<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/185030-dna-sequence-coding-for-a-protein-of-a-thaliana-having-a-delta-5-7-sterol-delta-7-reductase-activity-delta-7-red-protein-production-process-strains-of-transformed-yeasts-uses by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:58:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 185030:&quot;DNA SEQUENCE CODING FOR A PROTEIN OF A. THALIANA HAVING A DELTA-5,7 STEROL, DELTA-7 REDUCTASE ACTIVITY, DELTA 7-RED PROTEIN, PRODUCTION PROCESS, STRAINS OF TRANSFORMED YEASTS, USES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;DNA SEQUENCE CODING FOR A PROTEIN OF A. THALIANA HAVING A DELTA-5,7 STEROL, DELTA-7 REDUCTASE ACTIVITY, DELTA 7-RED PROTEIN, PRODUCTION PROCESS, STRAINS OF TRANSFORMED YEASTS, USES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A cDNA sequence coding for an A. thaliana protein having a delta-5,7 sterol, delta reductase activity (SEQ ID NO. 1). Cloning process. Reduction process for a sterol unsaturated in position C-7. Production process for pregnenolone.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DNA sequence coding for a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity, delta7-Red protein, production process, strains of transformed yeasts, uses.<br>
The present invention relates to a DNA sequence coding for a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity, delta7-Red protein, theirproduction process, strains of transformed yeasts, and their uses.<br>
Delta-5,7 sterol,delta-7 reductase (E.G.1.3.1.21) is a microsomal enzyme whose presence has been revealed by its activity in homogenates of rat's liver (M. E. Dempsey et al., Methods Enzymol., 15, 501-514, 1969) as well as in plant preparations of Zea mays (M. Taton et al., Biochem. Biophys. Res. Commun., 181, 465-473, 1991).  This reductase is dependent on NADPH and reduces 7-dehydrocholesterol to cholesterol in vitro.<br>
Sterols are the main constituents of the membranes in eucaryotes but show structural differences depending on the species.  In the cells of eucaryotes such as yeasts, the principal sterol is ergosterol which contains a double unsaturation in position C-5 and C-7, a branched side chain in position C-24 and an unsaturation in position C-22 while in mammals cholesterol is characterized by an unsaturation in position C-5 and a saturated side chain.  Sitosterol, stigmasterol and campesterol, which represent the most common sterols in plants, have a branched but saturated side chain and, as with the sterols of vertebrates, do not have an unsaturation in position C-7.  The enzyme responsible for this difference in structure of the sterol nucleus is delta-5,7 sterol,delta-7 reductase.<br>
Delta-5,7 sterol,delta-7 reductase has never been purified to homogeneity and only a partial purification has<br>
been described (M. E. Dempsey et al.; M. Taton et al., already quoted).  The protein has not been characterized by its physical-chemical properties.  No information on the sequence of the protein and no antibody directed against it have been described.  Furthermore, an apparent deficiency in 7-dehydrocholesterol reductase has been described in man associated with RSH/Smith-Lemli-Opitz (SLO) syndrome (J. M. Opitz et al., Am. J. Med. Genet., 50, 326-338, 1994).<br>
Cloning of a cDNA coding for delta-5,7 sterol,delta-7 reductase, which may enable the sequence of the corresponding protein to be identified as well as the characterization of the human gene or a congenital deficiency of it to be revealed, is not therefore achievable with known methods which make use for example of hybridization and/or immunological detection techniques.  The obtaining of inventive screening methods allowing cloning to be carried • out, in particular in the absence of information about the protein, is therefore particularly sought-after.<br>
Ergosterol, the principal sterol of the membranes of fungi, contains a pair of conjugated double bonds in position C-5,7 which confers an antimycotic activity on compounds of the family of polyenes such as nystatin (R. Bittman et al., J. Biol. Chem., 260, 2884-2889, 1985).  The strong dependency of the action of nystatin on the membrane concentration of unsaturated sterols in position C-5,7 has allowed the selection of mutant strains vis-a-vis accumulated sterols in S. cerevisiae (S. W. Molzahn et al., J. Gen.  Microbiol., 72, 339-348, 1972).  Thus the mutants erg2 and erg3 accumulate sterols which do not have conjugated double bonds in position C-5,7 because of a deficiency in sterol delta-8,7 isomerase (W. Ashman et al., Lipids, 26, 628, 1991) and in sterol delta-5 dehydrogenase (B. Arthinton et al., Gene, 102, 39, 1991) respectively.  Such mutants are viable because ergosterol, the principal natural sterol of yeast, can be replaced under certain conditions by different substitution sterols including cholesterol.<br>
The advantage of increasing the resistance to nystatin of yeast strains enriched with sterols not having the double<br>
unsaturation in position C-5,7 was perceived by the inventors for cloning a cDNA coding for a heterologous protein having a delta-5,7 sterol,delta-7 reductase activity with a screening method using a metabolic interference in S. cerevisiae.  The success of this approach which offers the advantage of being independent of knowing the DNA sequences or the protein as well as a detection solely based on the enzymatic activity, is not however foreseeable due to the many technical difficulties to be overcome.<br>
A first limitation comes from the fact that the way nystatin works is not entirely elucidated (L. W. Parks et al., CRC Critical Reviews in Microbiology, 301-304, 1978). For example, the low specificity of the selection by nystatin is foreseeable due to the indirect nature of it which leads in yeast to the selection of spontaneous genomic mutations such as the erg mutants (S. W. Molzahn et al. already quoted) or genomic mutations involving resistances independent of the sterol pathway.  Similarly, cells transformed by a gene library can express heterologous genes conferring a resistance to nystatin unrelated to the sterol pathway.<br>
Another example of the expected limitation relates to the fact that the heterologous protein could be weakly active in the cell, leading to the absence of or to a low resistance to nystatin for example for one of the following reasons: 1) the gene coding for delta-5,7 sterol,delta-7 reductase is weakly expressed; 2) the protein is weakly or not active because of poor folding or as a result of an incorrect subcellular orientation; 3) the plant protein does not recognise the yeast's own sterols as substrates or 4) the sterols which can be substrates are in the esterified form or stored in vesicles and are not in contact with the enzyme. It can therefore be anticipated that the sterols accumulated in this way escape the action of delta-5,7 sterol,delta-7 reductase which, in eucaryotes, is reputed to be localized in the microsomes.<br>
The present invention relates to the cloning of a cDNA of A. thaliana coding for a protein having a delta-5,7 sterol,delta-7 reductase activity, designated delta-7Red,<br>
according to a cloning process carried out in a yeast by metabolic interference based on the resistance to nystatin. The delta-VRed protein allows the sterols having an unsaturation in position C-7 to be reduced by a biological reduction process, which in addition provides an advantageous solution to the problem of the selective reduction in position C-7 of sterols or steroids having a double unsaturation in position C-5,7 which the reduction methods using chemical routes do not allow.<br>
The invention also relates to transformed yeast cells expressing the delta-7Red which, in a surprising manner, accumulate products saturated in position C-7 and optionally saturated in position C-22 which, contrary to ergosterol, are substrates of the restriction enzyme of the side chain of cholesterol, i.e. cytochrome P450SCC.<br>
These unexpected properties allow the use of the transformed yeasts of the invention in a preparation process for sterols or steroids having an industrial and/or pharmacological use, in particular the preparation of pregnenolone by biological oxidation of endogenic yeast sterols, reduced in position C-7, by cytochrome P45QSCC (P450SCC) in the presence of adrenodoxin reductase (ADR) and adrenodoxin (ADX).  The transformed yeasts of the invention can be also used in a preparation process for intermediate steroids of the metabolization route of cholesterol into hydrocortisone in mammals and other vertebrates.  This use has the advantage of allowing the preparation of hydrocortisone or its intermediates in a biological oxidation process which does not require the use of an exogenic sterol such as cholesterol as the initial substrate and thus bypassing the problem of the penetration of a sterol into the yeast whose impermeability to exogenic sterols under aerobiosis conditions has been described (R. T. Lorentz et al., J. Bacteriology, 981-985, 1986).<br>
The invention also relates to the use of nucleic sequences obtained using the cloning process of the invention.  An RNA or DNA sequence coding for delta-5,7 sterol,delta-7 reductase can be used for the diagnosis or the<br>
treatment of affections implicating the product of the gene of delta-5,7 sterol,delta-7 reductase.  For example, a deficiency in delta-5,7 sterol,delta-7 reductase which converts 7-dehydro cholesterol into cholesterol is presumed to be implicated in RSH/SLO syndrome.  Thus a human DNA sequence can be used as a probe for diagnosing a deficiency in delta-5,7 sterol,delta-7 reductase and can also be used in gene therapy to correct such a deficiency.<br>
Therefore a subject of the present invention is a nucleic acid sequence containing a sequence coding for a protein having a delta-5,7 sterol,delta-7 reductase activity, said nucleic acid being a DNA or an RNA and in particular being a cDNA.<br>
The delta-5,7 sterol,delta-7 reductase activity can be revealed for example using an enzymatic test in vitro described further on in the experimental part.<br>
A more particular subject of the invention is a cDNA sequence in which the coding sequence codes for a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity and having the nucleotide sequence of the sequence SEQ ID No. 1:<br>
GTGTGAGTAA TTTAGGTCAA CACAGATCAG AATCTGAGGC TTTGGCCGAG ACGAAGAGAA     60<br>
AAGCAGAAGA AGAAA ATG GCG GAG ACT GTA CAT TCT CCG ATC GTT ACT TAG      111<br>
Met Ala Glu Thr Val His Ser Pro lie Val Thr Tyr<br>
15	10<br>
GCA TCG ATG TTA TCG CTT CTC GCC TTC TGT CCA CCT TTC GTC ATT CTC       159<br>
Ala Ser Met Leu Ser Leu Leu Ala Phe Cys Pro Pro Phe Val lie Leu<br>
15	20	25<br>
CTA TGG TAG ACA ATG GTT CAT CAG GAT GGT TCT GTT ACT CAG ACC TTT       207<br>
Leu Trp Tyr Thr Met Val His Gin Asp Gly Ser Val Thr Gin Thr Phe<br>
30	35	40<br>
GGC TTC TTT TGG GAG AAT GGA GTT CAA GGA CTT ATC AAC ATA TGG CCA       255<br>
Gly Phe Phe Trp Glu Asn Gly Val Gin Gly Leu lie Asn lie Trp Pro<br>
45	50	55	60<br><br>
AGA CCC ACT TTG ATT GCT TGG AAA ATT ATA TTT TGC TAT GGA GCA TTT<br>
Arg Pro Thr Leu lie Ala Trp Lya lie lie Phe Cys Tyr Gly Ala Phe<br>
65	70	75<br><br>
303<br><br><br><br>
GAA GCT ATT CTT CAG CTG CTT CTG CCT GGT AAA AGA GTT GAG GGT CCA<br>
Glu Ala lie Leu Gin Leu Leu Leu Pro Gly Lys Arg Val Glu Gly Pro<br>
80	85	90<br><br>
351<br><br><br><br>
ATA TCT CCA GCC GGA AAC CGA CCA GTT TAC AAG GCC AAT GGT CTG GCT<br>
He Ser Pro Ala Gly Asn Arg Pro Val Tyr Lys Ala Asn Gly Leu Ala<br>
95	100	105<br><br>
399<br><br><br><br>
GCT TAC TTT GTG ACA CTA GCA ACC CAT CTT GGT CTT TGG TGG TTT GGA<br>
Ala Tyr Phe Val Thr Leu Ala Thr His Leu Gly Leu Trp Trp Phe Gly<br>
110	115	120<br><br>
447<br><br><br><br>
ATC TTC AAC CCT GCA ATT GTC TAT GAT CAC TTG GGT GAA ATA TTT TCG<br>
He Phe Asn Pro Ala He Val Tyr Asp His Leu Gly Glu He Phe Ser<br>
125	130	135	140<br><br>
495<br><br><br><br>
GCA CTA ATA TTC GGA AGC TTC ATA TTT TGT GTT TTG TTG TAC ATA AAA<br>
Ala Leu He Phe Gly Ser Phe He Phe Cys Val Leu Leu Tyr He Lys<br>
145	150	155<br><br>
543<br><br><br><br>
GGC CAT GTT GCA CCT TCA TCA ACT GAC TCT GGT TCA TGT GGT AAC CTA<br>
Gly His Val Ala Pro Ser Ser Ser Asp Ser Gly Ser Cys Gly Asn Leu<br>
160	165	170<br><br>
591<br><br><br><br>
ATA ATT GAC TTC TAT TGG GGC ATG GAG TTG TAC CCT CGA ATT GGT AAG<br>
He He Asp Phe Tyr Trp Gly Met Glu Leu Tyr Pro Arg He Gly Lys<br>
175	180	185<br><br>
639<br><br><br><br>
AGC TTT GAC ATC AAG GTG TTT ACT AAT TGC AGA TTC GGA ATG ATG TCT<br>
Ser Phe Asp He Lys Val Phe Thr Asn Cys Arg Phe Gly Met Met Ser<br>
190	195	200<br><br>
687<br><br><br><br>
TGG GCA GTT CTT GCA GTC ACG TAC TGC ATA AAA CAG TAT GAA ATA AAT<br>
Trp Ala Val Leu Ala Val Thr Tyr Cys He Lys Gin Tyr Glu He Asn<br>
205	210	215	220<br><br>
735<br><br>
GGC AAA GTA TCT GAT TCA ATG CTG GTG AAC ACC ATC CTG ATG CTG GTG       783<br>
Gly Lys Val Ser Asp Ser Met Leu Val Asn Thr lie Leu Met Leu Val<br>
225	230	235<br>
TAT GTC ACA AAA TTC TTC TGG TGG GAA GCT GGT TAT TGG AAC ACC ATG      831<br>
Tyr Val Thr Lya Phe Phe Trp Trp Glu Ala Gly Tyr Trp Asn Thr Met<br>
240	245	250<br>
GAC ATT GCA CAT GAC CGA GCT GGA TTC TAT ATA TGC TGG GGT TGT CTA      879<br>
Asp lie Ala His Asp Arg Ala Gly Phe Tyr lie Cys Trp Gly Cys Leu<br>
255	260	265<br>
GTG TGG GTG CCT TCT GTC TAG ACT TCT CCA GGC ATG TAC CTT GTG AAC       927<br>
Val Trp Val Pro Ser Val Tyr Thr Ser Pro Gly Met Tyr Leu Val Asn<br>
270	275	280<br>
CAC CCC GTC GAA CTC GGA ACT GAG TTG GCA ATA TAC ATT CTC GTT GCA       975<br>
His Pro Val Glu Leu Gly Thr Gin Leu Ala lie Tyr lie Leu Val Ala<br>
285	290	295	300<br>
GGA ATT CTG TGC ATT TAC ATA AAG TAT GAC TGT GAT AGA CAA AGG CAA      1023<br>
Gly lie Leu Cys lie Tyr lie Lys Tyr Asp Cys Asp Arg Gin Arg Gin<br>
305	310	315<br>
GAG TTC AGG AGG ACA AAC GGG AAA TGT TTG GTT TGG GGA AGA GCC CCG      1071<br>
Glu Phe Arg Arg Thr Asn Gly Lys Cys Leu Val Trp Gly Arg Ala Pro<br>
320	325	330<br>
TCA AAG ATT GTG GCG TCG TAT ACT ACA ACA TCT GGT GAA ACT AAA ACT      1119<br>
Ser Lys lie Val Ala Ser Tyr Thr Thr Thr Ser Gly Glu Thr Lys Thr<br>
335	340	345<br>
ACT CTT CTC TTA ACG TCT GGA TGG TGG GGA TTG GCT CGT CAT TTC CAT      1167<br>
Ser Leu Leu Leu Thr Ser Gly Trp Trp Gly Leu Ala Arg His Phe His<br>
350	355	360<br>
TAT GTT CCT GAG ATC TTA ACT GCT TTC TTC TGG ACC GTA CCG GCT CTC      1215<br>
Tyr Val Pro Glu lie Leu Ser Ala Phe Phe Trp Thr Val Pro Ala Leu<br>
365	370	375	380<br>
TTC GAT AAC TTC TTG GCA TAC TTC TAC GTC CTC ACC CTT CTT CTC TTT      1263<br>
Phe Asp Asn Phe Leu Ala Tyr Phe Tyr Val Leu Thr Leu Leu Leu Phe<br>
385	390	395<br><br>
GAT CGA GCC AAG AGA GAC GAT GAG CGA TGC CGA TCA AAG TAT GGG AAA      1311<br>
Asp Arg Ala Lys Arg Asp Asp Asp Arg Cys Arg Ser Lys Tyr Gly Lys<br>
400	405	410<br>
TAT TGG AAG CTG TAT TGT GAG AAA GTC AAA TAC AGG ATC ATT CCG GGA      1359<br>
Tyr Trp Lys Leu Tyr Cys Glu Lys Val Lys Tyr Arg lie lie Pro Gly<br>
415	420	425<br>
ATT TAT TGATTGTAAC GAAGTCTGTT GTTCTCATTT TCTACTTATT ACGTTAATTC       1415 lie Tyr 430<br>
GAACGTTGGA ATCATCAAAA GACCGAGCCA AAACAAAAAT GCAAATTGAT GCGATAGACA    1475<br>
TTCTTTTGCT GAAAAAAAAA A	1496<br>
as well as the allelic variants of this sequence.<br>
The above cDNA sequence which codes for a protein having 430 amino acids corresponding to the sequence shown in Figure 3 (SEQ ID No. 2) is a cDNA sequence which can be obtained for example by cloning in a yeast, starting from an A. thaliana expression library, by using a screening method based on the appearance of a resistance of the yeast to nystatin, according to operating conditions a detailed description of which is given further on.<br>
Knowledge of the nucleotide sequence SEQ ID No. 1 above allows the present invention to be reproduced by methods known to a man skilled in the art, for example by chemical synthesis or by screening a gene library or a cDNA library using synthetic oligonucleotide probes by hybridization techniques or by PCR amplification.<br>
Also a subject of the invention is a DNA sequence coding for a protein having a delta-5,7 sterol,delta-7 reductase activity and which hybridizes with the nucleotide sequence SEQ ID No. 1 under conditions of average or high stringency or which have a sequence identity of approximately 60% and more with this sequence.<br>
The sequences which hybridize in a detectable manner with sequence SEQ ID No. 1 hybridize under standard conditions of average stringency, for example a hybridization<br>
at 42°C, for 12 hours in a 50% solution of formamide, SSCX6 followed by washings or under less stringent conditions, for example a hybridization at 42°C for 24 hours in a 20% solution of formamide, SSCX6 followed by washings under known standard conditions (T. Maniatis et al., Molecular cloning, Cold Spring Harbor Laboratory Press, 1989).<br>
The percentage of nucleotide sequence identity can be determined by using for example the BLAST program "basic local alignment search tool" (S. F. Altschul et al., J. Mol. Biol., 215, 403-410, 1990) on the NCBI server.<br>
The invention also relates to a DNA sequence coding for a protein having a delta-5,7 sterol,delta-7 reductase activity and amplifiable by the PCR technique using as primers the oligonucleotides coding for a consensus sequence having the amino acid sequence SEQ ID No. 3:<br>
Leu Leu Xaa Xaa Gly Trp Xaa Gly Xaa Xaa Arg Xaa Xaa Xaa Tyr<br>
15	10	15<br>
in which Xaa in position 7 is Trp or Tyr and Xaa in position 12 is His or Lys.<br>
The above sequence SEQ ID No. 3 corresponds to a new consensus sequence which has been defined by alignment of the identity of amino acid sequences between the new sequence SEQ ID No. 2, deduced from the nucleotide sequence SEQ ID No. 1, and known sequences either of other sterol reductases having a specificity of action at a position other than position C-7, or of lamin B receptors, as is detailed further on in the experimental part.  From the information given by the amino acid sequence SEQ ID No. 3, a primer constituted by at most 45 nucleotides can be defined and synthesized which, combined with a second primer oligodT (17 nucleotides) as rebound sequence, will allow using a commercially-available PCR assay kit (for example Stratagene) the amplification of a DNA coding for a protein having delta-5,7 sterol,delta-7 reductase activity.<br>
The invention also relates to a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity and<br>
having the amino acid sequence SEQ ID No. 2:<br>
Met Ala Glu Thr Val His Ser Pro lie Val Thr Tyr Ala Ser Met Leu<br>
15	10	15<br>
Ser Leu Leu Ala Phe Cys Pro Pro Phe Val He Leu Leu Trp Tyr Thr<br>
20	25	30<br>
Met Val His Gin Asp Gly Ser Val Thr Gin Thr Phe Gly Phe Phe Trp<br>
35	40	45<br>
Glu Asn Gly Val Gin Gly Leu He Asn He Trp Pro Arg Pro Thr Leu<br>
50	55	60<br>
He Ala Trp Lys He He Phe Cys Tyr Gly Ala Phe Glu Ala He Leu<br>
65	70	75	80<br>
Gin Leu Leu Leu Pro Gly Lys Arg Val Glu Gly Pro lie Ser Pro Ala<br>
85	90	95<br>
Gly Asn Arg Pro Val Tyr Lys Ala Asn Gly Leu Ala Ala Tyr Phe Val<br>
100	105	110<br>
Thr Leu Ala Thr His Leu Gly Leu Trp Trp Phe Gly He Phe Asn Pro<br>
115	120	125<br>
Ala He Val Tyr Asp His Leu Gly Glu He Phe Ser Ala Leu He Phe<br>
130	135	140<br>
Gly Ser Phe He Phe Cys Val Leu Leu Tyr He Lys Gly His Val Ala<br>
145	150	155	160<br>
Pro Ser Ser Ser Asp Ser Gly Ser Cys Gly Asn Leu lie He Asp Phe<br>
165	170	175<br>
Tyr Trp Gly Met Glu Leu Tyr Pro Arg He Gly Lys Ser Phe Asp He<br>
180	185	190<br>
Lys Val Phe Thr Asn Cys Arg Phe Gly Met Met Ser Trp Ala Val Leu<br>
195	200	205<br>
Ala Val Thr Tyr Cys He Lys Gin Tyr Glu He Asn Gly Lys Val Ser<br>
210	215	220<br>
Asp Ser Met Leu Val Asn Thr lie Leu Met Leu Val Tyr Val Thr Lys<br>
225	230	235	240<br>
Phe Phe Trp Trp Glu Ala Gly Tyr Trp Asn Thr Met Asp lie Ala His<br>
245	250	255<br>
Asp Arg Ala Gly Phe Tyr lie Cys Trp Gly Cys Leu Val Trp Val Pro<br>
260	265	270<br>
Ser Val Tyr Thr Ser Pro Gly Met Tyr Leu Val Asn His Pro Val Glu<br>
275	280	285<br>
Leu Gly Thr Gin Leu Ala lie Tyr lie Leu Val Ala Gly lie Leu Cys<br>
290	295	300<br>
lie Tyr lie Lys Tyr Asp Cys Asp Arg Gin Arg Gin Glu Phe Arg Arg<br>
305	310	315	320<br>
Thr Asn Gly Lys Cys Leu Val Trp Gly Arg Ala Pro Ser Lys lie Val<br>
325	330	335<br>
Ala Ser Tyr Thr Thr Thr Ser Gly Glu Thr Lys Thr Ser Leu Leu Leu<br>
340	345	350<br>
Thr Ser Gly Trp Trp Gly Leu Ala Arg His Phe His Tyr Val Pro Glu<br>
355	360	365<br>
lie Leu Ser Ala Phe Phe Trp Thr Val Pro Ala Leu Phe Asp Asn Phe<br>
370	375	380<br>
Leu Ala Tyr Phe Tyr Val Leu Thr Leu Leu Leu Phe Asp Arg Ala Lys<br>
385	390	395	400<br>
Arg Asp Asp Asp Arg Cys Arg Ser Lys Tyr Gly Lys Tyr Trp Lys Leu<br>
405	410	415<br>
Tyr Cys Glu Lys Val Lys Tyr Arg lie lie Pro Gly lie Tyr<br>
420	425	430<br>
as well as the allelic variants and analogues of this sequence.<br>
By alleles and analogues, are included sequences modified by substitution, deletion or addition of one or more amino acids as long as these products retain the same<br>
function.  The modified sequences can be for example prepared by using the site directed mutagenesis technique known to a man skilled in the art.<br>
A particular subject of the invention is a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity and having the amino acid sequence SEQ ID No. 2 above and designated delta-7Red.<br>
The invention also relates to a protein having a delta-5,7 sterol,delta-7 reductase activity having an amino acid sequence having an identity of sequence of approximately 60% and more with the sequence SEQ ID No. 2.<br>
The percentage of identity can be determined for exampleby using the BLAST program indicated above.<br>
The invention also relates to a protein having a delta-5,7 sterol,delta-7 reductase activity and having a crossed immunological reactivity with the protein of A. thaliana delta-7Red defined above.<br>
The protein can be detected for example by immuno-precipitation using an anti-serum directed against the delta-7Red protein, prepared according to known methods.<br>
One of the aspects of the invention relates to a protein having a delta 5,-7 sterol,delta-7 reductase activity such as that obtained by expression in a host cell containing a DNA sequence defined previously and in particular relates to a protein of A. thaliana such as that obtained by expression in a host cell containing a DNA sequence coding for the amino acid sequence SEQ ID No. 2 above.<br>
When the protein of the invention is obtained by expression in a host cell, this is carried out according to genetic engineering and cell culture methods known to a man skilled in the art.<br>
Expression can be carried out in a procaryotic host cell, for example E. coli or in a eucaryotic host cell, for example a mammalian cell, an insect cell or a yeast containing the sequence coding for delta-7 Red of the invention preceded by a suitable promoter.<br>
The recombinant protein obtained can be glycosylated or nonglycosylated.<br>
In particular the invention relates to a protein according to the invention such as that obtained by expression in a yeast.<br>
The invention also relates to an antibody directed against a protein having a delta-5,7 sterol,delta-7 reductase activity defined previously.  The antibody can be a polyclonal antibody or a monoclonal antibody prepared according to methods known to a man skilled in the art.<br>
The invention also relates to an expression vector containing a DNA sequence defined previously as well as to a host cell transformed by this vector.<br>
The expression vectors are known vectors allowing the expression of the protein under the control of a suitable promoter.  For the procaryotic cells, the promoter can be for example the lac promoter, the trp promoter, the tac promoter, the 6-lactamase promoter or the PL promoter.  For mammalian-cells, the promoter can be the SV40 promoter or the promoters of the adeno-virus.  Vectors of Baculovirus type can also be used for expression in insect cells.  For yeast cells, the promoter can be for example the PGK promoter, the ADH promoter, the CYC1 promoter or the GAL10/CYC1 promoter.<br>
The host cells can be procaryotic cells or eucaryotic cells.  Procaryotic cells are for example E. coli, Bacillus or Streptomyces.  Eucaryotic host cells include yeasts and filamentous fungi as well as cells of higher organisms, for example cells of mammals or cells of insects.  The mammalian cells can be hamster CHO cells or monkey Cos cells.  The insect cells are for example SF9 cells.  The yeast cells can be for example Saccharomyces cerevisiae, Schizosaccharomyces pombe or Kluyveromyces lactis.<br>
Also a subject of the invention is a cloning process for a nucleic acid coding for a protein having a delta-5,7 sterol,delta-7 reductase activity in a microorganism comprising a screening method chosen from<br>
-	the resistance of the microorganism to nystatin or to an<br>
analogous compound whose toxicity depends on the presence of<br>
sterols carrying an unsaturation in position C-7,<br>
-	the hybridization of the nucleic acid with the nucleotide<br>
sequence of the sequence SEQ ID No. 1 above,<br>
-	the identification of the nucleic acid by using data<br>
processing techniques from DNA sequences isolated at random,<br>
-	the direct expression of the protein followed by<br>
immunodetection using antibodies directed against the protein<br>
having the amino acid sequence SEQ ID No. 2 above.<br>
By microorganism is meant a yeast such as for example S. cerevisiae, S. pombe or K. lactis.<br>
The compounds analogous to nystatin include for example amphotericin B or filipin.<br>
By hybridization is meant a hybridization under average or high stringency conditions according to known standard conditions (T. Maniatis et al., already quoted).<br>
Identification using data-processing techniques can be carried out for example according to the BLAST program indicated above.<br>
An example of the above cloning process in which the screening method uses the resistance to nystatin in a yeast is described further on in the experimental part.<br>
A subject of the invention is also a DNA or RNA nucleic acid sequence, as obtained by the above cloning process.  The nucleic acid sequence can be of procaryotic or eucaryotic origin according to the material from which the cloning is carried out, for example of human origin.<br>
A subject of the invention is also a host cell transformed by a vector containing a DNA sequence as obtained by the above cloning process.  The host cell can be a procaryotic cell or a eucaryotic cell.  Examples of host cells and vectors have been indicated previously.<br>
A particular subject of the invention is a host cell, chosen from a yeast or a filamentous fungus, transformed by a vector containing a DNA sequence according to the invention defined previously or a DNA sequence such as that obtained by the above cloning process.<br>
One of the subjects of the invention also relates to a preparation process for a protein having a delta-5,7 sterol, delta-7 reductase activity in which a transformed host cell according to the invention is cultivated and the expressed<br>
protein is isolated and particularly relates to a process in which the host cell is a transformed yeast in which the coding DNA sequence is placed under the control of a yeast promoter.<br>
One of the subjects of the invention also relates to a reduction process in vitro of a sterol unsaturated in position C-7 in which the sterol to be reduced is incubated with the protein obtained by the above process and the reduced sterol obtained is optionally isolated.<br>
One of the subjects of the invention also relates to a reduction process in vivo of an exogenic sterol unsaturated in position C-7 in which the sterol is incubated with a transformed host cell according to the invention and the reduced sterol obtained is optionally isolated.  The host cell can be a procaryotic cell or a eucaryotic cell, in particular a yeast or a filamentous fungus.<br>
One of the subjects of the invention also relates to a reduction process in vivo of an endogenic sterol unsaturated in position C-7 in which a transformed host strain according to the invention chosen from a yeast or a filamentous fungus is cultivated and the accumulated reduced sterol is optionally isolated.<br>
In particular the invention relates to a reduction process in vitro or in vivo defined above in which the reduced sterol obtained is a substrate of the restriction enzyme of the side chain of cholesterol (P45QSCC) and quite particularly relates to a reduction process in vivo in which the endogenic sterol to be reduced is ergosta 5,7 diene 3-ol, ergosta 5,7,24(28) triene 3-ol or ergosta 5,7,22 triene 3-ol or a mixture of these.<br>
A subject of the invention is also a production process for pregnenolone in which a transformed host cell according to the invention chosen from a yeast or a filamentous fungus is cultured, the accumulated endogenic sterol or sterols reduced in position C-7 are optionally isolated, the reduced sterols are incubated in the presence of P450SCC, and optionally in the presence of adrenodoxin reductase (ADR) and adrenodoxin (ADX), and the pregnenolone obtained is<br>
optionally isolated.<br>
A particular subject of the invention is a production process for pregnenolone defined above in which the host cell is a yeast.<br>
The invention also relates to a production process for pregnenolone in which a yeast transformed by one or more vectors allowing the coexpression of a protein having the activity of delta-5,7 sterol,delta-7 reductase and P45QSCC and optionally of ADR and ADX is cultured and the free or esterified pregnenolone is optionally isolated.<br>
The invention particularly relates to the above process in which a transformed yeast coexpressing a protein having the activity of delta-5,7 sterol delta-7 reductase, P450SCC, ADR and ADX is cultured, quite particularly the above process in which the protein having the delta-5,7 sterol delta-7 reductase activity is the protein of A. thaliana delta-7Red . and especially the above process in which the yeast strain is the strain EC01/pCD63.<br>
Examples of the production of pregnenolone according to the invention are given further on in the experimental part.<br>
The transformed yeast used for carrying out the production process for pregnenolone above can be for example a yeast co-transformed by an expression vector of the invention containing a DNA sequence coding for a protein having a delta-5,7 sterol,delta-7 reductase activity and an expression vector of the cytochrome P450SCC and optionally of ADX and ADR.  The expression vectors of the cytochrome P450SCC and/or ADX are known and preparations are described for example in the European Patent Application EP 0340878 as well as further on in the experimental part.<br>
The transformed yeast used can also be a yeast in which the DNA sequence coding for the protein having delta-5,7 sterol,delta-7 reductase activity is integrated in a particular locus of the genome, for example at locus ADE2 and in which the ergosterol is no longer the majority sterol under the expression conditions of delta-7 reductase.  The "integrated" yeast obtained can then be transformed by integrative expression cassettes or by expression vectors<br>
containing a DNA sequence coding for the cytochrome P450SCC and optionally for ADX and ADR.<br>
An example of the construction of yeast strains producing pregnenolone or its acetic ester in vivo is given further on in the experimental part.<br>
Therefore a subject of the invention is also a transformed yeast strain coexpressing a protein having the activity of delta-5,7 sterol delta-7 reductase, P450SCC, ADR and ADX and accumulating free or esterified pregnenolone, particularly an above yeast strain in which the protein having the activity of delta-5,7 sterol delta-7 reductase is the protein of A. thaliana delta-7Red and especially a yeast strain called EC01/pCD63 the precise construction of which is given further on in the experimental part.<br>
The invention also extends to a human DNA sequence as obtained according to the cloning process defined above, used as a probe to diagnose a congenital deficiency in delta-5,7 sterol,delta-7 reductase.  The deficiency in delta-5,7 sterol,delta-7 reductase includes for example the deficiency in 7-dehydrocholesterol reductase responsible for the abnormally low levels of cholesterol in the plasma.<br>
The invention also relates to a detection method for the deficiency in delta-5,7 sterol,delta-7 reductase which includes the incubation of a sample containing human genomic DNA with the probe defined above under standard hybridization conditions and the revealing of the fixation or the absence of fixation of the probe to the genomic DNA, the absence of fixation or the reduction of the latter indicating a congenital deficiency in delta-5,7 sterol,delta-7 reductase.<br>
The method of the invention can allow for example the detection of a congenital deficiency in prenatal delta-5,7 sterol, delta-7 reductase or in new-born babies as well as in patients suffering from various illnesses and in particular in patients having clinical manifestations of RSH/SLO syndrome.<br>
The attached figures illustrate certain aspects of the invention:<br>
Figure 1 represents the profile in total sterols<br>
extracted from the clone F22 resistant to nystatin obtained by screening of the A. thaliana library in the yeast FY 1679. Analysis is carried out by RP-HPLC at 205 nm or at 285 nm (Figure 1A) or by GC compared with the non-transformed yeast FY1679 (Figure IB).<br>
Figure 2 represents a restriction map of the Not I fragment of the plasmid pF22 and the sequencing strategy.<br>
Figure 3 represents the nucleotide sequence of the cDNA delta-7Red (SEQ ID No. 1) and the corresponding amino acid sequence (SEQ ID No. 2).<br>
Figure 4 illustrates the measurement in vitro of the delta-5,7 sterol,delta-7 reductase activity of a microsomal fraction of FY1679 transformed with the inducible expression vector "delta-7/V8".  Analysis is carried out with GC and shows the transformation of the substrate 7-dehydrocholesterol (RT=16.528) into cholesterol (RT=15.887) -in the presence of the endogenic sterols 5,22 diene 3-ol (RT=16.682); ergosterol (RT = 17.249); ergosta 5-ene3-ol (RT=17.664).<br>
Figure 5 illustrates the production of pregnenolone in vitro by restriction of ergosta 5-ene 3-ol (Fig. 5A); ergosta 5,24(28) diene 3-ol (5B) or ergosta 5,22 diene 3-ol (5C) purified from a transformed yeast expressing delta-7Red, then incubated with P450SCC, ADX and ADR.  Analysis is carried out by GC compared with a control restriction of cholesterol (full line).<br>
Figure 6 illustrates the construction of the integrative plasmid pADdelta-7 allowing the integration of the cDNA coding for the delta-7Red (sterol delta-7 reductase) at locus ADE2.<br>
Figure 7 represents the analysis by GC of the total sterols extracted by alkaline saponification of the strain FY1679 and the integrated strain ELR01 cultured in the presence of galactose.<br>
Figure 8 schematizes the construction of the shuttle vector E. coli-S. cerevisiae V13 containing a single Sail (Sa) site in the multiple cloning site.<br>
Figure 9 represents the stages of construction of the<br>
integrative plasmids pCD62-l and pCD62-2 which allow the ADR expression cassette to be inserted in the intergenic region of genes Ieu2 and splldelta. MCS1 and MCS2:  multiple cloning sites.<br>
Figure 10 represents the construction strategy of the strain CDR01.<br>
Figure 11 represents stages of construction of the expression plasmid pCD63 containing the two expression cassettes ADX and P450SCC.<br>
Figure 12 represents the analysis by GC of the sterols extracted from cells (cellular lysate) or culture medium (medium) isolated from the strain EC01/pCD63 after an induction by galactose for 9 hours (Figure 12a) or 24 hours (Figure 12b).<br>
Figure 13 represents the structure of plasmid pTG 7457. Figure 14 represents the structure of plasmid pTG 7453. Figure 15 represents the structure of plasmid pTG 10014. Figure 16 represents the structure of plasmid pTG 10004. Figure 17 represents the structure of plasmid pTG 10031. Figure 18 represents the structure of plasmid pTG 10033.<br>
The following examples illustrate the invention without limiting it.<br>
EXAMPLE 1; Cloning of the cDNA coding for delta-5,7 sterol,delta-7 reductase (delta-7Red) of A. thaliana A - Screening of the expression library of A. thaliana in yeast<br>
The starting cDNA expression library is the library described by M. Minet et al. (Plant J., 2, 417-422, 1992) which was prepared from mRNA of A. thaliana at the two-leaf germination stage and whose cDNA's bordered by the Not I site have been inserted at the Bst XI site of the expression cassette of the shuttle vector E. coli/S. cerevisiae pFL61. This cassette contains the promoter and terminator sequences of the phosphoglycerate kinase gene (PGK).  The origin of replication of the yeast derived from the 2u sequence and a selection indicator URA3 ensure the propagation of the vector in the yeast.  The propagation of the vector in E. coli is derived from the plasmid pUC19.<br>
The yeast strain FY 1679 (Mata), which is an isogenic strain of the strain S288C described by A. Thierry et al. (Yeast, 6, 521-534, 1990), was transformed by the cDNA library using the lithium acetate method described by D. Gietz et al. (Nucleic Acids Res., 20, 1425, 1992).<br>
The cells were plated on a synthetic medium SGI containing 7 g/1 of "yeast nitrogen base" (Difco), 1 g/1 of bactocasaminoacids (Difco), 20 g/1 of glucose, 20 mg/1 of tryptophan and which is free from uracil.  105 prototrophic transformants for uracil were obtained then grouped and again plated on the same synthetic medium free from uracil and containing 2 or 5 /zg/ml of nystatin at the rate of 5xl04 cells per dish. 106 cells for each nystatin concentration were screened in this way.  After incubation for 3 days at 28 °C, approximately 100 clones, which have grown at a concentration of 2 ng/ml of nystatin, were collected by constituting groups of 5 clones whose sterol composition was analyzed by reversed-phase high performance liquid chromatography (called RP-HPLC in what follows) while a single resistant clone, called F22, was obtained at a concentration of 5 /xg/ml of nystatin. B - Analysis of the sterols accumulated in the clone F22<br>
The total sterols of the yeast are prepared according to the alkaline saponification method described by L. Parks et al. (Methods in Enzymol., Ill, 333-346, 1985), then analyzed by RP-HPLC and/or by gas chromatography (called GC).<br>
The residue of sterols obtained is dissolved in an ethanol-tetrahydrofuran-water mixture (65:10:25 v/v) then analyzed by RP-HPLC on an Applied Biosystems CIS bonded silica column (100 x 2.1 mm) at a flow rate of 1 ml/min and at 55°C using a linear gradient of methanol in water (50% to 100% over 18 min) and a photometric detection at 205 nm and 285 nm compared with the ergosterol, campesterol and cholesterol standards.<br>
The composition of sterols is also analyzed by GC on an Alltech SE-30 capillary column (30 m x 0.32 mm) with helium as carrier gas, a temperature of 280°C and 310°C for the injector and the detector respectively, with an initial<br>
increase in the temperature of 110°C to 245°C at a speed of 45°C/min, then of 3°C/min in order to reach 280°C.<br>
The analysis by RP-HPLC (Figure 1A) and the analysis by GC (Figure IB) show the profile of the accumulated sterols in the clone F22 obtained above characterized by the virtually complete disappearance of ergosterol, majority sterol of the non-transformed strain FY 1679, and its replacement by two majors sterols and in similar quantity which do not absorb at 285 nm and thus no longer have a conjugated double unsaturation according to analysis by RP-HPLC.<br>
In Figure 1A, the campesterol (Sigma) (24-R-ergosta 5-ene 3-ol) contains approximately 35% of dihydrobrassicasterol (24-S-ergosta 5-ene 3-ol). c - Cloning of the delta-7Red cDNA<br>
The plasmids originating from the clone F22 were amplified in E. coli according to the method described by J.-N. Strathern et al. (Methods in Enzymol., 194, 319-329, 1991), then digested by Not I.  A fragment of about 600 base pairs (bp) and a fragment of 1.6 kbp were obtained.  The strain FY1679 was transformed with each of the above fragments respectively.  The composition in sterols of each clone of the transformed yeast was analyzed as indicated above and allowed the plasmid carrying the gene responsible for the modified profile in sterols to be distinguished.  The plasmid identified in this way was named pF22. D - Determination of the cDNA sequence of delta-7Red<br>
The cDNA insert of pF22 was sub-cloned at the Not I site of the vector pUC9N derived from pUC9 (Pharmacia) in which the Eco RI site of the multiple cloning site is replaced by the insertion of a Not I restriction site while retainingthe reading frame of the LacZ gene.  A restriction map was then determined (Figure 2).<br>
Restriction fragments having the Not I external site and the EcoRI, PvuII or Hindlll internal sites respectively were sub-cloned in the pBluescript plasmid (Stratagene).  The nucleotide sequence was determined by the Sanger method with DNA polymerase Sequenase (Stratagene kit) on the two strands by using direct and inverse primers of pBluescript pUC9, T3<br>
and T7 or deduced specific primers of the cDNA nucleotide sequence.<br>
The compilation of all of the sequences obtained gives the delta-7Red cDNA nucleotide sequence of A. thaliana (SEQ ID No. 1) represented in Figure 3.  It comprises 1496 nucleotides ending with a polyadenylation sequence.  It has an open reading frame starting with a methionine initiator at nucleotide 76 and ending with a termination codon at nucleotide 1366.  This results in an open reading frame of 1290 nucleotides coding for a protein of 430 amino acids. The coding region of the delta-7Red cDNA codes for the protein delta-7Red the deduced amino acid sequence of which (SEQ ID No. 2) is shown in Figure 3.  The sequence of the protein includes 430 amino acids amines having a calculated molecular mass of 49.286 kDa.<br>
A sample of the E. coli DH5-1 strain containing delta--7Red cDNA in the vector pUC9N (designated delta7red/pUC9N cDNA) was deposited at the CNCM on 10th February 1995 under the number 1-1535.<br>
E - Determination of the consensus sequence SEQ ID No. 3 Using computerized searching of sequence databases (Genbank and EMBL), it has been shown that the sequence of the delta-7Red protein of A. thaliana has some sequence similarities with other sterol reductases, in particular sterol C-14 reductase and sterol C-24(28) reductase of S. cerevisiae described by R. T. Lorentz et al. (DNA Cell Biol., 11, 685-692, 1992) and W. Chen et al. (Yeast, 7, 305-308, 1991) respectively as well as the product of the sts 1+ gene of S. pombe described by M. Shimanuki et al. (Mol. Biol. Cell, 3, 263-273, 1992) and sterol C-14 reductase of Neurospora crassa (No. X77955 in the EMBL database).  In addition, the delta-7Red protein shows a similarity with the 400 amino acids of the C-terminal end of the chicken lamine B receptor and with that of the corresponding human receptor described by H. J. Worman et al. (J. Cell Biol., Ill, 1535-1542, 1990) and E. Schuler et al. (J. Biol. Chem., 269, 11312-11317, 1994).<br>
Sequence identity alignments were established between<br>
the amino acid sequence SEQ ID No.   2  deduced from the delta-7Red cDNA obtained above and those of the three yeast sterol reductases and the two lamine B receptors,   then a new consensus sequence having the amino acid sequence   (SEQ ID No. 3):<br>
Leu Leu Xaa Xaa Gly Trp Xaa Gly Xaa Xaa Arg Xaa Xaa Xaa Tyr<br>
15	10	15<br>
in which Xaa in position 7 is Trp or Tyr and Xaa in position<br>
12 is His or Lys,<br>
was defined in order to prepare oligonucleotides which can be<br>
used as primers to amplify, by PCR, new genomic DNA or cDNA<br>
sequences coding for a protein having a delta-5,7 sterol,<br>
delta-7 reductase activity.<br>
F - Expression of the delta-7Red protein in yeast.<br>
a) construction of the expression vector delta-7/V8 inducible<br>
in yeast<br>
The deletion of the non-coding regions of cDNA of pF22 was carried out by the PCR amplification using the following specific oligonucleotides:<br>
5'CGCGGATCCA TGGCGGAGAC TGTACATTC 3' (SEQ ID No. 4) and 5'CAGGGTACCT CAATAAATTC CCGGAATG 3' (SEQ ID No. 5)<br>
which were defined in order to introduce a BamH I restriction site immediately upstream of the initiation codon and a Kpn I site immediately downstream of the stop codon.<br>
The cDNA was amplified starting from 1 ng of "delta7red/pUC9N cDNA" plasmid in the presence of 2 units of Pfu DNA polymerase and 0.2 uM of each of the above primers by using the following amplification conditions: 94°C, 10s; 52°C, 50s; 74°C, 90s; 33 cycles with the Stratagene PCR kit.<br>
A fragment of approximately 1300 bp was obtained, then digested by the restriction enzymes BamH I and Kpn I and inserted into the BamH I and Kpn I sites of the E. coli/S. serevisiae pYeDPl/8-2 shuttle vector (called V8) described by<br>
C. Cullin et al. (Gene, 65, 203-217, 1988).  V8 carries the selection indicator URA3 and contains an expression cassette in the yeast containing the promoter GAL10/CYC1 and the terminator sequence of the PGK gene.  The vector thus obtained, called delta-7/V8, allows the inducible expression by galactose of the delta-7Red protein.<br>
b)	Production of the delta-7Red protein<br>
The yeast strain FY 1679 (Mata) was transformed with the delta-7/V8 plasmid obtained above by using the lithium acetate method described by D. Gietz et al. (already quoted).<br>
The transformed yeast was cultured at 27°C, in the SGI selective medium described above but in which the glucose concentration is 5 g/1, until a cell density saturation (OD6Q0nin=12) is obtained.  The culture is then diluted by the addition of a volume of complete medium YP (10 g/1 in yeast extract (Difco) and 10 g/1 in bactopeptone (Difco)), then the addition of ethanol (1.5% v/v) as a source of carbon.  When the growth has been allowed to reach a cellular concentration of at least 7xl07 cells/ml, the expression of delta-7Red was induced by the addition of galactose at a concentration of 20 g/1.<br>
The induction was also carried out in an SLI selective minimum medium, which corresponds to the SGI medium in which the glucose is replaced by galactose (20 g/1), until a cellular concentration of 2 x 107 cells/ml is obtained.<br>
c)	Enzymatic test in vitro of the delta-5,7 sterol,delta-7<br>
reductase activity<br>
Expression of the delta-7Red protein was revealed by using an enzymatic test described by M. Taton et al. (Biochem. Bioph. Res. Commun., 181, 465-473, 1991) but without an NADPH regeneration system, with microsomal or cytosolic cell preparations of the above induced yeast.<br>
The cellular fractions were obtained by mechanical rupture of the induced cells and isolation of the fractions by ultracentrifugation according to the method described by P. Urban et al. (Eur. J. Biochem., 222, 843-850, 1994).  The cells are collected then washed twice with TE-KCl buffer (50 mM Tris-HCl, pH 7.4; ImM EDTA, 0.1 M KC1) and resuspended in<br>
0.6 M TE-sorbitol lysis buffer.  Glass beads with a diameter of 0.45-0.5 mm (Braun) are added until they show through the surface of the cellular suspension which is then agitated for 5 min at 4°C.  The cellular lysate on the surface is collected and the glass beads are washed three times with the lysis buffer.  The lysate and the washings are united then centrifuged at 20,000 g for 13 min at 4°C so as to eliminate the mitochondrial fractions.  The collected supernatant is centrifuged for 1 hour at 100,000 g and at 4°C.  The pellet which contains the microsomal fraction and the supernatant which represents the cytosolic fraction are collected separately.<br>
The microsomal fraction or the cytosolic fraction obtained are incubated respectively for 90 min at 37°C in 100 mM Tris/HCl buffer at pH 7.3 containing as substrate 150 urn of 7-dehydrocholesterol emulsified with Tween 80 (1.5 g/1) -and in the presence of 2 mM of NADPH.  The sterols are extracted by the addition of 3 volumes of a methanol-dichloromethane mixture (50:50, v/v) then analyzed by GC in comparision with standard products.<br>
The formation of cholesterol (RT = 15.887 min) from 7-dehydrocholesterol (RT = 16.528 min) is shown in Figure 4 with a microsomal fraction (3.5 mg/ml of protein) obtained above from the yeast FY 1679 transformed with the delta-7/V8 vector, induced for 3 hours and the endogenic sterols of which have higher retention times (RT = 16.682 min for ergosta 5-22 diene 3-ol; RT = 17.249 min for ergosterol and RT = 17.664 min for ergosta 5-ene 3-ol).<br>
These results show that the delta-7Red protein, on the one hand is expressed in the transformed yeast, and on the other hand has a delta-5,7 sterol,delta-7 reductase activity.<br>
EXAMPLE 2; Reduction in vivo of endogenic sterols of yeast unsaturated in position C-7<br>
Yeast strains whose majority sterols have a double bond in position C-5,7 were transformed with the delta-7/V8 vector obtained in Example 1, then cultured and induced as indicated in Example 1.  The endogenic sterols, whose profile is<br><br>
analyzed by GC, were extracted and purified by RP-HPLC as indicated in Example 1 by using a preparative CIS column (100 X 4.6 mm), then identified by IR, UV, MS and NMR.  The following three strains were used respectively:<br>
-	Strain FY1679 described in Example 1;<br>
-	Mutant strain erg5, called PLC 1051, characterized by a<br>
deficiency in sterol C-22 desaturase, which was constructed<br>
by crossing between the strain FY1679 and the original strain<br>
po!5 described by S. W. Molzahn et al. (J. Gen. Microbiol.,<br>
72, 339-348, 1972) and which accumulates the ergosta 5,7-<br>
diene 3-ol.<br>
-	The mutant double strain erg4,5, called PLC 1451,<br>
characterized by a deficiency in sterol C-22 desaturase<br>
(erg5) and sterol C-24(28) reductase (erg4), was obtained by<br>
crossing between the strain FY1679 and a strain po!5<br>
described by S. W. Molzahn et al. (already quoted) having<br>
acquired a spontaneous resistance to nystatin during a<br>
systematic screening of yeasts searching for mutants of<br>
sterols and which accumulates ergosta 5,7,24(28) triene 3-ol.<br>
The resultant haploid strain, which carries the double<br>
mutation erg4, erg5, grows in the presence of galactose and a<br>
non-fermentable substrate and is auxotrophic for uracil,<br>
tryptophan and histidine.  The strains PLC 1051 and PLC 1451<br>
were deposited at the CNCM on 10th February 1995 under the<br>
numbers 1-1536 and 1-1537 respectively.<br>
The major reduced sterols identified respectively from the above three transformed strains are indicated in the following table:EXAMPLE 3: Production of pregnenolone in vitro by restriction of yeast endogenic sterols reduced in position C-<br>
7<br>
Production of pregnenolone is carried out by using the enzymatic restriction test of the side chain of cholesterol in vitro described by Wada et al. (Arch. Biochem. Biophys., 290, 376-380, 1991) in which 260 uM of a sterol reduced in position C-7 obtained in Example 2 is incubated in 150 ul of 10 mM phosphate buffer, pH 7.4, 100 mM NaCl, 0.3% Tween 20 in the presence of 140 nM of adrenodoxin reductase, 1.16 uM of adrenodoxin and 0.68 uM of cytochrome P450SCC of bovine origin obtained from the suprarenal glands for example according to D. W. Seybert et al., J. Biol. Chem., 254, 12088-12098, 1979.<br>
The reaction is triggered by the addition of 150 uM of NADPH.  After incubation for 80 min at 37°C, the reaction is stopped by the addition of a volume of a methanol-dichloromethane mixture (50:50 v/v).  The sterols are extracted and analyzed by GC as indicated in Example 1.<br>
Figure 5 shows respectively that ergosta 5-ene 3-ol<br>
 (Fig. 5A), ergosta 5,24(28) diene 3-ol (Fig. 5B) or ergosta 5,22 diene 3-ol (Fig. 5C) is the substrate of cytochrome P450SCC and leads to a product having an identical RT to that of pregnenolone obtained under the same conditions by restriction of cholesterol.<br>
The results obtained show that a transformed yeast expressing delta-VRed accumulates usable sterols directly to prepare pregnenolone by biological oxidation in vitro. EXAMPLE 4; Construction of yeast strains producing pregnenolone or its acetic ester in vivo<br>
A - Construction of the strain ELR01 containing an expression cassette for the delta-7Red of A. thaliana integrated in the locus ADE2 of the haploid strain FY1679 mating type a (FY1679 Mata)<br>
a) Construction of the integrative plasmid pADdelta-7 (pADA7):<br>
Construction of the plasmid pAD 7 was carried out as described in Figure 6.  The Bglll fragment (2244pb) containing the ADE2 gene of S. cerevisiae was isolated from the plasmid pASZ 11 (A. Stotz et al., Gene, 95, 91, 1990) and inserted in the pBluescriptll KS+ vector (Stratagene) at the BamHI site of the multiple cloning site.<br>
The plasmid obtained, called pBS-ADE2, was then linearized at its single Stu I site and dephosphorylated.<br>
A fragment of approximately 2.44 kb containing the promoter GAL10/CYC1, the coding phase of delta-7Red (sterol 7 reductase) and the terminator PGK (tPGK) was obtained from the plasmid delta-7/V8, obtained in Example IF by the PCR technique using the following oligonucleotides as primers:<br>
5' GATTACGCCA AGCTTTTCGA AAC 3'         (SEQ ID No. 6) and 5' AGATCTTGAG AAGATGCGGC CAGCAAAAC 3'   (SEQ ID N o. 7)<br>
which have been defined to pair with the 3' end of tPGK and the 3' end of the URA3 gene respectively.  The following were used: the plasmid delta-7/V8 as a template (80 ng), the above oligonucleotides (0.5 /xM each), native Pfu DNA polymerase (1<br>
unit in the buffer recommended by Stratagene) and the following amplification conditions: 35 cycles; 1 min at 95°C; 5 sec. at 95°C; 30 sec. at 56°C; 4 min 30 sec. at 70°C).  The amplification fragment obtained was then cloned at the blunt ends in the above linearized plasmid pBS-ADE2 to give the plasmid pADA? in which the Notl-PstI fragment of approximately 4720 pb carries the ADE2 gene interrupted by the expression cassette of delta-7Red. b) Chromosomic integration in the yeast strain FY1679 Mata:<br>
The Notl-PstI fragment (4720 bp), isolated from the plasmid pADA7 digested with the restriction enzymes NotI and PstI, was introduced into the yeast FY1679 Mata by transformation using the lithium acetate method described by D. Gietz et al. (already quoted).<br>
The transformants having integrated this fragment by homologous recombination were selected by their resistance to nystatin, this phenotype being due to the expression of delta-7Red which converts the delta-5,7 sterols of the yeast into delta-5 sterols.<br>
The transformed cells were incubated at 28°C for 4 hours in the complete medium YP described in Example IF containing glucose (20 g/1) and supplemented with adenine (20 mg/1). They were then concentrated then plated on an SLI-agar minimum medium (lg/1 of bactocasaminoacids; 7 g/1 of "yeast nitrogen base"; 20 g/1 of galactose; 20 mg/1 of adenine; 20 g/1 of agar) and incubated overnight at 28°C to induce the expression of the delta-7Red gene.  The absence of uracil complementation allows the growth of cells to be limited. The clones were collected, grouped then plated on dishes on a complete medium YP containing galactose (20 g/1), supplemented with adenine (20 mg/1) and in the presence of increasing concentrations of nystatin (0 fig/ml/ 1 /ig/ml, 2 jug/ml, 5 /Lig/ml, 20 /zg/ml respectively) .  On the 4th day, about twenty clones having grown at a concentration of 5 /ug/ml of nystatin were obtained.  Twelve of these clones were sub-cultured on dishes in minimum medium WO (7g/l of "yeast nitrogen base" without amino acids, 20 g/1 of glucose) enriched with uracil, leucine, tryptophan, histidine and<br>
adenine (20 mg/1 each).<br>
The auxotrophy of the adenine due to the disruption of the ADE2 gene was then confirmed by observing the absence of growth on the minimum medium WO described above enriched with uracil, leucine, tryptophan, histidine but free of adenine.<br>
The presence of the expression cassette in the genome of the yeast was controlled by PCR amplification from the genomic DNA of the clones obtained and by using the primers having the sequence SEQ ID No. 6 and SEQ ID No. 7 above.<br>
The functionality of the integrated delta-7Red gene was confirmed by GC analysis of the composition in sterols accumulated in the yeast and extracted by alkaline saponification according to the operating method described in Example IB with a five meter SE 30 capillary column (Alltech).  Analysis shows a modified profile containing sterols saturated in position C-7 when the clones are cultivated in the presence of galactose.  The strain obtained, called ELR01, accumulates ergosta 5 ene 3-ol and ergosta 5,22 diene 3-ol instead of ergosta 5, 7, 22 triene 3-ol (ergosterol), the majority sterol of the initial strain FY1679 as is shown in Figure 7.<br>
The strain ELR01 constructed in this way expresses the delta-7Red gene when it is cultured in the presence of galactose due to the fact of transcriptional control by the promoter GAL10/CYC1.  Although the expression unit for delta-7Red has the same transcription direction as that of the ADE2 gene, no expression of the delta-7Red is detectable by analysis of the composition of sterols by GC when the culture is carried out in the presence of glucose, as a result of the repression of the promoter GAL10/CYC1.<br>
B - Construction of the strain CDR01 containing an expression cassette for the mature form of bovine adrenodoxin reductase (ADRm) integrated between the loci LEU2 and SPL1 of the haploid strain FY1679 mating type alpha (mat. alpha). a) Construction of the shuttle vector E. coli-S. cerevisiae V13<br>
The V13 vector corresponds to the V8 vector (C. Cullin et al., already quoted) which carries the selection indicator<br>
URA3 and an expression cassette in yeast containing the GAL10/CYC1 promoter (pG/C) and in which an additional Sail site (Sa) has been introduced into the multiple cloning site, according to the construction diagram given in Figure 8.<br>
The V8 vector was digested by the restriction enzymes Hindlll and BamHI and the BamHI-Hindlll fragment obtained (1722 pb) containing the URA3 gene and the GAL10/CYC1 promoter (called further on ura-gal"), was sub-cloned between the corresponding sites of the pUC18 vector (Pharmacia) digested by the restriction enzymes Hindlll and BamHI.<br>
The "ura-gal" fragment was then amplified by PCR from 30 ng of the pUCIS/"ura-gal" plasmid obtained denatured for 30 sec. at 95°C using the following conditions: 30 cycles; 5 sec. at 86°C; 10 sec. at 95°C;  40 sec. at 38°C; 5 sec. at 55°C and 2 min at 74°C), 2 units of Tag DNA polymerase (Boehringer) in the manufacturer's buffer and 1/iM of each of the primers having the following nucleotide sequences:<br>
5' GGGGATCCGT GGTCGACGTA ATTTAGTGTG TGTATTTGTG TTTGCG 3'<br>
(SEQ ID NO. 8) and<br>
5' GTAAAACGAC GGCCAGT 3'	(SEQ ID No. 9)<br>
The primer SEQ ID No. 8 contains a BamHI GGATCC site identical to that of the "ura-gal" fragment, 3 consecutive nucleotides at the BamHI site not hybridizing with the template, an Sail GTCGAC site and a sequence homologous to that of the GAL10/CYC1 promoter.  The primer SEQ ID No. 9 matches the sequence preceding the Hind III site of the multiple cloning site pUCIS.  The Hindlll-BamHI fragment of 1722 pb obtained after amplification was digested by Xhol and Bam HI releasing a fragment of 254 bp containing the GAL10/CYC1 promoter (pG/C) which was then sub-cloned in the V8 vector digested by the restriction enzymes BamHI and Xhol.<br>
The resultant V13 vector contains restriction sites allowing easy sub-cloning of the cDNA's coding for mature bovine adrenodoxin reductase (ADRm), mature bovine adrenodoxin (ADXm) and the bovine cytochrome P450SCC.<br><br>
Starting from the V13 vector, the V13-ADR vector, the V13-ADX vector and the V13-SCC10 vector were prepared respectively in the following manner:<br>
a)	Construction of the VI3-ADR vector<br>
A Sall-Kpnl fragment of 1478 pb carrying the cDNA coding for ADRm was isolated from the plasmid pGBADR-2 described in Example 25 of the European Patent Application EP 340878 and sub-cloned in the corresponding sites of the V13 vector to give the V13-ADR vector.<br>
b)	Construction of the V13-ADX vector<br>
A Sail BamHI fragment of 390 pb carrying the cDNA coding for ADXm was isolated from the plasmid pGBADX-1 described in Example 23 of the European Patent Application EP 340878 and sub-cloned in the corresponding sites of the V13 vector to give the V13-ADX vector.<br>
c)	Construction of the V13-SCC10 vector<br>
A Sall-EcoRI fragment of 1554 pb carrying the cDNA coding for P45QSCC was isolated from the plasmid pGBSCC-10 described in Example 6 of the European Patent Application EP 340878 and sub-cloned in the corresponding sites of the V13 vector to give the V13-SCC10 vector.<br>
b) Construction of the integrative plasmids pCD62-l and PCD62-2:<br>
The construction of the plasmids pCD62-l and pCD62-2 was carried out as described in Figure 9.<br>
a) Construction of the plasmid pFL26CD<br>
A NotI site was introduced into the plasmid pFL26 (N. Bonneaud et al., Yeast, 7, 609-615, 1991), in the intergenic region separating the Ieu2 gene from the 5' end of the spll gene (called spll ) (C. Kolman et al., J. Bacteriol., 175, 1433, 1993) according to the following operating method:<br>
Two DNA fragments of 704 bp and 347 bp carryingthe 5' end of Ieu2 and the 3' end of Spll respectively were synthesized by PCR using primers having the following nucleotide sequences:<br>
5' TTGAAGGTTC AACATCAATT GATTG 3'        (SEQ ID No. 10) and<br>
5' GTGTGGCGGC CGCCTCCTTG TCAATATTAA TGTTAAAG 3'<br>
(SEQ ID No. 11)<br>
for the amplification of the 704 bp fragment and the nucleotide sequences<br>
5' CAAGGAGGCG GCCGCCACAC AAAAAGTTAG GTGT 3' (SEQ ID No. 12) and<br>
5' TCTGCTTCCC TAGAACCTTC TTATG 3'	(SEQ ID No. 13)<br>
for the amplification of the 347 bp fragment.<br>
The primers SEQ ID No. 11 and SEQ ID No. 12 have a sequence GGCGGCCG which corresponds to a NotI site and in which 3 bases do not match with the template.  The primer SEQ ID No. 10 matches with a sequence situated 536 bp upstream from the stop codon of Ieu2 and the primer SEQ ID No. 13 with a sequence situated 194 bp upstream that of spllA.<br>
The 704 bp and 347 bp fragments are first amplified by PCR by using the plasmid pFL26 as the template and Pfu DNA polymerase as the enzyme under standard conditions described by the supplier (Stratagene).<br>
The two amplified fragments obtained matched over 20 bp at the level of the ends generated by the primer SEQ ID No. 11 (704 bp fragment) and the primer SEQ ID No. 12 (347 bp fragment) starting from the 5' end; these 20 bp correspond to the first 20 nucleotides of each of these primers respectively.<br>
The product resulting from the pairing of the DNA fragments of 704 bp (1 ng) and 347 bp (2 ng) was amplified using primer SEQ ID No. 10 and SEQ ID No. 13 and the following conditions: 30 cycles of 10 sec. at 95°C, 5 sec. at 60°C, 1 min at 45°C, 5 sec. at 65°C and 2 min at 72°C followed by a cycle of 7 min at 72°C;  50 pmol of each primer and 1 unit of Pfu DNA polymerase in 50 jul of reaction buffer<br>
 (Stratagene).  An amplified fragment of 1031 bp containing a NotI restriction site was obtained.  This fragment was then digested by the BstXI and Nsil enzymes and the fragment of 920 bp obtained, containing the NotI site, was inserted in place of the initial BstXI-Nsil fragment of pFL26 to give the plasmid pFL26CD, whose map is represented in Figure 9a. b) Construction of the plasmid pCD60<br>
-	Preparation of the plasmid pDP10036:<br>
A Sall-BamHI fragment of 390 bp carrying the cDNA coding for mature bovine adrenodoxin (ADXm) was isolated from the plasmid V13-ADX then sub-cloned in the Sall-Bglll sites of the multiple cloning site of the plasmid pTG10033 which is flanked by the inducible promoter GAL10/CYC1 and the terminator ter PGK.  The plasmid pTG10033, whose map is represented in Figure 18 and which corresponds to the expression vector pTG10031 (Figure 17), containing the promoter CYC1 and terPGK, in which the promoter CYC1 has been replaced by the promoter GAL10/CYC1 was prepared according to the operating method described further on.<br>
The plasmid thus obtained, called pDP10034, carries the ADX expression cassette, i.e. the gene coding for ADXm under the transcriptional control of GAL10/CYC1 and terPGK. Subsequently, the term "expression cassette" will be used for any gene under the transcriptional dependence of GAL10/CYC1 and terPGK.<br>
A Hindlll fragment of 3593 bp carrying the selection indicator URA3 and the ADR expression cassette was isolated from the plasmid V13-ADR digested by the restriction enzyme Hindlll then inserted in the corresponding site of plasraid pDP10034 digested by the restriction enzyme Hindlll.  The plasmid obtained, called pDP10036, contains the ADX and ADR expression cassettes separated from each other by the marker URA3 (Figure 9b).<br>
-	Preparation of the plasmid pCD60:<br>
The Afllll-AccI fragment of 2770 bp carrying the ADR cassette was isolated by partial digestion of the plasmid pDP10036 with the restriction enzymes Afllll and AccI, bluntends were created by treatment with the klenow fragment<br>
of DNA polymerase I and sub-cloned after ligation at the blunt ends in the Smal site of the plasmid pBlue-Script KS+ (Stratagene).  In the plasmid obtained, called pCD60, the ADR expression cassette is framed on either side by a NotI site, one situated at 209 bp upstream of the Afllll/Smal ligation site and originating from the sub-cloned fragment and the other originating from the multiple cloning site (MCS1) of pBlue-Script KS+ (Figure 9b).<br>
c) Construction of plasmids pCD62-l and pCD62-2:<br>
The NotI fragment of 2558 bp, isolated from the plasmid pCD60 digested by the restriction enzyme NotI, was then sub-cloned in the single NotI site of the plasmid pFL26CD. Depending on the direction of the insertion of the fragment, two plasmids were obtained, called pCD62-l and pCD62-2 (Figure 9c).<br>
In the plasmid pCD62-l, the ADR expression cassette is -oriented in the direction of the transcription of the Ieu2 gene while this orientation is reversed in the plasmid pCD62-2.<br>
The plasmid pCD62-l was retained for subsequent constructions.<br>
c) Chromosomic integration in the yeast strain FY1679 (Matalpha):<br>
The plasmid pCD62-l contains regions homologous to a chromosomic locus of the strain FY1679.  These regions correspond respectively to fragments Bglll-Clal of 1060 bp (A), EcoRI-NotI of 707 bp (B) and Notl-Bglll of 602 bp (C) indicated in Figure 10.<br>
The region corresponding to the Clal-EcoRI fragment of 486 bp of the plasmid pCD62-l was deleted in the strain FY1679, implying an auxotrophy of this strain vis-a-vis leucine (strain LEU2") (R. S. Sikorski et al., Genetics, 122, 19, 1989).<br>
The plasmid pCD62-l was linearized by digestion with the restriction enzyme Xbal whose restriction site is situated outside the homologous regions, then was introduced by transformation in the strain FY1679 (Matalpha) using the lithium acetate method (D. Gietz et al., already quoted).<br>
The cellular repair capacity of the DNA by the yeast ("gap repair") and the selection of recombinants having the phenotype LEU2+ have allowed in the first instance two types of recombinants to be selected: the 1st type is obtained after homologous recombinations at the level of fragments A and B, the 2nd type is obtained after homologous recombinations at the level of fragments A and C.  Only the latter type of recombination allowed the integration of the ADR expression cassette in addition to the restoration of phenotype LEU2+.<br>
To select this 2nd type of clone, a screening by PCR was carried out using the primer SEQ ID No. 10 above and the primer having the following nucleotide sequence:<br>
5' TACATTAGGT CCTTTGTAGC 3'	(SEQ ID No. 14)<br>
so as to confirm both the presence and the correct localization of the ADR expression cassette in the genome of the strain FY1679 (Matalpha).<br>
In this screening, the primer SEQ ID No. 14 exclusively matches the sequence coding for ADRm and the primer SEQ ID No. 10 matches a chromosomic sequence (Figure 10).<br>
The amplification reaction was carried out using as template the genomic DNA (20 ng) isolated from the strain FY1679 (Matalpha) (C. Hoffman et al., Gene, 57, 267, 1987), Taq DNA polymerase (1U, Boehringer), 50 pmol of each primerand 30 PCR cycles (10 sec. at 95°C, 1 min at 55°C, 3 min at 72°C) under the standard conditions described by the supplier.<br>
The amplification led to the isolation of a corresponding fragment of 2.9 kb in the case where the expression cassette has been integrated.  In the opposite case, no amplification product was detected.<br>
The strain FY1679 (Matalpha) selected in this way containing the ADR integrated expression cassette was called CDR01.<br>
The expression of ADR by this strain was revealed from cytosolic cellular fractions prepared according to the<br>
protocol described in Example IF, by immunodetection of the protein recognised by the anti-ADR antibodies.<br>
The functionality of the ADR expressed by the strain CDR01 was confirmed in the enzymatic restriction test of the side chain of cholesterol in vitro described in Example 3 and in which the purified ADR (0.28 pmol) was replaced by a cytosolic cellular fraction of the strain CDR01 containing 100 ug of total proteins.  A bioconversion rate of cholesterol into pregnenolone of approximately 25% was observed, comparable to that obtained with purified ADR. C - Construction of the diploid strain EC01 coexpressing delta-7Red of A. Thaliana and ADRm.<br>
The diploid strain EC01 was obtained by crossing the haploid strains CDR01 and ELR01 obtained above according to the protocol described by G. Sprague et al. (Methods in Enzymology, 194, 77, 1991):<br>
A first selection on minimum medium WO described previously, enriched with uracil, tryptophan and histidine (20 mg/1 each) but free of leucine, allowed the diploid clones LEU2+ (prototrophic character which shows the presence of the ADR expression cassette) to be isolated.  These clones were then tested for their resistance to nystatin at 5 jug/ml (resistance character which shows the presence of the expression cassette for delta-7Red) on the solid synthetic SLI-agar medium described previously.<br>
A strain, called EC01, was in this way arbitrarily isolated from the clones resistant to nystatin. D - Construction of the strain EC01/pCD63 producing pregnenolone and pregnenolone 3-acetate. a) Construction of the expression plasmid pCD63<br>
The construction of the plasmid pCD63 was carried out as described in Figure 11.<br>
The NotI fragment of 4961 bp containing the ADX expression cassette, the selection indicator URA3 and the ADR expression cassette, were isolated from the plasmid pDP10036 prepared above and digested with the restriction enzyme NotI, then cloned in the NotI site of the multiple cloning site of plasmid pFL45L (N. Bonneaud et al., Yeast, 7, 609, 1991).<br>
The vector thus obtained, called pDP10037, is represented in Figure 11.<br>
On the one hand, the plasmid pDP10037 was linearized by digestion with the enzyme Tthllll whose restriction site is situated in the gene coding for ADRm.<br>
On the other hand, a PvuII-EcoRV fragment of 3476 bp containing the expression cassette for P45QSCC and the 5' end of the URA3 marker were purified from the plasmid V13-SCC10 obtained previously and digested with the restriction enzymes PvuII and EcoRV.<br>
The two linear DNA's respectively obtained have homologous regions which correspond on the one hand to the 5' end of the gene URA3 and to GAL10/CYC1, on the other hand to ter PGK as represented in Figure lla.<br>
These two fragments were then introduced into the yeast strain FY1679 (Mata) by co-transformation using the lithium • acetate method (D. Gietz et al., already quoted).<br>
The following selection of prototrophic recombinants for uracil and tryptophan (URA3+, RTP1+), allowed clones to be isolated in which the double strand break generated by digestion by the restriction enzyme Tthllll has been repaired as a result of the integration of the expression cassette P450SCC by homologous recombination.<br>
Selection of the recombinants (URA3+, RTP1+) was carried out on minimum medium WO enriched with leucine, histidine and leucine (20 mg/1 each) but free from uracil and tryptophan. Starting from 50 collected clones, the total DNA was extracted by the method described by C. Hoffman et al., (Gene, 57, 267, 1987), then introduced by electroporation into the strain of E. coli XLl-Blue (Stratagene).  The clones transformed by the plasmid generated by "gap repair" were selected on the rich medium LB (tryptone 1%, yeast extract 0.5%, NaCl 1%) containing 50 mg/1 of ampicillin.  Starting from one of the selected clones, the plasmid, called pCD63, was extracted according to the method described by J. Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratory Press, 1989.  The plasmid pCD63 obtained contains expression cassettes ADX and P45QSCC separated by the<br>
selection indicator URA3, as represented in Figure lib.<br>
b)	Transformation of the strain EC01 by the plasmid pCD63<br>
The plasmid pCD63 was introduced into the strain EC01 obtained above by transformation by the lithium acetate method (D. Gietz et al., already described).  The transformed yeast was then cultured on the minimum medium WO described previously and free from uracil, tryptophan, adenine and leucine but supplemented with histidine (20 mg/1).<br>
The strain, called EC01/pCD63, was isolated in this way. A sample of the strain, under the reference EC01/pCD63, was deposited at the CNCM on 10th February 1995 under the number 1-1538.<br>
c)	Production in vivo of pregnenolone and pregnenolone 3-<br>
acetate.<br>
The strain EC01/pCD63 was cultured at 28°C under aerobic conditions (130 r/min) in a 3-litre Erlenmeyer until the stationary growth phase is reached (D0600 = 12 to 13) in the selective medium SGI described in Example 1A and in which the glucose concentration is 5 g/1.  The culture is then diluted by the addition of a volume of the complete medium YP described above, then the addition of ethanol (0.5% v/v) as a source of carbon. When a new stationary growth phase has been reached (DO600 = 12 to 13), galactose (20 g/1) is added to the culture, in the form of a concentrated solution (500 g/1), so as to simultaneously induce the expression of genes coding for ADXm, ADRm, P450SCC and delta-7 Red which are respectively under the control of the promoter GAL10/CYC1.<br>
The co-expression of the four genes was revealed by analysis of the sterols accumulated in the cells and in the supernatant of the culture respectively according to the following operating method:<br>
50 ml samples of the culture are harvested after 9 hours and 24 hours of induction.  Each sample is centrifuged (4000 g, 10 min, 4°C) so as to separate the cells from the culture medium.<br>
On the one hand, the cells are lyzed by mechanical rupture in the presence of glass beads according to the<br>
method indicated in Example IF.  Starting from the lysate thus obtained, the intracellular sterols are then extracted by the addition of a volume of hexane.<br>
On the other hand, the sterols present in the culture medium are extracted directly by the addition of a volume of hexane.<br>
The composition of sterols extracted is analyzed by GC as indicated in Example 1, compared with standard products.<br>
The results obtained after 9 hours or after 24 hours of induction are shown in Figure 12a and in Figure 12b respectively. After 9 hours of induction, Figure 12a shows:<br>
-	in the cellular lysate, the presence of a majority compound<br>
having an identical retention time to that of standard<br>
pregnenolone acetate (RT =11.8 min) while only a very weak<br>
peak is present at the retention time of the pregnenolone (RT<br>
= 9.9 min).  Low quantities of endogenic sterols of the yeast<br>
reduced in position C-7 (ergosta 5-ene 3-ol and ergosta 5,22<br>
diene 3-ol) are identified respectively at RT = 18 min and RT<br>
= 17 min.  The alkaline saponification of the cellular lysate<br>
before analysis leads to the presence of a majority compound<br>
co-migrating with the pregnenolone.  This allows confirmation<br>
that the accumulated compound having an RT of 11.8 min<br>
corresponds to pregnenolone acetate.<br>
-	in the culture medium, the significant absence of<br>
pregnenolone or its acetate.<br>
After 24 hours of induction, Figure 12b shows:<br>
-	in the cellular lysate, the presence of low quantities of<br>
pregnenolone (RT =10.2 min) and of pregnenolone acetate (RT<br>
= 12 min) as well as of reduced endogenic sterols of the<br>
yeast (RT = 17 min and RT = 18 min).  The cholesterol (RT =<br>
16.2 min) is an internal standard added before the<br>
extraction.<br>
-	in the culture medium, the majority presence of<br>
pregnenolone acetate and a minor quantity of pregnenolone.<br>
The experiments carried out in parallel with the strain EC01 transformed by a control plasmid such as pFL45L already quoted, have shown no peak corresponding to free pregnenolone<br>
or pregnenolone in the acetate form.<br>
Identification of the sterols carried out in this way shows that the yeast strain EC01/pCD63 has accumulated the pregnenolone and the pregnenolone acetate in the absence of any source of exogenic sterols, after induction in the presence of galactose with a major production after an induction of 9 hours.<br>
These results demonstrate on the one hand the effective mobilization of the endogenic sterols reduced in position C-7 of the strain EC01, on the other hand the extreme effectiveness of the coupling of the restriction reaction of the side chain of the endogenic sterols.<br>
Preparation of Example 4: construction of the plasmid PT610033<br>
1.  Derivative of pUC19 having a new multiple cloning site:<br>
The cloning vector M13mpl9 (C. Yanish-Peron et al., Gene, 33, 103, 1985) was mutagenized using the following oligonucleotide:<br>
5' GCGCTCAGCG GCCGCTTTCC AGTCG 3'	SEQ ID No. 15<br>
to introduce a NotI site into the sequence of the truncated lacl gene and to give the plasmid M13TG724.<br>
A "polylinker11 containing the EcoRI, SnaBI and NotI sites was then introduced into the EcoRI site of the plasmid M13TG724 by using the following oligonucleotides:<br>
5' AATTGCGGCC GCGTACGTAT G3'	SEQ ID No,16<br>
and<br>
5' AATTCATACG TACGCGGCCG C 3'	SEQ ID No. 17<br>
to give the plasmid M13TG7244 in which a modification of the insert was observed during the amplification stage.  The insert of the plasmid M13TG7244 has the following nucleotide sequence:<br>
GAATTCATACGTACGCGGCCGCAATTGCGGCCGGTACGTATAATTCACrGGCCGr<br>
in which the EcoRI, SnaBI and SstI sites are underlined and the lacZ sequence of pUC19 is in italics.<br>
After digestion of the plasmid M13TG7244 by the restriction enzymes EcoRI and SstI, a "polylinker11 containing the Mlul and Avrll sites was introduced using the following oligonucleotides:<br>
5' CAACGCGTCC TAGG 3'	SEQ ID No. 18<br>
and<br>
5' AATTCCTAGG ACGCGTTGAG CT 3'	SEQ ID No. 19.<br>
After digestion with the enzyme PvuII, the PvuII fragment obtained was sub-cloned in pUC19 (C. Yanish-Porch et al. already quoted) to give the plasmid pTG7457 (Figure 13).<br>
2.	Sub-cloning of the PGK terminator:<br>
pUC19 was digested with the restriction enzymes BamHI and EcoRI and a new "polylinker" BamHI SstI was introduced using the following oligonucleotides:<br>
5' GATCCGCAGA TATCATCTAG ATCCCGGGTA GAT 3'   SEQ ID No. 20,<br>
5' AGAGCTCAAG ATCTACCCGG GATCTAGATG ATATCTGCG 3'<br>
SEQ ID No. 21,<br>
5' CTTGAGCTCT ACGCAGCTGG TCGACACCTA GGAG 3' SEQ ID No. 22 and 5' AATTCTCCTA GGTGTCGACC AGCTGCGT 3'         SEQ ID No. 23. •<br>
The plasmid pTG7453 was obtained in this way (Figure 14) then was digested by the restriction enzymes BamHI and SstI. The sites of the "polylinker11 between BamHI and SstI were introduced into a derivative of plasmid pTG7457 obtained above and digested by the restriction enzymes BamHI and SstI. The new plasmid obtained contains the PvuII, Hindlll, BamHI, EcoRI, Xbal, Smal, Bglll, SstI (=SacI), Mlul, Avrll, EcoRI, SnaBI, NotI, SnaBI, Pvul sites.<br>
This new plasmid was digested by the restriction enzymes Bglll and Hindlll and a Bglll-Hindlll fragment containing the PGK promoter (R. A. Hitzeman et al., Nucleic Acids Res., 10, 7791, 1982;  G. Loison et al., Yeast, 5, 497, 1989) was inserted in the former to give the plasmid pTG10014 (Figure 15).<br>
3.	Sub-cloning of promoters:<br>
a) the CYC1 promoter<br>
The sites of the "polylinker" between BamHI and SstI of the plasmid pTG7453 were introduced into a derivative of the plasmid pTG7457 as indicated above.  The new plasmid obtained was restricted by the restriction enzyme SnaBI, then the<br>
Rsal-Dral fragment of 456bp of the plasmid pEMBLS (L. Dente et al., Nucleic Acid Res., 11, 1645, 1983) containing the origin of replication of the phage fl was introduced to give the plasmid pTG7503.<br>
The BamHI Hindlll fragment of 0.78 kb of the plasmid pGBSCC-9, prepared in Example 6 of the European Patent Application EP 0340378 and containing the CYC1 promoter of S. cerevisiae, a "polylinker" and the lactase terminator of K.lactis, was sub-cloned in the plasmid pTG7503 digested with the restriction enzymes Hindlll and BamHI to give the plasmid PTG10004 (Figure 16).<br>
The Xhol and Mlul sites of the CYC1 promoter were then eliminated by site directed mutagenesis of the double strand DNA of the plasmid pTG10004 using the following oligonucleotide:<br>
5' GCGGATCTGC TCGAAGATTG CCTGCGCGTT GGGCTTGATC 3'<br>
SEQ ID NO. 24.<br>
The plasmid pTGlOOOS thus obtained was then digested by the restriction enzymes Sail and Xhol then a Mlul site was introduced by using the following oligonucleotides:<br>
5' TCGACGGACG CGTGG 3'	SEQ ID No. 25<br>
and<br>
5' TCGACCACGC GTCC 3'	SEQ ID No. 26<br>
to give the plasmid pTG10006.<br>
b) the GAL10/CYC1 promoter<br>
The plasmid pYeDPl/8-2 (C. Cullin et al., Gene, 203, 1988) was opened with the restriction enzyme Xhol.  The cohesive ends created were filled using the Klenow fragment of DNA polymerase then the plasmid was religated.  The plasmid pTGlOOlO thus obtained in which the GAL10/CYC1 promoter no longer contains the Xhol site is used as a template for a PCR amplification.<br>
4.  Construction of the expression vector pTG10031<br>
The remaining part of the sequence coding for lacZ was eliminated in the plasmid pTG7503 by site directed mutagenesis using the following oligonucleotide:<br>
5' TGGCCGTCGT TTTACTCCTG CGCCTGATGC GGTAT 3' SEQ ID No. 27<br>
to give the plasmid pTG7549.<br>
The LacZ promoter present in the plasmid pTG7549 wasthen deleted using the following oligonucleotides:<br>
5' GGCCGCAAAA CCAAA 3'	SEQ ID No. 28 and<br>
5' AGCTTTTGGT TTTGC 3'	SEQ ID No. 29<br>
which are inserted in the plasmid digested beforehand by the restriction enzymes NotI and Hindlll and which restore the two sites to give the plasmid pTG7553.<br>
A BamHI Mlul fragment containing the CYC1 promoter was obtained from the plasmid pTG10006 digested by the restriction enzymes BamHI and Mlul and a Mlul Hindlll fragment containing the PGK promoter was isolated from the plasmid pTG10015 digested by the restriction enzymes Mlul and Hindlll.  These two fragments were ligated and the ligation product obtained was inserted in the plasmid pTG7553 digested beforehand by the restriction enzymes Mlul and Hindlll.<br>
The following oligonucleotide:<br>
5' GATCTATCGA TGCGGCCGCG 3'	SEQ ID No. 30<br>
hybridized with the following oligonucleotide:<br>
5' CGCGCGCGGC CGCATCGATA 3'	SEQ ID No. 31<br>
which constitutes a BamHI Mlul "linker" containing the Clal and NotI sites, was added and ligated to give the expression vector pTG 10031 (Figure 17).<br>
The fragment amplified by PCR obtained above from the plasmid pYeDPl/8-2 was digested with the restriction enzymes Clal and Sail then was introduced into the plasmid pTG10031 digested beforehand with the same restriction enzymes to give the plasmid pTG10033 (Figure 18).<br>
SEQUENCE LISTING<br>
(1) GENERAL INFORMATION:<br>
(i) APPLICANT:<br>
(A)	NAME: ROUSSEL UCLAF<br>
(B)	STREET: 102, Route de Noisy<br>
(C)	CITY: ROMAINVILLE<br>
(E)	COUNTRY: FRANCE<br>
(F)	POSTAL CODE (ZIP): 93230<br>
(G)	TELEPHONE: 49.91.49.91<br>
(H)	TELEFAX: 49.91.46.10<br>
(ii) TITLE OF INVENTION: DNA sequence coding for a protein of A. thaliana having a delta-5,7 sterol,delta-7 reductase activity, delta7-Red protein, production process, strains of transformed yeasts, uses.<br>
(iii) NUMBER OF SEQUENCES: 31<br>
(iv) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Floppy disk<br>
(B)	COMPUTER: IBM PC compatible<br>
(C)	OPERATING SYSTEM: PC-DOS/MS-DOS<br>
(D)	SOFTWARE: PatentIn Release #1.0, Version #1.30 (EPO)<br>
(vi) PRIOR APPLICATION DATA:<br>
(A)	APPLICATION NUMBER: FR 9501723<br>
(B)	FILING DATE: 15-FEB-1995<br>
(vi) PRIOR APPLICATION DATA:<br>
(A)	APPLICATION NUMBER: FR 9506517<br>
(B)	FILING DATE: Ol-JUN-1995<br>
(2) INFORMATION FOR SEQ ID NO: 1:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1496 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
 (vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Arabidopsis thaliana<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: CDS<br>
(B)	LOCATION:76..1365<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 1:<br>
GTGTGAGTAA TTTAGGTCAA CACAGATCAG AATCTGAGGC TTTGGCCGAG ACGAAGAGAA     60<br>
AAGCAGAAGA AGAAA ATG GCG GAG ACT GTA CAT TCT CCG ATC GTT ACT TAC      111<br>
Met Ala Glu Thr Val His Ser Pro lie Val Thr Tyr<br>
15	10<br>
GCA TCG ATG TTA TCG CTT CTC GCC TTC TGT CCA CCT TTC GTC ATT CTC      159<br>
Ala Ser Met Leu Ser Leu Leu Ala Phe Cys Pro Pro Phe Val lie Leu<br>
15	20	25<br>
CTA TGG TAC ACA ATG GTT CAT CAG GAT GGT TCT GTT ACT CAG ACC TTT      207<br>
Leu Trp Tyr Thr Met Val His Gin Asp Gly Ser Val Thr Gin Thr Phe<br>
30	35	40<br>
GGC TTC TTT TGG GAG AAT GGA GTT CAA GGA CTT ATC AAC ATA TGG CCA      255<br>
Gly Phe Phe Trp Glu Asn Gly Val Gin Gly Leu lie Asn lie Trp Pro<br>
45	50	55	60<br>
AGA CCC ACT TTG ATT GCT TGG AAA ATT ATA TTT TGC TAT GGA GCA TTT      303<br>
Arg Pro Thr Leu lie Ala Trp Lys lie lie Phe Cys Tyr Gly Ala Phe<br>
65	70	75<br>
GAA GCT ATT CTT CAG CTG CTT CTG CCT GGT AAA AGA GTT GAG GGT CCA       351<br>
Glu Ala lie Leu Gin Leu Leu Leu Pro Gly Lys Arg Val Glu Gly Pro<br>
80	85	90<br>
ATA TCT CCA GCC GGA AAC CGA CCA GTT TAC AAG GCC AAT GGT CTG GCT       399<br>
lie Ser Pro Ala Gly Asn Arg Pro Val Tyr Lys Ala Asn Gly Leu Ala<br>
95	100	105<br>
GCT TAC TTT GTG ACA CTA GCA ACC CAT CTT GGT CTT TGG TGG TTT GGA       447<br>
Ala Tyr Phe Val Thr Leu Ala Thr His Leu Gly Leu Trp Trp Phe Gly<br>
110	115	120<br><br><br>
ATC TTC AAC CCT GCA ATT GTC TAT GAT CAC TTG GGT GAA ATA TTT TCG<br>
lie Phe Asn Pro Ala lie Val Tyr Asp His Leu Gly Glu lie Phe Ser<br>
125	130	135	140<br><br>
495<br><br><br><br>
GCA CTA ATA TTC GGA AGC TTC ATA TTT TGT GTT TTG TTG TAC ATA AAA<br>
Ala Leu He Phe Gly Ser Phe He Phe Cys Val Leu Leu Tyr He Lys<br>
145	150	155<br><br>
543<br><br><br><br>
GGC CAT GTT GCA CCT TCA TCA ACT GAG TCT GGT TCA TGT GGT AAC CTA<br>
Gly His Val Ala Pro Ser Ser Ser Asp Ser Gly Ser Cys Gly Asn Leu<br>
160	165	170<br><br>
591<br><br><br><br>
ATA ATT GAC TTC TAT TGG GGC ATG GAG TTG TAC CCT CGA ATT GGT AAG<br>
He He Asp Phe Tyr Trp Gly Met Glu Leu Tyr Pro Arg He Gly Lys<br>
175	180	185<br><br>
639<br><br><br><br>
AGC TTT GAC ATC AAG GTG TTT ACT AAT TGC AGA TTC GGA ATG ATG TCT<br>
Ser Phe Asp He Lys Val Phe Thr Asn Cys Arg Phe Gly Met Met Ser<br>
190	195	200<br><br>
687<br><br><br><br>
TGG GCA GTT CTT GCA GTC ACG TAC TGC ATA AAA GAG TAT GAA ATA AAT<br>
Trp Ala Val Leu Ala Val Thr Tyr Cys He Lys Gin Tyr Glu He Asn<br>
205	210	215	220<br><br>
735<br><br><br><br>
GGC AAA GTA TCT GAT TCA ATG CTG GTG AAC ACC ATC CTG ATG CTG GTG<br>
Gly Lys Val Ser Asp Ser Met Leu Val Asn Thr He Leu Met Leu Val<br>
225	230	235<br><br>
783<br><br><br><br>
I TAT GTC ACA AAA TTC TTC TGG TGG GAA GCT GGT TAT TGG AAC ACC ATG<br>
Tyr Val Thr Lys Phe Phe Trp Trp Glu Ala Gly Tyr Trp Asn Thr Met<br>
240	245	250<br><br>
831<br><br><br><br>
GAC ATT GCA CAT GAC CGA GCT GGA TTC TAT ATA TGC TGG GGT TGT CTA<br>
Asp He Ala His Asp Arg Ala Gly Phe Tyr He Cys Trp Gly Cys Leu<br>
255	260	265<br><br>
879<br><br><br><br>
GTG TGG GTG CCT TCT GTC TAC ACT TCT CCA GGC ATG TAC CTT GTG AAC<br>
Val Trp Val Pro Ser Val Tyr Thr Ser Pro Gly Met Tyr Leu Val Asn<br>
270	275	280<br><br>
927<br><br><br><br>
CAC CCC GTC GAA CTC GGA ACT CAG TTG GCA ATA TAC ATT CTC GTT GCA<br>
His Pro Val Glu Leu Gly Thr Gin Leu Ala He Tyr He Leu Val Ala<br>
285	290	295	300<br>
975<br>
GGA ATT CTG TGC ATT TAG ATA AAG TAT GAC TGT GAT AGA CAA AGG CAA      1023<br>
Gly lie Leu Cys lie Tyr lie Lys Tyr Asp Cys Asp Arg Gin Arg Gin<br>
305	310	315<br>
GAG TTC AGG AGG ACA AAC GGG AAA TGT TTG GTT TGG GGA AGA GCC CCG      1071<br>
Glu Phe Arg Arg Thr Asn Gly Lys Cys Leu Val Trp Gly Arg Ala Pro<br>
320	325	330<br>
TCA AAG ATT GTG GCG TCG TAT ACT ACA ACA TCT GGT GAA ACT AAA ACT     1119<br>
Ser Lys lie Val Ala Ser Tyr Thr Thr Thr Ser Gly Glu Thr Lys Thr<br>
335	340	345<br>
ACT CTT CTC TTA ACG TCT GGA TGG TGG GGA TTG GCT CGT CAT TTC CAT      1167<br>
Ser Leu Leu Leu Thr Ser Gly Trp Trp Gly Leu Ala Arg His Phe His<br>
350	355	360<br>
TAT GTT CCT GAG ATC TTA ACT GCT TTC TTC TGG ACC GTA CCG GCT CTC      1215<br>
Tyr Val Pro Glu He Leu Ser Ala Phe Phe Trp Thr Val Pro Ala Leu<br>
365	370	375	380<br>
TTC GAT AAC TTC TTG GCA TAG TTC TAG GTC CTC ACC CTT CTT CTC TTT      1263<br>
Phe Asp Asn Phe Leu Ala Tyr Phe Tyr Val Leu Thr Leu Leu Leu Phe<br>
385	390	395<br>
GAT CGA GCC AAG AGA GAC GAT GAC CGA TGC CGA TCA AAG TAT GGG AAA     1311<br>
Asp Arg Ala Lys Arg Asp Asp Asp Arg Cys Arg Ser Lys Tyr Gly Lys<br>
400	405	410<br>
TAT TGG AAG CTG TAT TGT GAG AAA GTC AAA TAC AGG ATC ATT CCG GGA      1359<br>
Tyr Trp Lys Leu Tyr Cys Glu Lys Val Lys Tyr Arg He lie Pro Gly<br>
415	420	425<br>
ATT TAT TGATTGTAAC GAAGTCTGTT GTTCTCATTT TCTACTTATT ACGTTAATTC       1415 He Tyr 430<br>
GAACGTTGGA ATCATCAAAA GACCGAGCCA AAACAAAAAT GCAAATTGAT GCGATAGACA    1475<br>
TTCTTTTGCT GAAAAAAAAA A	1496<br>
(2) INFORMATION FOR SEQ ID NO: 2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 430 amino acids<br>
(B)	TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
 (ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 2:<br>
Met Ala Glu Thr Val His Ser Pro lie Val Thr Tyr Ala Ser Met Leu<br>
15	10	15<br>
Ser Leu Leu Ala Phe Cys Pro Pro Phe Val lie Leu Leu Trp Tyr Thr<br>
20	25	30<br>
Met Val His Gin Asp Gly Ser Val Thr Gin Thr Phe Gly Phe Phe Trp<br>
35	40	45<br>
Glu Asn Gly Val Gin Gly Leu lie Asn lie Trp Pro Arg Pro Thr Leu<br>
50	55	60<br>
He Ala Trp Lys He He Phe Cys Tyr Gly Ala Phe Glu Ala He Leu<br>
65	70	75	80<br>
Gin Leu Leu Leu Pro Gly Lys Arg Val Glu Gly Pro He Ser Pro Ala<br>
85	90	95<br>
Gly Asn Arg Pro Val Tyr Lys Ala Asn Gly Leu Ala Ala Tyr Phe Val<br>
100	105	110<br>
Thr Leu Ala Thr His Leu Gly Leu Trp Trp Phe Gly He Phe Asn Pro<br>
115	120	125<br>
Ala He Val Tyr Asp His Leu Gly Glu He Phe Ser Ala Leu He Phe<br>
130	135	140<br>
Gly Ser Phe He Phe Cys Val Leu Leu Tyr He Lys Gly His Val Ala<br>
145	150	155	160<br>
Pro Ser Ser Ser Asp Ser Gly Ser Cys Gly Asn Leu He He Asp Phe<br>
165	170	175<br>
Tyr Trp Gly Met Glu Leu Tyr Pro Arg He Gly Lys Ser Phe Asp He<br>
180	185	190<br>
Lys Val Phe Thr Asn Cys Arg Phe Gly Met Met Ser Trp Ala Val Leu<br>
195	200	205<br>
Ala Val Thr Tyr Cys He Lys Gin Tyr Glu He Asn Gly Lys Val Ser<br>
210	215	220<br>
Asp Ser Met Leu Val Asn Thr lie Leu Met Leu Val Tyr Val Thr Lys<br>
225	230	235	240<br>
Phe Phe Trp Trp Glu Ala Gly Tyr Trp Asn Thr Met Asp lie Ala His<br>
245	250	255<br>
Asp Arg Ala Gly Phe Tyr He Cys Trp Gly Cys Leu Val Trp Val Pro<br>
260	265	270<br>
Ser Val Tyr Thr Ser Pro Gly Met Tyr Leu Val Asn His Pro Val Glu<br>
275	280	285<br>
Leu Gly Thr Gin Leu Ala He Tyr He Leu Val Ala Gly He Leu Cys<br>
290	295	300<br>
He Tyr He Lys Tyr Asp Cys Asp Arg Gin Arg Gin Glu Phe Arg Arg<br>
305	310	315	320<br>
Thr Asn Gly Lys Cys Leu Val Trp Gly Arg Ala Pro Ser Lys He Val<br>
325	330	335<br>
Ala Ser Tyr Thr Thr Thr Ser Gly Glu Thr Lys Thr Ser Leu Leu Leu<br>
340	345	350<br>
Thr Ser Gly Trp Trp Gly Leu Ala Arg His Phe His Tyr Val Pro Glu<br>
355	360	365<br>
He Leu Ser Ala Phe Phe Trp Thr Val Pro Ala Leu Phe Asp Asn Phe<br>
370	375	380<br>
Leu Ala Tyr Phe Tyr Val Leu Thr Leu Leu Leu Phe Asp Arg Ala Lys<br>
385	390	395	400<br>
Arg Asp Asp Asp Arg Cys Arg Ser Lys Tyr Gly Lys Tyr Trp Lys Leu<br>
405	410	415<br>
Tyr Cys Glu Lys Val Lys Tyr Arg He He Pro Gly He Tyr<br>
420	425	430<br>
(2) INFORMATION FOR SEQ ID NO: 3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 15 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
 (ii) MOLECULE TYPE: peptide<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: Peptide<br>
(B)	LOCATION:?<br>
(D) OTHER INFORMATION:/note= "residue 7 = Trp or Tyr"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: Peptide<br>
(B)	LOCATION:12<br>
(D) OTHER INFORMATION:/note= "residue 12 » His or Lys"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 3:<br>
Leu Leu Xaa Xaa Gly Trp Xaa Gly Xaa Xaa Arg Xaa Xaa Xaa Tyr<br>
15	10	15<br>
(2) INFORMATION FOR SEQ ID NO: 4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 29 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATION:10..29<br>
(D) OTHER INFORMATION:/note= "SEQUENCE ID No 1 FROM 76 TO 95"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 4:<br>
CGCGGATCCA TGGCGGAGAC TGTACATTC	29<br>
(2) INFORMATION FOR SEQ ID NO: 5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 28 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
 (ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATION:complement (10..28)<br>
(D) OTHER INFORMATION:/note= "COMPLEMENTARY SEQUENCE ID No 1 FROM 1350 TO 1368"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 5:<br>
CAGGGTACCT CAATAAATTC CCGGAATG	28<br>
(2) INFORMATION FOR SEQ ID NO: 6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 23 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..23)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 6:<br>
GATTACGCCA AGCTTTTCGA AAC	23<br>
(2) INFORMATION FOR SEQ ID NO: 7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 29 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 7:<br>
AGATCTTGAG AAGATGCGGC CAGCAAAAC	29<br>
(2) INFORMATION FOR SEQ ID NO: 8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 46 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 8:<br>
GGGGATCCGT GGTCGACGTA ATTTAGTGTG TGTATTTGTG TTTGCG	46<br>
(2) INFORMATION FOR SEQ ID NO: 9:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 17 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..17)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 9:<br>
GTAAAACGAC GGCCAGT	17<br>
(2) INFORMATION FOR SEQ ID NO: 10:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 25 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
 (ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 10:<br>
TTGAAGGTTC AACATCAATT GATTG	25<br>
(2) INFORMATION FOR SEQ ID NO: 11:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 38 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc « "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..38)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 11:<br>
GTGTGGCGGC CGCCTCCTTG TCAATATTAA TGTTAAAG	38<br>
(2) INFORMATION FOR SEQ ID NO: 12:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 34 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 12:<br>
CAAGGAGGCG GCCGCCACAC AAAAAGTTAG GTGT	34<br><br>
 (2) INFORMATION FOR SEQ ID NO: 13:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 25 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..25)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 13:<br>
TCTGCTTCCC TAGAACCTTC TTATG	25<br>
(2) INFORMATION FOR SEQ ID NO: 14:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 20 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..20)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 14:<br>
TACATTAGGT CCTTTGTAGC	20<br>
(2) INFORMATION FOR SEQ ID NO: 15:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 25 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
 (ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 15:<br>
GCGCTCAGCG GCCGCTTTCC AGTCG	25<br>
(2) INFORMATION FOR SEQ ID NO: 16:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 21 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 16:<br>
AATTGCGGCC GCGTACGTAT G	21<br>
(2) INFORMATION FOR SEQ ID NO: 17:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 21 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATION:complement (1..21)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 17:<br>
AATTCATACG TACGCGGCCG C	21<br><br>
 (2) INFORMATION FOR SEQ ID NO: 18:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 14 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 18:<br>
CAACGCGTCC TAGG	14<br>
(2) INFORMATION FOR SEQ ID NO: 19:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 22 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..22)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 19:<br>
AATTCCTAGG ACGCGTTGAG CT	22<br>
(2) INFORMATION FOR SEQ ID NO: 20:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 33 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 20:<br>
GATCCGCAGA TATCATCTAG ATCCCGGGTA GAT	33<br>
(2) INFORMATION FOR SEQ ID NO: 21:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 39 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..39)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 21:<br>
AGAGCTCAAG ATCTACCCGG GATCTAGATG ATATCTGCG	39<br>
(2) INFORMATION FOR SEQ ID NO: 22:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 34 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 22:<br>
CTTGAGCTCT ACGCAGCTGG TCGACACCTA GGAG	34<br>
(2) INFORMATION FOR SEQ ID NO: 23:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 28 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
 (ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc « "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATION:complement (1..28)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 23:<br>
AATTCTCCTA GGTGTCGACC AGCTGCGT	28<br>
(2) INFORMATION FOR SEQ ID NO: 24:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 40 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 24:<br>
GCGGATCTGC TCGAAGATTG CCTGCGCGTT GGGCTTGATC	40<br>
(2) INFORMATION FOR SEQ ID NO: 25:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 15 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 25:<br>
TCGACGGACG CGTGG	15<br>
(2) INFORMATION FOR SEQ ID NO: 26:<br><br>
 (i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 14 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc - "OLIGONUCLEOTIDE1<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_£eature<br>
(B)	LOCATIONcomplement (1..14)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 26:<br>
TCGACCACGC GTCC	14<br>
(2) INFORMATION FOR SEQ ID NO: 27:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 35 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 27:<br>
TGGCCGTCGT TTTACTCCTG CGCCTGATGC GGTAT	35<br>
(2) INFORMATION FOR SEQ ID NO: 28:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 15 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 28:<br>
GGCC6CAAAA CCAAA	15<br>
(2) INFORMATION FOR SEQ ID NO: 29:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 15 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..15)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 29:<br>
AGCTTTTGGT TTTGC	15<br>
(2) INFORMATION FOR SEQ ID NO: 30:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 20 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: 1inear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 30:<br>
GATCTATCGA TGCGGCCGCG	20<br>
(2) INFORMATION FOR SEQ ID NO: 31:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 20 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
 (ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION:   /desc = "OLIGONUCLEOTIDE'<br>
(ix) FEATURE:<br>
(A)	NAME/KEY: misc_feature<br>
(B)	LOCATIONcomplement (1..20)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 31:<br>
CGCGCGCGGC CGCATCGATA	20<br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	Nucleic acid sequence containing a sequence coding for a protein having a delta-5,7 sterol, delta-reductase activity, said nucleic acid being a DNA or an RNA, wherein the nucleic acid is a cDNA, in which the coding sequence codes for an A. thaliana protein having a delta-5,7 sterol,delta-7 reductase activity and having the nucleotide sequence of SEQ ID NO. 1:<br>
(Sequence Removed) <br>
2.	DNA   sequence   coding   for   a   protein   having   a   delta-5,7   sterol,delta-7<br>
reductase activity and which hybridizes with the sequence as claimed in<br>
claim   1   under  average   or  high   stringency  conditions  or  which  has  a<br>
sequence identity of approximately 60% and more with this sequence.<br>
3.	A. thaliana protein obtained by expressing SEQ ID N°. 1 as claimed in claim<br>
1 and having a delta-5,7 sterol, delta-7 reductase activity and having the<br>
amino acid sequence SEQ ID No. 2:<br>
(Sequence Removed) <br>
as well as the allelic variants and analogue of this sequence.<br>
4.	Protein having a A7Red activity as protein claimed in claim 3 and having<br>
an amino acid sequence having a sequence identity of about 60% and more<br>
with the sequence SEQ ID N°. 2.<br>
5.	Protein having a ATRed activity as protein claimed in claim 3 and having<br>
a crossed immunological reactivity with the A. thaliana as protein claimed in<br>
claim 3 and having protein as claimed in claim 3.<br>
6.	Protein having a A7Red activity as protein claimed in claim 3 obtain by<br>
expression of a DNA sequence as claimed in claims 1 and 2 in a host cell.<br>
7.	A  protein  as  claimed  in  claim  3   obtained  by  expression  of a  DNA<br>
sequence coding for SEQ ID N°. 2 in a host cell.<br>
8.	Protein as claimed in claim 6 or claim 7, wherein the host cell is a yeast.<br>
9.	Antibody directed against a protein having a delta-5,7   sterol,delta-7<br>
reductase activity as claimed in any one of claims 3 to 8.<br>
10.	Expression vector containing a DNA sequence as claimed in any one of<br>
claims land 2.<br>
11.	Host cell transformed by a vector as claimed in claim 10.<br>
12. Cloning process for a nucleic acid coding for a protein having a delta-5,7 sterol,delta-7 reductase activity in a microorganism comprising a screening method chosen from<br>
-	the resistance of the microorganism to nystatin or to an analogous compound<br>
whose toxicity depends on the presence of sterols carrying an unsaturation in<br>
position C-7,<br>
-	the hybridization of the nucleic acid with the nucleotide sequence of the<br>
sequence SEQ ID No. 1,<br>
-	the identification of the nucleic acid by using data processing techniques from<br>
DNA sequences isolated at random,<br>
-the direct expression of the protein followed by immunodetection using antibodies directed against the protein having the amino acid sequence SEQ ID No. 2.<br>
13.	DNA or RNA nucleic acid sequence as obtained by the process as claimed<br>
in claim 12.<br>
14.	Host cell transformed by a vector containing a DNA sequence as claimed<br>
in claim 13.<br>
15.	Host    cell    transformed    as    claimed    in    claim    11    or    claim    14<br>
in which the host is a yeast or a filamentous fungus.<br>
16.	Preparation       process       for       a       protein       having       a       delta-5,7<br>
sterol,delta-7 reductase activity in which a host cell transformed as claimed in<br>
any one of claims   11,   14  or   15 is cultured and the expressed protein is<br>
isolated.<br>
17.	Process as claimed in claim 16, wherein the host cell is a transformed<br>
yeast in which the coding DNA sequence is placed under the control of a yeast<br>
promoter.<br>
18.	Reduction process in vitro for a sterol unsaturated in position C-7 in<br>
which the  sterol to be reduced is incubated with the protein  obtained  as<br>
claimed in claim 16 or claim 17 and the reduced sterol obtained is optionally<br>
isolated.<br>
19.	Reduction       process       in       vivo       for       an       exogenic        sterol<br>
unsaturated     in     position     C-7     in     which     the     sterol     is     incubated<br>
with a host cell transformed as claimed in  any one of claims 11, 14 or 15 and the reduced sterol obtained is optionally isolated.<br>
20.	Reduction process in vivo for an endogenic sterol unsaturated in position<br>
C-7 in which a host strain transformed as claimed in claim 15 is cultured and<br>
the reduced sterol accumulated is optionally isolated.<br>
21.	Reduction process in vitro or in vivo as claimed in any one of claims 18<br>
to 20 in which the reduced sterol obtained is a substrate of the restriction<br>
enzyme of the side chain of cholesterol (P45oSCC).<br>
22.	Reduction process in vivo as claimed in claim 21 in which the endogenic<br>
sterol to be reduced is ergosta 5,7 diene 3-ol, ergosta 5,7,24(28) triene 3-ol or<br>
ergosta 5,7,22 triene 3-ol or a mixture of these.<br>
23.	Production process for pregnenolone in which a host cell transformed as<br>
claimed in claim 15 is cultured, the accumulated endogenic sterol or sterols<br>
reduced in position C-7 is/are  optionally isolated,  the reduced  sterols are<br>
incubated     in     the     presence     of     P450SCC,     and     optionally     in     the<br>
presence      of     adrenodoxin      reductase      (ADR)      and      of     adrenodoxin<br>
(ADX), and the pregnenolone obtained is optionally isolated.<br>
24.	Process   as   claimed   in   claim   23   in   which   the   host   cell   is   a<br>
yeast.<br>
25.	Production      process      for      pregnenolone      in      which      a      yeast<br>
transformed by one or more vectors allowing the coexpression of a protein<br>
having the activity of delta-5,7 sterol,delta-7 reductase and of P4soSCC, and<br>
optionally of ADR and ADX, is cultured and the free or esterified pregnenolone<br>
is optionally isolated.<br>
26.	Production process for pregnenolone as claimed in claim 25 in which a<br>
transformed   yeast   coexpressing   a   protein   having   delta-5,7   sterol,delta-7<br>
reductase, P45oSCC, ADR and ADX activity, is cultured.<br>
27.	Process as claimed in claim 26, in which the protein having the delta-5,7<br>
sterol,delta-7 reductase activity is the A. thaliana delta-7Red protein.<br>
28.	Process    as    claimed    in    claim    27    in    which    the    yeast    strain<br>
is the strain EC01/pCD63.<br>
29.	Transformed   yeast   strain   coexpressing   a   protein   having   delta-5,7<br>
sterol,delta-7 reductase, P450SCC, ADR and ADX activity and accumulating free<br>
or esterified pregnenolone.<br>
30.	Yeast strain as claimed in claim 29 in which the protein having the<br>
delta-5,7 sterol,delta-7 reductase activity is the A. thaliana delta-7Red protein.<br>
31.	Yeast strain as claimed in claim 30 named EC01/pCD63.<br>
32.	Human   DNA   sequence   as   claimed   in   claim   13   used   as   a   probe<br>
for diagnosing a congenital deficiency in delta-5,7 sterol,delta-7 reductase.<br>
33.	Nucleic acid sequence containing a sequence coding for a protein having<br>
a delta-5,7 sterol, delta-7 reductase activity substantially herein described with<br>
reference to the foregoing examples and the accompanying drawings.<br>
34.	cDNA sequence substantially as herein described with reference to the<br>
foregoing examples and the accompanying drawings.<br>
35.	A. thaliana protein substantially as herein described with reference to the<br>
foregoing examples and the accompanying drawings.<br>
36.	Preparation   process  for  a  protein  having  a  delta-5,7   sterol,   delta-7<br>
reductase  activity  substantially  as  herein  described  with  reference  to  the<br>
foregoing examples and the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0176-del-1996-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">0176-del-1996-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0176-del-1996-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTEwLnBkZg==" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-10.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1mb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE3Ni1kZWwtMTk5Ni1ncGEucGRm" target="_blank" style="word-wrap:break-word;">0176-del-1996-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUFic3RyYWN0LSgyMC0xMS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Abstract-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUNsYWltcy0oMjAtMTEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Claims-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUNsYWltcy0oMjItMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Claims-(22-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjAtMTEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Correspondence-Others-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Correspondence-Others-(22-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDIwLTExLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Description (Complete)-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LURyYXdpbmdzLSgyMC0xMS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Drawings-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUZvcm0tMS0oMjAtMTEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Form-1-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LWRlbC0xOTk2LWZvcm0tMTMtKDIwLTExLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">176-del-1996-form-13-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUZvcm0tMy0oMjItMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Form-3-(22-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LUdQQS0oMjAtMTEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-GPA-(20-11-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LVBldGl0aW9uLTEzNy0oMjItMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Petition-137-(22-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2LURFTC0xOTk2LVBldGl0aW9uLTEzOC0oMjItMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">176-DEL-1996-Petition-138-(22-04-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="184879-a-process-for-the-preparation-of-milled-malted-cereals-flour.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="185328-method-and-compositions-for-treatment-of-diseases.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>185030</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>0176/DEL/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Oct-2000</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Jun-2001</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Jan-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ROUSSEL UCLAF</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>102, ROUTE DE NOISY F-93230 ROMAINVILLE, FRANCE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DENIS POMPON</td>
											<td>9, PLACE DU MARCHE NEUF, F-91190 GIF SUR YVETTE, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>XAVIER CHENIVESSE</td>
											<td>64,RUE JEAN JACQUES ROUSSEAU, F-94200 IVRY SUR SEINE, FRANCE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CATHERINE DUPORT</td>
											<td>6, RUELLE DU ROYAUME, F-91440 BURES SUR YVETTE, FRANCE</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ERIC LECAIN</td>
											<td>70 AVENUE CARNOT, F-942300 CACHAN, FRANCE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>95 01723</td>
									<td>1995-02-15</td>
								    <td>France</td>
								</tr>
								<tr>
									<td>2</td>
									<td>95 06517</td>
									<td>1995-06-01</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/185030-dna-sequence-coding-for-a-protein-of-a-thaliana-having-a-delta-5-7-sterol-delta-7-reductase-activity-delta-7-red-protein-production-process-strains-of-transformed-yeasts-uses by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:58:17 GMT -->
</html>
