<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/187836-method-of-producing-protein-hydrolysates by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:34:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 187836:METHOD OF PRODUCING PROTEIN HYDROLYSATES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD OF PRODUCING PROTEIN HYDROLYSATES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Abstract The present invention relates to methods of producing protein hydrolysates, comprising lidding to a proteinaceous material one or more aminopeptidase(s) having glycine releasing properties and one or more additional proteases wherein the amount of glycine produced is greater than the amount of glycine produced by the one or more additional proteases alone under the same conditions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br><br>
Background of the Invention<br>
Field of the Invention<br>
The present mvention relates to methods of producing a protein hydrolysate, comprising adding to a proteinaceous material one or more polypeptide(s) having glycine releasing activity and one or more additional proteases wherein the amount of glycine produced is greater than the amount of glycine produced by the one or more additional proteases alone under the same conditions.<br>
Description of the Related Art<br>
Various food products, e.g., soups, sauces and seasonings, contain flavoring agents obtained by hydrolysis of proteinaceous materials. This hydrolysis is conventionally accomplished using strong hydrochloric acid, followed by neutralization with sodium hydroxide. However, such chemical hydrolysis leads to severe degradation of the amino acids obtained during the hydrolysis, and also to hazardous byproducts formed in the course of this chemical reaction. Increasing concern over the use of flavoring agents obtained by chemical hydrolysis has led to the development of enzymatic hydrolysis processes.<br>
Enzymatic hydrolysis processes of proteinaceous materials aim at obtaining a high degree of hydrolysis (DH), and this is usually attained using a complex of unspecific acting proteolytic enzymes (i.e., unspecific-acting endo- and exo-peptidases). For example, WO 94/25580 describes a method for hydrolyzing proteins by use of an unspecific acting<br><br>
enzyme preparation obtained from Aspergillus oryzae. Specific acting proteolytic enzymes have not been used for this purpose because such enzymes only lead to an inadequate degree of hydrolysis.<br>
Polypeptides having aminopeptidase activity catalyze the removal of one or more amino acid residues from the N-terminus of peptides, polypeptides, and proteins. Such polypeptides are classified under the Enzyme Classification Number E.G. 3.4.11.- of the International Union of Biochemistry and Molecular Biology.<br>
WO 96/28542 discloses an aminopeptidase which has a moleculer weight of 35 kDa.<br>
JP-7-5034631 (Noda) discloses a leucine aminopeptidase obtained from yellow koji mold,<br>
which includes Aspergillus oryzae.   JP-7-4021798 (Zaidan Hojin Noda Sangyo) discloses<br>
the production of miso by adding a leucine aminopeptidase II prepared by cultivating a<br>
number of strains, including Aspergillus oryzae strain 460 (ATCC 20386) and strain lAM<br>
2616.     Aspergillus  oryzae  strain  460   is  known  to  produce  a  number  of leucine<br>
aminopeptidases of which three have a molecular weight of 26.5, 56, and 61 kDa by gel<br>
filtration (Nakada etai, 1972, Agricultural and Biological Chemistry 37: 757-765; Nakada<br>
et ai, 1972, Agricultural and Biological Chemistry 37: 161-11 A\ and Nakada et at., 1972,<br>
Agricultural and Biological Chemistry 37: 115-1S2; respectively).    Penicillium citrium<br>
produces an intracellular leucine aminopeptidase with a molecular weight of 65 kDa by<br>
SDS-PAGE (Kwon et at., 1996, Journal of Industrial Microbiology 17: 30-35).<br>
WO 97/04108 (Roehm) discloses DNA encoding an Aspergillus sojae leucine aminopeptidase. Chang and Smith (1989, Journal of Biological Chemistry 264: 6979-6983) disclose the molecular cloning and sequencing of a gene encoding a vacuolar aminopeptidase from Saccharomyces cerevisiae. Chang et ai (1992, Journal of Biological Chemistry 267: 8007-8011) disclose the molecular cloning and sequencing of a gene encoding a methionine aminopeptidase from Saccharomycse cerevisiae.<br>
The production of protein hydrolysates with desirable organoleptic properties and high degrees of hydrolysis generally requires the use of a mixture of peptidase activities. It would be desirable to provide a single component peptidase enzyme which has activity useful for improving the organoleptic properties and degree of hydrolysis of protein hydrolysates used in food products either alone or in combination with other enzymes.<br>
It is an object of the present invention to provide improved methods for producing protein hydrolysates with desirable organoleptic qualities and/or high degrees of hydrolysis.<br><br>
Accordingly the present invention provides a method of producing a protein hydrolysate from a proteinaceous material such as herein described comprising the steps of treating said proteinaceous material with atleast two enzymes, one of said two enzymes being one or more polypeptides having glycine releasing activity, and the other being one or more proteases, under known protein hydrolysing conditions and isolating the protein hydrolysate from the reaction mixture by known means, the glycine content of said protein hydrolysate being greater by atleast 10% to 400% than that produced by treating said proteinaceous material with one or more proteases alone under the same condtions.<br>
With reference to the accompanying drawings:<br>
Fgiure 1 shows the nucleic acid sequence and the ieduced amino acid sequence of an Aspergillus oryzae ATCC 20386 aminopeptidase (SEQ ID N0S:1 and 2, respectively).<br>
Figure 2 shows a restriction map of pMWR3.<br>
Figure 3 shows a restriction map of pEJG19.<br>
Figure 4 shows a restriction map of pDMlSl.<br>
Figure 5 shows a restriction map of pEJG28.<br>
Figure 6 shows the activity of a Sphingomonas capsulata aminopeptidase versus pH measured at room temperature using a .00mg/ml solution of Ala-pNA in DMSO.<br><br>
Figure 7 shows the activity of a Sphingomonas oapsulata aminopeptidase versus the temperature in 50mM sodium phosphate buffer pH 7.5.<br>
Figure 8 shows the activity of a SphingonQnas o&amp;psulata aminopeptidase versus the temperature in 50mM sodium phosphate buffer pH 7.5.<br>
Figure  9  shows  the effect on the DH	when  adding a<br>
Sphingomonas oapgulata crude enzyme extract and	a SPhlngQMOnaa<br>
capsulata   aminopeptidase  to  respectively	low  and   high<br>
Flavourayme " background dosage.<br>
Figure  10  shows the effect on the DH	when  adding  a<br>
Sphingomonas capsulata crude enzyme extract and	a Sphinflomonas<br>
capsulata   aminopeptidase   to  respectively	low  and   high<br>
Flavourzyme" background dosage.<br>
Figure 11 shows the DH obtained by different combinations of Flavourzyme ' Alcalase" and a Sphingomonas Gapsulata aminopeptidase.<br><br>
Figure 12 she wins the DH obtained by different combinations of Flavourzyme™, Alcalase® and a Sphingomonas capsulata aminopeptidase.<br>
Detailed Description of the Invention<br>
The present invention relates to method of producing a protein hydrolysate, comprising adding to a proteinaceous material one or more polypeptide(s) having glycine releasing activity and one or more additional proteases wherein the amount of glycine produced is greater than the amount of glycine produced by the one or more additional jroteases alone under the same conditions. Because of the activity of the glycine-releasing polypeptide, the methods of the present invention are able to produce protein hydrolysates laving a higher degree of hydrolysis or the same degree of hydrolysis using lower amounts of enzyme.<br>
In a preferred embodiment, the amount of glycine produced is between 10-400%, referably 50-350%, more preferably 100-350%, even more preferably 150-350%, and lost preferably 200-350%, than the amount of glycine produced by the one or more dditional proteases under the same conditions..<br>
The enzymatic treatment(s) may take place at any convenient temperature at which le enzymes do not become mactivated, preferably in the range of from about 20°C to 30Ut 70°C. In accordance with established practice, the enzymes may be inactivated by icreasing the temperature of the incubation mixture to a temperature where the enzymes ;come inactivated, e.g., to above about 70*'C, or by decreasing the pH of the incubation ixture to a point where the enzymes become inactivated, e.g., below about 4.0.<br>
In a preferred embodiment, the methods of the present invention produce a protein 'drolysate having a degree of hydrolysis between 35-90%, preferably 45-80%, more eferably 50-75%, even more preferably 55-75%, and most preferably 60-70%.<br>
The degree of hydrolysis (DH) is the percentage of the total number of amino bonds<br>
a protein that has been hydrolyzed by a proteolytic enzyme.   The DH is defined by the<br>
llowing formula:<br>
Number of peptide bonds cleaved<br>
I =	X 100%<br>
Total number of peptide bonds<br><br>
The DH may be calculated according to Adler-Nissen; Enzyme Hydrolysis of Food Proteins; Elsvier Applied Science Publishers Ltd. (1986), page 122.<br>
Proteinacous Material<br>
The methods of the present invention comprise treating a proteinaceous material with at least two enzymes. The proteinaceous material may consist of intact proteins, prehydrolyzed proteins {i.e., peptides), or a mixture thereof.<br>
In a preferred embodiment, the proteinaceous material is a foodsmff. The proteinaceous material may be of vegetable or animal origin.<br>
In another preferred embodiment, the proteinaceous material is of animal origin, preferably milk protein, whey protein, casein, meat protein, fish protein, blood protein, egg white gelatin, or lactoalbumin.<br>
In another preferred embodiment, the proteinaceous material is of vegetable origin, preferably soy protein, grain proteins, e.g.. wheat gluten, corn gluten, barley, rye, oat, rice, zein, lupine, cotton seed protein, rape seed protein, peanut, alfalfa protein, pea protein, fabaceous bean protein, sesame seed protein, or sunflower.<br>
Polypeptides Having Glycine Releasing Activity<br>
One of the enzymes added to the proteinaceous material is a polypeptide having 'lycine releasing activity. The term "glycine releasing activity" is defined herein as a peptidase activity which catalyzes the removal of glycine from the N-terminal end of &gt;eptides, oligopeptides or proteins. The polypeptides may also have peptidase activity vhich is capable of cleaving one or more amino acids X from the N-terminus of a peptide, polypeptide, or protein, wherein X represents any amino acid residue selected from the 'roup consisting of Ala, Arg, Asn, Asp, Cys, Gin, Glu, His, He, Leu, Lys, Met, Phe, Pro, !er, Thr, Trp, Tyr, and Val, but at least Leu, Glu, Gly, Ala, and/or Pro.<br>
The polypeptide(s) may be added to the proteinaceous substrate in an effective mount conventionally employed in protein hydrolysis processes, preferably in the range of rom about 0.1 to about 100,000 aminopeptidase units per 100 g of protein, and more referably in the range of from about 1 to about 10,000 aminopeptidase units per 100 g of rotein.    As defined herein, one aminopeptidase unit (APU) is the amount of enzyme<br><br>
needed to release 1 micromole of p-nitroanilide per minute from Leu-p-nitroanilide (Sigma Chemical Co., St. Louis MO) under the specified conditions. Alternatively, the aminopeptidase may be employed preferably in the range of from about 0.5 to about 500 LAPU/g of protein, and more preferably in the range of from about 5 to about 50 LAPU/g of protein. LAPU is defined as the leucine aminopeptidase activity which is determined as described in AF 298/1-GB (available on request from Novo Nordisk A/S, Denmark).<br>
In a preferred embodiment, the polypeptide preferentially cleaves glycine.<br>
In another preferred embodiment, the polypeptide is selected from the group consisting of:<br>
(a)	a polypeptide having an amino acid sequence which has at least 50% identity<br>
with the amino acid sequence of SEQ ID N0:2;<br>
(b)	a polypeptide encoded by a nucleic acid sequence which hybridizes under<br>
mediuni stringency conditions with (i) the nucleic acid sequence of SEQ ID N0;1, (ii) its<br>
complementary strand, or (iii) a subsequence thereof;<br>
(c)	an allelic variant of (a) or (b); and<br>
(4) a fragment of (a), (b), or (c), wherein the fragment has aminopeptidase activity;<br><br>
(b) a polypeptide encoded by a nucleic acid sequence which hybridi/cs under medium stringency conditions with (i) the nucleic acid sequence or "FO ID NO:!0, (ii) its complementary strand, or (iii) a subsequence thereof;<br><br>
(c)	an allelic variant of (a) or (b); and<br>
(d)	a fragment of (a), (b), or (c), wherein the fragment has aminopeptidase activity; and<br>
(e)	a polypeptide which (a) has aminopeptidase activity in the pH range between pH<br>
5.0-8.5 measured at 37°C, (b) an isoelectric point (pi) in die range of 7.4-8.5; (c) has<br>
aminopeptidase activity in the temperature range of 20-55°C, measured at pH 7.5 using Gly-<br>
pNA m Tris-HCl buffer; (d) hydrolyzes Ala-pNA, Gly-pNA, Leu-pNA, Glu-pNA, Asp-pNA,<br>
Lys-pNA, Ile-pNA and Val-pNA; (e) does not hydroiyze Phe-pNA nor Pro-pNA; (f) is not<br>
inhibited by phenylmethanesulfonyl fluoride, slighdy inhibited by EDTA, di-isopropyl fluoro<br>
phosphate, p-chloromercuribenzoic acid and iodoacetic acid, completely inhibited by o-<br>
phenanthrolin, and/or (g) is obtained from a strain belonging to Sphingomonas and has a<br>
molecular mass of 67 ± 5 kDa.<br>
In a preferred embodiment, the polypeptide has an amino acid sequence which has a degree of identity to the amino acid sequence of SEQ ID N0:2 or 11 of at least about 50%, preferably at least about 60%, preferably at least about 70%, more preferably at least about 80%, even more preferably at least about 90%, most preferably at least about 95%, and even most preferably at least about 97%, which have aminopeptidase activity (hereinafter "homologous polypeptides"). In a preferred embodiment, the homologous polypeptides have an amino acid sequence which differs by five amino acids, preferably by four amino acids, more preferably by three amino acids, even more preferably by two amino acids, and most preferably by one amino acid from the amino acid sequence of SEQ ID N0:2 or 11. For purposes of the present invention, the degree of identity between two amino acid sequences is determined by the Clustal method (Higgins, 1989, CABIOS 5: 151-153) with an identity table, a gap penalty of 10, and a gap length penalty of 10.<br>
Preferably, the polypeptides comprise the amino acid sequence of SEQ ID N0:2 or 11, an allelic variant; and a fragment thereof, wherein the fragment has aminopeptidase activity. In a more preferred embodiment, the polypeptides comprise the amino acid sequence of SEQ ID N0:2 or 11. In another preferred embodiment, the polypeptide has the amino acid sequence of SEQ ID N0:2 or 11 or a fragment thereof, wherein the fragment has aminopeptidase activity. A fragment of SEQ ID N0:2 is a polypeptide having one or more amino acids deleted from the amino and/or carboxy terminus of this amino<br><br>
acid sequence.    In a most preferred embodiment, the polypeptide has the amino acid sequence of SEQ ID N0:2 or 11.<br>
Preferably, a fragment contains at least 330 amino acid residues, more preferably at least 380 amino acid residues, and most preferably at least 430 amino acid residues.<br>
An allelic variant denotes any of two or more alternative forms of a gene occupying the same chomosomal locus. Allelic variation arises naturally through mutation, and may result in phenotypic polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. The term allelic variant is also used to denote a protein encoded by an allelic variant of a gene.<br>
The amino acid sequences of the homologous polypeptides may differ from the amino acid sequence of SEQ ID NO:2 or 11 by an insertion or deletion of one or more amino acid residues and/or the substitution of one or more amino acid residues by different amino acid residues. Preferably, amino acid changes are of a minor nature, that is conservative amino acid substimtions that do not significantly affect the folding and/or activity of the protein; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain.<br>
Examples of conservative substimtions are within the group of basic amino acids (such as arginine, lysine and histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine and valine), aromatic amino acids (such as phenylalanine, tryptophan and tyrosine), and small amino acids (such as glycine, alanine, serine, threonine and methionine). Amino acid substitutions which do not generally alter the specific activity are known in the art and are described, for example, by H. Neurath and R.L. Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/Ile, Asp/Glu, Thr/Ser, Ala/Gly, Ala/Thr, Ser/Asn, Ala/Val, Ser/GIy, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/Ile, Leu/Val, Ala/Glu, and Asp/Gly as well as these in reverse.<br><br>
In another preferred embodiment, the polypeptide are encoded by nucleic acid sequences which hybridize under low stringency conditions, more preferably medium stringency conditions, and most preferably high stringency conditions. with an oligonucleotide probe which hybridizes under the same conditions with the nucleic acid sequence of SEQ ID N0:1 or 10 or their complementary strand (J. Sambrook, E.F. Fritsch, and T. Maniatus, 1989, Molecular Cloning. A Laboratory Manual, 2d edition, Cold Spring Harbor, New York); or allelic variants and fragments of the polypeptides, wherein the fragments have aminopeptidase activity.<br>
Hybridization indicates that the nucleic acid sequence hybridizes to the oligonucleotide probe corresponding to the polypeptide encoding part of the nucleic acid sequence shown in SEQ ID N0:1 or 10, under low to high stringency conditions {i.e., prehybridization and hybridization at 42°C in 5X SSPE, 0.3 % SDS, 200 ng/ml sheared and denamred salmon sperm DNA, and either 25, 35 or 50% formamide for low, medium and high stringencies, respectively), following standard Southern blotting procedures.<br>
The amino acid sequence of SEQ ID NO:2 or 11 or a partial sequence thereof may be used to design an oligonucleotide probe, or a nucleic acid sequence encoding a polypeptide of the present invention, such as the nucleic acid sequence of SEQ ID N0:1 or 10, or a subsequence thereof, may be used to identify and clone DNA encoding polypeptides having aminopeptidase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 15, preferably at least 25, and more preferably at least 40 nucleotides in length. Longer probes can also be used. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with "P, H, S, biotin, or avidin).<br>
Thus, a genomic, cDNA or combinatorial chemical library prepared from such other organisms may be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having aminopeptidase activity. Genomic or other DNA from such other organisms may be separated by agarose or polyacrylamide gel electrophoresis, or other separation techniques.   DNA from the libraries or the separated DNA may be<br><br>
transferred to and immobilized on nitrocellulose or other suitable carrier material. In order to identify a clone or DNA which is homologous with SEQ ID N0:1 or 10, the carrier material is used in a Southern blot in which the carrier material is finally washed three times for 30 minutes each using 2 x SSC, 0.2% SDS preferably at least SCC, more preferably at least 55°C, more preferably at least 60°C, more preferably at least 650, even more preferably at least 70°C, and most preferably at least 75°C. Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using X-ray film.<br>
In another preferred embodiment, the polypeptides have the following physicochemical properties: (a) a pH optimum in the range of from about pH 7.27 to about pH 10.95 determined at ambient temperature in the presence of Ala-para-nitroanilide; (ii) a temperature stability of 90% or more, relative to initial activity, at pH 7.5 determined after incubation for 20 minutes at 60°C in the absence of substrate; and (iii) an activity towards Xaa-para-nitroaniiide wherein Xaa is selected from the group consisting of Leu, Gtu, Gly, Ala, and Pro.  The polypeptides also have the ability to hydrolyze other substrates, or<br>
In a preferred embodiment, the pH optimum in the range of from about pH 7.27 to about pH 10.95, more preferably in the range of from about pH 8.03 to about pH*10.95, and most preferably in the range of from about pH 8.62 to about pH 10.51 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide.<br>
In another preferred embodiment, the polypeptide has a molecular mass of 67 ± 5 kDa, more preferably a molecular mass of 67 ± 2 kDa and is obtained from a strain, belonging to Sphingomonas, more preferably Sphingomonas capsulata and most preferably Sphingomonas capsulata IFO 12533. The molecular mass is measured by SDS-polyacrylamide gel electrophoresis (SDS-PAGE) using a 10-15% gradient gel on the apparams of FAST System from Pharmacia (Uppsala, Sweden). The molecular massof the enzyme was estimated from the regression line of molecular weight markers as follows: phosphorylase b (94 kDa) albumin (67 kDa)<br>
ovalbumin (43 kDa)	:<br>
carbonic anhydrase (30 kDa) trypsine inhibitor (20.1 kDa)<br><br>
a-lactalbumine (14.4 kDa)<br>
In another preferred embodiment, the polypeptide (a) has aminopeptidase activity in the pH range between pH 5.0-8.5 measured at 37°C, more preferably in the pH range 6.5-8.0, and even more preferably having the highest aminopeptidase activity in the pH range 7.0-7.5 measured at 37°C; (b) an isoelectric point (pi) in the range of 7.4-8.5; (c) has aminopeptidase activity in the temperature range of 20-55°C, measured at pH 7.5 using Gly-pNA in Tris-HCl buffer; (d) hydrolyzes Ala-pNA, Gly-pNA, Leu-pNA, Glu-pNA, Asp-pNA, Lys-pNA, Ile-pNA and Val-pNA; (e) does not hydrolyze Phe-pNA nor Pro-pNA; and/or (f) is not inhibited by phenyhnethanesulfonyl fluoride, slightly mhibited by EDTA, di-isopropyl fluoro phosphate, p-chloromercuribenzoic acid and iodoacetic acid, completely inhibited by o-phenanthrolin.<br>
The isoelectric point of the aminopeptidase is variable since the pi of the freshly prepared enzyme is estimated as 8.4 using an activity staining method, however, pi varies to 7.4-8.5 during the course of purification and/or storage. The uncertainty of pi may be due to the self digestion of the aminopeptidase from its amino terminal.<br>
In a more preferred embodunent, the polypeptide is obtained from an Aspergillus oryzae strain, and most preferably from Aspergillus oryzae ATCC 20386 or a mutant strain thereof, e.g., the polypeptide with the amino acid sequence of SEQ ID N0:2.<br>
In another preferred embodiment, the polypeptide is obtained from a Sphingomonas capsulata strain, e.g., Sphingomonas capsulata is IFO 12533, which is a bacterial strain kept at the Institute for Fermentation, Osaka, Japan, e.g.y the polypeptide with the amino acid sequence of SEQ ID NO: 11 (the predicted signal peptide consists of the first 32 amino acids).<br>
In another embodiment, the polypeptides have immunochemical identity or partial immunochemical identity to the polypeptide having the amino acid sequence of SEQ ID N0:2 or 11. The immunochemical properties are determined by immunological cross-reaction identity tests by the well-known Ouchterlony double immunodiffusion procedure. Specifically, an antiserum containing antibodies which are immunoreactive or bind to epitopes of the polypeptide having the amino acid sequence of SEQ ID N0:2 or 11 are prepared by immunizing rabbits (or other rodents) according to the procedure described by Harboe and Ingild, In N.H. Axelsen, J. Kr0ll, and B. Weeks, editors, A Manual of<br><br>
Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or Johnstone and Thorpe, Immunochemistry in Practice, Blackwell Scientific Publications, 1982 (more specifically pages 27-31). A polypeptide having immunochemical identity is a polypeptide which reacts with the antiserum in an identical fashion such as total fusion of precipitates, identical precipitate morphology, and/or identical electrophoretic mobility using a specific immunochemical technique. A further explanation of immunochemical identity is described by Axelsen, Bock, and Krell, In N.H. Axelsen, J. KrcfU, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Pubhcations, 1973, Chapter 10. A polypeptide having partial immunochemical identity is a polypeptide which reacts with the antiserum in a partially identical fashion such as partial fusion of precipitates, partially identical precipitate morphology, and/or partially identical electrophoretic mobility using a specific immunochemical technique. A further explanation of partial hnmimochemical identity is described by Bock and Axelsen, In N.H. Axelsen, J. Kr0ll, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 11.<br>
Polypeptides encoded by nucleic acid sequences which hybridize with an oligonucleotide probe which hybridizes with the nucleic acid sequence of SEQ ID N0:1 or 10, their complementary strands, or allelic variants and subsequences of SEQ ID N0:1 or 10; allelic variants and fragments of the polypeptides; or the homologous polypeptides and polypeptides having identical or partially identical immunological properties may be obtained from microorganisms of any genus.<br>
In a preferred embodiment, these polypeptides may be obtained from a bacterial source. For example, these polypeptides may be obtained from a gram positive bacterium 3uch as a Bacillus strain, e.g.. Bacillus alkalophilus, Bacillus amyloliquefaciens. Bacillus brevis, Bacillus circulans, Bacillus coagulans. Bacillus lautus. Bacillus lentus. Bacillus licheniformis. Bacillus megaterium. Bacillus stearothermophilus, Bacillus subtilis, or Sacillus thuringiensis; or a Streptomyces strain, e.g., Streptomyces lividans or Streptomyces nurinus; or fi-om a gram negative bacterium, e.g., E. coli or Pseudomonas sp.<br>
The polypeptides may be obtained from a fungal source, and more preferably from a east strain such as a Candida, Kluyveromyces, Pichia, Saccharomyces, )chizosaccharomyces, or Yarrowia strain; or a filamentous fiingal strain such as an icremonium,    Aspergillus,    Aureobasidium,    Cryptococcus,    Filibasidium,    Fusarium,<br><br>
Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, PaecilomyceSy Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Trichoderma strain.<br>
In a preferred embodiment, the polypeptides are obtained from a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis strain.<br>
In another preferred embodiment, the polypeptides are obtained from a Fusarium bactridioidesy Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminumy Fusarium heterosporum Fusarium negundi Fusarium oxysporumy Fusarium reticulatumy Fusarium roseum, Fusarium sambucinumy Fusarium sarcochroumy Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatumy Humicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophilay Neurospora crassa, Penicillium purpurogenum, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatumy Trichoderma reesei, or Trichoderma viride strain.<br>
The polypeptides are preferably obtained from species of Aspergillus including, but not limited to, Aspergillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, or Aspergillus oryzae.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g.y anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents. The polypeptides may also be obtained from microorganisms which are synonyms of Aspergillus as defined by Raper, K.D. and Fermel, D.I., 1965, The Genus Aspergillus, The Wilkins Company, Baltimore. Aspergilli are mitosporic fungi characterized by an aspergillum conprised of a conidiospore stipe with no known teleomorphic states terminating in a vesicle, which in turn bears one or two layers of synchronously formed specialized cells, variously referred to as sterigmata or phialides, and asexually formed spores referred to as conidia. Known teleomorphs of Aspergillus include Eurotium, Neosartorya, and Emericella. Strains of Aspergillus and teleomorphs thereof are readily accessible to the public in a number of culture collections, such  as   the  American  Type  Culmre  Collection  (ATCC),   Deutsche  Sammlung  von<br><br>
Mikroorganismen und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such polypeptides may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The nucleic acid sequence may then be derived by similarly screening a genomic or cDNA library of another microorganism. Once a nucleic acid sequence encoding the polypeptide has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e.g., Sambrook et al., 1989, supra).<br>
For purposes of the present invention, the term "obtained from" as used herein in connection with a given source shall mean that the polypeptide is produced by the source or by a cell in which a gene from the source has been mserted.<br>
The polypeptide(s) may be a fiised polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fiised polypeptide is produced by fiismg a nucleic acid sequence (or a portion thereof) encoding another polypeptide to a nucleic acid sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
Additional Protease(s)<br>
In the methods of the present invention, the proteinaceous material also is treated widi one or more additional proteases. The additional protease(s) may be added simultaneously with the polypeptide(s) or consecutively. The additional protease(s) may be added to the proteinaceous material in an effective amount conventionally employed in protein hydrolysis processes.<br>
The one or more additional proteases may be endopeptidases. The endopeptidase(s) may be added to the proteinaceous substrate in an effective amount conventionally employed in protein hydrolysis processes, preferably in the range of from about 0.05 to<br><br>
about 15 AU/lOO g of protein, and more preferably from about 0.1 to about 8 AU/lOO g of protein. One AU (Anson Unit) is defined as the amount of enzyme which under standard conditions (i.e., 25*C, pH 7.5 and 10 min. reaction time) digests hemoglobin at an initial rate such that there is liberated per minute an amount of TCA soluble product which gives the same color with phenol reagent as one milli-equivaient of tyrosine. The analytical method AF 4/5 is, available upon request from Novo Nordisk A/S, Denmark, which is incorporated herein by reference.<br>
In a preferred embodiment, the endopeptidase is a glutamyl endopeptidase (EC 3.4.21.19); a lysyl endopeptidase (EC 3.4.21.50); a leucyl endopeptidase (EC 3.4.21.57); a glycyl endopeptidase (EC 3.4.22.25); a prolyl endopeptidase (EC 3.4.21.26); trypsin (EC 3.4.21.4) or a trypsin-Iike (lysine/argmine specific) endopeptidase; or a peptidyl-Asp metalloendopeptidase (EC 3.4,24.33).<br>
The glutamyl endopeptidase (EC 3.4.21.19) may preferably be obtained from a Bacillus strain, in particular Bacillus licheniformis and Bacillus subtilis, a Staphylococcus strain, in particular Staphylococcus aureus, a Streptomyces strain, in particular Streptomyces thermovulgaris and Streptomyces griseus, or a Actinomyces strain.<br>
The lysyl endopeptidase (EC 3.4.21.50) may preferably be obtained from a Achromobacter strain, in particular Achromobacter lyticus, a Lysobacter strain, in particular Lysobacter enzymogenes, or a Pseudomonas strain, in particular Pseudomonas aeruginosa.<br>
The leucyl endopeptidase (EC 3.4.21.57) may be of plant origin. The glycyl endopeptidase (EC 3.4.22.25) may preferably be obtained from the papaya plant {Carica papaya).<br>
The prolyl endopeptidase (EC 3.4.21.26) may preferably be obtained from a Flavobacterium strain, or it may be of plant origin.<br>
The trypsin-like endopeptidase may preferably be obtained from a Fusarium strain, in particular Fusarium oxysporum, e.g., as described in WO 89/06270 or WO 94/25583.<br>
The peptidyl-Asp metalloendopeptidase (EC 3.4.24.33) may preferably be obtained from a Pseudomonas strain, in particular Pseudomonas fragi.<br>
The endopeptidase may be obtained from a strain of Bacillus, preferably Bacillus amyloliquefaciens. Bacillus lentus, Bacillus licheniformis or Bacillus subtilis, a strain of Staphylococcus, preferably Staphylococcus aureus, a strain of Streptomyces, preferably<br><br>
Streptomyces thermovularis or Streptomyces griseus, a strain of Actinomyces species, a strain of Aspergillus, preferably Aspergillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, or Aspergillus sojae, or a strain of Fusarium, preferably Fusarium venenatum.<br>
In another preferred embodiment, the specific acting proteolytic enzyme is an exo-peptidase that may act from either end of the peptide.<br>
In a preferred embodiment, the exopeptidase is an aminopeptidase such as a leucyl aminopeptidase (EC 3.4.11.1); a dipeptide aminopeptidase or a tripeptide aminopeptidase (EC 3.4.11.4).<br>
In another preferred embodiment, the exopeptidase is a carboxypeptidase such as a proline carboxypeptidase (EC 3.4.16.2); a carboxypeptidase A (EC 3.4.17,1); a carboxypeptidase B (EC 3.4.17.2); a carboxypeptidase C (EC 3.4.16.5); a carboxypeptidase D (EC 3.4.16.6); a lysine (arginine) carboxypeptidase (EC 3.4.17.3); a glycine carboxypeptidase (EC 3.4.17.4); an alanine carboxypeptidase (EC 3.4.17.6); a glutamate carboxypeptidase (EC 3.4.17.11); a peptidyl-dipeptidase A (EC 3.4.15.1); or a peptidyl-dipeptidase (EC 3.4.15.5).<br>
In a most preferred embodiment, the additional protease is a plurality of proteases sold under the name FLAVOURZYME™ manufactured by Novo Nordisk A/S, Denmark. The protease formulation contains at least five Aspergillus oryzae-dtnvtd proteases and peptidases of different molecular weight.<br>
Additional Enzymes<br>
In a preferred embodiment, the proteinaceous material is treated with one or more idditional enzymes. The one or more additional enzymes may be an amylase, :arbohydrase, cellulase, esterase, alpha-gaiactosidase, beta-galactosidase, glucoamylase, ilpha-glucosidase, beta-glucosidase, lipase, pectinolytic enzyme, peptidoglutaminase, )hytase, transglutaminase, or xylanase.<br>
The additional enzyme(s) may be obtained from a microorganism belonging to the ;enus Aspergillus, preferably Aspergillus aculeatus, Aspergillus awamori, Aspergillus iger, or Aspergillus oryzae, Trichoderma, Humicola, preferably Humicola insolens, or ■usarium, preferably Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, 'usarium    culmorum,     Fusarium    graminearum,    Fusarium    graminum,    Fusarium<br><br>
heterosporum, Fusarium negundi,	Fusarium oxysporum, Fusariwn reticulatum, Fusarium<br>
roseum,   Fusarium  sambucinum,	Fusarium  sarcochroum,   Fusarium  sporotrichioides,<br>
Fusarium sulphureum,  Fusarium	torulosum,  Fusarium  trichothecioides,   or Fusarium<br>
venenatum.<br>
Protein Hydrolysates<br>
The present invention also relates to protein hydrolysates produced by the methods of the present invention.<br>
The present invention also relates to protein hydrolysates enriched in free glutamic acid and/or peptide bound glutamic acid residues. Such protein hydrolysates can be produced by subjecting the proteinaceous material to a deamidation process either prior to or simultaneously with the enzymatic treatment(s).<br>
These protein hydrolysates have excellent flavor because glutamic acid (GIu), whether free or peptide bound, plays an important role in the flavor and palatability of protein hydrolysates. These protein hydrolysates also have improved functionality, in particular, improved solubility, improved emulsifying properties, increased degree of hydrolysis, and improved foaming properties.<br>
The deamidation process, Le., the conversion of amides (glutamine or asparagine) into charged acids (glutamic acid or aspartic acid) via the liberation of ammonia, may take place as a non-enzymatic or as an enzymatic deamidation process.<br>
In a preferred embodiment, the deamidation is carried out as an enzymatic deamidation process, e.g., by subjecting the substrate to a transglutaminase and/or peptidoglutaminase.<br>
The transglutaminase may be of any convenient source including mammals, see e.g., JP 1050382 and JP 5023182, including activated Factor XIH, see e.g., WO 93/15234; those derived from fish, see e.g., EP 555,649; and those obtained from microorganisms, see e.g., EP 379,606, WO 96/06931 and WO 96/22366. In a preferred embodiment, the transglutaminase is obtained from an Oomycete, including a strain of Phytophthora, preferably Phytophthora cactorum, or a strain of Pythium, preferably Pythium irregulare, Pythium sp., Pythium intermedium, Pythium ultimum, or Pythium periilum (or Pythium periplocum). In another preferred embodiment, the transglutaminase is of bacterial origin and is obtained from a strain of Bacillus,  preferably Bacillus subtilis,  a strain of<br><br>
Streptoverticillium, preferably Streptoverticillium mobaraensis, Streptoverticillium griseocameum, or Streptoverticillium cinnamoneum, and a strain of Streptomyces, preferably Streptomyces lydicus.<br>
The peptidoglutaminase may be a peptidoglutaminase I (peptidyl-glutaminase; EC 3.5.1.43), or a peptidoglutaminase II (protein-glutamine glutaminase; EC 3.5.1.44), or any mixture thereof. The peptidoglutaminase may be obtained from a strain of Aspergillus, preferably Aspergillus japonicus, a strain of Bacillus, preferably Bacillus circulans, a strain of Cryptococcus, preferably Cryptococcus albidus or a strain of Debaryomyces, preferably Debaryomyces kloecheri.<br>
The transglutaminase is added to the proteinaceous substrate in an effective amount conventionally employed in deamidation processes, preferably in the range of from about 0.01 to about 5% (w/w), and more preferably in the range of from about 0.1 to about 1% (w/w) of enzyme preparation relating to the amount of substrate.<br>
The peptidoglutaminase is added to the proteinaceous substrate in an effective amount conventionally employed in deamidation processes, preferably in the range of from about 0.01 to about 100,000 PGase Units per 100 g of substrate, and more preferably in the range of from about 0.1 to about 10,000 PGase Units per 100 g of substrate.<br>
The peptidoglutaminase activity may be determined according to the procedure of Cedrangoro et al. (1965, Enzymologia 29: 143). According to this procedure, 0.5 ml of an enzyme sample, adjusted to pH 6.5 with 1 N NaOH, is charged into a small vessel. Then 1 ml of a borate pH 10.8 buffer solution is added to the vessel. The discharged ammonia is absorbed by 5 N sulphuric acid, and by use of Nessler's reagent the mixture is allowed to form color which is measured at 420 nm. One PGase unit is the amount of enzyme capable of producing 1 micromole of ammonia per minute under these conditions.<br>
Alternatively, the peptidoglutaminase activity may be determined according to the procedure described in US 3,857,967 or Example 20 below.<br>
The present invention also relates to food products, e.g., baked products, and inimal feed additives comprising a protein hydrolysate obtained by the methods of the Dresent invention. Such food products exhibit enhanced organoleptic qualities, such as inprovement in flavor, palatability, mouth feel, aroma and crust color.<br>
In the present context, the term "baked products" includes any food prepared from lough, either of a soft or a crisp character.   Examples of baked products, whether of a<br><br>
white, light or dark type, which may be advantageously produced by the present invention, are bread, in particular white, whole-meal or rye bread, typically in the form of loaves or rolls; French baguette-type breads; pita breads; tacos; cakes; pancakes; biscuits; crisp breads; and the like.<br>
Such baked products are conventionally prepared from a dough which comprises flour and water, and which is typically leavened. The dough may be leavened in various ways, such as by adding sodium bicarbonate or the like, or by adding a leaven (fermenting dough), but ±e dough is preferably leavened by adding a suitable yeast culture such as a culmre of Saccharomyces cerevisiae (baker's yeast). Any of the commercially available Saccharomyces cerevisiae strainsmay be employed.<br>
Further, the dough used in the preparation of the baked products may be fresh or frozen. The preparation of frozen dough is described by K. Kulp and K. Lorenz in "Frozen and Refrigerated Doughs and Batters". A flavor improving composition of the present invention is typically included in the dough in an amount in the range of 0.01-5%, more preferably 0.1-3 %.<br>
The present invention also relates to the use of a hydrolysate produced by the methods of the invention as an additive to food products, such as baked foods, to enhance organoleptic qualities, such as flavor, palatability and aroma.<br>
The present invention is further described by the following examples which should not be construed as limiting the scope of the invention.<br>
Examples<br>
Example 1: Purification of FLAVOURZYME™ aminopeptidase 11<br>
Aminopeptidase was purified from a FLAVOURZYME™ broth (Novo Nordisk A/S, Bagsvaerd, Denmark). The FLAVOURZYME™ broth was produced by cultivation of Aspergillus oryzae strain 1568 (ATCC 20386) in a medium composed of carbon and nitrogen sources and trace metals. First, the broth (20 ml containing 720 mg of protein) was diluted with 180 ml of 20 mM sodium phosphate pH 7.0 buffer and filtered using Nalgene Filterware equipped with a 0.45 im filter (Nalgene, Rochester, NY). The filtered solution was loaded onto a 24 x 130 mm column containing 31 ml of Q-Sepharose, Big Beads (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM sodium<br><br>
phosphate pH 7.0 buffer. The protein was eluted using pH gradients from 7.0 (20 mM sodium phosphate buffer) to 5.0 (20 mM sodium acetate buffer), from 5.0 to 3.5 (20 mM sodium acetate buffer), and then from 3.5 to 3.0 (20 mM sodium acetate buffer). Fractions eluting between pH 3.5 and 3.0 were collected, pooled, and concentrated to 20 ml by ultrafiltration with a PM 10 membrane (Amicon, New Bedford, MA).<br>
The concentrated solution was diluted with 100 ml of 20 mM sodium phosphate pH 7.0 buffer and then loaded onto a 20 x 100 mm column containing Pharmacia MonoQ Beads (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The protein was eluted with a 0 to 0.4 M NaCl gradient in 20 nM sodium phosphate pH 7.0 buffer. The fractions between 0.330 and 0.343 M NaCl were collected, pooled, and concentrated using ultrafiltration against 20 mM sodium acetate )H 4.0 buffer.<br>
The purified preparation was found to contain three major bands judged by )DS-PAGE analysis. The sample consisted of components with molecular weights of pproximately 65, 50 and 33 kDa.<br>
ilxample 2: Amino acid sequencing of aminopeptidase II<br>
An aliquot of the purified aminopeptidase 11 preparation described in Example 1 was lectrophoresed and subsequently blot-transferred to a PVDF membrane (Novex, San )iego, CA) using 10 mM CAPS (3-[cyclohexylamino]-l-propanesulfonic acid) pH U in 0% methanol for 2 hours. The PVDF membrane was stained with 0.1% Coommassie ■lue R-250 in 40% methanol/1% acetic acid for 20 seconds and destained in 50% ethanol &gt; observe the protein bands. Three components at 65, 50, and 33 kDa were excised and ibjected to amino terminal sequencing on an Applied Biosystems Model 476A Protein squencer (Applied Biosystems, Inc., Foster City, CA) using a blot cartridge and liquid lase TFA delivery accordmg to the manufacmrer's instructions. All three components elded the same amino terminal sequence, RALVSPDEFPEDIQLEDLLEGSQQLEDFAY !EQ ID NO:2).<br>
A 300 fil sample of the protein was dried on a Savant Speed Vac AS 160 (Savant struments, Farmingdale, NY) and then reconstimted with 300 p.1 of 70% formic acid queous). A few crystals of cyanogen bromide were added and incubated at room mperature in the dark overnight.    The sample was redried in the Speed Vac and<br><br>
reconstimted in Tricine sample buffer (Novex, San Diego, CA). The cyanogen bromide cleavage fragments were separated using a 10-20% Tricine SDS-poIyacrylamide gel into bands of 6, 10. 15, 22, 27, 40 and 50 kDa and blot-transferred to a PVDF membrane. The 6, 10, 15, and 22 kDa bands were excised and subjected to amino terminal sequencing.<br>
The amino terminal sequences of the 15 and 22 kDa bands were identical to the amino terminal sequence above, while the sequences of the 6 and 10 kDa bands were both determined to contain the sequence TYSPSVEVTADVAWKNLGTSEADYPDVEGKVAL (SEQ ID N0:2).<br>
Example 3: Aspergillus oryzae strain 1568 RNA isolation<br>
Aspergillus oryzae strain 1568 was cultivated in a fermentation tank in a medium composed of 7.5 g of potato starch, 10 g of soy bean meal, 2 g of KH2PO4, 5 g of Na2HP04-2H20, and 0.1 g of ZnS04-7H20 per liter. A two liter sample was taken after five days of growth at 30°C, and the mycelia were collected, frozen in liquid N2, and<br>
stored at -SCC. Total RNA was prepared from the frozen, powdered mycelia of Aspergillus oryzae 1568 by extraction with guanidinium thiocyanate followed by ultracentrifugation through a 5.7 M cesium chloride cushion (Chirgwin et al, 1979, Biochemistry 18: 5294-5299). Poly(A)+ RNA was isolated by oligo(dT)-cellulose affinity chromatography according to Aviv and Leder (1972, Proceedings of the National Academy of Sciences USA 69: 1408-1412).<br>
Example 4: Construction of a cDNA library<br>
Double-stranded cDNA was synthesized from 5 g of Aspergillus oryzae 1568 po]y(A)+ RNA of Example 3 using the procedure described by Gubler and Hof&amp;nan (1983, Gene 25: 263-269) and Sambrook et al. (1989, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, New York), except that an o\igo{dT)-Notl anchor primer, instead of an oIigo(dT)12-18 primer, was used in the first strand reaction. After synthesis, the cDNA was treated with Mung bean nuclease (Life Technologies, Gaidiersburg, MD), blunt-ended with T4 DNA polymerase (Boehringer Mannheim, Indianapolis, IN), and ligated to non-palindromic BstXl adaptors (Invitrogen, San Diego, CA), using about 50-fold molar excess of the adaptors.   The adapted cDNA<br><br>
was digested with Notl, size-fractionated for 1.2-3.0 kb cDNAs by agarose gel electrophoresis, and ligated into pYES2.0 (Invitrogen, San Diego, CA) cleaved with BstXUNotL The ligation mixture was transformed into electrocompetent E. coli DHIOB cells (Life Technologies, Gaithersburg, MD) according to the manufacturer's instructions. The library consisting of 1 x 10* independent clones was stored as individual pools (25,000-30,000 colony forming units/pool) in 20% glycerol at -80°C, and as double stranded cDNA and ligation mixture at -20*'C.<br>
Example 5: Genomic DNA extraction<br>
Aspergillus oryzae 1568 ..was grown in 25 ml of 0.5% yeast extract-2% glucose (YEG) medium for 24 hours at 37*'C and 250 rpm.    Mycelia were then collected by filtration through Miracloth (Calbiochem, La JoUa, CA) and washed once with 25 ml of 10 mM Tris-1 mM EDTA (TE) buffer.    Excess buffer was drained from the mycelia preparation which was subsequently frozen in liquid nitrogen.     The  frozen mycelia preparation was ground to a fine powder in an electric coffee grinder, and the powder was added to a disposable plastic centrifuge tube containing 20 ml of TE buffer and 5 ml of 20% w/v sodium dodecylsulfate (SDS).   The mixture was gently inverted several times to ensure mixing, and extracted twice with an equal volume of phenol:chloroform:isoamyl alcohol (25:24:1 v/v/v).   Sodium acetate (3 M solution) was added to the extracted sample to a final concentration of 0.3 M followed by 2.5 volumes of ice cold ethanol to precipitate the DNA.  The tube was centrifuged at 15,000 x g for 30 minutes to pellet the DNA.  The DNA pellet was allowed to air-dry for 30 minutes before resuspension in 0.5 ml of TE buffer.    DNase-free ribonuclease A was added to the resuspended DNA pellet to a concentration of 100 g per ml and the mixture was then incubated at 37**C for 30 minutes. Proteinase K (200 /g/ml) was added and the tube was incubated an additional one hour at 37°C. Finally, the sample was extracted twice with phenol:chloroform:isoamyl alcohol and the DNA precipitated with ethanol.   The precipitated DNA was washed with 70% ethanol, dried under vacuum, resuspended in TE buffer, and stored at 4°C.<br>
Example 6: PCR amplification oi Aspergillus oryzae 1568 aminopeptidase II<br>
Based on the amino acid sequences of the Aspergillus oryzae 1568 aminopeptidase II partial peptides described in Example 2, the degenerate oligonucleotide primers shown<br><br>
below were synthesized with an Applied Biosystems Model 394 DNA/RNA Synthesizer,<br>
according to the manufacturer's instructions, to PCR amplify aminopeptidase n gene<br>
fragments from Aspergillus oryzae 1568 genomic DNA.<br>
Forward primer: 5'-CCIGAYGARTTYCCIGARGA-3' (SEQ ID N0:3)<br>
Reverse   primer:      5'-RTTYTTIACIACIGCIACRTCIGCIGTIACYTCIAC-3*   (SEQ   ID<br>
N0:4)<br>
(R= A or G, Y= C or T, N= G or A or C or T, H= A or C or T, 1= Inosine) Amplification reactions (50 /il) were prepared using approximately 1 fig of Aspergillus oryzae 1568 genomic DNA, prepared as described m Example 5, as the template. Each reaction contained the followmg components: 1 /ig of genomic DNA, 40 pmol of the forward primer, 40 pmol of the reverse primer, 200 fM each of dATP, dCTP, dGTP, and dTTP, 1 x Taq polymerase buffer (Perlcin-Ehner Corp., Branchburg, NJ), and 2.5 Units of Taq polymerase (Perkin-Elmer Corp., Branchburg, NJ). The reactions were incubated in a Perkin-Elmer Model 480 Thermal Cycler programmed as follows: Cycle 1 at 94*'C for 5 minutes, 50'"C for 2 minutes, and 72°C for 2 minutes; and Cycles 2-26 at 94'C for 2 minutes, 50'C for 1 minute, and 72*C for 2 minutes. The reaction products were isolated on a 1.5% agarose gel (Eastman Kodak, Rochester, NY) where a 309 bp product band was excised from the gel and purified using Qiaex II (Qiagen, Chatsworth, CA) according to the manufacturer's instructions. The purified PCR product was subsequently cloned into a pCRII vector (Invitrogen, San Diego, CA) and the DNA sequence was determined using lac forward and reverse primers (New England BioLabs, Beverly, MA).<br>
The aminopeptidase II gene segment (309 bp) consisting of 103 codons was amplified from Aspergillus oryzae 1568 with the aminopeptidase Il-specific PCR primers described above. DNA sequence analysis showed that the amplified gene segment encoded a portion of the corresponding Aspergillus oryzae 1568 aminopeptidase n gene. The aminopeptidase 11 gene segment was used to probe the Aspergillus oryzae 1568 cDNA library described in Example 5.<br>
Example 7: Identincation of Aspergillus oryzae 1568 aminopeptidase II clones<br>
The Aspergillus oryzae 1568 cDNA library was plated on Luria plus 50 |J.g/nii carbeniciilin agar plates.    Colony lifts (Maniatis et al,  1982, Molecular Cloning, A<br><br>
Laboratory Manual, Cold Spring Harbor Press, Cold Spring Harbor, New York) were performed  on  approximately   10,000  colonies  and  the  DNA  was  cross-linked  onto membranes (Hybond N-I-, Amersham, Arlington Heights, IL) using a UV Stratalinker (Stratagene, La JoUa, CA).   The membranes were soaked for three hours at 45'C in a hybridization solution containing 5 x SSPE, 0.3% SDS, 50% formamide, and 10 ug/ml of denatured and sheared herring sperm DNA.   The aminopeptidase n gene fragment isolated from the Aspergillus oryzae 1568 as described in Example 6 was radiolabeled using the Random Primed  DNA Labeling  Kit  (Boehringer  Mannheun,   Mannheim,   Germany), denatured by adding NaOH to a final concentration of 0. IM, and added to the hybridization solution at an activity of approximately 1 x 10"* cpm per ml of hybridization solution.  The mixture was incubated overnight at 45 °C in a shaking water bath.   Following incubation, the membranes were washed three times in 2 x SSC with 0.2% SDS at 55°C.    The membranes were then dried on blotting paper for 15 minutes, wrapped in SaranWrap™, and exposed to X-ray film for 48 hours at -70°C with intensifying screens (Kodak, Rochester, NY).<br>
Eleven colonies, produced strong hybridization signals with the probe. The eleven :olonies were inoculated into five mi of LB plus 50 ug/ml carbenicillin medium and grown overnight at 37°C. Miniprep DNA was prepared from each of these clones using the <izard dna purification kit madison wi clone and aminopeptidase ii encoding sequence as confirmed by sequencing. was fiiu length. the plasmid pejgls subcloned in e. coli dh5a cells to ejg18.></izard>
jxample 8: DNA sequence analysis of Aspergillus oryzae 1568 aminopeptidase II gene<br>
DNA sequencing of the aminopeptidase II gene contained in pEJGlS in E. coli )H5a EJG18 described in Example 7 was performed with an Applied Biosystems Model 73A Automated DNA Sequencer (Applied Biosystems, Inc., Foster City, CA) on both trands using the primer walking technique with dye-terminator chemistry (Giesecke et aL, 992, Journal of Virology Methods 38: 47-60). Oligonucleotide sequencing primers were esigned to complementary sequences in the aminopeptidase 11 gene and were synthesized<br><br>
on   an   Applied   Biosystems   Model   394   DNA/RNA   Synthesizer   according   to   the manufacturer's instructions.<br>
The nucleotide sequence of the gene encoding the Aspergillus oryzae 1568 aminopeptidase n and the deduced amino acid sequence thereof is shown in Figure 1 (SEQ ID N0S:1 and 2, respectively). Sequence analysis of the cloned insert revealed a large open reading frame of 1488 nucleotides (excluding the stop codon) encoding a protein of 496 amino acids sequence (SEQ ID N0:2). The G+C content of this open reading frame is 58%. Based on the rules of van Heijne (van Heijne, 1984, Journal of Molecular Biology 173: 243-251), the first 15 amino acids likely comprise a secretory signal peptide which directs the nascent polypeptide mto the endoplasmic reticulum (double underlined m Figure<br>
1).<br>
The amino acid sequences of the partial peptides derived from the purified aminopeptidase II as described in Example 2 are underlined in Figure 1 and were consistent with those found in the deduced amino acid sequence (SEQ ID N0:2) of the Aspergillus oryzae 1568 aminopeptidase II cDNA.<br>
Using the Clustal alignment program (Higgins, 1989, supra) to compare the deduced amino acid sequence of the Aspergillus oryzae 1568 aminopeptidase H to that of the Saccharmyces cerevisiae aminopeptidase II Y (SEQ ID N0:5), a 33.7% identity was observed.<br>
Example 9:   Construction of an Aspergillus oryzae 1568 aminopeptidase n expression vector for an Aspergillus host<br>
Two synthetic oligonucleotide primers shown below were designed to PCR amplify the Aspergillus oryzae A1568 aminopeptidase II gene coding sequence from plasmid pEJGlS (£". coli DH5a-EJG18) for subcloning and expression in an Aspergillus host. Forward primer:  5'-ATGATGAGGTCGCTTTTGTGGGC-3' (SEQ ID N0:6) Reverse primer:  5'-GGGATGCATCTATGCCTCGACTT-3' (SEQ ID N0:7) Bold letters represent coding sequence.<br>
In order to facilitate the subcloning of the gene fragment into an expression vector designated pMWR3 (Figure 2), a Nsil restriction enzyme site was introduced at the 3' end of the aminopeptidase II gene. The 5' end was left blunt with an addition of ATG for insertion into the Swal site.    The vector pMWR3 contained the TAKA promoter and<br><br>
terminator as regulatory sequences. Since the plasmid does not contain a selectable marker for fungal transformations, it was cotransformed with pToC90 (WO 91/17243) which contains amdS as the selectable marker.<br>
Fifty picomoles of each of the primers above were used in a PCR reaction (50 \yX) containing 70 ng of pEJGlS (an Aspergillus oryzae 1568 cDNA clone in pYES2), IX Pwo Buffer (Boehringer Mannheim, Indianapolis, IN), 8 fil of a 10 mM blend of dATP, dTTP, dGTP, and dCTP, and 2.5 units of Pwol (Boehringer Mannheim, Indianapolis, IN).   The amplification conditions were one cycle at 94°C for 2 minutes, SS'C for 30 seconds, and 72°C for 1 minute; 9 cycles at 94"="0 for 15 seconds, 5S°C for 30 seconds, and 72'C for 1 minutes; 15 cycles at 94°C for 15 seconds, 55°C for 30 seconds, and 12°C for 1 minute, with an extension of 20 seconds per cycle; and a final cycle of 94°C for 15 seconds, S5°C for 30 seconds, and 72*'C for 7 minutes.  The heat block then went to a 4*0 soak cycle. The amplified 1500 bp DNA fragment was purified by gel electrophoresis and Qiaex E. The aminopeptidase clone was digested with Nsii (using conditions specified by the manufacturer).   The fragment was phenol-chloroform extracted and ethanol precipitated. The cut fragment was cloned into pMWR3 that had been previously cut with Swal and Nsil resulting in the expression plasmid pEJG19 (Figure 3) in which transcription of the aminopeptidase II gene was under the control of the TAKA promoter.    The plasmid pEJG19 was transformed into E. coli DH5a cells (Life Technologies, Gaithersburg, MD). An E. coli transformant containing the pEJG19 plasmid was isolated and plasmid DNA was prepared according to procedures described by Sambrook et al., 1989, supra.<br>
Example 10:   Expression of the Aspergillus oryme 1568 aminopeptidase II gene in<br>
Aspergillus oryzae<br>
Plasmid pEJG19 was introduced into an alkaline protease-deficient Aspergillus oryzae host JaL142-6 using the following protoplast transformation methods. The ransformation was conducted with protoplasts at a concentration of ca. 2x10' protoplasts ler ml. One hundred nl of protoplasts were placed on ice with ca. 5 ptg of pEJG19 and 5 4g of pTOC90; 250 /xl of 60% PEG 4000, 10 mM Tris-HCl, pH 7.5, 10 mM CaClj was idded, and the protoplasts were incubated at 37'C for 30 minutes. Three mis of STC (1.2 A Sorbitol, 10 mM Tris-HCl, pH 7.5, and 10 mM CaClj ) was added.   The solution was<br><br>
mixed gently and poured onto COVE transformation plates (per liter: 0.52 g of KCl, 0.52 g of MgSO4-7H20, 1.52 g of KH2PO4, 1 ml of trace metals described below, 342.3 g of sucrose, 25 g of Noble agar, 10 ml of 1 M acetamide, 10 mi of 3 M CsCl). The trace metals solution (lOOOX) was comprised of 22 g of ZnS04-7H20. 11 g of H3BO3, 5 g of MnCl2-4H20, 5 g of FeS04-7HjO, 1.6 g of CoCl-SHjO, 1.6 g of (NH4)sMo,04, and 50 g of NaiEDTA per liter. Plates were incubated 7 days at 37'*C. Transformants were transferred to plates of the same medium and incubated 2 days at 37°C. Totally, 140 transformants were recovered by their ability to grow on COVE medium using acetamide as sole nitrogen source.<br>
The transformants were grown for 4 days at 34°C, 200 rpm in 24 well plates containing 1 ml per well of 25% MY50 medium diluted with 75% MY50 salts. MY50 was composed per liter of 50 g of maltodextrin, 2.0 g of MgS04-7H20, 10 g of KH2PO4, 2 g of citric acid, 10 g of yeast extract, 2.0 g of urea, 2 g of K2SO4 and 0.5 ml of trace elements solution adjusted to pH 6.0. The trace metals solution was compsoed per liter of 14.3 g of ZnS04-7H20, 2.5g of CUSO4-5H2O, 0.5g of NiCl2-6H20, 13.8 g of FeS04-7H20, 8.5 g of MnS04-H20, 3 g of citric acid. The MY50 salts was composed per liter of 2.0 g of MgS04-7H20, 10 g of KH2PO4, 2 g of citric acid, and 2 g of K2SO4, pH 6.0.<br>
Each of the 140 wells were assayed for aminopeptidase 11 activity using Leu-pNA (hydrochloride salt) as substrate. In a 96 well microtiter plate, 4 1 of supernatant was added to 100 fjX of 1 mg/ml of Leu-pNA in 50 mM sodium phosphate pH 7.5 buffer. The absorbance at 405 nm was monitored.<br>
Four transformants, 20, 88, 90, and 137, with the highest level of aminopeptidase II activity were then grown in 125 ml shake flasks for 4 days at 34°C containing 25 ml of MY50 medium.<br>
Samples were assayed on day 2, 3, and 4 for aminopeptidase n activity by mixing 100 fil of 10-foid diluted supernatant with 100 fA of 2 mg/ml Leu-pNA in 50 mM sodium phosphate pH 7.5 buffer. Transformants 20, 90, and 137 were the highest producers, for For purification of the aminopeptidase II, transformants 20 and/or 90 were grown in shake flasks as above or in a fermentation medium composed of suitable carbon and nitrogen sources.<br><br>
Example 11:   Purification of recombinant Aspergillus oryzae 1568 aminopeptidase II produced in Aspergillus<br>
The combined supematants from the shake flask broths described in Example 10 were combined (approximately 100 mg of protem in approximately 100 mi) and was diluted to 3.7 mS and adjusted to pH 7.0. The diluted sample was then loaded onto Q-Sepharose, Big Beads, pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The aminopeptidase II was eluted with a 0-0.4 M NaCl gradient in 20 mM sodium phosphate pH 7.0 buffer followed by a wash with 0.4 M NaCl. Fractions were assayed for aminopeptidase II activity by mixing 100 /zl of each fraction with 100 /xl of 2 mg of Leu-pNA per ml of 50 mM sodium phosphate pH 7.5 buffer. The assay results indicated that the aminoepeptidase II eluted at the end of the gradient and during the 0.4 M NaCl wash. Analysis by SDS-PAGE revealed that the enzyme was homogeneous.<br>
Example 12:  Construction of an Aspergillus oryzae 1568 aminopeptidase II expression<br>
vector for a Fusarium host<br>
Two synthetic oligonucleotide pruners shown below were designed to PCR amplify<br>
the Aspergillus oryzae A1568 aminopeptidase n gene coding sequence from plasmid<br>
pEJGlS (£". coli DH5a-EJG18) for subcloning and expression in a Fusarium host.<br>
Forward  primer:      5'-ATTTAAATcaccATGAGGTCGCTTrTGTGGGC-3'   (SEQ   ID<br>
N0:8)<br>
Reverse primer:     5'-GGGTTAATTAACTATGCCTCGACTTGAGAATG-3'  (SEQ ID<br>
N0:9)<br>
Bold letters represent coding sequence.  Small case represents a Kozak consensus sequence to enhance expression (Kozak, 1981, Nucleic Acids Research 12. 857-872).<br>
In order to facilitate the subcloning of the gene fragment into an expression vector designated pDMi81 (Figure 4), Swal and Pad restriction enzyme sites were introduced at the 5' and 3' end of the aminopeptidase H gene, respectively. The vector pDMlSl contained the Fusarium oxysporum trypsin-like protease (SP387) promoter and terminator (WO 96/00787) as regulatory sequences. The plasmid also contained the bar gene as a selectable marker for fungal transformations (de Block et al., 1987, EMBO Journal 6:2513-2518).<br><br>
Fifty picomoles of each of ±e primers above were used in a PCR reaction containing 70 ng of pEJGlS. IX Pwo Buffer, 5 l of 10 mM blend of dATP. dTTP, dGTP. and dCTP, and 2.5 units of Pwol. The amplification conditions were one cycle at 94'C for 2 minutes, 55°C for 30 seconds, and 72°C for 1 minute; 9 cycles each at 94X for 15 seconds, 55°C for 30 seconds, and 72°C for 1 minute; 15 cycles each at 94'C for 15 seconds, 55 °C for 30 seconds, and 12°C for 1 minute, with an extension of 20 seconds per cycle; and a final cycle at 94'C for 15 seconds, 55°C for 30 seconds, and 72'C for 7 minutes.  The heat block was then held at a 4°C soak cycle.   The amplified 1500 bp DNA fragment was purified by gel electrophoresis and Qiaex n, and then subcloned mto pCRU TOPO TA cloning vector (Stratagene, San Diego, CA). The pCRII aminopeptidase clone was cut with restriction endonucleases Swal and Pad (using conditions specified by the manufacturer).   The fragment was purified by gel electrophoresis and Qiaex II.   The cut fragment was cloned into pDMlSl that had been previously cut with 5wal and Pad resulting m the expression plasmid pEJG28 (Figure 5) in which transcription of the aminopeptidase II gene was under the control of the Fusarium oxysporum trypsin-Iike protease promoter.    The plasmid pEJG28 was transformed into E. coli ABLE K cells (Stratagene, San Diego, CA).  The E. coli transformant containing the pEJG28 plasmid was isolated and plasmid DNA was prepared according to procedures described by Sambrook et al, 199, supra.<br>
Example 13: Transformation oi Fusarium CCl-3 and analysis of transformants<br>
Fusarium strain CC1-3, a highly branched morphological mutant of Fusarium strain A3/5 (ATCC 20334) (Wiebe etai, 1992. Mycological Research 96: 555-562; Wiebe etal., 1991, Mycological Research 95: 1284-1288; Wiebe et aL, 1991, Mycological Research 96: 555-562), was grown in a liquid medium containing Vogers salts (Vogel, 1964, Am. Nature 98: 435-446), 25 mM NaN03, and 1.5% glucose for 4 days at 28'C and 150 rpm. Conidia were purified by filtration through 4 layers of cheesecloth and finally through one layer of Miracloth. Conidial suspensions were concentrated by centrifligation. Fifty ml of YPG medium comprised of 1% yeast extract, 2% bactopeptone, and 2% glucose were inoculated with approximately 10 conidia, and incubated for 14 hours at 24*0 and 150 rpm. Resulting hyphae were trapped on a sterile 0.4 mm filter and washed successively ;vith sterile distilled water and 1.0 M MgS04.   The hyphae were resuspended in 10 ml of<br><br>
NOVOZYM 234™ solution (2-10 mg/ml in 1.0 M MgS04) and digested for 15-30 minutes at 34°C with agitation at 80 rpm. NOVOZYM 234"* was obtained from Novo Nordisk A/S, Bagsvaerd, Denmark. Undigested hyphal material was removed from the resulting protoplast suspension by successive filtration through 4 layers of cheesecloth and through Miracloth. Twenty ml of 1 M sorbitol were combined with the protoplast solution. After mixing, the protoplasts were pelleted by centrifugation and washed successively by resuspension and centrifugation in 20 ml of 1 M sorbitol and in 20 ml of STC (0.8 M sorbitol, 0.05 M Tris pH 8.0, 0.05 M CaCy. The washed protoplasts were resuspended in 4 parts STC and 1 part SPTC (0.8 M sorbitol, 40% PEG 4000, 0.05 M Tris pH 8.0, 0.05 M CaCy at a concentration of 5 x lOVml.<br>
One hundred ml of protoplast suspension were added to 10 fig of pEJG28 in polypropylene tubes (17 x 100 mm), mixed and mcubated on ice for 30 minutes. One ml of SPTC was mixed gently into the protoplast suspension and incubation was continued at room temperature for 20 minutes. 12.5 ml of molten solution (cooled to 40°C) consisting of IX Vogel's salts, 25 mM NaNOj, 0.8 M sucrose and 1% low melting agarose (Sigma Chemical Company, St. Louis, MO) were mixed with the protoplasts and then plated onto an empty 100 mm Petri plate. Incubation was continued at room temperature for 10 to 14 days. After incubation at room temperature for 24 hours, 12.5 ml of the identical medium plus 10 mg of BASTA™ (Hoechst Schering, Rodovre, Denmark) per ml were overlayed onto the Petri plate. BASTA™ was extracted twice with phenol:chloroform:isoamyI alcohol (25:24:1), and once with chloroform:isoamyl alcohol (24:1) before use.<br>
After two weeks, 2 transformants designated #1 and #2 were apparent. A mycelial fragment from the edge of each transformant was transferred to individual wells of a 24 well plate containing Vogel's/ BASTA™ medium. The medium contained 25 g of sucrose, 25 g of Noble agar, 20 ml of 50X Vogel's salts (Vogel, 1964, supra), 25 mM NaNOj, and 10 g of BASTA™ per liter. The plate was sealed in a plastic bag to maintain moisture and incubated approximately one week at room temperature.<br>
Example  14:     Expression of Aspergillus oryzae  1568 aminopeptidase II gene  in<br>
Fusarium<br><br>
A mycelial fragment from each of the 2 Fusarium CCl-3 transformants described in Example 13 was inoculated into 20 ml of M400Da medium composed of 50 g of maltodextrin, 2.0 g of MgS04-7H20, 2.0 g of KH2PO4, 4.0 g of citric acid, 8.0 g of yeast extract, 2.0 g of urea, and 0.5 ml of trace metals solution per liter and incubated for 7 days at 30°C and 150 rpm. The medium was adjusted to pH 6.0 with 5 N NaOH. The trace metals solution contained 14.3 g of ZnS04-7H20, 2.5 g of CUSO4-5H2O, 0.5 g of NiClj-6H2O, 13.8 g of FeS04-7H20, 8.5 g of MnS04-H20, and 3.0 g of citric acid per liter. The untransformed host was also run as a control. One ml of culture supernatant was harvested at 7 days and stored and assayed. Aminopeptidase II activity was determined by mixing 10 1 of supernatant with 200 l of a substrate stock solution containing 2 mg of Leu-para-nitroanilide per ml of 50 mM sodium phosphate pH 7.5 buffer and monitoring the change in absorbance at 405 nm over 4 minutes. Both transformants exhibited activity towards Leu-pNA greater than the untransformed control.<br>
The primary Fusarium transformant #2 described in Example 13 was cultivated in 125 ml shake flasks for 5 days at 30°C in 25 ml of M400Da medium. The whole culture broth was filtered using a double layer of Miracloth. The filtrate was recovered and then frozen at-20°C.<br>
Example 15:   Purification of recombinant Aspergillus oryzae 1568 aminopeptidase II produced by Fusarium<br>
A 20 ml volume of a 5 day Fusarium broth described in Example 14 was filtered through a 0.45 micron syringe filter. The sample was then diluted 8-fold m 20 mM sodium phosphate buffer pH 7.0 buffer. The conductivity and pH of the sample was 3.1 mS and 7.10, respectively. The sample was loaded onto a XK-26 (Pharmacia Biotech AB, Uppsala, Sweden) column containing 60 ml of Q-Sepharose, Big Beads, which had been pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The column was washed until a baseline was reached and dien the sample was eluted with a linear gradient from 0-0.5 M NaCl in 20 mM sodium phosphate buffer pH 7.0 over 8.3 column volumes and at a flow rate of 5 ml/min. Fractions were assayed using Leu-pNA as substrate by mixing 10 1 of each fraction with 90 il of 50 mM sodium phosphate pH 7.5 buffer and 100 1 of a substrate stock solution containing 2 mg of Leu-pNA per ml of 50 mM sodium phosphate<br><br>
pH 7.5 buffer and monitoring the change in absorbance at 405 nm over 4 minutes. All fractions active on Leu-pNA were then pooled, diluted, and concentrated using an Amicon Ultrafiltration unit utilizing a PM 10 membrane.<br>
The concentrated sample was then loaded onto a Mono Q 16/10 (Pharmacia Biotech AB, Uppsala, Sweden) column which had been pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The column was then washed with 0.15 M NaCI. A gradient was performed from 0.15-0.5 M NaCl over 10 column volumes at a flow rate of 2 ml/min. Active fractions were then equilibrated in 1.7 M (NH4)2S04 in 50 mM sodium phosphate buffer pH 7.0.<br>
The sample was then loaded onto a Phenyl Superose 5/5 column (Pharmacia Biotech AB, Uppsala, Sweden) which had been pre-equilibrated with 1.7 M (NH4)2S04 in 50 mM sodium phosphate pH 7.0 buffer. The column was washed with 1.7 M (NH4)2S04 in 50 mM sodium phosphate pH 7.0 buffer until baseline was achieved. The enzyme was eluted with a gradient from 1.7 M to 0 M (NH4)2S04 over 30 column volumes at a flow rate of 0.5 ml/min. The flow through had activity on Leu-pNA, as did fractions that were eluted. The enzyme appeared as a series of differentially glycosylated forms based on SDS-PAGE analysis. When the various forms of the enzyme were treated with Endoglycosidase F/N glycosidase F (Boehringer Mannheim, Indianapolis, IN), according to the manufacturer's suggested protocol, a single band with a molecular weight of "58 kDa appeared in all the samples analyzed. The differentially glycosylated forms were then pooled, desalted using 50 mM sodium phosphate buffer pH 7.5, and submitted to biochemical characterization.<br>
Example 16:  Characterization of recombinant Aspergillus oryzae 1568 aminopeptidase II<br>
The purified aminopeptidase H described in Example 11 was used in the following characterization.<br>
Kinetic parameters of the aminopeptidase II were determined for several p-nitroanilides (pNA) including Leu-pNA, Gly-pNA, Ala-pNA, and Pro-pNA (Sigma Chemical Co., St. Louis, MO, or Bachem, Torrance, CA). Stock solutions of 100 mg of each p-nitroanilide per ml of dimethylsulfoxide were diluted with 50 mM potassium phosphate pH 7.0 buffer to concentrations ranging from 0.0064 to 9.56 mM. It should be noted that the solubility of the substrates is not always sufficient to have concentrations<br><br>
comparable to K, which may result in errors which could be higher than expected normally. The reaction of the aminopeptidase H with the p-nitroanilide was initiated when a 100 I aliquot of the enzyme solution in 50 mM potassium phosphate pH 7.0 was added to 100 IJ.1 of a substrate solution in a microtiter plate well and monitored at 405 nm and 25"C using a THERMOmax Microplate Reader (Molecular Devices Corp., Sunnyvale, CA).   Analysis of initial rates of hydrolysis of the p-nitroanilides produced the results<br><br>
The results showed that the aminopeptidase II possesses a substrate specificity where the specificity toward Ala is much worse than toward Gly.<br>
Inhibition of the aminopeptidase II with 1,10-phenanthroline was evaluated using Leu-pNA as substrate at pH 7.5 m 50 mM Tris pH 7.5 buffer where hydrolysis was monitored at 405 nm. A 200 mM solution of 1,10-phenanthroline in methanol was prepared. The inhibition reaction was conducted by mixing 100 fii of 2 mg of Leu-pNA per ml of 50 mM sodium phosphate pH 7.5 solution and 10 1 of the 1,10-phenanthroUne solution with 100 1 of aminopeptidase II diluted 5-fold in 50 mM sodium phosphate pH 7.5 buffer. A control was run where 10 1 of 20 mM Tris pH 7.6 buffer was used in place of the 1,10-phenantholine solution.<br>
The results indicated that 1,10-phenanthroline inhibited the aminopeptidase n suggesting that the aminopeptidase II is a metalloprotease. The rate of hydrolysis of Leu-pNA decreased from 285 mOD/minute to 21 mOD/minute in the presence of 1,10-phenanthroline.<br>
The purified aminopeptidase II described in Example 15 was used in the following characterizations.<br>
The pH optimum was determined using Ala-pNA as substrate in the universal buffer composed of 0.125 M citric acid, 0.125 M mono basic sodium phosphate, and 0.125 M<br><br>
boric acid pH was adjusted to 4.5-11 with 10 N NaOH, in 0.5 pH increments. The Ala-pNA substrate was prepared by dissolving 100 mg of Ala-pNA in 1 ml of DMSO and adding 20 lA of the Ala-pNA/DMSO solution to 980 pd of the universal buffer at the various pH values. The assay was initiated by adding a 15 1 aliquot of the aminopeptidase II in 50 mM sodium phosphate pH 7.5 buffer to 200 (xl of 2 mg/ml Ala-pNA at the various pH values at ambient temperature. The change in absorbance at 405 nm was monitored for 5 minutes. Autohydrolysis of the substrate as a control was determined by adding 15 1 of 50 mM sodium phospliate pH 7.5 buffer to 200 (A of 2 mg/mJ Ala-pNA at the various pH values.<br>
The results shown in Table 2 below demonstrated that the aminopeptidase U possessed activity with Ala-pNA as substrate over the measured pH range 4.91 to 10.91 with optima] activity at pH "9.5-10, No autohydrolysis of the substrate was observed at pH values of 11 or less.<br><br><br>
20 fi\ of purified aminopeptidase II was added and the sample was then incubated for an additional 20 minutes. The samples were then placed on ice. Once the iacubations were completed for all the temperatures, the samples were then assayed for activity using Leu-pNA as substrate.<br>
The assay was performed by mixing 100 /xl of the mcubation mixtures for the various temperatures with 100 1 of 2 mg/ml Leu-pNA in 50 mM sodium phosphate pH 7.5 buffer at ambient temperature. The absorbance at 405 nm was monitored for 5 minutes.<br>
The results shown in Table 3 demonstrated that the aminopeptidase II retained 90% of its activity after a 20 minute incubation at 60'C, pH 7.5.<br><br>
The kinetic parameters for various aminopeptidase II substrates was determined using the following protocol. Purified aminopeptidase II with an A280 of 0.581 was used. Each substrate was dissolved in DMSO to a concentration of 100 mg/ml and then diluted 50 fold in 50 mM sodium phosphate pH 7.5 buffer to 2 mg/ml. The substrates included Leu-pNA, Glu-pNA (Bachem, Torrance, CA), and Ala-pNA. In a 96 well microtiter plate, 10 fi\ of purified aminopeptidase H was incubated with each substrate as follows except 50 /l of purified aminopeptidase II was incubated with Glu-pNA, and the absorbance at 405 nm was measured for 4 minutes:<br>
1.	200 jul of 2 mg/ml substrate + 0 1 of 50 mM sodium phosphate buffer pH 7.5<br>
2.	100 1 of 2 mg/ml substrate + 100 1 of 50 mM sodium phosphate buffer pH 7.5<br>
3.	50 jul of 2 mg/ml substrate + 150 tl of 50 mM sodium phosphate buffer pH 7.5<br>
4.	25 jLtl of 2 mg/ml substrate + 175 fi\ of 50 mM sodium phosphate buffer pH 7.5<br><br>
A Lineweaver-Burke plot was constructed to determine the K and the kat for each substrate, using an average molecular weight of 97 kDa for the differentially glycosylated forms.<br>
For Leu-pNA, the K, and k were determined to be 5.78 mM and 230.9 min"', respectively.<br>
For Glu-pNA, the K and k were determined to be 1.17 mM and 8.217 min' respectively.<br>
For Ala-pNA, the K and k were determined to be 1.49 mM and 34.638 min', respectively.<br>
Example  17:     Preparation  of protein hydrolysates  with Aspergillus oryzae  1568 aminopeptidase II<br>
The purified aminopeptidase U described in Example 11 was tested in degree of hydrolysis assays using soy, wheat gluten, and casein as substrates according to the following procedure.<br>
The degree of hydrolysis (DH) assays were performed at 50°C for 18 hours as a mini-hydrolysis on a 10 ml scale using soy bean meal tablets, wheat gluten meal tablets, and sodium-caseinate at a 2% concentration adjusted to pH 7, if necessary, with no pH adjustment during hydrolysis.   The hydrolysates were inactivated at 85 "C for 3 minutes in a waterbath.   The enzymes used were FLAVOURZYME'" and aminopeptidase H.   The enzymes were dosed as follows. For soy, 2 LAPUs and 5 LAPUs of aminopeptidase II (recombinant) were added compared to 3 LAPUs of FLAVOURZYME"".   For gluten, 2 LAPUs and 5 LAPUs of aminopeptidase U (recombinant) were added compared to 3 LAPUs of FLAVOURZYME™.    For casein,  1  and 2 LAPUs of ammopeptidase D (recombinant) were added compared to 3 LAPUs of FLAVOURZYME.    One LAPU (Leucine Amino Peptidase Unit) is the amount of enzyme which decomposes 1 micromole of L-leucine-p-nitroanilide per minute under the following conditions: 26 mM L-leucine-p-nitroanilide in O.I M Tris pH 8.0 buffer at 40°C for 10 minutes.   Upon hydrolysis, p-nitroanilide is liberated turning the solution yellow which is monitored 405 nm.<br>
The degree of hydrolysis (DH), as defined as described by Adler-Nissen (1986, Enzymic Hydrolysis  of Food Proteins,   Elsevier  Applied  Science  Publishers),   was<br><br>
determined  by  reaction of the  supernatant  with OP A  (ortho-phtaldialdehyde,  Sigma Chemical Co., St. Louis, MO) according to the following procedure  The hydrolysate was diluted 100-fold into distilled water.    Then 120 /xl was transferred to 900 /zl of OPA reagent.   For the OPA reagent, 160 mg of OPA was dissolved in 4 ml of ethanol and transferred to a 200 ml volumetric flask containing a solution of 7.62 g of disodium tetraborate decahydrate, 200 mg of sodium dodecylsulphate, and 176 mg of dithiothreitol and the flask was filled to 200 ml with water.  The solution was then shaken well and after 2 minutes exactly, the absorbance at 340 nm was measured and compared to the absorbance of a 0.95 mM L-serine (distilled water) solution after subtraction of the blank value (water reacted with OPA reagent).  To determine the true DH, the serine equivalents measured in the hydrolysates were corrected with the factors suggested by Adler-Nissen for the trinitrobenzenesulfonic acid method (Adler-Nissen, 1979, Agricultural and Food Chemistry 17: 1256) which gave the same response as the described OPA method.   The DH was calculated on the basis of the total amount of protein in the hydrolysis mixmre (not on the basis of soluble protein).<br>
A volume of 25 |il of suitably diluted supernatant was mixed with 200 \il of OPA reagent in a microtiter plate well and allowed to react for exactly 2 minutes at 25'C. The absorbance at 340 nm was measured in a microtiter plate reader and compared to the absorbance of a 95 mM L-serine standard solution after subtraction of the blank value (water reacted with OPA-reagent). To determine the true DH, the serine equivalents measured m the supematants were corrected with the factors suggested by Adler-Nissen for the trinitrobenzenesulfonic acid method (Adler-Nissen, 1979, Agricultural and Food Chemistry 17: 1256) which gave the same response as the described OPA method. The degree of hydrolysis was calculated on basis of the total amount of protein in the hydrolysis mixture (not on basis of soluble protein).<br>
For soy, the addition of 2 LAPUs and 5 LAPUs of aminopeptidase U to 3 LAPUs of FLAVOURZYME'" increased absolute DH at least 8% and 10%. respectively, above the samples with 3 LAPUs of FLAVOURZYME" alone.<br>
For gluten, the addition of 2 LAPUs and 5 LAPUs of aminopeptidase II to 3 LAPUs of FLAVOURZYME" increased absolute DH 6% and 9%, respectively.<br><br>
For gelatin, the addition of 2 LAPUs and 5 LAPUs of dipeptide aminopeptidase 11 to 3 LAPUs of FLAVOURZYME™ mcreased absolute DH 4.9% and 5.3% respectively.<br>
For casein, the addition of 1 and 2 LAPUs of aminopeptidase n to 3 LAPUs of FLAVOURZYME' increased absolute DH 7% and 9%, respectively, over the addition of 3 LAPUs of FLAVOURZYME™ alone.<br>
Example 18: Hydrolysis of soy protein with Aspergillus oryzae aminopeptidase H<br>
Soy protein in the form of defatted soy bean meal was hydrolysed on a 10 ml-scale (mini-hydrolysis) with a start pH of 7.0 and a protein concentration of 2%.  The hydrolysis time and temperature were 18 hours and 50°C, respectively.   Enzymes were inactivated at 85 °C for 5 minutes and the hydrolysates were centrifiiged.  The supematants were analysed for DH using the OPA-method. The DH, as defined as described by Adler-Nissen (1986, Enzymic  Hydrolysis   of Food  Proteins,   Elsevier  Applied   Science   Publishers),   was determined  by  reaction of the  supernatant  with  OPA  (ortho-phtaldialdehyde,   Sigma Chemical Co., St. Louis, MO) according to Church et al., 1983, Journal of Dairy Science 66: 1219-1227.   For the OPA reagent, 160 mg of OPA was dissolved in 4 ml of ethanol and transferred to a 200 ml volumetric flask containing a solution of 7.62 g of disodium tetraborate decahydrate, 200 mg of sodium dodecylsulphate, and 176 mg of dithiothreitol and the flask was filled to 200 ml with water.   Selected samples were analysed for the content of free amino acids using the PicoTag HPLC method (Waters Associates, Milford, MA) according to the manufacturer's instructions.<br>
The dosages of enzymes to each hydrolysis flask containing 200 mg of soy protein are shown in Table 4 below. The dosage of FLAVOURZYME™ lOOOL was in percent of the soy protein substrate such that 1.5% FLAVOURZYME™ lOOOL equals 3 mg of FLAVOURZYME™ lOOOL added to 200 mg of soy protein. In addition to FLAVOURZYME™ lOOOL, 1.5% ALCALASE® 2.4L was added to all hydrolysis flasks which equals 3 mg per 200 mg of soy protein. The aminopeptidase 11 was produced recombinantly in Aspergillus oryzae as described in Example 10 and purified accordingly. The aminopeptidase II solution had an Ajm of -   estimated protein content of 5 mg/ml from amino acid determination.<br><br>
The results of the DH analysis are presented in Table 4.   DH was calculated from the total protein concentration of 2% - not the soluble protein content.<br>
Table 4. The DH results for the hydrolysates<br><br>
* The concentration of aminopeptidase II used for this calculation is 5 mg/ml<br>
Table 5 shows the relative % increase in the individual free amino acids upon addition of a maximal aminopeptidase II dosage (0.5 g per kg soy protein) to a background dosage of FLAVOURZYME™ and 1.5% ALCALASE® 2.4L. The increase was measured relative to the release of free amino acids resulting from the background enzyme dosages.<br>
Table 5. Relative increase in free amino acids due to addition of aminopeptidase II<br><br><br><br><br>
The results showed that when aminopeptidase II was added to a low dosage of FLAVOURZYME™ (1.5%) plus 1.5% ALCALASE® 2.4L, Gly showed the highest relative % increase followed by Ser, Asp, Asn, Pro, Cys, and Ala. When aminopeptidase a was added to a high FLAVOURZYME™ dosage (6%) plus 1.5% ALCALASE® 2.4L, Pro showed the highest relative % increase followed by Asp, GIu, Cys, Gly, and Gin.<br>
Sxample 19: Increased protein solubility and release of glutamate by deamidation<br>
Wheat gluten (WG) was obtained from Cargill (JOB 5141) and deamidated wheat jluten (DWG) was obtained from StaPro Consultancy B.V., Lemdijk 32, 9422 TH Smilde, L. Suspensions of 8% protein were made by mixing 11 g of gluten with 89 g of water. The pH was adjusted to 6.5 with NaOH. Glutamate/aspartate specific protease (SP446), jbtainable as described in WO 91/13554, or lysine/arginine specific protease (SP387) jbtainable as described in WO 89/06270, was added to the suspensions.   The dosage was<br><br>
0.01 AU/g protein for SP446 and 0.006 AU/g protein for SP387. FLAVOURZYME™ (an un-specifically acting protease preparation available from Novo Nordisk A/S, Bagsvserd, Denmark, containing endo- and exo-peptidase activities, and obtained by fermentation of Aspergillus oryzae) was added to some of the hydrolysates at a dosage of 20 LAPU/g protein.<br>
The hydrolyses were carried out at 50°C without further pH adjustment for 18 hours. The enzymes were inactivated by heating at SSC for 15 minutes. The pH was adjusted to 5 and the hydrolysates were centrifuged. The content of protein and free glutamate in the supernatant was determined.<br>
The protein content was determined by Kjeldahl analysis, using a Kjeldahl factor of 6.25.<br>
The content of free glutamate was determined by use of a glutamate determination kit according to the manufacturer's instructions (Boehringer-Mannheim, Indianapolis, IN). The method was adapted for use in microtiter plates.<br>
When comparing wheat gluten (WG) to deamidated wheat gluten (DWG), the results shown in Table 6 demonstrated that deamidation increased the susceptibility of the gluten to specific proteases, such that more protein became soluble. By addition of FLAV0UR2YME™ with a specific protease, the release of glutamate was doubled due to deamidation.<br><br>
Example 20: Enzymatic deamidation and release of glutamate<br>
Peptidoglutaminase 11 was produced by growing Bacillus circulans strain ATCC 21590 in shake flasks (400 ml) containing 200 ml of a medium composed of 1% polypeptone, 0.5% lactose. 0.025% MgS04-7H20, 0.005% FeS04-7H20, 0.025% KH2PO4, and 17% Na2HP04-12H20 (pH adjusted to 7.2), at 30°C for 20 hours with mixing at 270<br><br>
rpm.  The cells were harvested by centriftigation at 4000 rpm in 1 litre flasks.   The cells were then frozen.<br>
The purification of peptidoglutaminase n from Bacillus circulans was performed at room temperature. The frozen Bacillus circulans cells were thawed and suspended in Lysis buffer (50 mM Tris/HCl; 25% (w/v) sucrose; 1 mM EDTA, pH 8.0) until a homogeneous suspension was obtained - 100 g wet cells per liter of Lysis buffer. Lysozyme (10 mg/ml) and DNAse I (Sigma DN-25, 10 mg/ml) were dissolved in Lysis buffer. Then 100 ml of lysozyme solution, 10 ml of 1.0 M MgClj, and 1 ml of DNAse I solution were added per litre of cell suspension.  The enzymes were allowed to act for 1 hour.<br>
The suspension was filtered through a Seitz depth filter plate and the filtrate was transferred to a 10 mM KHjPO/NaOH, pH 8.0 (Buffer A) on a Sephadex G25 column (Pharmacia Biotech AB, Uppsala, Sweden). The enzyme solution was applied to a SOURCE Q column (Pharmacia Biotech AB, Uppsala, Sweden) equilibrated in Buffer A and eluted with a linear NaCl gradient (0 -&gt; 500 mM) in Buffer A. Fractions from the column were analysed for peptidoglutaminase II activity as described below and fractions with activity were pooled. The absorbance of the pooled fractions at 280 nm was 1.78, thus the protein content was estimated to 1.8 mg/ml.<br>
The purity of the protein in the peptidoglutaminase II pool was approximately 25% as judged from a SDS-PAGE gel. Thus, the preparation contained approximately 0.5 mg/ml of pure peptidoglutaminase II.<br>
The peptidoglutaminase activity was determined by measuring the ammonia formed during hydrolysis of y-carboxyamide of N-tert-Butoxycarbonyl-Gln-Pro (N-t-BOC-Gln-Pro; SIGMA No. B-4403) using the Boehringer-Mannheim kit for ammonia determination (Cat. No. 1112732). In this kit, ammonia is measured by determination of the consumption of NADH by glutamate dehydrogenase, and blanks without the addition of N-t-BOC-Gln-Pro were also applied in order to subtract the effect of other NADH consuming enzymes.<br>
A total of 200 mg of wheat gluten protein was added to 9 mi of boiling water and after cooling, the pH was adjusted to 7.0. Then 250 \i\ of the peptidoglutaminase II preparation (PEP) described above was added. The glutamate/aspartate specific protease (SP446) described in Example 19 was added in an amount of 0.04 ATI/d nrnrpin   anH<br><br>
FLAVOURZYME™ described in Example 19 was added in an amount of 20 LAPU/g protein.<br>
Hydrolysis was allowed to proceed without pH adjustment for 18 hours at 50°C. Controls without the addition of peptidoglutaminase were also run.  The hydrolysates were centrifliged and glutamate was measured as described in Example 19.    The DH was determined as described in Example 18.<br>
The results as shown below in Table 7 demonstrated ±at hydrolysis with the peptidoglutaminase preparation increased the DH as well as the release of glutamate.<br>
Table 7<br><br>
Example 21: Production of crude enzyme extract powder from Spingomonas capsulata<br>
Sphingomonas capsulata IFO 12533 was cultivated at 30°C for 9 hours with ieration and agitation in a 5 liter fermentor containing 3 liters of medium composed of 3.5% sucrose, 1% gelatin, 0.5% yeast extract, 1% com steep liquor, 0.3% sodium ;hloride, 0.2% dipotassium hydrogen phosphate, and 0.1% magnesium sulfate-7H20. The :ell mass was collected from the culture broth and washed twice in 10 mM Tris-HCI pH J.O buffer yielding 43 g wet weight of cell mass. The cell mass was suspended in 200 ml )f 10 mM Tris-HCI pH 8.0 buffer and lysed by adding lysozyme and Triton X-100 at final ;oncentrations of 1 mg/ml and 0.1%, respectively, and incubating the solution at 37*'C for tne hour. The solution was ultrasonicated and then centrifiiged at 15,000 g for 10 minutes. The supernatant was recovered (200 ml) and protamine sulfate was added to a final oncentration of 0.1% to precipitate nucleic acids. The precipitate was discarded through entrifugation in the same fashion, and the resulting solution was used as a crude enzyme xtract.<br>
The activity of the crude enzyme extract was determined toward various synthetic eptides as shown in Table 8.   The results demonstrated that the crude enzyme extract ossessed dipeptidyl peptidase IV, leucine aminopeptidase, glycine aminopeptidase, and rolyl oligopeptidase activity. Table 8. Activities of various peptidase activities in the crude enzyme extract solution<br><br><br>
The crude enzyme extract was precipitated by adding an equal volume of cold acetone. The enzyme precipitate was dissolved in a small volume of 10 mM phosphate pH 6.0 buffer and insoluble substances were removed by centrifugation. The enzyme was lyophilized to produce a crude enzyme powder.<br>
Example 22: Purification of Sphingomonas capsulata aminopeptidase I from crude enzyme powder<br>
The crude enzyme powder described in Example 21 was dissolved in 20 mM phosphate pH 7.0 buffer and diluted until the conductivity of the sample was equivalent to the loading buffer, 20 mM phosphate pH 7.0. The sample was loaded onto a Pharmacia Q-Sepharose or Mono Q column pre-equilibrated with 20 mM phosphate pH 7.0 buffer. The flow-throughs from these columns were assayed for aminopeptidase activity with alanine-para-nitroanilide (Ala-pNA) using the procedure described below.<br>
A stock solution of 100 mg of Ala-pNA per ml in dimethylsulfoxide was diluted with 50 mM sodium phosphate pH 7.5 buffer to a concentration of 2 mg per ml. The reaction of the aminopeptidase with the para-nitroanilide was initiated when a 10-50 |al aliquot of the enzyme solution was added to 200 yd of the substrate solution in a microtiter plate well. Analysis of initial rates of hydrolysis of the para-nitroanilide was monitored at 405 nm at room temperamre in a THERMOmax Microplate Reader (Molecular Devices Corp., Sunnyvale, CA, USA).<br>
Aminopeptidase activity was present in the flow-through. The flow-through was concentrated using DIAFLO® PM 10 ultrafiltration membrane (Amicon, Inc., USA) and the pH of the concentrated flow-through was then adjusted to pH 6.0 using 70 mM acetate buffer pH 4.0 buffer<br><br>
The concentrated flow-through was applied to a Pharmacia Mono S 5/5 pre-packed 7 X 50 mm column (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 50 mM MES pH 6.0 buffer. Fractions were assayed as above for aminopeptidase activity. The bound aminopeptidase activity was eluted with a gradient from 0 to 0.2 M NaCl in 50 mM MES pH 6.0 buffer. The fractions with significant activity were then concentrated again using a PM 10 ultrafiltration membrane and then equilibrated in 50 mM phosphate pH 7.0 buffer containing 0.5 M (NH4)2S04.<br>
The concentrated sample was then loaded onto a Phenyl Superose column pre-equilibrated with 50 mM phosphate pH 7.0 buffer contaming 0.5 M (NH4)2S04. The enzyme was eluted with a gradient from 0.5 M ammonium sulfate containing 50 mM phosphate buffer to 50 mM phosphate buffer containing no ammonium sulfate. Active fractions were pooled.<br>
SDS-PAGE analysis of the purified aminopeptidase revealed one band with a molecular weight of 67 kDa. The purified aminopeptidase was designated Sphingomonas capsulata aminopeptidase L<br>
Example 23: Purification of Sphingomonas capsulata aminopeptidase I from crude enzyme powder   .<br>
The crude enzyme powder described in Example 21 was dissolved in distilled water and loaded onto a CM Sepharose CL-6B column (Pharmacia Biotech, Uppsala, Sweden) pre-equilibrated with 15 mM sodium phosphate buffer pH 6.0 buffer. Aminopeptidase I activity was eluted with a linear gradient of 15 mM to 60 mM sodium phosphate buffer (pH 6.0). Fractions of 7.4 ml are collected and assayed for aminopeptidase activity as described in Example 22.<br>
Active fractions were pooled and dialyzed against 15 mM phosphate pH 6.0 buffer. The dialysate was sequentially loaded onto a hydroxylapatite column equilibrated with 15 mM sodium phosphate pH 6.0 buffer. Aminopeptidase I activity was eluted with a linear gradient of 15 mM to 300 mM sodium phosphate 6.0 buffer. Fractions were collected and assayed for aminopeptidase activity. Active fractions were pooled, dialyzed against distilled water, and lyophilized.<br><br>
Example 24: Purification of Sphingomonas capsuUUa aminopeptidase I<br>
Sphingomonas capsulata aminopeptidase I was pm-ified from a culture brodi supernatant produced by cultivation of Sphingomonas capsulata strain IFO 12533 for 15 hours at 31 °C, 250 rpm, and an initial pH of 7.45 in 1.5 liters of medium composed of per liter 10 g of bactopeptone, 5 g of yeast extract, 3 g of NaCl, 2 g of K2HPO4, 0.1 g of MgS04-7H20, and 5 g of glucose (autoclaved separately).<br>
Aminopeptidase I activity was measured with alanine-para-nitroanilide (Ala-pNA) as described in Example 22.<br>
The culture broth supernatant (approximately 1 liter) prepared by centrifugation, filtered using a Whatman glass microfiber filter (Whatman, Maidstone, England), filtered using Nalgene Filterware equipped with a 0.22 mm filter, and concentrated using an Amicon Spiral Ultrafiltration System (Amicon, New Bedford, MA, USA), was equilibrated with 10 mM sodium phosphate pH 6.0 buffer until the conductivity and pH were equal to the loading buffer, 50 mM MES pH 6.0. The filtered solution was loaded onto a 24 x 390 mm column containing approximately 180 ml of SP-Sepharose, Fast Flow (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 50 mM MES pH 6.0 buffer. Protein with aminopeptidase I activity was eluted with a 240 ml gradient from 0 to 0.2M NaCl in 50 mM MES pH 6.0 buffer. Fractions with aminopeptidase I activity were pooled, desalted using a DIAFLO"' PM 10 ultrafiltration membrane (Amicon, New Bedford, MA, USA), and equilibrated with 20 mM sodium phosphate pH 7.0 buffer.<br>
The pooled solution was then loaded onto a Pharmacia MonoQ Beads column pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. Protein with aminopeptidase I activity did not bind to the column and was collected in the flow-through. The flow-through was concentrated using a PM 10 membrane system as above, and the pH adjusted to 6.0 with 70 mM sodium acetate pH 4.0 buffer.<br>
The concentrated flow-through was loaded onto a Pharmacia Mono S column pre-equilibrated with 50 mM MES pH 6.0 buffer. The aminopeptidase was eluted with a 60 ml gradient from 0 to 0.2 M NaCl in 50 mM MES pH 6.0 buffer. The fractions with significant Ala-pNA activity were then pooled, concentrated, and equilibrated with 50 mM phosphate pH 7.0 buffer containing 0.5 M (NH4)2S04 using a PM 10 membrane system as ibove.<br><br>
Finally, the concentrated sample was loaded onto a Pharmacia Phenyl Superose 5/5 pre-packed 7 x 50 mm column (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 50 mM phosphate pH 7.0 buffer containing 0.5 M (NHJSO. Protein with aminopeptidase I activity was then eluted with a 30 ml gradient from 0.5 to 0 M (NH4)2S04 in 50 mM phosphate pH 7.0 buffer. Fractions containing aminopeptidase I activity were analyzed by SDS-PAGE and then pooled.<br>
SDS-PAGE analysis of the purified Sphingomonas capsulata aminopeptidase I revealed one band with a molecular weight of 67 kDa.<br>
Example 25:  Amino acid sequencing of Sphingomonas capsulata aminopeptidase I (67 kDa)<br>
A 20 \x\ sample of the purified Sphingomonas capsulata aminopeptidase I described in Example 22 was electrophoresed and subsequently blot-transferred to a PVDF membrane using 10 mM CAPS (3-[cyclohexylamino]-l-propanesulfonic acid) pH 11 in 10% methanol for 2 hours. The PVDF membrane was stained with 0.1% Coommassie Blue R-250 in 40% methanol/1% acetic acid for 20 seconds and destained in 50% ethanol to observe the protein bands. The major band of 67 kDa was excised and subjected to amino terminal sequencing on an Applied Biosystems Model 476A Protein Sequencer using a blot cartridge and liquid phase TFA delivery according to the manufacturers instructions. The protein was found to be N-terminally blocked to Edman sequencing chemistry.<br>
A 1.0 ml sample of the purified aminopeptidase I described in Example 22 was dried on a Savant Speed Vac AS160 and then reconstituted with 300 fil of 70% formic acid (aqueous). A few crystals of cyanogen bromide were added and incubated at room temperature in the dark overnight. The sample was redried in the Speed Vac and reconstituted in Tricine sample buffer (Novex, San Diego, CA, USA), The cyanogen bromide cleavage fragments were separated using a 10-20% Tricine SDS-poIyacrylamide gel into bands of 42, 30, 17, 15, 10, 6, and 4 kDa and blot-transferred to a PVDF membrane. The 6, 10, 15, 17, 30, and 42 kDa bands were excised and subjected to amino terminal sequencing. The N-terminal sequence of the 10 kDa band was determined to contain the following sequence while the N-terminal sequences of the other bands were inconclusive: AVNGDAYDADKLKGAITNAKTGNPGAGRPI (SEQ ID NO. 11).<br><br>
Example 26: Characterization of Sphingomonas capsulata aminopeptidase I (67 kDa)<br>
Stock solutions of 100 mg of each para-nitroanilide per ml of dimethylsulfoxide were diluted with 50 mM sodium phosphate pH 7.5 buffer to concentrations of 2 mg per ml. Where the substrates were incompletely soluble, their suspensions were used (shown with an asterisk in Table 9 below). The reaction of the Sphingomonas capsulata aminopeptidase I with each para-nitroanilide was initiated when a 10 \i\ aliquot of the enzyme solution in 50 mM sodium phosphate pH 7.0 was added to 190 il of a substrate solution in a 96 well microtiter plate. Analysis of initial rates of hydrolysis of the para-nitroanilides was monitored at 405 lun and 25 "C in a THERMOmax Microplate Reader (Molecular Devices Corp., Sunnyvale, CA, USA). The results (Table 9) showed that the aminopeptidase I preferably hydrolyzed Ala-pNA, but also hydrolyzed Gly-pNA. However, among the dipeptide substrates the aminopeptidase I hydrolysed Gly-Phe-pNA more quickly than Ala-Ala-pNA.<br><br>
The pH optimum for the 67 kDa aminopeptidase was determined using the following protocol.  The buffer solutions at different pH's were prepared by adding distilled water to 4.25 g of sodium acetate and 3.78 g of Tris (free base) to a final volume of 250 ml and adjusting the pH to 8.5, 8.0, 7.5, 7.0, 6.5, 5.80, 5.5, and 5.0 with 36.5-38% HCl. Aliquots of 10 ml were removed at each pH.   A 20 yX volume of a 100 mg/ml solution of<br><br>
Ala-pNA in DMSO was added to 980 /xl of each buffer solution. The pH was then determined to be 8.5, 8.0, 7.57, 7.30, 7.09, 6.68, 6.14, and 5.25 for the buffer solutions after the addition of the substrate. Another solution of substrate at a concentration of 2 mg/ml was prepared at the various pH's. The reaction was initiated by the addition of 10 lA of enzyme solution, diluted 5-fold in 10 mM Tris-HCl pH 7.5, to 200 1 of substrate at the various pHs at room temperature and monitored at 405 nm.<br>
The results shown in Table 10 demonstrated that aminopeptidase I has a pH optimum in the range of 5.3 to 8.5, and more preferably in the range of 7.0 to 7.5 (see Figure 6).  There was no detectable autohydrolysis of Ala-pNa in the pH range 5.25-8.5. At pH 5.5, hydrolysis of Ala-pNA catalyzed by aminopeptidase I did not follow Michaelis-<br><br>
sodium phosphate pH 7.5 buffer over the temperature range of 30'*C to 55°C. The results showed that aminopeptidase I has a temperature optimum in the range of 35*0 to 46°C, and more preferably in the range of 40"C to 45°C at pH 7.5 (see Figure 7).<br>
The temperamre stability of aminopeptidase I was determined by incubating the enzyme for 20 minutes in 50 mM sodium phosphate pH 7.5 buffer at temperatures in the range of 35°C to 55°C followed by cooling on ice and measuring the residual activity using Ala-pNA as substrate at pH 7.5 as described above. The results showed (see Figure 8) that the aminopeptidase was approximately 100% stable up to 40°C at pH 7.5.   At 45'C, the<br><br>
enzyme retained approximately 60% residual activity, while at 500 the enzyme was<br>
completely mactivated.<br>
Example  27:   Comparison between  DH  increasing  effect  of crude  and  puriHed Sphingomonas capsulata aminopeptidase I<br>
The DH increasing effect of Sphingomonas capsulata aminopeptidase I (67 kDa) was evaluated in soy protein hydrolysis and compared to the performance of the crude enzyme powder prepared as described in Example 21.<br>
The degree of hydrolysis (DH) of the soy protein was determined as follows. The DH, defined as described m by Adler-Nissen (1986, Enzymic Hydrolysis of Food Proteins, Elsevier Applied Science Publishers), was determined by reaction of the supernatant with OPA (ortho-phtaldialdehyde, Sigma Chemical Co., St. Louis, MO) essentially as described by Church et al, 1983, Journal of Dairy Science 66: 1219-1227 with correction factors as determined by Adler-Nissen, 1979, Agricultural and Food Chemistry 27: 1256-1262.<br>
The aminopeptidase I was added in increasing dosage to either a low background (1.5% of the substrate protein) or a high background (6% of the substrate protein) of FLAVOURZYME lOOOL™ and a dosage of ALCALASE 2.4L® of 1.5% of the substrate protein.  The experiments were carried out as described below:<br>
Hydrolysis was carried out on a 10 ml scale at SOC for 18 hours. The initial pH was 7 with no pH readjustment during hydrolysis. Inactivation of the enzyme was carried out at 85°C for 3 minutes in a water bath. The substrate concentration was 2% protein from soy bean meal. The substrate was heat treated in a water bath at 85°C for 3 minutes. The enzymes used were FLAVOURZYME IOOOLTM, ALCALASE 2.4L®, crude enzyme powder (see Example 21), and the purified Sphingomonas capsulata aminopeptidase 1 described in Example 24.<br>
The aminopeptidase dosages used were based on alanyl-aminopeptidase units (AAU) determined by the hydrolysis of Ala-pNa at pH 7.5. One AAU unit of the aminopeptidase I is defined as the quantity of an enzyme which hydrolyses 1.0 micromole of Ala-pNa per ninute in a 9.1 mM solution of Ala-pNa at pH 7.5, 22°C, and ionic strength 50 mM.<br>
The specific activity of the purified aminopeptidase I was approximately 12 "AU/mg enzyme. Therefore, a dosage of 0.24 AAU per 200 mg soy protein, for example, :orresponded to 0.1 g enzyme per kg soy protein.<br><br>
The DH as function of dosage of AAU is shown in Figure 9 for low background dosages of FLAVOURZYME™ and in Figure 10 for a high background dosage of FLAVOURZYME™.<br>
Figures 9 and 10 show that the hydrolysates were close to being saturated with AAU at die highest dosages of AAU. The crude enzyme was capable of increasing DH from 44% to 72% when added to a low dosage of FLAVOURZYME'™. The purified aminopeptidase I increased DH to 61-62%. Thus, the purified aminopeptidase I was responsible for (62)/(72)xl00%=64% of die DH mcreasing effect of the crude enzyme. The crude enzyme increased DH from 61% to 77% when added to a high dosage of FLAVOURZYME™. The purified aminopeptidase I increased DH to 71%. Thus, the purified aminopeptidase I was responsible for (71-61)/(77-61)xl00%=63% of the increasing effect of the crude enzyme.<br>
An alternative to addition of aminopeptidase I is the addition of more FLAVOURZYME™. The addition of 0.5% extra FLAVOURZYME™ to the background dosage of 1.5% increased DH from 44% to 48%. The effect of adding aminopeptidase I was calculated to be (62-48)/(728)xl0O% =58% of the effect of the crude enzyme.<br>
Likewise, addition of 1% extra FLAVOURZYME™ to the background dosage of 6% FLAVOURZYME™ increased DH from 61% to 63%. The effect of the aminopeptidase I was calculated to be (71-63)/(77-63)xl00%=57% of the effect of the crude enzyme.<br>
The highest DH obtained was 77%. Since the DH value was based on total protein and not on soluble protein, the protein solubility was around 85%. Therefore, DH in the soluble protein was around 91%, which was very close to 100%.<br>
Example 28: Free amino acid analysis<br>
The % relative increase in the individual free amino acids (FAA) due to AAU additions are shown in Table 11. Hyp, methioninsulphonic acid, and Trp were not included in the Table due to very fluctuating and uncertain results.<br><br><br>
The results showed that when the crude or purified Sphingomonas capsulata ninopeptidase I was added, Gly was the amino acid which showed the highest increase. iier amino acids, which showed increased release were Ala, GIu, Gin, Asp and Ser, but r these amino acids the crude enzyme appeared to release more than the purified linopeptidase I alone probably due to the presence of other aminopeptidases in the crude lyme.<br><br>
A high DH hydrolysate was produced by use of a high dosage of FLAVOURZYME™. The same high DH and release of FAA were obtained by using a low dosage of FLAVOURZYME™ supplemented with 67 kDa aminopeptidase I or the crude enzyme. According to Table 12, a hydrolysate obtained by use of a low dosage of FLAVOURZYME™ supplemented with 67 kDa aminopeptidase I will have a higher level of especially Gly, but also Ala, Glu and Asp and a lower level of Met, Pro and lie as compared with hydrolysates obtained by using a high dosage of FLAVOURZYME"" alone.<br>
Example 29: DH increasing effect of Sphingomonas capsulata aminopeptidase I in<br>
gelatin hydrolysis<br>
The DH increasing effect of the Sphingomonas capsulata aminopeptidase I was<br>
tested in gelatin hydrolysis.<br>
The aminopeptidase I was added in increased dosage to either a low or a high<br>
background of FLAVOURZYME™ and ALCALASE® as performed in Example 27.<br>
Hydrolysis was carried out in 200 il reactions in Eppendorf tubes. The substrate gelatin (Merck) was dissolved in distilled water at 85*0 for 5 minutes. After cooling to 50°C, the pH was adjusted to 6.5, The final gelatin concentration was adjusted to 2% after the addition of enzymes. The substrate concentration in each reaction was calculated to be 4mg.<br>
The enzymes used were FLAVOURZYME™, ALCALASE®, crude enzyme powder, purified aminopeptidase I. Glycine aminopeptidase unit (GAPU) of both crude and purified enzyme was measured in 50 mM Tris-HCl pH 7.5 buffer at 37°C using Gly-pNA as a substrate. Aminopeptidase I was assayed at 37'C in 50 mM Tris-HCl (pH 7.5). Gly-pNA (Bachem Feinchemikalien AG, Bubendorf, Switzerland) was dissolved in 40% lioxane at a concentration of 2.5 mg/ml and 1.5 volume of 50 mM Tris-HCl (pH 7.5) was idded to a substrate concentration of 1 mg/ml. The enzyme sample was diluted in 500 \JX )f the same buffer and 100 \d of the substrate solution was added and incubated at 37°C for j minutes. The reaction was terminated by the addition of 300 (il of 1 M sodium acetate )H 4.0 buffer. For the blank reaction, the stop solution was added to the enzyme solution .nd incubated for 5 minutes at 37°C, and then the substrate solution was added. The bsorbance at 410 nm was measured for both the sample and blank reactions.  The enzyme<br><br>
activity was calculated using the molecular extinction coeficient at 410 nm for pNA as 9480 M"' cm"'. One unit of GAPU was defined as the quantity of enzyme that hydroiyses 1 micromole of Gly-pNA at 37*0 per minute under the conditions described above. The specific activities of the crude and purified aminopeptidase I used were 0.725 GAPU/mg and 6.45 GAPU/mg, respectively. The enzymes were dosed to the 4 mg protein according to the scheme in Table 12.<br><br>
The hydrolysis reactions were performed at 50°C for 18 hours. Enzymes were ictivated at 85 °C for 5 minutes followed by centrifugation. DH was calculated on the sis of the total protein content of hydrolysate using the OPA method.<br>
The DH-increasing effect of purified aminopeptidase I shown in Figures 11 and 12 nonstrated that addition of the crude and the purified enzyme (0.2 U/4 mg gelatin) to the<br><br>
low FLAVOURZYME™ background increased DH from 38% to 67% and 61%, respectively. The DH increasing effect was almost saturated at this dose.<br>
When the dose of FLAVOURZYME™ was increased up to 7%, the control DH increased to 43%. However, the DH obtained with purified aminopeptidase I was 61% which ahnost coincided with those obtained in the low FLAVOURZYME"" background.<br>
These results indicated that the Sphingomonas capsulata aminopeptidase I hydrolyses proteins to an extremely high DH. Furthermore, using aminopeptidase I, the dosage of other proteolytic enzymes can be reduced to obtain the same DH.<br>
Deposit of Biological Materials<br>
The following biological material has been deposited under the terms of the<br>
Budapest  Treaty  with  the  Agricultural  Research  Service	Patent  Culture  Collection,<br>
Northern Regional Research Center, 1815 University Street,	Peoria, Illinois, 61604, and<br>
given the following accession number:<br>
Deposit                                         Accession Number	Date of Deposit<br>
E. coli DH5a pEJGlS                     NRRL B-21677	April 4, 1997<br><br>
SEQUENCE LISTING<br>
[1) GENERAL INFORMATION<br>
(i) APPLICANT:  Novo Nordisk Biotech, Inc 1445 Drew Avenue Davis, California 95616<br>
(i) APPLICANT;  Novo Nordisk A/S Novo Alle DK-2880 Basvaerd<br>
Denmark<br>
(i) APPLICANT:  Asahi Chemical Industry Co., Inc. 2-1 Samejima, Fuji City, Shizuoka Tokyo 416 Japan<br>
(ii) TITLE OF THE INVENTION:  Methods for Producing Protein<br>
Hydrolysates<br>
(iii) NUMBER OF SEQUENCES: 9<br>
(iv) CORRESPONDENCE ADDRESS:<br>
(A)	ADDRESSEE: Novo Nordisk of North America, Inc.<br>
(B)	STREET; 405 Lexington Avenue<br>
(C)	CITY: New York<br>
(D)	STATE: NY<br>
(E)	COUNTRY: U.S.A.<br>
(F)	ZIP: 10174<br>
{v) COMPUTER READABLE FORM:<br>
(A)	MEDIUM TYPE: Diskette<br>
(B)	COMPUTER: IBM Compatible {C) OPERATING SYSTEM: DOS<br>
(D) SOFTWARE: FastSEQ for Windows Version 2.0<br>
{vi) CURRENT APPLICATION DATA:<br>
(A) APPLICATION NUMBER: To Be Assigned {B) FILING DATE: 15-MAY-1998<br>
(C)	CLASSIFICATION:<br>
Viii) ATTORNEY/AGENT INFORMATION:<br>
(A)	NAME: Gregg, Valeta A.<br>
(B)	REGISTRATION NUMBER: 35,127<br>
(C)	REFERENCE/DOCKET NUMBER: 5253.500-IN<br>
(ix) TELECOMMUNICATION INFORMATION:<br>
(A)	TELEPHONE: 212-867-0123<br>
(B)	TELEFAX: 212-878-9655<br>
(C)	TELEX:<br>
I)    INFORMATION FOR SEQ ID NO:1:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1491 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:1:<br>
ATGAGGTCGC TTTTGTGGGC TTCGTTGCTT TCGGGCGTGT TGGCTGGGAG GGCGCTTGTT	60<br>
TCGCCGGATG AGTTCCCCGA GGATATTCAG TTGGAAGATC TGCTGGAAGG ATCCCAACAG	120<br>
CTTGAGGACT TCGCCTATGC CTACCCCGAG CGCAATCGCG TCTTTGGTGG TAAAGCCCAC	180<br>
GACGACACGG TTAACTATCT CTACGAGGAG CTGAAGAAGA CTGGCTACTA TGATGTCTAC	240<br>
AAGCAGCCTC AGGTGCACCT GTGGAGCAAT GCCGACCAGA CGCTCAAGGT GGGCGATGAG	300<br>
GAAATCGAGG CGAAGACCAT GACCTACAGT CCCAGCGTCG AGGTCACCGC CGATGTAGCC	360<br>
GTCGTCAAGA ACCTGGGATG CAGCGAGGCG GATTACCCAT CCGATGTCGA GGGCAAGGTC	420<br>
GCCCTGATCA AGCGTGGAGA ATGCCCGTTC GGCGACAAGT CGGTTCTCGC TGCCAAAGCC	48 0<br>
AAGGCCGCGG CTTCGATTGT CTATAACAAT GTGGCCGGAT CCATGGCGGG CACCCTTGGC	54 0<br>
GCGGCGCAGA GTGATAAGGG ACCGTATTCG GCCATTGTCG GTATCAGCTT GGAGGATGGC	600<br>
CAGAAGCTGA TCAAGCTTGC TGAGGCTGGA TCGGTATCTG TGGATCTGTG GGTGGATAGT	660<br>
AAGCAGGAGA ACCGTACGAC GTATAACGTT GTCGCGCAGA CGAAGGGCGG CGATCCGAAC	72 0<br>
AACGTCGTCG CGCTGGGTGG CCACACGGAC TCAGTCGAGG CGGGCCCTGG TATCAACGAC	780<br>
GATGGCTCGG GCATTATTAG CAACTTGGTC ATTGCCAAAG CGCTCACGCA GTACTCCGTC	840<br>
AAGAATGCCG TGCGCTTCCT CTTCTGGACA GCAGAGGAGT TCGGTCTGCT GGGCAGCAAC	900<br>
TACTACGTCT CCCATCTGAA TGCCACCGAG CTGAACAAGA TCCGACTGTA CCTGAACTTC	960<br>
GACATGATCG CCTCACCTAA CTACGCCCTC ATGATCTATG ACGGTGATGG ATCGGCGTTC	102 0<br>
AACCAGAGCG GACCGGCCGG TTCCGCCCAG ATCGAGAAAC TGTTCGAGGA CTACTACGAC	1080<br>
TCCATCGACC TGCCTCATAT CCCCACCCAG TTTGACGGAC GTTCCGACTA CGAGGCCTTT	1140<br>
ATCCTGAACG GCATTCCGTC CGGTGGACTC TTCACGGGCG CCGAGGGCAT CATGTCCGAA	1200<br>
GAGAACGCAA GCCGCTGGGG AGGTCAAGCC GGCGTGGCCT ACGACGCCAA CTACCACGCC	1260<br>
GCGGGAGACA ACATGACCAA CCTCAACCAT GAAGCCTTCC TGATCAACTC CAAAGCCACC	132 0<br>
GCCTTCGCCG TCGCCACCTA CGCCAACGAC CTCTCCTCGA TCCCCAAACG GAATACCACA	1380<br>
TCCTCCTTGC ACCGACGAGC CCGCACCATG CGACCATTCG GCAAGAGAGC TCCGAAGACA	1440<br>
CACGCTCACG TATCAGGATC CGGATGCTGG CATTCTCAAG TCGAGGCATA G	1491<br>
(2) INFORMATION FOR SEQ ID NO:2:<br>
{i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 496 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single {D} TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:<br>
Met Arg Ser Leu Leu Trp Ala Ser Leu Leu Ser Gly Val Leu Ala Gly<br>
15	10	15<br>
Arg Ala Leu Val Ser Pro Asp	Glu Phe Pro Glu Asp lie Gin Leu Glu<br>
20	25	30<br>
Asp Leu Leu Glu Gly Ser Gin	Gin Leu Glu Asp Phe Ala Tyr Ala Tyr<br>
35	40	45<br>
Pro Glu Arg Asn Arg Val Phe	Gly Gly Lys Ala His Asp Asp Thr Val<br>
50	55	60<br>
Asn Tyr Leu Tyr Glu Glu Leu	Lys Lys Thr Gly Tyr Tyr Asp Val Tyr<br>
65                    70	75	80<br>
Lys Gin Pro Gin Val His Leu	Trp Ser Asn Ala Asp Gin Thr Leu Lys<br>
85	90	95<br><br>
Val Gly Asp Glu Glu lie Glu Ala Lys Thr Met Thr Tyr Ser Pro Ser<br>
100	105	110<br>
Val Glu Val Thr Ala Asp Val Ala Val Val Lys Asn Leu Gly Cys Ser<br>
115	120	125<br>
Glu Ala Asp Tyr Pro Ser Asp Val Glu Gly Lys Val Ala Leu lie Lys<br>
130	135	140<br>
Arg Gly Glu Cys Pro Phe Gly Asp Lys Ser Val Leu Ala Ala Lys Ala<br>
145	ISO	155	160<br>
Lys Ala Ala Ala Ser lie Val Tyr Asn Asn Val Ala Gly Ser Met Ala<br>
165	170	175<br>
Gly Thr Leu Gly Ala Ala Gin Ser Asp Lys Gly Pro Tyr Ser Ala lie<br>
180'	185	190<br>
Val Gly lie Ser Leu Glu Asp Gly Gin Lys Leu lie Lys Leu Ala Glu<br>
195	200	205<br>
Ala Gly Ser Val Ser Val Asp Leu Trp Val Asp Ser Lys Gin Glu Asn<br>
210	215	220<br>
Arg Thr Thr Tyr Asn Val Val Ala Gin Thr Lys Gly Gly Asp Pro Asn<br>
225	230	235	240<br>
Asn Val Val Ala Leu Gly Gly His Thr Asp Ser Val Glu Ala Gly Pro<br>
245	250	255<br>
Gly lie Asn Asp Asp Gly Ser Gly lie lie Ser Asn Leu Val lie Ala<br>
260	265	270<br>
Lys Ala Leu Thr Gin Tyr Ser Val Lys Asn Ala Val Arg Phe Leu Phe<br>
275	280	285<br>
Trp Thr Ala Glu Glu Phe Gly Leu Leu Gly Ser Asn Tyr Tyr Val Ser<br>
290	295	300<br>
His Leu Asn Ala Thr Glu Leu Asn Lys lie Arg Leu Tyr Leu Asn Phe<br>
305	310	315	320<br>
Asp Met lie Ala Ser Pro Asn Tyr Ala Leu Met He Tyr Asp Gly Asp<br>
325	330	335<br>
Gly Ser Ala Phe Asn Gin Ser Gly Pro Ala Gly Ser Ala Gin He Glu<br>
340	345	350<br>
Lys Leu Phe Glu Asp Tyr Tyr Asp Ser He Asp Leu Pro His He Pro<br>
355	360	365<br>
Thr Gin Phe Asp Gly Arg Ser Asp Tyr Glu Ala Phe He Leu Asn Gly<br>
370	375	380<br>
He Pro Ser Gly Gly Leu Phe Thr Gly Ala Glu Gly He Met Ser Glu<br>
385	390	395	400<br>
Glu Asn Ala Ser Arg Trp Gly Gly Gin Ala Gly Val Ala Tyr Asp Ala<br>
405	410	415<br>
Asn Tyr His Ala Ala Gly Asp Asn Met Thr Asn Leu Asn His Glu Ala<br>
420	425	430<br>
Phe Leu He Asn Ser Lys Ala Thr Ala Phe Ala Val Ala Thr Tyr Ala<br>
435	440	445<br>
Asn Asp Leu Ser Ser He Pro Lys Arg Asn Thr Thr Ser Ser Leu His<br>
450	455	460<br>
Arg Arg Ala Arg Thr Met Arg Pro Phe Gly Lys Arg Ala Pro Lys Thr<br>
465	470	475	480<br>
His Ala His Val Ser Gly Ser Gly Cys Trp His Ser Gin Val Glu Ala<br>
485	490	495<br>
2) INFORMATION FOR SEQ ID NO:3:<br>
:i} SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 20 amino acids<br>
(B)	TYPE: amino acid<br><br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii)   MOLECULE  TYPE:   None<br>
(xi)    SEQUENCE  DESCRIPTION:   SEQ   ID  N0:3:<br>
Cys  Cys   He Gly Ala Tyr Gly Ala Arg Thr Thr Tyr Cys  Cys   He Gly<br>
15	10	15<br>
Ala Arg Gly Ala 20<br>
(2) INFORMATION FOR SEQ ID NO:4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 36 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear {ii) MOLECULE TYPE: None<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:<br>
Arg Thr Thr Tyr Thr Thr He Ala Cys He Ala Cys He Gly Cys He<br>
15	10	15<br>
Ala Cys Arg Thr Cys He Gly Cys He Gly Thr He Ala Cys Tyr Thr<br>
20	25	30<br>
Cys He Ala Cys 35<br>
(2) INFORMATION FOR SEQ ID NO:5:<br>
{i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 537 amino acids<br>
(B)	TYPE: amino acid<br>
{C} STRANDEDNESS: single (D) TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:<br>
Met His Phe Ser Leu Lys Gin Leu Ala Val Ala Ala Phe Tyr Ala Thr<br>
15	10	15<br>
Asn Leu Gly Ser Ala Tyr Val He Pro Gin Phe Phe Gin Glu Ala Phe<br>
20	25	30<br>
Gin Gin Glu Glu Pro He Glu Asn Tyr Leu Pro Gin Leu Asn Asp Asp<br>
35	40	45<br>
Asp Ser Ser Ala Val Ala Ala Asn He Pro Lys Pro His He Pro Tyr<br>
50	55	60<br>
Phe Met Lys Pro His Val Glu Ser Glu Lys Leu Gin Asp Lys He Lys<br>
65	70	75	80<br>
Val Asp Asp Leu Asn Ala Thr Ala Trp Asp Leu Tyr Arg Leu Ala Asn<br>
85	90	95<br>
Tyr Ser Thr Pro Asp Tyr Gly His Pro Thr Arg Val He Gly Ser Lys<br>
lOO	105	110<br><br>
Gly His Asn Lys Thr Met Glu Tyr He Leu Asn Val Phe Asp Asp Met<br>
115	120	125<br>
Gin Asp Tyr Tyr Asp Val Ser Leu Gin Glu Phe Glu Ala Leu Ser Gly<br>
130	135	140<br>
Lys He He Ser Phe Asn Leu Ser Asp Ala Glu Thr Gly Lys Ser Phe<br>
145	150	155	160<br>
Ala Asn Thr Thr Ala Phe Ala Leu Ser Pro Pro Val Asp Gly Phe Val<br>
165	170	175<br>
Gly Lys Leu Val Glu He Pro Asn Leu Gly Cys Glu Glu Lys Asp Tyr<br>
180	185	190<br>
Ala Ser Val Val Pro Pro Arg His Asn Glu Lys Gin He Ala Leu He<br>
195	200	205<br>
Glu Arg Gly Lys Cys Pro Phe Gly Asp Lys Ser Asn Leu Ala Gly Lys<br>
210	215	220<br>
Phe Gly Phe Thr Ala Val Val He Tyr Asp Asn Glu Pro Lys Ser Lys<br>
225	230	235	240<br>
Glu Gly Leu His Gly Thr Leu Gly Glu Pro Thr Lys His Thr Val Ala<br>
245	250	255<br>
Thr Val Gly Val Pro Tyr Lys Val Gly Lys Lys Leu He Ala Asn He<br>
260	265	270<br>
Ala Leu Asn lie Asp Tyr Ser Leu Tyr Phe Ala Met Asp Ser Tyr Val<br>
275	280	285<br>
Glu Phe He Lys Thr Gin Asn He He Ala Asp Thr Lys His Gly Asp<br>
290	295	300<br>
Pro Asp Asn He Val Ala Leu Gly Ala His Ser Asp Ser Val Glu Glu<br>
305	310	315	320<br>
Gly Pro Gly He Asn Asp Asp Gly Ser Gly Thr He Ser Leu Leu Asn<br>
325	330	335<br>
Val Ala Lys Gin Leu Thr His Phe Lys He Asn Asn Lys Val Arg Phe<br>
340	345	350<br>
Ala Trp Trp Ala Ala Glu Glu Glu Gly Leu Leu Gly Ser Asn Phe Tyr<br>
355	360	365<br>
Ala Tyr Asn Leu Thr Lys Glu Glu Asn Ser Lys He Arg Val Phe Met<br>
370	375	380<br>
Asp Tyr Asp Met Met Ala Ser Pro Asn Tyr Glu Tyr Glu He Tyr Asp<br>
385	390	395	400<br>
Ala Asn Asn Lys Glu Asn Pro Lys Gly Ser Glu Glu Leu Lys Asn Leu<br>
405	410	415<br>
Tyr Val Asp Tyr Tyr Lys Ala His His Leu Asn Tyr Thr Leu Val Pro<br>
420	425	430<br>
Phe Asp Gly Arg Ser Asp Tyr Val Gly Phe He Asn Asn Gly He Pro<br>
435	440	445<br>
Ala Gly Gly He Ala Thr Gly Ala Glu Lys Asn Asn Val Asn Asn Gly<br>
450	455	460<br>
Lys Val Leu Asp Arg Cys Tyr His Gin Leu Cys Asp Asp Val Ser Asn<br>
465	470	475	480<br>
Leu Ser Trp Asp Ala Phe He Thr Asn Thr Lys Leu He Ala His Ser<br>
485	490	495<br>
Val Ala Thr Tyr Ala Asp Ser Phe Glu Gly Phe Pro Lys Arg Glu Thr<br>
500	505	510<br>
Gin Lys His Lys Glu Val Asp lie Leu Asn Ala Gin Gin Pro Gin Phe<br>
515	520	525<br>
Lys Tyr Arg Ala Asp Phe Leu He He<br>
530	535<br>
(2) INFORMATION FOR SEQ ID NO:6:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 23 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6 ATGATGAGGT CGCTTTTGTG GGC<br>
(2) INFORMATION FOR SEQ ID NO:7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 23 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
{xi) SEQUENCE DESCRIPTION: SEQ ID NO:7: GGGATGCATC TATGCCTCGA CTT<br>
(2) INFORMATION FOR SEQ ID NO:8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 32 base pairs<br>
(B)	TYPE; nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY; linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO;8; ATTTAAATCA CCATGAGGTC GCTTTTGTGG GC<br>
(2) INFORMATION FOR SEQ ID NO:9:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 32 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: cDNA<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO;9: 3GGTTAATTA ACTATGCCTC GACTTGAGAA TG<br>
I)    INFORMATION FOR SEQ ID NO:10:<br>
(i) SEQUENCE CHARACTERISTICS: (A) LENGTH: base pairs<br><br>
(B) TYPE: nucleic acid {C) STRANDEDNESS: single (D) TOPOLOGY: linear<br>
(xi) SEQXffiNCE DESCRIPTION: SEQ ID KO:10:<br>
CAGGTGCAGC CGGCGAGCAA CAGCCGCCCG ATGGCAGTGC CGATCGCTCA TGGGGTGCCC GATGCGCAGG ACGTGCCCTA TCCCGGCACG ATCGGGCTGC AGATCGATGC GACCGATCTG GCCACCGGGG CGTTCCGGGT GGTGGAAACC GTGCCGGTGG CGGCCGATGC CAAGGAACTG ATCCTGCAAC TGCCGGCCTG GCTGCCGGGT GAGCATGGCA ATCGCGGCCC CGTGGCCGAG CTGGCCGGCA TCACGTTTGA AGCCAAGGGC CAGAAGCTGG CCTGGACCCG CGACCCGGTG GAAGTGAACG CGTTCCACAT CCCCCTGCCC GCCGGCACCA GCGAAGTGGT GGCCCGCTTC ATCCACACCT CGCCGCTGCG CGACAGCGAA GGCCGCATCA CCGTTACGCG CGAAATGCTC AACGTGCAGT GGGAGAAGAT GAGCCTCTAT CCCGCCGGTC ACTATGTGCG GCAGATCAAG GTGCGTCCCA CCGTCAGCTT CCCGCAGGGT TGGACCGTGT TCACCGCGCT GGATGGCAAG ACGCAGAGCG GCGCGGGCAA TACCGTGACT TGGGCCGAAA CCGACTATGA AACCCTGGTC GATTCGCCGA TCTTTGCCGG GCTCTATGCC GCGCGGCATG ATCTGGGCCA CAACGTCTAT TTCGATCTGG TGGCCGACAA GCCCGAGCTG CTGGCGATCA AGCCGGAAAA CCTGGCCGCC TATCGCAACC TGGCCGACGA AGCCGTGGGC GCATTCGGCG CGCGCCATTT CGATCACTAC GATTTCCTGC TCGCGCTGAC CGATCGCATG GGCAGCATCG GCCTGGAACA CCACCGTTCC AGCGAAAACC AGCAGGAACC CAAGAGCCTG ACCGACTGGG CCGCCTATGA CTGGGACCGC AACGTGATCG CCCACGAATT CAGCCACAGC TGGGATGGCA AGTATCGCCG CTCGGCCAAG CTGTGGACGC CCGACTATCG CCAGCCGATG CAGGACAACC TGCTGTGGGT CTATGAAGGG CAGACGCAGT TCTGGGGCCT GGTCCTGGCC GCACGCTCGG GCGTGCAGAG CAAGGACGTG GTCTTGGGCA GCCTCGCCAA CTATGCCGGC ACGTTCACCC AGACCGCCGG GCGCGACTGG CGCTCGGTGG AAGACACGAC GATGGATCCC ATCTTCGCCG CCCGCAAGCC CAAGCCCTAT TCCTCGCTTA CCCGTAACGA GGACTATTAC ACCGAAGGCG CGCTGGTGTG GCTGGAAGCG GACCAGATCA TCCGCGATGG CACCGGCGGC AAGAAGGGCC TGGATGATTT CGCCAAGGCG TTCTTTGGCG TGCGCGACGG CGATTGGGGC GTGCTGACCT ATGAATTCGA TGACGTGGTC AAGACCCTCA ACGGCGTCTA TCCCTATGAC TGGGCCACGT TCCTCAAGAC CCGCCTGCAG ACGCCGGGCC AGCCGGTGCC GCTCGGCGGG ATCGAGCGCG GCGGCTACAA GCTGGAATTC AAGGACGAGC CCAACCCCTA TGACAAGGCG CGCATGGCCG ATGCCAAGGT GCTCAGCCTG TTCAACTCGC TGGGCGTGAC GCTGGACAAG GACGGCAAAG TCACCGCCTC GCGCTGGGAT GGCCCGGCGT TCAAGGCGGG GCTGGTTTCG GGCATGCAGG TGATGGCCGT GAACGGCGAC GCCTATGACG CGGACAAGCT CAAGGGCGCG ATCACCAATG CCAAGACCGG CAACCCCGGC GCCGGCCGCC CGATCGAACT GCTGGTCAAG CGTGACGATC GCTTTGTCAC GCTGCCGATC ACCTATGCCG ATGGCCTGCG CTGGCCGTGG CTGGTGCGCA CGGCGCCGGG CACGGCACCG ACCGGGCTGG ACAAGCTGCT GGCCCCGCAC GCCAGCAAGC TGCCCGTGGG CAAGGCTGCC AAGTGA<br>
(2) INFORMATION FOR SEQ ID MO:11:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:<br>
MRKTPQGIGL LSALSTSTLA LATLILAQPA LAQVQPASNS RPMAVPIAHG VPDAQDVPYP GTIGLQIDAT DLATGAFRW ETVPVAADAK ELILQLPAWL PGEHGNRGPV AELAGITFEA KGQKLAWTRD PVEVNAFHIP LPAGTSEWA RFIHTSPLRD SEGRITVTRE MLNVQWEKMS LYPAGHYVRQ IKVRPTVSFP QGWTVFTALD GKTQSGAGNT VTWAETDYET LVDSPIFAGL YAARHDLGHN VYFDLVADKP ELLAIKPENL AAYRNLADEA VGAFGARHFD HYDFLLALTD RMGSIGLEHH RSSENQQEPK SLTDWAAYDW DRNVIAHEFS HSWDGKYRRS AKLWTPDYRQ<br><br>
PMQDNLLWVY EGQTQFWGLV LAARSGVQSK DWLGSLAl AGTFTQTAGR DWRSVEDTTM DPIFAARKPK PYSSLTRNED YYTEGALVWL EADQIIRDGT GGKKGLDDFA KAFFGVRDGD WGVLTYEPDD WKTLNGVYP YDWATFLKTR LQTPGQPVPL GGIERGGYKL EFKDEPNPYD KARMADAKVL SLFNSLGVTL DKDGKVTASR WDGPAFKAGL VSGMQVMAVN GDAYDADKLK GAITNAKTGN PGAGRPIELL VKRDDRFVTL PITYADGLRW PWLVRTAPGT APTGLDKLLA PHASKLPVGK AAK<br><br><br><br><br><br>
WE CLAIM:<br>
1. A method of producing a protein hydrolysate from a proteinaceous material such as herein described comprising the steps of treating said proteinaceous material with atleast two enzymes, one of said two enzymes being one or more polypeptides having glycine releasing activity, and the other being one or more proteases, under known protein hydrolysing conditions and isolating the protein hydrolysate from the reaction mixture by known means, the glycine content of said protein hydrolysate being greater by atleast 10% to 400% than that produced by treating said proteinaceous material with one or more proteases alone under the same condtions.<br>
2.	The method as claimed in claim 1, wherein the percent<br>
increase of glycine is between 25-400%, preferably 50-350%, more<br>
preferably 100-350%,  even more preferably 150-350%  and most<br>
preferably 200-350%.<br>
3.	The method as claimed in claim 1 or 2, wherein the degree of protein hydrolysis is between 35-90%, preferably 45-80%, more preferably 50-75%, even more preferably 55-75%, and most preferably 60-70%.<br>
4.	The. method as claimed in any of claims 1-3, wherein said polypeptide is selected from the group consisting of:<br>
(a) a polypeptide having an amino acid sequence which has at least 50% identity with the amino acid sequence of SEQ ID NO. 2;<br><br>
(b)	a polypeptide encoded by a nucleic acid sequence<br>
which hybridizes under known medium stringency conditions with<br>
(1) the nucleic acid sequence of SEQ ID NO.l, (ii) its<br>
complementary strand, or (iii) a subsequence thereof;<br>
(c)	an allelic variant of (a) or (b); and<br>
(d)	a fragment of (a), (b), or (c) wherein the fragment has aminopeptidase activity; and<br>
(e)	a polypeptide having aminopeptidase activity with<br>
physicochemical properties of (i) a pH optimum in the range of<br>
from about pH 7.27 to about pH 10.95 determined at ambient<br>
temperature in the presence of Ala-para-nitroani1ide; (ii) a<br>
temperature stability of 90% or more, relative to initial<br>
activity, at pH 7.5 determined after incubation for 20 minutes at<br>
60oC in the absence of substrate; and (iii) an activity towards<br>
Xaa-para nitroani1ide wherein Xaa is selected from the group<br>
consisting of Leu, Glu, Gly, Ala and Pro.<br><br>
5.	The method as claimed in claim 4, wherein said<br>
polypeptide has an amino acid sequence which has a degree of<br>
identity to the amino acid sequence of SEQ ID N0;2 of at least<br>
about 50% preferably at least about 60%, preferably at least<br>
about 70%, more preferably at least about 80%, even more<br>
preferably at least about 90%, most preferably atleast about 95%,<br>
and even most preferably at least about 97%, which have<br>
aminopeptidase activity(hereinafter "homologous polypept ides).<br>
6.	The  method  as  claimed  in  claim  4,  wherein  said<br><br>
a mutant strain thereof, e.g., the polypeptide with the amino acid sequence of SEQ ID N0:2.<br>
7.	The method as claimed in any of claims 1-3, wherein<br>
said polypeptide is selected from the group consisting of:<br>
(a)	a polypeptide having an amino acid sequence which<br>
has  atleast 50% identity with the amino acid sequence of SEQ  ID<br>
(b)	a polypeptide encoded by a nucleic acid sequence<br>
vhich hybridizes under known medium stringency conditions with<br>
(i) the nucleic acid sequence of SEQ ID NO:10, (ii) its<br>
complementary strand, or {i i i) a subsequence thereof;<br>
(c)	an allelic variant of (a) or (b); and<br><br>
(d) a fragment of (a), (b), or (c), wherein the fragment has aminopeptidase activity; and<br>
(e) a polypeptide which (a&gt; has aminopeptidase activity in the pH range between pH 5.0-8.5 measured at 37°C, (b) an isoelectric  point  (pI)  in  the  range  of  7.4-8.5  (c)   has<br><br>
chloromercuribeuzoic acid and iodoacetie acid, completely inhibited  by  o~phenanthrolin,  and/or (g) is  obtained  from  a<br><br>
8. The method as claimed in claim 7, wherein said polypeptide has an amino acid sequence which has a degree of identity to the amino acid sequence of SEQ ID HO:11 of atleast about 50%, preferably at least about 60%, preferably at least about 70%, more preferably at least about 80%, even more preferably at least abaout 90%, most preferably at least about 95%, and even most preferably at least about 97%, which have aminopeptidase activity (hereinafter "homologous polypeptides").<br><br>
9.        The  method  as  claimed  in  claim  7,  wherein  said<br><br>
which is a bacterial strain kept at the Institute for Fermentation, Osaka, Japan, e.g., the polypeptide with the amino acid sequence of SEQ ID HO:11 (the predicted signal peptide consists of the first 32 amino acids).<br>
10.	The method as claimed in any of the claims 4,5,7 or  8,<br>
wherein  said  polypeptide  is obtained by  known  means  from a<br>
bacterial  source  such  as a gram postiive  bacterium,  e.g.,  a<br><br>
11.	The method as claimed in claims 4,5,7 or 8, wherein<br>
said polypeptide is obtained by known means from a fungal<br>
source,  preferably  from  a  yeast strain  such  as  a  Candida.<br><br><br><br>
12.	The method as claimed in any of the claims 4,5,7 or 8<br>
wherein  said  polypeptide  is obtained by  known  means  from  a<br><br>
13.	The method as claimed in claims 4,5,7 or 8, wherein the<br>
polypeptide   is  obtained  by  known means  from  a  Fusarium<br><br><br>
14.	The method as olaimed in any of the claims 4,5,7 or  8,<br><br>
15.	The method as olaimed in any of claims  1-14,  wherein<br>
the one or more additional proteases are endopeptidases.<br>
16.	The method as olaimed in claim 15, wherein the endopeptidase is a glutamyl endopeptidase (EC 3.4.21.19); a lysyl endopeptidase (EC 3.4,21.50); a leuoyl endopeptidase (EC 3.4.21.57); a glyoyl endopeptidase (EC 3.4.22.25); a prolyl endopeptidase (EC 3.4.21.26); trypsin (EC 3.4.21.4) or a trysin-like (lysine/arginine specific&gt;endopeptidase; or a peptidyl-Asp metalloendopeptidase (EC 3.4.24.33).<br>
17.	The method as claimed in any of claims 1-16, wherein the one or more additional proteases are an exo-peptidase that may act from either end of the peptide.<br>
La. The method as claimed in claim 17, wherein the exopeptidase is an aminopeptidase such as a leuoyl aminopeptidase EC 3.4.11.1); a dipeptide aminopeptidase or a trlpeptide minopeptidase <ec></ec><br>
19. The method as claimed in claim 18, wherein the exopeptidase  is a carboxypeptidase such as a proline oarboxypeptidase (EC 3.4.16.2); a carboxypeptidase A (EC 3.4.17.1); a carboxypeptidase B (EC 3.4.17.2); a carboxypeptidase C (EC 3.4.16,5); a carboxypeptidase D (EC 3.4.16.6); a lysine (arginine) carboxypeptidase (EC 3,4.17.3); a glycine carboxypeptidase (EC 3.4.17.4); an alanine carboxypeptidase (EC 3.4.17.6); a glutamate carboxypeptidase (EC 3.4.17.11); a peptidyl-dipeptidase A (EC 3,4.15.1); or a peptidyl-dipeptidase (EC 3.4.15.5).<br>
20.	The method as claimed in any of the claims 1-14,<br>
wherein  the  one or more addtiional proteases is  a mixture  of<br><br>
21.	The method as claimed in any of the claims 1-20,<br>
further comprisisng adding to the proteinaceous material one or<br>
more other enzymes selected from the group consisting of amylase.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1063-mas-1998 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1063-mas-1998 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1063-mas-1998 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1063-mas-1998 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">1063-mas-1998 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1063-mas-1998 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1063-mas-1998 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1063-mas-1998 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1063-mas-1998 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">1063-mas-1998 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">1063-mas-1998 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA2My1tYXMtMTk5OCBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">1063-mas-1998 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="187834-a-process-for-the-preparation-of-n-4-fluorophenyl-n-2-carboxy-4-chlopophe-nyl-glycine.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="187837-a-process-for-producing-a-pharmaceutical-lozenge-formulation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>187836</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1063/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jan-2003</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-May-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVO NORDISK BIOTECH INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1445 DREW AVENUE, DAVIS, CALIFORNIA 95616</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ALEXANDER BLINKOVSKY</td>
											<td>955 CRANBROOK COURT #277, DAVIS, CA 95616</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07B41/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/069,719</td>
									<td>1997-12-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>08/857,886</td>
									<td>1997-05-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/062,893</td>
									<td>1997-10-20</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/187836-method-of-producing-protein-hydrolysates by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:34:57 GMT -->
</html>
