<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/188176-an-apparatus-for-detecting-nucleotide-target-sequence-in-a-liquid-medium by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:47:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 188176:&quot;AN APPARATUS FOR DETECTING NUCLEOTIDE TARGET SEQUENCE IN A LIQUID MEDIUM&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN APPARATUS FOR DETECTING NUCLEOTIDE TARGET SEQUENCE IN A LIQUID MEDIUM&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides a method for rapidly, economically and efficiently sequencing and assaying nucleotides in a liquid medium using PNA probes. The method is particularly advantageous in not requiring the use of a solid support to bind the probe or the target sequence. The method is capable of providing information regarding the nature and existence of hybridization.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>invention farset realates to an apparatus for sefeching<br>
amdesodisequence in a kuqid medium  construct. Description of Related Art PNAs are polyamide analogs of DNA and RNA. See, e.g., U.S. Patent No. 5,539,082 to Nielsen et al. Nielsen et al. discloses that PNAs mimic natural polynucleotides by binding complementary single stranded (ss) DNA and RNA strands. PNAs generally comprise ligands linked to a peptide backbone. Representative ligands include either the four main naturally occurring DNA bases (i.e., thymine, cytosine, adenine or guanine) or other naturally occurring nucleobases (e.g., inosine, uracil, 5-methylcytosine or thiouracil) or artificial bases (e.g., bromothymine, azaadenines or azaguanines, etc.) attached to a peptide backbone through a<br>
Probes comprising PNA sequences have been employed to detect target nucleotide sequences. U.S. Patent No. 5,503,980 to Cantor suggests employing PNA probes in a method of sequencing a nucleic acid by hybridizing the nucleic acid with a set of PNA probes containing random, but determinable, base sequences within the single stranded portion adjacent to a double stranded portion, wherein the single stranded portion of the set preferably comprises every possible combination of sequences over a predetermined range. Hybridization occurs by complementary recognition of the single stranded portion of a target with the single stranded portion of the probe and is thermodynamically favored by the presence of adjacent double strandedness of the probe.However, although Cantor discloses that the nucleic acids can be PNAs, it does not disclose or suggest utilizing such probes in the absence of a solid support. Moreover, the present invention does not require the adjacent construct of DNA material being tested.<br>
In addition to teaching the use of a solid support like Cantor, Perry-O'Keefe et al., "Peptide Nucleic Acid Pre-Gel Hybridization: An Alternative to Southern Hybridization," 93 Proc. Natl. Acad. Sci. USA 14670 (December 1996) also teaches that PNA does not generally bind well to double stranded DNA (dsDNA) . See Perry-O'Keefe et al. at page 14 673, footnote. Moreover, the homopyrimidine PNA constructs which have been found to bind dsDNA well would not be useful as probes. Applicants have discovered that the qualification which suggests that only homopyrimidine can bind with dsDNA by strand inversion is incorrect and arises from the hybridization conditions employed.<br>
Smulevitch et al., "Enhancement of Strand Inversion by Oligonucleotides Through Manipulation of Backbone Charge," 14 Nature Biotechnology 1700 (Dec. 1996) (disclosed in Landsdorp, "Close Encounters of the PNA Kind," 14 Nature Biotechnology 1653 (Dec. 1996)) discloses" using PNA primers to hybridize with dsDNA. However, Smulevitch et al. teaches the use of gels in detecting hybridization, and does not suggest the use of fluorescent markers.<br>
Many types of sample analysis rely upon the fluorescent properties of a stain. Fluorescence occurs when a molecule excited by light of one wavelength returns to the unexcited (ground) state by emitting light of a longer wavelength. The exciting and emitted light, being of different wavelengths, can be separated from one another using optical filters, a camera or a CCD. Fluorescence has been used to visualize certain molecules (and hence structures) by light microscopy for many years, and is also used in other analytical techniques, such as flow cytometry.  Further, the emission of<br>
fluorescence showing different colors can be detected by a human eye, a camera or a charge coupled device (CCD).<br>
For example, U.S. Patent No. 5,594,138 to Dykstra et al. discloses a method of fluorescent detection of a nucleic acid. The method comprises contacting the nucleic acid with a fluorescent marker that is a bis-dicationic aryl furan compound and exposing the nucleic acid to light at a frequency inducing fluorescence of the fluorescent marker. The fluorescent marker may be conjugated to a nucleotide sequence as a probe for hybridization studies, or it may be conjugated to numerous reagents for in situ labeling studies.<br>
U.S. Patent No. 4,963,477 to Tchen discloses a probe of high sensitivity containing a modified nucleic acid, which can be recognized by specific antibodies.<br>
Fluorescent In Situ Hybridization (FISH) is a technique comprising detecting fluorescent probe binding to human Chromosomes by attaching DNA to a solid support, such as a glass slide. See, e.g., K.H. Andy Choo, Ed., "In Situ Hybridization Protocols," Chapters 2 and 4 (Humana Press, Totowa, NJ, 1994). Like all other conventional detection methods comprising hybridization with probes, this method relies on the solid support to keep the two complementary strands of DNA apart while the probe hybridizes with one of the strands. In addition, FISH requires a complicated buffer and temperature control protocol, with overnight incubation.<br>
Until the present invention, however, it has not been possible to rapidly test for the presence of nucleotide sequences in solution using a method which does not destroy the sample, is less hazardous to laboratory personnel than radiation based assays, does not require the cost and delay of preparing solid supports, and is readily automated. Time and cost efficient detection of mutant genetic sequences has been the rate limiting step in correlating mutant genotypes with altered phenotypes. Although conventional DNA sequencing methods have been considered to be the most accurate means of<br>
 identifying mutations, these methods have been relatively slow and labor intensive, and are not particularly well-suited to rapidly screening large number of samples of genomic DNA.<br>
All references cited herein are incorporated herein by reference in their entireties .<br>
SUMMARY OF THE INVENTION<br>
The present invention provides methods for detecting nucleic acid sequence and/or sequence information from nucleic acids Probes according to the present invention include PNA.<br>
The invention provides a method for detecting at least one single stranded or double stranded nucleotide sequence in a liquid medium, the method comprising adding to the liquid medium PNA probes having at least one marker such as fluorescein each to form at least one hybridization complex with at least one nucleotide sequence in the medium, detecting the at least one nucleotide sequence by detecting at least one signal correlated with an amount of the at least one hybridization complex in the liquid medium.<br>
The method can be conducted without binding the PNA probes, nucleotide sequences or hybridization complexes to a solid support or gel<br>
The present invention relates to an apparatus for detecting at least one single stranded or double stranded nucleotide target sequence in a liquid medium, said apparatus comprising:<br>
a container adapted to contain said liquid medium and at least one PNA probe having a fluorescent marker, capable of forming a hybridization complex with said at least one said target sequence,,<br>
means provided in sequence with the said container for separating unhybndized PNA probe from said hybridization complex to form a test medium;<br>
a laser source provided in sequence with the said means for irradiating said test medium with a laser beam having a wavelength which excites said fluorescent marker and causes said fluorescent marker to emit fluorescent light;<br>
a fluorescence detector for measuring an intensity of said emitted fluorescent light emitted by the liquid medium in the said container wherein said measured intensity is inversely proportional to a number of base mismatches between said at least one target sequence and said PNA probe, over a range inclusive of 0 base mismatches through at least 3 base mismatches<br>
an output device for receiving and comparing said measured intensity with a reference intensity received from the detector to detect whether said liquid medium contains said at least one target sequence,<br>
wherein said PNA probe, said nucleotide sequence and said hybridization complex are not bound to a solid support or gel<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
The invention will be described in conjunction with the accompanying drawings in which like reference numerals designate like elements and wherein;<br>
Figure 1 is a schematic depiction of an apparatus according to the invention<br>
Figures 2, 3A, 3B, 4A, 4B, 4C, 5A, 5B, 6A, 6B, 7A, 7B, 8A, 8B, 8C, 9A, 9B, 9C, 9D, 10A, 10B, 11A, 11B, 12A, 12B, 13A, 13B, 13C, 14A, 14B, 15A, 15B, 16A, 16B, 16C, 16D, 17A, 17B, 17C, 18A, 18B, 19A, 19B, 19C, 19D and 19Eare fluorescent spectra<br>
DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS The invention utilizes PNA probes to detect and/or characterize nucleotide sequences in a sample. PNA probes are able to recognize dsDNA by binding one strand, thereby presumably hybridizing with the other strand to generate a PNA-DNA complex. Such recognition can take place to dsDNA target sequences 20 or more base pairs long. Probe sequences having any length from 8 to 20 bases are preferred since this is the range within which the smallest unique DNA sequences of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome. However, a plurality of shorter probes can be used to detect a nucleotide sequence having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence.<br>
The probes of the invention are able to form triplex complexes with dsDNA and duplex complexes with RNA or ssDNA. The compounds of the invention are also able to form triplex complexes wherein a first PNA probe binds with RNA or ssDNA and a second ssDNA strand binds with the resulting duplex complex. See, e.g., Egholm et al., "PNA Hybridizes to Complementary Oligonucleotides Obeying the Watson-Crick Hydrogen-Bonding Rules," 365 Nature 566 (1993), and Tomac et al., "Ionic Effects on the Stability and Conformation of Peptide Nucleic Acid Complexes," 118 J.Am.Chem.Soc. 5544 (1996) .<br>
In the PNA probes according to the invention, the bases attached to the polyamide backbone are primarily naturally occurring nucleobases attached at the position required by probe manufacture. Alternatively, the bases may be non-naturally occurring nucleobases (nucleobase analogs) , other base-binding moieties, aromatic moieties, (C1-C4) alkanoyls, hydroxyls or even hydrogens. It will be understood that the term  nucleobase  includes  nucleobases  bearing  removable<br>
protecting groups. Furthermore, at least one base on the polyamide skeleton can be replaced with, or substituted with, a DNA intercalator, a reporter ligand such as, for example, a fluorophore, radio label, spin label, hapten, or a protein-recognizing ligand such as biotin. Preferred detectable labels include a radioisotope, a stable isotope, an enzyme, a fluorescent chemical, a luminescent chemical, a chromatic chemical, a metal, an electric charge, or a spatial structure.<br>
In particularly preferred embodiments, the PNA probe comprises a PNA sequence covalently bonded to a fluorescent marker, which fluoresces when irradiated with a laser. Preferred fluorescent markers include biotin, rhodamine and fluorescein.<br>
It is preferred to provide the fluorescent marker at the 5' terminal of the PNA with a short linker to minimize interaction with the PNA.<br>
In order to distinguish a mutant nucleotide sequence from a reference nucleotide sequence, wherein the two sequences differ by as little as a single base, it is preferred to design the PNA probe so that the mutant portion of the mutant nucleotide corresponds to the center of the PNA probe. This design results in a higher hybridization yield and a more stable hybrid than when the mutant portion of the nucleotide corresponds to a terminus of the probe, since the bonding mismatch between probe and nucleotide is located centrally within the probe.<br>
PNA probes are added to a liquid medium suspected of containing at least one nucleotide sequence, and/or a mutant version of the at least one sequence. The liquid medium can be any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al., "Molecular Cloning: A Lab Manual," 2d (1989). For example, the liquid medium can comprise nucleotides, water, buffers and surfactants.<br>
The nucleotides in the liquid medium can be obtained from clinical samples by any conventional method, including an automated method. Examples of such methods are summarized in, e.g., Sambrook et al., Vol. 2, pp. 9.16-9.19 and 7.6 to 7.7. An example of an automated nucleic acid purifying apparatus is the BioRobot 9600 manufactured by Quiagen.<br>
For example, a variety of diseases are known to be linked with the presence of mutant DNA in an individual's genome. If the sequences of the wild type DNA and the mutant DNA are known, it is possible to isolate these nucleotide sequences from clinical samples using conventional technology. PCR is the preferred method of isolating nucleotides from clinical samples. PCR is conducted using a primer which is capable of amplifying the wild type DNA and the mutant DNA.<br>
The nucleotide sequences are added to the liquid medium in a known concentration, since the concentration can affect the magnitude of the signal (e.g., fluorescent intensity) generated in subsequent steps in the inventive method. The nucleotide concentration can be determined by, e.g., measuring the UV absorption at 260 nm.<br>
The isolated nucleotides are added to the liquid medium and denatured prior to being detected. Preferably, the denaturation is conducted at about 90°C to about 100°C from about 30 seconds to about 5 hours in the presence of PNA<br>
Preferably, PNA probes are added to the liquid medium in a concentration 1 to 20 times the concentration of the nucleotide sequence to be detected.<br>
Hybridization between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are known in the art. See, e.g., Perry-O'Keefe et al., Egholm et al., Tomac et al., Sambrook et al., Vol. 2<br>
pp. 9.47-9.55 and the Pre-Gel Hybridization Technique taught in Vol. 4, No. 3 of PerSeptive Biosystems Magazine.<br>
It is preferred that hybridization complexes be formed at a temperature of about 4°C to about 75°C for about 2 minutes to about 24 hours. It is particularly preferred to conduct denaturing for no more than 60 minutes in the presence of PNA probe, after which the temperature is passively cooled to room temperature without quenching.<br>
It is possible to facilitate hybridization in solution by using certain reagents. Preferred examples of these reagents include single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, divalent ions, polyvalent ions, and intercalating substances such as ethidium bromide, actinomycin D, psoralen, and angelicin.<br>
In embodiments of the inventive method, hybridization complexes are separated from unhybridized PNA probes prior to detecting the signal of the hybridized PNA probes. The separation is accomplished by at least one of filtration, centrifugation, precipitation, ion exchange resin separation and free solution electrophoresis (i.e., electrophoresis in a liquid medium as opposed to gel electrophoresis).<br>
Separation can be accomplished by G50 column. After hybridization, the mixture of hybridization complexes and unhybridized DNA and PNA is transferred into a G50 column. The column is centrifuged at 500 to 1000 rpm for 1 to 2 minutes. The unbound PNA is filtered and retained in the column. The solution with the hybrids of PNA-DNA is passed through column and collected in a cuvette.<br>
In fact, centrifugation can be avoided. The solution can flow through the column by gravity or washing using extra amounts of buffer. However, centrifugation is for the purpose of reducing the separation time and avoiding the sample being diluted.<br>
Separation can be accomplished by centrifugation. After the hybridization, the sample (without transferring) is separated into two layers (gradient) by centrifugation at 1000 to 10000 rpm for 4 to 20 hours. The lighter unhybridized PNA is abundant in the upper layer, while the heavier hybridization complex is concentrated in the lower layer. The lower layer is collected in a cuvette for fluorescence measurement.<br>
In certain embodiments, a filtration step is avoidable if an electrical method is used to concentrate hybridized PNA. In these embodiments, an electric field is applied to the liquid medium prior to or concurrently with detecting the desired nucleotide sequence, and a change in an intensity of fluorescent emission as a function of the electric field is detected as an indication of whether the PNA probe is hybridized to at least one of a completely complementary nucleotide sequence and an incompletely complementary nucleotide sequence.<br>
The preferred markers for use in the invention are fluorophores. As will be appreciated by the skilled artisan, the wavelength preferably selected to induce fluorescence of the fluorescent marker is known in the art as the "excitation maximum," i.e., that wavelength which is absorbed by a molecule and excites that molecule to a higher electronic state. When the marker molecule passes from the higher to a lower electronic state, the molecule emits a type of visible radiation, i.e., fluorescence, at a wavelength referred to as the "emission maximum." It is this fluorescence that is detected in the present invention. The detectable signal emitted by the compound can be detected using techniques known in the art, for example by observation with the human eye, using electronic means for detecting a generated wavelength (e.g., cameras and CCDs), and the like. Advantageously,   the   wavelength   of   fluorescence   is<br>
sufficiently removed from that of the exciting light to allow good separation of the two wavelengths by optical filters.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond<br>
preferably 400 to 1000 nm, more preferably 400 to 750 nm. For example, when the marker is fluoroscein, the preferred wavelength of excitation is about 488 nm.<br>
In preferred embodiments, an argon ion laser is used to irradiate- the marker with light having a wavelength in a range of 400 to 520 nm, and fluorescent emission is detected in a range of 500 to 750 nm. The duration of irradiation is preferably about 10 milliseconds to about 1 minute.<br>
An apparatus for performing the inventive method can comprise a liquid medium container for containing the liquid medium; a laser for irradiating the nucleotide; a fluorescence detector for detecting fluorescence induced by the laser; a data analysis device for analyzing data generated by the fluorescence detector; and an output device which reports the data analysis generated by the data analysis device. See, e.g., Fig. 1, which shows a schematic diagram of a fluorescence detection system suitable for use with the method of the invention.<br>
In certain embodiments, fluorescent emission generated by irradiating hybridized PNA probes with a light source is distinguished from fluorescent emission of unhybridized PNA probes, without separating hybridized and unhybridized PNA probes. In certain embodiments, the fluorescent emission of one type of PNA probe hybridized to one nucleotide sequence is distinguished from the fluorescent emission of another type of PNA probe hybridized to a nucleotide sequence other than the first nucleotide sequence. For example, the presence of either of two nucleotide sequences differing by as little as a single nucleotide can be detected on the basis that a fluorescent PNA probe precisely complementary to one<br>
of the two nucleotides will bond more effectively with the precise complement than with the imprecise complement, thus providing a higher concentration of hybridization complexes in solution and a higher fluorescent intensity aftei of unhybridized PNA probe.<br>
A plurality of PNA probes can be employed simult; to achieve a variety of effects. Several probes targeted for different segments of a single nucleotide sequence can be employed to enhance the reliabilircy of the detection method. Similarly, one probe can target one strand of dsDNA, while another probe can target the complementary strand of dsDNA.<br>
A preferred method of detecting whether DNA is mutant type or the correspondng wild type comprises the simultaneous use of (a) a first type of PNA probe targeted to a sequence that occurs in both the wild type and mutant type DNA but is otherwise unique, and (b) a second type of PNA probe targeted to a sequence unique to the mutant type DNA, wherein the first and second types of PNA probe have different markers that produce distinguishable signals. Thus, detection of the first probe signal indicates that the test was run properly (i.e., the first probe functions as a positive control) and detection of the second probe signal indicates that the mutant type DNA is present. For example, one probe can have a fluorescein marker exhibiting a fluorescent emission intensity peak at 525 nm while the other probe can have a rhodamine marker exhibiting a fluorescent emission intensity peak at 580 nm.<br>
liquid medium (i.e., the sample to be ; embodiments to no more than about 200 :<br>
When testing for mutant dsDNA using PNA, if obtained for which there remains doubt, a further<br>
immediately performed on the sample by adding the complementary PNA probe to test the complementary strand of DNA. If centrifugated as part of the first test, the sample would be removed to a new tube and hybridized with the complementary PNA. Alternatively, the PNA test can be done with both the PNA and complementary PNA probes hybridized to each of the denatured DNA strands in the first instance and at the same time.<br>
For forensic applications, samples can be tested, stored and then retested because PNA is expelled from hybridization over a couple of days, and DNA recombines over time and does not degrade by this procedure. Accordingly, a sample frozen after testing can be subsequently retested in the same tube a number of times.<br>
Figure 2 shows DNA hybridization over time. A PNA probe was allowed to hybridize with wild type DNA (WT DNA) and mutant DNA (MT DNA). Hybridization between the probe and the DNA was followed over time by measuring fluorescent intensity. Figure 2 shows that hybridization of the PNA probe with the DNA eventually decreases over time, suggesting that the DNA resumes its native structure over time by displacing PNA.<br>
Clinical samples can be tested using at least 100 times less chemicals or genomic material (100 microliters vs. 10 milliliters) than is typical in conventional methods. Therefore, even using 10 or 20 times the concentration of PNA conventionally used, the tests still only consume l/5th to l/10th the amount of PNA , while obtaining a very decisive<br>
The invention will be illustrated in more detail with reference to the following Examples, but it should be understood that the present invention is not deemed to be limited thereto.<br>
EXAMPLE 1<br>
A 150 bp (base pair) fragment of genomic DNA from wild type p53 gene (SEQ ID NO:l) was amplified by PCR. PCR was conducted using a GeneAmp PCR system 2400 from Perkin Elmer, with a hot start at 95°C for 5 minutes. After adding Taq enzyme, 35 cycles were carried out as follows: Denaturing: 94°C for 30 mins. Annealing:     45°C for 45 mins. (45ºC is 5°C lower<br>
than the primer Tm) Extension:     72°C for-30 mins. (1 kb/min.) x 35.<br>
PCR was conducted using a mixture comprising in 100 ml volume 10 x reaction buffer (10 µl) , 20 mM of MgCl2 (10 µl) , 10 mM of dNTP mixture (2), 25 pmol/µl of primer 1 (1 µl), 25 pmol/µl of primer 2 (1 µl), 100 µg/µl of DNA template (1 µl), dd H20 (74 µl) and Taq polymerase (1 µl) (5 unit/µl) .<br>
Similarly, a mutant fragment (SEQ ID NO:2) of the same genomic 150 bp fragment was also amplified by PCR. The mutant fragment was identical to the wild type fragment except for a point mutation at amino acid position 344 at which the DNA wild type sequence CTG was changed to CAG.<br>
A 12-mer PNA probe was obtained from PerSeptive Biosystems, Inc. (Framingham, MA, USA) and was designed to be complementary to a 12 nucleotide segment of the 150 bp p53 wild type fragment (SEQ ID NO:l). The probe had the following structure:<br>
5' H-FluO CAT TCA GCT CTC Lys -CONH2<br>
The buffer solution for hybridization was lOmM Tris-HCl, pH 7.1, 1% by weight BSA (bovine serum albumin), and 0.1% Triton X-100 (t-octylphenoxypolyethoxyethanol). 50 pmol of wild type genomic DNA was added to 300 pmol of PNA probe and mixed in 100 ml of buffer. 50 pmol of mutant genomic DNA was added to 300 pmol of PNA probe and mixed in 100 microliters of buffer.<br>
Each sample was heated at 95°C for 30 minutes. Each sample was then added to buffer preheated to 30°C and allowed to hybridize for 1 hour.<br>
The components of each sample were separated by G50 spin columns (Pharmacia Biotech, Uppsala, Sweden) by spinning at 600g for 2 minutes. The unbound PNA was filtered and retained in the column. The solution with the hybrids of PNA-DNA passed through the column and was collected in a cuvette for fluorescent detection. The cuvette was then placed in a fluorescence spectrometer for laser induced fluorescence (irradiation wavelength of 488 nm in all of the Examples) and detection of the fluorescent probe attached to the target genomic DNA. As shown in Figures 3B and 3C, the relative fluorescence intensity at 525 nm of the wild type genomic DNA sample (Figure 3B) was four times higher than that of the mutant sample (Figure 3C) , allowing the mutant genomic DNA sample to be distinguished from the wild type genomic DNA sample. Figure 3A shows the fluorescent intensity of the probe alone. EXAMPLE 2<br>
This Example was similar to Example 1 except for the following details. The mutant genomic DNA sample (SEQ ID NO: 3) had the base sequence at amino acid position 344 changed from the wild type sequence CTG to CGG. Both samples were heated at 94°C for 20 minutes. The buffer was preheated to 25°C and the samples were allowed to hybrid for 30 minutes. The components of each sample were separated by using G50 spin columns and spinning at 800 rpm for 1.5 minutes. The relative intensity at 525 nm of the detected fluorescence from the wild type genomic DNA sample (Figure 4B) was six times higher than that of the mutated genomic DNA sample (Figure 4C), allowing the wild type genomic sample to be distinguished from the mutated genomic sample. Figure 4A shows the fluorescent intensity of the probe alone in the<br>
This Example was similar to Examples 1 and 2 except for the following details. The mutant genomic DNA sample (SEQ ID NO: 4) had a 2 base change from the wild type genomic DNA at amino acid position 344, from CTG to AAG. Each sample was initially heated at 100°C for 45 minutes. The buffer was preheated to 45°C and each sample was allowed to hybridize for 20 minutes. The components of each sample were separated by using G75 spin columns and spinning at 1000 rpm for 1 minute. The relative intensity at 525 nm of the detached fluorescence from the wild type genomic DNA sample (Figure 5A) was eight times higher than that of the mutated genomic sample (Figure 5B), allowing the wild type genomic sample to be distinguished from the mutated genomic sample. EXAMPLE 4<br>
This Example was similar to Example 3 except for the following details. The mutant genomic DNA sample (SEQ ID NO: 5) had a 2 base change from the wild type genomic DNA at amino acid position 344, from CTG to GCG. Each sample was initially heated at 100°C for 15 minutes. The buffer was preheated 50°C and each sample was allowed to hybridize for 2 hours. The components of each sample were separated using G50 spin columns and spinning at 1200 rpm for 1 minute. The relative intensity at 525 nm of the detected fluorescence from the wild type genomic DNA sample was seven times higher than that from the mutated genomic sample, allowing the wild type genomic sample (Figure 6A) to be distinguished from the mutated genomic sample (Figure 6B).<br>
This Example was similar to the other Examples except for the following details. The mutant genomic DNA sample (SEQ ID NO:6) had a 3 base change from the wild type genomic DNA at amino acid position 344, from CTG to TAC. Each sample was initially heated at 95°C for 30 minutes. The buffer was preheated to 25°C and each sample was allowed to hybridize<br>
for 60 minutes. The components of each sample were separated using G50 spin columns and spinning at 750 rpm for 2 minutes. As shown in Figures 7A and 7B, the relative intensity at 525 nm of the detected fluorescence from the wild type genomic DNA sample (Figure 7A) was ten times higher than that from the mutated genomic sample (Figure 7B) , allowing the wild type genomic sample to be distinguished from the mutated genomic sample.<br>
This Example was similar to Example 1 except for the following details. 100 pmol of the 12 bp PNA of Example 1 labeled with fluorescein (PerSeptive Biosystems) , 25 pmol of 150 bp PCR amplified dsDNA, including wild type (CTG) (SEQ ID N0:1) and one base mutated DNA (CAG, SEQ ID NO:2, and CGG, SEQ ID NO:3) and 115 microliters buffer (0.5xTBE working solution (0.0225M Trisborate/0.0005M EDTA), pH 6.5) were mixed at room temperature. The mixture was then heated for 30 minutes at 95°C and hybridized at room temperature for about 1 hour.<br>
Two metal wires serving as electrodes were placed into each sample. Fluorescent intensity was monitored at 525 nm while a voltage of 5 volts at a current of 10mA was switched on and off, with an electrode separation of about 7 or 8 mm. When the voltage was applied, the DNA and DNA-PNA migrated to the anode, since the DNA is negative charged, while the unbound PNA did not migrate, since it is neutral.<br>
Figure 8A shows the fluorescent spectrum of the wild type DNA. After hybridizing at 25°C for 1 hour, the solution was transferred into a cuvette. The cuvette was placed in a fluorescence spectrometer and then two electrodes were put into the solution for detecting the change of fluorescent intensity at a wavelength of 525 nm with the applied electric field. The fluorescent intensity at 525 nm decreased with the application of the voltage and increased with the removal of the voltage.     Figures 8B and 8C show the fluorescent<br>
spectra of one base pair mismatched DNA (SEQ ID NO: 2 and SEQ ID NO:3, respectively). After hybridizing at 25°C for 1 hour, the fluorescent spectra of mutant DNA showed a difference from the spectrum of wild type DNA. When voltage was applied, the intensity quickly increased, and when voltage was removed, the intensity decreased. EXAMPLE 7<br>
This Example was similar to Example 1 except for the following details. A 375 bp fragment of p53 wild type DNA (SEQ ID NO:7), a corresponding 375 bp mutant type DNA fragment (SEQ ID NO:8), a corresponding 375 bp mutant type DNA fragment (SEQ ID NO: 9) and a 633 bp DNA fragment (SEQ ID NO: 10) were obtained by PCR. The 375 bp wild type DNA contained a target sequence of DNA complementary to the 12 base PNA from Example 1. The 633 bp DNA lacked such a target sequence and was used as a negative control in this example. SEQ ID NO:8 was identical to the wild type fragment (SEQ ID NO:7) except for a point mutation at amino acid position 344 at which the DNA wild type sequence CTG was changed to CAG. SEQ ID NO:9 was identical to the wild type fragment (SEQ ID NO:7) except for a point mutation at ammo acid position 344 at which the DNA wild type sequence CTG was changed to CGG.<br>
Each sample comprised 50 pmol of the respective DNA fragment, 200 pmol of PNA probe and 130 ml of buffer (0.5xTBE at pH 6.5) . Each sample was heated at 95°C for about 30 minutes. Each sample was then allowed to hybridize for 1 hour under ambient conditions (25°C). Before fluorescence measurement, unbound probe was filtered from solution using a G50 column.<br>
Each sample was placed in a fluorescence spectrometer for detection of the fluorescent probe attached to the target genomic DNA. As with the previous examples, the relative fluorescence intensity at 525 run was highest for samples containing DNA fragments completely complementary to the probe,  thus  allowing  the  samples  containing mutant  DNA<br>
fragments (Figures 9B-9C) to be distinguished from the sample containing wild type DNA fragments (Figure 9A).  As expected, the  negative  control  DNA  fragment  showed  the  lowest fluorescent intensity of all (Figure 9D). EXAMPLE 8<br>
A clinical sample is obtained from an individual suspected of suffering from a disease which is known to be caused by a single nucleotide mutation within a known segment of the human genome. A fragment of this gene segment is amplified by PCR using a primer capable of amplifying the fragment regardless of whether it contains wild type DNA or mutant type DNA.<br>
A PNA probe of 12 bases is provided which is complementary to a portion of the mutant type DNA fragment containing the mutant nucleotide, wherein the mutant nucleotide has a base complementary to a centrally located base of the probe. The probe is marked with fluorescein at its 5' end.<br>
50 pmol of the DNA obtained from PCR and 300 pmol of PNA probe are added to 100 ml of buffer solution. The sample is heated at 95°C for 30 minutes. The sample is then added to buffer preheated to 30°C and allowed to hybridize for 1 hour.<br>
The components of the sample are separated by G50 spin columns by spinning at 600 rpm for 2 minutes. The unhybridized PNA is filtered with a column, and the liquid medium containing hybrids and DNA passes through the column and is collected in a cuvette.<br>
The cuvette is placed in a fluorescence spectrometer for detection of the fluorescent probe attached to the DNA. If the sample's fluorescent intensity at 525 nm exceeds a predetermined intensity value, then the mutant type DNA has been detected in the individual. If the sample's fluorescent intensity at 525 nm does not exceed a predetermined intensity value, then the mutant type DNA has not been detected in the individual.  The predetermined intensity value is previously<br>
determined by calibrating the spectrometer using negative and positive control samples. EXAMPLE 9~<br>
This Example is similar to Example 8, except for the following details. Two unlabeled DNA (one is wild type (SEQ ID NO:l), another is mutated type (SEQ ID N0:3)) need to be identified. A probe, which is complementary to the mutated DNA (SEQ ID N0:3), with the following sequence 5'CAT TCC GCT CTC (synthesized by PerSeptive Biosystems) was used to identify the two unknown DNA.<br>
Each sample comprising 100 pmol probe, 25 pmol DNA (WT DNA or mutated DNA) and 115 µl buffer (O.SxTBE, pH 6.5) was heated at 95°C for 30 minutes. Each sample was then allowed to hybridize for 1 hour at 25°C. Before fluorescent measurement, each sample was transferred into a G50 column and centrifuged at 750 rpm for 2 minutes. The hybrid flowed through the column and was collected in a cuvette, while the unhybridized probe was filtered and retained in the column. The cuvette was placed into a fluorescent spectrometer for<br>
As shown in Figures 10A and 10B, the relative fluorescent intensity at 525 nm shows a large difference in spectra. The sample with a stronger fluorescent signal (Figure 10A) is identified as the mutant DNA (SEQ ID NO:3). The other sample with weaker fluorescent intensity is wild type DNA (SEQ ID NO:l). EXAMPLE 10<br>
This Example is similar to Example 8 except for the following details. Rather than using a single probe as in Example 8, two probes are used in this example. the first probe is the probe of Example 8, which is completely complementary to a portion of the mutant type DNA fragment. The second probe is identical to the first probe, except that it has the wild type nucleobase sequence instead of the mutant type nucleobase sequence.   A first portion of the<br>
sample is probed in accordance with Example 8 using the first probe. A second portion equivalent in size to the first portion is similarly probed using the second probe. After denaturation, hybridization and separation, the fluorescence of each portion is measured. If the fluorescent intensity of the first portion exceeds that of the second portion, then the mutant type DNA has been detected in the individual. If the fluorescent intensity of the second portion exceeds that of the first portion, then the mutant type DNA has not been detected in the individual.<br>
The method of this example could be adapted for use with a diagnostic device comprising a solid support having the two contrasting type of PNA probes bonded to different areas of the support. The sample is not<br>
divided, but rather is uniformly dispersed onto the support so as to contact both probe types equally. The fluorescent intensity of the two areas of the device could be compared in the same way that the two separate samples are in Example 9. EXAMPLE 11<br>
Biotinylated double stranded DNA (SEQ ID N0:1 and SEQ ID NO:3) was amplified by PCR. Single stranded DNA was prepared using Dynabeads (from DYNAL company) as follows. 25 µl of 2x binding and washing buffer (from M-280 streptativin kit) and 25 µl (25 pmol) of biotinylated DNA (SEQ N0:1 and NO:2) were added into 2mg of the prewashed dynabeads of M-280 streptativin. The mixture was incubated at room temperature for 15 minutes, keeping the beads suspended. The mixture was then placed onto a magnetic particle concentrator (MPC, Dynal, Inc.) at room temperature for 2 minutes, followed by removal of the supernatant with a pipette. 20 µl of freshly made 0. IN NaOH was added and the mixture was kept at room temperature for 5 minutes. The dynabeads were collected with the immobilized biotinylated strand on the side of the tube by using the MPC, and the NaOH supernatant was transferred with the non-biotinylated single stranded DNA to a clean<br>
tube.  The NaOH supernatant was neutralized with 2.5 µl of freshly made 0.2N HC1.<br>
200 pmol of probe and 150 µl of 0. 5xTBE buffer were added into the supernatant containing a single stranded DNA. The mixture was heated at 94°C for 5 minutes and then incubated at room temperature for 1 hour. After separation by G50 column at 650 rpm for 2 minutes, the unbound PNA was filtered in the column and the hybrid passed through the column and collected in a cuvette. The sample was placed in a fluorescent spectrometer for fluorescence detection. As shown in Figures 11A and 11B, the relative fluorescent intensity at 525 nm of single stranded wild type DNA sample (Figure 11A) was five times higher than that of mutant single stranded DNA (Figure 11B) . EXAMPLE 12<br>
Each sample comprising 400 pmol probe, 100 pmol DNA (SEQ ID NO.:l) and 100 µl buffer (O.5xTBE, pH 6.5) was heated at 95°C for 30 minutes. Each sample was then allowed to hybridize for 1 hour at 25°C. Before fluorescent measurement, each sample was transferred into a G50 column and centrifuged at 720 rpm for 2 minutes. The hybrids flowed through the column and were collected in a cuvette, while the unhybridized probe was filtered and retained in the column. The cuvette was placed into a fluorescent spectrometer for fluorescence measurement.<br>
As shown in Figures 12A and 12B, the relative fluorescence intensity at 525 nm shows a large difference in the spectra. The spectrum with a stronger fluorescent signal (Figure 12A) shows a perfect matched probe hybridization with WT DNA (SEQ ID NO:l). The other spectrum with weaker fluorescent intensity (Figure 12B) shows a one bp mismatched probe hybridization with WT DNA (SEQ ID NO:l). EXAMPLE 13<br>
Two PNA probes having the same dye (fluorescein) hybridized with two perfectly matched targets in different strands.<br>
Example 13a: One probe perfectly matching one target sequence.<br>
A 150 bp fragment of genomic DNA from mutated type p53 DNA (SEQ ID NO: 11) was amplified by PCR and purified using the QIAquick PCR Purification Kit (QIAGEN Inc., Chatsworth, CA, USA) . The mutated fragment was identical to the wild type fragment (SEQ ID NO:l) except for a two base mutation at amino acid position 340 (i.e., bases 88-90) at which the DNA wild type sequence ATG was changed to GAG.<br>
A 12-mer _PNA probe (Probe No. 1) was synthesized by PerSeptive Biosystems, Inc. of Framingham, MA, USA. The probe, having the structure:<br>
5' H-Fluo-0-CAT TCA GCT CTC Lys-CONH2/ was designed to be complementary to a 12 nucleotide segment of the 150 bp p53 ME DNA, starting at base pair position 95 and ending at base pair position 106 (see SEQ ID NO:11).<br>
0.5xTBE solution (0.0225 M Tris-borate/0.0005 M EDTA, pH 6.5) was used as hybridization buffer. 5 pmol of DNA (SEQ ID NO:ll) was added into 115 µl 0.5XTBE buffer. Then 2.5 pmol of Probe No. 1 was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column (purchased from Pharmacia Biotech, AB, Uppsala, Sweden). The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 13A shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO:ll) hybridized with 2.5 pmol of Probe No.1. A peak appears at 525 nm.<br>
Example 13b: One probe perfectly matching one target<br>
A  12-mer  PNA  probe  (Probe  No.  2)  synthesized  by PerSeptive Biosystems, Inc., having the structure: 5' H-Fluo-O-TCG AGG AGT TCC Lys-CONH2,<br>
was prepared to be completely complementary to a target in the strand of ME DNA complementary to the strand targeted in Example 13a, starting at base pair position 83 and ending at base pair position 94 (see SEQ ID NO:11).<br>
5 pmol of DNA (SEQ ID NO: 11) was added into 115 ul 0.5xTBE buffer. Then the probe was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 13B shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO:11) hybridized with 5 pmol of Probe No. 2.<br>
Example 13c: Two probes perfectly matching two target sequences in different strands.<br>
5 pmol of DNA (SEQ ID NO: 11) was added into 115 ul 0.5xTBE buffer. Then 2.5 pmol of Probe No. 1 and 5 pmol of Probe No. 2 were added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 13C shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 11) hybridized with 2.5 pmol of Probe No. 1 and 5 pmol of Probe No. 2. As shown in Fig. 13C, the fluorescent intensity at 525 nm is approximately about the sum of the intensities at 525 nm shown in Figs. 13A and 13B.<br>
Two PNA probes having different dyes iamine) hybridxzed with one perfectly matched target.<br><br>
A 150 bp fragment of genomic DNA from wild type p53 DNA (SEQ ID NO:l) was amplified by PCR and purified by using the QIAquick PCR Purification Kit.<br>
A 12-mer rhodamine labeled PNA probe (Probe No. 3) synthesized by PerSeptive Biosystems, Inc., having the structure:<br>
5' H-Rho-O-CAT TCA GCT CTC Lys-CONH2, was designed to be complementary to a 12 nucleotide segment of the 150 bp p53 DNA, starting at base pair position 95 and ending at base pair position 106 (see SEQ ID NOS: 1, 11, and 12) .<br>
20 pmol of DNA (SEQ ID NO:l) was added into 50 ul 0.5xTBE buffer. Then 20 pmol of the Probe No. 3 was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solutions with the hybrids of PNA-DNA were placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 14A shows a fluorescence spectrum of DNA hybridized with the rhodamine labeled PNA probe. The emission peak in the fluorescence spectrum emerges at about 580 nm.<br>
Example 14b<br>
20 pmol of DNA (SEQ ID NO:l) was added into 50 ul O.SxTBE buffer. Then 15 pmol of rhodamine labeled probe (Probe No. 3) and 5 pmol of fluorescein labeled probe (Probe No. 1) were added into the solution. The sample was heated at 95°C for 10 minutes and then the sample was hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 14B shows a fluorescence spectrum of DNA hybridized with the mixed probes of rhodamine labeled PNA probe (Probe No. 3) and fluorescein labeled PNA probe(Probe No. 1). A shoulder emerged at 580 nm due at least in part to rhodamine labeled probe hybridized to the target sequence and a peak emerged at 525 nm partly contributed by the fluorescein labeled probe hybridized to the target sequence. EXAMPLE 15<br>
Two PNA probes having different dyes (fluorescein and rhodamine) hybridized with two perfectly matched targets on two strands.<br>
Example 15a<br>
20 pmol of DNA (SEQ ID NO: 11) was added into 50 µl 0.5xTBE buffer. Then 20 pmol of rhodamine labeled probe (Probe No. 3) was added into the solution. The sample was heated at 95°C for 10 minutes and then the sample was hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 15A shows a fluorescence spectrum of DNA hybridized with the rhodamine labeled PNA probe (Probe No. 3). A broad peak emerges at 580 nm contributed by hybridization complexes labeled with rhodamine.<br>
Example 15b<br>
20 pmol of DNA (SEQ ID NO: 11) was added into 50 pi 0.5xTBE buffer. Then 20 pmol of rhodamine labeled probe (Probe No. 3) and 5 pmol of fluorescein labeled probe (Probe No. 2) were added into the solution. The sample was heated at 95°C for 10 minutes and then the sample was hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column.  The solution<br>
with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 15B shows a fluorescence spectrum of DNA hybridized with Probe Nos. 2 and 3. A broad peak emerges at 580 nm contributed by the rhodamine labeled probe and a peak rises at 525 nm contributed by the fluorescein labeled probe.<br>
A PCR amplified and purified 633 bp DNA (SEQ ID NO: 10) lacking a target sequence was used as a negative control for probes.<br>
5 pmol DNA (SEQ ID NO:10) was added into 115 µl 0.5xTBE buffer and then Probe Nos. 1-3 were added into the solution. Each sample was heated at 95°C for 10 minutes and then hybridized at 25°C for 30 minutes.<br>
Before the fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution  was  placed  into  a  cuvette  and  subjected  to<br>
Fig. 16A shows a fluorescence spectrum of probe alone (i.e., no DNA was added to the sample, which included 15 pmol of the probes — 5 pmol of each of Probe Nos. 1, 2 and 3).<br>
Fig. 16B shows a fluorescence spectrum of 5 pmol of Probe No. 1 and negative control DNA (SEQ ID NO:10).<br>
Fig. 16C shows a fluorescence spectrum of 5 pmol of Probe No. 2 and negative control DNA (SEQ ID NO:10).<br>
Fig. 16D shows a fluorescence spectrum of 5 pmol of Probe No. 3 and negative control DNA (SEQ ID NO:10).<br>
All  the  spectra  show  the  fluorescent  intensity  at background level at 525 nm and 580 nm. Example 17<br>
Two PNA probes having the same dye (fluorescein) hybridized with a one base pair mismatched target sequence and a perfectly matched target sequence.<br>
Example 17a<br>
A 150 bp fragment of genomic DNA from p53 DNA gene MQ (SEQ ID NO:12) was amplified by PCR and purified by using the QIAguick PCR Purification Kit. The mutated fragment was identical to the ME DNA {SEQ ID NO:'ll) fragment except for a one base mutation at amino acid position 340 (bases 88-90) at which the DNA sequence CTC was changed to GTC.<br>
5 pmol of DNA (SEQ ID NO: 12) was added into 115 µl 0.5xTBE buffer. Then 5 pmol of Probe No. 2, which has a one base pair mismatch to the target sequence, was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 17A shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 12) hybridized with 5 pmol of Probe No. 2. The spectrum shows the fluorescent intensity at background<br>
5 pmol of DNA (SEQ ID NO: 12) was added into 115 µl 0.5xTBE buffer. Then Probe No. 1 was added into the solution. This probe was perfectly matched with the DNA target. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and<br>
Fig. 17B gives the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 12) hybridized with S pmol of Probe No. 1. The positive signal at 525 nm is clearly shown in Fig. 17B.<br><br>
Example 17c<br>
5 pmol of DNA (SEQ ID NO: 12) was added into 115 µl 0.5xTBE buffer. Then 5 pmol Probe No. 1 and 5 pmol Probe No. 2 were added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 17C shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO:12) hybridized with 5 pmol of Probe No. 1 and 5 pmol of Probe No. 2.  The fluorescent intensity at 525 nm is approximately equal to the intensity shown in Fig. 17B. Example 18<br>
Two PNA probes having different dyes (fluorescein and rhodamine) hybridized with a one bp mismatched target and a perfectly matched target.<br>
5 pmol of DNA (SEQ ID NO: 12) was added into 115 µl O.5xTBE buffer. Then 5 pmol of rhodamine labeled probe (Probe No. 3) was added into the solution. This probe was perfectly matched to the target sequence. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 18A shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 12) hybridized with 5 pmol of Probe No. 3. The positive signal at 580 nm is clearly shown in Fig. 18A.<br>
5 pmol of DNA (SEQ ID NO: 12) was added into 115 µl 0.5xTBE buffer. Then 5 pmol of Probe No. 3, which was perfectly matched  to the target sequence, and 5 pmol of<br>
Probe No. 2, which has one base pair mismatched to the target sequence, were added to the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 18B shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO:12) hybridized with 5 pmol of Probe No. 3 and 5 pmol of Probe No. 2. The fluorescent intensity at 525 nm is approximately equal to the intensity shown in Fig. 18A. The difference between the intensity at 580 nm and at 525 nm is approximately equal to that shown in Fig. 18A. EXAMPLE 19<br>
Three PNA probes having the same dye (fluorescein) hybridized with two one base pair mismatched target sequences and one perfectly matched target.<br>
Example 19a<br>
A 150 bp fragment of genomic DNA from p53 DNA gene QR (SEQ ID NO:13) was amplified by PCR and purified by using the QIAquick PCR Purification Kit. The mutated fragment was identical to the ME DNA (SEQ ID NO: 11) fragment except for one base mutation at amino acid position 340 (bases 85-87) at which the DNA sequence CTC was changed to GTC and one base mismatch at amino acid position 344 (bases 100-102) at which the DNA sequence CTG was changed to CGG.<br>
5 pmol of DNA (SEQ ID NO: 13) was added into 115 µl 0.5xTBE buffer. Then 5 pmol of Probe No. 1, which has a one base pair mismatch to the target sequence at base pair position 101, was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column.  The solution<br>
with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 19A shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 13) hybridized with 5 pmol of Probe No. 1. The spectrum shows the fluorescent intensity at 525 nm at background level.<br>
5 pmol of DNA (SEQ ID NO: 13) was added into 115 ul 0.5xTBE buffer. Then Probe No. 2, which has a one base pair mismatch to the target sequence at base pair position 85, was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 19B shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 13) hybridized with 5 pmol of Probe No. 2. The spectrum shows the fluorescent intensity at 525 nm at background level.<br>
5 pmol of DNA (SEQ ID NO: 13) was added into 115 ul 0.5xTBE buffer. A 12-mer fluorescein labeled PNA probe (Probe No. 4) synthesized by PerSeptive Biosystems, Inc., having the structure:<br>
5'H-Fluo-O-CAT TCC GCT CTC Lys-CONH2, was designed to be completely complementary to a 12 nucleotide segment of SEQ ID NO:13, starting at base pair position 95 and ending at base pair position 106. This probe was added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°c for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and<br><br>
Fig. 19C shows the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO: 13) hybridized with 5 pmol of Probe No. 4. The positive signal at 525 ma is clearly shown in Fig. 19C.<br>
Example 19d<br>
5 pmol of DNA (SEQ ID NO: 13) was added into 115 µl 0.5xTBE buffer. Then 5 pmol of Probe No. 1 and 5 pmol of Probe No. 2 were added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 19D gives the fluorescence spectrum of 5 pmol of DNA (SEQ ID NO:13) hybridized with 5 pmol of Probe No. 1 and 5 pmol of Probe No. 2. The spectrum shows the fluorescent intensity at 525 nm at background level.<br>
Example 19e<br>
5 pmol of DNA (SEQ ID NO:13) was added to 115 pi 0.5xTBE buffer. Then 5 pmol of Probe No. 1, 5 pmol of Probe No. 2 and 5 pmol of Probe No. 4 were added into the solution. The sample was heated at 95°C for 10 minutes and hybridized at 25°C for 30 minutes.<br>
Before fluorescence measurement, the unbound probe was filtered from the solution by G50 spin column. The solution with the hybrids of PNA-DNA was placed into a cuvette and subjected to fluorescence measurement.<br>
Fig. 19E gives the fluorescence spectrum of QR DNA (SEQ ID NO: 13) hybridized with PNA Probe Nos. 1, 2 and 4. The fluorescent intensity at 525 nm is approximately equal to the intensity shown in Fxg. 19C.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to one skilled in the art that various changes and<br>
modifications can be made therein without departing from the spirit and scope thereof.<br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	An apparatus for detecting at least one single stranded or double<br>
stranded nucleotide target sequence in a liquid medium, said apparatus<br>
comprising:<br>
a container adapted to contain said liquid medium and at least one PNA probe having a fluorescent marker, capable of forming a hybridization complex with said at least one said target sequence,;<br>
means provided in sequence with the said container for separating unhybridized PNA probe from said hybridization complex to form a test medium;<br>
a laser source provided in sequence with the said means for irradiating said test medium with a laser beam having a wavelength which excites said fluorescent marker and causes said fluorescent marker to emit fluorescent light;<br>
a fluorescence detector for measuring an intensity of said emitted fluorescent light emitted by the liquid medium in the said container wherein said measured intensity is inversely proportional to a number of base mismatches between said at least one target sequence and said PNA probe, over a range inclusive of 0 base mismatches through at least 3 base mismatches<br>
an output device for receiving and comparing said measured intensity with a reference intensity received from the detector to detect whether said liquid medium contains said at least one target sequence,<br>
wherein said PNA probe, said nucleotide sequence and said hybridization complex are not bound to a solid support.<br>
2.	. The apparatus for detecting at least one nucleotide sequence as<br>
claimed in claim 1, wherein means for separation are filters, centrifuge,<br>
precipitator or free solution electrophoresis device.<br>
3.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said laser beam has a wavelength of about 450 to about 530 nm.<br>
4.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said emitted fluorescent light is measured in a range of 400 to 1000 nm.<br>
5.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 13, wherein the first and second markers are selected from the group consisting of fluorescein and rhodamine.<br>
6.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said emitted fluorescent light has a wavelength longer than said laser beam wavelength.<br>
7.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, comprising heating means for denaturing said nucleotide sequence in said liquid medium prior to said detecting step, at a temperature of about 85 °C to about 100 °C for about 30 seconds to about 5 hours.<br>
8.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein a total volume of said test medium is no more than about 5 milliliters.<br><br>
9.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 8, wherein said total volume is no more than about 10 microliters.<br>
10.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said detected fluorescent light consists essentially of said fluorescent light emitted by said marker.<br>
11.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said PNA probe is a one-base mismatch of a segment of said target sequence.<br>
12.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said PNA probe is a two-base mismatch of a segment of said target sequence.<br>
13.	The apparatus for detecting at least one nucleotide sequence as claimed in claim 1, wherein said PNA probe is a three base mismatch of a segment of said target sequence.<br>
14.	An apparatus as claimed in claim 1 for detecting at least one single stranded or double stranded nucleotide target sequence in a liquid medium, substantially as hereinbefore described with reference to the accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">486-del-1998-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">486-del-1998-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">486-del-1998-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">486-del-1998-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">486-del-1998-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWZvcm0tNi5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDg2LWRlbC0xOTk4LXBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">486-del-1998-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="188173-a-process-for-preparing-a-protein-isolate.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="188177-a-process-for-the-production-of-a-beta-lactam.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>188176</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>486/DEL/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2002</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Aug-2002</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Apr-2003</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Feb-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LORNE PARK RESEARCH, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>20 QUEEN STREET WEST, TORONTO, ONTARIO, CANADA M5H, 3R3.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NIE, EILEEN XIAO-FENG</td>
											<td>4, DALMENY ROAD, THORNHILL, ONTARIO, CANADA L3T, 1L9.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WU, YUAN MIN</td>
											<td>4, DALMENY ROAD, THORNHILL, ONTARIO, CANADA L3T, 1L9.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>B01J 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>08/870,370</td>
									<td>1997-06-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>08/807,901</td>
									<td>1997-02-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/188176-an-apparatus-for-detecting-nucleotide-target-sequence-in-a-liquid-medium by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:47:55 GMT -->
</html>
