<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/190892-a-method-for-obtaining-a-hydrolysate-from-a-proteinaceous-substrate by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:44:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 190892:A METHOD FOR OBTAINING A HYDROLYSATE FROM A PROTEINACEOUS SUBSTRATE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR OBTAINING A HYDROLYSATE FROM A PROTEINACEOUS SUBSTRATE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a method for obtaining a hydrolysate from a proteinaceous substrate. The substrate is subjected to the action of a polypeptide having dipeptidyl aminopeptidase activity and endopeptidase action. Polypeptides having dipeptidyl aminopeptidase activity is selected from (a) a polypeptide having amino acid sequence of at least 70% with 17 to 771 of SEC ID No.2 (b) a polypeptide encoded by a nucleic acid sequence, (c) an allelic variant of (a) or (b), a fragment of (a), (b), or (c) having dipeptidyl aminopeptidase activity and a polypeptide having dipeptidyl aminopeptidase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The invention relates to a method for obtaining protein hydrolysates useful as flavour improving agents.<br>
Various food products, e.g., soups, sauces and seasonings, contain flavouring agents obtained by hydrolysis of proteinaceous materials. This hydrolysis is conventionally accomplished using strong hydrochloric acid, followed . by neutralization with sodium hydroxide. However, such chemical hydrolysis leads to severe degradation of the amino acids obtained during the hydrolysis, and also to hazardous by products formed in the course of this chemical reaction. Increasing concern over the use of flavoring agents obtained by chemical hydrolysis has led to the development of enzymatic hydrolysis processes.<br><br>
Enzymatic hydrolysis processes of proteinaceous materials aim at obtaining a high degree of hydrolysis (DH), and this is usually attained using a complex of unspecific acting proteolytic enzymes (i.e., unspecific-acting endo- and exo-peptidases). For example, WO 94/25580 describes a method for hydrolyzing proteins by use of an unspecific acting enzyme preparation obtained from Aspergillus oryzae. Specific acting proteolytic enzymes have not been used for this purpose because such enzymes only lead to an inadequate degree of hydrolysis.<br>
Polypeptides having dipeptidyl aminopeptidase activity catalyze the removal of dipeptides from the N-terminus of peptides, polypeptides, and proteins. Such polypeptides are classified under the Enzyme Classification Number E.G. 3.4.14.- of the International Union of Biochemistry and Molecular Biology.<br>
Beauvais et al. (1997, Journal of Biological Chemistry 272: 6238-6244) disclose a dipeptidyl-peptidase from Aspergillus fumigatus which has a molecular weight of 88 kDa by SDS-PAGE and a substrate specificity limited to the hydrolysis of X-Ala, His-Ser, and Ser-Tyr dipeptides at a neutral pH optimum. Tachi et al. (1992, Phytochemistry 31: 3707-3709) disclose an X-prolyl dipeptidyl aminopeptidase from Aspergillus oryzae which has a molecular weight of 145 kDa by SDS-PAGE and a substrate specificity toward the peptide bond at the carboxyl site of a proline residue in the penultimate position of N-terminal free dipeptides and amides at a neutral pH optimum.<br>
The production of protein hydrolysates with desirable organoleptic properties and high degrees of hydrolysis generally requires the use of a mixture of peptidase activities. It would be desirable to provide a single component peptidase enzyme which has activity useful for improving the organoleptic properties and degree of hydrolysis of protein hydrolysates used in food products either alone or in combination with other enzymes.<br>
It is an object of the present invention to provide improved polypeptides having dipeptidyl aminopeptidase activity as well as methods for obtaining protein hydrolysates with desirable organoleptic qualities and high degrees of hydrolysis.<br><br>
Summary of the Invention<br>
The present invention relates to isolated polypeptides having dipeptidyl aminopeptidase activity selected from the group consisting of:<br>
(a)	a polypeptide having an amino acid sequence which has at least 50% identity with the amino acid sequence of SEQ ID N0:2;<br>
(b)	a polypeptide encoded by a nucleic acid sequence which hybridizes under medium stringency conditions with (i) the nucleic acid sequence of SEQ ID N0:1, (ii) its complementary strand, or (iii) a subsequence thereof;<br>
(c)	an allelic variant of (a) or (b); and<br>
(d)	a fragment of (a), (b), or (c), wherein the fragment has dipeptidyl aminopeptidase activity; and<br>
(e)	a polypeptide having dipeptidyl aminopeptidase activity with physicochemical<br>
properties of (i) a pH optimum in the range of from about pH 4.4 to about pH 9.8 determined<br>
after incubation for 5 minutes at ambient temperature in the presence of<br>
Ala-Pro-para-nitroanilide; (ii) a temperature stability of 90% or more, relative to initial activity,<br>
at pH 7.5 determined after incubation for 20 minutes at 65Â°C in the absence of substrate; and<br>
(iii) an activity towards Xaa-Pro-para-nitroanilide or Xaa-Ala-para-nitroanilide wherein Xaa is<br>
selected from the group consisting of Ala, Arg, Asp, Gly, and Val.<br>
The present invention also relates to isolated nucleic acid sequences encoding the polypeptides and to nucleic acid constructs, vectors, and host cells comprising the nucleic acid sequences as well as methods for producing the polypeptides.<br>
The present invention also relates to methods for obtaining hydrolysates from proteinaceous substrates which comprise subjecting the proteinaceous material to a polypeptide with dipeptidyl aminopeptidase activity alone or in combination with an endopeptidase, and to hydrolysates obtained from the method.<br>
The present invention also relates to methods for obtaining from a proteinaceous substrate a hydrolysate enriched in free glutamic acid and/or peptide bound glutamic acid residues, which methods comprise subjecting the substrate to a deamidation process and to the<br><br>
action of a polypeptide having dipeptidyl aminopeptidase activity.<br>
The present inventin further relates to flavor-improving compositions comprising a polypeptide with dipeptidyl aminopeptidase activity.<br>
In a final aspect, the methods of the invention may be used in food related applications to improve flavor, such as baking. Alternatively, flavor improvement in foods may be achieved by the addition of hydroly sates obtained by the methods of the invention.<br>
Accordingly the present invention provides a method for obtaining a hydrolysate from a proteinaceous substrate such as herein described which comprises the steps of:<br>
(i) subjecting in a known manner the substrate to the action of a polypeptide having   dipeptidyl   aminopeptidase activity; and<br>
(ii) subjecting in a known manner the substrate to the action of an endopeptidase;<br>
wherein    the   polypeptide   having    dipeptidyl aminopeptidase activity is selected from the group consisting of:<br>
(a) a polypeptide having an amino acid sequence which has at least 70% identity with amino acids 17 to 771 of SEQ ID No.2;<br><br>
(b)	a polypeptide which is encoded by a nucleic acid sequence which<br>
hybridizes under medium stringency conditions with (i) nucleotides 49 to 2396<br>
ofSEQ ID No. 1, (ii) the cDNA sequence contained in nucleotides 49 to 2396<br>
of SEQ ID NO.l, (iii) a subsequence of (i) or (ii) of at least 100 nucleotides,<br>
or (iv) a complementary strand of (i), (ii), or (iii); wherein medium<br>
stringency conditions are   defined as prehybridization   and hybridization at<br>
42Â°C in 5X SSPE. 0.3% SDS, 200 Âµg/ml sheared and denatured salmon sperm DNA, and 35% formamide;<br>
(c)	an allelic variant of (a) or (b);<br>
(d)	a fragment of (a), (b), or (c), wherein the fragment has dipeptidyl aminopeptidase activity; and<br>
(e)	a polypeptide having dipeptidyl aminopeptidase activity with physicochemical properties of (i) a pH optimum in the range of from pH 4.4 to pH 9.8 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide; (ii) a temperature stability of 90% or more, relative to initial activity, at pH 7.5 determined after incubation for 20 minutes at 65oC in the absence of substrate; and (iii) an activity towards Xaa-Pro-para-nitroanilide or Xaa-Ala-para-nitroanilide wherein Xaa is selected from the group consisting of Ala, Arg, Asp, Gly and Val.<br>
With reference to the accompanying drawings:<br>
Figures lA and IB show the nucleic acid sequence and the deduced acid sequence of an Aspergillus oryzae ATCC 20386 dipeptidyl aminopeptidase (SEQ ID NOS.l and 2 respectively).<br>
Figure 2 shows a restriction map of pDM181.<br>
Figure 3 shows a restriction map of pMWR54.<br><br>
Detailed Description of the Invention<br>
Polypeptides Having Dipeptidyl Aminopeptidase Activity<br>
The term "dipeptidyl aminopeptidase activity" is defined herein as a peptidase activity which cleaves dipeptides from the N-terminal end of a peptide, polypeptide, or protein sequence. Defined in a general manner, the dipeptidyl aminopeptidase is capable of cleaving the dipeptide XY from the unsubstituted N-terminal amino group of a peptide, polypeptide, or protein, wherein X or Y may represent any amino acid residue selected from the group consisting of Ala, Arg, Asn, Asp, Cys, Gin, Glu, Gly, His, He, Leu, Lys, Met, Phe, Pro, Ser, Thr, Trp, Tyr, and Val, but at least Ala, Arg, Asp, Gly, and/or Val. All of X and Y may be different or identical. It will be understood that the isolated polypeptides having dipeptidyl aminopeptidase activity of the present invention may be unspecific as to the amino acid sequence of the dipeptide to be cleaved.<br><br>
In a first embodiment, the present invention relates to isolated polypeptides having an amino acid sequence which has a degree of identity to the amino acid sequence of SEQ ID N0:2 of at least about 50%, preferably at least about 60%, preferably at least about 70%, more preferably at least about 80%, even more preferably at least about 90%, most preferably at least about 95 %, and even most preferably at least about 97 %, which have dipeptidyl aminopeptidase activity (hereinafter "homologous polypeptides"). In a preferred embodiment, the homologous polypeptides have an amino acid sequence which differs by five amino acids, preferably by four amino acids, more preferably by three amino acids, even more preferably by two amino acids, and most preferably by one amino acid from the amino acid sequence of SEQ ID N0:2. For purposes of the present invention, the degree of identity between two amino acid sequences is determined by the Clustal method (Higgins, 1989, CABIOS 5: 151-153) with an identity table, a gap penalty of 10, and a gap length penalty of 10.<br>
Preferably, the polypeptides of the present invention comprise the amino acid sequence of SEQ ID N0:2 or an allelic variant; and a fragment thereof, wherein the fragment has dipeptidyl aminopeptidase activity. In a more preferred embodiment, the polypeptides of the present invention comprise the amino acid sequence of SEQ ID NO:2. In another preferred embodiment, the polypeptide of the present invention has the amino acid sequence of SEQ ID N0:2 or a fragment thereof, wherein the fragment has dipeptidyl aminopeptidase activity. A fragment of SEQ ID N0:2 is a polypeptide having one or more amino acids deleted from the amino and/or carboxy terminus of this amino acid sequence. In a most preferred embodiment, the polypeptide has the amino acid sequence of SEQ ID N0:2.<br>
Preferably, a fragment contains at least 455 amino acid residues, more preferably at least 555 amino acid residues, and most preferably at least 655 amino acid residues.<br>
An allelic variant denotes any of two or more alternative forms of a gene occupying the same chomosomal locus. Allelic variation arises naturally through mutation, and may result in phenotypic polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. The term allelic variant is also used to denote a protein encoded by an allelic variant of a gene.<br><br>
The amino acid sequences of the homologous polypeptides may differ from the amino acid sequence of SEQ ID N0:2 by an insertion or deletion of one or more amino acid residues - and/or the substitution of one or more amino acid residues by different amino acid residues. Preferably, amino acid changes are of a minor nature, that is conservative amino acid substitutions that do not significantly affect the folding and/or activity of the protein; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain.<br>
Examples of conservative substitutions are within the group of basic amino acids (such as arginine, lysine and histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine and valine), aromatic amino acids (such as phenylalanine, tryptophan and tyrosine), and small amino acids (such as glycine, alanine, serine, threonine and methionine). Amino acid substitutions which do not generally alter the specific activity are known in the art and are described, for example, by H. Neurath and R.L. Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/De, Asp/Glu, Thr/Ser, Ala/Gly, AlaTThr, Ser/Asn, AlaA^ai, Ser/Gly, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/Ile, LeuA^al, Ala/Glu, and Asp/Gly as well as these in reverse.<br>
In a second embodiment, the present invention relates to isolated polypeptides having dipeptidyl aminopeptidase activity which are encoded by nucleic acid sequences which hybridize under low stringency conditions, more preferably medium stringency conditions, and most preferably high stringency conditions, with an oligonucleotide probe which hybridizes under the same conditions with the nucleic acid sequence of SEQ ID N0:1 or its complementary strand (J. Sambrook, E.F. Fritsch, and T. Maniatus, 1989, Molecular Cloning, A Laboratory Manual, 2d edition. Cold Spring Harbor, New York); or allelic variants and fragments of the polypeptides, wherein the fragments have dipeptidyl aminopeptidase activity.<br>
Hybridization indicates that the nucleic acid sequence hybridizes to the oligonucleotide probe corresponding to the polypeptide encoding part of the nucleic acid sequence shown in<br><br>
SEQ ID N0:1,  under  low   to  high  stringency  conditions   (i.e.,   prehybridization  and hybridization at 42Â°C in 5X SSPE, 0.3% SDS, 200Âµg/ml sheared and denatured salmon sperm<br>
DNA, and either 25,  35 or 50% formamide for low,   medium and high  stringencies, respectively), following standard Southern blotting procedures.<br>
The amino acid sequence of SEQ ID NO: 2 or a partial sequence thereof may be used to design an oligonucleotide probe, or a nucleic acid sequence encoding a polypeptide of the present invention, such as the nucleic acid sequence of SEQ ID N0;1, or a subsequence thereof, may be used to identify and clone DNA encoding polypeptides having dipeptidyl aminopeptidase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 15, preferably at least 25, and more preferably at least 40 nucleotides in length. Longer probes can also be used. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with 32P, 3H, 35S, biotin, or avidin).<br>
Thus, a genomic, cDNA or combinatorial chemical library prepared from such other organisms may be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having dipeptidyl aminopeptidase activity. Genomic or other DNA from such other organisms may be separated by agarose or polyacrylamide gel electrophoresis, or other separation techniques. DNA from the libraries or the separated DNA may be transferred to and immobilized on nitrocellulose or other suitable carrier material. In order to identify a clone or DNA which is homologous with SEQ ID N0:1, the carrier material is used in a Southern blot in which the carrier material is finally washed three times for 30 minutes each using 2 x SSC, 0.2% SDS preferably at least 50Â°C, more preferably at least 55Â°C, more preferably at least 60Â°C, more preferably at least 65Â°C, even more preferably at least 70Â°C, and most preferably at least 75Â°C. Molecules to which the oligonucleotide probe hybridizes under these conditions are detected using X-ray film.<br><br>
In a third embodiment, the present invention relates to isolated polypeptides having the following physicochemical properties: (a) a pH optimum in the range of from about pH 4.4 to about pH 9.8 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide; (b) a temperature stability of 90% or more, relative to initial activity, at pH 7.5 determined after incubation for 20 minutes at 65Â°C in the absence of substrate; and (c) an activity towards Xaa-Pro-para-nitroanilide or Xaa-Ala-para-nitroanilide wherein Xaa is selected from the group consisting of Ala, Arg, Asp, Gly, and Val. The polypeptides of the present invention also have the ability to hydrolyze other substrates.<br>
In a preferred embodiment, the pH optimum in the range of from about pH 4.4 to about pH 9.8, more preferably in the range of from about pH 5.8 to about pH 9.8, and most preferably in the range of from about pH 7.5 to about pH 9.3 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide.<br>
In another preferred embodiment, a polypeptide of the present invention acts synergistically with an aminopeptidase to hydrolyze another polypeptide. The term "acts synergistically with an aminopeptidase to hydrolyze another polypeptide" is defined herein as the combination of a dipeptidyl aminopeptidase and an aminopeptidase which increases at least 5-fold, more preferably at least 10-fold, even more preferably at least 25-foId, most preferably at least 50-fold, and even most preferably at least 100-fold the hydrolysis of a peptide or a polypeptide relative to either individual enzyme alone. The aminopeptidase may be any aminopeptidase, but is preferably an aminopeptidase obtained from Aspergillus oryzae, and more preferably aminopeptidase I obtained from Aspergillus oryzae as described in WO 96/28542. The polypeptide is preferably a tripeptide, but may be any larger peptide or protein.<br>
In a fourth embodiment, the present invention relates to isolated polypeptides having immunochemical identity or partial immunochemical identity to the polypeptide having the amino acid sequence of SEQ ID NO:2. The immunochemical properties are determined by immunological cross-reaction identity tests by the well-known Ouchterlony double immunodiffusion procedure. Specifically, an antiserum containing antibodies which are immunoreactive or bind to epitopes of the polypeptide having the amino acid sequence of SEQ ID N0:2 are prepared by immunizing rabbits (or other rodents) according to the procedure<br><br>
described by Harboe and Ingild, In N.H. Axelsen, J. Kr0ll, and B, Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or Johnstone and Thorpe, Immunochemistry in Practice, Blackwell Scientific Publications, 1982 (more specifically pages 27-31). A polypeptide having immunochemical identity is a polypeptide which reacts with the antiserum in an identical fashion such as total fusion of precipitates, identical precipitate morphology, and/or identical electrophoretic mobility using a specific immunochemical technique. A further explanation of immunochemical identity is described by Axelsen, Bock, and Kr0ll, In N.H. Axelsen, J. Kr0ll, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 10. A polypeptide having partial immunochemical identity is a polypeptide which reacts with the antiserum in a partially identical fashion such as partial fusion of precipitates, partially identical precipitate morphology, and/or partially identical electrophoretic mobility using a specific immunochemical technique. A further explanation of partial immunochemical identity is described by Bock and Axelsen, In N.H. Axelsen, J. Kr0ll, and B. Weeks, editors, A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 1973, Chapter 11.<br>
Polypeptides encoded by nucleic acid sequences which hybridize with an oligonucleotide probe which hybridizes with the nucleic acid sequence of SEQ ID N0:1, its complementary strand, or allelic variants and subsequences of SEQ ID N0:1; allelic variants and fragments of the polypeptides; or the homologous polypeptides and polypeptides having identical or partially identical immunological properties may be obtained from microorganisms of any genus.<br>
In a preferred embodiment, these polypeptides may be obtained from a bacterial source. For example, these polypeptides may be obtained from a gram positive bacterium such as a Bacillus strain, e.g.. Bacillus alkalophilus. Bacillus amyloliqy faciens. Bacillus brevis. Bacillus circulans. Bacillus coagulans. Bacillus lautus. Bacillus lentus. Bacillus licheniformis. Bacillus megaterium. Bacillus stearothermophilus. Bacillus subtilis, or Bacillus thuringiensis', or a Streptomyces strain, e.g., Streptomyces lividans or Streptomyces murinus; or from a gram negative bacterium, e.g., E. coli or Pseudomonas sp.<br><br>
The polypeptides may be obtained from a fungal source, and more preferably from a yeast strain such as a Candida, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces or Yarrowia strain; or a filamentous fungal strain such as an Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filibasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicilliumy Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Trichoderma strain.<br>
In a preferred embodiment, the polypeptides are obtained from a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis strain.<br>
In another preferred embodiment, the polypeptides are obtained from a Fusarium bactridioides Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium solani, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpurogenum, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride strain.<br>
The polypeptides of the present invention are preferably obtained from species of Aspergillus including, but not limited to, Aspergillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, or Aspergillus oryzae.<br>
In a more preferred embodiment, a polypeptide of the present invention is obtained from an Aspergillus oryzae strain, and most preferably from Aspergillus oryzae ATCC 20386 or a mutant strain thereof, e.g., the polypeptide with the amino acid sequence of SEQ ID N0:2.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e,g.,  anamorphs,<br><br>
regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents. The polypeptides of the present invention may also be obtained from microorganisms which are synonyms of Aspergillus as defined by Raper, K.D. and Fennel, D.L, 1965, The Genus Aspergillus, The Wilkins Company, Baltimore. Aspergilli are mitosporic fungi characterized by an aspergillum conprised of a conidiospore stipe with no known teleomorphic states terminating in a vesicle, which in turn bears one or two layers of synchronously formed specialized cells, variously referred to as sterigmata or phialides, and asexually formed spores referred to as conidia. Known teleomorphs of Aspergillus include Eurotium, Neosartorya, and Emericella. Strains of Aspergillus and teleomorphs thereof are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such polypeptides may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The nucleic acid sequence may then be derived by similarly screening a genomic or cDNA library of another microorganism. Once a nucleic acid sequence encoding the polypeptide has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e,g,, Sambrook et al, 1989, supra).<br>
For purposes of the present invention, the term "obtained from" as used herein in<br>
connection with a given source shall mean that the polypeptide is produced by the source or by<br>
a cell in which a gene from the source has been inserted.	<br>
As defined herein, an "isolated" polypeptide is a polypeptide which is essentially free of other non-dipeptidyl aminopeptidase polypeptides, e.g., at least about 20% pure, preferably at least about 40% pure, more preferably about 60% pure, even more preferably about 80% pure, most preferably about 90% pure, and even most preferably about 95% pure, as determined by<br><br>
Nucleic Acid Sequences<br>
The present invention also relates to isolated nucleic acid sequences which encode a polypeptide of the present invention. In a preferred embodiment, the nucleic acid sequence encodes a polypeptide obtained from Aspergillus, e.g., Aspergillus oryzae, and in a more preferred embodiment, the nucleic acid sequence is obtained from Aspergillus oryzae ATCC 20386, e.g., the nucleic acid sequence of SEQ ID N0:1. In another more preferred embodiment, the nucleic acid sequence is the sequence contained in plasmid pMWR52 which is contained in Escherichia coli NRRL B-21682. The present invention also encompasses nucleic acid sequences which encode a polypeptide having the amino acid sequence of SEQ ID N0:2, which differ from SEQ ID NO:l by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO: 1 which encode fragments of SEQ ID N0:2 which have dipeptidyl aminopeptidase activity. A subsequence of SEQ ID NO: 1 is a nucleic acid sequence encompassed by SEQ ID NO: 1 except that one or more nucleotides from the 5' end and/or 3' end have been deleted. Preferably, a subsequence contains at least 990 nucleotides, more preferably at least 1140 nucleotides, and most preferably at least 1290 nucleotides.<br>
The nucleic acid sequences may be obtained from microorganisms which are taxonomic equivalents of Aspergillus as defined by Raper, K.D. and Fennel, D.I., 1965, supra., regardless of the species name by which they are known.<br>
The techniques used to isolate or clone a nucleic acid sequence encoding a polypeptide are known in the art and include isolation from genomic DNA, preparation from cDNA, or a combination thereof. The cloning of the nucleic acid sequences of the present invention from such genomic DNA can be effected, e.g., by using the well known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared structural features. See, e.g., Innis et al, 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other nucleic acid amplification procedures such as ligase chain reaction (LCR), ligated activated transcription (LAT) and nucleic acid sequence-<br><br>
based amplification (NASBA) may be used. The nucleic acid sequence may be cloned from a strain of Aspergillus, or another or related organism and thus, for example, may be an allelic or species variant of the polypeptide encoding region of the nucleic acid sequence.<br>
The term "isolated nucleic acid sequence" as used herein refers to a nucleic acid sequence which is essentially free of other nucleic acid sequences, e.g., at least about 20% pure, preferably at least about 40% pure, more preferably at least about 60% pure, even more preferably at least about 80% pure, and most preferably at least about 90% pure as determined by agarose electrophoresis. For example, an isolated nucleic acid sequence can be obtained by standard cloning procedures used in genetic engineering to relocate the nucleic acid sequence from its natural location to a different site where it will be reproduced. The cloning procedures may involve excision and isolation of a desired nucleic acid fragment comprising the nucleic acid sequence encoding the polypeptide, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host cell where multiple copies or clones of the nucleic acid sequence will be replicated. The nucleic acid sequence may be of genomic, cDNA, RNA, semisynthetic, synthetic origin, or any combinations thereof.<br>
The present invention also relates to nucleic acid sequences which have a degree of homology to the nucleic acid sequence of SEQ ID NO: 1 of at least about 50%, preferably about 60%, preferably about 70%, preferably about 80%, more preferably about 90%, even more preferably about 95%, and most preferably about 97% homology, which encode an active polypeptide. For purposes of the present invention, the degree of homology between two nucleic acid sequences is determined by the Clustal method (Higgins, 1989, supra) with an identity table, a gap penalty of 10, and a gap length penalty of 10.<br>
Modification of a nucleic acid sequence encoding a polypeptide of the present invention may be necessary for the synthesis of polypeptides substantially similar to the polypeptide. The term "substantially similar" to the polypeptide refers to non-naturally occurring forms of the polypeptide. These polypeptides may differ in some engineered way from the polypeptide isolated from its native source. For example, it may be of interest to synthesize variants of the polypeptide where the variants differ in specific activity, thermostability, pH optimum, or the like using, e.g., site-directed mutagenesis. The analogous sequence may be constructed on the<br><br>
basis of the nucleic acid sequence presented as the polypeptide encoding part of SEQ ID NO: 1, e.g., a subsequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the polypeptide encoded by the nucleic acid sequence, but which corresponds to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution, see, e.g.. Ford et al., 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active polypeptide. Amino acid residues essential to the activity of the polypeptide encoded by the isolated nucleic acid sequence of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g., Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in tile molecule, and the resultant mutant molecules are tested for dipeptidyl aminopeptidase activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et al, 1992, Science 255: 306-312; Smith et ai, 1992, Journal of Molecular Biology 224: 899-904; Wlodaver et al, 1992, FEES Utters 309: 59-64).<br>
Polypeptides of the present invention also include fused polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fused polypeptide is produced by fusing a nucleic acid sequence (or a portion thereof) encoding another polypeptide to a nucleic acid sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br><br>
The present invention also relates to isolated nucleic acid sequences encoding a polypeptide of the present invention, which hybridize under low stringency conditions, more preferably medium stringency conditions, and most preferably high stringency conditions, with an oligonucleotide probe which hybridizes under the same conditions with the nucleic acid sequence of SEQ ID NO: 1 or its complementary strand; or allelic variants and subsequences thereof (Sambrook et al, 1989, supra).<br>
Nucleic Acid Constructs<br>
The present invention also relates to nucleic acid constructs comprising a nucleic acid sequence of the present invention operably linked to one or more control sequences which direct the expression of the coding sequence in a suitable host cell under conditions compatible with the control sequences. Expression will be understood to include any step involved in the production of the polypeptide including, but not limited to, transcription, post-transcriptional modification, translation, post-translational modification, and secretion.<br>
"Nucleic acid construct" is defined herein as a nucleic acid molecule, either single- or double-stranded, which is isolated from a naturally occurring gene or which has been modified to contain segments of nucleic acid which are combined and juxtaposed in a manner which would not otherwise exist in nature. The term nucleic acid construct is synonymous with the term expression cassette when the nucleic acid construct contains all the control sequences required for expression of a coding sequence of the present invention. The term "coding sequence" as defined herein is a sequence which is transcribed into mRNA and translated into a polypeptide of the present invention. The boundaries of the coding sequence are generally determined by a ribosome binding site (prokaryotes) or by the ATG start codon (eukaryotes) located just upstream of the open reading frame at the 5' end of the mRNA and a transcription terminator sequence located just downstream of the open reading frame at the 3' end of the mRNA. A coding sequence can include, but is not limited to, DNA, cDNA, and recombinant nucleic acid sequences.<br>
An isolated nucleic acid sequence encoding a polypeptide of the present invention may be manipulated in a variety of ways to provide for expression of the polypeptide.  Manipulation<br><br>
of the nucleic acid sequence encoding a polypeptide prior to its insertion into a vector may be desirable or necessary depending on the expression vector.   The techniques for modifying<br>
s<br>
nucleic acid sequences utilizing cloning methods are well known in the art.<br>
The term "control sequences" is defined herein to include all components which are necessary or advantageous for the expression of a polypeptide of the present invention. Each control sequence may be native or foreign to the nucleic acid sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, a polyadenylation sequence, a propeptide sequence, a promoter, a signal sequence, and a transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleic acid sequence encoding a polypeptide. The term "operably linked" is defined herein as a configuration in which a control sequence is appropriately placed at a position relative to the coding sequence of the DNA sequence such that the control sequence directs the production of a polypeptide.<br>
The control sequence may be an appropriate promoter sequence, a nucleic acid sequence which is recognized by a host cell for expression of the nucleic acid sequence. The promoter sequence contains transcriptional control sequences which mediate the expression of the polypeptide. The promoter may be any nucleic acid sequence which shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or intracellular polypeptides either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in a bacterial host cell, are the promoters obtained from the E, coli lac operon, the Streptomyces coelicolor agaraser gene (dagA), the Bacillus subtilis levansucrase gene (sacB), the Bacillus licheniformis alpha-amylase gene (amyL), the Bacillus stearothermophilus maltogenic amylase gene (amyM), the Bacillus amyloliquefaciens alpha-amylase gene (amyQ), the Bacillus licheniformis penicillinase gene (penP), the Bacillus subtilis xylA and xylB genes, and the prokaryotic beta-lactamase gene (Villa-Kamaroff et al,<br><br>
1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et al., 19S3, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook et aL, 1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes encoding Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamori glucoamyiase iglaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae triose phosphate isomerase, Aspergillus nidulans acetamidase, Fusarium oxysporum trypsin-like protease (U.S. Patent No. 4,288,627), and mutant, truncated, and hybrid promoters thereof. Particularly preferred promoters for use in filamentous fungal host cells are the TAKA amylase, NA2-tpi (a hybrid of the promoters from the genes encoding Aspergillus niger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase), and glaA promoters.<br>
In a yeast host, useful promoters are obtained from the Saccharomyces cerevisiae enolase (ENO-l) gene, the Saccharomyces cerevisiae galactokinase gene (GALl), the Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase genes (ADH2/GAP), and the Saccharomyces cerevisiae 3-phosphoglycerate kinase gene. Other useful promoters for yeast host cells are described by Romanes et a/., 1992, Yeast 8: 423-488. In a mammalian host cell, useful promoters include viral promoters such as those from Simian Virus 40 (SV40), Rous sarcoma virus (RSV), adenovirus, bovine papilloma virus (BPV), and human cytomegalovirus (CMV).<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to terminate transcription. The terminator sequence is operably linked to the 3' terminus of the nucleic acid sequence encoding the polypeptide. Any terminator which is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes encoding Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamyiase, Aspergillus<br><br>
nidulans anthranilate synthase, Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-like protease.<br>
Preferred terminators for yeast host cells are obtained from the genes encoding Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYCl), or Saccharomyces cerevisiae gIyceraldehyde-3-phosphate dehydrogenase. Other useful terminators for yeast host cells are described by Romanos et al., 1992, supra. Terminator sequences are well known in the art for mammalian host cells.<br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleic acid sequence encoding the polypeptide. Any leader sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred leaders for filamentous fungal host cells are obtained from the genes encoding Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase.<br>
Suitable leaders for yeast host cells are obtained from the Saccharomyces cerevisiae enolase (ENO-1) gene, the Saccharomyces cerevisiae 3-phosphoglycerate kinase gene, the Saccharomyces cerevisiae alpha-factor, and the Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase genes (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence which is operably linked to the 3' terminus of the nucleic acid sequence and which, when transcribed, is recognized by the host cell as a signal to add poly adenosine residues to transcribed mRNA. Any polyadenylation sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes encoding Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, and Aspergillus niger alpha-glucosidase.<br>
Useful polyadenylation sequences for yeast host cells are described by Guo and Sherman, 1995, Molecular Cellular Biology 15: 5983-5990. Polyadenylation sequences are well known in the art for mammalian host cells.<br><br>
The control sequence may also be a signal peptide coding region, which codes for an amino acid sequence linked to the amino terminus of the polypeptide which can direct the encoded polypeptide into the cell's secretory pathway.  The 5' end of the coding sequence of the nucleic acid sequence may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region which encodes the secreted polypeptide.  Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region which is foreign to the coding sequence.   The foreign signal peptide coding region may be required where the coding sequence does not normally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to obtain enhanced secretion of the polypeptide. The signal peptide coding region may be obtained from a glucoamylase or an amylase gene from an Aspergillus species, a lipase or proteinase gene from a Rhizomucor species, the gene for the alpha-factor from Saccharomyces cerevisiae, an amylase or a protease gene from a Bacillus species, or the calf preprochymosin gene. However, any signal peptide coding region which directs the expressed polypeptide into the secretory pathway of a host cell of choice may be used in the present invention.<br>
An effective signal peptide coding region for bacterial host cells is the signal peptide coding region obtained from the maltogenic amylase gene from Bacillus NCIB 11837, the Bacillus stearothermophilus alpha-amylase gene, the Bacillus licheniformis subtilisin gene, the Bacillus licheniformis beta-lactamase gene, the Bacillus stearothermophilus neutral proteases genes (nprT, nprS, nprM), or the Bacillus subtilis PrsA gene. Further signal peptides are described by Simonen and Palva, 1993, Microbiological Reviews 57: 109-137.<br>
An effective signal peptide coding region for filamentous fungal host cells is the signal peptide coding region obtained from the Aspergillus oryzae TAKA amylase gene, Aspergillus niger neutral amylase gene, Rhizomucor miehei aspartic proteinase gene, Humicola lanuginosa cellulase gene, or Humicola lanuginosa lipase gene.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Saccharomyces cerevisiae invertase. Other useful signal peptide coding regions are described by Romanos et al, 1992, supra.<br><br>
The control sequence may also be a propeptide coding region, which codes for an amino acid sequence positioned at the amino terminus of a polypeptide. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases). A propolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the Bacillus subtilis alkaline protease gene (aprE), the Bacillus subtilis netural protease gene (nprT), the Saccharomyces cerevisiae alpha-factor gene, the Rhizomucor miehei aspartic proteinase gene, or the Myceliophthora thermophila laccase gene (WO 95/33836j.<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a polypeptide, the propeptide region is positioned next to the amino terminus of a polypeptide and the signal peptrJe region is positioned next to the amino terminus of the propeptide region.<br>
The nucleic acid constructs of the present invention may also comprise one or more nucleic acid sequences which encode one or more factors that are advantageous for directing the expression of the polypeptide, e.g., a transcriptional activator {e.g., a trans-acting factor), a chaperone, and a processing protease. Any factor that is functional in the host cell of choice may be used in the present invention. The nucleic acids encoding one or more of these factors are not necessarily in.- tandem with the nucleic acid sequence encoding the polypeptide.<br>
A transcrip:;'jnal activator is a protein which activates transcription of a nucleic acid sequence encoding i polypeptide (Kudla et al, 1990, EMBO Journal 9: 1355-1364; Jarai and Buxton, 1994, Current Genetics 26: 2238-244; Verdier, 1990, Yeast 6: 271-297). The nucleic acid sequence encoding an activator may be obtained from the genes encoding Bacillus stearothermophilus SprA {nprA), Saccharomyces cerevisiae heme activator protein 1 (hapl), Saccharomyces cerevisiae galactose metabolizing protein 4 (gal4), Aspergillus nidulans ammonia regulation protein (areA), and Aspergillus oryzae alpha-amylase activator (amyR), For further example., see Verdier, 1990, supra and MacKenzie et al, 1993, Journal of General Microbiology 139:1:95-2307.<br>
A chaperone  is a protein which assists another polypeptide in folding properly (Hard et al., 1994, 7755 19: 10-25; Bergeron et al, 1994, TIBS 19: 124-128; Demolder et al, 1994,<br><br>
Journal of Biotechnology 32: 179-189; Craig, 1993, Science 260: 1902-1903; Gething and Sambrook, 1992, Nature 355; 33-45; Puig and Gilbert, 1994, Journal of Biological Chemistry 269: 7764-7771; Wang and Tsou, 1993, The FASEB Journal 1: 151541157; Robinson et al., 1994, Bio/Technology 1: 381-384; Jacobs et al, 1993, Molecular Microbiology 8: 957-966). The nucleic acid sequence encoding a chaperone may be obtained from the genes encoding Bacillus subtilis GroE proteins. Bacillus subtilis PrsA, Aspergillus oryzae protein disulphide isomerase, Saccharomyces cerevisiae calnexin, Saccharomyces cerevisiae BiP/GRP78, and Saccharomyces cerevisiae Hsp70. For further examples, see Gething and Sambrook, 1992, supra, and Hartl et al, 1994, supra.<br>
A processing protease is a protease that cleaves a propeptide to generate a mature biochemically active polypeptide (Enderlin and Ogrydziak, 1994, Yeast 10: 67-79; Fuller et al, 1989, Proceedings of the National Academy of Sciences USA 86: 1434-1438; Julius et al, 1984, Cell 37: 1075-1089; Julius et al, 1983, Cell 32: 839-852; U.S. Patent No. 5,702,934). The nucleic acid sequence encoding a processing protease may be obtained from the genes encoding Saccharomyces cerevisiae dipeptidyl aminopeptidase, Saccharomyces cerevisiae Kex2, Yarrowia lipolytica dibasic processing endoprotease (xpr6), and Fusarium oxysporum metalloprotease (p45 gene).<br>
It may also be desirable to add regulatory sequences which allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems would include the lac, tac, and trp operator systems. In yeast, the ADH2 system or GALl system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and the Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are those which allow for gene amplification. In eukaryotic systems, these include the dihydrofolate reductase gene which is amplified in the presence of methotrexate, and the metallothionein genes which are amplified with heavy metals. In these cases, the nucleic acid sequence encoding the polypeptide would be operably linked with the regulatory sequence.<br><br>
The present invention also relates to nucleic acid constructs for altering the expression of an endogenous gene encoding a polypeptide of the present invention. The constructs may contain the minimal number of components necessary for altering expression of the endogenous gene. In one embodiment, the nucleic acid constructs preferably contain (a) a targeting sequence, (b) a regulatory sequence, (c) an exon, and (d) a splice-donor site. Upon introduction of the nucleic acid construct into a cell, the construct inserts by homologous recombination into the cellular genome at the endogenous gene site. The targeting sequence directs the integration of elements (a)-(d) into the endogenous gene such that elements (b)-(d) are operably linked to the endogenous gene. In another embodiment, the nucleic acid constructs contain (a) a targeting sequence, (b) a regulatory sequence, (c) an exon, (d) a splice-donor site, (e) an intron, and (f) a splice-acceptor site, wherein the targeting sequence directs the integration of elements (a)-(f) such that elements (b)-(f) are operably linked to the endogenous gene. However, the constructs may contain additional components such as a selectable marker.<br>
In both embodiments, the introduction of these components results in production of a new transcription unit in which expression of the endogenous gene is altered. In essence, the new transcription unit is a fusion product of the sequences introduced by the targeting constructs and the endogenous gene. In one embodiment in which the endogenous gene is altered, the gene is activated. In this embodiment, homologous recombination is used to replace, disrupt, or disable the regulatory region normally associated with the endogenous gene of a parent cell through the insertion of a regulatory sequence which causes the gene to be expressed at higher levels than evident in the corresponding parent cell. The activated gene can be further amplified by the inclusion of an amplifiable selectable marker gene in the construct using methods well known in the art (see, for example, U.S. Patent No. 5,641,670). In another embodiment in which the endogenous gene is altered expression of the gene is reduced.<br>
The targeting sequence can be within the endogenous gene, immediately adjacent to the gene, within an upstream gene, or upstream of and at a distance from the endogenous gene. One or more targeting sequences can be used.    For example, a circular plasmid or DNA<br><br>
fragment preferably employs a single targeting sequence, while a linear plasmid or DNA fragment preferably employs two targeting sequences.<br>
The regulatory sequence of the construct can be comprised of one or more promoters, enhancers, scaffold-attachment regions or matrix attachment sites, negative regulatory elements, transcription binding sites, or combinations of these sequences.<br>
The constructs further contain one or more exons of the endogenous gene. An exon is defined as a DNA sequence which is copied into RNA and is present in a mature mRNA molecule such that the exon sequence is in-frame with the coding region of the endogenous gene. The exons can, optionally, contain DNA which encodes one or more amino acids and/or partially encodes an amino acid. Alternatively, the exon contains DNA which corresponds to a 5' non-encoding region. Where the exogenous exon or exons encode one or more amino acids and/or a portion of an amino acid, the nucleic acid construct is designed such that, upon transcription and splicing, the reading frame is in-frame with the coding region of the endogenous gene so that the appropriate reading frame of the portion of the mRNA derived from the second exon is unchanged.<br>
The splice-donor site of the constructs directs the splicing of one exon to another exon. Typically, the first exon lies 5' of the second exon, and the splice-donor site overlapping and flanking the first exon on its 3' side recognizes a splice-acceptor site flanking the second exon on the 5' side of the second exon. A splice-acceptor site, like a splice-donor site, is a sequence which directs the splicing of one exon to another exon. Acting in conjunction with a splice-donor site, the splicing apparatus uses a splice-acceptor site to effect the removal of an intron.<br>
Expression Vectors<br>
The present invention also relates to recombinant expression vectors comprising a nucleic acid sequence of the present invention, a promoter, and transcriptional and translational stop signals. The various nucleic acid and control sequences described above may be joined together to produce a recombinant expression vector which may include one or more convenient restriction sites to allow for insertion or substitution of the nucleic acid sequence encoding the polypeptide at such sites.  Alternatively, the nucleic acid sequence of the present<br><br>
invention may be expressed by inserting the nucleic acid sequence or a nucleic acid construct comprising the sequence into an appropriate vector for expression. In creating the expression vector, the coding sequence is located in the vector so that the coding sequence is operably linked with the appropriate control sequences for expression, and possibly secretion.<br>
The recombinant expression vector may be any vector (e.g., a plasmid or virus) which can be conveniently subjected to recombinant DNA procedures and can bring about the expression of the nucleic acid sequence. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. The vectors may be linear or closed circular plasmids. The vector may be an autonomously replicating vector, i.e,, a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g., a plasmid, an extrachromosomal element, a minichromosome, or an artificial chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which it has been integrated. The vector system may be a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be introduced into the genome of the host cell, or a transposon.<br>
The vectors of the present invention preferably contain one or more selectable markers which permit easy selection of transformed cells. A selectable marker is a gene the product of which provides for biocide or viral resistance, resistance to heavy metals, prototrophy to auxotrophs, and the like. Examples of bacterial selectable markers are the dal genes from Bacillus subtilis or Bacillus licheniformis, or markers which confer antibiotic resistance such as ampicillin, kanamycin, chloramphenicol or tetracycline resistance. Suitable markers for mammalian cells are the dihydrofolate reductase (dfhr), hygromycin phosphotransferase (hygB), aminoglycoside phosphotransferase II, and phleomycin resistance genes. Suitable markers for yeast host cells are ADE2, HISS, LEU2, LYS2, MET3, TRPl, and URA3. A selectable marker for use in a filamentous fungal host cell may be selected from the group including, but not limited to, amdS (acetamidase), argB (ornithine carbamoyltransferase), bar (phosphinothricin acetyltransferase), hygB (hygromycin phosphotransferase), niaD (nitrate<br><br>
reductase), pyrG (orotidine-5'-phosphate decarboxylase), sC (sulfate adenyltransferase), trpC (anthranilate synthase), as well as equivalents from other species. Preferred for use in an Aspergillus cell are the amdS and pyrG genes of Aspergillus nidulans or Aspergillus oryzae and the bar gene of Streptomyces hygroscopicus. Furthermore, selection may be accomplished by co-transformation, e.g., as described in WO 91/17243, where the selectable marker is on a separate vector.<br>
The vectors of the present invention preferably contain an element(s) that permits stable integration of the vector into the host cell genome or autonomous replication of the vector in the cell independent of the genome of the cell.<br>
For integration into the host cell genome, the vector may rely on the nucleic acid sequence encoding the polypeptide or any other element of the vector for stable integration of the vector into the genome by homologous or nonhomologous recombination. Alternatively, the vector may contain additional nucleic acid sequences for directing integration by homologous recombinadon into the genome of the host cell. The additional nucleic acid sequences enable the vector to be integrated into the host cell genome at a precise location(s) in the chromosome(s). To increase the likelihood of integration at a precise location, the integrational elements should preferably contain a sufficient number of nucleic acids, such as 100 to 1,500 base pairs, preferably 400 to 1,500 base pairs, and most preferably 800 to 1,500 base pairs, which are highly homologous with the corresponding target sequence to enhance the probability of homologous recombination. The integrational elements may be any sequence that is homologous with the target sequence in the genome of the host cell. Furthermore, the integrational elements may be non-encoding or encoding nucleic acid sequences. On the other hand, the vector may be integrated into the genome of the host cell by non-homologous recombination.<br>
For autonomous replication, the vector may further comprise an origin of replication enabling the vector to replicate autonomously in the host cell in question. Examples of bacterial origins of replication are the origins of replication of plasmids pBR322, pUC19, pACYC177, and pACYC184 permitting replication in E. coli, and pUBl 10, pE194, pTA1060, and pAMBl permitting replication in Bacillus, Examples of origins of replication for use in a yeast host cell<br><br>
are the 2 micron origin of replication, ARSl, ARS4, the combination of ARS1 and CEN3, and the combination of ARS4 and CEN6. The origin of replication may be one having a mutation which makes its functioning temperature-sensitive in the host cell (see, e.g., Ehrlich, 1978, Proceedings of the National Academy of Sciences USA 75: 1433).<br>
More than one copy of a nucleic acid sequence encoding a polypeptide of the present invention may be inserted into the host cell to amplify expression of the nucleic acid sequence. Stable amplification of the nucleic acid sequence can be obtained by integrating at least one additional copy of the sequence into the host cell genome or by including an amplifiable selectable marker gene with the nucleic acid sequence where cells containing amplified copies of the selectable marker gene, and thereby additional copies of the nucleic acid sequence, can be selected for by culturing the cells in the presence of the appropriate selectable agent.<br>
The procedures used to ligate the elements described above to construct the recombinant expression vectors of the present invention are well known to one skilled in the art (see, e.g., Sambrook et al, 1989, supra).<br>
Host Cells<br>
The present invention also relates to recombinant host cells, comprising a nucleic acid sequence of the invention, which are advantageously used in the recombinant production of the polypeptides. The term "host cell" encompasses any progeny of a parent cell which is not identical to the parent cell due to mutations that occur during replication.<br>
A vector comprising a nucleic acid sequence of the present invention is introduced into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector. Integration is generally considered to be an advantage as the nucleic acid sequence is more likely to be stably maintained in the cell. Integration of the vector into the host chromosome may occur by homologous or non-homologous recombination as described above.<br>
The choice of a host cell will to a large extent depend upon the gene encoding the polypeptide and its source. The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g., a eukaryote.  Useful unicellular cells are<br><br>
bacterial cells such as gram positive bacteria including, but not limited to, a Bacillus cell, e.g., Bacillus alkalophilus. Bacillus amyloliquefaciens. Bacillus brevis. Bacillus circulans, Bacillus 'clausii. Bacillus coagulans, Bacillus lautus. Bacillus lentus. Bacillus licheniformis. Bacillus megaterium. Bacillus stearothermophilus. Bacillus subtilis, and Bacillus thuringiensis; or a Streptomyces cell, e.g., Streptomyces lividans or Streptomyces murinus, or gram negative bacteria such as E. coli and Pseudomonas sp. In a preferred embodiment, the bacterial host cell is a Bacillus lentus. Bacillus licheniformis. Bacillus stearothermophilus, or Bacillus subtilis cell. The introduction of a vector into a bacterial host cell may, for instance, be effected by protoplast transformation (see, e.g., Chang and Cohen, 1979, Molecular General Genetics 168: 111-115), by using competent cells (see, e.g.. Young and Spizizin, 1961, Journal of Bacteriology 81: 823-829, or Dubnau and Davidoff-Abelson, 1971, Journal of Molecular Biology 56: 209-221), by electroporation (see, e.g., Shigekawa and Dower, 1988, Biotechniques 6: 742-751), or by conjugation (see, e.g., Koehler and Thome, 1987, Journal of Bacteriology 169: 5771-5278).<br>
The host cell may be a eukaryote, such as a mammalian cell, an insect cell, a plant cell or a fungal cell. Useful mammalian cells include Chinese hamster ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK) cells, COS cells, or any number of other immortalized cell lines available, e.g., from the American Type Culture Collection.<br>
In a preferred embodiment, the host cell is a fungal cell. "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota, Chytridiomycota, and Zygomycota (as defined by Hawksworth et al.. In, Ainsworth and Bisby's Dictionary of The Fungi, 8th edition, 1995, CAB International, University Press, Cambridge, UK) as well as the Oomycota (as cited in Hawksworth et al., 1995, supra, page 171) and all mitosporic fungi (Hawksworth et al., 1995, supra). Representative groups of Ascomycota include, e.g., Neurospora, Eupenicillium {=Penicillium), Emericella (=Aspergillus), Eurotium {-Aspergillus), and the tme yeasts listed below. Examples of Basidiomycota include mushrooms, rusts, and smuts. Representative groups of Chytridiomycota include, e.g., Allomyces, Blastocladiella, Coelomomyces, and aquatic fungi. Representative groups of Oomycota include, e.g., Saprolegniomycetous aquatic fungi (water molds) such as Achlya.   Examples of mitosporic fungi include Aspergillus,<br><br>
Penicillium, Candida, and Altemaria.   Representative groups of Zygomycota include, e.g., Rhizopus and Mucor.<br>
In a more preferred embodiment, the fungal host cell is a yeast cell.  "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfecti (Blastomycetes).  The ascosporogenous yeasts are divided into the families Spermophthoraceae and Saccharomycetaceae.  The latter is comprised of four subfamilies, Schizosaccharomycoideae (e.g., genus Schizosaccharomyces), Nadsonioideae, Lipomycoideae,   and   Saccharomycoideae   (e.g.,   genera   Kluyveromyces,   Pichia,   and Saccharomyces),      The  basidiosporogenous  yeasts  include  the   genera   Leucosporidim, Rhodosporidium, Sporidiobolus, Filobasidium, and Filobasidiella,   Yeast belonging to the Fungi  Imperfecti  are   divided   into   two   families,   Sporobolomycetaceae   (e.g.,   genera Sporobolomyces and Bullera) and Cryptococcaceae (e.g.,  genus  Candida),     Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast (Skinner, F.A., Passmore, S.M., and Davenport, R.R., eds, Soc. App. Bacteriol. Symposium Series No. 9, 1980.  The biology of yeast and manipulation of yeast genetics are well known in the art (see, e.g.. Biochemistry and Genetics of Yeast, Bacil, M., Horecker, B.J., and Stopani, A.O.M., editors, 2nd edition, 1987; The Yeasts, Rose, A.H.,  and Harrison, J.S.,  editors, 2nd edition, 1987; and 77ie Molecular Biology of the Yeast Saccharomyces, Strathem et al, editors, 1981).<br>
In an even more preferred embodiment, the yeast host cell is a cell of a species of Candida, Hansenula, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia,<br>
In a most preferred embodiment, the yeast host cell is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis cell. In another most preferred embodiment, the yeast host cell is a Kluyveromyces lactis cell. In another most preferred embodiment, the yeast host cell is a Yarrowia lipolytica cell.<br>
In another more preferred embodiment, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota<br><br>
(as defined by Hawksworth et ah, 1995, supra). The filamentous fungi are characterized by a mycelial wall composed of chitin, cellulose, glucan, chitosan, mannan, and other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative. In a more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium, and Trichoderma.<br>
In an even more preferred embodiment, the filamentous fungal host cell is an Aspergillus cell. In another even more preferred embodiment, the filamentous fungal host cell is an Acremonium cell. In another even more preferred embodiment, the filamentous fungal host cell is a Fusarium cell. In another even more preferred embodiment, the filamentous fungal host cell is a Humicola cell. In another even more preferred embodiment, the filamentous fungal host cell is a Mucor cell. In another even more preferred embodiment, the filamentous fungal host cell is a Myceliophthora cell. In another even more preferred embodiment, the filamentous fungal host cell is a Neurospora cell. In another even more preferred embodiment, the filamentous fungal host cell is a Penicillium cell. In another even more preferred embodiment, the filamentous fungal host cell is a Thielavia cell. In another even more preferred embodiment, the filamentous fungal host cell is a Tolypocladium cell. In another even more preferred embodiment, the filamentous fungal host cell is a Trichoderma cell.<br>
In a most preferred embodiment, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, or Aspergillus oryzae cell. In another most preferred embodiment, the filamentous fungal host cell is a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium solani, Fusarium sporotrichioides Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell.  In<br><br>
an even most preferred embodiment, the filamentous fungal parent cell is a Fusarium venenatum (Nirenberg sp. nov.) cell. In another most preferred embodiment, the filamentous fungal host cell is a Humicola insolens or Humicola lanuginosa cell. In another most preferred embodiment, the filamentous fungal host cell is a Mucor miehei cell. In another most preferred embodiment, the filamentous fungal host cell is a Myceliophthora thermophilum cell. In another most preferred embodiment, the filamentous fungal host cell is a Neurospora crassa cell. In another most preferred embodiment, the filamentous fungal host cell is a Penicillium purpurogenum cell. In another most preferred embodiment, the filamentous fungal host cell is a Thielavia terrestris cell. In another most preferred embodiment, the Trichoderma cell is a Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei or Trichoderma viride cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus host ceils are described in EP 238 023 and Yelton et al, 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardier et aL, 1989, Gene 78: 147-156 and WO 96/00787. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc., New York; Ito et aL, 1983, Journal of Bacteriology 153: 163; and Hinnen et al, 1978, Proceedings of the National Academy of Sciences USA 75: 1920. Mammalian cells may be transformed by direct uptake using the calcium phosphate precipitation method of Graham and Van der Eb (1978, Virology 52: 546).<br>
Methods of Production<br>
The present invention also relates to methods for producing a polypeptide of the present invention comprising (a) cultivating a strain, which in its wild-type form is capable of producing the polypeptide, to produce a supematant comprising the polypeptide; and (b) recovering the polypeptide. Preferably, the strain is of the genus Aspergillus.<br><br>
The present invention also relates to methods for producing a polypeptide of the present invention comprising (a) cultivating a host cell under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
The present invention further relates to methods for producing a polypeptide of the present invention comprising (a) cultivating a homologously recombinant cell, having incorporated therein a new transcription unit comprising a regulatory sequence, an exon, and/or a splice donor site operably linked to a second exon of an endogenous nucleic acid sequence encoding the polypeptide, under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide. The methods are based on the use of gene activation technology, for example, as described in U.S. Patent No. 5,641,670. Gene activation technology is based on activating a gene which is normally unexpressed in a cell or increasing expression of a gene which is expressed at very low levels in a cell. Gene activation technology includes methods of inserting an exogenous DNA construct containing a regulatory sequence, an exon, and/or a splice donor site into the genomic DNA of a cell in such a manner that the insertion results in the production of a new transcription unit in which the regulatory sequence, the exon, and/or the splice donor site are operably linked to and activate expression of the endogenous gene.<br>
In the production methods of the present invention, the cells are cultivated in a nutrient medium suitable for production of the polypeptide using methods known in the art. For example, the cell may be cultivated by shake flask cultivation, small-scale or large-scale fermentation (including continuous, batch, fed-batch, or solid state fermentations) in laboratory or industrial fermentors performed in a suitable medium and under conditions allowing the polypeptide to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium comprising carbon and nitrogen sources and inorganic salts, using procedures known in the art (see, e.g., references for bacteria and yeast; Bennett, J.W. and LaSure, L., editors. More Gene Manipulations in Fungi, Academic Press, CA, 1991). Suitable media are available from commercial suppliers or may be prepared according to published compositions (e.g., in catalogues of the American Type Culture Collection). If the polypeptide is secreted into the nutrient medium, the polypeptide can be recovered directly from the  medium.     If the<br><br>
polypeptide is not secreted, it can be recovered from cell lysates.<br>
The polypeptides may be detected using methods known in the art that are specific for the polypeptides. These detection methods may include use of specific antibodies, formation of an enzyme product, or disappearance of an enzyme substrate. For example, an enzyme assay may be used to determine the activity of the polypeptide. Procedures for determining dipeptidyl aminopeptidase activity are known in the art and include, e.g., measuring the initial rate of hydrolysis of a p-nitroanilide at 405 nm.<br>
The resulting polypeptide may be recovered by methods known in the art. For example, the polypeptide may be recovered from the nutrient medium by conventional procedures including, but not limited to, centrifugation, filtration, extraction, spray-drying, evaporation, or precipitation.<br>
The polypeptides of the present invention may be purified by a variety of procedures known in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures (e.g., preparative isoelectric focusing, differential solubility {e.g., ammonium sulfate precipitation), SDS-PAGE, or extraction (see, e.g.. Protein Purification, J.-C. Janson and Lars Ryden, editors, VCH Publishers, New York, 1989).<br>
Removal or Reduction of Dipeptidyl Aminopeptidase Activity<br>
The present invention also relates to methods for producing a mutant cell of a parent cell, which comprises disrupting or deleting a nucleic acid sequence encoding the polypeptide or a control sequence thereof, which results in the mutant cell producing less of the polypeptide than the parent cell.<br>
The construction of strains which have reduced dipeptidyl aminopeptidase activity may be conveniently accomplished by modification or inactivation of a nucleic acid sequence necessary for expression of the polypeptide having dipeptidyl aminopeptidase activity in the cell. The nucleic acid sequence to be modified or inactivated may be, for example, a nucleic acid sequence encoding the polypeptide or a part thereof essential for exhibiting dipeptidyl aminopeptidase activity, or the nucleic acid sequence may have a regulatory function required<br><br>
for the expression of the polypeptide from the coding sequence of the nucleic acid sequence. An example of such a regulatory or control sequence may be a promoter sequence or a functional part thereof, i.e., a part which is sufficient for affecting expression of the polypeptide. Other control sequences for possible modification include, but are not limited to, a leader, a polyadenylation sequence, a propeptide sequence, a signal sequence, and a termination terminator.<br>
Modification or inactivation of the nucleic acid sequence may be performed by subjecting the cell to mutagenesis and selecting for cells in which the dipeptidyl aminopeptidase producing capability has been reduced or eliminated. The mutagenesis, which may be specific or random, may be performed, for example, by use of a suitable physical or chemical mutagenizing agent, by use of a suitable oligonucleotide, or by subjecting the DNA sequence to PCR generated mutagenesis. Furthermore, the mutagenesis may be performed by use of any combination of these mutagenizing agents.<br>
Examples of a physical or chemical mutagenizing agent suitable for the present purpose include ultraviolet (UV) irradiation, hydroxylamine, N-methyl-N'-nitro-N-nitrosoguanidine (MNNG), O-methyl hydroxylamine, nitrous acid, ethyl methane sulphonate (EMS), sodium bisulphite, formic acid, and nucleotide analogues.<br>
When such agents are used, the mutagenesis is typically performed by incubating the cell to be mutagenized in the presence of the mutagenizing agent of choice under suitable conditions, and selecting for cells exhibiting reduced or no expression of dipeptidyl aminopeptidase activity.<br>
Modification or inactivation of production of a polypeptide of the present invention may be accomplished by introduction, substitution, or removal of one or more nucleotides in the nucleic acid sequence encoding the polypeptide or a regulatory element required for the transcription or translation thereof. For example, nucleotides may be  inserted or removed so as to result in the introduction of a stop codon, the removal of the start codon, or a change of the open reading frame. Such modification or inactivation may be accomplished by site-directed mutagenesis or PCR generated mutagenesis in accordance with methods known in the art. Although, in principle, the modification may be performed in vivo, i.e., directly on the cell<br><br>
expressing the nucleic acid sequence to be modified, it is preferred that the modification be performed in vitro as exemplified below.<br>
An example of a convenient way to inactivate or reduce production by a host cell of choice is based on techniques of gene replacement or gene interruption. For example, in the gene interruption method, a nucleic acid sequence corresponding to the endogenous gene or gene fragment of interest is mutagenized in vitro to produce a defective nucleic acid sequence which is then transformed into the host cell to produce a defective gene. By homologous recombination, the defective nucleic acid sequence replaces the endogenous gene or gene fragment. It may be desirable that the defective gene or gene fragment also encodes a marker which may be used for selection of transformants in which the gene encoding the polypeptide has been modified or destroyed.<br>
Alternatively, modification or inactivation of the nucleic acid sequence encoding a polypeptide of the present invention may be performed by established anti-sense techniques using a nucleotide sequence complementary to the polypeptide encoding sequence. More specifically, production of the polypeptide by a cell may be reduced or eliminated by introducing a nucleotide sequence complementary to the nucleic acid sequence encoding the polypeptide which may be transcribed in the cell and is capable of hybridizing to the polypeptide mRNA produced in the cell. Under conditions allowing the complementary anti-sense nucleotide sequence to hybridize to the polypeptide mRNA, the amount of polypeptide translated is thus reduced or eliminated.<br>
It is preferred that the cell to be modified in accordance with the methods of the present invention is of microbial origin, for example, a fungal strain which is suitable for the production of desired protein products, either homologous or heterologous to the cell.<br>
The present invention further relates to a mutant cell of a parent cell which comprises a disruption or deletion of a nucleic acid sequence encoding the polypeptide or a control sequence thereof, which results in the mutant cell producing less of the polypeptide than the parent cell.<br>
The polypeptide-deficient mutant cells so created are particularly useful as host cells for the expression of homologous and/or heterologous polypeptides. Therefore, the present invention further relates to methods for producing a homologous or heterologous polypeptide<br><br>
comprising (a) culturing the mutant cell under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide. In the present context, the term "heterologous polypeptides" is defined herein as polypeptides which are not native to the host cell, a native protein in which modifications have been made to alter the native sequence, or a native protein whose expression is quantitatively altered as a result of a manipulation of the host cell by recombinant DNA techniques.<br>
In a still further aspect, the present invention relates to a method for producing a protein product essentially free of dipeptidyl aminopeptidase activity by fermentation of a cell which produces both a polypeptide of the present invention as well as the protein product of interest. The method comprises adding an effective amount of an agent capable of inhibiting dipeptidyl aminopeptidase activity to the fermentation broth either during or after the fermentation has been completed, recovering the product of interest from the fermentation broth, and optionally subjecting the recovered product to further purification.<br>
In a still further alternative aspect, the present invention relates to a method for producing a protein product essentially free of dipeptidyl aminopeptidase activity, wherein the protein product of interest is encoded by a DNA sequence present in a cell encoding a polypeptide of the present invention. The method comprises cultivating the cell under conditions permitting the expression of the product, subjecting the resultant culture broth to a combined pH and temperature treatment so as to reduce the dipeptidyl aminopeptidase activity substantially, and recovering the product from the culture broth. Altematively, the combined pH and temperature treatment may be performed on an enzyme preparation recovered from the culture broth. The combined pH and temperature treatment may optionally be used in combination with a treatment with a dipeptidyl aminopeptidase inhibitor.<br>
The combined pH and temperature treatment is preferably carried out at a pH in the range of 9-11 and a temperature in the range of 40-75Â°C for a sufficient period of time to attain the desired effect, where typically, 30 to 60 minutes is sufficient.<br>
In accordance with this aspect of the invention, it is possible to remove at least 60%, preferably at least 75%, more preferably at least 85%, still more preferably at least 95%, and most preferably at least 99% of the dipeptidyl aminopeptidase activity. It is contemplated that a<br><br>
complete removal of dipeptidyl aminopeptidase activity may be obtained by use of these methods.<br>
The methods used for cultivation and purification of the product of interest may be performed by methods known in the art.<br>
The methods of the present invention for producing an essentially dipeptidyl aminopeptidase-free product is of particular interest in the production of eukaryotic polypeptides, in particular fungal proteins such as enzymes. The enzyme may be selected from, e.g,, an amylolytic enzyme, a lipolytic enzyme, a proteolytic enzyme, a cellulytic enzyme, an oxidoreductase or a plant cell-wall degrading enzyme. Examples of such enzymes include an aminopeptidase, an amylase, an amyloglucosidase, a carbohydrase, a carboxypeptidase, a catalase, a cellulase, a chitinase, a cutinase, a cyclodextrin glycosyltransferase, a deoxyribonuclease, an esterase, a galactosidase, a beta-galactosidase, a glucoamylase, a glucose oxidase, a glucosidase, a haloperoxidase, a hemicellulase, an invertase, an isomerase, a laccase, a ligase, a lipase, a lyase, a mannosidase, an oxidase, a pectinolytic enzyme, a peroxidase, a phytase, a phenoloxidase, a polyphenoloxidase, a proteolytic enzyme, a ribonuclease, a transferase, a transglutaminase, or a xylanase. The dipeptidyl aminopeptidase-deficient cells may also be used to express heterologous proteins of pharmaceutical interest such as hormones, growth factors, receptors, and the like.<br>
It will be understood that the term "eukaryotic polypeptides" includes not only native polypeptides, but also those polypeptides, e.g., enzymes, which have been modified by amino acid substitutions, deletions or additions, or other such modifications to enhance activity, thermostability, pH tolerance and the like.<br>
In a further aspect, the present invention relates to a protein product essentially free from dipeptidyl aminopeptidase activity which is produced by a method of the present invention.<br>
Methods of Producing Protein Hydrolysates<br>
The polypeptides of the present invention may be used in the production of protein hydrolysates for enhancing the degree of hydrolysis and flavor development.<br><br>
The present invention further relates to methods for using a polypeptide of the present invention in combination with an endopeptidase to produce a high degree of hydrolysis of a<br>
0<br>
protein-rich material. The method comprises treating of a proteinaceous substrate with the polypeptide and an endopeptidase. The substrate may be treated with the enzymes concurrently or consecutively.<br>
A polypeptide of the present invention is added to the proteinaceous substrate in an effective amount conventionally employed in protein hydrolysis processes, preferably in the range of from about 0.1 to about 100,000 dipeptidyl aminopeptidase units (DPAPU) per 100 g of protein, and more preferably in the range of from about 1 to about 10,000 dipeptidyl aminopeptidase units per 100 g of protein. As defined herein, one dipeptidyl aminopeptidase Unit (DPAPU) is the amount of enyzme needed to release 1 micromole of p-nitroanilide per minute from Ala-Pro-p-nitroanilide (Sigma Chemical Co., St, Louis MO) under specified conditions.<br>
The endopeptidase may be obtained from a strain of Bacillus, preferably Bacillus licheniformis or Bacillus subtilis, a strain of Staphylococcus, preferably Staphylococcus aureus, a strain of Streptomyces, preferably Streptomyces thermovularis or Streptomyces griseus, a strain of Actinomyces species, a strain of Aspergillus, preferably Aspergillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus nidulans, Aspergillus niger, or Aspergillus oryzae, or a strain of Fusarium, preferably Fusarium venenatum.<br>
The endopeptidase is added to the proteinaceous substrate in an effective amount conventionally employed in protein hydrolysis processes, preferably in the range of from about 0.05 to about 15 AU/100 g of protein, and more preferably from about 0.1 to about 8 AU/lOO g of protein. One AU (Anson Unit) is defined as the amount of enzyme which under standard conditions (i,e,, 25Â°C, pH 7.5 and 10 min. reaction time) digests hemoglobin at an initial rate such that there is liberated per minute an amount of TCA soluble product which gives the same color with phenol reagent as one milli-equivalent of tyrosine. The analytical method AF 4/5 is available upon request from Novo Nordisk A/S, Denmark, which is incorporated herein by reference.<br><br>
The enzymatic treatment, i.e,, the incubation of the substrate with the enzyme preparations, may take place at any convenient temperature at which the enzyme preparation does not become inactivated, preferably in the range of from about 20Â°C, to about 70Â°C. In accordance with established practice, the enzyme preparations may be suitably inactivated by increasing the temperature of the incubation mixture to a temperature where the enzymes become inactivated, e.g., to above about 70Â°C, or similarly by decreasing the pH of the incubation mixture to a point where the enzymes become inactivated, e.g., below about 4.0.<br>
Furthermore, the methods of the present invention result in enhancement of the degree of hydrolysis of a proteinaceous substrate. As used herein, the degree of hydrolysis (DH) is the percentage of the total number of amino bonds in a protein that has been hydrolyzed by a proteolytic enzyme.<br>
In another aspect of the present invention, the hydrolysates have an increased content of Ala, Arg, Asp, Gly, and/or Val, e.g,, 1.1 times greater.<br>
In another aspect of the present invention, a polypeptide of the present invention acts synergistically with an aminopeptidase to hydrolyze a tripeptide where either enzyme activity alone does not hydrolyze the tripeptide. In a preferred embodiment, the aminopeptidase is aminopeptidase I obtained from Aspergillus oryzae as described in WO 96/28542.<br>
The present invention also relates to methods for obtaining a protein hydrolysate enriched in free glutamic acid and/or peptide bound glutamic acid residues, which method comprises:<br>
(a)	subjecting the substrate to a deamidation process; and<br>
(b)	subjecting the substrate to the action of a polypeptide having dipeptidyl<br>
aminopeptidase activity.<br>
The two steps may be performed simultaneously, or the second step may be performed subsequent to the first step.<br>
These methods of the present invention produce protein hydrolysates of excellent flavor because glutamic acid (Glu), whether free or peptide bound, plays an important role in the flavor and palatability of protein hydrolysates. These method also produce protein hydrolysates having improved functionality, in particular, improved solubilitv. improved<br><br>
emulsifying properties, increased degree of hydrolysis, and improved foaming properties.<br>
The conversion of amides (glutamine or asparagine) into charged acids (glutamic acid or aspartic acid) via the liberation of ammonia is known as deamidation. Deamidation may take place as a non-enzymatic or as an enzymatic deamidation process.<br>
In a preferred embodiment, the deamidation is carried out as an enzymatic deamidation process, e.g., by subjecting the substrate to a transglutaminase and/or peptidoglutaminase.<br>
The transglutaminase may be of any convenient source including mammals, see e.g., JP 1050382 and JP 5023182, including activated Factor XIII, see e.g., WO 93/15234; those derived from fish, see e.g., EP 555,649; and those obtained from microorganisms, see e.g., EP 379,606, WO 96/06931 and WO 96/22366. In a preferred embodiment, the transglutaminase is obtained from an Oomycete, including a strain of Phytophthora, preferably Phytophthora cactorum, or a strain of Pythium, preferably Pythium irregulare, Pythium sp., Pythium intermedium, Pythium ultimum, or Pythium periilum (or Pythium periplocum). In another preferred embodiment, the transglutaminase is of bacterial origin and is obtained from a strain of Bacillus, preferably Bacillus subtilis, a strain of Streptoverticillium, preferably Streptoverticillium mobaraensis, Streptoverticillium griseocameum, or Streptoverticillium cinnamoneum, and a strain of Streptomyces, preferably Streptomyces lydicus.<br>
The peptidoglutaminase may be a peptidoglutaminase I (peptidyl-glutaminase; EC 3.5.1.43), or a peptidoglutaminase II (protein-glutamine giutaminase; EC 3.5.1.44), or any mixture thereof. The peptidoglutaminase may be obtained from a strain of Aspergillus, preferably Aspergillus japonicus, a strain of Bacillus, preferably Bacillus circulans, a strain of Cryptococcus, preferably Cryptococcus albidus, or a strain of Debaryomyces, preferably Debaryomyces kloecheri.<br>
The transglutaminase is added to the proteinaceous substrate in an effective amount conventionally employed in deamidation processes, preferably in the range of from about 0.01 to about 5% (w/w), and more preferably in the range of from about 0.1 to about 1% (w/w) of enzyme preparation relating to the amount of substrate.<br>
The peptidoglutaminase is added to the proteinaceous substrate in an effective amount conventionally employed in deamidation processes, preferably in the range of from about 0.01<br><br>
to about 100,000 PGase Units per 100 g of substrate, and more preferably in the range of from about 0.1 to about 10,000 PGase Units per 100 g of substrate.<br>
The peptidoglutaminase activity may be determined according to the procedure of Cedrangoro et al (1965, Enzymologia 29: 143). According to this procedure, 0.5 ml of an enzyme sample, adjusted to pH 6.5 with 1 N NaOH, is charged into a small vessel. Then 1 ml of a borate pH 10.8 buffer solution is added to the vessel. The discharged ammonia is absorbed by 5 N sulphuric acid, and by use of Nessler's reagent the mixture is allowed to form color which is measured at 420 nm. One PGase unit is the amount of enzyme capable of producing 1 micromole of ammonia per minute under these conditions.<br>
Alternatively, the peptidoglutaminase activity may be determined according to the procedure described in US 3,857,967 or Example 17 below.<br>
In step (b) of the methods of the present invention, the substrate is subjected to a polypeptide of the present invention. A polypeptide of the present invention is added to the proteinaceous substrate in an effective amount conventionally employed in protein hydrolysis processes, preferably in the range of from about 0.001 to about 0.5 AU/100 g of substrate, more preferably in the range of from about 0.01 to about 0.1 AU/100 g of substrate.<br>
In another embodiment, the methods of the present invention may be used to produce a hydrolysate enriched in free glutamic acid and/or peptide bound glutamic acid residues further comprise:<br>
(c) subjecting the substrate to one or more unspecific acting endo- and/or exo-peptidase enzymes.<br>
This step may take place simultaneously with steps (a) and (b), or may follow steps (a) and (b).<br>
In a preferred embodiment, the unspecific acting endo- and/or exo-peptidase enzyme is obtained from a strain of Aspergillus, preferably Aspergillus niger, Aspergillus oryzae, or Aspergillus sojae, or a strain of Bacillus, preferably Bacillus amyloliquefaciens. Bacillus lentus. Bacillus licheniformis, or Bacillus subtilis.<br>
The unspecific acting endo- and/or exo-peptidase enzyme is added to the substrate in an effective amount conventionally employed in protein hydrolysis processes, preferably in the<br><br>
range of from about 0.05 to about 15 CPU/100 g of substrate, and more preferably in the range of from about 0.1 to about 5 CPU/100 g of substrate. One CPU (Casein Protease Unit) is defined as the amount of enzyme liberating 1 micromole of primary amino groups (determined by comparison with a serine standard) per minute from casein under standard conditions, i.e., incubation for 30 minutes at 25'Â°C,and pH 9.5. The analytical method AF 228/1, which is incorporated herein by reference, is available upon request from Novo Nordisk A/S, Bagsvaerd, Denmark.<br>
Each enzymatic treatment may take place at any temperature at which the enzyme preparation does not become inactivated, preferably in the range of from about 20Â°C to about 70Â°C. The enzyme preparation may then be inactivated by increasing the temperature, e.g., to above about 70Â°C, or by decreasing the pH, e.g., below about 4.0.<br>
The proteinaceous substrate used in the methods of the present invention may consist of intact proteins, prehydrolyzed proteins (i.e., peptides), or a mixture thereof. The proteinaceous substrate may be of vegetable or animal origin. Preferably, the proteinaceous substrate is of vegetable origin, e.g., soy protein, grain protein, e,g,, wheat gluten, com gluten, barley, rye, oat, rice, zein, lupine, cotton seed protein, rape seed protein, peanut, alfalfa protein, pea protein, fabaceous bean protein, sesame seed protein, or sunflower. A proteinaceous substrate of animal origin may be whey protein, casein, meat proteins, fish protein, red blood cells, egg white, gelatin, or lactoalbumin.<br>
The present invention also relates to protein hydrolysates produced by these methods.<br>
Other Uses<br>
The present invention also relates to methods of deactivating enzymes with a polypeptide of the present invention.<br>
Furthermore, a polypeptide of the present invention may be useful for a number of purposes in which a specific cleavage of peptide sequences is desirable. For instance, some proteins or peptides are synthesized in the form of inactive precursors comprising a number of additional amino acid residues at the N-terminal of the mature protein. A polypeptide of the present invention could provide the necessary post-translational processing to activate such<br><br>
precursor proteins.<br>
Compositions<br>
In a still further aspect, the present invention relates to polypeptide compositions comprising a polypeptide of the present invention. Preferably, the compositions are enriched in a polypeptide of the present invention. In the present context, the term "enriched" indicates that the dipeptidyl aminopeptidase activity of the polypeptide composition has been increased, e.g., with an enrichment factor of 1.1.<br>
The polypeptide composition may comprise a polypeptide of the invention as the major<br>
enzymatic component, e.g., a mono-component polypeptide composition. Alternatively, the<br>
composition may comprise multiple enzymatic activities, such as an aminopeptidase, an<br>
amylase, a carbohydrase, a carboxypeptidase, a catalase, a cellulase, a chitinase, a cutinase, a<br>
cyclodextrin glycosyltransferase, a deoxyribonuclease, an esterase, an alpha-galactosidase, a<br>
beta-galactosidase, a glucoamylase, an alpha-glucosidase, a beta-glucosidase, a<br>
haloperoxidase, an invertase, a laccase, a lipase, a mannosidase, an oxidase, a pectinolytic<br>
enzyme, a peptidoglutaminase, a peroxidase, a phytase, a polyphenoloxidase, a proteolytic<br>
enzyme, a ribonuclease, a transglutaminase, or a xylanase. The additional enzyme(s) may be<br>
producible by means of a microorganism belonging to the genus Aspergillus, preferably<br>
Aspergillus aculeatus, Aspergillus awamori, Aspergillus niger, or Aspergillus oryzae, or<br>
Trichoderma, Humicola, preferably Humicola insolens, or Fusarium, preferably Fusarium<br>
bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium<br>
graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium<br>
oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium<br>
sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum,<br>
Fusarium trichothecioides, or Fusarium venenatum.	<br>
In a preferred embodiment, the invention relates to a flavor-improving composition comprising a polypeptide with dipeptidyl aminopeptidase activity and a carrier. Any suitable carrier known in the art may be used. In another preferred embodiment, the flavor-improving composition further comprises an endopeptidase.    In another preferred embodiment, the<br><br>
flavoring composition further comprises one or more unspecific-acting endo- and/or exo peptidase enzymes.    In another preferred embodiment, the flavoring composition further comprises one or more specific-acting endo- and/or exo-peptidase enzymes.<br>
In a preferred embodiment, the specific acting proteolytic enzyme is an endopeptidase such as a glutamyl endopeptidase (EC 3.4.21.19); a lysyl endopeptidase (EC 3.4.21.50); a leucyl endopeptidase (EC 3.4.21.57); a glycyl endopeptidase (EC 3.4.22.25); a prolyl endopeptidase (EC 3.4.21.26); trypsin (EC 3.4.21.4) or a trypsin-like (lysine/arginine specific) endopeptidase; or a peptidyl-Asp metalloendopeptidase (EC 3.4.24.33).<br>
The glutamyl endopeptidase (EC 3.4.21.19) may preferably be obtained from a Bacillus strain, in particular Bacillus licheniformis and Bacillus subtilis, a Staphylococcus strain, in particular Staphylococcus aureus, a Streptomyces strain, in particular Streptomyces thermovulgaris and Streptomyces griseus, or a. Actinomyces strain.<br>
The lysyl  endopeptidase  (EC  3.4.21.50)   may  preferably  be  obtained  from  a Achromobacter strain, in particular Achromobacter lyticus, a Lysobacter strain, in particular Lysobacter enzymogenes, or a Pseudomonas strain, in particular Pseudomonas aeruginosa. The leucyl endopeptidase (EC 3.4.21.57) may be of plant origin. The glycyl endopeptidase (EC 3.4.22.25) may preferably be obtained from the papaya plant {Carica papaya).<br>
The prolyl endopeptidase (EC 3.4.21.26) may preferably be obtained from a Flavobacterium strain, or it may be of plant origin.<br>
The trypsin-like endopeptidase may preferably be obtained from a Fusarium strain, in particular Fusarium oxysporum, e.g., as described in WO 89/06270 or WO 94/25583.<br>
The peptidyl-Asp metalloendopeptidase (EC 3.4.24.33) may preferably be obtained from a Pseudomonas strain, in particular Pseudomonas fragi.<br>
In another preferred embodiment, the specific acting proteolytic enzyme is an exo-peptidase that may act from either end of the peptide.<br>
In a preferred embodiment, the specific acting proteolytic enzyme is an aminopeptidase such as a leucyl aminopeptidase (EC 3.4.11.1); or a tripeptide aminopeptidase (EC 3.4.11.4).<br><br>
In another preferred embodiment, the specific acting proteolytic enzyme is a carboxypeptidase such as a proline carboxypeptidase (EC 3.4.16.2); a carboxypeptidase A (EC 3.4.17.1); a carboxypeptidase B (EC 3.4.17.2); a carboxypeptidase C (EC 3.4.16.5); a carboxypeptidase D (EC 3.4.16.6); a lysine (arginine) carboxypeptidase (EC 3.4.17.3); a glycine carboxypeptidase (EC 3.4.17.4); an alanine carboxypeptidase (EC 3.4.17.6); a glutamate carboxypeptidase (EC 3.4.17.11); a peptidyl-dipeptidase A (EC 3.4.15.1); or a peptidyl-dipeptidase (EC 3.4.15.5).<br>
The polypeptide compositions may be prepared in accordance with methods known in the art and may be in the form of a liquid or a dry composition. The polypeptide may be stabilized by methods known in the art.<br>
The present invention also relates to food products, e.g., baked products, comprising a protein hydrolysate obtained by the methods of the present invention. Such food products exhibit enhanced organoleptic qualities, such as improvement in flavor, palatability, mouth feel, aroma and crust color.<br>
In the present context, the term "baked products" includes any food prepared from dough, either of a soft or a crisp character. Examples of baked products, whether of a white, light or dark type, which may be advantageously produced by the present invention, are bread, in particular white, whole-meal or rye bread, typically in the form of loaves or rolls; French baguette-type breads; pita breads; tacos; cakes; pancakes; biscuits; crisp breads; and the like.<br>
Such baked products are conventionally prepared from a dough which comprises flour and water, and which is typically leavened. The dough may be leavened in various ways, such as by adding sodium bicarbonate or the like, or by adding a leaven (fermenting dough), but the dough is preferably leavened by adding a suitable yeast culture such as a culture of Saccharomyces cerevisiae (baker's yeast). Any of the commercially available Saccharomyces cerevisiae strains may be employed.<br>
Further, the dough used in the preparation of the baked products may be fresh or frozen. The preparation of frozen dough is described by K. Kulp and K. Lorenz in "Frozen and Refrigerated Doughs and Batters". A flavor improving composition of the present invention is typically included in the dough in an amount in the range of 0.01-5%, more<br><br>
preferably 0.1-3%.<br>
In the methods of the present invention, a polypeptide of the present invention, an endopeptidase, a transglutaminase, a peptidoglutaminase, one or more specific and/or unspecific acting endo- and/or exo-peptidase enzymes, and/or one or more enzymes specified above may be added, either separately or concurrently, to the mixture from which the dough is made or to any ingredient, e.g., flour, from which the dough is to be made.<br>
The present invention further relates to a pre-mix, e.g., in the form of a flour composition, for dough and/or baked products made from dough, wherein the pre-mix comprises a polypeptide or a flavor-improving composition of the invention and a carrier or baking ingredient, and optionally one or more other enzymes specified above.<br>
In another embodiment, the pre-mix comprises a hydrolysate obtained by the methods of the invention.<br>
The pre-mix may be prepared by mixing the relevant enzymes with a suitable carrier such as flour, starch, a sugar or a salt. The pre-mix may contain other dough-improving and/or bread-improving additives.<br>
In the present context, the term "pre-mix" is a mixture of baking agents, normally including flour, which has been prepared to permit storage under designated conditions and provide convenience in handling during dough preparation processes. Such a pre-mix may be of advantageous use in industrial and commercial bread-baking plants and facilities, as well as in retail bakeries.<br>
The present invention also relates to the use of a hydrolysate produced by the methods of the invention as an additive to food products, such as baked foods, to enhance organoleptic qualities, such as flavor, palatability and aroma.<br>
The hydrolysates enriched in free glutamic acid and/or peptide bound glutamic acid residues obtained by the methods of the present invention may becaused in various industrial applications, in particular, where there is a need for the incorporation of functional proteins.<br>
For example, the present invention also relates to food products comprising a hydrolysate enriched in free glutamic acid and/or peptide bound glutamic acid residues obtained by the method of the invention and to animal feed additives comprising a hydrolysate enriched<br><br>
in free glutamic acid and/or peptide bound glutamic acid residues obtained by the methods of the present invention.<br>
The present invention is further described by the following examples which should not be construed as limiting the scope of the invention.<br>
Examples<br>
Example 1: Purification of FLAVOURZYMEâ¢ dipeptidyl aminopeptidase I<br>
Dipeptidyl aminopeptidase I was purified from a FLAVOURZYMEâ¢ broth (Novo Nordisk A/S, Bagsvasrd, Denmark). The FLAVOURZYMEâ¢ broth was produced by cultivation of Aspergillus oryzae strain 1568 (ATCC 20386) in a medium comprised of carbon and nitrogen sources and trace metals. First, the broth (20 ml containing 720 mg of protein) was diluted with 180 ml of 20 mM sodium phosphate pH 7.0 buffer and filtered using Nalgene<br>
Filterware equipped with a 0.45Âµm filter.  The filtered solution was loaded onto a 24 x 130<br>
mm column containing 31 ml of Q-Sepharose, Big Beads (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The dipeptidyl aminopeptidase I was eluted using 20 mM sodium acetate buffer, pH 3.5. Dipeptidyl aminopeptidase I activity was monitored at 405 nm using 1 mg/ml Ala-Pro-p-nitroanilide as substrate in 50 mM sodium phosphate pH 7.5 buffer. The resultant solution containing dipeptidyl aminopeptidase I activity was concentrated to 38 ml by ultrafiltration with a PM10 membrane (Amicon, New Bedford, MA) and then the pH was adjusted to pH 7.0 using a 20 mM Na2HPO4 solution.<br>
The resultant solution was loaded onto a 24 x 130 mm column containing 31 ml of Q-Sepharose Big Beads (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The dipeptidyl aminopeptidase was eluted with a 0 to 0.3 M NaCl gradient in 20 mM sodium phosphate pH 7.0 buffer. Fractions were monitored for dipeptidyl aminopeptidase I activity as described above. The fractions containing dipeptidyl aminopeptidase I activity were collected, pooled, desalted and concentrated using ultrafiltration<br><br>
against 20 mM sodium phosphate pH 7.0 buffer.<br>
The resultant solution was loaded onto a MonoQ 16/10 (20 ml) pre-packed column (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM sodium phosphate pH 7.0 buffer. The dipeptidyl aminopeptidase I was eluted with a 0 to 0.27 M NaCl gradient in 20 mM sodium phosphate pH 7.0 buffer. Fractions were monitored for dipeptidyl aminopeptidase I activity as described above. The fractions between 0.200 and 0.212 M NaCl were collected, pooled, and rebuffered with 1.7 M (NH4)2SO4/20 mM sodium phosphate pH 7.0 buffer using ultrafiltration as described above.<br>
The resultant solution was loaded onto a 7 x 50 mm column containing Phenyl Superose resin (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 1.7 M (NH4)2SO4/20 mM sodium phosphate pH 7.0 buffer solution. The dipeptidyl aminopeptidase I was eluted with a reverse 1.7 to 0 M (NH4)2SO4 gradient in 20 mM sodium phosphate pH 7.0 buffer. Fractions were monitored for dipeptidyl aminopeptidase I activity as described above. Two fractions possessing highest activity toward Ala-Pro-/7-nitroanilide were found to be at least 95% homogeneous based on SDS-PAGE analysis. The major band had a molecular weight of approximately 95 kDa (range of 93-96 kDa).<br>
Example 2:   Protein sequencing and amino acid analysis methods<br>
N-terminal sequencing of the partially purified dipeptidyl aminopeptidase I described in Example 1 and a digested fragment of the dipeptidyl aminopeptidase I was performed on an Applied Biosystems 476A Protein Sequencer (Perkin Ehner/Applied Biosystems Division, Foster City, CA) with on-line HPLC and liquid phase trifluoroacetic acid (TFA) delivery. Samples of the purified dipeptidyl aminopeptidase I were transblotted onto Novex PVDF membranes (Novex, San Diego, CA) from SDS-PAGE gels and sequenced from a blott cartridge using sequencing reagents (Perkin Elmer/Applied Biosystems Division, Foster City, CA). Detection of phenylthiohydantoin-amino acids was accomplished by on-line HPLC using Buffer A containing 3.5% tetrahydrofuran in water with 15 ml of the Premix concentrate (Perkin Elmer/Applied Biosystems Division, Foster City, CA) containing acetic acid, sodium acetate, and sodium hexanesulfonate and Buffer B containing acetonitrile.  Data was collected<br><br>
f<br><br>
ana analyzed on a Macintosh llsi using Applied Biosystems 610 Data Analysis software.<br>
The partially purified dipeptidyl aminopeptidase I (MonoQ peak of Example 1) was also subjected to in-gel digestion with trypsin to generate peptide fragments of the enzyme for protein sequencing. The trypsin-digested dipeptidyl aminopeptidase I was separated by SDS-PAGE electrophoresis using 8-16% Novex Tris-glycine gels (Novex, San Diego, CA). Eight Coommassie blue stained gel pieces corresponding to a molecular weight of 95 kDa were excised and washed extensively with 200 mM NH4HCO3 in 50% acetonitrile. The gel pieces were subsequently reduced in 1 mM dithiothreitol (DTT) at 37Â°C for 20 minutes and alkylated with an equal volume of 100 mM iodoacetic acid at room temperature for 20 minutes in the dark. Gel pieces were washed repeatedly with 200 mM NH4HCO3 in 50% acetonitrile. Supematants were removed and gel pieces were dried on a Speed-Vac (Savant Instruments, Farmingdale, NY). The gel pieces were rehydrated in a solution containing 0.033 mg of sequencing grade modified porcine trypsin (Promega, Madison, WI) per ml of 135 mM NH4HCO3. The solution was prepared by diluting 1 part of 0.1 mg of the trypsin per ml of trypsin resuspension buffer (Promega, Madison, WI) into 2 parts of 2(X) mM NH4HCO3.  The<br>
gel pieces were incubated at 37Â°C for 20 hours. The peptide fragments were extracted from the<br>
gel in repetitive washes of 0.1% TFA in 60% acetonitrile for 1 hour each. The extracted peptides were dried and reconstituted in 0.05% TFA in 25% acetonitrile and then further diluted<br>
into 0.05% TFA.  The peptide fragments were filtered using a Micropure 0.45Âµm filter unit<br>
(Amicon, Inc., Beverly, MA). The peptide fragments were then separated by reverse-phase HPLC using a Hewlett-Packard 1090L HPLC equipped with a 2.1 x 250 mm Vydac C18-RP column (5 micron). A step gradient was used with 0.1% TFA in 80% acetonitrile as the eluant. The peptide samples were hand collected and then subjected to N-terminal sequencing.<br>
The partially purified dipeptidyl aminopeptidase I was also subjected to cyanogen bromide to generate peptide fragments of the enzyme for sequencing. The dipeptidyl aminopeptidase I was digested with cyanogen bromide by reconstituting a dried sample of the partially purified n 70% formic acid with a few crystals of cyanogen bromide and incubating for 18 hours at room temperature in the dark.  The peptide fragments were separated by SDS-<br><br>
PAGE electrophoresis using 10-20% Novex Tricine gels (Novex,  San Diego, CA) and sequenced as described above.<br>
N-terminal sequencing of the dipeptidyl aminopeptidase I revealed that the N-terminus was apparently blocked. A weak sequence was obtained as follows where amino acid residues in parentheses are not 100% certain and residues marked with an X could not be determined: Peptide 1: XEGSKRLTFXETVVKQAIT(P) (SEQ ID N0:3)<br>
The cyanogen bromide-degraded fragments had the following amino acid sequences where amino acid residues underlined matched 100% with a dipeptidyl aminopeptidase I of Saccharomyces cerevisiae (Anna-Arriola and Herskowitz, 1994, Yeast 10: 801-810; Galisson and Dujon, 1996, Yeast 12: 877-885): Peptide 2: QRLPPGFSPDICKYPILFTPYGG (SEQ ID NO:4) Peptides: KYIGPIK(SEQIDN0:5) Peptide 4: GEGSKRL (SEQ ID NO:6)<br>
In-gel digestion with trypsin produced the following peptides where amino acid residues underlined matched 100% with the deduced amino acid sequence of the Aspergillus oryzae dipeptidyl aminopeptidase I nucleic acid sequence described in Example 7: Peptides: XPILFTPY(SEQ ID NO:7) Peptide 6: XVPLMPDO(Q)GDIOYAO (SEP ID N0:8)<br>
Example 3:   Genomic DNA extraction<br>
Aspergillus oryzae 1568 was grown in 25 ml of 0.5% yeast extract-2% glucose (YEG) medium for 24 hours at 37Â°C and 250 rpm. Mycelia were then collected by filtration through Miracloth (Calbiochem, La Jolla, CA) and washed once with 25 ml of 10 mM Tris-1 mM EDTA (TE) buffer. Excess buffer was drained from the mycelia preparation which was subsequently frozen in liquid nitrogen. The frozen mycelia preparation was ground to a fine powder in an electric coffee grinder, and the powder was added to a disposable plastic centrifuge tube containing 20 ml of TE buffer and 5 ml of 20% w/v sodium dodecylsulfate (SDS). The mixture was gently inverted several times to ensure mixing, and extracted twice with an equal volume of phenol:chloroform:isoamyl alcohol (25:24:1 v/v/v). Sodium acetate (3<br><br>
M solution) was added to the extracted sample to a final concentration of 0.3 M followed by 2.5 volumes of ice cold ethanol to precipitate the DNA. The tube was centrifuged at 15,000 x g for 30 minutes to pellet the DNA. The DNA pellet was allowed to air-dry for 30 minutes before resuspension in 0.5 ml of TE buffer. DNase-free ribonuclease A was added to the resuspended DNA pellet to a concentration of 100 M-g/ml and the mixture was then incubated at 37Â°C for 30 minutes Proteinase K (200 Âµg/ml) was added and the tube was incubated an additional one hour at 37Â°C. Finally, the sample was extracted twice with phenol:chloroform:isoamyl alcohol and the DNA precipitated with ethanol. The precipitated DNA was washed with 70% ethanol, dried under vacuum, resuspended in TE buffer, and stored at 4oC.<br>
Example   4:      PCR   amplification   of   Aspergillus    oryzae   1568    dipeptidyl aminopeptidase I<br>
The forward degenerate oligonucleotide primer was designed to the peptide sequence DW(W)YEEE, a conserved motif found in most published dipeptidyl aminopeptidase I protein sequences. The reverse degenerate oligonucleotide primer was designed to a partial peptide, PPGFSDKKYP, of peptide 2 (SEQ ID N0:4) as described in Example 2. The degenerate oligonucleotide primers shown below were synthesized with an Applied Biosystems Model 394 DNA/RNA Synthesizer, according to the manufacturer's instructions, for use to PCR amplify dipeptidyl aminopeptidase I gene fragments from Aspergillus oryzae 1568 genomic DNA:<br>
Forward primer: 5'-GAYTGGrnTAYGARGARGAR-3' (SEQ ID NO:9) Reverse primer: 5'-GGRTAYTTYTTRTCIGGISWRAAICCIGGIGG-3' (SEQ ID NO: 10)<br>
(R= A or G, Y= C or T, S= G or C, W= A or T, 1= Inosine)<br>
Amplification reactions (100 Âµl) were prepared using approximately 1 Âµg of genomic DNA isolated from an Aspergillus oryzae 1568 as described in Example 3 as the template. Each reaction contained the following components:   1 fig genomic DNA, 40 pmol forward primer, 40 pmol reverse primer, 200 ÂµM each of dATP, dCTP, dGTP, and dTTP, 1 x Taq polymerase buffer (Perkin-Elmer Corp., Branchburg, NJ), and 2.5 Units of Taq polymerase<br><br>
(Perkin-Elmer Corp., Branchburg, NJ). The reactions were incubated in a Perkin-Elmer Model 480 Thermal Cycler programmed as follows: Cycle 1 at 94Â°C for 2 minutes, 45Â°C for one minutes, and 72Â°C for one minute; and cycles 2-30 each at 94Â°C for one minute, 45Â°C for one minute, and 72Â°C for one minute. The reaction products were isolated on a 1% agarose gel (Eastman Kodak, Rochester, NY). An approximately 1.0 kb product band was excised from the gel and purified using GenElute spin columns (Supelco, Bellefonte, PA) according to the manufacturer's instructions. The purified PCR product was subsequently cloned into a pCRII vector (Invitrogen, San Diego, CA) and the DNA sequence was determined using lac forward and reverse primers (New England BioLabs, Beverly, MA).<br>
A dipeptidyl aminopeptidase I gene segment consisting of approximately 321 codons (963bp) was amplified from Aspergillus oryzae 1568 with the dipeptidyl aminopeptidase I PCR primers described above. DNA sequence analysis shows that the amplified gene segment encoded a portion of the corresponding Aspergillus oryzae 1568 dipeptidyl aminopeptidase I gene. The dipeptidyl aminopeptidase I gene segment was used to probe an Aspergillus oryzae 1568 genomic DNA library.<br>
Example 5:    Construction of DNA libraries<br>
A genomic DNA library was constructed in the bacteriophage cloning vector AZipLox<br>
(Life Technologies, Gaithersburg, MD). First, total cellular DNA was partially digested with Tsp509I and size-fractionated on 1% agarose gels. DNA fragments migrating in the size range 3-7 kb were excised and eluted from the gel using Prep-a-Gene reagents (BioRad Laboratories, Hercules,   CA).     The  eluted  DNA  fragments  were  ligated   with   EcoRI-cleaved   and<br>
dephosphorylated AZipLox vector arms (Life Technologies, Gaithersburg, MD), and the<br>
ligation mixtures were packaged using commercial packaging extracts (Stratagene, La JoUa, CA). The packaged DNA libraries were plated and amplified in Escherichia coli Y10902ZL cells (Life Technologies, Gaithersburg, MD). The unamplified genomic DNA library contained 3.1 X 106 pfu/ml (background titers with no DNA were 2.0 X 104 pfu/ml).<br><br>
Example 6:    Identification of dipeptidyl aminopeptidase I clones<br>
Approximately 10,000 plaques from the library described in Example 5 were screened by plaque-hybridization using the dipeptidyl aminopeptidase I PCR fragment from Aspergillus oryzae 1568 as the probe. The DNA was cross-linked onto membranes (Hybond N+, Amersham, Arlington Heights, IL) using a UV Stratalinker (Stratagene, La Jolla, CA). The membranes were soaked for three hours at 45Â°C in a hybridization solution containing 5 x<br>
SSPE, 0.3% SDS, 50% formamide, and 10Âµg/ml of denatured and sheared herring sperm<br>
DNA. The dipeptidyl aminopeptidase I gene fragment isolated from the Aspergillus oryzae 1568 genomic DNA as described in Example 2 was radiolabeled using the Random Primed DNA Labeling Kit (Boehringer Mannheim, Mannheim, Germany), denatured by adding NaOH to a final concentration of O.IM, and added to the hybridization solution at an activity of approximately 1 x 106 cpm per ml of hybridization solution. The mixture was incubated overnight at 45Â°C in a shaking water bath. Following incubation, the membranes were washed once in 2 x SSC with 0.2% SDS at 55Â°C followed by two washes in 2 x SSC at the same temperature. The membranes were dried on blotting paper for 15 minutes, wrapped in Saran Wrapâ¢, and exposed to X-ray film ovemight at -70Â°C with intensifying screens (Kodak, Rochester, NY).<br>
Based on the production of strong hybridization signals with the probe, three plaques,<br>
designated E. coli DH5a MWR52A, E. coli DH5a MWR52B, and E, coli DH5a MWR52C<br>
were chosen for further study. The three plaques were purified twice in E, coli Y1090ZL cells and the dipeptidyl aminopeptidase I genes were subsequently excised from the AZipLox vector<br>
as pZLl-derivatives (D'Alessio et al., 1992, FocusÂ® 14:76) using in vivo excision by infection of E, coli DHIOBZL cells (Life Technologies, Gaithersburg,-MD).    The three plasmid<br>
containing colonies were inoculated into three ml of LB plus 50Âµg/ml carbenicillin medium and<br>
grown ovemight at 37Â°C.  Miniprep DNA was prepared from each of these cultures using the Wizard 373 DNA Purification Kit (Promega, Madison, WI).  The dipeptidyl aminopeptidase I<br><br>
encoding plasmid (pMWR52) was confirmed by DNA sequencing.<br>
Example 7:    DNA  sequence   analysis   of Aspergillus   oryzae 1568   dipeptidyl aminopeptidase I gene<br>
DNA sequencing of the dipeptidyl aminopeptidase I gene contained on pMWR52 in E,<br>
coli DH5a MWR52 described in Example 6 was performed with an Applied Biosystems Model<br>
377 Automated DNA Sequencer (Applied Biosystems, Inc., Foster City, CA) on both strands using the primer walking technique with dye-terminator chemistry (Giesecke et al, 1992, Journal of Virology Methods 38: 47-60). Oligonucleotide sequencing primers were designed to complementary sequences in the dipeptidyl aminopeptidase I gene and were synthesized on an Applied Biosystems Model 394 DNA/RNA Synthesizer according to the manufacturer's instructions.<br>
The nucleotide sequence of the gene encoding the Aspergillus oryzae 1568 dipeptidyl aminopeptidase I is shown in Figure 1 (SEQ ID NO: 1). Sequence analysis of the cloned insert revealed an open reading frame of 2396 nucleotides (excluding the stop codon) interrupted by an 83 bp intron. The G+C content of this open reading frame was 55.3%. The deduced amino acid sequence encoded a protein of 771 amino acids (SEQ ID N0:2). Based on the rules of van Heijne (van Heijne, 1984, Journal of Molecular Biology 173: 243-251), the first 16 amino acids likely comprise a secretory signal peptide which directs the nascent polypeptide into the endoplasmic reticulum (boxed in Figure 1).<br>
The amino acid sequences of the partial peptides derived from the purified dipeptidyl aminopeptidase I as described in Example 2 are underlined in Figure 1 and are consistent with those found in the deduced amino acid sequence (SEQ ID N0:2) of the Aspergillus oryzae 1568 dipeptidyl aminopeptidase I cDNA.<br>
Using the Clustal alignment program (Higgins, 1989, CABfOS 5: 151-153) to compare the deduced amino acid sequence of the Aspergillus oryzae 1568 dipeptidyl aminopeptidase I to that of a Saccharomyces cerevisiae dipeptidyl aminopeptidase I (Anna-Arriola and Herskowitz, 1994, Yeast 10: 801-810) (SEQ ID NO: 11), a 23.2% identity was observed.<br><br>
Example    8:        Construction    of    a   Aspergillus     oryzae    1568     dipeptidyl aminopeptidase Fusarium expression vector<br>
The coding region of Aspergillus oryzae dipeptidyl aminopeptidase I was amplified and the resulting fragment was cloned into pDMlSl for expression in Fusarium, pDMlSl provides the Fusarium trypsin (SP387) promoter and terminator, and the bar selectable marker gene. Specifically, the fragment was amplified by PCR using a sense primer (PI) designed to the first in-frame ATG and extending 13 bp downstream and an antisense primer (P2) designed to a region of the transcriptional stop codon and extending 10 bp downstream. To facilitate the cloning of the amplified fragment the sense and antisense primers contain a Swal and a Pad restriction site, respectively. The oligonucleotide primers shown below were synthesized using an ABI Model 394 DNA/RNA Synthesizer (Applied Biosystems, Inc., Foster City, CA) according to the manufacturer's instructions.<br>
Swal PI: 5'-GATTTAAATCACCATGAAGGTACGTCAATTCCACTG-3' (SEQ ID N0:12)<br>
Pad P2: 5'-GTTAATTAATCTACTCCTCCAAGTCCTTCTTAGTCC-3' (SEQ ID NO: 13)<br>
The 50 Âµl PCR solution (10 mM Tris-HCl pH 8.3, 50 mM KCl, 1.5 mM MgCl,,<br>
0.01% w/v gelatin) contained approximately 200 ng of pMWR52 DNA, 200 ÂµM each of<br>
dATP, dCTP, dGTP, and dTTP, and 50 pmol of each PCR primer described above. Five<br>
units of PWO polymerase (Boehringer Mannheim, Indianapolis, IN) were added and the<br>
reaction was incubated at 95 Â°C for 3 minutes and cooled to 80Â°C. The reaction was then cycled<br>
30 times, each cycle at 95Â°C for 30 seconds, 57Â°C for 1 minute, and 72Â°C for 1 minute, in a<br>
Perkin-Elmer 9600 Themial Cycler. Following the last cycle, the reaction incubated for 5<br>
minutes at 72Â°C.	<br>
A predicted 2.4 kb fragment was isolated by digestion with Swal and Pad and was cloned into pDMlSl (Figure 2) digested with the same restriction endonucleases to create pMWR54 (Figure 3). To verify the fidelity of the cloned PCR fragment, the fragment was sequenced according to the method of Hattori and Sakaki (1986, Analytical Biochemistry 152:<br><br>
232-237)   using   an   automated  Applied  Biosystems  Model   373A   Sequencer   (Applied Biosystems, Inc., Foster City, CA) according to the manufacturer's instructions.<br>
Sequencing of the cloned dipeptidyl aminopeptidase amplified insert of pMWR54 confirmed that there were no differences in the sequence described in SEQ ID NO: 1.<br>
Example 9:   Transformation of Fusarium CCl-3 and analysis of transformants<br>
Fusarium strain CCl-3, a highly branched morphological mutant of Fusarium strain A3/5 (ATCC 20334) (Wiebe et aL, 1992, Mycological Research 96: 555-562; Wiebe et a/., 1991, Mycological Research 95: 1284-1288; Wiebe et al, 1991, Mycological Research 96: 555-562), was grown in a liquid medium containing Vogel's salts, (Vogel, 1964, Am, Nature 98: 435-446), 25 mM NaNO3, and 1.5% glucose for 4 days at 28Â°C and 150 rpm. Conidia were purified by filtration through 4 layers of cheesecloth and finally through one layer of Miracloth. Conidial suspensions were concentrated by centrifugation. Fifty ml of YPG medium comprised of 1% yeast extract, 2% bactopeptone, and 2% glucose were inoculated with approximately 108 conidia, and incubated for 14 hours at 24Â°C and 150 rpm. Resulting hyphae were trapped on a sterile 0.4 mm filter and washed successively with sterile distilled water and 1.0 M MgSO4. The hyphae were resuspended in 10 ml of NOVOZYM 234â¢ solution (2-10 mg/ml in 1.0 M MgSO4) and digested for 15-30 minutes at 34Â°C with agitation at 80 rpm. Undigested hyphal material was removed from the resulting protoplast suspension by successive filtration through 4 layers of cheesecloth and through Miracloth. Twenty ml of 1 M sorbitol were combined with the protoplast solution. After mixing, the protoplasts were pelleted by centrifugation and washed successively by resuspension and centrifugation in 20 ml of 1 M sorbitol and in 20 ml of STC (0.8 M sorbitol, 0.05 M Tris pH 8.0, 0.05 M CaCl2). The washed protoplasts were resuspended in 4 parts STC and 1 part SPTC (0.8 M sorbitol, 40% PEG 4000,0.05 M Tris pH 8.0, 0.05 M CaCl2) at a concentration of 5 x 107/ml.<br>
One hundred ml of protoplast suspension were added to  10Âµg of pMWR54 in<br>
polypropylene tubes (17 x 1(X) mm), mixed and incubated on ice for 30 minutes.  One ml of SPTC was mixed gently into the protoplast suspension and incubation was continued at room<br><br>
temperature for 20 minutes. 12.5 ml of molten solution (cooled to 40Â°C) consisting of IX Vogel's salts, 25 mM NaNO3, 0.8 M sucrose and 1% low melting agarose (Sigma Chemical Company, St. Louis, MO) were mixed with the protoplasts and then plated onto an empty 100 mm Petri plate. Incubation was continued at room temperature for 10 to 14 days. After incubation at room temperature for 24 hours, 12.5 ml of the identical medium plus 10 mg of<br>
BAST Aâ¢ (Hoechst Schering, Rodovre, Denmark) per ml were overlayed onto the Petri plate.<br>
BASTAâ¢ was extracted twice with phenol:chloroform;isoamyl alcohol (25:24:1), and once<br>
with chloroform:isoamyl alcohol (24:1) before use.<br>
After two weeks, 21 transformants were apparent.  A mycelial fragment from the edge of each transformant was transferred to individual wells of a 24 well plate containing<br>
Vogel's/BASTAâ¢ medium. The medium was composed per liter of 25 g of sucrose, 25 g of<br>
Noble agar, 20 ml of 50X Vogel's salts (Vogel, 1964, supra), 25 mM NaNO3, and 10 g of BASTAâ¢.    The plate was sealed in a plastic bag to maintain moisture and incubated<br>
approximately one week at room temperature.<br>
Example 10:   Expression of Aspergillus oryzae 1568 dipeptidyl aminopeptidase I in Fusarium<br>
A mycelial fragment from each of the 21 Fusarium CCl-3 transformants described in Example 9 was inoculated into 20 ml of M400Da medium composed per liter of 50 g of maltodextrin, 2.0 g of MgSO4-THp, 2.0 g of KH2PO4, 4.0 g of citric acid, 8.0 g of yeast extract, 2.0 g of urea, and 0.5 ml of trace metals solution and incubated for 5 days at 30Â°C and 2(X) rpm. The medium was adjusted to pH 6,0 with 5 N NaOH. The trace metals solution was compsoed per liter of 14.3 g of ZnS04-7H20, 2.5 g of CUSO4-5H2O, 0.5 g of NiCVeHp, 13.8 g of FeSO^4TH2O, 8.5 g of MnS04-H20, and 3.0 g of citric acid. The untransformed host was also run as a control. One ml of each culture supernatant was harvested at 5 days and<br>
stored at 4Â°C.   Dipeptidyi aminopeptidase I activity was determined by mixing 1Âµl of the<br><br>
enzyme supernatant with 200Âµl of a Ala-Pro-pNA substrate stock solution containing 2 mg of<br>
Ala-Pro-pNA per ml of 50 mM sodium phosphate pH 7.5 and monitoring the change in absorbance at 405 nm and ambient temperature.<br>
Culture supematants from 19 of the 21 primary transformants of pMWR54 were positive when assayed for dipeptidyl aminopeptidase I activity.<br>
The Fusarium primary transformants #1 and #16 were cultivated in 125 ml shake flasks for 5 days at 30Â°C in 25 ml of M400Da medium. The whole culture broths from die Fusarium primary transformants were filtered using a double layer of Miracloth. The filtrate was recovered and then frozen at -20Â°C.<br>
Example 11:    Purification of recombinant Aspergillus   oryzae 1568   dipeptidyl aminopeptidase I produced in Fusarium<br>
The recombinant dipeptidyl aminopeptidase I was purified from the Fusarium broth of primary transformant #1 described in Example 10. The broth (20 ml) was filtered through Nalgene filterware equipped with a 0.45 micron filter (Nalgene, Rochester, NY). The sample was diluted 10-fold using 20 mM sodium phosphate pH 7.0 buffer and concentrated using a ultrafiltration system (Amicon, Beverly, MA) utilizing a PMIO ultrafiltration membrane. The conductivity of the sample was 2.5 mS.<br>
The sample was then loaded onto a column (XK-26) containing 60 ml of Q-Sepharose Big Beads, which had been pre-equilibrated with 400 ml of 20 mM sodium phosphate pH 7.0 buffer. The column was washed until baseline was reached. At a flow rate of 5 ml/min, the dipeptidyl aminopeptidase I was eluted with a linear gradient from 0-0.40 M NaCl in 20 mM sodium phosphate buffer pH 7.0 buffer over 10 column volumes. The dipeptidyl aminopeptidase I eluted at -0.24 M NaCl. SDS-PAGE was performed on the fractions active on Ala-Pro-pNA according to the following protocol. The substrate was prepared by dissolving 2 mg of Ala-Pro-pNA (Bachem, Torrance, CA) in 20Âµl of DMSO. Then, 980 Âµl of 50 mM sodium phosphate pH 7.5 buffer was added. In a 96 well microtiter plate, 100 Âµl of the substrate solution was added to 100Âµl of Aspergillus oryzae dipeptidyl aminopeptidase I<br><br>
(100 fold diluted in 50 mM sodium phosphate buffer pH 7.5) and the increase in absorbance at 405 nm was measured for 4 minutes using a SpectroMax 340 plate reader (Molecular Devices, Sunnyvale, Ca). The homogeneous fractions were then pooled.<br>
Example 12: Characterization of recombinant Aspergillus oryzae 1568 dipeptidyl aminopeptidase I<br>
The purified dipeptidyl aminopeptidase I described in Example 11 was characterized with respect to pH optimum, temperature stability, substrate specificity, and kinetic parameters.<br>
The pH optimum was determined using Ala-Pro-pNA (HCl salt) as substrate in the universal buffer composed of 0.125 M citric acid, 0.125 M mono basic sodium phosphate, and 0.125 M boric acid pH was adjusted to 4.35-9.83 with 10 N NaOH, in 0.5 pH increments. The Ala-Pro-pNA substrate was prepared by dissolving 100 mg of Ala-Pro-pNA in 1 ml of DMSO and adding 20Âµl of the Ala-Pro-pNA-DMSO solution to 980Âµl of the universal buffer at the various pH values at ambient temperature. The assay was initiated by adding a 10 Âµl aliquot of the dipeptidyl aminopeptidase I diluted 20-fold in water to 200 Âµl of 2 mg/ml Ala-Pro-pNA at the various pH values. The absorbance at 405 nm was monitored for 5 minutes. Autohydrolysis of the substrate as a control was determined by adding 10 |il of water to 200 jil of 2 mg/ml Ala-Pro-pNA at the various pH values.<br>
The results shown in Table 1 below demonstrated that the dipeptidyl aminopeptidase I possessed activity toward Ala-Pro-pNA as substrate over the measured pH range 4.35 to 9.83 with optimal activity at pH -8.7. Autohydrolysis of the substrate was observed at pH values greater than 7.<br><br><br>
5.36 75.75	0	75.75	0.20<br>
5.86 113.8	0	113.8	0.307<br>
6.38 135.48	0	135.48	0.365<br>
6.85  168.45	2	166.45	0.45<br>
7.2  188.86	2	186.86	0.503<br>
7.51 230.97	3	227.97	0.615<br>
7.97 308.52	5.8	302.72	0.817<br>
8.71 383.15	12.5	370.65	1<br>
9.32 247.68	29.8	217.88	0.588<br>
9.83 171	74	97	0.261<br>
The temperature stability of the dipeptidyl aminopeptidase I was determined using the following protocol: 490Âµl of 50 mM sodium phosphate buffer pH 7.5 was preincubated at 37Â°, 45Â°, 55Â°, 60Â°, 65Â°, 70Â°, and 75Â°C for 30 minutes in a 1.7 ml Eppendorf tube. Then 10Âµl of purified dipeptidyl aminopeptidase I was added and the sample was then incubated for an additional 20 minutes. The samples were then placed on ice. Once the incubations were completed for all the temperatures, the samples were then assayed for activity using Ala-Pro-pNA as substrate.<br>
The assay was performed by mixing 30Âµl of the incubation mixtures for the various temperatures with 200Âµl of 2 mg/ml Ala-Pro-pNA in 50 mM sodium phosphate pH 7.5 buffer) at ambient temperature. The absorbance at 405 nm was monitored for 5 minutes.<br>
The results shown in Table 2 demonstrateld that the dipeptidyl aminopeptidase I retained 90% of its activity after a 20 minute incubation at 65Â°C, pH 7.5.<br>
Table 2<br>
Temperature (Â°C)	Percent activity relative to 37Â°C<br>
37	100 <br>
45	103<br>
55	103<br>
65	92.5<br><br>
75	0.7<br>
The relative activity of various mono-, di-, and tri-peptide para-nitroanilide substrates were assayed relative to Ala-Pro-pNA with the purified dipeptidyl aminopeptidase I diluted 100-fold in 50mM sodium phosphate pH 7.5 buffer. Each substrate was dissolved in DMSO to a concentration of 100 mg/ml and then diluted 50 fold in 50 mM sodium phosphate pH 7.5 buffer to 2 mg/ml. The assay was performed by mixing 100 Âµl of the substrate solution with 100 Âµl of the dipeptidyl aminopeptidase I solution and monitoring the change in absorbance at 405 nm and ambient temperature.<br>
The results shown in Table 3 demonstrated that the dipeptidyl aminopeptidase I preferably hydrolyzed Xaa-Pro-pNA and Xaa-Ala-pNA substrates where Xaa corresponds to any natural amino acid.<br>
Table 3<br>
Substrate:	Activity (mOD/min)	Percent Activity Relative to<br>
Ala-Pro-pNA<br>
Gly-Arg-pNA	0	0<br>
Gly-Pro-pNA	125	51.25<br>
Arg-Pro-pNA	103	42.17<br>
Val-AIa-pNA	49.5	20.3<br>
Gly-Glu-pNA	
Ala-Pro-pNA	244	100<br>
Ala-Ala-pNA	30.2	12.4<br>
Asp-Pro-pNA	70.95	29<br>
Leu-pNA	0	0<br>
Ala-pNA	0	0    <br>
Ala-Ala-Pro-pNA	10.6	4.3<br>
The kinetic parameters for various dipeptidyl aminopeptidase I substrates was determined using the following protocol. The substrates included Ala-Pro-pNA, Asp-Pro-pNA, and Ala-Ala-pNA.   Purified dipeptidyl aminopeptidase I with an A280 of 1.521 was<br><br>
diluted 25-fold for Ala-Pro-pNA, 20-fold for Asp-Pro-pNA, and 10-fold for Ala-Ala-pNA in water. Each substrate was dissolved in DMSO to a concentration of 100 mg/ml and then diluted 50 fold in 50 mM sodium phosphate pH 7.5 buffer to 2 mg/ml. In a 96 well microtiter plate, 10 Âµ1 of purified dipeptidyl aminopeptidase I was incubated with each substrate as follows except the 200Âµl substrate assay was not performed with the Asp-Pro-pNA substrate, and the absorbance at 405 nm was measured for 3 minutes:<br>
1.	200Âµl of 2 mg/ml substrate + 0 Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
2.	100Âµl of 2 mg/ml substrate + 100 Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
3.	50 Âµl of 2 mg/ml substrate + 150Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
4.	25 Âµ1 of 2 mg/ml substrate + 175 Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
5.	10 Âµl of 2 mg/ml substrate + 190 Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
6.	5 Âµl of 2 mg/ml substrate + 195Âµl of 50 mM sodium phosphate buffer pH 7.5<br>
A Line weaver-Burke plot was constructed to determine the Km and the kcst for each substrate, using an average molecular weight of 97 kDa for the differentially glycosylated forms.<br>
For Ala-Pro-pNA, the Km and kcat were determined to be 0.140 mM and 576.3 min-1 respectively.<br>
For Asp-Pro-pNA, the Km and kcat were determined to be 0,632 mM and 244.3 min-1, respectively.<br>
For Ala-Ala-pNA, the Km and kcatwere determined to be 1.08 mM and 106.5 min-1, respectively.<br>
Example 13:   Purincation of Aspergillus oryzae 1568 aminopeptidase I<br>
A 50 ml volume of FLAVOURZYMEâ¢ preparation described in Example 1 was<br>
centrifuged at 10,000 rpm for 10 minutes. The supernatant was filtered with a 0.2 |im Nalgene<br>
filter and the filtrate was then diluted to 350 ml with 20 mM sodium phosphate pH 7.5 buffer. The diluted filtrate was concentrated with an Amicon ultrafiltration cell equipped with a PM10 membrane. More 20 mM phosphate pH 7.5 buffer was added and concentrated 3 more times<br><br>
to adjust the sample to the proper pH and conductivity.<br>
The enzyme solution was then loaded onto a XK-26 column containing Q-Sepharose Big Beads pre-equilibrated with 20 mM phosphate buffer pH 7.5 buffer. A gradient was run from 0 to 300 mM NaCI, and then washed with 350 ml of 300 mM NaCl. Fractions were assayed for Leu-pNA activity according to the following protocol. The substrate was prepared by dissolving 2 mg of Leu-pNA (Sigma Chemical Co., St. Louis, MO) in 20 Âµl of DMSO. Then, 980Âµl of 50 mM sodium phosphate pH 7.5 buffer was added. In a 96 well microtiter plate, 100 Âµl of the substrate solution was added to 100 Âµl of Aspergillus oryzae aminopeptidase I (100 fold diluted in 50 mM sodium phosphate pH 7.5 buffer) and the increase in absorbance at 405 nm was measured for 4 minutes using a SpectroMax 340 plate reader. The most active fractions were pooled and concentrated as above using PM10 ultrafiltration.<br>
The concentrated sample was diluted to 250 ml with 20 mM phosphate pH 7.5 buffer and loaded onto a Mono-Q 16/10 column (Pharmacia Biotech AB, Uppsala, Sweden). A gradient was run from 0 to 400 mM NaCl. Fractions were assayed for Leu-pNA activity as above. The most active and pure fractions (by SDS-PAGE) were pooled and concentrated as above using using PM10 ultrafiltration.<br>
The sample was diluted in 50 mM phosphate pH 7.0 buffer with 1.7 M ammonium sulfate and concentrated as above. This step was repeated three times. The sample was loaded onto Phenyl Sepharose column (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 50 mM sodium phosphate pH 7.0 buffer containing 1.7 M ammonium sulfate. A gradient was run from 1.7 M to 0 M ammonium sulfate, and then washed with 50 mM sodium phosphate pH 7.0 buffer. Fractions were assayed for Leu-pNA activity and purity was checked by SDS-PAGE. Two small bands near 70 kDa were still present.<br>
A 1 ml volume of the most pure fraction was concentrated with a Microcon  10<br>
microconcentrator (Amicon, New Bedford, MA).   A 100Âµl volume of the concentrate was<br>
loaded onto a Superose 12 column (Pharmacia Biotech AB, Uppsala, Sweden) pre-equilibrated with 20 mM phosphate buffer pH 7.0 buffer, and eluted in 30 ml of the same buffer. Only one peak was observed. SDS-PAGE analysis revealed two small bands near 70 kDa. These bands<br><br>
were probably an aggregation of the dipeptidyl aminopeptidase, and no further purification was necessary.<br>
All active fractions from the Phenyl Sepharose column were pooled, concentrated, and desalted using PM10 ultrafiltration and 20 mM phosphate pH 7.0 buffer.<br>
Example 14:    Synergism  between Aspergillus   oryzae 1568   aminopeptidase  I and Aspergillus oryzae 1568 dipeptidyl aminopeptidase I<br>
The substrate was prepared by dissolving 2 mg of Ala-Phe-Pro-pNA (Bachem, Torrance, CA) in 20 Âµl of DMSO. Then, 980Âµl of 50 mM sodium phosphate buffer pH 7.5 and 100 Âµl of ethanol were added. In a 96 well microtiter plate, 100 |xl of the Ala-Phe-Pro-pNA solution was added to 100 Âµl of 100-fold diluted aminopeptidase I (Example 12); 100Âµl of the Ala-Phe-Pro-pNA solution was added to 100Âµ1 of 100-fold diluted dipeptidyl aminopeptidase I (Example 11); and 100Âµl of the Ala-Phe-Pro-pNA solution was added to a mixture of 50Âµl of 100-fold diluted aminopeptidase I and 50 Âµl of 100-fold diluted dipeptidyl aminopeptidase I. The absorbance at 405 nm was measured for 10 minutes.<br>
The purified aminopeptidase I was determined to have an activity of 26.822 LAPUs/ml for Leu-pNA using an extinction coefficient of 10,000 for para-nitroaniline at 405 nm. LAPU is defined as the leucine aminopeptidase activity which is determined as described in AF 298/1-GB (available on request from Novo Nordisk A/S, Denmark). The purified dipeptidyl aminopeptidase I was determined to have an activity of 11.54 DPAPU/ml for Ala-Pro-pNA, also using an extinction coefficient of 10000 for paranitroaniline. At a 100-fold dilution, aminopeptidase I exhibited less than a 2 mOD/min velocity on Ala-Pro-pNA and dipeptidyl aminopeptidase I exhibited less than a 2 mOD/min velocity on Leu-pNA. One hundred-fold diluted aminopeptidase I exhibited no activity on Ala-Phe-Pro-pNA over 10 minutes. One hundred-fold diluted dipeptidyl aminopeptidase I exhibited no activity on Ala-Phe-Pro-pNA over 10 minutes. When a mixture of 100-fold diluted dipeptidyl aminopeptidase I and 100-fold diluted aminopeptidase I were assayed with Ala-Phe-Pro-pNA, a velocity of 122 mOD/min was observed over 10 minutes.<br><br>
Example 15:    Preparation of protein hydrolysates with Aspergillus oryzae 1568 dipeptidyl aminopeptidase I<br>
The purified dipeptidyl aminopeptidase I described in Example 11 was tested in degree of hydrolysis assays using geletin, soy, gluten, and casein as substrates according to the following procedure.<br>
The degree of hydrolysis (DH) assays were performed at 50Â°C for 18 hours as a mini-hydrolysis on a 10 ml scale using gelatin, soy bean meal tablets, wheat gluten, and sodium-caseinate at a 2% concentration adjusted to pH 7, if necessary, with no pH adjustment during hydrolysis.   The hydrolyses were inactivated at 85Â°C for 3 minutes in a waterbath.   The<br>
enzymes used were FLAVOURZYMEâ¢ with ALCALASEâ¢ 2.4L (Novo Nordisk A/S, Bagsvrd,   Denmark)   and   dipeptidyl   aminopeptidase   I   with   FLAVOURZYMEâ¢   and<br>
ALCALASEâ¢ 2.4L.   FLAVOURZYMEâ¢, ALCALASEâ¢, and dipeptidyl aminopeptidase I<br>
were dosed at 3 mg, 3 mg, 0.125 mg, respectively, per 200 mg of protein per ml.<br>
The DH, as defined by Adler-Nissen (1986, Enzymic Hydrolysis of Food Proteins, Elsevier Applied Science Publishers), was determined by reaction of the supernatant with OPA (ortho-phtaldialdehyde, Sigma Chemical Co., St. Louis, MO). For the OPA reagent, 160 mg of OPA was dissolved in 4 ml of ethanol and transferred to a 200 ml volumetric flask containing a solution of 7.62 g of disodium tetraborate decahydrate, 200 mg of sodium dodecylsulphate, and 176 mg of dithiothreitol and the flask was filled to 2(X) ml with water.<br>
A volume of 25 Âµl of suitably diluted supernatant was mixed with 200 Âµl of OPA<br>
reagent in a microtiter plate well and allowed to react for exacdy 2 minutes at 25Â°C. The absorbance at 340 nm was measured in a microtiter plate reader and compared to the absorbance of a 95 mM L-serine standard solution after subtraction of the blank value (water reacted with OPA-reagent). To determine the true DH, the serine equivalents measured in the supematants were corrected with the factors suggested by Adler-Nissen for the trinitrobenzenesulfonic acid method (Adler-Nissen, 1979, Agricultural and Food Chemistry 17:<br><br>
1256) which gave the same response as the described OPA method. The degree of hydrolysis was calculated on basis of the total amount of protein in the hydrolysis mixture (not on basis of soluble protein).<br>
The results showed that dipeptidyl aminopeptidase increased the degree of hydrolysis<br>
5% above the samples with FLAVOURZYMEâ¢ and ALCALASEâ¢ alone for all the proteins tested.<br>
Example    16:    Increased    protein   solubility    and   release    of   glutamate    by deamidation<br>
Wheat gluten (WG) was obtained from Cargill (JOB 5141) and deamidated wheat gluten (DWG) was obtained from StaPro Consultancy B.V., Lemdijk 32, 9422 TH Smilde, NL. Suspensions of 8% protein were made by mixing 11 g of gluten with 89 g of water. The pH was adjusted to 6.5 with NaOH. Glutamate/aspartate specific protease (SP446), obtainable as described in WO 91/13554, or lysine/arginine specific protease (SP387) obtainable as described in WO 89/06270, was added to the suspensions. The dosage was 0.01 AU/g protein<br>
for SP446 and 0.006 AU/g protein for SP387.    FLAVOURZYMEâ¢ (an non-specifically<br>
acting protease preparation available from Novo Nordisk A/S, Bagsvasrd, Denmark, containing endo- and exo-peptidase activities, and obtained by fermentation of Aspergillus oryzae) was added to some of the hydrolysates at a dosage of 20 LAPU/g protein. One LAPU (Leucine Amino Peptidase Unit) is the amount of enzyme which decomposes 1 micromole of L-leucine-p-nitroanilide per minute under the following conditions: 26 mM L-leucine-p-nitroanilide in 0.1 M Tris pH 8.0 buffer at 40Â°C for 10 minutes. Upon hydrolysis, p-nitroanilide is liberated tuming the solution yellow which is monitored 405 nm.<br>
The hydrolyses were carried out at 50Â°C without further pH adjustment for 18 hours. The enzymes were inactivated by heating at 85Â°C for 15 minutes.  The pH was adjusted to 5 and the hydrolysates were centrifuged.    The content of protein and free glutamate in the supernatant was determined.<br><br>
The protein content was determined by Kjeldahl analysis, using a Kjeldahl factor of 6.25.<br>
The content of free glutamate was determined by use of a glutamate determination kit according to the manufacturer's instructions (Boehringer-Mannheim, Indianapolis, IN). The method was adapted for use in microtiter plates.<br>
When comparing wheat gluten (WG) to deamidated wheat gluten (DWG), the results as shown in Table 4 demonstrated that deamidation increased the susceptibility of the gluten to<br>
specific proteases, such that more protein became soluble. By addition of FLAVOURZYMEâ¢ with a specific protease, the release of glutamate was doubled due to deamidation.<br><br>
Example 17: Enzymatic deamidation and release of glutamate<br>
Peptidoglutaminase II was produced by growing Bacillus circulans strain ATCC 21590 in shake flasks (400 ml) containing 200 ml of a medium composed of 1% polypeptone, 0.5% lactose, 0.025% MgSO4-7H2O, 0.005% FeSO4-TH2O, 0.025% KH2PO4 and 17% Na2HPO4-12H2O (pH adjusted to 7,2), at 30Â°C for 20 hours with mixing at 270 rpm. The cells were harvested by centrifugation at 4000 rpm in 1 litre flasks. The cells were then frozen<br><br>
The purification of peptidoglutaminase II from Bacillus circulans was performed at room temperature. The frozen Bacillus circulans cells were thawed and suspended in Lysis buffer (50 mM Tris/HCl; 25% (w/v) sucrose; 1 mM EDTA, pH 8.0) until a homogeneous suspension was obtained - 100 g wet cells per liter of Lysis buffer. Lysozyme (10 mg/ml) and DNAse I (Sigma DN-25, 10 mg/ml) were dissolved in Lysis buffer. Then 100 ml of lysozyme solution, 10 ml of 1.0 M MgCl2, and 1 ml of DNAse I solution were added per litre of cell suspension. The enzymes were allowed to act for 1 hour.<br>
The suspension was filtered through a Seitz depth filter plate and the filtrate was transferred to a 10 mM KH2PO4/NaOH, pH 8.0 (Buffer A) on a Sephadex G25 column (Pharmacia). The enzyme solution was  applied to a SOURCE Q column (Pharmacia)<br>
equilibrated in Buffer A and eluted with a linear NaCl gradient (0 -&gt;500 mM) in Buffer A.<br>
Fractions from the column were analysed for Peptidoglutaminase n activity as described below and fractions with activity were pooled. The absorbance of the pooled fractions at 280 nm was 1.78, thus the protein content was estimated to 1.8 mg/ml.<br>
The purity of the protein in the peptidoglutaminase 11 pool was approximately 25% as judged from a SDS-PAGE gel. Thus, the preparation contained approximately 0.5 mg/ml of pure peptidoglutaminase II.<br>
The peptidoglutaminase activity was determined by measuring the ammonia formed<br>
during hydrolysis of y-carboxyamide of N-tert-Butoxycarbonyl-Gln-Pro (N-t-BOC-Gln-Pro;<br>
SIGMA No. B-4403) using the Boehringer-Mannheim kit for ammonia determination (Cat. No. 1112732). In this kit, ammonia is measured by determination of the consumption of NADH by glutamate dehydrogenase, and blanks without the addition of N-t-BOC-Gln-Pro were also applied in order to subtract the effect of other NADH consuming enzymes.<br>
A total of 200 mg of wheat gluten protein was added to 9 ml, of boiling water and after<br>
cooling, the pH was adjusted to 7.0.   Then 250Âµl of the peptidoglutaminase II preparation<br>
(PEP) described above was added. The glutamate/aspartate specific protease (SP446) described   in   Example   16   was   added   in   an   amount   of  0.04   AU/g   protein,   and<br><br>
FLAVOURZYMEâ¢ described in Example 16 was added in an amount of 20 LAPU/g protein.<br>
Hydrolysis was allowed to proceed without. pH adjustment for 18 hours at 50Â°C.<br>
Controls without the addition of peptidoglutaminase were also run. The hydrolysates were centrifuged and glutamate was measured as described in Example 16. The DH was determined as described in Example 15.<br>
The results as shown below in Table 5  demonstrated that hydrolysis  with  the peptidoglutaminase preparation increased the DH as well as the release of glutamate.<br><br>
Deposit of Biological Materials<br>
The following biological material has been deposited under the terms of the Budapest<br>
Treaty with the Agricultural Research Service Patent Culture Collection, Northern Regional<br>
Research Center, 1815 University Street, Peoria, Illinois, 61604, and given the following<br>
accession number;<br>
Deposit	Accession Number	Date of Deposit<br>
E. coli DH5apMWR52	NRRLB-21682	April 18, 1997<br>
This application is divided out of Patent Application No.<br>
1064/MAS/98 which relates to method of obtaining polypeptide having dipeptidyl aminopeptidase.<br><br><br><br><br>
WE CLAIM;<br>
1.      A method for obtaining a hydrolysate  from   a proteinaceous substrate such as herein described which comprises the steps of:<br>
(i)      subjecting in a known manner the substrate to the action    of a polypeptide having  dipeptidyl  aminopeptidase activity; and<br>
(ii)     subjecting in a known manner the substrate to   the action of an endopeptidase;<br>
wherein    the   polypeptide    having    dipeptidyl aminopeptidase activity is selected from the group consisting of:<br>
(a)     a polypeptide having an amino acid sequence which has at least 70% identity with amino acids 17 to 771 of SEQ ID No.2;<br>
(b) a polypeptide which is encoded by a nucleic acid sequence which hybridizes under medium stringency conditions with (i) nucleotides 49 to 2396 of SEQ ID No. 1, (ii) the cDNA sequence contained in nucleotides 49 to 2396 of SEQ ID NO.l, (iii) a subsequence of (i) or (ii) of at least 100 nucleotides, or (iv) a complementary strand of (i), (ii), or (iii); wherein medium stringency conditions are defined as prehybridization and hybridization at 42Â°C in 5X SSPE, 0.3% SDS, 200 Âµg/ml sheared and denatured salmon sperm DNA, and 35% formamide;<br><br>
(c)     an allelic variant of (a) or (b);<br>
(d)    a   fragment   of  (a), (b),   or   (c),   wherein   the fragment has dipeptidyl aminopeptidase activity; and<br>
(e) a polypeptide having dipeptidyl aminopeptidase activity with physicochemical properties of (i) a pH optimum in the range of from pH 4.4 to pH 9.8 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide; (ii) a temperature stability of 90% or more, relative to initial activity, at pH 7.5 determined after incubation for 20 minutes at 65'oC in the absence of substrate; and (iii) an activity towards Xaa-Pro-para-nitroanilide or Xaa-Ala-para-nitroanilide wherein Xaa is selected from the group consisting of Ala, Arg, Asp, Gly and Val.<br>
2.	The method as claimed in claim 1, in which step (i) and step (ii) are accomplished concurrently.<br>
3.	The method as claimed in claim 1, in which step (i) and step (ii) are accomplished consecutively.<br>
4.	The method as claimed in claim 1, wherein the polypeptide of (a) has an amino acid sequence which has at least 80% identity with amino acids 17 to 771 of SEQ ID NO.2.<br>
5.	The method as claimed in claim 4, wherein the polypeptide of (a) has an<br>
amino acid sequence which has at least 90% identity with amino acids 17 to 771<br>
of SEQ ID NO.2.<br><br>
6. The method as claimed in claim 5, wherein the polypeptide of (a) has an amino acid sequence which has at least 95% identity with amino acids 17 to 771of SEQ ID NO.2.<br>
7.	The method as claimed in claim 6, wherein the polypeptide of (a) has an amino acid sequence which has at least 97% identity with amino acids 17 to 771 of SEQI D NO,2.<br>
8.	The method as claimed in claim 1, wherein the polypeptide of (a) has the amino acid sequence of SEQ ID N0.2 or a fragment thereof<br>
9.	The method as claimed in claim 8, wherein the polypeptide has the<br>
amino acid sequence of SEQ ID NO,2.<br>
10. The method as claimed in claim 1, wherein the polypeptide of (a) is obtained from an Acremonium, Aspergillus, Fusarium, Humicola, Mucon Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium and Trichoderma strain.<br>
11.	The method as claimed in claim 11, wherein the polypeptide is obtained from an Aspergillus oryzae strain.<br>
12.	The method as claimed in claim 1, wherein the polypeptide of(b) is encoded by a nucleic acid sequence which hybridizes under medium stringency conditions with (i) nucleotides 49 to 2396 of SEQ ID NOT, (ii) the   cDNA  sequence contained   in nucleotides 49 to 2396 of SEQ ID NO.l,<br><br>
(iii) a subsequence of (i) or (ii) of at least 100 nucleotides or (iv) a complementary strand of (i), (ii), or (iii) wherein medium stringency conditions are defined as prehybridization and hybridization at 42Â°C in 5X SSPE, 0.3% SDS, 200Âµg/ml sheared and denatured salmon sperm DNA, and 35% formamide.<br>
13. The method as claimed in claim 12, wherein the polypeptide of (b) is encoded by a nucleic acid sequence which hybridizes under medium stringency condtitions with (i) nucleotides 49 to 2396 of SEQ ID NO.l, (ii) the cDNA sequence contained in nucleotides 49 to 2396 of SEQ ID NO.l, or (iii) a complementary strand of (i) or (ii) of at least 100 nucleotides wherein medium stringency conditions are defined as prehybridization and hybridization at 42oC in 5X SSPE, 0.3% SDS, 200 Âµg/ml sheared and denatured salmon sperm DNA, and 35% formamide.<br>
14. The method as claimed in claim 12, wherein the polypeptide is obtained from an Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium and Trichoderma strain.<br>
15.	The method as claimed in claim 14, wherein the polypeptide is obtained from an Aspergillus oryzae strain.<br>
16.	The method as claimed in claim 1, wherein the polypeptide of(b)is encoded by a nucleic acid sequence which hybridizes under high stringency conditions   with (i) nucleotides  49 to 2396 of SEQ ID No. l,(ii)the cDNA<br><br>
sequence contained in nucleotides 49 to 2396 of SEQ ID No. 1, or (iii) a complementary strand of (i) or (ii) of at least 100 nucleotides said high stringency conditions are defined as prehybndization and hybridization at 42Â°C in 5X SSPE, 0.3% SDS, 200 Âµg/ml sheared and denatured salmon sperm DNA, and 35% formamide.<br>
17.	The method as claimed in claim 16, wherein the polypeptide of(b)<br>
is encoded by a nucleic acid sequence which hybridizes under high<br>
stringency conditions with the nucleic acid sequence of SEQ ID No. 1 or its<br>
complementary strand said high stringency conditions are defined as<br>
prehybridization and hybridization at 42'oC in 5X SSPE, 0.3% SDS, 200Âµg/ml<br>
sheared and denatured salmon sperm DNA, and 35% formamide.<br>
18.	The method as claimed in claim 16, wherein the polypeptide is<br>
obtained from an Acremonium, Aspergillus, Fusarium, Humicola,<br>
Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium<br>
and Trichoderma strain.<br>
19.	The method as claimed in claim 18, wherein the polypeptide is obtained from an Aspergillus oryzae strain.<br>
20.	The method as claimed in claim 1, wherein the polypeptide of(e) has the following physicochemical properties: (a) a pH optimum in the range of from pH 4.4 to pH 9.8 determined after incubation for 5 minutes at ambient temperature in the presence of Ala-Pro-para-nitroanilide; (b) a temperature stability  of 90%  or  more, relative to initial activity, at pH 7.5<br><br>
determined after incubation for 20 minutes at 65Â°C in the absence of substrate; and (c) an activity towards Xaa-Pro-para-nitroanilide or Xaa-Ala-para-nitroanilide wherein Xaa is selected from the group consisting of Ala, Arg, Asp, Gly and Val.<br>
21. The method as claimed in claim 20, wherein the polypeptide of(e) is obtained from an Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium and Trichoderma strain.<br>
22.     The method as claimed in claim 21, wherein the polypeptide of(e) is obtained from an Aspergillus oryzae strain.<br>
23. The method as claimed in claim 1, wherein the polypeptide of(b) is encoded by the nucleic acid sequence contained in plasmid pMWR52 which is contained in E.coli NRRL B-21682.<br>
24.	The method as claimed in claim 1, in which the dipeptidyl aminopeptidase is added to the substrate in an amount in the range of from 0.1 to 100,000 dipeptidyl aminopeptidase units (DPAPU) per 100 g of protein.<br>
25.	The method as claimed in claim 1, in which the endopeptidase is derived from a strain of Bacillus, Staphylococcus, Streptomyces, Actinomyces or Aspergillus.<br><br>
26.	The method as claimed in claim I, in which the endopeptidase is<br>
added to the substrate in an amount in the range of from 0.5 to 500 LAPU/lOOg<br>
of protein.<br>
27.	The method as claimed in claim 1, in which the proteinaceous<br>
substrate is of vegetable origin.<br>
28.	The method as claimed in claim 1, in which the proteinaceous<br>
substrate is of animal origin.<br>
29.	The method as claimed in claim 1, wherein the proteinaceous<br>
substrate is optionally deamidated first to enrich the hydrolysate in free<br>
glutamic acid and/or peptide bound glutamic acid residues.<br>
30.	The method as claimed in claim 1, wherein the proteinaceous substrate is treated with an aminopeptidase which acts synergistically with the polypeptide having dipeptidyl aminopeptidase activity to hydrolyze a peptide.<br>
31.	The method as claimed in claim 30, wherein the peptide is a tripeptidc.<br>
32.	The method for obtaining a hydrolysate from a proteinaceous<br>
substrate substantially as herein described.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">372-mas-2000-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">372-mas-2000-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">372-mas-2000-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gMTMucGRm" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gNC5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzcyLW1hcy0yMDAwLW90aGVyIGRvY3VtZW50cy5wZGY=" target="_blank" style="word-wrap:break-word;">372-mas-2000-other documents.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="190891-a-process-for-the-manufacture-of-flowable-non-dusty-free-riboflavin-granulates.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="190894-a-process-for-preparing-beadlets-containing-fat-soluble-substances.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>190892</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>372/MAS/2000</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Mar-2004</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-May-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVOZYMES BIOTECH INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1445 DREW AVENUE, DAVIS, CA 95616,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ALEXANDER BLINKOVSKY</td>
											<td>C/O NOVOZYMES BIOTECH INC.,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A23L 1/227</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/062,892</td>
									<td>1997-10-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>08/057,884</td>
									<td>1997-05-16</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/190892-a-method-for-obtaining-a-hydrolysate-from-a-proteinaceous-substrate by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:44:37 GMT -->
</html>
