<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/192690-continuous-fermentaion-process by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:00:43 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 192690:CONTINUOUS FERMENTAION PROCESS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CONTINUOUS FERMENTAION PROCESS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a fermentation assembly comprising (a) a vessel for culturing living cells; (b) at least two storage flasks in fluid communication with the vessel for supply of liquids and a first transport means for transferring the liquids from the storage flasks to the vessel; (c) individual volumetric or mass flow rate measuring instruments operably connected to the transport means, for monitoring the supply of the contents of the storage flasks to the vessel; (d) a harvest flask in fluid communication with the vessel and a second transport means for transferring the fermentation broth from the vessel to the harvest flask; and (e) a system containing a measuring insturment that monitors the flow from the storage flasks and a controlling unit which system is operably connected to the first transport means for controlling and maintaining a constant dilution rate in the vessel with varying rates of individual supply of liquid from the storage flasks to the vessel.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a  fermentation  assembly.<br>
In accordance with the present invention it has been found that splitting of cultivation media used in a continuous fermentation process allows to study the influence on growth and metabolite-production of microorganisms and thus to determine optimal conditions for the fermentation process. A continuously delivered fermentation medium can generally be split into as many fractions as it contains ingredients. Examples of such ingredients are carbon, nitrogen, phosphorus and sulfur sources as well as vitamins and complex substrates such as corn steep, yeast extract and other natural products. Furthermore, every required mineral, micro- or trace element can be provided separately as a solution of a water-soluble salt, such as a chloride, sulfate or nitrate. In this manner a fermentation medium of any desired composition can be obtained, provided that the desired amounts of the ingredients are (water)-soluble and no disturbing interactions (e.g., precipitation, reaction) occur in the individual feed solutions or in the fermentation medium.<br>
In one aspect, the present invention is concerned with a continuous process for the manufacture of proteins by means of protein-producing microorganism.<br><br>
More particularly, the invention is concerned with a continuous process for the manufacture of proteins by means of protein-producing microorganism in rich process the microorganism is optionally immobilized on a shod carrier and/w the nutrients and other agents required for the growth of the microorganism and the optimal production of protein are fed into the reactor individually at a constant dilution rate.<br>
Accordingly, the present invention provides a fermentation assembly comprising (a) a vessel for culturing living cells; (b) at least two storage flasks in fluid communication with the vessel for supply of liquids and a first transport means for transferring the Squids from the storage flasks to the vessel; (c) individual volumetric or mass flow rate measuring instruments operably connected to the transport means, for monitoring the supply of the contents of the storage flasks to the vessel; (d) a harvest flask in fluid communication with the vessel and a second transport means for transferring the fermentation broth from the vessel to the harvest flask; and (e) a system containing a measuring instrument that monitors the flow from the storage flasks and a controlling unit, which system is operably connected to the first transport means for controlling and maintaining a constant dilution rate in the vessel with varying rates of individual supply of liquid from the storage flasks to the vessel, wherein (i) vessel is equipped with inlet tubes fern sac 'age flasks for softly of liquids; (ii) pumps are for transporting liquids from the storage flasks to vessel; (iii) scales are for monitoring the amount of Squids supped to and discharged from the vessel; (iv) pun^ is for discharging fermentation broth via outlet tubes to a harvest flask; (v) controlling unit is for monitoring and steering individual cc troll systems for temperature, pH, gas pressure, vessel content and antifoam agents; (vi) gas inlet tubes<br><br>
and outlet tubes are present; (vii) main controlling unit is fact  overall process monitoring and steering; (viii) circuit having pump is for gas supply and taking samples; (ix) gas inlet and outlet flow control are present; and, optionally, sterile filters and thermo stating unit are present<br>
Any conventional fermentation vessel can be used for the purpose of this invention. The vessel may be made of materials such as stainless steel,  ass or ceramics and may have a volume of from eg., 100 ml to 2500m although  these figures are not critical to the invention For continuous operation the inside of the vessel is optionally equipped with, eggs., a receptacle or sieve plate for uptake of immobilized cells. Further, the fermentation vessel is connected to a series of storage flasks that cetin nutrient solutions and solutions â€”<br><br>
for maintaining and controlling a desired pH and other parameters, such as foam formation, redo potential etc. in the fermentation broth. Depending on the particular needs of the fermentation, there may be separate storage flasks for individual supply of substrates that serve as carbon or nitrogen or mineral source for the Using cells.<br>
It has been found in accordance with the invention that the process is advantageously carried out at a constant dilution rate in the fermentation vessel. As used herein, the term "dilution rate" denotes the total volume of liquids supplied to the fermentation vessel per volume of the fermentation vessel per hour [h'^].<br>
Accordingly, it is a particular feature of the present invention to carry out the fermentation process at a constant dilution rate in the fermentation vessel while varying the supply of individual nutrient components or other additives during the fermentation process. To facilitate this task a storage flask containing an inert component, e.g., water is optionally provided that allows to complement the supply of liquids thus keeping the total supply of liquid constant.<br>
The assembly that is preferably used to carry out the process of this invention further comprises means to transport the individual components of the fermentation medium from the storage flasks to the fermentation vessel, and appliances for monitoring the amount of liquid supplied to the fermentation vessel. Every combination of measuring instruments (e.g., volumetric or mass flow rate by either gravimetric, anemometric, magnetic, ultrasonic, Venture, J, cross-relation, thermal, Carioles, radiometric) and transfer units (e.g., pumps or pressure difference) can be used for this purpose. Additionally, every transfer unit can be applied as a dosing unit (e.g., gear, peristaltic, piston, membrane or exciter pump). For operation on small scale the supply is suitably monitored by weighing the storage flasks that contain nutrient or additive solutions in a predetermined concentration.<br>
The device for controlling and maintaining a constant dilution rate in the fermentation vessel is suitably a system comprising a measuring instrument that monitors the flow from the storage flasks and a controlling unit, e.g., a computer-software control that calculates the actual mass flow rates, compares them to the desired value and adjusts the pump<br><br>
setting accordingly. An appropriate system is, e.g., the Process Automation System, National Instruments, Bridge View, USA, for Windows NT 4.0 (represented by National Instruments, Sonnenbergstrasse 53, 5408 Ennetbaden, Switzerland) that is connected to the various operating units (scales, pumps) through a serial-interface box (Rocket Port, Control Europe Ltd, Great Britain, represented by Techno software AG Rothackerstrasse 13, 5702 Nederland, Switzerland).<br>
An assembly that can be used in the process of this invention is depicted in Figure 1 of   the  alcohol panning drawings-<br>
The fermentation vessel 1 (Fermentor) is equipped with inlet tubes 2a from storage flasks 2 (suitably equipped with a stirrer) for supply of salt solution (Salts), nutrient solution (Nutrients), particular substrates (Substrate 1 and Substrate 2) for supply of, e.g., distinct carbon sources, agent for controlling the pH (Base), water for controlling a constant dilution rate, and antifoam. Pumps 3 transport liquids from the storage flasks 2 to the fermentor 1. Scales 4 monitor the amount of liquids supplied to and discharged from the fermentor. Further, the fermentor has inlet tubes 9 for oxygen supply and outlet tubes 10 for exhaust controlled by untits 14 and 15. Pump 6 discharges fermentation broth via outlet tubes 5a to a harvest flask 5. A main controlling unit 7 monitors and steers the overall process. Controlling unit 11 monitors and steers individual control systems 17 for temperature, pH, gas pressure, fermentor content and supply of antifoam agents. Circuit 12 including pump 13 is used for taking samples from the fermentation broth and for providing a controlled gas flow for moving the fermentation broth. Inlet and outlet gas flow is controlled by flow control 14 and 15. Sterile filters 16 are provided optionally. Optionally, the fermentation vessel 1 is equipped with a thermosetting unit 8.<br>
In the process of the present invention, any protein-producing microorganism either natural, e.g. fungal origin or bacterial origin or microorganisms which have been transformed by protein encoding DNA whereby such transformed microorganisms can be bacteria or fungi or yeasts, preferably from the genus Peripheral, Aspergillums, Pansexual or Picric, especially Aspergillums nicer, Aspergillum avatar, Aspergillum solace, Aspergillums orison or Pansexual polymorph or Picric pastries.<br><br>
In this context, the skilled person in the art selects such a protein-producing microorganism which is known to be useful for the production of a desired protein.<br>
In a preferred embodiment of the present invention the protein is selected from the group consisting of proteins having the activity of an enzyme such as catalase, lactase, phenoloxidase, oxidize, oxidoreductase, glutamate cellulase, xylanase and other polysaccharide, peroxides, lipase, hydrolase, esterase, cuttings, protease and other proteolyses enzymes, amino peptidase, carboxypeptidaseâ€™s, photoset, lease, pectinâ€™s and other pectinolytic enzymes, amylase, glycosidase, mannosidase, isomeraseâ€™s, inverts, transferees, rib nuclease, chitins, and desoxyribonuclease. Furthermore, in a preferred embodiment of the present invention the protein is selected from the group of therapeutic proteins such as antibodies, vaccines, antigens, or of antibacterial and/or health-beneficial proteins such as lactoternin, lactoperoxidase or lysozyme.<br>
It will be understood by those skilled in the art that the term "activity" includes not only native activities referring to naturally occurring enzymes or therapeutic functions, but also those activities or functions which have been modified by amino acid substitutions, deletions, additions, or other modifications which may be made to enhance or modify the desired activity, or the thermo stability, pH tolerance and/or further properties.<br>
In a most preferred embodiment of the invention the selected protein is a protein having the activity of a phytase.<br>
Examples of proteins having the activity of a phytase are described in EP 684 313, EP 897 010, EP 897 985 or in Examples 6 to 16 and Figures 2-22 of the present invention.<br>
Figure 2: Design of the consensus phytase sequence. The letters represent the amino acid residues in the one-letter code. The following sequences were used for the alignment: phyA from Aspergillus terreus 9A-1 (Mitchell et al, 1997; from amino acid (aa) 27), phyA from A. terreus cbsl 16.46; (van Loon et al., 1998; from aa 27), phyA from Aspergillus niger var. awamori (Piddington et al, 1993; from aa 27), phyA from A. niger T213; Mitchell et al. 1997 from aa 27), phyA from A. niger strain NRRL3135 (van Hartingsveldt et al, 1993; from aa 27), phyA from Aspergillus fumigatus ATCC 13073 (Pasamontes et al, 1997; from aa 25), phyA from A. fumigatus ATCC 32722 (EP 897 985; Figure 1; from aa 27), phyA from A.<br><br>
fumigatus ATCC 58128 (EP 897 985; Figure 1; from aa 27), phyA from A fumigatus ATCC 26906 (EP 897 985; Figur 1; from aa 27), phyA from A fumigatus ATCC 32239 (EP 897 985; Figur 1; from aa 30),p/iyA from Emericella indulines (Pasamontes et al, 1997a; from aa 25), phyA from Talaromyces thermopiles (Pasamontes et al, 1997a; from aa 24), and phyA from Myceliophthora thermopiles (Mitchell et al, 1997; from aa 19). The alignment was calculated using the program PILEUP. The location of the gaps was refined by hand. Capitalized amino acid residues in the alignment at a given position belong to the amino acid coalition that establish the consensus residue. In bold, beneath the calculated consensus sequence, the amino acid sequence of the finally constructed consensus phytase (Fcp) is shown. The gaps in the calculated consensus sequence were filled by hand according to principals stated in Example 6.<br>
Figure 3: DNA sequence of the consensus phytase-1 gene (fcp) and of the primers used for the gene construction. The calculated amino acid sequence (Figure 2) was converted into a DNA sequence using the program BACKTRANSLATE (Devereux et a/., 1984) and the codon frequency table of highly expressed yeast genes (GCG program package, 9.0). The signal peptide of the phytase from A. terreus cbs.l 16.46 was ftised to the N-terminus. The bold bases represent the sequences of the oligonucleotides used to generate the gene. The names of the respective oligonucleotides are alternately noted above or below the sequence. The underlined bases represent the start and stop codon of the gene. The bases written in italics show the two introduced Eco RI sites.<br>
Figure 4: Alignment and consensus sequence of five Basidiomycetes phytases. The letters represent the amino acid residues in the one-letter code. The amino acid sequences of the phytases from Lapillus involutes, phyla (aa 21) and phyA2 (aa 21, WO 98/28409), Trametespubescens (aa 24, WO 98/28409), Agrocybepediades (aa 19, WO 98/28409), and Peripheral lycii (aa 21, WO 98/28409) starting with the amino acid residues mentioned in parentheses, were used for the alignment and the calculation of the corresponding consensus sequence called "Basidio" (Example 7). The alignment was performed by the program PILEPUP. The location of the gaps was refined by hand. The consensus sequence was calculated by the program PRETTY. While a vote weight of 0.5 was assigned to the two P. involutes phytases, all other genes were used with a vote weight of 1.0 for the consensus sequence calculation. At positions, where the program was not able to determine a consensus residue, the Basidio sequence contains a dash. Capitalized amino acid residues in the alignment at a given position belong to the amino acid coalition that establish the consensus residue.<br><br>
Figure 5: Design of consensus phytase-10 amino acid sequence. Adding the phytase sequence of Thermometer lanuginose (Break et ah, 1998) and the consensus sequence of the phytases from five Basidiomycetes to the alignment of Figure 2, an improved consensus sequence was calculated by the program PRETTY. Additionally, the amino acid sequence of A. niger T213 was omitted; therefore, a vote weight of 0.5 was used for the remaining A. niger phytase sequences. For further information see Example 8.<br>
Figure 6: DNA and amino acid sequence of consensus phytase-10. The amino acid sequence is written above the corresponding DNA sequences using the one-letter code. The sequence of the oligonucleotides which were used to assemble the gene are in bold letters. The labels of oligonucleotides and the amino acids which were changed compared to those for consensus phytase -1 are underlined. The/cplO gene was assembled from the following oligonucleotides: CP-1, CP-2, CP-3.10, CP-4.10, CP-5.10, CP-6, CP-7.10, CP-8.10, CP-9.10, CP-10,10, CP-11.10, CP-12.10, CP-13.1G, CP-14.10, CP-15.10, CP-16.10, CP-17.10, CP18.10, CP-19.10, CP-20.10, CP-21.10, CP-22.10. The newly synthesized oligonucleotides are additionally marked by number 10. The phytase contains the following 32 exchanges relative to consensus phytase -1: Y54F, E58A, D69K, D70G, A94K, N134Q, I158V, S187A, Q188N, D197N, S204A, T214L, D220E, L234V, A238P, D246H, T251N, Y259N, E267D, E277Q, A283D, R291I, A320V, R329H, S364T, 1366V, A379K, S396A, G404A, Q415E, A437G, A463E. The mutations accentuated in bold letters revealed a stabilizing effect on consensus phytase-1 when tested as single mutations in consensus phytase-1.<br>
Figure 7: Alignment for the design of consensus phytase-11. In contrast to the design of consensus phytase-10, for the design of the amino acid sequence of consensus phytase-11, all Basidiomycete phytases were used as independent sequences using an assigned vote weight of 0.2 for each Basidiomycete sequence. Additionally, the amino acid sequence of A. niger T213 phytase was used in that alignment, again.<br>
Figure 8: DNA and amino acid sequence of consensus phytase-l-thermo[8]-Q50T-K91A. The amino acid sequence is written above the corresponding DNA sequence using the one-letter code. The replaced amino acid residues are underlined. The stop codon of the gene is marked by a star ("*").<br>
Figure 9: DNA and amino acid sequence of consensus phytase-10-thermo[3]-Q50T-K91A, The amino acid sequence is written above the corresponding DNA sequence using the<br><br>
one-letter code. The replaced amino acid residues are underlined. The stop codon of the gene is marked by a star (*).<br>
Figure 10: DNA and amino acid sequence of A. fumigates ATCC 13073 phytase (X-mutant. The amino acid sequence is written above the corresponding DNA sequence using the one-letter code. The replaced amino acid residues are underlined. The stop codon of the gene is marked by a star (*).<br>
Figure 11: DNA and amino acid sequence of consensus phytase-7. The amino acids are written above the corresponding DNA sequence using the one-letter code. The sequences of the oligonucleotides used to assemble the gene are in bold letters. Oligonucleotides and amino acids that were exchanged are underlined and their corresponding triplets are highlighted in small cases. The/cp7 gene was assembled from the following oligonucleotides: CP-1, CP-2, CP-3, CP-4.7, CP-5.7, CP-6, CP-7, CP-8.7, CP-9, CP-10.7, CP-1L7, CP-12.7, CP-13.7, CP-14.7, CP-15.7, CP-16, CP-17.7, CP-18.7, CP-19.7, CP-20, CP-21, CP-22. The newly synthesized oligonucleotides are additionally marked by number 7. The phytase contains the following 24 exchanges in comparison to the original consensus phytase -1: S89D, S92G, A94K, D164S, P201S, G203A, G205S, H212P, G224A, D226T, E255T, D256E, V258T, P265S, Q292H, G300K, Y305H, A314T, S364G, M365I, A397S, S398A, G404A, and A405S.<br>
Figure 12: Differential scanning calorimetry (DSC) of consensus phytase-1 and consensus phytase-10. The protein samples were concentrated to ca. 50-60 mg/ml and extensively dialyzed against 10 mM sodium acetate, pH 5.0. A constant heating rate of 10 Â°C/min was applied up to 95 Â°C. DSC of consensus phytase-10 (upper graph) yielded a melting temperature of 85.4 Â°C, which is 7.3 Â°C higher than the melting point of consensus phytase-1 (78.1 "'C, lower graph).<br>
Figure 13: Differential scanning calorimetry (DSC) of consensus phytase-10-thermo[3]-Q50T and consensus phytase-10-thermo[3]-Q50T-K91A. The protein samples were concentrated to ca. 50-60 mg/ml and extensively dialyzed against 10 mM sodium acetate, pH 5.0. A constant heating rate of 10 Â°C/min was applied up to 95 Â°C. DSC of consensus phytase-10-thermo-[3]-Q50T (upper graph) yielded a melting temperature of 88.6 Â°C, while the melting point of consensus phytase-10-thermo[3]-Q50T-K91A was found at 89.3<br><br>
Figure 14: Comparison of the temperature optimum between consensus phytase-1, consensus phytase-10 and consensus phytase-10-thermo[3]-Q50T. For the determination of the temperature optimum, the phytase standard assay was performed at a series of temperatures between 37 and 86 Â°C. The diluted supernatant of transformed S. cerevisiae strains was used for the determination. The other components of the supernatant showed no influence on the determination of the temperature optimum: A, consensus phytase-1;<br>
V, consensus phytase-10; B, consensus phytase 10-thermo[3]-Q50T.<br>
Figure 15: pH-dependent activity profile and substrate specificity of consensus phytase-10 and its variants thermo[3]-Q50T and thermo[3]-Q50T-K91A, Graph a) shows the pH-dependent activity profile of consensus phytase-10 (D), consensus phytase-10-thermo[3]-Q50T (A), and consensus phytase-10-thermo[3]-Q50T-K91A (A).The phytase activity was determined using the standard assay in appropriate buffers (see Example 15) at different pH-values. Graph b) shows the corresponding substrate specificity tested by replacement of phytate by the indicated compounds in the standard assay; open bars, consensus phytase-10 (white bars, consensus phytase-10-thermo-Q50T; dark bars, consensus phytase-10-thermo-Q50T-K91A). The numbers correspond to the following compounds: 1, phytate; 2, p-nitro phenyl phosphate; 3, phenyl phosphate; 4, fructose-1,6-bisphosphate; 5, fructose-6-phosphate; 6, glucose-6-phosphate; 7, ribose-5-phosphate; 8, DL-glycerol-3-phosphate; 9, glycerol-2-phosphate; 10, 3-phosphoglycerate; 11, phosphoenolpyruvate; 12, AMP; 13, ADP; 14, ATP.<br>
Figure 16: pH-dependent activity profile and substrate specificity of consensus phytase-1-thermo[8]-Q50T and of consensus phytase-l-thermo[8]-Q50T-K91A.. Graph a) shows the<br>
pH-dependent activity profile of the Q50T- (â– ) and the Q50T-K91A-variant (A).The phytase activity was determined using the standard assay in appropriate buffers (see Example 15) at different pH-values Graph b) shows the corresponding substrate specificities tested by replacement of phytate by the indicated compounds in the standard assay (open bars, consensus phytase-l-thermo[8]-Q50T; filled bars, consensus phytase-1-thermo[8]-Q50T-K91A). The substrates are listed in the legend of Figure 15.<br>
Figure 17: Differential scanning calorimetry (DSC) of consensus phytase-l-thermo[8]-Q50T and consensus phytase-l-thermo[8]-Q50T-K91 A. The protein samples were concentrated to ca. 50-60 mg/ml and extensively dialyzed against 10 mM sodium acetate, pH 5.0. A constant heating rate of 10 Â°C/min was appUed up to 95 Â°C. DSC of consensus phytase-l-thermo[8]-Q50T (upper graph) showed a melting temperature of 84.7 Â°C, while the melting point of consensus phytase-l-thermo[8]-Q50T-K91A was found at 85.7 Â°C.<br><br>
Figure 18: Comparison of the temperature optimum between consensus phytase-1, consensus phytase-l-thermo[3] and consensus phytase-l-thermo[8]. For the determination of the temperature optimum, the phytase standard assay was performed at a series of temperatures between 37 and 86 Â°C. Purified protein from the supernatant of transformed S. cerevisiae strains was used for the determination. O, consensus phytase-l;n, consensus phytase-1-thermo[3]; A, consensus phytase -l-thermo[8].<br>
Figure 19: Comparison of the pH-dependent activity profile and substrate specificity of consensus phytase-1, consensus phytase-7, and of the phytase from A. in errs 3135..<br>
Graph a) shows the pH-dependent activity profile of consensus phytase-1 (B), the phytase from A. niger NRRL 3135 (O), and of consensus phytase-7 (A).The phytase activity was determined using the standard assay in appropriate buffers (see Example 15) at different pH-values Graph b) shows the corresponding substrate specificity tested by replacement of phytate by the indicated compounds in the standard assay (black bars, A. niger NRRL 3135 phytase; grew bars, consensus phytase-1, dashed bars, consensus phytase-7). The substrates are listed in the legend of Figure 15.<br>
Figure 20: Differential scanning calorimetry (DSC) of the phytase from A. fumigates ATCC 13073 and of its stabilized a-mutant, which contains the following amino acid exchanges: F55Y, VIOOI, F114Y, A243L, S265P, N294D.<br>
The protein samples were concentrated to ca. 50-60 mg/ml and extensively dialyzed against 10 mM sodium acetate, pH 5.0. A constant heating rate of 10 *^C/min was applied up to 95 Â°C. DSC of consensus A. fumigatus 13073 phytase (lower graph) revealed a melting temperature of 62.5 Â°C, while the melting point of the a-mutant was found at 67.0 Â°C<br>
Figure 21: Comparison of the temperature optimum of A. fumigatus 13073 wild-type phytase, its a-mutant, and a further stabilized a-mutant (E59A-S126N-R329H-S364T-G404A). For the determination of the temperature optimum, the phytase standard assay was performed at a series of temperatures between 37 and 75 Â°C. The diluted supernatants of transformed S. cerevisiae strains were used for the determination. The other components of the supernatant showed no influence on the determination of the temperature optimum. O, A. fumigatus ATCC 13073 phytase; A, A. fumigatus ATCC 13073 a-mutant; D, A fumigatus ATCC 13073 alpha-mutant-(E59A-S126N-R329H-S364T-<br><br>
G404A)-Q27T; U ^ A, fumigatus ATCC 13073 a-mutant-(E59A-S126N-R329H-S364T-G404A)-Q27T-K68A. The mutations Q51T and K92A in the A formicates (X-mutants correspond to -1 Q50T and K91A in consensus phytase, respectively.<br>
Figure 22: Amino acid sequence of consensus phytase -12 (consphyl2) which contains a number of active site residues transferred from the "basidio" consensus sequence to consensus phytase-10-thermo[3]-Q50T-K91A.<br>
The culture medium used in the fermentation process in accordance with the present invention usually contains nutrients for the cells or microorganisms such as digestible nitrogen sources and inorganic substances, vitamins, micro- and trace elements and other growth-promoting factors. In addition, the culture medium contains a carbon source. Various organic or inorganic substances maybe used as nitrogen sources in the fermentation process in accordance with the present invention, such as nitrates, ammonium salts, yeast extract, meat extract, peptone, casein, corn steep liquor, amino acids and urea. Typical inorganic substances that can be used in the fermentation are calcium, iron, zinc, nickel, manganese, cobalt, copper, molybdenum, and alkali salts such as chlorides, caliphates and phosphates as well as boric acid. As a carbon source, glycerol or sugar-like mono-, di-, oblige- or polysaccharides, e.g., glucose, fructose, sucrose, maltose, starch, glycogen, cellulose or substrates containing such substances, e.g., molasses, glucose syrups and fructose syrups can be used. The concentration of glucose and / or methanol in the total feed stream may vary from about 10 to about 500 g/1 for each component and is preferably from about 200 to about 300 g/1. While the fermentation medium is principally an aqueous medium such medium may contain organic solvents such as alcohols, e.g. methanol, ethanol or isopropanol. Further, the fermentation medium may also be a dispersion or suspension, in which case the fermentation is suitably carried out with stirring.<br>
For continuous operation, the cells are optionally immobilized on a solid porous carrier. Any solid porous carrier with any porosity, size and geometry conventionally used in fermentation processes and exerting no toxic effects on the particular cell or microorganism which is to be immobilized can be used for the purpose of this invention. Examples of such carriers are those made from inorganic material and having a pore diameter of from about 0.5 to about 100 preferably from about 10 to about 30 [Am diameter. Examples of inorganic materials are ceramics and natural minerals such as<br><br>
steatite, zeolite, bentonite, silicates (glasses), aluminum silicates, aluminum oxide, magnesium aluminum silicates and magnesium aluminum oxides. Such carriers are commercially available, e.g., from Cremate, Marktredwitz, Germany, Schott Engineering GmbH, Mainz, Germany and others. Preferably, the carriers are spherical with a mean diameter of from about 0,2 to about 20 mm diameter. The carriers can be loaded with the living cells in a manner known per se by contacting the carrier particles with an appropriate cell culture. If desired, the carrier particles loaded with the cells can be further processed by applying a membrane-type coating layer, such as described in German Offenlegungsschrift DE 3421049. Suitably, the carrier is present in the fermentation vessel on a fixed bed. Further, the culture medium, its components and their containments, respectively are suitably sterilized prior to use if auto sterilization (e.g., by methanol, ethanol, ammonia) cannot be guaranteed. Heat sterilization with steam (e.g., at 121Â°C and 1 bar pressure during 20 minutes) and filtration (0.2 m) for sensitive components are preferred. Alternative sterilization methods maybe applied. Media components need not necessarily be sterilized when running the process in continuous mode.<br>
Depending on the particular cell or organism used the fermentation may be carried out at a pH between about 2 and about 11. In a preferred aspect of the invention, the fermentation process for the manufacture of phytase is carried out using the microorganism, Hinesville polymorphs transformed by a phytase encoding DNA sequence as described in EP 897 010, EP 897 985, or Example 11 of the present case. According to that particular aspect of the invention, the preferred carbon source is a mixture of glucose and methanol. Further, in accordance with that particular aspect of the invention, the fermentation may be carried out at a pH betweeen about 4 and 5, preferably at about pH 4.6. A preferred temperature range for carrying out such fermentation process is between about 10 and 50 Â°C, more preferably the fermentation temperature is about 30 Â°C. The aeration rate is preferably adjusted to between about 0.01 and about 1.5 volume of gas per volume of liquid with a dissolved oxygen concentration (DO) of in between 0.01 and about 500 %. A DO of 100 % denotes oxygen saturation of the solution at atmospheric pressure (1 bar) and reactor temperature. The fermentation can be carried out at a pressure of from about 0.1 to about 100 bar, preferably, the fermentation is carried out at atmospheric pressure, i.e., at about 1 bar. The dilution rate can vary from about 0.001 to about 0.5 per hour.<br>
The invention is illustrated further by the Examples given below.<br><br><br><br>
This solution was sterilized at 121 Â°C for 20 minutes.<br><br>
(D-biotin,600mg/l<br>
thiamin-HCl 200 g/1 in 50 % isopropanol/water)<br>
The vitamin solution was sterilized by filtration (0.2) and added to the salt solution that was sterilized at 121 ^C for 20 minutes.<br>
1.5	Glucose Solution<br>
770 g of D-glucose â€¢ H20 were dissolved in 480 g of water and sterilized (121 Â°C, 20 min) to yield 11 solution containing 57 % (by weight) of D-glucose.<br>
1.6	Methanol<br>
Pure methanol was assumed to be sterile and filled into a sterilized flask.<br><br><br><br><br><br>
Analytics:<br>
Bio-Rad Protein Assay Kit I (Bio-Rad, Glattbrugg, Switzerland) was used to determine the total protein concentration. A factor for the calculation of phytase concentration (Cphyt) from total protein concentration (ctp) was determined as Cphyt = 0.76 â€¢ Ctp.<br>
To determine the biomass in the medium two samples of 1 ml were centrifuged, washed with 1 ml of water, centrifuged again, dried at 85 ^C for two days and weighed.<br>
Results:<br>
Under the above process conditions the biomass was 59 g/1. Given a dilution rate of 0.0067 per hour the productivity was 0.078 g of phytase per litre per hour.<br>
In a fermentation that was run fed-batch-wise the biomass was 125 g/1; the productivity, however, was calculated to 0.054 g phytase per litre per hour.<br>
Example 3<br>
A fermentation in analogy to Example 2 but omitting the steatite spheres (i.e., without immobilisation of the microorganism) was carried out. A nutrient and a salt and vitamin solution of the following composition were pumped into the reactor separately:<br><br><br><br><br>
The supply of these two solutions was adjusted to provide in the feed stream a concentration of 51 g/1 of Nutrient Solution and 61 g/1 of Salts + Vitamins Solution. The dilution rate was adjusted to 0.009 h'^ The pH was kept at 4.6 by addition of 12.5 wt% ammonium hydroxide.<br>
Furthermore, Glucose Solution as in Example 1 and methanol were fed into the reactor separately to maintain a glucose concentration of 275 g/1 and a methanol concentration of 260 g/1 in the feed stream.<br>
The productivity of this fermentation was 0.088 g phytase per litre per hour. Biomass in outflow was 58 g/1.<br>
Example 4<br>
In a fermentation process in analogy to Example 3 but adjusting glucose concentration to 290 g/1, methanol concentration to 260 g/1, and keeping the dilution rate constant at 0.009 h'\ the productivity was 0.092 g phytase per litre per hour. Biomass in outflow was 60.4<br>
g/1-<br>
Example 5<br>
In a fermentation process in analogy to Example 3 but adjusting glucose concentration to 270 g/1, methanol concentration to 280 g/1, and keeping the dilution rate constant at 0.009 h'\ the productivity was 0.094 g phytase per litre per hour. Biomass in outflow was 56.8<br>
g/1-<br><br>
Example 6: Design of the amino acid sequence of consensus phytase-1<br>
Alignment of the amino acid sequences<br>
The alignment was calculated using the program PILEUP from the GCG Sequence Analysis Package Release 9.0 (Devereux etal.y 1984) with the standard parameters (gap creation penalty 12, gap extension penalty 4). The location of the gaps was refined using a text editor. Table 1 shows the sequences (see Figure 2), without the signal sequence, that were used for the performance of the alignment starting with the amino acid (aa) as mentioned in Table 1.<br>
Table 1: Origin and vote weight of the photoset amino acid sequences used for the design of consensus phvtase-1<br>
-	phyA from Aspergillus terreus 9A-1, aa 27, vote weight 0.5 (Mitchell et a/., 1997)<br>
-	phyA from Aspergillus terreus cbsll6.46, aa 27, vote weight 0.5 (EP 897 985; Figur<br>
1)<br>
-	phyA from Aspergillus niger var. awamori, aa 27, vote weight 0.33 (Piddington et aL,<br>
1993)<br>
-	phyA from Aspergillus niger T213, aa 27, vote weight 0.33<br>
-phyA from Aspergillus niger strain NRRL3135, aa 27, vote weight 0.33 (van Hartingsveldt era/., 1993)<br>
-	phyA from Aspergillus fumigatus ATCC 13073, aa 26, vote weight 0.2 (Pasamontes et al, 1997)<br>
-	phyA from Aspergillus fumigatus ATCC 32722, aa 26, vote weight 0.2 (EP 897 985; Figur 1)<br>
-phyA from Aspergillus fumigatus ATCC 58128, aa 26, vote weight 0.2 (EP 897 985; Figur 1)<br>
-	phyA from Aspergillus fumigatus ATCC 26906, aa 26, vote weight 0.2 (EP 897 985; Figur 1)<br>
-	phyA from Aspergillus fumigatus ATCC 32239, aa 30, vote weight 0.2 (EP 897 985; Figur 1)<br>
-	phyA from Emericella nidulans , aa 25, vote weight 1.0 (Pasamontes et a/., 1997a)<br>
-	phyA from Talaromyces thermophihis ATCC 20186, aa 24, vote weight 1.0 (Pasamontes etai, 1997a)<br>
-	phyA from Myceliophthora thermopiles, aa 19, vote weight 1.0 (Mitchell et al., 1997)<br><br>
Calculation of the amino acid sequence of consensus phytase-1<br>
Using the refined alignment as input, the consensus sequence was calculated by the program PRETTY from the GCG Sequence Analysis Package Release 9.0 (Devereux et al, 1984). PRETTY prints sequences with their columns aligned and can display a consensus sequence for an alignment. A vote weight that pays regard to the similarity between the amino acid sequences of the aligned phytases was assigned to all sequences. The vote weight was set in such a way that the combined impact of all phytases from one sequence subgroup (same species, but from different strains), e. g. the amino acid sequences of all phytases from A. fumigates on the election was set one, that means that each sequence contributes with a value of 1 divided by the number of strain sequences (see Table 1), By this means, it was possible to prevent that very similar amino acid sequences, e, g. of the phytases from different A. ftimigatus strains, dominate the calculated consensus sequence.<br>
The program PRETTY was started with the following parameters: The plurality defining the number of votes below which there is no consensus was set on 2.0. The threshold, which determines the scoring matrix value below which an amino acid residue may not vote for a coalition of residues, was set on 2. PRETTY used the PrettyPep.Cmp consensus scoring matrix for peptides.<br>
Ten positions of the alignment (position 46, 66, 82, 138,162, 236, 276, 279, 280, 308; Figure 2), for which the program was not able to determine a consensus residue, were filled by hand according to the following rules: if a most frequent residue existed, this residue was chosen (138, 236, 280); if a prevalent group of similar equivalent residues occurred, the most frequent or, if not available, one residue of this group was selected (46, 66, 82, 162, 276, 308). If there was neither a prevalent residue nor a prevalent group, one of the occurring residues was chosen according to common assumptions on their influence on the protein stability (279). Eight other positions (132, 170, 204, 211, 275, 317, 384,447; Figure 2) were not filled with the amino acid residue selected by the program but normally with amino acids that occur with the same frequency as the residues that were chosen by the program. In most cases, the slight underrating of the three A. niger sequences (sum of the vote weights: 0.99) was eliminated by this correction.<br>
Conversion of the consensus phytase-1 amino acid sequence to a DNA sequence<br>
The first 26 amino acid residues of A. terreus cbsl 16.46 phytase were used as signal peptide and, therefore, fused to the N-terminus of all consensus phytases. For this stretch, we used a special method to calculate the corresponding DNA sequence. Purvis et al<br><br>
(1987) proposed that the incorporation of rare co dons in a gene has an influence on the folding efficiency of the protein. The DNA sequence for the signal sequence was calculated using the approach of Purvis et al (1987) and optimized for expression in S. cerevisiae. For the remaining parts of the protein, we used the codon frequency table of highly expressed S. cerevisiae genes, obtained from the GCG program package, to translate the calculated amino acid sequence into a DNA sequence.<br>
The resulting sequence of the/cp gene is shown in Figure 3.<br>
Construction and cloning of the consensus phytase-1 gene<br>
The calculated DNA sequence of consensus phytase-1 (Jip) was divided into oligonucleotides of 85 bp, alternately using the sequence of the sense and the anti-sense strand. Every oligonucleotide overlaps 20 bp with its previous and its following oligonucleotide of the opposite strand. The location of all primers, purchased from Microsynth, Balzac (Switzerland) and obtained in a PAGE-purified form, is indicated in Figure 3.<br>
PCR-Reactions<br>
In three PCR reactions, the synthesized oligonucleotides were composed to the entire gene. For the PCR, the High Fidelity Kit from Bushranger Mannheim (Bushranger Mannheim, Germany) and the thermo cycle The Protocol (TM) from AMS Biotechnology (Europe) Ltd. (Lugano, Switzerland) were used.<br>
Oligonucleotides CP-1 to CP-10 (Mix 1, Figure 3) were mixed to a concentration of 0.2 pmol/|il of each oligonucleotide. A second oligonucleotide mixture (Mix 2) was prepared with CP-9 to CP-22 (0.2 pmol/|Al of each oUgonucleotide). Additionally, four short primers were used in the PCR reactions:<br>
CP-a:	Eco RI<br>
5'-TATATGAATTCATGGGCGTGTTCGTC-3'	(SEQ ID No. 1)<br>
CP-b:<br>
5'-TGAAAAGTTCATTGAAGGTTTC-3'	(SEQ ID No. 2)<br>
CP-c:<br>
5'-TCTTCGAAAGCAGTACAAGTAC-3'	(SEQ ID No. 3)<br><br><br><br>
step 1	2 min - 94X<br>
step 2	30 sec - 94X<br>
step 3	30 sec - 55C<br>
step 4	1 min - 72C<br>
Steps 2 to 4 were repeated 31-times.<br>
The resulting PCR product (1.4 kb) was purified as mentioned above, digested with Eco RI, and ligated in an Eco Rl-digested and dephosphorylated pBsk(-)-vector (Stratagene, La Jolla, CA, USA). 1 \i\ of the ligation mixture was used to transform E. colt XL-1 competent cells (Stratagene, La Jolla, CA, USA), All standard procedures were carried out as described by Sambrook et al (1987). The DNA sequence of the constructed consensus phytase gene {fcp. Figure 3) was controlled by sequencing as known in the art.<br>
Example 7 Design of an improved consensus phytase (consensus phytase-10) amino acid<br>
sequence<br>
The ahgnments used for the design of consensus phytase-10 were calculated using the program PILEUP from the GCG Sequence Analysis Package Release 9.0 (Devereux et aLj 1984) with the standard parameters (gap creation penalty 12, gap extension penalty 4) The location of the gaps was refined using a text editor.<br>
The following sequences were used for the alignment of the Basiodiomycete phytases starting with the amino acid (aa) mentioned in Table 2:<br>
Table 2: Origin and vote weight of five Basidiomycete phytases used for the calculation of the corresponding amino acid consensus sequence (basidio)<br>
-	phyAl from Paxillus invohitus NN005693, aa 21, vote weight 0.5 (WO 98/28409)<br>
-	phyA2 from Paxillus invohitus NN005693, aa 21, vote weight 0.5 (WO 98/28409)<br>
-	phyA from Teammates pubescens NN9343, aa 24, vote weight 1.0 (WO 98/28409)<br>
-	phyA from Aerodyne pediades NN009289, aa 19, vote weight 1.0 (WO 98/28409)<br>
-	phyA from Peripheral loci NN006113, aa 21, vote weight 1.0 (WO 98/28409)<br>
The alignment is shown in Figure 4.<br>
In Table 3 the genes, which were used for the performance of the final alignment, are arranged. The first amino acid (aa) of the sequence which is used in the alignment is mentioned behind the organism's designation.<br><br>
Table 3: Origin and vote weight of the photos sequences used for the design of consensus phytase 10<br>
-	phyA from Aspergillus terreus 9A-1, aa 27, vote weight 0.5 (Mitchell et aL, 1997)<br>
'phyA from Aspergillus terreus cbsl 16.46, aa 27, vote weight 0.5 (EP 897 985; Figur 1)<br>
-	phyA from Aspergillus niger var. awamori, aa 27, vote weight 0.5 (Piddington et at., 1993)<br>
-	phyA from Aspergillus niger strain NRRL3135, aa 27, vote weight 0.5 (van Hartingsveldt etal, 1993)<br>
' phyA from Aspergillus fumigatus ATCC 13073, aa 26, vote weight 0.2 (Pasamontes et<br>
ai, 1997) 'phyA from Aspergillus fumigatus ATCC 32722, aa 26, vote weight 0.2 (EP 897 985;<br>
Figur 1) -phyA from Aspergillus fumigatus ATCC 58128, aa 26, vote weight 0.2 (EP 897 985;<br>
Figur 1)<br>
-	phyA from Aspergillus fumigatus ATCC 26906, aa 26, vote weight 0.2 (EP 897 985;<br>
Figur 1)<br>
-phyA from Aspergillus fumigatus ATCC 32239, aa 30, vote weight 0.2 (EP 897 985;<br>
Figur 1) -phyA from Emericella nidulans , aa 25, vote weight 1.0 (Pasamontes et aL, 1997a) -phyA from Talaromyces thermophilus ATCC 20186, aa 24, vote weight 1.0 (Pasamontes<br>
etaL, 1997a)<br>
-	phyA from Myceliophthora thermophila, aa 19, vote weight 1.0 (Mitchell et aL, 1997) -phyA from Thermomyces lanuginose, aa 36, vote weight 1.0 (Berka et aL, 1998)<br>
-	Consensus sequence of five Basidiomycete phytases, vote weight 1.0 (Basidio, Figure 4)<br>
The corresponding alignment is shown in Figure 5.<br>
Calculation of the amino acid sequence of consensus phytase-10<br>
To improve the alignment, we combined the consensus sequence of five phytases from four different BasidiomyceteSy called Basidio, still containing the undefined sequence positions (see Figure 4), nearly all phytase sequences used for calculation of the original consensus phytase and one new phytase sequence from the Ascomycete Thermomyces lamiginosus to a larger alignment.<br>
We set plurality on 2.0 and threshold on 3. The used vote weights are listed in Table 3. The alignment and the corresponding consensus sequence are presented in Figure 5. The new consensus phytase -10 sequence has 32 different amino acids in comparison to the original consensus phytase. Positions for which the program PRETTY was not able to calculate a<br><br>
consensus amino acid residue were filled according to rules mentioned in Example 6. None of the residues suggested by the program was replaced.<br>
Furthermore, we included all Basidiomycete phytases as single amino acid sequences but assigning a vote weight of 0.2 in the alignment. The corresponding alignment is shown in Figure 7. The calculated consensus amino acid sequence (consensus phytase-11) has the following differences to the sequence of consensus phytase-10: D35X, X(K)69K, X(E)100E, AIOIR, Q134N, X(K)153N, X(H)190H, X(A)204S, X(E)220D, E222T, V227A, X(R)271R, H287A, X(D)288D, X(K)379K, X(I)389I, E390X, X(E)415E, X(A)416A, X(R)446L, E463A, where the numbering is as in Fig. 6.<br>
Letter X means that the program was not able to calculate a consensus amino acid; the amino acid in parenthesis corresponds to the amino acid finally included into the consensus phytase-10.<br>
We also checked single amino acid replacements suggested by the improved consensus phytase sequences 10 and 11 on their influence on the stability of the original consensus phytase -1. The approach is described in example 8.<br>
Conversion of consensus phytase-10 amino acid sequence to a DNA sequence<br>
The first 26 amino acid residues of A. terreus cbs 116.46 phytase were used as signal peptide and, therefore, fused to the N-terminus of consensus phytase-10. The used procedure is further described in Example 6.<br>
The resulting sequence of the/cplO gene is shown in Figure 6.<br>
Construction and cloning of the consensus phytase-10 gene (fcplQ)<br>
The calculated DNA sequence of/cplO was divided into oligonucleotides of 85 bp, alternately using the sequence of the sense and the anti-sense strand. Every oligonucleotide overlaps 20 bp with its previous and its following oligonucleotide of the opposite strand. The location of all primers, purchased from Microsynth, Balgach (Switzerland) and obtained in a PAGE-purified form, is indicated in Figure 6.<br><br>
PCR-Reactions<br>
In three PCR reactions, the synthesized oligonucleotides were composed to the entire gene. For the PCR, the High Fidelity Kit from Boehringer Mannheim (Boehringer<br>
TM<br>
Mannheim, Mannheim, Germany) and the thermocycler The Protokol     from AMS Biotechnology (Europe) Ltd. (Lugano, Switzerland) were used. The following oligonucleotides were used in a concentration of 0.2 pmol/ml.<br>
Mix 1.10: CP-1, CP-2, CP-3.10, CP-4.10, CP-5.10, CP-6, CP-7.10, CP-8.10, CP-9.10, CP-10.10<br>
Mix 2.10: CP-9.10, CP-10.10, CP-11.10, CP-12.10, CP-13.10, CP-14.10, CP-15.10, CP-16.10, CP-17.10, CP-18.10, CP-19.10, CP-20.10, CP-21.10, CP-22.10<br>
The newly synthesized oligonucleotides are marked by number 10. The phytase contains the following 32 exchanges, which are underlined in Figure 6, in comparison to the original consensus phytase -1: Y54F, E58A, D69K, D70G, A94K, N134Q, 1158V, S187A, Q188N, D197N, S204A, T214L, D220E, L234V, A238P, D246H, T251N, Y259N, E267D, E277Q, A283D, R291I, A320V, R329H, S364T, I366V, A379K, S396A, G404A, Q415E, A437G, A463E.<br>
Four short PCR primers were used for the assembling of the oligonucleotides:<br>
CP-a:	Eco RI<br>
5'-TATATGAArrCATGGGCGTGTTCGTC-3'	(SEQ, ID No. 1)<br>
CP-b:<br>
5'-TGAAAAGTTCATTGAAGGTTTC-3'	(SEQ, ID No. 2)<br>
CP-c.lO:<br>
5'-TCTTCGAAAGCAGTACACAAAC-3'	(SEQ, ID No. 5)<br>
CP-e:	Eco RI<br>
5'-TATATGAATTCTTAAGCGAAAC-3'	(SEQ, ID No. 4)<br>
PCR reaction a:	10 Mix 1.10 (2.0 pmol of each oligonucleotide)<br>
2 nucleotides (10 mM each nucleotide) 2 primer CP-a (10 pmol/ml) 2 primer CP- (10 pmol/ml) 10,0 1^1 PCR buffer 0.75 \x\ polymers mixture (2.6 U) 73.25 H2O<br><br>
PCR reaction b\	10 Mix 2.10 (2.0 pmol of each oligonucleotide)<br>
2 nucleotides (10 mM each nucleotide) 2 ^1 primer CP-b (10 pmol/ml) 2 \\\ primer CP-e (10 pmol/ml) 10,0 III PCR buffer 0.75 |A1 polymerized mixture (2.6 U) 73.25 \x\ H2O<br><br>
Reaction conditions for PCR reactions a and b:<br>
step 1	2 min - 45 Â°C<br>
step 2	30 sec - 72 Â°C<br>
step 3	30 sec - 94 Â°C<br>
step 4	30 sec - 52 Â°C<br>
step 5	1 min - 72 "^C<br>
Steps 3 to 5 were repeated 40-times.<br>
The PCR products (670 and 905 bp) were purified by an agarose gel electrophoresis (0.9% agarose) and a following gel extraction (QIAEXII Gel Extraction Kit, Qiagen, Hilden, Germany). The purified DNA fragments were used for the PCR reaction c.<br>
PCR reaction c:	6 |il PCR product of reaction a( =50 ng)<br>
6 \i\ PCR product of reaction b( =50 ng) 2 111 primer CP-a (10 pmol/ml) 2 |il primer CP-e (10 pmol/ml) 10,0 ^l PCR buffer 0.75 \x\ polymerase mixture (2.6 U) 73.25 \x\ H2O<br>
Reaction conditions for PCR reaction c:<br>
step 1	2 min - 94 Â°C<br>
step 2	30 sec - 94 Â°C<br>
step 3	30 sec - 55 Â°C<br>
step 4	1 min - 72 Â°C<br>
Steps 2 to 4 were repeated 31-times.<br>
The resulting PCR product (1.4 kb) was purified as mentioned above, digested with Eco RI, and ligated in an Eco Rl-digested and dephosphorylated pBsk(-)-vector (Stratagene, La Jolla, CA, USA). 1 \x\ of the ligation mixture was used to transform Â£. coli XL-1 competent cells (Stratagene, La Jolla, CA, USA). All standard procedures were carried out as described by Sambrook et a/. (1987). The DNA sequence of the constructed gene ifcplO) was checked by sequencing as known in the art.<br><br>
Example 8<br>
Increasing the thermostability of consensus phvtase-1 by introduction of single<br>
mutations suggested by the amino acid sequence of consensus phytase-10 and/or<br>
consensus phvtase-11<br>
In order to increase the thermostability of homologous genes, it is also possible to test the stability effect of each differing amino acid residue between the protein of interest and the calculated consensus sequence and to combine all stabilizing mutations into the protein of interest. We used the consensus phytase -1 as protein of interest and tested the effect on the protein stability of 34 amino acids, which differed between consensus phytase -1 on one hand and consensus phytases 10 and/or -11 on the other hand, by single mutation..<br>
To construct mutinies for expression in A. nagger, S. cerevisiaCy or H. polymorpha, the corresponding expression plasmid containing the consensus phytase gene was used as template for site-directed mutagenesis (see Examples 11 - 13). Mutations were introduced using the "quick exchange     site-directed mutagenesis kit" from Stratagene ( La Jolla, CA, USA) following the manufacturer's protocol and using the corresponding primers. All mutations made and their corresponding primers are summarized in Table 4. Plasmids harboring the desired mutation were identified by DNA sequence analysis as known in the art.<br>
Table 4: Primers used for site-directed mutagenesis of consensus phytases<br>
(Exchanged bases are highlighted in bold. The introduction of a restriction site is marked above the sequence. When a restriction site is written in parenthesis, the mentioned site was destroyed by introduction of the mutation.)<br>
mutation      Primer set<br>
Q50T	5'-CACTTGTGGGGTACCTACTCTCCATACTTCTC-3' (SEQ ID No. 6)<br>
5'-GAGAAGTATGGAGAGTAGGTACCCCACAAGTG-3' (SEQ ID No. 7)<br>
Y54F	5'-GGTCAATACTCTCCATTCTTCTCTTTGGAAG-3' (SEQ ID No. 8)<br>
5'-CTTCCAAAGAGAAGAATGGAGAGTATTGACC-3' (SEQ ID No. 9)<br>
E58A	5'-CATACTTCTCTTTGGCAGACGAATCTGC-3' (SEQ ID No. 10)<br>
5'-GCAGATTCGTCTGCCAAAGAGAAGTATG-3' (SEQ ID No. 11)<br><br>
D69K<br>
D70G<br><br>
Aatll 5'-CTCCAGACGTCCCAAAGGACTGTAGAGTTAC-3' (SEQ ID No. 12) 5'-GTAACTCTACAGTCCTTTGGGACGrCTGGAG-3' (SEQ ID No. 13)<br>
Aatll 5'-CTCCAGACGTCCCAGACGGCTGTAGAGTTAC-3' (SEQ ID No. 14) 5'-GTAACTCTACAGCCGTCTGGGACGTCTGGAG-3' (SEQ ID No. 15)<br><br><br><br>
K91A<br>
A94K<br><br>
5'-GATACCCAACTTCTTCTGCGTCTAAGGCTTACTCTG-3' (SEQ ID No. 16)<br>
5'-CAGAGTAAGCCTTAGACGCAGAAGAAGTTGGGTATC-3' (SEQ ID No. 17)<br>
Sen I 5'-CTTCTAAGTCTAAGAAGTACrCTGCTTTG-3' (SEQ ID No. 18) 5'-CAAAGCAGAGTACITCTTAGACTTAGAAG-3'(SEQ ID No. 19)<br><br><br><br>
AIOIR<br><br>
5'-GCTTACTCTGCTTTGATTGAACGGATTCA7UAGAACGCTAC-3'<br><br>
5'-GTAGCGTTCTTTTGAATCCGTTCAATCAAAGCAGAGTAAGC-3'<br><br>
1158V<br>
N134Q<br>
5'-CCATTCGGTGAACAGCAAATGGTTAACTC-3' (SEQ ID No. 22) 5'-GAGTTAACCATTTGCTGTTCACCGAATGG-3' (SEQ ID No. 23)<br>
K153N<br>
Nrul 5'-GATACAAGGCTCrCGCGAGAAACATTGTTC-3' (SEQ ID No. 24). 5'-GGAACAATGTTTCTCGCGAGAGCCTTGTATC-3' (SEQ ID No. 25)<br>
Bss HI 5'-GATTGTTCCATTCGTGCGCGCTTCTGGTTC-3' (SEQ ID No. 26) 5'-GAACCAGAAGCGCGCACGAATGGAACAATC-3' (SEQ ID No. 27)<br>
D197N<br>
Bell 5'-CTCCAGTTATTAACGTGATCATTCCAGAAGG-3' (SEQ ID No. 28) 5'-CCTTCTGGAATGATCACGTTAATAACTGGAG-3' (SEQ ID No. 29)<br>
SI 87 A<br>
Apal 5'-GGCTGACCCAGGGGCCCAACCACACCAAGC-3' (SEQ ID No. 30) 5'-GCTTGGTGTGGTTGGGCCCCTGGGTCAGCC-3' (SEQ ID No. 31)<br>
T214L<br>
Ncol 5'-CACTTTGGACCATGGTCTTTGTACTGCTTTCG-3' (SEQ ID No. 32) 5'-CGAAAGCAGTACAAAGACCATGGTCCAAAGTG-3' (SEQ ID No. 33)<br>
E222T 34)<br>
Avr II 5'-GCTTTCGAAGACTCTACCCTAGGTGACGACGTTG-3' (SEQ ID No.<br>
5'-CAACGTCGTCACCrAGGGTAGAGTCTTCGAAAGC-3' (SEQ ID No. 35)<br><br>
V227A	5'-GGTGACGACGCTGAAGCTAACTTCAC-3' (SEQ ID No. 36)<br>
5'-GTGAAGTTAGCTTCAGCGTCGTCACC-3'(SEQ ID No. 37)<br>
Sac II<br>
L234V	5'-CTAACTTCACCGCGGTGTTCGCTCCAG-3' (SEQ ID No. 38)<br>
5'-CTGGAGCGAACACCGCGGTGAAGTTAG-3' (SEQ ID No. 39)<br>
A238P	5'-GCTTTGTTCGCTCCACCTATTAGAGCTAGATTGG-3' (SEQ ID No.<br>
40)<br>
5'-CCAATCTAGCTCTAATAGGTGGAGCGAACAAAGC-3' (SEQ ID No. 41)<br>
Hpal<br>
T251N	5'-GCCAGGTGrrAACTTGACTGACGAAG-3' (SEQ ID No. 42)<br>
5'-TTCGTCAGTCAAGrrAyiCACCTGGC-3' (SEQ ID No. 43)<br>
Aatll<br>
Y259N	5'-GACGAAGACGrCGTTAACTTGATGGAC-3' (SEQ ID No. 44)<br>
5'-GTCCATCAAGTTAACGACGTCTTCGTC-3' (SEQ ID No. 45)<br>
Asp I<br>
E267D	5'-GTCCATTCGACACrGrCGCTAGAACTT C-3' (SEQ ID No. 46)<br>
5'-GAAGTTCTAGCGACAGrGTCGAATGGAC-3' (SEQ ID No. 47)<br>
E277Q	5'-CTGACGCTACTCAGCTGTCTCCATTC-3' (SEQ ID No. 48)<br>
5'-GAATGGAGACAGCTGAGTAGCGTCAG-3' (SEQ ID No. 49)<br>
A283D	5'-GTCTCCATTCTGTGATTTGTTCACTCAC-3' (SEQ ID No. 50)<br>
5'-GTGAGTGAACAAATCACAGAATGGAGAC-3' (SEQ ID No. 51)<br>
Kspl<br>
H287A	5'-GCTTTGTTCACCGCGGACGAATGGAG-3' (SEQ ID No. 52)<br>
5'-CTCCATTCGTCCGCGGTGAACAAAGC-3' (SEQ ID No. 53)<br>
BamHl<br>
R291I	5'-CACGACGAATGGArCCAATACGACTAC-3' (SEQ ID No. 54)<br>
5'-GTAGTCGTATTGGArCCATTCGTCGTG-3' (SEQ ID No. 55)<br>
Bsi WI<br>
Q292A	5'-GACGAATGGAGAGCGTACGACTACTTG-3' (SEQ ID No. 56)<br>
5'-CAAGTAGTCGrACGCTCTCCATTCGTC-3' (SEQ ID No. 57)<br>
Hpal<br>
A320V	5'-GGTGTTGGTTTCGTTAACGAATTGATTGC-3' (SEQ ID No. 58)<br>
5'-GCAATCAATTCGTrAACGAAACCAACACC-3' (SEQ ID No. 59)<br><br>
(Bgl II)<br>
R329H	5'-GCTAGATTGACTCACrcrCCAGTTCAAG-3' (SEQ ID No. 60)<br>
5'-CTTGAACTGGAGAGrGAGrCAATCTAGC-3' (SEQ ID No. 61)<br>
Eco RV<br>
S364T	5'-CTCACGACAACACTATGATATCTATTTTCTTC-3' (SEQ ID No. 62)<br>
5'-GAAGAAAATAGATATCATAGTGTTGTCGTGAG-3' (SEQ ID No. 63)<br>
Ncol<br>
I366V	5'-CGACAACTCCATGGTTTCTATTTTCTTCGC-3' (SEQ ID No. 64)<br>
5'-GCGAAGAAAATAGAAACCATGGAGTTGTCG-3' (SEQ ID No. 65)<br>
Kpnl<br>
A379K	5'-GTACAACGGTACCAAGCCATTGTCTAC-3' (SEQ ID No. 66)<br>
5'-GTAGACAATGGCTTGGrACCGTTGTAC-3' (SEQ ID No. 67)<br>
S396A	5'-CTGACGGTTACGCTGCTTCTTGGAC-3' (SEQ ID No. 68<br>
5'-GTCCAAGAAGCAGCGTAACCGTCAG-3' (SEQ ID No. 69)<br>
G404A	5'-CTGTTCCATTCGCTGCTAGAGCTTAC-3' (SEQ ID No. 70)<br>
5'-GTAAGCTCTAGCAGCGAATGGAACAG-3' (SEQ ID No. 71)<br>
Q415E	5'-GATGCAATGTGAAGCTGAAAAGGAACC-3' (SEQ ID No. 72)<br>
5'-GGTTCCTTTTCAGCTTCACATTGCATC-3' (SEQ ID No. 73)<br>
Sail<br>
A437G	5'-CACGGTTGTGGTGrCGACAAGTTGGG-3' (SEQ ID No. 74)<br>
5'-CCCAACTTGTCGACACCACAACCGTG-3' (SEQ ID No. 75)<br>
Muni<br>
A463E	5'-GATCTGGTGGCAArTGGGAGGAATGTTTCG-3' (SEQ ID No. 76)<br>
5'-CGAAACATTCCTCCCAATTGCCACCAGATC-3' (SEQ ID No. 77)<br>
and accordingly for other mutations.<br>
The temperature optimum of the purified phytases, expressed in Saccharomyces crevassing (Example 14), was determined as outlined in Example 14. Table 5 shows the effect on the stability of consensus phytase -1 for each mutation introduced.<br>
Table 5: Stability effect of the individual amino acid replacements in consensus phytase-1 (+ or - means a positive, respectively, negative effect on the protein stability up to 1 Â°C, ++ and â€” means a positive, respectively, negative effect on the protein stability between 1 and<br><br>
3 Â°C; the number 10 or 11 corresponds to the consensus phytase sequence that suggested the amino acid replacement,)<br><br><br><br>
We combined eight positive mutations (E58A, D197N, E267D, R291I, R329H, S364T, A379K, G404A) in consensus phytase-1 using the primers and the technique mentioned above in this example. Furthermore, the mutations Q50T and K91A were introduced which mainly influence the catalytical characteristics of the phytase (see EP 897 985 as well as Example 14). The DNA and amino acid sequence of the resulting phytase gene (consensus phytase-l-thermo[8]-Q50T-K91 A) is shown in Figure 8. In this way, the temperature optimum and the melting point of the consensus phytase was increased by 7 Â° C(Figurel6, 17, 18).<br>
Using the results of Table 5, we further improved the thermostability of consensus phytase 10 by the following back mutations K94A, VI581, and A396S that revealed a strong negative influence on the stability of consensus phytase -1. The resulting protein is consensus phytase-10-thermo [3]. Furthermore, we introduced the mutations Q50T and K91A which mainly influence the catalytical characteristics of consensus phytase (see EP 897 485 as well as Example 14 and Figures 15 and 16). The resulting DNA and amino acid sequence is shown in Figure 9. The optimized phytase showed a 4 ^C higher temperature optimum and melting point than consensus phytase -10 (Figures 13 and 14). Furthermore, the phytase has also a strongly increased specific activity with phytate as substrate of 250 U/mg at pH 5.5 (Figure 15).<br>
Example 9 Stabilization of the phytase of A. fumisatus ATCC 13073 by replacement of amino acid residues with the corresponding consensus phytase-1 and consensus phytase-<br>
10 residues<br>
At six typical positions where the A. fumigatus 13073 phytase is the only or nearly the only phytase in the alignment of Figure 2 that does not contain the corresponding consensus phytase amino acid residue, the non-consensus amino acid residue was replaced by the consensus one. In a first round, the following amino acids were substituted in A. fumigatus 13073 phytase, containing the Q51T substitution and the signal sequence of A. terreus cbs.116.46 phytase (see Figure 10):<br>
F55(28)Y, V100(73)I, FI14(87)Y, A243(220)L, S265(242)P, N294(282)D.<br>
The numbers in parentheses refer to the numbering of Figure 2.<br>
In a second round, four of the seven stabilizing amino acid exchanges (E59A, R329H, S364T, G404A) found in the consensus phytase-10 sequence and, tested as single<br><br>
mutations in consensus phytase-1 (Table 5), were additionally introduced into the A. fumigatus (X-mutant. Furthermore, the amino acid replacement S154N, shown to reduce the protease susceptibility of the phytase, was introduced.<br>
The mutations were introduced as described in example 8 (see Table 6) and expressed as described in example 11 to 13. The resulting A. fumigatus 13073 phytase variants were called a-mutant and a-mutant-E59A-S154N-R329H-S364T-G404A.<br>
The temperature optimum (60 Â°C, Figure 21) and the melting point (67.0 Â°C, Figure 20) of the A. fumigatus 13073 phytase a-mutant were increased by 5 - 7Â°C in comparison to the values of the wild-type (temperature optimum: 55 Â°C, T^: 60 Â°C). The five additional amino acid replacements further increased the temperature optimum by 3 Â°C (Figure 21).<br>
Table 6: Mutagenesis primers for stabilization of A. fumigatus phytase ATCC 13073<br>
Mutation      Primer<br>
F55Y	5'-CACGTACTCGCCATACTTTTCGCTCGAG-3' (SEQ ID No. 78)<br>
5'-CTCGAGCGAAAAGTATGGCGAGTACGTG-3' (SEQ ID No. 79)<br>
{Who I)<br>
E58A	5'-CCATACTTTTCGCTCGCGGACGAGCTGTCCGTG-3' (SEQ ID NO. 80)<br>
5'-CACGGACAGCTCGTCCGCGAGCGAAAAGTAGG-3' (SEQ ID NO. 81)<br>
VIOOI	5'-GTATAAGAAGCTTATTACGGCGATCCAGGCC-3' (SEQ ID No. 82)<br>
5'-GGCCTGGATCGCCGTAATAAGCTTCTTATAC-3' (SEQ ID No. 83)<br>
114Y	5'-CTTCAAGGGCAAGTACGCCTTTTTGAAGACG-3' (SEQ ID No. 84)<br>
5'-CGTCTTCAAAAAGGCGTACTTGCCCTTGAAG-3' (SEQ ID No. 85)<br>
^2431	5'-CATCCGAGCTCGCCTCGAGAAGCATCTTC-3' (SEQ ID No. 86)<br>
5'-GAAGATGCTTCTCGAGGCGAGCTCGGATG-3' (SEQ ID No. 87)<br>
S265P	5'-CTAATGGA TGTGTCCGTTTGATACGGTAG-3' (SEQ ID No. 88)<br>
5'-CTACCGTATCAAACGGACACATGTCCATTAG-3' (SEQ ID No. 89)<br><br>
N294D<br>
R329H<br>
S364T<br>
G404A<br><br>
5'-GTGGAAGAAGTACGACTACCTTCAGTC-3' (SEQ ID No. 90) 5'-GACTGAAGGTAGTCGTACTTCTTCCAC-3' (SEQ ID No. 91)<br>
{Mlu I) 5'-GCCCGGTTGACGCArrCGCCAGTGCAGG-3' (SEQ ID No. 92) 5'-CCTGCACTGGCGAATGCGTCAACCGGGC-3' (SEQ ID No. 93)<br>
Ncol 5'-CACACGACAACACCArGGTTTCCATCTTC-3' (SEQ ID No. 94) 5'-GAAGATGGAAACCATGGTGTTGTCGTGTG-3' (SEQ ID No. 95)<br>
(Bss HI) 5'-GTGGTGCCTTTCGCCGCGCGAGCCTACTTC-3' (SEQ ID No. 96) 5'-GAAGTAGGCTCGCGCGGCGAAAGGCACCAC-3' (SEQ ID No. 97)<br><br>
Example 10 Introduction of the active site amino acid residues of the A. niser NRRL 3135<br>
phytase into the consensus phytase-1<br>
We used the crystal structure of the Aspergillus niger NRRL 3135 phytase to define all active site amino acid residues (see Reference Example and EP 897 010). Using the alignment of Figure 2, we replaced the following active site residues and additionally the non-identical adjacent ones of the consensus phytase -1 by those of the A. niger phytase:<br>
S89D, S92G, A94K, D164S, P201S, G203A, G205S, H212P, G224A, D226T, E255T, D256E, V258T, P265S, Q292H, G300K, Y305H, A314T, S364G, M365I, A397S, S398A, G404A, and A405S<br>
The new protein sequence consensus phytase -7 was back translated into a DNA sequence (Figure 11) as described in Example 6. The corresponding gene {fcp7) was generated as described in Example 6 using the following oligonucleotide mixes:<br>
Mix 1.7: CP-1, CP-2, CP-3, CP-4.7, CP-5.7, CP-6, CP-7, CP-8.7, CP-9, CP-10.7<br>
Mix 2.7: CP-9, CP-10.7, CP-11.7, CP-12.7, CP-13.7, CP-14.7, CP-15.7, CP-16, CP-17.7, CP-18.7, CP-19.7, CP-20, CP-21, CP-22.<br>
The DNA sequences of the oligonucleotides are indicated in Figure 11. The newly synthesized oligonucleotides are additionally marked by number 7. After assembling of the<br><br>
oligonucleotides using the same PCR primers as mentioned in Example 6, the gene was cloned into an expression vector as described in Examples 11-13.<br>
The pH-profile of consensus phytase-7, purified after expression in Pansexual " polymorpha, was very similar to that of A. niger NRRL 3135 phytase (see Figure 19).<br>
Example 11 Expression of the consensus phytase genes in Hansen polymorpha<br>
The phytase expression vectors, used to transform H. polymorpha RBI 1 (Gellissen et al, 1994), were constructed by inserting the Eco RI fragment of pesky"/cp or variants thereof into the multiple cloning site of the K polymorpha expression vector pFPMT121, which is based on an ura3 selection marker from S. cerevisiaCy a formate dehydrogenises {FMD) promoter element and a methanol oxidize {MO) termimator element from H. polymorpha. The 5' end of the/cp gene is fused to the FMD promoter, the 3' end to the MOX terminator (GeUissen et al.y 1996; EP 0299 108 B). The resulting expression vectors were designated pFPMT/cp, pFPMT/cpiO, pFPMT/cp7.<br>
The constructed plasmids were propagated in E. colt, Plasmid DNA was purified using standard state of the art procedures. The expression plasmids were transformed into the H. polymorpha strain RPll deficient in orotidine-5'-phosphate decarboxylase {ura3) using the procedure for preparation of competent cells and for transformation of yeast as described in Gelissen et al (1996). Each transformation mixture was plated on YNB (0.14% w/v Difco YNB and 0.5% ammonium sulfate) containing 2% glucose and 1.8% agar and incubated at 37. After 4 to 5 days individual transformant colonies were picked and grown in the liquid medium described above for 2 days at 37 *^C. Subsequently, an aliquot of this culture was used to inoculate fresh vials with YNB-medium containing 2% glucose. After seven further passages in selective medium, the expression vector is integrated into the yeast genome in multimeric form. Subsequently, mitotically stable transformants were obtained by two additional cultivation steps in 3 ml non-selective Uquid medium (YPD, 2% glucose, 10 g yeast extract, and 20 g peptone). In order to obtain genetically homogeneous recombinant strains an aliquot from the last stabilization culture was plated on a selective plate. Single colonies were isolated for analysis of phytase expression in YNB containing 2% glycerol instead of glucose to derepress the find promoter. Purification of the consensus phytases was done as described in Example 12.<br><br>
Example 12<br>
Expression of the consensus phytase genes in Saccharomvces cerevisiae and<br>
purification of the phytases from culture supernatant<br>
The consensus phytase genes were isolated from the corresponding Bluescript-<br>
plasmid (pBsk'/cp, pBSK'/cpiO, pBsk'/cp7) and ligated into the Eco RI sites of the<br>
expression cassette of the Saccharomyces cerevisiae expression vector pYES2 (Invitrogen,<br>
San Diego, CA, USA) or subcloned between the shortened GAPFL (glyceraldhyde-3-<br>
phosphate dehydrogenase) promoter and the pho5 terminator as described by Janes et aL<br>
(1990). The correct orientation of the gene was checked by PCR. Transformation of S.<br>
cerevisiae strains, e. g. INVScl (Invitrogen, San Diego, CA, USA) was done according to<br>
Hinnen et al (1978). Single colonies harboring the phytase gene under the control of the<br>
GAPFL promoter were picked and cultivated in 5 ml selection medium (SD-uracil,<br>
Sherman et al.y 1986) at SO'^C under vigorous shaking (250 rpm) for one day. The<br>
preculture was then added to 500 ml YPD medium (Sherman et al, 1986) and grown<br>
under the same conditions. Induction of the gall promoter was done according to the<br>
manufacturer's instructions. After four days of incubation cell broth was centrifuged (7000<br>
rpm, GS3 rotor, 15 min, 5) to remove the cells and the supernatant was concentrated by<br>
way of ultrafiltration in Amicon 8400 cells (PM30 membranes) and ultrafree-15<br>
centrifugal filter devices (Biomax-30K, Millipore, Bedford, MA, USA). The concentrate<br>
(10 ml) was desalted on a 40 ml Sephadex G25 Superfine column (Pharmacia Biotech,<br>
Freiburg, Germany), with 10 mM sodium acetate, pH 5.0, serving as elution buffer. The desalted sample was brought to 2 M (NH4)2S04 and directly loaded onto a 1 ml Butyl<br>
Sepharose 4 Fast Flow hydrophobic interaction chromatography column (Pharmacia<br>
Biotech, Freiburg, Germany) which was eluted with a linear gradient from 2 M to 0 M (NH4)2S04 in 10 mM sodium acetate, pH 5.0. Phytase was eluted in the break-through,<br>
concentrated and loaded on a 120 ml Sephacryl S-300 gel permeation chromatography column (Pharmacia Biotech, Freiburg, Germany). Consensus phytase -1 and consensus phytase -7 eluted as a homogeneous symmetrical peak and was shown by SDS-PAGE to be approx. 95% pure.<br>
Example 13 Expression of the consensus phytase genes in Aspergillus niser<br>
The Bluescript-plasmids pBsk'/cp, pBSK'/cpiO, and pBsk"/cp7 were used as template for the introduction of a Bsp Hl-site upstream of the start codon of the genes and an Eco RV-site downstream of the stop codon. The Expand High Fidelity PCR Kit (Boehringer Mannheim, Mannheim, Germany) was used with the following primers:<br><br>
Primer Asp-1:<br>
Bsp HI<br>
5'-TATATCATGAGCGTGTTCGTCGTGCTACTGTTC-3' (SEQ ID No. 98)<br>
Primer Asp-2 used for cloning of/cp and/cp7:<br>
EcoRV a'-ACCCGACTTACAAAGCGAATTCTATAGATATAT-S' (SEQ ID No. 99)<br>
Primer Asp-3 used for cloning offcplO:<br>
EcoRV 3'-ACCCTTCTTACAAAGCGAATTCTArAGATATAT-5' (SEQ ID No. 100)<br>
The reaction was performed as described by the suppUer. The PCR-ampHfied/cp-genes had a new Bsp HI site at the start codon, introduced by primer Asp-1, which resulted in a replacement of the second amino acid residue glycine by serine. Subsequently, the DNA-fragment was digested with Bsp HI and Eco RV and ligated into the Nco I site downstream of the glucoamylase promoter of Aspergillus niger (glciA) and the Eco RV site upstream of the Aspergillus nidulans tryptophan C terminator {trpC) (Mullaney et a/., 1985). After this cloning step, the genes were sequenced to detect possible failures introduced by PCR. The resulting expression plasmids which basically correspond to the pGLAC vector as described in Example 9 of EP 684 313 contained the orotidine-5'-phosphate decarboxylase gene ipyr4) of Neurospora crassa as a selection marker. Transformation of Aspergillus niger and expression of the consensus phytase genes was done as described in EP 684 313. The consensus phytases were purified as described in Example 12.<br>
Example 14 Determination of phytase activity and of temperature optimum<br>
Phytase activity was determined basically as described by Mitchell et al (1997). The activity was measured in an assay mixture containing 0.5% phytic acid (=5 mM) in 200 mM sodium acetate, pH 5.0. After 15 min of incubation at 37 Â°C, the reaction was stopped by addition of an equal volume of 15% trichloroacetic acid. The liberated phosphate was quantified by mixing 100 of the assay mixture with 900 \x\. H2O and 1 ml of 0.6 M H2SO4, 2% ascorbic acid and 0.5% ammonium molybdate. Standard solutions of potassium phosphate were used as reference. One unit of enzyme activity was defined as the amount of enzyme that releases 1 phosphate per minute at 37 Â°C. The protein concentration was determined using the enzyme extinction coefficient at 280 nm<br><br>
calculated according to Pace et al (1995): consensus phytase -1.101; consensus phytase -7, 1.068; consensus phytase -1 10, 1.039.<br>
In case of pH-optimum curves, purified enzymes were diluted in 10 mM sodium acetate, pH 5.0. Incubations were started by mixing aliquots of the diluted protein with an equal volume of 1% phytic acid (=10 mM) in a series of different buffers: 0.4 M glycine/HCl, pH 2.5; 0.4 M acetate/NaOH, pH 3.0, 3.5,4.0,4.5, 5.0, 5.5; 0.4 M imidazole/HCl, pH 6.0, 6.5; 0.4 M Tris/HCl pH 7.0, 7.5, 8.0, 8.5, 9.0. Control experiments showed that pH was only slightly affected by the mixing step. Incubations were performed for 15 min at 37 Â°C as described above.<br>
For determinations of the substrate specificities of the phytases, phytic acid in the assay mixture was replaced by 5 mM concentrations of the respective phosphate compounds. The activity tests were performed as described above.<br>
For determination of the temperature optimum, enzyme (100) and substrate solution (100 |il) were pre-incubated for 5 min at the given temperature. The reaction was started by addition of the substrate solution to the enzyme. After 15 min incubation, the reaction was stopped with trichloroacetic acid and the amount of phosphate released was determined.<br>
The pH-optimum of the original consensus phytase was around pH 6.0-6.5 (80 U/mg). By introduction of the Q50T mutation, the pH-optimum shifted to pH 6.0 (130 U/mg). After introduction of K91A, the pH optimum shifted one pH-unit into the acidic pH-range showing a higher specific activity between pH 2.5 and pH 6.0. That was shown for the stabilized mutants and for consensus phytase-10, too (Figure 15 and 16).<br>
Consensus phytase-7, which was constructed to transfer the catalytic characteristics of the A. niger NRRL 3135 phytase into consensus phytase-1, had a pH-profile very similar to that of A. niger NRRL 3135 phytase (see Figure 19). The substrate specificity of consensus phytase-7 also resembled more to that of A. niger NRRL 3135 phytase than to that of consensus phytase-1.<br>
The temperature optimum of consensus phytase-1 (71 Â°C) was 16-26 Â°C higher than the temperature optimum of the wild-type phytases (45-55 Â°C, Table 7) which were used to calculate the consensus sequence. The improved consensus phytase-10 showed a further increase of its temperature optimum to 80 Â°C (Figure 12). The temperature optimum of<br><br>
the consensus phytase-l"thermo[8] phytase was found in the same range (78 Â°C) when using the supernatant of an overproducing S. cerevisiae strain. The highest temperature optimum reached of 82 Â°C was determined for consensus phytase-10-thermo[3]-Q50T-K91A.<br>
Table 7: Temperature optimum and Tm-value of consensus phytase and of the phytases from A. fumigatuSy A. nigevy E. nidiilans, and M. thermophila. The determination of the temperature optimum was performed as described in Example 14. The T-values were determined by differential scanning calorimetry as described in Example 15.<br><br><br>
Example 15 Determination of the melting point by differential scanning calorimetry (DSC)<br>
In order to determine the unfolding temperature of the phytases, differential scanning calorimetry was applied as previously published by Lehman et al (2000). Solutions of 50-60 mg/ml homogeneous phytase were used for the tests. A constant heating rate of 10 Â°C/min was applied up to 90-95 Â°C.<br>
The determined melting points reflect the results obtained for the temperature optima (Table 7). The most stable consensus phytase designed is consensus phytase-10-thermo[3]-Q50T-K91A showing a melting temperature under the chosen conditions of 89.3 '. This is 26 to 33.6 Â°C higher than the melting points of the wild-type phytases used.<br>
Example 16<br>
Transfer of basidiomycete phytase active site into consensus phvtase-10-thermo[3l-O50T-K91A<br>
As described previously (Example 8), mutations derived from the basidiomycete phytase active site were introduced into the consensus phytase -10. The following five constructs a) to e) were prepared:<br>
a)	This construct is called consensus phytase -12, and it comprises a selected number of active site residues of the basidio consensus sequence. Its amino acid sequence (consphyl2) is shown in Fig. 22 (the first 26 amino acids forms the signal peptide, amended positions are underlined);<br>
b)	a cluster of mutations (Cluster II) was transferred to the consensus phytase 10 sequence, viz.: S80Q, Y86F, S90G, K91A, S92A, K93T, A94R, Y95I;<br>
c)	analogously, another cluster of mutations (Cluster III) was transferred, viz.: T129V, E133A, Q143N, M136S, V137S, N138Q, S139A;<br>
d)	analogously, a further cluster of mutations (Cluster IV) was transferred, viz.: A168D, E171T,K172N,F173W;<br><br><br><br><br>
WE CLAIM;<br>
1.       A fermentation assembly comprising<br>
(a)	a vessel for cohering living cells;<br>
(b)	at least two storage flasks in fluid communication with the vessel for supply of liquids and a first transport means for transferring the liquids from the storage flasks to the vessel;<br>
(c)	individual volumetric cow mass flow rate measuring instruments operably connected to the transport means, for monitoring the supply of the contents of the stage flasks to the vessel;<br>
(d)	a harvest flask in fluid communication with the vessel and a coccid transport means for transferring the fermentation broth from the vessel to the harvest flask; and<br>
(e)	a system containing a measuring instrument that monitors the flow from the storage flap:  and a controlling unit, whydah system is operably connected to the first transport means for c(Â»trolley  and maintaining a constant dilution rate in the vessel with varying rates of individual supply of liquid from the stage flasks to town vessel, wherein<br>
(i)      vessel (1) is equipped with inlet tubes (2a) from storage flacks (2) for<br>
supply of liquids; (ii)     pumps (3) are for transporting liquids from time stone flat  (2) to<br>
vessel (1);<br><br>
(iii)    scales (4) are for monitoring the amount of liquids supplied to and<br>
discharged from the vessel; (iv)    pump (6) is for discharging fermentation broth via outfit tubes (5a) to a<br>
harvest flap  (5); (v)      Misruling unit (11) is for  mitering and steering individual control<br>
systems (17) for temperature, pH, gas pressure, vessel content and<br>
antifoam agents; (vi)    gas inlet tubes (9) and outlet tubes (10) are present; (vii)   main controlling unit (7) is for overall process monitoring and steering; (viii) circuit (12) having pump (13) is for gas supply and taking samples; (ix)    gas inlet and outlet flow control (14) and (15) are present;<br>
and, optionally, sterile filters (16) and thermo stating unit (8) are present<br>
2.       A fermentation assembly substantially as herein described with particular reference to figure I in the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">849-mas-2000-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">849-mas-2000-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWRlc2NyaXB0aW9uIGNvbXBsZXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">849-mas-2000-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">849-mas-2000-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gNC5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLWZvcm0gNi5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODQ5LW1hcy0yMDAwLW90aGVyIGRvY3VtZW50cy5wZGY=" target="_blank" style="word-wrap:break-word;">849-mas-2000-other documents.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="192689-a-process-for-the-extraction-of-dietary-fibre-oleresin-and-fixed-oils-from-the-seeds-of-fenugreek-trigonella-foenum-graecum-and-an-extraction-module-apparatus-for-carrying-out-the-same.3f3e.delaye">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="192691-a-semiconductor-device-and-a-method-of-producing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>192690</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>849/MAS/2000</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Aug-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Oct-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, 6411 TE HEERLEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ATTILA BARTOK</td>
											<td>5 RIETERPLATZ, CH-8002 ZUERICH,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>THORSTEN MUEH</td>
											<td>6 AM BLAUEN BERG, D-51375 LEVERKUSEN,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARKUS RUCCKEL</td>
											<td>25 BIRKENSTRASSE, D-82377 PENZBERG,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12M1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>00119676.5</td>
									<td>2000-09-08</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>99120289.6</td>
									<td>1999-10-11</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/192690-continuous-fermentaion-process by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:00:44 GMT -->
</html>
