<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/193087-a-process-for-producing-a-carotenoid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:38:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 193087:A PROCESS FOR PRODUCING A CAROTENOID</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PRODUCING A CAROTENOID</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a process for producing a carotenoid, which comprises the steps of cultivating an organism selected from organisms obtained by cultivating a parent organism such as herein described, which is capable of producing a carotenoid, under the conditions for reducing an alternative oxidase activity and selecting an organism with enhanced productivity of carotenoid and a mutant strain of which productivity of carotenoid is enhanced with the aid of alteration of the resistance against an alternative oxidase inhibitor, in a suitable medium, such as YPD medium, and at a suitable temperature such as 2oc, and recovering the resulting carotenoid in a known manner.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br><br>
Case 20685<br>
FIELD OF THE INVENTION<br>
The present invention relates to molecular biology for the manufacture of carotenoids and biological materials useful therefor.<br>
BACKGROUND OF THE INVENTION<br>
Astaxanthin is known to be distributed in a wide variety of organisms such as animal (e.g. birds such as flamingo and scarlet ibis, and fish such as rainbow trout and salmon), algae and microorganisms.   It is also recognized that astaxanthin has a strong antioxidation property against reactive oxygen species, which is expected to apply to pharmaceutical usage to protect living cells against some diseases such as a cancer.   Moreover, from a viewpoint of industrial application, a demand for astaxanthin as a coloring reagent is increasing especially in the industry of farmed fish, such as salmon, because astaxanthin imparts distinctive orange-red coloration to the animals and contributes to consumer appeal in the marketplace.<br>
Phaffia rhodozyma is known as a carotenogenic yeast strain which produces astaxanthin specifically. Different from the other carotenogenic yeast, Rhodotorula species, Phaffia rhodozyma (P. rhodozyma) can ferment some sugars such as D-glucose.   This is an important feature from a viewpoint of industrial application.   In a recent taxonomic study, a sexual cycle of P. rhodozyma was revealed and its telemorphic state was designated under the name of Xanthophyllomyces dendrorhous (W.I. Golubev; Yeast 11, 101 - 110, 1995).   Some strain improvement studies to obtain hyper producers of astaxanthin from P. rhodozyma have been conducted, but such efforts have been restricted to employ the method of conventional mutagenesis and protoplast fusion in this decade.   Recently, Wery et al.<br>
NS/15.05.2001<br><br>
developed a host vector system using P. rhodozyma in which a non-replicable plasmid was used to be integrated onto the genome of P. rhodozyma at the locus of ribosomal DNA in multicopies (Wery et al., Gene, 184, 89-97, 1997).   And Verdoes et al. reported more improved vectors to obtain a transformant of P. rhodozyma as well as its three carotenogenic genes which code the enzymes that catalyzes the reactions from geranylgeranyl pyrophosphate to beta-carotene (International patent publication W097/23633). The importance of genetic engineering method on the strain improvement study of P. rhodozyma will increase in near future to break through the reached productivity by the conventional methods.<br>
Many researchers have speculated that astaxanthin might play a role as an antioxidant in Phaffia rhodozyma because its production is stimulated in a respiration phase of growth rather than in the fermentation phase.   In general, reactive oxygen species tend to be generated in the respiration phase as a result of electron overflow in the respiratory chain which is caused by the imbalance of the electron transfer between reduction speed of ubiquinone pool and electron transfer in the downstream of the respiratory chain.   In such a speculation, astaxanthin might quench such reactive oxygen species as super oxide dismutase does in living organisms.<br>
Schroeder et al. reported that respiratory chain oi Phaffia rhodozyma was shifted from KCN-sensitive respiration to KCN-resistant one in the late phase of growth when astaxanthin production was stimulated (J. Biol. Chem., 270, 18374-18379, 1995).   KCN-sensitive respiratory chain is the common electron transfer'chain, in which the electron within the ubiquinone pool is transferred to complex IV via complex III which is distributed in a wide variety of organisms.   It is known that this respiratory chain is inhibited by KCN or antimycin A.   On the other hand, KCN-resistant respiratory chain is distributed in plant and fungi.   In this respiratory chain, mitochondrial membrane protein called as alternative oxidase (AOX) plays a substantial role to transfer the electron within the ubiquinone pool to H2O molecule by using an oxygen molecule as a<br><br>
receptor.   AOX activity is known to be inhibited by n-propyl gallate (n-PG) or salicylhydroxamic acid (SHAM).<br>
In their characterization study for antimycin-sensitive hyper producers of astaxanthin derived from Phaffia rhodozyma. An et al. made a speculation that such mutants produced much more astaxanthin to quench the reactive oxygen species, which might be produced by the electron overflowed from the electron transfer chain (Appl. Env. Microbiol, 55, 116-124,1989).<br>
This invention is created based on the assumption that biosynthesis of astaxanthin might be upregulated under the condition which the electron transfer chain would be in the reduced state.   Such a reduced state might be induced by an addition of a specific inhibitor such as antimycin A, KCN, n-PG or SHAM.   Such a state also might be induced by some mutation which \Vould result in an imbalance of electron transfer.<br>
In accordance with this invention, mutants on which the resistance against SHAM was conferred were obtained. Such mutants acquired 50 % higher productivity of astaxanthin than their parent strain.<br>
This invention involves the cloning of a gene, which codes for alternative oxidase from Phaffia rhodozyma.   This invention also involves the enzymatic characterization as a result of the expression of such a gene in suitable host organisms such as E. coli or Saccharomyces cerevisiae.  The gene thus cloned may be used for the reduction in AOX activity by using iite-directed mutagenesis of promoter sequence or anti-sense method in a suitable host, such as P. rhodozyma.  And their effects on the :arotenogenesis can be confirmed by the cultivation of such a transformant in an appropriate medium under an appropriate cultivation condition.<br><br>
SUMMARY OF THE INVENTION<br>
The present invention provides a n.ovel process for producing carotenoids, which comprises cultivating an organism which is obtainable by treating a parent organism producible of carotenoids under the condition to induce a reduction of an alternative oxidase activity and selecting an organism of which productivity of carotenoids is enhanced. The organism utilized in the process of the present invention may be a mutant strain of which productivity of carotenoids is enhanced with the aid of alteration of the resistance against an alternative oxidase inhibitor.  The said organism may be a mutant resistant to an alternative oxidase inhibitor. A process according to the present invention can be practiced by using an organism which belongs to the kingdom of Protista or Fungi, more preferably to the genus Synechococcus, Synechocystis, Haematococcus, Dunaliella, Phaffia, Xanthophyllomyces, Neurospora, Rhodotorula, Blakeslea, or Phycomyces, wherein the most preferable organism may be a strain of Phaffia rhodozyma and Xanthophyllomyces dendrorhous.<br>
The alternative oxidase inhibitor which may be used for the present invention may be selected from the group consisting of n-propyl gallate and salicylhydroxamic acid.<br>
The present invention also provides a method for establishing a mutant strain capable of producing carotenoids in an enhanced level relative to a parent organism, which comprises cultivating an organism producible of carotenoids under the condition for reducing an alternative oxidase activity and selecting an organism capable of producing carotenoids in the level higher than the said parent organism.   The said condition for reducing an alternative oxidase activity may comprise the presence of an alternative oxidase inhibitor.  The alternative oxidase inhibitor for this purpose may be selected from the group consisting of n-propyl gallate and salicylhydroxamic acid.   The organism for a mutant strain of the present invention may belong to the kingdom of Protista or Fungi, more preferably to the genus Synechococcus, Synechocystis, Haematococcus, Dunaliella, Phaffia, Xanthophyllomyces, Neurospora, Rhodotorula, Blakeslea, or Phycomyces, wherein the most preferable<br><br>
I<br>
organism may be a strain of Phaffia rhodozyma and Xanthophyllomyces dendrorhous.<br>
In another aspect, the present invention also relates to a mutant strain of an organism capable of producing carotenoids in enhanced level relative to a parent organism which is obtainable by the method described above. The said mutants may be more specifically characterized in that they can grow even in the medium containing 0.3 to 0.45 mg/ml of SHAM in a similar growth rate in the medium which does not contain SHAM.<br>
As an embodiment of the present invention, there are provided SHAM-resistant mutants, which were derived from Phaffia rhodozyma ATCC96594. Such SHAM - resistant mutants were deposited at the DSMZ (Deutsche Sammlung der Mikroorganismen and. Zellkulturen GmbH, Mascheroder Weg lb, D-38124 Braunschweig, Germany) under DSM 13429, DSM 13430 und DSM 13431, respectively, on April 3, 2000.<br>
The organism used in the process of the present invention may be a recombinant organism of which gene expression of the alternative oxidase is altered to reduce efficiency compared to a parent organism.   The present invention further provides a recombinant organism capable of producing carotenoids in an enhanced level relative to a host organism which is characterized in that whose gene expression of the alternative oxidase is altered to reduce efficiency compared to the host organism.   Such an organism may be that the gene expression of the alternative oxidase is altered with the aid of the technique selected from antisense technology, site-directed mutagenesis, chemical mutagenesis, etc.  The organism for this purpose may belong to the kingdom of Protista or Fungi, more preferably to the genus Synechococcus-, Synechocystis, Haematococcus-, Dunaliella, Phaffia, Xanthophyllomyces, Neurospora, Rhodotorula, Blakeslea, or Phycomyces, wherein the most preferable organism may be a strain of Phaffia rhodozyma and Xanthophyllomyces dendrorhous - especially the above mentioned deposits.<br>
The present invention also provides a recombinant DNA sequence coding for an alternative oxidase derived from an organism producible of carotenoids. The recombinant DNA can be obtained from an organism<br><br>
which belongs to the kingdom of Protista or Fungi, more preferably to the genus Synechococcus, Synechocystis, Haematococcus, Dunaliella, Phaffia, Xanthophyllomyces, Neurospora, Rhodotorula, Blakeslea, or Phycomyces, wherein the most preferable organism may be a strain of Phaffia rhodozyma and Xanthophyllomyces dendrorhous - especially the above mentioned deposits. The recombinant DNA sequence according to the present invention may be that identified by SEQ ID NO: 2 or that having identity with SEQ ID NO: 2 higher than 55%, more preferably higher than 75%, most preferably higher than 95%.<br>
The said recombinant DNA sequence may be more specifically characterized in that (a) it codes for the said enzyme having an amino acid sequence described in SEQ ID NO: 1, or (b) it codes for a variant of the said enzyme selected from (i) an allelic variant, and (ii) an enzyme having one or more amino acid addition, insertion, deletion and/or substitution and having the stated enzyme activity.   Particularly specified isolated DNA sequence mentioned above may be that which can be derived from a gene of Phaffia rhodozyma and is selected from (i) a DNA sequence represented in SEQ ID NO: 2, (ii) an isocoding or an allelic variant for the DNA sequence represented in SEQ ID NO: 2, and (iii) a derivative of a DNA sequence represented in SEQ ID NO: 2, with addition, insertion, deletion and/or substitution of one or more nucleotide(s), and coding for a polypeptide having the said enzyme activity.<br>
The present invention also provides the use of the said recombinant DNA to transform a host organism.   A convenient form of the recombinant DNA may be a vector. The recombinant organism obtained by use of the recombinant DNA is capable of decreasing an enzyme activity for alternative oxidase. The host organism transformed with the recombinant DNA may be useful in the improvement of the production process of carotenoids, in particular astaxanthin. Thus the present invention also provides such a recombinant organism.<br>
Moreover,    the   present   invention   provides    a    method   for   the biological   production   of   carotenoids   which    comprises    introducing   a<br><br>
recombinant DNA described above into an appropriate host organism and cultivating thus obtained recombinant organism. Thus the method for producing a carotenoid, characterized in that a recombinant organism described above is cultivated under conditions conductive to the production of the carotenoid is one of the aspects of the present invention. This method may be applied to the biological production of astaxanthin.<br>
Accordingly the present invention provides a process for producing a carotenoid, which comprises the steps of cultivating an organism selected from organisms obtained by cultivating a parent organism such as herein described, which is capable of producing a carotenoid, under the conditions for reducing an alternative oxidase activity and selecting an organism with enhanced productivity' of carotenoid and a mutant strain of which productivity of carotenoid is enhanced with the aid of alteration of the resistance against an alternative oxidase inhibitor, in a suitable medium, such as YPD medium, and at a suitable temperature such as 20°C, and recovering the resulting carotenoid in a known manner.<br>
The invention will now be described in more detail with reference to the accompanying drawings, in which;<br>
Figure 1 depicts a working model for respiratory chain in P.rhodozyma.<br><br>
As described above, many researchers have speculated that astaxanthin might play a role as an antioxidant in Phaffia rhodozyma because its production is stimulated in a respiration phase of growth rather than in the fermentation phase.   In general, reactive oxygen species tends to be produced in the respiration phase as a result of electron overflow in the respiratory chain which is caused by the imbalance of the electron transfer between reduction speed of ubiquinone pool and electron transfer in the downstream of the respiratory chain (FIG. 1).   In such a speculation, astaxanthin might quench such reactive oxygen species as super oxide dismutase does.<br>
Based on this speculation, overproduction of astaxanthin could be realized by the inhibition of respiratory chain in Phaffia rhodozyma.   In fact, An ct al. isolated mutants whose KCN-sensitive respiration were blocked and which produced much more astaxanthin from Phnffin rhodozyma (Appl. Env. Microbiol, 55, 116-124, 1989).<br><br>
On the other hand, as Schroeder et al. reported, respiratory chain of Phaffia rhodozyma was shifted from KCN-sensitive respiration to KCN-resistant one in the late phase of growth when astaxanthin production was stimulated (J. Biol. Chem., 270, 18374-18379, 1995).   In this context, KCN-resistant respiration, which is mediated by alternative oxidase would give more effect on the respiration in the production phase of astaxanthin.  The inhibition of alternative oxidase might lead to the overproduction of astaxanthin.<br>
In order to examine the effect of specific inhibition of respiration activity on production of astaxanthin in Phaffia rhodozyma, SHAM, which is known to inhibit the alternative oxidase was added to growing agar medium at serial diluted concentration.   In the course of this study, several spontaneous mutants, which showed similar growth activity even in the presence of 0.3 to 0.45 mg/ml SHAM to that in the medium which did not include SHAM appeared.   Surprisingly it was found that such mutants produced 50 % higher astaxanthin than their parent.   This indicated that some mutation which led to overproduction of astaxanthin could complement growth inhibition by the reduced state of respiratory chain which would be caused by a decrease of alternative oxidase activity.<br>
The present invention provides thus isolated mutants which are resistant against 0.3 to 0.45 mg/ml of the specific inhibitor for alternative oxidase, SHAM.   Likewise the above, as it is readily understood by those ordinary skilled in the art, mutant strains which are capable of producing astaxanthin in higher productivity can be established by cultivation of an appropriate organisms in the* presence of any one or more of the inhibitors to the alternative oxidase, and by screening the organisms showing growth activity in the presence of the said inhibitor(s) and higher productivity of astaxanthin than parent organisms.   Productivity of astaxanthin can be determined by extracting carotenoids from the cells of P. rhodozyma and measuring astaxanthin level as exemplified in Example 2.  An increase of the productivity in about 10 % will be a possible criteria to select a mutant strain capable of producing astaxanthin in higher level than a parent strain.<br><br>
Re-cultivation and screening of thus obtained mutant strain under the pressure of the alternative oxidase inhibitor can be employed to improve the productivity.   Thus obtained mutant strain can be applied to astaxanthin production in an appropriate medium.<br>
In order to decrease the activity for alternative oxidase, an approach employed by genetic engineering has several advantages against an approach by the addition of specific inhibitor such as SHAM to the culture medium.   One of those advantages is an economic reason.   Addition of such an inhibitor would increase the production cost.   And addition of such an inhibitor to the culture medium has another disadvantage in the purification step to remove the added inhibitors from the final product.<br>
Further, the present invention provides an isolated recombinant DNA sequence which codes for alternative oxidase from Phaffia rhodozyma.<br>
The said DNA of the present invention can mean a cDNA which contains only open reading frame flanked between the short fragments in its 5'- and 3'- untranslated region, and also a genomic DNA which contains its introns and regulatory sequences such as its promoter and terminator which are involved in the expression of the gene of interest.<br>
At first, we cloned a partial gene fragment containing a portion of AOX gene by using degenerate PCR method.  The said degenerate PCR is a method to clone a gene of interest which has high homology of amino acid sequence to the known enzyme from other species which has a same or similar function.   Degenerate primer, which is used as a primer in degenerate PCR, was designed by a reverse translation of the amino acid sequence to corresponding nucleotides ("degenerated").   In such a degenerate primer, a mixed primer which consists any of A, C, G or T, or a primer containing inosine at an ambiguity code is generally used.   In this invention, the mixed primers were used for degenerate primers to clone<br><br>
above gene.   PCR condition used is varied depending on primers and gene to clone as described hereinafter.<br>
An entire gene containing its coding region with its intron as well as its regulatory region such as a promoter or terminator can be cloned from a chromosome by screening of genomic library which is constructed in phage vector or plasmid vector in an appropriate host, by using a partial DNA fragment obtained by degenerate PCR as described above as a probe after it was labeled.   Generally, E. coli as a host strain and E. colt vector, a phage vector such as lambda phage vector, or a plasmid vector such as pUC vector is often used in the construction of library and a following genetic manipulation such as a sequencing, a restriction digestion, a ligation and the like.   In this invention, an EcoRl genomic library of P. rhodozyma was constructed in the derivatives of lambda vector, lambda gtll.   An insert size, what length of insert must be cloned, was determined by the Southern blot hybridization before a construction of a library.   In this invention, a DNA which was used for a probe was labeled with digoxigenin (DIG), a steroid hapten instead of conventional 32P label, following the protocol which was prepared by the supplier (Boehringer-Mannheim, Mannheim, Germany).   A genomic library constructed from the chromosome of P. rhodozyma was screened by using a DIG-labeled DNA fragment which had a portion of a gene of interest as a probe.   Hybridized plaques were picked up and used for further study.   After the isolation"of positive plaque, insert fragment was subcloned into appropriate plasmid vector which can be conveniently used for sequencing.   In this invention, the insert fragment in the positive phage vector was subcloned into pOCUS-2 vector, which was used for construction of transposon-inserted sequencing derivatives (Locus Pocus System, Novagene, Madison, U.S.A.).<br>
In this invention, the automated fluorescent DNA sequencer, ALFred system (Pharmacia, Uppsala, Sweden) was used with an autocycle sequencing protocol in which the Taq DNA polymerase is employed in most cases of sequencing.<br><br>
After the determination of the genomic sequence, a sequence of a coding region was used for a cloning of cDNA of corresponding gene.   The PCR method was also exploited to clone cDNA fragment.  The PCR primers whose sequences were identical to the sequence at the 5'- and 3'- end of the open reading frame (ORF) were synthesized with an addition of an appropriate restriction site, and PCR was performed by using those PCR primers.   In this invention, a cDNA pool was used as a template in this PCR cloning of cDNA.   The said cDNA pool consists of various cDNA species which were synthesized in vitro by the viral reverse transcriptase and Taq polymerase (CapFinder Kit manufactured by Clontech, Palo Alto, U.S.A.) by using the mRNA obtained from P. rhodozyma as a template.   cDNA of interest thus obtained can be confirmed in its sequence.   Furthermore, cDNA thus obtained can be used for a confirmation of its enzyme activity after the cloning of the cDNA fragment into an expression vector which functions in E. coli or S. cerevisiae under the appropriate promoter.<br>
To express a gene which was derived from Eukaryote, a procedure in which cDNA is cloned into an expression vector in E. coli or S. cerevisiae is often used.   This is caused from a fact that a specificity of intron structure varies among the organisms and an inability to recognize the intron sequence from other species.   In fact, Prokaryote has no intron structure in its own genetic background.   Even in the yeast, genetic background is different between Ascomycetes to which Saccharomyces cerevisiae belongs and Basidiomycetes to which P. rhodozyma belongs.   Wery et al. showed that the intron structure of actin gene from P. rhodozyma cannot be recognized nor spliced by the ascomycetous yeast, Saccharomyces cerevisiae (Yeast, 12, 641-651, 1996).<br>
Some other researchers reported that intron structures of some kinds of the genes involve regulation of their gene expressions (Dabeva, M. D. et al, Proc. Natl. Acad. Sci. U.S.A., 83, 5854, 1986).   It might be important to use a genomic fragment which has its introns in a case of self-cloning of . the gene of an interest whose intron structure involves such a regulation of its own gene expression.<br><br>
To apply a genetic engineering method for a strain improvement study, it is necessary to study its genetic mechanism in the event such as transcription and translation.   It is important to determine a genetic sequence for its upstream activation sequence (UAS), promoter, intron structure and terminator, not only for its exon, to study the genetic mechanism.<br>
According to this invention, the gene which codes for the alternative oxidase was cloned from genomic DNA of P. rhodozyma, and their genomic sequence containing alternative oxidase (AOX) gene including its 5'- and 3'-adjacent regions as well as its intron structures were determined.<br>
Succeeding to the confirmation of the enzyme activity, gene modification study to decrease the alternative oxidase activity can be employed.<br>
In the present invention, the polypeptide sequence includes SEQ ID NO: 2 and fragments thereof having AOX activity and polypeptide sequences which hybridize to SEQ ID NO: 2 under stringency conditions which are sufficient to identify specific binding to SEQ ID NO: 2, and which hybrids encode a polypeptide that has the function of an alternative oxidase. For example, any combination of the following hybridization and wash conditions may be used to achieve the required specific binding:<br>
High Stringency Hybridization: 6X SSC 0.5% SDS<br>
100 micrograms/ml denatured salmon sperm DNA 50% formamide<br><br>
Incubate overnight with gentle rocking at 42°C overnight.<br>
High Stringency Wash:<br>
1 wash in 2X SSC, 0.5% SDS at Room Temperature for 15 minutes, followed by another wash in O.IX SSC, 0.5% SDS at Room Temperature for 15 minutes.<br>
Low Stringency Hybridization:<br>
6X SSC<br>
0.5% SDS<br>
100 micrograms/ml denatured salmon sperm DNA<br>
50% formamide<br>
Incubate overnight with gentle rocking at 37°C overnight.<br>
Low Stringency Wash:<br>
1 wash in O.IX SSC, 0.5% SDS at Room Temperature for 15 minutes.<br>
Moderately stringent conditions may be obtained by varying the temperature at which the hybridization reaction occurs and/or the wash conditions as set forth above.   In the present invention, it is preferred to use high stringency hybridization and wash conditions.<br>
To decrease a gene expression with genetic methods, some methods can be exploited.   One of such methods is anti-sense method.   Anti-sense method is a method to decrease an expression of gene of interest by introducing an artificial gene fragment, whose sequence is complementary<br><br>
to that of gene of interest.   And such an anti-sense gene fragment would form complex with mature mRNA fragment of objective gene in vivo and inhibit an efficient translation from mRNA, as a consequence.   For construction of anti-sense RNA for AOX gene, a PCR method can be employed to clone the complementary cDNA strand for AOX gene.<br>
The other method is a mutation of promoter region.   In general, the gene consists of several parts which have different functions from each other.  In Eukaryotes, genes which encode corresponding proteins are transcribed to premature messenger RNA (pre-mRNA) differing from the genes for ribosomal RNA (rRNA), small nuclear RNA (snRNA) and transfer RNA (tRNA).   Although RNA polymerase II (PolII) plays a central role in this transcription event, PolII can not solely start transcription without cis element covering an upstream region containing a promoter and an UAS, and a trans-acting protein factor.   At first, a transcription initiation complex which consists of several basic protein components recognizes the promoter sequence in'the 5'-adjacent region of the gene to be expressed. In this event, some additional participants are required in the case of the gene which is expressed under some specific regulation, such as a heat shock response, or adaptation to a nutrition starvation, and so on.   In such a case, a UAS is required to exist in the 5'-untranslated upstream region around the promoter sequence, and some positive or negative regulator proteins recognize and bind to the UAS.  The strength of the binding of transcription initiation complex to the promoter sequence is affected by such a binding of the trans-acting factor around the promoter, and this enables the regulation of the transcription activity.<br>
ff<br>
After the activation of a transcription initiation complex by the phosphorylation, a transcription initiation complex initiates transcription from the transcription start site.   Some parts of the transcription initiation complex are detached as an elongation complex from the promoter region to the 3'-direction of the gene (this step is called as a promoter clearance event) and an elongation complex continues the transcription until it<br><br>
reaches to a termination sequence that is located in the 3'-adjacent downstream region of the gene.<br>
To decrease an expression of gene of interest, mutation by conventional chemical mutagenesis or genetical site-directed mutagenesis in the promoter region of objective gene containing UAS sequence described above was often used.   In this approach, a gene cassette, containing a reporter gene which is fused to a promoter region derived from a gene of interest at its 5'-end and a terminator region from a gene of interest at its 3'-end, is mutagenized and then introduced into P. rhodozyma.   By detecting the difference of the activity of the reporter gene, an effective mutation would be screened.   Mutant strain in which expression of enzyme of interest might decrease can be obtained by transforming host strain with a recombinant DNA having such a mutated promoter region.<br>
Such constructs as a vector containing anti-sense AOX gene or mutated promoter for AOX gene can be transferred into appropriate host strain.   In the case of using Phaffia rhodozyma as a host strain, this is realized by the cloning of such constructs into the appropriate vector on which a selectable marker that functions in P. rhodozyma is harbored.   A drug resistance gene which encodes the enzyme that enables the host to survive in the presence of a toxic antibiotic is often used for the selectable marker.   G418 resistance gene harbored in pGB-Ph9 (Wery et al., Gene, 184, 89-97, 1997) is an example of a drug resistance gene.   Such a plasmid can be integrated on the chromosome of Phaffia rhodozyma through the homologous recombination between the chromosome and the plasmid.<br>
As a transformation method, LiAc method and electroporation method (Wery et al., Gene, 184, 89-97, 1997) were applied to transform P. rhodozyma.<br><br>
Such a genetically engineered P. rhodozyma would be cultivated in an appropriate medium and evaluated in its productivity of astaxanthin.<br>
The present invention is further illustrated with Examples described below by referring to the attached drawing.<br>
Examples<br>
The following materials and methods were employed in the Examples described below:<br><br><br>
Vectors<br>
lambda gtll (Clontech)<br>
pCR2.1-T0P0 (Invitrogen)<br>
pOCUS-2<br>
pBluescript II SK-(Stratagene)<br>
pGBPh9 (Wery et al., Yeast, 12, 641-651, 1996)<br>
Media<br>
P. rhodozyma strain is maintained routinely in YPD medium (DIFCO, Detroit, U.S.A.).   E. coli strain is maintained in LB medium (10 g Bacto-trypton (DIFCO), 5 g yeast extract (DIFCO) and 5 g NaCl per liter).   NZY medium (5 g NaCl, 2 g MgS04-7H20, 5 g yeast extract (DIFCO), 10 g NZ amine type A (WAKO, Osaka, Japan) per liter) is used for lambda phage propagation in a soft agar (0.7 % agar; WAKO).   When an agar medium was prepared, 1.5 % of agar (WAKO) was supplemented.   Salicylhydroxamic acid (SHAM) was purchased from Aldrich (Milwaukee, U.S.A.).<br>
Methods<br>
A general niethod for techniques of molecular genetics was according to the method in Molecular cloning: a Laboratory Manual, 2nd Edition (Cold Spring Harbor Laboratory Press, 1989).   Restriction enzymes and T4 DNA ligase were purchased from Takara Shuzo (Ohtsu, Japan).<br>
f<br>
Isolation of a chromosomal DNA from P. rhodozyma was performed by using QIAGEN Genomic Kit (QIAGEN, Hilden, Germany) following the protocol supplied by the manufacturer.   Mini-prep of plasmid DNA from transformed E. coli was performed with the Automatic DNA isolation system (PI-50, Kurabo, Co. Ltd., Osaka, Japan).   Midi-prep of plasmid DNA from an E. coli transformant was performed by using QIAGEN column<br><br>
(QIAGEN).   A DNA fragment was isolated and purified from agarose by-using QIAquick or QIAEX II (QIAGEN).<br>
Isolation of total RNA from P. rhodozyma was performed with phenol method by using Isogen (Nippon Gene, Toyama, Japan).   mRNA was purified from total RNA thus obtained by using mRNA separation kit (Clontech).   cDNA was synthesized by using CapFinder cDNA construction kit (Clontech).<br>
In vitro packaging was performed by using Gigapack III gold packaging extract (Stratagene, La Jolla, U.S.A.).   Isolation of lambda DNA was performed by Wizard lambda preps DNA purification system (Promega, Madison, U.S.A.) following the protocol prepared by the manufacturer.<br>
Polymerase chain reaction (PCR) was performed with the thermal cycler from Perkin Elmer model 2400.   Each PCR condition is described in the Examples.   PCR primers were purchased from a commercial supplier. DNA sequencing was performed with the automated fluorescent DNA sequencer (ALFred, Pharmacia).<br>
Competent cells of DH5alpha were purchased from Toyobo.   All the chemicals were purchased from WAKO otherwise stated.<br>
Example  1     Isolation of SHAM-resistant mutants,  SHAMl,  SHAM2  and SHAM3 from P. rhodozyma ATCC96594<br>
In order to examine the effect of the inhibition for KCN-resistant respiration mediated by alternative oxidase on the growth of P. rhodozyma.<br><br>
addition of SHAM to culture of P. rhodozyma on YPD-agar medium was conducted.  SHAM was resolved in ethanol and added to the YPD-agar medium by 0.05, 0.15, 0.30, 0.45 and 0.90 mg/ml at final concentration, respectively.   On those medium, 2 x 10' cells / ml of P. rhodozyma ATCC96594 were spread after dilution.  After 3 day-cultivation at 20 °C, yielded colonies were counted.  As a result, the number of colonies which grew on SHAM-containing YPD agar was almost the same as that on control medium which did not contain SHAM.   But, size of colonies which grew on SHAM-containing medium was smaller than control culture. TABLE 1 shows the relative colony diameter against the control colony.<br>
TABLE 1   Relative size of colony which grew on SHAM-containing YPD-agar<br>
SHAM (mg/ml)	0.05       0.15       0.30       0.45       0.90<br>
Relative colony diameter (%)	100	70	40	30	12<br>
Interestingly, among the colonies, which grew on the medium containing 0.3 and 0.45 mg/ml of SHAM, some colonies showed the similar size to the control colony.   Such colonies also showed deeper pigmentation than the control colony.   Four colonies, which showed similar colony size to the control colony were picked up and streaked onto YPD-agar medium. All the colonies showed deeper pigmentation than the control colony even on YPD-agar medium, which did not contain SHAM.   From this result, it was suggested that these strains might be spontaneous mutants and were named as SHAMl, SHAM2, SHAM3 and SHAM4.<br><br>
Example 2   Flask fermentation of resistant mutants, SHAMl, SHAM2 and SHAM3<br>
To evaluate productivity of astaxanthin by SHAM-resistant mutants, SHAMl, SHAM2 and SHAM3, fermentation in shaking flasks was conducted.   These mutants and their parent strain, ATCC96594 were inoculated from freshly prepared agar-culture to 50 ml of YPD medium in baffle flask of 500 ml in size at final OD at 660 nm of 0.05.   Fermentation was conducted at 20 °C at 200 r.p.m.   At an appropriate interval, 3 ml of broth was withdrawn and was analyzed for cell yield and astaxanthin content.<br>
Cell yield was measured as OD at 660 nm.   As dry cell weight, it was measured by weighing cells derived from 1.0 ml of broth after heating at 120 °C overnight in 1.5 ml of microcentrifugation tube.   Astaxanthin content of P. rhodozyma was measured with HPLC method after extraction of carotenoids from cells of P. rhodozyma by disruption with glass beads as follows.   Cells obtained from 1 ml of broth after the centrifugation was concentrated by two-fold with distilled water and 10.0 gram of glass beads was added to the cell suspension (0.5 ml) in brown-shaded test tube (13.5 mm, 11 cm).   Next, 1.5 ml of acetone / butylated hydroxy toluene (BHT) / water (45.mg of BHT in 450 ml acetone and 50 ml water) was added and then test tube was shaken with horizontal table top shaker for an hour. After extraction, 5 ml of acetone/BHT/water containing appropriate concentration of bixin (nacalai tesque, Kyoto, Japan) as an internal standard was added.   Supernatant was analyzed for astaxanthin content with following HPLC system.   (Hardware for HPLC system was purchased from Tosoh (Tokyo, Japan)).<br>
HPLC column; YMC-Pak ODS-A (6 mm, 150 mm (YMC, Inc., Milford, U.S.A.).<br>
Temperature; room temperature<br><br>
Eluent; acetonitrile / methanol / isopropanol (85 / 10 / 5) Injection volume; 10 microliter Flow Rate; 2.0 ml/minute Detection; UV at 471 nm<br>
Results are summarized in TABLE 2.  All the mutants showed 50 % higher productivity of astaxanthin than parent strain, ATCC96594.   From this result, some mutation might occur in these mutants to compensate the inhibition of alternative oxidase activity by increase of astaxanthin production.<br>
TABLE 2   Productivitv of astaxanthin by SHAM-resistant mutants<br><br><br>
Example 3     Isolation of mRNA from P.   rhodozyma  and construction of cDNA library<br>
To construct cDNA library of P. rhodozyma, total RNA was isolated by phenol extraction method right after the cell disruption and the mRNA from P. rhodozyma ATCC96594 strain was purified by using mRNA separation kit (Clontech).<br>
At first, cells of ATCC96594 strain from 10 ml of two-day-culture in YPD medium were harvested by centrifugation (1500 x g for 10 min.) and washed once with extraction buffer (10 mM Na-citrate / HCl (pH 6.2) containing 0.7 M KCl).  After suspending in 2.5 ml of extraction buffer, the cells were disrupted by French press homogenizer (Ohtake Works Corp., Tokyo, Japan) at 1500 kgf/cm2 and immediately mixed with two times of volume of isogen (Nippon gene) according to the method specified by the manufacturer.   In this step, 400 microgram of total RNA was recovered.<br>
Then this total RNA was purified by using mRNA separation kit (Clontech) according to the method specified by the manufacturer. Finally, 16 gram of mRNA from P. rhodozyma ATCC96594 strain was obtained.<br>
To construct cDNA library, CapFinder PCR cDNA construction kit (Clontech) was used according to the method specified by the manufacturer.   One microgram of purified mRNA was applied for a first strand synthesis followed by PCR amplification.  After this amplification by PCR, 1 mg of cDNA pool was obtained.<br><br>
Example 4   Cloning of the partial AOX (alternative oxidase) gene from P. rhodozyma<br>
To clone a partial AOX gene from P. rhodozyma, a degenerate PCR method was exploited.   Species and accession number to database whose sequence for alternative oxidase were used for multiple alignment analysis (ClustalW, Thompson J. D., et al., Nucleic Acids Research, 22, 4673-4680, 1994) are as follows.<br><br>
Two mixed primers whose nucleotide sequences were designed and lynthesized as shown in TABLE 3 based on the common sequence of known ilternative oxidase genes from other species.<br>
TABLE 3 Sequence of primers used in the cloning of AOX gene<br><br>
aox3 ; AAYGARMGNATGCAYYTNYTNACNTT (sense primer) (SEQ ID NO: 3)<br>
aox5 ; GCYTCYTCYTCNARRTANCCNACRAA (antisense primer) (SEQ ID NO: 4)<br>
(N=A, C, G or T; R=A or G, Y=C or T, M=A or C)<br>
After the PCR reaction of 25 cycles of 95 °C for 30 seconds, 50 °C for 3Q seconds and 72°C for 15 seconds by using ExTaq (Takara Shuzo) as a DNA polymerase and cDNA pool obtained in Example 1 as a template, reaction mixture was applied to agarose gel electrophoresis.   A PCR band that had a desired length was recovered and purified by QIAquick (QIAGEN) according to the method by the manufacturer and then ligated to pCR2.1-TOPO (Invitrogen).   After the transformation of competent E. coli TOPIO, 6 white colonies were selected and plasmids were isolated with Automatic DNA isolation system.   As a result of sequencing, it was found that 3 clones had a sequence whose deduced amino acid sequence was similar to known alternative oxidase genes.   This isolated cDNA clone was designated as pAOX514 and used for further study.<br>
Example 5   Isolation of genomic DNA from P. rhodozyma<br>
To isolate a genomic DNA from P. rhodozyma, QIAGEN genomic kit was used according to the method specified by the manufacturer.<br>
At first, cells of P. rhodozyma ATCC96594 strain from 100 ml of overnight culture in YPD medium were harvested by centrifugation (1500 x g for 10 min.) and washed once with TE buffer (10 mM Tris / HCl (pH 8.0) containing 1 mM EDTA).   After suspending in 8 ml of Yl buffer of the QIAGEN genomic kit, lyticase (SIGMA, St. Louis, U.S.A.) was added at the<br><br>
concentration of 2 mg/ml to disrupt cells by enzymatic degradation and the reaction mixture was incubated for 90 minutes at 30 °C and then proceeded to the next extraction step.   Finally, 20 microgram of genomic DNA was obtained.<br>
Example 6   Southern blot hybridization by using pAOX514 as a probe<br>
Southern blot hybridization was performed to clone a genomic fragment which contains AOX gene from P. rhodozyma.   Two microgram of genomic DNA was digested by EcoRl and subjected to agarose gel electrophoresis followed by acidic and alkaline treatment.   The denatured DNA was transferred to nylon membrane (Hybond N+, Amersham, Buckinghamshire, U.K.) by using transblot (Joto Rika, Tokyo, Japan) for an hour.   The DNA which was transferred to nylon membrane was fixed by a heat treatment (80 °C,'90 minutes).   A probe was prepared by labeling a template DNA (EcoRI-digested pAOX514) with DIG multipriming method (Boehringer Mannheim).   Hybridization was performed with the method specified by the manufacturer.   As a result, hybridized band was visualized in the range from 5.5 to 7.0 kilobases (kb).<br>
Example 7   Cloning of a genomic fragment containing AOX gene<br>
Four microgram of the genomic DNA was digested by EcoRl and subjected to agarose gel electrophoresis.  Then, DNAs whose length is within the range from 5.0 to 7.0 kb was recovered by QIAEX II gel extraction kit (QIAGEN) according to the method specified by the manufacturer.   The purified DNA was ligated to 0.5 microgram of EcoRl-digested and CIAP (calf intestine alkaline phosphatase)-treated lambda gtll (Clontech) at 16 °C overnight, and packaged by Gigapack III gold<br><br>
packaging extract (Stratagene).   The packaged extract was infected to E. coli Y1090 strain and over-laid with NZY medium poured onto LB agar medium.   About 6000 plaques were screened by using EcoRI-digested pAOX514 as a probe.   One plaque was hybridized to the labeled probe.<br>
This lambda gtll derivative containing putative AOX gene from P. rhodozyma was prepared by using Wizard lambda preps DNA purification system (Promega).   As a result of digestion by EcoRl, it was revealed that this lambda gtll derivative contained 6 kb EcoRl insert.   Next, PCR was conducted by using this lambda gtll derivative as a template and two primers, aox3 and aox5 as primers.  As a result of PCR under the same PCR condition as described in Example 4, expected 0.3 kb band was yielded.   It was suggested that this lambda gtll derivative might contain putative AOX gene from P. rhodozyma.   6.0 kb insert EcoRl fragment in this lambda gtll derivative was purified by using QIAquick (QIAGEN) and subjected to subcloning into pOCUS-2 vector (Novagen) by using DH5alpha as a host strain and yielded pOCUSAOX607.<br>
Example 8   Sequencing of a genomic fragment containing AOX gene<br>
pOCUSAOX607 was transferred into competent gamma delta donor cells and used for preparation of sequencing derivatives which were used for Locus Pocus system (Novagen).   Method for preparation of sequencing derivatives was followed by the protocol supplied by the manufacturer.   As<br>
t<br>
sequencing primers, Cy5-labelled primers whose sequence are listed in TABLE 4 were synthesized and used for sequencing by using AutoCycIe sequencing kit (Pharmacia).<br>
TABLE 4 Sequence of primers-used for sequencing AOX gene<br><br>
poc1 ; (Cy5-) AGCTACAACATACGAAAGGG (SEQ ID NO: 5) poc2 ; (Cy5-) GGGGAACTGAGAGCTCTAAA (SEQ ID NO: 6)<br>
As a result of sequencing, nucleotide sequence comprising 2561 base pairs of genomic fragment containing AOX gene from P. rhodozyma was determined.<br>
Coding region was in 1206 base pairs that consisted of 10 exons and 9 introns.   Introns were dispersed all through the coding region without 5' or 3' bias.   By using genetic analysis software, GENETYX-SV/RC (Software Development Co., Ltd., Tokyo, Japan) version 4.0.1, it   was found that open reading frame consists of 402 amino acids (SEQ ID NO: 1) whose sequence is strikingly similar to the known amino acid sequence of alternative oxidase from other species (51.5 % identity to alternative oxidase from Aspergillus niger).   A stretch of hydrophobic amino acid residues at amino terminal end which was expected to form alpha-helix structure indicated that this amino terminal region might be a membrane spanning domain or transit peptide for mitochondria.   PSORTII program (http://psort.nibb.ac.ip:8800/) predicted that this protein might be mitochondrial protein by 82.6 % prediction value.<br>
Example 9   Cloning of upstream region of AOX gene<br>
Cloning of 5'- adjacent region of AOX gene was performed by using Genome Walker Kit (Clontech), because it seemed that pAOX514 might not have sufficient length to contain promoter for AOX gene.   At first, the PCR primers whose sequences were shown in TABLE 5 were synthesized.<br><br>
TABLE 5<br>
Sequence of primers used in the cloning of 5'- adjacent region of AOX<br>
gene   -<br>
aoxl3 ; GTGTCAGAAACCTCAGATCAACAGGC (primary primer) (SEQ ID NO: 7)<br>
aoxl4 ; CAACAGGCAGTACAGTCAGCAGATTC (nested primer) (SEQ ID NO: 8)<br>
Protocols for library construction and PCR condition were the same as-those specified by the manufacturer. The genomic DNA preparation obtained in Example 5 was used as a PCR template.  The PCR fragments that had Scal site at the 5'- end (1.2 kb), and that had Dral site at the 5'-end (3.0 kb) were recovered and cloned into pCR2.1-TOPO by using E. coli TOPIO as a host strain.   As a result of sequencing of each 2 independent clones from both constructs, it was confirmed that the 5'-adjacent region of AOX gene was cloned.   The clone obtained by the Scal construct in the above experiment was designated as pAOXSc702 and used for further study. Based on the sequence of insert fragment in pAOXSc702, 4 PCR primers whose sequence are listed in TABLE 6 were synthesized.<br>
TABLE 6 Sequence of primers used for the cloning of AOX promoter region<br>
aoxl5 ; GAATTCAACAGGTCAAATGA (sense primer) (SEQ ID NO: 9) aoxl6 ; ATCCACCCACGCCTGTTTCC (antisense primer) (SEQ ID NO: 10) aoxl7 ; GGAAACAGGCGTGGGTGGAT (sense primer) (SEQ ID NO: 11) aoxl8 ; GAATTCAGTAAACGCATTAG (antisense primer) (SEQ ID NO: 12)<br><br>
I<br>
The PCR condition was the same as shown in Example 4 except that HF polymerase (Clontech) was used as DNA polymerase.   In the combination of aoxl5 and aoxl6, the fragment which has 0.7 kb in its length was amplified.   In the combinationof aoxl7 and aoxl8, the fragment which has 0.5 kb in its length was amplified.   These fragments were cloned into pCR2.1-T0P0 and transformed E. coli TOP1O.   Plasmids were prepared from 6 independent white colonies, respectively and subjected to the sequencing.   As a result, expected clones whose sequence of the insert fragment were identical to each other were obtained.   In the combination of aoxl5 and aoxl6, obtained clone was named as pAOX714 #1516.   In the combination of aoxl7 and aoxl8, obtained clone was named as pAOX714 #1718.   As a result of sequencing, sequence of pAOX714 #1516 and pAOX714 #1718 which contained promoter region for AOX gene from P. rhodozyma were determined.   Determined sequence containing AOX promoter was 1406 base pairs in its length.<br>
Combining the sequence which was obtained in Example 8 and 9, it was determined that the nucleotide sequence (3.7 kb) contained AOX gene and its promoter and terminator (SEQ ID NO: 2).<br>
Example 10   Construction of antisense plasmid for AOX gene<br>
An antisense gene fragment which covered the entire structure gene for AOX gene was amplified by PCR method and then cloned into integration vector in which antisense AOX gene was transcribed by AST promoter in P. rhodozyma.<br>
TABLE 7 Sequence of primers used in the antisense construction for AOX gene<br><br>
aoxlOl ; GGCCATTATGGCCTCAATTGGTCTGAGACATGC (SEQ ID NO: 13)<br>
aoxl02 ; GGCCGAGGCGGCCATGTCTCTTGCTAGATGTCT (SEQ ID NO:<br>
14)<br>
Both primers, aoxlOl and 102 had asymmetrical recognition sequence for restriction enzyme, Sfil (GGCCNNNNNGGCC) but their asymmetrical hang-over sequence was designed to be different.   This might enable a directional cloning into expression vector which has the same asymmetrical sequence at their ligation sequence.<br>
PCR was performed by using HP polymerase (Clontech) as Taq polymerase and the cDNAs prepared in Example 3 as a template under the<br>
condition as follows; 30 cycles of 94°C for 15 seconds, 55°C for 30 seconds and 72°C for 45 seconds.   The amplified PCR fragment was purified and cloned into pCR2.1-T0P0 vector.   As a result of sequencing, it was revealed that one clone had the correct fragment and this clone was named as pAOXlOO7 #0102. The sequence of antisense fragment for AOX gene is listed in SEQ ID NO: 15.<br>
For the promoter and terminator fragment which drove the transcription of the antisense AOX gene, AST promoter and terminator were cloned from the chromosome prepared in Example 5.<br>
TABLE 8 Sequence of primers used for the cloning of AST promoter and terminator<br><br>
ast49 ; GCGGCCGCACGTACAGACTAAGATCGAC (sense primer) (SEQ ID<br>
NO: 16)<br>
ast5O ; GGCCATAATGGCCATGGAGAAAGTAGGTGGCAA (antisense primer) (SEQ ID NO: 17)<br>
ast36 ; CCTGCAGGCCGCCTCGGCCGTTGATTCTTCATATGTTAA (sense primer) (SEQ ID NO: 18)<br>
ast37 ; GGTACCCTGCAGTCGACAAACATGAA (antisense primer) (SEQ ID NO: 19)<br>
The PCR condition was as follows; 25 cycles of 94 for 15 seconds, 55°C for 30 seconds and 72°C for 90 seconds.   In the combination of ast49 and ast5O, the fragment which has 1.25 kb in its length was amplified.   In the combination of ast36 and ast37, the fragment which has 0.3 kb in its length was amplified.   These fragments were cloned into pCR2.1-T0P0 and transformed E. coli TOPIO.   Plasmids were prepared from 6 independent white colonies, respectively and subjected to the sequencing. As a result, the clones that had correct sequence of AST promoter and terminator (EP 1 035 206 Al) were selected for further study (pUAST407 for AST promoter and pAST526 #3637 for AST terminator).<br>
Next, AST terminator sequence was fused to G418 resistant cassette by ligating Notl- plus Kpnl- digested pAST526 #3637 and Kpnl- plus Sad-digested pG418Sa330 (EP 1 035 206 Al) to Notl- and Sad- digested pBluescriptll SK- (Stratagene).   The ligation mixture was transformed into competent KB822 cells.   As a result of restriction analysis, one clone (pUAST418) which had the correct structure was selected for further study.<br>
Then, 3.1 kb of Sad fragment containing ribosomal DNA (rDNA) locus (Wery et al., Gene, 184, 89-97, 1997) was inserted in the downstream of G418 cassette of pUAST418.   rDNA fragment exists in multicopies on the chromosome of Eukaryote.   The integration event via the rDNA fragment results in multicopied integration onto the chromosome of the host used<br><br>
and this enables the overexpression of foreign genes which are harbored in expression vector.   For this purpose, Sad fragment from pGBPh9 containing rDNA gene was ligated to Sacl-digested and bacterial alkaline phosphatase-treated pUAST418.  The ligation mixture was transformed into competent KB822 cells.  As a result of restriction analysis, two clones, in which rDNA fragment was inserted in different orientation each other were selected for further study (pURDNA421 and pURDNAR421).<br>
Subsequently, AST promoter was inserted in the upstream of AST terminator to construct of expression vector which functions in P. rhodozyma.   l.Okb of Notl- and Bg/11- fragment of pUAST407 and 0.25kb fragment of Bg/11 and PstIfragment of pUAST407 were ligated to Notl- and Sse8387I- digested pURDNA421 or pURDNAR421. The competent KB822 cells were transformed by the ligation mixture and each 6 resultant colonies were subjected to restriction analysis.  The clones which had the correct insertion of AST promoter were selected for further study (pF718 and pR'718 which had the opposite orientation of rDNA fragment) to each other.<br>
Finally, antisense AOX construct was completed by inserting the 1.2kb of Sfil fragment of pAOX1007#0102 into the S/il-digested pF718 or pR718. The resultant plasmids were named as pFAOX828 and pRAOX828.<br>
Example 11   Transformation of P. rhodozyma with AOX-antisense vector<br>
The AOX-antisense vectors, pFAOX828 and pRAOX828 were transformed into P. rhodozyma wild type strain, ATCC96594.   Biolistic transformation was performed according to the method described in Methods in Molecular Biology (Johnson et al., 53, 147-153, 1996).  P. rhodozyma strain, ATCC96594 was cultured in YPD medium to the<br><br>
stationary phase.  After centrifugation of the broth, cells were concentrated by 10-fold with sterilized water and 200 microliter of the cell suspension was spread on YPD medium containing 100 microgram/ml of geneticin, and 0.75M of D-mannitol and D-sorbitol.   Five microgram of plasmids was coated on 1.5 mg of 0.9 micrometer gold particle, and used as donor DNAs for the Biolistic transformation.  One geneticin resistant colony which was transformed with pFAOX828 and showed enhanced pigmentation was selected for further characterization in view of its productivity of astaxanthin and its decreased activity of alternative oxidase which was encoded by AOX gene.<br>
Example 12 Characterization of the pFAOX828 integrant derived from P. rhodozyma, ATCC96594<br>
The P. rhodozyma transformant, ATCC96594 :: pFAOX828, together with its parent strain ATCC96594 was cultured in 50 ml of YPD medium in<br>
500 ml Erlenmeyer flask at 20 °C for 3 days by using their seed culture which grew in 10 ml of YPD medium in test tubes (21 mm in diameter) at 20 °C for 3 days.   At different time points, e.g. at 24, 43 and 65 hours after the inoculation, appropriate volume of culture broth was withdrawn and used for analysis of their growth, productivity of astaxanthin and their oxygen uptake activity under the presence or absence of KCN.   The 24, 43 and 65 hour-time point corresponded to their growth phase of late log-phase, mid-stationary and late stationary, respectively.<br>
For analysis of growth, optical density at 660 nm was measured by using UV-1200 photometer (Shimadzu Corp., Kyoto, Japan) in addition to the determination of their dried cell mass by drying up the cells derived from 1 ml of broth after microcentrifugation at 100 °C for one day.<br><br>
For analysis of content of astaxanthin and total carotenoids, cells were harvested from 1.0 ml of broth after microcentrifugation and used for the extraction of the carotenoids from cells of P. rhodozyma by disruption with glass beads.  After extraction, disrupted cells were then removed by centrifugation and the resultant was analyzed for carotenoid content with HPLC.  The HPLC condition used was as follows;<br>
HPLC column; Chrompack Lichrosorb si-60 (4.6 mm, 250 mm)<br>
Temperature; room temperature<br>
Eluent; acetone / hexane (18/82) add 1 ml/L of water to eluent<br>
Injection volume; 10 microliter<br>
Flow rate; 2.0 ml/minute<br>
Detection; UV at 450 nm<br>
A reference sample of astaxanthin was obtained from F. Hoffmann-La Roche AG (Basel, Switzerland).<br>
For determination of respiration activity by measuring oxygen uptake activity in the presence or absence of KCN, the DO meter model, B-505 and the DO probe, GU-BM manufactured by lijima Electronics Corporation (Aichi, Japan) were used.   Harvested cells   were resuspended with 0.5 M KPB (pH 7.4).   Then, 200 microliter of this cell suspension was diluted with 2.3 ml of water in the chamber of DO analysis.   The measurement was initiated by the addition of 0.2 ml of 1 M glucose in the presence or absence of 0.48 mM KCN.<br>
Results are summarized in TABLE 9.<br><br><br>
(Activity of respiration is expressed by nmol 02-uptake / min x mg-dried cells.)<br>
As shown in TABLE 9, antisense AOX transformant ATCC96594 :: pFAOX828 showed similar cell yield to parent strain, ATCC96594 at 43 hours of culture although it showed slower growth at 24 hours. Astaxanthin and carotenoid content of the transformant increased by 15 %. In view of its respiration activity, the transformant had the similar activity of KCN-sensitive respiration to its host strain (95 %) but KCN-resistant respiration mediated by alternative.oxidase was decreased to 20 % levels of<br><br>
its host strain at 24 hours culture and perfectly impaired at 43 hours of culture.<br>
From this result, it was indicated that the decrease of alternative oxidase activity which mediated KCN-resistant respiration might lead to the overproduction of astaxanthin and carotenoids in P. rhodozyma.<br><br><br><br>
WE CLAIM:<br>
1.	A process for producing a carotenoid, which comprises the steps of cultivating an organism selected from organisms obtained by cultivating a parent organism such as herein described, which is capable of producing a carotenoid, under the conditions for reducing an alternative oxidase activity and selecting an organism with enhanced productivity of carotenoid and a mutant strain of which productivity of carotenoid is enhanced with the aid of alteration of the resistance against an alternative oxidase inhibitor, in a suitable medium, such as YPD medium, and at a suitable temperature such as 20°C, and recovering the resulting carotenoid in a known manner.<br>
2.	The process according to claim 1 wherein the conditions for reducing an alternative oxidase activity comprises the presence of an alternative oxidase inhibitor.<br>
3.	The process according to claim 2, wherein the alternative oxidase inhibitor is selected from the group consisting of n-propyl gallate and salicylhydroxamic acid.<br>
4.	The process according to claim 1 wherein the gene expression of the alternative oxidase is altered with the aid of the technique selected from antisense technology, sitedirected mutagenesis, chemical mutagenesis, etc.<br><br>
5.	The process according to claim 4, wherein the nucleic acid sequence as illustrated by SEQ ID NO: 2 or having identity with SEQ ID NO: 2 higher than 55% is involved.<br>
6.	The process according to claim 5, wherein the nucleic acid sequence as illustrated by SEQ ID NO: 2 encodes an amino acid sequence as illustrated by SEQ ID NO: I or having identity with SEQ ID NO: 1 higher than 51.5%.<br>
7.	The process according to claim 1, where in the said organism belongs to the kingdom of Protista or Fungi, more preferably to the genus Synechococcus, Synechocystis, Haematococcus, Dunaliella, Phaffia, Xanthophyllomyces, Neurospora, Rhodotorula, Elakeslea, or Phycomyces.<br>
8.	The process according to claim 7, wherein the said organism is a strain of Phaffia rhodozyma.<br>
9.	The process according to claim 8, wherein the said organism is a strain selected from the group consisting of DSM 13429, DSM 13430 and DSM 13431.<br><br>
10.    A process for producing a carotenoid, substantially as herein described with reference to the accompanying drawing.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0411-mas-2001 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">0411-mas-2001 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">0411-mas-2001 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0411-mas-2001 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0411-mas-2001 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQxMS1tYXMtMjAwMSBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">0411-mas-2001 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="193086-method-of-and-apparatus-for-producing-beverage-containers.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="193089-a-process-for-preparing-ethylbenzene.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>193087</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>411/MAS/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2010</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Sep-2010</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td></td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-May-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F, HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>124 GRENZACHERSTRASSE CH 4070 BASLE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TATSUO HOSHINO,</td>
											<td>FUETA 808-47 KAMAKURA-SHI KANAGAWA-KEN 248-0027</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KAZUYUKI OJIMA,</td>
											<td>ZENGYOUDANCHI 3-22-304, FUJISAWA-SHI, KANAGAWA-KEN 251-0877</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YUTAKA SETOGUCHI,</td>
											<td>KATASEYAAMA 4-27-2, FUJISAWA-SHI, KANAGAWA-KEN, 251-0033,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K45/05</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>00111148.3</td>
									<td>2000-05-24</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/193087-a-process-for-producing-a-carotenoid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:38:24 GMT -->
</html>
