<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/194265-a-method-for-preparing-a-dna-construct-with-glyphosate-tolerance by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:47:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 194265:A METHOD FOR PREPARING A DNA CONSTRUCT WITH GLYPHOSATE TOLERANCE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR PREPARING A DNA CONSTRUCT WITH GLYPHOSATE TOLERANCE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for preparing a DNA construct comprising a first and a second expression cassette that when expressed in a monocot crop plant confers tolenrance to glyphosate comprising: (a) preparing a first cassette which is in operable linkage comprises (i) a rice actin 1 promoter; (ii) a rice actin 1 intron; (iii) a chloroplast transit peptide encoding DNA molecule; Civ) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (v) a transcriptional terminator DNA molecule; (b) preparing a second cassette which m in operable linkage comprises (a) a cauliflower mosaic virus (CaMV) 35S promoter; (b) a Hsp70 intron; (c) a chloroplast transit peptide encoding DNA molecule; (d) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (e) a transcriptional terminator DNA molecule; and (c) operably joining the first and second cassettes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present jinventxon relates to a method for preparing a 'DNA construct with glyphosate tolerance.<br>
The present invention relates to the field of plant molecular biology, specifically the<br>
invention relates to a DNA construct for conferring glyphosate tolerance to a plant. The<br>
invention more specifically relates to a glyphosate tolerant com plant PV-ZMGT32(nk603) and to assays for detecting the presence of com plant PV-ZMGT32(nk603) DNA in a sample and<br>
compositions thereof<br>
BACKGROUND OF THE INVENTION<br>
This invention relates to the glyphosate herbicide tolerant com (Zea mays) plant PV-<br>
 ZMGT32(nk603) and to the DNA plant expression constmct of com plant PV-ZMGT32(nk603) and the detection of the transgene/genomic insertion region in com PV-ZMGT32(nk603) and progeny thereof Com is an important crop and is a primar&gt;' food source in many areas of the world. The ly methods of biotechnology have been applied to com for improvement of the agronomic traits and the quality of the product. One such agronomic trait is herbicide tolerance, in particular, tolerance to glyphosate herbicide. This trait in com has been conferred by the expression of a transgene in the com plants (U.S. Patent No. 6,040,497).<br>
The  expression  of foreign  genes  in  plants  is  known  to  be  influenced  by  their<br>
if chromosomal position, perhaps due to chromatin stmcture (e.g.. heterochromatin) or the proximity of transcriptional regulation elements (e.g., enhancers) close to the integration site (Weising et al., Ann. Rev. Genet 22:421-477, 1988). For this reason, it is often necessary to screen a large number of events in order to identify an event characterized by optimal expression of a introduced gene of interest.   For example, it has been observed in plants and in other<br>
/25 organisms that there may be a wide variation in levels of expression of an introduced genes among events. There may also be differences in spatial or temporal patterns of expression, for example, differences in the relative expression of a transgene in various plant tissues, that may not correspond to the pattems expected from transcriptional regulatory elements present in the introduced gene constmct.  For this reason, it is common to produce hundreds to thousands of<br>
30^   different events and screen those events for a single event that has desired transgene expression<br>
'      levels and pattems for commercial purposes. An event that has desired levels or pattems of<br>
D'!--' '^K\Z	j^-2      15 OCT Z003<br><br>
transgene expression is useful for introgressing the transgene into other genetic backgrounds by sexual outcrossing using conventional breeding methods. Progeny of such crosses maintain the transgene expression characteristics of the original transformant. This strategy is used to ensure reliable gene expression in a number of varieties that are well adapted to local growing 5    conditions.<br>
It would be advantageous to be able to detect the presence of a particular event in order to determine whether progeny of a sexual cross contain a transgene of interest. In addition, a method for detecting a particular event would be helpful for complying with regulations requiring the premarket approval and labeling of foods derived from recombinant crop plants, for<br>
10 example. It is possible to detect the presence of a transgene by any well known nucleic acid detection method such as the polymerase chain reaction (PCR) or DNA hybridization using nucleic acid probes. These detection methods generally focus on frequently used genetic elements, such as promoters, terminators, marker genes, etc. As a result, such methods may not be useful for discriminating between different events, particularly those produced using the same<br>
15 DNA construct unless the DNA sequence of chromosomal DNA adjacent to the inserted DNA ("flanking DNA") is known. An event-specific PCR assay is discussed, for example, by Windels et al. (Med. Fac. Landbouww, Univ. Gent 64/5b:459-462, 1999), who identified glyphosate tolerant soybean event 40-3-2 by PCR using a primer set spanning the junction between the insert and flanking DNA, specifically one primer that included sequence from the insert and a<br>
20    second primer that included sequence from flanking DNA.<br>
SUMMARY OF THE INVENTION<br>
According to one aspect of the invention, a DNA construct is provided that when expressed in plant cells and plants confers tolerance to glyphosate herbicide. This invention<br>
25 relates preferably to the methods for producing and selecting a glyphosate tolerant monocot crop plant. The DNA construct consists of two transgene expression cassettes. The first expression cassette comprising a DNA molecule of a rice {Oryzae saliva) actin 1 promoter and rice actin 1 intron operably joined to a DNA molecule encoding a chloroplast transit peptide sequence, operably   cormected   to   a   DNA   molecule   encoding   a   glyphosate   resistant   5-enol-<br>
30    pyruvylshikimate-3-phosphate  synthase  (EPSPS),  operably  connected to  a DNA  molecule<br><br>
comprising a 3' transcriptional terminator. The second transgene expression cassette of the DNA construct comprising a DNA molecule of the cauliflower mosaic virus (CaMV) 35S promoter, operably connected to a DNA molecule comprising a Hsp70 intron, operably connected to a DNA molecule encoding a chloroplast transit peptide sequence, operably connected to a DNA 5 molecule encoding a glyphosate resistant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS), operably connected to a DNA molecule comprising a 3' transcriptional terminator.<br>
More specifically, a DNA construct is provided that when expressed in plant cells and plants confers tolerance to glyphosate herbicide. This invention relates preferably to the methods for producing and selecting a glyphosate tolerant com plant. The DNA construct consists of two 10 transgene expression cassettes. The first expression cassette consisting of a DNA molecule of a rice (Oryzae sativa) actin 1 promoter and rice actin 1 intron operably joined to a DNA molecule encoding an Arabidopsis EPSPS chloroplast transit peptide sequence, operably connected to a DNA molecule encoding a glyphosate resistant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) isolated from Agrobacterium tumefaciens sp. strain CP4, operably connected to a DNA 15 molecule consisting of a nopaline synthase transcriptional terminator. The second transgene expression cassette consisting of a DNA molecule of the cauliflower mosaic virus (CaMV) 35S promoter containing a tandem duplication of the enhancer region, operably connected to a DNA molecule consisting of a Zea mays Hsp70 intron, operably connected to a DNA molecule encoding an Arabidopsis EPSPS chloroplast transit peptide sequence, operably connected to a 20 DNA molecule encoding a glyphosate resistant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) isolated from Agrobacterium tumefaciens sp. strain CP4. operably connected to a DNA molecule consisting of a nopaline synthase transcriptional terminator.<br>
According to another aspect of the invention, compositions and methods are provided for detecting the presence of the transgene/genomic insertion region from a novel com plant 25 designated PV-ZMGT32(nk603). DNA molecules are provided that comprise at least one junction sequence of PV-ZMGT32(nk603) selected from the group consisting of 5' TGTAGCGGCCCACGCGTGGT 3' (SEQ ID N0:9), 5' TACCACGCGACACACTTC 3' (SEQ ID NO:10), and 5' TGCTGTTCTGCTGACTTT 3' (SEQ ID NO:ll) and complements thereof; wherein a junction sequence spans the junction between heterologous DNA inserted into<br><br>
the genome and the DNA from the com cell flanking the insertion site and is diagnostic for the event. The com plant and seed comprising these molecules is an aspect of this invention.<br>
A novel DNA molecule 5'ACCAAGCTTTTATAATAG 3' (SEQ ID N0:12) and the complement thereof, wherein this DNA molecule is novel in PV-ZMGT32(nk603) and its 5    progeny. The com plant and seed comprising this molecule is an aspect of this invention.<br>
According to another aspect of the invention, DNA molecules that comprise the novel transgene/genomic insertion region, SEQ ID N0:7 and SEQ ID N0:8 and are homologous or complementary to SEQ ID N0:7 and SEQ ID N0:8 are an aspect of this invention.<br>
DNA molecules that comprise a sufficient length of a transgene portion of the DNA 10 sequence of SEQ ID N0:7 and a similar sufficient length of a 5' flanking com DNA sequence of SEQ ID N0:7; or a similar sufficient length of a transgene portion of the DNA sequence of SEQ ID N0:8 and a similar sufficient length of a 3' DNA sequence flanking the transgene, wherein these DNA molecules are useful as DNA primers in DNA amplification methods so as to provide a DNA amplicon product specifically produced from PV-ZMGT32(nk603) DNA and its progeny 15 are another aspect of the invention. DNA primers homologous or complementary to a length of SEQ ID N0:7 and SEQ ID N0:8 are an aspect of the invention. The amplicons produced using DNA primers that are diagnostic for com event PV-ZMGT32(nk603) and its progeny are a subject of this invention.<br>
According to another aspect of the invention, methods of detecting the presence of DNA<br>
20    corresponding to the com event PV-ZMGT32(nk603) event in a sample are provided.   Such<br>
methods comprise: (a) contacting the sample comprising DNA with a DNA primer set, that when<br>
used in a nucleic acid amplification reaction with genomic DNA extracted from com event PV-<br>
ZMGT32 (nk603) produces an amplicon that is diagnostic for com event PV-ZMGT32(nk603);<br>
(b) performing a nucleic acid amplification reaction, thereby producing the amplicon; and (c)<br>
25    detecting the amplicon.    A pair of DNA molecules comprising a DNA primer set that are<br>
homologous or complementary to SEQ ID N0:7 or SEQ ID N0:8 that function in a nucleic acid<br>
amplification   reaction   to   produce   an   amplicon   DNA   molecule   diagnostic   for   PV-<br>
ZMGT329nk603).  More specifically, a pair of DNA molecules comprising a DNA primer set,<br>
wherein the DNA molecules are identified as SEQ ID NO: 13 or complements thereof and SEQ<br>
30    ID NO: 14 or complements thereof; SEQ ID NO: 15 or complements thereof and SEQ ID NO: 16<br><br>
or complements thereof.  The amplicon comprising the DNA molecules of SEQ ID NO: 13 and<br>
SEQ ID NO: 14.  The amplicon comprising the DNA molecules of SEQ ID NO: 15 and SEQ ID<br>
NO: 16. The amplicon produce by the afore described method that can hybridize under stringent<br>
conditions to SEQ ID N0:9, SEQ ID NO: 10, SEQ ID NO: 11, or SEQ ID NO: 12.<br>
5	According to another aspect of the invention, methods of detecting the presence of a<br>
DNA molecule corresponding to the PV-ZMGT32(nk603) event in a sample, such methods<br>
comprising: (a) contacting the sample comprising DNA extracted from a com plant with a DNA<br>
probe molecule that hybridizes under stringent hybridization conditions with genomic DNA from<br>
com event PV-ZMGT32(nk603) and does not hybridize under the stringent hybridization<br>
10    conditions with a control com plant DNA; (b) subjecting the sample and probe to stringent<br>
hybridization conditions; and (c) detecting hybridization of the probe to the DNA.    More<br>
specifically, a method for detecting the presence of a DNA molecule corresponding to the PV-<br>
ZMGT32(nk603) event in a sample, such methods, consisting of (a) contacting the sample<br>
comprising DNA extracted from a com plant with a DNA probe molecule that consists of SEQ<br>
15    ID N0:9, SEQ ID NO:10, SEQ ID N0:11, or SEQ ID N0:12, wherein said DNA probe<br>
molecule hybridizes under stringent hybridization conditions with genomic DNA from corn<br>
event PV-ZMGT32(nk603) and does not hybridize under the stringent hybridization conditions<br>
with a control com plant DNA; (b) subjecting the sample and probe to stringent hybridization<br>
conditions; and (c) detecting hybridization of the probe to the DNA.<br>
20	According to another aspect of the invention, methods of producing a com plant that<br>
tolerates application of glyphosate are provided that comprise the steps of: (a) sexually crossing a first parental com line comprising the expression cassettes of the present invention, which confers tolerance to application of glyphosate, and a second parental com line that lacks the glyphosate tolerance, thereby producing a plurality of progeny plants; and (b) selecting a 25 progeny plant that tolerates application of glyphosate. Such methods may optionally comprise the further step of back-crossing the progeny plant to the second parental com line to producing a tme-breeding com plant that tolerates application of glyphosate.<br>
According to another aspect of the invention, a method is provided to select for<br>
glyphosate tolerant com plants of the present invention and progeny thereof comprising<br>
30    extracting DNA from a plant sample, contacting a DNA with a marker nucleic acid molecule<br><br>
selected from the group consisting of SEQ ID N0:9, SEQ ID NO: 10, SEQ ID NO: 11, or SEQ ID NO: 12, or complements thereof, detecting the hybridization of said marker nucleic acid molecule to the DNA, and performing a marker assisted breeding analysis for the genetic linkage of the glyphosate tolerant trait to the marker nucleic acid molecule.<br>
The present invention provides a method of producing a com plant tolerant to glyphosate herbicide comprising transforming a com cell with the DNA constmct (pMON25496), selecting the com cell for tolerance to the treatment with an effective dose of glyphosate, and growing the com cell into a fertile com plant. The fertile com plant can be self pollinated or crossed with compatible com varieties to produce glyphosate tolerant progeny.<br>
The invention further relates to a DNA detection kit comprising at least one DNA molecule of sufficient length of contiguous nucleotides homologous or complementary to SEQ ID N0:7 or SEQ ID N0:8 that functions as a DNA primer or probe specific for com event PV-ZMGT32(nk603) or its progeny.<br>
This invention further relates to the plants and seeds of glyphosate tolerant com {Zea mays) PV-ZMGT32 (nk603) having ATCC Accession No. PTA-2478 and the progeny derived thereof The com plant or its parts produced by growing of the glyphosate tolerant com plant PV-ZMGT32(nk603), the pollen and ovules of the com plant PV-ZMGT32(nk603). The nuclei of vegetative cells, the nuclei of pollen cells, and the nuclei of egg cells of the com plant PV-ZMGT32 (nk603) and the progeny derived thereof The com plant and seed PV-ZMGT32(nk603) from which the DNA primer molecules of the present invention provide a specific amplicon product is an aspect of the invention.<br>
15 OCT im<br><br>
Accordingly, the present invention provides a method for preparing a DNA construct comprising a first and a second expression cassette that when expressed in a monocot crop plant confers tolenrance to glyphosate comprising: (a) preparing a first cassette in a known manner wherein said first expression cassette in operable linkage comprises (i) a rice actin 1 promoter; (ii) a rice actin 1 intron; (iii) a chloroplast transit peptide encoding DNA molecule; (iv) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (v) a transcriptional terminator DNA molecule; (b) preparing a second cassette in a known manner wherein said second expression cassette in operable linkage comprises (a) a cauliflower mosaic virus (CaMV) 35S promoter; (b) a Hsp70 intron; (c) a chloroplast transit peptide encoding DNA molecule; (d) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (e) a transcriptional terminator DNA molecule; and (c) operably joining the first and second cassettes in a known manner.<br>
The foregoing and other aspects of the invention will become more apparent from the following detailed description and with reference to the accompanying drawings, in which<br>
Figure 1.    Plasmid map of pMON25496<br>
This application claims the benefit of U.S. Provisional Application<br>
No.60/213,567, filed 06/22/00; U.S. Provisional Application No.60/241,215, filed<br>
10/13/00; and U.S. Provisional	<br><br>
Application No. 60/240,014, filed 10/13/00. The following definitions and methods are provided to better define the present invention and to guide those of ordinary skill in the art in the practice of the present invention. Unless otherwise noted, terms are to be understood according to conventional usage by those of ordinary skill in the relevant art. Definitions of common terms in 5 molecular biology may also be found in Rieger et al. Glossary of Genetics: Classical and Molecular, 5th edition. Springer-Verlag: New York. 1991; and Lewin, Genes V, Oxford University Press: New York, 1994. The nomenclature for DNA bases as set forth at 37 CFR § 1.822 is used.<br>
As used herein, the term "com" means Zea mays or maize and includes all plant varieties 10    that can be bred with com, including wild maize species.<br>
As used herein, the term "comprising" means "including but not limited to". "Glyphosate" refers to N-phosphonomethylglycine and its salts, Glyphosate is the active ingredient of Roundup® herbicide (Monsanto Co.). Treatments with "glyphosate herbicide" refer to treatments with the Roundup®, Roundup Ultra®, Roundup UltraMax® herbicide or any 15 other herbicide formulation containing glyphosate. The selection of application rates for a glyphosate formulation that constitute a biologically effective dose is within the skill of the ordinary agricultural technician.<br>
A DNA construct is an assembly of DNA molecules linked together that provide one or more expression cassettes. The DNA construct is preferably a plasmid that is enabled for self 20 replication in a bacterial cell and contains various endonuclease enzyme restriction sites that are useful for introducing DNA molecules that provide functional genetic elements, i.e.. promoters, introns, leaders, coding sequences, 3' termination regions, among others. The expression cassettes contained within a DNA constmct comprise the necessary genetic elements to provide transcription of a messenger RNA. The expression cassettes can be designed to express in 25 prokaryote cells or eukaryotic cells. The expression cassettes of the present invention are designed to express most preferably in plant cells. The combination of expression cassettes in transgenic plants can lead to unexpected phenotypes. In the present invention, the combination of the rice actin 1 promoter cassette and the CaMV 35S promoter cassette provided in a DNA construct (pMON25496) enhanced transgenic com plant tolerance to glyphosate herbicide when<br>
)0<br><br>
compared to the single cassettes alone and to the commercial standard GA21 that contains three copies of a rice actin 1 promoter cassette.<br>
A transgenic "event" is produced by transformation of plant cells with heterologous DNA construct, including a nucleic acid expression cassette that comprises a transgene of interest, the 5    regeneration of a population of plants resulting from the insertion of the transgene into the genome of the plant, and selection of a particular plant characterized by insertion into a particular genome location.   The term "event" refers to the original transformant and progeny of the transformant that include the heterologous DNA.    The term "event" also refers to progeny produced by a sexual outcross between the transformant and another variety that include the 10    heterologous DNA.  Even after repeated back-crossing to a recurrent parent, the inserted DNA and flanking DNA from the transformed parent is present in the progeny of the cross at the same chromosomal location.   The term "event" also refers to DNA from the original transformant comprising the inserted DNA and flanking genomic sequence immediately adjacent to the inserted DNA that would be expected to be transferred to a progeny that receives inserted DNA 15    including the transgene of interest as the result of a sexual cross of one parental line that includes the inserted DNA (e.g., the original transformant and progeny resulting from selfing) and a parental line that does not contain the inserted DNA. A glyphosate tolerant com plant PV-ZMGT32 (nk603) can be breed by first sexually crossing a first parental com plant consisting of a com plant grown from the transgenic com plant PV-ZMGT32 (nk603) having ATCC 20    Accession No. PTA-2478 and progeny thereof derived from transformation with the expression cassettes of the present invention that tolerates application of glyphosate herbicide, and a second parental com plant that lacks the tolerance to glyphosate herbicide, thereby producing a plurality of first progeny plants; and then selecting a first progeny plant that is tolerant to application of glyphosate herbicide; and selfing the first progeny plant, thereby producing a plurality of second 25    progeny plants; and then selecting from the second progeny plants a glyphosate herbicide tolerant plant. These steps can further include the back-crossing of the first glyphosate tolerant progeny plant or the second glyphosate tolerant progeny plant to the second parental com plant or a third parental com plant, thereby producing a com plant that tolerates the application of glyphosate herbicide.<br><br>
It is also to be understood that two different transgenic plants can also be mated to produce offspring that contain two independently segregating added, exogenous genes. Selfing of appropriate progeny can produce plants that are homozygous for both added, exogenous genes. Back-crossing to a parental plant and out-crossing with a non-transgenic plant are also contemplated, as is vegetative propagation. Descriptions of other breeding methods that are commonly used for different traits and crops can be found in one of several references, e.g., Fehr, in Breeding Methods for Cultivar Development, Wilcox J. ed., American Society of Agronomy, Madison WI (1987).<br>
A "probe" is an isolated nucleic acid to which is attached a conventional detectable label or reporter molecule, e.g., a radioactive isotope, ligand, chemiluminescent agent, or enzyme. Such a probe is complementary to a strand of a target nucleic acid, in the case of the present invention, to a strand of genomic DNA from com event PV-ZMGT32(nk603) whether from a com plant or from a sample that includes DNA from the event. Probes according to the present invention include not only deoxyribonucleic or ribonucleic acids but also polyamides and other probe materials that bind specifically to a target DNA sequence and can be used to detect the presence of that target DNA sequence.<br>
"Primers" are isolated nucleic acids that are annealed to a complementary target DNA strand by nucleic acid hybridization to form a hybrid between the primer and the target DNA strand, then extended along the target DNA strand by a polymerase, e.g., a DNA polymerase. Primer pairs of the present invention refer to their use for amplification of a target nucleic acid sequence, e.g.. by the polymerase chain reaction (PCR) or other conventional nucleic-acid amplification methods.<br>
Probes and primers are of sufficient nucleotide length to bind to the target DNA sequence specifically in the hybridization conditions or reaction conditions determined by the operator. This length may be of any length that is of sufficient length to be useful in the detection method of choice. Generally, 11 nucleotides or more in length, preferably 18 nucleotides or more, more preferably 24 nucleotides or more, and most preferably 30 nucleotides or more are used. Such probes and primers hybridize specifically to a target sequence under high stringency hybridization conditions. Preferably, probes and primers according to the present invention have complete DNA sequence similarity of contiguous nucleotides with the target sequence, although<br><br>
probes differing from the target DNA sequence and that retain the ability to hybridize to target DNA sequences may be designed by conventional methods.<br>
Methods for preparing and using probes and primers are described, for example, in Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, ed. Sambrook et al.. Cold Spring 5 Harbor Laboratory Press, Cold Spring Harbor, NY, 1989 (hereinafter, "Sambrook et al., 1989"); Current Protocols in Molecular Biology, ed. Ausubel et al., Greene Publishing and Wiley-Interscience, New York, 1992 (with periodic updates) (hereinafter, "Ausubel et al., 1992"); and Innis et al., PCR Protocols: A Guide to Methods and Applications, Academic Press: San Diego, 1990. PCR-primer pairs can be derived from a known sequence, for example, by using computer 10 programs intended for that purpose such as Primer (Version 0.5, © 1991, Whitehead Institute for Biomedical Research, Cambridge, MA).<br>
Primers and probes based on the flanking DNA and insert sequences disclosed herein can<br>
be used to confirm (and, if necessary, to correct) the disclosed sequences by conventional<br>
methods, e.g., by re-cloning and sequencing such sequences.<br>
15	The nucleic acid probes and primers of the present invention hybridize under stringent<br>
conditions to a target DNA  sequence.     Any conventional  nucleic acid hybridization or<br>
amplification method can be used to identify the presence of DNA from a transgenic event in a<br>
sample. Nucleic acid molecules or fragments thereof are capable of specifically hybridizing to<br>
other nucleic acid molecules under certain circumstances.   As used herein, two nucleic acid<br>
20    molecules are said to be capable of specifically hybridizing to one another if the two molecules<br>
are capable of forming an anti-parallel, double-stranded nucleic acid structure.   A nucleic acid<br>
molecule is said to be the "complement" of another nucleic acid molecule if they exhibit<br>
complete  complementarity.     As   used   herein,   molecules   are   said  to   exhibit  "complete<br>
complementarity" when every nucleotide of one of the molecules is complementary to a<br>
25    nucleotide of the other.  Two molecules are said to be "minimally complementary" if they can<br>
hybridize to one another with sufficient stability to permit them to remain annealed to one<br>
another under at least conventional "low-stringency" conditions.   Similarly, the molecules are<br>
said to be "complementary" if they can hybridize to one another with sufficient stability to<br>
permit them to remain annealed to one another under conventional "high-stringency" conditions.<br>
30    Conventional stringency conditions are described by Sambrook et al., 1989, and by Haymes et<br><br>
ai. In: Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, DC (1985), Departures from complete complementarity are therefore permissible, as long as such departures do not completely preclude the capacity of the molecules to form a double-stranded structure. In order for a nucleic acid molecule to serve as a primer or probe it need only be sufficiently 5 complementary in sequence to be able to form a stable double-stranded structure under the particular solvent and salt concentrations employed.<br>
As used herein, a substantially homologous sequence is a nucleic acid molecule that will<br>
specifically hybridize to the complement of the nucleic acid molecule to which it is being<br>
compared under high stringency conditions.  Appropriate stringency conditions which promote<br>
10    DNA hybridization, for example, 6.0 x sodium chloride/sodium citrate (SSC) at about 45°C,<br>
followed by a wash of 2.0 x SSC at 50°C, are known to those skilled in the art or can be found in<br>
Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6.   For<br>
example, the salt concentration in the wash step can be selected from a low stringency of about<br>
2.0 x SSC at 50°C to a high stringency of about 0.2 x SSC at 50°C. In addition, the temperature<br>
15    in the wash step can be increased from low stringency conditions at room temperature, about<br>
22°C, to high stringency conditions at about 65°C. Both temperature and salt may be varied, or<br>
either the temperature or the salt concentration may be held constant while the other variable is<br>
changed.   In a preferred embodiment, a nucleic acid of the present invention will specifically<br>
hybridize to one or more of the nucleic acid molecules set forth in SEQ ID NO: 9, 10, 11 and 12<br>
20    or complements thereof or fragments of either under moderately stringent conditions, for<br>
example at about 2.0 x SSC and about 65°C.  In a particularly preferred embodiment, a nucleic<br>
acid of the present invention will specifically hybridize to one or more of the nucleic acid<br>
molecules set forth in SEQ ID N0:9 through SEQ ID NO: 12 or complements or fragments of<br>
either under high stringency conditions. In one aspect of the present invention, a preferred<br>
25    marker nucleic acid molecule of the present invention has the nucleic acid sequence set forth in<br>
SEQ ID N0:9 through SEQ ID NO: 12 or complements thereof or fragments of either. In another<br>
aspect of the present invention, a preferred marker nucleic acid molecule of the present invention<br>
shares between 80% and 100% or 90% and 100% sequence identity with the nucleic acid<br>
sequence set forth in SEQ ID N0:9 through SEQ ID N0:12 or complement thereof or fragments<br>
30    of either. In a further aspect of the present invention, a preferred marker nucleic acid molecule<br><br>
of the present invention shares between 95% and 100% sequence identity with the sequence set<br>
forth in SEQ ID N0:9 through SEQ ID NO; 12 or complement thereof or fragments of either.<br>
SEQ ID N0:9 through SEQ IN NO: 12   may be used as markers in plant breeding methods to<br>
identify the progeny of genetic crosses similar to the methods described for simple sequence<br>
5    repeat DNA marker analysis, in "DNA markers: Protocols, applications, and overviews: (1997)<br>
173-185, Cregan, et al., eds., Wiley-Liss NY; all of which is herein incorporated by reference in<br>
its' entirely. The hybridization of the probe to the target DNA molecule can be detected by any<br>
number of methods known to those skilled in the art, these can include, but are not limited to,<br>
fluorescent tags, radioactive tags, antibody based tags, and chemiluminescent tags.<br>
10	Regarding the amplification of a target nucleic acid sequence (e.g., by PCR) using a<br>
particular amplification primer pair, "stringent conditions" are conditions that permit the primer<br>
pair to hybridize only to the target nucleic-acid sequence to which a primer having the<br>
corresponding wild-type sequence (or its complement) would bind and preferably to produce a<br>
unique amplification product, the amplicon, in a DNA thermal amplification reaction.<br>
15	The term "specific for (a target sequence)" indicates that a probe or primer hybridizes<br>
under stringent hybridization conditions only to the target sequence in a sample comprising the target sequence.<br>
As used herein, "amplified DNA" or "amplicon" refers to the product of nucleic-acid<br>
amplification of a target nucleic acid sequence that is part of a nucleic acid template.   For<br>
20    example, to determine whether the com plant resulting from a sexual cross contains transgenic<br>
event genomic DNA from the com plant of the present invention, DNA extracted from a corn<br>
plant tissue sample may be subjected to nucleic acid amplification method using a DNA primer<br>
pair that includes a first primer derived from flanking sequence in the genome of the plant<br>
adjacent to the insertion site of inserted heterologous DNA, and a second primer derived from<br>
25    the inserted heterologous DNA to produce an amplicon that is diagnostic for the presence of the<br>
event DNA. The amplicon is of a length and has a sequence that is also diagnostic for the event.<br>
The amplicon may range in length from the combined length of the primer pairs plus one<br>
nucleotide base pair, preferably plus about fifty nucleotide base pairs, more preferably plus about<br>
two hundred-fifty nucleotide base pairs, and even more preferably plus about four hundred-fifty<br>
30    nucleotide base pairs. Alternatively, a primer pair can be derived from flanking sequence on both<br><br>
sides of the inserted DNA so as to produce an amplicon that includes the entire insert nucleotide sequence (e.g., the Mlul DNA fragment of the pMON25496 expression construct. Figure 1, approximately 6706 nucleotide base pairs). A member of a primer pair derived from the plant genomic sequence may be located a distance from the inserted DNA sequence, this distance can 5 range from one nucleotide base pair up to the limits of the amplification reaction, or about twenty thousand nucleotide base pairs. The use of the term "amplicon" specifically excludes primer dimers that may be formed in the DNA thermal amplification reaction.<br>
Nucleic-acid amplification can be accomplished by any of the various nucleic-acid<br>
amplification methods known in the art, including the polymerase chain reaction (PCR).   A<br>
10    variety of amplification methods are known in the art and are described, inter alia, in U.S. Patent<br>
Nos. 4,683,195 and 4,683,202 and in FCR Protocols: A Guide to Methods and Applications, ed.<br>
Innis et ai. Academic Press, San Diego, 1990.      PCR amplification methods have been<br>
developed to amplify up to 22 kb of genomic DNA and up to 42 kb of bacteriophage DNA<br>
(Cheng et al, Proc. Natl. Acad. Sci. USA 91:5695-5699, 1994). These methods as well as other<br>
15    methods known in the art of DNA amplification may be used in the practice of the present<br>
invention. The sequence of the heterologous DNA insert or flanking DNA sequence from corn<br>
event PV-ZMGT32(nk603) can be verified (and corrected if necessary) by amplifying such<br>
sequences from DNA extracted from the ATCC deposit Accession No. PTA-2478 seed or plants<br>
using DNA primers derived from the sequences provided herein followed by standard DNA<br>
20    sequencing of the PCR amplicon or of the cloned DNA.<br>
The amplicon produced by these methods may be detected by a plurality of techniques. One such method is Genetic Bit Analysis (Nikiforov, et al. Nucleic Acid Res. 22:4167-4175. 1994) where an DNA oligonucleotide is designed which overlaps both the adjacent flanking genomic DNA sequence and the inserted DNA sequence . The oligonucleotide is immobilized in 25 wells of a microwell plate. Following PCR of the region of interest (using one primer in the inserted sequence and one in the adjacent flanking genomic sequence), a single-stranded PCR product can be hybridized to the immobilized oligonucleotide and serve as a template for a single base extension reaction using a DNA polymerase and labelled ddNTPs specific for the expected next base.   Readout may be fluorescent or ELISA-based.   A signal indicates presence of the<br><br>
insert/flanking  sequence  due  to  successful  amplification,  hybridization,  and  single  base extension.<br>
Another method is the Pyrosequencing technique as described by Winge (Innov. Pharma. Tech. 00:18-24, 2000). In this method an oligonucleotide is designed that overlaps the adjacent 5 genomic DNA and insert DNA junction. The oligonucleotide is hybridized to single-stranded PCR product from the region of interest (one primer in the inserted sequence and one in the flanking genomic sequence) and incubated in the presence of a DNA polymerase, ATP, sulfiirylase, luciferase, apyrase, adenosine 5' phosphosulfate and luciferin. DNTPs are added individually and the incorporation results in a light signal which is measured. A light signal 10 indicates the presence of the transgene insert/flanking sequence due to successful amplification, hybridization, and single or multi-base extension.<br>
Fluorescence Polarization as described by Chen, et al., (Genome Res. 9:492-498. 1999) is a method that can be used to detect the amplicon of the present invention. Using this method an oligonucleotide is designed which overlaps the genomic flanking and inserted DNA junction. 15 The oligonucleotide is hybridized to single-stranded PCR product fi-om the region of interest (one primer in the inserted DNA and one in the flanking genomic DNA sequence) and incubated in the presence of a DNA polymerase and a fluorescent-labeled ddNTP. Single base extension results in incorporation of the ddNTP. Incorporation can be measured as a change in polarization using a fluorometer. A change in polarization indicates the presence of the 20 transgene insert/flanking sequence due to successful amplification, hybridization, and single base extension.<br>
Taqman® (PE Applied Biosystems, Foster City, CA) is described as a method of detecting and quantifying the presence of a DNA sequence and is fully understood in the instructions provided by the manufacturer. Briefly, a FRET oligonucleotide probe is designed 25 which overlaps the genomic flanking and insert DNA junction. The FRET probe and PCR primers (one primer in the insert DNA sequence and one in the flanking genomic sequence) are cycled in the presence of a thermostable polymerase and dNTPs. Hybridization of the FRET probe results in cleavage and release of the fluorescent moiety away from the quenching moiety on the FRET probe. A fluorescent signal indicates the presence of the flanking/transgene insert 30    sequence due to successful amplification and hybridization.<br><br>
Molecular Beacons have been described for use in sequence detection as described in Tyangi, et al. (Nature Biotech. 14:303-308, 1996)   Briefly, a FRET oligonucleotide probe is designed that overlaps the flanking genomic and insert DNA junction. The unique structure of the FRET probe results in it containing secondary structure that keeps the fluorescent and quenching moieties in close proximity.  The FRET probe and PCR primers (one primer in the insert DNA sequence and one in the flanking genomic sequence) are cycled in the presence of a thermostable polymerase and dNTPs. Following successful PCR amplification, hybridization of the FRET probe to the target sequence results in the removal of the probe secondary structure and spatial separation of the fluorescent and quenching moieties. A fluorescent signal results. A fluorescent signal indicates the presence of the flanking/transgene insert sequence due to successful amplification and hybridization.<br>
The following examples are included to demonstrate examples of certain preferred embodiments of the invention. It should be appreciated by those of skill in the art that the techniques disclosed in the examples that follow represent approaches the inventors have found function well in the practice of the invention, and thus can be considered to constitute examples of preferred modes for its practice. However, those of skill in the art should, in light of the present disclosure, appreciate that many changes can be made in the specific embodiments that are disclosed and still obtain a like or similar result without departing from the spirit and scope of the invention.<br>
EXAMPLES<br>
EXAMPLE 1<br>
The PV-ZMGT32(nk603) (hence forth referred to as nk603) transgenic com event was generated by microprojectile bombardment of maize embryos (Songstad et al.. In Vitro Ceil Plant 32:179-183, 1996) using a linear Mlu I DNA fragment derived from pMON25496 (Figure 1). This DNA fragment contains two transgene expression cassettes that collectively confer corn plant tolerance to glyphosate. The first cassette is composed of the rice actin 1 promoter and intron (P-Os.Actl and I-Os.Actl, U.S. Patent No. 5,641,876), operably connected to an Arabidopsis EPSPS chloroplast transit peptide (TS-At.EPSPS:CTP2, Klee et al., Mol. Gen. Genet. 210:47-442, 1987), operably connected to a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) from Agrobacterium sp. strain CP4 (AGRTU.aro^:CP4, U.S.<br><br>
Patent No. 5,633,435) and operably connected to a nopaline synthase transcriptional terminator (T-AGRTU.nos, Fraley et al.   Proc. Natl. Acad. Sci. USA 80:4803-4807, 1983).   The second transgene expression cassette consists of the cauliflower mosaic virus 35S promoter containing a tandem duplication of the enhancer region (P-CaMV.35S, Kay et al. Science 236:1299-1302, 5     1987; U.S. Patent No. 5,164,316), operably connected to a Zea mays Hsp70 intron (I-Zm.Hsp70, U.S. Patent No. 5,362,865), operably connected to an Arabidopsis EPSPS chloroplast transit peptide (TS-At.EPSPS:CTP2, Klee et al.,  Mol. Gen. Genet.  210:47-442,  1987), operably connected to a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) from Agrobacterium sp. strain CP4 (AGRTU.arov4:CP4, U.S. Patent No. 5,633,435) and operably 10    connected to a nopaline synthase transcriptional terminator (T-AGRTU.nos, Fraley et al.   Proc. Natl. Acad. Sci. USA 80:4803-4807, 1983).    Post-bombardment, glyphosate-tolerant transgenic calli were selected on media containing 3 mM glyphosate and plants were subsequently regenerated.  Three hundred-four plants from 91 independent transgenic events were produced, nk603 was selected from this population based on a superior combination of characteristics,<br>
15 including glyphosate tolerance, agronomic performance, and single transgenic insertion. Greenhouse and field evaluations of the nk603 event and its derived progeny indicated this transgenic insertion confers tolerance that exceeds commercial specifications for full vegetative and reproductive tolerance to 340 g glyphosate/acre (840 g glyphosate/hectare; 32 oz Roundup Ultra/acre) when applied at the V4 and V8 leaf stages.<br>
20    EXAMPLE 2<br>
The glyphosate tolerant nk603 com event was compared to the current commercial standard, GA21 (U.S. Patent No. 6,040,497), for tolerance to glyphosate vegetative injury and effect on yield. GA21 contains at least 3 transgene expression cassettes arranged in tandem in the com genome of GA21  event (SCP/GMO/232-Final,  European Commission Health &amp;<br>
25 Consumer Protection Directorate-General). The transgene cassette of GA21 consists of a rice actin 1 promoter and intron linked to a ribulose 1,5-bisphosphate carboxylase chloroplast transit peptide linked to a modified maize glyphosate resistant EPSPS and a 3' nopaline synthase transcription termination region. Plants of nk603 and GA21 were planted in rows in replicated field plots.  The treatments were 1) unsprayed, 2) sprayed at a 64 ounces/acre rate of Roundup<br>
30    Ultra® at V4 leaf stage and again with 64 ounces/acre Roundup Ultra® at V8 leaf stage, 3)<br><br>
sprayed at a 96 ounces/acre rate of Roundup Ultra® at V4 leaf stage and again with 96 ounces/acre Roundup Ultra® at V8 leaf stage.    Vegetative tolerance was measured as percent vegetative injury determined by the amount of leaf malformation observed 10 days after the herbicide treatment at the V8 leaf stage.  The yield of each plot was measured in bushels/acre and the percent reduction in yield determined for each herbicide treatment relative to the unsprayed treatment. The results shown in Table 1 illustrates that nk603 shows a lower percent vegetative injury level than the GA21 plants and the observed percent yield reduction is also less for the nk603 event. A low incidence of vegetative injury was observed in the unsprayed plots, this observation is attributable to various environmental factors other than glyphosate herbicide exposure. The double expression cassette of pMON25496 in nk603 was compared to vegetative injury and fertility rating of 3 independent com events obtained from only the CaMV.35S promoter driving expression of the glyphosate tolerance gene (AGRTU.aroA:CP4).    It was observed that the double expression cassette conferred a higher level of vegetative tolerance and reproductive tolerance than three independent com events (ev I, ev 2, and ev 3) containing only the expression cassette where the glyphosate tolerance gene expression was driven by the CaMV.35S promoter. A higher level of vegetative tolerance to glyphosate herbicide injury was observed for nk603 plus 3 additional com events derived from pMON25496 when compared to the average injury of 6 com events derived from a construct where the glyphosate tolerance gene expression was driven only by the rice actin promoter and intron (P-Os.Actl/I-Os.Actl).   The plants transformed with the double expression cassette possessed a higher level of glyphosate tolerance to vegetative and fertility injury than plants derived from transformation with the single expression cassettes this resulted in improved resistance to yield loss due to glyphosate herbicide application.     The pMON25496 construct provides two plant expression cassettes at a single location in nk603 that confers a higher lever of glyphosate tolerance than the triple tandem insertion occurring in the commercial standard, GA21.<br><br><br><br>
EXAMPLE 3<br>
The corresponding flanking DNA molecule from nk603 was cloned by using ligated adapters and nested PCR as described in the Genome Walker™ kit (catalog # K1807-1, CloneTech Laboratories, Inc, Palo Alto, CA).   First, genomic DNA from the nk603 event was purified by the CTAB purification method (Rogers et al.. Plant Mol. Biol. 5:69-76, 1985).  The genomic DNA libraries for amplification were prepared according to manufacturer's instructions (Genome Walker^*^, CloneTech Laboratories, Inc, Palo Alto, CA).    In separate reactions, genomic DNA was digested overnight at  37°C with the following blunt-end restriction endonucleases:   EcoRV, Seal, Dral, Pvull, and Stul (CloneTech Laboratories, Inc, Palo Alto, CA). The reaction mixtures were extracted with phenolxhloroform, the DNA was precipitated by the addition of ethanol to the aqueous phase, pelleted by centrifugation, then resuspended in Tris-EDTA buffer (lOmM Tris-.HCl, pH 8.0, ImM EDTA).  The purified blunt-ended genomic DNA fragments were ligated to the Genome Walker^" adapters according to the manufacturer's protocol. After ligation, each reaction was heat treated (70°C for 5 min) to terminate the reaction and then diluted 10-fold in Tris-EDTA buffer.   One 1   of each respective ligation was then amplified in a 50 ]x\ reaction that included 1 fxl of respective adapter-ligated library, 1 |il of 10 ^M Genome Walker™ adapter primer API (5'GTATATCGACTCACTATAGGGC 3', SEQ ID N0:1), 1 |il of 10 |iM nk603 transgene-specific oligonucleotide<br>
(5' TGACGTATCAAAGTACCGACAAAAACATCC 3' SEQ ID N0:2), 1 jil 10 mM deoxyribonucleotides, 2.5 ^1 dimethyl sulfoxide, 5 |il of lOX PCR buffer containing MgCh, 0.5 [i\ (2.5 units)of Amplitaq thermostable DNA polymerase (PE Applied Biosystems, Foster City, CA), and H2O to 50 \i\. The reactions were performed in a thermocycler using calculated temperature control and the following cycling conditions: 1 cycle of 95°C for 9 min; 7 cycles of 94°C for 2 s, 70°C for 3 min; 36 cycles of 94°C for 2 s, 65°C for 3 min; 1 cycle of 65°C for 4 min. One |il of each primary reaction was diluted 50-fold with water and amplified in a secondary reaction (1 \i\ of respective diluted primary reaction, 1 |il of 10 j^M Genome Walker^^ nested adapter primer AP2 (5'ACTATAGGGCACGCGTGGT 3', SEQ ID N0:3, supplied by manufacturer), 1 |il of 10 |aM nk603 transgene-specific nested oligonucleotide<br><br>
(5' CTTTGTTTTATTTTGGACTATCCCGACTC 3', SEQ ID N0:4), 1 |al 10 mM deoxyribonucleotides, 2.5 ^1 dimethyl sulfoxide, 5 |Lil of lOX PCR buffer containing MgCh, 0.5 fil (2.5 units)of Amplitaq thermostable DNA polymerase (PE Applied Biosystems, Foster City, CA), and H2O to 50 |il] using the following cycling conditions: 1 cycle of 95°C for 9 min; 5 5 cycles of 94°C for 2 s, 70°C for 3 min; 24 cycles of 94°C for 2 s, 65°C for 3 min; 1 cycle of 65°C for 4 min.<br>
PCR products, representing 5' regions that span the junction between the nk603<br>
transgenic insertion and the neighboring flanking com DNA were purified by agarose gel<br>
electrophoresis followed by purification from the agarose matrix using the QIAquick Gel<br>
10 Extraction Kit (catalog # 28704, Qiagen Inc., Valencia, CA) and direct cloning into the pGEM-T<br>
Easy vector (catalog. # A1360, Promega, Madison, WI). The identity of the cloned PCR<br>
products and relationship to the Mlu I fragment of pMON25496 was confirmed by DNA<br>
sequence analysis (ABI Prism^"^ 377, PE Biosystems, Foster City, CA and DNASTAR sequence<br>
analysis software, DNASTAR Inc., Madison, WI).<br>
15	Similarly, the nk603 3' flanking com DNA sequence was amplified and cloned using<br>
nested gene specific primers, such as, SEQ ID N0:5 (5' AGATTGAATCCTGTTGCCGGTCTTGC 3') and SEQ ID N0:6<br>
(5' GCGGTGTCATCTATGTTACTAGATCGGG 3') that anneal to the T-AGRTU.nos<br>
transcriptional terminator.   Two T-AGRTU.nos transcriptional terminators are present in the<br>
20    nk603 transgenic/genomic insertion, one internal in the construct and one at the 3' end of the<br>
construct adjacent to com genomic DNA.    The PCR products produced in this reaction were<br>
sequenced and the DNA sequence that spans the junction between transgene and flanking<br>
sequence was distinguished from products of the intemal T-AGRTU.nos by comparison to the<br>
known genetic element sequences of the pMON25496 constmct as previously described.<br>
25	Com DNA sequence flanking both sides of the transgenic insertion was determined for<br>
nk603 by sequencing the Genome Walker^^ -derived amplification products and alignment to<br>
known transgene sequence.  At the 5' end of the transgenic insertion, the sequence of a 498 bp<br>
segment around the insertion junction was determined (SEQ ID N0:7).  This consisted of 304<br>
base pairs (bp) of the flanking maize genomic DNA sequence (nucleotides 1-304 of SEQ ID<br>
30    N0:7), 45 bp of pMON25496 constmct DNA sequence (nucleotides 305-349 of SEQ ID N0:7)<br><br>
and 149 bp of DNA sequence from the 5' end of P-Os.Actl (nucleotides 350-498 of SEQ ID N0:7).<br>
The DNA sequence was determined for a 1183 bp segment around the 3' insertion junction (SEQ ID N0:8), that begins with 164 bp of the T-AGRTU.nos transcriptional 5 terminator (nucleotides 1-164 of SEQ ID N0:8), 217 bp of pMON25496 construct DNA sequence (nucleotides 165-381 of SEQ ID N0:8), 305 bp of the maize plastid genes, rpsU and rpoA (partial segments of each gene corresponding to bases 63-363 of Genbank accession X07810, corresponding to bases 382-686 of SEQ ID N0:8), and the remaining DNA sequence consisting of maize genomic DNA sequence flanking the integration site (corresponding to bases<br>
10    687-1183 of SEQ ID N0:8).<br>
The junction DNA molecules, SEQ ID N0:9, SEQ ID NO: 10, SEQ ID N0:11, and SEQ ID NO: 12 are novel DNA molecules in nk603 and are diagnostic for com plant nk603 and its progeny. The junction molecules in SEQ ID N0:9, SEQ ID NO:10, and SEQ ID N0:11 represent about 9 nucleotides on each side of an insertion site of a transgene DNA fragment and<br>
15 com genomic DNA. SEQ ID N0:9 is found at nucleotide positions 295-314 of SEQ ID N0:7. The junction sequences SEQ ID NO: 10 and 11 are located at nucleotide positions 373-390 and 678-695, respectively, of SEQ ID N0:8, representing a junction DNA molecule of construct DNA sequence with com plastid DNA sequence (SEQ ID NO: 10) and construct sequence with com genomic DNA sequence (SEQ ID N0:11). SEQ ID NO: 12 is located at nucleotide position<br>
20     156-173 of SEQ ID N0:8 and represents a novel DNA molecule in nk603 due to it being a fusion of the T-AGRTU.nos terminator sequence with an inverted fragment of the rice actin promoter DNA sequence. EXAMPLE 4<br>
DNA event primer pairs are used to produce an amplicon diagnostic for nk603.   These<br>
25 event primer pairs include but are not limited to SEQ ID NO: 13 and SEQ ID NO: 14 for the 5* amplicon DNA molecule and SEQ ID NO: 15 and SEQ ID NO: 16 for the 3' amplicon DNA molecule. In addition to these primer pairs, any primer pair derived from SEQ ID N0:7 and SEQ ID N0:8 that when used in a DNA amplification reaction produces a DNA amplicon diagnostic for nk603 is an aspect of the present invention. The amplification conditions for this<br>
30    analysis is illustrated in Table 2 and Table 3 for the 5' transgene insert/genomic junction region.<br><br>
The same method is applied for amplification of the 3' amplicon DNA molecule using primer<br>
DNA molecules SEQ ID NO: 15 and SEQ ID NO: 16, however, any modification of these<br>
methods that use DNA molecules or complements thereof to produce an amplicon DNA<br>
molecule diagnostic for nk603 is within the ordinary skill of the art. In addition, a control primer<br>
pair (SEQ ID NO: 17 and 18) for amplification of an endogenous com gene is included as an<br>
internal standard for the reaction conditions.   The analysis of nk603 plant tissue DNA extract<br>
sample should include a positive tissue DNA extract control from nk603, a negative DNA extract<br>
control from a com plant that is not nk603, and a negative control that contains no template com<br>
DNA extract. Additional DNA primer molecules of sufficient length can be selected from SEQ<br>
ID N0:7 and SEQ ID N0:8 by those skilled in the art of DNA amplification methods, and<br>
conditions optimized for the production of an amplicon that may differ from the methods shown<br>
in Table 2 and Table 3 but result in an amplicon diagnostic for nk603.  The use of these DNA<br>
primer sequences with modifications to the methods of Table 2 and 3 are within the scope of the<br>
invention. The amplicon wherein at least one DNA primer molecule of sufficient length derived<br>
fi-om SEQ ID N0:7 and SEQ ID N0:8 that is diagnostic for nk603 is an aspect of the invention.<br>
The amplicon wherein at least one DNA primer of sufficient length derived from any of the<br>
genetic elements of pMON25496 that is diagnostic for nk603 is an aspect of the invention. The<br>
assay for the nk603 amplicon can be performed by using a Stratagene Robocycler, MJ Engine,<br>
Perkin-Elmer 9700, or Eppendorf Mastercycler Gradient thermocycler as shown in Table 3, or<br>
by methods and apparatus known to those skilled in the art.<br>
Table 2. PCR procedure and reaction mixture for the confirmation of nk603 5' transgene insert/genomic junction region.<br><br><br><br><br><br>
A deposit of the Monsanto Company, com seed of event PV-ZMGT32(nk603) disclosed above has been made under the Budapest Treaty with the American Type Culture Collection 5 (ATCC), 10801 University Boulevard, Manassas, Va. 20110. The ATCC accession number is PTA-2478. The deposit will be maintained in the depository for a period of 30 years, or 5 years after the last request, or for the effective life of the patent, whichever is longer, and will be replaced as necessary during that period.<br>
Having illustrated and described the principles of the present invention, it should be 10    apparent to persons skilled in the art that the invention can be modified in arrangement and detail without departing from such principles. We claim all modifications that are within the spirit and scope of the appended claims.<br>
All   publications  and  published  patent  documents  cited   in  this  specification  are incorporated herein by reference to the same extent as if each individual publication or patent 15    application was specifically and individually indicated to be incorporated by reference.<br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A method for preparing a DNA construct comprising a first and a second<br>
expression cassette that when expressed in a monocot crop plant confers<br>
tolenrance to glyphosate comprising:<br>
(a)	preparing a first cassette in a known manner wherein said first expression cassette in operable linkage comprises (i) a rice actin 1 promoter; (ii) a rice actin 1 intron; (iii) a chloroplast transit peptide encoding DNA molecule; (iv) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (v) a transcriptional terminator DNA molecule;<br>
(b)	preparing a second cassette in a known manner wherein said second expression cassette in operable linkage comprises (a) a cauliflower mosaic virus (CaMV) 35S promoter; (b) a Hsp70 intron; (c) a chloroplast transit peptide encoding DNA molecule; (d) a glyphosate tolerant 5-enol-pyruvylshikimate-3-phosphate synthase (EPSPS) encoding DNA molecule; and (e) a transcriptional terminator DNA molecule; and<br>
(c)	operably joining the first and second cassettes in a known manner.<br><br>
2.	The method as claimed in claim 1, wherein the glyphosate tolerant EPSPS encoding DNA molecule comprises an Agrobacterium sp. Strain CP4 (AGRTU.aroA:CP4) DNA molecule.<br>
3.	The method as claimed in claim 1, wherein the monocot crop plant is a com plant.<br>
4.	The method as claimed in claim 1, wherein the DNA construct is inserted into a replicable plasmid.<br>
X\ b OCT 1003</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0477-mas-2001 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0477-mas-2001 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">0477-mas-2001 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpICBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 description (complete)  duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">0477-mas-2001 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBkcmF3aW5ncyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ3Ny1tYXMtMjAwMSBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">0477-mas-2001 form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="194263-a-method-of-producing-a-hot-rolled-stainless-steel-strip.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="194266-a-cleaning-method-for-a-sugar-cane-harvesting-machine-and-a-sugar-cane-harvesting-machine.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>194265</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>477/MAS/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>20/2006</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-May-2006</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Dec-2005</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Jun-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY LIC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERG,BOULEVARD ST.LOUIS, MISSOURI 63167</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CARL F BEHR</td>
											<td>800 NORTH LINDBERG,BOULEVARD ST.LOUIS, MISSOURI 63167</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/213,567</td>
									<td>2000-06-22</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/194265-a-method-for-preparing-a-dna-construct-with-glyphosate-tolerance by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:47:49 GMT -->
</html>
