<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/194800-a-method-of-producing-an-enzyme-exhibiting-endoglucanase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:31:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 194800:A METHOD OF PRODUCING AN ENZYME EXHIBITING ENDOGLUCANASE ACTIVITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PRODUCING AN ENZYME EXHIBITING ENDOGLUCANASE ACTIVITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT 738/MAS/96 &quot;A method of producing an enzyme exhibiting endoglucanase activity&quot; The present invention relates to a method of producing an enzyme exhibiting endoglucanase activity, the method comprising culturing a cell having (a) a DNA construct comprising the DNA sequence shown in SEQ IS No.8, or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisisae DSM 10081, or (b) a DNA construct comprising an analogue of the DNA sequence shown in SEQ ID No.8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, which DNA sequence has at least 75% identity with the DNA sequence shown in SEQ ID No.8 or the DNA sequence obtainable from the plasmid in Sachharomyes cerevisiae DSM 10081, under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>synthetic oligonucleotide primers using an Applied Biosystems 373A automated seqviencer according to the manufacturers instructions.  Analysis of the sequence data was performed according to Devereux et al.<br>
Isolation of a cDNA gene for expression in Aspergillus:<br>
An endoglucanase-producing yeast colony was inoculated into 20 ml YPD broth in a 50 ml glass test tube. The tube was shaken for 2 days at 30°C. The cells were harvested by centrifugation for 10 min. at 3000 rpm.<br>
DNA was isolated according to WO 94/14953 and dissolved in 50 111  water. The DNA was transformed into E.   coli  by standard procedures. Plasmid DNA was isolated from E. coli  using standard procedures, and analyzed by restriction enzyme analysis. The cDNA insert was excised using appropriate restriction enzvTues and ligated into an Aspergillus expression vector.<br>
Transformation of Aspergillus oryzae  or Aspergillus niger<br>
Protoplasts may be prepared as described in WO 95/02043, p. 16, line 21 - page 17, line 12, which is hereby incorporated by reference.<br>
100 fil  of protoplast suspension is mixed with 5-25 jxq  of the appropriate DNA in 10 ^1 of STC (1.2 M sorbitol, 10 mM Tris-HCl, pH = 7.5, 10 mM CaClj) . Protoplasts are mixed with p3SR2 (an A.   nidulans  amdS gene carrying plasmid). The mixture is left at room temperature for 25 minutes. 0.2 ml of 60% PEG 4000 (BDH 29576), 10 mM CaCl, and 10 mM Tris-HCl, pH 7.5 is added and carefully mixed (twice) and finally 0.85 ml of the same solution is added and carefully mixed. The mixture is left at room temperature for 25 minat,^s^ spun at 2500 g for 15 minutes and the pellet is resuspended in 2 ml of 1.2 M sorbitol. After one more sedimentation the protoplasts are spread<br><br>
on minimal plates (Cove, Biochem. Biophys. Acta 113 (1966) 51-56) containing 1.0 M sucrose, pH 7.0, 10 mM acetamide as nitrogen source and 2 0 mM CsCl to inhibit background growth. After incubation for 4-7 days at 37°C spores are picked and spread for single colonies. This procedure is repeated and spores of a single colony after the second reisolation is stored as a defined transformant.<br>
Test of A.   ojryzae  transformants<br>
Each of the transformants were inoculated in 10 ml YPM and propagated. After 2-5 days of incubation at 37°C, 10 ml supernatant was removed. The endoglucanase activity was identified by AZCL HE cellulose as described above.<br>
Hybridization conditions (to be used in evaluating property ii) of the DNA construct of the invention): Suitable conditions for determining hybridization between a nucleotide probe and a homologous DNA or RNA seguence involves presoaking of the filter containing the DNA fragments or RNA to hybridize in 5 x SSC (standard saline citrate) for 10 min, and prehybridization of the filter in a solution of 5 x SSC (Sambrook et al. 1989), 5 x Denhardt's solution (Sambrook et al. 1989), 0.5 % SDS and 100 /xg/ml of denatured sonicated salmon sperm DNA (Sambrook et al. 1989), followed by hybridization in the same solution containing a random-primed (Feinberg, A. P. and Vogelstein, B. (1983) Anal.   Biochem.   132:6-13), "p-dCTP-labeled (specific activity &gt; 1 x 10^ cpm//xg ) probe for 12 hours at ca. 45°C. The filter is then washed two times for 30 minutes in 2 x SSC, 0.5 % SDS at preferably not higher than 50°C, more preferably not higher than 55°C, more preferably not higher than 60°C, more preferably not higher than 65°C, even more preferably not higher^^EHan 70°e, especially not higher than 75°C. The nucleotide probe to be used in the hybridization is the DNA sequence corresponding to the endoglucanase<br><br>
encoding part of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, or 16, resepctively, and/or the DNA sequence obtainable from the plasmid in S.   cerevisiae, DSM 9770, DSM 10082, DSM 10080, DSM 10081, E.   coll,    DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
Immunological cross-reactivity: Antibodies to be used in determining immunological cross-reactivity may be prepared by use of a purified cellulase. More specifically, antiserum against the cellulase cf the invention may be raised by immunizing rabbits (or other rodents) according to the procedure described by N. Axelsen et al. in: A Manual of Quantitative Immunoelec¬trophoresis, Blackwell Scientific Publications, 1973, Chapter 23, or A. Johnstone and R. Thorpe, Immunochemistrv in Practice, Blackwell Scientific Publications, 1982 (more specifically pp. 27-31). Purified immunoglobulins may be obtained from the anti-sera, for example by salt precipitation ((NH4)2 SO4) , followed by dialysis and ion exchange chromatography, e.g. on DEAE-Sephadex. Imanunoc'hemical characterization of proteins may be done either by Outcherlony double-diffusion analysis (O. Ouchterlony in: Handbook of Experimental Immunology (D.M. VJeir, Ed.), Blackwell Scientific Publications, 1967, pp. 655-706), by crossed immunoelectrophoresis (N. Axelsen et al., supra. Chapters 3 and 4), or by rocket immunoelectrophoresis (N. Axelsen et al., Chapter 2).<br>
Media<br>
YPD: 10 g yeast extract, 20 g peptone, HjO to 900 ml. Autoclaved, 100 ml 20% glucose (sterile filtered) added.<br>
YPM: 10 g yeast extract, 20 g peptone, H2O to 900 ml. Autoclaved, 100 ml 2 0%--««ltodextrin (sterile filtered) added.<br><br>
10 X Basal salt: 75 g yeast nitrogen base, 113 g succinic acid, 68 g NaOH, Hp ad 1000 ml, sterile filtered.<br>
SC-URA: 100 ml 10 x Basal salt, 28 ml  20% casamino acids without vitamins, 10 ml 1% tryptophan, HjO ad 900 ml, autoclaved, 3.6 ml 5% threonine and 100 ml 20% glucose or 20% galactose added.<br>
SC-URA agar: SC-URA, 20 g/1 agar added.<br>
PD agar: 39g potato dextrose agar, DIFCO 0013; add deionized water up to 1000ml; autoclave (121°C for 15-20 min) .<br>
PC agar: Potatoes and carrots (grinded, 20 g of each) and water, added up to 1000ml, are boiled for 1 hr; agar (20g/l of Merck 1614);  autoclave (121°C for 20 min)<br>
PC liquid broth: as PC agar but without the Agar<br>
PD liquid broth: 24g potato dextrose broth, Difco 0549, deionized water up to 1000ml; autoclave (121'C for 15-20 min)<br>
PC and PD liquid broth with cellulose: add 30 g Solcafloc (Dicacel available from Dicalite-Europe-Nord, 9000 Gent, Belgium) per 1000ml<br>
PB-9 liquid broth: 12 g Rofec (Roquette 101-0441)  and 24 g glucose are added to 1000ml water; pH is adjusted to 5.5; 5inl mineral oil and 5 g CaCo3 are added per lOOOml. autoclave (121°C for 4 0 min)<br>
YFG  liquid broth: 4g yeast extract (Difco 0127), Ig KH2PO4 (Merck4873), 0. 5g MgS04.7H20 Merck 5886, 15g Dextrose, Roquette 101-0441, 0.1ml Pluronic (101-3088); deionized fiater  up to lOOOmI; autoclave (20inin at 121°C)<br>
"	r<br><br>
Dilute  salt  solution   (DS):   Make up     two  stock  solutions:<br>
P-stock:    13.61g     KH2PO4;    13.21g   (NH4)2P04,    17.4 2g  KH2PO4;<br>
deionized water up to 100ml<br>
Ca/Mg stock: 7.35g  CaCl,, 2H2O, lO.lVg MgClj, 6H2O,<br>
deionized water up to lOOml; pH adjusted to 7.0;<br>
autoclaving (121°C; 20min)<br>
Mix 0..5ml P-stock with 0.1ml Ca/Mg stock<br>
add deionized water up to 1000ml<br>
AZCL HE cellulose (Megazyme, Australia).<br>
Uses<br>
During washing and wearing, dyestuff from dyed fabrics or garment will conventionally bleed from the fabric which then looks faded and worn. Removal of surface fibers from the fabric will partly restore the original colours and looks of the fabric. By the term "colour clarification", as used herein, is meant the partly restoration of the initial colours of fabric or garment throughout multiple washing cycles.<br>
The term "de-pilling" denotes removing of pills from the fabric surface.<br>
The term "soaking liquor" denotes an aqueous liquor in which laundry may be immersed prior to being subjected to a conventional v/ashing prcccsc. The soaking liquoz- may contain one or more ingredients conventionally used in a washing or laundering process.<br>
The term "washing liquor" denotes an aqueous liquor in which laundry is subjected to a washing process, i.e. usually a combined chemical and mechanical action either manually or in a washing machine. Conventionally, the<br><br>
washing liquor is an aqueous solution of a powder or liquid detergent composition.<br>
The term "rinsing liquor" denotes an aqueous liquor in which laundry is immersed and treated, conventionally immediately after being subjected to a washing process, in order to rinse the laundry, i.e. essentially remove the detergent solution from the laundry. The rinsing liquor may contain a fabric conditioning or softening composition.<br>
The laundry subjected to the method of the present inven¬tion may be conventional washable laundry. Preferably, the major part of the laundry is sewn or unsewn fabrics, including knits, wovens, denims, yarns, and toweling, made from cotton, cotton blends or natural or manmade cellulosics (e.g. originating from xylan-containing cel¬lulose fibers such as from wood pulp) or blends thereof. Examples of blends are blends of cotton or rayon/viscose with one or more companion material such as wool, syn¬thetic fibers (e.g. polyamide fibers, acrylic fibers, polyester fibers, polyvinyl alcohol fibers, polyvinyl chloride fibers, polyvinylidene chloride fibers, polyurethane fibers, polyurea fibers, aramid fibers), and cellulose-containing fibers (e.g. rayon/viscose, ramie, flax/linen, jute, cellulose acetate fibers, lyocell).<br>
Detergent Compositions<br>
According to one aspect of the present invention, the present endoglucanases may typically be components of a detergent composition. As such, they may be included in the detergent composition in the form of a non-dusting granulate, a stabilized liquid, or p^^otected enzymes. Non-dusting granulates may be produced, e.g., as dis¬closed in US 4,106,991 and 4,661,452 (both to Novo<br><br>
Industri A/S) and may optionally be coated by methods known in the art. Examples of waxy coating materials are poly(ethylene oxide) products (polyethyleneglycol, PEG) with mean molecular weights of 1000 to 20000; ethoxylated nonylphenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in patent GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to established methods. Other enzyme stabilizers are well known in the art. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any convenient form, e.g. as powder, granules, iDaste or liquid. A liquid detergent may be aqueous, typically con¬taining up to 70% water and 0-30% organic solvent, or nonaqueous.<br>
The detergent composition comprises one or more surf¬actants, each of which may be anionic, nonionic, cationic, or zwitterionic. The detergent will usually contain 0-50% of anionic surfactant such as linear alkylbenzenesulfonate (LAS), alpha-olefinsulfonate (AOS), alkyl sulfate (fatty alcohol sulfate) (AS), alcohol ethoxysulfate (AEOS or AES), secondary alkanesulfonates (SAS), alpha-sulfo fatty acid methyl esters, alkyl- or alkenylsuccinic acid, or soap. It may also contain 0-40% of nonionic surfactant such—as—alcohol ethoxylate (AEO or AE) , carboxylated alcohol ethoxylates, n.onylphenol ethoxylate, alkylpolyglycoside, alkyldimethylamine oxide,<br><br>
ethoxylated fatty acid monoethanolamide, fatty acid monoethanolamide, or polyhydroxy alkyl fatty acid amide (e.g. as described in WO 92/06154).<br>
5  The detergent composition may additionally comprise one or more other enzymes such as amylase, lipase, cutinase, protease, peroxidase, and oxidase, e.g. laccase.<br>
The detergent may contain 1-65% of a detergent builder or )  complexing agent such as zeolite, diphosphate,<br>
triphosphate, phosphonate, citrate, nitrilotriacetic acid (NTA), ethylenediaminetetraacetic acid (EDTA), diethylenetriaminepentaacetic acid (DTMPA), alkyl- or alkenylsuccinic acid, soluble silicates or layered 1  silicates (e.g. SKS-6 from Hoechst). The detergent may also be unbuilt, i.e. essentially free of detergent builder.<br>
The detergent may comprise one or more polymers. Examples<br>
are carboxymethylcellulose (CMC), poly(vinylpyrrolidone) (PVP), polyethyleneglycol (PEG), poly(vinyl alcohol) (PVA), polycarboxylates such as polyacrylates,<br>
maleic/acrylic acid copolymers and lauryl<br>
methacrylate/acrylic acid copolymers.*<br>
The detergent may contain a bleaching system which may comprise a HjOj source such as perborate or percarbonate which may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine (TAED) or nonanoyloxybenzenesulfonate (NOBS). Alternatively, the bleaching system may comprise peroxyacids of, e.g.&gt; the amide, imide, or sulfone type.<br>
The enzymes of the detergent composition of the invention may be stabilized using conventional sta-biiizing agents, e.g. a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a<br><br>
boric acid derivative such as, e.g., an aromatic borate ester, and the composition may be formulated as described in, e.g., WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional deter¬gent ingredients such as, e.g., fabric conditioners in¬cluding clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil-redeposition agents, dyes, bactericides, optical brighteners, or perfume.<br>
The pH (measured in aqueous solution at use concentra¬tion) will usually be neutral or alkaline, e.g. in the range of 7-11.<br>
Particular forms of detergent compositions within the scope of the invention include:<br>
1)  A detergent composition formulated as a granulate having a bulk density of at least 600 g/1 comprising<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
12)	A detergent composition formulated as a granulate<br>
having a bulk density of at least 600 g/1 comprising<br><br>
13)	Detergent formulations as described in 1) - 12)<br>
wherein all or part of the linear alkylbenzenesulfonate<br>
is replaced by (C|2-C,8) alkyl sulfate.<br><br>
14)  A detergent composition formulated as a granulate having a bulk density of at least 600 g/1 comprising<br><br><br>
15)	A detergent composition formulated as a granulate<br>
having a bulk density of at least 600 g/1 comprising<br><br>
16)	Detergent formulations as described in 1) - 15) which<br>
contain a stabilized or encapsulated peracid, either as<br>
an additional component or as a substitute for already-<br>
specified bleach systems.<br>
17)	Detergent compositions as described in 1), 3), 7), 9) and 12) wherein perborate is replaced by percarbonate.<br>
18)	Detergent compositions as described in 1), 3), 7), 9), 12), 14) and 15) which additionally contain a manganese catalyst. The manganese catalyst may, e.g., be one of the compounds described in "Efficient manganese catalysts for low-temperature bleaching", Nature 369. 1994, pp. 637-639.      	^<br>
19)	Detergent composition formulated as a nonaqueous<br><br>
detergent liquid comprising a liquid nonionic surfactant such as, e.g., linear alkoxylated primary alcohol, a builder system (e.g. phosphate), enzyme and alkali. The detergent may also comprise anionic surfactant and/or a bleach system.<br>
The endoglucanase may be incorporated in concentrations conventionally employed in detergents. It is at present contemplated that, in the laundry composition of the in¬vention, the cellulase may be added in an amount corre¬sponding to 0.0001-10 mg (calculated as pure enzyme protein) of cellulase per liter of wash liquor.<br>
According to yet another aspect of the present invention, endoglucanase may typically be a component of a fabric conditioning or softener composition. Examples of conven¬tional softener compositions are disclosed in e.g. EP 0 233 910.<br>
Textile applications<br>
In another embodiment, the present invention relates to use of the endoglucanase of the invention in the bio-polishing process. Bio-Polishing is a specific treatment of the yarn surface which improves fabric quality with respect to handle and appearance without loss of fabric wettability.  The most important effects of Bio-Polishing can be characterized by less fuzz and pilling, increased gloss/luster, improved fabric handle, increased durable softness and altered water absorbency. Bio-Polishing usually takes place in the wet processing of the manufacture of knitted and woven fabrics.  Wet processing comprises such steps as e.g. desizing, scouring, bleaching, washing, dying/printing and finishi-Bg^,— During each of these steps, the fabric is more or less subjected to mechanical action. In general, after the textiles have<br><br>
been knitted or woven, the fabric proceeds to a desizing stage, followed by a scouring stage, etc.  Desizing is the act of removing size from textiles.  Prior to weaving on mechanical looms, warp yarns are often coated with size starch or starch derivatives in order to increase their tensile strength.  After weaving, the size coating must be removed before further processing the fabric in order to ensure a homogeneous and wash-proof result. It is known that in order to achieve the effects of Bio-Polishing, a combination of cellulytic and mechanical action is required.  It is also known that "super-softness" is achievable when the treatment with a cellulase is combined with a conventional treatment with softening agents. It is contemplated that use of the endoglucanase of the invention for bio-polishing of cellulosic fabrics is advantageous, e.g. a more thorough polishing can be achieved. Bio-polishing may be obtained by applying the method described e.g. in WO 93/20278.<br>
Stone-washing<br>
It is known to provide a "stone-washed" look (localized abrasion of the colour) in dyed fabric, especially in denim fabric or jeans, either by washing the denim or jeans made from such fabric in the presence of pumice stones to provide the desired localized lightening of the colour of the fabric or by treating the fabric enzymatically, in particular with cellulytic enzymes. The treatment with an endoglucanase of the present invention may be carried out either alone such as disclosed in US 4,832,864, together with a smaller amount of pumice than required in the traditional process, or together with perlite such as disclosed in WO 95/09225.<br><br>
Pulp and paper applications<br>
In the papermaking pulp industry, the endoglucanase of the present invention may be applied advantageously e.g. as follows:<br>
-	For debarking: pretreatment with the endoglucanase may degrade the cambium layer prior to debarking in mechanical drums resulting in advantageous energy savings.<br>
-	For defibration: treatment of a material containing cellulosic fibers with the endoglucanase prior to refin¬ing or beating may result in reduction of the energy consumption due to the hydrolysing effect of the cellulase on the interfibre surfaces. Use of the endoglucanase may result in improved energy savings as compared to the use of known enzymes, since it is believed that the enzyme composition of the invention may possess a higher ability to penetrate fibre walls.<br>
-	For fibre modification, i.e. improvement of fibre prop¬erties where partial hydrolysis across the fibre wall is needed which  requires deeper penetrating enzymes (e.g: in order to make coarse fibers more flexible). Deep treatment of fibers has so far not been possible for high yield pulps e.g. mechanical pulps or mixtures of recycled pulps. This has been ascribed to the nature of the fibre wall structure that prevents the passage of enzyme molecules due to physical restriction of the pore matrix of the fibre wall. It is contemplated that the present endoglucanase is capable of penetrating into the fibre wall.<br>
-	FoT drainage improvement. The drainability of papermaking pulps may be improved by treatment of the pulp with hydrolysing enzymes, e.g. cellulases. Use of<br><br>
the present endoglucanase may be more effective, e.g. result in a higher degree of loosening bundles of strongly hydrated micro-fibrils in the fines fraction (consisting of fibre debris) that limits the rate of drainage by blocking hollow spaces between fibers and in the wire mesh of the paper machine. The Canadian standard freeness (CSF) increases and the Schopper-Riegler drainage index decreases when pulp in subjected to cellulase treatment, see e.g. US patent 4,923,565; TAPPI T227, SCAN C19:65.ence.<br>
-	For inter fibre bonding. Hydrolytic enzymes are applied<br>
in the manufacture of papermaking pulps for improving the<br>
inter fibre bonding. The enzymes rinse the fibre surfaces<br>
for impurities e.g. cellulosic debris, thus enhancing the<br>
area of exposed cellulose with attachment to the fibre<br>
wall, thus improving the fibre-to-fibre hydrogen binding<br>
capacity. This process is also referred to as<br>
dehornification. Paper and board produced with a cellulase containing enzyme preparation may have an improved strength or- a reduced grammage, a smoother surface and an improved printability.<br>
-	For enzymatic deinking. Partial hydrolysis of recycled<br>
paper during or upon pulping by use of hydrolysing<br>
enzymes such as cellulases are known to facilitate the<br>
removal and agglomeration of ink particles. Use of the<br>
present endoglucanse may give a m.ore effective loosening<br>
of ink from the surface structure due to a better pen¬<br>
etration of the enzyme molecules into the fibrillar<br>
matrix of the fibre wall, thus softening: the surface<br>
whereby ink particles are effectively loosened. The<br>
agglomeration of loosened ink particles are also<br>
improved, due to a more efficient hydrolysis of cellulo¬<br>
sic fragments found attached to ink particles originating	<br>
from the fibres.<br><br>
The treatment of lignocellulosic pulp may, e.g., be performed as described in WO 91/14819, WO 91/14822, WO 92/17573 and WO 92/18688.<br>
Degradation of plant material<br>
In yet another embodiment, the present invention relates to use of the endoglucanase and/or enzyme preparation according to the invention for degradation of plant material e.g. cell walls.<br>
It is contemplated that the novel endoglucanase and/or enzyme preparation of the invention is useful in the pre¬paration of wine, fruit or vegetable juice in order to increase yield. Endoglucanases according to the invention m.ay also be applied for enzymatic hydrolysis of various plant cell-wall derived materials or waste materials, e.g. agricultural residues such as wheat-straw, corn cobs, whole corn plants, nut shells, grass, vegetable hulls, bean hulls, spent grains, sugar beet pulp, and the like. The plant material may be degraded in order to improve different kinds of processing, facilitate purification or extraction of other components like purification of beta-glucan or beta-glucan oligomers from cereals, improve the feed value, decrease the water binding capacity, improve the degradability in waste water plants, improve the conversion of e.g. grass and corn to ensilage, etc.<br>
The following examples illustrate the invention.<br><br>
EXAMPLE 1<br>
Cellulytic enzymes from 4 fungi, belonging to 3 families under two orders within the Ascomycetes were detected by expression cloning; corresponding DNA sequences were determined; the enzymes heterologously expressed, and produced by liquid fermentation, characterized and demonstrated to give good performance in colour clarifi¬cation assays.<br>
Isolate CBS 117.65, CBS 478.94, NRRL 8126, and ATCC 10523 were grown in shake flask cultures on cellulose enriched potato dextrose broth, incubated for 5 days at 2 6'^C (shaking conditions, 150 rpm).<br>
A. Cloning and expression of an endoglucanase from Myceliophthora thermophila,   Acremonium sp.,   and Thlelavia tenrestris  and Volartella colletotTichoides<br>
mRNA was isolated from Myceliophthora  thermophila, Acremonium sp.,   Tbielavia  terrestris  and Volutella colletotrichoides,   respectively, grown in a cellulose-containing fermentation medium with agitation to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C. Libraries from Myceliophthora thermophila,   Acremonium  sp.,   Thielavia   terrestris   and Volutella  colletotrichoides,   respectively, each consisting of approx. 10* individual clones were constructed in E.   coli   as described with a vector background of 1%.<br>
Plasmid DNA from some of the pools from each library was transformed into yeast, and 50-100 plates containing 250-4 00 yeast colonies were obtained from each^o«k.l.<br>
Endoglucanase-positive colonies were identified and<br><br>
isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Myceliophthora   thermophila   is shown in SEQ ID No. 1 and the corresponding amino acid sequence is also shown in SEQ ID No. 1. The cDNA is obtainable from the plasmid in DSM 9770.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Acremonium  sp.   is shown in SEQ ID No. 4 and the corresponding amino acid sequence is shown in SEQ ID No. 5. The cDNA is obtainable from the plasmid in DSM 10082.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Thielavia  terrestris   is shown in SEQ ID No. 8 and the corresponding amino acid sequence is shown in SEQ ID No. 9. The cDNA is obtainable from the plasmid in DSM 10081.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Volutella  colletotrichoides   is shown in SEQ ID No. 16 and the corresponding amino acid sequence is shown in SEQ ID No. 17. The cDNA is obtainable from the plasmid in DSM 10571.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli   as described above. In order to express the endoglucanases in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene from Myceliophthora  thermophila,   Acremonium sp.,   Thielavia terrestris  and Volutella  colletotrichoides,   respectively, was purified. The genes were subsequently ligated to<br><br>
pHD414, digested with appropriate restriction enzymes, resulting in the plasmids pA2C193, pA2C357, pA2C385 and pA2C488, respectively.<br>
After amplification of the DNA in E.   coll  the plasmids were transformed into Aspergillus  oryzae  as described above.<br>
Test of A.   oryzae  transformants<br>
Each of the transformants were tested for endoglucanase activity as described above. Some of the transformants had endoglucanase activity which was significantly larger than the Aspergillus oryzae  background. This demonstrates efficient expression of the endoglucanases in Aspergillus oryzae.   The transformants with the highest endoglucanase activity were selected and inoculated in a 500 m&gt;l shake flask with YPM media. After 3-5 days of fermentation with sufficient agitation to ensure good aeration, the culture broth was centrifuged for 10 minutes at 2 000 g and the supernatant recovered.<br>
B. Determination of endoglucanase activity<br>
The cellulytic activity of the endoglucanase may be determined relative to an analytical standard and expressed in the unit S-CEVU.<br>
Cellulytic enzym.es hydrolyse CMC, thereby decreasing the viscosity of the incubation mixture. The resulting reduction in viscosity may be determined by a vibration viscosimeter (e.g. MIVI 3000 from Sofraser, France).<br>
Determination of the cellulytic activity, measured in term.s of S-CEVU, may be determined according to the analysis methQd_AF 3 01.1 which is available from the Applicant upon request.<br><br>
The S-CEVU assay quantifies the amount of catalytic activity present in the sample by measuring the ability of the sample to reduce the viscosity of a solution of carboxy-methylcellulose (CMC). The assay is carried out at 40°C, pH 7.5 using a relative enzyme standard for reducing the viscosity of the CMC substrate.<br>
Assay for determination of endoglucanase activity in terms of SAVI units using phosphoric-acid swollen cellu-1  lose (PASC):<br>
Definition:<br>
1	SAVI-U is the amount of enzyme which forms an amount of<br>
reducing carbohydrates equivalent to 1 /ixmol of glucose<br>
per minute.<br>
Assay condition:<br>
Enzyme solution: 0,5 m.l<br>
4 g/1 PASC in 0,1 M Buffer: 2.0 ml<br>
2	0 min, 4 0 °C<br>
Sensitivity:<br>
Max 0.1 SAVlU/ml =  approx. 1 S-CEVU/ml (CMC viscosity)<br>
Min 0.01 SAVIU/ml = approx. 0.1 S-CEVU/ml<br>
Determination of formation of reducing sugars:<br>
The reducing groups assay was performed according to Lever, M. A new reaction for colormetric determination of carbohydrates. Anal- Biochem. 1972. Vol 47 (273-279). Reagent mixture was  prepared by mixing 1,5 gram p-hydroxybenzoic-acide hydracide (PHBAH) with 5 gram sodium tartrate in 100 ml 2 % sodium hydroxide.<br>
Substrate:<br>
PASC stock solution was prepared the following way using.<br><br>
ice cold acetone and phosphoric acid. 5 gram of cellulose (Avicel*) was moistered with water, and 150 ml ice cold 85% ortho-phosphoric acid was added. The mixture was placed in ice bath under slow stirring for 1 hr. Then 100 ml ice cold acetone was added with stirring. The slurry was transferred to a Buchner filter with pyrex sintered disc number 3 and then washed three times with 100 ml ice cold acetone, and sucked as dry as possible after each wash. Finally, the filter cake was washed twice with 500 ml water, sucked as dry as possible after each wash. The PASC was mixed with deionized water to a total volume of 300 ml, blended to homogeneity (using the Ultra Turrax Homogenizer) and stored in refrigerator (up to one month).<br>
Substrate equilibration with buffer: 2 0 gram phosphoric acid swollen cellulose PASC stock solution was centrifu-ged for 2 0 min at 5000 rpm., the supernatani: was poured of; the sediment was resuspended in 3 0 ml of buffer and centrifuged for 20 min. at 5000 rpm., the supernatant was poured of, and the sediment was resuspended in tuffer to a total of 60 g corresponding to a substrate concentra¬tion of 5 g cellulose/litre.<br>
Buffer for pH 8,5 determination: 0.1 M Barbital. Buffer for pH 10 determination: 0.1 M Glycine.<br>
Procedure:<br>
1.	Dilution of enzyme samples<br>
The enzyme solution is diluted in the same buffer as the substrate.<br>
2.	Enzyme reaction<br>
The substrate in buffer solution is preheated for 5 min.<br>
at 40°C  (2 ml).		.<br>
Then the enzyme solution (diluted to between 0.2 and 1 S-CEVU/ml) 0,5 ml is added and mixed for 5 sec. Enzymes<br><br>
blanks are obtained by adding the stop reagent before enzyme solution. Incubate for 2 0 min. at 4 0 °C. The reaction is stopped by adding 0.5 ml 2% NaOH solution and mixing for 5 sec.<br>
The samples are centrifuged for 20 min. at 5000 rpm. 1 ml supernatant is mixed with 0.5 ml PHBAH reagent and boiled for 10 min. The test tubes are cooled in a ice water bath.<br>
3.	Determination of reducing end groups:<br>
The absorbancy at 410 nm is measured using a spectrophotometer. Blanks are prepared by adding sodium hydroxide before adding enzynne solution. A standard glucose curve was obtained by using glucose concentrations of 5, 10, 15 and 25 mg/1 in the same buffer and adding PHBAH reagent before boiling. The release of reducing glucose equivalent is calculated using this standard curve.<br>
4.	Calculation of catalytic activity:<br>
Measure absdrbance at 410 nm<br>
1)	Standard curve<br>
(Glucose)  -  (H2O)   vs   concentration of glucose<br>
2)	Enzyme sample<br>
(Sample) - (Blank)<br>
Calculate glucose concentration according to a standard curve<br><br>
Activity (SAVIU/ml):<br><br>
C. Purification and characterisation of the endoglucanase from M.   thermophila<br>
Aspergillus  oryzae  transformed with pA2C193 was grown on YPM medium for 4 days. The liquid was then centrifuged and sterile filtered.<br>
The sample was concentrated by ultrafiltration on AMICON cells using a DOW membrane GR51PP with cut-off 20 kD. The Uf-concentrate was analyzed for S-CEVU/ml and SaviU/ml with the following result:<br><br>
Purification:<br>
2 ml of the UF-concentrate was diluted 5 times to lower the ionic strength and filtered through 0.22 iim  disk filter. This sample was applied to a Mono Q* HR5/5 Phar¬macia column, equilibrated with 50 mM Tris/HCl buffer, pH 7.5, (buffer A) and a flow of 1 ml/min. After wash to baseline, with buffer A, the column was eluted with a Tris/HCl buffer, pH 7.5, containing 1 M NaCl (buffer B), the elution gradient was 0-50% buffer B in 1 hour.<br>
After 3 6 min. a peak complex showed up, 1 ml fractions were picked up and the first 10 fractions showed cellulase activity on CMC/Agarose/congo-red plates.<br><br>
These fractions were pooled and concentrated, by ultrafiltration on AMICON cells using a DOW membrane GR61PP with cut-off 20 kD, to 3 ml.<br>
This sample was applied to a HiLoad 26/60 Superdex 75™ prep grade Pharmacia column, equilibrated with 100 mM Na-Acetate buffer, pH 6.35, and a 1 ml/min flow.<br>
After 82 rain, a peak showed up, 1 ml fractions were picked up and the first 10 fractions showed cellulase activity on CMC/Agarose/congo-red plates.<br>
These fractions were pooled and the following results were obtained:<br>
A2go=0.15<br>
^280/^260"=! • 62<br>
Mw(SDS)=22 kD<br>
pl=3.5 - 5<br>
Purity on SDS-PAGE =100%<br>
S-CEVU/ml=28.5<br>
S-CEVU/A28o=188<br>
S-CEVU/mg=4 3 6<br>
Extinction coefficient=54880 (calculated)<br>
Mw(calculated)=22 kD<br>
The Extinction coefficient is based on the content of tyrosine, tryptophane and cystein calculated from the sequence of the enclosed SEQ ID No. 1 (the amino acid sequence). SDS-Page was performed on NOVEX Pre-Cast Gels 4-20% Tris-Glycine Gel 1.0 mm x 10 Well<br>
lEF was performed on Pharmacia PAGplate pH 3.5 - 9.5, the activity was visualized by CMC-Congored overlaying.<br>
Determinat ion, of K^, &amp; k„,:<br>
k„ and k^^t was determined in the same manner as the<br>
determination of SAVI Units at pH 8.5 with a substrate<br><br>
concentration up to 8 g/1.<br>
The following results were obtained:<br>
k„, 38 per sec.<br>
K  5 g/1,<br>
phosporic acid swollen cellulose, pH 8.5.<br>
Specific activity on CMC at pH 7.5: 436 S-CEVU per mg protein.<br>
D. Determination of pH and temperature profile of the endoglucanase from M. thermophila<br>
The pH profile was determined at the following conditions:<br>
Buffers of pH values between 2.5 and 10.0 were made by mixing O.IM Tri-sodium phosphate with O.lM citric acid. Purified endoglucanase was diluted to ensure the assay response to be within the linear range of the assay. The substrate was a 0.4% suspension of AZCL-HE-cellulose (MegaZyme) mixed 1:1 with the citrate/phosphate buffer to a final substrate concentration of 0.2% AZCL-HE-cellulose. 1 ml substrate in Eppendorf® 1.5ml polypropylene tubes were added 10 ^1 of enzyme solution and incubated for 15 minutes in Eppendorf® temperature controlled Thermomixers before heat-inactivation of enzymes for 2 0 minutes at 95°C in a separate Thermomixer. The tubes were cenrrituged and 2 00 ^1;of each supernatant was transferred to a well in a 96 well microtiter plate and OD was measured at 620nm in an ELISA reader (Labsystems Multiskan® MCC/340).<br>
For the pH optimum incubations took place at 3 0°C. For each pH value, three tubes were added enzyme and<br><br>
incubated before heat-inactivation, whereas one tube (the blank) was added enzyme and heat-inactivated immediately. The mean value of the three incubated samples was calculated and the blank value was substracted.<br>
The following pH profile was determined:<br><br>
It is seen that the endoglucanase has more than 60% activity between pH 4.0 and 8.0 and optimal activity at pH 5.0-6.0'.<br><br>
The temperature optimum was determined in the same manner at pH 5.5. The temperatures ranged from 30°C to 80°C. For each temperature three incubations were carried out and the mean calculated. Three blanks were produced by immediate heat-inactivation of enzyme and the mean was subtracted from the incubated sample values.<br>
It is seen that the endoglucanase has optimal activity at 50-70°C.<br><br>
The temperature stability was determined in the same manner at pH 5.5 and 30°C, and, further, the enzyme solu¬tions were preheated for 1 hour at the actual temperature and cooled on ice. The residual activity is shown below in % of the activity of a non-preheated enzyme sample:<br><br><br>
E. Color clarification of Myceliophthora cellulase (SEQ ID No. 1) measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br>
Apparatus      :    Terg-o-tometer<br>
Liquid volume  :    100 ml<br>
Agitation      :    150 movements/min with vertical<br>
stirrer<br>
Rinse time     :    5 min in tapwater<br>
Washing temp   :    40°<br>
Washing liqour :    0.05 M phosphate buffer<br>
pH	:    7.0<br>
Washing time   :    3 0 min<br>
Repetitions    :    2<br>
Enzymes        :    Myceliophthora SEQ ID No. IB<br>
Dosage	:    500 and 2500 S-CEVU/1<br>
Textile        :    2 swatches of aged black 100% cotton<br>
5x6 cm (0.9 gram)<br>
Drying        :    Tumble dry<br>
Evaluation     :    The light remission is measured by a<br>
Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values) . When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br>
The sample is; compared with a blind sample, i.e. washed without enzyme:<br>
No cellulase   500 ECU/1 2500 ECU/1<br>
0.00	-1.41"    -1.91<br>
Delta L-values compared to blind sample.<br><br>
The data shows that Myceliophthora  cellulase without CBD gives good color clarification under the conditions tested.<br>
F. Construction of the gene fusions between the endoglucanase from Myceliophthora thermophila and the 43]cD endoglucanase from Humicola insolens<br>
The purpose of the two constructions was to make derivatives of the endoglucanase from M. thermophila with the linker and CBD from the 43kD endoglucanase from H. insolens   (disclosed in WO 91/17243). The native endoglucanase from M.   thermophila  do not have a linker and/or a cellulose binding domain, CBD.<br>
CMl: Construction 1 consist of the endoglucanase from M.thennophila (2 25 amino acids) and the 7 2 C-terminal amino acids from the H. insolens 43kD endoglucanase.<br>
CM2: Construction 2 consist of the endoglucanase from M.thermophila (225 aiaino acids) and the 83 C-terminal amino acids from the H. insolens 43kD endoglucanase.<br>
The 43kD endoglucanase cDNA from H. insolens was cloned into pHD414 in such a way that the endoglucanase gene was transcribed from the Taka-promoter. The resulting plasmid was named pCaHj418.<br>
In a similar way the cDNA encoding the endoglucanase from M.   thermophila  was cloned into pHD414 and the resulting plasmid was named pA2C193.<br><br><br>
Construction 1:<br>
Reaction A: The Polymerase Chain Reaction (PCR) was used'<br>
to amplify the fragment of pCaHj418 between primer 1 and<br>
AMG-term. primer (the linker and CBD from the 43kD<br>
endoglucanase from H.insolens).<br>
Reaction B: PCR amplification of the fragment between<br>
Taka-pro. primer and primer 2 in pA2C193, the<br>
endoglucanase gene from M.thermophila.<br>
Reaction C: The two purified fragments were used in a<br>
third PCR in the presence of the primers flanking the<br>
total region, i.e. Taka-pro. primer and AMG-term. primer.<br>
CohstructTon 2:<br>
The same procedure was used where primer 3 and primer 4<br><br>
had replaced respectively primer 1 and primer 2.<br>
The fragment amplified in reaction C was purified, digested with restriction enzymes Xba I and BsstE II. The purified digested fragment was ligated into pA2C193 digested with restriction enzymes Xba I and BsstE II.<br>
Competent cells from E. coli strain DH5aF' (New England Biolabs.) were transformed with the ligated plasmid and colonies containing the gene fusion were isolated. The sequence of the cloned part was verified by DNA sequencing.<br>
The sequence of the gene in the two constructs are shown in SEQ ID No. 2A and SEQ ID No. 3A.<br>
Polymerase Chain Reactions were carried out under standard conditions, as recommended by Perkin-Elmer.<br>
Reaction A and B started with 2 min. at 94°C followed by 20"cycles of (30 sec. at 94°C, 30 sec. at 50°C and 1 min. at 72°C) and end with 4 min. at 72 °C.<br>
Reaction C started with (2 min. at 94°C, 1 min. at 52°C and 2 min. at 72°C), followed by 15 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 90 sec. at 72°C) and end with 4 min. at 72°C.<br>
The two constructs were transformed into Aspergillus ory-zae as described above.<br>
G. Purification and characterisation of cloned cellulases with cellulose binding domains:<br>
The cloned product is recovered after fermentation by ■ separation of the extracellular fluid from the production<br><br>
organism.<br>
About one gram of cellulase is then highly purified by affinity chromatography using 150 gram of Avicel in a slurry with 2 0 mm Sodium- phosphate pH 7.5.<br>
The Avicel is mixed with the crude fermentation broth which contain total about 1 gram of cellulase. After mixing at 4 C for 20 min the Avicel enzyme is packed into a column with a dimension of 50 times 200 mm about 400 ml total.<br>
The column is washed with the 2 00 ml buffer, then washed with 0.5 M NaCl in the same buffer until no more protein elutes. Then washed with 500 ml 20 mm Tris pH 8.5. Finally the pure full length enzyme is eluted with 1% triethylamine pH 11.8.<br>
The eluted enzyme solution is adjusted to pH 8 and concentrated using a Amicon cell unit with a membrane DOW GR61PP (polypropylene v/ith a cut off of 20 KD) to above 5 mg protein per ml. The purified cellulases were characterised as follow:<br>
Mw     pi     Molar E.28 0    S-CEVU per<br>
SDS-PAGE	A.280<br>
Myceliophthora<br>
(SEQ ID No.2)  43 kD     4    74.950	135<br>
Acremonium<br>
(SEQ ID No.5)  40 kD     5    68.020	185<br>
Thielavia<br>
(SEQ ID No.9)  35 kD     4.3  52.470	75<br><br>
pH       N-terminal	Termo-<br>
Activity	stability<br>
above 50%	DSC<br>
Myceliophthora<br>
(SEQ ID No.2)  5.0-9.0   Blocked.	80°C<br>
Acremonium<br>
(SEQ ID No.5)  6.0-9.5   Blocked.	61°C<br>
Thielavia<br>
(SEQ ID No.9)  5.0-9.0   ASGSG		83°C<br>
The purified cellulases was analysed for MW by SDS-PAGE and using standard LMW protein marker kit from Pharmacia the MVJ was calculated for the cellulases. The MW is apparently higher than the MW of the composition of the coding amin acids and is due to the fact the linker region are 0-glycosylated resulting in this higher MW. The pi was determined using a Pharmacia Ampholine PAG plates pH 3.5 to 9.5 and again using a Pharmacia kit with known pi proteins.<br>
The molar extinction coefficient was calculated based on the amin acids composition using the known absorbance of Tryptophan, Tyrosine and Cystein.<br>
pH activity profile was obtained using CMC substrate, incubation for 20 min at 40° C at a 0.5 pH interval and measuring the formation of reducing sugars. The relative activity at the different pH was calculated and the table contain the interval with more than 50% relative activity has been measured.<br>
The N-terminal was determined for the-pwrified cellulase using a Applied Biosystems model 473A sequencer. The protein sequenceer was run according to the manufacturer<br><br>
instructions.<br>
Two of the cellulases were blocked, this is due to the N-terminal glutamine which form a pyroglutamate which can not be detected and which block for further sequencing. DSC Differential scanning calometry was done at neutral pH (7.0) using a MicroCalc Inc. MC calorimeter with a constant scan rate and raising the temperature from 2 0 to 90° at a rate of 90° per hour.<br>
Raising antibody. The cellulases from Myceliophthora, Acremonium and Thielavia were used for raising antibody in rabits. 0.1 mg of the purified cellulase in 0.9 % NaCl solution mixed with Freunds adjuvant immediately prior to injection. The rabits were immunized 10 times with one week interval. The immunoglobulin G fraction (IgG) was purified by ammonium sulfate precipitation (25% saturation). the precipitate was solubilized in water and then dialyzed extensively against sodium acetate buffer (pH 5.0, 50 mM) altering with deionized water. After filtration, the IgG fraction was stabilized with sodium azide (0.01%) .<br>
Using immunodiffusion in agar plates all three cellulases form a single immunoprecipitate with its homologous antiserum and no precipitate was seen between the 3 cloned cellulases and the sera raised against the other two cellulases.<br><br>
H-I. Performance of endoglucanase of construction 1 (SEQ ID No. 2) measured in buffer as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br>
Apparatus	:	Terg-o-tometer<br>
Liquid volume	:	100 ml<br>
Agitation	:	150 movements/min (rpm)<br>
Rinse time	:	5 min in tap water<br>
Washing temp	:	40"C<br>
Water Hardness	:	1 mM CaClj<br>
Washing liquor	:	0.05 M phosphate buffer<br>
pH	:	7.0<br>
Washing time	:	30 min<br>
Repetitions	:	2<br>
Textile	:	2 swatches of aged black,<br>
100% cotton 5x6 cm<br>
Drying	:	Tumble dry<br>
Evaluation:<br>
The light remission was measured by a Macbeth Color Eye 7000 Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn were removed by the cellulase, the surface appeared more bright, and lower L values were obtained.<br><br>
The data show that the enzyme of the invention gives very good color clarification under the conditions tested.<br><br>
I-II. Performance of cloned endoglucanase from Thielavia terrestris   (SEQ ID No.9) in buffer measured as removal of surface fibrils and fibers protruding from the yarn of a :extile containing cellulosic fibers<br>
Apparatus     :    Terg-o-tometer<br>
jiquid volume  :    100 ml<br>
i.gitation      :    150 movements/min with vertical<br>
stirrer iinse time    :    10 min in tapwater ?ashing temp   :    40°<br>
?ashing liqour :    0.05 M phosphate buffer.<br>
)H	:    7.0<br>
?ashing time   :    30 min Lepetitions    :    2 'extile        :    2 swatches of aged black cotton 5x6<br>
cm (app. 150 g/m2) !rying        :    Tumble dry Ivaluation :<br>
'he light remission was measured by a Datacolor Elrepho [emission spectrophotometer. Remission is calculated as lelta L (Hunter Lab-values). When the surface fibrils and ibers protruding from the yarn are removed by the lellulase, the surface of the black fabric appears darker nd nicer, and lower L values are obtained.<br><br><br>
H-III. Performance of endoglucanase of Volutella colletrichoides   (SEQ ID No. 17) measured in buffer as removal of surface fibrils and fibers protruding from thi yarn of a textile containing cellulosic fibers<br>
Apparatus      :    Terg-o-tometer<br>
Liquid volume  :    100 ml<br>
Agitation      :    150 movements/min with vertical<br>
stirrer Rinse time     :    5 min in tapwater Washing temp   :    40°<br>
Washing liqour :    0.05 M phosphate buffer<br>
pH	:    7.0<br>
Washing time   :    30 min<br>
Repetitions    :    2<br>
Dosage	:    2.5 S-CEVU/ml<br>
Textile        :    2 swatches of aged black 100% cotton<br>
5x6 cm (0.9 gram) Drying        :    Tumble dry Evaluation:<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils anc fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br>
The sample is compared with a blind sample, i.e. washed without enzyme:<br>
No cellulase        With cellulase<br>
0.00	-0.57<br>
Delta L remission values compared to blind sample.<br>
The data 'shows that~ tire" Volutella  colletrichoides cellulase gives good color clarification under the conditions tested.<br><br>
H-IV. Performance of cloned cellulases from Thielavia terrestris  and Acremonlum sp.   CBS 478.94 in high pH heavy duty detergent measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br>
Apparatus      :    Terg-o-tometer<br>
Liquid volume  :    150 ml<br>
Agitation     :    150 movements/min with vertical<br>
stirrer Rinse time     :    10 min in tapwater Washing temp   :    35°C Washing liqour :    1.0 g/1 US type HDG<br>
(zeolite/soda built, anionic/nonionic<br>
weight ratio &gt; 2.5)<br>
pH	:    10.0<br>
Hardness       :    1.0 mM CaC12<br>
0.3 4 mM MgC12 Washing time   :    12 min Repetitions    :    6 Textile        :    2 swatches of aged black cotton 5x6<br>
cm (app. 150 g/m2)<br>
2 swatches of heavy knitted cotton<br>
5x6 cm   (app- 600 g/m2) Drying        :    Tumble dry Evaluation :<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker and nicer, and lower L values are obtained. Different dosages of cloned cellulases from Thielavia  terrestris (SEQ ID No. 9) and Acremonium  sp.   CBS 478.94 (SEQ ID No.5), respectively, (denoted A and B, respectively) were tested.<br><br><br>
The data show that both cellulases gives good color clarification under the conditions tested.<br>
H-V. Performance of cellulases cloned from Thielavia torrastris  and Acremonium sp.   CBS 4 78.94, and construction 1 (SEQ ID No.2) measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers<br>
Apparatus     :    Terg-o-tometer<br>
Liquid volume j    150 ml<br>
Agitation      :    150 movements/min with vertical<br>
stirrer<br>
Rinse time    :    10 min in tapwater<br>
Washing temp   :    35°C<br>
Hardness       :    1.0 mM CaCl2<br>
0.34 mM MgC12<br>
Washing liqour :    2.0 g/1 HDL (neutral, citrate built<br>
HDL, with nonionic/anionic weight ra¬tion &gt; 0.5)<br>
kjn	.       ,' . D<br>
Washing time   :    3 0 min<br>
Repetitions    :    2<br>
Textile        :    2 swatches of aged black cotton 5x6<br>
cm (app. 150 g/m2)<br>
2 swatches of heavy knitted cotton<br>
4x7 cm (app. 600 g/m2)<br><br>
Drying        :    Tumble dry Evaluation :<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (CIE Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker and nicer, and lower L values are obtained. Three different dosages Different dosages of cloned cellulases from Thielavia  terrestris   (SEQ ID No. 9) and Acremonium sp.   CBS 478.94 (SEQ ID No.5) and the construction 1 (SEQ ID No.2), respectively, (denoted A and B and C, respectively) were tested. Results:<br><br>
The data show that all cellulases gives very good color clarification under the conditions tested.<br>
I. Application of endoglucanases from Thielavia terre¬stris, Acremonium sp. and construction 1 (SEQ ID No. 2) in denim finishing<br>
Experimental<br>
Apparatus:	" Washing'machine Wascator FL 12 0<br>
Liquid volume:     20 L<br>
Fabric:	1.1 kg denim fabric, 14% oz 100 %<br><br>
cotton<br>
Desizing:	10 min, 55°C, pH 7<br>
50 ml Aquazyme 120L 2.5 g/1 Phosphate buffer<br>
Abrasion:	2 hours;<br>
pH and temperature varied according to the following table:<br>
Enzyme  Activity      pH/temp   Buffer system<br>
SEQ ID<br>
No. 2    1400 S-CEVU/g  6/55°C    2.5 g/1 phosphate<br>
buffer No. 9    292 S-CEVU/g   5/65°C    1 g/1 citrate buffer No. 5    782 S-CEVU/g   7/45°C    2.5 g/1 phosphate<br>
buffer<br>
Inactivation: 15 min, 80°C<br>
1 g/1 sodium carbonate Rinses:  Three rinse cycles of 5 min in cold tap water<br>
Evaluation:<br>
Abrasion:   The remission from the fabric was determined at<br>
420 nm using a Texflash 2000 as a measure of the abrasion<br>
level.<br>
The results from the treatment of the denim fabric with different endoglucanases of the invention is shown in the following table:<br><br><br>
All tested cellulases show excellent performance in denim finishing, although each enzyme is unique in its own way. When applying the enzyme corresponding to SEQ ID No. 2 for denim finishing it is possible to reach a high abrasion level with a minimum of strength loss. When treating denim with the enzyme corresponding to SEQ ilD No. 9, a very high wash down can be reached which leaves the fabric with an almost bleached appearance. Denim finishing with the enzyme corresponding to SEQ ID No. 5 gives a high abrasion level at a low temperature optimum which makes it possible to reduce the processing tempera¬ture and save energy.<br>
J. Use of cloned cellulases from Acremonium sp,   and Thie-lavia  terrestris  for Biopolishing of lyocell fibers<br>
Lyocell fibers which are sold under the trade name Tencel<br><br>
re spun from wood pulp cellulose in a more nvironmentally friendly waterbased solvent than is the ase for normal viscose production). However, the fibers ave a tendency to fibrillate when they are processed nto textiles which is seen on the surface and denoted fuzz". By using cellulases it is possible to permanently emove the exposed and fuzzy fibers and significantly mprove the look of the finished fabric, the treatment generally known^as Biopolishing. The endoglucanases of he present invention are especially suited for the removal of Lyocell surface fibers.<br>
[ATERIALS AND METHODS<br>
the textile substrate was either 100 % woven or different<br>
kinds of jersey knitted dark blue Tencel. The dark colour<br>
and jersey knit was preferred in order to enhance the<br>
visual effects which simplifed the evaluation. A woven<br>
0/30 Tencel/Rayon blend was also used to a lesser<br>
extent.<br>
the assays were either performed in 200 ml scale using a<br>
aunder-o-meter or in the 20 1 scale using a Wascator.<br>
?he treatment time was 60min at 55° C in Wascator and 60-<br>
30 min in LOM. The buffer was 2 g/1 sodium acetate<br>
adjusted to pH 5 with acetic acid. The fabric to liquid<br>
ratio was 1:10 but in the Launder-o-meter 20 steel balls<br>
with a diameter of 14 mm (11 g each) was used to obtain<br>
sufficient mechanical abrasion. The biopolishing was<br>
Immediately followed by inactivation using 2 g/lit sodium<br>
carbonate at 80° C for 15 min followed by rinsing in cold<br>
later.<br>
The results were evaluated using a fuzz note scale from 1<br>
 5 were 1 is the fibrillated look of the starting<br>
naterial and 5 is a high quality look with no visible<br>
uibers on the surface. Since the performance of<br>
andacellulases is specific towards a surface treatment<br>
the weightless is below 2 % and is therefore not included<br>
Ln the evaluation. Two cellulases were evaluated: the<br><br>
cellulases cloned from Acremonium sp.    (SEQ ID No. 5) and from Thielavia   terrestris   (SEQ ID No. 9).<br>
The two cellulases are able to defibrillate both Tencel and Tencel blended fabrics. By using an endoglucanase of the invention, only small fibrils are removed rather than whole fibers such as is the case when using acid cellulase mixtures from Trichoderma.   The strength loss of the treated fabric is threrefore kept at a minimum when using endoglucanases of the present invention.<br>
The following dosages gave a superior defibrillation, i.e. fuzz note 4 or above:<br>
15 S-CEVU/g fabric of cellulase from Acremonium sp   (SEQ ID No. 5); and<br>
10 S-CEVU/g fabric of cellulase from Thelavia  terrestris (SEQ ID No.9).<br>
EXAMPLE 2<br>
A new cellxxlytic enzyme was by expression cloning as well as by PCR cloning detected to be produced by a plant pat¬hogen, isolated from soy bean seeds and identified"as Ma-crophomina phaseolina.<br>
Production of biomass for PCR and expression cloning<br>
procedures:<br>
Isolate CBS 281.96 was grown in shake flask cultures on<br>
cellulose enriched potato dextrose broth, incubated for 5<br>
days at 260C (shaking conditions: 150 rpm).<br>
A. Cloning and expression of an endoglucanase from Macrophomina phaseolina<br>
mRNA was isolated from Macrophomina phaseolina,   grown in a cellulose-containing fermentation medium with agitation<br><br>
to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C. A library from Macrophomina phaseolina,   consisting of approx. 10* individual clones was constructed in E.   coli   as described with a vector background of 1%.<br>
Plasmid DNA from some of the pools was transformed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained from each pool.<br>
Endoglucanase-positive colonies were identified and isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above. The DNA sequence of the cDNA encoding the endoglucanase is shown in SEQ ID No. 10 and the corresponding amino acid sequence is shown in SEQ ID No. 11.<br>
The cDNA is obtainable from the plasmid in DSM 10512.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was' rescued by transformation of E.   coli   as described above. In order to express the endoglucanse in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene was purified. The gene was subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmid pA2C477.<br>
After amplification of the DNA in E.   coli   the plasmid was transformed into Aspergillus  oryzae  as described above.<br>
Screening of the cDNA library by hybridization and characterization of the positive clones. Approximately<br><br>
6000 colony forming units (c.f.u,) from the Macrophomina phaseolina  cDNA library in E. coli was screened by colony hybridization using a random-primed VP-labeled PCR product from M.   phaseolina  as probe. The PCR product was generated as described in the Materials and methods section. The positive cDNA clones were characterized by sequencing the ends of the cDNA inserts, and by determining the nucleotide seuence of the longest cDNA from both strands. The DNA sequence of the cDNA encoding the endoglucanase is shown in SEQ ID No. 10 and the corresponding amino acid sequence is shown in SEQ ID No. 11.<br>
B. Construction of gene fusion between the endoglucanase from Macrophomina phaseolina and the 43 kD endoglucanase from Humicola insolens<br>
One construction was prepared in order to make a derivative of the endoglucanase from M.   phaseolina  with the linker and CBD from the 43 kO endoglucanase from H. insolens   (disclosed in WO 91/17243). The native endoglucanase from M.   phaseolina  does not have a linker and/or a cellulose binding domain, CBD.<br>
The construction consists of the endoglucanase from M. phaseolina   (223 amino acids) and the 72 C-terminal amino acids from the H.   insolens   4 3 kO endoglucanase.<br>
The 4 3 kD endoglucanase cDNA from H.   insolens   is cloned into pHD414 in such a way that the endoglucanase gene is transcribed from the Taka-promoter. The resulting plasmid is named pCaHj418.<br>
The cDNA encoding the endogTQcanase from M.   phaseolina  is cloned into pYES2.0 as a BstX I/Not I fragment and the resulting plasmid is named pClC477.<br><br>
Primers:<br>
primer 1: 5'-<br>
GGTCGCCCGGACTGGCTGTTCCCGTACCCCCTCCAGCAGCACCAGCTCTCCGG -3' primer 2: 5'<br>
CCGGAGAGCTGGTGCTGCTGGAGGGGGTACGGGAACAGCCAGTCCGGGCGACC -3' pYES2.0 F.HT primer:  5' CGGACTACTAGCAGCTGTAATACG -3' AMG-term. primer:  5' CCCCATCCTTTAACTATAGCG -3'<br>
The endoglucanase fusion is constructed by the PCR overlap-extension method as described by Higuchi et al. 1988.<br>
Reaction A: The Polymerase Chain Reaction (PCR) is used<br>
to amplify the fragment of pCaHj418 between primer 1 and<br>
AMG-term. primer (the linker and CBD from the 4 3 kD<br>
endoglucanase from H.insolens).<br>
Reaction B: PCR amplification ot  the fragment between<br>
pYES2.0 F.HT primer and primer 2 in pClC477, the<br>
endoglucanase gene from M.   phaseolina.<br>
Reaction C: The two purified fragments are used in a<br>
third PCR in the presence of the primers flanking the<br>
total region, i.e. pYES2.0 F.HT primer and AMG-term.<br>
primer.<br>
The fragment amplified in reaction C is purified, digested with restriction enzymes, e.g. Xba I and BamH I. The purified digested fragment is ligated into pHD414 digested with restriction enzymes, e.g. Xba I and BamH I.<br>
Competent cells from E.   coli   strain DH5aF' (New England Biolabs) are transformed with the ligated plasmid and colonies containing the gene fusion are isolated. The sequence of the cloned part was verified by DNA sequen¬cing<br>
Polymerase Chain Reactions are carried out under standard<br><br>
conditions, as recommended by Perkin-Elmer.<br>
Reaction A and B start with 2 min. at 94°C followed by 20 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 1 min. at 72°C) and ends with 4 min. at 72 °C.<br>
Reaction C starts with (2 min. at 94°C, 1 min. at 52°C and 2 min. at 72°C), followed by 20 cycles of (30 sec. at 94°C, 30 sec. at 52°C and 90 sec. at 72°C) and ends with 4 min. at 72°C.<br>
The construct may be transformed into Aspergillus  oryzae as described above.<br>
EXAMPLE 3<br>
Cloning and expression of an endoglucanase from AcrBMonium sp.   and Sordaria fimicola<br>
Production of biomass for expression cloning procedures: Isolates CBS 478.94 and ATCC 52644, respectively, were grown in shake flask cultures on cellulose enriched pota¬to dextrose broth, incubated for 5 days at 2 60C (shaking conditions: 150 rpm).<br>
mRNA was isolated from Acremonium sp.,   CBS 478.94, and Sordaria  fimicola,   ATCC 52644, respectively, grown in a cellulose-containing fermentation medium with agitation to ensure sufficient aeration. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C. Libraries from Acremonium sp.,   and Sordaria  fimicola,   respectively, each consisting of approx. 10* individual clones were constructed in E. coli  as dpscr.ibed with a vector background of '1%.<br>
Plasmid DNA from some of the pools from each library was<br>
transforraed into yeast, and 50-100 plates containing 250-400 yeast colonies were obtained from each pool.<br>
Endoglucanase-positive colonies were identified and isolated on SC-agar plates with the AZCL HE cellulose assay. cDNA inserts were amplified directly from the yeast colonies and characterized as described in the Materials and Methods section above.<br>
The DNA sequence of the cDNA encoding the endoglucanase from Acremonium  sp.   is shown in SEQ ID No. 6 and the corresponding amino acid sequence is shown in SEQ ID No.7. The cDNA is obtainable from the plasmid in DSM 10080.<br>
The partial DNA sequence of the cDNA encoding the endoglucanase from Sordaria  fimicola   is shown in SEQ ID No. 19 (Nucleotide sequence of the 5'-end of the cDNA) and the corresponding amino acid sequence is shown in SEQ ID No, 20. The cDNA is obtainable from the plasmid in DSM 10576.<br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli  as described above. In order to express the endoglucanase in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene from Acremonium  sp.   and Sordaria  fimicola,   respectively, was purified. The genes were subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmids pA2C371 and pA2C502, respectively.<br>
After amplification of the DNA in E.   coli  the plasmids were transformed into Aspergillus  oryzae  as described above.<br><br>
EXAMPLE 4<br>
A. Cloning by PCR an endoglucanase from Crinipellis sca-bella,   CBS 280.96<br>
Isolate CBS 280.96 was grown in static flask cultures, holding wheat bran medium (per flask: 300g wheat bran added 450 ml salt solution), incubated for 6 days at 26C. After incubation the wheat bran was extracted with destined water (300ml per flask) and the extract tested for endoglucanase activity (0.1% AZCL-HE-Cellulose (megazyme) in 1% agarose (Litex agarose, Medinova), Activity was observed on the plates holding pH of 3.0, 7.0 and 9.5.<br>
nRNA was isolated from Crinipellis scabella grown as describe above. Mycelia were harvested after 3-5 days' growth, immediately frozen in liquid nitrogen and stored at -80°C. A library from Crinipellis  scabella,   consisting of approx. 10* individual clones was constructed in E. coli  as described with a" vector background of 1%.<br>
Approximately 10 000 colony forming units (c.f.u.) from the Crinipellis  scabella  cDNA library in E. coli was screened by colony hybridization using a random-primed ^^P-labeled PCR product from C. scabella  as probe. The PCR product was generated as described in the Materials and methods section. The positive cDNA clones were characterized by sequencing the ends of the cDNA inserts, and by determining the nucleotide seuence of the longest cDNA from both strands.<br>
The DNA sequence of the cDNA encoding the endoglucanase is shown in SEQ ID No. 12 and the corresponding amino acid sequence l-s—sb.own in SEQ ID No. 13.<br>
The cDNA is obtainable from the plasmid in DSM 10511.<br><br>
Total DNA was isolated from a yeast colony and plasmid DNA was rescued by transformation of E.   coli  as described above. In order to express the endoglucanse in Aspergillus,   the DNA was digested with appropriate restriction enzymes, size fractionated on gel, and a fragment corresponding to the endoglucanase gene was purified. The gene was subsequently ligated to pHD414, digested with appropriate restriction enzymes, resulting in the plasmid pA2C475.<br>
After amplification of the DNA in E.   coli  the plasmid was transformed into Aspergillus  oryzae  as described above.<br>
Construction of two gene fusions between the endoglucanase from Crinipellis scabella  and the linker/CBD region of the 4 3 kDa endoglucanase from Eumxcola insolens.<br>
The native endoglucanase from Crinipellis  scabella  neit¬her has a linker nor a cellulose binding domain (CBD). In addition, the full-length cDNA contains an ATG start codon upstream from the endoglucanase encoding open reading frame (ORF), presumably resulting in scrambled translation initiation upon heterologous expression of the cDNA, such as in the yeast Saccharomyces  cerevisiae and the filamentous fungus Aspergillus  oryzae.   Thus, two gene fusions between the endoglucanase from Crinipellis scabella  and the linker/CBD region of the 4 3kD endoglucanase from Humicola insolens   (disclosed in WO 91/17243) has been constructed using splicing by overlap extension (SOE) (Horton et al, 1989).<br>
Construction 1 consists of the cDNA encoding the 226-re-sidue endoglucanase from_CL__s;cajbelia fused by PCR with the 3'-end cDNA of H.   insolens  coding for the linker and CBD region (72 amino acids) at the COOH-terminus of the<br><br>
H.   insolens  4 3kD endoglucanase. The second hybrid construct is identical to the abovementioned gene fusion, except that the first five residues from the putative signal peptide have been deleted by PCR resulting in a shorter signal, which starts with the second in-frame ATG start codon.<br>
Plasmid constructs<br>
The plasmid pClC475 contains the full-length cDNA from C. scahella,   cloned into BstXI/Notl-cut yeast expression vector pYES 2.0, the plasmid pClC144 contains the full-length cDNA from H.   insolens,   cloned into the BstXI site of pYES 2.0.<br>
Splicing by overlap extension<br>
Two PCR fragments encoding the core region of the endog¬lucanase from C. scabella  were generated in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgClj, 0.01 % gelatin; containing 200 ^M each dNTP), using 50-100 ng of pClC475 as template, and 250 pmol of the reverse primer (5'-GACCGGAGAGCTGGTGCTGCTGGAGGGTTTACGAACACAGCCCGAGATATTAG TG- 3') in two combinations with 300-350 pmol of each forward primer (forward no. 1 5'-<br>
CCCCAAGCTTGACTTGGAACCAATGGTCCATCC-3', forward no. 2 5'-CCCCAAGCTTCCATCCAAACATGCTTAAAACGCTCG- 3'), a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Tag polymerase (Perkin-Elmer, Cetus, USA). Thirty cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 55 °C for 2 min, and extension at 72 °C for 3 min. The PCR fragment coding for the linker and CBD of the endoglucanase of H.   insolens was generated in PCR buffer (10-«M-Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgClj, 0.01 % gelatin; containing 200 /xM each dNTP) using 100 ng of the pClC144 template, 250 pmol<br><br>
forward primer (5'-CACTAATATCTCGGGC-<br>
TGTGTTCGTAAACCCTCCAGCAGCACCA-GCTCTCCGGTC-3'), 250 pmol of the pYES 2.0 reverse primer<br>
(5'-GGGCGTGAATGTAAGCGTGACATA-3') a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Taq polymerase (Perkin-Elmer, USA). Thirty cycles of PCR were performed as above. The PCR products were electrophoresed in 0.7 % low gelling temperature agarose gels (SeaPlaque, FMC), the fragments of interest were excised from the gel and recovered by treatment with agarase (New England Biolabs, USA) according to the manufacturer's instructions, followed by phenol extraction and ethanol precipitation at - 20 °C for 12 h by adding  2 vols of 95 % EtOH and 0.1 vol of 3M NaAc.<br>
The recombinant hybrid genes between the endoglucanase from Crinipellis  scabella  and the linker/CBD region of the 43 kD endoglucanase from Humicola  insolens  were generated by combining the overlapping PCR fragments from above (ca. 50 ng of each template) in two combinations in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgClj, 0.01 % gelatin; containing 200 ^M each dNTP). The SOE reaction was carried out using the DNA thermal cycler (Landgraf, Germany) and 2.5 units of Taq polymerase (Perkin-Elmer, Cetus, USA). Two cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 55 °C for 2 min, and extension at 72 °C for 3 min, the reaction was stopped, 250 pmol of each end-primer (forward no. 1<br>
5'-CCCCAAGCTTGACTTGGAACCAATGGTCCATCC-3', forward no. 2 5 ' -CCCCAAGCTTCCATCCAAACATGCTTAJyyi.CGCTCG-3 ' , reverse primer 5'-GGGCGTGAATGTAAGCGTGACATA-3') was added to the reaction mixture, and an additional 3 0 cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 55 °C for z min"; arid extension at 72 °C for 3 min.<br><br>
Construction of the expression cassettes for heterologous expression in Aspergillus oryzae<br>
The PCR-generated, recombinant fragments were electrophoresed in a 0.7 % low gelling temperature agarose gel (SeaPlaque, FMC), the fragments of interest were excised from the gel and recovered by treatment with agarase (New England Biolabs, USA) according to the manufacturer's instructions, followed by phenol extraction and ethanol precipitation at - 20 °C for 12 h. The DNA fragments were digested to completion with Hindlll and Xbal, and ligated into HijjdIII/Xjbal-cleaved pHD414 vector followed by electroporation of the constructs into E.   coli  DHIOB cells according to the manufacturer's instructions (Life Technologies, DSA).<br>
The nucleotide sequence of the resulting gene fusions were determined from both strands as described in the Materials and methods section, SEQ ID no. 14A and 15A. The constructs may be transformed into A.   oryzae  as described.<br>
EXAMPLE 5<br>
PCR facilitated detection of said type of cellulytic en¬zyme from 4 6 filamentous and monocentric fungi, representing 32 genera, from 23 families, belonging to 15 orders of 7 classes, covering all in all ail four groups of the true Fungi: Ascomycetous, Basidiomycetous, Chytri-diomycetous and Zygomycetous fungi<br>
5.1 Materials<br>
l.Diplodia gossypina Cooke<br>
Deposit of Strain, Ace No: CBS 274.96<br>
—	II ii.i —rf<br>
2.Ulospora bilgramii (Hawksw. et al.) Hawksw. et al. Ace No of strain: NKBC 1444,<br><br>
3.	Microsphaeropsis sp<br>
4.	Ascobolus stictoideus Speg.<br>
Ace No of strain: Q026 (Novo Nordisk collection) Isolated from goose dung, Svalbard, Norway<br>
5.	Saccobolus dilutellus (Fuck) Sacc. Deposit of strain: Ace No CBS 275.96<br>
6.	PenicilliuiTi verruculosum Peyronel Ex on Ace No of species: ATCC 62396<br>
7.	Penicillium chrysogenum Thorn Ace No of Strain: ATCC 9480<br>
8.	Thermomyees verrucosus Pugh et al Deposit of Strain, Ace No.: CBS 285.96<br>
9.	Xylaria hypoxylon L. ex Greville Deposit of Strain, Ace No: CBS 2 8 4.96<br><br>
10.	Poronia punctata (Fr.ex L.) Fr. Ref:A.MunJc: Danish Pyrenomyeetes, Darisk Botanisk Arkiv, Voll7,l 1957<br>
11.	Nodulisporum sp<br>
Isolated from leaf of Camellia reticulata (Theaceae,<br>
Guttiferales),<br>
Kunming Botanical Garden, Yunnan Province, China<br>
12.	Cylindroearpon sp<br>
Isolated from marine sample, the Bahamas<br>
13.	Fusarium anguioides Sherbakoff<br>
AceNo of strain: IFO 44 67<br><br>
14.	Fusarium poae (Peck) Wr.<br>
Ex on Ace No of species: ATCC 60883<br>
15.	Fusarium  solani (Mart.)Sacc.emnd.Snyd &amp; Hans. Ace No of strain: IMI 107.511<br>
16.	Fusarium oxysporum ssp lycopersici (Sacc.)Snyd. &amp; Hans.<br>
Ace No of strain: CBS 645.78<br>
17.	Fusarium oxysporum ssp passiflora Ace No of strain: CBS 744.79<br>
18.	Glioeladium catenulatum Gillman &amp; Abbott Ace. No. of strain: ATCC 10523<br>
19.	Nectria pinea Dingley<br>
Deposit of Strain, Ace. No. CBS 279.96<br>
20.	Sordaria macrospora Auerswald Ex on Ace No of species: ATCC 602 55<br>
21.	Humicola grisea Traeen<br>
ex on Ace No for the species: ATCC 22726<br>
22.	Humicola nigrescens Omvik Ace No of strain: CBS 819.73<br>
23.	Seytalidium thermophilum (Cooney et Emerson) Austwiek Ace No of strain: ATCC 28085<br>
24.	Thielavia thermophila Fergus et Sinden (syn Corynascus thermophilus)<br>
Ace No of strain: CBS 174.70, IMI 145.136<br>
25.	Cladorrhinum foecundissimum Saccardo et Marchal<br>
Ex on Ace No of species: ATCC 623 73<br><br>
26.	Syspastospora boninensis<br>
Ace No of strain: NKBC 1515 (Nippon University, profe Tubaki Collection)<br>
27.	Chaetomium cuniculorum Fuckel Ace, No. of strain: CBS 799.83<br>
28.	Chaetomium brasiliense Batista et Potual Ace No of strain: CBS 122.65<br>
29.	Chaetomium murorum Corda Ace No of strain: CBS 163.52<br>
30.	Chaetomium virescens (von Arx) Udagawa Aac.No. of strain: CBS 547.75<br>
31.	Nigrospora sp<br>
Deposit of strain, Ace No: CBS 2 7 2.96<br>
32.	Nigrospora sp Isolated from:<br>
33.	Diaporthe syngenesia<br>
Deposit of strain, Ace No:,CBS 278.96<br>
34.	Colletotrichum lagenarium (Passerini) Ellis et<br>
Halsted<br>
syn Glomerella cingulata var orbiculare Jenkins et<br>
Winstead<br>
Ex on dcc No of species: ATCc 52609<br>
35.	Exidia glandulosa Fr.<br>
Deposit of Strain, Ace No: CBS 277.96<br>
36.	Femes fomentarius (L.) Fr.<br>
Deposit of strain: Ace No. CBS 276.96<br><br>
37.	Spongipellis (?)<br>
Deposit of Strain: Ace No CBS 283.96<br>
38.	Rhizophlyctis rosea (de Bary &amp; Wor) Fischer Deposit of Strain: Ace No.: CBS 282.96<br>
39.	Rhizomucor pusillus (Lindt) Schipper syn: Mucor pusillus<br>
Ace No of strain: IFO 4578<br>
40.	Phycomyces nitens (Kunze) van Tieghem &amp; Le Monnier<br>
Ace No of strain: IFO 4814<br>
41 Chaetostylum fresenii van Tieghem &amp; Le Monnier syn. Helicostylum fresenii Ace No of strain NRRL 2 3 05<br>
42.	Triehotheeium roseum, Ace No of strain: IFO 5372<br>
43.	Coniothecium sp<br>
Endophyte, "isolated from leaf of flowering plant, Kurmiug , Yunnan, China<br>
•44. Deposit of strain. Ace No.: CBS 271.96 Coelomycete, Isolated from leaf of Artocarpus altilis (Moraeeae, Urticales), Christiana, Jamaica<br>
45.	Deposit of strain. Ace No.: CBS 273.96 Coelomycete, isolated from leaf of Pimenta dioica (Myrtaceae, Myrtales), Dallas Mountain, Jamaica<br>
46.	Deposit of strain: CBS 270.96 Coelomycete, isolated from leaf of Pseudocalymma alliaceum (Bignoniaceae, Solanales) growing in Dallas Mountain, Jamaica<br><br>
5.2 Procedure<br>
Maintenance of strains and production of biomass:<br>
The strains were maintained on agar in petrie dishes (9cm) or on  slants (see list of Media: PCA and PDA). 44 of the strains were grown in shake flasks under the following  growth conditions: general fungal media as PC, PD and PB 9 or YPG (see list of media);  incubation time from 3 to 9 days; temperature 2 6°C; rpm between 150 and 175 . Strain No 14 (F.poae) was grown on wheat bran for 15 days (26'C; static) . Strain No 38 was grown in dilute salt solution (DS/2), added 1 cm^ pieces of autoclaved filter paper.<br>
Activity test:<br>
Activity was tested on 0.1% AZCL-HE-Cellulose (Megazyme) plates (14 cm Petrie dishes), made up in 1% agarose (HSB, Litex Agarose, Medinova). All tests were done in triplicate, viz. AZCL-HE-Cellulose dissolved in three buffers, adjusted to pH 3, 7 or 9.5 (using various proportions of the following two ingredients Citric acid monohydrat, Merck art. No 100244 (21.0 g) dissolved in water, making a total of 1000 ml; O.lM tri-Sodium dodeca-brohydrate, Merck art.no. 6578 (38 g), dissolved in wa¬ter, making a total of 1000 ml. The mixing is done immidiately before use.<br>
Harvesting of Biomass:<br>
The biomass was harvested by filtering (mesh adjusted to the growth of the fungus, the finest used for fungi which have highly sporulating mycelium as e.g. Fusarium spp.). The biomass on the filter was scraped into a sterile plastic bag and immidiately frozen (by submerging into liquid nitrogen).<br><br>
5.3 Results<br>
I. Using the PCR screening and amplification techniques described in Materials and Methods the following partial cDNA sequences were obtained:<br>
Saccobolus dilutellus (Fuck) Sacc, CBS 275.96: SEQ ID<br>
No. 21 (and the deduced amino acid sequence in SEQ ID No.<br>
22) ;<br>
Thermomyces verrucosus, CBS 285.96: SEQ ID No. 2 3 (and<br>
the deduced amino acid sequence in SEQ ID No. 24) ;<br>
Xylaria hypoxylon, CBS 284.96: SEQ ID No. 25 (and the<br>
deduced amino acid sequence in SEQ ID No. 26);<br>
Fusarium oxysporum ssp lycopersici, CBS 645.78: SEQ ID<br>
No. 27 (and the deduced amino acid sequence in SEQ ID No.<br>
28);<br>
Nectria pinea, CBS 279.96: SEQ ID No. 29 (and the deduced<br>
amino acid sequence in SEQ ID No. 30);<br>
Humicola grisea, ATCC 2272 6: SEQ ID No. 31 (and the<br>
deduced amino acid sequence in SEQ ID No. 32);<br>
Humicola nigrescens, CBS 819.73: SEQ ID No. 33 (and the<br>
deduced amino acid sequence in SEQ ID No. 34) ;<br>
Cladorrhinum foecundissimum, ATCC 62373: SEQ ID No. 35<br>
(and the deduced amino acid sequence in SEQ ID No. 36);<br>
Syspastospora boninensis, NKBC 1515: SEQ ID No. 37 (and<br>
the deduced amino acid sequence in SEQ ID No. 38);<br>
Nigrospora sp., CBS 272.96: SEQ ID No. 39 (and the<br>
deduced amino acid sequence in SEQ ID No. 40);<br>
Chaetostylum fresenii: SEQ ID No. 41 (and the deduced<br>
amino acid sequence in SEQ ID No. 42);<br>
Exidia glandulosa, CBS 277,96: SEQ ID No. 43 (and the<br>
deduced amino acid sequence in SEQ ID No. 44);<br>
Coniothecium sp.: SEQ ID No. 45 (and the deduced amino<br>
acid sequence in SEQ ID No. 46);<br>
Deposition No. CBS 271.96: SEQ ID No. 47 (and the deduced<br>
amino acid sequence in SEQ ID No. 48)^;	r<br>
Deposition No. CBS 270.96: SEQ ID No. 49 (and the deduced amino acid sequence in SEQ ID No. 50);<br><br>
Diplodia gossypina, CBS 274.96: SEQ ID No. 51 (and the<br>
deduced amino acid sequence in SEQ ID No. 52);<br>
Ulospora bilgramii, NKBC 1444: SEQ ID No. 53 (and the<br>
deduced amino acid sequence in SEQ ID No. 54);<br>
Penicillium verruculosum, ATCC 62396: SEQ ID No. 55 (and<br>
the deduced amino acid sequence in SEQ ID No. 56);<br>
Poronia punctata: SEQ ID No. 57 (and the deduced amino<br>
acid sequence in SEQ ID No. 58);<br>
Fusarium anguioides, IFO 4 4 67: SEQ ID No. 59 (and the<br>
deduced amino acid sequence in SEQ ID No. 60);<br>
Thielavia thermophila, CBS 174.70: SEQ ID No. 61 (and the<br>
deduced amino acid sequence in SEQ ID No. 62);<br>
Chaetomium cuniculorum, CBS 799.83: SEQ ID No. 63 (and<br>
the deduced amino acid sequence in SEQ ID No. 64);<br>
Chaetomium virescens: SEQ ID No. 65 (and the deduced<br>
amino acid sequence in SEQ ID No, 66);<br>
Colletotrichum lagenarium: SEQ ID No. 67 (and the deduced<br>
amino acid sequence in SEQ ID No. 68);<br>
Phycomyces nitens: SEQ ID No. 69 (and the deduced amino<br>
acid sequence in SEQ ID No. 70); and<br>
Tricho€hecium roseum: SEQ ID No. 71 (and the deduced<br>
amino acid sequence in SEQ ID No. 72);<br>
II, Using the PCR screening and amplification techniques described in Materials and Methods partial cDNA encoding partially for the enzyme of the invention was obtained and the plasmid was deposited according to the Budapest Treaty:<br>
Escherichia  coli,   DSM 10583, deposition date 13 March, 1996;<br>
cDNA from Trichothecium roseum;<br>
Escherichia  coli,   DSM 10584, deposition date 13 March, 1996;<br>
cDNA from Syspastospora boninensis; I—Escherichia  coli,   DSM 10585, deposition "date" 13 March, 1996; cDNA from Cheatomium murorum<br><br>
Escherichia  coli,   DSM 10587, deposition date 13 March,<br>
1996;<br>
cDNA froin Sordaria fimicola;<br>
Escherichia  coli,   DSM 10588, deposition date 13 March,<br>
1996;<br>
cDNA from the unidentified strain CBS 273.96;<br>
Escherichia  coli,   DSM 1058 6, deposition date 13 March,<br>
1996;<br>
cDNA from Spongipellis sp.<br>
Color clarification of crude supernatants<br>
During normal wash the fabric will often fade. However, the fabric appearance is improved and the original colours are much better preserved or maintained if the fabric is washed with a cellulase giving color clarification. Color clarification is measured as removal of surface fibrils and fibers protruding from the yarn of a textile containing cellulosic fibers.<br>
Apparatus	:    Terg-o-tometer<br>
Liquid volume      :    100 ml<br>
Agitation	:    150 movements/min with vertical<br>
stirrer<br>
Rinse time	:    5 min in tapwater<br>
Washing temp       :    40°<br>
Washing liqour     :    0.05 M phosphate buffer<br>
pH	:    7. 0<br>
Washing time       :    30 min<br>
Repetitions        :    2<br>
Enzymes	:    Crude supernatants from the<br>
strains shown below.<br>
Dosage	:    Two dosages from : 2 00, 500,<br>
1000 or 2500 S-CEVU/1<br>
Textile	:    2 swatches of aged black 100%<br>
cotton	5x6 cm (0.9 gram)<br><br>
Drying	:    Tumble dry<br>
Evaluation:<br>
The light remission is measured by a Datacolor Elrepho Remission spectrophotometer. Remission is calculated as delta L (Hunter Lab-values). When the surface fibrils and fibers protruding from the yarn are removed by the cellulase, the surface of the black fabric appears darker, and lower L values are obtained.<br>
The samples are compared with a blind sample, i.e. washed without enzyme. Below is shown the delta L remission values compared to a blind sample.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Asp Asp Ser Ser Phe Pro Val Phe Gin Gly Ser Gly Ser Gly Asp Val<br>
225 .	230	235	240<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Tyr Ser Gin Cys Phe Pro<br>
305	310<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
family-specific sequences to clone Cellulase homologue cDNAs from Fusarium oxysporum,   Gene, (1994), Vol. 15, pp. 163-167.<br>
Saloheimo, A., et al., "A novel, small endoglucnaase gene, egI5,   from Trichoderma reesei   isolated by expression in yeast", Molecular Microbiology (1994), Vol. 13(2), pp. 219-228.<br>
van Arsdell, J.N. et al, (1987) Cloning, characterization, and expression in Saccharomyces cerevisiae  of endoglucanase I from Trichoderma  reesei, Bio/Technology 5: 60-64.<br>
Pentxila, M. et al, (1986) Homclogy between cellulase gnees o:f Trichoderma reesei:   complete nucleotide sequence of the endoglucanase I gene. Gens   45:253-263.<br>
Saloheimo, M. et al, (1988) EGIII, a new endoglucanase from Trichoderma reesei:   the characterization of both gene and enzyme. Gene  63:11-21.<br>
GoTJzales, R. , et al., "Cloning, sequence analysis and yeast expression of the egil gene from Trichoderma longibrachiatum",   Appl. Microbiol. Biotechnol. (1992), Vol. 38, pp. 370-375.<br>
Ooi, T. et al. "Cloning and sequence analysis of a cDNA for cellulase (FI-CMCase) from Aspergillus  aculeatus" Curr. Genet. (1990), Vol. 18, pp. 217-222.<br>
Ooi, T. et al, "Complete nucleotide sequence of a gene coding for Aspergillus  aculeatus  cellulase (FI-CMCase)" Nucleic Acids Research (1990), Vol. 18, No. 19, p. 5884.<br>
Xue, G. et al., "Cloning and expression of multiple<br>
cellulase   cDNAs   frnTnthp   anaoT-nhir-.   yiimpn    ■fiinrm?^<br><br>
Neocallimastix patriciarum  in E.   coli,   J.   Gen. Microbiol. (1992), Vol. 138, pp. 1413-1420.<br>
Xue, G. et al., "A novel polysaccharide hydrolase cDNA (celD)   from Neocallimastix patriciarum  encoding three multi-functional catalytical domains with high endoglucanase, cellobiohydrolase and xylanase activities", J. Gen. Microbiol. (1992), Vol. 13S, pp. 2397-2403.<br>
Zhou, L. et al., "Intronless celB   from the anaerobic fungus Neocallimastix patriciarum  encodes a modular family A endoglucanase", Biochem. J. (1994), Vol. 297, pp. 35S-3 64.<br>
Dalboge, H. and Heldt-Hansen, H.P., "A novel method for efficient expression cloning of fungal enzyme genes", Mol. Gen. Genet. (1994), Vol. 243, pp. 253-260.<br>
All, B.R.S. et al., "Cellulases and hemicellulases of the anaerobic fungus Piromyces  constitute a multiproteln cellulose-bin.d.lng complex and are encxDded by multigene families-"*, FEMS Microbiol. Lett. (1995), Vol. 125, No. 1, pp. 15-21.<br>
DNA Data Bank of Japan (DDBJ).<br>
Wang, H.Y. and Jones, R.W.: "Cloning, characterization and functional expression of an endoglucanase-encoding gene from the phytopathogenic fungus Macrophomina phaseolina",   Gene, 158:125-128, 1995.<br>
Wang, H.Y. and Jones, R.W.: "A unique endoglucanase-encoding gene cloned from the phytopathogenic fungus Macrophomina phaseolina^y Applr and Environm. Microbiology, 61:2004-2006, 1995.<br><br>
B. Henrissat: Biochem. J., 280:309-316, 1991.<br>
Schauwecker, F., Wanner, G., Kahmann, R.: "Filament-specific expression of a cellulase gene in the dimorphic fungus Ustilago maydis", 1995, Biological Chemistry Hoppe-Seyler, 376:617-625.<br>
WO 93/20193<br>
WO 94/21801<br>
WO 94/26880<br>
WO 95/02043<br>
The Drawings:<br>
Feng and Doolittle, 1987, J. Mol. Evol. 25: 351-360.<br>
NIH Data Base (Entrez, version spring 1996) available on<br>
World Wide Web: (htt-<br>
p: //www3 . ncbi. nlm.nih.gov/irtbin/ef/entrezTAX) .<br>
Eriksson, O.E. &amp; Hawksworth, D.L.: Systema Ascomyc6tum vol 12 (1993) .<br>
Jiilich, W. : Higher Taxa of Basidiomycetes, Bibliotheca Mycologia 85, 485pp (1981).<br>
O'Donnell, K.: Zygomycetes in culture. University of Georgia, US,  257pp (1979).<br>
Hawksworth, D.L., Kirk, P.M., Sutton, B.C. and Pegler,<br>
D.N.: Dictionary of the fungi. International Mycological<br>
Institute, 616pp (1995);		^<br>
Von Arx, J.A.: The genera of fungi sporulating in<br><br>
culture, 424pp (1981).<br>
Detailed Description:<br>
Ford et al.. Protein Expression and Purification 2:   95-107, 1991,<br>
Cunningham and Wells, Science 244, 1081-1085, 1989,<br>
de Vos et al., Science 255: 306-312, 1992.<br>
Smith et al., J, Mol. Biol. 22A:   899-904, 1992.<br>
Wlodaver et al., FEES Lett. 309: 59-^4, 1992.<br>
ToTtnne, P. et al. "Cellulose-Binding Domains: Classi¬fication and Properties" in "Enzymatic Degradatiion of Insoluble Carbohydrates", John M. Saddler and Michael H. Penner (Eds.), ACS Symposium Series, No. 618, 1996.<br>
WO 90/00609<br>
WO 95/16782<br>
Needleman, S.B. and Wunsch, CD., Journal  of Molecular Biology,   48.: 443-453, 1970.<br>
WO 94/14953<br>
Sambrook, J., Fritsch, E, F. &amp; Maniatis, T. 1989. Molecular Cloning: A Laboratory Manual. Cold Spring Harbor Lab., Cold Spring Harbor, NY.<br>
Beaucage and Caruthers, Tetrahedron Letters 22 (1981), 1859 - 1869.<br><br>
Matthes et al., EMBO Journal 3 (1984). 801 - 805.<br>
US 4,683,202<br>
Saiki et al.. Science 239 (1988), 487 - 491.<br>
Hitzeman et al., J. Biol. Chem. 255 (1980), 12073 -12080.<br>
Alber and Kawasaki, J. Mol. Appl. Gen. 1 (19 82), 419 -434) .<br>
Young et al., in Genetic Engineering of Microorganisms for Chemicals (HoLlaender et al, eds.), Plenum Press, New York, 1982).<br>
US 4,599,311<br>
Russell et al., Nature 304 (1983), 652 - 654.<br>
McKnight et al.. The EABO J. 4. (1S85) , 2093 - 2099<br>
P.R. Russell, Gene 4-0, 1985, pp. 125-130.<br>
US 4,870,008<br>
0. Hagenbuchle et al., Nature 289, 1981, pp. 643-646.<br>
L.A. Vails et al.. Cell 48. 1987, pp. 887-897.<br>
WO 87/02670<br>
M. Egel-Mitani et al.. Yeast 6. 1990, pp. 127-137.<br>
US 4,54 6,082	,<br>
EP 16 201<br><br>
EP 123 294<br>
EP 123 544<br>
EP 163 529<br>
WO 89/02463<br>
WO 92/11378<br>
US 4,599,311<br>
US 4,931,373<br>
US 4,870,008<br>
US 5,037,743<br>
US 4,845,075<br>
US 4,931,373<br>
Gleeson et al., J. Gen. Microbiol. 132, 1986, pp. 3459-3465.<br>
US 4,882,279<br>
EP 272 277<br>
EP 230 023<br>
Malardier et al., 1989, Gene 78: 147-156.<br>
WO 93/11249.<br>
WO 94/14953.<br><br>
wo 95/02043.<br>
Horton, R. M., Hunt, H. D., Ho, S. N., Pullen, J. K., and Pease, L. R. (1989) Gene  11,   61-68<br>
Dalb0ge, H., and Heldt-Hansen, H. (1994) Uol.   Gen.   Genet. 243, 253-260<br>
Christensen, T., W0ldike, H., Boel, E., Mortensen, S. B., Hjortshoj, K., Thim, L., and Hansen, M. T. (1988) Bio/Technology 6,   1419-142 2<br>
Sanger, F., Nicklen, S., and Coulson, A. R. (1977) Proc. Na-tl.   Acad.   Sci.   U.   S.   A.   lA,   5463-5467<br>
Devereux, J., Haeberli, P., and Smithies, O. (1984) Nucleic Acids  Res.   12,   387-395<br>
Becker, D. M. &amp; Guarante, L. 1991. Methods Enzyinol. 194: 182-187.<br>
Gubler, D. &amp; Hoffman, B. J. 1983. &amp;ejifi 25: 263-269.<br>
R. Higuchi, B. Krummel, and "R.K. Saiki (1988). A general method of in vitro preparation and specific mutagenesis of DNA fragments: study of protein and DNA interactions. Nucl. Acids Res. 16: 7351-7367.<br>
Sanger, F., Nicklen, S. &amp; Coulson, A. R. 1977. Proc. Natl. Acad. Sci. U. S. A. 74: 5463-5467.<br>
N. Axelsen et al., A Manual of Quantitative Immunoelectrophoresis, Blackwell Scientific Publications, 197 3, Chapters 2,3,4 and 23.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
The present invention relates to a method of producing an enzyme exhibing endoglucanase activity.<br>
The present invention relates to novel enzyme preparations comprising an enzyme exhibiting endoglucanase activity which performs very good in indus¬trial applications such as laundry compositions, for biopolishing of newly manufactured textiles, for provid¬ing an abraded look of cellulosic fabric or garment, and for treatment of paper pulp. Further, the invention relates to DNA constructs encoding such enzymes, a method for providing a gene encoding for such enzymes, a method of producing the enzymes, enzyme preparations containing such enzymes, and the use of these enzymes for a number of industrial applications.<br>
BACKGROUND OF THE INVENTION<br>
Cellulases or cellulytic enzymes are enzymes involved in hydrolyses of cellulose. In the hydrolysis of native cel¬lulose, it is known that there are three major types of cellulase enzymes involved, namely cellobiohydrolase (l,4-j8-D-glucan cellobiohydrolase, EC 3.2.1.91), endo-/3-1,4-glucanase (endo-1,4-i8-D-glucan 4-glucanohydrolase, EC 3.2.1.4) and j3-glucosidase (EC 3.2.1.21).<br>
Cellulases are synthesized by a large number of microor¬ganisms which include fungi, actinomycetes, myxobacteria and true bacteria but also by plants. Especially endoglucanases of a wide variety of specificities have been identified.<br>
A very important industrial use of cellulytic enzymes is the use for treatment of cellulosic textile or fabric, e.g. as ingredients in detergent compositions or fabric<br><br>
softener compositions, for bio-polishing of new fabric (garment finishing), and for obtaining a "stone-washed" look of cellulose-containing fabric, especially denim, and several methods for such treatment have been sug¬gested, e.g. in GB-A-1 368 599, EP-A-0 307 564 and EP-A-0 435 876, WO 91/17243, WO 91/10732, WO 91/17244, PCT/DK95/000108 and PCT/DK95/00132.<br>
Another important industrial use of cellulytic enzymes is the use for treatment of paper pulp, e.g. for improving the drainage or for deinking of recycled paper.<br>
Especially the endoglucanases (EC No. 3.2.1.4) constitute an interesting group of hydrolases for the mentioned industrial uses. Endoglucanases catalyses endo hydrolysis of 1,4-β-D-glycosidic linkages in cellulose, cellulose derivatives (such as carboxy methyl cellulose and hydroxy ethyl cellulose), lichenin, (3-1,4 bonds in mixed i3-l,3 glucans such as cereal β-D-glucans or xyloglucans and other plant material containing cellulosic parts. The authoirized name,  is endo-1,4-/8—D-glucan 4-glucano hydrolase, but the abbreviated term endoglucanase is used in the present specification. Reference can be made to T.-M. Enveri, "Microbial Cellulases" in W.M. Fogarty, Microbial Enzymes and Biotechnology, Applied Science Pub¬lishers, p. 183-224 (1983); Methods in Enzymology, (1988) Vol. 160, p. 200-391 (edited by Wood, W.A. and Kellogg, S.T.); Beguin, P., "Molecular Biology of Cellulose Degra¬dation", Annu. Rev. Microbiol. (1990), Vol. 44, pp. 219-248; Beguin, P. and Aubert, J-P., "The biological degra-dation of cellulose", FEMS Microbiology Reviews 13. (1994) p.25-58; Henrissat, B,, "Cellulases and their interaction with cellulose". Cellulose (1994), Vol. 1, pp. 169-196.<br>
Fungal endoglucanases have been described in numerous publications, especially those derived from species as e.g. Fusarium oxysporum,   Trichoderma reesei,   Trlchoderma<br><br>
longibrachiatum,   Aspergillus aculeatus,   Neocallimastix patriciarum,   and e.g. from species of the genera Piromyces,   Humicola,   Myceliophthora,   Geotricum, Penicillium,   Irpex,   Coprinus.<br>
For example, fungal endoglucanases have been described by Sheppard, P.O., et al., "The use of conserved cellulase family-specific sequences to clone Cellulase homologue cDNAs from Fusarium  oxysporum,   Gene, (1994), Vol. 15, pp. 163-167, Saloheimo, A., et al., "A novel, small endoglucanase gene, egI5,   from Trichoderma reesei   iso¬lated by expression in yeast". Molecular Microbiology (1994), Vol. 13(2), pp. 219-228; van Arsdell, J.N. et al, (1987) , Cloning, charact-erization, and expression in Saccharomyces cerevisia^  of endoglucanase I from Trichoderma reesei,   Bio/Technology 5: 60-64; Penttila, M. et al., (1986), *'Homology between cellulase genes of Trichoderma reesei:   complete nucleotide sequence of the endoglucanase I gene". Gene  45:253-263; Saloheimo, M. et al, (1988), "EGIII, a new endoglucanase from Trichoderma reesei:   the characterization of both gene and enzyme". Gene  63:11-21; GxymzAles,   R. , et al., "Cloning, sequence analysis and yeast expression of the egll gene from Trichoderma longibrachiatum",   Appl. Microbiol. Biotechnol., (1992), Vol. 38, pp. 370-375; Ooi, T. et al. "Cloning and sequence analysis of a cDNA for cellulase (FI-CMCase) from Aspergillus  aculeatus",   Curr. Genet., (1990), Vol. 18, pp. 217-222; Ooi, T. et al, "Complete nucleotide sequence of a gene coding for Aspergillus aculeatus  cellulase (FI-CMCase)", Nucleic Acids Research, (1990), Vol. 18, No. 19, p. 5884; Xue, G. et al., "Clon¬ing and expression of multiple cellulase cDNAs from the anaerobic rumen fungus Neocallimastix patriciarum  in E. coli",   J. Gen. Microbiol., (1992), Vol. 138, pp. 1413-1420; Xue, G. et airv~,—A novel polysaccharide hydrolase cDNA (celD)   from Neocallimastix patriciarum  encoding three multi-functional catalytical domains with high<br><br>
endoglucanase, cellobiohydrolase and xylanase activ¬ities", J. Gen. Microbiol., (1992), Vol. 138, pp. 2397-2403; Zhou, L. et al., "Intronless celB  from the anaerobic fungus Neocallimastix patriciarum  encodes a modular family A endoglucanase", Biochem. J., (1994), Vol. 297, pp. 359-364; Dalboge, H. and Heldt-Hansen, H.P., "A novel method for efficient expression cloning of fungal enzyme genes", Mol. Gen. Genet., (1994), Vol. 243, pp. 253-260; Ali, B.R.S. et al., "Cellulases and hemicellulases of the anaerobic fungus Piroiayces  consti¬tute a multiprotein cellulose-binding complex and are encoded by multigene families", FEMS Microbiol. Lett., (1995), Vol. 125, No. 1, pp. 15-21. Further, the DNA Data Bank of Japan (DDBJ database publicly available at Internet) comprises two DNA sequences cloned from Penicillium janthinellum  encoding endoglucanases (cloned by A. Koch and G. Mernitz, respectively) and a DNA sequence cloned from Humicola  grisea  var.   theirmoidea encoding an endoglucanase (cloned by T. Uozumi). Two endoglucanases from Macrophomina phaseolina  have been cloned and sequerrced, see Wang, H.Y. and Jones, R.W. : "Cloning, characterization and functional expression of an endoglucana^e-encoding gene from the phytopathogenic fungus Macrophomina phaseolina"   in Gene, 158:125-128, 1995, and Wang, H.Y. and Jones, R.W.: "A unique endoglucanase-encoding gene cloned from the phytopathogenic fungus Macrophomina phaseolina"   in Applied And Environmental Microbiology, 61:2004-2006, 1995. One of these endoglucanases shows high homology to the egl3 endoglucainase from the fungus Trichoderma reesei,   the other shows homology to the egll from the microbial phytopathogen Pseudomonas solanacearum  indicat¬ing that both endoglucanases belong to family 5 of glycosyl hydrolases (B. Henrissat, Biochem J 280:309-316 (1991)). Filainent-specific expression of a cellulase gene in the dimorphic fungus Ustilago maydis  is disclosed in Schauweoker, F. et al. (1995).<br><br>
wo 91/17243 (Novo Nordisk A/S) discloses a cellulase pre¬paration consisting of a homogenous endoglucanase compo¬nent immunoreactive with an antibody raised against a highly purified 43 kDa endoglucanase derived from Humicola insolens,   DSM 1800; WO 91/17244 (Novo Nordisk A/S) discloses a new (hemi)cellulose degrading enzyme, such as an endoglucanase, a cellobiohydrolase or a 0-glucosidase, which may be derived from fungi other than Trichoderma  and Phanerochaete;   WO 93/20193 discloses an endoglucanase derivable from Aspergillus aculeatus;  WO 94/21801 (Genencor Inc.) concerns a cellulase system iso¬lated from Trichoderma longibrachiatum  exhibiting endoglucanase activity; WO 94/26880 (Gist Brocades N.V.) discloses an isolated mixture of cellulose degrading enzymes, which preferable are obtained from Trichoderma, Aspergillus  or Disporotrichum, comprising endoglucanase, cellobiohydrolase, and xyloglxKranase activity; and WO 95/02043 (Novo Nordisk A/S) describes an enzyme with endoglucanase activity derived from Trichoderma harzianum,   which can be used for a number of purposes including e.g.   degradation or mcKiification of plant cell walls.<br>
It is also known that cellulases may or may not have a cellulose binding domain (a CBD). The CBD enhances the binding of the enzyme to a cellulose-containing fiber and increases the efficacy of the catalytic active part of the enzyme.<br>
There is an ever existing need for providing novel cellulase enzyme preparations which may be used for applications where cellulase, preferably an endoglucanase, activity is desirable.<br>
The object of the present- in-vention is to provide novel enzyme preparations having substantial cellulytic activ¬ity at acid, neutral or alkaline conditions and improved<br><br>
performance in paper pulp processing, textile treatment, laundry processes or in animal feed; preferably novel cellulases, more preferably well-performing endoglucana-ses, which are contemplated to be producible or produced by recombinant techniques.<br>
SUMMARY OF THE INVENTION<br>
Surprisingly, it has been found that a group of endoglucanases having certain unique characteristics per¬form very good in those industrial applications for which endoglucanases are conventionally used. These unique characteristics can be described in terms of conserved regions of the amino acid sequence of the enzyme protein and the inventors have found that cellulytic enzymes, i.e. enzymes exhibiting cellulytic activity, having cer¬tain conserved reg^ions axe very effednrve e.g. in th-e treatment of laundry, in the treatment of newly manufac¬tured textile, in the treatment of papermaking pulp.<br>
Accordingly, in its first aspect t±ie present invention relates to an enzyme preparation consisting essentially of an enzyme having cellulytic activity and comprising a first amino acid sequence consisting of 14 amino acid residues having the following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp Xaa 1   2   3   4   5   6   7   8   9   10  11  12  13  14<br>
and a second amino acid sequence consisting of 5 amino acid residues having the following sequence<br>
Trp Cys Cys Xaa Cys<br>
12   3   4   5'	'<br><br>
in position 3 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 4 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 8 of the first sequence, the amino acid is<br>
Arg, Lys or His;<br>
in position 9, 10, 12 and 14, respectively, of the first<br>
sequence, and in position 4 of the second sequence, the<br>
amino acid is any of the 2 0 naturally occurring amino<br>
acid residues with the provisos that, in the first amino<br>
acid sequence, (i) when the amino residue in position 12<br>
is Ser, then the amino acid residue in position 14 is not<br>
Ser, and (ii) when the amino residue in position 12 is<br>
Gly, then the amino acid residue in position 14 is not<br>
Ala.<br>
This surprising finding of clearly recognisable conserved regions, in spite of rather prominent variations founti within well-performing endoglucanase enzymes, is a result of studies of a number of fungal DNA sequences encGdxng fox speclfdc amino acid sequences of enzymes having sig¬nificant cellulytic, especiaJLly endoglucanase, activ¬ities.<br>
Based on this finding, a novel molecular method taylored to screen specifically for genomic DNA or cDNA characterised by encoding the enzymes of the invention has been developed. As tools for this three sets of degenerated primers were constructed. Accordingly, in its second aspect, the invention relates to a method for providing a gene encoding fojr cellulytic enzymes having the  above conserved regions.<br>
By using this method, i.e. the set of primers for a PCR screening on genomic DNA, it was surprisingly found that'^ DNA encoding for said enzymes can be found from a broad range of fungi, belonging to taxonomically very different<br><br>
organisms and inhabiting ecologically very different niches.<br>
Further, by using this method it has been possible to find DNA sequences encoding for the core regions (cata-lytically active regions or domains) of said enzymes without any attached cellulose binding domain (CBD) which core regions of enzymes would not have been selected by using conventional performance based screening approaches. The inventors have verified experimentally that the linking of a CBD region to a core region enzyme (comprising the catalytically active region or domain of the enzyme) of the present invention results in a sig¬nificantly improved performance, e.g. a fifty times higher performance, of the mrultiple domain enzyme.<br>
Accordingly, the present ixsvention provides novel cellulases, especially endoglucanases, having improved performance in industial applications, either in their native form, or homo- or heterologously produced.<br>
In further aspects, the present invention relates to novel cellulytic enzyme preparations which are derivable from taxonomically specific phyli, classes, orders, families, genera, and species; e.g. from Basidiomycotous Hymenomycetes,   Zygomycota,   Chytridiomycota;   or from the classes Discomycetes,     Loculoascomycetes,   Plectomycetes; Archaeascomycetes,   Hemiascomycetes  or from the orders Di-aportales,   Xylariales,   Trichosphaeriales,   Phyllachorales; or from the families Nectriaeae,   Sordarlaceae, Chaetomiaceae,   CeratostomacBae,   Lasiosphaeriaceae;   or from the genera Cylindrocarpon,   Gliocladxum,   Volutella, Scytalldium,   Acremonium,   or from the species Fusarium lycopersici,   Fusarium passiflora,   Fusarium solani, Fusarium anguioides,~Fuc&amp;i!^ium poae,   Humicola nigrescens, Humicola grisea,   especially such consisting essentially of an enzyme comprising an amino acid sequence selected<br><br>
from the group consisting of the sequences Xaa Thr Arg Xaa Phe Asp Xaa 12   3   4   5   6   7; Xaa Thr Arg Xaa Tyr Asp Xaa 12   3   4   5   6   7; and<br>
Xaa Thr Arg Xaa Trp Asp Xaa<br>
12   3   4   5   6   7<br>
wherein, in position 4, Xaa is Trp, Tyr or Phe; and<br>
in position 1 and 7, Xaa is any of the 2 0 naturally<br>
occurring amino acid residues.<br>
More specifically, the enzyme preparation of the<br>
invention is preferably obtainable from the taxonomically<br>
specific phyli, classes, orders, families, genera, and<br>
species mentioned above which all produce endoglucanases<br>
comprising a first peptide consisting of 13 amino acid<br>
residues  harvi.ng the  followinig  sequenrie<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13<br>
and a second peptide oonsdsting of 5 amino acid residues<br>
having the following sequRnr.e<br>
Trp Cys Cys Xaa Cys<br>
12   3   4   5<br>
wherein, in position 3 of the first sequence, the amino<br>
acid is Trp, Tyr or Phe; in position 4 of the first<br>
sequence, the amino acid is Trp, Tyr or Phe; in position<br>
8 of the first sequence, the amino acid is Arg, Lys or<br>
His; in position 9, 10, and 12, respectively, of the<br>
first sequence, and in position 4 of the second sequence,<br>
the amino acid is any of the 2 0 naturally occurring amino<br>
acid residues.<br>
In yet further aspects, the present invention provides DNA constructs comprising a DNA sequence encoding an en¬zyme exhibiting endoglucanase activity, which DNA sequence comprises the DNA sequence shown in SEQ ID Nos.<br><br>
1, 4, 6, 8, 10, 12, 16, and 19, respectively, or analogues thereof.<br>
The present invention also relates to a recombinant expression vector comprising a DNA construct of the invention; to a cell comprising a DNA construct or a recombinant expression vector of the invention; to a method of producing an enzyme, e.g a recombinant enzyme, of the invention; to a method of providing colour clari¬fication of laundry by using the enzyme of the invention; to a laundry composition comprising the enzyme of the invention; to uses of the enzyme of the invention for degradation or modification of plant material, e.g. cell walls, for treatment of fabric, textile or garment, for treatment of paper pulp; and to an enzyme preparation which is enriched in an enzyme of the present invention.<br><br>
Accordingly, the present invention provides a method of producing an enzyme exhibiting endoglucanase activity, the method comprising culturing a cell having (a) a DNA construct comprising the DNA sequence shown in SEQ IS No.8, or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisisae DSM 10081, or (b) a DNA construct comprising an analogue of the DNA sequence shown in SEQ ID No.8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, which DNA sequence has at least 75% identity with the DNA sequence shown in SEQ ID No.8 or the DNA sequence obtainable from the plasmid in Sachharomyes cerevisiae DSM 10081, under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.<br>
With reference to the accompanying drawings, in which.<br>
Figure 1 is an alignment of the deduced encoded amino acid sequences of Acremonium sp. (I), Volutella coUetotrichoides, Crinipellis scabella, Acremonium sp. (II), Myceliophthora thermophila, Thielavia terrestris, Macrophomina phaseolina. The Pileup program (Feng and Doolittle, 1987) (GCG package, version 8.0) was used to create the best alignment. Identical residues in at least four sequences (boxed) are indicated around the corresponding amino acids.<br>
Figure 2<br>
Figure 2a,b,c illustrates the taxonomic classification within the Fungal Kingdom of all the microorganisms disclosed herein as being capable of producing said enzyme preparations and enzymes of the invention.<br>
The taxonomic classification used herein builds primarily	.<br><br>
on the system used in the :NIH Data Base (Entrez, version<br>
spring 1996) available on World Wide Web:<br>
(http://www3.ncbi.nlm.nih.gov/htbin/ef/entrezTAX).<br>
Regarding classification of organisms which are not<br>
included in the Entrez data base the following generally<br>
available and world wide accepted reference books have<br>
been used:<br>
For Ascomycetes:   Eriksson, O.E. &amp; Hawksworth, D.L.:<br>
Systema Ascomycetum vol 12 (1993).<br>
For Basidiomycetes:   Jiilich, W, : Higher Taxa of<br>
Basidiomycetes, Bibliotheca Mycologia 85, 485pp (1981).<br>
For Zygomycetes:   O'Donnell, K.: Zygomycetes in culture.<br>
University of Georgia, US,  257pp (1979) .<br>
General  mycological reference books:<br>
Hawksworth, D.L., Kirk, P.M., Sutton, B.C. and Pegler,<br>
D.N.: Dictionary of the fungi. International Mycological<br>
Institute, 616pp (1995);<br>
Von Arx, J.A. : The genera of fungi sporula1:ing in<br>
culture, 424pp (1981).<br>
The taxonomic implacement of the genus Humicola has until! recently remaii^ad unclecir. However, studies of 18SRNA of a wide selection of Sordariales has given strong indications of referring Humicola to the order Sordariales (Taylor, Clausen &amp; Oxenb0ll, unpublished). Further these data suggests Humicola along with Scytalid-ium to be only rather distantly related to the families Sordariaceae, Chaetomiaceae, Ceratostomataceae, and Lasiosphaeriaceae. In accordance with the above Humicola and Scytalidium are here placed within the order Sordariales, with unclassified Family.<br>
Figure 3 is an alignment of the deduced partial amino<br>
acid sequences derived from a selection of 26 of the 46<br>
"microorganisms described in Example 5.      	'<br><br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In the present context, the term "the 20 naturally occuring amino acid residues" denotes the 20 amino acid residues usually found in proteins and conventionally known as alanine (Ala or A), valine (Val or V), leucine (Leu or L), isoleucine (lie or I), proline (Pro or P), phenylalanine (Phe or F), tryptophan (Trp or W), methionine (Met or M), glycine (Gly or G), serine (Ser or S), threonine (Thr or T), cysteine (Cys or C), tyrosine (Tyr or Y), asparagine (Asn or N), glutamine (Gin or Q), aspartic acid (Asp or D), glutamic acid (Glu or E), lysine (Lys or K), arginine (Arg or R), and histidine (His or H),<br>
According to the present invention there is provided novel well—performing endoglucanases comprising conserved amino acid sequence regions, especially a first amino acid sequence consisting of 14 amino acid residues having the following sequence<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp Xaa 1 ' 2   3   4   5   6   7   8   9   10  11  12  13  14 and a second amino acid sequence consisting of 5 amino acid residues having the following sequence<br>
Trp Cys Cys Xaa Cys<br>
12   3   4   5<br>
wherein,<br>
in position 3 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in position 4 of the first sequence, the amino acid is<br>
Trp, Tyr or Phe;<br>
in -position 8 of the first sequence, the amino acTd is<br>
Arg, Lys or His;<br>
in position 9, 10, 12 and 14, respectively, of the first<br><br>
sequence, and in position 4 of the second sequence, the amino acid is any of the 20 naturally occurring amino acid residues with the provisos that, in the first amino acid sequence, (i) when the amino residue in position 12 is Ser, then the amino acid residue in position 14 is not Ser, and (ii) when the amino residue in position 12 is Gly, then the amino acid residue in position 14 is not Ala.<br>
Preferably, the enzyme of the invention is of microbial origin, i.e. obtainable from a microorganism such as a fungus.<br>
In a preferred embodiment, the amino acid residue in position 9 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, aspaxagine, glutramine, tyrosine, serine, raethionin^e and tryptophan, preferably from the group consisting of proline and threonine.<br>
In anot±ier preferred embodiment, tiie amino acid residue in position 10 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine,' leucine, Isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, preferably serine.<br>
In yet another preferred embodiment, the amino acid resi¬due in position 12 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cy¬steine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, preferably from the group con-si^tlhg~"of"alanine and glycine.<br>
In yet another preferred embodiment, the amino acid resi-<br><br>
due in position 14 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cy¬steine, asparagine, glutamine, tyrosine, serine, methionine, tryptophan, glutamic acid and aspartic acid, preferably from the group consisting of proline, threonine, serine, alanine, glutamic acid and aspartic acid.<br>
Preferably, the amino acid residue in position 4 of the second sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine, tryptophan, glutamic acid and aspartic acid, more preferably from the group con¬sisting of alanine, glycine, and glutamine.<br>
Examples of more prefenred embodiments are such wherein, in the first sequence, the amino acid residue in position 3 is tyrosine; or the amino acid residue in position 4 is tryptophaTi; or the amino acid residue in position 8 is lysine.<br>
In an especially preferred embodiment, the enzyme 'of the<br>
invention has a first sequence comprising the amino acid<br>
sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Pro Ser Cys Ala Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  ,<br>
or the amino acid sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Thr Ser Cys Ala Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  ,<br>
or the amino acid sequence<br>
Thr Arg Tyr Trp Asp Cys Cys Lys Pro Ser Cys Gly Trp<br>
1   2   3   4   5   6   7   8   9   10  11  12  13  .<br>
	—	      	r<br>
In a second aspect, the present invention provides a method for providing a microbial strain comprising a gene<br><br>
encoding such an enzyme which method comprises hybridization, e.g. PCR amplification, under standard conditions with an oligonucleotide derived from any of the conserved regions, illustrated in Fig.l.<br>
A useful oligonucleotide comprises a nucleotide sequence encoding at least a pentapeptide comprised in a peptide selected from the group consisting of a.<br>
Thr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp Xaa 1   2   3   4   5   6   7   8   9   10  11  12  13  14 the amino acid in position 3 or 4 being Trp, Tyr or Phe; the amino acid in postion 8 being Arg, Lys or His; he amino acid in position 9, 10, 12 and 14, respectively, being any of the 20 naturally occurring amino acid resi¬dues ; and b.<br>
Trp Cys Cys Xaa Cys Tyr 12   3   4   5   6<br>
the amino acid in position 4 being any of the 2 0 natural¬ly occurring amino acid residues ; and c.<br>
Xaa Pro Gly Gly Gly Xaa Gly Xaa Phe 12345   6   789* the amino acid in position l being Met or lie; the amino acid in position 6 and 8, respectively, being Leu, lie or Val; and d.<br>
Gly Cys Xaa Xaa Arg Xaa Asp Trp Xaa 123   4   56789<br>
the amino acid in position 3 being any of the 2 0 natural¬ly occurring amino acid residues;<br>
the amino acid in position 4 and 6, respectively, being Trp, Tyr or Phe; and the amino acid in position 9 being Pho or Met;<br>
The useful oligonucleotides also comprises nucleotide<br><br>
sequences complementary to the sequences mentioned.<br>
In a preferred embodiment of the method of the invention, the oligonucleotide corresponds to a PCR primer selected from the PCR primers sense:<br>
5 ' -CCCCAAGCTTACI'^/cGITA'^/-rTGGGA'^/TTG^/TTGVTAA'^/G'^/cC-3 '<br>
antisense 1:<br>
5'- CTAGTCTAGATA^/GCAIGC^/GCA^/GCACC -3 ' ;<br>
antisense 2:<br>
CTAGTCTAGAAAIA^/G/'^ICCIA^/^/°ICCICCICCIGG -3 ' ; and<br>
antisense 3:<br>
5 ' - CTAGTCTAGAIAACCA'^/GTCA^/G'^/TAIC°/TCC-3 .<br>
In a third aspect, the present invention provides an enzyme preparation which essentially consists of an enzyme having cellulytic activity and having the conserved regions found by the inventors, i.e. which com¬prises a peptide consisting of 7 amino acid residues ha¬ving the following sequence Xaa Thr Arg Xaa Phe Asp Xaa 12   3   4   5   6   7; Xaa Thr Arg Xaa Tyr Asp Xaa 1   2   3   4   5   6   7 " ; and Xaa Thr Arg Xaa Trp Asp Xaa 12   3   4   5   6   7<br>
wherein, in position 4, Xaa is Trp, Tyr or Phe; and in position 1 and 7, Xaa is any of the 2 0 naturally occurring amino acid residues.<br>
This enzyme is obtainable from a strain belonging to Basidiomycotous Hymenomycetes     (see Fig.2 ), more preferably to the group consisting of the orders Agarica-les,   Auriculariales,   and Aphyllophorales,   even more preferably to the group consisting of the families Exidiaceae,   Tricholomataceae,   Coprinaceae, Schizophyllaceae,   Bjerkanderaceae  and Polyporaceae,<br><br>
especially to the group consisting of the genera Exidia, Crinipellis,   Fames,   Panaeolus,   Trametes,   Schizophyllum, and Spongipellis.<br>
Specific examples are endoglucanases obtainable from a strain belonging to the group consisting of the species Exidia  glandulosa,   Crinipellis scabella,   Fomes fomentarius,   and Spongipellis  sp.,   more specific examples being Exidia  glandulosa,   CBS 277.96, Crinipellis scabella,   CBS 280.96, Fomes  fomentarius,   CBS 276.96, and Spongipellis  sp.,   CBS 283.96.<br>
Exidia  glandulosa  was deposited at Centraalbureau voor Schimmelcultures, Oosterstraat 1, Postbus 273, NL-3740 AG Baarn, the Netherlands, on 12 March, 1996, under the deposition number CBS 277.96; Crinipellis  scabella  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under th=e deposition number CBS 2 8 0.96, Fomes fomentarius  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 276.9^, and "Spongif^llis sp.   was deposited at Centraalbureau voor Schimmelcultures on 12 March, 199 6, under the deposition number CBS 283.96; all deposited under the Budapest Treaty.<br>
The enzyme preparation of the invention is also obtain¬able from a strain belonging to Chytridiomycota, preferably from a strain belonging to the class of Chytridiomycetes,   more preferably belonging to the group consisting of the order Spizellomycetales   , even more preferably to the family Spizellomycetaceae.,   especially belonging to the genus Rhizophlyctis.   A specific example is a strain belonging to the species Rhizophlyctis rosea, more specifically to Rhizophlyctis rosea,     CBS 282.96.<br>
Rhizophlyctis rosea  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition<br><br>
number CBS 2 8 2.96; under the Budapest Treaty.<br>
The enzyme preparation of the invention is also obtai able from a strain belonging to Zygomycota,   preferabl belonging to the class Zygomycetes,   more preferably t the order Mucorales,   even more preferably to the grou families consisting of Mucoraceae and Thamnidiaceae, especially belonging to the group consisting of the gi nera Rhizomucor,   Phycomyces and Chaetostylum.   Specifii examples are strains belonging to the genera Rhizomui pusillus,     Phycomyces nitens,   and Chaetostylum fresen. more specifically to Rhizomucor pusillus^   IFO 4578, ai Phycomyces nitens,   IFO 4814 and Chaetostylum fresenli NRRL 2 3 05.<br>
Further, the enzyme preparation of the invention is a'. obtainable from a strain belonging to the group consis irrg of ArctraeascoTnycetes, Discomycetes,   Hemiascomyceti Loculoascomycetes,   and Plectomycetes,   preferably belonging to the group consisting of the orders Pezizales,   Rhytismatales,   Dothideales,   and Surotiales. Especially, the enzyme is obtainable fxom a strain belonging the the group consisting of the families CucurJbitariaceae, Ascobolaceae,   Rhytismataceae,     and Trichocomaceae,   preferably belonging the the group cor sisting of the genera Diplodia,   Microsphaeropsis, Ulospora,     Macrophomina,   Ascobolus,   Saccobolus,     Penlc Hum,   and Thermomyces.   Specific examples are enzymes obtainable from a strain belonging the the group consi ing of the species Diplodia gossypina,   Microsphaeropsi sp.,   Ulospora bilgramii,   Aureobasidium sp.,   Macrophomi phaseolina,   Ascobolus  stictoides,   Saccobolus  dilutellv Peziza,   Penicillium verruculosum,   Penicillium chrysogenum,   and Thermomyces  verrucosus;  more specifically Diplodia gossypina,   CBS 274.96,   Ulospora bilgramii,   NKBC 1444,   Macrophomina phaseolina,   CBS 281.96,   Saccobolus  dilutellus,   CBS 275.96,   Penicilliun<br><br>
verruculosum^   ATCC 62396,   Penicillium chrysogenum,   ATCC 9480, and Thermomyces  verrucosus,   CBS 285.96.<br>
Diplodia  gossyplna  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 274.96, Macrophoraina phaseolina  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 281.96, Saccobolus dilutellus  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 199 6, under the deposition number  CBS 275.96; Thermomyces  verrucosus  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 285.96; all under the Budapest Treaty.<br>
Yet further, the enzyme is obtainable from a strain belonging to the group consisting of the orders Dia-portales,   Xylariales,   Trichosphaeriales  and Phyllachorales,   preferably from a strain belonging to the group consisting of the families Xylariaceae,   Valsaceae, and Phyllachoracese,   more preferably belonging to the ge¬nera Diaporthe,   ColletJDt.ricimm,   Slgrospora,   Xylaria, Nodulispojrum  and Poronla.   Specific escaraples are the species Diaporthe 'syngenesia,   Colletotrichum lagenarium, Xylaria hypoxyIon,  Nigrospora sp.,  Nodulisporum sp.,   and Poronia punctata,   more specifically Diaporthe  syngenesia, CBS 278.96,   Colletotrichum lagenarium,   ATCC 52609, Nigrospora sp.,   CBS 272.96,   Xylaria hypoxylon,   CBS 284.96.<br>
Diaporthe  syngenesia  was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 278.96, Nigrospora  sp.   wa;s deposited at Centraalbureau voor Schimmelcultures on 12 March, 199 6, under the deposition number CBS 272.96, Xylaria hypoxylon was deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition number CBS 284.96;<br><br>
all under the Budapest Treaty,<br>
The enzyme is also obtainable from the unidentified fungal, mitosporic, coleomycetous deposited at Centraalbureau voor Schimmelcultures on 12 March, 1996, under the deposition numbers CBS 270.96, CBS 271.96 and CBS 273.96, respectively, under the Budapest Treaty.<br>
The enzyme is also obtainable from a strain belonging to the group consisting of the genera Cylindrocarpon^ Gliocladium,   Nectria,   Volutella,   Sordaria,   Scytalidium, Thielavia,   Syspastospora,   Cladorrhinum,   Chaetomium,   Myce-liphthora  and Acremoninm,   especially from a strain belon¬ging to the group consisting of the species Cylin-drocarpon  sp.,   Nectria pinea,   Volutella colletotrichoides,   Sordaria fimicola,   Sordaria macrospora^   Thielavia  terrestris,   Thielavia  thermophila, Syspastospora boninensis,   Cladorrhinvrm t^oecxmdissimam, Chaetomium murorum,   Chaetomixxm vxrescens,   Chaetomium brasiliensis,   Chaetomium cunicolorum,   Myceliophthora thermophila,   Gliocladium  ca tanuJ a turn,   Scytalidium thenaophila,   and Acremonium sp. ,   more specifically from Nectria pinea,   CBS 279.96,   Volutella  colletotrichoides, CBS 400.58,   Sordaria  fimicola,   ATCC 52644,   Sordaria macrospora,   ATCC  60255,   Thielavia   terrestris,   NRRL  8126, Thielavia  thermophila,   CCBS 174.70,   Chaetomium murorum, CBS 163.52,   Chaetomium  virescens,   CBS 547.75,   Chaetomium brasiliensis,   CBS 122.65,   Chaetomium  cunicolorum,   CBS 799.83,   Syspastospora boninensis,   NKBC 1515,   Cladorrhinum foecundissimum,   ATCC 62373, Myceliophthora  thermophila, CBS 117.65, Scytalidium  th-ermoph±la,   ATCC 28085, Gliocladium catenulatum, ATCC 10523, and Acremonium sp., CBS 478.94.<br>
Nectria pinea  was deposited aTt Centraalbureau voor Schimmelcultures on 12 March, 1996, under the depositioh number CBS 279.96, and Acremonium  sp.   was deposited on<br><br>
28 September 1994 under the deposition number CBS 478,94, both according to the Budapest Treaty.<br>
The enzyme is also obtainable from a strain belonging to the group consisting of the species Fusarium solani, Fusarium anguioides,   Fusarium poae,   Fusarium oxysporum ssp. lycopersici,  Fusarium oxysporum ssp.  passiflora, Humicola nigrescens  and Humicola  grisea,   especially Fusarium oxysporum ssp lycopersici,   CBS  645.78,   Fusarium oxysporum ssp passiflora,   CBS  744.79,   Fusarium solani, IMI 107.511,   Fusarium anguioides,   IFO 4467,   Fusarium poae,   ATCC 60883,   Humicola  nigrescens,   CBS 819.73 and Humicola  grisea,   ATCC 22726. It is to be noted that Hum.i-ziola grisea  is different from Humicola grisea var. thermoidea.<br>
[n a preferred embodiment, the enz&gt;Tne preparation of the Invention is derived from the disclosed classes, orders, families, genera and species and essentially consists of in enzyme comprising a first peptide consisting of 13 imino acid residiies having the following sequence<br>
Chr Arg Xaa Xaa Asp Cys Cys Xaa Xaa Xaa Cys Xaa Trp<br>
L   2   3   4   5   6   7   8   9   lb  11  12  13<br>
ind a second peptide consisting of 5 amino acid residues laving the following sequence<br>
?rp Cys Cys Xaa Cys 2   3   4   5<br>
therein, in position 3 of the first sequence, the amino icid is Trp, Tyr or Phe; in position 4 of the first sequence, the amino acid is Trp, Tyr or Phe; in position 1 of the first sequence, the amino acid is "Arg, Lys or [is; in position 9, 10, and 12, respectively, of the irst sequence, and in position 4 of the second sequence.<br><br>
the amino acid is any of the 20 naturally occurring amino acid residues.<br>
Preferably, the amino acid residue in position 9 of the first sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, more preferably from the group consisting of proline and threonine; the amino acid residue in position 10 of the first sequence which is selected from the group consist¬ing of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, pireferably serine; the amino acid residue in position 12 of the first sequence is selected from the group consist¬ing of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine and tryptophan, preferably from the group consistiing o:f alanine and glycine; and the amino acid residue in p.ositrLon 4 of the second sequence is selected from the group consisting of proline, threonine, valine, alanine, leucine, isoleucine, phenylalanine, glycine, cysteine, asparagine, glutamine, tyrosine, serine, methionine, tryptophan, glutamic acid and aspartic acid, more preferably from the group con¬sisting of alanine, glycine, and glutamine.<br>
In further aspects, the present invention provides a DNA construct comprising a DNA sequence encoding an enzyme exhibiting endoglucanase^ activity, which DNA sequence comprises<br>
a) the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevislae DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia<br><br>
coli,   DSM 10512, DSM 10511, DSM 10571, DSM 1057 6, respectively; or<br>
b) an analogue of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia   coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, which<br>
i)    is homologous with the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae  DSM 977 0, DSM 10082, DSM 10080, DSM 10081, Escherichia coli, DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, ii)   hybridizes with the same oligonucleotide probe as the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the.  DWA sequ¬ence obtainable from the plasmid in Saccharomyces cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 105X2, DSM 10511, DSM 10571, DSM 10576, respectively, iii)  encodes a polypeptide which is hcaaologous with the polypeptide encoded by a DNA sequence comprising the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequ¬ence obtainable from the plasmid in Saccharomyces cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, iv)   encodes a polypeptide which is immunologically re¬active with an antibody raised against the purified endoglucanase encoded by the DNA sequence shown in SEQ ID No 1, 4, 6, 8, 10, 12, 16, or 19, respectively, or the DNA sequence—obtainable from the plasmid in Saccharomyces  cerevisiae  DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,<br><br>
DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively.<br>
Escherichia  coli  DSM 10512 was deposited under the Buda¬pest Treaty on 2 February, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10511 was deposited under the Buda¬pest Treaty on 2 February, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-3812 4 Braunschweig, Germany).<br>
Escherichia coli  DSM 10571 was deposited xrnder the Buda¬pest Treaty on 6 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10576 was deposited under the Buda¬pest Treaty on 12 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkultxiren GmbH, Mascheroder Weg 16, D—38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10583 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10584 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10585 was deposited under the Buda-peet Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br><br>
Escherichia  coli  DSM 10586 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganisxnen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10587 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Escherichia  coli  DSM 10588 was deposited under the Buda¬pest Treaty on 13 March, 1996, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccharomyces  cerevisia^  DSM 9770 was deposited under the Budapest Treaty on 24 February, 1995, at DSm   (Deutsche Samcmlung von Mikroorganismien und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Sacchajromyces  aerevisla^  DSM 10082 was deposited under the Budapest Trea^ty on 3 0 June, 1995, at D^I (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccharomyces  cerevisiae  DSM 1008 0 was deposited under the Budapest Treaty on 30 June, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany).<br>
Saccharomyces  cerevisLae  DSM 10081 was deposited under the Budapest Treaty on 30 June, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-3 8124 Braunschweig, Germany).<br>
The DNA construct of the invention relating to SEQ ID No. 1 can be isolated from or produced on the basis of a DNA<br><br>
library of a strain of Myceliophthora,   in particular a strain of M.   thermophila,   especially M. thermophila,   CBS 117.65.<br>
The DNA constructs of the invention relating to SEQ ID Nos. 4 and 6 can be isolated from or produced on the basis of a DNA library of a strain of Acremonium, especially Acremonium sp.,   CBS 478.94.<br>
The DNA construct of the invention relating to SEQ ID No. 8 can be isolated from or produced on the basis of a DNA library of a strain of Thielavia   in particular a strain of Thielavia  terrestris,   especially Thielavia  terrestris, NRRL 8126.<br>
The DNA construct of the invention relating to SEQ ID No. 10 can be isolated from or produced on the basis of a DNA library of a strain of Macrophomina,   in paarticular a strain of M.   phaseolina,   especially M.phaseolina,   CBS 281.96.<br>
The DNA coistruct of the invention relating to SEQ ID No. 12 can be isolated from or produced on the basis of a DNA library of a strain of Crinipellis^   in particular a strain of C. scabella,   especially C.scabella,   CBS 280.96.<br>
The DNA construct of the invention relating to SEQ ID No. 19 can be isolated from or produced on the basis of a DNA library of a strain of Sordaria,   in particular a strain of Sordaria fimicola.<br>
In the present context, the "analogue" of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, is intended to indicate any DNA sequence.encoding an enzyme exhibiting endoglucanase activity, which has any or all of the properties i)-iv).'<br><br>
The analogous DNA sequence<br>
a)	may be isolated from another or related (e.g. the same) organism producing the enzyme with endoglucanase activity on the basis of the DNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, e.g. using the procedures described herein; the homologue may be an allelic variant of the DNA sequence comprising the DNA sequences shown herein, i.e. an alternative form of a gene that arises through mutation; mutations can be silent (no change in the encoded enzyme) or may encode enzymes having altered amino acid sequence; the homologue of the present DNA sequence may also be a genus or spe¬cies homologue, i.e. encoding an enzyme with a similar activity derived from another species,<br>
b)	may be constructed on the basis of the DNA sequences shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, e.g. by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the endog 1 ucama se enicoded by the DNA sequence, but which correspond t.o the codon usage of the host organism intended for prcaiuction of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. However, in the latter case amino acid changes are preferably of a minor nature, that is conservative amino acid substitu¬tions that do not significantly affect the folding or activity of the protein, small deletions, typically of one to about 3 0 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 2 0-25 residues, or a small extension that facilitates purifica¬tion, such as a poly-histidine tract, an antigenic epitope or a binding domain.-See in general Ford et al.. Protein Expression and Purification 2.: 95-107,. 1991. Examples of conservative substitutions are within the<br><br>
group of basic amino acids (such as arginine, lysine, histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine, valine), aromatic amino acids (such as phenylalanine, tryptophan, tyrosine) and small amino acids (such as glycine, alanine, serine, threonine, methionine).<br>
It will be apparent to persons skilled in the art that such substitutions can be made outside the regions criti¬cal to the function of the molecule and still result in an active polypeptide. Amijio acids essential to the activity of the polypeptide encoded by the DNA construct of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning miatagenesis (CunninghaTti and Wells, Science 244. 1081-1085, 1989). In the  latter technique mutations are introduced at every residue in the mol¬ecule, and the resultant mutant molecules are tested for biological (i.e. endoglucanase) activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic reson¬ance, crystallography or photoaffinity labeling. See, for example, de Vos et al.. Science 255: 306-312, 1992; Smith et al., J. Mol. Biol. 224; 899-904, 1992; Wlodaver et al., FEES Lett. 3 09: 59-64, 1992.<br>
The endoglucanase encoded by the DNA sequence of the DNA construct of the invention may comprise a cellulose bind¬ing domain (CBD) existing as an integral part of the encoded enzyme, or a CBD from another origin may be introduced into the endoglucanase enzyme thus creating an enzyme hybride. In this context, the term "cellulose-<br><br>
binding domain" is intended to be understood as defined by Peter Tomme et al. "Cellulose-Binding Domains: Classi¬fication and Properties" in "Enzymatic Degradation of Insoluble Carbohydrates", John N. Saddler and Michael H. Penner (Eds.), ACS Symposium Series, No. 618, 1996. This definition classifies more than 120 cellulose-binding domains (CBDs) into 10 families (I-X), and it demon¬strates that CBDs are found in various enzymes such as cellulases, xylanases, mannanases, arabinofuranosidases, acetyl esterases and chitinases. CBDs have also been found in algae, e.g., the red alga Porphyra purpurea as a non-hydrolytic polysaccharide-binding protein, for refer¬ence see Peter Tomme et al., supra. However, most of the CBDs axe from cellulases and xylanases. CBDs are found at the N or C teirmini of proteins or are internal. Enzyme hybrids are known in the art, see e.g. WO 90/00609 and WO 95/16782, and may be prepared by transforming into a host cell a DNA construct comprising at least a fragment of DNA encoding the cellulose-binding domain ligated, with or without a linker, to a DNA sequence encoding the enzyme of interest and growing the host cell to express the fused gene. Enzyme hybrids may be described by the following formula:<br><br>
wherein CBD is the N-terminal or the C-terminal region of an amino acid sequence corresponding to at least the cellulose-binding domain; MR is the middle region (the linker), and may be a bond, or a short linking group preferably of from about 2 to about 100 carbon atoms, more preferably of from 2 to 40 carbon atoms; or is preferably from about 2 to about 100 amino acids, more preferably of from 2 to 40 amino acids; and X is an N-terminal or C-terminal region of a polypeptide encoded by the DNA sequence of the invention.<br><br>
The homology referred to in i) above is determined as the degree of identity between the two sequences indicating a derivation of the first sequence from the second. The homology may suitably be determined by means of computer programs known in the art such as GAP provided in the GCG program package (Needleman, S.B. and Wunsch, CD., Jour¬nal   of Molecular Biology,   48.: 443-453, 1970). Using GAP with the following settings for DNA sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3, the coding region of the DNA sequence exhibits a degree of identity preferably of at least 50%, more preferably at least 65%, more preferably at least 70%, even moxe preferably at least 80%, especially at least 90%, with the coding region of the DNA sequence shown in SEQ ID No.l, 4, 6, 8, 10, 12, or 16, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevislae,   DSM 3770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571, ox DSM 10576, respect ively.<br>
The hybridization referred to in ii) above is intended to indicarte that the analogous DNA sequence hybridizes to the same probe as tJie DNA sequence encoding ttie endoglucanase enzyme under certain specified conditions which are described in detail in the Materials and Methods section hereinafter. The oligonucleotide probe to be used is the DNA sequence corresponding to the endoglucanase encoding part of the DNA sequence shown in SEQ ID NO 1, 4, 6, 8, 10, 12, or 16  respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
The homology" referred to in iii) above is determined as the degree of identity between the two sequences indicat¬ing a derivation of the first sequence from the second.<br><br>
The homology may suitably be determined by means of com¬puter programs known in the art such as GAP provided in the GCG program package (Needleman, S.B. and Wunsch, CD,, Journal   of Molecular Biology,   48_:   443-453, 1970). Using GAP with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1, the polypeptide encoded by an analogous DNA sequence exhibits a degree of identity pre¬ferably of at least 55%, more preferably at least 60%, more preferably at least  65%, even more preferably at least 70%, more preferably at^least 80%, especially at least 90%, with the enzyme encoded by a DNA construct comprising the DNA sequence shown in SEQ ID No.l, 4, 6, 8, 10, 12, 16 or 19, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae, DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respect ive1y.<br>
In connection with property iv) above it is intended to indicate an endoglucanase encoded by a DNA sequence iso¬lated from strain Saccharomyces  cerevisiae, DSM BllQ,   DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively, and produced in a host organism transformed with said DNA sequence or the corresponding endoglucanase naturally produced by Myceliophthora  thermophila,   Acremonium sp., Thielavia  terrestris,   Macrophomina phaseolina, Crinipellis scabella,   Volutella  colletotrichoides,   or Sordaria fimicola,   respectively. The immunological reac¬tivity may be determined by the method described in the Materials and Methods section below.<br>
In further aspects the invention relates to an expression vector Harbour ing a DNA construct of the invention, a cell comprising the DNA construct or expression vector and a method of producing an enzyme exhibiting<br><br>
endoglucanase activity which method comprises culturing said cell under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.<br>
In a still further aspect the invention relates to an enzyme exhibiting endoglucanase activity, which enzyme<br>
a)	is encoded by a DNA construct of the invention<br>
b)	produced by the method of the invention, and/or<br>
c)	is immunologically reactive with an antibody raised against a purified endoglucanase encoded by the DNA sequence shown in SEQ ID No.l, 4, 6, 8, 10, 12, or 16, respectively, or the DNA sequence obtainable from the plasmid in Saccharomyces  cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively.<br>
The endoglucanase mentioned in c) above may be encoded by the DNA sequence isolated from the strain Saccharomyces cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, Escherichia  coli,   DSM 10512, DSM 10511, DSM 10571 or DSM 10576, reispEtrtively, and produced in a host organism transfonffi^ with, said DNA. sequence ox tbe corresponding endoglucanase naturally produced by Myceliophthora thermophila,   Acremonium sp.,   Thielavia  terrestris, Macrophomina phaseolina,   Crinipellis  scahella,   Volutella colletotrichoides  or Sordaria  fimicola,   respectively.<br>
Generally, in the present context the term "enzyme" is understood to include a mature protein or a precursor form thereof as well to a functional fragment thereof which essentially has the activity of the full-length enzyme. Furthermore, the term "enzyme" is intended to include homologues of said enzyme.<br>
Homologues of the present enzyme may have one or more amino acid substitutions, deletions or additions. These changes are preferably of a minor nature, that is conser-<br><br>
vative amino acid substitutions that do not significantly affect the folding or activity of the protein, small deletions, typically of one to about 3 0 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue, a small linker peptide of up to about 2 0-25 residues, or a small extension that facilitates purification, such as a poly-histidine tract, an antigenic epitope or a binding domain. See in general Ford et al. , Protein Expression and Purification 2.: 95-107, 1991. Examples of conservative substitutions are within the group of basic amino acids (such as arginine, lysine, histidine), acidic amino acids (such as glutamic acid and aspartic acid), polar amino acids (such as glutamine and asparagine), hydrophobic amino acids (such as leucine, isoleucine, valine), aromatic amino acids (such as phenylalanine, tryptophan, tyrosine) and small cusino acids (such as glycine, alanine, serine, threonine, methionine).<br>
It will be apparent to persons skiU-ed in the art that such substitutions can be made outside the regions criti¬cal to the function of the aoXecule and still result in an active enzyme. Amino acids essential to the activity of the enzyme of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham, 1989). In the  latter technique mutations are introduced at every residue in the molecule, and the resultant rautaiTt molecules are tested for cellulytic activity to identify amino acid residues that are critical to the activity of the molecule. Sites of ligand-receptor interaction can also be determined by analysis of crystal structure as determined by such techniques as nuclear magnetic resonance, crystal logr&amp;phy or photoaffinity labelling. See, for example, de Vos et al., 1992; Smith et al., 1992. Wlodaver et al., 1992.<br><br>
The homologue may be an allelic variant, i.e. an alterna¬tive form of a gene that arises through mutation, or an altered enzyme encoded by the mutated gene, but having substantially the same activity as the enzyme of the invention. Hence mutations can be silent (no change in the encoded enzyme) or may encode enzymes having altered amino acid sequence.<br>
The homologue of the present enzyme may also be a genus or species homologue, i.e. an enzyme with a similar activity derived from another species.<br>
A homologue of the enzyme may be isolated by  using the procedures described herein.<br>
Molecular screening and cloning by polymerase chain reaction (PCR)<br>
MoleculaLr screening for DNA sequences of the invention may "be carried out by polyTiierase chain react ioxi (PCR) using genomic DNA or double-stranded ca3HA isolated from a suitable source, such as any of the herein mentioned organisms, and synthetic oligonucleotide primers prepared on the basis of the DNA sequences or the amino acid sequences disclosed herein. For instance, suitable oligonucleotide primers may be the primers described in the Materials and Methods section.<br>
In accordance with well-known procedures, the PCR fragment generated in the molecular screening may be isolated and subcloned into a suitable vector. The PCR fragment may be used for screening DNA libraries by e.g. colony or plaque hybridization.<br><br>
Expression cloning in yeast<br>
The DNA sequence of the invention encoding an enzyme exhibiting endoglucanase activity may be isolated by a general method involving<br>
cloning, in suitable vectors, a DNA library from a<br>
suitable source, such as any of the herein<br>
mentioned organisms<br>
transforming suitable yeast host cells with said<br>
vectors,<br>
culturing the host cells under suitable conditions<br>
to express any enzyme of interest encoded by ci<br>
clone in the DNA library,<br>
screeTi±ng for positive clones by determining any<br>
endoglucanase activity of the enzyme produced by<br>
such clones, and<br>
isolating the enzyme encoding DNA from such<br>
clones.<br>
The general method is further disclosed in WO	94/14953<br>
the contents of which are hereby incorporated	by<br>
reference. A more detajLled description of tiie	screening<br>
method is given in Example 1 below.<br>
The DNA sequence coding for the enzyme may for instance be isolated by screening a cDNA library of Macrophomina phaseolina,   Crinipellis scabella,   Sordaria fimicola  or Volutella  colletotrichoides,   and selecting for clones ex¬pressing the appropriate enzyme activity (i.e. endoglucanase activity) or from Escherichia  coli  DSM :10512 deposited under the Budapest Treaty on 2 February, 199 6, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany), or from Escherichia coli  DSM 10511 deposited under the Budapest Treaty on 2 February, 1996, at DSM, or  from Escherichia coli DSM 10576, deposited under the Budapest Treaty on 12 March, 1996, at<br><br>
DSM; or  from Escherichia  coli  DSM 10571 deposited under the Budapest Treaty on 6 March, 1996, at DSM; or by screening a cDNA library of Myceliphthora  thermophila, CBS 117.65, Acremonium sp.,   CBS 478.94, or Thielavia terrestris,   NRRL 8126, and selecting for clones expres¬sing the appropriate enzyme activity (i.e. endoglucanase activity) or from Saccharomyces  cerevisiae  DSM 9770 deposited under the Budapest Treaty on 24 February, 1995, at DSM (Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 16, D-38124 Braunschweig, Germany), or from Saccharomyces  cerevisiae DSM 10082 deposited under the Budapest Treaty on 30 June, 1995, at DSM, from Saccharomyces  cerevisiae  DSM 10080 deposited under the Budapest Treaty on 30 June, 1995, or from S^ccharomycGs cerevisiae  DSM 10081 deposited under the Budapest Treaty on 3 0 June, 1995, at DSM. The appropriate DNA sequence may then be isolated from the clone by standard procedures, e.g. as described in EXcimple 1.<br>
Nucleic acjjd construct<br>
As used herein the term "nucleic acid construct" is intended to indicate any nucleic acid molecule of cDNA, genomic DNA, synthetic DNA or RNA origin. The term "construct" is intended to indicate a nucleic acid segment which may be single- or double-stranded, and which may be based on a complete or partial naturally occurring nucleotide sequence encoding an enzyme of interest. The construct may optionally contain other nucleic acid segments.<br>
The nucleic acid construct encoding the enzyme of the invention may suitably be of genomic_jQLE_cDNA origin, for instance obtained by preparing a genomic or cDNA library and screening for DNA sequences coding for all or part of<br><br>
the enzyme by hybridization using synthetic oligonucleotide probes in accordance with standard techniques (cf. Sambrook et al., 1989).<br>
The nucleic acid construct encoding the enzyme may also be prepared synthetically by established standard methods, e.g. the phosphoamidite method described by Beaucage and Caruthers, (1981), or the method described by Matthes et al., (1984). According to the phospho¬amidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in suitable vectors.<br>
Furthermore, the nucleic acid construct may be of mixed synthetic and genomic, mixed synthetic and cDNA or mixed genomic and cDNA origin prepared by ligating fragments of synthetic, genomic or cDNA origin (as appropriate), the fragmeiits cxarrespanding to various parts of the entire nucleic acid constract, in accordance with standard tech¬niques.<br>
The nucleic ac±d construct loay also be prepared by polymerase chain reaction using specific primers, for instance as. described in US 4,683,202 or Saiki et al., (1988) .<br>
The nucleic acid construct is preferably a DNA construct which term will be used exclusively in this specification and claims.<br>
Recombinant, vector<br>
A recombinant vector comprising a DNA construct encoding<br>
the enzyme of the invention may be any vector which may^ 	<br>
conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host<br><br>
cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA sequence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DFA. In general, the expression vector is derived from plasmid or viral DNA, or may contain element:s of both. The term, "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The proiaoter may be any DNA sequence which sihows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for use in yeast host cells include promoters from yeast glycolytic genes (Hitzeman et al., J. Biol. Chem. 255 (1980), 12073 -12080; Alber and Kawasaki, J. Mol. APPI. Gen. 1 (1982), 419 - 434) or alcohol dehydrogenase genes (Young et al., in Genetic Engineering of Microorganisms for Chemicals (Hollaender et al, eds.). Plenum Press, New York, 1982), or the TPIl (US 4,599,311) or ADH2-4C (Russell et al., tiature 304 (1983), 652 - 654) promoters.<br>
Examples of suitable promoters for use in filamentous<br><br>
fungus host cells are, for instance, the ADH3 promoter (McKnight et al.. The EMBO J. 4. (1985), 2093 - 2099) or the tpJA promoter. Examples of other useful promoters are those derived from the gene encoding A.   oryzae  TAKA amylase, Rhizomucor miehei   aspartic proteinase, A.  niger neutral a-amylase, A.   niger  acid stable ot-amylase, A. niger  or A.   awamori  glucoamylase (glviA) , Rhizomucor miehei   lipase, A. oryzae  alkaline protease, A.   oryzae triose phosphate isomerase or A.   nidulans  acetamidase. Preferred are the TAKA-amylase and gluA promoters.<br>
Examples of suitable promoters for use in bacterial host cells include the promoter of the Bacillus stearothermophi.7us  maltogenic amylase gene, the Bacillus licheniformls  alpha-amylase gene, the Bscillus ajgyloliquefaciens  BAN amylase gene, the Bacillus subtilis alkaline protease gen, or the Bacillus pmnilus  xylosidase gerre, or by the phage Lambda PR or PL promoters or the E. coli lac, trp or tac promoters.<br>
The DMA sequerrce encoding the enzyme of the invention may also, if necessary, be operabily connected tx&gt;  a suitable terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the gene coding for dihydrofolate reductase (DHFR) or the Schizosaccharomyces pombe  TPI gene (described by P.R. Russell, Gene 40, 1985, pp. 125-130). For filamentous fungi, selectable markers include amdS. pyrG.arqB, nTaP,   sC.<br>
To direct an enzyme of the present invention into the<br><br>
secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is joined to the DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the enzyme. The secretory signal sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br>
For secretion from yeast cells, the secretory signal sequence may encode any signal peptide which ensures efficient direction of the expressed enzyme into the secretory pathway of the cell. The signal peptide may be naturally occurring signal peptide, or a functional part thereof, or it may be a synthetic peptide. Suitable signal peptides have been found to be the a-factor signal peptide (cf. US 4,870,008), the signal peptide of mouse salivary amylase (cf. O. Hagenbuchle et al.. Nature 289, L981", pp. 643-646) , a modified carboxypeptidase signal peptide (cf. L.A. Vails et al., Cell 48. 1987, pp. 887-397), the yeast BARl signal peptide (cf. WO 87/02670), or :he yeast aspartic protease 3 (YAP3) signal peptide (cf. 1.   Egel-Mitani et al.. Yeast 6,   1990, pp. 127-137).<br>
='or efficient secretion in yeast, a sequence encoding a Leader peptide may also be inserted downstream of the signal sequence and upstream of the DNA sequence encoding :he enzyme. The function of the leader peptide is to illow the expressed enzyme to be directed from the mdoplasmic reticulum to the Golgi apparatus and further ;o a secretory vesicle for secretion into the culture ledium (i.e. exportation of the enzyme across the cell rail or at least through the cellular membrarre" into the •eriplasmic space of the yeast cell). The leader peptide lay be the yeast a-factor leader, (the use of which is<br><br>
described in e.g. US 4,546,082, EP 16 201, EP 123 294, EP 123 544 and EP 163 529). Alternatively, the leader peptide may be a synthetic leader peptide, which is to say a leader peptide not found in nature. Synthetic leader peptides may, for instance, be constructed as described in WO 89/02463 or WO 92/11378.<br>
For use in filamentous fungi, the signal peptide may conveniently be derived from a gene encoding an Aspergillus  sp. amylase or glucoamylase, a gene encoding a Rhizomucor miehei   lipase or protease, a Humicola lanuginosa   lipase. The signal peptide is preferably derived from a gene encoding A.   oryzae  TAKA amylase, A. niger  neutral a-amylase, A.   niger  acid-stable amylase, or A.   niger  glucoamylase.<br>
The procedures used to ligate the DNA sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit.).<br>
Host cells<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question. If homologous to the host cell, i.e. produced by the host cell in nature, it will typically be operably connected to another promoter sequence or, if applicable, another secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a cDNA sequence encoding an enzyme native to the host organism in question. The term "heterologous" is<br><br>
intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may be from another organism, or it may be a synthetic sequence.<br>
The host cell into which the DNA construct or the recombinant vector of the invention is introduced may be any cell which is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells.<br>
Examples of bacterial host cells which, on cultivation, are capable of producing the enzyme of the invention are gram-positive bacteria such as strains of Bacillus,   such as strains of B.   subtilis,   B.   licheniformis,   B.   lentus, B.   brevis,   B.   stearothermophilus,   B.   alkalophilus,   B. amyloliquefaciens,   B.   coagulans,   B.   circulans,   B.   lautus, B.   megatherium  or B. thuringiensis,   or strains of Streptomyces,   such as S.   lividans  or S. murinus,   or gram-negative bacteria such as Echerichia  coli.   The transformation of the bacteria may be effected by protoplast transformation or by using competent cells in a manner known per se (cf. Sambrook et al., supra).<br>
When expressing the enzyme in bacteria such as E.   'coli, the enzyme may be retained in the cytoplasm, typically as insoluble granules (known as inclusion bodies), or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme.<br>
Examples of suitable yeasts cells include cells of Saccharomyces  spp. or Schizosaccharomyces  spp., in<br><br>
particular strains of Saccharomyces  cerevisiae  or Saccharomyces kluyveri.   Methods for transforming yeast cells with heterologous DNA and producing heterologous enzymes therefrom are described, e.g. in US 4,599,311, US 4,931,373, US 4,870,008, 5,037,743, and US 4,845,075, all of which are hereby incorporated by reference. Transformed cells are selected by a phenotype determined by a selectable marker, commonly drug resistance or the ability to grow in the absence of a particular nutrient, e.g. leucine. A preferred vector for use in yeast is the POTl vector disclosed in US 4,931,373. The DNA sequence encoding the enzyme of the invention may be preceded by a signal sequence and optionally a leader sequence , e.g. as described above. Further examples of suitable yeast cells are strains of Kluyveroinyces, such as K.   lactis, Hansenula,   e.g. H.   polymorpha,   or Pichia,   e.g. P. pastoris   (cf. Gleeson et al., J. Gen. Microbiol. 132, 1986, pp. 3459-3465; US 4,882,279).<br>
Examples of other fungal cells are cells of filamentous fungi, e.g. Aspergillus  spp., Neurospora  spp., Fusarium spp. or Trichoderma  spp., in particular strains of A. oryzae,   A.   nidulans,   A.   niger,   or Fusarium graminearum. The use of Aspergillus  spp. for the expression of pro¬teins is described in, e.g., EP 272 277, EP 230 023. The transformation of F.   oxysporum  may, for instance, be carried out as described by Malardier et al., 1989, Gene 78: 147-156.<br>
When a filamentous fungus is used as the host cell, it may be transform.ed with the DNA construct of the invention, conveniently by integrating the DNA construct in the host chromosome to obtain a recombinant host cell. This integration is- generally considered to be an advantage as the DNA sequence~~rs' more likely to be stably maintained in the cell.  Integration of the DNA constructs into the host chromosome mav be performed<br><br>
according to conventional methods, e.g. by homologous or heterologous recombination.<br>
The transformed or transfected host cell described above is then cultured in a suitable nutrient medium under conditions permitting the expression of the present enzyme, after which the resulting enzyme is recovered from the culture.<br>
The medium used to culture the cells may be any conventional medium suitable for growing the host cells, such as minimal or complex media containing appropriate supplements. Suitable media are available from commercia] suppliers or may be prepared according to published recipes (e.g. in catalogues of the American Type Culture Collection). The enzyme produced by the cells may then be recovered from the culture medium by conventional procedures including separating the host cells from the medium by centrifugation or filtration, precipitating the proteinaceous components of the supernatant or filtrate by means of a salt, e.g. ammonium sulphate, purification by a variety of chromatographic procedures, e.g. ion exchange chromatography, gel filtration chromatography, affinity chromatography, or the like, dependent on the type of enzyme in question.<br>
In a still further aspect, the present invention relates to a method of producing an enzyme according to the invention, wherein a suitable host cell transformed with a DNA sequence encoding the enzyme is cultured under conditions permitting the production of the enzyme, and the resulting enzyme is recovered from the culture.<br>
Enzyme Screening driven by taxonomy as well as ecology:<br>
A powerful tool like the molecular screening disclosed herein, designed to detect and select said type of<br><br>
interesting enzymes, can still not stand on its own. In order to maximize the chances of making interesting discoveries the molecular screening approach was in the present investigation combined with careful selection of which fungi to screen. The selection was done through a thorough insight in the identification of fungi, in taxonomical classification and in phylogenetic relationships.<br>
A taxonomic hot spot for production of cellulytic enzymes can further only be fully explored if also the ecological approach is included. Thorough knowledge about the adaptation to various substrates (especially saprotrophic, necrotrophic or biotrophic degradation of plant materials) are prerequisites for designing an intelligent screening and for managing a successful selection of strains and ecological niches to be searc¬hed.<br>
Both the taxonomy and the ecological approach disclosed herein aim at maximizing discovery  of said enzymes in the moleculax screening program. However, still several hundreds (or if all preliminary work is included) several thousand fungi have been brought in'culture in order to detect the 53 hits of said type of cellulytic enzyme here reported.<br>
The screening and cloning may be carried out using the following:<br>
MATERIALS AND METHODS<br>
List of organisms:<br>
Saccharomyces cerevisiae, DSM 9770, DSM-X0082, DSM 10080, DSM 10081, or Escherichia coli, DSM 10512, DSM 10511, DSM 10571, DSM 10576, respectively, containing the plasmid<br><br>
comprising the full length DNA sequence, coding for the endoglucanase of the invention, in the shuttle vector pYES 2.0.<br>
Escherichia   coli   DSM 10583, 10584, 10585, 10586, 10587, and 10588.<br>
Diplodia  gossypina  Cooke Deposit of Strain, Ace No: CBS 274.96 Classification: Ascomycota, Loculoascomycetes, Dothideales, Cucurbitariaceae<br>
Ulospora  bilgramii   (Hawksw.   et  al.)   Hawksw.   et  al. Ace No of strain: NKBC 1444, Nippon University, (Prof. Tubaki collection)<br>
Classification: Ascomycota, Loculoascomycetes, Dothideales, (family unclassified)<br>
Microsphaeropsis  sp.<br>
Isolated from: Leaf of Camellia japonica (Theaceae,<br>
Guttiferales), grown in Kunming Botanical garden, Yunnan<br>
ProV ince, China<br>
Classification: Ascomycota, Loculoascomycetes,<br>
Dothideales, (family unclassified)<br>
Macrophomina  phaseolina     (Tassi)   Goidannich<br>
Syn: Rhizoctonia bataticola<br>
Deposit of Strain, Ace No.:CBS 281.96<br>
Isolated from seed of Glycine max (Leguminosa), cv CMM<br>
60, grown in Thailand, 1990<br>
Classification: Ascomycota, Discomycetes, Rhytismatales,<br>
Rhytismataceae<br>
AscoJboIus stictoideus  Speg.<br>
Isolated from goose dung, Svalbard, Norway<br>
Classification: Ascomycota, Discomycetes, Pezizales,<br>
Ascobolaceae<br><br>
Saccobolus  dilutellus   (Fuck.)   Sacc. Deposit of strain: Ace No CBS 275.96<br>
Classification: Ascomycota, Discomycetes, Pezizales, Ascobolaceae<br>
Penicillium  verruculosum     Peyronel<br>
Ex on Ace No of species: ATCC 62396<br>
Classification: Ascomycota, Plcctomycetes, Eurotiales,<br>
Trichocomaceae<br>
Penicillium chrysogenum Thorn<br>
Ace No of Strain: ATCC 9480<br>
Classification: Ascomycota, Plectomycetes, Eurotiales,<br>
Trichocomaceae<br>
Thermomyces  verrucosus Pugh  et al Deposit of Strain, Ace No.: CBS 285.96<br>
Classification: Ascomycota, Plectomycetes, Eurotiales, (family unclassified; affiliation based on 18S RNA, sequencing and  homologies)<br>
Xylaria hypoxylon L.   ex Greville<br>
Deposit of Strain, Ace No: CBS 284.96<br>
Classification: Ascomycota, Pyrenomyeetes, Xylariales,<br>
Xylariaceae<br>
Poronia  punctata   (Fr.ex L.)   Fr.<br>
Classification: Ascomycota, Pyrenomyeetes, Xylariales,<br>
Xylariaceae<br>
Nodulisporum sp<br>
Isolated from leaf of Camellia reticulata (Theaceae,<br>
Guttiferales), grown in Kunming Botanical Garden, Yunnan<br>
Province, China<br>
Classification: Ascomycota, Pyrenomyeetes, Xylariales,<br>
Xylariaceae<br><br>
Cylindrocarpon  sp<br>
Isolated from marine sample, the Bahamas<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales<br>
(unclassified)<br>
Acremonium sp<br>
Deposit of Strain, Ace. No.: CBS 478.94<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium anguioides Sherbakoff<br>
Ace No of strain: IFO 4467<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium poae   (Peck)   Wr.<br>
Ex on Ace No of species: ATCC 60883<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium     solani   (Mart.)Sacc.emnd.Snyd  &amp; Hans.<br>
Ace No of strain: IMI 107.511<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium oxysporum ssp lycopersici   (Sacc.)Snyd.   &amp;  Hans.<br>
Ace No of strain: CBS 645.78<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Fusarium oxysporum ssp passiflora<br>
Ace No of strain: CBS 744.79<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Hypocreaceae<br>
Gliocladium  catenulatum Gillman  &amp; Abbott Ace. No. of strain: CBS 227.48<br>
- 1<br><br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales, Hypocreaceae<br>
Nectria pinea    Dingley<br>
Deposit of Strain, Ace. No. CBS 279.96<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales,<br>
Nectriaceae<br>
Volutella  colletotrichoides<br>
Ace No of Strain: CBS 400.58<br>
Classification: Ascomycota, Pyrenomycetes, Hypocreales<br>
(unclassified)<br>
Sordaria macrospora Auerswald<br>
Ex on Ace No of species: ATCC 60255<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Sordariaceae<br>
Sordaria    fimicola   (Roberge)   Cesati  et De No taris Ex on Ace. No. for the species: ATCC 52644 Isolated from dung by H.Dissing, ISP, KU, Denmark Classification: Ascomycota, Pyrenomycetes, Sordariales, Sordariaceae<br>
Humicola  grisea  Traeen<br>
ex on Ace No for the species: ATCC 22726<br>
Source: Hatfield Polytechnic<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
(fam. unclassified)<br>
Humicola liigrescens Omvik<br>
Ace No of strain: CBS 819.73<br>
Classification:" Ascomycota, Pyrenomycetes, Sordariales,<br>
(fam. unclassified)<br>
Scytalidium  thermophilum   (Cooney et Emerson)   Austwick Aee No of strain: ATCC 28085<br><br>
classification: Ascoraycota, Pyrenomycetes, Sordariales, (fam. unclassified)<br>
Thielavia  thermophila  Fergus  et  Sinden<br>
(syn Corynascus thermophilus)<br>
Ace No of strain: CBS 174.70, IMI 145.136<br>
Classification: Ascoroycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Isolated from Mushroom compost<br>
Thielavia  terrestris   (Appinis)   Malloch  et  Cain<br>
Ace No of strain: NRRL812 6<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Cladorrhinum foecundissimum Saccardo  et Marchal<br>
Ex on Ace No of species: ATCC 62373<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Lasiosphaeriaceae<br>
Isolated from leaf of Selandin sp. (Compositaceae,<br>
Asterales), Dallas Mountain, Jamaica<br>
Syspastospora boninensis<br>
Ace No of strain: NKBC 1515 (Nippon University, profe<br>
Tubaki Collection)<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Cerastomataceae<br>
Chaetomium  cuniculorum Fuckel<br>
Ace. No. of strain: CBS 799.83<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Chaetomium Jbrasiliense Batista  et  Potual<br>
Ace No of strain: CBS 122.65	.<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales, Chaetomiaceae<br><br>
Chaetomium murorum  Corda<br>
Ace No of strain: CBS 163.52<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Chaetomium  virescens   (von  Arx)   Udagawa<br>
Ace.No.   of strain: CBS 547.75<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Myceliophthora  thermophila   (Apinis)   Oorschot<br>
Deposit of Strain, Ace No:CBS 117.65<br>
Classification: Ascomycota, Pyrenomycetes, Sordariales,<br>
Chaetomiaceae<br>
Nigrospora sp<br>
Deposit of strain. Ace No: CBS 272.96<br>
Isolated from leaf of Artocarpus altilis, Moraceae,<br>
Urticales grown in Christiana, Jamaica<br>
Classification: Ascomycota, Pyrenomycetes,<br>
Trichosphaeriales, (family unclassified)<br>
Nigrospora sp<br>
Isolated from leaf of Pinus yuannanensis. Botanical<br>
Garden, Kuning, Yunnan.<br>
Classification: Ascomycota, Pyrenomycetes,<br>
Trichosphaeriales, Abietaceae, Pinales.<br>
Diaporthe  syngenesia<br>
Deposit of strain. Ace No: CBS 278.96<br>
Classification: Ascomycota, Pyrenomycetes, Diaporthales,<br>
Valsaceae<br>
Colletotrichum lagenarium   (Passerini)   Ellis  et  Halsted<br>
syn Glomerella cingulatavar orbiculare Jenkins et<br>
Winstead<br>
Ex on ace No of species: ATCC 52609<br><br>
Classification: Ascomycota, Pyrenomycetes, Phyllachorales<br>
Exidia  glandulosa Fr.<br>
Deposit of Strain, Ace No: CBS 277.96 Classification: Basidiomycota, Hymenomycetes, Auriculariales, Exidiaceae<br>
Crinipellis  scabella   (Alb.&amp;Schw.:Fr.)Murr<br>
Deposit of strain: Ace No CBS 280.96<br>
Classification: Basidiomycota, Hymenomycetes, Agaricales,<br>
Panaeolus retirugis   (Fr.)   Gill.<br>
Ace.No. of strain: CBS 275.47<br>
Classification: Basidiomycota, Hymenomycetes, Agaricales,<br>
Coprinaceae<br>
Fames  fomentarius   (L.)   Fr.<br>
Deposit of strain: Ace No. CBS 276.96<br>
Classification: Basidiomycota, Hymenomycetes,<br>
Aphyllophorales,<br>
Fomitaceae<br>
Spongipellis  sp.<br>
Deposit of Strain: Ace No CBS 283.96<br>
Classification: Basidiomycota, Hymenomycetes,<br>
Aphyllophorales,<br>
Bjerkanderaeeae (identified and affiliated taxonomieally<br>
by IBS sequence and homology)<br>
Trametes  sanguinea   (Fr.)   Lloyd<br>
syn: Polyporus sanguineus; Pycnoporus sanguineus (L.:Fr.)<br>
Murrill<br>
Ace No of strain: AKU 5062 (Kyoto University Culture<br>
Collection)<br>
Classification: Basidiomycota, Aphyllophorales-^<br>
Polyporaceae<br><br>
Schizophyllum commune Fr<br>
Ace. No. of species: ATCC 38548<br>
Classification: Basidiomycota, Aphyllophorales,<br>
Schizophyllaceae<br>
Rhizophlyctis rosea   (de  Bary  &amp; Wor)   Fischer Deposit of Strain: Ace No.: CBS 282.96 Classification: Chytridiomycota, Chytridiomycetes, Spizellomycetales, Spizellomycetaceae<br>
Rbizomucor pusillus   (Lindt)   Schipper<br>
syn: Mucor pusillus<br>
Ace No of strain: IFO 4578<br>
Ex on Ace No of species: ATCC 46883<br>
Classification: Zygomycota, Zygomycetes, Mucorales,<br>
Mucoraceae<br>
Phycomyces  nltens   (Kunze)   van  Tieghem  &amp;  Le  Monnier<br>
Ace  No of strain: IFO 4814<br>
Ex on Ace No of species: ATCC 163 27<br>
Classification: Zygomycota, Zygomycetes, Mucorales,<br>
Mucoraceae<br>
Chaetostylum  fresenii  van Tieghem &amp; Le Monnier<br>
syn. Helicostylum fresenii<br>
Ace No of strain NRRL 2 3 05<br>
Classification: Zygomycota, Zygomycetes, Mucorales,<br>
Thamnidiaceae<br>
Unclassified:<br>
Trichothecium roseum<br>
Ace No of strain: IFO 5372<br>
Coniothecium sp<br>
Endophyte, isolated from leaf of unidentifed higher<br><br>
plant, growing in Kunming, Yunnan, China<br>
Unclassified and Un-identified:<br>
Deposit of strain. Ace No.: CBS 271.96<br>
Isolated from leaf of Artocarpus altilis (Moraceae,<br>
Urticales), grown in Christiana, Jamaica<br>
Deposit of strain. Ace No.: CBS 273.96<br>
Isolated from leaf of Piroenta dioica (Myrtaceae,<br>
Myrtales) grown in Dallas Mountain, Jamaica<br>
Deposit of strain: CBS 270.96<br>
Isolated from leaf of Pseudocalymma alliaceum<br>
(Bignoniaceae, Solanales) growing in Dallas Mountain,<br>
Jamaica<br>
Other strains:<br>
Escherichia   coli  MC1061 and DHIOB.<br>
Yeast strain: The Saccharomyces  cerevisiae  strain used was W3124 (MATa; ura 3-52; leu 2-3, 112; his 3-D200; pep 4-1137; prcl::HIS3; prbl:: LEU2; cir+).<br>
Plasmids:<br>
The Aspergillus  expression vector pHD414 is a derivative of the plasmid p775 (described in EP 238 023). The construction of pHD414 is further described in WO 93/11249.<br>
pYES 2,0 (Invitrogen)<br>
PA2C477, PA2C193, pA2C357, pA2C371, pA2C385, pA2C475, pA2C488, pA2C502 (See example 1, 2, 3 and 4).<br><br>
Isolation of the DNA sequence shown in SEQ ID No. 1, 4, 6/ 8, 10, 12, 16, or 19 respectively:<br>
The full length DNA sequence, comprising the cDNA sequence shown in SEQ ID No. 1, 4, 6, 8, 10, 12, 16 or 19, respectively, coding for the endoglucanase of the invention, can be obtained from the deposited organism S. cerevisiae,   DSM 9770, DSM 10082, DSM 10080, DSM 10081, E. coli,    DSM 10512, DSM 10511, DSM 10571 or DSM 10576, respectively, by extraction of plasmid DNA by methods known in the art (Sambrook et al. (1989) Molecular cloning: A laboratory manual. Cold Spring Harbor lab.. Cold Spring Harbor, NY) .<br>
PCR primers for molecular screening of cellulases of the present invention:<br>
The four degenerate, deoxyinosine-containing oligonucleotide primers (sense; s and antisense; asl, as2 and as3) corresponding to four highly conserved amino acid regions found in the deduced amino acid sequences of Thielavia  terrestris  cellulase, Myceliophthora thermophiluM  cellulase, and two cellulases from Acremonium sp.   The residues are numbered according to the Myceliophthora  thermophilum  sequence. The deoxyinosines are depicted by an I in the primer sequences, and the restriction sites are underlined.<br>
(klippe-klistre fra SKa<br><br>
Molecular screening by polymerase chain reaction (PCR):<br>
In vitro amplification of genomic DNA and double-stranded CDNA.<br>
Directional, double-stranded cDNA was synthesized from 5 ^q  of poly(A)'^RNA as described below. Genomic DNA was isolated according to Yelton et al.<br>
Approximately 10 to 20 ng of double-stranded, cellulase-induced cDNA or 100 to 200 ng of genomic DNA from a selection of fungal strains was PCR amplified in PCR buffer (10 mM Tris-HCl, pH 8.3, 50 mM KCl, 1.5 mM MgClj, 0.01 % (w/v) gelatin) containing 200 /xM of each dNTP and 100 pmol of each degenerate primer in three combinations:<br><br>
a DNA thermal cycler (Landgraf, Germany) and 2.5 units of Tag polymerase (Perkin-Elmer, Cetus', USA) . Thirty cycles of PCR were performed using a cycle profile of denaturation at 94 °C for 1 min, annealing at 64 °C for 2<br><br>
min, and extension at 72 °C  for 3 min. Ten-μ1 aliquots of the amplification products were analyzed by electrophoresis in 3 % agarose gels (NuSieve, FMC) with Haelll-digested 0X174  RF DNA as a size marker.<br>
Direct sequencing of the PCR products. Eighty-μ1 aliquots of the PCR products were purified using the QIAquick PCR purification kit (Qiagen, USA) according to the manufacturer's instructions. The nucleotide sequences of the amplified PCR fragments were determined directly on the purified PCR products by the dideoxy chain-termination method, using 50-150 ng template, the Tag deoxy-terminal cycle sequencing kit (Perkin-Elmer, USA), fluorescent labeled terminators and 5 pmol of the sense primer: 5'- CCCCAAGCTTACIA/CGITAC/T-TGGGAC/XT-GC/TTGC/IAAA/GA/CC-S ' . Analysis of the sequence data were performed according to Devereux et al.<br>
Cloning by polymerase chain reaction (PCR): Subcloning of PCR fragments.<br>
Twentyfive-μl aliquots of the PCR products generated as described above were electrophoresed in 0.8 % low gelling temperature agarose (SeaPlaque GTG, FMC) gel's, the relevant fragments were excised from the gels, and recovered by agarase treatment by adding 0.1 vol of 10 x agarase buffer (New England Biolabs) and 2 units per 100 μl molten agarose to the sample, followed by incubation at 45 °C for 1.5 h. The sample was phenol and chloroform extracted, and precipitated by addition of  2 vols of 96 % EtOH and 0.1 of 3 M NaAc, pH 5.2. The PCR fragments were recovered by centrifugation, washed in 70 % EtOH, dried and resuspended in 20 μl  of restriction enzyme buffer (10 mM Tris-HCl, 10 m.M MgC12, 50 mM NaCl, 1 mM DTT) , The fragments were digested with Hindlll  and XbaI, phenol and chloroform extracted, recovered by precipitation with  2 vols of 96 % EtOH and 0.1 of 3 M<br><br>
NaAc, pH 5.2, and subcloned into HindIII/Xbal-cleaved pYES 2.0 vector.<br>
Screening of cDNA libraries and characterization of the positive clones. cDNA libraries in S. cerevisiae or E. coli, constructed as described below, were screened by colony hybridization (Sambrook, 1989) using the corresponding random-primed (Feinberg and Vogelstein) ^^P-labeled (&gt;1 x 10^cpm//ig) PCR products as probes. The hybridizations were carried out in 2 x SSC (Sambrook, 1989), 5 X Denhardt's solution (Sambrook, 1989), 0.5 % (w/v) sbs, 100 ^g/ml denatured salmon sperm DNA for 20 h at 65°C followed by washes in 5 x SSC at 25°C (2 x 15 min), 2 X SSC, 0,5 % SDS at 65°C (30 min), 0.2 x SSC, 0.5 % SDS at 65°C (30 min) and finally in 5 x SSC ( 2 x 15 min) at 25"C. The positive cDNA clones were characterized by sequencing the ends of the cDNA inserts with pYES 2.0 polylinker primers (Invitrogen, DSA), and by determining the nucleotide seuence of the longest cDNA from both strands by the dideoxy chain termination method (Sanger et al.) using fluorescent labeled terminators. Qiagen purified plasmid DNA (Qiagen, USA) was sequenced with the Tag deoxy terminal cycle sequencing kit (Perkin Elmer, USA) and either pYES 2.0 polylinker primers (Invitrogen, USA) or synthetic oligonucleotide primers using an Applied Biosystems 373A automated sequencer according to the manufacturers instructions.  Analysis of the sequence data was performed according to Devereux et al.<br>
Extraction of total RNA was performed with guanidinium thiocyanate followed by ultracentrifugation through a 5.7 M CsCl cushion, and isolation of poly(A)*RNA was carried out by oligo(dT)-cellulose affinity chromatography using the procedures described in WO 94/14953.<br>
CDNA synthesis: Double-stranded cDNA was synthesized from 5 μg  poly (A)+ RNA by the RNase H method (Gubler and<br><br>
Hoffman (1983) Gene 25:263-269, Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY) using the hair-pin modification developed by F. S. Hagen (pers. comm.)• The poly (A)+ RNA (5 μg in 5 μl of DEPC-treated water) was heated at 70°C for 8 min. in a pre-siliconized, RNase-free Eppendorph tube, quenched on ice and combined in a final volume of 50 μl  with reverse transcriptase buffer (50 mM Tris-Cl, pH 8.3, 75 mM KCl, 3 mM MgClj, 10 mM DTT, Bethesda Research Laboratories) containing 1 mM of dATP, dGTP and dTTP and 0.5 mM  5-methyl-dCTP (Pharmacia), 40 units human placental ribonuclease inhibitor (RNasin, Promega) , 1.45 μg of oligo(dT) N-Not I primer (Pharmacia) and 1000 units Superscript II RNase H reverse transcriptase (Bethesda Research Laboratories). First-strand cDNA was synthesized by incubating the reaction mixture at 45°C for 1 hour. After synthesis, the mRNA:cDNA hybrid mixture was gelfiltrated through a MicroSpin S-400 HR (Pharmacia) spin column according to the manufacturer's instructions.<br>
After the gelfiltration, the hybrids were diluted in 250 μl second strand buffer (20 mM Tris-Cl, pH 7.4, 90 mM KCl, 4.6 mM MgCl2, 10 mM (NH4)2S04, 0.16 mM /3NAD+) containing 200 juM of each dNTP, 60 units E.   coli   DNA polymerase I (Pharmacia), 5.25 units RNase H (Promega) and 15 units E.   coli   DNA ligase (Boehringer Mannheim). Second strand cDNA synthesis was performed by incubating the reaction tube at 16°C  for 2 hours and additional 15 min. at 25°C. The reaction was stopped by addition of EDTA to a final concentration of 20 mM followed by phenol and chloroform extractions.<br>
Mung bean nuclease treatment: The double-stranded cDNA was precipitated at -2 0°e for 12 hours by addition of 2 vols 96% Eton, 0.2 vol 10 M NH4AC, recovered by centrifugation, washed in 7 0% EtOH, dried and resuspended<br><br>
in 3 0 μl Mung bean nuclease buffer (30 mM NaAc, pH 4.6, 300 mM NaCl, 1 mM ZnSO2, 0.35 mM DTT, 2% glycerol) containing 25 units Mung bean nuclease (Pharmacia). The single-stranded hair-pin DNA was clipped by incubating the reaction at 30°C for 30 min., followed by addition of 70 μl 10 mM Tris-Cl, pH 7. 5, 1 mM EDTA, phenol extraction and precipitation with 2 vols of 96% EtOH and 0.1 vol 3 M NaAc, pH 5.2 on ice for 30 min.<br>
Blunt-ending with T4 DNA polymerase: The double-stranded cDNAs were recovered by centrifugation and blunt-ended in 30 μl T4 DNA polymerase buffer (20 mM Tris-acetate, pH 7.9, 10 mM MgAc, 50 mM KAc, 1 mM DTT) containing 0.5 mM of each dNTP and 5 units T4 DNA polymerase (New England Biolabs) by incubating the reaction mixture at 16°C for 1 hour. The reaction was stopped by addition of EDTA to a final concentration of 20 mM, followed by phenol and chloroform extractions, and precipitation for 12 hours at -20°C by adding 2 vols 96% EtOH and 0.1 vol 3 M NaAc pH 5.2.<br>
Adaptor ligation. Not I digestion and size selection:<br>
After the fill-in reaction the cDNAs were recovered by centrifugation, washed in 7 0% EtOH and dried. The cDNA pellet was resuspended in 25 Ml ligation buffer (30 mM Tris-Cl, pH 7.8, 10 mM MgCl2, 10 mM DTT, 0.5 mM ATP) containing 2,5 /ig non-palindromic BstXI adaptors (Invitrogen) and 30 units T4 ligase (Promega) and incubated at 16°C for 12 hours. The reaction was stopped by heating at 65°C for 20 min. and then cooling on ice for 5 min. The adapted cDNA was digested with Not I restriction enzyme by addition of 20 μl water, 5 Ml lOx Mot I restriction enzyme buffer (New England Biolabs) and 50 units Not I (New England Biolabs), followed by Incubation for 2.5 hours at 37°C The reaction was stopped by heating at 65°C for 10 min. The cDNAs were ;ize-fractionated by gel electrophoresis on a 0.8%<br><br>
SeaPlaque GTG low melting temperature agarose gel (FMC) in Ix TBE to separate unligated adaptors and small cDNAs. The cDNA was size-selected with a cut-off at 0.7 kb and rescued from the gel by use of i3-Agarase (New England Biolabs) according to the manufacturer's instructions and precipitated for 12 hours at -20°C by adding 2 vols 96% EtOH and 0.1 vol 3 M NaAc pH 5.2.<br>
Construction of libraries: The directional, size-selected cDNA was recovered by centrifugation, washed in 7 0% EtOH, dried and resuspended in 30 μl   10 mM Tris-Cl, pH 7.5, 1 mM EDTA. The cDNAs were desalted by gelfiltration through a MicroSpin S-300 HR (Pharmacia) spin column according to the manufacturer's instructions. Three test ligations were carried out in 10 μl   ligation buffer (30 mM Tris-Cl, pH 7.8, 10 mM MgCl2, 10 mM DTT, 0.5 mM ATP) containing 5 μl  double-stranded cDNA (reaction tubes #1 ar.d #2) , 15 units T4 ligase (Promega) and 3 0 ng (tube #1), 4 0 ng (tube #2) and 40 ng (tube #3, the vector background control) of BstXI-NotI cleaved pYES 2.0 vector. The ligation reactions were performed by incubation at 16°C for 12 hours, heating at 7 0°C for 2 0 min. ajid addition of 10 μl  water to each tube. 1 μl of each ligation mixture was electroporated into 40 μl  electrocbmpetent E.   coli DHIOB cells (Bethesda research Laboratories) as described (Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab., Cold Spring Harbor, NY). Using the optimal conditions a library was established in E.   coli  consisting of pools. Each pool was made by spreading transformed E.   coli   on LB+ampicillin agar plates giving 15.000-30.000 colonies/plate after incubation at 37°C for 24 hours. 20 ml LB+ampicillin was added to the plate and the cells were suspended herein. The cell suspension was shaked in a 50 ml tube for 1 hour at 37°C. Plasmid DNA was isolated from the cells according to the manufacturer's instructions using QIAGEN plasmid kit and stored at -20°C.<br><br>
1 μ1  aliquots of purified plasmid DNA (100 nq/μl)   from individual pools were transformed into S.   cerevislae W3124 by electroporation (Becker and Guarante (1991) Methods Enzymol. 194:182-187) and the transformants were plated on SC  agar containing 2% glucose and incubated at 30°C.<br>
Identification of positive colonies:  After 3-5 days of growth, the agar plates were replica plated onto a set of SC + galactose-uracil agar plates containing 0.1% AZCL HE cellulose. These plates were incubated for 3-7 days at 30°C. Endoglucanase positive colonies were identified as colonies surrounded by a blue halo.<br>
Cells from enzyme-positive colonies were spread for single colony isolation on agar, and an enzyme-producing single colony was selected for each of the endoglucanase-prodiacing colonies identified.<br>
Characterization of positive clones: The positive clones  were obtained as single colonies, the cDNA inserts were amplified directly froin the yeast colony using biotinylated polylinker primers, purified by magnetic beads (Dynabead M-280, Dynal) system and characterized individually by sequencing the 5'-end of each cDNA clone using the chain-termination method (Sanger et al. (1977) Proc. Natl. Acad. Sci. U.S.A. 74:5463-5467) and the Sequenase system (United States Biochemical).<br>
The nucleotide sequence was determined of the longest cDNA from both strands by the dideoxy chain termination method (Sanger et al.) using fluorescent labeled terminators. Plasmid DNA was rescued by transformation into E.   coli   as described below. Qiagen purified plasmid DNA (Qiagen, USA) was sequenced with the Taq deoxy terminal cycle sequencing kit (Perkin Elmer, USA) and either pYES 2.0 polylinker primers (Invitrogen, USA) or<br><br><br>
WE CLAIM:<br>
1.	A method of producing an enzyme exhibiting endoglucanase activity, the<br>
method comprising culturing a cell having<br>
(a)	a DNA construct comprising the DNA sequence shown in SEQ IS No.8, or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisisae DSM 10081, or<br>
(b)	a DNA construct comprising an analogue of the DNA sequence shown in SEQ ID No.8 or a DNA construct comprising the DNA sequence obtainable from the plasmid in Saccharomyces cerevisiae DSM 10081, which DNA sequence has at least 75% identity with the DNA sequence shown in SEQ ID No.8 or the DNA sequence obtainable from the plasmid in Sachharomyes cerevisiae DSM 10081, under conditions permitting the production of the enzyme, and recovering the enzyme from the culture.<br><br>
2.	The method according to claim 1, in which the DNA construct is isolated from or produced on the basis of a DNA library of a strain belonging to the family Chae-tomiaceae, preferably to the genus Thielavia, in particular a strain of Thielavia terrestris, especially Thielavia terrestris, NRRL 8126.<br>
3.	The method according to claim 1 or 2, wherein the DNA construct further comprises a DNA sequence encoding a cellulose-binding domain.<br>
4.	The method according to claim 3, wherein in the DNA construct the cellulose-binding domain and the enzyme core (catalytically active domain) of the enzyme encoded by the DNA sequence of the DNA construct are operably linked.<br><br>
5.	The method according to claim 1, wherein the DNA construct is comprised in a<br>
recombinant expression vector.<br>
6.	The method according to claim 1, wherein the cell comprises a recombinant<br>
expression vector according to claim 5.<br>
7.	The method according to claim 1, wherein the cell is a eukaryotic cell, in particular a fungal cell, such as a yeast cell or a filamentous fungal cell, or an endogenous cell from which the gene originates.<br>
8.	The method according to claim 7, wherein the cell belongs to a strain of Aspergillus, Fusarium, or Trichoderma, in particular a strain of Fusarium graminearum, Aspergillus niger or Aspergillus oryzae.<br>
9.	A method of producing an enzyme exhibiting endoglucanase activity<br>
substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">738-mas-1996 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">738-mas-1996 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGNvcnJlc3BvbmRlbmNlIHBvLnBkZg==" target="_blank" style="word-wrap:break-word;">738-mas-1996 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktMS5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 description (complete)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktMi5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 description (complete)-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSktMy5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 description (complete)-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">738-mas-1996 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">738-mas-1996 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">738-mas-1996 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">738-mas-1996 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IGZvcm0tNi5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IHBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">738-mas-1996 pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">738-mas-1996 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM4LW1hcy0xOTk2IHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">738-mas-1996 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="194799-method-and-apparatus-for-defeating-effects-of-color-burst-modification-to-a-video-signal.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="194807-an-improved-process-for-the-production-of-niobium-carbide.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>194800</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>738/MAS/1996</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>20/2006</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-May-2006</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Dec-2005</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-May-1996</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SCHULEIN MARTIN</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ANDERSEN LENE NONBOE</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LASSEN SOREN FLENSTED</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KAUPPINEN MARKUS SAKARI</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NELSEN RUBY ILUM</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>IHARA MICHIKO</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>LANGE LENE</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>TAKAGI SHINOBU</td>
											<td>C/O. NOVOZYMES A/S, KROGSHOJVEJ 36 DK-2880 BAGSVAERD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C11D10/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/194800-a-method-of-producing-an-enzyme-exhibiting-endoglucanase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:31:07 GMT -->
</html>
