<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/195459-a-method-for-the-production-of-1-3-propanediol by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:32:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 195459:A METHOD FOR THE PRODUCTION OF 1,3-PROPANEDIOL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR THE PRODUCTION OF 1,3-PROPANEDIOL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for the production of 1,3-propanediol from a recombinant organism is provided comprising 1) transforming a suitable host organism with one or more transformation cassettes, ii) culturing the transformed host organism in the presence of at least one carbon source whereby 1,3-propanediol is produced, and in) recovering the 1,3-propanediol.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION <br>
The present invention relates to a method for the production of 1,3-propanediol<br>
The present invention relates to the field of molecular biology and the use of recombinant organisms for the production of desired compounds More specifically it describes the expression of cloned genes for glycerol-3-phosphate dehydrogenase (G3PDH) and glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT). either separately or together, for the enhanced production of 1,3-propanediol<br>
BACKGROUND<br>
1.3-Propanediol is a monomer having potential utility in the production of polyester fibers and the manufacture of polyurethanes and cyclic compounds<br>
A variety of chemical routes to 1,3-propanediol are known For example ethylene oxide may be converted to 1,3-propanediol over a catalyst in the presence of phosphine, water, carbon monoxide, hydrogen and an acid, by the catalytic solution phase hydration of acrolein followed by reduction, or from hydrocarbons such as glycerol, reacted in the presence of carbon monoxide and hydrogen over catalysts having atoms from group VIII of the periodic table. Although it is possible to generate 1,3-propanediol by these methods, they are expensive and generate waste streams containing environmental pollutants<br>
It has been known for over a century that 1,3-propanediol can be produced from the fermentation of glycerol Bacterial strains able to produce 1,3-propanediol have been found, for example, in the groups Citrobacter, Clostridium, Enterobacter, llyobacter, Klebsiella, Lactobacillus, and Pelobacter In each case studied, glycerol is converted to 1,3-propanediol in a two step, enzyme catalyzed reaction sequence In the first step, a dehydratase catalyzes the conversion of glycerol to 3-hydroxypropionaldehyde (3-HP) and water (Equation 1) In the second step, 3-HP is reduced to 1,3-propanediol by a NAD+-linked oxidoreductase (Equation 2)<br>
Glycerol ? 3-HP + H2O	(Equation 1)<br>
3-HP + NADH + H+ ? 1,3-Propanediol + NAD+	(Equation 2)<br>
The 1,3-propanediol is not metabolized further and, as a result,accumulates in high concentration in the media The overall reaction consumes a reducing equivalent in the form of a cofactor, reduced ?-nicotinamide adenine dinucleotide (NADH), which is oxidized to nicotinamide adenine dinucleotide (NAD+)<br>
-2-<br><br>
The production of 1.3-propanediol from glycerol is generally performed under anaerobic conditions using glycerol as the sole carbon source and in the absence of other exogenous reducing equivalent acceptors Under these conditions, in for example, strains of Citrobacter, Clostridium, and Klebsiella, a parallel pathway for glycerol operates which first involves oxidation of glycerol to dihydroxyacetone (DHA) by a NAD+- (or NADP+-) linked glycerol dehydrogenase (Equation 3) The DHA, following phosphorylation to dihydroxyacetone phosphate (DHAP) by a DHA kinase (Equation 4), becomes available for biosynthesis and for supporting ATP generation via, for example, glycolysis<br>
Glycerol + NAD+ ? DHA + NADH + H+	(Equation 3)<br>
DHA + ATP ? DHAP + ADP	(Equation 4)<br>
In contrast to the 1,3-propanediol pathway, this pathway may provide carbon and energy to the cell and produces rather than consumes NADH<br>
In Klebsiella pneumoniae and Citrobacter freundu, the genes encoding the functionally linked activities of glycerol dehydratase (dhaB), 1,3-propanediol oxidoreductase (dhaT), glycerol dehydrogenase (dhaD), and dihydroxyacetone kinase (dhaK) are encompassed by the dha regulon The dha regulons from Citrobacter and Klebsiella have been expressed in Escherichia coli and have been shown to convert glycerol to 1,3-propanediol<br>
Biological processes for the preparation of glycerol are known The overwhelming majority of glycerol producers are yeasts, but some bacteria, other fungi and algae are also known to produce glycerol Both bacteria and yeasts produce glycerol by converting glucose or other carbohydrates through the fructose-1,6-bisphosphate pathway in glycolysis or by the Embden Meyerhof Parnas pathway, whereas, certain algae convert dissolved carbon dioxide or bicarbonate in the chloroplasts into the 3-carbon intermediates of the Calvin cycle In a series of steps, the 3-carbon intermediate, phosphoglyceric acid, is converted to glyceraldehyde 3-phosphate which can be readily interconverted to its keto isomer dihydroxyacetone phosphate and ultimately to glycerol.<br>
Specifically, the bacteria Bacillus licheniformis and Lactobacillus lycopersica synthesize glycerol, and glycerol production is found in the halotolerant algae Dunaliella sp and Asteromonas gracilis for protection against high external salt concentrations (Ben-Amotz et al, Experientia 38, 49-52, (1982)) Similarly, various osmotolerant yeasts synthesize glycerol as a protective measure Most strains of Saccharomyces produce some glycerol during alcoholic fermentation, and this can be increased physiologically by the application of<br>
 3<br><br>
osmotic stress (Albertyn et al , Mol Cell Biol 14.4135-4144 (1994)) Earlier this century commercial glycerol production was achieved by the use of Saccharomyces cultures to which 'steering reagents" were added such as sulfites or alkalis Through the formation of an inactive complex, the steering agents block or inhibit the conversion of acetaldehyde to ethanol, thus excess reducing equivalents (NADH) are available to or "steered" towards DHAP for reduction to produce glycerol. This method is limited by the partial inhibition of yeast growth that is due to the sulfites. This limitation can be partially overcome by the use of alkalis which create excess NADH equivalents by a different mechanism In this practice, the alkalis initiated a Cannizarro disproportionation to yield ethanol and acetic acid from two equivalents of acetaldehyde<br>
The gene encoding glycerol-3-phosphate dehydrogenase (DAR1, GPD1) has been cloned and sequenced from S diastaticus (Wang et al, J Bact 176, 7091-7095, (1994)) The DAR1 gene was cloned into a shuttle vector and used to transform E coli where expression produced active enzyme Wang et al (supra) recognize that DAR1 is regulated by the cellular osmotic environment but do not suggest how the gene might be used to enhance 1,3-propanediol production in a recombinant organism<br>
Other glycerol-3-phosphate dehydrogenase enzymes have been isolated for example, sn-glycerol-3-phosphate dehydrogenase has been cloned and sequenced from S cerevmae (Larason et al, Mol Microbiol 10, 1101, (1993)) and Albertyn et al, (Mol Cell Biol 14, 4135, (1994)) teach the cloning of GPD1 encoding a glycerol-3-phosphate dehydrogenase from S cerevisiae . Like Wang et al (supra), both Albertyn et al and Larason et al recognize the osmo-sensitivity of the regulation of this gene but do not suggest how the gene might be used in the production of 1,3-propanediol in a recombinant organism<br>
As with G3PDH, glycerol-3-phosphatase has been isolated from Saccharomyces cerevisiae and the protein identified as being encoded by the GPP1 and GPP2 genes (Norbeck et al., J Biol Chem 271, 13875,(1996)) Like the genes encoding G3PDH, it appears that GPP2 is osmosensitive<br>
Although biological methods of both glycerol and 1,3-propanediol production are known, it has never been demonstrated that the entire process can be accomplished by a single recombinant organism<br>
Neither the chemical nor biological methods described above for the production of 1,3-propanediol are well suited for industrial scale production since the chemical processes are energy intensive and the biological processes require the expensive starting material, glycerol A method requiring low energy input and an inexpensive starting material is needed A more desirable process would<br><br>
incorporate a microorganism that would have the ability to convert basic carbon sources such as carbohydrates or sugars to the desired 1,3-propanediol end-product<br>
Although a single organism conversion of fermentable carbon source other than glycerol or dihydroxyacetone to 1,3-propanediol would be desirable, it has been documented that there are significant difficulties to overcome in such an endeavor For example, Gottschalk et al (EP 373 230) teach that the growth of most strains useful for the production of 1,3-propanediol, including Citrobacter freundii, Clostridium autobutylicum, Clostridium butylicum, and Klebsiella pneumoniae, is disturbed by the presence of a hydrogen donor such as fructose or glucose Strains of Lactobacillus brevis and Lactobacillus buchner, which produce 1,3-propanediol in co-fermentations of glycerol and fructose or glucose, do not grow when glycerol is provided as the sole carbon source, and, although it has been shown that resting cells can metabolize glucose or fructose, they do not produce 1,3-propanediol (Verga DA Cunha et al, J Bacteriol 174, 1013 (1992)) Similarly, it has been shown that a strain of llyobacter polytropus, which produces 1,3-propanediol when glycerol and acetate are provided, will not produce 1,3-propanediol from carbon substrates other than glycerol, including fructose and glucose (Steib et al ,Arch Microbiol 140, 139 (1984)) Finally Tong et al (Appl Biochem Biotech 34, 149 (1992)) has taught that recombinant Escherichia coli transformed with the dha regulon encoding glycerol dehydratase does not produce 1,3-propanediol from either glucose or xylose in the absence of exogenous glycerol<br>
Attempts to improve the yield of 1,3-propanediol from glycerol have been reported where co-substrates capable of providing reducing equivalents, typically fermentable sugars, are included in the process Improvements in yield have been claimed for resting cells of Citrobacter freundu and Klebsiella pneumoniae DSM 4270 cofermenting glycerol and glucose (Gottschalk et al, supra., and Tran-Dinh et al, DE 3734 764), but not for growing cells of Klebsiella pneumoniae ATCC 25955 cofermenting glycerol and glucose, which produced no 1,3-propanediol (I-T Tong, PhD Thesis, University of Wisconsin-Madison (1992)) Increased yields have been reported for the cofermentation of glycerol and glucose or fructose by a recombinant Escherichia coli, however, no 1,3-propanediol is produced in the absence of glycerol (Tong et al., supra ) In these systems, single organisms use the carbohydrate as a source of generating NADH while providing energy and carbon for cell maintenance or growth These disclosures suggest that sugars do not enter the carbon stream that produces 1,3-propanediol In no case is 1,3-propanediol produced in the absence of an<br>
5<br><br>
exogenous source of glycerol Thus the weight of literature clearly suggests that the production of 1,3-propanediol from a carbohydrate source by a single organism is not possible<br>
The problem to be solved by the present invention is the biological production of 1,3-propanediol by a single recombinant organism from an inexpensive carbon substrate such as glucose or other sugars The biological production of 1,3-propanediol requires glycerol as a substrate for a two step sequential reaction in which a dehydratase enzyme (typically a coenzyme B12-dependent dehydratase) converts glycerol to an intermediate, 3-hydroxy-propionaldehyde, which is then reduced to 1,3-propanediol by a NADH- (or NADPH) dependent oxidoreductase The complexity of the cofactor requirements necessitates the use of a whole cell catalyst for an industrial process which utilizes this reaction sequence for the production of 1,3-propanediol. Furthermore, in order to make the process economically viable, a less expensive feedstock than glycerol or dihydroxyacetone is needed. Glucose and other carbohydrates are suitable substrates, but, as discussed above, are known to interfere with 1,3-propanediol production As a result no single organism has been shown to convert glucose to 1,3-propanediol<br>
Applicants have solved the stated problem and the present invention provides for bioconverting a fermentable carbon source directly to 1,3-propanediol using a single recombinant organism Glucose is used as a model substrate and the byconversion is applicable to any existing microorganism Microorganisms harboring the genes encoding gIycerol-3-phosphate dehydrogenase (G3PDH), glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT), are able to convert glucose and other sugars through the glycerol degradation pathway to 1,3-propanediol with good yields and selectivities Furthermore, the present invention may be generally applied to include any carbon substrate that is readily converted to 1) glycerol, 2) dihydroxyacetone, or 3) C3 compounds at the oxidation state of glycerol (e g., glycerol 3-phosphate) or 4) C3 compounds at the oxidation state of dihydroxyacetone (e g , dihydroxyacetone phosphate or glyceraldehyde 3-phosphate)<br>
SUMMARY OF THE INVENTION<br>
The present invention provides a method for the production of 1,3-propanediol from a recombinant organism comprising<br>
(i) transforming a suitable host organism with a transformation cassette comprising at least one of (a) a gene encoding a glycerol-3-phosphate dehydrogenase activity, (b) a gene encoding a glycerol-3 phosphatase activity,<br><br>
(c) genes encoding a dehydratase activity, and (d) a gene encoding 1 3-propanediol oxidoreductase activity, provided that if the transformation cassette comprises less than all the genes of (a)-(d), then the suitable host organism comprises endogenous genes whereby the resulting transformed host organism comprises at least one of each of genes (a)-(d),<br>
(11) culturing the transformed host organism under suitable conditions in the presence of at least one carbon source selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, or a one carbon substrate whereby 1,3-propanediol is produced, and<br>
(iii) recovering the 1,3-propanediol<br>
The invention further provides transformed hosts comprising expression cassettes capable of expressing gIycerol-3-phosphate dehydrogenase, glycerols-phosphatase, glycerol dehydratase and 1,3-propanediol oxidoreductase activities for the production of 1,3-propanediol<br>
The suitable host organism used in the method is selected from the group consisting of bacteria, yeast, and filamentous fungi The suitable host organism is more particularly selected from the group of genera consisting of Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Hansenula, Debaryomyces, Mucor, Torulopsis, Methylobacter, Escherichia, Salmonella, Bacillus, Streptomyces and Pseudomonas Most particularly, the suitable host organism is selected from the group consisting of E coll, Klebsiella spp , and Saccharomyces spp Particular transformed host organisms used in the method are 1) a Saccharomyces spp transformed with a transformation cassette comprising the genes dhaB1, dhaB2, dhaB3, and dhaT, wherein the genes are stably integrated into the Saccharomyces spp. genome, and 2) a Klebsiella spp transformed with a transformation cassette comprising the genes GPD1 and GPD2,<br>
The preferred carbon source of the invention is glucose<br>
The method further uses the gene encoding a glycerol-3-phosphate dehydrogenase enzyme selected from the group consisting of genes corresponding to amino acid sequences given in SEQ ID NO 11, in SEQ ID NO 12, and in SEQ ID NO 13, the amino acid sequences encompassing amino acid substitutions, deletions or additions that do not alter the function of the glycerol-3-phosphate dehydrogenase enzyme The method also uses the gene encoding a glycerol-3-phosphatase enzyme selected from the group consisting of genes corresponding to amino acid sequences given in SEQ ID NO 33 and in SEQ ID NO 17, the amino acid sequences encompassing amino acid substitutions, deletions or additions that<br>
-7 -<br><br>
do not alter the function of the glycerol-3-phosphatase enzyme The method also uses the gene encoding a glycerol kinase enzyme that corresponds to an amino acid sequence given in SEQ ID NO 18, the amino acid sequence encompassing amino acid substitutions deletions or additions that do not alter the function of the glycerol kinase enzyme The method also uses the genes encoding a dehydratase enzyme comprise dhaB1, dhaB2 and dhB3. the genes corresponding respectively to amino acid sequences given in SEQ ID NO 34, SEQ ID NO 35, and SEQ ID NO 36, the amino acid sequences encompassing amino acid substitutions, deletions or additions that do not alter the function of the dehydratase enzyme The method also uses the gene encoding a 1,3-propanediol oxidoreductase enzyme that corresponds to an amino acid sequence given in SEQ ID NO 37, the amino acid sequence encompassing amino acid substitutions, deletions or additions that do not alter the function of the 1,3-propanediol oxidoreductase enzyme<br>
The invention is also embodied in a transformed host cell comprising<br>
(a)	a group of genes composing<br>
(1) a gene encoding a glycerol-3-phosphate dehydrogenase enzyme corresponding to the amino acid sequence given in SEQ ID NO 11,<br>
(2) a gene encoding a glycerol-3-phosphatase enzyme corresponding to the amino acid sequence given in SEQ ID NO 17,<br>
(3) a gene encoding the a subunit of the glycerol dehydratase enzyme corresponding to the amino acid sequence given in SEQ ID NO 34,<br>
(4) a gene encoding the ß subunit of the glycerol dehydratase enzyme corresponding to the amino acid sequence given in SEQ ID NO 35,<br>
(5) a gene encoding the ? subunit of the glycerol dehydratase enzyme corresponding to the amino acid sequence given in SEQ ID NO 36, and<br>
(6) a gene encoding the 1,3-propanediol oxidoreductase enzyme corresponding to the amino acid sequence given in SEQ ID NO 37,<br>
the respective amino acid sequences of (a)(l)-(6) encompassing amino acid substitutions, deletions, or additions that do not alter the function of the enzymes of genes(1)-(6), and<br>
(b)	a host cell transformed with the group of genes of (a), whereby<br>
the transformed host cell produces 1,3-propanediol on at least one substrate<br>
selected from the group consisting of monosaccharides, oligosaccharides, and<br>
polysaccharides or from a one-carbon substrate<br><br>
Accordingly, the present invention relates to a method for the<br>
production of 1,3-propanediol from a recombinant organism comprising :<br>
(i) transforming in the manner such herein described a suitable<br>
host organism of the kind such as herein described with one<br>
or more transformation cassettes each of which comprises at<br>
least one of:<br>
(a) a gene encoding a glycerol-3-phosphate dehydrogenase activity,<br>
(b) a gene encoding a glyceroI-3-phosphatase activity;<br>
(c) genes encoding a dehydratase activity;<br>
(d) a gene encoding 1,3-propanediol oxidoreductase activity,<br>
wherein all of the genes (a) - (d) are introduced into the host microorganism,<br>
(11) culturing the transformed host organism in the manner such as herein described in the presence of at least one carbon source selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, or a one-carbon substrate whereby 1,3-propanediol is produced, and<br>
(iii) recovering in the manner such as herein described the 1,3-propanediol<br>
BRIEF DESCRIPTION OF BIOLOGICAL DEPOSITS AND SEQUENCE LISTING<br>
The transformed E coli W2042 (comprising the E. coli host W1485 and plasmids pDT20 and pAH42) containing the genes encoding glycerol-3-phosphate<br><br>
dehydrogenase (G3PDH) and glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB). and 1.3-propanediol oxidoreductase (dhaT) was deposited on 26 September 1996 with the ATCC under the terms of the Budapest Treaty on the International Recognition of the Deposit of Micro-organisms for the Purpose of Patent Procedure and is designated as ATCC 98188<br>
S cerevisiae YPH500 harboring plasmids pMCK10, pMCK17, pMCK30 and pMCK35 containing genes encoding glycerol-3-phosphate dehydrogenase (G3PDH) and glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT) was deposited on 26 September 1996 with the ATCC under the terms of the Budapest Treaty on the International Recognition of the Deposit of Micro-organisms for the Purpose of Patent Procedure and is designated as ATCC 74392.<br>
"ATCC" refers to the American Type Culture Collection international depository located at 12301 Parklawn Drive, Rockville, MD 20852 USA The designations refer to the accession number of the deposited material<br>
Applicants have provided 49 sequences in conformity with Rules for the Standard Representation of Nucleotide and Amino Acid Sequences in Patent Applications (Annexes I and II to the Decision of the President of the EPO, published in Supplement No 2 to OJ EPO, 12/1992) and with 37 C.F R 1 821-1 825 and Appendices A and B (Requirements for Application Disclosures Containing Nucleotides and/or Amino Acid Sequences)<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention provides a method for a biological production of 1,3-propanediol from a fermentable carbon source in a single recombinant organism The method incorporates a microorganism containing genes encoding glycerol-3-phosphate dehydrogenase (G3PDH), glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT) The recombinant microorganism is contacted with a carbon substrate and 1,3-propanediol is isolated from the growth media<br>
The present method provides a rapid, inexpensive and environmentally responsible source of 1,3-propanediol monomer useful in the production of polyesters and other polymers<br>
The following definitions are to be used to interpret the claims and specification<br>
The terms "glycerol dehydratase" or "dehydratase enzyme" refer to the polypeptide(s) responsible for an enzyme activity that is capable of isomerizing or converting a glycerol molecule to the product 3-hydroxypropionaldehyde For the purposes of the present invention the dehydratase enzymes include a glycerol<br>
10<br><br>
dehydratase (GenBank U09771 U30903) and a diol dehydratase (GenBank D45071) having preferred substrates of glycerol and 1,2-propanediol, respectively Glycerol dehydratase of K pneumonia ATCC 25955 is encoded by the genes dhaB1. dhaB2, and dhaB3 identified as SEQ ID NOS 1, 2 and 3, respectively The dhaB1. dhaB2, and dhaB3 genes as SEQ ID NOS 1, 2 and 3, respectively<br>
glycerol dehydratase enzyme, respectively<br>
The terms "oxidoreductase" or "1,3-piopanediol oxidoreductase refer to the polypeptide(s) responsible for an enzyme activity that is capable of catalyzing the reduction of 3-hydroxyproptonaldehyde to 1 ,3-propanediol 1 ,3-PropanedioJ oxidoreductase includes, for example, the polypeptide encoded by the dhaT gene (GenBank U09771, U30903) and is identified as SEQ ID NO 4<br>
The terms "glycerol-3-phosphate dehydrogenase" or "G3PDH" refer to the polypeptide(s) responsible for an enzyme activity capable of catalyzing the conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) In vivo G3PDH may be NADH-, NADPH- ,or FAD-dependent Examples of this enzyme activity include the following: NADH-dependent enzymes (EC 11,1 8) are encoded by several genes including GPDI (GenBank Z74071 x2) or GPD2 (GenBank Z35169x1) or GPD3 (GenBank G984182) orDARl (GenBank Z74071x2), a NADPH-dependent enzyme (EC 1 1 1 94) is encoded by gpsA (GenBank U32164, G466746 (eds 197911-196892), and L45246), and FAD-dependent enzymes (EC 1 1 99.5) are encoded by GUT2 (GenBank Z47047x23) or glpD (GenBank G147838) or glpABC (GenBank M20938)<br>
The terms "glycerol-3-phosphatase" or "sn-glycerol-3-phosphatase" or "d, 1-giycerol phosphatase" or "G3P phosphatase" refer to the polypeptide(s) responsible for an enzyme activity that is capable of catalyzing the conversion of glycerophosphate to glycerol G3P phosphatase includes, for example, the polypeptides encoded by GPP1 (GenBank Z47047xl25) or GPP2 (GenBank<br>
U18813xll).	<br>
The term "glycerol kinase" refers to the polypeptide(s) responsible for an<br>
enzyme activity capable of catalyzing the conversion of glycerol to glycero-phosphate or glycerol-3-phosphate to glycerol, depending on reaction conditions Glycerol kinase includes, for example, the polypeptide encoded by GUT1 (GenBank Ul 1583x19)<br>
The terms "GPD1 ""DARH "OSG1", "D2830", and "YDL022W will be used interchangeably and refer to a gene that encodes a cytosolic glycerophosphate dehydrogenase and characterized by the base sequence given as SEQ ID NO 5<br><br>
The term "GPD2" refers to a gene that encodes a cytosolic glycerol-3-phosphate dehydrogenase and characterized by the base sequence given as SEQ ID NO 6<br>
The terms "GUT2" and "YIL155C" are used interchangably and refer to a gene that encodes a mitochondrial glycerol-3-phosphate dehydrogenase and characterized by the base sequence given in SEQ ID NO 7<br>
The terms "GPP1", "RHR2" and "YIL053W" are used interchangably and refer to a gene that encodes a cytosolic gIycerol-3-phosphatase and characterized by the base sequence given as SEQ ID NO 8<br>
The terms "GPP2". "HOR2" and "YER062C" are used interchangably and refer to a gene that encodes a cytosolic glycerol-3-phosphatase and characterized by the base sequence given as SEQ ID NO.9<br>
The term "GUT1" refers to a gene that encodes a cytosolic glycerol kinase and characterized by the base sequence given as SEQ ID NO-10<br>
The terms "function" or "enzyme function" refer to the catalytic activity of an enzyme in altering the energy required to perform a specific chemical reaction It is understood that such an activity may apply to a reaction in equilibrium where the production of either product or substrate may be accomplished under suitable conditions<br>
The terms "polypeptide" and "protein" are used interchangeably The terms "carbon substrate" and "carbon source" refer to a carbon source capable of being metabolized by host organisms of the present invention and particularly carbon sources selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, and one-carbon substrates or mixtures thereof<br>
The terms "host cell" or "host organism" refer to a microorganism capable of receiving foreign or heterologous genes and of expressing those genes to produce an active gene product<br>
The terms "foreign gene", "foreign DNA", "heterologous gene" and "heterologous DNA" refer to genetic material native to one organism that has been placed within a host organism by various means<br>
The terms "recombinant organism" and "transformed host" refer to any organism having been transformed with heterologous or foreign genes The recombinant organisms of the present invention express foreign genes encoding glyceroI-3-phosphate dehydrogenase (G3PDH) and glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT) for the production of 1,3-propanediol from suitable carbon substrates<br><br>
Gene" refers to a nucleic acid fragment that expresses a specific protein, including regulatory sequences preceding (5' non-coding) and following (3' non-codmg) the coding region The terms "native" and "wild-type* refer to a gene as found in nature with its own regulatory sequences<br>
The terms 'encoding" and "coding" refer to the process by which a gene through the mechanisms of transcription and translation, produces an amino acid sequence It is understood that the process of encoding a specific amino acid sequence includes DNA sequences that may involve base changes that do not cause a change in the encoded amino acid, or which involve base changes which may alter one or more amino acids, but do not affect the functional properties of the protein encoded by the DNA sequence It is therefore understood that the invention encompasses more than the specific exemplary sequences Modifications to the sequence, such as deletions, insertions, or substitutions in the sequence which produce silent changes that do not substantially affect the functional properties of the resulting protein molecule are also contemplated For example, alteration in the gene sequence which reflect the degeneracy of the genetic code, or which result in the production of a chemically equivalent amino acid at a given site, are contemplated. Thus, a codon for the amino acid alanine, a hydrophobic amino acid, may be substituted by a codon encoding another less hydrophobic residue, such as glycine, or a more hydrophobic residue, such as valine, leucine, or isoleucine Similarly, changes which result in substitution of one negatively charged residue for another, such as aspartic acid for glutamic acid, or one positively charged residue for another, such as lysine for arginine, can also be expected to produce a biologically equivalent product Nucleotide changes which result in alteration of the N-terminal and C-terminal portions of the protein molecule would also not be expected to alter the activity of the protein In some cases, it may in fact be desirable to make mutants of the sequence in order to study the effect of alteration on the biological activity of the protein. Each of the proposed modifications is well within the routine skill in the art, as is determination of retention of biological activity in the encoded products Moreover, the skilled artisan recognizes that sequences encompassed by this invention are also defined by their ability to hybridize, under stringent conditions (0 IX SSC, 0 1% SDS, 65 °C), with the sequences exemplified herein<br>
The term "expression" refers to the transcription and translation to gene product from a gene coding for the sequence of the gene product<br>
The terms "plasmid", "vector", and "cassette" refer to an extra chromosomal element often carrying genes which are not part of the central metabolism of the cell, and usually in the form of circular double-stranded DNA<br><br>
molecules Such element;, may be autonomously replicating sequences, genome integrating sequences, phage or nucleotide sequences, linear or circular, of a single- or double-stranded DNA or RNA, derived from any source, in which a number of nucleotide sequences have been joined or recombined into a unique construction which is capable of introducing a promoter fragment and DNA sequence for a selected gene product along with appropriate 3' untranslated sequence into a cell "Transformation cassette" refers to a specific vector containing a foreign gene and having elements in addition to the foreign gene that facilitate transformation of a particular host cell "Expression cassette" refers to a specific vector containing a foreign gene and having elements in addition to the foreign gene that allow for enhanced expression of that gene in a foreign host<br>
The terms "transformation" and "transfection" refer to the acquisition of new genes in a cell after the incorporation of nucleic acid The acquired genes may be integrated into chromosomal DNA or introduced as extrachromosomal replicating sequences The term "transformant" refers to the product of a transformation<br>
The term "genetically altered" refers to the process of changing hereditary material by transformation or mutation CONSTRUCTION OF RECOMBINANT ORGANISMS<br>
Recombinant organisms containing the necessary genes that will encode the enzymatic pathway for the conversion of a carbon substrate to 1,3-propanediol may be constructed using techniques well known in the art In the present invention genes encoding glycerol-3-phosphate dehydrogenase (G3PDH), glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT) were isolated from a native host such as Klebsiella or Saccharomyces and used to transform host strains such as E. coli DH5a, ECL707, AA200, or W1485, the Saccharomocyes cerevisiae strain YPH500, or the Klebsiella pneumoniae strains ATCC 25955 or ECL 2106. Isolation of Genes<br>
Methods of obtaining desired genes from a bacterial genome are common and well known in the art of molecular biology For example, if the sequence of the gene is known, suitable genomic libraries may be created by restriction endonuclease digestion and may be screened with probes complementary to the desired gene sequence. Once the sequence is isolated, the DNA may be amplified using standard primer directed amplification methods such as polymerase chain reaction (PCR) (U S 4,683,202) to obtain amounts of DNA suitable for transformation using appropriate vectors<br>
14<br><br>
Alternatively, cosmid libraries may be created where large segments of genomic DNA (35-45kb) may be packaged into vectors and used to transform appropriate hosts Cosmid vectors are unique in being able to accommodate large quantities of DNA Generally, cosmid vectors have at least one copy of the cos DNA sequence which is needed for packaging and subsequent circularization of the foreign DNA In addition to the cos sequence these vectors will also contain an origin of replication such as ColEl and drug resistance markers such as a gene resistant to ampicillin or neomycin Methods of using cosmid vectors for the transformation of suitable bacterial hosts are well described in Sambrook et al, Molecular Cloning A Laboratory Manual. Second Edition (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbon, NY (1989)<br>
Typically to clone cosmids, foreign DNA is isolated and ligated, using the appropriate restriction endonucleases, adjacent to the cos region of the cosmid vector Cosmid vectors containing the linearized foreign DNA is then reacted with a DNA packaging vehicle such as bacteriophage X. During the packaging process the cos sites are cleaved and the foreign DNA is packaged into the head portion of the bacterial viral particle These particles are then used to transfect suitable host cells such as E coli Once injected into the cell, the foreign DNA circularizes under the influence of the cos sticky ends In this manner large segments of foreign DNA can be introduced and expressed in recombinant host cells<br>
Isolation and cloning of genes encoding glycerol dehydratase (dhaB) and 1.3-DroDanediol oxidoreductase (dhaT)<br>
Cosmid vectors and cosmid transformation methods were used within the context of the present invention to clone large segments of genomic DNA from bacterial genera known to possess genes capable of processing glycerol to 1,3-propanediol Specifically, genomic DNA from K. pneumoniae ATCC 25955 was isolated by methods well known in the art and digested with the restriction enzyme Sau3A for insertion into a cosmid vector Supercos 1 and packaged using Gigapackll packaging extracts Following construction of the vector E coli XL 1 -Blue MR cells were transformed with the cosmid DNA Transformants were screened for the ability to convert glycerol to 1,3-propanediol by growing the cells in the presence of glycerol and analyzing the media for 1,3-propanediol formation<br>
Two of the 1,3-propanediol positive transformants were analyzed and the cosmids were named pKPl and pKP2 DNA sequencing revealed extensive homology to the glycerol dehydratase gene (dhaB) from C freundn, demonstrating that these transformants contained DNA encoding the glycerol dehydratase gene Other 1,3-propanediol positive transformants were analyzed<br><br>
and the cosmids were named pKP4 and pKP5 DNA sequencing revealed that<br>
these cosmids carried DNA encoding a diol dehydratase gene<br>
Although the instant invention utilizes the isolated genes from within a<br>
Klebsiella cosmid. alternate sources of dehydratase genes include, but are not<br>
limited to. Citrobacter, Clostridia, and Salmonella<br>
Genes encoding G3PDH and G3P phosphatase<br>
The present invention provides genes suitable for the expression of<br>
G3PDH and G3P phosphatase activities in a host cell<br>
Genes encoding G3PDH are known For example, GPD1 has been<br>
isolated from Saccharomyces and has the base sequence given by SEQ ID NO 5.<br>
encoding,the amino acid sequence given in SEQ ID NO 11 (Wang et al, supra)<br>
Similarly, G3PDH activity has also been isolated from Saccharomyces encoded by GPD2 having the base sequence given in SEQ ID NO 6, encoding the amino acid sequence given in SEQ ID NO 12 (Eriksson et al, Mol Microbiol 17, 95, (1995)<br>
It is contemplated that any gene encoding a polypeptide responsible for G3PDH activity is suitable for the purposes of the present invention wherein that activity is capable of catalyzing the conversion of dihydroxyacetone phosphate (DHAP) to glycerol-3-phosphate (G3P) Further, it is contemplated that any gene encoding the amino acid sequence of G3PDH as given by any one of SEQ ID NOS 11, 12, 13, 14, 15 and 16 corresponding to the genes GPD1, GPD2, GUT2, gpsA, glpD, and the a subunit of glpABC, respectively, will be functional in the present invention wherein that amino acid sequence encompasses amino acid substitutions, deletions or additions that do not alter the function of the enzyme It will be appreciated by the skilled person that genes encoding G3PDH isolated from other sources are also be suitable for use in the present invention For example, genes isolated from prokaryotes include GenBank accessions M34393, M20938, L06231, U12567, L45246, L45323, L45324, L45325, U32164, and U39682, genes isolated from fungi include GenBank accessions U30625, U30876 and X56162; genes isolated from insects include GenBank accessions X61223 and X14179, and genes isolated from mammalian sources include GenBank accessions U12424, M25558 and X78593<br>
Genes encoding G3P phosphatase are known For example, GPP2 has been isolated from Saccharomyces cerevisiae and has the base sequence given by SEQ ID NO 9 which encodes the amino acid sequence given in SEQ ID NO 17 (Norbecketal,J Biol Chem 271, p 13875,1996)<br>
It is contemplated that any gene encoding a G3P phosphatase activity is suitable for the purposes of the present invention wherein that activity is capable<br>
-16-<br><br>
of catalyzing the conversion of glycerol-3-phosphate to glycerol. Further, it is contemplated that any gene encoding the amino acid sequence of G3P phosphatase as given by SEQ ID NOS 33 and 17 will be functional in the present invention wherein that amino acid sequence encompasses amino acid substitutions, deletions or additions that do not alter the function of the enzyme It will be appreciated by the skilled person that genes encoding G3P phosphatase isolated from other sources are also suitable for use in the present invention For example, the dephosphorylation of gIycerol-3-phosphate to yield glycerol may be achieved with one or more of the following general or specific phosphatases alkaline phosphatase (EC 3 1 3 I) [GenBank M19159, M29663, U02550 or M33965], acid phosphatase (EC 3 1 3.2) [GenBank U5I210, U19789, U28658 or L20566], glycerol-3-phosphatase (EC 3 1 3 -) [GenBank Z38060 or U18813xl 1], glucose-1-phosphatase (EC 3 13 10) [GenBank M33807], glucose-6-phosphatase (EC 3 1 3 9) [GenBank U00445], fructose-1,6-bisphosphatase (EC 3 1 3 11) [GenBank X12545 or J03207] or phosphotidyl glycero phosphate phosphatase (EC 3 1 3 27) [GenBank M23546 and M23628]<br>
Genes encoding glycerol kinase are known For example, GUT1 encoding the glycerol kinase from Saccharaomyces has been isolated and sequenced (Pavlik et al, Curr Genet 24, 21, (1993)) and the base sequence is given by SEQ ID NO 10 which encodes the amino acid sequence given in SEQ ID NO 18 It will be appreciated by the skilled artisan that although glycerol kinase catalyzes the degradation of glycerol in nature the same enzyme will be able to function in the synthesis of glycerol to convert glycerol-3-phosphate to glycerol under the appropriate reaction energy conditions Evidence exists for glycerol production through a glycerol kinase Under anaerobic or respiration-inhibited conditions, Trypanosoma brucei gives rise to glycerol in the presence of Glycerol-3-P and ADP The reaction occurs in the glycosome compartment (D Hammond, J Biol Chem 260, 15646-15654,(1985)) Host cells<br>
Suitable host cells for the recombinant production of glycerol by the expression of G3PDH and G3F phosphatase may be either prokaryotic or eukaryotic and will be limited only by their ability to express active enzymes Preferred hosts will be those typically useful for production of glycerol or 1,3-propanediol such as Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Hansenula, Debaryomyces, Mucor, Torulopsis, Methylobacter, Escherichia, Salmonella,<br><br>
Bacillus, Streptomyces and Pseudomonas Most preferred in the present invention are E coli Klebsiella species and Saccharomyces species<br>
Adenosyl-cobalamin (coenzyme B12) is an essential cofactor for glycerol dehydratase activity The coenzyme is the most complex non-polymeric natural product known, and its synthesis in vivo is directed using the products of about 30 genes Synthesis of coenzyme B12 is found in prokaryotes, some of which are able to synthesize the compound de novo, while others can perform partial reactions E coli, for example, cannot fabricate the cornn ring structure, but is able to catalyse the conversion of cobmarmde to comnoid and can introduce the 5'-deoxyadenosyl group.<br>
Eukaryotes are unable to synthesize coenzyme B12 de novo and instead transport vitamin B12 from the extracellular milieu with subsequent conversion of the compound to its functional form of the compound by cellular enzymes Three enzyme activities have been described for this series of reactions.<br>
1) aquacobalamin reductase (EC 1.6 99 8) reduces Co(III) to Co(II),<br>
2) cob(II)alamin reductase (EC 1 6 99 9) reduces Co(II) to Co(I), and<br>
3) cob(I)alamin adenosyltransferase (EC 2 5 117) transfers a 5'deoxyadenosine moiety from ATP to the reduced comnoid This last enzyme activity is the best characterized of the three, and is encoded by cobA in S typhimunum, btuR in<br>
E coli and cobO in P denitrificans These three cob(I)alamin adenosyltransferase genes have been cloned and sequenced Cob(I)alamin adenosyltransferase activity has been detected in human fibroblasts and in isolated rat mitochondria (Fenton et al, Biochem Biophys Res Commun 98,283-9,(1981)) The two enzymes involved in cobalt reduction are poorly characterized and gene sequences are not available There are reports of an aquacobalamin reductase from Euglena gracilis (Watanabe et al, Arch Biochem Biophys. 305, 421-7, (1993)) and a microsomal cob(III)alamin reductase is present in the microsomal and mitochondrial inner membrane fractions from rat fibroblasts (Pezacka, Biochim Biophys Acta, 1157,<br>
167-77,(1993))<br>
Supplementing culture media with vitamin B12 may satisfy the need to produce coenzyme B12 for glycerol dehydiatase activity in many microorganisms, but in some cases additional catalytic activities may have to be added or increased in vivo Enhanced synthesis of coenzyme B12 in eukaryotes may be particularly desirable Given the published sequences for genes encoding cob(I)alamin adenosyltransferase, the cloning and expression of this gene could be accomplished by one skilled in the art For example, it is contemplated that yeast, such as Saccharomyces, could be constructed so as to contain genes encoding cob(I)alamin adenosyltransferase in addition to the genes necessary to effect<br>
18<br><br>
conversion of a carbon substrate such as glucose to 1,3-propanediol Cloning and expression of the genes for cobalt reduction requires a different approach This could be based on a selection in E coli for growth on ethanolamine as sole N2 source In the presence of coenzyme B12 ethanolamine ammonia-lyase enables growth of cells in the absence of other N2 sources If E colt cells contain a cloned gene for cob(I)alamin adenosyltransferase and random cloned DNA from another organism, growth on ethanolamine in the presence of aquacobalamin should be enhanced and selected for if the random cloned DNA encodes cobalt reduction properties to facilitate adenosylation of aquacobalamin<br>
In addition to E coli and Saccharomyces, Klebsiella is a particularly preferred host Strains of Klebsiella pneumoniae are known to produce 1,3-propanediol when grown on glycerol as the sole carbon It is contemplated that Klebsiella can be genetically altered to produce 1,3-propanediol from monosaccharides, oligosaccharides, polysaccharides, or one-carbon substrates In order to engineer such strains, it will be advantageous to provide the Klebsiella host with the genes facilitating conversion of dihydroxyacetone phosphate to glycerol and conversion of glycerol to 1,3-propanediol either separately or together, under the transcriptional control of one or more constitutive or inducible promoters The introduction of the DAR1 and GPP2 genes encoding glycerol-3-phosphate dehydrogenase and glycerol-3-phosphatase, respectively, will provide Klebsiella with genetic machinery to produce 1,3-propanediol from an appropriate carbon substrate<br>
The genes (e g , G3PDH, G3P phosphatase, dhaB and/or dhaT) may be introduced on any plasmid vector capable of replication in K pneumoniae or they may be integrated into the K pneumoniae genome For example, K pneumoniae ATCC 25955 and K pneumoniae ECL 2106 are known to be sensitive to tetracycline or chloramphenicol; thus plasmid vectors which are both capable of replicating in K pneumoniae and encoding resistance to either or both of these antibiotics may be used to introduce these genes into K pneumoniae Methods of transforming Klebsiella with genes of interest are common and well known in the art and suitable protocols, including appropriate vectors and expression techniques may be found in Sambrook, supra Vectors and expression cassettes<br>
The present invention provides a variety of vectors and transformation and expression cassettes suitable for the cloning, transformation and expression of G3PDH and G3P phosphatase into a suitable host cell Suitable vectors will be those which are compatible with the bacterium employed Suitable vectors can be derived, for example, from a bacteria, a virus (such as bacteriophage T7 or a M-13<br><br>
derived phage), a cosmid. a yeast or a plant Protocols for obtaining and using such vectors are known to those in the art (Sambrook et al. Molecular Cloning A Laboratory Manual - volumes 1,2,3 (Cold Spring Harbor Laboratory. Cold Spring Harbor, NY, (1989))<br>
Typically, the vector or cassette contains sequences directing transcription and translation of the relevant gene, a selectable marker, and sequences allowing autonomous replication or chromosomal integration Suitable vectors comprise a region 5' of the gene which harbors transcriptional initiation controls and a region 3' of the DNA fragment which controls transcriptional termination. It is most preferred when both control regions are derived from genes homologous to the transformed host cell although it is to be understood that such control regions need not be derived from the genes native to the specific species chosen as a production host.<br>
Initiation control regions or promoters, which are useful to drive expression of the G3PDH and G3P phosphatase genes in the desired host cell, are numerous and familiar to those skilled in the art. Virtually any promoter capable of driving these genes is suitable for the present invention including but not limited to CYC1, HIS3, GAL1, GAL10, ADH1, PGK, PH05, GAPDH, ADC1, TRP1, URA3, LEU2, ENO, TP1 (useful for expression in Saccharomyces), AOX1 (useful for expression in Pichia), and lac, tip, ?PL, ?PR, T7, tac, and trc (useful for expression in E coli).<br>
Termination control regions may also be derived from various genes native to the preferred hosts. Optionally, a termination site may be unnecessary, however, it is most preferred if included.<br>
For effective expression of the instant enzymes, DNA encoding the enzymes are linked operably through initiation codons to selected expression control regions such that expression results in the formation of the appropriate messenger RNA.<br>
Transformation of suitable hosts and expression of genes for the production of 1,3-propanediol<br>
Once suitable cassettes are constructed they are used to transform appropriate host cells Introduction of the cassette containing the genes encoding glycerol-3-phosphate dehydrogenase (G3PDH) and glyceroI-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT), either separately or together into the host cell may be accomplished by known procedures such as by transformation (e g , using calcium-permeabilized cells, electroporation) or by transfection using a recombinant phage virus (Sambrook et al, supra ).<br><br>
In the present invention, E colt W2042 (ATCC 98188) containing the genes encoding glycerol-3-phosphate dehydrogenase (G3PDH) and glycerols-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and i.3-propanediol oxidoreductase {dhaT) was created Additionally. S cerevisiae YPH500 (ATCC 74392) harboring plasmids pMCK10, pMCK17, pMCK30 and pMCK35 containing genes encoding glycerol-3-phosphate dehydrogenase (G3PDH) and glycerol-3-phosphatase (G3P phosphatase), glycerol dehydratase (dhaB), and 1,3-propanediol oxidoreductase (dhaT) was constructed Both the above-mentioned transformed E coli and Saccharomyces represent preferred embodiments of the invention Media and Carbon Substrates<br>
Fermentation media in the present invention must contain suitable carbon substrates. Suitable substrates may include but are not limited to monosaccharides such as glucose and fructose, oligosaccharides such as lactose or sucrose, polysaccharides such as starch or cellulose, or mixtures thereof, and unpurified mixtures from renewable feedstocks such as cheese whey permeate, cornsteep liquor, sugar beet molasses, and barley malt Additionally, the carbon substrate may also be one-carbon substrates such as carbon dioxide, or methanol for which metabolic conversion into key biochemical intermediates has been demonstrated<br>
Glycerol production from single carbon sources (e g , methanol, formaldehyde, or formate) has been reported in methylotrophic yeasts (Yamada et al, Agric Biol Chem, 53(2) 541-543, (1989)) and in bacteria (Hunter et al, Biochemistry, 24, 4148-4155, (1985)). These organisms can assimilate single carbon compounds, ranging in oxidation state from methane to formate, and produce glycerol The pathway of carbon assimilation can be through nbulose monophosphate, through serine, or through xylulose-momophosphate (Gottschalk, Bacterial Metabolism, Second Edition, Spnnger-Verlag New York (1986)) The nbulose monophosphate pathway involves the condensation of formate with ribulose-5-phosphate to form a 6 carbon sugar that becomes fructose and eventually the three carbon product glyceraldehyde-3-phosphate Likewise, the serine pathway assimilates the one-carbon compound into the glycolytic pathway via methylenetetrahydrofolate.<br>
In addition to utilization of one and two carbon substrates, methyl otrophic organisms are also known to utilize a number of other carbon-containing compounds such as methylamine, glucosamine and a variety of amino acids for metabolic activity For example, methylotrophic yeast are known to utilize the carbon from methylamine to form trehalose or glycerol (Bellion et al, Microb Growth C1 Compd, [Int Symp ], 7th (1993), 415-32 Editor(s) Murrell, J<br><br>
Collin, Kelly, Don P Publisher Intercept. Andover. UK) Similarly, various species of Candida will metabolize alanine or oleic acid (Suiter et al , Arch Microbiol, 153(5). 485-9 (1990)) Hence, the source of carbon utilized in the present invention may encompass a wide variety of carbon-containing substrates and will only be limited by the requirements of the host organism<br>
Although it is contemplated that all of the above mentioned carbon substrates and mixtures thereof are suitable in the present invention, preferred carbon substrates are monosaccharides, oligosaccharides, polysaccharides, and one-carbon substrates More preferred are sugars such as glucose, fructose, sucrose and single carbon substrates such as methanol and carbon dioxide Most preferred is glucose.<br>
In addition to an appropriate carbon source, fermentation media must contain suitable minerals, salts, cofactors, buffers and other components, known to those skilled in the art, suitable for the growth of the cultures and promotion of the enzymatic pathway necessary for glycerol production Particular attention is given to Co(II) salts and/or vitamin B12 or precursors thereof. Culture Conditions<br>
Typically, cells are grown at 30 °C in appropriate media Preferred growth media in the present invention are common commercially prepared media such as Luna Bertani (LB) broth, Sabouraud Dextrose (SD) broth or Yeast Malt Extract (YM) broth. Other defined or synthetic growth media may also be used and the appropriate medium for growth of the particular microorganism will be known by someone skilled in the art of microbiology or fermentation science The use of agents known to modulate catabolite repression directly or indirectly, e g , cyclic adenosine 2' 3'-monophosphate or cyclic adenosine 2':5'-monophosphate, may also be incorporated into the reaction media Similarly, the use of agents known to modulate enzymatic activities (e.g , sulphites, bisulphites and alkalis) that lead to enhancement of glycerol production may be used in conjunction with or as an alternative to genetic manipulations.<br>
Suitable pH ranges for the fermentation are between pH 5 0 to pH 9 0, where pH 6 0 to pH 8 0 is preferred as range for the the initial condition<br>
Reactions may be performed under aerobic or anaerobic conditions where anaerobic or microaerobic conditions are preferred Batch and Continuous Fermentations.<br>
The present process uses a batch method of fermentation A classical batch fermentation is a closed system where the composition of the media is set at the beginning of the fermentation and not subject to artificial alterations during the fermentation Thus, at the beginning of the fermentation the media is inoculated<br><br>
with the desired organism or organisms and fermentation is permitted to occur adding nothing to the system Typically, however, a batch fermentation is "batch with respect to the addition of the carbon source and attempts are often made at controlling factors such as pH and oxygen concentration The metabolite and biomass compositions of the batch system change constantly up to the time the fermentation is stopped Within batch cultures cells moderate through a static lag phase to a high growth log phase and finally to a stationary phase where growth rate is diminished or halted If untreated, cells in the stationary phase will eventually die Cells in log phase generally are responsible for the bulk of production of end product or intermediate<br>
A variation on the standard batch system is the Fed-Batch fermentation system which is also suitable in the present invention In this variation of a typical batch system, the substrate is added in increments as the fermentation progresses Fed-Batch systems are useful when catabohte repression is apt to inhibit the metabolism of the cells and where it is desirable to have limited amounts of substrate in the media Measurement of the actual substrate concentration in Fed-Batch systems is difficult and is therefore estimated on the basis of the changes of measurable factors such as pH, dissolved oxygen and the partial pressuie of waste gases such as CO2 Batch and Fed-Batch fermentations are common and well known in the art and examples may be found in Brock, supra<br>
It is also contemplated that the method would be adaptable to continuous fermentation methods Continuous fermentation is an open system where a defined fermentation media is added continuously to a bioreactor and an equal amount of conditioned media is removed simultaneously for processing Continuous fermentation generally maintains the cultures at a constant high density where cells are primarily in log phase growth<br>
Continuous fermentation allows for the modulation of one factor or any number of factors that affect cell growth or end product concentration For example, one method will maintain a limiting nutrient such as the carbon source or nitrogen level at a fixed rate and allow all other paiameters to moderate In other systems a number of factors affecting growth can be altered continuously while the cell concentration, measured by media turbidity, is kept constant Continuous systems strive to maintain steady state growth conditions and thus the cell loss due to media being drawn off must be balanced against the cell growth rate in the fermentation Methods of modulating nutrients and growth factors for continuous fermentation processes as well as techniques for maximizing the rate<br>
23<br><br>
of product formation are well known in the art of industrial microbiology and a variety of methods are detailed by Brock, supra<br>
The present invention may be practiced using either batch, fed-batch or continuous processes and that any known mode of fermentation would be suitable Additionally, it is contemplated that cells may be immobilized on a substrate as whole cell catalysts and subjected to fermentation conditions for 1.3-propanediol production Alterations in the 1,3-propanediol production pathway<br>
Representative enzyme pathwav The production of 1,3-propanediol from glucose can be accomplished by the following series of steps This senes is representative of a number of pathways known to those skilled in the art Glucose is converted in a senes of steps by enzymes of the glycolytic pathway to dihydroxyacetone phosphate (DHAP) and 3-phosphoglyceraldehyde (3-PG) Glycerol is then formed by either hydrolysis of DHAP to dihydroxyacetone (DHA) followed by reduction, or reduction of DHAP to glycerol 3-phosphate (G3P) followed by hydrolysis The hydrolysis step can be catalyzed by any number of cellular phosphatases which are known to be specific or non-specific with respect to their substrates or the activity can be introduced into the host by recombination The reduction step can be catalyzed by a NAD+ (or NADP+) linked host enzyme or the activity can be introduced into the host by recombination It is notable that the dha regulon contains a glycerol dehydrogenase (EC 1116) which catalyzes the reversible reaction of Equation 3<br>
Glycerol? 3-HP + H2O	(Equation 1)<br>
3-HP + NADH + H+ ? 1,3-Propanediol + NAD+	(Equation 2)<br>
Glycerol + NAD+ ? DHA + NADH + H+	(Equation 3)<br>
Glycerol is converted to 1,3-propanediol via the intermediate 3-hydroxy-propionaldehye (3-HP) as has been described in detail above. The intermediate 3-HP is produced from glycerol (Equation 1) by a dehydratase enzyme which can be encoded by the host or can introduced into the host by recombination This dehydratase can be glycerol dehydratase (EC 4 2 1 30), diol dehydratase (EC 4 2.1 28), or any other enzyme able to catalyze this transformation Glycerol dehydratase, but not diol dehydratase, is encoded by the dha regulon. 1,3-Propanediol is produced from 3-HP (Equation 2) by a NAD+- (or NADP+) linked host enzyme or the activity can introduced into the host by recombination This final reaction in the production of 1,3-propanediol can be catalyzed by 1,3-propanediol dehydrogenase (E C 1 1 1 202) or other alcohol dehydrogenases<br><br>
Mutations and transformations that affect carbon channeling A variety of mutani organisms comprising variations in the 1,3-propanediol production pathway will be useful in the present invention The introduction of a triosephosphate isomerase mutation (tpi-) into the microorganism is an example of the use of a mutation to improve the performance by carbon channeling Alternatively, mutations which diminish the production of ethanol (adh) or lactate (Idh) will increase the availability of NADH for the production of 1,3-propanediol Additional mutations in steps of glycolysis after glyceraldehyde-3-phosphate such as phosphoglycerate mutase (pgm) would be useful to increase the flow of carbon to the 1,3-propanediol production pathway Mutations that effect glucose transport such as PTS which would prevent loss of PEP may also prove useful Mutations which block alternate pathways for intermediates of the 1,3-propanediol production pathway such as the glycerol catabolic pathway (g/p) would also be useful to the present invention The mutation can be directed toward a structural gene so as to impair or improve the activity of an enzymatic activity or can be directed toward a regulatory gene so as to modulate the expression level of an enzymatic activity<br>
Alternatively, transformations and mutations can be combined so as to control particular enzyme activities for the enhancement of 1,3-propanediol production Thus it is within the scope of the present invention to anticipate modifications of a whole cell catalyst which lead to an increased production of 1,3-propanediol. Identification and purification of 1.3-propanediol<br>
Methods for the purification of 1,3-propanediol from fermentation media are known in the art For example, propanediols can be obtained from cell media by subjecting the reaction mixture to extraction with an organic solvent, distillation and column chromatography (U S 5,356,812) A particularly good organic solvent for this process is cyclohexane (U S 5,008,473)<br>
1,3-Propanediol may be identified directly by submitting the media to high pressure liquid chromatography (HPLC) analysis Preferred in the present invention is a method where fermentation media is analyzed on an analytical ion exchange column using a mobile phase of 0.01 N sulfuric acid in an isocratic fashion Identification and purification of G3PDH and G3P phosphatase<br>
The levels of expression of the proteins G3PDH and G3P phosphatase are measured by enzyme assays, G3PDH activity assay relied on the spectral properties of the cosubstrate, NADH, in the DHAP conversion to G-3-P NADH has intrinsic UV/vis absorption and its consumption can be monitored<br><br>
spectrophotometncally at 340 nm G3P phosphatase activity can be measured by any method of measuring the inorganic phosphate liberated in the reaction The most commonly used detection method used the visible spectroscopic determination of a blue-colored phosphomolybdate ammonium complex<br>
EXAMPLES GENERAL METHODS<br>
Procedures for phosphorylations, ligations and transformations are well known in the art Techniques suitable for use in the following examples may be found in Sambrook, J et al, Molecular Cloning A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989)<br>
Materials and methods suitable for the maintenance and growth of bacterial cultures are well known in the art Techniques suitable for use in the following examples may be found as set out in Manual of Methods for General Bacteriology (Phillipp Gerhardt, R G E Murray, Ralph N Costilow, Eugene W. Nester, Willis A Wood, Noel R Krieg and G Bnggs Phillips, eds), American Society for Microbiology, Washington, DC. (1994)) or by Thomas D Brock in Biotechnology A Textbook of Industrial Microbiology, Second Edition, Sinauer Associates, Inc , Sunderland, MA (1989) All reagents and materials used for the growth and maintenance of bacterial cells were obtained from Aldrich Chemicals (Milwaukee, WI), DIFCO Laboratories (Detroit, MI), GIBCO/BRL (Gaithersburg, MD), or Sigma Chemical Company (St Louis, MO) unless otherwise specified The meaning of abbreviations is as follows "h" means hour(s), "mm" means minute(s), "sec" means second(s), "d" means day(s), "mL" means milliliters, "L" means liters ENZYME ASSAYS<br>
Glycerol dehydratase activity in cell-free extracts was determined using 1,2-propanediol as substrate The assay, based on the reaction of aldehydes with methylbenzo-2-thiazolone hydrazone, has been described by Forage and Foster (Biochim Biophys Acta, 569,249 (1979)) The activity of 1,3-propanediol oxidoreductase, sometimes referred to as 1,3-propanediol dehydrogenase, was determined in solution or in slab gels using 1,3-propanediol and NAD+ as substrates as has also been described Johnson and Lin, J Bacteriol, 169, 2050 (1987) NADH or NADPH dependent glycerol 3-phosphate dehydrogenase (G3PDH) activity was determined spectrophotometncally, following the disappearance of NADH or NADPH as has been described (R M Bell and J E Cronan,Jr,y Biol Chem 250 7153-8(1975))<br><br>
Assay for glycerol-3-phosphatase GPP<br>
The assay for enzyme activity was performed by incubating the extract with an organic phosphate substrate in a bis-Trts or MES and magnesium buffer. pH 6 5 The substrate used was 1-a-glycerol phosphate, d,l-a-g!ycerol phosphate The final concentrations of the reagents in the assay are buffer (20 mM, bis-Tns or 50 mM MES), MgCl2 (10 mM), and substrate (20 mM) If the total protein in the sample was low and no visible precipitation occurs with an acid quench, the sample was conveniently assayed in the cuvette This method involved incubating an enzyme sample in a cuvette that contained 20 mM substrate (50 µL, 200 mM), 50 mM MES, 10 mM MgCl2. pH 6 5 buffer The final phosphatase assay volume was 0 5 mL The enzyme-containing sample was added to the reaction mixture, the contents of the cuvette were mixed and then the cuvette was placed in a circulating water bath at T = 37 °C for 5 to 120 min -- depending on whether the phosphatase activity in the enzyme sample ranged from 2 to 0 02 U/mL The enzymatic reaction was quenched by the addition of the acid molybdate reagent (0 4 mL) After the Fiske SubbaRow reagent (0.1 mL) and distilled water (1 5 mL) were added, the solution was mixed and allowed to develop After 10 min, the absorbance of the samples was read at 660 nm using a Cary 219 UV/V is spectophotometer The amount of inorganic phosphate released was compared to a standard curve that was prepared by using a stock inorganic phosphate solution (0 65 mM) and preparing 6 standards with final inorganic phosphate concentrations ranging from 0 026 to 0 130 µmol/mL Isolation and Identification 1,3-propanediol<br>
The conversion of glycerol to 1,3-propanediol was monitored by HPLC Analyses were performed using standard techniques and materials available to one skilled in the art of chromatography One suitable method utilized a Waters Maxima 820 HPLC system using UV (210 nm) and RI detection Samples were injected onto a Shodex SH-1011 column (8 mm x 300 mm, purchased from Waters, Milford, MA) equipped with a Shodex SH-101 1P precolumn (6 mm x 50 mm), temperature controlled at 50 °C, using 0 01 N H2SO4 as mobile phase at a flow rate of 0 5 mL/mm When quantitative analysis was desired, samples were prepared with a known amount of tnmethylacetic acid as external standard Typically, the retention times of glycerol (RI detection), 1,3-propanediol (RI detection), and tnmethylacetic acid (UV and RI detection) were 20 67 min, 26 08 min, and 35 03 min, respectively<br>
Production of 1,3-propanediol was confirmed by GC/MS Analyses were performed using standard techniques and materials available to one of skill in the art of GC/MS One suitable method utilized a Hewlett Packard 5890 Series II gas<br><br>
chromatograph coupled to a Hewlett Packard 5971 Series mass selective detector (EI) and a HP-INNOWax column (30 m length, 0 25 mm i d . 0 25 micron film thickness). The retention time and mass spectrum of 1,3-propanediol generated were compared to that of authentic 1.3-propanediol (m/e 57. 58)<br>
An alternative method for GC/MS involved derealization of the sample To 1 0 mL of sample (e g . culture supernatant) was added 30 uL of concentrated (70% v/v) perchloric acid After mixing, the sample was frozen and lyophihzed A 1 1 mixture of bis(tnmethylsilyl)trifluoroacetamide pyridine (300 uL) was added to the lyophihzed material, mixed vigorously and placed at 65 °C for one h The sample was clarified of insoluble material by centrifugation The resulting liquid partitioned into two phases, the upper of which was used for analysis The sample was chromatographed on a DB-5 column (48 m, 0 25 mm I D , 0.25 um film thickness, from J&amp;W Scientific) and the retention time and mass spectrum of the 1,3-propanediol denvative obtained from culture supernatants were compared to that obtained from authentic standards The mass spectrum of TMS-denvatized 1,3-propanediol contains the characteristic ions of 205, 177,130 and 115 AMU<br>
EXAMPLE 1 CLONING AND TRANSFORMATION OF E COLI HOST CELLS WITH<br>
COSMID DNA FOR THE EXPRESSION OF 1.3-PROPANEDIOL Media<br>
Synthetic S12 medium was used in the screening of bacterial transformants for the ability to make 1,3-propanediol S12 medium contains' 10 mM ammonium sulfate, 50 mM potassium phosphate buffer, pH 7 0, 2 mM MgCl2, 0 7 mM CaCl2, 50 uM MnCl2, 1 uM FeCl3, 1 uM ZnCl, 1 7 uM CuS04, 2 5 uM CoCl2, 2 4 uM Na2MoO4, and 2 uM thiamine hydrochloride<br>
Medium A used for growth and fermentation consisted of 10 mM ammonium sulfate; 50 mM MOPS/KOH buffer, pH 7 5, 5 mM potassium phosphate buffer, pH 7 5, 2 mM MgCl2, 0 7 mM CaCl2, 50 uM MnCl2, 1 uM FeCl3, 1 uM ZnCl, 1.72 uM CuSO4, 2 53 uM CoCl2; 2 42 uM Na2MoO4, 2 uM thiamine hydrochloride, 0 01% yeast extract, 0 01% casamino acids, 0 8 µg/mL vitamin B12, and 50 µg/mL amp Medium A was supplemented with either 0 2% glycerol or 0 2% glycerol plus 0 2% D-glucose as required Cells<br>
Klebsiella pneumoniae ECL2106 (Ruch et al, J Bacteriol, 124, 348 (1975)), also known in the literature as K aerogenes or Aerobacter aerogenes, was obtained from E C C. Lin (Harvard Medical School, Cambridge, MA) and was maintained as a laboratory culture<br><br>
Klebsiella pneumoniae ATCC 25955 was purchased from American Type Culture Collection (Rockville, MD)<br>
E colt DH5a was purchased from Gibco/BRL and was transformed with the cosmid DNA isolated from Klebsiella pneumoniae ATCC 25955 containing a gene coding for either a glycerol or diol dehydratase enzyme Cosmids containing the glyceiol dehydratase were identified as pKP1 and pKP2 and cosmid containing the diol dehydratase enzyme were identified as pKP4 Transformed DH5ct cells were identified as DH5a-pKPl, DH5a-pKP2, and DH5a-pKP4<br>
E coli ECL707 (Sprenger et al, J Gen Microbiol, 135, 1255 (1989)) was obtained from E C C Lin (Harvard Medical School, Cambridge, MA) and was similarly transformed with cosmid DNA from Klebsiella pneumoniae These transformants were identified as ECL707-pKPl and ECL707-pKP2, containing the glycerol dehydratase gene and ECL707-pKP4 containing the diol dehydratase gene<br>
E coli AA200 containing a mutation in the tpi gene (Anderson et al, J Gen Microbiol, 62, 329 (1970)) was purchased from the E coli Genetic Stock Center, Yale University (New Haven, CT) and was transformed with Klebsiella cosmid DNA to give the recombinant organisms AA200-pKPl and AA200-pKP2, containing the glycerol dehydratase gene, and AA200-pKP4. containing the diol dehydratase gene DH5a<br>
Six transformation plates containing approximately 1,000 colonies of E coli XL 1-Blue MR transfected with K pneumoniae DNA were washed with 5 mL LB medium and centrifuged. The bacteria were pelleted and resuspended in 5 mL LB medium + glycerol An aliquot (50 uL) was inoculated into a 15 mL tube containing S12 synthetic medium with 0 2% glycerol + 400 ng per mL of vitamin B12 + 0 001% yeast extract + 50amp The tube was filled with the medium to the top and wrapped with parafilm and incubated at 30 °C A slight turbidity was observed after 48 h Aliquots, analyzed for product distribution as described above at 78 h and 132 h, were positive for 1,3-propanediol, the later time points containing increased amounts of 1,3-propanediol<br>
The bacteria, testing positive for 1,3-propanediol production, were serially diluted and plated onto LB-50amp plates in order to isolate single colonies Forty-eight single colonies were isolated and checked again for the production of 1,3-propanediol Cosmid DNA was isolated from 6 independent clones and transformed into E coli strain DH5ct The transformants were again checked for the production of 1,3-propanediol Two transformants were characterized further and designated as DH5a-pKPl and DH5a-pKP2<br><br>
A 12 1 kb EcoRI-Sall fragment from pKPl, subcloned into pIBI31 (IBI Biosystem. New Haven, CT), was sequenced and termed pHK28-26 (SEQ ID NO 19) Sequencing revealed the loci of the relevant open reading frames of the dha operon encoding glycerol dehydratase and genes necessary for regulation Referring to SEQ ID NO 19 a fragment of the open reading frame for dhaK encoding dihydroxyacetone kinase is found at bases 1-399, the open reading frame dhaD encoding glycerol dehydrogenase is found at bases 983-2107, the open reading frame dhaK encoding the repressor is found at bases 2209-4134, the open reading frame dhaT encoding 1,3-propanediol oxidoreductase is found at bases 5017-6180, the open reading frame dhaB1 encoding the alpha subunit glycerol dehydratase is found at bases 7044-8711, the open reading frame dhaBl encoding the beta subunit glycerol dehydratase is found at bases 8724-9308, the open reading frame dhaB3 encoding the gamma subunit glycerol dehydratase is found at bases 9311 -9736, and the open reading frame dhaBX, encoding a protein of unknown function is found at bases 9749-11572<br>
Single colonies of E coli XL 1-Blue MR transfected with packaged cosmid DNA from K pneumoniae were inoculated into microliter wells containing 200 uL of S15 medium (ammonium sulfate, 10 mM, potassium phosphate buffer, pH 7 0, 1 mM, MOPS/KOH buffer, pH 7 0, 50 mM, MgCl2, 2 mM, CaCl2, 0 7 mM, MnCl2, 50 uM, FeCl3, 1 uM, ZnCl, 1 uM, CuSO4, 1 72 uM, CoCl2, 2 53 uM, Na2MoO4, 2 42 uM, and thiamine hydrochloride, 2 uM) + 0 2% glycerol + 400 ng/mL of vitamin B12 + 0 001% yeast extract + 50 µg/mL ampicillin In addition to the microtiter wells, a master plate contaimng LB-50 amp was also inoculated. After 96 h, 100 uL was withdrawn and centrifuged in a Rainin microfuge tube contaimng a 0 2 micron nylon membrane filter Bacteria were retained and the filtrate was processed for HPLC analysis Positive clones demonstrating 1,3-propanediol production were identified after screening approximately 240 colonies Three positive clones were identified, two of which had grown on LB-50 amp and one of which had not. A single colony, isolated from one of the two positive clones grown on LB-50 amp and verified for the production of 1,3-propanediol, was designated as pKP4 Cosmid DNA was isolated from E coli strains contaimng pKP4 and E coli strain DH5a was transformed. An independent transformant, designated as DH5a-pKP4, was verified for the production of 1,3-propanediol ECL707<br>
E coli strain ECL707 was transformed with cosmid K pneumoniae DNA corresponding to one of pKPl, pKP2, pKP4 or the Supercos vector alone and named ECL707-pKPl, ECL707-pKP2, ECL707-pKP4, and ECL707-sc,<br><br>
respectively ECL707 is defective in glpK, gld, and ptsD which encode the ATP-dependent glycerol kinase, NAD+-linked glycerol dehydrogenase, and enzyme II for dihydroxyacetone of the phosphoenolpyruvate-dependent phosphotransferase system, respectively<br>
Twenty single colonies of each cosmid transformation and five of the Supercos vector alone (negative control) transformation, isolated from LB-50 amp plates, were transferred to a master LB-50 amp plate These isolates were also tested for their ability to convert glycerol to 1,3-propanediol in order to determine if they contained dehydratase activity The transformants were transferred with a sterile toothpick to microtiter plates containing 200 uL of Medium A supplemented with either 0 2% glycerol or 0 2% glycerol plus 0 2% D-glucose After incubation for 48 hr at 30 °C, the contents of the microtiter plate wells were filtered through an 0 45 micron nylon filter and chromatographed by HPLC The results of these tests are given in Table 1<br>
Table 1 Conversion of glycerol to 1,3-propanediol by transformed ECL707<br><br>
Transformant<br>
Glycerol*<br>
Glvcerol plus Glucose*<br>
ECL707-pKPl<br>
19/20<br>
19/20<br>
ECL707-pKP2<br>
18/20<br>
20/20<br>
ECL707-pKP4<br>
0/20<br>
20/20<br>
ECL707-sc<br>
0/5<br>
0/5<br>
*(Number of positive isolates/number of isolates tested)<br>
AA200<br>
E coli strain AA200 was transformed with cosmid K pneumoniae DNA corresponding to one of pKPl, pKP2, pKP4 and the Supercos vector alone and named AA200-pKP 1, AA200-pKP2, AA200-pKP4, and AA200-sc, respectively Strain AA200 is defective in tnosephosphate isomerase (tpr)<br>
Twenty single colonies of each cosmid transformation and five of the empty vector transformation were isolated and tested for their ability to convert glycerol to 1,3-propanediol as described for E coli strain ECL707 The results of these tests are given in Table 2<br>
-31 -<br><br>
Table 2<br>
Conversion of glycerol to 1.3-propanediol bv transformed AA200<br>
Transtrormant<br>
Glycerol*<br>
Glycerol plus Glucose*<br>
AA200-pKPl<br>
17/20<br>
17/20<br>
AA200-pKP2<br>
17/20<br>
17/20<br>
AA200-pKP4<br>
2/20<br>
16/20<br>
AA200-sc<br>
0/5<br>
0/5<br>
* (Number of positive isolates/number of isolates tested)<br>
EXAMPLE 2 CONVERSION OF D-GLUCOSE TO 1.3-PROPANEDIOL BY RECOMBINANT E coli USING DAR1. GPP2. dhaB. and dhaT Construction of general purpose expression plasmids for use in transformation of<br>
Escherichia coli<br>
The expression vector DTacIO<br>
The E coli expression vector, pTacIQ, contains the laclq gene (Farabaugh, Nature 274, 5673 (1978)) and tac promoter (Amann et al, Gene 25, 167 (1983)) inserted into the EcoRI of pBR322 (Sutcliffe et al, Cold Spring Harb Symp Quant Biol 43, 77 (1979)) A multiple cloning site and terminator sequence (SEQ ID NO 20) replaces the pBR322 sequence from EcoRI to SphI Subclaiming the glycerol dehydratase genes (dhaB1. 2, 3)<br>
The open reading frame for dhaB3 gene (incorporating an EcoRI site at the 5' end and a Xbal site at the 3' end) was amplified from pHK28-26 by PCR using primers (SEQ ID NOS.21 and 22) The product was subcloned into pLitmus29 (New England Biolab, Inc , Beverly, MA) to generate the plasmid pDHAB3<br>
containing dhaB3<br>
The region containing the entire coding region for the four genes of the dhaB operon from pHK28-26 was cloned into pBluescriptll KS+ (Stratagene, La Jolla, CA) using the restriction enzymes KpnI and EcoRI to create the plasmid<br>
pM7<br>
The dhaBX gene was removed by digesting the plasmid pM7, which<br>
contains dkaB(1.2.3.4). with Apal and Xbal (deleting part of dhaB3 and all of dhaBK) The resulting 5.9 kb fragment was purified and ligated with the 325-bp Apal-Xbal fragment from plasmid pDHAB3 (restonng the dhaB3 gene) to create<br>
pM 11, which contains dhaB(l, 2,3)<br>
The open reading frame for the dhaBl gene (incorporating a Hmdlll site and a consensus RBS ribosome binding site at the 5' end and a Xbal site at the 3' end) was amplified from pHK28-26 by PCR using primers (SEQ ID NO 23 and<br><br>
SEQ ID NO 24) The product was subcloned into pLitmus28 (New England Biolab. Inc ) to generate the plasmid pDTl containing dhaBl<br>
A Notl-Xbal fragment from pMl 1 containing part of the dhaBl gene the dhaBl gene and the dhaB3 gene was inserted into pDTl to create the dhaB expression plasmid, pDT2 The HindIII-XbaI fragment containing the dhaB(l 2 J) genes from pDT2 was inserted into pTacIQ to create pDT3 Subclaining the 1 1-nronanediol dehydrogenase gene (dhaT)<br>
The KpnI-SacI fragement of pHK28-26, containing the complete 1,3-propanedioI dehydrogenase (dhaT) gene, was subcloned into pBluescnptII KS+ creating plasmid pAH1. The dhaT gene (incorporating an Xbal site at the 5' end and a BamHI site at the 3' end) was amplified by PCR from pAHl as template DNA using synthetic primers (SEQ ID N0.25 with SEQ ID N0.26) The product was subcloned into pCR-Scnpt (Stratagene) at the Srfl site to generate the plasmids pAH4 and pAH5 containing dhaT. The plasmid pAH4 contains the dhaT gene in the correct orientation for expression from the lac promoter in pCR-Scnpt and pAH5 contains the dhaT gene in the opposite orientation The Xbal-BamHI fragment from pAH4 containing the dhaT gene was inserted into pTacIQ to generate plasmid pAH8 The HmdIII-BamHI fragment from pAH8 contaimng the RBS and dhaT gene was inserted into pBluescnptll KS+ to create pAH11 The HindIII-Sall fragment from pAH8 contaimng the RBS, dhaT gene and terminator was inserted into pBluescnptll SK+ to create pAH12 Contraction of an expression cassette for dhaB (l.2.3) and dhaT<br>
An expression cassette for the dhaB(l,2,3) and dhaT was assembled from the individual dhaB(l,2,3) and dhaT subclones described above using standard molecular biology methods The Spel-Kpnl fragment from pAH8 contaimng the RBS, dhaT gene and terminator was inserted into the Xbal-KpnI sites of pDT3 to create pAH23 The SmaI-EcoRI fragment between the dhaB3 and dhaT gene of pAH23 was removed to create pAH26. The SpeI-NotI fragment containing an EcoRI site from pDT2 was used to replace the Spel-NotI fragment of pAH26 to generate pAH27 Construction of expression cassette for dhaT and dhaB(l.2,3)<br>
An expression cassette for dhaT and dhaB(l,2,3) was assembled from the individual dhaB(l,2,3) and dhaT subclones descnbed previously using standard molecular biology methods A SpeI-SacI fragment containing the dhaB(l,2,3) genes from pDT3 was inserted into pAH11 at the SpeI-SacI sites to create pAH24.<br><br>
Cloning and expression of glycerol 3-phosphatase for increased glycerol production in E coli<br>
The Saccharomyces cetevisiae chromosomeV lamda clone 6592 (Gene Bank, acession # U18813xl 1) was obtained from ATCC The glycerol 3-phosphate phosphatase (GPP2) gene (incorporating an BamHI-RBS-XbaI site at the 5' end and a SmaI site at the 3' end) was cloned by PCR cloning from the lamda clone as target DNA using synthetic primers (SEQ ID NO 27 with SEQ ID NO 28) The product was subcloned into pCR-Scnpt (Stratagene) at the SrfI site to generate the plasmids pAH 15 containing GPP2 The plasmid pAH 15 contains the GPP2 gene in the inactive orientation for expression from the lac promoter in pCR-Scnpt SK+ The BamHI-Smal fragment from pAHI5 containing the GPP2 gene was inserted into pBlueScnptII SK+ to generate plasmid pAH19. The pAH19 contains the GPP2 gene in the correct orientation for expression from the lac promoter The XbaI-PstI fragment from pAH19 containing the GPP2 gene was inserted into pPH0X2 to create plasmid pAH21 Plasmids for the expression of dhaT. dhaB(l.2.3) and GPP2 genes<br>
A SaII-EcoRI-XbaI linker (SEQ ID NOS 29 and 30) was inserted into pAH5 which was digested with the restriction enzymes, SaII-XbaI to create pDT16. The linker destroys the XbaI site The 1 kb SaII-MIuI fragment from pDT16 was then inserted into pAH24 replacing the existing Sall-Mlul fragment to create pDT 18<br>
The 4 1 kb EcoR1-Xba1 fragment containing the expression cassette for dhaTand dhaB(1,2,3) from pDT18 and the 1 0 kb XbaI-SaII fragement containing the GPP2 gene from pAH21 was inserted into the vector pMMB66EH (Fuste et al, GENE, 48, 119 (1986)) digested with the restriction enzymes EcoRI and SaII to create pDT20<br>
Plasmids for the over expression of DAR1 in E coli<br>
DAR1 was isolated by PCR cloning from genomic S cerevistae DNA using synthetic primers (SEQ ID NO 46 with SEQ ID NO 47) Successful PCR cloning places an NcoI site at the 5' end of DAR1 where the ATG within Ncol is the DAR1 initiator methionine At the 3' end of DAR1 a BamHI site is introduced following the translation terminator The PCR fragments were digested with Ncol + BamHl and cloned into the same sites within the expression plasmid pTrc99A (Pharmacia, Piscataway, New Jersey) to give pDAR1A<br>
In order to create a better ribosome binding site at the 5' end of DAR1, a Spel-RBS-Ncol linker obtained by annealing synthetic primers (SEQ ID NO 48 with SEQ ID NO 49) was inserted into the Ncol site of pDAR1A to create pAH40 Plasmid pAH40 contains the new RBS and DAR1 gene in the correct<br><br>
orientation for expression from the trc promoter of Trc99A (Pharmacia) The NcoI-BamHI fragment from pDARl A and a second set of SpeI-RBS-Ncol linker obtained by annealing synthetic primers (SEQ ID NO 31 with SEQ ID NO 32) was inserted into the Spel-BamHI site of pBluescript II-SK+ (Stratagene) to create pAH41 The construct pAH41 contains an ampicillin resistance gene The NcoI-BamHI fragment from pDARl A and a second set of Spel-RBS-Ncol linker obtained by annealing synthetic primers (SEQ ID NO 31 with SEQ ID NO 32) was inserted into the Spel-BamHI site of pBC-SK+ (Stratagene) to create pAH42 The construct pAH42 contains a chloroamphenicol resistance gene Construction of an expression cassette for DAR1 and GPP2<br>
An expression cassette for DARl and GPP2 was assembled from the individual DARl and GPP2 subclones described above using standard molecular biology methods The BamHI-PstI fragment from pAH19 containing the RBS and GPP2 gene was inserted into pAH40 to create pAH43 The BamHI-PstI fragment from pAH19 containing the RBS and GPP2 gene was inserted into pAH41 to create pAH44 The same BamHI-PstI fragment from pAHl9 containing the RBS and GPP2 gene was also inserted into pAH42 to create pAH45 E coli strain construction<br>
E coli W1485 is a wild-type K-12 strain (ATCC 12435) This strain was transformed with the plasmids pDT20 and pAH42 and selected on LA (Luna Agar, Difco) plates supplemented with 50 µg/mL carbencillim and 10 µg/mL chloramphenicol Production of 1.3-propanediol from glucose<br>
E coli W1485/pDT20/pAH42 was transferred from a plate to 50 mL of a medium containing per liter 22 5 g glucose, 6 85 g K2HPO4, 6 3 g (NH4)2SO4, 0 5 g NaHCO3, 2 5 g NaCl, 8 g yeast extract, 8 g tryptone, 2 5 mg vitamin B12, 2 5 mL modified Balch's trace-element solution, 50 mg carbencilhm and 10 mg chloramphenicol, final pH 6 8 (HCl), then filter sterilized. The composition of modified Balch's trace-element solution can be found in Methods for General and Molecular Bacteriology (P Gerhardt et al, eds, p 158, American Society for Microbiology, Washington, DC (1994)) After incubating at 37 °C, 300 rpm for 6 h, 0 5 g glucose and IPTG (final concentration = 02 mM) were added and shaking was reduced to 100 rpm Samples were analyzed by GC/MS After 24 h, W1485/pDT20/pAH42 produced 1.1 g/L glycerol and 195 mg/L 1,3-propanediol<br>
35<br><br>
EXAMPLE 3 CLONING AND EXPRESSION OF dhaB AND dhaT LN Saccharomvces cerevisiae Expression plasmids that could exist as replicating episomal elements were constructed for each of the four dha genes For all expression plasmids a yeast ADH1 promoter was present and separated from a yeast ADH1 transcription terminator by fragments of DNA containing recognition sites for one or more restriction endonucleases Each expression plasmid also contained the gene for {3-lactamase for selection in E coli on media containing ampicillin, an origin of replication for plasmid maintainence in E coli, and a 2 micron origin of replication for maintainence in S cerevisiae The selectable nutritional markers used for yeast and present on the expression plasmids were one of the following HIS3 gene encoding imidazoleglycerolphosphate dehydratase, URA3 gene encoding orotidine 5'-phosphate decarboxylase, TRP1 gene encoding N-(5'-phosphonbosyl)-anthranilate isomerase, and LEU2 encoding p-isopropylmalate dehydrogenase<br>
The open reading frames for dhaT, dhaBS, dhaB2 and dhaBl were amplified from pHK28-26 (SEQ ID NO 19) by PCR using primers (SEQ ID NO 38 with SEQ ID NO 39, SEQ ID NO 40 with SEQ ID NO 41, SEQ ID NO 42 with SEQ ID NO 43, and SEQ ID NO 44 with SEQ ID NO 45 for dhaT, dhaB3, dhaB2 and dhaBl, respectively) incorporating EcoRl sites at the 5' ends (10 mM Tns pH 8 3, 50 mM KC1, 1 5 mM MgCl2, 0 0001% gelatin, 200 uM dATP, 200 uM dCTP, 200 uM dGTP, 200 \iU dTTP, 1 uM each primer, 1-10 ng target DNA, 25 units/mL Amphtaq(tm) DNA polymerase (Perkin-Elmer Cetus, Norwalk CT)) PCR parameters were 1 min at 94 °C, 1 mm at 55 °C, 1 mm at 72 °C, 35 cycles The products were subcloned into the EcoRl site of pHIL-D4 (Phillips Petroleum, Bartlesville, OK) to generate the plasmids pMP13, pMP14, pMP20 and pMP15 containing dhaT, dhaB3, dhaB2 and dhaBl, respectively Construction of dhaBl expression plasmid pMCKlO<br>
The 7 8 kb replicating plasmid pGADGH (Clontech, Palo Alto, CA) was digested with Hindlll, dephosphorylated, and hgated to the dhaBl Hindlll fragment from pMP 15 The resulting plasmid (pMCK 10) had dhaBl correctly oriented for transcription from the ADH1 promoter and contained a LEU2 marker Construction of dhaBl expression plasmid pMCK17<br>
Plasmid pGADGH (Clontech, Palo Alto, CA) was digested with Hmdlll and the single-strand ends converted to EcoRl ends by ligation with Hindlll-Xmnl and EcoRI-XmnI adaptors (New England Biolabs, Beverly, MA) Selection for plasmids with correct EcoRl ends was achieved by ligation to a kanamycm<br>
-2C<br><br>
resistance gene on an EcoRI fragment from plasmid pUC4K (Pharmacia Biotech. Uppsala) transformation into E coli strain DH5ct and selection on LB plates containing 25 Mg/mL kanamycm The resulting plasmid (pGAD/KAN2) was digested with SnaBI and EcoRI and a 1 8 kb fragment with the ADH1 promoter was isolated Plasmid pGBT9 (Clontech. Palo Alto, CA) was digested with SnaBI and EcoRJ. and the 1 5 kb ADH1/GAL4 fragment replaced by the 1 8 kb ADH1 promoter fragment isolated from pGAD/KAN2 by digestion with SnaBI and EcoRI The resulting vector (pMCKI 1) is a replicating plasmid in yeast with an ADH1 promoter and terminator and a TRPl marker Plasmid pMCKI 1 was digested with EcoRI, dephosphorylated, and hgated to the dhaB2 EcoRI fragment from pMP20 The resulting plasmid (pMCK 17) had dhaB2 correctly oriented for transcription from the ADH1 promoter and contained a TRP1 marker Construction of dhaB3 expression Dlasmid PMCK30<br>
Plasmid pGBT9 (Clontech) was digested with Nael and PvuII and the 1 kb TRPl gene removed from this vector The TRPI gene was replaced by a URA3 gene donated as a 1 7 kb Aatll/Nael fragment from plasmid pRS406 (Stratagene) to give the intermediary vector pMCK32 The truncated ADH1 promoter present on pMCK32 was removed on a 1 5 kb SnaBl/EcoRI fragment, and replaced with a full-length ADH1 promoter on a 1 8 kb SnaBI/EcoRI fragment from plasmid pGAD/KAN2 to yield the vector pMCK26 The unique EcoRI site on pMCK26 was used to insert an EcoRI fragment with dhaB3 from plasmid pMP14 to yield pMCK30 The pMCK30 replicating expression plasmid has dhaB3 orientated for expression from the ADHI promoter, and has a URA3 marker Construction of dhaT expression plasmid PMCK.35<br>
Plasmid pGBT9 (Clontech) was digested with Nael and PvuII and the 1 kb TRPl gene removed from this vector The TRPI gene was replaced by a HIS3 gene donated as an Xmnl/Nael fragment from plasmid pRS403 (Stratagene) to give the intermediary vector pMCK33 The truncated ADHI promoter present on pMCK.33 was removed on a 1.5 kb SnaBI/EcoRI fragment, and replaced with a full-length ADHI promoter on a 1 8 kb SnaBI/EcoRI fragment from plasmid pGAD/KAN2 to yield the vector pMCK31 The unique J icoRI site on pMCK31 was used to insert an EcoRI fragment with dhaT from plasmid pMP 13 to yield pMCK35 The pMCK35 replicating expression plasmid has dhaT orientated for expression from the ADHI promoter, and has a HIS3 marker Transformation of S cerevistae with dha expression plasmids<br>
S cerevisiae strain YPH500 {ura3-52 lys2-801 ade2-10J trpl-A63 his3-A200 Ieu2-Al) (Sikorski R S and Hieter P , Genetics 122, 19-27, (1989)) purchased from Stratagene (La Jolla, CA) was transformed with 1-2 |ig of plasmid<br><br>
DNA using a ^rozen-EZ Yeast Transformation Kit (Catalog #T200i) (Zymo Research. Orange. CA) Colonies were grown on Supplemented Minimal Medium (SMM - 0 67% yeast nitrogen base without amino acids, 2% glucose) for 3-4 d at 29 °C with one or more of the following additions adenine sulfate (20 mg/L). uracil (20 mg/L), L-tryptophan (20 mg/L). L-histidine (20 mg/L), L-leucine (30 mg/L), L-lysine (30 mg/L) Colonies were streaked on selective plates and used to inoculate liquid media Screening of S cerevisiae transformants for dha genes<br>
Chromosomal DNA from URA+, HIS+, TRP+, LEU+ transformants was analyzed by PCR using primers specific for each gene (SEQ ID NOS 38-45) The presence of all four open reading frames was confirmed Expression of dhaB and dhaT activity in transformed S cerevisiae<br>
The presence of active glycerol dehydratase {dhaB) and 1,3-propanediol oxido-reductase (dhaT) was demonstrated using in vitro enzyme assays Additionally, western blot analysis confirmed protein expression from all four open reading frames.<br>
Stiam YPH500, transformed with the group of plasmids pMCKlO, pMCKl 7, pMCK30 and pMCK35, was grown on Supplemented Minimal Medium containing 0 67% yeast mtrogen base without amino acids 2% glucose 20 mg/L adenine sulfate, and 30 mg/L L-lysme Cells were homogenized and extracts assayed for dhaB activity A specific activity of 0 12 units per mg protein was obtained for glycerol dehydratase, and 0 024 units per mg protein for 1,3-propanediol oxido-reductase<br>
EXAMPLE 4 PRODUCTION OF 1,3-PROPANEDIOL FROM D-GLUCOSE USING RECOMBINANT Saccharomvces cerevisiae S cerevisiae YPH500, harboring the groups of plasmids pMCKlO, pMCKl 7, pMCK30 and pMCK35, was grown in a BiostatB fermenter (B Braun Biotech, Inc ) in 1.0 L of minimal medium initially containing 20 g/L glucose, 6 7 g/L yeast nitrogen base without amino acids, 40 mg/L adenine sulfate and 60 mg/L L-lysme HCI. During the course of the growth, an addition.il equivalent of yeast nitrogen base, adenine and lysine was added The fermenter was controlled at pH 5 5 with addition of 10% phosphoric acid and 2 M NaOH, 30 °C, and 40% dissolved oxygen tension through agitation control After 38 h, the cells (OD600 = 58 AU) were harvested by centrifugation and resuspended in base medium (6 7 g/L yeast nitrogen base without amino acids, 20 mg/L adenine sulfate, 30 mg/L L-lysineHCl, and 50 mM potassium phosphate buffer, pH 7 0)<br><br>
Reaction mixtures containing cells (OD600 = 20 AU) in a total volume of 4 mL of base media supplemented with 0 5% glucose, 5 µg/mL coenzyme B12 and 0. 10, 20. or 40 mM chloroquine were prepared, in the absence of light and oxygen (nitrogen sparging) in 10 mL crimp sealed serum bottles and incubated at 30 °C with shaking After 30 h. aliquots were withdrawn and analyzed by HPLC The results are shown in the Table 3<br><br>
Table 3 Production of 1,3-propanediol using recombinant S cerevisiae<br>
reaction<br>
chloroquine (mM)<br>
1,3-propanediol (mM)<br>
1<br>
0<br>
02<br>
2<br>
10<br>
02<br>
3<br>
20<br>
03<br>
4<br>
40<br>
07<br>
EXAMPLE 5<br>
USE OF A S cerevisiae DOUBLE TRANSFORMANT FOR PRODUCTION<br>
OF 1.3-PROPANEDIOL FROM D-GLUCOSE WHERE dhaB AND dhaT ARE<br>
INTEGRATED INTO THE GENOME<br>
Example 5 phrophetically demonstrates the transformation of S cerevisiae with dhaB1, dhaB2, dhaB3, and dhaT and the stable integration of the genes into the yeast genome for the production of 1,3-propanediol from glucose Construction of expression cassettes<br>
Four expression cassettes (dhaBl, dhaB2, dhaB3, and dhaT) are constructed for glucose-induced and high-level constitutive expression of these genes in yeast, Saccharomyces cerevisiae These cassettes consist of (l) the phosphoglycerate kinase (PGK) promoter from S cerevisiae strain S288C, (n) one of the genes dhaBl, dhaB2, dhaB3, or dhaT, and (in) the PGK terminator from S cerevisiae strain S288C The PCR-based technique of gene splicing by overlap extension (Horton et al, BioTechniques, 8 528-535, (1990)) is used to recombine DNA sequences to generate these cassettes with seamless joints for optimal expression of each gene These cassettes are cloned individually into a suitable vector (pLITMUS 39) with restriction sites amenable to multi-cassette cloning in yeast expression plasmids Construction of veast integration vectors<br>
Vectors used to effect the integration of expression cassettes into the yeast genome are constructed These vectors contain the following elements (0 a polycloning region into which expression cassettes are subcloned, (ii) a unique marker used to select for stable yeast transformants, (iii) replication origin and<br><br>
selectable marker allowing gene manipulation in E coli prior to transforming yeast. One integration vector contains the URA3 auxotrophic marker (YIp352b), and a second integration vector contains the LYS2 auxotrophic marker (pKP7) Construction of yeast expression plasmids<br>
Expression cassettes for dhaB1 and dhaB2 are subcloned into the polyclomng region of the YIp352b (expression plasmid #1), and expression cassettes for dhaB3 and dhaT are subcloned into the polycloning region of pKP7 (expression plasmid #2) Transformation of yeast with expression plasmids<br>
S cerevisiae (ura3, lys2) is transformed with expression plasmid #1 using Frozen-EZ Yeast Transformation kit (Zymo Research, Orange, CA), and transformants selected on plates lacking uracil Integration of expression cassettes for dhaBl and dhaB2 is confirmed by PCR analysis of chromosomal DNA Selected transformants are re-transformed with expression plasmid #2 using Frozen-EZ Yeast Transformation kit, and double transformants selected on plates lacking lysine Integration of expression cassettes for dhaB3 and dhaT is confirmed by PCR analysis of chromosomal DNA The presence of all four expression cassettes (dhaBl, dhaB2, dhaB3, dhaT) in double transformants is confirmed by PCR analysis of chromosomal DNA Protein production from double-transformed veast<br>
Production of proteins encoded by dhaBl, dhaB2, dhaB3 and dhaT'from double-transformed yeast is confirmed by Western blot analysis Enzyme activity from double-transformed veast<br>
Active glycerol dehydratase and active 1,3-propanediol dehydrogenase from double-transformed yeast is confirmed by enzyme assay as described in General Methods above Production of 13 -propanediol from double-transformed veast<br>
Production of 1,3-propanediol from glucose in double-transformed yeast is demonstrated essentially as described in Example 4<br>
EXAMPLE 6<br>
CONSTRUCTION OF PLASMIDS CONTAINING DAR1/GPP2<br>
OR dhaT/dhaBl-3 AND TRANSFORMATION INTO KLEBSIELLA SPECIES<br>
K pneumoniae (ATCC 25955), K pneumoniae (ECL2106), and K oxytoca (ATCC 8724) are naturally resistant to ampicillin (up to 150 µg/mL) and kanamycin (up to 50 µg/mL), but sensitive to tetracycline (10 µg/mL) and chloramphenicol (25 µg/mL) Consequently, replicating plasmids which encode resistance to these latter two antibiotics are potentially useful as cloning vectors for these Klebsiella strains The wild-type K pneumoniae (ATCC 25955), the<br><br>
glucose-derepressed K pneumonia (ECL2106), and K oxytoca (ATCC 8724) were successfully transformed to tetracycline resistance by electroporation with the moderate-copy-number plasmid, pBR322 (New England Biolabs, Beverly MA) This was accomplished by the following procedure Ten mL of an overnight culture was inoculated into 1 L LB (1% (w/v) Bacto-tryptone (Difco. Detroit MI), 0 5% (w/v) Bacto-yeast extract (Difco) and 0 5% (w/v) NaCl (Sigma. St Louis, MO) and the culture was incubated at 37 °C to an OD600 of 0 5-0 7 The cells were chilled on ice, harvested by centrifugation at 4000 x g for 15 mm, and resuspended in 1 L ice-cold sterile 10% glycerol The cells were repeatedly harvested by centrifugation and progressively resuspended in 500 mL, 20 mL and, finally, 2 mL ice-coid sterile 10% glycerol For electroporation, 40 uL of cells were mixed with 1-2 uL DNA in a chilled 0 2 cm cuvette and were pulsed at 200 O, 2 5 kV for 4-5 msec using a BioRad Gene Pulser (BioRad, Richmond, CA) One µL of SOC medium (2% (w/v) Bacto-tryptone (Difco), 0 5% (w/v) Bacto-yeast extract (Difco), 10 µM NaCl, 10 µM MgCl2,10 µM MgSO4- 2 5 µM KCl and 20 µM glucose) was added to the cells and, after the suspension was transferred to a 17 x 100 mm sterile polypropylene tube, the culture was incubated for 1 hr at 37 °C, 225 rpm Aliquots were plated on selective medium, as indicated Analyses of the plasmid DNA from independent tetracycline-resistant transformants showed the restriction endonuclease digestion patterns typical of pBR322, indicating that the vector was stably maintained after overnight culture at 37 °C in LB containing tetracycline (10 µg/mL) Thus, this vector, and derivatives such as pBR329 (ATCC 37264) which encodes resistance to ampicillin, tetracycline and chloramphenicol, may be used to introduce the DAR1/GPP2 and dhaT/dhaBl-3 expression cassettes into K pneumoniae and K oxytoca<br>
The DAR1 and GPP2 genes may be obtained by PCR-mediated amplification from the Saccharomyces cerevisiae genome, based on their known DNA sequence The genes are then transformed into K pneumoniae or K oxytoca under the control of one or more promoters that may be used to direct their expression in media containing glucose. For convenience, the genes were obtained on a 2 4 kb DNA fragment obtained by digestion of plasmid pAH44 with the PvuII restriction endonuclease, whereby the genes are already arranged in an expression cassette under the control of the E coli lac promoter This DNA fragment was ligated to PvuII-digested pBR329, producing the msertional inactivation of its chloramphenicol resistance gene The hgated DNA was used to transform E coli DH5a (Gibco, Gaithersberg, MD) Transformants were selected by their resistance to tetracycline (10 µg/mL) and were screened for their<br><br>
sensitivity to chloramphenicol (25 µg/mL) Analysis of tie plasmid DNA from tetracychne-resistant, chloramphemcol-sensitive transrormants confirmed the presence of the expected plasmids, in which thePlac-darl-gpp2 expression cassette was subcloned in either orientation into the pBR329 Pvull site These plasmids, designated pJSPlA (clockwise orientation) and pJSPlB (counterclockwise orientation), were separately transformed by eiectroporation into K pneumonia (ATCC 25955), K pneumonia (ECL2106) and K oxytoca (ATCC 8724) as described Transformants were selected by their resistance to tetracycline (10 µg/mL) and were screened for their sensitivity to chloramphenicol (25 µg/mL) Restriction analysis of the plasmids isolated from independent transformants showed only the expected digestion patterns, and confirmed that they were stably maintained at 37 °C with antibiotic selection The expression of the DAR1 and GPP2 genes may be enhanced by the addition of IPTG (0 2-2 0 mM) to the growth medium<br>
The four K pneumoniae dhaB(l-3) and dhaT genes may be obtained by PCR-mediated amplification from the K pneumoniae genome, based on their known DNA sequence These genes are then transformed into K pneumoniae under the control of one or more promoters that may be used to direct their expression in media containing glucose For convenience, the genes were obtained on an approximately 4 0 kb DNA fragment obtained by digestion of plasmid pAH24 with the KpnI/SacI restriction endonucleases, whereby the genes are already arranged in an expression cassette under the control of the E coli lac promoter This DNA fragment was ligated to similarly digested pBC-KS+ (Stratagene, LaJolla, CA) and used to transform E coli DH5ct Transformants were selected by their resistance to chloramphenicol (25 µg/mL) and were screened for a white colony phenotype on LB agar containing X-gal Restriction analysis of the plasmid DNA from chloramphenicol-resistant transformants demonstrating the white colony phenotype confirmed the presence of the expected plasmid, designated pJSP2, in which the dhaT-dhaB(l-3) genes were subcloned under the control of the E coli lac promoter<br>
To enhance the conversion of glucose to 3G, this plasmid was separately transformed by eiectroporation into K pneumoniae (ATCC 25955) (pJSPl A), K pneumoniae (ECL2106) (pJSPl A) and K oxytoca (ATCC 8724) (pJSPl A) already containing the VXac-darl-gpp2 expression cassette Cotransforrnants were selected by their resistance to both tetracycline (10 µg/mL) and chloramphenicol (25 µg/mL). Restriction analysis of the plasmids isolated from independent cotransformants showed the digestion patterns expected for both pJSPl A and<br><br>
pJSP2 The expression of the DARl. GPP2, dhaB(l-3). and dhuT genes may be enhanced by the addition of IPTG (0 2-2 0 mM) to the medium<br>
EXAMPLE 7 Production of 1,3 propanediol from glucose by K pneumoniae<br>
Klebsiella pneumoniae strains ECL 2106 and 2106-47, both transformed with pJSPlA, and ATCC 25955, transformed with pJSPl A and pJSP2. were grown in a 5 L Applikon fermenter under various conditions (see Table 4) for the production of 1,3-propanediol from glucose Strain 2106-47 is a fluoroacetate-tolerant derivative of ECL 2106 which was obtained from a fluoroacetate/lactate selection plate as described in Bauer et al, Appl Environ Microbiol 56, 1296 (1990) In each case, the medium used contained 50-100 mM potassium phosphate buffer, pH 7 5, 40 mM (NH4)2SO4, 0 1% (w/v) yeast extract, 10 µM CoCl2, 6 5 µM CuCl2, 100 µM FeCl3, 18 µM FeSO4, 5 µM H3BO3, 50 ?M MnCl2, 0 1 µM Na2MoO4, 25 µM ZnCl2, 0 82 mM MgSO4, 0 9 mM CaCl2, and 10-20 g/L glucose Additional glucose was fed, with residual glucose maintained in excess Temperature was controlled at 37 °C and pH controlled at 7 5 with 5N KOH or NaOH Appropnate antibiotics were included for plasmid maintenance, IPTG (isopropyl-P-D-thiogalactopyranoside) was added at the indicated concentrations as well For anaerobic fermentations, 0 1 vvm nitrogen was sparged through the reactor, when the dO setpoint was 5%, 1 vvm air was sparged through the reactor and the medium was supplemented with vitamin B12 Final concentrations and overall yields (g/g) are shown in Table 4<br>
Table 4 Production of 1,3 propanediol from glucose by K pneumoniae<br><br>
Organism<br>
dO<br>
IPTG, mM<br>
vitamin B12, mg/L<br>
Titer, g/L<br>
Yield, g/g<br>
25955[pJSPlA/pJSP2]<br>
0<br>
05<br>
0<br>
8 1<br>
16%<br>
25955[pJSPlA/pJSP2]<br>
5%<br>
02<br>
05<br>
52<br>
4%<br>
2106[pJSPlA]<br>
0<br>
0<br>
0<br>
49<br>
17%<br>
2106[pJSPlA]<br>
5%<br>
0<br>
5<br>
65<br>
12%<br>
2106-47[pJSPlA]<br>
5%<br>
02<br>
05<br>
10 9<br>
12%<br>
43<br><br>
EXAMPLE 8 Conversion of carbon substrates to 1.3-propanediol bv recombinant K pneumoniae containing dar1, SDD2. dhaB. and dhdT A Conversion of D-fructose to 1,3-propanediol by various K pneumoniae recombinant strains<br>
Single colonies of K pneumoniae (ATCC 25955 pJSPIA), K pneumoniae (ATCC 25955 pJSPlA/pJSP2), K pneumoniae (ATCC 2106 pJSP1A), and K pneumoniae (ATCC 2106 pJSPl A/pJSP2) were transferred from agar plates and in separate culture tubes were subcultured overnight in Luna-Bertani (LB) broth containing the appropriate antibiotic agent(s) A 50-mL flask containing 45 mL of a steri-filtered minimal medium defined as LLMM/F which contains per liter 10 g fructose, 1 g yeast extract, 50 mmoles potassium phosphate, pH 7 5, 40 mmoles (NH4)2SO4; 0 09 mmoles calcium chloride; 2 38 mg CoCl2?6H2O, 0 88 mg CuCl2?2H2O, 27 mg FeCl3-6H2O; 5 mg FeSO4?7H2O, 0 31 mg H3BO3, 10 mg MnCl2?4H2O, 0 023 mg Na2MoO4?2H2O, 3 4 mg ZnCl2, 0 2 g MgSO4?7H20 Tetracycline at 10 µg/mL was added to medium for reactions using either of the single plasmid recombinants, 10 µg/mL tetracycline and 25 µg/mL chloramphenicol for reactions using either of the double plasmid recombinants. The medium was thoroughly sparged with nitrogen pnor to inoculation with 2 mL of the subculture IPTG (I) at final concentration of 0 5 mM was added to some flasks The flasks were capped, then incubated at 37 °C, 100 rpm in a New Brunswick Senes 25 incubator/shaker Reactions were run for at least 24 hours or until most of the carbon substrate was converted into products Samples were analyzed by HPLC Table 5 describes the yields of 1,3-propanediol produced from fructose by the various Klebsiella recombinants<br><br><br><br>
Table 5<br><br><br>
Production of 1,3-propanediol from D-fructose using recombinant Klebsiella<br>
Klebsiella Strain<br>
Medium<br>
Conversion<br>
[3G] (B/L)<br>
0<br>
Yield Carbon (%)<br>
2I06pBR329<br>
LLMM/F<br>
100<br><br><br>
0<br>
2I06pJSPlA<br>
LLMM/F<br>
50<br>
0 66<br>
155<br>
2106pJSPlA<br>
LLMM/F+ 1<br>
100<br>
011<br>
1 4<br>
2106pJSPlA/pJSP2<br>
LLMM/F<br>
58<br>
0 26<br>
5<br>
25955 pBR329<br>
LLMM/F<br>
100<br>
0<br>
0<br>
25955 pJSPI A<br>
LLMM/F<br>
100<br>
03<br>
4<br>
25955 pJSPlA<br>
LLMM/F +1<br>
100<br>
0 15<br>
2<br>
25955 pJSPlA/pJSP2<br>
LLMM/F<br>
100<br>
09<br>
11<br>
25955 pJSPlA/pJSP2<br>
LLMM/F +1<br>
62<br>
10<br>
20<br>
44<br><br>
B Conversion of various carbon substrates to 1.3-propanediol by K pneumoniae (ATCC 25955 pJSPlA/pJSP2)<br>
An aliquot (0 1 mL) of frozen stock cultures of K pneumoniae (ATCC 25955 pJSPl A/pJSP2) was transferred to 50 mL Seed medium in a 250 mL baffled flask The Seed medium contained per liter 0 1 molar NaK/PO buffer. pH 7 0, 3 g (NH4)2SO4, 5 g glucose, 0 15 g MgSO4?7H2O, 10 mL 100X Trace Element solution, 25 mg chloramphenicol, 10 mg tetracycline, and 1 g yeast extract The 100X Trace Element contained per liter 10 g citric acid, 1 5 g " CaCl2?2H2O. 2 8 g FeSO4?7H2O, 0 39 g ZnSO4?7H2O, 0 38 g CuSO4?5H2O. 0 2 g CoCl2?6H2O, and 0 3 g MnCl2?4H2O. The resulting solution was titrated to pH 7 0 with either KOH or H2SO4 . The glucose, trace elements, antibiotics and yeast extracts were sterilized separately. The seed inoculum was grown overnight at 35 °C and 250 rpm<br>
The reaction design was semi-aerobic The system consisted of 130 mL Reaction medium in 125 mL sealed flasks that were left partially open with aluminum foil stop The Reaction Medium contained per liter 3 g (NH4)2S04, 20 g carbon substrate, 0.15 molar NaK/P04 buffer, pH 7 5, 1 g yeast extract, 0 15 g MgSO4?7H2O, 0 5 mmoles IPTG, 10 mL 100X Trace Element solution, 25 mg chloramphenicol, and 10 mg tetracycline The resulting solution was titrated to pH 7 5 with KOH or H2SO4 The carbon sources were D-glucose (Glc), D-fructose (Frc), D-lactose (Lac), D-sucrose (Suc), D-maltose (Mal), and D-mannitol (Man) A few glass beads were included in the medium to improve mixing The reactions were initiated by addition of seed inoculum so that the optical density of the cell suspension started at 0 1 AU as measured at ?600 The flasks were incubated at 35 °C 250 rpm 3G production was measured by HPLC after 24 hr Table 6 describes the yields of 1,3-propanediol produced from the various carbon substrates<br>
Table 6<br>
Production of 1,3-propanediol from various carbon substrates<br>
using recombinant Klebsiella 25955 pJSPlA/pJSP2<br><br><br><br>
l,3-Propanediol(g/L)<br><br>
Carbon Substrate<br>
Expt 1<br>
Expt 2<br>
Expt 3<br>
Glc<br>
0 89<br>
1<br>
1 6<br>
Frc<br>
0 19<br>
0 23<br>
0 24<br>
Lac<br>
0 15<br>
0 58<br>
0 56<br>
Suc<br>
0 88<br>
0 62<br><br>
Mal<br>
0 05<br>
0 03<br>
0 02<br>
Man<br>
0 03<br>
0 05<br>
0 04<br><br>
A^<br>
45<br><br>
SEQUENCE LISTING (1) GENERAL INFORMATION<br>
(2) APPLICANT<br>
(A) ADDRESSEE E I DU PONT DE NEMOURS AND COMPWI:<br>
(B) STREET 1007 MARKET STREET<br>
(C) CITY WILMINGTON<br>
(D) STATE DELAWARE<br>
(E)	COUNTRY USA<br>
(F)	ZIP 19898<br>
(G)	TELEPHONE 302-892-8112<br>
(H)	TELEFAX 302-7 7 3-0164<br>
(I)	TELEX 6717325<br>
(A) ADDRESSEE GENENCOR INTERNATIONAL, INC<br>
(B) STREET 4 CAMBRIDGE PLACE<br>
1870 SOUTH WINTON ROAD<br>
(C) CITY ROCHESTER<br>
(D) STATE NEW YORK<br>
(E)	COUNTRY U S.A<br>
(F)	POSTAL CODE {ZIP}: 14618<br>
(II) TITLE OF INVENTION METHOD FOR THE RECOMBINANT<br>
PRODUCTION OF 1,3-PROPANEDIOL<br>
(III) NUMBER OF SEQUENCES 49<br>
(IV) COMPUTER READABLE FORM.<br>
(A) MEDIUM TYPE: 3 50 INCH DISKETTE<br>
(B) COMPUTER IBM PC COMPATIBLE<br>
(C) OPERATING SYSTEM MICROSOFT WORD FOR WINDOWS 95<br>
(D) SOFTWARE- MICROSOFT WORD VERSION 7 0A<br>
(V) CURRENT APPLICATION DATA<br>
(A) APPLICATION NUMBER<br>
(B) FILING DATE:<br>
(C) CLASSIFICATION<br>
(VI) PRIOR APPLICATION DATA<br>
(A) APPLICATION NUMBER 60/030,601<br>
(B) FILING DATE? NOVEMBER 13, 1996<br>
(Vll) ATTORNEY/AGENT INFORMATION:<br>
(A) NAME FLOYD, LINDA AXAMETHY<br>
(B) REGISTRATION NO . 33,692<br>
(C) REFERENCE/DOCKET NUMBER CR-9982<br>
(A) <br>
(2) INFORMATION FOR SEQ ID NO 1<br>
(1) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 1668 base pairs (3) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(III) HYPOTHETICAL NO<br>
(IV) ANTI-SENSE NO<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM DHABI<br>
(XI) SEQUENCE DESCRIPTION SEQ ID NO 1<br>
ATGAAAAGAT CAAAACGATT TGCAGTACTG GCCCAGCGCC CCGTCAATCA GGACGGGCTG	60<br>
ATTGGCGAGT GGCCTGAAGA GGGGCTGATC GCCATGGACA GCCCCTTTGA CCCGGTCTCT	120<br>
TCAGTAAAAG TGGACAACGG TCTGATCGTC GAACTGGACG GCAAACGCCG GGACCAGTTT	180<br>
GACATGATCG ACCGATTTAT CGCCGATTAC GCGATCAACG TTGAGCGCAC AGAGCAGGCA	24 0<br>
ATGCGCCTGG AGGCGGTGGA AATAGCCCGT ATGCTGGTGG ATATTCACGT CAGCCGGGAG	300<br>
GAGATCATTG CCATCACTAC CGCCATCACG CCGGCCAAAG CGGTCGAGGT GATGGCGCAG	360<br>
ATGAACGTGG TGGAGATGAT GATGGCGCTG CAGAAGATGC GTGCCCGCCG GACCCCCTCC	4 20<br>
AACCAGTGCC ACGTCACCAA TCTCAAAGAT AATCCGGTGC AGATTGCCGC TGACGCCGCC	4 80<br>
GAGGCCGGGA TCCGCGGCTT CTCAGAACAG GAGACCACGG TCGGTATCGC GCGCTACGCG	540<br>
CCGTTTAACG CCCTGGCGCT GTTGGTCGGT TCGCAGTGCG GCCGCCCCGG CGTGTTGACG	600<br>
CAGTGCTCGG TGGAA.GAGGC CACCGAGCTG GAGCTGGGCA TGCGTGGCTT AACCAGCTAC	660<br>
GCCGAGACGG TGTCGGTCTA CGGCACCGAA GCGGTATTTA CCGACGGCGA TGATACGCCG	720<br>
TGGTCAAAGG CGTTCCTCGC CTCGGCCTAC GCCTCCCGCG GGTTGAAAAT GCGCTACACC	780<br>
TCCGGCACCG GATCCGAAGC GCTGATGGGC TATTCGGAGA GCAAGTCGAT GCTCTACCTC	840<br>
GAATCGCGCT GCATCTTCAT TACTAAAGGC GCCGGGGTTC AGGGACTGCA AAACGGCGCG	900<br>
GTGAGCTGTA TCGGCATGAC CGGCGCTGTG CCGTCGGGCA TTCGGGCGGT GCTGGCGGAA	960<br>
AACCTGATCG C'TCTATGCT CGACCTCGAA GTGGCGTCCG CCAACGPCCA GACTTTCTCC	1020<br>
CACTCGGATA TTCGCCGCAC CGCGCGCACC CTGATGCAGA TGCTGCCGGG CACCGACTTT	1080<br>
ATTTTCTCCG GCTACAGCGC GGTGCCGAAC TACGACAACA TGTTCGCCGG CTCGAACTTC	1140<br>
GATGCGGAAG ATTTTGATGA TTACAACATC CTGCAGCGTG ACCTGATGGT TGACGGCGGC	1200<br>
CTGCGTCCGG TGACCGAGGC GGAAACCATT GCCATTCGCC AGAAAGCGGC GCGGGCGATC	1260<br>
CAGGCGGTTT TCCGCGAGCT GGGGCTGCCG CCAATCGCCG ACGAGGAGGT GGAGGCCGCC	1320<br>
ACCTACGCGC ACGGCAGCAA CGAGATGCCG CCGCGTAACG TGGTGGAGGA TCTGAGTGCG	1380<br>
GTGGAAGAGA TGATGAAGCG CAACATCACC GGCCTCGATA TTGTCGGCGC GCTGAGCCGC	14 40<br>
47<br><br>
AGCGGCTTTG AGGATATCGC CAGCAATATT CTCAATATGC TGCGCCAGCG GGTCACCGGC	1500<br>
GPTTACCTGC AGACCTCGGC CATTCTCGAT CGGCAGTTCG AGGTGGTGAG TGCGGTCAAC	1560<br>
GACATCAATG ACTATCAGGG GCGGGGCACC GGCTATCGCA TCTCTGCCGA ACGCTGGGCG	1620<br>
GAGATCAAAA ATATTCCGGG CGTGGTTCAG CCCGACACCA TTGAATAA	1668<br>
(2) INFORMATION FOR SEQ ID NO 2<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 585 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM DHAB2<br>
(xI) SEQUENCE DESCRIPTION: SEQ ID NO.2-<br>
GTGCAACAGA CAACCCAAAT TCAGCCCTCT TTTACCCTGA AAACCCGCGA GGGCGGGGTA	60<br>
GCTTCTGCCG ATGAACGCGC CGATGAAGTG GTGATCGGCG TCGGCCCTGC CTTCGATAAA	120<br>
CACCAGCATC ACACTCTGAT CGATATGCCC CATGGCGCGA TCCTCAAAGA GCTGATTGCC	180<br>
GGGGTGGAAG AAGAGGGGCT TCACGCCCGG GTGGTGCGCA TTCTGCGCAC GTCCGACGTC	24 0<br>
TCCTTTATGG CCTGGGATGC GGCCAACCTG AGCGGCTCGG GGATCGGCAT CGGTATCCAG	300<br>
TCGAAGGGGA CCACGGTCAT CCATCAGCGC GATCTGCTGC CGCTCAGCAA CCTGGAGCTG	360<br>
TTCTCCCAGG CGCCGCTGCT GACGCTGGAG ACCTACCGGC AGATTGGCAA AAACGCTGCG	420<br>
CGCTATGCGC GCAAAGAGTC ACCTTCGCCG GTGCCGGTGG TGAACGATCA GATGGTGCGG	4 80<br>
CCGAAATTTA TGGCCAAAGC CGCGCTATTT CATATCAAAG AGACCAAACA TGTGGTGCAG	540<br>
GACGCCGAGC CCGTCACCCT GCACATCGAC TTAGTAAGGG AGTGA	585<br>
(2) INFORMATION FOR SEQ ID NO.3<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 426 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS. single<br>
(D) TOPOLOGY- linear<br>
(n) MOLECULE TYPE DNA (genomic)<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM- DHAB3<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO•3.<br>
ATGAGCGAGA AAACCATGCG CGTGCAGGAT TATCCGTTAG CCACCCGCTG CCCGGAGCAT	60<br>
ATCCTGACGC CTACCGGCAA ACCATTGACC GATATTACCC TCGAGAAGGT GCTCTCTGGC	120<br>
GAGGTGGGCC CGCAGGATGT GCGGATCTCC CGCCAGACCC TTGAGTACCA GGCGCAGATT	180<br>
GCCGAGCAGA TGCAGCGCCA TGCGGTGGCG CGCAATTTCC GCCGCGCGGC GGAGCTTATC	24 0<br>
48<br><br>
GCCATTCCTG ACGAGCGCAT TCTGGCTATC TATAACGCGC TGCGCCCGTT CCGCTCCTCG 300<br>
CAGGCGGAGC, TGCTGGCGAT CGCCGACGAG CTGGAGCACA CGTGGGAGC GACAGTGAAT 360<br>
CCCCCCTTTG TCCGGGAGTC GGCGGAAGTG TATCAGCAGC GGCATMGCT GCGTAAAGGA<br>
AGCTAA<br>
426 (2) INFORMATION FOR SEQ ID NO 4<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 1164 base pairs<br>
(B) TYPE nucleic acid CO STRANDEDNESS single (D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM DHAT<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO-4<br>
ATGAGCTATC GTATQTTTGA TTATCTGGTG CCAAACGTTA ACTTTTTTGG CCCCAACGCC 60<br>
ATTTCCGTAG TCGGCGAACG CTGCCAGCTG CTGGGGGGGA AAAAAGCCCT GCTGGTCACC 120<br>
GACAAAGGCC TGCGGGCAAT TAAAGATGGC GCGGTGGACA AAACCCTGCA TTATCTGCGG 180<br>
GAGGCCGGGA TCGAGGTGGC GATCTTTGAC GGCGTCGAGC CGAACCCGAA AGACACCAAC 240<br>
GTGCGCGACG GCCTCGCCGT GTTTCGCCGC GAACAGTGCG ACATCATCGT C^CCGTGGGC 300<br>
GGCGGCAGCC CGCACGATTG CGGCAAAGGC ATCGGCATCG CCGCCACCCA TGAGGGCGAT 360<br>
CTGTACCAGT ATGCCGGAAT CGAGACCCTG ACCAACCCGC TGCCGCCTAT CGTCGCGGTC	4 20<br>
AATACCACCG CCGGCACCGC CAGCGAGGTC ACCCGCCACT GCGTCCTGAC CAACACCGAA 4 80<br>
ACCAAAGTGA AGTTTGTGAT CGTCAGCTGG CGCAAACTGC CGTCGGTCTC TATCAACGAT	540<br>
CCACTGCTGA TGATCGGTAA ACCGGCCGCC CTGACCGCGG CGACCGGGAT GGATGCCCTG	600<br>
ACCCACGCCG TAGAGGCCTA TATCTCCAAA GACGCTAACC CGGTGACGGA CGCCGCCGCC	660<br>
ATGCAGGCGA TCCGCCTCAT CGCCCGCAAC CTGCGCCAGG CCGTGGCCCT CGGCAGCAAT	7 20<br>
CTGCAGGCGC GGGAAAACAT GGCCTATGCT TCTCTGCTGG CCGGGATGGC TTTCAATAAC	780<br>
GCCAACCTCG GCTACGTGCA CGCCATGGCC CACCAGCTGG GCGGCCTGTA CGACATGCCG	840<br>
CACGGCGTGG CCAACGCTGT CCTGCTGCCC CATGTQGCGC GCTACAACCT GATCGCCAAC	900<br>
CCGGAGAAAT TCGCCGATAT CGCTGAACTG ATGGGCGAAA ATATCACCGG ACTGTCCACT	960<br>
CTCGACGCGG CGGAAAAAGC CATCGCCGCT ATCACGCGTC TGTCGATGGA TATCGGTATT	1020<br>
CCGCAGCATC TGCGCGATCT GGGGGTAAAA GAGGCCGACT TCCCCTACAT GGCGGAGATG	1080<br>
GCTCTAAAAG ACGGCAATGC GTTCTCGAAC CCGCGTAAAG GCAACGAGCA GGAGATTGCC	114 0<br>
GCGATTTTCC GCCAGGCATT CTGA	1164<br>
49<br><br>
(2) INFORMATION FOP SEQ ID NO 5<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 1380 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DN^ (genome)<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM GPD1<br>
(xi} SEQUENCE DESCRIPTION SEQ ID NO 5*<br>
CTTTAATTTT CTTTTATCTT ACTCTCCTAC ATAAGACATC AAGAAACAAT TGTATATTGT	60<br>
ACACCCCCCC CCTCCACAAA CACAAATATT GATAATATAA AGATGTCTGC TGCTGCTGAT	120<br>
AGATTAAACT TAACTTCCGG CCACTTGAAT GCTGGTAGAA AGAGAAGTTC CTCTTCTGTT	180<br>
TCTTTGAAGG CTGCCGAAAA GCCTTTCAAG GTTACTGTGA TTGGATCTGG TAACTGGGGT	240<br>
ACTACTATTG CCAAGGTGGT TGCCGAAAAT TGTAAGGGAT ACCCAGAAGT TTTCGCTCCA	300<br>
ATAGTACAAA TGTGGGTGTT CGAAGAAGAG ATCAATGGTG AAAAATTGAC TGAAATCATA	360<br>
AATACTAGAC ATCAAAACGT GAAATACTTG CCTGGCATCA CTCTACCCGA CAATTTGGTT	4 20<br>
GCTAATCCAG ACTTGATTGA TTCAGTCAAG GATGTCGACA TCATCGTTTT CAACATTCCA	4 80<br>
CATCAATTTT TGCCCCGTAT CTGTAGCCAA TTGAAAGGTC ATGTTGATTC ACACGTCAGA	540<br>
GCTATCTCCT GTCTAAAGGG TTTTGAAGTT GGTGCTAAAG GTGTCCAATT GCTATCCTCT	600<br>
TACATCACTG AGGAACTAGG TATTCAATGT GGTGCTCTAT CTGGTGCTAA CATTGCCACC	660<br>
GAAGTCGCTC AAGAACACTG GTCTGAAACA ACAGTTGCTT ACCACATTCC AAAGGATTTC	720<br>
AGAGGCGAGG GCAAGGACGT CGACCATAAG GTTCTAAAGG CCTTGTTCCA CAGACCTTAC	780<br>
TTCCACGTTA GTGTCATCGA AGATGTTGCT GGTATCTCCA TCTGTGGTGC TTTGAAGAAC	840<br>
GTTGTTGCCT TAGGTTGTGG TTTCGTCGAA GGTCTAGGCT GGGGTAACAA CGCTTCTGCT	900<br>
GCCATCCAAA GAGTCGGTTT GGGTGAGATC ATCAGATTCG GTCAAATGTT TTTCCCAGAA	960<br>
TCTAGAGAAG AAACATACTA CCAAGAGTCT GCTGGTGTTG CTGATTTGAT CACCACCTGC	1020<br>
GCTGGTGGTA GAAACGTCAA GGTTGCTAGG CTAATGGCTA CTTCTGGTAA GGACGCCTGG	1080<br>
GAATGTGAAA AGGAGTTGTT GAATGGCCAA TCCGCTC7.AG GTTTAATTAC CTGCAAAGAA	1140<br>
GTTCACGAAT GGTTGGAAAC ATGTGGCTCT GTCGAAGACT TCCCATTATT TGAAGCCGTA	1200<br>
TACCAAATCG TTTACAACAA CTACCCAATG AAGAACCTGC CGGACATGAT TGAAGAATTA	1260<br>
GATCTACATG AAGATTAGAT TTATTGGAGA AAGATAACAT ATCATACTTC CCCCACTTTT	1320<br>
TTCGAGGCTC TTCTATATCA TATTCATAAA TTAGCATTAT GTCATTTCTC ATAACTACTT	1380<br>
(2) INFORMATION FOR SEQ ID NO 6<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 2946 base pairs<br>
(B) TYPE nucleic acid<br>
50<br><br>
(C) STRANDEDNES3 single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(Vl) ORIGINAL SOURCE<br>
(A) ORGANISM GPD2<br>
(XI) SEQUENCE DESCRIPTION SEQ ID NO 6<br>
GAATTCGAGC CTGAAGTGCT GATTACCTTC AGGTAGACTT CATCTTGACC CATCAACCCC	60<br>
AGCGTCAATC CTGCAAATAC ACCACCCAGC AGCACTAGGA TGATAGAGAT AATATAGTAC	120<br>
GTGGTAACGC TTGCCTCATC ACCTACGCTA TGGCCGGAAT CGGCAACATC CCTAGAATTG	180<br>
AGTACGTGTG ATCCGGATAA CAACGGCAGT GAATATATCT TCGGTATCGT AAAGATGTGA	24 0<br>
TATAAGATGA TGTATACCCA ATGAGGAGCG CCTGATCGTG ACCTAGACCT TAGTGGCAAA	300<br>
AACGACATAT CTATTATAGT GGGGAGAGTT TCGTGCAAAT AACAGACGCA GCAGCAAGTA	360<br>
ACTGTGACGA TATCAACTCT TTTTTTATTA TGTAATAAGC AAACAAGCAC GAATGGGGAA	4 20<br>
AGCCTATGTG CAATCACCAA GGTCGTCCCT TTTTTCCCAT TTGCTAATTT AGAATTTAAA	4 80<br>
GAAACCAAAA GAATGAAGAA AGAAAACAAA TACTAGCCCT AACCCTGACT TCGTTTCTAT	54 0<br>
GATAATACCC TGCTTTAATG AACGGTATGC CCTAGGGTAT ATCTCACTCT GTACGTTACA	600<br>
AACTCCGGTT ATTTTATCGG AACATCCGAG CACCCGCGCC TTCCTCAACC CAGGCACCGC	660<br>
CCCAGGTAAC CGTGCGCGAT GAGCTAATCC TGAGCCATCA CCCACCCCAC CCGTTGATGA	720<br>
CAGCAATTCG GGAGGGCGAA AATAAAACTG GAGCAAGGAA TTACCATCAC CGTCACCATC	780<br>
ACCATCATAT CGCCTTAGCC TCTAGCCATA GCCATCATGC AAGCGTGTAT CTTCTAAGAT	84 0<br>
TCAGTCATCA TCATTACCGA GTTTGTTTTC CTTCACATGA TGAAGAAGGT TTGAGTATGC	900<br>
TCGAAACAAT AAGACGACGA TGGCTCTGCC ATTGGTTATA TTACGCTTTT GCGGCGAGGT	960<br>
GCCGATGGGT TGCTGAGGGG AAGAGTGTTT AGCTTACGGA CCTATTGCCA TTGTTATTCC	1020<br>
GATTAATCTA TTGTTCAGCA GCTCTTCTCT ACCCTGTCAT TCTAGTATTT TTTTTTTTTT	]080<br>
TTTTTGGTTT TACTTTTTTT TCTTCTTGCC TTTTTTTCTT GTTACTTTTT TTCTAGTTTT	1140<br>
TTTTCCTTCC ACTAAGCTTT TTCCTTGATT TATCCTTGGG TTCTTCTTTC TACTCCTTTA	1200<br>
GATTTTTTTT TTATATATTA ATTTTTAAGT TTATGTATTT TGGTAGATTC AATTCTCTTT	1260<br>
CCCTTTCCTT TTCCTTCGCT CCCCTTCCTT ATCAATGCTT GCTGTCAGAA GATTAACAAG	1320<br>
ATACACATTC CTTAAGCGAA CGCATCCGGT GTTATATACT CGTCGTGCAT ATAAAATTTT	1380<br>
GCCTTCAAGA TCTACTTTCC TAAGAAGATC ATTATTACAA ACACAACTGC ACTCAAAGAT	14 4 0<br>
GACTGCTCAT ACTAATATCA AACAGCACAA ACACTGTCAT GAGGACCATC CTATCAGAAG	1500<br>
ATCGGACTCT GCCGTGTCAA TTGTACATTT GAAACGTGCG CCCTTCAAGG TTACAGTGAT	1560<br>
TGGTTCTGGT AACTGGGGGA CCACCATCGC CAAAGTCATT GCGGAAAACA CAGAATTGCA	1620<br>
TTCCCATATC TTCGAGCCAG AGGTGAGAAT GTGGGTTTTT GATGAAAAGA TCGGCGACGA	1680<br>
49--S"J-<br><br>
AAATCTGATG GATATCATAA ATACAAGACA CCAGAACGTT AAATATCTPC CCAATATTGA	740<br>
CCTGCCCCAT AATCTAGTGG CCGATCCTGA TCTTTTACAC TCCATCAAGG GTGCTG^CAT	980<br>
CCTTGTTTTC AACATCCCTC ATCAATTTTT ACCAAACATA GTCAAACAAT TGCAAGGCC^	1860<br>
CCTGGCCCCT CATGTAAGGG CCATCTCGTG TCTAAAAGGG TTCGAGTTGG GCTCCAAGGG	1920<br>
TGTGCAATTG CTATCCTCCT ATGTTACTGA TGAGTTAGGA ATCCAATGTG GCGCACT&amp;7C	1980<br>
TGGTGCAAAC TTGGCACCGG AAGTGGCCAA GGAGCATTGG TCCGAAACCA CCGTGGCTTA	2040<br>
CCAACTACCA AAGGATTATC AAGGTGATGG CAAGGATGTA GATCATAAGA TTTTGAAATT	2100<br>
GCTGTTCCAC AGACCTTACT TCCACGTCAA TGTCATCGAT GATGTTGCTG GTATATCCAT	2160<br>
TGCCGGTGCC TTGAAGAACG TCGTGGCACT TGCATGTGGT TTCGTAGAAG GTATGGGATG	2220<br>
GGGTAACAAT GCCTCCGCAG CCATTCAAAG GCTGGGTTTA GGTGAAATTA TCAAGTTCGG	2280<br>
TAGAATGTTT TTCCCAGAAT CCAAAGTCGA GACCTACTAT CAAGAATCCG CTGGTGTTGC	2340<br>
AGATCTGATC ACCACCTGCT CAGGCGGTAG AAACGTCAAG GTTGCCACAT ACATGGCCAA	24 00<br>
GACCGGTAAG TCAGCCTTGG AAGCAGAAAA GGAATTGCTT AACGGTCAAT CCGCCCAAGG	2460<br>
GATAATCACA TGCAGAGAAG TTCACGAGTG GCTACAAACA TGTGAGTTGA CCCAAGAATT	2520<br>
CCCAATTATT CGAGGCAGTC TACCAGATAG TCTACAACAA CGTCCGCATG GAAGACCTAC	2580<br>
CGGAGATGAT TGAAGAGCTA GACATCGATG ACGAATAGAC ACTCTCCCCC CCCCTCCCCC	264 0<br>
TCTGATCTTT CCTGTTGCCT CTTTTTCCCC CAACCAATTT ATCATTATAC ACAAGTTCTA	2700<br>
CAACTACTAC TAGTAACATT ACTACAGTTA TTATAATTTT CTATTCTCTT TTTCTTTAAG	27 60<br>
AATCTATCAT TAACGTTAAT TTCTATATAT ACATAACTAC CATTATACAC GCTATTATCG	2820<br>
TTTACATATC ACATCACCGT TAATGAAAGA TACGACACCC TGTACACTAA CACAATTAAA	2880<br>
TAATCGCCAT AACCTTTTCT GTTATCTATA GCCCTTAAAG CTGTTTCTTC GAGCTTTTCA	2940<br>
CTGCAG	2946<br>
{2) INFORMATION FOR SEQ ID NO 7:<br>
(l) SEQUENCE CHARACTERISTICS.<br>
(A) LENGTH 3178 base pairs<br>
(B) TYPE: nucleic acid<br>
(C) STRANDEDNESS. single<br>
(D) TOPOLOGY linear<br>
(ll) MOLECULE TYPE DNA (genomic)<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM GUT2<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 7.<br>
CTGCAGAACT TCGTCTGCTC TGTGCCCATC CTCGCGGTTA GAAAGAAGCT GAATTGTTTC	60<br>
ATGCGCAAGG GCATCAGCGA GTGACCAATA ATCACTGCAC TAATTCCTTT TTAGCAACAC	120<br>
ATACTTATAT ACAGCACCAG ACCTTATGTC TTTTCTCTGC TCCGATACGT TATCCCACCC	180<br>
AACTTTTATT TCAGTTTTGG CAGGGGAAAT TTCACAACCC CGCACGCTAA AAATCGTATT	24 0<br><br>
TAAACTTAAA AGAGAACAGC CACAAATAGG GAACTTTGGT CTAAACGAAG GACTCTCCCT	300<br>
CCCTTATCTT GACCGTGCTA TTGCCATCAC TGCTACAAGA CTAAATACGT ACTAATATAT	360<br>
GTTTTCGGT^ ACGAGAAGAA GAGCTGCCGG TGCAGCTGCT GCCATGGCCA CAGCCACGGG	420<br>
GACGCTGTAC TGGATGACTA GCCAAGGTGA TAGGCCGTTA GTGCACAATG ACCCGAGCTA	4 80<br>
CATGGTGCAA TTCCCCACCG CCGCTCCACC GGCAGGTCTC TAGACGAGAC CTGCTGGACC	540<br>
GTCTGGACAA GACGCATCAA TTCGACGTGT TGATCATCGG TGGCGGGGCC ACGGGGACAG	600<br>
GATGTGCCCT AGATGCTGCG ACCAGGGGAC TCAATGTGGC CCTTGTTGAA AAGGGGGATT	660<br>
TTGCCTCGGG AACGTCGTCC AAATCTACCA AGATGATTCA CGGTGGGGTG CGGTACTTAG	720<br>
AGAAGGCCTT CTGGGAGTTC TCCAAGGCAC AACTGGATCT GGTCATCGAG GCACTCAACG	780<br>
AGCGTAAACA TCTTATCAAC ACTGCCCCTC ACCTGTGCAC GGTGCTACCA ATTCTGATCC	840<br>
CCATCTACAG CACCTGGCAG GTCCCGTACA TCTATATGGG CTGTAAATTC TACGATTTCT	900<br>
TTGGCGGTTC CCAAAACTTG AAAAAATCAT ACCTACTGTC CAAATCCGCC ACCGTGGAGA	960<br>
AGGCTCCCAT GCTTACCACA GACAATTTAA AGGCCTCGCT TGTGTACCAT GATGGGTCCT	1020<br>
TTAACGACTC GCGTTTGAAC GCCACTTTAG CCATCACGGG TGTGGAGAAC GGCGCTACCG	1080<br>
TCTTGATCTA TGTCGAGGTA CAAAAATTGA TCAAAGACCC AACTTCTGGT AAGGTTATCG	114 0<br>
GTGCCGAGGC CCGGGACGTT GAGACTAATG AGCTTGTCAG AATCAACGCT AAATGTGTGG	1200<br>
TCAATGCCAC GGGCCCATAC AGTGACGCCA TTTTGCAAAT GGACCGCAAC CCATCCGGTC	1260<br>
TGCCGGACTC CCCGCTAAAC GACAACTCCA AGATCAAGTC GACTTTCAAT CAAATCTCCG	1320<br>
TCATGGACCC GAAAATGGTC ATCCCATCTA TTGGCGTTCA CATCGTATTG CCCTCTTTTT	1380<br>
ACTCCCCGAA GGATATGGGT TTGTTGGACG TCAGAACCTC TGATGGCAGA GTGATGTTCT	14 4 0<br>
TTTTACCTTG GCAGGGCAAA GTCCTTGCCG GCACCACAGA CATCCCACTA AAGCAAGTCC	1500<br>
CAGAAAACCC TATGCCTACA GAGGCTGATA TTCAAGATAT CTTGAAAGAA CTACAGCACT	1560<br>
ATATCGAATT CCCCGTGAAA AGAGAAGACG TGCTAAGTGC ATGGGCTGGT GTCAGACCTT	1620<br>
TGGTCAGAGA TCCACGTACA ATCCCCGCAG ACGGGAAGAA GGGCTCTGCC ACTCAGGGCG	1680<br>
TGGTAAGATC CCACTTCTTG TTCACTTCGG ATAATGGCCT AATTACTATT GCAGGTGGTA	1740<br>
AATGGACTAC TTACAGACAA ATGGCTGAGG AAACAGTCGA CAAAGTTGTC GAAGTTGGCG	18 00<br>
GATTCCACAA CCTGAAACCT TGTCACACAA GAGATATTAA GCTTGCTGGT GCAGAAGAAT	1860<br>
GGACGCAAAA CTATGTGGCT TTATTGGCTC AAAACTACCA TTTATCATCA AAAATGTCCA	192 0<br>
ACTACTTGGT TCAAAACTAC GGAACCCGTT CCTCTATCAT TTGCGAATTT TTCAAAGAAT	198 0<br>
CCATGGAAAA TAAACTGCCT TTGTCCTTAG CCGACAAGGA AAATAACGTA ATCTACTCTA	204 0<br>
GCGAGGAGAA CAACTTGGTC AATTTTGATA CTTTCAGATA TCCATTCACA ATCGGTGAGT	2100<br>
TAAAGTATTC CATGCAGTAC GAATATTGTA GAACTCCCTT GGACTTCCTT TTAAGAAGAA	2160<br>
CAAGATTCGC CTTCTTGGAC GCCAAGGAAG CTTTGAATGC CGTGCATGCC ACCGTCAAAG	2220<br><br>
TTATGGGTGA TGAGTTCAAT TGGTCGGAGA AAAAGAGGCA GTGGGAACTT GAAAAAACTG	2260<br>
TGAACTTCAT CCAAGGACGT TTCGGTGTCT AAATCGATCA TGATAGTTAA GGGTGACAAA	2340<br>
GATAACATTC ACAAGAGTAA TAATAATGGT AATGATGATA ATAATAATAA TGATAGTAAT	2400<br>
AACAATAATA ATAATGGTGG TAATGGCAAT GAAATCGCTA TTATTACCTA TTTTCCTTAA	2460<br>
TGGAAGAGTT AAAGTAAACT AAAAAAACTA CAAAAATATA TGAAGAAAAA AAAAAAAAGA	2520<br>
GGTAATAGAC TCTACTACTA CAATTGATCT TCAAATTATG ACCTTCCTAG TGTTTATATT	2580<br>
CTATTTCCAA TACATAATAT AATCTATATA ATCATTGCTG GTAGACTTCC GTTTTAATAT	2 64 0<br>
CGTTTTAATT ATCCCCTTTA TCTCTAGTCT AGTTTTATCA TAAAATATAG AAACACTAAA	2700<br>
TAATATTCTT CAAACGGTCC TGGTGCATAC GCAATACATA TTTATGGTGC AAAAAAAAAA	27 60<br>
ATGGAAAATT TTGCTAGTCA TAAACCCTTT CATAAAACAA TACGTAGACA TCGCTACTTG	2820<br>
AAATTTTCAA GTTTTTATCA GATCCATGTT TCCTATCTGC CTTGACAACC TCATCGTCGA	2880<br>
AATAGTACCA TTTAGAACGC CCAATATTCA CATTGTGTTC AAGGTCTTTA TTCACCAGTG	2940<br>
ACGTGTAATG GCCAfGATTA ATGTGCCTGT ATGGTTAACC ACTCCAAATA GCTTATATTT	3000<br>
CATAGTGTCA TTGTTTTTCA ATATAATGTT TAGTATCAAT GGATATGTTA CGACGGTGTT	3060<br>
ATTTTTCTTG GTCAAATCGT AATAAAATCT CGATAAATGG ATGACTAAGA TTTTTGGTAA	3120<br>
AGTTACAAAA TTTATCGTTT TCACTGTTGT CAATTTTTTG TTCTTGTAAT CACTCGAG	3178<br>
(2) INFORMATION FOR SEQ ID NO*8<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 816 base pairs<br>
(B) TYPE nucleic acid {C) STRANDEDNESS. single (D) TOPOLOGY linear<br>
{ii) MOLECULE TYPE DNA (genomic)<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GPP1<br>
(Xl) SEQUENCE DESCRIPTION. SEQ ID NO:8:<br>
ATGAAACGTT TCAATGTTTT AAAATATATC AGAACAACAA AAGCAAATAT ACAAACCATC	60<br>
GCAATGCCTT TGACCACAAA ACCTTTATCT TTGAAAATCA ACGCCGCTCT ATTCGATGTT	120<br>
GACGGTACCA TCATCATCTC TCAACCAGCC ATTGCTGCTT TCTGGAGAGA TTTCGGTAAA	180<br>
GACAAGCCTT ACTTCGATGC CGAACACGTT ATTCACATCT CTCACGGTTG GAGAACTTAC	24 0<br>
GATGCCATTG CCAAGTTCGC TCCAGACTTT GCTGATGAAG AATACGTTAA CAAGCTAGAA	300<br>
GGTGAAATCC CAGAAAAGTA CGGTGAACAC TCCATCGAAG TTCCAGGTGC TGTCAAGTTG	360<br>
TGTAATGCTT TGAACGCCTT GCCAAAGGAA AAATGGGCTG TCGCCACCTC TGGTACCCGT	420<br>
GACATGGCCA AGAAATGGTT CGACATTTTG AAGATCAAGA GACCAGAATA CTTCATCACC	4 80<br>
GCCAATGATG TCAAGCAAGG TAAGCCTCAC CCAGAACCAT ACTTAAAGGG TAGAAACGGT	54 0<br><br>
TTGGGTTTCC CAATTAATGA ACAAGACCCA TCCAAATCTA AGGTTGTTGT CTTTGAAGAC	600<br>
GCACCAGCTG GTATTGCTGC TGGTAAGGCT GCTGGCTGTA AAATCGTTGG TATTGCTACC	660<br>
ACTTTCGATT TGGACTTCTT GAAGGAAAAG GGTTGTGACA TCATTGTCAA GAACCACGAA	720<br>
TCTATCAGAG TCGGTGAATA CAACGCTGAA ACCGATGAAG TCGAATTGAT CTTTGATGAC	780<br>
TACTTATACG CTAAGGATGA CTTGTTGAAA TGGTAA	816<br>
(2) INFORMATION FOR SEQ ID NO 9<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 753 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GPP2<br>
(xx) SEQUENCE DESCRIPTION SEQ ID NO 9<br>
ATGGGATTGA CTACTAAACC TCTATCTTTG AAAGTTAACG CCCCTTTGTT CGACGTCGAC	60<br>
GGTACCATTA TCATCTCTCA ACCAGCCATT GCTGCATTCT GGAGGGATTT CGGTAAGGAC	120<br>
AAACCTTATT TCGATGCTGA ACACGTTATC CAAGTCTCGC ATGGTTGGAG AACGTTTGAT	180<br>
GCCATTGCTA AGTTCGCTCC AGACTTTGCC AATGAAGAGT ATGTTAACAA ATTAGAAGCT	240<br>
GAAATTCCGG TCAAGTACGG TGAAAAATCC ATTGAAGTCC CAGGTGCAGT TAAGCTGTGC	300<br>
AACGCTTTGA ACGCTCTACC AAAAGAGAAA TGGGCTGTGG CAACTTCCGG TACCCGTGAT	360<br>
ATGGCACAAA AATGGTTCGA GCATCTGGGA ATCAGGAGAC CAAAGTACTT CATTACCGCT	4 20<br>
AATGATGTCA AACAGGGTAA GCCTCATCCA GAACCATATC TGAAGGGCAG GAATGGCTTA	4 80<br>
GGATATCCGA TCAATGAGCA AGACCCTTCC AAATCTAAGG TAGTAGTATT TGAAGACGCT	540<br>
CCAGCAGGTA TTGCCGCCGG AAAAGCCGCC GGTTGTAAGA TCATTGGTAT TGCCACTACT	600<br>
TTCGACTTGG ACTTCCTAAA GGAAAAAGGC TGTGACATCA TTGTCAAAAA CCACGAATCC	660<br>
ATCAGAGTTG GCGGCTACAA TGCCGAAACA GACGAAGTTG AATTCATTTT TGACGACTAC	720<br>
TTATATGCTA AGGACGATCT GTTGAAATGG TAA	7 53<br>
(2) INFORMATION FOR SEQ ID NO.10<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 2520 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
{ii) MOLECULE TYPE DNA (genomic)<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GUT1<br><br>
(xi) SEQUENCE DESCRIPTION SEC ID NO 10<br>
TGTATTGGCC ACGATAACCA CCCTTTGTAT ACTGTTTTTG TTTTTCACAT GGTAAATAAC	60<br>
GACTTTTATT AAACAACGTA TGTAAAAACA TAACAAGAAT CTACCCATAC AGGCCATTTC	120<br>
GTAATTCTTC TCTTCTAATT GGAGTAAAAC CATCAATTAA AGGGTGTGGA GTAGCATAGT	180<br>
GAGGGGCTGA CAGCATTGAC AAAAAAATTG AAAAAAAAAA AGGAAAAGGA AAGGAAAAAA	240<br>
AGACAGCCAA GACTTTTAGA ACGGATAAGG TGTAATAAAA TGTGGGGGGA TGCCTGTTCT	300<br>
CGAACCATAT AAAATATACC ATGTGGTTTG AGTTGTGGCC GGAACTATAC AAATAGTTAT	360<br>
ATGTTTCCCT CTCTCTTCCG ACTTGTAGTA TTCTCCAAAC GTTACATATT CCGATCAAGC	4 20<br>
CAGCGCCTTT ACACTAGTTT AAAACAAGAA CAGAGCCGTA TGTCCAAAAT AATGGAAGAT	4 80<br>
TTACGAAGTG ACTACGTCCC GCTTATCGCC AGTATTGATG TAGGAACGAC CTCATCCAGA	54 0<br>
TGCATTCTGT TCAACAGATG GGGCCAGGAC GTTTCAAAAC ACCAAATTGA ATATTCAACT	600<br>
TCAGCATCGA AGGGCAAGAT TGGGGTGTCT GGCCTAAGGA GACCCTCTAC AGCCCCAGCT	660<br>
CGTGAAACAC CAAACGCCGG TGACATCAAA ACCAGCGGAA AGCCCATCTT TTCTGCAGAA	7 20<br>
GGCTATGCCA TTCAAGAAAC CAAATTCCTA AAAATCGAGG AATTGGACTT GGACTTCCAT	7 80<br>
AACGAACCCA CGTTGAAGTT CCCCAAACCG GGTTGGGTTG AGTGCCATCC GCAGAAATTA	84 0<br>
CTGGTGAACG TCGTCCAATG CCTTGCCTCA AGTTTGCTCT CTCTGCAGAC TATCAACAGC	900<br>
GAACGTGTAG CAAACGGTCT CCCACCTTAC AAGGTAATAT GCATGGGTAT AGCAAACATG	960<br>
AGAGAAACCA CAATTCTGTG GTCCCGCCGC ACAGGAAAAC CAATTGTTAA CTACGGTATT	1020<br>
GTTTGGAACG ACACCAGAAC GATCAAAATC GTTAGAGACA AATGGCAAAA CACTAGCGTC	1080<br>
GATAGGCAAC TGCAGCTTAG ACAGAAGACT GGATTGCCAT TGCTCTCCAC GTATTTCTCC	114 0<br>
TGTTCCAAGC TGCGCTGGTT CCTCGACAAT GAGCCTCTGT GTACCAAGGC GTATGAGGAG	1200<br>
AACGACCTGA TGTTCGGCAC TGTGGACACA TGGCTGATTT ACCAATTAAC TAAACAAAAG	1260<br>
GCGTTCGTTT CTGACGTAAC CAACGCTTCC AGAACTGGAT TTATGAACCT CTCCACTTTA	1320<br>
AAGTACGACA ACGAGTTGCT GGAATTTTGG GGTATTGACA AGAACCTGAT TCACATGCCC	1380<br>
GAAATTGTGT CCTCATCTCA ATACTACGGT GACTTTGGCA TTCCTGATTG GATAATGGAA	14 40<br>
AAGCTACACG ATTCGCCAAA AACAGTACTG CGAGATCTAG TCAAGAGAAA CCTGCCCATA	1500<br>
CAGGGCTGTC TGGGCGACCA AAGCGCATCC ATGGTGGGGC AACTCGCTTA CAAACCCGGT	1560<br>
GCTGCAAAAT GTACTTATGG TACCGGTTGC TTTTTACTGT ACAATACGGG GACCAAAAAA	1620<br>
TTGATCTCCC AACATGGCGC ACTGACGACT CTAGCATTTT GGTTCCCACA TTTGCAAGAG	1680<br>
TACGGTGGCC AAAAACCAGA ATTGAGCAAG CCACATTTTG CATTAGAGGG TTCCGTCGCT	17 4 0<br>
GTGGCTGGTG CTGTGGTCCA ATGGCTACGT GATAATTTAC GATTGATCGA TAAATCAGAG	1800<br>
GATGTCGGAC CGATTGCATC TACGGTTCCT GATTCTGGTG GCGTAGTTTT CGTCCCCGCA	1860<br>
TTTAGTGGCC TATTCGCTCC CTATTGGGAC CCAGATGCCA GAGCCACCAT AATGGGGATG	1920<br><br>
TCTCAATTCA CTACTGCCTC CCACATCGCC AGAGCTGCCG TGGAAGGTGT TTGCTTTCAA	1980<br>
GCCAGGGCTA TCTTGAAGGC AATGAGTTCT GACGCGTTTG GTGAAGGTTC CAAAGACAGG	2040<br>
GACTTTTTAG AGGAAATTTC CGACGTCACA TATGAAAAGT CGCCCCTGTC GGTTCTGGCA	2100<br>
GTGGATGGCG GGATGTCGAG GTCTAATGAA GTCATGCAAA TTCAAGCCGA TATCCTAGGT	2160<br>
CCCTGTGTCA AAGTCAGAAG GTCTCCGACA GCGGAATGTA CCGCATTGGG GGCAGCCATT	2220<br>
GCAGCCAATA TGGCTTTCAA GGATGTGAAC GAGCGCCCAT TATGGAAGGA CCTACACGAT	2280<br>
GTTAAGAAAT GGGTCTTTTA CAATGGAATG GAGAAAAACG AACAAATATC ACCAGAGGCT	2340<br>
CATCCAAACC TTAAGATATT CAGAAGTGAA TCCGACGATG CTGAAAGGAG AAAGCATTGG	2400<br>
AAGTATTGGG AAGTTGCCGT GGAAAGATCC AAAGGTTGGC TGAAGGACAT AGAAGGTGAA	24 60<br>
CACGAACAGG TTCTAGAAAA CTTCCAATAA CAACATAAAT AATTTCTATT AACAATGTAA	2 520<br>
(2) INFORMATION FOR SEQ ID NO:11<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 391 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown {D) TOPOLOGY: unknown<br>
(u) MOLECULE TYPE. protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GPD1<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO:ll<br>
Met Ser Ala Ala Ala Asp Arg Leu Asn Leu Thr Ser Gly His Leu Asn<br>
15	10	15<br>
Ala Gly Arg Lys Arg Ser Ser Ser Ser Val Ser Leu Lys Ala Ala Glu<br>
20	25	30<br>
Lys Pro Phe Lys Val Thr Val Ile Gly Ser Gly Asn Trp Gly Thr Thr<br>
35	40	45<br>
Ile Ala Lys Val Val Ala Glu Asn Cys Lys Gly Tyr Pro Glu Val Phe<br>
50	55	60<br>
Ala Pro Ile Val Gln Met Trp Val Phe Glu Glu Glu Ile Asn Gly Glu<br>
65	70	75	80<br>
Lys Leu Thr Glu Ile Ile Asn Thr Arg His Gln Asn Val Lys Tyr Leu<br>
85	90	95<br>
Pro Gly Ile Thr Leu Pro Asp Asn Leu Val Ala Asn Pro Asp Leu Ile<br>
100	105	HO<br>
Asp Ser Val Lys Asp Val Asp Ile ILe Val Phe Asn Ile Pro His Gln<br>
115	120	125<br>
Phe Leu Pro Arg Ile Cys Ser Gln Leu Lys Gly His Val Asp Ser His<br>
130	135	140<br>
Val Arg Ala Ile Ser Cys Leu Lys Gly Phe Glu Val Gly Ala Lys Gly<br>
145	150	155	160<br><br>
57<br><br>
Val Gln Leu Lea Ser Ser Tyr lie Thr Glu Glu Leu Gly lie Gln Cys<br>
165	170	175<br>
Gly Ala Leu Ser Gly Ala Asn Ile Ala Thr Glu Val Ala Gln Glu His<br>
130	185	190<br>
Trp Ser Glu Thr Thr Val Ala Tyr His Ile Pro Lys Asp Phe *rg G'V<br>
195	200	205<br>
Glu Gly Lys Asp Val Asp his Lys Val Leu Lys Ala Leu Phe HIS Arg<br>
210	215	220<br>
Pro Tyr Phe His Val Ser Val Ile Glu Asp Val Ala Gly Ile Ser Ile<br>
225	230	235	240<br>
Cys Gly Ala Leu Lys Asn Val Val Ala Leu Gly Cys Gly Phe Val Glu<br>
245	250	255<br>
Gly Leu Gly Trp Gly Asn Asn Ala Ser Ala Ala Ile Gln Arg Val Gly<br>
260	265	270<br>
Leu Gly Glu Ile Ile Arg Phe Gly Gln Met Phe Phe Pro Glu Ser Arg<br>
275	280	285<br>
Glu Glu Thr Tyr Tyr Gln Glu Ser Ala Gly Val Ala Asp Leu Ile Thr<br>
290	295	300<br>
Thr Cys Ala Gly Gly Arg Asn Val Lys Val Ala Arg Leu Met Ala Thr<br>
305	310	315 - - *	320<br>
Ser Gly Lys Asp Ala Trp Glu Cys Glu Lys Glu Leu Leu Asn Gly Gln<br>
325	330	r	335<br>
Ser Ala Gln Gly Leu Ile Thr Cys Lys Glu Val His Glu Trp Leu Glu<br>
340	345	^ 350<br>
Thr Cys Gly Ser Val Glu Asp Phe Pro Leu Phe Glu Ala Val Tyr Gln<br>
355	360	365<br>
Ile Val Tyr Asn Asn Tyr Pro Met Lys Asn Leu Pro Asp Met Ile Glu<br>
370	375	 380<br>
Glu Leu Asp Leu His Glu Asp	<br>
385	390<br>
(2) INFORMATION FOR SEQ ID NO 12:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH. 384 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) T0POL0GY unknown<br>
(n) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: GPD2<br>
(Xl) SEQUENCE DESCRIPTION- SEQ ID NO 12-<br>
Met Thr Ala His Thr Asn Ile Lys Gln His Lys His Cys His Glu Asp<br>
15	10	15<br>
His Pro He Arg Arg Ser Asp Ser Ala Val Ser He Val His Leu Lys<br>
20	25	30<br>
58<br><br>
Arg Ala Pro Phe Lys Val Thr Val lie Gly Ser Gly Asn Trp Gly Tbr<br>
35	40	45<br>
Thr lie Ala Lys Val lie Ala Glu Asn Thr Glu Leu His Ser His lie<br>
50	55	60<br>
Phe Glu Pro Glu Val Arg Met Trp Val Phe Asp Glu Lys Ile Gly Asp<br>
65	70	75	80<br>
Glu Asn Leu Thr Asp Ile Ile Asn Thr Arg His Gln Asn Val Lys Tyr<br>
85	90	95<br>
Leu Pro Asn Ile Asp Leu Pro His Asn Leu Val Ala Asp Pro Asp Leu<br>
100	105	110<br>
Leu His Ser Ile Lys Gly Ala Asp Ile Leu Val Phe Asn Ile Pro His<br>
115	120	125<br>
Gln Phe Leu Pro Asn Ile Val Lys Gln Leu Gln Gly His Val Ala Pro<br>
130	135	140<br>
His Val Arg Ala Ile Ser Cys Leu Lys Gly Phe Glu Leu Gly Ser Lys<br>
145	150	155	160<br>
Gly Val Gln Leu Leu Ser Ser Tyr Val Thr Asp Glu Leu Gly Ile Gln<br>
165	170	175<br>
Cys Gly Ala Leu Ser Gly Ala Asn Leu Ala Pro Glu Val Ala Lys Glu<br>
180	185	190<br>
His Trp Ser Glu Thr Thr Val Ala Tyr Gln Leu Pro Lys Asp Tyr Gln<br>
195	200	205<br>
Gly Asp Gly Lys Asp Val Asp His Lys Ile Leu Lys Leu Leu Phe His<br>
210	215	220<br>
Arg Pro Tyr Phe His Val Asn Val Ile Asp Asp Val Ala Gly Ile Ser<br>
225	230	235	" 240<br>
Ile Ala Gly Ala Leu Lys Asn Val Val Ala Leu Ala Cys Gly Phe Val<br>
245	250	255<br>
Glu Gly Met Gly Trp Gly Asn Asn Ala Ser Ala Ala Ile Gln Arg Leu<br>
260	265	270<br>
Gly Leu Gly Glu Ile Ile Lys Phe Gly Arg Met Phe Phe Pro Glu Ser<br>
275	280	285<br>
Lys VaJ Glu Thr Tyr Tyr Gln Glu Ser Ala Gly Val Ala Asp Leu Ile<br>
290	295	300<br>
Thr Thr Cys Ser Gly Gly Arg Asn Val Lys Val Ala Thr Tyr Met Ala<br>
305	310	315	320<br>
Lys Thr Gly Lys Ser Ala Leu Glu Ala Glu Lys Glu Leu Leu Asn Gly<br>
325	330	335<br>
Gln Ser Ala Gln Gly Ile Ile Thr Cys Arg Glu Val His Glu Trp Leu<br>
340	345	350<br>
Gln Thr Cys Glu Leu Thr Gln Glu Phe Pro Ile Ile Arg Gly Ser Leu<br>
355	360	365<br>
Pro ASD Ser Leu Gln Gln Arg Pro His Gly Arg Pro Thr Gly Asp Asp<br>
370	375	380<br><br>
2,	INFORMATION FOR SEQ ID NO 13<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 614 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(II) MOLECULE TYPE protein<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM GUT2<br>
(xI) SEQUENCE DESCRIPTION SEQ ID NO 13<br>
Met Thr Arg Ala Thr Trp Cys Asn Ser Pro Pro Pro Leu His Arg Gln<br>
15	10	15<br>
Val Ser Arg Arg Asp Leu Leu Asp Arg Leu Asp Lys Thr His Gln Phe<br>
20	25	30<br>
Asp Val Leu Ile Ile Gly Gly Gly Ala Thr Gly Thr Gly Cys Ala Leu<br>
35	40	45<br>
Asp Ala Ala Thr Arg Gly Leu Asn Val Ala Leu Val Glu Lys Gly Asp<br>
50	55	60<br>
Phe Ala Ser Gly Thr Ser Ser Lys Ser Thr Lys Met Ile His Gly Gly<br>
65	70	75	80<br>
Val Arg Tyr Leu Glu Lys Ala Phe Trp Glu Phe Ser Lys Ala Gln Leu<br>
85	90	95<br>
Asp Leu Val Ile Glu Ala Leu Asn Glu Arg Lys His Leu Ile Asn Thr<br>
100	105	110<br>
Ala Pro His Leu Cys Thr Val Leu Pro Ile Leu Ile Pro Ile Tyr Ser<br>
115	120	125<br>
Thr Trp Gln Val Pro Tyr Ile Tyr Met Gly Cys Lys Phe Tyr Asp Phe<br>
130	135	140<br>
Phe Gly Gly Ser Gln Asn Leu Lys Lys Ser Tyr Leu Leu Ser Lys Ser<br>
145	150	155	160<br>
Ala Thr Val Glu Lys Ala Pro Met Leu Thr Thr Asp Asn Leu Lys Ala<br>
165	170	175<br>
Ser Leu Val Tyr His Asp Gly Ser Phe Asn Asp Ser Arg Leu Asn Ala<br>
180	185	190<br>
Thr Leu Ala Ile Thr Gly Val Glu Asn GLy Ala Thr Val Leu Ile Tyr<br>
195	200	205<br>
Val Glu Val Gln Lys Leu Ile Lys Asp Pro Thr Ser Gly Lys Val Ile<br>
210	215 220<br>
Gly Ala Glu Ala Arg Asp Val Glu Thr Asn Glu Leu Val Arg Ile Asn<br>
225	230	235	240<br>
Ala Lys Cys Val Val Asn Ala Thr Gly Pro Tyr Ser Asp Ala Ile Leu<br>
245	250	255<br>
Gln Met Asp Arg Asn Pro Ser Gly Leu Pro Asp Ser Pro Leu Asn Asp<br>
260	265	270<br><br>
Asn Ser Lys Ile Lys Set Thr Phe Asn Gln Ile Ser Val Met Asp Pro<br>
275	280	285<br>
Lys Met Val Ile Pre Ser Ile Gly Val His Ile Val Leu Pre Ser Pre<br>
290	295	300<br>
Tyr Ser Pro Lys Asp Met Gly Leu Leu ASD Val Arg Thr Ser ASD Gly<br>
305	310	315	 320<br>
Arg Val Met Phe Phe Leu Pro Trp Gln Gly Lys Val Leu Ala Gly Thr<br>
325	330	335<br>
Thr Asp Ile Pro Leu Lys Gln Val Pro Glu Asn Pro Met Pro Thr Glu<br>
340	345	350<br>
Ala Asp Ile Gln Asp Ile Leu Lys Glu Leu Gln His Tyr Ile Glu Phe<br>
355	360	365<br>
Pro Val Lys Arg Glu Asp Val Leu Ser Ala Trp Ala Gly Val Arg Pro<br>
370	375	380<br>
Leu Val Arg Asp Pro Arg Thr Ile Pro Ala Asp Gly Lys Lys Gly Ser<br>
385	390	395	400<br>
Ala Thr Gln Gly Val Val Arg Ser His Phe Leu Phe Thr Ser Asp Asn<br>
405	410	415<br>
Gly Leu Ile Thr Ile Ala Gly Gly Lys Trp Thr Thr Tyr Arg Gln Met<br>
420	425	430<br>
Ala Glu Glu Thr Val Asp Lys Val Val Glu Val Gly Gly Phe His Asn<br>
435	440	445<br>
Leu Lys Pro Cys His Thr Arg Asp Ile Lys Leu Ala Gly Ala Glu Glu<br>
450	455	460<br>
Trp Thr Gln Asn Tyr Val Ala Leu Leu Ala Gln Asn Tyr His Leu Ser<br>
465	470	475	480<br>
Ser Lys Met Ser Asn Tyr Leu Val Gln Asn Tyr Gly Thr Arg Ser Ser<br>
485	490	495<br>
Ile Ile Cys Glu Phe Phe Lys Glu Ser Met Glu Asn Lys Leu Pro Leu<br>
500	505	510<br>
Ser Leu Ala Asp Lys Glu Asn Asn Val Ile Tyr Ser Ser Glu Glu Asn<br>
515	520	525<br>
Asn Leu Val Asn Phe Asp Thr Phe Arg Tyr Pro Phe Thr Ile Gly Glu<br>
530	535	540<br>
Leu Lys T/r Ser Met Gln Tyr Glu Tyr Cys Arg Thr Pro Leu Asp Phe<br>
545	550	555	560<br>
Leu Leu Arg Arg Thr Arg Phe Ala Phe Leu Asp Ala Lys Glu Ala Leu<br>
565	570	575<br>
Asn Ala Val His Ala Thr Val Lys Val Met Gly Asp Glu Phe Asn Trp<br>
580	585	590<br>
Ser Glu Lys Lys Arg Gln Trp Glu Leu Glu Lys Thr Val Asn Phe Ile<br>
595	600	605<br>
Gln Gly Arg Phe Gly Val 610<br><br>
(2)	INFORMATION FOR SEQ ID NO 14<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 339 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(11) MOLECULE TYPE protein<br>
(u) ORIGINAL SOURCE<br>
{A) ORGANISM GPSA<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 14<br>
Met Asn Gln Arg Asn Ala Ser Met Thr Val Ile Gly Ala Gly Ser Tyr<br>
15	10	15<br>
Gly Thr Ala Leu Ala Ile Thr Leu Ala Arg Asn Gly His Glu Val Val<br>
20	25	30<br>
Leu Trp Gly His Asp Pro Glu His Ile Ala Thr Leu Glu Arg Asp Arg<br>
35	40	45<br>
Cys Asn Ala Ala Phe Leu Pro Asp Val Pro Phe Pro Asp Thr Leu His<br>
50	55	60<br>
Leu Glu Ser Asp Leu Ala Thr Ala Leu Ala Ala Ser Arg Asn Ile Leu<br>
65	70	75	80<br>
Val Val Val Pro Ser His Val Phe Gly Glu Val Leu Arg Gln Ile Lys<br>
85	. 90	95<br>
Pro Leu Met Arg Pro Asp Ala Arg Leu Val Trp Ala Thr Lys Gly Leu<br>
100	105	110<br>
Glu Ala Glu Thr Gly Arg Leu Leu Gln Asp Val Ala Arg Glu Ala Leu<br>
115	120	125<br>
Gly Asp Gln Ile Pro Leu Ala Val Ile Ser Gly Pro Thr Phe Ala Lys<br>
130	135	140<br>
Glu Leu Ala Ala Gly Leu Pro Thr Ala Ile Ser Leu Ala Ser Thr Asp<br>
145	150	155	160<br>
Gln Thr Phe Ala Asp Asp Leu Gln Gln Leu Leu His Cys Gly Lys Ser<br>
165	170	175<br>
Phe Arg Val Tyr Ser Asn Pro Asp Phe Ile Gly Val Gln Leu Gly Gly<br>
180	185	190<br>
Ala Val Lys Asn Val Ile Ala Ile Gly Ala Gly Met Ser Asp Cly Ile<br>
195	200	205<br>
Gly Phe GJy Ala Asn Ala Arg Thr Ala Leu Ile Thr Arg Gly Leu Ala<br>
210	215	220<br>
Glu Met Ser Arg Leu Gly Ala Ala Leu Gly Ala Asp Pro Ala Thr Phe<br>
225	230	235	240<br>
Met Gly Met Ala Gly Leu Gly Asp Leu Val Leu Thr Cys Thr Asp Asn<br>
245	250	255<br>
Gln Ser Arg Asn Arg Arg Phe Gly Met Met Leu Gly Gln Gly Met Asp<br>
260	265	270<br><br>
Val Gln Ser Ala Gln Glu Lys lie Gly Gln Val Vai Glu Sly Tyr Arg<br>
275	230	285<br>
Asn Thr Lys Glu Val Arg Glu Leu Ala His Arg Phe Gly Val Glu Men<br>
290	295	300<br>
Pro lie Thr Glu Glu lie Tyr Gln Val Leu Tyr Cys Gly Lys Asn Aia<br>
305	310	315 320<br>
Arg Glu Ala Ala Leu Thr Leu Leu Gly Arg Ala Arg Lys Asp Glu Arg<br>
325	330	335<br>
Ser Ser His<br>
(2) INFORMATION FOR SEQ ID NO 15<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 501 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(ll) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
{A) ORGANISM GLPD<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 15<br>
Met Glu Thr Lys Asp Leu lie Val lie Gly Gly Gly lie Asn Gly Ala<br>
15	10	15<br>
Gly lie Ala Ala Asp Ala Ala Gly Arg Gly Leu Ser Val Leu Met Leu<br>
20	25	30<br>
Glu Ala Gln Asp Leu Ala Cys Ala Thr Ser Ser Ala Ser Ser Lys Leu<br>
35	40	45<br>
lie His Gly Gly Leu Arg Tyr Leu Glu His Tyr Glu Phe Arg Leu Val<br>
50	55	60<br>
Ser Glu Ala Leu Ala Glu Arg Glu Val Leu Leu Lys Met Ala Pro His<br>
65	70	75	80<br>
lie Ala Phe Pro Met Arg Phe Arg Leu Pro His Arg Pro His Leu Arg<br>
85	90	95<br>
Pro Ala Trp Met Ile Arg Ile Gly Leu Phe Met Tyr Asp His Leu Gly<br>
100	105	HO<br>
Lys Arg Thr Ser Leu Pro Gly Ser Thr Gly Leu Arg Phe Gly Ala Asn<br>
115	120	125<br>
Ser Val Leu Lys Pro Glu Ile Lys Arg Gly Phe Glu Tyr Ser Asp Cys<br>
130	135	140<br>
Trp Val Asp Asp Ala Arg Leu Val Leu Ala Asn Ala Gln Met Val Val<br>
145	150	155	160<br>
Arq Lys Gly Gly Glu Val Leu Thr Arg Thr Arg Ala Thr Ser Ala Arg<br>
165	170	175<br>
Arg Glu Asn Gly Leu Trp Ile Val Glu Ala Glu Asp Ile Asp Thr Gly<br>
180	185	190<br><br>
Lys Lys Tyr Ser Trp Gln Ala Arg Gly Leu Val Asr aia Ter Gly Pro<br>
195	200	205<br>
Trp Val Lys Gln Phe Phe Asp Asp Gly Met His Leu Pro Ser Pro Tyr<br>
210	215	220<br>
Gly lie Arg Leu Ile Lys Gly Ser His Ile Val Val Pro Arg Val His<br>
225	230	235	240<br>
Thr Gln Lvs Gln Ala Tyr Ile Leu Gln Asn Glu Asp Lys Arg Ile Val<br>
245	250	255<br>
Phe Val Ile Pro Trp Met Asp Glu Phe Ser Ile Ile Gly Thr Thr Asp<br>
260	265	270<br>
Val Glu Tyr Lys Gly Asp Pro Lys Ala Val Lys Ile Glu Glu Ser Glu<br>
275	280	285<br>
Ile Asn Tyr Leu Leu Asn Val Tyr Asn Thr His Phe Lys Lys Gln Leu<br>
290	295	300<br>
Ser Arg Asp Asp Ile Val Trp Thr Tyr Ser Gly Val Arg Pro Leu Cys<br>
305	310	315	320<br>
Asp Asp Glu Ser Asp Ser Pro Gln Ala Ile Thr Arg Asp Tyr Thr Leu<br>
325	330	335<br>
Asp Ile His Asp Glu Asn Gly Lys Ala Pro Leu Leu Ser Val Phe Gly<br>
340	345	350<br>
Gly Lys Leu Thr Thr Tyr Arg Lys Leu Ala Glu His Ala Leu Glu Lys<br>
355	360	365<br>
Leu Thr Pro Tyr Tyr Gln Gly Ile Gly Pro Ala Trp Thr Lys Glu Ser<br>
370	375	380<br>
Val Leu Pro Gly Gly Ala Ile Glu Gly Asp Arg Asp Asp Tyr Ala Ala<br>
385	390	395	400<br>
Arg Leu Arg Arg Arg Tyr Pro Phe Leu Thr Glu Ser Leu Ala Arg His<br>
405	410	415<br>
Tyr Ala Arg Thr Tyr Gly Ser Asn Ser Glu Leu Leu Leu Gly Asn Ala<br>
420	425	430<br>
Gly Thr Val Ser Asp Leu Gly Glu Asp Phe Gly His Glu Phe Tyr Glu<br>
435	440	445<br>
Ala Glu Leu Lys Tyr Leu Val Asp His Glu Trp Val Arg Arg Ala Asp<br>
450	455	460<br>
Asp Ala Leu Trp Arg Arg Thr Lys Gln Gly Met Trp Leu Asn Ala Asp<br>
465	470	475	480<br>
Gln Gln Ser Arg Val Ser Gln Trp Leu Val Glu Tyr Thr Gln Gln Arg<br>
485	490	495<br>
Leu Ser Leu Ala Ser 500<br>
(2) INFORMATION FOR SEQ ID NO 16.<br>
(L) SEQUENCE CHARACTERISTICS<br>
{A) LENGTH 542 amino acids (B) TYPE amino acid<br>
64<br><br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(II) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GLPABC<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 16<br>
Met Lys Thr Arg Asp Ser Gln Ser Ser Asp Val Ile Ile lie Gly Gly<br>
15	10	15<br>
Gly Ala Thr Gly Ala Gly Ile Ala Arg Asp Cys Ala Leu Arg Gly Leu<br>
20	25	30<br>
Arg Val Ile Leu Val Glu Arg His Asp Ile Ala Thr Gly Ala Thr Gly<br>
35	40	45<br>
Arg Asn His Gly Leu Leu His Ser Gly Ala Arg Tyr Ala Val Thr Asp<br>
50	55	60<br>
Ala Glu Ser Ala Arg Glu Cys Ile Ser Glu Asn Gln Ile Leu Lys Arg<br>
65	70	75	80<br>
Ile Ala Arg His Cys Val Glu Pro Thr Asn Gly Leu Phe Ile Thr Leu<br>
85	90	95<br>
Pro Glu Asp Asp Leu Ser Phe Gln Ala Thr Phe Ile Arg Ala Cys Glu<br>
100	105	110<br>
Glu Ala Gly Ile Ser Ala Glu Ala Ile Asp Pro Gln Gln Ala Arg Ile<br>
115	120	125<br>
Ile Glu Pro Ala Val Asn Pro Ala Leu Ile Gly Ala Val Lys Val Pro<br>
130	135	140<br>
Asp Gly Thr Val Asp Pro Phe Arg Leu Thr Ala Ala Asn Met Leu Asp<br>
145	150	155	160<br>
Ala Lys Glu His Gly Ala Val Ile Leu Thr Ala His Glu Val Thr Gly<br>
165	170	175<br>
Leu Ile Arg Glu Gly Ala Thr Val Cys Gly Val Arg Val Arg Asn His<br>
180	185	190<br>
Leu Thr Gly Glu Thr Gln Ala Leu His Ala Pro Val Val Val Asn Ala<br>
195	200	205<br>
Ala Gly Ile Trp Gly Gln His Ile Ala Glu Tyr Ala Asp Leu Arg Ile<br>
210	215	220<br>
Arg Met Phe Pro Ala Lys Gly Ser Leu Leu lie Met Asp His Arg Ile<br>
225	230	235	240<br>
Asn Gln His Val Ile Asn Arg Cys Arg Lys Pro Ser Asp Ala Asp Ile<br>
245	250	255<br>
Leu Val Pro Gly Asp Thr Ile Ser Leu Ile Gly Thr Thr Ser Leu Arg<br>
260	265	270<br>
Ile Asp Tyr Asn Glu Ile Asp Asp Asn Arg Val Thr Ala Glu Glu Val<br>
275	280	285<br>
Asp Ile Leu Leu Arg Glu Gly Glu Lys Leu Ala Pro Val Met Ala Lys<br>
290	295	300<br><br>
Thr Arg Ile Leu Arq Ala Tyr Ser Gly Val Arg Pro Leu Val Ala Ser<br>
305	310	315	320<br>
ASD Asp Asp Pro Ser Gly Arg Asn Leu Ser Arg Gly Ile Val Leu Leu<br>
325	330	335<br>
Asp His Ala Glu Arg Asp Gly Leu Asp Gly Pne Ile Thr Ile Thr Gly<br>
340	345	350<br>
Gly Lys Leu Met Thr Tyr Arg Leu Met Ala Glu Trp Ala Thr Asp Ala<br>
355	360	365<br>
Val Cys Arg Lys Leu Gly Asn Thr Arg Pro Cys Thr Thr Ala Asp Leu<br>
370	375	380<br>
Ala Leu Pro Gly Ser Gln Glu Pro Ala Glu Val Thr Leu Arg Lys Val<br>
385	390	395	400<br>
Ile Ser Leu Pro Ala Pro Leu Arg Gly Ser Ala Val Tyr Arg His Gly<br>
405	410	415<br>
Asp Arg Thr Pro Ala Trp Leu Ser Glu Gly Arg Leu His Arg Ser Leu<br>
420	425	430<br>
Val Cys Glu Cys Glu Ala Val Thr Ala Gly Glu Val Gln Tyr Ala Val<br>
435	440	445<br>
Glu Asn Leu Asn Val Asn Ser Leu Leu Asp Leu Arg Arg Arg Thr Arg<br>
450	455	460<br>
Val Gly Met Gly Thr Cys Gln Gly Glu Leu Cys Ala Cys Arg Ala Ala<br>
465	470	475	480<br>
Gly Leu Leu Gln Arg Phe Asn Val Thr Thr Ser Ala Gln Ser Ile Glu<br>
485	490	495<br>
Gln Leu Ser Thr Phe Leu Asn Glu Arg Trp Lys Gly Val Gln Pro Ile<br>
500	505	510<br>
Ala Trp Gly Asp Ala Leu Arg Glu Ser Glu Phe Thr Arg Trp Val Tyr<br>
515	520	525<br>
Gln Gly Leu Cys Gly Leu Glu Lys Glu Gln Lys Asp Ala Leu<br>
530	535	540<br>
(2} INFORMATION FOR SEQ ID NO 17-<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 250 amino acids<br>
(B) TYPE: amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
{li) MOLECULE TYPE- protein<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM GPP2<br>
{xi) SEQUENCE DESCRIPTION SEQ ID NO 17<br>
Met Gly Leu Thr Thr Lys Pro Leu Ser Leu Lys Val Asn Ala Ala Leu<br>
15	10	15<br>
Phe Asp Val Asp Gly Thr Ile Ile Ile Ser Gln Pro Ala Ile Ala Ala<br>
20	25	30<br><br>
Phe Trp Arg Asp Phe Gly Lys Asp Lys Pro Tyr Phe Asp Ala Glu Pis<br>
35	40	45<br>
Val lie Gln Val Ser His Gly Trp Arg Thr Phe Asp Ala Ile Ala Lys<br>
50	55	60<br>
Phe Ala Pre Asp Phe Ala Asn GLY Glu Tyr Val Asn Lys Leu Glu Ala<br>
65	70	75	80<br>
Glu Ile Pro Val Lys Tyr Gly Glu Lys Ser Ile Glu Val Pro Gly Ala<br>
85	90	95<br>
Val Lys Leu Cys Asn Ala Leu Asn Ala Leu Pro Lys Glu Lys Trp Ala<br>
100	105	110<br>
Val Ala Thr Ser Gly Thr Arg Asp Met Ala Gln Lys Trp Phe Glu His<br>
115	120	125<br>
Leu Gly Ile Arg Arg Pro Lys Tyr Phe Ile Thr Ala Asn Asp Val Lys<br>
130	135	140<br>
Gln Gly Lys Pro His Pro Glu Pro Tyr Leu Lys Gly Arg Asn Gly Leu<br>
145	150	155	160<br>
Gly Tyr Pro Ile Asn Glu Gln Asp Pro Ser Lys Ser Lys Val Val Val<br>
165	170	175<br>
Phe Glu Asp Ala Pro Ala Gly Ile Ala Ala Gly Lys Ala Ala Gly Cys<br>
180	185	190<br>
Lys lie Ile Gly Ile Ala Thr Thr Phe Asp Leu Asp Phe Leu Lys Glu<br>
195	200	205<br>
Lys Gly Cys Asp Ile Ile Val Lys Asn His Glu Ser Ile Arg Val Gly<br>
210	215	220<br>
Gly Tyr Asn Ala Glu Thr Asp Glu Val Glu Phe Ile Phe Asp Asp Tyr<br>
225	230	235	240<br>
Leu Tyr Ala Lys Asp Asp Leu Leu Lys Trp<br>
245	250<br>
(2) INFORMATION FOR SEQ ID NO.18"<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH- 709 amino acids<br>
(B) TYPE. amino acid<br>
(C) STRANDEDNESS: unknown<br>
(D) TOPOLOGY unknown<br>
(ii) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GUT1<br>
(xi) SEQUENCE DESCRIPTION- SEQ ID NO.18<br>
Met Phe Pro Ser Leu Phe Arg Leu Val Val Phe Ser Lys Arg Tyr Ile<br>
1	5	10	15<br>
Phe Arg Ser Ser Gln Arg Leu Tyr Thr Ser Leu Lys Gln Glu Gln Ser<br>
20	25	30<br>
Arg Met Ser Lys Ile Met Glu Asp Leu Arg Ser Asp Tyr Val Pro Leu<br>
35	40	45<br><br>
Ile Ala Ser lie Asp Val Gly Thr Thr Ser Ser Arg Cys lie Leu Phe<br>
50	55	60<br>
Asn Arg Trp Gly Gln Asp Val Ser Lys His Gln lie GIJ Tyr Ser Thr<br>
65	70	75	80<br>
Ser Ala Ser Lys Gly Lys lie Gly Val Ser Gly Leu Arg Arg ?ro Ser<br>
85	90	95<br>
Thr Ala Pro Ala Arg Glu Thr Pro Asn Ala Gly Asp Ile Lys Thr Ser<br>
100	105	110<br>
Gly Lys Pro Ile Phe Ser Ala Glu Gly Tyr Ala Ile Gln Glu Thr Lys<br>
115	120	125<br>
Phe Leu Lys Ile Glu Glu Leu Asp Leu Asp Phe His Asn Glu Pro Thr<br>
130	135	140<br>
Leu Lys Phe Pro Lys Pro Gly Trp Val Glu Cys His Pro Gln Lys Leu<br>
145	150	155	160<br>
Leu Val Asn Val Val Gln Cys Leu Ala Ser Ser Leu Leu Ser Leu Gln<br>
165	170	175<br>
Thr Ile Asn Ser Glu Arg Val Ala Asn Gly Leu Pro Pro Tyr Lys Val<br>
180	185	190<br>
Ile Cys Met Gly Ile Ala Asn Met Arg Glu Thr Thr Ile Leu Trp Ser<br>
195	200	205<br>
Arg Arg Thr Gly Lys Pro Ile Val Asn Tyr Gly Ile Val Trp Asn Asp<br>
210	215	220<br>
Thr Arg Thr Ile Lys Ile Val Arg Asp Lys Trp Gln Asn Thr Ser Val<br>
225	230	235	240<br>
Asp Arg Gln Leu Gln Leu Arg Gln Lys Thr Gly Leu Pro Leu Leu Ser<br>
245	250	255<br>
Thr Tyr Phe Ser Cys Ser Lys Leu Arg Trp Phe Leu Asp Asn Glu Pro<br>
260	265	270<br>
Leu Cys Thr Lys Ala Tyr Glu Glu Asn Asp Leu Met Phe Gly Thr Val<br>
275	280	285<br>
Asp Thr Trp Leu Ile Tyr Gln Leu Thr Lys Gln Lys Ala Phe Val Ser<br>
290	295	300<br>
Asp Val Thr Asn Ala Ser Arg Thr Gly Phe Met Asn Leu Ser Thr Leu<br>
305	310	315	320<br>
Lys Tyr Asp Asn Glu Leu Leu Glu Phe Trp Gly Ile Asp Lys Asn Leu<br>
325	330	335<br>
Ile Kis Met Pro Glu Ile Val Ser Ser Ser Gln Tyr Tyr Gly Asp Phe<br>
340	345	350<br>
Gly Ile Pro Asp Trp Ile Met Glu Lys Leu His Asp Ser Pro Lys Thr<br>
355	360	365<br>
Val Leu Arg Asp Leu Val Lys Arg Asn Leu Pro Ile Gln Gly Cys Leu<br>
370	375	380<br>
Gly Asp Gln Ser Ala Ser Met Val Gly Gln Leu Ala Tyr Lys Pro Gly<br>
385	390	395	400<br>
68<br><br>
Ala Ala Lys Cys Thr Tyr Gly Thr Gly Cys Phe Leu Leu Tyr Asn Thr<br>
405	 410	415<br>
Gly Thr Lys Lys Leu Ile Ser Gln His Gly Ala Leu Thr Thr Leu Ala<br>
420	425 	430<br>
Phe Trp Phe Pro His Leu Gln Glu Tyr Gly Gly Gln Lys Pro Glu Leu<br>
435	440	445<br>
Ser Lys Pro His Phe Ala Leu Glu Gly Ser Val Ala Val Ala Gly Ala<br>
450	455	460<br>
Val Val Gln Trp Leu Arg Asp Asn Leu Arg Leu Ile Asp Lys Ser Glu<br>
465	470	475	480<br>
Asp Val Gly Pro Ile Ala Ser Thr Val Pro Asp Ser Gly Gly Val Val<br>
485	490	495<br>
Phe Val Pro Ala Phe Ser Gly Leu Phe Ala Pro Tyr Trp Asp Pro Asp<br>
500	505	510<br>
Ala Arg Ala Thr Ile Met Gly Met Ser Gln Phe Thr Thr Ala Ser His<br>
515	520	525<br>
Ile Ala Arg Ala Ala Val Glu Gly Val Cys Phe Gln Ala Arg Ala Ile<br>
530	535	540<br>
Leu Lys Ala Met Ser Ser Asp Ala Phe Gly Glu Gly Ser Lys Asp Arg<br>
545	550	555	560<br>
Asp Phe Leu Glu Glu Ile Ser Asp Val Thr Tyr Glu Lys Ser Pro Leu<br>
565	570	575<br>
Ser Val Leu Ala Val Asp Gly Gly Met Ser Arg Ser Asn Glu Val Met<br>
580	585	590<br>
Gln Ile Gln Ala Asp Ile Leu Gly Pro Cys Val Lys Val Arg Arg Ser<br>
595	600	605<br>
Pro Thr Ala Glu Cys Thr Ala Leu Gly Ala Ala Ile Ala Ala Asn Met<br>
610	615	620<br>
Ala Phe Lys Asp Val Asn Glu Arg Pro Leu Trp Lys Asp Leu His Asp<br>
625	630	635	640<br>
Val Lys Lys Trp Val Phe Tyr Asn Gly Met Glu Lys Asn Glu Gln Ile<br>
645	650	655<br>
Ser Pro Glu Ala His Pro Asn Leu Lys Ile Phe Arg Ser Glu Ser Asp<br>
660	665	670<br>
Asp Ala Glu Arg Arg Lys His Trp Lys Tyr Trp Glu Val Ala Val Glu<br>
675	680	685<br>
Arg Ser Lys Gly Trp Leu Lys Asp Ile Glu Gly Glu His Glu Gln Val<br>
690	695	700<br>
Leu Glu Asn Phe Gln 705<br>
(2) INFORMATION FOR SEQ ID NO 19<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 12145 base pairs<br>
(B) TYPE nucleic acid<br>
69<br><br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(11) MOLECULE T/PE DNA (genomic)<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM PHK28-2 6<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 19<br>
GTCGACCACC ACGGTGGTGA CTTTAATGCC GCTCTCATGC AGCAGCTCGG TGGCGGTCTC	60<br>
AAAATTCAGG ATGTCGCCGG TATAGTTTTT GATAATCAGC AAGACGCCTT CGCCGCCGTC	120<br>
AATTTGCATC GCGCATTCAA ACATTTTGTC CGGCGTCGGC GAGGTGAATA TTTCCCCCGG	180<br>
ACAGGCGCCG GAGAGCATGC CCTGGCCGAT ATAGCCGCAG TGCATCGGTT CATGTCCGCT	24 0<br>
GCCGCCGCCG GAGAGCAGGG CCACCTTGCC AGCCACCGGC GCGTCGGTGC GGGTCACATA	300<br>
CAGCGGGTCC TGATGCAGGG TCAGCTGCGG ATGGGCTTTA GCCAGCCCCT GTAATTGTTC	360<br>
ATTCAGTACA TCTTCAACAC GGTTAATCAG CTTTTTCATT ATTCAGTGCT CCGTTGGAGA	420<br>
AGGTTCGATG CCGCCTCTCT GCTGGCGGAG GCGGTCATCG CGTAGGGGTA TCGTCTGACG	480<br>
GTGGAGCGTG CCTGGCGATA TGATGATTCT GGCTGAGCGG ACGAAAAAAA GAATGCCCCG	54 0<br>
ACGATCGGGT TTCATTACGA AACATTGCTT CCTGATTTTG TTTCTTTATG GAACGTTTTT	600<br>
GCTGAGGATA TGGTGAAAAT GCGAGCTGGC GCGCTTTTTT TCTTCTGCCA TAAGCGGCGG	660<br>
TCAGGATAGC CGGCGAAGCG GGTGGGAAAA AATTTTTTGC TGATTTTCTG CCGACTGCGG	720<br>
GAGAAAAGGC GGTCAAACAC GGAGGATTGT AAGGGCATTA TGCGGCAAAG GAGCGGATCG	7 80<br>
GGATCGCAAT CCTGACAGAG ACTAGGGTTT TTTGTTCCAA TATGGAACGT AAAAAATTAA	840<br>
CCTGTGTTTC ATATCAGAAC AAAAAGGCGA AAGATTTTTT TGTTCCCTGC CGGCCCTACA	900<br>
GTGATCGCAC TGCTCCGGTA CGCTCCGTTC AGGCCGCGCT TCACTGGCCG GCGCGGATAA	960<br>
CGCCAGGGCT CATCATGTCT ACATGCGCAC TTATTTGAGG GTGAAAGGAA TGCTAAAAGT	1020<br>
TATTCAATCT CCAGCCAAAT ATCTTCAGGG TCCTGATGCT GCTGTTCTGT TCGGTCAATA	1080<br>
TGCCAAAAAC CTGGCGGAGA GCTTCTTCGT CATCGCTGAC GATTTCGTAA TGAAGCTGGC	114 0<br>
GGGAGAGAAA GTGGTGAATG GCCTGCAGAG CCACGATATT CGCTGCCATG CGGAACGGTT	1200<br>
TAACGGCGAA TGCAGCCATG CGGAAATCAA CCGTCTGATG GCGATTTTGC AAAAACAGGG	12 60<br>
CTGCCGCGGC GTGGTCGGGA TCGGCGGTGG TAAAACCCTC GATACCGCGA AGGCGATCGG	1320<br>
TTACTACCAG AAGCTGCCGG TGGTGGTGAT CCCGACCATC GCCTCGACCG ATGCGCCAAC	1380<br>
CAGCGCGCTG TCGGTGATCT ACACCGAAGC GGGCGAGTTT GAAGAGTATC TGATCTATCC	1440<br>
GAAAAACCCG GATATGGTGG TGATGGACAC GGCGATTATC GCCAAAGCGC CGGTACGCCT	1500<br>
GCTGGTCTCC GGCATGGGCG ATGCGCTCTC CACCTGGTTC GAGGCCAAAG CTTGCTACGA	1560<br>
TGCGCGCGCC ACCAGCATGG CCGGAGGACA GTCCACCGAG GCGGCGCTGA GCCTCGCCCG	1620<br>
CCTGTGCTAT GATACGCTGC TGGCGGAGGG CGAAAAGGCC CGTCTGGCGG CGCAGGCCGG	1680<br><br>
GGTAGTGACC GAAGCGCTGG AGCGCATCAT CGAGGCGAAC ACTTACCTCA CCGGCATTGC CTTTGAAAGC AGTGGCCTGG CCGCTGCCCA TGCAATCCAC AACGGTTTCA CCATTCTTGA 900 AGAGTGCCAT CACCTGTATC ACGGTGAGAA AGTGGCCTTC GGTACCCTGG CGCAGCTGGT 1860 GCTGC^GAAC AGCCCGATGG ACGAGATTGA AACGGTGCAG GGCTTCTGCC AGCGCGTGGG 1920 CCTGCCGGTG ACGCTCGCGC AGATGGGCGT CAAAGAGGGG ATCGACGAGA AAATCGCCGC 1980 GGTGGCGAAA GCTACCTGCG CGGAAGGGGA AACCATCCAT AATATGCCGT TTGCGGTGAC 2040 CCCGGAGAGC GTCCATGCCG CTATCCTCAC CGCCGATCTG TTAGGCCAGC AGTGGCTGGC 2100 GCGTTAATTC GCGGTGGCTA AACCGCTGGC CCAGGTCAGC GGTTTTTCTT TCTCCCCTCC 2160<br>
GGCAGTCGCT GCCGGAGGGG TTCTCTATGG TACAACGCGG AAAAGGATAT GACTGTTCAG	2220<br>
ACTCAGGATA CCGGGAAGGC GGTCTCTTCC GTCATTGCCC AGTCATGGCA CCGCTGCAGC	2280<br>
AAGTTTATGC AGCGCGAAAC CTGGCAAACG CCGCACCAGG CCCAGGGCCT GACCTTCGAC	2340<br>
TCCATCTGTC GGCGTAAAAC CGCGCTGCTC ACCATCGGCC AGGCGGCGCT GGAAGACGCC	2400<br>
TGGGAGTTTA TGGACGGCCG CCCCTGCGCG CTGTTTATTC TTGATGAGTC CGCCTGCATC	24 60<br>
CTGAGCCGTT GCGGCGAGCC GCAAACCCTG GCCCAGCTGG CTGCCCTGGG ATTTCGCGAC	2520<br>
GGCAGCTATT GTGCGGAGAG CATTATCGGC ACCTGCGCGC TGTCGCTGGC CGCGATGCAG	2580<br>
GGCCAGCCGA TCAACACCGC CGGCGATCGG CATTTTAAGC AGGCGCTACA GCCATGGAGT	264 0<br>
TTTTGCTCGA CGCCGGTGTT TGATAACCAC GGGCGGCTGT TCGGCTCTAT CTCGCTTTGC	2700<br>
TGTCTGGTCG AGCACCAGTC CAGCGCCGAC CTCTCCCTGA CGCTGGCCAT CGCCCGCGAG	2760<br>
GTGGGTAACT CCCTGCTTAC CGACAGCCTG CTGGCGGAAT CCAACCGTCA CCTCAATCAG	2820<br>
ATGTACGGCC TGCTGGAGAG CATGGACGAT GGGGTGATGG CGTGGAACGA ACAGGGCGTG	2880<br>
CTGCAGTTTC TCAATGTTCA GGCGGCGAGA CTGCTGCATC TTGATGCTCA GGCCAGCCAG	2940<br>
GGGAAAAATA TCGCCGATCT GGTGACCCTC CCGGCGCTGC TGCGCCGCGC CATCAAACAC	3000<br>
GCCCGCGGCC TGAATCACGT CGAAGTCACC TTTGAAAGTC AGCATCAGTT TGTCGATGCG	3060<br>
GTGATCACCT TAAAACCGAT TGTCGAGGCG CAAGGCAACA GTTTTATTCT GCTGCTGCAT	3120<br>
CCGGTGGAGC AGATGCGGCA GCTGATGACC AGCCAGCTCG GTAAAGTCAG CCACACCTTT	3180<br>
GAGCAGATGT CTGCCGACGA TCCGGAAACC CGACGCCTGA TCCACTTTGG CCGCCAGGCG	324 0<br>
GCGCGCGGCG GCTTCCCGGT GCTACTGTGC GGCGAAGAGG GGGTCGGGAA AGAGCTGCTG	3300<br>
AGCCAGGCTA TTCACAATGA AAGCGAACGG GCGGGCGGCC CCTACATCTC CGTCAACTGC	3360<br>
CAGCTATATG CCGACAGCGT GCTGGGCCAG GACTTTATGG GCAGCGCCCC TACCGACGAT	34 20<br>
GAAAATGGTC GCCTGAGCCG CCTTGAGCTG GCCAACGGCG GCACCCTGTT TCTGGAAAAG	34 80<br>
ATCGAGTATC TGGCGCCGGA GCTGCAGTCG GCTCTGCTGC AGGTGATTAA GCAGGGCGTG	354 0<br>
CTCACCCGCC TCGACGCCCG GCGCCTGATC CCGGTGGATG TGAAGGTGAT TGCCACCACC	3600<br>
ACCGTCGATC TGGCCAATCT GGTGGAACAG AACCGCTTTA GCCGCCAGCT GTACTATGCG	3660<br><br>
CTGCACTCCT TTGAGATCGT CATCCCGCCG CTGCGCGCCC GACGCAACAG TATTCCGTCG 3720 CTGGTGCATA ACCGGTTGAA GAGCCTGGAG AAGCGTTTCT CTTCGCGACT GAAAGTGGAC 3780 GATGACGCGC TGGCACAGCT GGTGGCCTAC TCGTGGCCGG GGAATGATTT TGAGCTCAAC 3840 AGCGTCATTG AGAATATCGC CATCAGCAGC GACAACGGCC ACATTCGCCT GAGTAATCTG 3900 CCGGAATATC TCTTTTCCGA GCGGCCGGGC GGGGATAGCG CGTCATCGCT GCTGCCGGCC 3960<br>
AGCCTGACTT TTAGCGCCAT CGAAAAGGAA GCTATTATTC ACGCCGCCCG GGTGACCAGC	4 020<br>
GGGCGGGTGC AGGAGATGTC GCAGCTGCTC AATATCGGCC GCACCACCCT GTGGCGCAAA	4 080<br>
ATGAAGCAGT ACGATATTGA CGCCAGCCAG TTCAAGCGCA AGCATCAGGC CTAGTCTCTT	414 0<br>
CGATTCGCGC CATGGAGAAC AGGGCATCCG ACAGGCGATT GCTGTAGCGT TTGAGCGCGT	4200<br>
CGCGCAGCGG ATGCGCGCGG TCCATGGCCG TCAGCAGGCG TTCGAGCCGA CGGGACTGGG	4 2 60<br>
TGCGCGCCAC GTGCAGCTGG GCAGAGGCGA GATTCCTCCC CGGGATCACG AACTGTTTTA	4 320<br>
ACGGGCCGCT CTCGGCCATA TTGCGGTCGA TAAGCCGCTC CAGGGCGGTG ATCTCCTCTT	4 380<br>
CGCCGATCGT CTGGCTCAGG CGGGTCAGGC CCCGCGCATC GCTGGCCAGT TCAGCCCCCA	4440<br>
GCACGAACAG CGTCTGCTGA ATATGGTGCA GGCTTTCCCG CAGCCCGGCG TCGCGGGTCG	4 500<br>
TGGCGTAGCA GACGCCCAGC TGGGATATCA GTTCATCGAC GGTGCCGTAG GCCTCGACGC	4 560<br>
GAATATGGTC TTTCTCGATG CGGCTGCCGC CGTACAGGGC GGTGGTGCCT TTATCCCCGG	4 620<br>
TGCGGGTATA GATACGATAC ATTCAGTTTC TCTCACTTAA CGGCAGGACT TTAACCAGCT	4 680<br>
GCCCGGCGTT GGCGCCGAGC GTACGCAGTT GATCGTCGCT ATCGGTGACG TGTCCGGTAG	4 740<br>
CCAGCGGCGC GTCCGCCGGC AGCTGGGCAT GAGTGAGGGC TATCTCGCCG GACGCGCTGA	4800<br>
GCCCGATACC CACCCGCAGG GGCGAGCTTC TGGCCGCCAG GGCGCCCAGC GCAGCGGCGT	4860<br>
CACCGCCTCC GTCATAGGTT ATGGTCTGGC AGGGGACCCC CTGCTCCTCC AGCCCCCAGC	4 920<br>
ACAGCTCATT GATGGCGCCG GCATGGTGCC CGCGCGGATC GTAAAACAGG CGTACGCCTG	4 980<br>
GCGGTGAAAG CGACATGACG GTCCCCTCGT TAACACTCAG AATGCCTGGC GGAAAATCGC	504 0<br>
GGCAATCTCC TGCTCGTTGC CTTTACGCGG GTTCGAGAAC GCATTGCCGT CTTTTAGAGC	5100<br>
CATCTCCGCC ATGTAGGGGA AGTCGGCCTC TTTTACCCCC AGATCGCGCA GATGCTGCGG	5160<br>
AATACCGATA TCCATCGACA GACGCGTGAT AGCGGCGATG GCTTTTTCCG CCGCGTCGAG	5220<br>
AGTGGACAGT CCGGTGATAT TTTCGCCCAT CAGTTCAGCG ATATCGGCGA ATTTCTCCGG	528 0<br>
GTTGGCGATC AGGTTGTAGC GCGCCACATG CGGCAGCAGG ACAGCGTTGG CCACGCCGTG	534 0<br>
CGGCATGTCG TACAGGCCGC CCAGCTGGTG CGCCATGGCG TGCACGTAGC CGAGGTTGGC	54 00<br>
GTTATTGAAA GCCATCCCGG CCAGCAGAGA AGCATAGGCC ATGTTTTCCC GCGCCTGCAG	54 60<br>
ATTGCTGCCG AGGGCCACGG CCTGGCGCAG GTTGCGGGCG ATGAGGCGGA TCGCCTGCAT	5520<br>
GGCGGCGGCG TCCGTCACCG GGTTAGCGTC TTTGGAGATA TAGGCCTCTA CGGCGTGGGT	5580<br>
CAGGGCATCC ATCCCGGTCG CCGCGGTCAG GGCGGCCGGT TTACCGATCA TCAGCAGTGG	5640<br><br>
ATCGTTGATA GAGACCGACG GCAGTTTGCG CCAGCTGACG ATCACAAACT TCACTTTGGT	5700<br>
TTCGGTGTTG GTCAGGACGC AGTGGCGGGT GACCTCGCTG GCGGTGCCGG CGGTGGTATT	5760<br>
GACCGCGACG ATAGGCGGCA GCGGGTTGGT CAGGGTCTCG ATTCCGGCAT ACTGGTACAG	5320<br>
ATCGCCCTCA TGGGTGGCGG CGATGCCGAT GCCTTTGCCG CAATCGTGCG GGCTGCCGCC	5880<br>
GCCCACGGTG ACGATGATGT CGCACTGTTC GCGGCGAAAC ACGGCGAGGC CGTCGCGCAC	5940<br>
GTTGGTGTCT TTCGGGTTCG GCTCGACGCC GTCAAAGATC GCCACCTCGA TCCCGGCCTC	6000<br>
CCGCAGATAA TGCAGGGTTT TGTCCACCGC GCCATCTTTA ATTGCCCGCA GGCCTTTGTC	6060<br>
GGTGACCAGC AGGGCTTTTT TCCCCCCCAG CAGCTGGCAG CGTTCGCCGA CTACGGAAAT	6120<br>
GGCGTTGGGG CCAAAAAAGT TAACGTTTGG CACCAGATAA TCAAACATAC GATAGCTCAT	6180<br>
AATATACCTT CTCGCTTCAG GTTATAATGC GGAAAAACAA TCCAGGGCGC ACTGGGCTAA	624 0<br>
TAATTGATCC TGCTCGACCG TACCGCCGCT AACGCCGACG GCGCCAATTA CCTGCTCATT	6300<br>
AAAAATAACT GGCAGGCCGC CGCCAAAAAT AATAATTCGC TGTTGGTTGG TTAGCTGCAG	6360<br>
ACCGTACAGA GATTGTCCTG GCTGGACCGC TGACGTAATT TCATGGGTAC CTTGCTTCAG	6420<br>
GCTGCAGGCG CTCCAGGCTT TATTCAGGGA AATATCGCAG CTGGAGACGA AGGCCTCGTC	64 80<br>
CATCCGCTGG ATAAGCAGCG TGTTGCCTCC GCGGTCAACT ACGGAAAACA CCACCGCCAC	6540<br>
GTTGATCTCA GTGGCTTTTT TTTCCACCGC CGCCGCCATT TGCTGGGCGG CGGCCAGGGT	6600<br>
GATTGTCTGA ACTTGTTGGC TCTTGTTCAT CATTCTCTCC CGCACCAGGA TAACGCTGGC	6660<br>
GCGAATAGTC AGTAGGGGGC GATAGTAAAA AACTATTACC ATTCGGTTGG CTTGCTTTAT	67 20<br>
TTTTGTCAGC GTTATTTTGT CGCCCGCCAT GATTTAGTCA ATAGGGTTAA AATAGCGTCG	6780<br>
GAAAAACGTA ATTAAGGGCG TTTTTTATTA ATTGATTTAT ATCATTGCGG GCGATCACAT	6840<br>
TTTTTATTTT TGCCGCCGGA GTAAAGTTTC ATAGTGAAAC TGTCGGTAGA TTTCGTGTGC	6900<br>
CAAATTGAAA CGAAATTAAA TTTATTTTTT TCACCACTGG CTCATTTAAA GTTCCGCTAT	6960<br>
TGCCGGTAAT GGCCGGGCGG CAACGACGCT GGCCCGGCGT ATTCGCTACC GTCTGCGGAT	7020<br>
TTCACCTTTT GAGCCGATGA ACAATGAAAA GATCAAAACG ATTTGCAGTA CTGGCCCAGC	7 080<br>
GCCCCGTCAA TCAGGACGGG CTGATTGGCG AGTGGCCTGA AGAGGGGCTG ATCGCCATGG	714 0<br>
ACAGCCCCTT TGACCCGGTC TCTTCAGTAA AAGTGGACAA CGGTCTGATC GTCGAACTGG	7200<br>
ACGGCAAACG CCGGGACCAG TTTGACATGA TCGACCGATT TATCGCCGAT TACGCGATCA	7 260<br>
ACGTTGAGCG CACAGAGCAG GCAATGCGCC TGGAGGCGGT GGAAATAGCC CGTATGCTGG	7320<br>
TGGATATTCA CGTCAGCCGG GAGGAGATCA TTGCCATCAC TACCGCCATC ACGCCGGCCA	7380<br>
AAGCGGTCGA GGTGATGGCG CAGATGAACG TGGTGGAGAT GATGATGGCG CTGCAGAAGA	74 4 0<br>
TGCGTGCCCG CCGGACCCCC TCCAACCAGT GCCACGTCAC CAATCTCAAA GATAATCCGG	7500<br>
TGCAGATTGC CGCTGACGCC GCCGAGGCCG GGATCCGCGG CTTCTCAGAA CAGGAGACCA	7560<br>
CGGTCGGTAT CGCGCGCTAC GCGCCGTTTA ACGCCCTGGC GCTGTTGGTC GGTTCGCAGT	7 620<br><br>
GCGGCCGCCC CGGCGTGTTG ACGCAGTGCT CGGTGGAAGA GGCCACCGAG CTGGAGCTGG	7680<br>
GCATGCGTGG CTTAACCAGC TACGCCGAGA CGGTGTCGGT CTACGGCACC GAAGCGGTAT	7740<br>
TTACCGACGG CGATGATACG CCGTGGTCAA AGGCGTTCCT CGCCTCGGCC TACGCCTCCC	7800<br>
GCGGGTTGAA AATGCGCTAC ACCTCCGGCA CCGGATCCGA AGCGCTGATG GGCTATTCGG	7860<br>
AGAGCAAGTC GATGCTCTAC CTCGAATCGC GCTGCATCTT CATTACTAAA GGCGCCGGGG	7920<br>
TTCAGGGACT GCAAAACGGC GCGGTGAGCT GTATCGGCAT GACCGGCGCT GTGCCGTCGG	7980<br>
GCATTCGGGC GGTGCTGGCG GAAAACCTGA TCGCCTCTAT GCTCGACCTC GAAGTGGCGT	8040<br>
CCGCCAACGA CCAGACTTTC TCCCACTCGG ATATTCGCCG CACCGCGCGC ACCCTGATGC	8100<br>
AGATGCTGCC GGGCACCGAC TTTATTTTCT CCGGCTACAG CGCGGTGCCG AACTACGACA	8160<br>
ACATGTTCGC CGGCTCGAAC TTCGATGCGG AAGATTTTGA TGATTACAAC ATCCTGCAGC	8220<br>
GTGACCTGAT GGTTGACGGC GGCCTGCGTC CGGTGACCGA GGCGGAAACC ATTGCCATTC	8280<br>
GCCAGAAAGC GGCGCGGGCG ATCCAGGCGG TTTTCCGCGA GCTGGGGCTG CCGCCAATCG	834 0<br>
CCGACGAGGA GGTGGAGGCC GCCACCTACG CGCACGGCAG CAACGAGATG CCGCCGCGTA	84 00<br>
ACGTGGTGGA GGATCTGAGT GCGGTGGAAG AGATGATGAA GCGCAACATC ACCGGCCTCG	84 60<br>
ATATTGTCGG CGCGCTGAGC CGCAGCGGCT TTGAGGATAT CGCCAGCAAT ATTCTCAATA	8520<br>
TGCTGCGCCA GCGGGTCACC GGCGATTACC TGCAGACCTC GGCCATTCTC GATCGGCAGT	8580<br>
TCGAGGTGGT GAGTGCGGTC AACGACATCA ATGACTATCA GGGGCCGGGC ACCGGCTATC	8 64 0<br>
GCATCTCTGC CGAACGCTGG GCGGAGATCA AAAATATTCC GGGCGTGGTT CAGCCCGACA	8700<br>
CCATTGAATA AGGCGGTATT CCTGTGCAAC AGACAACCCA AATTCAGCCC TCTTTTACCC	8760<br>
TGAAAACCCG CGAGGGCGGG GTAGCTTCTG CCGATGAACG CGCCGATGAA GTGGTGATCG	8820<br>
GCGTCGGCCC TGCCTTCGAT AAACACCAGC ATCACACTCT GATCGATATG CCCCATGGCG	8880<br>
CGATCCTCAA AGAGCTGATT GCCGGGGTGG AAGAAGAGGG GCTTCACGCC CGGGTGGTGC	894 0<br>
GCATTCTGCG CACGTCCGAC GTCTCCTTTA TGGCCTGGGA TGCGGCCAAC CTGAGCGGCT	9000<br>
CGGGGATCGG CATCGGTATC CAGTCGAAGG GGACCACGGT CATCCATCAG CGCGATCTGC	9060<br>
TGCCGCTCAG CAACCTGGAG CTGTTCTCCC AGGCGCCGCT GCTGACGCTG GAGACCTACC	9120<br>
GGCAGATTGG CAAAAACGCT GCGCGCTATG CGCGCAAAGA GTCACCTTCG CCGGTGCCGG	9180<br>
TGGTGAACGA TCAGATGGTG CGGCCGAAAT TTATGGCCAA AGCCGCGCTA TTTCATATCA	92 4 0<br>
AAGAGACCAA ACATGTGGTG CAGGACGCCG AGCCCGTCAC CCTGCACATC GACTTAGTAA	9300<br>
GGGAGTGACC ATGAGCGAGA AAACCATGCG CGTGCAGGAT TATCCGTTAG CCACCCGCTG	9360<br>
CCCGGAGCAT ATCCTGACGC CTACCGGCAA ACCATTGACC GATATTACCC TCGAGAAGGT	94 20<br>
GCTCTCTGGC GAGGTGGGCC CGCAGGATGT GCGGATCTCC CGCCAGACCC TTGAGTACCA	94 80<br>
GGCGCAGATT GCCGAGCAGA TGCAGCGCCA TGCGGTGGCG CGCAATTTCC GCCGCGCGGC	9540<br>
GGAGCTTATC GCCATTCCTG ACGAGCGCAT TCTGGCTATC TATAACGCGC TGCGCCCGTT	9600<br><br>
CCGCTCCTCG CAGGCGGAGC TGCTGGCGAT CGCCGACGAG CTGGAGCPCA CCTGGCATGC	5660<br>
GACACTGAAT GCCGCCTTTG TCCGGGAGTC GGCGGAAGTG TATCAGCAGC GGCATAAGCT	9720<br>
GCGTAAAGGA AGCTAAGCGG AGGTCAGCAT GCCGTTAATA GCCGGGATTG ATATCGGCAA	9780<br>
CGCCACCACC GAGGTGGCGC TGGCGTCCGA CTACCCGCAG GCGAGGGCGT TTGTTGCCAG	9340<br>
CGGGATCGTC CCGACGACGG GCATGAAAGG GACGCGGGAC AATATCGCCG GGACCCTCGC	9900<br>
CGCGCTGGAG CAGGCCCTGG CGAAAACACC GTGGTCGATG AGCGATGTCT CTCGCATCTA	9960<br>
TCTTAACGAA GCCGCGCCGG TGATTGGCGA TGTGGCGATG GAGACCATCA CCGAGACCAT	10020<br>
TATCACCGAA TCGACCATGA TCGGTCATAA CCCGCAGACG CCGGGCGGGG TGGGCGTTGG	10080<br>
CGTGGGGACG ACTATCGCCC TCGGGCGGCT GGCGACGCTG CCGGCGGCGC AGTATGCCGA	1014 0<br>
GGGGTGGATC GTACTGATTG ACGACGCCGT CGATTTCCTT GACGCCGTGT GGTGGCTCAA	10200<br>
TGAGGCGCTC GACCGGGGGA TCAACGTGGT GGCGGCGATC CTCAAAAAGG ACGACGGCGT	10260<br>
GCTGGTGAAC AACCGCCTGC GTAAAACCCT GCCGGTGGTG GATGAAGTGA CGCTGCTGGA	10320<br>
GCAGGTCCCC GAGGGGGTAA TGGCGGCGGT GGAAGTGGCC GCGCCGGGCC AGGTGGTGCG	10380<br>
GATCCTGTCG AATCCCTACG GGATCGCCAC CTTCTTCGGG CTAAGCCCGG AAGAGACCCA	10440<br>
GGCCATCGTC CCCATCGCCC GCGCCCTGAT TGGCAACCGT TCCGCGGTGG TGCTCAAGAC	10500<br>
CCCGCAGGGG GATGTGCAGT CGCGGGTGAT CCCGGCGGGC AACCTCTACA TTAGCGGCGA	10560<br>
AAAGCGCCGC GGAGAGGCCG ATGTCGCCGA GGGCGCGGAA GCCATCATGC AGGCGATGAG	10620<br>
CGCCTGCGCT CCGGTACGCG ACATCCGCGG CGAACCGGGC ACCCACGCCG GCGGCATGCT	10680<br>
TGAGCGGGTG CGCAAGGTAA TGGCGTCCCT GACCGGCCAT GAGATGAGCG CGATATACAT	107 4 0<br>
CCAGGATCTG CTGGCGGTGG ATACGTTTAT TCCGCGCAAG GTGCAGGGCG GGATGGCCGG	10800<br>
CGAGTGCGCC ATGGAGAATG CCGTCGGGAT GGCGGCGATG GTGAAAGCGG ATCGTCTGCA	10860<br>
AATGCAGGTT ATCGCCCGCG AACTGAGCGC CCGACTGCAG ACCGAGGTGG TGGTGGGCGG	10920<br>
CGTGGAGGCC AACATGGCCA TCGCCGGGGC GTTAACCACT CCCGGCTGTG CGGCGCCGCT	10980<br>
GGCGATCCTC GACCTCGGCG CCGGCTCGAC GGATGCGGCG ATCGTCAACG CGGAGGGGCA	1104 0<br>
GATAACGGCG GTCCATCTCG CCGGGGCGGG GAATATGGTC AGCCTGTTGA TTAAAACCGA	11100<br>
GCTGGGCCTC GAGGA7CTTT CGCTGGCGGA AGCGATAAAA AAATACCCGC TGGCCAAAGT	11160<br>
GGAAAGCCTG TTCAGTATTC GTCACGAGAA TGGCGCGGTG GAGTTCTTTC GGGAAGCCCT	11220<br>
CAGCCCGGCG GTGTTCGCCA AAGTGGTGTA CATCAAGGAG GGCGAACTGG TGCCGATCGA	11280<br>
TAACGCCAGC CCGCTGGAAA AAATTCGTCT CGTGCGCCGG CAGGCGAAAG AGAAAGTGTT	11340<br>
TGTCACCAAC TGCCTGCGCG CGCTGCGCCA GGTCTCACCC GGCGGTTCCA TTCGCGATAT	114 00<br>
CGCCTTTGTG GTGCTGGTGG GCGGCTCATC GCTGGACTTT GAGATCCCGC AGCTTATCAC	11460<br>
GGAAGCCTTG TCGCACTATG GCGTGGTCGC CGGGCAGGGC AATATTCGGG GAACAGAAGG	11520<br>
GCCGCGCAAT GCGGTCGCCA CCGGGCTGCT ACTGGCCGGT CAGGCGAATT AAACGGGCGC	11580<br>
75<br><br>
TCGCGCCAGC CTCTCTCTTT AACGTGCTAT TTCAGGATGC CGATAATGAA CCAGACTTCT 11640 ACCTTAACCG GGGAGTGCGT GGCCGAGTTT CTTGGCACCG GATTGCICAT TTTCTTCGGC 11700<br>
GCGGGCTGCG TCGCTGCGCT GCGGGTCGCC GGGGCCAGCT TTGGTCAGTG GGAGATCAGT 11760 ATTATCTGGG GCCTTGGCGT CGCCATGGCC ATCTACCTCA CGGCCGGTGT CTCCGGCGCG 11820<br>
CACCT^TC CGGCGGTGAG CATTGCCCTG TGGCTGTTCG CCTGTTTTGA ACGCCGCA.G	11880<br>
GTGCTGGCGT TTATTGTTGC CCAGACGGCG GGGGCCTTCT GCGCCGCCGC GCTGGTGTAT	11940<br>
GGGCTCTATC GCCAGCTGTT TCTCGATCTT GAACAGAGTC AGCATATCGT GCGCGGCACT	12000<br>
GCCGCCAGTC TTAACCTGGC CGGGGTCTTT TCCACGTACC CGCATCCACA TATCACTTTT	12060<br>
ATACMGCGT TTGCCGTGGA GACCACCATC ACGGCAATCC TGATGGCGAT GATCATGGCC	12120<br>
CTGACCGACG ACGGCAACGG AATTC<br>
12145<br>
(2) INFORMATION FOR SEQ ID NO:20.<br>
(a) SEQUENCE CHARACTERISTICS<br>
{A) LENGTH 94 base paxrs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS- single<br>
(D) TOPOLOGY linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO:20:<br>
AGCTTAGGAG TCTAGAATAT TGAGCTCGAA TTCCCGGGCA TGCGGTACCG GATCCAGAAA 60<br>
AAAGCCCGCA CCTGACAGTG CGGGCTTTTT TTTT	94<br>
(2) INFORMATION FOR SEQ ID NO-21<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 37 base pairs CBJ TYPE- nucleic acid CO STRANDEDNESS single (D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO-21:<br>
GGAATTCAGA TCTCAGCAAT GAGCGAGAAA ACCATGC	37<br>
(2) INFORMATION FOR SEQ ID NO 22:<br>
(I) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH 27 base pairs<br>
(B) TYPE- nucleic acid<br>
(C) STRANDEDNESS: single<br>
(D) TOPOLOGY. linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 22<br>
GCTCTAGATT AGCTTCCTTT ACGCAGC	2 7<br>
(2) INFORMATION FOR SEQ ID NO 23:<br><br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 33 base pairs (3) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOG/ linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 23<br>
GGCCAAGCTT AAGGAGGTTA ATTAAATGAA AAG<br>
(2) INFORMATION FOR SEQ ID NO 24<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 26 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(ii) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO:24.<br>
GCTCTAGATT ATTCAATGGT GTCGGG<br>
(2) INFORMATION FOR SEQ ID NO 25<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 42 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single (D) TOPOLOGY: linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 25<br>
GCGCCGTCTA GAATTATGAG CTATCGTATG TTTGATTATC TG<br>
(2) INFORMATION FOR SEQ ID NO"26<br>
(l) SEQUENCE CHARACTERISTICS.<br>
(A) LENGTH 36 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS- Single<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO 26<br>
TCTGATACGG GATCCTCAGA ATGCCTGGCG GAAAAT<br>
(2) INFORMATION FOR SEQ ID NO:27-<br>
(l) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 51 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(ii) MOLECULE TYPE DNA (genomic)<br><br>
(.1) SEQUENCE DESCRIPTION SEQ ID NO 27 GCGCGGATCC AGGAGTCTAG AATTATGGGA TTGACTACTA AACCTCTATC T (2) INFORMATION FOP SEQ ID NO 28<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 36 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS Single<br>
(D) TOPOLOGY linear<br>
(11) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO:28<br>
GATACGCCCG GGTTACCATT TCAACAGATC GTCCTT<br>
(2) INFORMATION FOR SEQ ID NO-29<br>
U) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 18 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(n) MOLECULE TYPE DNA (genomic) <br><br><br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO:29 <br>
TCGACGAATT CAGGAGGA	<br>
(2) INFORMATION FOR SEQ ID NO:30<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 18 base pairs <br>
(B) TYPE nucleic acid	<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(li) MOLECULE TYPE DNA (genomic) (xi) SEQUENCE DESCRIPTION" SEQ ID NO 30<br>
CTAGTCCTCC TGAATTCG<br>
(2) INFORMATION FOR SEQ ID NO:31<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH 19 base pairs<br>
(B) TYPE: nucleic acid<br>
(C) STRANDEDNESS. single<br>
(D) TOPOLOGY linear<br>
(li) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 31.<br>
CTAGTAAC-GA GGACAATTC<br>
(2) INFORMATION FOR SEQ ID NO.32<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 19 base pairs<br>
(B) TYPE. nucleic acid<br>
(A) <br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(11) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 32<br>
CATGGAATTG TCCTCCTTA	19<br>
(2) INFORMATION FOR SEQ ID NO 33<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 271 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown {D) TOPOLOGY unknown<br>
(11) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM GPP1<br>
(xi) SEQUENCE DESCRIPTION- SEQ ID NO 33<br>
Met Lys Arg Phe Asn Val Leu Lys Tyr lie Arg Thr Thr Lys Ala Asn<br>
15	10	15<br>
lie Gln Thr lie Ala Met Pro Leu Thr Thr Lys Pro Leu Ser Leu Lys<br>
20	25	30<br>
lie Asn Ala Ala Leu Phe Asp Val Asp Gly Thr Ile lie Ile Ser Gln<br>
35	40	45<br>
Pro Ala Ile Ala Ala Phe Trp Arg Asp Phe Gly Lys Asp Lys Pro Tyr<br>
50	55	60<br>
Phe Asp Ala Glu His Val Ile His Ile Ser His Gly Trp Arg Thr Tyr<br>
65	70	75	80<br>
Asp Ala Ile Ala Lys Phe Ala Pro Asp Phe Ala Asp Glu Glu Tyr Val<br>
85	90 	95<br>
Asn Lys Leu Glu Gly Glu Ile Pro Glu Lys Tyr Gly Glu His Ser Ile<br>
100	105	110<br>
Glu Val Pro Gly Ala Val Lys Leu Cys Asn Ala Leu Asn Ala Leu Pro<br>
115	120	125<br>
Lys Glu Lys Trp Ala Val Ala Thr Ser Gly Thr Arg Asp Met Ala Lys<br>
130	135	140<br>
Lys Trp Phe Asp Ile Leu Lys* Ile Lys Arg Pro Glu Vyr Phe Ile Thr<br>
145	150	155	160<br>
Ala Asn Asp Val Lys Gln Gly Lys Pro His Pro Glu Pro Tyr Leu Lys<br>
165	170	175<br>
Gly Arg Asn Gly Leu Gly Phe Pro Ile Asn Glu Gln Asp Pro Ser Lys<br>
180	185	190<br>
Ser Lys Val Val Val Phe Glu Asp Ala Pro Ala Gly Ile Ala Ala Gly<br>
195	200	205<br>
Lys Ala Ala Gly Cys Lys Ile Val Gly Ile Ala Thr Thr Phe Asp Leu<br>
210	215	220<br><br>
Asp Phe Leu LiS Giu Lys Gly Cys Asp Ile lie Val Lys Asn His Gis<br>
225	230	235 243	<br>
Ser lie Arg Val Gly Glu Tyr Asn Ala Glu Thr Aso Glu Vai Gia Leu<br>
245	250	255<br>
Ile Phe Asp Asp Tyr Leu Tyr Ala Lys Asp Aso Leu Leu Lys Tro<br>
260	265	270<br>
(2) INFORMATION FOR SEQ ID NO 34<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 555 amino acids CB) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(n) MOLECULE TYPE- protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM: DHABI<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO"34<br>
Met Lys Arg Ser Lys Arg Phe Ala Val Leu Ala Gln Arg Pro Val Asn<br>
1	5	10	15<br>
Gln Asp Gly Leu Ile Gly Glu Trp Pro Glu Glu Gly Leu Ile Ala Met<br>
20	25	30<br>
Asp Ser Pro Phe Asp Pro Val Ser Ser Val Lys Val Asp Asn Gly Leu<br>
35	40	45<br>
Ile Val Glu Leu Asp Gly Lys Arg Arg Asp Gln Phe Asp Met Ile Asp<br>
50	55	60<br>
Arg Phe Ile Ala Asp Tyr Ala Ile Asn Val Glu Arg Thr Glu Gln Ala<br>
65	70	75	80<br>
Met Arg Leu Glu Ala Val Glu Ile Ala Arg Met Leu Val Asp Ile His<br>
85	90	95<br>
Val Ser Arg Glu Glu Ile Ile Ala Ile Thr Thr Ala Ile Thr Pro Ala<br>
100	105	HO<br>
Lys Ala Val Glu Val Met Ala Gln Met Asn Val Val Glu Met Met Met<br>
115	120	125<br>
Ala Leu Gln Lys Met Arg Ala Arg Arg Thr Pro Ser Asn Gln Cys His<br>
130	135	140<br>
Val Thr Asn Leu Lys Asp Asn Pro Val Gln Ile Ala Ala Asp Aia Ala<br>
145	150	155	160<br>
Glu Ala Gly Ile Arg Gly Phe Ser Glu Gln Glu Thr Thr Val Gly Ile<br>
165	170	175<br>
Ala Arg Tyr Ala Pro Phe Asn Ala Leu Ala Leu Leu Val Gly Ser Gln<br>
180	185	190<br>
Cvs Glv Arg Pro Gly Val Leu Thr Gln Cys Ser Val Glu Glu Ala Thr<br>
195	200	205<br>
Glu Leu Glu Leu Gly Met Arg Gly Leu Thr Ser Tyr Ala Glu Thr Val<br>
210	215	220<br><br>
Ser Val Tyr Gly Tnr Glu Ala Vax Phe Thr ASD Gly Asp Asp Tnr Pro<br>
225	230	235 	2-10<br>
Trp Ser Lys Ala Phe Leu Ala Ser Ala Tyr Ala Ser Arg Gly Leu Lys<br>
245	250	255<br>
Met Arg Tyr Thr Ser Gly Thr Gly Ser G±u Ala Leu Met Gly Tyr Ser<br>
260	265	270<br>
Glu Ser Lys Ser Met Leu Tyr Leu Glu Ser Arg Cys lie Phe lie Thr<br>
275	280	285<br>
Lys Gly Ala Gly Val Gln Gly Leu Gln Asn Gly Ala Val Ser Cys lie<br>
290	295	300<br>
Gly Met Thr Gly Ala Val Pro Ser Gly lie Arg Ala Val Leu Ala Glu<br>
305	310	315	320<br>
Asn Leu lie Ala Ser Met Leu Asp Leu Glu Val Ala Ser Ala Asn Asp<br>
325	330	335<br>
Gln Thr Phe Ser His Ser Asp lie Arg Arg Thr Ala Arg Thr Leu Met<br>
340	345	350<br>
Gln Met Leu Pro Gly Thr Asp Phe lie Phe Ser Gly Tyr Ser Ala Val<br>
355	360	365<br>
Pro Asn Tyr Asp Asn Met Phe Ala Gly Ser Asn Phe Asp Ala Glu Asp<br>
370	375	380<br>
Phe Asp Asp Tyr Asn lie Leu Gln Arg Asp Leu Met Val Asp Gly Gly<br>
385	390	395	400<br>
Leu Arg Pro Val Thr Glu Ala Glu Thr lie Ala Ile Arg Gln Lys Ala<br>
405	410	415<br>
Ala Arg Ala Ile Gln Ala Val Phe Arg Glu Leu Gly Leu Pro Pro Ile<br>
420	425	430<br>
Ala Asp Glu Glu Val Glu Ala Ala Thr Tyr Ala His Gly Ser Asn Glu<br>
435	440	445<br>
Met Pro Pro Arg Asn Val Val Glu Asp Leu Ser Ala Val Glu Glu Met<br>
450	455	460<br>
Met Lys Arg Asn Ile Thr Gly Leu Asp Ile Val Gly Ala Leu Ser Arg<br>
465	470	475	480<br>
Ser Gly Phe Glu Asp Ile Ala Ser Asn Ile Leu Asn Met Leu Arg Gln<br>
485	490	495<br>
Arg Val Thr Gly Asp Tyr Leu Gln Thr Ser Ala Ile Leu Asp Arg Gln<br>
500	505	510<br>
Phe Glu Val Val Ser Ala Val Asn Asp Ile Asn Asp Tyr Gln Gly Pro<br>
515	520	525<br>
Gly Thr Gly Tyr Arg Ile Ser Ala Glu Arg Trp Ala Glu Ile Lys Asn<br>
530	535	540<br>
Ile Pro Gly Val Val Gln Pro Asp Thr Ile Glu<br>
545	550	555<br><br>
81<br><br>
(2) INFORMATION FOR SEQ ID NO 35<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 194 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(II) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM DHAB2<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO.35<br>
Met Gln Gln Thr Thr Gln lie Gln Pro Ser Phe Thr Leu Lys Thr Arg<br>
15	10	15<br>
Glu Gly Gly Val Ala Ser Ala Asp Glu Arg Ala Asp Glu Val Val lie<br>
20	25	30<br>
Gly Val Gly Pro Ala Phe Asp Lys His Gln His His Thr Leu lie Asp<br>
35	40	45<br>
Met Pro His Gly Ala lie Leu Lys Glu Leu lie Ala Gly Val Glu Glu<br>
50	55	60<br>
Glu Gly Leu His Ala Arg Val Val Arg lie Leu Arg Thr Ser Asp Val<br>
65	70	75	80<br>
Ser Phe Met Ala Trp Asp Ala Ala Asn Leu Ser Gly Ser Gly lie Gly<br>
85	90	95<br>
lie Glv lie Gln Ser Lys Gly Thr Thr Val lie His Gln Arg Asp Leu<br>
100	105	110<br>
Leu Pro Leu Ser Asn Leu Glu Leu Phe Ser Gln Ala Pro Leu Leu Thr<br>
115	120	125<br>
Leu Glu Thr Tyr Arg Gln lie Gly Lys Asn Ala Ala Arg Tyr Ala Arg<br>
130	135	140<br>
Lys Glu Ser Pro Ser Pro Val Pro Val Val Asn Asp Gln Met Val Arg<br>
145	150	155	160<br>
Pro Lys Phe Met Ala Lys Ala Ala Leu Phe His lie Lys Glu Thr Lys<br>
165	170	175<br>
His Val Val Gln Asp Ala Glu Pro Val Thr Leu His lie Asp Leu Val<br>
180	185	190<br>
Arg Glu (2) INFORMATION FOR SEQ ID NO 36<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 140 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(II) MOLECULE TYPE protein<br>
(VI) ORIGINAL SOURCE<br>
(A) ORGANISM- DHAB3<br><br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 36<br>
Met Ser Glu Lys Thr Met Arg Val Gln Asp Tyr Pro Leu Ala Thr Arg<br>
5	10 15<br>
Cys Pro Glu His Ile Leu Thr Pro Thr Gly Lys Pro Leu Thr Asp Ile<br>
20	25	30<br>
Tnr Leu Glu Lys Val Leu Ser Gly Glu Val Gly Pro Gln Asp Val Arg<br>
35	40	45<br>
Ile Ser Arg Gln Thr Leu Glu Tyr Gln Ala Gln Ile Ala Glu Gln Met<br>
50	55	60<br>
Gln His Ala Val Ala Arg Asn Phe Arg Arg Ala Ala Glu Leu Ile Ala<br>
65	70	75 80<br>
Ile Pro Asp Glu Arg Ile Leu Ala Ile Tyr Asn Ala Leu Arg Pro Phe<br>
85	90	95<br>
Arg Ser Ser Gln Ala Glu Leu Leu Ala Ile Ala Asp Glu Leu Glu His<br>
100	105	110<br>
Thr Trp His Ala Thr Val Asn Ala Ala Phe Val Arg Glu Ser Ala Glu<br>
115	120	125<br>
Val Tyr Gln Gln Arg His Lys Leu Arg Lys Gly Ser<br>
130	135	140<br>
(2) INFORMATION FOR SEQ ID NO 37:<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 387 amino acids<br>
(B) TYPE amino acid<br>
(C) STRANDEDNESS unknown<br>
(D) TOPOLOGY unknown<br>
(11) MOLECULE TYPE protein<br>
(vi) ORIGINAL SOURCE<br>
(A) ORGANISM: DHAT<br>
(xi) SEQUENCE DESCRIPTION. SEQ ID NO 37<br>
Met Ser Tyr Arg Met Phe Asp Tyr Leu Val Pro Asn Val Asn Phe Phe<br>
15	10	15<br>
Gly Pro Asn Ala Ile Ser Val Val Gly Glu Arg Cys Gln Leu Leu Gly<br>
20	25	30<br>
Gly Lys Lys Ala Leu Leu Val Thr Asp Lys Gly Leu Arg Ala Ile Lys<br>
35	40	45<br>
Asp Gly Ala Val Asp Lys Thr Leu His Tyr Leu Arg Glu Ala Gly Ile<br>
50	55	60<br>
Glu Val Ala Ile Phe Asp Gly Val Glu Pro Asn Pro Lys Asp Thr Asn<br>
65	70	75 80<br>
Val Arg Asp Gly Leu Ala Val Phe Arg Arg Glu Gln Cys Asp Ile Ile<br>
85	90	95<br>
Val Thr Val Gly Gly Gly Ser Pro His Asp Cys Gly Lys Gly Ile Gly<br>
100	105	110<br><br>
Ale Aia Ala Thr His Glu Gly ASD Leu Tyr Gln Tvr Ala Gly Ile Gli<br>
115	120	125<br>
Thr Leu Thr Asn Pro Leu Pro Pro Ile Val Ala Val Asn Thr Thr Ala<br>
130	135	140<br>
Gly Thr Ala Ser Glu Val Thr Arg His Cys Val Leu Thr Asn Thr Glu<br>
145	150	155	160<br>
Thr Lys^Val Lys Phe Val Ile Val Ser Trp Arg Lys Leu Pro Ser Val<br>
165	170	175<br>
Ser Ile Asn Asp Pro Leu Leu Met Ile Gly Lys Pro Ala Ala Leu Thr<br>
180	185	190<br>
Ala Ala Thr Gly Met Asp Ala Leu Thr His Ala Val Glu Ala Tyr Ile<br>
195	200	205<br>
Ser Lys Asp Ala Asn Pro Val Thr Asp Ala Ala Ala Met Gln Ala Ile<br>
210	215	220<br>
Arg Leu Ile Ala Arg Asn Leu Arg Gln Ala Val Ala Leu Gly Ser Asn<br>
225	230	235	_ 240<br>
Leu Gln Ala Arg Glu Asn Met Ala Tyr Ala Ser Leu Leu Ala Gly Met<br>
245	250	255<br>
Ala Phe Asn Asn Ala Asn Leu Gly Tyr Val His Ala Met Ala His Gln<br>
260	265	270<br>
Leu Gly Gly Leu Tyr Asp Met Pro His Gly Val Ala Asn Ala Val Leu<br>
275	280	285<br>
Leu Pro His Val Ala Arg Tyr Asn Leu Ile Ala Asn Pro Glu Lys Phe<br>
290	295	300<br>
Ala Asp Ile Ala Glu Leu Met Gly Glu Asn Ile Thr Gly Leu Ser Thr<br>
305	310	315	320<br>
Leu Asp Ala Ala Glu Lys Ala Ile Ala Ala Ile Thr Arg Leu Ser Met<br>
325	330	335<br>
Asp Ile Gly Ile Pro Gln His Leu Arg Asp Leu Gly Val Lys Glu Ala<br>
340	345	350<br>
Asp Phe Pro Tyr Met Ala Glu Met Ala Leu Lys Asp Gly Asn Ala Phe<br>
355	360	365<br>
Ser Asn Pro Arg Lys Gly Asn Glu Gln Glu Ile Ala Ala Ile Phe Arg<br>
370	375	380<br>
Gln Ala Phe 385<br>
(2) INFORMATION FOR SEQ ID NO:38.<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH: 27 base pairs<br>
(B) TYPE- nucleic acid<br>
(C) STRANDEDNESS- single<br>
(D) TOPOLOGY: linear<br>
(li) MOLECULE TYPE DNA (genomic)<br>
84<br><br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 38 GCGAATTCAT GAGCTATCGT ATGTTTG (2) INFORMATION FOR SEQ ID NO 39<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 28 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(n) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 39<br>
GCGAATTCAG AATGCCTGGC GGAAAATC<br>
(2) INFORMATION FOR SEQ ID NO 40<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 28 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic) (xi) SEQUENCE DESCRIPTION SEQ ID NO 40 GGGAATTCAT GAGCGAGAAA ACCATGCG (2) INFORMATION FOR SEQ ID NO 41:<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH. 27 base pairs<br>
(B) TYPE: nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO-41<br>
GCGAATTCTT AGCTTCCTTT ACGCAGC<br>
(2) INFORMATION FOR SEQ ID NO 42.<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH: 30 base pairs<br>
(B) TYPE nucleic acid (C STRANDEDNESS: single {D) TOPOLOGY• linear<br>
(ii) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 42<br>
GCGAATTCAT GCAACAGACA ACCCAAATTC<br>
(2) INFORMATION FOR SEQ ID NO 43<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 25 base pairs<br>
(B) TYPE nucleic acid<br>
(A) <br>
(C) STRPNDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 43<br>
GCGAATTCAC TCCCTTACTA AGTCG<br>
(2) INFORMATION FOR SEQ ID NO 44<br>
(l) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH- 30 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic) (xi) SEQUENCE DESCRIPTION SEQ ID NO:44 GGGAATTCAT GAAAAGATCA AAACGATTTG (2) INFORMATION FOR SEQ ID NO-45:<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 29 base pairs<br>
(B) TYPE, nucleic acid<br>
(C) STRANDEDNESS. Single<br>
(D) TOPOLOGY. linear<br>
(II) MOLECULE TYPE DNA (genomic) (xi) SEQUENCE DESCRIPTION: SEQ ID NO 45 GCGAATTCTT ATTCAATGGT GTCGGGCTG<br>
(2) INFORMATION FOR SEQ ID NO:46<br>
(i) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 34 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS. single<br>
(D) TOPOLOGY linear<br>
(ii) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION- SEQ ID NO 46.<br>
TTGATAATAT AACCATGGCT GCTGCTGCTG ATAG<br>
(2) INFORMATION FOR SEQ ID NO 47<br>
(I) SEQUENCE CHARACTERISTICS.<br>
(A) LENGTH 39 base pairs<br>
(B) TYPE* nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE- DNA (genomic) (xi) SEQUENCE DESCRIPTION SEQ ID NO 47 GTATGATATG TTATCTTGGA TCCAATAAAT CTAATCTTC<br>
86<br><br>
(2)	INFORMATION FOR SEQ ID NO 48<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH 24 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE DNA (genomic)<br>
(xi) SEQUENCE DESCRIPTION SEQ ID NO 48<br>
CATGACTAGT AAGGAGGACA ATTC<br>
(2) INFORMATION FOR SEQ ID NO 4 9<br>
(I) SEQUENCE CHARACTERISTICS<br>
(A) LENGTH. 24 base pairs<br>
(B) TYPE nucleic acid<br>
(C) STRANDEDNESS single<br>
(D) TOPOLOGY linear<br>
(II) MOLECULE TYPE: DNA (genomic) (xi) SEQUENCE DESCRIPTION SEQ ID NO:49 CATGGAATTG TCCTCCTTAC TAGT<br>
87<br><br>
We Claim <br>
1. A method for the production of 1,3-propanediol from a recombinant organism comprising<br>
(i) transforming in the manner such herein described a suitable host organism of the kind such as herein described with one or more transformation cassettes each of which comprises at least one of:<br>
(a) a gene encoding a glycerol-3-phosphate dehydrogenase activity;<br>
(b) a gene encoding a glycerol-3-phosphatase activity;<br>
(c) genes encoding a dehydratase activity;<br>
(d) a gene encoding 1,3-propanediol oxidoreductase activity,<br>
wherein all of the genes (a) - (d) are introduced into the host microorganism,<br>
(ii) culturing the transformed host organism in the manner such as herein described in the presence of at least one carbon source selected from the group consisting of monosaccharides, oligosaccharides, polysaccharides, or a one-carbon substrate whereby 1,3-propanediol is produced, and<br>
(iii) recovering in the manner such as herein described the 1,3-propanediol.<br>
2 The method as claimed in claim 1 wherein the suitable host organism is selected from the group consisting of bacteria, yeast, and filamentous fungi.<br>
3 The method as claimed in claim 2 wherein the suitable host organism is selected from the group of genera consisting of Citrobacter, Enterobacter, Clostridium, Klebsiella, Aerobacter, Lactobacillus, Aspergillus, Saccharomyces, Schizosaccharomyces, Zygosaccharomyces, Pichia, Kluyveromyces, Candida, Hansenula,<br>
2 <br>
Debaryomyces, Mucor, Torulopsis, Methylobacter, Escherichia, Salmonella, Bacillus, Streptomyces and Pseudomonas.<br>
4	The method as claimed in claim 3 wherein the suitable host<br>
organism is selected from the group consisting of E. coli, Klebsiella spp., and Saccharomyces spp.<br>
5. The method as claimed in claim 1 wherein the transformed host organism is a Saccharomyces spp transformed with a transformation cassette comprising the genes dhaBl, dhaB2, dhaB3, and dhaT, wherein the genes are stably integrated into the Saccharomyces spp genome<br>
6	The method as claimed in claim 1 wherein the transformed host<br>
organism is a Klebsiella spp transformed with a transformation cassette comprising the genes GPD1 and GPD2<br>
7. The method as claimed in claim 1 wherein the carbon source is glucose<br>
8	The method as claimed in claim 1 wherein the gene encoding a<br>
glycerol-3-phosphate dehydrogenase enzyme is selected from the group consisting of genes corresponding to amino acid sequences given in SEQ ID NO: 11, in SEQ ID NO: 12, and in SEQ ID NO. 13, the amino acid sequences encompassing amino acid substitutions, deletions or additions that do not alter the function of the glycerol-3-phosphate dehydrogenase enzyme.<br>
9. The method as claimed in claim 1 wherein the gene encoding a glycerol-3-phosphatase enzyme is selected from the group consisting of genes corresponding to amino acid sequences given in SEQ ID NO:33 and in SEQ ID NO. 17, the amino acid sequences encompassing amino acid substitutions, deletions or additions that do not alter the function of the glycerol-3-phosphatase enzyme.<br>
10 The method as claimed in claim 1 wherein the gene encoding a glycerol kinase enzyme corresponds to an amino acid sequence given in SEQ ID NO: 18, the amino acid sequence encompassing amino acid substitutions,' deletions or additions that do not alter<br><br>
the function of the glycerol kinase enzyme<br>
11. The method as claimed in claim 1 wherein the genes encoding a dehydratase enzyme comprise dhaBl, dhaB2 and dhB3, the genes corresponding respectively to amino acid sequences given in SEQ ID NO 34, SEQ ID NO 35, and SEQ ID NO 36, the amino acid sequences encompassing amino acid substitutions, deletions or additions that do not alter the function of the dehydratase enzyme.<br>
12. The method as claimed in claim 1 wherein the gene encoding a 1,3-propanediol oxidoreductase enzyme corresponds to an amino acid sequence given in SEQ ID NO.37, the amino acid sequence encompassing amino acid substitutions, deletions or additions that do not alter the function of the 1,3-propanediol oxidoreductase enzyme<br>
13 A method for the production of 1,3-propanediol substantially such<br>
as herein described with the help of forgoing examples. Dated this 13th day of November, 1997.<br><br>
[DR. RAMESH KUMAR MEHTA]<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPLICANTS<br>
90<br>
A method for the production of 1,3-propanediol from a recombinant organism is provided comprising 1) transforming a suitable host organism with one or more transformation cassettes, ii) culturing the transformed host organism in the presence of at least one carbon source whereby 1,3-propanediol is produced, and in) recovering the 1,3-propanediol.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">02142-cal-1997-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">02142-cal-1997-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">02142-cal-1997-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">02142-cal-1997-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">02142-cal-1997-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctb3RoZXIgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">02142-cal-1997-other document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDIxNDItY2FsLTE5OTctcHJpb3JpdHkgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">02142-cal-1997-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny0oMTctMDUtMjAxMiktQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-(17-05-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny0oMTctMDUtMjAxMiktRk9STS0yNy5wZGY=" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-(17-05-2012)-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny0oMTctMDUtMjAxMiktUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-(17-05-2012)-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny0oMjctMTAtMjAxMSktQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-(27-10-2011)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny0oMjctMTAtMjAxMSktUEEucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-(27-10-2011)-PA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1DT1JSRVNQT05ERU5DRSAxLjEucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1DT1JSRVNQT05ERU5DRS0xLjIucGRm" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-CORRESPONDENCE-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1GT1JNIDI3LTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-FORM 27-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1GT1JNIDI3LnBkZg==" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1QQS50aWY=" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-PA.tif</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE0Mi1DQUwtMTk5Ny1QRVRJVElPTiBVTkRFUiBSVUxFIDEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">2142-CAL-1997-PETITION UNDER RULE 138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="195458-a-device-determining-the-orientation-of-a-multi-component-detector-using-data-from-a-multi-component-seismic-survey.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="195460-adjustable-wrench.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>195459</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2142/CAL/1997</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Nov-2005</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Nov-1997</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>E. I. DU PONT DE NEMOURS AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1007 MARKET STREET, WILMINGTON, DELAWARE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHARON LORETTA HAYNIE</td>
											<td>963 NORTH RANDOLPH STREET, PHILADELPHIA, PENNSYLVANIA 19123</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAMESCH V. NAIR</td>
											<td>905-D CLOISTER ROAD, WILMINGTON, DELAWARE 19809</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARK SCOTT PAYNE</td>
											<td>2408 NEWELL DRIVE, WILMINGTON, DELAWARE 19808</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MARIA DIAZ-TORRES</td>
											<td>58, N EL CAMINO REAL #118 SAN MATEO, CALIFORNIA 94401</td>
										</tr>
										<tr>
											<td>5</td>
											<td>RICHARD D. LAREAU</td>
											<td>197A EASY STREET, MOUNTAIN VIEW, CALIFORNIA 94043</td>
										</tr>
										<tr>
											<td>6</td>
											<td>GREGORY M. WHITED</td>
											<td>304 SOUTH ROAD, BELMONT, CALIFORNIA 94002</td>
										</tr>
										<tr>
											<td>7</td>
											<td>GATENBY ANTHONY ARTHUR</td>
											<td>2309, BAYNARD BOULEVARD, WILMINGTON, DELAWARE 19802</td>
										</tr>
										<tr>
											<td>8</td>
											<td>NAKAMURA CHARLES E.</td>
											<td>2 MOUNT VERNON DRIVE, CLAYMONT, DELAWARE 19703</td>
										</tr>
										<tr>
											<td>9</td>
											<td>NAGARAJAN VASANTHA</td>
											<td>13, DICKINSON LANE, WILMINGTON, DELAWARE 19807</td>
										</tr>
										<tr>
											<td>10</td>
											<td>PICATAGGIO STEPHEN KENNETH</td>
											<td>17 MEADOW WOOD LANE, LANDENBER, PENNSYLVANIA 19350</td>
										</tr>
										<tr>
											<td>11</td>
											<td>HSU AMY KUANG-HUA</td>
											<td>528, KEELSON CIRCLE, REDWOOD CITY, CALIFORNIA - 94065</td>
										</tr>
										<tr>
											<td>12</td>
											<td>TRIMBUR DONALD E.</td>
											<td>349, ORCHARD AVENUE, REDWOOD CITY, CALIFORNIA 94601</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/53, 15/55</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/030,601</td>
									<td>1996-11-13</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/195459-a-method-for-the-production-of-1-3-propanediol by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:32:32 GMT -->
</html>
