<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/197016-a-process-for-preparation-of-panel-for-cell-based-detection-and-differentiation-of-lung-cancer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:14:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 197016:A PROCESS FOR PREPARATION OF PANEL FOR CELL-BASED DETECTION AND DIFFERENTIATION OF LUNG CANCER</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PREPARATION OF PANEL FOR CELL-BASED DETECTION AND DIFFERENTIATION OF LUNG CANCER</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a method of detecting and differentiating disease states with high sensitivity and specificity. The method allows for a determination of whether a cell-based sample contains abnormal cells and, for certain disease, is capable of determining the histologic type of disease present. The method detects changes in the level and pattern of expression of the molecular markers in the cell-based sample. Panel selection and validation procedures are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>CELL-BASED DETECTION AND DIFFERENTIATION OF DISEASE STATES<br>
BACKGROUND OF THE INVENTION<br>
The present invention relates to early detection of a general disease state in a patient. The present invention also relates to discrimination (differentiation) between specific disease states in their early stages.<br>
Early detection of a specific disease state can greatly improve a patient's chance for survival by permitting early diagnosis and early treatment while the disease is still localized and its pathologic effects limited anatomically and physiologically . Two key evaluative measures of any test or disease detection method are its sensitivity (Sensitivity = True Positives/(True Positives + False Negatives) and specificity (Specificity = True Negatives/(False Positives + True Negatives), which measure how well the test performs to accurately detect all affected individuals without exception, and without falsely including individuals who do not have the target disease. Historically, many diagnostic tests have been criticized due to poor sensitivity and specificity.<br>
Sensitivity is a measure of a test's ability to detect correctly the target disease in an individual being tested. A test having poor sensitivity produces a high rate of false negatives, i.e., individuals who have the disease but are falsely identified as being free of that particular disease. The potential danger of a false negative is that the diseased individual will remain undiagnosed and untreated for some period of time, during which the disease may progress to a later stage wherein treatments, if any, may be less effective. An example of a test that has low sensitivity is a protein-based blood test for HIV. This type of test exhibits poor sensitivity because it fails to detect the presence of the virus until the disease is well established and the virus has invaded the bloodstream in substantial numbers. In contrast, an example of a test that has high sensitivity is viral-load detection using the polymerase chain reaction (PCR). High sensitivity is achieved because this type of test can detect very small quantities of the virus (see Lewis, D.R. et al. "Molecular Diagnostics: The Genomic Bridge Between Old and New Medicine: A White Paper on the Diagnostic Technology and Services Industry" Thomas Weisel Partners, June 13, 2001).<br>
Specificity, on the other hand, is a measure of a test's ability to identify accurately patients who are free of the disease state. A test having poor specificity produces<br>
a high rate of false positives, i.e., individuals who are falsely identified as having the disease. A drawback of false positives is that they force patients to undergo unnecessary medical procedures treatments with their attendant risks, emotional and financial stresses, and which could have adverse effects on the patient's health. A feature of diseases which makes it difficult to develop diagnostic tests with high specificity is that disease mechanisms often involve a plurality of genes and proteins. Additionally, certain proteins may be elevated for reasons unrelated to a disease state. An example of a test that has high specificity is a gene-based test that can detect a p53 mutation. A p53 mutation will never be detected unless there are cancer cells present (see Lewis, D.R. et al. "Molecular Diagnostics: The Genomic Bridge Between Old and New Medicine: A White Paper on the Diagnostic Technology and Services Industry" Thomas Weisel Partners, June 13, 2001).<br>
Cellular markers are naturally occurring molecular structures within cells that can be discovered and used to characterize or differentiate cells in health and disease. Their presence can be detected by probes, invented and developed by human beings, which bind to markers enabling the markers to be detected through visualization and/or quantified using imaging systems. Four classes of cell-based marker detection technologies are cytopathology, cytometry, cytogenetics and proteomics, which are identified and described below.<br>
Cytopathology relies upon the visual assessment by human experts of cytomorphological changes within stained whole-cell populations. An example is the cytological screening and cytodiagnosis of Papanicolaou-stained cervical-vaginal specimens by cytotechnologists and cytopathologists, respectively. Unlike cytogenetics, proteomics and cytometry, cytopathology is not a quantitative tool. While it is the state-of-the-art in clinical diagnostic cytology, it is subjective and the diagnostic results are often not highly sensitive or reproducible, especially at early stages of cancer (e.g., ASCUS, LSIL).<br>
Tests that rely on morphological analyses involve observing a sample of a patient's cells under a microscope to identify abnormalities in cell and nuclear shape, size, or staining behavior. When viewed through a microscope, normal mature epithelial cells appear large and well differentiated, with condensed nuclei. Cells characterized by dysplasia, however, may be in a variety of stages of differentiation,with some cells being very immature. Finally, ceils characterized by invasive carcinoma often appear undifferentiated, with very little cytoplasm and relatively large nuclei.<br>
A drawback to diagnostic tests that rely on morphological analyses is that cell morphology is a lagging indicator. Since form follows function, often the disease state has<br>
already progressed to a critical stage by the time the disease becomes evident by morphological analysis. The initial stages of a disease involve chemical changes at a molecular level. Changes that are detectable by viewing cell features under a microscope are not apparent until later stages of the disease. Therefore, tests that measure chemical changes on a molecular level, referred to as "molecular diagnostic" tests, are more likely to provide early detection than tests that rely on morphological analyses alone.<br>
Cytometry is based upon the flow-microfluorometric instrumental analysis of fiuorescently stained cells moving in single file in solution (flow cytometry) or the computer-aided microscope instrumental analysis of stained cells deposited onto glass microscope slides (image cytometry). Flow cytometry applications include leukemia and lymphoma immunophenotyping. Image cytometry applications include DNA ploidy, Malignancy-Associated Changes (MACs) and S-phase analyses. The flow and image cytometry approaches yield quantitative data characterizing the cells in suspension or on a glass microscope slide. Flow and image cytometry can produce good marker detection and differentiation results depending upon the sensitivity and specificity of the cellular stains and flow/image measurement features used.<br>
Malignancy-Associated Changes (MACs) have been qualitatively observed and reported since the early to mid-1900's (OC Gruner: "Study of the changes met with leukocytes in certain cases of malignant disease" in Brit J Surg 3: 506-522, 1916) (HE Neiburgs, FG Zak, DC Allen, H Reisman, T Clardy: "Systemic cellular changes in material from human and animal tissues" in Transactions, 7th Ann Mtg Inter Soc Cytol Council, pp 137-144, 1959). From the mid-1900's through 1975, MACs were documented in independent qualitative histology and cytology studies in buccal mucosa and buccal smears (Nieburgs, Finch, Klawe), duodenum (Nieburgs), liver (Elias, Nieburgs), megakaryocytes (Ramsdahl), cervix (Nieburgs, Howdon), skin (Kwitiken), blood and bone marrow (Nieburgs), monocytes and leukocytes (van Haas, Matison, Clausen), and lung and sputum (Martuzzi and Oppen Toth). Before 1975 these qualitative studies reported MAC-based sensitivities for specific disease detection from 76% to 97% and specificities from 50% to 90%. In 1975 Oppen Toth reported a sensitivity of 76% and specificity of 81% in a qualitative sputum analysis study.<br>
Quantitative observations regarding MAC-based probe analysis began two to three decades ago (H Klawe, J Rowinski: "Malignancy associated changes (MAC) in cells of buccal smears detected by means of objective image analysis" in Acta Cytol 18: 30-33, 1974) (GL Wied, PH Battels, M Bibbo, JJ Sychra: "Cytomorphometric markers for uterine cancer<br>
in intermediate cells" in Analyt Quant Cytoi 2: 257-263, 1980) (G Burger, U Jutting, K Rodenacker: "Changes m benign population in cases of cervical cancer and its precursors" in Analyt Quant Cytol 3: 261-271, 1981). MACs were documented in independent quantitative histology and cytology studies in buccal mucosa and smears Klawe, Burger), cervix (Wied, Burger, Bartels, Vooijs, Reinhardt, Rosenthal, Boon, Katzke, Haroske, Zahniser), breast (King, Bibbo, Susnik), bladder and prostate (Sherman, Montironi), colon (Bibbo), lung and sputum (Swank, MacAulay, Payne), and nasal mucosa (Reith) studies with MAC-based sensitivities from 70% to 89% and specificities from 52% to 100%. Marek and Nakhosteen showed (1999, American Thoracic Society annual meeting) the results from two quantitative pulmonary studies showing (a) sensitivity of 89% and specificity of 92%, and (b) sensitivity of 91% and specificity of 100%.<br>
Clearly, Malignancy-Associated Changes (MACs) are potentially useful probes that result from the image-cytornetry marker detection technology. MAC-based features from DNA-stained nuclei can be used in conjunction with other molecular diagnostic probes to create optimized molecular diagnostic panels for the detection and differentiation of lung cancer and other disease states.<br>
Cytogenetics detects specific chromosome-based intracellular changes using, for example, in situ hybridization (ISH) technology. ISH technology can be based upon fluorescence (FISH), multi-color fluorescence (M-FISH), or light-absorption-based chromogenics imaging (CHRISH) technologies. The family of ISH technologies uses DNA or RNA probes to detect the presence of the complementary DNA sequence in cloned bacterial or cultured eukaryotic cells. FISH technology can, for example, be used for the detection of genetic abnormalities associated with certain cancers. Examples include probes for Trisomy 8 and HER-2 neu. Other technologies such as polymerase chain reactions (PCR) can be used to detect B-cell and T-cell gene rearrangements. Cytogenetics is a highly specific marker detection technology since it detects the causative or "trigger" molecular event producing a pathology condition. It may be less sensitive than the other marker detection technologies because fewer events may be present to detect. In situ hybridization (ISH) is a molecular diagnostic method uses gene-based analyses to detect abnormalities on the genetic level such as mutations, chromosome errors or genetic material inserted by a specific pathogen. For example, in situ hybridization may involve measuring the level of a specific mRNA by treating a sample of a patient's cells with labeled primers designed to hybridize to the specific mRNA, washing away unbound primers and measuring the signal of the label. Due to the uniqueness of gene sequences, a test involving the detection of gene<br>
sequences will likely have a high specificity, yielding very few false positives   However, because the amount of genetic material in a sample of cells may be very low only a very weak signal may be obtained. Therefore, in situ hybridization tests that do not employ pre-amplification techniques will likely have a poor specificity, yielding many false negatives. Proteomics depends upon cell charactenzation and differentiation resulting from the over-expression, under-expression, or presence/absence of unique or specific proteins m populations of normal or abnormal cell types   Proteomics includes not only the identification and quantification of proteins, but also the determination of their localization, modifications, interactions, chemical activities, and cellular/extracellular funcnons. Immunochemistry (lmmunocytochemistry in cells and lmmunohistochemistry (THC) in tissues) is the technology used, either qualitatively or quantitanvely (QIHC) to stain antigens (i.e , proteomes) using antibodies   Immunostaming procedures use a dye as the detection indicator. Examples of IHC applications include analyses for ER (estrogen receptor), PR progesterone receptor), p53 tumor suppressor genes, and EGRF prognostic markers. Proteomics is typically a more sensitive marker detection technology than cytogenetics because there are often orders of magnitude more protem molecules to detect using proteomics than there are cytogenehc mutations or gene-sequence alterations to detect using cytogenetics   However, proteomics may have a poorer specificity than the cytogenetic marker detecnon technology since mulnple pathologies may result m similar changes m protem over-expression or under-expression.Immunochemistry involves histological or cytological localization of lmmunoreactive substances in tissue sections or cell preparations, respectively, often utilizing labeled antibodies as probe reagents. Immunocnemistry can be used to measure the concentration of a disease marker (specific protein) m a sample of cells by treating the cells with an agent such as a labeled antibody (probe) that is specific for an epitope on the disease marker, then washing away unbound antibodies and measuring the signal of the label. Immunochemistry is based on the property that cancer cells possess different levels of certain disease markers than do healthy cells. The concentration of a disease marker in a cancer cell is generally large enough to produce a large signal  Therefore, tests that rely on immunochemistry will likely have a high sensitivity, yielding few false negatives. However, because other factors in addition to the disease state may cause the concentration of a disease marker to become raised or lowered, tests that rely on immunochemical analysis of a specific disease marker will likely have poor specificity, yielding a high rate of false positives<br>
The present invention provides a noninvasive disease state detection and discrimination method with both high sensitivity and high specificity. The method involves contacting a cytological sample suspected of containing diseased cells with a panel of probes comprising a plurality of agents, each of which quantitatively binds to a specific disease marker, and detecting and analyzing the partem of binding of the probe agents. The present invention also provides methods of constructing and validating a panel of probes for detecting a specific disease (or group of diseases) and discriminating among its various disease states. Illustrative panels for detecting lung cancer and discriminating among different types of lung cancer are also provided.<br>
A human disease results from the failure of the human organism's adaptive mechanisms to neutralize external or internal insults which result in abnormal structures or functions within the body's cells, tissues, organs or systems. Diseases can be grouped by shared mechanisms of causation as illustrated below, in Table 1.<br>
Table 1:<br>
(Table Removed)<br>
Disease states are either caused by or result in abnormal changes (i.e., pathological conditions) at a subcellular, cellular, tissue, organ, or human anatomic or physiological system level. Many disease states (e.g., lung cancer) are charactenzed by abnormal changes at a subcellular or cellular level. Specimens (e.g., cervical PAP smears, voided urine, blood, sputum, colonic washings) can be collected from patients with suspected disease states to diagnose those patients for the presence and type of the disease state. Molecular pathology is the discipline that attempts to identify and diagnostically exploit the molecular changes associated with these cell-based diseases.<br>
Lung cancer is an illustrative example of a disease state in which screening of high-risk populations and at-risk individuals can be performed using diagnostic tests (e.g., molecular diagnostic panel assays) to detect the presence of the disease state . Also, for patients in which lung cancer or other disease states have been detected by these means, related diagnostic tests can be employed to differentiate the specific disease state from related or co-occurring disease states. For example, in this lung cancer illustration, additional molecular diagnostic panel assays may indicate the probabilities that the patient's disease<br>
state is consistent with one of the following types of lung cancer: (a) squamous cell carcinoma of the lung, (b) adenocarcinoma of the lung, (c) large cell carcinoma of the lung, (d) small cell carcinoma of the lung, or (e) mesothelioma. Early detection and differentiatioii of cell-based disease states is a hypothesized means to improve patient outcomes.<br>
Cancer is a neoplastic disease the natural course of which is fatal. Cancer cells, unlike benign tumor cells, exhibit the properties of invasion and metastasis and are highly anaplastic. Cancer includes the two broad categories of carcinoma and sarcoma, but in normal usage it is often used synonymously with carcinoma. According to the World Health Organization (WHO), cancer affects more than 10 million people each year and is responsible for in excess of 6.2 million deaths.<br>
Cancer is, in reality, a heterogeneous collection of diseases that can occur in virtually any part of the body. As a result, different treatments are not equally effective in all cancers or even among the stages of a specific type of cancer. Advances in diagnostics (e.g., mammography, cervical cytology, and serum PSA testing) have, in some cases, allowed for the detection of early-stage cancer when there are a greater number of treatment options, and therapies tend to be more effective. In cases where a solid tumor is small and localized, surgery alone may be sufficient to produce a cure. However, in cases where the tumor has spread, surgery may provide, at best, only limited benefits. In such cases the addition of chemotherapy and/or radiation therapy may be used to treat metastatic disease. While somewhat effective in prolonging life, treatment of patients with metastatic disease rarely produces a cure. Even through there may be an initial response, with rime the disease progresses and the patient ultimately dies from its effects and/or from the toxic effects of the treatments.<br>
While not proven, it is generally accepted that early detection and treatment will reduce the morbidity, mortality and cost of cancer. Early detection will, in many cases, permit treatment to be initiated prior to metastasis. Furthermore, because there are a greater number of treatment options, there is a higher probability of achieving a cure or significant improvement in long-term survival.<br>
Developing a test that can be used to screen an "at-risk" population has long been a goal of health practitioners. While there have been some successes such as mammography for breast cancer, PSA testing for prostate cancer, and the PAP smear for cervical cancer, in most cases cancer is detected at a relatively late stage where the patient is symptomatic and the disease is almost always fatal. For most cancers, no test or combination<br>
of tests has exhibited the necessary sensitivity and specificity to permit cost-effective identification of patients with early stage disease.<br>
For a cancer screening program to be successful and gain acceptance by patients, physicians, and third party payers, the test must have implied benefit (changes the outcome), be widely available and be able to be carried out readily within the framework of general healthcare. The test should be relatively noninvasive, leading to adequate compliance, have high sensitivity, and reasonable specificity and predictive value. In addition, the test must be available at relatively low cost.<br>
For patients who are suspected of having cancer, the diagnosis must be confirmed and the tumor properly staged cytologically and clinically in order for physicians to undertake appropriate therapeutic intervention. Some tests currently being used in the diagnosis and staging of cancer, however, either lack sufficient sensitivity or specificity, are too invasive, or are too costly to justify their use as a population-based screening test. Shown below in Tables 2 and 3, for example, are estimates of sensitivity and specificity of lung cancer diagnostics and estimated costs for diagnostic tests used to detect lung cancer.<br>
Table 2:<br>
ESTIMATES OF SENSITIVITY AND SPECIFICITY OF LUNG CANCER DIAGNOSTICS [1]<br>
(Table Removed)<br>
*Dependent upon the stage of the disease at the time of diagnosis<br>
Table 3:<br>
ESTIMATED COSTS FOR DIAGNOSTIC TESTS USED IN LUNG CANCER [1]<br>
(Table Removed)The chest radiograph (X-ray) is often used to detect and localize cancer lesions due to its reasonable sensitivity, high specificity and low cost. However, small lesions are often difficult to detect and although larger tumors are relatively easy to visualize on a chest film, at the time of detection most have already metastasized. Thus, chest X-rays lack the necessary sensitivity for use as an early detection method.<br>
Computed tomography (CT) is useful in the confirmation and characterization of pulmonary nodules and allows the detection of subtle abnormalities that are often missed on a standard chest X-ray [2]. CT, and Spiral CT methods in particular, remains the test of choice for patients who present with a prior mahgnant sputum cytology result or vocal chord paralysis. CT, with its improved sensitivity over the conventional chest film, has become the primary tool for imaging the central airway [3]. While capable of exarnining large areas, CT is subject to artifacts from cardiac and respiratory motion although improved resolution can be achieved through the use of iodinated contrast material.<br>
Spiral CT is a more rapid and sensitive form of CT that has the potential to detect early cancer lesions more reliably than either conventional CT or X-ray. Spiral CT appears to have greatly improved sensitivity in diagnosing early disease. However, the test has relatively low specificity with a 20% false positive rate [4]. Spiral CT is also less sensitive in detecting the central lesions that represent one-third of all lung canceTs. Furthermore, while the cost of the initial test is relatively low (S300), the cost of follow-up can be high. Cytology using molecular diagnostic panel assays offers significant promise as an adjunctive test with Spiral CT to improve the specificity of Spiral CT testing by minimizing false positive results through the evaluation of fine needle aspirations (FNAs) or biopsies (FNBs) from Spiral CT-suspicious pulmonary nodules.<br>
Fluorescence bronchoscopy provides increased sensitivity over conventional white light bronchoscopy, significantly improving the detection of small lesions within the central airway [5]. However, fluorescence bronchoscopy is unable to detect peripheral lesions, it takes a long time for bronchoscopists to examine a patient's airways, and it is an expensive procedure. Additionally, the procedure is moderately invasive, creating an insurmountable barrier to its use as a population-based screening test.<br>
Positron Emission Tomography (PET) is a highly sensitive test that utilizes radioactive glucose to identify the presence of cancer cells within the lung [6-8]. The cost of establishing a testing facility is high and there is the need for a cyclotron on site or nearby.<br>
This, coupled with the high cost of the test, has limited the use of PET scans to staging lung cancer patients rather than for early detection of the disease.<br>
Although used for some time as a means of screening for lung cancer, sputum cytology has enjoyed only limited success due to its low sensitivity and its failure to reduce disease-specific mortality. In conventional sputum cytology, the pathologist uses characteristic changes in cellular morphology to identify malignant cells and make a diagnosis of cancer. Today only 15% of patients who are 'at-risk" or who are suspected of having lung cancer undergo sputum cytology testing, and less than 5% undergo multiple evaluations [9]. A number of factors including tumor size, location, degree of differentiation, cell clumping, inefficiency of clearing mechanisms to release cells and sputum to the external environment, and the poor stability of cells within the sputum contribute to the overall poor performance of the test.<br>
Cancer diagnostics has traditionally relied upon the detection of single molecular markers. Unfortunately, cancer is a disease state in which single markers have typically failed to detect or differentiate many forms of the disease. Thus, probes that recognize only a single marker have been shown to be largely ineffective. Exhaustive searches for "magic bullet" diagnostic tests have been underway for many decades though no universal successful magic bullet probes have been found to date.<br>
A major premise of this invention is that cell-based cancer diagnostics and the screening, diagnosis for, and therapeutic monitoring of other disease states will be significantly improved over the state-of-the-art that uses single marker/probe analyses rather than kits of multiple, simulaneously labeled probes. This multiplexed analytical approach is particularly well suited for cancer diagnostics since cancer is not a single disease. Furthermore, this multi-factorial "panel" approach is consistent with the heterogeneous nature of cancer, both cytologically and clinically.<br>
Key to the successful implementation of a panel approach to cell-based diagnostic tests is the design and development of optimized panels of probes that can chemically recognize the pattern of markers that characterizes and distinguishes a variety of disease states. This patent application describes an efficient and unique methodology to design and develop such novel and optimized panels.<br>
Improved methods for specimen collection (e.g., point-of-care mixers for sputum cytology) and preparation (e.g., new cytology preservation and transportation fluids, and liquid-based cytology preparation instruments) are under development and becoming commercially available. In conjunction with existing and these emerging methods, a<br>
successful implementation of this molecular diagnostics cell-based panel assay will lead to (a) characterization of the molecular profile of malignant tumors and other disease states, (b) improved methods for early cancer and other disease state detection and differentiation, and (c) opportunities for improved clinical diagnoses, prognoses, customized patient treatments, and therapeutic monitoring.<br>
SUMMARY OF THE INVENTION<br>
The present invention is directed to a panel for detecting a generic disease state or discriminating between specific disease states using cell-based diagnosis. The panel comprises a plurality of probes each of which specifically binds to a marker associated with a generic or specific disease state, wherein the pattern of binding of the component probes of the panel to cells in a cytology specimen is diagnostic of the presence or specific nature of said disease state. The present invention is also directed to a method of forming a panel for detecting a disease state or discriminating between disease states in a patient using cell-based diagnosis. The method involves determining the sensitivity and specificity of binding of probes each of which specifically binds to a member of a library of markers associated with a disease state and selecting a limited plurality of said probes whose pattern of binding is diagnostic for the presence or specific nature of said disease state. The present method is also directed to a method of detecting a disease or discriminating between disease states comprising . The method involves contacting a cytological sample suspected of containing abnormal cells characteristic of a disease state with a panel according to claim 1 and detecting a pattern of binding of said probes that is diagnostic for the presence or specific nature of said disease state.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1.	Molecular markers that are preferable markers to be included in<br>
a panel for identifying different histologic types of lung cancer. The column labeled "%" indicates the percentage of tumor specimens that express a particular marker.<br>
Figure 2.	Potential ways in which different markers may be used to<br>
discriminate between specific types of lung cancer. SQ indicates squamous cell carcinoma, AD indicates adenocarcinoma, LC indicates large cell carcinoma, SC indicates small cell carcinoma and ME indicates mesothelioma. The numbers appearing in each cell represent frequency of marker change in one cell type versus another. To be included in the table, the<br>
ratio must be greater than 2.0 or less than 0.5. A number larger than 100 generally indicates that the second marker is not expressed. In such cases the denominator was set ai 0.1 for the purpose of the analysis. Finally, empty cells represent either no difference in expression or the absence of expression data.<br>
Figure 3.	Comparisons between H-scores for probes 7 and 15 in control<br>
tissue and m cancerous tissue. The x-axis shows the H-scores while the y-axis shows the percent of cases.<br>
Figure 4.	Correlation matrix, in which correlation measures the amount<br>
of linear association between a pair of variables. All markers in this matrix with a correlation number of 50% or higher are considered correlate markers.<br>
Figure 5.	Detection panel compositions, pair-wise discrimination panel<br>
compositions and joint discrimination panel compositions. Panel compositions using decision tree analysis, stepwise LR and stepwise LD are shown.<br>
Figure 6.	Detection panel compositions wherein probe 7 was not<br>
included as a probe. Panel compositions using decision tree analysis, stepwise LR and stepwise LD are shown.<br>
Figure 7.	Detection panel compositions using only commercially<br>
preferred probes. Panel compositions using decision tree analysis, stepwise LR and stepwise LD are shown.<br>
DETAILED DESCRIPTION OF THE INVENTION 1.   Introduction<br>
The present invention provides a noninvasive disease state detection and discrimination method with high sensitivity and specificity. The method involves contacting a cytological sample suspected of containing diseased cells with a panel comprising a plurality of agents, each of which quantitatively binds to a disease marker, and detecting a partem of binding of the agents. This pattern includes the localization and density/concentration of binding of the component probes of the panel. The present invention also provides methods of making a panel for detecting a disease and also for discriminating between disease states as well as panels for detecting lung cancer in early stages and discriminating between different types of lung cancer. Panel tests have been used in medicine. For example, panels are used in blood serum analysis. However, because a cytology analysis involves imaging and localization of specific markers within individual cells and tissues, prior to the present invention it was not apparent that the panel approach<br>
would be effective for cytology samples. Additionally, it was not apparent which, if any statistical analyses could be applied to design and develop an optimized cell-based diagnostic panel of probes.<br>
One of the few examples of a cytology-based screening program is the PAP Smear, which screens for cervical cancer. For over 50 years this method has been practiced and has greatly contributed to the fact that today, almost no woman who has regular PAP smears dies of cervical cancer. There are drawbacks, however, to the PAP smear screening program. For example, PAP smears are labor intensive and are not universally accessible. The present molecular diagnostic cell-based screening method utilizing probe panels does not suffer from these drawbacks. The method may be fully automated and thereby made less expensive, increasing access to this type of testing.<br>
The present invention provides a method, having both high specificity and high sensitivity, for detecting a disease state and for discnminatmg between disease states. The invention is applicable to any cell-based disease state, such as cancer and infectious diseases.<br>
The panel is diagnostic of the presence or specific nature of the disease state. The present invention overcomes the limitations and drawbacks of known disease state detection methods by enabling quick, accurate, relatively noninvasive and easy detection and discrimination of diseased cells in a cytological sample while keeping costs low.<br>
A feature of the inventive method for making a panel of the present invention is the rapidity with which the panel may be developed.<br>
There are several benefits to using a panel of agents in a memod for detecting a disease state, and for discriminating between types of disease states. One benefit is that a panel of agents has sufficient redundancy to permit detection and characterization of disease states thereby increasing the sensitivity and specificity of the test. Given the heterogeneous nature of many disease states, no single agent is capable of identifying the vast majority of cases.<br>
An additional benefit to using a panel is that use of a panel permits discriminanon between the various types of a disease state based on specific patterns (probe localization and density/concentration) of expression. As the various types of a disease may exhibit dramatic differences in their rate of progression, response to therapy, and lethality, knowledge of me specific type can help physicians choose the optimal therapeutic approach.<br>
2.    The Panel<br>
The panel of the present invention comprises a plurality of agents, each of which quantitatively binds to a disease marker, wherein the pattern (localization and density/concentration) of binding of the component agents of the panel is diagnostic of the presence or specific nature of a disease state. Therefore, the panel may be a detection panel or a discrimination panel. A detection panel detects whether a generic disease state is present in a sample of cells, while a discrimination panel discriminates among different specific disease states in a sample of cells known to be affected by a disease state which comprises different types of diseases. The difference between a detection panel and a discrimination panel lies in the specific agents that the panels comprise. A detection panel comprises agents having a pattern of binding that is diagnostic of the presence of a disease state, while a discrimination panel comprises agents having a pattern of binding that allows for detennining the specific nature (i.e., each type) of the disease state.<br>
A panel, by definition, contains more than one member. There are several reasons why it is beneficial to use a panel of markers rather than just one marker alone to detect a generic disease state or to discriminate among specific disease states. One reason is the unlikely existence of a probe for one single marker, that is present in all diseased cells yet not present in healthy cells, whose behavior can be measured with a high specificity and sensitivity to yeild an accurate test result. If such a single probe existed for detection of a particular disease with high sensitivity and specificity, it would already have been utilized for clinical testing. Rather, it is the directed selection of panel tests, each consisting of multiple probes, that together can provide the range of detection capability to ensure clinically adequate testing.<br>
If one nevertheless chooses to construct a panel test comprising one or a very few probes, then the failure of any single marker/probe combination to perform its labeling function for any reason (for example, diminished reactivity of the specimen cells due to biological variability; inherent variability between lots of probe reagents; a weak, outdated or defective processing reagent; improper processing time or conditions for that probe) could result in a catastrophic failure of the test to detect or discriminate the target disease. The inclusion of multiple, and even redundant probes in each pasel test greatly enhances the probability that a failure of any one probe will not cause a catastrophic failure of the test.<br>
A probe is any molecular structure or substructure that binds to a disease marker. The term "agent" as used herein, may also refer to a molecular structure or<br>
substructure that binds to a disease marker. Molecular probes are horning devices used by biologists and clinicians to detect and locate markers indicative of the specific disease states. For example, antibodies may be produced that bind specifically to a protein previously identified as a marker for small cell lung cancer. This antibody probe can then be used to localize the target protein marker in cells and tissues of patients suspected of having the disease by using appropriate immunochemical protocols and incubations. If the antibody probe binds to its target marker in a stoichiometric (i.e., quantitative) fashion and is labeled with a chromogenic or colored "tag", then localization and quantitation of the probe and, indirectly, its target marker may be accomplished using an optical microscope and image cytometry technology.<br>
The present invention contemplates detecting changes in molecular marker expression at the DNA, RNA or protein level using any of a number of methods available to an ordinary skilled artisan.   Exemplary probes may be a polyclonal or monoclonal antibody or fragment thereof or a nucleic acid sequences that is complementary to the nucleic acid sequence encoding a molecular marker in the panel. A probe may also be a stain, such as a DNA stain. Many of the antibodies used in the present invention are specific to a variety of cell surface or intracellular antigens as marker substances. The antibodies may be synthesized using techniques generally known to those of skill in the art. For example, after the initial raising of antibodies to the marker, the antibodies can be sequenced and subsequently prepared by recombinant techniques. Alternatively, antibodies may be purchased.<br>
In embodiments of the present invention, the probe contains a label. A probe containing a label is often referred to herein' as a "labeled probe". The label may be any substance that can be attached to a probe so that when the probe binds to the marker a signal is emitted or the labeled probe can be detected by a human observer or an analytical instrument. This label may also be referred to as a "tag". The label may be visualized using reader instrumentation. The term "reader instrumentation" refers to the analytical equipment used to detect a probe. Labels envisioned by the present invention are any labels that emit a signal and allow for identification of a component in a sample. Preferred labels include radioactive, fluorogenic, chromogenic or enzymatic moieties. Therefore, possible methods of detection include, but are not limited to, irnmunocytochemistry, imrnunohistochemistry, in situ hybridization, fluorescent in situ hybridization, flow cytometry and image cytometry. The signal generated by the labeled probe is of sufficient intensity to permit detection by a medical practitioner.<br>
A '"marker", "disease marker" or "molecular marker" is any molecular structure or substructure that is correlated with a disease state or pathogen. The term "antigen" may be used interchangeably with '"marker". Broadly defined, a marker is a biological indicator that may be deliberately used by an observer or instrument to reveal, detect, or measure the presence or frequency and/or amount of a specific condition, event or substance. For example, a specific and unique sequence of nucleotide bases may be used as a genetic marker to track patterns of genetic inheritance among individuals and through families. Similarly, molecular markers are specific molecules, such as proteins or protein fragments, whose presence within a cell or tissue indicates a particular disease state. For example, proliferating cancer cells may express novel cell-surface proteins not found on normal cells of the same type, or may over-express specific secretory proteins whose increased or decreased abundance (e.g., overexpression or underexpression. respectively) can serve as markers for a particular disease state.<br>
Suitable markers for cytology panels are substances that are localized in or on the nucleus, cytoplasm or cell membrane. Markers may also be localized in organelles located in any of these locations in the cell. Exemplary markers localized in the nucleus include but are not limited to retinoblastoma gene product (Rb), Cyclin A, nucleoside diphosphate kinase/nm23, telomerase, Ki-67, Cyclin Dl, proliferating cell nuclear antigen (PCNA), pi20 (proliferation-associated nucleolar antigen) and thyroid transcription factor 1 (TTF-1). Exemplary markers localized in the cytoplasm include but are not limited to VEGF, surfactant apoprotein A (SP-A), nucleoside nm23, melanoma antigen-1 (MAGE-1), Mucin 1, surfactant apoprotein B (SP-B), ER related protein p29 and melanoma antigen-3 (MAGE-3). Exemplary markers localized in the cell membrane include but are not limited to VEGF, thrombomodulin, CD44v6, E-Cadherin, Mucin 1, human epithelial related antigen (HERA), fibroblast growth factor (FGF), heptocyte growth factor receptor (C-MET), BCL-2, N-Cadherin, epidermal growth factor receptor (EGFR) and glucose transporter-3 (GLUT-3). An example of a marker located in an organelle of the cytoplasm is BCL-2, located (in part) in the mitochondrial membrane. An example of a marker located in an organelle of the nucleus is pi20 (proliferating-associated nucleolar antigen), located in the nucleoli.<br>
Preferred are markers where changes in expression: occur early in disease progression, are exhibited by a majority of diseased cells, allow for detection of in excess of 75% of a given disease type, most preferably in excess of 90% of a given disease type and/or allow for the discrimination between the nature of different types of a disease state.<br>
It is noted that the inventive panel may be referred ro as a panel of probes or a panel of markers, since the probes bind to the markers. Therefore, the panel may comprise a number of markers or it may comprise a number of probes that bind to specific markers. For the sake of consistency, the present panel is referred to as a panel of probes; however, it could also be referred to as a panel of markers.<br>
Markers can also include features such as malignancy- associated changes (MACs) in the cell nucleus or features related to the patient's family history of cancer. Malignancy-associated changes, or MACs, are typically sub-visual changes that occur in normal-appearing cells located in the vicinity of cancer cells. These exceedingly subtle changes in the cell nucleus may result biologically from changes in the nuclear matrix and the chromatin distribution partem. They cannot be appreciated even by trained observers through the visual observation of individual cells, but may be determined from statistical analysis of cell populations using highly automated, computerized high-speed image cytometry. Techniques for detection of MACs are well known to those of skill in the art and are described in more detail in: Gruner, O.C. BntJ. Surg. 3 506-522 (1916); Neiburgs, H.E. ex al., Transaction, 7th Annual Mtg. Inter. Soc. Cytol. Council 137-144 (1959); Klawe, H. Acta. Cytol. 18 30-33 (1974); Wied, G.L., et al, Analty. Quant. Cytol. 2 257-263 (1980); and Burger, G., et al.,Analyt. Quant. Cytol. 3 261-271 (1981).<br>
The present invention encompasses any marker that is correlated with a disease state. The individual markers themselves are mere tools of the present invention. Therefore, the invention is not limited to specific markers. One way to classify markers is by their functional relationship to other molecules. As used herein, a "functionally related" marker is a component of the same biological process or pathway as the marker in question and would be known by a person of skill in the art to be abnormally expressed together with the marker in question. For example, many markers are associated with a cell proliferation pathway, such as fibrobast growth factor (FGF), (vascular endothelial growth factor) VEGF, CyclinA and Cyclin Dl. Other markers are glucose transporters, such as Glut-1 and Glut-3.<br>
A person of ordinary skill in the art is well equipped to determine a functionally related marker and may research various markers or perform experiments in which the functional behavior of a marker is determined. By way of non-limiting example, a marker may be classified as a molecule involved in angiogenesis, a transmembrane glycoprotein, a cell surface glycoprotein, a pulmonary surfactant protein, a nuclear DNA-binding phosphoprotein, a transmembrane Ca"+ dependent cell adhesion molecule, a regulatory subunit of the cyclin-dependent kinases (CDK's), a nucleoside diphosphate kinase,<br>
a ribonucleoprotein enzyme, a nuclear protein that is expressed in proliferating normal and neoplastic cells, a cofactor for DNA polymerase delta, a gene that is silent in normal tissues yet when it is expressed in malignant neoplasms is recognized by autologous, tumor-directed and specific cytotoxic T cells (CTL's), a glycosylated secretory protein, the gastrointestinal tract or genitourinary tract, a hydrophobic protein of a pulmonary surfactant, a transmembrane glycoprotein, a molecule involved in proliferation, differentiation and angiogenesis, a proto-oncogene, a homeodomain transcription factor, a mitochondrial membrane protein, a molecule found in nucleoli of a rapidly proliferating cell, a glucose transporter, or an estrogen-related heat shock protein.<br>
Classes of biomarkers and probes include, but are not limited to: (a) morphologic biomarkers, including DNA ploidy, MACs and premalignant lesions; (b) genetic biomarkers including DNA adducts, DNA mutations and apoptotic indices; (c) cell cycle biomarkers including cellular proliferation, differentiation, regulatory molecules and apoptosis markers, and; (d) molecular and biochemical biomarkers including oncogenes, tumor suppressor genes, tumor antigens, growth factors and receptors, enzymes, proteins, prostaglandin levels and adhesion molecules.<br>
A "disease state" may be any cell-based disease. In some embodiments the disease state is cancer. In other embodiments, the disease state is an infectious disease. The cancer may be any cancer, including, but not limited to epithelial cell-based cancers from the pulmonary, urinary, gastrointestinal, and genital tracts; solid and/or secretory tumor-based cancers, such as sarcomas, breast cancer, cancer of the pancreas, cancer of the liver, cancer of the kidneys, cancer of the thyroid, and cancer of the prostate; and blood-based cancers, such as leukemias and lymphomas. Exemplary cancers which may be detected by the present invention are lung, bladder, gastrointestinal, cervical, breast or prostate cancer. Exemplary infectious diseases which may be detected are cell-based sieases in which the infectious organism is a virus, bacteria, protozoan, parasite, or fungus. The infectious disease, for example, may be HIV, hepatitis, influenza, meningitis, mononucleosis, tuberculosis and sexually transmitted diseases (STDs), such as chlamydia, trichomonas, gonorrhea, herpes and syphilis.<br>
As used herein, the term "generic disease state" refers to a disease which comprises several types of specific diseases, such as lung cancer, sexually transmitted diseases and immune-based diseases. Specific disease states are also referred to as histologic types of diseases. For example, the term "lung cancer" comprises several specific diseases, among which are squamous cell carcinoma, adenocarcinoma, large cell carcinoma, small cell<br>
lung cancer and mesothelioma. The term 'sexually transmitted diseases" comprises several specific diseases, among which are Gonorrhea, Human Papilloma Virus (HPV), herpes and Syphilis. The term "'immune-based diseases" comprises several specific diseases, such as systemic lupus erythematosus (Lupus), rheumatoid arthritis and pernicious anemia.<br>
As used herein, the term "high-risk population" refers to a group of individuals who are exposed to disease causing agents, e.g., carcinogens, either at home or in the workplace (i.e., a "high risk population" for lung cancer might be exposed to smoking, passive smoking and occupational exposure). Individuals in a "high-risk population" may also have a genetic predisposition.<br>
The term "at-risk" refers to individuals who are asymptotic but, because of a family history or significant exposure are at a significant risk of developing a disease state (i.e., an individual at risk for lung cancer with a &gt; 30 pack-year history of smoking; "pack-year" is a measurement unit computed by multiplying the number of packs smoked per day, times the number of years for this exposure).<br>
Cancer is a disease in which cells divide without control due to, for example, altered gene expression. In the methods and panels of the present invention, the cancer may be any malignant growth in any organ. For example, the cancer may be lung, bladder, gastrointestinal, cervical, breast or prostate cancer. Each cancer may comprise a collection of diseases or histological types of cancer. The term "histologic type" refers to cancers of different histology. Depending on the cancer there can be one or several histologic types. For example, lung cancer includes, but is not limited to, squamous cell carcinoma, adenocarcinoma, large cell carcinoma, small cell carcinoma and mesothelioma. Knowledge of the histologic type of cancer affecting a patient is very useful because it helps the medical practitioner to localize and characterize the disease and to determine the optimal treatment strategy.<br>
Infectious diseases include cell-based diseases in which the infectious organism is a virus, bacteria, protozoan, parasite or fungus.<br>
Exemplary detection and discrimination panels are panels that detect lung cancer, a general disease state, and panels that discriminate a single lung cancer type, specific disease state, against all other types of lung cancer and false positives. False positives can include metastatic cancer of a different type, such as metastasized liver, kidney or pancreatic cancer.<br>
3.   Methods of Making a Panel<br>
The method of making a panel for detecting a generic disease state or discriminating between specific disease states in a patient involves determining the sensitivity and specificity of binding of probes to a library of markers associated with a generic or specific disease state and selecting a plurality of said probes whose pattern of binding (localization and density/concentration) is diagnostic of the presence or specific nature of the disease state. In some embodiments, optional prelirninary pruning and preparation steps are performed. The method of making a panel of the present invention involves analyzing the partem of binding of probes to markers in known histologic pathology samples, i.e. gold standards. The classifier designed on the gold standard data can then be used to design a classifier for cytometry, especially automated cytometry. Therefore, the set of marker probes selected from the pathology analysis is used to prepare a new training data set taken from a cytology sample, such as sputum, fine needle aspirations, urine, etc. Cells shed from the specified lesions will stain in a similar fashion to the gold standards. The method described here eliminates the experimental error in selecting the best features set because the integrity of the diagnosis based on gold standard histologic pathology samples is high. Although it is, in principle, possible to use cytology samples to produce a panel, this is less desireable because cytology samples contain debris, there may be deterioration of the cells in a cytology sample, and the pathology diagnosis may be difficult to confirm clinically.<br>
A library of markers is a group of markers. The library can comprise any number of markers. However, in some embodiments the number of markers in the library is limited by technical and/or commercial practicalities, such as specimen size. For example, in some embodiments, each specimen is tested against all of the markers in the panel. Therefore, the number of markers must not be larger than the number of samples into which the specimen may be divided. Another technical practicality is time. Typically, the library contains less than 60 markers. Preferably, the library contains less than 50 markers. More preferably, the library contains less than 40 markers. Most preferably the library contains 10-30 markers. It is preferable that the library of potential panel members contain more than 10 markers so that there is opportunity to optimize the performance of the panel. As used herein, the term "about" means plus or minus 3 markers.   <br>
In some embodiments, a library is obtained by consulting sources which contain information about various markers and correlations between the markers and generic/specific disease states. Exemplary sources include experimental results, theoretical<br>
or predicted analyses and literary sources, such as journals, books, catalogues and web sites. These various sources may use histology or cytology and may reiy on cytogenetics, such as in situ hybridization; proteomics, such as immunohistochernistry; cytometry, such as MACs or DNA ploidy ; and/or cytopathology, such as morphology. The markers may be localized anywhere in or on a cell. For example, the markers may be locahzed in or on the nucleus, the cytoplasm or the cell membrane. The marker may also be localized in an organelle within any of the aforementioned localizations.<br>
In some embodiments, the library may be of an unsuitable size. Therefore, one or more pruning steps may be required prior to initiating the basic method for making a panel. The pruning step may involve one or several successive pruning steps. One pnining step may involve, for example, setting an arbitrary threshold for sensitivity and/or specificity. Therefore, any marker whose experimental or predicted sensitivity and/or specificity falls below the threshold may be removed from the library.  Other exemplary pruning steps, which may be performed alone or in sequence with other pruning steps, may rely on detection technology requirements, access constraints and irreproducibility of reported results. With respect to detection technology requirements, it is possible that the machinery required to detect a particular marker is unavailable. With respect to access constraints, it is possible that licensing restrictions make it difficult or impossible to obtain a probe that binds to a particular marker. In some embodiments, a due diligence study is performed on each marker.<br>
In some embodiments, prior to beginning the basic method for making a panel, it may be necessary to perform preparation steps. Exemplary preparation steps include optimizing the protocols for objective quantitative detection of the markers in the library and collecting histology specimens. Optimization of the protocols for objective quantitative detection of the markers is within the skill of an ordinary artisan. For example, the necessary reagents and supplies must be obtained, such as buffers, reagents, software and equipment. It is possible that the concentration of reagents may need to be adjusted. For example, if nonspecific binding is observed, a person of ordinary skill in the art may dilute the concentration of the probe solution.<br>
In some embodiments, the histology specimens are Gold Standards. The term "Gold Standard" is known by a person of ordinary skill in the art to mean that the histology and clinical diagnosis of the specimen is known. The gold standards are often referred to as a 'training" data set. The gold standards comprise a set of measurements, or reliable estimates, of all the features that may contribute to the discriminating process. Such features are collected from samples collected from a representative number of patients with known<br>
disease states. The standard samples can be cytology samples but this is less desireable for panel selection.<br>
The histology samples may be obtained by any technique known to those of skill in the art, for example biopsy. In some embodiments, it is necessary that the size of the specimen per patient be large enough so that enough tissue sections can be obtained to test each marker in the library.<br>
In some embodiments, specimens are obtained from multiple patients diagnosed with each specific disease state. One specimen per patient may be obtained, or multiple specimens per patient may be obtained. In embodiments in which multiple specimens are obtained from individual patients, the expertise of the surgeon is rehed upon to establish that each specimen obtained from a single patient is similar to the other specimens obtained from that patient. Specimens are also obtained from a control group of patients. The control group of patients may be healthy patients or patients that are not suffering from the generic or specific disease state that is being tested.<br>
The first step of the basic method is determining the sensitivity and specificity of binding of probes to a library of markers associated with the desired disease state. In this step, a probe that is specific for each marker in the library is applied to a sample of the patients' specimens. Therefore, in some embodiments, if there are, for example, 30 markers in the library, each patient's specimen will be divided into 30 samples and each sample will be treated with a probe that is specific for one of the 30 markers. The probe contains a label that may be visualized. Therefore, the pattern and level of binding of the probe to the marker can be detected. The partem and level of binding may be detected either quantitatively, i.e., by an analytical instrument, or qualitatively, by a human, such as a pathologist.<br>
In some embodiments, an objective and/or quantitative scoring method is developed to detect the pattern and level of binding of the probe to the markers. The scoring method may be heuristically designed. Scoring methods are used to objectify a subjective interpretation, for example, by a pathologist. It is within the skill of an ordinary artisan to determine a suitable scoring method. In some embodiments, the scoring method may comprise categorizing features, such as the density of a marker probe stain as: none, weak, moderate, or intense. In another embodiment, these features may be measured with algorithms operating on microscope slide images. An exemplary scoring method is one in which the proportions and density are consolidated into a single "H Score" obtained by grading the intensity as: none = 0, weak = 1, moderate = 2, intense = 3, and the percentage cells as: 0-5% = 0, 6-25% = 1, 26-50% * 2, 51-75% = 3, &gt;75% = 4, and then multiplying the<br>
two grades together. For example, 50% weakly stained plus 50% moderately stained would score 6 = (1 x 2) + (2 x 2).    The "H score" honors the late Kenneth Hirsch, one of the present inventors.<br>
An ordinary artisan is capable of addressing issues related to rrunimizing potential biases related to pathologists and samples. For example, randomizing may be used to minimize the chance of having a systematic error. Blinding may be used to eliminate experimental biases by the people conducting the experiments. For example, in some embodiments, pathologist-to-pathologist variation may be minimized by conducting a double blind study. As used herein, the term "double blind study" is a well establish method for avoiding biases, where the data collection and data analysis are done independently. In other embodiments, sample-to-sample variation is minimized by randomizing the samples. For example, the samples are randomized before the pathologist analyzes them. There is also randomization involved in the experimental protocols. In some embodiments, each sample is analyzed by at least two pathologists. For each patient, a reliable assessment of the binding of the probe to the marker is obtained. In one embodiment, this diagnosis is made by qualified pathologists, using two pathologists per patient, to check for reliability.<br>
A sufficient number of samples should be collected to produce reliable designs and reliable statistical performance estimates. It is within the skill of a normal artisan to determine how many samples are sufficient to produce reliable designs and reliable statistical performance estimates. Most standard classifier design packages have methods for determining the reliability of the performance estimates and the sample size should be progressively increased until reliable estimates are achieved. For example, sufficient estimates to produce reliable designs may be achieved with 200 samples collected and 27 different features estimated from each sample.<br>
The second step is selecting a limited plurality of probes. The selecting step may employ statistical analysis and/or pattern recognition techniques. In order to perform the selecting step, the data may be consolidated into a database. In some embodiments, the probes may be numbered to render their method of action as unseen during the analysis of their effectiveness and further minimize biases. Rigorous statistical techniques are used because of the large amount of data that is generated by this method. Any statistical method may be used and an ordinary skilled statistician will be able to identify which and how many methods are appropriate.<br>
Any number of statistical analysis and/or pattern recognition methods may be employed. Since the structure of the data is initially unknown, and since different classifier<br>
design methods perform better for different structures, it is preferred to use at least two design methods on the data. In some embodiments, three different methodologies may be used. One of ordinary skill in the art of statistical analysis and/or partem recognition of data sets would recognize from characteristics of the data set structures that certain statistical methods would be more likely to yield an efficient result than others, where efficient in this case means achieving a certain level of sensitivity and specificity with a desired number of probes. A person of ordinary skill in the art would know that the efficiency of the statistical analysis and/or method is data dependent.<br>
Exemplary statistical analysis and/or pattern recognition methods are described below:<br>
a)	A decision tree method, known as C4.5. C4.5 is public domain software available via ftp from http://www.cse.unsw.edu.au/~quinlan/. This is well suited to data that can be best classified by sequentially applying a decision threshold to specific features in turn. This works best with uncorrelated data; it also copes with data with similar means provided the variances differ. The C4.5 package was used to provide the examples shown herein.<br>
b)	Linear Discrirninant Analysis. This involves finding weighted combinations of the features that give the best separation of the classes. These methods work well with correlated data, but not in data with similar means and different variances. Several statistical packages were used (SPSS, SAS and R), depending on the performance estimates and graphical outputs required. Fisher's linear discriminant function was used to obtain the classifier that minimized the error rate. A canonical discriminant function was used to compute receiver operating characteristic (ROC) curves showing the trade-off between sensitivity and selectivity as the decision threshold is changed.<br>
c)	Logistic Regression. This is a non-linear transformation of the linear regression model: the dependent variable is replaced by a log odds ratio (logit). Linear regression, like discrirninant analysis, belongs to a class of statistical methods founded on linear models. Such models are based on linear relationships between the explanatory variables.<br>
With a sufficient number of samples it is possible, using the above techniques and software packages, to search for combinations of features giving good discrimination between the classes. Other exemplary statistical analysis and/or pattern recognition methods are the linear Discriminant Function Method in SPSS and Logistic Regression Method in R and SAS. SPSS is the full product name and is available from SPSS, Inc., located at SPSS,<br>
Inc. Headquarters, 233 S. Wacker Drive, 11th floor, Chicago, Illinois 60606 (www.spss.com). SAS is the full product name and is available from SAS Institute, Inc., 100 SAS Campus Drive, Cary, NC 27513-2414, USA (www.sas.com). R is the full product name and is available as Free Software under the terms of the Free Software Foundation's GNU (General Public License). http://www.r-project.org/.<br>
In some embodiments, a correlation matrix is obtained. Correlation measures the amount of linear association between a pair of variables. A correlation matrix is obtained by correlating the data obtained with one marker to data obtained with another marker. A threshold correlation number may be set, for example, 50% correlation. In this case, all markers with a correlation number of 50% or higher would be considered correlate markers.<br>
In some embodiments of the present invention, user supplied weighting factors may be used to obtain optimized panels. Weighting may be related to any factor. For example, certain markers may be weighted higher than others due to cost, commercial considerations, misclassifications or error rates, prevalence of a generic disease state in a geographic location, prevalence of a specific disease state in a geographic location, redundancy and availability of probes. Some factors related to cost that may encourage a user to weight certain markers higher than others is the cost of the probe and commercial access issues, such as license terms and conditions. Some factors related to commercial considerations that may encourage a user to weight certain markers higher than others are Research and Development (R&amp;D) time, R&amp;D cost, R&amp;D risk, i.e., the probability that the probe will work, cost of final analytical instrument, final performance and the time to market. In a detection panel, for example, some factors related to misclassifications or error rates that may encourage a user to weight some markers higher than others is that it may be desirable to rninimize false negatives. In a discrimination panel, on the other hand, it may be desirable to minimize false positives. Some factors related to prevalence of a generic or specific disease state in a geographic area that may encourage a user to weight some probes higher than others are that in some geographic locations the incidence of certain generic or specific diseases are more or less prevalent. With respect to redundancies, in some instances it is desirable to have redundancies in the panel. For example, if for some reason one probe fails to be detected, due to the biological variability of the markers in the panel, a disease state will still be detected by the other markers. In some embodiments, markers that are preferred redundant markers may be weighted more heavily.<br>
The invention is flexible in being adaptable to the availability of features where cost or supply problems may not allow the very best combination. In one embodiment, the invention can simply be applied to the available features to find an alternative combination. In another embodiment, the algorithm is used to select features that allow cost weightings to be included in the selection process to arrive at a minimum cost solution. In the examples, marker performance estimates for combinations selected from all the markers collected or for only a group of commercially preferred probes are shown. The examples also demonstrate how the C4.5 package can be used to down weight certain probes on the basis of their high cost. These probe combinations may not perform as well as the optimum combination, but the performance might be acceptable in circumstances where cost is a significant factor.<br>
Some of the methods used allow weightings to be applied to the classes. This is available in C4.5 where the tree design can optimize the cost. Also, the Discriminant Function method gives a single parameter output which can be used to give a desired false positive or false negative probability. A plot of these parameters for different threshold settings is known as the receiver operating characteristic (ROC) curve. An ROC curve shows the estimated percentage of false positive against true positive scores for different threshold levels of a classifier.<br>
Given the heterogeneous nature of many generic disease states, the panels may be constructed with a degree of redundancy to ensure that the tests have sufficient sensitivity, specificity, positive predictive value (Positive Predictive Value = True Positives/(True Positives + False Positives) and negative predictive value (Negative Predictive Value = True negatives/(False Negatives + True Negatives) to justify their use as a population-based screen. However, local and regional differences may dictate specific use of the tests in different segments of the global market, and so may significantly influence the criteria used to construct the final panel test for a given market. While the optimization of clinical utility is of utmost importance, local factors including affordability (cost), technical competence, laboratory and healthcare provider resources, workflow issues, manpower requirements, and availability of the probes and labels will contribute to a final, local selection of the markers used in the panel. Well known linear discriminant function analysis is used to include and assess all potential selection factors, by which each local factor is represented by a term in the equation, and each is weighted according to its locally determined significance. In this way, a panel test optimized for use in one world region may differ from a panel test optimized for use in a different region.<br>
Once detection or discrmination panels have been designed using the above described method, the next step is to validate the panel using known cytology samples. Prior to validation, optional optimization steps may be performed. In some embodiments, the method for collecting cytology samples may be improved. This encompasses methods of obtaining the sample from the patient as well as methods for mixing the cytology sample. In other embodiments, the cytology presentation methods may be improved. For example, identifying optimal fixatives (preservation fluids) or transportation fluids.<br>
The cytology samples used to validate the panels produced using the gold standard histology samples are cytology samples with known diagnoses. These samples may be collected using any method known by those of skill in the art. For example, sputum samples can be collected by spontaneous production, induced production and through the use of agents that enhance sputum production. The sample is contacted with each probe in the panel and the level and partem of binding of the probes is analyzed to determine the performance of the panel. In some embodiments, it may be necessary to further optimize the panel. For example, it may be necessary to remove a probe from the panel. Or, it may be necessary to add an additional probe to the panel. Additionally, it may be necessary to replace one probe on the panel with another probe. If a new probe is added, this probe may be a correlate marker as determined from a correlation matrix. Alternatively, the probe may be a functionally similar marker. Once the panel is optimized, the panel may proceed for further testing in clinical studies.<br>
In other embodiments, it is not necessary to optimize the panel. If the results with the cytology samples correlate with the results from the histology samples, there may not be a need to optimize the panel and the panel may proceed for further testing in clinical studies.<br>
4.   Methods of Use<br>
Once a panel is obtained using the above described method, it may be applied to cytologic samples. To illustrate the method, cancer, especially lung cancer, will be exemplified. Similar steps and procedures will be applied for other disease states. It is to be expected that cells shed from the specified lesions will stain in a similar fashion and show in a cytologic sample, such as a fine need aspiration, sputum,-urine, in a similar fashion as in the histologic pathology samples used to obtain the panel.<br>
The basic method of the present invention typically involves two steps. First, a cytological sample suspected of containing diseased cells is contacted with a panel<br>
containing a plurality of agents, each of which quantitatively binds to a disease marker. Then, the level or pattern of binding of each agent to a disease marker is detected. The results of the detection may be used to diagnose the presence of a generic disease or to discriminate among specific disease states. An optional preliminary step is identifying an optimized panel of agents that will aid in the detection of a disease or the discrimination between disease states in a cytologic sample.<br>
Cytology specimens may include, but are not limited to, cellular samples collected from body fluids, such as blood, urine, spinal fluids, and lymphatic systems; epithelial cell-based organ systems, such as the pulmonary tract, e.g., lung sputum, urinary tract, e.g., bladder washings, genital tract, e.g., cervical PAP smears, and gastrointestinal tract, e.g., colonic washings; and fine needle aspirations from solid tissue sites in organs and systems such as the breast, pancreas, liver, kidneys, thyroid, bone marrow, muscles, prostate, and lungs; biopsies from solid tissue sites in organs and systems such as the breast, pancreas, liver, kidneys, thyroid, bone marrow, muscles, prostate, and lungs; and histology specimens, such as tissue from surgical biopsies.<br>
An illustrative panel of agents according to the present invention includes any number of agents that allows for accurate detection of malignant cells in a cytological sample. Molecular markers envisioned by the present invention may be any molecule that aids in the detection of malignant cells. Markers may be selected for inclusion in a panel based on several different criteria relating to changes in level or pattern of expression of the marker. Preferred are molecular markers where changes in expression: occur early in tumor progression, are exhibited by a majority of tumor cells, allow for detection of in excess of 75% of a given tumor type, most preferably in excess of 90% of a given tumor type and/or allow for the discrimination between histologic types of cancer.<br>
The first step of the basic method is the detection of changes in the level or pattern of expression of the panel of agents in a cytological sample. This step typically involves contacting the cytologic sample with an agent, such as a labeled polyclonal or monoclonal antibody or fragment thereof or a nucleic acid probe, and observing the signal in individual cells.   Detection of cells where there is a change in signal is indicative of a change in the level of expression of the molecular marker to which the label probe is directed. The changes are based on an increase or decrease in the level of expression relative to nonmalignant cells obtained from the tissue or site being examined..<br>
An analysis of the changes in the level or pattern of expression of a panel of agents enables a skilled artisan to determine, with high sensitivity and high specificity,<br>
whether malignant cells are present in the cytologic sample. The term "sensitivity" refers to the conditional probability that a person having a disease will be correctly identified by a clinical test, (the number of true positive results divided by the number of true positive and false negative results). Therefore, if a cancer detection method has high sensitivity, the percentage of cancers detected is high e.g., 80%, preferably greater than 90%. The term "specificity" refers to the conditional probability that a person not having a disease will be correctly identified by a clinical test, (i.e., the number of true negative results divided by the number of true negative and false positive results). Therefore, if a cancer detection method has high specificity, 80%, preferably 90%, more preferably 95%, the percentage of false positives the method produces is low. A "cytologic sample" encompasses any sample collected from a patient that contains that patient's cells. Examples of cytological samples envisioned by the present invention include body fluids, epithelial cell-based organ system washings, scrapings, brushings, smears or effusions, and fine-needle aspirates and biopsies.<br>
Use of the markers described in this invention assumes that it is possible to obtain an adequate cytologic sample routinely and that the samples can be adequately preserved for subsequent evaluation. The cytologic sample may be processed and stored in a suitable preservative. Preferably, the cytologic sample is collected in a vial containing the preservative. The preservative is any molecule or combination of molecules known to maintain cellular morphology and inhibit or block degradation of cellular proteins and nucleic acids. To ensure proper fixation, the sample may be mixed at the collection site at high speeds to disaggregate the sample and/or break up obscuring material such as mucus, thereby exposing the cells to the preservative.<br>
Once a specimen is obtained, it is desirable to homogenize it, using an appropriate mixing device. This permits using aliquots for multiple purposes, including the possibility of sending aliquots to more than one testing site, as well as preparing multiple slides and/or multiple depositions on a slide. The initial homogenization of the specimen and of each aliquot before use will ensure that each individual slide will have substantially the same distribution of cells, so that comparisons of results from one slide to another will be meaningful.<br>
Preparation of a specimen for analysis involves applying a sample to a microscope slide using methods including, but not limited to, smears, centrifugation, or deposition of a monolayer of cells. Such methods may be manual, semi-automated, or fully automated The cell suspension may be aspirated depositing the cells on a filter and a monolayer of cells transferred to a prepared slide that may be processed for further<br>
evaluation. By repeating this process additional slides may be prepared as necessary. The present invention encompasses detection of one molecular marker per slide. Detection of several molecular markers per slide is also envisioned. Preferably, 1-6 markers are detected per slide. In some embodiments 2"markers are detected per slide. In other embodiments, 3 markers arexdetected per slide.<br>
The present invention contemplates detecting changes in molecular marker expression at the DNA, RNA or protein level using any of a number of methods available to an ordinary skilled artisan. Detection of the changes in the level or pattern of expression of the molecular markers in a cytologic sample generally involves contacting a cytologic sample with a polyclonal or monoclonal antibody or fragment thereof or a nucleic acid sequence that is complementary to the nucleic acid sequence encoding a molecular marker in the panel, collectively "probes", and a label. Typically, the probe and label components are operatively linked so that when the probe reacts with the molecular marker a signal is emitted (a "labeled probe"). Labels envisioned by the present invention are any labels that emit or enable a signal and allow for identification of a component in a sample. Preferred labels include radioactive, fluorogenic, chromogenic or enzymatic moieties. Therefore, possible methods of detection include, but are not limited to, immunocytochemistry; proteomics, such as immunochemistry, cytogenetics, such as in situ hybridization, and fluorescence in situ hybridization; radiodetection, cytometry and field effects, such as MACs and DNA ploidy (the quantitation of stoichiometrically-stained nuclear DNA using automated computerized cytometry) and; cytopathology, such as quantitative cytopathology based on morphology. The signal generated by the labeled probe is preferrably of sufficient intensity to permit detection by a medical practitioner or technician.<br>
Once the slide is prepared, a medical practitioner conducts a microscopic review of the slides in order to identify cells that exhibit a change in marker expression characteristic of a diagnosis of cancer. The medical practitioner may use an image analysis system and automated microscope to identify cells of interest. Analysis of the data may make use of an information management system and algorithms that will assist the physician in making a definitive diagnosis and select the optimal therapeutic approach. A medical practitioner may also examine the sample using an instrument platform that is capable of detecting the presence of the labeled agent<br>
A molecular diagnostic panel assay will result in one or more glass microscope slides with labeled cells and/or tissue sections. The challenge for human experts to assess these (cyto)pathology multilabeled-cell preparations objectively and with clinically<br>
meaningful results is a virtually insurmountable detection and. perception problem for any<br>
human being.<br>
Computer-aided imaging systems (i.e., Photonic Microscopes™) can be developed and used to assess quantitatively and reproducibly the amount and location of probe-labeled cells and tissues. Such Photonic Microscopes™ combine robotic slide-handling capabilities, data management systems (e.g., medical informatics), and quantitative digital (optical and electronic) image analysis hardware and software modules to detect and report cell-based probe content and localization data that cannot be obtained by human visualization with comparable sensitivity and accuracy. These probe data can be used to characterize and differentiate cellular samples based, upon their related characteristics and differences in their respective cell-based markers for a variety of disease states.<br>
The present methodology is a methodology whereby the molecular diagnostic panels are applied to cell-based specimens and samples, and whereby computer-aided imaging systems are subsequently used to quantify and report the results of the molecular diagnostic panel tests. Such imaging systems can be used to evaluate cell-based samples in which multiple probes are used simultaneously on a given slide-based sample, and in which the probes can be separately analyzed, quantified, and reported because the probes are differentiated by color on the microscope cytology or histology slide.<br>
The signals generated by a labeled agent in the sample may, if they are of appropriate type and of sufficient intensity, be detected by a human reviewer (e.g., pathologist) using a standard microscope or a Computer-Aided Microscope [167]. The Computer-Aided Microscope is an ergonomic, computer-interfaced microscope workstation that integrates mouse-driven control of microscope operation (e.g., stage movement, focusing) with computerized automation of key functions (e.g., slide scanning patterns). A centralized Data Management System stores, organizes and displays relevant patient information as well as results from all specimen screenings and pathologist reviews. An identification number that is imprinted onto barcodes and affixed to each sample slide uniquely identifies each sample in the database, and relates it to the original specimen and the patient.<br>
In a preferred embodiment the signals generated by a labeled agent in the sample will be detected and quantitated using an automated image analysis system, or Photonic Microscope, interfaced to the centralized Data Management System. The Photonic Microscope provides fully automated software control of the microscope operations and<br>
incorporates detectors and other components appropriate for quantitation even of signals not detectable by human reviewers, such as very faint signals or signals from radiolabeled moieties. The location of detected signals is stored electronically for rapid relocation by automated instruments, and for human review-using a Computer-Aided Microscope [168].<br>
The centralized Data Management System archives all patient and sample data using the bar-coded identification number. The data may be acquired asynchronously, from a multiplicity of sites, and may be derived from multiple reviews and analyses by human cytologists and/or automated analyzers. These data may include results from multiple sample slides representing aliquots from a single previously homogenized patient specimen. Part or all of the data may be transferred to or from a hospital's Laboratory Information System to meet reporting, archiving, billing or regulatory requirements. A single, comprehensive report with integrated results from panel tests and human reviews may be generated and delivered to the physician in hardcopy, or electronically through networked computers or the Internet.<br>
In some embodiments, the instant method allows for differential discrimination of different diseases, such as different histologic types of cancers. The term "histologic type" refers to specific disease states. Depending on the general disease state there can be one or several histologic types. For example, lung cancer includes, but is not limited to, squamous cell carcinoma, adenocarcinoma, large cell carcinoma, small cell carcinoma and mesothelioma. Knowledge of the histologic type of cancer affecting a patient is very useful because it helps the medical practitioner to localize and characterize the disease and to determine the optical treatment strategy.<br>
In order to determine die specific disease state, a panel of markers is selected that allows for discrimination between specific disease states. For example, within a panel of molecular markers, a pattern of expression may be identified that is indicative of a particular histologic type of cancer. The detection of the level of expression of the panel of molecular markers is achieved by the above-described methods. Preferably, a panel of 1-20 molecular markers is employed to discriminate among the various histologic types of lung cancer. However, most preferably, 4-7 markers are used. Decision trees may be developed to aid in discriminating between different histologic types based on patterns of marker expression.<br>
In addition to allowing for the detection of malignant cells in a cytologic sample, the instant invention has utility in the molecular characterization of the disease state. Such information is often of prognostic significance and can assist the physician in the selection of the optimal therapeutic approach for a particular patient In addition, the panel of<br>
markers described in this invention may have utility in monitoring the patient for either recurrence or to measure the efficacy of the therapy being used to treat the disease.<br>
By way of non-limiting example, the presence of lung cancer may be detected by a lung cancer detection panel and the specific type of lung cancer may be detected by a discrimination panel.s If the medical practitioner determines that malignant cells are present in the cytologic sample, a further analysis of the histologic type of lung cancer may be performed. The histologic type of lung cancer encompassed by the present invention includes but is not limited to squamous cell carcinoma, adenocarcinoma, large cell carcinoma, small cell carcinoma and mesothelioma. Figure 1 illustrates molecular markers that are preferable markers to be included in a panel for identifying different histologic types of lung cancer. The column labeled "%" indicates the percentage of tumor specimens that express a particular marker.<br>
In determining the various histologic types of lung cancer, the relative level of expression of a marker is analyzed. Figure 2 illustrates how different markers may be used to discrirninate among different histologic types of cancer. In this table, SQ indicates squamous cell carcinoma, AD indicates adenocarcinoma, LC indicates large cell carcinoma, SC indicates small cell carcinoma and ME indicates mesothelioma. The numbers appearing in each cell represent frequency of marker change in one cell type versus another. To be included in the table, the ratio must be greater than 2.0 or less than 0.5. A number larger than 100 generally indicates that the second marker is not expressed. In such cases the denominator was set at 0.1 for the purpose of the analysis. Finally, empty cells represent either no difference in expression or the absence of expression data.<br>
One method for analyzing the data collected is to construct decision trees. Schemes 1-4 are examples of decision trees that may be constructed to enable a differential determination of a histologic type of lung cancer using the patterns of expression. The present invention is in no way limited to the decision trees presented in Schemes 1-4. The relative level of expression of a marker can be higher, lower, or the same (ND) as the level of expression of the molecular marker in a malignant cell of a different histologic type. Each scheme enables a distinction between five histologic types of lung cancer through the use of the indicated panel of molecular markers.<br>
For example, in Scheme 1 the panel consists of HERA, MAGE-3, Thrombomodulin and Cyclin Dl. First the sample is contacted with a labeled probe directed toward HERA. If the expression of HERA is lower than the control, the test indicates that the histologic type of lung cancer is mesothelioma (ME). If, however, the expression is higher or<br>
the same as the control, the sample is contacted with a probe directed toward MAGE-3. If the expression of MAGE-3 is lower than the control, the sample is contacted with a labeled probe directed toward Cyclin Dl and a determination of small cell carcinoma (SC) or adenocarcinoma (AD) is possible. If the expression of MAGE-3 is higher than or the same as the control, the sample is contacted with a labeled probe directed toward Thrombomodulin and a determination of squamous cell carcinoma (SC) or large cell carcinoma (LC) is possible.<br>
(Scheme Removed)<br>
In Scheme 2 the panel consists of E-Cadherin, Pulmonary Surfactant B and Thrombomodulin. First the sample is contacted with a labeled probe directed toward E-Cadherin. If the expression of E-Cadherin is lower than the control, the test indicates that the histologic type of lung cancer is mesothelioma (ME). If, however, the expression is higher or the same as the control, the sample is contacted with a probe directed toward Pulmonary Surfactant B. If the expression of Pulmonary Surfactant B is lower than the control, the sample is contacted with a labeled probe directed toward Thrombomodulin and a determination of squamous cell carcinoma (SQ) or large cell carcinoma (LC) is possible. If the expression of Pulmonary Surfactant B is higher than or the same as the control, the sample is contacted with a labeled probe directed toward CD44v6 and a determination of adenocarcinoma (AD) and small cell carcinoma (SC) is possible. (See Schemes 3 and 4 for more examples of decision trees). (Scheme 3,4 Removed) <br>
Scheme 4<br>
(Scheme Removed)<br>
A preferred method involves using panels of molecular markers where differences in the partem of expression permits the discrimination between the various histologic type of lung cancer.<br>
Many different decision trees may be constructed to analyze the patterns of marker expression. This information may be used by physicians or other healthcare providers to make patient management decisions and select an optimal treatment strategy.<br>
5.   Reporting of Results ofPanel Analysis<br>
The results from the panel analysis may be reported in several ways. For example, the results may be reported as a simple "yes or no" result. Alternatively, the result may be reported as a probability that the test results are correct For example, the results from a detection panel study may indicate whether a patient has a generic disease state or not. As the panel also reports the specificity and sensitivity, the results may also be reported as the probability that the patient has a generic disease state. The results from a discrimination panel analysis will discriminate among specific disease states. The results may be reported as a "yes or no" with respect to whether the specific disease state is present. Alternatively, the results may be reported as a probability that a specific disease state is present. It is also<br>
possible to perform several discrimination panel analyses on a specimen from one patient and report a profile of the probabilities that the disease state present is a specific disease state with respect to the other possibilities. The other possibilities may also include false positives.<br>
In embodiments in which a profile of the probabilities of each specific disease state being present is produced, mere are several possible outcomes. For example, it is possible that all of the probabilities will be a very small probability. In this instance, it is possible that the doctor will conclude that the patient's specimen diagnosis is a false positive. It is also possible that all of die probabilities will be low except for one that is above 80-90%. In this instance, it is possible that the doctor will conclude that the test verifies that the patient has the specific disease state that indicated the high probability. It is also possible that most of the probabilities will be low, but similarly high probabilities are reported for two specific disease states. In this case, a doctor may recommend more extensive panel testing to ensure that the correct disease state is identified. Another possibility is that all of the probabilities reported will be low, with one being slightly higher than the rest but not high enough to be in the 80-90% range. In this case, a doctor may recommend more extensive panel testing to ensure that the correct disease state is identified and/or to rule out metastatic cancer from a remote primary tumor of a different cancer type.<br>
The following Example is illustrative of the method of the invention for selecting a disease detection panel, disease discrimination panels, validation of the panels and use of the panels in the clinic to screen for a disease and to discriminate among different subtypes of the disease. Lung cancer was selected for this illustrative example, in part because of its importance to world health, but it will be appreciated that similar procedures will apply to other types of cancer, as well as to infectious, degenerative and autoimmune diseases, according to the foregoing general disclosure.<br>
ILLUSTRATIVE EXAMPLE<br>
The present method was used to develop lung cancer detection panels as well as single lung cancer type specific discrimination panels. Lung cancer is an extremely complex collection of diseases that can be segregated into two main classes. Non-small cell lung carcinoma (NSCLC) that accounts for approximately 70 to 80% of all lung cancers can be further subdivided into three main histologic types including squamous cell carcinoma, adenocarcinoma, and large cell carcinoma. The remaining 20 to 30% of lung cancer patients present with small cell lung carcinoma (SCLC). In addition, malignant mesothelioma of the<br>
pleural space, can develop in individuals exposed to asbestos and will often spread widely invading other thoracic structures. Different forms of lung cancer tend to localize in different regions of the lung, have different prognoses, and respond differently to various forms of therapy.<br>
According to the latest statistics from the World Health Organization (Globocan 2000), lung cancer has become the most common fatal malignancy in both men and women with an estimated 1.24 million new cases and 1.1 million deaths each year. In the U.S. alone, the National Cancer Institute reports that there are approximately 186,000 new cases of lung cancer and each year 162,000 people die of the disease, accounting for 25% of all cancer-related deaths. In the U.S., overall 1-year survival for patients with lung cancer is 40%, however, only 14% live 5 years. In other parts of rise world, 5-year survival is significantly lower (5% in the UK). The high mortality of lung cancer can be attributed to the fact that most patients (85%) are diagnosed with advanced disease when treatment options are limited and the disease is likely to have metastasized. In these patients, 5-year survival is between 2-30% depending of the stage at the time of diagnosis. This is in sharp contrast to cases where patients are diagnosed early and 5-year survival is greater than 75%. While it is true that a number of new chemotherapeutic agents have been introduced into clinical practice for the treatment of advanced lung cancer, to date, none have yielded a significant improvement in long-term survival. Even though patients with early stage disease can presumably be cured by surgery, they remain at significant risk, as there is a high probability that they will develop a second malignancy. Thus, for the lung cancer patient, early detection and treatment followed by aggressive monitoring provides the best chance of achieving significant improvements in long-term survival along with a reduction in morbidity and cost.<br>
At the present time, a patient is suspected of having lung cancer either because of a suspicious lesion on X-ray or because the patient becomes symptomatic. As a result, most patients are diagnosed with relatively late stage disease. In addition, because most methods lack sufficient sensitivity with respect to the detection of early stage disease, the current policy of the U.S. National Cancer Institute (NCI), National Institutes of Health, recommends against screening for lung cancer even in populations of patients who are at significant risk. In this embodiment of the present invention, however, sputum cytology is employed to provide a relatively noninvasive, more effective and cost-effective means for the early detection of lung cancer.<br>
The specificity of sputum cytology is relatively high. Recent studies have indicated that experienced cytotechnologists are able to recognize malignant or severely<br>
dysplastic cells with a high degree of accuracy and reliability [10]. While the detection rate can be as high as 80 to 90% when samples are collected from patients with a relatively advanced disease [11,12], overall, sputum cytology has a sensitivity of only 30-40% [13,14]. The low sensitivity of sputum cytology is particularly important given that obtaining and preparing the specimen can be relatively expensive. Furthermore, failing to detect a malignancy can significantly delay treatment thereby reducing the chance of achieving a cure.<br>
The selection of an "at-risk" population can also influence the value of sputum cytology as a screening tool. Individuals who are at significant risk include those with a prior diagnosis of lung cancer, long-term smokers or former smokers (&gt;30 pack years) and individuals with long-term exposure to asbestos or pulmonary carcinogens. People with a genetic predisposition or familial history are also included in an "at-risk" population. Such individuals are likely to benefit from testing. While the inclusion of individuals with lower risk may result in an increase in the absolute number of cases detected, it would be hard to justify the substantial increase in healthcare costs.<br>
Other factors that contribute to the relatively poor performance of conventional sputum cytology include the location of the lesion, tumor size, histologic type, and the quality of the sample. Squamous-cell carcinoma accounts for 31% of all primary pulmonary neoplasms. Most of these tumors arise from segmental bronchi and extend to the proximal lobar and distal subsegmental branches [15]. For this reason, sputum cytology is reasonably effective (79%) in detecting these lesions. Currently, squamous cell carcinoma is viewed as the only type of lung cancer that is amenable to cytologic detection in an in situ and radiologically occult stage [15], as sloughed cells are more likely to be available for evaluation. In one large study where patients were followed with both chest X-ray and sputum cytology, 23% of all lung cancers were detected by cytology alone, suggesting that the tumors were early stage and radiologically occult [16]. In another study [17], sputum cytology detected 76% of patients with radiologically occult tumors.<br>
In the case of adenocarcinoma, 70% of tumors occur in the periphery of the lung making it less likely that malignant cells will be found in a conventional sputum specimen. For this reason, adenocarcinomas are rarely detected by sputum cytology (45%) [12,18,19], an important consideration, since the incidence of adenocarcinoma appears to be increasing, particularly in women [20-22].<br>
Tumor size can also affect the likelihood of achieving a correct diagnosis, a factor that is particularly important when considering a screening test for the detection of<br>
disease in asymptomatic individuals. While there is only a 50% chance that tumors 
[12].<br>
Recent reports also indicate that the cellularity of the specimen will affect the sensitivity of sputum cytology [14,23]. In general, patients with squamous cell carcinoma produce specimens with significant numbers of tumor cells, thereby increasing the likelihood of a correct diagnosis [14,23]. For patients with adenocarcinoma, the presence of tumor cells in a sputum specimen is reported to be less than 10% in 95% of the specimens and less than 2% in 75% of specimens, making the diagnosis significantly more difficult.<br>
The degree of differentiation can also influence the ability of a pathologist to detect malignant cells, particularly in cases of adenocarcinoma. Well-differentiated tumor cells frequently resemble nonneoplastic respiratory epithelial cells. In the case of small-cell lung carcinoma, sputum samples often contain nests of loosely aggregated cells that have a distinct appearance. However, techniques currently used to process sputum samples tend to disaggregate the cells, making a diagnosis more difficult.<br>
Sample quality is another factor that can contribute to the low sensitivity of sputum cytology. Recent reports suggest that it is possible to obtain adequate samples from 70-85% of subjects. However, achieving this measure of success often requires that patients provide multiple specimens [13]. This procedure is inconvenient, time-consuming and costly. Patient compliance is also generally low, as patients are frequently asked to collect over several days [13]. Of equal importance is the observation that former smokers, while at significant risk for developing lung cancer, often fail to produce an adequate specimen. Sample preservation and processing is another critical factor that can affect the value of sputum cytology as a diagnostic test.<br>
Lastly, even if adequate samples could be obtained and optimally prepared, cytotechnologists generally still have to review 2-4 slides per specimen, each typically taking up to four minutes [24]. Given the low sensitivity, high technical complexity and labor intensity of conventional sputum cytology, it is not surprising that this test has been almost universally rejected as a population-based screen for the early detection of lung cancer [25].<br>
Even if these technical issues were resolved, the low sensitivity of sputum cytology remains a significant problem. The high incidence of false negative results can significantly delay the patient receiving potentially curative therapy. While it may be possible to develop tests with greater sensitivity, such improvements must not come at the cost of specificity. An increase in the number of false positive results would subject patients<br>
to unnecessary, often invasive and costly, follow-up and would have a negative impact on the patient's quality of life. The present invention overcomes many of the limitations associated with previous methods of early cancer detection, including those related to the use of sputum cytology for the early detection of lung cancer.<br>
Lung cancer is a heterogeneous collection of diseases. To ensure that a test has the necessary level of sensitivity and specificity to justify its use as a population based screen, the present invention envisions using, for example, a library of 10 to 30 cellular markers to develop panels. Selection of the library of this invention was based on a review and reanalysis of the relevant scientific literature where, in most cases, marker expression was measured in biopsy specimens taken from patients with lung cancer in an attempt to link expression with prognosis.<br>
For example, a preferred panel for early detection, characterization, and/or monitoring of lung cancer in a patient's sputum may include molecular markers for which a change in expression occurred in at least 75% of tumor specimens. An exemplary panel includes markers selected from VEGF, Thrombomodulin, CD44v6, SP-A, Rb, E-Cadherin, cyclin A, nm23, telomerase, Ki-67, cyclin Dl, PCNA, MAGE-1, Mucin, SP-B, HERA, FGF-2, C-MET, thyroid transcription factor, Bcl-2, N-Cadherin, EGFR, Glut-1, ER-related (p29), MAGE-3 and Glut-3. A most preferred panel includes molecular markers for which a change in expression occurs in more than 85% of tumor specimens. An exemplary panel includes molecular markers selected from Glutl, HERA, Muc-1, Telomerase, VEGF, HGF, FGF, E-cadherin, Cyclin A, EGF Receptor, Bcl-2, Cyclin Dl and N-cadherin. With the exception of Rb and E-cadherin, a diagnosis of lung cancer is associated with an increase in marker expression. A brief description of die library of probes/markers utilized in the present example is provided below in Table 4. It is noted that the numbering of the antibodies in the table below is consistent with the number of the antibodies/probes/markers throughout this example.<br>
Table 4:<br><br>
(Table Removed)<br>
Each molecular marker in the preferred panel is described below.  Table 5, reciting the percentage of expression of the markers in tissue for each type of lung cancer is provided at the end of this section.<br>
Glucose Transporter Proteins (Glut 1 and Glut 3) [26-28]<br>
Glucose Transporter-1 (Glut 1) and Glucose Transporter-3 (Glut-3) are a ubiquitously expressed high affinity glucose transporter. Tumor cells often display higher rates of respiration, glucose uptake, and glucose metabolism than do normal cells, and the elevated upcake of glucose in tumor cells is thought to be mediated by glucose transporters. Overexpression of certain types of GLUT isoforms has been reported in lung cancer. The cellular localization of Glut 1 is in the cell membrane. GLUT-1 and GLUT-3 are disease markers useful for detection of a disease state.<br>
Malignant cells exhibit an increase in glucose uptake that appears to be mediated by a family of glucose transporter proteins (Gluts). Oncogenes and growth factors appear to regulate the expression of these proteins as well as their activities. Members of the Glut family of proteins exhibit different patterns of distribution in various human tissues and rapid proliferation is often associated with their overexpression. Recent evidence suggests that Glutl is expressed by a large percentage of NSCLC and by a majority of SCLC.<br>
While the expression of Glut 3 is relatively low in both NSCLC and SCLC a significant percentage (39.5%) of large cell carcinomas express the protein. In stage I tumors, 83% express Glutl at some level with 75-100% of cells staining in 25% of cases.<br>
These data would suggest that Glutl overexpression is a relatively early event in tumor progression. Glutl immunoreactivity has also been detected in &gt; 90% of stage II and IIIA cancers. There also appears to be an inverse correlation between Glutl and Glut3 immunoreactivity and tumor differentiationrTumors expressing high levels of Glutl appear to be particularly aggressive that are associate with a poor prognosis. In cases were tumors were negative for the proteins better survival was observed.<br>
Human Epithelial Related Antigen (HERA) [29,30]<br>
HERA is a transmembrane glycoprotein with an, as yet, unknown function. HERA is present on most normal and malignant epithelia. Recent reports suggest that the while HERA expression is high in all histologic types of NSCLC making it useful as a detection marker. In contrast HERA expression is absent in mesothelioma and thus suggesting would have utility as a discrimination marker. The cellular localization of HERA is the cell surface.<br>
Basic Fibroblast Growth Factor (FGF) [31-34]<br>
Basic Fibroblast Growth Factor (FGF) is a polypeptide growth factor with a high affinity for heparin and other glycosaminoglycans. In cancer, FGF functions as a potent mitogen, plays a role in angiogenesis, differentiation, and proliferation, and is involved in tumor progression and metastasis. FGF overexpression frequently occurs in both SCLC and squamous cell carcinoma. In many cases (62%), die cells also express the FGF receptor suggesting the presence of an autocrine loop. Forty-eight percent of Stage 1 tumors overexpress FGF. The frequency of FGF in Stage II lung cancer is 84%. Expression of either die growth factor or its receptor was associated with the poor prognosis. Five-year survival rates for those patients with stage I disease were 73% for those expressing FGF versus 80% for those who were FGF negative. The cellular localization is the cell membrane.<br>
Telomerase [35-42]<br>
Telomerase is a ribonucleoprotein enzyme that extends and maintains telomeres of eukaryotic chromosomes. It consists of a catalytic protein subunit with reverse transcriptase activity and an RNA subunit with reverse transcriptase activity and an RNA subunit that serves as the template for telomere extension. Cells that do not express telomerase have successively-shortened telomeres with each cell division, which ultimately leads to chromosomal instability, aging and cell death. The cellular localization of telomerase is nuclear.<br>
Expression of telomerase appears to occur in immortalized cells and enzyme activity is a common feature of the malignant phenotype. Approximately 80-94% of lung<br>
tumors exhibit high levels of telomerase activity. In addition, 71% of hyperplasia, 80% of metaplasia, and 82% of dysplasia express enzyme activity. All the carcinoma in situ (CIS) specimens exhibit enzyme activity. The low levels of expression in premaligant tissues is probably related to the fact that only a smaffpercentage of cells (5 and 20%) in the sample express enzyme activity. This is in contrast to tumors where 20-60% of cells may express enzyme activity. Based on a limited number of samples it would appear expression of telomerase activity is also common in SCLC.<br>
Proliferating Cell Nuclear Antigen (PCNA) [43-51]<br>
PCNA functions as a cofactor for DNA polymerase delta. PCNA is expressed in both S phase of the cell cycle and during periods of DNA synthesis associated with DNA repair. PCNA is expressed in proliferating cells in a wide range of normal and malignant tissues. The cellular localization of PCNA is nuclear.<br>
Expression of PCNA is a common feature of rapidly dividing cells and is detected in 98% of tumors. Immunohistochemical staining is nuclear with moderate to intense staining detected in 83% of NSCLC. Intense PCNA staining was observed in 51% of p53-negative tumors. However, when both PCNA (&gt; 50% of cells staining) and p53 are overexpressed (&gt;10% of cells stained) the prognosis tends to be poorer with a shorter time to progression. Although frequently detected in all stages of lung cancer, intense staining for PCNA is more common in metastatic disease. Thirty-one percent of CIS also overexpress PCNA.<br>
CD44_[51-58]<br>
CD44v6 is a cell surface glycoprotein that acts as a cellular adhesion molecule. It is expressed on a wide range of normal and malignant cells in epithelial, mesothelial and hematopoietic tissues. The expression of specific CD44 splice variants has been shown to be associated with metastasis and poor prognosis in certain human malignancies. It is expected to be used for detection and discrimination between squamous cell carcinoma and adenocarcinoma. CD44 is a cell adhesion molecule that appears to play a role in tumor invasion and metastasis. Alternative splicing results in the expression of several variant isoforms. CD44 expression is generally lacking in SCLC and is variably expressed in NSCLC. Highest levels of expression occur in squamous cell carcinoma, thus making it valuable in discriminating between tumor types. In non-neoplastic tissue, CD44 staining is observed in bronchial epithelial cells, macrophages, lymphocytes, and alveolar pneumocyies. There was no significant correlation between CD44 expression and tumor stage, recurrence, or survival particularly when overexpression occurs in early stage disease.<br>
In metastatic lesions 100% of squamous cell carcinoma and 75% of adenocarcinoma showed strong CD44v6 positivity. These data would tend to indicate that changes in CD44 expression occur relatively late in tumor progression that could limit its value as an early detection marker. Recent findings suggest that the CD44v8-10 variant is expressed by a majority of NSCLC making it a possible candidate marker.<br>
Cyclin A [59-62]<br>
Cyclin A is a regulatory subunit of the cyclin-dependent kinases (CDK's) which control the transition points at specific phases of the cell cycle. It is detectable in S phase and during progression into G2 phase. The cellular localization of Cyclin A is nuclear.<br>
Protein complexes consisting of cyclins and cyclin-dependent kinases function to regulate cell cycle progression. Changes in cyclin expression are associated with genetic alterations affecting the CCDN1 gene. While the cyclins act as regulatory molecules, the cyclin-dependent kinases function as catalytic subunits activating and inactivating Rb.<br>
Imrnunohistochemical analysis has revealed that the overexpression of the cyclins is associated widi an increase in cellular proliferation as indicated by a high Ki-67 labeling index. Cyclin overexpression occurs in 75% of NSCLC and appears to occur relatively early in tumor progression. Recent reports indicate that 66.7% of stage I/n and 70.9% of stage EI tumors overexpress Cyclin A. Nuclear staining is common in poorly differentiated tumors. Expression of cyclin A is often associated with a decrease in mean survival time and a tendency towards the development of drug resistance. However, increased expression has also been associated with a greater response to doxorubicin.<br>
Cvclin Dl [63-73]<br>
Cyclin Dl, as with Cylcin A, is a regulatory subunit of the cyclin-dependent kinases (CDK's) which control the transition points at specific phases of the cell cycle. Cyclin Dl regulates the entry of cells into S phase of the cell cycle. This gene is frequently amplified and/or its expression deregulated in a wide range of human malignancies. The cellular localization of Cyclin Dl is nuclear.<br>
Like Cyclin A, cyclin D1 functions to regulate cell cycle progression. Staining of cyclin Dl is predominately cytoplasmic and independent of histologic type. Reports suggest that cyclin Dl overexpression occurs in 40-70% of NSCLC and 80% of SCLC. Cyclin Dl, staining was observed in 37.9% of stage I, 60% stage II, and 57.9% of stage HI tumors. Cyclin Dl expression has also been seen in dysplastic and hyperplastic tissue providing evidence that these changes occur relatively early in tumor progression.<br>
Patients who. overexpress cyclin Dl exhibit shorter mean survival time and lower five-year survival rate.<br>
Hepatocvte Growth Factor Receptor (C-MET) [74-77]<br>
C-MET is a proto-oncogene that encodes a transmembrane receptor tyrosine kinase for HGF. HGF is a mitogen for hepatocytes and endothelial cells, and exerts pleitrophic activity on several cell types of epithelial origin. The cellular localization of C-MET is the cell surface.<br>
Hepatocyte growth factor/scatter factor (HGF/SF) stimulates a broad spectrum of epithelial cells causing them to proliferate, migrate, and carry out complex differentiation programs including angiogenesis. HGF/SF binds to a receptor encoded by the c-MET oncogene. While both normal and malignant tissues express the HGF receptor, expression of HGF/SF appears to be limited to malignant tissue.<br>
While the human lung generally expresses low levels of HGF/SF, expression increases markedly in NSCLC. Using Western blot analysis, 88.5% of lung cancers exhibited an increase in the protein expression. Ail histologic types of tumors expressed the protein at increased concentrations. While increased levels of protein occur in ail stages of the disease, recent evidence suggests that in addition to the cancer cells, stromal cells and/or inflammatory cells may be responsible for the production of the growth factor.<br>
Mucin -MUC-1 (78-82]<br>
Mucin-1 comes from a family of highly glycosylated secretory proteins which comprise the major protein constituents of the mucous gel which coats and protects the tracheobronchial tree, gastrointestinal tract and genitourinary tract. Mucin-1 is atypically expressed in epithelial tumors. The cellular localization of Mucin-1 is cytoplasm and the cell surface.<br>
Mucins are a family of high molecular weight glycoproteins that are synthesized by a variety of secretory epithelial cells that are either membrane bound or secreted. Within the respiratory tract, these proteins contribute to the mucus gel that coats and protects that tracheobronchial tree. Changes in mucin expression commonly occur in conjunction with mahgnaut transformation including lung cancer. Evidence exists suggesting at these changes may contribute to alterations in cell growth regulation, recognition by the immune system, and the metastatic potential of the tumor.<br>
Although normal lung tissue expresses MUC-1, significantly higher levels of expression are found in lung cancer with highest levels occurring in adenocarcinoma. Staining appears to occur independently of stage and is more common in smokers than in<br>
former smokers or nonsmokers. Some premalignant lesions also exhibit increased MLC-1 expression.<br>
Thvroid Transcription Factor-1 (TTF-1') [83,84]<br>
TTF-1 belongs to a family of homeodomain transcription factors that activate thyroid-specific and pulmonary-specific differentiation genes. The cellular localization of TTF-1 is nuclear.<br>
TTF-1 is a protein originally found to mediate the transcription of thyroglobulin. Recently, TTF-1 expression was also found in the diencephalon and brohchioloalveolar epithelium. Within the lung TTF-1 functions as a transcription factor regulating the synthesis of surfactant proteins and clara secretory protein. Overexpression of TTF-1 occurs in a large proportion of lung adenocarcinomas and can aid in distinguishing between primary lung cancer and cancers that metastasize to the lung. Adenocarcinomas that express TTF-1 and are cytokeratin 7 positive and cytokeratin 20 negative can be detected with 95% sensitivity.<br>
Vascular Endothelial Growth Factor (VEGF) [33,61,85-89]<br>
VEGF plays an important role in angiogenesis, which promotes tumor progression and metastasis. There are multiple forms of VEGF; the two smaller isofonns are secreted proteins and act as diffusible agents, whereas the larger two remain cell associated. The cellular localization of VEGF is cytoplasmic, cell surface, and extracellular matrix.<br>
Vascular Endothelial Growth Factor (VEGF) is an important angiogenesis factor and endothelial cell-specific mitogen. Angiogenesis is an important process in the latter stages of carcinogenesis, tumor progression and is particularly important in the development of distant metastasis. VEGF binds to a specific receptor Fit that is often present in the tumors expressing the growth factor suggesting the presence of an autocrine loop.<br>
Immunohistochemical analysis reveals that cells expressing VEGF exhibit a pattern of staining that is diffuse and cytoplasmic. While not expressed by nonneoplastic cells, VEGF is present in the majority of NSCLC and in a smaller percentage of SCLC. Several reports have shown high levels of VEGF in early stage lung cancer.<br>
Expression of VEGF has been associated with an increased frequency of metastasis. Studies have shown that VEGF expression is indicative of a poor prognosis and shorter disease-free interval in adenocarcinoma but not in squamous cell carcinoma. Three year and five year survival rates in the group expressing high levels of VEGF were 50% and 16.7% as compared to 90.9 and 77.9% respectively for the low VEGF group.<br>
Epidermal Growth Factor Receptor (EGFR) [90-104]<br>
Epidermal Growth Factor Receptor (EGFR) is a transmembrane glycoprotein, which can bind and become activated by various ligands. Binding initiates a chain of events that result in DNA synthesis, cell proliferation, and cell differentiation. EGFR has been demonstrated in a broad spectrum of normal tissues, and EGFR overexpression is found in a variety of neoplasms.   Increased expression has been observed in adenocarcinomas of the lung and large cell carcinomas but not in small cell lung carcinomas. The cellular localization of EGFR is the cell surface.<br>
The EGFR plays an important role in cell growth and differentiation. The EGFR is uniformly present in the basal cell layer but not in more the superficial layers of histologically normal bronchial epithelium. With this exception, there is no consistent staining of normal tissue. Recent evidence suggests that the overexpression of the EGF receptor may not be an absolute requirement for the development of invasive lung cancer. However, it appear that in cases where EGFR overexpression occurs it is a relatively early event with greater staining intensity in more advanced disease.<br>
For patients with invasive carcinomas, 50-77% of tumors stain for EGF. Overexpression of the EGFR is more common in squamous cell carcinoma than in adenocarcinoma and common in SCLC. Highest levels of EGFR occur in conjunction with late stage and metastatic disease that have approximately twice the concentration of EGFR as that seen in stage I/n tumors. Estimates suggest that the level of the EGFR observed in stage I tumors is approximately twice that seen in normal tissue. In addition, 48% of bronchial lesions also show EGFR staining including, metaplasia, atypia, dysplasia, and CIS. In the "normal" bronchial mucosa, of these same cancer patients, overexpression of the EGFR was observed in 39% of cases but was absent in the bronchial epithelium of the non-cancer. In addition, overexpression of the EGFR occurs more frequently in the tumors of smokers than in nonsmokers, particularly in the case of squamous cell carcinoma.<br>
While several studies have suggested that overexpression of the EGFR is associated with the poor prognosis, other studies have failed to make this correlation.<br>
Nucleoside Diphosphate Kinase/nm23 [105-111]<br>
Nucleoside diphosphate kinase (NDP kinase)/nm23 is a nucleoside diphosphate kinase. Tumor cells with high metastatic potential often lack or express only a low amount of nm23 protein, hence the nm23 protein has been described as a metastasis suppressor protein. The cellular localization of nm23 is nuclear and cytoplasmic.<br>
Expression of nm23/nucleoside diphosphate/kinase A (nm23) is a marker of tumor progression where there is an inverse relationship between expression and metastatic<br>
potential. In cases where stage I tumors overexpress mn23- no evidence of metastasis was seen during an average follow-up period of 35 months. Immunohistochemical analysis reveals staining that is diffuse, cytoplasmic and generally limited to malignant cells. Alveolar macrophages also express the protein. Given that high levels of expression are associated with a low metastatic potential, there is currently no explanation as to why normal epithelial cells do not express nm23.<br>
Intense staining has been observed in high percentage of NSCLC particularly large cell lung cancer and 74% of SCLC suggesting that this protein plays an important role in tumor progression. With the exception of squamous cell carcinoma, staining intensity tends to increase with stage. Based on the available evidence, it would appear that nm23 is a prognostic factor in both SCLC and NSCLC.<br>
Bcl-2 [10l.112-1251<br>
Bcl-2 is a mitochondrial membrane protein that plays a central role in the inhibition of apoptosis. Overexpression of bcl-2 is a common feature of cells in which programmed cell death has been arrested. The cellular localization of Bcl-2 is the cell surface.<br>
Bcl-2 is a protooncogene believed to play a role in promoting the terminal differentiation of cells, prolonging the survival of non-cycling cells and blocking apoptosis in cycling cells. Bcl-2 can exist as a homodimers or can form a heterodimer with Bax. As a homodimer, Bax functions to induce apoptosis. However, the formation of a Bax-bcl-2 complex blocks apoptosis. By blocking apoptosis, bcl-2 expression appears to confer a survival advantage upon affected cells. Bcl-2 expression may also play a role in the development of drug resistance. The expression of bcl-2 is negatively regulated by p53.<br>
Immunohistochemistry analysis of bcl-2 reveals a heterogeneous pattern of cytoplasmic staining. In adenocarcinoma, expression of bcl-2 was significantly associated with smaller tumors (
Overexpression of bcl-2 is not present in preneoplastic lesions suggesting that changes in bcl-2 occur relatively late in tumor progression. In addition to tumor cells, bcl-2 immunostaining also occurs in basal cells and on the luminal surfaces of normal bronchioles but is generally not detected in more differentiated cell types.<br>
Association of  bcl-2 immunoreactivity with improved prognosis in NSCLC is controversial. Several reports of suggested that patients with tumors expressing bcl-2 have a superior prognosis and a longer time to recurrence. Several reports indicate that bcl-2 expression tends to be lower in those patients who develop metastatic disease. For patients with squamous cell carcinoma, expression of bcl-2 has been linked to an improvement in 5-year survival. However, in three relatively large studies there was no survival benefit linked to bcl-2 expression, particularly for patients with early stage disease.<br>
Estrogen Receptor-related Protein (p29) [126]<br>
ER related protein p29 is an estrogen-related heat shock protein that has been found to correlate with the expression of estrogen-receptor. The cellular localization of p29 is cytoplasmic.<br>
Estrogen-dependent intracellular processes are important in the growth regulation of normal tissue and may play a role in the regulation of malignancies. In one study expression of p29 was detected in 109 (98%) of 111 lung cancers. The relation between p29 expression and survival time was different for men and women. Expression of p29 was associated with poorer survival particularly in women with Stage I and II disease. There was no correlation between p29 expression and long-term survival in men.<br>
Retinoblastoma Gene Product (Rb) r68.73.123.127-1411<br>
Retinoblastoma Gene Product (Rb) is a nuclear DNA-binding phosphoprotein. Under phosphorylated Rb binds oncoproteins of DNA tumor viruses and gene regulatory proteins thus inhibiting DNA replication. Rb protein may act by regulating transcription; loss of Rb function leads to uncontrolled cell growth. The cellular localization of Rb is nuclear.<br>
Retinoblastoma protein (pRb) is a protein that is encoded by the retinoblastoma gene and is phosphorylated and dephosphorylated in a cell cycle dependent manner. pRb is considered an important tumor suppressor gene that functions to regulate the cell cycle at G0/G1. In its hypophosphorylated state, pRb inhibits the transition from Gl to S. During Gl, inactivation of the growth suppressive properties of pRb occurs when the cyclin dependent kinases (CDK's) phosphorylate the protein. The hyperphosphorylation of pRb prevents it from forming a complex with E2F that functions as a transcription factor proteins that are required for DNA synthesis.<br>
Inactivation of the retinoblastoma (Rb) gene has been documented in various types of cancer, including lung cancer. Small-cell carcinomas fail to stain for pRb indicating loss of Rb function. Overall, 17.6% of the tumors fail to express pRb with no correlation being seen with respect to stage or nodal status. A reduction in staining has also seen in 31%<br>
dysplastic bronchial biopsies. However, there appears to be no correlation between pRb expression and the severity of dysplasia. In contrast, normal bronchial epithelium and cells taken from areas adjacent to tumors expressed pRb positive nuclei. These data suggest that alterations in the expression of the Rb protein may arise early in the development of some lung cancers.<br>
Patients with Rb-positive carcinomas tend to have a somewhat better prognosis but, in most studies, the difference is not significant. However, patients with adenocarcinoma whose tumors are both pRb negative and either p53 or ras positive exhibit a decrease in 5-year survival. A similar relationship does not occur in squamous cell carcinoma. pRb negative tumors have been reported to be more likely to exhibit resistant to doxorubicin than Rb-positive carcinomas.<br>
Thrombomodulin T142-1471<br>
Thrombomodulin is a transmembrane glycoprotein. Through its accelerated activation of protein C (which in turn acts as an anticoagulant by binding protein S and thrombin), synthesis of TM is one of several mechanisms important m reducing clot formation on the surface of endothelial cells. The cellular localization of thrombomodulin is the cell surface.<br>
Aggregation of host platelets by circulating tumor cells appears to play an important role in the metastatic process. Thrombomodulin plays an important role in the activation of the anticoagulant protein C by thrombin and is an important modulator of intravascular coagulation. In addition to its expression in normal squamous epithelium, expression of thrombomodulin also occurs in squamous metaplasia, carcinoma in situ, and invasive squamous cell carcinomas. Although present in 74% of primary squamous cell carcinomas, only 44% of metastatic lesions stained for thrombomodulin. These data suggest that, with progression, there is a decrease in thrombomodulin expression. Higher levels of expression tend to occur in well and moderately differentiated rumors when compared to poorly differentiated tumors.<br>
Patients with thrombomodulin-negative squamous cell carcinoma tend to have a worse prognosis. Eighteen percent of patients with thrombomodulin-negative have a five-year survival as compared to 60% in cases where the tumors stained positive for the protein. Progression to metastatic disease was also more common in thrombomodulin-negative tumors (69% vs. 37%) and there was a greater tendency for these tumors to develop at extrathorasic sites. Thus, loss of thrombomodulin expression appears to be prognostic in cases of squamous cell carcinoma. The observation that changes in thrombomodulin<br>
expression occur in later stages of NSCLC and that the protein is expressed by normal bronchial epithelial cells would tend to limit its utility as a marker for early detection. However, since a majority of mesotheliomas and only a small percentage of adenocarcinomas express thrombomodulin, the marker has potential utility in discrirninatmg between these two tumor types.<br>
E-cadherin &amp; N-cadherin [148-151]<br>
E-cadherin is a transmembrane Ca2+ dependent cell adhesion molecule. It plays an important role in the growth and development of cells via the mechanisms of control of tissue architecture and the maintenance of tissue integrity. E-cadherin contributes to intercellular adhesion of epithelial cells, the establishment of epithelial polarization, glandular differentiation, and stratification. Down-regulation of E-cadherin expression has been observed in a number of carcinomas and is usually associated with advanced stage and progression. The cellular localization of E-cadherin is the cell surface.<br>
E-cadherin is a calcium-dependent epithelial cell adhesion molecule. A decrease in E-cadhenn expression has been associated with tumor dedifferentiation and metastasis and decreased survival. Reduced expression has been observed in moderately and poorly differentiated squamous cell carcinoma and in SCLC. There was no change in E-cadherin expression in adenocarcinoma. Furthermore, while adenocarcinomas express E-cadherin theses tumors fail to express N-cadherin which is in contrast to mesotheliomas that express N-cadherin but not E-cadherin. Thus, these markers can be used to discriminate between adenocarcinoma and mesothelioma.<br>
Expression of E-cadherin can also be used to assess the prognosis of patients with squamous cell carcinoma. Whereas 60% of patients with tumors expressing E-cadherin survived three-year survival, only 36% of patients exhibiting a reduction in expression survived 3 years.<br>
MAGE-1 and MAGE-3 [152-156]<br>
Melanoma Antigen-1 (MAGE-1) and Melanoma Antigen-3 (MAGE-3) are members of a family of genes that are normally silent in normal tissues but when expressed in malignant neoplasms are recognized by autologous, tumor-directed and specific cytotoxic T cells (CTL's). The cellular localization of MAGE-1 and MAGE-3 is cytoplasmic.<br>
MAGE-1, MAGE-3 and MAGE 4 gene products are tumor-associated antigens that are recognized by cytotoxic T lymphocytes. As such, they could have utility as targets for immunotherapy in NSCLC. MAGE proteins are also expressed by some SCLCs but not by normal cells. While the frequency of MAGE expression falls below the level<br>
necessary for use as a detection marker, differences in the pattern of expression between histologic types suggest that MAGE expression may have utility as differentiation markers. This utility is also supported by the observation mat, in 50% of squamous cell carcinoma greater than 90% of tumor cells showed evidence of MAGE-3 overexpressipn with 30% to tumors exhibiting overexpression in at least 50% of cells.<br>
Nucleolar Protein (p120) [157]<br>
p120 (proliferation-associated nucleolar antigen) is found in the cells of nucleoli of rapidly proliferating cells during early Gl phase. The cellular localization of p120 is nuclear.<br>
Nucleolar protein pl20 is a proliferation-associated protein whose function has yet to be elucidated. Strong staining has been detected in tumor tissue but not in macrophages or normal tissue. Overexpression of p120 was more common in squamous cell carcinoma that in adenocarcinoma or large cell carcinoma raising the possibility that this marker may have utility in discriminating between tumor types.<br>
Pulmonary Surfactants [83,158-166]<br>
Pulmonary surfactants are a phospholipid-rich mixture that functions to reduce the surface tension at the alveolar-liquid interface, thus providing the alveolar stability necessary for ventilation- Surfactant proteins appear to be expressed exclusively in the airway and are produced by alveolar type II cells. In the non-neoplastic lung, pro-surfactant-B irnmunoreactivity is detected in normal and hyperplastic alveolar type II cells and some non-ciliated bronchiolar epithelial cells. Sixty percent of adenocarcinomas contained strong cytoplasmic irnmunoreactivity with 10-50% of tumor cells exhibiting staining the majority of cases. Squamous cell carcinoma and large cell carcinoma failed to stain for pro-surfactant-B.<br>
Surfactant Apoprotein B (SP-B) is one in four hydrophobic proteins that make up the pulmonary surfactant, which is a phospholipid and protein complex secreted by type II alveolar cells. Squamous cell and large cell carcinomas of the lung and nonpulmonary adenocarcinomas do not express SP-B. The cellular localization of SP-B is cytoplasmic.<br>
SP-A is a pulmonary surfactant protein that plays an essential role in keeping alveoli from collapsing at the end of expiration. SP-A is a unique differentiation marker of pulmonary alveolar epithelial cells (type II pneumocytes); the antigen is preserved even in the neoplastic state. The cellular localization of SP-A is cytoplasmic.<br>
Pulmonary surfactant A appears to be specific for non-mucinous bronchoiolo-alveolar carcinoma with 100% staining as compared to none of the of mucinous type. Pulmonary surfactants potentially have utility in discriminating lung cancer from other<br>
cancers metastasized to rung. In addition to tumor cells, non-neoplastic pheumocytes also stain for pulmonary surfactant A. As with pulmonary surfactant B staining for pulmonary surfactant A is relatively common in adenocarcinoma but not in other forms of NSCLC or m SCLC. Mesothelioma also fails to express pulmonary surfactant A leading to the suggestion that pulmonary surfactant A may" have utility in the discrimination between adenocarcinoma and mesothelioma.<br>
Ki-67<br>
Ki-67 is a nuclear protein that is expressed in proliferating normal and neoplastic cells and is down-regulated in quiescent cells. It is present in Gl, S, G2, and M phases of the cell cycle, but is absent in Go phase. Commonly used as a marker of proliferation. The cellular localization of Ki-67 is nuclear.<br>
Table 5:<br>
(Table Removed)<br>
+ percent of tumors exhibiting a change in marker expression * No Data Available<br>
a.        Obtaining a Library of Marker of a Suitable Size<br>
Preliminary pruning steps were required in order to obtain a suitable size library of markers that were correlated with lung cancer. More than a hundred markers correlated to lung cancer are known in the literature. A partial listing of candidate probes identified in the literature and evaluated for potential inclusion in panels tests include antibodies to: bax, Bcl-2, c-MET (HGFr), CD44S, CD44v4, CD44v5, CD44v6, cdk2 kinase, CEA (carcino-embryonic antigen), Cyclin A, Cyclin Dl (bcl-1), E-cadherin, EGFR, ER-related (p29), erbB-l, erbB-2, FGF-2 (bFGF), FOS, Glut-1, Glut-2, Glut-3, Glut-4, Glut-5,<br>
HERA (MOC-31), HPV-16, HPV-18, HPV-31, HPV-33, HPV-51, integrin VLA2, integrin VLA3, integrin VLA6, JUN, keratin, keratin 7, keratin 8, keratin 10, keratin 13, keratin 14, keratin Id, keratin 17, keratin 18, keratin 19, A-type larnins (A; C), B-type larnins (Bl; B2), MAGE-1, MAGE-3, MAGE-4, melanoma-associated antigen clone NKI/C3, mdm2, mib-1 (Ki-67), mucin 1 (MUC-1), mucin'2 (MUC-2), mucin 3 (MUC-3), mucin 4 (MUC-4), MYC, N-cadherin, NCAM (neural cell adhesion molecule), nm23, p120, p16, p21, p27, p53, P-cadherin, PCNA Retinoblastoma, SP-A, SP-B, Telomerase, Thrombomodulin, Thyroid Transcription Factor 1, VEGF, vimentin, and wafl. The initial list of markers was pruned by initially assessing, from the literature, the apparent effectiveness of the probes in detecting early stage cancer cells, discriminating between cells of differing cancer states, and localizing the label to the target cancer cells.   This list of markers was further pruned by removing markers whose utilization would be difficult to reduce to practice because they are difficult to produce or obtain, have unsuitable detection technology requirements or poor reproducibility of reported results. After all of the pruning steps were complete, a library of 27 markers was obtained.<br>
b.        Optimizing Protocols and Obtaining Gold Standard Lung Cancer Samples<br>
Preliminary preparation steps were also required prior to obtaining the panels. The probes containing appropriate labels were available from commercial vendors. The protocols of the probes were analyzed for optimum objective quantitative detection. For example, it was determined that the concentration of PCNA was too low. Originally, PCNA was diluted 1:4000 in S809 buffer. A second dilution was made, which was 1:3200 in S809. The optimized protocols for each marker is shown in below. It is noted that the second column is labeled "Antibody Name". Except for MOC-31, the probes in this list are listed by the marker name because many of the vendors refer to the antibody by the name of the marker. It is noted that an alternative way these reagents might be listed is, for example, anti-VEGF, ann-Thrombomodulin, anti-CD44v6, etc.<br>
Gold standard tissue specimens were obtained from UCLA Tissue specimens were received from two sources. Cases had been diagnosed using standard procedures including review of hematoxylin and eosin (H&amp;E)-stained slides and the clinical history . Specimen slides were coded and labeled with arbitrary numbers to blind the study pathologists to the historical diagnosis and antibody marker and to protect patient confidentiality.<br>
Specimen slides with tissue sections from cancerous and non cancerous (control) tissues were used. A total of 175 separate cases were analyzed. Within this set, the following diagnoses, located in Table 6 were present with the following frequencies:<br>
Table 6:<br>
(Table Removed)<br>
c	Determination of the Level of Expression of the Panel of Molecular<br>
Markers<br>
Sufficient specimen slides were prepared for each case so that only one probe was tested per slide.   In general, a microscope slide is prepared which contains the cytologic sample contacted with one or more labeled probes that are directed at particular molecular markers. Independently, each study pathologists examined an H&amp;E-stained slide to make a diagnosis for each case, and then examined each probe-reacted and imrnunochemically-stained slide to assess the level of probe binding, recording the results on a standardized data form.<br>
In greater detail, the immunohistochemical staining was performed on formalin fixed, paraffin embedded (FFPE) tissue. Tissue sections were cut at 4 microns thick on poly-L-Lysine coated slides and dried at room temperature overnight. De-paraffinization and rehydration of the tissue sections were performed as follows: To completely remove all of the embedding medium from the specimen the slides were incubated in two consecutive Xylene-substitute (Histoclear) baths for five minutes each. All liquid was tapped off the slides before incubation in two consecutive baths of 100% reagent grade alcohol for three minutes each. Once again all excess liquid was tapped off the slides before being incubated in two final baths of 95% reagent grade alcohol for three minutes each. After the last bath of 95% the slides were rinsed in tap water and held in wash buffer (Tris-buffered saline wash buffer containing 0.05% Tween 20 corresponding to a 1:10 dilution of DAKO Autostainer<br>
Wash buffer, code S3306). Table 7, below, presents a complete list of the reagents used in this study along widi corresponding product code numbers. Detection systems used in the study were DAKO EnVision+ HRP mouse (code K4007) or rabbit (code K4003) and LSAB+ HRP (code K0690). The protocols for irnraunoassaying were followed according to the package inserts. The kits contained liquid two component DAB+ substrate chromogen (code K3468).<br>
Table 7:             (Table Removed)<br>
Pretreatments were critical in optimizing these" antibodies on lung tissue. For antibodies requiring enzyme digestion, DAKO Proteinase K (code S3020) was used for 5 minutes at room temperature. Antibodies requiring heat induced target retrieval received pretreatment using either DAKO Target Retrieval Solution (code SI700) or DAKO High pH Target Retrieval Solution (code S3307). Tissues were placed in a pre-heated Target Retrieval Solution and incubated in a 95°C water bath for 20 or 40 minutes depending on the specific protocol. Tissue sections were then allowed to cool at room temperature for an additional 20 minutes.<br>
After de-paraffinization, rehydration and tissue pretreatment, all specimens were incubated in a solution of 3% hydrogen peroxide to quench endogenous peroxidase<br>
activity. Blocking reagents were used specifically for the two antibodies FGF and Telomerase in order to minimize nonspecific background.<br>
As shown in Table 8, below, tissue specimens were incubated for a specified length of time with 200 micro liters of the optimally diluted primary antibody. It is noted that the numbering of the markers/antibodies in Table 8 is consistent with the numbering of the antibody probes and markers throughout this document. Slides were then washed in DAKO IX Autostainer Buffer (code S3306). Depending on the antibody, the correct detection system was applied. The steps and total incubation times for the DAKO EnVision+ HRP and LSAB+ HRP detection systems are shown in Table 9, below. The color reaction is developed using 3,3'-diaminobenzidine (DAB) resulting in a brown color precipitate at the site of the reaction.<br>
Table 8: <br>
(Table Removed)<br>
Table 9:<br><br>
(Table Removed)<br>
Following immunostaining all shdes were incubated in DAKO Hematoxylin (code S3302) for 3 minutes and coverslipped using DAKOMount Mounting Media (S3025). All protocols were run on DAKO Autostainers (serial #'s 3400-6612-03 &amp; 3400-6142R-03) using the IHC software version 3.0.2.<br>
Immunostaining was viewed under a light microscope to determine that controls were correctly stained and tissues were intact Slides were labeled, boxed and sent to designated pathologists for results interpretation. Trained pathologists identified the type of cancer or other lesion seen in the samples. Trained pathologists assessed the sensitivity to the marker probe by estimating the staining density and proportion of cells stained. These scores were entered in a data sheet for that patient. The pathologists were blinded to the original diagnosis and antibody marker used in the immunostaining. Each slide was read by at least two pathologists and results recorded on a data collection form. To provide additional integrity to the process, the method is repeated with a second or third pathologist. The scores obtained can then be matched to identify data entry errors. The additional data also facilitates a better classifier design.<br>
For each case, up to 27 slides were analyzed, each stained for a marker coded with numbers 1 through to 17,19 through to 28. Staining for marker 18 (C-MET) could not<br>
be optimized and the marker/probe was therefore not used. Pathologist 1 scored slides from all 175 cases. Pathologist 2 scored slides from 99 of the cases. Pathologist 3 scored slides from 80 of the cases.<br>
Table 10 below shows how many cases of each diagnosis each pathologist scored slides from:<br>
Table 10:<br>
(Table Removed)<br>
For the purposes of some selected statistical analysis techniques, it was necessary to consider only those cases that had scores for all 27 slides present. Table 11 below shows how many cases of each diagnosis were complete in terms of having scores from all 27 slides. Table 11:<br>
(Table Removed)<br>
From this table, it can be calculated that each pathologist scored the following total number of complete cases. Pathologist 1 scored all 27 slides for 119 of the cases Pathologist 2 scored all 27 slides for 83 of the cases. Pathologist 3 scored all 27 slides for 23 of the cases.<br>
The total number of cancer data points is 172. This comprises 113 data points from Pathologist 1 and 60 data points from Pathologist 2. The total number of control data<br>
points is 101. This comprises 62 data points from Pathologist 1 and 39 data points from Pathologist 2.<br>
Figure 3 shows a comparisons between H-scores for probes 7 and 15 in control tissue and in cancerous tissue. The x-axis shows the H-scores while the y-axis shows the percent of cases with that particular H-score. The difference in H-scores is apparent.<br>
For each, patient the scores were entered electronically into a Pathology Review Form which consolidates the scores into a data base showing the patient identifier together with diagnosis, proportion of cells stained, and staining density. The proportions and density were consolidated into a single "H-Score" obtained by grading the intensity as: none = 0, weak = 1, moderate = 2, intense = 3, and the percentage cells as: 0-5% = 0, 6-25% = 1, 26-50% = 2, 51-75% = 3, &gt;75% = 4, and then multiplying the two grades together. For example, 50% weakly stained plus 50% moderate stained would score 10 - 2x2 + 2x3. This is the standard scoring system throughout the analysis, except for the section 3(f), below, titled "Effect of Using other (non-H-score) objective scoring parameters", which investigates alternative scoring systems.<br>
Standard classification procedures were used to find the best combination of probes. Typically these use a search procedure such as the "Branch and Bound Algorithm" to find a hierarchy of the best features, Tanked according to a test of mscriminating power, and truncated according to a test of significance. This process also defines the decision rule or rules for best classification.<br>
The performance of a classifier designed with these features can be estimated from the data used to design the classifier. The straightforward application of all the design data to the classifier gives a very unsound estimate of performance.<br>
The analysis of the data collected in the present example provide the optimum selection of probes which provided the best separation of classes. Therefore, panels were obtained that only needed a few probes to perform the analysis. However the data showed that near-optimum performance could be obtained with other combinations of probes. Hence, the invention is flexible in being adaptable to the availability of probes where cost or supply problems may not allow the very best combination. In some cases, the invention can simply be applied to the available features to find an alternative combination. In other cases, the algorithm may be used to select features which allows cost weightings to be included in the selection process to arrive at a low cost solution.<br>
The design of data collection and analysis experiment was chosen to avoid biases through the well established double blind procedures where data collection and data analysis were done independently.<br>
In the first case the pathologists reviewed shdes with conventional staining to allow a diagnosis to be made. This diagnosis was entered on the Pathology Review form. The pathologists were then presented, in random order, with shdes stained by the marker probes for scoring the percentage of cells stained and the relative intensity of the staining. The slides were numbered to exclude information about the probe from the pathologist. To allow data integrity to be checked two pathologists reviewed all patients.<br>
Data were consolidated into a database that was then reviewed by a team of statisticians. Probes were numbered to render their method of action as unseen during the analysis of their effectiveness.<br>
The first stage of the analysis was to check the integrity of the data by comparing entries for each patient. Where large differences were found, the data entries were checked and any obvious errors weTe corrected. Unexplained differences were left in the data.<br>
The data were then separately analyzed by four statisticians, using different techniques in recognition of the fact that different statistical methodologies are suited to different types of Discriminating information in the data.<br>
The first step in the process of selecting the best probe combination is to divide the data into two sets, one for designing a classifier and one for testing the performance of the classifier. By selecting the design made with the design (train) set, but showing the best performance evaluated on the test set, it can be concluded with confidence that the classifier has generalized to the structure of the data and not adapted to particular cases seen in the training set.<br>
In order to test for reliability the analysis was typically repeated with many randomly selected sets of training data and test data. This approach is generally accepted as giving good estimates of the classifier performance. Where these tests showed inconsistent selections of probes such probe selections were discounted as unreliable.<br>
d.        Statistical Analysis and/or Pattern Recognition<br>
1.        Introduction to Data Analysis<br>
a.        Input Data<br>
i.	Raw data<br>
For each patient fee scores were entered electronically into a Pathology Review Form that consolidates the scores into a database showing the patient identifier together with diagnosis, proportion of cells stained, and staining density, ii.       Computed datar<br>
The efficiency of the score for each probe used in the analysis is computed from the intensity/percentage tables. The proportions and density are consolidated into a single "H-Score" with a simple rule H= proportion stained x (3 if intense+2 if moderate* 1 if weakly stained). This is the feature value associated with that probe, iii.       Alternative computed data parameters<br>
The H-score described above was heuristically derived, a simple analysis to find a better way of combining percentages and intensity failed to show a significant improvement over H-score (Section 3(f), titled "Effect of Using other (non-H-score) objective scoring parameters"). A larger data base may allow the extraction of a better rule in future.<br>
iv.       User supplied weiehting criteria per marker<br>
The invention is flexible in being adaptable to the availability of features where cost or supply problems may not allow the very best combination. For example, the invention can simply be applied to the available features to find and alternative combination. Alternatively, the algorithm used to select features allows cost weightings to be included in the selection process to arrive at a minimum cost solution. Marker performance estimates are shown for combinations selected from all the markers collected or only those from one supplier. It is also shown how the C4.5 package can be used to down weight certain probes, say on the basis of their high cost. These probe combinations do not perform as well as the optimum combination, but the performance might be acceptable in circumstances where cost is a significant factor.<br>
v.        User supplied weighting criteria per class<br>
Some of the methods used allow weightings to be applied to the classes. This is available in C4.5 where the tree design can optimize the cost. Also the Discriminant Function method gives a single parameter output which can be used to give a desired false positive or false negative probability. A plot of these parameters for different threshold settings is known as the Receiver Operating Curve.<br>
vi.     Detection  panels - assumptions<br>
A low probability of a false negatives was assumed to be desirable for the cancer detection process (to avoid positive patients being missed at the cost of an increased number of false positives who would require re-screening). It was also assumed that the cancer discrimination process would require a lower false positive score (to minimize patients receiving the wrong treatment).<br>
It was assumed that detection panels requiring 6 or more probes to achieve an acceptable performance would not be cost effective. It was also assumed that a detection panel with a false negative error rate of more than 5% would not be acceptable. Panels falling outside this box are not accepted. This assumption acknowledges that cytometric panels are likely to have a worse performance than the histology based panels analyzed here. The ultimate aim will be a cytometric panel which performs better than 20% error rate, this being approximately the performance of cervical PAP smear screeners.<br>
vii.       Discrimination panels - assumptions<br>
It was assumed that panels requiring 6 or more probes are not cost effective and it was assumed that an error rate of better than 20% is required. Panels falling outside this box were not accepted.<br>
b.        Output data<br>
Outputs provided by the present analysis included: Confusion Matrices, showing how data from the test set was classified as either true positive, false positive, true negative or false negative. These may be shown as actual counts or as percentages. Confusion matrices are discussed in section 2(d) titled 'Terformance Metrics". A confusion matrix shows how data from a test set was classifiefd as either true positive, false positive, true negative or false negative. An exemplary confusion matrix, obtained from data analyzed by decision trees, is shown below in table 12 for simultaneous discrmination of adenocarcinoma, squamous cell carcinoma, large cell carcinoma, mesothelioma and small cell carcinoma<br>
Table 12 (Table Removed)   <br>
•	Error Rates, summarizing data in the confusion matrix as the sum of all false classifications divided by the total number of classifications made expressed as a percentage<br>
•	Receiver Operating Characteristic (ROC) curves show the estimated percentage (or per unit probability) of false positive and false negative scores for different threshold levels in the classifier. An indifferent classifier, unable to discriminate better than random choice would present a ROC curve with equal true and false readings. The area under this curve would be 50% (0.5 probability).<br>
•	Area Under the Curve (AUC) is often used as an overall estimate of classifier performance and most standard discriminant function packages provide this AUC figure. A perfect classifier would have 100% Area Under the Curve, and a useless classifier would have an AUC near 50% (0.5).<br>
•	Sensitivity and specificity (can be derived from the confusion matrix). See section 2(d)(iii) titled "Sensitivity and Specificity".<br>
•	Marker correlation matrices. See Figure 4.<br>
i.	Detection panels: composition<br>
These panels are trained on data divided into two classes, patients with any of the five cancers and patients with none of the cancers. Not all probes were present for all patients. Where one or more probes were missing for a particular analysis these cases were excised from the data. Hence, where analysis was undertaken on reduced numbers of probes die data set might include slightly more cases.<br>
The number of probes included in the analysis was 27. Although in many cases a false probe was added where the data entered for that probe was from a random number generator set to generate numbers uniformly between zero and 12. This false probe<br>
was included in much of the early analysis to ensure integrity in the probe selection process. This false probe was also used in one approach to progressively eliminate probes from the analysis. Probes that contributed less information than the false probe could be readily identified and excluded from the selection process. Early elimination of such probes speeds the analysis and renders the analysis less vulnerable to variations in results (noise) caused by these probes.<br>
ii.        Detection Panel Performance<br>
As outputs from this study, the probe combinations selected by the different methodologies and their performance estimates in terms of the confusion matrix, % error rate, and AUC are reported.<br>
iii.       Detection Panels - alternative compositions<br>
Detection panels were also selected from reduced sets of probes. In one set of panels, performance measures of panels weighted for commercially preferred markers were obtained. The performances obtained when the best probe was removed from the analysis to find a new combination of discriminating probes was also analyzed. The performance of a single probe acting on its own was found to be very high (probe 7). However, as shown below in the performance diagrams, Table 13, evaluated using linear discriminant analysis, the performance was improved as more markers were added. The best subsets of probes were determined using best subsets logistic regression. The improvement is statistically significant.   (Table Removed)<br><br>
Table 13:<br><br>
The best and second best subsets of probes (determined using best subsets logistic regression) and evaluated using logistic regression is shown below. AUC=Area under ROC curve. It is noted that mean AUC is the average from 100 trials on random train and test partitions (70%:30%). The results are shown below, in Table 14.<br>
T«ble 14:<br>
(Table Removed)<br>
iv.       Discrimination panels - composition<br>
For this part of the study five classifiers were designed and tested, each designed to detect the presence of one of the cancer from all patients with cancer. The application of this five way pair-wise system allows doubtful cases to appear more than once in the analysis, or not at all. Such cases can be identified and subjected to closer scrutiny, re-testing or alternative testing regimes.<br>
Again the number of probes in the study was 27, with a false probe used in the early stage to reduce the numbers in the analysis<br>
V.	Discriminant panels - performance<br>
The performance estimators described above were used to show the performance of the best probe combinations discovered by the different techniques vi.       Discriminant panels — alternative composition<br>
The analysis was repeated for a probe combination comprising commercially preferred probes. Performance was degraded, but not unusable for several reduced-set classifiers. Below, the best subsets of probes without probe 7, determined using best subsets logistic regression), is shown, as Table 15. The data was evaluated using linear disciiminat analysis.<br>
Table 15:<br>
(Table Removed)<br>
The best and second best subsets of probes with probe 7 (determined using best subsets logistic regression) and evaluated using logistic regression is shown below. AUC=Area under ROC curve. It is noted that mean AUC is the average from 100 trials on random train and test partitions (70%:30%). The results are shown below, in Table 16. Table 16:<br>
(Table Removed)<br>
2.        Data analysis methodology<br>
In this section, the process of gaining an initial understanding of the structure of the data as a guide to interpreting results^ from the different methodologies used is<br>
described.<br>
a.        Analysis of variance<br>
i.	Pathologist-to-pathologist variability and pooling pathologist<br>
scores.<br>
(1)      t - Test<br>
Two pathologists reviewed each patient's slides in this clinical trial. Pathologist 1 reviewed all patients, Pathologist 2 also reviewed approximately half of this set and Pathologist 3 reviewed the remainder. With two independent estimates of the H-score, the consistency of pathologist performance could be tested.<br>
A readily available statistical tool was used to test the variability between pathologists. This is the paired-sample t-test. This takes the difference between each pair of estimates, averages these and expresses this as a proportion of the overall variances. The t-test then converts this ratio into a probability estimating the likelihood that the two samples sets came from the same population (the P value).<br>
This test was applied to the scores for each marker probe, for all cases reviewed by Pathologist 1 and Pathologist 2, and also for all cases reviewed by Pathologist 1 and Pathologist 3. Since there were 27 tests applied (to cover all probes) a low value of P=0.01 was selected as the "significant threshold". Results, showing the P scores for each probe, and for the two pairs of pathologists, are shown below, in Tables 17,18,19 and 20. It is clear that Pathologist 1 and Pathologist 2 were more consistent than Pathologist 1 and Pathologist 3.<br>
Table 17:<br>
Pathologist 1, Pathologist 2 scores:<br><br>
(Table Removed)<br>
Table 18:<br>
Pathologist 1, Pathologist 2 scores thresholded at 0.01 (a = 1% level of significance):<br>
(Table Removed)<br>
Table 19:<br>
Pathologist 2, Pathologist 3 scores:<br>
(Table Removed)<br>
Table 20;<br><br>
(Table Removed)<br>
Because the H score is subjective it isprone to scale factor differences and noise at marginal cases. So, in spite of the three feaores which showed statistically different scores between Pathologist 1 and Pathologist 2, this Joint data was accepted as representative of a measuring instrument Pathologist 1 and Pathologist 2 were combined into a single data set for the analysis process. The results for Pathologist 3 were withheld for independent testing purposes. Such tests using the Pathologist 3 iota, would be biased towards showing an under-performance because of the significant diffenoces.<br>
The data from Pathologist 1 and Pathologist 2 were combined by considering them as separate cases, with the variability giving a degree of independence between the results for any one case. When testing with such datathe performance estimates will be biased towards a more optimistic value. This is becaase samples coming from the same patient may occur simultaneously in the training a tetii subsets. This does not however invalidate the processes used to find the best combinafon of features, it merely biases the estimate of performance.<br>
(2)       Analysis ofVarmtce qfHScores<br>
(a)       Backgroand<br>
Within each probe, the H-scores may iary due to many reasons. To the extent they vary consistently due to the type of disease this kuseful, variation due to which pathologist read the slide is instructive, whereas randan variation sets a limit on the detection of the previous two sources of variation.<br>
Analysis of Variance (ANOVA) is a staadard technique for splitting up the sources of variation in data and for testing its statistic^ significance. ANOVA summarizes<br>
the total variation of a set of data as a sum of terms which can be attributed to specific sources, or causes, of variation.<br>
ANOVA is available in many statistical packages. The public domain package "R" was chosen ("The R Project for Statistical Computing", http://www.R-project.org/).<br>
(b)	Aim<br>
■<br>
To perform ANOVA analyses on the H-score data from pathologists 1 and 2 and to consider whether this data can be safely merged into a single consistent set for further analysis for the selection of panels.<br>
(c)	Methodology<br>
From the database, data was selected from pathologists 1 and 2. Only data which was complete for a given probe was used in the ANOVA for that probe.<br>
The control categories of Emphysema, Granulomatous Disease, and Interstitial Lung Disease were grouped together and called "Normal" giving 6 levels within factor Disease.<br>
Pathologist was coded as a factor with 2 levels (Pathologist 1, Pathologist 2).<br>
An R script was written to perform a standard ANOVA analysis for each probe in turn, using the factors: Disease, Pathologist, and the interaction term Disease:Pathologist. The results are shown in below, in Table 21.   "Df' is defined as the degrees of freedom. In a dataset of n observations, knowing n-1 deviations from the mean, the nth is automatically determined. N-1 is the number of degrees of freedom. Sum Sq and mean Sq are measures of variation. F is a test statistic concerning the equality of two variances based on the F distribution. Pr(&gt;F) is the probability used to determine whether or not the variability is statistically significant.<br>
Table 21:<br>
Analysis of Variance of H-Scores<br><br>
(Table Removed)<br>
 (d)	Analysis of Results<br>
In all cases (except for probe 21) the response of the probes was related to disease. This is not surprising since the probes have presumably been selected for this purpose. In no case is the response of the probe related to pathologist (at the p=0.05 level). This indicates that it would be safe to merge this data and use the two pathologists as two measurements on the data<br>
In a few cases, probes 7,14,17, there is some evidence of an interaction term gaining significance. This indicates that there may be some difference between pathologists in their scoring of some diseases. Some of these cases may well be due to an occasional outlier in the data.<br>
(e)	Conclusions<br>
The results indicate that it is safe to merge this data for further analysis. The data indicate that the slight interactions in some cases between pathologist and disease appear to be attributed to random sources.<br>
ii.        Patient to patient variability<br>
The variability from patient to patient was measured by the disease: disease variability of section 2(a)(i)(2) (see above, "Analysis of Variance of H-Scores"). iii.       Marker-to-marker variability<br>
Histograms were plotted (PathologisfData.xls, worksheet: Histograms) showing the distribution of marker scores for each probe for Control vs. Cancer.<br>
b.	Marker correlation matrix analyses<br>
The population correlation coefficient ("Applied Multivariate Statistical Analysis", R. A. Johnson and D. W. Wichem, 2nd Ed,1988, Prentice-Hall, NJ.) measures the amount of linear association between a pair of random variables. Typically the distributions and associated parameters of the random variables are not known and the population correlation coefficient cannot be directly computed. In this case it is possible to compute the sample correlation coefficient from sample data. See Figure 4. The sample correlation coefficient is, however, only an estimate of the population correlation coefficient. Moreover, because it is calculated on the basis of sample data it is possible, purely by chance, that it may indicate a strong positive or negative correlation when in reality there may be no actual relationship between the corresponding random variables ("Modem Elementary Statistics", J. E. Freund, 6th Ed, 1984, Prentice-HallrN.J.).<br>
The correlation coefficient measures the ability of one variable to predict the other. A strong linear association does not, however, imply a causal relationship. The square of the correlation coefficient is called the coefficient of determination. The coefficient of determination computed for a bivariate data set measures the proportion of the variability in one variable that can be accounted for by its linear relationship to the other. When dealing with several variables, the correlation coefficient can be calculated for each pair in turn and the set of coefficients can be written as a matrix called the correlation matrix. See Figure 4.<br>
The H-scores for the individual markers can be modeled as random variables. The sample correlation matrix for this multivariate data set can be computed from the input data described in the section titled "Input Data", above.<br>
c.	Pattern recognition<br>
Statistical partem recognition is an approach to classifying signals or geometric objects on the basis of quantitative measurements (called features). Statistical pattern recognition essentially reduces to the problem of dividing the n-dimensional feature space into regions that correspond to the categories or classes of interest.<br>
Three different classifier methodologies employed in this study are sensitive to different structural forms within the data.<br>
For the Decision Tree method a preliminary analysis of different data combinations identified markers which were never used by C4.5 for the detection panel. These were removed from the analysis and this resulted in more consistent results, symptomatic of the left-out probes only contributing noise to the selection process.<br>
Similarly a preliminary analysis of probes used in the detection panels identified the noisy probes for removal prior to the detailed analysis.<br>
The Linear Discriminant Function method in SPSS has built-in stepwise processes for reducing the numbers of markers in the analysis. Typically, this reduced the probes used in the analysis to between 2 and 7.<br>
The Logistic Regression method in R and SAS implement stepwise procedures for variable selection. In SAS, a best subsets variable selection option is also provided. In R, the stepwise methodology was used in conjunction with multiple random trials to develop a heuristic method for selecting variables based on the number of times a given feature was used in 100 random selections of training and test data (split 70%:30% respectively). Features with counts comparable to the count for artificial random feature were progressively eliminated until a minimal consistent set of features was obtained over 100 runs.<br>
i.	Statistical methods<br>
From the point of view of multivariate statistical analysis, the problem is one of estimating density functions in high-dimensional space (and partitioning this space into the regions of interest). Assuming that the distributions of random (feature) vectors are known, the theoretically best classifier is the Bayes classifier because it rninirnizes the probability of classification error (K. Fukunaga, "Statistical Pattern Recognition", 2nd Ed., Academic Press 1990, p.3). Unfortunately the implementation of the Bayes classifier is difficult because of its complexity, especially when the dimensionality of the feature space is high. In practice, simpler parametric classifiers are used. Parametric classifiers are based on assumptions about the underlying density or discriminant functions. The most common such classifiers are linear and quadratic classifiers. In multivariate statistical analysis such classifiers fall under the heading of discriminant analysis. Discriminant analysis techniques are closely related to multivariate linear regression models and generalized linear models (encompassing logistic and multinomial regression).<br>
(1)       Logistic Regression with a Binomial Response<br>
(a)	Background<br>
The problem of selecting a set of markers to be used on a detection panel can be formulated as a logistic regression problem with a binomial response. The response variable is a factor with two levels: normal (no cancer) and abnormal (cancer). The explanatory variables are the marker H-scores.<br>
The problem of selecting a set of markers to be used on a cancer discrimination panel can also be formulated as a logistic regression problem with a binomial response. The response variable is a factor with two levels: normal (not the cancer of interest) and abnormal (cancer of interest). The explanatory variables are the marker H-scores.<br>
Stepwise variable selection can be used to select a subset of the original variables (markers) for use in discriminating between the two classes. This is a computationally expensive exercise and is best suited to a computer. Several commercial and public domain software packages—e.g., R, S-plus, and SAS—implement stepwise logistic regression.<br>
Two different approaches to feature selection were investigated based on the stepwise variable selection procedures found in R and SAS respectively.<br>
(b)	Experimental data<br>
The data used for the present analysis consists of the H-scores for markers 1-17, and 19-28 for the cases examined by Pathologist 1 and Pathologist 2 and described elsewhere in this report. In addition, a dummy marker, 18, was added to the data set. The dummy marker consists of integer values from 0 to 12 selected at random from a uniform distribution.<br>
(c)	Method 1: Using the R package (version 1.4.1)<br>
Computerized model fitting procedures generally cannot deal with missing data. This is the case for the glm (glm stands for generalized linear model) procedure used in R. Consequently when fitting a model using glm it was necessary to exclude all the cases for which there are one or more missing values . When fitting the initial full model, containing the 27 real markers and the single dummy marker, this reduces the data set to only 202 cases. With so few observations it was decided that the best way to perform variable selection, to<br>
train a classifier using the selected variables, and to assess its performance was to undertake 100 trials on random partitions of the data into train and test sets.<br>
_   (i)       Partitioning the data into train and test sets<br>
At the start of each trial, the data is partitioned into a test set and a training set. This is done by randomly choosing 30% of the abnormals and 30% of the normals to form the test set, and using the remaining observations to form the training set.<br>
(ii)       Variable (marker) selection<br>
At the start of each trial, the full model, which includes all of the variables (markers), is fitted to the training data. In R the logistic regression model is fitted using glm. The code fragment used is as follows:<br>
my.model 
my.glm 
The procedure stepAIC is then used to perform stepwise variable selection based on the Akaike Information Criterion (AIC). This procedure is part of the publicly available MASS library. The library and the procedure are described in "Modem Applied Statistics with S-PLUS" (W. N. Venables and B.D. Ripley, Springer-Verlag, Pathologist 3ew York, 1999). The R code fragment to do this is as follows:<br>
my.step   
The resulting model is then assessed on the test data. The code fragment used is as follows:<br>
probability_is_abnormal 
predict (my. step, testing.data, type=# response# )<br>
The performance of the classifier is recorded in terms of the actual error rate of misclassification (AER) and the area under the ROC curve (AUC). After the 100 trials, 100 models and their associated AERs and AUCs remain. A frequency table is constructed, recording the number of times each variable made an appearance in the 100 models. An example is shown in Table 22:<br>
Table 22:<br><br>
(Table Removed)<br>
This table is used to decide which markers to discard. First, all of the markers that have a frequency less than or equal to 10 are discarded. Next a cut-off frequency is chosen based on the frequency of the dummy marker (typically this is 1 or 1.5 times that of the dummy marker). All markers with a frequency less than this cut-off value are discarded. The remaining markers, along with the dummy marker, are then used as the full model for another 100 trials and the pruning process is repeated. If necessary, the severity of the pruning can be increased to force one or more markers out of the model. If necessary, the remaining markers can be used as the full model for yet another 100 trials. Pruning stops when the desired number of panel members is reached or the average AUC for the current model is less than that for the preceding model.<br>
To illustrate the pruning process consider the table above. The table was obtained using the detection panel data. The shaded entries indicate those markers that are retained after pruning. Another 100 trials is performed using the following full model:<br>
ray.model  
Again, a frequency table, Table 23 is constructed: Table 23:<br>
(Table Removed)<br>
The shaded entries show the markers retained after pruning (using a cutoff of 47). Another 100 trials is performed using the following full model:<br>
my-model   
Again, a frequency table, Table 24 is constructed:<br><br>
(Table Removed)<br>
At this point a cut-off of 50 is chosen. The shaded entries show the remaining markers for use on a 5 member panel. In each step, the average AUC increases: 94.37%→ 95.45% -» 95.78%.<br>
(iii)     Assessing the performance of the panel<br>
To assess the performance of the panel, 100 trials were performed, as before, but without the stepwise selection procedure. For each trial, the AUC, sensitivity, and specificity are recorded. For the detection panel example above, the results are:<br>
&gt;	my .model 
&gt;	summary(AUC)<br>
Min. 1st Qu. Median   Mean 3rd Qu.    Max. 0.9289 0.9590 0.9615 0.9601 0.9630 0.9630<br>
&gt;	summary(sensitivity)<br>
Min. IstQu. Median   Mean 3rd Qu.    Max. 0.8519 0.9630 0.9630 0.9737 1.0000 1.0000<br>
&gt;	summary(specificiiy)<br>
Min. 1st Qu. Median   Mean 3rd Qu.    Max. 0.8378 0.9730 0.9730 0.9749 1.0000 1.0000<br>
In summary, the panel has a sensitivity of 97.37% and a specificity of 97.49%. The area under the ROC is 96.01%.<br>
(d)       Method 2: Using SAS (version 8.2)<br>
Logistic regression can be performed in SAS using the procedure LOGISTIC. When the response variable is a two-level factor, the procedure fits a binary logit model (equivalent to glm in R with family=binomial and link=logit). SAS automatically excludes all of the missing multivariate observations for the model specified. Unlike R, SAS is able to<br>
perform a best subsets variable selection procedure. The code fragment in SAS needed to do this is as follows:<br>
PROC LOGISTIC DATA=WORK.panel;<br>
CLASS Class;<br>
MODEL Class - XI X2 X3 X4 X5 X6 X7 X8 X9 X10 XI1 X12 X13 X14 X15 X16 X17 X18 X19 X20 X21 X22 X23 X24 X25 X26 X27 X28 /SELECTION-SCORE BEST-28;<br>
RUN;<br>
This procedure is applied to the entire data set. The parameter BEST=28 directs SAS to find the best 28 single-variable models, the best 28 two-variable models, the best 28 three-variable models, up to the best 28 28-variable models.<br>
(i)       Assessing the performance of the panels<br>
The procedure described in method 1 is used to assess the performance of each of the panels. The following, Table 25,was generated from the detection panel data. It lists results only for the two best one-, two-, three-, four-, and five-marker panels.<br>
Table 25:<br>
(Table Removed)<br>
 (2)       Linear Discriminant Analysis<br>
(a)       Background<br>
The commercial statistical package SPSS has procedures allowing simple linear discriminant functions to be design and tested.<br>
A commonly used method is Fisher's Linear discriminant function. This finds the hyper-plane in feature space which gives a good separation of classes. For a two class problem where the class distributions have different means, but similar multivariate Gaussian distributions, this classifier gives optimum performance. The method can be extended heuristically to multi-class problems, but this was not applied in the study.<br>
The method is simplistic in its approach but robust to problems associated with data sets containing a large number of features (the probes in our case number 27, giving problem for a data set comprising only some two hundred exemplars (cases)).<br>
This package has a procedure for identifying the features which contribute well to the discrimination process. This "stepwise method" first finds the most discriminating feature. Other features are then sequentially added and evaluated against the classifier. Combinations are explored so the final solution may exclude features initially selected if better combinations are found. The number of features is gradually increased until a statistical test shows the remaining features do not contribute reliably to the classification process.<br>
An estimate of the performance is gained by using the leave one out method. This removes one sample from the data set to form the training set The left out sample is retained as the test set, applied to the classifier, and the resulting classification accumulated in the confusion matrix. The procedure is repeated for case in the data. This procedure gives an unbiased estimate of performance, but the estimate will have a high variance.<br>
Method<br>
In SPSS select the appropriate data set for analysis, select "Analyze", select "Classify", select "Discriminant...", on the table select "Fishers method", "leave one out testing" and "use stepwise method". Enter the diagnosis as the grouping variable and enter all the features as the independents. Enter 'OK" to complete the analysis. Pre-set values for other parameters were left as set.<br>
The analysis output includes a list of the features used in the analysis, the canonical discriminant function and a confusion matrix and the correct-classification rate (1-error rate).<br>
In order to compute an ROC curve the Canonical discriminant function is applied to the selected features to generate a new feature. In SPSS use Graphs, ROC to plot this curve<br>
ii.        Hierarchical methods: Decision trees (I)       Background<br>
Decision tree learning is one of the most widely used and practical methods for inductive inference. It is a method for classification that is robust to noisy data and capable of learning disjunctive expressions (Tom M. Mitchell, "Machine Learning', McGraw-Hill, New York, NY, 1997.)<br>
The most popular and accessible machine learning package is "C4.5" the source code of which is published in: (J. Ross Quinlan, "C4.5: Programs for Machine Learning", Morgan Kaufmann, San Mateo CA, 1993).<br>
When a decision tree is being trained (on training data), the algorithm decides at each node of the tree which single attribute of the data to use at this node to best make a decision. Therefore when the tree is completely constructed, it will have selected some set of attributes to use and ignored others. In our application, using decision trees to process measurements gained from molecular probes, the decision tree has effectively chosen a panel of probes, and a method of combining the probe scores, which best explains the classification of the data. To obtain an unbiased estimate of the panel performance, the resulting tree must be evaluated on data which was not used in the training. One standard technique for doing this is cross-validation. A 10-fold cross-validation was employed.<br>
Cross-validation is a technique for making the very best use of limited data. In 10-fold cross-validation the data is randomly split into 10 nearly-equal sized partitions, taking care to have approximately the same number of cases in a class across each partition. Then, the decision tree is trained on partitions 2-9 combined and tested on partition 1, then trained on partitions 1,3-9 combined and tested on partition 2, and so on for 10 trials rotating the held-out test set through the data once. In this manner tests are only ever performed on held-out data and so are unbiased, and all data is tested exactly once so an aggregate error rate across the whole data set can be computed.<br>
Trees are usually constructed until they are a very good fit to the training data, then they are "pruned" back by clipping off "noisy" branches and leaves. This improves the<br>
generalization ability of the decision tree on unseen data and is essential to obtain good performance. The C4.5 package includes two methods for pruning trees first a standard tree pruning algorithm, second a rule extraction algorithm. In general, the tree based method was found to give superior results on this data. Therefore, the rule-based method is not reported.<br>
(2)       Data Preparation<br>
Data on the response of various probes to normal tissue and five different cancers (Adenocarcinoma, Large Cell Carcinoma, Mesothelioma, Small Cell Lung Cancer, and Squamous Cell Carcinoma) was obtained as described elsewhere. The H-scores for probes 1-28, and pathologists Pathologist 1 and Pathologist 2 were extracted from the database and put into a flat data file. For the decision tree analysis each data point (even by two pathologists on a same physical slide) was taken to be an independent observation of the effect of disease on staining. This may slightly positively bias the performance of classification but should have no effect on panel selection.<br>
•	The control categories of Emphysema, Granulomatous Disease, and Interstitial Lung Disease were grouped together and called " Normal".<br>
•	For the detection panel all the cancers were grouped together and called "Abnormal" making this a 2-class problem.<br>
•	For the single discrimination panel, the Normal cases were removed from the data to form a 5-class problem.<br>
•	For the hold-out discrimination panels, each cancer was held out in turn and the remaining cancers grouped into "Other" to give a set of five 2-class problems.<br>
C4.5 requires a ".names" file which describes the data and the attributes to be included in the analysis. An example names file for the discrimination panel is, Table 26:<br>
Table 26:<br>
(Table Removed)<br>
Probe 18 was missing from the data and was set to "ignore" in all the designs. Setting attributes to "ignore" in the names file is an easy and effective way of trimming probes from the panels and is used in the data analysis.<br>
(3)       Data Analysis<br>
Ten-fold cross validation was run on each data set using the "xval. sh" script supplied with C4.5. Standard (default) parameters for the package were used. Cross validation is a technique developed for classifier training and testing on small data sets. It involves randomly splitting the data into N equal sized partitions. The clasifier is then trained on N-l partitions together and tested on the rermariing partition. This is repeated N times.<br>
Since the decision tree trained in one cross-validation(CV) trial may differ from the tree obtained in another (different in both probes selected, and tree coefficients) the number of times each probe was selected by the tree in 10 trials was computed.<br>
The first cull of probes was done by setting to ignore any probe which did not occur in a pruned tree 5 or more times out of the 10 CV trials.<br>
Then the cross-validation was repeated with this smaller set of candidate probes. The second cull of probes was done by setting to ignore any probe which did not occur in a pruned tree 5 or more times out of the 10 CV trials. If any further probes dropped out, a diird CV run was done.<br>
The panels were selected by the various runs, and their estimated error performance are shown in the results tables. The panel performance for decision tree analysis is shown below, in Table 27. Table 27:<br>
(Table Removed)<br>
An example decision tree structure is shown in below, in Tables 28 and 29, for discriminating between Small Cell Lung Cancer and the remaining four types of cancer.<br>
C4.5 output format: Table 28:   (Table Removed)<br><br><br>
Table 29:<br>
Pictorial format: (Table Removed)<br>
 The panel performance for stepwise linear discrminant is shown below, in<br>
Table 30:-<br>
Table 30:<br><br>
(Table Removed)<br>
The panel performance for stepwise logistic regression analysis is shown below, in Table 31:<br>
Table 31: (Table Removed)<br><br><br>
Artificial neural networks ANN's are candidate pattern recognition techniques which could readily be applied to select features and design classifiers in association with this invention. However such techniques give little insight to the structure of the data and the influence of particular probes in the way that LDF gives. For this reason this class of algonthm was not used in this study. LDF stands for linear discriminant function, a linear combination of features whose result is thresholded to determine the classification.<br>
This class of techniques includes algorithms such as Multi-Layer Perceptron MLP, Back-Prop, Kohonen's Self-Organizing Maps, Learning Vector Quantization, K-nearest neighbors and Genetic Algonthms.<br><br>
iv.       Special topics<br>
(1)	Assumptions<br>
Linear discriminant analysis<br>
•	Assumes the covariance matrices for the two classes are equal.<br>
•	Minimizes the cost of misclassification only when the two classes are multivariate normal.<br>
•	Assumes that the explanatory variables are continuous rather than categorical (in this study, the H-scores are categorical while in practice (i.e., in an automated system) intensity can be measured on a continuous scale).<br>
Logistic regression (binomial generalized linear models)<br>
See Venerables and Ripley, chapter 7 ("Modern Applied Statistics with S-PLUS" (W.N. Venables and B.D. Ripley, Springer-Verlag, New York, 1999)).<br>
(2)	Marker rejection (de-selection)<br>
Computerized implementations of discriminant analysis and regression procedures include stepwise variable selection procedures; e.g., stepAIC in R These procedures are designed to select the best subset of variables for use as explanatory variables. In reality, because of the step-by-step nature of these procedures, there is no guarantee that the best variables are selected for prediction (Johnson and Wichem, p. 299). Nevertheless such procedures do provide the basis for marker selection and de-selection.<br>
(3)	Pairwise tests<br>
Inherent problems in designing multiclass classifiers is discussed in "Applied Mulitvaiiate Statistical Analysis", R. A. Johnson and D. W. Wichem, 2nd Ed, 1988, Prentice-Hall, N.J. This is motivation for developing several separate two-class classifiers (discrimination panel).<br>
(4)	Redundancy consideration in panel composition<br>
"Linear models form the core of classical statistics and are still the basis of much of statistical practice" "Modern Applied Statistics with S-PLUS" (W.N. Venables and B.D. Ripley,Springer-Verlag, New York -1999.   Linear models are the foundation for the t-test, analysis of variance (ANOVA), regression analysis, as well as a variety of multivariate methods including discriminant analysis. Explanatory variables may or may not enter the model as first-order terms. This is true also of (non-linear) logistic regression. The logistic regression model is simply a non-linear transformation of the linear regression model: the dependent variable is replaced by a log odds ratio (logit). In summary these statistical methods are based on linear relationships between the explanatory variables. Consequently, one avenue for seeking redundancy in panels is to identify highly correlated variables (markers). It may be possible to replace one marker with the other in a panel to achieve similar performance.<br>
Another avenue for seeking redundancy in panels is to undertake a "best subsets" regression analysis. Given a starting model with all of the explanatory variables of interest, the aim is to find the best single-variable regression models, the best two-variable regression, etc. This methodology is implemented in the SAS statistical package.<br>
(5)       Use of weighting scores<br>
(a)	Commercial and clinical considerations<br>
For many reasons, including strategic and commercial factors; cost; availability; ease of use, it may be preferred to encourage the selection of certain probes in a panel and penalize the selection of others, at the same time trading this off against panel size or performance.<br>
(b)	Attribute costing<br>
Methods for such attribute weighting (in decision trees) have been proposed in the machine learning literature in other contexts such as the incorporation of background knowledge (M. Nunez, "The Use of Background Knowledge", Machine Learning 6: 231-250, 1991.), and the differential cost of obtaining information from robotic sensors (M. Tan, "Cost-sensitive Learning of Classification Knowledge and its Applications in Robotics", Machine Learning. 13: 7-33, 1993.)<br>
Both of these cost-sensitive algorithms have been implemented in the literature by minor changes to the standard machine learning software package known as "C4.5 (J. Ross Quinlan, "C4.5: programs for machine learning", Morgan Kaufinann, CA.<br>
1993.) For convenience, this approach was followed to implement the "EG2" algorithm of Nunez.<br>
In the C4.5 decision tree construction phase, the algorithm compares each available attribute to split on and chooses the single one which maximizes the information gain, Gi. In the EG2 algorithm, (2G| -1)/ (Ci + 1) is maximized which incorporates the cost of information for attribute i, Ci. The vector of weights need to be set a priori by the user.<br>
(i)       Code Modifications<br>
The C4.5 source code was modified to implement the economic generalize! "EG2" algorithm proposed by M. Nunez (The Use of Background Knowledge, Machine Learning 6: 231-250, 1991.)<br>
The exact modifications to the C4.5 package are as follows.<br>
After the following lines in file "R8/Src/contm.c". (J. Ross Quinlan, "C4.5: programs for machine learning", Morgan Kaufmann, CA. 1993)<br><br>
(Table Removed)<br>
is inserted. Where the vector of attribute costs has been previously read in from a text file maintained by the user.<br>
(ii)      Experimental Methodology.<br>
The commercially preferred probes are: 2,4,5,6,8,10,11,12,16,19,20,22,23,28.<br>
For the sake of example, suppose the above probes are commercially preferred due to cost and it is desired to reselect the detection panel taking this cost into account.<br>
The modified C4.5 decision tree software was used to give the commercially preferred probes a penalty of zero and non-commercially preferred probes a penalty of two. The 10-fold cross validated panel selection methodology (as described elsewhere) was run using the modified C4.5 algorithm<br>
(iii)     Results<br>
The standard decision tree detection panel consists of probes 3, 7,19,25,28. Resulting Panel Members: are 2, 6, 7,10, 19,25,28 which used only 2 commercially preferred probes, P7 and P25. Note these probes have been selected by the method in spite of their increased cost due to their, superior performance on this data. The panel is now larger: 7 probes versus 5 originally. There is no demonstratable drop in panel performance on this data although the performance will now be sub-optimal as a trade off against the reduced cost of probes.<br>
(iv)      Conclusion<br>
A straightforward way has been established for incorporating costs of using probes into the panel selection methodology.<br>
(c)       Misclassification costing (i)        Background<br>
For many reasons it may be desired to select an optimal panel bearing in mind that the costs of the different kinds of classification errors may vary. For example, it may be desired to select a panel which has an increased sensitivity to one disease (say Large Cell Carcinoma) and be willing to trade this off against reduced specificity and sensitivity elsewhere in the confusion matrix.<br>
In theory a matrix of misclassification costs (of the same dimensions as the confusion matrix) to incorporate all the possible combinations of costs may be needed. In practice, only those costs which are non unity (the default) are entered.<br>
The commercial decision tree software See5. (RuleQuest Research Pty Ltd, 30 Athena Avenue ,St Ives Pathologist 3SW 2075, Australia, (http://www.rulequest.com)) incorporates tins capability and was used in the following demonstration.<br>
(ii)       Aim<br>
The standard joint discrimination panel (described elsewhere) consists of the members: P2, 3, 4, 5, 12,14, 16, 19, 22, 23, 28. And gives the following estimated confusion<br>
matrix:<br>
(Table Removed)<br>
The sensitivity of Large Cell Carcinoma is low at 26 percent. If one wished to increase this sensitivity in a newly designed panel, the following method may be employed.<br>
(iii)     Methodology The following costs file was generated:<br><br>
(Table Removed)<br>
This file upweights the misciassification of Large Cell Carcinoma as any of the other cancers by a factor of 10. This will tend to increase the sensitivity of detection in this class (with reduced performance elsewhere) but no weighting can ensure perfect classification.<br>
The standard decision tree panel selection methodology was applied (using See5 instead of C4.5).<br>
(iv)      Results<br>
The new panel members are: P2, 3, 4, 5, 6, 9, 12, 14, 16,17, 25, 28. With an estimated performance of:<br>
(Table Removed)<br>
The above demonstrates that the estimated sensitivity of Large Cell Carcinoma has now increased to 48%.<br>
(v)      Conclusion<br>
A straightforward way has been demonstrated for incorporating the differential costs of misclassification into the panel selection methodology. d.        Performance metrics<br>
Outputs provided by the analysis indicating the estimated performance of each method include:<br>
i.	ROC analyses<br>
Receiver Operating Characteristic (ROC) curves show the estimated percentage (or per unit probability) of false positive and false negative scores for different threshold levels in the classifier. An indifferent classifier, unable to discriminate better than random choice, would present a ROC curve with equal true and false readings. The area under this curve would be 50% ( 0.5 probability).<br>
Area Under the Curve (AUC) is often used as an overall estimate of classifier performance and most commercial discriminant function packages compute this figure. A perfect classifier would have 100% Area Under the Curve, a useless classifier would have an AUC near 50% (0.5).<br>
ii.        Confusion matrices: counts and percentages<br>
Confusion matrices show how data from the test set was classified. For pah-wise tests these are counts of true positive, false positive, true negative or false negative scores. These may be shown as actual counts or as percentages. For the multi-way Panel, which attempts to give a unique diagnosis with one panel only, the confusion matrix would show counts for each correct classification. For instance, each time Small Cell carcinoma is detected as such it would be entered in one diagonal of the matrix. Incorrect scores; for instance how often a small cell carcinoma is incorrectly identified as squamous cell cancer would be entered in the appropriate off-diagonal element of the matrix. Error Rates are used to summarize data in the confusion matrix as the sum of all false classifications divided by the total number of classifications made, expressed as a percentage, iii.       Sensitivity and specificity<br>
Specificity refers to the extent to which any definition excludes invalid cases. If a definition has poor specificity, it is high in false positives. This means that it labels individuals as having a disorder when there is really no disorder present. Sensitivity refers to the extent to which any definition includes all valid cases. If a definition has poor sensitivity, it is high in false negatives (individuals who have a disorder present are falsely being diagnosed as not having one).<br>
3.	Data analysis and results<br>
a.	Sample size and variability<br>
Of the 354 cases in the combined Pathologist 1 and Pathologist 2 data set, only 202 cases possessed an H score for every marker (variable or feature).<br>
•	The small number of complete observations and the large number of variables leads to estimation problems (curse of dimensionality). Hence it is necessary to prune severely back the number of variables used to build a classifier.<br>
•	Due to the small number of observations it is not prudent to divide the data into separate training and testing sets (necessary for the robust estimation of classifier performance). For this reason, it was necessary to use resampling methods (such as cross-validation and multiple random trials).<br>
•	The design of a multiclass classifier for cancer diiscrimination is difficult because there are so few observations for each type of cancer.<br>
b.	De-selected markers<br>
Markers were de-selected using the methodology described above. Markers<br>
that were de-selected are represented by non-selection in the panels.<br>
c	Detection panel(s) composition<br>
i.	Selected marker probes<br>
The selected marker probes for all three methods are summarized in Figure 5. ii.        Minimum selected marker set<br>
For the detection panel it is clear that probe 7 delivered the best detection performance for a single marker. Combinations of probes were analyzed to see if a rehable panel could be obtained with more probes.<br>
(1)	Method<br>
The Logistic Regression method allows best subsets to be ranked in terms of a performance measure (Fisher'score). This analysis was used to select the combinations from 1 through 5 probes. Fishers linear discriminant function and logit models (logistic regression) were used to illustrate the performance of these combinations. Data shown above.<br>
(2)	Conclusions<br>
Probe 7 performs well on its own as a classifier, however, a drawback to using probe 7 alone is that probe 7 has a high false negative score. The best performance using Fishers linear discriminant function as a classifier was with probes 7 and 16. The variability of results amongst panels using other combinations suggests the noise added by more features is outweighing any potential to improve classification scores. The small number of incorrectly scored samples gives a poor representation of the statistics of these rarer events. A classifier designed with a larger number of cases may allow a better classifier to be designed. Techniques to select best combinations of probes using different classifiers may produce a different best panel, depending on the structure of the data.<br>
iii.       Supplemental markers<br>
It is shown that panels can be designed to suit the availability of different probes. Different methodologies can be used for selecting these subsets: Decision Trees, Logistic Regression, and Linear Discriminant Functions. Data are shown above.<br>
Method<br>
Using SPSS a Fisher's Linear Discriminant function was applied to the scores obtained from the panel in which constrains were applied due to access constraints. For example, all of the probes come from one vendor. Again, the stepwise option was selected to find the best combination of features. Performance was estimated using the Leave-One-Out cross validation test.<br>
iv.       Alternative markers: biological mechanisms of action (functionally equivalent markers')<br>
A person of ordinary skill inihe art is able to determine functionally equivalent markers. The functional behaviors of the markers used in the panel are described throughout this document.<br>
v.        Marker localization<br>
The localizations of the various markers used in this study are described elsewhere in this document.<br>
vi.       Panel Performance<br>
The performance of the three methods is shown above.<br>
vii.       Limitations on interpretation of panel performance<br>
•	Due to small data set and the need to employ resampling methods, there is the danger that the classifiers have been over-trained (made to fit the data too closely).<br>
•	The panel performance using cytology specimens is difficult to forecast accurately since it is not clear whether sputum cytology samples will contain adequate numbers of cells that are representative of the cells analyzed in the histological validation studies. Nevertheless, given an adequate cellular sample size, one would expect the optimized panel to behave similarly with cytological specimens.<br>
d.        Discriminant Panel Composition<br>
i.	A single 5-wav panel for all cancers<br>
Of the three analysis techniques, only a decision tree is amenable to a single 5-way panel. A single decision tree was therefore constructed to simultaneously classify all types of lung cancer. The panel members are shown Figure 5. The panel performance is shown above in the panel performance tables.<br>
ii.        Panels for discriminating a single type of lung cancer against<br>
all others<br>
Linear discniminant functions are not well suited to performing simultaneous multi-class cliscrimination. The performance of five separate classifiers, each designed<br>
separately to discriminate one of the cancers from a pooled set of all the cancers, was analyzed. Such combinations have the potential to classify none of the cases as having one of the candidate cancers, or classify a single case as having two or more of the candidate cancers. This has a potential advantage in identifying inconsistent cases for further review.<br>
It has been seen that the overall error rate of a single discriminant panel for all cancer types has a fairly high error rate (a five way classifier). In the panel performance data shown above, the performance of five pair-wise classifiers, each designed to identify one cancer from the four other possible cancers is shown. This approach is amenable to analysis using Decision Trees, and Linear Discriminant functions. The technique has the potential to deliver an ambiguous finding when applied, giving two or more diagnoses for a single patient, suggesting further clinical investigation. The technique has the potential to deliver no finding, again suggesting further investigation (perhaps a re-test with the detection panel).<br>
iii.       Panels to account for possibility of false positive cases from<br>
detection panels<br>
A further panel can be trained to discriminate among the false positive cases (from the detection panel) and the five cancer types. This involves selecting those individual cases from the detection panel that were incorrectly classified as abnormal. This trains a dedicated classifier on the 'harder' problem of detecting these 'special' cases. However, while this is a theoretically sound task, the data set only yielded four of these cases and the population was deemed to be under-represented for analysis.<br>
iv.       Selected Markers<br>
The selected marker probes for all three methods are summarized in Figure 5. v.        Minimum selected marker set<br>
This topic is addressed below under "Robustness of Approach Demonstrated by Similar Results Using Different Methods."<br>
vi.       Supplemental markers<br>
This topic is addressed below under "Robustness of Approach Demonstrated by Similar Results Using Different Methods."<br>
vii.      Alternative markers: biological mechanisms of action<br>
A person of ordinary skill in the art is able to determine functionally equivalent markers. The functional behaviors of the markers used in the panel are described throughout this document<br>
viii.     Marker localization<br>
The localization of the various markers used in this study are described throughout this document.<br>
ix.       Panel Performanne<br>
The performance of the three methods is summarized in Figure 5.<br>
e.	Effect of weighting parameters<br>
In addition to user-supplied weighting criteria for markers and also for disease states (classes) as discussed earlier, one can also use a binary weighting scheme. For example, if all non-DAKO supplied probes are weighted "0" and all DAKO-supplied probes are weighted "1", then the optimized panel will contain only DAKO-supplied probes. This is an improtant product design capability for any vendor who intends to develop and market molecular diagnostic panel kits using only their supplies.<br>
f.	Effect of using other (non H-score) objective scoring parameters<br>
i.	Background<br>
The Pathology Review sheet contains a set of boxes as follows, in Table 32:<br>
Table 32:<br>
(Table Removed)<br>
The standard scoring system uses the "H score" which is obtained by grading the intensity as: none = 0, weak = 1, moderate = 2, intense = 3, and the percentage cells as: 0-5% = 0, 6-25% = 1, 26-50% = 2,51-75% = 3, &gt;75% = 4, and then multiplying the two<br>
grades together. For example, 50% weakly stained plus 50% moderate stained would score 10 = 2x2 + 2x3.<br>
ii.        Method<br>
An alternative scoring method was analyzed in which the response was divided into low, medium and high as follows:<br>
(a)	if more than 50% of cells had moderate or above stain → HIGH<br>
(b)	if more than 50% of cells had no stain →LOW<br>
(c)	otherwise → MEDIUM<br>
The decision tree detection panel selection methodology was repeated using this 3-level factor instead of H-score. This caused the tree to split into 3 branches at each node, if required.<br>
iii.       Results<br>
The panel selected was: Probes 3, 7,10,11, 16,19, 20,28 With an estimated performance of:<br>
(Table Removed)<br>
This should be compared to the reference performance with H-scores of:<br>
(Table Removed)<br>
iv.       Conclusions<br>
There is a substantial loss of performance (larger panels, lower sensitivity and lower specificity when the proposed alternative scoring system is used.<br>
Treating the H-score as a continuous variable (in the range 0 to 12) seems to be near optimal for panel selection on the data examined.<br>
•	The many other possible scoring systems have not been examined, but may be<br>
feasible and applicable to the experimentally tested panel design and development<br>
methodology.<br>
4.        Lung Cancer Detection and Discrimination Panels<br>
Listed below are exemplary lung cancer detection and discrimination panels determined by the above illustrative example. It is noted that although the panels listed below recite specific probes, each specific probe may be substituted by a correlate probe or a functionally related probe. Detection (No Constraints)<br>
•	anti-Cyclin A combined with one or more additional probes<br>
•	anti-Cyclin A, anti-human epithelial related antigen (MOC-31) anti-Cyclin A, anti-ER-related P29<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B<br>
•	anti-Cyclin A, anti-human epithelial related antigen (MOC-31), anti-VEGF<br>
•	anti-Cyclin A, anti-human epithelial related antigen (MOC-31), anti-mature surfactant apoprotein B<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF<br>
anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-surfactant apoprotein A<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF, anti-surfactant apoprotein A<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF, anti-Cyclin Dl<br>
•	anti-Cyclin A, anti-human epithelial related antigen (MOC-31) combined with one or more additional probes<br>
•	anti-Cyclin A, anti-ER-related P29 combined with one or more additional probes<br>
anti-Cyclin A, anti-mature surfactant apoprotein B combined with one or more additional probes<br>
•	anti-Cyclin A, anti-human epithelial related antigen (MOC-31), anti-VEGF combined<br>
with one or more additional probes<br>
anti-Cyclin A, anti-human epithelial related antigen (MOC-31), anti-mature surfactant apoprotein B combined with one or more additional probes<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF combined with one or more additional probes<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-surfactant apoprotein A combined with one or more additional probes<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF, anti-surfactant apoprotein A combined with one or more additional probes<br>
•	anti-Cyclin A, anti-mature surfactant apoprotein B, anti-human epithelial related antigen (MOC-31), anti-VEGF, anti-Cyclin Dl combined with one or more additional probes<br>
Detection fW/O anti-Cvclin A)<br>
•	anti-Ki-67 combined with one or more additional probes.<br>
•	anti-Ki-67 combined with any one probe selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B.<br>
anti-Ki-67 combined with any two probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B.<br>
" anti-Ki-67 combined with any three probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein 8.<br>
•	anti-Ki-67 combined with any four probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B.<br>
•	anti-Ki-67 combined with any five probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B.<br>
anti-Ki-67, anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B<br>
•	anti-Ki-67 combined with any one probe selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B, and with one or more additional probes,<br>
•	anti-Ki-67 combined with any two probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferatmg cell nuclear antigen and anti-mature surfactant apoprotein B, and with one or more additional probes.<br>
•	anti-Ki-67 combined with any three probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B, and with one or more additional probes.<br>
•	anti-Ki-67 combined with any four probes selected from the group consisting of anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen and anti-mature surfactant apoprotein B, and with one or more additional probes.<br>
•	anti-Ki-67 combined with any five probes selected from the group consisting of anti-<br>
VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-<br>
proliferating cell nuclear antigen and anti-mature surfactant apoprotein B, and with<br>
one or more additional probes.<br>
anti-Ki-67, anti-VEGF, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen, anti-mature surfactant apoprotein B and one or more additional probes.<br>
Detection With CommericaUv Preferred Probes<br>
•	anti-Ki-67 combined with one or more additional probes.<br>
•	anti-TTF-1 combined with one or more additional probes. ••	anti-EGFR combined with one or more additional probes.<br>
•	anti-proliferating cell nuclear antigen combined with one or more additional probes.<br><br>
•	two probes selected from the group consisting of anti-Ki-67, anti-TTF-1, anti-EGFR and anti-proliferating cell nuclear antigen.<br>
•	three probes selected from the group consisting of anti-Ki-67, anti-TTF-1, anti-EGFR and anti-proliferating cell nuclear antigen.<br>
anti-Ki-67, anti-TTF-1, anti-EGFR and anti-proliferating cell nuclear antigen<br>
•	two probes selected from the group consisting of anti-Ki-67, anti-TTF-1, anti-EGFR and anti-proliferating cell nuclear antigen, and one or more additional probes.<br>
•	three probes selected from the group consisting of anti-Ki-67, anti-TTF-1, anti-EGFR and anti-proliferating cell nuclear antigen, and one or more additional probes.<br>
anti-Ki-67, anti-TTF-1, anti-EGFR, anti-proliferating cell nuclear antigen, and one or more additional probes.<br>
Discrimination Between Adenocarcinoma And Other Lnng Cancers<br>
•	anti-mucin 1 and anti-TTF-1<br>
anti-mucin 1 and anti-TTF-1 combined with any one probe selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3<br>
•	anti-mucin 1 and anti-TTFrl combined with and two probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3<br>
•	anti-mucin 1 and anti-TTF-1 combined with any three probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3<br>
•	anti-mucin 1 and anti-TTF-1 combined with any four probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3<br>
•	anti-VEGF, anti-surfactant apoprotein A, anti-mucin 1, anti-TTF-1, anti-BCL2, anti-ER-related P29 and anti-Glut 3<br>
•	anti-mucin 1, anti-TTF-1 and one or more additional probes<br>
•	anti-mucin 1 and anti-TTF-1 combined with any one probe selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3, and with one or more additional probes<br>
•	anti-mucin 1 and anti-TTF-1 combined with and two probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3, and with one or more additional probes<br>
•	anti-mucin 1 and anti-TTF-1 combined with any three probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3, and with one or more additional probes<br>
•	anti-mucin 1 and anti-TTF-1 combined with any four probes selected from the group consisting of anti-VEGF, anti-surfactant apoprotein A, anti-BCL2, anti-ER-related P29 and anti-Glut 3, and with one or more additional probes<br>
•	anti-VEGF, anti-surfactant apoprotein A, anti-mucin 1, anti-TTF-1, anti-BCL2, anti-<br>
ER-related P29, anti-Glut 3 and one or more additional probes<br>
Discrimination Between Squamous Cell Carcinoma And Other Lung Cancers<br>
•	anti-CD44v6 combined with one or more additional probes<br>
•	anti-CD44v6 combined with any one probe selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3<br>
•	anti-CD44v6 combined with any two probes selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3<br>
•	anti-CD44v6 combined with any three probes selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3<br>
"	anti-CD44v6 combined with any four probes selected from the group consisting of<br>
anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3<br>
•	anti-CD44v6, anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3<br>
•	anti-CD44v6 combined with any one probe selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3, and with one or more additional probes<br>
•	anti-CD44v6 combined with any two probes selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3, and with one or more additional probes<br>
•	anti-CD44v6 combined with any three probes selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3, and with one or more additional probes<br>
anti-CD44v6 combined with any four probes selected from the group consisting of anti-VEGF, anti-thrombomodulin, anti-Glut 1, anti-ER-related P29 and anti-melanoma-associated antigen 3, and with one or more additional probes<br>
anti-CD44v6, anti-VEGF„anti-thrombomodulin, anti-Glut 1, anti-ER-related P29, anti-melanoma-associated antigen 3 and one or more additional probes<br>
Discrimination Between Large Cell Carcinoma And Other Lung Cancers<br>
•	anti-VEGF combined with one or more additional probes.<br>
anti-VEGF and anti-pl20 anti-VEGF and anti-Glut 3 anti-VEGF, anti-p 120 and anti-Cyclin A<br>
•	anti-VEGF, anti-pl20 and one or more additional probes<br>
•	anti-VEGF, anti-Glut 3 and one or more additional probes<br>
•	anti-VEGF, anti-p 120, anti-Cyclin A and one or more additional probes<br>
Discrimination Between Mesothelioma And Other Lane Cancers<br>
•	anti-CD44v6 combined with one or more additional probes.<br>
•	anti-proliferating cell nuclear antigen combined with one or more additional probes.<br>
•	anti-human epithelial related antigen (MOC-31) combined with one or more additional probes.<br>
•	two probes selected from the group consisting of anti-CD44v6, anti-proliferating cell nuclear antigen and anti-human epithelial related antigen (MOC-31), combined with one or more additional probes<br>
anti-CD44v6, anti-proliferating cell nuclear antigen, anti-human epithelial related antigen (MOC-31) and one or more additional probes.<br>
Discrimination Between Small Cell And Other Lung Cancers<br>
•	anti-proliferating cell nuclear antigen combined with one or more additional probes.<br>
•	anti-BCL2 combined with one or more additional probes.<br>
•	anti-EGFR combined with one or more additional probes.<br>
•	two probes selected from the group consisting of anti-proliferating cell nuclear antigen, anti-BCL2 and anti-EGFR<br>
•	anti-proliferating cell nuclear antigen, anti-BCL2, anti-EGFR<br>
•	two probes selected from the group consisting of anti-proliferating cell nuclear antigen, anti-BCL2 and anti-EGFR, combined with one or more additional probes<br>
•	anti-proliferating cell nuclear antigen, anti-BCL2, anti-EGFR and one or more additional probes<br>
Simultaneous Discriminarion Of Adenocarcinoma, Squamous Cell Carcinoma, Large Cell Carcinoma. Mesothelioma And Small Cell Carcinoma<br>
•	two or more probes selected from anti-VEGF, anti-thrombomodulin, anti-CD44v6, anti-surfactant apoprotein A, anti-proliferating cell nuclear antigen, anti-mucin 1, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-N-cadherin, anti-EGFR and anti-proliferatirig cell nuclear antigen<br>
•	anti-VEGF, anti-thrombomodulin, anti-CD44v6, anti-surfactant apoprotein A, anti-proliferating cell nuclear antigen, anti-mucin 1, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-N-cadherin, anti-EGFR and anti-proliferating cell nuclear antigen<br>
•	two or more probes selected from anti-VEGF, anti-thrombomodulin, anti-CD44v6, anti-surfactant apoprotein A, anti-proliferating cell nuclear antigen, anti-mucin 1, anti-human epithelial related antigen (MOC-31), anti-TTF-1, anti-N-cadherin, anti-EGFR and anti-proliferating cell nuclear antigen, combined with one or more additional probes<br>
anti-VEGF, anti-thrombomodulin, anti-CD44v6, anti-surfactant apoprotein A, anti-proliferating cell nuclear antigen, anti-mucin 1, anti-human epithehal related antigen (MOC-31), anti-TTF-1, anti-N-cadherin, anti-EGFR and anti-proliferating cell nuclear antigen, combined with one or more"additional probes<br>
5.        Conclusions<br>
a.        Validity of panel approach to molecular diagnostics<br>
i.	Non-intuitive solutions<br>
Histograms were plotted (PathologistData.xls, worksheet: Histograms) showing the distribution of marker scores for each probe for Control vs. Cancer. It is clear from these histograms that an intuitive selection of probes for specific panels is certainly not obvious and the invention described does allow effective combinations to be found in the absence of an obvious method.<br>
ii.        Optimization for varied product applications<br>
iii.       Robustness of approach demonstrated by similar results using different methods<br>
Detailed scrutiny of the results obtained by the various analyses in the body of this report, and as summarized in the tables and figures, shows the following findings.<br>
1.	Careful scrutiny of the performance of individual probes does not make apparent probe combinations that might perform better than any one probe alone.<br>
2.	All three classification methodologies evaluated hone in on similar sets of features. The small differences can be attributed to the data structure that may favor one classifier over another.<br>
3.	All the classifiers designed with one of these methods were shown to give good performance when tested on data from an independent pathologist, unseen during the design process. This gives high confidence in the invention.<br>
4.	A detection panel based on probe 7 alone gives a high performance.<br>
5.	If probe 7 is combined with probe 16 or 25 then a better performance is obtained.<br>
6.	While combinations of other probes with probe 7 appear to improve performance further, the number of extra cases captured is so low that they may be unrepresentative and the classifier so designed may not generalize.<br>
7.	The performance of panels selected from probes excluding probe 7 provided some discrimination, good enough in comparison with current practice using human screening, but perhaps not good enough for an automated cytometer in tomorrow's clinical diagnostic cytology world (see Figure 6).<br>
8.	Other combinations of probes can provide a useful, but lesser, performance.<br>
9.	If some probes become unavailable this invention allows the selection of other combinations of probes. This was illustrated by classifier designs based on a commercially preferred set of probes only. See Figure 7.<br>
10.	The invention allows a weighting to be applied against costly probes. Rather than totally excluding them from the analysis this allows their inclusion in the panel if their contribution is important.<br>
11.	The invention allows the design of single lung cancer type specific discrmination panels that can discriminate one type of lung cancer from among all other cancers.<br>
12.	Analysis of the performance of a single panel to classify five cancers showed discrimination was possible but the overall error rate was worse than a set of five panels each designed to discriminate one of the cancers from the others.<br>
13.	A very useful discrimination was obtained with the combination of five two way classifiers.<br>
14.	Common sets of probes were selected by the three classification methodologies for the five discrimination panels, again giving confidence in this result.<br>
15.	Probes for isolating cases of Adenocarcinoma are 1, 14,19,20,25, and 27.<br>
16.	Probes for isolating cases of Squamous Cell cancer are 1,2, 3,24,25, and 26.<br>
17.	Probes for isolating cases of Large Cell cancer are 1 and 7 or 1, and 21.<br>
18.	Probes for isolating cases of Mesothelioma are 3, 12, and 16.<br>
19.	Probes for isolating cases of Small Cell cancer are 12, 20, and 23.<br>
20.	Probes for recognizing all cancers simultaneously are 1,2, 3, 4,12,14, 19,22,23, and 28.<br><br>
21.      An advantage of using the multiple pair-wise panels as defined by this invention is that doubtful cases may not score on any of the five panels, also confusing cases may show on two or more panels. Such anomalous reports would alert the cytologist that further analysis is indicated.<br>
iv.       Risk Management Study<br>
All the tests applied in this study were statistical in nature. There is a risk that probes selected on the basis of small improvements in performance will have statistical variations when tested on new data. To give confidence in the results, the best classifier emerging from the Linear Discriminant analysis on the Pathologist 1 and Pathologist 2 data was tested. It should be remembered that the Pathologist 3 data was statistically different from the Pathologist 1 and Pathologist 2 data, so if good performances are obtained when tests using the Pathologist 3 data, then this would be encouraging indeed.<br>
(1)       Report on Testing with unseen data - Detection panel<br>
(a)	Method<br>
In the Section titled "Detection Panel(s) Composition" above, we showed that good classification is obtained with features 7 and 16. Using SPSS all the Pathologist 3 data that reported H scores for both 7 and 16 was selected. Then, using Transform and Compute, the canonical discrimination function was generated as a new feature. The performance of this feature alone was then tested.<br>
(b)	Results<br>
These are the results of testing the classifier designed on Pathologist 1 and Pathologist 2 data and testing on Pathologist 3 data. The classifier was designed using the linear discriminant function on probes 7 and 16. The Canonical Pathologist 2 function was =0.965*Probe7-0.298*Probel6.<br>
Classification Results on Pathologist 3 data using probes 7 and 16<br>
Predicted	Total<br>
Group<br>
Membership<br>
Diagnosis	0	1<br>
(UCLA)<br>
Original Count	0	20	1	21<br>
1	6	41	47<br>
%	0	95.2	4.8	100.0<br>
1	12.8        87.2	100.0<br>
Cross-validated Count	0	20	1	21<br>
1	6	41	47<br>
%	0	95.2	4.8	100.0<br>
1	12.8        87.2	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by die functions derived from all cases other than that case, b 89.7% of original grouped cases correctly classified, c 89.7% of cross-validated grouped cases correctly classified.<br>
This is better than classifying the Pathologist 3 data on probe 7 only show as follows<br>
Classification Results on Pathologist 3 data using probe 7 only<br>
Predicted	Total<br>
Group<br>
Membership<br>
Diagnosis	0	1<br>
(UCLA)<br>
Original Count	0	20	1	21<br>
1	8	39	47<br>
%	0	95.2	4.8	100.0<br>
1	17.0	83.0	100.0<br>
Cross-validated Count	0	20	1	21<br>
1	8	39	47<br>
%	0	95.2	4.8	100.0<br>
1	17.0        83.0	100.0<br>
a Cross validation is done only for those cases in die analysis. In cross validation, each case is classified by the functions derived from all cases other than tiiat case, b 86.8% of original grouped cases correcdy classified, c 86.8% of cross-validated grouped cases correctly classified.<br>
(c)       Conclusion<br>
This gives confidence that the two-probe classifier based on 7 and 16 is better than probe 7 alone<br>
(2)       Report on testing with unseen data - Discrimination Panel<br>
(a)       Background<br>
Reported below is the performance of the classifier designed with Pathologist 1 and Pathologist 2 data using LDF and tested with the unseen Pathologist 3 data. The numbers of cases at the design stage was relatively small and the numbers in the test data are also small, so a good degree of variability can be expected between performance on the first and second set<br>
(b)	Method<br>
In SPSS, the canonical discrimination functions derived in die section titled 'Tattem recognition", were built and tested on Pathologist 3 data for all five classes of cancer<br>
(c)	Results<br>
MesotheliomaLDF=probe3sc * .385 - probel2s * .317 + probel6s * 1.006<br>
Classification Results<br>
Predicted	Total<br>
Group<br>
Membership<br>
Meso = 1,	0	'  1<br>
others = 0<br>
Original        Count	0	38	2	40<br>
1	17	8<br>
%	0	95.0	5.0	100.0<br>
1	12.5        87.5	100.0<br>
Cross-        Count	0	38	2	40<br>
validated<br>
1	17	8<br>
%	0	95.0	5.0	100.0<br>
1	12.5        87.5	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case, b 93.8% of original grouped cases correctly classified, c 93.8% of cross-validated grouped cases correctly classified.<br>
Small cell cancer LDF= probel2s * .575 - probe20s * .408 - probe22s * .423 + probe23s* .344<br>
Classification Results<br>
Predicted	Total<br>
Group Membersh<br>
ip   -<br>
Small = 1,	0	1<br>
others = 0<br>
Original        Count	0	39	3	42<br>
1*1	5	6<br>
%	0	92.9	7.1	100.0<br>
1	16.7	83.3	100.0<br>
Cross-        Count	0	39	3	42<br>
validated<br>
115	6<br>
%	0	92.9	7.1	100.0<br>
1	16.7	83.3	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case, b 91.7% of original grouped cases correctly classified, c 91.7% of cross-validated grouped cases correctly classified.<br>
Squamous cell cancer LDF= - probelsc * .328 - probe2sc * .295 + probe3sc * .741 + probe24s * .490 + probe25s * .393 + probe26s * .426<br>
Classification Results<br>
Predicted	Total<br>
Group<br>
Membership<br>
Squamous	0	1<br>
= 1, others<br>
= 0<br>
Original        Count	0	31	4	35<br>
1	2	9	11<br>
%	0	88.6     11.4      100.0<br>
1	18.2     81.8       100.0<br>
Cross-        Count	0	31	4	35<br>
validated<br>
1	2	9	11<br>
%	0	88.6     11.4      100.0<br>
1	18.2     81.8       100.0<br>
a Cross validation is done only for those cases in the analysts. In cross validation, each case is classified by the functions derived from all cases other than that case, b 87.0% of original grouped cases correctly classified, c 87.0% of cross-validated grouped cases correctly classified.<br>
i-arge cell cancer LDF= probelsc * .847+ probe7sc * .452<br>
Classification Results<br>
Predicted	Total<br>
Group<br>
Membership —<br>
Large = 1,	0	1<br>
others = 0<br>
Original        Count	0	23	15	38<br>
1	4	5	9<br>
%	0	60.5        39.5	100.0<br>
1	44.4        55.6	100.0<br>
Cross-        Count	0	23	15	38<br>
validated<br>
1	4	5	9<br>
%	0	60.5        39.5	100.0<br>
1	44.4        55.6	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case, b 59.6% of original grouped cases correctly classified, c 59.6% of cross-validated grouped cases correctly classified.<br>
The lower, but useful, performance was on a classifier designed and tested with a very small number of cases of large cell cancer, so this result is still very encouraging.<br>
Adenocarcinoma, LDF= - probe4sc * .515 + probe5sc * .299 - probel4s * .485 - probel9s * .347 + probe20s * .723 + probe25s * .327 + probe27s * .327<br>
Classification Results<br>
Predicted	Total<br>
Group Membersh<br>
ip<br>
Adeno = 1,	0	1<br>
Others = 0<br>
Original        Count	0	29	5	34<br>
1	0	14	14<br>
%	0	85.3	14.7	100.0<br>
1   ■	.0	100.0	100.0<br>
Cross-        Count	0	29	5	34<br>
validated<br>
1	0	14	14<br>
%	0	85.3	14.7	100.0<br>
1	.0	100.0	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case, b 89.6% of original grouped cases correctly classified, c 89.6% of cross-validated grouped cases correctly classified.<br>
• (d)      Conclusion<br>
It is very encouraging to note the performance of these classifiers stand up to the tests of applying unseen data. This gives a very high confidence in the ability to detect the individual cancers.<br>
(3)       Training and testing on data from different patients and pathologists<br>
As a "final final" test of robustness a LDF was trained on the data that was reviewed by both Pathologist 1 and Pathologist 2. This removes data reviewed by Pathologist 3. Hence testing on data reviewed by both Pathologist 3 plus Pathologist 1 data is not biased. Previously the test process was biased through using data from the same patient for test and train.<br>
LDF produced the same set of features except for probe 4 which was not included. The LDF was = probelsc * .288 + probe7sc * .846 - probel5s * .249 - probel6s * .534<br>
Classification Results<br>
Area under the Curve = .977<br>
Predicted	Total<br>
Group<br>
Membership<br>
Diagnosis	0       1<br>
(UCLA)<br>
Original	Count	0	20       0	20<br>
1	9     37	46<br>
%	0	100.0      .0	100.0<br>
1	19.6 80.4	100.0<br>
Cross-	Count	0	20       0	20<br>
validated<br>
1	9     37	46<br>
%	0	100.0      .0	100.0<br>
1	19.6 80.4	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case. b 86.4% of original grouped cases correctly classified, c 86.4% of cross-validated grouped cases correctly classified.<br>
Still a reasonable result, but a similar result, but with a smaller area under the curve, was obtained with probe7 alone on Pathologist 3 only data<br>
Classification Results<br>
Area under the curve = .908<br>
Predicted  _	Total<br>
Group Membersh<br>
'P<br>
Diagnosis	0	1<br>
(UCLA)<br>
Original        Count	0	19	1	20<br>
1	7	39	46<br>
%	0	95.0	5.0	100.0<br>
1	15.2	84.8	100.0<br>
Cross-	Count	0	19	1	20<br>
validated<br>
1	7	39	46<br>
%	0	95.0	5.0	100.0<br>
1	15.2	84.8	100.0<br>
a Cross validation is done only for those cases in the analysis. In cross validation, each case is classified by the functions derived from all cases other than that case, b 87.9% of original grouped cases correctly classified, c 87.9% of cross-validated grouped cases correctly classified.<br>
References:<br>
[I]	Goldberg-Kahn, B., Healy, J.C. and Bishop, J.W. (1997) The cost of diagnosis: a<br>
comparison of four different strategies in the workup of solitary radiographic lung<br>
lesions. Chest 111, 870-6.<br>
[2]       O'Donovan, PJB. (1997) The radiologic appearance of lung cancer. Oncology (Huntingt) 11,1387-402; discussion 1402-4.<br>
[3]       Worrell, J.A. (1995) Radiology of the central airways. Otolaryngol Clin North Am 28, 701-20.<br>
[4]       Henschke, C.I., Miettinen, O.S., Yankelevitz, D.F., Libby, D.M. and Smith, JP.<br>
(1994) Radiographic screening for cancer. Proposed paradigm for requisite research. Clin Imaging 18,16-20.<br>
[5]       Lam, S., Kennedy, T., Unger, M., Miller, Y.E., Gelmont, D., Rusch, V., Gipe, B., Howard, D., LeRiche, J.C, Coldman, A. and Gazdar, A.F. (1998) Localization of bronchial intraepithelial neoplastic lesions by fluorescence bronchoscopy. Chest 113, 696-702.<br>
[6]       Sazon, D.A., Santiago, S.M., Soo Hoo, G.W., Khonsary, A., Brown, C, Mandelkem, M., Bland, W. and Williams, A.J. (1996) Fluorodeoxyglucose-positron emission tomography in the detection and staging of lung cancer. Am J Respir Crit Care Med 153,417-21.<br>
[7]       Lowe, V.J., DeLong, DM., Hoffman, J.M. and Coleman, RJE. (1995) Optimum<br>
scanning protocol for FDG-PET evaluation of pulmonary malignancy. J Nucl Med 36, 883-7.<br>
[8]       Lowe, V.J., Fletcher, J.W., Gobar, L., Lawson, M., Kirchner, P., Valk, P., Karis, J., Hubner, K., Delbeke, D., Heiberg, E.V., Patz, E.F. and Coleman, R.E. (1998) Prospective investigation of positron emission tomography in lung nodules. J Clin Oncol 16, 1075-84.<br>
[9]       Raab, S.S., Homberger, J. and Raffin, T. (1997) The importance of sputum cytology in the diagnosis of lung cancer a cost-effectiveness analysis. Chest 112, 937-45.<br>
[10]     Franklin, W.A. (1998) New molecular and cellular approaches to lung cancer detection. In: Biology of Lung Cancer, pp. 529-570.<br>
[II]	Kern, W.H. (1988) The diagnostic accuracy of sputum and urine cytology. Acta Cytol<br>
32, 651-4.<br>
[12]     Mehta, A.C., Marty, J.J. and Lee, F.Y. (1993) Sputum cytology. Clin Chest Med 14, 69-85.<br>
[13]     Gledhill, A., Bates, C, Henderson, D., DaCosta, P. and Thomas, G. (1997) Sputum cytology: a limited role. J Clin Pathol 50, 566-8.<br>
[14] Steffee, C.H., Segletes, L.A. and Geisinger, K.R. (1997) Changing cytologic and histologic utilization patterns in the diagnosis of 515 primary lung malignancies. Cancer 81, 105-15.<br>
[15]     Zaman, M.B. (1991) Pulmonary cytology. Clin Lab Med 11, 293-315.<br>
[16]     Flehinger, B.J. and Melamed, M.R. (1994) Current status of screening for lung cancer. Chest Surg Clin N Am 4, 1-15.<br><br>
Koss, L.G., Melamed, M.R. and Goodner, J.T. (1964) Pulmonary cytology: A brief survey of diagnostic results from July 1st, 1952 until December 31st, 1960. Acta Cytol 8,104.<br>
Saccomanno, G., Saunders, R.P., Ellis, H., Archer, V.E., Wood, B.G. and Beckler, P.A. (1963) Concentration of carcinoma or atypical cells in sputum. Acta Cytol 5, 305-310.<br>
Miura, H., Konaka, C, Kawate, N., Tsuchida, T. and Kato, H. (1992) Sputum cytology-positive, bronchoscopically negative adenocarcinoma of the lung [see comments]. Chest 102,1328-32.<br>
Valatis, J., Warrens, D. and Gamble, D. (1981) Increased incidence of adenocarcinoma of the lung. Cancer 47,1042-1046.<br>
Baldini, E.H. and Strauss, G.M. (1997) Women and lung cancer: waiting to exhale. Chest 112, 229S-234S.<br>
Caldwell, C J. and Berry, C.L. (1996) Is the incidence of primary adenocarcinoma of the lung increasing? Virchows Arch 429, 359-63.<br>
Risse, E.K., Vooijs, G.P. and van't Hof, M.A. (1987) Relationship between the cellular composition of sputum and the cytologic diagnosis of lung cancer. Acta Cytol 31, 170-6.<br>
Holiday, D.B., McLarty, J.W., Farley, M.L., Mabry, L.C., Cozens, D., Roby, T., Waldron, E., Underwood, R£&gt;., Anderson, E., Culbreth, W. and et al. (1995) Sputum cytology within and across laboratories. A reliability study. Acta Cytol 39,195-206.<br>
Eddy, D.M. (1989) Screening for lung cancer [see comments]. Ann Intern Med 111, 232-7.<br>
Younes, M., Brown, R.W., Stephenson, M., Gondo, M. and Cagle, P.T. (1997) Overexpression of Glutl and Glut3 in stage I nonsmall cell lung carcinoma is associated with poor survival. Cancer 80,1046-51.<br>
Ogawa, J., Inoue, H. andKoide, S. (1997) Glucose-transporter-type-I-gene amplification correlates with sialyl- Lewis-X synthesis and proliferation in lung cancer. Int J Cancer 74, 189-92.<br>
Ito, T., Noguchi, Y., Satoh, S., Hayashi, H., Inayama, Y. and Kitamura, H. (1998) Expression of facilitative glucose transporter isoforms in lung carcinomas: its relation to histologic type, differentiation grade, and tumor stage [see comments]. Mod Pathol 11,437-43.<br>
Sosolik, R.C., McGaughy, VJL and De Young, B.R. (1997) Anti-MOC-31: a potential addition to the pulmonary adenocarcinoma versus mesothelioma immunohistochemistry panel. Mod Pathol 10, 716-9.<br>
Ordonez, N.G. (1998) Value of the MOC-31 monoclonal antibody in differentiating epithelial pleural mesothelioma from lung adenocarcinoma. Hum Pathol 29,166-9.<br>
Takanami, I., Tanaka, F., Hashizume, T., Kikuchi, K., Yamamoto, Y., Yamamoto, T. and Kodaira, S. (1996) The basic fibroblast growth factor and its receptor in pulmonary adenocarcinomas: an investigation of their expression as prognostic markers. Eur J Cancer 32A, 1504-9.<br>
[32]     Takanami, I., Imamura, T., Hashizume, T., Kikuchi, K., Yamamoto, Y, Yamamoto, T. and Kodaira, S. (1996) Irnmunohistochernical detection of basic fibroblast growth<br>
factor as a prognostic indicator in pulmonary adenocarcinoma. Jpn J Clin Oncol 26, 293-7.<br>
[33]     Ohta, Y., Endo, Y., Tanaka, M., Shimizu, J., Oda, M., Hayashi, Y., Watanabe, Y. and Sasaki, T. (1996) Significance of vascular endothelial growth factor messenger RNA expression in primary lung cancer. Clin Cancer Res 2,1411-6 1996.<br>
[34]     Volm, M., Koomagi, R., Mattern, J. and Stammler, G. (1997) Prognostic value of basic fibroblast growth factor and its receptor (FGFR-1) in patients with non-small cell lung carcinomas. Eur J Cancer 33, 691-3.<br>
[35] Hiyama, K., Hiyama, E., Ishioka, S., Yamakido, M., Inai, K., Gazdar, A.F., Piatyszek, M.A. and Shay, J.W. (1995) Telomerase activity in small-cell and non-small-cell lung cancers [see comments]. J Natl Cancer Inst 87, 895-902.<br>
[36]     Yashima, K_, Litzky, L.A., Kaiser, L., Rogers, T., Lam, S-, Wistuba, II Milchgrub, S., Srivastava, S., Piatyszek, M.A., Shay, J.W. and Gazdar, AJ. (1997) Telomerase expression in respiratory epithelium during the multistage pathogenesis of lung carcinomas. Cancer Res 57, 2373-7.<br>
[37]     Ahrendt, S A., Yang, S.C., Wu, L., Westra, W.H., Jen, J., Califano, J.A. and<br>
Sidransky, D. (1997) Comparison of oncogene mutation detection and telomerase activity for the molecular staging of non-small cell lung cancer. Clin Cancer Res 3, 1207-14.<br>
[38]     Albanell, J., Lonardo, F., Rusch, V., Engelhardt, M., Langenfeld, J., Han, W., Klimstra, D., Venkatraman, E., Moore, M.A. and Dmitrovsky, E. (1997) High telomerase activity in primary lung cancers: association with increased cell proliferation rates and advanced pathologic stage. J Natl Cancer Inst 89,1609-15.<br>
[39]     Hiyama, K., Ishioka, S., Shay, J.W., Taooka, Y., Maeda, A., Isobe, T., Hiyama, E., Maeda, H. and Yamakido, M. (1998) Telomerase activity as a novel marker of lung cancer and immune- associated lung diseases. Int J Mol Med 1, 545-9.<br>
[40]     Yahata, N., Ohyashiki, K., Ohyashiki, J.H., Iwama, H., Hayashi, S., Ando, K., Hirano, T., Tsuchida, T., Kato, H., Shay, J.W. and Toyama, K (1998) Telomerase activity in lung cancer cells obtained from bronchial washings. J Natl Cancer Inst 90, 684-90.<br>
[41]     Lee, J.C., Jong, H.S., Yoo, C.G., Han, S.K., Shim, Y.S. and Kim, Y.W. (1998)<br>
Telomerase activity in lung cancer cell lines and tissues. Lung Cancer 21,99-103.<br>
[42]     Arai, T., Yasuda, Y., Takaya, T., Ito, Y., Hayakawa, K., Toshirna, S., Shibuya, C, Yoshimi, N. and Kashiki, Y. (1998) Application of telomerase activity for screening of primary lung cancer in broncho-alveolar lavage fluid. Oncol Rep 5, 405-8.<br>
[43]     Fujii, M., Motoi, M., Saeki, H., Aoe, K. and Moriwaki, S. (1993) Prognostic<br>
significance of proliferating cell nuclear antigen (PCNA) expression in non-small cell lung cancer. Acta Med Okayama 47, 103-8.<br>
[44]     Kawai, T., Suzuki, M., Kono, S., Shinorniya, N., Rokutanda, M., Takagi, K., Ogata, T. and Tamai, S. (1994) Proliferating cell nuclear antigen and Ki-67 in lung carcinoma. Correlation with DNA flow cytometric analysis. Cancer 74,2468-75.<br>
[45]     Ogawa, J., Tsurumi, T., Yamada, S., Koide, S. and Shohtsu, A. (1994) Blood vessel invasion and expression of sialyl Lewisx and proliferating cell nuclear antigen in stage I non-small cell lung cancer. Relation to postoperative recurrence. Cancer 73,<br>
1177-83.<br>
[46]    Ebina, M., Steinberg, S.M., Mulshine, J.L. and Linnoila, R.I. (1994) Relationship of p53 overexpression and up-regulation of proliferating cell nuclear antigen with the clinical course of non-small cell lung cancer. Cancer Res 54,2496-503.<br>
[47]     Fontanini, G., Vignati, S., Bigini, D2 Merlo, G.R., Ribecchini, A, Angeletti, C.A., Basolo, F., Pingitore, R. and Bevilacqua, G. (1994) Human non-small cell lung cancer p53 protein accumulation is an early event and persists during metastatic progression [see comments]. J Pathol 174,23-31.<br>
[48]     Wiethege, T., Voss, B. and Muller, K M (1995) P53 accumulation and proliferating-cell nuclear antigen expression in human lung cancer. J Cancer Res Clin Oncol 121, 371-7.<br>
[49]     Esposito, V., Baldi, A., De Luca, A., Micheli, P., Mazzarella, G., Baldi, F., Caputi, M. and Giordano, A. (1997) Prognostic value of p53 in non-small cell lung cancer: relationship with proliferating cell nuclear antigen and cigarette smoking. Hum Pathol 28,233-7.<br>
[50]     Caputi, M., Esposito, V., Groger, A.M., Pacilio, C, Murabito, M., Dekan, G., Baldi, F., Wolner, E. and Giordano, A. (1998) Prognostic role of proliferating cell nuclear antigen in lung cancer an immunohistochemical analysis. In Vivo 12, 85-8.<br>
[51]     Hirata, T., Fukuse, T., Naiki, H., Hitomi, S. and Wada, H. (1998) Expression of CD44 variant exon 6 in stage I non-small cell lung carcinoma as a prognostic factor. Cancer Res 58,1108-10.<br>
[52]     Ariza, A., Mate, J.L., Isamat, M., Lopez, D., Von Uexkull-Guldeband, C.RoselL R., Fernandez-Vasalo, A. and Navas-Palacios, JJ. (1995) Standard and variant CD44 isoforms are commonly expressed in lung cancer of the non-small cell type but not of the small cell type. J Pathol 177,363-8.<br>
[53]     Fasano, M., Sabatini, M.T., Wieczorek, R., Sidhu, G., Goswami, S. and Jagirdar, J. (1997) CD44 and its v6 spliced variant in lung tumors: a role in histogenesis? Cancer 80,34-41.<br>
[54]     Miyoshi, T., Kondo, K., Hino, N., Uyama, T. and Monden, Y. (1997) The expression of the CD44 variant exon 6 is associated with lymph node metastasis in non-small cell lung cancer. Clin Cancer Res 3,1289-97.<br>
[55]     Tran, T.A., Kailakury, B.V., Sheehan, C.E. and Ross, J.S. (1997) Expression of CD44 standard form and variant isoforms in non-small cell lung carcinomas. Hum Pathol 28, 809-14.<br>
[56]     Takigawa, N., Segawa, Y., Mandai, K., Takata, I. and Fujimoto, N. (1997) Serum CD44 levels in patients with non-small cell lung cancer and their relationship with clinicopathological features. Lung Cancer 18, 147-57.<br>
[57]     Kondo, K., Miyoshi, T., Hino, N., Shimizu, E., Masuda, N., Takada, M., Uyama, T. and Monden, Y. (1998) High frequency expressions of CD44 standard and variant forms in non- small cell lung cancers, but not in small cell lung cancers. J Surg Oncol 69,128-36.<br>
[58]     Sasaki, J.I., Tanabe, KJL, Takahashi, K., Okamoto, I., Fujimoto, H., Matsumoto, M., Suga, M., Ando, M. and Saya, H. (1998) Expression of CD44 splicing isoforms in lung cancers: dominant expression of CD44v8-10 in non-small cell lung carcinomas. Int J Oncol 12,525-33.<br>
[59] Voixn. M, Koomagi, R., Mattem, J. and Stammier. G. (199") Cychn A :s associated with an unfavourable outcome in patients with non-small-csil lung carcinomas. 3r J Cancer 75, 1774-3.<br>
[60]     Dobashi, Y., Shoji, M., Jiang, S.X., Kobayashi, M, Kawakubo. Y. and Kimeya. 7.<br>
(1998) Active cyciin A-CDK2 complex, a possible critical factor for ceil proliferation in human primary lung carcinomas. Am J Pathol 152, 963-~2.<br>
[61]     Volm, M., Rittgen, W. and Drings, P. (1998) Prognostic value of ERBB-i, VEGF, cyciin A, FOS, JTJN and MYC in patients with squamous cell lung carcinomas [published erratum appears in Br J Cancer 1998 Apr;77(7): 1198]. Br J Cancer 7", 663-9.<br>
[62]     Shoji. M., Dobashi, Y., Morinaga, S., Jiang, S.X. and Kameya, T. (1999) Tumor extension and cell proliferation in adenocarcinomas of the lung. Am J Pathol 154, 909-18.<br>
[63]     Shapiro, G.L, Edwards, CD., Kobzik, L., Godlesid, J., Richards, W., Sugarbaker, D.J. and Rollins, B.J. (1995) Reciprocal Rb inactivation and pl6INK4 expression in primary lung cancers and cell lines. Cancer-Res 55, 505-9.<br>
[64].     Berticher, D.C., Heighway, J., Hasieton, P.S., Altermatt, H.J., Ryder. W.D., Cemy, T. and Thatcher, N. (1996) Prognostic significance of CCND1 (cyciin Dl) overexpression in primary resected non-smali-cell lung cancer. Br J Cancer 73, 294-300.<br>
[65]     Mate, J.L., Ariza, A., Aracil, C, Lopez, D., Isamat, M., Perez-Piteira, J. andNavas-Palacios, J.J. (1996) Cyciin Dl overexpression in non-small cell lung carcinoma: correlation with Ki67 labelling index and poor cytoplasmic differentiation. J Pathol 180, 395-9.<br>
[66]     Yang, W.I., Chung, K.Y., Shin, D.H. and Kim, Y.B. (1996) Cyciin Dl protein expression in lung cancer. Yonsei Med J 37, 142-50.<br>
[67]     Berticher, D.C., Heighway, I, Thatcher, N. and Hasieton, P.S. (1997) Abnormal<br>
expression of CCND1 and RB 1 in resection margin epithelia of lung cancer patients. Br J Cancer 75, 1761-8.<br>
[68]     Nishio, M., Koshikawa, T., Yatabe, Y., Kuroishi, T., Suyama, M., Nagatake, M., Sugiura, T., Ariyoshi, Y., Mitsudomi, T. and Takahashi, T. (1997) Prognostic significance of cyciin Dl and retinoblastoma expression in combination with p53 abnormalities in primary, resected non-small cell lung cancers. Clin Cancer Res 3, 1051-8.<br>
[69]     Caputi, M., De Luca, L., Papaccio, G., A, D.A., Cavallotti, I., Scala, P., Scarano, F., Manna, M., Gualdiero, L. andDe Luca, B. (1997) Prognostic role of cyciin DI in non small cell lung cancer: an irnmunohistochemical analysis. Eur J Histochem 41, 133-8.<br>
[70] Berticher, D.C., White, GJL, Vonlanthen, S., Liu, X., Kappeler, A., Altermatt, H.J., Thatcher, N. and Heighway, J. (1997) Gl control gene status is frequently altered in resectable non-small cell lung cancer. Int J Cancer 74, 556-62.<br>
[71]     Volm, M., Koomagi, R. and Rittgen, W. (1998) Clinical implications of cyclins, cyclin-dependent kinases, RB and E2F1 in squamous-cell lung carcinoma Int J Cancer 79,294-9.<br>
[72]     Kurasono, Y., Iio, T., Kameda, Y., Nakamura. N. and Kitamura, H. (1998) Expression of cyclin DI, retinoblastoma gens protein, and plo MTS1 protein in atypical adenomatous hyperplasia and adenocarcinoma of the lung. An immunohistochemical analysis. Virchows Arch 432, 207-15.<br>
[73] Tanaka. H., Fujii. Y., Hirabayashi. H.. Miyoshi. S.. Sakaguchi. M., Yoon. E.E. and Matsuda. H. (1998) Disruption of the RB pathway and ceil-proiiferauve activity in non- .small-cell lung cancers"Int J Cancer 79, 111-5.<br>
[74]     Olivero, M., Rizzo, M., Madeddu, R.. Casadio, C. Pennacchietti. S.. Nicotra. M.R., Prat, M., Maggi, G., Arena, N., Nataii, P.G., Comogiio, P.M. and Di Renzo, M.5.<br>
(1996)	Overexprsssion and activation of hepatocyte growth factor/scatter factor in<br>
human non-small-cell iung carcinomas. Br J Cancer 74, 1862-8.<br>
[75]     Harvey, P., Warn, A, Newman, P., Perry, L.J., Ball, R.Y. and Warn, R.M. (1996)<br>
Immunoreactivity for hepatocyte growth factor/scatter factor and its receptor, met, in human iung carcinomas and malignant mesotheliomas. J Pathol 180, 389-94.<br>
[76]     Takanami, L, Tanana, F., Hashizume, T., Kikuchi, K., Yamamoto, Y., Yamamoto, T. and Kodaira, S. (1996) Hepatocyte growth factor and c-Met/hepatocyte growth factor receptor in pulmonary adenocarcinomas: an evaluation of their expression as prognostic markers. Oncology 53, 392-7.<br>
[77]     Siegfried, J.M., Weissfeld. L.A., Luketich, J.D., Weyant, R.J., Gubish C.T. and<br>
Landreneau, RJ. (1998) The clinical significance of hepatocyte growth factor for non-small cell lung cancer. Ann Thorac Surg 66, 1915-8.<br>
[78]     Nguyen, P.L., Niehans, G.A., Cherwitz, DX., Kim, Y.S. and Ho, S3. (1996)<br>
Membrane-bound (MUC1) and secretory (MUC2, MUC3, and MUC4) mucin gene expression in human lung cancer. Tumour 3iol 17,176-92.<br>
[791     Yu, C.J., Yang, P.C., Shun, C.T., Lee, Y.C., Kuo, S.H. and Luh, K.T. (1996)<br>
Overexpression of MUC5 genes is associated with early post-operative metastasis in non-small-cell lung cancer. Int J Cancer 69, 457-65.<br>
[80]     Yu, C.J., Shun, C.T., Yang, P.C., Lee, Y.C., Shew, J.Y., Kuo, SJEL and Luh, K.T.<br>
(1997)	Sialomucin expression is associated with erbB-2 oncoprotein overexpression,<br>
early recurrence, and cancer deam in non-small-cell lung cancer [published erratum<br>
appears in Am J Respir Crit Care'Med 1997 Aug; 156(2 Pt l):677-8]. Am J Respir Crit<br>
Care Med 155, 1419-27.<br>
[81]     Jarrard, LA., Linnoila, RJ., Lee, H., Steinberg, SM, Witschi, H. and Szabo, E.<br>
(1998)	MUC1 is a novel marker for the type II pneumocyte lineage during lung<br>
carcinogenesis. Cancer Res 58, 5582-9.<br>
[82]     Ohgami, A., Tsuda, T., Osaki, T., Mitsudomi, T., Morimoto, Y., Higashi, T. and Yasumoto, K. (1999) MUC1 mucin mRNA expression in stage I lung adenocarcinoma and its association with early recurrence. Ann Thorac Surg 67, 810-4.<br>
[83]     Bejarano, P.A., Baughman, R.P., Biddinger, P.W., Miller, M.A., Fenoglio-Preiser, C, al-Kafaji, B., Di Lauro, R and Whitsett, J.A. (1996) Surfactant proteins and thyroid transcription factor-1 in pulmonary and breast carcinomas. Mod Pathol 9, 445-52.<br>
[84]     Harlamert, H.A., Mira, J., Bejarano, P.A., Baughman, R.P., Miller, M.A., Whitsett,<br>
J.A. and Yassin, R. (1998) Thyroid transcription factor-1 and cytokeratins 7 and 20 in pulmonary and breast carcinoma. Acta Cytol 42, 1382-8.<br>
[351     Fontanini, G., Vignati, S., Lucchi, M., Mussi, A., Calcinai, A.. Boldrmi. L.. Chine. S.. Silvestn, V., Angeletri, C A., Basolo, F. and Bevilacqua. G. (1997) Neoangiogenesis and p53 protein in lung cancer: their prognostic roie and their relation with vascular endothelial growth factor (VEGF) expression [see comments]. Br J Cancer "5. 1295-301.<br>
[So]     Shibusa, T., Shijubo, N. and Abe, S. (1998) Tumor angiogenesis and vascuiar<br>
endothelial growth factor expression in stage I iung adenocarcinoma. Clin Cancer Res 4, 1483-7.<br>
[87]     Giatromanolaki, A, Koukourakis, MJ., Kakolyris, S., Turiey. H., O'Byme, K., Scott, PA., Pezzella, F., Georgoulias, V., Harris, A.L. and Garter, K.C. (199S) Vascular endothelial growth factor, wild-type p53, and angiogenesis in eariy operable non-small cell lung cancer. Clin Cancer Res 4, 3017-24.<br>
[88]     Fontanini, G., Boldrini, L., Vignati, S., Chine. S., Basolo, F., Silvestri. V., Lucchi, M.. Mussi, A., Angeletri, C.A. and Bevilacqua, G. (1998) Bcl2 and p53 regulate vascular endothelial growth factor (VEGF)- mediated angiogenesis in non-small ceil lung carcinoma. Eur J Cancer 34, 718-23.<br>
[89]     Takahama, M., Tsutsumi, M., Tsujiuchi, T., Kido, A, Okajirna, E., Nezu, K., Tojo, T., Kushibe, K_, Kitamura, S. and Konishi, Y. (1998) Frequent expression of the vascular endothelial growth factor in human non-small-ceil lung cancers. Jpn J Clin Oncol 28, 176-81.<br>
[90]     Sozzi, G., Miozzo, M., Tagliabue, E., Calderone, C, Lombardi, L., Pilotti, S.,<br>
Pastormo, U., Pierotti, M.A. and Delia Porta, G. (1991) Cytogenetic abnormalities and overexpression of receptors for growth factors in normal bronchial epithelium and tumor samples of lung cancer patients. Cancer Res 51, 400-4.<br>
[91]     Volm, M., Efferth, T., Mattern, J. and Wodrich, W. (1992) Overexpression of c-fos<br>
and c-erbBl encoded proteins in squamous cell carcinomas of the lung of smokers. Int J Oncol 1, 69-71 1992.<br>
[92]     Wodrich, W. and Volm, M. (1993) Overexpression of oncoproteins in non-small cell lung carcinomas of smokers. Carcinogenesis 14, 1121-4.<br>
[93]     Pastorino, U., Sozzi, G., Miozzo, M., Tagliabue, E., Pilotti, S'. arid Pierotti, M.A. (1993) Genetic changes in lung cancer. J Cell Biochem Suppl 17F, 237-48.<br>
[94]     Gorgoulis, V., Sfikakis, P.?., Karameris, A., Papastamatiou, H., Trigidou, R,<br>
Veslemes, M., Spandidos, D.A., Sfikakis, P. and Jordanoglou, J. (1995) Molecular and immunohistochemical study of class I growth factor receptors in squamous cell lung carcinomas. Pathol Res Pract 191, 973-81.<br>
[95]     Rusch, V., Klimstra, D., Linkov, I. and Dmitrovsky, E. (1995) Aberrant expression of p53 or the epidermal growth factor receptor is frequent in early bronchial neoplasia and coexpression precedes squamous cell carcinoma development. Cancer Res 55, 1365-72.<br>
[96]     Rusch, V.W. and Dmitrovsky, E. (1995) Molecular biologic features of non-small cell lung cancer. Clinical implications. Chest Surg Clin N Am 5,39-55.<br>
[97]     Fontanini, G., Vignati, S., Bigini, D., Mussi, A., Lucchi, H., Angeietti, C.A.,<br>
Pingitore, R., Pepe, S., Basolo, F. and Bevilacqua, G. (1995) Epidermal growth factor receptor (EGFr) expression in non-small cell lung carcinomas correlates with<br>
metastanc involvement of hilar and mediastinal iymph nodes in the squamous subtype. Eur J Cancer 31 A. 178-83.<br>
[98]     Pfiug, B. and Djakisw, D. (1996) Expression of:he low affinity nerve growth factor receptor in prostate epithelial cells negatively regulates nerve growth factor- mediatec growth via induction of apoptosis (Meeting abstract;. Proc Annu Meet Am Assoc Cancer Res 37, A262 1996.<br>
[99]     Rusch, V., Klimstra, D., Venkatraman, E., Langenfeid, J., Pisters. P. and Drnitrovsky, E. (1996^) Overexpression of EGFR and TGF-alpha is frequent in early stage non-small cell lung cancer, but does not predict tumor progression (Meeting aostract). Proc Annu Meet Am Assoc Cancer Res 3", A131- 1996.<br>
[100]   Fujino, S., Enokibori, T., Tezuka, N., Asada, Y., Inoue, S.. Kato, H. and Mori, A.<br>
(1996) A comparison of epidermal growth factor receptor levels and other prognostic parameters in non-small cell lung cancer. Eur J Cancer 32A 2070-4.<br>
[101]   Pastorino, U., Andreola, S., Tagliabue, E., Pezzella, F., Incarbone, M., Sozzi, G., Buyse, M., Menard, S., Pierotti, M. and Rilke, F. (1997) Immunocytochemical markers in stage I lung cancer: relevance to prognosis. J Clin Oncol 15, 2858-65.<br>
[102]   Sekine, I., Takami, S., Guang, S.G., Yokose, T., Kodama, T., Nishiwaki, Y.,<br>
Kinoshita, M., Matsumoto, H., Ogura, T. and Nagai, K. (1998) Role of epidermal growth factor receptor overexpression, K-ras pomt mutation and c-myc amplification in the carcinogenesis of non-small ceil lung cancer. Oncol Rep 5, 351-4.<br>
[103]   Pfeiffer, P., Nexo, E., Bentzen, S.M., Clausen, P.P., Andersen, K, Rose, C. and Nex, E. (1998) Enzyme-linked immunosorbent assay of epidermal growth factor receptor in lung cancer: comparisons with immunohistochemistry, clinicopathological features and prognosis. Br J Cancer 78, 96-9.<br>
[104]   D'Amico, T.A., Massey, M., Herndon, J.E., 2nd, Moore, M.B. and Harpole, D.H., Jr. (1999) A biologic risk model for stage I lung cancer, immunohistochemical analysis of 408 patients with the use often molecular markers. J Thorac Cardiovasc Surg 117, 736-43.<br>
[105]   Engel, M., Theisinger, B., Seib, T., Seitz, G., Huwer, H., Zang, K.D., Welter, C. and Dooley, S. (1993) High levels of nm23-Hl and nm23-H2 messenger RNA in human squamous- cell lung carcinoma are associated with poor differentiation and advanced tumor stages. Int J Cancer 55, 375-9.<br>
[106]   Ozeki, Y., Takishima, K and Mamiya, G. (1994) Immunohistochemical analysis of nm23/NDP kinase expression in human lung adenocarcinoma: association with tumor progression in Clara cell type. Jpn J Cancer Res 85, 840-6.<br>
[107]   Lai, W.W., Wu, M.H., Yan, JJ. and Chen, F.F. (1996) Immunohistochemical analysis of nm23-Hl in stage I non-small cell lung cancer: a useful marker in prediction of metastases. Ann Thorac Surg 62,1500-4.<br>
[108]   Gazzeri, S., Brambilla, E., Negoescu, A., Thoraval, D., Veron, M., Moro, D. and<br>
Brambilla, C. (1996) Overexpression of nucleoside diphosphate/kinase A/nm23-Hl protein in human lung tumors: association with tumor progression in squamous carcinoma. Lab Invest 74, 158-67.<br>
[109]   MacKinnon, M., Kerr, KM., King, G., Kennedy, MM., Cockbum, J.S. and Jeffrey, R.R. (1997) p53, c-erbB-2 andnm23 expression have no prognostic significance in primary pulmonary adenocarcinoma. Eur J Cardiothorac Surg 11, 838—2.<br>
[110]   Bosnar, M.E., Paveiic, K., krizanac, S., Slobodniak. Z. and Pavelic, J. (199H<br>
Squamous cell lung carcinomas: the role of nm23-Hl gene. J Mol Med 75. 609-15.<br>
[111]   Kawakubo, Y., Sato, Y.. Koh. T., Kcno. H. and Kameya. T. (1997) Expression of nm23 protein in pulmonary adenocarcinomas: inverse correlation to rumor progression. Lung Cancer 17,103-13.<br>
[112] Ritter, LH., Dresler, CM. aad Wick, M.R. (1995) Expression of bcl-2 protein in stage T1N0M0 non-small ceil lung carcinoma. Hum Pathol 26, 1227-32.<br>
[113]   Kitagawa, Y., Wong, F., Lo, P., Elliott. M.. Verburgt. L.M., Eogg, J.C. and Daya. M. (1996) Overexpression of Bcl-2 and mutations in p53 and K-ras in resected human non-small cell lung cancers. Am J Respir Cell Mol Biol 15, 45-54.<br>
[114]   Rao, S.K., Krishna, M., Woda, B A.. Savas, L. and Fraire, A.E. (1996)<br>
Lmmunohistochemical detection of bcl-2 protein in adenocarcinoma and nonneoplastic cellular compartments of the lung. Mod Pathol 9, 555-9.<br>
[115] Boers, J.E., ten Velde, G.P. and Thunrussen, F.B. (1996) P53 in squamous metaplasia: a marker for risk of respiratory tract carcinoma. Am J Respir Crit Care Med 153, 411-6.<br>
[116]   Coppola, D., Clarke, M., Landreneau, R„ Weyant, RJ., Cooper, D. and Yousem, S.A. (1996) Bcl-2, p53, CD44, and CD44v6 isoform expression in neuroendocnne tumors of the lung. Mod Pathol 9, 4S4-90.<br>
[117] Higasniyama, M., Doi, O., Kodama, K., Yokouchi, H. and Tateishi, R. (1996) Bcl-2 oncoprotein expression is increased especially in the portion of small cell carcinoma within the combined type of small cell lung cancer. Tumour Biol 17, 341-4.<br>
[118]   Strauss, GJV1. (1997) Prognostic markers in resectable non-small cell lung cancer. Hematol Oncol Clin North Am 11, 409-34.<br>
[119]   Anton, R.C., Brown, R.W., Younes, M., Gondo, MM., Stephenson, MA and Cagle, P.T. (1997) Absence of prognostic significance of bcl-2 immunopositivity in non-small cell lung cancer: analysis of 427 cases. Hum Pathol 28,1079-S2.<br>
[120]   Ishida, EL, Irie, K., Itoh, T., Furukawa, T. and Tokunaga, O. (1997) The prognostic significance of p53 and bcl-2 expression in lung adenocarcinoma and its correlation with Ki-67 growth fraction. Cancer 80,1034-45.<br>
[121]   Stefanaki, K., Rontogiannis, D., Vamvouka, C, Bolioti, S., Chaniotis, V., Sotsiou, F., Vlychou, M., Delidis, G., Kakolyris, S., Georgoulias, V. and Kanavaros, P. (1998) lmmunohistochemical detection of bcl2, p53, mdm2 and p21/wafl proteins in small-cell lung carcinomas. Anticancer Res 18, 1167-73.<br>
[122]   Brambilla, E., Gazzeri, S., Lantuejoul, S., Coll, J.L., Moro, D., Negoescu, A. and Brambilla, C. (1998) p53 mutant immunophenotype and deregulation of p53 transcription pathway (Bcl2, Bax, and Wafl) in precursor bronchial lesions of lung cancer. Clin Cancer Res 4, 1609-18.<br>
[123]   Salgia, R. and Skarin, A.T. (1998) Molecular abnormalities in lung cancer. J Clin Oncol 16, 1207-17.<br>
[124] Kim, Y.C., Park, K.O., Kern, J.A., Park, C.S., Lim, S.C., Jang, A.S. and Yang, J.B. (1998) The interactive effect of Ras, HER2, P53 and Bcl-2 expression in predicting the survival of non-small cell lung cancer patients. Lung Cancer 22, 181-90.<br>
[125]   Groeger, A.M., Caputi, M., Esposiio, V.. De Luca. A., Salar. A., Murabitc. M.. Giordano, G.G., Baidi, F., Giordano, A. and Wolner, E. (1999) 3ci-2 protein expression correlates with nodal status in non snxzil ceil lung cancer. .Anticancer Res 19* S21-4.<br>
[126]   Vargas, S.O., Leslie, K.O., Vacak, P.M., Socinski, M.A. and Weaver, D.L. (1998) Estrogen-receptor-reiated protein p29 in primary nonsmall ceil lung carcinoma: pathologic and prognostic correlations. Cancer 82. 1495-500.<br>
[127]   Higasbiyama, M.. Doi. O., Kodama, K., Yokouchi, H. and Tateishi, R. (1994)<br>
Retinoblastoma protein expression in iung cancer: an irnmimohistochernicai analysis. Oncology 51, 544-51.<br>
[128]   Xu, H.J., Quinlan, D.C., Davidson, A.G. and et.al. (1994) Altered retinoblastoma<br>
protein expression and prognosis in early stage non-small cell lung carcinoma. J. Natl. Cancer Inst. 86, 695-699. "<br>
[129]   Lee, J.S., Kalapurakal, S., Ro, J.Y. and Hong, W K (1995) Prognostic significance of retinoblastoma protein expression in non- small cell lung cancer (Meeting abstract). Proc Annu Meet Am Assoc Cancer Res 36, A3787 1995.<br>
[130] Dixon, G., Salisbury, J. and Walker, C. (1995) Expression of the retinoblastoma protein in normal and dysplastic bronchial epithelium and lung cancer (Meeting abstract). J Pathol 176, 32A 1995.<br>
[131] Shapiro, G.L, Edwards, CD.. Kobzik, L., Godleski, J., Richards, W., Sugarbaker, D.J. and Rollins, B.J. (1995) Reciprocal Rb inactivation and pl6 expression in lung cancer (Meeting abstract). Proc Annu Meet Am Assoc Cancer Res 36, A164 1995.<br>
[132] Volm, M. and Stammler, G. (1996) Retinoblastoma (Rb) protein expression and resistance in squamous cell lung carcinomas. Anticancer Res 16, 891-4.<br>
[133]   Dosaka-Akita, H., Hu, S.X., Kinoshita, I., Fujino, M., Harada, M., Kawakami, Y. and Benedict, W.F. (1996) Prognostic significance of Rb protein expression in non-small cell lung cancer (NSCLC) (Meeting abstract). Proc Annu Meet Am Assoc Cancer Res 37, A1401 1996.<br>
[134]   Kinoshita, I., Dosaka-Akita, H., Akie, K., Mishina, T., Hiroumi, H. and Kawakami, Y. (1996) Significance of abnormal p16TNK4 and RB protein expression in non-small cell lung cancer (NSCLC) (Meetmg abstract). Proc Annu Meet Am Assoc Cancer Res 37, A3979 1996.<br>
[135]   Kratzke, RA., Greatens, T.M., Rubins, J.B., Maddaus, M.A., Niewoehner, D.E.,<br>
Niehans, G.A. and Geradts, J. (1996) Rb and pl6INK4a expression in resected non-small cell lung tumors. Cancer Res 56, 3415-20.<br>
[136]   Sakaguchi, M., Fujii, Y., Hirabayashi, H., Yoon, H.E., Komoto, Y., Oue. T.,<br>
Kusafuka, T., Okada, A. and Matsuda, H. (1996) Inversely correlated expression of pl6 and Rb protein in non-small cell lung cancers: an rmmunohistochemical study. Int J Cancer 65, 442-5.<br>
[137]   Xu, H.J., Cagle, P.T., Hu, S.X., Li, J. and Benedict, W.F. (1996) Altered<br>
retinoblastoma and p53 protein status in non-small cell carcinoma of the lung: potential synergistic effects on prognosis. Clin Cancer Res 2,1169-76 1996.<br>
[138]   Dosaka-Akita, H., Hu, S.X., Fujino, M., Harada, M., Kinoshita, I., Xu, H.J.,<br>
Kuzumaki, N., Kawakami, Y. and Benedict, W.F. (1997) Altered retinoblastoma<br>
protein expression in nonsmall cell lung cancer, its synergistic erects with altered ras and p53 protein status on prognosis. Cancer 79, 1329-37.<br>
[139]   Cagle, P.T., el-Naggar, A.K., Xu, K.J., Eu. S.X. and Benedict. W.F. (199'")<br>
Differential retinoblastoma protein expression in neuroendocrine tumors of ±e iung. Potential diagnostic implications. Am J Pathol 150, 393-400.<br>
[140]   Kashiwabara, K., Oyama. t, Sano, T., Fukuda, T. and Nakajima, T. (1998)<br>
Correlation between methylauon status of the pl6/CDKN2 gene and the expression of pl6 and Rb proteins in primary non-small cell lung cancers, hat J Cancer 79, 215-20.<br>
[141]   Caputi, M, Esposito, V., Groger, AM., De Luca, A., Paciiio. C, Dekan, G.,<br>
Giordano, G.G., Baldi, F., Woiner, E. and Giordano, A. (1998) RB growth control evasion in lung cancer. Anticancer Res 18, 2371-4.<br>
[142]   Tamura, A., Matsubara, O., Hirokawa, K. and Aoki, N. (1993) Detection of thrombomodulin in human lung cancer cells. Am J Pathol 142, 79-85.<br>
[143]   Tamura, A., Komatsu, H., Hebisawa, A., Kurashima, A., Mori, M. and Katayama, T. (1996) Is thrombomodulin useful as a tumor marker of a lung cancer? Lung Cancer 15.189-95.<br>
[144]   Collins, C.L., Ordonez, N.G., Schaefer, R, Cook, CD., Xie, S.S., Granger, J., Hsu,<br>
P.L., Fink, L. and Hsu, S.M. (1992) Thrombomodulin expression in malignant pleural mesothelioma and pulmonary adenocarcinoma. Am J Pathol 1-1, 827-33.<br>
[145]   Hamatake, M., Ishida, T., Mitsudomi, T., Akazawa, K. and Sugimachi, K. (1996)<br>
Prognostic value and clinicopathological correlation of thrombomodulin in squamous cell carcinoma of the human lung. Clin Cancer Res 2, 763-6 1996.<br>
[146]   Ordonez, N.G. (1997) Value of thrombomodulin immunostaining in the diagnosis of mesothelioma. Histopathoiogy 31, 25-30.<br>
[147] Tolnay, E., Wiethege, T. and Muller, K.M. (1997) Expression and localization of thrombomodulin in preneoplastic bronchial lesions and in lung cancer. Virchows Arch 430,209-12.<br>
[ 148]   Bohm, M., Totzeck, B. and Wieland, I. (1994) Differences of E-cadherin expression levels and patterns in human lung cancer. Ann Hematol 68, 81-3.<br>
[149]   Bohm, M., Totzeck, B., Birchmeier, W. and Wieland, I. (1994) Differences of E-<br>
cadherin expression levels and patterns in primary and metastatic human lung cancer. Clin Exp Metastasis 12, 55-62.<br>
[ 150]   Peralta Soler, A., Knudsen, KA., Jaurand, M.C., Johnson, KJL, Wheelock, M.J.,<br>
Klein-Szanto, AJ. and Salazar, H. (1995) The differential expression of N-cadherin and E-cadherin distinguishes pleural mesotheliomas from lung adenocarcinomas [see comments]. Hum Pathol 26,1363-9.<br>
[151]   Han, A.C., Peralta-Soler, A., Knudsen, K.A., Wheelock, MJ., Johnson, KJL and<br>
Salazar, H. (1997) Differential expression of N-cadherin in pleural mesotheliomas and E- cadherin in lung adenocarcinomas in formalin-fixed, paraffin-embedded tissues [see comments]. Hum Pathol 28,641-5.<br>
[152]   Weynants, P., Lethe, B., Brasseur, F., Marchand, M. and Boon, T. (1994) Expression of mage genes by non-small-cell lung carcinomas. Int J Cancer 56, 826-9.<br><br>
[153]   Shicbijo, S., Hayashl, A." Takamori, S., Tsunosue, R_, Hoshino, T., Sakau. M.,<br>
Kuranioto, T., Oiznnri, K. and Itoh, K. (1995) Detection of MAGE-4 protein in lung cancers, int J Cancer 64,158-65.<br>
[154] Sakata, M. (1996) Exnression of MAGE gene farailv in lung cancers. Kurume Med J 43, 55-61.<br>
[155]   Fischer, C, Gudat, F., Stulz, 2., Noppen, C, Schaefer, C. Zajac. P., Trutmann, M., Kocher, T., Zuber, M., Haider, F., Heberer, M. and Spagnoli, G.C. (1997) High expression of MAGE-3 protein in squamous-cell lung carcinoma [letter]. Int J Cancer 71,1119-21.<br>
[156]   Gotoh, K., Yatabe, Y., Sugiura, T., Takagi, K., Ogawa, M., Takahashi, T. and<br>
Mitsudomi, T. (1998) Frequency of MAGE-3 gene expression in HLA-A2 positive patient: with non-small cell lung cancer. Lung Cancer 20, 117-25.<br>
[157] Uchiyama, B., Saijo, Y., Kumano, N., Abe, T., Fujimura, S., Ohkuda, K, Handa, M., Satoh, K. and Nukiwa, T. (1997) Expression of nucleolar protein pi20 in human lung cancer: difference in histological types as a marker for proliferation. Clin Cancer Res 3, 1873-7.<br>
[158]   Singh, G., Scheithauer, B.W. and Katyal, S.L. (1986) The pathobiologic features of carcinomas of type II pneumocytes. An immunocyto logic study. Cancer 57, 994-9.<br>
[159]   Mizutani, Y., Nakajima, T., Morinaga, S., Gotoh, M., Shimosato, Y., Akino, T. and Suzuki, A (1988) hnmunohistochemical localization of pulmonary surfactant apoproteins in various lung tumors. Special reference to nonmucus producing lung adenocarcinomas. Cancer 61,532-7.<br>
[160]   Noguchi, M., Nakajima, T., Hirohasbi, S., Akiba, T. and Shimosato, Y. (1989) Tmmunohistochemical distinction of malignant mesothehoma from pulmonary adenocarcinoma with anti-surfactant acoprotein, anti- Lewisa, and anti-Tn antibodies. Hum Pathol 20, 53-7.<br>
[161]   Linnoila, Rl, Jensen, S.M., Steinberg, SJVL, Mulshine, J L, Eggleston, J.C. and<br>
Gazdar, AF. (1992) Peripheral airway cell marker expression in non-small cell lung carcinoma. Association with distinct clinicopathologic features [see comments]. Am J Clin Pathol 97, 233-43.<br>
[162]   Shijubo, R, Tsutahara, S., Hirasawa, M., Takahashi, H., Honda, Y., Suzuki, A,<br>
Kuroki, Y. and Akino, T. (1992) Pulmonary surfactant protein A in pleural effusions. Cancer 69, 2905-9.<br>
[163]   Shijubo, N., Honda, Y.", Fujishima, T., Takahashi, H., Kodama, T., Kuroki, Y., Akino, T. and Abe, S. (1995) Lung surfactant protein-A and carcinoembryonic antigen in pleural effusions due to lung adenocarcinoma and malignant mesothehoma. Eur RespirJ8,403-6.<br>
[164]   Nicholson, A.G., McCormick, C.J., Shimosato, Y., Butcher, D.N. and Sheppard, M.N. (1995) The value of PE-10, a monoclonal antibody against pulmonary surfactant, in distinguishing primary and metastatic lung tumours. Histopathology 27, 57-60.<br>
[165]   Khoor, A., Whitsett, JA., Stahhnan, M.T. and Halter, S.A. (1997) Expression of surfactant protein B precursor and surfactant protein B mNA in adenocarcinoma of<br>
the lung. Mod Pathol 10, 62-7.<br>
[166] Saitoh, E.,SMmura, S., Fushirni, T., Okayama, E. and Shiraio, K. (199') Detection of surfactant protein-A gene transcript in the cells from pleural effusion for the diagnosis of lung adenocarcinoma. Am J Med 103, 400-i.<br>
[167]   Grohs at al, Acta Cytologica, 1996, 40(1):26-30.<br>
[168]   Grohs et al, Acta Cytologica, 1997, 41(1):1A4-152.<br><br><br><br><br><br>
I / We claim:<br>
1.	(Amended) A diagnostic kit comprising a panel for detecting a generic disease state or discriminating between specific disease states using cell-based diagnosis, comprising a plurality of probes each of which specifically binds to a marker associated with a generic or specific disease state, wherein the pattern of binding of the component probes of the panel to cells in a cytology specimen is diagnostic of the presence or specific nature of said disease state.<br>
2.	(Amended) The diagnostic kit of claim 1, wherein said generic disease state is selected from the group consisting of cancer and infectious diseases.<br>
3.	(Amended) The diagnostic kit of claim 2, wherein said cancer is selected from the group consisting of epithelial cell-based cancers, solid tumor-based cancers, secretory tumor based cancers, and blood based cancers.<br>
4.	(Amended) The diagnostic kit of claim 2 wherein said infectious disease is selected from the group consisting of cell-based diseases in which the infectious organism is a virus, bacterium, protozoan, parasite, or fungus.<br>
5.	(Amended) The diagnostic kit of claim 1, wherein said panel is optimized by using weighting factors selected from the group consisting of cost, prevalence of a generic disease state in a geographic location, prevalence of a specific disease state in a geographic location, availability of probes and commercial considerations.<br>
6.	(Amended) The diagnostic kit of claim 1, wherein each of said probes comprises a detectable label.<br>
7.	(Amended) The diagnostic kit of claim 6, wherein said probes comprise antibodies.<br>
8.	(Amended) The diagnostic kit of claim 6, wherein said label is selected from the group consisting of a chromophore, a fluorophore, a dye, a radioisotope and an enzyme.<br>
9.	(Amended) The diagnostic kit of claim 8, wherein said label is a chromophore detected using electromagnetic radiation selected from the group consisting of beta rays, gamma rays, X rays, ultraviolet radiation, visible light, infrared radiation and microwaves.<br>
10.	(Amended) The diagnostic kit of claim 1, wherein said pattern of binding is detected using photonic microscopy.<br>
11.	(Amended) The diagnostic kit of claim 10, wherein said photonic microscopy utilizes at least one electromagnetic radiation selected from the group consisting of gamma rays, X rays, beta rays, ultraviolet radiation, visible light, infrared radiation and microwaves.<br>
12.	(Amended)The diagnostic kit of claim 3, wherein said detecting is for lung cancer and said discriminating is between squamous cell carcinoma, adenocarcinoma, large cell carcinoma, small cell carcinoma and mesothelioma.<br>
13.	(Amended) The diagnostic kit of claim 1, wherein said detecting is for sexually transmitted diseases and said discriminating is between chlamydia, trichomonas, gonorrhea, herpes and syphilis.<br>
14.	A method of forming a panel for detecting a disease state or<br>
discriminating between disease states in a patient using cell-based diagnosis, comprising:<br>
(a)	determining the sensitivity and specificity of binding of probes each of which specifically binds to a member of a library of markers associated with a disease state; and<br>
(b)	selecting a limited plurality of said probes whose pattern of binding is diagnostic for the presence or specific nature of said disease state.<br><br>
15.	The method of claim 14, wherein said determining comprises: (a) separately contacting a histological or cytological sample from a patient known to be suffering from said disease and a histological or cytological sample from a patient known not to be suffering from said disease with each of said probes; (b) measuring the amount of specific binding of each probe with its complementary disease marker at loci where said marker is known to be present in cells of said samples; and (c) correlating each said amount with the presence or specific nature of said disease.<br>
16.	The method of claim 14, wherein said selecting comprises one or more of statistical analytical methods, pattern recognition methods and neural network analysis.<br>
17.	The method of claim 14, where said selecting comprises the use of weighting factors.<br>
18.A method of detecting a disease or discriminating between disease states comprising: (a) contacting a cytological sample suspected of containing abnormal cells characteristic of a disease state with a panel according to claim 1 ; and (b) detecting a pattern of binding of said probes that is diagnostic for the presence or specific nature of said disease state.<br>
19.	The method of claim 18, wherein said cytological sample is a cellular sample collected from a body fluid, an epithelial cell-based organ system, a fine needle aspiration or a biopsy.<br>
20.	The method of claim 19, wherein said cytological sample is sputum<br>
21.	A panel for detecting a generic disease state or discriminating between specific disease states using cell-based diagnosis, wherein said panel is formed according to the method of claim 14.<br>
22.	The panel of claim 21, wherein said disease state is lung cancer.<br>
23.	The panel of claim 1, wherein said disease marker is selected from the group consisting of a morphologic biomarker, a genetic biomarker, a cell cycle biomarker, a molecular biomarker and a biochemical biomarker.<br>
24.	The panel of claim 3, wherein said epithelial cell-based cancer is from the pulmonary, urinary, gastrointestinal or genital tract.<br><br>
25.	The panel of claim 3, wherein said solid tumor-based cancer is selected from the group consisting of a sarcoma, breast cancer, pancreatic cancer, liver cancer, kidney cancer, thyroid cancer, and prostate cancer.<br>
26.	The panel of claim 3, wherein said secretory tumor-based cancer is selected from the group consisting of a sarcoma, breast cancer, pancreatic cancer, liver cancer, kidney cancer, thyroid cancer, and prostate cancer.<br>
27.	The panel of claim 3, wherein said blood-based cancer is selected from the group consisting of leukemia and lymphoma.<br>
28.	The panel of claim 24, wherein said disease state is lung cancer and said probes bind to markers selected from the group consisting Glutl, HERA, FGF, telomerase, PCNA, CD44v6, cyclin A, HGF, MUC1, thyroid transcription factor, VEGF, EGF receptor, PCNA, nm23, E-cadherin, Bcl-2, cyclinDl, RB, N-Cadherin, thrombomodulin and functional equivalents thereof.<br>
29.	The method of claim 19, wherein said body fluid is selected from the group consisting of blood, urine, spinal fluid and lymph.<br>
30.	The method of claim 19, wherein said epithelial cell based organ system is selected from the group consisting of the pulmonary tract, the urinary tract, the genital tract and the gastrointestinal tract.<br>
31.	The method of claim 19, wherein said final needle aspiration is from solid tissue types in organs and systems.<br>
32. The method of claim 19, wherein said biopsy is from solid tissue types in organs and systems.<br>
33., The method of claim 31 or 32, wherein said organs and systems are selected from the group consisting of breast, pancreas, liver, kidney, thyroid, bone marrow, muscle, prostate and lung.<br>
34.	The panel of claim 23, wherein said morphologic biomarker is selected from the group consisting of DNA ploidy, MACs, and premalignant lesions.<br>
35.	The panel of claim 23, wherein said genetic biomarker is selected from the group consisting of DNA adducts, DNA mutations and apoptotic indices.<br>
36.	The panel of claim 23, wherein said cell cycle biomarker is selected from the group consisting of cellular proliferation markers, differentiation markers, regulatory molecules and apoptosis markers.<br>
37.	The panel of claim 23, wherein said molecular biomarker or<br>
biochemical biomarker is selected from the group consisting of oncogenes, tumor<br>
suppressor genes, tumor antigens, growth factors and receptors, enzymes, proteins,<br>
prostaglandins and adhesion molecules.<br>
38.	The panel of claim 24, wherein said pulmonary cancer is lung cancer.<br>
39.	The panel of claim 38 for detecting lung cancer wherein said plurality of probes comprises a probe that binds to cyclin A, or a correlate marker or functionally related marker thereof.<br>
40.	The panel of claim 39, wherein said plurality of probes further<br>
comprises one or more probes that bind to SP-B, HERA or ER-related (p29), or correlate<br>
markers or functionally related markers thereof.<br>
41.	The panel of claim 38 for discriminating adenocarcinoma among other<br>
types of lung cancers wherein said plurality of probes comprises a probe that binds to<br>
Mucin 1 and thyroid transcription factor 1, or correlate markers or functionally related<br>
markers thereof.<br>
42.	The panel of claim 41, wherein said plurality of probes further comprises one or more probes that bind to VEGF, SP-A, BCL-2, ER-related (p29) and Glut 3, or correlate markers or functionally related markers thereof.<br>
43.	The panel of claim 38 for discriminating squamous cell carcinoma among other types of lung cancers wherein said plurality of probes comprises a probe that binds to one or both of CD44v6 and ER-related(p29) or a correlate marker or functionally related marker thereof.<br>
44.	The panel of claim 43, wherein said plurality of probes further comprises one or more probes that bind to VEGF, thrombomodulin, Glut 1 and MAGE 3, or correlate markers or functionally related markers thereof.<br>
45.	The panel of claim for discriminating large cell carcinoma among other types of lung cancers wherein said plurality of probes comprises a probe that binds to one or more probes that bind to VEGF, cyclin A and P120, or correlate markers or functionally related markers thereof.<br>
46.	The panel of claim 38 for discriminating mesothelioma among other types of lung cancers wherein said plurality of probes comprises a probe that binds to one or more probes that bind to CD44v6, PCNA and HERA, or correlate markers or functionally related markers thereof.<br>
47.	The panel of claim 38 for discriminating small cell carcinoma among other types of lung cancers wherein said plurality of probes comprises a probe that binds to one or more probes that bind to PCNA, BCL-2 and EGFR, or correlate markers or functionally related markers thereof.<br>
48.	(NEW) A cell-based method with high specificity and high sensitivity for detecting lung cancer or discriminating between types of lung cancer, comprising contacting a monolayer of cells from a cytology sample on a microscope slide with a panel comprising a plurality of probes and analyzing the pattern of binding of the component probes of the panel to cells in said cytology sample to detect lung cancer or discriminate between types of lung cancer.<br>
49.	(NEW) The method of claim 48, wherein said method is for detecting lung cancer and said plurality of probes comprises a probe that binds to cyclin A or a correlate marker thereof and at least one probe that binds to a moiety selected from the group consisting of SP-B, HERA, ER-related(p29), and correlate markers thereof.<br><br>
50.	(NEW) The method of claim 48 wherein said method is for discriminating adenocarcinoma among other types of lung cancer and said plurality of probes comprises a probe that binds to Mucin 1 or a correlate marker thereof and a probe that binds to thyroid transcription factor 1 or a correlate marker thereof.<br>
51.	(NEW) The method of claim 50 wherein said plurality of probes further comprises at least one probe that binds to a moiety selected from the group consisting of VEGF, SP-A, BCL-2, ER-relalcd(p29), Glut 3 and correlate markers thereof.<br>
52.	(NEW) The method of claim 48 wherein said method is for<br>
discriminating squamous cell carcinoma among other types of lung cancer and said<br>
plurality of probes comprises a probe that binds to CD44v6 or a correlate marker thereof<br>
and a probe that binds to ER-related(p29) or a correlate marker thereof.<br>
53.	(NEW) The method of claim 52 wherein said plurality of probes<br>
further comprises at least one probe that binds to a moiety selected from the group<br>
consisting of VEGF, thrombomodulin, Glut 1, MAGE 3, and correlate markers thereof.<br>
54.	(NEW) The method of claim 48 wherein said method is for<br>
discriminating large cell carcinoma among other types of lung cancer and said plurality<br>
of probes comprises a probe that binds to VEGF or a correlate marker thereof and a<br>
probe that binds to PI20 or a correlate marker thereof.<br>
55.	(NEW) The method of claim 54 wherein said plurality of probes<br>
further comprises a probe that binds to Cyclin A or a correlate marker thereof.<br>
56.	(NEW) The method of claim 48 wherein said method is for discriminating mesothelioma among other types of lung cancer and said plurality of probes comprises a probe that binds to CD44v6 or a correlate marker thereof, a probe that binds to PCNA or a correlate marker thereof and a probe that binds to HERA or a correlate marker thereof.<br>
57.	(NEW) The method of claim 48 wherein said method is for discriminating mesothelioma among other types of lung cancer and said plurality of probes comprises a probe that binds to CD44v6 or a correlate marker thereof.<br>
58.	(NEW) The method of claim 48 wherein said method is for discriminating mesothelioma among other types of lung cancer and said plurality of probes comprises a probe that binds to PCNA or a correlate marker thereof.<br>
59.	(NEW) The method of claim 48 wherein said method is for discriminating small cell carcinoma among other types of lung cancer and said plurality of probes comprises a probe that binds to PCNA or a correlate marker thereof, a probe that binds to BCL-2 or a correlate marker thereof and a probe that binds to EGFR or a correlate marker thereof.<br>
60.	(NEW) A panel for detecting lung cancer or discriminating between<br>
types of lung cancer using cell-based diagnosis involving contacting a monolayer of cells<br>
from a cytology sample on a microscope slide with said panel, wherein<br>
(a) said panel comprises a plurality of probes each of which specifically binds to a marker associated with lung cancer or a specific type of lung cancer, and<br>
(b) the pattern of binding of the component probes of the panel to cells in said cytology sample is analyzed to detect lung cancer or discriminate between types of lung cancer.<br>
61.	(NEW) The panel of claim 60, wherein said panel is for detecting<br>
lung cancer and said plurality of probes comprises a probe that binds to cyclin A or a<br>
correlate marker thereof and at least one probe that binds to a moiety selected from the<br>
group consisting of SP-B, HERA and ER-re!ated(p29).<br>
62.	(NEW) The panel of claim 60 wherein said panel is for<br>
discriminating adenocarcinoma among other types of lung cancer and said plurality of<br>
probes comprises a probe that binds to Mucin 1 or a correlate marker thereof and a probe<br>
that binds to thyroid transcription factor 1 or a correlate marker thereof.<br>
63.	(NEW) The panel of claim 62 wherein said plurality of probes<br>
further comprises at least one probe that binds to a moiety selected from the group<br>
consisting of VEGF, SP-A, BCL-2, ER-related(p29), Glut 3, and correlate markers<br>
thereof.<br>
64.	(NEW) The panel of claim 60 wherein said panel is for<br>
discriminating squamous cell carcinoma among other types of lung cancer and said<br>
plurality of probes comprises a probe that binds to CD44v6 or a correlate marker thereof<br>
and a probe that binds to ER-related(p29) or a correlate marker thereof.<br>
65.	(NEW) The panel of claim 64 wherein said plurality of probes<br>
further comprises at least one probe that binds to a moiety selected from the group<br>
consisting of VEGF, thrombomodulin, Glut 1, MAGE 3, or correlate markers thereof.<br>
66.	(NEW) The panel of claim 60 wherein said panel is for<br>
discriminating large cell carcinoma among other types of lung cancer and said plurality<br>
of probes comprises a probe that binds to VEGF or a correlate marker thereof and a<br>
probe that binds to P120 or a correlate marker thereof; or<br>
67.	(NEW) The panel of claim 66 wherein said plurality of probes further<br>
comprises a probe that binds to Cyclin A or a correlate marker thereof.<br>
68.	(NEW) The panel of claim 60 wherein said panel is for<br>
discriminating mesothelioma among other types of lung cancer and said plurality of<br>
probes comprises a probe that binds to CD44\ 6 or a correlate marker thereof, a probe<br>
that binds to PCNA or a correlate marker thereof and a probe that binds to HERA or a<br>
correlate marker thereof.<br>
69.	(NEW) The panel of claim 60 wherein said panel is for discriminating mesothelioma among other types of lung cancer and said plurality of probes comprises a probe that binds to CD44v6 or a correlate marker thereof.<br>
70.	(NEW) The panel of claim 60 wherein said panel is for discriminating mesothelioma among other types of lung cancer and said plurality of probes comprises a probe that binds to PCNA or a correlate marker thereof.<br>
71.	(NEW). The panel of claim 60 wherein said panel is for discriminating small cell carcinoma among other types of lung cancer and said plurality of probes comprises a probe that binds to PCNA or a correlate marker thereof, a probe that binds to BCL-2 or a correlate marker thereof and a probe that binds to EGFR or a correlate marker thereof.<br><br>
72.	A diagnostic kit comprising a panel for detecting a generic disease state or discriminating between specific disease using cell-based diagnosis as herein described with reference to the examples and accompanying drawings.<br>
73.	A method of forming a panel for detecting a generic disease state or discriminating between specific disease using cell-based diagnosis as herein described with reference to the examples and accompanying drawings.<br>
74.	A cell based method with high specificity and high sensitivity for detecting lung cancer or discriminating between types of lung cancer as herein described with reference to the examples and accompanying drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wYS5wZGY=" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMjA2LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-206.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMjI0LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-224.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMzAxLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMzA4LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtMzMyLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-332.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtNDAxLnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtNDA4LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE0NjItZGVsbnAtMjAwMy1wY3QtNDI4LnBkZg==" target="_blank" style="word-wrap:break-word;">01462-delnp-2003-pct-428.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="197013-a-plate-and-internal-nozzle-assembly-for-use-in-a-distributor-in-the-steel-industry.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="197018-inflatable-balls.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>197016</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>01462/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>36/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Sep-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONOGEN INC., a US corporation</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1033,BUTTERFIELD ROAD, VERNON HILLS, IL 60061-1360, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PRESSMAN NORMAN JULES</td>
											<td>390,GREENWOOD AVENUE, GLENCOE IL 60022 U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HIRSCH KENNETH STEVEN</td>
											<td>807,PORTMAN DRIVE, REDWOOD CITY, CA 94065 U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/569</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB02/01125</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-03-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/274,638</td>
									<td>2001-03-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/197016-a-process-for-preparation-of-panel-for-cell-based-detection-and-differentiation-of-lung-cancer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:14:28 GMT -->
</html>
