<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/197371-a-process-for-the-preparation-of-a-semisynthetic-amplicon-useful-for-the-idintification-of-the-presence-of-rust-resistance-gene-lr-28-in-wheat by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:02:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 197371:&quot;A PROCESS FOR THE PREPARATION OF A SEMISYNTHETIC AMPLICON USEFUL FOR THE IDINTIFICATION OF THE PRESENCE OF RUST RESISTANCE GENE LR 28 IN WHEAT&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PROCESS FOR THE PREPARATION OF A SEMISYNTHETIC AMPLICON USEFUL FOR THE IDINTIFICATION OF THE PRESENCE OF RUST RESISTANCE GENE LR 28 IN WHEAT&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A process for the preparation of semisynthetic amplicon useful in the identification of the presence of rust resistant gene Lr 28 in wheat by isolating DNA from leaf and stem tissues of the wheat plant by conventional method enriching the said DNA for the low copy sequence DNA in a conventional Random Amplification Polymorphic DNA polymerase chain reaction resolving the amplified products by conventional electrophoresis method eluting the resistance specific , double stranded, amplified product from the gel piece by known methods synthesizing the single stranded chain of synthetic oligonucleotide by known method based on the said sequence data amplifying the said unenriched DNA in conventional Sequence Tagged Site Polymerase Chain Reaction using a pair of single stranded chain of synthetic oligonucleotide as primer to obtain semisynthetic amplicon.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to a process for the preparation of a semisynthetic amplicon useful for the identification of the presence of rust resistance gene Lr28 in wheat. More particularly it relates to the process which involves amplification of the target DNA in a Polymerase Chain Reaction (PCR) using synthetic oligonucleotide.<br>
The present invention has enabled for the first time to determine the presence of specific resistance gene L/28 at a very young stage of wheat plant non-destructively.<br>
Bread wheat (Triticum aestivum L. em Thell), belonging to festucoid tribe of family Poaceae (Gramineae) along with barley and rye, is an allopolyploid crop with three genomes (A,B and D), each with seven pairs of chromosomes that makeup the 42 chromosomes of hexapioid species (2n=6x=42). Wheat is believed to have originated from Mediterranean area and Western Asia. It is one of the most important annual winter cereals in India contributing 31.5 percent to total food grain basket of the country. It provides almost 20% of the total food calones for the people of world. India ranks third in the world, In terms of area (23.0 million hectares) as well as production (65.0 million tones). As a food, wheat is the major ingredient in most breads, rolls, chapaties, crackers, biscuits, cakes, doughnuts, muffins, pancakes, waffles, noodles, pie crust, ice cream cones, macaroni, spaghetti, puddings, pizza, bulgur, rolled flakes, many ready-to-eat breakfast foods, and baby foods. It is a common thickener in soup, gravies, and sauces and occurs in candies and beverages. Germ, barn, and malt are additional forms of wheat products.<br>
The wheat rusts, historically, have been diseases of great importance. The losses caused by these diseases world wide, over the centuries, have been substantial. The most common rust, called leaf rust or brown rust, caused by Puccinia recondita Rob. ex Desm. f. sp tritici, occurs on the leaf blade and leaf sheath. It is a widely distributed and most frequently occurring disease in all the wheat growing areas of India and can take a heavy toll of wheat production particularly in epidemic years, where the varieties can incur 5-30% loss in yield.<br>
Management of the wheat rust continues to be a major challenge to the breeders in India. Although, cultural and chemical methods have been advocated to contain the occurrence and severity of rust, the most economical and cost-effective method remains the genetic control, i.e., incorporation of resistance genes in agronomically superior varieties. Over the past 20 to 30 years, much progress has been made in relation to breeding for resistance to the rust diseases. Nonetheless, it should be emphasized that because of ever-evolving virulence in these pathogens, the rust diseases remain a concern and are economically important. If continuous and active efforts to maintain resistance level and improve its durability are not on-going, the resurgence of these diseases is virtually assured. Successful wheat production in the rust areas of the world continues to depend on the use of rust resistant cultivars. One way to fight ever-evolving pathogen is to breed for the varieties having more than one resistant genes pyramided, this will retain the variety for longer period in the field while<br>
the pathogen takes time to evolve virulence for ail the pyramided resistant genes in it.<br>
By studying the genetics of rust resistance and by a combination of allelic tests and spectrum of resistance to leaf rust pathotypes, a number of genes for leaf rust resistance have been identified. These have been given the symbol "Lr" followed by a number signifying the sequence it was being identified. These include several genes transferred from the species and genera of the tribe Triticeae.<br>
In order to determine the presence of the gene in a complex genetic background of other resistance genes, a genetic marker for it is needed. In crop species, genes conferring resistance to diseases or pests often exhibit dominance and epistasis. On the other hand it becomes very difficult to identify the presence of one resistance gene for a disease in the presence of other. As a result, It is generally considered difficult or impossible to accumulate, or pyramid resistance genes with both large and small effects in a single population or cultivars, because phenotypic selection at minor loci is ineffective when major genes are also present. The availability of modem tools of molecular bioiogy has opened new avenues to deal with such complex situations. The molecular markers, unlike resistance genes, express in codominant fashion and are less susceptible to environmental effects. Hence they are more reliable and accurate. For plant breeding purposes, selection for the resistance gene could profitably be employed by selecting indirectly with adjacent molecular markers. This would speed  breeding<br>
programs, as one type of molecular analysis could replace several disease tests. Also, indirect selection with molecular marker is very helpful in elucidating rarely-occurring recombination between resistance genes and so facilitating the combination of these closely-linked resistance genes into cultivars.<br>
The term "oligonucleotide" refers to a molecule comprising of two or more deoxyribonucleotides or ribonucleotides.<br>
The term "DNA enrichment" refers to enrichment of DNA for low copy unique sequences and removal of high copy repetitive DNA.<br>
The term "Polymerase Chain Reaction (PCR)" is an in vitro method of nucleic acid synthesis by which a particular segment of DNA can be specifically replicated. The process of PCR involves repeated cycles of heat denaturation of the DNA, annealing of the primers to their complementary sequences and extension of the annealed primers with thermostable DNA polymerase.<br>
The "Primer" refers to an oligonucleotide, whether natural or synthetic, capable of acting as a point of initiation of DNA synthesis complementary to a nucleic acid strand, in the presence of four different nucleotide triphosphates and an agent for polymerization (i.e. DNA polymerase or reverse transcriptase).<br>
The term "semisynthetic ampiicon" refers to a deoxyribonucleotide polymer in double-stranded form, amplified artificially in a polymerase chain reaction.<br>
The term "resistance specific amplicon" refers to the PCR amplification product amplified only in the rust resistant lines under specific conditions.<br>
The term "specific conditions" refers to the reaction conditions and experimental materials used herein.<br>
"Amplifying or amplification", as used herein describes both linear and exponential increase in the number of target sequence of nucleic acid.<br>
The term "Random Amplification of polymorphic DNA Polymerase Chain Reaction (RAPD-PCR) " as used herein describes a process wherein exponential increase in the number of target sequence of nucleic acid takes place by using random decamer primers having arbitrary sequence.<br>
The term "Recurrent Parent" used here in refers to the variety used repeatedly for crossing in order to develop the Near Isogenic Lines (NILs).<br>
The term "Near Isogenic Line (NIL)" used here in refers to the plants which differ from recurrent parent from which it is developed for only one character, rest of the characters remain same. The repeated crosses to the recurrent parent and selection for the character in question, ensures that the NILs receive in full all other characters of the recurrent parent except for the character under study.<br>
The term "Sequence Tagged Site Polymerase Chain Reaction" as used herein describes a process wherein exponential increase in the number of<br>
Selected target sequence of nucleic acid takes place by a pair of specific primers having sequence specificity to the selected target sequence.<br>
The object of present inventions to provide a process for the preparation of a semisyntheitic amplicon useful for the identification of the presence of rust resistance gene Lr28 in wheat.<br>
Accordingly, the present invention provides a process for the preparation of a semi synthetic amplicon useful in identification of the presence of rust resistance gene Lr28 in wheat which comprises, isolating DNA from leaf and stem tissues of the wheat plant by conventional method, enriching the said DNA for the low-copy sequences by Hydroxyl petite column, amplifying so obtained low-copy DNA in a conventional Random Amplification Polymorphic DNA Polymerase Chain Resection (RAPD-PCR), resolving the amplified products by conventional electrophoresis methods, eluting the resistance specific, double standard, amplified product from the gel piece by known methods, coloning the said product in a known vector by conventional method, sequencing the said cloned product by known methods, synthesizing the single stranded chain of synthetic oligonucleotides by known method based on the said sequence data, amplifying the said unenriched DNA in conventional Sequence Tagged Site Polymerase Chain Reaction (STS-PCR) using a pair of single stranded chain of synthetic oligonucleotides as primers to get a resistance specific semisynthetic amplicon<br>
In an embodiment of the present invention, the polymerase chain reaction is carried out by using a thermaostable DNA polymerase enzyme.<br>
In another embodiment of the present invention, the Random Amplification Polymorphic DNA Polymerase Chain Reaction (RAPD-PCR) may be effected by using one or more single stranded oligonucleotide primers, such as decamers.<br>
In another embodiment of the present invention, the Random Amplification Polymorphic DNA Polymerase Chain Reaction (RAPD-PCR) may be effected by using enriched DNA.<br>
In another embodiment of the present invention, the single copy enriched DNA may be effected by column chromatography such as hydroxiapetite column chromatography.<br>
In an another embodiment of the present invention, separation of the products of Polymerase Chain Reaction may be effected by conventional electrophoresis method using agarose gel, polyacrylamide gel and mixtures thereof.<br>
In another embodiment of the present invention, elution of the resistance specific amplification product from the gel may be effected by conventional elution methods, such as electroelution, freeze-thaw method.<br>
In another embodiment of the present invention, cloning of the resistance specific amplified, eluted product may be effected by using a plasmid vector, such as PCR product cloning vector.<br>
In another embodiment of the present invention, nucleotide sequencing of the cloned product may be effected by conventional DNA sequencing methods.<br>
In another embodiment of the present invention, Sequence Tagged Site Polymerase Chain Reaction may be effected by using synthetic oligonucleotide primers designed from the sequence of the said resistance specific amplicon.<br>
In another embodiment of the present invention, the cloned resistance specific amplicon has the following sequence:-<br>
(Sequence Removed)<br><br>
In a feature of the present invention, the enrichment of DNA for low copy sequences was effected by known methods, such as reported by Eastwood et at., 1994 (Genome 37:311-319).<br>
In a feature of the present invention, synthesis of synthetic oligonucleotide chain is effected by using known methods, such as phosphoramidite chemistry.<br>
In another feature process of the present invention, DNA was isolated from a any wheat plant tissues. A variety of techniques for extracting nucleic acids from  biological  samples   are   known,   for  example  those  described   by<br>
Sambrook et al. Molecular cloning -A laboratory manual, Cold Spring Harbor Laboratory, NY (1985) and by Roger and Bendich (Roger ef a/. 1988, PMB manual A6 pp 1-10).<br>
In yet another feature process of the present invention, oligonucleotides can be prepared by any suitable method for example, direct chemical synthesis by using phosphoramidite chemistry (Gait M.J., Oligonucleotide synthesis a practical approach, IRL press Ltd).<br>
In yet another feature process of the present invention, the amplified products of a polymerase chain reaction may be resolved by known methods for example, on agarose gel, on polyacrylamide gel, on a mixture of polyacrylamide and agarose, as described in Maniatis ef al. Molecular Cloning : A Laboratory Manual, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory, 1982.<br>
In yet another feature process of the present invention, detection of the resolved amplified products on the gel can be carried out by various conventional methods for example autoradiography of the gel, staining of the gel by known methods, such as, ethidium bromide staining, or silver staining as mentioned in Maniatis et al., Molecular Cloning : A Laboratory Manual, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory, 1982.<br>
In a process of the present invention, oligonucleotide primers and nucleotides used in the polymerase chain reaction may be detectably labeled, using any reporter element that is capable of generating a detectable signal.   Such<br>
detectable labels include radioactive markers such as 32P, 3H, 14C, or 125l, and nonradioactive markers, such as alkaline phosphate, biotin bromodeoxyuridine, fluorescent or chromogenic molecules.<br>
The method of the present invention is described herein below with reference to examples which are illustrative only and should not be construed to limit the scope of the present invention in any manner.<br>
EXAMPLE NO. 1<br>
Seeds of a Near Isogenic line (NIL) HW 2031, its recurrent parent (var. Sonalika) and the source (CS 2A/2M 4/2, from where the resistance gene, L/28 was transferred in to Sonalika to develop the NIL HW 2031) (Table 1), were grown in plastic trays in the culture room. Young leaf tissue comprising of leaves and stem of the 10-15 days old seedling was harvested and frozen in liquid nitrogen. Ten grams of frozen tissue of each sample was mechanically ground to a fine powder using mortar and pestle in liquid nitrogen.<br>
The DNA isolations were carried out by Roger and Bendich method (Roger et al. 1988, PMB manual A6 pp 1-10 ), where 15 ml of extraction buffer containing 2% CTAB ( Cetyltriethyl ammonium bromide), 100 mM Tns-HCI (pH 8.0), 20 mM EDTA (pH 8.0), 1.4 M NaCI, and 1% polyvinyl pyrrolidone was added per 10gm of frozen tissue. The slurry was incubated at 60°C for 20   minutes   and    cooled    to   room   temperature.    Equal   volume    of<br>
chlorofomwsoamyl alcohol (24:1) mixture was added and mixed thoroughjy to form an emulsion, which was centrifuged for 10 minutes at 10,000 rpm in a SS-34 rotor. To the supernatant, equal volume of CTAB precipitation buffer containing, 1% CTAB, 50 mM Tris-HCl (pH 8.0) and 10 mM EDTA (pH 8.0) was added, mixed gently and centrifuged at 10,000 rpm. The DNA pellet was dissolved in high salt TE buffer [1M NaCI, 10 mM Tris-HCl (pH 8.0), 1 mM EDTA (pH 8.0) ] and was precipitated with twice volume of absolute ethanol. The DNA precipitate was washed with 70% ethanol, centrifuged and the pellet was redissolved in TE buffer [10 mM Tris-HCl (pH 8.0), 1 mM EDTA (pH 8.0) ]. For removal of RNA, the DNA was incubated at 37°C for 1 hour with the enzyme RNAse A.<br>
The isolated DNA was quantified spectrophotometrically and subjected to DNA enrichment using following protocol reported by Eastwood er a/., 1994 (Genome 37:311-319). 115 ug of total DNA was dissolved in TE buffer [10 mM Tris-HCl (pH 8.0), 1 mM EDTA (pH 8.0) ] in 1500 µl eppendorf tubes. The DNA solution was sonicated for 10 seconds so as to get the DNA fragment size of 600 bp to 6.0 Kb. The fragment size was checked on 0.8% neutral agarose horizontal slab gel in TAE buffer (40.0 mM Tris-acetate and 2.0 mM EDTA) at a constant current with JHindIII and X174/Haelll as standards to check for the fragment sizes. Sonicated DNA was precipitated by adding 25 µl of 3M Sodium acetate (pH 5.2) followed by 1000 µl of chilled ethanol and precipitated for 30 minutes at -70°C. It was then spun at 10,000 rpm for 10 minutes at 4°C.    Supernatant was discarded and pellet was<br>
washed with 70% ethanol, dried and resuspended in 500 µl of 0.12 M Sodium Phosphate buffer (pH 6.8). The resuspended DNA was denatured by boiling for 10 minutes. Denatured DNA was incubated in the waterbath at 60°C for over 24 hours to achieve a Cot 1/2 value of over 100 by the end of time. After 24 hours the DNA was spun briefly in a microfuge, then was loaded on the column preset with 60°C water circulation to maintain the temperature. The column consisted of Hydroxylapetite (HTP) to a height of 4 cm prepared in the 10 ml of 0.12 M Sodium Phosphate buffer (pH 6.8), prewashed with 4 column volumes of 0.12 M Sodium Phosphate buffer (pH 6.8). The low copy DNA from the column was collected with 0.15 M Phosphate buffer (pH 6.8). After the collection of one sample and before loading next sample, the column was washed with 0.4 M Sodium Phosphate buffer (pH 6.8) to remove the remaining high copy DNA from the column. The concentration of the low copy DNA samples was determined using spectrophotometer.<br>
The enriched DNA was subjected to Random Amplification of Polymorphic DNA Polymerase Chain Reaction (RAPD-PCR) with a synthetic, arbitrary, 10bp primer. The final volume of the reaction mixture was 25 µl, which contained 50 ng of template DNA, 1.5 mM MgC12, 50 mM KCI, 10 mM TAPS [ 3-tri (hydroxymethyl) methyl amino propane sulphonic acid], 0.01% gelatin, 100 µl of each dATP, dGTP, dTTP, dCTP, 1.0 Unit of thermostable DNA Polymerase and 15 pmoles of decamer primer having the sequence 5'CCCGGCATAA 3'. The reaction mixture was overlaid with 30 µl of mineral oil.<br>
Amplification reaction was performed in a thermocycler, where the reaction mixture was loaded in 500 µl eppendorf tube and run on thermocycler under following conditions: Initial denaturation of 94°C for 5 minutes followed by five cycles of denaturation at 92°C for 30 s, annealing at 35°C for 2 min., and extension at 72°C for 1.5 min.; 35 cycles of denaturation at 92°C for 5 s, annealing at 40°C for 20 s, and extension at 72°C for 1.5 min.; one cycle of denaturation at 92°C for 10 s, annealing at 40°C for 20 s, and extension at 72°C for 5 min. The amplification products were analyzed on 2.0% neutral agarose horizontal slab gel in TAE buffer (40.0 mM Tris-acetate and 2.0 mM EDTA) at a constant current. After electrophoresis, gel was stained with ethidium bromide (1µg/ml) and was visualized on a long wavelength (302nm) UV transilluminator.<br>
A L/28 leaf rust resistance specific amplicon having molecular size of 0.38 Kb was identified in the resistant wheat cultivars. A gel slice containing DNA of the Lr28 leaf rust resistance specific amplicon was cut out from the gel using a sharp blade and the DNA was eluted from the gel piece as described in Maniatis et al., Molecular Cloning : A Laboratory Manual, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory, 1982.<br>
100ng of the eluted DNA was ligated to 100ng of a PCR product cloning vector in a reaction containing, 1 µl 10X DNA ligase buffer [500 mM Tris-HCI (pH 7.4), 100mM spermidine, 1mg/ml BSA (Bovine Serum Albumin)], 1 µl of 1MATP, 1 µl<br>
of 0.1M MgCI2 and 2.0 Units of the enzyme DNA ligase. Final volume of the reaction was adjusted to 10 µl by adding sterile water. The ligation reaction was carried out at 16°C for 16 hours and the ligated products were directly transformed into the competent cells of E. coli. The transformed E. coli cells were plated on LB Agar (Lurra-Bertani) medium (Bacto tryptone 10gm/lit, Bacto yeast extract 5gm/lit, Sodium chloride 10gm/lit, 1.5% Bacto agar) containing the antibiotic ampicillin (100µg/ml) Resulting colonies were screened for the presence of the recombinants as described in Maniatis ef a/., Molecular Cloning : A Laboratory Manual, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory, 1982. The plasmid DNA was isolated by inoculating a colony positive for the insert in 1ml LB medium containing the antibiotic ampicillin (100ug/ml). The culture was grown at 37°C, for 16 hours with constant shaking at 175rpm. The culture was spun in 1.5ml plastic tube for 10 minutes to pellet the cells and the supernatant was discarded. The cell pellet was suspended in 10 ul solution of GTE buffer [50mM Glucose, 25mM Tris-HCI (pH 8.0), and 10mM EDTA (pH 8.0)], vortexed and incubated at room temperature for 10 minutes. 200 µl of freshly prepared solution containing 1% SDS and 0.2N NaOH was added to the cell suspension, mixed well by tapping the tube and incubated on ice for 10 minutes, and further purified as described by Maniatis ef al., [Molecular Cloning : A Laboratory Manual, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory, 1982],<br>
The cloned PCR fragment was sequenced by Sanger's dideoxy chain termination method, as described by Sanger et at. PNAS 74, 1977:54-63.<br>
To denature the template DNA, 5 ug of plasmid DNA was dried in an eppendorf tube and dissolved in 40 ul denaturation buffer (0.2M NaOH, 0.2 mM EDTA, pH 8.0) and kept at 37°C for 30 minutes. 4 µl of 3M Sodium acetate, pH 5.2 was added, followed by 100 ul of chilled ethanol and precipitated for 30 minutes at -70°C. The sample was spun at 10,000 rpm for 10 minutes at 4°C. Supernatant was discarded and pellet was washed with 70% ethanol, dried and dissolved in 7µl of sterile water.<br>
To anneal the sequencing primer to the template DNA, 1ul of sequencing primer and 2ul of 5X reaction buffer were added to the template DNA and incubated at 65°C for 2 minutes.<br>
The labeling reaction was carried out by adding, 1µl DTT (0.1 M), 2ul labeling nucleotide mix (7.5 µm each of dGTP, dTTP, dATP and dCTP), 5µCi of α-32P dATP and 0.6 units of enzyme to the annealed template-primer. The sample was incubated for 5 minutes at room temperature.<br>
2.5µl of dideoxynucleotide mixture (ddGTP, ddTTP, ddATP and ddCTP) was taken in four labeled tubes and wanned to 37°C for 2 minutes. 3.5 ul of the template primer mixture was added to each of the labeled tubes, mixed and incubated at 37°C for 5 minutes. After the incubation, 4 ul of formamide buffer (95% Formamide, 20 mM EDTA, 0.05% Xylene Cyanol FF) was added to stop the reaction.<br>
The sequencing reaction was then resolved on a 6%, 0.4 mm thick, denaturing polyacrylamide gel. Electrophoresis was carried out at a constant voltage (2000V) in 1X TBE buffer (89 mM Tris-borate and 2 mM EDTA). After the electrophoresis, the gel was covered in a thin plastic sheet and exposed to an x-ray film at -70°C for 16 hours.<br>
Sequencing of the cloned amplicon revealed .a sequence as shown in the sequence identity no. 1<br>
SEQUENCE IDENTITY NO. 1:<br>
(Sequence Removed)<br><br>
From the sequence data, the primers were designed so as to amplify the entire sequence of the amplicon. The sequence of the oligonucleotide primers is as given in sequence identity no. 2 and sequence identity no. 3.<br>
SEQUENCE IDENTITY NO. 2:<br>
5' CCCGGCATAAGTCTATGGTT 3' SEQUENCE IDENTITY NO. 3:<br>
5' CAATGAATGAGATACGTGAA 3'<br>
The isolated unennched DNA of all the three plant samples were subjected to Sequence Tagged Polymerase Chain Reaction (STS-PCR) using the pair of synthetic oligonucleotide primers as mentioned in sequence identity no. 2<br>
and sequence identity no. 3. The reaction mixture contained, 50 ng of template DNA, 1.5 mM of MgCI2, 50 mM KCI, 10 mM TAPS [3-tri(hydroximethyl) methyl amino propane sulphonic acid], 0.01% gelatin, 100 uM of each dATP, dCTP, dGTP and dTTP, 0.5 units of thermostable DNA polymerase, 15 pmoles of each of the synthetic oligonucleotide primers and the final volume was made up to 25 µl by adding sterile water. The reaction mixture was overlaid with 30 µl of mineral oil.   .<br>
Amplification reaction was performed in a thermocycler, where the reaction mixture was incubated at 94°C for 6 minutes, followed by 35 cycles of 94°C for 1 minute, 50°C for 1 minute, and 72ºC for 2 minutes. The amplification reaction was concluded by a final extension of 72°C for 5 minutes. The amplification product was analyzed on a 2.0% neutral agarose horizontal slab gel in TAE buffer(0.09M Tns-acetate and 2 mM EDTA) at a constant current. After electrophoresis, gel was stained with ethidium bromide (µg/ml) and was visualized on a long wavelength (302 nm) UV transilluminator.<br>
It was observed that using the said pair of synthetic oligonucleotides in a Sequence Tagged Polymerase Chain Reaction, only a single amplicon of 378 bp in size was amplified specific to the resistant lines, source (CS 2A/2M 4/2) and NIL (HW 2031), enabling us to accurately identify the resistant genotype.<br>
The following examples 2 and 3 relate to the identification of leaf rust resistant genotypes of wheat containing Lr28 gene using semisynthetic amplicon.<br>
EXAMPLE NO. 2<br>
Seeds of 8 Near Isogenic Lines (NILs) (Table 1) apart from HW 2031 and their recurrent parents, along with the source (CS 2A/2M 4/2) were grown in plastic trays. Young tissue comprising of leaves and stem of the 10-15 days old seedling was harvested and frozen in liquid nitrogen. Ten grams of frozen tissue of each sample was mechanically ground to a fine powder using mortar and pestle in liquid nitrogen.<br>
The DNA isolations were carried out by Roger and Bendich method (Roger et al. 1988, PMB manual A6 pp 1-10 ), where 15 ml of extraction buffer containing 2% CTAB ( Cetyltriethyl ammonium bromide), 100 mM Tris-HCI (pH 8.0), 20 mM EDTA (pH 8.0), 1.4 M NaCI, and 1% polyvinyl pyrrolidone was added per 10gm of frozen tissue. The slurry was incubated at 60°C for 20 minutes and cooled to room temperature. Equal volume of chloroform:isoamyl alcohol (24:1) mixture was added and mixed thoroughly to form an emulsion, which was centrifuged for 10 minutes at 10,000 rpm in a SS-34 rotor. To the supernatant, equal volume of CTAB precipitation buffer containing, 1% CTAB, 50 mM Tris-HCI (pH 8.0) and 10 mM EDTA (pH 8.0) was added, mixed gently and centrifuged at 10,000 rpm. The DNA pellet was dissolved in high salt TE buffer [1M NaCI, 10 mM Tris-HCI (pH 8.0), 1 mM EDTA (pH 8.0) ] and was precipitated with twice volume of absolute ethanol. The DNA precipitate was washed with 70% ethanol, centrifuged and the pellet was redissolved in TE buffer [10 mM Tris-HCI (pH 8.0), 1 mM EDTA<br>
(pH 8.0) ]. For removal of RNA, the DNA was incubated at 37°C for 1 hour with the enzyme RNAse A.<br>
The isolated DNA was quantified spectrophotometrically and subjected to polymerase chain reaction with a pair of synthetic oligonucleotide primers having the sequence identity no. 2 and sequence identity no 3.<br>
SEQUENCE IDENTITY NO. 2:<br>
5' CCCGGCATAAGTCTATGGTT 3' SEQUENCE IDENTITY NO. 3:<br>
5' CAATGAATGAGATACGTGAA 3'<br>
The reaction mixture contained, 50 ng of template DNA, 1.5 mM Of MgCI2, 50 mM KCI, 10 mM TAPS [3-tri(hydroximethyl) methyl amino propane sulphonic acid], 0.01% gelatin, 100 µM of each dATP, dCTP, dGTP and dTTP, 0.5 units of thermostable DNA polymerase, 15 pmoles of each of the synthetic oligonucleotide primers and the final volume was made up to 25 µl by adding sterile water. The reaction mixture was overlaid with 30 µl of mineral oil.<br>
Amplification reaction was performed in a thermocycler, where the reaction mixture was incubated at 94°C for 6 minutes, followed by 35 cycles of 94°C for 1 minute, 50°C for 1 minute, and 72°C for 2 minutes. The amplification reaction was concluded by a final extension of 72°C for 5 minutes. The amplification product was analyzed on a 2.0% neutral agarose horizontal slab gel in TAE buffer(0.09M Tris-acetate and 2 mM EDTA) at a constant current.<br>
After electrophoresis, gel was stained with ethidium bromide (1µg/ml) and was visualized on a long wavelength (302 nm) UV transilluminator.<br>
It was observed that using the said pair of synthetic oligonucleotides in a sequence tagged site polymerase chain reaction, only a single amplicon of 378 bp in size was amplified specific to the resistant lines, source (CS 2A/2M 4/2) and eight NILs but absent in the recurrent parents for the corresponding NILs (table 1), enabling us to accurately identify the resistant genotype.<br><br>
(Table Removed)<br>
Seeds from the 5 resistant and 5 susceptible lines identified from the F3 population derived from the cross of HW 2035 and Nl 5439, along with the source (CS 2A/2M 4/2) were grown in plastic trays. Young tissue comprising of leaves and stem of the 10-15 days old seedling was harvested and frozen in  liquid  nitrogen.     Ten grams  of frozen  tissue of each  sample was<br>
mechanically ground to a fine powder using mortar and pestle in liquid nitrogen.<br>
The DNA isolations were carried out by Roger and Bendich method (Roger et al. 1988, PMB manual A6 pp 1-10 ), where 15 ml of extraction buffer containing 2% CTAB ( Cetyltriethyl ammonium bromide), 100 mM Tns-HCI (pH 8.0), 20 mM EDTA (pH 8.0), 1.4 M NaCI, and 1% polyvinyl pyrrolidone was added per 10gm of frozen tissue. The slurry was incubated at 60°C for 20 minutes and cooled to room temperature. Equal volume of chloroform:isoamyl alcohol (24:1) mixture was added and mixed thoroughly to form an emulsion, which was centrifuged for 10 minutes at 10,000 rpm in a SS-34 rotor. To the supernatant, equal volume of CTAB precipitation buffer containing, 1% CTAB, 50 mM Tris-HCI (pH 8.0) and 10 mM EDTA (pH 8.0) was added, mixed gently and centrifuged at 10,000 rpm. The DNA pellet was dissolved in high salt TE buffer [1M NaCI, 10 mM Tris-HCI (pH 8.0), 1 mM EDTA (pH 8.0) ] and was precipitated with twice volume of absolute ethanol. The DNA precipitate was washed with 70% ethanol, centrifuged and the pellet was redissolved in TE buffer [10 mM Tris-HCI (pH 8.0), 1 mM EDTA (pH 8.0) ]. For removal of RNA, the DNA was incubated at 37°C for 1 hour with the enzyme RNAse A.<br>
The isolated DNA was quantified spectrophotometrically. Equal amount of DNA from all the resistant lines was mixed to make a resistant DNA bulk. Likewise, equal amount of DNA from ail the susceptible lines was mixed to make a susceptible DNA bulk.    Both the DNA bulks, all the resistant and<br>
susceptible lines and the source DNA after quantification were subjected to polymerase chain reaction with a pair of synthetic oligonucleotide primers having the sequence identity no. 2 and sequence identity no. 3.<br>
SEQUENCE IDENTITY NO. 2:<br>
5' CCCGGCATAAGTCTATGGTT 3' SEQUENCE IDENTITY NO. 3:<br>
5' CAATGAATGAGATACGTGAA 3'<br>
The reaction mixture contained, 50 ng of template DNA, 1.5 mM Of MgCI2, 50 mM KCI, 10 mM TAPS [3-tri(hydroximethyl) methyl amino propane suiphonic acid], 0.01% gelatin, 100 µM of each dATP, dCTP, dGTP and dTTP, 0.5 units of thermostable DNA polymerase, 15 pmoles of each of the synthetic oligonucleotide primers and the final volume was made up to 25 µl by adding sterile water. The reaction mixture was overlaid with 30 µl of mineral oil.<br>
Amplification reaction was performed in a thermocycler, where the reaction mixture was incubated at 94°C for 6 minutes, followed by 35 cycles of 94°C for 1 minute, 50°C for 1 minute, and 72°C for 2 minutes. The amplification reaction was concluded by a final extension of 72°C for 5 minutes. The amplification product was analyzed on a 2.0% neutral agarose horizontal slab gel in TAE buffer(0.09M Tris-acetate and 2 mM EDTA) at a constant current. After electrophoresis, gel was stained with ethidium bromide (1ug/ml) and was visualized on a long wavelength (302 nm) UV transilluminator.<br>
It was observed that using the said pair of synthetic oligonucleotides in a sequence tagged polymerase chain reaction, only a single amplicon of 378 bp in size was amplified specific to the resistant bulk and lines and source (CS 2A/2M 4/2) but absent in the susceptible bulk and lines, enabling us to accurately identify the resistant genotype.<br>
MAIN ADVANTAGES OF PRESENT INVENTION:<br>
1.	The important advantage of the present invention is to provide a duplex polynucleotide useful for the identification of plants carrying a rust resistance gene, designated as Lr28.<br>
2.	With this invention, resistant genotypes can be identified in the seedling stage of the breeding population, without having to infect the plants artificially either in glass house or in the field, with rust spores. This invention will thus be more economical and precise, since the development of rust disease in field or glass house depends on the environmental conditions and the selection of plants carrying the gene can be defective in the absence of proper infection.<br>
3.	The present invention will be useful in pyramiding Lr28 gene with other effective resistance genes in a single genotype, which is not possible by conventional host parasite interactions. This pyramiding or combination of more than one resistant gene in a single plant will prolong the life span of a high-yielding variety by protecting against the new races of leaf rust and reduce the frequency of replacement of variety.<br><br><br><br><br><br><br>
We Claim:<br>
1.	A process for the preparation of a semi synthetic amplicon useful in identification of the presence of rust resistance gene Lr28 in wheat which comprises, isolating DNA from leaf and stem tissues of the wheat plant by conventional method, enriching the said DNA for the low-copy sequences by Hydroxyl petite column, amplifying so obtained low-copy DNA in a conventional Random Amplification Polymorphic DNA Polymerase Chain Resection (RAPD-PCR), resolving the amplified products by conventional electrophoresis methods, eluting the resistance specific, double standard, amplified product from the gel piece by known methods, coloning the said product in a known vector by conventional method, sequencing the said cloned product by known methods, synthesizing the single stranded chain of synthetic oligonucleotides by known method based on the said sequence data, amplifying the said unenriched DNA in conventional Sequence Tagged Site Polymerase Chain Reaction (STS-PCR) using a pair of single stranded chain of synthetic oligonucleotides as primers to get a resistance specific semisynthetic amplicon.<br>
2.	A process claimed in claim 1 wherein, the polymerase chain reaction is carried out using a thermo stable DNA polymerase.<br>
3.	A process as claimed in claims 1 and 3 wherein, the Random Amplification Polymorphic DNA Polymerase Chain Reaction (RAPD-PCR) may be effected by the process of DNA enrichment and selection of proper DNA fragment sizes.<br>
4.	A process as claimed in claims 1, 3 and 4 wherein, the Random Amplification Polymorphic DNA Polymerase Chain Reaction (RAPAD- PCR) may be effected by using one or more single stranded oligonucleotide primers, such as known decamers.<br>
5.	A process as claimed in claims 1, 3-5 wherein, separation of resistance specific products of Polymerase Chain Reaction my be effected by conventional electrophoresis method using agarose gel, polyacrylamide gel mixtures thereof.<br>
6.	A process as claimed in claims 1, 3-6 wherein, elution of the resistance specific amplification product from the gel piece may be effected by conventional elution methods, such as electro elution; freeze-thaw method.<br>
7.	A process as claimed in claims 1, 3-7 wherein, cloning of the resi stance specific amplified, eluted product may be effected by using a plasmid vector, such as PCR product cloning vector.<br>
8.	A process as claimed in claims 1-8, wherein, Sequence Tagged,Site Polymerase Chain Reaction is effected by using synthetic oligonucleotide/primers spanning the sequence of the said resistance specific amplicon.<br>
9.	A process for the preparation of semisynthetic amplicon useful for the identification of the presence of one of the important rust resistance gene Lr28 in wheat as herein described with reference to the examples.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">380-del-1998-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">380-del-1998-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">380-del-1998-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">380-del-1998-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">380-del-1998-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">380-del-1998-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">380-del-1998-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">380-del-1998-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzgwLWRlbC0xOTk4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">380-del-1998-form-2.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="197370-a-process-for-preparing-a-hydrate-salt-of-5-4-2-n-methyl-n-2-pyridyl-amino-ethoxy-benzyl-thiazolidine-2-4-dione-maleic-acid.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="197375-method-and-apparatus-for-analyzing-image-structures.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>197371</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>380/DEL/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Nov-2006</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Oct-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Feb-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG,NEW DELHI-110 001, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VIDYA SHRIKANT GUPTA</td>
											<td>NATIONAL CHEMICAL LABORATORY, PUNE -411 008, MAHARASHTRA, INDIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SURESH NAIK</td>
											<td>NATIONAL CHEMICAL LABORATORY, PUNE -411 008, MAHARASHTRA, INDIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PRABHAKAR KAMALAKAR RANJEKAR</td>
											<td>NATIONAL CHEMICAL LABORATORY, PUNE -411 008, MAHARASHTRA, INDIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VELIVENTI SURYA PRAKASA RAO</td>
											<td>AGHARKAR RESEARCH INSTITUTE, PUNE-411 004, MAHARASHTRA, INDIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/197371-a-process-for-the-preparation-of-a-semisynthetic-amplicon-useful-for-the-idintification-of-the-presence-of-rust-resistance-gene-lr-28-in-wheat by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:02:47 GMT -->
</html>
