<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/199576-a-method-for-providing-the-multiplex-structure by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:05:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 199576:&quot;A METHOD FOR PROVIDING THE MULTIPLEX STRUCTURE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR PROVIDING THE MULTIPLEX STRUCTURE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A multiplex structure, such as a nucleic acid quadruplex, includes: a first strand containing a first sequence of nucleobases; a second strand containing a second sequence of nucleobases, wherein the second strand is associated with the first strand by Watson-Crick bonding; a third strand containing a third sequence of nucleobases; and a fourth strand containing a fourth sequence of nucleobases, wherein the fourth strand is associated with the second strand and the third strand by Watson-Crick bonding. Formation of the multiplex structure is promoted by monovalent cations (e.g., sodium and potassium), divalent cations, multivalent cations, intercalating agents and/or molecules known to bind within the minor grooves of nucleic acids. The multiplex structure and the process of forming it have diagnostic, therapeutic, prophylactic and nanoengineering applications.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of Invention:<br>
This invention relates to a method for providing the multiplex structure and more particularly to methods wherein quadruplexes are formed by specific binding between double-stranded nucleic acid probes and double-stranded nucleic acid target sequences.<br>
2.       Description of related Art:<br>
Although nucleic acid duplexes are the most widely studied type multiple-strand nucleic acid structures, it has been discovered that nucleic acids also form triplex and quadruplex structures under certain conditions.<br>
Until recently, hybridization among three nucleic acid strands to form a triplex was widely believed to be confined to very limited species of nucleic acids (e.g., polypurine or polypyrimidine sequences). See, e.g., Floris et al., "Effect of cations on purine-purine-pyrimidine triple helixformation in mixed-valence salt solutions," 260 Eur. J. Biochem. 801-809 (1999). Moreover, triplex formation or hybridization was thought to be based on Hoogsteen binding between limited varieties of adjacent nucleobases, rather than Watson-Crick base pairing. See, e.g., Floris et al. and U.S. Patent No. 5,874,555 to Dervan et al. However, the inventors have recently disclosed in several patent applications that triplex nucleic acids based on Watson-Crick base pairing can be created and used as the basis for a highly accurate and sensitive assay for specific binding. See U.S. patent applications Nos. 09/613,263 and 09/468,679, respectively filed July 10, 2000 and December 21, 1999.<br>
As was the case with triplex nucleic acids, the conventional wisdom regarding quadruplex nucleic acids has been that such peculiar structures only exist under relatively extreme conditions for a relatively narrow class of nucleic acids. In particular, Sen et al. (Nature 334:364-366 (1988)) disclosed that guanine-rich cdigonucleotides can spontaneously self-assemble into four-stranded helices in vitro. Sen et al. (Biochemistry 31:65-70 (1992)) disclosed that these four-stranded complexes can further associate into superstructures composed of 8, 12, or 16 oligomeref.<br>
Marsh et al. (Biochemistry 33:10718-10724 (1994), and Nucleic Acids Research 23:696-700 (1995)) disclosed that some guanine-rich oligonucleotides can also assemble in an offset, parallel alignment, forming long "G-wires". These higher-order structures are stabilized by G-quartets that consist of four guanosine residues arranged in a plane and held together through Hoogsteen base pairings. According to Sen et al. (Biochemistry 31:65-70 (1992)), at least three contiguous guanines within the oligomer are critical for the formation of these higher order structures.<br>
It has been suggested that four-stranded DNAs play a role in a variety of biological processes,  such as inhibition  of  HIV-1  integrase  (Mazumder  et  al., Biochemistry 35:13762-13771 (1996)), formation of synapsis during meiosis (Senetal., Nature 334 :364-366 (1988)), and telomere maintenance (Williamson et al., Cell 59:871-880 (1989)); Baran et al., Nucleic Acids Research 25:297-303 (1997)) .<br>
It has been further suggested that controlling the production of guanine-rich quadruplexes might be the key to controlling such biological processes. For example, U.S. Patent No. 6,017,709 to Hardin et al. suggests that<br>
telomerase activity might be controlled through drugs that inhibit the formation of guanine quartets.<br>
U.S. Patent No. 5,888,739 to Pitner et al. discloses that G-quartet based quadruplexes can be employed in an assay for detecting_jriucleic acids. Upon hybridization to a complementary oligonucleotide, the G-quartet structure unfolds or linearizes, thereby increasing the distance between a donor and an acceptor on different parts of the G-quartet structure, resulting in a decrease in their interaction and a detectable change in a signal (e.g., fluorescence) emitted from the structure.<br>
U.S. Patent No. 5,912,332 to Agrawal et al. discloses a method for the purification of synthetic oligonucleotides, wherein the synthetic oligonucleotides hybridize specifically with a desired, full-length oligonucleotide and concomitant ly form a mul timer aggregate, such as quadruplex DNA, The multimer aggregate containing the oligonucleotide to be purified is then isolated using size-exclusion techniques.<br>
Despite the foregoing developments, the full potential of quadruplex nucleic acid has neither been fully appreciated nor fully exploited.<br>
All references cited herein are incorporated herein by reference in their entireties.<br>
SUMMARY OF THE   INVENTION<br>
The invention provides a multiplex structure comprising: a first strand containing a first sequence of nucleobases; a second strand containing a second sequence of nucleobases, wherein the second strand is associated with the first strand by Watson-Crick bonding/ a third strand containing a third sequence of nucleobases; and a fourth strand containing a fourth sequence of nucleobases,<br>
wherein the fourth strand is associated with the second strand and the third strand by Watson-Crick bonding.<br>
Also provided is a    method for providing the    multiplex structure comprising:<br>
a first strand containing a first sequence of nucleobases;<br>
a second strand containing a second sequence of nucleobases, wherein said second strand is associated with said first strand by Wats on-Crick bonding;<br>
a third strand containing a third sequence of nucleobases; and<br>
a fourth strand containing a fourth sequence of nucleobases, said fourth strand is associated with said second strand and said third strand by Watson-Crick bonding, and at least one nucleobase of said fourth sequence of nucleobases is associated by Watson-Crick bonding to at least one nucleobase of said third sequence of nucleobases and to at least one nucleobase of said second sequence of nucleobases,<br>
wherein the method comprising:<br>
providing a hybridization medium comprising said first strand, said second strand, said third strand, said fourth strand, water, a buffer and at least one promoter, and<br>
incubating said hybridization medium for an incubation time effective to hybridize said second strand to said fourth strand to provide said multiplex structure.<br>
Still further provides is an assay for single-stranded and double-stranded targets employing a double-stranded probe.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention will be described in conjunction with the following drawings in which like reference numerals designate like elements and wherein:<br>
Figs. 1, 2, 3, 4 and 5 are composite graphs of fluorescent intensity plotted as a function of wavelength for each sample analyzed.<br>
DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS<br>
Unlike the quadruplexes discussed in the Background Section above, the preferred multiplex structures of the invention contain at least four strands of nucleic acid bonded together according to traditional Watson-Crick bonding rules.<br>
As used herein, the term "Watson-Crick bonding" is intended to define specific association between opposing pairs of nucleic acid (and/or nucleic acid analogue) strands via matched, opposing bases. While the formation of a Watson-Crick quadruplex may sometimes be referred to as a hybridization event herein, . that is merely for convenience and is not intended to limit the scope of the<br><br>
invention with respect  to  how  the  formation  of  a Watson-Crick quadruplex can be best characterized.<br>
The multiplex structures of the invention are preferably quadruplexes. Each strand of the multiplex independently comprises a nucleic acid or a nucleic acid analogue. Suitable nucleic acids include, e.g., DNA or RNA. Preferred nucleic acid analogues contain an uncharged or partially charged backbone (i.e., a backbone having a charge that is not as negative as a native DNA backbone) .<br>
In certain embodiments, one of the second and fourth strands of the four-stranded quadruplex comprises DNA and the other of the second and fourth strands comprises RNA, mRNA, hnRNA, rRNA, tRNA or cDNA.<br>
In certain embodiments, the second strand and the fourth strand are anti-parallel to each other. These embodiments are defined as having mirror complementarity. In these embodiments, a major groove of the first and second strands is placed in a major groove of the third and fourth strands.<br>
In other embodiments, the second and fourth strands are parallel to each other. In these embodiments, which possess "nested complementarity," a major groove of the first and second strands is placed in a minor groove of the third and fourth strands.<br>
In certain embodiments, each nucleobase binds to no more than two other nucleobases. In some of these embodiments, the bases of the second strand specifically bond (via Watson-Crick rules) to the matching bases of the first strand and to the matching bases of the fourth strand, and the bases of the fourth strand specifically bond (via Watson-Crick rules) to the matching bases of the third strand and to the matching bases of the second strand, wherein the bases of the first and third strands<br>
bind to no more than one other base each. Thus, in addition to the traditional Watson-Crick base pairs, such embodiments include the following Watson-Crick base triplets: A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and/or C-G-C (including C+-G-C, and/or any other ionized species of base).<br>
In certain embodiments, it is believed that opposing bases of the first and third strands also bind to each other, in addition to: (a) the binding between opposing bases of the first and second strands; (b) the binding between opposing bases of the third and fourth strands; and (c) the binding between opposing bases of the second and fourth strands. _-<br>
In certain embodiments of the multiplex structure of the invention, no strand is contiguous with another strand. That is, there are at least four separate strands. Although folded conformations and the like (e.g., hairpin turns, etc.) are within the scope of the invention, folded portions of a single strand do not make the strand count more than once toward the minimum of four separate strands.<br>
Multiplex structures of the invention preferably do not rely on Hoogsteen bonding or G-G quartets for maintenance of the multiplex structure, although insignificant amounts of Hoogsteen bonding and/or G-G quartets may be present. That is, multiplex structures of the invention are preferably substantially free of Hoogsteen bonding, and substantially free of G-G quartets.<br>
In certain embodiments, the first and second strands of the multiplex are 5 to 50 bases long (more preferably 5 to 30 bases long) and the third and fourth strands are 8 to 3.3 X 109 base pairs long. For example, the first and second strajgda^J2an__cjQnstitute a double-stranded probe and the third and fourth strands can constitute a<br><br>
double- stranded target, such as genomic DNA, which can<br>
'<br><br>
In embodiments, the third_sjtrancrand^ the fourth, strand<br>
are PCR amplified products.<br>
The multiplexes of the invention can be present in solution, on a solid support, in vitro or in vivo. The solid support can be ^lectrica.lly conductive (e.g., an electrode) or non- conductive*<br>
Quadruplex formation according to the invention is suitable for a variety of uses. For example, double- stranded probes covalently bound to a double- stranded nucleic acid cleaving agent can be used to specifically cleave target sequences of double -stranded nucleic acids. Double-stranded probes covalently bound to a chemotherapeutic agent can be used to specifically treat V target sequences of double- stranded nucleic acids. Thus, v-.the invention encompasses multiplex structures further comprising a therapeutic, prophylactic or diagnostic agent bound to at least one of the first, second, third and fourth strands.<br>
In addition, multiplexes of the invention are suitable for use in nanoengineering, such as to provide electrical circuitry on a molecular (i.e., nanoscale) level. Further details regarding nanoengineering with nucleic acids can be found in U.S. Patent No. 5,948,897 to Sen et al. and the references cited therein.<br>
Multiplex structures of the invention can be provided by a method comprising: providing a hybridization medium comprising the first strand, the second strand, the third strand, the fourth strand, water, a buffer and a promoter; and incubating the hybridization medium for an incubation time effective to hybridize the second strand to the fourth strand.<br>
The hybridization medium can include any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al. , "Molecular Cloning: A Lab Manual," Vol. 2 (1989). For example, the medium can comprise nucleotides, water, buffers and standard salt concentrations. When divalent cations are used exclusively to promote quadruplex formation, chelators such as EDTA or EGTA should not be included in the reaction mixtures.<br>
Specific binding between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition. Examples of these conditions and methods for applying them are known in the art.<br>
Unlike many Hoogsteen-type multiplexes, which are unstable or non-existent at pH levels above about 7.6, the Watson-Crick multiplexes of the invention are stable over a wide range of pH levels, preferably from about pH 5 to about pH 9.<br>
Moreover, the inventive multiplexes do not require the presence of homopyrimidine sequences or homopurine sequences, as in certain prior art quadruplexes. For example, the target sequence can contain 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order.<br>
It is preferred that multiplexes be formed at a temperature of about 5°C to about 25°C for about two hours or less. The incubation time is preferably less than five minutes, even at room temperature. Longer reaction times are not required, but incubation for up to 24 hours in most cases did not adversely affect the quadruplexes. The fast binding times of Watson-Crick quadruplexes of the invention contrast with the much longer binding times for Hoogsteen quadruplexes.<br>
The promoter in the hybridization medium is preferably an intercalating agent or a cation. The intercalating agent can be, e.g., a fluorophore, such as a member selected from the group consisting of YOYO-1, TOTO-1, ethidium bromide, ethidium homodimer-1, ethidium homodimer-2 and acridine.<br>
Suitable cations include, e.g., monovalent cations, such as Na+ (preferably at a concentration of 50mM to !25mM), K+, and other alkali metal ions; divalent cations, such as alkaline earth metal ions (e.g., Mg+2 and Ca+2) and divalent transition metal ions (e.g., Mn+2, Ni*2, Cd+2, Co*2 and Zn+2) ; and cations having a positive charge of at least three,  such  as  Co(NH3)6+3,  trivalent  spermidine  and tetravalent spermine.  Mn+2 is preferably provided at a concentration of lOmM to 30mM. Mg*2 is preferably provided at a concentration of 15mM to 20mM.  Ni+2 is preferably provided at a concentration of about 20mM. In embodiments, Mg"*2 and Mn+2 are provided in combination at a concentration of lOmM each, 15mM each/ 20mM each, 25niM each or 30mM each (i.e., 10-30 mM each).<br>
The amount of cation added to the medium in which the multiplex forms depends on a number of factors, including the nature of the cation, the concentration of probe, the concentration of target, the presence of additional cations and the base content of the probe and target. The preferred cation concentrations and mixtures can routinely be discovered experimentally.<br>
Although not required, other promoters include, e.g., single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, viologen and additional intercalating substances such as actinomycin D, psoralen, and angelicin. Such facilitating<br>
reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures.<br>
The invention also enables a method in which hybridization of the second strand to the fourth strand inactivates an activity associated with at least one of the third strand and the fourth strand. Thus, at least one of the first strand and the second strand further comprises a pharmaceutical agent, wherein hybridization of the second strand to the fourth strand places the pharmaceutical agent an effective distance from a target on the third strand, the fourth strand or on another molecule associated with at least one of the third strand and the fourth strand. The pharmaceutical agent is preferably a member selected from the group consisting of nucleic acids designed to bind promoter sequences of clinically relevant genes, nucleic acids designed to bind clinically relevant genes, or nucleic acids designed to bind origin of replication sites of pathogens.<br>
In preferred embodiments, the invention provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between a single-stranded or double-stranded target and a double-stranded probe, wherein the target comprises a nucleic acid sequence or a nucleic acid analogue sequence and the probe comprises a nucleic acid sequence or a nucleic acid analogue sequence.<br>
The inventive assay can be used to, e.g., identify accessible regions in folded nucleotide sequences, to determine the number of mismatched base pairs in a hybridization complex, and to map genomes.<br>
The invention not only detects the presence of specific probe-target binding, but also provides qualitative and quantitative information regarding the<br>
nature of interaction between a probe and target. Thus, the invention enables the practitioner to distinguish among a perfect match, a one base pair mismatch, a two base pair mismatch, a three base pair mismatch, a one base pair deletion, a two base pair deletion and a three base pair deletion arising between a sequence in the double-stranded probe and in a sequence in the double-stranded target.<br>
Embodiments of the invention comprise calibrating the measured signal (e.g., fluorescence, chemiluminescence, I electrochemiluminescence or electrical properties) for a \ first probe-target mixture against, the same type of signal<br>
i<br>
^exhibited by other probes combined with the same target, jwherein each of the other probes differs from the first probe by at least one base.<br>
A calibration curve can be generated, wherein the<br>
magnitude of  the measured signal  (e.g.,  fluorescent<br>
intensity) is a function of the binding affinity between<br>
the target and probe. As the binding affinity between the<br>
,target and a plurality of different probes varies with the<br>
(number of mismatched bases, the nature of the mismatch(es)<br>
i (A-G vs. A-C vs. T-G vs. T-C, etc.), the location of the<br>
mismatch (es) within the quadruplex, etc., the assay of the<br>
invention can be used to sequence the target.<br>
In embodiments, the signal measured can be the fluorescent intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the probe and target can be directly or inversely correlated with the intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Under selected conditions, the fluorescent intensity generated by intercalating agents can be directly correlated with probe-target binding affinity, whereas the intensity of preferred embodiments employing a<br>
non-intercalating fluorophore covalently bound to the probe can be inversely correlated with probe-target binding affinity. The fluorescent intensity decreases for non-intercalating fluorophores as the extent of matching Between the probe and target increases, preferably over a rangeinclusive of 0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions. '<br>
The invention enables quantifying the binding affinity between probe and target. Such information can_be valuable for a variety of uses, including designing antisense drugs with optimized binding characteristics.<br>
V... ,  _ 	   -    - •  "	  "  '""  "<br>
The assay of the invention is preferably homogeneous. The  assay  can be  conducted  without  separating  the probe-target complex from the free probe and free target prior to detecting the magnitude of the measured signal. The assay does not require a gel separation step, thereby allowing  a  great  increase  in  testing  throughput. Quantitative   analyses   are   simple  and  accurate. Consequently the binding assay saves a lot of time and expense, and can be easily automated.  Furthermore, it enables binding variables such as buffer,  pH,  ionic concentration,  temperature,  incubation time,  relative concentrations of probe and target sequences, intercalator concentration, length of target sequences, length of probe sequences,  and  possible  cofactor  (i.e.,  promoter) requirements to be rapidly determined.<br>
The assay can be conducted in, e.g., a solution within a well or microchannel, on an impermeable surface or on a biochip. In certain embodiments, the third and fourth strands are provided in the hybridization medium before the first and second strands, and the first and second strands<br>
are provided in dehydrated form prior to rehydration by contact with the hybridization medium.<br>
Moreover, the inventive assay is preferably conducted without providing a signal quenching agent on the target or on the probe.<br>
Although the inventors have previously disclosed the advantages of fluorescent intensity assays for hybridization (see, e.g., U.S. Patent Application No. 09/224,505, filed December 31, 1998), certain embodiments of the inventive assay specifically detect quadruplexes of the probe and the double-stranded target, thus obviating the need to denature the target. It is surprising that the inventors have been able to specifically assay quadruplexes formed between double-stranded probes and double-stranded targets, wherein the interaction between the probes and targets is based on Watson-Crick base pairing (at least in the sense that A binds to T (or U, in the case of RNA) and G binds to C) , rather than the very limited Hoogsteen model of quadruplex hybridization of, e.g., Pitner et al., supra.<br>
Suitable probes for use in the inventive assay include, e.g., dsDNA, dsRNA, DMA:RNA hybrids, dsPNA, PNArDNA hybrids and other double-stranded nucleic acid analogues having uncharged or partially-charged backbones. Probe sequences having any length from 8 to 20 bases are preferred since this is the range within which the smallest unique DNA sequences of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome. In embodiments, probes of 5 to 30 bases are most preferred. However, a plurality of shorter probes can be used to detect a nucleotide sequence having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence. The<br>
length of the probe can be selected to match the length of the target.<br>
The instant invention does not require the use of radioactive probes, which are hazardous, tedious and time-consuming to use, and need to be constantly regenerated. Probes of the invention are preferably safe to use and stable for years. Accordingly, probes can be made or ordered in large quantities and stored.<br>
In embodiments, the probe is labeled with a multi-molecule signaling complex or a redox pair, or with a label that elicits .chemiluminescent or electrochemiluminescent properties.<br>
When a fluorescent intercalator is not present in the hybridization medium, it is preferred that the probe or target (preferably the probe) have a fluorescent label covalently bound thereto. The label is preferably a non-intercalating fluorophore or an intercalating fluorophore. In such embodiments, the fluorophore is preferably bound to the probe at either end. Preferred fluorescent markers include biotin, rhodamine, acridine and fluorescein, and other markers that fluoresce when irradiated with exciting energy.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Fluorophores are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from 5 to 85°C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay. On the other hand, the invention can be conducted at room temperature or below (e.g., at a temperature below 25°C).<br>
The reliability of the invention is independent of guanine and cytosine content in said target. Since G-C base pairs form three hydrogen bonds, while A-T base pairs form only two hydrogen bonds, target and probe sequences with a higher G or C content are more stable, possessing higher melting temperatures. Consequently, base pair mismatches that increase the GC content of the hybridized probe and target region above that present in perfectly matched hybrids may offset the binding weakness associated with a mismatched probe.<br>
The inventive assay is extremely sensitive, thereby obviating the need to conduct PCR amplification of the target. For example, it is possible to assay a test sample having a volume of about 20 microliters, which contains about 10 femtomoles of target and about 10 femtomoles of probe. Embodiments of the invention are sensitive enough to assay targets at a concentration of 5xlO"9 M, preferably at a concentration of not more than 5xlO"10 M. Embodiments of the invention are sensitive enough to employ probes at a concentration of 5xlO"9 M, preferably at a concentration of not more than 5xlO"10 M. It should go without saying that the foregoing values are not intended to suggest that the method cannot detect higher concentrations.<br>
The ratio of probe (e.g., first and second strands) to target (e.g., third and fourth strands) is 30:1 to 1:1, preferably about 10:1.<br>
The invention will be illustrated in more detail with reference to the following Examples, but it should be understood that the present invention is not deemed to be limited thereto.<br>
EXAMPLES<br>
Example 1<br>
Sense and antisense 50-mer ssDNA target sequences, derived from exon 10 of the human cystic fibres is gene {Nature 380, 207 (1996)) and modified such that the percent GC content was changed from 30% to 52%, were synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems) and purified by HPLC.  Equimolar amounts of complementary oligonucleotides were denatured at 95°C for 10 minutes and allowed to anneal gradually as the temperature cooled to 21°C over 1.5 hours. DsDNA oligonucleotides were dissolved in ddH2O at a concentration of 1 pmole//zl.<br>
Sequence for the sense strand (SEQ ID NO:1) of the wild-type dsDNA Target A: 5'-GAG CAC CAT GAG AGA CAC TGT CAT CTC TGG TGT GTC CTA CGA TGA CTC TG-3 ' .<br>
Sequence for the antisense strand (SEQ ID NO:2) of the wild-type dsDNA Target A: 5'-GAG AGT CAT CGT AGG ACA CAC CAG AGA TGA CAG TGT CTG TCA TGG TGC TC-3 ' .<br>
Target B was a 50-mer mutant dsDNA target identical to the wild-type target DNA except for a one base pair mutation (underlined) on the sense and antisense strands, wherein the wild-type bases CAT and ATG were replaced by bases CGT and ACG, respectively.<br>
Sequence for the sense strand (SEQ ID NO:3) of mutant Target B: 5'-GAG CAC CAT GAG AGA CAC TGT CGT CTC TGG TGT GTC CTA CGA TGA CTC TG-31.<br>
Sequence for the antisense strand (SEQ ID NO:4) of mutant Target B: 5' -CAG AGT CAT CGT AGG ACA CAC CAG AGA CGA CAG TGT CTG TCA TGG TGC TC-31.<br>
Target C was a 50-mer mutant dsDNA target identical to the wild-type target DNA except for a one base pair mutation (underlined) on the sense and antisense strands, wherein the wild-type bases CAT and ATG were replaced by bases CTT and AAG, respectively.<br>
Sequence for the sense strand (SEQ ID NO: 5) of mutant Target C: 5' -GAG CAC CAT GAC AGA CAC TGT CTT CTC TGG TGT GTC CTA CGA TGA CTC TG-31.<br>
Sequence for the antisense strand (SEQ ID NO: 6} of mutant Target C: 5'-CAG ACT CAT CGT AGG ACA CAC CAG AGA AGA CAG TGT CTG TCA TGG TGC TC-3 ' .<br>
Target D was a 50-mer mutant dsDNA target identical to the wild-type DNA except for a one base pair mutation (underlined) on the sense and antisense strands, wherein the wild-type bases CTC and GAG were replaced by bases CTT and AAG, respectively.<br>
Sequence for the sense strand (SEQ ID NO:7) of mutant Target D: 5 ' -GAG CAC CAT GAC AGA CAC TGT CAT CTT TGG TGT GTC CTA CGA TGA CTC TG-3' .<br>
Sequence for the antisense strand (SEQ ID NO:8) of mutant Target D: 5'-CAG AGT CAT CGT AGG = ACA CAC CAA AGA TGA CAG TGT CTG TCA TGG TGC TC-31.<br>
Target E was a 50-mer mutant dsDNA target identical to the wild-type DNA except for a one base pair mutation (underlined) on the sense and antisense strands, wherein the wild-type bases CTC and GAG were replaced by bases CCC and GGG, respectively.<br>
Sequence for the sense strand (SEQ ID NO:9) of mutant Target E: 5' -GAG CAC CAT GAC AGA CAC TGT CAT CCC TGG TGT GTC CTA CGA TGA CTC TG-31.<br>
Sequence for the antisense strand (SEQ ID NO: 10) of mutant Target E: 5'-CAG AGT CAT CGT AGG ACA CAC CAG GGA TGA CAG TGT CTG TCA TGG TGC TC-3 ' .<br>
Probe A was a 15-mer dsDNA probe with an attached fluorescein moiety at each 5' position and was designed to be mirror complementary to 15 nucleotide segments of the sense strand and antisense strand near the center of the 50-mer wild-type Target A. The strands of the probe were synthesized on the DNA synthesizer mentioned above and purified by HPLC. Equimolar amounts of the probe strands were denatured at 95°C for 10 min and allowed to anneal as the temperature cooled to 21°C over 1.5 hours. DsDNA probe was dissolved in ddH2O at a concentration of 1 pmole//zl.<br>
Sequence for the sense strand (SEQ ID NO: 11) of Probe A: 5'-Flu-CTG TCA TCT CTG GTG-3 ' .<br>
Sequence for the antisense strand (SEQ ID NO: 12) of Probe A: 5' -Flu-CAC GAG AGA TGA CAG-3' .<br>
Each hybridization reaction mixture (40/il) contained the following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled dsDNA probe, 10 mM Tris-HCl, pH 7.5 and 100 mM KCl. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The maximum fluorescent intensities occurred at a wavelength of 525 nm, the emission wavelength for fluorescein. Fig. 1 shows the intensity of fluorescence plotted as a function of wavelength for each sample analyzed.<br>
In the absence of KCl, no hybridization between the dsDNA targets and the Probe A was detected, resulting in similar fluorescent intensities observed when wild-type dsDNA Target A or mutant dsDNA Target D were mixed with dsDNA Probe A or when dsDNA Probe A was present alone (data not shown).<br>
After a 1 hour incubation at 21 °C in the presence of 100 mM.KCl, dsDNA target:dsDNA-F quadruplexes consisting of perfectly complementary sequences on dsDNA Target A and dsDNA Probe A formed readily, resulting in a 62% decrease in the intensity of fluorescent emission compared to that emitted by dsDNA Probe A alone (labeled dsDNA-F) (Fig. 1) . In contrast, incompletely complementary dsDNA Target D:dsDNA-F Probe A quadruplexes, containing a 1 base pair G-T mismatch, were less stable in these reaction conditions, yielding only an 18% decrease in fluorescent intensity compared to that exhibited by dsDNA Probe A alone.<br>
The presence of monovalent cations, such as K*, at specific concentrations was sufficient to allow quadruplex formation between dsDNA targets and dsDNA probes labeled with fluorescein in the absence of prior denaturation. Quadruplex formation occurred on the basis of Watson-Crick base pair affinities, with a measurable and significantly greater amount of quadruplex formation between fully complementary homologous duplex strands. Moreover, the reaction occurred at room temperature within just 1 hour of incubation at a ratio of probe to target of 10 to 1, using natural dsDNA. The dsDNA targets and dsDNA probe used in this example were homologous, contained 53% GC content, and did not contain homopurine or homopyrimidine stretches on any DNA strand. DNA quadruplexes formed readily despite the inferred conclusion that the formation required the duplex probe to give up its right-handed chirality. The assay of the invention was able to identify perfectly complementary dsDNA sequences and those containing a pair of mismatched bases, using dsDNA probe.<br>
Example 2<br>
The quadruplex DNA assays performed in Example 1 were facilitated by the addition of monovalent cations to the reaction mixtures. The specificity of the assay was further examined utilizing divalent cations to facilitate quadruplex DNA formation with dsDNA targets and dsDNA-F probes possessing 53% GC content.<br>
Each hybridization reaction mixture (40/il) contained the following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled dsDNA probe; 10 mM Tris-HCl, pH 7.5 and 20mM MnCl2 and 20 mM MgCl2. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. Fig. 2 shows the intensity of fluorescence plotted as a function of wavelength for each sample analyzed.<br>
When dsDNA-F Probe A (with a 53% GC content) was incubated with 50-mer wild-type dsDNA Target A or mutant dsDNA Target D in the presence of 20 mM MnCl2 and 20 mM MgCl2/ quadruplexes were formed at room temperature under non-denaturing conditions. While perfectly matched DNA quadruplexes yielded the maximum decrease in fluorescent intensity, a 34% decrease, the less favorable dsDNA:dsDNA-F quadruplexes containing a 1 bp T-G mismatch (dsDNA Target D + dsDNA Probe A) produced a fluorescent intensity that was about the same as that observed with dsDNA Probe A alone (Fig. 2) .<br>
The presence of divalent cations, such as Mn*2 and Mg+2, facilitated quadruplex formation under non-denaturing conditions to allow accurate discrimination betweenhomologous, that is mirror complementary, dsDNA target and dsDNA probe quadruplexes, and quadruplex sequences containing a pair of bases which are mismatched.<br>
Divalent cations facilitated the formation of mirror complementary Watson-Crick quadruplexes despite the inferred conclusion that the formation required the duplex probe to give -up its right-handed chirality.<br>
Example 3<br>
The quadruplex DNA assays performed in Examples 1 and 2 were facilitated by. the addition of either monovalent cations or divalent cations in the reaction mixtures. The next Examples demonstrate the specificity of the quadruplex DNA assays when a DNA intercalator is employed.<br>
Probe B was a 15-mer dsDNA probe identical to probe A, without the attached fluorescein marker, and was prepared similarly.<br>
Each hybridization reaction mixture (40 /il) contained the following: 0.4 pmoles of dsDNA target, 4 pmoles of dsDNA Probe B, 0.5 x TBE and 100 nM of the DNA intercalator YOYO-1 (Molecular Probes, Eugene, OR, USA) . The reaction mixtures were incubated at 21°C for 5 minutes and then assayed. Samples were placed into a quartz cuvette, and irradiated with an argon ion laser beam having a wavelength of 488 nm. The maximum fluorescent intensity occurred at a wavelength of 535 nm, indicative of intercalation of YOYO-1.<br>
The fluorescent intensities observed when no target or probe was present (YOYO-1 only) are shown in Pig. 3. Fig. 3 also shows the fluorescent intensities observed when the reaction mixtures combined dsDNA Probe B with wild-type 50-mer dsDNA Target A, which contained homologous sequences, or with four other dsDNA targets which, but for one mismatched pair of bases, contained sequences which<br>
were homologous to the base sequences in the dsDNA Probe B. Homologous, that is to say mirror complementary, wild-type target dsDNA when present in the reaction mixture with the dsDNA Probe B produced the greatest fluorescent intensity. Mismatched dsDNA targets when incubated with dsDNA Probe B in the  reaction mixture  yielded  lesser  fluorescent intensity values ranging from 20% less for dsDNA Target C to 80% less for dsDNA Target E, compared to that achieved by perfectly matched quadruplexes (Fig. 3) .<br>
It was observed that Watson-Crick quadruplexes, stabilized by YOYO-1 intercalation, formed more readily between dsDNA target and dsDNA probe when that probe contained perfectly homologous, that is to say mirror complementary sequences, than when there was a single pair of bases which were not homologous, that is to say not mirror complementary to sequences in the dsDNA targets. Watson-Crick quadruplexes formed readily despite the inferred conclusion that the quadruplex formation required the duplex probe to give up its right-handed chirality.<br>
Example 4<br>
In this example, 50-mer dsDNA targets were exposed to the 53% GC 15-mer dsDNA Probe C, whose Watson-Crick complementarity exists between bases of the strands of the probe and bases of the strands of the target when the major groove of the duplex probe is placed in the minor groove of the duplex target, herein referred to as nested complementarity. The sequences of bases in the duplex probe are not homologous but inverted in relation to those in the duplex target.<br>
The strands of the probe were synthesized on the DNA synthesizer mentioned above and purified by HPLC. Equimolar amounts of the probe strands were denatured at 95°C for 10 min and allowed to anneal as the temperature<br>
ooled to 21 °C over 1.5 hours.  DsDNA probe was dissolved in ddH2O at a concentration of lpmole//iil.<br>
Sequence for the sense strand (SEQ ID NO: 13) of dsDNA Probe C: 5'-GAG ACT AGA GAG CAC-3'.<br>
Sequence for the antisense strand (SEQ ID NO: 14) of dsDNA Probe C: 5'-GTG GTC TCT ACT GTC-3 ' .<br>
Each hybridization reaction mixture (40 /il) contained the following: 0.4 pmoles of target dsDNA, 4 pinoles of dsDNA Probe C, 0.5 x TBE and 100 nM of the DNA intercalator YOYO-1. The reaction mixtures were incubated at room temperature (21°C) for 5 minutes, placed in a quartz cuvette, and irradiated with an argon ion laser beam having a wavelength of 488 nm. The maximum fluorescent intensity occurred at a wavelength of 535 nm, indicative of intercalation of YOYO-1.<br>
Fig. 4 illustrates that in the absence of prior denaturation, the highest fluorescent intensities were achieved when wild-type 50-mer dsDNA Target A was reacted with 15-mer dsDNA Probe C, which was a perfect match on a nested complementary basis to dsDNA Target A. The fluorescent intensity is indicative of DNA binding taking place, in this case quadruplex formation between the dsDNA target and nested complementary dsDNA probe.<br>
Mutant dsDNA targets which were mismatched with the duplex probe by a single pair of bases when matching was assessed on the inverted homology basis of nested complementarity, formed measurably fewer quadruplex complexes with the dsDNA probe, than did the fully complementary wild-type dsDNA target. The various mismatches, which were assayed on a mirror complementary basis in Example 3 were assayed on a nested complementary basis in this example.<br>
As shown in Fig. 4, the fluorescent intensities produced by the quadruplexes formed with the 1 bp mismatched dsDNA targets plus dsDNA Probe C, ranged from 8% to 16% less than that achieved by perfectly matched quadruplexes (dsDNA Target A + dsDNA Probe C) .<br>
Greater discrimination in fluorescence was observed between perfectly matched and mismatched quadruplexes in Example 3. This suggests that fully complementary or 1 base pair mismatched dsDNA probes prefer binding in a nested complementary orientation rather than a mirror complementary orientation to dsDNA target.<br>
This example shows that Watson-Crick quadruplex binding between nested complementary duplex DNA readily occurs in the presence of YOYO-1. This facility arises in part from the fact that nested complementary quadruplex binding allows the right-handed chirality of each interacting duplex to be maintained.<br>
Example 5<br>
50-mer ssDNA target sequences, derived from exon 10 of the human cystic fibrosis gene (Nature 380. 207 (1996)) and modified such that the percent GC content was changed from 30% to 52%, were synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems) and purified by HPLC.<br>
Sequence for the strand (SEQ ID NO:2) being the 50-mer wild-type ssDNA Target F: 5'- GAG AGT CAT CGT AGG ACA CAC CAG AGA TGA GAG TGT CTG TCA TGG TGC TC-3' .<br>
Target G was a 50-mer mutant ssDNA target identical to wild-type ssDNA Target F except for a one base mutation (underlined).<br>
Sequence for the strand (SEQ ID NO:4) of mutant ssDNA Target G: 5'- CAG AGT CAT CGT AGG ACA CAC CAG AGA CGA CAG TGT CTG TCA TGG TGC TC-3' .<br>
arget H was a 50-mer mutant ssDNA target identical to wild-type ssDNA Target F except for a one base mutation (underlined).<br>
Sequence for the strand (SBQ ID NO:6) of mutant ssDNA Target H: 5'- GAG ACT CAT CGT AGG ACA CAC GAG AGA AGA CAG TGT CTG TCA TGG TGC TC-3' .<br>
Target I was a 50-mer mutant ssDNA target identical to wild-type ssDNA Target F except for a one base mutation (underlined) .<br>
Sequence for the strand (SEQ ID NO:8) of mutant ssDNA Target I: 5'- CAG ACT CAT CGT .AGG ACA CAC CAA AGA TGA CAG TGT CTG TCA TGG TGC TC-3'.<br>
Target J was a 50-mer mutant ssDNA target identical to wild-type ssDNA Target F except for a one base mutation (underlined) .<br>
Sequence for the strand (SEQ ID NO: 10) of mutant ssDNA Target J: 5'-CAG AGT CAT CGT AGG ACA CAC CAG GGA TGA CAG TGT CTG TCA TGG TGC TC-3'.<br>
Target K was a 50-mer mutant ssDNA target identical to wild-type ssDNA Target F except for a one base mutation (underlined) .<br>
Sequence for the strand (SEQ ID NO: 15) of mutant ssDNA Target K: 5'-CAG AGT CAT CGT AGG ACA CAC CAG CGA TGA CAG TGT CTG TCA TGG TGC TC-3'.<br>
Probe B was a 15-mer dsDNA probe identical to Probe A without  the  fluorescein  marker,  and  was  prepared accordingly.   One of the strands of Probe B  (i.e., SEQ ID NO: 11) was complementary to a 15 nucleotide segment near the center of 50-mer wild-type ssDNA Target F.<br>
Each hybridization reaction mixture (40/zl) contained the following: 0.4 pmoles of target ssDNA, 4 pmoles of dsDNA probe, 0.5 x TBE and 100 nM YOYO-1. The reaction mixtures were incubated at room temperature (21°C) for 5<br>
inutes, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. Integration time of all laser irradiation was 80 msec. The maximum fluorescent intensities occurred at a wavelength'of 535 nm, indicative of intercalation of YOYO-1. Fig. 5 shows the intensity of fluorescence plotted as a function of wavelength for each sample analyzed.<br>
In the absence of target ssDNA, some hybridization between the dsDNA probes occurred on a mirror complementary basis (Fig. 5) . Such binding between homologous duplex DNA is disclosed in prior examples. SsDNA targets incubated alone with YOYO-1 resulted in low intensity emission values, which were less than one-half the value of the intensity emission of the dsDNA probes alone incubated with YOYO-1 (data not shown).<br>
After a 5 minute incubation at 21 °C in the presence of lOOmM YOYO-1, ssDNA Target F readily formed binding complexes with perfectly complementary sequences on dsDNA Probe B, resulting in a 79% increase in the intensity of fluorescent emission compared to that emitted by dsDNA Probe B alone (Fig 5) . In contrast, incompletely complementary ssDNA targets containing various one base pair mismatches were less stable in these reaction conditions, yielding only a 19% to 31% increase in fluorescent intensity compared to that exhibited by dsDNA Probe B alone (Fig. 5) .<br>
The  presence  of  YOYO-1,  an  intercalator,  was sufficient to allow binding between dsDNA probes and ssDNA targets in the absence of prior denaturation.  Binding occurred on the basis of Watson-Crick base pair affinities, with a measurable and significantly greater amount of<br>
binding between fully complementary strands. The reaction occurred at room temperature within just 5 minutes of incubation at a ratio of probe to target of 10 to 1, using natural DNA. The ssDNA targets and dsDNA probe used in this example contained 53% GC content and did not contain homopurine or homopyrimidine stretches on any DNA strand. The assay of the invention was able to identify perfectly complementary ssDNA sequences and those containing mismatched bases, using dsDNA probes.<br>
The assay of the invention functioned notwithstanding that binding occurred to a substantial extent between duplex probes which were fully self-complementary, i.e., of homologous strand sequence and which bound to one another in a mirror complementary fashion, that is to say major groove to major groove.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to one skilled in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br><br><br><br><br><br>
We Claim:<br>
1.	A method for providing the multiplex structure comprising:<br>
a first strand containing a first sequence of nucleobases;<br>
a second strand containing a second sequence of nucleobases, wherein said second strand is associated with said first strand by Watson-Crick bonding;<br>
a third strand containing a third sequence of nucleobases; and<br>
a fourth strand containing a fourth sequence of nucleobases, said fourth strand is associated with said second strand and said third strand by Watson-Crick bonding, and at least one nucleobase of said fourth sequence of nucleobases is associated by Watson-Crick bonding to at least one nucleobase of said third sequence of nucleobases and to at least one nucleobase of said second sequence of nucleobases,<br>
wherein the method comprising:<br>
providing a hybridization medium comprising said first strand, said second strand, said third strand, said fourth strand, water, a buffer and at least one promoter, and<br>
incubating said hybridization medium for an incubation time effective to hybridize said second strand to said fourth strand to provide said multiplex structure.<br>
2.	A method as claimed in claim 1 wherein said hybridization medium is<br>
buffered to a pH of 5 to 9.<br>
3.	A method as claimed in claim 1 wherein said at least one promoter is an<br>
intercalating agent.<br>
4.	A method as claimed in claim 3 wherein said at least one promoter is an<br>
intercalating fluorophore, and a fluorescent intensity of a test medium<br>
containing said multiplex structure is directly correlated with a binding<br>
affinity of said second strand for said fourth strand.<br>
5.	A method as claimed in claim 4 wherein said intercalating fluorophore is<br>
a member selected from the group consisting of YOYO-1, TOTO-1,<br>
ethidium bromide, ehtidium homodimer-1, ethidium homodimer-2 and<br>
acridine.<br>
6.	A method as claimed in claim 1 wherein said at least one promoter is<br>
tethered to at least one of said first strand, said second strand, said third<br>
strand and said fourth strand.<br>
7.	A method as claimed in claim 1 wherein at least one promoter is a<br>
monovalent cation.<br>
8.	A method as claimed in claim 1 wherein said at least one promoter is a<br>
cation having a valency greater than one.<br>
9.	A method as claimed in claim 8 wherein said cation is at least one<br>
member selected from the group consisting of alkali metal cations,<br>
alkaline   earth  metal  cations,  transition  metal  cations,   Co(NH3)6+3,<br>
trivalent spermidine and tetravalent spermine.<br><br>
10.	A method as claimed in claim 8 wherein said cation is K+ or Na+<br>
provided at a concentration of 50mM to 125mM.<br>
11.	A method as claimed in claim 1 wherein said third strand and said fourth<br>
strand are provided in said hybridization medium before said first strand<br>
and said second strand, and wherein said first strand and said second<br>
strand are provided in dehydrated from prior to rehydration by contact<br>
with said hybridization medium.<br>
12.	A method as claimed in claim 1 wherein said incubation time is not more<br>
than two hours.<br>
13.	A method as claimed in claim 1 wherein said incubating is conducted at<br>
room temperature.<br>
14.	A method as claimed in claim 1 wherein hybridization of said second<br>
strand to said fourth strand is detected as a change in a fluorescent,<br>
chemilunescent, electrochemiluminescent or electrical signal.<br>
15.	A method as claimed in claim 14 wherein an intensity of said signal is<br>
correlated with a binding affinity between said second strand and said<br>
fourth strand.<br>
16.	A method as claimed in claim 15 wherein at least one of said first strand<br>
and said second strand is covalently labeled with a non-intercalating<br>
fluorophore and said intensity is inversely correlated with said binding<br>
affinity.<br>
17.	A method  as   claimed  in claim  16  wherein  said  non-intercalating<br>
fluorophore is a member selected from the group consisting of biotin,<br>
rhodamine and fluorescein.<br>
18.	A method as claimed in claim 1 wherein a ratio of said first strand and<br>
said second strand to said third stand and said fourth strand is 10:1.<br>
19.	A method as claimed in claim 1 wherein concentrations of each of said<br>
first strand, said second strand, said third strand and said fourth strand<br>
are not more that 5 x 10"10 M.<br>
20.	A method as claimed in claim 1 wherein said at least one promoter is a<br>
minor groove nucleic acid binding molecule, which binds in   a non- j<br>
intercalating manner and binds with an association constant of at least<br>
103 M1.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtREVMTlAtMjAwMy1Bc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00423-DELNP-2003-Assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtREVMTlAtMjAwMy1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00423-DELNP-2003-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtREVMTlAtMjAwMy1Gb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00423-DELNP-2003-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1mb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtREVMTlAtMjAwMy1Gb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">00423-DELNP-2003-Form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtMTk5LnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-199.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtMzA4LnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wY3QtNDAyLnBkZg==" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wZXRpdGlvbi0xMzcucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0MjMtZGVsbnAtMjAwMy1wZXRpdGlvbi0xMzgucGRm" target="_blank" style="word-wrap:break-word;">00423-delnp-2003-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="199575-process-for-preparing-orally-administrable-pharmaceutical-formulation-comprising-ephedrine-hydrochloride.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="199577-an-apparatus-for-contacting-solids-in-the-form-of-flowable-lumps-with-liquids.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>199576</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00423/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Sep-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INGENEUS CORPORATION, A Corporation organized and existing under the laws the Barbados</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHANCERY HOUSE, HIGH STREET, BRIDGETOWN, BARBADOS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DAKSIS JASMINE I.</td>
											<td>36 CARTIER CRESCENT, RICHMOND HILL, ONTARIO L4C 2N2, CANADA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ERIKSON GLEN HENRYK</td>
											<td>P.O. BOX 799, PROVIDENCIALES, TURKS &amp; CAICOS ISLANDS.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB01/01643</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-09-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/664,827</td>
									<td>2000-09-19</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/199576-a-method-for-providing-the-multiplex-structure by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:05:36 GMT -->
</html>
