<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/200710-a-quantitative-detection-method-for-quantitatively-determining-a-content-ratio-of-a-genetic-recombinant-in-a-sample-containing-at-least-one-genetic-recombinant-fine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:27:32 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 200710:A QUANTITATIVE DETECTION METHOD FOR QUANTITATIVELY DETERMINING A CONTENT RATIO OF A GENETIC RECOMBINANT IN A SAMPLE CONTAINING AT LEAST ONE GENETIC RECOMBINANT FINE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A QUANTITATIVE DETECTION METHOD FOR QUANTITATIVELY DETERMINING A CONTENT RATIO OF A GENETIC RECOMBINANT IN A SAMPLE CONTAINING AT LEAST ONE GENETIC RECOMBINANT FINE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A quantitative detection method for quantitatively determining a content ratio of a genetic recombinant in a sample containing at least one genetic recombinant The present invention relate to a method of detecting a genetic recombinant by using the PCR method. A method of quantitatively detecting method is provided whereby the total content ratio of the genetic recombinants and the individual content ratio of the genetic recombinant in a population containing plural genetic recombinant lines can be quantified. The method of the present invention comprises performing PCR for the DNA sequence specific to the recombinant and the endogenous DNA sequence shared by the species corresponding to the recombinant using, as a standard molecule, a molecule containing the DNA sequence specific to the recombinant and the endogenous DNA sequence shared by the species on the single molecule, and determining the content ratio of the number of molecules thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS OF QUANTITATIVE DETECTION OF<br>
GENETIC RECOMBINANTS<br>
AND STANDARD MOLECULES FOR THE METHODS<br>
Technical Field of the Invention<br>
The present invention relates to a method of detecting a genetic recombinant by using PCR (polymerase chain reaction). Particularly, the present invention relates to a molecular biological method of detecting a DNA sequence which is specific to a genetic recombinant using PCR. More specifically, the present invention relates to a method of detecting the content ratio of each genetic recombinant line which may be exist in a population containing various genetic recombinants harboring various recombinant DNA sequences. Furthermore, the present invention relates to a standard molecule which is used in such a molecular biological method of detecting the genetic recombinant.<br>
Background of the Invention<br>
The worldwide practical use of genetic recombinants, which are produced by using genetic recombinant techniques, is in progress. As the genetic recombinants which have been already practically used, genetic recombinant microorganisms such as those represented by interferon producing bacteria and genetic recombinant crops such as those represented by insect-resistant maize have been known. For example, an insect-resistant gene such as the CrylA(b) protein coding region (herein after, crylA(b)) from Bacillus thuringiensis or herbicide-resistant gene such as phosphinotricin acetyl transferase(PAT) protein coding region (hereinafter, pat) has been used in the previously developed genetic recombinant crops. In such cases, these transgenes were introduced as the expression units combined with various DNA sequences such that the transgenes can express in the crops.<br>
The DNA sequences which can be used include 35S promoter from<br><br>
cauliflower mosaic virus (hereinafter, CaMVSSS promoter), the promoter of phdsphoenolpyruvate carboxykinase (PEPC) gene from maize, the promoter of calcium dependent protein kinase (CDPK) from maize, the introns such as the region containing the sixth intron of alcohol dehydrogenase 1S gene from maize {Adhl'S IVS6), the region containing the second intron of alcohol dehydrogenase 1S gene from maize {Adhl-S IVS2), the 9th intron of PEPC {PEPC#9) or the intron region of heat-shock protein 70 (hsp70) from maize, and the terminators such as nopaline synthase terminator (herein after, NOS terminator) from Agrobacterium tumefaciens or 35S terminator from cauliflower mosaic virus. The actually commercially distributed genetic recombinant crops include, for example, the progeny varieties from Bt11 line of Novartis, the progeny varieties from Event176 of Novartis, the progeny varieties from MON810 line of Monsanto, the progeny varieties from GA21 line of Monsanto and the progeny varieties from T25 line of Aventis, for maize. For soybean, they include, for example, progeny varieties from Roundup Ready Soy line. The constructions of DNAs that had been introduced into these varieties are shown in Figure 1.<br>
Generally, a genetic recombinant has been developed to impart an industrially preferable property to an original organism compared with the original organism. The major approach for such purpose is isolating a gene expressing the property from an organism which intrinsically exhibits the property, and introducing the gene into the interested organism such that the gene can be expressed in the organism. Thus, the DNAs from genetic recombinants include such recombinant DNA sequences which have been introduced into the recombinants.<br>
For example, genetic recombinant crops have been developed to impart them preferable properties as agricultural products, such as insect-resistance as well as herbicide-resistance and the like, and which crops are produced by introducing the gene responsible for insect-resistance or herbicide-resistance and the like into the original crops, in the form where the gene can be expressed in the crops. Thus, the DNAs from genetic recombinants include such recombinant DNA<br><br>
sequences that have been introduced into the recombinants.<br>
However, when such crops are actually commercialized, it is likely that the crops are the progeny hybrids of the genetic recombinant crops. Therefore, it should be confirmed that the introduced DNA sequences stably exist in such crops.<br>
Additionally, European Community (EU) has enacted the regulation affecting the labeling of genetic recombinants and processed foods produced from them (Regulation (EC) No. EC/258/97, Council Regulation (EC) No. 1139/98), and the regulation affecting the labeling of genetic recombinants and processed foods produced from them was also issued in Japan, which arose the need in the food industry and its relating filed for the information about the existence and the content of recombinants in crops or in foods.<br>
Thus, there is a need for a technique which allows the determination of the existence of genetic recombinants or the content thereof in foods, feeds and in the source crops thereof, particularly the content ratio thereof in the raw materials. When plural genetic recombinant lines are expected to be contained, a techniques is desired which makes it possible to detect them individually and to define the content ratio of each line, that is, the development of a practical technique having high sensitivity and quantitativity to detect the individual line of genetic recombinants is desired.<br>
Although there are many reports showing that it is advantageous to use the molecular biological techniques using polymerase chain reaction (PCR) (Science Vol. 230, 1350-1354 (1985), Science Vol. 239, 487-491 (1988)) for detecting genetic recombinants, these techniques qualitatively determine whether the genetic recombinants exist or not and they do not generate the information about the content ratio of the genetic recombinants contained in the samples (see, for example, Z. Lebensm Unters Forsch A., Vol.203, 339-344 (1996), Mitt. Gebiete Lebensm. Hyg., Vol.87, 307-367 (1996), Deutsche Lebensmittel-Rundschau, Vol.93, Jahrg., Heft 2, 35-38 (1997), Z. Lebensm Unters Forsch A., Vol.205, 442-445 (1997), Zeitschrift fur Ernahrungwissenscaft Vol.36, 155-160<br><br>
(1997), BGW Hefte 1, 115-117 (1997), Mitt. Gebiete Lebensm. Hyg., Vol.88, 164-175 (1997), Mitt. Gebiete Lebensm. Hyg., Vol.88, 515-524 (1997), Food Additives and Contaminants, Vol.15, No.7, 767-774 (1998), Lebensm. -Wiss. u. -Technol., Vol.31, 664-667 (1998), Z. Lebensm Unters Forsch A., Vol.206, 203-207 (1998), Z. Lebensm Unters Forsch A., Vol.206, 237-242 (1998), Z. Lebensm Unters Forsch A., Vol.207, 264-267 (1998), BioSci. Biotecnol. Biochem. Vol.62, No.7, 1461-1464 (1998), Deutsche Lebensmittel-Rundschau, Vol.95, Jahrg., Heft 2, 44-48 (1999), Deutsche Lebensmittel-Rundschau, Vol.95, Jahrg., Heft 2, 48-51 (1999), Deutsche Lebensmittel-Rundschau, Vol.95, Jahrg., Heft 2, 52-56 (1999), Deutsche Lebensmittel-Rundschau, Vol.95, Jahrg., Heft 7, 275-278 (1999), Journal of AOAC International Vol.82, No.4, 923-928 (1999), GIT Labor-Fachzetschrift, 2/99, 156-160 (1999), Bio Industry Vol.16, No.4, 17-21 (1999), Eur. Food Res. Technol. Vol.209, 77-82 (1999), Analytica Chimica Acta Vol.393, 177-179 (1999), Food Control Vol.10, 339-349 (1999), Journal of Agricultural and Food Chemistry, Vol.47, No.12, 5038-5043 (1999), Journal of Food Hygienic Society of Japan, Vol.41, No.2, 137-143 (2000)).<br>
There are also several reports for known techniques that can provide the information of the content ratio of each genetic recombinant contained in the sample such as the reports for techniques for quantitatively determining the recombinant DNA sequences form soybean and maize.<br>
For example, Deutsche Lebensmittel-Rundschau, Vol.95, Jahrg., Heft 2, 57-59 (1999) and Eur. Food. Res. Technol, Vol.209, 83-87 (1999) reports the quantitative detecting techniques for soybean using competitive PCR. These techniques, however, were limited to only a single line of genetic recombinant soybean. Moreover, the accuracy of quantification was relatively low due to the use of competitive PCR and, the purity and yield of the DNA solution extracted from samples affected the result of quantification because they do not use an internal standard.<br>
The quantitative detection technique for genetic recombinant maize using competitive PCR was also reported in Z. Lebensm Unters Forsch A,<br><br>
VoL207, No.3, 207-213 (1998), but the target of the detection was also limited to only a single line of genetic recombinant maize, which is similar to the above-mentioned case. Furthermore, similarly to the above-described case, the competitive PCR and the lack of internal standard made the reliability of the result of quantification insufficient.<br>
Food Control Vol.10, 353-358 (1999) reports a quantitative detection technique using competitive PCR for genetic recombinant maize and genetic recombinant soybean, but both techniques were limited to only a single line, respectively. Furthermore, the use of competitive PCR and the lack of an internal standard made the reliability of the result of quantification insufficient, as above described.<br>
Quantitative PCR using a fluorescent probe and the like (also referred to as real time PCR or in line PCR etc.) is known as a quantitative analysis which is superior to competitive PCR in the accuracy of quantification. There are several reports regarding the quantitative detection techniques using quantitative PCR for recombinant DNA sequences.<br>
For example. Journal of Agricultural and Food Chemistry, Vol.47, No. 12, 5261-5266 (1999) reports that the quantification of recombinant soybean and maize can be performed by quantitative PCR with fluorescent probes. In this reports, the internal standard procedure was employed, which improved the reliability of the quantification result. The target of detection, however, was a single line for genetic recombinant soybean and also a single line for genetic recombinant maize.<br>
Food Control Vol.10, 385-389 (1999) also reports two techniques, the quantitative detection techniques for genetic recombinant soybean using quantitative PCR with fluorescent probes and the quantitative detection techniques for genetic recombinant soybean using competitive PCR. The internal standard procedures were also used in these reports, which improved the reliability of the result of quantification. In contrast to the above-referred reports, the plasmid DNA containing a DNA sequence for internal standard and the<br><br>
plasmid DNA containing the target DNA sequence to be examined were used as a standard molecule. However, since both plasmids were provided separately, there remained the possibility that the manner of diluting the plasmid DNA and the amount of plasmid DNA added into the reaction system affected the result of quantification. Again the target of detection was limited to a single line of genetic recombinant soybean. Chemie in Labor und Biotechnik. Vol.50, Jahrg., Heft 1, 6-8 (1999) is similar to the above-mentioned reports.<br>
Furthermore, since the genetic recombinants that are commercially available as the standard materials are only two lines for maize and one line for soybean, it is very difficult for the analysts to analyze other lines.<br>
A technique for quantifying the content of genetic recombinants was also reported where the protein expressed from a the recombinant DNA sequence is determined by Enzyme Linked Immunosorbent Assay (ELISA), but this technique was also the technique of quantitative quantification for a particular single line.<br>
Summary of the Invention<br>
As above described, the previously reported quantitative detection techniques for genetic recombinants are those which quantitatively detect only limited lines or those which lack the accuracy in quantification.<br>
On the other hand, it is very difficult to quantitatively examine the content ratio of genetic recombinants in a sample when the techniques for detecting only particular one of such lines, because the crops which are conventionally distributed are in the sate where various varieties are mixed during the distribution process. For example, the genetic recombinant maize lines distributed in Japan are up to 7 lines as of July, 2000. Therefore, the analysis would be insufficient if only a particular single line of genetic recombinant maize in the sample is quantitatively determined. Thus, it is understand that it should be noticed that the technique for detecting only a particular single line as well as the technique lacking the accuracy of quantification is insufficient as a quantitative analysis method and lacks the actual utility.<br><br>
One of the factors limiting the number of the target lines to be detected is the insufficient supply of the assay standards. It is very difficult to obtain a pure standard sample which contains only a particular single line of genetic recombinants and which does not contain any other lines. The currently provided genetic recombinants as standard samples for conventional analysts are only two lines for maize and one line for soybean. Indeed, in the previously mentioned prior reports regarding the quantitative detection techniques, all the targets were limited to those for which the respective standard samples can be available (This is also mentioned in the previously referred reference, for example, in Food Control Vol.10, 385-389 (1999)). Namely, the availability of the standard samples leads to the result that the applicability of the above-mentioned detection techniques is limited.<br>
Additionally, since the type and copy number of the introduced recombinant DNA sequence per genome in each line of the genetic recombinants differ among the lines, if the analysis is carried out assuming that only particular line(s) is/are contained in the sample without considering the effects of such differences, it is likely to obtain the results far from the actual content ratio of the genetic recombinants. Thus, even if all the genetic recombinants lines are provided as the standard samples for the analysis, these techniques could not be considered as actual quantitative detection techniques for genetic recombinants unless any means are figured out to take the diversity of recombinant DNA sequences among lines into account.<br>
As mentioned above, it is desired to develop a sensitive and quantitative practical detection technique for genetic recombinants and the development thereof has been actively attempted, but the development and popularization of molecular biological detection techniques for recombinant DNA sequences and/or genetic recombinants are difficult due to the conventional distribution manners, the availability of the standard samples and the diversity of the recombinant DNA sequences among lines.<br>
Accordingly, the object of the present invention is to provide a molecular<br><br>
biological method of detecting genetic recombinants, wherein the accurate total content ratio of genetic recombinants in a population containing plural lines of genetic recombinants can be quantitatively determined by strictly taking the diversity of the recombinant DNA sequences among the plural genetic recombinant lines into account.<br>
Particularly, the object of the present invention is to provide a molecular biological method of detecting genetic recombinants, wherein the accurate individual content ratio of each of the genetic recombinant in a population containing plural lines of genetic recombinants can be quantitatively determined by strictly taking the diversity of the recombinant DNA sequences among the plural genetic recombinant lines into account.<br>
Another object of the present invention is to provide a recombinant DNA molecule as a standard sample used for the above-mentioned quantitative detection method, which especially has the property of being able to be unlimitedly supplied, wherein the molecule is designed so that the quantification can be performed by strictly taking the diversity of the recombinant DNA sequences among the plural genetic recombinant lines into account.<br>
The inventors of the present invention produced a molecule containing, on a single molecule, five genetic recombinant maize line specific DNA sequences, two DNA sequences which are frequently used for the genetic recombinants but are not line-specific and additional one DNA sequence of endogenous maize gene, a molecule containing, on a single molecule, one genetic recombinant soybean line specific DNA sequence and one DNA sequence of endogenous soybean gene and a molecule containing, on a single molecule, one genetic recombinant soybean line specific DNA sequence, one DNA sequence of endogenous soybean gene and two DNA sequences which are frequently used for the genetic recombinants but are not line-specific and found that the previous qualitative and quantitative detection methods can be remarkably improved by using these molecules as the standard molecules in PCR, which allow the inventors to establish the present invention.<br><br>
Accordingly, the present invention is a quantitative detection method for quantitatively determining a content ratio of a genetic recombinant in a sample containing at least one genetic recombinant lines, which comprises: (i) performing quantitative PCR for a DNA sequence specific to the genetic recombinant(s) which may exist in a DNA sample derived from genetic recombinants in the sample and quantitative PCR for an endogenous DNA sequence shared by the species corresponding to the genetic recombinant using, as a standard molecule, a molecule containing, on a single molecule, the DNA sequence specific to the genetic recombinant and the endogenous DNA sequence;<br>
(ii) determining the number of the DNA sequence specific to the genetic recombinant(s) in the sample based on the result of the quantitative PCR; (iii) determining the number of the endogenous DNA sequence in the sample based on the result of the quantitative PCR; and<br>
(iv) determining the content ratio of the genetic recombinant(s) according to formula (I): (content ratio of the genetic recombinant(s) in the sample) =<br>
100 X [(the number of molecules of the DNA sequence specific to the genetic<br>
recombinant(s) in the sample) / (the number of molecules of the endogenous DNA<br>
sequence in the sample)]/ (quantification ratio) (%)	(I)<br>
wherein the quantification ratio is any of the values which is pre-calculated according to formula (II):<br>
(quantification ratio) (%) = (the number of molecules of the DNA sequence specific to a genetic recombinant from the individual genetic recombinant line) / (the number of molecules of the endogenous DNA sequence in the genetic recombinant)       (II),<br>
More specifically, the present invention is a quantitative detection method for quantitatively determining an individual content ratio of an individual genetic recombinant line in a sample containing at lest one genetic recombinant<br><br>
line, which comprises:<br>
(i) performing quantitative PCR for a DNA sequence specific to the individual<br>
genetic recombinant line which may exist in a DNA sample derived from genetic<br>
recombinants in the sample and quantitative PCR for an endogenous DNA<br>
sequence shared by the species corresponding to the genetic recombinants<br>
using, as a standard molecule, a molecule containing, on a single molecule, the<br>
DNA sequence specific to the individual genetic recombinant line and the<br>
endogenous DNA sequence;<br>
(ii) determining the number of the DNA sequence specific to the individual genetic<br>
recombinant line in the sample based on the result of said quantitative PCR;<br>
(iii) determining the number of the endogenous DNA sequence in the sample<br>
based on the result of the quantitative PCR; and<br>
(iv) determining content ratio of the genetic recombinant according to formula (III):<br>
(content ratio of the individual genetic recombinant line in the sample) =<br>
100 X [(the number of molecules or the DNA sequence corresponding to each<br>
genetic recombinant in the sample) / (the number of molecules of the endogenous<br>
DNA sequence in the sample)]/ (quantification ratio) (%)	(III)<br>
wherein the quantification ratio is calculated according to the above-identified formula (II).<br>
The recombinant DNA molecule of the present invention is the recombinant DNA molecule characterized in that it contains, on a single molecule, a DNA sequence specific to the line(s) of genetic recombinant and at least one endogenous DNA sequence shared by the species corresponding to the genetic recombinants. Particularly, the DNA molecule of the present invention is the recombinant DNA molecule characterized in that it contains, on a single molecule, two  or  more   DNA  sequences  specific to  the  individual   line  of  genetic<br><br><br>
Brief Description of Drawings<br>
Figure 1 shows the construction of the recombinant DNA sequence introduced into each genetic recombinant line and the location of the designed primers for the molecular biological analysis, respectively. Each primer pair for amplifying the DNA sequence specific to each genetic recombinant line was designed so that it can amplify the region extending over the plural DNA sequences. The primer pair for amplifying the DNA sequence which is not line-specific but which is frequently used for genetic recombinants was designed so that it can be reacted with all of the corresponding lines.<br>
Figure 2 shows the result of specificity verification test using the primer pair designed for detecting T25 line progeny variety. The amplification reaction, was not observed for the DNA extracted from the samples other than those from T25 line progeny varieties and the amplified product was observed only with the DNA extracted from the progeny varieties from T25 line. The molecular weight of the amplified product was consistent with the designed one.<br>
Figure 3 shows the result of specificity verification test using the primer pair and the probe designed for detecting NOS terminator. The experiments were performed with 11 DNA templates. As can be seen in Figure 1, NOS terminator is introduced into the progeny varieties from Btl 1 line, the progeny varieties from Event176 line and the progeny varieties from Roundup Ready Soy line. The experimental results showed that the amplified products were observed only with the template DNAs extracted from these three varieties and were not observed when other template DNAs were subjected to the reaction, which confirmed the designed specificity.<br>
Figure 4 shows the scheme of the procedure for combining the PCR products.<br>
Figure 5 shows the nucleotide sequence of the standard molecule for maize. The locations of each amplification target in the combined DNA sequences were shown in the figure. The binding regions of the primers and probe used in the experiment were also shown in the figure.<br><br>
Figure 6 shows the nucleotide sequence of another standard molecule for maize. The locations of the combined DNA sequences for each amplification target were shown in the figure. The binding regions of the primers and the probe used in the experiment were also shown in the figure.<br>
Figure 7 shows the schemes of the plasmids pMul4 and pMul5. For pMul4 the DNA shown in Figure 5 was inserted into the vector. For pMul5 the DNA sequence shown in Figure 6 was inserted into the vector.<br>
Figure 8 shows the nucleotide sequence of the standard molecule for soybean. The locations of each target of amplification in the combined DNA sequences were shown in the figure. The binding regions of the primers and the probe used in the experiment were also shown in the figure.<br>
Figure 9 shows the nucleotide sequence of the standard molecule for soybean. The locations of each target of amplification in the combined DNA sequences were shown in the figure. The binding regions of the primers and probe used in the experiment were also shown in the figure.<br>
Figure 10 shows the scheme of the plasmid pMulSL. The sequence introduced into the vector is the DNA sequence shown in Figure 8.<br>
Figure 11 shows the scheme of the plasmid pMulSL2. The sequence introduced into the vector is the DNA sequence shown in Figure 9.<br>
Figure 12 confirmed the genetic recombinant maize contained in the blind maize sample. The progeny varieties from Btl 1 line (A) and the progeny varieties form GA21 line and MON810 line (B) were detected. SamHI digest of pMu14 was used as a control.<br>
Figure 13 shows the result confirming the quantification ratio by performing PCR using the DNA extracted from a single maize seed and calculating the ratio according to formula (II). The arrows in the figure indicate the predicted values which F1 seed from the progeny varieties form T25 line may exhibit.<br>
Figure 14 show the graph of (cycles - fluorescence intensity) during the quantitative PCR for maize standard recombinant DNA sequence.<br><br>
Figure 15 shows the standard curve obtained by the quantitative PCR for maize standard recombinant DNA sequence.   The standard curve was derived<br>
from figure 14.<br>
Figure 16 shows the result confirming the quantification ratio by conducting PCR using the DNA extracted from a single soybean seed and calculating the quantification ratio according to formula (II).<br>
Preferred Embodiments of the Invention<br>
The present invention will be illustrated in the following description by genetic recombinant crops as the example of genetic recombinants, but the application of the present invention are not limited to crops and the present invention does not exclude other genetic recombinants including genetic recombinant animals, plants, microorganisms and the like. In one embodiments of the present invention, recombinant DNA sequences are detected by using PCR method with nucleic acids from test plants to specifically detect various genetic recombinants. Accordingly, it is required to design the primer pair to detect the DNA sequence which is specific to the genetic recombinant line, the primer pair to detect a DNA sequence which is not line-specific but is frequently used for genetic recombinants and the primer pair to detect a DNA sequence which is specific to the crops which may be either a genetic recombinant or a non-recombinant.<br>
The test plant samples may be any samples from which the nucleic acids such as genome DNA can be extracted, including raw seeds, dry seeds, processed materials such as corn grits and soybean flour. Such samples can be used after grinding, if necessary.<br>
According to the present invention, it is possible to determine the content ratio of the genetic recombinants, especially the content ratio of the individual genetic recombinant, in the sample which may possibly contains one or two or more genetic recombinant lines, based on the number of molecules of the recombinant DNA sequences, the number of molecules of the endogenous gene and the quantification ratio calculated for the individual genetic recombinant in the<br><br>
sample.<br>
According   to  the   present   invention,   content   ratio   of  a   genetic recombinant is calculated by the following procedure:<br>
(i) performing quantitative PCR using a molecule as a standard molecule, which contains, on a single molecule, a DNA sequence specific to the genetic recombinant(s) which may exist in a sample and an endogenous DNA sequence shared by the species corresponding to the genetic recombinant; (ii) determining the number of the DNA sequence specific to the genetic recombinant(s) in the sample based on the result of the quantitative PCR; (iii) determining the number of the endogenous DNA sequence in the sample based on the result of the quantitative PCR; and<br>
(iv) determining the content ratio of the genetic recombinant(s) according to formula (I): (content ratio of the genetic recombinant(s) in the sample) =<br>
100 X [(the number of molecules of the DNA sequence specific to the genetic<br>
recombinant(s) in the sample) / (the number of molecules of the endogenous DNA<br>
sequences in the sample)]/ (quantification ratio) (%)	(I)<br>
wherein the quantification ratio is any of the values which is pre-calculated according to the formula (II):<br>
(quantification ratio) (%) = (the number of molecules of the DNA sequence specific to a genetic recombinant from the individual genetic recombinant line) / (the number of molecules of the endogenous DNA sequence in the genetic recombinant)       (II).<br>
Particularly, when the individual content ratio of each genetic recombinant is determined, formula (III) will be used instead of formula (I):<br>
(content ration of the individual genetic recombinant line in the sample) = 100 X [(the number of molecules of the DNA sequence specific to the individual genetic recombinant in the sample) / (the number of molecules of the<br><br>
endogenous DNA sequence in the sample)]/ (quantification ratio) (%)	(III)<br>
The term "DNA sequence" and "DNA sequence or.the partial region thereof as used herein are used interchangeably. Thus, for example, "DNA sequence specific to a genetic recombinant line" means a full length DNA or at least one partial region thereof, all of which are specific to the genetic recombinant line. This is similar in other context. This is also applied to the meaning of "gene sequence".<br>
The term "genetic recombinant" includes the whole organism into which an exogenous gene has been introduced or a portion of organism into which an exogenous gene has been introduced such as organs, tissue explants, cells (including culture cells), seeds, pollens or embryos from such organisms.<br>
The term "quantitative PCR" generally means a series of reactions which uses PCR for quantifying the template DNA which existed at the beginning of the amplifying reaction. Quantitative PCR includes Internal Standard Method which employs an endogenous sequence as a standard and Competitive Method which employs a molecule competing the amplification reaction. In the present invention, the internal method and PCR were employed where the template DNA which is referred to as a standard molecule and which serves as a standard to determine the number of molecules of each sequence accurately and easily, and the progression of the reaction, namely the extent of amplification, is monitored at any time during the reaction. Thus, "Quantitative PCR" herein used refers to the PCR for quantitatively determining the amount of the template DNA which existed at the beginning of the reaction, wherein the target molecule to be amplified can be monitored for its amplification at any time during the reaction. For the quantification, such PCR and a standard curve is generally combined where the standard curve can associate the number of DNA molecule which existed at the beginning of the reaction with the signal indicating the extent of the amplification of the molecule. The standard curve can be generated by using a standard molecule of which number is known.<br><br>
Sample DNA, primers, probes, conditions for PCR, and the number of tine recombinant DNA molecules, the number of molecules of endogenous gene, and the methods for calculating the quantification ratio, which may be used in the present invention, will be described in details hereinafter.<br>
The nucleic acids from test plants are preferably the genome DNA from the test plants. The methods of extracting the nucleic acids from the test plants are not limited and any methods of extracting nucleic acids from test plant samples can be used as long as the sufficient quality for PCR is obtained. For example, commercially available kit such asQIAGEN Plant Maxi Kit (QIAGEN GmbH) may be used.   Additionally, the methods can be modified, if necessary.<br>
The nucleic acids extracted by such methods are preferably kept in a form which is suitable to be used as a template in PCR, for example, in the form of a solution in a buffer. The purity of the obtained nucleic acids can be estimated by the known methods, for example by determining the absorbance at 230nm, 260nm and 280nm. In such a way, it is preferable for performing PCR method that the ratio of 260nm/230nm is more than 2 and the ratio of 260nm/280nm is between 1.8 and 2.<br>
The amplification by PCR may be performed using primers corresponding to the endogenous gene in order to confirmed that the prepared DNA solution is purified sufficient to be subjected to PCR and is not degraded.<br>
PCR is performed using thus obtained nucleic acids obtained from test plants. The primer used in the PCR is the pair of primers that can amplify the region containing the entire recombinant DNA sequence or a portion thereof. By using these primers, it is possible to specifically detect a recombinant DNA sequence depending on the structure of the recombinant DNA sequence. Such primer pairs may be designed as described below depending on the genetic recombinants to be detected.<br>
The target genetic recombinant to be detected may be any of the aforementioned genetic recombinants and the types thereof are not limited. Any genetic recombinant may be the target of detection as long as the construct of the<br><br>
introduced DNA sequence is determined. Such a genetic recombinant may not be restricted to the generation where the gene introduction was performed but it may be a variety from its progenies.<br>
Firstly, the nucleotide sequence of the recombinant DNA contained in the genetic recombinant to be tested should be obtained. The nucleotide sequence is not necessary the entire sequence of the recombinant DNA and it may be a nucleotide sequence located in the neighborhood of the target region. Many of such nucleotide sequences are available in the known literatures. Even if the nucleotide sequence is not available from the known literatures, the sequence can be obtained by an experiment if the information of the partial nucleotide sequence thereof is available. In designing a line specific primer pair, it would be easy to select the pair with high specificity by selecting the region extending over plural types of DNA sequences (for example, the region extending from CaMV 35S promoter to crylA(b)). Each primer pair may be any primer pair as long as it can specifically amplify the target DNA sequence, but it is preferable that the amplified fragment is between 80bp and 200bp and the GC content of each primer is between 40% to 60%, respectively, and it is more preferable that each primer does not form intramolecular high-order structures and does not form base pairing over 3 or more contiguous nucleotides.<br>
For example, the primers can be designed from the region shown in Figure 1 when the progeny varieties from Bt11 line, the progeny varieties from Event176 line, the progeny varieties from MON810 line, the progeny varieties form GA21 line, the progeny varieties from T25 line or the progeny varieties from Roundup Ready Soy line are intended to be specifically and individually detected.<br>
It is also known that the number of the amplified region which existed at the beginning of the reaction, and which is amplified by the primers, can be quantified by quantitative PCR method. Several quantitative PCR methods have been known, but it is required in many cases to prepare a probe complementary to the region which is encompassed by the primer pair. The probe may be any probe which generates a signal corresponding with the number of the molecules<br><br>
which were produced by the amplification, for example, a substance which is suitable to detect, during PCR, the formation of double strand formation of DNA or the dissociation reaction from double strand to single strand, or the extension reaction of nucleic acids. Generally, fluorescence-labeled nucleic acids are used for probes. Specifically, it is preferable that the probe may specifically hybridize to the template DNA under the condition used for the extension reaction by polymerase during the amplification reaction by PCR and may cause the changes in the fluorescence intensity depending on the extension of the DNA strand, that is, the amplification of the template DNA, and said changes preferably indicate the extent of the amplification, more preferably, the probe degrades depending on the amplification of the template DNA to release a fluorophor, which leads to the increase in fluorescent intensity in the reaction mixture and furthermore the increase in florescent intensity is preferably the index of the extent of the amplification. By using such probes, the progress of amplification during PCR may be easily monitored in real time. Such fluorescence-labeled probes are known to those skilled in the art, and suitable fluorescent probes having such properties may also be synthesized. Additionally, it is preferable that the probe has the Tm value about 10**C higher than the corresponding primer pair and the full length of the probe is preferably between about 18 nucleotides and about 25 nucleotides, and that the probe does not have G at its terminal. In one embodiment of the methods of the present invention, a probe is used where the probe degrades as the amplification progresses, which causes the increase in the fluorescent intensity, and the increase in the fluorescent intensity is the indicator of the amplification.<br>
The qualitative PCR can be performed using the nucleic acids from the test plant sample as the template and using the primer pair designed as described above. Furthermore, the quantitative PCR can be performed using the nucleic acids from the test plant sample as the template and using the primer pair and the<br><br>
probe thus designed. There are not any limitations for the reaction mixture, because the reaction mixture can be easily prepared by those skilled in the art. For example, the reaction mixture can be prepared by using an appropriate amount of template nucleic acids, the primer pair, PCR buffer solution, dNTP, magnesium chloride, DNA polymerase, etc. For example, about 50 ng of DNA extracted from the sample can be used, the reaction can be performed with a final primer concentration of about 0.2 - 0.5[iMin a total volume of 25. \xl The conditions for PCR are also not limited and it can be performed by maintaining 10 minutes at 95X, 40 cycles of 30 seconds at 95X, 30 seconds at 58X, 30 seconds at 72**C, and maintaining 7 minutes at 72°C after the 40 cycles and keeping at 4'*C. Those skilled in the art could easily optimize the condition and thus, the temperature and the time for each step can be optionally changed. Moreover, such a reaction can be performed using a well-knov\/n device.<br>
The number of molecules of the recombinant DNA in the sample can be obtained by performing quantitative PCR. Regarding quantitative PCR, it is known that the value obtained by the determination of the initial amount of the target DNA sequence(s) to be amplified does not always directly indicate the content of the genetic recombinant(s). For example, as is indicated in Figure 1, since the DNA sequences introduced into genetic recombinants highly differs among lines and the copy number per plant genome of each genetic recombinant has diversity, the content ratio can not be simply calculated. For example, the coding region for ctylA(b) is introduced into Event176 line and Bt11 line as well as MON810 line, and the developers report that two or more copies has been introduced into Event176, one copy for Bt11 and at least one copy has been introduced into MON810. Additionally, PCR is likely to be inhibited by contaminants in the reaction system.<br>
According to the present invention, the number of molecules of a recombinant DNA sequence is determined by, for example, preparing a standard curve by performing quantitative PCR using a known amount of an internal<br><br>
standard, performing the similar quantitative PCR for the sample and determining said number of the molecules using the standard curve.<br>
The number of molecules of the endogenous sequence may be determined by the similar procedure. On the other hand, for pure genetic recombinant, the number of molecules of the DNA sequence in the genetic recombinant and the number of molecules of the internal DNA sequence are predetermined by quantitative PCR to define the ratio of these determined values as the "quantification ratio". Since the quantification ratio represents the unique value depending on the number and type of the recombinant DNA sequence which has been introduced into each genetic recombinant, the diversity of recombinant DNA sequences among the lines is strictly taken into account by converting the quantitative value to the content ratio of the genetic recombinant using Formula (1). According to the present invention, it becomes possible to avoid the errors in the determined values due to, for example, the deranging factor as to the reaction system itself such as PCR inhibitor which may be contained in the sample DNA solution and the differences of yield during the DNA . extraction.<br>
The quantification ratio is preferably calculated for each of all of the genetic recombinants which may be possibly contained in the sample, but it is also acceptable that the quantification ratio is obtained only for a part of them. Once the value of quantification ratio is obtained, the content ratio of the genetic recombinant in the original sample can be obtained from the initial amount of the target DNA sequence to be amplified and the initial amount of the internal standard sequence at the beginning of the reaction by using the value of quantification ratio as a coefficient, because the value of the quantification ratio is unique for the target region to be amplified by PCR and the line of the genetic recombinant.<br>
Such an internal standard may be, for example, the specific internal DNA sequence which are share by maize, especially, an internal gene sequence, a<br><br>
pecific internal DNA sequence which are share by soybean, especially, an iternal gene sequence or an artificially constructed standard molecule.<br>
The  standard  molecule  suitable for the  present  invention  is  the<br>
Bcombinant DNA molecule which include, on a single molecule, at least one<br>
ecombinant DNA sequence which is specific to at least one, preferably two or<br>
nore lines of genetic recombinants and at least one internal DNA sequences<br>
\/hich  are shared  by the corresponding  species to the  recombinants. A<br>
ecombinant DNA molecule which contains, on a single molecule, at least one<br>
ecombinant DNA which is not specific to each line of the genetic recombinant but<br>
s frequently use for genetic recombinants and at least one internal DNA<br>
sequences shared by the corresponding species can also be used in the present<br>
nvention.     The  recombinant  DNA  molecule  may further  contain  various<br>
sequences to be used for replication in a suitable host or a marker gene for<br>
selecting  hosts   harboring  the  standard  molecule  and  in  addition  to  the<br>
aforementioned sequences.   Only the minimum region which can be used as the<br>
standard molecule can also be obtained from the recombinant DNA molecule by<br>
using appropriate restriction enzymes or by amplifying the part of the molecule by<br>
PCR.    The recombinant DNA molecule according to the present invention,<br>
however, should have the feature that the number of molecules to be subjected to<br>
PCR can be easily controlled.     For example,  when the entire nucleotide<br>
sequence according to the present invention is known or the molecular weight is<br>
known, the number of molecules to be applied to PCR can be controlled.   The<br>
nucleotide sequence or the molecular weight of the recombinant DNA molecule<br>
can be determined, if necessary.<br>
As the internal DNA sequence, for example, the specific internal gene<br>
sequence shared by maize or a part thereof, or the specific internal gene<br>
sequence shared by soybean or a part thereof can be used, as described above.<br>
As   the   recombinant   DNA   sequence   which   is   shared   by   the   genetic<br>
recombinants, for example, CaMV 35S promoter sequence, Nos terminator<br><br>
sequence, and a part thereof may be used. As the DNA sequence specific to the individual recombinant line, for example, the DNA sequence derived from the individual gene v^hich has been introduced into each line may be used.<br>
Such a standard molecule may be constructed according to the molecular biological techniques which are well known to those skilled in the art. For example, the molecule may be constructed by sequentially cloning the restriction fragments or repeatedly combining the PCR products using tailed primers. Preferably the molecule is constructed as a recombinant DNA molecule being capable of self-replicating in suitable hosts such as microorganisms, animal cells, and plant cells. When the standard molecule is constructed as a plasmid, it is preferably to use it as a linear molecule after cleaving it by a restriction enzyme because supercoil formation may destabilize the PCR reaction system. In such cases, any restriction enzymes can be used for cleaving as long as the enzymes do not cleave the target DNA sequence to be amplified.<br>
Particularly, in one embodiment of the present invention, the standard molecule contains a part of Ie1 gene sequence from soybean as the internal DNA sequence, and contains Roundup Ready Soy line specific DNA sequence as the recombinant DNA to be detected. In another embodiment of the present invention, the standard molecule contains a part of zSSIIb gene sequence from maize as the internal standard and contains the DNA sequences specific to GA21, T25, MON810, Event176 and Bt11 line, that it, n?-epsps-NOS terminator region, paf-35S terminator region, adh1'1S-crylA(b) region, ctylA(b)-PEPC#9 region and crylA(b)-HOS terminator region, respectively. In these embodiments, the standard molecules are the DNA molecules which can self-replicate in Escherichia coll.<br>
Particularly, the quantitative PCR according to the present invention and<br>
the determination of the number of molecules of the interested DNA seouence<br>
■<br>
may be performed, for example, according to the followings.<br><br>
(i) Preparation of standard curve<br>
PCR is performed using different numbers of the standard molecule, the primers for amplifying the DNA sequence specific to the genetic recombinant line or the internal DNA sequence shared by the corresponding species to the recombinant under the presence of the probe which increases fluorescent intensity depending on the progress of the amplification of the internal DNA sequence or of the amplification of the DNA sequence specific to the genetic recombinant. The fluorescence intensity is monitored every pre-determined number of times of cycles for each reaction which uses the defined number of the standard molecules as the template DNA which existed at the beginning of the reaction (Fig. 14(A), (B)). Then the threshold of fluorescence increase (ARn) is determined in the phase where an exponential relationship is observed between fluorescence intensity and the cycle numbers. For example, ARn is set to 10'' in Figure 14. The number of PCR cycles reaching the threshold may be plotted against the number of molecules of the DNA template which existed at the beginning of the reaction by taking the number of PCR cycles as a vertical axis and the number of molecules of the template DNA existing at the beginning of the reaction as a horizontal axis to generate the standard curve (Figure 15).<br>
(ii) PCR for sample DNAs<br>
Regarding with the DNA sequence specific to a line of a genetic recombinant and an internal DNA sequence shared by the corresponding species to the genetic recombinant, which may be contained in the sample, the quantitative PCR is performed for the DNA sequence specific to the genetic recombinant in the sample and for the internal DNA sequence, respectively. The DNA sequence specific to the genetic recombinant(s) may be the sequence which is specific to each of the genetic recombinant lines or it may be commonly shared by two or more genetic recombinant lines. The PCR for the DNA sequence specific to each genetic recombinant and for the internal DNA sequence may be<br><br>
performed in the same reaction or in the separate reaction as long as it is assured that the template DNA molecule should be identical in the both reactions.<br>
(ill) Determination of the number of molecules of the DNA sequence specific to the<br>
genetic recombinant(s) and the number of molecules of the internal  DNA<br>
sequence(s)<br>
PCR may be performed for each individual sequence as described<br>
above v\/ith monitoring the signal as the indicator of the amplification to determine<br>
the number of cycles where the signal reaches to the threshold defined in step (i).<br>
Then the obtained number of cycles is converted to the number of the molecules<br>
which existed at the beginning of the reaction by using the standard curve<br>
generated in step (i).<br>
When the standard curve as defined in (i) has already obtained, the standard curve may be used for the procedures after step (ii).<br>
Once the number of molecules of the recombinant specific DNA sequence in the sample, the number of molecules of the internal DNA sequence and the quantification ratio for each individual recombinant are obtained, the total content ratio of the genetic recombinants or the individual content ratio of each line contained in the sample can be calculated according to the aforementioned formula (I) or formula (III). In formula (I) or formula (III), either of the DNA sequence which is specific to each of the genetic recombinant lines or the DNA sequence which is not line-specific but which is frequently used for genetic recombinants may be selected as the recombinant DNA sequence to be quantified.<br>
When the former is selected, the total content ratio of the genetic recombinants in the sample may be exactly quantified by summing up the content ratio of each line in the sample after repeatedly analyzing all of the genetic recombinant lines, because the individual content ratio of each individual line in the sample can be exactly defined. Such a method may be a suitable method in many cases for a distribution mode where the samples to be analyzed are the<br><br>
mixture of plural lines in the distribution channel.<br>
When the latter is selected, since the approximate content ratio can be simultaneously determined for plural lines, it may be a suitable standard sample for conveniently quantifying the approximate total content ratio of the entire genetic recombinants in the sample. In such cases, any value of the quantification ratios calculated for the genetic recombinant lines can be selected as the quantification ratio in formula (I), but it is preferable to use the minimal quantification ratio among these values. This will make it possible to estimate the possible maximum value as the content ratio of the genetic recombinants.<br>
(Example)<br>
The following examples may illustrate the present invention but these examples are only for explanation and the scope of the present invention should not be limited to these examples.<br>
In the following examples, the following samples, reagents and devices were employed.<br>
(1) Samples<br>
The dry seeds of the following six varieties were used for maize {Zea mays):<br>
Genetic recombinant maize: progeny varieties from BT 11 line, Event176 line, MON810 line, T25 line and GA21 line. Non-recombinant maize: Dairyland 1412<br>
The dry seeds of the following two varieties were used for soybean {Glycine max): Genetic recombinant soybean: progeny varieties from Roundup Ready Soy. Non-recombinant soybean: species Murayutaka<br><br>
The dry seeds of the following variety were used for rice [Oryza sativa): Non-recombinant rice: species Kinuhikari<br>
The dry seeds of the following variety were used for wheat {Triticum aestivum): Non-recombinant wheat: species Haruyutaka<br>
The dry seeds of the following variety were used for barely {Hordeum<br>
vulgare): Non-recombinant barley: species Harrington<br>
(2) Reagents<br>
The following reagents were used for DMA extraction: Sodium lauryl sulfate (SDS) (guaranteed reagent) (Sigma Chemical Co.) QIAGEN DNeasy Plant Maxi Kit   (QIAGEN GmbH) QIAGEN DNeasy Plant Mini Kit   (QIAGEN GmbH)<br>
The following reagents were used for electrophoresis:<br>
Acetic acid (guaranteed reagent) (Wako Pure Chemical Industries, Ltd.)<br>
Tris[hydroxymethyl]aminomethane (Tris) (guaranteed reagent) (Sigma Chemical Co.)<br>
Ethylene-diamine-teraacetic acid (EDTA) (guaranteed reagent) (Sigma Chemical Co.)<br>
Agarose powder "103 TTaKaRaj " (TaKaRa Shuzo Co., Ltd.)<br>
Ethidium bromide (Sigma Chemical Co.)<br>
Ficoll 400 (Sigma Chemical Co.)<br>
Bromphenol blue (Sigma Chemical Co.)<br>
Xylene cyanol (Sigma Chemical Co.)<br>
DNA marker "H/ndlll digest of lambda" (New England Biolabs Inc.)<br>
DNA marker "1 kb ladder" (New England Biolabs Inc.)<br><br>
DNA marker "100 bp ladder" (New England Biolabs Inc.)<br>
The following reagents were used for qualitative PCR: DNA polymerase "AmpliTaq Gold" (PE Biosystems) X10 PCR buffer II (PE Biosystems)<br>
The following reagents were used for producing and purifying plasmids:<br>
DNA polymerase "AmpliTaq Gold" (PE Biosystems)<br>
xlO PCR buffer II (PE Biosystems)<br>
DNA polymerase "KOD" (TOYOBO Co, Ltd.)<br>
XlO PCR buffer II (TOYOBO Co., Ltd.)<br>
TOPO TA Cloning Kit with TOPIOP Cells (Invitrogen Co.)<br>
Yeast Extract (Difco Laboratories)<br>
Tryptone   Peptone (Difco Laboratories)<br>
NaCI    (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.)<br>
Agar powder    (Syoei Kanten Ltd.)<br>
D[-]-a-Aminobenzylpenicillin (Ampicillin) Sodium Salt (Sigma Chemical Co.)<br>
QIA(?EN Plasmid Maxi Kit   (QIAGEN GmbH)<br>
ethanol (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.) 2-propanol (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.)<br>
Tris[hydroxymethyl]aminomethane (Tris)     (guaranteed  reagent)     (Sigma Chemical Co.)<br>
Ethylene-diamine-teraacetic acid (EDTA)    (guaranteed reagent)     (Sigma Chemical Co.)<br>
Restriction Enzyme "HindWV (TaKaRa Shuzo Co., Ltd.)<br>
Restriction Enzyme "eamHI" (TOYOBO Co., Ltd.)<br>
Restriction Enzyme "Smal" (New England Biolabs Inc.)<br>
Restriction Enzyme "Srf\" (New England Biolabs Inc.)<br>
Phenol (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.)<br>
chloroform (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.)<br><br>
isoamyl alcohol (guaranteed reagent)    (Wako Pure Chemical Industries, Ltd.)<br>
The following reagent was use for quantitative PCR: TaqMan Universal PCR Master Mix (PE Biosystems)<br>
(3) Devices<br>
The following devices were used for extracting DNA from samples: Granulator "Multi Beads Shocker MB301" (Yasui Kikai Co.) Granulator "DM-6"    (Yu Chi Machinery Co., Ltd.) Touch-Mixer "Tube mixer" (Yamato Scientific Co., Ltd.) Ultrademineralizer "CPW-200"    (ADVANTEC Toyo Kaisya Ltd.) Incubator "Thermo Minder SD mini"    (TAITEC Co.) Centrifuge "himac CT13" (Hitachi Koki Co., Ltd.) Centrifuge "himac CF15D2" (Hitachi Koki Co., Ltd.) Centrifuge "AllegraTM 6KR" (Beckman Coulter, Inc.) Spectrophotometer "DU7400" (Beckman Coulter, Inc.)<br>
The following devices were used for DNA electrophoresis: Electrophoresis apparatus "Mupid 2" (Advance Co., Ltd.) Imaging analyzer "Molecular Imager' FX" (BioRad Laboratories Inc.)<br>
The following devices were used for qualitative PCR: Thermal Cycler "PTC-200" (MJ Research Inc.) Thermal Cycler "PCR System 9700" (PE Biosystems)<br>
The following devices were used for producing and purifying plasmids: Shaking incubator 'Thermostat Shaking Incubator AT24R" (Thomas Kagaku Co., Ltd.) Thermal Cycler "PTC-200" (MJ Research Inc.) Thermal Cycler "PCR System 9700" (PE Biosystems)<br><br>
Centrifuge "himac CT13" (Hitachi Koki Co., Ltd.) Centrifuge "liimac CF15D2" (Hitachi Koki Co., Ltd.)<br>
The following devices were used for quantitative PCR: Quantitative PCR apparatus "ABI PRISM 7700 Sequence Detector System"<br>
(PE Biosystems) Quantitative PCR apparatus "ABI PRISM 5700 Sequence Detector System"<br>
(PE Biosystems)<br>
(4) Others<br>
The primer synthesis was consigned to Greiner Japan K.K.<br>
The probe synthesis was consigned to PE Biosystems Japan K.K.<br>
The verification of DNA sequences was consigned to Greiner Japan K.K.<br><br>
Example 1      HNA Extraction<br>
The extraction of DNA from maize, soybean, rice, wheat, barley was carried out according to the following procedures. Firstly, the samples were ground to powder by granulator "DM-6" (Yu Chi Machinery Co., Ltd.), then 500-1,000 mg was weighed from the ground products and the DNAs were extracted from the ground products using QIAGEN DNeasy Plant Maxi Kit (QIAGEN GmbH) according to the manufacturer's protocol.<br>
To extract the DNA from a single grain of maize or soybean, the surface of the grain was firstly washed well with 1% SDS before crushing by granulator "Multi Beads Shocker MB301" (Yasui Kikai Co.), and then the whole ground product was used for DNA extraction.<br>
The DNA extraction from the blind samples described in Examples 12, 13 and 14 was carried out according to the following procedure. This will be described for maize as an example, but the similar procedure can be carried out for soybean. Firstly the genetic recombinant maize and non-recombinant maize were washed by 1% SDS solution, respectively, before drying. After that, they are ground separately by granulator "DM-6" (Yu Chi Machinery Co., Ltd.) and 1g from each of the ground products was weighed, the genetic recombinant maize and non-recombinant maize were mixed by using granulator "DM-6" (Yu Chi Machinery Co., Ltd.). 500 -1,000 mg was weighed from the mixed sample and the DNA was extracted form the ground products using QIAGEN DNeasy Plant Maxi Kit (QIAGEN GmbH) according to the manufacturer's protocol. 1 |iL each from all of the extracted DNA solutions was subjected to electrophoresis after mixing with 1 \iL of 10x loading buffer (20% Ficoll 400, 0.1M EDTA, 1.0% SDS, 0.25 bromphenol blue, 0.25% Xylene Cyanol) to confirm that the degradation was not occurred during the extraction process. Namely, the electrophoresis was carried out by using electrophoresis apparatus "Mupid 2" (Advance Co., Ltd.) and 0.8% agarose gel containing 50ng of ethidium bromide (Sigma Chemical Co.) in TAE buffer (0.04M Tris, 0.04M Acetic Acid, 0.001M EDTA) at 100V for 15<br><br>
minutes. The DMAs in the gel were confirmed by imaging analyzer "Molecular Imager*^ FX" (BioRad Laboratories Inc.).<br>
Additionally, 1 |iL each from all of the extracted DNA solutions was subjected to spectrophotometry to determine the concentration and the purity thereof. Namely, the absorbance of the sample at 230nm, 260nm, 280nm which was diluted with 49|aL TE buffer to bring a 50-fold dilution and the concentration and the purity thereof was calculated using 1 A260 Unit = 50|ig.<br>
All the extracted DNA was stored at -20X.<br>
Example 2. Selection of the regions for detection (Designing the primer pairs and probes<br>
The recombinant DNA sequences are determined by using the DNA extracted from each progeny variety of the genetic recombinant lines with the sequencing primers shown in Table 1.<br><br><br><br><br>
The primer pair for amplifying the DNA sequence which is specific to the genetic recombinant line was designed to amplify the region extending over plural DNA sequence contained in the introduced DNA sequence. Namely, the primer pairs were designed for amplifying crylA(b) - Nos terminator and adhl-S - crylA(b) for the progeny varieties from Bt11 lline, crylA(b) - PEPC#9 for the progeny varieties from Event176 line, hsp70 - crylA(b) for the progeny varieties from<br><br>
MON810 line, pat - 35S for the progeny varieties from T25 line, m-epsps - NOS and OPT - m-epsps for the progeny varieties from GA21 line and CPT4 - CP4 -epsps for the progeny varieties from Roundup Ready Soy line.<br>
The primer pair for amplifying the DNA sequence which is not line-specific but frequently used for the genetic recombinants was designed to amplify an internal sequence of CaMV 35S promoter sequence or an internal sequence of NOS terminator sequence.<br>
Additionally, an internal sequence of zSSIIb gene for maize or the internal sequence of Ie1 gene for soybean was selected as the DNA sequence of the specific endogenous gene which the organism possesses. These sequences were obtained by searching over genome database to design the primers.<br>
The probe having the Tm value about 10DC higher than the Tm values of the primers was designed between the primers (Table 2A dnd 2B).<br><br><br><br><br><br><br><br><br><sequence listing free text><br>
SEQ ID NOs: 45,  46. 48. 49: PCR primer<br>
SEQ ID NO: 47: Probe for soybean M<br>
SEQ ID NO: 50: Probe for Roundup Ready Soy<br>
Example 3: Confirmation of Specificity of Primer Pairs (Qualitative PCR)<br>
Confirmation was made whether the primer pairs designed in Example 2 were capable of specifically detecting only their target sequences in qualitative PCR.<br>
As the maize samples, progeny varieties from Bt11 line, Event176 line, MON810 line, GA21 line and T25 line and a non-recombinant maize Dairyland 1412 were used. As the soybean samples, a progeny variety from Roundup Ready Soy line and a non-recombinant soybean Murayutaka were used. In total eight kinds of DMA were extracted from the individual samples in the same manner as in Example 1.<br>
Since there is a possibility of contamination with main crops other than maize and soybean in real assay samples, three kinds of DMA were also extracted from rice (species Kinuhikari), wheat (species Haruyutaka) and barley<br><br>
I.'<br>
(species Harrington), respectively, in the same manner as in Example 1.<br>
The eleven kinds in total of DNA extracted above were used as the PCR templates and distilled water was used as a negative control in the qualitative PCR. The reaction was performed using Thermal Cycler "PTC-200" (MJ Research Inc.). In the experiment, each of the primers was used in the PCR solution at a final concentration of 0.5 |aM. The DNA extracted from the individual samples (templates) was used in an amount of 25 ng per reaction system. As a PCR enzyme, a DNA polymerase "AmpliTaq Gold" (PE Biosystems) was used in an amount of 0.625 Unit per reaction system. As a reaction buffer, x10 PCR Buffer II (PE Biosystems) was used in a volume of 2.5 |iL, and MgClj and dNTPs were used in concentrations of 1.5 mM and 200 \xM, respectively, per reaction system. The reaction system was made up to 20 laL with distilled water.<br>
The reaction conditions employed were as follows: maintaining for 10 minutes at 95X, 40 cycles of 30 seconds at 95X, 30 seconds at 58X and 30 seconds at 72°C, followed by maintaining for 7 minutes at 72°C and keeping at 4X.<br>
After the reaction was completed, 5 laL was sampled from the reaction solution, mixed with 1 i^L of a 10x loading buffer, and then subjected to electrophoresis on 3% agarose gel in TAE buffer at 100V for 15 minutes using an electrophoresis apparatus "Mupid 2" (Advance Co., Ltd.). After the gel was stained with ethidium bromide (Sigma Chemicals Co.) for 15 minutes, the presence of PCR amplification products was confirmed using an image analyzer "Molecular Imager'^ FX" (BioRad Laboratories Inc.).<br>
An example of the results of the experiment is shown in Fig. 2. It was confirmed that the primer pair designed for the detection of progeny variety from T25 line were capable of specifically detecting only the progeny variety from T25 line and showed no cross reaction with other samples.<br><br>
All of the results confirmed in the same manner are summarized in Table 3. As shown in Table 3, it was confirmed that all of the primer pairs were capable of specifically detecting only their target progeny varieties from genetic recombinant lines, respectively.<br><br><br>
Example 4: Confirmation of Specificity of Primer Pairs and Probes (Quantitative<br>
PCR)<br>
Confirmation was made whether the primer pairs and primers designed in Example 2 were capable of specifically detecting only their target sequences in the quantitative PCR.<br>
As in the case of Example 3, as the maize samples, progeny varieties from Bt11 line, Event176 line. MON810 line. GA21 line and T25 line and a non-recombinant maize Dairyland 1412 were used; and, as the soybean samples, a progeny variety from Roundup Ready Soy line and a non-recombinant soybean Murayutaka were used. In total eight kinds of DNA were extracted from the individual samples in the same manner as in Example 1.<br>
Since there is a possibility of contamination with main crops other than maize and soybean in real assay samples, three kinds of DNA were also extracted from rice (species Kinuhikari), wheat (species Haruyutaka) and barley (species Harrington) in the same manner as in Example 1.<br>
The 11 kinds in total of DNA extracted above were used as the PCR templates and distilled water was used as a negative control in the quantitative PCR. The reaction was performed using a quantitative PCR apparatus "ABI PRISM 7700 Sequence Detector System" (PE Biosystems). In the experiment, each of the primers was used in a final concentration of 0.5 ^iM and the primer was used in a final concentration of 0.2 jaM in the reaction solution. The DNA extracted from the individual sample (template) was used in an amount of 50 ng per reaction system, and TaqMan Universal PCR Master Mix (PE Biosystems; hereinafter, simply referred to as "Master Mix") was used in a volume of 12.5 ^iL per reaction system.   The reaction system was made up to 25 ^iL.<br>
The reaction conditions employed were as follows: maintaining the reaction solution for 2 minutes at SC'C and then 10 minutes at 95°C, 40 cycles of    " 30 seconds at 95X and 1 minute at SQ'^C, followed by keeping at 25°C.<br><br>
During the reaction, the fluorescent intensity in each reaction well was measured over time. Wells which show the increase in fluorescent intensity can be identified by analyzing the time course of change in fluorescent intensity in each well after the reaction is completed. The fluorescent intensity increased as a result of the degradation of the probe in association with the PCR amplification. Accordingly, in a well in which the increase in fluorescent intensity is observed, it is deemed that PCR amplification occurs and the probe is degraded.<br>
An example of the results of the experiment is shown in Fig. 3. It was confirmed that the primer pair and probe designed for detection of NOS terminator were capable of specifically detecting only the progeny varieties from Bt11 line, Event176 line and Roundup Ready Soy line and showed no cross reaction with other samples.<br>
All of the results confirmed in the same manner are summarized in Table 4. As shown in Table 4, it was confirmed that all of the primers and probes were capable of specifically detecting only their target progeny varieties from genetic recombinant lines, respectively.<br><br><br><br><br>
Example 5: Production of Standard Molecules (Maize)<br>
Integration of the regions to be detected which were selected in Example 2 was performed according to the procedure schematically shown in Fig. 4.<br>
Namely, PCR was performed using the tailed primers shown in Table 5 and the DNA extracted from the corresponding genetic recombinant lines as templates successively to give PCR products having on their terminus other sequences complementary to the target regions to be detected.<br>
PCR was performed using a reaction system containing 10 ng of DNA from maize as a template, 0.5 ^iM of each tailed primer, 0.156 Unit of DNA polymerase "KOD" (TOYOBO Co., Ltd.), 160 ^M of dNTPs, 1.5 mM of MgCljand 2.5 ^iL of xlO PCR buffer II (TOYOBO Co., Ltd.), which was made up to a total volume of 25 |aL with distilled water.<br><br>
The reaction conditions employed were as follows: maintaining for 1 minute at 98°C and then 35 cycles of 30 seconds at 98**C, 30 seconds at 54'C and 1 minute at 74'C, followed by maintaining for 2 minutes at 74X and keeping<br>
at 4X.<br>
Each of the resulting PCR products, together with one of the PCR product in which a region to be adjacent to the region contained in that PCR product was amplified, was subjected to integration reaction utilizing PCR. Namely, 0.25 \iL of the individual PCR products amplified previously were mixed with 0.156 Unit of DNA polymerase "KOD" (TOYOBO Co., Ltd.), 160 ^iM of dNTPs, 1.5 mM of MgClaand 2.5 laL of xlO PCR buffer 11 (TOYOBO Co., Ltd.), which was made up to a total volume of 24.5 i^L with distilled water. First, the reaction system was subjected to PCR amplification without any primer.<br>
The reaction conditions employed were as follows: maintaining for 1 minute at 98°C and then 8 cycles of 30 seconds at 98X, 30 seconds at 56*'C and 1 minute at 74°C, at which point of time the reaction was terminated.<br>
Next, the individual outmost primers were added to the reaction system in an amount of 0.5 |aM. and the PCR amplification was further performed to give an amplification product having the two regions combined.<br>
The reaction conditions employed in the second reaction were as follows: maintaining for 1 minute at 98°C and then 35 cycles of 30 seconds at 98°C, 30 seconds at 56°C and 1 minute at 74X, followed by maintaining for 2 minutes at 74°C and keeping at 4°C.<br>
By repeating the integration reaction in the similar manner, molecules shown in Figs. 5 and 6 were produced. The molecule shown in Fig. 5 has a zSSIIb (GENBANK Accession No. AF019297) DNA sequence of amplification target integrated between nucleotide 1 to 151, a CaMV 35S promoter DNA sequence of amplification target integrated between nucleotide 152 to 252, a NOS terminator DNA sequence of amplification target integrated between nucleotide<br><br>
'5 to 425, a GA21 line-specific DNA sequence of amplification target integrated 3tween nucleotide 441 to 581, a T25 line-specific DNA sequence of amplification rget integrated between nucleotide 582 to 730, a MON810 line-specific DNA squence of amplification target integrated between nucleotide 731 to 843, a ^ent176 line-specific DNA sequence of amplification target integrated between jcleotide 844 to 951 and a Btll line-specific DNA sequence of amplification irget integrated between nucleotide 952 to 1102. The sequence of this region is lown in SEQ ID NO:57. The nucleotide sequence of the molecule of Fig. 5 is lown in SEQ ID NO:73.<br>
The  molecule shown  in  Fig.  6  has a zSSIIb  DNA sequence  of<br>
mplification target integrated between nucleotide 1  to 151, a CaMV 35S<br>
remoter DNA sequence of amplification target integrated between nucleotide<br>
52 to 252, a NOS terminator DNA sequence of amplification target integrated<br>
letween  nucleotide  275  to 425,   a   GA21   line-specific  DNA  sequence  of<br>
implification target integrated between nucleotide 441 to 573, a T25 line-specific<br>
)NA sequence of amplification target integrated between nucleotide 574 to 722, a<br>
/1ON810 line-specific DNA sequence of amplification target integrated between<br>
nucleotide 723 to 835, a Event176 line-specific DNA sequence of amplification<br>
arget integrated between nucleotide 836 to 943 and a Bt11 line-specific DNA<br>
sequence of amplification target integrated between nucleotide 944 to 1071.<br>
The nucleotide sequence of the molecule of Fig. 6 is shown in SEQ ID NO:74.<br><br><br><br><br><sequence listing free text><br>
SEQ ID NOs: 51-72: PCR primers;<br>
SEQ ID NOs: 73-74: Target regions for amplification of standard molecules to be<br>
used for quantitative detection of maize genetic recombinants; and<br>
SEQ ID NO: 75: A target region for amplification of a standard molecule to be<br>
used for quantitative detection of soybean genetic recombinants.<br>
Each of the integrated molecules prepared above was re-amplified using DNA polymerase "AmpliTaq Gold" (PE Biosystems), and ligated into a plasmid vector using TOPO TA Cloning Kit with TOP 10F' Cells (Invitrogen Co.). An E. coli host vector system was used, so that the molecules could be supplied readily and infinitely.<br>
Namely, 1 laL of each of the integrated molecules prepared above (template) was subjected to PCR amplification under the same conditions as in Example 3. After the reaction, 1 laL of the reaction solution was mixed with 1 ^iL of plasmid vector pCR2.1 TOPO and 1 ^L of salt buffer, and the mixture was allowed to stand at room temperature for 5 minutes. Two [iL of the reaction solution was mixed with E coli strain TOP10F' Cells included in the kit, allowed to stand on ice for 5 minutes, and then subjected to heat shock treatment at 42^ for 30 seconds to perform transformation.<br>
One hundred |iL of the solution containing the transformants was plated<br><br>
on a LB (ampicillin) plate [composition per L: 10 g of tryptone peptone (Difco Laboratories), 5 g of yeast extract (Difco Laboratories), 5 g of NaCI (Wako Pure Chemical Industries, Ltd.), 15 g of agar powder (Syoei Kanten Ltd.) and 50 mg of sodium D[-]-a-aminoben2ylpenicillin (Ampicillin) (Sigma Chemical Co.)], and then allowed to stand at SZ^'C overnight to give transformants.<br>
Each colony of the resulting transformants was subjected to colony direct PCR to select correct transformants. Namely, Ml 3 forward primer and M13 reverse primer (0.5 \xM each) were mixed with 0.625 Unit of DNA polymerase "AmpliTaq Gold" (RE Biosystems) and 2.5 |aL of xlO PCR buffer II (PE Biosystems) as a reaction buffer. Each reaction system was also added with MgCIs and dNTPs in concentrations of 1.5 mM and 200 jxM, respectively. The reaction system was made up to a total volume of 25 ^L with distilled water. The colony was picked up with a toothpick and then suspended in the reaction system.<br>
The reaction conditions employed were as follows: maintaining for 5 minutes at 95X, 35 cycles of 30 seconds at 95X, 30 seconds at 50X and 90 seconds at 72X, followed by maintaining for 90 seconds at 72'*C and keeping at 4°C..<br>
The resulting PCR amplification products were subjected to agarose gel electrophoresis. Colonies containing amplification products that had be found to conform to the intended design were cultured in 40 mL of LB (ampicillin) liquid medium [composition per L: 10 g of tryptone peptone (Difco Laboratories), 5 g of yeast extract (Difco Laboratories), 5 g of NaCl (Wako Pure Chemical fndustries, Ltd.) and 50 mg of sodium D[-]-a-aminobenzylpenicillin (Ampicillin) (Sigma Chemical Co.)] at 37°C overnight.<br>
Plasmids were extracted from a large-scale culture of E. coli transformants using QIAGEN Plasmid Maxi Kit (QIAGEN GmbH). The extraction of the plasmids was performed following the protocol attached to the kit.<br><br>
The resulting plasmids were confirmed the correct nucleotide sequence and used as standard molecules (pMul4 and pMulS: Fig. 7). Both of E. coli-ASN-pMul4 which carries plasmid pMul4 and E. coli-ASN-pMul5 which carries plasmid pMu15 have been deposited at Research Institute of Biotechnological &amp; Industrial Science, Institute of Industrial Science and Technology, Ministry of International Trade and Industry, at 1-3, Higashi 1-chome, Tsukuba-shi, Ibaragi-ken, 305-8566 Japan (currently named "Independent Administrative Institution, National Institute of Advanced Industrial Science and Technology, International Patent Organism Depository", at 1 -1 -1 Higashi, Tsukuba-shi, Ibaragi-ken, Tsukuba Central 6, 305-8566 Japan) under Accession Nos. PERM BP-7319 and PERM BP-7320 on October 12, 2000.<br>
When these molecules were used in quantitative PCR, they were used in the form of a linearized molecule prepared by digesting with restriction enzyme SamHI. In the examples below, a BamHl digest of pMul4 (hereinafter, pMul4 BamHl digest) was used as the standard molecule.<br>
Example 6: Production of Standard Molecule (Soybean)<br>
The integration of the target regions to be detected which had been selected in Example 2 was performed according to the procedure schematically shown in Pig. 4, as in the case of Example 5.<br>
Namely, PCR was performed using the tailed primers shown in Table 5 and the DNA extracted from the corresponding genetic recombinant line (a template) to give a PCR product having on the terminus other sequence complementary to the region to be detected. The resulting PCR product was, together with the PCR product in which the region to be adjacent to the region contained in that PCR product was amplified, was subjected to integration reaction utilizing PCR.<br>
The detailed conditions employed in the experiment above were the<br><br>
same as those employed in Example 5.<br>
As a result of the integration reaction, molecules shown in Figs. 8 and 9 were obtained. The molecule shown in Fig. 8 has a Roundup Ready Soy line-specific DNA sequence of amplification target integrated between nucleotide 1 and 121 and a Ie1 (GENBANK Accession No. K00821 M30884) DNA sequence of amplification target integrated between nucleotide 122 to 239. The nucleotide sequence of the molecule of Fig. 8 is shown in SEQ ID NO:75.<br>
The molecule shown in Fig. 9 has a Roundup Ready Soy line-specific DNA sequence of amplification target integrated between nucleotide 1 and 121, a Ie1 DNA sequence to be amplified integrated between nucleotide 122 to 239, a NOS terminator DNA sequence of amplification target integrated between nucleotide 240 to 419 and a CaMV 35S promoter DNA sequence of amplification target integrated between nucleotide 428 to 528. The nucleotide sequence of the molecule of Fig. 9 is shown in SEQ ID NO:75. <sequence listing free text><br>
SEQ ID NOs: 75 and 76: Target regions for amplification of standard molecules to be used for quantitative detection of soybean genetic recombinants.<br>
Each of the integrated molecules prepared above was ligated into a plasmid vector using TOPO TA Cloning Kit with TOP 10F' Cells (Invitrogen Co.), and an E. coll host vector system was used so that the molecule could be supplied readily and infinitely, in the same manner as in Example 5. The resulting plasmids were confirmed the correct nucleotide sequence and used as standard molecules (pMulSL and pMulSL2: Figs. 10 and 11). Both of E. coli-ASN-pMulSL which carries plasmid pMulSL and E. coli-ASN-pMulSL2 which carries plasmid pMulSL2 have been deposited at Research Institute of Biotechnological &amp; Industrial Science, Institute of Industrial Science and Technology, Ministry of International Trade and Industry, at 1-3, Higashi 1-chome, Tsukuba-shi,   Ibaragi-ken,   305-8566   Japan   (currently   called   "Independent<br><br>
Administrative Institution, National Institute of Advanced Industrial Science and Technology, International Patent Organism Depository", 1-1-1 Higashi, Tsukuba-shi, Ibaragi-ken, Tsukuba Central 6, 305-8566 Japan) under Accession Nos. PERM BP-7321 and PERM BP-7322 on October 12, 2000.<br>
When these molecules v^ere used in quantitative PCR, they were used in the form of a linearized molecule prepared by digesting v^ith restriction enzyme SamHL In the examples belov^, a SamHI digest of pMulSL (hereinafter, pMulSL BamHl digest) v^as used as the standard molecule.<br>
Example 7: Use as Positive Control for Qualitative Analysis<br>
■ DNA samples extracted from 1 g each of progeny varieties from Btll line, GA21 line and MON810 line were mixed together in equal amounts. A DNA solution was prepared using the DNA mixture in a DNA concentration of 20 ng/|aL. The DNA solution was used as a blind maize sample, and the presence of genetic recombinant maize in the sample was examined by qualitative PCR using the standard molecule as a positive control.<br>
Devices and conditions (e.g., compo'sitions of reaction solutions) employed in the experiment were the same as those employed in Example 3. Qualitative PCR was conducted for all of the primer pairs shown in Table 2. In the experiment, the DNA solution prepared above was used as a PCR template, the standard molecule pMul4 BamHl digest described in Example 5 was used as a positive control, and distilled water was used as a negative control. The standard molecule pMul4 BamH\ digest was used in an amount of 500 molecules per reaction system.<br>
As in the case of Example 3, after the reaction was completed, 5 [xL was sampled from the reaction solution and subjected to electrophoresis on 3% agarose gel and stained with ethidium bromide, and the presence of a PCR amplification product was confirmed using an image analyzer.<br><br>
As a result of the experiment, as shown in Fig. 12, it was confirmed that progeny vaneties from Bt11 line, GA21 line and MON810 line were contained in the sample as genetic recombinant maize. It was also confirmed that the standard molecule pMul4 BamH\ digest could be used suitably as a positive control for qualitative analysis.<br>
In particular, with respect to the progeny varieties from GA21 line and MON810 line, since no standard sample is commercially available at present, positive controls for these lines have been unavailable to ordinary analysts.<br>
Example 8: Determination of Quantification ratio (Preliminary Test - Maize)<br>
Since maize is a heterotic hybrid, a genetically uniform F1 seed population or a seed population equivalent thereto is required for determining a correct quantification ratio. Thus, as the preliminary test, genetic uniformity of seeds was first confirmed by quantitative PCR.<br>
■^ As the samples of the preliminary test, F1 seeds w^re used for the progeny varieties from Bt11 line, Event176 line, MON810 line and GA21 line. For the progeny variety from T25 line, since F1 seeds was not available, F2 seeds were used instead. The details of the experiment are described in the following. DNA was extracted from each of eight seeds of the individual five varieties above, and the numbers of molecules of the target regions to be detected (which were selected in Example 2) in the extracted DNA were determined using the pMul4 BamHl digest described in Example 5 as a standard sample. Namely, for the DNA sequence of maize endogenous gene zSSIIb to be amplified, quantitative PCR was performed using the DNA extracted from each of the five varieties as a template to determine the number of molecule of the region in the sample. For the DNA sequence of CaMV 35S promoter to be amplified, quantitative PCR was performed using the DNA extracted from each of progeny varieties from Bt11 line, Event176ljne, MON810 line and T25 line as a template to<br><br>
determine the number of molecule of the region in the sample. For the DNA sequence of NOS terminator to be amplified, quantitative PCR was performed using the DNA extracted from each of the progeny varieties from Bt11 line and GA21 line as a template to determine the number of molecule of the region in the sample. The measurement values given by the experiment were used to calculate quantification ratio for the individual lines according to the formula (II) (Fig. 13)<br>
The experiments were performed using a quantitative PCR apparatus "ABI PRISM 7700 Sequence Detector System" (PE Biosystems). In the experiment, the final primer concentration of each PCR solution was 0.5 fxM for all of the primers used for quantification of CaMV 35S promoter, NOS terminator and zSSIIb] and the final probe concentration was 0.2 ^M for all of the probes used for quantification of CaMV 35S promoter, NOS terminator and zSSIIb. The DNA extracted from each sample as a template was used in an amount of 50 ng per reaction system, and TaqMan Universal PCR Master Mix (PE Biosystems; simply referred to as "Master Mix" hereinafter) was used in a volume of 12.5 |j.L per reaction system. Each reaction system was made up to 25 ^L The same reaction was performed in tetraplicate and the measurement values were averaged.<br>
The conditions employed in the reaction were as follows: maintaining the reaction solution for 2 minutes at 50X and then for 10 minutes at 95°C, 40 cycles of 30 seconds at 95X and 1 minute at 59'C, followed by keeping at 25X.<br>
For the quantitative determination of CaMV 35S promoter, a reaction tube was charged with each of 12 kinds of samples consisting of: the standard molecule pMul4 BamH\ digest prepared in three concentrations (250 molecules, 1,000 molecules and 50,000 molecules per reaction system) as a standard recombinant DNA sequence; the DNA solutions extracted from the individual<br><br>
seeds (eight samples in total for each variety) as test samples; and a salmon spermary DNA solution as a negative control. A primer pair and a probe for quantification of CaMV 35S promoter and Master Mix were added to each of the test tubes in the final concentrations indicated above to give a reaction solution.<br>
Similarly, for the quantitative determination of NOS terminator, a reaction tube was charged with each of 12 kinds of samples consisting of: the standard molecule pMul4 SamHI digest prepared in three concentrations (250 molecules, 1,000 molecules and 50,000 molecules per reaction system) as a standard recombinant DNA sequence; the DNA solutions extracted from the individual seeds (eight samples in total for each variety) as test samples; and a salmon spermary DNA solution as a negative control. A primer pair and a probe for quantification of NOS terminator and Master Mix were added to each of the test tubes in the final concentrations indicated above to give a reaction solution.<br>
Similarly, for the quantitative determination of endogenous gene zSSIIb, a reaction tube was charged with each of 12 kinds of samples consisting of: the standard molecule pMul4 BamHl digest prepared in three final concentrations (250 molecules, 1,000 molecules and 50,000 molecules per reaction system) as a standard recombinant DNA sequence; the DNA solutions extracted from the individual seeds (eight samples in total for each variety) as test samples; and a salmon spermary DNA solution as a negative control. A primer pair and a probe for quantification of zSSIIb and Master Mix were added to each of the test tubes in the final concentrations indicated above to give a reaction solution.<br>
During the reaction, the degree of degradation of the probe caused by the PCR amplification was determined over time in terms of the increase in fluorescent intensity. Accordingly by generating a graph (cycles - fluorescent intensity) for the three kinds of samples provided as the standards for the quantification (x 4) (see Fig. 14), a correlation function between the number of molecules of a target DNA sequence to be amplified and the time required for the<br><br>
fluorescent intensity to reach a specified value can be obtained (see Fig. 15). Using the correlation function as a standard curve, the numbers of molecule of the DNA sequence of CaMV 35S promoter to be amplified, the DNA sequence of NOS terminator to be amplified and the DNA sequence of zSSIIb to be amplified present in a quantification sample at the beginning of the reaction can be determined (Table 6).<br><br><br><br>
As shown in Fig. 13, for the progeny varieties from Bt11 line, Event176 line, MON810 line and GA21 line, the quantification ratios were almost constant among the DNA samples from the eight seeds, and the seed samples were confirmed to be genetically uniform F1 populations. For the progeny variety from T25 line of which F1 seeds were not available, however, the integrated DNA sequence showed a progeny distribution following Mendel's Laws as expected. Thus, for the progeny variety from T25 line, only the seeds which had been confirmed to have the identical genetic structure to that of the F1 population (marked with arrows in Fig. 13) were selected, and a population of the seeds was used in the subsequent experiments as the seed population equivalent to a genetically uniform F1 seed population.<br>
Example 9: Determination of Quantification ratio (Preliminary Test - Soybean)<br>
The preliminary test was performed on soybean in the same manner as in Example 8. F1 seeds of a progeny variety from Roundup Ready Soy line were used as the samples for the preliminary test.<br>
DNA was extracted from eight F1 seeds of the progeny variety from Roundup Ready Soy line separately in the same manner as in Example 8, and the numbers of molecules of the target regions to be detected (which had been selected in Example 2) in the extracted DNA samples were determined using the<br><br>
pMulSL BamHl digest described in Example 6 as a standard sample.<br>
Namely, quantitative PCR using the extracted DNA as a template was performed for a DNA sequence of soybean endogenous gene \Q1 of amplification target and a DNA sequence specific to Roundup Ready Soy line of amplification target separately, and the numbers of molecules of the individual regions in the samples were determined. The measurement values obtained in the experiment were used to calculate the quantification ratios according to the formula (II) (Fig.<br>
16).<br>
The devices and conditions (e.g., compositions of reaction solutions) employed in the experiment are the same as those in Example 8, except that primers and probes corresponding to the DNA sequence of soybean endogenous gene Le1 to be amplified and the DNA sequence specific to Round Ready Soy line to be amplified were used.<br>
As a result of the experiment, as shown in Fig. 16, for the progeny variety from Roundup Ready Soy line, the calculated quantification ratios was almost constant among the DNA samples from the eight seeds, and the seed samples were confirmed to be a genetically uniform F1 population, which were used in the subsequent experiments.<br>
Example 10: Determination of Quantification ratio (Maize)<br>
Test for determining the quantification ratio was performed using the genetically uniform F1 seed populations and the equivalent seed population obtained in the preliminary tests. For each variety, the numbers of molecules of the target regions to be detected (selected in Example 2) in the extracted DNA were determined using the DNA samples extracted from the genetically uniform populations as templates and the pMul4 BamHl digest described in Example 5 as a standard sample in the same manner as in the preliminary test.<br>
In the test, in addition to the DNA sequences of amplification target<br><br>
quantified in the preliminary test, the variety-specific DNA sequences of amplification target were also examined. For the variety-specific DNA sequences of amplification target, the numbers of molecules of the target regions in the sample were determined for the individual corresponding varieties.<br>
As in the case of Example 9, the devices and conditions (e.g., compositions of reaction solutions) employed in the experiment were the same as those in Example 8, except that different primers and probes were used for different targets to be amplified.<br>
As in the case of the preliminary test, the measurement values were used to calculate the quantification ratios according to the formula (II). Namely, each of the numbers of molecules of the DNA sequence of CaMV 35S promoter of amplification target, the DNA sequence of NOS terminator of amplification target and the variety-specific target DNA sequences was divided by the number of molecules of zSSIIb DNA sequence of amplification target which was respectively determined in the identical samples to give the quantification ratios unique to the individual DNA sequences (Table 7).<br><br>
As the primer to be used for the amplification of the endogenous gene zSSIIb, two sets of primers designed for a shorter sequence of amplification target (SSIIB 2-5VSSIB 2-3' and SSIIb3-57SSIB 3-3') were also used. The quantification ratios determined using these primers as the primers for amplification of the endogenous gene were also determined (Tables 8 and 9).<br><br><br>
Example 11: Determination of Quantification ratio (Soybean)<br>
For soybean, test for determining the quantification ratio was also performed using the genetically uniform seed population obtained in the preliminary test. For each variety, the numbers of molecules of the target DNA sequence to be amplified of soybean endogenous gene Ie1 in the extracted DNA and the DNA sequence specific to Roundup Ready Soy line were determined separately using the DNA extracted from the genetically uniform population as a template and the pMulSL BamH\ digest as a standard sample in the same manner as in the preliminary test.<br>
As in the case of Example 9, the devices and conditions (e.g., compositions of reaction solutions) employed in the experiment were the same as those in Example 8, except that different primers and probes were used for different targets.<br>
The measurement values were used to calculate the quantification ratios according to the formula (II) in the same manner as in the preliminary test. Namely, the number of molecules of the DNA sequence specific to Roundup<br><br>
Ready Soy line was divided by the number of the DNA sequence to be amplified of soybean endogenous gene M determined in the same sample to give the quantification ratios of the DNA sequences specific to Roundup Ready Soy line unique to the line (Table 8).<br><br>
Example 12: Quantification using blind samples<br>
A ground product of each of genetic recombinant line v^as mixed with a ground product of a non-recombinant line to prepare a blind sample. The blind sample was actually determined on the content ratio of the individual genetic recombinant lines to assess the efficacy of the present quantitative analysis method.<br>
Namely, blind samples were prepared using seeds of genetic recombinant lines and a non-recombinant line as described in Example 1, and DNA was extracted from the blind samples. In the test for maize, as the blind samples, in total 12 kinds of mixtures were prepared which contained 0.1%, 1% and 5% (by weight) of each of the seeds of progeny varieties from Bt11 line, Event176 line, MON810 line and T25 line, respectively<br>
Quantitative PCR was performed using the DNA extracted from each blind sample as a template and the standard molecule prepared in Example 5 which was linearized by digesting with restriction enzyme BamHl (pMuW SamHI digest) to determine the numbers of molecules of the DNA sequence to be amplified of the endogenous gene and the DNA partial sequence to be amplified of the recombinant DNA sequence separately in the blind sample. The content ratio of the genetic recombinant in the sample was determined from the results<br><br>
using the quantification ratio determined in Example 10 or 11.<br>
In the quantification reaction for every target, each of the primers and the probe to be used for quantification were used in a PCR reaction solution in final concentrations of 0.5 \xM and 0.2 laM, respectively. The DNA extracted from a sample (a template) and Master Mix were used in amounts of 50 ng and 12.5 laL, respectively, per reaction system. The reaction system was made up to 25 i^L. The same reaction was performed in tetraplicate, and the resulting measurements were averaged.<br>
The reaction conditions employed were as follows: maintaining the reaction solution for 2 minutes at 50'C and then for 10 minutes at 95X, 40 cycles of 30 seconds at 95°C and 1 minute at 59°C, followed by keeping at 25**C.<br>
In the quantitative determination of each of the target DNA sequences to be amplified, a reaction tube was charged with each of 9 kinds of samples consisting of: standard molecule pMul4 BamHl digest prepared in five concentrations (10, 50, 250, 1,000 and 50,000 molecules per reaction system); the DNA solutions extracted from the 0.1%, 1% and 5% blind samples as test samples; and a salmon spermary DNA solution. A primer pair, a probe for quantification and Master Mix were added to the reaction tube at the final concentrations described above to give a reaction solution.<br>
During the reaction, the degree of degradation of the probe caused by the PCR amplification was determined over time in terms of the increase in fluorescent intensity Accordingly by generating a graph (cycles - fluorescent intensity) for the five samples provided as the standard recombinant DNA sequences (x 4), a correlation function between the number of molecules of a target DNA sequence to be amplified and the time required for the intensity of the fluorescence emitted from the reaction system to reach a specified value can be obtained. Using the correlation function as a standard curve, the numbers of molecules of the target DNA sequences to be applied in the samples present at<br><br>
the beginning of the reaction can be determined (Table 9).<br><br>
The measurement values for the numbers of molecules of the DNA sequences can be converted into the content ratio of the individual genetic recombinants according to the formula (!) using the quantification ratios given in Examples 10 and 11.    The results of the conversion are shown in Table 8.    In<br><br>
either case, it was found that blind samples containing progeny varieties from genetic recombinant lines in amount of 0.1%, 1% or 5% can be determined quantitatively determined satisfactorily<br>
Example 13: Quantification of Content Ratio of Genetic Recombinant using Standard Molecule (1)<br>
Blind samples each containing three kinds of maize genetic recombinant lines were prepared. Quantitative PCR targeting CaMV 35S promoter and a specific endogenous gene zSSIIb from maize was performed on the blind samples to determine the content ratios of the maize genetic recombinant lines in the samples. As the standard material, pMul4 SamHI digest described in Example 5 was used.<br>
The blind maize samples used in the experiment were as follows: a sample containing ground products of progeny varieties from Btll line, Event176 line and MON810 line as maize genetic recombinants each in an amount of 1% (by weight) and a ground product of Dairyland 1412 in an amount of 97% (by weight) as a non-recombinant maize (1% mixture); and a sample containing ground products of progeny varieties from Bt11 line, Event176 line and MON810 line as maize genetic recombinants each in an amount of 5% (by weight) and a ground product of Dairyland 1412 in an amount of 85% (by weight) as a non-recombinant maize (5% mixture).<br>
In the experiment, in order to simulate real situations of analysis, it was assumed that analysts were given no information about the content of any of the three kinds of maize genetic recombinant lines in the samples.<br>
Again, in the PCR reaction solutions, the final primer concentration was 0.5 |iM for each of the primers for analysis of CaMV 35S promoter and the primers for analysis of zSSIIb] the final probe concentration was 0.2 iiM for each of the probe for analysis of CaMV 35S promoter and the probe for analysis of zSSIIb.<br><br>
The DNA extracted from the individual samples (template) and Master Mix were added in amounts of 50 ng and 12.5 |iL, respectively, per reaction system. The reaction system was made up to 25 |iL. The same reaction was performed in tetraplicate and the resulting measurements were averaged.<br>
The reaction conditions employed were as follows: maintaining the reaction solution for 2 minutes at 50X and then for 10 minutes at 95X, 40 cycles of 30 seconds at 95°C and 1 minute at 59X, followed by keeping at 25X.<br>
In the quantitative determination of CaMV 35S promoter, a reaction tube was charged with each of 7 kinds of samples consisting of: standard molecule pMul4 BamH\ digest prepared in five concentrations (10, 50, 250, 1,000 and 50,000 molecules per reaction system) as a standard recombinant DNA sequence; a DNA solution extracted from each of the blind maize samples as a test sample; and a salmon sperm DNA solution as a negative control. A primer pair for quantification of CaMV 35S promoter, a probe for quantification of CaMV 35S promoter and Master Mix were added to the reaction tube at the final concentrations described above to give a reaction solution.<br>
Similarly, in the quantitative determination of endogenous gene zSSIIb, a reaction tube was charged with each of 7 kinds of samples consisting of: standard molecule pMul4 SamHI digest prepared in five concentrations (10, 50, 250, 1,000 and 50,000 molecules per reaction system) as standard recombinant DNA sequence; a DNA solution extracted from each of the blind maize samples as a test sample; and a salmon sperm DNA solution as a negative control. A primer pair for quantification of zSSIIb, a probe for quantification of zSSIIb and Master Mix were added to the reaction tube at the final concentrations described above to give a reaction solution.<br>
During the reaction, the degree of degradation of the probe caused by the PCR amplification was determined over time in terms of the increase in fluorescent intensity.   Accordingly, by generating a graph (cycles - fluorescent<br><br>
intensity) for the five samples provided as the standard recombinant DNA sequence (x 4), a correlation function between the number of molecules of a target DNA sequence to be amplified and the time required for the intensity of the fluorescence emitted from the reaction tube to reach a specified value can be obtained. Using the correlation function as a standard curve, the numbers of molecules of the DNA sequences of CaMV 35S promoter to be amplified and the DNA sequence of zSSIIb to be amplified present in the samples at the beginning of the reaction can be determined (Table 10).<br>
The apparatus used for the quantitative determination ABI PRISM 7700 Sequence Detector System (PE Biosystems) is capable of conducting the reactions for 96 samples simultaneously. Hence, the test mentioned above can be made in one pass of quantitative PCR by using the apparatus.<br>
The measurement values of the numbers of molecules of the DNA sequences can be converted into the content ratio of the individual genetic recombinants carrying CaMV 35S promoter according to the formula (I) using the quantification ratios given in Example 10. In the experiment, since no information about the content of any of the genetic recombinant lines in the samples was given, it was necessary to calculate the quantification ratios and expressed them as converted values for all of the three kinds of genetic recombinant lines. However, the content ratios of the genetic recombinant lines in the samples could be determined only in one pass of quantitative PCR advantageously (Table 10).<br><br><br>
*1 : A blind sample containing 1% of Bt11, Event176 and MON810, respectively<br>
(containing 3% genetic recombinant maize in total) *2: A blind sample containing 5% of Bt11, Event176 and MON810, respectively ( containing 15% genetic recombinant maize in total)<br><br>
Example 14: Quantification of Content Ratio of Genetic Recombinant using Standard Molecule (2)<br>
The same blind samples as prepared in Example 13 were subjected to quantitative PCR targeting the DNA sequences specific to the three kinds of genetic recombinant maize lines, respectively, and a specific endogenous gene zSSIIb from maize to determine the content ratio of the individual genetic recombinant maize lines in the samples. The results and the sum of the quantification ratios obtained were used to determine the content ratio of the genetic recombinants in the samples. As in the case of Example 13, pMul4 6a/7?HI digest described in Example 5 was used as the standard material.<br>
In the experiment, in order to simulate real situations of analysis, it was assumed that analysts were given no information about the content of any of the three kinds of maize genetic recombinant lines in the samples.<br>
The compositions of the reaction solutions, conditions for the reaction and the analysis methods were the same as in those employed in Example 13, except that the sequences specific to the progeny varieties from Bt11 line, Event176 line and MON810 line were separately quantified instead of CaMV 35S promoter. Namely, in the experiment, in total four kinds of the target DNA sequence to be amplified including zSSIIh were subjected to quantitative PCR separately. Therefore, in the experiment, ABI PRISM 7700 Sequence Detector System (PE Biosystems) was run three times in total.<br>
The correlation between the number of molecules of the individual target DNA sequences to be amplification present at the beginning of the reaction and the time required for the intensity of the fluorescence emitted from the reaction tubes to reach a specified value was represented as a standard curve. Using the standard curve, the numbers of the individual line-specific target DNA sequences to be amplified in the assay samples at the beginning of the reaction and the<br><br>
number of the target region to be amplified oi zSSIIb could be determined (Table 11).<br>
As in the case of Example 13. the measurement values for the numbers of molecules of the DNA sequences can be converted into the content ratio of the individual genetic recombinant maize lines in the samples according to the formula (I) using the quantification ratio determined in Examples 10. In the experiment, since the quantitative determination v^as made on DNA sequences specific to the genetic recombinant lines, there v\/as no concern that measurement values might vary which was found in Example 13. The method described above enabled to determine the content ratio of the individual genetic recombinants in the samples more accurately, although the quantitative PCR apparatus had to be run plural times for performing the determination (Table 11).<br><br>
*1 : A blind sample containing 1% of Btll, Event176 and MON810, respectively<br>
(containing 3% genetic recombinant maize in total) *2: A blind sample containing 5% of Btl 1. Event176 and MON810, respectively<br>
(containing 15% genetic recombinant maize in total)<br>
According to the method of the present invention, the presence of a<br><br>
Possible genetic recombinant line(s) in a sample in which the presence of at least One genetic recombinant line is suspected can be determined;.the content ratio of :he genetic recombinant line(s) in the sample in which that content ratio is nknown in a simple manner; the content ratio of a genetic recombinant(s) in a population can be determined accurately; and, in particular, the content ratio of a certain genetic recombinant line in a sample in which that content ratio of the line is unknown can be determined accurately. In particular, the method of the present invention utilizes a standard molecule having in the molecule both of regions to be used for the quantitative determination of recombinant DNA sequences and a DNA sequence of an endogenous gene shared by the species corresponding to the recombinant DNA sequences. Since the standard molecule usually contains the regions to be used for the quantitative determination in an integral proportion depending on the pattern of the regions introduced, the quantification ratio represented by formula (II) can be determined with good reproducibility Consequently, the method of the present invention enables to determine the content ratio of the individual specific genetic recombinant lines in a sample accurately.<br>
Since the genetic recombinants in pure form become unnecessary after the determination of quantification ratios any longer, the difficulty of obtaining the standard molecules for analyst can be eliminated.<br>
Moreover, by providing the standard molecules for the detection of genetic recombinants according to the present invention, the practical utility of molecular biological assays for genetic recombinants for which accurate analytical methods with wide applicability are widely available can be improved.<br><br><br><br>
WHAT IS CLAIMED IS:<br>
1. A quantitative detection method for quantitatively determining a content ratio of<br>
a genetic recombinant in a sample containing at least one genetic recombinant<br>
lines, which comprises:<br>
(i) performing quantitative PCR for a DNA sequence specific to the genetic<br>
recombinant(s) which  may  exist  in  a  DNA  sample  derived from  genetic<br>
recombinants in the sample and PCR for an endogenous DNA sequence shared<br>
by the species corresponding to the genetic recombinant using, as a standard<br>
molecule, a molecule containing, on a single molecule, the DNA sequence<br>
specific to the genetic recombinant and the endogenous DNA sequence;<br>
(ii) determining the number of the DNA sequence specific to the genetic<br>
recombinant(s) in the sample based on the result of the quantitative PCR;<br>
(iii) determining the number of the endogenous DNA sequence in the sample<br>
based on the result of the quantitative PCR; and<br>
(iv) determining the content ratio of the genetic recombinant(s) according to<br>
formula (I):      *.<br>
(content ratio of the genetic recombinant(s) in the sample) =<br>
100 X [(the number of molecules of the DNA sequence specific to the genetic<br>
recombinant(s) in the sample) / (the number of molecules of the endogenous DNA<br>
sequence in the sample)]/ (quantification ratio) (%)	(I)<br>
wherein the quantification ratio is any of the values which is pre-calculated according to fornnula (II):<br>
(quantification ratio) (%) = (the number of molecules of the DNA sequence specific to a genetic recombinant from the individual genetic recombinant line) / (the number of molecules of the endogenous DNA sequence in the genetic recombinant)       (II).<br>
2. A quantitative detection method for quantitatively determining an individual content ratio of an individual genetic recombinant line in a sample containing at lest one genetic recombinant line, which comprises:<br><br>
(i) performing quantitative PCR for a DNA sequence specific to the individual genetic recombinant line v\/hich may exist in a DNA sample derived from genetic recombinants in the sample and for an endogenous DNA sequence shared by the species corresponding to the genetic recombinants using, as a standard molecule, a molecule containing, on a single molecule, the DNA sequence specific to the individual genetic recombinant line and the endogenous DNA sequence;<br>
(ii) determining the number of the DNA sequence specific to the individual genetic recombinant line in the sample based on the result of said quantitative PCR; (iii) determining the number of the endogenous DNA sequence in the sample based on the result of the quantitative PCR; and<br>
(iv) determining content ratio of the genetic recombinant according to formula (III): (content ratio of the individual genetic recombinant line in the sample) =<br>
100 X [(the number of molecules or the DNA sequence corresponding to each<br>
genetic recombinant in the sample) / (the number of molecules of the endogenous<br>
DNA sequence in the sample)]/ (quantification ratio) (%)	(111)<br>
wherein the quantification ratio is any of the values which is pre-calculated according to formula (II):<br>
(quantification ratio) (%) = (the number of molecules of the DNA sequence specific to a genetic recombinant from the individual genetic recombinant line) / (the number of molecules of the endogenous DNA sequence in the genetic recombinant)       (II).<br>
3. The method according to claim 1 or 2, wherein the determination of the number of the DNA sequence specific to the genetic recombinant line and the number of the endogenous DNA sequence in the sample comprises (i) Performing quantitative PCR for the DNA sequence specific to the genetic recombinant line which may exist in the sample and quantitative PCR for the endogenous DNA sequence shared by the species corresponding to the genetic recombinant; and<br><br>
(ii) monitoring a signal which is an indicator of the amplification in the PCR,<br>
determining a number of cycles of PCR at which the signal reaches a predefined<br>
threshold and then converting the number to a number of molecules which existed<br>
at the beginning of the reaction using a standard curve,<br>
wherein the standard curve represent a relationship between the number of the<br>
DNA sequence to be amplified and the signal which is the indicator of the<br>
amplification.<br>
4.	The method of claim 3, wherein the signal which is the indicator of the amplification is fluorescence, said fluorescence deriving from a fluorescence-labeled probe and said fluorescence being capable of changing depending of the degradation of said probe depending on the amplification of the DNA sequence by the PCR.<br>
5.	The method of claim 3, wherein the standandard curve is obtained by<br><br>
(a)	performing PCR for a standard molecule using primers for amplifying the DNA sequence specific to the genetic recombinant line or the internal DNA sequence shared by the corresponding species to the recombinant under the presence of a probe which increases fluorescent intensity depending on the progress of the amplification of the internal DNA sequence or of the amplification of the DNA sequence specific to the genetic recombinant;<br>
(b)	monitoring the fluorescence intensity every pre-determined number of times of cycles for each reaction which uses the defined number of the standard molecules as the template DNA which existed at the beginning of the reaction;<br>
(c)	determining a threshold of fluorescence increase (ARn) in the phase where an exponential relationship is observed between fluorescence intensity and the cycle numbers; and<br>
(d)	plotting a number of PCR cycles reaching the threshold against the number of molecules of the DNA template which existed at the beginning of the reaction by taking the number of PCR cycles as a vertical axis and the number of molecules<br><br>
of the template DNA existing at the beginning of the reaction as a horizontal axis<br>
6.	A recombinant DNA molecule characterized in that it contains, on a single molecule, a DNA sequence specific to a line of genetic recombinant (s) and at least one endogenous DNA sequence shared by the species corresponding to the genetic recombinant(s).<br>
7.	A recombinant DNA molecule characterized in that it contains, on a single molecule, two or more DNA sequences specific to the individual line of genetic recombinants respectively, and at least one endogenous DNA sequence shared by two or more non-transformants corresponding to said genetic recombinants.<br>
8.	A recombinant DNA molecule which is capable of self-replicating in a host cell and which comprises the recombinant molecule according to claim 6 or 7.<br>
9.	The recombinant DNA molecule according to claim 8 which is plasmid pMul4 contained in E.coli PERM BP-7319.<br><br>
10.	The recombinant DNA molecule according to claim 8 which is plasmid pMul5 contained in E.coli PERM BP-7320.<br>
11.	The recombinant DNA molecule according to claim 8 which is plasmid pMulSL contained in E.coli PERM BP-7321.<br>
12.	The recombinant DNA molecule according to claim 8 which is plasmid pMulSL2 contained in E.coli PERM BP-7322.<br>
13.	The method of claim 1, wherein the standard molecule is the recombinant DNA molecule according to any one of claim 6-12.<br><br>
14.	An oligonucleotide comprising the sequence of SEQ ID NO: 15 or NO: 16 as the 3'-terminal sequence.<br>
15.	An oligonucleotide comprising the sequence of SEQ ID NO: 18 or NO: 19 as the 3'-terminal sequence.<br>
16.	An oligonucleotide comprising the sequence of SEQ ID NO:21orNO:22 as the 3'-terminal sequence.<br>
17.	An oligonucleotide comprising the sequence of SEQ ID NO:24 or NO:25 as the 3'-terminal sequence.<br>
18.	An oligonucleotide compnsing the sequence of SEQ ID NO:27 or NO:28 as the 3'-terminal sequence,<br>
19.	An oligonucleotide comprising the sequence of SEQ ID NO:30 or N0:31 as the 3'-terminal sequence.<br>
20.	An oligonucleotide comprising the sequence of SEQ ID NO:33 or N0:34 as the 3'-terminal sequence.<br>
21.	An oligonucleotide comprising the sequence of SEQ ID NO:36or NO:37 as the 3'-terminal sequence.<br>
22.	An oligonucleotide comprising the sequence of SEQ ID NO:39 or NO:40 as the 3'-terminal sequence.<br>
23.	An oligonucleotide comprising the sequence of SEQ ID NO:42 or NO:43 as the 3'-terminal sequence.<br><br>
24.	An oligonucleotide comprising the sequence of SEQ ID NO:45 or NO:46 as the 3'"terminal sequence.<br>
25.	An oligonucleotide comprising the sequence of SEQ ID NO:48 or NO:49 as the 3'"terminal sequence.<br>
26.	An oligonucleotide comprising the sequence of SEQ ID NO: 17.<br>
27.	An oligonucleotide comprising the sequence of SEQ ID N0:20.<br>
28.	An oligonucleotide comprising the sequence of SEQ ID NO:23.<br>
29.	An oligonucleotide comprising the sequence of 6EQ ID NO:26.<br>
30.	An oligonucleotide comprising the sequence of SEQ ID NO:29.<br>
31.	An oligonucleotide comprising the sequence of SEQ ID NO:32.<br>
32.	An oligonucleotide comprising the sequence of SEQ ID NO:35.<br>
33.	An oligonucleotide comprising the sequence of SEQ ID NO:38.<br>
34.	An oligonucleotide comprising the sequence of SEQ ID N0:41.<br>
35.	An oligonucleotide comprising the sequence of SEQ ID N0:44.<br>
36.	An oligonucleotide comprising the sequence of SEQ ID NO:47.<br>
37.	An oligonucleotide comprising the sequence of SEQ ID N0:50.<br><br>
38.	An oligonucleotide compnsing the sequence of SEQ ID NO:77 or NO:78 as the 3'-terminal sequence.<br>
39.	An oligonucleotide comprising the sequence of SEQ ID NO:79 or NO:80 as the 3*-terminal sequence.<br><br>
A quantitative detection method substantially as herein described with reference to the accompanying drawings.<br>
An oligonucleotide substantially as herein described with reference to the accompanying drawings.<br><br></sequence></sequence></sequence></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">619-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">619-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtY2xhaW1zIG9yaWdpbmFsLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">619-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZGVzY3JpcHRpb24oY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">619-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZGVzY3JpcHRpb24oY29tcGxldGUpIG9yaWdpbmFsLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">619-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">619-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjE5LWNoZW5wLTIwMDMtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">619-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="200709-scanner-system-and-method-for-transmission-of-scanned-image-over-data-network-to-a-file-server.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="200711-a-process-for-the-preparation-of-intrinsically-conducting-pre-vulcanized-natural-rubber-latex-films-for-electronic-and-optoelectronic-applications.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>200710</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>619/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>8/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td></td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Apr-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NATIONAL FOOD RESEARCH INSTITUTE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2-1-12 KANNONDAI, TSUKUBA-SHI, IBARAKI, JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NE</td>
											<td>NE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP01/09344</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-10-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-326738</td>
									<td>2000-10-26</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/200710-a-quantitative-detection-method-for-quantitatively-determining-a-content-ratio-of-a-genetic-recombinant-in-a-sample-containing-at-least-one-genetic-recombinant-fine by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:27:33 GMT -->
</html>
