<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/200700-a-method-for-detecting-or-quantifying-a-traget-nucleic-acid-in-a-sample by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:29:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 200700:A METHOD FOR DETECTING OR QUANTIFYING A TRAGET NUCLEIC ACID IN A SAMPLE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR DETECTING OR QUANTIFYING A TRAGET NUCLEIC ACID IN A SAMPLE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for detecting or quantifying a target nucleic acid in a sample A method for detecting and/or quantifying a target DNA or RNA present in a sample by a isometric primer extension method is disclosed. The method includes carrying out a primer extension reaction in the absence of a free nucleotide so that the primer extension reaction is stopped where the absent nucleotide would have been inserted. Thus, as the amount of incorporation of a labeled nucleotide on the primer extended product is detected, the amount of the target RNA or DNA in the sample is measured.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention is related to a method for detecting or quantifying a target nucleic acid in a sample.<br>
Conventional methods for detecting and quantifying special sequences of nucleic acids such as DNA and RNA include southern blotting, northern analysis, and RNase protection assays, and Polymerase Chain Reaction (PCR), among other methods. However, if the detection of a specific RNA species in a sample is considered, Northern blotting and RNase protection assay present limitations in efficiency, labor intensiveness, accuracy, high cost, sensitivity, greater RNA sample requirement, specialized equipment, and a large amount of biohazardous and radioisotopic waste material that are generated. In particular, both Northern blotting and RNase protection assay require 2-3 days for completion of the analyses. In addition. Northern blotting requires running a RNA gel, transferring the RNA to a solid support, preparing a probe, and carrying out a hybridization reaction. The sensitivity requirement is a 5 .mu.g sample for adequate sensitivity. Northern blot is based on the principle of hybridization between the target and the probe nucleic acid. Moreover, the cost per reaction is fairly high, as is the amount of biohazardous and radioisotopic waste material that are generated.<br>
Similarly, RNase protection assay requires 2-3 days to obtain the appropriate results. The experimental procedure requires preparing template DNA, preparing RNA probe, carrying out hybridization reaction, enzyme digestion reaction, and running a gel. In order to obtain good results, a 1 ^ig sample of the target RNA is required. The principle behind RNase protection assay is a combination of<br><br>
hybridization and enzyme digestion reactions. As in the Northern blotting method, it is expensive to carry out this reaction. Furthermore, biohazardous and radioisotopic waste products are generated in high amounts. A chart comparing the various factors associated with Northern blotting, RNase protection assay and the multiple primer extension method of the present invention is set forth in Table 1.<br>
U.S. Patent No. 5,846,710 discloses using a primer extension technique to screen for variant DNA molecules. However, this patent does not disclose detecting a target DNA or RNA in a sample.<br>
U.S. Patent No. 5,994,079 discloses forming an RNA/DNA hybrid by annealing a DNA primer to a specific RNA and extending the primer by using reverse transcriptase. The hybrid is detected by an antibody specific for the RNA/DNA hybrid. However, this patent does not disclose detecting a target DNA or RNA in a sample as in the present invention.<br>
It is recognized that there is a need in the art for a nucleic acid detection method that is simple, costs less time, is sensitive, cost effective and has a low adverse environmental impact. The present invention as described hereinbelow meets all of these needs.<br>
SUMMARY OF THE INVENTION<br>
The present invention has met the hereinbefore described need. The present invention is directed to a method for detecting or quantifying a target nucleic acid in a sample comprising:<br>
(a)	preparing a primer or primers specifically matched to a<br>
predetermined position of the target nucleic ac;d;<br>
(b)	annealing the primer or primers from (a) with the target nucleic<br>
acid under high stringency conditions to obtain a primer-<br><br>
nucleic acid duplex at the predetermined position of the target nucleic acid;<br>
(c)	mixing the primer-nucleic acid duplex from (b) with a mixture<br>
comprising:<br>
(1)	one or two or three types of free non-terminator<br>
nucleotides and at least one type of non-terminator nucleotide that is optionally labeled with a detectable marker, and<br>
(2)	with or without a type of terminator nucleotide that is<br>
different from the one or two or three types of non-terminator nucleotides in (1);<br>
(d)	performing the primer extension by enzymatic or chemical<br>
reaction in an appropriate buffer; and either<br>
(e)	detecting or quantifying the amount of labeling signal on the<br>
primer extended nucleotides, or<br>
(f)	detecting or quantifying the amount of extended primers by mass<br>
spectrometry.<br>
In the above method, the primer can be a nucleic acid primer, an oligodeoxyribonucleotide, an oligoribonucleotide, or a copolymer of deoxyribonucleic acid and ribonucleic acid. The nucleic acid of interest can be a deoxyribonucleic acid, a ribonucleic acid, or a copolymer of deoxyribonucleic acid and ribonucleic acid.<br>
In a preferred embodiment, the method may comprise using a mixture comprising a combination of non-terminator and terminator nucleotides as follows:<br>
(a)	dATP, dCTP, dGTP, ddTTP or ddUTP,<br>
(b)	dATP, dCTP, dTTP or dUTP, ddGTP,<br>
(c)	dATP, dGTP, dTTP or dUTP, ddCTP,<br>
(d)	dCTP, dGTP, dTTP, or ^UTP, ddATP,<br><br>
(e)	dATP, dCTP, dGTP,<br>
(f)	dATP, dCTP, dTTP or dUTP,<br>
(g)	dATP, dGTP, dTTP or dUTP, or (h) dCTP, dGTP, dTTP or dUTP.<br>
The method of the invention may use at least one non-terminator nucleotide that is labeled with a detectable marker. The detectable marker may comprise an enzyme or protein moiety, radioactive isotope, a fluorescent moiety, or a chemical group such as biotin. Moreover, the detecting or quantifying method step may be carried out by mass spectrometry.<br>
Some of the enzymes used in the primer extension reaction of the invention include a template-dependent enzyme such as E. coli DNA polymerase I or the "Klenow fragment" thereof, T4 DNA polymerase, T7 DNA polymerase, T. aquaticus DNA polymerase, a retroviral reverse transcriptase, or a combination thereof<br>
These and other objects of the invention will be more fully understood from the following description of the invention, the referenced drawings attached hereto and the claims appended hereto.<br>
A method for detecting or quantifying a target nucleic acid in a sample comprising: (a) preparing at least one primer specifically matched to a predetermined position of the target nucleic acid; (b) annealing the primer or primers from (a) with the target nucleic acid under high stringency conditions to obtain a primer-nucleic acid duplex at the predetermined position of the target nucleic acid; (c) mixing the primer-nucleic acid duplex from (b) with a non-terminator nucleotide mixture, wherein at least one A,T,G, or C non-terminator nucleotides is omitted from the non-terminator<br><br>
nucleotide mixture, and wherein at least one non-terminator nucleotide of the non-terminator nucleotide mixture which is not omitted is labeled with a detectable marker; (d) performing isometric primer extension by enzymatic or chemical reaction in an appropriate buffer to form isometric primer extension products, wherein the primer extension terminates at a target nucleic acid nucleotide complementary to the omitted non-terminator nucleotide of (c); and (e) detecting or quantifying the amount of labeling signal on the isometric primer extension products.<br>
With reference to the accompanying drawings, in which<br>
Figure 1- a schematic of multiple primer extension reaction to detect and quantify a specific DNA sequence.<br>
Figure 2 - a schematic for using the multiple primer extension method to detect and quantify RNA.<br>
The present invention is directed to a method for detecting and quantifying a specific nucleotide sequence by using an isometric primer extension (iPE) method.<br><br>
To summarize the invention, the target DNA or RNA in the sample is hybridized to a single or multiple oligonucleotide primer. The primer(s) is then extended by DNA polymerase or reverse transcriptase in the presence of one, two or three types of the pre-labeled free nucleotide. At least one of the four required types of nucleotides for continuous extension is left out of the reaction or is replaced by the corresponding type of terminator nucleotide such as dideoxynucleotides. The specific target nucleic acid can then be detected and quantified by measuring the presence or absence of the signal generated by the label on the extended primer(s). As the extended primer is separated away from the free nucleotides, the extended primer is assayed for incorporation of the label.<br>
The primer corresponding to a position on the target nucleic acid will be extended, and a population of equal length (isometric) primer extended nucleic acid will be made, because a definite number of nucleotides is sequence-dependently incorporated. Quantification of these equally extended primers will accurately quantify the number or amount of the target nucleic acid. If there are many copies of the target DNA or RNA in the sample, the number of copies of the primer extended product incorporating the labeled nucleotide will be correspondingly increased, contributing to a stronger overall signal. Thus, by comparing the strength of the signal observed in the unknown sample with a standardized known amount of DNA or RNA, it is possible to detect and/or quantify the amount of the target DNA or RNA in the sample. In another embodiment, the specific target nucleic acid can be detected or quantified by measuring the amount of these equal length primer extended nucleic acid species using mass spectrometry method.<br>
The lack of a free nucleotidein the reaction buffer causes the primer extension to terminate where the missing nucleotide would have been inserted. Thus, a discrete length of the primer extension product is obtained<br>
Many obvious variants are possible within the realm of the present invention. For example, not just one type of nucleotide but two or three types of nucleotides<br><br>
maybe absent in the primer extension reaction. Also, various labels could be used, which are not limited to radioactive nucleotides but can be fluorescent, as well as enzymatic.<br>
As used herein, "nucleic acid" or "nucleotide" can be a deoxyribonucleic acid, a ribonucleic acid, or a copolymer of deoxyribonucleic acid and ribonucleic acid. The sample of nucleic acids can be natural or synthetic. The sample of nucleic acid can be naturally occurring nucleic acid, and can be obtained from any organism. Some examples of organisms to which the method of the present invention is applicable include plants, microorganisms, viruses, birds, vertebrates, invertebrates, mammals, human beings, horses, dogs, cows, cats, pigs, or sheep.The target nucleic acid can occur naturally, or can be synthesized enzymatically in vivo, synthesized enzymatically in vitro, or synthesized non-enzymatically.<br>
The sample containing the nucleic acid or acids of interest can comprise genomic DNA from an organism, RNA transcripts thereof, or cDNA prepared from RNA transcripts thereof The sample containing the nucleic acid or acids of interest can also comprise extragenomic DNA from an organism, RNA transcripts thereof, or cDNA prepared from RNA transcripts thereof Also, the nucleic acid or acids of interest can be synthesized by the polymerase chain reaction.<br>
The nucleic acid of interest can comprise non-natural nucleotide analogs such as deoxyinosine or 7-deaza-2-deoxyguanosine. These analogues destabilize DNA duplexes and could allow a primer annealing and extension reaction to occur in a double-stranded sample without completely separating the strands.<br>
The nucleic acid of interest can comprise one or more moieties that permit affinity separation of the nucleic acid of interest from the unincorporated reagent and/or the primer. For example, the nucleic acid of interest can comprise biotin which permits affinity separation of the nucleic acid of interest from the unincorporated reagent and/or the primer via binding of the biotin to the avidin family of molecules, which is attached to a solid support. The sequence of the nucleic acid of interest can<br><br>
comprise a DNA or RNA sequence that permits affinity separation of the nucleic acid of interest from the unincorporated reagent and/or the primer via base pairing to a complementary sequence present in a nucleic acid attached to a solid support. The nucleic acid of interest can be labeled with a detectable marker; this detectable marker can be different from any detectable marker present in the reagent or attached to the primer.<br>
In this regard, the term "normal nucleotide" or "normal base" is defined as the wild-type or previously known standard nucleotide base from which a mutation is sought to be identified at the base site. By "standard nucleotide base", it includes any known base, which may include wild-type or a known mutant base so long as the base is known and it is desired to know its variant. Thus, as an example, normal base can be a known wild-type base for which a mutation is sought at the position. Reversely, the known base can be a known mutant for which the presence of a wild-type base is sought at the position. Alternatively, the known normal base can be a known mutant for which another mutant variant base is sought. Therefore, the method of the invention can be applied to any known sequence that can be used to determine the presence of any other base variant at the site.<br>
As used herein, the term "primer" or "oligonucleotide primer" refers to an oligonucleotide which is capable of acting as a point of initiation of synthesis when placed under conditions that allow for synthesis of a primer extension product which is complementary to a nucleic acid (template) strand, in the presence of various factors such as for example, nucleotides and enzymes such as DNA polymerase, and at a suitable temperature and pH.<br>
The term "primer" is alternatively defined as any nucleic acid fragment obtained from any source. For example, the primer can be produced by fragmenting larger nucleic acid fragments such as genomic DNA, cDNA or DNA that has been obtained through PCR. In other words, the nature of the primer is not limited by how the primer is obtained, whether it be by fragmenting naturally or synthetically<br><br>
occurring nucleic acid or by synthesizing the nucleic acid primer. Furthermore, the primer can be oligodeoxyribonucleotide, a copolymer of oligodeoxyribonucleotides, an oligoribonucleotides, a copolymer of ribonucleotides, or a copolymer of deoxyribonucleotides and ribonucleotides. The primer can be either natural or synthetic. The oligonucleotide primer can be synthesized either enzymatically in vivo, enzymatically in vitro, or non-enzymatically in vitro. The primer can be labeled with a detectable marker; this detectable marker can be different from any detectable marker present in the reagent or attached to the nucleic acid of interest. In addition, the primer must possess sequence corresponding to the flanking sequence at a specific position of interest adjacent to, and upstream of, the nucleotide base to be identified.<br>
In addition, the primer must be capable of hybridizing or annealing with nucleotides present in the nucleic acid of interest. One way to accomplish the desired hybridization is to have the template-dependent primer be substantially complementary or fially complementary to the known base sequence.<br>
The oligonucleotide primer can comprise one or more moieties that link the primer to a solid support for affinity separation of the primer from the unincorporated reagent and/or the nucleic acid of interest. Such affinity moeties include, but are not limited to, digitonin, magnetic beads, and ligands, such as protein ligands, including antibodies. Preferably, the moiety is biotin. In the case of using biotin, the primer comprising biotin permits affinity separation of the primer from the unincorporated reagent and/or nucleic acid of interest via binding of the biotin to streptavidin which is attached to a solid support. The sequence of the oligonucleotide primer can comprise a DNA sequence that permits affinity separation of the primer from the unincorporated reagent and/or the nucleic acid of interest via base pairing to a complementary sequence present in a nucleic acid attached to a solid support.<br>
As used herein, the term "primer extension reaction" refers to the reaction conditions in which the template-dependent nucleic acid synthesis reaction is carried out. The conditions for the occurence of the template-dependent, primer extension<br><br>
reaction can be created, in part, by the presence of a suitable template-dependent enzyme. Some of the suitable template-dependent enzymes are DNA polymerases. The DNA polymerase can be of several types. The DNA polymerase must, however, be primer and template dependent. For example, E. coli DNA polymerase I or the "Klenow fragment" thereof, T4 DNA polymerase, T7 DNA polymerase ("Sequenase"), T. aquaticiis DNA polymerase, or a retroviral reverse transcriptase can be used. RNA polymerases such as T3 or T7 RNA polymerase could also be used in some protocols. Depending upon the polymerase, different conditions must be used, and different temperature ranges may be required for the hybridization and extension reactions.<br>
As used herein, the term "primer extension strand" includes the strand that is formed opposite the template in a duplex after the primer has been added. Preferably, the extension of the primer is terminated by the incorporation of the terminator to the primer extension strand.<br>
As used herein, the term "template" is defined as a nucleic acid, including double strand DNA, single strand DNA and RNA or any modification thereof, and can be any length or sequence.<br>
As used herein, the term "terminator" or "chain terminator" is meant to refer to a nucleic acid base, such as A, G, C, T or U, or an analogue that effectively terminates the primer extension reaction when it is incorporated into the primer extension strand opposite the template strand. Preferably, the terminator is a dideoxynucleotide. Also preferably, the terminator is either unlabeled or is labeled so that it is distinguished from the label on the non-terminator. Also as used herein, . when the term "terminator" or "chain terminator" are referred to in the singular, it does not mean that a single nucleotide molecule is used. Rather, the singular form of the term "terminator" refers to the type of nucleotide, nucleic acid base or nucleic acid analogue that is used in the assay. For example, if the terminator is ddA, then all of the ddA's in the aggregate are referred to in the singular form, and not just a single<br><br>
molecule of ddA. Alternatively, the "terminator" may be the absence of the specific type of nucleotide so that primer extension is stopped by the lack of the specific nucleotide at the locus. For example, if it is desired that the primer extension reaction , be stopped opposite a "C" on the template strand, the non-terminating bases A, T and G should be included in the primer extension reaction mixture, but not "G", which is the complement of "C". Thus, the absence of the complementary base will cause termination of the primer extension reaction with a similar result as adding a dideoxy terminator nucleotide, for example.<br>
As used herein, the term "non-terminator" or "non-chain terminator" includes a nucleotide base that does not terminate the extension reaction when it is incorporated into the primer extension strand. Preferably, at least one non-terminator in the primer extension reaction is labeled. Also as used herein, when the term "non-terminator" or "non-chain terminator" are referred to in the singular, it does not mean that a single nucleotide molecule is used. Rather, the singular form of the term "non-terminator" refers to the type of nucleotide, nucleic acid base or nucleic acid analogue that is used in the assay. For example, if the terminator is G, then all of the G's in the aggregate are referred to in the singular form, and not just a single molecule of G.<br>
As used herein, the term "mutant" or "mutation" indicates any base on the template strand that is different from the wild-type or normal base. The mutation that can be detected using the method of the instant invention can be any type of mutation at all, including, single base mutation, insertion, deletion, or gene translocation, so long as the base on the template directly opposite to the base immediately 3' to the annealed primer is affected.<br>
As used herein, the term "label" refers to any molecule that is linked to the terminator or non-terminator nucleotide to provide a detectable signal. The label may be radioactive, chemiluminescent, protein ligand such as an antibody, or if a fluorescent group is used, a different fluorescent group may be used for each type of<br><br>
non-terminating nucleotide base. These fluorescent tags would have the property of having spectroscopically distinguishable emission spectra.<br>
Alternatively, the method of determining the level of incorporation of a nucleotide base in the primer extension product can be measured by mass spectrometry techniques as exemplified in U.S. Patent No. 5,885,775, which is incorporated herein by reference in its entirety.<br>
As used herein, the phrase "high stringency hybridization conditions" refers to nucleic hybridization conditions, such as but not limited to a wash condition of 0. IXSSC, at 42°C. Hybridization conditions generally can be found in general Molecular Biology protocol books, such as Ausubel et al., Current Protocols in Molecular Biology Greene and Wiley, pub. (1994), which is incorporated herein by reference in its entirety.<br>
As used herein, "thin layer chromatography (TLC)" can be carried out in paper medium based on cellulose products, but can be made of any substance that allows for molecules to be finely divided and formed into a uniform layer. This substance includes, but is not limited to, inorganic substances such as silica gel, aluminum oxide, diatomaceous earth or magnesium silicate. Organic substances include, but are not limited to, cellulose, polyamide, or polyethylene powder. Thin layer chromatography methods are described generally in Chemical protocol books, such as generally set forth in Freifelder, Physical Biochemistry - Applications to Biochemistry arid Molecular Biology, seconded., published by Freeman and Co. (1982), which is incorporated herein by reference in its entirety, especially Chapter 8, which discusses chromatographic techniques, and in particular thin layer chromatography at pages 229-232.<br>
A modification of the method for indentifying and/or quantifying a nucleic acid of interest is to separate the primer extended strand from the nucleic acid of interest after the extension reaction by using appropriate denaturing conditions. The denaturing conditions can comprise heat, alkali, formamide, urea, glyoxal, enzymes,<br><br>
and combinations thereof. The denaturing conditions can also comprise treatment with2.0NNaOH.<br>
It can be appreciated by a person of skill in the art that the terminator can be labeled with a different label from the non-terminator, which can then be used to differentiate between incorporation of terminator or non-terminator in the primer extension strand. The terminator exemplified as being the absence of the particular type of nucleotide in the present application only for purposes of simplicity of illustration, but this illustration should not be construed to limit the claims in any way. Differentially labeled or unlabeled terminator is also encompassed by the invention, so long as the label on the terminator is different from the label on the non-terminator.<br>
It can also be appreciated by a person of skill in the art that so long as the sequence of the template is at least partially known, a primer can be designed that binds to the template strand such that the binding of the primer on the template strand can occur. It can also be appreciated by a person of skill in the art that the method of the invention can be practiced by using several primers in one or more assay tube.<br>
A feature of the method of the invention is that strong signal can be generated if the non-terminators are uniformly labeled because of the additive signal effect achieved by the incorporation of several labeled non-terminators incorporated in the primer extension strand. Accuracy is enhanced when signals are observed from using different labels specific to various terminators or non-terminators.<br>
It is also an object of this invention to provide a kit and reagents for rapidly and accurately determining the presence or absence of a target nucleic acid in a sample quantitatively or non-quantitatively as desired. Each component of the kit(s) may be individually packaged in its own suitable container. The individual containers may also be labelled in a manner which identifies the contents. Moreover, the individually packaged components may be placed in a larger container capable of holding all desired components. Associated with the kit may be instructions which explain how to use the kit. These instructions may be written on or attached to the kit.<br><br>
The following examples are offered by way of illustration of the present invention, and not by way of limitation.<br>
EXAMPLE<br>
Total RNA was extracted from rat brain by RNAzol B (Tels-tel, TX) method. The concentration of total RNA was measured by O.D. 260 nm absorbance. Total RNA was diluted by RNase-free diethylpyrocarbonate (DEPC) treated water. 5 fil of diluted RNA solution with different amounts as indicated in Table 2 was aliquoted into each tube and then mixed with 1 Μ1 synthetic oligonucleotide primer 5'-GTGGGAACCGTGTCA-3' (SEQ ID NO.i), which is a sequence matched to a rat brain specific cDNA (unpublished data). The RNA-primer mixture was heated at 70° C for 3 minutes and incubated on ice for 3 minutes. After quick spinning the tube, the primer extension reaction was started by adding 14 \x\ reaction mix containing Tris-HCl buffer (pH 7.5) at final concentration of 20 mM, 15 units RNase inhibitor, 0.5 mM dATP, dGTP, 1 ^μ dCTPa^-P and 100 units MMVL-reverse transcriptase. The reaction was performed at 37° C for 20 minutes and stopped by heating the reaction tube at 100° C for 2 minutes. A 1 |il reaction mixture was applied to a thin layer chromatography (TRIM USA Maryland) to separate out fi-ee dCTPa^P. The radioactivity of the labeled primer was then subjected to counting by scintillation counter (Beckman LS 5000). The results are shown in Table 2.<br>
All of the above steps involve chemistries, manipulations, and protocols that have been, or are amenable to being, automated. Thereby, incorporation of the preferred mode of practice of this invention into the operation of a suitably programmed robotic workstation should resuh in significant cost savings and increases in productivity for virtually any diagnostic procedure that depends on the<br><br>
detection of specific nucleotide sequences or sequence differences in nucleic acids derived from biological samples.<br>
All of the references cited herein are incorporated by reference in their entirety.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WE CLAIM;<br>
1.	A method for detecting or quantifying a target nucleic acid in a sample comprising: (a) preparing at least one primer specifically matched to a predetermined position of the target nucleic acid; (b) annealing the primer or primers from (a) with the target nucleic acid under high stringency conditions to obtain a primer-nucleic acid duplex at the predetermined position of the target nucleic acid; (c) mixing the primer-nucleic acid duplex from (b) with a non-terminator nucleotide mixture, wherein at least one A,T,G, or C non-terminator nucleotides is omitted from the non-terminator nucleotide mixture, and wherein at least one non-terminator nucleotide of the non-terminator nucleotide mixture which is not omitted is labeled with a detectable marker; (d) performing isometric primer extension by enzymatic or chemical reaction in an appropriate buffer to form isometric primer extension products, wherein the primer extension terminates at a target nucleic acid nucleotide complementary to the omitted non-terminator nucleotide of (c); and (e) detecting or quantifying the amount of labeling signal on the isometric primer extension products.<br>
2.	The method according to claim 1, wherein the primer is a nucleic acid primer, an oligodeoxyribonucleotide, an oligoribonucleotide, or a copolymer of deoxyribonucleic acid and ribonucleic acid.<br>
3.	The method according to claim 1, wherein the nucleic acid of interest is a deoxyribonucleic acid, a ribonucleic acid, or a copolymer of deoxyribonucleic acid and ribonucleic acid.<br><br>
4.	The method according to claim 1, wherein the non-terminator nucleotides is deoxyribonucleotides or ribonucleotides.<br>
5.	The method according to claim 1, wherein the non-terminator nucleotide mixture comprises:<br><br>
(a)	dATP, dCTP, dGTP;<br>
(b)	dATP, dCTP, dTTP or dUTP;<br>
(c)	dATP, dGTP, dTTP or dUTP; or<br>
(d)	dCTP, dGTP, dTTP or dUTP;<br><br>
6.	The method according to claim 1, wherein said detectable marker comprises an enzyme or protein moiety, radioactive isotope, a fluorescent moiety or a chemical group.<br>
7.	The method according to claim 1, wherein the primer extension products are formed using a template-dependent enzyme.<br>
8.	The method according to claim 7, wherein the template-dependant enzyme is DNA polymerase, RNA polymerase or reverse transcriptase.<br>
9.	The method according to claim 7, wherein the template-dependant enzyme is E.coli DNA polymerase I, a Klenow fragment thereof, T4 DNA polymerase, T7 DNA polymerase. Thermophilic DNA polymerase, retroviral reverse transcriptase, or a combination thereof.<br><br>
10.	The method according to claim 1, wherein the target nucleic acid is synthesized enzymatically in vivo, or in vitro, or synthesized non-enzymatically.<br>
11.	The method according to claim 1, wherein the target nucleic acid is synthesized by polymerase chain reaction.<br>
12.	The method according to claim 1, wherein the target nucleic acid comprises non-natural nucleotide analogs.<br>
13.	The method according to claim 12, wherein the non-natural nucleotide analogs comprise deoxyinosine or 7-deaza-2'-deoxyguanosine.<br>
14.	The method according to claim 1, wherein the target nucleic acid comprises genomic DNA from an organism, RNA transcripts thereof, and cDNA prepared from RNA transcripts thereof<br>
15.	The method according to claim 14, wherein the organism is a plant,<br>
microorganism, bacteria, or virus.<br>
16.	The method according to claim 14, wherein the organism is a vertebrate or<br>
invertebrate.<br>
17.	The method according to claim 14, wherein the organism is a mammal.<br>
18.	The method according to claim 17, wherein the mammal is a human being.<br><br>
19.	The method according to claim 1, wherein an amplification step is performed on the target nucleic acid.<br>
20.	The method according to claim 19, wherein the amplification step comprises cloning, transcription, polymerase chain reaction (PCR), ligase chain reaction (LCR), strand displacement amplification (SDA), or loop mediated isothermal amplification (LAMP).<br>
21.	The method according to claim 1, wherein the primer comprises one or more moieties that permits affinity separation of the primer from the unincorporated reagent and/or the nucleic acid of interest.<br>
22.	The method according to claim 1, wherein the primer comprises one or more moieties that allows immobilization of the primer onto a solid support to produce an immobilized primer sequence.<br>
23.	The method according to claim 21 or 22, wherein the moieties comprises a special chemical groups such as biotin or digitonin.<br>
24.	The method according to claim 21 or 22, wherein the moieties comprises a DNA or RNA sequence that allows the primer to link to a solid support via base pairing to a complementary sequence present in the solid support.<br>
25.	The method according to claim 1, wherein the primer is directly synthesized on a solid support to produce an immobilized primer sequence.<br><br>
26.	The method according to claim 25, wherein the synthesis is accomplished by enzymatic or chemical or physical method.<br>
27.	The method according to claim 1, wherein the primer is immobilized onto a solid support to produce immobilized isometric primer extension products.<br>
28.	The method according to claim 1, wherein the primer is reversibly immobilized onto a solid support.<br>
29.	The method according to claim 28, wherein the primer is cleaved from the solid support by a chemical, enzymatic or physical process.<br>
30.	The method according to claim 1, wherein the target nucleic acid is immobilized onto a solid support to produce immobilized isometric primer extension products.<br>
31.	The method according to claim 1, wherein the target nucleic acid is reversibly immobilized onto a solid support.<br>
32.	The method according to claim 31, wherein the target nucleic acid is cleaved from the solid support by a chemical, enzymatic or physical process.<br>
33.	The method according to claim 27, 28, 30 or 31, wherein immobilization is accomplished via a photocleavable bond.<br>
34.	The method according to claim 22, 24, 25, 27, 28, 30 or 31, wherein the solid support comprises beads, flat surfaces, chips, capillaries, pins, combs or wafers.<br><br>
35.	The method according to claim 27, 28, 30 or 31, wherein said immobilization is<br>
accomplished by hybridization between a complimentary capture nucleic acid<br>
molecule, which has been previously immobilized to a solid support, and a portion of<br>
the nucleic acid molecule, which is distinct from the target nucleic acid sequences.<br>
36.	The method according to claim 27, 28, 30 or 31, wherein said immobilization is<br>
accomplished via direct bonding between the solid support and a portion of the<br>
nucleic acid molecule, which is distinct from the target nucleic acid sequence.<br>
37.	A method for detecting or quantifying a target nucleic acid in a sample<br>
substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSAgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">0451-mas-2001  others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBhYnN0cmFjdCBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0451-mas-2001 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBjbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">0451-mas-2001 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBjb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">0451-mas-2001 correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">0451-mas-2001 description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">0451-mas-2001 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBkcmF3aW5ncyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0451-mas-2001 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">0451-mas-2001 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0451-mas-2001 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">0451-mas-2001 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDQ1MS1tYXMtMjAwMSBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">0451-mas-2001 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="200699-polyethylene-glycol-modified-polyester-fibers-and-method-for-making-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="200701-a-fuel-additive-composition-for-controlling-pollution-emission-a-method-for-the-treatment-of-liquid-hydrocarbon-containing-fuels-and-a-fuel-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>200700</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>451/MAS/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>8/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-May-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Jun-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>XIAO BING WANG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1315 MALBAY DRIVE, LUTHERVILLE, MARYLAND 21093</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>XIAO BING WANG</td>
											<td>1315 MALBAY DRIVE, LUTHERVILLE, MARYLAND 21093</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/209,987</td>
									<td>2001-05-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/200700-a-method-for-detecting-or-quantifying-a-traget-nucleic-acid-in-a-sample by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:29:53 GMT -->
</html>
