<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/201062-a-method-of-producing-a-plant-that-is-resistant-to-a-fungus-that-produces-a-trichothecene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 201062:A METHOD OF PRODUCING A PLANT THAT IS RESISTANT TO A FUNGUS THAT PRODUCES A TRICHOTHECENE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PRODUCING A PLANT THAT IS RESISTANT TO A FUNGUS THAT PRODUCES A TRICHOTHECENE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of preventing fungal contamination in a crop plant The preseni invention relates to a method of preventing fungal contamination a crop plant comprising the steps of: _a) transforming a plant cell with iii) a heterologous gene encoding the amino acid sequence of SEQ D NO; 2, SEQ ID NO; 6 or SEQ ID NO: 8, or iv) a heterologous gene encoding an amino acid sequence that is 95% identical to the amino acid sequence of SEQ ID NO: 2, SEQ ID NO: 6 or SEQ ID NO: 8, wherein said amino acid sequence has trichothecene 3-0 acetyltransferase activity; b) expressing the gene product at a biologically significant level; and c) regenerating the plant cell into a plant.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The invention relates to a novel elongase gene with the sequences stated in sequence SEQ ID NO:l, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 or their homologs, derivatives or analogs, to a gene construct comprising this gene or its homologs, derivatives and analogs, and to its use. The invention also relates to vectors or transgenic organisms comprising an elongase gene with the sequence SEQ ID NO:l, SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11,or its homologs, derivatives and analogs. The invention furthermore relates to the use of the elongase gene sequences alone or in combination with further elongases and/or further fatty acid biosynthesis genes. The present invention relates to a novel elongase gene with the sequence SEQ ID N0:1 or its homologs, derivatives and analogs.<br>
Furthermore, the invention relates to a process for the preparation of polyunsaturated fatty acids and to a process for introducing DNA into organisms which produce large amounts of oils and, in particular, oils with a high content of unsaturated fatty acids. Moreover, the invention relates to an oil and/or a fatty acid preparation with a higher content of polyunsaturated fatty acids with at least two double bonds and/or a triacyiglycerol preparation with a higher content of polyunsaturated fatty acids with at least two double bonds.<br>
Background of the invention<br>
Certain products and byproducts of naturally occurring metabolic processes in cells can be used for a wide spectrum of industries, including the animal feed industry, food industry, cosmetics industry and pharmaceuticals industry These molecules, which are joinly referred to as "fine chemicals'^, also include lipids and fatty acids, amongst which the polyunsaturated fatty acids constitute an exEunple of one class. Polyunsaturated fatty acids (PUFAs) are added, for example, to children's formula to increase its nutritional value. For example, PUFAs have a positive effect on the cholesterol level in the blood of humans and are therefore suitable for protection against heart disease. Fine chemicals and polyunsaturated fatty acids (PUFAs) can be isolated from animal sources, for example fish, or microorganisms. Culturing these<br><br>
microorganisms allows large founts of one or more of the desired molecules to be produced and isolated-<br>
Microorganisms which are especially suitable for preparing PUFAs are microorganisms such as^Thraustochytria or Schizochytria strains, algae such as Phaeodactyluin tricornutum or Crypthecodinium species, Ciliata such as Stylonychia or Colpidium, fungi such as Mortierella, Entomophthora or MucorliA number of mutant strains of the microorganisms in question which produce a series of desirable compounds, including PUFAs, have been developed by strain selection. The selection of strains with an improved production of a certain molecule is, however, a time-consuming and difficult procedure. Also disadvantageous is the fact that only specific unsaturated fatty acids, or only a specific fatty acid spectrum, can be produced by a defined microorganism.<br>
As an alternative, fine chemicals can suitably be produced on a large scale via the production of plants which have been developed in such a way that they produce the abovementioned PUFAs. Plants which are particularly well suited to this purpose are oil c^^ps which contain large amounts of lipid compounds, such as oilseed rape, canola, linseed, soya, sunflowers, borage and evening primrose. However, other crops which contain oils or lipids and fatty acids are well suited, as mentioned in the detailed description of the present invention. Conventional plant breeding has led to the development of a series of mutant plants which produce a spectrum of desirable lipids and fatty acids, cofactors and enzymes. However, the selection of novel plant varieties with an improved production of a certain molecule is a time-consuming and difficult procedure or even impossible if the compound does not occur naturally in the plant in question, such as in the case of polyunsaturated C2o-fatty acids, and C22-fatty acids and those with longer carbon chains.<br>
Abstract of the invention<br>
The invention provides novel nucleic acid molecules which are suitable for identifying and isolating elongase genes of PUFA biosynthesis and which can be used for the modification of oils, fatty acids, lipids, lipid-derived compounds and, most preferably, for the preparation of polyunsaturated fatty acids, since there remains a great demand for novel genes which encode enzymes which are involved in the biosynthesis of unsaturated fatty acids and which make it possible for these to be prepared on an industrial scale. In particular, there is a demand for fatty acid biosynthesis enzymes which make possible the<br><br>
elongation of polyunsaturated fatty acids, preferably with two or more double bonds in the molecule. The nucleic acids according to the invention encode enzymes which have this ability.<br>
Microorganisms such as Phaeodactylum, Colpidium, Mortierella, Entomophthora, Mucor, Crypthecodinium and other algae and fungi and plants, in particular oil crops, are generally used in industry for the production of a large number of fine chemicals on a large scale.<br>
As long as cloning vectors and techniques are available for the genetic manipulation of the abovementioned microorganisms and Ciliata, as disclosed in WO 98/01572 and WO 00/23604, or algae and related organisms, such as Phaeodactyluin tricornutum, described in Falciatore et al. [1999, Marine Biotechnology 1(3):239-251]; and Dunahay et al. [1995, Genetic transformation of diatoms, J. Phycol. 31:10004-1012] and the references cited therein, the nucleic acid molecules according to the invention can be used for the recombinant modification of these organisms so that they become better or more efficient producers of one or more fine chemicals, especially unsaturated fatty acids. This improved production or production efficiency of a fine chemical can be caused by a direct effect of manipulating a gene according to the invention or by an indirect effect of this manipulation.<br>
Mosses and algae are the only known plant systems which produce considerable amounts of polyunsaturated fatty acids, such as arachidonic acid (= ARA) and/or eicosapentaenoic acid {= EPA) and/or docosahexaenoic acid (= DHA). Mosses contain PUFAs in membrane lipids, while algae, organisms related to algae and some fungi also accumulate considerable amounts of PUFAs in the triacylglycexol fraction. Thus, nucleic acid molecules which are isolated from such strains which also accumulate PUFAs in the triacylglycerol fractions are particularly suitable for modifying the lipid and PUFA production systems in a host, in partiuclar in Tiicroorganisms, such as the abovementioned microorganisms, and Dlants, such as oil crops, for example oilseed rape, canola, linseed, soya, sunflower, borage, castor-oil plant, oil palm, safflower (Carthamus tinctorius), coconut, peanut or cacao bean. Furthermore, nucleic acids from triacylglycerol-accumulating tiicroorganisms can be used for identifying such DNA sequences and anzymes in other species which are suitable for modifying the Diosynthesis of PUFA precursor molecules in the organisms in guestion. Microorganisms which accumulate PUFAs such as ARA, EPA 3r DHA in triacylglycerols are, in particular, microorganisms such as Crypthecodinium cohnii and Thraustochytrium species, rhraustochytria are also closely related to the Schizochytria<br><br>
strains in terms of phylogenetics. Even though these organisms are not closely related to mosses such as Physcomitrella, sequence similarities at the DNA sequence and, in particular, polypeptide level can be observed to such an extent that DNA molecules can be identified, isolated and characterized functionally in heterologous hybridization experiments, sequence alignments and experiments using the polymerase chain reaction, even from organisms which are distantly related in terms of evolution. In particular, consense sequences can be derived which are suitable for the heterologous screening or the functional complementation and prediction of gene functions in third species. The ability to identify these functions, for example to predict the substrate specificity of enzymes, can therefore be of significant importance. Furthermore, these nucleic acid molecules may act as reference sequences for mapping related genomes or for deriving PCR primers.<br>
The novel nucleic acid molecules encode proteins termed in the present context PUFA-specific elongases (= PSEs, or PSE in the singular). These PSEs can, for example, exert a function which is involved in the metabolism (for example in the biosynthesis or in the breakdown) of compounds required for lipid or fatty acid synthesis, such as PUFAs, or which participate in the transmembrane transport of one or more iipid/fatty acid compositions, either into the cell or out of the cell.<br>
This novel application shows the isolation of such novel elongase genes in greater detail. For the first time, we have isolated elongase genes which are suitable for producing long-chain polyunsaturated fatty acids, preferably having more than eighteen or twenty carbon atoms in the carbon skeleton of the fatty acid and/or at least two double bonds in the carbon chain while being derived from typical organisms which contain high amounts of PUFAs in the triacylglycerol fraction. This means, in the singular, a PSE gene or PSE protein or, in the plural, PSE genes or PSE proteins. Other known patent applications and publications disclose, or show, no functionally active PSE gene, even though various known patent applications exist which show the elongation of saturated fatty acids of short or medium chain length (WO 98/46776 and US 5,475,099) or the elongation or production of long-chain fatty acids, but which then have no more than one double bond or lead to long-chain fatty acid wax esters (see WO 98/54954, WO 96/13582, WO 95/15387). The invention presented here describes the isolation of novel elongases with novel properties. Starting from the sequence stated in SEQ ID N0:1, it was possible<br><br>
to find further nucleic acids which encode elongases which elongate unsaturated fatty acids.<br>
WO 99/64616, WO 98/46763, WO 98/46764 and WO 98/46765 describe the production of PUFAs in transgenic plants and demonstrate the cloning and functional expression of corresponding desaturase activities, in particular from fungi, but demonstrate no PSE-encoding gene and no functional PSE activity. The expression of the desaturase activities leads to a shift in the fatty acid spectrum in the transgenic plants, but no increased content of unsaturated fatty acids was observed. The production of a trienoic acid with Ci8-carbon chain has been demonstrated and claimed with reference to gamma-linolenic acid, but the production of very long-chain polyunsaturated fatty acids (with a C20- and longer carbon chain and of trienoic acids and higher unsaturated types) has, however, not been demonstrated to date.<br>
To prepare long-chain PUFAs, the polyunsaturated Cie- or Cig-fatty acids must be elongated by at least two carbon atoms by the enzymatic activity of an elongase. The jiucleic acid sequence SEQ ID N0:1 according to the invention enodpdes the first plant elongase which is capable of elongating the Cig- or Cig-fatty acids with at least two double bonds in the fatty acid by at least two carbon atoms. After one elongation cycle, this enzyme activity leads to Cjo-fatty acids, and after two, three and four elongation cycles to €22-1  C24- or C26-fatty acids. Longer-chain PUFAs can also be synthesized with the aid of the other elongases which are disclosed (SEQ ID NO: 3, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID N0:9 and SEQ ID N0:11). They can be employed individually, multiply or, for example, in addition to the PUFA elongase from the moss Physcomitrella patens (SEQ ID N0:1) for increasing the PUFA content in a novel process for the preparation of PUFAs. The activity of the elongases according to the invention preferably leads to C2o-fat'ty acids with at least two double bonds in the fatty acid molecule, preferably with three or four double bonds, especially preferably three double bonds, in the fatty acid molecule and/or C22-fa'tty acids with at least two double bonds in the fatty acid molecule, preferably with four, five or six double bonds, especially preferably with five or six double bonds, in the molecule. After the elongation by the enzyme according to the invention has taken place, further desaturation steps may be carried out in order to obtain the highly desaturated fatty acids. The products of the elongase activities and of the further desaturation with is a possibility therefore lead to preferred PUFAs with a higher degree of desaturation, such as docosadienoic acid, arachidonic acid, (u6-eicosatrienedihomo-Y-linolenic acid, eicosapentaenoic acid, (j)3-eicosatrienoic acid.<br><br>
to3-eicosatetraenoic acid, docosapentaenoic acid or docosahexaenoic acid. Substrates of the enzyme activity according to the invention are, for example, taxol acid;<br>
7,10,13-hexadecatrienoic acid, 6,9-octadecadienoic acid, linolic acid, linolenic acid, a- or 7-linolenic acid or stearidonic acid, and also arachidonic acid, eicosatetraenoic acid, docosapentaenoic acid, eicosapentaenoic acid. Preferred substrates are linolic acid, v-linolenic acid and/or a-linolenic acid, and also arachidonic acid, eicosatetraenoic acid, docosapentaenoic acid and eicosapentaenoic acid. Arachidonic acid, docosapentaenoic acid and eicosapentaenoic acid are especially preferred. The Cie- or Cig-fatty acids with at least two double bonds in the fatty acid can be elongated by the enzymatic activity according to the invention in the form of the free fatty acid or in the form of the esters, such as phospholipids, glycolipids, sphingolipids, phosphoglycerides, monoacylglycerol, diacyiglycerol or triacylglycerol. Of particular importance for human nutrition is conjugated linolic acid "CLA". Chh  is to be understood as meaning, in particular, fatty acids such as C18:2 ' =^S' n^ans or the isomer C18:2 lotrans, 12 cis, which can be desaturated or elongated after uptake in the body owing to human enzyme systems and can contribute to health-promoting effects. Elongases according to the invention also allow those conjugated fatty acids which have at least two double bonds in the molecule to be elongated and thus make available such health-promoting fatty acids for human nutrition. Other examples of conjugated fatty acids are alpha-parinaric acid, eleostearic acid and calendulic acid.<br>
Given cloning vectors for use in plants and in the transformation of plants, such as those which are published, and cited, in: Plant Molecular Biology and Biotechnology (CRC Press, Boca Raton, Florida), chapter 6/7, pp. 71-119 (1993); r.F. White, Vectors for Gene Transfer in Higher Plants; in: Transgenic Plants, Vol- If Engineering and Utilization, Ed.: Kung and R. Wu, Academic Press, 1993, 15-38; B. Jenes et al., Techniques for Gene Transfer, in: Transgenic Plants, Vol. 1, Engineering and Utilization, Ed.: Kung and R. wu, Academic Press (1993), 128-143; Potrykus, Anna- Rev. Plant Physiol. Plant Molec. Biol. 42 (1991), 205-225)), the nucleic acids according to the invention can be used for the recombinant modification of a broad spectrum of plants so that they become a better, more efficient or modified producer of one or more lipid-derived products, such as PUFAs. This improved production or production efficiency of a lipid-derived product, such as PUFAs, can be caused by the direct effect of the manipulation or by an indirect effect of this manipulation.<br><br>
ft	ja	n	■a	D"	.—.	D"	rt	.	.	ft	rt	a	a	a&gt;	•-3	3	0)	n	0	tr	3	O	"1	P	e	O	tr	o	3	P	n	rt	o	H	P	(n	&gt;p	tr	13	p-	t^	Hi	'^<br>
o	c	3*	H	H-	s	fD	:T	«	3"	3-	ID	H-	H'	rr	O	3	o	3	p-	d	cr	p	cr	p-	0	ID	cr	d	O	0	0	H	cr	a	ID	d	p-	M	3	cr	H	p-		3-<br>
	(U	m	o	0	cr	Hi	fD	3-	(D	fD	0)	H	en	fD	M	a	3	tD	0	1	ID	rt	H-	ft	hti		ID	rt	rt	M	t.	^	ID	
rt	3	3	a.	[fl	H-	O		0			p-	fD	0		(D	^	-d		01	<br>
	3	rt	P*	cr	P	p-	3	H	p-	H	P	P		p	d	01	01	a	►1		3	M	cn	h<br>
ET	rt	H-	p	*
(D	H-	n	n	3	3*	fD	m	tD	O	U)	(D	rt	M	d	d	H	c	h	3	H	o		ft	a	rt	0	o	ID	p-	tn	a	P-	3	rl-	3	p-	3	n	P	d	p			<br>
	rt	P	rt	rt			01		H	P]	D-	M	(D	rt	h-'		3		rt	^	p	P	"
O	H-	M	H-	3*	§	rt	H-	•a	3			&gt;
		13	M	X<br>
(D	m	^-^	0	01	0	3*	h	c	w	ifl	Hi		p.	-Q	0)	M	01	o	ID	0	0)	p-	O	cr	01	p	tn	to		a	cr		O	ID	01		ID		a	ID	o	tl	o	p-<br>
H'	U)	•	3	tt	c	(D	fD	0"	(-■	tD	H-	0		(D		O	-*	H	01	Hi	'H*	a	Hi	ID		01			O	a	ID	o	a	0			01	a		H	p-	O	rt	01<br>
H'				H-	H-	•	P-	ID	01		p"	a	ID	rt<br>
	0	"1	o	O	a			rt	3	tD	tD		O	fD	H	d	n		U	0	0		rt	[0	H	a	Hi	ID		3	fD		n	01	cr	P-	01		0	0		d	P-	01<br>
0	Hi	0	H			a	HI	H-	(D			H-		(0	O	n	p	13		3	13	a&gt;	cr	rt		'^	0	A	0	^	3	p	rt		p-	01		3		H	p	p	3	<br>
H		H		13	M	fD	t--	rt	rt	P&gt;	O	3	0&gt;	p-	s	rt	3	V)	0	fD	rt	3	fD	o	p-		H	d	rt		a		P-	p-	o	o	P	o	rt		H	rt		P<br>
	rt	rt	-u	Pi	(D	(Q	a	•	O	cr	a		tn		p-		t4	Hi		H-	a		M	3	01		p-	cr	r^		o	O	3	3"		Cr	
3	3"	3-	H	rt	a	rt	It		cr	n	tD	H-	1J		T3	0	3	01		O	3		0	p	rt	0	ID	H	fD	h	13	fD	3				p-	O	P	0	13	0	P	01<br>
tu	0)	fD	0	5"	c	H		3	o	0	3	H	w	o	M	3	a		rt	H	p-	P"	ID	UQ	ID		a	ID	H	P-	H	P-		rt	a	rt	P"		ft	rt		a	o	ID<br>
'^		H	a	
	a	3	d	(U	(t	►
H-	n&gt;	o	n	'
D	ID	h	rt	tn	0	0	3	ft		3	H	M	h	3"	01	ttl	13	O	ID	ti	P'	a		0	a	rt	p"	0	3	p"	n	P	P		tD	C		n	ID	rt	P	a	p-	01<br>
rt	H-	fl)	H-		(D	rt	M-		*:	iQ	01	•
(D	M	^	O	0	!--■	3-	0	cr	P)		•		rt		0	n	01		n		3		0	13	p	rt			01	P"			13	o		ID	H		rt		g	0	iQ	o<br>
M	(D		a	Hi	M	(D	(u	(D	01	rt		[U	tD	►ti	H	p-	01	P	g	13		rt		P	rt	cr	Hi	rt		^^		3	U)	a	P		P-	rt	ID	0	p-	H		Hi<br>
l-h	a	rt				H	1—		rt	O	T)	Hi	H-	tn		ID	p-	1-1	M	cn	0	3-	•d	H	fD	fD	p	a-	S	n	tr	O	PI	rt	rt	O	P	cr	h	Hi	o		rf	<br>
(D		a-	(D	rt	
M	Hi	(D	Hi	3-	H-	3		3		rt	M	fD			p-	n	p-		o	U)		3	O	3		o	rt		p-	H	rt		tP	X	rt				•Q	rt	O	H		P<br>
m	H-		Hi	tD	P)	O	P)	n	XI	u-		O	«	iQ	o	•
	D	h	H-		tr	(J	1:5	H	H	fD	tD	ft	H'	tD	H			3	3	£	cr	fD	d	3	01	3		p-	cr	H	P	3	3	•	fD	P	O	H	P	tD	M	a	cr	3C<br>
C	{D	(D	n	Hi	H-	(D	n	(D	O		X		ft	3	O	o	p	
H-		H-	H-	H-	(-■	O		(U	a	h-'	0}	ft	»■	fD	O	Hi	3	O	in	p-		
rt	n	fll	m	3	H-	c	CO	01	d	3	g	ET			H			P-	^	n	P	ID			3	a	p-	■"
cr	3-	rt	3	(D	rt	M	i	c	rf	p-	rt	PI	P-	p-	3	01<br>
	(D	p.	n		^	(D	•a	a	rt	(D	H-		0	O	P	p	3	tD	0		ft		q	cr	H	h	0)	rt		13	p-	a				01	t)	01	p-		01	0	<j></j>
10	3	
P	H-	{D		tr			h	Q		rt	■*	H	H-	o	m	rt	M			H		01	p	rt		ID	3	tn	O	H			ID		a	01	01	H			3	<br>
N	n	H	0	(D	0	0	rt	S	0	H-		0	Hi	h	H-	^a	a	c			0		ID	rt	cr<br>
1	p)	'^	Hi	3	H	H	<d h- m rt p. a p id hi p- d n o></d><br>
	M			H-			a	3		3	3"	C	(D	p-	0)	CO	U)		01	p-	'
(D		1-"	rt	O	S	-d		lO	cr		(D	n	p.	3	a	n	ID	ft	rt	3	•*	ID	P	H			P		
	rt	m	tr	P)	o	H	H-		^	O		rt		iQ		p-		cr	M	
M\	3-	H	at	M	c	o	a	rt	<br>
	0&gt;	3	H-	P&gt;			a	P-	fD	O	0	0	p-		cr	p-	Hi		Hi	a	■		rt	P"		tD	IP	3	ID	3"		iP	p-	O	p-<br>
m	(D	la			f-"	o		0	H	3	C	o	O	rt			3		*
ID	3	fD	a	^--^	a	m	p]		0		^	3	rt	o		3		tr	p-	<: fd h cr rt o p- w id p a tn m></:>
a	en		m	rt		01	3	rt	Q.	tr	o*		H-			0	*
	Ifl		p-	tr	a	3	3	'S	M		
D-	(D	!-•■	[n	3"	H-	01		3-	C	(0	tD	0	
		cr			
(V	t--	tJ	H-	C	3	(D	(D	fD	o		M	ttl	H-	3*		fD	ID	p			O	ID	h	a		tfl	a		rt	P	O	p-	0		ID	tn	rt	p-	O	tn	P		p-	cr<br>
n	
?^	(B	h	(D		(D		Hi	0	tn	3	O	o	P-			c	a	a	cr	3	ID	P	cr	M	cr		d	H	ID	P-	ID	cn				kQ	tD		3		g	Hi	p	<br>
	cn	p)	a	H	H	«	h--	
g		n		(0	Hi	H-	O	m	0	h		tD	01	3		(D		c		rt	01		a	p		H	H	3	P-		rt	3	tn		3	ID	iQ	O	0	ID	a	P	rt	6<br>
(0	3	(D	Hi	P-	fD	rt	H-	h	Hi	tD	pj			
o	01	»-•	H-	P	H	tr	(D	-d		a	n	0	S	fD			H		o	fD	3	P-	cr	p-	ID	o	p-	n	13	rt		cr			^	P	3	P'	3	ID	Hi	Hi	■
rr	■
0&gt;		c	(H	M-		3	rt	0	cr	a	M-		H-	rt		3	a	HI	H-			ID	t(	iQ	H	h	*-&gt;		h	ID	ID	01	ID		tr	01		p.	01	n	ID	3	p	p-<br>
P	cr	t--		3	H			a	tD	»	
H-	(D	p)		ifl	H-	rt	3	c			P-	o	cr	0			o	rt	p-		cr		p-	rt		0	13	P-		3	3	01	ID				3	^	p	3			ttl	p<br>
D)		n			rt	3-	(U	o	o	D)	ft	n		3		p-	rt	cr	rt		(D		3	cr	0	h	P-	a	O	d	O		01			P-			H		13	O	ID	rt<br>
3	rt			^	3"	tD	b	rt	fD	0	^	fD				p-	P-	tD	^				tr	ID	H	01	a	ID	ttl	1	
01	0			H'			a	H-	M					3		T3	0	cn					ID			■s	01			<br>
	o	P"	rt			ID		3		ID	0	ID	rt	0<br>
	H-0			n&gt; a	rt <t>	n<br>
(D	fD<br>
H	O<br>
3<br>
0 H)	t-"	rt<br>
PJ<br>
rt	0<br>
Ml			P)		P-<br>
a	3	ID	0<br>
Hi				p-<br>
0				3<br>
p			tD H<br>
O<br>
H	rt<br>
cr ID	g	o p<br>
3			&gt;		a			rt ID<br>
P-3	3 p.<br>
P P<br>
H		3 0<br>
Ml<br><br>
r<br>
such as  allosteric regulation'; for example, they might increase the allocation of the PUFA into the triacylglycerol fraction owing to an increased activity or number of other enzymes or detoxifying enzymes of the PUFA pathway which follow downstream; the viability of the seed cells might increase which, in turn, leads to better development of cells in culture or to seeds which produce the desired fine chemical. Alternatively, the PSE gene according to the invention can be manipulated in such a way that the corresponding quantities of the various lipid molecules and fatty acid molecules are produced. This can have a decisive effect on the lipid composition of the cell membrane and generates novel oils in addition to the occurrence of PUFAs which have been synthesized de novo. Since each type of lipid has different physical properties, a change in the lipid composition of a membrane can substantially modify membrane fluidity. Changes in membrane fluidity can have an effect on the transport of molecules via the membrane and on cell integrity, both of which have a decisive effect on the production of fine chemicals. In plants, moreover, these changes can also have an effect on other traits such as the tolerance to abiotic and biotic stress situations.<br>
Biotic and abiotic stress tolerance is a general trait which it is desirable to impart to a broad spectrum of plants such as maize, wheat, rye, oats, triticale, rice, barley, soybean, peanut, cotton, oilseed rape and canola, cassava, pepper, sunflower and tagetes, Solanaceae plants such as potato, tobacco, aubergine and tomato, Vicia species, pea, alfalfa, shrub plants (coffee, cacao, tea), Salix species, trees (oil palm, coconut)<br>
' and perennial grasses and fodder crops. As a further embodiment according to the invention, these crops are also preferred target plants for genetic engineering, very especially preferred plants according to the invention are oil crops such as soybean, peanut, oilseed rape, canola, sunflower, safflower, trees (oil palm,<br>
■ coconut) or crops such as maize, wheat, rye, oats, triticale, rice, barley, alfalfa, or shrub plants (coffee, cacao, tea).<br>
Accordingly, one aspect of the invention relates to isolated nucleic acid molecules (for example cDNAs), encompassing<br>
I nucleotide sequences which encode a PSE or several PSEs or biologically active parts thereof, or nucleic acid fragments which are suitable as primers or hybridisation probes for the detection or amplification of PSE-encoding nucleic acids (for example DNA or mRNA). In a specially preferred embodiment, the<br>
i nucleic acid molecule encompasses one of the nucleotide sequences shown in SEQ ID NO:l, SEQ ID N0:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID NO:11, or the coding region or a complement of<br><br>
f<br>
one of these nucleotide sequences. In other especially preferred embodiments, the isolated nucleic acid molecule according to the invention encompasses a nucleotide sequence which hybridizes with a nucleotide sequence as shown in the sequence SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11, sr a part thereof or which has at least approximately 50%, preferably at least approximately 60%, more preferably at least approximately 70%, 80% or 90% and even more preferably at least approximately 95%, 96%, 97%, 98%, 99 % or more homology thereto. In other preferred embodiments, the isolated nucleic acid molecule encodes one of the amino acid sequences shown in the sequence SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12. Preferably, the preferred PSE gene according to the invention also has at least one of the PSE activities described herein.<br>
In a further embodiment, the isolated nucleic acid molecule encodes a protein or part thereof, the protein or the part thereof comprising an amino acid sequence which has sufficiently homology with an  amino acid sequence of the sequence SEQ ID N0:2, SEQ ID K0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12, that the protein or the part thereof retains a PSE activity. Preferably, the protein or the part thereof which is encoded by the nucleic acid molecule retains the ability to participate in the metabolism of compounds required for the synthesis of cell membranes of plants or in the transport of molecules via these membranes. In one embodiment, the protein encoded by the nucleic acid molecule has at least approximately 50%, preferably at least approxiamtely 60% and more preferably at least approximately 70%, 80% or 90% and most preferably at least approximately 95%, 96%, 97%, 98%, 99% or more homology with an amino acid sequence of the sequence SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:Z0 and SEQ ID N0:12. In a further preferred embodiment, the protein is a full-length protein, parts of which are essentially homologous to a complete amino acid sequence of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID N0:12, (which is due to the open reading frame shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, BtQ  ID N0:7, SEQ ID N0:9 and SEQ ID NO:U) and which can be isolated in its full length by methods and experiments with which the skilled worker is familiar.<br>
In another preferred embodiment, the isolated nucleic acid molecule originates from Phytophthora infestans, Physcomitrella patens, Crypthecodinium cohnii or Thraustochytrium and encodes a protein (for example a PSE fusion protein) comprising a biologically active domain which has at least approximately 50%<br><br>
or more homology with an amino acid sequence of the sequence SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12 and retains the ability to participate in the metabolism of compounds required for the synthesis of cell membranes of plants or in the transport of molecules via these membranes or which has at least one of the elongation activities resulting in PUFAs such as ARA, EPA or DHA or their precursor molecules or one of the activities listed in Table 1, and also encompasses heterologous nucleic acid sequences which encode a heterologous polypeptide or regulatory proteins.<br>
Table 1:    Fatty acid profile of five transgenic yeast strains in mol%. The proportions of 7-linolenic acid which has been added and taken up are emphasized by numbers printed in bold, those of the elongated products are underlined and those of the elongated y-linolenic acid are emphasized by numbers printed in bold (last line).<br><br>
Fatty acids [mol%]	pYES2	pY2PSEla	pY2PSElb	pY2PSElc	pY2PSEld<br>
16:0	17.0	17.6	16.4	16,3	17.6<br>
15:1A9	28.0	26,8	28.0	27.9	25.1<br>
18:0	6.5	6.0	6.4	5.6	6.1<br>
18:iA9	25.9	23.5	27.0	25.2	21,4<br>
18:3^6,9,12	22.6	15.7	13.2	16.4	22.8<br>
20:3A8'11'1^	-	10.3	9.0	8.6	7.1<br>
Elongation	-	39.6	40.5	34.4	23,7<br>
In another embodiment, the isolated nucleic acid molecule is at least 15 nucleotides in length and hybridizes under stringent conditions with a nucleic acid molecule comprising a nucleotide sequence of SEQ ID K0:1, SEQ ID N0:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11. The isolated nucleic acid molecule preferably corresponds to a naturally occurring nucleic acid molecule. More preferably, the isolated nucleic acid molecule encodes naturally occurring Crypthecodinium, Phytophthora or Thraustochytrium PSE or a biologically active part thereof.<br>
A further aspect of the invention relates to vectors, for example recombinant expression vectors, comprising at least one nucleotide molecule according to the invention and host cells into which these vectors have been introduced, in particular J microorganisms, plant cells, plant tissues, plant organs or intact plants, in one embodiment, such a host cell can store fine chemicals, in particular PUFAs; to isolate the desired compound.<br><br>
the cells are harvested. The compound {oiis, iipias, triacylglycerides, fatty acids) or the PSE can then be isolated from the medium or from the host cell which, in the case of plants, are cells comprising or storing the fine chemicals, most preferably cells of storage tissues such as seed coats, tubers, epidermis cells and seed cells.<br>
Yet another aspect of the invention relates to a genetically modified plant, preferably an oil crop as mentioned above, especially preferably a rapeseed, linseed or Physcomitrella patens plant into which a PSE gene has been introduced. In one embodiment, the genome of oilseed rape, linseed or Physcomitrella patens has been modified by introducing, as transgene, a nucleic acid molecule according to the invention encoding a wild-type or mutated PSE sequence. In another embodiment, an endogenous PSE gene in the genome of the donor organisms Physcomitrella patens, Phytophthora infestans, Crypthecodinium or Thraustochytrium has been modified, that is to say functionally destroyed, for example by homologous recombination with a modified PSE gene or by mutagenesis and detection by means of DHA sequences. In a preferred embodiment, the plant organism belongs to the genus Physcomitrella, Ceratodon, Funaria, oilseed rape or linseed, with Physcomitrella, oilseed rape or linseed being preferred, in a preferred embodiment, Physcomitrella, oilseed rape or linseed is also used to produce a desired compound such as lipids or fatty acids, with PUFAs being especially preferred.<br>
In yet another preferred embodiment, the moss Physcomitrella patens can be used for demonstrating a function of an elongase gene using homologous recombination on the basis of the nucleic acids described in the present invention.<br>
Yet another aspect of the invention relates to an isolated PSE gene or a part, for example a biologically active part, thereof. In a preferred embodiment, the isolated PSE or a part thereof can participate in the metabolism of compounds required for the synthesis of cell membranes in a microorganism or a plant cell or in the transport of molecules via its membranes. In a further preferred embodiment, the isolated PSE or the part thereof has sufficient homology with an amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0;8, SEQ ID NO:10 and SEQ ID NO:12 for this protein or the part thereof to retain the ability to participate in the metabolism of compounds required for the synthesis of cell membranes in microorganisms or plant cells or in the transport of molecules via these membranes.<br><br>
The invention also provides an isolated preparation of a PSE. in preferred embodiments, the PSE gene encompasses an amino acid sequence of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID NO:12. In a further preferred embodiment, the invention relates to an isolated full-length protein which is essentially homologous with a complete amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12 (which are encoded by the open reading frames shown in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID K0:11). In a further embodiment, the protein has at least approximately 50%, preferably at least approximately 60%, more preferably at least approximately 70S, 801 or 90%  and most preferably at least approximately 95%, 96%, 97%, 98%, 99% or more homology with an amino acid sequence of sequence SEQ ID H0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12. In other embodiments, the isolated PSE encompasses an amino acid sequence which has at least approximately 50% homology with one of the amino acid sequences of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 and which can participate in the metabolism of compounds required for the synthesis of fatty acids in a microorganism or a plant cell or in the transport of molecules via these membranes or has one or more of the PUFA-elongating activities, the elongation advantageously concerning desaturated Cie- or Cig- or C2o-carbon chains with double bonds in at least two positions.<br>
As an alternative, the isolated PSE can encompass an amino acid sequence which is encoded by a nucleotide sequence hybridizing with a nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID NO: 5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11, for example under stringent conditions, or which has at least approximately 50%, preferably at least approximately 60%, more preferably at least approximately 70%, 80% or 90% and even more preferably at least approimately 95%, 96%, 97%, 98%, 99% or more homology thereto. It is also preferred for the preferred PSE forms also to have one of the PSE activities described herein.<br>
The PSE polypeptide or a biologically active part thereof can be linked functionally to a non-PSE polypeptide to form a fusion protein. In preferred embodiments, this fusion protein has an activity which differs from that of PSE alone. In other preferred embodiments, this fusion protein participates in the metabolism of compounds which are required for the synthesis of lipids and fatty acids, cofactors and enzymes in microorganisms or plants, or in the transport of molecules via these membranes. In especially preferred embodiments, the introduction of this fustion protein into a host cell modulates the production of a<br><br>
desired compound by the cell; In a preferred embodiment, these fusion proteins also contain A4-, A5- or A6-, A8-, A15-, A17- or A19-desaturase activities, alone or in combination.<br>
Another aspect of the invention relates to a process for the production of a fine chemical. This process either comprises culturing a suitable microorganism or culturing plant cells, plant tissues, plant organs or intact plants encompassing the nucleotide sequences according to the invention of SEQ ID N0:1, SEQ ID HO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID K0:11 or their homologs, derivatives or analogs or a gene construct which compasses SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0;9 and SEQ ID N0:11 or their homologs, derivatives or analogs, or a vector encompassing these sequences or the gene construct which brings about the expression of a PSE nucleic acid molecule according to the invention so that a fine chemical is produced. In a preferred embodiment, the process furthermore encompasses the step of obtaining a cell comprising such an elongase nucleic acid sequence according to the invention, in which a cell is transformed with an elongase nucleic acid sequence, a gene construct or a vector which bring about the expression of a PSE nucleic acid according to the invention. In a further preferred embodiment, this process furthermore comprises the step of obtaining the fine chemical from the culture. In an especially preferred embodiment, the cell belongs to the order of the Ciliata, to microorganisms such as fungi, or to the plant kingdom, in particular to oil crops, with microorganisms or oil  crops  being especially preferred.<br>
A further aspect of the invention relates to methods of modulating the production of a molecule by a microorganism. These methods encompass combining the cell with a substance which modulates the PSE activity or the expression of the PSE nucleic acid so that a cell-associated activity is modified relative to the same activity in the absence of the substance. In a preferred embodiment, a metabolic pathway, or two metabolic pathways, of the cell for lipids and fatty acids, cofactors and enzymes is, or are, modulated or the transport of compounds via these membranes is modulated so that the yield or the production rate of a desired fine chemical by this microorganism is improved. The substance which modulates the PSE activity can be a substance which stimulates the PSE activity or the expression of the PSE nucleic acid or which can be used as intermediate in fatty acid biosynthesis. Examples of substances which stimulate the PSE activity or the expression of PSE nucleic acids are, inter alia, small molecules, active PSEs and nucleic acids encoding PSEs which have been introduced into the cell. Examples of substances<br><br>
which inhibit the PSE activity or PSE expression are,   inter alia, small molecules and/or antisense PSE nucleic acid moldeules.<br>
A further aspect of the invention relates to methods of modulating the yields of a desired compound from a cell, which encompass introducing, into a cell, a wild-type or mutant PSE gene which is either kept on a separate plasmid or integrated into the genome of the host cell. In the case of integration into the genome, integration can be random or take place by recombination in such a way that the native gene is replaced by the copy which is introduced, thus modulating the production of the desired compound by the cell, or by using a gene intron, so that the gene is functionally linked to a functional expression unit comprising at least one sequence which ensures the expression of a gene and at least one sequence which ensures the polyadenylation of a functionally transcribed gene.<br>
In a preferred embodiment, the yields are modified. In a further embodiment, the desired chemical is increased, it being possible to reduce undesired compounds which have a negative effect. In an especially preferred embodiment, the desired fine chemical is a lipid or fatty acid, a cofactor or an enzyme, in an especially preferred embodiment, this chemical is a polyunsaturated fatty acid. More preferably, it is selected from amongst arachidonic acid (= ARA), eicosapentaenoic acid (= EPA) or docosahexaenoic acid (= DHA).<br>
Detailed description of the invention<br>
The present invention provides PSE nucleic acids and PSE protein molecules which participate in the metabolism of lipids and fatty acids, PUFA cofactors and enzymes in the moss Physcomitrella patens, Phytophthora infestans. Crypthecodinium or Traustochytrium or in the transport of lipophilic compounds via membranes. The compounds according to the invention can be used for modulating the production of fine chemicals from organisms, for example microorganisms, such as ciliates, fungi, yeasts, bacteria, algae, and/or plants such as maize, wheat, rye, oats, triticale, rice, barley, soybean, peanut, cotton, Brassica<br>
I species, such as oilseed rape, canola and turnip rape, pepper, sunflower, borage, evening primrose and tagetes, Solanaceae plants such as potato, tobacco, aubergine and tomato, Vicia species, pea, cassava, alfalfa, shrub plants (coffee, cacao, tea), Salix species, trees (oil palm, coconut) and perennial<br>
i  grasses and fodder crops, either directly (for exEimple when the overexpression or optimization of a fatty acid biosynthesis protein has a direct effect on the yield, production and/or<br><br>
production efficiency of the fatty acid from modified organisms) or they can have an indirect effect which nevertheless leads to an increased yield, production and/or production efficiency of a desired compound or to a decrease in undesired compounds (for example when the modulation of the lipid and fatty acid, cofactor and enzyme metabolism leads to changes in yield, production and/or production efficacy or in the composition of the desired compounds within the cells, which, in turn, may affect the production of one or more fine chemicals). Aspects of the invention are illustrated in greater detail hereinbelow.<br>
I.  Fine chemicals and PUFAs<br>
The term "fine chemicals" is known in the art and encomp&amp;sses molecules which have been produced by an organism and which are used in a variety of industries such as, by way of example but not by way of limitation, the pharmaceuticals industry, agroindustry, food industry and cosmetics industry. These compounds encompass lipids, fatty acids, cofactors and etizymes and the like (as described, for example, in Kuninaka, A. (1996) Nucleotides and related compounds, pp. 561-612, in Biotechnology Vol. 6, Rehm et al., Ed.: VCH Weinheim and references cited therein), lipids, saturated and unsaturated fatty acids [for example arachidonic acid), vitamins and cofactors (as described in Ullmann's Encyclopedia of Industrial Chemistry, Vol. A27, Vitamins, pp. 443-613 (1996): VCH Weinheim, and references cited therein; and Ong, A.S., Niki, E., &amp; Packer, L. (1995) Nutrition, Lipids, Health and Disease Proceedings of the<br>
UNESCO/Confederation of Scientific and Technological Associations in Malaysia and the Society for Free Radical Research - Asia, held September 1-3,  1994 in Penang, Malaysia, AOCS Press (1995)), enzymes and all other chemicals described by Gutcho (1983) in Chemicals by Fermentation, Noyes Data Corporation, ISBN: 0818805086 and references cited therein. The metabolism and the uses of certain fine chemicals are illustrated in greater detail hereinbelow.<br>
The combination of various precursor molecules and biosynthetic enzymes leads to the production of various fatty acid molecules, which has a decisive effect on membrane composition. It can be assumed that PUFAs are not only just incorporated into triacylglycerol, but also into membrane lipids.<br>
Membrane synthesis is a well characterized process in which a number of components, inclusive of lipids as part of the bilayer membrane, are involved. The production of novel fatty acids such<br><br>
as PUFAs can therefore gener^e novel properties of membrane functions within a cell or an organism.<br>
Cell membranes serve a multiplicity of functions in a cell. First and foremost, a membrane delimits the contents of a cell from the environment, thus imparting integrity to the cell. Membranes can also act as barriers against the influx of dangerous or undesired compounds or else against the efflux of desired compounds.<br>
For more detailed descriptions of involvements of membranes and the mechanisms involved, see Bamberg, E., et al, (1993) Charge transport of ion pumps on lipid biiayer membranes, Q. Rev. Biophys. 26:1-25; Gennis, R.B. (1989) Pores, Channels and Transporters, in: Biomembranes, Molecular Structure and Function, Springer: Heidelberg, pp. 270-322; and Nikaido, H., und Saier, H. (1992) Transport proteins in bacteria: common themes in their design, Science  258:936-942, and the citations contained in each of these references.<br>
Lipid synthesis can be divided into two parts: the synthesis of fatty acids and their binding to sn-glycerol-3-phosphate, and the addition or modification of a polar head group. Customary lipids used in membranes encompass phospholipids, glycolipids, sphlngolipids and phosphoglycerides. Fatty acid synthesis starts with the conversion of acetyl-CoA either into malonyl-CoA by acetyi-CoA carboxylase or into acetyl-ACP by acetyl transacylase. After a condensation reaction, these two product molecules together form acetoacetyl-ACP, which is converted via a series of condensation, reduction and dehydration reactions to give a saturated fatty acid molecule with the desired chain length. The production of the unsaturated fatty acids from these molecules is catalyzed by specific desaturases, either aerobically by means of molecular oxygen or anaerobically (as regards fatty acid synthesis in microorganisms, see F.C. Neidhardt et al. (1996) E. coli and Salmonella. ASM Press: Washington, D.C., pp. 612-636 and references contained therein; Lengeler et al. (Ed.) (1999) Biology of Procaryotes. rhieme: Stuttgart, New York, and the references contained therein, and Magnuson, K., et al. (1993) Microbiological Reviews 57:522-542 and the references contained therein).<br>
Examples of precursors for PUFA biosynthesis are linolic and Linolenic acid. These Cig-carbon fatty acids must be elongated to :2o or C22 to give fatty acids of the eicosa and docosa chain bype. Various desaturases such as enzymes which have ^6-desatura5e, A5- and A4-desaturase activity can lead to arachidonic acid, eicosapentaenoic acid and docosahexaenoic acid<br><br>
and various other long-chain PUFAs which can be extracted and used for variovis purposes in food and feed, cosmetic or pharmaceutical applications.<br>
To produce long-chain PUFAs, the polyunsaturated Ci6-or Cig- or C2o-fatty acids must, as mentioned above, be elongated by at least two carbon atoms by the enzymatic activity of an elongase. The nucleic acid sequences according to the invention encode first microbial elongases from typical producers containing PUFA in the triacylglycerol fraction, which elongases are capable of elongating the Cie- or Cig- or  C2o-fatty acids with at least two double bonds in the fatty acid by at least two carbon atoms or which convert these, for example sequentially in succession, by converting a Cie- or Cig-fatty acid into a C2o-fatty acid and then a C20- into a C22- or higher even-numbered fatty acid containing units with 2 C atoms. After one elongation cycle, this enzyme activity leads to C2o-fatty acids, and after two, three and four elgonation cycles to C22-, C24- or C26-fatty acids. Longer PUFAs can also be synthesized with the elongase according to the ivnention. The activity of the elongases according to the invention preferably leads to C20- and/or C22-fatty acids with at least two double bonds in the fatty acid molecule, C2o-fatty acids, preferably with three, four or five double bonds, especially preferably three double bonds, in the fatty acid molecule, C22-fatty acids, preferably with three, four, five or six double bonds, especially preferably five or six double bonds, in the fatty acid molecule. After elongation with the enzyme according to the invention, further desaturation steps may be carried out. Thus, the products of the elongase activities and of the further desaturation which is possible lead to preferred PUFAs with a higher degree of desaturation, such as docosadienoic acid, arachidonic acid, oje-eicosatrienedihomo-v-linolenic acid, eicosapentaenoic acid, mS-eicosatrienoic acid, «i3-eicosatetraenQic acid, docosapentaenoic acid or docosahexaenoic acid. Examples of substrates of this enzyme activity according to the invention are taxol acid, 7,10,13-hexadecatrienoic acid, 6,9-octadecadienoic acid, linolic acid, Y-linolenic acid, linolenic acid, a-linolenic acid, arachidonic acid, eicosapentaenoic acid or stearidonic acid. Preferred substrates are linolic acid, 7-linolenic acid and/or cc-linolenic acid or arachidonic acid, eicosatetraenoic acid or eicosapentaenoic acid. The Cig- or Cig- or C2o-fatty acids with at least two double bonds in the fatty acid can be elongated by the enzyme activity according to the invention in the form Qf the free fatty acid or in the form of the esters, such as<br><br>
phospholipids, glycolipids, ephingolipids, phosphoglycerides, monoacylglycerol, diacylglycerol or triacylglycerol.<br>
Furthermore, fatty acids must subsequently be transported to various locations and incorporated into the triacylglycerol storage lipid. Another important step in lipid synthesis is the transfer of fatty acids to the polar head groups, for example by glycerol fatty acid acyltransferase (see Frentzen, 1998, Lipid, 100(4-5):161-166).<br>
For publications on plant fatty acid biosynthesis, desaturation, lipid metabolism and the membrane transport of fatty compounds, beta-oxidation, fatty acid modification and cofactors, triacylglycerol storage and assembly inclusive of the references cited therein, see the following articles: Kinney, 1997, Genetic Engineering, Ed.: JK Setlow, 19:149-166; Ohlrogge and Browse, 1995, Plant Cell 7:957-970; Shanklin and Cahoon, 1998, Annu. Rev. Plant Physiol. Plant Mol. Biol. 49:611-641; Voelker, 1996, Genetic Engineering, Ed.: JK Setlow, 18:111-13; Gerhardt, 1992, Prog. Lipid R. 31:397-417; Giihnemann-Schafer &amp; Kindl, 1995, Biochira. Biophys Acta 1256:181-186; Kunau et al., 1995, Prog. Lipid Res. 34:267-342; Stymne et al., 1993, in: Biochemistry and Molecular Biology of Membrane and Storage Lipids of Plants, Ed.: Murata and Somerville, Rockville, American Society of Plant Physiologists, 150-158, Murphy &amp; Ross 1998, Plant Journal-13(1):1-16.<br>
Vitamins, cofactors and "nutraceuticals", such as PUFAs, encompass a group of molecules which higher animals can no longer synthesize and therefore have to take up, or which higher animals can no longer synthesize themselves to a sufficient degree and must therefore take up additionally, even though they are readily synthesized by other organisms such &amp;s bacteria. The biosynthesis of these molecules in organisms which are capable of producing them, such as in bacteria, has been more or less characterized (Ullmann's Encyclopedia of industrial Chemistry, "Vitamins", Vol. A27, pp. 443-613, VCH Weinheim, 1996; Hichal, G. (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley &amp; Sons; Ong, A.S., Niki, E., &amp; Packer, L. (1995) "Nutrition, Lipids, Health and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia and the Society for Free Radical Research Asia, held September 1-3, 1994, in Penang, Malaysia, AOCS Press, Champaign, IL X, 374 pp).<br><br>
The abovementioned molecules are either biologically active molecules themselves or precursors of biologically active substances which act either as electron carriers or as intermediates in a multiplicity of metabolic pathways. Besides their nutritional value, these compounds also have a significant industrial value as colorants, antioxidants and catalysts or other processing auxiliaries. (For a review over structure, activity and industrial applications of these compounds, see, for example, Ullmann's Encyclopedia of Industrial Chemistry, "Vitamins", Vol. A27, pp. 443-613, VCH Weinheim, 1996). Polyunsaturated fatty acids have a variety of functions and health-promoting effects, for example in the case of coronary heart disease, inflammatory mechanisms, children's nutrition and the like. For publications and references including the references cited therein, see: Simopoulos, 1999, Am, J. Clin. Nutr. 70 (3rd Suppl.):560-569, Takahata et al., Biosc. Biotechnol. Biochem. 1998, 62(11) :2079-2085, Willich and Winther, 1995, Deutsche Hedizinische wochenschrift 120(7):229 et seq.<br>
II. Elements and processes of the invention<br>
The present invention is based at least in part on the discovery Df novel molecules termed herein PSE nucleic acid and PSE protein nolecules, which exert an effect on the production of ceil nembranes in Physcomitrella patens. Crypthecodinium cohnii, Phytophthora infestans, Thraustochytrium and/or Ceratodon Durpureus and, for example, have an effect on the movement of nolecules via these membranes. In one embodiment, the PSE nolecules participate in the metabolism of compounds required for the synthesis of cell membranes in organisms such as microorganisms and plants or indirectly affect the transport of molecules via these membranes. In a preferred embodiment, the activity of the PSE molecules according to the invention for regulating the production of membrane components and membrane transport has an effect on the production of the desired fine chemical by this organism. In an especially preferred embodiment, the activity of the PSE molecules according to the invention is modulated so that the yield, production and/or production efficiency of the metabolic pathways of microorganisms or plants which regulate the PSSs according to the invention are modulated and the transport efficiency of compounds through the membranes is modified, which either directly or indirectly modulates the yield, production and/or production efficiency of a desired fine chemical by microorganisms and plants.<br><br>
The term PSE or PSE polypeptide encompasses proteins which participate in the metabolism of compounds required for the synthesis of cell membranes in organisms such as microorganisms and plants or in the transport of molecules via these membranes. Examples of PSEs are disclosed in SEQ ID N0:1, SEQ ID K0:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 or their homologs, derivatives or analogs. The terms PSE or PSE nucleic acid sequence(s) encompass nucleic acid sequences which encode a PSE and part of which can be a coding region and also corresponding 5'- and 3'-untranslated sequence regions. Examples of PSE genes are the sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID K0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11. The terms production and productivity are known in the art and encompass the concentration of the fermentation product (for example of the desired fine chemical) which is formed within a certain period and in a certain fermentation volume (for example kg product per hour per liter). The term production efficiency encompasses the time required for achieving a particular product quantity (for exaimple the time required by the cell to establish a particular throughput rate of a fine chemical). The term yield or product/carbon yield is known in the art and encompasses the efficiency with which the carbon source is converted into the product (i.e. the fine chemical). This is usually expressed as, for example, kg product per kg carbon source. Increasing the yield or production of the compound increases the amount of the ctiolecules obtained or of the suitable molecules of this compound obtained in a specific quantity of culture over a defined period. The terms biosynthesis or biosynthetic pathway are known in the art and encompass the synthesis of a compound, preferably of an organic compound, by a cell from intermediates, for example in a multi-step process which is subject to strong regulation. The terms catabolism or catabolic pathway are known in the art and encompass the cleavage of a compound, preferably of an organic compound, by a cell into catabolites (in general smaller or less complex molecules), for example in a multi-step process which is subject to strong regulation. The term metabolism is )tnown in the art and encompasses the totality of the biochemical reactions tfhich take place in an organism. The metabolism of a certain ::ompo\ind (for example the metabolism of a fatty acid) thus ancompasses the totality of the biosynthetic, modification and :atabolic pathways of this compound in the cell which are relevant to this compound.<br>
[n another embodiment, the PSE molecules according to the Invention can modulate the production of a desired molecule, such is a fine chemical, in a microorganism or in plants. There exists i series of mechanisms by which the modification of a PSE<br><br>
according to the invention can directly affect the yield, production and/or production efficiency of a fine chemical from a microorganism strain or plant strain comprising this modified protein. The number or activity of PSEs participating in the transport of molecules of fine chemicals within, or out of, the cell can be increased, so that greater amounts of these compounds are transported via membranes, from which they can be obtained and converted into each other with greater ease. Furthermore, fatty acids, triacylglycerols and/or lipids are desirable fine chemicals themselves; optimizing the activity or increasing the number of one or more PSEs according to the invention which participate in the biosynthesis of these compounds, or by interfering with the activity of one or more PSEs which participate in the catabolism of these compounds makes increasing the yield, production and/or production efficiency of fatty acid molecules and lipid molecules from organisms such as microorganisms or plants, possible.<br>
The mutagenesis of the PSE gene according to the invention can also give rise to PSEs with modified activities which indirectly affect the production of one or more desired fine chemicals from microorganisms or plants. For example, PSEs according to the invention which participate in the export of waste products can exhibit a greater number or higher activity, so that the normal metabolic waste products of the cell (whose quantity might be increased owing to the overproduction of the desired fine chemical) are exported efficiently before they can damage the molecules in the cell (which would reduce the cell's viability) or interfere with the biosynthetic pathways of the fine chemicals (which would reduce the yield, production or production efficiency of a desired fine chemical). The relatively large intracellular amounts of the desired fine chemical themselves can furthermore be toxic to the cell, so that increasing the activity or number of transporters capable of exporting these compounds from the cell results in an increased viability of the cell in culture, which, in turn, leads to a higher number of cells in the culture which produce the desired fine chemical. The PSEs according to the invention can also be manipulated in such a way that the corresponding amounts of different lipid molecules and fatty acid molecules are produced. This can have a substantial effect on the lipid composition of the cell membrane. Since each lipid type has different physical properties, a modification of the lipid composition of a membrane can significantly modify membrane fluidity- Modifications of the membrane fluidity can affect the transport of molecules via the membrane and cell integrity, each of which has a substantial effect on the production of fine chemicals from microorganisms and plants in<br><br>
large-scale fermentation culture. Plant membranes impart specific properties such as tolerance to high and low temperatures, salt, drought and tolerance with respect to pathogens such as bacteria and fungi. The modulation of the membrane components may therefore have a critical effect on the ability of the plants to survive under the aboveraentioned stress parameters. This can take place via changes in signal cascades or directly via the modified membrane composition (see, for example. Chapman, 1998, Trends in Plant Science, 3(11):419-426) and signal cascades (see Wang 1999, Plant Physiology, 120:645-651) or affect the tolerance of low temperatures, as disclosed in WO 95/18222.<br>
The isolated nucleic acid sequences according to the invention are present, for example, in the genome of a Thraustochytrium strain which is available via the American Type Culture Collection (ATCC) with the strain number ATCC26185 (Thraustochytrium), or, in the case of Crypthecodinium, for example, accessible via the Provasoli-Guillard National Center for Culture of Marine Phytoplankton ((CCMP) West Boothbay Harbour, ME, USA) with the strain culture No. CCMP316. In the case of Phytophthora infestans, the stated nucleic acid molecules are isolated from the strain ATCC 48886.<br>
The nucleotide sequence of the isolated Physcomitrella, Crypthecodinium, Phytophthora infestans or Thraustochytrium cDHA and the deduced amino acid sequences of the Physcomitrella patens PSEs are shown in SEQ ID N0:1 to SEQ ID N0:12. Computer analyses were carried out which classify and/or identify these nucleotide sequences as sequences which encode proteins participating in the metabolism of cell membrane components or which participate in the transport of compounds via cell membranes, or of PUFA biosynthesis. ESTs with the database input Ko. PP001019019F, :C001042041R, PI001002014R, TC002034029R, TC002034029R-11 and rC002014093R in the database of the inventor constitute the sequences according to the invention in SEQ ID N0:1, SEQ ID NO:3, 3EQ ID NO:5, SEQ ID NO:?, SEQ ID NO:9 and SEQ ID K0:11. In a similar manner, the partial polypeptides were termed ?P001019019F, CC001042041R, PI001002014R, TC002034029R, DC002034029R-11 and TC002014093R and constitute the sequences according to the invention in SEQ ID N0:2, SEQ ID N0:4, SEQ ID ^0:6, SEQ ID N0:8, SEQ ID NOilO and SEQ ID N0:12 in. accordance ^ith Table 2. The complete fragment insert of the ESTs CC002034029R was sequenced and resulted in SEQ ID N0:3, which is :he complete sequence of TC002034029R. TC002034029R-11 describes i full-length sequence of an elongase from Thraustochytrium. The laming of the remaining clones is similar. Also, corresponding jene names were assigned to the various clones. Abbreviations: Tc<br><br>
= Thraustochytrium,   Cc = Crypthecodinium,   Pp =  Physcomitrella patens,   P:   Phytophthora  infestans.<br>
Table  2<br><br>
Name/EST name	Gene name	Polypeptide<br>
SEQ ID NO	Nucleic acid SEQ ID NO<br>
PP0Q1019019F	Pp_PSEl	2	1<br>
TC002034029R	Tc PSEl	4	3<br>
TC002014093R	Tc PSE2	6	.5<br>
CC001042041R	Cc PSEl	8	7<br>
TC002034029R-11	Tc PSEl 1	10	9<br>
PI001002014R	Pi PSEl	12	11<br>
The present invention also relates to proteins with an eimino acid sequence which is essentially homologous with an amino acid sequence of SEQ ID K0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID NO:12. As used in the present context, a protein with an amino acid sequence which is essentially homologous with a selected amino acid sequence has at least approximately 50% homology with the selected amino acid sequence, for example the complete amino acid sequence selected. A protein with an amino acid sequence which is essentially homologous with a selected amino acid sequence can also have at least approximately 50 to 60%, preferably at least approximately 60 to 70%, more preferably at least approximately 70 to 80%, 80 to 90% or 90 to 95%, and most preferably at least approximately 96%, 97%, 98%, 99% or more homology with a selected amino acid sequence.<br>
The PSE according to the invention or the biologically active part or the fragment thereof can participate in the metabolism of compounds required for the synthesis of cell membranes in nicroorganisms or plants or in the transport of molecules via these membranes or have one or more of the activities required for the elongation of Cie- or Cia- or C20-PUFAS, so that C20-. C22-or C24-PUFAS and related PUFAs are obtained.<br>
Various aspects of the invention are described in greater detail in the subsections which follow.<br>
A.  Isolated nucleic acid molecules<br>
One embodiment of the invention comprises isolated nucleic acids derived from pUFA-producing microorganisms and encoding polypeptides which elongate Cie- or Cie-fatty acids with at least two double bonds in the fatty acid by at least two carbon atoms<br><br>
or which elongate C2o-fatty acids with at least two double bonds in the fatty acid by at least two carbon atoms.<br>
A further embodiment according to the invention comprises isolated nucleic acids emcompassing nucleotide sequences encoding polypeptides which elongate Cie-, Cig- or C2o-fatty acids with at least two double bonds in the fatty acid and which are selected from the group consisting of<br>
a)	the nucleic acid sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID K0:9 and SEQ ID N0:11,<br>
b)	a nucleic acid sequence which, in accordance with the degeneracy of the genetic code, is derived from one of the sequences shown in  SEQ ID N0:1, SEQ ID N0:3, SEQ ID H0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 or<br>
c)	derivatives of the sequence shown in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 which encode polypeptides of the amino acid sequence shown in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12 and which have at least 50% homology at the amino acid level without the enzymatic action of the polypeptides being substantially reduced.<br>
The abovementioned nucleic acids according to the invention, which act as Cig-, C^g- or C20-elongase, are derived from organisms such as ciliates, fungi, algae, plants or dinofl age Hates which are capable of synthesizing PUFAs, preferably from plants or algae, especially preferably from the genus Phytophthora, Physcomitrella, Crypthecodinium, Thraustochytrium or Schizochytrium, most preferably from phytophthora infestans, Physcomitrella patens, Crypthecodinium cohnii or Thraustochytrium sp., Schizochytrium sp. or closely related organisms.<br>
One aspect of the invention relates to isolated nucleic acid molecules which encode PSE polypeptides or biologically active parts thereof, and to nucleic acid fragments which suffice for use as hybridization probes or primers for identifying or amplifying a PSE-encoding nucleic acid (for example PSE DNA). The term "nucleic acid molecule" as used in the present context is intended to encompass DNA molecules (for example cDNA or genomic DNA) and RNA molecules (for example mRNA) and DNA or RNA analogs which are generated by means of nucleotide analogs. This term additionally encompasses the untranslated sequence at the 3' and the 5' end of the coding gene region: at least approximately 100 nucleotides of the sequence upstream of the 5' end of the coding<br><br>
region and at least approximtn.eiy ^\j  imuj-eotiaes or tne sequence downstream of the 3' end of the coding gene region. The nucleic acid molecule can be single- or double-stranded, but is preferably double-stranded DNA. An "isolated" nucleic acid molecule is separated from other nucleic acid molecules which are present in the natural source of the nucleic acid. An "isolated" nucleic acid preferably has no sequences which naturally flank the nucleic acid in the genomic DNA of the organism from which the nucleic acid is derived {for example sequences located at the 5' and 3' ends of the nucleic acid). In various embodiments, the isolated PSE nucleic acid molecule can contain, for example, less than approximately 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5 kb or 0.1 kb nucleotide sequences which naturally flank the nucleic acid molecule in the genomic DNA of the cell from which the nucleic acid is derived (for example a Physcomitrella patens cell). An "isolated" nucleic acid molecule, such as a cDNA molecule, can moreover be essentially free from other cellular material or culture medium if it is generated by recombinant techniques, or free from chemical precursors or other chemicals if it is synthesized chemically.<br>
A nucleic acid molecule according to the invention, for example a nucleic acid molecule with a nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 or a part thereof, can be isolated using standard techniques of molecular biology and the sequence information provided herein. Also, for example a homologous sequence or homologous, conserved sequence regions at the DNA or amino acid level can be identified with the aid of alignment algorithms. For example, a Phytophthora, Physcomitrella, Crypthecodinium or Thraustochytrium cDNA can be isolated from a Phytophthora, Physcomitrella, Crypthecodinium or Thraustochytrium library by using the complete SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and/or SEQ ID N0:11 or a part thereof as hybridization probe and standard hybridization techniques (such as, for example, as described in Sambrook et al.. Molecular Cloning: A Laboratory Manual. 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989). Moreover, a nucleic acid molecule encompassing a complete sequence of SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID NO:il or a part thereof can be isolated by polymerase chain reaction, where oligonucleotide primers which are generated on the basis of this sequence or parts thereof, in particular regions around His-box motifs of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0;8, SEQ ID NO: 10 and SEQ ID NO: 12 or modifications of the scime in individual, defined amino acids are used (for example, a nucleic acid molecule<br><br>
encompassing the complete sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 or a part thereof can be isolated by polymerase chain reaction using oligonucleotide primers which have been generated on the basis of this same sequence of SEQ ID N0;1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID K0:11). Furthermore, especially suitable for this purpose are those partial sequences as they are shown in figure 10. For example, mRNA can be isolated from cells (for example by the guanidinium thiocyanate extraction method by 'chirgwin et al. (1979) Biochemistry 18:5294-5299), and cPNA can be generated by means of reverse transcriptase (for example Moloney MLV reverse transcriptase, available from Gibco/BRL, Bethesda, MD, or AMV reverse transcriptase, availabe from Seikagaku America, Inc., St. Petersburg, FL). Synthetic oligonucleotide primers for amplification by means of polymerase chain reaction can be generated on the basis of one of the nucleotide sequences shown in SEQ ID N0:1, 3, 5, 7, 9 or 11 or with the aid of the amino acid sequences shown in figure 10. A nucleic acid according to the invention can be amplified using cDNA or, alternatively, using genomic DNA as template and suitable oligonucleotide primers, in accordance with standard PCR amplification techniques. The nucleic acid thus amplified can be cloned into a suitable vector and characterized by means of DNA sequence analysis. Oligonucleotides which correspond to a PSE nucleotide sequence can be generated by standard synthesis methods, for example with an automatic DNA synthesizer.<br>
The cDNA shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO-.5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 encompasses sequences which encode PSEs (i.e. the "coding region") and also 5'-untranslated sequences and 3'-untranslated sequences. Alternatively, the nucleic acid molecule can only encompass the coding region of one of the sequences in SEQ ID N0:1, SEQ ID NO: 3, SEQ ID HO-.i,   SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO;11 or can comprise complete genomic fragments isolated from genomic DNA.<br>
SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID N0:12 are identified by the same EST input number code as SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID NO:11 for ease of correlation.<br>
In a further preferred embodiment, an isolated nucleic acid molecule according to the invention encompasses a nucleic acid molecule which is a complement of one of the nucleotide sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID N0:9 and SEQ ID NO:11 or a part thereof. A nucleic acid molecule which is complementary to one of the nucleotide<br><br>
sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11 is sufficiently complementary if it is capable of hybridizing with one of the sequences stated in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11, giving rise to a stable duplex.<br>
Homologs of the new elongase nucleic acid sequences with the sequence SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 means, for example, allelic variants with at least approximately 50 to 60%, preferably at least approximately 60 to 70%, more preferably at least approximately 70 to 80%, 80 to 90% or 90 to 95%, and even more preferably at least approximately 95%, 96%, 97%, 98%, 99% or more homology with one of the nucleotide sequences shown in SEQ ID N0:1, SEQ ID HO-.3, SEQ ID H0:5, SEQ ID H0:7, SEQ ID K0:9 and SEQ ID N0:11 or their homologs, derivatives or analogs or parts thereof. In a further preferred embodiment, an isolated nucleic acid molecule according to the invention encompasses a nucleotide sequence which hybridizes with one of the nucleotide sequences shown in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID NO:H or a part thereof, under stringent conditions. Allelic variants encompass, in particular, functional variants which can be obtained by the deletion, insertion or substitution of nucleotides from/into the sequence shown in SEQ ID N0:1, SEQ ID N0:3, SEQ ID K0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11, it being intended, however, for the enzyme activity of the resulting proteins which are synthesized to be advantageously retained for the insertion of one or more genes. Proteins which retain the enzymatic activity of elongase means proteins with at least 10%, preferably 20%, especially preferably 30%, very especially preferably 40% of the original enzyme activity compared with the protein encoded by SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID K0:12. Elongases which retain the abovementioned activities are elongases whose enzymatic activity is not substantially reduced.<br>
Homologs of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID NO: 9 and SEQ ID NO: 11 also means, for example, bacterial, fungal and plant homologs, truncated sequences, single-stranded DNA or RNA of the coding and noncoding DNA sequence.<br>
Homologs of SEQ ID N0:1, SEQ ID NO: 3, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 9 and SEQ ID NO: 11 also means derivatives such as, for Bxample, promoter variants. The promoters upstream of the nucleotide sequences stated can be modified by one or more nucleotide substitutions, by insertion(s) and/or deletion(s), without, however, interfering with the functionality or activity<br><br>
of the promoters. It is furthermore possible for the activity of the promoters to be increased by modification of their sequence or for them to be replaced completely by more active promoters, even from heterologous organisms.<br>
Moreover, the nucleic acid molecule according to the invention can only encompass part of the coding region of one of the sequences in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11, for example a fragment which can be used as probe or primer or a fragment which encodes a biologically active segment of a PSE. The nucleotide sequences determined from cloning the PSE gene of Physcomitrella patens, Phytophthora infestans, Thraustochytrium and Crypthecodinium allow the generation of probes and primers which are designed for identifying and/or cloning PSE homologs in other cell types and organisms and PSE homologs from other mosses or related species. The probe/primer normally encompasses essentially purified oligonucleotide. The oligonucleotide normally encompasses a nucleotide sequence region which hybridizes under stringent conditions with at least approximately 12, preferably approximately 16, more preferably approximately 25, 40, 50 or 75 successive nucleotides of a sense strand of one of the sequences stated in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID NO:11, of an antisense strand of one of the sequences stated in SEQ ID N0;1, SEQ ID N0:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID K0:11 or its homologs, derivatives and analogs or naturally occurring mutants thereof. Primers based on a nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID H0:11 can be used in PCR reactions for cloning PSE homologs. Probes based on the PSE nucleotide sequences can be used for detecting transcripts or genomic sequences which encode the same or homologous proteins. In preferred embodiments, the probe additionally encompasses a labeling group bound thereto, for example a radioisotope, a fluorescent compound, an enzyme or an enzyme cofactor. These probes can be used as part of a test kit for genomic markers for identifying cells which misexpress a PSE, for example by measuring an amount of a PSE-encoding nucleic acid in a cell sample, for example measuring the PSE mRNA level, or for determining whether a genomic PSE gene is mutated or deleted.<br>
In one embodiment, the nucleic acid molecule according to the invention encodes a protein or a part thereof which encompasses an amino acid sequence which has sufficient homology with an amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 for the protein or the part thereof to retain the ability to participate in the<br><br>
metabolism of compounds required for the synthesis of cell membranes in microorganisms or plants or in the transport of molecules via these membranes. As used in the present context, the term "sufficient homology" relates to proteins or parts thereof whose amino acid sequences have a minimum number of amino acid residues {for example an amino acid residue with a similar side chain, such as an amino acid residue in one of the sequences of SEQ ID N0:2 to 12) which are identical with or equivalent to an amino acid sequence of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID N0:10 and SEQ ID N0:12 so that the protein or the part thereof can participate in the metabolism of compounds required for the synthesis of cell membranes in microorganisms or plants or in the transport of molecules via these membranes. As described herein, protein components of these metabolic pathways for membrane components or membrane transport systems can play a role in the production and secretion of one or more fine chemicals. Examples of these activities are also described herein. Thus, the "function of a PSE" contributes either directly or indirectly to the yield, production and/or production efficiency of one or more fine chemicals. Examples of PSE substrate specificities of the catalytic activity are stated in Table :.<br>
In a further embodiment, derivatives of the nucleic acid molecule according to the invention encode proteins with at least approximately 50 to 60%, preferably at least approximately 60 to 70% and more preferably at least approximately 70 to 80%, 80 to 90%, 90 to 95% and most preferably at least approximately 96%, 97%, 98%, 99% or more homology with a complete amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO: 10 and SEQ ID NO: 12. The homology of the amino acid sequence was determined over the entire sequence region using the program Pileup (J. Mol. Evolution., 25, 351-360, 1987, Higgins et al., CABIOS, 5, 1989:151-153) or BESTFIT or GAP (Henikoff, S. and Henikoff, J. G. (1992). Amino acid substitution matrices from protein blocks. Proc. Natl. Acad. Sci. USA 89: 10915-10919.)<br>
Parts of proteins encoded by the PSE nucleic acid molecules according to the invention are preferably biologically active parts of one of the PSEs. As used herein, the term "biologically active part of a PSE" is intended to encompass a segment, for example a domain/motif, of a PSE which can participate in the metabolism of compounds required for the synthesis of cell membranes in microorganisms or plants or in the transport of molecules via these membranes or which has an activity stated in Table 1. An assay of the enzymatic activity can be carried out in order to determine whether a PSE or a biologically active part<br><br>
thereof can participate in the  metabolism of compounds required for the synthesis of cell membranes in microorganisms or plants or in the transport of molecules via these membranes. These assay methods as described in detail in Example 8 of the examples section are known to the skilled worker.<br>
Additional nucleic acid fragments which encode biologically active segments of a PSE can be generated by isolating part of one of these sequences in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:l2, expressing the encoded segment of the PSE or of the peptide (for example by recombinant expression in vitro) and determining the activity of the encoded part of the PSE or of the peptide.<br>
Moreover, the invention encompasses nucleic acid molecules which differ from one of the nucleotide sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 (and parts thereof) owing to the degeneracy of the genetic code and which thus encode the same PSE as the one encoded by the nucleotide sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11. In another embodiment, an isolated nucleic acid molecule according to the invention has a nucleotide sequence which encodes a protein with an amino acid sequence shown in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12. In a further embodiment, the nucleic acid molecule according to the invention encodes a full-length PSE protein which is essentially homologous with an amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 (which is encoded by an open reading frame shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID NO:11) and which can be identified and isolated by customary methods.<br>
In addition to the PSE nucleotide sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID N0:11, the skilled worker recognizes that DNA sequence polymorphisms may exist which lead to changes in the amino acid sequences of the PSEs within a population {for example the Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium population). These genetic polymorphisms in the PSE gene can exist between individuals within a population owing to natural variation. As used in the present context, the terms "gene" and "recombinant gene" refer to nucleic acid molecules with an open reading frame which encodes a PSE, preferably a Phytophthora, Physcomitrella, Crypthecodinium or Thraustochytrium PSE. These natural variants usually cause a variance of 1 to 5% in the<br><br>
nucleotide sequence of the PSE gene. All of these nucleotide variations and resulting amino acid polymorphisms in PSE which are the result of natural variation and do not alter the functional activity of PSEs are intended to come within the scope of the invention.<br>
Nucleic acid molecules which correspond to the natural variants and non-Physcomitrella, -Phytophthora, -Crypthecodinium or -Thraustochytrium homologs, derivatives and analogs of the Phytophthora, Physcomitrella, Crypthecodinium or Thraustochytrium cDNA can be isolated in accordance with standard hybridization techniques under stringent hybridization conditions owing to their homology with the Phytophthora, Physcoinitrella, Crypthecodinium or Thraustochytrium PSE nucleic acid disclosed herein using the Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium cDNA or a part thereof as hybridization probe. In another embodiment, an isolated nucleic acid molecule according to the invention has a minimum length of 15 nucleotides and hybridizes under stringent conditions with the nucleic acid molecule which encompasses a nucleotide sequence of SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11. In other embodiments, the nucleic acid has a minimum length of 25, 50, 100, 250 or more nucleotides. The term "hybridizes under stringent conditions" as used in the present context is intended to describe hybridization and wash conditions under which nucleotide sequences which have at least 60% homology with each other usually remain hybridized with each other. The conditions are preferably such that sequences which have at least approximately 65%, more preferably at least approximately 70% and even more preferably at least approximately 75% or more homology with each other usually remain hybridized with each other. These stringent conditions are known to the skilled worker and can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N. Y. (1989), 6.3.1-6.3.6. A preferred, nonlimiting example of stringent hybridization conditions are hybridizations in 6 x sodium chloride/sodium citrate (podium chloride/s.odium citrate = SSC) at approximately 45°C followed by one or more wash steps in 0.2 X SSC, 0.1% SDS at 50 to 65°C. It is known to the skilled worker that these hybridization conditions differ depending on the type of the nucleic acid and, for example when organic solvents are present, with regard to buffer temperature and concentration. For example, the temperature differs under "standard hybridization conditions" depending on the type of the nucleic acid between 42°C and 58°C in aqueous buffer with a concentration of 0.1 to 5 x SSC (pH 7.2). If organic solvent is present in the abovementioned buffer, for example 50% formamide, the temperature under standard conditions is approximately 42^C.<br><br>
The hybridization conditions/for DNA:DNA hybrids are preferably for example 0.1 x SSC and 20°C to 45'C, preferably between 30°C and 45°C. The hybridization conditions for DNA:RNA hybrids are preferably for example 0.1 x SSC and 30°C to SS'^C, preferably between 45°C and 55°C. The abovementioned hybridization temperatures are determined for example for a nucleic acid approximately 100 bp (= base pairs) in length and a G + C content of 50% in the absence of formamide. The skilled worker knows how the hybridization conditions required can be determined with reference to textbooks, such as the one mentioned above or from the following textbooks: Sambrook et al., "Molecular Cloning", Cold Spring Harbor Laboratory, 1989; Hames and Higgins (Ed.) 1985, "Nucleic Acids Hybridization: A Practical Approach", IRL Press at Oxford University Press, Oxford; Brown (Ed.) 1991, "Essential Molecular Biology: A Practical Approach", IRL Press at Oxford University Press, Oxford.<br>
Preferably, an isolated nucleic acid molecule according to the invention which hybridizes under stringent conditions with a sequence of SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 corresponds to a naturally occurring nucleic acid molecule. As used in the present context, a "naturally occurring" nucleic acid molecule refers to an RNA or DNA molecule with a nucleotide sequence which occurs in nature (for example which encodes a natural protein). In one embodiment, the nucleic acid encodes a naturally occurring Physcomitrella patens PSE, Phytophthora infestans PSE, Crypthecodinium cohnii PSE or Thraustochytrium PSE.<br>
In addition to naturally occurring variants of the PSE sequence which may exist in the population, the skilled worker furthermore recognizes that changes by means of mutation may also be introduced into a nucleotide sequence of SEQ ID N0:1, SEQ ID K0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID N0:11, which leads to changes in the amino acid sequence of the encoded PSE without adversely affecting the functionality of the PSE protein. For example, nucleotide substitutions which lead to amino acid substitutions on "nonessential" amino acid residues can be generated in a sequence of SEQ ID N0;1, SEQ ID NO;3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11. A "nonessential" amino acid residue is a residue which can be altered in a wild-type sequence of one of the PSEs (SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID N0:12) without altering the activity of the PSE, while an "essential" amino acid residue is required for the PSE activity. Other amino acid residues (for example those which are not conserved, or only semi-conserved, in the domain with PSE<br><br>
activity), however, may not "fee essential for the activity and can therefore probably be altered without altering the PSE activity.<br>
Accordingly, a further aspect of the invention relates to nucleic acid molecules which encode PSEs comprising altered amino acid residues which are not essential for the PSE activity. These PSEs differ from a sequence in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 with regard to the amino acid sequence while still retaining at least one of the PSE activities described herein. In one embodiment, the isolated nucleic acid molecule encompasses a nucleotide sequence encoding a protein, the protein encompassing an amino acid sequence with at least approximately 50% homology with an amino acid sequence of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12 and being able to participate in the metabolism of compounds required for the synthesis of cell membranes in Phytophthora, Physcomitrella, Crypthecodinium or Thraustochytrium or in the transport of molecules via these membranes. The protein encoded by the nucleic acid molecule preferably has at least approximately 50 to 60% homology with one of the sequences in SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID H0:12, more preferably at least approximately 60 to 70% homology with one of the sequences in SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID N0:12, even more preferably at least approximately 70 to 80%, 80 to 90%, 90 to 95% homology with one of the sequences in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID NO:8, SEQ ID 110:10 and SEQ ID 150:12, and most preferably at least approximately 96%, 97%, 98% or 99% homology with one of the sequences in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID NO:B, SEQ ID NO:10 and SEQ ID N0:12.<br>
To determine the percentage homology of two amino acid sequences (for example one of the sequences of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 and a nutated form thereof) or of two nucleic acids, the sequences are written one underneath the other to allow optimal comparison (for sxample, gaps may be introduced into the sequence of a protein or 3f a nucleic acid in order to generate an optimal alignment with the other protein or the other nucleic acid). Then, the amino acid residues or nucleotides on the corresponding amino acid positions or nucleotide positions are compared, if a position in a sequence (for example one of the sequences of SEQ ID NO:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID H0:8, SEQ ID NO:10 and SEQ ID NO:12) is occupied by the same amino acid residue or the same nucleotide as the corresponding position in the other sequence (for exEimple a mutated form of the sequence selected from SEQ ID KG:2, SEQ ID<br><br>
N0:4, SEQ ID N0:6, SEQ ID NOrS, SEQ ID N0:10 and SEQ ID N0:I2), then the molecules are homologous at this position (i.e. amino acid or nucleic acid "homology" as used in the present context ::orresponds to amino acid or nucleic acid "identity"). The percentage homology between the two sequences is a function of the number of identical positions which the sequences share (i.e. i  homology = number of identical positions/total number of positions x 100).<br>
\n  isolated nucleic acid molecule which encodes a PSE which is lomologous with a protein sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID K0:8, SEQ ID NQ:10 and SEQ ID N0:12 can be jenerated by introducing one or more nucleotide substitutions, idditions or deletions into a nucleotide sequence of SEQ ID K0:1, JEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SSQ ID 10:11 so that one or more amino acid substitutions, additions or ieletions are introduced into the encoded protein. Mutations can je introduced into one of the sequences of SEQ ID N0:1, SEQ ID 10:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11 by standard techniques, such as site-directed mutagenesis and 'CR-mediated mutagenesis. Preferably, conservative amino acid jubstitutions are generated at one or more of the predicted lonessential amino acid residues. In a "conservative amino acid substitution", the amino acid residue is exchanged for an amino tcid residue with a similar side chain. Families of amino acid residues with similar side chains have been defined in the specialist field. These families encompass amino acids with basic lide chains (for example lysine, arginine, histidine), acidic iide chains (for example aspartic acid, glutamic acid), uncharged jolar side chains (for example glycine, asparagine, glutamine, lerine, threonine, tyrosine, cysteine), unpolar side chains, (for •xample alanine, valine, leucine, isoleucine, proline, &gt;henylalanine, methionine, tryptophan), beta-branched side chains<br>
for example threonine, valine, isoleucine) and aromatic side ;hains (for example tyrosine, phenylalanine, tryptophan, listidine). A predicted nonessential amino acid residue in a PSE .s thus preferably exchanged for another amino acid residue from :he same side-chain family. As an alternative, in another imfaodiment, the mutations can be introduced randomly over all or lart of the PSE-encoding sequence, for example by saturation mtagenesis, and the resulting mutants can be screened for the 'SE activity described herein in order to identify mutants which<br>
etain PSE activity. After the mutagenesis of one of the lequences of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, ;EQ ID NO:9 and SEQ ID NO:11, the encoded protein can be ixpressed recombinantly, and the activity of the protein can be<br><br>
determined, for example using the assay described herein (see examples section).<br>
In addition to the nucleic acid molecules which encode the above-described PSEs, a further aspect of the invention relates to isolated nucleic acid molecules which are "antisense" thereto. An "antisense" nucleic acid encompasses a nucleotide sequence which is complementary to a "sense" nucleic acid which encodes a protein, for example complementary to the coding strand of a double-stranded cDNA molecule or complementary to an mRNA sequence. Accordingly, an antisense nucleic acid can bind to a sense nucleic acid via hydrogen bonds. The antisense nucleic acid can be complementary to a complete PSE-encoding strand or only to part thereof. In one embodiment, an antisense nucleic acid molecule is "antisense" to a  "coding region" of the coding strand of a nucleotide sequence encoding a PSE. The term "coding region" refers to the region of the nucleotide sequence which encompasses codons which are translated into amino acid residues (for example the entire coding region which starts and ends with the stop codon, i.e. the last codon before the stop codon). In a further embodiment, the antisense nucleic acid molecule is "antisense" to a "noncoding region" of the coding strand of a nucleotide sequence encoding PSE. The term "noncoding strand" refers to 5' and 3' sequences which flank the coding region and are not translated into amino acids (i.e. which are also termed 5'- and 3'-untranslated regions).<br>
Taking into consideration the PSE-encoding sequences disclosed herein of the coding strand {for example the sequences shown in SEQ ID N0:1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID NO:9 and SEQ ID NO:ll), antisense nucleic acids according to the invention can be designed in accordance with the rules of the Watson-Crick base pairing. The antisense nucleic acid molecule can be complementary to all of the coding region of PSE mRNA, but is more preferably an oligonucleotide which is "antisense" to only part of the coding or noncoding region of PSE mRNA. For example, the antisense oligonucleotide can be complementary to the region around the translation start of PSE mRNA. An antisense oligonucleotide can have a length of, for example, approximately 5, 10, 15, 20, 25, 30, 35, 40, 45 or 50 and more nucleotides. An antisense oligonucleotide is advantageously 15 to 25 nucleotides in length. An antisense nucleic acid according to the invention can be constructed by processes known in the art using chemical synthesis and enzymatic ligation reactions. For example, an antisense nucleic acid (for example an antisense oligonucleotide) can be synthesized chemically, making use of naturally occurring nucleotides or various modified nucleotides which are such that<br><br>
they increase the biological stability of the molecules or increase the physical stability of the duplex formed between the antisense and the sense nucleic acid; for example, ^ phosphorothioate derivatives and acridin-substituted nucleotides may be used. Examples of modified nucleotides which may be used for generating the antisense nucleic acid are, inter alia, 5-fluorouracil, 5-bromouraciI, S-chlorouracil, 5-iodouracil, hypoxanthin, xanthin, 4-acetylcytosine, 5-{carboxyhydroxylmethyl)uracil, 5-carboxymethylaininomethyl-2-thiouridine, 5-carboxymethylaminomethyluracil, dihydrouracil, beta-D-galsctosylqueosine, inosine, N6-isopentenyladenine, 1-methylguanine, 1-methylinosine, 2,2-dimethylguanine, 2-methyladenine, 2-methylguanine, 3-methylcytosine, 5-methylcytosine, H6-adenine, 7-inethylguanine, S-methylaminomethyluracil, 5-methoxyaminomethyl-2-thiouracil, beta-D-mannosylqueosine, 5'-methoxycarboxymethyluracil, 5-methoxyuracil, 2-niethylthio-N6-isopentyladenine, uracil-5-oxyacetic acid (v), wybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-inethyl-2-thiouracil, 2-thiouracil, 4-thiouracil, 5-methyluracii, methyl uracil-5-oxyacetate, uracil-5-oxyacetic acid (v), 5-methyl-2-thiouracil, 3-(3-aniino-3-N-2-carboxypropyl)uracil, (acp3)w and 2,6-diaminopurine. Alternatively, the antisense nucleic acid can be generated biologically using an expression vector to which a nucleic acid has been subcloned in antisense orientation (i.e. RNA which is transcribed by the nucleic acid introduced is in antisense orientation relative to a target nucleic acid of interest, which is described in greater detail in the subsection which follows).<br>
The antisense nucleic acid molecules according to the invention are usually administered to a cell or generated in situ so that they hybridize with, or bind to, the cellular mRNR and/or the genomic DNA encoding a PSE, thus inhibiting expression of the protein, for example by inhibiting transcription and/or translation- Hybridization can be effected by conventional nucleotide complementarity with formation of a stable duplex or, for example in the case of an antisense nucleic acid molecule which binds DNA duplices, by specific interactions in the large cleft of the double helix. The antisense molecule can be modified in such a manner that it specifically binds to a receptor or to an antigen expressed at a selected cell surface, for example by Dinding the antisense nucleic acid molecule to a peptide or an antibody, each of which binds to a cell surface receptor or an antigen. The cells can also be provided with the antisense lucleic acid molecule using the vectors described herein. Vector constructs in which the antisense nucleic acid molecule is under<br><br>
the control of a strong proKaryotic, vxrai or eukaryotic promoter, inclusive of a plant promoter, are preferred for achieving sufficient intracellular concentrations of the antisense molecules.<br>
In a further embodiment, the antisense nucleic acid molecule according to the invention is an a-anomeric nucleic acid molecule. An a-anomeric nucleic acid molecule forms specific double-stranded hybrids with complementary RNA, the strands running parallel to each other, in contrast to ordinary p-units [Gaultier et al. (1987) Nucleic Acids Res. 15:6625-6641]. Moreover, the antisense nucleic acid molecule can encompass a 2'-o-methylribonucleotide [Inoue et al. (1987) Nucleic Acids Res. 15:6131-6148] or a chimeric RHA-DNA analogon [Inoue et al. (1987) FEBS Lett. 215:327-330].<br>
In a further embodiment, an antisense nucleic acid according to the invention is a ribozyme. Ribozymes are catalytic RNA molecules with ribonuclease activity which can cleave a single-stranded nucleic acid, such as an mRNA, to which they have a complementary region. Thus, ribozymes, for example hammerhead ribozymes [described in Haselhoff and Gerlach (1988) Nature 334:585-591], can be used for the catalytic cleavage of PSE mRNA transcripts in order to inhibit the translation of PSE mRNA. A ribozyme with specificity for a PSE-encoding nucleic acid can be designed on the basis of the nucleotide sequence of a PSE cDNA disclosed herein (i.e. 38°Ck21_g07fwd in SEQ ID N0:1, SEQ ID N0:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11) or on the basis of a heterologous sequence to be isolated in accordance with the processes taught in the present invention. For example, a derivative of a Tetrahymena-L-19-lvS RNA can be constructed, in which the nucleotide sequence of the active site is complementary to the nucleotide sequence to be cleaved in PSE-encoding mRNA. See, for example, Cech et al., US 4,987,071 and Cech et al., US 5,116,742. AS an alternative, PSE mRNA can be used for selecting a catalytic RNA with a specific ribonuclease activity from amongst a pool of RNA molecules [see, for example,  Bartel, D., and Szostak, J.W. (1993) Science 261:1411-1418].<br>
As an alternative, PSE gene expression can be inhibited by directing nucleotide sequences which are complementary to the regulatory region of a PSE nucleotide sequence (for example a PSE promoter and/or enhancer) in such a way that triple helix structures are formed, which inhibit the transcription of a PSE gene in target cells [see generally Helene, C. (1991) Anticancer<br><br>
(D    (D 3    3<br>
tn   ^<br>
0- ij<br><a></a>
 <br>
tn    n<br><br><br>
►d	n	&gt;	CD	w	t3<br>
3"	o			o	M<br>
^	B	Ml		y-	C<br>
rt	■D	d			iQ<br>
O	M	•-1	(6		<br>
'V	H-	rt-	<br>
	o\	w<br>
tr	tn	»•	3 (D	o\	(D<br>
rt	M-	(D	<br>
	o	Ul<br>
tr	3	H	<br>
	• &gt;	•<br>
0	iQ		o	PO	<br>
h		ID	0 3 [n	-J	CTl<br>
CD	3	§	<br>
	1	Ol<br>
		0	ft	o&gt;	■—^<br>
o	H-	a	H	'••	<br>
h	m	M'	c o<br>
rt		Ut<br>
"
	(U	CTl<br>
•a	(-■	(D	<br>
	3	■l}<br>
ti-	0)	a	<br>
	a	1<br>
er	r+	rt			oa<br>
pj   ~. IB   as<br>
•	I--<br>
a<br>
•	n<br>
u -<br>
VD	(I<br>
to	fi<br>
m	h<br>
p.	.<br>
o<br>
W	1-<br>
pi	vc<br>
■^	K<br>
(0	—<br>
N)<br><br>
CO	<br>
o	W<br>
-J 1	<br>
1<br>
CO	&gt;<br>
M	o<br>
m	ni<br>
■— a<br><br>
or in the plant promoters CaMV/35S [Franck et al., Cell 21 (1980) 285-294], PRPl [Ward et al.. Plant. Mol. Biol. 22 (1993)], SSU, DCS, lib4, usp, STLSl, B33, nos or in the ubiquitin or pHaseolin promoter. Advantageous in this context are also inducible promoters, such as the promoters described in EP-A-0 388 186 (benzylsulfonamide-inducible), Plant J. 2, 1992:397-404 (Gatz et al., tetracyclin-inducible), EP-A-0 335 528 (abscisic acid-inducible) or WO 93/21334 (ethanol- or<br>
cyclohexenol-inducible). Other suitable plant promoters are the cytosolic FBPase or the potato ST-LSI promoter (Stockhaus et al., EMBO J. 8, 1989, 2445), the Glycine max<br>
phosphoribosylpyrophosphate amidotransferase promoter (Genbank Accession No. U87999) or the node-specific promoter described in EP-A-0 249 676. Especially advantageous promoters are promoters which allow expression in tissues which are involved in fatty acid biosynthesis, very especially advantageous are seed-specific promoters, such as the usp, the LEB4, the phaseolin or the napin promoter. Further especially advantageous promoters are seed-specific promoters which can be used for monocots or dicots which are described in US 5,608,152 (oilseed rape napin promoter), WO 98/45461 (Arabidopsis phaseolin promoter), US 5,504,200 (Phaseolus vulgaris phaseolin promoter), WO 91/13980 (Brassica Bce4-promoter), Baeumlein et al.. Plant J., 2, 2, 1992:233-239 (leguminous LEB4 promoter), these promoters being suitable for dicots. The following promoters are suitable, for example, for monocots; the barley lpt-2 or lpt-1 promoter (WO 95/15389 and WO 95/23230), the barley hordein promoter, and ather suitable promoters described in WO 99/16890.<br>
In principle, it is possible to use all natural promoters with their regulatory sequences, such as those mentioned above, for the novel process. It is also possible and advantageous additionally to use synthetic promoters.<br>
^s described above, the gene construct can also encompass further jenes which are to be introduced into the organisms. It is aossible and advantageous to introduce into the host organisms, and to express therein, regulatory genes such as genes for Lnductors, repressors or enzymes which, owing to their enzymatic activity, engage in the regulation of one or more genes of a Diosynthetic pathway. These genes can be of heterologous or homologous origin. The inserted genes can have their own promoter or else be under the control of the promoter of sequence SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID NO:11 or its homologs, derivatives or analogs.<br><br>
To express the other genes wmcn are present, the gene construct advantageously encompasses further 3'- and/or 5'-terminal regulatory sequences for enhancing expression, and these are selected for optimal expression as a function of the host organism chosen and the gene(s).<br>
As mentioned above, these regulatory sequences are intended to make possible the specific expression of the genes and protein expression. Depending on the host organism, this may mean, for example, that the gene is expressed or overexpressed only after induction, or that it is expressed and/or overexpressed immediately.<br>
Moreover, the regulatory sequences or regulatory factors can preferably have an advantageous effect on the expression of the genes which have been introduced, thus enhancing it. In this manner, it is possible that the regulatory elements are advantageously enhanced at the transcriptional level, using strong transcription signals, such as promoters and/or enhancers. However, it is furthermore also possible to enhance translation, for example by improving mRNA stability. The nucleic acid sequences according to the invention are advantageously cloned into a gene construct (= expression cassette, nucleic acid construct) together with at least one reporter gene, and this gene construct is introduced into the organism via a vector or directly into the genome. This reporter gene should allow easy detectability by means of a growth, fluorescence, chemoluminescence, bioluminescence or resistance assay or via photometric measurement. Examples of reporter genes which may be mentioned are genes for resistance to antibiotics or herbicides, hydrolase genes, fluorescence protein genes, bioluminescence genes, sugar or nucleotide metabolism genes or biosynthesis genes such as the Ura3 gene, the Ilv2 gene, the luciferase gene, the p-galactosidase gene, the gfp gene, the 2-deoxygIucose-6-phosphate phosphatase gene, the p-glucuronidase gene, the p-lactamase gene, the neomycin phosphotransferase gene, the hygromycin phosphotransferase gene or the<br>
BASTA {= glufosinate) resistance gene. These genes make it possible for the transcriptional activity, and thus gene expression, to be measured and quantified readily. This allows the identification of positions in the genome which show different productivity.<br><br>
The nucleic acid sequences according to the invention with SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9 and SEQ ID NO:11, which encode elongases, can be present in the expression cassette (= gene construct) in one or more copies.<br>
The expression cassette {= gene construct, nucleic acid construct) can additionally comprise at least one further nucleic acid which encodes a gene, preferably from fatty acid biosynthesis, to be introduced into the host organisms. These genes can be under separate regulation or under the same regulatory region as the genes for the elongase according to the invention. These genes are, for example, further biosynthesis genes, advantageously of fatty acid biosynthesis, which make possible an increased synthesis. Genes which may be mentioned by way of example are those for A19-, A17-, A15-, A12-, A9-, A8-, A6-, A5-, A4-desaturase, the various hydroxylases, A12-acetylenase, acyl-ACP thioesterases, p-ketoacyl-ACP synthases or p-ketoacyl-ACP reductases. The desaturase genes are advantageously used in the nucleic acid construct. Again, these genes may be present in the gene construct in one or more copies.<br>
C.  Recombinant expression vectors and host cells<br>
A further aspect of the invention relates to vectors, preferably expression vectors, comprising a nucleic acid according to the invention or a gene construct according to the invention which encode a PSE (or part thereof). As used in the present context, the term "vector" refers to a nucleic acid molecule which can transport another nucleic acid to which it is bound. One type of vector is a "plasmid", which represents a circular double-stranded DBA loop into which additional DNA segments can be ligated. A further type of vector is a viral vector, it being possible for additional DNA segments to be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they have been introduced (for example bacterial vectors with bacterial origin of replication and episomal mammalian vectors). Other vectors (for example nonepisomal mairanalian vectors) are integrated into the genome of a host cell upon introduction into the host cell and so replicate together with the host genome. In addition, certain vectors can govern the expression of genes to which they are functionally linked. These vectors are referred to as "expression vectors" herein. Usually, expression vectors which are suitable for recombinant DNA techniques take the form of plasmids. In the present description, "plasmid" and "vector" may be used interchangeably since the plasmid is the most frequently used form of a vector. However, the invention is intended to encompass<br><br>
these other forms of expression vectors, such as viral vectors <for example replication-deficient retroviruses adenoviruses and adeno-related viruses which exert similar functions. furthermore the term vector is also intended to encompass other i vectors known skilled worker such as phages sv40 cmv baculovirus adenovirus transposons elements phasmids phagemids cosmids linear or circular dna rha.></for>
The recombinant expression vectors according to the invention I encompass a nucleic acid according to the invention or a gene construct according to the invention in a form which is suitable for expressing the nucleic acid in a host cell, which means that the recombinant expression vectors encompass one or more regulatory sequences, selected on the basis of the host cells to be used for expression, which is or are functionally linked to the nucleic acid sequence to be expressed. In a recombinant expression vector, "functionally linked" means that the nucleotide sequence of interest is bound to the regulatory sequence(s) in such a way that expression of the nucleotide ' sequence is possible and they are bound to each other so that both sequences fulfill the predicted function which has been ascribed to the sequence (for example in an in-vitro transcription/translation system or in a host ceil, when the vector is introduced into the host cell). The term "regulatory sequence" is intended to encompass promoters, enhancers and other expression control elements (for example polyadenylation signals). These regulatory sequences are described, for example, in Goeddel: Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990), or see: Gruber and Crosby, in: Methods in Plant Molecular Biology and Biotechnology, CRC Press, Boca Raton, Florida, Ed.: Click and Thompson, Chapter 7, 89-108, including the references therein. Regulatory sequences encompass those which control the constitutive expression of a nucleotide sequence in many types of host cell and those which control the direct expression of the nucleotide sequence only in certain host cells under certain conditions. The skilled worker knows that the design of the expression vector may depend on factors such as the choice of the host cell to be transformed, the extent to which the desired protein is expressed, and the like. The expression vectors according to the invention can be introduced into host cells in order to produce proteins or peptides, including fusion proteins or fusion peptides, which are encoded by the nucleic acids as described herein (for example PSEs, mutant forms of PSEs, fusion proteins and the like).<br><br>
The recombinant expression vectors according to the invention can be designed for expressing PSEs in prokaryotic or eukaryotic cells. For example, PSE genes can be expressed in bacterial cells, such as C. glutamicum, insect cells (using baculovirus expression vectors), yeast and other fungal cells [see Romanos, M.A., et al. (1992) "Foreign gene expression in yeast: a review". Yeast 8:423-488; van den Hondel, C.A.M.J.J., et al. (1991) "Heterologous gene expression in filamentous fungi", in: More Gene Manipulations in Fungi, J.W. Bennet &amp; L.L. Lasure, Ed., pp. 396-428: Academic Press; San Diego; and van den Hondel, C.A.M.J.J., &amp; Punt, P.J. (1991) "Gene transfer systems and vector development for filamentous fungi, in: Applied Molecular Genetics of Fungi, Peberdy, J.F., et al., Ed., pp. 1-28, Cambridge University Press: Cambridge], algae [Falciatore et al., 1999, Marine Biotechnology. 1, 3:239-251], ciliates of the following types: Holotrichia, Peritrichia, Spirotrichia, Suctoria, Tetrahymena, Paramecium, Colpidium, Glaucoma, Platyophrya, Potomacus, Pseudocohnilembus, Euplotes, Engelmaniella and Stylonychia, in particular of the genus Stylonychia lemnae, using vectors and following a transformation method as described in WO 98/01572, and cells of multicelled plants [see Schmidt, R. and Willmitzer, L. (1988) "High efficiency Agrobacterium tumefaciens-mediated transformation of Arabidopsis  thaliana  leaf and cotyledon explants" Plant Cell Rep.:583-586; Plant Molecular Biology and Biotechnology, C Press, Boca Raton, Florida, Chapter 6/7, pp.71-119 (1993); F.F. White, B, Jenes et al.. Techniques for Gene Transfer, in: Transgenic Plants, Bd. 1, Engineering and Utilization, Ed.: Rung and R. Wu, Academic Press (1993), 128-43; Potrykus, Annu. Rev. Plant Physiol. Plant Molec. Biol. 42 (1991), 205-225 (and references cited therein)] or mammalian cells. Suitable host cells are furthermore discussed in Goeddel, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990). As an alternative, the recombinant expression vector can be transcribed and translated in vitro, for exeunple using T7 promoter regulatory sequences and T7 polymerase.<br>
In prokaryotes, proteins are usually expressed with vectors containing constitutive or inducible promoters which control the expression of fusion proteins or nonfusion proteins. Fusion vectors add a series of amino acids to a protein encoded therein, usually at the amino terminus of the recombinant protein, but also at the C terminus or fused within suitable regions in the proteins. These fusion vectors usually have three tasks: 1) to enhance the expression of recombinant protein; 2) to increase the solubility of the recombinant protein and 3) to support the purification of the recombinant protein by acting as ligand in affinity purification, for example via so-called his tags. In the<br><br>
case of fusion expression vefctors, a proteolytic cleavage site is frequently introduced at the site where the fusion unit and the recombinant protein are linked, so that the recombinant protein can be separated from the fusion unit after purification of the fusion protein. These enzymes and their corresponding recognition sequences encompass factor Xa, thrombin and enterokinase.<br>
Typical fusion expression vectors are, inter alia, pGEX [Pharmacia Biotech Inc; Smith, D.B., and Johnson, K.S. (1988) Gene 67:31-40], pMAL [New England Biolabs, Beverly, MA] and pRIT5 [Pharmacia, Piscataway, NJ], where glutathione S-transferase (GST), maltose-E-binding protein or protein A is fused to the recombinant target protein. In one embodiment, the PSE-encoding sequence is cloned into a pGEX expression vector to generate a vector encoding a fusion protein which encompasses, from the N terminus to the C terminus, GST-thrombin cleavage site-X-protein. The fusion protein can be purified by affinity chromatography using glutathione-agarose resin. Recombinant PSE which is not fused with GST can be obtained by cleaving the fusion protein with thrombin.<br>
Examples of suitable inducible nonfusion E. coli expression vectors are, inter alia, pTrc (Amann et al. (1988) Gene 69:301-315) and pET lid (Studier et al.. Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, California (1990) 60-89). Target gene expression of the pTrc vector is based on transcription by host RNA polymerase from a hybrid trp-lac fusion promoter. Target gene expression from the pET lid vector is based on transcription from a T7-gnlO-lac fusion promoter which is mediated by a coexpressed viral RNA polymerase (T7 gnl). This viral polymerase is provided by the host strains BL21 (DE3) or HMS174 (DE3) by a resident k   prophage which harbors a T7 gnl gene under the transcriptional control of the lacUV 5 promoter.<br>
Other vectors which are suitable for use in prokaryotic organisms are known to the skilled worker; these vectors are, for example, in E. coli pLG338, pACYC184, the pBR series such as pBR322, the pUC series such as pUC18 or pUC19, the M113mp series, pKC30, pRep4, pHSi, pHS2, pPLc236, pMBL24, pLG200, pUR290, pIN-III^^^-Bl, pgtll or pBdCI, in Streptomyces pIJlOl, pIJ364, pIJ702 or pIJ361, in Bacillus pUBllO, pC194 or pBD214, in Corynebacterium pSA77 or pAJ667.<br>
A strategy of maximizing the expression of recombinant protein is to express the protein in a host bacterium whose ability to cleave the recombinant protein proteolytically is disrupted<br><br>
[Gottesman, S., Gene Expression Technology; Methods in Enzymology 185, Academic Press, San Diego, California (1990) 119-128]. A further strategy is to modify the nucleic acid sequence of the nucleic acid to be inserted into an expression vector, so that the individual codons for each amino acid are those which are preferentially used in a bacterium selected for expression, such as C. glutamicum  [Wada et al. (1992) Nucleic Acids Res. 20:2111-2118]. Modification of these nucleic acid sequences according to the invention is carried out by standard techniques of DNA synthesis.<br>
In a further embodiment, the PSE expression vector is a yeast expression vector. Examples of vectors for expression in the yeast S. cerevisiae include pYepSecl fBaldari et al. (1987) Embo J. 6:229-234], pMFa [Kurjan and Herskowitz (1982) Cell 30:933-943], pJRY88 [Schultz et al. (1987) Gene 54:113-123] and pYES2 [Invitrogen Corporation, San Diego, CA]. Vectors and methods for the construction of vectors which are suitable for use in other fungi, such as the filamentous fungi, include those which are described in detail in: van den Hondel, C.A.H.J.J., S Punt, P.J. (1991) "Gene transfer systems and vector development for filamentous fungi, in: Applied Molecular Genetics of fungi, J.F. Peberdy et al., Ed., pp. 1-28, Cambridge University Press: Cambridge, or in: More Gene Manipulations in Fungi [J.W. Bennet &amp; L.L. Lasure, Ed., pp. 396-428: Academic Press: San Diego]. Further suitable yeast vectors are, for example, pAG-1, YEp6, yEpl3 or pEMBLYe23.<br>
As an alternative, the PSEs according to the invention can be expressed in insect cells using baculovirus expression vectors. Baculovirus vectors which are available for expressing proteins in cultured insect cells (for example Sf9 cells) include the pAc series [Smith et al. (1983) Mol. Cell Biol. 3:2156-2165] and the pVL series [Lucklow and Summers (1989) Virology 170:31-39].<br>
The abovementioned vectors are just a short review of suitable vectors which are possible. Further plasmids are known to the skilled worker and are described, for example, in: Cloning Vectors (Ed. Pouwels, P.H., et al., Elsevier, Amsterdam-New York-Oxford, 1985, ISBN 0 444 904018).<br>
In yet a further embodiment, a nucleic acid according to the invention is expressed in mammalian cells using a mammalian expression vector. Examples of mammalian expression vectors include pCDM8 [Seed, B. (1987) Nature 329:840] and pMT2PC [Kaufman et al. (1987) EMBO J. 6:187-195]. When used in mammalian cells, the control functions of the expression vector are<br><br>
frequently provided by viral regulatory elements. Promoters which are usually used are derived, for example, from polyoma, adenovirus2, cytomegalovirus and simian virus 40. Other suitable expression systems for prokaryotic and eukaryotic cells can be found in Chapters 16 and 17 of Sambrook, J., Fritsch, E.F., and Maniatis, T., Molecular Cloning: A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
In another embodiment, the recombinant mammalian expression vector can control the expression of the nucleic acid preferably in a specific cell type (for example, tissue-specific regulatory elements are used for expressing the nucleic acid). Tissue-specific regulatory elements are known in the art. Nonlimiting examples of suitable tissue-specific promoters are, inter alia, the albumin promoter {liver-specific; Pinkert et al. (1987) Genes Dev. 1:268-277], lymphoid-specific promoters [Calame and Eaton (1988) Adv. Immunol. 43:235-275], in particular promoters of T-cell receptors [Winoto and Baltimore (1989) EMBO J. 8:729-733] and immunoglobulins [Banerji et al. (1983) Cell 33:729-740; Queen and Baltimore (1983) Cell 33:741-748], neuron-specific promoters [for example neurofilament promoter; Byrne and Ruddle (1989) PNAS 86:5473-5477], pancreas-specific promoters [Edlund et al., (1985) Science 230:912-916] and mamma-specific promoters [for example milk serum promoter; OS 4,873,316 and EP-A-0 264 166). Also included are development-regulated promoters, for example the mouse hox promoters [Kessel and Gruss (1990) Science 249:374-379] and the fetoprotein promoter [Campes and Tilghman (1989) Genes Dev. 3:537-546].<br>
In a further embodiment, the PSEs according to the invention can be expressed in single-celled plant cells (such as algae), see Falciatore et al., 1999, Marine Biotechnology 1 (3):239-251 and references cited therein, and in plant cells from higher plants (for example spermatophytes such as crops). Examples of plant expression vectors include those which are described in detail in: Becker, D., Kemper, E., Schell, J., and Hasterson, R. (1992) "New plant binary vectors with selectable markers located proximal to the left border". Plant Mol. Biol. 20:1195-1197; and Bevan, M.w. (1984) "Binary Agrobacterium vectors for plant transformation", Nucl. Acids Res. 12:8711-8721; Vectors for Gene Transfer in Higher Plants; in: Transgenic Plants, Vol. 1, Engineering and Utilization, Ed.; Kung and R, wu. Academic Press, 1993, pp. 15-38. Further suitable plant vectors are described, inter alia, in "Methods in Plant Molecular Biology and Biotechnology" (CRC Press), Chapter 6/7, pp. 71-119. Advantageous<br><br>
vectors are so-called shuttle vectors or binary vectors, which replicate in E. coli and Agrobacterium.<br>
A plant expression cassette preferably comprises regulatory sequences which can control gene expression in plant cells and which are functionally linked, so that each sequence can fulfill its function, such as transcriptional termination, for example polyadenylation signals. Preferred polyadenylation signals are those derived from Agrobacterium tumefaciens T-DNA, such as gene 3 of the Ti plasmid pTiACHS, which is known as octopine synthase [Gielen et al,, EMBO J. 3 (1984) 835 et seq.] or functional equivalents thereof, but all other terminators which are functionally active in plants are also suitable.<br>
Since plant gene expression is very frequently not limited to the iranscriptional level, a plant expression cassette preferably comprises other functionally linked sequences, such as iranslation enhancers, for example the overdrive sequence, which :ontains the 5'-untranslated tobacco mosaic virus leader sequence, which increases the protein/RNA ratio [Gallie et al., .987, Nucl. Acids Research 15:8693-8711].<br>
'lant gene expression must be functionally linked to a suitable iromoter which effects gene expression in a cell- or .issue-specific manner with the correct timing. Preferred promoters are those which lead to constitutive expression [Benfey et al., EMBO J. 8 (1989) 2195-2202], such as those which are derived from plant viruses such as 35S CAMV [Franck et al., Cell 21 (1980) 285-294], 19S CaMV (see also US 5,352,605 and WO B4/02913) or plant promoters such as the Rubisco small subunit promoter described in US 4,962,028.<br>
Other sequences which are preferred for use for functional linkage in plant gene expression cassettes are targeting sequences, which are required for targeting the gene product in its corresponding cell compartment [for a review, see Kermode, Crit. Rev. Plant Sci. 15, 4 (1996) 285-423 and references cited therein], for example into the vacuole, the nucleus, all types of plastids such as amyloplasts, chloroplasts, chromoplasts, the extracellular space, the mitochondria, the endoplasmatic reticulum, elaioplasts, peroxisomes and other compartments of plant cells.<br>
Plant gene expression can also be facilitated via a chemically inducible promoter [for a review, see Gatz 1997, Annu. Rev. Plant Physiol. Plant Mol. Biol., 48:89-108]. Chemically inducible promoters are particularly suitable when it is desired for gene<br><br>
expression to take place in i specific manner with regard to timing. Examples of such promoters are a salicylic acid-inducible promoter (WO 95/19443), a tetracyclin-inducible promoter (Gatz et al. (1992) Plant J. 2, 397-404] and an ethanol-inducible promoter.<br>
Other suitable promoters are promoters which respond to biotic or<br>
abiotic stress conditions, for example the pathogen-induced PRpl<br>
gene promoter [Ward et al.. Plant. Mol. Biol- 22 (1993) 361-366],<br>
the heat-inducible totomato hspBO promoter (US 5,187,267), the<br>
low-temperature-inducible potato alpha-amylase promoter<br>
(WO 96/12814) or the wound-inducible pinll promoter (EP-A-0 375<br>
091).<br>
Promoters which are particularly preferred are those which lead to gene expression in tissues and organs in which lipid and oil Diosynthesis take place, in seed cells such as endosperm cells and cells of the developing embryo. Promoters which are suitable are the oilseed rape napin gene promoter (US 5,608,152), der /icia faba USP promoter [Baeumlein et al,, Mol Gen Genet, 1991, &gt;25 (3):459-67l, the Arabidopsis oleosin promoter (WO 98/45461), :he Phaseolus vulgaris phaseolin promoter (US 5,504,200), the irassica Bce4 promoter (WO 91/13980) or the legumin E4 promoter LeB4; Baeumlein et al., 1992, plant Journal, 2 (2):233-9], and )romoters which lead to the seed-specific expression in monocots iuch as maize, barley, wheat, rye, rice and the like. Notable iromoters which are suitable are the barley lpt2 or Iptl gene iromoter (WO 95/15389 and WO 95/23230), or the promoters [escribed in WO 99/16890 (promoters from the barley hordein gene, he rice glutelin gene, the rice orysin gene, the rice prolamin ene, the wheat gliadin gene, the wheat glutelin gene, the maize ein gene, the oat glutelin gene, the sorghum kasirin gene, and _he rye secalin gene).<br>
Promoters which are also particularly suitable are those which lead to plastid-specific expression, since plastids are the compartment in which the precursors and some end products of lipid biosynthesis are synthesized. Suitable promoters such as the viral RNA polymerase promoter are described in WO 95/16783 and WO 97/06250, and the Arabidopsis clpP promoter, described in WO 99/46394.<br>
The invention furthermore provides a recombinant expression vector encompassing a DNA molecule according to the invention which is cloned into the expression vector in antisense orientation, i.e. the DNA molecule is functionally linked to a regulatory sequence in such a way that it allows the expression<br><br>
(by transcribing the DNA moledule) of an RNA molecule which is "antisense" to the PSE mRNA. Regulatory sequences may be selected which are functionally linked to a nucleic acid cloned in antisense orientation and which control the continuous expression of the antisense RHA niolecule in a multiplicity of cell types, for example, viral promoters and/or enhancers or regulatory sequences may be selected which control the constitutive, tissue-specific or cell type-specific expression of antisense RNA. The antisense expression vector may be present in the form of a recombinant plasmid, phagemid or attenuated virus in which the antisense nucleic acids are produced under the control of a highly effective regulatory region whose activity can be determined by the cell type into which the vector has been introduced. For an explanation of the regulation of gene expression by means of antisense genes, see weintraub, H., et al., Antisense-RNA as a molecular tool for genetic analysis. Reviews - Trends in Genetics, Vol. 1(1) 1986.<br>
a further aspect of the invention relates to host cells into rfhich a recombinant expression vector according to the invention has  been introduced. The terms "host cell" and "recombinant host :ell" are used interchangeably in the present context. Naturally, :hese terms do not only refer to the particular target cell, but ilso to the progeny or potential progeny of this cell. Since specific modifications may occur in subsequent generations owing :o mutation or environmental effects, this progeny is not lecessarily identical with the parental cell, but remains within ;he scope of the term as used in the present context.<br>
I host cell may be a prokaryotic or eukaryotic cell. For example, I PSE can be expressed in bacterial cells such as C. glutamicum, .nsect cells, fungal cells or mammalian cells (such as Chinese lamster ovary cells (CHO) or COS cells), algae, ciliates, plant :ells, fungi or other microorganisms, such as C. glutamicum. vther suitable host cells are known to the skilled worker.<br>
Vector DNA can be introdued into prokaryotic or eukaryotic cells via conventional transformation or transfection techniques. The terms "transformation" and "transfection", conjugation and transduction as used in the present context are intended to encompass a multiplicity of methods known in the art for introducing foreign nucleic acid (for example DNA) into a host cell, including calcium phosphate or calcium chloride coprecipitation, DEAE-dextran-mediated transfection, lipofection, natural competence, chemically mediated transfer, electroporation or particle bombardment. Suitable methods for the transformation or transfection of host cells, including plant cells, can be<br><br>
found in Sambrook et al. (Molecular Cloning: A Laboratory Manual., 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, m,   1989) and other laboratory textbooks, such as Methods in Molecular Biology, 1995, Vol. 44, Agrobacterium protocols, Ed.: Gartland and Davey, Humana Press, Totowa, New Jersey.<br>
It is known about the stable transfection of mammalian cells that only a minority of the cells integrate the foreign DNA into their genome, depending on the expression vector used and the transfection technique used. To identify and select these integrants, a gene which encodes a selectable marker (for example resistance to antibiotics) is usually introduced into the host cells together with the gene of interest. Preferred selectable markers encompass those which impart resistance to drugs such as G418, hygromycin and methotrexate, or, in plants, those which impart resistance to a herbicide such as glyphosphate or glufosinate. Further suitable markers are, for example, markers which encode genes which are involved in the biosynthesis pathways of, for example, sugars or amino acids, such as ^-galactosidase, ura3 or ilv2. Markers which encode genes such as Luciferase, gfp or other fluorescence genes are also suitable. These markers can be used in mutants in which these genes are not functional since they have been deleted for example by means of :onventional methods. Furthermore, markers which encode a nucleic icid which encodes a selectable marker can be introduced into a lost cell on the same vector as the one which encodes a PSE, or !an be introduced on a separate vector. Cells which have been ■.ransfected stably with the nucleic acid introduced can be .dentified for example by drug selection (for example, cells rhich have the selectable marlcer integrated survive, whereupon ither cells die).<br>
'o generate a homologously recombinant microorganism, a vector is enerated which contains at least one segment of a PSE gene into hich a deletion, addition or substitution has been introduced in rder to modify the PSE gene hereby, for example to functionally uisrupt it. This PSE gene is preferably a Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium PSE gene, but a homolog or analog from other organisms, even from a mainmalian, fungal or insect source, can also be used. In a preferred embodiment, the vector is designed in such a way that the endogenous PSE gene is functionally disrupted <i.e. no longer encodes a functional protein also teinned knock-out vector upon homologous recombination. as an alternative the can be designed such that endogenous pse gene mutates or is modified otherwise recombination while still encoding></i.e.><br>
/<br>
functional protein (for example, the upstream regulatory region can be modified in such a way that this leads to a modification of the expression of the endogenous PSE). To generate a point mutation via homologous recombination, DNA-RNA hybrids, which are also known as chimeraplasty and which are known from Cole-Strauss at al., 1999, Nucleic Acids Research 27(5):1323-1330 and Kmiec, Gene therapy, 1999, American Scientist, 87(3):240-247 can also be used.<br>
In the vector for homologous recombination, the modified segment of the PSE gene is flanked at its 5' and 3' end by additional nucleic acid of the PSE gene, so that homologous recombination is possible between the exogenous PSE gene which is present on the vector and an endogenous PSE gene in a microorganism or a plant. The additional flanking PSE nucleic acid is sufficiently long for successful homologous recombination with the endogenous gene. Usually, several hundred base pairs up to kilobases of flanking DNA (both on the 5' and on the 3' end) are present in the vector [for a description of vectors for homologous recombination, see, for example, Thomas, K.R., and Capecchi, M.R. (1987) Cell 51:503 or for the recombination in Physcomitrella patens on cDSA basis, see Strepp et al., 1998, Proc. Natl. Acad. Sci. USA 95 (8) :4368-4373l. The vector is introduced into a microorganism or plant cell (for example by means of polyethylene glycol-mediated DNA), and cells in which the PSE gene introduced has undergone homologous recombination with the endogenous PSE gene are selected using techniques known in the art.<br>
En another embodiment, recombinant organisms such as nicroorganisms of the plants can be generated which contain selected systems which allow regulated expression of the gene Introduced. The inclusion of a PSE gene in a vector, where it is ilaced under the control of the lac-operon, allows, for eXEimple, •xpression of the PSE gene only in the presence of IPTG. These regulatory systems are known in the art.<br>
L host cell according to the invention, such as prokaryotic or iukaryotic host cells, growing either in culture or in a field, :an be used for producing (i.e. expressing) a PSE. In plants, an ilternative method can additionally be used by directly xansferring DNA into developing flowers via electroporation or .grobacterium-mediated gene transfer. Accordingly, the invention ■urthermore provides methods of producing PSEs using the host ;ells according to the invention. In one embodiment, the method incorapasses growing the host cell according to the invention \into which a recombinant expression vector encoding a PSE has been introduced or into whose genome a gene encoding a wild-type<br><br>
or modified PSE has been introduced) in a suitable medium until the PSE has been produced. In a further embodiment, the method encompasses isolating the PSEs from the medium or the host cell.<br>
Host cells which are suitable in principle for taking up the nucleic acid according to the invention, the novel gene product according to the invention or the vector according to the invention are all prokaryotic or eukaryotic organisms- The host organisms which are used advantageously are organisms such as bacteria, fungi, yeasts, animal cells or plant cells. Further advantageous organisms are animals or, preferably, plants or parts thereof. The term "animal" is understood here as not including humans. Fungi, yeasts or plants are preferably used, especially preferably fungi or plants, very especially preferably plants such as oil crops which contain large amounts of lipid compounds, such as oilseed rape, evening primrose, castor-oil plant, canola, peanut, linseed, soya, safflower, sunflower, borage, oil palm, coconut or plants such as maize, wheat, rye, oats, triticale, rice, barley, cotton, cassava, pepper, tagetes, Solanaceae plants such as potato, tobacco, aubergine and tomato, Vicia species, pea, alfalfa, shrub plants, (coffee, cacao, tea), Salix species, trees (oil palm, coconut) and perennial grasses and fodder crops. Especially preferred plants according to the invention are oil crops such as soya, peanut, oilseed rape, canola, castor-oil plant, linseed, evening primrose, sunflower, safflower, trees (oil palm, coconut).<br>
A particularly preferred aspect of the invention relates to a plant cell which comprises the polynucleotide according to the invention or the vector according to the invention. Preference is furthermore given to transgenic plants or plant tissue comprising the plant cell according to the invention. A further aspect of the present invention relates to those parts of the plants according to the invention which can be harvested and to the material suitable for propagating the transgenic plants according to the invention, containing either plant cells according to the invention which express the nucleic acid according to the invention or containing cells which have an elevated level of the protein according to the invention. In principle, all parts of a plant can be harvested, in particular flowers, pollen, fruits, seedlings, roots, leaves, seeds, tubers, stems, etc. propagation material includes, for example, seeds, fruits, seedlings, tubers, cuttings and rhizomes.<br>
D.  Isolated PSE<br><br>
A further aspect of the invention relates to isolated PSEs and biologically active parts thereof. An "isolated" or "purified" protein or a biologically active part thereof, is essentially free of cellular material when it is produced by  recombinant DKA techniques, or free of chemical precursors or other chemicals when it is synthesized chemically. The term "essentially free of cellular material" encompasses PSE preparations in which the protein is separate from cellular components of the cells in which it is produced naturally or recombinantly. In one embodiment, the term "essentially free of cellular material" encompasses PSE preparations with less than approximately 30% (based on the dry weight) of non-PSE (also referred to herein as "contaminating protein"), more preferably less than approximately 20% of non-PSE, even more preferably less than approximately 10% of non-PSE and most preferably less than approximately 5% of non-PSE. When the PSE or a biologically active part thereof has been produced by recombinant technology, it is also essentially free of culture medium, i.e. the culture medium amounts to less than approximately 20%, more preferably less than approximately 10% and most preferably less than approximately 5% of the volume of the protein preparation. The term "essentially free of chemical precursors or other chemicals" encompasses PSE preparations in which the protein is separate from chemical precursors or other chemicals which are involved in the synthesis of the protein,. In one embodiment, the term "essentially free of chemical precursors or other chemicals" encompasses PSE preparations with less than approximately 30% (based on the dry weight) of chemical precursors or non-PSE chemicals, more preferably less than approximately 20% of chemical precursors or non-PSE chemicals, even more preferably less than approximately 10% of chemical precursors or non-PSE chemicals and most preferably less than approximately 5% of chemical precursors or non-PSE chemicals. In preferred embodiments, isolated proteins or biologically active parts thereof exhibit no contaminating proteins from the same organism from which the PSE originates. In the case of the protein according to the invention which contains the sequence shown in SEQ ID NO: 10 or which is encoded by a gene which comprises SEQ ID NO: 9, however, it has to be taken into account that the sequence starts with two Met. In the translation of a corresponding encoding nucleic acid sequence, this may result in the expression of two derivatives of the protein according to the invention starting with the first or the second Met. The expression ratio between the two derivatives can vary between 0 and 1, depending on the type of expression or the host organism. The invention accordingly comprises PSE containing both of the derivatives mentioned, or only one of the derivatives. The two derivatives can have different activities, localizations,<br><br>
half-lives, regulation mechanisms, etc. These proteins are usually produced by recombinant expression, for example Physcomitreila, Phytophthora, Crypthecodinium or Thraustochytrium PSE in plants such as Physcomitreila patens or the abovementioned, or microorganisms for example bacteria such as E. coli, Bacillus subtiiis, C. glutamicum, fungi such as Hortierella, yeast such as Saccharomyces, or ciliates such as Colpidium or algae such as Phaeodactylum.<br>
An isolated PSE according to the invention or part thereof can also participate in the metabolism of compounds required for the synthesis of cell membranes in Physcomitreila, Phytophthora, Crypthecodinium or Thraustochytrium or in the transport of nolecules via these membranes. In preferred embodiments, the protein or the part thereof encompasses an amino acid sequence tfhich has sufficient homology with an amino acid sequence of SEQ ro N0:2, SEQ ID K0:4, SEQ ID N0:6, SEQ ID K0:8, SEQ ID NO:10 and 5EQ ID H0-. 12 for the protein or the part thereof to retain the ability to participate in the metabolism of compounds required for the synthesis of cell membranes in Physcomitreila, Phytophthora, Crypthecodinium or Thraustochytrium or in the transport of molecules via these membranes. The part of the protein is preferably a biologically active part as described herein. In a further preferred embodiment, a PSE according to the invention has one of the amino acid sequences shown in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12. In a further preferred embodiment, the PSE has an amino acid sequence which is encoded by a nucleotide sequence which hybridizes with a nucleotide sequence of SEQ ID N0:1, SEQ ID J10:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID H0:9 and SEQ ID NO: 11, for example under stringent conditions. In yet another preferred embodiment, the PSE has an amino acid sequence encoded by a nucleotide sequence which has at least approximately 50 to 60%, preferably at least approximately 60 to 70%, more preferably at least approximately 70 to 80%, 80 to 90%, 90 to 95% and more preferably at least approximately 96%, 97%, 98%, 99% or more honrology with one of the amino acid sequences of SEQ ID N0;2, SEQ ID B0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12. The PSE preferred in accordance with the invention preferably also has at least one of the PSE activities described herein. For example, a preferred PSE according to the invention encompasses an amino acid sequence encoded by a nucleotide sequence which hybridizes with a nucleotide sequence of SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9 and SEQ ID NO:11, for example under stringent conditions, and which can participate in the metabolism of compounds required for the synthesis of cell membranes in Physcomitreila, Phytophthora, Crypthecodinium or<br><br>
Thraustochytriurn or in the transport of molecules via these membranes and is capable of elongating one or more polyunsaturated fatty acids with at least two double bonds and a chain length of Cie or Cis-<br>
In other embodiments, the PSE is essentially homologous with an amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID N0:12 and retains the functional activity of the protein of one of the sequences of SEQ ID N0:2, BEQ ID N0:4, SEQ ID N0:6, SEQ ID KO-.B, SEQ ID HO:10 and SEQ ID NO:12, their amino acid sequence differs, owing to natural variation or mutagenesis as described in detail in the above subsection I. In a further embodiment, the PSE is, accordingly, a protein encompassing an amino acid sequence which has at least approximately 50 to 60%, preferably at least approximately 60 to 70% and more preferably at least approximately 70 to 80%, 80 to 90%, 90 to 95% and most preferably at least approximately 96%, 97%, 98%, 99% or more homology with a complete eimino acid sequence of SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID NO:12 and has at least one of the PSE activities described herein. In another embodiment, the invention relates to a complete Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytriurn protein which is essentially homologous with a complete amino acid sequence of SEQ ID NO:2, SEQ ID N0:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO:10 and SEQ ID NO:12.<br>
Biologically active parts of a PSE encompass peptides encompassing amino acid sequences derived from the amino acid sequence of a PSE, for example an amino acid sequence shown in SEQ ID K0:2, SEQ ID NO:4, SEQ ID N0:6, SEQ ID NO:8, SEQ ID NO:10 and SEQ ID NO:12 or the amino acid sequence of a protein which is homologous with a PSE, which peptides have fewer amino acids than the full-length PSE or the full-length protein which is homologous with a PSE and have at least one activity of a PSE. Biologically active parts (peptides, for example peptides with a length of, for example, 5, 10, 15, 20, 30, 35, 36, 37, 38, 39, 40, 50, 100 or more amino acids) usually encompass a domain or a motif with at least one activity of a PSE. Moreover, other biologically active parts in which other regions of the protein are deleted can be generated by recombinant techniques and studied with regard to one or more of the activities described herein. The biologically active parts of a PSE preferably ,encompass one or more selected domains/motifs or parts thereof with biological activity.<br><br>
Some of such domains and motifs can be identified by sequence analysis, for example using computer-aided methods.<br>
The sequences according to the invention were found to contain, for example, KK motifs.<br>
Kermode 1996, Critical Reviews in Plant Sciences 15 (4): 285-423, describes KK motifs, a double lysine, which is found mainly as KKXX or K X K XXX motif and which influences recycling from the ER to the Golgi apparatus and thus the residence time of the protein and its enzyme activity at a certain location, in particular the ER.<br>
Double lysine motifs have also been found, for example in A12-desaturases (Arondel et al. 1992, Science 258:1353), and they are also present in the elongases according to the invention. In particular motifs which can be localized at the C-terminus have been described. In the sequences according to the invention, there is a noticeable accumulation of lysines at the C-terminus.<br>
Moss elongase	PSEl: C-terminus  KQKGAKTE<br>
SEQ ID NO 2:	KTKKA<br>
5EQ ID NO 4	KKSTPAAKKTN<br>
SEQ ID NO 6:	KHLK<br>
These may be a possible gene variation.<br>
[■here are Lys radicals which influence targeting, addressing and Localization at or in the ER. A masking of this sequence, nodification or spatial modification, in the vicinity of the end )f the C-terminus, for example by fusion with GFP "green fluorescent protein" may be utilized to influence lorapartmentalization.<br>
■SEs are preferably produced by recombinant DNA techniques. For ixample, a nucleic acid molecule encoding the protein is cloned into an expression vector (as described above), the expression vector is introduced into a host cell (as described above), and the PSE is expressed in the host cell. The PSE can then be isolated from the cells by a suitable purification scheme using standard techniques of protein purification. As an alternative Eor the recombinant expression, a PSE, a PSE polypeptide or a PSE aeptide can be synthesized chemically by standard techniques of peptide synthesis. Moreover, native PSE can be isolated from :ells (for example endothelial cells), for example using an<br><br>
anti-PSE antibody which can be raised by standard techniques, using a PSE according to the invention or a fragment thereof.<br>
The invention also provides chimeric PSE proteins or PSE fusion 'proteins. As used in the present context, a "chimeric PSE protein" or "PSE fusion protein" encompasses a PSE polypeptide which is functionally bound to a non-PSE polypeptide. A "PSE polypeptide" refers to a polypeptide with an amino acid sequence which corresponds to a PSE, while a "non-PSE polypeptide" refers to a polypeptide with an amino acid sequence which corresponds to a protein which is essentially not homologous with PSE, for exeimple a protein which differs from PSE and which originates from the same or another organism. Within the fusion protein, the term "functionally linked" is to be understood as meaning that the PSE polypeptide and the non-PSE polypeptide are fused to each other in such a way that both sequences fulfill the predicted function which has been ascribed to the sequence used. The non-PSE polypeptide can be fused to the N terminus or the C terminus of the PSE polypeptide. In one embodiment the fusion protein is, for example, a GST-PSE fusion protein in which the PSE sequences arc fused to the C terminus of the GST sequences. These fusion proteins can facilitate the purification of the recombinant PSEs. In a further embodiment, the fusion protein is a PSE which has a  heterologous signal sequence at its N terminus. In certain host cells (for example mammalian host cells), expression and/or secretion of a PSE can be increased by using a heterologous signal sequence.<br>
A chimeric PSE protein or PSE fusion protein according to the invention is produced by standard recombinant DMA techniques. For example, DNA fragments which encode different polypeptide sequences are ligated to each other in correct reading frame using conventional techniques, for example by employing blunt ends or sticky ends for ligation, restriction enzyme cleavage for providing suitable ends, filling up cohesive ends, as required, treatment with alkaline phosphatase to avoid undesired linkages, and enzymatic ligation. In a further embodiment, the fusion gene :;an be synthesized by conventional techniques including DNA synthesizers. As an alternative, PCR amplification of gene fragments can be carried out using anchor primers which generate ::oraplementary overhangs between successive gene fragments which :an subsequently be hybridized and reamplified to give rise to a :himeric gene sequence (see, for example, Current Protocols in tolecular Biology, Ed. Ausubel et al., John Wiley &amp; Sons: 1992). Moreover, a large number of expression vectors which already sncode a fusion unit (for example a GST polypeptide) are rommercially available. PSE-encoding nucleic acid can be cloned<br><br>
-..^w ^ui.11 ail tsApression vector so that the fusion unit is linked in correct reading frame to the PSE protein-<br>
PSE homologs can be generated by mutagenesis, for example by specific point mutation or truncating the PSE. The term "homologs" as used in the present context refers to a variant form of PSE which acts as agonist or antagonist of the pSE activity. A PSE agonist can essentially retain the same activity as PSE, or some of the biological activities. A PSE antagonist can inhibit one or more activities of the naturally occurring PSE form, for example by competitive binding to an upstream or downstream element of the metabolic cascade for cell membrane components which encompasses the PSE, or by binding to a PSE which mediates the transport of compounds via cell membranes, thus inhibiting translocation.<br>
In an alternative embodiment, PSE homologs can be identified by screening combinatory libraries of mutants, for example truncated mutants, of pSE with regard to pSE agonist or<br>
PSE antagonist activity. In one embodiment, a variegated library of PSE variants is generated at the nucleic acid level by combinatory mutagenesis and encoded by a variegated genetic library. A variegated library of PSE variants can be generated for example by enzymatic ligation of a mixture of synthetic oligonucleotides into gene sequences so that a degenerate set of potential PSE sequences can be expressed as individual polypeptides or, alternatively, as a set of larger fusion proteins (for example for phage display) which comprise this set of.  PSE sequences. There is a multiplicity of methods which can be jsed for generating libraries of potential PSE homologs from a Regenerate oligonucleotide sequence. The chemical synthesis of a degenerate gene sequence can be carried out in a DNA synthesizer, and the synthetic gene can then be ligated into a suitable expression vector. The use of a degenerate set of genes allows all sequences which encode the desired set of potential PSE sequences to be provided in a mixture. Methods for the synthesis ■}f  degenerate oligonucleotides are known in the art [see, for 3xample, Narang, S.A. (1983) Tetrahedron 39:3; Itakura et al. (1984) Annu. Rev. Biochem. 53:323; Itakura et al., (1984) Science 198:1056; Ike et al. (1983) Nucleic Acids Res. 11:477].<br>
[n addition, libraries of PSE fragments can be used for jenerating a variegated population of PSE fragments for screening ind for the subsequent selection of homologs of a pSE. In one embodiment, a library of fragments of the coding sequence can be generated by treating a double-stranded PCR fragment of a coding PSE sequence with a nuclease under conditions under which<br><br>
—-^^^-ij^j.a,iiu. wi-eat^s onxy occur approximately once per molecule, denaturing the double-stranded DNA, renaturing the DKA with the formation of double-stranded DNA which can encompass sense/antisense pairs of various products with double-strand breaks, removal of single-stranded sections from newly formed duplices by treatment with SI nuclease, and ligating the resulting fragment library into an expression vector. This method allows an expression library to be derived which encodes K-terminal, C-terminal and internal fragments of variously sized PSEs.<br>
A number of techniques for screening gene products in combinatory libraries which have been generated by point mutations or truncation and for screening cDNA libraries for gene products with a selected property are known in the art. These techniques can be adapted to rapid screening of the genetic libraries which have been generated by combinatory mutagenesis of PSE homologs. The most frequently used techniques for screening large genetic libraries which can be subjected to high-throughput analysis usually encompass cloning the genetic library into replicable sxpression vectors, transforming suitable cells with the resulting vector library, and expressing the combinatory genes under conditions under which detecting the desired activity facilitates the isolation of the vector encoding the gene whose oroduct has been detected. Recursive ensemble mutagenese (REM), a :iovel technique which increases the frequency of functional nutants in the libraries, can be used in combination with the screening assays for identifying PSE homologs [Arkin and Yourvan<br>
(1992)	Proc. Natl. Acad. Sci. USA 89:7811-7815; Delgrave et al.<br>
(1993)	Protein Engineering 6(3):327-331]. Combinations of the ibovementioned methods can also be used advantageously.<br>
k  further known technique for modifying catalytic properties of jnzymes or the genes encoding them is gene shuffling (see, for ixample, WO 97/20078 or WO 98/13487), which is a combination of jene fragments where this new combination can additionally be varied by erroneous polymerase chain reactions thus creating a ligh sequence diversity to be assayed. However, the prerequisite :or using such an approach is a suitable screening system to test ;he resulting gene diversity for functionality, i. screening method which identifies a PUFA-dependent enzyme ictivity, or activities, is a prerequisite in particular for icreening elongase activities. As regards elongase activities rith a specificity for PUFAs, the toxicity of arachidonic acid in :he presence of a toxic metabolite (here: salicylic acid or salicylic acid derivatives), can be exploited in Hucor species 7hich can be transformed with desired gene constructs by known<br><br>
transformation methods (Eroshin et al., Mikrobiologiya, Vol. 63, NO.l, 1996, pages 31-36) to carry out a growth-based primary screening. Resulting clones can then be analyzed for their lipid constituents by means of gas chromatography and mass spectroscopy in order to identify the nature and quantity of starting materials and products.<br>
In a further embodiment, cell-based assays can be made use of for analyzing a variegated PSE library using further processes known in the art.<br>
In a further embodiment, the present invention relates to an antibody which binds specifically to the polypeptide of the present invention or to parts, for example epitopes, of such a protein. The antibody according to the invention can be used to identify and isolate other elongases, in particular PSEs. These antibodies can be monoclonal antibodies, polyclonal antibodies or synthetic antibodies, and also fragments of these antibodies, such as, for example. Fab, Fv or scFV fragments, etc. Monoclonal antibodies can be prepared, for example, by methods such as those described originally by Kbhler and Milstein in Nature 256 (1975), 485, and Galfro in Meth. Enzymol. 73 (1981). Antibodies and fragments thereof can also be prepared, for example, according to Harlow &amp; Lane, "Antibodies, a Laboratory Manual", CSH Press, Cold Spring Harbor, 1988. These antibodies can be used to precipitate and localize, for example, the proteins according to the invention, or to monitor the synthesis of these proteins, for sxample in recombinant organisms, and to identify compounds which Interact with the proteins according to the invention. In many rases, the binding of antibodies to antigens is equivalent to the binding of other ligands and antiligands.<br>
?he present invention furthermore relates to a process for .dentifying an agonist or antagonist of elongases, in particular 'SEs, comprising<br>
I)  bringing the cells which express the polypeptide of the<br>
present invention into contact with a candidate substance;<br>
i)  testing the PSE activity;<br>
;)  comparing the PSE activity with a standard activity in the<br>
absence of the candidate substance, where a PSE activity that is higher than that of the standard indicates that the candidate substance is an agonist and where a PSE activity<br><br>
that is lower than that of the standard indicates that the candidate substance is an antagonist.<br>
The candidate substance mentioned can be a substance that is synthesized chemically or produced microbiologically, being present, for example, in cell extracts of, for example, plants, animals or microorganisms. The substance mentioned may furthermore be known in the prior art but hitherto unknown as increasing or repressing the activity of the PSEs. The reaction mixture may be a cell-free extract or comprise a cell or cell culture. Suitable methods are known to the person skilled in the art and described in a general manner, for example, in Alberts, Molecular Biology of the cell, 3^^  edition (1994), for example Chapter 17. The substances mentioned may be added, for example, to the reaction mixture or the culture medium or be injected into the cells or sprayed onto a plant.<br>
If a sample which comprises a substance which is active according to the method according to the invention has been identified, it is then possible either to isolate the substance directly from the original sample or to divide the sample into various groups, for example when it consists of a large number of different components, to reduce the number of different substances per sample, and then to repeat the process according to the invention with such a "subsample" of the original sample. Depending on the complexity of the sample, the steps described above can be repeated a plurality of times, preferably until the sample identified by the method according to the invention comprises only a small number of substances or only one substance. The substance identified by the method according to the invention, or derivatives thereof, are preferably formulated further, such that they are suitable for use in plant breeding or plant cells or tissue culture.<br>
Substances which can be identified and tested by the process according to the invention can be: expression libraries, for Bxample cDNA expression libraries, peptides, proteins, nucleic acids, antibodies, small organic substances, hormones, PNAs or :he like (Milner, Nature Medicin 1 (1995), 879-880; Hupp, Cell. J3 (1995), 237-245; Gibbs, Cell. 79 (1994), 193-198 and Literature quoted therein). These substances can also be ■unctional derivatives or analogs of known inhibitors or ictivators. Processes for preparing chemical derivatives or inalogs are known to the person skilled in the art. The known derivatives and analogs can be tested using processes of the prior art. It is furthermore possible to use computer-aided design or peptidomimetics to prepare suitable derivatives and<br><br>
analogs. The cell or the tissue used for the process{es) according to the invention is preferably a host cell, plant cell or a plant tissue according to the invention, as described in the embodinients above.<br>
Correspondingly, the present invention also relates to a substance which was identified by the process according to the invention mentioned above. The substance is, for example, a homolog of the PSE according to the invention. Homologs of the PSEs can be generated by mutagenesis, for example by point mutation or deletion of PSE. Here, the term "homolog" is used to mean a variant form of the PSEs which acts as agonist or antagonist for the PSE activity. An agonist may have substantially the same or part of the biological activity of the PSEs. An antagonist of the PSEs may inhibit one or more activities of the naturally occurring forms of the PSEs, for example bind competitively to a downstream or upstream member of the metabolic paths of fatty acid synthesis including the PSEs, or bind to PSEs and reducing or inhibiting the activity in the process.<br>
Accordingly, the present invention also relates to an antibody or a fragment thereof as described herein which inhibits the activity of the PSEs according to the invention.<br>
One aspect of the present invention relates to an antibody which specifically recognizes or binds to the above-described agonist or antagonist according to the invention.<br>
A further aspect relates to a composition which comprises the antibody, the stop or the antisense molecule identified by the process according to the invention.<br>
E.  Uses and processes/methods according to the invention<br>
The nucleic acid molecules, proteins, protein homologs, fusion proteins, antibodies, primers, vectors and host cells described herein can be used in one or more of the methods which follow: identification of Physcomitrella patens, Crypthecodinium, Phytophthora infestans or Thraustochytrium and related organisms, genome mapping of organisms which are related with Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium, identification and localization of Physcomitrella, phytophthora, Crypthecodinium or Thraustochytrium sequences of interest, evolutionary studies, determination of PSE protein regions required for the function, modulation of a PSE activity; modulation of the metabolism of one or more cell<br><br>
membrane components; modulation of the transmembrane transport of one or more compounds, and modulation of the cellular production of a desired compound such as a fine chemical. The PSE nucleic acid molecules according to the invention have a multiplicity of uses. Firstly, they can be used for identifying an organism as Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium or a close relative of these. They can also be used for identifying the presence of Physcomitrella, Crypthecodinium, Phytophthora or Thraustochytrium or a relative of these in a mixed population of microorganisms. The invention provides the nucleic acid sequences of a series of Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium genes; the presence or absence of this organism can be determined by screening the extracted genomic DNA of a culture of a uniform or mixed population of microorganisms under stringent conditions with a probe covering a region of a Physcoroitrella, Crypthecodinium, Phytophthora or Thraustochytrium gene which is unique for this organism, or of parts of this gene. Physcomitrella, Crypthecodinium, Phytophthora or Thraustochytrium themselves are used for the commerical production of polyunsaturated acids. Moreover, the nucleic acids according to the invention are suitable for the production of PUFAs, also in other organisms, in particular when it is intended for the resulting PUFAs to be incorporated into the triacylglycerol fraction.<br>
Furthermore, the nucleic acid and protein molecules according to the invention can act as marker for specific regions of the genome. They are not only suitable for mapping the genome, but also for functional studies of Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium proteins. To identify the genome region to which a certain DHA-binding protein of Physcomitrella, Crypthecodinium, Phytophthora or Thraustochytrium binds, the Physcomitrella, Crypthecodinium, Phytophthora or Thraustochytrium genome can be fragmented, for example, and the fragments incubated with the DHA-binding protein. Those which bind the protein can additionally be screened with the nucleic acid molecules according to the invention, preferably with readily detectable markers; the binding of such a nucleic acid molecule to the genome fragment makes possible the localization of the fragment on the genome map of Physcomitrella, Phytophthora, Crypthecodinium or Thraustochytrium and, if this is carried out repeatedly with different enzymes, facilitates a rapid determination of the nucleic acid sequence to which the protein binds. Moreover, the nucleic acid molecules according to the invention can have sufficient homology with the sequences of related species for these nucleic acid molecules to be able to<br><br>
act as markers for the construction of a genomic map in related fungi or algae.<br>
The PSE nucleic acid molecues according to the invention are also suitable for evolutionary studies and studies of the protein structure. The metabolic and transport processes in which the molecules according to the invention are involved are utilized by a large number of prokaryotic and eufcaryotic cells; the evolutionary degree of relatedness of the organisms can be determined by comparing the sequences of the nucleic acid molecules according to the invention which those which encode similar enzymes from other organisms. Accordingly, such a comparison allows the determination of which sequence regions are conserved and which are not conserved, which may be helpful when determining regions of the protein which are essential for enzyme function. This type of determination is valuable for protein engineering studies and may provide a clue of how much mutagenesis the protein can tolerate without losing its function.<br>
Manipulation of the PSE nucleic acid molecules according to the invention can lead to the production of PSEs with functional differences to the wild-type PSEs. The efficacy or activity of these proteins can be improved; they may be present in the cell in larger numbers than usual; or their efficacy or activity can be reduced. An improved efficacy or activity means, for example, that the enzyme has a higher selectivity and/or activity, preferably an activity which is at least 10% higher, very especially an activity which is at least 20% higher, very especially preferably an activity which is at least 30% higher, than the original enzyme.<br>
There exists a series of mechanisms by which modification of a PSE according to the invention can directly affect yield, production and/or production efficacy of a fine chemical comprising such a modified protein. Obtaining fine chemical compounds from cultures of ciliates, algae or fungi on a large scale is significantly improved when the cell secretes the desired compounds, since these compounds can readily be isolated from the culture medium (in contrast to extraction from the biomass of the cultured cells). Otherwise, purification can be improved when the cell stores compounds in vivo in a specialized compartment with a sort of concentration mechanism. In plants which express PSEs, an increased transport may lead to better distribution within the plant tissue and the plant organs. Increasing the number or the activity of the transporter molecules which export fine chemicals from the cell may allow the quantity of the fine chemical produced, which is present in the<br><br>
extracellular medium, to be increased, thus facilitating harvesting and purification or, in the case of plants, more efficient distribution. In contrast, increased amounts of cofactors, precursor molecules and intermediates for the suitable biosynthetic pathways are required for an efficient overproduction of one or more fine chemicals. Increasing the number and/or the activity of transporter proteins involved in the import of nutrients such as carbon sources (i.e. sugars), nitrogen sources {i.e. amino acids, ammonium salts), phosphate and sulfur can improve the production of a fine chemical owing to the elimination of all limitations of the nutrients available in the biosynthetic process. Fatty acids such as PUFAs and lipids comprising PUFAs are desirable fine chemicals themselves. Optimizing the activity or increasing the number of one or more PSEs according to the invention involved in the biosynthesis of these compounds, or disrupting the activity of one or more PSEs involved in the breakdovm of these compounds, can thus increase the yield, production and/or production efficacy of fatty acid and lipid molecules in ciliates, algae, plants, fungi, yeasts or other microorganisms.<br>
The manipulation of one or more PSE genes according to the invention can likewise lead to PSEs with modified activities ■which indirectly affect the production of one or more desired fine chemicals from algae, plants, ciliates or fungi. The normal biochemical metabolic processes lead, for example, to the production of a multiplicity of waste products (for example hydrogen peroxide and other reactive oxygen species) which can actively disrupt these metabolic processes [for example,<br>
' peroxynitrite is known to nitrate tyrosin side chains, thus inactivating some enzymes with tyrosin in the active center; Groves, J.T. (1999) Curr. Opin. Chem. Biol. 3(2);226-235]. While these waste products are normally excreted, the cells used for fermentative production on a large scale are optimized for the<br>
' overproduction of one or more fine chemicals and can therefore produce more waste products than is customary for a wild-type cell. Optimizing the activity of one or more PSEs according to the invention which are involved in the export of waste molecules allows the improvement of the viability of the cell and the maintenance of an efficient metabolic activity. Also, the presence of high intracellular amounts of the desired fine chemical can in fact be toxic to the cell, so that the viability of the ceil can be improved by increasing the ability of the cell to secrete these compounds.<br><br>
Furthermore, the PSEs according to the invention can be manipulated in such a way that the relative amounts of various lipid and fatty acid molecules are modified. This can have a decisive effect on the lipid composition of the cell membrane. Since each lipid type has different physical properties, the modification of the lipid composition of a membrane can significantly modify membrane fluidity. Changes in membrane fluidity can affect the transport of molecules via the membrane which, as explained above, can modify the export of waste products or of the fine chemical produced or the import of nutrients which are required. These changes in membrane fluidity can also have a decisive effect on cell integrity; cells with comparatively weaker membranes are more susceptible to abiotic and biotic stress conditions which can damage or kill the cell. Manipulation of PSEs which are involved in the production of fatty acids and lipids for membrane synthesis so that the resulting membrane has a membrane composition which is more susceptible to the environmental conditions prevailing in the cultures used for the production of fine chemicals should allow more cells to survive and multiply. Larger numbers of producing cells should manifest themselves in greater yields, higher production or higher production efficacy of the fine chemical from the culture.<br>
The abovementioned mutagenesis strategies for PSEs intended to lead to elevated yields of a fine chemical are not to be construed as limiting; variations of these strategies are readily obvious to the skilled worker. Using these mechanisms, and with the aid of the mechanisms disclosed herein, the nucleic acid and protein molecules according to the invention can be used for generating algae, ciliates, plants, animals, fungi or other microorganisms such as C. glutamicum which express mutated PSE nucleic acid and protein molecules so that the yield, production and/or production efficacy of a desired compound is improved. This desired compound can be any natural product of algae, ciliates, plants, animals, fungi or bacteria which encompasses the end products of biosynthetic pathways and intermediates of naturally occurring metabolic pathways, and also molecules which do not naturally occur in the metabolism of these cells, but which are produced by the cells according to the invention.<br>
A further embodiment according to the invention is a process for the production of PUFAs, which comprises culturing an organism which contains a nucleic acid according to the invention/ a gene construct according to the invention or a vector according to the invention which encode a polypeptide which elongates Ci(,~i   Cig- or C2o-fatty acids with at least two double bonds in the fatty acid<br><br>
rnolecuie by at least two carbon atoms under conditions under which PUFAs are produced in the organism. PUFAs prepared by this process can be isolated by harvesting the organisms either from the culture in which they grow or from the field, and disrupting and/or extracting the harvested material with an organic solvent. The oil, which contains lipids, phospholipids, sphingolipids, glycolipids, triacylglycerols and/or free fatty acids with a higher PUFA content, can be isolated from this solvent. The free fatty acids with a higher content of PUFAs can be isolated by basic or acid hydrolysis of the lipids, phospholipids, sphingolipids, glycolipids and triacylglycerols. A higher content of PUFAS means at least 5%, preferably 10%, especially preferably 20%, very especially preferably 40% more PUFAs than the original organism which does not have additional nucleic acid encoding the elongase according to the invention.<br>
The PUFAs produced by this process are preferably Cig-, C20- or C22-fatty acid molecules with at least two double bonds in the fatty acid molecule, preferably three, four, five or six double bonds, especially preferably three or five double bonds. These C18-. ^20- or C22-fatty acid molecules can be isolated from the organism in the form of an oil, lipid or a free fatty acid. Examples of suitable organisms are those mentioned above. Preferred organisms are microorganisms, animals or plants, especially preferably plants or algae, very especially preferably transgenic plants.<br>
An eznbodiment according to the invention are oils, lipids or fatty acids or fractions thereof which have been prepared by the above-described process, especially preferably oil, lipid or a fatty acid composition encompassing PUFAs and originating from transgenic plants.<br>
One embodiment of the invention are oils, lipids or fatty acids which have been prepared by the process according to the invention. Other embodiments of the invention are oil, lipid or fatty acid compositions which comprise PUFAs produced by the process according to the invention and which are derived from transgenic plants which comprise a nucleic acid, a gene ' construct or vector according to the invention.<br>
A further embodiment according to the invention is the use of the oil, lipid or the fatty acid composition in feeding stuffs, foodstuffs, cosmetics or pharmaceuticals.<br><br>
A further embodiment of the invention relates to a kit, comprising the nucleic acid according to the invention, the gene construct according to the invention, the amino acid sequence according to the invention, the antisense nucleic acid molecule according to the invention, the antibody and/or composition according to the invention, an antagonist or agonist prepared by the process according to the invention and/or oils, lipids and/or fatty acids according to the invention, or a fraction thereof. The kit may also comprise the host cells, organisms or plants according to the invention, or parts thereof, parts of the plants according to the invention which can be harvested, or propagation material, or else the antagonist or agonist according to the invention. The components of the kit of the present invention can be packed in suitable containers, for example together with or in buffers or other solutions. One or more of the components mentioned may be packed into one and the same container. Additionally or alternatively, one or more of the components mentioned can, for example, be absorbed on a solid surface, for example a nitrocellulose filter, glass plates, chips, nylon membranes or microtiter plates. The  kit can be used for any of the methods and embodiments described herein, for example for producing host cells, transgenic plants, for detecting homologous sequences, for identifying antagonists or agonists, and the like. Furthermore, the kit may contain instructions on how to use the kit for one of the applications mentioned.<br>
This invention is illustrated in greater detail by the examples which follow, which are not to be construed as limiting. The content of all references, patent applications, patents and published patent applications cited in this patent application is incorporated herein fay reference.<br>
Examples<br>
Example 1:  General methods<br>
a)  General cloning methods:<br>
Cloning methods, such as, for example, restriction cleavages, agarose gel electrophoresis, purification of DNA fragments, transfer of nucleic acids to nitrocellulose and nylon membranes, linkage of DNA fragments, transformation of Escherichia coli and yeast cells, the culture of bacteria and the sequence analysis of recombinant DNA were carried out as described in Sambrook et al. [(1989), Cold Spring Harbor Laboratory Press: ISBN 0-87969-309-6] or Kaiser, Michaelis and Mitchell [(1994), "Methods in Yeast Genetics", Cold Spring Harbor Laboratory Press: ISBN<br><br>
0-87969-451-3]. The transformation and culture of algae such as Chlorella or Phaeodactylum are carried out as described by El-Sheekh ((1999), Biologia Plantarum 42:209-216] or Apt et al. [(1996) Molecular and General Genetics 252 (5):872-9].<br>
b)	Chemicals<br>
Unless otherwise specified in the text, the chemicals used were obtained in analytical grade quality from Fluka (Neu-Ulm), Merck (Darmstadt), Roth (Karlsruhe), Serva (Heidelberg) and Sigma (Deisenhofen). Solutions were prepared using pure pyrogen-free water, referred to in the following text as H2O, from a Milli-Q water system water purification unit (Millipore, Eschborn). Restriction endonucleases, DNA-modifying enzymes and molecular biology kits were obtained from AGS (Heidelberg), Amersham (Brunswick), Biometra (Gottingen), Boehringer (Mannheim), Genomed (Bad Oeynhausen), New England Biolabs (Schwalbach/Taunus), Novagen (Madison, Wisconsin, USA), Perkin-Elmer (Weiterstadt), Pharmacia (Freiburg), Qiagen (Hilden) and Stratagene (Amsterdam, Netherlands). Unless otherwise specified, they were used following the manufacturer's instructions.<br>
c)	Cell material<br>
The present studies were carried out using a Thraustochytrium strain which is available via the American Type culture Collection (ATCC) with the strain number ATCC26185 (Thraustochytrium) or, in the case of Crypthecodinium, from the Provasoli-Guillard National Center for Culture of Marine Phytoplankton ((CCMP) West Boothbay Harbour, ME, USA), with the strain culture No. CCMP316. The algae were cultured in the dark at 25 degrees Celsius in glass tubes into which air was passed in from the bottom. As an alternative, Thraustochytrium was grown as described in detail by Singh &amp; ward (1997, Advances in Microbiology, 45, 271-311).<br>
The culture medium used for Crypthecodinium was the f/2 culture medium supplemented with 10% organic medium of Guillard, R.R.L. [1975; Culture of phytoplankton for feeding marine invertebrates. In: Smith, W.L. and Chanley, M.H. (Eds.) Culture of marine Invertebrate animals, NY Plenum Press, pp. 29-60.]. It comprises<br>
995.5 ml of (artificial) salt water<br>
1 ml NaNOs (75 g/1), 1 ml NaH2P04 (5 g/1), 1 ml trace metal<br>
solution, 1 ml Tris/Cl pH 8.0, 0.5 ml f/2 vitamin solution<br><br>
Trace metal solution: Na2EDTA (4.36  g/1}, FeCls (3.15 g/1).<br>
Primary Trace metals: CUSO4 (10 g/1), ZnS04 (22 g/1), C0CI2<br>
(10 g/1), MnCl2 (18 g/1), NaMo04 (6.3 g/1)<br>
f/2 vitamin solution: biotin: 10 mg/1, thiamin 200 mg/1, vitamin<br>
B12 0.1 mg/1<br>
org medium: sodium acetate (1 g/1), glucose (6 g/1), sodium<br>
succinate (3 g/1), Bacto-tryptone {4 g/1), yeast extract (2 g/1)<br>
d)	Moss material (= plant material)<br>
For this study, plants of the species Physcomitrella patens (Hedw.) B.S.G. from the collection of the department for genetic studies. University of Hamburg, were used. They are derived from strain 16/14, which had been collected by H.L.K. Whitehouse in Gransden Wood, Huntingdonshire (England) and subcultured by Engel (1968, Am J Hot 55, 438-446) from a spore. Proliferation of the plants was done by means of spores and by regenerating the gametophytes. The protonema developed from the haploid spore into chloroplast-rich chloronema and chloroplast-depleted caulonema, which budded after approximately 12 days. These buds grew into gametophores with antheridia and archegonia. Fertilization gave rise to diploid sporophyte with short seta and spore capsule in which the meiospores mature.<br>
e)	Plant culture<br>
Plants were grown in a controlled-environment cabinet at an air temperature of 25°C and a light intensity of 55 ^mol-s-^.m-^ (white light; Philips TL 65W/25 fluorescent tube) and a light/dark region of 16/8 hours. The moss was grown either in liquid culture using Reski and Abel's modified knop medium (1985, Planta 165, 354-358) or on solid knop medium using 1% Oxoid agar (Unipath, Basingstoke, England).<br>
The protonemata used for RNA and DNA isolation were grown in aerated liquid cultures. The protonemata were comminuted every 9 days and transferred into fresh culture medium.<br>
f)	Cultivation of Phytophthora infestans<br>
Initially, a cDKA library of Phytophthora infestans was prepared-To this end, it is possible to obtain strain ATCC 48886 from the American Type Culture Collection Rockville, USA. As a variation of the culture protocol described for the strain ATCC 48S86, Phytophthora spores were washed with cold, doubly distilled water and kept in a fridge for 6 hours to induce sporulation. The material was then transferred into pea medium. To this end, 150 g<br><br>
of deep-frozen peas (Igiu, obtainable from local supermarkets) were autoclaved under sterile conditions and I liter of water for 20 minutes. The material was grown in 100-ml-flasks at room temperature, on an orbitalshaker (200 rpm). After two days, 2 flasks were filtered off and the filter residue was comminuted in liquid nitrogen, using mortar and pestle, and for the following 4 days, this procedure was repeated for in each case 2 flasks.<br>
Example 2:   Isolation of total DNA from plants and microorganisms such as Thraustochytrium and Crypthecodinium for hybridization experiments<br>
The details on the isolation of total DNA refer to the work-up of plant material with a fresh weight of one gram.<br>
CTAB buffer: 2% (w/v) N-acetyl-N,N,N-trimethylairanonium bromide (CTAB); 100 mM Tris-HCl, pH 8.0; 1.4 M NaCl; 20 mM EDTA.<br>
K-Laurylsarcosine buffer: 10% (w/v) N-laurylsarcosine; 100 mM Tris-HCl, pH 8.0; 20 mM EDTA.<br>
The plant material or Crypthecodinium or Thraustochytrium cell material was triturated under liquid nitrogen in a mortar to give a fine powder and transferred into 2 ml Eppendorf vessels. The frozen plant material was then covered with a layer of 1 ml of decomposition buffer (1 ml CTAB buffer, 100 ml K-laurylsarcosine buffer, 20 ml B-mercaptoethanol and 10 ml proteinase K solution, 10 mg/ml) and incubated at 60 °C for one hour with continuous shaking. The homogenate obtained was distributed into two Eppendorf vessels (2 ml) and extracted twice by shaking with an equal volume of chloroform/isoamyl alcohol (24:1). For phase separation, centrifugation was carried out at 8000 x g and RT (= room temperature = ~ 23°C) for 15 minutes in each case. The DNA was then precipitated at -70°c for 30 minutes using ice-cold isopropanol. The precipitated DNA was sedimented at 4°C and 10,000 g for 30 minutes and resuspended in 180 ml of TE buffer (Sambrook et al., 1989, Cold Spring Harbor LaJDoratory Press: ISBN 0-87969-309-6). For further purification, the DNA was treated with NaCl (1.2 M final concentration) and precipitated again at -70°C for 30 minutes using twice the volume of absolute ethanol. After a wash step with 70% ethanol, the DNA was dried and subsequently taken up in 50 ml of H2O + RNase (50 mg/ml final concentration). The DWA was dissolved overnight at 4°C and the RNase cleavage was subsequently carried out for  1 hour at 37'c. The DNA was stored at 4°C.<br><br>
Exajnple 3:   Isolation of total RNA and poly(A)"^-RKA from plants and microorganisms (Crypthecodinium and Thraustochytrium)<br>
Total RHA was isolated from plants such as linseed and oilseed rape by a method described by Logemann et al (1987, Anal. Biochem. 163, 21). The total RNA from moss can be obtained from protonema tissue using the GTC method (Eeski et al,, 2994, Mol, Gen. Genet., 244:352-359).<br>
RNA isolation from Crypthecodinium and Thraustochytrium:<br>
Frozen samples of algae (- 70°C) are triturated in an ice-cold mortar under liquid nitrogen to give a fine powder. 2 volumes of homogenization medium (12.024 g sorbitol, 40.0 ml IM Tris-HCl, pH 9 (0.2 M); 12.0 ml 5 M NaCl (0.3 H), 8.0 ml 250 mM EDTA, 761.0 mg EGTA, 40.0 ml 10% SDS are made up to 200 ml with H2O and the pH is brought to 8.5) and 4 volumes of phenol with 0.2% mercaptoethanol are added to the frozen cell powder at 40-50°C while mixing thoroughly. Then, 2 volumes of chloroform are added and the mixture is stirred vigorously for 15 minutes. The mixture is centrifuged for 10 minues at 10,000 g and the aqueous phase is extracted with phenol/chloroform (2 vol/2 vol) and then with chloroform.<br>
The resulting volume of the aqueous phase is treated with 1/20 volume of 4 M sodiiom acetate (pH 6) and 1 volume of (ice-cold) isopropanol, and the nucleic acids are precipitated at -20°C. The mixture is centrifuged for 30 minutes at 10,000 g and the supernatant is removed by suction. This is followed by a wash step with 70% EtOH and another centrifugation step. The sediment is taken up in Tris borate buffer (80 mM Tris borate buffer, 10 mM EDTA, pH 7.0). The supernatant is then treated with 1/3 vol of 8 M LiCl, mixed and incubated for 30 minutes at 4°C. After recentrifugation, the sediment is washed with 70% ethanol, ::entrifuged, and the sediment is dissolved in RNase-free water.<br>
Poly(A)+-RHA is isolated using Dyna Beads® (Dynal, Oslo, Finland) following the instructions in the manufacturer's protocol.<br>
^fter the RNA or poly(A)*-RNA concentration has been determined, the RNA is precipitated by adding 1/10 volume 3 M sodium acetate, pH 4.6 and 2 volumes of ethanol and stored at -70'^c.<br>
For the analysis, 20 [ig portions of RNA are separated in a formaldehyde-containing 1.5% strength agarose gel and transferred to nylon membranes (Hybond, Amersham). Specific transcripts are<br><br>
detected as described by Amasino ((1986) Anal. Biochem. 152, 304)).<br>
Isolation of total RMA and poly-(A)+ RNA from Phytophthora infestans:<br>
Total RNA was obtained using the RNeasy Plant Total RNA kit (Quiagen, Milden) and the buffer contained therein, following the instructions of the manufacturer. From the total RNA thus obtained, the poly-(A)+ RNA was isolated using the Poly Attract in RNA Isolation System III from Promega (Heidelberg), following the instructions of the manufacturer.<br>
Example 4: Construction of the cDNA library<br>
To construct the cDNA library from Physcomitrella, Crypthecodinium and Thraustochytrium, respectively, the first-strand synthesis was carried out using murine leukemia virus reverse transcriptase (Roche, Mannheim, Germany) and oligo-d{T) primers, while the second-strand synthesis was carried out by incubation with DNA polymerase I, Klenow enzyme and cleavage with RKAse H at 12oc (2 hours), le^C (1 hour) and 22°C (1 hour). The reaction was quenched by incubation at 65°C (iO minutes) and subsequently transferred to ice. Double-stranded DNA molecules were made blunt-ended with T4 DNA polymerase (Roche, Mannheim) at 37°C (30 minutes). The nucleotides were removed by extraction with phenol/chloroform and Sephadex G50 spin columns. EcoRI/Xhol adapters (Pharmacia, Freiburg, Germany) were ligated to the cDNA ends by means of T4 DNA ligase (Roche, 12°C, overnight), recut with Xhol and phosphorylated by incubation with polynucleotide kinase (Roche, 37^0, 30 minutes). This mixture was subjected to separation on a low-melting agarose gel. DNA molecules of over 300 base pairs were eluted from the gel, extracted with phenol, concentrated on Elutip D columns (Schleicher and Schiill, Dassel, Germany), ligated to vector arms and packaged into lambda-ZAPII phages or lambda-ZAP-express phages using the Gigapack Gold kit {Stratagene, Amsterdaim, the Netherlands), using the manufacturer's material and following their instructions.<br>
The construction of a cDNA library for Phytophthora infestans was carried out as described above.<br><br>
Example 5: DNA sequencing and computer analysis<br>
cDHA libraries as described in Example 4 were used for DNA sequencing by standard methods, in particular the chain termination method using the ABI PRISM Big Dye Terminator Cycle Sequencing Ready Reaction Kit (Perkin-Elmer, Weiterstadt, Germany). Following the plasmid preparation from cDNA libraries, individual random clones were sequenced via in-vivo mass excision and retransformation of DHIOB on agar plates (details on materials and protocol: Stratagene, Amsterdam, the Netherlands). Plasmid DNA was prepared from E. coli cultures grown overnight in Luria broth supplemented with ampicillin (see Sambrook et al. (1989) (Cold Spring Harbor Laboratory Press: ISBN 0-87969-309-6)) using a Qiagen DNA preparation robot (Qiagen, Hilden) following the manufacturer's protocols. Sequencing primers with the following nucleotide sequences were used:<br>
5'-CAGGAAACAGCTATGACC-3' 5'-CTAAAGGGAACAAAAGCTG-3' 5'-TGTAAAACGACGGCCAGT-3'<br>
The sequences were processed and recorded using the EST-MAX standard software package which is commercially available from Bio-Max (Munich, Germany). Exploiting comparative algorithms and using a searching sequence, homologous genes were searched for using the BLAST program (Altschul et al. (1997) "Gapped BLAST and PSI-BIAST: a new generation of protein database search programs". Nucleic Acids Res. 25:3389-3402). One sequence from Crypthecodinium and Thraustochytrium with homologies with the search sequence of the Physcomitrella patens moss elongase were characterized in greater detail.<br>
Example 6a:  Identification of the Tc_PSEl and Tc_PSE2 gene (Tc = Thraustochytrium) and of the Cc_PSEl and Cc_pSE2 gene (Cc = Crypthecodinium cohnii) by comparison with the Physcomitrella patens Pp_PSEl gene.<br>
The full-length sequence of the Pp_PSEl moss elongase according to the invention (name: see also Table 2) was employed for the sequence comparisons in the TBLASTN search algorithm:<br>
MEWERFYGE LDGKVSQGVN ALLGSFGVEL TDTPTTKGLP LVDSPTPIVL GVSVYLTIVI GGLLWIKARD LKPRASEPFL LQALVLVHNL FCFALSLYMC VGIAYQAITW RVSLWGHAYH PKHKEMAILV YLFYMSKYVE FMDTVIMILK RSTRQISFLH VYHHSSISLI WWAIAHHAPG<br><br>
GEAYWSAALN SGVHVLMYAY YFLAACLRSS PKLKNKVLFW GRYLTQFQMF QFMLNLVQAY YDMKTNAPYP QWLIKILFYY MISLLFLFGH FYVQKYIKPS DGKQKGAKTE.<br>
The complete nucleotide sequence of the moss elongase Pp_PSEl CDNA is composed of approximately 1200 bp. It contains an open reading frame of 873 bp which encodes 290 amino acids with a calculated molecular mass of 33.4 Da. The protein sequence only has 38.5% identity and 48-3% similarity with a Saccharomyces cerevisiae gene product, for example the Saccharomyces cerevisiae PSEl gene product, which is required in yeast for the elongation of fatty acids with medium chain length (Toke &amp; Martin, 1996, Isolation and characterization of a gene affecting fatty acid elongation in Saccharomyces cerevisiae. Journal of Biological Chemistry 271, 18413-18422).<br>
The EST sequences CC001042041R, TC002034029R and TC002014093R were first considered as target gene amongst other candidate genes owing to initially weak homologies with the Physcomitrella patens elongase (see Table 2),   the PSEl gene. Figure 5 shows the result of the comparison of the Pp_PSEl peptide sequence with the found sequence. It is part of the nucleic acid of Seq ID N0:3 according to the invention (gene name: rcPSEl, inventors' database No. TC002034029R). Letters indicate identical amino acids, while the plus symbol denotes a chemically similar amino acid. The identities and homologies of all sequences found in accordance with the invention can be seen from the summary in Table 3.<br>
Sequencing of the complete cDNA fragment from clone TC002034029R resulted in a sequence of 693 base pairs starting with the first base in the open reading frame. The sequence encodes a polypeptide of 195 amino acids shown in Seq ID NO: 4 with a stop codon in base pair position translated from Seq ID NO:3 in base pair position 586-588. Clone TC002014093R comprises a virtually complete elongase polypeptide as can be seen from the sequence alignment in Figure 7. Lines denote identical amino acids, while colons and dots represent chemically exchangeable, i.e. chemically equivalent, amino acids- The alignment was carried out using Henikoff &amp; Henikoff's BLOSUM62 amino acid substitution matrix  ((1992) Amino acid substitution matrices from protein blocks.  Proc. Natl. Acad. Sci. USA 89: 10915-10919). Parameters used: Gap Weight: 8; Average Match: 2.912, Length weight: 2, Average Mismatch: -2.003-<br>
Furthermore, a second EST was identified by the sequence alignment. The alignment of the Pp_PSEl peptide sequence with the found sequence is shown in Figure 6. Even though the homology<br><br>
amongst the parameters chosen is restricted to a few Eimino acids, this refers to a highly conserved region of the PUFA specific elongases. The sequence of the complete cloned fragment was therefore determined.<br>
sequencing of the complete cDNA fragment of clone TC002014093R resulted in a sequence of 955 base pairs starting with the first oase in the open reading frame. This is referred to by SEQ ID HO:5 according to the invention. The sequence encodes a polypeptide of 297 amino acids with a stop codon in base pair position 892-894 shown in accordance with the invention in SEQ ID NO: 6.<br>
The Crypthecodinium cohnii EST CC001042041R which encodes the Cc_PSEl gene was identified with the aid of the sequence PpPSEl. The isolated EST CC001042041R, shown in accordance with the invention as SEQ ID NO:7, is 708 base pairs long and has an open reading frame of 642 base pairs from the first base which encodes 214 amino acids and has a stop codon in position 643-645. The amino acid sequence up to the stop codon is shown in accordance with the invention in SEQ ID NO:8.<br>
Besides the similarity with the PSEl gene product, the similarity with the Saccharomyces cerevisiae  elongase (see elo IP), which is required in yeast for the elongation of fatty acids with medium chain length, may also be resorted to (Toke &amp; Martin, 1996, Isolation and characterization of a gene affecting fatty acid elongation in Saccharomyces cerevisiae. Journal of Biological Chemistry 271, 1S413-18422). Table 3 shows the identities and homologies of elongases according to the invention with each other and with the physcomitrella patens and yeast elongases. The data were obtained with the aid of the GAP program as subprogram of the following software: Wisconsin Package Version 10.0 (Genetics Computer Group (GCG), Madison, Wise, USA).<br>
Table 3:<br><br>
Identi¬ty/ homology	Tc^PSEl	TC_PSE2	Pp_PSEl	See elo IP<br>
CC_PSE1	47.1% / 40.2%	50.6% / 43.5%	38.5% / 29.4%	45.1% / 33.5%<br>
Tc_PSEl	100 / 100	n.d.	43.2% / 32.7%	41.9% / 29.9%<br>
Tc_PSE2	41.7% / 29.5%	100 / 100	39.2% / 30.0	35.4% / 27.8%<br><br>
In particular, Figures 5 to 10 can be used to derive the following sequence motifs as regions of high homology and corresponding consensus sequences derived therefrom which, by back translating the amino acids into three-base-pair codons, .ead to oligonucleotides which can be exploited for isolating lovel elongases by means of polymerase chain reaction. They are, .n particular, the sequence motifs shown in Figure 10. These lotifs can be used for deriving oligonucleotides which, in :ombination with two oligonucleotides, can be employed in PCR experiments for isolating further elongase fragments. To do this, Lt is expedient to construct and synthesize one oligonucleotide [latching the conventionally defined 5'-3' strand and a second one tfith an oligonucleotide matching the 3'-5'  strand downstream. This results in a definable number of primer combinations, which is limited by permutation of the variants which are possible.<br>
En this context, use may also be made of oligo-dT primers and variants thereof, for example by the last base allowing specificity for a transcript pool, such as, for example, oligo dT (12-20) X, where X can be a G, C or T. Also, a second base oligo IT (12-20) XY can be made use of, where X can be a G, C or A, while the Y  can be an A, G, C or T.<br>
The above-defined sequences allow 17- to 20mer oligonucleotides to be derived which can be exploited for isolating gene fragments by varying the primer combinations and experimental parameters such as the temperature program, Mg ion concentration and the like. The resulting fragments can be cloned into suitable vectors and the sequence of resulting clones can be determined by current methods for identifying novel elongases.<br>
Example 6b: Isolation of the cDMA clone from Phvtophthora infestans<br>
The cDNA clone designated PI001002014R from Phytophthora infestans was identified from cDHAs sequenced at random, using homologies to PUFA elongase from the moss Physcoraitrella patens (ATCCC 48886) . The clone contains the consensus motif MyxYYF shown in Figure 8, where, different from PUFA elongases which have hitherto been identified, a threonine radical was found as variable amino acid x. This further variation can be used for deriving PCR primers.<br><br>
Example 7:   Identification 6f  genes by means of hybridization (TC002034029R-11 iGenTc-PCEl)<br>
Gene sequences can be used for identifying homologous or heterologous genes from cDNA libraries or genomic libraries.<br>
Homologous genes (i.e. full-length cDNA clones which are homologous, or homologs) can be isolated via nucleic acid hybridization using, for example, cDNA libraries: the method can be made use of in particular for isolating functionally active fuii-length genes of SEQ ID NO:I, SEQ ID N0:3, SEQ  ID N0:5, SEQ ID NO:7, SEQ ID HO:9 and SEQ ID NO:11. Depending on the frequency of the gene of interest, 100,000 up to 1,000,000 recombinant bacteriophages are plated and transferred to a nylon membrane. After denaturation with alkali, the DHA is immobilized on the membrane, for example by UV crosslinking. Hybridization is performed under highly stringent conditions. The hybridization and the wash steps are carried out in aqueous solution at an ionic strength of 1 M NaCl and a temperature of 68°C. Hybridization probes were generated for example by labeling with radioactive (32p-) nick transcription (High Prime, Roche, Mannheim, Germany). The signals are detected by autoradiography.<br>
Partially homologous or heterologous genes which are related but not identical can be identified analogously to the process described above using low-stringency hybridization and wash conditions. For the aqueous hybridization, the ionic strength was usually kept at 1 M KaCl, and the temperature was lowered gradually from 68 to 42°C.<br>
The isolation of gene sequences which only exhibit homologies with an individual domain of, for example, 10 to 20 amino acids can be carried out using synthetic, radiolabeled oligonucleotide probes. Radiolabeled oligonucleotides are generated by phosphorylating the 5' end of two complementary oligonucleotides with T4 polynucleotide kinase. The complementary oligonucleotides are hybridized and ligated with each other to give rise to concatemers. The double-stranded concatemers are radiolabeled for example by nick transcription. Hybridization is usually carried out under low stringency conditions using high oligonucleotide concentrations.<br>
Oligonucleotide hybridization solution:<br>
6 X SSC<br>
0.01 M sodium phosphate<br>
1 mM EDTA (pH 8)<br><br>
0.5S SDS<br>
100 ng/ml denaturated salmon sperm DNA<br>
0.1% dry low-fat milk<br>
During the hybridization, the temperature is lowered stepwise to 5 to 10°C below the calculated oligonucleotide temperature or to room temperature (unless otherwise specified, RT = ~ 23°C in all experiments), followed by wash steps and autoradiography. Washing is carried out at extremely low stringency, for example 3 wash steps using 4 x SSC. Further details are as described by Sambrook, J., et al. (1989), "Molecular Cloning: A Laboratory Manual", Cold Spring Harbor Laboratory Press, or Ausubel, F.M., et al. (1994) "Current Protocols in Molecular Biology", John Wiley &amp; Sons.<br>
I<br>
The clone TCO02034029R-11 with the gene name Tc_PCEl_l is a full-length sequence of an elongase from Thraustochytrium and thus longer than the clone TC002034029R from Seq. ID No. 3 and Seq. ID No. 4. The clone was isolated using a hybridization method, as described above (Example 7). It is a DNA sequence of a length of 1 050 base pairs encoding for 271 amino acids having a start codon in base pair position 43-45 and a stop codon in base pair position 856-858.<br>
Example 8:   Identification of target genes by screening expression libraries with antibodies<br>
To generate recombinant protein, for example in E. coli, cDNA sequences are used (for example Qiagen QiAexpress pQE system). The recombinant proteins are then affinity-purified, usually via Ni-NTA affinity chromatography (Qiagen). The recombinant proteins are then used for raising specific antibodies, for example using standard techniques for iiraitunizing rabbits. The antibodies are then affinity-purified using an Ni-NTA column which is presaturated with recombinant antigen, as described by Gu et al., (1994) BioTechniques 17:257-262. The antibody can then be used for screening expression cDNA libraries by immunological screening (Sambrook, j., et al. (1989), "Molecular Cloning: \  Laboratory Manual", Cold Spring Harbor Laboratory Press, or liusubel, F.M., et al. (1994) "Current Protocols in Molecular Biology", John Wiley &amp; Sons).<br>
Example 9:   Plasmids for plant transformation<br>
Binary vectors such as pBinAR can be used for plant transformation (Hofgen and willmitzer. Plant Science 66 (1990) 221-230). The binary vectors can be constructed by ligating the<br><br>
CDNA in sense or antisense otientation into T-DKA. 5' of the cDNA, a plant promoter activates cDNA transcription. A polyadenylation sequence is located 3' of the cDNA.<br>
Tissue-specific expression can be achieved using a tissue-specific promoter. For example, seed-specific expression can be achieved by cloning in the napin or the LeB4 or the USP promoter 5' of the cDNA. Any other seed-specific promoter element can also be used. The CaMV-35S promoter may be used for constitutive expression in all of the plant.<br>
The protein expressed can  be targeted into a cellular cojnpartment using a signal peptide, for example for plastids, mitochondria or the endoplasmatic reticulum (Kermode, Crit. Rev. Plant Sci. 15, 4 (1996) 285-423). The signal peptide is cloned 5' in correct reading frame with the cDNA in order to achieve subcellular localization of the fusion protein.<br>
Example 10:  Transformation of Agrobacterium<br>
Agrobacterium-mediated plant transformation can be carried out for example using the Agrobacterium tumefaciens strain GV3101 (PMP90) (Koncz and Scheil, Mol. Gen. Genet. 204 (1986) 383-396) or LBA4404 (Clontech). The transformation can be carried out by standard transformation techniques (Deblaere et al., Nucl, Acids. Tes. 13 (1984), 4777-4788).<br>
ExEunple 11: Plant transformation<br>
Agrobacterium-mediated plant transformation can be carried out using standard transformation and regeneration techniques (Gelvin, Stilton B., Schilperoort, Robert A., plant Molecular Biology Manual, 2nd Ed., Dordrecht: Kluwer Academic Publ., 1995, in Sect., Ringbuc Zentrale Signatur: BTll-P ISBN 0-7923-2731-4; Glick, Bernard R., Thompson, John E., Methods in Plant Molecular Biology and Biotechnology, Boca Raton: CRC Press, 1993, 360 pp. ISBN 0-8493-5164-2).<br>
For example, oilseed rape can be transformed by means of cotyledon or hypocotyledon transformation (Moloney et al.. Plant Cell Report 8 (1989) 238-242; De Block et al.. Plant Physiol. 91 (1989) 694-701). The use of antibiotics for the selection of agrobacteria and plants depends on the binary vector and the agrobacterial strain used for the transformation. The selection of oilseed rape is normally carried out using kanamycin as selectable plant marker.<br><br>
Agrobacterium-mediated gene transfer in linseed (Linum usitatissimum) can be carried out for example using a technique described by Mlynarova et al. (1994) Plant Cell Report 13:282-285.<br>
The transformation of soya can be carried out for example using a technique described in EP—A—0 424 047 (Pioneer Hi-Bred international) or in EP-A-0 397 687, US 5,376,543, US 5,169,770 (University of Toledo).<br>
Plant transformation using particle bombardment, polyethylene-glycol-mediated DNA uptake or via the silicon carbonate fiber technique is described, for example, by Freeling and Walbot "The maize handbook" (1993) ISBH 3-540-97826-7, Springer Verlag New York).<br>
Example 12: Plasmids for plant transformation<br>
Binary vectors such as pBinAR can be used for plant transformation (Hdfgen and Willmitzer, Plant Science 66 (1990) 221-230). The binary vectors can be constructed by ligating the cDHA in sense or antisense orientation into T-DNA. 5' of the CDMA, a plant promoter activates cDNA transcription. A polyadenylation sequence is located 3' of the cDNA. Tissue-specific expression can be achieved using a tissue-specific promoter. For example, seed-specific expression can be achieved by cloning in the napin or the LeB4 or the USP promoter 5' of the cDNA. Any other seed-specific promoter element can also be used. The CaMV-35S promoter may be used for constitutive expression in all of the plant.<br>
In particular, genes encoding elongases and desaturases can be cloned into a binary vector by constructing a plurality of expression cassettes in succession in order to imitate the metabolic pathway in the plant.<br>
Within an expression cassette, the protein expressed can be targeted into a cellular compartment using a signal peptide, for example for plastids, mitochondria or the endoplasmatic reticulum (Kermode, Crit. Rev. Plant Sci. 15, 4   (1996) 285-423). The signal peptide is cloned 5' in correct reading frame with the cDNA in order to achieve subcellular localization of the fusion protein.<br>
Example 13: In  vivo  mutagenesis<br>
The in-vivo mutagenesis of microorganisms can be performed by passaging the plasmid DNA (or any other vector DHA) via E. coli or other microorganisms (for example Bacillus spp. or yeasts such<br><br>
as Saccharomyces cerevisiae), in which the ability of retaining the integrity of their genetic information is disrupted. Conventional mutator strains have mutations in the genes for the DKA repair system (for example mutHLS, mutD, mutT and the like; as reference, see Rupp, W.D. (1996) DNA repair mechanisms, in: Escherichia coli and Salmonella, pp. 2277-2294, ASM: Washington), These strains are known to the skilled worker. The use of these strains is illustrated for example in Greener, A., and Callahan, M. (1994) Strategies 7:32-34. Mutated DNA molecules are preferably transferred to plants after the microorganisms have been selected and tested. Transgenic plants are generated in accordance with various examples in the examples section of the present document.<br>
Example 14:  Studying the expression of a recombinant gene product in a transformed organism<br>
The activity of a recombinant gene product in the transformed host organism was measured at the transcriptional and/or the translational level.<br>
A suitable method for determining the amount of transcription of the gene (which indicates the amount of RNA available for translation of the gene product) is to carry out a northern blot as specified hereinbelow (for reference, see Ausubel et al. (1988) Current Protocols in Molecular Biology, Wiley: New York, or the abovementioned excimples section) in which a primer which is designed such that it binds to the gene of interest is labeled with a detectable label (usually radioactivity or chemiluminescence) so that, when the total RNA of a culture of the organism is extracted, separated on a gel, transferred to a stable matrix and incubated with this probe, binding and the extent of binding of the probe indicates the presence as well as the quantity of the mRNA for this gene. This information indicates the degree of transcription of the transformed gene. Total cell RNA can be prepared from cells, tissues or organs by a plurality of methods, all of which are known in the art, such as, for example, the method of Bormann, E.R., et al. (1992) Mol. Microbiol. 6:317-326.<br>
Northern hybridization:<br>
For the RNA hybridization, 20 jig of total RNA or 1 fig of poly(A)"^-RNA were separated by gel electrophoresis in 1.25% strength agarose gels using formaldehyde as described by Amasino (1986, Anal. Biochem. 152, 304), transferred to positively charged nylon membranes (Hybond H+, Amersham, Brunswick) by capillary<br><br>
attraction using 10 x SSC, iiranobilized by UV light and prehybridized for 3 hours at 68°C using hybridization buffer (10% dextran sulfate w/v, 1 M NaCl, 1% SDS, 100 mg herring sperm DHA) . The DNA probe had been labeled with the Highprime DNA labeling ' kit (Roche, Mannheim, Germany) during the prehybridization stage using a-32p-dCTP (Amersham, Brunswick, Germany). The hybridization was carried out after adding the labeled DNA probe in the same buffer at 68°C overnight. The wash steps were carried out twice for 15 minutes using 2 x SSC and twice for 30 minutes using 1 x SSC, 1% SDS, at 68°C. The sealed filters were exposed at -70°C for a period of 1 to 14 days.<br>
Standard techniques, such as a Western blot (see, for example, Ausubel et al. (1988) Current Protocols in Molecular Biology, Wiley: New York) can be employed for studying the presence or the relative quantity of protein translated by this mRNA. In this method, the total cell proteins are extracted, separated by means of gel electrophoresis, transferred to a matrix such as nitrocellulose and incubated with a probe such as an antibody which specifically binds to the desired protein- This probe is usually provided with a chemiluminescent or colorimetric label which can be detected readily. The presence and the quantity of the label observed indicates the presence and the quantity of the desired mutated protein which is present in the cell.<br>
Example 15: Analysis of the effect of the recombinant proteins on the production of the desired product<br>
The effect of the genetic modification in plants, fungi, algae, ciliates or on the production of a desired compound (such as a fatty acid) can be determined by growing the modified microorganisms or the modified plant under suitable conditions (such as those described above) and analyzing the medium and/or the cell components for the increased production of the desired ' product (i.e. of lipids or a fatty acid). These analytical techniques are known to the skilled worker and encompass spectroscopy, thin-layer chromatography, various staining methods, enzymatic and microbiological methods, and analytical chromatography such as high-performance liquid chromatography (see, for example, Ullman, Encyclopedia of Industrial Chemistry, vol. A2, pp. 89-90 and pp. 443-613, VCH Weinheim (1985); Fallon, A., et al., (1987) "Applications of HPLC in Biochemistry" in: Laboratory Techniques in Biochemistry and Molecular Biology, Vol. 17; Rehm et al. (1993) Biotechnology, Vol. 3, Chapter III "Product recovery and purification", pp. 469-714, VCH Weinheim; Belter, P.A., et al. (1988) Bioseparations: downstream processing for Biotechnology, John Wiley and Sons; Kennedy, J.F., and<br><br>
Cabral, J.M.S. (1992) Recovery processes for biological Materials, John Wiley and Sons; Shaeiwitz, J.A., and Henry, J.D. (1988) Biochemical Separations, in: Ullmann's Encyclopedia of Industrial Chemistry, Bd. B3; Chapter 11, Vol. 1-27, VCH Weinheim; and Dechow, F.J. (1989) Separation and purification techniques in biotechnology, Noyes Publications).<br>
In addition to the abovementioned methods, plant lipids are extracted from plant material as described by Cahoon et al. (1999) Proc. Natl. Acad. Sci. USA 96 (22):12935-12940, and Browse et al. (1986) Analytic Biochemistry 152:141-145. Qualitative and quantitative lipid or fatty acid analysis is described in Christie, William W., Advances in Lipid Methodology, Ayr/Scotland: Oily Press (Oily Press Lipid Library; 2); Christie, William W., Gas Chromatography and Lipids. A Practical Guide -Ayr, Scotland: Oily Press, 1989, Repr. 1992, IX, 307 pp. (Oily Press Lipid Library; 1); "Progress in Lipid Research, Oxford: Pergamon Press, 1 (1952) - 16 (1977) under the title: Progress in the Chemistry of Fats and Other Lipids CODEN.<br>
In addition to measuring the end product of the fermentation, it is also possible to analyze other components of the metabolic pathways which are used for producing the desired compound, such as intermediates and byproducts, in order to determine the overall production efficiency of the compound. The analytical methods encompass measurements of the nutrient quantities in the medium (for example sugars, carbohydrates, nitrogen sources, phosphate and other ions), biomass composition and growth measurements, analysis of the production of customary metabolites of biosynthetic pathways, and measurements of gases which are generated during fermentation. Standard methods for these measurements are described in Applied Microbial Physiology; A Practical Approach, P.M. Rhodes and P.P. Stanbury, Ed., IRL Press, pp. 131-163 and 165-192 (ISBN: 0199635773) and references stated therein.<br>
One  example is the analysis of fatty acids (abbreviations: FAME, fatty acid methyl ester; GC-MS, gas-liquid chromatography/mass spectrometry; TAG, triacylglycerol; TLC, thin-layer chromatography).<br>
The unambiguous detection of the presence of fatty acid products can be obtained by analyzing recombinant organisms by analytical standard methods: GC, GC-MS or TLC, as they are described on several occasions by Christie and the references therein (1997, in: Advances on Lipid Methodology, Fourth edition: Christie, Oily<br><br>
Press, Dundee, 119-169; ly^a;   gas cnromatography/mass spectrometry methods, Lipide 33:343-353).<br>
The material to be analyzed can be disrupted by sonication, grinding in a glass mill, liquid nitrogen and grinding or via other applicable methods. After disruption, the material must be centrifuged. The sediment is resuspended in distilled water, heated for 10 minutes at 100°C, ice-cooled and recentrifuged followed by extraction in 0.5 M sulfuric acid in methanol with 2% dimethoxypropane for 1 hour at 90°C, which leads to hydroiyzed oil and lipid compounds which give transmethylated lipids. These fatty acid methyl esters are extracted in petroleum ether and finally subjected to GC analysis using a capillary column (Chrompack, WCOT Fused Silica, CP-Wax-52 CB, 25 \m,   0.32 mm) at a temperature gradient of between 170°C and 240°C for 20 minutes and 5 minutes at 240°C. The identity of the resulting fatty acid methyl esters must be defined using standards which are commercially available (i.e. Sigma).<br>
In the case of fatty acids for which no standards are available, the identity must be demonstrated via derivatization followed by GC/MS analysis. For example, the localization of fatty acids with triple bond must be demonstrated via GC/MS following derivitization with 4,4-dimethoxyoxazolin derivatives (Christie, 1998, see above).<br>
Example 16:  Expression constructs in heterologous microbial systems<br>
Strains, growth conditions and plasmids<br>
The Escherichia coli strain XLl Blue MRF' kan (Stratagene) is used for subcloning the novel Physcomitrella patens elongases, such as PpPSEl. For functionally expressing this gene, we used the Saccharomyces cerevisiae strain INVSc 1 (Invitrogen Co.). E. coli is cultured at 37°C in Luria-Bertini broth (LB, Duchefa, Haarlem, the Netherlands). If necessary, ampicillin (100 mg/liter) is added, and 1.5% of agar (w/v) is added for solid LB media. S. cerevisiae is cultured at 30°C either in YPG medium or in complete minimal medium without uracil (CMdum; see: Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A., Struhl, K., Albright, L.B., Coen, D.M., and Varki, A. (1995) Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York) together with 2% (w/v) of either raffinose or glucose. For solid media, 2% (w/v) of Bacto"" agar (Difco) are<br><br>
added. The plasmids used for cloning and expression are pUC18 (Pharmacia) and pYES2 (Invitrogen Co.).<br>
Example 17:  Cloning and expression of PUFA-specific<br>
Physcomitrella, Crypthecodinium and Thraustochytrium elongases.<br>
The full-length genes of sequences according to the invention can be isolated as described in Example 7 and processed as illustrated hereinbelow. Concrete expression examples are shown with regard to the use.<br>
A)  Elongation of fatty acids by the moss elongase pp_PSEl:<br>
For expression in yeast, the P. patens cDNA clone PpPSEl (earlier database sequence name: 08_ckl9_b07, new name: pp001019019f), which encodes the PUFA-specific elongase (PSEl) gene, was first modified in such a way that a BamHI restriction site and the yeast consensus sequence for highly effective translation (Kozak, M. (1986) Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes, Cell 44, 283-292) was obtained next to the start codon and a BamHI restriction site was obtained which flanked the stop codon. To amplify the open reading frame, a primer pair which was complementary to its 5' and 3' ends was synthesized.<br>
The gene of the sequence according to the invention shown in SEQ ID N0:1 was cloned into pYES by means of polymerase chain reaction, giving rise to the plasmid pYPp_PSEl:<br>
The following oligonucleotides were employed for the PCR experiment:<br>
Ppex6:    ggatccacataatggaggtcgtggagagattc Ppex6r:    ggatcctcactcagttttagctccttttgc<br>
The PCR reaction was carried out with plasmid DNA of clone PP001019019F as matrix in a thermocycler (Biometra) with Pfu DHA (Stratagene) polymerase and the following temperature program: 3 minutes at 96°C followed by 25 cycles with 30 seconds at 96'C, 30 seconds at 55°C and 1 minute at 72°C, 1 cycle with 10 minutes at 72°C.<br>
The correct size of the amplified DNA fragment was confirmed by agarose TBE gel electrophoresis. The amplified DNA was extracted from the gel using the QIAquick gel extraction kit (QIAGEN) and initially cloned into pUClS using the Sure Clone Ligation Kit<br><br>
(Pharmacia) . The fragment clo'ned thus was cut vith BamHl and ligated into pXES, giving rise to pYpp_pSEl. The fragment orientation was checked by means of Hindlll. Following the transformation of E. coli XLl Blue MRF' kan, a DNA minipreparation (Riggs, M.G., &amp; McLachlan, A. (1986) A simplified screening procedure for large numbers of plasmid<br>
mini-preparation. BioTechniques 4, 310-313) of transformants was carried out, and positive clones were identified by means of BamHI restriction analysis. The sequence of the cloned PCR product was confirmed by resequencing using the ABI PRISM Big Dye Terminator Cycle Sequencing Ready Reaction Kit (Perkin-Elmer, Weiterstadt).<br>
One clone was grown with the Nucleobond® AX 500 plasmid DNA extraction kit (Macherey-Nagel, Diiringen) for the DNA maxipreparation.<br>
Saccharomyces INVScl was transformed with pYPp_PSEl or with pYES2 as control by means of a modified PEG/lithium acetate protocol (Ausubel et al., 1995). Following selection on CMdum-agar plates with 2% glucose, transformants and a pYES2 transformant were selected for further cultivation and functional expression as already stated and fed various fatty acids in the medium.<br>
i)  Lipid patterns of yeasts which are transformed with the pYES plasmid without fragment insertion or which express the Pp-PSEl gene {data in mol%) after feeding with 250 [m hexadecatrienoic acid (16 :3'^'^'icc,i3c) .<br>
Table 4:<br><br>
PYES 2	PYES2	pyPp_PSEl	PYPp_PSEl<br>
16:0	11.8%	16:0	11.1%<br>
16:1	28.7%	16:1	23.9%<br>
I6:3^'^'i°='i3
18:0	10.6%	18:0	8.6%<br>
18:1^9c	34.9%	18:1'^9'^	20.6%<br>
18:l^ii
18:3'9c,i2c,i5c	3.7%	18:3^9C'12c,15c	21.4%<br>
ii) Lipid patterns of yeasts which are transformed with the pYES plasmid without fragment insertion or which express the Pp-PSEl gene (data in mol%) after feeding with 500 yim pinolenic acid (18 :3'^6c,9c,i2c) .<br><br>
Table 5:<br><br>
PYES 2	PYES 2	pYPp_PSEl	PYPp_PSEl<br>
16:0	18.3%	16:0	16.9%<br>
15:X'i9c	16.0%	16:l^9c	15.3%<br>
18:0	8.6%	18:0	8.4%<br>
IBil'^^c	16.7%	18:1^9=	17.5%<br>
18:l^iic	0.7%	18:1 "lie	2.0%<br>
X8:3'"c.9c-i2c	39.8%	18:3'^c,9c,i2c	32.6%<br>
20:3'^'=.lie,14c	0%	20 :3 ^'c,lie,14c	5.1%<br>
iii)Lipid patterns of yeasts which are transformed with the pYES plasinid without fragment insertion or which express the Pp-PSEl gene (data in moll) after feeding with 500 (irn stearidonic acid (is :4^6'='^='i^'='i5c) .<br>
Table 6:<br><br>
PYES2	pYES2	pYPp_PSEl	PYPp_PSEl<br>
16:0	15.2%	16:0	15.6%<br>
16:1^^9^	13.1%	le-livsc	14.9%<br>
18:0	12.3%	18:0	10.7%<br>
18:i^9c	12.9%	ISil'iac	14.0%<br>
Ig-l'Mlc	0.7%	18:l'iiic	1.2%<br>
18:3'6='9='i2c.i5c	45.4%	18 :3^6c,9c,i2c,i5c	23.8%<br>
20:4<br>
i8c,lie,14c,170	0.4%	20:4<br>
A8c,llc,14c,nc	19.8%<br>
iv) Lipid patterns of yeasts which are transformed with the pYES plasmid without fragment insertion or which express the Pp-PSEl gene (data in mol%) after feeding with 500 \jm linoleic acid (18 :2^9c,i2c) .<br>
Table 7:<br><br>
pYES2	pYES2	pYPp_PSEl	PYPp_PSEl<br>
16-.0	7.9%	16:0	8.7%<br>
16:1^9=	1.2%	16:1^9c	1.3%<br>
18:0	5.3%	18:0	5.1%<br>
18:1 ^^9=	1.3%	18:l^9c	1.3%<br>
18:2"9'^'i2=	83.9%	18:2^9^.12=	80.4%<br>
20: 2^ii'='H'=	0.5%	20:2^ii='i«=	3.2%<br>
B)  Elongation of fatty acids by a Thraustochytrium elongase<br>
For expression in yeast, the Thraustochytrium cDNA clone of<br><br>
SEQ ID NO: 3 (Tc_PSE2), which encodes a PUFA-specific elongase (PSE) gene, is first modified in such a way that it constitutes a functionally active polypeptide. To this end, the N-terminus of the protein is elongated at DNA level by 42 base pairs by the few missing bases from the Physcomitrella patens elongase. However, it is also possible only to add a start codon in the correct reading frame for the sequence.<br>
The following oligonucleotides are employed for the PCR experiment:<br>
pTCPSE2-5' :<br>
aaaggatccacataatggaggtcgtggagagattctacggtgagttggatggga agGTCATTTCGGGCCTCGACC • pTCPSE2-3': aaggatccctgagttttagctcccttttgctttcc<br>
In addition, both oligonucleotides contain a BamHI restriction site and the yeast consensus sequence for highly efficient translation (Kozak, M. (1986) Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell 44, 283-292).<br>
The PCR reaction is carried out with plasmid DMA as matrix in a thermocycler (Biometra) with Pfu DNA (Stratagene) polymerase and the following temperature program: 3 minutes at 96'C followed by 25 cycles with 30 seconds at 96°C, 30 seconds at 55°C and 3 minutes at 72°C, 1 cycle with 10 minutes at 72°C and stop at 4°C.<br>
The correct size of the amplified DKA fragment is confirmed by agarose TBE gel electrophoresis. The amplified DNA is extracted from the gel using the QIAquick gel extraction kit (QIAGEN) and ligated into the Smal restriction site of the dephosphorylated vector pUClS using the Sure Clone Ligation Kit (Pharmacia), giving rise to pUC-hybrid-Tc_PSE2. Following the transformation of E. coli XLl Blue MRF' kan, a DNA minipreparation (Riggs, H.G., &amp; HcLachlan, A. (1986) A simplified screening procedure for large numbers of plasmid mini-preparation- BioTechniques 4, 310-313) of 24 ampicillin-resistant transformants was carried out, and positive clones were identified by means of BamHI restriction analysis. The sequence of the cloned PCR product was confirmed by resequencing using the ABI PRISM Big Dye Terminator Cycle Sequencing Ready Reaction Kit (Perkin-Elmer, weiterstadt).<br>
The plasmid DNA of pUC-PSEl and pUC-liybrid-Tc_PSE2 was first cleaved with BamHI and the fragments obtained were ligated into the BamHI restriction site of the dephosphorylated yeast/E.coli shuttle vector pYES2, giving rise to pY2 hybrid-Tc_PSE2.<br><br>
Following transformation of E. coli and DHA minipreparation from the transformants, the orientation of the DNA fragment in the vector was checked by cleavage with Hindlll. One clone was grown with the Nucleobond® AX 500 plasmid DNA extraction kit (Macherey-Wagel, Duringen) for the DNA maxipreparation. Saccharomyces INVScl is transformed with py2PSEl, pYES2, pY2-hybrid-Tc_PSE2 and pyES2 by means of a modified PEG/lithium acetate protocol (Ausubel et al., 1995). Following selection on CMdum agar plates with 2% glucose, in each case four pY2PSEl transformants (py2PSEla-d), pY2-hybrid-Tc_PSE2 transformants (pY2-hybrid-Tc_PSE2 la-d) and one pYES2 transformant are selected for further culture and functional expression.<br>
Functional expression of an elongase activity in yeast<br>
Preculture;<br>
20 ml of CMdum liquid medium with 2% (w/v) raffinose were inoculated with the transgenic yeast clones (pY2-hybrid-Tc_PSE2 la-d, pYES2) and cultured for 3 days at 30°C, 200 rpm, until an optical density at 600 nm {ODgoo) of 1-5-2 had been reached.<br>
Main culture:<br>
For expression, 20 ml of CMdum liquid medium with 2% raffinose and 1% (v/v) Tergitol KP-40 were supplemented with fatty acid substrates to a final concentration of 0.003% (w/v). The media are inoculated with the precultures to an ODgQo of 0.05. Expression was induced for 16 hours at an ODgoo of 0.2, using 2% (w/v) galactose, whereupon the cultures were harvested at an ODeoo of 0.8-1.2.<br>
Fatty acid analysis<br>
The overall fatty acids were extracted from yeast cultures and analyzed by means of gas chromatography. To this end, cells of 5 ml culture were harvested by centrifugation (1000 x g, 10 minutes, 4°C) and washed once with 100 iiiM NaHCOs, pH 8.0, to remove residual medium and fatty acids. To prepare the fatty acid methyl esters (FAMES), the cell sediments were treated for 1 hour at 80°C with 1 M methanolic H2SO4 and 2% (v/v) dimethoxypropane. The FAMES were extracted twice with 2 ml of petroleum ether, washed once with 100 mM NaHCOs, pH 8.0, and once with distilled water, and dried with Na2S04. The organic solvent was evaporated under a stream of argon, and the FAMEs were dissolved in 50 ^1 petroleum ether. The samples were separated on a ZEBRON ZB Wax capillary column (30 m, 0.32 mm, 0.25 [im; Phenomenex) in a Hewlett<br><br>
Packard 5850 gas chromatograph equipped with a flame ionization detector. The oven temperature was programmed from 70°C (hold for 1 minute) to 200°C at a rate of 20°C/minute, then to 250'C {hold for 5 minutes) at a rate of 5°C/minute and finally to 260°C at a ' rate of 5°C/minute. Nitrogen was used as the carrier gas (4.5 ml/minute at 70°C}. The fatty acids were identified by comparison with retention times of FAME standards (SIGMA).<br>
The fatty acid patterns of five transgenic yeast strains are shown in Table 1 in mol%.<br>
The ratios of the y-linolenic acid which had been added and taken up are emphasized by numbers printed in bold, those of the elongated products by numbers in red and those of the elongated Y-linolenic acid by numbers printed in bold (last line).<br>
The GC analysis of FAMES which from total lipids of the yeasts transformed with pYES2 (i/control) and pY2PSEl (ii-iv c+d/ in each case transformed with pY2PSElA, py2PSElB, py2PSElC, pY2PSElD) is shown in Figure 2a-e. For the analysis, the transgenic yeasts were cultured in the presence of 7-18:3. Table 1 shows their fatty acid patterns in mol%. The uptake of Y-18:3 is emphasized by numbers printed in bold, the elongation product dihomo-y-linolenic acid (20:3A8,11,14) is underlined and the ratio Y-18:3-elongation product (also in mol%) is emphasized by numbers printed in bold (last line). The structure and the mass spectra of the DMOX derivative of cis-A6,9,12 C18:3 can be seen from Fig. 3a+b. The structure and the mass spectra of the DMOX derivative of A8,ll,14 C20:3 can be seen from Fig. 4a+b.<br>
The results demonstrate that Y-18:3 has been incorporated into all transgenic yeasts in large aimounts. All four transgenic yeast clones which had been transformed with pY2PSEl exhibit an additional peak in the gas chromatogram, which was identified as 20:3 AS,11,14 by a comparison of the retention times. A gas chromatography/mass spectroscopy can provide additional proof to confirm this identity. The percentage of elongated Y-18:3 was 23.7 to 40.5%, as shown in Table 1. Furthermore, no significant elongation of palmitic acid (16:0), palmitoleic acid (16:1), stearic acid (18:0) or oleic acid (18:1 A9) was observed.<br>
The products identified demonstrate that the nucleotide sequence of PpPSEl encodes a A6-selective fatty acid elongase from the moss Physcomitrella patens, which leads to the formation of novel fatty acids in transgenic yeasts.<br><br>
The ratios of the fatty acid substrates which have been added and taken up can be determined as above, so that quantity and quality of the elongase reaction can be detected.<br>
The structure and the mass spectra of DMOX derivatives also reveal the respective position of a double bond.<br>
Further feeding experiments with a wide range of other fatty acids (for example arachidonic acid, eicosapentaenoic acid and the like) can be carried out for confirming the substrate selectivity of this elongase in greater detail.<br>
Example 18:  Isolation of the desired product from transformed organisms in general<br>
The desired product can be obtained from plant material or fungi, algae, ciliates, animal cells or from the supernatant of the above-described cultures by various methods known in the art. If the desired product is not secreted from the cells, the cells can be harvested from the culture by slow centrifugation, and the cells can be lyzed by standard techniques, such as mechanical force or sonication. Plant organs can be separated mechanically from other tissue or other organs. After homogenization, the cell debris is removed by centrifugation, and the supernatant fraction, which contains the soluble proteins, is retained for further isolating the desired compound. If the product is secreted from desired cells, the cells are removed from the culture by slow centrifugation, and the supernatant fraction is retained for the further isolation.<br>
The supernatant fraction from each isolation step is subjected to a chromatography with a suitable resin, the desired molecule either being retained on the chromatography resin while many contaminants in the sample are not, or the contaminants remaining on the resin while the sample does not. These chromatography steps can be repeated, if desired, using either the same or other chromatography resins. The skilled worker is familiar with selecting suitable chromatography resins and with their most effective use for a particular molecule to be isolated. The product isolated can be concentrated by filtration or ultrafiltration and stored at a temperature at which the stability of the product is highest.<br>
A broad spectrum of isolation methods is known in the art, and the isolation method above is not intended to be limiting. These isolation methods are described, for example, in Bailey, J.E., &amp;<br><br>
Ollis, D.F., Biochemical Engxutsexxng tunaamentals, McGraw-Hill: New York (1986).<br>
Identity and purity of the compounds isolated can be determined by standard techniques of the art. They include high-performance liquid chromatography (HPLC), spectroscopic methods, staining methods, thin-layer chromatography, NIRS, enzyme assays or microbiological methods. For a review of these analytical methods, see: Patek et al. (1994) Appl. Environ. Microbiol. 60:133-140; Malakhova et al. (1996) Biotekhnologiya 11:27-32; and Schmidt et al. (1998) Bioprocess Engineer. 19:67-70. Ulmann's Encyclopedia of Industrial Chemistry (1996) Vol. A27, VCH Weinheim, pp. 89-90, pp. 521-540, pp. 540-547, pp. 559-566, 575-581 and pp. 581-587; Michal, G (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley and Sons; Fallon, A., et al. (1987) Applications of HPLC in Biochemistry in; Laboratory Techniques in Biochemistry and Molecular Biology, Vol. 17.<br>
Equivalents<br>
The skilled worker knows, or can identify, a number of squivalents of the specific embodiments according to the Invention which have been described herein by simply resorting to routine experiments. These equivalents are intended to be covered jy the patent claims.<br><br><br>
WE CLAIM:<br>
1. An isolated nucleic acid encompassing a nucleotide sequence encoding a polypeptide which etongates Ci,,' Cig- or CV fatty acids with at least two double bonds in the fatty acid moiecuie. selected from the group consisting of<br>
a)	a nucleic acid sequence shown in SEQ ID N0:1, SEQ ID N0:3, SEQ ID<br>
NO:5orSEQIDNO:7,<br>
b)	a nucleic acid sequence which, in accordance with the degeneracy of the genetic code, is derived from the sequence shown in SEQ ID NO; 1, SEQ ID N0:3, SEQ ID N0:5 or SEQ ID N0:7,<br>
c)	derivatives of the sequence shown in SEQ ID NO:I, SEQ ID N0:3, SEQ ID NO:5 or SEQ ID NO:7 which encode polypeptides of the amino acid sequence shown in SEQ ID N0:2. SEQ ID N0:4, SEQ ID N0:6 or SEQ ID N0:8 and which have at least 50% homology at the amino acid level without the<br>
	enzymatic action of the polypeptides being substantially reduced.<br>
       2.       The isolated nucleic acid sequence as claimed in claim 1, wherein the sequence         is derived from a plant, alga or fungus.<br>
      3.        The isolated nucleic acid sequence as claimed in claim I or 2, wherein the        sequence is derived from Physcomilrella, Thraustochytrium or Crypthecodinium.<br>
 4. A gene construct encompassing an isolated nucleic acid as claimed in any of claims 1 to 3, wherein the nucleic acid is functionally linked to one or more regulatory signals.<br>
5.	"ITie gene construct as claimed in claim 4, whose gene expression is enhanced<br>
by the regulatory signals.<br>
6.	A gene construct comprising a nucleic acid sequence as claimed in any of<br>
clahns 1 to 3 or a gene construct as claimed in claim 4 or 5 and at least one further<br>
nucleic acid which encodes a fatty acid biosynthesis gene.<br><br>
7.	The gene consirucl as claimed in claim 6, wherein the nucleic acid which encodes a fatty acid biosynthesis gene is selected from the group consisting of A19-, A17-, A15-, A12-. A9-, A8-. A6-, A5-, A4-desaturase, the various hydroxylases, A12-acetylenase, ac}'l-ACP thioesterases, p-ketoacyl-ACP synthases or P-ketoacyl-ACP reductases.<br>
8.	An amino acid sequence which is encoded by the isolated nucleic acid sequence as claimed in any of claims 1 to 3 or the gene construct as claimed in any of claims 4 to 7.<br>
9.	A vector encompassing a nucleic acid as claimed in any of claims 1 to 3 or a gene construct as claimed in any of claims 4 to 7.<br>
10.	An micro organism encompassing at least one nucleic acid as claimed in any of claims 1 to 3, a gene constmct as claimed in any of claims 4 to 7 or a vector as claimed in claim 9.<br>
11.	An antibody which specifically binds a polypeptide which is encoded by one of the nucleic acids as claimed in any of claims 1 to 3.<br>
12.	An anlisense nucleic acid molecule which encompasses the complementary sequence of the nucleic acid as claimed in any of claims 1 to 3.<br>
13.	A process for the preparation of PUFAs, which comprises culturing an organism which encompasses a nucleic acid as claimed in claim 1, a gene constmct as claimed in claim 5 or a vector as claimed in claim 9, encoding a polypeptide which elongates Ci^-, Cig- or Ci,,- fatty acids with at least two double bonds in the fatty acid molecule by at least two carbon atoms under conditions under which PUFAs are formed in the organism.<br><br>
14.	The process as claimed in claim 13, wherein the PUFAs prepared by the process are Cjg-, C20- or Cir fatty acid molecules with at least two double bonds in the fatty acid molecule.<br>
15.	The process as claimed in claim 13 or 14, wherein the Cig-, C20- or C22- fatty acid molecules are isolated from the organism in the form of an oil, lipid era free fatty acid.<br>
16.	The process as claimed in any of claims 13 to 15, wherein the organism is a microorganism, an animal or a plant.<br>
17.	The process as claimed in any of claims 13 to 16, wherein the organism is a transgenic plant.<br>
18.	The process as claimed in any of claims 13 to 17, wherein theCi^-, Cig- orC20-fally acid is a fatty acid with three double bonds in the molecule.<br>
19.	An oil, lipid or fatly acid or a fraction thereof, prepared by the process as<br>
claimed in any of claims 13 10 18.<br>
20.	An oil, lipid or fatty acid composition which comprises PUFAs and which is derived from a transgenic plant.<br>
21.	An oil, lipid or fetty acid composition as claimed in claim 20, wherein the PUFAs are derived from transgenic plants which encompass a nucleotide sequence as claimed in any of claims 1 to 3, the gene construct as claimed in any of claims 4 to 7, the antisense nucleic acid molecule as claimed in claim 11 or the vector as claimed in claim 9.<br><br>
22.	A composition, comprising the aniibody as claimed in claim 11, the antisense<br>
nucleic acid constmci as claimed in claim 12 or an antagonist or agonist identified as<br>
claimed in claim 24.<br>
23.	A kit, comprising the nucleic acid as claimed in any of claims 1 to 3, the gene<br>
construct as claimed in any of claims 4 to 7, the antisense nucleic acid molecule as<br>
claimed in claim 12, the antibody as claimed in claim 11, an antagonist or agonist<br>
identified as claimed in claim 24, the composition as claimed in claim 22, the amino<br>
acid sequence as claimed in claim 8 and/or oil, lipids and/or fatty acids or a fraction<br>
thereof as claimed in any of claims 19 to 21.<br>
24.	A process for identifying an antagonist or agonist of elongases, comprising<br>
a)	bringing the cells which express the polypeptide of the present invention into<br>
contact with a candidate substance;<br>
b)	testing the PSl-activity;<br>
c)	comparing (he PSli aciivity wilh a standard activity in the absence of the candidate substance, where a PSli activity tliat is higher than that of the standard indicates that the candidate substance is an agonist and where a PSE activity that is lower than that of the standard indicates thai the candidate substance is an antagonist.<br><br></t></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTMxMC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1310-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="201061-a-standing-type-feeding-device-of-a-narrow-fabric-loom.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="201063-a-wind-power-installation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>201062</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1310/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td></td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Sep-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SYNGENTA PARTICIPATIONS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCHWARZWALDALLEE 215, CH-4058 BASEL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HOHN, THOMAS, M</td>
											<td>102 BLAKELY DRIVE, CHAPEL HILL, NC 27517</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PETERS, CHERYL</td>
											<td>6308 VALLEY ESTATES DRIVE, RALEIGH, NC 27612</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SALMERON, JOHN, MANUEL</td>
											<td>1308 BLACKBERRY LANE, HILLSBOROUGH, NC 27278</td>
										</tr>
										<tr>
											<td>4</td>
											<td>REED, JANET, N</td>
											<td>5415 LAKE VISTA DRIVE, DURHAM, NC 27712</td>
										</tr>
										<tr>
											<td>5</td>
											<td>DAWSON, JOHN, L</td>
											<td>214 EAST AVONDALE DRIVE, GREENSBORO, NC 27403</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP00/02769</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-03-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/282,995</td>
									<td>1999-03-31</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/502,852</td>
									<td>2000-02-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/201062-a-method-of-producing-a-plant-that-is-resistant-to-a-fungus-that-produces-a-trichothecene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:28:16 GMT -->
</html>
