<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/201309-method-of-producing-prenyl-alcohols by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:03:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 201309:METHOD OF PRODUCING PRENYL ALCOHOLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD OF PRODUCING PRENYL ALCOHOLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method of producing a prenyl alcohol 1164/CHENP/2003 A method of producing a prenyl alcohol, comprising a step of producing a recombinant obtained by transferring into a cell a recombinant DNA for expression or a DNA fragment for genomic integration each comprising: (i) a hydroxymethylglutaryl-CoA reductase gene, an isopentenyl-diphosphate D-isomerase gene or a famesyl-diphosphate synthase gene, or a mutant of anyone of said genes, (ii) a transcription promoter as herein described, and (iii) a transcription terminator as herein described; a step of culturing said recombinant in a known culture medium under known condition; and a step of recovering the prenyl alcohol from the resultant culture in a known manner.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br>
METHOD OF PRODUCING PRENYL ALCOHOLS<br>
TECHNICAL FIELD<br>
The present invention relates to a method of producing prenyl alcohols.<br>
BACKGROUND ART<br>
The biosynthesis of terpenoids (isoprenoids) begins with the synthesis of geranyl diphosphate (GPP; Cio), famesyl diphosphate (FPP; C15) and geranylgeranyl diphosphate (GGPP; C20), which are straight chain prenyl diphosphates, through the sequential condensation reactions of isopentenyl diphosphate (IPP; C5) with an allylic diphosphate substrate (Fig. 1). In Fig. 1, the abbreviations and words in boxes represent enzymes. Specifically, hmgR represents hydroxymethylglutaryl-CoA reductase; GGPS represents GGPP synthase; and FPS represents FPP synthase.<br>
Among prenyl diphosphates, FPP is the most significant biosynthetic intermediate. It is a precursor for the synthesis of tremendous kinds of terpenoids, e.g. steroids including ergosterol (provitamin D2), the side chains of quinone (vitamin K; VK), sesquiterpenes, squalene (SQ), the anchor molecules of famesylated proteins, and natural rubber.<br>
GGPP is also a key biosynthetic intermediate in vivo, and is essential for the biosynthesis of such compounds as retinol (vitamin A; VA), P-carotene (provitamin A), phylloquinone (vitamin Ki; VKi), tocopherols (vitamin E; VE), the anchor molecules of geranylgeranylated proteins, the side chain of chlorophyll, gibberellins, and the ether lipid of Archaea.<br>
Famesol (FOH; C15) and nerolidol (NOH; C15), which are alcohol derivatives of FPP, and geranylgeraniol (GGOH; C20), which is an alcohol derivative of GGPP, are known as fragrant substances in essential oils used as the ingredients of perfumes. FOH, NOH and GGOH are also important as the starting materials for the synthesis of various compounds (including the above-mentioned vitamins) useful as pharmacological agents (Fig. 1),<br><br>
It is desired to establish a system in which a pure product of the so-called active-type prenyl alcohol, not a mixture containing isomers, can be produced in a large quantity.<br>
Although it had been believed that all the biosynthesis of IPP is performed via the mevalonate pathway (a pathway in which IPP is synthesized from acetyl-CoA through mevalonate), M. Rohmer et aL elucidated a novel pathway for IPP synthesis using bacteria at the end of 1980's. This is called non-mevalonate pathway or DXP (1-deoxyxylulose 5-phosphate) pathway, in which IPP is synthesized from glyceraldehyde-3-phosphate and pyruvate through 1-deoxyxylulose 5-phosphate.<br>
FOH and NOH are currently produced by chemical synthesis except for small amounts of them prepared from natural products such as essential oils. Chemically synthesized FOH and NOH generally have the same carbon skeletons, but they are obtained as mixtures of(E) type {trans type) and (Z) type (cis type) in double bond geometry. (E, £)-FOH or (£)-NOH, both of which are of (all-E) type, is the form synthesized in metabolic pathways in organisms and is industrially valuable. In order to obtain (E, £)-FOH or (£)-NOH in a pure form, refining by column chromatography, high precision distillation, etc, is necessary. However, it is difficult to carry out high precision distillation of FOH, a thermolabile ally I alcohol, or its isomer FOH. Also, the refining of these substances by column chromatography is not suitable in industrial practice since it requires large quantities of solvent and column packings as well as complicated operations of analyzing and recovering serially eluting fractions and removing the solvent; thus, this method is complicated and requires high cost. Under circumstances, it is desired to establish a method of biosynthesis of (E, £)-FOH (hereinafter, just referred to as "FOH") by controlling the production of (£)- and (Z)-geometrical isomers or by utilizing the repeat structure of reaction products. However, such a method has not been established yet. The substrates for FOH synthesis are provided via the mevalonate pathway in cells of, for example, Saccharomyces cerevisiae, a budding yeast. However, even when HMG-CoA reductase that is believed to be a key enzyme for FOH synthesis was used, it has only been discovered that the use of the reductase increases squalene synthesis ability (Japanese Unexamined Patent Publication No. 5-192184; Donald et aL, (1997) Appl. Environ. Microbiol. 63, 3341-3344).    Further, even when a squalene synthase gene-deficient<br><br>
strain of a special budding yeast that had acquired sterol intake ability was cultured, accumulation of 1.3 mg of FOH per liter of culture broth was only revealed (Chambon et ai, (1990) Curr. Genet. 18, 41-46); no method of biosynthesis of (E)-NOH (hereinafter, just referred to as "NOH") has been known.<br>
DISCLOSURE OF THE INVENTION<br>
It is an object of the invention to provide a method for producing a prenyl alcohol by culturing a recombinant prepared by transferring into a host cell a recombinant DNA for expression comprising an HMG-CoA reductase gene, an IPP A-isomerase gene or an FPP synthase gene, or a mutant of any one of these genes.<br>
As a result of intensive and extensive researches toward solution of the above problems, the present inventors attempted to develop a prenyl alcohol production system by introducing into a host a gene of an enzyme involved in prenyl diphosphate synthesis. As the host, an unicellular eucaryote, in particular, yeast or procaryotes (such as bacterium, in particular, E. coll) that had been widely used in the fermentation industry from old times, that carries out the synthesis of prenyl diphosphate via the mevalonate pathway or DXP pathway; and that can be subjected to various genetic engineering techniques was used. In order to construct systems with which a gene of an enzyme involved in prenyl diphosphate synthesis {e.g., HMG-CoA reductase gene) in yeast can be expressed artificially in a host cell, expression shuttle vectors were created which comprised a constitutive or inducible transcription promoter and various auxotrophic markers. Then, a gene of interest or a mutant thereof was inserted into these vectors, which were then introduced into various host cells. The inventors have succeeded in obtaining NOH or FOH from the culture of the resultant recombinant. Thus, the above-mentioned object has been achieved, and the present invention has been completed. When E. coli was used as a host, a gene of an enzyme involved in prenyl diphosphate synthesis [e.g., FPP synthase gene or IPP A-isomerase gene) was introduced into the host cell using a conventional vector. Then, FOH was obtained from the culture of the resultant recombinant after dephosphorylation. Thus, the above-mentioned object has been achieved, and the present invention has been completed.<br><br>
The present invention relates to a method of producing a prenyl a]cohol(s), comprising creating a recombinant obtained by introducing into a host a recombinant DNA(s) for expression or a DNA fragment(s) for genomic integration each comprising: (i)    a hydroxymethylglutaryl-CoA reductase gene, an isopentenyl-diphosphate A-isomerase<br>
gene or a famesyl-diphosphate synthase gene, or a mutant of any one of these genes, (ii)    a transcription promoter, and (iii)    a transcription terminator;<br>
culturing the recombinant; and recovering the prenyl alcohol(s) from the resultant culture. Specific examples of the prenyl alcohol include C15 prenyl alcohols such as FOH or NOH. Specific examples of the HMG-CoA reductase gene and mutant thereof include a gene encoding the amino acid sequence as shown in SEQ ID NO: 2, 4 or 6, or a deletion mutant thereof For example, an HMG-CoA reductase gene comprising one nucleotide sequence selected from the group consisting of SEQ ID NOS: 1, 3, 5 and 7-16 may be given. Specific examples of the FPP synthase gene or mutant thereof include a gene encoding the amino acid sequence as shown in SEQ ID NO: 76, 78, 80, 82 or 84. For example, an FPP synthase gene comprising one nucleotide sequence selected from the group consisting of SEQ ID NOS: 75, 77, 79, 81 and 83 may be given. Specific examples of the IPPA-isomerase gene or mutant thereof include a gene encoding the amino acid sequence as shown in SEQ ID NO: 86. For example, an IPPA-isomerase gene comprising the nucleotide sequence as shown in SEQ ID NO: 85 may be given. As the transcription promoter, one selected from the group consisting of ADHl promoter, TDH3 (GAP) promoter, PGKJ promoter, TEF2 promoter, GALl promoter and tac promoter may be used. Other transcription promoters may also be used which are functionally equivalent to these promoters in activity. As the transcription terminator, ADHl terminator or CYCl terminator may be used. Other transcription terminators may also be used which are functionally equivalent to these terminators in activity. As the host, yeast may be used, e.g. budding yeast such as Saccharomyces cerevisia. Specific examples of preferable S. cerevisiae strains include A451, YPH499, YPH500, W303-1A and W303-1B, or strains derived therefrom. Alternatively, a bacterium, e.g. Escherichia coli may be used. Specific examples of preferable E. coli strains include JM109 or strains derived therefrom.<br><br>
According to the present invention, it is possible to produce a prenyl alcohol such as NOH or FO.H at a concentration that cannot be achieved by merely culturing the untransformed host cell (at least 0.05 mg/L medium).<br>
Further, the present invention relates to a recombinant obtained by transferring into a<br>
host a recombinant DNA for expression or a DNA fragment for genomic integration each<br>
comprising:<br>
(i)    a hydroxymethylglutaryl-CoA reductase gene, an isopentenyl-diphosphate A-isomerase<br>
gene or a famesyl-diphosphate synthase gene, or a mutant of any one of these genes,<br>
(ii)    a transcription promoter, and<br>
(iii)    a transcription terminator, the recombinant being capable of producing at least 0.05 mg/L of FOH or NOH.    Specific<br>
examples of the host, the promoter and the terminator are the same as described above.<br>
Hereinbelow, the present invention will be described in detail. The present specification encompasses the contents described in the specification and the drawings of Japanese Patent Application No. 2000-401701 based on which the present application claims priority.<br>
The inventors have attempted to develop a system with which an active-type prenyl alcohol (i.e., (all-E)-prenyl alcohol) can be produced in vivo, by using metabolic engineering techniques. Generally, FPP is synthesized by the catalytic action of famesyl-diphosphate synthase (FPS) from IPP and DMAPP (3,3-dimethylanyl diphosphate) as substrates. Usually, this reaction does not proceed toward the synthesis of FOH, but proceeds toward the synthesis of squalene by squalene synthase, the synthesis of GGPP by geranygeranyl-diphosphate synthase, the synthesis of hexaprenyl diphosphate by hexaprenyl-diphospfcate synthase, and so on (Fig. 1). In the present invention, transformant cells capable of producing not the usually expected squalene or major final products (sterols) but prenyl alcohols such as NOH and FOH not indicated in conventional metabolic pathway maps have been obtained by introducing into host cells an HMG-CoA reductase gene, FPP synthase gene or IPP A-isomerase gene that are believed to be involved in the activation of prenyl diphosphate synthesis via two different, independent pathways (the mevalonate<br><br>
pathway and DXP pathway) depending on organisms. Thus, biological, mass-production systems for prenyl alcohols have been developed. Furthermore, deletion mutants of HMG-CoA reductase gene with various paUerns of deletions (Fig. 2) have been introduced into hosts in such a manner that the genes come under the control of a transcription promoter; or mutants of FPP synthase with amino acid substitutions have been introduced into hosts. Thus, biological, mass-production systems for the above-mentioned prenyl alcohols have been developed.<br>
1. Preparation of Recombinant DNAs for Expression or DNA Fragments for Genomic Integration In the present invention, the recombinant DNA for expression used in the transformation of hosts may be obtained by ligating or inserting a transcription promoter DNA and a transcription terminator DNA into a gene of interest to be expressed. Specifically, the gene to be expressed may be, for example, an HMG-CoA reductase genes (e.g., HMGI), Escherichia coli FPP synthase gene ispA, Bacillus stearothermophilus FPP synthase gene or IPPA-isomerase gene idi (ORF182) (hereinafter, referred to as an "HMG-CoA reductase gene or the like"). These genes can be isolated by cloning tecliniques using PCR or commercial kits.<br>
It is also possible to prepare in advance a gene expression cassette comprising an HMG-CoA reductase gene or the like to which a transcription promoter and a transcription terminator have been ligated, and to incorporate the cassette into a vector. The ligation of the promoter and the terminator may be performed in any order. However, the promoter is ligated upstream of the HMG-CoA reductase eene or the like, and the terminator downstream of the gene. Alternatively, in the present invention, an HMG-CoA reductase gene or the like, a transcription promoter and a transcription terminator may be incorporated into an appropriate DNA, e.g. a vector, in succession. If the direction of transcription is properly considered, the incorporation may be performed in any order<br>
The DNA used for this purpose is not particularly limited as long as it may be retained in host cells hereditarily.    Specific examples of DNA that may be used include plasmid DNA, bacteriophage, retrotransposon DNA and artificial chromosomal DNA (YAC: yeast artificial chromosome).    With respect to recombinant DNA fragments for the gene expression by genomic integration, replication ability is not necessarily required in that DNA.   The DNA<br><br>
ents prepared by PCR or chemical synthesis may also be used.<br>
Specific examples of useful plasmid DNA include YCp-type E, co//-yeast shuttle 5 such as pRS413, pRS414, pRS415, pRS416, YCp50, pAUR112 or pAUR123; type E. coli-ytast shuttle vectors such as pYES2 or YEpl3; YIp-type E. co/z-yeast<br>
vectors such as pRS403, pRS404, pRS405, pRS406, pAURlOl or pAUR135; E. :rived plasmids such as ColE plasmids (e.g., pBR322, pBR325, pUC18, pUC19, 8, pUC119, pTVllSN, pTV119N, pBluescript, pHSG298, pHSG396 or pTrc99A), plasmids (e.g., pACYC177 or pACYC184) and pSClOl plasmids (e.g., pMW118, 19, pMW218 or pMW219); and Bacillus subtilis-derived plasmids (e.g., pUBllO, Specific examples of useful phage DNA include X phage (Charon4A, Charon21A, 3, EMBL4, X-gtlO, Xgtll, XZAP), (t)X174, M13mpl8 and M13mpl9. Specific les of useful retrotransposon DNA include Ty factor. Specific examples of YAC J include pYACC2.<br>
^hen recombinant DNAs are introduced into hosts, selection marker genes are used in :ases.    However, the use of the marker genes are not necessarily required if there is an riate assay to select recombinants. \s the transcription promoter, a constitutive promoter or an inducible promoter may be<br>
The "constitutive promoter" means a transcription promoter of a gene involved in a metabolic pathway. Such a promoter is believed to have transcription activity under owth conditions. The "inducible promoter" means a promoter that has transcription  only under specific growth conditions and whose activity is suppressed under other  conditions.<br>
Any transcription promoter may be used as long as it has activity in hosts such as yeast, ample, GAL] promoter, GALIO promoter, TDH3 (GAP) promoter, ADHl promoter, promoter or TEF2 promoter may be used to direct expression in yeast. To direct sion in E. coli, trp promoter, lac promoter, tix promoter or tac promoter may be used, mple.<br>
The recombinant DNA may further comprise cis-elements such as an enliancer, a splicing signal, a poly A addition signal, selection markers, or the like, if desired. Specific examples of useful selection markers include marker genes such as URA3, LEU2, TRP] and H1S3 that have non-auxotrophic phenotypes as indicators, and drug resistance genes such as<br><br>
Amp, Tet' Cm' Km and A URl-C.<br>
A transcription terminator derived from any gene may be used as long as it has activity in hosts such as yeast. For example, ADHJ terminator or CYC] terminator may be used to direct the expression in yeast. To direct the expression in E. coli, nnB terminator may be used, for example. It is also possible to incorporate an SD sequence (typically, 5'-AGGAGG-3') upstream of the initiation codon of the gene of a bacterium (eg., E. coli) as a ribosome binding site for translation.<br>
Expression vectors prepared in the present invention as recombinant DNAs for gene transfer may be designated and identified by indicating the name of the gene after the name of the plasmid used, unless otherwise noted. For example, when HMGl gene has been ligated to plasmid pRS434GAP having TDH3 (GAP) promoter, the resultant plasmid is expressed as "pRS434GAP-HMGr'. Except for special cases, this notational system applies to other expression vectors comprising other plasmids, promoters and genes.<br>
In the present invention, an HMG-CoA reductase gene or the like may be a mutant in which a part of its regions (2217 nucleotides at the maximum) has been deleted, or a mutant that has deletion, substitution or addition of one or several to ten-odd nucleotides in the nucleotide sequence of a wild-type gene or a deletion mutant thereof With respect to amino acid sequences, an HMG-CoA reductase may be a deletion mutant in which 739 amino acids at the maximum have been deleted in the amino acid sequence of a wild-type HMG-CoA reductase (SEQ ID NO: 2), or it may be a mutant that has deletion, substitution or addition of one or several (e.g., one to ten, preferably one to three) amino acids in the amino acid sequence of the wild-type enzyme or a deletion mutant thereof. Specifically, an HMG-CoA reductase gene may be a wild-type gene or a deletion mutant thereof as shown in Fig. 2B. Also, the amino acid sequence encoded by such a gene may have site-specific substitution(s) at one to ten sites as a result of nucleotide substitution(s), for example, as shown in Fig. 2A. An FPP synthase gene may also be a mutant that has deletion, substitution or addition of one or several to ten-odd nucleotides. Specifically, various mutant genes (SEQ ID NOS: 79, 81 and 83) each of which has substitution of five nucleotides in a wild-type FPP synthase gene (SEQ ID NO: 77) may be used.   These mutant genes encode mutant enzymes in which the<br><br>
79th amino acid residue Tyr of the wild-type FPP synthase (SEQ ID NO: 78) has been changed to Asp (SEQ ID NO: 80), Glu (SEQ ID NO: 82) or Met (SEQ ID NO: 84), respectively.<br>
Substitution mutations of nucleotides that occur in DNA fragments obtained by amplifying wild-type DNA by PCR (polymerase chain reaction) using a DNA polymerase of low fidelity, such as Taq DNA polymerase, are called "PCR errors". In the present invention, for example, an HMG-CoA reductase gene in which encoded polypeptide has substitution mutations attributable to those nucleotide substitutions resulted from PCR errors when a wild-type HMG-CoA reductase gene (SEQ ID NO: 1) was used as a template may also be used. This HMG-CoA reductase gene is called 'HMGl' ". An embodiment of nucleotide substitutions resulted from PCR errors when the wild-type HMG-CoA reductase gene (SEQ ID NO: 1) was used as a template is shovm in Fig. 2A. HMGl' has the nucleotide sequence as shown in SEQ ID NO: 3, and the amino acid sequence encoded thereby is shown in SEQ ID NO: 4. In Fig. 2A, the mutations of nucleotides are expressed in the following order: the relevant nucleotide before substitution (in one letter abbreviation), the position of this nucleotide when the first nucleotide in the initiation codon of the HMG-CoA reductase gene is taken as position 1, and the nucleotide after substitution (in one letter abbreviation). The mutations of amino acids contained in the amino acid sequence of tne PCR error-type HMG-CoA reductase are expressed in the following order: the relevant amino acid residue before substitution (in one letter abbreviation), the position of this amino acid in the HMG-CoA reductase, and the amino acid residue after substitution (in one letter abbreviation). Further, the PCR error-type nucleotide sequence described above may be corrected partially by techniques such as site-directed mutagenesis. Such a corrected HMG-CoA reductase gene may also be used in the invention. Further, those HMG-CoA reductase genes (including PCR error-type) may also be used in the invention that encode deletion mutants in which predicted transmembrane domains are deleted. For example, Fig. 2B shows examples of HMGl A genes that are deletion mutants of the PCR error-type HMG-CoA reductase gene HMGl . In Fig. 2B, the upper most row represents HMGl' gene without deletion.. The portion indicated with thin solid line (—) is the deleted region.    Table 1 below shows which<br><br><br><br>
2.    Preparation of Recombinants<br>
The recombinant of the invention can be obtained by introducing into a host the recombinant DNA of the invention in such a manner that the HMG-CoA reductase gene or the like (including various mutants; the same applies to the rest of the present specification unless otherwise noted) can be expressed. The host used in the invention is not particularly limited. Any host may be used as long as it can produce a prenyl alcohol(s). Preferably, E. coli or yeast is used.<br>
In the present invention, the recombinant DNA comprising a promoter, an HMG-CoA reductase gene or the like, and a terminator may be introduced into fungi including unicellular eucaryotes such as yeast; procaryotes such as E. coli; animal cells; plant cells; eic, to obtain recombinants.<br>
Fungi useful in the invention include Myxomycota, Phycomycetes, Ascomycota, Basidiomycota, and Fungi Imperfecti. Among fungi, some unicellular eucaryotes are well known as yeast that is important in industrial applicability. For example, yeast belonging to Ascomycota, yeast belonging to Basidiomycota, or yeast belonging to Fungi Imperfecti may be enumerated. Specific examples of yeast include yeast belonging to Ascomycota, in particular, budding yeast such as Saccharomyces cerevisiae (known as Baker's yeast), Candida utilis or Pichia pastris; and fission yeast such as Shizosaccharomyces pombe. The yeast strain is not particularly limited as long as it can produce a prenyl alcohol(s). In the case of 5. cerevisiae^ specific examples of useful strains include A451, EUG8, EUG12, EUG27, YPH499, YPH500, W303-1A, W303-1B and AURGGlOl strains as shown below. As a method for introducing the recombinant DNA into yeast, such method as electroporation, the spheroplast method, or the lithium acetate method may be employed.<br>
A451 (ATCC200589; MATa can] leu2 trpl ura3 aroT)<br>
YPH499 (ATCC76625; MATa ura3-52 lys2-80l ade2A0l trp]-A63 his3-A200 leu2'Al; Stratagene, La Jolla, CA)<br>
YPH500 (ATCC76626; MATa ura3-52 lys2'SQ\ ade2-lQ\ trpI'A63 his3-A200 Ieu2-A\\ Stratagene)<br><br><br>
As prokaryotes, archaea and bacteria may be enumerated. As archaea, methane producing microorganisms such as Metanobacierium; halophilic microorganisms such as Halobacterium, thermophilic acidophilic microorganisms such as Sulfolobus, may be enumerated. As bacteria, various Gram-negative or Gram-positive bacteria that are highly valuable in industrial or scientific applicability may be enumerated, e.g. Escherichia such as E. coli. Bacillus such as B, subtilis or B. brevis, Pseudomonas such as P, putida, Agrobacterium such as A. tumefaciens or A. rhizogenes, Corynebacterium such as C. glutamicum, Lactobacillus such as L plantarum, and Actinomycetes such as Actinomyces ox Streptmyces.<br>
When a bacterium such as E. coli is used as a host, the recombinant DNA of the invention is preferably not only capable of autonomous replication in the host but also composed of a promoter, an SD sequence as a ribosome RNA binding site, and the gene of the invention. A transcription terminator may also be inserted appropriately. The recombinant DNA may also contain a gene that controls the promoter. Specific examples of E. coli strains include, but are not limited to, BL21, DH5a, HBlOl, JMlOl, MBV1184, TH2, XLl-Blue and Y-1088. As the transcription promoter, any promoter may be used as long as it can direct the expression of a gene in a host such as E. coli. For example, an E. coli- or phage-derived promoter such as trp promoter, lac promoter, PL promoter or PR promote may be used. An artificially altered promoter such as tac promoter may also be used. As a method for introducing the recombinant DNA into a bacterium, any method of DNA transfer into bacteria may be used. For example, a method using calcium ions, electroporation, or a method using a commercial kit may be employed.<br>
Whether the gene of the invention has been transferred into the host cell or not<br><br>
can be confirmed by such methods as PCR or Southern blot hybridization. For example, DNA is prepared from the resultant recombinant, designed a primer(s) specific to the introduced DNA and subjected to PCR. Subsequently, the amplified product is subjected to agarose gel electrophoresis, polyacrylamide gel electrophoresis or capillary electrophoresis, followed by staining with ethidium bromide, SYBR Green solution or the like, or detection of DNA with a UV detector. Thus, by detecting the amplified product as a single band or peak, the introduced DNA can be confirmed. Alternatively, PCR may be performed using a primer(s) labeled with a fluorescent dye or the like to detect the amplified product.<br>
3.     Production of Prenyl Alcohols<br>
In the present invention, a prenyl alcohol(s) can be obtained by culturing the above-described recombinant comprising a transferred HMG-CoA reductase gene or the like, and recovering the prenyl alcohol(s) from the resultant culture. The term "culture" used herein means any of the following materials: culture supernatant, cultured cells or microorganisms per se, or disrupted products from cultured cells or microorganisms. The recombinant of the invention is cultured by conventional methods used in the culture of hosts. As the prenyl alcohol, C15 prenyl alcohols such as farnesol (FOH) or nerolidol (NOH) may be enumerated. These prenyl alcohols are accumulated in the culture independently or as a mixture.<br>
As a medium to culture the recombinant obtained from a microorganism host, either a natural or synthetic medium may be used as long as it contains carbon sources, nitrogen sources and inorganic salts assimilable by the microorganism and is capable of effective cultivation of the recombinant. As carbon. sources, carbohydrates such as glucose, galactose, fructose, sucrose, raffinose, starch; organic acids such as acetic acid, propionic acid; and alcohols such as ethanol and propanol may be used. As nitrogen sources, ammonia; ammonium salts of inorganic or organic acids such as ammonium chloride, ammonium sulfate, ammonium acetate, ammonium phosphate; other nitrogen-containing compounds; Peptone; meat extract; com steep liquor, various amino acids, etc. may be used.<br><br>
As inorganic substances, potassium dihydrogcn phosphate, clipotassium hydrogen phosphate, magnesium phosphate, magnesium sulfate, sodiuni chloride, iron(II) sulfate, manganese sulfate, copper sulfate, calcium carbonate and the like may be used. Usually, the recombinant is subjected to shaking culture or aeration agitation culture under aerob'c conditions at 26 to 36 °C. Preferably, when the host is S. cerevisiae, the recombinant is cultured at 30°C for 2 to 7 days. When the host is E. coli, the recombinant is cultured at 37°C for 12 to 18 hours. The adjustment of pH is carried out using an inorganic or organic acid, an alkali solution or the like. During the cultivation, antibiotics such as ampicillin, chloramphenicol or aureobasidin A may be added to the medium if necessary.<br>
When a recombinant incorporating an expression vector containing an inducible transcription promoter is cultured, an inducer may be added to the medium if necessary. For example, when GALl promoter was used, galactose may be used as a carbon source. When a microorganism (e.g.. E. coli) transformed with an expression vector containing a promoter that is inducible by isopropyl-P-D-thiogalactopyranoside (IPTG) is cultured, IPTG may be added to the medium.<br>
When cultured under the above-described conditions, the recombinant of the invention can produce prenyl alcohol(s) at high yield(s). In particular, when the host is AURGGlOl and the vector is pYHMG044, the recombinant can produce 32 mg or more of prenyl alcohols per liter of the medium. It can produce even 150 mg/L or more depending on the culture conditions.<br>
In the present invention, it is possible to increase the production efficiency of prenyl alcohols by adding to the above-described medium such substances as terpenoids, oils, or surfactants.    Specific examples of these additives include the following.<br>
Terpenoids: squalene, tocopherol, IPP, DMAPP<br><br><br>
The concentrations of oils are 0.01% or more, preferably 1-3%. The concentrations of surfactants are 0.005-1%, preferably 0.05-0.5%). The concentrations of terpenoids are 0.01% or more, preferably 1-3%.<br>
After the cultivation, the prenyl alcohol of interest is recovered by disrupting the microorganisms or cells by, e.g., homogenizing, when the alcohol(s) is produced within the microorganisms or cells. Alternatively, the alcohol(s) may be extracted directly using organic solvents without disrupting the cells. When the prenyl alcohol(s) of the invention is produced outside the microorganisms or cells, the culture broth is used as it is or subjected to centrifugation or the like to remove the microorganisms or cells. Thereafter, the prenyl alcohol(s) of interest is extracted from the culture by, e.g., extraction with an organic solvent. If necessary, the alcohol(s) may be further isolated and purified by various types of chromatography or the like.<br>
In the present invention, preferable combinations of host strains and vectors as recombinant DNAs, as well as relationships between these combinations and yields of prenyl alcohols are as illustrated in Table 2 below.<br><br><br><br>
From Table 2, the following yields can be presented, for example.<br>
(1)	When a DNA comprising HMGl or a mutant thereof (e.g., HMGl') or a deletion mutant of this mutant (HMGxxy) ligated downstream of a constitutive promoter had been introduced into S. cerevisiae cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-18.3 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-0.3 mg/L.<br>
(2)	When a DNA comprising HMGl or a mutant thereof (e.g., HMGl') or a deletion mutant of this mutant (HMGxxy) ligated downstream of an inducible promoter had been introduced into S. cerevisiae cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-158 mg/L, more preferably at 53-158 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-23 mg/L, more preferably at 2.4-23 mg/L.<br>
(3)	When two DNAs comprising HMGl ligated downstream of a constitutive promoter and HMG04y (a deletion mutant of HMGl ) ligated downstream of an inducible promoter, respectively, had been introduced into S. cerevisiae cells, the cells produced FOH at least at 22 mg/L, preferably at 22-66 mg/L, and produced NOH at least at 12 mg/L, preferably at 12-28 mg/L.<br>
(4)	When a DNA comprising HMGl or a mutant thereof (e.g., HMGl 0 or a deletion mutant of this mutant (HMGxxy) had been introduced into S. cerevisiae A45] cells or A451-derived cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-158 mg/L, more preferably at 53-158 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-23 mg/L, more preferably at 2.4-23 mg/L.<br>
(5)	When a DNA comprising HMGl or a mutant thereof (e.g., HMGl') or a deletion mutant of this mutant (HMGxxy) had been introduced into S, cerevisiae YPH499 cells or YPH499-derived cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-18.3 mg/L, more preferably at 5.9-18.3 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.13-0.30 mg/L.<br><br>
(6)	When a DNA comprising HMGl or a mutant thereof (e.g., HMGl') or a deletion mutant of this mutant (HMGxxy) had been introduced into S. cerevisiae YPH500 cells or YPH500-derived cells, the cells produced FOH at least at 3.2 mg/L, preferably at 3.2-13.6 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-0.22 mg/L.<br>
(7)	When a DNA comprising HMGl or a mutant thereof (e.g., //A/Gi ') had been introduced into S. cerevisiae cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-18.3 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-2.7 mg/L.<br>
(8)	When a DNA comprising HMGxxy (a deletion mutant oi HMGl') had been introduced into S. cerevisiae cells, the cells produced FOH at least at 0.05 mg/L, preferably at 0.05-158 mg/L, more preferably at 53-158 mg/L, and produced NOH at least at 0.05 mg/L, preferably at 0.05-23 mg/L, more preferably at 2.4-23 mg/L.<br>
(9)	A plasmid comprising a substitution mutant of E, coli FPP synthase gene ispA was introduced into E. coli. When the resultant cells were cultured in a liquid medium containing IPP and DMAPP and then treated with phosphatase, the cells produced FOH at least at 11 mg/L, preferably at 11-90 mg/L, more preferably at 64-90 mg/L.<br>
(10)	When ispA and idi had been introduced into E. coli, the cells produced FOH at least at<br>
0.15 mg/L, preferably at 0.15-0.16 mg/L, as a result of phosphatase treatment even without<br>
the addition of IPP and DMAPP.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig.   1   is   a   diagram   showing   a   metabolic   pathway   in   which   mevalonate<br>
pathway-related enzymes are involved.<br>
Fig. 2A is a diagram showing construction of deletion mutants of HMGl gene.<br>
Fig. 2B shows patterns of substitution mutations. Fig. 3 is a diagram showing plasmid pRS414.<br><br><br><br>
Fig. 15 presents graphs showing prenyl alcohol yields when pRS434GAP-HMGl or pRS444GAP-HMGlhas been transferred into A451 strain.<br>
Fig. 16 presents graphs showing prenyl alcohol yields when pRS414PTadh-HMGl, pRS414TPadh-HMGl, pRS434GAP-HMGl, pRS444GAP-HMGl, pRS434PGK-HMGU pRS444PGK-HMGl, pRS434TEF-HMGl or pRS444TEF-HMGl has been transferred into YPH499 strain.<br>
Fig. 17 presents graphs showing prenyl alcohol yields when pRS434GAP-HMGl or pRS444GAP-HMGl has been transferred into EUG8 strain.<br>
Fig. 18 presents graphs showing prenyl alcohol yields when pRS434GAP-HMGl or pRS444GAP-HMGl has been transferred into EUG12 strain.<br>
Fig. 19 presents graphs showing prenyl alcohol yields when pRS434GAP-HMGl or pRS444GAP-HMGl has been transferred into EUG27 strain.<br>
Fig. 20A presents graphs showing prenyl alcohol yields when pYES-HMGl or pYHMG044 has been transferred into A451 strain.<br>
Fig. 20B presents graphs showing prenyl alcohol yields when pYES-HMGl or pYHMG044 has been transferred into AURGGlOl strain.<br>
Fig. 21 presents graphs showing prenyl alcohol yields when pYES-HMGl has been transferred into W303-1A or W303-1B.<br>
Fig. 22 presents graphs showing prenyl alcohol yields when pYHMG026, pYHMG044, pYHMG056, pYHMG062, pYHMG076, pYHMGOSl, pYHMGlOO, pYHMGll2 or pYHMG122 has been transferred into A451 strain.<br>
Fig. 23 presents graphs showing prenyl alcohol yields when pYHMG026, pYHMG044, pYHMG056, pYHMG062, pYHMG076, pYHMGOSl, pYHMGlOO, pYHMG112 or pYHMG122 has been transfened into AURGGlOl strain.<br>
Fig. 24 presents graphs showing prenyl alcohol yields when pYHMG026, pYHMG044, pYHMG056, pYHMG062, pYHMG076, pYHMGOS 1, pYHMG 100, pYHMG112 or pYHMG122 has been transferred into AURGGlOl strain (the graphs in Fig. 23 are enlarged).<br>
Fig. 25 is a graph showing prenyl alcohol yields when pRS434GAP"HMGl or pRS444GAP-HMGl has been introduced into AURGGlOl strain together with pYHMG044.<br>
Fig. 26 is a graph showing prenyl alcohol yields when a mutant ispA gene-transferred E. coli was cultured in a liquid medium containing IPP and DMAPP.<br><br>
Fig. 27 is a graph showing prenyl alcohol yields when a mutant ispA gene-transferred E coll was cultured in a liquid medium without IPP and DMAPR<br>
Fig. 28 is a graph showing prenyl alcohol yields and cell counts when a recombinant 15-2 clone (pYHMG044/AURGG101) was cultured in ajar fermenter.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
Hereinbelow, the present invention will be described more specifically with reference to the following Examples. However, the technical scope of the present invention is not limited to these Examples.<br>
[EXAMPLE 1]    Construction of Expression Vectors<br>
Vectors were constructed using E. coli SURE2 supercompetent cells purchased from Stratagene (La Jolla, CA) as a host. For the preparation of genomic DNA from S. cerevisiae and for testing the introduction of resultant vectors, YPH499 strain (Stratagene) was used.<br>
(1)	E. coli'S. cerevisiae Shuttle Vectors<br>
Plasmids pRS404 and pRS414 (Fig. 3) were purchased from Stratagene. Plasmid pAUR123 was purchased from Takara, and plasmid pYES2 (Fig. 4) was purchased from Invitrogen (Carlsbad, CA).<br>
(2)	Genomic DNA<br>
Dr. GenTLE , a genomic DNA preparation kit for yeast, was purchased from Takara. Genomic DNA was prepared from S. cerevisiae YPH499 according to the protocol attached to the kit.<br>
(3)   Insertion of ADHJp-ADHJt Fragment into pRS414<br>
Plasmid pRS414 (Fig. 3) was digested with Nael and Pvull to obtain a 4.1 kbp fragment without fl ori and LacZ moieties. This fragment was purified by agarose gel electrophoresis. Plasmid pAUR123 was digested with BamHl and blunt-ended with Klenow enzyme.    Then, a TO kbp fragment containing ADHl transcription promoter (ADHIip) and<br><br>
ADHl transcription tenninator (ADHJi) (Fig. 5; SEQ ID NO: 17) was purified by agarose gel electrophoresis. The 4.1 kbp fragment from pRS414 still retained the replication origins for E. coli and yeast, a transfonnation marker Amp' for E. coli, and an auxotrophic marker TRP] for yeast. On the other hand, the 1.0 kbp fragment from pAUR123 contained ADHlp, ADHh, and a cloning site flanked by them. These two fragments were ligated to each other with a DNA ligation kit (Takara) and transformed into SURE2 cells.<br>
Plasmid DNA was prepared from the resultant recombinant. Mapping of the DNA with Sail and Seal revealed that the ADHlp-ADHt fragment has been inserted into pRS414 in opposite directions to thereby yield two plasmids pRS414PTadh and pRS414TPadh (Fig. 6).<br>
(4)    Insertion of CYClt Fragment into pRS Vectors<br>
CYClX  (CYCl  transcription  terminator)  fragment  was prepared  by  PCR.     The<br>
following oligo-DNAs, XhoI-TcyclFW and Apal-TcyclRV, were used as PCR primers.    As<br>
a template, pYES2 was used.<br>
XhoI-TcyclFW : 5'- TGC ATC TCG AGO GCC GCA TCA TGT AAT TAG -3'<br>
(SEQ ID NO:  18)<br>
ApaI-TcyclRV:5'- CAT TAG GGC CCG GCC GCA AAT TAA AGC CTT CG<br>
-3'  (SEQ ID NO: 19)<br>
Briefly, 50 \i\ of a reaction solution containing 0.1 \xg of pYES2, 50 pmol of each primer DNA, Ix Pfu buffer containing MgSO4 (Promega, Madison, WI), 10 nmol dNTPs, 1.5 units of Pfu DNA polymerase (Promega) and 1 µl of Perfect Match polymerase enhancer (Stratagene) was prepared. The reaction conditions were as follows: first denaturation at 95°C for 2min; 30 cycles of denaturation at 95°C for 45 sec, annealing at 60°C for 30 sec, and extension at 72°C for 1 min; and final extension at 72°C for 5 min. After completion of the reaction, the solution was stored at 4°C. The amplified DNA was digested with Xhol and Apal, and the resuhant 260 bp DNA fragment was purified by agarose gel electrophoresis to obtain CYC It-XA.<br>
CYClt-XA was inserted into the Xhol-Apal site of pRS404 and pRS406 to thereby obtain pRS404Tcyc and pRS406Tcyc, respectively.<br><br>
(5)    Preparation of Transcription Promoters<br>
DNA fragments comprising transcription promoters were prepared by PCR using pAUR123 or yeast genomic DNA as a template.    The DNA primers used are as follows.<br>
SacI-PadhlFW: 5'-GAT CGA GCT CCT CCC TAA CAT GTA GGT GGC GG-3' (SEQ ID NO: 20)<br>
SacII-PadhlRV: 5'-CCC GCC GCG GAG TTG ATT GTA TGC TTG GTA TAG C-3' (SEQ ID NO: 21)<br>
SacI-Ptdh3FW: 5'-CAC GGA GCT CCA GTT CGA GTT TAT CAT TAT CAA-3' (SEQ ID NO: 22)<br>
SacII-Ptdh3RV: 5'-CTC TCC GCG GTT TGT TTG TTT ATG TGT GTT TAT TC -3' (SEQ ID NO: 23)<br>
SacI-PpgklFW: 5'-TAG GGA GCT CCA AGA ATT ACT CGT GAG TAA GG-3' (SEQ ID NO: 24)<br>
SacII-PpgklRV: 5'-ATA ACC GCG GTG TTT TAT ATT TGT TGT AAA AAG TAG-3' (SEQ ID NO: 25)<br>
SacI-Ptef2FW: 5'-CCG CGA GCT CTT ACC CAT AAG GTT GTT TGT GAC G-3 (SEQ ID NO: 26)<br>
SacII-Ptef2RV: 5'-CTT TCC GCG GGT TTA GTT AAT TAT AGT TCG TTG ACC-3' (SEQ ID NO: 27)<br>
For the amplification of ADHI transcription promoter (ADHIp), SacI-PadhlFW and SacII-PadhlRV were used as PCR primers and pAUR123 as a template. For the amplification of TDH3 (GAP) transcription promoter (TDHSp (GAPp)), SacI-Ptdh3FW and SacII-Ptdh3RV were used as PCR primers; for the amplification of PGKJ transcription promoter (PGK]p), SacI-PpgklFW and SacII-PpgklRV were used as PCR primers; and for the amplification of TEF2 transcription promoter {TEF2p), SacI-Ptef2FW and SacII-Ptef2RV were used as PCR primers. For these promoters, yeast genomic DNA was used as a template. As a reaction solution, a 100 jil solution containing 0.1 )ig of pAUR123 or 0.46 µg of yeast genomic DNA, 100 pmol of each primer DNA, 1 x ExTaq buffer (Takara), 20 nmol dNTPs, 0.5 U of ExTaq DNA polymerase (Takara) and 1 µl of Perfect Match polymerase enhancer was prepared. The reaction conditions were as follows: first denaturation at 95°C for 2min; 30 cycles of denaturation at 95°C for 45 sec, annealing at 60°C for 1 min, and extension at 72°C for 2 min; and final extension at 72°C for 4 min.   After<br><br>
completion of the reaction, the solution was stored at 4°C. The amplified 4 types of DNAs were digested with Sad and Sacll, and the resultant 620 bp, 680 bp, 710 bp and 400 bp DNA fragments were purified separately by agarose gel electrophoresis to thereby obtain ADHJp, TDH3p, PGKlp and TEF2p, respectively.<br>
(6)	Preparation of 2^ DNA Replication Origin Site<br>
pYES2, which is a YEp vector, was digested whh Sspl and Nhel. The resultant 1.5 kbp fragment containing 2^ DNA replication origin (2)a ori) was purified by agarose gel electrophoresis and then blunt-ended.    This DNA fragment was designated 2|aOriSN.<br>
(7)	Preparation of YEp Type Expression Vectors<br>
2|iOriSN was inserted into the Nael site of pRS404Tcyc and pRS406Tcyc pretreated<br>
with BAP (bacterial alkaline phosphatase: Takara).    The resultant plasmids were transformed<br>
into E. coli SURE2, and then plasmid DNA was prepared.    The plasmid DNA was digested<br>
with Dralll; and EcdBl, Hpal or Pstl; and PvwII, followed by agarose gel electrophoresis to<br>
examine  the  insertion  and  the  direction  of 2µa  ori.     The  resultant pRS404Tcyc  and<br>
pRS406Tcyc into which 2µ a ori had been inserted in the same direction as in pYES2 were<br>
designated   pRS434Tcyc2µOri    and   pRS436Tcyc2µOri,    respectively.        The   resultant<br>
pRS404Tcyc and pRS406Tcyc into which 2µ. ori had been inserted in the opposite direction to<br>
that in pYES2 were designated pRS444Tcyc2µ0ri and pRS446Tcyc2µOri, respectively.<br>
A transcription promoter-containing fragment, i.e., ADHIp, TDH3p (GAPp), PGKlp or<br>
TEF2p, was inserted into the SachSacll site of the above-described four plasmids<br>
pRS434Tcyc2iiOri, pRS436Tcyc2µOri, pRS444Tcyc2µOri and pRS446Tcyc2|iOri to clone<br>
the DNA.    As a result, the following plasmids were obtained: (i) pRS434ADH, pRS434GAP,<br>
pRS434PGK and pRS434TEF from pRS434Tcyc2|aOri; (ii) pRS436ADH, pRS436GAP,<br>
pRS436PGK and pRS436TEF from pRS436Tcyc2|.iOri; (iii) pRS444ADH, pRS444GAP,<br>
pRS444PGK and pRS444TEF from pRS444Tcyc2µOri; (iv) pRS446ADH, pRS446GAP,<br>
pRS446PGK and pRS446TEF from pRS446Tcyc2µOri (Figs. 7A-7H).<br>
The expression vectors prepared in the present invention are summarized in Table 3<br>
below.<br><br><br>
(8)    Introduction of YEp Type Expression Vectors into Yeast<br>
In order to examine whether the DNA replication region of the prepared YEp type expression vectors functions or not, about 40 ng of each YEp type expression vector was introduced into YPH499 strain using Frozen-EZ Yeast Transformation II (Zymo Research, Orange, CA). (The procedures followed the protocol attached to the kit.) Then, colonies growing on SD-W (DOB+CMS (-Trp); BIOlOl, Vista, CA) agarplate at 30°C were examined. The results are shown in Table 4 below.<br><br><br>
The results shown in Table 4 revealed that each of the YEp type vectors prepared in the invention functions normally as a vector.<br>
[EXAMPLE 2]    Cloning of Genes<br>
(1)    Cloning of HMG-CoA Reductase Gene {HMGl' Gene) by PCR<br>
The cloning of 5. cerevisiae HMGl' gene was carried out as described below.<br>
Based on information on S. cerevisiae-dcrivQd HMGl gene (Accession No. M22002) (M.E. Basson, et al, Mol. Cell. Biol. 8, 3797-3808 (1988): SEQ ID NO: 1) registered in the GenBank, a pair of primers were designed which are specific to those nucleotide sequences corresponding to an N-terminal and a C-terminal region of the protein encoded by this gene. Using these primers and a yeast cDNA library (Clontech; No. CL7220-1 derived from S. cerevisiae DBY746) as a template, PCR was carried out.<br>
N-terminal primer (Primer 1): 5'-ATG CCG CCG CTA TTC AAG GGA CT-3*    (SEQ<br>
ID NO: 28)<br>
C-terminal primer (Primer 2): 5'-TTA GGA TTT AAT GCA GOT GAC GG-3*<br>
(SEQ ID NO; 29)<br>
The PCR was carried out in the reaction solution as described below under the<br>
following conditions: 30 cycles of denaturation at 94°C for 45 sec, annealing at 55°C<br>
for 1 min and extension at 72°C for 2 min.<br>
10 X ExTaq buffer (Takara)	5 jil<br>
2.5 mM dNTP mix	4 |al<br>
5 U/^1 ExTaq (Takara)	1 ^l<br>
10 pmol    Primer 1 10 pmol    Primer 2<br>
0.5 ng    cDNA	<br>
To give a 50 |il solution in total<br>
Agarose gel electrophoresis performed after the PCR confirmed a fragment at the expected location (3.2 kbp). This 3.2 kbp DNA fragment was cloned into pT7Blue T vector (Novagen, Madison,  WI) capable of TA cloning, to thereby obtain pT7HMGl.     The<br><br>
nucleotide sequence of the thus cloned HMG-CoA reductase gene was determined. As a result, the nucleotide sequence as shown in SEQ ID NO: 3 and the amino acid sequence as shown in SEQ ID NO: 4 were obtained. The thus determined nucleotide sequence was partially different from the corresponding nucleotide sequence registered in the GenBank (http://www.ncbi.nlm.nih.gov/Genbank/index.html) (Fig. 2A). This gene that comprises PCR errors and encodes the amino acid sequence of a mutant HMG-CoA reductase (SEQ ID NO: 4) is designated HMGl'.<br>
(2)    Correction of PCR Errors in HMGl'<br>
An HMOr fragment was subcloned from plasmid pT7HMGl comprising HMGl' encoding a mutant HMG-CoA reductase. Then, the amino acid substitutions resulted from the PCR errors occurred in the coding region of the wild-type HMG-CoA reductase gene were corrected by site-directed mutagenesis to thereby prepare pALHMG106. The details of this preparation are as described below.<br>
Plasmid pT7HMGl was used as cloned HMGl \ As a vector for introducing site-directed mutations, pALTER-1 (Promega) was used.<br>
Site-directed mutagenesis was carried out according to the procedures described in "Protocols and Application Guide, 3rd edition, 1996 Promega, ISBN 1-882274-57-1" published by Promega. As oligos for introducing mutations, the following three oligos were synthesized chemically.<br>
HMG1(190-216) 5*-CCAAATAAAGACTCCAACACTCTATTT-3' (SEQ ID NO: 30) HMGl (1807-1833) 5'-GAATTAGAAGCATTATTAAGTAGTGGA-3'  (SEQ ID NO: 31) HMGl (2713-2739) 5'-GGATTTAACGCACATGCAGCTAATTTA-3'  (SEQ ID NO: 32)<br>
First, pT7HMGl was digested with Smal, ApaU and 5a/I, and a 3.2 kbp HMGl' fragment was prepared by agarose gel electrophoresis. This fragment was inserted into the SmaVSall site of pALTER-1 to prepare pALHMGl. After denaturation of this plasmid with alkali, the above-described oligos for introducing mutations, Amp repair oligo (Promega) as repair oligos, and Tet knockout oligo (Promega) as knockout oligos were annealed thereto. The resultant plasmid was introduced into E. coli ESI301 (Promega).. Transformants that retained plasmids into which site-directed mutations had been introduced were selected and<br><br>
cultured with 125 µg/ml ampicillin to prepare plasmid DNA. The nucleotide sequence of the resultant plasmid DNA was examined with primers having the sequences as shown below. As a result, all the sequences corresponding to HMGl (190-216), HMGl (1807-1833) and HMGl (2713-2739) were corrected so that they had the sequences of these oligonucleotides (SEQ ID NO: 5). The amino acid sequence encoded by the corrected nucleotide sequence (SEQ ID NO: 6) was consistent with the amino acid sequence encoded by the wild-type HMGl (SEQ ID NO: 2); the corrected sequence retained only silent mutations. Since this PCR error-corrected HMGl encodes a polypeptide having the same amino acid sequence as that of the wild-type enzyme though it has a partially different nucleotide sequence, this gene is also designated HMGl and used herein without distinction between this and the wild-type gene HMGl.<br>
HMGl (558-532) S'-GTCTGCTTGGGTTACATTTTCTGAAAAO'  (SEQ ID NO: 33) HMGl (1573-1599) 5'-CATACCAGTTATACTGCAGACCAATTG-3'  (SEQ ID NO: 34) HMGl (2458-2484) 5'-GAATACTCATTAAAGCAAATGGTAGAA-3'  (SEQ ID NO: 35)<br>
The plasmid carrying the thus corrected HMGl sequence was designated pALHMG106 (Fig. 8).<br>
(3)    Cloning of Geranylgeranyl Diphosphate Synthase Gene BTSl<br>
S. cerevisiae BTSl gene (also called GGPP synthase gene) was cloned as described below.<br>
Based on information on S, cerevisiae-denwed GGPP synthase gene registered in the GenBank (Accession No. U31632) (Y. Jiang, et aL, J. Biol. Chem. 270 (37), 21793-21799 (1995)), a pair of primers described below matching an N-terminal and a C-terminal region of the enzyme were designed. Using these primers and a yeast cDNA library (CL7220-1) as a template, PCR was carried out.<br>
N-terminal primer: 5'-ATG GAG GCC AAG ATA GAT GAG CT-3' (SEQ ID NO: 36) C-terminal primer: 5'-TCA CAA TTC GGA TAA GTG GTC TA-3' (SEQ ID NO: 37)<br><br>
The PCR was performed in a reaction solution having a composition similar to that of the reaction solution described in (1) above under the following conditions: 30 cycles of denaturation at 94°C for 45 sec, annealing at 55°C for 1 min and extension at 72°C for 2 min. Agarose gel electrophoresis performed after the PCR confirmed a fragment having the proper mobility (corresponding to approx. 1.0 kbp). This BTSl gene was cloned into pT7Blue T vector capable of TA cloning, followed by sequencing of the entire region of this BTSl gene. The results revealed that the nucleotide sequence of this gene was completely identical with the nucleotide sequence registered in the GenBank. Thus, it was confirmed that this gene is the S. cerevisiae-d^nv^d GGPP synthase gene.<br>
The pT7Blue T vector was digested with 5(3/wHI and SQI\ to cut out the BTSl gene, which was then introduced into the BamUl-Xhol site of pYES2 (Invitrogen). The recombinant plasmid obtained was designated pYESGGPS.<br>
(4)    Cloning of Escherichia coli-derived FPP Synthase Gene ispA<br>
E. coli genomic DNA was prepared from E. coli JM109 (Takara) by the following procedures. JM109 cells were cultured in 1.5 ml of 2xYT medium and harvested by centrifugation. To these cells, 567 µl of TE (pH 8.0), 3 µ1 of 20 mg/ml proteinase K (Boehringer Mannheim, Mannheim, Germany) and 30 µl of 10% SDS were added. The resultant mixture was left at 37°C for 1 hr, and then 100 µl of 5M NaCl was added thereto and mixed. Eightyµl of CTAB/NaCl solution (10% CTAB, 0.7 M NaCl) was added thereto, and the resultant mixture was heated at 65°C for 10 min. This mixture was then treated with 700 \x\ of chloroform/isoamyl alcohol (24:1) extraction, and a fiirther extraction was carried out with 600 |il of phenol/chloroform/isoamyl alcohol (25:24:1) to the obtained aqueous layer,, which was then centriftjged. The precipitate was washed with 70% ethanol, dried, and then dissolved in 100 \i\ of TE (pH 8.0) to thereby obtain an E. coli genomic DNA solution. The DNA was measured and quantitatively determined at OD260- Then, TE was added to the solution to give a DNA concentration of 1µg/)µl.<br>
Using the thus obtained E. coli genomic DNA as a template and the following synthetic oligo-DNA primers, E. coli-derived FPP synthase gene ispA was cloned by PCR.<br>
ISPAl: 5'-TGA GGC ATG CAA TTT CCG CAG CAA CTC G-3' (SEQ ID NO: 68) ISPA2: 5'-TC AGA ATT CAT CAG GGG CCT ATT AAT AC-3' (SEQ ID NO: 69)<br><br>
PCR was carried out in a 100 µl reaction solution containing Ix ExTaq buffer, 0.5 mM dNTP, 100 pmol of ISPAl, 100 pmol of ISPA2, 0.2 fig of E coli genomic DNA and 5 units of ExTaq under the following conditions: 30 cycles of denaturation at 94°Cfor 1 min, annealing at 55°C for 1 min and extension at 72°C for 1.5 min. The PCR product was digested with ^caRI and Sphl. Then, the resultant 1.0 kbp fragment was purified by agarose gel electrophoresis and inserted into the EcoK[-Sph\ site of pALTER-Ex2 (Promega), which was then introduced into E, coli JM109 for the cloning of the gene. Restriction enzyme mapping using EcoKl, Sphl, Ndel, Smal, and Bamm revealed that ispA gene (SEQ ID NO: 77) had been introduced correctly into earned tliree plasmids, i.e., pALispA4, pALispA16 and pALispAlS.<br>
(5)    Preparation of Mutant FPP Synthase Genes<br>
Using plasmid pALispA16, the codon encoding the amino acid residue Tyr at position 79 of the polypeptide encoded by E. coli ispA was modified by substitution according to the procedures described in "Protocols and Applications Guide, the 3rd edition, 1996, Promega, ISBN 1-882274-57-1" published by Promega. The following oligos for introducing mutations (also called "mutant oligos") were prepared by chemical synthesis.<br>
ISPA-D: 5'-ATC ATG AAT TAA TGA GTC AGC GTG GAT GCA TTC AAC GGC GGCAGC-3'  (SEQ ID NO: 70)<br>
ISPA-E: 5'-ATC ATG AAT TAA TGA TTC AGC GTG GAT GCA TTC AAC GGC GGCAGC-3*  (SEQ ID NO: 71)<br>
ISPA-M: 5'-ATC ATG AAT TAA TGA CAT AGC GTG GAT GCA TTC AAC GGC GGCAGC-3'  (SEQ ID NO: 72)<br>
The above-described mutant oligo ISPA-M was designed so that the nucleotides from position 16 to position 18 (the three nucleotides underlined) encode Met, which nucleotides correspond to the codon for the 79th amino acid residue Tyr in the wild-type gene. Similarly, mutant oligos ISPA-D and ISPA-E were designed so that the corresponding codons encode Asp and Glu, respectively. In these mutant oligos, the nucleotides from position 26 to position 31 (the six nucleotides underlined) were designed so that EcoT22l (Nsil) site is<br><br>
newly formed by the substitution mutation. Thus, it is so an-anged that these mutant genes can be easily distinguished from the wild-type gene by restriction enzyme mapping. The mutant oligos were treated with T4 polynucleotide kinase (Promega) in advance to phosphorylate their 5' end and purified by gel filtration with Nick Column (Pharmacia Biotech, Uppsala, Sweden) before use. For the introduction of mutations, Cm repair oligo (Promega) as the repair oligo, and Tet knockout oligo (Promega) as the knockout oligo were also used. Cm repair oligo, Tet knockout oligo and the mutant oligos were annealed to alkali-denatured pALispA16, which was then transformed into E. coli ESI301 mutS (Promega). Plasmid DNA was prepared from E. coli colonies growing in the presence of 20 |j.g/ml chloramphenicol (Cm), and transformed into E. coli JM109. Plasmid DNA was prepared from E, coli colonies growing on agar plates containing 20 |ig/ml Cm. Plasmids containing substitution-mutated ispA genes (designated ispAm genes) that were prepared using pALispA4 as a template and ISPA-D, ISPA-E and ISPA-M as mutant oligos were designated p4D, p4E and p4M, respectively. Those plasmids prepared similarly using pALispA16 as a template were designated pl6D, pl6E and pl6M, respectively. Those plasmids prepared similarly using pALispAlS as a template were designated pl8D, pl8E and pl8M, respectively.<br>
(6)	Cloning of IPPA-Isomerase Gene idi<br>
E. coli IPPA-isomerase gene was formerly called as ORF182 (according to NCBI BLAST search; GenBank Accession No. AE000372), but Hahn et al ((1999) J. Bacteriol., 181: 4499-4504) designated this gene idi. As plasmids in which idi (SEQ ID NO: 85; encoding the amino acid sequence as shown in SEQ ID NO: 86) is cloned, p3-47-ll and p3"47-13 described in Hemmi et al, (1998) J. Biochem., 123: 1088-1096 were used in the invention.<br>
(7)	Cloning of Bacillus stearothermophilus FPP Synthase Gene<br>
Plasmid pFE15 described in Japanese Unexamined Patent Publication No. 5-219961 was digested with Notl and Smal, The resultant 2.9 kbp Bacillus stearothermophilus FPP synthase gene (hereinafter, referred to as "fts") (SEQ ID NO: 75; encoding the amino acid sequence as shown in SEQ ID NO: 76) fragment containing a transcription unit was purified and   inserted   into   the  Seal   site   of pACYC177   (Nippon   Gene)   to   obtain  plasmid<br><br>
pFE15NS2.9-l.<br>
[EXAMPLE 3]    Insertion of Genes into Expression Vectors<br>
(1)	Subcloning into pRS Expression Vectors<br>
HMGl gene was introduced into individual pRS vectors (Figs. 6 and 7) prepared in the present invention which are E. coli-S. cerevisiae YEp shuttle vectors containing a constitutive transcription promoter.<br>
pALHMG106 (Fig. 8) containing the PCR eiTor-corrected HMG-CoA reductase gene was digested with Smal and Sail. The resultant 3.2 kbp HMGl fragment was purified by agarose gel electrophoresis and inserted into the Smal-Sall site of pRS434GAP, pRS444GAP, pRS434TEF, pRS444TEF, pRS434PGK and pRS444PGK. Those plasmids into which the gene had been subcloned were examined for their physical maps by restriction enzyme mapping with ATzoI, Spel, Nael and Sphl, and by confirmation of the nucleotide sequences of the border regions of the inserted 3.2 kbp HMGl fragment. Then, those plasmids created exactly as planned were selected and designated pRS434GAP-HMGl, pRS444GAP-HMGl, pRS434TEF-HMGl, pRS444TEF-HMGl, pRS434PGK-HMGl and pRS444PGK-HMGl.<br>
(2)	Preparation of pRS414PTadh-HMGl and pRS414TPadh-HMGl<br>
Vectors pRS414PTadh and pRS414TPadh (Fig. 6) containing a constitutive transcription promoter ADHlp were digested with Smal and Sail, followed by the same operations as described in (1) above. As a result, plasmids pRS414PTadh-HMGl and pRS414TPadh-HMGl each containing HMGl gene inserted thereinto were created.<br>
(3)	Preparation of HMGl' Expression Plasmid pYES-HMGl<br>
pTVHMGl prepared in (1) in Example 2 was digested with BamHl, Sail and Seal to cut out the HMGl' gene encoding the mutant HMG-CoA reductase resulted from PCR errors. Then, this gene was^inserted into the BamHl-Xhol site of pYES2 (Invitrogen, Carlsbad, CA). The resultant recombinant vector was designated pYES-HMG 1. As a result of determination of the nucleotide sequence within this vector, it was confirmed that the sequence is identical with the nucleotide sequence as shown in SEQ ID NO: 3. The above plasmid pYES2 is a shuttle vector for expression in yeast that has yeast 2|im DNA ori as a replication origin and GALl transcription promoter inducible by galactose (Fig. 4).<br><br>
(4)    Preparation of Deletion Mutant HMGl' Expression Plasmid pYHMGxxy<br>
In order to prepare vectors for expressing deletion mutants of HMG-CoA reductase gene having deletion of a nucleotide sequence encoding a region upstream of a domain that is believed to be the catalytic domain of HMG-CoA reductase, a fragment lacking a part of the HMGl' coding region together with the vector moiety was prepared by PCR using pYES-HMGl created in (3) above as a template. The resultant fragment was blunt-ended with Klenow enzyme and then circularized again by self-ligation, followed by transformation into E. coli JM109. Then, plasmid DNA was prepared from the transformant. The sequences of the synthetic DNAs used as primers and their combinations are shovwi in Table 1.<br>
For each of the plasmid DNA obtained, it was confirmed with 373A DNA sequencer (Perkin Elmer, Foster City, CA) that there was no shift in the reading frame of amino acids upstream and downstream oiHMGl, and that there was no amino acid substitution resulting from PCR errors around the junction site. As a result, the following plasmids were obtained which have no amino acid substitution resulting from PCR errors around the junction site and in which a deletion could be made successively without any shift in the reading frame. Deletion mutants of HMGl gene are expressed as, e.g., "A02y" according to the deletion pattern (where y represents a working number that may be any figure), and pYES2 vectors comprising A02y are expressed as, e.g., pYHMG026. (This is applicable to other deletion mutants.)<br><br>
HMGl A02y: SEQ ID NO HMGlA04y: SEQ ID NO HMGlA05y: SEQ ID NO HMGl A06y: SEQ ID NO HMGlA07y: SEQ ID NO HMGlA08%y: SEQ ID NO HMGlM10y: SEQ ID NO HMGlA11y: SEQ ID NO HMGlA12y: SEQ ID NO HMGlA13y: SEQ ID NO<br><br>
7<br>
8<br>
9<br>
10<br>
11<br>
12<br>
13<br>
14<br>
15<br>
16<br><br>
Vprtnrs;- YHMG026. nYHMG027. nYHMG044. tDYHMG045. DYHMG062. DYHMG063,<br><br>
PYHMG065, pYHMG076, pYHMG081, pYHMG083, pYHMG085, pYHMG094, pYHMGlOO, pYHMG106, pYHMGlO, pYHMGlOS, pYHMG109, pYHMG112, pYHMGI22, pYHMG123, pYHMG125 and pYHMG133<br>
[EXAMPLE 4]    Preparation of AURGGlOl<br>
A 1.9 kbp Sail fragment having a primary structure of GAL J transcription promoter-^r^y-Cyc; transcription terminator (GAUp-BTSJ-CYCJX) was prepared by PCR using pYESGGPS described in (3) in Example 2 as a template and the following primers PYES2 (1-27) and PYES2 (861-835).<br>
PYES2 (1-27): 5'-GGC CGC AAA TTA AAG CCT TCG AGC GTC-3' (SEQ ID NO: 73)<br>
PYES2 (861-835): 5'-ACG GAT TAG AAG CCG CCG AGC GGG TGA-3' (SEQ ID NO: 74)<br>
This fragment was inserted into the Sail site of pAURlOl (Takara) to obtain pAURGGllS. It was confirmed by DNA sequencing that the BTSl gene in pAURGG115 had no PCR error.<br>
pAURGG115 was linearized with Eco065I and introduced into A451 strain by the lithium acetate method. Then, colonies growing on YPD agar plates (1% yeast extract, 2% peptone, 2% dextrose, 2% agar) containing lµg/ml aureobasidin at 30°C were selected as transformants. The resultant transformants were cultured again on aureobasidin selection plates to select a single colony.<br>
As a result, two clones AURGGlOl and AURGG102 were obtained as recombinants from A451 strain. In the present invention, AURGGlOl was used as one of A451-derived host clones.<br>
As revealed by Southern blot hybridization (Fig. 9) and PCR mapping (Fig. 10), BTSl is integrated in the genome in AURGG102 but not integrated therein in AURGGlOl.    In AURGGlOl, it was found that AURl has been merely replaced withAURI-C?7-C (a marker gene). Since AURl .is not directly involved in the synthesis of prenyl alcohol or prenyl diphosphate, it is possible to use AURGGlOl as one example of A451-derived host clones.<br>
Details of the Southern blot hybridization, Northern blot hybridization and PCR<br><br>
mapping are provided in Example 6 described later.<br>
[EXAMPLE 5]    Preparation of EUG Strains<br>
A gene map around squalene synthase gene ERG9 was obtained from a yeast genome database. Based on this map, PCR primer DNAs for amplifying DNA fragments for replacing ERG9 transcription promoter {ERG9p) were designed (Fig. 2). On the other hand, a 1.8 kbp DNA fragment comprising a transformant selection marker gene URA3 and a transcription promoter GALlp was prepared by PCR amplification using, as a template, pYES2A obtained by digesting pYES2 with Nael and Nhel, blunt-ending with Klenow enzyme and deleting 2|a oh by self-ligation.<br>
The primers used in the PCR are as follows. E-MCSf: 5'- GCC GTT GAC AGA GGG TCC GAG CTC GGT ACC AAG-3'  (SEQ<br>
ID NO: 49)<br>
E"URA3r: 5'- CAT ACT GAC CCA TTG TCA ATG GGT AAT AAC TGA T-3' (SEQ<br>
ID NO: 50)<br>
In the above primers, an Eaw 11051 recognition site (the underlined portion) is added so that T/A ligation can be conducted by using (i) a 0.7 kbp DNA fragment comprising a downstream portion of the open reading frame YHR189W in the genome of 5. cerevisiae and (ii) a 0.9 kbp DNA fragment comprising an upstream portion of ERG9. The YHR189W fragment was prepared by PCR using the following primers YHRl 89Wf and YHRl 89Wr, and YPH499 genomic DNA as a template. The ERG9 fragment was prepared by PCR using the following primers ERG9f and ERG9r, and YPH499 genomic DNA as a template. YPH499 genomic DNA was prepared with Dr. GenTLE   .<br>
YHR189Wf: 5'-TGT CCG GTA AAT GGA GAC-3' (SEQ ID NO: 51) YHR189Wr: 5'-TGT TCT CGC TGC TCG TTT-3' (SEQ ID NO: 52) ERG9f: 5'-ATG GGA AAG CTA TTA CAA T-3' (SEQ ID NO: 53) ERG9r: 5'-CAA GGT TGC AAT GGC CAT-3' (SEQ ID NO: 54)<br>
The 1.8 kbp DNA fragment was digested with Eam11051 and then ligated to the 0.7 kbp DNA fragment. With the resultant fragment as a template, 2nd PCR was carried out using the above-described primers YHR189Wf and E-MCSf. The amplified 2.5 kbp DNA fragment was digested with Eam ll05I and then ligated to the 0.9 kbp fragment. With the<br><br>
resultant fragment as a template, 3rd PCR was carried out using the following primers YHR189W-3f and ERG9-2r. As a result, a 3.4 kbp DNA fragment was amplified. This was used as a DNA fragment for transformation.<br>
YHR189W-3f: 5'-CAA TGT AGO GCT ATA TAT G-3' (SEQ ID NO: 55) ERG9-2r: 5'-AAC TTG GGG AAT GGC ACA-3' (SEQ ID NO: 56)<br>
A vector was introduced into yeast strains using Frozen EZ Yeast Transformation II kit purchased from Zymo Research (Orange, CA). The resultant recombinants were cultured on an agar medium called SGR-U medium that had been obtained by adding CSM (-URA) (purchased from BIO 101, Vista, CA) and adenine sulfate (final concentration 40 mg/L) to SGR medium (a variation of SD medium in which the glucose component is replaced with galactose and raffinose), at 30°C. Colonies grown on the medium were spread on the same medium again, cultured and then subjected to single colony isolation.<br>
The resultant recombinants were designated EUG {ERG9p::URA3-GAL]p) strain. Of these, clones derived from A451 strain were designated EUGl through EUGIO; clones derived from YPH499 strain were designated EUGll through EUG20; and clones derived from YPH500 strain were designated EUG21through EUG30.<br>
They were cultured on SD medium to select those clones that exhibit growth exhibition as a result of the inhibition of ERG9 expression due to glucose repression. As a result, EUG8 from A451, EUG12 from YPH499 and EUG27 from YPH500 were obtained.<br>
Genomic DNA was prepared from EUG8, EUG 12 and EUG27, separately, using Dr. GenTLE"^. The results of PCR using the genomic DNA as a template confirmed that the 1.8 kbp PCR fragment containing URA3 and GALJp is integrated into the genome of each strain upstream of the ERG9 coding region.<br>
[EXAMPLE 6]    Analysis of Genes and Enzyme Activity<br>
In this Example, the expression of genes in various recombinant yeasts prepared in the invention (for the preparation thereof, see Examples 7 and 8 describing the production of prenyl alcohols) was analyzed by determining the enzyme activity of prenyl-diphosphate synthase and by various techniques including Northern blot hybridization, Southern blot hybridization and PCR mapping.    Of the prepared recombinants, the host strain and the<br><br>
recombinants listed below were used in this Example. The introduction of individual vectors into the host was carried out according to the lithium acetate method described in Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc., pp. 13.7.1-13.7.2 or by a method using Frozen EZ Yeast Transformation II kit (Zymo Research, Orange, CA) according to the protocol attached to the kit. Clone 1-2 was obtained by introducing pYES-HMGl into A451; clone 3-2 was obtained by introducing pYHMG044 A451; clone 13-2 was obtained by introducing pYES-HMGl into AURGGlOl; and clone 15-2 was obtained by introducing pYHMG044 into AURGGlOl.<br>
No.l host strain: A451<br>
No.2 GALlp-BTSl (YIp): AURGGlOl (A451, aurl\:AURl-C)<br>
No.3 GAUp'BTS] (YIp): AURGG102 (A451, aurlwBTSl-AURl-C)<br>
No.4 GAUp'HMGl (YEp): 1-2 (pYES-HMGl/A451)<br>
No.5 GALlp'HMGlA (YEp): 3-2 (pYHMG044/A451)<br>
No.6 GALIp-HMGl (YEp): 13-2 (pYES-HMGl/AURGGlOl)<br>
No.7 GALlp'HMGlA (YEp): 15-2 (pYHMG044/AURGG101)<br>
Clones No. 1 to No. 7 were precultured separately at 26°C. One milliliter of the preculture was washed with physiological saline, added to 100 ml of a culture broth and cuhured in a 300 ml Erlenmeyer flask at 26°C with reciprocal shaking at 120 times/min. SD medium or SG medium (in which the glucose component of SD medium is replaced with galactose) was used for the cultivation. Recombinants retaining URA3 marker were cultured in SD-U [CSM (-URA)-added SD medium] or SG-U [GSM (-URA)-added SG medium]. AURGG clones were cultured in the presence of aureobasidin at 1 |ig/L.<br>
Cell growth was determined at OD^oo- CuUivation was stopped when the value at OD600 reached about 3-4 (23-52 hours). The culture was cooled in ice and then subjected to the preparation of DNA, RNA and crude enzyme solution, as described below.<br>
(1)   Southern Blotting<br>
Yeast DNA was prepared using the yeast DNA preparation kit Dr.  GenTLE^'^<br><br>
according to the protocol attached to the kit.<br>
The DNA thus prepared from yeast was digested with Ndel and Stul, followed by 0.8% agarose gel electrophoresis (3 |ag/lane). As molecular weight markers, 0.5 fig each of 1 kb ladder and XIHindIII (both from Promega, Madison, WI) were used. After the electrophoresis, the DNA was denatured with alkali, neutralized and then transferred onto Hybond N nylon membrane (Amersham, Buckinghamshire, England) by capillary blotting with 20 X SSC according to conventional methods. The resultant membrane was subjected to UV irradiation with a UV cross-linker (Stratagene) under conditions of optimal cross-linking, to thereby fix the DNA on the membrane.<br>
(2)	Northern Blotting<br>
RNA was prepared according to the method described in Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc., pp. 13.12.2-13.12.3 with partial modification. The modification was that once prepared RNA samples were fiirther treated with DNase I.<br>
After separation of RNA by formaldehyde-denatured agarose gel electrophoresis, the RNA was transferred onto Hybond N nylon membrane by capillary blotting with 20 x SSC according to conventional methods. Five micrograms of total RNA was electrophoresed per lane. As a molecular marker, 20 ng of DIG-RNA Marker I was used. The resultant membrane was subjected to UV irradiation with a UV cross-linker (Stratagene) under conditions of optimal cross-linking, to thereby fix the RNA on the membrane.<br>
(3)	PCR Mapping<br>
In order to examine how a fragment from pAURGGllS (a Yip vector prepared in Example 4) is integrated into the genome, PCR was carried out using 0.3-0.6 |ig of the yeast DNA prepared above as a template and a combination of synthetic oligonucleotide primers AUR-FWc and AUR-RVc, or AUR-SALl and AUR-SAL2. PCR conditions were as follows: 30 cycles of denaturation at 94°C for 30 sec, annealing at 55°C for 1 min and extension at 72°C for 3 min.<br>
AUR-FWc: 5'-TCT CGA AAA AGG GTT TGC CAT-3' (SEQ ID NO: 57)<br><br>
AUR-RVc: 5'-TCA CTA GGT GTA AAG AGG GCT-3' (SEQ ID NO: 58) AUR-SALl: 5'-TGT TGA AGG TTG CAT GCC TGC-3' (SEQ ID NO: 59) AUR-SAL2: 5'-TTG TAA AAC GAG GGC GAG TGA-3' (SEQ ID NO: 60)<br>
(4)    Preparation of DIG-Labeled Probe DNAs<br>
As hybridization probes, Probes I, II, III and V were prepared (Table 5).<br>
Table 5.  Hybridization Probes<br><br>
Probe No.<br>
I<br>
II III V<br><br>
Gene<br>
ERG20<br>
BTS1<br>
HMG1<br>
AUR1<br><br>
Template<br>
pT7ERG20<br>
pYES2-GGPS6<br>
pYHMGI<br>
PAUR123<br><br>
Prinner 1<br>
SCFPS1<br>
BTS1 (1-21)<br>
HMG1 (1267-1293)<br>
AUR-RV<br><br>
Primer 2<br>
SGFPS2<br>
BTS1 (1008-982)<br>
HMG1 (2766-2740)<br>
AUR-FW<br><br>
Probe I:<br>
Using the following synthetic oligonucleotides SCEPSl and SCEPS2 as primers, a PCR<br>
fragment was obtained from an S. cerevisiae cDNA library (Clontech, Palo Alto, CA) and<br>
cloned into pT7blue T vector.   Thus, pT7ERG20 was prepared.<br>
SCFPSl: 5'-ATG GCT TCA GAA AAA GAA ATT AG-3' (SEQ ID NO: 61) SCFPS2: 5'-CTA TTT GCT TCT CTT GTA AAC TT-3' (SEQ ID NO: 62)<br>
Using pT7ERG20 as a template and SCEPSl and SCEPS2 as primers, a DIG (digoxigenin)-labeld probe DNA was synthesized with PCR DIG Probe Synthesis Kit (Roche Diagnostics, Marmheim Germany). Experimental conditions were in accordance with the manufacturer's protocol attached to the kit.<br>
PCR conditions were as follows: 30 cycles of denaturation at 94°C for 30 sec, annealing at 58°C for 1 min and extension at 72°C for 3 min. The resultant DIG-Iabeled probe DNA was subjected to agarose gel electrophoresis to examine the state of synthesis.<br><br>
Probe II:<br>
A DIG-labeled probe DNA was synthesized in the same manner as used for Probe I,<br>
using  the following synthetic oligonucleotides as primers and pYESGGPS  (see (3)  in<br>
Example 2) as a template.<br>
BTSl (1-21): 5'-ATG GAG GCC AAG ATA GAT GAG-3' (SEQ ID NO: 63)<br>
BTSl (1008-988): 5'-TCA CAA TTC GGA TAA GTG GTC-3' (SEQ ID NO: 64)<br>
Probe III:<br>
A DIG-labeled probe DNA was synthesized in the same manner as used for Probe I,<br>
using the following synthetic oligonucleotides as primers and pYES-HMGl (see (3) in<br>
Example 3) as a template.<br>
HMGl (1267-1293): 5*-AAC TTT GGT GGA AAT TGG GTC AAT GAT-3' (SEQ ID<br>
NO: 42)<br>
HMGl  (2766-2740): 5'-TCC TAA TGC CAA GAA AAC AGO TGT CAC-3'<br>
(SEQ ID NO: 65)<br>
Probe V:<br>
A DIG-labeled probe DNA was synthesized in the same manner as used for Probe I,<br>
using the following synthetic oligonucleotides as primers and pAUR123 (Takara) as a<br>
template.<br>
AUR-FW: 5'-ATG GCA AAC CCT TTT TCG AGA-3' (SEQ ID NO: 66)<br>
AUR-RV: 5'-AGC CCT CTT TAG ACC TAG TGA-3' (SEQ ID NO: 67)<br>
(5)    Hybridization and Detection of Probes<br>
Southern blot hybridization was carried out at a probe concentration of 20 ng/ml at 42°C for 24 hr using DIG Easy Hyb (Roche). Northern blot hybridization was carried out at a probe concentration of 100 ng/ml at 50°C for 24 hr using DIG Easy Hyb. Prior to each hybridization, prehybridization was carried out for 24 hr in DIG Easy Hyb solution at the same temperature used for the hybridization. After the hybridization, the membrane was washed 3 times with 2x SSC, 0.1% SDS at 65°C for 10 min each, and then 2 times with 0.2x<br><br>
SSC, 0.1% SDS at 65X for 15-20 min each. Thereafter, the DIG-labeled probe in the membrane was allowed to generate chemiluminescence by using DIG Luminescent Detection Kit (Roche), followed by exposure of the blot to X-ray film for visualization.<br>
(6)    Determination of Enzyme Activity<br>
Cells were harvested from each culture broth by centrifugation and disrupted at 4°C with glass beads in the same manner as in the preparation of RNA, Then, cells were suspended in sterilized water. The suspension was centrifuged at 12,000 r.p.m. for 10 min with a refrigerated microcentrifuge, and the resultant supernatant was recovered as a crude enzyme fraction. The protein concentration in the crude enzyme fraction was determined by Bio-Rad Protein Assay (Bio-Rad, Hercules, CA) using BSA as a standard protein. Ten |ig of the crude enzyme fraction was reacted in 200 |al of the following reaction cocktail at 37°C for 40 min.<br>
0.125 mM [*^C] IPP (185 GBq/mol)<br>
0.125 mM geranyl diphosphate (Sigma Chemical, St. Louis, MO)<br>
100mMTrisHCl(pH7.0)<br>
10mMNaF,5 mMMgCl2<br>
5 mM 2-mercaptoethanol<br>
0.05% Triton X-100<br>
0.005% BSA<br>
After the reaction, extended prenyl diphosphate was extracted with water-saturated butanol. An aliquot of the prenyl diphosphate was subjected to determination of radioactivity with a liquid scintillation counter. The remaining sample was dephosphorylated with potato acid phosphatase, spotted onto thin layer chromatography plate [plate: LKC18 (Whatman, Clifton, NJ], and then the plate was developed [developer solvent: H20/acetone = 1:19]. The autoradiogram was visualized with Bio Image Analyzer BAS2000 (Fuji Film) and the relative radioactivities were determined, according to the method of Koyama et al (Koyama T, Fujii, H. and Ogura, K., 1985, Meth. Enzymol. 110:153-155).<br><br>
(7)    Results and Observations<br>
(7-1)    Southern Blot Hybridization and PCR Mapping<br>
The results of southern blot hybridization are shown in Fig. 9. The results of PCR mapping in the vicinity of AURl are shown in Fig. 10. In Figs. 9 and 10, lanes 1 to 7 correspond to the numbers of clones (No. 1 to No. 7) used in (6). "N" represents DNA digested with Nde\\ and "S" represents DNA digested with StuL DNAs used in individual lanes were prepared from the following strains.<br>
Lane 1: A451; Lane 2: AURGGl 01; Lane 3: AURGGl 02; Lane 4: pYES-HMGl/A451; Lane 5: pYHMG044/A451; Lane 6: pYES-HMGl/AURGGlOl; Lane 7: pYHMG044/AURGG101<br>
It was found that ERG20 (FPP synthase gene) is contained in the same manner in all of the clones tested and that there is no change in the vicinity of ERG20 in the genome of each clone (Fig. 9).<br>
When BTSi (GGPP synthase gene) and AURl were used as probes, it was found that BTSl is integrated into the region of AURl in AURGG102, but the bands appearing in AURGGlOl are the same as those appearing in the host strain A45L In AuRGGlOl, only AURl gene is replaced with pAURl01-derived AURl-C gene; it was found that the GALl-BTSl fragment is not integrated into the genome of this clone. Duplication of AURl locus resulting from genomic integration was detected by PCR. As expected, a band was not detected in AURGGlOl but detected only in AURGG102 (Fig. 10).<br>
In Fig. 9, when HMGl was used as a probe, a plasmid-derived band appeared in Ndel-digtsted DNAs (lanes 4-7). In iStul-digested DNAs, it is expected that a 8.2 kbp band derived from the plasmid (overlapping a 8.3 kbp band derived from the genome) should appear as in clone 1-2 (No. 4). However, a band shift was observed in clone 13-2 (No. 6) and clone 15-2 (No. 7) as a result of recombination between the vicinity of HMGl in the genome and the plasmid introduced.<br>
From the results of Southern blot hybridization and PCR mapping, the genotypes of the clones used this time can be summarized as shown in Table 6 below. In this Table, "AUR" means a medium to which aureobasidin has been added. "Medium 1" means a medium for preculture, and "Medium 2" means a medium for subsequent culture.<br><br><br>
(7-2)   Northern Blot Hybridization<br>
The resuhs of Northern blot hybridization are shown in Fig. 11. Probes I, II and III as shown in Table 5 were used as probe.<br>
In Fig. 11, the clones used in lanes 1 to 7 are the same as used in Fig. 9. Mark "-" indicates transcripts in SD medium, and mark "+" indicates transcripts in SG medium.<br>
ERG20 transcript showed a tendency to decrease in clone 13-2 (No. 6) and clone 15-2 (No, 7) when GALlp transcriptional induction was applied by SG medium.<br>
When the transcription of genes under the control of GALI transcription promoter was induced by SG medium, the induction of BTS] transcript increased only in a clone in which GALJp-BTSl fragment has been integrated into the genome, i.e., AURGG102 (No. 3).<br>
However, when compared with HMGl transcript, it is seen that the degree of transcription induction of BTS 1 is lower. When transcription was induced by SG medium, HMGl transcript increased remarkably in clones No.4 to No. 7 in which GAL]p-HMGI fragment has been transferred by a plasmid.<br>
(7-3)    Prenyl-diphosphate synthase Activity<br>
The activity of prenyl-diphosphate synthase in the crude enzyme fraction was determined using geranyl diphosphate (GPP) and [14CJ-labeled IPP as allylic diphosphate substrates.<br>
The prenyl diphosphates synthesized with GPP and [14C] IPP as substrates were dephosphorylated and analized by TLC.    Then, the ratioactivity of each spot on the plate was<br><br>
examined. As a result, FPP synthase activity was high, and next to that, HexPP (hexaprenyl diphosphate) synthase activity was detected that was by far higher than GGPP synthase activity. Then, relative amounts of reaction products were calculated from autoradiogram, followed by calculation of specific activity per gross protein. The results are shown in Fig. 12. In Fig. 12A, the upper panel shows FPP synthase (FPS) activity, and the lower panel shows GGPP synthase (GGPS) activity. In Fig. 12B, the upper panel shows HexPP synthase (HexPS) activity, and the lower panel shows PTase (total prenyl-diphosphate synthase) activity. Gray columns show the results in SD medium, and white columns show the results in SG medium. A large part of the total prenyl-diphosphate synthase activity is FPP synthase activity. An increase in this activity caused by SG medium was observed. In particular, total prenyl-diphosphate synthase activity remarkably increased in clone 13-2 (No. 6) and clone 15-2 (No. 7) that produce FOH in a large quantity (see Example 9). As a whole, when GPP is used as an allylic substrate, GGPP synthase activity is about 1/20000 of FPP synthase activity and about 1/300 of HexPP synthase activity. HexPP synthase activity decreased in SG medium.<br>
[EXAMPLE 7]    Cultivation of Recombinants and Production of Prenyl Alcohols<br>
(1) Production of Prenyl Alcohols When HMGl Gene with a Constitutive Promoter Was Introduced into A451 (such a recombinant is expressed as "Constitutive Promoter; HMG1\ A451"; this way of expression applies to the remaining part of the specification).<br>
For industrial application of FOH high yielding recombinants, the use of a constitutive promoter is advantageous since it allows the use of cheap, conventional media. Then, HMGl gene was expressed under the control of a constitutive promoter using as a host S. Cerevisiae A451 (ATCC200589) that was recognized in preliminary experiments to have potentiality to produce FOH.<br>
HMGl gene (PCR error-corrected gene) was introduced into vector pRS434GAP or pRS444GAP each containing a constitutive promoter GAPp {=TDH3p) to thereby prepare pRS434GAP-HMGl and pRS444GAP-HMGl, respectively. These plasmids were introduced into A451 to obtain recombinants, which were designated pRS434GAP-HMGl/A451 and pRS444GAP-HMGl/A451.<br>
Ten colonies were selected randomly from each of the yeast recombinants into which<br><br>
HMG-CoA reductase gene had been introduced. These colonies were inoculated into SD-W medium [obtained by adding CSM (-TRP) to SD] that is an SD selection medium for a marker gene TRPl, and precultured therein. Then, 250 \x\ of the preculture (when a yeast recombinant with a constitutive promoter was precultured, this amount was added not only in this experiment but in other experiments described later) was added to 2.5 ml of YM medium and cultured at 26°C for 4 days with rotary shaking at 130 r.p.m.<br>
After completion of the cultivation, 2.5 ml of methanol was added to the culture broth and mixed. Then, about 5 ml of pentane was added thereto and agitated vigorously. The resultant mixture was left stationary. The pentane layer was transferred into a new glass tube, followed by evaporating the pentane in a draft to thereby concentrate solute components. Then, the resultant solution was subjected to gas chromatography/mass spectrography (GC/MS) to identify prenyl alcohols and quantitatively determine them with undecanol as an internal standard. At that time, in order to examine the degree of cell growth, 50 |il of the culture broth was diluted 30-fold with water, followed by determination of absorbance at 600 nm.<br>
GC/MS was carried out with HP6890/5973 GC/MS system (Hewlett-Packard, Wilmington, DE). The column used was HP-5MS (0.25 mm x 30 m; film thickness 0.25 p.m). Analytical conditions were as described below. The same conditions were used for all the GC/MS analyses in this specification.<br>
Inlet temperature: 250°C Detector temperature: 260°C [MS zone temperatures] MSQuad:150°C<br>
MS Source: 230°C Mass scan range: 35-200 [Injection parameters]<br>
Automated injection mode<br>
Sample volume: 2µ1<br>
Methanol washing: 3 times; hexane washing: twice<br>
Split ratio: 1/20<br>
Carrier gas: helium 1.0 ml/min<br><br>
Solvent retardation: 2 min [Oven heating conditions]<br>
115°Cfor90sec<br>
Heating up to 250°C at 70°C /min and retaining for 2 min<br>
Heating up to 300°C at 70°C /min and retaining for 7 min After Time 0<br>
Internal standard:    0.01 |il of 1-undecanol in ethanol Reliable standards:    (E)-Nerolidol (Eisai)<br>
(all-E)'-Farnsol (Sigma)<br>
(all-F)-Geranylgeraniol (Eisai)<br>
Squalene (Tokyo Kasei Kogyo)<br>
The results of determination of prenyl alcohol yields are shown in Figs. 13-15. Fig. 14 shows a result selecting 10 colonies from clone No. 3 of pRS434 shown in Fig. 13. Fig. 15 shows a summary of data shown in Fig. 13. An FOH yield of 4.9 mg/L was recognized in colony No. 10 (pRS434) in Fig. 14. In Fig. 15, "434" and "444" represent the results when pRS434GAP and pRS444GAP vectors were used, respectively. The column at the utmost right represents the results when the host (A451) before gene transfer was cultured.<br>
These results revealed that, when A451 was used as a host, the productivity of both NOH and FOH increased in pRS434GAP-HMG/A451. FOH could be produced at 3.8 mg/L on the average, with 11.2 mg/L at the highest, by merely activating the transcription of HMGl gene (Fig. 13). In pRS444GAP-HMGl/A451, the yield of NOH was 0.16 mg/L at the highest; this clone was found to be effective mainly in the production of FOH.<br>
It is believed that A451 is different from conventionally used recombinant DNA host strains (such as YPH499) in the balance between squalene synthase activity and mevalonate pathway activity, and that famesyl diphosphate (FPP), an interaiediate metabolite, is accumulated when multiple copies of HMGl gene are present or the transcription of this gene is activated; as a result, FOH (a dephosphorylated product of FPP) is produced. Alternatively, it is believed that the ability to produce FOH was rendered to A451 as a result of mutation of CANl or AR07 seen in the genotype of A451.    This means that any strain<br><br>
having a balance similar to that of A451 between squalene synthase activity and mevalonate pathway activity, or any strain having mutation in CANl and/or AR07 can be expected to produce FOH upon introduction of HMGJ. With respect to FOH production, a tendency was observed that the use of pRS434GAP vector exhibits better productivity than pRS444GAP vector.<br>
(2)    Constitutive Promoter; HMG]; YPH499<br>
The plasmids listed below that had been obtained by inserting HMGl gene (PCR<br>
error-corrected gene) into vector pRS414PTadh, pRS414TPadh, pRS434GAP, pRS444GAP,<br>
pRS434PGK, pRS444PGK, pRS434TEF or pRS444TEF comprising a constitutive promoter<br>
ADHJp, GAPp (^TDH3p\ PGKlp or TEF2p, were introduced into YPH499.<br>
pRS414PTadh-HMGl<br>
pRS414TPadh-HMGl<br>
pRS434GAP-HMGl<br>
pRS444GAP-HMGl<br>
pRS434PGK-HMGl<br>
pRS444PGK-HMGl<br>
pRS434TEF-HMGl<br>
pRS444TEF-HMGl The resultant recombinants were cultured in YM medium supplemented with adenine sulfate at 40|ig/ml (the same medium was also used for other recombinants when YPH499 was used as a host). Culture conditions were the same as in (1) above. After completion of the cultivation, the pentane extract fraction from the culture broth was subjected to GC/MS analyses.    The yields of prenyl alcohols (NOH and FOH) were determined.<br>
The results are shown in Fig. 16. In Fig. 16, "414Pr', "414TP", "434" and "444" represent the results when pRS414PTadh, pRS414TPadh, pRS434xxx and pRS444xxx (where XXX indicates the alphabetical part of the name of the gene used in the promoter) vectors were used, respectively. The right utmost column represents the results when the host (YPH499) before gene transfer was cultured. As shown in Fig. 16, the yield of FOH is improved in every recombinant, and an increase in NOH productivity is observed in pRS434GAP-HMGl-, PRS444GAP-HMG1-, pRS434TEF-HMGl-, pRS444TEF-HMGl-, pRS434PGK-HMGl- or<br><br>
pRS444PGK-HMGl-introduced YPH499 clone.<br>
(3)	Constitutive Promoter; HMG1 EUG<br>
A451-, YPH499- or YPH500-derived EUG clones that exhibit Glc growth inhibition<br>
and have integrated the DNA of interest into the genome completely were selected (i.e., EUG8, EUG12 and EUG27). Then, plasmid pRS434GAP-HMGl or pRS444GAP-HMGl obtained by inserting HMGl gene (PCR error-corrected gene) into vector pRS434GAP or pRS444GAP comprising a constitutive promoter GAPp (^TDH3p) was introduced into EUG8 (NOH yield: 0.021 mg/L; FOH yield: 0.20 mg/L), EUG12 (NOH yield: 0.13 mg/L; FOH yield: 5.9 mg/L) and EUG27 (NOH yield: 0.038 mg/L; FOH yield: 3.2 mg/L). The yields of prenyl alcohols in the resultant recombinants were determined.<br>
The results are shown in Fig. 17 (EUG8), Fig. 18 (EUG12) and Fig. 19 (EUG27).<br>
EUG clones produce FOH when cultured in YM medium containing glucose (Glc) as the carbon source. The introduction of HMGl gene improved the productivity of FOH. A451-derived EUG8 is different from YPH499-derived EUG12 and YPH500-derived EUG27 in production profile. It is believed that clones derived from YPH strains are more suitable for production.<br>
These results revealed that it is possible to improve the productivity of prenyl alcohols in A451-derived clones, YPH499-derived clones and YPH500-derived clones by introducing HMGJ thereinto.<br>
(4)	Inducible promoter; HMGJ; A451 or AURGGlOl<br>
Plasmid pYES2-HMG obtained by inserting HMGl' (a PCR error mutant oi HMGl) into vector pYES2 comprising an inducible promoter GALlp was introduced into A451 and AURGGlOl (A451, aurlwAURl-Q prepared in Example 4.<br>
Each of the resultant recombinants was precultured. Then, 25 ^1 of the preculture (when a yeast recombinant with an inducible promoter was precultured, this amount was added not only in this experiment but in other experiments described later) was added to 2.5 ml of SG medium and cultured at 26°C for 4 days with rotary shaking at 130 r.p.m. Prior to the addition to SG medium, cells were washed with physiological saline so that no glucose component was brought into SG medium. After completion of the cultivation in SG medium, the yields of prenyl alcohols (NOH and FOH) were determined.<br><br>
As a resuh, pYES-HMGl/AURGGlOl clones produced NOH at 1.43 mg/L on the average and FOH at 4.31 mg/L on the average. Thus, prenyl alcohol high yielding clones were obtained even in those recombinants in which pYES-HMGl comprising HMGl' (a mutant MMGl) has been transferred (Fig. 20). Fig. 20A shows the results when A451 was used. Fig. 20B shows the results when AURGGlOl was used. pYES is a vector that was used for the gene transfer.<br>
When AURGGlOl derived from A451 was used as a host and GALJp as a promoter, clones were obtained that highly produced FOH in particular.<br>
(5)    Inducible Promoter; HMG]; W303-1A or W303-1B<br>
Plasmid pYES2-HMG obtained by inserting HMGJ into vector pYES2 comprising an inducible promoter GALlp was introduced into W303-1A and W303-1B. The resultant recombinants were cultured in SG medium. Thereafter, the yields of prenyl alcohols (NOH and FOH) were determined (Fig. 21).<br>
When HMGJ was introduced (the column at the left in each graph), the yields of both products increased. W303 clones were characterized by their effectiveness in the production of NOH.<br>
[EXAMPLE 8]    Production of Prenyl Alcohols by High Expression of Deletion Mutant Type<br>
HMG-CoA Reductase Gene In Example 7, prenyl alcohol-producing recombinant yeasts were developed using a full-length HMG-CoA reductase gene or a mutant thereof     In this Example, prenyl alcohol-producing recombinant yeasts were developed using a deletion mutant of HMG-CoA reductase gene, and alcohol production was carried out.<br>
(1)    Inducible Promoter; HMGJA; A451<br>
The following plasmids (described in (4) in Example 3) obtained by inserting a deletion mutant of HMGJ' gene into a vector pYES2 comprising an inducible promoter GALlp were introduced separately into A451.<br>
pYHMG026<br>
pYHMG044<br>
pYHMG056<br><br>
pYHMG062<br>
pYHMG076<br>
pYHMGOSl<br>
pYHMGlOO<br>
pYHMG112<br>
pYHMG122 After completion of cultivation in SG medium, the yields of prenyl alcohols (NOH and FOH) were determined (Fig. 22).<br>
When a deletion mutant type HMGl gene was expressed with an inducible promoter, FOH high yielding clones were obtained. For FOH production, HMGIA044 and HMGJA122 were effective (FOH yield was 0.0 mg/L on the average in HMG1/A451 clones).<br>
(2)    Inducible promoter;//MG7A; AURGGl 01<br>
The following plasmids (described in (4) in Example 3) obtained by inserting a deletion mutant of HMGl' gene into a vector pYES2 comprising an inducible promoter GALlp were introduced separately into AURGGIOI.<br>
pYHMG026<br>
pYHMG044<br>
pYHMG056<br>
pYHMG062<br>
pYHMG076<br>
pYHMGOSl<br>
pYHMGlOO<br>
pYHMG112<br>
pYHMG122<br>
pYHMG133<br>
After completion of cultivation in SG medium, the yields of prenyl alcohols (NOH and FOH) were determined (Figs. 22 and 23). In Fig. 23, the right utmost columns represent the yields of host AURGGIOI before gene transfer.    Fig. 24 shows enlarged graphs of Fig. 23.<br>
In particular, when HMGlA044 was expressed with an inducible promoter, a prenyl alcohol high yielding clone (clone 15-2) was obtained. NOH yield and FOH yield in this recombinant were 12 mg/L and 31.7 mg/L on the average, respectively (Fig. 23).     The<br><br>
maximum yields were 23 mg/L and 53 mg/L, respectively. In those recombinants integrating HMGIA other than HMGJA044, clones were obtained that produce NOH and FOH at about 0.05-0.06 mg/L (Fig. 24). The recombinant integrating //A/G7A062 produced NOH at 0.11 mg/L and FOH at 0.13 mg/L at the maximum.<br>
(3)    Constitutive Promoter; HMGl, Inducible promoter; HMGJA; AURGGlOl<br>
pRS434GAP-HMGl  or pRS444GAP-HMGl  prepared in (2) in Example 7 was<br>
introduced into clone 15-2 prepared in (2) above in this Example.    After completion of<br>
cultivation in SG medium, the yields of prenyl alcohols (NOH and FOH) were determined<br>
(Fig 25).<br>
As a result, a clone was obtained that produced FOH at 66 mg/L at the maximum,<br>
improving the FOH yield of 53 mg/L of clone 15-2.<br>
[EXAMPLE 9]    Production of Prenyl Alcohols by Escherichia coli<br>
(1)	The plasmids obtained in (4), (5) and (7) in Example 2 were introduced separately into E.<br>
coli JM109. To a 50 ml medium containing 2x YT and 1 mM IPTG in a 300 ml flask, 0.5 ml<br>
of a preculture was added. Antibiotics (ampicillin and chloramphenicol), if necessary, 5 mM<br>
(about 0.12% (w/v)) IPP and 5 mM DMAPP were added thereto, and the cells were cultured<br>
at 37°C for 16 hr under shaking.<br>
After completion of the cultivation at 37°C for 16 hr, potato acid phosphatase was added to the culture supernatant and the cell pellet disrupted by sonication, followed by extraction of prenyl alcohols v^th pentane as an organic solvent. Then, the prenyl alcohols were identified and quantitatively determined by GC/MS as described in (1) in Example 7.<br>
As a result, FOH yield in the presence of IPP and DMAPP was 86.4 mg/L when wild type ispA was introduced (pALispA in Fig. 29) and 12.0 mg/L when wild type j^5 was introduced (pFE15NS2.9-l in Fig. 26). Even when a mutant ispA was introduced, JM109 retaining pl8M or pl8E produced FOH at 11.1 mg/L and 16.3 mg/L, respectively; JM109 retaining p4D produced FOH at 72.7 mg/L; and in JM109 retaining pl6D, FOH yield reached 93.3 mg/L (Fig. 26).<br>
(2)	In order to ascertain whether or not prenyl alcohol production can be carried out without<br>
the addition of IPP and DMAPP, plasmids pALispA4 and p3-47-ll or plasmids pALispA4<br><br>
and p3-47-13 obtained in (4) and (6) in Example 2 were introduced into E. coli JM109. To a medium containing 50 ml of 2x YT per 300 ml flask and 1 mM IPTG, 0.5 ml of a preculture was added. Antibiotics (ampicillin and chloramphenicol) were added thereto, if necessary. Then, the cells were cultured at 37°C for 16 hr under shaking. The results revealed that JMI09 retaining pALispA4 and p3-47-ll has FOH production ability of 0.15 mg/L and that JM109 retaining pALispA4 and p3-47-13 has FOH production ability of 0.16 mg/L (Fig. 27). Thus, it was found that E. coli retining plasmid p3-47-ll or p3-47-13 containing idi and plasmid pALispA4 containing ispA, i.e., E. coli incorporating idi and ispA has ability to produce FOH at 0.15-0.16 mg/L even without the addition of IPP and DM APR<br>
[EXAMPLE 10]    Mass Production of FOH 1.    Culture Conditions<br>
One platinum loopful of the recombinant yeast clone 15-2 (AURGGlOl retaining pYHMG044) described in (2) in Example 8 was inoculated from slants into CSM-URA medium (BIO 101 Inc.) and DOB medium (BIO 101 Inc.) (200 ml in a 500 ml Erlenmeyer flask with baffle plates) and cultured at 30°C for 2 days under shaking at 130 r.p.m. Then, in order to remove the glucose contained in the culture broth completely, centrifugation (at 1500 g, for 5 min, at 4°C) and washing with sterilized physiological saline were repeated 3 times. Subsequently, 50 ml of the culture was inoculated into a fermenter (1%) and cultured under the conditions described below.<br>
Fermenter medium:<br>
5% galactose<br>
Amino acid-containing YNB (Difco)<br>
1% soybean oil (Nacalai Tesque)<br>
0.1% Adekanol LG109 (Asahi Denka) Operational conditions:<br>
Cultivation apparatus: MSJ-U 10 L Cultivation Apparatus (B. E. Marubishi)<br>
Medium volume: 5 L<br>
Cultivation temperature: 26°C<br>
Aeration rate: 1 vvm<br>
Agitation: 300 rpm<br><br>
pH: controlled proportionally using 4 N sodium hydroxide solution and 2N hydrochloric<br>
acid solution, and with the following parameters:<br>
Proportional Band	LOO<br>
Non Sensitive Band	0.15<br><br>
Control Period	16 Sec<br>
Full Stroke	1 Sec<br>
Minimum Stroke	0 Sec<br>
2.	Cell Counting<br>
One hundred microliters of the culture broth was diluted 1- to 20-fold with physiological saline. Then, cells were counted with a hematometer (Hayashi Rikagaku). The number of cells in 0.06 mm (corresponding to 9 minimum grids) was counted, followed by calculation of the average of 4 counts. Then, using the formula below, cell count per liter of the culture broth was calculated.<br>
Cell count (IxlO^/L broth) = 0.444 x (cell count in 0.06 mm2 ) x dilution rate<br>
3.	Quantitative Determination of FOH<br>
FOH was identified and quantitatively determined in the same manner as in Example 8.<br>
4.	Results<br>
The results are shown in Fig. 28. As seen from Fig. 28, it was demonstrated that a recombinant yeast obtained by introducing HMG J A044 (a deletion mutant of the mutant type HMG-CoA reductase gene HMGl') into A451-derived AURGGlOl can produce 146 mg of FOH per liter of the culture broth on the average and 158 mg/L at the maximum.<br>
All publications, patents and patent applications cited herein are incorporated herein by reference in their entirety.<br>
INDUSTRIAL APPLICABILITY<br>
According to the present invention, a method of producing prenyl alcohols is provided. According to the present invention, biologically active prenyl alcohols can be obtained in lai'ge   quantities.      From   these   prenyl   alcohols,   isoprenoids/terpenoids   with   various<br><br>
physiological activities can be synthesized. The active prenyl alcohols provided in the invention may also be used as materials to find out those substances having a novel physiological activity.<br>
SEQUENCE LISTING FREE TEXT SEQ ID NOS: 18-74:    synthetic DNA<br><br><br><br><br><br>
CLAIMS<br>
1.	A method of producing a prenyl alcohol, comprising creating a recombinant obtained by<br>
transferring into a host a recombinant DNA for expression or a DNA fragment for genomic<br>
integration each comprising:<br>
(i)       a   hydroxymethylglutaryl-CoA   reductase    gene,   an   isopentenyl-diphosphate A-isomerase gene or a famesyl-diphosphate synthase gene, or a mutant of any one of said genes, (ii)    a transcription promoter, and (iii)    a transcription terminator; culturing said recombinant; and recovering the prenyl alcohol from the resultant culture.<br>
2.	The method according to claim 1, wherein the prenyl alcohol is a C15 prenyl alcohol.<br>
3.	The method according to claim 2, wherein the C15 prenyl alcohol is farnesol or nerolidol.<br>
4.	The method according to claim 3, wherein the concentration of famesol or nerolidol in the resultant culture is at least 0.05 mg/L.<br>
5.	The method according to any one of claims 1 to 4, wherein the<br>
hydroxymethylglutaryl-CoA reductase gene or mutant thereof comprises one nucleotide<br>
sequence selected from the group consisting of SEQ ID NOS: 1, 3, 5 and 7-16.<br>
6.	The method according to any one of claims 1 to 4, wherein the famesyl-diphosphate synthase gene or mutant thereof comprises one nucleotide sequence selected from the group consisting of SEQ ID NOS: 75, 77, 79, 81 and 83.<br>
7.	The method according to any one of claims 1 to 4, wherein the isopentenyl-diphosphate<br><br>
A-isomerase gene or mutant thereof comprises the nucleotide sequence as shown in SEQ ID NO: 85.<br>
8.	The method according to any one of claims 1 to 7, wherein the transcription promote: is one selected from the group consisting of ADHl promoter, TDH3 (GAP) promoter, PGKJ promoter, TEF2 promoter, GALl promoter and tac promoter.<br>
9.	The method according to any one of claims 1 to 7, wherein the transcription terminator is ADHl terminator or CYCl terminator.<br>
10.    The method according to any one of claims 1 to 9, wherein the host is yeast or Escherichia coli.<br>
11.	The method according to claim 10, wherein the yeast is Saccharomyces cerevisiae.<br>
12.	The method according to claim 11, wherein the Saccharomyces cerevisiae is A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
13.	A recombinant obtained by transferring into a host a recombinant DNA for expression or a DNA fragment for genomic integration each comprising:<br>
(i)    a hydroxymethylglutaryl-CoA reductase gene, an isopentenyl-diphosphate A-isomerase<br>
gene or a famesyl-diphosphate synthase gene, or a mutant of any one of said genes, (ii)    a transcription promoter, and (iii)    a transcription terminator, said recombinant being capable of producing at least 0.05 mg/L of famesol or nerolidol.<br>
14.	The recombinant according to claim  13, wherein the hydroxymethylglutaryl-CoA<br>
reductase gene or mutant thereof comprises one nucleotide sequence selected from the group<br><br>
consisting of SEQ ID NOS: 1, 3, 5 and 7-16.<br>
15.	The recombinant according to claim 13, wherein the farnesyl-diphosphate synthase gene or mutant thereof comprises one nucleotide sequence selected from the group consisting of SEQ ID NOS: 75, 77, 79, 81 and 83.<br>
16.	The recombinant according to claim 13, wherein the isopentenyl-diphosphate A-isomerase gene or mutant thereof comprises the nucleotide sequence as shown in SEQ ID NO: 85.<br><br>
17.	The recombinant according to any one of claims 13 to 16, wherein the transcription promoter is one selected from the group consisting of ADHl promoter, TDH3 (GAP) promoter, PGKl promoter, TEF2 promoter, GALl promoter and tac promoter.<br>
18.	The recombinant according to any one of claims 13 to 16, wherein the transcription terminator is ADHl terminator or CYCl terminator.<br>
19.    The recombinant according to any one of claims 13 to 18, wherein the host is yeast or Escherichia coli.<br>
20.	The recombinant according to claim 19, wherein the yeast is Saccharomyces cerevisiae,<br>
21.	The recombinant according to claim 20, wherein the Saccharomyces cerevisiae is A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br><br>
22.	A method of producing a prenyl alcohol substantially as herein described with reference to the accompanying drawings.<br>
23.	A recombinant substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE2NC1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1164-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="201234-bulk-handung-apparatus.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="201310-a-method-of-manufacturing-integral-wheel-rim-and-disc-assembly-of-a-5-taper-bead-seat-of-flat-or-semi-drop-center-rim-and-integral-wheel-construction.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>201309</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1164/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Jul-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Jul-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TOYOTA JIDOSHA KABUSHIKI KAISHA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 TOYOTA-CHO, TOYATA-SHI, AICHI-KEN 471-8571, JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OHTO, CHIKARA</td>
											<td>C/O TOYOTA JIDOSHA KABUSHIKI KAISHA, 1, TOYOTA-CHO, TOYOTA-SHI, AICHI 471-8571, JAPAN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>OBATA, SHUSEI</td>
											<td>C/O TOYOTA JIDOSHA KABUSHIKI KAISHA, 1, TOYOTA-CHO, TOYOTA-SHI, AICHI 471-8571, JAPAN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/52</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP01/11213</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-12-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-401701</td>
									<td>2000-12-28</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/201309-method-of-producing-prenyl-alcohols by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:03:37 GMT -->
</html>
