<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/201498-a-method-for-the-preparation-of-compounds-for-making-emamectin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:22:07 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 201498:A METHOD FOR THE PREPARATION OF COMPOUNDS FOR MAKING EMAMECTIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR THE PREPARATION OF COMPOUNDS FOR MAKING EMAMECTIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;A method for the preparation a compounds for making emamectin&quot; This invention relates to a method of preparing compounds of the formula I by reacting compound of the formula II into contact with a polypeptide that exhibits enzymatic activity of P450 monooxygenase to form a compound of formula III and reacting said compound with an amine of formula HN(R8)R9.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS AND COMPOSITIONS FOR MAKING EMAIVIKCTIN<br>
The invention relates to the field ofagrochemicals, and in particular, to insecticides. More specifically, this invention relates to the derivati/ation of avermectin, paiticiilarly for the svnthesis of emamectin.<br>
l^mamcclin is a potent insecticide and controls many pests such as Ihrips, leafminers, and worm pests including alfalfa caterpillar, beet am^iyworm, cabbage loopcr, com carworm, cutworms, diamondback moth, tobacco budworm, tomato fniitworm, and tomato pinworm. Emamectin (4"-dcoxy-4"-epi-N-mcthyIamino avermectin Bla/Bib) is described in U.S. Patent No. 4,874,749 and in Cvetovich, R.J. a r//., J. Ors^anic Clicm. 59:7704-7708, 1994 (as MK-244).<br>
U.S. Patent No. 5,288,710 describes salts of emamectin that arc especially valuable agrochemically. These salts of emamectin arc valuable pesticides, especially for combating insects and representatives of the order Acarina. Some pests for which emamectin is useful in combating are listed in European Patent Application EP-A 736,252.<br>
One drawback to the use of emamectin is the difficulty of its synthesis from avermectin. This is due to the first step of the process, which is the most costly and time-consuming step of producing emamectin, in which the 4'-carbinol group of avermectin must be oxidized to a ketone. The oxidation of the 4*'-carbinol group is problematic due to the presence of two other hydroxy! groups on the molecule that must be chemically protected before oxidation and dcprotected after oxidation. Thus, this first step, significantly increases the overall cost and time of producing emamectin from avermectin.<br>
Because of the efficacy and potency of emamectin as an insecticide, there is a need to develop a cost and time effective method and/or reagent for regioselectively oxidizing the 4"-carbinol group of avermectin to produce 4"-keto-avermectin, which is a necessary intermediate for producing emamectin from avermectin.<br>
The invention provides a novel family of P450 monooxygenases, each member of which is able to regioselectively oxidize the 4'-carbinol group of unprotected avermectin, thereby resulting in a cheap, effective method to produce 4"-keto-avermeclin, a necessary intermediate in the production of emamectin. The invention allows elimination of the costly, time-consuming steps of (1) chemically protecting the two other hydroxy! groups on the avermectin<br><br>
molecule prior to oxidation of the 4"-carbinol group that must be chemically protected before oxidation; and (2) chemically dcprotecting these two other hydroxyl groups after oxidation. The invention thus provides reagents and methods for significantly reducing the overall cost of producing emamcctin from avermectin.<br>
■	Accordingly, in one aspect, the invention provides a purified nucleic acid molecule encoding a polypeptide that exhibits an cn/ymatic activity of a P45() monooxygenasc and rcgioselectively oxidizes avermectin to 4"-keto-avermectin.<br>
■	In a specific embodiment , the invention relates to an purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a P450 monooxygenasc and rcgioselectively oxidizes avermectin to 4"-keto-avcrmectin, which polypeptide is substantially similar, and preferably has between at least 50%, and 99% amino acid sequence identity to the polypeptide of SEQ ID NO:2, with each individual number within this range of between 50% and 99%; also being part of the invention.<br>
■	In a further specific embodiment , the invention relates to an purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a P450 monooxygenasc and rcgioselectively oxidizes avermectin to 4"-keto-avermectin, which polypeptide is immunologically reactive with antibodies raised against a polypeptide of SEQ ID NO:2.<br>
■	The invention further provides a purified nucleic acid molecule comprising a nucleotide sequence<br><br>
a)	as given in SEQ ID NO: 1;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO: 1, or the complement thereof;<br>
e)	complementary to (a), (b) or (c);<br>
0    which is the reverse complement of (a), (b) or(c), or<br><br>
g) which is a (unclional part of (a), (b), (c), (d), (e) or (0 encoding a polypeptide that still exhibits an enzymatic activity of a P450 monooxygcnase and regiosclcctively oxidizes avermectin to 4'-kcto-avermcctin.<br>
■	In a specific embodiment , the invention relates to a purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a P450 monooxygcnase and regiosclcctively oxidizes avermectin to 4'-keto-avermeclin, which polypeptide is substantially similar, and preferably has at least between 60%, and 99% amino acid sequence identity to the polypeptide of SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20. SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID N0:9.S, with each individual number within this range of between 60% and 99%J also being part of the invention.<br>
■	In a further specific embodiment , the invention relates to an purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a P450 monooxygcnase and regioselectivcly oxidizes avermectin to 4"-keto-avermectin, which poKpcptide is immunologically reactive with antibodies raised against a polypeptide of SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
•	The invention further provides a purified nucleic acid molecule comprising a<br>
nucleotide sequence<br>
a)	as given in SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NO:l 1, SEQ ID NO:13, SEQ ID NO:15, SEQ ID NO:l7, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29. SEQ ID NO:31. SEQ ID NO:33, or SEQ ID NO:94;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO:l, SEQ ID<br><br>
NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO:!!, SEQ ID NO: 13,<br>
SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:2!, SEQ ID NO:23,<br>
SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:3 1, SEQ ID NO:33, or<br>
SEQ ID NO:94 or the complement thereof: e)    complementary to (a), (b) or (c); 0     which is the reverse complement of (a), (h) t)r(c): or g)    which is a functional part of (a), (b), (c), (d), (e) or (f) encoding a polypeptide that still<br>
exhibits an enzymatic activity of a P450 monooxygcnase and rcgiosclcctivcly oxidizes<br>
avcrmcctin to 4"-kelo-avermcclin.<br>
In certain embodiments, the nucleic acid molecule comprises or consists essentially of a nucleic acid secjuence that is at least between 66^^-, and 99^/^- identical to SEQ ID NO: 1, with each individual number within this range of between 66%, and 99% also being part of the invention..<br>
■	In certain embodiments, the nucleic acid molecule comprises or consists essentially of<br>
a nucleic acid sequence that is at least between 70%, and 99% identical to SEQ ID NO:I,<br>
SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID<br>
N0:I3, SEQIDNO:I5,SEQIDNO:I7, SEQIDNO:19,SEQIDNO:2!,SEQIDNO:23,<br>
SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, or SEQ<br>
ID NO:94, with each individual number within this range of between 70%, and 99% also<br>
being part of the invention..<br>
•	In some embodiments, the nucleic acid molecule comprises or consists essentially of a<br>
nucleic acid sequence that is at least 80% identical to SEQ ID NO:l, SEQ ID NO:3, SEQ ID NO:5. SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 1 L SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:2!, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94.<br>
■	In certain embodiments, the nucleic acid molecule comprises or consists essentially of<br>
a nucleic acid sequence that is at least 90% identical to SEQ ID NO:l, SEQ ID NO:3, SEQ<br>
ID N0:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID N0:11, SEQ ID NO: 13, SEQ ID NO: 15,<br>
SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID N0:21. SEQ ID NO:23, SEQ ID NO:25, SEQ ID<br>
NO:27, SEQ ID NO:29, SEQ ID N0:3!, SEQ ID NO:33, or SEQ ID NO:94.<br><br>
■	In certain embodiments, the nucleic acid molecule comprises or consists essentially of a nucleic acid sequence that is at least 95% identical to SEQ ID NO: I, SKQ ID N0:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID NO:9, SEQ ID NO: 1 K SEQ ID NO: 13, SEQ ID N0:15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94.<br>
■	In some embodiments, the nucleic acid molecule comprises or consists essentially of a nucleic acid sequence selected from the group consisting of SEQ ID NO: I, SEQ ID N0:3, SEQ ID N0:5, SEQ TD NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID N0:17, SEQ ID NO: 19, SEQ ID N0:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, and SEQ ID NO:94.<br>
■	In particular embodiments, the nucleic acid molecule is isolated from a Strcptomyces strain. In certain embodiments, the Strcptomyces strain is selected from the group consisting of Streptomyces luhercidicus, Strcptomyces lydicus, Strcptomyces platensis, Strcptomyces chattanoogensis, Strcptomyces kasu^aensis, and Strcptomyces rimosiis and Streptomyccs albofaciens..<br>
■	In some embodiments of Ihis aspect, the nucleic acid molecule further comprises a nucleic acid sequence encoding a tag which is linked to the P450 monooxygenasc via a covalent bond. In certain embodiments, the tajz is selected from the eroup consisting of a His tag, a GST tag, an HA tag, a HSV tag, a Myc-tag, and VSV-G-Tag.<br>
■	In another aspect, the invention provides a purified polypeptide that exhibits an enzymatic activity of a P450 monooxygenasc and regiosclectively oxidizes avermectin to 4'-keto-avermectin.<br>
■	In some embodiments, the polypeptide comprises or consists essentially of an amino acid sequence that is encoded by a nucleic acid molecule<br><br>
a)	as given in SEQ ID N0:1 or the complement thereof;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO: 1, or the complement thereof;<br>
e)	complementary to (a), (b) or (c);<br><br>
0     which is the reverse complement of (a), (b) or (c); or.<br>
g)    which is a runctional part of (a), (b), (c). (d), (e) or (0 encoding a polypeptide that still<br>
exhibits an cn/ymatic activity of a P45() mtinooxygcnasc and rcgioselectivcly oxidizes<br>
avermeclin to 4'-kelo-avcrmcctin. In some embodiments, the polypeptide comprises or consists essentially of an amino acid sequence that is between at least 50*/^', and W^/( identical to STIQ II) N0;2, with each individual number within this range of between 507o and 99% also being part of the invention..<br>
	In some embodiments, the polypeptide comprises or consists essentially of an amino<br>
acid sequence that is encoded by a nvicicic acid molecule<br>
a)	as given in SEQ ID NO: I, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID N0:9, SEQ ID NO:l L SEQ ID NO:I3, SEQ ID NO:l5, SEQ ID N0:I7, SEQ ID N0:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33. or SEQ ID NO:94 or the complement thereof;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof:<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO: 1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID N0:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94 or the complement thereof, or the complement thereof;<br>
e)	complementary to (a), (b) or (c);<br>
f)	which is the reverse complement of (a), (b) or (c); or<br>
g)	which is a functional part of (a), (b), (c), (d), (e) or (0 encoding a polypeptide that still exhibits an enzymatic activity of a P450 monooxygenase and regiosclcctivcly oxidizes avermectin to 4'-keto-avermectin.<br>
	In some embodiments, the P450 monooxygenase comprises or consists essentially ofan amino acid sequence that is between at least 60%, and 99% identical to SEQ ID NO:2,<br>
SEQ ID N0:4, SEQ ID N0:6, SEQ FD N0:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID<br>
NO: 14, SEQ ID NO:I6, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24,<br><br>
SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:?(). SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95, with each individual number within this range of between 60% and 99% also being part of the invention..<br>
	In certain embodiments, the P450 monoo.wgenase comprises or consists essentially of<br>
an amino acid sequence that is at least 70%. identical to SEQ ID NO:2, SEQ ID N0:4, SEQ<br>
ID NO:6, SEQ ID N0:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16,<br>
SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID<br>
NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
•	In some embodiments, the P450 monooxygenase comprises or consists essentially of<br>
an amino acid sequence that is at least 80%. identical to SEQ ID NO:2, SEQ ID NO:4, SEQ<br>
ID NO:6. SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16,<br>
SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22. SEQ ID NO:24, SEQ ID NO:26, SEQ ID<br>
NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID N0:y5.<br>
	In some embodiments, the P450 monooxygenase comprises or consists essentially of<br>
an amino acid sequence that is at least 90% identical to SEQ ID NO:2, SEQ ID N0:4, SEQ<br>
ID NO:6, SEQ ID NO;8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16,<br>
SEQ [D NO: 18, SEQ ID NO:20. SEQ ID NO:22. SEQ ID NO:24, SEQ ID NO:26, SEQ ID<br>
NO:28, SEQ ID NO;30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
	In certain embodiments, the P450 monooxygenase comprises or consists essentially of<br>
an amino acid sequence that is at least 95% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ<br>
ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID N0:12, SEQ ID NO: 14, SEQ ID NO: 16,<br>
SEQ ID NO: 18, SEQ ID NO:20. SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID<br>
NO:28, SEQ ID NO:30- SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
•	In some embodiments of this aspect of the invention, the P450 monooxygenase<br>
comprises or consists essentially of an amino acid sequence selected from the group<br>
consistmg of SEQ ID NO:2. SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10.<br>
SEQ ID NO: 12, SEQ ID NO: 14. SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID<br>
NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32,<br>
SEQ ID NO:34, and SEQ ID NO:95.<br>
■	In certain embodiments, the polypeptide according to the invention exhibiting an<br>
enzymatic activity of a P450 monooxygenase further comprises a tag.  In some<br><br>
embodiments, the tag is selected from the group consisting of a His tag, a GST lag, an HA tag, a HSV tag, a Myc-tag, and VSV-G-Tag.<br>
In another aspect, the invention provides a binding agent that specifically binds to a polypeptide according to the invention exhibiting an enzymatic activity of a P45() monooxygcnase that rcgiosclcctively oxidizes avermcctin to 4"-kelo-avermcctin. In some embodiments, the binding agent is an antibody.   In certain embodiments, the antibody is a polyclonal antibody or a monoclonal antibody.<br>
In yet another aspect, the invention provides a family of P450 monooxygcnase polypeptides, wherein each member of the family regioselectively oxidizes avermcctin to 4'-keto-avermectin.<br>
In certain embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is between at least 50%, and 99% identical to SEQ ID NO:2, with each individual number within this range of between 50% and 99% also being part of the invention..<br>
In certain embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is between at least 60%, and 99% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18. SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95, with each individual number within this range of between 60% and 99% also being part of the invention..<br>
In some embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is at least 70% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID N0:i2, SEQ ID NO:I4, SEQ ID NO:I6, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
In certain embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is at least 80% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8. SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95. In<br><br>
some embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is at least 90% identical to SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQIDNO:8,SEQIDNO:10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID N0:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
In certain embodiments, each member of the family comprises or consists essentially of an amino acid sequence that is at least 95% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95-<br>
In some embodiments of this aspect of the invention, each member of the family comprises or consists essentially of an amino acid sequence selected from the group consisting of SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO:12, SEQ ID N0:14, SEQ ID NO:16, SEQ ID NO:I8, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, and SEQ ID NO:95.<br>
In still another aspect, the invention provides a cell genetically engineered to comprise a nucleic acid molecule encoding a polypeptide which exhibits an enzymatic activity of a P450 monooxygenase that regioselectively oxidizes avermectin to 4"-kelo-avermcctin.<br>
In some embodiments, the nucleic acid molecule is positioned for expression in the cell. In certain embodiments, the cell further comprises a nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin protein.<br>
In some embodiments, the cell further comprises a nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin reductase protein.<br>
In certain embodiments, the cell is a genetically engineered Streptomyces strain. In certain embodiments, the cell is a genetically engineered Streptomyces lividans strain. In particular embodiments, the genetically engineered Streptomyces lividans strain has NRRL Designation No. B-30478. In some embodiments, the cell is a genetically engineered Pseudomonas strain. In some embodiments, the cell is a genetically engineered<br><br>
Psaulomonas putida strain.  In certain embodiments, the genetically engineered Pseiidomonds putidci strain has NRRL Designation No. B-30479.  in some embodiments, the cell is a genetically engineered Escherichia coli strain.<br>
In another aspect, the invention provides a purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide according to the invention exhibiting an cn/ymatic activity of a ferredoxin, wherein the nucleic acid molecule is isolated from a Strcptomyces strain comprising a P450 monooxygenasc that regiosclcctively oxidi/es avermectin to 4'-keto-avermcctin.<br>
In a specific embodiment , the invention relates to an purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits the cn/ymatic activity of a ferredoxin, which polypeptide is substantially similar, and preferably has between at least 80%, and 99% amino acid sequence identity to the polypeptide of SEQ ID NO:36 or SEQ ID NO: 38, with each individual number within this range of between 80%' and 99% also being part of the invention.<br>
In still a further specific embodiment , the invention relates to an purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide that exhibits the enzymatic activity of a ferredoxin, which polypeptide is immunologically reactive with antibodies raised against a polypeptide of SEQ ID NO: 36 or SEQ ID NO: 38.<br>
The invention further provides a purified nucleic acid molecule comprising a nucleotide sequence<br>
a)	as given in SEQ ID NO:35 or SEQ ID NO: 37;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing lo a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO: 35 or SEQ ID NO: 37, or the complement thereof;<br>
e)	complementary to (a), (b) or (c);<br>
f)	which is the reverse complement of (a), (b) or (c); or<br>
g)	which is a functional part of (a), (b), (c), (d), (e) or (f) encoding a polypeptide that still exhibits an enzymatic activity of a ferredoxin and regioselectively oxidizes avermectin to 4'-keto-avermectin.<br><br>
In certain embodiments, the nucleic acid molecule encoding a ferredoxin of the invention comprises or consists essentially of a nucleic acid sequence that is at least 81% identical to SEQ ID NO:35 or SEQ ID NO:37.  In some embodiments, the nucleic acid molecule comprises or consists esscniially ot" a nucleic acid sequence that is at least 85%, or at least ')()%. or at least 95%, or at least 99% identical to SEQ ID NO:35 or SEQ ID NO:37.  In certain embodiments, the nucleic acid molecule encoding a ferredoxin of the invention comprises or consists essentially of the nucleic acid sequence of SEQ ID NO:35 orSEQIDNO:37.<br>
In yet another aspect, the invention provides a purified ferredoxin protein, wherein the ferredoxin protein is isolated from a Streptomyces strain comprising a P450 monooxygenasc that rcgioselcctively oxidi/es avcrmectm to 4"-ket()-avermectin.  In certain embodiments, the ferredoxin of the invention comprises or consists essentially of an amino acid sequence that is at least 80%^ identical to SEQ ID NO:36 or SEQ ID NO:38.  In some embodiments, the nucleic acid molecule comprises or consists essentially of an amino acid sequence that is at least 85%, or at least 90%', or at least 95%, or at least 99% identical to SEQ ID NO:36 or SEQ ID NO:38.<br>
In particular embodiments, the ferredoxin of the invention comprises or consists essentially of the amino acid sequence of SEQ ID NO:36 or SEQ ID NO:38.<br>
In another aspect, the invention provides a purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin reductase, wherein the nucleic acid molecule is isolated from a Streptomyces strain comprising a P450 monooxygenasc that regioselectively oxidizes avermcctin to 4"-kcto-avermectin.<br>
In certain embodiments, the nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin reductase comprises or consists essentially of the nucleic acid sequence of SEQ ID NO:98, SEQ ID NO: 100. SEQ ID NO: 102, or SEQ ID NO: 104.<br>
In yet another aspect, the invention provides a purified polypeptide exhibiting an enzymatic activity of a ferredoxin reductase protein, wherein the said polypeptide is isolated from a Streptomyces strain comprising a P450 monooxygenasc that regioselectively oxidizes avermectin to 4"-keto-avermectin. In certain embodiments, the<br><br>
polypeptide of the invention comprises or consists essentially of the amino acid sequence<br>
of SEQ ID NO:99, SEQ ID NO: 101, SEQ ID NO: 103, or SEQ ID NO: 105.<br>
■	In another aspect, the invention provides a process for the preparation a compound of<br>
tlic fi)rmula<br><br>
in which<br>
Ri-Ro represent, independently of each other hydrogen or a subslituent;<br>
m is 0, 1 or 2;<br>
n is 0. 1,2 or 3; and<br>
the bonds marked with A, B, C, D, E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a taulomer<br>
thercot", in each case in free form or in salt form,<br>
which process comprises<br>
I) bringing a compound of the formula<br><br>
wherein<br>
RpR?, m, n. A, B, C, D. E and F have the same meanings as given for formula (I) above,<br>
into contact with a polypeptide according to the invention thai is capable of<br>
regioselectively oxidising the alcohol at position 4" in order to form a compound of the<br>
formula<br><br><br>
in which Ri, R2, R3, R4, R5, Rf,. Ry. rn, n. A, B, C, D, E and Fliavc ihc meanings given for formula (i); and<br>
2) reacting Ihc compound of the formula (11!) with an amine of the formula HN(RH)R'), wherein RH and R(&gt; have the same meanings as given for formula (I), and which is known, in the presence of a reducing agent;<br>
and, in each case, if desired, converting a compound of formula (!) obtainable in accordance with the process or by another method, or an E/Z isomer or tautomcr there &gt;' in each case in free form or in salt form, into a different compound of formula (!) or ;m E/Z isomer or tautomer thereof, in each case in free form or in salt form, separating a mixture ot'E/T. isomers obtainable in accordance with the process and isolating the desired isomer, and/or converting a free compound of formula (!) obtainable in accordance with the process or by another method, or an E/Z isomer or tautomer then &gt; f into a salt or converting a salt, obtainable in accordance with the process or by anotlu ' method, of a compound of formula (I) or of an E/Z isomer or tautomer thereof into liu free compound of formula (I) or an E/Z isomer or tautomer thereof or into a differcni salt. In some embodiments, the compound of formula (U) is further brought into conlae! with a polypeptide according to the invention exhibiting an enzymatic activity of a<br><br>
fcrrcdoxin. In certain embodiments, the compound of formula (II) is I'urlher brought into<br>
contact with a polypeptide according to the invention exhibiting an enzymatic activity of a<br>
ferredoxin reductase. In some embodiments, the compound of formula (11) is further<br>
brought into contact with a reducing agent (^'.,1,'.. NADU or NADPf I).<br>
■	In still a further embodiment, the invention provides a process for the preparation of a<br>
compound of the formula<br><br>
in which R|, R2, R3, Ra* R5, Rf,- R7, m, n. A, B, C, D, E and F have the meanings given for formula (I) of claim K which process comprises<br>
1) bringing a compound of the formula<br><br><br>
wnercin<br>
RrR7, m, n. A, B, C, D, E and F have the same meanings as given for formula (I) above,<br>
into contact with a polypeptide according to the invention that is capable of<br>
regioselcctively oxidising the alcohol at position 4", maintaining said contact for a time<br>
sufficient for the oxidation reaction to occur and isolating and purifying the compound<br>
of formula (il).<br>
■	In yet another embodiment, the invention provides a process according to the invention<br>
for the preparation of a compound of the formula (I), in which<br>
n is 1;<br>
m is 1;<br>
A is a double bond;<br>
B is single bond or a double bond,<br>
C is a double bond,<br>
D is a single bond,<br>
E is a double bond,<br>
F is a double bond; or a single bond and a epoxy bridge; or a single bond and a<br>
methylene bridge;<br>
Ri, R2 and R3 are H;<br><br>
R4 is methyl;<br>
R5 is C1-C10-alkyl, C.vCs-cycloalkyI or C:-Cu&gt;-a!kcnyl;<br>
R(, isH;<br>
R7isOH;<br>
RH and R9 arc independently of each (Mher H; Cj-Cio-alkyl or C|-Cn)-acyl: or together<br>
form -(CI I2X,-; and<br>
q is 4, 5 or 6.<br>
	In still another embodiment, the invention provides a process according to the<br>
invention for the preparation of a compound of the formula (f), in which<br>
n is 1;<br>
m is 1;<br>
A, B, C, E and F are double bonds;<br>
D is a single bond;<br>
Ri, R2, and R3 arc H;<br>
R4 is methyl;<br>
R5 is s-butyl or isopropyl;<br>
Rft is H:<br>
R7 is OH;<br>
Rg is methyl<br>
RoisH.<br>
•	In still another embodiment, the invention provides a process according to the<br>
invention for the preparation of 4"-deoxy-4"-N-methyIamino avermectin Bi/Bih benzoate salt-<br>
■	In another aspect, the invention provides a method for making emamectin. The<br>
method comprises adding a polypeptide according to the invention exhibiting an enzymatic<br>
activity of a P450 monooxvgenase that resioselectivelv oxidizes avermectin to 4"-keto-<br>
avermectin to a reaction mixture comprising avermectin and incubating the reaction<br>
mixture under conditions that allow the polypeptide to regioselectively oxidize avermectin<br>
to 4"-keto-avermectin. In some embodiments, the reaction mixture further comprises a<br>
polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin. In<br><br>
certain embodiments, the reaction mixture further comprises a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin reductase.  In some embodiments, the reaction mixture further comprises a reducing agent (c.^., NADH or NADPH).<br>
■	In still another aspect, the invention provides a formulation for making a compound of lormula (I) comprising a polypeptide according to the invention exhibiting a P450 monooxygenase activity that is capable of regioseleclively oxidising the alcohol at position 4" in order to form a compound of formula (H). In some embodiments, the formulation further comprises a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin (e.i^., a ferredoxin from cell or strain from which the P430 monooxygenase was isolated or derived).<br>
■	In still another aspect, the invention provides a formulation for making cmamectin comprising a P45() monooxygenase that regioselectively oxidizes avermectin to 4"-keto-avcrmectin.  In some embodiments, the formulation further comprises a ferredoxin {e.f^,, a ferredoxin from cell or strain from which the P450 monooxygenase was isolated or derived).<br>
■	In certain embodiments, the formulation further comprises a polypeptide according to the invention exhibiting an enzymatic activity of a ferredoxin reductase {e.f^., a ferredoxin from cell or strain from which the P45() monooxygenase was isolated or derived). In some embodiments, the formulation further comprises a reducing agent (c\g., NADH or NADPH).<br>
Brief Description of the Drawings<br>
Figure 1 is a diagrammatic representation showing a map of plasmid pTBBKA. Recognition sites bv the indicated restriction endonucleases are shown, aloniz w'ith the location of the site in the nucleotide sequence of the plasmid. Also shown are genes {e.g., kanamycin resistance "KanR'), and other functional aspects (e.g.. Tip promoter) contained in the plasmid.<br>
Figure 2 is a diagrammatic representation showing a map of plasmid pTUAlA. Recognition sites by the indicated restriction endonucleases are shown, along with the<br><br>
location of the site in the nucleotide sequence of the plasmid. Also shown are genes (e,g., ampicillin resistance "AmpR") and other functional aspects (e.g.. Tip promoter) contained in the plasmid.<br>
Figure 3 is a diagrammatic representation showing a map of plasmid pRK-emal/fd233 This plasmid was derived by ligating a Bglll fragment containing the emal and fd233 genesJ genes organized on a single transcriptional unit into the Bglli site of the broad host-range plasmid pRK29(). The emal/fd233 genes are expressed by the tac promoter (Ptac), and they are followed by the tac terminator (Ttac). Restriction endonuclcase recognition sites shown are Bglll ^'B'^; EcoRl ^'E"; Pad 'Tc"; Pmel ^Tm'; and Sail 'Sr<br>
The present invention provides a family of polypeptides which exhibit an enzymatic activity of a P45() monooxygcnases and arc capable of regioselectively oxidizing the alcohol at position 4" of a compound of formular (11) such as avermcclin in order to produce a compound of the formula (HI), but especially 4"-keto-avcrmcctin.<br>
More particularly, the family of polypeptides according to the invention may be used in a process for the preparation a compound of the formula<br><br>
in which<br>
R1-R9 represent, independently of each other hydrogen or a substituent;<br><br>
m is 0, 1 or 2;<br>
n is 0, 1, 2 or 3; and<br>
(he bonds marked with A, B, C, D, E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a tautomcr thereof, in each case in free form or in salt form, which process comprises<br>
1) bringing a compound of the formula<br><br>
wherein<br>
R1-R7, m, n, A, B, C, D, E and F have the same meanings as given for formula (1) above,<br><br>
into contact with a polypeptide according to the invention which exhibits an cn/.ymatic activity of a P45() monooxygcnases and is capable of rcgiosclcctivcly oxidizing the alcohol at position 4" of formular (IJ) in order to produce a compound of the formula (III)<br><br>
in which R|, R^, R3, R4. R5, R*., R?. rn, n. A, B, C, D, E and F have the meanings given for formula (I); and<br>
2) reacting the compound of the formula (III) with an amine of the formula HN(R8)RQ, wherein RH and Ro have the same meanings as given for formula (1), and which is known, in the presence of a reducing agent:<br>
and, in each case, if desired, converting a compound of formula (I) obtainable in accordance with the process or by another method, or an E/Z isomer or lautomer thereof, in each case in free form or in salt form, into a different compound of formula (I) or an E/Z isomer or tautomer thereof, in each case in free form or in salt form, separating a mixture of E/Z isomers obtainable in accordance with the process and isolating the desired isomer, and/or converting a free compound of formula (I) obtainable in accordance with the process or by another method, or an E/Z isomer or tautomer thereof, into a salt or converting a salt, obtainable in accordance with the process or by another method, of a compound of formula (I) or of an E/Z isomer or tautomer thereof into the free compound of formula (I) or an E/Z isomer or tautomer thereof or into a different salt.<br><br>
Methods of synthesis lor the compounds of formula (I) are described in the literature. It has been found, however, that the processes known in the literature cause considerable problems during production basically on account of the low yields and the tedious procedures which have to be used. Accordingly, the knov\ n processes are not satisfactory in that respect, giving rise to the need to make available improved preparation processes for those compounds.<br>
The compounds (I), (II) and (III) may be in the form of tautomers. Accordingly, hereinbefore and hereinafter, where appropriate the compounds (I), (II) and (HI) are to be understood to include corresponding tautomers, even if the latter arc not specifically mentioned in each case.<br>
The compounds (I), (II) and (III) arc capable of forming acid addition salts. Those salts are formed, for example, with strong inorganic acids, such as mineral acids, for example perchloric acid, sulfuric acid, nitric acid, nitrous acid, a phosphoric acid or a hydrohalic acid, with strong organic carboxylic acids, such as unsubstitutcd or substituted, for example halo-substituted, C1-C4alkanecarboxylic acids, for example acetic acid, saturated or unsaturated dicarboxylie acids, for example oxalic, malonic, succinic, malcic, fumaric or phlhalic acid, hydroxycarboxylic acids, for example ascorbic, lactic, malic, tartaric or citric acid, or benzoic acid, or with organic sulfonic acids, such as unsubslituled or substituted, for example halo-substituted, C1-C4alkane- or ar\l-suIfonic acids, for example methane- or p-toluene-sulfonic acid. Furthermore, compounds of formula (I), (11) and (III) having at least one acidic group are capable of forming salts with bases. Suitable salts with bases are, for example, metal salts, such as alkali metal or alkaline earth metal salts, for example sodium, potassium or magnesium salts, or salts with ammonia or an organic amine, such as morpholine, piperidine, pyrrolidine, a mono-, di- or tri-lower alkylamine, for example ethyl-, diethyl-, triethyl- or dimethyl-propyl-amine, or a mono-, di- or tri-hydroxy-lower alkylamine, for example mono-, di- or tri-ethanolamine. In addition, corresponding internal salts may also be formed. Preference is given within the scope of the invention to agrochemically advantageous salts. In view of the close relationship between the compounds of formula (I), (II) and (III) in free form and in the form of their salts, any reference hereinbefore or hereinafter to the free compounds of formula (I), (11) and (III) or to their respective salts is to be understood as including also the corresponding salts or the free compounds of formula (I), (11) and (HI), where appropriate and<br><br>
expedient. The same applies in the case of taulomers of compounds of tbrmula (I), (fl) and (III) and the salts thereof. The free form is generally preferred in each case.<br>
Preferred within the scope of this invention is a process for the preparation of compounds of the formula (1), in which<br>
n is 1;<br>
m is I:<br>
A is a double bond;<br>
B is single bond or a double bond,<br>
C is a double bond,<br>
D is a single bond,<br>
H is a double bond,<br>
F is a double bond; or a single bond and a epoxiy brdge; or a single bond and a methylene brdge;<br>
r, R2 and R3 are H;<br>
R4 is methyl;<br>
R5 is Ci-Cin-alkyK C.vCK-cycloalkyI or C^-Cio-alkcnyl;<br>
RftisH;<br>
RyisOH;<br>
Rg and R9 are independently of each other H; Ci-Cio-alkyI or Ci-Cio-acyl; or together form -(CH2)q-; and q is 4, 5 or 6.<br>
Especially preferred within the scope of this invention is a process for the preparation of a compound of the formula (I) in which n is I; m is 1;<br>
A, B, C, E and F are double bonds; D is a single bond; r, R2, and R3 are H; R4 is methyl; R5 is s-butyl or isopropyl;<br><br>
R6 is H; RvisOH; Rs is methyl R
Very especially preferred is a process for the preparation of emamectin, more particularly the ben/oate salt of emamectin. Fimamcctin is a mixture of 4'^Hlc(&gt;xy-4"-N-melhylamino avermectin Bu/Bib and is descrbed in US-P-4,4874,749 and as MK-244 in Journal of Organic Chemistry, Vol. 59 (1994), 7704-7708. Salts of emamectin that are especially valuable agrochcmicaily arc descrbed in US-P-5,288,710. Each member of this family of peptides exhibiting an enzymatic activity of a P450 monooxygcnases as descrbed hereinbefore is able to oxidize unprotected avermectin regioselectively at position 4'\ thus opening a new and more economical route for the production of emamectin.<br>
The family members each catalyze the following reaction:<br><br>
Accordingly, the invention provides a purfied nucleic acid molecule encoding a polypeptide that exhibits an enzymatic activity of a P450 monooxygenase and is capable of regioselectively oxidizing the alcohol at position 4" of a compound of formular (11) such as avermectin in order to produce a compound of formula (III), but especially 4"-kelo-avermectin.<br>
In particular, the invention provides a purfied nucleic acid molecule encoding a P450 monooxygenase that regioselectively oxidizes avermectin to 4'-keto-avermectin. A 'nucleic<br><br>
acid molecule" refers to single-stranded or double-stranded polynucleotides, such as deoxyrbonucleic acid (DNA), rbonucleic acid (rNA), or analogs of either DNA or RNA.<br>
The invention also provides a purfied polypeptide that exhibits an enzymatic activity of a P45() monooxygcnase and is capable of regioselectively oxidizing the alcohol at position 4" of a compound of formular (11) such as avcrmectin in order to produce a compound of formula (III), but especially 4"-keto-avcrmectin.<br>
In particular, the invention also provides a purfied P450 monooxygcnase that regioselectively oxidizes avermectin to 4"-keto-avermectin.<br>
As used herein, by 'purfied' is meant a nucleic acid molecule or polypeptide {e.g., an enzyme or antibody) that has been separated from components which naturally accompany it. An example of such a nucleotide sec|uence or segment of interest 'purfied" from a source, would be nucleotide sequence or segment that is excised or removed from said source by chemical means, e.g., by the use of restrction cndonucleases, so that it can be further manipulated, e.g., amplified, for use in the invention, by the methodology of genetic engineerng. Such a nucleotide sequence or segment is commonly referred to as "recombinant.'". In one specific aspect, the purfied nucleic acid molecule may be separated from nucleotide sequences, such as promoter or enhancer sequences, that flank the nucleic acid molecule as it naturally occurs in the chromosome.<br>
In the case of a protein or a polypeptide, the purfied protein and polypeptide, respectively, is separated from components, such as other proteins or fragments of cell membrane, that accompany it in the cell. Of course, those of ordinary skill in molecular biology will understand that water, buffers, and other small molecules may additionally be present in a purfied nucleic acid molecule or purfied protein preparation. A purfied nucleic acid molecule or purfied polypeptide {e.g., enzyme) of the invention that is at least 95% by weight, or at least 98% by weight, or at least 99% by weight, or 100% by weight free of components which naturally accompany the nucleic acid molecule or polypeptide.<br>
According to the invention, a purfied nucleic acid molecule may be generated, for example, by excising the nucleic acid molecule from the chromosome. It may then be ligated into an expression plasmid. Other methods for generating a purfied nucleic acid molecule encoding a P450 monooxygcnase of the invention are available and include, without limitation, artificial synthesis of the nucleic acid molecule on a nucleic acid synthesizer.<br><br>
Similarly, a purfied P450 monooxygcnasc of the invention may be generated, for example, by recombinant expression of a nucleic acid molecule encoding the P450 monooxygenase in a cell in which the P45() monooxygenase does not naturally occur. Of course, other methods for obtaining a purfied P450 monooxygenase of the invention include, without limitation, artificial synthesis of the P45() monooxygenase on a polypeptide synthesi/cr and isolation of the P45() monooxygenase from a cell m which it naturally occurs using, 6^^^, an antibody that specifically binds the P450 monooxygenase.<br>
Biotransformations of secondary alcohols to ketones by Streptomyces bactera are known to be catalyzed by dehydrogenases or oxidases. However, pror to the present discovery of the cytochrome P450 monooxygenase from Streptomyces tubercidicus strain R-^.^22 responsible for the regioselectivc oxidation of avermectin to 4'-keto-avcrmectin, no expermental data of another cytochrome P45() monooxygenase from Streptomyces to oxidize a secondary alcohol to a ketone had been reported.<br>
According to some embodiments of the invention, the nucleic acid molecule and/or the polypeptide encoded by the nucleic acid molecule are isolated from a Streptomyces strain. Thus, the nucleic acid molecule (or polypeptide encoded thereby) may be isolated from, without limitation, Streptomyces tnhercidicus^ Streptomyces lydicus, Streptotnycesplatensis. Streptomyces chattanoogensis, Streptomyces kasugaensis, Streptomyces rmosus, and Streptomyces alhofaciens.<br>
As mentioned above and descrbed in more detail below, an entire family of polypeptides exhibiting an enzymatic activity of P450 monooxygenases capable of regioselectively oxidizing avermectin to 4"-keto-avermectin are provided herein. All of these family members are related by at least 609r identity at the amino acid level. A useful nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenase comprses or consists essentially of a nucleic acid sequence that is at least 70^r identical to SEQ ID iNO: 1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO:17, SEQ ID NO:19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94. In certain embodiments, the nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenase<br><br>
comprses or consists essentially of a nucleic acid set|iicnce that is at least 80% identical; or at least 85% identical; or at least 90% identical; or at least 95%; identical; or at least 98% identical to SEQ ID N0:1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: I 1, SEQ ID NO: 13, SEQ ID NO: 15, SE:Q ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SPiQ ID NO:25, SEQ ID NO:27. SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33.orSEiQlDNO:94.<br>
Similarly, the invention provides a purfied polypeptide exhibiting an enzymatic activity of a P450 monooxygenase that regioselectively oxidizes avermectin to 4"-keto-avcnncctin which, in some embodiments, comprses or consists essentially of an amino acid sequence that is at least 60% identical to SEQ ID NO:2. SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14. SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.  In certain embodiments, the purfied polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenasc comprses or consists essentially of an amino acid sequence that is at least 70%) identical; or at least 80% identical; or at least 90%^ identical; or at least 95% identical to SEQ ID N0:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8. SEQ ID NO: 10. SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20. SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32. SEQ ID NO:34, or SEQ ID NO:95.<br>
In some embodiments, the nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenase comprses or consists essentially of the nucleic acid sequence of SEQ ID NO: 1. SEQ ID NO:3. SEQ ID N0:5. SEQ ID NO:7. SEQ ID NO;9. SEQ ID NO: 11. SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29. SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94. Similarly, the purfied polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenase, in some embodiments, comprses or consists essentially of the amino acid sequence of SEQ ID NO:2. SEQ ID NO:4, SEQ ID NO:6, SEQ ID N0;8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14. SEQ ID NO: 16. SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24. SEQ ID NO:26. SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br><br>
To dcscnbc the sequence relationships between two or more nucleic acids or polynucleotides the following terms are used: (a) "reference sequence", (b) "comparson window", (c) "sequence identity", (d) "percentage of sequence identity", and (e) "substantial identity".<br>
(a)	As used herein, "reference scc|ucnce" is a defined sequence used as a basis for sctiucnce comparson. A reference sec|uence may be a subset or the entirety of a specified sequence; for example, as a segment of a full length cDNA or gene sequence, or the complett^ cDNA or gene sequence.<br>
(b)	As used herein, "comparson window" makes reference to a contiguous and specified segment of a polynucleotide sequence, wherein the polynucleotide sequence in the comparson window may comprse additions or deletions (i.e., gaps) compared to the reference sequence (which does not comprse additions or deletions) for optimal alignment of the two sequences. Generally, the comparson window is at least 20 contiguous nucleotides in length, and optionally can be 30, 40, 50, 100, or longer. Those of skill in the art understand that to avoid a high similarty to a reference sequence due to inclusion of gaps in the polynucleotide sequence a gap penalty is typically introduced and is subtracted from the number of matches.<br>
Methods of alignment of sequences for comparson are well known in the art. Thus, the determination of percent identity between any two sequences can be accomplished using a mathematical algorthm. Preferred, non-limiting examples of such mathematical algorthms are the algonthm of Myers and Miller, 1988; the local homology algorthm of Smith ct al. 1981; the homology alignment algorthm of Needleman and Wunsch 1970; the search-for-similarty-method of Pearson and Lipman 1988; the algorthm of Karlin and Altschul. 1990, modified as in Karlin and Altschul, 1993.<br>
Computer implementations of these mathematical algorthms can be utilized for companson of sequences to determine sequence identity. Such implementations include, but are not limited to: CLUSTAL in the PC/Gene program (available from Intelligenetics, Mountain View, Califomia); the ALIGN program (Version 2.0) and GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Version 8 (available from Genetics Computer Group (GCG), 575 Science Drve, Madison, Wisconsin, USA). Alignments using these programs can be performed using the default parameters. The<br><br>
CLUSTAL program is well descrbed by fliggins e( al. 1988; Higgins ct al. 1989; Corpel et al. 1988; Huang el al. 1992; and Pearson ct al. 1994. The ALIGN program is based on the algorthm of Myers and Miller, supra. 1'he BLAST programs of Altschul et al., 1990, are based on the algorthm of Karlin and Altschul supra.<br>
Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Informalicm (http://w\\\\ .ncbi.nlm.nih.gcw/). This algorthm involves first identifying high scorng sequence pairs (MSPs) by identifying shoit words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al., 1990). These initial neighborhood word hits act as seeds for initiatmg searches to find longer I ISPs containing them. The word hits are then extended in both directions along each sequence for as far as the cumulative alignment score can be increased.  Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always &gt; 0) and N (penalty score for mismatching residues; always 
In addition to calculating percent sequence identity, the BLAST algorthm also performs a statistical analysis of the similarty between two sequences (see, e.g., Karlin &amp; Altschul (1993). One measure of similarty provided by the BLAST algorthm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a test nucleic acid sequence is considered similar to a reference sequence if the smallest sum probability in a comparson of the test nucleic acid sequence to the reference nucleic acid sequence is less than about 0.1, more preferably less than about 0.01, and most preferably less than about 0.001.<br>
To obtain gapped alignments for comparson purposes. Gapped BLAST (in BLAST 2.0) can be utilized as descrbed in Altschul et al. 1997. Alternatively, PSI-BLAST (in BLAST 2.0) can be used to perform an iterated search that detects distant relationships<br><br>
between molecules. See Altschul et al., supra.  When utilizing BLAST, Gapped BLAST, PSI-BLAST, the default parameters of the respective programs (e.g. BLASTN for nucleotide sequences, BLASTX for proteins) can be used. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (\V) of 1 1, an expectation (E) of 10, a cutoff of 100, M-5, N=-4, and a comparson of both strands. For amino acid sequences, the BLASTP program uses as defaults a wordlength (VV) of 3. an expectation (E) of 10, and the BLOSUM62 scorng matrx (see Henikoff &amp; Henikoff, 1989). See lUtpV/www.ncbi.n 1 m.nih.gov. Alignment may also be performed manually by inspection. For purposes of the present invention, comparson of nucleotide sequences for determination of percent sequence identity to the nucleotide sequences disclosed herein is preferably made using the BlaslN program (version 1.4.7 or later) with its default parameters or any e(|uivalent program.  By "equivalent program" is intended any sequence comparson program that, for any two sequences in question, generates an alignment having identical nucleotide or amino acid residue matches and an identical percent sequence identity when compared to the corresponding alignment generated by the preferred program.<br>
(c) As used herein, "sequence identity" or "identity" in the context of two nucleic acid or polypeptide sequences makes reference to the residues in the two sequences that are the same when aligned for maximum correspondence over a specified comparson window. When percentage of sequence identity is used in reference to proteins it is recognized that residue positions which are not identical often differ by conservative amino acid substitutions, where amino acid residues are substituted for other amino acid residues with similar chemical properties (e.g., charge or hydrophobicity) and therefore do not change the functional properties of the molecule. When sequences differ in conservative substitutions, the percent sequence identity may be adjusted upwards to correct for the conservative nature of the substitution. Sequences that differ by such conservative substitutions are said to have "sequence similarty" or "similarty." Means for making this adjustment are well known to those of skill in the art. Typically this involves scorng a conservative substitution as a partial rather than a full mismatch, thereby increasing the percentage sequence identity. Thus, for example, where an identical amino acid is given a score of 1 and a non-conservative substitution is given a score of zero, a conservative substitution is given a score between zero<br><br>
and I. The scorng of conservative substitutions is calculated, e.g., as implemented in the program PC/GENE (Intelligenelics, Mountain View, Calirornia).<br>
(d) As used herein, "percentage of sequence identity" means the value determined by comparng two optimally aligned sequences over a comparson window, wherein the portion of the polynucleotide sequence in the comparson window may comprse additions or deletions (i.e., gaps) as compared to the reference .sequence (which does not comprse additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical nucleic acid base or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparson, and multiplying the result by 100 to yield the percentage of sequence identity. (c)(i) The term "substantial identity" of polynucleotide sequences means that a polynucleotide comprses a sequence that has at least 66%. 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, or 79%, preferably at least 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, or 89%, more preferably al least 90%, 91 %, 92%, 93%, or 94%, and most preferably at least 95%, 96%, 97%, 98%, or 999^ sequence identity, compared to a reference sequence using one of the alignment programs descrbed using standard parameters. One of skill in the art will recognize that these values can be approprately adjusted to determine corresponding identity of proteins encoded by two nucleotide sequences by taking into account codon degeneracy, amino acid similarty, reading frame positioning, and the like. Substantial identity of amino acid sequences for these purposes normally means sequence identity of at least 70%, more preferably at least 80%, 90%, and most preferably at least 95%.<br>
Another indication that nucleotide sequences are substantially identical is if two molecules hybrdize to each other under strngent conditions (see below). Generally, strngent conditions are selected to be about 5°C lower than the thermal melting point (T^) for the specific sequence at a defined ionic strength and pH. However, strngent conditions encompass temperatures in the range of about 1°C to about 20'C, depending upon the desired degree of strngency as othenvise qualified herein. Nucleic acids that do not hybrdize to each other under strngent conditions are still substantially identical if the polypeptides they encode are substantially identical. This may occur, e.g., when a copy of a nucleic acid is created using the maximum codon degeneracy permitted by the genetic code. One indication that two<br><br>
nucleic acid sequences are substantially identical is when the polypeptide encoded by the first nucleic acid is immunologically cross reactive with the polypeptide encoded by the second nucleic acid.<br>
(e)(ii)The term "substantial identity" in the context of a polypeptide indicates that a polypeptide comprses a sequence with at least 50%, 60%, 70%', 71%, 72%-, 737r. 74%^ 75%, 76%, 11%, 78%, or 79%, preferably 80%, 81%, S2%, 83^;^, 84^/r, 85^'f, 86^;^., 87%, 88%, or 89%, more preferably al least 90%, 91%, 92%, 93%, or 94%, or even more preferably, 95%, 96%, 97%, 98% or 99%, sequence identity to the reference sequence over a specified comparson window. Preferably, optimal alignment is conducted using the homology alignment algorthm of Needleman and Wunsch (1970). An indication that two polypeptide secjuences are substantially identical is that one polypeptide is immunologically reactive with antibodies raised against the second polypeptide. Thus, a polypeptide is substantially identical to a second polypeptide, for example, where the two peptides differ only by a conservative substitution.<br>
For sequence comparson, typically one sequence acts as a reference sequence to which test sequences arc compared. When using a sequence comparson algorthm, test and reference sequences are input into a computer, subsequence coordinates are designated if necessary, and sequence algorthm program parameters arc designated. The sequence comparson algorthm then calculates the percent sequence identity for the test sequence(s) relative to the reference sequence, based on the designated program parameters.<br>
As noted above, another indication that two nucleic acid sequences arc substantially identical is that the two molecules hybrdize to each other under strngent conditions. The phrase "hybrdizing specifically to" refers to the binding, duplexing, or hybrdi/.mg of a molecule only to a particular nucleotide sequence under strngent conditions when that sequence is present in a complex mixture (e.g., total cellular) DNA or RNA. "Bind(s) substantially" refers to complementar&gt;' hybrdization between a probe nucleic acid and a target nucleic acid and embraces minor mismatches that can be accommodated by reducing the strngency of the hybrdization media to achieve the desired detection of the target nucleic acid sequence.<br>
'Strngent hybrdization conditions" and 'strngent hybrdization wash conditions' in the context of nucleic acid hybrdization experments such as Southern and Northem<br><br>
hybrdi/.alion arc sequence dependent, and are different under different environmental parameters. The T„, is the temperature (under defined ionic strength and pH) at which 50% of llic target sequence hybrdizes to a perfectly matched probe. Specificity is typically the function of post-hybrdi/ation washes, the crtical factors being the ionic strength and temperature of the final wash solution. For DNA-DNA hybrds, the Tm can be approximated from the equation of Meinkoth and Wahl, 1984; T,,, 81.5^C + 16.6 (log M) +0.41 (%GC) -0.61 (% form) - 500/L; where M is the molarty of monovalent cations, %GC is the percentage of guanosine and cytosine nucleotides m the DNA, % form is the percentage of formamide in the hybrdization solution, and L is the length of the hybrd in base pairs. T^ is reduced by about 1C for each 1% of mismatching; thus, Tm, hybrdization, and/or wash conditions can be adjusted to hybrdize to sequences of the desired identity.  For example, if sequences with &gt;90% identity are sought, the Tm can be decreased 10°C. Generally, strngent conditions are selected to be about 5C lower than the thermal melting point I for the specific sequence and its complement at a defined ionic strength and pH. However, severely strngent conditions can utilize a hybrdization and/or wash at i, 2, 3, or 4C lower than the thermal melting point I; moderately strngent conditions can utilize a hybrdization and/or wash at 6, 7, 8, 9, or iC lower than the thermal melting point I; low strngency conditions can utilize a hybrdization and/or wash at 11, 12, 13, 14, 15, or 20C lower than the thermal melting point I. Using the equation, hybrdization and wash compositions, and desired T, those of ordinary skill will understand that varations in the strngency of hybrdization and/or wash solutions are inherently descrbed. If the desired degree of mismatching results in a T of less than 45^C (aqueous solution^ or 32'^C (formamide solution), it is preferred to increase the SSC concentration so that a higher temperature can be used. An extensive guide to the hybrdization of nucleic acids is found in Tijssen, 1993. Generally, highly strngent hybrdization and wash conditions are selected to be about 5*'C lower than the thermal melting point Tm for the specific sequence at a defined ionic strength and pH.<br>
An example of highly strngent wash conditions is 0.15 M NaCl at 72*'C for about 15 minutes. An example of strngent wash conditions is a 0.2X SSC wash at 65"C for 15 minutes (see, Sambrook, infra, for a descrption of SSC buffer). Often, a high strngency wash is preceded by a low strngency wash to remove background probe signal. An example medium<br><br>
strngency wash for a duplex of, e.g., more than 100 nucleotides, is IX SSC at 45'C for 15 minutes. An example low strngency wash for a duplex of, e.g., more than 100 nucleotides, is 4-6X SvSC at 40**C for 15 minutes.  For short probes (e.g., about 10 to 50 nucleotides), strngent conditions typically involve salt concentrations of less than about 1.5 M, more preferably about 0.01 to 1.0 M, Na ion concentration (or other salts) at pH 7.0 to 8.3, and the temperature is typically at least about 3(y'C and at least about GifC for long robes (e.g., &gt;50 nucleotides). Strngent conditions may also be achieved with the addition of destabilizing agents such as formamide.  In general, a signal to noise ratio of 2X (or higher) than that observed for an unrelated probe in the pailicular hybrdization assay indicates detection of a specific hybrdization.  Nucleic acids that do not hybrdize to each other under strngent conditions are still substantially identical if the proteins that they encode are substantially identical. This occurs, e.g., when a copy of a nucleic acid is created using the maximum codon degeneracy permitted by the genetic code.<br>
Very strngent conditions are selected to be equal to the Tm for a particular probe. An example of strngent conditions for hybrdization of complementary nucleic acids which have more than 100 complementary residues on a filler in a Southern or Northern blot is 50% formamide, e.g., hybrdization in 50% formamide, 1 M NaCI, 1% SDS at 37°C, and a wash in 0. 1X SSC at 60 to 65°C. Exemplary low strngency conditions include hybrdization with a buffer solution of 30 to 35% formamide, 1 M NaCI, 1%&gt; SDS (sodium dodecyl sulphate) at 37^C, and a wash in IX to 2X SSC (20X SSC = 3.0 M NaCl/0.3 M trsodium citrate) at 50 to 55°C. Exemplary moderate strngency conditions include hybrdization in 40 to 45% formamide, 1.0 M NaCI, 1% SDS at 3TC. and a wash in 0-5X to IX SSC at 55 to 60X.<br>
The following are examples of sets of hybrdization/wash conditions that may be used to clone orthologous nucleotide sequences that are substantially identical to reference nucleotide sequences of the present invention: a reference nucleotide sequence preferably hybrdizes to the reference nucleotide sequence in 7% sodium dodecyl sulfate (SDS), 0.5 M NaP04, 1 mM EDTA at 50X with washing in 2X SSC, 0.1%^ SDS at 50X, more desirably in 7% sodium dodecyl sulfate (SDS), 0.5 M NaP04, 1 mM EDTA at 50°C with washing in IX SSC, 0T% SDS at 50X, more desirably still in 7% sodium dodecyl sulfate (SDS), 0.5 M NaPOa, 1 mM EDTA at 50X with washing in 0.5X SSC. 0A% SDS at SO'^C, preferably in 7% sodium<br><br>
dodccyl sulfalc (SDS). 0.5 M NaPOj. 1 mM EDTA at 5()°C with washing in 0.1X SSC, 0.1% SDS at 5()X, more preferably in 1% sodium dodccyl suitate (SDS), 0.5 M NaPO^. I mM RDTA at 50^C with washing in 0.1X SSC, 0.1 % SDS at 65X.<br>
One non-limiling source of a par tied polypeptide of the invention exhibiting an en/ymatic activity of a P45() monooxygenase that regioselectively oxidi/es avcrmcctin to 4"-keto-avermectin is the cell-free extract descrbed in the examples below. Another method for purfying a polypeptide exhibiting a P450 monooxygenase activity in accordance with the invention is to attach a tag to the protein, thereby facilitating its purfication. Accordingly, the invention provides a purfied polypeptide exhibiting an enzymatic activity of a P450 monooxygenase which regioselectively oxidizes avermectin to 4"-keto-avermcctin, wherein the polypeptide is covalcntly bound to a tag. The invention further provides a nucleic acid molecule encoding such a tagged polypeptide.<br>
As used herein, a "tag' is meant a polypeptide or other molecule covalcntly bound to a polypeptide of the invention, whereby a binding agent {c.^., a polypeptide or molecule) specifically binds the tag. In accordance with the invention, by "specifically binds" is meant that the binding agent {e.g., an antibody or Ni^^ resm) recognizes and binds to a particular polypeptide or chemical but does not substantially recognize or bind to other molecules in the sample. In some embodiments, a binding agent that specifically binds a ligand forms an association with that ligand with an affinity of at least 10^ M\ or at least 10 M  , or at least 10**M"\ or at least 10*^ M* either in water, under physiological conditions, or under conditions which approximate physiological conditions with respect to ionic strength, e..t,'., 140 mM NaCI, 5 mM MgCl^- For example, a His tag is specifically bound by nickel {e.g.. the Ni^^-charged column commercially available as His-Bind® Resin from Novagen Inc, Madison, WI). Likewise, a Mye tag is specifically bound by an antibody that specifically binds Myc.<br>
As descrbed below, a His tag is attached to the purfied polypeptide of the invention exhibiting an enzymatic activity of a P450 monooxygenase by generating a nucleic acid molecule encoding the His-tagged polypeptide, and expressing the polypeptide in E. coli. These polypeptides, once expressed by E. coli, are readily purfied by standard techniques {e.g., using one of the His'Bind® Kits commercially available from Novagen or using the TALON'^'* Resin (and manufacturer's instructions) commercially available from Clontcch Laboratores, Inc., Palo Alto, CA).<br><br>
Additional tags may be attached to any or all of the polypeptides of the invention to facihlalc purfication. These lags inckide, without limitation, the HA-Tag (amino acid sequence: YPYDVPDYA (SEQ ID NO:39)), the Myc-tag (amino acid sequence: liQKLlSHfiDI. (SHO \0 NO:40)), the IISV tag (amino acid sequence: QPELAF^EDPED (SEQ ID N0:41)), and the VSV-G-Tag (ammo acid sequence: YTDIEMNRLGK (SEQ ID NO:42)). Covalent attachmenl (ci;., via a polypeptide hond) of these tags to a polypeptide of the invention allows purfication of the tagged polypeptide using, respectively, an anti-HA antibody, an anli-Myc antibody, an anli-HSV antibody, or an anti-VSV-G antibody, all of which arc commercially available (for example, from MBL International Corp., Watertown, MA; Novagen Inc.; Research Diagnostics Inc., Fhmders, NJ).<br>
rhc tagged polypeptides of the invention exhibiting a P45() monooxygenase activity may also be tagged by a covalent bond to a chemical, such as biotin, which is specifically bound by streptavidin, and thus may be purfied on a streptavidin column. Similarly, the tagged P45() monooxygenases of the invention may be covalently bound (t'.^'., via a polypeptide bond) to the constant region of an antibody. Such a tagged P450 monooxygenase may be purfied, for example, on protein A sepharose.<br>
The tagged P450 monooxygenases of the invention may also be tagged to a GST (glutathione-S-transfcrase) or the constant region of an immunoglobulin. For example, a nucleic acid molecule of the invention {e.g., comprsing SEQ ID NO:l) can be cloned into one of the pGEX plasmids commercially available from Amersham Pharmacia Biotech, Inc. (Piscataway NJ), and the plasmid expressed in E. coli. The resulting P450 monooxygenase encoded by the nucleic acid molecule is covalently bound to a GST (glulathionc-S-transferase). These GST fusion proteins can be purfied on a glutathione agarose column (commercially available from, e.g., Amersham Pharmacia Biotech), and thus purfied. Many of the pGEX plasmids enable easy removal of the GST portion from the fusion protein. For example, the pGEX-2T plasmid contains a thrombm recognition site between the mseiled nucleic acid molecule of interest and the GST-encoding nucleic acid sequence. Similarly, the pGES-3T plasmid contains a factor Xa site. By treating the fusion protein with the approprate enzyme, and then separating the GST portion from the P450 monooxygenase of the invention using glutathione agarose (to which the GST specifically binds), the P45() monooxygenase of the invention can be purfied.<br><br>
Yet another method to obtain a punfied polypeptide of the invention exhibiting a P450 monooxygenase activity is to use a binding agent that specifically binds to such a polypeptide. Accordingly, the invention provides a binding agent that specifically binds to a P450 monooxygenase of the invention. This binding agent of the invention may be a chemical compound (^'..t,'., a protein), a metal ion, or a small molecule.<br>
in particular embodiments, the binding agent is an antibody. The term "antibody" encompasses, without limitation, polyclonal antibody, monoclonal antibody, antibody fragments {e.i^.. Fab, Fv, or Fab' fragments), single chain antibody, chimerc antibody, bi-specific antibody, antibody of any isotype (r,i,'., IgG, IgA, and IgE), and antibody from any specifies (r.i,'., rabbit, mouse, and human).<br>
In one non-limiting example, the binding agent of the invention is a polyclonal antibody. In another non-limiting example, the binding agent of the invention is a monoclonal antibody. Methods for making both monoclonal and polyclonal antibodies are well known {sc(\ c.t^.. Current Protocols in Immunology, ed. John E. Coligan, John Wiley &amp; Sons, Inc. 1993; Current Protocols in Molecular Biology, cds, Ausubel et ai. John Wiley &amp; Sons, Inc. 2000).<br>
The polypeptides descrbed herein exhibiting an enzymatic activity of a P450 monooxygenase that rcgioselectivcly oxidizes avermectin to 4"-keto-avermectin belong to a family of novel P450 monooxygenases. Accordingly, the invention also provides a family of P450 monooxygenase polypeptides, wherein each member of the family rcgioselectivcly oxidizes avermectin to 4"-keto-avemiectin. In some embodiments, each member of the family comprses or consists of an amino acid sequence that is at least 50% identical to SEQ ID NO:2, SEQ ID NO:4, SEQ ID NO:6, SEQ ID NO:8, SEQ ID NO:I0, SEQ ID NO: 12, SEQ ID NO: 14. SEQ ID NO: 16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95. In particular embodiments, each member of the family is encoded by a nucleic acid molecule comprsing or consisting of a nucleic acid sequence that is at least 66%' identical to SEQ ID NO: 1, SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQIDNO:I3, SEQIDN0:15, SEQIDNO:17, SEQIDN0:19, SEQIDNO:2I,SEQID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29. SEQ ID NO:31, SEQ ID NO:33, or SEQ ID NO:94.<br><br>
The present invention, which provides an entire family of P45() monooxygenases, each member of which is able to regiosclectively oxidi/e avermcclin to 4'-keto-avenTicctin, allowed for the generation of an improved P450 monooxygcnase, which may not be naturally occurrnu, but which reuioselectivelv oxidi/es a\crmcctin to 4"-keto-avermectin with efficiency and with reduced undesirable side product.  For instance, one of the members of the P4;S() monooxygcnasc family o\' the invention, P45()i.;„i;,i en/yme catalyzes a further oxidation that is not desirable, since the formation of 3"-0-demcthyl-4'-kelo-avcrmectin has been detected in the reaction by Streptoniyces tnbcrcidicus strain R-922 and by Streptomyces lividans containing the emal gene. The formation of 3'-0-dcmethyl-4"-keto-avermectin is brought about by the oxidation of the 3"-0-methyl group, whereby the hydrolylically labile 3"-0-hydroxymethyl group is formed which hydroly/es to form formaldehyde and the 3'-hydroxyl group.<br>
By providing a family of polypeptides exhibiting an cn/ymatic activitiy of P45() monooxygenases that regioselcclively oxidize avermcctin to 4"-kcto-avenncctin (see, e.^,. Table 3 below), individual members of the family can be subjected to family gene shuffling efforts in order to produce new hybrd genes encodmg optimized P450 monooxygenases of the invention. In one non-limiting example, a portion of the C77ial gene encoding the O2 binding site of the P450Emai protein can be swapped with the portion of the etna! gene encoding the O2 binding site of the P450tm:i: protein. Such a chimerc emal/2 protein is within defmition of a P45() monooxvsenase of the invention.<br>
Site-directed mutagenesis or directed evolution technologies may also be employed to generate dervatives of the emal gene that encode enzymes with improved properties, including higher overall activity and/or reduced side product formation. One method for derving such a mutant is to mutate the Streptomyces strain itself, in a manner similar to the UV mutation of Streptomyces tubercidicus strain R-922 descrbed below.<br>
Additional dcnvatives mav be made bv makmsz conservative or non-conservative changes to the amino acid sequence of a P450 monooxygenase. Conservative and non-conservative amino acid substitutions are well known {see, e.g., Stryer, Biochemistry, 3^"* Ed., W.H. Freeman and Co.. NY 1988). Similarly, tnmcations of a P450 monooxygenase of the invention may be generated by truncating the protein at its N-terminus (e.g., see the emalA<br><br>
gene descrbed below), at its C-terminus, or truncating (i.e.. removing amino acid residues) from the middle of the protein.<br>
Such a mutant, dervative, or truncated P45() monooxygcnasc is a P450 monooxygenase of the invention as long as the mutant, dervative, or truncated P450 monooxygcnasc is able to rcgioselectively oxidize avermectin to 4"-kcto-avermcctin.<br>
In another aspect, the invention provides a cell genetically engineered to comprse a nucleic acid molecule encoding a polypeptide which exhibits an enzymatic activity of a P450 monooxygenase that rcgioselectively oxidizes avermectin to 4'-keto-avermectin. By 'genetically engineered' is meant that the nucleic acid molecule is exogenous to the cell into which it is introduced. Introduction of the exogenous nucleic acid molecule into the genetically engineered cell may be accomplished by any means, including, wiihout limitation, transfection, transduction, and transformation.<br>
In certain embodiments, the nucleic acid molecule is positioned for expression in the genetically engineered cell. By "positioned for expression' is meant that the exogenous nucleic acid molecule encoding the polypeptide is linked to a regulatory sequence in such a way as to permit expression of the nucleic acid molecule when introduced into a cell.  By "regulatory sequence" is meant nucleic acid sequences, such as initiation signals, polyadenylation (polyA) signals, promoters, and enhancers, which control expression of protein coding sequences with which they are operably linked. By "expression" of a nucleic acid molecule encoding a protein or polypeptide fragment is meant expression of that nucleic acid molecule as protein and/or mRNA.<br>
A genetically engineered cell of the invention may be a prokayolic cell {e.g., £. coli) or a eukaryotic cell {e.g., Saccharomyces cerevisiae or mammalian cell {e.g., HeLa)). According to some embodiments of the invention, the genetically engineered cell is a cell wherein the wild-type {i.e., not genetically engineered) cell does not naturally contain the inserted nucleic acid molecule and does not naturally express the protein encoded by the inserted nucleic acid molecule. Accordingly, the cell may be a genetically engineered Streptomyces strain, such as a Streptomyces lividans or a Streptomyces avermitilis strain. Alternatively, the cell may be a genetically engineered Pseudomonas strain, such as a Pseiidomonas putida strain or a Pseudomonas tluorescens strain. In another alternative, the cell may be a genetically engineered Escherchia coli strain.<br><br>
Note that in some types of cells genetically engineered to comprse a nucleic acid molecule encoding a polypeptide which exhibits an enzymatic activity of a P450 monooxygcnase that regioselectivcly oxidizes avcrmcctin to 4'-kclo-avcrmcclin, the actual genetically engineered cell, itself, may not be able to convert avermectin into 4"-keto-avermectin. Rather, the P450 monooxygcnase heterogously expressed by such a genetically engineered cell may be purfied from that cell, where the purfied P45() monooxygcnase of the invention can be used to regioselectivcly oxidize avermectin lo 4"-kcto-avermectin. Thus, the genetically engineered cell of the invention need not, itself, be able to regioselectivcly convert avennectin to 4"-keto-avcrmcctin; rather, the genetically engineered cell of the invention need only comprse a nucleic acid molecule encoding a polypeptide which exhibits an enzymatic activity of a P450 monooxygcnase that regioselectivcly oxidizes avermectin to 4'kcto-avermeclin, regardless of whether the polypeptide is active inside that cell.<br>
In addition, a cell {e.}^., E. coli) geneticially engineered to comprse a nucleic acid molecule encoding a polypeptide of the invention which exhibits an enzymatic activity of a P450 monooxygcnase may not be able to regioselectivcly oxidize avermectin to 4'-keto-avcrmectin, although the P450 monooxygcnase purfied from the genetically engineered cell is able to regioselectivcly oxidize avermectin to 4"-keto-avermectin.  However, if the same cell were genetically engineered to comprse a polypeptide of the invention which exhibits an enzymatic activity of a P450 monooxygcnase, a ferredoxin of the invention, and/or a ferredoxin reductase of the invention, then the P450 monooxygcnase together wiih the ferredoxin and the ferredoxin reductase, all purfied from that cell, and in the presence of a reducing agent {e,g,, NADH or NADPH), would be able to regioselectivcly oxidize avermectin lo 4"-keto-avermectin. Furthermore the genetically engineered cell comprsing a P450 monooxygcnase of the invention, a ferredoxin of the invention, and a ferredoxin reductase of the invention, itselfrnight be able to carry out this oxidation.<br>
Moreover, in a non-limiting example where a cell {e.g-, E. coli) is genetically engniecred lo express P450 monooxygcnase, a ferredoxin, and a ferredoxin reductase proteins of the invention, all three of these proteins, when purfied from the genetically engineered E. coli. are together and in the presence of a reducing agent (e.g., NADH or NADPH) would be active and able to regioselectivcly oxidize avermectin to 4"-keto-avermectin, and so are useful in a method for making emamectin.<br><br>
In accordance with the present invention, the following matera! has been deposited with the Agrcultural Research Service, Patent Culture Collection (NRRL), 1815 North University Street, Peora, Illinois 61604, under the terms of the Budapest Treaty on the International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure: (1) Strcptomyces lividans ZX7 {cmal/fd2.^3'l\it\\\) NRRL Designation No. B-30478; and (2) rsciuhmumas piaiila NRRL 8-4067 containing plasmid pRK290-w/^////^/2:?-?, NRRL Designation No.B-30479<br>
In identifying the novel family of polypeptides exhibiting an enzymatic activity of P450 monooxygenascs that regioseleclively oxidize avermectin to 4'-keto-avcrmectin, novel ferrcdoxins and novel ferredoxin reductases were also identified in the same strains of bactera m which the P430 monooxygenascs were found.  Accordingly, in a further aspect, the invention provides a purfied nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a ferredoxin, wherein the nucleic acid molecule is isolated from a Streptomyces strain comprsing a polypeptide that regioseleclively oxidizes avermectin to 4'-keto-avermectin. Similarly, the invention provides a purfied nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a ferredoxin reductase, wherein the nucleic acid molecule is isolated from a Streptomyces strain comprsing a polypeptide that rcgioselectively oxidizes avermectin to 4'"-keto-avermectin. The invention also provides a purfied protein that exhibits an enzymatic activity of a ferredoxin, as well as a purfied protein that exhibits an enzymatic activity of a ferredoxin reductase, wherein the ferredoxin protein and the ferredoxin reductase protein are isolated from a Streptomyces strain comprsing a polypeptide that resiosclectivelv oxidizes avermectin to 4"-keto-avermectin.<br>
A useful nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a ferredoxin comprses or consists essentially of a nucleic acid sequence that is at least 81% identical to SEQ ID NO:35 or SEQ ID NO:37. Alternatively, the nucleic acid molecule comprses or consists essentially of a nucleic acid sequence that is at least 85%, or at least 90%, or at least 95%, or at least 99% identical to SEQ ID NO:35 or SEQ ID NO:37. The nucleic acid molecule comprsing a nucleotide sequence encoding a polypeptide that exhibits an enzymatic activity of a ferredoxin<br><br>
may comprse or consist essentially of the nucleic acid sequence of SEQ ID NO:35 or SEQ ID NO:37.<br>
The protein of the invention exhibiting a fcrrcdoxin activity may comprse or consist essentially of an amino acid sequence that is at least 80% identical to SEQ ID NO:36 or SEQ ID NO:38.  In some embodiments, the nucleic acid molecule comprses or consists essentially an amino acid sequence that is at least 85^^', or at least 90^;?', or at least 95%, or at least 99%^ identical to SEQ ID NO:36 or SEQ ID NO:38. The ferredoxin of the invention may comprse or consist essentially of the amino acid sequence of SEQ ID NO:36 or SEQ ID NO:38.<br>
A useful nucleic acid molecule comprsing a nucleotide sequence encoding a protein of the invention exhibiting a ferredoxin reductase comprses or consists essentially of the nucleic acid sequence that is at least 857f', or at least 9()9f. or at least 95^'(\ or at least 99%' identical to SEQ ID NO:98, SEQ ID NO: 100, SEQ ID NO: 102, or SEQ ID NO: 104.  In a particular embodiment of the invention, the nucleic acid molecule encoding a ferredoxin reductase of the invention may comprse or consist essentially of the amino acid sequence of SEQ ID NO:98, SEQ ID NO: 100, SEQ ID NO: 102, or SEQ ID NO: 104.<br>
The ferredoxin reductase of the invention may comprse or consist essentially of the amino acid sequence that is at least 85%, or at least 90%, or at least 95%\ or at least 99% identical to SEQ ID NO:99, SEQ ID NO: 101, SEQ ID NO: 103, or SEQ ID NO: 105. In a particular embodiment of the invention, the ferredoxin reductase of the invention may comprse or consist essentially of the amino acid sequence of SEQ ID NO:99, SEQ ID NO: 101, SEQ ID NO: 103, or SEQ ID NO: 105.<br>
Methods for purfying ferredoxin and ferredoxin reductase proteins and nucleic acid molecules encoding such ferredoxin and ferredoxin reductase proteins arc known in the art and arc the same as those descrbed above for purfying P450 monooxygenases of the invention and nucleic acid molecules encoding P450 monooxygenases of the invention.<br>
In one non-limiting example to obtain a puntied P450 monooxygenasc of the invention with a purfied ferredoxin, a 5. lividans strain (or F. puiida strain, or any other ceil in which the P450 monooxygenase of the invention does not naturally occur) may be genetically engineered to contain a first nucleic acid molecule encoding a P450 monooxygenase of the invention and a second nucleic acid molecule encoding a ferredoxin protein, where both the first and second nucleic acid molecules are positioned for expression in the genetically<br><br>
engineered cell. The first imd the second nucleic acid molecules can be on separate plasmids, t)r can be on the same plasmid. Thus, the same engineered cell or strain will produce both the P45() monooxygenase of the invention and the ferredoxin protein of the invention.<br>
In a further non-limiting example to obtain a purfied P45() monooxygenase of the mvention with a purfied ferredoxin and with a purfied ferredoxin reductase of the invention, a .v. Hviclcms strain (or P. putida strain, or any other cell in which the P45() monooxygenase of the invention does not naturally occur) may be genetically engineered to contain a first nucleic acid molecule encoding a P450 monooxygenase of the invention and a second nucleic acid molecule encoding a ferredoxin protein of the invention and a third nucleic acid molecule encoding a ferredoxin reductase protein of the invention, where all the first and second and third nucleic acid molecules are positioned for expression in the genetically engineered cell. The first and the second and the third nucleic acid molecules may be provided on separate plasmids, or on the same plasmid. Thus, the same engineered cell or strain will produce all the P450 monooxygenase of the invention and the ferredoxin and the ferredoxin reductase proteins of the invention.<br>
As descrbed above for the P450 monooxygenases of the invention, the ferredoxin protein and/or the ferredoxin reductase protein may further comprse a tag. Moreover, the invention contemplates binding agents {e.g., antibodies) that specifically bind to the ferredoxin protein, and binding agents that specifically bind to the ferredoxin reductase proteins of the invention. Methods for generating tagged ferredoxin protein, tagged ferredoxin reductase protein, and binding agents {e.g., antibodies) that specifically bind to ferredoxin or ferredoxin reductase are the same as those as descrbed above for generating lagged P450 monooxygenases of the invention and generating binding agents that specifically bind P450 monooxygenases of the invention.<br>
The invention also provides a method for making emamectin. In this method, a P450 monooxysenase that resioseleclivelv oxidizes avermectin to 4"-keto-avermectin is added to a reaction mixture containing avermectin. The reaction mixture is then incubated under conditions that allow the P450 monooxygenase to regioselectively oxidize avermectin to 4'-keto-avermectin. The reaction mixture may further comprse a ferredoxin, such as a fenedoxin of the present invention.  In particular embodiments, the reaction mixture funhcr<br><br>
comprses a forrcdoxin reductase such as a ferrcdoxin of the present invention. The reaction mixture may fuilher comprse a reducing agent, such as NADH or NADPH.<br>
Additionally, the invention provides a method for making 4'-kelo-avcrmectin. The method comprses addmg a P45() monooxygenasc that regioselectively oxidi/.es avermectin to 4"-kcto-avcrmectin to a reaction mixture comprsing avermectin and incubating the reaction mixture under conditions that allow the f*4S() monooxygenasc to regioselectively oxidize avermectin to 4'-kcto-avermectin.  In some embodiments, the reaction mixture further comprses a ferredoxin, such as a fciTcdoxin of the present invention. The reaction mixture may also further comprse a fen'edoxin reductase such as a ferredoxin of the present invention, hi pailicular embodiments, the reaction mixture further comprses a reducing agent, such as NADHorNADPH.<br>
The invention also provides a formulation for making emamectin comprsing a P45() monooxygenasc that regioselectively oxidi/es avermectin to 4'-keto-avermcctin. In some embodiments, the formulation further comprses a ferrcdoxin, such as a ferredoxin of the present invention. In particular embodiments, the ferredoxin is isolated from the same species of cell or strain from which the P450 monooxygenasc was isolated or derved. The formulation may further comprse a ferredoxin reductase , such as a ferrcdoxin reductase of the present invention. In particular embodiments, the ferredoxin reductase is isolated from the same species of cell or strain from which the P450 monooxygenasc was isolated or derved. . In some embodiments, the formulation further comprses a reducing agent, such as NADH or NADPH.<br>
In addition, the invention provides a formulation for making 4"-keto-avenTiectin comprsing a P450 monooxygenasc that regioselectively oxidizes avermectin to 4*'-kcto-avermectin. In some embodiments, the formulation further comprses a ferredoxin, such as a ferredoxin of the present invention. In particular embodiments, the ferredoxin is isolated from the same species of cell or strain from which the P450 monooxygenase was isolated or derved. In some embodiments, the formulation further comprses a ferredoxin reductase, such as a ferredoxin reductase of the present invention. In particular embodiments, the ferredoxin reductase is isolated from the same species of cell or strain from which the P450 monooxygenase was isolated or derved. The formulation may further comprse a reducing agent, such as NADH or NADPH.<br><br>
The following examples are intended to further illustrate eertain preferred embodiments of the invention and are not limitinu in nature.<br>
[':XAMPLE1<br>
Optimized Growth Conditions \'o\- Sircptoifixces tuhercidicus Strain R-922<br>
In one non-limiting example the fermentation conditions needed to provide a steady supply of cells of Streptomyces tuhercidicus strain R-922 highly capable of regioseleclively oxidizing avermectin to 4'-keto-avermectin were optimized.<br>
First, the following solutions were made.  For ISP-2 agar, 4 g of yeast extract (commercially available from Oxoid Ltd, Basingstoke, UK), 4 g of D(+)-glucose, 10 g of bacto malt extract (Difco No. 0186-17-7 (Difco products commercially available from, e.f^., Voigt Global Distrbution, Kansas City, MO)), and 20 g of agar (Difco No. 01404)1) were dissolved in one liter of demineralized water, and the pFI is adjusted to 7.0. The solution was sterlized at 12rC for 20 min., cooled down, and kept al 55"C for the time needed for the immediate preparation of the agar plates.<br>
For PUG medium, 10 g of peptone (Sigma 0521; commercially available from Sigma Chemical Co., St. Louis, MO), 10 g of yeast extract (commercially available from Difco), 10 g of D-(+)-glucose, 2 g of NaCl. 0.15 g of MgSOa x 7 H2O, 1.3 g of NaH^POj x FLO, and 4.4 g of K:HP04 were dissolved in 1 liter of demineralized water, and the pH was adjusted to 7.0.<br>
Streptomyces tuhercidicus strain R-922 was grown in a Petr dish on ISP-2 agar at 28°C. This culture was used to inoculate four 500 ml shaker flasks with a baffle, each containing 100 ml PHG medium. These pre-cultures were grown on an orbital shaker at 120 rpm at 28'^C for 72 hours and then used to inoculate a 10-liter fermenter equipped with a mechanical stirrer and containing 8 liters of PHG medium. This main culture was grown at 28'C with stimne at 500 rpm and with aeration of L75 vvm (14 I/min.) and a pressure of 0.7 bar. At the end of the exponential growth, after about 20 hours, the cells were harvested by centrfugation. The yield of wet cells was 70-80 s/1 culture.<br>
EXAMPLE n<br><br>
Whole Cell Biocalalvsis Assav<br>
	,1	,.*.<br>
As determined in accordance with the present invention, the following whole cell hiocatalysis assay was employed to determine thai the activity from Streptoniyccs cells capable of regioselectivcly oxidi/mg avermectin to 4"-keto-avermectm is cataly/ed by a P'.SO monooxygenase.<br>
Streptomyces tuhercidicus .strain R-922 was grown in PUG medium, and Sireptomyces tuhcrcidicns strain 1-1529 was grown in M-I7 or PUG medium.  PHG medium contains 10 g/1 Peptone (Sigma, 0.521), 10 g/1 Yeast Extract (Difco, 0127-17-9), 10 g/l D-G!ucosc, 2 g/1 NaCKO.ISg/lMgSOjxTH.O, 1.3 g/1 NaH.POa x I Il^O, and 4.4 g/1 K.HPOj at pH 7.0. M-17 medium contains 10 g/1 glycerol, 20 g/1 Dextrn while, 10 g/i Soytonc (Dil'co 0437-17), 3 g/1 Yeast Extract (Difco 0127-17-9), 2 g/1 (NH4):S04, and 2 g/1 CaCO^ at pH 7.0<br>
To grow the cells, an 1SP2 agar plate (not older than 1-2 weeks) was inoculated and incubated for 3-7 days until good growth was achieved. Next, an overgrown agar piece was transferred (with an inoculation loop) to a 250ml Erlenmeycr tlask with 1 baffle containing 50 ml PHG medium. This pre-culturc is incubated at 28"C and 120 rpm for 2-3 days. Next, 5 ml of the pre-culturc were transferred to a 500 ml Erlenmeycr flask with 1 bafHe containing 100 ml PHG medium. The main culture was incubated at 28'C and 120 rpm for 2 days. Next, the culture was centrfuged for 10 min. at 8000 rpm on a Beckman Rotor J A-14. The cells were next washed once with 50 mM potassium phosphate buffer. pH 7.0.<br>
To perform the whole cell biocatalysis assay, 500 mg wet cells were placed into a 25 ml Erlenmeycr flask, to which were added 10 ml of 50 mM potassium phosphate buffer, pH 7.0. The cells were stirred with a magnetic stir bar to distrbute the cells.  Next, 15 \i\ of a solution of avermectin Bla in isopropanol (30 mg/ml) were added, and the mixture shaken on an orbital shaker at 160 rpm and 28'C. Strain R-922 was reacted for 2 hours, and strain 1-1529 was reacted for 30 hours.<br>
To work up the cultures in the whole cell biocatalysis assay, 10 ml methyl-t-butyl-ether was added to an Erlenmeycr flask containing the resting cells and the entire cell mixture was transferred to a 30 ml-centrfuge tube, shaken vigorously, and then centrfuged at 16000 rpm for 10 min. The ether phase was pipetted into a 50 ml pear tlask, and evaporated in vacuo by means of a rotary evaporator (
 <br>
;ind transferred to an HPLC-sample vial. The conversion of avermcclin B la to 4"-hydroxy-avermcclin Bla and 4'-keto-avcnTiectin B la (also called 4'-oxo-avenncc(in Bla) and the formation of a side product from the biocalalysis reacli(^n could he observed by HI^LC analysis using UPLC protocol 1.<br>
For HPLC protocol I, the followiniz parameters were used:<br>
Hardware<br>
Pump:	L-6250 Merck-Hitachi<br>
Autosampler:	AS-200()A Merck-Hitachi<br>
Interface Module:	D-6000 Mcrck-JIitachi<br>
dianncl 1-Dclector:	L-7430A UV-Diode Array Merck-Hitachi<br>
Column Oven:	none<br>
Column:	70mm x 4mm<br>
Adsorbent:	KromasiI 100A-3.5|.i-C 18<br>
Gradient Mode:	Low<br>
Pressure Limit:	5-300har<br>
Column Temperature	ambient (-20'^C)<br>
Solvent A:	acetonitrle<br>
Solvent B:	water<br>
Flow:	1.5 ml/min<br>
Detection:	243 nm<br>
Pump Table:	0.0 min	75% A      25% B<br>
linear gradient    7.0 min	100% A        0%- B<br>
9.0 min	100% A       0%'B<br>
jump	9A min	75% A      25%' B<br>
12.0 min	75% A      25%&gt; B<br>
Stop time:	12 min<br>
Sampling Perod:	every 200 msec<br>
Retention tmie table:	time	References<br>
2A2 min     4'-hydroxy-avermectin Bla<br><br>
3.27 min     avermcclin Bla<br>
3.77 min     3"-0-dcmcthyl-4"-kclo-avcrmcctin Bla<br>
4.83 min     4"-kcto-avcrmcclin Bla<br>
FXAMPLE fll Biotransrormation With Ccll-Frcc Extract From Streptomyccs Strain R-922<br>
To prepare an active ccll-frcc extract from Streptomyces inhercidicus strain R-922 capable oCrcgiosclcctive oxidation of avermcctin to 4"-keto-avermectin, the following solutions were made, stored at 4"C, and kept on ice when used.<br><br>
Six grams of wet cells from Streptomyces strain R-922 were washed in PP-buffer and then resuspcndcd in 35 ml disruption buffer and disrupted in a French press at 4'C. The resulting suspension was centrfuged for I hour at 35000 x g. The supernatant of the cell free extract was collected. One \i\ substrate was added to 499|.il of cleared cell free extract and incubated at 30°C for 1 hour. Then, 1 ml methyl-t-butyl ether was added to the reaction mixture and thoroughly mixed. The mixture was next centrfuged for 2 min. at 14000 rpm, and the methyl-t-butyl ether phase was transferred into a 10 ml flask and evaporated in vacuo<br><br>
by means of a rotary evaporator. The residue was dissolved in 200 |j.l acetonitrlc and transferred into an UPLC-samplc vial.<br>
For HPLC, Ihc HPLC protocol I was used.<br>
When 1 |.il substrate was added to 4W yil of cleared cell free extract and incubated at 30"C, no conversion of aveimectin to 4"-kcto-a\ermectin was observed by HPLC analysis using HPLC protocol 1.<br>
However, the possibility of addition of spinach ferredoxin and spinach ferredoxin reductase and NADPH to the cell free extract to restore the biocatalytic activity was explored (see, generally, D.E. Cane and E.I. Graziani, J. Amer. Chem. Sac. 120:2682, 1998). Accordingly, the following solutions were made:<br><br>
Thus, to 475 |.il of cleared cell free extract the following solutions were added: 10 |il ferredoxin. 10 |,U ferredoxin reductase and 1 \x\ substrate. After the addition of substrate to the cells, the mixture was immediately and thoroughly mixed and aerated. Then, 5 \x\ of NADPH were added and the mixture incubated at SO^'C for 30 min. Then, 1 ml methyl-t-butyl ether was added to the reaction mixture and thoroughly mixed. The mixture was next centrfuged for 2 min. at 14000 rpm, and the methyl-t-butyl ether phase was transferred into a 10 ml flask<br><br>
and cvnporaled in vacuo by means of a rotary evaporator. The residue was dissolved in 200 ^1 acetonitrle and transferred into an UPLC-sample vial, and HPLC analysis performed using UPLC protocol I.<br>
Formation of 4"-kcto-avermectm was observable by HPLC analysis. Thus, addition of spinach ferrcdoxin and spinach fenvdoxin reductase and NADPH to the cell free extract restored (he biocatalytic activity.<br>
Upon injection of a 30 |il sample, a peak appeared at 4.83 min., indicating the presence of 4'-kcto-avermectin Bla. A mass of 870 D could be assigned to this peak by HPLC-mass spectrometry which corresponds to the molecular weight of 4"-kcto-avcrmectin Bla.<br>
Note that when analyzing product formation by HPLC and HPLC-mass spectrometry, in addition to the 4"-keto-avcrmcctin, the corresponding kelohydrate 4"-hydroxy-avcrmectin was also found giving a peak at 2.12 min. This finding indicated that the P450 monooxygenase converts avermectin by hydroxylation to 4"-hydroxy-avermectin, from which 4"-kcto-avermectin is formed by dehydration.  Interestingly, when the spinach fcrredoxin was replaced by ferrcdoxin from the bacterum Clostrdium pastcuraniun or from the red alga Porphyra umbilicalis, the biocatalytic conversion of avermectin to 4"-kelo-avermectin still took place, indicating that the enzyme d(x;s not depend on a specific ferrcdoxin for receiving reduction equivalents.<br>
EXAMPLE IV<br>
Isolation of a Mutant Strcptonn'ces Strain R-922 With Enhanced Activity<br>
To obtain strains of Streptomyccs strain R-922 that have an enhanced ability to rcgioselectively oxidize avermectin to 4"-keto-avermectin, UV mutants were generated. To do this, spores of Sireptomyces strain R-922 were collected and stored in 15% glycerol at -20'^C. This stock solution contained 2x 10 spores.<br>
The spore stock solution was next diluted and transferred to petr plates containing 10ml of sterle water, and the suspension was exposed to UV light in a Stratalinker UV crosslinker 2400 (commercially available from Stratagene, La JoUa, CA). The Stratalinker UV crosslinker uses a 254-nm lisht source and the amount of enerey used to irradiate a sample can be set in the "energy mode."<br><br>
Through expermentation, it was determined that an exposure of 8000 microjoulcs of UV irradiation (254 nm) was required to kill 99.9% of the spores. This level of UV exposure was used in the mutagenesis.<br>
Surviving UV-mutagcni/ed spores were plated, cultured, and transferred to minimal media. Approximately 0.3-0.4% of the viable spores were determined to be auxotrophie, indicating a good level of mutagenesis in the population.<br>
The mutagcnizcd clones were screened for activity in the whole cell biocatalysis assay descrbed in Example 11. As shown in an HPLC chromatogram, one mutant C'R-922 UV mutant") showed a two to three fold increase in an ability to regioselcctively oxidize avermectin to 4'-keto-avermectin as compared to wild-type strain R-922-  Although the gene encoding the P45() monooxygenase responsible for the regioselcctively oxidation activity, cmal, is not mutated in the R-922 UV mutant, this mutant nonetheless provides an excellent source for a cell-free extract containing cmal protein.<br>
EXAMPLE V Isolation of the P450 Monooxygenase from Streptomvccs Strain R-922<br>
To enrch the P450 enzyme, 35 ml of active cell free extract were Hllercd through a 45<br>
f,im filter and fractionated by anion exchange chromatography. Anion exchange<br>
chromatography conditions were as follows:<br>
FPLC instrument:	Akta prme (from Pharmacia Biotech)<br>
FPLC-column:    HiTrap ' Q (5 ml) stacked onto Resource® Q (6 ml) (from Pharmacia<br>
Biotech)<br>
eluents	buffer A: 25 mM Trs/HCI (pH 7.5)<br>
buffer B: 25 mM Trs/HCI (pH 7.5) containing 1 M KCl<br>
temperature	eluent bottles and fractions in ice bath.<br>
How	3 ml/min<br>
detection	UV 280nm<br>
Pump table:	0.0 min       100% A	0% B<br>
linear gradient tol.O min	90% A       10% B<br>
5.0 min	90% A       10% B<br><br>
linear gradient /o3().() min       50% A        507r. B<br>
linear gradient toAO.i) min	0% A       100% B<br>
50.0min	0^7r A       100% B<br>
En/ymc activity elulcd with 35^&gt;'-40'/f butTcr B. The active fractions were pooled and concentrated by centrfugal filtration through Biomax''^^ filters with an exclusion limit of 5kD (commercially available from Milliporc Coip., Bedford, MA) at 5000 rpm and then rcdiluted in disruption buffer containing 20% glycerol to a volume of 5 ml containing 3-10 mg/ml protein. This enrched enzyme solution contained at least 25% of the orginal enzyme activity.<br>
The en/ymc was further purfied by size exclusion chromatography. Size exclusion<br>
chromatography conditions were as follows:<br>
FPLC instrument:	Akta prme (from Pharmacia Biotech)<br>
FPLC-column:	MiLoad 26/60 Superdex® 200 prep grade (from Pharmacia Biotech)<br>
sample:	3-5 ml enrched enzyme solution from the anion chromatography step<br>
sample preparation:       filtered through 45 (im filter<br>
clucnt buffer:	PP-buffer (pH 7.0) + 0.1 M KCl<br>
temperature:	4°C<br>
flow:	2 ml/min<br>
deletion:	UV 280nm<br>
Enzyme activity eluled between 205-235 ml eluent buffer. The active fractions were<br>
*rvi<br>
pooled, concentrated by centrfugal filtration through Biomax '   filters with an exclusion limit of 5 kD (from Millipore) at 5000 rpm, and rediluted in disruption buffer containing 20%-glycerol to form a solution of 0.5-1 ml containing 2-5mg/ml protein. This enrched enzyme solution contained 10% of the orginal enzyme activity. This enzyme preparation, when checked for purty by SDS page. {sec. generally. Laemmli, U.K., Nature 227:680-685, 1970 and Current Protocols in Molecular Biology, supra) and stained with Coomassie blue, showed one dominant protein band with a molecular weight of 45-50 kD, according to reference proteins of known molecular weight.<br>
EXAMPLE VI<br><br>
Attempted Isolation of P450 Monooxygcnasc Genes From Strcptomxci's .Vtrains R-922 and I~I529<br>
Based on results descrbed ahove that sus:ges!cd the enzyme from strain R-922 that is responsible for the regiospeeilic oxidation of avermectin to 4'-keto-avermectin is a P45() monooxygenase. a direct PCR-based approacli to clone P45() monooxygenasc genes from this strain was initiated {set\ generally, Hyun ct ai, J. MicrohioL Biotcchnol. 8(3):295-299, 1998). This approach is based on the fact that all P450 monooxygenasc enzymes contain highly conserved oxygen-binding and heme4^inding domains that arc also conserved at the nucicolide level. PCR prmers were designed to prme to these conserved domains and to amplify the DNA fragment from P45() genes using R-922 or 1-1329 genomic DNA as a template. The PCR prmers used are shown in Table 1.<br><br><br><br>
PCR amplification using any of (lie prmers specific to nucleotide sequences encoding the 02-binding domain with any of the prmers specific to the nucleotide sequences encoding the heme-binding domain and genomic DNA from Strcplomyccs strains R-922 or M529 resulted in the amplification of an approximately 350 bp DNA fragment. This is exactly the size that would be expected from this PCR amplification due to the approximately 350 bp separation in P450 genes of the gene segments encoding the 07-binding and heme-binding sites.<br>
The 350 bp PCR fragments were cloned into the pCR2-. 1-TOPO TA cloning plasmid (commercially available Invitrogcn, Carlsbad, CA) and transformed into E. coli strain TOPIO (Invitrogen, Carlsbad. CA). Approximately 150 individual clones from strains R-922 and I-1529 were sequenced to determine how many unique P450 gene fragments were represented. Analysis of the sequences revealed that they included 8 unique P450 gene fragments from strain R-922 and 7 unique fragments from 1-1529.<br>
Blast analysis (alignment of the deduced amino acid sequences of P450 gene-specific PCR fragments derved from Streptomyccs tubercidicus strain R-922 and Streptomyces strain 1-1529, respectively, and the P450 monooxygenase from S. thennotolenins that is involved in the synthesis of carbomycin (Stol-ORFA) (GenBank Accession No. D30759) by the program Pretty from the University of Wisconsin Package version lOT (Altschul et uL. NiicL Acids Res. 25:3389-3402). demonstrated that all of the unique P450 gene fragments from both the R-922 and 1-1529 strains were denved from P450 iiencs and encoded the reeion between the 02-binding and heme-binding domains.<br>
Next, in order to clone the full-length genes from which the PCR fragments were derved, the DNA fragments cloned by PCR were used as hybrdization probes to gene librares containing genomic DNA from strains R-922 and 1-1529. To do this, genomic DNA from the R-922 and 1-1529 strains was partially digested with Sau3A 1, dephosphorylated with<br><br>
calf intestinal alkaline phosphatase (CAP) and ligated into the cosmid pPl:U215, a modified version of SupcrCos 1 (commercially available from Stratagcne, La Jolla, CA). Ligation products were packaged using the Gigapack HI XL packaging extract and transfectcd into /:. coli XLl Blue MR host cells. Twelve cosmids that strongly hyiuidi/cd to the l^CR-gencrated P450 gene fragments were identified from the R-922 library, from which three unic|ue F^-450 genes were subcloned and sequenced. The hybrdizations were performed at high stnngency conditions according to the protocol of Church and Gilbert (Church and Gilbert, Froc. Natl. Acad. Sci. USA 81:1991-1995, 1984). In bref, these high strngency conditions include Hybrd Buffer containing 500 mMNa-phosphate, 1 mM EDTA,7%SDS, 1% BSA; Wash Buffer 1 containing 40 mM Na-phosphate, 1 mM EDTA, 5% SDS, 0.5% BSA; and Wash Buffer 2 containing 40 mM Na-phosphate, 1 mM LDTA, 1% SDS (Note that other high strngency hybrdizations conditions are descrbed, for example, in Current Protocols in Molecular Biolouv, supra.) Nineteen strongly hybrdizing cosmids were identified from the I-1529 library, and from these, four unicjue P-450 genes were subcloned and sequenced.<br>
In yet a further approach to isolate diverse P450 monooxygenasc genes from strains R-922 and 1-1529, a known P450 gene from another bacterum was used as a hybrdization probe to identify cosmid clones containing homologous P450 genes from strains R-922 and I-1529. The epoF P450 gene from Soran^iiim celhilosum strain So ce90 that is involved in the synthesis of epothilones (Molnar ct al., Chem Biol. 7(2):97-109, 2000) was used as a probe in this effort. Using the epoF P450 gene probe, one cosmid was identified from strain R-922 (clone LC), and threewere identified from strain 1-1529 (clones LA, LB, and EA).  In each case, the homologous gene fragment was subcloned and sequenced, and found to code for P450 monooxygenasc enzymes.<br>
However, a comparson of the 17 polypeptide sequences identified in Example VII (below) failed to match any of these cloned genes. Two of the polypeptide sequences (namely. LVKDDPALLPR and AVHELMR) mapped to the region between the O: and heme binding domains, and so these should have identified any of the partial gene fragments derved by the PCR approach. Thus, the standard approaches based on the known PCR technique of Hyun et al., supra, and using known P450 genes as hybrdization probes failed to identify the gene that encodes the specific P450 monooxygenasc responsible for the regioselective<br><br>
iixiclation ofavcrmcctin. Accordingly, it was determined that additional cxpenmentation was rec]uired to isolate the gene encoding the P450 monooxygcnasc of the invention.<br>
EXAMPLE VII Partial Seciucncing of the P450 Monooxyeenase from Strcptomxccs Strain R-922<br>
Partial amino acid sequencing of the P450 monooxygenase from Streptomyces strain R-922 was carred out by the Fredrch Micscher Institute, Basel Switzerland. The protein of the dominant band on the SDS page was tryptically digested and the formed peptides separated and sequenced by mass spectrometry and Edman degradation {sec, generally, Zerbc-Burkhardl ct a}., J. Biol. Clicni. 273:6508, 1998). The sequence of the following 17 peptides were found:<br>
Sequence	Sequence I.D. No.<br>
HPGEPNVMDPALrTDPFTGYGALR    (SEQ ID NO:61)<br>
FVNNPASPSLNYAPEDNPLTR	(SEQ ID NO:62)<br>
LLTHYPDISLGIAPEHLER	(SEQ ID NO:63)<br>
VYLLGSILNYDAPDHTR	(SEQ ID NO:64)<br>
TWGADLISMDPDR	(SEQ ID NO:65)<br>
EALTDDLLSELIR	(SEQ ID NO:66)<br>
FMDDSPVWLVTR	(SEQ ID NO:67)<br>
LMEMLGLPEHLR	(SEQ ID NO:68)<br>
VEQIADALLAR	(SEQ ID NO:69)<br>
LVKDDPALLPR	(SEQ ID NO:70)<br>
DDPALLPR	(SEQIDNO:71)<br>
TPLPGNWR	(SEQ ID NO:72)<br>
LNSLPVR	(SEQ ED NO:73)<br>
ITDLRPR	(SEQ ID NO:74)<br>
EQGPVVR	(SEQ ID NO:75)<br>
AVHELMR	(SEQ ID NO:76)<br><br>
AFTAR	(SEQIDNO:77)<br>
FUEVR	(SEQ ID NO:78)<br>
Alignment of these peptides to a selection otactinomycete P450 monooxygenase sec|uenccs indicated that all the peptides were fragments of a single P450 mono-oxygenasc.<br>
EXAMPLE Vlll<br>
Cloning the P450 Monooxy^^cnase Gene from Strain R-922 that Encodes the Enzyme<br>
Responsible for the Oxidation of Avermectin to 4'-Kcto-Avermectin<br>
PCR prmers were designed by reverse translation from the amino acid sequences of several of the peptides derved from the P450 en/yme of strain R-922 (sec Example VII and Table 2 below).  Each of five forward prmers (2aF, 2bF, 3F, IF, and 7F) was paired with one reverse prmer (5R) in PCR reactions with R-922 genomic DNA as a template. In each reaction, a DNA fragment of the expected si/c was produced.<br><br><br>
**  Expected size of PCR product when the prmer is when paired with prmer 5R<br>
The 580 and 600 bp PCR fragments generated by using prmers (2bF and 5R) and (2aF and SR), respectively, were cloned into the pCR-Blunt II-TOPO cloning plasmid (commercially available tVom Invilrogcn, Carlsbad, CA) and transformed into E. coli strain TOPIC (Invitrogen, Carlsbad, CA). The inserted DNA fragments were then sequenced, Examination of the sequences revealed that the 600 and 580 bp fragments were identical in the 580 bp of sequence that they have in common. Also, there was a perfect match between ihe deduced amino acid sequence (SEQ ID NO:2) derved from the nucleotide sequence of the 600 bp and 580 bp fragments and the amino acid sequences of peptides isolated from the purfied P450Hm:ii cn/yme that aligned in this region of the isolated gene. This result strongly suggested thai the gene fragments isolated in these clones arc derved from the gene that encodes the P450uniai enzyme that is responsible for the oxidation of avermectin to 4"-kcto-avermectin.<br>
The 600 bp PCR fragment produced using pnmers 2aF (SEQ ID No:80) and 5R (SEQ ID No:90) was used as a hybrdization probe to a cosmid library of genomic DNA isolated from strain R-922 (cosmid library descrbed in Example VI). Two cosmids named pPEH249 and pPEH250 were identified that hybrdized strongly with the probe. The portion of each cosmid encoding the P450 enzyme was sequenced and the sequences were found to be identical between the two cosmids. The complete coding sequence of the emal gene was identified (SEQ ID NO:!). The amino acid sequence of all polypeptide fragments from P450Eniai rnatched perfectly with the deduced amino acid sequence from the emal gene. Comparson of the deduced amino acid sequence of the protein encoded by the emal gene using BLASTP (Altschul et ai. supra) determined that the closest match in the databases is to a P450 monooxygenase from S. thermotolerans that has a role in the biosynthesis of carbomycin (Arsawa et ai, Biosci, Biotech, Biochem. 59(4):582-588, 1995) and whose identity with emal is only 497r (Identities = 202/409 (49%). Positives = 271/409 (65%), Gaps = 2/409 {()%)). In the Blast analysis, the following settings were employed:<br><br>
BLASTP   2.0.10<br>
Lambda	K	II<br>
0.3 22	0.140	0.428<br>
(Japped<br>
Lambda	K	H<br>
0.270	0.0470	0-230<br>
Matrx :    HLO.';UM62<br>
(Jap   Penal I ie^;:    Kxistence :    11,    Extenr; i on :    1<br>
Number   of	Hit:s   to   DB:    37500 1765<br>
Number   of	.Socjuences:    1271323<br>
Nuniben"   
Numbei■   of	suecr.'^'&gt;5i f u 1   extension^;:    4G7 38<br>
Number   of	sequences   better   than   10.0:    2211<br>
Number   of	HSP's   better   than   10.0   without   gapping:   628<br>
Number   of	HSP's   successfully gapped   in   prelim   test:   1583<br>
Number   of	HSP's   that   attempted  gapping   in  prelim  test:   43251<br>
Number   of	HSP's   gapped   (non-prelim):    2577<br>
length   of	query:    4 30<br>
length   of	database:   409,691,007<br>
effective	HSF'   length:   55<br>
of feet ive	length   of   query:    37 5<br>
effective	length   of   database:    339,768,242<br>
effective	search   space:    127413090750<br>
cneclivc search space used: 127413090750<br>
A similar comparson of the nucleotide sequences of these two genes demonstrated that they arc 65% identical at the nucleotide level. These results demonstrate that P450i:,„ai is a new enzyme.<br>
EXAMPLE IX Heterologous Expression of the cnml Gene in Streptomxces lividans Strain ZX7<br>
The coding sequence of the emal gene was fused to the thiostrepton-inducible promoter (tipA) (Murakami et aL,J. BacteroL 171:1459-1466, 1989). The tipA promoter was derved from plasmid pSIT151 (Hcrron and Evans. FEMS Microbiology Letters 171:215-221, 1999).<br>
The fusion of the tipA promoter and the emal coding sequence was achieved by first amplifying the emal coding sequence with the following prmers to introduce a Pad cloning site at the 5' end and a Pmel compatible end on the 3' end.<br>
Forward Prmer: The underlined sequence is a Pad recognition sequence; the sequence in bold-face type is the start of the coding sequence oi emal.<br>
5   AGATTAATTAATGTCGGAArTAATGAACTGTC(?GTT 3  (SEQ ID NO:91)<br><br>
Reverse Prmer: The underlined sequence is half of a Pmcl recognition sequence; the bold-face type sequence is the reverse complement of the cnuti translation slop codon followed by the 3' end of the enial coding sequence.<br>
3'-AAACTCACCCCAACCGCACCGGCA(lCGAGTTC-3' (SEQ ID NO:92)<br>
The Pad-digested PCR fragment containing the cmal coding sequence was cloned inl(^ plasmid pTBBKA {see Figure 1) that was restrcted (i.e., digested) with Pad and Pmel, and the ligatcd plasmid transformed into /:'. eolL Four clones were sequenced. Three of the four contained the complete and correct emal coding sequence. The fourth emal gene clone contained a truncated version of the eftuil gene. The full-length enuil gene encodes a protein that begins with the amino acid sequence MSELMNS (SEQ ID NO:93). The tmncated gene encodes a protein that lacks the first 4 amino acids and begins with the second methionine residue. This gene has been named emal A,        The nucleotide and amino acid sequence of emal A are provided as SEQ ID NO:33 and SEQ ID NO:34, respectively. The emal and emal A genes in these plasmids, pTBBKA-emal and pTBBKA-^'wa//\, arc in the correct juxtaposition with the tipA promoter to cause expression of the genes from this promoter.<br>
Plasmid pTBBKA contains a gene from the Streptomyces insertion element ISl 17 that encodes an integrase that cataly/xs site-specific integration of the plasmid into the chromosome of Streptomyces species (Henderson et al., MoL Microbiol. 3:1307-1318, 1989 and Lydiate et ai, Moi Gen. Genet. 203:79-88, 1986). Since plasmid pTBBKA has only an E. coli replication orgin and contains a mobilization site, it can be transferred from E, coli to Streptomyces strains by conjugation where it will not replicate. However, it is able to integrate into the chromosome due to the IS 117 integrase and Streptomyces clones containing chromosomal integrations can be selected by resistance to kanamycin due to the plasmid-borne kanamvcin resistance sene.<br>
The emal coding sequence was also cloned into other plasmids that are either replicative in Streptomyces or, like pTBBKA, integrate into the chromosome upon introduction into a Streptomyces host. For example, emal was cloned into plasmid pEAA. which is similar to plasmid pTBBKA but the Kpnl/Pad fragment containing the tipA promoter was replaced with the ermE gene promoter (Schmitt-John and Engels, Appl<br><br>
Microbiol Biotcchnol. 36(4):493-498, 1992). In addition, pHAA docs not contain the kanamycin resistance gene. The cnial gene was c!t)ned into pEAA as a Pacl/Pmel fragment to create phismid pEAA-cmal in which the cnuil gene is expressed from the constitutive emiE promoter.<br>
Plasmid pTUAl A is a Strcptmnyces-E.coli shuttle plasmid {sec Figure 2) that contains the tipA promoter.   The cfucil gene was also cloned into the I^acl/Pmel site in plasmid pTUAl A to create plasmid \i\\JA-emaL<br>
The emalA gene fragment was also ligated as a Pacl/Pmei fragment into plasmids pTUAl A, and pEAA in the same way as the emu I gene fragment to create plasmids pTUA-cmal A, and pEAA-cmalA, respectively.<br>
The pIliBKA, pTUAl A, and pEAA hased plasmids containing the aual or cnmlA genes were introduced into .V. lividans 7.X1 and in each case transformants were obtained and verfied {S. lividans strains 7.X1 w^iX^MY^A'CmaI or cmatA. ZX7 (\)\\)A-cm(il or-emalA), and ZXl::pEAA-emaI or -cnialA, respectively).<br>
Wild-type Streptomyces lividans strain ZX7 was tested and found to be incapable of the oxidation of avermectin to 4"-keto-avermectin. Transformed S. lividans strains ZX7::pTBBKA-t'/mi/, ZX7::pTBBKA-c'Am///l, ZX7 (pTUA-t-ma/), ZX7 (pTUA-mw//\), ZX7::pEAA-emaL and ZX7::pEAA-r/m///\ were each tested for the ability to oxidize avermectin to 4"-keto-avcrmcctin using resting cells. To do this, the whole cell biocatalysis assay descrbed above (including analysis method) was performed. Note that for the whole cell biocatalysis assay, transformed Streptomyces lividans, like strain R-922, was grown in PHG medium and, again like strain R-922, had a reaction time of 16 hours {i.c\, durng which time the 500 mg transformed Slrcpiomyccs lividans wet cells in 10 ml of 50 mM potassium phosphate buffer, pH 7.0, were shaken at 160 rpm at 28*^0 in the presence of 15 |al of a solution of avermectin in isopropanol (30 mg/ml)).<br>
In the presence of the inducer, thiostrepton (5 ug/ml), the cnial- or 
Table 3<br><br><br>
These results conclusively demonstrate that the P450Hmai enzyme encoded by the emal gene is responsible for the oxidation of avermectin to 4"-keto-avermectin in S. tiibercidicus strain R-922. Furthermore, the data demonstrates that the emal A gene that is 4 amino acids shorter on the N-terminus than the native emal gene also encodes an active P450r:niiii en/.yme.    As can be demonstrated by HPLC analysis, oxidation of avermectin to 4"-keto-avermectin by S.<br><br>
lividans strain ZX7::pTBBKA-ema/ following induction of cmal expression with 0, 0,5, or 5.0 l-ig/ml thiostrcpton. is varable depending upon the amount of thioslreplon used to induce expression of cmal.  Note that S. lividans strains ZXl::pEAA-emaI and ZX7:.pllAA-cfnalA {see Table 3) demonstrated this oxidation activity in the absence of thioslreplon since in these strains the cmal or cmalA genes are expressed from the crmE promoter that does not require mduction.<br>
EXAMPLE X<br>
Isolation of an f^m
Strcptoniyces tiihercidicus strain I-1529 was also found to be active in biocatalysis of avermectin to form the 4'-kcto-avcrmcctin dervative. The cosmid library from strain M529, descrbed in Example VI, was probed at the high strngency conditions of Church and Gilbert (Church and Gilbert, Proc, Natl. Acad. Sci USA 81:1991-1995, 1984) with the 600 bp cmal PCR fragment produced using prmers 2aF (SEQ ID No:80) and 5R (SEQ ID No;90) descrbed previously to identify clones containing the emal homofog from stram 1-1529. Three strongly hybrdizing cosmids were identified. The P450 gene regions in two of the cosmids, pPEH252 and pPEH253, were sequenced and found to be identical. Analysis of the DNA sequence revealed the presence of a gene with high homology to the emal gene of strain R-922. A comparson of the deduced amino acid sequence of Ema2 {i.e., P450Emj:), Emal {i.e., P450Emai). ^nd a P450 monooxygenase from Streptomyces thermotolerans that is mvolved in the biosynthesis of carbomycin (Carb-450) (GenBank Accession No. D30759). demonstrated that all of the unique P450 gene fragments from both the R-922 and 1-1529 strains were derved from P450 genes and encoded the region between the 02-binding and heme-bindins domains.<br>
The gene from Streptomyces tiihercidicus strain I-1529, named ema2, encodes an enzyme with 90% identity at the amino acid level and 90.6% identity at the nucleotide level to the P450Emai enzyme. The nucleotide sequence of the emal gene and the deduced amino acid sequence of P450£:ma2 are provided in SEQ ID NO:3 and SEQ ID NO:4, respectively.<br>
The emal coding sequence was cloned in the same manner as the emal and emal A genes into plasmids pTBBKA, pTUAlA, and pEAA such that the coding sequence was<br><br>
functionally fused to the tipA or enuE'^ promoter in these plasmids. The resulting plasmids, pTBBKA-
Strains ZX7:;TBBKA-^7m/2, ZX7 (pTUA-r/?/^/2), and ZX7::pHAA-^7//^/2 were next tested for the ability to oxidize avermectin to 4'-keto-avermectin. The emal gene was also shown to provide biocatalysis activity, although at a lower level compared to the cnuil gene (sec Table 3).<br>
These results demonstrate that the cma2 gene from S. tiibercidiciis strain M529 also encodes a P45() en/yme (P450u„ia2) capable of oxidizing avennectin to 4"-keto-avermectin.<br>
EXAMPLE XI Characterzation of emal Homolo^s From Other Biocatalysis Strains<br>
Seventeen Strcptomyccs sp. strains, including strains R-922 and 1-1529, were identified that arc capable of catalyzing the rcgiospccific oxidation of the 4"-carbinol of avermectin to a ketone. Next, the isolation and characterzation of the genes encoding the biocatalysis enzyme from all of these strains was accomplished.<br>
To do this, genomic DNA was isolated from the strains and was evaluated by restrction with several restnction endonucleases and Southern hybrdization with the emal gene. A specific restrction endonuclease was identified for each DNA that would generate a single DNA fragment of a defined size to which the emal gene hybrdizes. For each strain, there was only one strongly hybrdizing DNA fragment, thus suggesting that other P450 genes were not detected under the high strngency hybrdization conditions used in these experments. Each DNA was digested with the approprate restnction endonuclease, and the DNA was subjected to agarose gel electrophoresis. DNA in a narrow size range that included the size of theemal7-hybrdizing fragment was excised from the gel. The size selected DNA was ligated into an approprate cloning plasmid and this ligated plasmid was used to transform E. coli. The E. coli clones from each experment were screened by colony hybrdization with the emal gene fragment to identify clones containing the emal -homologous, DNA fragment.<br><br>
The nucleotide sequence of the cloned DNA in each ^'mc/Z-homologous clone was determined and examined for the presence of a gene encoding a P450 enzyme with homology to emal.  In this way, t7;/a/-homologous genes were isolated from 14 of the 15 other active strains. The nucleotide and deduced amino acid sequences of these are referenced in Table 4 as SEQ ID NOS:5-32 and 94-95. The relationship of these enzymes can be shown in the form of a phylogcnclic tree. Such a phylogenetic tree can be generated using the commercially available GCG Wisconsin software program version 1.0 (Madison, WI).<br><br>
EXAMPLE XIJ<br>
Construction of His-taeged emal and emal Homologs to Facilitate Enzyme Purfication<br><br>
In order to purfy the P450v.mai en/ymc and the P45() cn/ymcs encoded by the emal homologs IVom other biocatalysis strains, each of the P45() genes was cloned into the E. coli expression plasmid pET-28b(+) (commercially available from Novagen, Madison, Wl). The pCT-28 plasmids are designed to facilitate His-tag fusions at cither the N-, or C-terminus and to provide strong expression of the genes in /:. coli from the T7 phage promoter. In many cases, the coding secjuence of the emu genes begins with the sequence ATC7r. These genes were amplified by PCR such that the prmers on the 5' end incorporated a Pcil recognition site (5' ATATGT 3') at the 5' terminus. The last four bases of the Pcil site correspond to the ATGT at the beginning of the etna gene coding sequence.<br>
PCR prmers at the 3' end of the genes were designed to remove the translation stop codon at the end of the cma gene coding scciucnce and to add an Xhof recognition site to the 3' terminus. The resulting PCR fragments were restrcted with Pcil and Xhol to generate Pcil ends at the 5' termini and Xhol ends at the 3' termini, thereby facilitating cloning of the fragments into pET-28b(+) previously restrcted with Ncol and Xhol. vSincc Pcil and Ncol ends are compatible, the fragments were cloned into pET-28b(+) in the proper orentation to the T7 promoter and rbosomc binding site in the plasmid to provide expression of the genes.<br>
At the 3' end of each etna gene, the coding sequence was fused in frame at the Xhol site to the His-tag sequence followed by a translation stop codon. This results in the production of an Ema enzyme with six histidine residues added to the C-terminus to facilitate punfication on nickel columns.<br>
In the case oi ema genes in which the ATG translation initiation codon is not followed by a T nucleotide, the etna genes were amplified by PCR using a different strategy for the 5' end. The prmers at the 5' end were designed to incorporate a C immediately preceding the ATG translation initiation codon and the prmers at the 3' end were the same as descrbed above. The PCR fragments that were amplified were restrcted with Xhol to create an Xhol end at the 3'-terminus and the 5' end was left as a blunt end. These fragments were cloned into pET-28b(+) that had been restrcted with Ncol, but the Ncol ends were made blunt-ended by treatment with mung bean exonuclease, and restrcted with Xhol.<br>
In this manner, the ema genes were cloned into pET-28b(+) to create a functional fusion with the T7 promoter and the His-tag at the C-terminus as descrbed previously. All His-tagged ema genes were sequenced to ensure that no errors were introduced by PCR.<br><br>
Large amounts of the P450Rma! ^nd P450r:m;.2 enzymes were isolated and purfied by standard protocols. /:. coli strain BL21 DE3 (commercially available from Invitrogen; Carlsbad, CA) containing the T7 RNA polymerase gene under the control of the inducible tac promoter and the approprate pET-28A7//a plasmid was cultured and the cells were harvested and lysed. 1'he lysates were applied to Ni-NTA columns (commercially available from Qiagen Inc.. Valencia. CV\) and the protein were purfied according to the procedure recommended by the manufacturer.<br>
Purfied His-tagged P450r:m;.i and P450[:nui2 were highly active in in vitro activity assays as evidenced by a high rate of conversion of avermcclin to 4'-keto-avcmectin.<br>
EXAMPLE Xm Expression of cnial in Pseudomonas<br>
The cmal gene constructs were next introduced into P. putida (wildtype P. piitida commercially available from the Amercan Type Culture Collection, Manassas, Virginia; ATCC Nos. 700801 and 17453). The cmal and emcil/fd233 gene fragments were cloned as Pacl/Pmel fragments into the plasmid pUK21 (Vicra and Messing, G^v/e' 100:189-194, 1991). The fragments were cloned into a position located between the tac promoter (Puc) and terminator (T,ac) on pUK2l in the proper orentation for expression from the tac promoter. The Ptac-^"'«^-Ttac and Vx^^^-emal/fdl^S-Ttxc gene fragments were removed from pUK21 as Bglll fragments and these were cloned into the broad host-range, transmissible plasmid, pRK290 (Ditta et aL. Proc. Natl. Acad. Sci. USA 77:7347-7351, 1980) to create plasmids pRK-emal and pRK-cmal/fdZJJ (Figure 3). These plasmids were introduced into P. putida strains ATCC 700801 and ATCC 17453 by conjugal transfer from E. coli hosts by standard methodology (Ditta ct ai. Proc. Natl. Acad. Sci. USA 77:7347-7351, 1980).<br>
P. putida ATCC 700801 and ATCC 17453 containing plasmids pRK-cmal or pRK-emal/fd233 were tested for the ability to catalyze the oxidation of avermectin. The results shown in Table 3 demonstrate that these strains are able to catalyze this reaction.<br>
EXAMPLE XIV<br><br>
Idcntircalion of'Genes Fincodini; Ferrcdoxins That Arc Active With theP450{.,n:i I Monooxygcnase<br>
P45() m()n(X)xygcnases require two electrons lor each hydroxylation reaction catalyzed (Mueller et ai, 'IXventy-Hve years ol" P450,;,„i research: Mechanistic Insights into Oxygenase Catalysis/' Cytochrome P45(). 2'"' lulition, P.R. Ortiz de Montellano (ed.), pp. 83-124; Plenum Press, NY 1995). These electrons are translerred to the P45() monooxygcnasc one at a time by a ferredoxin. The electrons arc ultimately dcfived from NAD(P)H and are passed to the ferredoxin by a ferredoxin reductase. Specific P-450 monooxygenase enzymes have a higher activity when they interact with a specific ferredoxin. In many cases, the gene encoding a ferredoxin that interacts specifically with a given FM5() monooxygenase is located adjacent to the gene encoding the P45() enzyme.<br>
As descrbed above, in addition to the emal gene, four P45() genes from strain R-922 and seven P450 genes from strain 1-1529 {see Example VI) were isolated and sequenced.  In some of these, there was sufficient sequence information about the DNA flanking the P-450 genes to look for the presence of associated ferredoxin genes.  By this approach, two unique ferredoxin genes were identified from each of the two strains. Ferredoxin genes/J229 and fd230 were identified from strain R-922, and/J2Ji and/^/EA were identified from strain I-1529. In addition, a ferredoxin reductase gene was found to reside adjacent to the/i:/EA gene from strain 1-1529.<br>
In order to test the biological activity of each of these ferredoxins in combination with P450Emai' ^^ch individual ferredoxin gene was amplified by PCR to produce a gene fragment that included a blunt 5"-end. the native nbosome-bindins site and ferredoxin sene coding sequence, and a Pmel restrction site on the 3'-end. Each such ferredoxin gene fragment was cloned into the Pmel site located 3' to the emal gene in plasmid pTUA-e^AA/a/. In this way, artificial operons consisting of the emal gene and one of the ferredoxin genes operably linked to a functional promoter were created.<br>
In the case of the//EA ferredoxin gene in which a ferredoxin reductase gene,/reEA, was found to be located adjacent to theyi:/EA gene, a DNA fragment containing both the/^/EA and/reeA genes was generated by a similar PCR strategy. This gene fragment was also cloned in the Pmel site of plasmid pTUA-emal as descrbed for the other ferredoxin genes.<br><br>
Each c7^/c//~rcrrcdoxin gene combination was tested for biological activity by introduction of the individual enui14crvcdo\\u gene plasmids into S. lividcms vStrain ZX7. The biocataiysis activity derved from each plasmid in .V. lividans was determined. Of the four different constructs, only the ferredoxin gene/(/2--fJ derved from strain I-1529 provided increased activity when compared to the expression oiemal alone in the same plasmid and host background {sec Table 3). The pWA-('fH(il/l(l2^^^ plasmid in .V. lividans provided approximately 1.5 to 3- fold higher activity compared to the pTVA-emal plasmid. The other three plasmids containing the other ferredoxin genes gave results essentially the same as the plasmid with only the cmal gene. Likewise, the pTUA-eniaI/JdEA/frcEA plasmid did not yield results different from those of pTUA-(v//c//. The nucleotide and deduced amino acid sequences of \hc fd2SJ gene are shown m SLQ ID NOs:35 and 36, respectively.<br>
A BLAST analysis of the nucclolide and amino acid sequences ol'fd23J revealed that the closest matches were to fcrrcdoxins from .V. coclicolor (GenBank Accession AL445945) and S. lividans (GenBank Accession AF()727()9), At the nucleotide lcvel,/i^/2.?J shares 80 and 79.8 % identity with the ferredoxin genes from S. coclicolor and i\ lividans, respectively. At the peptide \c\tl, fd2J3 shares 79.4 and 77.8% identity with the fcrrcdoxins from 5. coclicolor and S. lividans, respectively.<br>
Since fd233 is derved from strain I-1529 and cmal is from strain R-922, the proteins encoded by the two genes cannot interact with each other in nature. In an approach designed to identify a ferredoxin gene from strain R-922 that is homologous to ihcfdZSS gene and that might encode a ferredoxin that interacts optimally with the P450EmDi» ihcfd233 gene was used as a hybrdization probe to a gene library of DNA from strain R-922. A strongly hybrdizing cosmid, pPEH232, was identified and the hybrdizing DNA was cloned and sequenced. Comparson of the deduced amino acid sequences from fd233 and the ferredoxin gene on cosmid pPEH232, fd232, revealed that they differed in only a single amino acid.<br>
In a similar manner, plasmid pJ\JA'CnHil-Jd232 was constructed and tested in 5. lividans ZX7. This plasmid gave similar results as those obtained with plasmid pTVA-emaJ-fd233 {see Table 3). The nucleotide and deduced amino acid sequences of fd232 are shown in SEQ ID NOs:37 and 38, respectively.<br>
The emal~fd233 operon was also subcloned, as a Pacl-Pmel fragment, into pTBBKA and pEAA that had been digested with the same restrction enzymes.  S. lividans<br><br>
ZX7::pTBBKA-^wa/-./^/2-?J, and S. lividans 7.Xl::pliAA-cmal-f(l2J3 were tested in the avcrmcctin conversion assay and found to have higher activities than the strains harborng the cmal gene alone in the comparable plasmids (sec Table 3).<br>
EXAMPLE XV Heterologous Expression of 1M50K„,,.I and IM5()Kni:i2 in Other C ells<br>
The expression constructs pRK-emal (Example XllI) and pRK-^^/;/c/2 (created in a way analogous to that descrbed in Example XlII for pRK-nnal) were mobilized by conjugation into three fluorescent soil Pscudomomis strains. Conjugation was performed according to standard methods (Dilta et uL. Proc. Natl. Acad. Sci. USA 77:7347-735 1, 1980). The strains were: P. jJuorescens MOCG134, P. Jlnoresccns Pf-5, and /'. fhiorcsccns CI lAO. Standard resting cell assays for the conversion of avermectin to 4"-ketoavermectin were conducted for each of the transconjugants. For strains Pf-5 and CHAO, the levels of conversion were below the detection limit. Strain MOCG 134 vielded 3% conversion for cmal and 5% for cmal.<br>
Tn addition, the constructs listed in the Table 5 were introduced into Strcptomyccs avermitilis MOS-OOOl by protoplast-mediated transformation (Kieser, T.; Bibb, MJ.; Buttner, M.J.; Chater, K.F.; Hopwood, D.A. (eds.): Practical Strcptomyccs Genetics. The John Inncs Foundation, Norwich (England), 2000), (Stulzman-Engwall, K. et al. (1999) Strcptomyccs avermitilis gene directing the ratio of B2:Bi avcrmectins, WO 99/41389).<br><br><br>
Wild-type Str. avennitilis MOS-OOOl was tested and found to be incapable of the oxidation ofavermcctin to 4"-ketoavermcctin.<br>
Transformed S, avermitilis strains MOS-OOOl::pTBBKA-(77K//, MOS-OOOl (pTUA-anal). MOS-OOOl::pEAA-t'ma/, MOS-0001::pTBBKA-^7/7^W^Vy^/2,?3, and MOS-OOOl ({iYV)!\-cfna 1 /\/fd233) were each tested for their ability to oxidi/e avermcctin to 4"-keto-avcrmectin using resting cells. To do this, the whole cell biocatalysis assay descnbcd above (including analysis method) was performed. Note that for the whole cell biocatalysis assay, transformed Streptomyces avermitilis, like strain R-922, was grown in PI IG medium and, again like strain R-922, had a reaction time of 16 hours {i.e.. durng which lime the 500 mg transformed Streptomyces avermitilis wet cells in 10 ml of 50 mM potassium phosphate buffer, pH 7.0, were shaken at 160 rpm at 28'C in the presence of 15 j.il of a solution of avcrmectin in isopropanol (30 mg/ml)).<br>
As shown in Table 5, in the presence of the inducer, thiostrepton (5 |,ig/ml), (he emal- or ^7Nr^/A^^y2.?J-containing strains MOS-OOOl::pTBBKA-m(a/, MOS-0001::pTBBKA-emaI/Vf(1233, MOS-OOOl (pTUA-tvm//), MOS-OOOl (pTVA-emaIA/fd233) were found to oxidize avcrmectin to 4"-keto-avermectin as evidenced by the appearance of the oxidized 4"-keto-avermectin compound. Note that the 5. avennitilis strain MOS-000I::pEAA-t'ma/ demonstrated this oxidation activity in the absence of thiostrepton since in this strain the emal gene is expressed from the ermE promoter thai does not require induction.<br>
Thus, expression of the emal P450 monooxygenase gene in varous Streptomyces and Pseudomonas strains provided recombinant cells that were able to convert avcrmectin to 4'-ketoavermectin in resting cell assays.<br>
Next, expression and activity of P450Emai monooxygenase was tested in E. coli. To do this, the emal gene was cloned into the E. co^/expression plasmid pET-28b(+) (commercially available from Novagen, Madison, Wl) as descrbed previously. /:. coli strain BL21 DE3 (commercially available from Invitrogen; Carlsbad, CA) that contains the T7 RNA polymerase gene under control of the inducible tac promoter and the pET-28/(?/na/ plasmid was cultured in 50 ml LB medium containing 5 mg/1 kanamycin in a 250-ml flask with one baffle, for 16 hours at ST^'C, with shaking at 130 rpm. 0.5 ml of this culture was used to inoculate 500 ml LB medium with 5 mg/1 kanamycin in a 2-liter flask with one baffle, and the<br><br>
culture was incubalcd for 4 hours al 37"C followed by 4 hours and 30'C, with shaking at 130 rpm throughout. The cells were harvested by ccntrfugation, washed in 50 mM potassium phosphate buffer, and centrfuged again.<br>
For the resting cell assays, 90 mg wet cells were weighed into deep-well plates in trplicate and resuspended in 0.5 ml 50 mM potassium phosphate buffer.  F\)r cell-free extracts, 4 grams wet cells in 8 ml disruption buffer were disrupted in French press.<br>
For the resting cell assays, 5 |,il of substrate (2.5 mg/ml in 2-propanol) was added to the cell suspension. The plate was sealed with air permeable foil, and the reaction was incubated on an orbital shaker at 1000 rpm at 28"C for 22 hours. No conversion of aveimectin to 4"-kctoavermectin was detected.<br>
F\)r the celMVee assays, 100 |.il cell free extract, l|.d substrate solution (20 mg/ml) in 2-propanol, 5 ix\ 100 mM NADPIl, 10 (.d ferredoxin, 10 ^il ferrcdoxin reductase, and 374 |xl potassium phosphate buffer pH 7.0 were added as descrbed in Example HI, and the assay was incubated at 3(rC with shaking at 600 rpm for 20 hours. 9.2% +/- 0.3% of avermcctin was converted to 4'*-kctoavermcctin.<br>
Thus, expression of the emal gene in /:'. coli resulted in the production of the active Emal P450 monooxygenasc en/yme which, when purfied from the cells, was able to convert avermectin to 4"-ketoavermectin.<br>
EXAMPLE XVI<br>
Identification and Cloning of Genes Encoding Ferredoxin Reductases that Support Increased<br>
Activity of the P450Fma2 Monooxygenasc<br>
The electron transport pathway that supports the activity of P450 monooxygenases also includes ferredoxin reductases. These proteins donate electrons to the ferredoxin and, as is the case with ferredoxms and P450 monooxygenases, specific ferredoxin reductases are known to be better electron donors for certain ferredoxins than others.<br>
According, a number of ferredoxin reductase genes from Streptomyces strains were cloned and were evaluated for their impacts on the biocatalysis reaction. To do this, numerous bacteral ferredoxin reductase (Fre) protein sequences were retreved from NCBI and aligned with the program Pretty from the GCG package. Two conserved regions.<br><br>
approximately 266 amino acid residues apart, were used lo make degenerate oligonucleotides For PCR. The forward prmer (CGSCCSCCSCTSWSSAAS (SEQ ID NO:96; where '^S" is C or G; and 'W is A or G)) and the reverse prmer (SASSGCSTTSBCCCARTGYTC (SEQ ID NO:97; where ^\S^' is C or G; ^13" is C, G, or T; ^^R" is A or G; and ^'Y" is C or T)) were used to amphfy 800 bp products from the biocatalytically active Streptomyces strains R-922 and I-1529. These pools of products were cloned into ^FOPO TA cloning vectors (commercially available from Invitrogen Inc., Carlsbad, CA), and 20 clones each from R922 and 1-1529 were sequenced according to standard methods (see, e.}^.. Current Protocols in Molecular Biology, cds. Ausubei el al., John Wiley &amp; Sons, Inc. 2000). Sequencing revealed that 4 unique/re gene fragments were isolated from the strains: three from R922 (frc3Jrcl2,frel4) and one from M529 (J)rl6). The /;7'3,/r
In order to assess the biological activity of cnch fre gene in relation to the activity of Emal, each gene was inserted into the emal/fd233 opcron descrbed above, 3' to ihcfd233 gene. This resulted in the formation of artificial operons consisting of the emal,fd233. and individual/rf genes that were expressed from the same promoter. The emal/fd233/fre operons were cloned into the Pseudomomis plasmid pRK290 and introduced into 3 different P. piitida strains. These strains were then analysed for Emal biocatalysis activity using the whole cell assay and one of the genes. Xhcfre gcx\cfre\6 from strain 1-1529, was found to increase the activity of P450Emai monooxygenase by approximately 2-fold. This effect was strain specific, as it was seen only in one of the P. piitida strains, ATCC Desposit No. 17453, and not in the other two. In P. pulida strain ATCC 17453, the presence of fre gcncfre\(y resulted in 44% conversion of avermectin to 4"-kelo-avermectin, as compared to 23% vvithout this gene. The oiYiCX fre genes had no impact on the biocatalysis activity in any of the P, putida strains tested.<br>
In a similar approach, each of the emcil/fd233/fre operons were cloned into the Streptomyces plasmids pTUA, pTBBKA, and pEAA, and introduced into 5. lividans strain<br><br>
ZX7.  In each case there was no impact in S. Uvidans by any of {hcfre genes on biocatalysis activity.<br>
EQUIVALENTS<br>
Those skilled in the art will recognize, or be able to ascertain, using no more than routine expeninenion, numerous equivalents to the specific substances and procedures descrbed herein. Such equivalents are considered to be within the scope of this invention. The patent and scientific literature referred to herein establishes knowledge that is available to those with skill in the art. The issued patents, applications, and references, including GenBank database sequences, that arc cited herein are hereby incorporated by reference to the same extent as if each was specifically and individually indicated to be incorporated by reference. Any conflict between any reference cited herein and the specific teachings of this specification shall be resolved in favor of the latter.  Likewise, any conflict between an art-understood definition of a word or phrase and a definition of the word or phrase as specifically taught in this specification shall be resolved in favor of the latter.<br><br><br><br>
What is claimed is:<br>
1.    A purified nucleic acid molecule encoding a polypeptide thai exhibits an en/ymatic<br>
activity of a P45() monooxygcnase and is capable of regioselectively oxidi/ing the alcohol at position 4" of a compound of formiilar (11)<br><br>
wherein R1-R7 represent, independently of each other hydrogen or a substituent;<br>
m is 0, 1 or 2;<br>
n is 0, 1, 2 or 3; and<br>
the bonds marked with A, B, C, D. E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br><br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, ancl/or a lautomer thereof, in each case in free form (or m salt form, in order to produce a compound of the formula (III)<br><br>
wherein Ri-R?. m. n. A, B, C, D, E and F have the same meanings as given for formula (11) above.<br>
2.	The nucleic acid molecule of claim 1, comprising a nucleic acid sequence that encodes a<br>
polypeptide that exhibits an enzymatic activity of a P450 monooxygenase and<br>
regioselectively oxidizes avennectin to 4"keto-avermectin.<br>
3.	The nucleic acid molecule of claims 1 or 2, comprising a nucleic acid sequence that<br>
encodes a polypeptide that exhibits an enzymatic activity of a P450 monooxygenase,<br>
which polypeptide is substantially similar, and has between at least 50%, and 99% amino<br>
acid sequence identity to the polypeptide of SEQ ID NO:2.<br><br>
4.    The nucleic acid molecule of claim 3 comprising a nucleotide sequence<br>
a)	as given in SEQ ID NO: I;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or (he complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides o\ a nucleotide set|ucnce given in SEQ ID NO; 1, or the complement thereof;<br>
c) complementary to (a), (b) or (c);<br>
0 which is the reverse complement of (a), (b) or (c); or<br>
g) which is a functional part of (a), (b), (c), (d), (e) or (0 encoding a polypeptide that still<br>
exhibits an enzymatic activity of a P450 monooxygenase and regioselectivcly oxidizes<br>
avcrmectin to 4"-keto-avermectin. .<br>
5.	The nucleic acid molecule of claims I or 2, comprising a nucleic acid sequence that is at least 66 % identical to SEQ ID NO: I.<br>
6.	The nucleic acid molecule of claims I or 2, comprising a nucleic acid sequence that encodes a polypeptide that exhibits an enzymatic activity of a P450 monooxygenase, which polypeptide is substantially similar, and has at least between 60%, and 997c amino acid sequence identity to the polypeptide of SEQ ID NO:2, SEQ ID NO:4, SEQ ID N0:6, SEQ ID NO:8, SEQ ID NO: 10, SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO:16, SEQ ID NO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26. SEQ ID NO:28, SEQ ID NO:30. SEQ ID NO:32, SEQ ID NO:34, or SEQ ID NO:95.<br>
7.	The nucleic acid molecule of claims 1 or 2. comprising a nucleic acid sequence that encodes a polypeptide that exhibits an enzymatic activity of a P450 monooxygenase,which polypeptide is immunologically reactive with antibodies raised against a polypeptide of SEQ ID N0:2, SEQ ID NO:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO: 10. SEQ ID NO: 12, SEQ ID NO: 14, SEQ ID NO: 16, SEQ ID N0:18,<br><br>
SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO;34, or SEQ ID NO:93.<br>
The nucleic acid molecule of claims 1 or 2 comprising a nucleotide sequence<br>
a)	as given in SEQ ID NO: 1, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7. SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13. SEQ ID NO: 15, SEQ ID NO: 17. SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, or SEQ ID NO:94;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 30 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO: I, SEQ ID NO:3, SEQ ID N0:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:3 1, SEQ ID NO:33, or SEQ ID NO:94 or the complement thereof;<br>
c) complementary to (a), (b) or (c): and<br>
0 which is the reverse complement of (a), (b) or (c).<br>
g) . which is a functional part of (a), (b), (c), (d), (e) or (f) encoding a polypeptide that<br>
still exhibits an enzymatic activity of a P450 monooxygenasc and regioselectively<br>
oxidizes avermectin to4"-keto-avermectin.<br>
The nucleic acid molecule of claim 8. composing a nucleic acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO:3. SEQ ID NO:5, SEQ ID NO:7, SEQ IDNO:9,SEQIDNO:ll,SEQIDNO:I3,SEQIDNO:I5,SEQIDNO:I7,SEQID N0:I9, SEQ ID NO:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27. SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, and SEQ ID NO:94.<br>
). The nucleic acid molecule of anyone of claims 1 to 9, wherein the nucleic acid molecule is isolated from a Streptomyccs strain.<br><br>
11.	The nucleic acid molecule of anyone of claims 1 to 10 further comprising a nucleic acid scc|ucnce encoding a lag which is linked to the P450 monooxygenase via a covalcnt bond.<br>
12.	A polypeptide that exhibits an enzymatic activity of a P430 monooxygenase and is capable of regioselectively oxidi/ing the alcohol at position 4" of a compound of formular<br>
(11)<br>
R7<br>
wherein Ri-R? represent, independently of each other hydrogen or a substitucnt:<br>
m is 0, 1 or 2;<br>
n is 0, 1, 2 or 3; and<br>
the bonds marked with A, B, C, D, E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br><br><br>
including, where applical")lc, an il/Z isomer, a mixture of E/Z isomers, and/or a tautomer thereof, in each case in free form or in salt form, in order to produce a compound of the formula (III)<br><br>
wherein R1-R7, m, n. A, B, C, D, E and F have the same meanings as given for formula (II) above.<br>
13.	A polypeptide that exhibits an enzymatic activity of a P450 monooxygenase and regioselectively oxidizes avermectin to 4**keto-avermectin.<br>
14.	The polypeptide of claims 12 or 13 that comprises an amino acid sequence that is encoded by a nucleic acid molecule<br><br>
a)	as given in SEQ ED NO: 1 or the complement thereof;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br><br>
d)	capcibic of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO:l, or the complement thereof;<br>
e)	complementary to (a), (b) or (c);<br>
0 which is the reverse complement of (a), (b) or (c); or<br>
g) which is a functional part of (a), (b), (c). (d), (e) or (0 encoding a polypeptide that still<br>
exhibits an enzymatic activity of a P45() monooxygcnasc and regioseicctively oxidizes<br>
avcrmectin to 4'-kcto-avermcctin.<br>
15.	The polypeptide of claims 12 to 14, comprising an amino acid sequence that is at least 50% identical to SEQ ID NO:2.<br>
16.	The polypeptide of claims 12 or 13 compnsing an amino acid sequence that is encoded by a nucleic acid molecule<br><br>
a)	as given in SEQ ID NO:I, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9, SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15, SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ FD N0:31, SEQ ID NO:33, or SEQ ID NO:94 or the complement thereof;<br>
b)	having substantial similarity to (a);<br>
c)	capable of hybridizing to (a) or the complement thereof;<br>
d)	capable of hybridizing to a nucleic acid molecule comprising 50 to 200 or more consecutive nucleotides of a nucleotide sequence given in SEQ ID NO:l, SEQ ID NO:3, SEQ ID NO:5, SEQ ID NO:7, SEQ ID NO:9. SEQ ID NO: 11, SEQ ID NO: 13, SEQ ID NO: 15. SEQ ID NO: 17, SEQ ID NO: 19, SEQ ID NO:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:3I, SEQ ID NO:33, or SEQ ID NO:94 or the complement thereof, or the complement thereof:<br>
e)	complementarN' to (a), (b) or (c);<br>
f)	which is the reverse complement of (a), (b) or (c); or<br><br>
g) which is a functional part of (a), (b), (c), (d), (c) or (f) encoding a polypeptide that still exhibits an enzymatic activity of a P450 monoo.xygenase and regioselectivciy oxidizes avcrmectin to 4'-kcto-avermectin.<br>
17.	The polypeptide of claim 16, comprising an amino acid sccjiicncc selected from the group consisting of SEQ ID NO:2, SEQ ID tNO;4, SEQ ID NO:6. SVX) ID NO:8. SEQ ID NO:10, SEQIDN0:12, SEQIDN0:I4, SEQIDNO:16, SEQIDNO:18,SEQID N0:2(), SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, and SEQ ID NO:95.<br>
18.	The polypeptide of anyone of claims 12 to 17, further comprising a lag.<br>
19.	A binding agent that specifically binds to the polypeptide of anyone of claims 12 to 18.<br>
20.	The binding agent of claim 20, wherein the binding agent is an antibody.<br>
21.	A family of polypeptides exhibiting an enzymatic activity of a P450 monooxygenase, wherein each member of the familv is capable of rcuioselcctivclv oxidizing the alcohol at position 4" of a compound of formular (II)<br><br><br>
wherein R1-R7 represent, independently of ciich other hydrogen or a substituent;<br>
m is 0, 1 or 2;<br>
n is 0, 1,2 or 3; and<br>
the bonds marked with A, B, C, D, E and F indicate, independently of each other, that two adjacent carbon atoms arc connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a lautomer thereof, in each case in free form or in salt form, in order to produce a compound of the formula (III)<br><br><br>
wherein R1-R7, m, n. A, B, C, D. E and F have the same meanings as given for formula (11) above.<br>
22.	A family of polypeptides exhibiting an enzymatic activity of a P450 monooxygenase,<br>
wherein each member of the family oxidizes avermectin to 4"keto-avermcctin.<br>
23.	The family of claims 21 or 22, wherein each member of the family is comprises an amino acid sequence that is at least 50% identical to SEQ ID NO:2.<br>
24.	A purified nucleic acid molecule comprising a nucleotide sequence encoding a polypeptide exhibiting an enzymatic activity of a ferredoxin and a ferredoxin reductasejespectively. wherein the nucleic acid molecule is isolated from a Streptomyces strain comprising a P450 monooxygenase that is capable of regioselectively oxidizing the alcohol at position 4" of a compound of formular (II)<br><br><br>
wherein R1-R7 represent, independently of each other hydrogen or a substituent;<br>
m is 0, 1 or 2;<br>
n is 0, 1, 2 or 3; and<br>
the bonds marked with A, B, C, D, E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a tautomer thereof, in each case in free form or in salt form, in order to produce a compound of the formula (III)<br><br><br>
wherein R1-R7, m, n. A, B, C, D, E and F have Ihe same meanings as given for tonnula (11) above.<br>
25.	A purified nucleic acid molecule according to claim 24 comprising a nucleotide sequence encoding a polypeptide exhibiting an enzymatic activity of a fcrrcdoxin and a ferredoxin reductase, respectively, wherein the nucleic acid molecule is isolated from a Sireptomyces strain comprising a P450 monooxygenase that regioselectively oxidizes avcrmectin to 4"keto-avermectin.<br>
26.	The nucleic acid molecule of claim 25, comprising a nucleic acid sequence selected from the group consisting of SEQ ID NO:35 and SEQ ID NO:37.<br>
27.	The nucleic acid molecule of claim 25, comprising a nucleic acid sequence selected from the group consisting of SEQ ID NO:98, SEQ ID NO: 100, SEQ ID NO: 102. and SEQ ID NO: 104.<br>
28.	A polypeptide exhibiting an enzymatic activity of a feiredoxin and a ferredoxin reductase, respectively, wherein the polypeptide is isolated from a Streptomyces strain comprising a<br><br>
P450 monooxygcnasc th;it is capable of regioselcclivcly oxidizing Ihc alcohol at position 4" of a compound of formular (H)<br><br>
wherein R1-R7 represent, independently of each other hydrogen or a substitucnt;<br>
m isO, I or 2;<br>
n is 0, 1, 2 or 3; and<br>
the bonds marked with A, B, C, D. E and F indicate, independently of each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a taulomer ihcreof, in each case in free form or in salt ("orm, in order to produce a compound ol' thcformula (III)<br><br>
wherein R1-R7, m, n. A, B, C, D, E and F have the same meanings as given for formula (II) above.<br>
29.	A polypeptide exhibiting an enzymatic activity of a ferredoxin and a fcrredoxin reductase,<br>
respectively, wherein the ferredoxin protein is isolated from a Streptomyccs strain<br>
comprising a P450 monooxygenase that regiosclectively oxidizes avermectin to 4"keto-<br>
avermectin.<br>
• ^.	■■■•■<br>
30.	The ferredoxin protein of claim 29, comprising an amino acid sequence selected from the<br>
group consisting of SEQ fD NO:36 and SEQ ID NO:38.<br>
31.	The ferredoxin reductase protein of claim 29, comprising an amino acid sequence<br>
selected from the group consisting of SEQ ID NO:99, SEQ ID NOrlOl, SEQ ID NO:103,<br>
and SEQ ID NO: 105.<br><br>
32.	A cell genetically engineered to comprise a nucleic acid molecule encoding a polypeptide<br>
that exhibits an enzymatic activity of a P45() monooxygenase according to anyone of<br>
claims I to 1 I.<br>
33.	The cell of claim 32 further comprising a nLicieic acid molecule encoding a ferredoxin<br>
protem and a ferredoxin reduclrasc protein, respectively, or a combination thereof.<br>
34.	The cell of claims 32 or 33, wherein the nucleic acid molecule is positioned for<br>
expression in the cell.<br>
35.	The cell of anyone of claims 32 to 34. wherein the cell is a genetically engineered cell<br>
selected from the group consisting of a Strcptomyccs strain cell and a Pscndomona .vtrain<br>
cell, and an Escherichia coli strain cell.<br>
36.	The cell of claim 35, wherein the cell has NRRI. Designation No. B-3()478 and NRRL Designation No.B-3()479, respectively.<br>
37.	A method for the preparation a compound of the formula<br><br><br>
in which<br>
Ri-R.j represent, independently of each other hydrc^gen or a substilucnl:<br>
m is 0, I or 2;<br>
n is 0, 1,2 or 3; and<br>
the bonds marked with A, B, C\ D, E and F indicate, independently ot each other, that two adjacent carbon atoms are connected by a double bond, a single bond, a single bond and a epoxide bridge of the formula<br><br>
including, where applicable, an E/Z isomer, a mixture of E/Z isomers, and/or a taulomer<br>
thereof, in each case in free form or in salt form,<br>
which process comprises<br>
1) bringing a compound of the formula<br><br><br>
wherein<br>
R1-R7, m, n, A, B, C, D, E and F have Ihc same meanings as given for formula (I) above,<br>
into contact with a polypcplidc according to the invention that is capable of<br>
regioseleclively oxidising the alcohol at position 4" in order lo form a compound of the<br>
formula<br><br>
in which R|, R2, R3, R4, R5, R^,. R?, m, n, A. B, C, D, E and F have the meanings given for formula (I); and<br>
2) reacting the compound of the formula (III) with an amine of the formula HN(RH)R*&gt;, wherein RH and R9 have the same meanings as given for formula (I), and which is known, in the presence of a reducing agent;<br>
and, in each case, if desired, converting a compound of formula (I) obtainable in accordance with the process or by another method, or an E/Z isomer or tautomer thereof, in each case in free form or in salt form, into a different compound of formula (I) or an E/Z isomer or tautomer thereof, in each case in free form or in salt form, separating a mixture of E/Z isomers obtainable in accordance with the process and isolating the desired isomer, and/or converting a free compound of formula (I) obtainable in accordance with the process or by another method, or an E/Z isomer or tautomer thereof, into a salt or converting a salt, obtainable in accordance with the process or by another<br><br>
method, of a compound of formula (I) or of an E/Z isomer or tautomer thereof into the free compound of formula (I) or an E/Z isomer or tautomer thereof or into a different salt.<br>
&gt;8.  A method for the preparation of a compound of the formula<br><br>
in which Ri, R:. R3, R4. R5, Rr*, R7, rn, n. A, B, C, D, E and F have the meanings given for formula (III) of claim 37, which process comprises<br>
1) bringing a compound of the formula<br><br><br>
wherein<br>
RrR7, m, n. A, B, C, D, E and F have the same meanings as given for formula (I) above, into eontact with a polypeptide according to the invention that is capable of rcgioselectively oxidising the alcohol at position 4", maintaining said contact for a time sufficient for the oxidation reaction to occur and isolating and purifying the compound of formula (11).<br>
39.	A method according to anyone of claims 37 or 38 for making emamectin, comprising adding a polypeptide that exhibits an enzymatic activity of a P450 monooxygenase and rcgioselectively oxidizes avermectin to 4'"keto-avermcctin to a reaction mixture comprising avermectin and incubating the reaction mixture under conditions that allow the polypeptide to regioselectively oxidize avermectin to 4"*keto~avermectin.<br>
40.	The method of anyone of claims 37 to 39, wherein the reaction mixture further comprises a ferredoxin protein.<br>
41.	The method of anyone of claims 37 to 40, wherein the reaction mixture further comprises a ferredoxin reductase protein.<br><br>
A formulation for making emamcctin comprising a polypeptide that exhibits an cn/ymatic activity of a P450 monooxygcnasc and regiosclcctivcly oxidi/es avermectin to 4"kcto-avcrmcclin.<br>
The formukilion ofckiims 42 further comprising a Icrredoxin protein.<br>
The formukition ofck^iim 42 or 43 further comprising a ferrcdoxin reductase protein.<br><br>
A purified nucleic acid substantially as herein described and exemplified.<br>
A polypeptide exhibiting an enzymatic activity substantially as herein described and exemplified<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5Ni1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1796-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="201497-a-mobile-messaging-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="201499-a-projection-type-image-display-apparatus.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>201498</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1796/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Jul-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Nov-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SYNGENTA PARTICIPATIONS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Schwarzwaldallee 215, CH-4058 Basel</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MOLNAR, Istvan</td>
											<td>Syngenta Biotechnology, Inc., 3054 Cornwallis Road, Research Triangle Park, NC 27709</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LIGON, James, Madison</td>
											<td>Syngenta Biotechnology, Inc., 3054 Cornwallis Road, Research Triangle Park, NC 27709</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ZIRKLE, Ross, Eric</td>
											<td>Syngenta Biotechnology, Inc., 3054 Cornwallis Road, Research Triangle Park, NC 27709</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HAMMER, Philip, Eugene</td>
											<td>212A WALDO STREET CARY, NC 27511</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HILL, Dwight, Steven</td>
											<td>Syngenta Biotechnology, Inc., 3054 Cornwallis Road, Research Triangle Park, NC 27709</td>
										</tr>
										<tr>
											<td>6</td>
											<td>PACHLATKO, Johannes, Paul</td>
											<td>SYNGENTA CROP PROTECTION AG SCHWARZWALDALLEE 215 CH-4058 BASEL</td>
										</tr>
										<tr>
											<td>7</td>
											<td>BUCKEL, THOMAS, GUNTER</td>
											<td>SYNGENTA CROP PROTECTION AG SCHWARZWALDALLEE 215 CH-4058 BASEL</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2002/005363</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-05-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/291,149</td>
									<td>2001-05-16</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/201498-a-method-for-the-preparation-of-compounds-for-making-emamectin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:22:08 GMT -->
</html>
