<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/201606-a-method-for-determining-differences-in-the-level-of-gene-expression by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:49:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 201606:A METHOD FOR DETERMINING DIFFERENCES IN THE LEVEL OF GENE EXPRESSION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR DETERMINING DIFFERENCES IN THE LEVEL OF GENE EXPRESSION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>he present invention relates to a highly accurate and reproducible mathematically-based methodology for quantifying the levels of differential gene expression from microassay protocols. Specifically, the present invention provides a simple deductive approach, grounded in a Bayesian framework, to circumvent the heuristic-based limitation of previous methodologies. Rather than seeking a point-estimate of the fold-change of the level of gene expression, the present invention utilizes the derivation of mathematical formula to determine the a posteriori distribution of all the fold-changes of differential gene expression which may be inferred from the given measurements. From this a posteriori distribution the following information may be obtained: (i) an estimator for the fold-change of the level of gene expression; (ii) confidence limits for the fold-change, at any given confidence level; and (iii) a P-value for assessing the statistical significance of change. The present invention also possesses the advantage that fold-change estimates and confidence limits may even be assigned to signal pairs where both signals are zero or negative, without resorting to heuristic thresholds.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
The present invention relates to a method determining differences in the level of gene expression.<br>
BACKGROUND OF THE INVENTION<br>
Cells rely upon their numerous protein components for a wide variety of ninctions. These functions include, e.g., the production of energy, the biosynthesis of all component macromolecules, the maintenance of cellular architecture, the ability to act upon intra- and extracellular stimuli, and the like. Each cell within an organism in contains the information necessary to produce the repertoire of proteins that organism can expressed. This information is stored as genes within the organism"s genome. The number of unique human genes is estimated to be 30,000 to 100,000.<br>
For any given cell, only a portion of the gene set is expressed in the form of protein. Some of the proteins are likely to be present in all cells (i.e., are ubiquitously expressed) because they serve biological function(s) which are required in every type of cell, and can be thought of as "housekeeping" proteins. In contrast, other proteins serve specialized functions that are only required in particular cell types. For example, muscle cells contain specialized proteins that form the dense contractile fibers of a muscle. Given that a large part of a cell"s specific functionality is determined by the genes it is expressing, it is logical that transcription, the first step in the process of converting the genetic information stored in an organism"s genome into protein, would be highly regulated by the control network that coordinates and directs cellular activity.<br>
Regulation of gene expression is readily observed in studies that examine activities evident in cells configuring themselves for a particular function (e.g., specialization into a muscle cell) or state (e.g., active multiplication or quiescence). Hence, as cells alter their status, the coordinated transcription of the protein(s) which are requited for this molecular biological/physiological "state" can be observed. This highly detailed, global knowledge of the cell"s transcriptional state provides information on the cell"s status, as well as on the biological system(s) controlling this status. For example, knowledge of when and in what types of cell the protein product<br><br>
of a gene of unknown function is expressed would provide useful clues as to the likely function of that gene. Determination of gene expression patterns in normal cells could-provide detailed knowledge of the way in which the control system achieves the highly coordinated activation and deactivation required for development and differentiation of a mature organism from a single fertilized egg. Comparison of gene expression patterns in normal and pathological cells could provide useful diagnostic "fingerprints" and help identify aberrant functions that would be reasonable targets for therapeutic intervention.<br>
Unfortunately, the ability to carry out studies in which the transcriptional state of a large number of genes is determined has, until recently, been inhibited by limitations on the ability to survey cells for the presence and abundance of a large number of gene transcripts in a single experiment. One limitation can be in the small number of identified genes. In the case of humans, only a few thousand proteins encoded within the human genome have been physically purified and quantitatively characterized to any extent. Another limitation can be in the manner of transcription analysis.<br>
Two recent technological advances address have aided analyses of gene transcription. The cloning of molecules derived from mRNA transcripts in particular tissues, followed by the application of high-throughput sequencing to the DNA ends of the members of these libraries has yielded a catalog of expressed sequence tags (ESTs). See e.g., Boguski and Schuler, Nat. Genetics 10: 369-370 (1995). These "signature sequences" can provide unambiguous identifiers for a large cohort of genes.<br>
In addition, the clones from which these sequences were derived provide analytical reagents that can be used in the quantitation of transcripts front biological samples. The nucleic acid polymers, DNA and RNA, are biologically synthesized in a copying reaction in which one polymer serves as a template for the synthesis of an opposing strand, which is termed its complement. Following the separation of the strands from one another (i.e., denaturation), these strands can be induced to pair, quite specifically, with other nucleic acid strands possessing a complementary sequence in a process called hybridization. This specific binding can be the basis of<br><br>
analytical procedures for measuring the amount of a particular species of nucleic acid, such as the mRNA specifying a particular protein gene product.<br>
A second advance involves microarray/microassay technology. This is a hybridization-based process which allows simultaneous quantitation of many nucleic acid species. See e.g., DeRisi era/., Nat. Genetics 14: 457-460 (1996); Schena et al, Proc. Natl. Acad. Set. USA 93: 10614-10619 (1996). This technique combines robotic placement (i.e., spotting) of small amounts of individual, pure nucleic acid species on a glass surface, hybridization to this array with multiple fluorescently-labeled nucleic acids, and detection and quantitation of the resulting fluorescent-labeled hybrids with, for example, a scanning confocal microscope. When used to detect transcripts, a particular RNA transcript (i.e., an mRNA) can be copied into DNA (i.e., a cDNA) and this copied form of the transcript is subsequently immobilized onto, for example, a glass surface.<br>
A problem in the analysis of gene expression data is the estimation of the overall fold-change in the expression level of a gene in one experiment relative to its expression in another experiment. Given these two raw measurements of the fold-change in gene expression level, the simplest approach, as utilized by previous methodologies, has been to take the arithmetic ratio of the values as an estimate of the overall fold-change. While for very strong signals this leads to a meaningful estimate of the fold-change in the underlying mRNA concentrations, for weaker signals the results are much more ambiguous because of contamination by the "noise" which is indigenous to the particular experimental system utilized. Another previously utilized technology for the estimation of the fold-change in gene expression level is based upon differential-signal intensities (e.g., the Affymetrix® chip). However, the values assigned to expression levels by use of the aforementioned methodology can be negative, thus leading to the awkward situation of negative or undefined gene expression ratios.<br><br>
SUMMARY OF THE INVENTION<br>
The present invention provides a highly accurate and reproducible mathematically-based methodology for quantifying the levels of differential gene expression from microassay protocols.<br>
The methods of the present invention can be used to calculate differences in the level of gene expression in one or more arrays of genes. The methods involve defining the experimental noise associated with intensity of hybridization signal for each gene in the array(s). The experimental noise is variations in observed levels on chips or other microarrays rather than biological noise, which is the variation of expression level seen in biological systems. Detection of genes is often, but not always, based on fluorescence. Other detection systems have been used which may be adapted here. Such systems include luminescent or radioactive labels, biotinylated, haptenated, or other chemical tags that allow for easy detection of labeled probes.<br>
For a mathematical description, see Section I - Formulation of the Noise Model below. The noise is assumed to be Gaussian and Bayes Theorem is applied. The defined experimental noise term, sigma, is then used to define an analytical (i.e., analytical in the mathematical sense meaning that it is a continuous function) probability distribution function ("pdf) describing distribution values of intensity for each gene. These pdfs are used to derive an analytical joint pdf describing possible ratios or fold changes for any differentially-expressed gene or gene product in the array(s). The joint pdfs are applied using experimentally-derived intensities and noise values from the genes on the array(s) (1) to estimate fold changes in the concentration of gene transcripts, (2) to use the jpdf to establish the confidence limits on the fold change given specific confidence intervals, and (3) to derive a p-value, or quality metric (the probability a fold change could be less than 1, when the estimate is greater than 1, or, the probability that the fold change is greater than 1, when the estimate is less than 1), associated with the fold change estimate. The estimated fold change determined by the methods of the present invention represents the difference in level of gene expression observed. The total variance (i.e., noise) may still be high even as the concentration of transcript goes to zero. The methods of the present invention use<br><br>
a mathematical formula to describe an a posteriori statistical distribution of all the levels of gene expression which may be derived from the measurements obtained of levels of gene expression in one or more cells or tissue types represented in the array(s).<br>
Microarrays are an ordered array of double stranded or single stranded DNA molecules positioned on a support material in a spatially separated organization. In contrast to filter "macroarrays", which are typically large sheets of nitrocellulose, microarrays position the DNA more densely packed organization such that up to 10000 DNA molecules can be fit into a region typically 1-4 square centimeters. Microarrays typically use coated glass as the solid support, in contrast to the nitrocellulose-based material of filter arrays. By having an ordered array of DNA samples, the position of each sample can be tracked and linked to the original sample from which the DNA on the array was generated. Methods and apparatus for preparing a microarray have been described. See, e.g., United States Patents 5,445,934 and 5,800,992, both incorporated herein by reference.<br>
The DNA samples on the microarray are hybridized with RNA or DNA probes that have been fluorescently labeled to identify whether the probe sample contains a molecule that is similar or identical to the DNA sample on the microarray. Under the appropriate conditions, probe molecules hybridize to a DNA molecule on the microarray. Generally, identical or near identical sequences form productive hybrids. The presence of DNA-probe hybrid molecules is detected by a fluorescence detection instrument. If the hybridization signal is weak or non-existent at a particular DNA site, then the corresponding DNA or RNA molecule in the probe is absent. Current microarray instruments can hybridize up to four different fluorescent probe samples at one time. With improvements to the technology, more probes can be hybridized at once.<br>
Up until recently, DNA hybridizations were performed on nitrocellulose filters. In contrast to microarrays where DNA is spotted directly onto the microarray, filter arrays are generated by spotting bacterial colonies on the filters, placing the filters over a agar growth media, and incubating the filters under conditions that promote the<br><br>
bacterial colonies to grow. The DNA within the bacterial colonies is released by lysing the colony and treating the filters to fix the DNA to the filter material. The process of generating a bacterial filter array can take typically 2-4 days. Microarrays have a number of advantages over filter array methods. For example, filter methods generally array bacterial colonies in which the cloned cDNA is contained. The colonies must be grown up over several days, lysed to release DNA and fix DNA onto the filter. Hybridization to filter arrays of colonies is less reliable due to bacterial debris and the low amount of DNA released from the colony. A second advantage is that the iterations are quicker with microarrays than with filters. This is due to the time needed to grow colonies on the filters and prepare them for the next round of hybridization. In contrast, probing of a subsequent microarray can begin less than 24 hr after analysis of an array is completed. Another advantage of microarrays is the ability to use fluorescently labeled probes. This provides for a non-radioactive method for hybridization detection. In contrast, filter hybridization generally uses probes labeled with radioactive phosphorus or sulfur. Microarrays can be hybridized with multiple probes simultaneously. In contrast, filter arrays can only be hybridized with one probe at a time. One of the most important advantages of microarrays is their reproducibility and sensitivity of hybridization signals. Typically, hybridization signals are higher and sensitivity is greater on microarrays versus filter arrays. In addition, filter arrays often exhibit spurious background signals that are unrelated to productive hybridization between the probe and DNA on the filter.<br>
Once the random sample of nucleic acid fragments is immobilized to a solid surface (e.g., glass) in a microarray, the random sample of nucleic acid fragments can then be hybridized to one or more labeled probes complementary to genes or sequences of interest. Generally, the unhybridized probes are removed. The labeled probes are then detected by methods known in the art (e.g., confocal microscopy). For example, slide images can be with ArrayVision image analysis software (Imaging Research) for spot finding analysis, localized background determination, distribution of signal intensities in a spot, and signal to noise ratios. Statistical assessment is then performed as described below.<br><br>
The present invention utilizes a mathematically-based methodology to quantitative the fold-change in the levels of differentially expressed genes. Specifically, the present invention uses a simple deductive approach, grounded in a Bayesian framework, to circumvent the heuristic-based limitation of previous methodologies used in the mathematical analysis of differential gene expression. The present invention, rather than immediately seeking a point-estimate of the fold-change of the level of gene expression, derives a mathematical formula for the a posteriori distribution of all the fold-changes of differential gene expression which may be inferred from the given measurements. From this a posteriori distribution the following information may be obtained: (/) an estimator for the fold-change of the level of gene expression; (ii) confidence limits for the fold-change, at any given confidence level; and (Hi) a P-value for assessing the statistical significance of change. An additional advantage of the present invention is that fold-change estimates and confidence limits may even be assigned to signal pairs where both signals are zero or negative, without resorting to heuristic thresholds. Hence, the mathematical framework disclosed herein unifies estimation for all signals within a given sample.<br>
Accordingly the present invention provides a method for determining differences in the level of gene expression in at least one array of genes comprising:<br>
(a)	defining experimental noise associated with intensity of hybridization signal for each gene in the at least one array;<br>
(b)	determining an analytical probability distribution function describing distribution values of intensity for each gene on the bases of said experimental noise by applying Bayes Theorem.<br>
(c)	deriving an analytical joint probability distribution function describing at least one of a possible ratio and a fold change for at least one differentially-expressed gene on the basis of said analytical probability distribution function.<br>
(d)	by applying the joint probability distribution function of each differentially-expressed gene and gene product with experimentally-derived intensities and noise values from the genes to determine values related to fold changes of the genes in at least an array of genes.<br><br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 is a line graph illustrating an a posteriori distribution of the fold change, Equation (16), for a series of measurement pairs (xt, x2). In all cases but (0, 0), the ratio of measurements is 4. The standard deviation of both noise terms is kept constant at a; = a2 = 20.<br>
FIG. 2 is a qualitative illustration of the derivation of Equation(16), explaining the behavior of the distributions shown in FIG. 1. For each pair of signals (xu x2), a box is drawn with extents ± cc about a point in the plane. Lines are drawn from the origin to the point in the box: the distribution of slopes of these lines is the a posteriori distribution of fold-changes. Panel A represents construction for signals (100, 400). Panel B represents construction for signals (5, 20).<br>
FIG. 3 is a graphical mapping of intensities (x}, x2) into the (R, P) plane, as induced by Equations(30) and (33). Lines of constant X] and x2 are shown. The dark line is the upper limit PU(R) of the range for P.<br>
FIG. 4 illustrates, in tabular form, the results of fold-change estimates R, 68% confidence intervals (Rp, Rp.]), and P-values for significant positive fold-change, for all the measurement pairs of FIG. 1.<br>
FIG. 5 illustrates, in tabular form, the results of the performance of three statistical methodologies utilized for the detection of 1000 genes with actual fold-change of b = 3 against a background of 4000 genes with no change (b = 1), based upon Monte Carlo simulations described in Section 6 herein (i.e., the a priori probability of a gene undergoing the 3-fold change is P\ = 0.2). Specific results are reported for a fixed false positive rate P(0|/&gt;) = 0.3.<br>
FIG. 6 is a scatter of scatter plots in the (R, P) plane generated by: (1) 1000 genes from class 0, the no-change class (Panel A), and (2) 1000 genes from class 1 (Panel B).<br>
FIG. 7 is a line graph depicting a receiver operating characteristic (ROC) for the statistic / = P.<br>
FIG. 8 is a line graph depicting a receiver operating characteristic (ROC) for the statistic / = -P.<br><br>
FIG. 9 is a line graph illustrating the compared sensitivity as a function of Pi = fraction of genes which underwent a change (b = 3), for the R and P statistics. A fixed relative false positive rate P(0 \p) = 0.3 is imposed throughout.<br>
FIG. 10 is a block diagram of a computer system for implementing features of the invention.<br>
FIG. 11 is a flow diagram of a process according to the invention. DETAILED DESCRIPTION OF THE INVENTION<br>
Referring to FIG. 10, a computer system 102 includes a processor 104, memory 106, disk drives 108, a display 110, a keyboard 112, and a mouse 114. The processor 104 can be a personal computer central processing unit (CPU) such as a Pentium® III processor made by Intel® Corporation. Memory 106 includes random access memory (RAM) and read-only memory (ROM). The disk drives 108 include a hard-disk drive and can include floppy-disk drives, a CD-ROM drive, and/or a zip drive. The display 110 is a cathode-ray tube (CRT), although other forms of displays are acceptable, e.g., liquid-crystal displays (LCD) including TFT displays. The keyboard 112 and mouse 114 provide data input mechanisms for a user (not shown). The components 104, 106, 108, 110, 112, and 114 are connected by a bus 116. The computer system 102 can store, e.g., in memory 106, software code containing instructions for controlling the processor 102 to perform functions described below.<br><br><br>
where a is the proportionate variation parameter, such that var(sp) = a2p = (an) .  The<br>
proportionate variation term a is similar to the coefficient of variation c (initially defined by Chen et al, J. Biomed. Optics 2: 364 (1997)). Other noise terms intervene<br><br>
in this model as well, so that the total variance of the noise may remain large even as n -» 0. In the following equations it is assumed that e is normally distributed.<br>
Utilizing the Affymetrix® chips as an example, the combined background and cross-hybridization noise variance a] are computed by taking the variance of the measurements x for all genes signaled as absent by the absence/presence decision algorithm. The proportionate term has been estimated by comparing the highest quartile of intensities in replication experiments.<br>
To illustrate the magnitude of the terms within Equation(6) for a typical Affymetrix® chip-based experiment the median value of the expression levels x is Med(x) « 80, with ac « 25 and a « 0.25. Thus, the median signal over noise ratio Med(x)/ac is only approximately 3. The standard deviation of the background noise alone is a^ « 3-4, so that Ob « ac with cross-hybridization noise dominating background noise by almost an order of magnitude.<br><br><br><br>
where P(x<br>) is given by Equation (9) and where the denominator is now:<br><br><br><br><br><br><br><br>
In this limit, the distribution of R is completely independent of the concentrations, the influence of which has been overwhelmed by the noise. Equation(26) defines a so-called Cauchy distribution (see e.g.. Keeping, E. S., Introduction to Statistical Inference, (Dover, New York, 1995)), which is very broad and does not have a finite mean. One indigenous "pathological" property of the Cauchy distribution is that the mean of many independent samples does not converge, in any sense, to a single number, but rather remains distributed according to Equation (26), regardless of the total number of samples utilized. Conversely, the median over the Cauchy distribution is exactly 1, and the sample median converges to 1, without any of the aforementioned pathologies associated with the median.<br>
The cumulative distribution function of R is given by:<br>
P(R<p tan .></p>
(27) For example, the 90% confidence limits are (0.16, 6.3), showing that the distribution of Equation (26) is very broad, since these bounds will obtain even when RQ = 1, provided that the signal is weak enough when compared to the noise.<br>
Finally, under the transformation \x = log R, the distribution function of Equation (26) becomes completely symmetrical:<br>
^H)"7cosh(«)    "<br>
(28)<br>
so that a logarithmic representation may be useful in some cases, although additional uses of the transformation shall not be pursued further herein.<br>
FIG. 1 displays the a posteriori distribution /R(R) for a series of pairs (x]t x2), for constant standard deviation of both noise terms a{ = a2 = 20. In this figure, the ratio xi/x\ is always 4 (except for the case where both signals are 0), but the signal-to-noise ratio is highly variable. At the very highest signal levels, (*], x2) = (100,400), and fR(R) is strongly peaked about R = 4. Even in this limit, however, visual<br><br>
inspection indicates that the 68% confidence interval (corresponding to a width of two standard deviations for a normal distribution) is roughly (3, 5). This shows that even when the lowest signal-to-noise ratio is 100/20 = 5, the actual fold-change cannot be inferred to a value better than 3 
With decreasing signal-to-noise ratio, the distribution _/^(.tf) not only broadens, but its peak shifts downwards. Thus, in FIG. 1 for the measurement pair (40, 10), the median of the distribution is about 2.2, with the actual maximum occurring very close to a value of 1. This broadening and shifting of the distribution function indicates that for weakening signals, the ratio of the measurements becomes an increasingly less reliable indication of the actual fold-change. Finally, in the limit of measured values both zero, (0, 0), the recovery of Equation (26) indicates that the distribution is very broad, with median R = 1 and a peak at R = 0.<br>
FIG. 2 qualitatively illustrates the behavior of the distribution/R(.fl), which was previously quantified in FIG. 1, through utilization of the following construction: (/) for each pair of values (xh X2), draw a box in the plane about the point (xls x2) with extents ± aE in each dimension, excluding regions which fall along negative axes and (//) then draw a set of lines from the origin to all points in the box. The distribution of the slopes of these lines represents the a posteriori distribution/R(#). 4.        Bayesian Estimation of Fold-Changes<br>
Bayesian estimation of the fold-change R can be performed based on Equation (16) and knowledge of the measurements x, and x2. Initially, the cumulative distribution function is defined:<br>
F{R,)=P(RsR,)=]*flt(R)dR .<br>
(29) F(R) is preferably evaluated by use of numerical integration.    Based upon the numerical values of F(R), the following information may be readily obtained.<br><br>
4.1	Fold-change estimator R:<br>
The median estimator was chosen as an estimator R for the fold-change:<br>
k= Med(R)      ,<br>
(30) that is, the value of R for which F(R) = 111.<br>
Other estimators are possible, for instance the MAP (Maximum a posteriori Probability) or the mean estimators. See e.g., Van Trees, H. L., Detection, Estimation and Modulation Theory, Pari J (John Wiley and Sons), New York, 1998). However, the mean estimator is not used here, as/R(K ) does not have a finite mean (i.e., it will always have a "tail" with dependence l/R2, even in the near-normal limit of Equation(21)). Thus, the median estimator has the dual advantages of robustness and symmetry under the transformation (R -» l/R), and is the one which is adopted herein. Formally, the median estimator is one that reduces, e.g., minimizes, the absolute value of the (estimate-actual value) error term. See e.g., Van Trees, H. L., Detection, Estimation and Modulation Theory, Part I (John Wiley and Sons), New York, 1998).<br>
4.2	Confidence limits Rp and Rj.p:<br>
Given p 
HRP)=P,<br>
(31) F(R,p)=l-p.<br>
(32)<br>
4.3	F-value for significance of change:<br>
The hypothesis R &gt; 1 ("a significant, positive fold-change occurred in experiment 2 relative to 1") may be tested by evaluating the probability of the complementary hypothesis, R ^ 1 and defining this as the P-value P of the hypothesis for significant change. This is simply represented as:<br>
P=F(R=\)  .	(33)<br><br>
Results for all the measurement pairs discussed in connection with FIG. 1 are also illustrated in FIG. 4 below, with confidence limits determined by p = 0.16. Note that having the P-value provides a powerful selection criterion for retaining only those measurement pairs deemed significant. Thus, while all ratios of the measurements shown in FIG. 4 are equal to 4 (except of course for (0, 0)), only the first three entries ((100, 400), (50, 200), (25, 100)) are found to indicate a significant change (i.e., at the 0.05 confidence level). In turn, for each of these aforementioned tabular entries, confidence limits for the fold-change are known. Thus, for the measurement pair (25, 100), the estimate of R = 3.6 is bracketed by the values (2.0, 8.8), showing that in this example the fold-change can not be ascertained more accurately than this reported interval {i.e., actual fold-changes as small as 2 and as large as 8.8 are consistent with the data). 5.       Mapping Intensity Pairs (x,, x2) into the (R, F) Plane<br>
The pair of intensities (xu x2) are mapped by Equations (30) and (33) into pairs of numbers (R, P). This mapping, which provides a significance-weighted representation of the fold-changes, is illustrated in FIG. 3, For the sake of simplicity, at = o2 = a was chosen.<br>
FIG. 3 shows the lines of constant X] and constant xi in the (log(^), P) plane. The figure is symmetrical under the interchange R -» {MR) {i.e., log(R) -&gt; -log(J?)), because the value CT] = (72 was chosen for this particular example. For a given R, the range of P is finite, with an upper bound PJR) such that:<br>
O<p .></p>
(34) Because PU(R) -» 0 as R -&gt; oo or R -» 0, large fold-changes are necessarily correlated with small P-values, albeit always in a finite non-zero range.<br><br><br>
The estimation scheme described hereinabove has been implemented in a C++ program called PFOLD. For a given set of input parameters (xu x2, ci|, a2), specifying the two intensities and the corresponding standard deviations of the noise terms, PFOLD first numerically evaluates the distribution function fc\(R) (see Equation (16)) over a finite range Rmm <r at points on a regular mesh r rmm iar i="0," ...n where rmax and ar are automatically chosen to capture all of the variation function cumulative distribution f equation is then found by numerical integration following which estimators section fold change confidence limits p-value p></r><br>
can be readily evaluated by numerically solving for Equations (30), (31), (32) and (33), respectively.   In finding the roots of these aforementioned equations, a simple bisection method was used.   See e.g., Press, W., et ah, Numerical Recipes in C, Td Edition, p. 353 (Cambridge University Press, Cambridge, 1997). 7.        Monte Carlo Simulations<br>
A central problem in the analysis of expression data is separating significant from non-significant fold-change. In order to evaluate the usefulness of the statistics (R, P) in this process, a series of Monte Carlo simulations (see, e.g., Cowan, G., Statistical Data Analysis, p. 41 (Claredon Press, Oxford, 1998) were preformed which were aimed to approximate an actual experiment. Concentration values n were generated according to a log-normal distribution (see e.g., Cowan, G., Statistical Data Analysis, p. 34 (Claredon Press, Oxford, 1998), by computing:<br>
*=exp(&gt;&gt;),	(36)<br>
where y is a Gaussian random variable, generated with parameters:<br><y>=7.25,	(37)<br>
a, = 1.22,	(38)<br>
where <y> and oy, are the mean and standard deviation of y, respectively. The parameters of Equations (37) and (38) result in a distribution with 25th percentile, median and 75th percentile with values:<br>
«Z5 = 618,	(39)<br>
»5C = 1408,	(40)<br>
n,, = 3208,	(41)<br>
respectively. The choice of a log-normal distribution for the concentration n is dictated by the empirical observation that in actual experiments, the distribution of intensities of present genes is approximately log-normal. For Affymetrix® chips, with an antibody staining procedure following hybridization, the intensities indicated by the percentiles in Equations (39), (40) and (41) are typical.<br><br><br><br>
however, more general regions will be considered as well.<br>
For any choice of D, it is possible to derive an estimate of the probabilities: P(p\Q) = probability that a gene in Class 0 gets assigned to Class 1, P(a\0) = probability that a gene in Class 1 gets assigned to Class 0, by simply counting, in each case, the number of instances (R, P) which belong or, alternately, do not belong to D.   If in turn, values are assumed for the a priori probabilities that, in a large set of genes, some will undergo no fold-change, and others a fold-change of 3 (for the sake of simplicity, it is assumed that no other fold-changes are possible outside of 1 or 3),<br>
Po = a priori probability that a gene is in class 0, Pi = a priori probability that a gene is in class 1. Bayes" theorem may then be utilized to obtain the a posteriori probabilities:<br>
P(p\0) = probability that a gene assigned a significant fold-change did not really change,<br><br><br>
The two quantities which are of greatest interest include:<br>
P(a/l) = absolute false negative rate,	(51)<br>
P(0\p) = relative false positive rate.	(52)<br>
The definitions of Equations (51) and (52) are not symmetrical because the calculation of P(0\p) requires the value of the prior Plt whereas the computation of P(a/l) does not. The absolute false negative rate is a measure of the fraction of all expressed genes that will be missed by the detection scheme at a given stringency. The relative false positive rate, on the other hand, refers to the fraction of the detected genes that is misclassified, and which did not in fact undergo a change. Therefore, the false negative rate, as defined herein, is thus a measure of the efficiency (see e.g., Cowan, G., Statistical Data Analysis, p. 47 (Claredon Press, Oxford, 1998), or sensitivity of the detection scheme (i.e., the smaller value is preferred), while the false positive rate is a measure of the purity (see e.g.. Cowan, G., Statistical Data Analysis, p. 47 (Claredon Press, Oxford, 1998) or selectivity of the scheme (i.e., the smaller value is preferred). 7.2     Results of experimental simulation:<br>
In FIG. 6 (Panels A and B), there is a comparison of scatter plots within the (R, P) plane generated by 1000 genes from Class 0 (the no-change class) and 1000 genes in Class 1 (the genes which exhibited a 3-fold change), respectively. In order to facilitate understanding of this concept, the following equations are based upon a scenario with an a priori probability of a gene changing of:<br>
P, = 0.2.	(53)<br>
This defines a configuration with 1000 genes changing 3-fold, against a background of 4000 unchanging genes.<br>
A method of choice for displaying the dependence of error rates on the position of a decision surface is the construction, in graphical format, of the so-called receiver operating characteristic (ROC) of the decision scheme. See e.g.. Van Trees, H. L., Detection, Estimation and Modulation Theory, Part I (John Wiley and Sons), New<br><br>
York, 1998). The ROC enables one to clearly visualize the tradeoff between reducing, e.g., minimizing, false positive rates, thereby increasing the selectivity of the scheme, and reducing, e.g., minimizing, false negative rates, thereby also increasing the overall sensitivity of the given scheme.<br>
In FIG. 7, the receiver operating characteristic (ROC) are plotted which are obtained when one use as statistic for a decision boundary the fold-change itself, l = R. Thus, in the previous case, the acceptance region is defined simply as:<br>
0=1**4	(54)<br>
so that the decision surface is a vertical line within the (R, P) plane. In FIG. 7, the false positive rate P(0\p) and the false negative rate i*(a|l) are plotted as a function of Rc. It can be seen that as the stringency of the test for acceptance is increased (i.e., increasing Rc), the relative false-positive rate decreases (i.e., the purity of the detected sample increases), but that there is also a concomitant increase in the absolute false negative rate as well. In addition, FIG. 8 illustrates the receiver operating characteristic (ROC) which is obtained for the statistic / = -P.<br>
FIG. 5 illustrates in tabular form the performance of the three statistical methodologies utilized for the detection of 1000 genes with actual fold-change of b = 3, against a background of 4000 genes with no fold-change of b = 1, based upon the Monte Carlo experimental simulations described in Section. 7 (i.e., the a priori probability of a gene undergoing the 3-fold change is Pi = 0.2). Specific results are reported for a fixed false positive rate P(Q\p) = 0.3. FIG. 9 illustrates the compared sensitivity as a function of Pi = fraction of genes which underwent a change (b = 3), for the R and P statistics. A fixed relative false positive rate P(0 \p) = 0.3 is imposed throughout.<br>
The results illustrated in FIG. 5 and FIG. 7 show that the use of the P-value (f = -/ statistic) instead of, or in conjunction with, the fold-change R, can markedly increase the sensitivity of the various ranges of parameters which were, heretofore, extremely hard to detect.<br><br>
In conclusion, due to the low signal-to-noise ratio which is inherent in expression data, this noise must be carefully accounted for. The PFOLD algorithm disclosed herein offers a theoretical and practical framework for dealing with noise. For example, the PFOLD algorithm allows for two important metrics for fold-change of the level of gene expression: (i) the P-value, which reflects the overall "quality" of the ratio; and (ii) R, which reflects the "quantity" of the fold-change in the expression of the gene(s). In addition, the PFOLD p-statistic is essential for quantifying small populations of changing genes and/or small fold-changes of the expression level of such genes.<br><br><br><br><br>
(e.g., all the levels) of gene expression that may be derived from the measurements obtained of levels of gene expression in one or more cells or tissue types represented in the at least one array.<br>
At stage 152, an experimental noise associated with intensity of hybridization signal for each gene in the at least one array is defined. This noise is experimental noise, being variations in observed levels on chips or other microarrays rather than biological noise, which is the variation of expression level seen in biological systems.<br>
At stage 154, the defined experimental noise is used to define an analytical probability distribution function (pdf) describing distribution values of intensity for each gene. The noise is assumed to be Gaussian and the analytical pdf is defined using Bayes Theorem. The analytic pdf is a continuous function.<br>
At stage 156, the analytic pdfs are used to derive an analytical joint pdf describing possible ratios or fold changes for differentially-expressed genes or gene products in the at least one array. The analytical joint pdf describing possible ratios or fold changes may be derived for any differentially-expressed genes or gene products.<br>
At stage 158, the joint pdfs are applied, using experimentally-derived intensities and noise values from the genes on the at least one array, to determine values related to fold changes associated with the genes. The fold changes in the concentration of gene transcripts are estimated. The confidence limits on the fold change given specific confidence intervals are established. A p-value, or quality metric associated with the fold change estimate is also derived. This value represents the probability that a fold change could be less than 1 when the estimate is greater than 1, or the probability that the fold change is greater than 1 when the estimate is less than 1.<br>
The estimated fold change represents the difference in level of gene expression observed.   The total variance (noise) may still be high even as the concentration of transcript goes to zero. EQUIVALENTS<br>
From the foregoing detailed description of the specific embodiments of the invention, a unique mathematically-derived a posteriori distribution for the estimation<br><br>
of alterations in gene expression has been described. Although particular embodiments have been disclosed herein in detail, this has been done by way of example for purposes of illustration only, and is not intended to be limiting with respect to the scope of the appended claims which follow. In particular, it is contemplated that various substitutions, alterations, and modifications will become apparent to those skilled within the art from the foregoing description and accompanying figures and may be made to the invention without departing from the spirit and scope of the invention as defined by the claims. Such modifications are intended to fall within the scope of the appended claims.<br>
Also, due to the nature of software, functions described above can be implemented using software, hardware, firmware, hardwiring, or combinations of any of these. Items implementing functions may also be physically located at various positions, including being distributed such that portions of functions are implemented at different physical locations.<br><br><br>
WE CLAIMS:<br>
1.	A method for determining differences in the level of gene expression in at least<br>
one array of genes comprising:<br>
(a)	defining experimental noise associated with intensity of hybridization signal for each gene in the at least one array;<br>
(b)	determining an analytical probability distribution function describing distribution values of intensity for each gene on the bases of said experimental noise by applying Bayes Theorem.<br>
(c)	deriving an analytical joint probability distribution flmcfion describing at least one of a possible ratio and a fold change for at least one differentially-expressed gene on the basis of said analytical probability distribution fiinction.<br>
(d)	by applying the joint probability distribution function of each differentially-expressed gene and gene product with experimentally-derived intensities and noise values from the genes to determine values related to fold changes of the genes in at least an array of genes.<br><br>
2.	The method as claimed in claim 1 wherein the determined values has estimated fold changes in the concentration of gene transcripts.<br>
3.	The method as claimed in claim 2 wherein the determined values has a quality metric associated with at least one of the estimated fold changes.<br>
4.	The method as claimed in claim 3 wherein the quality metric represents at least one of a probability that a fold change is less than 1 when an estimated fold change is greater than 1, and a fold change is greater than 1 when the estimated fold change is less than 1.<br>
5.	The method as claimed in claim 1 wherein the determined values has confidence limits on the fold change given specific confidence intervals.<br><br>
6.	The method as claimed in claim 1 wherein the analytic probability distribution<br>
function is defined assuming that the noise is Gaussian.<br>
7.	The method as claimed in claim 6 wherein the analytic probability distribution function is a continuous function.<br>
8.	The method as claimed in any of claims 1 to 7 wherein steps (a) to (d) are performed by a computer.<br>
9.	The method as claimed in claim 1 wherein the values related to fold changes of the genes has p-values.<br><br>
10.	The method as claimed in claim 1 wherein determining the experimental noise comprises modeling noise in terms of at least cross-hybridization.<br>
11.	The method as claimed in claim 10 wherein modeling noise includes modeling noise in terms of cross-hybridization, background, and coefficient of variation contributions.<br>
12.	The method as claimed in claim 1 wherein the analytical joint probability distribution iunction is an estimated ratio of differential gene expression levels.<br>
13.	A method for determining differences in the level of gene expression in at least<br>
one array of genes substantially as hereinbefore described with reference to the<br>
accompanying drawings.</y></y></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgYWJzdHJhY3QtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che abstract-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMS1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1621-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="201605-magnetically-responsive-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="201607-drive-mechanism-for-a-vehicle-especially-a-multilane-electromobile.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>201606</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1621/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Aug-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Nov-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. AVENTIS PHARMACEUTICALS INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Route 202-206 P.O. Box 6800 Bridgewater, NJ 08807-0800</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THEILHABER, Joachim</td>
											<td>10 Sacramento Place Cambridge, MA 02138</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BUSHNELL, Steven</td>
											<td>41 South Street Medfield, MA 02052</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FUCHS, Rainer</td>
											<td>40 Bowker Drive Sudbury, MA 02193</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G06F 19/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-05-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/135,853</td>
									<td>1999-05-25</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/577,634</td>
									<td>2000-05-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/201606-a-method-for-determining-differences-in-the-level-of-gene-expression by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:49:47 GMT -->
</html>
