<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/201908-an-isolated-nucleic-acid-molecule-and-a-process-of-preparing-a-host-cell-which-contains-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:23:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 201908:AN ISOLATED NUCLEIC ACID MOLECULE AND A PROCESS OF PREPARING A HOST CELL WHICH CONTAINS THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED NUCLEIC ACID MOLECULE AND A PROCESS OF PREPARING A HOST CELL WHICH CONTAINS THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides isolated nucleic acids molecules which encode pyruvate decarboxylase enzymes having improved decarboxylase activity, substrate affinity, thermostability, and activity at different pH. The nucleic acids of the invention also have a codon usage which allows for high expression in a variety of host cells. Accordingly, the invention provides recombinant expression vectors containing such nucleic acid molecules, recombinant host cells comprising the expression vectors, host cells further comprising other ethanologenix enzymes, and methods for producing useful substances, e.g., acetaldehyde and ethanol, using such host cells.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>CLONING AND SEQUENCING OF PYRUVATE DECARBOXYLASE (PDC) GENES FROM BACTERIA AND USES THEREFOR<br>
Related Information<br>
This application claims priority to U.S. provisional application number 60/288,638, entitled "High-Level Production of Active Sarcina ventriculi Pyruvate Decarboxylase in Recombinant Bacillus megatehum^; U.S. provisional application number 60/288,671, entitled ^'Cloning, Expression, and Characterization of Pyruvate Decarboxylase from the Acid-Tolerant, Anaerobic Gram-Positive Bacterium Sarcina ventriculi Goodsir"; U.S. provisional application number 60/288,698, entitled '^Acetobacterpasteurianus Pyruvate Decarboxylase: Biochemical, Genetic, and Physiological Properties"; U.S. provisional application number 60/288,622, entitled "Biochemical and Biophysical Characterization of Pyruvate Decarboxylase from the Acetic Acid Bacterium Acetohacter pasteurianus'; and U.S. provisional application number 60/288,699, entitled 'Pyruvate Decarboxylase: A Key Enzyme for the Oxidative Metabolism of Lactic Acid hy Aceiobacier pasteuriamis'"; all of which were filed on May 4, 2001 and are incorporated herein in their entireties by this reference. The contents of all patents, patent applications, and references cited throughout this specification are hereby incorporated by reference in their entireties.<br>
Government Sponsored Research<br>
This work was supported, in part, by grants from the U.S. Department of Energy's National Renewable Energy Laboratory (ZDH-9-29009-04), Energy Biosciences Program (FG02-96ER20222), and the Florida Agricultural Experiment Station.<br>
Background of the Invention<br>
Many environmental and societal benefits would result from the replacement of petroleum-based automotive fuels with renewable fuels obtained from plant materials (Lynd e( ai, (1991) Science 251:1318-1323; Olson et al. (1996) Enzyme Microh. Technol. 18:1-17; Wymane/a/., {\99S)Am€r. Chem. Sac. Symp. 618:272-290). Each year, the United States bums over 120 billion gallons of automotive fuel, roughly equivalent to the total amount of imported petroleum. The development of ethanol as a renewable alternative fuel has the potential to eliminate United States dependence on imported oil, improve the environment, and provide new employment (Sheehan, (1994) ACS Symposium Series No. 566, ACS Press, pp 1-53).<br>
In theory, the solution to the problem of imported oil for automotive fuel appears quite simple. Rather than using petroleum, a finite resource, ethanol, a renewable resource, can be produced efficiently by the fermentation of plant material. Indeed,<br><br>
Brazil has demonstrated the feasibility of producing ethanol and the use ot ethanol as a primary automotive fuel for more than 20 years. Similarly, the United States produces over 1.2 billion gallons of fuel ethanol each year. Currently, fuel ethanol is produced from corn starch or cane syrup utilizing either Saccharomyces cerevisiae or Zymomonas mohilis (Z mohilis). However, neither of these sugar sources can supply the volumes needed to realize a replacement of petroleum-based automotive fuels. In addition, both cane sugar and com starch are relatively expensive starting materials, which have competing uses as food products.<br>
Moreover, these sugar substrates represent only a fraction of the total carbohydrates in plants. Indeed, the majority of the carbohydrates in plants are in the form of lignocellulose, a complex structural polymer containing cellulose, hemicellulose, pectin, and lignin. Lignocellulose is found in, for example, the stems, leaves, hulls, husks, and cobs of plants. Hydrolysis of these polymers releases a mixture of neutral sugars including glucose, xylose, mannose, galactose, and arabinose. No known natural organism can rapidly and efficiently metabolize all of these sugars into ethanol.<br>
Nonetheless, in an effort to exploit this substrate source, the Gulf Oil Company developed a method for the production of ethanol from cellulose using a yeast-based process termed simultaneous saccharification and fermentation (SSF) (Gauss et ai (1976) U.S.P.N. 3,990,944). Fungal cellulase preparations and yeasts were added to a slurry of the cellulosic substrate in a single vessel. Ethanol was produced concurrently during cellulose hydrolysis. However, Gulfs SSF process has some shortcomings. For example, the cell cycle time for yeast is relatively long (24-36 hours) and they are unable to ferment complex sugars. Further, fungal cellulases have to be added which have been considered, thus far, to be too expensive for use in large scale bioethanol processes (Himmel et al, (1997) Amer. Chem. Soc. pp. 2-45; Ingram et ai, (1987) Appl Environ. Microbiol 53:2420-2425; Okamoto e/a/., {\99A) Appl Microbiol Biotechnol 42:563-568; Philippidis, G., (1994) Amer. Chem. Soc. pp. 188-217; Saito et al, (1990) J. Ferment. Bioeng. 69:282-286; Sheehan, J., (1994) Amer. Chem. Soc. pp 1-52; Su et al, {\993) Biotechnol Lett. 15:979-984).<br>
Moreover, producing ethanol using other organisms is difficult because pyruvate decarboxylase (PDC), a key enzyme for fermenting ethanol, is common only to plants, yeast, and fungi; and is rarely found in bacteria and is absent in animals (9, 25).<br>
Summary of the Invention<br>
The development of inexpensive enzymatic methods for ethanol fermentation has great potential for improving the efficiency of substrate utilization and the economics of the fermentation process. Accordingly, developing enzymes and.<br><br>
advantageously, biocatalysts that produce such enzymes which can be used for the efficient depolymerization of complex sugars and subsequent rapid fermentation of the sugar into alcohol, would be of great benefit.<br>
Certain microbes, such as Gram-negative and Gram-positive bacteria produce a number of fermentation enzymes, which are capable of catalyzing, for example, the depolymerization of cellulose and hemicellulose to produce fermentable sugars, conversion of a sugar into pyruvate, the substrate pyruvate into acetaldehyde. and finally, the substrate acetaldehyde into ethanol. However, such organisms rarely produce all of the necessarily enzymes at the most desirable levels.<br>
Accordingly, the invention provides genes encoding pyruvate decarboxylases which can be expressed at high levels in a range of organisms. Thus, when expressed in an organism, or cultured with an organism, that produces the remaining key enzymes needed for ethanol fermentation, superior levels of ethanol production can be achieved. These enzymes, for example pyruvate decarboxylase (PDC), alone or in combination with alcohol dehydrogenase (ADH), can be used as a crude extract having a desired mixture of activity or, can be used as a purified composition.<br>
Moreover, a biocatalyst, advantageously a recombinant bacterium, more advantageously a ethanologenic bacterium, can be engineered to express one or more of these enzymatic activities in particular amounts sufficient for fermenting a sugar(s). Such a biocatalyst is suitable for the efficient degradation of complex sugars and subsequent fermentation into alcohol by a process known as simultaneous saccharification and fermentation (SSF).<br>
The present invention is based, at least in part, on the discovery of key enzyme-encoding genes of ethanol fermentation in bacteria. In particular, the identification of ihQ pdc gene of Zymobacter palmae. Acetohacterpasteurianus, and Sarcina ventriculi has been achieved. These genes have been determined to encode pyruvate decarboxylase enzymes having superior pyruvate decarboxylase activity, substrate affinity, for, e.g., pyruvate, as well as thermostability, and superior activity at different pH. Still further, the pdc genes of the invention have a codon usage that affords for their high expression in a range of organisms.<br>
Accordingly, in one aspect, the invention provides isolated nucleic acid molecules encoding pyruvate decarboxylase polypeptides (PDC) or biologically active portions thereof, as well as nucleic acid fragments suitable as primers or hybridization probes for the detection of PDC-encoding nucleic acids.<br>
In one embodiment, an pyruvate decarboxylase ipdc) nucleic acid molecule of the invention is at least about 50%, 60%, 70%, 75%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%&gt;, 98%), or more<br><br>
identical to the nucleotide sequence {e.g., when compared to the overall length of the nucleotide sequence) shown in SEQ ID N0:1, 3, 5, or a complement thereof.<br>
In a particular embodiment, the isolated nucleic acid molecule includes the nucleotide sequence shown in SEQ ID N0:1, 3, 5, or a complement thereof.<br>
In another embodiment, apc/c nucleic acid molecule includes a nucleic acid sequence encoding a polypeptide having an amino acid sequence sufficiently homologous to the amino acid sequence of SEQ ID N0:2, 4, or 6. In a particular embodiment, apt/c nucleic acid molecule includes a nucleotide sequence encoding a (PDC) polypeptide having at least about 50%, 60%, 70%, 75%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%. 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or more identity (e.g., when compared to the overall length of the amino acid sequence) to the amino acid sequence shown in SEQ ID N0:2, 4, or 6.<br>
In one particular embodiment, an isolated nucleic acid molecule encodes the amino acid sequence of the pyruvate decarboxylase enzyme of Zymohacier palmae having the amino acid sequence of SEQ ID NO: 2.<br>
In another particular embodiment, an isolated nucleic acid molecule encodes the amino acid sequence of the pyruvate decarboxylase enzyme of Acetohacter pasteurianus having the amino acid sequence of SEQ ID NO: 4.<br>
In yet another particular embodiment, an isolated nucleic acid molecule encodes the amino acid sequence of the pyruvate decarboxylase enzyme ofSarcina ventriculi having the amino acid sequence of SEQ ID NO: 6.<br>
In another particular embodiment, the nucleic acid molecule is at least about 1600 nucleotides in length and encodes a polypeptide having pyruvate decarboxylase activity (as described herein).<br>
In a more particular embodiment, the invention provides a plasmid, pJAM3440, encoding dipdc gene derived from Zymobacter palmae represented by a deposit with the<br>
American Type Culture Collection designated as deposit number ATCC	. In a<br>
related embodiment, the invention provides a plasmid, pJAM304, encoding ^pdc gene derived from Acetobacter pasteurianus represented by a deposit with the American<br>
Type Culture Collection designated as deposit number ATCC	. In another related<br>
embodiment, the invention provides a plasmid, pJAM419, encoding ?ipdc gene derived<br>
from Sarcina ventriculi represented by a deposit with the American Type Culture<br>
Collection designated as deposit number ATCC	.<br>
Another embodiment of the invention features nucleic acid molecules, advantageously pyruvate decarboxylase nucleic acid molecules, which specifically detect pyruvate decarboxylase nucleic acid molecules {i.e., pdc gene(s)) relative to nucleic acid molecules encoding non-pyruvate decarboxylase (PDC) polypeptides. For example, in one embodiment, such a nucleic acid molecule is at least 50, 60, 70, 80, 90,<br><br>
100, 150,200,300,400,500,500-1000, 1000-1500, 1500-1500 or more nucleotides in length and/or hybridizes under stringent conditions to a nucleic acid molecule comprising the nucleotide sequence shown in SEQ ID NO: 1, 3, or 5, or a cohiplement thereof. It should be understood that the nucleic acid molecule can be of a length within a range having one of the numbers listed above as a lower limit and another number as the upper limit for the number of nucleotides in length, e.js'., molecules that are 60-80, 300-1000, or 150-400 nucleotides in length.<br>
In particular embodiments, the nucleic acid molecules are at least 15 (e.g., contiguous) nucleotides in length and hybridize under stringent conditions to the nucleotide sequence of SEQ ID N0:1, 3, or 5. Accordingly, the invention provides a method for detecting the presence ofsipdc nucleic acid of the invention using the foregoing nucleic acid.<br>
In other particular embodiments, the nucleic acid molecule encodes a naturally occurring allelic variant of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, 4, or 6, wherein the nucleic acid molecule hybridizes to a nucleic acid molecule comprising SEQ ID NO.i, 3, or 5, respectively, under stringent conditions.<br>
In another embodiment, the nucleic acid molecule of the invention is in a vector and may be optionally linked to a surrogate promoter and/or additional nucleic acid sequences encoding a heterologous polypeptide.<br>
In a particular embodiment, the invention provides a host cell containing a nucleic acid molecule of the invention, for example, contained in a vector or stably integrated into the host cell genome.<br>
In one particular embodiment, the host cell comprises a heterologous nucleic acid sequence encoding pyruvate decarboxylase derived from a bacterial cell such as Zymohacter palmae, Acetobacter pasteurianus, ox Sarcirta ventriculi, for example, as provided, in SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5.<br>
In another embodiment, the host cell containing a pdc gene can be ethanologenic, for example, naturally ethanologenic and/or further comprise an ethanologenic gene(s) encoding alcohol dehydrogenase, glucanase, secretase, or a combination thereof. In a related embodiment, the host cell is suitable for fermenting ethanol from a sugar. In a particular embodiment, the host cell is a recombinant ethanologenic host cell comprising a heterologous nucleic acid encoding a PDC shown in SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6. The heterologous nucleic acid can be under the control of exogenous surrogate promoter.<br>
The aforementioned host cell can be a Gram-negative bacterial cell or a Gram-positive bacterial cell.<br><br>
A Gram-negative host cell of the invention can be, e.g., Gluconohacter, Rhizobium, Bradyrhizobium, Alcaligenes, Rhodobacter, Rhodococcus. Azospirillum, Rhodospirillum, Sphingomonas, Burkholderia, Desulfomonas, Geospirillum, Succinomonas, Aeromonas, Shewanella, Halochromatiuni Citrobacier, Escherichia, Klebsiella, Zymomonas Zymobacter, or Acetohacier.<br>
A Gram-positive host cell of the invention can be, e.g., Fibrohacler, Acidobacter, Bacteroides, Sphingobacterium, Actinomyces, Corynebacterium, Nocardia, Rhodococcus, Propionibacterium, Bifidobacterium, Bacillus, Geobacillus, Paenibacillus, Sulfobacillus, Clostridium, Anaerobacter, Eubacterium, Streptococcus, Lactobacillus, Leuconostoc, Enterococcus, Lactococcus, Thermobifida, Cellulomonas, or Sarcina,<br>
In another aspect, the invention provides an isolated polypeptide having at least about 50%, 60%, 70%, 75%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, or more identity (c^^^, when compared to the overall length of the amino acid sequence) to the amino acid sequence shown in SEQ ID N0:2, 4, or 6,<br>
In one embodiment, the isolated polypeptide of the invention has the amino acid sequence of SEQ ID N0:2.4, or 6.<br>
In a related embodiment, the isolated polypeptide has pyruvate decarboxylase activity. The pyruvate decarboxylase of the invention can be selected for improved activity, e.g., pyruvate decarboxylase activity, but also, for example, improved codon usage, substrate (e.g., pyruvate) affinity, thermal stability, and/or activity at a certain pH. Such a pyruvate decarboxylase of the invention can be an altered or chimeric polypeptide to achieve any of the aforementioned properties. In addition, the polypeptide can further comprises heterologous amino acids, for example, an immunotag for purification or detection.<br>
In another aspect, the invention provides an antibody which selectively binds to a polypeptide of invention (or fragment thereof), for example, a pyruvate decarboxylase enzyme as shown, for example, in SEQ ID N0:2, 4, or 6. Accordingly, the invention provides a method for detecting a pyruvate decarboxylase of the invention using such an antibody.<br>
In another aspect, the invention provides a method for producing a pyruvate decarboxylase of the invention by expressing in a host cell one of the foregoing nucleic acids of the invention under suitable culture conditions. The nucleic acid may be altered or mutated to improve the codon usage of the nucleic acid or decarboxylase activity of the encoded product (e.g., thermal stability, substrate affinity, or activity at various pH).<br><br>
In another aspect, the invention provides a method for producing acetaldehyde )y culturing one of the aforementioned hosts under conditions whereby pyruvate lecarboxylase is expressed at sufficient levels such that acetaldehyde is produced from pyruvate. In a related embodiment, the method for producing acetaldehyde is performed by contacting a cell lysate obtained from the above host cell under conditions whereby acetaldehyde is produced from pyruvate. Accordingly, the invention provides enzyme extracts having improved decarboxylase activity, and having, for example, thermal stability, activity at various pH, and/or superior substrate affinity.<br>
In another aspect, the invention provides a method for producing ethanol by culturing the above host cell under conditions whereby pyruvate decarboxylase and alcohol dehydrogenase are expressed at sufficient levels such that ethanol is produced as a primary fermentation product.<br>
In another as aspect, the invention provides a method for selecting a pyruvate decarboxylase enzyme with improved decarboxylase activity (e.g., improved affinity for pyruvate, thermal stability, activity at different pH) by comparing the amino acid sequence of a pyruvate decarboxylase with the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6; and altering at least one amino acid residue of the pyruvate decarboxylase to have identity with the corresponding amino acid residue of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, such that a polypepfide with improved pyruvate decarboxylase activity is achieved.<br>
In a related embodiment, the invention provides a method for selecting a pyruvate decarboxylase enzyme for expression in a recipient host cell by comparing the' nucleic acid sequence encoding a pyruvate decarboxylase with the nucleic acid sequence of SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5; and altering at least one codon of the nucleic acid encoding the pyruvate decarboxylase enzyme to have identity with the corresponding codon of SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5, such that improved expression of the altered nucleic acid encoding pyruvate decarboxylase enzyme is achieved in the host cell.<br>
In another related embodiment, the invention provides a method for selecting a pyruvate decarboxylase enzyme with improved expression in a recipient host cell by comparing the nucleic acid sequence encoding a pyruvate decarboxylase with the codon usage of the recipient host cell and altering at least one codon of the nucleic acid encoding the pyruvate decarboxylase enzyme to correspond with the codon usage of the recipient host cell such that improved expression of the altered nucleic acid encoding pyruvate decarboxylase enzyme is achieved in the host cell.<br>
In yet another related embodiment, the invention provides a method for selecting a pyruvate decarboxylase enzyme with improved expression in a recipient host cell by comparing the nucleic acid sequence encoding a pyruvate decarboxylase with the codon<br><br>
usage of the recipient host cell and altering the recipient host cell to recombinantly produce at least one tRNA corresponding to a codon of the nucleic acid encoding the pyruvate decarboxylase enzyme such that improved expression of the nucleic acid encoding pyruvate decarboxylase enzyme is achieved in the altered host cell.<br>
Other features and advantages of the invention will be apparent from the following detailed description and claims.<br>
Brief Description of the Drawings<br>
Figure 1 shows a schematic representation of the Zymohacter palmae pdc gene. Plasmid pJAM3400 carries a 6-kb Bam\\\ fragment of Z palmae genomic DNA blunt-ended and ligated into plasmid vector pLITMUS28. The remaining plasmids were derived from pJAM3400 and were used for DNA sequence analysis of a 2.9-kb region that included thepc/c gene. Plasmid pJAM3440 was used to produce the Z palmae PDC polypeptide in recombinant E. coli. The arrowhead indicates the direction o^pdc gene transcription.<br>
Figure 2 shows a schematic representation of the Acetobacter pasteurianus pdc gene. The plasmid pJAM301 carries a 6,337-bp AatW fragment of ^. pasteurianus genomic DNA ligated into the Xhol site of plasmid vector pLlTMUS28. The 3'- and 5'-overhangs generated by AatW and Xho\ were blunt-ended with Vent DNA polymerase prior to ligation. The remaining plasmids were derived from pJAM30l and were used for DNA sequence analysis. Plasmid pJAM304 was used to produce the A. pasteurianus PDC protein in recombinant £ coli. Arrowheads indicate the directions of transcription and translation of the deduced ORFs.<br>
Figure 3 shows a schematic representation of the S. ventriculi pdc gene, various plasmids produced for sequencing and characterizing the pdc locus and the plasmids pJAM413, pJAM419, and pJAM418 which comprise the pdc open reading frame and are suitable for the high expression of the pdc gene in bacteria.<br>
Figure 4 shows the nucleic acid sequence (SEQ ID NO: 1) and amino acid sequence (SEQ ID NO: 2) for thepc/c gene and gene product of Zymobacter palmae.<br>
Figure 5 shows the nucleic acid sequence (SEQ ID N0:3) and amino acid sequence (SEQ ID N0:4) for thept/c gene and gene product of Acetobacter pasteurianus. A putative promoter is double underlined with the -35 and -10 promoter consensus sequence indicated directly below the sequence predicted for PDC and highlighted by capital letters in the nucleic acid sequence of pdc. Arrowheads above the DNA sequence indicate transcription start sites. Underlined bases indicate a "Shine-Dalgamo" ribosome binding-site. An asterisk indicates translation stop codon. Arrows below the DNA sequence indicate a stem-loop structure, which can facilitate p-independent transcription termination.<br><br>
Figure 6 shows the nucleic acid sequence (SEQ ID N0:5) and amino acid sequence (SEQ ID N0:6) for the pdc gene and gene product of Sarcina ventriculi.<br>
Figure 7 shows a graph depicting the thermostability of different bacterial PDC enzymes. 'Recombinant' Zmo (•), Zpa (■), Apa (o) and Sve (a) PDC polypeptides were pre-incubated at the temperatures indicated in 50 mM sodium citrate buffer at pH 5.0 with 1 mM TPP and 1 mM MgCb for 30 min, cooled to O'C, and assayed for residual activity at 25X in the same buffer.  100% is activity after pre-incubated at OT. SvePDC was purified from recombinant E. coli BL21-CodonPlus-RIL (pJAM419).<br>
Figure 8 shows a graph depicting the effect of pH on the activity of bacterial PDC enzymes. 'Recombinant' ZpaPDC (•) and ApaPDC (■). 'Native' ZpaPDC (o) and ApaPDC (a). Activity was measured at 25'C in 50 mM sodium citrate buffer from pH 4.0 to 5.0 and 50 mM sodium phosphate buffer from pH 5.5 to pH 8.0.  100% is activity at optimal pH.<br>
Figure 9 shows a multiple amino acid sequence alignment of the deduced PDC polypeptide of Z. palmae (SEQ ID N0:2) aligned with selected PDC polypeptides, i.e., Z mohilis (SEQ ID N0:8), A. pasteurianns (SEQ ID N0:4), S. ventriculi (SEQ ID NO:6), Z mays (SEQ ID N0:9), and S. cerevisiac (SEQ ID NO; 10). Functionally conserved (black highlight) and semi-conserved (gray highHghl) amino acid residues. Gaps introduced in alignment (--). Residues within 0.4 nm of the Mg^^ and TPP binding sites of the yeast and Z mobilis PDC polypeptides (*). Yeast PDCl residues forming hydrogen bonds with pyruvamide (■). Double underlined residues are conserved among TPP-dependent enzymes. Abbreviation and GenBank or SwissProt accession number: Zpa, Z. palmae, AF474I45; Apa, A. pasteurianus, AR368435; Sec, Saccharomyces cerevisiae, P06169; Sve, Sarcina ventriculi, AF354297; Zma, Zea mays, P28516; Zmo. Zymomonas mobilis, P06672.<br>
Detailed Description of the Invention<br>
In order for the full scope of the invention to be clearly understood, the following definitions are provided.<br>
/.	Definitions<br>
As used herein the term "alcohol dehydrogenase" is intended to include the enzyme capable of converting acetaldehyde into an alcohol, advantageously, ethanol.<br>
The term "chimeric" includes a mutant or altered PDC where an entire domain derived from another PDC is engineered (fused, exchanged) with a corresponding domain in a PDC, using, e.g., genetic engineering.<br><br>
Ihe term "codon usage" is intendea to incluae analyzing a given nucieic acia being considered for expression in a recipient host organism (or acellular extract thereof) for the occurrence or "usage" of certain codons that the host organism will require (advantageously at sufficient levels) in order to translate the nucleic acid into a corresponding polypeptide. Based on such observations the recipient host may be recombinantly supplemented with any necessary codon. Alternatively, another host can be selected with superior codon usage or the nucleic acid can be altered to no longer comprise a limiting codon {e.g.. by introducing a silent mutation(s)).<br>
The term 'decarboxylase activity" is intended to include the ability of a polypeptide to enzymatically convert pyruvate into acetaldehyde. Typically, the activity of a selected polypeptide encompasses the total enzymatic activity associated with the produced polypeptide, comprising, e.g,, the superior substrate affinity of the enzyme, thermostability, stability at different pHs, or a combination of these attributes.<br>
The term 'derived from" is intended to include the isolation (in whole or in part) of a polynucleotide segment from an indicated source. The term is intended to include, for example, direct cloning, PCR amplification, or artificial synthesis from, or based on, a sequence associated with the indicated polynucleotide source.<br>
The term "ethanologenic" is intended to include the ability of a microorganism to produce ethanol from a carbohydrate as a primary fermentation product. The term is intended to include naturally occurring ethanologenic organisms, ethanologenic organisms with naturally occurring or induced mutations, and ethanologenic organisms which have been genetically modified.<br>
The terms "fermenting" and "fermentation" are intended to include the enzymatic process {e.g., cellular or acellular, e.g., a lysate or purified polypeptide mixture) by which ethanol is produced from a carbohydrate, in particular, as a primary product of fermentation.<br>
The term "gene involved in ethanologenesis" is intended to include any gene capable of conferring on a cell ethanologenic properties or capable of improving any aspect of cellular ethanologenesis, such as, e.g., substrate uptake, substrate processing, ethanol tolerance, etc. Genes involved in ethanologenesis are, e.g., alcohol dehydrogenase, pyruvate decarboxylase, secretory polypeptide/s, and polysaccharases, and these genes, or their homologs, can be derived from any appropriate organism.<br>
The term "glucanase" is intended to include a polypeptide capable of catalyzing the degradation or depolymerization of any linked sugar moiety, e.g.. disaccharides, trisaccharides, oligosaccharides, including, complex carbohydrates, also referred to herein as complex sugars, e.g., cellooligosaccharide and lignocellulose, which comprises cellulose, hemicellulose, and pectin. The terms are intended to include cellulases such as glucanases, including advantageously, endoglucanases but also including, e.g.,<br><br>
exoglucanase, P-glucosidase, cellobiohydrolase, endo-l,4-p-xylanase, p-xylosidase, a-glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase. a-amylase, P-amylase, glucoamylase, pullulanase, P-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination of any of these cellulases.<br>
The term "Gram-negative bacterial cell" is intended to include the art recognized definition of this term. Typically, Gram-negative bacteria include Gluconohacter, Rhizobium, Bradyrhizobium, Alcaligencs, Rhodohacter, Rhodococciis. Azospirillum, Rhodospirillum, Sphingomonas, Burkholderia, Desulfomonas, GeospirUlum, Succinomonas, Aeromonas, Shewanella, Halochromatium, Citrobacter, Escherichia, Klebsiella, Zymomonas (e.g., Zymomonas mobilis), Zymobacter (e.g., Zymobacter palmae), andAcetobacter (e.g., Acetobacterpasteuhanus).<br>
The term "Gram-positive bacteria" is intended to include the art recognized definition of this term. Typically, Gram-positive bacteria include Fibrobacter, Acidobacter, Bacteroides, Sphingobacterium, Actinomyces, Corynehacterium, Nocardia, Rhodococcus, Propionibacterium, Bifidobacterium, Bacillus, Geohacillus, Paenibacillus, Sulfobacillus, Clostridium, Anaerobacter, Euhacterium, Streptococcus, Lactobacillus, Leuconostoc, Entcrococcus, Lactococcus, Thermobifida, Celhilomonas, and Sarcina (e.g. Sarcina ventriculi).<br>
The term "heterologous polypeptide" is intended to include a polypeptide or portion thereof that can be encoded by a heterologous nucleic acid derived from any source, e.g., eukaryotes, prokaryotes, virii, or synthetic nucleic acid fragments.<br>
The term "liomologous" is intended to include a first amino acid or nucleotide sequence which contains a sufficient or minimum number of identical or equivalent amino acid residues or nucleotides, e.g., an amino acid residue which has a similar side chain, to a second amino acid or nucleotide sequence such that the first and second amino acid or nucleotide sequences share common structural domains and/or a common functional activity.<br>
The terms "host cell" and "recombinant host cell" are intended to include a cell suitable for genetic manipulation, e.g., which can incorporate heterologous polynucleotide sequences, e.g., which can be transfected. The cell can be a microorganism or a higher eukaryotic cell, such as an animal cell or a plant cell. The term is intended to include progeny of the cell originally transfected. In particular embodiments, the cell is a bacterial cell, e.g., a Gram-negative bacterial cell or a Gram-positive cell. Particularly, the term recombinant host cell is intended to include a cell that has already been selected or engineered to have certain desirable properties and suitable for further modification using the compositions and methods of the invention.<br><br>
ihe term an "isolatea poiypepuae   {e.g., an isoiaieu or puriueu Diobynineuc enzyme) is substantially free of cellular material or other contaminating polypeptides from the microorganism from which the polypeptide is derived, or substantially free from chemical precursors or other chemicals when chemically synthesized.<br>
The term "nucleic acid" is intended to include nucleic acid molecules, e.g., polynucleotides which include an open reading frame encoding a polypeptide, and can further include non-coding regulatory sequences, and introns. In addition, the terms are intended to include one or more genes that map to a functional locus. In addition, the terms are intended to include a specific gene for a selected purpose. The gene can be endogenous to the host cell or can be recombinantly introduced into the host cell, e.g., as a plasmid maintained episomally or a plasmid (or fragment thereoO that is stably integrated into the genome. In one embodiment, the gene of polynucleotide segment is involved in at least one step in the bioconversion of a carbohydrate to ethanol. Accordingly, the term is intended to include any gene encoding a polypeptide such as a pyruvate decarboxylase, an alcohol dehydrogenase, a secretory polypeptide/s, or a polysaccharase, e.g., a glucanase, or a combination thereof<br>
The phrase 'mutant nucleic acid molecule" or 'mutant gene" is intended to include a nucleic acid molecule or gene having a nucleotide sequence which includes al least one alteration {e.g., substitution, insertion, deletion) such that the polypeptide or polypeptide that can be encoded by said mutant exhibits an activity that differs from the polypeptide or polypeptide encoded by the wild-type nucleic acid molecule or gene.<br>
The phrase "operably linked" means that the nucleotide sequence of the nucleic acid molecule or gene of interest is linked to the regulatory sequence(s) in a manner which allows for expression (t^g., enhanced, increased, constitutive, basal, attenuated, decreased or repressed expression) of the nucleotide sequence, advantageously expression of a gene product encoded by the nucleotide sequence (e.g., when the recombinant nucleic acid molecule is included in a recombinant vector, as defined herein, and is introduced into a microorganism).<br>
The term "pH" is intended to include its art recognized meaning. Typically, the pyruvate decarboxylase enzymes of the invention exhibit decarboxylase activity at a pH of about 4 to about 8, more particularly at a pH of about 5 to about 7, even more particularly at a pH of about 5.5 to about 6.0.<br>
The term "pyruvate decarboxylase" is intended to include the enzyme described herein capable of decarboxylating pyruvate into acetaldehyde. By convention, the term "pdc " refers to a pyruvate decarboxylase gene whereas the term "PDC" refers to apc/c gene product, i.e., a pyruvate decarboxylase polypeptide or enzyme.<br><br>
The term "recombinant nucleic acid molecule"' includes a nucleic acid molecule (e.g., a DNA molecule) that has been altered, modified or engineered such that it differs in nucleotide sequence from the native or natural nucleic acid molecule from which the recombinant nucleic acid molecule was derived (e.g.^ by addition, deletion or substitution of one or more nucleotides). Advantageously, a recombinant nucleic acid molecule (e.g., a recombinant DNA molecule) includes an isolated nucleic acid molecule or gene of the present invention (e.g., an isolated pt/c gene) operably linked vO regulatory sequences.<br>
The term 'secretase' is intended to include any polypeptide/s, alone or in combination with other polypeptides, that facilitate the transport of another polypeptide from the intracellular space of a cell to the extracellular milieu. In one embodiment, the secretory polypeptide/s encompass all the necessary secretory polypeptides sufficient to impart secretory activity to a Gram-negative or Gram-positive host cell.<br>
The term "simultaneous saccharification and fermentation" or "SSF" is intended to include the use of one or more recombinant hosts (or extracts thereof, including purified or unpurified extracts, and if desired, other enzyme additions, e.g., from one or more different sources) for the contemporaneous degradation or depolymerization of a complex sugar and bioconversion of that sugar residue into acetaldehyde and subsequently, if desired, into ethanol by fermentation.<br>
The term "substrate affinity" is intended to include the binding kinetics of an enzyme for a substrate, for example, the KM of the enzyme pyruvate decarboxylase for its substrate pyruvate (or analog thereof). Typically, the pyruvate decarboxylase enzymes of the invention exhibit a substrate affinity {e.g., for pyruvate) having a KM of about 0.1 to about 1, more particularly a KM of about 0.1 to about 0.5, even more particularly a KM of about 0.2 to about of about 0.4.<br>
The term "sugar" is intended to include any carbohydrate source comprising a sugar molecule(s). Such sugars are potential sources of sugars for depolymerization (if required) and subsequent bioconversion to acetaldehyde and subsequently to ethanol by fermentation according to the products and methods of the present invention.<br>
The term "surrogate promoter" is intended to include a polynucleotide segment that can transcriptionally control a gene-of-interest, e.g., a pyruvate decarboxylase gene, that it does not transcriptionally control in nature. In one embodiment, the transcriptional control of a surrogate promoter results in an increase in expression of the gene-of-interest. In another embodiment, a surrogate promoter is placed 5' to the gene-of-interest. A surrogate promoter can be used to replace the natural promoter, or can be used in addition to the natural promoter. A surrogate promoter can be endogenous with regard to the host cell in which it is used or it can be a heterologous polynucleotide sequence introduced into the host cell, e.g., exogenous with regard to the host cell in<br><br>
which it is used. Other promoters suitable for use in bacteria include, e.g., lacZ, T7, and SP6 (see, e.g., Ausubel et al. infra).<br>
The terms "thermal stability" and "thermostability" are used interchangeably and are intended to include the ability of a enzyme {e.g., whether expressed in a cell, present in an cellular extract, cell lysate. or in purified or partially purified form) to exhibit the ability to catalyze a reaction {e.g., the conversion of pyruvate to acetaldehyde) at least at about 20'C, advantageously at about 25°C to 35'C, more advantageously at about 37 C or higher, in particular, at about SO^'C or higher, for example, at least about 60 C or higher.<br>
//.       Isolated Nucleic Acid Molecules and Genes<br>
The present invention features nucleic acid molecules comprising pyruvate decarboxylase genes {pdc) that encode gyruvate decarboxylase polypeptide enzymes (PDC) wherein the nucleic acids have been isolated from Gram-negative and Gram-positive bacteria, for example, the Gram-negative bacteria Zymohacter palmae, Acetobacter pasteurianus and from the Gram-positive bacterium Sarcina veniriculi. Also featured are isolated genomic nucleic acids comprising any one of the above mentioned pyruvate decarboxylase genes {i.e., pdc) but also other flanking regions which comprise regulatory regions {e.g., promoter(s) and ribosome binding sites(s)) as well as other associated genes involved in ethanologenesis, e.g., alcohol dehydrogenase {adh)).<br>
The nucleic acid molecule includes DNA molecules {e.g., linear, circular, cDNA or chromosomal DNA) and RNA molecules {e.g., tRNA, rRNA, mRNA) and analogs of the DNA or RNA generated using nucleotide analogs. The nucleic acid molecule can be single-stranded or double-stranded, but advantageously is double-stranded DNA. The isolated nucleic acid molecule of the invention includes a nucleic acid molecule which is free of sequences which naturally flank the nucleic acid molecule {i.e., sequences located at the 5' and 3' ends of the nucleic acid molecule) in the chromosomal DNA of the organism from which the nucleic acid is derived. In various embodiments, an isolated nucleic acid molecule can contain less than about 10 kb, 5 kb, 4 kb, 3 kb, 2 kb, 1 kb, 0.5 kb, O.l kb, 50 bp, 25 bp or 10 bp of nucleotide sequences which naturally flank the nucleic acid molecule in chromosomal DNA of the microorganism from which the nucleic acid molecule is derived. Moreover, an "isolated" nucleic acid molecule, such as a cDNA molecule, can be substantially free of other cellular materials when produced by recombinant techniques, or substantially free of chemical precursors or other chemicals when chemically synthesized.<br><br>
ThQ pdc genes, as described herein (and italicized by convention), include a nucleic acid molecule (e.g., a DNA molecule or segment thereoO, for example, a polypeptide or RNA-encoding nucleic acid molecule that, in an organism, is separated from another gene or other genes, by intergenic DNA {i.e., intervening or spacer DNA which naturally flanks the gene and/or separates genes in the chromosomal DNA of the organism). A gene can direct synthesis of an enzyme or other polypeptide molecule {e.g., can comprise coding sequences, for example, a contiguous open reading frame (ORF) which encodes a polypeptide) or can itself be functional in the organism. A gene in an organism can be clustered in an operon, as defined herein, wherein the operon is separated from other genes and/or operons by intergenic DNA. Individual genes contained within an operon can overlap without intergenic DNA between the individual genes.<br>
An isolated gene as described herein, includes a gene which is essentially free of sequences which naturally flank the gene in the chromosomal DNA of the organism from which the gene is derived {i.e., is free of adjacent coding sequences which encode a second or distinct polypeptide or RNA molecule, adjacent structural sequences or the like) and optionally includes 5' and 3' regulatory sequences, for example promoter sequences and/or terminator sequences. In one embodiment, an isolated gene includes predominantly coding sequences for a polypeptide {e.g., sequences which encode PDC polypeptides).<br>
In another embodiment, an isolated gene includes coding sequences for a polypeptide {e.g., for a PDC polypeptide) and adjacent 5' and/or 3' regulatory sequences from the chromosomal DNA of the organism from which the gene is derived {e.g.. adjacent 5' and/or 3' pdc regulatory sequences). Advantageously, an isolated gene contains less than about 10 kb, 5 kb, 2 kb, 1 kb, 0.5 kb, 0.2 kb, 0.1 kb, 50 bp, 25 bp or 10 bp of nucleotide sequences which naturally flank the gene in the chromosomal DNA of the organism from which the gene is derived.<br>
In one aspect, the present invention features isolated/?Jc nucleic acid sequences or genes, isolated alcohol dehydrogenase {adh) nucleic acid sequences or genes advantageously derived from a Gram-positive or Gram-negative bacteria. Advantageously, the pdc nucleic acid or gene is derived from a Gram-negative microorganism selected from the group consisting of Gluconobacter, Rhizobium, Bradyrhizobium, Alcaligenes, Rhodobacter, Rhodococcus, Azospirillum, Rhodospirillum, Sphingomonas, Burkholderia, Desulfomonas, Geospirillum, Succinomonas, Aeromonas, Shewanella, Halochromatium, Citrobacter, Escherichia, Klebsiella, Zymomonas (e.g., Zymomonas mobilis), Zymobacter (e.g., Zymobacter palmae), andAcetobacter (e.g., Acetobacterpasteurianus).<br><br>
In another embodiment, the pdc nucleic acid or gene is derived from a Gram-positive microorganism selected from the group consisting ot^Fihrohacler, Acidobacter, Bacieroides, Sphingobacterium, Actinomyces, Corynebacteriimt Nocardia, Rhodococcus, Propionibacterium, Bifidobacterium, Bacillus, Geobacillus, Paenibacillus, Sulfobacillus, Clostridium, Anaerobacter, Eubacterium, Streptococcus, Lactobacillus, Leuconostoc, Enterococcus, Lactococcus, Thermobifida, Celhdomonas, and Sarcina (e.g. Sarcina ventricidi).<br>
In one embodiment, Xh^pdc nucleic acid or gene is derived from Gram-negative bacteria Zymobacter palmae.<br>
In another embodiment, Xhcpdc nucleic acid or gene is derived from the Gram-negative Acetobacter pasteurianus.<br>
In yet another embodiment, ihepdc nucleic acid or gene is derived from the Gram-positive bacterium Sarcina ventriculi.<br>
In another embodiment, an isolated nucleic acid molecule of the present invention comprises a nucleotide sequence which is at least about 60-65%, advantageously at least about 70-75%, more preferable at least about 80-85%, and even more advantageously at least about 90-95% or more identical to a nucleotide sequence set forth as SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5.<br>
In another embodiment, an isolated nucleic acid molecule hybridizes under stringent conditions to a nucleic acid molecule having a nucleotide sequence set forth as SEQ ID NO:K SEQ ID N0:3, or SEQ ID N0:5. Such stringent conditions are known to those skilled in the art and can be found in Current Protocols in Molecular Biology. John Wiley &amp; Sons, N,Y. (1989), 6.3.1-6.3.6. A particular, non-limiting example of stringent (e.g. high stringency) hybridization conditions are hybridization in 6X sodium chloride/sodium citrate (SSC) at about 45°C, followed by one or more washes in 0.2 X SSC, 0.1% SDS at 50-65°C. Advantageously, an isolated nucleic acid molecule of the invention that hybridizes under stringent conditions to the sequence of SEQ ID NO: 1, SEQ ID N0:3, or SEQ ID N0:5 corresponds to a naturally-occurring nucleic acid molecule. Typically, a naturally-occurring nucleic acid molecule includes an RNA or DNA molecule having a nucleotide sequence that occurs in nature.<br>
A nucleic acid molecule of the present invention (e.g., a nucleic acid molecule having the nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5 can be isolated using standard molecular biology techniques and the sequence information provided herein. For example, nucleic acid molecules can be isolated using standard hybridization and cloning techniques {e.g., as described in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual 2nd, ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989) or can be isolated by the polymerase chain reaction using synthetic<br><br>
oligonucleotide primers designed based upon the sequence of SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5. A nucleic acid of the invention can be amplified using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. In another embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule which is a complement of the nucleotide sequence shown in SEQ ID NO:K SEQ ID N0:3, or SEQ ID N0:5.<br>
Additional pc/c nucleic acid sequences are those that comprise the nucleotide sequence of SEQ IDN0:1, SEQ ID N0:3. or SEQ ID N0:5, that encode a homologue of the polypeptide having the amino acid sequence set forth in SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6 (e.g., encode a polypeptide having at least 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95% or more identity to the polypeptide having the amino acid sequence as set forth in SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, and having a substantially identical activity as the polypeptide), hybridize under stringent conditions to all or a portion of a nucleic acid molecule having the nucleotide sequence of SEQ ID NO: 1, SEQ ID N0:3, or SEQ ID N0:5 or to all or a portion of a nucleic acid molecule that encodes a polypeptide having the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, or are complementary to apdc nucleotide sequence as set forth herein.<br>
In yet another embodiment, an isolated/7c/c nucleic acid molecule or gene encodes a homologue of the PDC polypeptide having the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6. Typically, the term "homologue" includes a polypeptide or polypeptide sharing at least about 30-35%, advantageously at least about 35-40%, more advantageously at least about 40-50%, and even more advantageously at least about 60%, 70%, 80%, 90% or more identity with the amino acid sequence of a wild-type polypeptide or polypeptide described herein and having a substantially equivalent functional or biological activity as the wild-type polypeptide or polypeptide. For example, a PDC homologue shares at least about 30-35%, advantageously at least about 35-40%, more advantageously at least about 40-50%, and even more advantageously at least about 60%, 70%, 80%, 90% or more identity with the polypeptide having the amino acid sequence set forth as SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, and has a substantially equivalent flinctional or biological activity {Le,, is a fiinctional equivalent) of the polypeptide having the amino acid sequence set forth as SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6 {e.g., has a substantially equivalent pyruvate decarboxylase activity).<br>
In an embodiment, an isolated pJc nucleic acid molecule or gene comprises a nucleotide sequence that encodes a polypeptide as set forth in any one of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6.<br><br>
In another embodiment, an isolated/?t/c nucleic acid molecule hybridizes to all or a portion of a nucleic acid molecule having the nucleotide sequence set forth in SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5 or hybridizes to all or a portion of a nucleic acid molecule having a nucleotide sequence that encodes a polypeptide having the amino acid sequence of any of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6.<br>
Such hybridization conditions are known to those skilled in the art and can be found in Current Protocols in Molecular Biology. Ausubel et al, eds., John Wiley &amp; Sons, Inc. (1995), sections 2, 4 and 6. Additional stringent conditions can be found in Molecular Cloning: A Laboratory Manual^ Sambrook et ai. Cold Spring Harbor Press, Cold Spring Harbor, NY (1989), chapters 7, 9 and 11. A particular, non-limiting example of stringent hybridization conditions includes hybridization in 4X sodium chloride/sodium citrate (SSC), at about 65-70°C (or hybridization in 4X SSC plus 50% formamide at about 42-50°C) followed by one or more washes in IX SSC, at about 65-70°C. A particular, non-limiting example of highly stringent hybridization conditions includes hybridization in IX SSC, at about 65-70°C (or hybridization in IX SSC plus 50% formamide at about 42-50'^C) followed by one or more washes in 0.3X SSC, at about 65-70°C. A particular, non-limiting example of reduced stringency hybridization conditions includes hybridization in 4X SSC, at about 50-60°C (or alternatively hybridization in 6X SSC plus 50% formamide at about 40-45°C) followed by one or more washes in 2X SSC, at about 50-60°C. Ranges intermediate to the above-recited values, e.g., at 65-70°C or at 42-50°C are also intended to be encompassed by the present invention. SSPE (IX SSPE is 0.15 M NaCl, lOmM NaH2P04, and 1.25 mM EDTA, pH 7.4) can be substituted for SSC (IX SSC is 0.15 M NaCl and 15 mM sodium citrate) in the hybridization and wash buffers; washes are performed for 15 minutes each after hybridization is complete. The hybridization temperature for hybrids anticipated to be less than 50 base pairs in length should be 5-1 OX less than the melting temperature (Tm) of the hybrid, where Tm is determined according to the following equations. For hybrids less than 18 base pairs in length, Tm(°C) = 2(# of A + T bases) + 4(# of G + C bases). For hybrids between 18 and 49 base pairs in length, Tm(°C) = 81.5 + 16.6(logio[Na^]) + 0.41(%G+C) - (600/N), where N is the number of bases in the hybrid, and [Na"^] is the concentration of sodium ions in the hybridization buffer ([Na^] for IX SSC = 0.165 M).<br>
It will also be recognized by the skilled practitioner that additional reagents can be added to hybridization and/or wash buffers to decrease non-specific hybridization of nucleic acid molecules to membranes, for example, nitrocellulose or nylon membranes, including but not limited to blocking agents {e.g., BSA or salmon or herring sperm carrier DNA), detergents (e.g., SDS), chelating agents {e.g., EDTA), Ficoll, PVP and the like. When using nylon membranes, in particular, an additional, non-limiting example<br><br>
ot stringent hybridization conditions is hybridization in 0.25-0.5M NaH2P04, 7% SDS at about 65T, followed by one or more washes at 0.02M NaH2P04, 1% SDS at 65T, see e.g., Church and Gilbert (1984) Proc. Natl. Acad Sci. USA 81:1991-1995, (or, alternatively, 0.2X SSC, 1% SDS). In another embodiment, an isolated nucleic acid molecule comprises a nucleotide sequence that is complementary to a/?Jc nucleotide sequence as set forth herein (e.g., is the full complement of the nucleotide sequence set forth as SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5).<br>
Yet another embodiment of the present invention features mutant or chimeric pdc nucleic acid molecules or genes. Typically, a mutant nucleic acid molecule or mutant gene as described herein, includes a nucleic acid molecule or gene having a nucleotide sequence which includes at least one alteration (e.g., substitution, insertion, deletion) such that the polypeptide or polypeptide that can be encoded by said mutant exhibits an activity that differs from the polypeptide or polypeptide encoded by the wild-type nucleic acid molecule or gene. Typically, a chimeric pdc includes an entire domain derived from another PDC that is engineered (fused, exchanged) with a corresponding domain in a PDC. Advantageoulsy, a mutant nucleic acid molecule or mutant gene {e.g., a muisinipdc gene) encodes a PDC polypeptide having improved activity, e.g.. decarboxylase activity (e.g., substrate affinity (e.g., to pyruvate); thermostability; activity at a different pH; or codon usage (e.g.. for improved expression in the recipient host cell).<br>
///.      Recombinant Nucleic Acid Molecules and Vectors<br>
The present invention further features recombinant nucleic acid molecules (e.g.. recombinant DNA molecules) that include nucleic acid molecules and/or genes described herein (e.g., isolated nucleic acid molecules and/or genes), advantageously pdc genes, more advantageously pc/c genes derived from a Gram-negative or Gram-positive bacterium.<br>
The present invention further features vectors (e.g., recombinant vectors) that include nucleic acid molecules (e.g., isolated or recombinant nucleic acid molecules and/or genes) described herein. In particular, recombinant vectors are featured that include nucleic acid sequences that encode bacterial gene products as described herein, advantageously/?^c gene products, more advantageously pc/c gene products of a Gram-negative or a Gram-positive bacterium, even more advantageously p<ic gene products derived from zymobacterpalmae acetobacterpasteuhanus or sarcina ventriculi.></ic>
The recombinant vector (e.g., plasmid, phage, phasmid, virus, cosmid or other purified nucleic acid vector) can been altered, modified or engineered such that it contains greater, fewer or different nucleic acid sequences than those included in the native or natural nucleic acid molecule from which the recombinant vector was derived.<br><br>
Advantageously, the recombinant vector includes ap^/c gene or recombinant nucleic      ' acid molecule including such 3.pdc gene, operably linked to regulatory sequences, for example, promoter sequences, terminator sequences and/or artificial ribosome binding sites (RBSs), as defined herein.<br>
Typically, thQpdc gene is operably linked to regulatory sequence(s) in a manner which allows for expression (e.g.. enhanced, increased, constitutive, basal, attenuated, decreased or repressed expression) of the nucleotide sequence, advantageously expression of a gene product encoded by the nucleotide sequence (e.g., when the recombinant nucleic acid molecule is included in a recombinant vector, as defined herein, and is introduced into a microorganism).<br>
The regulatory sequence includes nucleic acid sequences which affect (e.g., modulate or regulate) expression of other nucleic acid sequences. In one embodiment, a regulatory sequence is included in a recombinant nucleic acid molecule or recombinant vector in a similar or identical position and/or orientation relative to a particular gene of interest as is observed for the regulatory sequence and gene of interest as it appears in nature, e.g., in a native position and/or orientation. For example, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence which accompanies or is adjacent to the gene of interest in the natural organism (e.g., operably linked to 'native" regulatory sequences, for example, to the 'native" promoter). Alternatively, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence which accompanies or is adjacent to another (e.g., a different) gene in the natural organism. Alternatively, a gene of interest can be included in a recombinant nucleic acid molecule or recombinant vector operably linked to a regulatory sequence from another organism. For example, regulatory sequences from other microbes (e.g., other bacterial regulatory sequences, bacteriophage regulatory sequences and the like) can be operably linked to a particular gene of interest.<br>
In one embodiment, a regulatory sequence is a non-native or non-naturally-occurring sequence (e.g., a sequence which has been modified, mutated, substituted, derivatized, deleted including sequences which are chemically synthesized). Advantageous regulatory sequences include promoters, enhancers, termination signals, anti-termination signals and other expression control elements (e.g., sequences to which repressors or inducers bind and/or binding sites for transcriptional and/or translational regulatory polypeptides, for example, in the transcribed mRNA). Such regulatory sequences are described, for example, in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd, ed, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. Regulatory sequences include those which direct constitutive expression of a nucleotide sequence in<br><br>
a microorganism (e.g., constitutive promoters and strong constitutive promoters); those which direct inducible expression of a nucleotide sequence in a microorganism (e.g., inducible promoters, for example, xylose inducible promoters); and those which attenuate or repress expression of a nucleotide sequence in a microorganism (e.g., attenuation signals or repressor sequences). It is also within the scope of the present invention to regulate expression of a gene of interest by removing or deleting regulatory sequences. For example, sequences involved in the negative regulation of transcription can be removed such that expression of a gene of interest is enhanced.<br>
In one embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes a nucleic acid sequence or gene that encodes at least one bacterial pdc gene product operably linked to a promoter or promoter sequence. Advantageous promoters of the present invention include native promoters, surrogate promoters and/or bacteriophage promoters. In one embodiment, a promoter is a promoter associated with a biochemical housekeeping gene or a promoter associated with a ethanologenic pathway. In another embodiment, a promoter is a bacteriophage promoter. Other promoters include re/(the translational elongation factor (TEF) promoter) and/?yc (the pyruvate carboxylase (PYC) promoter), which promote high level expression in Bacillus (e.g.. Bacillus suhtilis). Additional advantageous promoters, for example, for use in Gram positive microorganisms include, but are not limited to, the amyE promoter or phage SP02 promoters. Additional advantageous promoters, for example, for use in Gram negative microorganisms include, but are not limited to (ac. trp, iei, trp-tet, Ipp, lac, Ipp-lac, laclq, T7, T5, T3, gal, ire, ara, SP6, X-Pnor k-Pi.<br>
In another embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes a terminator sequence or terminator sequences (e.g., transcription terminator sequences). Typically, terminator sequences refer to the regulatory sequences which serve to terminate transcription of a gene. Terminator sequences (or tandem transcription terminators) can further serve to stabilize mRNA (^ ,g-. by adding structure to mRNA), for example, against nucleases.<br>
In yet another embodiment, a recombinant nucleic acid molecule or recombinant vector of the present invention includes sequences which altow for detection of the vector containing said sequences (/.e., detectable and/or selectable markers), for example, sequences that overcome auxotrophic mutations, for example, ura3 or ilvE, fluorescent markers, and/or colorimetric markers (e.g., /acZ/p-galactosidase), and/or antibiotic resistance genes (e.g., bla or let).<br>
It is understood that any one of iki^ pdc genes of the invention can be introduced into a vector also comprising one or more ethanologenic genes (e.g., alcohol dehydrogenase (/.e., adh)) and/or a gene encoding a gene product suitable for fermenting a sugar or degrading a sugar for subsequent fermentation as described for example, in<br><br>
Examples 8 and 9 and as provided in U.S. patent Nos. 5,821,093; 5,482,846; 5,424,202; 5,028,539; 5,000,000; 5,487,989, 5,554,520, and 5,162,516. Such two or more genes can be expressed independently using separate regulatory elements (e.g.^ promoters), common regulatory element(s), native regulatory e!ement(s), or a combination thereof.<br>
IK      Isolated Polypeptides<br>
Another aspect of the present invention features isolated polypeptides {e.g., isolated ethanologenic enzymes, for example, pyruvate decarboxylase (PDC)). In one embodiment, PDC polypeptides are produced by recombinant DNA techniques and can be isolated from microorganisms of the present invention by an appropriate purification scheme using standard polypeptide purification techniques. In another embodiment, polypeptides are synthesized chemically using standard peptide synthesis techniques.<br>
An isolated or purified polypeptide {e.g., an isolated or purified PDC) is substantially free of cellular material or other contaminating polypeptides from the microorganism from which the polypeptide is derived, or substantially free from chemical precursors or other chemicals when chemically synthesized. In one embodiment, an isolated or purified polypeptide has less than about 30% (by dry weight) of contaminating polypeptide or chemicals, more advantageously less than about 20% of contaminating polypeptide or chemicals, still more advantageously less than about 10% of contaminating polypeptide or chemicals, and most advantageously less than about 5% contaminating polypeptide or chemicals.<br>
In one embodiment, the PDC polypeptide or gene product is derived from a Gram-positive or Gram-negative bacterium. Advantageously, the PDC polypeptide or gene product is derived from a Gram-negative microorganism selected from the group consisting of Gluconobacter, Rhizobium, Bradyrhizobium, Alcaligenes, Rhodobacter, Rhodococcus. Azospirillum, Rhodospirillum, Sphingomonas, Burkholderia, Desulfomonas, Geospirillum, Succinomonas, Aeromonas, Shewanella, Halochromatium, Citrobacter, Escherichia, Klebsiella, Zymomonas (e,g, Zymomonas mobilis), Zymobacter (e.g., Zymobacier palmae), and Acetohacter (e.g., Acetobacter pasteurianus).<br>
In another embodiment, the PDC polypeptide or gene product is derived from a Gram-positive microorganism selected from the group consisting of Fibrobacter, Acidobacter, Bacteroides, Sphingobacterium, Actinomyces, Corynebacterium, Nocardia, Rhodococcus, Propionibacterium, Bifidobacterium. Bacillus, Geobacillus, Paenibacillus, Sulfobacillus, Clostridium, Anaerobacter, Eubacterium, Streptococcus, Lactobacillus, Leuconostoc, Enterococcus, Lactococcus, Thermobifida, Cellulomonas, and Sarcina (e.g. Sarcina ventricuU), Advantageously, the gene product is derived from a microorganism selected from the group consisting of the Gram-negative bacteria<br><br>
Zymob'acter palmae, Acetohacter pasteurianus, and the Gram-positive bacterium Sarcina ventriculi.<br>
Included within the scope of the present invention are PDC polypeptides or gene products that are Zymobacier palmae, Acetobacter pasteurianus, or Sarcina ventriculi -derived polypeptides or gene products encoded by naturally-occurring bacterial genes. Further included within the scope of the present invention are bacterial-derived polypeptides or gene products which differ from naturally-occurring bacterial and/or Zymohacter palmae, Acetobacter pasteurianus, or Sarcina ventriculi genes {e.g., pdc), for example, genes which have nucleic acids that are mutated, inserted or deleted, but which encode polypeptides substantially similar to the naturally-occurring gene products of the present invention, e.g.^ comprise a pyruvate decarboxylase activity.<br>
For example, it is well understood that one of skill in the art can mutate (e.g., substitute) nucleic acids which, due to the degeneracy of the genetic code, encode for an identical amino acid as that encoded by the naturally-occurring gene. This may be desirable in order to improve the codon usage of a nucleic acid to be expressed in a particular organism. Moreover, it is well understood that one of skill in the art can mutate (e.g., substitute) nucleic acids which encode for conservative amino acid substitutions. It is further well understood that one of skill in the art can substitute, add or delete amino acids to a certain degree without substantially affecting the function of a gene product (e.g., decarboxylase activity) as compared with a naturally-occurring gene product, each instance of which is intended to be included within the scope of the present invention. The decarboxylase activity and, for example, enzyme/substrate affinity, enzyme thermostability, and/or enzyme activity at various pHs, can be readily determined using the assays described herein.<br>
In an embodiment, an isolated polypeptide of the present invention (e.g., an isolated pyruvate decarboxylase enzyme, for example isolated PDC polypeptide has an amino acid sequence shown in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, or SEQ ID N0:8. In other embodiments, an isolated polypeptide of the present invention is a homologue of the at least one of the polypeptides set forth as SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, or SEQ ID N0:8 (e.g.. comprises an amino acid sequence at least about 30-40% identical, advantageously about 40-50% identical, more advantageously about 50-60% identical, and even more advantageously about 60-70%, 70-80%, 80-90%, 90-95% or more identical to the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, or SEQ ID N0:8, and has an activity that is substantially similar to that of the polypeptide encoded by the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, or SEQ ID N0:8, respectively.<br><br>
To determine the percent identity of two amino acid sequences or of two nucleic acids, the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in the sequence of a first amino acid or nucleic acid sequence for optimal alignment with a second amino or nucleic acid sequence). When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e.. % identity = # of identical positions/total # of positions x 100), advantageously taking into account the number of gaps and size of said gaps necessary to produce an optimal alignment.<br>
The comparison of sequences and determination of percent identity between two sequences can be accomplished using a mathematical algorithm. A particular, non-limiting example of a mathematical algorithm utilized for the comparison of sequences is the algorithm of Karlin and Altschul (1990) Proc. Nail. Acad Sci. USA 87:2264-68, modified as in Karlin and Altschul (1993) Proc. Natl. Acad Sci, USA 90:5873-77. Such an algorithm is incorporated into the NBLAST and XBLAST programs (version 2.0) of Altschul e( al. (1990) J. Mol. Biol. 215:403-10. BLAST nucleotide searches can be performed with the NBLAST program, score = 100, wordlength = 12 to obtain nucleotide sequences homologous to nucleic acid molecules of the invention. BLAST polypeptide searches can be performed with the XBLAST program, score = 50, wordlength = 3 to obtain amino acid sequences homologous to polypeptide molecules of the invention. To obtain gapped alignments for comparison purposes. Gapped BLAST can be utilized as described in Altschul et al. (1997) Nucleic Acids Research 25(17):3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs {e.g., XBLAST and NBLAST) can be used. See http://www.ncbi.nlm.nih.gov. Another particular, non-limiting example of a mathematical algorithm utilized for the comparison of sequences is the algorithm of Myers and Miller (1988) Comput Appl Biosci. 4:11-17. Such an algorithm is incorporated into the ALIGN program available, for example, at the GENESTREAM network server, IGH Montpellier, FRANCE or at the ISREC server. When utilizing the ALIGN program for comparing amino acid sequences, a PAM120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 can be used.<br>
In another embodiment, the percent identity between two amino acid sequences can be determined using the GAP program in the GCG software package, using either a Blossom 62 matrix or a PAM250 matrix, and a gap weight of 12, 10, 8, 6, or 4 and a length weight of 2, 3, or 4. In yet another embodiment, the percent homology between two nucleic acid sequences can be accomplished using the GAP program in the GCG<br><br>
software package (available at http://www.gcg.com), using a gap weight of 50 and a length weight of 3.<br>
Based on the foregoing isolated PDC polypeptides, immunospecific antibodies can be raised against a PDC polypeptide, or portion thereof, using standard techniques as described herein.<br>
V.        Methods of Use<br>
Methods for Producins Acetaldehyde<br>
The clean and efficient production of acetaldehyde as a desirable product has widespread commercial and industrial applications. For example, acetaldehyde is used in the production of acetic acid, flavorings, foods, beverages, perfumes, plastics, aniline dyes, synthetic rubber manufacturing, the silvering of mirrors, gelatin fiber hardening, and in laboratory research. In addition, acetaldehyde serves as a substrate for the production of ethanol through, for example, fermentation.<br>
Accordingly, the present invention includes methods for the conversion of a substrate, for example, pyruvate or a pyruvate analog, into acetaldehyde, using the PDC enzymes of the invention. In one embodiment, the invention provides methods for producing acetaldehyde form pyruvate using microorganisms {e.g., a recombinant microorganisms) expressing ^pdc gene and gene product (PDC) as described herein. The methods also include biological processes which result in the production {e.g.. transformation or conversion) of pyruvate, or convertible analog thereof, which can be converted or decarboxylated to acetaldehyde, or an analog thereof<br>
The method comprises contacting a microorganism which expresses apJc gene and gene product of the invention, and optionally, one or more ethanologenic genes, with a sugar, carbon skeleton capable of being converted into pyruvate, pyruvate, or an analog thereof, under culture conditions such that acetaldehyde (or analog thereof) is produced.<br>
In another embodiment of the invention, the above-mentioned microorganism can be processed into a enzymatic lysate for performing the above conversion reaction. In yet another embodiment, X\ycpdc gene product is purified, as described herein, for carrying out the conversion reaction.<br>
The microorganism(s) and/or enzymes {e.g., in the form of a lysate, partially purified, or purified)) used in the conversion reactions are in a form allowing them to perform their intended function {e.g., producing a desired compound, e.g., acetaldehyde). The microorganisms can be whole cells, or can be only those portions of the cells necessary to obtain the desired end result. The microorganisms can be suspended {e.g., in an appropriate solution such as buffered solutions or media), rinsed {e.g., rinsed free of media from culturing the microorganism), acetone-dried.<br><br>
immobilized (e.g., with polyacrylamide gel or k-carrageenan or on synthetic supports, for example, beads, matrices and the like), fixed, cross-linked or permeabilized {e.g., have permeabilized membranes and/or walls such that compounds, for example, substrates, intermediates or products can more easily pass through said membrane or wall).<br>
Purified or unpurified PDC enzymes (alone or in combination with other ethanologenic enzyme(s)) can also be used in the conversion reactions. The enzyme can be in a form that allows it to perform its intended function {e.g., obtaining the desired compound, e.g., acetaldehyde, or, if in the presence of the necessary ethanologenic gene products, ethanol). For example, the enzyme can be in free form or immobilized.<br>
Methods for Producins Ethanol<br>
In one embodiment of the present invention, the host cell having the above mentioned attributes is also ethanologenic. Accordingly, the invention provides methods for producing ethanol using such host cells (or extracts/enzymes derived therefrom). In addition, the host cells can be applied in synergistically degrading or depolymerizing a complex saccharide into a monosaccharide. Subsequently, the cell can catabolize the simpler sugar into ethanol by fermentation. This process of concurrent complex saccharide depolymerization into smaller sugar residues followed by fermentation is referred to as simultaneous saccharification and fermentation (SSF).<br>
Typically, fermentation conditions are selected that provide an optimal pH and temperature for promoting the best growth kinetics of the producer host cell strain and catalytic conditions for the enzymes produced by the culture (Doran ef a/., (1993) Biotechnol Progress, 9:533-538). For example, for Klebsiella, e.g., the P2 strain, optimal conditions were determined to be between 35-37° C and pH 5.0- pH 5.4. Under these conditions, even exogenously added fungal endoglucanases and exoglucanases are quite stable and continue to function for long periods of time. Other conditions are discussed in the Examples. Moreover, it will be appreciated by the skilled artisan, that only routine experimentation is needed, using techniques known in the 2irt, for optimizing a given fermentation reaction of the invention. See. for example, U.S. patent Nos. 5,424,202 and 5,916,787, which are specifically incorporated herein by this reference.<br>
This invention is further illustrated by the following examples, which should not be construed as limiting.<br><br>
Exemplification<br>
Throughout the examples, the following materials ar\d methods are used unless otherwise stated. Abbreviation are as follows: PDC, pyruvate decarboxylase; ADH, alcohol dehydrogenase; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis; TPP, thiamine diphosphate; PCMS,para-chloromercuriphenyl-sulfonic acid; PCMB,/7ara-chloromecuribenzoic acid; DTT, dithiothreitol; Apa, Acetobacter pasteiirianus', Sve, Sarcina ventriculi\ Zpa, Zymohacter palmae; Zmo Z mohilis.<br>
Materials and Methods<br>
Strains and Media - Z palmae simnT\09 (lAM 14233, ATCC 51623) was cultivated in ATCC 1956 MY medium (10 g yeast extract, 20 g maltose, 2 g KH2PO4, and 5 g NaCl per liter) at 26T (200 rpm). A. pasteurianus strain NCIB8618 (ATCC 12874) was grown with aeration at 25X in minimal medium (pH 5.0 to 5,5) supplemented with 2% (v/v) D-L-lactate as described previously (3, 13) with the addition of 0.1 ml per liter of 1% (v/v) antifoam. E. coli strains ERl 648 F" fhuA2^{lacZ) rl supE44 trp31 mcrA]272::TnI0(Tci') his-1 rpsLl()4 {Siv')xyl^7 mll'2 metBl ^{mcrC-mrr)102\'.lnlO{lQX') (New England Biolabs, Beverly, MA), DH5a F" recAl endAl hsdR17{x]^mC) supE44 thi-1 gyrA relAl (Life Technologies, Rockville, MD), BL21-CodonPlus-RIL F' ompThsdSix^" mif) dent' Tct'gcd X (DE3) cndA lite [argUHeYleuWCBXVL] (an E.coli B strain) (Stratagene, LaJoUa, CA), and Rosetta (DE3) F'ompThsdSs irQ'MB') gal dcm lacYl (pRARE) (Novagen, Madison, WI) were used for recombinant DNA experiments. £ coli strains were grown at ST^C (200 rpm) in Luria-Bertani (LB) medium. Medium was supplemented with 2% (w/v) glucose and antibiotics including ampicillin (100 mg per liter), kanomycin (100 mg per liter), and chloramphenicol (30 mg per liter) as needed.<br>
DNA Isolation and Cloning - Plasmid DNA was isolated using a Quantum Prep Plasmid Miniprep Kit from BioRad (Hercules, CA). DNA fragments were eluted from 0.8% SeaKem GTG agarose (FMC Bioproducts, Rockland, ME) gels with IX TAE buffer (40 mM Tris acetate, 2 mM EDTA, pH 8.5) using the QIAquick gel extraction kit from Qiagen (Valencia, CA). Genomic DNA was isolated using the method described by Harwood and Cutting (17).<br>
Analysis of DNA and Polypeptide Sequences ofZ. palmae - Plasmids were sequenced using the Sanger dideoxy method (42) using a LICOR sequencer (DNA Sequencing Facility, Department of Microbiology and Cell Science, University of Florida). Genpro 5.0 (Hoefer Scientific, San Francisco, CA), ClustalW version 1.81 (46), Treeview version 1.5 (39), and MultiAln (12) were used for DNA and polypeptide sequence alignments and comparisons. Deduced amino acid sequences were compared<br><br>
to polypeptide sequences in the GenBank, EMBL, and SwissProt at the National Center for Biotechnology Information (Bethesda, MD) using the BLAST network server.<br>
Purification of PDC Polypeptides - PDC purifications were performed as presented in Table 2. All procedures were performed at room temperature and all purification buffers contained 1 mM TPP and 1 mM MgCla unless otherwise indicated. S. ventriculi PDC was purified from recombinant E. coli BL21-CodonPlus-RIL(pJAM419) and Rosetta (DE3)(pRARE, pJAM419) as previously described (45). Z mobilis PDC was purified from recombinant E. coli DH5a (pLOI276) (II) using thermal treatment, Q-Sepharose, and Superdex 200 procedures as described below for the Z palmae PDC with the exception that the Z mobilis PDC was eluted from Q-Sepharose at 0.22 to 0.32 M NaCl.<br>
(i) Purification ofZ. palmae PDC - Z palmae and E. coli DH5a (pJAM3440) cells were grown to stationary phase and harvested by centrifugation at 10,000 X jtf (10 min, 4°C). Cells (12 to 15 g wet wt) were resuspended in 6 volumes of 50 mM sodium phosphate buffer at pH 6.5 containing 1 mM TPP and I mM MgCh (buffer A) and passed through a chilled French pressure cell at 20,000 lb per in^. Cell debris was removed by centrifugation at 16,000 X g (20 min, 4T). Cell lysate was heated to 60T for 30 min, chilled on ice for 15 min, and centrifuged at 16,000 X ^ (30 min at 4"C) to remove denatured polypeptides. The supernatant was applied to a HiLoad Q-Sepharose 26/10 column (Amersham Pharmacia Biotech, Piscataway, NJ) equilibrated in 50 mM buffer A. The column was washed with 200 ml of 50 mM buffer A and a 400-ml linear 0 - 1 M NaCl gradient was applied at 4.0 ml per min. The peak of PDC activity eluted between 0.4 and 0.5 M NaCl and was pooled. The recombinant Z palmae PDC from E. coli was purified to homogeneity by injecting aliquots (0.5 ml) onto a Superdex 200 10/30 column (Amersham Pharmacia Biotech) equilibrated with 50 mM buffer A containing 150 mM NaCl and 10% glycerol at 0.2 ml per min.<br>
Purification of "native' PDC from Z palmae required additional steps. Heat-treated lysate was applied to a hydroxyapatite CHT5-1 column (BioRad) equilibrated with 10 mM buffer A. The column was washed with 30 ml of 10 mM buffer A and a 30-ml linear 10 mM - IM sodium phosphate gradient was applied at 0.5 ml per min. The peak of PDC activity eluted at 0.45 to 0.5 M sodium phosphate and was pooled. Aliquots were further purified using a Superdex 200 10/30 column as described above. Solid ammonium sulfate was added to the combined active fractions to 1 M and then loaded onto a Phenyl Sepharose 6 Fast Flow (low sub) column (Amersham Pharmacia Biotech) equilibrated with 50 mM buffer A containing 1 M ammonium sulfate. After a 15-ml wash, the column was developed with a 15-ml decreasing linear gradient of 1 - 0 M ammonium sulfate at 0.5 ml per min. The peak of "native" PDC activity was eluted between 0.55 and 0.3 M salt.<br><br>
(ii) Purification of A. pasteurianus PDC - PDC was purified from A. pasteuhanus and E. coli ER1648(pJAM304) using heat-treatment, Q-Sepharose, Superdex 200, and hydroxyapatite as described for the Z palmae PDC with the following modifications. Cells were passed through a French pressure cell at 30,000 lb per in^. The PDC activity, which was eluted at 0.17 to 0.26 M NaCl from the Q-Sepharose column, was pooled and concentrated by dialysis against PEG8000 prior to loading onto the Superdex 200 column.<br>
The Z palmae and A. pasteurianus PDCs were stored for up to one month in 10% glycerol under liquid nitrogen without loss of activity. The S. ventriculi and Z mohilis PDCs were stored at 4'C.<br>
PDC Quantitation and Enzyme Activity Assay - Polypeptide concentration was determined using Bradford reagent with bovine serum albumin as the standard (BioRad). PDC activity was measured by the coupled assay with baker's yeast alcohol dehydrogenase (ADH) (Sigma- Aldrich, St. Louis, MO) as previously described (11). The reaction mixture contained 0.15 mM NADH, 5 mM MgCb, 0.1 mM TPP, 5 mM pyruvate, and 10 U ADH in 50 mM sodium citrate buffer at pH 5.0 and was measured at 25°C unless otherwise indicated. One unit of enzyme activity is defined as the amount of enzyme that generates 1 |imol of acetaldehyde per min.<br>
Molecular Mass and Amino Acid Sequence Determination of the PDC Polypeptide - Subunit molecular masses and enzyme purity were determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) using 12% polyacrylamide gels stained with Coomassie blue R-250 (26). Molecular weight standards for SDS-PAGE were as follows: phosphorylase b (97.4 kDa), serum albumin (66.2 kDa), ovalbumin (45 kDa), carbonic anhydrase (31 kDa), trypsin inhibitor (21.5 kDa), and lysozyme (14.4 kDa). For the determination of native molecular mass, samples were applied to a Superdex 200 10/30 column equilibrated in 50 mM sodium phosphate buffer at pH 6.5 with 150 mM NaCl. Molecular mass standards included: serum albumin (66 kDa), alcohol dehydrogenase (150 kDa), a-amylase (200 kDa), apoferritin (443 kDa), and thyroglobulin (669 kDa).<br>
The N-terminal sequences of the purified PDC polypeptides were determined after SDS-PAGE and electroblotting of the polypeptides onto a polyvinylidene difluoride membrane (PVDF-PLUS) (Micron Separations Inc., Westborough, MA). Sequences were determined by automated Edman degradation at the Polypeptide Chemistry Core Facility of the University of Florida Interdisciplinary Center for Biotechnology Research.<br>
Apoenzyme Preparation - Purified PDC (0,75 mg per 0.5 ml) was diluted with 1.5 ml of 50 mM sodium phosphate buffer at pH 9.0 and immediately concentrated 8-fold using a Centricon YM30 concentrator (Millipore, Bedford, MA). After adjusting<br><br>
the volume to 0.5 ml, the polypeptide was incubated at 25°C for 45 min. PDC was purified from unbound cofactors by application to a Superdex 200 10/30 column equilibrated with 50 mM sodium phosphate buffer at pH 9.0 with 10% glycerol and 150 mM NaCl. Immediately after elution, the apoenzyme was diluted 5-fold into 50 mM sodium citrate buffer at pH 5.0, stored at 4'C, and used within 16 h for reconstitution assays, TPP was measured by fluorescence after oxidation to thichrome diphosphate using K3Fe(CN)6 in 15% NaOH (15). Excitation and emission wavelengths were 375nni and 430nm, respectively. For reconstitution, apoenzyme (755 ng) was diluted into 1 ml of 50 mM sodium citrate buffer at pH 5.0 with 1 mM TPP and/or 1 mM of MgC^. The mixture was assayed for PDC activity in the same buffer.<br>
Materials - Biochemicals were purchased from Sigma-Aldrich. Other organic and inorganic analytical grade chemicals were from Fisher Scientific (Atlanta, GA). Restriction endonucleases and DNA-modifying enzymes were from New England BioLabs. Digoxigenin-11-dUTP (2'-deoxyurid!ne-5'-triphosphate coupled by an 11-atom spacer to digoxigenin), alkaline phosphatase conjugated antibody raised against digoxigenin, and nylon membranes for colony and plaque hybridizations were from Roche Molecular Biochemicals. Positively charged nylon membranes for Southern hybridization were from Ambion (Austin, TX).<br>
EXAMPLE 1<br>
IDENTIFICATION AND CHARACTERIZATION OF PDC GENE FROM<br>
GRAM-NEGATIVE BACTERIUM ZYMOBACTER PALMAE<br>
In this example, the identification and characterization of a PDC gene from the Gram-negative bacterium Z palmae is described.<br>
Briefly, a pyruvate decarboxylase ipdc) operon was isolated from a genomic library of Zymobacter palmae using a degenerate oligonucleotide probe based on the N-terminal amino acid sequence of the purified protein (Fig. 1). A 1.7-kb Pstl to BamUl subclone of this fragment (pJAM3440) was found to be required for PDC activity in recombinant E. coli. Based on DNA sequence, a 1668-bp ORF (55 mol% G+C content) was identified in this region that encoded a putative PDC polypeptide (Fig. 1). A canonical Shine-Dalgamo sequence (GGAGG) was 10 bp upstream of the translation start codon (ATG). In addition, a putative -35 and -10 promoter (TTcACt-Nn-atTAAT, where N is any nucleotide and uppercase letters match the bacterial promoter consensus) was located 16 to 44 bp upstream of the start codon (18). Downstream (21 bp) of the translation stop codon was a 62 bp sequence predicted to form two stem-loop structures followed by a 16 bp AT-rich region, consistent with p-independent transcription termination (18). All four bacterial pdc genes, including those from Z<br><br>
mohilis, S. ventriculi, and A. pasteuricmus, are predicted to be independently transcribed from a monocistronic operon (11, 40, 45).<br>
The deduced PDC polypeptide of Z palmae (ZpaPDC) contained 556 amino acids (including the N-terminal methionine) with an anhydrous molecular mass of 60,113 Da (Fig. 4). This is similar to the three other bacterial PDCs which range from 552 to 568 amino acids and 59,830 to 61,809 Da. GenBank accession number AF474145 has been assigned to the Z palmae sequence.<br>
EXAMPLE 2<br>
IDENTIFICATION AND CHARACTERIZATION OF PDC GENE FROM<br>
GRAM-NEGATIVE AEROBIC BkCTER\\}MACETOBACTER PASTEURIANUS<br>
In this example, the identification and characterization of a PDC gene from the Gram-negative bacterium Acetohacter pasteurianus is described.<br>
Briefly, a chromosomal DNA was isolated from cells of i4. pasteurianus using the method described by Harwood et al. (17). For Southern analysis, the genomic DNA (I to 2 |ig per lane) was cleaved with restriction enzymes (.4^/11, BamWl, Clal, EcoRl, and Hindi), separated by gel electrophoresis (0.8% agarose), and transferred by downward capillary action to positively charged nylon membranes. Membranes were equilibrated at 60^C for 2 h in 5X SSC (IX SSC is 0.15 M NaCl plus 0.015 M sodium citrate) containing 1% blocking reagent, 0.1% N-lauroylsarcosine, and 0.02% sodium dodecyl sulfate (SDS). Random primers were used to label a probe with dixoigenin-11-dUTP using a 0.7-kb Kprtl fragment of the Z mohilis pdc gene as the template. After the labeled-probe (1 ng per ml) was added, membranes were incubated at GOT for 14 h and washed twice with a solution containing 2X SSC and 0.1% SDS (5 min per wash) and twice with a solution containing 0.5X SSC and 0.1 % SDS (15 min per wash at 60'C). Signals were visualized by chemiluminescence using X-ray film.<br>
A sub-genomic library was generated using 5- to 7-kb Aatll fragments of ^. pasteurianus chromosomal DNA. Overhangs were converted to blunt ends using Vent DNA polymerase and ligated into the blunt (Vent DNA polymerase), de-phosphorylated (calf intestinal alkaline phosphatase) Xhol site of pLITMUS28. After transformation, recombinants were screened by hybridization using conditions similar to those for Southern analysis, which allowed the isolation of plasmid pJAM301 containing the full-length/&gt;(/c and aldl genes (Fig. 2).<br>
The A. pasteurianus DNA fragment (4.2 kb) in pJAM301 was then subcloned (Fig. 2) and sequenced by the Sanger dideoxy method (42) using a LICOR sequencer (DNA Sequencing Facility, Department of Microbiology and Cell Science, University of Florida). GenBank accession number AF368435 has been assigned to this sequence.<br><br>
Genepro 5.0 (Riverside Scientific, Seattle, WA), ClustalW version 1.81 (Thompson et al, 1994), Tree view version 1.5 (Page, 1996), and MultiAln (Corpet, 1988) were used for DNA and/or polypeptide sequence alignments and comparisons. Deduced amino acid sequences were compared to polypeptide sequences available in the GenBank, EMBL, and SwissProt databases at the National Center for Biotechnology Information (Bethesda, MD) using the BLAST network server.<br>
A fragment of the pyruvate decarboxylase (pdc) gene from Z mohilis was used as a DNA-hybridization probe to isolate a 6,337 bp Aatll fragment with 58.5% G+C content from A. pasteurianus genomic DNA (Fig. 2). Based on a Blast analysis of the DNA sequence, apc/c gene was identified. This gene is predicted to encode a polypeptide of 548 amino acids (including the N-terminal formyl-methionine) with a calculated pi of 5.49 and an anhydrous molecular mass of 58,873 Da (Fig. 5). The codon usage of the/7t/c gene was compared tp the two other bacterial/7(^c genes, Z mobilis (11, 34, Reynen and Sahm, J Bacteriol. 170:3310-3313 (1988)) and S. ventriculi, as well as the E. coli K-12 genome (Table 1). Consistent with high G+C content for the A. pasteurianus pdc gene (60.4%), the codons had a predominance of C and/or G in the third base position with exception of those for Glu, His, and Tyr (Table 1). This is in contrast to the Z mobilis (52.4%) and S. ventriculi {}Q.9%)pdc genes that have lower G+C content and differ in their codon usage.<br>
Analysis of the 5'-end of the A. pasteurianus pdc gene revealed a canonical Shine-Dalgamo sequence that is 8 bp upstream of the predicted translation start codon of GTG. In addition, a region 72 to 101 bp upstream oixh^pdc gene has high identity to eubacterial -35 and -10 consensus sequence for a a^^-dependent promoter. Immediately downstream (17 bp) of thep^c translation stop codon is a series of predicted hairpin loop structures that are followed by an AT-rich region, indicating p-independent termination of pdc transcription. Thus, the A. pasteurianus pdc gene was determined to be transcribed as a monocistronic operon like that of Z mobilis (11) and S. ventriculi pdc genes.<br>
Additional open reading frames (ORFs) were identified in the pdc gene region including an ORF denoted as aldl, which is divergently transcribed from pdc (Fig. 2). The putative aW/gene is predicted to encode a polypeptide of 357 amino acids (including the N-terminal formyl-methionine) with a calculated molecular mass of 38,108 Da and pl of 6.32. The deduced polypeptide sequence (Aldl) is very similar to members of the Zn^"*"- and NAD'^-dependent medium-chain ADH family (class III) most notably the glutathione-dependent formaldehyde dehydrogenases (GSH-FDH) (Jomvall et al, Eur. J. Biochem 167:195-201 (1987)). Amino acid residues that are probable ligands for both the structural (C-92, C-95, C-98, and C-106) and catalytic (C-40, H-62, and C-169) zinc ions important for GSH-FDH activity are conserved. A Rossman fold<br><br>
consensus sequence (residues 194-GLGGIG-199) is also present and indicates that Aldl can bind NADCP)"^. The predicted A. pasteurianus Aldl polypeptide has little sequence identity to the three subunits of the membrane-bound ALD or ADH enzymes, necessary for ethanol oxidation in Acetobacter (Thumer et al, 1997; Kondo and Horinouchi, J Bacteriol. 177:5048-5055 (1997)). Instead, this predicted polypeptide was determined to have close evolutionary ties to GSH-FDH polypeptides. Most notably, the A. pasteurianus polypeptide encoded by aW/clusters with GSH-FDH enzymes from the a-proteobacteria, the cynaobacterium Anahaena azolla, and to a lesser extent with the P-and y-proteobacteria consistent with the taxonomic classification of the acetic acid bacteria.<br>
Subclones of the ^. pasteurianus AatW 6.3-kb genomic fragment carrying the pdc gene on a high copy number plasmid were analyzed for PDC activity by the ADH coupled assay. Significant levels of PDC activity were only detected for strains carrying plasmid pJAM304, which has the complete/?£/c' open reading frame in addition to 124 bp upstream and 236 bp downstream of the gene (Figs. 2 and 5). GenBank accession number AR368435 has been assigned to the Acetobacter pasteurianus sequence.<br>
EXAMPLE 3<br>
IDENTIFICATION AND CHARACTERIZATION OF PDC GENE FROM<br>
GRAM-POSITIVE BACTERIUM SARCINA VENTRICULI<br>
In this example, the identification and characterization of a PDC gene from the Gram-positive bacterium iSarc/wa ventriculi is described.<br>
Briefly, a degenerate oligonucleotide 5'-AARGARGTNAAYGTNGARCAYATGTTYGGNGT-3' (SEQ ID N0:11) was synthesized based on the N-terminal amino acid sequence of PDC purified from S. ventriculi (Lowe and Zeikus, 1992)(where, R is A or G; N is A, C, G, or T; Y is C or T). This oligonucleotide was labeled at the 3'-end using terminal transferase with digoxigenin-11-dUTP and dATP as recommended by the supplier (Roche Molecular Biochemicals) and was used to screen genomic DNA from S, ventriculi.<br>
For Southern analysis, genomic DNA was digested with BgR, EcoKl, or Hindi, separated by 0.8% agarose electrophoresis, and transferred to positively charged nylon membranes (Southern, 1975). Membranes were equilibrated at 58°C for 2 h in 5 X SSC (1 X SSC is 0.15 M NaCl plus 0.015 M sodium citrate) containing 1% blocking reagent (Roche Molecular Biochemicals), 0.1% A^-lauroylsarcosine, and 0.02% SDS. After the probe (0.2 pmol per ml) and Poly(A) (0.0 Img per ml) were added, membranes were incubated at 58*'C for 18.5 h. Membranes were washed twice with 2 X SSC containing 0.1% SDS (5 min per wash) at 25°C and twice with 0.5 X SSC containing 0.1% SDS (15<br><br>
min per wash) at 58°C. Signals were visualized using colorimetric detection accordir to supplier (Roche Molecular Biochemicals).<br>
For generation of a subgenomic library in plasmid pBR322, S. ventricidi chromosomal DNA was digested with Hindi and fractionated by electrophoresis. The 2.5- to 3.5-kb Hindi DNA fragments were ligated into the EcoRV site of pBR322 and transformed into £. coli SE2309. Colonies were screened with the degenerate oligonucleotide by colorimetric detection. By this method, plasmid pJAM400 that carries Hindi fragment containing 1,350 bp of the pdc gene was isolated (Fig. 3).<br>
The XBlueSTAR Vector System (Novagen) was used to create an additional subgenomic library to facilitate isolation of the full-length/76/c gene from S. ventriculi. Genomic DNA was digested with fic/I, separated by electrophoresis in 0.8% agarose, and the 6.5- to 8,5-kb fragments were ligated with the A^BlueSTAR BamHl arms. In vitro packaging and plating of phage was performed according to the supplier (Novagen). A DNA probe was generated using an 800-bp EcoRl fragment of the/^Jc gene from pJAM400 that was labeled with digoxigenin-11-dUTP using the random primed method as recommended by the supplier (Roche Molecular Biochemicals). Plaques were screened using colorimetric detection. Cre-Zox/'-mediated subcloning was used to circularize the DNA of the positive plaques by plating XBlueSTAR phage with E. coli BM25.8 that expresses Cre recombinase (Novagen). The circularized plasmid pJAM410 was then purified and electroporated into E. coli DH5a.<br>
For generation of apc/c expression vector, the promoterless/Jt/c gene was subcloned into pET21d after amplification from pJAM413 (Fig. 3) by the polymerase chain reaction (PCR). Primers were designed for directional insertion using BspHl (oligo 1) and ATzoI (oligo 2) restriction sites. The resulting fragment was ligated into compatible Ncol and^ol sites of pET2Id (Novagen) to produce pJAM419 (Fig. 3). The fidelity ofxhepdc gene was confirmed by DNA sequencing.<br>
To determine the S. ventriculi pyruvate decarboxylase operon, the N-terminal amino acid sequence of the PDC polypeptide purified from S. ventriculi (Lowe and Zeikus, 1992) was used to generate a degenerate oligonucleotide for hybridization to genomic DNA. This approach facilitated the isolation of a 7.0-kb BcR genomic DNA fragment from S. ventriculi. The fragment was further subcloned in order to sequence both strands of a 3,886 bp Hindl-to-Hincll region that hybridized to the oHgonucleotide probe (Fig. 3). Analysis of the DNA sequence reveals an open reading frame (ORF) of 1,656 bp encoding a polypeptide with an N-terminus identical to that of the previously purified S. ventriculi PDC (Fig. 6). The ORF is therefore designated pt/c. A canonical Shine-Dalgamo sequence is present 7 bp upstream of the pdc translation start codon. In addition, a region 82 to 110 bp upstream of pdc has limited identity to the eubacterial -35 and -10 promoter consensus sequence. Downstream (43 bp) of the pdc translation<br><br>
stop codon is a region predicted to form a stem-loop structure followed by an AT-ric region, consistent with a p-independent transcription terminator. Thus, the S. ventricuu pdc is transcribed as a monocistronic operon like the Z mobilis pdc gene (Conway et al, 1987).<br>
A partial ORP was identified 722 bp upstream of pdc which encodes a 177 amino acid polypeptide fragment (ORFl*) (Fig. 3). ORFl* has identity (28-29 %) to several hypothetical membrane polypeptides (GenBank accession numbers CACl 1620, CAC24018, CAA22902) and is predicted to form several transmembrane spanning domains (data not shown). The gene encoding ORFl * is not predicted to be transcribed from the putative -35 and -10 promoter of the pdc operon.<br>
Based on the above studies, the S. ventricidi pdc gene was determined to encode a polypeptide of 552 amino acids (including the N-terminal methionine) with a calculated pi of 5.16 and anhydrous molecular mass of 61,737 Da. GenBank accession number AF354297 has been assigned to the Sarcina ventriculi sequence.<br>
EXAMPLE 4 METHODS FOR PRODUCING AN ANTIBODY THAT SPECIFICALLY BINDS<br>
A PDC POLYPEPTIDE<br>
In this example, methods for making an immunospecitlc antibody against a pyruvate decarboxylase are described.<br>
In brief, purified recombinant A. pasteurianus PDC polypeptide (300 \xg) was separated by SDS-PAGE. A gel fragment containing the PDC protein was excised prior to staining with Coomassie blue R-250. This was used as antigen for polyclonal antibody production in rabbits (anti-ApPDC) as recommended by the supplier (Cocalico Biologicals, Reamstown, Penn.). For immunoblot analysis, proteins were separated by SDS-PAGE and transferred to PVDF membranes in 10 mM MES at pH 6.0 with 10 % methanol for 16 h at 20 vohs at 4°C. For detection of antigen, immunoblot procedures and colorimetric detection were using standard techniques. The primary antibody, anti-ApPDC, was diluted 1:7,000. The alkaline phosphatase linked goat anti-rabbit secondary antibody (Fisher Biotech) was diluted 1:7,500. Using the foregoing method, a polyclonal antisera that specifically bound a PDC polypeptide was identified.<br>
If desired, the antibody molecules directed against PDC can be isolated from the mammal (e.g., from the blood) and further purified by well known techniques, such as protein A chromatography to obtain the IgG fraction. In addition, at an appropriate time after immunization, e.g., when the anti-PDC antibody titers are highest, antibody-producing cells can be obtained from the subject and used to prepare monoclonal antibodies by standard techniques, such as the hybridoma technique originally described<br><br>
by Kohler and Milstein (1975) Nature 256:495-497; see also, e.g.. Using Antibodies: Laboratory Manual, Harlow et al, C.S.H.L. Press, Pub. (1999)).<br>
EXAMPLE 5 METHODS FOR IDENTIFYING CODON USAGE OF PDC GENES IN<br>
DIFFERENT HOST CELLS<br>
In this example, methods for identifying PDC polypeptides suitable for improved expression based on codon usage are described.<br>
Bacterial PDC can be used to engineer ethanol pathways in a wide variety of organisms. However, compatible patterns of codon usage are advantageously determined in order to ensure for functional expression in heterologous systems (4, 14, 16, 22). A simple tabulation of G+C content can serve as an initial guide. G+C contents for these foxxxpdc genes range from 60% for .4. pasteurianus to 31% for S. ventriculi. The average G+C content for E. coli ORFs is 52%, identical to that for Z mobilis pdc (52%) and similar to Z palmaepdc (55%). For most amino acids, patterns of codon usage for these two organisms were quite similar to each other and to E. coli. The Z palmae pdc was functionally expressed at high levels (approximately 1/3 of polypeptide based on activity) in recombinant E. coli. The Z mobilis pdc and A. pasteurianus pdc were expressed at lower levels (7 to 9% of polypeptide based on activity) and S. ventriculi pdc was poorly expressed in recombinant E. coli (less than 0.3% of polypeptide based on activity)(45).<br>
To produce larger amounts of functional recombinant 5". ventriculi PDC, the use of a modified E. coli host containing additional tRNA genes for rare codons such as AGA (arginine), GGA (glycine), AUA (isoleucine), and CUA (leucine) proved important for high-level polypeptide production (almost 1/4 of polypeptide based on activity) (45). The pattern of codon usage in S. ventriculi pdc is particularly useful for the genetic engineering of an ethanol pathway in low G+C Gram-positive bacteria as illustrated by the comparison to B. subtilis ORFs (Table 1). In contrast, A. pasteurianus pdc can be selected for engineering a homoethanol pathway in to high G+C cellulolytic bacteria such as Thermobifidia sp. (23) or Cellulomonas sp. (35).<br>
*<br><br><br><br>
EXAMPLE 6 BIOCHEMICAL CHARACTERIZATION OF PDC ACTIVITY<br>
In this example, the biochemical characterization of several representative PDC polypeptides is described.<br>
The purification of four representative bacterial PDC polypeptides was performed as presented in Table 2, infra. ZpaPDC was purified to homogeneity from Z palmae and recombinant £ coli. For comparison, three other bacterial PDCs were purified from recombinant E. coli. PDC was also purified from A. pasteurianus. In general, purification of the 'native' PDCs required additional steps {i.e., phenyl Sepharose and hydroxyapatite chromatography). This was due to the 10- to 250-fold lower levels of PDC activity of'native' compared to ^recombinant' cell lysate. All of the PDC polypeptides had subunit molecular masses of 55 to 60 kDa, as estimated by reducing SDS-PAGE, which were consistent with the masses calculated from the deduced polypeptide sequences.<br>
The N-terminal amino acid sequence of PDC purified from Z palmae (MYTVGMYLAE) was identical to the sequence deduced from the gene and included the N-terminal methionine. Previous studies have demonstrated that the PDC purified from S. ventriculi also retains the N-terminal methionine which is amino to a lysine residue (28, 45). In contrast, the N-terminal sequence of PDC purified from A. pasteurianus (TYTVGMYLAERL) lacked an N-terminal methionine indicating that cleavage by a native methionine aminopeptidase. The PDC purified from Z mohilis also was determined to be cleaved to expose an N-terminal serine (34). These results are consistent with the highly conserved substrate specificity of methionine aminopeptidases which is dictated by the Pi' residue (5). The N-terminal methionine is only removed if the second residue is small and uncharged. This substrate preference is opposite to the 'N-end rule' for polypeptide degradation (47).<br>
A determination of the quaternary structure of the PDC polypepfides was also performed. The ZmoPDC associates as a tetramer of 240 kDa even after removal of the cofactors TPP and Mg^"**(15). Likewise the SvePDC and ZpaPDC associated as tetramers of 240 kDa even after cofactor dissociation (data not shown). Interestingly, the ApaPDC formed both tetramers and octamers (224 and 447 kDa) of similar specific activity, and dissociated into dimers (120 kDa) after cofactor extraction. Activity and tetrameric configuration of the ApaPDC apoenzyme were fully restored after addition of Mg^"*" and TPP with half saturation constants of 3.1 |aM and 5.8 |iM and Hill constants of 1.17 to 1.22, respectively. The tetrameric configuration observed for all four bacterial PDCs is consistent with the quaternary structure of the majority oCeukaryotic PDCs. However, similar to ApaPDC higher order structures have been observed for the PDC<br><br>
from the filamentous fungus Neurospora crassa, which associates as a homopolymeric filament of 8 to 10 nm (1). Likewise, in plants (maize, pea, and wheat germ) PDC forms complexes up to 1 MDa (24, 27, 32). The dissociation of ApaPDC into dimers after removal of the cofactors is also consistent with eukaryotic PDCs (25).<br><br>
EXAMPLE 7 CHARACTERIZATION OF PDC THERMOSTABILITY<br>
In this example, the thermostability of several representative PDC polypeptides is described.<br>
It has been observed that the ZmoPDC is thermostable after heating cell lysate to eO'C in the presence of cofactors (TPP and Mg^"^) (11). To fiirther address this phenomenon, activity of the purified bacterial PDCs was analyzed at 25'C after exposure to various temperatures in the presence of saturating cofactors (Fig. 7).   All three Gram-negative PDCs were relatively thermostable and retained 60 to 100%<br><br>
activity after heating to 60'C for 30 min. In contrast, the purified SvePDC was completely inactivated after exposure to temperatures of SO'C or higher. Although amino acid composition cannot be used as a universal guide to thermostability (30, 31, 48), it was observed that the three thermostable PDCs contained higher levels of alanine and cysteine and lower levels of phenylalanine. These changes are consistent with an increase in thermostability based on compositional comparisons of homologous enzymes (2, 31). Increased levels of alanine can stabilize Gram-negative PDC polypeptides at high temperature by facilitating the formation of helices and a more compact polypeptide core. Accordingly, the present invention also includes pdc genes which have been altered to express a PDC polypeptide comprising corresponding cysteine and/or alanine amino acid changes to achieve improved thermostability. In addition, analysis of the thermostable PDCs from the Gram-negative bacteria revealed an over 2.5-fold increase in activity when assayed at their temperature optima of 60°C compared to room temperature. Based on Arrhenius plots, these enzymes were estimated to have activation enthalpy in the range of 12.6 to 14.2 kJ mof' and entropy values of-92.8 to -98.2 Jmol*' K'*.<br>
EXAMPLE 8 CHARACTERIZATION OF PDC SUBSTRATE AFFINITIES AND pH OPTIMA<br>
In this example, the characterization of substrate affinity for several representative PDC polypeptides as well as pH optima are described.<br>
In particular, significant differences in pH optima were observed for the bacterial PDC polypeptides. The ZpaPDC was most active at pH 5.5 to 6.0 (Fig. 8), similar to ZmoPDC which has a pH optimum of 6.0 (34). The pH optimum of ApaPDC, however, was significantly lower than that observed for the other two Gram-negative PDCs and ranged from 5.0 to 5.5 (Fig. 8). The pH optimum for the Gram-positive SvePDC (pH 6.3 to 7.6) has been shown to be much broader and more neutral than the Gram-negative PDCs (28). This indicates that there are differences in the conformation and/or composition of charged residues at or near the active site.<br>
The eukaryotic and bacterial SvePDC exhibit positive cooperative kinetics in the presence of the substrate pyruvate (9, 25, 28). The ZpaPDC and ApaPDC, however, exhibited normal Michaelis-Menten kinetics with K^ values of 0.2 to 0.4 mM for pyruvate and kcat values of 20,500 to 30,500 min'* (pH optimum, 25"C) (Table 3). The lack of allosteric control with respect to pyruvate, which was observed for these Gram-negative PDC enzymes, is similar to that established for the highly related PDC from Z mobilis (7, 34). As a further comparison, the kinetic constants of all four bacterial PDC polypeptides were determined at their pH optimum as well as at pH 7.0 (Table 3). This approach was chosen to analyze the bacterial PDC enzymes to more closely reflect the<br><br>
neutral cytosol of recombinant hosts used previously for high-level ethanol production (e.g. E. coli, Erwinia spp.). These modest pH changes had only a slight effect on the maximum rate of the reaction for all four bacterial PDC polypeptides. In contrast, pH significantly influenced the affinity of the Gram-negative PDCs for pyruvate. Most notably, an over 13-fold increase in /^m was observed when ApaPDC was shifted from its pH optimum to neutral conditions.<br><br>
These results indicate that all three Gram-negative PDCs have an amino acid residue(s) that enables the enzyme to more efficiently bind pyruvate when protonated (at the pH optimum) than deprotonated (neutral pH). However, the protonated slate of this residue does not influence the overall rate of decarboxylation. Because the changes in A^m for the Gram-negative PDCs were observed between pH values of 5 to 7, it is possible that the residue involved in modulating substrate binding for all three enzymes is a histidine. The pKa for free histidine is 6.04; whereas, the pKa for pyruvate as well as the other ionizable amino acid residues {i.e. Asp, Glu, Lys, and Arg) does not fall within this pH range.<br>
Protonation of one or more histidines is important to facilitate substrate binding by forming an ion pair with the carboxyl group of pyruvate or by making the active site more substrate accessible. It is also possible that a different residue(s) is involved modulating substrate binding; however, its pKa is modified by the polypeptide environment.<br>
The effect of pH on k^JK^ compared to A:cat for the PDC enzyme of Z mobilis was determined using titration curves with pKa values estimated at 6.23 to 6.45 for the residue involved in modulating substrate binding (21, 43). The deprotonation of Hisl 13 has been indicated to lead to conformational changes that result in a flexible loop (residues 105 to 112) to close over the active site during catalysis (43). Both Hisl 13 and<br><br>
Hisl 14 are conserved in all PDC polypeptides that have been characterized. Thus, Hisl 13 is a logical candidate for the observed pH-dependent changes in Km observed for all three Gram-negative PDCs. However, five crucial residues with ionizable side-chains protruding into the active site of Z mobilis PDC (Asp27, Glu50, Hisl 13, Hisl 14, and Glu473) were modified to residues that were non-ionizable or had altered pKa values (21). The pH-dependence oikcJK^ was relatively unaffected by these modifications indicating that these residues, including Hisl 13, is not involved. Importantly, in this same study modification of Hisl 13 to a residue with significantly higher pKa value {i.e, Arg and Lys) increased the affinity of ZmoPDC for pyruvate by over 20-fold.<br>
These results show that the positively charged form of Hisl 13 keeps the active site open for substrate binding.<br>
EXAMPLE 9 COMPARISON ALIGNMENT OF PDC GENES FROM DIFFERENT<br>
ORGANISMS<br>
In this example, a comparison alignments of the amino acid sequences of pyruvate decarboxylase enzymes (PDC) from different organisms are compared and contrasted.<br>
The deduced PDC polypeptide of Z palmae (ZpaPDC) contained 556 amino acids (including the N-terminal methionine) with an anhydrous molecular mass of 60,113 Da. This is similar to the three other bacterial PDCs which range from 552 to 568 amino acids and 59,830 to 61,809 Da. The ZpaPDC had a calculated pi of 4.93 analogous to the pi determined experimentally (4.87 to 5.3) for the Z mobilis PDC (7, 33). The relatively high alanine content of ZpaPDC (13.1%) was comparable to that of the Gram-negative Z mobilis and A. pasteuhanus PDCs (Zmo and ApaPDC) but almost 2-fold higher than that of the Gram-positive S. ventriculi PDC (SvePDC) (6.9%). Multiple amino acid sequence alignment (Fig. 9) and dendrogram cluster analysis revealed that ZpaPDC was most closely related to ApaPDC (72% identity) and highly related to ZmoPDC (62% identity). All three of the Gram-negative PDCs clustered in sequence similarity with the plant PDCs {e.g. Zea mays PDC; 39 to 40% identity).<br>
In contrast, the Gram-positive SvePDC was only 30 to 31% identical to the Gram-negative PDCs and instead grouped with the majorit}' of filamentous fungi and yeast PDCs {e.g. Saccharomyces cerevisiae PDCl; 38% identity). The N-terminal extension common to PDCs of the plant kingdom was absent from all four of the bacterial PDCs. Residues involved in TPP and Mg^"^ binding, based on the crystal structures of the Z mobilis and yeast PDCs, were conserved in all four bacterial PDCs. In contrast, residues of the yeast PDCl (Tyrl57 and Arg224) are involved in substrate<br><br>
activation, based on binding to the pyruvate analog pyruvamide (29), were only noted in the bacterial SvePDC and not found in the remaining three bacterial PDCs.<br>
The deduced A. pasteuhanus PDC amino acid sequence was aligned with PDC sequences from Z mobilis and yeast (PDCl), both of which have been analyzed by X-ray crystallography (Dyda et al, 1993; Arjunan et al, 1996; Lu et ai, 2000; Dobritzsch et al, 1998), as well as from Zea mays (Fig. 9).   The multiple alignments revealed thai the A, pasteurianus PDC is most similar to Z mobilis PDC (62% identity and 74% similarity). The N-terminal extension of the Z mays PDC, common to other plant PDCs, is not found in the fungal or bacterial PDC polypeptides including that o^A. pasteurianus. Residues demonstrated to be within 0.4 nm of the TPP and Mg^^ binding sites of the yeast and Z mobilis PDC enzymes were highly conserved in the A. pasteuhanus PDC polypeptide sequence (V24, D27, E50, T72, V75, N82, HI 13, HI 14, T384, D386. G408,1410, D435, S437, L440,1460, N462, G464,1467, E468). In addition, the motif identified in TPP-dependent enzymes which was proposed by Hawkins et al (1989) to be involved in Mg^^ and TPP binding is also conserved in the A, pasteurianus PDC polypeptide sequence (G435 to N462). Residues of the yeast PDCl (Tyrl57 and R224) proposed to be involved in allosteric control and demonstrated to bind the pyruvate analog pyruvamide (Lu et ai, 2000), were not conserved in the A. pasteurianus, Z mobilis, or Z mays PDC polypeptides. Instead these residues are restricted to the majority of fungal PDCs and related PDC polypeptide from the Gram-positive bacterium 5*. ventriculi which is substrate activated (Lowe and Zeikus, 1992). Based on the kinetics of CO2 production from pyruvate observed for cell lysate of acetic acid bacteria (King and Cheldelin, 1953), it is concluded that the A, pasteurianus PDC polypeptide, similar to the Z mobilis PDC, is not substrate activated. However, the plant PDCs do not contain these conserved residues but are regulated by substrate, indicating some differences in the mechanism of allosteric control among PDC polypeptides.<br>
Based on cluster analysis of PDC polypeptides, the A. pasteurianus PDC is most closely related to the Z mobilis PDC. This is consistent with the classification of these two bacteria as Gram-negative a-proteobacteria. In contrast to the low G+C Gram-positive S. ventriculi PDC which shares close evolutionary roots with the majority of fungal PDCs, the PDC polypeptides from the a-proteobacteria are more closely related to plant and a couple of out-grouping fungal PDCs.<br><br>
EQuivalents<br>
Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims. Moreover, any number of genetic constructs, host cells, and methods described in United States Patent Nos. 5,821,093; 5,482,846; 5,424,202; 5,028,539; 5,000,000; 5,487,989, 5,554,520, and 5,162,516, can be employed in carrying out the present invention and are hereby incorporated by reference.<br>
Literature Cited<br>
1.	Alvarez, M. E., A. L. Rosa, E. D. Temporini, A. Wolstenholme, G. Panzetta, L. Patrito, and H. J. Maccioni. 1993. The 59-kDa polypeptide constituent of 8-10-nm cytoplasmic filaments in Neurospora crassa is a pyruvate decarboxylase. Gene 130:253-258.<br>
2.	Argos, P., M. G. Rossman, U. M. Grau, H. Zuber, G. Frank, and J. D. Tratschin. 1979. Thermal stability and polypeptide structure. Biochemistry 18:5698-5703.<br>
3.	Atkinson, D. E. 1956. Hydrogen metabolism mAcetohacter peroxydam. J.Bacteriol. 72:189-194.<br>
4.	Barbosa, M. F. S. and L. O. Ingram. 1994. Expression of the Zymomonas mohilis alcohol dehydrogenase II (adhB) and pyruvate decarboxylase (pdc) genes in Bacillus. Curr.Microbiol. 28:279-282.<br>
5.	Bradshaw, R. A., W. W. Brickey, and K. W. Walker. 1998. N-terminal processing: the methionine aminopeptidase and N"-acetyl transferase families. Trends Biochem.Sci. 23:263-267.<br>
6.	Braii, B. and H. Sahm. 1986. Cloning and expression of the structural gene for pyruvate decarboxylase of Zymomonas mobilis in Escherichia coli. Arch.Microbiol. 144:296-301.<br>
7.	Bringer-Meyer, S., K.-L. Schimz, and H. Sahm. 1986. Pyruvate decarboxylase from Zymomonas mobilis. Isolation and partial characterization. Arch.Microbiol. 146:105-110.<br><br>
Bucher, M., R. Braendle, and C. Kuhlemeier. 1994. Ethanolic fermentation in transgenic tobacco expressing Zymomonas mobilis pyruvate decarboxylase, EMBOJ. 13:2755-2763.<br>
Candy, J. M. and R. G. Duggleby . 1998. Structure and properties of pyruvate decarboxylase and site-directed mutagenesis of the Zymomonas mobilis enzyme. Biochim.Biophys.Acta 1385:323-338.<br>
Candy, J. M., R. G. Duggleby, and J. S. Mattick. 1991. Expression of active yeast pyruvate decarboxylase in Escherichia coli. J.Gen.Microbiol. 137:2811-2815.<br>
Conway, T., Y. A. Osman, J. I. Konnan, E. M. Hoffmann, and L. O, Ingram. 1987. Promoter and nucleotide sequences of the Zymomonas mobilis pyruvate decarboxylase. J.Bacteriol. 169:949-954.<br>
Corpet, F. 1988. Multiple sequence alignment with hierarchical clustering. Nucleic Acids Res. 16:10881-10890.<br>
DeLey, J. 1958. Studies on the metabolism of Aceiohacier peroxydans. Part I. General properties and taxonomic position of the species. Antonie Van Leeuwenhoek J.Microbiol.Serol. 24:281-297.<br>
Deng, M. D. and J. R. Coleman. 1999. Ethanol synthesis by genetic engineering in cyanobacteria. Appl.Environ.Microbiol. 65:523-528.<br>
Diefenbach, R. J. and R. G. Duggleby. 1991. Pyruvate decarboxylase from Zymomonas mobilis. Structure and re-activation of apoenzyme by the cofactors thiamin diphosphate and magnesium ion. Biochem.J. 276:439-445.<br>
Gold, R. S., M. M. Meagher, S. Tong, R. W. Hutkins, and T. Conway. 1996. Cloning and expression of the Zymomonas mobilis "production of ethanol" genes in Lactobacillus casei. Curr.Microbiol. 33:256-260.<br>
Harwood, C. R. and S. M. Cutting. 1990. In Molecular Biological Methods for Bacillus, Wiley, New York.<br>
Henaut, A. and A. Danchin. 1996. Analysis and predictions from Escherichia coli sequences, or E. coli in silico, p. 2047-2066. In F. C. Neidhardt and e( al (eds.), Escherichia coli and Salmonella Cellular and Molecular Biology. ASM Press, Washington,DC.<br><br>
Hoppner, T. C. and H. W. Doelle. 1983. Purification and kinetic characteristics of pyruvate decarboxylase and ethanol dehydrogenase from Zymomonas mohilis in relation to ethanol production. Eur.J.Appl.Microbiol.Biotechnol. 17:152-157.<br>
Horn, S. J., I. M. Aasen, and K. 0stgaard. 2000. Production of ethanol from mannitol by Zymohacter palmae. J.Indust.Microbiol.Biotech. 24:51-57.<br>
Huang, C. Y., A. K. Chang, P. F. Nixon, and R. G. Duggleby. 2001. Site-directed mutagenesis of the ionizable groups in the active site of Zymomonas mohilis pyruvate decarboxylase: effect on activity and pH dependence. Eur.J.Biochem. 268:3558-3565.<br>
Ingram, L. O., H. C. Aldrich, A. C. Borges, T. B. Causey, A. Martinez, F. Morales, A. Saleh, S. A. Underwood, L. P. Yomano, S. W. York, J. Zaidivar, and S. Zhou. 1999. Enteric bacterial catalysts for fuel ethanol production. Biotechnol.Prog. 15 :855-866.<br>
Irwin, D. C, S. Zhang, and D. B. Wilson. 2000. Cloning, expression and characterization of a Family 48 exocellulase, Cel48A, from Thermobifida fusca. Eur.J.Biochem. 267:4988-4997.<br>
Kenworthy, P. and D. D. Davies . 1976. Kinetic aspects of regulation of pyruvic decarboxylase. Phytochemistry 15:279-282.<br>
Konig, S. 1998. Subunit structure, function and organisation of pyruvate decarboxylases from various organisms. Biochim.Biophys.Acta 1385:271-286.<br>
Laemmli, U. K. 1970. Cleavage of structural polypeptides during the assembly of the head of bacteriophage T4. Nature 227:680-685.<br>
Lee, T. C. and P. J. Langston-Unkefer. 1985. Pyruvate decarboxylase from Zea mays L. I. Purification and partial characterization from mature kernals and anaerobically treated roots. Plant Physiol. 79:242-247.<br>
Lowe, S. E. and J. G. Zeikus. 1992. Purification and characterization of pyruvate decarboxylase from Sarcina ventriculi. J.Gen.Microbiol. 138:803-807.<br>
Lu, G., D. Dobritzsch, S. Baumann, G. Schneider, and S. Konig. 2000. The structural basis of substrate activation in yeast pyruvate decarboxylase. A crystallographic and kinetic study. Eur.J.Biochem. 267:861-868.<br><br>
Matthews, B. W. 1993. Structural and genetic analysis of polypeptide stability. Annu.Rev.Biochem. 62:139-160.<br>
Mozhaev, V. V. and K. Martinek . 1984. Structure-stability relationships in polypeptides: new approaches to stabilizing enzymes. Enzyme Microb.Technol. 6:50-59.<br>
Mucke, U., S. Konig, and G. Hiibner. 1995. Purification and characterisation of pyruvate decarboxylase from pea seeds (Pisum sativum cv. Miko). Biol.Chem.Hoppe Seyler 376:111-117.<br>
Neale, A. D., R. K. Scopes, R. E. Wettenhall, and N. J. Hoogenraad. 1987. Nucleotide sequence of the pyruvate decarboxylase gene from Zymomonas mobilis. Nucleic Acids Res. 15:1753-1761.<br>
Neale, A. D., R. K. Scopes, R. E. Wettenhall, and N. J. Hoogenraad. 1987. Pyruvate decarboxylase oi Zymomonas mobilis: isolation, properties, and genetic expression in Escherichia coli, J.Bacteriol. 169:1024-1028.<br>
Notenboom, V., C. Birsan, R, A. Warren, S. G. Withers, and D. R. Rose. 1998. Exploring the cellulose/xylan specificity of the beta-l,4-glycanase cex from Cellulomonas fimi through crystallography and mutation. Biochemistry 37:4751-4758.<br>
Okamoto, T., H. Taguchi, K. Nakamura, and H, Ikenaga. 1994. Production of ethanol from maltose by Zymobacter palmae fermentation. Biosci.Biotechnol.Biochem. 58:1328-1329.<br>
Okamoto, T., H. Taguchi, K. Nakamura, H. Ikenaga, H. Kuraishi, and K. Yamasato. 1993. Zymobacter palmae gen. nov., sp. nov., a new ethanol-fermenting peritrichous bacterium isolated from palm sap. Arch.Microbiol. 160:333-337.<br>
Or, E., J. Baybik, A. Sadka, and A. Ogrodovitch. 2000. Fermentative metabolism in grape berries: isolation and characterization of pyruvate decarboxylase cDNA and analysis of its expression throughout berry development. Plant Sci. 156:151-158.<br>
Page, R. D. 1996. TreeView: an application to display phylogenetic trees on personal computers. ComputAppl.Biosci. 12:357-358.<br><br>
Raj, K. C, L. O. Ingram, and J. A. Maupin-Furlow. 2001. Pyruvate decarboxylase: a key enzyme for the oxidative metabolism of lactic acid by Acetobacter pasteuhanus. Arch.Microbiol. 176:443-451.<br>
Reid, M. F. and C. A. Fewson. 1994. Molecular characterization of microbial alcohol dehydrogenases. Crit Rev.Microbiol. 20:13-56.<br>
Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. ProcNatl.Acad.Sci.U.S.A. 74:5463-5467.<br>
Schenk, G., F. J. Leeper, R. England, P. F. Nixon, and R. G. Duggleby. 1997. The role of Hisl 13 and Hisl 14 in pyruvate decarboxylase from Zymomonas mobilis. Eur.J.Biochem. 248:63-71.<br>
Tadege, M., R. Brandle, and C. Kuhlemeier. 1998. Anoxia tolerance in tobacco roots: effect of overexpression of pyruvate decarboxylase. Plant J. 14:327-335.<br>
Talarico, L. A., L. O. Ingram, and J. A. Maupin-Furlow. 2001, Production of the Gram-positive Sarcina ventriculi pyruvate decarboxylase in Escherichia coli. Microbiology 147:2425-2435.<br>
Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673-4680.<br>
Tobias, J. W., T. E. Shrader. G. Rocap, and A. Varshavsky. 1991. The N-end rule in bacteria. Science 254:1374-1377.<br>
Vieille, C. and G. J. Zeikus. 2001. Hyperthermophilic enzymes: sources, uses, and molecular mechanisms for thermostability. Microbiol.Mol.Biol.Rev. 65:1-43.<br><br><br><br>
Claims<br>
1.	An isolated nucleic acid molecule selected from the group consisting of:<br>
a)	a nucleic acid molecule comprising a nucleotide sequence which is at least 60% homologous to the nucleotide sequence of SEQ ID NO: 1, SEQ ID N0:3, or SEQ ID N0:5, or a complement thereof;<br>
b)	a nucleic acid molecule comprising a fragment of at least 000 nucleotides of a nucleic acid comprising the nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, or a complement thereof;<br>
c)	a nucleic acid molecule which encodes a polypeptide comprising an amino acid sequence at least about 50% homologous to the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6;<br>
d)	a nucleic acid molecule which encodes a fragment of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6; wherein the fragment comprises at least 15 contiguous amino acid residues of the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6; and<br>
e)	a nucleic acid molecule which encodes a naturally occurring allelic variant of a polypeptide comprising the amino acid sequence^of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6; wherein the nucleic acid molecule hybridizes to a complement of a nucleic acid molecule comprising SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5, under stringent conditions.<br>
2.	The isolated nucleic acid molecule of claim 1 which is selected from the<br>
group consisting of:<br>
a)	a nucleic acid molecule comprising the nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, or a complement thereof; and<br>
b)	a nucleic acid molecule which encodes a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6.<br><br>
3.	The nucleic acid molecule of claim 1 further comprising vector nucleic acid sequences.<br>
4.	The nucleic acid molecule of claim 1 operably linked to a surrogate promoter.	'<br>
5.	The nucleic acid molecule of claim 1 further comprising nucleic acid sequences encoding a heterologous polypeptide.<br><br>
6.	A host cell which contains the nucleic acid molecule of claim 1.<br>
7.	The host cell of claim 6, wherein the host cell is selected from the group consisting of a Gram-negative bacterial cell and a Gram-positive bacterial cell.<br>
8.	The host cell of claim 6, wherein the Gram-negative bacterial cell is selected from the group consisting of Gliwonobacter, Rhizobium, Bradyrhizobium, Alcaligenes, Rhodobacter, Rhodococcus. Azospirillum, Rhodospirillum, Sphingomonas, Burkhoideria, Desulfomonas, Geospirilliim, Succinomonas, Aeromonas, Shewanella, Halochromaiium, Citrobacier, Escherichia, Klebsiella, Zymomonas, Zymobacter, and Acetobacter.<br>
9.	The host cell of claim 6, wherein the Gram-positive bacterial cell is selected from the group consisting of Fibrobacter, Acidobacter, Bacteroides, Sphingobacterium, Actinomyces, Corynebacterium, Nocardia, Rhodococcus, Propionibacterium, Bifidobacterium, Bacillus, Geobacillus, Paenibacillus, Sulfiybacillus, Clostridium, Anaerobacter, Eubacterium, Streptococcus, Lactobacillus, Leuconostoc, Enterococcus, Lactococcus, Thermohifida, Cellulomonas, and Sarcina.<br>
10.	An isolated polypeptide selected from the group consisting of:<br>
a)	a fragment of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, wherein the fragment comprises at least 15 contiguous amino acids of SEQ ID NO:2, SEQ ID N0:4, or SEQ ID N0:6;<br>
b)	a naturally occurring allelic variant of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID NO:6, wherein the polypeptide is encoded by a nucleic acid molecule which hybridizes to a complement of a nucleic acid molecule comprising SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5, under stringent conditions;<br>
c)	a polypeptide which is encoded by a nucleic acid molecule comprising a nucleotide sequence which is at least 50% homologous to a nucleic acid comprising the nucleotide sequence of SEQ ID N0:1, SEQ ID N0:3, or SEQIDN0:5;and<br>
d)	■       a polypeptide comprising an amino acid sequence which is at<br>
least 30% homologous to the amino acid sequence of SEQ ID N0:2, SEQ ID<br>
NO:4,orSEQIDNO:6.<br><br>
11.	The isolated polypeptide of claim 10 comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6.<br>
12.	The polypeptide of any one of claims 10 or 11, wherein the polypeptide has improved decarboxylase activity.<br>
13.	The polypeptide of any one of claims 10 or 11, wherein the polypeptide has improved thermal stability.<br>
14.	The polypeptide of any one of claims 10 or 11, wherein the polypeptide has improved substrate affinity.<br>
15.	The polypeptide of any one of claims 10 or 11 further comprising heterologous amino acid sequences.<br>
16.	An antibody which selectively binds to a polypeptide of claim 10 or 11.<br>
17.	A method for producing a polypeptide selected from the group consisting of<br><br>
a)	a polypeptide comprising the amino acid sequence SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6;<br>
b)	a fragment of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6; wherein the fragment comprises at least 15 contiguous amino acids of SEQ ID NO:2, SEQ ID N0:4, or SEQ ID N0:6;<br>
c)	a naturally occurring allelic variant of a polypeptide comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, wherein the polypeptide is encoded by a nucleic acid molecule which hybridizes to a complement of a nucleic acid molecule comprising SEQ ID N0:1, SEQ ID N0:3, or SEQ ID NO:5, under stringent conditions; and<br>
comprising culturing the host cell of claim 6 under conditions in which the nucleic acid molecule is expressed.<br>
18.	A method for detecting the presence of a polypeptide of claim 10 or 11 in<br>
a sample comprising:<br>
a)        contacting the sample with a compound which selectively binds to the polypeptide; and<br><br>
b)        determining whether the compound binds to the polypeptide in th^<br>
sample to thereby detect the presence of a polypeptide of claim 10 in the sample.<br>
19.	The method of claim 18, wherein the compound which binds to the polypeptide is an antibody.<br>
20.	A method for detecting the presence of a nucleic acid molecule in claim 1 in a sample comprising:<br><br>
a)	contacting the sample with a nucleic acid probe or primer which selectively hybridizes to a complement of the nucleic acid molecule; and<br>
b)	determining whether the nucleic acid probe or primer binds to a nucleic acid molecule in the sample to thereby detect the presence of a nucleic acid molecule of claim 1 in the sample.<br><br>
21.	The method of claim 20, wherein the sample comprises mRNA molecules and is contacted with a nucleic acid probe.<br>
22.	A kit comprising a compound which selectively hybridizes to a complement of a nucleic acid molecule of claim 1 and instructions for use.<br>
23.	A recombinant host cell comprising a heterologous nucleic acid sequence encoding pyruvate decarboxylase wherein the nucleic acid sequence is selected for improved codon usage in said host cell.<br>
24.	A recombinant host cell comprising a heterologous nucleic acid sequence encoding pyruvate decarboxylase selected for improved decarboxylase activity.<br>
25.	A recombinant host cell comprising a heterologous nucleic acid sequence encoding pyruvate decarboxylase selected for improved thermal stability.<br>
26.	The host cell of claim 23, wherein the heterologous nucleic acid sequence encoding pyruvate decarboxylase is operably linked to a surrogate promoter.<br>
27.	The host cell of claim 23, wherein the heterologous nucleic acid sequence encoding pyruvate decarboxylase is derived from a bacterial cell selected from the group consisting of Zymobacter palmae, Acetobacter pasteurianus, and Sarcina ventriculi.<br><br>
28.	The host cell of any one of claims 23-26, wherein the heterologou nucleic acid sequence is selected from the group consisting of SEQ ID N0:1, SEg IL) N0:3, and SEQ ID N0:5.<br>
29.	The host cell of any one of claims 6 and 23-26, wherein the host cell further comprises a nucleic acid encoding a polypeptide selected from the group consisting of an alcohol dehydrogenase, glucanase, and secretase.<br>
30.	The host cell of claim 29, wherein the host cell further comprises a nucleic acid encoding alcohol dehydrogenase.<br>
31.	The host cell of any one of claims 6 and 23-26, wherein the host cell is ethanologenic.<br>
32.	The host cell of any one of claims 6 and 23-26, wherein the host cell is suitable for fermenting ethanol from a sugar.<br>
33.	The recombinant host cell any one of claims 23-26, wherein said host cell is a bacterial cell selected from the group consisting of Gram-negative bacterial cell and Gram-positive bacterial cell.<br>
34.	The host cell of claim 33, wherein the Gram-negative bacterial cell is selected from the group consisting of Gluconobacter, Rhizobium, Bradyrhizobium, Alcaligenes, Rhodobacter, Rhodococcus. Azospirillum, Rhodospirillum, Sphingomonas, Burkholderia, Desulfomonas, Geospirillum, Succinomonas, Aeromonas, Shewanella, Halochromatiuniy Citrobacter, Escherichia, Klebsiella, Zymomonas Zymobacter, and Acetobacter.<br>
35.	The host cell of claim 33, wherein the Gram-positive bacterial cell is<br>
selected from the group consisting o{ Fibrobacter, Acidobacter, Bacteroides,<br>
Sphingobacterium, Actinomyces, Corynebactehum, Nocardia, Rhodococcus,<br>
Propionibacterium, Bifidobacterium, Bacillus, Geobacillus, Paenibacillus,<br>
Sulfobacillus, Clostridium, Anaerobacter, Eubacterium, Streptococcus, Lactobacillus,<br>
LeUconostoc, Enterococcus, Lactococcus, Thermobifida, Cellulomonas, and Sarcina.<br><br>
36.	A recombinant ethanologenic host cell comprising a heterologous nucleic acid encoding a PDC selected from the group consisting of SEQ ID N0:2, SEQ ID N0:4, and SEQ ID N0:6, wherein the nucleic acid is under the transcriptional control of an exogenous surrogate promoter.<br>
37.	A method for producing acetaldehyde comprising culturing the host<br>
cell of any one of claims 6 and 23-26, under conditions wherein pyruvate decarboxylase is expressed at sufficient levels such that acetaldehyde is produced from pyruvate.<br>
38.	The method of claim 37, wherein the host cell further comprises an ethanologenic gene selected from the group consisting of alcohol dehydrogenase, secretase, and glucanase.<br>
39.	A method for producing acetaldehyde comprising contacting a cell lysate obtained from the host cell of any one of claims 6 and 23-26, under conditions wherein acetaldehyde is produced from pyruvate.<br>
40.	A method for producing ethanol comprising culturing the host cell of any one of claims 6 and 23-26, under conditions wherein pyruvate decarboxylase and alcohol dehydrogenase are expressed at sufficient levels such that ethanol is produced as a primary fermentation product.<br>
41.	The method of claim 38, wherein said method is conducted in an aqueous solution.<br><br>
42.	The method of claim 40, wherein said method is conducted in an aqueous solution.<br>
43.	An enzyme extract comprising detectable levels of pyruvate decarboxylase derived from the host cell of any one of claims 6 and 23-26.<br><br>
44.	The enzyme extract of claim 37, wherein the pyruvate decarboxylase has improved decarboxylase activity.<br>
45.	The enzyme extract of claim 40, wherein the pyruvate decarboxylase has improved decarboxylase activity.<br><br>
46.	The enzyme extract of claim 43, wherein the pyruvate decarboxylase has improved thermal stability.<br>
47.	The enzyme extract of claim 43, wherein the pyruvate decafboxylase has improved substrate affinity.<br>
48.	A method for selecting a pyruvate decarboxylase enzyme with improved decarboxylase activity comprising,<br>
comparing the amino acid sequence of a pyruvate decarboxylase with the amino acid sequence of SEQ ID N0:2. SEQ ID N0:4, or SEQ ID N0:6; and<br>
altering at least one amino acid residue of said pyruvate decarboxylase to have identity with the corresponding amino acid residue of SEQ ID N0:2, SEQ ID N0:4, or SEQ ID N0:6, such that improved pyruvate decarboxylase activity is achieved.<br>
49.	The method of claim 48, wherein the pyruvate decarboxylase activity comprises improved pyruvate decarboxylase affinity for pyruvate.<br>
50.	The method of claim 48, wherein the pyruvate decarboxylase activity comprises improved thermal stability.<br>
51.	A method for selecting a pyruvate decarboxylase enzyme with improved expression in a recipient host cell comprising,<br>
comparing the nucleic acid sequence encoding a pyruvate decarboxylase with the nucleic acid sequence of SEQ ID N0:1, SEQ ID N0:3, or SEQ ID N0:5; and<br>
altering at least one codon of said nucleic acid encoding said pyruvate decarboxylase enzyme to have identity with the corresponding codon of SEQ ID NO: 1, SEQ ID N0:3, or SEQ ID NO;5, such that improved expression of said altered nucleic acid encoding pyruvate decarboxylase enzyme is achieved in said host cell.<br>
52.	A method for selecting a pyruvate decarboxylase enzyme with improved<br>
expression in a recipient host cell comprising,<br>
comparing the nucleic acid sequence encoding a pyruvate decarboxylase with the codon usage of said recipient host cell; and<br>
altering at least one codon of said nucleic acid encoding said pyruvate decarboxylase enzyme to correspond with the codon usage of said recipient host cell such that improved expression of said altered nucleic acid encoding pyruvate decarboxylase enzyme is achieved in said host cell.<br><br>
53.	A method for selecting a pyruvate decarboxylase enzyme with improved<br>
expression in a recipient host cell comprising,<br>
comparing the nucleic acid sequence encoding a pyruvate decarboxylase with the codon usage of said recipient host cell; and<br>
altering said recipient host cell to recombinantly produce at least one tRNA corresponding to a codon of said nucleic acid encoding said pyruvate decarboxylase enzyme such that improved expression of said nucleic acid encoding pyruvate decarboxylase enzyme is achieved in said altered host cell.<br>
54.	The plasmid pJAM3440 encoding apdc gene derived from Zymobacter palmae represented by a deposit with the American Type Culture Collection designated as deposit number ATCC	.<br>
55.	The plasmid pJAM304 encoding dipdc gene derived from Acetobacter pasteurianus represented by a deposit with the American Type Culture Collection designated as deposit number ATCC	.<br>
56.	The plasmid pJAM419 encoding dipdc gene derived from Sarcina ventriculi represented by a deposit with the American Type Culture Collection designated as deposit number ATCC	_.<br><br>
An isolated nucleic acid molecule substantially as herein above described with reference to the accompanying drawings. A method of selecting a pyruvate decarboxylase enzyme with improved decarboxylase  activity  substantially  as  herein  above described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg5Ny1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1897-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="201907-wind-energy-plant-with-an-asynchronous-machine-for-determining-the-azimuth-position.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="201909-a-flow-meteting-system-and-a-method-of-operating-a-flow-metering-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>201908</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1897/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Aug-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Dec-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. UNIVERSITY OF FLORIDA RESEARCH FOUNDATION INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1938 W. University Avenue, Gainesville, FL 32611</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MAUPIN-FURLOW, Julie, A</td>
											<td>4117 NW 23rd Terrace, Gainesville, FL 32605</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TALARICO, Lee, Ann</td>
											<td>2600 SW Williston Road, #701, Gainesville, FL 32608</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RAJ, Krishnan, Chandra</td>
											<td>Manthanai Village, Thangadu Horanalli Post, The Nilgiris-643003, Tamil Nadu</td>
										</tr>
										<tr>
											<td>4</td>
											<td>INGRAM, Lonnie, O.</td>
											<td>3132 NW 57th Terrace, Gainesville, FL 32611</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/88</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2002/015038</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-04-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/288,699</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/288,638</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/288,698</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/288,622</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/288,671</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/201908-an-isolated-nucleic-acid-molecule-and-a-process-of-preparing-a-host-cell-which-contains-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:23:19 GMT -->
</html>
