<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/202036-methods-of-producing-prenyl-alcohols by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:58:59 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 202036:METHODS OF PRODUCING PRENYL ALCOHOLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS OF PRODUCING PRENYL ALCOHOLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of producing a prenyl alcohol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof, culturing the resultant recombinant, and recovering the prenyl alcohol from the resultant culture.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br>
METHODS OF PRODUCING PRENYL ALCOHOLS<br>
TECHNICAL FIELD<br>
The present invention relates to methods of producing phenyl alcohols<br>
BACKGROUND ART<br>
The biosynthesis of terpenoids (isoprenoids) begins with the synthesis of geranyl diphosphate (GPP; Cio), famesyl diphosphate (FPP; C15) and geranylgeranyl diphosphate (GGPP; C20), which are straight chain prenyl diphosphates, through the condensation reaction of isopentenyl diphosphate (IPP; C5) with an allylic diphosphate substrate in succession (Fig. 1). In Fig. 1, the abbreviations and words in boxes represent enzymes. Specifically, hmgR represents hydroxymethylglutaryl-CoA (HMG-CoA) reductase; GGPS represents GGPP synthase; and FPS represents FPP synthase.<br>
Among prenyl diphosphates, FPP is the most important intermediate for the biosynthesis, and is a precursor for the synthesis of numerous kinds of terpenoids, e.g., steroids including ergosterol (provitamin D2), the side chains of quinone (vitamin K; VK), sesquiterpenes, squalene (SQ), the anchor molecules of famesylated proteins, natural rubber, etc.<br>
GGPP is also an important intermediate for the terpenoid biosynthesis, and is<br>
essential for the biosynthesis of such compounds as retinol (vitamin A; VA), p-carotene<br>
(provitamin    A),    phylloquinone    (vitamin    K1;    VK1),    the    anchor   molecules    of<br>
geranylgeranylated proteins, the side chains of chlorophyll, gibberellins, and the ether lipid<br>
 of archaea.<br>
Famesol   (FOH;   C15)  and  geranylgeraniol  (GGOH;  C20),   which  are  alcohol<br>
derivatives of FPP and GGPP, respectively, and their isomers such as nerolidol (NOH; C15)<br>
are known as fragrant substances in essential oils used in perfumes.    They are also<br>
important as starting materials for the synthesis of various compounds including the<br><br>
above-mentioned vitamins useful as pha4macological agents (Fig. 1).<br>
Thus, it is desired to establish a system in which a pure product of the so-called active-type prenyl alcohol, not a mixture of c15- and tram- ((Z)- and (E)-) isomers, can be produced in a large quantity.<br>
Although it had been believed that all the biosynthesis of IPP is performed via the mevalonate pathway (the pathway in which IPP is synthesized from acetyl-CoA through mevalonate), M. Rohmer ei al. elucidated a novel pathway for IPP synthesis using bacteria at the end of 1980's. This is called the non-mevalonate pathway or DXP (1-deoxyxylulose 5-phosphate) pathway, in which IPP is synthesized from glyceraldehyde-3-phosphate and pyruvate through 1-deoxyxylulose 5-phosphate.<br>
GGOH is currently produced by chemical synthesis (see, for example, Japanese Unexamined Patent Publication No. 8-133999). However, the chemical synthesis of GGOH requires more steps than that of FOH or NOH with shorter carbon chains, and thus requires a higher cost. Besides, though chemically synthesized GGOH has the same carbon skeleton as that of naturally occurring GGOH, it is obtained as a mixture of (E)-type {trans type) and (Z)-type {cis type) in double bond pattern. (E,, E, E)-GGOH (hereinafter, abbreviated to (all-e)-GGOH) is the form synthesized in metabolic pathways in organisms and is industrially valuable. In order to obtain (all-E)-GGOH in a pure form, refining by column chromatography, high precision distillation, etc. is necessary. However, it is difficult to carry out high precision distillation of GGOH that is a thermally unstable allyl alcohol. Also, refining by column chromatography is not suitable for industrial practice since it requires large quantities of solvents and column packings, as well as complicated operations of analyzing and recovering serially eluting fractions and removing the solvent; thus, this method is complicated and requires a high cost. Under circumstances, it is desired to establish a method of biosynthesis of (all-E)-GGOH by controlling the generation of (E)- and (Z)-geometrical isomers or by utilizing characteristics such as the repeat structures of reaction products. However, such a method has not been established yet. The substrates for GGOH synthesis are provided via the mevalonate pathway in cells of, for example, budding yeast Saccharomyces cerevisiae.    However, even when HMG-CoA<br><br>
reductase that is believed to be a key enzyme for GGOH synthesis was used, the use only increased the ability of squalene synthesis through FPP synthesis (Japanese Unexamined Patent Publication No. 5-192184; Donald et al, (1997) Appl. Environ, Microbiol 63, 3341^3344). Further, even when a squalene synthase gene-deficient strain of a special budding yeast that had acquired sterol intake ability was cultured, accumulation of 1.3 mg of FOH per liter of culture broth was only revealed (Chambon et ai, (1990) Ciirr Genet. 18, 41-46); no method of biosynthesis of NOH has been known. With respect to the biosynthesis of GGOH, production of 0.66-3.25 mg per liter of culture broth is achieved by culturing plant cells in Japanese Unexamined Patent Publication No. 9-238692. However, this method needs an expensive plant cell culture medium inappropriate for industrial application and also requires light for culturing cells. Thus, this method is less practical even compared to the conventional GGOH preparation from natural products such as essential oils. There is known no method of biosynthesis of GGOH suitable for industrialization, e.g., biosynthesis by culturing microorganisms.<br>
DISCLOSURE OF THE INVENTION<br>
It is an object of the invention to provide a method of producing a prenyl alcohol by culturing a recombinant that has been transformed with a recombinant DNA for expression comprising a prenyl diphosphate synthase gene.<br>
As a result of intensive and extensive researches toward solution of the above problem, the present inventors attempted to develop prenyl alcohol production systems by introducing genes of enzymes involved in prenyl diphosphate synthesis into hosts. As such hosts, those microorganisms which have been widely used in the fermentation industry from old times, which carry out the synthesis of prenyl diphosphate via the mevalonate pathway or DXP pathway, and which can be subjected to various genetic engineering techniques, e.g., unicellular eucaryotes (in particular yeast) or procaryotes (such as bacteria, in particular E. coli), were used. In order to construct systems with which genes of enzymes involved in prenyl diphosphate synthesis in yeast {e.g., genes of mevalonate<br><br>
pathway-related enzymes represented by HMG-CoA reductase gene, IPP A-isomerase gene, various prenyl diphosphate synthase genes, or mutants or fusion genes thereof) are expressed artificially in host cells, expression shuttle vectors were created which comprise a constitutive (permanent expression type) or inducible expression promoter and various auxotrophic markers. Then, a gene of interest or a mutant thereof was incorporated into the vector, which was then introduced into a host cell. The inventors have succeeded in obtaining prenyl alcohols (in particular geranylgeraniol) from the culture of the resultant recombinant, achieving the above-mentioned object. Thus, the present invention has been completed. When bacteria, in particular E. coli, were used as a host, a gene of an enzyme involved in prenyl diphosphate synthesis (e.g., a mutant of FPP synthase gene, or IPPA-isomerase gene) was introduced into the host cell using a conventional vector. The recombinant was cultured, and geranylgeraniol was obtained from the resultant culture after dephosphorylation. Thus, the above-mentioned object has been achieved, and the present invention has been completed.<br>
The present invention is summarized as follows.<br>
(1)	A method of producing a prenyl alcohol (eg, geranylgeraniol), comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof, culturing the resultant recombinant, and recovering the prenyl alcohol from the   ' resultant culture.<br>
(2)	A method of producing a prenyl alcohol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a recombinant DNA for expression or a DNA for genomic integration each comprising a hydroxymethylglutaryl-CoA reductase gene or a mutant thereof, culturing the resultant recombinant, and recovering the prenyl alcohol from the resultant culture.<br>
(3)	A method of producing geranylgeraniol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a<br><br>
recombinant DNA for expression or a DNA for genomic integration each comprising an isopentenyl diphosphate A-isomerase gene, culturing the resultant recombinant, and recovering geranylgeraniol from the resultant culture.<br>
(4)	The prenyl diphosphate synthase gene may be selected from the group consisting of<br>
the following genes (a) and (b) and fiision genes (c) and (d):<br>
(a)	famesyl diphosphate synthase gene or a mutant thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of famesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added.<br>
Specific examples of famesyl diphosphate synthase gene include a gene encoding the amino acid sequence as shown in SEQ ID NO: 2 or 4, and specific examples of geranylgeranyl diphosphate synthase gene include a gene encoding the amino acid sequence as shown in SEQ ID NO: 6.<br>
(5)	A method of producing geranylgeraniol, comprising creating a recombinant by<br>
transferring into a host a recombinant DNA for expression or a DNA for genomic<br>
integration each comprising a hydroxymethylglutaryl-CoA reductase gene or a mutant<br>
thereof, culturing the resultant recombinant, and recovering geranylgeraniol from the<br>
resultant culture.<br>
(6)	A method of producing geranylgeraniol, comprising creating a recombinant by<br>
transferring into a host a recombinant DNA for expression or a DNA for genomic<br>
integration each comprising a hydroxymethylglutaryl-CoA reductase gene or a mutant<br>
thereof and a recombinant DNA for expression or a DNA for genomic integration<br>
comprising a gene selected from the group consisting of the following (e) through (j):<br>
(e)	isopentenyl diphosphate A-isomerase gene<br>
(f)	mevalonate kinase gene<br>
(g)	acetyl-CoA acetyltransferase gene<br>
(h)   hydroxymethylglutaryl-CoA synthase gene<br><br>
(i)   phosphomevalonate kinase gene<br>
(j)   diphosphomevalonate decarboxylase gene; culturing the resultant recombinant, and recovering geranylgeraniol from the resultant culture.<br>
(7)	According to the above-described methods, geranylgeraniol can be produced at a<br>
concentration of at least 0.05 mg/L. Specific examples of hosts useflil in these methods<br>
include yeast (e.g., Saccharomyces cerevisiae) and Escherichia coli. Preferable S.<br>
cerevisiae strains useful in these methods include A451 strain, YPH499 strain, YPH500<br>
strain, W303-1A strain and W303-1B strain, or strains derived from any one of these strains.<br>
(8)	A recombinant DNA for expression comprising any gene selected from the<br>
above-described group consisting of genes (a) and (b) and fusion genes (c) and (d), as well<br>
as a transcription promoter and a transcription terminator.<br>
(9)	The transcription promoter may be any one selected from the group consisting of<br>
ADHl promoter, TDH3 {GAP) promoter, TEF2 promoter, GALl promoter and tac promoter;<br>
and the transcription terminator may be CYCl terminator.<br>
(10)	A recombinant obtained by transferring the above-described recombinant DNA into a host.    Specific examples of the host are as described above.<br>
(11)	A method of producing a prenyl alcohol, comprising culturing a microorganism having an ability to produce the prenyl alcohol using a medium comprising any one of the follov^ing components (i) through (vi):<br>
(i)   sugar<br>
(ii)   alcohol<br>
(iii)    ammonia gas, aqueous ammonia and/or an ammonium salt<br>
(iv)    a mixture of sodium hydroxide and sulfuric acid<br>
(v)   a mixture of KH2PO4, magnesium sulfate, ammonium sulfate, com steep<br>
liquor, calcium chloride and a surfactant (vi)    a mixture of two or more of the above components (i) through (v); and recovering the prenyl alcohol from the resultant culture.<br><br>
In the method described in (11) above, the microorganism may be cultured using a feed solution comprising the following component (i), (ii) or (iii) or a mixture of two or more of these components:<br>
(i)   sugar<br>
(ii)    alcohol<br>
(iii)    ammonia gas, aqueous ammonia and/or an ammonium salt.<br>
The feed solution may have the component as described below and may be added to the medium in the following manner, for example.<br>
Briefly, the carbon source component of the feed solution consists of glucose alone up to 12-24 hours after the start of cultivation, and then the carbon source component is shifted to a component containing ethanol. This shifting may be made in such a manner that the ratio of ethanol to the total carbon source component of the feed solution is 50% or more after 12-24 hours after the start of cultivation. Alternatively, the carbon source component of the feed solution may consist of ethanol alone after 12-24 hours after the start of cultivation.<br>
The term "feed" means that a specific solution or components are supplied or added to a culture broth by any arbitrary method during cultivation. A cultivation method in which a specific component(s) is/are supplied or added to a fermenter is called "fed-batch culture".<br>
The concentration of the prenyl alcohol accumulated in the culture is at least 0.1 g/L or more, preferably 1 g/L or more. As a specific example of the prenyl alcohol, geranylgeraniol may be given, and specific examples of the microorganism include yeast such as Saccharomyces cerevisiae. In the present invention, Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of these strains may be used.<br>
Further, in the method described in (11) above, the microorganism is preferably a recombinant. As a specific example of such a recombinant, the following a) or b) may be given:<br><br>
a)	a recombinant created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a mevalonate pathway-related gene or a mutant thereof or a prenyl diphosphate synthase gene or a mutant thereof<br>
b)	a recombinant created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl-phosphate synthase gene or a mutant thereof and a recombinant DNA for expression or a DNA for genomic integration each comprising a mevalonate pathway-related gene or a mutant thereof.<br>
Specific examples of the host include Saccharomyces cerevisiae. More specifically, Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of these strains may be used.<br>
As a specific example of the mevalonate pathway-related gene, hydroxymethylglutaryl-CoA reductase gene (e.g., HMGl gene) may be given.<br>
Specific examples of the prenyl diphosphate synthase gene include any gene selected from the group consisting of the following genes (a) and (b) and fusion genes (c) and (d):<br>
(a)	famesyl diphosphate synthase gene or a mutant thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of famesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added.<br>
Further, the microorganism useful in the present invention is a prototroph, a diploid cell, or a prototroph and, at the same time, a diploid cell.<br>
Further, the present invention is characterized by controlling the pH of the medium. The pH control is carried out using, e.g., ammonium gas, an ammonium salt solution, a sodium hydroxide solution or sulfuric acid.<br>
Hereinbelow, the present invention will be described in more detail.   The present<br><br>
specification encompasses the contents disclosed in the specification and/or drawings of Japanese Patent Application No. 2000-403067 based on which the present application claims priority.<br>
Using metabolic engineering techniques, the present inventors attempted to establish a system in which an active prenyl alcohol, in particular (all-E)-geranylgeraniol (hereinafter, referred to as "GGOH") is produced.<br>
It is believed that GGOH is synthesized from geranylgeranyl diphosphate (GGPP) as a precursor. Generally, simply increasing GGPP synthase activity will only results in acceleration of the synthesis of GGPP from isopentenyl diphosphate (IPP) and 3,3-dimethylallyl pyrophosphate (DMAPP), and it is unpredictable that such increasing would result in production of GGOH (Fig. 1). Besides, GGPP in vivo is known only as a precursor for the synthesis of various final products such as carotenoids and prenylated proteins (Fig. 1). Thus, even when GGPP synthesis rate is increased, the level of those final products are expected to increase, but whether an industrially valuable GGPP synthesis system can be established or not is unpredictable. Even when the expression level of HMG-CoA reductase (a key enzyme in the mevalonate pathway) is increased by enhancing the enzyme activity of HMG-CoA reductase or the activities of the enzymes mentioned in (e) through (j) above, it is unpredictable which synthesis (i.e., FPP synthesis or GGPP synthesis) would be increased; thus, the increasing of the above expression level could not be expected to be effective for GGPP synthesis. Further, according to the information accumulated so far, it cannot be expected also that the expression of the gene of FPP synthase (which catalyzes the synthesis of famesyl diphosphate (FPP), a precursor of FOH) would be effective for GGOH production (Fig. 1).<br>
In the present invention, the inventors have developed mass production systems for prenyl alcohols, in particular GGOH, by constructing recombinant DNAs to introduce prenyl diphosphate synthase genes, HMG-CoA reductase gene and/or IPPA-isomerase gene into host cells, and creating recombinants with the DNAs.<br><br>
1.    Preparation of Recombinant DNAs for Expression or DNA Fragments for Genomic Integration<br>
In the present invention, one example of the recombinant DNA for expression that is used in the transformation of hosts may be obtained by ligating or inserting a transcription promoter DNA and a transcription terminator DNA into a gene prenyl diphosphate synthase gene. It is also possible to prepare in advance a gene expression cassette comprising a prenyl diphosphate synthase gene to which a transcription promoter and a transcription terminator have been ligated, and to incorporate the cassette into a vector. The ligation or insertion of the promoter and terminator may be performed in any arbitrary order, but it is preferable to ligate the promoter upstream of the prenyl diphosphate synthase gene and he terminator downstream of the gene. Alternatively, in the present invention, a prenyl diphosphate synthase gene, a transcription promoter and a transcription terminator may be incorporated successively into an appropriate DNA, e.g., a vector. If the direction of transcription is properly considered, the incorporation may be performed in any arbitrary order.<br>
Specific examples of prenyl diphosphate synthase gene include famesyl diphosphate synthase gene (called "FPP synthase gene") and geranylgeranyl diphosphate synthase gene (called "GGPP synthase gene"). Specific examples of FPP synthase gene include Saccharomyces cerevisiae ERG20 (SEQ ID NO: 1), Escherichia coli ispA (SEQ ID NO: 3) and Bacillus stearothermophilus-derived FPP synthase genes (Japanese Unexamined Patent Publication No. 5-219961; United States Patent No. 5,786,192). Specific examples of GGPP synthase gene include Saccharomyces cerevisiae BTSI (SEQ ID NO: 5), Sulfolobus acidocaldarius crtE (Japanese Unexamined Patent Publication No. 7-308913; United States Patent No. 5,773,273) and Thermus thermophilus Tth (Japanese Unexamined Patent Publication No.9-107974; United States Patent No. 6,107,072). These genes can be obtained by conventional gene isolation methods or by using commercial kits. In the present invention, it is also possible to use a mutant of FPP synthase gene or a mutant of GGPP synthase gene.<br>
Further, in the present invention, a vector comprising a fusion gene composed of<br><br>
GGPP synthase gene or a mutant thereof and FPP synthase gene or a mutant thereof may be constructed so that the polypeptides produced by the expression of the GGPP synthase gene and the FPP synthase gene take a form of a fusion protein. In the present invention, such a gene constructed from two or more genes so that a fusion protein is produced as an expression product is called a "fusion gene". In order to prepare a fusion gene, such a method may be used in which one DNA is digested with an appropriate restriction enzyme, and then the other DNA predigested with the same restriction enzyme is ligated thereto in such a manner that the reading frame of the amino acid sequence of the protein encoded by the latter DNA is not shifted.<br>
Further, in the present invention, for the purpose of adding an endoplasmic reticulum (ER) transition signal (an amino acid sequence represented by His Asp Glu Leu (SEQ ID NO: 24); hereinafter, referred to as "HDEL sequence") to the C-terminal of the protein produced by the expression of a prenyl diphosphate synthase gene or a mutant thereof or the above-described fusion gene, a nucleotide sequence encoding the amino acid sequence may be added to the prenyl diphosphate synthase gene or the fusion gene to thereby create a modified gene.<br>
Further, in the present invention, it is also possible to produce prenyl alcohols (in particular GGOH) by transferring into a host a hydroxymethylglutaryl-CoA (HMG-CoA) reductase gene (SEQ ID NO: 7) or a mutant thereof alone or as a fusion gene with the above-mentioned prenyl diphosphate synthase gene (including a mutant thereof) and expressing the gene. Altematively, it is also possible to transfer into a host both the prenyl diphosphate synthase gene or mutant thereof and the HMG-CoA reductase gene or mutant thereof and to co-express the two genes. Specific examples of HMG-CoA reductase gene include Saccharomyces cerevisiae HMGl and HMG2,<br>
The above-described mutants of prenyl diphosphate synthase genes and HMG-CoA reductase gene may be deletion mutant genes having a deletion of one part of region (e.g., deletion of 2217 nucleotides at the maximum for HMG-CoA reductase gene), or mutant genes having a deletion, substitution or addition of one or several to ten nucleotides in the nucleotide sequences of wild type genes or the above-mentioned deletion mutant genes.<br><br>
Accordingly, the amino acid sequence encoded by such a mutant gene may have a mutation(s).    Specifically, the amino acid sequences of wild-type prenyl diphosphate synthases (FPP synthase: SEQ ID NO: 2 or 4; GGPP synthase: SEQ ID NO: 6) or the amino acid sequence of wild-type MMG-CoA reductase (SEQ ID NO: 8) may have a mutation(s), such as deletion, substitution or addition of one or several (e.g., one to ten, preferably, one to three) amino acids.   The amino acid sequence of wild-type HMG-CoA reductase (SEQ ID NO: 8) may have a deletion of 739 amino acids at the maximum, and such a deletion mutant type enzyme may further have a mutation(s), such as deletion, addition, substitution or insertion of one or several(e.g., one to ten, preferably, one to three) amino acids. Specifically, wild-type HMG-CoA reductase gene or its deletion mutants as illustrated in Fig. 2B may be used in the invention, and the amino acid sequences encoded by them may have one to ten site-specific substitutions as a result of nucleotide substitutions, such as shown in Fig. 2A.   Further, when a wild-type prenyl diphosphate synthase gene(e.g, SEQ ID NO: 1, 3 or 5) or wild-type HMG-CoA reductase gene (SEQ ID NO: 7) is amplified by PCR (polymerase chain reaction), substitution mutations of nucleotides that occur in the resultant DNA fragments due to the low fidelity of a DNA polymerase, such as Taq DNA polymerase, are called "PCR errors".    In the present invention, for example, an HMG-CoA reductase gene may also be used in which encoded polypeptide has substitution mutations attributable to nucleotide substitutions resulted fi-om PCR errors when wild-type HMG-CoA reductase gene (SEQ ID NO: 7) was used as a template; this HMG-CoA reductase gene is designated "HMG1' ".    Embodiments of nucleotide substitutions resulted from PCR errors when wild-type HMG-CoA reductase gene (SEQ ID NO: 7) was used as a template are shown in Fig. 2A.   HMGl' has the nucleotide sequence as shown in SEQ ID NO: 9, and the amino acid sequence encoded thereby is shown in SEQ ID NO; 10.   In Fig. 2A, the mutations of nucleotides are expressed in the following order: the relevant nucleotide before substitution (in one letter code), the position of this nucleotide counted taking the first nucleotide in the initiation codon of the wild-type HMG-CoA reductase gene as 1, and the nucleotide after substitution (in one letter code).     The mutations of amino acids contained in the amino acid sequence of the PCR error-type HMG-CoA reductase are expressed in the<br><br>
following order: the relevant amino acid before substitution (in one letter code), the position of this amino acid in the HMG-CoA reductase, and the amino acid after substitution (in one letter code). Further, the PCR en'or-type nucleotide sequence described above may be modified partially by techniques such as site-directed mutagenesis. Such a modified HMG-CoA reductase gene may also be used in the invention. Embodiments of nucleotide substitutions resulted from PCR errors are shown in Fig. 2A. Further, the above-described PCR error-type nucleotide sequence may be modified partially by techniques such as site-directed mutagenesis. A gene (SEQ ID NO: 11) encoding such a modified-type HMG-CoA reductase (SEQ ID NO: 12) may also be used in the invention.<br>
Further, as examples of HMG-CoA reductase genes (including PCR error-type) encoding deletion mutants in which predicted transmembrane domains are deleted, HMGIA genes that are deletion mutants of the PCR error-type HMG-CoA reductase gene HMGl' are sho2n (Fig. 2B). The upper most row in this Figure represents HMGl' gene without deletion. The portion indicated with thin solid line (—) represents the deleted region. Table 1 below shows which region of HMGT gene (SEQ ID NO: 9) has been deleted in each deletion mutant gene. HMGl' deletion mutant genes are expressed as "//MG7Axxy" according to the deletion pattern, in which "xx" represents the deletion pattern and "y" a working number (any arbitrary number). In Fig. 2B, "A026" is shovm as one example of HMGl A02y,    (Examples of other deletion patterns are also shown in a similar manner.)<br><br><br><br>
Further, in the present invention, it is also possible to produce prenyl alcohols, in particular GGOH, by transferring into a host an isopentenyl diphosphate A-isomerase (IPP A-isomerase) gene together with the above-described prenyl diphosphate synthase gene or mutant thereof Specific examples of IPP A-isomerase gene include E coli-derived idi (SEQ ID NO: 32). Specific examples of prenyl diphosphate synthase genes include E coli-derived ispAm mutant genes (Y79M: SEQ ID NO: 37; Y79E: SEQ ID NO: 35; Y79D: SEQ ID NO: 33) and Bacillus stearothermophilus-derived fpsm (Y81M: SEQ ID NO: 39). Mutant genes derived from this ispA encode mutant enzymes in which the amino acid residue Tyr at position 79 of wild-type FPP synthase (SEQ ID NO: 4) is changed to Asp (SEQ ID NO: 34), Glu (SEQ ID NO: 36) or Met (SEQ ID NO: 38) by substitution mutation.<br>
The DNA used in the invention is not particularly limited as long as it may be retained in host cells hereditarily. Specific examples of DNA that may be used include plasmid DNA, bacteriophage, retrotransposon DNA, yeast artificial chromosomal DNA (YAC: yeast artificial chromosome), etc. With respect to DNA fragments for genomic integration, these fragments do not need replication ability. Thus, DNA fragments prepared by PCR or chemical synthesis may be used.<br>
Specific examples of useful plasmid DNA include YCp-type E. coli-yeast shuttle vectors such as pRS413, pRS414, pRS415, pRS416, YCp50, pAUR112 or pAUR123; YEp-type E. coli-yeast shuttle vectors such as pYES2 or YEpl3; YIp-type E, coli-yeast shuttle vectors such as pRS403, pRS404, pRS405, pRS406, pAURlOl or pAUR135; E. coli-derived plasmids(e.g., ColE plasmids such as pBR322, pBR325, pUClS, pUC19, pUCllS, pUC119, pTVllSN, pTV119N, pBluescript, pHSG298, pHSG396 or pTrc99A; p15A plasmids such as pACYC177 or pACYC184; and pSC101 plasmids such as pMW118, pMW119, pMW218 or pMW219) and Bacillus subtilis-derived plasmids (e.g., pUBllO, pTP5). Specific examples of useful phage DNA include X phage (Charon4A, Charon21A, EMBL3, EMBL4, XgtlO, Xgtll, XZAP), (t)X174, M13mpl8 and M13mpl9. Specific examples of useful retrotransposon include Ty factor. Specific examples of YAC vectors include pYACC2.<br>
When recombinant DNAs are transferred into hosts, selectable marker genes are<br><br>
used in many cases.    However, the use of marker genes are not necessarily required if there is an appropriate assay method.<br>
As the transcription promoter, a constitutive (permanent expression type) promoter or an inducible promoter may be used. The term "constitutive promoter" means a transcription promoter of a gene involved in a major metabolic pathway. Such a promoter has transcription activity under any growth conditions. The "inducible promoter" means a promoter that has transcription activity only under specific growth conditions and whose activity is repressed under other growth conditions.<br>
Any transcription promoter may be used as long as it has activity in hosts such as yeast. For example, GALJ promoter, GALJO promoter, TDH3 (GAP) promoter, ADHJ promoter, TEF2 promoter or the like may be used for expression in yeast. For expression in E. coll, trp promoter, lac promoter, trc promoter, tac promoter or the like may be used.<br>
Further, the recombinant DNA may comprise cis-elements such as an enhancer, a splicing signal, a poly A addition signal, selectable markers, etc., if desired. Specific examples of useful selectable markers include marker genes such as URA3, LEU2, TRP J and HISS whose indicators are non-auxotrophic phenotypes, and antibiotic resistance genes such as Amp', Tef, Cm, Km' and A URl-C,<br>
A transcription terminator derived from any gene may be used as long as it has activity in hosts such as yeast. For expression in yeast, ADHl terminator, CYCl terminator or the like may be used. For expression in E. coli, rrnB terminator may be used, for example. In order to express a gene of interest in bacterial cells, an SD sequence (typically, 5'-AGGAGG-3') may also be incorporated upstream of the initiation codon of the gene as a ribosome binding site for effective translation.<br>
The expression vectors prepared in the present invention as recombinant DNAs for gene transfer may be designated and identified by indicating the name of the relevant gene after the name of the plasmid used. Table 2 shows relations between the designations of expression vectors and their constitutions when pRS435GAP was used as a plasmid. When pRS434, pRS444 and pRS445 plasmids were used in combination with the above-mentioned promoters, such relations may be described in the same manner as used<br><br><br>
When HMGl gene is ligated to plasmid pRS434GAP, the resuhant vector is expressed as "pRS434GAP-HMGl". Table 3 shows relations between the designations of expression vectors and their constitutions when pRS434GAP was used as a plasmid. When the plasmid was pRS435GAP, pRS445GAP or the like, such relations may be described in the same manner as used for pRS434GAP.<br><br><br>
2.     Preparation of Recombinants<br>
The recombinants of the invention can be obtained by transferring into hosts the recombinant DNAs of the invention in such a manner that various prenyl diphosphate synthase genes or fusion genes thereof, and/or HMG-CoA reductase gene (including mutants of these genes; the same applies to the rest of the present specification unless otherwise noted), or IPP A-isomerase gene can be expressed. The host used in the invention is not particularly limited. Any host may be used as long as it can produce a prenyl alcohol(s).    Preferably, yeast or E. coli is used.<br>
In the present invention, the recombinant DNA comprising a transcription promoter and a transcription terminator, as well as a prenyl diphosphate synthase gene, HMG-CoA reductase gene, IPP A-isomerase gene or one of the genes listed in (e) through (j) above may be introduced into fungi including unicellular eucaryotes such as yeast; procaryotes; animal cells; plant cells; etc. to obtain recombinants.<br>
Fungi useful in the invention include Myxomycota, Phycomycetes, Ascomycota, Basidiomycota, and Fungi Imperfecti. Among fungi, some unicellular eucaryotes are well knovra as yeast that is important in industrial applicability. For example, yeast belonging to Ascomycota, yeast belonging to Basidiomycota, or yeast belonging to Fungi Imperfecti may be enumerated. Specific examples of yeast useful in the invention include yeast belonging to Ascomycota, in particular, budding yeast such as Saccharomyces cerevisiae (known as Baker's yeast), Candida utilis or Pichia pastris; and fission yeast such as<br><br>
Shizosaccharomyces pombe. The yeast strain useful in the invention is not particularly limited as long as it can produce a prenyl alcohol(s). In the case of S. cerevisiae, specific examples of useful strains include A451, EUG5, EUG8, EUG12, EUG27, YPH499, YPH500, W303-1A, W303-1B, ATCC28382, AURGG101, AURGG102, AHl and YHl as shown below. As a method for transferring the recombinant DNA into yeast, such method as electroporation, the spheroplast method, or the lithium acetate method may be employed.<br><br>
in the present invention; comprising GALl promoter, BTSJ and CYCJ terminator together with AURI'C gene in AURl locus.<br>
EUG5, EUG8 (A451, ERG9p::URA3-GAL]p): A451-derived strains established in the present invention; comprising squalene synthase gene ERG9, transformant selection marker gene URA3 and transcription promoter GAL]p,<br>
EUG12 (YPH499, ERG9p::URA3-GALlp): YPH499-derived strain established in the present invention; comprising ERG9, URA3 and GALlp.<br>
EUG27 (YPH500, ERG9p::URA3-GALlp): YPH500-derived strain established in the present invention; comprising ERG9, URA3 and GALlp.<br>
AHl strain (pRS434GAP-HMGl/A451): A451-derived strain established in the present invention; pRS434GAP-'HMGl is transferred into A451.<br>
YHl strain (pRS434GAP-HMGl/YPH499): YPH499-derived strain established in the present invention; pRS434GAP-HMGl is transferred into YPH499.<br><br>
Procaryotes useful in the invention include archaea and bacteria. As archaea, methane producing microorganisms such as Metanobacterium; halophilic microorganisms such as Halobactehum, and thermophilic acidophilic microorganisms such as Sulfolobus may be enumerated. As bacteria, various Gram-negative or Gram-positive bacteria that are highly valuable in industrial or scientific applicability may be enumerated, e.g., Escherichia such as E. coli. Bacillus such as B. subtilis or B. brevis, Pseudomonas such as P. putida, Agrobacterium such as A. tumefaciens or A. rhizogenes, Corynebacterium such as C glutamicum, Lactobacillus such as L plantarum, and Actinomycetes such as Actinomyces or Streptmyces,<br>
When a bacterium such as E. coli is used as a host, preferably, the recombinant DNA of the invention is not only capable of autonomous replication in the host but also composed of a transcription promoter, an SD sequence as ribosome RNA binding site, and the gene of the invention. A transcription terminator may also be inserted appropriately into the recombinant DNA. The DNA may also contain a gene that controls the promoter. Specific examples of E". coli strains useful in the invention include, but are not limited to, BL21, DH5a, HBlOl, JMlOl, JM109, MV1184, TH2, XLl-Blue and Y-1088. As the transcription promoter, any promoter may be used as long as it can direct the expression of the gene of the invention in a host such as E. coli. For example, an E, coli- or phage-derived promoter such as trp promoter, lac promoter, PL promoter or PR promoter may be used. A promoter whose design is artificially altered may also be used. As a method for introducing the recombinant vector into a bacterium, any method of DNA transfer into bacteria may be used. For example, a method using calcium ions, electroporation, etc. may be used.<br>
Whether the gene of the invention has been introduced into the host cell or not can be confirmed by such methods as PCR (polymerase chain reaction) or Southern blot hybridization. For example, DNA is prepared from the resultant recombinant and subjected to PCR using a pair of primers specific to the transferred DNA. Subsequently, the amplified product is subjected to agarose gel electrophoresis, polyacrylamide gel electrophoresis or capillary electrophoresis, followed by staining with ethidium bromide.<br><br>
SYBR Green solution or the like, or detection of DNA with a UV detector. By detecting the amplified product as a single band or peak, the transferred DNA can be confirmed. Alternatively, PCR may be performed using primers labeled with a fluorescent dye or the like to thereby detect the amplified product.<br>
3.    Production of Prenyl Alcohols<br>
In the present invention, a prenyl alcohol(s) can be obtained by culturing the above-described recombinant comprising a prenyl diphosphate synthase gene or a mutant thereof (including a fusion gene), and/or an HMG-CoA reductase gene or a mutant thereof, or a mevalonate pathway-related enzyme gene selected from the above-described (e) through (j) transferred thereinto, and recovering the prenyl alcohol(s) from the resultant culture. The term "culture" used herein means any of the following materials: culture supernatant, cultured cells or microorganisms per se, or disrupted products from cultured cells or microorganisms. The recombinant of the invention is cultured by conventional methods used in the cultivation of its host. As a specific example of the prenyl alcohol, GGOH may be given. These prenyl alcohols are accumulated in culture independently or as a mixture.<br>
As a medium to culture the recombinant obtained from a microorganism host, either a natural or synthetic medium may be used as long as it contains carbon sources, nitrogen sources and inorganic salts assimilable by the microorganism and is capable of effective cultivation of the recombinant. As carbon sources, carbohydrates such as glucose, galactose, fructose, sucrose, raffinose, starch; organic acids such as acetic acid, propionic acid; and alcohols such as ethanol and propanol may be enumerated. As nitrogen sources, ammonia; ammonium salts of inorganic or organic acids such as ammonium chloride, ammonium sulfate, ammonium acetate, ammonium phosphate; other nitrogen-containing compounds; Peptone; meat extract; com steep liquor, various amino acids, etc. may be enumerated. As inorganic substances, potassium dihydrogen phosphate, dipotassium hydrogen phosphate, magnesium phosphate, magnesium sulfate, sodium chloride, iron(ll) sulfate, manganese sulfate, copper sulfate, calcium carbonate, etc. may be<br><br>
enumerated. Usually, the recombinant is cultured under aerobic conditions (such as shaking culture or aeration agitation culture) at 26 to 42°C. Preferably, when the host is S. cerevisiae, the recombinant is cultured at 30°C for 2 to 7 days; when the host is E. coli, the recombinant is cultured at 37°C for 12 to 18 hours. The adjustment of pH is carried out using an inorganic or organic acid, an alkali solution, etc.<br>
When a recombinant integrating an expression vector containing an inducible transcription promoter is cultured, an inducer may be added to the medium if necessary. For example, when GALl promoter was used in the vector, galactose may be used as a carbon source. When a microorganism(E. coli) transformed with an expression vector containing a promoter inducible by isopropyl-p-D-thiogalactopyranoside (IPTG) is cultured, IPTG may be added to the medium.<br>
When cultured under the above-described conditions, the host can produce a prenyl alcohol(s) at a high yield(s). For mass-production of prenyl alcohols, a jar fermenter culture apparatus may be employed. In particular, when the host is S. cerevisiae YPH499 and the transferred plasmid DNA is pRS435GGF or pRS434GAP-HMGl, the recombinant can produce 1.5 mg or more of a prenyl alcohol per liter of the medium; depending on culture conditions, the recombinant can produce 128 mg or even more.<br>
In the present invention, it is possible to increase the production efficiency of prenyl alcohols by adding to the above-described medium such substances as terpenoids, oils, or surfactants.    Specific examples of these additives include the following.<br>
Terpenoids: squalene, tocopherol, IPP, DMAPP<br>
Oils: soybean oil, fish oil, almond oil, olive oil<br>
Surfactants: Tergitol, Triton X-305, Span 85, ADEKANOL LG109 (Asalii Denka), ADEKANOL LG294 (Asahi Denka), ADEKANOL LG295S (Asahi Denka), ADEKANOL LG297 (Asahi Denka), ADEKANOL B-3009A (Asahi Denka), ADEKA PLURONIC L-61 (Asahi Denka)<br>
The concentration of oils is 0.01% or more, preferably 1-3%. The concentration of surfactants is 0.005-1%, preferably 0.05-0.5%. The concentration of terpenoids is 0.01% or more, preferably 1-3%.<br><br>
Further, in the present invention, it is also possible to culture a microorganism having an ability to produce a prenyl alcohol using a medium comprising any one of the components (i) through (vi) described below and recovering the prenyl alcohol from the resultant culture. Further, fed-batch culture may be conducted using a feed solution comprising any one of the components (i) through (v) described below.<br>
(i)    sugar<br>
(ii)   alcohol<br>
(iii)   ammonia gas, aqueous ammonia and/or an ammonium salt<br>
(iv)    a mixture of sodium hydroxide and sulfuric acid<br>
(v)    a mixture of KH2PO4, magnesium sulfate, ammonium sulfate, com steep liquor, calcium chloride and a surfactant<br>
(vi)    a mixture of two or more of the above components (i) through (v).<br>
Specific examples of the above sugar include glucose, sucrose, galactose and lactose. Specific examples of the above alcohol include methanol, ethanol, propanol, isopropanol and butanol.<br>
As carbon source components in the feed solution, a combination of glucose and ethanol is preferable, and it is more preferable to add to the medium ammonia gas or an ammonium salt (e.g., ammonium acetate) for pH control. As a method of adding a feed solution, it is preferable to use a feed solution whose carbon source is glucose alone up to 12-24 hours after the start of the cultivation and then to shift to another feed solution containing ethanol in its carbon source component. Alternatively, glucose may remain the sole carbon source throughout the cultivation. The ratio of ethanol to the total carbon source of the feed solution may be any ratio. The ratio may be 50% or more, or even 100%.<br>
Strains that are able to propagate without supplementation of specific nutrients to the medium are called prototrophs. Generally, a prototroph is a strain that has the same phenotype as that of its corresponding wild-type strain in nutritional requirements. On the other hand, auxotrophs (auxotrophic mutant strains) are frequently used as host strains for creating recombinants.   The phenotype of such an auxotroph may be changed to the same<br><br>
phenotype as that of the corresponding prototroph by complementing the auxotrophic mutation. Briefly, a wild-type gene corresponding to the mutant gene causing the auxotrophic mutation is transferred into the auxotroph. When the wild-type gene is dominant to the mutant gene causing the auxotrophic mutation, it is also possible to complement the mutation by mating or conjugating the auxotroph with a strain having the wild-type gene. In the present invention, a prototroph can be obtained, for example, by replacing some of the mutant genes causing nutritional requirements in the genome of a GGOH producing clone (YHl strain comprising a fusion gene composed of GGPP synthase gene and FPP synthase gene) with corresponding wild-type genes, and then mating the resultant clone with a YPH500-derived clone that has dominant wild-type genes to the remaining mutant genes. In the present invention, it is preferable to use a microorganism that is a diploid cell and, at the same time, a prototroph.<br>
After the cultivation, the prenyl alcohol of interest is recovered by disrupting microorganisms or cells by, e.g., homogenizer treatment, if the alcohol is produced within the microorganisms or cells. Alternatively, the alcohol may be extracted directly using organic solvents without disrupting the cells. If the prenyl alcohol of the invention is produced outside the microorganisms or cells, the culture broth is used as it is or subjected to centriftigation, etc. to remove the microorganisms or cells. Thereafter, the prenyl alcohol of interest is extracted from the culture by, e.g., extraction with an organic solvent. If necessary, the prenyl alcohol may be further isolated and purified by various types of chromatography or the like.<br>
In the present invention, preferable combinations of host strains and vectors, as well as relationships between these combinations and yields of prenyl alcohols are as illustrated in Table 4 below.<br><br><br><br><br><br><br><br><br><br>
(i) When plasmid pRS445GG is prepared by integrating GGPP synthase gene BTSl into pRS445GAP and transferred into A451 or YPH499 strain as a host, GGOH yield increases (about 0.4 mg/L on the average).<br>
(ii) When plasmid pRS435FGG pRS445FGG pRS435GGF or pRS445GGF is prepared by integrating a fusion gene composed of GGPP synthase gene BTSl and FPP synthase gene ERG20 into plasmid pRS435GAP or pRS445GAP and transferred into A451 or YPH499 strain as a host; or when plasmid pRS435FGGHDEL, pRS445FGGHDEL or pRS435GGFHDEL comprising a nucleotide sequence encoding an HDEL sequence ligated to one end of the above fusion gene(i.e., plasmid comprising a gene modified so that an HDEL sequence is added to the C-terminal of the polypeptide to be produced by the expression of the fusion gene) is prepared and transferred into A451 or YPH499 strain; 0.20 mg/L of GGOH is produced on the average with ERG20-BTS1 fusion; 0.39 mg/L of GGOH is produced on the average with BTS1-ERG20 fusion, and 0.62 mg/L of GGOH is produced on the average with BTS1-ERG20-HDEL fusion.<br>
(iii) When plasmid pRS434GAP-HMGl (pRS434GAP integrating HMG-CoA reductase gene HMGl) and plasmid pRS435GGF comprising the above-described fusion gene are transferred into YPH499 as a host and co-expressed therein, the recombinant produces 1.55 mg/L of GGOH on the average. When this recombinant is cultured in YMO medium (YM medium supplemented vdth soybean oil, etc.) at 30°C for 7 days, it produces 5.61 mg/L of GGOH.<br>
(iv) When both pRS435GGFHDEL and pRS434GAP.HMGl are transferred into YPH499 as a host and co-expressed therein, the recombinant produces 1.50 mg/L of GGOH on the average. When this recombinant is cultured in YMO medium at 30°C for 7 days, it produces 5.64 mg/L of GGOH.<br>
(v) When both pRS435GGF and pRS434GAP-HMGl are transferred into YPH499 as a host and the resultant recombinant is cultured in a jar fermenter for 109 hours, the recombinant produces 128 mg/L of GGOH.<br>
(vi) When HMG-CoA reductase gene and GGF fusion gene are co-expressed, most of the recombinant clones produce 100 mg/L or more of GGOH, and produce 189 mg/L of<br><br>
GGOH at the maximum.<br>
(vii) When pRS435GGF/YH3 clone obtained by converting a clone co-expressing HMG-CoA reductase gene and GGF fusion gene into a prototroph and then diploidizing is subjected to fed-batch culture, the clone produces 0.47 g/L of GGOH when 500 g/L glucose solution is used as a feed solution after 20 hours after the start of cultivation; and produces 1.16 g/L of GGOH when 400 g/L ethanol solution is used as the feed solution.<br>
(viii) When pRS435GGF/YH3 clone is subjected to fed-batch culture, the clone produces 2.5 g/L of GGOH under the following conditions: the ratio of ethanol to the total carbon source of the feed solution after 21 hours after the start of cultivation is 71%, and ammonium acetate is added to the feed solution.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig.  1  is a chart showing metabolic pathways of mevalonate pathway-related enzymes.<br>
Fig. 2 A is a table showing a pattern of substitution mutations.<br>
Fig. 2B is a construction diagram for deletion type HMGl genes.<br>
Fig. 3 is a diagram showing plasmid pRS414.<br>
Fig. 4 is a diagram showing plasmid pYES2.<br>
Fig. 5 shows the sequences of ADHJ promoter and terminator.<br>
Fig. 6A is a diagram showing plasmid pRS414PTadh.<br>
Fig. 6B is a diagram showing plasmid pRS414TPadh.<br>
Fig. 7A is a diagram showing plasmid pRS434GAP.<br>
Fig. 7B is a diagram showing plasmid pRS434TEF.<br>
Fig. 7C is a diagram showing plasmid pRS435GAP.<br>
Fig. 7D is a diagram showing plasmid pRS444GAP.<br>
Fig. 7E is a diagram showing plasmid pRS444TEF.<br>
Fig. 7F is a diagram showing plasmid pRS445GAP.<br>
Fig. 8 is a diagram showing the direction of each of the mevalonate pathway-related<br><br>
enzymes inserted into pT7 vector.<br>
Fig. 9 is a physical map of plasmid pALHMG106.<br>
Fig. 10 is a diagram showing the restriction enzyme recognition sites on ORF182 fragment.<br>
Fig. 11 is a diagram showing expression vectors for B. stearothermophilus FPP synthase mutant gene (Y81M).<br>
Fig. 12 presents photographs showing results of Southern blot hybridization.<br>
Fig. 13 presents photographs showing results of PCR mapping.<br>
Fig. 14 presents photographs showing results of Northern blotting.<br>
Fig. 15A presents graphs showing the specific activities of prenyl diphosphate synthases in crude enzyme solution.<br>
Fig. 15B presents graphs showing the specific activities of prenyl diphosphate synthases in crude enzyme solution.<br>
Fig. 16 is a graph showing the GGOH yields of recombinants obtained by transferring into A451 HMGl gene to which a constitutive promoter is ligated.<br>
Fig, 17 is a graph showing the GGOH yield of each recombinant obtained by using A451 or AURGGlOl (each retaining YEp expression vector comprising GALIp-HMGI) as a host.<br>
Fig. 18 is a graph showing the GGOH yield of each recombinant obtained by transferring plasmid pYES2-HMG into AURGG102 or AURGG703.<br>
Fig. 19 is a graph showing the GGOH yield of each recombinant when a deletion-type HMGl '  gene is inserted into G/4Z7p-containing pYES2 vector.<br>
Fig. 20 is a graph showing the GGOH yield of each recombinant when a deletion-type HMGl '  gene is inserted into G/4Z7p-containing pYES2 vector.<br>
Fig. 21 is a graph showing the GGOH yield of each recombinant when a deletion-type HMGl'  gene is inserted into G^Z7p-containing pYES2 vector.<br>
Fig. 22 is a graph showing the GGOH yield of each recombinant when a deletion-type HMGl '  gene is inserted into GAlIp-containing pYES2 vector.<br>
Fig. 23 is a graph showing the GGOH yields of recombinant E, coli retaining p4M,<br><br>
pl6M, etc. when cultured in a medium containing IPP and DMAPR<br>
Fig. 24 is a diagram showing the primers used in the creation of BTSI-ERG20 fusion genes, as well as the locations and directions of these primers.<br>
Fig. 25 is a graph showing the results of determination of GGOH yields when ERGIO'BTSl fusion genes were transferred into A451-derived clones.<br>
Fig. 26 is a graph showing the results of determination of GGOH yields when ERG20'BTS1 fusion genes were transferred into YPH499-derived clones.<br>
Fig. 27 is a graph showing the results of determination of GGOH yields in rEF2p-HMG7-transferred YPH499-derived clones.<br>
Fig. 28 is a graph showing the results of determination of GGOH yields in TDH35p-HMG7-transferred YPH499-derived clones.<br>
Fig. 29A is a graph showing the results of determination of GGOH yields in A451-derived clones.<br>
Fig. 29B is a graph showing the results of determination of GGOH yields in A451-derived clones.<br>
Fig. 30A is a graph showing the results of determination of GGOH yields in YPH499-derived clones.<br>
Fig. 3 OB is a graph showing the results of determination of GGOH yields in YPH499-derived clones.<br>
Fig. 31A is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred A451 strain when cultured for 2 days with indicated sugar compositions.<br>
Fig. 31B is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred A451 strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 31C is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred A451 strain when cultured for 7 days with indicated sugar compositions.<br>
Fig.   32A   is   a   graph   showing   the   GGOH   yields   of   pRS435GGF-   or<br><br>
pRS435GGFHDEL-transferred AHl strain when cultured for 2 days with indicated sugar compositions.<br>
Fig. 32B is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred AHl strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 32C is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred AHl strain when cultured for 7 days with indicated sugar compositions.<br>
Fig. 33A is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred EUG5 strain when cultured for 2 days with indicated sugar compositions.<br>
Fig. 33B is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred EUG5 strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 33C is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL"transferred EUG5 strain when cultured for 7 days with indicated sugar compositions.<br>
Fig. 34A is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred YPH499 strain when cultured for 2 days with indicated sugar compositions.<br>
Fig. 34B is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL4ransferred YPH499 strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 34C is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred YPH499 strain when cultured for 7 days with indicated sugar compositions.<br>
Fig. 35A is a graph showing the GGOH yields of pRS435GGF" or pRS435GGFHDEL-transferred YHl strain when cultured for 2 days with indicated sugar compositions.<br><br>
Fig. 35B is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred YHl strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 35C is a graph showing the GGOH yields of pRS435GGF" or pRS435GGFHDEL-transferred YHl strain when cultured for 7 days with indicated sugar compositions.<br>
Fig. 36A is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred EUG12 strain when cultured for 2 days with indicated sugar compositions.<br>
Fig. 36B is a graph showing the GGOH yields of pRS435GGF" or pRS435GGFHDEL-transferred EUG12 strain when cultured for 4 days with indicated sugar compositions.<br>
Fig. 36C is a graph showing the GGOH yields of pRS435GGF- or pRS435GGFHDEL-transferred EUG12 strain when cultured for 7 days with indicated sugar compositions.<br>
Fig. 37 is a graph showing the results of determination of prenyl alcohol yields in pRS435GGFA'Hl (pRS434GAP-HMGlA^PH499) clone when cultured in a soybean oil-containing medium in ajar fermenter.<br>
Fig. 38 is a graph showing the results of determination of prenyl alcohol yields in 15-2 clone when cultured in a soybean oil-containing medium in a jar fermenter.<br>
Fig. 39 presents photographs showing the results of Northern blot hybridization conducted in order to confirm the expression of fusion genes.<br>
Fig. 40 presents photographs showing the results of Western blotting.<br>
Fig. 41A presents graphs showing the results of determination of GGOH yields in clones co-expressing HMG-CoA reductase gene and a fusion gene composed of GGPP synthase gene and FPP synthase gene.<br>
Fig. 41B presents graphs showing the results of determination of GGOH yields in clones co-expressing HMG-CoA reductase gene and a fusion gene composed of GGPP synthase gene and FPP synthase gene.<br><br>
Fig. 41C presents graphs showing the results of determination of GGOH yields in clones co-expressing HMG-CoA reductase gene and a fusion gene composed of GGPP synthase gene and FPP synthase gene.<br>
Fig. 42 is a graph showing flow rate conditions of a feed solution.<br>
BEST MODES FOR CARRYING OUT THE INVENTION<br>
Hereinbelow, the present invention will be described more specifically with reference to the following Examples. However, the technical scope of the present invention is not limited to these Examples.<br>
The Examples encompass the following contents.<br>
(1)	Expression vectors, such as pRS435GAP, are prepared using pRS vectors (Stratagene), pYES vector (Invitrogen), Saccharomyces cerevisiae YPH499-derived genomic DNA, etc.<br>
(2)	Cloning of Mevalonate Pathway-Related Enzyme Genes<br>
Acetyl-CoA acetyltransferase gene, HMG-CoA reductase gene or mutants thereof, mevalonate kinase gene, phosphomevalonate kinase gene, diphosphomevalonate decarboxylase gene, isopentenyl diphosphate A-isomerase gene, famesyl diphosphate synthase gene or substitution-mutants thereof, and geranylgeranyl diphosphate synthase gene are cloned, and then their expression vectors are prepared.<br>
(3)	Plasmids comprising mevalonate pathway-related enzyme genes are transferred into<br>
hosts such as A451, YPH499, etc. Also, mutant strains (EUG strains) were created by<br>
replacing the ERG9 transcription promoter in the genomic DNA of A451 or YPH499 with<br>
pYES2"derived GALl transcription promoter, and used as hosts for transferring prenyl<br>
diphosphate synthase genes thereinto.<br>
(4)	AURGG, which is an A451-derived strain integrating AURl-C gene, and AURGG102, which is an A451-derived strain comprising GALl promoter, BTSl and CYCl terminator together with AURl-C gene in its AURl locus, are created and used as hosts for gene transfer.<br>
(5)	Prenyl diphosphate synthase gene expression vectors are transferred into hosts, which<br><br>
are then cultured in YM7 medium (YM medium whose pH is adjusted to 7 with NaOH), YMO medium, IPP and DMAPP-containing medium, etc. Each culture broth is subjected to extraction to isolate and quantitatively determine prenyl alcohols (in particular GGOH). Recombinants that have shown favorable results are cultured in a jar fermenter for a long period to obtain prenyl alcohols in large quantities. Further, effects upon GGOH production when recombinants are converted into prototrophs and then diploidized are examined. Also, effects of compounds (such as ethanol or ammonia) added to the medium upon GGOH production are examined.<br>
[EXAMPLE 1 ]   CONSTRUCTION OF EXPRESSION VECTORS<br>
(1)	E. coli-S, cerevisiae Shuttle Vectors<br>
Plasmids pRS404, pRS404 and pRS414 (Fig. 3) were purchased from Stratagene. Plasmid pAUR123 was purchased from Takara, and plasmid pYES2 (Fig. 4) was purchased from Invitrogen (Carlsbad, CA).<br>
(2)	Genomic DNA<br>
S. cerevisiae genomic DNA was prepared from 5. cerevisiae YPH499 using Dr. GenTLE (a genomic DNA preparation kit for yeast) purchased from Takara and according to the protocol attached to the kit.<br>
E. coli genomic DNA was prepared from E. coli JM109 (Takara) by the following procedures. Briefly, JM109 cells were cultured in 1.5 ml of 2x YT medium and harvested by centrifugation. To these cells, 567 μl of TE (pH 8.0), 3 |4.1 of 20 mg/ml proteinase K (Boehringer Mannheim, Mannheim, Germany) and 30 μ1 of 10% SDS were added. The resultant mixture was kept at 37°C for 1 hr, and then 100 Μ1 of 5M NaCI was added thereto and mixed. Eighty μl of CTAB/NaCl solution (10% CTAB, 0.7 M NaCl) was added thereto, and the resultant mixture was heated at 65°C for 10 min. This mixture was extracted with 700μl of chloroform/isoamyl alcohol (24:1), and the aqueous layer was further extracted with 600 |μl of phenol/chloroform/isoamyl alcohol (25:24:1).   After the<br><br>
extraction, 0.6 volumes of isopropanol was added to the aqueous layer, which was then centrifuged. The precipitate fraction was washed with 70% ethanol, dried, and then dissolved in 100 μl of TE (pH 8.0) to thereby obtain an E. coli genomic DNA solution. OD260 of the DNA was measured, and the DNA was quantitatively determined. Then, TE was added to the solution to give a DNA concentration of lμg/μl.<br>
Plasmid DNA from E. coli was prepared using Wizard PureFection Plasmid DNA Purification System (Promega, Madison, WI).<br>
(3)	Insertion of ADHlp-ADHh Fragment into pRS414<br>
Plasmid pRS414 (Fig. 3) was digested with Nael and pvull to obtain a 4.1 kbp fragment without fl ori and LacZ moieties. This fragment was purified by agarose gel electrophoresis. Plasmid pAUR123 was digested with BamUl and blunt-ended with Klenow enzyme. Then, a 1.0 kbp fragment containing ADHl transcription promoter (ADHJp) and ADHl transcription terminator (ADHlt) (Fig. 5; SEQ ID NO: 23) was purified by agarose gel electrophoresis. The 4.1 kbp fragment from pRS414 still retained the replication origins for E. coli and yeast, a transformation marker Amp' for E, coli, and an auxotrophic marker TRPl for yeast. On the other hand, the 1.0 kbp fragment from pAUR123 contained ADH]p, ADHlt, and a cloning site flanked by them. These two fragments were ligated to each other with a DNA ligation kit (Takara) and transformed into E, coli SURE2 supercompetent cells (Stratagene, La Jolla, CA).<br>
Plasmid DNA was prepared from the resultant recombinant. Mapping of the DNA with Sail and Seal revealed that the ADHlp-ADHt fragment has been inserted into pRS414 in two directions to thereby yield two plasmids pRS414PTadh (Fig. 6A) and pRS414TPadh (Fig. 6B).<br>
(4)	Insertion of C7C7t Fragment into pRS Vectors<br>
CYCJt (CYCl transcription terminator) fragment was prepared by PCR. The following oligo-DNAs, XhoI-TcyclFW and Apal-TcyclRV, were used as PCR primers. As a template, pYES2 was used.<br><br>
XhoI-TcyclFW: 5'- TGC ATC TCG AGG GCC GCA TCA TGT AAT TAG -3' (SEQ<br>
ID NO: 40)<br>
Apal-TcyclRV: 5'- CAT TAG GGC CCG GCC GCA AAT TAA AGC CTT CG -3'<br>
(SEQ ID NO: 41)<br>
Briefly, 50 μ1 of a reaction solution containing 0.1 )ig of pYES2, 50 pmol of each primer DNA, Ix Pfu buffer containing MgSO4 (Promega, Madison, WI), 10 nmol dNTPs, 1.5 units of P/w DNA polymerase (Promega) and 1 μl of Perfect Match polymerase enhancer (Stratagene) was prepared. The reaction conditions were as follows: first denaturation at 95°C for 2 min; 30 cycles of denaturation at 95°C for 45 sec, annealing at 60°C for 30 sec, and extension at 72°C for 1 min; and final extension at 72°C for 5 min. After completion of the reaction, the solution was stored at 4°C. The amplified DNA was digested with Xhol and Apal, and the resultant 260 bp DNA fragment was purified by agarose gel electrophoresis to obtain CYClt-XA.<br>
CYClt-XA was inserted into the Xhol-Apal site of pRS404 and pRS406 to thereby obtain pRS404Tcyc and pRS406Tcyc, respectively.<br>
(5)    Preparation of Transcription Promoters<br>
DNA fragments comprising transcription promoters were prepared by PCR using pAUR123 or yeast genomic DNA as a template.   The DNA primers used are as follows.<br>
SacI-Ptdh3FW: 5'-CAC GGA GCT CCA GTT CGA GTT TAT CAT TAT CAA-3'<br>
(SEQ ID NO: 42)<br>
SacII-Ptdh3RV: 5'-CTC TCC GCG GTT TGT TTG TTT ATG TGT GTT TAT TC-3'<br>
(SEQ ID NO: 43)<br>
SacI-Ptef2FW: 5'-CCG CGA GCT CTT ACC CAT AAG GTT GTT TGT GAC G-3'<br>
(SEQ ID NO: 44)<br>
SacII-Ptef2RV: 5'-CTT TCC GCG GGT TTA GTT AAT TAT AGT TCG TTG<br><br>
ACC-3'(SEQ ID NO:45)<br>
For the amplification of ADHI transcription promoter (ADHlp), SacI-PadhlFW and SacII-PadhlRV were used as PCR primers and pAUR123 as a template. For the amplification of TDH3 (GAP) transcription promoter (TDH3p (GAPp)), SacI-Ptdh3FW and SacII-Ptdh3RV were used as PCR primers; and for the amplification of TEF2 transcription promoter (TEF2p), SacI-Ptef2FW and SacII-Ptef2RV were used as PCR primers. For these promoters, yeast genomic DNA was used as a template. As a reaction solution, a 100 |il solution containing 0.1μg of pAUR123 or 0.46 [xg of yeast genomic DNA, 100 pmol of each primer DNA, 1 x ExTaq buffer (Takara), 20 nmol dNTPs, 0.5 U of ExTaq DNA polymerase (Takara) and 1 μl of Perfect Match polymerase enhancer was prepared. The reaction conditions were as follows: first denaturation at 95°C for 2 min, then 30 cycles each consisting of 45 sec at 95°C5 1 min at 60°C and 2 min at 72oC, and final extension at 72°C for 4 min. After completion of the reaction, the solution was stored at 4°C. The amplified 4 types of DNAs were digested with Sad and Sacll, and the resultant 620 bp, 680 bp, 710 bp and 400 bp DNA fragments were purified separately by agarose gel electrophoresis to thereby obtain TDHip and TEF2p, respectively.<br>
(6)	Preparation of 2μDNA Replication Origin Region<br>
pYES2, which is a YEp vector, was digested with Sspl and Nhel. The resultant 1.5 kbp fragment containing 2μDNA replication origin(2\xori) was purified by agarose gel electrophoresis and then blunt-ended.    This DNA fragment was designated 2μOriSN.<br>
(7)	Preparation of YEp Type Expression Vectors<br>
2μOriSN was inserted into the Nael site of pRS404Tcyc and pRS406Tcyc pretreated with BAP (bacterial alkaline phosphatase: Takara). The resultant plasmids were transformed into E. coli SURE2, and then plasmid DNA was prepared. The plasmid DNA was digested with Dralll; and EcoRI, Hpal or Pstl; and PvwII, followed by agarose gel electrophoresis to examine the insertion and the direction of 2μ ori.    The resultant<br><br>
pRS404Tcyc and pRS405Tcyc into which 2μ ori had been inserted in the same direction as in pYES2 were designated pRS434Tcyc2μOri and pRS435Tcyc2μOri, respectively.    The resuhant pRS404Tcyc and pRS405Tcyc into which 2μ ori had been inserted in the opposite direction to that in pYES2 were designated pRS444Tcyc2μOri and pRS445Tcyc2|iOri, respectively.<br>
A transcription promoter-containing fragment, i.e., ADHJp, TDH3p (GAPp), PGKJp or TEF2p, was inserted into the Sacl-Sacll site of the above-described four plasmids pRS434Tcyc2μOri, pRS435Tcyc2μOri, pRS444Tcyc2μOri and pRS445Tcyc2μOri to clone the DNA. As a result, the following plasmids were obtained: (i) pRS434GAP and pRS434TEF from pRS434Tcyc2μOri; (ii) pRS435GAP from pRS435Tcyc2μOri; (iii) pRS444GAP and pRS444TEF from pRS444Tcyc2μOri; (iv) pRS445GAP from pRS445Tcyc2μOri (Figs. 7A-7F).<br>
The expression vectors prepared in the present invention are summarized in Table 5 below.<br><br><br>
♦ The "+" and "-" marks appearing after markers and gene expression transcription units indicate downstream and direction upstream direction, respectively. The "+" mark appearing after ori indicates that the ori is inserted in the same direction as that in pRS (for YCp vectors) or pYES (for YEp vectors); the "-" mark indicates that the ori is inserted in the direction opposite to that in pRS (for YCp vectors) or pYES (for YEp vectors).<br>
(8)   Introduction of YEp Type Expression Vectors into Yeast<br>
In order to examine whether the DNA replication region of the prepared YEp type expression vectors functions or not, about 40 ng of each YEp type expression vector was introduced into YPH499 strain using Frozen-EZ Yeast Transformation II (Zymo Research, Orange, CA). (The procedures followed the protocol attached to the kit.) Then, colonies growing on SD-W (DOB+CMS (-Tip); BIOlOl, Vista, CA) agar plate at 30°C were examined.    The results are shown in Table 6 below.<br><br><br><br>
The results shown in Table 6 revealed that each of the YEp type vectors prepared in the invention is retained normally as a vector.<br>
[EXAMPLE   2]      CLONING  OF   MEVALONATE  PATHWAY-RELETED   ENZYME<br>
GENES In the cloning of genes from yeast cDNA, an S. cerevisiae DBY746-derived cDNA library "Quick-Clone cDNA" purchased from Clontech (Palo Alto, CA) was used.<br>
(1)   Cloning of Famesy 1 Diphosphate Synthase Genes<br>
(1-1)   Saccharomyces cerevisiae-derived FPP synthase gene ERG20:<br>
An approximately 0.9 kbp DNA fragment encoding S. cerevisiae FPP synthase gene ERG20 (SEQ ID NO: 1) was amplified by PCR (polymerase chain reaction) using the above cDNA as a template.    The PCR primers used are as follows.<br>
Primer 1 (SCFPSl): 5'-ATG GCTTCAGAAAAAGAAATTAG-3'<br>
(SEQ ID NO: 46)<br>
Primer 2 (SCFPS2): 5'-CTATTT GCT TCT CTT GTA AAC TT-3'<br>
(SEQ ID NO: 47)<br><br><br>
The PCR was carried out in the reaction solution described above for 30 cycles each consisting of 45 sec at 94°C, 1 min at 55°C and 2 min at 72oC.<br>
Unless otherwise indicated, PCR reactions in the following Examples were carried out under the same conditions as described above.<br>
The amplified fragment was purified by agarose gel electrophoresis and then cloned into pT7Blue-T (Novagen, Madison, WI) by T/A ligation. It was found that ERG20 was inserted into pT7Blue-T in the same direction as that of locZ in this plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was determined and compared to the corresponding nucleotide sequence registered in SGD (Saccharomyces Genome Database, http://genome-www.stanford.edu/Saccharomvces/). As a result, no PCR errors were found in nucleotide positions 1-300 and 610-1059.<br>
The plasmid DNA prepared was designated pT7ERG20. (1-2)   Escherichia coli-derived FPP Synthase Gene ispA:<br>
E. coli-derived FPP synthase gene ispA (SEQ ID NO: 3) was cloned by PCR using E. coli genomic DNA as a template and the following synthetic oligo DNAs as primers.<br>
ISPAl: 5'-TGA GGC ATG CAA TTT CCG CAG CAA CTC G-3' (SEQ ID NO: 48)<br>
ISPA2: 5'-TC AGA ATT CAT CAG GGG CCT ATT AAT AC-3' (SEQ ID NO: 49)<br>
The PCR was carried out in 100μl of a reaction solution containing Ix EXTaq buffer, 0.5 mM dNTPs 100 pmol ISPAl, 100 pmol ISPA2, 0.2 μg of E. coli genomic DNA and 5 units of ExTaq for 30 cycles each consisting of 1 min at 94°C, 1 min at 55°C and 1.5 min at 72°C. The PCR product was digested with EcoRl and Sphl, and subjected to agarose gel electrophoresis to purify a 1.0 kbp DNA fragment. This fragment was inserted into the EcoRl-Sphl site of pALTER-Ex2 (Promega), which was then transferred into E. coli JM109 for cloning the gene. As a result, plasmids pALispA4, pALispAS, pALispA15, pALispA16 and pALispAlS were obtained; restriction enzyme mapping with EcoRI, SphI,<br><br>
Ndel, Smal and BamHl confirmed that ispA gene was transferred correctly into these<br>
plasmids.<br>
(1-3) Bacillus stearothermophilus-derived  FPP Synthase Gene<br>
pFE15 disclosed in Japanese Unexamined Patent Publication No. 5-219961was digested with Noil and Smal, followed by purification of an FPP synthase gene fragment containing a 2.9 kbp transcription unit. This gene fragment was inserted into the Seal site of pACYC177 (Nippon Gene) to thereby prepare an expression vector comprising B. stearoihermophilus-derived FPP synthase gene fps  (SEQ ID NO: 25).<br>
(2)    Cloning of Geranylgeranyl Diphosphate Synthase Gene<br>
S. cerevisiae-derived GGPP synthase gene BTSl (SEQ ID NO: 5) was cloned as described below.<br>
Briefly, based on information about the S, cerevisiae-derived GGPP synthase gene registered at the GenBank (http://www.ncbi.nlm.nih.gov/Genbank/index.html/) (A.N.U31632) (Y. Jiang, ei al, 1 Biol Chem. 270 (37), 21793-21799 (1995)), a pair of primers matching the N-terminal and C-terminal of the protein encoded by the gene were designed. Using these primers and a yeast cDNA library (Clontech; No. CL7220-1) as a template, PCR was carried out.<br>
N-terminal primer: 5'-ATG GAG GCC AAG ATA GAT GAG CT-3'<br>
(SEQ ID NO: 50)<br>
C-terminal primer: 5'-TCA CAA TTC GGA TAA GTG GTC TA-3'<br>
(SEQ ID NO: 51)<br>
The PCR was performed using Perfect Match Polymerase Enhancer (Stratagene) for 30 cycles each consisting of denaturation for 45 sec at 94°C, annealing for 1 min at 55°C and extension for 2 min at 72°C.<br>
A fragment of interest (approx. 1.0 kbp) was confrmed. This BTSl fragment was cloned into pT7Blue T vector capable of TA cloning, followed by sequencing of the entire region of BTSL    The results revealed that the nucleotide sequence of this gene was<br><br>
completely identical with the corresponding nucleotide sequence registered at the GenBank (SEQ ID NO: 5). Thus, it was confirmed that this gene is the S. cerevisiae-derived GGPP synthase gene.<br>
(3)	Cloning of Acetyl-CoA Acetyltransferase Gene<br>
An approximately 1.2 kbp genomic DNA fragment encoding S. cerevisiae acetyl-CoA acetyltransferase gene ERGIO (SEQ ID NO: 26) was amplified by PCR using ExTaq DNA polymerase. The resultant fragment was cloned into the Sacll-Xbal site of pRS435GAP and pRS445GAR   The PCR primers used are as follows.<br>
Primer 1 (SacII-ERG10): 5'-TCC CCG CGG ATG TCT CAG AAC GTT TAC ATT<br>
GT-3'(SEQ ID NO:52)<br>
Primer 2 (Xbal-ERGIO): 5'-TGC TCT AGA TCA TAT CTT TTC AAT GAC AAT<br>
GGA-3'(SEQ ID NO:53)<br>
(Underlined portions indicate restriction enzyme recognition sites.)<br>
The PCR was performed in the co-presence of Perfect Match Polymerase Enhancer for 30 cycles each consisting of 45 sec at 95°C, 1 min at 60°C and 2 min at 72°C. The resultant plasmids were subjected to Smal, Seal, Ncol and BamHI recognition site mapping to examine whether they were prepared as designed. The successfully prepared plasmids were designated pRS43 5 GAP-ERG 10 and pRS445 GAP-ERG 10, respectively.<br>
(4)	Cloning of HMG-CoA Synthase Gene<br>
An approximately 1.5 kbp fragment encoding S, cerevisiae HMG-CoA synthase gene HMOS (SEQ ID NO: 27) was amplified by PCR using cDNA as a template. For the amiealing temperature, 50°C was employed.    The PCR primers used are as follows.<br>
Primer 1 (HMGS-1-2): 5'-ATGAAACTCTCAACTAAACTTTGTT-3' (SEQ ID NO: 54)<br><br>
Primer 2 (scHMGS-15): 5'-GTT CAG CAA GAT GCA ATC GAT GGG G-3' (SEQ ID NO:55)<br>
The PCR fragment was purified by agarose gel electrophoresis and then cloned into pT7Blue-T by T/A ligation. It was found that HMGS was inserted into pTTBlue in the opposite direction to that of lacZ in this plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was determined. Comparison of this sequence with the corresponding sequence in SGD revealed that the nucleotide A at position 39 (counted taking the first nucleotide A of the initiation codon ATG as position 1) was changed to G as a result of PCR error (A39G; hereinafter, PCR errors are expressed in the same manner).<br>
Further, 5 additional errors of T144C, T223C, T1038C, C1122T and A1370G were also found. Of these PCR errors, T223C and A1370G caused changes in the encoded amino acids. T223C changed Ser at position 75 to Pro (S75P; hereinafter, amino acid sequence errors are expressed in the same manner), and A1370G caused another amino acid sequence error K457R.<br>
The resultant plasmid was designated pT7HMGS.<br>
(5)   Cloning of HMG-CoA Reductase Gene<br>
S. cerevisiae-derived HMG-CoA reductase gene HMGl was cloned as described below.<br>
Briefly, based on information about S. cerevisiae-derived HMG-CoA reductase gene HMGl (A.N. M22002) (M.E. Basson, et at, Mol Cell Biol 8, 3797-3808 (1988): SEQ ID NO: 7) registered at the GenBank, a pair of primers matching the N-terminal and the C-terminal of the protein encoded by this gene were designed. Using these primers and the yeast cDNA library (Clontech) as a template, PCR was carried out.<br>
N-terminal primer: 5'-ATG CCG CCG CTATTC AAG GGA CT-3'<br>
(SEQ ID NO: 56)<br>
C-terminal primer: 5'-TTA GGATTT AAT GCA GGT GAC GG-3'<br><br>
(SEQ ID NO:57)<br>
The PCR was performed using Perfect Match Polymerase Enhancer for 30 cycles each consisting of denaturation for 45 sec at 94°C, annealing for 1 min at 55°C and extension for 2 min at 72°C.<br>
A fragment of interest (3.2 kbp) was confirmed. This fragment(HMGl) was cloned into pT7Blue T vector capable of TA cloning to thereby obtain pT7-HMGl. The nucleotide sequence of the thus cloned HMGl was determined. As a result, the nucleotide sequence as shown in SEQ ID NO: 9 and the amino acid sequence as shown in SEQ ID NO: 10 were obtained. The thus determined nucleotide sequence was partially different from the corresponding nucleotide sequence registered at the GenBank due to PCR errors (Fig. 2A). This mutant type HMG-CoA reductase gene containing PCR errors is designated HMGl (6)   Correction of PCR Errors in HMG-CoA Reductase Gene<br>
PCR errors were corrected by subcloning the HMGl fragment from pT7HMGl and correcting those errors in the HMGl region that would cause amino acid substitution mutations.<br>
Briefly, an HMGl' gene fragment was subcloned from plasmid pT7HMGl comprising HMGl ', a PCR error type DNA of HMG-CoA reductase gene HMGL Then, the PCR errors in the HMGl region that would cause amino acid substitution mutations were corrected by site-directed mutagenesis to thereby prepare pALHMG106. The details of this preparation are as described below.<br>
Plasmid pT7HMGl was used as cloned HMGL    As a vector for introducing<br>
site-directed mutations, pALTER-1 (Promega) was purchased.<br>
Site-directed mutagenesis was carried out according to the procedures described in<br>
"Protocols and Application Guide, 3rd edition,  1996 Promega, ISBN  1-882274-57-1"<br>
published by Promega.   As oligos for introducing mutations, the following three oligos<br>
were synthesized chemically.<br><br>
HMGl (190-216): 5'-CCAAATAAAGACTCCAACACTCTATTT-3'<br>
(SEQ ID NO:58)<br>
HMGl (1807-1833): 5'- GAATTAGAAGCATTATTAAGTAGTGGA-3'<br>
(SEQ ID NO: 59)<br>
HMGl (2713-2739): 5'- GGATTTAACGCACATGCAGCTAATTTA-3'<br>
(SEQ ID NO: 60)<br>
First, pT7HMGl was digested with Smal, ApaLl and Sail, followed by preparation of a 3.2 kbp HMGl fragment by agarose gel electrophoresis. This fragment was inserted into the Smal-Sall site of pALTER-1 to prepare pALHMGl. After denaturation of this plasmid with alkali, the above-described oligos for introducing mutations. Amp repair oligo (Promega) as repair oligo, and Tet knockout oligo (Promega) as knockout oligo were annealed thereto. The resultant plasmid was introduced into E. coli ES1301 (Promega). Then, transformants retaining the plasmid into which site-directed mutations had been introduced was subjected to enrichment culture using 125|ig/ml of ampicillin, followed by preparation of plasmid DNA. The nucleotide sequence of the resultant plasmid DNA was examined with primers having the sequences as shown belov/. The results revealed that all the sequences corresponding to HMGl (190-216), HMGl (1807-1833) and HMGl (2713-2739) were corrected to the intended sequences (SEQ ID NO: 11). The amino acid sequence encoded by the corrected nucleotide sequence (SEQ ID NO: 12) was consistent with the amino acid sequence encoded by HMGl' (SEQ ID NO: 10) (silent mutations).<br>
HMGl (558-532) 5'-GTCTGCTTGGGTTACATTTTCTGAAAA-3'<br>
(SEQ ID NO: 61)<br>
HMGl (1573-1599) 5'-CATACCAGTTATACTGCAGACCAATTG-3'<br>
(SEQ ID NO: 62)<br>
HMGl (2458-2484) 5'-GAATACTCATTAAAGCAAATGGTAGAA-3'<br>
(SEQ ID NO: 63)<br><br>
The plasmid where the sequence within the HMGl region had been corrected was designated pALHMG106 (Fig. 9).<br>
(7)	Cloning of Mevalonate Kinase Gene<br>
An approximately 1.3 kbp fragment encoding S. cerevisiae mevalonate kinase gene ERG12 (SEQ ID NO: 28) was amplified by PCR using cDNA as a template. The PCR primers used are as follows.<br>
Primer 1 (ATM-1): 5'-AAC TGC AGA TGT CAT TAC CGT TCT TAA CTT C-3'<br>
(SEQ ID NO: 64)<br>
Primer 2 (ATM-2): 5'-CCG AGC TCT TAT GAA GTC CAT GGT AAA TTC G-3'<br>
(SEQ ID NO: 65)<br>
(Underlined portions indicate restriction enzyme recognition sites.)<br>
The resultant fragment was digested with Pstl and Sacly purified by agarose gel electrophoresis, and then cloned into the Pstl-Sacl site of pTTBlue. By these procedures, ERG 12 was inserted into pT7Blue in the opposite direction to that of lacZ in this plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was sequenced and compared with the corresponding sequence registered at SGD.    As a result, no PCR error was found.<br>
The plasmid DNA prepared was designated pT7ERG12.<br>
(8)	Cloning of Phosphomevalonate Kinase Gene<br>
An approximately 1.3 kbp fragment encoding S, cerevisiae ERGS (SEQ ID NO: 29) was amplified by PCR using cDNA as a template.    The PCR primers used are as follows.<br>
Primer 1 (YSCE-1): 5'-AAC TGC AGA TGT CAT TAC CGT TCT TAA CTT C-3'<br>
(SEQ ID NO: 66)<br>
Primer 2 (YSCE -2): 5'-CCG AGC TCT TAT GAA GTC CAT GGT AAA TTC G-3'<br><br>
(SEQ ID NO: 67)<br>
The PCR fragment was purified by agarose gel electrophoresis and then cloned into pT7Blue-T by T/A ligation. By these procedures, ERGS was inserted into pT7Blue-T in the opposite direction to that of lacZ in this plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was determined and compared with the corresponding sequence registered at SGD. As a result, the following PCR errors were found: A70C, A72G, G146A, C171Q G224C, A306Q T387C, G574T, C637G, G638C, G729A, G739A, T759A, A879G and A1222G Of these errors, A70C and A72G caused an amino acid error of T24P; G146A caused an amino acid error of G49E; G224C caused an amino acid error of S75T; G574T caused an amino acid error of A192S; C637G and G638C caused an amino acid error of R213A; G739A caused an amino acid error of D247N; and A1222G caused an amino acid error of T408A.<br>
The plasmid DNA prepared was designated pT7ERG8.<br>
(9)   Cloning of Diphosphomevalonate Decarboxylase Gene<br>
An approximately 1.2 kbp fragment encoding S. cerevisiae diphosphomevalonate decarboxylase gene ERG19 (MVDI) (SEQ ID NO: 30) was amplified by PCR using cDNA as a template.    The PCR primers used are as follows.<br>
Primer 1 (SCU-1): 5'-AAC TGC AGA TGA CCG TTT ACA CAG CAT CCG T-3'<br>
(SEQ ID NO: 68)<br>
Primer 2 (SCU-2): 5'-CGGAATTCT TAT TCC TTT GGT AGA CCA GTC T-3'<br>
(SEQ ID NO: 69)<br>
(Restriction enzyme recognition sites are underlined.)<br>
The amplified fragment was digested with Psil and EcoKl, purified by agarose gel electrophoresis, and cloned into the Pstl-EcoRl site of pT7Blue. By these procedures, ERG19(MVDI) was inserted into pT7Blue in the direction opposite to that of lacZ in this<br><br>
plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was determined and compared with the corresponding sequence registered in SGD. As a result, no PCR error was found.<br>
The plasmid DNA prepared was designated pT7ERG19.<br>
(10)    Cloning of Isopentenyl Diphosphate A-Isomerase Gene (10-1)    iS. cerevisiae-Derivcd IPP A-Isomerase Gene IDIl<br>
An approximately 0.9 kbp fragment encoding S. cerevisiae IDIl gene (SEQ ID NO: 31) was amplified by PCR using cDNA as a template. As PCR primers. Primer 1 (SCIPP-1) and Primer 2 (SCIPP-2) were used.<br>
Primer 1 (SCIPP-1): 5'-ATG ACTGCC GAC AAC AATAGT AT-3' (SEQ ID NO: 70)<br>
Primer 2 (SCIPP-2): 5'-TTATAG CAT TCT ATG AAT TTG CC-3' (SEQ ID NO: 71)<br>
The PCR fragment was purified by agarose gel electrophoresis and then cloned into pT7Blue-T by T/A ligation. By these procedures, IDIl was inserted into pT7Blue-T in the opposite direction to that of lacZ in this plasmid (Fig. 8). The nucleotide sequence of the cloned fragment was determined and compared with the corresponding sequence registered at SGD.    As a result, no PCR error was found.<br>
The plasmid DNA prepared was designated pT7IDIl.<br>
(10-2)   E. coli-Derived IPP A-Isomerase Gene idi<br>
Using, as a template, plasmid p3-47-13 (Hemmi et al, (1998) J. Biochem. 123, 1088-1096) in which a genomic DNA comprising E. coli ORF182 (an open reading frame expected to encode a polypeptide homologous to IPP A-isomerase; gene name: idi) is cloned, an ORF182 fragment of approx. 0.55 kbp was amplified by PCR. The PCR was performed in the co-presence of Perfect Match Polymerase Enhancer for 30 cycles each consisting of<br><br>
45 sec at 95°C, 1 min at 60°C and 2 min at 72°C.    The PCR primers used are as follows.<br>
Primer 1 (SacII-ORF182(1.23)): 5'-TCC CCG CGG ATG CAA ACG GAA CAC<br>
GTC ATT TT-3' (SEQ ID NO: 72)<br>
Primer 2 (Xbal-ORF 182(549-525)): 5'-TGC TCT AGA TTA TTT AAG CTG GGT<br>
AAATGC AGA-3' (SEQ ID NO: 73)<br>
(Underlined portion indicates a restriction enzyme recognition site.)<br>
The PCR product was digested with Spel, Dralll and AluI, and then cut by agarose gel electrophoresis. As a result, the physical map as shown in Fig. 10 was obtained which was consistent with the nucleotide sequence data (SEQ ID NO: 32) of the ORF182 fragment (idi) of EcoGene (http://bmb.med.miami.edu/EcoGene/EcoWeb/). Then, the amplified 0.55 kbp fragment was digested with Sacll and Xbal, purified by agarose gel electrophoresis and cloned into the SaclhXbal site of pRS435GAP and pRS445GAP.' The resultant plasmids were designated pRS435GAP-ORF182 and pRS445GAP-ORF182, respectively.<br>
The E. coli IPPA-isomerase gene (SEQ ID NO: 32) was previously called ORF182 (according to NCBI BLAST search; GenBank Accession No. AE000372), but Hahn et aL, (1999) J. BacterioL, 181: 4499-4504 designated this gene idi. As plasmids in which idi is cloned, p3-47-ll and p3-47-13 described in Hemmi et al., (1998) J. Biochem,, 123: 1088-1096 were used in the invention.<br>
[EXAMPLE 3]    CLONING OF MUTANT GENES<br>
(1)    Conversion of Escherichia coli FPP Synthase Gene into GGPP Synthase Gene (Cloning of Mutants of FPP Synthase Gene) The codon encoding the amino acid residue Tyr at position 79 of the polypeptide encoded by E. coli ispA was modified by substitution mutation using pALispA4, pALispAS, pALispA15, pALispA16 and pALispA18 obtained in section (1-2) in Example 2 and according to the protocol described in the 'Protocols and Applications Guide, 3rd edition.<br><br>
1996 Promega, ISBN 1-882274-57-1" published by Promega. The following oligonucleotides for introducing mutations (sometimes referred to as "mutation oligo(s)") were prepared by chemical synthesis.<br>
ISPA-D: 5'-ATC ATG AAT TAA TGA GTC AGC GTG GAT GCA TTC AAC GGC<br>
GGC AGC-3' (SEQ ID NO: 74)<br>
ISPA-E: 5'-ATC ATG AAT TAA TGA TTC AGC GTG GAT GCA TTC AAC GGC<br>
GGC AGC-3' (SEQ ID NO: 75)<br>
ISPA-M: 5'-ATC ATG AAT TAA TGA CAT AGC GTG GAT GCA TTC AAC GGC<br>
GGC AGC-3' (SEQ ID NO: 76)<br>
In the above mutation oligo ISPA-M, nucleotides at positions 16-18 (the underlined 3 nucleotides) correspond to the codon encoding the amino acid residue Tyr at position 79 of the wild-type FPP synthase; these three nucleotides are.designed so that this codon encodes Met. Similarly, mutation oligos ISPA-D and IDPA-E are designed so that this codon encodes Asp and Glu, respectively. The nucleotides at positions 26-31 (the underlined 6 nucleotides) in the above mutation oligos are designed so that an EcoT221 (Nsil) site is newly formed as a result of the substitution mutation. With this site, mutant genes can be easily discriminated by restriction enzyme mapping. These mutation oligos are phosphorylated at 5' end with T4 polynucleotide kinase (Promega) and purified by gel filtration with Nick Column (Pharmacia Biotech, Uppsala, Sweden) before use. In the introduction of mutations. Cm repair oligo (Promega) was used as repair oligo and Tet knockout oligo (Promega) as knockout oligo. Cm repair oligo, Tet knockout oligo and the mutation oligo were annealed to alkali-denatured pALispA16, which was then transformed into E. coli ESI301 mutS (Promega). Plasmid DNA was prepared from E. coli colonies growing in the presence of 20 |ig/ml of Cm (chloramphenicol) and transformed into E. coli JM109. Then, plasmid DNA was prepared from colonies growing on agar plates containing 20 |ig/ml of Cm. Those plasmids comprising the substitution mutant type ispA (called 'ispAM") created by using pALispA4 as a template and ISPA-D,<br><br>
ISPA-E or ISPA-M as a mutation oligo are designated p4D, p4E and p4M, respectively. Likewise, those plasmids prepared by using pALisp8 as a template were designated p8D, p8E and p8M, respectively; those plasmids prepared by using pALispl5 as a template were designated pl5D, pl5E and pl5M, respectively; those plasmids prepared by using pALispl6 as a template were designated PI6D5 pl6E and PI6M5 respectively; and those plasmids prepared by using pALispl8 as a template were designated pi 80, pl8E and pl8M, respectively.<br>
The gene encoding the Y79D mutant type amino acid sequence (SEQ ID NO: 34) is shown in SEQ ID NO: 33; the gene encoding the Y79E mutant type amino acid sequence (SEQ ID NO: 36) is shown in SEQ ID NO: 35; and the gene encoding the Y79M mutant type amino acid sequence (SEQ ID NO: 38) is shown in SEQ ID NO: 37. The thus obtained plasmids were appropriately selected and used.<br>
(2)    Cloning of a Mutant of Bacillus stearothermophilus FPP Synthase Gene<br>
Expression vectors comprising a substitution mutant of B. stearothermophilus FPP synthase gene(fps\ SEQ ID NO: 39) was prepared from pFPS(Y81M) disclosed in Ohnuma et al, (1996) J. Biol Chem., 271, 30748-30754.<br>
pFPS is a plasmid integrating fps downstream of lac promoter in pTV118N (Takara), and this plasmid expresses B, stearothermophilus FPP synthase gene in E. coli in the presence of IPTG First, Y81M mutation (i,e., substitution mutation that changes Tyr at position 81 of the amino acid sequence encoded by the FPP synthase gene to Met) was introduced into the FPP synthase gene by site-directed mutagenesis [to thereby obtain pFPS(Y81M)]. As a result of the introduction of Y81M mutation, the reaction product specificity of the enzyme encoded in pEPS(Y81M) was changed; here, the FPP synthase gene was modified to a GGPP synthase gene without decrease of the specific activity of the encoded enzyme. Subsequently, pFPS(Y81M) was digested with PshBl and blunt-ended with Klenow enzyme. Then, a 2.7 Kbp fragment containing the transcription unit was purified and inserted into the Hindi site of Amp' gene in pACYC177. The resultant plasmid in which the mutant fps gene fragment was inserted in the same direction as that of<br><br>
Amp' gene was designated pFPS21m, and the plasmid in which the mutant fps gene fragment was inserted in the opposite direction to that of Amp' gene was designated pFPS31m(Fig. 11).<br>
(3)    Cloning of Deletion Mutants of HMG-CoA Reductase Gene<br>
Vectors pRS414PTadh and pRS414TPadh, each comprising a constitutive promoter ADHlp, were digested with restriction enzymes, followed by insertion of HMGl to thereby prepare plasmids pRS414PTadh-HMGl and pRS414TPadh-HMGl.<br>
pT7-HMGl prepared in (5) in Example 2 was digested with BamHI, Sail and Seal to obtain HMGl' gene having PCR errors. This gene was transferred into the BamHl-Xhol site of pYES2 (Invitrogen, Carlsbad, CA) to thereby obtain a recombinant vector pYES-HMGl. The nucleotide sequence within the vector was confirmed to be the nucleotide sequence of SEQ ID NO: 3. pYES is a shuttle vector for expression in yeast having the ori of yeast 2μmDNA as a replication origin and GALl promoter inducible by galactose (Fig. 4).<br>
In order to prepare expression vectors for deletion mutants of HMG-CoA reductase gene having deletion of regions corresponding to transmembrane domains of HMG-CoA reductase, PCR was carried out using pYES-HMGl prepared above as a template to thereby generate DNA fragments (including the vector moiety) in which a part of the HMGl coding region is deleted. The resultant fragments were blunt-ended with Klenow enzyme, circularized by self-ligation, and transformed into E. coli JM109. Then, plasmid DNA was prepared. Synthetic DNA sequences used as primers and their combinations are shown in Table 1 above.<br>
For each of the plasmid DNA obtained, it was confirmed with 3 73A DNA sequencer (Perkin Elmer, Foster City, CA) that there was no shift in the reading frame of amino acids between the upstream and downstream of the deleted region in HMGl, and that there was no amino acid substitution resulting from PCR errors around the junction site. As a result, the following plasmids were obtained which had no amino acid substitution resulting from PCR errors around the junction site and in which a part of the gene could be deleted without<br><br>
any shift in the reading frame.    Deletion mutants of HMGl gene are expressed as, e.g.,<br>
"A02y" according to the deletion pattern (where y represents any working number), and a<br>
pYES2 vector comprising A02y is expressed as, e.g., pYHMG026.    (This expression<br>
method is also applied to other deletion mutants.)<br>
HMGiA026:SEQIDNO:13<br>
HMG7A044: SEQ ID NO: 14<br>
HMG/A056:SEQIDNO:15<br>
HMGIA062: SEQ ID NO: 16<br>
HMGlA076: SEQ ID NO: 17<br>
HMG7A081:SEQIDNO:18<br>
HMG7A100:SEQIDNO:19<br>
HMG7A112: SEQ ID NO: 20<br>
HMG7A122:SEQIDNO:21<br>
HMG7A133:SEQIDNG:22<br>
Plasmids:<br>
pYHMG026,   pYHMG027,   pYHMG044,    pYHMG045,   pYHMG059,   pYHMG062,<br>
pYHMG063,    pYHMG065,   pYHMG076,    pYHMGOSl,   pYHMG083,   pYHMG085,<br>
pYHMG094,   pYHMGlOO,   pYHMG106,    pYHMG107,   pYHMG108,   pYHMG109,<br>
pYHMGin,pYHMG122, pYHMG123, pYHMG125,pYHMG133 andpYHMG134.<br>
[EXAMPLE 4]   SUBCLONING OF GENES INTO VECTORS<br>
As E. coli-S. cerevisiae YEp shuttle vectors having a constitutive transcription promoter, pRS vectors prepared in Example 1 were used.<br>
(1)   Subcloning of FPP Synthase Gene<br>
(1 -1)   S. cerevisiae-Derivd FPP Synthase Gene ERG20:<br>
pT7ERG20 described in section (1-1) in Example 2 was digested with Xbal and BatnRl, and subjected to agarose gel electrophoresis to thereby purify an ERG20 gene<br><br>
fragment of 1.1 kbp.    This fragment was inserted into the Xbal-BamHl site of pRS435GAP<br>
and pRS445GAP to obtain pRS435GAP-ERG20 and pRS445GAP-ERG20, respectively.<br>
(1-2)    E. coli-Derived FPP Synthase Gene ispA:<br>
pALispA4 described in section (1-2) in Example 2 was digested with Sphl and<br>
EcoRly and subjected to agarose gel electrophoresis to thereby purify an ispA gene fragment<br>
of 1.0 kbp.    To this fragment, Sphl-Sacll linker DNA (5'-pTTT CCG CGG AAA CAT G-3';<br>
SEQ ID NO: 86) and EcoRl-Eco52l linker DNA (5'-pAAT TGA CGG CCG TC -3'; SEQ ID<br>
NO: 87) were ligated.    Then, the fragment was digested with Sacll and EcoSll.    The<br>
resultant iS'acII-Eco52I fragment of 1.0 kbp was inserted into the iS'acII-E'co52I site of<br>
pRS435GAP and pRS445GAP for subcloning.    For each of the subcloned plasmids,<br>
recognition sites of Sad, Sacll, Ndel, Nsil (EcoT22l) AorSlHl Xbal, Smal, BamUl, Pstl,<br>
Ndel, Pvull and EcoT14I were mapped, followed by selection of plasmids that were<br>
constructed as designed.    The selected plasmids were designated pRS435GAP-ispA and<br>
pRS445GAP-ispA, respectively.<br>
(1-3)   B. stearothermophiIus-Dcn\td FPP Synthase Gene<br>
B. stearothermophilus-dcnwQd FPP synthase gene was cloned into a vector directly from a genomic PCR fragment.<br>
(2)    Subcloning of GGPP Synthase Gene or Mutants Thereof (2-1)   S. cerevisiae-Derived GGPP Synthase Gene BTSJ:<br>
The pT7Blue-T vector described in section (2) in Example 2 was digested with BamHl and Sail to obtain a fragment encoding BTSl, which was then introduced into the BamHl-Xhol site of pYES2 (Invitrogen). The resultant recombinant vector was designated pYESGGPS.<br>
pYESGGPS was digested with BamHl and Mwl, and subjected to agarose gel electrophoresis to purify a 1.3 kbp fragment.    This fragment was inserted into the BamUhMlul site of pRS435GAP and pRS445GAP to obtain pRS435GAP-BTSl  and pRS445GAP-BTSl, respectively. (2-2)   E. coli-Derived GGPP Synthase Gene (Substitution Mutant Type FPP Synthase<br><br>
Gene) ispAm:<br>
pl6M described in section (1) in Example 3 was digested with SphI and EcoRl, and<br>
subjected to agarose gel electrophoresis to purify an 1.0 kbp fragment encoding ispAm gene.<br>
To this fragment, Sphl-SacII linker DNA and EcoRI-Eco52I linker DNA described in<br>
section (1-2) in this Example were ligated, followed by digestion with Sacll and Eco52l.<br>
The resultant SacII-'Eco52l fragment (1.0 kbp) was inserted into the iS'acII-E'co52I site of<br>
pRS435GAP and pRS445GAP for subcloning.    For each of the subcloned plasmids,<br>
recognition sites of iSacI, Sacll, Ndel, Nsil (EcoT22l) AorSlUl, Xbal, Smal, BamRl, Pstl,<br>
Pvull and EcoT14I were mapped, followed by selection of plasmids that were constructed<br>
as designed.    Of these recognition sites, Nsil (EcoT22l) recognition site is a site that was<br>
newly introduced when a substitution mutation was introduced into ispA.    If the plasmid<br>
can be cut with this restriction enzyme, it is confirmed that the gene in the plasmid is the<br>
ispA mutant gene ispAm.    The selected plasmids were designated pRS435GAP-ispAm and<br>
pRS445GAP-ispAm, respectively.<br>
(3)	Subcloning of Acetyl-CoA Acetyltransferase Gene<br>
Acetyl-CoA acetyltransferase gene ERGIO was cloned into pRS vector directly fi-om a genomic PCR fragment.<br>
(4)	Subcloning of HMG-CoA Synthase Gene<br>
A 1.5 kbp BamHl-Sall fragment encoding HMGS gene was prepared from pT7HMGS described in section (4) in Example 2, and inserted into the BomUl-Sall site of pRS435GAP and pRS445GAP. The HMG5-subcloned plasmids were examined by Kpnl restriction site mapping, followed by selection of plasmids that were constructed as designed. The selected plasmids were designated pRS435GAP-HMGS and pRS445GAP-HMGS, respectively.<br>
(5)	Subcloning of HMG-CoA Reductase Gene or Mutants Thereof<br>
The pT7Blue-T vector described in section (5) in Example 2 was digested with<br><br>
BamUl, Sail and Seal to thereby cut out HMGl' gene encoding a PCR error-type mutant HMG-CoA reductase. This gene was inserted into the BamUl-Xhol site of pYES2 (Invitrogen).    The resultant plasmid was designated pYES-HMGl.<br>
Vectors pRS414PTadh and pRS414TPadh, each comprising a constitutive promoter ADHIp, were digested with Smal and Sail. Then, HMGl gene was inserted thereinto to prepare pRS414PTadh-HMGl andpRS414TPadh-HMGl.<br>
Further, pALHMG106 (Fig. 9) described in section (6) in Example 2 was digested with Smal and Sail, and subjected, to agarose gel electrophoresis to purify a 3.2 kbp fragment encoding the PCR error-corrected HMGl gene. This fragment was inserted into the Smal-Sall site of pRS434GAP, pRS444GAP, pRS434TEF, pRS444TEF, pRS434PGK and pRS444PGK. Physical maps of the HMGi-subcloned plasmids were examined by restriction enzyme mapping using Xhol, Spel, Nael and Sphl, and by confirming the nucleotide sequences of the border regions of the inserted 3.2 kbp HMGl gene. Then, those plasmids constructed exactly as designed were selected and designated pRS434GAP-HMGl, pRS444GAP-HMGl, pRS434TEF-HMGl, pRS444TEF-HMGl, pRS434PGK-HMGl and pRS444PGK-HMGl, respectively.<br>
Deletion mutants of HMG-CoA reductase gene were obtained from pYES2-derived plasmids incorporating corresponding deletion mutants of HMGl described in Example 3 and cloned into pRS434GAP in a manner similar to that described in the preceding paragraph.<br>
(6)    Subcloning of Mevalonate Kinase Gene<br>
A Smal-Sall 1.3 kbp fragment encoding ERG12 gene was prepared from pT7ERG12 described in section (7) in Example 2, and inserted into the Smal-Sall site of pRS435GAP and pRS445GAP. The Ei?G72-subcloned plasmids were examined by Kpnl recognition site mapping, followed by selection of those plasmids constructed exactly as designed. The selected plasmids were designated pRS435GAP-ERG 12 and pRS445GAP-ERG12, respectively.<br><br>
(7)	Subcloning of Phosphomevalonate Kinase Gene<br>
A Baml'Sall 1.3 kbp fragment encoding ERGS gene was prepared from pT7ERG8 described in section (8) in Example 2, and inserted into the Smal-Sall site of pRS435GAP and pRS445GAP. The ERGS-subcloned plasmids were examined by Xbal recognition site mapping, followed by selection of those plasmids constructed exactly as designed. The selected plasmids were designated pRS435GAP-ERG8 and pRS445GAP"ERG8, respectively.<br>
(8)	Subcloning of Diphosphomevalonate Decarboxylase Gene<br>
pT7ERG19 described in section (9) in Example 2 was digested with BamUl and Sail, and subjected to agarose gel electrophoresis to purify a BamHl-Sall 1.5 kbp fragment encoding ERG 19 gene. This fragment was inserted into the BamUl-Sall site of pRS435GAP and pRS445GAP. The Ei?G79-subcloned plasmids were examined by Xbal recognition site mapping, followed by selection of those plasmids constructed exactly as designed. The selected plasmids were designated pRS43 5 GAP-ERG 19 and pRS445GAP-ERGl 9, respectively.<br>
(9)	Subcloning of Isopentenyl Diphosphate A-Isomerase Gene<br>
(9-1)    S. cerevisiae-Dexivtd IPP A-Isomerase Gene IDIl:<br>
A BamUl-Sall 0.9 kbp fragment was prepared from pT7IDIl described in section<br>
(10-1) in Example 2 and inserted into the BammSall site of pRS435GAP and pRS445GAR<br>
The subcloned plasmids were examined by recognition site mapping using Ncol and BamUl,<br>
followed by selection of those plasmids constructed exactly as designed.   The selected<br>
plasmids were designated pRS435GAP-IDIl and pRS445GAP-IDIl, respectively.<br>
(9«2)   E. coli-Derived IPP A-Isomerase Gene ORF182 (idi):<br>
ORF182 (idi) was cloned into pRS vector directly from a genomic PCR fragment<br>
as described in section (10-2) in Example 2.<br>
[EXAMPLE 5]   PREPARATION OF AURGG101, AURGG102 AND AURGG703<br><br>
A 1.9 kbp SaII fragment having a primary structure of GAL 1 promotor=BTSJ=CYCl terminator (GALlp-BTSl-CYClt) was prepared by PCR using pYESGGPS described in section (2-1) in Example 4 as a template and the following primers PYES2 (1-27) and PYES2 (861-835).<br>
PYES2 (1-27): 5'-GGC CGC AAA TTA AAG CCT TCG AGC GTC-3'<br>
(SEQ ID NO: 88) PYES2 (861-835): 5'-ACG GAT TAG AAG CCG CCG AGC GGG TGA-3'<br>
(SEQ ID NO: 89)<br>
This fragment was inserted into the Sail site of pAUR101 (Takara) to obtain pAURGGllS. It was confirmed by DNA sequencing that the BTSl gene in pAURGG115 had no PCR error.<br>
pAURGGl 15 was linearized with Eco065l and introduced into A451 strain and YPH499 strain by the lithium acetate method. Then, colonies growing on YPD agar plates (1% yeast extract, 2% peptone, 2% dextrose, 2% agar) containing lμg/ml aureobasidin at 30°C were selected as transformants.<br>
The resultant transformants were cultured again on aureobasidin selection plates for single colony selection.<br>
As a result, two strains AURGG101 and AURGG102 were obtained as A451-derived recombinants. Also, AURGG703 was obtained as a YPH499-derived recombinant. Southern blot hybridization (Fig. 12) and PCR mapping (Fig. 13) described later revealed that BTSl gene is not integrated in AURGG101 and that AURl has been replaced With AURl-C, a marker gene, in this strain. On the other hand, it was found that GALl promoter=BTSl=CYCl terminator is integrated in the AURl locus of AURGG102.<br>
[EXAMPLE 6]    CREATION OF EUG STRAINS<br>
A gene map around squalene synthase gene ERG9 was obtained from SGD.   Based on this map, PCR primer DNAs for amplifying DNA fragments for replacing ERG9<br><br>
transcription promoter(ERG9p) were designed. On the other hand, a 1.8 kbp DNA fragment comprising a transformant selection marker gene URA3 and a transcription promoter GALlp was prepared by PCR amplification using, as a template, pYES2A obtained by digesting pYES2 with Nael and Nhel, blunt-ending with Klenow enzyme and deleting lixori by self-ligation.<br>
The primers used in the PCR are as follows.<br>
E-MCSf 5'- GCC GTT GAC AGAGGGTCC GAG CTC GGT ACC AAG -3'<br>
(SEQ ID NO:90)<br>
E-URA3r: 5'- CAT ACT GAC CCA TTGTCAATG GGT AAT AAC TGA T -3'<br>
(SEQ ID NO:91)<br>
In each of the above primers, an Eam11051 recognition site (the underlined portion) was added so that a 0.7 kbp DNA fragment comprising a downstream portion of YHR189W and a 0.9 kbp DNA fragment comprising an upstream portion of ERG9 can be ligated to the 1.8 kbp fragment by T/A ligation. The YHR189W fragment was prepared by PCR using the following primers YHR189Wf and YHR189Wr, and YPH499 genomic DNA as a template. The ERG9 fragment was prepared by PCR using the following primers ERG9f and ERG9r, and YPH499 genomic DNA as a template. YPH499 genomic DNA was prepared with a yeast genomic DNA preparation kit "Dr. GenTLE    " (Takara).<br>
YHR189Wf: 5'-TGT CCG GTA AAT GGA GAC-3' (SEQ ID NO: 92) YHRl 89Wr: 5'-TGT TCT CGC TGC TCG TTT-3' (SEQ ID NO: 93) ERG9f: 5'-ATG GGA AAG CTATTA CAAT-3' (SEQ ID NO: 94) ERG9r: 5'-CAA GGT TGC AAT GGC CAT-3' (SEQ ID NO: 95)<br>
Briefly, the 1.8 kbp DNA fragment was digested with Eam11051 and then ligated to the 0.7 kbp DNA fragment. With the resultant fragment as a template, 2nd PCR was carried out using the above-described primers YHR189Wf and E-MCSf   The amplified<br><br>
2.5 kbp DNA fragment was digested with Eam11051 and then ligated to the 0.9 kbp fragment. With the resultant fragment as a template, 3rd PCR was carried out using the following primers YHR189W-3f and ERG9-2r. As a result, a 3.4 kbp DNA fragment was amplified.   This was used as a DNA fragment for transformation.<br>
YHR189W-3f: 5'-CAATGT AGG GCTATATAT G-3' (SEQ ID NO: 96) ERG9-2r: 5'-AAC TTG GGG AAT GGC ACA-3' (SEQ ID NO: 97)<br>
The vector was introduced into yeast strains using Frozen EZ Yeast Transformation II kit purchased from Zymo Research (Orange, CA). The resultant recombinants were cultured on an agar medium (called SGR(-URA) medium) that had been obtained by adding CSM(-URA) (purchased from BIO 101, Vista, CA) and adenine sulfate (final concentration 40 mg/L) to SGR medium, at 30°C. Colonies grown on the medium were spread on the same medium again, and single colony isolation was conducted.<br>
The resultant recombinants were designated EUG (ERG9p::URA3-GAL]p) clones. Of these clones, those derived from A451 were designated EUGl through EUGIO; those derived from YPH499 were designated HUGH through EUG20; those derived from YPH500 were designated EUG21 through EUG30; those derived from W303-1A were designated EUG31 through EUG50; and those derived from W303-1B were designated EUG51 through EUG70.<br>
Those clones that exhibit a decrease in growth rate as a result of repression of ERG9 expression by the glucose repression in SD medium were selected. As a result, EUG5 and EUG8 were obtained from A451; EUG12 was obtained from YPH499; and EUG27 was obtained from YPH500.<br>
Genomic DNA was prepared from EUG5, EUG8, EUG12 and EUG27 using Dr. GenTLE , and PCR was carried out using the genomic DNA as a template. The results confirmed that the 1.8 kbp PCR fragment comprising URA3 and GALlp is integrated upstream of the ERG9 coding region in each genome.<br><br>
[EXAMPLE 7]   ANALYSIS OF GENES AND ENZYME ACTIVITIES<br>
In this Example, the expression of genes in various recombinant yeast clones prepared (for the preparation thereof, see Examples 8-13 describing prenyl alcohol production) was analyzed by various techniques including determination of the enzyme activities of prenyl diphosphate synthases. Northern blot hybridization. Southern blot hybridization, PCR mapping and determination of prenyl alcohol yields.<br>
(1)	Southern Blotting<br>
Yeast DNA was prepared using the yeast DNA purification kit Dr. GenTLETM according to the protocol attached to the kit.<br>
The DNA thus prepared from yeast was digested with Ndel and Stul, followed by 0.8% agarose gel electrophoresis using 3 μg of the DNA per lane. As molecular weight markers, 0.5 μg each of 1 kb ladder and X/HindlU (both from Promega, Madison, WI) were used. After the electrophoresis, the DNA was denatured with alkali, neutralized and transferred onto Hybond N nylon membrane (Amersham, Buckinghamshire, England) by capillary blotting with 20x SSC according to conventional methods. The resultant membrane was subjected to UV irradiation with a UV cross-linker (Stratagene) under conditions of optimal cross-linking, to thereby fix the DNA on the membrane.<br>
(2)	Northem Blotting<br>
RNA was prepared according to the method described in Current Protocols in Molecular Biology, John Wiley &amp; Sons, pp. 13.12.2-13.12.3 with a modification. The modification was that once prepared RNA samples were further treated with DNase I.<br>
After separation of RNA by formaldehyde-denatured agarose gel electrophoresis, the RNA was transferred onto Hybond N nylon membrane by capillary blotting with 20 x SSC according to conventional methods. Five micrograms of total RNA was electrophoresed per lane. As a molecular marker, 20 ng of DIG-RNA Marker I was used. The resultant membrane was subjected to UV irradiation with a UV cross-linker (Stratagene) under<br><br>
conditions of optimal cross-linking, to thereby fix the RNA on the membrane.<br>
(3)	PCR Mapping<br>
In order to examine how a fragment from pAURGG115 (a YIp vector prepared in Example 5) is integrated into the genome, PCR was carried out using 0.3-0.6 )ig of the yeast DNA prepared above as a template and a combination of synthetic oligonucleotide primers AUR-FWc and AUR-RVc, or AUR-SALl and AUR-SAL2. PCR conditions were as follows: 30 cycles of denaturation at 94°C for 30 sec, annealing at 55°C for 1 min and extension at 72°C for 3 min.<br>
AUR-FWc: 5'-TCT CGA AAA AGO GTT TGC CAT-3' (SEQ ID NO: 98) AUR-RVc: 5'-TCA CTA GGT GTA AAG AGG GCT-3' (SEQ ID NO: 99) AUR-SALl: 5'-TGTTGA AGC TTG CAT GCC TGC-3' (SEQ ID NO: 100) AUR-SAL2: 5'-TTG TAA AAC GAC GGC CAG TGA-3' (SEQ ID NO: 101)<br>
(4)	Preparation of DIG-Labeled Probe DNAs<br>
As hybridization probes, Probes I, II, III and V were prepared (Table 7).<br><br>
Probe I:<br>
Using pT7ERG20 prepared in section (1-1) in Example 2 as a template and SCEPSl and SCEPS2 as primers, a DIG-labeled probe DNA was synthesized with PCR DIG Probe Synthesis Kit (Roche Diagnostics, Marmheim Germany).   Experimental conditions were in<br><br>
accordance with the manufacturer's protocol attached to the kit. PCR was performed for 30 cycles each consisting of 30 sec at 94°C, 1 min 58°C and 3 min at 72°C. The resultant DIG-labeled probe DNA was subjected to agarose gel electrophoresis to examine the state of synthesis.<br>
Probe II:<br>
A DIG-labeled probe DNA was synthesized in the same manner as described for Probe I, using synthetic oligonucleotides BSTS1(1-21) and BTSl(1008-988) as primers and pYESGGPS (see section(2-l) in Example 4) as a template.<br>
BTSl (1-21): 5'-ATG GAG GCC AAG ATA GAT GAG-S' (SEQ ID NO: 102) BTSl (1008-988): 5'-TCACAATTCGGATAAGTGGTC-3'(SEQ ID NO: 103)<br>
Probe III:<br>
A DIG-labeled probe DNA was synthesized in the same manner as described for Probe I, using synthetic oligonucleotides HMGl (1267-1293) and HMGl (2766-2740) as primers and pYES-HMGl (see section (3) in Example 3) as a template.<br>
HMGl (1267-1293): 5'-AAC TTT GGT GCA AAT TGG GTC AAT GAT-3' (SEQ ID NO: 80)<br>
HMGl (2766-2740): 5'-TCC TAATGC CAA GAAAAC AGC TGT CAC-3' (SEQ ID NO: 104)<br>
Probe V:<br>
A DIG-labeled probe DNA was synthesized in the same manner as described for Probe I, using synthetic oligonucleotides AUR-FW and AUR-RV as primers and pAUR123 (Takara) as a template.<br>
AUR-FW: 5'-ATG GCAAAC CCTTTTTCG AGA-3' (SEQ ID NO: 105)<br><br>
AUR-RV: 5'-AGCCCTCTTTACACCTAGTGA-3'(SEQIDNO: 106)<br>
(5)	Hybridization and Detection of Probes<br>
Southem blot hybridization was carried out at a probe concentration of 20 ng/ml at 42°C for 24 hr using DIG Easy Hyb (Roche). Northern blot hybridization was carried out at a probe concentration of 100 ng/ml at 50°C for 24 hr using DIG Easy Hyb. Prior to each hybridization, prehybridization was carried out for 24 hr in DIG Easy Hyb solution at the same temperature used for each hybridization. After the hybridization, the membrane was washed 3 times with 2x SSC, 0.1% SDS at 65°C for 10 min each, and then 2 times with 0.2x SSC, 0.1% SDS at 65°C for 15-20 min each. Thereafter, the DIG-labeled probe in the membrane was allowed to generate chemiluminescence by using DIG Luminescent Detection Kit (Roche), followed by exposure of the blot to X-ray film for visualization.<br>
(6)	Determination of Enzyme Activities<br>
Of the recombinants prepared, the host strain and the recombinants listed below were used in this experiment. The introduction of individual vectors into the host was carried out according to the lithium acetate method described in Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc., pp. 13.7.1-13.7.2 or by a method using Frozen EZ Yeast Transformation II kit (Zymo Research, Orange, CA) (procedures were in accordance with the protocol attached to the kit). In the list below, clone 1-2 was obtained by introducing pYES-HMGl into A451; clone 3-2 was obtained by introducing pYHMG044 into A451; clone 13-2 was obtained by introducing pYES-HMGl into AURGGlOl; and clone 15-2 was obtained by introducing pYHMG044 into AURGGIOI.<br>
No.l host strain: A451<br>
No.2 GALlp-BTSl (Yip): AURGG101 (A451, aur]::AURl-Q<br>
No.3 GALlp-BTSl (Yip): AURGG102 (A451, aurl\:BTSl-AURl-C)<br>
No.4 GALlp-HMGl (YEp): 1-2 (pYES-HMGl/A451)<br>
No.5 GALlp-HMGlA (YEp): 3-2 (pYHMG044/A451)<br><br>
No.6 GALlp-HMGI (YEp) &amp;   GALlp-BTS] (Yip) : 13-2 (pYES-HMGl/AURGGlOl) No.7 GAL]p-HMG]A (YEp) &amp;    GALlp-BTSl (Yip) : 15-2 (pYHMG044/AURGG101) No.8 GALlp-HMGI (YEp) &amp; GALlp-BTSl (Yip): 24-1 (pYES-HMGl/AURGG102) No.9 GALlp-HMGI A (YEp) &amp; GALlp-BTSl (Yip): 27-2 (pYHMG045/AURGG102) No.lO GALlp-HMG7A (YEp) &amp; GALlp-BTSl (Yip): 31-2 (pYHMG076/AURGG102)<br>
Strains/clones No. 1 to No. 10 were precultured separately at 26°C. One milliliter of the preculture was washed with physiological saline, added to 100 ml of a culture broth and cultured in a 300 ml Erlenmeyer flask at 26°C with reciprocal shaking at 120 times/min. SD medium or SG medium (in which the glucose component of SD medium is replaced with galactose) was used for the cultivation. Recombinants retaining URA2 marker were cultured in SD-U [CSM(-URA)-added SD medium] or SG-U [CSM(-URA)-added SG medium].   AURGG strains were cultured in the presence of aureobasidin at 1 |ig/ml.<br>
OD600 of cells was determined to monitor cell growth. Cultivation was stopped when OD600 value reached about 3-4 (23-52 hours after the start of cultivation). The culture was cooled in ice and then subjected to the preparation of DNA, RNA and crude enzyme solution, as described below<br>
Cells were harvested from each culture broth by centrifugation and disrupted at 4°C with glass beads in the same maimer as in the preparation of RNA. Then, cells were suspended in sterilized water. The suspension was centrifuged at 12,000 rpm for 10 min with a refrigerated microcentrifuge, and the resultant supernatant was recovered as a crude enzyme fraction. The protein concentration in the crude enzyme fraction was determined by Bio-Rad Protein Assay (Bio-Rad, Hercules, CA) using BSA as a standard protein. Briefly, 10 μg of the crude enzyme fraction was reacted in 200 \i\ of the following reaction cocktail at 37°C for 40 min.<br><br><br>
After the reaction, extended prenyl diphosphate was extracted with water-saturated butanol. An aliquot of the prenyl diphosphate was subjected to determination of radioactivity with a liquid scintillation counter. The remaining sample was dephosphorylated with potato acid phosphatase and developed by thin layer chromatography [plate: LKC18 (Whatman, Clifton, NJ); developer: H20/acetone = 1:19], followed by visualization of the autoradiogram with Bio Image Analyzer BAS2000 (Fuji Film) to determine relative radioactivity, according to the method of Koyama et al. (KoyamaT., Fujii, H. and Ogura, K., 1985, Meth Enzymol 110:153-155).<br>
(7)   Results and Observations<br>
(7-1)    Southern Blot Hybridization and PCR Mapping<br>
The results of southern blot hybridization are shown in Fig. 12. The results of PCR mapping in the vicinity of AURI are shovra in Fig. 13. In Figs. 12 and 13, lanes 1 to 10 correspond to the numbers of strains/clones (No. 1 to No. 7) used in section (6) above.<br>
"N" appearing below each lane number represents DNA digested v^th Ndel; and "S" represents DNA digested v^th StuL DNAs used in individual lanes were prepared from the following strains/clones.<br>
Lane 1: A451; Lane 2: AURGGlOl; Lane 3: AURGG102; Lane 4: pYES-HMGl/A451; Lane 5: pYHMG044/A451; Lane 6: pYES-HMGl/AURGGlOl; Lane 7: p YHMG044/AURGG101; Lane 8: p YES-HMG1 /AURGG102; Lane 9: pYHMG045/AURGG102; and LanelO: pYHMG076/AURGG102<br>
It was found that ERG20 (FPP synthase gene) is identical in all of the strains/clones tested and that there is no change in the vicinity of ERG20 in the genome of each strain/clone (Fig. 12).<br>
When BTSl (GGPP synthase gene) and AURJ were used as probes, it was found that<br><br>
BTSl is integrated into the region of AURl in AURGG102, but the bands appearing in AURGG101 are the same as those appearing in the host strain A451. In AURGG101, only AURl gene is replaced with pAURl01-derived AURl-C gene; it was found that the GALl-BTSl fragment is not integrated into the genome of this strain. When duplication of AURl locus resulting from genomic integration was detected by PCR, no band was detected in AURGG101 as expected, but the band was detected only in AURGG102 (Fig. 13).<br>
In Fig. 12, when HMGl was used as a probe, a plasmid-derived band appeared in Ndel-digested DNAs (lanes 4-7). In iSrwI-digested DNAs, it is expected that a 8.2 kbp band derived from the plasmid (overlapping a 8.3 kbp band derived from the genome) should appear as in clone 1-2 (No. 4). However, a band shift was observed in clone 13-2 (No. 6) and clone 15-2 (No. 7) as a result of recombination between the vicinity of HMGl in the genome and the introduced plasmid.<br>
From the results of Southern blot hybridization and PCR mapping, the genotypes of the strains/clones used this time can be summarized as shown in Table 8 below. In this Table, "AUR" means a medium to which aureobasidin has been added. "Medium 1" means a medium for preculture, and "Medium 2" means a medium for main culture.<br><br><br>
(7-2)   Northern Blot Hybridization<br>
The results of Northern blot hybridization are shown in Fig. 14. Probes I, II, III and V as shown in Table 7 were used in the hybridization.<br>
In Fig. 14, the strains/clones used in lanes 1 to 10 are the same as used in Fig. 12. Mark "-" indicates transcripts in SD medium, and mark "+" indicates transcripts in SG medium.<br>
ERG20 transcript showed a tendency to decrease in clone 13-2 (No. 6) and clone 15-2 (No. 7) when GALlp transcriptional induction was applied by SG medium.<br>
When the transcription of genes under the control of GALl transcription promoter was induced by SG medium, BTSl transcript increased only in a strain in which GAL]p-BTSl fragment has been integrated into the genome, i.e., AURGG102 (No. 3).<br><br>
However, when compared with HMGl transcript, it is seen that the degree of transcription induction of BTSl is lower. When transcription was induced by SG medium, HMGl transcript increased remarkably in clones No.4 to No. 7 into which GALlp-HMGl fragment was transferred by a plasmid.<br>
(7-3)    Prenyl Diphosphate Synthase Activities<br>
The activities of prenyl diphosphate synthases in crude enzyme fraction was determined using geranyl diphosphate (GPP) and 14-labeled IPP as allylic diphosphate substrates.<br>
Briefly, individual prenyl diphosphates synthesized with GPP and [14C] IPP as substrates were dephosphorylated and developed by TLC, followed by examination of the radioactivity of each spot. As a result, FPP synthase activity was high, and next to that, HexPP (hexaprenyl diphosphate) synthase activity was detected that was by far higher than GGPP synthase activity. Then, relative amounts of reaction products were calculated from autoradiogram, followed by calculation of specific activity per gross protein. The results are shown in Fig. 15. In Fig. 15A, the upper panel shows FPP synthase (FPS) activity and the lower panel shows GGPP synthase (GGPS) activity. In Fig. 15B, the upper panel shows HexPP synthase (HexPS) activity and the lower .panel shows PTase (total prenyl diphosphate synthase) activity. Gray columns show the results in SD medium, and white columns show the results in SG medium. A large part of the total prenyl diphosphate synthase activity is FPP synthase activity. Increase in this activity caused by SG medium was observed. In particular, FPP synthase activity remarkably increased in clone 13-2 (No. 6) and clone 15-2 (No. 7). As a whole, when GPP is used as an allylic substrate, GGPP synthase activity is about 1/20000 of FPP synthase activity and about 1/300 of HexPP synthase activity.   HexPP synthase activity decreased in SG medium.<br>
[EXAMPLE 8]    DNA TRANSFER INTO HOSTS AND CULTIVATION THEREOF<br>
(EXPRESSION BY Saccharomyces cerevisiae)<br><br>
In this Example, in order to construct systems where mevalonate pathway-related enzyme genes are expressed permanently in S, cerevisiae cells, expression vectors for mevalonate pathway-related enzyme genes were prepared by transferring S. cerevisiae-derived genes into expression shuttle vectors comprising a constitutive promoter and various auxotrophic markers. Then, the effect of high expression of these genes upon prenyl alcohol production was evaluated.<br>
(1)	Transformation of Yeast<br>
The resultant expression vector for each mevalonate pathway-related enzyme gene was introduced into hosts. In the newly introduced vector, each mevalonate pathway-related enzyme gene was inserted downstream of TDH3 transcription promoter TDH3p(=GAPp).    As hosts, the following strains/clones were used.<br>
A451<br>
AURGG101<br>
YPH499<br>
AURGG703<br>
YPH500<br>
W303-1A<br>
W303-1B<br>
EUG5 (derived from A451)<br>
EUG8 (derived from A451)<br>
EUG12 (derived from YPH499)<br>
EUG24 (derived from YPH500)<br>
EUG27 (derived from YPH500)<br>
15-2 (pYHMG044/AURGG101) (derived from AURGGlOl)<br>
(2)	Cultivation of Yeast<br>
Each of the mevalonate pathway-related enzyme gene-transferred-yeast clones was precultured in an SD selection medium corresponding to the marker gene used.    Then, 25<br><br>
μl of preculture broth was added to 2.5 ml of YM or SG (the glucose component of SD is replaced with galactose) medium and cultured under reciprocal shaking at 130 rpm at 26°C for 4 days. When the preculture broth was added to SG medium, cells were washed with physiological saline in advance so that no glucose component was brought into the medium. When YPH499-derived clones were used, adenine sulfate was added to the medium to give a concentration of 40 |ig/ml.<br>
(3)	Pentane Extraction<br>
After the cultivation of mevalonate pathway-related enzyme gene-transferred yeast clones, OD600 was determined using 30-fold dilutions of the culture. Then, 2.5 ml of methanol was added thereto and mixed. To this mixture, about 5 ml of pentane was added and agitated vigorously. Then, the resultant mixture was left stationary. The pentane layer was transferred into a fresh, glass test tube. This test tube was placed in a draft where the pentane was evaporated to concentrate the solute. Subsequently, 10 jal of 1.0 ml/L iindecanol was added thereto as an internal standard substance to thereby prepare a sample for GC/MS.<br>
(4)	GC/MS Analysis<br>
The fraction extracted with pentane was separated, identified and quantitatively determined with HP6890/5973 GC/MS system (Hewlett-Packard, Wilmington, DE). The column used was HP-5MS (0.25 mm x 30 m; film thickness 0.25 jam). Analytical conditions were as described below. The same conditions were used for all the GC/MS analyses in this specification.<br>
Inlet temperature: 250°C Detector temperature: 260°C [MS zone temperatures]<br>
MS Quad:150°C<br>
MS Source: 230°C<br><br>
Mass scan range: 35-200 [Injection parameters]<br>
Automated injection mode<br>
Sample volume: 2μl<br>
Methanol washing 3 times and hexane washing twice<br>
Split ratio: 1/20<br>
Carrier gas: helium 1.0 ml/min Solvent retardation: 2 min [Oven heating conditions]<br>
115°Cfor90sec<br>
Heating up to 250°C at 70°C /min and retaining for 2 min<br>
Heating up to 300°C at 70°C /min and retaining for 7 min After 0 hr<br>
Internal standard:    0.01 μl of 1-undecanol in ethanol Reliable standards:    (all-E)-Nerolidol (Eisai)<br>
(all-E)-Famesol (Sigma) (all-E)-Geranylgeraniol (Eisai) Squalene (Tokyo Kasei Kogyo)<br>
(5)   Results<br>
Relations between genes, expression vectors, hosts, culture conditions (medium, temperature and cultivation period) and maximum GGOH yields are summarized in Table 9 below.<br><br><br><br><br><br><br><br><br><br>
In the above Table, "Ec" in the host column means E. coli and "Sc" S. cerevisiae. In the "medium" column, "YM(20)" means that this YM medium has initial sugar composition of 20% Glc-80% Gal and that Glc is further added to this medium at a final concentration of 5% on 2 day of the cultivation. Other combinations of medium and numerical value have similar meaning.<br>
(5-1)    Production of GGOH by ERG20 Expression<br>
When pRS435GAP-ERG or pRS445GAP-ERG was introduced into A451, the recombinant produced GGOH at high efficiency.    When pRS445GAP-ERG was used, 0.73 mg/L of GGOH was produced (Table 9). (5-2)   Production of GGOH by BTSl Expression<br>
When pRS435GAP-BTSl or pRS445GAP.BTSl was introduced into A451 or YPH499, GGOH yields increased (Table 9) at a high yield. When the host was A451, the recombinant produced 0.10-0.11 mg/L of GGOH on the average and 0.585 mg/L at the maximum (Table 9). Further, when pRS435GAP-BTSl or pRS445GAP-BTSl was introduced into W303-1A or W303-1B, 0.19-0.85 mg/L of GGOH was produced at the maximum (Table 9). (5-3)    Production of GGOH by Expression of HMG-CoA Reductase Gene or Mutant<br>
Thereof (i)    GGOH Production when Constitutive Promoter-Ligated HMGl Gene Is Transferred<br>
into A451<br>
(expressed as "Constitutive Promoter; HMGl; A451"; this way of expression is also<br>
applied to other recombinants described below) The results of determination of GGOH yields are shown in Fig. 16.    In Fig. 16, 434 and 444 represent the results when pRS434GAP and pRS444GAP vectors were used, respectively.    The right column in this graph indicates the results when the host (A451) before gene transfer was cultured.<br>
These results revealed that GGOH productivity was improved in pRS343GAP-HMG/A451, and that GGOH was produced at 0.105 mg/L on the average and, depending on colonies, at 0.348 mg/L at the maximum by merely activating the transcription of HMGl gene (Table 9).   Thus, this recombinant was found effective for<br><br>
GGOH production, (ii)   Inducible Promoter; HMGl; A451 &amp;AURGG101<br>
Plasmid pYES2-HMG obtained by inserting an HMGl gene(HMGl', a PCR error type HMGl) into inducible promoter GALIp-containing vector pYES2 was introduced into A451 andAURGGlOl(M51 ,aurl::AURl-C).<br>
As a result, GGOH high-yielding clones were obtained. The GGOH yield of AURGGl01-derived clones reached 1.1 mg/L on the average, and they produced 2.2 mg/L of GGOH at the maximum (Fig. 17).<br>
(iii)   Inducible Promoter; HMGl &amp; BTSl; AURGG 102 &amp; AURGG703<br>
Plasmid pYES-HMG obtained by inserting HMGl' into inducible promoter G^E7p-containing vector pYES2 was introduced into A451-derived AURGG 102 and YPH499-derived AURGG703(BTSl is integrated in the host genome).<br>
As a result, when either AURGG 102 or AURGG703 was used as the host, GGOH high   yielding   clones   were   obtained   as   long   as   GALlp   was   used   (Fig.   18). AURGG 102-derived clone produced 1.28 mg/L of GGOH at the maximum, (iv)   Inducible Type Promoter; HMGl A; A451<br>
The following plasmids obtained by inserting a deletion mutant of HMGl' gene into an inducible promoter G^X7p-containing vector pYES2 were introduced separately into A451.<br>
pYHMG026<br>
pYHMG044<br>
pYHMG056<br>
pYHMG062<br>
pYHMG076<br>
pYHMG081<br>
pYHMG100<br>
pYHMG112<br>
pYHMG122 The resultant recombinants were cultured in SG medium, followed by determination<br><br>
of GGOH yields (Fig. 19). In Fig. 19, "HMGlA026" represents the result when pYHMG026 was introduced into A451. Introduction of other genes is expressed in the same manner.<br>
When deletion mutants of HMGJ gene were expressed with the inducible promoter, GGOH high-yielding clones were obtained. HMGlA056 and HMGlA062 were effective for GGOH production. (HMG062/A451 clone produced 0.063 mg/L of GGOH on the average.)<br>
(v)   Inducible Promoter; HMGJ A; AURGG101<br>
The following plasmids obtained by inserting a deletion mutant of HMGl' gene into an inducible promoter GALIp-containing vector pYES2 were introduced separately into AURGG101.<br>
pYHMG026<br>
pYHMG044<br>
pYHMG056<br>
pYHMG062<br>
pYHMG076<br>
pYHMGO81<br>
pYHMG100<br>
pYHMG112<br>
pYHMG122<br>
pYHMG133<br>
The resultant recombinants were cultured in SG medium, followed by determination of GGOH yields (Fig. 20). As a result, GGOH yield of approximately 3.1 mg was observed in HMGlA044. In Fig. 20, the right utmost column represents the GGOH yield of the host AURGGIOI before gene transfer.<br>
(vi)   Inducible Promoter; HMGl A &amp; BTSl; AURGG 102<br>
The following plasmids obtained by inserting a deletion mutant of HMGl' gene into an inducible promoter Gy417p-containing vector pYES2 were introduced separately into AURGG 102.<br><br>
pYHMG027<br>
pYHMG044<br>
pYHMG045<br>
pYHMG059<br>
pYHMG062<br>
pYHMG063<br>
pYHMG076<br>
pYHMG083<br>
pYHMG094<br>
pYHMG106<br>
pYHMG12<br>
pYHMG123<br>
pYHMG134<br>
The resultant recombinants were cultured in SG medium, followed by determination of GGOH yields.<br>
As a result, a clone producing 0.36 mg/L of GGOH on the average was obtained when pYHMG045 was introduced (Fig. 21).<br>
(vii)   Inducible Promoter; HMG1∆ &amp; BTSl; AURGG703<br>
Plasmids pYHMG044 and pYHMG062 obtained by inserting a deletion mutant of HMGl' gene into an inducible promoter GALIp-containing vector pYES2 were introduced separately into AURGG703. The resultant recombinants were cultured in SG medium, followed by determination of GGOH yields.<br>
As a result, pYHMG062-introduced clone produced 0.21 mg/L of GGOH on the average (Fig. 22).<br>
(5-4)   Production of GGOH by Co-Expression of BTSl and HMG-CoA Reductase Gene<br>
pRS435GAP-HMGl or pRS445GAP-HMGl was introduced into YPH499 together with BTSl, and GGOH yields of the resultant recombinants were determined. As a result, a   clone  producing  0.58   mg/L  of GGOH   at  the   maximum   was   obtained   when<br><br>
pRS435GAP-HMGl was introduced (Table 9).<br>
(5-5)   Production of GGOH by Co-Expression of ispAm, ORF182(/J/), HMGS, ERGS, ERG10 or ERG19 together with a Deletion Mutant HMG-CoA Reductase Gene (HMGJA) ispAm, ORF182(idi), HMGS, ERGS, ERGIO or ERG19 was transferred into<br>
AURGGlOl together with HMGIA, and GGOH yields of the resultant recombinants were<br>
determined.    As a result, clones producing 0.6-2.7 mg/L of GGOH at the maximum were<br>
obtained (Table 9).<br>
[EXAMPLE 9]   DNA TRANSFER INTO HOST AND CULTIVATION OF THE HOST<br>
(Expression in Escherichia coli)<br>
The following vectors were transferred into E. coli JM109: pALisp4, pALisplS, pALispl6 and pALispl8 as expression vectors for E. coli FPP synthase gene \spA\ p4D5 p4E, p4M, p8M, pl5D, pl5E, pl6D, pl6E, pl6M, pl8E and pl8M as expression vectors for ispA(Y19D\ ispA{Y79E) and ispA{Y79M) that are GGPP synthase genes converted from ispA through a substitution mutation; and pFPSm21 and pFPSm31 as expression vectors for Y81M mutants of B. stearothermophilus FPP synthase gene fps. The resultant recombinants were precultured. To a 50 ml medium containing 2x YT and 1 mM IPTG in a 300 ml flask, 0.5 ml of the preculture broth was added. Antibiotics (ampicillin and chloramphenicol), if necessary, 5 mM (about 0.12% (w/v)) IPP and 5 mM DMAPP were added thereto, and the cells were cultured at 37°C for 16 hr under shaking.<br>
After completion of the cultivation, potato acid phosphatase was added to the culture supernatant and the precipitate disrupted by sonication, followed by extraction of prenyl alcohols with pentane as an organic solvent. Then, the prenyl alcohols were identified and quantitatively determined by GC/MS. Further, in order to ascertain whether prenyl alcohol production can be carried out without the addition of IPP and DMAPP, plasmid pl6M obtained in section (1) in Example 3 (designated pALispA16m) and /PPA-isomerase gene<br><br>
idi-retaining p3-47-13 obtained in section (10-2) in Example 2 were introduced into E. coli JM109, which was then precultured. To a 50 ml medium containing 2x YT and 1 mM IPTG in a 300 ml flask, 0.5 ml of the preculture broth was added. Antibiotics (ampiciUin and chloramphenicol) were added thereto, if necessary. Then, the cells were cultured at 37oC for 16 hr under shaking.<br>
As a result, GGOH yields when IFF and DMAFF were added to the medium were as follows. When mutant j^5 was introduced (pFFSm21 and pFPSm31 in Fig. 23), GGOH yields were 16.1 mg/L and 6.9 mg/L. When mutant ispA was introduced, p4M-, pl6M-and pl8M-retaining JM109 cells produced GGOH at 15.5 mg/L, 21.9 mg/L and 6.0 mg/L, respectively (Fig. 23). In p4M and pl6M where Y79M mutation is introduced, a high GGOH activity was recognized in the precipitate fraction. From these results, it is considered that pALispA4 and pALispA16 express FPP synthase gene active in E. coli cells, and that their substitution mutant type plasmids p4M and pl6M also have sufficient expression activity.<br>
When IPP and DMAPP were not added to the medium, the GGOH yield was 0.07 mg/L in pALispA16m-retaining JM109. When pALispA16m and p3-47-13 (retaining /PPA-isomerase gene) were co-expressed, prenyl alcohol productivity was 0.12 mg/L as calculated for GGOH.<br>
[EXAMPLE 10]   PRENYL ALCOHOL PRODUCTION BY EXPRESSION OF FUSION<br>
GENES<br>
It is assumed that the GGPP synthase encoded by S. cerevisiae BTSl prefers FPP to DMAPP (dimethyl allyl diphosphate) as a primer substrate. Therefore, it was believed that enforcement of FPP synthesis ability is required simultaneously for enforcing the ability to synthesize GGPP (the precursor of GGOH) from IPR<br>
In view of this, it was attempted in this Example to create fusion genes composed of BTSl and ERG20, to express them in S, cerevisiae cells and to ascertain whether GGOH productivity improves or not.   Further, it was also attempted to incorporate a nucleotide<br><br>
sequence encoding an ER transition signal downstream of BTSl, ERG20 or fusion genes thereof and to examine its effect upon prenyl alcohol production.<br>
(1)    Preparation of Plasmid DNA<br>
PCR reactions were carried out using pYESGGPS, which is a pYES2 plasmid incorporating GGPP synthase gene BTSl, and pT7ER20, which is a pT7 plasmid incorporating FPP synthase gene ERG20, as templates. The PCR primers used are as follows.<br>
SacII-BTS 1: 5'-TCC CCG CGG ATG GAG GCC AAG ATA GAT-3'<br>
(SEQ ID NO:107)<br>
BTS1 -Xhol: 5'-CAA CTC GAG TCA CAA TTC GGA TAA GTG-3'<br>
(SEQ ID NO: 108)<br>
ERG20HDEL-XbaI: 5'-GCT CTA GAG TTC GTC GTG TTT GCT TCT CTT GTA<br>
AAC TT-3' (SEQ ID NO: 109)<br>
BTSIHDEL-Xhol: 5'-TAT CTC GAG TCA CAA TTC GTC ATG TAA ATT GG-3'<br>
(SEQ ID NO: 110)<br>
BTSI-109I: 5'-GCA GGG ACC CCA ATT CGG ATAAGT GGT C-3'<br>
(SEQ ID NO: 111)<br>
109I-BTS1: 5'-GTA GGG TCC CTG GAG GCC AAG ATA GAT G-3'<br>
(SEQ ID NO: 112)<br>
ERG20-109I: 5'-GCA GGG ACC CTT TGC TTC TCT TGT AAA CT-3'<br>
(SEQ ID NO: 113)<br>
109I-ERG20: 5'-GTA GGG TCC TCA GAA AAA GAAATT AGG AG-3'<br>
(SEQIDNO: 114)<br>
-21: 5'-TGTAAAACGACGGCCAGT-3'(SEQ ID NO: 115)<br>
T7: 5'-TAA TAC GAC TCA CTA TAG GG-3' (SEQ ID NO: 116)<br>
ERG20HDEL-XbaI: 5'-GCT CTA GAG TTC GTC GTG TTT GCT TCT CTT GTA<br>
AAC TT-3'(SEQ ID NO: 117)<br><br>
BTSIHDEL-Xhol: 5'-TAT CTC GAG TCA CAA TTC GTC ATG TAA ATT GG-3' (SEQ ID NO: 118)<br>
The nucleotides from position 3 to position 8 of ERG20HDEL-XbaI and the nucleotides from position 4 to position 9 of BTSIHDEL-Xhol (underlined portions) represent the Sacll, Xhol or Xbal recognition site for vector ligation. The nucleotides from position 4 to position 10 of BTSM09I, 109I-BTS1, ERG20-109I and 109I-ERG20 (underlined portions) individually represent the EcoO\091 recognition site for fusion gene preparation.<br>
The PCR was carried out in the following reaction solution.<br>
Ix KOD-Plus buffer (Toyobo)<br>
0.2 mM dNTPs<br>
0.25 mM MgS04<br>
15 pmol Primer 1<br>
15 pmol Primer 2<br>
0.01-0.1 μg Template DNA<br>
1 unit KOD-Plus DNA polymerase (Toyobo)<br>
Total: 50μl<br>
KOD-Plus contains 1.6 μg/μl of KOD antibody. Following an initial denaturation of 2 min at 94°C, the PCR was carried out for 30 cycles each consisting of 15 sec at 94°C, 30 sec for 55°C and 1 min at 68°C.    Then, the solution was retained at 68°C for 2 min.<br>
First PCR was carried out using the combinations of a template and primers (Primer 1 + Primer 2) as indicated in Table 10 and Fig. 24. The designations of PCR products are also shown in Table 10 and Fig. 24. In Fig. 24, the designations of final plasmids are shown in the leftmost column. The sequences written in gray letters represent amino acid sequences. Of these, GS was introduced into the binding sequence of the fusion gene, and HDEL was inserted as an ER transition signal.   Arrowheads indicate the location and<br><br>
direction of individual primers used in the PCR.<br><br><br>
PCR products #9, #10, #11, #12, #13 and #14 were digested with the restriction enzyme Eco0109I. Then, #9 and #11, #10 and #12, #9 and #13, and #10 and #14 were ligated to each other individually. Using the resultant ligation solution as a template and combinations of SacII-BTSl and -21, T7 and BTSl-XhoI, SacII-BTSl and ERG20HDEL-XbaI, and T7 and BTSIHDEL-Xhol as primer 1 and primer 2, second PCR was carried out under the same conditions as in the first PCR. As a result, second PCR products #9-#ll, #10-#12, #9-#13 and #i0-#14 were obtained.<br>
The product #9-#ll was digested with Sacll and BamHl, and inserted into the Sacll-Bamm site of pRS435GAP and pRS445GAP to obtain pRS435GGFand pRS445GGF, respectively.<br>
The product #10-#12 was digested with Xbal and Xhol, and inserted into the Xbal-Xhol site of pRS435GAP and pRS445GAP to obtain pRS435FGG and pRS445FGG, respectively.<br>
The product #9-#13 was digested with Sacll and Xbal, and inserted into the Sacll-Xbal site of pRS435GAP to obtain pRS435GGFHDEL.<br>
The product #10-#14 was digested with Xbal and Xhol, and inserted into the Xbal-Xhol site of pRS435GAP and pRS445GAP to obtain pRS435FGGHDEL and pRS445FGGHDEL, respectively.<br><br>
The product #7 was digested with SacU and Xbal, and inserted into the SacU-Xbal site of pRS435GAP and pRS445GAP to obtain pRS435FHDEL and pRS445FHDEL, respectively.<br>
The product #6 was digested with BamUl and XJwl, and inserted into the BamUl-Xhol site of pRS435GAP and pRS445GAP to obtain pRS435GGHDEL and pRS445GGHDEL, respectively.<br>
It was confirmed by DNA sequencing that each of the resultant plasmid DNAs has the exact nucleotide sequence as designed.<br>
As plasmids for expressing non-fused BTSl and ERG20 genes separately, pRS435GAP^BTSl (called pRS435GG), pRS445GAP-BTSl (called pRS445GG), pRS435GAP-ERG20 (called pRS435F) and pRS445GAP-ERG20 (called pRS445F) were used. As plasmids for expressing HMGl, pRS434TEF-HMGl and pRS434GAP-HMGl were used.<br>
(2)    Preparation of Recombinants<br>
Recombinants were prepared by introducing the plasmid prepared above into the host using Frozen EZ yeast transformation kit (Zymo Research, Orange, CA). As the host, A451, YPH499, AHl (pRS434GAP-HMGl/A451), YHl (pRS434GAP-HMGl/YPH499), EUG5 and EUG12 were used.<br>
(3)   Determination of Prenyl Alcohol Yields<br>
Recombinants except for EUG strains were inoculated into SD (synthetic dextrose) selection liquid medium. EUG strains were inoculated into SGR medium (a medium in which the glucose component of SD medium is replaced with galactose and raffmose). All of them were cultured at 30oC to prepare preculture broth. Ten or 25l of the preculture broth was added to 1.0 or 2.5 ml of YM7+ade medium (YM, pH 7, 40 |ag/ml adenine sulfate) or YMO medium [YM7+ade, 1% soybean oil, 0.1% ADEKANOL LG-109 (Asahi Denka Kogyo, Tokyo, Japan)] and cultured at 30°C for 4 days or 7 days under reciprocal shaking at 130 rpm.<br><br>
After completion of the cultivation, an equal volume of methanol was added to the culture broth and mixed. Approximately 2 volumes of pentane was added to this mixture, agitated vigorously and then left stationary. The resultant pentane layer was transferred into a fresh glass tube, which was then placed in a draft. Pentane was evaporated therein to condense the solute components. Subsequently, prenyl alcohols were identified and quantitatively determined by GC/MS using undecanol as an internal standard for quantitative determination. At that time, the degree of cell growth was also examined by diluting 20 |al of the culture broth 30-fold with water and measuring the absorbance at 600 nm.<br>
For GC/MS analysis, PH6890/5973 GC/MS system (Hewlett-Packard, Wilmington, DE) was used.<br>
(4)    Results and Observations<br>
Maximum GGOH yields obtained by expressing fusion genes are listed in Table 11.<br><br><br><br><br><br>
(4-2)   Expression of ERG20 and BTSl in A451<br>
Changes in prenyl alcohol yields when fusion genes were expressed in A451 are shown in Fig. 25. In Fig. 25, "435GGF" represents pRS435GGF, and "435GGFHDEL" represents pRS435GGFHDEL (in the following Figures, these terms have the same meaning). ODeoo represents absorbance at 600 nm. Fig. 25 also shows the resuhs when an expression vector integrating non-fused BTSl was introduced into A451 (435GG).<br>
Even when pRS445GAP"BTSl was introduced (expressed as "445GG/A45r' in this Figure), GGOH yield of 0.44 mg/L was observed on the average.<br>
(4-3)    Expression of ERG20 and BTSl in YPH499<br>
Changes in prenyl alcohol yields when fusion genes were expressed in YPH499 are shown in Fig. 26. In Fig. 26, "499" represents YPH499; "435GGF" represents pRS435GGF; and "445GGFHDEL" represents pRS445GGFHDEL (in the following Figures, these terms have the same meaning). Fig. 26 also shows the results when non-fused BTSi-integrated expression vector was introduced into YPH499.<br>
When pRS435GAP-BTSl was introduced (expressed as "435GG/499" in this Figure), GGOH yield of 0.11 mg/L was observed on the average. When ERG20-BTS1 fusion gene-integrated pRS435FGG was introduced (expressed as "435FGG/499" in this Figure), GGOH yield of 0.20 mg/L was observed on the average. When pRS435GGF was introduced (expressed as "435GGF/499" in this Figure), GGOH yield of 0.39 mg/L was observed on the average. When pRS35GGFHDEL was introduced (expressed as "435GGFHDEL/499" in this Figure), GGOH yield of 0.62 mg/L was observed on the average. Thus, it was recognized that fusion genes and HDEL sequence were effective for improving GGOH productivity.<br>
(4-4)   Expression of HMGl, ERG20 and BTSl in YPH499<br>
The present inventors considered that it may be possible to obtain clones of still higher GGOH productivity by co-expressing HMGl and other gene(s) in clones pRS434GAP-HMGl/YPH499   (YHl)   and   pRS434TEF-HMGl/YPH499   which   were<br><br>
obtained by introducing an HMGl expression vector into YPH499.<br>
Fig. 27 shows prenyl alcohol yields when a non-fused ERG20 or 5757-integrated expression vector was further introduced into a host clone that had been already prepared by introducing pRS434TEF-HMGl into YPH499. In Fig. 27, "434TEFp-HMGr' represents a clone to which pRS434TEF-HMGl is introduced. TEFp is the transcription promoter of TEF2 gene. "499" represents YPH499. "435F" represents pRS435F and "445F" pRS445F. (These terms have the same meaning in the following Figures.) When BTSl was introduced alone into YPH499, GGOH yield was only 0.11 mg/L. On the other hand, when a BTSl expression vector was introduced into the TEF2p-//A/G7-transferred clone, GGOH yield was 0.40 mg/L on the average (see "435GG &amp; 434TEFp-HMG 1/499" in Fig. 27); and when a BTSl expression vector was introduced into the G^Pp-HMG7-transferred clone, 0.49 mg/L of GGOH was produced (see "435GG &amp; 434GAPp-HMG 1/499" in Fig. 27). Thus, a possibility of prenyl alcohol mass production system by co-expression of HMG-CoA reductase gene and a prenyl diphosphate synthase gene was indicated.<br>
Subsequently, using GAPp-HMG7-transferred YHl (pRS434GAP-HMGl/YPH499) as a host, ERG20-BTS1 fusion genes or HDEL signal-containing genes prepared in the invention were expressed therein with TDH3 transcription promoter GAPp (TDH3p), Prenyl alcohol yields of the resultant clones were determined.<br>
The results are shown in Fig. 28. In Fig. 28, "434GAPp-HMGl" represents a clone into which pRS434GAP-HMGl is transferred. GAPp is the transcription promoter of TDH3 gene. GGOH productivity improved when an HDEL signal-ligated prenyl diphosphate synthase gene and HMGl were co-expressed. The productivity was further improved by the introduction of ERG20-BTS1 fusion gene. In particular, pRS435GGF-and pRS435GGFHDEL"transferred clones exhibited a remarkable improvement. They produced 1.55 mg/L and 1.50 mg/L of GGOH on the average, respectively (see "435GGF &amp; 434GAPp-HMGl/499" and "435GGFHEDL &amp; 434GAPp-HMG 1/499" in Fig. 28).<br>
(4-5)   Prenyl Alcohol Productivity in Soybean Oil-Containing Medium<br>
ERG20-BTS1   fusion   gene-transferred   clones,   which   are   GGOH-producing<br><br>
recombinants created in the invention, were cultured in YM7 (YM, pH 7) medium and YMO (YM7, 0.1% ADEKANOL LG109, 1% soybean oil) medium for 4 to 7 days, followed by determination of prenyl alcohol yields. The results obtained using A451-derived clones as hosts are shown in Figs. 29A and 29B. The results obtained using YPH499-derived clones as hosts are shown in Figs. 30A and 30B. In Figs. 29A and 29B, "AHl" represents pRS434GAPp-HMGl/451, and "GGFHDEL" represents pRS435GGFHDEL. "-1" represents the yield after 4 days cultivation, and "-2" represents the yield after 7 days cultivation. Since cells are suspended in soybean oil in YMO medium, the amount of cells is expressed as the number of cells. "10-3 cell/μl" means the number of cells per microliter divided by 1000.<br>
While pRS435GGF/A451 produced 0.26 mg/L of GGOH on the average when cultured in YM7 medium for 7 days (Fig. 29A; upper panel; GGF/A451 -2), the yield increased to 0.98 mg/L on the average in YMO medium (Fig. 29B; upper panel; GGF/A451 -2). Also, when AHl (pRS434GAP-HMGl-transferred A451) was used as a host, pRS435GGFHDEL-transferred recombinant produced 2.5 mg/L of GGOH on the average (Fig. 29B; middle panel; GGFHDEL/AHl -2) and 3.5 mg/L of GGOH at the maximum in YMO medium (Table 11; see the rows coming under the gene name "435GGFHDEL"). Even when EUG5 obtained by replacing the ERG9 transcription promoter of A451 with GALl promoter (see Example 6) was used as a host, pRS435GGF-transferred recombinant produced more GGOH in YMO medium. This recombinant produced 6.6 mg/L of GGOH on the average when cultured in YM7 medium for 7 days (Fig. 29A; lower panel; GGF-EUG5 -2), but the yield increased to 9.6 mg/L on the average when cultured in YMO medium (Fig. 29B; lower panel; GGF/EUG5 -2).<br>
Improvement in GGOH productivity by the use of YMO medium was also observed when YPH499-derived clones were used as hosts (Figs. 30A and 30B). While pRS435GGF-transferred YPH499 produced 0.19 mg/L of GGOH on the average when cultured in YM7 medium for 7 days (Fig. 30A; upper panel; GGF/YPH499 -2), it produced 2.5 mg/L of GGOH on the average in YMO medium (Fig. 30B; upper panel; GGF/YPH499 -2).   Further, when pRS435GGF or pRS435GGFHDEL was transferred into YHl that<br><br>
co-expresses HMGl and cultured in YMO medium for 7 days, both recombinants produced 5.6 mg/L of GGOH on the average (Fig. SOB; middle panel; GGF/YHl -2 and GGFHDEL/YHl -2). When EUG12 that was created from YPH499 in the same manner as used in the creation of EUG5 (see Example 6) was used as a host, pRS435GGF- or pRS435GGFHDEL-transferred recombinant produced approximately 3.7-4.0 mg/L of GGOH. Thus, it was suggested that the GGOH productivity of YPH499-derived clones can be improved with a combination of HMGl and a prenyl diphosphate synthase gene.<br>
[EXAMPLE 11]    EFFECTS OF VARIOUS GLUCOSE-GALACTOSE COMPOSITIONS<br>
IN MEDIA UPON PRENYL ALCOHOL PRODUCTION<br>
(1)   Introduction of Vectors into Hosts and Their Cultivation<br>
In this Example, how the prenyl alcohol production in budding yeast will change with varied glucose-galactose (Glc-Gal) composition ratios is examined. In addition, effects of the expression of BTS1-ERG20 fusion genes upon prenyl alcohol production are also examined.<br>
Vectors were introduced into yeast hosts using Frozen EZ yeast transformation II kit purchased from Zymo Research (Orange, CA). As expression vectors for BTS1-ERG20 fusion gene, pRS435GGF and pRS435GGFHDEL were used. As hosts, A451, YPH499, AHl, EUG5 and EUG12 were used. Each of the resultant transformants was grown on an agar plate of SGR-based selection medium that has an appropriate auxotrophy as an indicator. For the purpose of cloning, cultivation on the selection medium agar plate was carried out twice.<br>
The transformant prepared was precultured in SGR selection medium. Then, 0.01-0.05 ml of the preculture broth was added to 1-5 ml of YM7 medium and cultured in a test tube 18 mm in diameter at 30 °C under reciprocal shaking at 130 rpm. Five types of YM7 medium having the following sugar components (Glc-Gal composition ratios) were prepared in advance: 0% Glc-100% Gal; 20% Glc-80% Gal; 50% Glc-50% Gal; 75% Glc-25% Gal; and 100% Glc-0% Gal.   First, cells were cultured in these media at 30 °C<br><br>
under reciprocal shaking at 130 rpm. Two days after the start of cultivation, Glc was added further to each medium to give a final concentration of 5% (w/v). Cells were cultured further up to day 7.<br>
(2)    Results<br>
(2-1)    GGOH Production by A451<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into A451 separately are shown in Figs. 31A-31C.    In both cases, GGOH was detected little. Characteristically, pRS435GGFHDEL/A451  cultured for 4 days exhibited the highest GGOH yield (0.56 mg/L on the average) when the initial Glc ratio was 20%. (2-2)    GGOH Production by AHl<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into AHl separately are shown in Figs. 33A-32C. BTS1-ERG20 fusion gene-transferred AHl clones cultured for 2-4 days also exhibited the highest GGOH yield (3.32 mg/L) when the initial Glc ratio was 20%. When these clones were cultured for 7 days, they exhibited the highest GGOH yield (4.13 mg/L on the average) when the initial Glc ratio was 50-80%. (2-3)    GGOH Production by EUG5<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into EUG5 separately are shown in Figs. 33A-33C.    In both pRS435GGF-transferred clone and pRS435GGFHDEL-transferred clone, good results were obtained when they were cultured for 2-4 days with the initial Glc ratio of 20-80%. (2-4)   GGOH Production by YPH499<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into YPH499 separately are shown in Figs. 34A-34C.    Like in the case where A451 was used, GGOH was detected little. (2-5)   GGOH Production by YHl<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into YHl separately are shown in Figs. 35A-35C. When the recombinant clones were cultured for 7 days, high GGOH yields were obtained with the initial Glc ratio of 100%.<br><br>
(2-6)   GGOH Production by EUG12<br>
GGOH yields when pRS435GGF and pRS435GGFHDEL were transferred into EUG12 separately are shown in Figs. 36A-36C. When the initial Glc ratio was 20%, both of the recombinant clones exhibited high prenyl alcohol productivity. When pRS435GGF/EUG12 was cultured for 4 days with the initial Glc ratio of 20%, this clone produced 7.6 mg/L of FOH and 5.4 mg/L of GGOH though the amount of cells corresponded to ODeoo = LI. It is believed that these production results are very efficient as productivity per cell.<br>
[EXAMPLE 12]    JAR FERMENTER CULTIVATION OF pRS435GGF/YHl AND 15-2<br>
(1)   pRS435GGF/YHl<br>
In order to produce GGOH in a large quantity, pRS435GGF/YHl clone that had produced GGOH preferentially at 5.6 mg/L (see Example 10) was cultured in a jar fermenter under the conditions described below. <fermenter medium><br>
5% glucose   (YM itself contains 1% glucose.    Thus, the final concentration of glucose becomes 6%.) YM broth (Difco) 3%) soybean oil (Nacalai Tesque) 0.1% ADEKANOL LG109 (Asahi Denka) <operational conditions><br>
Cultivation apparatus: MSJ-U 10 L Cultivation Apparatus (B. E. Marubishi) Medium volume: 5 L Cultivation temperature: 33°C Aeration rate: 1 vvm Agitation: 300 rpm<br>
pH: controlled proportionally with the following parameters using 4 N sodium hydroxide solution and 2N hydrochloric acid solution:<br><br>
Proportional Band	1.00<br>
Non Sensitive Band	0.15<br>
Control Period	16 Sec<br>
Full Stroke	1 Sec<br>
Minimum Stroke	OSec<br>
As a result, RS435GGF/YH1 clone could produce 128 mg/L of GGOH when cultured for 115 hr. At that time, yields of squalene (SQ), FOH and nerolidol (NOH) were 15mg/L, 5 mg/L and almost 0, respectively. Thus, the inventors have succeeded to create a system that produces GGOH alone in a large quantity by fermentation (Fig. 37).<br>
(2)    15-2 Clone<br>
15-2 Clone (pYHMG044-retaining AURGGlOl) described in Example 7 was inoculated from a slant into GSM-URA (BIOlOl) + DOB (BIOlOl) medium (200 ml in a 500 ml three-necked flask equipped with a baffle) and cultured at 30°C at 130 rpm for 2 days. Subsequently, centrifugation (1500 rpm, 5 min, 4°C) and washing with sterilized physiological saline were repeated 3 times to completely remove the glucose contained in the culture broth. Then, 50 ml of this preculture broth was cultured in a jar fermenter under the same conditions as used in (1) above for pRS435GGF/YHl. However, the medium used was as described below and the cultivation temperature was 26°C. <fermenter medium><br>
5% galactose   (YM contains 1% glucose.    Thus, the final sugar concentration<br>
becomes 6%.)<br>
YMB without Amino Acids (Difco)<br>
1% soybean oil (Nacalai Tesque)<br>
0.1% ADEKANOL LG109 (Asahi Denka)<br>
As a resuh, 15-2 clone could produce 3 mg/L of GGOH when cultured for 150 hr (Fig. 38).<br><br>
[EXAMPLE 13]    EXPRESSION OF FUSION GENES<br>
In order to examine whether pRS435GGF-transferred cells and pRS435GGFHDEL-transferred cells are expressing the respective fusion gene products or not, analysis of transcription products and translation products were carried out by Northern blot hybridization and Western blotting.<br>
(1)   Northern Blot Hybridization<br>
Fig. 39 shows the results of Northern blot hybridization analysis of transcription<br>
products in YPH499-derived recombinants to which pRS435GGF and pRS435GGFHDEL<br>
are transferred separately and YHl-derived recombinants to which pRS435GGF and<br>
pRS435GGFHDEL are transferred separately.    As probes, a DNA fragment located in the<br>
coding region of TUBl gene encoding a-tubulin and the ERG20 probe, BTSl probe and<br>
HMGl probe described in Table 7 in Example 7.    The TUBl probe was prepared in the<br>
same manner as described in Example 7 using oligonucleotides TUBlf-2 and TUBlr-2<br>
described below.   The preparation of RNA and Northern blot hybridization were performed<br>
in the same manner as described in Example 7.<br>
TUBlf-2: 5'-ACG GTA AGA AAT CCA AGC-3' (SEQ ID NO: 119) TUBlr.2: 5'-TAT GAG TCG GCA CCC ACT-3' (SEQ ID NO: 120)<br>
In Fig. 39, — indicates RNA samples from those cells into which prenyl diphosphate synthase gene-retaining plasmids are not transferred; "GGF" indicates RNA samples from pRS435GGF-transferred recombinants; "HDEL" indicates RNA samples from pRS435GGFHDEL-transferred recombinants; and "HMGl" indicates RNA samples from YHl-derived recombinants. From the results obtained with probe TUBl (tubulin a gene), it is understood that an almost equal amount of messenger RNA is obtained from every sample prepared.   When probe ERG20 and probe BTSl were used, an over-expressed.<br><br>
common 3.1 kb band is detected, indicating that fusion genes are transcribed efficiently. The 1.8 kb band detected with probe ERG20 is believed to be the transcription product of the wild-type ERG20 gene in the genome. The results obtained with probe HMGJ show that 4.1 kb RNA (a transcription product of plasmid-derived HMGJ) is detected in a large quantity in all of the pRS434GAP-HMGl-pretransfen-ed clones (i.e., YHl-derived clones; lanes marked with "HMGl" in Fig. 39). This indicates that the transcription of HMGJ is performed efficiently even if a prenyl diphosphate synthase expression plasmid is introduced further as a second plasmid.<br>
(2)    Western Blotting<br>
According to the C-terminal sequences of the polypeptides encoded by ERG20 and BTSI, polypeptides having the amino acid sequences described below were chemically synthesized. Using these polypeptides as antigens, mouse antibodies were prepared by conventional methods (described in common experimental manuals such as F. M. Ausubel et al Eds, Short Protocols in Molocular Biology, 4th Edition, (1999) John Wiley &amp; Sons, Inc., New York). Two milligrams of each of the following peptides was crosslinked to KLH (Keyhole Limpet Hemocyanin) and used as an antigen.<br>
BTSl-C: NH2 Cys Tyr He He Asp His Leu Ser Glu Leu COOH (SEQ ID NO: 121) ERG20-C: NH2 Cys Leu Asn Lys Val Tyr Lys Arg Ser Lys COOH (SEQ ID NO:<br>
122)<br>
Protein was prepared from the 6 strains/clones of YPH499, pRS435F/YPH499, pRS435GGF/YPH499, pRS435FGG/YPH499, pRS435GGFHDELA^PH499 and pRS435GGFA'Hl as described below and subjected to Western blot analysis. Briefly, preculture broth (absorbance at 600 nm was measured, and each broth was diluted with physiological saline to have an equal amount of cells) of each strain/clone was inoculated into a selection medium [for YPH499, SD medium DOB (dropout base: minimum medium whose carbon source is glucose) to which CSM (complete supplement mixture) is added as<br><br>
amino acid or nucleic acid component; for pRS435F/YPH499, SD-L medium (SD medium minus Leu); for pRS435GGF/YPH499, SD-L medium; for pRS435FGG/YPH499, SD-L medium; for pRS435GGFHDEL/YPH499, SD-L medium; for pRS435GGF/YHl, SD-LW medium (SD medium minus Leu and Trp)] and cultured at 30°C under shaking at 130 rpm for 4 days. After harvesting cells with a centrifuge, 2 ml of Y-PER (PIERCE, Rockford, IL) was added per 1 g (wet weight) of cells and agitated vigorously at room temperature for 1 hr to prepare a total protein solution. Twenty micrograms of the resultant total protein was separated on the basis of molecular weights by SDS-polyacrylamide gel electrophoresis (SDS-PAGE; for procedures, see Short Protocols in Molecular Biology, 4th Edition, (1999) John Wiley &amp; Sons, Inc., New York), and examined the state of expression of the gene transferred into the relevant recombinant by Western blotting (see Short Protocols in Molecular Biology, 4th Edition, (1999) John Wiley &amp; Sons, Inc., New York). The Western blotting technique used this time was partially modified in the following points.<br>
1)	Conditions for treating PVDF membrane with primary antibody<br>
In the routine procedures, PVDF membrane is shaken in a 10- to 1000-fold dilution of primary antibody in TBST for 30-60 min.<br>
In the modified procedures, PVDF membrane is shaken in a 2000-fold dilution of the above-mentioned anti-peptide antibody in TBST for 60 min.<br>
2)	PVDF membrane washing conditions<br>
In the routine procedures, the membrane is washed 4 times with 200 ml of TBST solution for 15 min each time.<br>
In the modified procedures, the membrane is washed 5 times with 80 ml of TBST solution for 5 min each time.<br>
3)	Treatment of PVDF membrane with secondary antibody<br>
In the routine procedures, anti-IgG (H+L)-alkali phosphatase conjugate is diluted 200-to 2000-fold with a blocking solution, and then PVDF membrane is soaked and shaken for 30-60 min.<br>
In the modified procedures, PVDF membrane is soaked in a 4000-fold dilution of anti-IgG (H+L)-alkali phosphatase conjugate in TBST and shaken for 30 min.<br><br>
4)    Method of detection of antigen protein bands<br>
In the routine procedures, PVDF membrane washed with TEST and TBS is soaked in BCIP (4-bromo-4-chloro-3-indoxyl-phosphate)/NBT (nitro blue tetrazolium) mixture for 30 min for color formation.<br>
In the modified procedures, PVDF membrane washed with TBST and TBS is soaked in ProtoBlot II AP System with Stabilized Substrate • Mouse solution (Promega, Madison, WI) for 20-40 sec for color formation.<br>
The results of Western blot analysis are shown in Fig. 40. In Fig. 40, "M" represents the lane of molecular markers. "F", "GGF", "FGG", "GGFHDEL" and "GGF/YHl" represent those lanes in which proteins from pRS435F/YPH499, pRS435GGF/YPH499, pRS435FGG/YPH499, pRS435GGFHDEL/YPH499 and pRS435GGF/YHl, respectively, were electrophoresed.<br>
When anti-BTSl-C mouse antibody that detects the polypeptide encoded by BTSl was used, polypeptides corresponding to approx. 79 kDa fusion proteins (GGF, FGG and GGFHDEL, respectively) were detected (in Fig. 40, the bands shoving the mobility marked with an open triangle). From these results, it was found that an FPP synthase-GGPP synthase fusion protein derived from ERG20-BTS1 fusion gene is actually expressed in pRS435GGF-transferred and pRS435GGFHDEL-transferred recombinants. Also, it was expected that the GGPP synthase encoded by BTSl gene is expressed little in non-recombinant cells because no band was detected in the protein from YPH499.<br>
When anti-ERG20-C mouse antibody that detects the polypeptide encoded by<br>
ERG20 was used, it was shown that the expression level of a protein having a molecular<br>
weight (approx. 40 kDa) corresponding to the FPP synthase encoded by ERG20 is elevated<br>
in pRS435F-transferred clone (lane "F") (the band showing the mobility marked with a<br>
filled     triangle).	In     pRS435GGF-transferred     clone     (lane     "GGF")     and<br>
pRS435GGFHDEL-transferred clone (lane "GGFHDEL"), polypeptides corresponding to fusion proteins (GGF and GGFHDEL) were detected (the bands showing the mobility corresponding to approx. 79 kDa; marked with an open triangle).    The fusion gene that<br><br>
should be expressed in pRS435FGG-transferred clone was not detected with the anti-ERG20-C antibody. It is believed that this occurred because the anti-ERG20-C antibody that recognizes a C-terminal portion of FPP synthase could not recognize the fusion enzyme well since GGPP synthase is fused to the C-terminal of FPP synthase in the fusion enzyme. Of the other bands detected with the anti-ERG20-C antibody, an approx. 45 kDa band is believed to be a non-specifically detected protein. Less than 40 kDa bands are believed to be non-specifically detected proteins or degradation products of proteins comprising the amino acid sequence encoded by ERG20 gene.<br>
[EXAMPLE 14]    GGOH PRODUCTION WHEN HMG-CoA REDUCTASE GENE AND<br>
GGF FUSION GENE ARE CO-EXPRESSED<br>
In order to ascertain whether it is also possible to obtain industrially useful GGOH-producing clones frpm strains other than YPH499 (ATCC76625, MATa ura3-52 Iys2-801 ode2-101 trplA63 his3A200 leu2Al) by enhancing the expression of HMG-CoA reductase gene and BTS1'ERG20 fusion gene encoding GGPP synthase-FPP synthase fusion protein, the following strains were transformed with pRS434GAP-HMGl, pRS435GGF and pRS435GGFHDEL.<br><br>
Briefly, pRS434GAP-HMGl was transfeired into INVScl, YPH500, YPH501, W303-1A and W303-1B to prepare IHl, YH2, YH3, WHl and WH2, respectively. Using these recombinants as hosts, pRS435GGF was transferred thereinto to obtain GGF/IHl,<br><br>
GGF/YH2, GGF/YH3, GGF/WHl and GGF/WH2. Also, pRS435GGFHDEL was transferred into those hosts to obtain HDEL/IHl, HDEL/YH2, HDEL/YH3, HDEL /WHl and HDEL/WH2. These recombinants were cultured in YPD07rich medium, followed by determination of prenyl alcohol productivity. The results are shown in Fig. 41. Every recombinant produced GGOH preferentially, and most of them produced 100 mg/L or more of GGOH. In particular, HDEL/YH2 clone produced 189 mg/L of GGOH at the maximum.<br>
[EXAMPLE 15]    GGOH PRODUCTION WHEN A CLONE CO-EXPRESSING HMG-Co<br>
REDUCTASE GENE AND GGF FUSION GENE IS CONVERTED INTO A PROTOTROPH AND THEN DIPLOIDIZED<br>
A GGOH-producing clone GGF/YHl was converted into a prototroph (a strain capable of growing without supplementation of specific nutrients to the medium) by replacing its mutant genes causing auxotrophy with corresponding wild-type genes, and then diploidized by mating with a YPH500-derived clone to thereby obtain GGF/YH3-AHKU clone.    The preparation procedures are as described below.<br>
(1)     Introduction of HISS and ADE2 into GGF/YHl and Introduction of LYS2 and URA3<br>
into YPH500<br>
A HISS fragment was prepared by PCR using pRS403GAP digested with PvwII and Xhol as a template and oligonucleotides HIS3-L (5' TTT TAA GAG CTT GGT GAG CGC 3' (SEQ ID NO: 123)) and HIS3-R (5' TCG AGT TCA AGA GAA AAA AAA 3' (SEQ ID NO: 124)) as primer DNAs under the following conditions.<br><br><br>
In the same manner, an URA3 fragment was prepared using pRS406GAP digested with Pvull and Xhol as a template and oligonucleotides URA3-L (5' TTC AAT TCA TCA TTT TTT TTT 3' (SEQ ID NO: 125)) and URA3-R (5' GGG TAA TAA CTG ATA TAA TTA 3' (SEQ ID NO: 126)) as primer DNAs.<br>
An ADE2 fragment and a LYS2 fragment were also prepared under similar reaction conditions using A451 genomic DNA as a template and ADE-1 (5' ATG GAT TCT AGA ACA GTT GGT 3' (SEQ ID NO: 127)) and ADE-2 (5' TTA CTT GTT TTC TAG ATA AGC 3' (SEQ ID NO: 128)) or LYS-1 (5' ATG ACT AAC GAA AAG GTC TGG 3' (SEQ ID NO: 129)) and LYS-2 (5' TTA AGC TGC TGC GGA GCT TCC 3' (SEQ ID NO: 130)) as primer DNAs. The resultant HIS3 fragment and ADE2 fragment were introduced into pRS435GGFA'Hl successively to thereby obtain pRS435GGFA/Hl-AH that exhibited non-histidine requirement and non-adenine requirement as phenotypes. On the other hand, the LYS2 fragment and URA3 fragment were introduced into YPH500 successively to thereby obtain YPH500-KU that exhibited non-lysine requirement and non-uracil requirement as phenotypes.<br>
(2)    Mating<br>
pRS435GGF/YHl-AH and YPH500-KU were cultured in YM medium at 30°C and streaked onto DOB (dropout base) agar plate medium so that the two clones were crossed with each other. Then, the cells were incubated at 30°C for 3 days. Colonies appearing on the plate were picked up and cultured on a presporulation plate medium (containing 1.6 g<br><br>
of yeast extract, 0.6 g of polypeptone, 100 ml of 20% glucose and 4 g of agar per liter), followed by cultivation on a sporulation plate medium (containing 2 g of potassium acetate, 0.2 g of yeast extract, 500μ1 of 20%) glucose and 4 g of agar per liter). Sporulation was confirmed by microscopic observation. A clone was confirmed to have been converted into a prototroph because it grows on DOB plate (a minimum medium) and also confirmed to have been diploidized because it forms spores on the sporulation medium. This clone was designated GGF/YH3-AHKU.<br>
[EXAMPLE 16]    GGOH PRODUCTION BY FED-BATCH CULTURE (1)<br>
GGF/YH3-AHKU  was   subjected  to   fed-batch  culture  under  the  conditions described below, and GGOH yield was determined.<br>
(1)	Pre-seed Culture<br>
The medium composition was as follows: yeast extract 5 g/L, malt extract 5 g/L, Bacto-Peptone 10 g/L, and glucose 5 g/L.<br>
The pH of the medium was not adjusted. Fifty milliliters of the medium was placed in a 500 ml Sakaguchi flask and sterilized at 120°C for 20 min. One platinum loopful of GGF/YH3-AHKU was scratched from a slant and cultured at 30°C under reciprocal shaking at 120 rpm for 24 hr.    OD (in 26-fold dilution, at 562 nm) reached 0.4.<br>
(2)	Seed Culture<br>
The medium composition was as follows: glucose 20 g/L, MAMENO (Ajinomoto Co., Inc.) 310 mg/L (as calculated for the amount of total nitrogen), KH2PO4 3 g/L, MgSO4 0.5 g/L, ammonium sulfate 5 g/L, CaCl2 0.5g/L, and defoaming agent 0.1 ml/L. If the total nitrogen concentration in MAMENO is 63 g/L, the amount of MAMENO per se added to the medium is 4.9 ml/L.<br>
After dissolving the medium components completely, the pH of each component was adjusted to 5.0 with a KOH solution.    After adjusting the liquid volume, each component<br><br>
was sterilized at 120°C for 20 min.<br>
A one liter mini-jar was used for seed culture. Three hundred milliliters of the medium was placed in the jar, to which 0.06-1 ml of the pre-seed culture was inoculated. Prior to the inoculation, the pH of the medium was adjusted to 5.5. Aeration rate was 1/2 vvm, and the temperature was set at 30°C. The pH was controlled to 5.5 with ammonia. Agitation, which started at 500 rpm, was put under cascade control so that dissolved oxygen (DO) was &gt;20%. Seed culture was terminated at the time point when pH rose. OD (in 51-fold dilution, at 562 nm) reached to 0.18-0.2. The amount of dissolved oxygen mentioned above was calculated taking the amount at saturation as 100%.<br>
(3)   Main Culture<br>
The medium composition was as shown in Table 12 below. The liquid volumes of plot A, plot B, plot C and plot D to the total volume of main culture were 20%, 20%, 30% and 20%, respectively. Com steep liquor (CSL) was treated with sulfuric acid to adjust the pH to 2.0 and then pre-sterilized at 80°C for 1 hr. The concentration of CSL appearing in Table 12 is expressed in terms of the amount of total nitrogen. CSL per se is added at 31.4 g/L to give this concentration. Each plot was sterilized at 120°C for 20 min and mixed together.   After adjustment of the liquid volume, the mixture was put into a vessel.<br><br><br>
A one liter mini-jar was used for main culture. The total volume of main culture in the jar was made 300 ml by adding 10% seed culture to the above mixture. Since the pH of the medium after sterilization was around 2.6, it was raised prior to seed inoculation. Aeration rate was 1/2 vvm, and the temperature was set at 30°C. The pH was controlled to 5.5 with ammonia. Agitation, which started at 500 rpm, was put under cascade control so that dissolved oxygen (DO) was &gt;20%. The feeding of glucose was started 2 hr after the start of cultivation, and conducted in such a manner that the flow rate was increased gradually, as shown in Fig. 42. The maximum flow rate was 3.5 ml/hr, which corresponds to approx. 5.8 g of glucose/L/hr. When it was difficult to secure sufficient dissolved oxygen (DO&gt;20%) by agitation, aeration volume was increased. About 20 hours after the start of cultivation, the predetermined amount of feed was fed. At this time, OD (in 101-fold dilution at 562 nm) reached 0.9-1.0.<br>
Thereafter, feed and other parameters were changed according to culture conditions.<br>
(4)   Examination of Culture Conditions<br>
Cells were grown under the same conditions up to 20 hr after the start of cultivation, and then effects of glucose and ethanol upon GGOH production were examined.<br>
Briefly, 400 g/L of ethanol solution (plot 1) and 500 g/L of glucose solution (plot 2) were fed to the culture. Further, the ethanol solution of plot 1 and the glucose solution of plot 2 were mixed 1:1 to prepare plot 3. The flow rate of feed solutions was set at 3.5 ml/hr at the maximum, and controlled so that the substrate concentration in the culture broth was 1.0 g/L or less. The amounts of accumulated GGOH are shown in Table 13. It was found that GGOH accumulation increases by feeding ethanol as a carbon source. Table 13<br><br><br>
[EXAMPLE 17]    GGOH PRODUCTION BY FED-BATCH CULTURE (2)<br>
GGF/YH3-AHKU clone was inoculated into 200 ml of DOB (dropout base) glucose minimum medium (Q-BIOgene, Carlsbad, CA) and cultured at 30°C under rotating for 3 days. Subsequently, the total volume of the resultant culture was inoculated into 3.35 L of a medium (preadjusted to pH 5.5 with aqueous ammonia) containing 0.09% glucose, 0.075%, KH2PO4, 0.14% magnesium sulfate, 0.45% ammonium sulfate, 5.4% com steep liquor, 0.031% calcium chloride and 0.15% ADEKANOL LG109 (Asahi Denka), and cultured under the conditions described below. Bach culture was conducted using Jar 1, Jar 2 and Jar 3.<br>
Cultivation apparatus: MSJ-U2W (10 L fermenter) (B. E. Marubishi, Chiyoda-ku,<br>
Tokyo)<br>
Cultivation temperature: 33°C<br>
Aeration rate: 0.74 wm<br>
Agitation rate: 900 rpm<br>
pH 5.5 (adjusted with 4 N sodium hydroxide solution and 2 N hydrochloric acid<br>
solution)<br>
Four hours after the start of cultivation, feeding of 40% glucose solution was started. Twenty-one hours after the start of cultivation, the feed solution to Jar 2 was changed to 40% glucose + 3.3% ammonium acetate solution; and the feed solution to Jar 3 was changed to 1.65% ammonium acetate + 50% ethanol + 20% glucose solution. Then, cells were cultured further. Culture broth samples were taken aseptically to analyze and quantitatively determine prenyl alcohols in the same manner as in Example 8. In order to maintain the sugar concentration in the medium at 0.1% or below, the feeding rate was adjusted (18.7 g/h at the maximum).<br>
The results revealed that this method minimizes the generation of FOH and enables efficient production of GGOH by microorganisms (Table 14).    By feeding ethanol and<br><br>
ammonium acetate in addition to glucose, GGOH concentration in the medium reached 2.5 g/L.<br><br>
All publications, patents and patent applications cited herein are incorporated herein by reference in their entity.<br>
INDUSTRIAL APPLICABILITY<br>
According to the present invention, methods of producing prenyl alcohols are provided. Since it is possible to obtain prenyl alcohols (in particular geranylgeraniol) in large quantities according to the present invention, they can be utilized for the production of substances important in vivo and also utilized as reagents for discovering novel physiological activities of active prenyl alcohols. Thus, the methods of the invention are useful.<br>
SEQUENCE LISTING FREE TEXT<br><br>
SEQ ID NO: 24:    synthetic peptide SEQ ID NOS: 25-120:   synthetic DNA SEQ ID NO: 121:    synthetic peptide SEQ ID NO: 122:    synthetic peptide SEQ ID NOS: 123-130:   synthetic DNA<br><br><br><br><br><br><br>
CLAIMS<br>
1. A method of producing a prenyl alcohol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof, culturing the resultant recombinant, and recovering the prenyl alcohol from the resultant culture.<br>
2.	A method of producing a prenyl alcohol, comprising creating a recombinant by<br>
transferring into a host a recombinant DNA for expression or a DNA for genomic<br>
integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a<br>
recombinant DNA for expression or a DNA for genomic integration each comprising a<br>
hydroxymethylglutaryl-CoA reductase gene or a mutant thereof, culturing the resultant<br>
recombinant, and recovering the prenyl alcohol from the resultant culture.<br>
3.	The method according to claim 1 or 2, wherein the prenyl alcohol is<br>
geranylgeraniol.<br>
4.	A method of producing geranylgeraniol, comprising creating a recombinant by<br>
transferring into a host a recombinant DNA for expression or a DNA for genomic<br>
integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a<br>
recombinant DNA for expression or a DNA for genomic integration each comprising an<br>
isopentenyl diphosphate A-isomerase gene, culturing the resultant recombinant, and<br>
recovering geranylgeraniol from the resultant culture.<br>
5. The method according to any one of claims 1 to 4, wherein the prenyl diphosphate synthase gene is selected from the group consisting of the following genes (a) and (b) and fusion genes (c) and (d):<br>
(a)   famesyl diphosphate synthase gene or a mutant thereof<br><br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of famesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added.<br><br>
6.	The method according to claim 5, wherein the famesyl diphosphate synthase gene encodes the amino acid sequence as shown in SEQ ID NO: 2 or 4.<br>
7.	The method according to claim 5, wherein the geranylgeranyl diphosphate synthase gene encodes the amino acid sequence as shown in SEQ ID NO: 6.<br>
8.	A method of producing geranylgeraniol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a hydroxymethylglutaryl-CoA reductase gene or a mutant thereof, culturing the resultant recombinant, and recovering geranylgeraniol from the resultant culture.<br>
9. A method of producing geranylgeraniol, comprising creating a recombinant by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a hydroxymethylglutaryl-CoA reductase gene or a mutant thereof and a recombinant DNA for expression or a DNA for genomic integration each comprising a gene selected from the group consisting of the following (e) through Q):<br>
(e)	isopentenyl diphosphate A-isomerase gene<br>
(f)	mevalonate kinase gene<br>
(g)	acetyl-CoA acetyltransferase gene<br>
(h)   hydroxymethylglutaryl-CoA synthase gene<br>
(i)   phosphomevalonate kinase gene<br>
(j)   diphosphomevalonate decarboxylase gene;<br><br>
culturing the resultant recombinant, and recovering geranylgeraniol from the resultant culture.<br>
10.    The method according to any one of claims 3 to 9, wherein the concentration of geranylgeraniol in the resultant culture is at least 0.05 mg/L.<br>
11.    The method according to any one of claims 1 tolO, wherein the host is yeast or Escherichia coli.<br>
12.    The method according to claim 11, wherein the yeast is Saccharomyces cerevisiae.<br>
13.	The method according to claim 13, wherein the Saccharomyces cerevisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
14.	A recombinant DNA for expression comprising a gene selected from the group consisting of the following genes (a) and (b) and fusion genes(c) and (d):<br><br>
(a)	famesyl diphosphate synthase gene or a mutaift thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of fameyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added;<br>
as well as a transcription promoter and a transcription terminator.<br>
15. The recombinant DNA according to claim 14, wherein the transcription promoter is any one selected from the group consisting of ADHl promoter, TDH3 {GAP) promoter, TEF2 promoter, GALl promoter and tac promoter.<br><br>
14. (Amended) A recombinant DNA for expression comprising a gene selected from the group consisting of the following genes (a) and (b) and fusion genes (c) and (d):<br>
(a)	farnesyl diphosphate synthase gene or a mutant thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of farnesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added;<br>
as well as a transcription promoter and a transcription terminator, and conferring an ability to produce prenyl alcohol to a host when transferred thereinto.<br>
15. The recombinant DNA according to claim 14, wherein the transcription promoter is any one selected from the group consisting of ADHl promoter, TDH3 (GAP) promoter, TEF2 promoter, GALl promoter and tac promoter.<br>
16.    The recombinant DNA according to claim 14, wherein the transcription terminator is CYCl terminator.<br>
17.    A recombinant obtained by transferring into a host the recombinant DNA according to any one of claims 14 to 16.<br>
18.    The recombinant according to claim 17, wherein the host is yeast or Escherichia coli.<br>
19.      The   recombinant   according   to   claim   18,   wherein   the   yeast   is Saccharomyces cerevisiae.<br>
20.    The recombinant according to claim 19, wherein the Saccharomyces<br><br>
cercvisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-i A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
21.    The recombinant according to any one of claims 18 to 20, wherein the recombinant is a prototroph.<br>
22.    The recombinant according to any one of claims 18 to 20, wherein the recombinant is a diploid cell.<br>
23.      The recombinant according to any one of claims 18 to 20, wherein the recombinant is a prototroph and a diploid cell.<br>
24. (Amended) A method of producing a prenyl alcohol according to any one of claims 1 to 13, comprising culturing a microorganism having an ability to produce the prenyl alcohol using a medium comprising any one of the following components (i) through (vi):<br>
(i)    sugar<br>
(ii)    alcohol<br>
(iii)    ammonia gas, aqueous ammonia and/or an ammonium salt<br>
(iv)    a mixture of sodium hydroxide and sulfuric acid<br>
(v)    a mixture of KH2PO4, magnesium sulfate, ammonium sulfate, corn steep liquor, calcium chloride and a surfactant<br>
(vi)    a mixture of two or more of the above components (i) through (v); and recovering the prenyl alcohol from the resultant culture.<br>
25. The method according to claim 24, wherein the microorganism is cultured using a feed solution comprising the following component (i), (ii) or (iii) or a mixture of two or more of said components:<br>
(i)    sugar<br><br>
(ii)    alcohol<br>
(iii)    ammonia gas, aqueous ammonia and/or an ammonium salt,<br>
26. The method according to claim 24, wherein the carbon source component of the feed solution consists of glucose alone up to 12-24 hours after the start of cultivation, and then the carbon source component is sifted to a component containing ethanol.<br>
27. The method according to claim 24, wherein the ratio of ethanol to the total carbon source component of the feed solution is 50% or more after 12-24 hours after the start of cultivation.<br>
28. The method according to claim 24, wherein the carbon source component of the feed solution consists of ethanol alone after 12-24 hours after the start of cultivation.<br>
29.    The method according to claim 24, wherein the concentration of said prenyl alcohol accumulated in the culture is at least 0.1 g/L or more.<br>
30.    The method according to claim 24, wherein the concentration of said prenyl alcohol accumulated in the culture is at least 1 g/L or more.<br>
31.     The  method  according  to  claim  24,  wherein  the  prenyl  alcohol  is geranylgeraniol.<br>
32.    The method according to claim 24, wherein the microorganism is yeast.<br>
33.    The method according to claim 32, wherein the yeast is Saccharomyces cerevisiae.<br><br>
34. The method according to claim 33, wherein the Saccharomyces cerevisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
35.    The method according to claim 24, wherein the microorganism is a recombinant.<br>
36. The method according to claim 35, wherein the recombinant has been created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a mevalonate pathway-related gene or a mutant thereof or a prenyl diphosphate synthase gene or a mutant thereof.<br>
37. The method according to claim 35, wherein the recombinant has been created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a recombinant DNA for expression or a DNA for genomic integration each comprising a mevalonate pathway-related gene or a mutant thereof.<br>
38.      The   method   according   to   claim   36   or   37,   wherein   the   host   is Saccharomyces cerevisiae.<br>
39.	The method according to claim 38, wherein the Saccharomyces cerevisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1J^ strain or W303-1B strain, or a strain derived from any one of said strains.<br>
40.	The method according to claim 36 or 37, wherein the mevalonate pathway-related gene is a hydroxymethylglutaryl-CoA reductase gene.<br><br>
41.	The      method      according      to      claim      40,      wherein      the<br>
hycjlroxymethylglutaryl-CoA reductase gene is HMGl gene.<br><br>
syn and<br><br>
42.    The method according to claim 36 or 37, wherein the prenyl diphosphate thase gene is any one selected from the group consisting of the following genes (a) (b) and fusion genes (c) and (d):<br>
(a)	farnesyl diphosphate synthase gene or a mutant thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of farnesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added.<br><br>
43.    The method according to claim 24, wherein the microorganism is a protbtroph.<br>
44.    The method according to claim 24, wherein the microorganism is a diploid<br>
cell.<br>
45.    The method according to claim 24, wherein the microorganism is a protdtroph and a diploid cell.<br>
46.    The method according to claim 24, wherein the pH of the medium is contrblled.<br><br>
using<br><br>
47.    The method according to claim 46, wherein the pH control is carried out ammonium gas, an ammonium salt solution, a sodium hydroxide solution or<br><br>
sulfuric acid.<br><br>
48. (Added) A method of producing a prenyl alcohol, comprising culturing a microorganism having an ability to produce the prenyl alcohol using a medium comprising any one of the following components (i) through (vi):<br>
(i)    sugar<br>
(ii)    alcohol<br>
(iii)    ammonia gas, aqueous ammonia and/or an ammonium salt<br>
(iv)    a mixture of sodium hydroxide and sulfuric acid<br>
(v)    a mixture of KH2PO4, magnesium sulfate, ammonium sulfate, corn steep liquor, calcium chloride and a surfactant<br>
(vi)    a mixture of two or more of the above components (i) through (v); and   recovering   the   prenyl   alcohol   from   the   resultant   culture,   wherein   the microorganism   is  cultured  using  a   feed   solution   comprising  a   mixture   of the components (i), (ii) and (iii) or the components (ii) and (iii).<br>
49. (Added) The method according to claim 48, wherein the carbon source component of the feed solution consists of ethanol alone after 12-24 hours after the start of cultivation, and then the carbon source component is sifted to a component containing ethanol.<br>
50. (Added) The method according to claim 48, wherein the ratio of ethanol to the total carbon source component of the feed solution is 50% or more after 12-24 hours after the start of cultivation.<br>
51. (Added) The method according to claim 48, wherein the carbon source component of the feed solution consists of ethanol alone after 12-24 hours after the start of cultivation.<br>
52.    (Adied)       The    method    according    to    claim    48,    wherein    the<br><br>
concentration of said prenyl alcohol accumulated in the culture is at least 0.1 g/L or more.<br>
53. (Added) The method according to claim 48, wherein the concentration of said prenyl alcohol accumulated in the culture is at least 1 g/L or more.<br>
54.    (Added)        The method  according  to  claim 48,  wherein  the  prenyl alcohol is geranylgeraniol.<br>
55.    (Added)      The    method    according    to    claim    48,    wherein    the microorganism is yeast.<br>
56.    (Added)      The method  according  to claim 55, wherein the yeast is Saccharomyces cerevisiae.<br>
57. (Added) The method according to claim 56, wherein the Saccharomyces cerevisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
58.    (Added)      The    method    according    to    claim    48,    wherein    the microorganism is a recombinant.<br>
59. (Added) The method according to claim 58, wherein the recombinant has been created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a mevalonate pathway-related gene or a mutant thereof or a prenyl diphosphate synthase gene or a mutant thereof.<br><br>
60. (Added) The method according to claim 58, wherein the recombinant has been created by transferring into a host a recombinant DNA for expression or a DNA for genomic integration each comprising a prenyl diphosphate synthase gene or a mutant thereof and a recombinant DNA for expression or a DNA for genomic, integration each comprising a mevalonate pathway-related gene or a mutant thereof.<br>
61.    (Added)       The method according to claim 59 or 60, wherein the host is Saccharomyces cerevisiae.<br>
62. (Added) The method according to claim 61, wherein the Saccharomyces cerevisiae is Saccharomyces cerevisiae A451 strain, YPH499 strain, YPH500 strain, W303-1A strain or W303-1B strain, or a strain derived from any one of said strains.<br>
63.    (Added)        The  method  according  to  claim  59  or 60,  wherein  the mevalonate pathway-related gene is a hydroxymethylglutaryl-CoA reductase gene.<br>
64.    (Added)	The    method    according    to    claim    63,    wherein    the<br>
hydroxymethylglutaryl-CoA reductase gene is HMGl gene.<br>
65.    (Added)	The method according to claim 59 or 60, wherein the prenyl<br>
diphosphate synthase gene is any one selected from the group consisting of the following genes (a) and (b) and fusion genes (c) and (d):<br>
(a)	farnesyl diphosphate synthase gene or a mutant thereof<br>
(b)	geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(c)	a fusion gene composed of farnesyl diphosphate synthase gene or a mutant thereof and geranylgeranyl diphosphate synthase gene or a mutant thereof<br>
(d)	the above gene (a) or (b) or the fusion gene (c) to which a nucleotide sequence encoding an amino acid sequence of His Asp Glu Leu is added.<br><br>
66.    (Added)        The    method    according    to    claim    48,    wherein    the microorganism is a prototroph.<br>
67.    (Added)        The    method    according    to    claim    48,    wherein    the microorganism is a diploid cell.<br>
68.    (Added)        The    method    according    to    claim    48,    wherein    the microorganism is a prototroph and a diploid cell.<br>
69.    (Added)       The method according to claim 48, wherein the pH of the medium is controlled.<br>
70. (Added) The method according to claim 69, wherein the pH control is carried out using ammonium gas, an ammonium salt solution, a sodium hydroxide solution or sulfuric acid.<br><br>
71.	A method of producing a prenyl alcohol substantially as herein described with reference to the accompanying drawings.<br>
72.	A method of producing geranylgeraniol substantially as herein described with reference to the accompanying drawings.<br>
Dated this 23 day of July 2003<br><br></fermenter></operational></fermenter></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MS1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1141-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="202035-a-process-for-producing-acrylamide.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="202037-a-device-of-push-button-gear-and-clutch-actuation-by-vacuum-power-for-single-cylinder-engined-two-and-three-wheeler-vehicles.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>202036</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1141/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td></td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TOYOTA JIDOSHA KABUSHIKI KAISHA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 TOYOTA-CHO TOYATA-SHI AICHI-KEN 471-8571</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OHTO, CHIKARA</td>
											<td>C/O TOYOTA JIDOSHA KABUSHIKI KAISHA 1,TOYOTA-CHO TOYOTA-SHI AICHI 471-8571</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NISHI, KIYOHIKO</td>
											<td>C/O KYUSHU PLANT AJINOMOTO CO INC 450 IAZA-MORODOMITSU MORODOMI CHO SAGA-GUN SAGA 840 2193</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TOTSUKA, KAZUHIKO</td>
											<td>C/O AJINOMOTO CO INC 15-1 KYOBASHI 1-CHOME CHUO-KU TOKYO 104-8315</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/52</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP01/11214</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-12-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-403067</td>
									<td>2000-12-28</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/202036-methods-of-producing-prenyl-alcohols by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:59:00 GMT -->
</html>
