<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/202579-a-method-for-expressing-a-gene-in-bacterial-cell-a-system-for-expressing-a-protein-of-interest-and-a-method-for-producing-recombinant-proteins-and-immunogenic-compositions-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:53:59 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 202579:&quot;A METHOD FOR EXPRESSING A GENE IN BACTERIAL CELL ,A SYSTEM FOR EXPRESSING A PROTEIN OF INTEREST ,AND A METHOD FOR PRODUCING RECOMBINANT PROTEINS AND IMMUNOGENIC COMPOSITIONS THEREOF &quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR EXPRESSING A GENE IN BACTERIAL CELL ,A SYSTEM FOR EXPRESSING A PROTEIN OF INTEREST ,AND A METHOD FOR PRODUCING RECOMBINANT PROTEINS AND IMMUNOGENIC COMPOSITIONS THEREOF &quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The disclosure below provides A protein export system for efficiently producing recombinant protein from a host cell. In a preferred embodiment, the protein export system utilizes protein export machinery endogenous to the host bactenum into which the proteinexport system vector is introduced.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>"A METHOD FOR EXPRESSING A GENE IN A BACTERIAL CELL  A SYSTEM FOR EXPRESSING A PROTEIN OF INTEREST, AND A METHOD FOR PRODUCTING A RECOMBINANT PROTEINS AND IMMTNOGENIC COMPOSITION  THEREOF"<br>
Government Support<br>
[0001]	The protein export system defined herein was developed through support from grants 5 R01<br>
AI29471, RO1 AI40297, and Research Contract N01 A145251 (M. M. Levine, Principal Investigator), from the National Institutes of Health.<br>
Related Applications<br>
[0002]	This application claims priority to U.S. Provisional Patent Application No. 60/252,516, filed<br>
November 22,2000, which is hereby incorporated by reference in its entirety.<br>
Background of the Invention Field of the Invention<br>
[0003]	The disclosure below relate to the use of a protein export system.   The disclosed system<br>
provides effective methods and compositions useful for the production of recombinant proteins. Description of the Related Art<br>
[0004]	Protein expression systems have long used high copy number expression plasmids or expression<br>
vectors in an attempt to increase yields of recombinant proteins of interest. High copy number expression plasmids and the proteins of interest they encode can exert a negative effect on the fitness of a host containing an expression plasmid. The notable burden placed upon prokaryotic host cell;; carrying multicopy plasmids is the cumulative result of a metabolic cascade triggered by two processes: 1) the replication and maintenance of expression plasmids and 2) transcription and translation of the various plasmidencoded functions including the gene of interest. Such mechanisms could explain the observation that plasmid-bearing bacteria grow slower than plasmid-less bacteria. This burden can also explain the observation that growth rate decreases as copy number increases.<br>
[0005]	As the gene of interest is expressed, the growth rate of the recombinant host cell decreases.<br>
The decrease in growth rate may trigger the induction of various cellular proteases that can degrade recombinantly produced protein present in cytoplasm of the host cell. Reduced growth rate is therefore the inevitable consequence of metabolic burden, which in turn is the cumulative result of a number of physiological perturbations. Because this reduction in the growth rate creates a selective pressure for loss of resident plasmids in the absence of selection, significant loss of expression plasmids from the host cell carrying an expression vector may occur after transformation of the host cell.       <br>
[0006]	Host cells with reduced growth rates can spontaneously shed an expression plasmid to remove<br>
from the host celt an unnecessary metabolic burden and allow plasmid-less host cells to quickly outgrow the population<br><br>
of plasmid bearing host cells. Such s shift in protein expression within a population of host cells would be expected to reduce the protein production.<br>
[0007] Accordingly, it would be desirable to prepare a protein expression system that would optimize protein expression from the expression vector while minimizing the metabolic burden on the host cell generated by the expression vector.<br>
Summary of the Invention<br>
[0008] The disclosed material relates to the use of an export protein to facilitate export of a fusion protein out of a host cell. One disclosed embodiment provides a method for expressing a gene in a bacterial cell comprising providing an expression vector to a population of untransformed bacterial host cells, wherein the expression vector comprises an expression cassette comprising an export protein coding sequence genetically fused to a protein of interest coding sequence, expressing the expression cassette such that an export protein-protein of interest fusion protein is produced and exported or transported into the culture medium.<br>
[0009]	Another disclosed embodiment relates to a method for eliciting an immune response from an<br>
animal comprising providing to an animal a population of bacterial host cells transformed with an expression vector which comprises an expression cassette comprising an export protein coding sequence genetically fused to a protein of interest coding sequence, expressing the expression cassette such that an export protein::protein of interest fusion protein is produced and exported or transported into the animal, and eliciting an immune response from the animal against the fusion protein.<br>
[00010]	Another disclosed embodiment relates to a system for expressing a protein of interest<br>
comprising: an expression vector comprising an expression cassette, wherein the expression cassette comprises an export protein coding sequence genetically fused to a protein of interest coding sequence, a host cell transformed with the expression vector, and a culturing environment for the transformed host cell, wherein the expression cassette<br>
expresses an export protein:: protein of interest fusion protein, which is exported out of the transformed host cell.<br><br>
Brief Description of the accompanying Drawings<br><br>
	[0011]	Figure 1 provides examples of the expression vector of this invention.   Figure 1A illustrates<br>
pSEC84 expression vector. Figure IB illustrates pSEC84bla expression vector. Figure 1C illustrates pSEC84secB.<br>
Figure 1D illustrates pSEC84glpuv. 	<br>
[0012]	Figure 2 illustrates exportation of ClyA-SacB protein fusion which results in the metabolisis of<br>
sucrose in solid growth medium.  The strains were grown on media containing either 8% sucrose (2A and 2B), 16%<br>
sucrose (2C and 2D), or 8% sucrose +8% Larabinose (2E and 2F). Figures 2A, 2C, and 2E demonstrate the growth of<br>
CVD 908-htrA expressing ClyA. Figures 2B, 2D, and 2F demonstrate the growth of CVD 908-htrA expressing ClyA-SacB.<br>
[0013]	Figure 3 illustrates the growth of CVD 908-htrA expressing either ClyA (pSEC84) or ClyA-SacB<br>
(pSEC84sacB), grown in 2XLB50 broth supplemented with DHB and either 10% sucrose or 10% glucose.<br>
-2-<br><br>
[0014] Figure 4 illustrates Western immunoblot analysis of bacterial ceil fractions from either CVO 908-htrA (lanes 1-3) or CVD 908-htrA(pSEC84gfuv) (lanes 4-8). Cell fractions are loaded as follows: supematants, lanes 1 end 4; cytoplasmic, lanes 2 and 6; periplasmic, lane 5; insoluble, lane 7; whole cell, lanes 3 and 8; and 50 ng GFPuv, lane 9. Membranes with identical samples were probed with antibodies specific for GFPuv (panel A) or E coli GroEL (panel B).<br>
Detailed Description of the Preferred Embodiment<br>
[0015]	The disclosure below provides a protein export system for efficiently producing recombinant<br>
protein from a bacterial host. In a preferred embodiment, the protein export system utilizes protein export machinery endogenous to the host bacterium into which the protein export system vector is introduced.<br>
[0016] The protein export system has a number of useful applications. The system can be used to efficiently produce recombinant proteins of interest inside a bacterial host cell and export the recombinant protein of interest from the bacterial host cell. For example, the disclosed system can be used to efficiently produce recombinant proteins of interest in a bioreactor.<br>
|0017| The protein export system can be also be used to provide to an animal antigenic material against which an immune response may be mounted. For example, in one embodiment an attenuated bacterium, such as Salmonella, is transformed with the components of the protein export system. The recombinant Salmonella can then be used as a live vector immunogenic composition capable of facilitating the generation of an immune response in an animal. The protein export system can be used with a variety of antigens of interest. Specific embodiments include immunogenic compositions directed against typhoid fever and other diseases. Immunogenic compositions expressing antigens that are exported from recombinant bacteria with a minimum of bacterial lysis are also disclosed. A.         HlyE Family Protein Export System<br>
[0018]	The disclosure below relates to the use of members of the HlyE family in a protein export system<br>
to facilitate protein expression. Members of the HlyE family can be used to facilitate the export of recombinantly produced proteins from their bacterial hosts. Expression systems that export recombinantly produced proteins are believed to facilitate increased protein production. The disclosed protein export system can also be used to prepare immunogenic compositions with which to vaccinate animals.<br>
[0019]	Growth rates of recombment organisms containing expression vectors have been observed to<br>
decrease as the level of expression of a gene of interest increases. The decrease in growth may trigger the induction of various cellular proteases that can degrade the expressed recombinant protein. Reduced growth rate is therefore the inevitable consequence of metabolic burden, which, in turn, is the cumulative result of a number of physiological perturbations, for example, physiological perturbations result from the expression and accumulation of the protein of interest inside the host bacterium. This accumulation can be harmful to the viability of the host bacterium and thus a negative selection pressure.<br>
-3-<br><br><br>
[0020] Because metabolic burdens such as those discussed above create a selective pressure for loss of resident expression vectors in the absence of selection, significant toss of expression vector from the host bacterium can occur after the host bacterium has been transformed with the expression vector containing the gene of interest Spontaneous plasmid loss removes any metabolic burden from the host bacteria and allows plasmid less bacteria to quickly outgrow the population of plasmid-bearing bacteria. The overgrowth of bacterial cells that do not contain the expression vector and thus do not express the protein of interest reduces overall protein production levels. Therefore, host bacteria that are not genetically constrained to maintain expression vectors directing the synthesis of high levels of a given protein of interest may produce significantly less protein.<br>
[0021]	A preferred embodiment for exporting the recombinantiy expressed protein of interest comprises<br>
exploiting an endogenous export system in the host bacteria containing the expression vector. Exploitation of an endogenous export system is advantageous in part because it avoids the need for large amounts of heterologous DNA encoding exotic proteins to supply an exogenous export system. Nevertheless, protein export systems utilizing exogenous expert systems are also encompassed by the present disclosure.<br>
[0022]	An attractive endogenous export system candidate is the cryptic hemolysin encoded by Cytotysin<br>
A (clyA) within the chromosome of Salmonella entsrica serovar Typhi (hereinafter "S. Typhi"), a member of the HlyE<br>
family of proteins. The HlyE family consists of a single protein, HlyE, and its close homologs from E coll. Shigella<br>
flexneri and S. Typhi, and other bacteria. The E. coli protein is a functionally well characterized, pore-forming,<br>
chromosomally-encoded hemolysin also called ClyA, HlyE. and silent hemolysin A (SheA). It consist of 303 amino acid<br>
residues (34 kDa). Its transcription is positively controlled by SlyA, a regulator found in several enteric bacteria. HlyE<br>
forms stable, moderately cation-selective transmsmbrane pores with a diameter of 2.5-3.0 nm in lipid bilayers. The<br>
protein binds cholesterol, and pore formation in a membrane is stimulated if the membrane contains cholesterol. The<br>
crystal structure off. coil HlyE has been solved to 2.0A resolution, and visualization of the lipid-associated form of the<br>
toxin at low resolution has been achieved by electron microscopy. The structure exhibits an elaborate helical bundle<br>
some 100A long. It oligomerizes in the presence of lipid to form transmembrane pores.<br>
B.	Cytolysin A (CIvA) Protein Export System<br>
[0023]	A preferred embodiment of the present disclosure relates to the use of the ClyA protein, a<br>
member of the HlyE family, in a protein export system. An approximately 1 kb clyA gene was cloned from S. Typhi CVD 908-htrA for use in a protein export system. The Cly A protein is exported from both E. coli and S. Typhi and is capable of exporting passenger proteins that have been genetically fused to the 3'-terminus of the clyA open reading frame. Passenger protein refered to herein is also referred to as a protein of interest. It is demonstrated that the proper folding of these fusion proteins occurs such that the inherent biological activity of the domains involved is observed.<br>
[0024]	Cytolysin A (ClyA) from S. Typhi was first described by Wallace et al who also reported the<br>
crystal structure for the homologous hemolysin from E. coli. (Wallace, A. J., T. J. Stiltman, A. Atkins, S. J. Jamieson, P.<br>
-4-<br><br>
A. Butlough, J. Green, and P. J. Artymiuk. 2000. E. coil hemolysin E (HlyE, ClyA, SheA): X-ray crystal structure of the toxin and observation of membrane pores by electron microscopy. Cell 100:265-276.) This hemolysin has been described previously and variously referred to as ClyA. HlyE, or SheA. To avoid confusion, the E. coli hemolysin is referred to herein as HlyE and is encoded by hlyE. Also for clarity, the S. Typhi hemorysin is referred to herein as ClyA, which is encoded by clyA.<br>
[0025] For illustrative purposes, the protein structure of the HlyE family members is discussed referring to the E.  coli protein HlyE. HlyE is a kinked rod-shaped molecule with a hydrophobic 27 residue transmembrane region. This region comprises one terminus of the folded molecule and is proposed to form a pore within a target membrane. The formation of the pore ultimately leads to lysis of the target cell. In elegant electron microscopy studies, Wallace et al. showed that HlyE inserts into lipid vesicles to form pores comprised of 8 HlyE monomers.<br>
[0028] Although the pore formation facilitated by HtyE has been elucidated, the mechanism by which HlyE and HlyE homologs are exported out of a bacterium remains unclear. Moreover, the manner by which the hemorysin inserts into target membranes for assembly into pores is also not well understood. Del Castillo et al., described the growth-phase dependent secretion of hemolytic activity which peaked during mid-log phase and vanished at the onset of stationary phase, (del Castillo, F. J., S. C. Leal, F. Moreno, and I. del Castillo. 1997. The Escherichia coli K-12 sheA gene encodes a 34-kDa secreted haemolysin. Mol. Microbiol. 25:107-115.) Ludwing and colleagues have reported that secretion of this cryptic hemorysin is accompanied by leakage of periptasmically confined proteins, but is not accompanied by loss of cytoplasmic proteins, arguing against outright cell lysis to release HlyE. (Ludwig, A., S. Bauer, R. Benz, B. Bergmann, and W. Goebel. 1999. Analysis of the SlyA-controlled expression, subcellular localization and pore-forming activity of a 34 kDa haemolysin (ClyA) from Eschen'chia coB K-1Z Mol. Microbiol. 31:557-567.)<br>
[0027] In addition, when compared to the sequence encoded by hfyE, N-terminal sequencing of secreted HlyE revealed that HlyE is not N-terminally processed during transport. Oscarsson et al., reported that HlyE binds to cholesterol and that the presence of cholesterol in target membranes stimulates pore formation and lysis. (Oscarsson, J., Y. Mizunoe, L. Li, X. Lai, A. Wieslander, and B. E. Uhlin. 1999. Molecular analysis of the cytolytic protein ClyA (SheA) from Eschen'chia coli. Mol. Microbiol. 32:1226-1238.) It is estimated that -103 molecules of HlyE are required for lysis of a target erythrocyte suggesting significant accumulation of HlyE prior to detection of cell lysis. HlyE is remarkably stable within a range of pH values between 3.0 and 9.0, and is resistant to cleavage by proteases including trypsin and pepsin. (Atkins, A., N. R. Wyborn, A. J. Wallace, T. J. Stillman, L K. Black, A. B. Fielding, M. Hisakado, P. J. Artymiuk. and J. Green. 2000. Structure-function relationships of a novel bacterial toxin, hemolysin E. The role of G. J. Biol. Chem. 275:4115041155.)<br>
[0028]	The nucleotide and amino acid sequence for the isolated clyA gene and ClyA protein used in the<br>
disclosed work are provided as SEQ ID NO:21 and SEQ ID NO:2, respectively.  Other HlyE family members are also available and known to those of ordinary s kill in the art. For example, another S. Typhi clyA gene for cytolysin A is<br>
-5-<br><br>
available under GEN BANK Accession No. AI313034 (SEQ ID N0:22); Salmonella paratyphi clyA gene sequence for cytolysin A is available under GENBANK Accession No. AJ313O33 (SEQ ID N0:23); the Shigella flexneritruncated HlyE (hlyE) gene's complete coding sequence is available under GENBANK Accession No. AF200955 (SEQ ID N0;24); and the Escherichia coli hlyE gene sequence is available under GENBANK Accession No. AJOO1829 (SEQ ID NO:25).<br>
[0029]	The HlyE family of proteins typically cause hemolysis in target cells.  Hemolytically active or<br>
inactive HlyE family members can both be used with the disclosed teachings. For example, it is known that mutation of the htrE gene can reduce or eliminate hemolytic activity. For example, loss of hemolytic activity has been reported when<br>
hlyE is mutated such that amino add substitutions occur at positions 180, 185, 187, and 193.  Specifically, G180V, <br>
* V185S, A187S, and 11935 result in a loss of hemolytic activity from a HlyE protein expressed from a mutated hlyE gene.<br>
[0030]	The present disclosure utilizes the export characteristics of the HlyE family of proteins to<br>
produce a protein export system. For example, fusion proteins comprising any mermber of the HlyE family and a protein of interest are disclosed. More specifically, fusion proteins comprising ClyA and a protein of interest are disclosed. As discussed below, ClyA-containing fusion proteins are exported from the bacteria! host cell and into the surrounding medium. This feature of the expression system comprising an export protein::protein of interest fusion protein component which facilitates production of the protein of interest and exportation of the export protein::protein of interest fusion protein.<br>
Export Protein Expression Vectors<br>
(9031} The protein export system described herein can be used to express and export a wide variety of fusion proteins comprising an export protein and a protion of interest. The export protein is selected from the HlyE family of proteins. In one embodiment, the protein of interest is encoded by a gens of interest. The gene of interest can be foreign to the bacteria containing the protein export system or it can be a gene that is endogenous to the bacteria. Typically, an export protein-protein of interest fusion protein construct is present in in expression cassette, which in turn is present in an expression vector. Each of these units are discussed below. Expression Vectors<br>
[0032] The protein export system utilizes an expression vector to facilitate the recombinant production of the protein of interest. Typically the expression vector will comprise an origin of replication and other structural features that control and regulate the maintenance of the expression vector in the host cell. By definition, the term "expression vector" refers to a plasmid, virus or other vehicle town in the art that has been manipulated by insertion or incorporation of the expression cassette comprising the export protein-protein of interest fusion protein expression cassette. An example of an expression vector system which teaches expression vectors that confer plasmid stability at two independent levels as descried in Galen, et el, Immun. 87:8424-6433 (1999) and in U.S. Patent Appl, Nos. 09/204,117, filed December 2, 1998 and 09/453,313, filed December 2, 1999, which are hereby incorporated by reference to their entirety.<br>
-6-<br><br>
Export Protein-Fusion Protein Expression Cassettes<br>
[0033] The protein export system described herein can be used to express and export a wide variety of fusion proteins comprising an export protein and a protein of interest. The protein of interest is encoded by the protein of interest coding sequence which is also the gene of interest. The gene of interest can be foreign to the bacteria containing the protein export system or it can be a gene that is endogenous to the bacteria. The protein of interest can range from a single amino acid to proteins several times the size of the export protein molecule. More preferably, the protein of interest can range from ten amino acids to two times the size of the export protein. It is preferrable that the size of the protein of interest be such that it not interfere with the ability of the export protein to be exported entirely out of the bacterium. Exemplary proteins of interest are from 0 kDa to at least 50 kDa in mass. Greater masses', and thus longer proteins may also be used as proteins of interest For example, the proteins of interest may have a mass of 55 kDa, 60 kOa, 65 kDa, 70 kOa, 75 kDa, 80 kDa. 85 kDa, 90 kDa, 95 kOa, 100 kDa, or larger.<br>
[0034]	Alternatively, the protein of interest can consist of 1 to 1000 amino acids, or more.   For<br>
example, the protein of interest may have 10,20, 30,40,50,60,70, 80,90,100.150.200, 250,300, 350,400,450, 500,550,600,650,700,750.800,850,900,950,11000 amino acids, or more,<br>
[0035]	Typically, the gene of interest to be expressed is present in an expression cassette.   An<br>
expression cassette will typically contain suitable structural features, such as a promoter, terminator, etc., to permit transcription of the gene of interest.<br>
[0036]	Potynucleotide sequences encoding an export protein::protein of interest fusion protein (also<br>
known as "export proteim:protein of interest fusion protein coding sequences") can be operatively linked to expression control sequences to form an expression cassette. The term "operatively linked" refers to a juxtaposition wherein the components so described are in a relationship permitting them to function in their intended manner. An expression control sequence operatively linked to a coding sequence is ligated such that expression of the coding sequence is achieved under conditions compatible with the expression control sequences. As used herein, the term "expression control sequences" refers to nucleic acid sequences that regulate the expression of a nucleic acid sequence to which it is operatively linked. Expression control sequences are operatively linked to a nucleic acid sequence when the expression control sequences control and regulate the transcription and, as appropriate, translation of the nucleic acid sequence. Thus expression control sequences can include appropriate promoters, transcription terminators, optimized ribosome binding sequences, a start codon (i.e., ATG) in front of a protein-encoding gene, the correct reading frame of that gene to permit proper translation of mRNA, and stop codons. The term "control sequences" is intended to include, at a minimum, components whose presence can influence expression, and can also include additional components whose presence is advantageous, for example, leader sequences . Expression control sequences can include a promoter.<br>
[0037]	A "promoter" is the minimal sequence sufficient to direct transcription.  Also included in the<br>
invention are those promoter elements which are sufficient to render promoter-dependent gene expression controllable for<br>
-7-<br><br>
cell-type specific, tissue-specific, or inducibte by external signals or agents; such elements may be located in the 5' or 3' regions of the export protein-protein of interest fusion protein coding sequence. Both constitutive and inducible promoters are useful with the disclosed methods. The expression of export protein::protein of interest fusion protein coding sequences can be driven by a number of promoters. Although the endogenous promoter of an export protein can be utilized for transcriptional regulation of the expression cassette, preferably, the promoter is a foreign regulatory sequence. An example of an inducible endogenous promoter is the ompC promoter which can be used to drive transcription of the expression cassette.<br>
[0038] Promoters useful in the invention include both constitutive and inducible natural promoters as well as engineered promoters. A preferred inducible promoter should 1) provide low expression in the absence of the inducer; 2) provide high expression in the presence of the tnducer; 3} use an induction scheme that does not interfere with the normal physiology of the host ceil; and 4) havn little or no effect on the expression of other genes. Examples of inducible promoters include those induced by chemical means. Those of skill in the art wilt know other promoters, both constitutive and inducible.<br>
[0039]	The particular promoter selected should be capable of causing sufficient expression to result in<br>
the production of an effective amount of the export protein-protein of interest fusion protein. The effective amount of export protein:protein of interest fusion protein can vary depending on the goal of the expression. The promoters used in the vector constructs of the present disclosure can be modified, if desired, to affect their control characteristics.<br>
[0040]	The export protein::protein of interest fusion protein comprising the export protein and the protein<br>
of interest can further comprise purification tags engineered into the expression cassette to be expressed as a part of the export protein-protein of interest fusion protein. The tag is chosen to facilitate purification of the export protein::protein of interest fusion protein and/or the protein of interested produced by the described methods. For example, a plurality of histidine residues can be engineered into the C-terminal portion or N-terminal portion of the protein of intereset to facilitate protein purification. It is preferable that the introduction of the tag minimizes improper folding of the protein of interest.<br>
[0041]	In addition to the polyhistidine tag, there are a number of other protein tags that can be used to<br>
facilitate protein purification. For example, antigenic tags such as the maltose binding protein tag, a c-myc epitope tag, a green fluorescent protein tag, a luciferace tag, a beta galactosidase tag, a polyhistidine tag, or any other suitable protein expression tag that can be used with the described system.<br>
[0042|	The export protein::protein of interest fusion protein comprising the export protein and the protein<br>
of interest can further comprise additional features to facilitate the use of the expressed and exported protein. For example, protease recognition sites can be engineered between various components of export protein:protein of interest fusion protein, including, if applicable, the tags described above, to promote the separation of the components of the export protein::protein of interest fusion protein. For example, a protease recognition site can be introduced between the<br>
-8-<br><br>
export protein and protein of interest sequences in the expression cassette. Also a protease recognition site can be introduced between the tag and the protein of interest sequences in the expression cassette. These protease recognition sites facilitate the separation of the export protein from the protein of interest.<br>
[0043] Optionally, a selectable marker may be associated with the expression cassette. As used herein, the term "marker" refers to a gene encoding a trait or a phenotype that permits the selection of, or the screening for, a host cell containing the marker. The marker gene may be an antibiotic resistance gene whereby the appropriate antibiotic can be used to select for transformed host cells from among cells that are not transformed or the marker gene may be some other drug resistance gene. Examples of suitable selectable markers include adenosine deaminase, dihydrofolate reductase, hygromycin-B-phosphotransferase, thymidine kinase, xanthine-guanine phospho-ribosyttransferase, gtyphosphate and glufostnate resistance and amino-glycoside 3'-0-phosphotransferase II (kanamycin, neomycin and G418 resistance). Those of skill in the art will know other suitable markers that can be employed with the disclosed teachings.<br>
[0044] An example of an expression vector is shown in Figure 1. In Figure 1A, the pSEC84 expression vector is shown. The nucleotide sequence of the pSEC84 vector can be found at SEQ ID NO:3. The amino acid sequence of ClyA encoded by the clyA gene is found at SEQ ID NQ:2.<br>
[0045]	Each vector shown in Figures IAD comprises a promoter [PmvC - a modified osmotically<br>
controlled ompC promoter from E. coli. an export protein (clyA). an origin of replication, a transcriptional terminator (T1), a passive partitioning function [par), resistance to kanamycin laph), a post-segregational killing system (hok-sok), and an active partitioning system (parA). It should be noted that these vector components are merely exemplary of a single embodiment of the disclosed system.<br>
[0046]	Figure 18 illustrates the pSEC84bla expression vector. This expression vector contains the same<br>
features as the pSEC84 vector and further comprises a export protein-protein of interest fusion protein construct. Specifically, the bla gene encoding B-lactamase was cloned into the pSEC84 vector at the Nhe I site at position 1426 of the parent vector. Other fusion constructs are shown in Figure 1C (pSEC84SacB) and Figure 1D (pSEC84gfpuv) Genes of Interest<br>
[0047] The protein export system disclosed herein can be used with a variety of genes of interest. In one embodiment, the gene of interest encodes a desired protein. Any protein amenable to recombinant bacterial expression can be used with the disclosed export system. The gene of interest can encode for any polypeptide such as. for example, a mammalian polypeptide such as an enzyme, an enzyme inhibitor, a hormone, a lymphokine, a plasminogen activator, or any other protein of interest. The gene of interest can encode a eucaryotic gene, a procaryotic gene, a plant gene, or viral gene of interest.<br>
[0048]	One advantage of the disclosed system is that it provides a method by which proteins that were<br>
toxic to a host bacterium can now be expressed. For example, recombhiant expression of certain proteins is complicated or impossible when the expressed protein is not exported from the host bacterial cell. With the methods disclosed herein,<br>
-9-<br><br>
one of ordinary skill in the art could express a previously unexpressible or underexpressed protein to produce the desired protein in usable quantities.<br>
[0049] In another embodiment, the gene of interest is an immunogenie antigen-encoding gene, and the protein of interest is an antigen which may be a protein or antigenic fragment thereof from any pathogen, such as viral pathogens, bacterial pathogens, and parasitic pathogens. Alternatively, the gene of interest may be a synthetic gene, constructed using recombmant DMA methods, which encode antigens or parts thereof from viral, bacterial, parasitic pathogens, or another antigen of interest. These pathogens can be infectious in humans, domestic animals or wild animal hosts.<br>
[0050] Examples of particular viral pathogens, from which the viral antigens are derived, include, but are not limited to, Orthomyxoviruses, such as intluenza virus; Retroviruses, such as Rous sarcoma virus (RSV) and simian immunodeficiency virus (SIV). Herpesviruses, such as Epstein Barr virus (EBV); cytomegalovirus (CMV) or herpes simplex virus; Lentiviruses, such as human immunodeficiency virus; Rhabdoviruses, such as rabies; Picomoviruses, such as poliovirus; Poxviruses, such as vaccinia; Rotavirus; and Parvoviruses.<br>
[0051]	Examples of immunogenic antigens from viral pathogens include the human immunodeficiency<br>
virus antigens Nef, p24, gp120, np41. Tat, Rev, and Pol Additional examples of antigens include the T cell and B cell epitopes of gp120, the hepatitis B surface antigen, rotavirus antigens, such as VP4, VP6, and VP7, influenza virus antigens such as hemagglutinin or nucleoprotein, and herpes simplex virus thymidine kinase. The nucleic acid and amino acid sequences for each of these virus antigens are well known in the art and readily available.<br>
[0052]	Bacterial pathogens, from which the bacterial antigens can be derived, include, but are not<br>
limited to, Mycobacterium spp.. Helicobacter pylori, Salmonella spp., Shigella spp., E coli, Rickettsia spp., Listeria spp., Legionella pneumoniae, Pseudomonas spp., Vibrio spp., and Borellia burgdorferi.<br>
[53]	Examples of immunogenic antigens of bacterial pathogens include, but are not limited to, the<br>
Shigella sonnei form 1 antigen, the O-antigen of V. cholerae Inaba strain 569B, immunogenic antigens of enterotoxigenic<br>
E. co/i, such as the CFA/I timbrial antigen, and the nontoxic B-subunit of the heat-labile toxin, pertactin of Bordetella<br>
pertussis, adenylate cyclase hemolysin of B. pertussis, and fragment C of tetanus toxin of Cfostridium tetani.<br>
[54]	Examples of immunogenic antigens of parasitic pathogens, from which the parasitic antigens can<br>
be derived, include, but are not' limited to, Plasmodium spp., Trypanosome spp., Giardia spp., Boophilus spp., Babesla<br>
spp., Entamoeba spp., Eimeria spp., Leishmania spp., Schistosome spp., Brugia spp., Fascida spp., Dirofiiaria spp.,<br>
Wuchereria spp., and Onchocerea spp.<br>
[0055]	Examples of immunogenic antigens of parasitic pathogens include, but are not limited to, the<br>
circumsporozoite antigens of Plasmodium spp., such as the circumsporozoite antigen of P. bergerii or the circumsporazoite antigen of P. falciparum; the merozoite surface antigen of Plasmodium spp.; the galactose specific lectin of Entamoeba histolytica, gp63 of Leishmania spp., paramyosin of Brugia malayi, the triosephosphate isomerase<br>
-10-<br><br>
of Schistosoma mansoni, the secreted globin-Iike protein of Trichostrongylus colubriformis; the glutathione-S-transferase of frascio/a hepatica. Schistosoma bovis and S. japonicum, and KLH of Schistosoma bovis and S.japonicum.<br>
[0056] In another embodiment, the gene of interest can encode a therapeutic agent, such as, but not limited to, tumor-specific, transplant, or autoimmune antigens or parts thereof. Alternatively, the gene of interest can encode synthetic genes, which encode for tumor-specific, transplant, or autoimmune antigens or parts thereof.<br>
[0057] Examples of tumor specific antigens include prostate specific antigen. TAG-72 and CEA, MA6E-1 and tyrosinase. Recently it has been shown in mice that immunization with non-malignant cells expressing a tumor antigen provides a vaccine-type effect, and also helps the animal mount an immune response to clear malignant tumor cells displaying the same antigen.<br>
[0058]	Examples of transplant antigens include the CD3 receptor on T cells.   Treatment with an<br>
antibody to CD3 receptor has been shown to rapidly dear circulating T cells and reverse most rejection episodes.<br>
[0059]	Examples of autoimmune antigens include IAS   chain. Vaccination of mice with an 18 amino<br>
acid peptide from IAS   chain has been demonstrated to provide protection and treatment to mice with experimental autoimmune encephalomyelitis.<br>
[0060]	Alternatively,   the  gene  of  interest   can   encode  rnimunoregulatory  molecules.     These<br>
immunoregulatory molecules include, but are not limited to, growth factors, such as M-CSF, GM-CSF; and cytokines, such as IL-2,114, IL 5,IL-6, IL-10, IL-12 or IFN-gamma. Recently, localized delivery of cytokines to tumor tissue has been shown to stimulate potent systemic immunity and enhanced tumor antigen presentation without producing a systemic cytokine toxicity. Stabilized Plasmid-based Expression Systems<br>
[0061]	Bacterial expression systems, by design, typically utilize expression vectors to harness and<br>
exploit the protein synthesis machinery of a bacterial host cell to produce a protein of interest. Protein expression levels can often be increased by using high copy number plasmids, or high copy number expression vectors, with the host cells. As discussed above, the introduction of a high copy number expression vector into a bacterial host cell, however, places certain metabolic stresses on the host cell that can cause the host cell to expel the expression vector and thus reduce protein expression levels.<br>
[0062]	Often overlooked in expression vector engineering is the effect high copy number expression<br>
vectors frequently exert on the fitness of the host tell in which the expression vector is introduced. The burden placed upon host bacterial cells carrying multicopy plasmids is the cumulative result of a metabolic cascade. The cascade is triggered by the replication and maintenance of expression vectors (see Bailey, J. E., Host-vector interactions in Escherichia coli, p. 29-77. In A. Fiechter (el), Advances in Biochemical Engineering. Biotechnology. Springer-Verlag, Berlin (1993), Glick, B. R.. Biotechnol. Adv. 13:247 261 (1995), and Smith &amp; Bidochka. Can. J. Microbiol. 44:351-355 (1998)).   The cascade is also triggered by transcription and translation of the various expression vector-encoded<br>
-11-<br><br>
functions, including the protein of interest. Mechanisms such as those described above explain the observation that plasmid-bearing bacteria grow slower than plasmid-less bacteria. These mechanisms can also explain the observation that growth rate decreases as copy number increase;;.<br>
[0063] Growth rates of recombinant organisms containing expression vectors have been observed to decrease as the expression of a gene of interest increases. The decrease in growth may trigger the induction of various cellular proteases that can degrade the expressed recombinant protein of interest. Reduced growth rate is therefore the inevitable consequence of metabolic burden, which in turn is the cumulative result of a number of physiological perturbations. For example, physiological perturbations result from the expression and accumulation of the protein of interest inside the host bacterium. This accumulation can be harmful to the viability of the host organism and thus a negative selection pressure.<br>
[0064] Because metabolic burdens such as those discussed above create a selective pressure for loss of resident expression vectors in the absence of selection, significant loss of expression vectors from the host cell can occur after the host cell has been transformed with the expression vector containing the gene of interest. Spontaneous plasmid loss removes any metabolic burden from the host cell and allow plasmid-less host cell to quickly outgrow the population of plasmid-bearing host cell. The overgrowth of host cells that do not contain and thus do not express the protein of interest reduces overall protein production levels. Therefore, host cells that are not genetically constrained to maintain expression vectors directing the synthesis of high levels of a given protein of interest may produce significantly less protein.<br>
[0065]	There are a number of means by which this metabolic stress can be reduced.   Controlled<br>
expression of a protein of interest from multicopy expression vectors represents one solution for synthesis of high levels of protein of interest within host cells. This solution is one embodiment with which to practice the disclosed methods. Utilization of inducible promoters, for example, is one method by which expression from an expression vector can be controlled. Such inducible promoters are discussed in the expression cassette section of this disclosure.<br>
[0066]	Another embodiment of the methods disclosed herein relates to a plasmid-based expression<br>
system engineered to permit the stable expression of high levels of one or more proteins throughout a growing population of cells. Preferably, a stable expression vector is one that perpetuates the expression vector as the host cell replicates. Expression vectors that confer plasmid stability at two independent levels have recently been described in Galen, et ai, Immun. 67:6424-6433 (1999) and in U.S. Patent Appl. Nos. 09/204,117, filed December 2,1998 and 09/453,313, filed December 2,1999, both of which are hereby incorporated by reference in their entirety.<br>
[0067]	In this embodiment, partition functions can be incorporated into an expression vector to enhance<br>
the inheritance of the plasmid as a given bacterium or host cell grows and subsequently divides. In rare cases where a daughter cell does not inherit at least one copy of the expression vector, a latent post-segregational killing system becomes activated and removes this bacterium or host cell from the growing population through cell lysis.<br>
-12-<br><br>
C.	Bacterial Host Cells<br>
[0068] A number of species of bacteria are suitable for use with the teachings disclosed herein. Preferably, a suitable bacterial species wilt be capable of protein export such that the gene of interest can be suitably transcribed such that the protein of interest is translated and exported out of the bacteria. In one embodiment of the invention, the bacteria is administered to an animal, and thus the protein of interest must be exported out of the bacteria into the animal. Invasive and non-invasive bacteria may be used. Examples of some invasive bacteria include, Shigella spp., Listeria spp., Rickettsia spp., and enteroinvasive Escherichia coli. A preferred embodiment utilizes Salmonella species.<br>
[0069] The particular Salmonella strain employed with the disclosure below is not critical. Examples of Salmonella strains which can be employed in the present invention include Â£ Typhi (ATCC No. 7251) and S. Typhimurium (ATCC No. 13311). Attenuated Salmonella strains are preferably used in the present invention and include S. Typhi arvAaroD (Hone et al, Vacc. 9:810-816 (1991)) and S. Typhimurium aroA mutant (Mastroeni et al. Micro. Pathol., 13:477491 (1992))). Alternatively, new attenuated Salmonella strains can be constructed by introducing one or more attenuating mutations as described for Salmonella spp. above.<br>
D.	Bioreactors<br>
I0070I	The protein export system described herein is suited for use with bioreactors and similar devices<br>
that facilitate the growth of bacteria and the harvesting or use of a desired product or protein of interest Traditionally there are five stages for recovery of biomolecules from the prior art bioreactors: pre-treatment, solid/liquid separation, concentration, purification, and formulation. There can be a wide range of operations available within each stage. These ranges of operations for each stage are as follows: Pre-treatment: cell disruption, stabilization, sterilization, pasteurization, and ftocculation; Solid/liquid Separation: filtration, sedimentation, and centrifugation; Concentration: membranes, precipitation, evaporation, extraction, and freeze concentration; Purification: precipitation, extraction, diafiltration, adsorption, and chromatography; and Formulation: drying, prilling, extrusion, granulation, and tabletting.<br>
[0071]	In bioreactors where the bacteria do not export the desired product out of the bacteria, one has<br>
to scale up the bacteria, induce the bacteria to produce the desired product and then lyse the bacteria to release the contents. Typically this disruption is performed in the same medium in which the bacteria were grown. One can use a homogenizer or bead mill to mechanically disrupt the bacteria. For non-mechanical disruption, one can use heat shock (which may destroy proteins), detergents, solvent!!, sequestrants, and enzymes. (Krijgsman, "Releases of Intracellular Components", pp. 2742, in Product Recovery in Bioprocess Technology, publisher Butterworth-Heinemann Ltd, Oxford, England, 1992)<br>
[0072]	After the bacteria are disrupted one separates the solid participates from the fluids (solid/liquid<br>
separation). The desired product is usually in the liquid, which one then has to concentrate. Then one extracts the desired product from the concentrated liquid.<br>
-13-<br><br>
[0073]	Factors which affect separation of the desired product from either the undesired solids or liquids<br>
are size, diffusivity, ionic charge, solubility, and density. For size-dependent separation, one can use microfilters, cloth and fiber filters, ultrafHtration, screens/strainers, and gel chromatography. For diffusivity dependent separation, one can use reverse osmosis and dialysis. Ion exchange chromatography is used for ionic charge-dependent separation. To separate the desired product based on its solubility, one can use solvent extractions. For density-dependent separation, one can use ultracentrifuges, centrifuges, and gravity sedimentation. (Krijgsman, "Downstream Processing in Biotechnology", pp. 2-12, in Product Recovery in Bioprocess Technology, publisher Bulterworth-Heinemann Ltd, Oxford, England, 1992).<br>
[0074] One advantage of using the disclosed system is that a population of recombinant bacterial host cells can be transformed with an expression vector comprising the disclosed protein export system and that population of bacterial host cells can be maintained in culture and used to produce protein without having to harvest and lyse the bacterial host cells. The culturing of bacterial host cells and the harvesting of the culture medium containing the recombinantly expressed protein of interest can be performed in any type of bioreactor.<br>
[0075]	There are various types of bioreactors but the family of devices can be divided to two main<br>
categories, "free floating" and "bed" bioreactors. In "free floating" bioreactors, the bacteria are floating freely within the media. Examples of "free floating" bioreactors are conventional stirred tank bioreactors, bubble column, airlift loop, multi-purpose tower bioreactors, liquid impelled loop bioreactors, and pumped tower loop bioreactors. An example of the "bed"-type bioreactor is the packed bed bioreactor. In a "bed"-type bioreactor, the bacteria are attached to beads, a membrane, or other solid support. A hybrid type of bioreactor can be produced using a f luidized bed bioreactor where the bacteria are attached to beads or other support but can float in the media. (Mijnbeek, "The. Conventional Stirrer Tank Reactor" pp. 39-74; Mijnbeek. "Bubble Column, Airlift Reactors, and Other Reactor Designs" pp. 75-114; Geraats, "An Introduction to Immobilized Systems" pp 115-124; all in "Operational Modes of Bioreactors", publisher Butterworth-Heinemann Ltd, Oxford, England, 1992.)<br>
[0076]	Using the protein export system described herein with a "bed" bioreactor avoids the step of pre-<br>
treatment and solid/liquid separation because the desired protein of interest is exported out of the bacteria into the media. One only needs to remove the media from the bed prior to attempting to isolate the desired product. For "free floating" bioreactors, one can centrifuge the liquidfbacteria mixture to pellet the bacteria. Then one removes the liquid containing the desired protein of interest from the pelleted bacteria. Next one isolates the desired protein of interest from the media. A further benefit of the disclosed system is that the media will contain less undesired proteins than are present in media in which bacteria were disrupted; all the intracellular components of the disrupted bacteria are absent from the media in the present invention. Thus purification of the desired protein of interest is easier. Furthermore, having tags and protease cleavage sites present within the export protein::protein of interest fusion protein further facilitate the isolation and purification of the protein of interest.<br>
-14-<br><br>
[0077]	One example of a bioreactor is the apparatus taught in U.S. Patent No. 5.635,368, "Bioreactor<br>
with immobilized lactic acid bacteria and the use thereof," to Lommi, et a!., June 3,1997, which is hereby incorporated by reference in its entirety. The Lommi apparatus relates to a bioreactor with immobilized bacteria, which is characterized in that the bacteria are fixed on the surface of a substantially non-compressible carrier. Another example of a bioreactor is found at U.S. Patent No. 4,910,139, "Method for continuously producing citric acid by dual hollow fiber membrane bioreactor," to Chang, et al., March 20,1990, which is hereby incorporated by reference in its entirety. This invention relates to growing immobilized bacteria to produce citric acid continuously.<br>
[0078]	An additional bioreactor apparatus is disclosed in U.S. Patent No. 5,585,266, "Immobilized cell<br>
bioreactor,** to Plitt, et al., December 17,1996, which is hereby incorporated by reference in its entirety. The disclosed<br>
Plitt device relates to an immobilized cell bioreactor wherein the cells are harbored within or upon an immobilization<br>
matrix including cell support sheets comprised of common textile fabric U.S. Patent Nos. 4,665,027 and 5,512,480,<br>
both of which are incorporated by reference, disclose other bioreactor embodiments.<br>
E.	Vaccines<br>
[0079] The protein export system described herein has utility in the production of vaccines. For example, the production of subunit vaccines can be achieved using the protein export system as the system facilitates recombinant protein harvest and reduces the presence of contaminating proteins from the growth medium in which the recombinant host cells are propagated. Recombinant host cells can also be used to generate immunogenic compositions where the recombinant host cell is provided to a subject and the subject's immune system generates an immune response against the proteins exported from the recombinant host cell.<br>
[0080]	The protein export system described herein can be used with any antigen to prepare a vaccine<br>
therefrom, where the antigen is the protein of interest as described above. Vaccine preparation is generally described in New Trends and Developments in Vaccines, edited by Volter et at.. University Park Press, Baltimore, Md. U.S.A. 1978. Encapsulation within liposomes is described, for example, by Fullerton, U.S. Pat. No. 4,235,877. Conjugation of proteins to macromolecules is disclosed, for example, by Likhite, U.S. Pat. No. 4,372,945 and by Armor et al., U.S. Pat. No. 4,474,757.<br>
[0081] The amount of antigen in each vaccine dose is selected as an amount which induces an immunoprotective response without significant, adverse side effects in typical vaccinees. Such an amount will vary depending on which specific antigens are employed and the delivery technology used (by way of example only, purified proteins or live bacteria). Generally it is expected that doses comprising purified proteins will comprise 1-1000 ?g of total antigen, preferably 2-200 ?g. Generally it is expected that doses comprising live bacteria delivering proteins of interest will comprise 1-1000 ng of total antigen of interest.   An optimal amount for a particular vaccine can be<br>
-15-<br><br>
ascertained by standard studies involving observation of antibody titres and other responses in subjects,  following an initial vaccination, subjects (animal or human) may receive one or more booster doses, for example after 1 and 6 months.<br>
[0082] The protein export system can also be used with a live bacterial vector vaccine to increase the efficacy of the preparation. For example, United States Patent No. 5,387,744, to Curtiss et al., entitled "Avirulent microbes and uses therefor Salmonella typhi," which is hereby incorporated by reference, provides for a live bacterial vector vaccine against S. Typhi. More specifically, the Curtiss patent provides immunogenic compositions for the immunization of a vertebrate or invertebrate comprising an avirulent derivative of S. Typhi. The derivatives having a mutation of the cya and/or crp and/or cdt genes.<br>
[0083]	The avirulent derivatives taught by Curtiss et at., can be transformed with the protein export<br>
system described herein to allow the resulting recombinant organism to act as an immunogenic composition against S.<br>
Typhi, as well as any other antigen or antigens that are coupled to the protein export protein of the described system.<br>
F.	Additional Utility<br>
[0084]	In addition to therapeutic proteins and antigens which are useful for the pharmaceutical industry,<br>
the gene of interest may encode for enzymes, polypeptides, proteins, or amino acids which maybe useful for, by way of example only, the food industry, the nutritional supplement industry, the animal feed industry, the biomediation industry, the waste disposal industry, and the waste treatment industry. For these industries, the protein of interest encoded by the gene of interest may not need to be isolated from the medium of a bioreactor for the protein of interest to serve its function. The protein of interest may be a catalyst for a desired reaction or may act as a precursor component for a desired reaction.<br>
[0085]	The following examples are provided for illustrative purposes only, and are in no way intended to<br>
limit the scope of the present invention.<br>
EXAMPLES<br>
Example 1<br>
Cloning and mutagenesis of S. Typhi clyA<br>
[0086]	Identification of clyA was accomplished by BLASTN analysis of the recently completed S. Typhi<br>
genome sequence available from the Sanger Centre (Wellcome Trust Genome Campus, Hinxton, Cambridge, CB1O 1SA, UK) (See http://www.sanger.ac.Uk/Proiects/S typhi/blast server.shtml), using the DNA sequence from E. coli hlyE (GenBank accession number U57430).<br>
[0087]	The clyA open reading frame was identified as a 912 bp sequence predicted to encode a 304<br>
residue protein with a molecular mass of 33.8 kDa that is 89.4% identical to Â£ coli HlyE. Although clyA is 85.3% identical to the 915 bp E. coli hlyE open reading frame, the upstream transcriptional control region is distantly related with only 33.6% identical bases within a 250 bp region.<br>
-16-<br><br>
[0088] Based on this analysis, primers were designed for PCR amplification of a promoterless genetic cassette encoding ClyA in which an optimized ribosome-binding site was engineered 5'-proximal to the ATG start codon. The primer sequences are listed in Table 1.<br>
-17-<br>
TABLE 1. Primers used in construction and sequence analysis of the plasmid cassettes<br><br><br><br>
[0089]	'Relevant restriction sites are designated in bold case, underlined; ribosome binding sites and<br>
start codons are designated in italics.<br>
[0090]	To facilitate recovery, overlapping PCR techniques were used to create a promoterless 2252<br>
base pair clyAtetA genetic cassette synthesized by overlapping PCR as previously described using primers 1 and 2 with chromosomal template DNA from CVD 908-htrA, and primers 3 and 4 with template derived from pBR322, and recovered in pGEM-T (Promega, Madison Wl) transformed into E. cali DH5.<br>
[0091]	Recombinant clones were screened on solid agar medium containing sheep red blood cells.<br>
Specifically, screening for hemolytic activity was performed on freshly prepared 1XLB agar medium containing appropriate antibiotic selection and 5% sheep blood. Plates were then incubated at 37Â°C for 24 hours to detect zones of red blood cell (RBC) hemolysis. Several colonies were immediately identified which produced clear halos of hemolysis. This observation suggested that if clyA requires accessory proteins for translation out of the bacterium, these proteins are apparently common to both S. Typhi and E. coli, A positive isolate, designated pGEM-TclyA, was chosen for further use.<br>
[0092]	The functional roles of various regions of ClyA were examined to provide information for the<br>
proper engineering of recombinant fusion proteins encoding an antigen fused to ClyA. Specifically, the role played by the amino terminus, the carboxyl terminus, or both, in exportation of hemolysin out of the bacterium was examined.<br>
[00931 To accomplish this, clyA was randomly mutagenize using the transposon TnphoA. The "phoA" of "TnphoA" encodes alkaline phosphatase (See Manoil &amp; Bechwith, PNAS Vol 82, pp 8129-8133,1985). Transposition of TnphoA allows for random formation of in-frame fusions of the N-terminus of PhoA onto a given target protein. TnphoA mutagenesis was carried out after electroporation of pGEM-TclyA, expressing functional S. Typhi ClyA hemolysin, into DH5. to yield DH5(pGEM-TclyA)... A cross-streak mating was then performed between DH5 (pGEM-TclyA) and the TnphoA donor strain SM10(pRT733) and selecting transconjugants on 2XLB50 supplemented with tetracycline, carbenicillin, and kanamycin at 10 g/ml, 50 g/ml, and 10 g/ml respectively (2XLB50 + T10C50K10). Bacteria were then pooled and grown up in broth cultures for plasmid purification, and purified plasmids retransformed into the phoA 20 mutant E. coli strain CC118 for selection of Pho* transformants on 2XLB50 + T10C50K10 supplemented with 200 g/ml of the alkaline phosphatase substrate 5-Bromo-4-Chloro-3-lndolyl-Phosphate (BCIP; Sigma, St. Louis, MD). Target protein fusions that are N-terminally secreted into the periplasm, surface exposed, or exported out of the bacterium<br>
-18-<br><br>
entirely, can easily be screened using the chromogenic substrate BCIP to detect deep blue halos of hydrolysis; proteins which are C-terminally secreted will not be detected using this method.<br>
[0094] Using TnphoA mutagenesis, 4 of 621 PhoA* colonies were identified that no longer displayed hemolytic activity. Sequencing of one isolate confirmed the in-frame insertion of PhoA after residue 179 (Ala) of ClyA. This insertion truncated ClyA in the proposed hydrophobia transmembrane region and removes the remaining 125 carboxyt-terminal residues. It was therefore concluded that the carboxy -terminus of 5. Typhi ClyA is not required for transport of the cytoplasm of E. coli (and presumably from S. Typhi also), and that genetic fusion of heterologous genes potentially encoding exported protein fusions should be carried out at the 3'terminus of clyA.<br>
Example 2 Construction of carboxyl-terminal fusions of test antigens to CIyA.<br>
[0095] To test the ability to export passenger proteins fused at the carhoxyl terminus of ClyA, the bta gene encoding the RTEM-1 -lactamase protein which confers resistance to both ampicillin and carbenicillin, was chosen for experimentation.<br>
[0096]	This protein fusion was engineered as a genetic fusion of a Spal cassette inserted in-frame into<br>
the Nhel site adjacent to the tandem stop codons at the cfyA 3'terminus of pSEC84. Initially, an 807 bp Spel-Nhel fragment encoding the mature 268 amino acid lactamase without the 23 residue signal sequence was synthesized from a pBR322 derivative by PCR. The purified fragment was then inserted in-frame into the engineered carboxyl terminal Nhe\ site of cfyA to create a 1742bp clyAbla genetic fusion encoding a predicted 62.9 kDa fusion protein. The desired plasmid construct was easily recovered in isolated colonies from cultures grown in the presence of 5 jjg/ml carbenicillin, but plasmids recovered after selection with 50 ng/ml carbenicillin appeared to be unstable and genetically rearranged. blatetA fusion<br>
I00971	Because of the problem with plasmid stability and genetic rearrangement of the clyA-bla<br>
construct described above, X^z blatetA fusion was synthesized as a 2111 bp Spe\ cassette by overlapping PCR using primers 5 and 6 with pGEM-T template and primers 7 and 4 with template derived from pBR322; insertion of this cassette into pSEC84 cleaved with Nhe\ yielded pSEC&amp;W/a (see Figure IB).<br>
100981	After introduction into CVD 908-A//yJ, colonies were screened for retention of hemolytic activity,<br>
and then screened for -lactamase activity using the chromogenic substrate nitrocefin at a concentration of 100 g/ml in 2XLA50+DHB+T10; plates were incubated at 30Â°C for at least 16 hours and examined for the presence of red halos around colonies indicating cleavage of nitrocefin. Red halos were observed around CVD 908-A//4{pSEC846/d), indicating cleavage of nitrocefin, confirmed the presence of enzymatically active -lactamase. It was concluded that an approximate doubling of the molecular mass of ClyA from 34 kDa to 63 kDa resulted in a 2 domain fusion protein in which both domains apparently folded correctly to maintain the expected biological activity of each domain.<br>
-19-<br><br>
sacB-tetA fusion<br>
[00991 To investigate the versatility of ClyA as a fusion partner to export heterologous antigens out of S. Typhi, the efficiency of CfyA to export the potentially lethal levansucrase encoded by sacB from Bac/ffas subtffis was examined. Expression of the sscB gene is lethal when expressed within the cytoplasm of enteric bacteria, including S. Typhi growing in the presence of sucrose. Construction of a ClyA-SacB protein fusion with a predicted molecular mass of 83.9 kDa, for introduction into CVD SQBhtrA was attempted. This fusion was engineered as a sacB-tetA Spe\ cassette encoding the mature 445 residue 50.0 kDa levansucrase, without the 29 amino acid signal sequence, and inserted in-frame into the engineered carboxyl terminal ASM site of ClyA in pSEC64. CVD 908-Af/tf carrying the desired construct was selected using tetracycline and screened in the presence of sucrose for survival. If ClyA-SacB failed to be exported out of the cytoplasm, no isolates would be recovered, but for fusions either surface expressed or fully exported out of the bacterium into the surrounding medium, an enzymaticatty active SacB moiety would be expected to cleave sucrose to release glucose, which would immediately be transported into the bacterium and metabolized.<br>
[0100]	The sacB-tetA cassette ms synthesized using primers 8 and 9 with plB279 template and<br>
primers 10 and 4 as above to create a 2653 bp Spe\ cassette inserted into pSEC84 generating the dykisacB fusion of pSEC84sacB (SEQ ID N0:18) (see Figure iq. After introduction into CVD 908-AfM, colonies were again screened for retention of hemolytic activity, and then examined for levansucrase activity by plating on MacConkey agar base medium (Difcoj supplemented wraYDHB and either sucrose (8% or 16% w/v) or 8% sucrose + 8% arabinose as the sole carbohydrate source. Rates were incubated at 30Â°C for 16-24 hours to recover isolated cfus and determine fermentation of the carbohydrate; additional incubation at room temperature for several more days was required to observe formation of the porysaccharide-like domes over colonies.<br>
(0101] As shown in Figures 2B and 2D, growth of CVD QG$htrA{p$ECB4sacB) was excellent when grown on indicator medium containing either 8 % sucrose or 16% sucrose as the sole carbohydrate source (where grown on MacConkey agar base medium). Indeed, a polysaccharide-like dome was observed to form over isolated CVD 908-fttrAipStCSAsacB) colonies which was not observed for CVD 908-A//4 (Figures 2A and 2C), and intensified with increasing concentration of sucrose. Hypothesizing that this polysaccharide-like material was levan, formed by the levansucrase-catalyzed polymerization of fructose liberated from hydrolysis of sucrose, we attempted to block this polymerization by introducing 8% L-arabinose which is known to inhibit levansucrase. As shown in Figure 2F, domes were no longer observed, with CVD SQB-htrA and CVD 908-AfM(pSEC84wc5} colonies now appearing similar.<br>
(0102) If ClyA-SacB protein fusions are indeed exported out of CVD 908-Arr&gt;l(pSEC84Â«ffl), then cleavage of sucrose by the SacB domain to liberate free glucose should provide a metabolic advantage compared CVD 908-A/M when these strains are grown as broth cultures in the presence of sucrose. To test this hypothesis, 100 ml broth cultures of either CVD 908-Ar7tf(pSEC84) or CVD 908-A//rf(pSEC84tfc5) were set up in 1 liter baffle flasks containing 2XLB50+DHB+K10 plus 10% sucrose and growth was compared to CVO 908*A//4(pSEC84) cultures grown<br>
-20-<br><br>
in the presence of 10% glucose 8$ a positive control. As shown in Figure 3, CVO 908-/&gt;/7%SEC8ta:5) was observed to grow faster in the presence of sucrose than either CVD 908-Afr4(p$EC84) growing with glucose or sucrose, an observation confirmed with viable counts. When taken together with results observed above for ClyA-BIa, the data strongly suggest that ClyA is a versatile fusion partner for export out of out of bacteria properly folded fusion proteins in which the biological activity of the fused domains is preserved. ctvAiurfouv fusion<br>
10103] To further define the export properties of ClyA and specifically verify the presence of ClyA fusion products in the supernatant of exponentially growing CVD 908-A//A a genetic fusion of clyA was constructed where cfyA was fused to the fluorescent reporter green fluorescent protein (GFPuv) creating the clyA::gfpuv cassette of pSECBVjEwr (see Figure ID), and isogenic to both pSEC84Â£/* and pSEC84satf. Again, CVD 908-ArM(pSECB4#//wv) remained hemolytic but with reduced fluorescence when compared to cytoplasmically expressed GFPuv. Using GFP polyclonal antibody (BO Biosciences Clontech, Palo Alto, CA), the export of ClyA-GFPuv into the culture supernatant was examined using Western immunoblot analysis, as shown in Figure 4. Figure 4 illustrates a set of Western immunoblots analyzing bacterial cell fractions from either CVD 908-Af/yl {lanes 1-3) or CVD 9084tt4(pSEC84fffrw) (lanes 4-8). Cell fractions are loaded as follows: supematants, lanes 1 and 4; cytoplasmic, lanes 2 and 6; periplasmic, lane 5; insoluble, lane 7; whole cell, lanes 3 and 8; and 50 ng GFPuv, lane 9. Membranes with identical samples were probed with antibodies specific for GFPuv (panel A) or Â£ coli GroEL {panel B). As can be seen in this figure, a significant amount of the expected 61 kDa protein fusion is detected in 0.5 ml of TCA-precipitated supernatant from CVD 908-Af/&gt;4(pSEC84#5w/tf (lane 4); an irrelevant cross-reacting species of approximately 45 kDa is also detected in the cytoplasm of CVD 908 htrA (lane 2) and in the cytoplasmic, insoluble, and whole cell fractions of CVD 908-htrAtySKMgfptri; interestingly, lane 5 suggests that very little ClyA-GFPuv is recovered from the periplasmic space.<br>
-21-<br><br>
Conclusion<br>
101045 The results from this work clearly support the conclusion that the cryptic hemoiysin ClyA from S. Typhi can be used to facilitate the export of heterologous antigen domains out of the attenuated vaccine strain CVD SQB-fttrA and into the surrounding medium. Furthermore this work demonstrates that ClyA can be used to facilitate the export of a fusion protein out of bacteria into the surrounding medium. As illustrated above, the ability to export property folded proteins of interest fused at the cartioxyl terminus of ClyA was shown using the bla gene encoding the RTEM-1 -lactamase protein which confers resistance to both ampicillin and carbenicillin. The bla gene of pBR322 is 861 bp in length and encodes a 31.5 kQa protein with a 23 amino acid signal sequence directing N-terminal secretion of -lactamase into the periplasmic space. The work above indicates the successful engineering of a gene fusion encoding a functional ClyA- -lactamase protein fusion which retained both hemolytic activity and the ability to cleave the chromogenic -lactamase substrate nitrocefin to produce red halos against a yellow background of uncleaved nitrocefin.<br>
1010S| Interestingly, attempts to select for such expression vectors when growing transformants in rich medium supplemented with 50 gfmt of either carbenicillin or ampicillin were unsuccessful and only extensively rearranged ptasmids were recovered as judged by restriction mapping. It has been conclusively demonstrated that cytoplasmically expressed -lactamase confers resistance to "5 g/ml of ampicillin, while appropriately expressed periplasmic beta-lactamase confers resistance to &gt; 4000 g/ml of ampicillin. However, surface display of -lactamase protein fusions have been shown to confer resistance to "100 g/ml of ampicillin. Indeed, Chervaux et af. have reported that Hly A-mediated secretion of beta-lactamase fusions out off. colt again confer low-level resistance to "5 ng/ml of ampicillin. They demonstrated that even though the specific activity of the intact lactamase domain of the surface fusion remained similar to that of unmodified beta-lactamase, resistance to high levels of ampicillin was not observed, and they concluded that bacterial resistance to betalactam antibiotics requires significant concentrations of beta-lactamase within the periplasmic space close to the hilling targets. Based on such observations, it was concluded that properly folded ClyA-beta-lactamase protein fusions were synthesized within CVD 908-A//tf(pSEC84A/a) and exported to confer a hemolytic phenotype, as well as beta-lactamasemediated hydrolysis of the chromogenic cephalosporin nitrocefin, without conferring resistance to ampicillin or carbenicillin.<br>
101061	To more clearly define the nature of ClyA-mediated export of heterologous antigen domains out<br>
of CVD 908-Aftf, and perhaps rule out the involvement of periplasmic intermediates, fusions of sacB, encoding the potentially lethal levansucrase from B. subtilis were studied. Levansucrase is a 50 IcDa single polypeptide exoenzyme that catalyzes the hydrolysis of sucrose to yield free glucose and fructose, and in turn catalyzes the polymerization of fructose into long polymers called levan. Secretion of levansucrase from B. subtilis growing on medium containing sucrose results in the growth of isolated colonies covered by an impressive dome of viscous levan after extended incubation at room temperature.<br>
-22-<br><br>
[0107] It is well established that eyttoplasmic and periplasmic expression of levansucrase encoded by sacB is lethal for a variety of bacteria growing in the presence of sucrose. It has recently been shown using signal peptide mutations that levansucrase becomes lethal within the cytoplasm of B. subtilis grown in the presence of sucrose, and that inactivation of the fructose polymerase activity was essential for removal of sucrose-induced lethality. It was therefore reasoned that failure of ClyASacB fusions to be exported out of both the cytoplasm and periplasmic space of CVD 908-Ar/4 should result in significant intracellular accumulation of the fusion protein resulting in lethality for CVD 90&amp;htrA{pSlC&amp;4sacB) growing in the presence of sucrose.<br>
[0108]	As shown in Figure 2B, however, CVD Â§0QhtrA{p$iC84sacB) was observed not Drily to grow in<br>
the presence of 8% sucrose but to ferment the sugar, a phenotype not observed for CVD 908-Ato4(pSEC84) grown under the identical conditions. As the concentration of sucrose was increased from 8% to 16% sucrose, fermentation of sucrose also increased with the accumulation of impressive domes of (even-Eke material which vanished in the presence of the levansucrase inhibitor arabinose. Simitar observations of levansucrase activity were reported by Jung et at for a surface expressed levansucrase domain fused to the carboxyl terminus of the ice nucleation protein of Pseudomonas syringae and expressed within Â£. coli. In view of these results, it was concluded that the engineered CVO 908-A//4(pSEC845300) had the ability to utilize sucrose as a carbon source in broth culture experiments in which CVD 908-f)trA{pSECQ4s3cB) was observed to grow faster than CVD 908-M/v4(pSEC84) grown either in the presence of sucrose or pure glucose. It was again concluded that, as with the ClyA-betalactamase protein fusions described above, that properly folded ClyASacB protein fusions were synthesized within CVD 908-A//vI, and exported to confer both the expected hemolytic phenotype, as welt as levansucrase activity allowing for the extracellular catabolism of an alternate carbohydrate source not utilized by the plasmid-less host strain.<br>
Example 3 Bioreactor Protein Expression of a ClyA SacB fusion<br>
[01091	A bioreactor is prepared according to the teachings of U.S. Patent No. 5,635,368, which is<br>
hereby incorporated by reference in its entirety. Briefly, granular derivatized cellulose is manufactured according to U.S. Pat. No. 4,355,117 as follows: 25 parts of fibrous cellulose is mixed with 25 parts of titanium dioxide and the mixture is compounded with 50 parts of high-impact polystyrene using a twin-screw extruder. The extrudate is cooled in water, and sieved to a particle size of 0.35-0.85 mm. The sieved granular agglomerated cellulose particles are derivatized to form DEAE cellulose as described in the U.S. Patent above.<br>
[0110]	Next, ten (10) grams of the granular DEAE-cellulose is reduced to a slurry in distilled water and<br>
soaked for 5 hours with occasional stirring. The hyiirated carrier is then decanted with the distilled water and transferred into a glass column with an inner diameter of 15 mm where it forms a bed with a height of 145 mm.<br>
[0111]	Bacteria transformed with pSEC84Mc5 (see Example 2) are cultured for 48 hours at 3(PC.<br>
Fifty (50) milliliters of the cell suspension is pumped through the carrier bed at a flow velocity of 25 ml/hour.<br>
-23-<br><br>
Subsequently, additional amounts of culture medium is pumped through the carrier bed. The outflow of the column is collected and the recombinantly expressed OlyA-SacB fusion protein (encoded by SEQ ID NO: 19) is isolated and purified from the outflow. Cleavage of SacB would provide ample commercial amounts of leyansucrase for the generation of levan.<br>
Example 4 His-tag protein purification under denaturing conditions<br>
[0112]	A bacterial culture is transformed with an expression vector containing an expression cassette<br>
comprising the coding sequence for an attenuated ClyA protein fused to a sacB gene, which is fused to a coding sequence encoding a protease recognition site, which is fused to a polyhistidine tag encoding sequence. The bacterial culture is introduced into a bioreactor such as that described in Example 3.<br>
[0113] The culture is placed under conditions promoting expression of the recombinant fusion protein, which is exported into the culture medium. The culture medium is collected and applied to a Ni column (HISTRAP; Pharmacia) equilibrated with a urea containing buffer at a concentration sufficiently high to denature the protein. The column ts then washed and eluted. The eluate is analyzed by gel electrophoresis to determine the presence of the purified protein.<br>
[0114]	Purified protein containing fractions are dialyzed against an enzyme digestion buffer.   The<br>
diatyzed samples are then pooled and subjected proteolysis catalyzed by the appropriate enzyme. The proteolyzed sample is purified to eliminate the deleted polyhistidine tag, leaving the isolated, purified protein.<br>
-24-<br><br>
Example 5 Construction of Attenuated CVD 90B-A/B4 that Expresses Fran C and Raising an Immune Response Thereto<br>
[0115]	A ClyA-Frag C fusion protein is generated in CVD 908-^ according to the steps discussed in<br>
Example 1. Our approach is to express a codon-optimized toxC open reading frame encoding fragment C of tetanus toxin<br>
inserted into ClyA expressed from the expression vector disclosed herein. Export of fragment C is accomplished through<br>
an in-frame genetic fusion of toxC to the 3' terminus of cfyA and carried on the orfc\ replicon pSEC84 as a 1426 bp P.^<br>
-ciyA Â£coH\ - Nhe\ cassette. toxC encoding fragment C is re-engineered from prior art constructs using the forward<br>
primer	5'<br>
GCGCACIAGIAAAAACCTTGAnGnGGGTCGACAACGAAGAAGACATCGATGnATCCTGAAAAAGTCTACCATTCTGAA CTTGGACATCAAC- 3* (SEQ ID NO: 15) and the reverse primer 5' AACTACCGCAnAAAGCTTATCGATGATAAGCTGTCAAACATGAGCIAgCCTAGGTCAnAGTCGTTGGTCCAACCnCAT CGGTCGGAACGAAGTA- 3' (SEQ ID NO: 16) to generate the desired PCR product (1424 bp). The toxC cassette is then subcloned into pSEC84 digested with Nhe\ to construct pSEC84rarC. The DNA sequence of the intended clyA - toxC fusion junction is confirmed using the sequencing primer 5'-CGATGCGGCAAAATTGAAATTAGCCACTGA-3' (SEQ ID NO: 17) which hybridizes 172 bases upstream of the engineered Nhe\ site at the 3'terminus of ctyA. Constructs are screened for retention of hemolytic activity and confirmed for export of the ClyA-Frag C into the supernatant by Western immunoblot analysis.<br>
[0116]	Groups of ten 6 weeks old Balb/c mice are immunized intranasally with 1.0x10'Â° cfu of strain<br>
CVD 908A//yl expressing the ClyA-Frag C fusion protein. Mice are bled prior and 30 days after their immunization, and their serum is stored at -20Â°C until use. Antibodies present in the serum against ClyA and Frag C antigens are determined by ELISA. The results indicate that immunization with strain CVD 908-ArM expressing the ClyA-Frag C fusion protein elicits antibody levels against the Frag C antigen that are significantly higher than those obtained with strain 908-Afftf not expressing the ClyA-Frag C fusion protein. The results demonstrate that the expression of the Frag C antigen as a fusion protein with ClyA enhances the immune response against this antigen. Protective immunity against tetanus toxin is confirmed by challenging immunized mice with otherwise lethal doses of natural tetanus toxin.<br>
[0117]	White the disclosure above describes the invention in detail and with reference to specific<br>
embodiments thereof, it will be apparent to one of ordinary skill in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br>
-25-<br><br>
REFERENCES<br>
[0118] Atkins, A., N. R. Wyborn. A. J. Wallace. T. J. Stillman, L K. Black, A. B. Fielding, M. Hisakado, P. J. Artymiub, and J. Green. 2000. Structure-function relationships of a novel bacterial toxin, hemolysin E. The rote of E. J. BioL Chem. 275:4115041155.<br>
[0119]	Bailey. J. E., Host-vector interactions in Escherichia coli, p. 29-77. In A. Fiechter (ed.|. Advances<br>
in Biochemical Engineering. Biotechnology. Springer-Verlag, Berlin (1993).<br>
[00120]	Balbas, P., X. Soberon, L Merino, M. Zurita, H. lomeli, F. Valle, N. Flores, and F. Bolivar.<br>
1986. Plasmid vector pBR322 and its special-purpose derivatives - a review. Gene 50:340.<br>
[0121] Blomfield. I. C, V. Vaughn, R. F. Rest and B. I. Eisenstein. 1991. Adelic exchange in Escherichia coli using the Bacillus subtitis sacB gene and a temperature-sensitive pSCIOl replicon. Mot. Microbiol. 5:1447-1457.<br>
[0122]	Boe. L, K. Gerdes, and S. Molin. 1987. Effects of genes exerting growth inhibition and<br>
plasmid stability on plasmid maintenance. J. Bacteriol. 169:46464650.<br>
[0123]	Borehert T. V. and V. Nagnrajan. 1991. Effect of signal sequence alterations on the export of<br>
levansucrase m Bacillus subtilis. J. Bacteriol. 173:276-282.<br>
[0124]	Bramucci. M. G. and V. Negarajan. 1996. Direct selection of cloned DNA in Bacillus subtilis<br>
based on sucrose-induced lethality. Appl. Environ. Microbiol. 62:3948 3953.<br>
[0125]	Chervaux, C, N. Sauvonnut A. Le Clainche, B. Kenny. A. L Hunt J. K. Broome Smith,<br>
and I. B. Holland. 1995. Secretion of active -tactamase to the medium mediated by the Escherichia coli haemolysin transport pathway. Mol. Gen. Genet. 249:237-245.<br>
[0126]	Corchero, J. L and A. Villa verde. 1998. Plasmid maintenance in Escherichia co/zrecombinant<br>
cultures is dramatically, steadily, and specifically influenced by features of the encoded proteins. Biotechnol. Bioeng. 58:625-632.<br>
[0127]	Cserjan-Puschmann, M., W. Kramer. E. Duerrschmid, G. Streidner, and K. Bayer. 1999.<br>
Metabolic approaches for the optimisation of recombinant fermentation processes. Appl. Microbiol. Biotechnol. 53:43-50.<br>
[0128]	Datta, N. and P. Kontomichalou. 1965. Pemcillinase synthesis controlled by infectious R<br>
factors in Enterobacteriaceae. Nature 208:239-241.<br>
[0129]	Dedonder. R. 1966. Levansucrase from Bacillus subtilis, p. 500-505. In E. F. Meufeld and V.<br>
Ginsburg (eds.). Methods in Enzymology. Academic Press, New York.<br>
[0130]	del Castillo, F. J., S. C. Leal, F. Moreno, and I. del Castillo. 1997. The Escherichia a?//K-12<br>
sheA gene encodes a 34-kDa secreted haemolysin. Mol. Microbiol. 25:107-115.<br>
-26-<br><br>
[0131] Fouet A., M. Arnaud, A. Klier, and G. Rapoport 1984. Characterization of the precursor form of the eaocellular levansucrase from Bacillus sub tills. Biochem. Biophys. Res. Commun. 119:795-800.<br>
[0132] Galen. J. E., 0. G. Gomez Duarte, G. Losonsky, J. L Halpern, C. S. Lauderbaugh, S. Kalmuck. M. K. Reymann, and M. M. Lsvine. 1997. A murine model of intranasal immunization to assess the rmmunogenicity of attenuated Salmonella typhi \m vector vaccines in stimulating serum antibody responses to expressed foreign antigens. Vaccine 15:700-708.<br>
[0133]	Galen, J. E. and M. M. Levine. 2001. Can a 'flawless' live vector vaccine strain be engineered?<br>
Trends in Microbiology 9:372-376.<br>
[0134] Galen, J. L. J. Naif. J. Y. Wang, S. S. Wasserman, M. K. Tanner, M. Sztein. and M. M. Levine. 1999. Optimization of plasmid maintenance in the attenuated live vector vaccine strain Salmonella typhi CVD mtttrA. Infect.Immun. 67:6424-6433.<br>
[0135]	Gay, P., 0. Le Coq, M. Steinmetz, T. Berkelman, and C. I. Kado. 1985. Positive selection<br>
procedure for entrapment of insertion sequence elements m Gram-negative bacteria. J. Bacteriol. 164:918-921.<br>
[0136]	Gay. P., D. Le Coq, M. Steiiunetz, E. Ferrari, and J. A. Hoch- 1983. Cloning structural gene<br>
sacB, which codes for exoenzyme levansucrase of Bacillus subtilis:   expression of the gene in Escherichia colt. J. Bacteriol. 153:1424-1431.<br>
[0137]	Click. B. R., Biotechnol. Adv. 13:247 261 (1995).<br>
[0138]	Han, Y. W. 1990. Microbial levan. Advances in Applied Microbiology 3 5:171 -194.<br>
[0139]	Harcum and Bentley. 1993. Biotechnol. Bioeng. 42:675-685.<br>
[0140]	Hone, D. M., A. M. Harris. !&gt;, Chatfietd, G. Dougan, and M. M. Levine. 1991. Construction<br>
of genetically defined double aro mutants of Salmonella typhi Vaccine 9:810-816.<br>
[0141] Jung. H., J. Lebeault and J. Pan. 1998. Surface display of Zymomonas mobilis levansucrase by using the ice-nucleation protein of Pseudomonas syringae. Nat. Biotechnol. 16:576-580.<br>
[0142]	Lattetnann, C. T., J. Maurer, E. Gerland, and T. F. Meyer. 2000. Autodisplay:  functional<br>
display of active -lactamase on the surface of Escherichiacolity the AIDAI autotransporter. J. Bacteriol. 182:3726-3733.<br>
10143)	Le Coq, D;, P. Ratet M. SteinmeU, and P. Gay. 1984. A genetic approach to levansucrase<br>
secretion in Bacillus subtilis, p. 141-152. In A. T. Ganesan and J. A. Hoch (eds.). Genetics and biotechnology of bacilli.<br>
Academic Press, New York.<br>
10144)	LeBrun, E. and R. van Rapenbusch. 1980. The structure of Bacillus subtilis levansucrase at<br>
3.8 A resolution. J. Biol. Chem. 255:12034-12036.<br>
-27-<br><br>
[0145]	Ludwig, A., S. Bauer. R. Bent, B. Bergmann, and W. GoebeL 1999. Analysis of the SlyA-<br>
controlled expression, subr.ellular localization and pore-forming activity of a 34 kDa haemolysin (ClyAI from Escherichia C0//K-12. Mol. Microbiol. 31:557 567.<br>
[0146]	Matthew. M. and R. W. Hedges. 1976. Analytical isoelectric focusing of R factor-determined â¢<br>
lactamases: correlation with pfasmid compatibility. J. Bacteriol. 125:713-718.<br>
[0147]	McDermott P- J., P- G owl and, and P. C. G owl and. 1993. Adaptation of Eschehchia cofi<br>
growth rates to the presence of pBR322. Lett. Appl. Microbiol. 17:139-143.<br>
[0148] Orr, N., J. Â£. Galen, and M. M. Levine. 1999. Expression and immunogenicity of a mutant diphtheria toxin molecule, CRMm, and its fragments in Salmonella typhi vaccine strain CVD 9QBhtrA. Infect, fmmun. 67:42904294.<br>
[0149]	Qscarsson, J., Y. Mizunoc, L Li, X. Lai, A. Wieslartder. and B. E. Uhlin. 1999. Molecular<br>
analysis of the cytolytic protein ClyA (SheA) from Eschehchia cofi Mol. Microbiol. 32:1226 1238.<br>
[0150]	Oscars son, J., Y. Mizunoe, B. E. Uhlin, and D. J. Haydon. 1996. Induction of haemolytic<br>
activity in Escherichia cofi by the sfyA gene product. Mol. Microbiol. 20:191-199.<br>
[0151]	Pecota, 0. C, C. S. Kim, K. Wu, K. Gerdes. and T. K. Wood. 1997. Combining the fiokisok,<br>
parQE, and pnd postsegregational killer loci to enhance plasmid stability. Appl. Environ. Microbiol. 63:1917-1924.<br>
[0152]	Pluckthun. A. and J. R. Knowles. 1987. The consequences of stepwise deletions from the<br>
signal-processing site of -lactamase. J. Biol. Chem. 2152:3951-3957.<br>
[0153}	Ried, J. and A. Collmer. 19B7. An nplsacBsacR cartridge for constructing directed.unmarked<br>
mutations in Gram-negative bacteria by marker exchange-eviction mulagenesis. Gene 57:239-246.<br>
[0154]	Samfarook, J.. E. F. Fritsch. and T. Maniatis. 1989. AnonymousMolecular cloning:    a<br>
laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.<br>
[0155]	Sambrook, J. and D. W. Russell. 2001. Expression of cloned genes in Escherichia colt, p.<br>
15.35AnonymousMolecular cloning. A laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor.<br>
[0156]	Shaw, K. J., P. N. Rather, R. S. Hare, and G. H. Miller. 1993. Molecular genetics of<br>
aminoglycoside resistance genes and familial relationships of the aminoglycoside-modifying enzymes. Microbiol. Rev.<br>
57:138-163.	Â¦<br>
[0157]	Smith &amp; Bidochka. Can. J. Microbiol. 44:351-355 (1998).<br>
[0158]	Steinmetz, M.. D. Le Coq, H. B. Djemia, and P. Gay. 1983. Genetic analysis of sacB, the<br>
structural gene of a secreted enzyme, levansucrase of Bacillus subtifis Marburg. Mol. Gen. Genet. 191:138-144.<br>
[0159]	Summers, D. K. 1998. Timing, self-control and sense of direction are the secrets of multicopy<br>
plasmid stability. Mol. Microbiol. 29:1137-1145.<br>
-28-<br><br>
[0160] Sutcliffe, J. G. 1978. Nudeotids sequence of the ampicillin resistance gene of Escherichia col/ plasmid pBR322. Proceedings of the National Academy of Sciences USA 75:3737-3741.<br>
[0161] Tacket C. 0., M. Satein, G. Losonsky. S. S. Wasserman, J. P. Nataro, R. Edelman. D. Pickard, G. Dougan, S. Chatfiefld, and M. M. Levine. 1997. Safety of live oral Salmonella typfii vaccine strains with deletions in IttrA and aroCamD and immune response]; in humans. Infect. Immun. 65:452-456.<br>
[0162]	Wallace, A. J., T. J. Stillman. A. Atkins. S. J. Jatnieson, P. A. BuHough, J. Green, and P.<br>
J. Artymiuk. 2000. E cofi hemolysin E (HlyE, ClyA, She A I:  X-ray crystal structure of the toxin and observation of membrane pores by electron microscopy. Cell 100:26!S-276.<br>
[0163]	Wang. J. Y-, F. Noriega, J. E. Galen, L M. Barry, and M. M. Levine. 2000. Constititive<br>
expression of the Vi polysaccharide capsular antigen in attenuated Salmonella enterica serovar Typhi oral vaccine strain CVO 90S. Infect. Immun. 68:4647-4652.<br>
[0164] Wang, J. Y., M. F. Pasetti. F. Noriega, R. J. Anderson, S. S. Wasserman. J. E. Galen. M. Sztein, and M. M. Levine. 2001. Construction, genotypic and phenotypic characterization, and immunogenicity of attenuated UguaBA Salmonella enterica serovar Typhi strain CVD 915. Infect. Immun. 69:47344741.<br>
[0165]	Wu, K. and T. K. Wood. 1994. Evaluation of the hoklsok killer locus for enhanced plasmid<br>
stability. Biotechnol. Bioeng. 44:912-921.<br>
-29-<br><br>
WE CLAIM:<br>
1.	A method for expressing a gene in a bacterial cell comprising;<br>
providing an expression vector to a population of untraasformed bacterial cells,<br>
wherein the expression vector comprises an expression cassette comprising an export protein coding sequence genetically fused to a protein of interest;<br>
expressing the expression cassette such that an export protein:;protein of interest fusion protein is produced and exported into the culture medium; wherein said export protein coding sequence is selected from the group consisting of Salmonella enterica serovar Typhi (S. Typhi) cytolysin A (ClyA) protein coding sequence of SEQ ID NO:21, Salmonella paratyphi ClyA protein coding sequence of SEQ ID NO:23, and Escherichia coli (E. coif) hemolysin E (HlyE) protein coding sequence of SEQ ID NO;25.<br>
2.	The method as claimed in Claim 1, wherein said bacterial cell is a S. Typhi<br>
cell.<br>
3.	The method as claimed in Claim 1, wherein said bacterial cell is an E. coli<br>
ceil<br>
4.	The method as claimed in Claim 1, wherein the export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21.<br>
5.	The method as claimed in Claim 1, wherein export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21 with one or more<br>
mutations resulting in amino acid substitutions selected from the group consisting of<br>
amino acid substitutions at positions 180,185, 187, and 193 of the export protein, which<br>
attenuate the hemolytic activity of the export protein.<br>
6.	The method as claimed in Claim 1, wherein the protein of interest is an<br>
antigen.<br>
30<br><br>
7.	A system for expressing a protein of interest comprising:<br>
an expression vector comprising an expression cassette, wherein the expression cassette comprises an export protein coding sequence genetically fused to a protein of interest coding sequence;<br>
a host cell transformed with the expression vector; and<br>
a culturing environment for the: transformed host cell, wherein the expression cassette expresses an export protein: :protein of interest fusion protein, which is exported out of the transformed host cell; wherein said export protein coding sequence is selected from the group consisting of S. Typhi ClyA protein coding sequence of SEQ ID NO:21, Salmonellaparatyphi ClyA protein coding sequence of SEQ ID NO:23, and E. coli HlyE protein coding sequence of SEQ ID NO:25.<br>
8.	The system as claimed in claim 7, wherein said host cell is a S. Typhi cell.<br>
9.	The system as claimed in claim 7, wherein said host cell is an E. coli cell.<br>
10.	The system as claimed in Claim 7, wherein the export protein coding<br>
sequence isS1. Typhi ClyA protein coding sequence of SEQ IDNO:21.<br>
11.	The system as claimed in Claim 7, wherein export protein coding sequence<br>
is S. Typhi ClyA protein coding sequence of SEQ ID NO:21 with one or more mutation's<br>
resulting in amino acid substitutions selected from the group consisting of amino acid<br>
substitutions at positions 180, 185, 187, and 193 of the export protein, which attenuate<br>
the hemolytic activity of the export protein.<br>
12.	A method as claimed in claim 1, wherein said exported fusion protein is<br>
isolated.<br>
13.	A method for producing a recombinant protein comprising:<br>
providing an expression vector to a population of untransformed bacterial cells,<br>
wherein the expression vector comprises an expression cassette comprising an export<br>
31<br><br>
protein coding sequence genetically fused to a protein of interest;<br>
expressing the expression cassette such that an export protein::protein of interest fusion protein is produced and exported into the culture medium; wherein said export protein coding sequence is selected from the group consisting of Salmonella enterica serovar Typhi (S. Typhi) cytolysin A (ClyA) protein coding sequence of SEQ ID NO:21, Salmonellaparatyphi ClyA protein coding sequence of SEQ ID NO:23, and Escherichia coli (E. coli) hemolysin E (HlyE) protein coding sequence of SEQ ID NO:25; and<br>
collecting said export protein:: protein of interest fusion protein from said culture, medium.<br>
14.	The method as claimed in Claim 13, wherein said bacterial cell is a S.<br>
Typhi cell or an E. coli cell.<br>
15.	The method as claimed in Claim 13, wherein the export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21.<br>
16.	The method as claimed in Claim 13, wherein export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21 with one or more<br>
mutations resulting in amino acid substitutions selected from the group consisting of<br>
amino acid substitutions at positions 180, 185, 187, and 193 of the export protein, which<br>
attenuate the hemolytic activity of the export protein.'<br>
17.	The method as claimed in Claim 13, wherein the protein of interest is an<br>
antigen.<br>
18.	The method as claimed in Claim 13, wherein said export protein: :protein<br>
of interest fusion protein is capable of being used to elicit an immune response in a host.<br>
19.	A method for preparing an immunogenic composition, said method<br>
comprising providing an expression vector to a population of untransformed bacterial<br>
cells, wherein the expression vector comprises an expression cassette comprising an<br>
32<br><br>
export protein coding sequence genetically fused to a protein of interest, and wherein said export protein coding sequence is selected from the group consisting of Salmonella enterica serovar Typhi (S. Typhi) cytolysin A (ClyA) protein coding sequence of SEQ ID NO:21, Salmonellaparatyphi ClyA protein coding sequence of SEQ ID NO:23, and Escherichia coli (E. coli) hemolysin E (HlyE) protein coding sequence of SEO ID NO:25.<br>
20.	The method as claimed in Claim 19, wherein said bacterial cell is a S.<br>
Typhi cell or an E. coli cell.<br>
21.	The method as claimed in Claim 19, wherein the export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21.<br>
22.	The method as claimed in Claim 19, wherein export protein coding<br>
sequence is S. Typhi ClyA protein coding sequence of SEQ ID NO:21 with one or more<br>
mutations resulting in amino acid substitutions selected from the group consisting of<br>
amino acid substitutions at positions 180, 185, 187, and 193 of the export protein, which<br>
attenuate the hemolytic activity of the export protein.<br>
23.	The method as claimed in Claim 19, wherein the protein of interest is an<br>
antigen.<br>
The disclosure below provides A protein export system for efficiently producing recombinant protein from a host cell. In a preferred embodiment, the protein export system utilizes protein export machinery endogenous to the host bactenum into which the proteinexport system vector is introduced.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBhc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBjb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBkZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBmb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBmb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBnLnAuYS5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 g.p.a.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBsZXR0ZXJzIHBhdGVudC5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 letters patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1MzYta29sbnAtMjAwMyBwcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">00536-kolnp-2003 priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM2LUtPTE5QLTIwMDMtQ09SUkVTUE9OREVOQ0UgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">536-KOLNP-2003-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM2LUtPTE5QLTIwMDMtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">536-KOLNP-2003-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM2LUtPTE5QLTIwMDMtRk9STSAyNyAxLjEucGRm" target="_blank" style="word-wrap:break-word;">536-KOLNP-2003-FORM 27 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM2LUtPTE5QLTIwMDMtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">536-KOLNP-2003-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTM2LUtPTE5QLTIwMDMtRk9STS0yNy5wZGY=" target="_blank" style="word-wrap:break-word;">536-KOLNP-2003-FORM-27.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="202578-1-leaf-spring-arrangement-for-automotive-vehicle.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="202580-a-catalytic-exhaust-gas-purification-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>202579</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>536/KOLNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Mar-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Mar-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Apr-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>UNIVERSITY OF MARYLAND BALTIMORE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>520 west lombard street, baltimore ,md-21201-1727</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GALEN JAMES E</td>
											<td>42 CHINS COURT ,OWINGS MILLS MD-21117,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/44185</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/252,516</td>
									<td>2000-11-22</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/202579-a-method-for-expressing-a-gene-in-bacterial-cell-a-system-for-expressing-a-protein-of-interest-and-a-method-for-producing-recombinant-proteins-and-immunogenic-compositions-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:54:00 GMT -->
</html>
