<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/202531-nucleic-acids-encoding-a-g-protein-coupled-receptor-involved-in-sensory-transduction by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:39:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 202531:NUCLEIC ACIDS ENCODING A G-PROTEIN COUPLED RECEPTOR INVOLVED IN SENSORY TRANSDUCTION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NUCLEIC ACIDS ENCODING A G-PROTEIN COUPLED RECEPTOR INVOLVED IN SENSORY TRANSDUCTION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides amino acid sequences of novel sensory cell specific G-protein coupled receptors GPCR-B3 and the nucleic acids encoding the receptors. These receptors are generally characterized as having an amino acid sequence with greater than about 70% sequence identity to SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3. Antibodies that specifically recognize the novel GPCR-B3 receptors are also provided by this invention. Furthermore, this invention provides methods for detecting nucleic acids encoding the GPCR-B3 receptors and methods for detecting the receptors. In addition, the invention provides methods useful for screening candidate compounds for those capable of modulating sensory signaling in sensory cells through their ability to exert a functional effect on a GPCR-B3 receptor of this invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NUCLEIC ACIDS ENCODING A G-PROTEIN COUPLED RECEPTOR INVOLVED IN SENSORY TRANSDUCTION<br>
CROSS-REFERENCES TO RELATED APPLICATIONS This application claims priority to USSN 60/094,465, filed July 28, 1998, herein incorporated by reference in its entirety.<br>
STATEMENT AS TO FEDERALLY SPONSORED RESEARCH AND<br>
DEVELOPMENT<br>
This invention was made with government support under Grant No. 5R01 DC03160, awarded by the National Institutes of Health. The government has certain rights in this invention.<br>
FIELD OF THE INVENTION<br>
The invention provides isolated nucleic acid and amino acid sequences of sensory cell specific G-protein coupled receptors, antibodies to such receptors, methods of detecting such nucleic acids and receptors, and methods of screening for modulators of sensory cell specific G-protein coupled receptors.<br>
BACKGROUND OF THE INVENTION<br>
Taste transduction is one of the most sophisticated forms of<br>
chemotransduction in animals (see, e.g., Margolskee, BioEssays 15:645-650 (1993); Avenet &amp; Lindemann, J. Membrane Biol. 112:1-8 (1989)). Gustatory signaling is found throughout the animal kingdom, from simple metazoans to the most complex of vertebrates; its main purpose is to provide a reliable signaling response to non-volatile ligands. Each of these modalities is though to be mediated by distinct signaling pathways mediated by receptors or channels, leading to receptor cell depolarization, generation of a receptor or action potential, and release of neurotransmitter at gustatory afferent neuron synapses (see, e.g., Roper, Ann. Rev. Neurosci. 12:329-353 (1989)).<br>
1<br><br>
Mammals are believed to have five basic taste modalities: sweet, bitter, sour, salty and unami (the taste of monosodium glutamate) (see, e.g., Kawamura &amp; Kare, Introduction to Unami: A Basic Taste (1987); Kinnamon &amp; Cummings, Ann. Rev. Physiol. 54:715-731(1992); Lindemann, Physiol. Rev. 76:718-766 (1996); Stewart et al., Am. J. Physiol. 272:1-26 (1997)). Extensive psychophysical studies in humans have reported that different regions of the tongue display different gustatory preferences (see, e.g., Hoffmann, Mencken. Arch. Path. Anat. Physiol. 62:516-530 (1875); Bradley et al, Anatomical Record 212: 246-249 (1985); Miller &amp; Reedy, Physiol Behav. 47:1213-1219 (1990)). Also, numerous physiological studies in animals have shown that taste receptor cells may selectively respond to different tastants (see, e.g., Akabas et al., Science 242:1047-1050 (1988); Gilbertson et ai,J. Gen. Physiol. 100:803-24 (1992); Bemhardt et al., J. Physiol. 490:325-336 (1996); Cummings et al.,J. Neurophysiol. 75:1256-1263 (1996)).<br>
In mammals, taste receptor cells are assembled into taste buds that are distributed into different papillae in the tongue epithelium. Circumvallate papillae, found at the very back of the tongue, contain hundreds (mice) to thousands (human) of taste buds and are particularly sensitive to bitter substances. Foliate papillae, localized to the posterior lateral edge of the tongue, contain dozens to hundreds of taste buds and are particularly sensitive to sour and bitter substances. Fungiform papillae containing a single or a few taste buds are at the front of the tongue and are thought to mediate much of the sweet taste modality.<br>
Each taste bud, depending on the species, contain 50-150 cells, including precursor cells, support cells, and taste receptor cells (see, e.g., Lindemann, Physiol. Rev. 76:718-766 (1996)). Receptor cells are innervated at their base by afferent nerve endings that transmit information to the taste centers of the cortex through synapses in the brain stem and thalamus. Elucidating the mechanisms of taste cell signaling and information processing is critical for understanding the function, regulation, and "perception" of the sense of taste.<br>
Although much is known about the psychophysics and physiology of taste cell function, very little is known about the molecules and pathways that mediate these sensory signaling responses (reviewed by Gilbertson, Current Opn. in Neurobiol. 3:532-539 (1993)). Electrophysiological studies suggest that sour and salty tastants modulate taste cell function by direct entry of H+ and Na+ ions through specialized membrane channels on the apical surface of the cell. In the case of sour compounds, taste cell<br>
2<br><br>
depolarization is hypothesized to result from H+ blockage of K+ channels (see, e.g., Kinnamon et al., Proc. Nat'l Acad. Sei USA 85: 7023-7027 (1988)) or activation of pH-sensitive channels (see, e.g., Gilbertson et al, J. Gen. Physiol. 100:803-24 (1992)); salt transduction may be partly mediated by the entry of Na+ via amiloride-sensitive Na+ channels (see, e.g., Heck et al., Science 223:403-405 (1984); Brand et al., Brain Res. 207-214 (1985); Avenet et al., Nature 331: 351-354 (1988)).<br>
Sweet, bitter, and unami transduction are believed to be mediated by G-protein-coupled receptor (GPCR) signaling pathways (see, e.g., Striem et al.,Biochem. J. 260:121-126 (1989); Chaudhari etal.,J. Neuros. 16:3817-3826 (1996); Wong et al., Nature 381: 796-800 (1996)). Confusingly, there are almost as many models of signaling pathways for sweet and bitter transduction as there are effector enzymes for GPCR cascades (e.g., G protein subunits, cGMP phosphodiesterase, phospholipase C, adenylate cyclase; see, e.g., Kinnamon &amp; Margolskee, Curr. Opin. Neurobiol. 6:506-513 (1996)). However, little is known about the specific membrane receptors involved in taste transduction, or many of the individual intracellular signaling molecules activated by the   " individual taste transduction pathways. Identification of such molecules is important given the numerous pharmacological and food industry applications for bitter antagonists, sweet agonists, and modulators of salty and sour taste.<br>
The identification and isolation of taste receptors (including taste ion channels), and taste signaling molecules, such as G-protein subunits and enzymes involved in signal transduction, would allow for the pharmacological and genetic modulation of taste transduction pathways. For example, availability of receptor and channel molecules would permit the screening for high affinity agonists, antagonists, inverse agonists, and modulators of taste cell activity. Such taste modulating compounds could then be used in the pharmaceutical and food industries to customize taste. In addition, such taste cell specific molecules can serve as invaluable tools in the generation of taste topographic maps that elucidate the relationship between the taste cells of the tongue and taste sensory neurons leading to taste centers in the brain.<br>
SUMMARY OF THE INVENTION<br>
The present invention thus provides for the first time nucleic acids encoding a taste cell specific G-protein coupled receptor. These nucleic acids and the polypeptides that they encode are referred to as "GPCR-B3" for G-protein coupled<br>
3<br><br>
receptor ("GPCR") B3. These taste cell specific GPCRs are components of the taste transduction pathway.<br>
In one aspect, the present invention provides an isolated nucleic acid encoding a sensory transduction G-protein coupled receptor, the receptor comprising       greater than about 70% amino acid identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ED NO:3.<br>
In one embodiment, the nucleic acid comprises a nucleotide sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ ID NO:6. In another embodiment, the nucleic acid is amplified by primers that selectively hybridize under stringent hybridization conditions to the same sequence as degenerate primer sets encoding amino acid sequences selected from the group consisting of: IAWDWNGPKW (SEQ ID NO:7) and LPENYNEAKC (SEQ ID NO:8).<br>
In another aspect, the present invention provides an isolated nucleic acid encoding a sensory transduction G-protein coupled receptor, wherein the nucleic acid specifically hybridizes under highly stringent conditions to a nucleic acid having the sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ ID NO:6.<br>
In another aspect, the present invention provides an isolated nucleic acid encoding a sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3 wherein the nucleic acid selectively hybridizes under moderately stringent hybridization conditions to a nucleotide sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ ID NO:6.<br>
In another aspect, the present invention provides an isolated nucleic acid encoding an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain having greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1.<br>
In another aspect, the present invention provides an isolated nucleic acid encoding a transmembrane domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the transmembrane domain of SEQ ID NO:1.<br>
In another aspect, the present invention provides an isolated sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70%<br>
4<br><br>
amino acid sequence identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
In one embodiment, the receptor specifically binds to polyclonal antibodies generated against SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3. In another embodiment, the receptor has G-protein coupled receptor activity. In another embodiment, the receptor has an amino acid sequence of SEQ ID NO: 1, SEQ ID NO:2, or SEQ ID NO:3. In another embodiment, the receptor is from a human, a rat, or a mouse.<br>
In one aspect, the present invention provides an isolated polypeptide comprising an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain comprising greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO: 1.<br>
In one embodiment, the polypeptide encodes the extracellular domain of SEQ ID NO:1. In another embodiment, the extracellular domain is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide.<br>
In one aspect, the present invention provides an isolated polypeptide comprising a transmembrane domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the transmembrane domain of SEQ ID NO:1.<br>
In one embodiment, the polypeptide encodes the transmembrane domain of SEQ ID NO: 1, In another embodiment, the polypeptide further comprises a cytoplasmic domain comprising greater than about 70% amino acid identity to the cytoplasmic domain of SEQ ID NO: 1. In another embodiment, the polypeptide encodes the cytoplasmic domain of SEQ ID NO: 1. In another embodiment, the transmembrane domain is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide. In another embodiment, the chimeric polypeptide has G-protein coupled receptor activity.<br>
In one aspect, the present invention provides an antibody that selectively binds to the receptor comprising greater than about 70% amino acid sequence identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
In another aspect, the present invention provides an expression vector comprising a nucleic acid encoding a polypeptide comprising greater than about 70% amino acid sequence identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
5<br><br>
In another aspect, the present invention provides a host cell transfected with the expression vector.<br>
In another aspect, the present invention provides a method for identifying a compound that modulates sensory signaling in sensory cells, the method comprising the steps of: (i) contacting the compound with a polypeptide comprising an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain comprising greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3; and (ii) determining the functional effect of the compound upon the extracellular domain.<br>
In another aspect, the present invention provides a method for identifying a compound that modulates sensory signaling in sensory cells, the method comprising the steps of: (i) contacting the compound with a polypeptide comprising an extracellular domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3; and (ii) determining the functional effect of the compound upon the transmembrane domain.<br>
In one embodiment, the polypeptide is a sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70% amino acid identity to a polypeptide encoding SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3. In another embodiment, polypeptide comprises an extracellular domain that is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide. In another embodiment, the polypeptide has G-protein coupled receptor activity. In another embodiment, the extracellular domain is linked to a solid phase, either covalently or non-covalently. In another embodiment, the functional effect is determined by measuring changes in intracellular cAMP, IP3, or Ca2+. In another embodiment, the functional effect is a chemical effect. In another embodiment, the functional effect is a chemical effect. In another embodiment, the functional effect is determined by measuring binding of the compound to the extracellular domain. In another embodiment, the polypeptide is recombinant. In another embodiment, the polypeptide is expressed in a cell or cell membrane. In another embodiment, the cell is a eukaryotic cell.<br>
In one embodiment, the polypeptide comprises an transmembrane domain that is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide.<br>
In one aspect, the present invention provides a method of making a sensory  transduction G-protein coupled receptor, the method composing the step of<br>
6<br><br>
expressing the receptor from a recombinant expression vector comprising a nucleic acid encoding the receptor, wherein the amino acid sequence of the receptor comprises greater than about 70% ammo acid identity to a polypeptide having a sequence of SEQ ID NO: 1, SEQ ID NO:2, or SEQ ID NO:3.<br>
In one aspect, the present invention provides a method of making a recombinant cell comprising a sensory transduction G-protein coupled receptor, the method comprising the step of transducing the cell with an expression vector comprising a nucleic acid encoding the receptor, wherein the amino acid sequence of the receptor comprises greater than about 70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
In one aspect, the present invention provides a method of making an recombinant expression vector comprising a nucleic acid encoding a sensory transduction G-protein coupled receptor, the method comprising the step of ligating to an expression vector a nucleic acid encoding the receptor, wherein the amino acid sequence of the receptor comprises greater than about 70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
Figure 1 shows the proposed topology of the GPCR-B3 (SEQ ID NO:2), with a large extracellular domain extending from amino acid 1 to about amino acid 580 of the rat GPCR-B3 amino acid sequence (corresponding to nucleotide residues 1-1740 of the rat sequence, with the ATG initiator methionine defined as residue 1), and seven transmembrane domains. The large extracellular domain may extend into the first transmembrane domain. Dark residues indicate identities between GPCR-B3 and GPCR-B4 (for a description of GPCR-B4, see, e.g., USSN 60/095,464, filed July 28,1998, and USSN 60/112,747, filed December 17,1998; see also Hoon et al., Cell 96:541-551 (1999)).<br>
Figure 2 is a western blot showing GPCR-B3 protein expression in taste buds but not in non-taste tissue. Using PCR assays, the following non-tongue tissues were screened for GPCR-B3 expression-brain, liver, olfactory epithelium, VNO, and heart. GPCR-B3 was expressed only in taste tissue (data not shown).<br>
7<br><br>
Figure 3 shows in situ hybridization of tongue tissue sections showing labeling of GPCR-B3 in taste receptor cells of taste buds, but not in adjacent non-taste tissue.<br>
Figure 4 shows a chimeric receptor (SEQ ID NO:2), containing the entire extracellular domain of the murine mGIuRl receptor and the transmembrane domain comprising seven transmembrane regions and corresponding cytosolic loops, and C-terminal end from murine GPCR-B3.<br>
Figure 5 shows HEK cells transfected with the chimeric glutamate/GPCR-B3 receptor described in Figure 4. Figure 5 shows calcium response to glutamate, demonstrating robust coupling of the chimeric receptor to phospholipase C. These results indicate that the chimeric glutamate/GPCR-B3 can couple to the promiscuous G protein G? 15 and trigger calcium responses that are detectable using the indicator Fura-2.<br>
DETAILED DESCRIPTION OF THE INVENTION I. Introduction<br>
The present invention provides for the first time nucleic acids encoding a taste cell specific G-protein coupled receptor. These nucleic acids and the receptors that they encode are referred to as "GPCR" for G-protein coupled receptor, and are designated as GPCR-B3. These taste cell specific GPCR are components of the taste transduction pathway. These nucleic acids provide valuable probes for the identification of taste cells, as the nucleic acids are specifically expressed in taste cells. For example, probes for GPCR polypeptides and proteins can be used to identity subsets of taste cells such as foliate cells and circumvallate cells, or specific taste receptor cells, e.g., sweet, sour, salty, and bitter. They also serve as tools for the generation of taste topographic maps that elucidate the relationship between the taste cells of the tongue and taste sensory neurons leading to taste centers in the brain. Furthermore, the nucleic acids and the proteins they encode can be used as probes to dissect taste-induced behaviors.<br>
The invention also provides methods of screening for modulators, e.g., activators, inhibitors, stimulators, enhancers, agonists, and antagonists, of these novel taste cell GPCRs. Such modulators of taste transduction are useful for pharmacological and genetic modulation of taste signaling pathways. These methods of screening can be used to identify high affinity agonists and antagonists of taste cell activity. These modulatory compounds can then be used in the food and pharmaceutical industries to<br>
8<br><br>
customize taste. Thus, the invention provides assays for taste modulation, where GPCR-B3 acts as an direct or indirect reporter molecule for the effect of modulators on taste transduction. GPCRs can be used in assays, e.g., to measure changes in ion concentration, membrane potential, current flow, ion flux, transcription, signal transduction, receptor-ligand interactions, second messenger concentrations, in vitro, in vivo, and ex vivo. In one embodiment, GPCR-B3 can be used as an indirect reporter via attachment to a second reporter molecule such as green fluorescent protein (see, e.g., Mistili &amp; Spector, Nature Biotechnology 15:961-964(1997)). In another embodiment, GPCR-B3 is recombinantly expressed in cells, and modulation of taste transduction via GPCR activity is assayed by measuring changes in Ca2+ levels.<br>
Methods of assaying for modulators of taste transduction include in vitro ligand binding assays using GPCR-B3, portions thereof such as the extracellular domain, or chimeric proteins comprising one or more domains of GPCR-B3, oocyte GPCR-B3 expression; tissue culture cell GPCR-B3 expression; transcriptional activation of GPCR-B3; phosphorylation and dephosphorylation of GPCRs; G-protein binding to GPCRs; ligand binding assays; voltage, membrane potential and conductance changes; ion flux assays; changes in intracellular second messengers such as cAMP and inositol triphosphate; changes in intracellular calcium levels; and neurotransmitter release.<br>
Finally, the invention provides for methods of detecting GPCR-B3 nucleic acid and protein expression, allowing investigation of taste transduction regulation and specific identification of taste receptor cells. GPCR-B3 also provides useful nucleic acid probes for paternity and forensic investigations. GPCR-B3 is a useful nucleic acid probe for identifying subpopulations of taste receptor cells such as foliate, fungiform, and circumvallate taste receptor cells. GPCR-B3 receptors can also be used to generate monoclonal and polyclonal antibodies useful for identifying taste receptor cells. Taste receptor cells can be identified using techniques such as reverse transcription and amplification of mRNA, isolation of total RNA or poly A+ RNA, northern blotting, dot blotting, in situ hybridization, RNase protection, SI digestion, probing DNA microchip arrays, western blots, and the like.<br>
Functionally, GPCR-B3 represents a seven transmembrane G-protein coupled receptor involved in taste transduction, which interacts with a G-protein to mediate taste signal transduction (see, e.g., Fong, Cell Signal 8:217 (1996); Baldwin, Curr. Opin. Cell Biol 6:180(1994)).<br>
9<br><br>
Structurally, the nucleotide sequence of GPCR-B3 (see, e.g., SEQ ID NOS:4-6, isolated from rat, mouse, and human respectively) encodes a polypeptide of approximately 840 amino acids with a predicted molecular weight of approximately 97 kDa and a predicted range of 92-102 kDa (see, e.g., SEQ ID NOS:l-3). Related GPCR-B3 genes from other species share at least about 70% amino acid identity over a amino acid region at least about 25 amino acids in length, optionally 50 to 100 amino acids in length. GPCR-B3 is specifically expressed in foliate and fungiform cells, with lower expression in circumvallate taste receptor cells of the tongue. GPCR-B3 is an moderately rare sequence found in approximately 1 /150,000 cDNAs from an oligo-dT primed circumvallate cDNA library (see Example 1).<br>
The present invention also provides polymorphic variants of the GPCR-B3 depicted in SEQ ID NO:1: variant #1, in which an isoleucine residue is substituted for a leucine acid residue at amino acid position 33; variant #2, in which an aspartic acid residue is substituted for a glutamic acid residue at amino acid position 84; and variant #3, in which a glycine residue is substituted for an alanine residue at amino acid position 90.<br>
Specific regions of the GPCR-B3 nucleotide and amino acid sequence may be used to identify polymorphic variants, interspecies homologs, and alleles of GPCR-B3. This identification can be made in vitro, e.g., under stringent hybridization conditions or PCR (using primers encoding SEQ ID NOS:7-8) and sequencing, or by using the sequence information in a computer system for comparison with other nucleotide sequences. Typically, identification of polymorphic variants and alleles of GPCR-B3 is made by comparing an amino acid sequence of about 25 amino acids or more, e.g., 50-100 amino acids. Amino acid identity of approximately at least 70% or above, optionaliy 80%, optionally 90-95% or above typically demonstrates that a protein is a polymorphic variant, interspecies homolog, or allele of GPCR-B3. Sequence comparison can be performed using any of the sequence comparison algorithms discussed below. Antibodies that bind specifically to GPCR-B3 or a conserved region thereof can also be used to identify alleles, interspecies homologs, and polymorphic variants.<br>
Polymorphic variants, interspecies homologs, and alleles of GPCR-B3 are confirmed by. examining taste cell specific expression of the putative GPCR-B3 polypeptide. Typically, GPCR-B3 having the amino acid sequence of SEQ ID NO: 1-3 is used as a positive control in comparison to the putative GPCR-B3 protein to demonstrate the identification of a polymorphic variant or allele of GPCR-B3. The polymorphic<br>
10<br><br>
variants, alleles and interspecies homologs are expected to retain the seven transmembrane structure of a G-protein coupled receptor.<br>
GPCR-B3 nucleotide and amino acid sequence information may also be used to construct models of taste cell specific polypeptides in a computer system. These models are subsequently used to identify compounds that can activate or inhibit GPCR-B3. Such compounds that modulate the activity of GPCR B4 can be used to investigate the role of GPCR-B3 in taste transduction.<br>
The isolation of GPCR-B3 for the first time provides a means for assaying for inhibitors and activators of G-protein coupled receptor taste transduction . Biologically active GPCR-B3 is useful for testing inhibitors and activators of GPCR-B3 as taste transducers using in vivo and in vitro expression that measure, e.g., transcriptional activation of GPCR-B3; ligand binding; phosphorylation and dephosphorylation; binding to G-proteins; G-protein activation; regulatory molecule binding; voltage, membrane potential and conductance changes; ion flux; intracellular second messengers such as cAMP and inositol triphosphate; intracellular calcium levels; and neurotransmitter release. Such activators and inhibitors identified using GPCR-B3, can be used to further study taste transduction and to identify specific taste agonists and antagonists. Such activators and inhibitors are useful as pharmaceutical and food agents for customizing taste.<br>
Methods of detecting GPCR-B3 nucleic acids and expression of GPCR-B3 are also useful for identifying taste cells and creating topological maps of the tongue and the relation of tongue taste receptor cells to taste sensory neurons in the brain. Chromosome localization of the genes encoding human GPCR-B3 can be used to identify diseases, mutations, and traits caused by and associated with GPCR-B3.<br>
II. Definitions<br>
As used herein, the following terms have the meanings ascribed to them unless specified otherwise.<br>
"Taste receptor cells" are neuroepithelial cells that are organized into groups to form taste buds of the tongue, e.g., foliate, fungiform, and circumvallate cells (see, e.g., Roper et al., Ann, Rev. Neuroscl. 12:329-353 (1989)).<br>
"GPCR-B3," also called "TR1," refers to a G-protein coupled receptor is specifically expressed in taste receptor cells such as foliate, fungiform, and circumvallate cells (see, e.g., Hoon et al., Cell 96:541-551 (1999), herein incorporated by reference in<br>
11<br><br>
its entirety). Such taste cells can be identified because they express specific molecules such as Gustducin, a taste cell specific G protein (McLaughin et al., Nature 357:563-569 (1992)). Taste receptor cells can also be identified on the basis of morphology (see, e.g., Roper, supra).<br>
GPCR-B3 encodes GPCRs with seven transmembrane regions that have "G-protein coupled receptor activity," e.g., they bind to G-proteins in response to extracellular stimuli and promote production of second messengers such as IP3, cAMP, and Ca 2+ via stimulation of enzymes such as phospholipase C and adenylate cyclase (for a description of the structure and function of GPCRs, see, e.g., Fong, supra, and Baldwin, supra).<br>
The term GPCR-B3 therefore refers to polymorphic variants, alleles, mutants, and interspecies homologs that: (1) have about 70% amino acid sequence identity, optionally about 75, 80, 85, 90, or 95% amino acid sequence identity to SEQ ID NOS:l-3 over a window of about 25 amino acids, optionally 50-100 amino acids; (2) bind to antibodies raised against an immunogen comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 1-3 and conservatively modified variants thereof; (3) specifically hybridize (with a size of at least about 500, optionally at least about 900 nucleotides) under stringent hybridization conditions to a sequence selected from the group consisting of SEQ ID NO:4-6, and conservatively modified variants thereof; or (4) are amplified by primers that specifically hybridize under stringent hybridization conditions to the same sequence as a degenerate primer sets encoding SEQ ID NOS:7-8.<br>
Topologically, sensory GPCRs have an N-terminal "extracellular domain," a "transmembrane domain" comprising seven transmembrane regions and corresponding cytoplasmic and extracellular loops, and a C-terminal "cytoplasmic domain" (see Figure 1; see also Hoon et al, Cell 96:541-551 (1999); Buck &amp; Axel, Cell 65:175-187 (1991)). These domains can be structurally identified using methods known to those of skill in the art, such as sequence analysis programs that identify hydrophobic and hydrophilic domains (see, e.g., Kyte &amp; Doolittle, J. Mot. Biol. 157:105-132 (1982)). Such domains are useful for making chimeric proteins and for in vitro assays of the invention.<br>
"Extracellular domain" therefore refers to the domain of GPCR-B3 that protrudes from the cellular membrane and binds to extracellular ligand. This region starts at the N-terminus and ends approximately at the conserved glutamic acid at amino acid position 563 plus or minus approximately 20 amino acids. The region corresponding to<br>
12<br><br>
amino acids 1-580 of SEQ ID NO:1 (nucleotides 1-1740, with nucleotide 1 starting at the ATG initiator methionine codon; see also Figure 1) is one embodiment of an extracellular domain that extends slightly into the transmembrane domain. This embodiment is useful for in vitro ligand binding assays, both soluble and solid phase.<br>
"Transmembrane domain," comprising seven transmembrane regions plus the corresponding cytoplasmic and extracellular loops, refers to the domain of GPCR-B3 that starts approximately at the conserved glutamic acid residue at amino acid position 563 plus or minus approximately 20 amino acids and ends approximately at the conserved tyrosine amino acid residue at position 812 plus or minus approximately 10 amino acids.<br>
"Cytoplasmic domain" refers to the domain of GPCR-B3 that starts at the conserved tyrosine amino acid residue at position 812 plus or minus approximately 10 amino acids and continues to the C-terminus of the polypeptide.<br>
"Biological sample" as used herein is a sample of biological tissue or fluid that contains GPCR-B3 or nucleic acid encoding GPCR-B3 protein. Such samples include, but are not limited to, tissue isolated from humans, mice, and rats, in particular, ton. Biological samples may also include sections of tissues such as frozen sections taken for histological purposes. A biological sample is typically obtained from a eukaryotic organism, such as insects, protozoa, birds, fish, reptiles, and preferably a mammal such as rat, mouse, cow, dog, guinea pig, or rabbit, and most preferably a primate such as chimpanzees or humans. Tissues include tongue tissue, isolated taste buds, and testis tissue.<br>
"GPCR activity" refers to the ability of a GPCR to transduce a signal. Such activity can be measured in a heterologous cell, by coupling a GPCR (or a chimeric GPCR) to either a G-protein or promiscuous G-protein such as Gal5, and an enzyme such as PLC, and measuring increases in intracellular calcium using (Offermans &amp; Simon, J. Biol. Chem. 270:15175-15180 (1995)). Receptor activity can be effectively measured by recording ligand-induced changes in [Ca2+] using fluorescent Ca2+-indicator dyes and fluorometric imaging. Optionally, the polypeptides of the invention are involved in sensory transduction, optionally taste transduction in taste cells.<br>
The phrase "functional effects" in the context of assays for testing compounds that modulate GPCR-B3 mediated taste transduction includes the determination of any parameter that is indirectly or directly under the influence of the receptor, e.g., functional, physical and chemical effects. It includes ligand binding,<br>
13<br><br>
changes in ion flux, membrane potential, current flow, transcription, G-protein binding, GPCR phosphorylation or dephosphorylation, signal transduction, receptor-ligand interactions, second messenger concentrations (e.g., cAMP, IP3, or intracellular Ca2+), in vitro, in vivo, and ex vivo and also includes other physiologic effects such increases or decreases of neurotransmitter or hormone release.<br>
By "determining the functional effect" is meant assays for a compound that increases or decreases a parameter that is indirectly or directly under the influence of GPCR-B3, e.g., functional, physical and chemical effects. Such functional effects can be measured by any means known to those skilled in the art, e.g., changes in spectroscopic characteristics (e.g., fluorescence, absorbance, refractive index), hydrodynamic (e.g., shape), chromatographic, or solubility properties, patch clamping, voltage-sensitive dyes, whole cell currents, radioisotope efflux, inducible markers, oocyte GPCR-B3 expression; tissue culture cell GPCR-B3 expression; transcriptional activation of GPCR-B3; ligand binding assays; voltage, membrane potential and conductance changes; ion flux assays; changes in intracellular second messengers such as cAMP and inositol triphosphate (IP3); changes in intracellular calcium levels; neurotransmitter release, and the like.<br>
"Inhibitors," "activators," and "modulators" of GPCR-B3 are used interchangeably to refer to inhibitory, activating, or modulating molecules identified using in vitro and in vivo assays for taste transduction, e.g., ligands, agonists, antagonists, and their homologs and mimetics. Inhibitors are compounds that, e.g., bind to, partially or totally block stimulation, decrease, prevent, delay activation, inactivate, desensitize, or down regulate taste transduction, e.g., antagonists. Activators are compounds that, e.g., bind to, stimulate, increase, open, activate, facilitate, enhance activation, sensitize or up regulate taste transduction, e.g., agonists. Modulators include compounds that, e.g., alter the interaction of a receptor with: extracellular proteins that bind activators or inhibitor (e.g., ebnerin and other members of the hydrophobic carrier family); G -proteins; kinases (e.g., homologs of rhodopsin kinase and beta adrenergic receptor kinases that are involved in deactivation and desensitization of a receptor); and arrestin-like proteins, which also deactivate and desensitize receptors. Modulators include genetically modified versions of GPCR-B3, e.g., with altered activity, as well as naturally occurring and synthetic ligands, antagonists, agonists, small chemical molecules and the like. Such assays for inhibitors and activators include, e.g., expressing GPCR-B3 in cells or cell membranes, applying putative modulator compounds, and then determining the functional effects on taste transduction, as described above. Samples or assays comprising GPCR-<br>
14<br><br>
B3 that are treated with a potential activator, inhibitor, or modulator are compared to control samples without the inhibitor, activator, or modulator to examine the extent of inhibition. Control samples (untreated with inhibitors) are assigned a relative GPCR-B3 activity value of 100%. Inhibition of GPCR-B3 is achieved when the GPCR-B3 activity value relative to the control is about 80%, optionally 50%, 25-0%. Activation of GPCR-B3 is achieved when the GPCR-B3 activity value relative to the control is 110%, optionally 150%, 200-500%, 1000-3000% higher.<br>
"Biologically active" GPCR-B3 refers to GPCR-B3 having GPCR activity as described above, involved in taste transduction in taste receptor cells.<br>
The terms "isolated" "purified" or "biologically pure" refer to material that is substantially or essentially free from components which normally accompany it as found in its native state. Purity and homogeneity are typically determined using analytical chemistry techniques such as polyaerylamide gel electrophoresis or high performance liquid chromatography. A protein that is the predominant species present in a preparation is substantially purified. In particular, an isolated GPCR-B3 nucleic acid is separated from open reading frames that flank the GPCR-B3 gene and encode proteins other than GPCR-B3. The term "purified" denotes that a nucleic acid or protein gives rise to essentially one band in an electrophoretic gel. Particularly, it means that the nucleic acid or protein is at least 85% pure, optionally at least 95% pure, and optionally at least 99% pure.<br>
"Nucleic acid" refers to deoxyribonucleotides or ribonucleotides and polymers thereof in either single or double stranded form. The term encompasses nucleic acids containing known nucleotide analogs or modified backbone residues or linkages, which are synthetic, naturally occurring, and non-naturally occurring, which have similar binding properties as the reference nucleic acid, and which are metabolized in a manner similar to the reference nucleotides. Examples of such analogs include, without limitation, phosphorothioates, phosphoramidates, methyl phosphonates, chiral-methyl phosphonates, 2-O-methyl ribonucleotides, peptide-nucleic acids (PNAs).<br>
Unless otherwise indicated, a particular nucleic acid sequence also implicitly encompasses conservatively modified variants thereof (e.g., degenerate codon substitutions) and complementary sequences, as well as the sequence explicitly indicated. Specifically, degenerate codon substitutions may be achieved by generating sequences in which the third position of one or more selected (or all) codons is substituted with mixed-base and/or deoxyinosine residues (Batzer et al.. Nucleic Acid Res. 19:5081 (1991);<br>
15<br><br>
Ohtsuka et al, J. Biol Chem. 260:2605-2608 (1985); Rossolini et al., Mol Cell Probes 8:91-98 (1994)). The term nucleic acid is used interchangeably with gene, cDNA, mRNA, oligonucleotide, and polynucleotide.<br>
The terms "polypeptide," "peptide" and "protein" are used interchangeably herein to refer to a polymer of amino acid residues. The terms apply to amino acid polymers in which one or more amino acid residue is an artificial chemical mimetic of a corresponding naturally occurring amino acid, as well as to naturally occurring amino acid polymers and non-naturally occurring amino acid polymer.<br>
The term "amino acid" refers to naturally occurring and synthetic amino acids, as well as amino acid analogs and amino acid mimetics that function in a manner similar to the naturally occurring amino acids. Naturally occurring amino acids are those encoded by the genetic code, as well as those amino acids that are later modified, e.g., hydroxyproline, y-carboxyglutamate, and O-phosphoserine. Amino acid analogs refers to compounds that have the same basic chemical structure as a naturally occurring amino acid, i.e., an a carbon that is bound to a hydrogen, a carboxyl group, an amino group, and an R group, e.g., homoserine, norleucine, methionine sulfoxide, methionine methyl sulfonium. Such analogs have modified R groups (e.g., norleucine) or modified peptide backbones, but retain the same basic chemical structure as a naturally occurring amino acid. Amino acid mimetics refers to chemical compounds that have a structure that is different from the general chemical structure of an amino acid, but that functions in a manner similar to a naturally occurring amino acid.<br>
Amino acids may be referred to herein by either their commonly known three letter symbols or by the one-letter symbols recommended by the IUPAC-IUB Biochemical Nomenclature Commission. Nucleotides, likewise, may be referred to by their commonly accepted single-letter codes.<br>
"Conservatively modified variants" applies to both amino acid and nucleic acid sequences. With respect to particular nucleic acid sequences, conservatively modified variants refers to those nucleic acids which encode identical or essentially identical amino acid sequences, or where the nucleic acid does not encode an amino acid sequence, to essentially identical sequences. Because of the degeneracy of the genetic code, a large number of functionally identical nucleic acids encode any given protein. For instance, the codons GCA, GCC, GCG and GCU all encode the amino acid alanine. Thus, at every position where an alanine is specified by a codon, the codon can be altered to any of the corresponding codons described without altering the encoded polypeptide.<br>
16<br><br>
Such nucleic acid variations are "silent variations," which are one species of conservatively modified variations. Every nucleic acid sequence herein which encodes a polypeptide also describes every possible silent variation of the nucleic acid. One of skill will recognize that each codon in a nucleic acid (except AUG, which is ordinarily the only codon for methionine, and TGG, which is ordinarily the only codon for tryptophan) can be modified to yield a functionally identical molecule. Accordingly, each silent variation of a nucleic acid which encodes a polypeptide is implicit in each described sequence.<br>
As to amino acid sequences, one of skill will recognize that individual substitutions, deletions or additions to a nucleic acid, peptide, polypeptide, 'or protein sequence which alters, adds or deletes a single amino acid or a small percentage of amino acids in the encoded sequence is a "conservatively modified variant" where the alteration results in the substitution of an amino acid with a chemically similar amino acid. Conservative substitution tables providing functionally similar amino acids are well known in the art. Such conservatively modified variants are in addition to and do not exclude polymorphic variants, interspecies homologs, and alleles of the invention.<br>
The following eight groups each contain amino acids that are conservative substitutions for one another:<br>
1)	Alanine (A), Glycine (G);<br>
2)	Aspartic acid (D), Glutamic acid (E);<br>
3)	Asparagine (N), Glutamine (Q);<br>
4)	Arginine (R), Lysine (K);<br>
5)	Isoleucine (I), Leucine (L), Methionine (M), Valine (V);<br>
6)	Phenylalanine (F), Tyrosine (Y), Tryptophan (W);<br>
7)	Serine (S), Threonine (T); and<br>
8)	Cysteine (C), Methionine (M)<br>
(see, e.g., Creighton, Proteins (1984)).<br>
Macromolecular structures such as polypeptide structures can be described in terms of various levels of organization. For a general discussion of this organization, see, e.g., Alberts et al, Molecular Biology of the Cell (3rd ed., 1994) and Cantor and Schimmel, Biophysical Chemistry Part I: The Conformation of Biological Macromolecules (1980). "Primary structure" refers to the amino acid sequence of a particular peptide. "Secondary structure" refers to locally ordered, three dimensional structures within a polypeptide. These structures are commonly known as domains.<br>
17<br><br>
Domains are portions of a polypeptide that form a compact unit of the polypeptide and are typically 50 to 350 amino acids long. Typical domains are made up of sections of lesser organization such as stretches of p-sheet and a-helices. 'Tertiary structure" refers to the complete three dimensional structure of a polypeptide monomer. "Quaternary structure" refers to the three dimensional structure formed by the noncovalent association of independent tertiary units.   Anisotropic terms are also known as energy terms. A "label" or a "detectable moiety" is a composition detectable by spectroscopic, photochemical, biochemical, immunochemical, or chemical means. For example, useful labels include 32P, fluorescent dyes, electron-dense reagents, enzymes (e.g., as commonly used in an ELISA), biotin, digoxigenin, or haptens and proteins for which ant or 7 can be made detectable, e.g., by incorporating a radiolabel into the peptide, and used to detect antibodies specifically reactive with the peptide).<br>
A "labeled nucleic acid probe or oligonucleotide" is one that is bound, either covalently, through a linker or a chemical bond, or noncovalently, through ionic, van der Waals, electrostatic, or hydrogen bonds to a label such that the presence of the probe may be detected by detecting the presence of the label bound to the probe.<br>
As used herein a "nucleic acid probe or oligonucleotide" is defined as a nucleic acid capable of binding to a target nucleic acid of complementary sequence through one or more types of chemical bonds, usually through complementary base pairing, usually through hydrogen bond formation. As used herein, a probe may include natural (i.e., A, G, C, or T) or modified bases (7-deazaguanosine, inosine, etc.). In addition, the bases in a probe may be joined by a linkage other than a phosphodiester bond, so long as it does not interfere with hybridization. Thus, for example, probes may be peptide nucleic acids in which the constituent bases are joined by peptide bonds rather than phosphodiester linkages. It will be understood by one of skill in the art that probes may bind target sequences lacking complete complementarity with the probe sequence depending upon the stringency of the hybridization conditions. The probes are optionally directly labeled as with isotopes, chromophores, lumiphores, chromogens, or indirectly labeled such as with biotin to which a streptavidin complex may later bind. By assaying for the presence or absence of the probe, one can detect the presence or absence of the select sequence or subsequence.<br>
The term "recombinant" when used with reference, e.g., to a cell, or nucleic acid, protein, or vector, indicates that the cell, nucleic acid, protein or vector, has<br>
18<br><br>
been modified by the introduction of a heterologous nucleic acid or protein or the alteration of a native nucleic acid or protein, or that the cell is derived from a cell so modified. Thus, for example, recombinant cells express genes that are not found within the native (non-recombinant) form of the cell or express native genes that are otherwise abnormally expressed, under expressed or not expressed at all.<br>
The term "heterologous" when used with reference to portions of a nucleic acid indicates that the nucleic acid comprises two or more subsequences that are not found in the same relationship to each other in nature. For instance, the nucleic acid is typically recombinantly produced, having two or more sequences from unrelated genes arranged to make a new functional nucleic acid, e.g., a promoter from one source and a coding region from another source. Similarly, a heterologous protein indicates that the protein comprises two or more subsequences that are not found in the same relationship to each other in nature (e.g., a fusion protein).<br>
A "promoter" is defined as an array of nucleic acid control sequences that direct transcription of a nucleic acid. As used herein, a promoter includes necessary nucleic acid sequences near the start site of transcription, such as, in the case of a polymerase II type promoter, a TATA element. A promoter also optionally includes distal enhancer or repressor elements, which can be located as much as several thousand base pairs from the start site of transcription. A "constitutive" promoter is a promoter that is active under most environmental and developmental conditions. An "inducible" promoter is a promoter that is active under environmental or developmental regulation. The term "operably linked" refers to a functional linkage between a nucleic acid expression control sequence (such as a promoter, or array of transcription factor binding sites) and a second nucleic acid sequence, wherein the expression control sequence directs transcription of the nucleic acid corresponding to the second sequence.<br>
An "expression vector" is a nucleic acid construct, generated recombinantly or synthetically, with a series of specified nucleic acid elements that permit transcription of a particular nucleic acid in a host cell. The expression vector can be part of a plasmid, virus, or nucleic acid fragment. Typically, the expression vector includes a nucleic acid to be transcribed operably linked to a promoter.<br>
The terms "identical" or percent "identity," in the context of two or more nucleic acids or polypeptide sequences, refer to two or more sequences or subsequences that are the same or have a specified percentage of amino acid residues or nucleotides that are the same (i.e., 70% identity, optionally 75%, 80%, 85%, 90%, or 95% identity over a<br>
19<br><br>
specified region), when compared and aligned for maximum correspondence over a comparison window, or designated region as measured using one of the following sequence comparison algorithms or by manual alignment and visual inspection. Such sequences are then said to be "substantially identical." This definition also refers to the compliment of a test sequence. Optionally, the identity exists over a region that is at least about 50 amino acids or nucleotides in length, or more preferably over a region that is 75-100 amino acids or nucleotides in length.<br>
For sequence comparison, typically one sequence acts as a reference sequence, to which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are entered into a computer, subsequence coordinates are designated, if necessary., and sequence algorithm program parameters are designated. Default program parameters can be used, or alternative parameters can be designated. The sequence comparison algorithm then calculates the percent sequence identities for the test sequences relative to the reference sequence, based on the program parameters.<br>
A "comparison window", as used herein, includes reference to a segment of any one of the number of contiguous positions selected from the group consisting of from 20 to 600, usually about 50 to about 200, more usually about 100 to about 150 in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned. Methods of alignment of sequences for comparison are well-known in the art. Optimal alignment of sequences for comparison can be conducted, e.g., by the local homology algorithm of Smith &amp; Waterman, Adv. Appl Math. 2:482 (1981), by the homology alignment algorithm of Needleman &amp; Wunsch, J. Mol. Biol 48:443 (1970), by the search for similarity method of Pearson &amp; Lipman, Proc. Nat 1 Acad. Sci. USA 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, WI), or by manual alignment and visual inspection {see, e.g., Current Protocols in Molecular Biology (Ausubel et al., eds. 1995 supplement)).<br>
One example of a useful algorithm is PILEUP. PILEUP creates a multiple sequence alignment from a group of related sequences using progressive, pairwise alignments to show relationship and percent sequence identity. It also plots a tree or dendogram showing the clustering relationships used to create the alignment. PILEUP uses a simplification of the progressive alignment method of Feng &amp; Doolittle, J. Mol.<br>
20<br><br>
Evol. 35:351-360 (1987). The method used is similar to the method described by Higgins &amp; Sharp, CABIOS 5:151-153 (1989). The program can align up to 300 sequences, each of a maximum length of 5,000 nucleotides or ammo acids. The multiple alignment procedure begins with the pairwise alignment of the two most similar sequences, producing a cluster of two aligned sequences.  This cluster is then aligned to the next most related sequence or cluster of aligned sequences. Two clusters of sequences are aligned by a simple extension of the pairwise alignment of two individual sequences. The final alignment is achieved by a series of progressive, pairwise alignments. The program is run by designating specific sequences and their amino acid or nucleotide coordinates for regions of sequence comparison and by designating the program parameters. Using P1LEUP, a reference sequence is compared to other test sequences to determine the percent sequence identity relationship using the following parameters: default gap weight (3.00), default gap length weight (0.10), and weighted end gaps. PILEUP can be obtained from the GCG sequence analysis software package, e.g., version 7.0 (Devereaux et al, Nuc. Acids Res. 12:387-395 (1984).<br>
Another example of algorithm that is suitable for determining percent sequence identity and sequence similarity are the BLAST and BLAST 2.0 algorithms, which are described in Altschul et al, Nuc. Acids Res. 25:3389-3402 (1977) and Altschul et al, J. Mol Biol. 215:403-410 (1990), respectively. Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al, supra). These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always &gt; 0) and N (penalty score for mismatching residues; always 
21<br><br>
sequence is reached. The BLAST algorithm parameters W, T, and X determine the sensitivity and speed of the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (W) of 11, an expectation (E) or 10, M=5, N=-4 and a comparison of both strands. For amino acid sequences, the BLASTP program uses as defaults a wordlength of 3, and expectation (E) of 10, and the BLOSUM62 scoring matrix (see Henikoff &amp; Henikoff, Proc. Natl. Acad. Set USA 89:10915 (1989)) alignments (B) of 50, expectation (E) of 10, M=5, N=-4, and a comparison of both strands.<br>
The BLAST algorithm also performs a statistical analysis of the similarity between two sequences {see, e.g., Karlin &amp; Altschul, Proc. Nat 7. Acad. Sci. USA 90:5873-5787 (1993)). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a reference sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.2, more preferably less than about 0.01., and most preferably less than about 0.001.<br>
An indication that two nucleic acid sequences or polypeptides are substantially identical is that the polypeptide encoded by the first nucleic acid is   immunologically cross reactive with the antibodies raised against the polypeptide encoded by the second nucleic acid, as described below. Thus, a polypeptide is typically substantially identical to a second polypeptide, for example, where the two peptides differ only by conservative substitutions. Another indication that two nucleic acid sequences are substantially identical is that the two molecules or their complements hybridize to each other under stringent conditions, as described below. Yet another indication that two nucleic acid sequences are substantially identical is that the same primers can be used to amplify the sequence.<br>
The phrase "selectively (or specifically) hybridizes to" refers to the binding, duplexing, or hybridizing of a molecule only to a particular nucleotide sequence under stringent hybridization conditions when that sequence is present in a complex mixture (e.g., total cellular or library DNA or RNA).<br>
The phrase "stringent hybridization conditions" refers to conditions under which a probe will hybridize to its target subsequence, typically in a complex mixture of nucleic acid, but to no other sequences. Stringent conditions are sequence-dependent and will be different in different circumstances. Longer sequences hybridize specifically at<br>
22<br><br>
higher temperatures. An extensive guide to the hybridization of nucleic acids is found in Tijssen, Techniques in Biochemistry and Molecular Biology--Hybridization with Nucleic Probes, "Overview of principles of hybridization and the strategy of nucleic acid assays" (1993). Generally, stringent conditions are selected to be about 5-10C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength pH. The Tm is the temperature (under defined ionic strength, pH, and nucleic concentration) at which 50% of the probes complementary to the target hybridize to the target sequence at equilibrium (as the target sequences are present in excess, at Tm, 50% of the probes are occupied at equilibrium). Stringent conditions will be those in which the salt concentration is less than about 1.0 M sodium ion, typically about 0.01 to 1.0 M sodium ion concentration (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30C for short probes (e.g., 10 to 50 nucleotides) and at least about 60C for long probes (e.g., greater than 50 nucleotides). Stringent conditions may also be achieved with the addition of destabilizing agents such as formamide. For selective or specific hybridization, a positive signal is at least two times background, optionally 10 times background hybridization. Exemplary stringent hybridization conditions can be as following: 50% formamide, 5x SSC, and 1% SDS, incubating at 42C, or, 5x SSC, 1% SDS, incubating at 65C, with wash in 0.2x SSC, and 0.1% SDS at 65C.<br>
Nucleic acids that do not hybridize to each other under stringent conditions are still substantially identical if the polypeptides which they encode are substantially identical. This occurs, for example, when a copy of a nucleic acid is created using the maximum codon degeneracy permitted by the genetic code. In such cases, the nucleic acids typically hybridize under moderately stringent hybridization conditions. Exemplary "moderately stringent hybridization conditions" include a hybridization in a buffer of 40% formamide, 1 M NaCl, 1% SDS at 37C, and a wash in IX SSC at 45C. A positive hybridization is at least twice background. Those of ordinary skill will readily recognize that alternative hybridization and wash conditions can be utilized to provide conditions of similar stringency.<br>
"Antibody" refers to a polypeptide comprising a framework region from an immunoglobulin gene or fragments thereof that specifically binds and recognizes an antigen. The recognized immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon, and mu constant region genes, as well as the myriad immunoglobulin variable region genes. Light chains are classified as either kappa or<br>
23<br><br>
lambda. Heavy chains are classified as gamma, mu, alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG, IgM, IgA, IgD and IgE, respectively.<br>
An exemplary immunoglobulin (antibody) structural unit comprises a tetramer. Each tetramer is composed of two identical pairs of polypeptide chains, each pair having one "light" (about 25 kDa) and one "heavy" chain (about 50-70 kDa). The N-terminus of each chain defines a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. The terms variable light chain (VL) and variable heavy chain (VH) refer to these light and heavy chains respectively.<br>
Antibodies exist, e.g., as intact immunoglobulins or as a number of well-characterized fragments produced by digestion with various peptidases. Thus, for example, pepsin digests an antibody below the disulfide linkages in the hinge region to produce F(ab)'2. a dimer of Fab which itself is a light chain joined to VH-CH1 by a disulfide bond. The F(ab)'2 may be reduced under mild conditions to break the disulfide linkage in the hinge region, thereby converting the F(ab)'2 dimer into an Fab' monomer. The Fab' monomer is essentially Fab with part of the hinge region (see Fundamental Immunology (Paul ed., 3d ed. 1993). While various antibody fragments are defined in terms of the digestion of an intact antibody, one of skill will appreciate that such fragments may be synthesized de novo either chemically or by using recombinant DNA methodology. Thus, the term antibody, as used herein, also includes antibody fragments either produced by the modification of whole antibodies, or those synthesized de novo using recombinant DNA methodologies (e.g., single chain Fv) or those identified using phage display libraries (see, e.g., McCafferty et al, Nature 348:552-554 (1990)).<br>
For preparation of monoclonal or polyclonal antibodies, any technique known in the art can be used (see, e.g., Kohler &amp; Milstein, Nature 256:495-497 (1975); Kozbor et al., Immunology Today 4: 72 (1983); Cole et al., pp. 77-96 in Monoclonal Antibodies and Cancer Therapy (1985)). Techniques for the production of single chain antibodies (U.S. Patent 4,946,778) can be adapted to produce antibodies to polypeptides of this invention. Also, transgenic mice, or other organisms such as other mammals, may be used to express humanized antibodies. Alternatively, phage display technology can be used to identify antibodies and heteromeric Fab fragments that.specifically bind to selected antigens (see, e.g., McCafferty et al, Nature 348:552-554 (1990); Marks et al., Biotechnology 10:779-783 (1992)).<br>
A "chimeric antibody" is an antibody molecule in which (a) the constant region, or a portion thereof, is altered, replaced or exchanged so that the antigen binding<br>
24<br><br>
site (variable region) is linked to a constant region of a different or altered class, effector function and/or species, or an entirely different molecule which confers new properties to the chimeric antibody, e.g., an enzyme, toxin, hormone, growth factor, drug, etc.; or (b) the variable region, or a portion thereof, is altered, replaced or exchanged with a variable region having a different or altered antigen specificity.<br>
An "anti-GPCR-B3" antibody is an antibody or antibody fragment that specifically binds a polypeptide encoded by the GPCR-B3 gene, cDNA, or a subsequence thereof.<br>
The term "immunoassay" is an assay that uses an antibody to specifically bind an antigen. The immunoassay is characterized by the use of specific binding properties of a particular antibody to isolate, target, and/or quantify the antigen.<br>
The phrase "specifically (or selectively) binds" to an antibody or "specifically (or selectively) immunoreactive with," when referring to a protein or peptide, refers to a binding reaction that is determinative of the presence of the protein in a heterogeneous population of proteins and other biologics. Thus, under designated immunoassay conditions, the specified antibodies bind to a particular protein at least two times the background and do not substantially bind in a significant amount to other proteins present in the sample. Specific binding to an antibody under such conditions may require an antibody that is selected for its specificity for a particular protein. For example, polyclonal antibodies raised to GPCR-B3 from specific species such as rat, mouse, or human can be selected to obtain only those polyclonal antibodies that are specifically immunoreactive with GPCR-B3 and not with other proteins, except for polymorphic variants and alleles of GPCR-B3. This selection may be achieved by subtracting out antibodies that cross-react with GPCR-B3 molecules from other species. A variety of immunoassay formats may be used to select antibodies specifically immunoreactive with a particular protein. For example, solid-phase ELISA immunoassays are routinely used to select antibodies specifically immunoreactive with a protein {see, e.g., Harlow &amp; Lane, Antibodies, A Laboratory Manual (1988), for a description of immunoassay formats and conditions that can be used to determine specific immunoreactivity). Typically a specific or selective reaction will be at least twice background signal or noise and more typically more than 10 to 100 times background.<br>
The phrase "selectively associates with" refers to the ability of a nucleic acid to "selectively hybridize" with another as defined above, or the ability of an antibody to "selectively (or specifically) bind to a protein, as defined above.<br>
25<br><br>
By "host cell" is meant a cell that contains an expression vector and supports the replication or expression of the expression vector. Host cells may be prokaryotic cells such as E. coli, or eukaryotic cells such as yeast, insect, amphibian, or mammalian cells such as CHO, HeLa and the like, e.g., cultured cells, explants, and cells in vivo.<br>
III. Isolation of the nucleic acid encoding GPCR-B3<br>
A. General recombinanl DNA methods<br>
This invention relies on routine techniques in the field of recombinant genetics. Basic texts disclosing the general methods of use in this invention include Sambrook et al., Molecular Cloning, A Laboratory Manual (2nd ed. 1989); Kriegler, Gene Transfer and Expression: A Laboratory Manual (1990); and Current Protocols in Molecular Biology (Ausubel et al., eds., 1994)).<br>
For nucleic acids, sizes are given in either kilobases (kb) or base pairs (bp). These are estimates derived from agarose or acrylamide gel electrophoresis, from sequenced nucleic acids, or from published DNA sequences. For proteins, sizes are given in kilodaltons (kDa) or amino acid residue numbers. Proteins sizes are estimated from gel electrophoresis, from sequenced proteins, from derived amino acid sequences, or from published protein sequences.<br>
Oligonucleotides that are not commercially available can be chemically synthesized according to the solid phase phosphoramidite triester method first described by Beaucage&amp; Caruthers, Tetrahedron Letts. 22:1859-1862 (1981), using an automated synthesizer, as described in Van Devanter et. al, Nucleic Acids Res. 12:6159-6168 (1984). Purification of oligonucleotides is by either native acrylamide gel electrophoresis or by anion-exchange HPLC as described in Pearson &amp; Reanier, J, Chrom. 255:137-149 (1983).<br>
The sequence of the cloned genes and synthetic oligonucleotides can be verified after cloning using, e.g., the chain termination method for sequencing double-stranded templates of Wallace et al, Gene 16:21-26 (1981).<br>
B. Cloning methods for the isolation of nucleotide sequences encoding GPCR-B3<br>
In general, the nucleic acid sequences encoding GPCR-B3 and related nucleic acid sequence homologs are cloned from cDNA and genomic DNA libraries by<br>
26<br><br>
hybridization with a probe, or isolated using amplification techniques with oligonucleotide primers. For example., GPCR-B3 sequences are typically isolated from mammalian nucleic acid (genomic or cDNA) libraries by hybridizing with a nucleic acid probe, the sequence of which can be derived from SEQ ID NOS:4-6. A suitable tissue from which GPCR-B3 RNA and cDNA can be isolated is tongue tissue, optionally taste bud tissue or individual taste cells.<br>
Amplification techniques using primers can also be used to amplify and isolate GPCR-B3 from DNA or RNA. The degenerate primers encoding the following amino acid sequences can also be used to amplify a sequence of GPCR-B3: SEQ ID NOS:7-8 (see, e.g., Dieffenfach &amp; Dveksler, PCR Primer: A Laboratory Manual (1995)). These primers can be used, e.g., to amplify either the full length sequence or a probe of one to several hundred nucleotides, which is then used to screen a mammalian library for full-lengthGPCR-B3.<br>
Nucleic acids encoding GPCR-B3 can also be isolated from expression libraries using antibodies as probes. Such polyclonal or monoclonal antibodies can be raised using the sequence of SEQ ID NOS:l-3.<br>
GPCR-B3 polymorphic variants, alleles, and interspecies homologs that are substantially identical to GPCR-B3 can be isolated using GPCR-B3 nucleic acid probes, and oligonucleotides under stringent hybridization conditions, by screening libraries.   Alternatively, expression libraries can be used to clone GPCR-B3 and GPCR-B3 polymorphic variants, alleles, and interspecies homologs, by detecting expressed homologs immunologically with antisera or purified antibodies made against GPCR-B3, which also recognize and selectively bind to the GPCR-B3 homolog.<br>
To make a cDNA library, one should choose a source that is rich in GPCR-B3 mRNA, e.g., tongue tissue, or isolated taste buds. The mRNA is then made into cDNA using reverse transcriptase, ligated into a recombinant vector, and transfected into a recombinant host for propagation, screening and cloning. Methods for making and screening cDNA libraries are well known {see, e.g., Gubler &amp; Hoffman, Gene 25:263-269 (1983); Sambrook et at, supra; Ausubel et al, supra).<br>
For a genomic library, the DNA is extracted from the tissue and either mechanically sheared or enzymatically digested to yield fragments of about 12-20 kb. The fragments are then separated by gradient centrifugation from undesired sizes and are constructed in bacteriophage lambda vectors. These vectors and phage are packaged in vitro. Recombinant phage are analyzed by plaque hybridization as described in Benton &amp;<br>
27<br><br>
Davis, Science 196:180-182 (1977). Colony hybridization is carried out as generally described in Grunstein et al.,Proc. Natl. Acad. Sci. USA., 72:3961-3965 (1975). An alternative method of isolating GPCR-B3 nucleic acid and its homologs combines the use of synthetic oligonucleotide primers and amplification of an RNA or DNA template (see U.S. Patents 4,683,195 and 4,683,202; PCR Protocols: A Guide to Methods and Applications (Innis et al., eds, 1990)). Methods such as polymerase chain reaction (PCR) and ligase chain reaction (LCR) can be used to amplify nucleic acid sequences of GPCR-B3 directly from rnRNA, from cDNA, from genomic libraries or cDNA libraries. Degenerate oligonucleotides can be designed to amplify GPCR-B3 homologs using the sequences provided herein. Restriction endonuclease sites can be incorporated into the primers. Polymerase chain reaction or other in vitro amplification methods may also be useful, for example, to clone nucleic acid sequences that code for proteins to be expressed, to make nucleic acids to use as probes for detecting the presence of GPCR-B3 encoding mRNA in physiological samples, for nucleic acid sequencing, or for other purposes. Genes amplified by the PCR reaction can be purified from agarose gels and cloned into an appropriate vector.<br>
Gene expression of GPCR-B3 can also be analyzed by techniques known in the art, e.g., reverse transcription and amplification of mRNA, isolation of total RNA or poly AT RNA, northern blotting, dot blotting, in situ hybridization, RNase protection, probing DNA microchip arrays, and the like. In one embodiment, high density oligonucleotide analysis technology (e.g., GeneChip) is used to identify homologs and polymorphic variants of the GPCRs of the invention. In the case where the homologs being identified are linked to a known disease, they can be used with GeneChip as a diagnostic tool in detecting the disease in a biological sample, see, e.g., Gunthand et aL, AIDS Res. Hum. Retroviruses 14: 869-876 (1998); Kozal et aL, Nat. Med. 2:753-759 (1996); Matson et aL, Anal. Biochem, 224:110-106 (1995); Lockhart et aL, Nat. Biotechnol 14:1675-1680 (1996); Gingeras et aL, GenomeRes. 8:435-448 (1998); Hacia et aL, Nucleic Acids Res, 26:3865-3866 (1998).<br>
Synthetic oligonucleotides can be used to construct recombinant GPCR-B3 genes for use as probes or for expression of protein. This method is performed using a series of overlapping oligonucleotides usually 40-120 bp in length, representing both the sense and non-sense strands of the gene. These DNA fragments are then annealed, ligated and cloned. Alternatively, amplification techniques can be used with precise<br>
28<br><br>
primers to amplify a specific subsequence of the GPCR-B3 nucleic acid. The specific subsequence is then ligated into an expression vector.<br>
The nucleic acid encoding GPCR-B3 is typically cloned into intermediate vectors before transformation into prokaryotic or eukaryotic cells for replication and/or expression. These intermediate vectors are typically prokaryote vectors, e.g., plasmids, or shuttle vectors.<br>
Optionally, nucleic acids encoding chimeric proteins comprising GPCR-B3 or domains thereof can be made according to standard techniques. For example, a domain such as ligand binding domain, an extracellular domain, a transmembrane domain (e.g., one comprising seven transmembrane regions and cytosolic loops), the transmembrane domain and a cytoplasmic domain, an active site, a subunit association region, etc., can be covalently linked to a heterologous protein. For example, an extracellular domain can be linked to a heterologous GPCR transmembrane domain, or a heterologous GPCR extracellular domain can be linked to a transmembrane domain. Other heterologous proteins of choice include, e.g., green fluorescent protein, ?-gal, glutamate receptor, and the rhodopsin presequence.<br>
C. Expression in prokaryotes and eukaryotes<br>
To obtain high level expression of a cloned gene or nucleic acid, such as those cDNAs encoding GPCR-B3, one typically subclones GPCR-B3 into an expression vector that contains a strong promoter to direct transcription, a transcription/translation terminator, and if for a nucleic acid encoding a protein, a ribosome binding site for translational initiation. Suitable bacterial promoters are well known in the art and described, e.g., in Sambrook et ah and Ausubel et al. Bacterial expression systems for expressing the GPCR-B3 protein are available in, e.g., E. coli, Bacillus sp., and Salmonella (Palva et al, Gene 22:229-235 (1983); Mosbach et al., Nature 302:543-545 (1983). Kits for such expression systems are commercially available. Eukaryotic expression systems for mammalian cells, yeast, and insect cells are well known in the art and are also commercially available. In one embodiment, the eukaryotic expression vector is an adenoviral vector, an adeno-associated vector, or a retroviral vector.<br>
The promoter used to direct expression of a heterologous nucleic acid depends on the particular application. The promoter is optionally positioned about the same distance from the heterologous transcription start site as it is from the transcription<br>
29<br><br>
start site in its natural setting. As is known in the art, however, some variation in this distance can be accommodated without loss of promoter function.<br>
In addition to the promoter, the expression vector typically contains a transcription unit or expression cassette that contains all the additional elements required for the expression of the GPCR-B3 encoding nucleic acid in host cells. A typical expression cassette thus contains a promoter operably linked to the nucleic acid sequence encoding GPCR-B3 and signals required for efficient polyadenylation of the transcript, ribosome binding sites, and translation termination. The nucleic acid sequence encoding GPCR-B3 may typically be linked to a cleavable signal peptide sequence to promote secretion of the encoded protein by the transformed cell. Such signal peptides would include, among others, the signal peptides from tissue plasminogen activator, insulin, and neuron growth factor, and juvenile hormone esterase of Heliothis virescens.   Additional elements of the cassette may include enhancers and, if genomic DNA is used as the structural gene, introns with functional splice donor and acceptor sites.<br>
In addition to a promoter sequence, the expression cassette should also contain a transcription termination region downstream of the structural gene to provide for efficient termination. The termination region may be obtained from the same gene as the promoter sequence or may be obtained from different genes.<br>
The particular expression vector used to transport the genetic information into the cell is not particularly critical. Any of the conventional vectors used for expression in eukaryotic or prokaryotic cells may be used. Standard bacterial expression vectors include plasmids such as pBR322 based plasmids, pSKF, pET23D, and fusion expression systems such as GST and LacZ. Epitope tags can also be added to recombinant proteins to provide convenient methods of isolation, e.g., c-myc.<br>
Expression vectors containing regulatory elements from eukaryotic viruses are typically used in eukaryotic expression vectors, e.g., SV40 vectors, papilloma virus vectors, and vectors derived from Epstein-Barr virus. Other exemplary eukaryotic vectors include pMSG, pAV009/A+, pMTO10/A+, pMAMneo-5, baculovirus pDSVE, and any other vector allowing expression of proteins under the direction of the SV40 early promoter, SV40 later promoter, metallothionein promoter, murine mammary tumor virus promoter, Rous sarcoma virus promoter, polyhedrin promoter, or other promoters shown effective for expression in eukaryotic cells.<br>
Some expression systems have markers that provide gene amplification such as thymidine kinase, hygromycin El phosphotransferase, and dihydrofolate reductase.<br>
30<br><br>
Alternatively, high yield expression systems not involving gene amplification are also suitable, such as using a bacuiovirus vector in insect cells, with a GPCR-B3 encoding sequence under the direction of the polyhedrin promoter or other strong bacuiovirus promoters.<br>
The elements that are typically included in expression vectors also include a replicon that functions in E. coli, a gene encoding antibiotic resistance to permit selection of bacteria that harbor recombinant plasmids, and unique restriction sites in nonessential regions of the plasmid to allow insertion of eukaryotic sequences. The particular antibiotic resistance gene chosen is not critical, any of the many resistance genes known in the art are suitable. The prokaryotic sequences are typically chosen such that they do not interfere with the replication of the DNA in eukaryotic cells, if necessary.<br>
Standard transfection methods are used to produce bacterial, mammalian, yeast or insect cell lines that express large quantities of GPCR-B3 protein, which are then purified using standard techniques (see, e.g., Colley et al.,J. Biol. Chem. 264:17619-17622 (1989); Guide to Protein Purification, in Methods in Enzymology, vol. 182 (Deutscher, ed., 1990)). Transformation of eukaryotic and prokaryotic cells are performed according to standard techniques (see, e.g., Morrison, J. Bact. 132:349-351 (1977); Clark-Curtiss &amp; Curtiss, Methods in Enzymology 101:347-362 (Wu et al., eds, 1983).<br>
Any of the well known procedures for introducing foreign nucleotide sequences into host cells may be used. These include the use of calcium phosphate transfection, polybrene, protoplast fusion, electroporation, liposomes, microinjection, plasma vectors, viral vectors and any of the other well known methods for introducing cloned genomic DNA, cDNA, synthetic DNA or other foreign genetic material into a host cell (see, e.g., Sambrook et al, supra). It is only necessary that the particular genetic engineering procedure used be capable of successfully introducing at least one gene into the host cell capable of expressing GPCR-B3.<br>
After the expression vector is introduced into the cells, the transfected cells are cultured under conditions favoring expression of GPCR-B3, which is recovered from the culture using standard techniques identified below.<br>
IV. Purification of GPCR-B3<br>
Either naturally occurring or recombinant GPCR-B3 can be purified for use in functional assays. Optionally, recombinant GPCR-B3 is purified. Naturally<br>
31<br><br>
occurring GPCR-B3 is purified, e.g., from mammalian tissue such as tongue tissue, and any other source of a GPCR-B3 homolog. Recombinant GPCR-B3 is purified from any suitable expression system, e.g., bacterial and eukaryotic expression systems, e.g., CHO cells or insect cells.<br>
GPCR-B3 may be purified to substantial purity by standard techniques, including selective precipitation with such substances as ammonium sulfate; column chromatography, immunopurification methods, and others (see, e.g., Scopes, Protein Purification: Principles and Practice (1.982); U.S. Patent No. 4,673,641; Ausubel et al., supra; and Sambrook et al., supra).<br>
A number of procedures can be employed when recombinant GPCR-B3 is being purified. For example, proteins having established molecular adhesion properties can be reversible fused to GPCR-B3. With the appropriate ligand, GPCR-B3 can be selectively adsorbed to a purification column and then freed from the column in a relatively pure form. The fused protein is then removed by enzymatic activity. Finally GPCR-B3 could be purified using immunoaffinity columns.<br>
A. Purification of GPCR-B3 from recombinant cells<br>
Recombinant proteins are expressed by transformed bacteria or eukaryotic cells such as CHO or insect cells in large amounts, typically after promoter induction; but expression can be constitutive. Promoter induction with IPTG is a one example of an inducible promoter system. Cells are grown according to standard procedures in the art. Fresh or frozen cells are used for isolation of protein.<br>
Proteins expressed in bacteria may form insoluble aggregates ("inclusion bodies"). Several protocols are suitable for purification of GPCR-B3 inclusion bodies. For example, purification of inclusion bodies typically involves the extraction, separation and/or purification of inclusion bodies by disruption of bacterial cells, e.g., by incubation in a buffer of 50 mM TRIS/HCL pH 7.5, 50 mM NaCl, 5 mM MgCl2, 1 mM DTT, 0.1 mM ATP, and 1 mM PMSF. The cell suspension can be lysed using 2-3 passages through a French Press, homogenized using a Polytron (Brinkman Instruments) or sonicated on ice. Alternate methods of lysing bacteria are apparent to those of skill in the art (see, e.g., Sambrook et al, supra; Ausubel et al, supra).<br>
If necessary, the inclusion bodies are solubilized, and the lysed cell suspension is typically centrifuged to remove unwanted insoluble matter. Proteins that formed the inclusion bodies may be renatured by dilution or dialysis with a compatible<br>
32<br><br>
buffer. Suitable solvents include, but are not limited to urea (from about 4 M to about 8 M), formamide (at least about 80%, volume/volume basis), and guanidine hydrochloride (from about 4 M to about 8 M). Some solvents which are capable of solubilizing aggregate-forming proteins, for example SDS (sodium dodecyl sulfate), 70% formic acid, are inappropriate for use in this procedure due to the possibility of irreversible denaturation of the proteins, accompanied by a lack of immunogenicity and/or activity. Although guanidine hydrochloride and similar agents are denaturants, this denaturation is not irreversible and renaturation may occur upon removal (by dialysis, for example) or dilution of the denaturant, allowing re-formation of immunologically and/or biologically active protein. Other suitable buffers are known to those skilled in the art. GPCR-B3 is separated from other bacterial proteins by standard separation techniques, e.g., with Ni-NTA agarose resin.<br>
Alternatively, it is possible to purify GPCR-B3 from bacteria periplasm. After lysis of the bacteria, when GPCR-B3 is exported into the periplasm of the bacteria, the periplasmic fraction of the bacteria can be isolated by cold osmotic shock in addition to other methods known to skill in the art. To isolate recombinant proteins from the periplasm, the bacterial cells are centrifuged to form a pellet. The pellet is resuspended in a buffer containing 20% sucrose. To lyse the cells, the bacteria are centrifuged and the pellet is resuspended in ice-cold 5 mM MgSO4 and kept in an ice bath for approximately 10 minutes. The cell suspension is centrifuged and the supernatant decanted and saved. The recombinant proteins present in the supernatant can be separated from the host proteins by standard separation techniques well known to those of skill in the art.<br>
B. Standard protein separation techniques for purifying GPCR-B3 Solubility fractionation<br>
Often as an initial step, particularly if the protein mixture is complex, an initial salt fractionation can separate many of the unwanted host cell proteins (or proteins derived from the cell culture media) from the recombinant protein of interest. In one embodiment, the salt is ammonium sulfate. Ammonium sulfate precipitates proteins by effectively reducing the amount of water in the protein mixture. Proteins then precipitate on the basis of their solubility. The more hydrophobic a protein is, the more likely it is to precipitate at lower ammonium sulfate concentrations. A typical protocol includes adding saturated ammonium sulfate to a protein solution so that the resultant ammonium sulfate concentration is between 20-30%. This concentration will precipitate the most<br>
33<br><br>
hydrophobic of proteins. The precipitate is then discarded (unless the protein of interest is hydrophobic) and ammonium sulfate is added to the supernatant to a concentration known to precipitate the protein of interest. The precipitate is then solubilized in buffer and the excess salt removed if necessary, either through dialysis or diafiltration. Other methods that rely on solubility of proteins, such as cold ethanol precipitation, are well known to those of skill in the art and can be used to fractionate complex protein mixtures.<br>
Size differential filtration<br>
The molecular weight of GPCR-B3 can be used to isolated it from proteins of greater and lesser size using ultrafiltration through membranes of different pore size (for example, Amicon or Millipore membranes). As a first step, the protein mixture is ultrafiltered through a membrane with a pore size that has a lower molecular weight cutoff than the molecular weight of the protein of interest. The retentate of the ultrafiltration is then ultrafiltered against a membrane with a molecular cut off greater than the molecular weight of the protein of interest. The recombinant protein will pass through the membrane into the filtrate. The filtrate can then be chromatographed as described below.<br>
Column chromatography<br>
GPCR-B3 can also be separated from other proteins on the basis of its size, net surface charge, hydrophobicity, and affinity for ligands. In addition, antibodies raised against proteins can be conjugated to column matrices and the proteins immunopurified. All of these methods are well known in the art. It will be apparent to one of skill that chromatographic techniques can be performed at any scale and using equipment from many different manufacturers (e.g., Pharmacia Biotech).<br>
V. Immunological detection of GPCR-B3<br>
In addition to the detection of GPCR-B3 genes and gene expression using nucleic acid hybridization technology, one can also use immunoassays to detect GPCR-B3, e.g., to identify taste receptor cells and variants of GPCR-B3. Immunoassays can be used to qualitatively or quantitatively analyze GPCR-B3. A general overview of the applicable technology can be found in Harlow &amp; Lane, Antibodies: A Laboratory Manual (1988).<br>
34<br><br>
A. Antibodies to GPCR-B3<br>
Methods of producing polyclonal and monoclonal antibodies that react specifically with GPCR-B3 are known to those of skill in the art (see, e.g., Coligan, Current Protocols in Immunology (1991); Harlow &amp; Lane, supra; Goding, Monoclonal Antibodies: Principles and Practice (2d ed. 1986); and Kohler &amp; Milstein, Nature 256:495-497 (1975). Such techniques include antibody preparation by selection of antibodies from libraries of recombinant antibodies in phage or similar vectors, as well as preparation of polyclonal and monoclonal antibodies by immunizing rabbits or mice {see, e.g., Huse et al., Science 246:1275-1281 (1989); Ward et al., Nature 341:544-546 (1989)).<br>
A number of GPCR-B3 comprising immunogens may be used to produce antibodies specifically reactive with GPCR-B3. For example, recombinant GPCR-B3 or an antigenic fragment thereof, is isolated as described herein. Recombinant protein can be expressed in eukaryotic or prokaryotic cells as described above, and purified as generally described above. Recombinanl: protein is one embodiment of an immunogen for the production of monoclonal or polyclonal antibodies. Alternatively, a synthetic peptide derived from the sequences disclosed herein and conjugated to a carrier protein can be used an immunogen. Naturally occurring protein may also be used either in pure or impure form. The product is then injected into an animal capable of producing antibodies. Either monoclonal or polyclonal antibodies may be generated, for subsequent use in immunoassays to measure the protein.<br>
Methods of production of polyclonal antibodies are known to those of skill in the art. An inbred strain of mice (e.g., BALB/C mice) or rabbits is immunized with the protein using a standard adjuvant, such as Freund's adjuvant, and a standard immunization protocol. The animal's immune response to the immunogen preparation is monitored by taking test bleeds and determining the titer of reactivity to GPCR-B3. When appropriately high titers of antibody to the immunogen are obtained, blood is collected from the animal and antisera are prepared. Further fractionation of the antisera to enrich for antibodies reactive to the protein can be done if desired {see Harlow &amp; Lane, supra).<br>
Monoclonal antibodies may be obtained by various techniques familiar to those skilled in the art. Briefly, spleen cells from an animal immunized with a desired antigen are immortalized, commonly by fusion with a myeloma cell {see Kohler &amp; Milstein, Eur. J. Immunol. 6:511-519 (1976)). Alternative methods of immortalization<br>
35<br><br>
include transformation with Epstein Barr Virus, oncogenes, or retroviruses, or other methods well known in the art. Colonies arising from single immortalized cells are screened for production of antibodies of the desired specificity and affinity for the antigen, and yield of the monoclonal antibodies produced by such cells may be enhanced by various techniques, including injection into the peritoneal cavity of a vertebrate host. Alternatively, one may isolate DNA sequences which encode a monoclonal antibody or a binding fragment thereof by screening a DNA library from human B cells according to the general protocol outlined by Huse et al., Science 246:1275-1281 (1989).<br>
Monoclonal antibodies and polyclonal sera are collected and titered against the immunogen protein in an immunoassay, for example, a solid phase immunoassay with the immunogen immobilized on a solid support. Typically, polyclonal antisera with a titer of 104 or greater are selected and tested for their cross reactivity against non-GPCR-B3 proteins or even other related proteins from other organisms, using a competitive binding immunoassay. Specific polyclonal antisera and monoclonal antibodies will usually bind with a Kd of at least about 0.1 mM, more usually at least about 1 ?M, optionally at least about 0.1 ?M or better, and optionally 0.01 ?M or better.<br>
Once GPCR-B3 specific antibodies are available, GPCR-B3 can be detected by a variety of immunoassay methods. For a review of immunological and immunoassay procedures, see Basic and Clinical Immunology (Stites &amp; Terr eds., 7th ed. 1991). Moreover, the immunoassays of the present invention can be performed in any of several configurations, which are reviewed extensively in Enzyme Immunoassay (Maggio, ed.. 1980); and Harlow &amp; Lane, supra.<br>
B. Immunological binding assays<br>
GPCR-B3 can be detected and/or quantified using any of a number of well recognized immunological binding assays {see, e.g., U.S. Patents 4,366,241; 4,376,110; 4,517,288; and 4,837,168). For a review of the general immunoassays, see also Methods in Cell Biology: Antibodies in Cell Biology, volume 37 (Asai, ed. 1993); Basic and Clinical Immunology (Stites &amp; Terr, eds., 7th ed. 1991). Immunological binding assays (or immunoassays) typically use an antibody that specifically binds to a protein or antigen of choice (in this case the GPCR-B3 or antigenic subsequence thereof). The antibody (e.g., anti-GPCR-B3) may be produced by any of a number of means well known to those of skill in the art and as described above.<br>
36<br><br>
Immunoassays also often use a labeling agent to specifically bind to and label the complex formed by the antibody and antigen. The labeling agent may itself be one of the moieties comprising the antibody/antigen complex. Thus, the labeling agent may be a labeled GPCR-B3 polypeptide or a labeled anti-GPCR-B3 antibody. Alternatively, the labeling agent may be a third moiety, such a secondary antibody, that specifically binds to the antibody/GPCR-B3 complex (a secondary antibody is typically specific to antibodies of the species from which the first antibody is derived). Other proteins capable of specifically binding immunoglobulin constant regions, such as protein A or protein G may also be used as the label agent. These proteins exhibit a strong non-immunogenic reactivity with immunoglobulin constant regions from a variety of species (see, e.g., Kronval et al.,J. Immunol. 111:1401-1406(1973); Akerstrom et at., J. Immunol. 135:2589-2542 (1985)). The labeling agent can be modified with a detectable moiety, such as biotin, to which another molecule can specifically bind, such as streptavidin. A variety of detectable moieties are well known to those skilled in the art.<br>
Throughout the assays, incubation and/or washing steps may be required after each combination of reagents. Incubation steps can vary from about 5 seconds to several hours, optionally from about 5 minutes to about 24 hours. However, the incubation time will depend upon the assay format, antigen, volume of solution, concentrations, and the like. Usually, the assays will be carried out at ambient temperature, although they can be conducted over a range of temperatures, such as 10C to 40C.<br>
Non-competitive assav formats<br>
Immunoassays for detecting GPCR-B3 in samples may be either competitive or noncompetitive. Noncompetitive immunoassays are assays in which the amount of antigen is directly measured. In one embodiment "sandwich" assay, for example, the anti-GPCR-B3 antibodies can be bound directly to a solid substrate on which they are immobilized. These immobilized antibodies then capture GPCR-B3 present in the test sample. GPCR-B3 is thus immobilized is then bound by a labeling agent, such as a second GPCR-B3 antibody bearing a label. Alternatively, the second antibody may lack a label, but it may, in turn, be bound by a labeled third antibody specific to antibodies of the species from which the second antibody is derived. The second or third antibody is typically modified with a detectable moiety, such as biotin, to<br>
37<br><br>
which another molecule specifically binds, e.g., streptavidin, to provide a detectable moiety.<br>
Competitive assav formats<br>
In competitive assays, the amount of GPCR-B3 present in the sample is measured indirectly by measuring the amount of a known, added (exogenous) GPCR-B3 displaced (competed away) from an anti-GPCR-B3 antibody by the unknown GPCR-B3 present in a sample. In one competitive assay, a known amount of GPCR-B3 is added to a sample and the sample is then contacted with an antibody that specifically-binds to GPCR-B3. The amount of exogenous GPCR-B3 bound to the antibody is inversely proportional to the concentration of GPCR-B3 present in the sample. In one embodiment, the antibody is immobilized on a solid substrate. The amount of GPCR-B3 bound to the antibody may be determined either by measuring the amount of GPCR-B3 present in a GPCR-B3/antibody complex, or alternatively by measuring the amount of remaining uncomplexed protein. The amount of GPCR-B3 may be detected by providing a labeled GPCR-B3 molecule.<br>
A hapten inhibition assay is another competitive assay. In this assay the known GPCR-B3, is immobilized on a solid substrate. A known amount of anti-GPCR-B3 antibody is added to the sample, and the sample is then contacted with the immobilized GPCR-B3. The amount of anti-GPCR-B3 antibody bound to the known immobilized GPCR-B3 is inversely proportional to the amount of GPCR-B3 present in the sample. Again, the amount of immobilized antibody may be detected by detecting either the immobilized fraction of antibody or the fraction of the antibody that remains in solution. Detection may be direct where the antibody is labeled or indirect by the subsequent addition of a labeled moiety that specifically binds to the antibody as described above.<br>
Cross-reactivity determinations<br>
Immunoassays in the competitive binding format can also be used for crossreactivity determinations. For example, a protein at least partially encoded by SEQ ID NOS:l-3 can be immobilized to a solid support. Proteins (e.g., GPCR-B3 proteins and homologs) are added to the assay that compete for binding of the antisera to the immobilized antigen. The ability of the added proteins to compete for binding of the antisera to the immobilized protein is compared to the ability of GPCR-B3 encoded by<br>
38<br><br>
SEQ ID NO: 1 -3 to compete with itself. The percent crossreactivity for the above proteins is calculated, using standard calculations. Those antisera with less than 10% crossreactivity with each of the added proteins listed above are selected and pooled. The cross-reacting antibodies are optionally removed from the pooled antisera by immunoabsorption with the added considered proteins, e.g., distantly related homologs. The immuno absorbed and pooled antisera are then used in a competitive binding immunoassay as described above to compare a second protein, thought to be perhaps an allele or polymorphic variant of GPCR-B3, to the immunogen protein (i.e., GPCR-B3 of SEQ ID NOS:l-3). In order to make this comparison, the two proteins are each assayed at a wide range of concentrations and the amount of each protein required to inhibit 50% of the binding of the antisera to the immobilized protein is determined. If the amount of the second protein required to inhibit 50% of binding is less than 10 times the amount of the protein encoded by SEQ ID NOS: 1 -3 that is required to inhibit 50% of binding, then the second protein is said to specifically bind to the polyclona! antibodies generated to a GPCR-B3 immunogen.<br>
Other assay formats<br>
Western blot (immunoblot) analysis is used to detect and quantify the presence of GPCR-B3 in the sample. The technique generally comprises separating sample proteins by gel electrophoresis on the basis of molecular weight, transferring the separated proteins to a suitable solid support, (such as a nitrocellulose filter, a nylon filter, or derivatized nylon filter), and incubating the sample with the antibodies that specifically bind GPCR-B3. The anti-GPCR-B3 antibodies specifically bind to the GPCR-B3 on the solid support. These antibodies may be directly labeled or alternatively may be subsequently detected using labeled antibodies (e.g., labeled sheep anti-mouse antibodies) that specifically bind to the anti-GPCR-B3 antibodies.<br>
Other assay formats include liposome immunoassays (LIA), which use liposomes designed to bind specific molecules (e.g., antibodies) and release encapsulated reagents or markers. The released chemicals are then detected according to standard techniques (see Monroe et al., Amer. Clin. Prod. Rev. 5:34-41 (1986)).<br>
Reduction of non-specific binding<br>
One of skill in the art will appreciate that it is often desirable to minimize non-specific binding in immunoassays. Particularly, where the assay involves an antigen<br>
39<br><br>
or antibody immobilized on a solid substrate it is desirable to minimize the amount of non-specific binding to the substrate. Means of reducing such non-specific binding are well known to those of skill in the art. Typically, this technique involves coating the substrate with a proteinaceous composition. In particular, protein compositions such as bovine serum albumin (BSA), nonfat powdered milk, and gelatin are widely used.<br>
Labels<br>
The particular label or detectable group used in the assay is not a critical aspect of the invention, as long as it does not significantly interfere with the specific binding of the antibody used in the assay. The detectable group can be any material having a detectable physical or chemical property. Such detectable labels have been well-developed in the field of immunoassays and, in general, most any label useful in such methods can be applied to the present invention. Thus, a label is any composition detectable by spectroscopic, photochemical, biochemical, immunochemical, electrical, optical or chemical means. Useful labels in the present invention include magnetic beads (e.g., DYNABEADS), fluorescent dyes (e.g., fluorescein isothiocyanate, Texas red, rhodamine, and the like), radiolabels (e.g., 3H, 125I,35S, 14C, or 32P), enzymes (e.g., horse radish peroxidase, alkaline phosphatase and others commonly used in an ELISA), and colorimetric labels such as colloidal gold or colored glass or plastic beads (e.g., polystyrene, polypropylene, latex, etc.).<br>
The label may be coupled directly or indirectly to the desired component of the assay according to methods well known in the art. As indicated above, a wide variety of labels may be used, with the choice of label depending on sensitivity required, ease of conjugation with the compound, stability requirements, available instrumentation, and disposal provisions.<br>
Non-radioactive labels are often attached by indirect means. Generally, a ligand molecule (e.g., biotin) is covalently bound to the molecule. The ligand then binds to another molecules (e.g., streptavidin) molecule, which is either inherently detectable or covalently bound to a signal system, such as a detectable enzyme, a fluorescent    . compound, or a chemiluminescent compound. The ligands and their targets can be used in any suitable combination with antibodies that recognize GPCR-B3, or secondary antibodies that recognize anti-GPCR-B3.<br>
The molecules can also be conjugated directly to signal generating compounds, e.g., by conjugation with an enzyme or fluorophore. Enzymes of interest as<br>
40<br><br>
labels will primarily be hydrolases, particularly phosphatases, esterases and glycosidases, or oxidotases, particularly peroxidases. Fluorescent compounds include fluorescein and its derivatives, rhodamine and its derivatives, dansyl, umbelliferone, etc: Chemiluminescent compounds include luciferin, and 2,3-dihydrophthalazinediones, e.g., luminol. For a review of various labeling or signal producing systems that may be used, see U.S. Patent No. 4,391,904.<br>
Means of detecting labels are well known to those of skill in the art. Thus, for example, where the label is a radioactive label, means for detection include a scintillation counter or photographic film as in autoradiography. Where the label is a fluorescent label, it may be detected by exciting the fluorochrome with the appropriate wavelength of light and detecting the resulting fluorescence. The fluorescence may be detected visually, by means of photographic film, by the use of electronic detectors such as charge coupled devices (CCDs) or photomultipliers and the like. Similarly, enzymatic labels may be detected by providing the appropriate substrates for the enzyme and detecting the resulting reaction product. Finally simple colorimetric labels may be detected simply by observing the color associated with the label. Thus, in various dipstick assays, conjugated gold often appears pink, while various conjugated beads appear the color of the bead.<br>
Some assay formats do not require the use of labeled components. For instance, agglutination assays can be used to detect the presence of the target antibodies. In this case, antigen-coated particles are agglutinated by samples comprising the target antibodies. In this format, none of the components need be labeled and the presence of the target antibody is detected by simple visual inspection.<br>
VI. Assays for modulators of GPCR-B3<br>
A. Assays for GPCR-B3 activity<br>
GPCR-B3 and its alleles and polymorphic variants are G-protein coupled receptors that participate in taste transduction. The activity of GPCR-B3 polypeptides can be assessed using a variety of in vitro and in vivo assays that determine functional, physical and chemical effects, e.g., measuring ligand binding (e.g., by radioactive ligand binding), second messengers (e.g., cAMP, cGMP, IP3, DAG, or Ca2+), ion flux, phosphorylation levels, transcription levels, neurotransmitter levels, and the like. Furthermore, such assays can be used to test for inhibitors and activators of GPCR-B3.<br>
41<br><br>
Modulators can also be genetically altered versions of GPCR-B3. Such modulators of taste transduction activity are useful for customizing taste.<br>
The GPCR-B3 of the assay will be selected from a polypeptide having a sequence of SEQ ID NOS:l-3 or conservatively modified variant thereof. Alternatively, the GPCR-B3 of the assay will be derived from a eukaryote and include an amino acid subsequence having amino acid sequence identity SEQ ID NOS:l-3. Generally, the amino acid sequence identity will be at least 70%, optionally at least 85%, most optionally at least 90-95%. Optionally, the polypeptide of the assays will comprise a domain of GPCR-B3, such as an extracellular domain, transmembrane domain, cytoplasmic domain, ligand binding domain, subunit association domain, active site, and the like. Either GPCR-B3 or a domain thereof can be covalently linked to a heterologous protein to create a chimeric protein used in the assays described herein.<br>
Modulators of GPCR-B3 activity are tested using GPCR-B3 polypeptides as described above, either recombinant or naturally occurring. The protein can be isolated, expressed in a cell, expressed in a membrane derived from a cell, expressed in tissue or in an animal, either recombinant or naturally occurring. For example, tongue slices, dissociated cells from a tongue, transformed cells, or membranes can b used.. Modulation is tested using one of the in vitro or in vivo assays described herein. Taste transduction can also be examined in vitro with soluble or solid state reactions, using a chimeric molecule such as an extracellular domain of a receptor covalently linked to a heterologous signal transduction domain., or a heterologous extracellular domain covalently linked to the transmembrane and or cytoplasmic domain of a receptor. Furthermore, ligand-binding domains of the protein of interest can be used in vitro in soluble or solid state reactions to assay for ligand binding.<br>
Ligand binding to GPCR-B3, a domain, or chimeric protein can be tested in solution, in a bilayer membrane, attached to a solid phase, in a lipid monolayer, or in vesicles. Binding of a modulator can be tested using, e.g., changes in spectroscopic characteristics (e.g., fluorescence, absorbance, refractive index) hydrodynamic (e.g., shape), chromatographic, or solubility properties.<br>
Receptor-G-protein interactions can also be examined. For example, binding of the G-protein to the receptor or its release from the receptor can be examined. For example, in the absence of GTP, an activator will lead to the formation of a tight complex of a G protein (all three subunits) with the receptor. This complex can be detected in a variety of ways, as noted above. Such an assay can be modified to search<br>
42<br><br>
for inhibitors. Add an activator to the receptor and G protein in the absence of GTP, form a tight complex, and then screen for inhibitors by looking at dissociation of the receptor-G protein complex. In the presence of GTP, release of the alpha subunit of the G protein from the other two G protein subunits serves as a criterion of activation.<br>
An activated or inhibited G-protein will in turn alter the properties of target enzymes, channels, and other effector proteins. The classic examples are the activation of cGMP phosphodiesterase by transducin in the visual system, adenylate cyclase by the stimulatory G-protein, phospholipase C by Gq and other cognate G proteins, and modulation of diverse channels by Gi and other G proteins. Downstream consequences can also be examined such as generation of diacyl glycerol and IP3 by phospholipase C, and in turn, for calcium mobilization by IP3.<br>
Activated GPCR receptors become substrates for kinases that<br>
phosphorylate the C-terminal tail of the receptor (and possibly other sites as well). Thus, activators will promote the transfer of 32P from gamma-labeled GTP to the receptor, which can be assayed with a scintillation counter. The phosphorylation of the C-terminal tail will promote the binding of arrestin-like proteins and will interfere with the binding of G-proteins. The kinase/arrestin pathway plays a key role in the desensitization of many GPCR receptors. For example, compounds that modulate the duration a taste receptor stays active would be useful as a means of prolonging a desired taste or cutting off an unpleasant one. For a general review of GPCR signal transduction and methods of assaying signal transduction, see, e.g., Methods in Enzymology, vols. 237 and 238 (1994) and volume 96 (1983); Bourne et al, Nature 10:349:117-27 (1991); Bourne et al., Nature 348:125-32 (1990); Pitcher et al, Annu. Rev. Biochem. 67:653-92 (1998).<br>
Samples or assays that are treated with a potential GPCR-B3 inhibitor or activator are compared to control samples without the test compound, to examine the extent of modulation. Control samples (untreated with activators or inhibitors) are assigned a relative GPCR-B3 activity value of 100. Inhibition of GPCR-B3 is achieved when the GPCR-B3 activity value relative to the control is about 90%, optionally 50%, optionally 25-0%. Activation of GPCR-B3 is achieved when the GPCR-B3 activity value relative to the control is 110%, optionally 150%, 200-500%, or 1000-2000%.<br>
Changes in ion flux may be assessed by determining changes in<br>
polarization (i.e., electrical potential) of the cell or membrane expressing GPCR-B3. One means to determine changes in cellular polarization is by measuring changes in current (thereby measuring changes in polarization.) with voltage-clamp and patch-clamp<br>
43<br><br>
techniques, e.g., the "cell-attached" mode, the "inside-out" mode, and the "whole cell" mode (see. e.g., Ackerman et al, New Engl. J. Med. 336:1575-1595 (1997)). Whole cell currents are conveniently determined using the standard methodology (see, e.g., Hamil et al, PFlugers. Archiv. 391:85 (1981). Other known assays include: radiolabeled ion flux assays and fluorescence assays using voltage-sensitive dyes {see, e.g., Vestergarrd-Bogind et al, J. Membrane Biol. 88:67-75 (1988); Gonzales &amp; Tsien, Chem. Biol. 4:269-277 (1997); Daniel et al, J. Pharmacol Meth. 25:185-193 (1991); Holevinsky et al, J. Membrane Biology 137:59-70 (1994)). Generally, the compounds to be tested are present in the range from 1 pM to 100 mM.<br>
The effects of the test compounds upon the function of the polypeptides can be measured by examining any of the parameters described above. Any suitable physiological change that affects GPCR activity can be used to assess the influence of a test compound on the polypeptides of this invention. When the functional consequences are determined using intact cells or animals, one can also measure a variety of effects such as transmitter release, hormone release, transcriptional changes to both known and uncharacterized genetic markers (e.g., northern blots), changes in cell metabolism such as cell growth or pH changes, and changes in intracellular second messengers such as Ca2+, IP3 or cAMP.<br>
Assays for G-protein coupled receptors include cells that are loaded with ion or voltage sensitive dyes to report receptor activity. Assays for determining activity of such receptors can also use known agonists and antagonists for other G-protein coupled receptors as negative or positive controls to assess activity of tested compounds. In assays for identifying modulatory compounds (e.g., agonists, antagonists), changes in the level of ions in the cytoplasm or membrane voltage will be monitored using an ion -sensitive or membrane voltage fluorescent indicator, respectively. Among the ion-sensitive indicators and voltage probes that may be employed are those disclosed in the Molecular Probes 1997 Catalog. For G-protein coupled receptors, promiscuous G-proteins such as G??15 and G??l6 can be used in the assay of choice (Wilkie et al, Proc. Nat'l Acad. Sci. USA 88:10049-10053 (1991)). Such promiscuous G-proteins allow coupling of a wide range of receptors.<br>
Receptor activation typically initiates subsequent intracellular events, e.g., increases in second messengers such as IP3, which releases intracellular stores of calcium ions. Activation of some G-protein coupled receptors stimulates the formation of inositol<br>
44<br><br>
triphosphate (IP3) through phospholipase C-mediated hydrolysis of phosphatidylinositol (Berridge &amp; Irvine, Nature 312:315-21 (1984)). IP3 in turn stimulates the release of intracellular calcium ion stores. Thus, a change in cytoplasmic calcium ion levels, or a change in second messenger levels such as IP3 can be used to assess G-protein coupled receptor function. Cells expressing such G-protein coupled receptors may exhibit increased cytoplasmic calcium levels as a result of contribution from both intracellular stores and via activation of ion channels, in which case it may be desirable although not necessary to conduct such assays in calcium-free buffer, optionally supplemented with a chelating agent such as EGTA, to distinguish fluorescence response resulting from calcium release from internal stores.<br>
Other assays can involve determining the activity of receptors which, when activated, result in a change in the level of intraceliular cyclic nucleotides, e.g., cAMP or cGMP, by activating or inhibiting enzymes such as adenylate cyclase. There are cyclic nucleotide-gated ion channels, e.g., rod photoreceptor cell channels and olfactory neuron channels that are permeable to cations upon activation by binding of cAMP or cGMP (see, e.g., Altenhofen et al, Proc. Natl. Acad. Sci. U.S.A. 88:9868-9872 (1991) and Dhallan et al, Nature 347:184-187 (1990)). In cases where activation of the receptor results in a decrease in cyclic nucleotide levels, it may be preferable to expose the cells to agents that increase intracellular cyclic nucleotide levels, e.g., forskolin, prior to adding a receptor-activating compound to the cells in the assay. Cells for this type of assay can be made by co-transfection of a host cell with DNA encoding a cyclic nucleotide-crated ion channel, GPCR phosphatase and DNA encoding a receptor (e.g., certain glutamate receptors, muscarinic acetylcholine receptors, dopamine receptors, serotonin receptors, and the like), which, when activated, causes a change in cyclic nucleotide levels in the cytoplasm.<br>
In one embodiment, GPCR-B3 activity is measured by expressing GPCR-B3 in a heterologous cell with a promiscuous G-protein that links the receptor to a phospholipase C signal transduction pathway {see Offermanns &amp; Simon, J. Biol. Chem. 270:15175-15180 (1995)). Optionally the cell line is HEK-293 (which does not naturally express GPCR-B3) and the promiscuous G-protein is Gal5 (Offermanns &amp; Simon, supra). Modulation of taste transduction is assayed by measuring changes in intracellular Ca2+ levels, which change in response to modulation of the GPCR-B3 signal transduction pathway via administration of a molecule that associates with GPCR-B3. Changes in<br>
45<br><br>
  Ca + levels are optionally measured using fluorescent Ca2+ indicator dyes and fluorometric imaging.<br>
In one embodiment, the changes in intracellular cAMP or cGMP can be measured using immunoassays. The method described in Offermanns &amp; Simon, J. Biol. Chem. 270:15175-15180 (1995) may be used to determine the level of cAMP. Also, the method described in Felley-Bosco et al., Am. J. Resp. Cell and Mol. Biol. 11:159-164 (1994) may be used to determine the level of cGMP. Further, an assay kit for measuring cAMP and/or cGMP is described in U.S. Patent 4,115,538, herein incorporated by reference.<br>
In another embodiment, phosphatidyl inositol (PI) hydrolysis can be analyzed according to U.S. Patent 5,436,128, herein incorporated by reference. Briefly, the assay involves labeling of cells with 3H-myoinositol for 48 or more hrs. The labeled cells are treated with a test compound for one hour. The treated cells are lysed and extracted in chloroform-methanol-water after which the inositol phosphates were separated by ion exchange chromatography and quantified by scintillation counting. Fold stimulation is determined by calculating the ratio of cprn in the presence of agonist to cpm in the presence of buffer control. Likewise, fold inhibition is determined by calculating the ratio of cpm in the presence of antagonist to cpm in the presence of buffer control (which may or may not contain an agonist).<br>
In another embodiment, transcription levels can be measured to assess the effects of a test compound on signal transduction. A host cell containing the protein of interest is contacted with a test compound for a sufficient time to effect any interactions, and then the level of gene expression is measured. The amount of time to effect such interactions may be empirically determined, such as by running a time course and measuring the level of transcription as a function of time. The amount of transcription may be measured by using any method known to those of skill in the art to be suitable. For example, mRNA expression of the protein of interest may be detected using northern blots or their polypeptide products may be identified using immunoassays. Alternatively, transcription based assays using reporter gene may be used as described in U.S. Patent 5,436,128, herein incorporated by reference. The reporter genes can be, e.g., chloramphenicol acetyltransferase, firefly luciferase, bacterial luciferase, ?-galactosidase and alkaline phosphatase. Furthermore, the protein of interest can be used as an indirect<br>
46<br><br>
reporter via attachment to a second reporter such as green fluorescent protein {see, e.g., Mistili &amp; Spector, Nature Biotechnology 15:961-964 (1997)).<br>
The amount of transcription is then compared to the amount of transcription in either the same cell in the absence of the test compound, or it may be compared with the amount of transcription in a substantially identical cell that lacks the protein of interest. A substantially identical cell may be derived from the same cells from which the recombinant cell was prepared but which had not been modified by introduction of heterologous DNA. Any difference in the amount of transcription indicates that the test compound has in some manner altered the activity of the protein of interest.<br>
B. Modulators<br>
The compounds tested as modulators of GPCR-B3 can be any small chemical compound, or a biological entity, such as a protein, sugar, nucleic acid or lipid. Alternatively, modulators can be genetically altered versions of GPCR-B3. Typically, test compounds will be small chemical molecules and peptides. Essentially any chemical compound can be used as a potential modulator or ligand in the assays of the invention, although most often compounds can be dissolved in aqueous or organic (especially DMSO-based) solutions are used. The assays are designed to screen large chemical libraries by automating the assay steps and providing compounds from any convenient source to assays, which are typically run in parallel (e.g., in microtiter formats on microtiter plates in robotic assays). It will be appreciated that there are many suppliers of chemical compounds, including Sigma (St. Louis, MO), Aldrich (St. Louis, MO), Sigma-Aldrich (St. Louis, MO), Fluka Chemika-Biochemica Analytika (Buchs Switzerland) and the like.<br>
In one embodiment, high throughput screening methods involve providing a combinatorial chemical or peptide library containing a large number of potential therapeutic compounds (potential modulator or ligand compounds). Such "combinatorial chemical libraries" or "ligand libraries" are then screened in one or more assays, as described herein, to identify those library members (particular chemical species or subclasses) that display a desired characteristic activity. The compounds thus identified can serve as conventional "lead compounds" or can themselves be used as potential or actual therapeutics.<br>
47<br><br>
A combinatorial chemical library is a collection of diverse chemical compounds generated by either chemical synthesis or biological synthesis, by combining a number of chemical "building blocks" such as reagents. For example, a linear combinatorial chemical library such as a polypeptide library is formed by combining a set of chemical building blocks (amino acids) in every possible way for a given compound length (i.e., the number of amino acids in a polypeptide compound). Millions of chemical compounds can be synthesized through such combinatorial mixing of chemical building blocks.<br>
Preparation and screening of combinatorial chemical libraries is well known to those of skill in the art. Such combinatorial chemical libraries include, but are not limited to, peptide libraries (see, e.g., U.S. Patent 5,010,175, Furka, Int. J. Pept. Prot. Res. 37:487-493 (1991) and Houghton et al., Nature 354:84-88 (1991)). Other chemistries for generating chemical diversity libraries can also be used. Such chemistries include, but are not limited to: peptoids (e.g., PCT Publication No. WO 91/19735), encoded peptides (e.g., PCT Publication WO 93/20242), random bio-oligomers (e.g., PCT Publication No. WO 92/00091), benzodiazepines (e.g., U.S. Pat. No. 5,288,514), diversomers such as hydantoins, benzodiazepines and dipeptides (Hobbs et al, Proc. Nat. Acad. Sci. USA 90:6909-6913 (1993)), vinylogous polypeptides (Hagihara et al, J. Amer. Chem. Soc. 114:6568 (1992)), nonpeptidal peptidomimetics with glucose scaffolding (Hirschmann et al, J. Amer. Chem. Soc. 114:9217-9218 (1992)), analogous organic syntheses of small compound libraries (Chen et al., J. Amer. Chem. Soc. 116:2661 (1994)), oligocarbamates (Cho et al., Science 261:1303 (1993)), and/or peptidyl phosphonates (Campbell et al, J. Org. Chem. 59:658 (1994)), nucleic acid libraries (see Ausubel, Berger and Sambrook, all supra), peptide nucleic acid libraries (see, e.g., U.S. Patent 5,539,083), antibody libraries (see, e.g., Vaughn et al., Nature Biotechnology, 14(3):309-314 (1996) and PCT/US96/10287), carbohydrate libraries (see. e.g., Liang et al, Science, 274:1520-1522 (1996) and U.S. Patent 5,593,853), small organic molecule libraries (see, e.g., benzodiazepines, Baum C&amp;EN, Jan 18, page 33 (1993); isoprenoids, U.S. Patent 5,569,588; thiazolidinones and metathiazanones, U.S. Patent 5,549,974; pyrrolidines, U.S. Patents 5,525,735 and 5,519,134; morpholino compounds, U.S. Patent 5,506,337; benzodiazepines, 5,288,514, and the like).<br>
Devices for the preparation of combinatorial libraries are commercially available (see, e.g., 357 MPS, 390 MPS, Advanced Chem Tech, Louisville KY, Symphony, Rainin, Woburn, MA, 433A Applied Biosystems, Foster City, CA, 9050 Plus,<br>
48<br><br>
Millipore, Bedford, MA). In addition, numerous combinatorial libraries are themselves commercially available (see, e.g., ComGenex, Princeton, N.J., Tripos, Inc., St. Louis, MO, 3D Pharmaceuticals, Exton, PA, Martek Biosciences, Columbia, MD, etc.).<br>
C. Solid State and soluble high throughput assays<br>
In one embodiment the invention provide soluble assays using molecules such as a domain such as ligand binding domain, an extracellular domain, a transmembrane domain (e.g., one comprising seven transmembrane regions and cytosolic loops), the transmembrane domain and a cytoplasmic domain, an active site, a subunit association region, etc.; a domain that is covalently linked to a heterologous protein to create a chimeric molecule; GPCR-B3; or a cell or tissue expressing GPCR-B3, either naturally occurring or recombinant. In another embodiment, the invention provides solid phase based in vitro assays in a high throughput format, where the domain, chimeric molecule, GPCR-B3, or cell or tissue expressing GPCR-B3 is attached to a solid phase substrate.<br>
In the high throughput assays of the invention, it is possible to screen up to several thousand different modulators or ligands in a single day. In particular, each well of a microtiter plate can be used to run a separate assay against a selected potential modulator, or, if concentration or incubation time effects are to be observed, every 5-10 wells can test a single modulator. Thus, a single standard microtiter plate can assay about 100 (e.g., 96) modulators. If 1536 well plates are used, then a single plate can easily assay from about 100- about 1500 different compounds. It is possible to assay several different plates per day; assay screens for up to about 6,000-20,000 different compounds is possible using the integrated systems of the invention. More recently, microfluidic approaches to reagent manipulation have been developed, e.g., by Caliper Technologies (Palo Alto, CA).<br>
The molecule of interest can be bound to the solid state component, directly or indirectly, via covalent or non covalent linkage e.g., via a tag. The tag can be any of a variety of components. In general, a molecule which binds the tag (a tag binder) is fixed to a solid support, and the tagged molecule of interest (e.g., the taste transduction molecule of interest) is attached to the solid support by interaction of the tag and the tag binder.<br>
A number of tags and tag binders can be used, based upon known molecular interactions well described in the literature. For example, where a tag has a<br>
49<br><br>
natural binder, for example, biotin, protein A, or protein G, it can be used in conjunction with appropriate tag binders (avidin, streptavidin, neutravidin, the Fc region of an immunoglobulin, etc.) Antibodies to molecules with natural binders such as biotin are also widely available and appropriate tag binders; see, SIGMA Immunochemicals 1998 catalogue SIGMA, St. Louis MO).<br>
Similarly, any haptenic or antigenic compound can be used in combination with an appropriate antibody to form a tag/tag binder pair. Thousands of specific antibodies are commercially available and many additional antibodies are described in the literature. For example, in one common configuration, the tag is a first antibody and the tag binder is a second antibody which recognizes the first antibody. In addition to antibody-antigen interactions, receptor-ligand interactions are also appropriate as tag and tag-binder pairs. For example, agonists and antagonists of cell membrane receptors (e.g., cell receptor-ligand interactions such as; transferrin, c-kil, viral receptor ligands, cytokine receptors, chemokine receptors, interleukin receptors, immunoglobulin receptors and antibodies, the cadherein family, the integrin family, the selectin family, and the like; see, e.g., Pigott &amp; Power, The Adhesion Molecule Facts Book I (1993). Similarly, toxins and venoms, viral epitopes, hormones (e.g., opiates, steroids, etc.), intracellular receptors (e.g. which mediate the effects of various small ligands, including steroids, thyroid hormone, retinoids and vitamin D; peptides), drugs, lectins, sugars., nucleic acids (both linear and cyclic polymer configurations), oligosaccharides, proteins, phospholipids and antibodies can all interact with various cell receptors.<br>
Synthetic polymers, such as polyurethanes, polyesters, polycarbonates, polyureas, polyamides, polyethyleneimines, polyarylene sulfides, polysiloxanes, polyimides, and polyacetates can also form an appropriate tag or tag binder. Many other tag/tag binder pairs are also useful in assay systems described herein, as would be apparent to one of skill upon review of this disclosure.<br>
Common linkers such as peptides, polyethers, and the like can also serve as tags, and include polypeptide sequences, such as poly gly sequences of between about 5 and 200 amino acids. Such flexible linkers are known to persons of skill in the art. For example, poly(ethelyne glycol) linkers are available from Shearwater Polymers, Inc. Huntsville, Alabama. These linkers optionally have amide linkages, sulfhydryl linkages, or heterofunctional linkages.<br>
Tag binders are fixed to solid substrates using any of a variety of methods currently available. Solid substrates are commonly derivatized or functionalized by<br>
50<br><br>
exposing all or a portion of the substrate to a chemical reagent which fixes a chemical group to the surface which is reactive with a portion of the tag binder. For example, groups which are suitable for attachment to a longer chain portion would include amines, hydroxyl, thiol, and carboxyl groups. Aminoalkylsilanes and hydroxyalkylsilanes can be used to functionalize a variety of surfaces, such as glass surfaces. The construction of such solid phase biopolymer arrays is well described in the literature. See, e.g., Merrifield, J. Am. Chem. Soc. 85:2149-2154 (1963) (describing solid phase synthesis of, e.g., peptides); Geysen et al., J. Immun, Meth. 102:259-274 (1987) (describing synthesis of solid phase components on pins); Frank &amp; Doring, Tetrahedron 44:60316040 (1988) (describing synthesis of various peptide sequences on cellulose disks); Fodor et al., Science, 251:767-771 (1991); Sheldon et al., Clinical Chemistry 39(4):718-719 (1993); and Kozal et al, Nature Medicine 2(7):753759 (1996) (all describing arrays of biopolymers fixed to solid substrates). Non-chemical approaches for fixing tag binders to substrates include other common methods, such as heat, cross-linking by UV radiation, and the like.<br>
D. Computer-based assays<br>
Yet another assay for compounds that modulate GPCR-B3 activity involves computer assisted drug design, in which a computer system is used to generate a three-dimensional structure of GPCR-B3 based on the structural information encoded by the amino acid sequence. The input amino acid sequence interacts directly and actively with a preestabiished algorithm in a computer program to yield secondary, tertiary, and quaternary structural models of the protein. The models of the protein structure are then examined to identify regions of the structure that have the ability to bind, e.g., ligands. These regions are then used to identify ligands that bind to the protein.<br>
The three-dimensional structural model of the protein is generated by entering protein amino acid sequences of at least 10 amino acid residues or corresponding nucleic acid sequences encoding a GPCR-B3 polypeptide into the computer system. The amino acid sequence of the polypeptide of the nucleic acid encoding the polypeptide is selected from the group consisting of SEQ ID NOS:l-3 or SEQ ID NOS:4-6 and conservatively modified versions thereof. The amino acid sequence represents the primary sequence or subsequence of the protein, which encodes the structural information of the protein. At least 10 residues of the amino acid sequence (or a nucleotide sequence encoding 10 amino acids) are entered into the computer system from computer<br>
51<br><br>
keyboards, computer readable substrates that include, but are not limited to, electronic storage media (e.g., magnetic diskettes, tapes, cartridges, and chips), optical media (e.g., CD ROM), information distributed by internet sites, and by RAM. The three-dimensional structural model of the protein is then generated by the interaction of the amino acid sequence and the computer system, using software known to those of skill in the art.<br>
The amino acid sequence represents a primary structure that encodes the information necessary to form the secondary, tertiary and quaternary structure of the protein of interest. The software looks at certain parameters encoded by the primary sequence to generate the structural model. These parameters are referred to as "energy terms," and primarily include electrostatic potentials, hydrophobic potentials, solvent accessible surfaces, and hydrogen bonding. Secondary energy terms include van der Waals potentials. Biological molecules form the structures that minimize the energy terms in a cumulative fashion.   The computer program is therefore using these terms encoded by the primary structure or amino acid sequence to create the secondary structural model.<br>
The tertiary structure of the protein encoded by the secondary structure is then formed on the basis of the energy terms of the secondary structure. The user at this point can enter additional variables such as whether the protein is membrane bound or soluble, its location in the body, and its cellular location, e.g., cytoplasmic, surface, or nuclear. These variables along with the energy terms of the secondary structure are used to form the model of the tertiary structure. In modeling the tertiary structure, the computer program matches hydrophobic faces of secondary structure with like, and hydrophilic faces of secondary structure with like.<br>
Once the structure has been generated, potential ligand binding regions are identified by the computer system. Three-dimensional structures for potential ligands are generated by entering amino acid or nucleotide sequences or chemical formulas of compounds, as described above. The three-dimensional structure of the potential ligand is then compared to that of the GPCR-B3 protein to identify ligands that bind to GPCR-B3. Binding affinity between the protein and ligands is determined using energy terms to determine which ligands.have an enhanced probability of binding to the protein.<br>
Computer systems are also used to screen for mutations, polymorphic variants, alleles and interspecies homologs of GPCR-B3 genes. Such mutations can be associated with disease states or genetic traits. As described above, GeneChip and<br>
52<br><br>
related technology can also be used to screen for mutations, polymorphic variants, alleles and interspecies homologs. Once the variants are identified, diagnostic assays can be used to identify patients having such mutated genes. Identification of the mutated GPCR-B3 genes involves receiving input of a first nucleic acid or amino acid sequence encoding GPCR-B3, selected from the group consisting of SEQ ID NOS: 1-3, or SEQ ID NOS:4-6 and conservatively modified versions thereof. The sequence is entered into the computer system as described above. The first nucleic acid or amino acid sequence is then compared to a second nucleic acid or amino acid sequence that has substantial identity to the first sequence. The second sequence is entered into the computer system in the manner described above. Once the first and second sequences are compared, nucleotide or amino acid differences between the sequences are identified. Such sequences can represent allelic differences in GPCR-B3 genes, and mutations associated with disease states and genetic traits.<br>
VIII. Kits<br>
GPCR-B3 and its homologs are a useful tool for identifying taste receptor cells, for forensics and paternity determinations, and for examining taste transduction. GPCR-B3 specific reagents that specifically hybridize to GPCR-B3 nucleic acid, such as GPCR-B3 probes and primers, and GPCR-B3 specific reagents that specifically bind to the GPCR-B3 protein, e.g., GPCR-B3 antibodies are used to examine taste cell expression and taste transduction regulation.<br>
Nucleic acid assays for the presence of GPCR-B3 DNA and RNA in a sample include numerous techniques are known to those skilled in the art, such as Southern analysis, northern analysis, dot blots, RNase protection, S1 analysis, amplification techniques such as PCR and LCR, and in situ hybridization. In in situ hybridization, for example, the target nucleic acid is liberated from its cellular surroundings in such as to be available for hybridization within the cell while preserving the cellular morphology for subsequent interpretation and analysis. The following articles provide an overview of the art of in situ hybridization: Singer et al., Biotechniques 4:230-250 (1986); Haase et al., Methods in Virology, vol. VII, pp. 189-226 (1984); and Nucleic Acid Hybridization: A Practical Approach (Hames et aL, eds. 1987). In addition, GPCR-B3 protein can be detected with the various immunoassay techniques described above. The test sample is typically compared to both a positive control (e.g., a sample expressing recombinant GPCR-B3) and a negative control.<br>
53<br><br>
The present invention also provides for kits for screening for modulators of GPCR-B3. Such kits can be prepared from readily available materials and reagents. For example, such kits can comprise any one or more of the following materials: GPCR-B3, reaction tubes, and instructions for testing GPCR-B3 activity. Optionally, the kit contains biologically active GPCR-B3. A wide variety of kits and components can be prepared according to the present invention, depending upon the intended user of the kit and the particular needs of the user.<br>
IX. Administration and pharmaceutical compositions<br>
Taste modulators can be administered directly to the mammalian subject for modulation of taste in vivo. Administration is by any of the routes normally used for introducing a modulator compound into ultimate contact with the tissue to be treated, e.g., the tongue or mouth. The taste modulators are administered in any suitable manner, optionally with pharmaceutically acceptable carriers. Suitable methods of administering such modulators are available and well known to those of skill in the art, and, although more than one route can be used to administer a particular composition, a particular route can often provide a more immediate and more effective reaction than another route.<br>
Pharmaceutically acceptable carriers are determined in part by the particular composition being administered, as well as by the particular method used to administer the composition. Accordingly, there is a wide variety of suitable formulations of pharmaceutical compositions of the present invention (see, e.g., Remington's<br>
Pharmaceutical Sciences, 17thed. 1985)).<br><br>
The taste modulators, alone or in combination with other suitable components, can be made into aerosol formulations (i.e., they can be "nebulized") to be administered via inhalation. Aerosol formulations can be placed into pressurized acceptable propellants, such as dichlorodifluoromethane, propane, nitrogen, and the like.<br>
Formulations suitable for administration include aqueous and non-aqueous solutions, isotonic sterile solutions, which can contain antioxidants, buffers, bacteriostats, and solutes that render the formulation isotonic, and aqueous and non-aqueous sterile suspensions that can include suspending agents, solubilizers, thickening agents, stabilizers, and preservatives. In the practice of this invention, compositions can be administered, for example, by orally, topically, intravenously, intraperitoneally, intravesically or intrathecally. Optionally, the compositions are administered orally or nasally. The formulations of compounds can be presented in unit-dose or multi-dose<br>
54<br><br>
sealed containers, such as ampules and vials. Solutions and suspensions can be prepared from sterile powders, granules, and tablets of the kind previously described. The modulators can also be administered as part a of prepared food or drug.<br>
The dose administered to a patient, in the context of the present invention should be sufficient to effect a beneficial response in the subject over time. The dose will be determined by the efficacy of the particular taste modulators employed and the condition of the subject, as well as the body weight or surface area of the area to be treated. The size of the dose also will be determined by the existence, nature, and extent of any adverse side-effects that accompany the administration of a particular compound or vector in a particular subject.<br>
In determining the effective amount of the modulator to be administered in a physician may evaluate circulating plasma levels of the modulator, modulator toxicities,, and the production of anti-modulator antibodies. In general, the dose equivalent of a modulator is from about 1 mg/kg to 10 mg/kg for a typical subject.<br>
For administration, taste modulators of the present invention can be administered at a rate determined by the LD-50 of the modulator, and the side-effects of the inhibitor at various concentrations, as applied to the mass and overall health of the subject. Administration can be accomplished via single or divided doses.<br>
All publications and patent applications cited in this specification are herein incorporated by reference as if each individual publication or patent application were specifically and individually indicated to be incorporated by reference.<br>
Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be readily apparent to one of ordinary skill in the art in light of the teachings of this invention that certain changes and modifications may be made thereto without departing from the spirit or scope of the appended claims.<br>
EXAMPLES<br>
The following examples are provided by way of illustration only and not by way of limitation. Those of skill in the art will readily recognize a variety of noncritical parameters that could be changed or modified to yield essentially similar results.<br>
55<br><br>
Example I: Cloning and expression of GPCR-B3<br>
Since taste transduction occurs in taste receptor cells found in taste buds of the tongue and palate epithelium, a full-length cDNA library was generated from rat taste papillae. This library was made by oligo-dT priming of poly-A+ RNA isolated from several hundreds rat circumvallate papillae using a directional IZAP vector (Stratagene Inc; Hoon &amp; Ryba, J. Dent. Res. 76:831-838 (1997)) following standard molecular biology procedures {see, e.g., Ausubel et al, Current Protocols in Molecular Biology (1995). A collection of single-cell and single taste-bud cDNA libraries was also generated from individually isolated taste receptor cells and taste buds from rat and mouse circumvallate, foliate and fungiform papillae according to the method of Dulac &amp; Axel, Cell 83:195-206 (1995). Taste buds and single taste receptor cells were isolated by enzymatic digestion and micro-dissection of lingual epithelium from adult rats and mice. To maximize lysis efficiency in the taste bud preparations, the lysis buffer volume was increased 10 fold (Dulac &amp; Axel, supra).<br>
GPCR-B3 was isolated from the IZAP circumvallate cDNA library by first generating a subtracted library enriched in sequences expressed in taste tissue. Construction and initial analysis of a taste-receptor cell subtracted cDNA library was as described by Hoon &amp; Ryba, supra. Further enrichment of taste-specific transcripts was achieved by dot-blot screening of cDNA clones with non-taste cDNA probes. Non-taste probes included lingual epithelium tissue devoid of taste buds, muscle, liver and brain tissue. The individual hybridization probes were generated by preparing first strand cDNA and labeling it using random priming methods as described in Ausubel et al., supra. Hybridization conditions and washes were 65C, 2x SSC for hybridizations, and 65C, O.lx SSC for washing.<br>
All cDNAs that showed taste tissue enrichment in the differential screens with taste and non-taste tissue were picked for DNA sequencing analysis using standard dideoxy-termination methods and an automated ABI sequencing machine. DNA sequences were subjected to data analysis using a variety of homology and structure prediction programs (e.g. blast at http://www.ncbi.nlm.nih.gov/ Tmpred at http://dot.imgen.bcm.tmc.edu:9331/seq-search/struc-predict.html). Individual cDNA clones that encoded novel sequences, sequences with some similarity to known signaling components, sequences with multiple predicted transmembrane domains, or sequences with known motifs such as SH2, SH3, PDZ, etc (see for example pfam at http://pfam.wustl.edu/) were chosen as candidates for further analysis.<br>
56<br><br>
Candidate cDNAs were assayed for taste-cell expression by in situ hybridization to tissue sections of rat tongue. Tissue was obtained from adult rats. Fresh frozen sections (14 mm) were attached to silanized slides and prepared for in situ hybridization as described by Ryba &amp; Tirindelli, Neuron 19:371-379 (1997). All in situ hybridizations were carried out at high stringency (5x SSC, 50 % formamide, 72 C). For single-label detection, signals were developed using alkaline-phosphatase conjugated antibodies to digoxigenin and standard chromogenic substrates (Boehringer Mannheim) as described by Ryba &amp; Tirindelli, supra. Partial DNA sequencing reactions were performed on ~2000 subtracted and single-cell cDNA clones, and in situ hybridizations were carried out with ~400 different candidate cDNAs. This screen identified a number of genes expressed in taste receptor cells including a single clone encoding a 3' fragment of GPCR-B3.<br>
Full-length rat GPCR-B3 was isolated from the 1ZAP rat circumvallate cDNA library following standard plaque hybridization protocols (Ausubel et ah, supra). Approximately 2.5 x106 clones were plated at high density on LB plates (~100,000 phage/plate) and replica filters were hybridized with a radiolabeled GPCR-B3 probe at high stringency (65C, 2x SSC). Positive clones were picked, retested, purified and characterized by DNA restriction mapping and sequencing analysis. Several full-length GPCR-B3 clones were isolated and characterized (see SEQ ID NOS:4-6 and the amino acid sequences that they encode, SEQ ID NOS:l-3).<br>
The mouse interspecies homolog of rat GPCR-B3 was isolated by screening a mouse genomic Bac and 1 library (Genome Systems) at low and moderate stringency (48C, 7x SSC and 55C, 5x SSC). The clones were characterized by restriction mapping and DNA sequencing. A mouse cDNA was isolated by performing RACE reactions (Marathon Kit, Clonetech) using first-strand cDNA prepared from RNA isolated from mouse circumvallate and foliate papillae. The human homolog of GPCR-B3 was isolated from a human testis library (Clonetech Inc.) following the observation that other sensory receptors such as olfactory and visual receptors are expressed in testis (Axel &amp; Dulac, supra). See Figure 1 for a topological map of GPCR-B3, showing the extracellular domain, seven transmembrane domains, and an intracellular or C-terminal domain.<br>
57<br><br>
.   Example II: Western blot and in situ analysis<br>
To demonstrate specific expression of GPCR-B3 protein in taste cells, antibodies were generated against short peptides and GPCR-B3 fusion proteins. The peptides consisted of 18 amino acid residues from the N- or C-terminal end of the GPCR-B3 predicted protein (see, e.g., SEQ ID NO: 1 and 2). The fusion proteins consisted of GST-fusion polypeptides encompassing the entire N-terminal domain or the last 3 predicted transmembrane segments plus the C-term region. Fusions were generated using standard molecular techniques (Harlow &amp; Lane, Antibodies (1988)). Peptides were fused to carrier proteins, immunized into rabbits, and the serum affinity purified and assayed as described by Cassill et al., Proc. Nat 7 Acad. Sci. USA 88:11067-11070 (1991)).<br>
Antibodies were tested for specificity by western-blot analysis of protein homogenates from circumvallate or fungiform papillae. The blots also contained liver and brain protein extracts as negative controls. For immunohistochemistry, frozen sections were prepared as described by Ryba &amp; Tirindelli, supra for in situ hybridizations, except that blocking reactions used 10 % donkey immunoglobulin, 1 % bovine serum albumin, 0.3% Triton X-100. Sections were incubated in the appropriate dilution of anti-TRl (1:100) for 12-18 hrs., and detected using fluorescein-conjugated donkey anti-rabbit secondary antibodies (Jackson Immunolaboratory). Taste buds were counter-stained with the F-actin marker BODIPYRTR-X phallacidin (Molecular Probes). As a control for these studies, anti-NCAM antibodies were also used. Fluorescent images were obtained using a Leica TSC confocal microscope with an argon-krypton laser. Pre-treatment of the antibodies with the peptide immunogen abolished staining. See Figures 2 and 3 for western blot and in situ analysis, respectively.<br>
58<br><br>
WE CLAIM :<br>
1.         An isolated nucleic acid encoding a sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70% amino acid identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
2.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
encodes a receptor that specifically binds to polyclbnal antibodies generated against SEQ<br>
ID NO:I, SEQ ID NO:2, or SEQ ID NO:3.<br>
3.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
encodes a receptor that has G-coupled protein receptor activity.<br>
4.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
encodes a receptor comprising an amino acid sequence of SEQ ID NO: 1, SEQ ID NO:2,<br>
or SEQ ID NO:3.<br>
5.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
acid comprises a nucleotide sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ ID NO:6.<br>
6.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
from a human, a mouse, or a rat.<br>
7.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
amplified by primers that selectively hybridize under stringent hybridization conditions to<br>
the same sequence as degenerate primer sets encoding amino acid sequences selected<br>
from the group consisting of:<br>
IAWDWNGPKW (SEQ ID NO:7) and LPENYNEAKC (SEQ ID NO:8).<br>
8.	The isolated nucleic acid as claimed in claim 1, wherein the nucleic acid<br>
encodes a receptor having a molecular weight of about between 92 kDa to about 102 kDa.<br>
59<br><br>
9.	An isolated nucleic acid encoding a sensory transduction G-proteir<br>
coupled receptor, wherein the nucleic acid specifically hybridizes under highly stringent<br>
conditions to a nucleic acid having the sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ<br>
ID NO:6.<br>
10.	An isolated nucleic acid encoding a sensory transduction G-protein<br>
coupled receptor, the receptor comprising greater than about 70% amino acid identity to a<br>
polypeptidc having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3,<br>
wherein the nucleic acid selectively hybridizes under moderately stringent hybridization<br>
conditions to a nucleotide sequence of SEQ ID NO:4, SEQ ID NO:5, or SEQ ID NO:6.<br>
11.      An isolated nucleic acid encoding an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain having greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1.<br>
12.	The isolated nucleic acid as claimed in claim 11, wherein the nucleic acid<br>
encodes the extracellular domain linked to a nucleic acid encoding a heterologous<br>
polypeptide, forming a chimeric polypeptide.<br>
13.	The isolated nucleic acid as claimed in claim 11, wherein the nucleic acid<br>
encodes the extracellular domain of SEQ ID NO:1.<br>
14.      An isolated nucleic acid encoding a transmembrane domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the transmembrane domain of SEQ ID NO: 1.<br>
15.       The isolated nucleic acid as claimed in claim 14, wherein the nucleic acid encodes the transmembrane domain linked to a nucleic acid encoding a heterologous polypeptide, forming a chimeric polypeptide.<br>
16.      The isolated nucleic acid as claimed in claim 14, wherein the nucleic acid encodes the transmembrane domain of SEQ ID NO:1.<br>
60<br><br>
17.      The isolated nucleic acid as claimed in claim 14, wherein the nucleic acid further encodes a cytopiasmic domain comprising greater than about 70% ammo acid identity to the cytopiasmic domain of SEQ ID NO: 1.<br>
18.       The isolated nucleic acid as claimed in claim 17, wherein the nucleic acid encodes the cytopiasmic domain of SEQ ID NO:1.<br>
19.      An isolated sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70% amino acid sequence identity to an amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
20.      The isolated receptor as claimed in claim 19, wherein the receptor specifically binds to polyclonal antibodies generated against SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
21.	The isolated receptor as claimed in claim 19, wherein the receptor has G-<br>
protein coupled receptor activity.<br>
22.	The isolated receptor as claimed in claim 19, wherein the receptor has an<br>
amino acid sequence of SEQ ID NO: 1, SEQ ID NO:2, or SEQ ID NO:3.<br>
23.	The isolated receptor as claimed in claim 19, wherein the receptor is from a<br>
human, a rat, or a mouse.<br>
24.       An isolated polypeptide comprising an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain comprising greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1.<br>
25.       The isolated polypeptide as claimed in claim 24, wherein the polypeptide encodes the extracellular domain of SEQ ID NO: 1.<br>
61<br><br>
26.      The isolated polypeptide as claimed in claim 24, wherein the extracellular domain is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide.<br>
27.       An isolated polypeptide comprising a transmembrane domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the transmembrane domain of SEQ ID NO:1.<br>
28.      The isolated polypeptide as claimed in claim 27, wherein the polypeptide encodes the transmembrane domain of SEQ ID NO:1.<br>
29.      The isolated polypeptide as claimed in claim 27, comprising a cytoplasmic domain comprising greater than about 70% amino acid identity to the cytoplasmic domain of SEQ ID NO:1.<br>
30.      The isolated polypeptide as claimed in claim 29, wherein the polypeptide encodes the cytoplasmic domain of SEQ ID NO:1.<br>
31.	The isolated polypeptide as claimed in claim 27., wherein the transmembrane<br>
domain is covalently linked to a heterologous polypeptide, forming a chimeric<br>
polypeptide.<br>
32.	The isolated polypeptide as claimed in claim 31, wherein the chimeric<br>
polypeptide has G-protein coupled receptor activity.<br>
33.	An antibody that selectively binds to the receptor as claimed in claim 19.<br>
34.	An expression vector comprising the nucleic acid as claimed in claim 1.<br>
35..       A method for identifying a compound that modulates sensory signaling in sensory cells, the method comprising the steps of:<br>
62<br><br>
(i) contacting the compound with a. polypeptide comprising an extracellular domain of a sensory transduction G-protein coupled receptor, the extracellular domain comprising greater than about 70% amino acid sequence identity to the extracellular domain of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3; and<br>
(ii) determining the functional effect of the compound upon the extracellular domain.<br>
36.       The method as claimed in claim 35, wherein the polypeptide is a sensory transduction G-protein coupled receptor, the receptor comprising greater than about 70% amino acid identity to a polypeptide encoding SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
37.       The method as claimed in claim 36, wherein the polypeptide comprises an extracellular domain that is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide.<br>
38.      The method as claimed in claim 36 or 37, wherein the polypeptide has G-protein coupled receptor activity.<br>
39.      The method as claimed in claim 35, wherein the extracellular domain is linked to a solid phase.<br>
40.     The method as claimed in claim 39, wherein the extracellular domain is covalently linked to a solid phase.<br>
41.       The method as claimed in claim 36 or 37, wherein the functional effect is determined by measuring changes in intracellular cAMP, IP3, or Ca 2+.<br>
42.	The method as claimed in claim 35", wherein the functional effect is a chemical<br>
effect.<br>
43.	The method as claimed in claim 35, wherein the functional effect is a physical<br>
effect.<br>
63<br><br>
44.      The method as claimed in claim 35, wherein the functional effect  is determined by measuring binding of the compound to the extracellular domain.<br>
46.      The method as claimed in claim 35, wherein the polypeptide is a recombinant.<br>
4'6       The method as claimed in claim 35, wherein the polypeptide is from a rat, a mouse, or a human.<br>
47	The method as claimed in claim 36, wherein the polypeptide comprises an<br>
amino acid sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
48	The method as claimed in claim 36 or 37, wherein the polypeptide is<br>
expressed in a cell or cell membrane.<br>
49.      The method as claimed in claim 48, wherein the cell is a eukaryotic cell.<br>
50.       A method for identifying a compound that modulates sensory signaling in sensory cells, the method comprising the steps of:<br>
(i) contacting the compound with a polypeptide comprising an transmembrane domain of a sensory transduction G-protein coupled receptor, the transmembrane domain comprising greater than about 70% amino acid sequence identity to the transmembrane domain of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3; and<br>
(ii) determining the functional effect of the compound upon the transmembrane domain.<br>
51.      The method as claimed in claim 50, wherein the polypeptide comprises an transmembrane domain that is covalently linked to a heterologous polypeptide, forming a chimeric polypeptide.<br>
5Z.      The method as claimed in claim 5 /., wherein the chimeric polypeptide has G-protein coupled receptor activity.<br>
53,       The method as claimed in claim 50, wherein the functional effect is<br>
determined by measuring changes in intracellular cAMP, IP3, or Ca2+.<br>
64<br><br>
54.      The method as claimed in claim 50, wherein the functional effect is a chemical ; effect.<br>
55.	The method as claimed in claim 50, wherein the functional effect is a physical<br>
effect.<br>
56.	The method as claimed in claim 50, wherein the polypeptide is recombinant<br><br>
57.	The method as claimed in claim 50, wherein the polypeptide is from a rat, a<br>
mouse, or a human.<br>
58.	The method as claimed in claim 50 or 57., wherein the polypeptide is<br>
expressed in a cell or cell membrane.<br>
59.      The method as claimed in claim 58, wherein the cell is a eukaryotic cell.<br>
6V.      A method of making a sensory transduction G-protein coupled receptor, the method comprising the step of expressing the receptor from a recombinant expression vector comprising a nucleic acid encoding the receptor, wherein the amino acid sequence of the receptor comprises greater than about 70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
6.1.       A method of making a recombinant cell comprising a sensory transduction G-protein coupled receptor, the method comprising the step of transducing the cell with an expression vector comprising a nucleic acid encoding the receptor, wherein the amino acid sequence of the receptor comprises greater than about 70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3.<br>
62..     A method of making an recombinant expression vector comprising a nucleic acid encoding a sensory transduction G-protein coupled receptor, the method comprising the step of ligating to an expression vector a nucleic acid encoding the<br>
receptor, wherein the amino acid sequence of the receptor comprises greater than about<br>
65<br><br>
70% amino acid identity to a polypeptide having a sequence of SEQ ID NO:1, SEQ ID  NO:2,or SEQ ID NO:3.<br>
66<br>
The present invention provides amino acid sequences of novel sensory cell specific G-protein coupled receptors GPCR-B3 and the nucleic acids encoding the receptors. These receptors are generally characterized as having an amino acid sequence with greater than about 70% sequence identity to SEQ ID NO:1, SEQ ID NO:2, or SEQ ID NO:3. Antibodies that specifically recognize the novel GPCR-B3 receptors are also provided by this invention. Furthermore, this invention provides methods for detecting nucleic acids encoding the GPCR-B3 receptors and methods for detecting the receptors. In addition, the invention provides methods useful for screening candidate compounds for those capable of modulating sensory signaling in sensory cells through their ability to exert a functional effect on a GPCR-B3 receptor of this invention.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="202530-growth-hormone-releasing-compounds-and-their-antagonists.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="202532-improved-eco-friendly-recycling-process-of-post-consumer-waste-plastics-and-device-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>202531</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00224/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Mar-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Mar-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Feb-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE REGENTS OF THE UNIVERSITY OF CALIFORNIA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>12TH FLOOR, 1111 FRANKLIN STREET, OAKLAND, CA 94607-5200, UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ZUKER CHARLES S.</td>
											<td>4778, THURSTON PLACE, SAN DIEGO, CA 92130 UNITED STATES OF AMERICA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ADLER JON ELLIOTT</td>
											<td>1099 TURQUOISE # 10, PACIFIC BEACH CA 92109 UNITED STATES OF AMERICA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LINDEMEIER JUERGEN</td>
											<td>FRANZISKANERANGER 2, D-59457, WERL, GERMANY</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RYBA NICK</td>
											<td>9202 LUNDIGEN COURT, BETHESDA, MD 20817 UNITED STATES OF AMERICA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HOON MARK</td>
											<td>4218 WARNER STREET, KENSINGTON, MD 20895, UNITED STATES OF AMERICA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 07 H 21/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US99/17099</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-07-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/094,465</td>
									<td>1998-07-28</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/202531-nucleic-acids-encoding-a-g-protein-coupled-receptor-involved-in-sensory-transduction by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:39:54 GMT -->
</html>
