<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/202826-a-fuel-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:58:43 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 202826:A FUEL CELL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A FUEL CELL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A fuel cell which has the ability to start up instantly and can accept recaptured energy such as that of regenerative braking by operating in reverse as an electrolyzer. The instant startup fuel cells have increased efficiency and power availability (higher voltage and current) and a dramatic improvement in operating temperature range of about -20 to 1500 C. The fuel cell employs an anode active material which has hydrogen storage capacity, such as a hydrogen shortage alloy which has excellent catalytic activity for the formation of atomic hydrogen from molecular hydrogen amongst other characteristics. The anode active material has a gaseous hydrogen contacting surface, an electrolyte contacting surface and the bulk of said anode active material is disposed within said gaseous hydrogen contacting surface and electrolyte contacting surface.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to DNA which enables co-expression of protein disulfide isomers and a polypeptide having disulfide bonds, and a process for producing a polypeptide having disulfide bonds, which is obtained through expression of the DNA in bacteria of the genus Bacillus and in which correct disulfide bonds are formed.<br>
BACKGROUND ART<br>
Physiologically active substances, such as hormones and enzymes, which play important roles in life activity, are proteins (polypeptides), and the high order structure is important in maintaining the activity. A disulfide bond that is formed between two cytokines is a component forming a high order structure. For example, insulin is secreted from B cells in the islets of Lagerphones in the pancreas and is the most important hormone in storing or using sugar, amino acid and fatty acid, and in maintaining homeostasis of blood glucose levels. Insulin is a polypeptide that consists of an A chain comprising 21 amino acids and a B chain comprising 30 amino acids, and has one disulfide bond within the A chain and two disulfide bonds between the A chain and the B chain. When these 3 disulfide bonds air formed inaccurately, insulin will have no activity. Disulfide bonds are thought to be formed by enzymes, such as thioredoxin and protein disulfide isomers, under an oxidation-reduction environment formed by glutathione and the like in a process where translated proteins are secreted to the endoplasmic reticulum and then transferred to the Google complex (Hwang, C. et al., Science 257: 1496-1502, (1992)., Getting, M. &amp; Sam brook, J., Nature 355: 33-45, (1992), Bardwell, J.C.A. &amp; Beckwith, J., Cell 74: 769-771, (1993)). It has been shown that these enzymes act to form disulfide bonds in in vitro experiments where the disulfide bond formation of proteins denatured to have no disulfide bond is enhanced when they are placed under reduction conditions and under the presence of the enzymes (Lyles, M. M. &amp; Gilbert, H.<br><br>
R, Biochemistry 30: 613-619, (1991)., Lyles, M. M. &amp; Gilbert, H. R, Biochemistry 30: 619-625, (1991)., Pigiet, V. R &amp; Schuster, B. J. Proc. Natl. Acad. Sci. USA 83:7643-7647, (1986)). Further, protein disulfide isomerase is known to have an effect of rearranging incorrect disulfide bonds into correct disulfide bonds (Weissman, J. S. &amp; Kim, P. S, Nature 365:185-188, (1993)., Laboissiere, M. C. A. et al., J. Biol. Chem. 270: 28006-28009, (1995)).<br>
Some physiologically active polypeptides that are present only in trace amounts in vivo, such as insulin, glucagon, interferon, calcitonin and growth hormone, are now being produced using established gene recombination techniques in large amounts as gene-recombinant proteins in prokaryotes and eukaryotes. In particular, an expression system in prokaryotes is widely used because of the large amount produced and the low production costs. However, among expression systems of prokaryotes, particularly a system in which proteins are expressed as inclusions within the cells of Escherichia coli has the advantage of a large amount of expression, but such a system cannot provide an environment for disulfide bond formation. Hence, proteins having disulfide bonds expressed in this expression system must be denatured once after isolation and purification, and then an enviromnent for disulfide bond formation must be established. These steps are complex and may increase the cost. In contrast, a system in which proteins are expressed in the periplasm of Escherichia coli is an environment which enables disulfide bond formation therewithin, but the expression amount is limited. Moreover, a yeast expression system enables disulfide bond formation within the cells before secretion, but its expression amount is low.<br>
A recently developed expression system of gene-recombinant protein using Bacillus brevis {Bacillus brevis; referred to as bacteria of the genus Brevibacillus according to a new classification) has received attention as a mass production system of gene-recombinant proteins with disulfide bonds, because polypeptides having disulfide bonds (human epidermal growth factor and the like) in an active state, that is, having accurately formed disulfide bonds are secreted and expressed in large amounts in media (Japanese Patent No. 2082727, Japanese Patent Application Laying-Open (Kokai) No. 62-201583, Konishi, H. et al., Appl, Microbiol. BiotechnoL, 34:297-302 (1990), Shigezo UDAKA, Journal of Japan Society for Bioscience, Biotechnology and Agrochemislry 61, 669-676 (1987), Sagiya, Y. et al., Appl. Microbiol.<br><br>
Biotechnol., 42:358-363 (1994), Yamagata, H. el al., Proc. Natl. Acad. Sci. USA 86: 3589-3593 (1989)). However, it has recently been found that the rate of con-ect disulfide bond formation is not 100% in this expression system (Miyauchi, A. et al., J. Indust. Microbiol. Biotech., 21: 208-214, (1998)). The rate of coixect disulfide bond formation seems to differ depending on the types of polypeptides. For example, the formation rate of a human epidennal growth factor is 80%.<br>
To date, it is known that protein disulfide isomerase is involved in the intracellular formation of correct disulfide bonds of a protein. Particularly regarding yeast, there has been a repoil that co-expression of the gene of protein disulfide isomerase involved in disulfide bond foi-mation and the gene of a protein having disulfide bonds has enabled production of proteins having correct disulfide bonds (Japanese Patent Application Laying-Open (Kokai) Nos. 6-38771 and 7-508881). Generally, in yeast, the eukaryote, disulfide bonds are formed while the protein is transferred from inti"acellular endoplasmic reticulum to the Golgi complex, and then secreted extracellularly after completion of tlie high order structure of the protein. As an example in bacteria, there has been a report that a target protein gene has been ligated to the gene of protein disulfide isomerase of molds and expressed in an expression system of bacteria of the genus Bacillus, In this example, protein disulfide isomerase is expressed as a fusion protein with the protein (Japanese Patent Application Laying-Open (Kokai) No. 11-75879).<br>
An object of the present invention is to increase efficiency of the production of polypeptides with correct disulfide bonds in a bacterial expression system for genes encoding proteins where it is difficult or impossible for such bonds to fonn. This is done by allowing co-expression of a gene encoding a polypeptide having disulfide bonds and a gene encoding protein disulfide isomerase, so as to establish an environment wherein the polypeptide and protein disulfide isomerase co-exist. In this case, it is required to surmount problems of how an expression/secretion system should be established to allow co-expression of the gene encoding a polypeptide and the gene encoding protein disulfide isomerase within bacterial cells, to allow extracellular secretion of both the generated proteins, and to allow extracellular formation of correct disulfide bonds in the polypeptide.<br><br>
DISCLOSURE OF THE INVENTION<br>
To achieve the above object, we have cloned and ligated the gene of protein disulfide isomerase to the gene of a polypeptide having disulfide bonds, thereby designing and constructing an expression/secretion system in bacteria in which both gene products are simultaneously secreted in medium. Hence, we were able to estabHsh an environment in bacteria allowing co-existence of protein disulfide isomerase and a polypeptide having disulfide bonds, and thus increased correct disulfide bond fonnation in polypeptides having disulfide bonds.<br>
Specifically, the present invention is summarized as follows.<br>
In a first aspect of the present invention, there is provided a DNA which enables co-expression of a protein disulfide isomerase and a polypeptide having disulfide bonds,<br>
comprising one or two identical or different promoters required for gene expression (Promoter), two identical or different Shine-Dalgamo sequences (SD), two identical or different sequences encoding signal peptides of cell wall proteins (CWP) of a bacterium of the genus Bacillus (CWPsp), a gene encoding a polypeptide having disulfide bonds (Polypeptide), and a gene encoding protein disulfide isomerase (PDI) that are ligated to each other as represented by the following formula: 5"-Promoter-SD-CWPsp-X,-(Promoter)n-SD-CWPsp-X2-3", wherein Xi=Polypeptide and Xi^PDI, or Xi=PDI and X2=Polypepfide, and n=0 or 1,<br>
In an embodiment of the present invention, the promoter is derived from bacteria of the genus Bacillus, for example, it is derived from CWP (for example, MWP (Middle-Wall Protein)) of bacteria of the genus Bacillus.<br>
In another embodiment of the present invention, protein disulfide isomerase is derived fi"om a human.<br>
In  still  another embodiment  of the present  invention,  the  polypeptide having disulfide bonds is a fusion protein containing a human insulin and is represented by, for example, the following formula: MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain,<br><br>
wherein MWPmp9 represents 9 amino acids from the N teraiinus of MWP mature protein; GSLQPR, RGHRP and PR respectively represents an amino acid sequence shown by one-letter codes; A chain and B chain respectively represent the A chain and B chain of human insulin; and Linker represents a linker consisting of any amino acid.<br>
In a specific example of the present invention, a part of a modified insulin C peptide is used as a linker. However, examples of linkers are not specifically limited as long as they are capable of linking two domains together without affecting the functional domain of the protein. For example, the appropriate number of Ala, Gly, Pro, Ser and Val may be linked in appropriate combinations.<br>
In another aspect of the present invention, there is provided a vector comprising a DNA as defined above.<br>
In still another aspect of the present invention, there is provided a bacterium of the genus Bacillus which is transformed with the above vector. In an embodiment of the present invention, the bacterium of the genus Bacillus is Bacillus brevis.<br>
In another aspect of the present invention, there is provided a process for producing a polypeptide having disulfide bonds, which comprises the steps of: introducing the above vector into a bacterium of the genus Bacillus; culturing the obtained transformed bacterium in a medium;<br>
co-expressing a protein disulfide isomerase and the polypeptides having one or more disulfide bonds, thereby secreting together extracellularly; and<br>
collecting the polypeptide having correct disulfide bonds formed by the action of the protein disulfide isomerase.<br>
In an embodiment of the present invention, the above process further comprises the steps of:<br>
generating a polypeptide having correct disulfide bonds as a fusion protein; and treating the obtained fusion protein with protease to obtain the polypeptide.<br>
In another embodiment of ihe present invention, the polypeptide is a human insulin.<br>
This specification includes pari or all of the contents as disclosed in the specification<br><br>
and/or drawings  of Japanese Patent Application No.  2000-70753,  which  is  a priority document of the present application.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Figure 1 shows amino acid sequences of fusion proteins MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain (Fig. 1 A) and MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Unker-PR-A chain-MWPsp*-hPDI* (Fig. IB) and nucleotide sequences encoding the amino acid sequences.<br>
Figure 2 is a schematic view showing the steps for incorporating fusion DNAs obtained in Examples 1 and 2 into an expression vector (pNU211R2L5) of Bacillus brevis.<br>
Figure 3 shows a picture of electrophoresis of a medium taken after transfomiants were cultured.    In the figure, lane 1 indicates a marker peptide; lane 2 pNU-mPINS (nonreduclion), lane 3 pNU-niPINS-hPDJ* (nonreduction), lane 4 pNU-mPlNS (reduction), and lane 5 pNU-mPINS-hPDI* (reduction).<br>
Figure 4 shows a photograph of western blotting of a medium taken after transfomiants were cultured.    In the figure, lane 1 indicates pNU-mPINS (nonreduction); lane 2 pNU-mPINS-hPDI* (nonreduction), lane 3 pNU-mPINS (reduction), and lane 4 pNU-mPINS-hPDI* (reduction).<br>
Figure 5 shows an HPLC elution pattern for a fusion protein, MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain.<br>
Figure 6 is a schematic diagram of conversion from a fusion protein MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain to insulin.<br>
Figure 7 shows a comparison between peptide mappings of the insulin of the present invention and a commercially available insulin (Novolin).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention makes it possible to obtain increased efficiency of the production of polypeptides that have undergone correct disulfide bond formation by allowing protein disulfide isomerase and a polypeptide having disulfide bonds to co-exist in an<br><br>
environment in which no disulfide bond can be formed or where efficiency of disulfide bond formation is low.<br>
Examples of promoters of bacteria of the genus Bacillus include those adjustable, and capable of expressing a gene ligated 3" downstream, such as Spac promoters, promoters derived from    105 phage, and promoters derived from a structural gene of a host including amilase and protease (Molecular Biological Methods for Bacillus, Harwood, C. R. &amp; Cutting, S. M. eds.,    pp202-203,1990).<br>
According to the embodiment of the present invention, an appropriate promoter is CWP protein of bacteria of the genus Bacillus.    Examples of CWP protein include, but are not limited to, MWp protein (J. Bacteriol., 169:1239-1245, 1989) derived from Bacillus brevis strain 47 (PERM P-7224: Japanese Patent Application Laying-Open(Kokai) No. 60-58074, Japanese Patent Application Laying-Open (Kokai)No. 62-201589) and HWP protein (J, Bacteriol., 172:1312-1320, \990) derived f rom BrevibaciUus choshinensis,(PERM BP-1087: Japanese Patent Application Laying-Open (Kokai) No. 4-278091).    Further, in the present invention, a sequence which encodes a signal peptide of the CWP protein illustrated herein can be used as CWPsp in the above formula.<br>
Protein disulfide isomerase is present generally in organisms having polypeptides which require disulfide bonds, such as yeast, molds and mammals.    Any type of protein disulfide isomerase can be used in the present invention, as far as it is an enzyme involved in correct disulfide bond formation of a polypeptide.    For example, protein disulfide isomerase may beD sb of Escherichia coli (Bardwell, J.C.A. et ah. Cell, 67:582-589 (1991), Kamitani, S. et al., EMBO J., 11:57-62 (1992)), which is involved in disulfide bond formation of a protein and is thought to relate to protein disulfide isomerase.    According to the embodiment of the present invention, an appropriate protein disulfide isomerase is derived from a human.<br>
Examples of a polypeptide having disulfide bonds include an insulin, an epidermal growth factor (EGP), a growth honnone and albumin.    For example, a human insulin contains one disulfide bond within the A chain (Cys at position 6 and Cys al position ] ] from the N terminus) and two disulfide bonds between the A chain and the B chain (Cys at position 7 from the N terminus of the A chain and Cys at position 7 from the N terminus of the B chain.<br><br>
and Cys at position 20 from the N terminus of the A chain and Cys at position 19 from the N terminus of the B chain).    What is common among polypeptides having disulfide bonds is that an con-ect disulfide bond is essential for the function of the polypeptide.    According to the embodiment of the present invention, a polypeptide having disulfide bonds is a fusion protein containing a human proinsulin, MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain.    This fusion protein is highly expressed in Bacillus brevis, and the disulfide bonds are accurately and efficiently generated in co-existence with protein disulfide isomerase.    When the thus obtained fusion protein is treated with an appropriate protease (for example, thrombin and carboxypeptidase), insulin having physiological activity can be obtained.    Insulin is a phannaceutical preparation essential for the therapy of diabetes.    Examples of protease include factor Xa, enterokinase, V8 protease and TEV protease.    However in this case, it is necessary to introduce a cleavage site specific to each protease into a fusion protein so as to specifically free the targetted mature protein.<br>
With a method to allow co-expression of protein disulfide isomerase (PDI) and a polypeptide having disulfide bonds, genes encoding each protein are ligated to the 3" downstream of the above promoter so that they can be expressed polycistronically, that is, in the fomi of Promoter-SD-CWPsp-PoIypeptide-SD-CWPsp-PDI.      In such a case, it may also be Promoter-SD-CWPsp-PDI-SD-CWPsp-Polypeptide.    In the present specification, SD sequence (Shine-Daigamo sequence) is required to initiate translation of a protein at a site where mRNA binds to a ribosome, and it is preferably derived from bacteria of the genus Bacillus.    Another method involves the step of ligating a gene that encodes protein disulfide isomerase to the 3" downstream of the promoter and then ligating a gene that encodes a polypeptide having disulfide bonds to the 3" downstream of the promoter, so that the genes are ligated in a series.    That is, the series takes the fomi of<br>
Promoter-SD-CWPsp-Polypeptide-Promoter-SD-CWPsp-PDI.    In such a case, it may also be Prom oter-SD-CWPsp-PDI-Promoter-SD-CWPsp-Polypeptide.<br>
The DNA of the present invention can be prepared using a combination of leclmiques known in the ail. For example, a target DNA can be prepared by a combination of preparing each constitutive DNA sequence by a chemical synthesis method or a cloning method,<br><br>
ligating in sequence these constituents usiug ligase, and using a PCR amplification method. Specifically, the details of preparation can be understood by refemng to the examples. General tecliniques that can be used herein are described individually in Maniatis, T. et al.. Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory (1989), Innis, M.A. et al., PCR Protocols, A guide to methods and applications. Academic Press (1990) and the like.<br>
DNAs encoding human proinsulin which contains the B chain of insulin, C peptide and an A chain can be obtained from a commercially available mRJMA of human pancreas using a commercially available cDNA 1 st-strand synthesis kit or the like.    Further, if short-strand DNAs that are used as primers can be synthesized based on known DNA sequences using a commercially available DNA synthesizer, desired DNA fragments including B chain, C peptide, A chain and the like can be amplified by a general polymerase chain reaction (PCR).    DNA encoding human protein disulfide isomerase can also be obtained similarly from a commercially available mRNA of human pancreas by a PCR amplification method.    In the PCR amplification method, a cycle consisting of denaturation of DNA, amiealing with primers and an elongation reaction is repeated 20 cycles or more.<br>
According to the present invention, a vector comprising the above DNA of the present invention is further provided.    Vectors that can be used herein should have at least the following properties of: having an appropriate insertion site, that is, a restriction enzyme site, to which the DNA of the present invention can be incorporated; enabling expression of the DNA within a host cell; and being capable of autonomously replicating within the host cell.    The vector may contain a replication origin and a terminator sequence. In addifion, a selection marker, such as a drug resistance gene, a gene complementing auxotrophy or the like may be included.    Preferably, the vector of the present invention is a plasmid which is capable of replicating in bacteria of the genus Bacillus.    Examples of such vectors that can be used herein include, hut are not limited to, pNU200, pHYSOO (Proc. Natl. Acad. Sci. USA, 86:3589-3593,1989), pHY483] (J. Bacteriol., 169:1239-]245,1987), pNUlOO (Appl. Microbiol. BiotechnoL, 30:75-80,1989), pNU211(J. Biochem., 112:488-491,1992), pNU211R2L5 (Japanese Patent Application Laying-Open (Kokai) No. 7-170984), pHY700<br><br>
(Japanese Patent Application Laying-Open (Kokai) No. 4-278091), pHT210 (Japanese Patent Application Laying-Open (Kokai) No. 6-133782), pHTl 10R2L5 (AppL MicrobioL BiotechnoL, 42:358-363, 1994).    According to specific examples of the present invention, an expression vector, pNU-mPINS~hPDI* can be prepared by a construction method as shown in Fig. 2.<br>
According to the present invention, bacteria of the genus Bacillus transformed with the above-defined vector are provided.    Examples of bacteria of the genus Bacillus as hosts include, but are not limited to, Bacillus brevis strain 47 (PERM P-7224 : Japanese Patent Application Laying-Open (Kokai) No.60-58074, Japanese Patent Application Laying-Open (Kokai) No. 62-201589), 47K (Japanese Patent Application Laying-Open (Kokai) No.2"257876), from Brevibacillus choshinensis 31 OK (Japanese Patent Application Laying-Open (Kokai) No.6-296485 and HPD31 (FERJVI BP-1087 ; Japanese Patent Application Laying-Open (Kokai) No.4-278091).    Recombinant bacteria obtained by transferring the expression vector, pNU-mPINS-hPDI*, into Bacillus brevis strain 47 5Q was deposited under the terms of the Budapest Treaty on February 24, 2000 at the National Institute of Bioscience and Human, Agency of Industrial Science and technology. Ministry of economy, trade and industry, Japan (1-3, Higashi l-chome, Tsukuba-shi, Ibaraki, Japan) (the present name: the International Patent Organism Depository (IPOD), National Institute of Advanced Industrial Science and Technology, Japan (I-l, Higashi l-chome, Tsukuba-shi, Ibaraki, Japan)) under the accession No. PERM BP-7055.<br>
Expression vectors obtained as described above are transfen-ed into competent bacteria of the genus Bacillus by a method, such as electroporation (Methods in Enzymoi., 217 : 23-33,   1993).    The bacteria are cultured inappropriate media under conditions that enable expression, and then polypeptides having disulfide bonds and protein disulfide isomerase are secreted extracellularly, so that a state wherein the two can co-exist in the media can be provided.    Thus, protein disulfide isomerase can increase the efficiency of accurate formation of disulfide bonds of polypeptides having disulfide bonds.    When disulfide bonds are accurately formed in a polypeptide, the polypeptide will have physiological activity    Further, industrial applications are expected for the polypeptide<br><br>
isolated and purified from media by a combined use of methods including gel filtration chromatography, ion exchange choromatography, affinity choromatography, hydrophobic interaction chromatography, electrophoresis and isoelectric electrophoresis.    Further, a fusion protein containing proinsuiin that is used as a polypeptide having disulfide bonds in the present invention is converted to insulin by treating with tlirombin and carboxypeptidase B. Hence, the production amount of insulin having correct disulfide bonds, that is, insulin having physiological activity as a result of co-existence with protein disulfide isomerase, can be increased.<br>
Therefore, according to the present invention, there can be provided a method for increasing the efficiency of correct disulfide bond formation for gene-recombinanl polypeptides having disulfide bonds by culturing bacteria of the genus Bacillus transformed with the above expression vector in media, and providing a system wherein protein disulfide isomerase and polypeptides having disulfide bonds can co-exist extracellularly.<br>
EXAMPLES<br>
The present invention will be further described in the following examples.    The examples are not intended to limit the scope of the invention.<br>
A method employed in the following Examples to prepare DNA which encodes a fusion protein involves ligating DNA fragments amplified by a PCR reaction (polymerase chain reaction) by a ligation reaction using DNA ligase.    MWPsp represents a signal peptide of MWP protein, and MWPmp9 represents 9 amino acids from the N terminus of MWP mature protein.    Regarding MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, the letters in GSLQPR, RGHRP and PR are commonly used one-letter symbols for amino acids. B chain and A chain represent respectively the B chain and A chain of insulin.    In addition. Linker is a part of C peptide of insulin.<br>
Example 1<br>
Construction       of       vector       CprnPINS)       having       MWPsp-MWPn-iD9-GSL0PR-B<br>
chain-RGHRP-Linker-PR-A chain fusion DNA incoiporated therein<br><br>
(1) Preparation of DNA fragment, MWPsp-MWPmp9<br>
a.	Template DNA<br>
Genome DNA (840 ng) was extracted from Bacillus brevis (strain 47-5Q) by a known method (Molecular Cloning 2nd ed., A Laboratory Manual, Cold Spring Harbor Laboratory (1989)).<br>
b.	Primer<br>
Forward primer: 5"-ACACGCGCTTGCAGGATTCG-3"(SEQ ID NO: 1) Reverse primer: 5"-AGCTGTAGTAGTTGCTGC-3" (SEQ ID NO: 2)<br>
Primers were synthesized by organic chemical synthesis based on the nucleotide sequence of MWP protein determined by Yamagata, H. et a). (J. Bacteriol., 169, 1239-1245, 1987) and Tsuboi, A. et al. (J. Bacteriol., 170, 935-945, 1988). They were added at a final concentration of 0.1 fimol/L.<br>
c.	Taq DNA polymerase<br>
5 U of a commercially available product (GIBCO BRL) was added.<br>
d.	Others<br>
Tris-HCl (final concentration of 20 mmol/L, pH 8), MgCh (final concentration of 2.5 inmol/L), dNTPs (dATP, dGTP, dCTP and dTTP each at a final concentration of 50(imol/L) were added.<br>
l00 μ f of each reaction solution of a to d was put into 0.5 mL tubes, and then FOR reaction was perfonned by a known method (a reaction cycle was repeated 30 times, each cycle consisting of denaturation temperature of 94°C for 1 min, amiealing temperature of 55Cfor 1 min and DNA strand elongation temperature of 72""C for 1 min) (Imiis, M. A. et al., PCR Protocols, A guide to methods and applications. Academic Press, (1990)). Afler PCR reaction, the reaction solutions were concentrated with phenol, applied to 0.8% agarose gel, and then subjected to electrophoresis under normal conditions, thereby collecting a PCR product, that is, a DNA fragment, MWPsp-MWPmp9, from the agarose gel with ultra-free C3H (Millipore). The collected PCR product was extracted with phenol, subjected to ethanol precipitation, dried in vacuum, and then dissolved in an appropriate amount of distilled water.    Blunt end reaction was performed using a DNA blunting kit (TAKARA<br><br>
SHUZO CO., LTD.) by a method according to the instructions.<br>
(2)	Preparation of DNA fragment, proinsulin<br>
A blunt-ended DNA fragment, proinsulin, was obtained by procedures similar to (1) except for the following procedures.<br>
• As a template DNA, 10 ng of a plasmid vector having human preproinsulin DNA incorporated therein was used. The plasmid vector having the human preproinsulin DNA incorporated tlierein was obtained as follows. Human pancreas cDNA was synthesized using a 1^" strand cDNA synthesis kit (Pharmacia) from a commercially available niRNA of human pancreas (CLONTECH) according to the instructions. Using the cDNA as a template and a forward primer 5"-ATGGCCCTGTGGATGCGCC-3" (SEQ ID NO: 3) and a reverse primer 5"-CTAGTTGCAGTAGTTCTCC-3" (SEQ ID NO; 4) synthesized based on the nucleotide sequence of a human preproinsulin gene that had been determined by Bell, G. I. et al (Nature, 282, 525-527, (1979)), PCR reaction was perfomed (conditions: a reaction cycle was repeated 35 times, each cycle consisting of 94°C for I min, 60°C for 1 min and 72°C for 1 min). The resulting PCR product, that is, human preproinsulin DNA, was cloned into a pGEM-T vector (Promega).<br>
•As primers, a forward primer 5"-TTTGTGAACCAACACCTG-3" (SEQ ID NO: 5) and a reverse primer 5"-CTAGTTGCAGTAGTTCTCC-3" (SEQ ID NO: 6) were used. •PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94""C for 1 min, an aimealing temperature of 47°C for 1 min and a DNA strand elongation temperature of 72°C for 30 sec.<br>
(3)	Preparation of DNA fragment, GSLQPR-B chain-R<br>
A blunt-ended DNA fragment GSLQPR-B chain-R was obtained by procedures similar   to   (I)   except   for   the   following   procedures.      Further,   phosphorylation   (T4 poiynucleotide  kinase  (NIPPON  GENE)  was  used  according  to  the  instructions)  was performed to obtain a phosphorylated DNA fragment GSLQPR-B chain-R. •As a template DNA, the PCR product of proinsulin (10 ng) obtained in (2) was used.<br><br>
•As	primers,	forward	primer<br>
5"-GGTTCCTTGCAACCTCGTTTTGTGAACCAACACCTG-3" (SESQ ID NO: 7) and a reverse primer 5"-GCGGGTCTTGGGTGTGTA-3" (SEQ ID NO: 8) were used. •PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94°C for 1 min, an aiinealing temperature of 47°C for 1 min and a DNA strand elongation temperature of 72°C for 30 sec.<br>
(4)	Preparation of DNA fragment, Linker<br>
A blunt-ended DNA fragment, Linker, was obtained by procedures similar to (1) except for the following procedures.<br>
•As a template DNA, 10 ng of the PCR product of proinsulin obtained in (2) was used. •As primers, a forward primer 5"-GAGGCAGAGGACCTGCAG-3" (SEQ ID NO: 9) and a reverse primer 5"-CTGCAGGGACCCCTCCAG-3" (SEQ ID NO: 10) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature 94°C for 1 min, an armealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72""C for 30 sec.<br>
(5)	Preparation of DNA fragment, GHRP-Linker<br>
A blunt-ended DNA fragment GHRP-Linker was obtained by procedures similar to (4) except for the following procedures. Further, phosphorylation (T4 polynucleotide kinase (NIPPON GENE) was used according to the instructions) was performed to obtain a phosphorylated DNA fragment GHRP-Linker.<br>
•As a template DNA, 10 ngof the PCR product of the DNA fragment, Linker, obtained in (4) was used.<br>
•As a forward primer, 5"-GGTCACCGTCCAGAGGCAGAGGACCTGCAGGTGGGG-3" (SEQ ID NO: 11) was used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for I min, an amiealing temperature of<br>
55""C for 1 min and a DNA strand elongation temperature of 72°C for 30 sec.<br><br>
(6)	Preparation of DNA fragment, A chain<br>
A blunt-ended DNA fragment A chain was obtained by procedures similar to (1) except for the following procedures.<br>
•As a template DNA, 10 ng of the PCR product of proinsulin obtained in (2) was used. •As primers, a forward primer 5"-GGCATTGTGGAACAATGCTGT-3" (SEQ ID NO: 12) and a reverse primer 5"-CTAGTTGCAGTAGTTCTCCAGCTGGTA-3" (SEQ ID NO: 13) were used.<br>
• PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94c for 1 niin, an armealing temperature of 55°C for 1 min and a DNA strand elongation temperature of 72C for 30 sec.<br>
(7)	Preparation of DNA fragment, PR-A chain<br>
A blunt-ended DNA fragment PR-A chain was obtained by procedures similar to (6) except<br>
for the following procedures.    Further, phosphorylation (T4 polynucleotide kinase (NIPPON<br>
GENE) was used according to the instructions) was performed to obtain a phosphorylated<br>
DNA fragment PR-A chain.<br>
•As a template DNA, 10 ng of the PCR product of the DNA fragment A chain, obtained in (6)<br>
was used.<br>
•As a forward primer, 5"-CCACGTGGCATTGTGGAACAATGCTGT-3" (SEQ ID NO: 14)<br>
was used.<br>
•PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an amiealing temperature of<br>
55°Cfor 1 min and aDNA strand elongation temperature of 72°C for 30 sec.<br>
(8)	Preparation effusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-R<br>
A blunt-ended fusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-R was obtained by procedures similar to (1) except for the following procedures. •A template DNA used herein was prepared by mixing an appropriate amount of the DNA<br><br>
fragment MWPsp-MWPmp9 obtained in (1) and an appropriate amount of the DNA fragment<br>
GSLQPR-B chain-R obtained in (3), and then perfomiing reaction using a DNA ligation kit<br>
(TAKARA SHUZO CO., LTD.) at 16°C for 30 min.<br>
•As a reverse primer, 5"-GCGGGTCTTGGGTGTGTA-3" (SEQ ID NO: 8) was used.<br>
• PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
47°C for ] min and a DNA strand elongation temperature of 72°C for 30 sec.<br>
Subsequently, the PCR product was phosphorylated using T4 polynucleotide kinase (NIPPON GENE) according to the instructions. The phosphorylated PCR product was cut with a restriction enzyme Hinc II using a DNA ligation kit (TAKARA SHUZO CO., LTD.), and then incorporated into a vector (STRATAGENE, Blue Script SK-). According to a known method (Molecular Cloning 2nd ed., A Laboratory Majiuai, Cold Spring Harbor Laboratory (1989)), Escherichia coli DH5a was transformed, and then a piasmid DNA, the vector, was purified from the transformant. Using a forward primer (Ml3 forward primer) or a reverse primer (M13 reverse primer) for determining the nucleotide sequence of a vector, the nucleotide sequence was determined, thereby confirming that a fusion DNA MWPsp-MWPmp9-GSLQPR-B chain-R was prepared. Next, using a vector having MWPsp-MWPmp9-GSLQPR-B chain-R incorporated therein as a template DNA, a forward primer 5"-ACACGCGCTTGCAGGATTCG-3" (SEQ ID NO: 1) and a reverse primer 5"-GCGGGTCTTGGGTGTGTA-3" (SEQ ID NO: 8), a second PCR reacrion was performed by a method similar to the above method, thereby obtaining a blunt-ended fijsion DNA, MWPsp-MWPmp9-GSLQPR-B chain-R.<br>
(9) Preparation of fusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker<br>
A blunt-ended fusion DNA, MWPsp-MWPmp9-GSLQPR-B diain-RGHRP-Linker, was obtained by procedures similar to (8) except for the following procedures. •A template DNA for a first PCR reaction employed herein was prepared by mixing an appropriate amount of the fusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-R, obtained in (8) and an appropriate amount  of the DNA <br><br>
perfoming reaction using a DNA ligation kit (TAKARA SHUZO CO., LTD) at ]6°C for 30<br>
min.<br>
•As a reverse primer, 5"-CTGCAGGGACCCCTCCAG-3" (SEQ ID NO: 10) was used.<br>
(10) Preparation of vector having fusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Liiiker-PR-A chain, incorporated therein<br>
A vector (pmPINS) having a ftision DNA, MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, incoiporated therein was obtained by procedures similar to (8) except for the following procedures.<br>
•A template DNA for a first PCR reaction employed herein was prepared by mixing an appropriate amount of the fusion DNA, MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker, obtained in (9) and an appropriate amount of the DNA fragment, PR-A chain, obtained in (7), and performing reaction using a DNA ligation kit (TAKARA SHUZO CO., LTD) at 16°C for 30 min.<br>
•As	a	reverse	primer	for	a	first	PCR	reaction,<br>
5"-CTAGTTGCAGTAGTTCTCCAGCTGGTA-3"(SEQIDNO: 13) was used. • PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of 50°C for 1 min and a DNA strand elongation temperature of 72°C for 1 min.<br>
Example 2<br>
Construction       of       vector       fpmPINS     ^     hPDI*")       having       fusion       DNA.<br>
MWPsD-MWPmp9-GSL0PR-B     chain-RGHRP-Linker-PR-A     chain     -MWPsp*-hPDI*.<br>
incorporated therein<br>
(1)	Preparation	of	DNA	fragment,	MWPsp-MWPmp9-GSLQPR-B<br>
chain-RGHRP-Linker-PR-A chain<br>
A	blunt-ended	DNA	fragment,	MWPsp-MWPmp9-GSLQPR-B<br>
chain-RGHRP-Linker-PR-A chain was obtained by procedures similar to (1) of Example 1 except for the following procedures.<br><br>
•As   a   template   DNA,   10   ng   of   a   plasmid   vector   having   the   DNA   fragment,<br>
MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, incoi-porated therein was<br>
used.<br>
•As a reverse primer, 5"-CTAGTTGCAGTAGTTCTCCAGCTGGTA-3" (SEQ ID NO: 13)<br>
was used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
53°C for 1 min and a DNA strand elongation temperature of 72°C for 30 sec.<br>
(2) Preparation of DNA fragment, human protein disulfide isomerase (liPDI)<br>
A blunt-ended and phosphorylated DNA fragment, hPDI, was obtained by procedures similar to Example 1 (J) except for the following procedures.<br>
•	As a template DNA, a plasmid vector (10 ng) having hPDI DNA incorporated therein was<br>
used. The plasmid vector having hPDI DNA incorporated therein was obtained as follows.<br>
Human pancreas cDNA was synthesized using a 1^" strand cDNA synthesis kit (Pharmacia)<br>
from a commercially available niRNA of human pancreas (CLONTECH) according to the<br>
instructions.	Using the cDNA as a template and a forwai"d primer<br>
5"-GCCCCCGAGGAGGAGGACCACGTCCTG-3" (SEQ ID NO: 15) and a reverse primer<br>
5"-rTACAGTTCArCrTTCACAGCrrrCTG-3" (SEQ ID NO: 16) synthesized based on the<br>
nucleotide sequence of a human protein disulfide isomerase gene that had been determined by<br>
Pihiajaniemi, T et al., (The EMBO Journal 6:643-649, 1987), PCR reaction was performed<br>
(conditions: 35 repeated cycles, each cycle consisting of 94""C for 1 min, 60°C for 1 min and<br>
72°C for I min). The resulting PCR product, that is, DNA was cloned into a pGEM-T vector<br>
(Promega).<br>
•As primers, a forward primer 5"-GCCCCCGAGGAGGAGGACCACGTCCTG-3" (SEQ ID NO: 15) and a reverse primer 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94cfor 1 min, an annealing temperature of<br><br>
"    55°C for 1 min and a DNA strand elongation temperature of 72""C for 2 min.<br>
(3) Preparation of tiumaii protein disulfide isomerase (hPDl*) by removing restriction enzyme ApaL 1 cleavage site from human protein disulfide isomerase gene<br>
1)	Preparation of DNA fragment, hPDI-1<br>
A blunt-ended DNA fragment, hPDI-1   was obtained by procedures  similar to Example I (I) except for tlie following procedures.<br>
•A template DNA, 10 ng of a plasmid vector having hPDI DNA which had been obtained in Example 2 (2) incorporated therein was used.<br>
•As primers, a forward primer, 5"-GCCCCCGAGGAGGAGGACCACGTCCTG-3" (SEQ ID NO: 15), and a reverse primer, 5"-TTTGACGGCCTCCACCTCGT-3" (SEQ ID NO: 17) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94C for I min, an annealing temperature of<br>
55c or 1 min and a DNA strand elongation temperature of 72°C for 1 min and 30 sec.<br>
2)	Preparation of DNA fragment, hPDI-2<br>
A blunt-ended DNA fragment, hPDI-2 was obtained by procedures similar to Example 1 (1) except for the following procedures.<br>
•As a template DNA, 10 ng of a plasmid vector having hPDI DNA which had been obtained in Example 2 (2) incorpoi"aled therein was used.<br>
•As primers, a forwaid primer, 5"-AGCTTCCCCACACTCAAGTT-3" (SEQ ID NO: 18), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction condifions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature uf 72°C for 15 sec.<br>
3)	Preparation of DNA fragment, mhPDl-2<br><br>
A blunt-ended  and phosphorylated  DNA fragment, mhPDI-2 was  obtained  by procedures similar to Example 1 (1) except for the following procedures. •As a template DNA, 10 ng of the PCR product DNA, hPDl-2, obtained in Example 2 (3) 2) was used.<br>
•As primers, a forward primer, 5"-GTTCATAGCTTCCCCACACTCAAGTTC-3" (SEQ ID NO: 19), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72°C for 15 sec.<br>
4) Preparation of DNA fragment, hPDI*<br>
A blunt-ended and phosphorylated DNA fragment, hPDI* was obtained by procedures similar to Example 1 (8) except for the following procedures. •A template DNA used for a first PCR reaction was prepared by mixing an appropriate amount of the DNA, hPDI-1, obtained in Example 2 (3) 1) and an appropriate amount of the DNA fragment, mliPDl-2, obtained in 3) to perfonu reaction at 16°C for 30 min using a DNA ligation kit (TAKARA SHUZO CO. LTD).<br>
•As primers for a first PCR reaction, a forward primer, 5"-GCCCCCGAGGAGGAGGACCACGTCCTG-3" (SEQ ID NO: 15), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94""C for 1 min, an annealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72°C for 1 min and 30 sec.<br>
(4) Preparation of vector having fusion DNA, MWPsp-hPDl*, incorporated therein 1) Preparation of DNA fragment, MWPsp<br>
A biunt-ended DNA fragment, MWPsp, was obtained by procedures similar to Example 1 (1) except for the following procedures.<br><br>
•As a template DNA, 10 ng of an expression vector pNU211R2L5 of Bacillus brevis was<br>
used.<br>
•As primers, a foi-ward primer, 5"-TATACTAGAGGAGGAGAACAC-3" (SEQ ID NO: 20),<br>
and a reverse primer, 5"-TGCGAAAGCCATTGGAGCAAC-3" (SEQ ID NO: 21) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72°C for 30 sec.<br>
2) Preparation of vector having fusion DNA, MWPsp-hPDI*, incoiporated therein<br>
A vector having fusion DNA, MWPsp-hPDI*, incorporated therein was obtained by procedures similar to Example 1 (8) except for the following procedures. •A template DNA used for a first PCR reaction was prepared by mixing an appropriate amount of the DNA, MWPsp, obtained in Example 2 (4) 1) and an appropriate amount of the DNA fragment, hPDI*, obtained in Example 2 (3) to perform reaction at 16°C for 30 min using a DNA ligation kit (TAKARA SHUZO CO., LTD).<br>
•As primers for a first PCR reaction, a forward primer, 5"-TATACTAGAGGAGGAGAACAC-3" (SEQ ID NO: 20), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
55""C for 1 min and a DNA strand elongation temperature of 72°C for 1 min and 30 sec.<br>
(5) Preparation of vector having fusion DNA, MWPsp*-hPDl*, incoiporated therein<br>
MWPsp*-hPDI* prepared by fusing MWPsp*, which had been prepared by removing a cleavage site of a restriction enzyme ApaL I from MWPsp, with hPDI* was obtained as follows,<br>
1) Preparation of DNA fragment, MWPsp-1<br>
A blunt-ended DNA fragment, MWPsp-1, was obtained by procedures similar to Example 1 (1) except for the following procedures.<br><br>
•AS a template DNA, 10 ng of a vector having a fusion DNA, MWPsp-hPDl*, incorporated therein was used.<br>
•As primers, a forward primer, 5"-TATACTAGAGGAGGAGAACAC-3" (SEQ ID NO: 20), and a reverse primer, 5"-GCTAGCCAATACACTGTTAAC-3" (SEQ ID NO: 22) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
53°C for 1 min and a DNA strand elongation temperature of 72°C for 15 sec.<br>
2)	Preparation of DNA fragment, MWPsp-2-lTPDI*<br>
A blunt-ended DNA fragment, MWPsp-2-hPDI*, was obtained by procedures similar to Example I (1) except for the following procedures.<br>
•As a template DMA, 10 ng of a vector having a fusion DNA, MWPsp-hPDl*, incorporated therein was used.<br>
•As primers, a forward primer, 5"-GCTCTCGCACTTACTGTTGCTCCA-3" (SEQ ID NO: 23), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
53°C for 1 min and a DNA strand elongation temperature of 72""C for 1 min and 30 sec.<br>
3)	Preparation of vector having MWPsp*-hPDI* incorporated therein<br>
A vector having fusion DNA, MWPsp*-hPDI*, incorporated therein was obtained by procedures similar to Example 1 (8) except for the following procedures. •A template DNA used for a first PCR reaction was prepared by mixing an appropriate amount of the DNA, MWPsp-1 obtained in Example 2 (5) 1), and an appropriate amount of the DNA fragment, MWPsp-2-hPDI*, obtained in 2) to perform reaction at 16°C for 30 min using a DNA ligation kii (TAKARA SHUZO CO., LTD).<br>
•As primers for a first PCR reaction, a forward primer, 5"-TATACTAGAGGAGGAGAACAC-3"   (SEQ   ID   NO:    20),    and   a   reverse   primer,<br><br>
5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ]D NO; 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a deiiaturafion temperature of 94""C for 1 min, an aimealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72°C for 2 min.<br>
(6) Preparation of vector having MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain-MWPsp*-hPDI* incorporated therein<br>
1)	Preparation of DNA fragment, MWPsp*-hPDI*<br>
A blunt-ended and phosphorylated DNA fragment, MWPsp*-hPDI*, vyas obtained by procedures similar to Example 1 (1) except for the following procedures. •As a template DNA,   10  ng of a vector having MWPsp*-hPDI*   obtained  in (5) 4) incorporated therein was used.<br>
•As primers, a foi-ward primer, 5"-TATACTAGAGGAGGAGAACAC-3" (SEQ ID NO: 20), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
•	PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each<br>
cycle consisting of a denaturation temperature of 94°C for 1 min, an annealing temperature of<br>
55°C for 1 min and a DNA strand elongation temperature of 72°C for 2 min.<br>
2)	Preparation of vector having MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A<br>
chain-MWPsp*-hPDI* incorporated therein<br>
A vector having a fusion DNA, MWPsp-MWPiTip9-GSLQPR-B chain-RGHRP-Linker-PR-A chain-MWPsp^-hPDl*, incorporated therein was obtained by procedwes similar to Example 1 (8) except for the following procedures. •A template DNA used for a first PCR reaction was prepared by inixing an appropriate amount of the DNA, MWPsp-MWPmp9-GSLQPR-B cbain-RGHRP-Linker-PR-A chain, obtained in Example 2 (1), and an appropriate amount of the DNA fragment, MWPsp*-hPD!*, obtained in (6) 1) to perfonn reaction af I6°C for 30 min using a DNA ligation kit (TAKARA SHUZO CO., LTD).<br><br>
■ -As primers for a first PCR reaction, a forward primer, 5"-ACACGCGCTTGCAGGATTCG-3" (SEQ ID NO: 1), and a reverse primer, 5"-TTACAGTTCATCTTTCACAGCTTTCTG-3" (SEQ ID NO: 16) were used.<br>
• PCR reaction conditions employed herein comprise a reaction cycle repeated 25 times, each cycle consisting of a denaturation temperature of 94°C for ] min, an annealing temperature of 55°C for 1 min and a DNA strand elongation temperature of 72°C for 2 min and 30 sec.<br>
Example 3<br>
Expression and secretion of fusion DNA<br>
(]} Amino acid and niicJeolide sequences effusion DNA<br>
The amino acid sequence and the nucleotide sequence of the fusion DNA obtained in Example 1 corresponding to SEQ ID NOS: 24 and 25, respectively, are shown jn Fig- lA; the amino acid sequence and the nucleotide sequence (SEQ ID NO: 26) of the fusion DNA obtained in Example 2 are shown in Fig. IB; and the amino acid sequence of the fusion protein, MWPsp*-hPDI* (Fig. IB), is shown in SEQ ID NO: 27. (2) Expression and secretion of fusion DNA<br>
A fusion protein encoded by the fusion DNAs obtained in Examples 1 and 2 was expressed, A method for incorporating the fusion DNAs into an expression vector is shown in Fig. 2.<br>
Specifically, the vector pmPINS, pmPINS-hPDI* having the above fusion DNAs incorporated therein was treated with restriction enzymes ApaL I and Hind III, and then subjected to 0.8% agarose electrophoresis, thereby excising DNA fragments containing each fusion DNA. Appropriate amounts of the excised ftision DNAs and of an expression vector pNU2nR2L5 for Bacillus brevis (Japanese Patent Application Laying-Open (Kokai) No. 7-170984) that had been cut with ApaL I and Hind HI were mixed to perfonn reaction at I6X for 30 min using a DNA ligation kit (TAKARA SHUZO CO., LTD.), thereby incorporating (lie fusion DNAs into the expression vector. As described above, the expression vector pNU-mPINS, pNU-mPINS~liPDl* containing each fusion DNA was obtained. Bacillus brevis strain 47-5 (PERM BP-1664) was Iransformed with these expression vectors by a<br><br>
kjiown method (Methods in Enzymol., 217: 23-33, 1993), inoculated on T2 agar medium [polypeptone (1%), meat extract (0.5%), yeast extract (0.2%), uracil (0.1 mg/mL), glucose (1 %), erythromycin (10 pg/mL) and agar (1.5%), pH 7], thereby obtaining transformants.<br>
The transfoHTiants were cultured on T2 media that had been prepared by removing agar from T2 agar medium at 3TC for 1 day. Then, plasmid DNA was purified by a known method (Molecular Cloning 2""" ed., A Laboratory Manual, Cold Spring Harbor Laboratory (1989)), and then treated with ApaL I and Hind IH, so that incorporation of the fusion DNA was confirmed.<br>
Example 4<br>
Demonstration by SDS-PAGE of an increase in the amount of monomeric fusion protein<br>
containine insulin<br>
A fusion protein which contains insulin having disulfide bonds fonned within its molecule is a monomer. Here, an increase in the amount of monomers was examined with SDS-PAGE under non-reduction conditions. Transformants that had been confirmed to contain incorporated DNAs encoding the fusion protein containing insulin were cultured on T2 medium at 37°C for 1 day. Subsequently, the cell suspension solution was added at a proportion of 1/1000 volume to a medium [polypeptone (3%), yeast extract (0.4%), glucose (3%), MgS04-7H20 (0.01%.), MnS04-4H20 (0.001%) and erythromycin (10 ^ig/mL), pH 8] and then shake-cultured at 30 °C for 4 days.<br>
After culturing, the media were centrifuged at 15,000 rpm for 2 min to obtain the cuhure supernatant. Then, the protein was analyzed by electrophoresis using a known method (Laemmli, U. K., Nature, 227: 680-685 (1970)). Specifically, 2 jaL of a buffer under non-reduction conditions [125 mmol/L Tris-HCl (pH 6.8), 20% glycerol, 4% SDS] or a buffer under reduction conditions [125 mmol/L Tris-HCl (pH 6.8), 20% glycerol, 4%&gt; SDS, 10% 2-mercaptoethanol] was added to 18 JJL of the cuhure supernatant. The mixture was boiled for 5 min, and then suppjemented wilb 4 pL of a loading buffer [250 mmol/L Tris-HCl (pH 6.5), 50%) glycerol, 0.5% BPB]. Tlie solution was applied to a commercially available 15/25%) SDS polyacrylamide gel (Daiichi Pure Chemicals, Japan) to perform electrophoresis<br><br>
(electrophoresis buffer: 100 mmol/L Tris, 100 irunol/L Tricine, 0.1% SDS). After eleclrophoresis, Coomassie staining was performed so as Co examine the amount of monomers of a target fusion protein. As shown in Fig. 3, though the amount of the fusion protein (arrow B) containing ijisulin expi"essed and secreted from pNU-mPINS and thai fwrn pNU-mPINS-hPDI* were almost the same (predicted from the SDS-PAGE image of lanes 4 and 5 under reduction conditions), the amount of the monomer of the fusion protein expressed and secreted from pNU-mPINS-hPDI* that had been designed to allow co-existence in the media of the fusion protein containing insulin and protein disulfide isomerase (arrow A) was clearly increased (lane 3).<br>
Example 5<br>
Demonstration by western blotting of an increase in the amount of monomeric fusion protein<br>
containing insulin<br>
The amount of the monomeric fusion proteins containing insulin was increased or decreased using antibodies for C peptide (Linker poilion). Similar to Example 3, bacteria transformed with pNU-mPINS or pNU-mPINS-hPDI* vector were cultured. The resulting media were centrifuged at 15,000 rpm for 2 min. Then, 1 |iL of the culture supernatant was subjected to electrophoresis under reduction or non-reduction conditions similar to Example 3. Next, the product was blotted electrically onto a nitrocellulose membrane by a known method (Towbin, H. et al., 76: 4350-4354 (1979)). Subsequently, the membrane was immersed in a 5% skim milk solution [5% skim milk, 20 mmol/L Tris-HCl (pH 7.4), 150 mM NaCl, 0.1% Tween 20] for ] hour, immersed for 30 min in rabbit anti-C-peptide antibody (LINCO RESEARCH) diluted 2000-fold with a buffer [20 mmol/L Tris-HCl (pH 7.4), 150 mmol/L NaCl, 0.1% Tween 20] while being shaken, washed 3 times with the buffer for 10 min while being shaken, and then immersed in peroxidase-labeled anti-rabbit IgG antibodies (E-Y Laboratories) diluted 2000-fold with the buffer for 30 min while being shaken. After immersion, the membrane was washed 3 times with the buffer for 10 min while being shaken, and then increases and decreases in the amount of the monomeric fusion protein containing insulin were examined using an ECL detection kit (Amersham-Pharmacia).     This was<br><br>
performed by a method according to the kit"s instructions. As shown in Fig. 4, under rediiction conditions, almost the same amount of the monomeric fusion protein containing insulin was confirmed to be present in both pNU-mPINS (lane 3) and pNU-mPINS-hPDl* (lane 4). In contrast, under non-reduction conditions, the amount of the monomeric fusion protein was greater in pNU-mPINS-hPDI* (lane 2) than the amount in pNU-mPINS (lane 1). In pNU-mPINS, a smear yielded by reaction was observed in the high molecular range, suggesting that the fusion protein formed non-specific disulfide bonds with high polymers. That is, regarding pNU-mPINS-hPDI* wherein protein disulfide isomerase and the fusion protein containing insulin co-expressed, it was shown that a smaller number of non-specific disuJiJde bonds were formed with high polymers, and the formation of the monomers of the fusion proteins was enhanced.<br>
Example 6<br>
Demonstration by HPLC of increase in the amount of monomeric fusion protein containing<br>
insulin<br>
Bacteria transformed with pNU-mPINS, pNU-mPINS-hPDI* were cultured at 37°C for 1 day. The cell suspension solution 50 \iL was added to 50 mL of a medium [polypeptone (3%), yeast extract (0.4%), glucose (3%), MgS04 • IU2O (0.01 %), MnS04 • 4H2O (0.001 %), erythromycin (lO^g/mL), pH 8], put into a 500 mL Erienmeyer flask (6 flasks in total), and then shake-cultured at 30 °C for 4 days. The media were centrifiiged at 9,000 ipm for 20 min, and then the resulting supernatant was dialyzed with a buffer at 4°C (20 mmol/L Na-POfl, 150 mmol/L, pH 8). Subsequently, the product was centrifiiged at 10,000 ipm for 20 min, the resulting supernatant was applied to a Ni-chelate column (5x10 cm, Pharmacia), and then the above buffer was added with 60 mmol/L imidazole, thereby eluting target flision proteins. The eluted product was dialyzed against 20 mmoI/L Tris and I mmol/L EDTA (pH 8.0), supplemented with urea (final concentration of 1 mol/L) and 2-propanol (final concentration of 20%), and then applied lu a Q-Sepharose XL column (1.6 x 10 cm) (PhaiTiiacia). The product was sufficiently equilibrated with a buffer (20 mmo!/L Tris,I mmol/L EDTA, I mol/L Urea, 20% 2-propanoI, pH 8), and then eluted with a gradient<br><br>
of the buffer that was created using a solution containing IM NaCl. Fractions eluted with 160 mmoI/L to 200 mmol/L NaCl were coJJected, adjusted to pH 3 using IN UCl concentrated using an ultrafilter with a fraction molecular weight of 3,000, and then applied to Vydac214TP54 (C4 column, 4.6 x 250 mm. Cypress) to be purified by HPLC. The purified product was equilibrated with 25%i acetonitrile and 0.1%) TFA solution., and then eJuted with a gradient that was created using 33% acetonitrile and 0.1% TFA solution. Figure 5 shows the elution pattern. The fraction eluted with 30-31% acetonitrile (indicated with an arrow), namely the monomer of the fusion protein containing insulin, was dried and solidified by centrifugation and concentration. Table 1 shows the molar ratio (mol%) of the amino acid composition and total amount of the monomer of the fusion protein containing insulin that was obtained from pNU-mPINS or pNU-mPINS-hPDl*. The total amount of the monomer of the fiision protein containing insulin in pNU-mPINS-liPDI* was increased to about 4.5-fold more than the total amount of pNU-mPINS. Table 1 also shows the molar ratio (mol%)) of the amino acid composition and the total amount of the monomer of the fusion protein obtained from the transformant of MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, pNU-mPINS orpNU-mPINS-liPDl*.<br><br><br><br>
Demonstration of correct disulfide bond formation<br>
A comparison of peptide mappings was made between the fusion protein, MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, which had been converted to insulin with thi-ombin and carboxypeptidase B as shown below, and of commercially available insulin. Thus, it was demonstrated that con-ect disulfide bonds had been fonned. Figure 6 shows a schematic diagram of conversion of the fusion protein to insuHn using thrombin and carboxypeptidase B.<br>
The dried and solidified fusion protein, MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain, obtained in Example 6 was dissolved in an appropriate amount of 0.1 % TFA, and then added with 0. ] mo]/L Tris buffer (pH 8) to 20 nmoVmL. The mixture was cooled to 4°C, added with a tlirombin solution (250 [imoI/mL, substrate : enzyme = 25 : ] (molar ratio)), and 9 hours later added with an appropriate amount of 10 % trifluoroacetic acid (TFA) to pH 2 to stop the reaction. Thrombin used herein was prepared by re-purifying official thrombin (ITOHAM FOODS INC. Japan) with Macro Prep CM (Bio Rad) and Lysine Sepharose 4B (Pharmacia).<br>
To purify insulin-Arg having Arg at the C-terminus of the B chain cleaved with thrombin by reverse phase HPLC, the above reaction solution for which reaction had been ceased after treating with thrombin was applied to Mighlysil RP4 (20 x 250 mm, Cica-MERCK), equilibrated with 25% acetonitrile and 0.1% TFA solution, and then eluted with a gradient that was created using 35% acetonitrile and 0.1% TFA solution.<br>
Fractions eluted with 30-31% acetonitrile were centrifuged and concentrated. The dried and solidified fractions were subjected to the following experiment.<br>
The dried and solidified insulin-Arg was dissolved in an appropriate amount of 0.1% TFA, added with 0.1 mol/L Tris buffer (pH 8) to Img/mL, and then added with a carboxypeptidase B solution (substrate : enzyme = 500 : 1 (molar ratio), Sigma, 4.7 mg/niL) for treatment at 25°C for 12 hours. Then, an appropriate amount of 10% TFA was added to pH 2 to stop the reaction. To purify insulin from the reaction solution for which reaction had been ceased, reverse phase HPLC was performed similar to that employed for purification of<br><br>
Ihe above insulin-Arg.<br>
5 mole of the obtained insulin (the present invention) and a commercially available insulin, Novolin 40 needle (hereinafter referred to as Novolin) (Novo Nordisk Pharma), were each dissolved in 50 ^L of 0.1 mol/L ammonium bicarbonate, 2 mmol/L EDTA solution (pH 7.8). 1.35 mL of an aqueous solution V8 protease (Wake Pure Chemical Industries, Ltd., Japan, 2 )ig/mL) was added to the mixture for reaction to proceed at 25°C for 24 hours. Then, 1% TFA was added to PH2 to stop the reaction. Next, the reaction soJuiion for which reaction had been ceased was applied to Vydac218TP54 (4.6 x 250 mm, CIS column), equilibrated with 5% acetonitrile and 0.1% TFA solution, and then eluted with a gradient that was created using 35% acetonitrile and 0.]% TFA solution. Figure 7 shows the elution pattern. The insulin of the present invention and Novolin (commercially available insulin) showed similar patterns. It was concluded that the disulfide bond mechanisms of both insulins are identical.<br>
Industrial Applicability<br>
The formation of coirect disulfide bonds of a polypeptide having disulfide bonds expressed and secreted in an expression system of the bacteria of the genus Bacillus is drastically enhanced according to the present invention. This can lead to drastically increased yields of a gene recombinant polypeptide.<br>
Sequence Listing Free Text<br>
SEQIDNO: 1	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 2	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 3	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 4	Description of ailificia! sequence: a primer for PCR<br>
SEQ ID NO: 5	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 6	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 7	Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 8	Description of artificial sequence: a primer for PCR<br><br>
SEQ ID NO: 9   Description of artificial sequence: a primer for PCR<br>
SEQ ID NO; 10    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: II    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 12    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 13    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 14    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 15    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 16    Description of ajiificial sequence: a primer for PCR<br>
SEQ ID NO: 17    Description of ailificial sequence: a primer for PCR<br>
SEQ ID NO; 1S    Description of artificial sequence; a primer for PCR<br>
SEQ ID NO: 19    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 20    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 21    Description of artificial sequence: a primer for PCR<br>
SEQ ID NO: 22    Description of artificial sequence; a primer for PCR<br>
SEQ ID NO; 23    Description of artificial sequence; a primer for PCR<br>
SEQ ID NO: 24    Description of artificial sequence: sequence of fusion protein,<br>
MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain<br>
SEQ ID NO; 25 Description of artificial sequence; DNA sequence encoding fusion protein,<br>
MWPsp-MWPmp9-GSLQPR-B chain-RGHRP-Linker-PR-A chain<br>
SEQ ID NO: 26 Description of artificial sequence: DNA sequence encoding fusion protein,<br>
MWPsp-MWPmp9-GSLQPR-Bchain-RGHRP-Unker-PR-Achain-MWPsp*-hPDl*<br>
SEQ   ID   NO;   27   Description   of   artificial   sequence:   sequence   of   fusion   protein,<br>
MWPsp*-hPDJ*<br>
Various publications are cited in the above description. All of these publications and patent applications are incorporated herein by reference. Further, the present invention may be modified or changed in various ways within a scope equivalent lo that of the inventions described in the scope for attached claims, and all the equivalents are included within the scope of the present invention.<br><br><br>
WE CLAIM:<br>
1.	A fuel cell, said fuel cell having an anode (1) comprising:<br>
an anode active material (3) having hydrogen storage capacity;<br>
said anode active material having a gaseous hydrogen contacting surface, an electrolyte contacting surface (5), and the bulk of said anode active material disposed between said gaseous hydrogen contacting surface (6)and said electrolyte contacting surface;<br>
said gaseous hydrogen contacting surface is adapted to dissociate and adsorb said gaseous hydrogen; said bulk of said anode active material is adapted to store said adsorbed hydrogen;<br>
said electrolyte contacting surface is adapted to react said stored hydrogen with an electrolyte solution.<br>
2.	The fiiel cell as claimed in claun 1, wherein said anode comprises a substrate component (4) which provides for both electrical conductivity and mechanical support and comprises an electrically conductive mesh, grid, foam, matte, foil, plate, or expanded metal.<br>
3.	The fuel cell as claimed in claim 1, wherein said anode active material is a hydrogen storage alloy which does not comprise noble metals.<br>
4.	The fuel cell as claimed in claim 3, wherem said hydrogen storage alloy is selected from the group consisting of rare-earth/Misch metal alloys, zirconium alloys, titanium alloys, and mixtures of alloys thereof<br>
5.	The fuel cell as claimed in claim 4, wherein said hydrogen storage alloy has the following composition:<br>
(Base Alloy) aCObMncFedSne<br><br>
where the Base Alloy comprises 0.1 to 60 atomic percent Ti, 0.1 to 40 atomic percent Zr, 0 to 60 atomic percent V, 0.1 to 57 atomic percent Ni, and 0 to 56 atomic percent Cr; b is 0 to 7.5 atomic percent; c is 13 to 17 atomic percent; d is 0 to 3.5 atomic percent; e is 0 to 1.5 atomic percent; and a+b+c+d+e=100 atomic percent.<br>
6.	The fuel cell as claimed in claim 1, wherein said fuel cell comprises an anode which<br>
comprises a hydrophobic component.<br>
7.	The fuel cell as claimed in claim 6, wherein said hydrophobic component is polytetrafluoroethylene (PTFE).<br>
8.	The fuel cell as claimed in claim 1, wherein said anode comprises a component which provides for electrical conductivity and comprises an electrically conductive powder intimately mixed with said hydrogen storage material.<br>
9.	The fuel cell as claimed in claim 8, wherein said electrically conductive powder<br>
comprises at least one material selected from the group consisting of copper, a copper alloy,<br>
nickel, a nickel alloy, and carbon.<br>
10.	The fuel cell as claimed m claim 2, wherein said mesh, grid, foam, or expanded metal is<br>
formed from nickel, nickel alloy, copper, copper plated nickel or a copper-nickel alloy.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTQyNC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1424-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="202825-a-method-and-a-transcoding-device-for-modifying-data-in-an-encoded-signal.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="202827-a-process-for-the-polymerization-of-an-olefin-monomer-and-a-catalyst-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>202826</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/1424/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Oct-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Sep-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. ENERGY CONVERSION DEVICES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1675 W. Maple Road, Troy, MI 48084</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OVSHINSKY, Stanford, R.</td>
											<td>2700 Squirrel Road, Bloomfield Hills, MI 48304</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VENKATESAN, Srinivasan</td>
											<td>30115 Woodgate Drive, Southfield, MI 48076</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ALADJOV, Boyko</td>
											<td>1415 Brians Way, Rochester Hills, MI 48037</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YOUNG, Rosa</td>
											<td>4136 Golf Ridge Drive, East Bloomfield Hills, MI 48302</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HOPPER, Thomas</td>
											<td>9902 Hendrickson Clawson, MI 48017</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>H01M 4/86</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2001/007864</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-03-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/524,116</td>
									<td>2000-03-13</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/202826-a-fuel-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:58:44 GMT -->
</html>
