<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/202921-a-method-producing-a-dipeptide-forming-enzyme-and-a-method-of-producing-a-dipeptide-therewith by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:19:26 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 202921:A METHOD PRODUCING A DIPEPTIDE-FORMING ENZYME AND A METHOD OF PRODUCING A DIPEPTIDE THEREWITH</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD PRODUCING A DIPEPTIDE-FORMING ENZYME AND A METHOD OF PRODUCING A DIPEPTIDE THEREWITH</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a method for producing dipeptide using inexpensively acquirable starting materials by an industrially advantageous and simple pathway. Dipeptide is produced from l-amino acid ester and L-amino acid using a culture of microbes having the ability to produce a dipeptide from an L-amino acid user ester and an L-amino acid, using microbial cells isolated from the culture, or a treated microbial cell product of the microbe.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
SPECIFICATION<br>
TITLE OF THE INVENTION<br>
Peptide-forming enzyme gene, peptide-forming enzyme, and dipeptide producing method<br>
TECHNICAL FIELD<br>
The present invention relates to a method for producing dipeptide conveniently and economically without going through a complicated synthesis method, and more particularly to a peptide-forming enzyme gene, to a peptide-forming enzyme, and to a method for producing dipeptide using the enzyme.<br>
BACKGROUND ART<br>
Dipeptides are used in the field of pharmaceutical materials and functional foods and various fields.   For example, L-alanyl-L-glutamine is used as a component of serum-free media, and is used for infusion components since it has greater stability and higher solubility than L-glutamine.<br>
Chemical synthesis methods, which have been conventionally known as methods of producing dipeptides, are not necessarily simple. Known examples of such methods include a method that uses N-benzyloxycarbonylalanine (hereinafter, "Z-alanine") and protected L-glutamine (see Bull. Chem. Soc. Jpn., 34, 739 (1961), Bull. Chem. Soc. Jpn., 35, 1966 (1962)), a method that uses Z-alanine and protected<br><br>
L-glutamate-y-methyl ester (see Bull. Chem. Soc. Jpn., 37, 200 (1964)), a method that uses a Z-alanine ester and unprotected glutamic acid (see Japanese Patent Application Laid-Open Publication No. H1-96194), and a method that uses a 2-substituted-propionyl fraloidhalide as raw material and synthesizes an N-(2-substituted)-propionyl glutamine derivative as an intermediate (see Japanese Patent Application Laid-Open Publication No. H6-234715).<br>
However, in all of these methods, the introduction and elimination of a protecting group or the synthesis of an intermediate is required, so that these production methods have not been sufficiently satisfactory in view of their industrial advantages.    Known examples of typical dipeptide production methods using enzymes include a condensation reaction using an N-protected, C-unprotected carboxy component and an N-unprotected, C-protected amine component (Reaction 1), and a substitution reaction using an N-protected, C-protected carboxy component and an N-unprotected, C-protected amine component (Reaction 2).   An example of Reaction 1 is a production method of Z-aspartylphenylalanine methyl ester from Z-aspartic acid and phenylalanine methyl ester (see Japanese Patent Application Laid-Open Publication No. S53-92729), while an example of Reaction 2 is a production method of acetylphenylalanylleucine amide from acetylphenylalanine ethyl ester and leucine amide (see Biochemical J., 163, 531 (1977)).    There are extremely few examples of research reports that describe methods using N-unprotected, C-protected carboxy components. An example of a substitution reaction using an N-unprotected, C-protected carboxy component and an N-unprotected, C-protected amine component<br><br>
(Reaction 3) is described in Patent WO 90/01555, and example of such a reaction is a production method of arginyl leucine amide from arginine ethyl ester and leucine amide.   An example of a substitution reaction using an N-unprotected, C-protected carboxy component and an N-unprotected, C-unprotected amine component (Reaction 4) is described in Patent EP 278787A, and example of such a reaction is a production method of tyrosyl alanine from tyrosine ethyl ester and alanine.   Among the methods those that are able to serve as the least expensive production methods are naturally those that fall in the range of Reaction 4 involving the fewest number of protecting groups.<br>
However, the enzyme used in the example of the prior art of the Reaction 4 (see Patent EP 278787A) is a comparatively expensive carboxypeptidase preparation derived from molds and plants, and the dipeptides that were produced contained amino acids that are comparatively highly hydrophobic.    For the Reaction 4, there is no known method that uses an enzyme of bacterial or yeast origin, and there has been known no method for producing highly hydrophilic alanylglutamine or alanylasparagine. Under such circumstances, there has been a need for the development of an inexpensive industrial method for the production of such peptides. On the other hand, proline iminopeptidase is an enzyme that catalyzes a reaction that cleaves an N-terminal proline from a peptide having proline on its N-terminal, and this enzyme is known to exist in numerous species of organisms.    For example, it is known to exist in higher animals such as guinea pigs (brain) (see J. Biol. Chem., 258, 6147-6154 (1983)), rats (brain and kidneys) (see Eur. J. Biochem., 190, 509-515<br><br>
(1990)), higher plants such as apricot seeds (see J. Biochem., 92, 413-421 (1982)), oral cavity spirochetes such as Trichoderma denticola (see Infect. Immun., 64, 702-708 (1996), filamentous fungi such as Penicillium species (see Japanese Patent Application Laid-Open Publication No. H1-215288), Basidiomycetes such as shiitake mushrooms (see Japanese Patent Application Laid-Open Publication No. S58-36387), Actinomycetes such as Streptomvces bJJGatuspIicatus (see Biochem. Biophys. Res. Commun., 184, 1250-1255 (1992), and bacteria such as Corynebacterium variabilis (see J. Appl. Microbiol., 90, 449-456 (2001)).     ,<br>
In addition, concerning proline iminopeptidase gene, there have been reported cloning and base sequences of genes of Arthrobacter nicotiana (see FEMS Microbiol. Lett., 78, 191-197 (1999)), Escherichia coli (see Japanese Patent Application Laid-Open Publication No. H2-113887), Flavobacterium meningosepticum (see Arch. Biochem. Biophys., 336, 35-41 (1996)),    Hafnia alvei (see J. Biochem., 119, 468-474 (1996)), Lactobacillus debfueokiidelbrueckii (see Microbiology, 140, 527-535 (1994)), Bacillus coagulans source (see J. Bacterid., 174, 7919-1925 (1994)), Aeromonas sobria source (see J. Biochem., 116, 818-825 (1994)), Xanthomonas campestris (see   Japanese Patent Application Laid-Open Publication No. H9-121860), Neisseria gonorrhoeae (see Mol. Microbiol., 9, 1203-1211 (1993), Propionibacterium freundenreichii (see Appl. Environ. MicorbioL, 64, 4736-4742 (1998)), Serratia marcescens (see J. Biochem., 122, 601-605 (1997)) and Thermoplasma acidophilum (see FEBS Lett, 398, 101-105 (1996)).<br>
In addition, base sequences predicted to encode proline<br><br>
imjnopeptidase have recently been reported in numerous species of organisms as a result of analyses on the whole genomes of microbes.   For example, the whole genome base sequence of Pseudomonas aeruginosa has been reported (see Nature, 406, 959 (2000)), and a base sequence was . found therein that is predicted to encode proline iminopeptidase.<br>
On the other hand, it has been found that a proline-containing dipeptide is formed when an ester of L-proline or DL-proline and an alpha-amino acid are allowed to react using proline iminopeptidase (see Japanese Patent Application Laid-Open Publication No. H3-13391). However, although proline iminopeptidase is an enzyme that catalyzes a reaction that cleaves the N-terminal proline from a peptide having proline on its N-terminal, and a prolyl amino acid would be naturally considered to be produced from proline ester and amino acid, the synthesis of peptide from an amino acid and amino acid ester other than proline using proline iminopeptidase has been completely unknown.    Of course, the synthesis of L-alanyl-L-glutamine from L-alanine ethyl ester hydrochloride and L-glutamine has been also previously unknown.    In addition, although the partial base sequence of proline iminopeptidase of Pseudomonas putida strain ATCC 12633 was disclosed (AF032970), there has been no study conducted whatsoever on its activity, including its detection.<br>
DISCLOSURE OF THE INVENTION<br>
An object of the present invention is to provide a method for producing a dipeptide by an industrially advantageous and simple pathway using a starting material that can be acquired inexpensively and an enzyme<br><br>
source that can be supplied inexpensively (such as a microbial culture, microbial cells or a treated microbial cell product of a microbe).<br>
As a result of extensive research in consideration of the aforementioned object, the inventors of the present invention have found that proline iminopeptidase has the ability to produce a peptide from an L-amino acid ester and an L-amino acid.    In addition, the inventors of the present invention also have cloned and expressed the gene of the enzyme and also clearly demonstrated the broad substrate specificity of the enzyme in peptide production using purified recombinant enzymes, thereby leading to completion of the present invention.<br>
NametVr-tfreThe present invention is as described below.provides: [1]        A,protein indicated in (A) or (B)-belew:<br>
(A)	a protein having an amino acid sequence described in SEQ ID NO: 5 of the Sequence Listing,<br>
(B)	a protein consisting of an amino acid sequence that includes substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 5 of the Sequence Listing, and has activity to produce a dipeptide from an L-amino acid ester and an L-amino acid.<br>
[2]     A protein taefeateet-iR (C) or (D)-belew:<br>
(C)	a protein having an amino acid sequence described in SEQ ID NO: 15 of the Sequence Listing,<br>
(D)	a protein consisting of an amino acid sequence that includes substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 15 of the<br><br>
Sequence Listing, and has activity to produce a dipeptide from an L-amino<br>
acid ester and an L-amino acid.<br>
[3]        A protein inetiGate^-in (E) or (F)-feeiow:<br>
(E)	a protein having an amino acid sequence described in SEQ ID NO:<br>
17 of the Sequence Listing,<br>
(F)	a protein consisting of an amino acid sequence that includes<br>
substitution, deletion, insertion, addition or inversion of one or a plurality of<br>
amino acids in the amino acid sequence described in SEQ ID NO: 17 of the<br>
Sequence Listing, and has activity to produce a dipeptide from an L-amino<br>
acid ester and an L-amino acid.<br>
[4]        A DNA mdiGated-lf* (a) or (b)-betew:<br>
(a)	a DNA consisting of a base sequence consisting of bases numbers<br>
57 to 1295 in a base sequence described in SEQ ID NO: 4 of the Sequence<br>
Listing,<br>
(b)	a DNA that hybridizes under stringent conditions with a DNA<br>
consisting of a base sequence consisting of bases numbers 57 to 1295 in<br>
the base sequence described in SEQ ID NO: 4 of the Sequence Listing, and<br>
encodes a protein having activity to form a dipeptide from an L-amino acid<br>
ester and an L-amino acid.<br>
i     [5]	A DNA fndiGated4n (c) or (d)-beJew:<br>
(c)	a DNA consisting of a base sequence consisting of bases numbers 486 to 1496 in a base sequence described in SEQ ID NO: 14 of the Sequence Listing,<br>
(d)	a DNA that hybridizes under stringent conditions with a DNA<br>
i     consisting of a base sequence consisting of bases numbers 486 to 1496 in<br><br>
the base sequence described in SEQ ID NO: 14 of the Sequence Listing, and encodes a protein having activity to form a dipeptide from an L-amino acid ester and an L-amino acid. [6]        A DNA mdioated-ln (e) or (f)-betew:<br>
(e)	a DNA consisting of a base sequence consisting of bases numbers 311 to 1279 in a base sequence described in SEQ ID NO: 16 of the Sequence Listing,<br>
(f)	a DNA that hybridizes under stringent conditions with a DNA consisting of a base sequence consisting of bases numbers 311 to 1279 in the base sequence described in SEQ ID NO: 16 of the Sequence Listing, and encodes a protein having activity to form a dipeptide from an L-amino acid ester and an L-amino acid.<br>
[7]   The DNA according to any one of [4] to [6] above, wherein the stringent<br>
conditions are conditions under which washing is carried out at 60°C and at<br>
a salt concentration equivalent to 1 x SSC and 0.1% SDS.<br>
[8]   A recombinant DNA comprising incorporated therein the DNA<br>
according to any one of [4] to [7] above.<br>
[9]   A transformed cell comprising incorporated therein the DNA according<br>
to any one of [4] to [7] above in a state where the DNA is able to express a<br>
protein encoded thereby.<br>
[10]   A method for producing a dipeptide-forming enzyme, comprising:<br>
culturing the transformed cells according to [9] above in a medium, and<br>
accumulating a protein having activity to produce the dipeptide from an<br>
L-amino acid ester and an L-amino acid in the medium and/or in the<br>
transformed cells.<br><br>
[11]   A method for producing a dipeptide, comprising: producing a dipeptide from an L-amino acid ester and an L-amino acid using a protein having activity to form the dipeptide from an L-amino acid ester and an L-amino acid that is produced in the transformed cells according to [9] above. [12]   The method for producing a dipeptide according to [11] above, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-a-ester, an L-aspartic acid-p-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a,(3-dimethyl ester and an L-glutamine-y-ester. [13]   The method for producing a dipeptide according to [11 ] or [12] above, wherein the L-amino acid is one or more types selected from the group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine and L-glutamate. [14]      A method for producing a dipeptide comprising: allowing a protein having proline iminopeptidase activity to act on an L-amino acid ester and an L-amino acid to form the dipeptide.<br>
[15]      The method for producing dipeptide according to [14] above, wherein the protein having proline iminopeptidase activity is derived from a microbe belonging to genus Corynebacterium, Pseudomonas or Bacillus. [16]      The method for producing a dipeptide according to [14] above, &gt;     wherein the protein having proline iminopeptidase activity is derived from<br><br>
any of Corynebacterium glutamicum, Pseudomonas putida and Bacillus coagulans.<br>
[17]      The method for producing a dipeptide according to any one of [14] to [16] above, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-cc-ester, an L-aspartic acid-(3-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a.p-dimethyl ester and an L-glutamine-y-ester. [18]      The method for producing a dipeptide according to any one of [14] to [17] above, wherein the L-amino acid is one or more types selected from the group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine and L-glutamate. [19]      A method for producing a dipeptide, comprising: producing the dipeptide from an amino acid ester and an amino acid using a culture of a microbe belonging to the genus Corynebacterium, Pseudomonas or Bacillus and having the ability to produce the dipeptide from the amino acid ester and the amino acid, microbial cells isolated from the culture or a treated microbial product of the microbe.<br>
[20]      The method for producing a dipeptide according to [19] above, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an<br><br>
L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-a-ester, an L-aspartic acid-p-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a,[3-dimethyl ester and an L-glutamine-y-ester. [21]      The method for producing a dipeptide according to [19] above, wherein the L-amino acid is one or more types selected from the group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine and L-glutamate.<br>
	The other objects, features and advantages of the present invention<br>
are specifically set forth in or will become apparent from the following detailed descriptions of the invention when read in conjunction with the accompanying drawing.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Fig, 1 is a graph illustrating dipeptide-forming activity when an inhibitor is added.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
The present invention provides a novel protein having activity to produce a dipeptide from an L-amino acid ester and an L-amino acid, a DNA encoding that protein, and a method for producing a dipeptide using these.   The reaction in the dipeptide production method of the present invention is expressed as shown in the reaction scheme below.   As exemplified in the following chemical formulae, a "dipeptide" as used herein<br><br>
refers to a peptide polymer having one dipeptide bond.<br>
(GhemiGal-F©fmu-la-1)<br>
,   RrCH(NH22)-COOR + H22N-CH(COOH)-R2<br>
-&gt; RrCH(NH22)-CONH-CH(COOH)-R2 + ROH<br>
(in the formula, R represents a substituted or non-substituted hydrocarbon chain, Ri represents an amino acid ester side chain, and R2 represents an amino acid side chain.)<br>
Amino acid esters are compounds that can be acquired inexpensively.   The method of the present invention, in which starting materials in the form of an amino acid ester and unprotected amino acid are allowed to react in an aqueous solution using bacteria, yeast and so forth as the enzyme source, is a novel dipeptide production method not found conventionally, and is capable of inexpensively providing useful dipeptides for use in pharmaceutical materials and functional foods.<br>
Hereinafter, the present invention will be described in detail in the following order:<br>
[I]	Microbes having the ability to produce dipeptides from<br>
L-amino acid esters and L-amino acids<br>
[II]	Isolation of DNA encoding protein having peptide production<br>
»<br>
activity<br>
[III]	Properties of peptide-forming enzyme<br>
[IV]	Dipeptide production method.<br><br>
[I]	Microbes Having the Ability to Produce Dipeptides from L-Amino<br>
Acid Esters and L-Amino Acids<br>
Microbes having the ability to produce dipeptides from L-amino acid esters and L-amino acids can be used without any particular restrictions as the microbe used in the present invention.    Examples of microbes having the ability to produce dipeptides from L-amino acid esters and L-amino acids include Bacillus species, Corynebacterium species and Pseudomonas species, and specific examples thereof are indicated below. Bacillus subtilis ATCC 6633<br>
Bacillus coagulans EK01 (J. Bacterid. 174, 7919-7925 (1992)) Corynebacterium glutamicum ATCC 13286 Pseudomonas putida AJ-2402 FERM BP-8101 Pseudomonas putida ATCC 12633 Pseudomonas putida AJ-2048 FERM BP-8123<br>
Those strains of microbes listed above that are indicated with ATCC numbers are deposited at the American Type Culture Collection (P.O. Box 1549, Manassas, VA 20110), and subcultures can be furnished by referring to each number.<br>
Those strains of microbes listed above that are indicated with FERM numbers are deposited at the independent administrative corporation, International Patent Organism Depositary of the National Institute for Advanced Industrial Science and Technology (Chuo Dai-6, 1-1 Higashi 1-Chome, Tsukuba-shi, Ibaraki-ken, 305-8566, Japan) and have been assigned a deposit number.    Pseudomonas putida strain AJ-2402 was deposited at the independent administrative corporation, International<br><br>
Patent Organism Depositary of the National Institute of Advanced Industrial Science and Technology on October 1, 2001, was assigned the deposit number of FERM P-18544, control of this strain was later transferred to international deposition on July 1, 2002 and assigned the deposit number of FERM BP-8101.    Note that FERM BP-8101 (AJ-2402) has been identified as the aforementioned Pseudomonas putida based on the following classification experiment.    In addition, Pseudomonas putida strain AJ-2048 was deposited at the independent administrative corporation, International Patent Organism Depositary of the National Institute of Advanced Industrial Science and Technology on July 22, 2002, and assigned the deposit number of FERM BP-8123.<br>
Pseudomonas putida strain FERM BP-8101 is a motile, aconidial rod that was identified as a bacterium belonging to the genus Pseudomonas based on the properties of being a Gram negative rod (0.7 to 0.8 x 1.5 to 2.0 micrometers (i-im)) that forms no spore, is motile, forms round, glossy, cream-colored colonies with a completely smooth or undulating border, grows at 30°C, and is catalase positive, oxidase positive and OF test (glucose) negative.    Moreover, it was identified as Pseudomonas putida based on the physiological properties of being nitrate reduction negative, indole production negative, acid production from glucose negative, arginine dihydrolase positive, urease negative, esculin hydrolysis negative, gelatin hydrolysis negative, (3-galactosidase negative, glucose assimilation positive, L-arabinose assimilation negative, D-mannose assimilation positive, D-mannitol assimilation positive, N-acetyl-D-glucosamine assimilation negative, maltose assimilation negative, potassium gluconate assimilation<br><br>
positive, n-capric acid assimilation positive, adipic acid assimilation negative, dl-malicacid assimilation positive, sodium citrate assimilation positive, phenyl acetate assimilation positive, oxidase positive, fluorochrome production on King's B agar medium positive, levan production from sucrose positive, and weak assimilation of sorbitol.<br>
Wild strains or variant strains may be used for these microbes, and recombinant strains and so forth derived by cell fusion, genetic manipulation or other genetic techniques may also be used.<br>
To obtain microbial cells of these microbes, the microbes can be cultured and grown in a suitable medium.   There is no particular restriction on the medium used for this purpose so far as it allows the microbes to grow. This medium may be an ordinary medium containing ordinary carbon sources, nitrogen sources, phosphorous sources, sulfur sources, inorganic ions, and organic nutrient sources as necessary.<br>
For example, any carbon source may be used so far as it can be utilized by the aforementioned microbes, and specific examples of which that can be used include sugars such as glucose, fructose, maltose, and amylose, alcohols such as sorbitol, ethanol and glycerol, organic acids such as fumaric acid, citric acid, acetic acid and propionic acid and their salts, carbohvdrateshvdrocarbons such as paraffin as well as mixtures thereof.<br>
Examples of nitrogen sources that can be used include ammonium salts of inorganic salteacids such as ammonium sulfate and ammonium chloride, ammonium salts of organic acids such as ammonium fumarate and ammonium citrate, nitrates such as sodium nitrate and potassium nitrate, organic nitrogen compounds such as peptones, yeast extract, meat extract<br><br>
and corn steep liquor as well as mixtures thereof.<br>
In addition, nutrient sources used in ordinary media, such as inorganic salts, trace metals and vitamins, can also be suitably mixed and used.<br>
There is no particular restriction on culturing conditions, and<br>
culturing may be carried out, for example, for about 12 to 48 hours while<br>
suitably controlling the pH and temperature to a pH range of 5 to 8 and a<br>
temperature range of 15 to 40°C under aerobic conditions.<br>
[II]	Isolation of DNA Encoding Protein Having Peptide-forming enzyme<br>
Activity [11-1]     DNA Isolation<br>
The inventors of the present invention have isolated and determined the sequence of DNA of the present invention that encodes protein having activity to synthesize a dipeptide from an L-amino acid ester and an L-amino acid.   A DNA consisting of the base sequence consisting of bases numbers 57 to 1295 described in SEQ ID NO: 4 of the Sequence Listing was isolated from Corynebacterium glutamicum strain ATCC 13286.   In addition, a DNA consisting of the base sequence consisting of bases numbers 486 to 1496 described in SEQ ID NO:. 14 of the Sequence Listing was isolated from Pseudomonas putida strain ATCC 12633.    Moreover, a DNA consisting of the base sequence consisting of bases numbers 311 to 1279 described in SEQ ID NO: 16 of the Sequence Listing was isolated from Pseudomonas putida strain FERM BP-8123.    Note that in the present specification, "base sequence described in SEQ ID NO: 4", "base sequence described in SEQ ID NO: 14", and "base sequence described in SEQ ID NO: 16" refer to the<br><br>
CDS portions unless specifically indicated otherwise.<br>
An example of isolating DNA is shown.    First, the amino acid sequence of a purified peptide-forming enzyme is first determined.    The amino acid sequence can be determined using Edman's method (see Edman, P., Acta Chem. Scand., 4, 227 (1950)).    In addition, the amino acid sequence can also be determined using a sequencer manufactured by Applied Biosystems.   The amino acid sequence is determined for the N-terminal or about 10 to 30 residues of the peptide obtained by treatment with lysyl endopeptidase and so forth for the purified peptide-forming enzyme, and the base sequence of DNA that encodes it can be deduced based on the determined amino acid sequence.   Universal codons are employed for deducing the base sequence of the DNA.<br>
Then, a DNA molecule of about 30 base pairs is synthesized based on the deduced base sequence.   A method for synthesizing this DNA molecule is disclosed in Tetrahedron Letters, 22, 1859 (1981).    In addition, the DNA molecule can also be synthesized using a sequencer manufactured by Applied Biosystems.   A DNA that encodes peptide-forming enzyme can then be amplified from a chromosomal DNA by the PCR method using the DNA molecule as a primer.    However, since the DNA that has been amplified using the PCR method does not contain the full-length DNA that encodes peptide-forming enzyme, the full-length DNA that encodes peptide-forming enzyme is isolated from a chromosomal gene library of various microbial cells such as Corynebacterium glutamicum, Pseudomonas putida or Bacillus subtilis using the DNA amplified by the PCR method as a probe.<br><br>
Alternatively, in the case where a portion of the base sequence of the gene is known, the full-length DNA that encodes a peptide-forming enzyme can be isolated from a chromosomal gene library using the DNA having the known sequence as a probe.<br>
Moreover, in the case where the base sequence of the gene has homology with a known sequence, the full-length DNA that encodes a peptide-forming enzyme can be isolated from a chromosomal gene library using a DNA having the known sequence as a probe.<br>
The procedure for the PCR method is described in, for example, White, T.J. et al., Trends Genet., 5, 185 (1989).   A method for preparing a chromosomal DNA, as well as a method for isolating a target DNA molecule from a gene library using a DNA molecule as a probe, are described in Molecular Cloning, 2nd edition, Cold Spring Harbor Press (1989).<br>
A method for determining the base sequence of isolated DNA that encodes a peptide-forming enzyme is described in A Practical Guide to Molecular Cloning, John Wiley &amp; Sons, Inc. (1985).    In addition, the base sequence can also be determined using a DNA sequencer (Applied Biosystems).    DNAs that encode peptide-forming enzymes isolated from Corynebacterium glutamicum strain ATCC 13286, Pseudomonas putida strain ATCC 12633, and Pseudomonas putida strain FERM BP-8123 in this manner are shown in SEQ ID NOs: 4, 14, and 16, respectively.<br>
DNAs that can be used in the present invention are not only the DNAs specified by SEQ ID NOs: 4, 14, and 16.    For example, in explaining the DNA of SEQ ID NO: 4 isolated from Corynebacterium glutamicum strain ATCC 13286 below, even a DNA that has been artificially mutated to a DNA<br><br>
that encodes a peptide-forming enzyme isolated from the chromosomal DNA of Corynebacterium giutamicum strain ATCC 13286 is also a DNA of the present invention so far as it encodes a peptide-forming enzyme.   A frequently used method for artificial mutating a DNA is the site-directed mutagenesis method described in Methods in Enzymol., 154 (1987).<br>
In addition, a DNA having a base sequence which hybridizes under stringent conditions with a polynucleotide having a base sequence complementary to a base sequence described in SEQ ID NO: 4 of the Sequence Listing, and encodes a protein having peptide-forming enzyme activity, can also be used as a DNA in the present invention.<br>
Moreover, a DNA substantially identical to the DNA of the present invention can also be obtained by isolating a DNA that hybridizes under stringent conditions with a probe prepared based on a DNA consisting of the CDS described in SEQ ID NO: 4 of the Sequence Listing, and encodes protein having peptide-forming enzyme activity.   A probe can be prepared according to established methods based on, for example, the base sequence described in SEQ ID NO: 4 of the Sequence Listing.    In addition, a method in which a target DNA is isolated by using a probe and picking up a DNA that hybridizes with it may also be carried out in accordance with established methods.    For example, a DNA probe can be prepared by amplifying a base sequence cloned in a plasmid or phage vector and extracting the base sequence desired to be used as a probe by cleaving with a restriction enzyme.   The location where the sequence is cleaved can be adjusted corresponding to the target DNA.<br>
The "stringent conditions" mentioned herein refer to conditions<br><br>
under which a so-called specific hybrid is formed while non-specific hybrids are not formed.   Although it is difficult to clearly quantify these conditions, examples of such conditions include conditions under which a DNA having a high degree of homology, such as a DNA having a homology of 50% or more, preferably 80% or more, and more preferably 90% or more, hybridizes while a DNA having a low degree of homology does not hybridize, or conditions under which a DNA hybridizes at 60°C and at a salt concentration equivalent to 1xSSC and 0.1% SDS, which are the conditions for washing of ordinary Southern hybridization, preferably at 60°C and at a salt concentration equivalent to 0.1 x SCC and 0.1% SDS, and more preferably at 65°C and at a salt concentration equivalent to 0.1 x SSC, and 0.1% SDS.   The activity of a peptide-forming enzyme is as previously explained.    However, in the case of a base sequence that hybridizes under stringent conditions with a base sequence complementary to the base sequence described in SEQ ID NON: 4 of the Sequence Listing, it preferably retains 10% or more, and preferably 50% or more, of the enzyme activity of a protein having the amino acid sequence described in SEQ ID NO: 4 of the Sequence Listing under conditions of 50°C and pH 8.<br>
Moreover, a protein substantially identical to the peptide-forming enzyme encoded by a DNA described in SEQ ID NO: 4 of the Sequence Listing can also be used in the present invention.   Thus, a DNA that encodes "a protein having an amino acid sequence including substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 5 of the Sequence Listing, and having peptide-forming enzyme activity to catalyze a reaction<br><br>
that produces a dipeptide from an L-amino acid ester and an L-amino acid" can also be used in the present invention.   The term "a plurality of refers to that over a range that does not significantly impair the three-dimensional structure of the protein of the amino acid residues or the activity of a peptide-forming enzyme, and more specifically is a value of 2 to 50, preferably a value of 2 to 30, and more preferably a value of 2 to 10.    In addition, the activity of a peptide-forming enzyme is as previously explained. However, in the case of an amino acid sequence including substitution, deletion, insertion, addition or inversion of one or a plurality of amino acid residues in the amino acid sequence described in SEQ ID NO: 5 of the Sequence Listing, it preferably retains 10% or more, and preferably 50% or more, of the enzyme activity of a protein having the amino acid sequence described in SEQ ID NO: 5 of the Sequence Listing under conditions of 50°C and pH 8.<br>
As has been described above, in addition to a DNA isolated from<br>
Corynebacterium glutamicum strain ATCC 13286, a DNA isolated from<br>
Pseudomonas putida strain ATCC 12633 and a DNA isolated from<br>
Pseudomonas putida strain FERM BP-8123, DNAs substantially identical to<br>
these DNAs are also provided as DNAs of the present invention.    Namely,<br>
examples of DNAs provided by the present invention are as follows:<br>
(i)	A DNA consisting of a CDS described in SEQ ID NO: 4, 14, or 16 of<br>
the Sequence Listing;<br>
(ii)        A DNA that hybridizes under stringent conditions with a polynucleotide consisting of a base sequence complementary to the CDS described in SEQ ID NO: 4, 14, or 16 of the Sequence Listing, and encodes<br><br>
V<br>
a protein having peptide-forming enzyme activity to catalyze a reaction that produces a dipeptide from an L-amino acid ester and an L-amino acid; (iviii)     A DNA that encodes protein having the amino acid sequence described in SEQ ID NO: 5, 15, or 17 of the Sequence Listing; and, (viy)      A DNA that encodes a protein having an amino acid sequence that includes substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 5, 15 or 17 of the Sequence Listing, and has peptide-forming enzyme activity to catalyze a reaction that produces a dipeptide from an L-amino acid ester and an L-amino acid. [ll-2]Production of Transformant<br>
Next, production of a transformant that expresses a protein having peptide-forming enzyme activity will be described.    Numerous examples are known for producing enzymes, physiologically active substances and other useful proteins by using a recombinant DNA technology, and the use of recombinant DNA technology allows useful proteins present only in trace amounts in nature to be produced in large volumes.<br>
Preferable examples of transformants that can be used in the method of the present invention include the transformants capable of expressing a protein of (AA), (BB), (CC) below, or the like:<br>
(AA)     a protein having an amino acid sequence described in SEQ ID NO: 5, 15, or 17 of the Sequence Listing;<br>
(BB)     a protein having an amino acid sequence that includes substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 5, 15, or 17 of the<br><br>
Sequence Listing, and has peptide-forming enzyme activity to catalyze a reaction that produces a dipeptide from an L-amino acid ester and an L-amino acid; or,<br>
(CC)     a protein encoded by a DNA that hybridizes under stringent conditions with a polynucleotide consisting of a base sequence complementary to the base sequence of SEQ ID NO: 4, 16, or 18 of the Sequence Listing or a probe prepared based on the base sequence of SEQ ID NO: 4, 16, or 18, and encodes protein having peptide-forming enzyme activity to catalyze a reaction that produces a dipeptide from an L-amino acid ester and an L-amino acid.<br>
To produce transformants that expresses a protein having peptide-producing activity of the aforementioned (AA) to (CC), a DNA of (i), (ii). (iii), QL (iv) ^4v) indicated in the aforementioned section [11-1] may be inserted into host cells.    Namely, a DNA of (i), (ii), (iii), or (iv) 0f-{v} is incorporated into a recombinant DNA, and more specifically an expression vector, capable of being expressed in the host cells, followed by introduction of this expression vector into host cells.<br>
In addition, variants like those indicated in the aforementioned (BB) are obtained by modifying the base sequence so that an amino acid at a specific site of the enzyme gene is substituted, deleted, inserted or added by, for example, a site-directed mutagenesis method.    In addition, the modified DNA mentioned above can also be acquired by a conventional mutagenesis treatment.    Examples of mutagenesis treatment include a method in which a DNA encoding the present enzyme is treated in vitro with hydroxylamine and so forth, and a method in which Escherichia coli<br><br>
possessing a DNA encoding the present enzyme is treated with ultraviolet radiation or N-methyl-N'-nitro-N-nitrosoguanidine (NTG), nitrous acid or other mutagen ordinarily used in artificial mutagenesis.<br>
In the case of mass production of a protein using a recombinant DNA technology, a mode of conjugating the protein in a transformant that produces the protein to form an inclusion body of protein is also a preferable mode for carrying out the present invention.   Advantages of this expression and production method include protection of the target protein from digestion by proteases present in microbial cells, and simple purification of the target protein by crushing the microbial cells, followed by centrifugation. The inclusion bodies of protein obtained in this manner are solubilized with a protein denaturant and then converted to a properly folded, physiologically active protein after going through an activity regeneration procedure consisting primarily of removing the denaturant.   There are numerous examples of this, including regeneration of the activity of human interleukin-2 (Japanese Patent Application Laid-Open Publication No. S61-257931) and so forth.<br>
To obtain active protein from inclusion bodies of protein, a series of<br>
operations including solubilization and activity regeneration are required,<br>
and the procedure is more complex than in the case of producing active<br>
protein directly.    However, in the case of producing in microbial cells a large<br>
amount of protein that has a detrimental effect on microbial growth, that<br>
effect can be suppressed by accumulating the protein in the form of protein<br>
inclusion bodies of protein that are inactive in the microbial cells,<br>
i	Examples of methods for producing a large volume of a target<br><br>
protein in the form of inclusion bodies include a method in which a target protein is expressed alone under the control of a powerful promoter, and a method in which a target protein is expressed in the form of a fused protein with a protein that is known to be expressed in a large volume.<br>
Moreover, it is also effective to arrange the recognition sequence of a restricting protease at a suitable location in order to cleave the target protein after the expression in the form of a fused protein.<br>
In the case of mass production of a protein using a recombinant DNA technology, examples of host cells that can be used include bacterial cells, Actinomyces cells, yeast cells, mold cells, plant cells and animal cells, intestinal bacteria such as Escherichia coli are commonly used, with Escherichia coli being used preferably.   This is because there are numerous findings available regarding techniques for mass production of protein using Escherichia coli.    Hereinafter, one mode of the method for producing a peptide-forming enzyme using transformed Escherichia coli will be described.<br>
Promoters normally used in heterogeneous protein production in Escherichia coli can be used as a promoter for expressing DNA encoding peptide-forming enzyme, examples of which include powerful promoters such as T7 promoter, lac promoter, trp promoter, trc promoter, tac promoter, lambda phage PR promoter, and PL promoter.<br>
In order to produce a peptide-forming enzyme in the form of an inclusion body of fused protein, a gene that encodes another protein, and preferably a hydrophilic peptide, is coupled to the upstream or downstream of the peptide-forming enzyme gene to obtain a fused protein gene.    The<br><br>
gene that encodes another protein in this manner may be any one that increases the amount of fused protein accumulated, and enhances the solubility of the fused protein after the denaturing and regenerating steps. Candidates therefor include, for example, T7 gene 10, p-galactosidase gene, dehydrofolate reductase gene, interferon y gene, interleukin-2 gene and prochymosin gene.<br>
When coupling these genes to a gene encoding a peptide-forming enzyme, the codon reading frames are made consistent with each other. They may be coupled at a suitable restriction enzyme site or a synthetic DNA of a suitable sequence may be used.<br>
In addition, to increase the production amount, it is preferable in some cases to couple a transcription terminating sequence in the form of a terminator downstream of the fused protein gene.   Examples of this terminator include a T7 terminator, an fd phage terminator, a T4 terminator, a tetracycline resistance gene terminator and Escherichia coli trpA gene terminator.<br>
So-called multi-copy vectors are preferable as the vector for introducing into Escherichia coli a gene that encodes a peptide-forming enzyme or a fused protein consisting of a peptide-forming enzyme and another protein, examples of which include plasmids having a replicator derived from ColE1 such as pUC plasmid, pBR322 plasmid or derivatives thereof.   The "derivative" as used herein refers to those plasmids that are subjected to modification of the plasmid by base substitution, deletion, insertion, addition or inversion.    Note that the modification referred to here includes modification resulting from mutagenesis treatment by a mutagen or<br><br>
UV irradiation, or spontaneous mutation.   More specifically, examples of vectors that can be used include pUC19, pUC18, pBR322, pHSG299, pHSG298, pHSG399? pHSG398, RSF1010, pMW119, pMW118, pMW219 and pMW218.    Besides, vectors of phage DNA and transposon DNA can also be used.<br>
In addition, the vector preferably has a marker such as an ampicillin resistance gene in order to screen out the transformant.    Expression vectors having powerful promoters are commercially available for use as such plasmids (such as pUC vector (manufactured by Takara Shuzo), pPROK vector (manufactured by Clontech) and pKK233-2 (manufactured by Clontech) and others).<br>
A recombinant DNA is obtained by coupling a DNA fragment, in which a promoter, gene encoding peptide-forming enzyme or fused protein consisting of peptide-forming enzyme and another protein, and depending on the case, a terminator are coupled in that order, with a vector DNA.<br>
When Escherichia coli is transformed using the recombinant DNA and the resulting Escherichia coli is cultured, a peptide-forming enzyme or a fused protein consisting of the peptide-forming enzyme and another protein is expressed and produced.   Although a strain of the transformed host can be used that is normally used in the expression of a heterogeneous gene, Escherichia coli strain JM109 is preferable.    Methods for carrying out transformation and methods for screening out transformants are described in Molecular Cloning, 2nd Edition, Cold Spring Harbor Press (1989) and other publications.<br>
In the case of expressing the protein in the form of a fused protein,<br><br>
the fused protein may be prepared such that a peptide-forming enzyme can be cleaved out using a restriction protease that uses a sequence not present in the peptide-forming enzyme, such as blood coagulation factor Xa or kallikrein, as the recognition sequence.<br>
A medium normally used for culturing Escherichia coli, such as an M9-casamino acid medium or an LB medium, may be used as the production medium.    In addition, culturing conditions and production induction conditions are suitably selected according to the marker of the vector used, promoter, type of host microbe and so forth.<br>
The following method can be used to recover a peptide-forming enzyme or a fused protein consisting of the peptide-forming enzyme and another protein.    If the peptide-forming enzyme or its fused protein has been solubilized in the microbial cells, after recovering the microbial cells, the microbial cells are crushed or lysed so that they can be used as a crude enzyme liquid.    Moreover, the peptide-forming enzyme or its fused protein can be purified prior to use by ordinary techniques such as precipitation, filtration or column chromatography as necessary.    In this case, a purification method can also be used that uses antibody to the peptide-forming enzyme or its fused protein.<br>
In the case where inclusion bodies of protein are formed, the inclusion bodies are solubilized with a denaturant.   Although they may be solubilized with the microbial cell protein, in consideration of the following purification procedure, the inclusion bodies are preferably taken out and then solubilized.    Conventionally known methods may be used to recover the inclusion bodies from the microbial cells.    For example, inclusion<br><br>
bodies can be recovered by crushing the microbial cells, followed by centrifugation.    Examples of denaturants capable of solubilizing inclusion bodies include guanidine hydrochloride (for example, 6 M, pH 5 to 8) and urea (for example, 8 M).<br>
Protein having activity is regenerated by removing these denaturants by dialysis.   Tris-HCI buffer solution or phosphate buffer solution and so forth should be used as the dialysis solution used in dialysis, and the concentration may be, for example, 20 mM to 0.5 M, while the pH may be, for example, 5 to 8.<br>
The protein concentration during the regeneration step is preferably held to about 500 jig/ml or less.   The dialysis temperature is preferably 5°C or lower to inhibit the occurrence of self-crosslinking by the regenerated peptide-forming enzyme.    In addition, dilution or ultrafiltration may be used in addition to dialysis to remove the denaturants, and the enzyme activity can be expected to be regenerated no matter which a denaturant is used.<br>
In the case of using the DNA indicated in SEQ ID NO: 4 of the Sequence Listing for the DNA that encodes a peptide-forming enzyme, the peptide-forming enzyme is produced that has the amino acid sequence described in SEQ ID NO: 5.    In addition, in the case of using a DNA indicated in SEQ ID NO: 14 of the Sequence Listing as the DNA that encodes a peptide-forming enzyme, peptide-forming enzyme is produced that has the amino acid sequence described in SEQ ID NO: 15.    In addition, in the case of using a DNA indicated in SEQ ID NO: 16 of the Sequence Listing as the DNA that encodes a peptide-forming enzyme, peptide-forming enzyme is produced that has the amino acid sequence described in SEQ ID<br><br>
NO: 17.<br>
Note that genetic engineering techniques can be carried out in compliance with the techniques described in the literature such as Molecular Cloning, 2nd edition, Cold Spring Harbor Press (1989). [Ill]    Properties of Peptide-forming enzyme<br>
Next, the properties of a peptide-forming enzyme purified from Corynebacterium glutamicum strain ATCC 13286 as an example of the aforementioned microbes.<br>
Taking an example of using L-alanine ester and L-glutamine as raw materials (substrates), this peptide-forming enzyme has activity to produce L-alanyl-L-glutamine using an L-alanine ester and L-glutamine as substrates. In addition, taking an example of using an L-alanine amid-eester and L-qkrtamineasparagine as raw materials, this peptide-forming enzyme has activity to produce L-alanyl-L-asparagine using an L-alanine ester and L-asparagine as substrates.<br>
With respect to the enzyme action, taking an example of using an L-alanine ester and L-asparagine or L-glutamine as raw materials, this peptide-forming enzyme produces one molecule of L-alanyl-L-glutamine and one molecule of alcohol from one molecule of L-alanine ester and one molecule of L-glutamine, and produces one molecule of L-alanyl-L-asparagine and one molecule of alcohol from one molecule of L-alanine ester and one molecule of asparagine.<br>
The optimum pH is in the vicinity of 6.0 to 10.0, and the optimum temperature is in the vicinity of 30°C to 50°C.    The molecular weight of the subunit is calculated to be 42,000 to 46,000 as determined by<br><br>
SDS-polyacrylamide gel electrophoresis. [IV]       Dipeptide Production Method<br>
The dipeptide production method of the present invention produces a dipeptide from an L-amino acid ester and an L-amino acid using an enzyme or enzyme-containing substance having the ability to synthesize a dipeptide, and more specifically, produces a dipeptide from an L-amino acid ester and an L-amino acid using a peptide-forming enzyme derived from a culture of microbes, microbial cells isolated from the culture or treated microbial cell product of the microbe having the ability to produce a dipeptide from an L-amino acid ester and an L-amino acid.    Note that a protein having proline iminopeptidase activity derived from the aforementioned microbes or those listed in the following Table 1 can also be used so far as they have activity to produce a dipeptide from an L-amino acid ester and an L-amino acid.<br>
The peptide-forming enzyme produced by the aforementioned microbes has activity to produce a dipeptide by using an L-amino acid ester and an L-amino acid as substrates.<br>
As the method by which the peptide-forming enzyme produced by the aforementioned microbes is allowed to act on the L-amino acid ester and L-amino acid, the substrates may be added directly to the culture liquid while culturing the aforementioned microbes, or microbial cells may be separated from the microbial culture by centrifugation and so forth, followed .by either re-suspending in buffer either directly or after washing, and then adding an L-amino acid ester and an L-amino acid and allowing them to react.   Alternatively, microbial cells can be used that have been<br><br>
immobilized by a known method using polyacrylamide gel, carrageenan or alginic acid gel.<br>
In addition, crushed microbial cells, acetone-treated microbial cells or freeze-dried microbial cells may be used as the treated microbial cell product.    Methods such as ultrasonic crushing, French press crushing or glass bead crushing can be used for crushing microbial cells, while methods using egg white lysozyme, peptidase treatment or a suitable combination thereof are used in the case of lysing microbial cells.<br>
Moreover, a peptide-forming enzyme may be recovered from the treated microbial cell product and used as a crude enzyme liquid, or the enzyme may be purified before use as necessary.   Ordinary enzyme purification methods can be used for purifying the enzyme obtained from a culture.    More specifically, microbial cells are collected by centrifugation and so forth, the cells are then crushed by mechanical methods such as ultrasonic treatment, glass beads or a dynomili, and solid materials such as cell fragments are removed by centrifugation to obtain crude enzyme followed by purification of the aforementioned peptide-forming enzyme by performing ultracentrifugation fractionation, salting out, organic solvent precipitation, ion exchange chromatography, adsorption chromatography, gel filtration chromatography, hydrophobic chromatography and so forth.<br>
Note that the "peptide-forming enzyme derived from a microbe" includes not only an enzyme obtained from the treated microbial cell product by going through the aforementioned purification step, but also enzyme produced by so-called genetic engineering techniques in which the gene of the enzyme is expressed in a heterogeneous or homogeneous host.<br><br>
Namely, in the case of a fraction having activity to produce a dipeptide from an L-amino acid ester and an L-amino acid, the whole enzyme and enzyme-containing substance can be used.    Here, an "enzyme-containing substance" refers to that which contains the enzyme, and includes specific forms such as a culture of microbes that produce the enzyme, microbial cells separated from the culture and treated microbial cell products of the microbes.   A culture of microbes refers to that which is obtained by culturing microbes, and more specifically, refers to a mixture of microbial cells, the medium used to culture the microbes and substances produced by the cultured microbes.    In addition, the microbial cells may be washed and used as washed microbial cells.   The treated microbial cell product includes cells that have been crushed, lysed orfreeze-dried, and further include a crude enzyme recovered by treating the cells and so forth as well as purified enzyme resulting from its purification.   Partially purified enzymes and so forth obtained by various purification methods may also be used as purified enzymes, or immobilized enzyme may be used that have been immobilized by covalent bonding, adsorption or entrapment methods. In addition, since some microbes are partially lysed during culturing depending on the microbes used, the culture supernatant may also be used as the enzyme-containing substance in such cases.<br>
Note that in the case of using a culture, cultured microbial cells, washed microbial cells or processed microbial cells in which the cells have been crushed or lysed, there are many cases in which an enzyme is present that decomposes the peptide produced without being involved in peptide production, and in such cases, there are cases in which it is preferable to<br><br>
add a metallic enzyme inhibitor such as a metalloprotease inhibitor like, for example, ethylenediaminetetraacetic acid (EDTA).   The amount added is in the range of 0.1 mM to 100 mM, and preferably 1 mMto 50 mM.<br>
The amount of enzyme or enzyme-containing substance used may be enough if it is an amount in which the target effect is demonstrated (effective amount).   This effective amount can be easily determined through simple, preliminary experimentation by a person with ordinary skill in the art; for example, in the case of using washed cells, the amount used is 1 to 500 g/l of reaction liquid.<br>
Any L-amino acid ester can be used as the L-amino acid ester so far as it is an L-amino acid ester capable of producing dipeptide with L-amino acid at the substrate specificity of the peptide-forming enzyme, and examples of such include methyl esters, ethyl esters, n-propyl esters, iso-propyl esters, n-butyl esters, iso-butyl esters and tert-butyl esters of L-amino acids.    In addition, not only L-amino acid esters corresponding to naturally-occurring amino acids, but also L-amino acid esters corresponding to non-naturally-occurring amino acids or their derivatives can also be used. Examples of L-amino acid esters that can be used preferably in the present invention include L-alanine ester, glycine ester, L-valine ester, L-isoleucine i     ester, L-methionine ester, L-phenylalanine ester, L-serine ester, L-threonine ester, L-glutamine ester, L-tyrosine ester, L-arginine ester, L-aspartic acid-a-ester, L-aspartic acid-(3-ester, L-leucine ester, L-asparagine ester, L-lysine ester, L-aspartic acid-a,p-dimethyl ester and L-glutamine-y-ester.<br>
There is no particular restriction on the L-amino acid and any L-amino 5     acid can be used so far as it produces a dipeptide with an L-amino acid<br><br>
ester in the substrate specificity of the peptide-forming enzyme.   Examples of L-amino acids that can be used preferably in the present invention include L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine, and L-glutamate, with L-glutamine and L-asparagine being particularly preferable.<br>
Each concentration of the L-amino acid ester and L-amino acid used as starting materials is 1 mM to 10 M, and preferably 0.05 M to 2 M. However, there are cases in which it is preferable to add an equimolar amount or more of L-amino acid with respect to the amount of L-amino acid ester.    In addition, in the case where a high concentration of substrate inhibits the reaction, it can be adjusted to a concentration that does not cause inhibition and then successively added during the reaction.<br>
The reaction temperature is 3 to 70°C, and preferably 5 to 50°C, while the reaction pH is 2 to 12, and preferably 3 to 11.    By carrying out the reaction in this manner for about 2 to 48 hours, a dipeptide is produced and accumulates in the reaction mixture.   The resulting dipeptide can then be recovered by established methods and purified as necessary.<br>
Examples<br>
Hereinafter, the present invention will be described in detail by referring to the examples described below.    However, the present invention is not limited to these examples.    Note that in the examples, quantitative determination of L-alanine, L-alanyl-L-glutamine or L-alanyl-L-asparagine was carried out by a method using high performance liquid chromatography<br><br>
(column: Inertsil ODS-2 (GL Science), eluate: aqueous phosphate solution<br>
(pH 2.2, 5.0 mM sodium 1-octane sulfonate/methanol = 100/15), flow rate:<br>
1.0 mL/min, detection: 210 nm).<br>
(Example 1)      Effect of Addition of EDTA on Production of<br>
L-Alanyl-L-Glutamine<br>
50 mL of a medium (pH 7.0) containing 5 g of glucose, 5 g of ammonium sulfate, 1 g of monopotassium phosphate, 3 g of dipotassium phosphate, 0.5 g of magnesium sulfate, 10 g of yeast extract and 10 g of peptone in 1 L was transferred to a 500 mL Sakaguchi flask and sterilized for 15 minutes at 115°C.    One loopful of of Pseudomonas putida strain FERM BP-8101, which had been cultured for 24 hours at 30°C on an slant agar medium containing the same composition (agar: 20 g/L, pH 7.0), was inoculated into the aforementioned medium and cultured by shake culturing for 17 hours at 30°C and 120 strokes/minute.    Following culturing, the microbial cells were separated by centrifugation and suspended with 100 mM borate buffer (pH 9.0) to a wet cell density of 100 g/L.    1 mL each of the cell suspension was respectively added to 1 mL of 100 mM borate buffer (pH 9.0) containing 200 mM L-alanine ethyl ester hydrochloride and 400 mM L-glutamine either in the absence of EDTA or additionally containing 20 mM EDTA (substrate solution) to bring to a final volume of 2 mL followed by allowing to react for 1 hour at 30°C.   As a result, L-alanyl-L-glutamine was produced at 4.9 mM in the section added with no EDTA and at 10.1 mM in the section added with EDTA.<br>
Note that in this reaction system, under conditions in which 1 mL of 100 mM borate buffer (pH 9.0) instead of cell suspension was added to 1<br><br>
mL or substrate solution (cell-free lot) and under conditions in which 1 mL of 100 mM of borate buffer either free of EDTA or containing 20 mM EDTA but not containing L-alanine ethyl ester hydrochloride and glutamine was added to the cell suspension (substrate-free lot), production of L-alanyl-L-glutamine was not observed in either case. (Example 2)      Use of Amino Acid Ester as Substrate<br>
1 mL of cell suspension wet cells (100 g/L) of Pseudomonas putida strain FERM BP-8101 prepared in the same manner as Example 1 was respectively added to 1 mL of 100 mM borate buffer (pH 9.0) containing 20 mM EDTA and the following L-alanine ester hydrochlorides at 200 mM and L-glutamine at 400 mM to bring to a final volume of 2 mL, followed by allowing to react for 1 hour at 30°C.   As a result, 14.9 mM L-alanyl-L-glutamine was produced in the case of using L-alanine methyl ester hydrochloride and L-glutamine as substrates, 11.4 mM L-alanyl-L-glutamine was produced in the case of using L-alanine ethyl ester hydrochloride and L-glutamine as substrates, and 0.5 mM L-alanyl-L-glutamine was produced in the case of using L-alanine-tert-butyl ester hydrochloride and L-glutamine as substrates. (Example 3)      Use of L-Amino Acid as Substrate<br>
1 mL of a cell suspension wet cells (100 g/L) of Pseudomonas putida strain FERM BP-8101 prepared in the same manner as Example 1 was respectively added to 1 mL of 100 mM borate buffer (pH 9.0) containing 20 mM EDTA and the following L-alanine ester hydrochlorides at 200 mM and L-glutamine or L-asparagine at 400 mM to bring to a final volume of 2 mL, followed by allowing to react for 1 hour at 30°C.    As a result, 12.7 mM<br><br>
L-alanyl-L-glutamine was produced in the case of using L-alanine methyl ester hydrochloride and L-glutamine as substrates, and 4.8 mM L-alanyl-L-asparagine was produced in the case of using L-alanine methyl ester hydrochloride and L-asparagine as substrates. (Example 4)      Microbes Producing L-Alanyl-L-Glutamine<br>
50 mL of a medium (pH 7.0) containing 5 g of glucose, 5 g of ammonium sulfate, 1 g of monopotassium phosphate, 3 g of dipotassium phosphate, 0.5 g of magnesium sulfate, 10 g of yeast extract and 10 of peptone in 1 L was transferred to a 500 mL Sakaguchi flask and sterilized for 15 minutes at 115°C.   One loopful of of each of the bacteria shown in Table 1, which had been cultured for 24 hours at 30°C on an slant agar medium (agar: 2 g/L, pH 7.0) containing 5 g of glucose, 10 g of yeast extract, 10 g of peptone and 5 g of NaCI, was inoculated into the aforementioned medium and cultured by shake culturing for 17 hours at 30°C and 120 strokes/minute.   After the culturing, the microbial cells were separated by centrifugation and suspended with 0.1 M borate buffer (pH 9.0) containing 10 mM EDTA to 100 g/L as wet microbial cells.    0.1 mL of 100 mM borate buffer (pH 9.0) containing 10 mM EDTA, 200 mM L-alanine methyl ester hydrochloride and 400 mM L-glutamine was respectively added to 0.1 mL of these microbial cell suspensions to bring to a final volume of 0.2 mL followed by allowing to react for 2 hours at 25°C.   The amounts (mM) of L-alanyl-L- glutamine (Ala-Gin) produced at this time are shown in Table 1.<br><br><br><br>
(Example 5)      Effect of Temperature on Production of L-AIanyl-L-Glutamine<br>
1 mL of the suspension of Pseudomonas putida strain FERM BP-8101 cells (100 g/L) prepared in accordance with the microbe culturing method of Example 4 was respectively added to 1 mLof 100 mM borate buffer (pH 9.0) containing 10 mM EDTA, 200 mM L-alanine methyl ester hydrochloride and 400 mM L-glutamine to bring to a final volume of 2 mL, followed by allowing to react for 1 hour at temperatures of 20°C, 30°C and 40°C, respectively.   Those results are shown in Table 2.    Production of L-alanyl-L-glutamine (Ala-Gin) demonstrated the highest value at a temperature of 40°C in the case of Pseudomonas putida strain FERM BP-8101.<br><br>
(Example 6)      Purification of Peptide-forming enzyme from Corynebacterium glutamicum strain ATCC 13286 and Production of L-AIanyl-L-Glutamine by the Purified Enzyme<br><br>
500 mL of a medium containing 5 g of glycerol, 5 g of yeast extract, 5 g of peptone, 5 g of sodium chloride and 5 g of L-alanine amide hydrochloride in 1 L was transferred to a 5 L Sakaguchi flask and sterilized for 20 minutes at 120°C.    Culture liquid of Corynebacterium glutamicum strain ATCC 13286 cultured for 20 hours in medium of the same composition as above was inoculated into the medium to be 5% (v/v), and cultured for 20 hours at 30°C and 120 strokes/minute.    Microbial cells were collected from 8 L of this culture liquid by centrifugation.   The subsequent procedure was carried out either on ice or at 4°C.   After washing the microbial cells with 50 mM potassium phosphate buffer (pH 7.0), the cells were subjected to crushing treatment for about 10 minutes using glass beads having a diameter of 0.1 millimeter.   The glass beads and crushed cell liquid were then separated, and the crushed cell fragments were removed by centrifugation for 30 minutes at 20,000xgravity (g) to obtain a cell-free extract.    Moreover, the insoluble fraction was removed by ultracentrifugation for 60 minutes at 200,000xg to obtain a soluble fraction in the form of the supernatant.   Ammonium sulfate was then added to the resulting soluble fraction to 60% saturation followed by recovery of the precipitate by centrifuging for 30 minutes at 20,000xg.   The resulting precipitate was dissolved in a small amount of 50 mM potassium phosphate buffer (pH 7.0) and then dialyzed against 50 mM potassium phosphate buffer (pH 7.0).   This enzyme liquid was then applied to a Q-Sepharose HP column pre-equilibrated with 50 mM potassium phosphate buffer (pH 7.0), and the enzyme was eluted over a linear concentration gradient of 50 mM •     potassium phosphate buffer (pH 7.0) containing 0 to 1.0 M sodium chloride.<br><br>
The active fraction was collected and applied to a Superdex 200 pg column pre-equilibrated with 50 mM potassium phosphate buffer (pH 7.0), and the enzyme was then eluted with the same buffer.   The active fraction was collected and dialyzed against 20 mM potassium phosphate buffer (pH 7.0) containing 0.5 M ammonium sulfate, and then applied to a Phenyl-Sepharose HP column pre-equilibrated with 20 mM potassium phosphate buffer (pH 7.0) containing 0.5 M ammonium sulfate.   The enzyme was then eluted over a linear concentration gradient of 20 mM potassium phosphate buffer (pH 7,0) containing 0.5 to 0 M ammonium sulfate.   The active fraction was collected and dialyzed against 50 mM potassium phosphate buffer (pH 7.0), and this was then applied to a MonoQ column pre-equilibrated with 50 mM potassium phosphate buffer (pH 7.0), after which enzyme was eluted over a linear concentration gradient of 50 mM potassium phosphate buffer (pH 7.0) containing 0 to 1.0 M sodium chloride.   The purified peptide-forming enzyme was uniformly purified on the basis of electrophoresis in this manner.<br>
The specific activity of the purified enzyme was 9.841 U/mg, and the specific activity of the purified peptide-forming enzyme was increased about 246-fold as a result of going through these purification steps.    In addition, as a result of applying the molecular weight of the purified enzyme standard to SDS polyacrylamide electrophoresis, a uniform band was detected at the position calculated to represent a molecular weight of 42,000 to 46,000. Measurement of enzyme titer was carried out as described below.    200 |aM of Tris-HCI buffer (pH 9.0), 50 \.M of L-alanine amide and a suitable amount »     of enzyme liquid were added and mixed to a final volume 1 mL, and after<br><br>
allowing to react for 60 minutes at 30°C, 4 mL of aqueous phosphoric acid (pH 2.1) was added to stop the reaction.   The alanine produced was quantified by high-performance liquid chromatography, and the amount of enzyme that produces 1 ^M of L-alanine in 1 minute was defined as 1 unit.<br>
This purified enzyme was then added to borate buffer (pH 9.0) containing EDTA, L-alanine methyl ester hydrochloride and L-glutamine (or L-asparagine), mixed to the total volume of 1 mL (for the final concentrations, the amount of enzyme added was 2 units as alanine amide decomposition activity, EDTA was at 10 mM, L-alanine methyl ester hydrochloride was at 100 mM and L-glutamine (or L-asparagine) was at 200 mM borate buffer at 100 mM), and allowed to react for 4 hours at 30°C. (Note that the number of units of enzyme does not indicate the production activity with respect to producing L-alanyl-L-glutamine from L-alanine methyl ester and L-glutamine, but rather simply indicates L-alanine amide decomposition activity.)   The amount of L-alanyl-L-glutamine produced at this time was 50.2 mM, while the amount of L-alanyl-L-asparagine produced was 49.8 mM.<br>
(Example 7)      Isolation of Peptide-forming enzyme Gene from Corynebacteriurn glutamicum strain ATCC 13286 and Expression in Escherichia coli<br>
Hereinafter, the isolation of a peptide-forming enzyme gene from Corynebacteriurn glutamicum strain ATCC 13286 and its expression in Escherichia coli {Escherichia coli) will be described.    Escherichia coli JM109 was used as the host and pUC18 was used as the vector for both the isolation and expression of the gene.<br><br>
1.	Production of PCR Primer Based on Determined Amino Acid<br>
Sequence<br>
Mixed primers indicated in SEQ ID NO: 2 and SEQ ID NO: 3, respectively, were produced based on the N-terminal amino acid sequence of peptide-forming enzyme derived from the Corynebacterium glutamicum strain ATCC 13286 (SEQ ID NO:1) obtained in Example 1.<br>
2.	Microbe Acquisition<br>
The microbes were refreshed by culturing Corynebacterium glutamicum strain ATCC 13286 for 24 hours at 30°C on a CM2Gly agar medium (0.5 g/dL glycerol, 1.0 g/dL yeast extract, 1.0 g/dL peptone, 0.5 g/dL NaCI, 2 g/dL agar, pH 7.0).    One loopful of cells was then inoculated into a 500 mL Sakaguchi flask containing 50 mL of CM2Gly liquid medium, followed by shake culturing for 16 hours at 30°C under aerobic conditions.<br>
3.	Acquisition of Chromosomal DNA from Microbial Cells<br>
50 mL of culture liquid was centrifuged (12,000 revolutions per minute, 4°C, 15 minutes) to collect the microbial cells.    These microbial cells were then suspended in 10 mL of 50 mM Tris-HCI buffer (pH 8.0) containing 20 mM EDTA followed by recovery of the microbial cells by centrifugation. The microbial cells were again suspended in 10 mL of 50 mM Tris-HCI buffer (pH 8.0) containing 20 mM EDTA.    Moreover, after adding 0.5 mL of 20 mg/ml lysozyme solution and 1 mL of 10% SDS (sodium dodecyl sulfate) solution to this suspension, the solution was incubated for 20 minutes at 55°C.   This incubated solution was then deproteinized by the addition of an equimolar amount of phenol saturated with 10 mM Tris-HCI buffer (pH 8.0) containing 1 mM EDTA.   An equimolar amount of 2-propanol was added to<br><br>
the separated aqueous layer to precipitate DNA, followed by recovery of that precipitated DNA.   After dissolving the precipitated in DNA in 0.5 mL of 50 mM Tris-HCI buffer (pH 8.0) containing 20 mM EDTA, 5 jaL of 10 mg/ml RNase and 5 jaL of 10 mg/ml Proteinase K was added and allowed to react for 2 hours at 55°C.   After the reaction, this solution was deproteinized by the addition of an equal volume of phenol saturated with 10 mM Tris-HCI buffer (pH 8.0) containing 1 mM EDTA.    Moreover, an equal volume of 24:1 chloroform/isoamyl alcohol was added to the separated aqueous layer followed by stirring and recovery of the aqueous layer.   After performing this procedure two times, 3 M sodium acetate solution (pH 5.2) was added to the resulting aqueous layer to bring to a final concentration of 0.4 M followed by the addition of 2 volumes of ethanol.   The DNA that formed as a precipitate was recovered, and after washing with 70% ethanol, was dried and dissolved in 1 mL of 10 mM Tris-HCI buffer (pH 8.0) containing 1 mM EDTA.<br>
4.      Acquisition of DNA Fragment Containing Partial Peptide-forming enzyme Gene by Cassette PCR<br>
The TaKaRa LA PCR in vitro Cloning Kit (manufactured by Takara Shuzo) was used for isolation and amplification of DNA molecules containing gene (aah) encoding peptide-forming enzyme using the cassette PCR method.    Unless indicated otherwise, the experiment was carried out based on the method described in the manual.    In the cassette PCR method, in case of using Primer 1 (1 st PCR, SEQ ID NO: 2) and Primer 2 (2nd PCR, SEQ ID NO: 3) as primers, a roughly 0.5 kilobase (kb) band (Fragment 1) was amplified with the Eco Rl cassette.   As a result of<br><br>
determining the base sequence of this fragment, Fragment 1 was confirmed<br>
to be a portion of aah.<br>
5.      Cloning of Peptide-forming enzyme Gene from Gene Library<br>
Next, in order to acquire the whole length of aah, Southern hybridization was first carried out using Fragment 1 as a probe.<br>
The DNA fragment that serves as the probe was prepared to about 50 ng/|_il and the probe was labeled by incubating 16 |.iL of this DNA solution for 24 hours at 37°C in accordance with the protocol using DIG High Prime (Boehringer Mannheim).<br>
1 \xg of chromosomal DNA was completely digested by use of combinations of various restriction enzymes, and then electoph'oresed with 0.8% agarose gel.   Then, this was blotted onto positively charged Nylon membranes (Boehringer Mannheim, Nylon membranes positively charged). Southern hybridization was then carried out in accordance with the following established method.    Hybridization was carried out using DIG Easy Hyb (Boehringer Mannheim), and after hybridization for 30 minutes at 50°C, the probe was added, following by hybridization for 18 hours at 50°C. Detection was carried out using the DIG Nucleotide Detection Kit (Boehringer Mannheim).<br>
As a result, a band was detected at about the 7 kb position in the BglU cleaved product.   This 7 kb region fragment was collected and coupled to pUC18 to produce a library (120 strains) with Escherichia coli JM109. Colony hybridization was then carried out in accordance with the following established method.   The colonies were transferred to Nylon membranes for colony and plaque hybridization (Boehringer Mannheim) followed by<br><br>
alkaline denaturation, neutralization and immobilization treatment. Hybridization was carried out using DIG Easy Hyb.   The filter was immersed in buffer and pre-hybridized for 30 minutes at 42°C. Subsequently, the aforementioned labeled probe was added followed by hybridization for 18 hours at 42°C.   After washing with SSC buffer, one positive clone was selected using the DIG Nucleotide Detection Kit.<br>
6.	Base Sequence of Peptide-forming enzyme Gene Derived from<br>
Corynebacterium glutamicum strain ATCC 13286<br>
Plasmids retained by the selected transformant were prepared in accordance with the method described in Molecular Cloning, 2nd edition, Cold Spring Harbor Press (1989), and the base sequence in the vicinity that hybridized with the probe was determined.   An open reading frame (ORF) was present that encoded a protein containing 30 residues of the N-terminal amino acid sequence of a peptide-forming enzyme, and vyas confirmed to be gene aah that encodes the peptide-forming enzyme.   The base sequence of the full-length peptide-forming enzyme gene is shown in SEQ ID NO: 4 of the Sequence Listing.   The resulting ORF exhibited a base sequence homology of 57.6% with known proline iminopeptidase derived from Propionibacterium species bacteria.    Note that the numerical value of homology is the value obtained by Genetyx (hereinafter the same applies to this Example).<br>
7.	Expression of Peptide-forming enzyme Gene in Escherichia coli<br>
Plasmid pUCAAH coupled to aah downstream of the lac promoter of<br>
pUC18 was constructed in order to express aah in Escherichia coli. Fragments amplified by PCR using the chromosomal DNA of<br><br>
Corynebacterium glutamicum strain ATCC 13286 as template and the oligonucleotides shown in Table 3 as primers were treated with Sac\ and Sma\} and after ligating with the Sac\ and Smal cleaved products of pl)C18, were used to transform Escherichia coli JM109.    Strains having the target plasmid were selected from ampicillin-resistant strains, and the constructed expression plasmid was designated as pUCAAH.<br><br>
The transformant expressing peptide-forming enzyme in Escherichia coli having pUCAAH was seed-€a4twedpre-cultured for 16 hours at 37°C in an LB medium containing 0.1 mg/ml ampicillin.    1 mL of this pre-culture broth was see^edinoculated into a 500 mL Sakaguchi flask containing 50 mL of LB medium followed by final culturing at 37°C.   After 2 hours from the start of the culturing, isopropyl-1-thio-p-D-galactopyranoside (IPTG) was added to a final concentration of 1 mM followed by additionally culturing for 3 hours.<br>
After completion of the culturing, the microbes were collected and washed, suspended in 10 mL of 20 mM phosphate buffer (pH 8.0), and then subjected to ultrasonic crushing for 30 minutes at 180 W.   The solution was collected and centrifuged for 10 minutes at 12,000gx10 minutes, and the resulting supernatant was used as a cell-free extract.<br><br>
(Example 8)      Measurement of Activity of Peptide-forming enzyme<br>
1.      Activity of Enzyme Derived from Corynebacterium glutamicum strain<br>
ATCC 13286<br>
After completion of the culturing in the manner described above, a cell-free extract was prepared and the activity of peptide-forming enzyme was measured using this as the enzyme source.    Measurement of peptide-forming enzyme activity was carried out by incubating a reaction liquid containing 100 mM L-alanine methyl ester hydrochloride, 150 mM L-glutamine, 100 mM borate buffer (pH 9.0), 10 mM EDTA, and the enzyme solution for 60 minutes at 30°C, followed by stopping the reaction by adding 4 fold volume of phosphoric acid (pH 1.5).   The amount of L-alanyl-L-glutamine was determined by HPLC.    For the unit of enzyme activity, enzyme activity to produce 1 INITIO I of L-alanyl-L-glutamine in 1 minute under these conditions was defined as 1 unit (U).<br>
Conditions of HPLC used for analysis are as follows: Column: Inertsil ODS-2<br>
Mobile  phase:   (aqueous  phosphoric  acid  solution  (pH  2.1)),  2.5  mM sodium-1-octanesulfonate/methanol = 10/1 Column temperature: 40°C Flow rate: 1.0 ml/minute Detection: UV210 nanometers<br>
As a result, 0.54 U/mg of activity that synthesized L-alanyl-L-glutamine from L-alanine methyl ester hydrochloride was detected in the case of introducing pUC18 AAH, thereby confirming that the cloned aah gene was expressed in Escherichia coii.    Note that no activity was detected in the<br><br>
case of inserting pUC18 only as a control.<br>
2.      Expression of His-Tag Peptide-forming enzyme Gene in Escherichia<br>
coli<br>
Plasmid pQEAAH that expresses a peptide-forming enzyme as an His-Tag protein downstream of the lac promoter of pUC18 was constructed to express aah in Escherichia coli.    Fragments amplified by PCR using the chromosomal DNA of Corynebacterium glutamicum strain ATCC 13286 as template and the oligonucleotides shown in Table 4 as primers were treated with Sac\ and Smal, and after ligating with the Sacl and Smal cleaved products of pQE-30 (Qiagen), were used to transform Escherichia coli JM109.    Strains having the target plasmid were selected from ampicillin-resistant strains, and the constructed expression plasmid was designated as pQEAAH.<br><br>
When the activity of the transformant expressing peptide-forming enzyme in Escherichia coli having pQEAAH was measured using the same method as in Example 8, it was found to exhibit peptide-forming enzyme activity of 5.28 U/mg. i     3.      Preparation of His-Tag Purified Enzyme<br><br>
Microbial cells from 150 mL of culture broth of Escherichia coli JM109 possessing pQEAAH were crushed according to the aforementioned method, and His-Tag L-alanine amide hydrolase was purified using the His Trap Kit (manufactured by Amersham Pharmacia Biotech) according to the protocol provided with the kit.    24 mg of protein was acquired that exhibited a single band on SDS-PAGE, and the specific activity of synthesis of L-alanyl-L-glutamine from L-alanine methyl ester hydrochloride was 148.3 U/mg and 50.7% with respect to L-alanine methyl ester hydrochloride. 4.      Study of Substrate Specificity Using His-Tag Purified Enzyme<br>
The synthesis of peptides other than the L-alanyl-L-glutamine by the acquired peptide-forming enzyme was studied using His-Tag purified enzyme.<br>
(1)    Peptide Synthesis from L-alanine Methyl Ester and Other L-amino Acids<br>
The synthesis reaction was carried out by incubating a reaction liquid containing 100 mM L-alanine methyl ester hydrochloride, 150 mM test amino acid, 100 mM borate buffer (pH 9.0), 10 mM EDTA and enzyme solution (0.05 U/ml) for 3 hours at 25°C, followed by determination of the quantity of the peptides produced by HPLC.   As a result, 22.34 mM L-alanyl-L-asparagine was produced in the case of using L-asparagine as the other amino acid, 5.66 mM L-alanyl-glycine was produced in the case of using glycine, 10.63 mM L-alanyl-L-alanine was produced in the case of using L-alanine, 13.73 mM L-alanyl-L-leucine was produced in the case of using L-leucine, 48.80 mM L-alanyl-L-methionine was produced in the case of using L-methionine, 1.02 mM L-alanyl-L-proline was produced in the case<br><br>
of using L-proline, 16.13 mM L-alanyl-L-phenylalanine was produced in the case of using L-phenylalanine, 15.31 mM L-alanyi-L-tryptophan was produced in the case of using L-tryptophan, 26.14 mM L-alanyi-L-serine was produced in the case of using L-serine, 24.23 mM L-alanyl-L-threonine was produced in the case of using L-threonine, 0.96 mM L-alanyl-L-tyrosine was produced in the case of using L-tyrosine, 7.91 mM L-alanyl-L-lysine was produced in the case of using L-lysine, 24.87 mM L-alanyl-L-arginine was produced in the case of using L-arginine, 23.16 mM L-alanyl-L-histidine was produced in the case of using L-histidine, 1.11 mM L-alanyl-L-glutamate was produced in the case of using L-glutamate.<br>
(2)     Peptide Synthesis from Other L-Amino Acid Methyl Esters and L-Glutamine<br>
The reactions were carried out using amino acid methyl esters other than L-alanine methyl ester.<br>
The synthesis reaction was carried out by incubating a reaction liquid containing 100 mM test amino acid methyl ester, 150 mM L-glutamine, 100 mM borate buffer (pH 9.0), 10 mM EDTA and enzyme (0.05 U/ml) for 3 hours at25°C, followed by determination of the quantity of the produced peptides by HPLC.   As a result, 52.19 mM glycyl-L-glutamine was produced in the case of using glycine methyl ester as the L-amino acid methyl ester, 5.94 mM L-valyi-L-glutamine was produced in the case of using L-valine methyl ester, 0.59 mM L-isoleucyl-L-glutamine was produced in the case of using L-isoleucyl-L-glutamine, 4.31 mM L-methionyl-L-glutamine was produced in the case of using L-methionine methyl ester, 3.67 mM L-phenylalanyl-L-glutamine was produced in the case<br><br>
of using L-phenylalanine methyl ester, 40.44 mM L-seryl-L-glutamine was produced in the case of using L-serine methyl ester, 3.85 mM L-threonyl-L-glutamine was produced in the case of using L-threonine methyl ester, 0.23 mM L-glutaminyl-L-glutamine was produced in the case of using L-glutamine methyl ester, 1.24 mM L-tyrosyl-L-glutamine was produced in the case of using L-tyrosine methyl ester, 6.52 mM L-arginyl-L-glutarnine was produced in the case of using L-arginine methyl ester, and 8.22 mM L-aspartyl-oc-L-glutamine was produced in the case of using L-aspartic acid-a-methyl ester.    In addition, peptides composed of the corresponding amino acids and L-glutamines were also confirmed to be produced in the case of using L-leucine methyl ester, L-asparagine methyl ester, L-lysine methyl ester, L-aspartic acid-p-methyl ester, L-aspartic acid-a,(3-dimethyl ester or L-glutamic acid-Y-methyl ester as the amino acid methyl ester (determination of quantity of these was not performed since no standard preparation was available). (3)     Measurement of Proline Iminopeptidase (pepl) Activity<br>
The reaction was carried out at 30°C using a reaction liquid (composition; 50 mM borate buffer (pH 9.0), 5 mM EDTA, 1 mM proline 2-naphthyl amide (proline-pNA).   The liberation rate of naphthyl amide was measured as an increase in optical absorbance at 405 nanometers (nm) (e = 9.83).   The activity resulting in release of 1 |_LM of naphthyl amide in 1 minute was defined as 1 unit.<br>
The proline iminopeptidase activity of the purified enzyme was 5.83 x 103U/mg. (Example 9)      Isolation of Proline Iminopeptidase (pepl) Gene from<br><br>
Pseudomonas putida Strain ATCC 12633 and Expression in Escherichia coli<br>
1.	Acquisition of Partial Fragment of Proline Iminopeptidase (pepl) Gene<br>
A DNA was isolated from cultured cells of Pseudomonas putida strain<br>
ATCC 12633 using the same method as that in the section 3 of Example 7.<br>
On the other hand, synthetic DNA oligomers (SEQ ID NO: 10: GGC GGA TCC GGT GCT CAA AGC GCA A and SEQ ID NO: 11: GGC GGA TC AGG TCG CCG CGT TCT TC) were produced based on the partial base sequence (AF032970) of proline iminopeptidase of Pseudomonas putida strain ATCC 12633 published in Genbank, and a partial gene fragment was amplified by PCR with TaKaRa LA (manufactured by Takara Shuzo) using these as primers.<br>
2.	Cloning Whole Length of Proline Iminopeptidase Gene from Gene<br>
Library<br>
In order to acquire the whole length of proline iminopeptidase (pepl) gene of Pseudomonas putida strain ATCC 12633, Southern hybridization was first carried out in the same manner as described in part 5 of Example 7 using the partial fragment as a probe.   As a result, a band was detected at about the 2.8 kb position in the Xho\ cleaved product.    Next, this 2.8 kb region fragment was collected and coupled to the Sail site of pUC18 to produce a library (100 strains) with Escherichia coli JM109.    Colony hybridization was then carried out in the same manner as the Example and one positive clone was selected.<br>
3.	Base Sequence of Proline Iminopeptidase Gene of Pseudomonas<br>
putida strain ATCC 12633<br>
Plasmids possessed by the selected transformant were prepared in<br><br>
accordance with the method described in Molecular Cloning, 2nd edition, Cold Spring Harbor Press (1989), and the base sequence of a portion where hybridization with the probe occurred and nearby was determined. An open reading frame (ORF) was present that encoded a protein consisting of 337 amino acids, and it was verified that the whole length of this gene was acquired.   The base sequence of the whole length of proline iminopeptidase gene is shown in SEQ ID NO: 14 of the Sequence Listing.<br>
Note that the resulting ORF exhibited a base sequence homology of 46.3% with known proline iminopeptidase derived from Corynebacterium species bacteria, and exhibited a homology of 82.4% with proline iminopeptidase of Pseudomonas putida PA01. 4.      Expression of Proline Iminopeptidase Gene in Escherichia coii<br>
Plasmid pUCPPPEPI coupled to proline iminopeptidase (pepl) downstream of the lac promoter of pUC18 was constructed in order to express pepl in Escherichia coli.    Fragments amplified by PCR using a chromosomal DNA as a template and the oligonucleotides shown in SEQ ID NOs: 9 and 11 (SEQ ID NO: 9: GGC GGA TCC GGT GCT CAA AGC GCA A; SEQ ID NO: 11: CAC GCG CTG CAG CAA ACC CCT CAT) as primers were treated, and after ligating with the cleaved products of pUC18, were used to transform Escherichia coli JM109.   Strains having the target plasmid were selected from ampicillin-resistant strains, and the constructed expression plasmid was designated as pUCPPPEPI.<br>
The Escherichia coli transformant having pUCPPPEPI was see4-GU^faredpre-cultured for 16 hours at 37°C in an LB medium containing 0.1 mg/ml ampicillin.    1 mL of this pre-culture broth was seededinoculated into<br><br>
a 500 mL Sakaguchi flask containing 50 mL of LB medium followed by main culturing at 37°C.   Two hours after the start of culturing, isopropyl-1-thio-p-D-galactopyranoside (IPTG) was added to a final concentration of 1 mM, followed by additional culturing for 3 hours.<br>
After completion of the culturing, the microbes were collected and washed, suspended in 10 mL of 20 mM phosphate buffer (pH 8.0), and then subjected to ultrasonic crushing for 30 minutes at 180 W.   The solution was collected and centrifuged at 12,000gx10 minutes, and the resulting supernatant was used as a cell-free extract.   As a result, proline iminopeptidase activity (1.21 x 103 U/mg) was detected only in the case where pUCPPPEPI was introduced, and as a result, it was verified that the cloned pepl gene had been expressed in Escherichia colL (Example 10)    Synthesis of L-Alanyl-L-Glutamine by Proline Iminopeptidase of Pseudomonas putida strain ATCC 12633 1.      Detection of L-Alanyl-L-Glutamine-forming activity in Pseudomonas putida strain ATCC 12633<br>
Pseudomonas putida strain ATCC 12633 was liquid cultured overnight at 30°C in an L medium to obtain microbial cells.   The acquired microbial cells were suspended in 0.1 M borate buffer (pH 9.0), 10 mM EDTA, and the resulting suspension was used as an enzyme liquid.    Note that determination of L-alanyl-L-glutamine -forming    activity was carried out according to the enzyme activity measurement method described below. An enzyme reaction was carried out at 30°C in a reaction solution composed of borate buffer (pH 9.0) at a final concentration of 0.1 M, EDTA at 10 mM, L-alanine methyl ester hydrochloride at 100 mM and L-glutamine<br><br>
at 150 mM, and the amount of L-alanyl-L-glutamine produced accompanying the reaction was determined by HPLC.   The activity that produces 1 jaM of L-alanyl-L-glutamine in 1 minute was defined as 1 unit.<br>
L-alanyl-L-glutamine -forming   activity of 0.051 unit per 1 mL of culture liquid was detected.<br>
2.      Synthesis of L-Alanyl-L-Glutamine by Escherichia coli Expressing Proline Iminopeptidase<br>
L-alanyl-L-glutamine -forming activity was measured using the aforementioned cell-free extract.   As a result, an<br>
L-alanyl-L-glutamine-forming activity of 7.88 U/mg was detected in the case of introducing pUCPPPEPI, and it was verified that the cloned pepl gene was L-alanyl-L-glutamine -forming enzyme gene.   The maximum accumulation of L-alanyl-L-glutamine was 25 mM. (Example 11)    Isolation of Proline Iminopeptidase (pepl) Gene from Pseudomonas putida Strain FERM BP-8123 and Expression in Escherichia coli 1.      Acquisition of Proline Iminopeptidase (pepl) Gene Section<br>
A DNA was isolated from cultured cells (50 mL culture) of Pseudomonas putida strain FERM BP-8123 using the same method as Example 1 to acquire proline iminopeptidase (pepl) from Pseudomonas putida strain FERM BP-8123.<br>
On the other hand, Southern hybridization was first carried out in the same manner as that described in the section 5 of Example 9 using the partial fragment of pepl gene of Pseudomonas putida strain ATCC 12633 amplified in Example 9 as a probe.   As a result, a band was detected at<br><br>
about the 6.5 kb position in the Pst\ cleaved product.<br>
This 6.5 kb region fragment was collected and coupled to the Pst\ site of pUC18 to produce a library (200 strains) with Escherichia coli JM109. Colony hybridization was then carried out in the same manner as the example and two positive clones were selected.<br>
2.	Base Sequence of Proline Iminopeptidase Gene of Pseudomonas<br>
putida strain FERM BP-8123<br>
Plasmids retained by the selected transformants were prepared in accordance with the method described in Molecular Cloning, 2nd edition, Cold Spring Harbor Press (1989), and the base sequence of the position where hybridization with the probe occurred and nearby was determined. An open reading frame (ORF) was present that encoded protein consisting of 323 amino acids, so that it was verified that the whole length of this gene had been acquired.   The base sequence of the whole length of proline iminopeptidase gene is shown in SEQ ID NO: 16 of the Sequence Listing. The resulting ORF exhibited a base sequence homology of 83% and an amino acid homology of 85% with the proline iminopeptidase of Pseudomonas putida strain ATCC 12633.<br>
3.	Expression of Proline Iminopeptidase Gene in Escherichia coli<br>
A plasmid coupled to pepl gene downstream of the lac promoter of pUC18 was constructed in order to express pepl gene in Escherichia coli. Fragments amplified by PCR using a chromosomal DNA as a template and the oligonucleotides shown in SEQ ID NOs: 12 and 13 (SEQ ID NO: 12: CCC GAA TTC TTA CGG AGC GCG CAA TG; SEQ ID NO: 13: CGG GGA TCC CTT CAT GCT TCT TCA GG) as primers were treated, and after<br><br>
ligating with the cleaved products of pUC18, were used to transform Escherichia coti JM109.    Strains having the target plasmids were selected from ampicillin-resistant strains, and the constructed expression plasmid was designated as pUCPGPEPI.<br>
The Escherichia coli transformant having pUCPGPEPI was used to produce a cell-free extract using the same method as that in Example 8, and when proline iminopeptidase activity was measured, activity (3.48 x 101 U/mg) was detected, so that it was verified that the cloned pepl gene had been expressed in Escherichia coli,<br>
(Example 12)    Synthesis of L-Alanyl-L-Glutamine by Proline Iminopeptidase of Pseudomonas putida strain FERM BP-8123<br>
1.	Detection of L-Alanyl-L-Glutamine-Forming activity in Pseudomonas<br>
putida strain FERM BP-8123<br>
Enzyme activity was measured by culturing Pseudomonas putida strain FERM BP-8123 in the same manner as that in Example 10. L-alanyl-L-glutamine -forming activity of 0.054 unit per 1 ml_ of culture liquid was detected.<br>
2.	Detection of L-AIanyl-L-Glutamine -forming enzyme Activity<br>
L-alanyl-L-glutamine-forming activity was measured using a cell-free<br>
extract of Escherichia coli transformant retaining pUCPGPEPI.   As a result, an L-alanyl-L-glutamine-forming activity of 0.470 U/mg was detected in the case of introducing pUCPGPEPI, so that it was verified that the cloned pepl gene was L-alanyl-L-glutamine-forming enzyme gene.   The maximum accumulation of L-alanyl-L-glutamine was 30 mM. (Example 13)    Synthesis of L-Alanyl-L-Glutamine by Enzyme Having<br><br>
Proline iminopeptidase Activity of Bacillus coagulans strain EK01<br>
L-alanyl-L-glutamine -forming activity was measured according to the method disclosed in Example 10 using purified enzyme (3.93 x 105 U/mg) from Bacillus coagulans strain EK01 commercially available from Toyobo Co., Ltd.   As a result, an activity of 52.0 U/mg was detected, so that it was verified that this proline iminopeptidase is an enzyme having L-alanyl-L-glutamine-forming activity.   The maximum accumulation of L-alanyl-L-glutamine was 18 mM. (Example 14)    Effect of Inhibitors on Acquired Enzyme Activity<br>
The effects of inhibitors on acquired proline iminopeptidase were investigated.   The enzyme reaction was carried out for 30 minutes at 30°C with a reaction solution consisting of 0.1 M borate buffer (pH 9.0), 10 mM EDTA, 100 mM L-alanine methyl ester hydrochloride, 150 mM L-glutamine and 1 mM inhibitor, followed by measurement of L-alanyl-L-glutamine synthesis.    Enzyme activity was nearly completely inhibited when NEM (N-ethylmaleimide) was added at 1 mM for Bacillus coagulans strain EK01, Pseudomonas putida strain ATCC 12633, and Pseudomonas putida strain FERM BP-8123.    In addition, the activities were also decreased to certain extents after the addition of IAA (iodoacetoamide) at 1 mM.    On the other hand, the addition of PMSF (phenyl methyl sulfonyl fluoride) at 1 mM did not have an effect on the enzyme activity.   The activity was not affected by any of the inhibitors investigated for Corynebacterium glutamicum strain ATCC 13286.<br>
[Sequence Listing Free Text]<br><br>
SEQ ID NO: 1: N-terminal amino acid sequence of a peptide-forming enzyme derived from Corynebacterium glutamicum<br>
SEQ ID NO: 2: PCR primer<br>
SEQ ID NO: 3: PCR primer<br>
SEQ ID NO: 4: Code sequence of a peptide-forming enzyme derived from Corynebacterium glutamicum<br>
SEQ ID NO: 5: Amino acid sequence of a peptide-forming enzyme derived from Corynebacterium glutamicum<br>
SEQ ID NO: 6: Primer<br>
SEQ ID NO: 7: Primer<br>
SEQ ID NO: 8: Primer<br>
SEQ ID NO: 9: Primer<br>
SEQ ID NO: 10: Primer<br>
SEQ ID NO 11: Primer<br>
SEQ ID NO: 12: Primer<br>
SEQ ID NO: 13: Primer<br>
SEQ ID NO: 14: Code sequence of a peptide-forming enzyme derived from Pseudomonas putida ATCC 12633<br>
SEQ ID NO: 15: Amino acid sequence of a peptide-forming enzyme derived from Pseudomonas putida ATCC 12633<br>
SEQ ID NO: 16: Code sequence of a peptide-forming enzyme derived from Pseudomonas putida FERM BP-8123<br>
SEQ ID NO: 17: Amino acid sequence of a peptide-forming enzyme<br>
derived from Pseudomonas putida FERM BP-8123<br>
	Although the invention has been described with respect to a specific<br><br>
embodiment for a complete and clear disclosure, the appended claims are not to be thus limited but are to be construed as embodying all modifications and alternative constructions that may occur to one skilled in the art which fairly fall within the basic teaching herein set forth.<br>
INDUSTRIAL APPLICABILITY<br>
According to the dipeptide production method of the present invention, a dipeptide can be produced using an L-amino acid ester and an L-amino acid that can be acquired inexpensively without going through a complex synthesis method, making it possible to reduce the production cost of dipeptides useful as pharmaceutical materials, functional foods and so forth. In addition, according to the dipeptide production method of the present invention, various types of dipeptides can be produced using various types of L-amino acid esters and L-amino acids as raw materials.<br><br><br><br><br><br><br><br><br><br><br><br>
CLAIMS<br>
2.	A protein mSeated irv(C) or (D)-below:<br>
(C)	a protein having an amino acid sequence described in SEQ ID NO: 15 of the Sequence Listing,<br>
(D)	a protein consisting of an amino acid sequence that includes substitution, deletion, insertion, addition or inversion of one or a plurality of amino acids in the amino acid sequence described in SEQ ID NO: 15 of the Sequence Listing, and has activity to produce a dipeptide from an L-amino acid ester and an L-amino acid.<br>
3.	A protein iR€feaied-m-(E) or (F)-below:<br>
(E)	a protein having an amino acid sequence described in SEQ ID NO: 17<br>
of the Sequence Listing,<br>
(F)	a protein consisting of an amino acid sequence that includes<br>
substitution, deletion, insertion, addition or inversion of one or a plurality of<br>
amino acids in the amino acid sequence described in SEQ ID NO: 17 of the<br>
Sequence Listing, and has activity to produce a dipeptide from an L-amino<br>
acid ester and an L-amino acid.<br>
5.      A DNA mdiGate44n-(c) or (d)-betew:<br>
(c)	a DNA consisting of a base sequence consisting of bases numbers 486 to 1496 in a base sequence described in SEQ ID NO: 14 of the Sequence Listing,<br>
(d)	a DNA that hybridizes under stringent conditions with a DNA<br><br>
consisting of a base sequence complementary to a base sequence consisting of bases numbers 486 to 1496 in the base sequence described in SEQ ID NO: 14 of the Sequence Listing, and encodes a protein having activity to form a dipeptide from an L-amino acid ester and an L-amino acid.<br>
6.	A DNA m4f^ed-ifr(e) or (f)-betew:<br>
(e)	a DNA consisting of a base sequence consisting of bases numbers 311 to 1279 in a base sequence described in SEQ ID NO: 16 of the Sequence Listing,<br>
(f)	a DNA that hybridizes under stringent conditions with a DNA consisting of a base sequence complementary to a base sequence consisting of bases numbers 311 to 1279 in the base sequence described in SEQ ID NO: 16 of the Sequence Listing, and encodes a protein having activity to form a dipeptide from an L-amino acid ester and an L-amino acid.<br><br>
7.	The DNA according to claim 5 or 6, wherein the stringent conditions are conditions under which washing is carried out at 60°C and at a salt concentration equivalent to 1 x SSC and 0.1% SDS.<br>
8.	A recombinant DNA comprising incorporated therein the DNA according to any one of claims 5 to 7.<br>
9.	A transformed cell comprising incorporated therein the DNA according to any one of claims 5 to 7 in a state where the DNA is able to express a protein encoded thereby.<br><br>
10.	A method for producing a dipeptide-forming enzyme, comprising: culturing the transformed cells according to claim 9 in a medium, and accumulating a protein having activity to produce the dipeptide from an L-amino acid ester and an L-amino acid in the medium and/or in the transformed cells.<br>
11.	A method for producing a dipeptide, comprising: producing a dipeptide from an L-amino acid ester and an L-amino acid using a protein having activity to form the dipeptide from an L-amino acid ester and an L-amino acid that is produced in the transformed cells according to claim 9.<br>
12.	The method for producing a dipeptide according to claim 11, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-a-ester, an L-aspartic acid-p-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a,p-dimethyl ester and an L-glutamine-y-ester.<br>
13.	The method for producing a dipeptide according to claim 11 or 12, wherein the L-amino acid is one or more types selected from the group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine,<br><br>
L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine and L-glutamate.<br>
14.	A method for producing a dipeptide comprising: allowing a protein having proline iminopeptidase activity to act on an L-amino acid ester and an L-amino acid to form the dipeptide.<br>
15.	The method for producing a dipeptide according to claim 14, wherein the protein having proline iminopeptidase activity is derived from a microbe belonging to genus Corynebacterium, Pseudomonas or Bacillus.<br>
16.	The method for producing a dipeptide according to claim 14, wherein the protein having proline iminopeptidase activity is derived from any of Corynebacterium glutamicum, Pseudomonas putida and Bacillus coagulans.<br>
17.	The method for producing a dipeptide according to any one of claims 14 to 16, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-a-ester, an L-aspartic acid-(3-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a,(3-dimethyl ester and an L-glutamine-y-ester.<br>
18.	The method for producing a dipeptide according to any one of claims<br>
&gt;     14 to 17, wherein the L-amino acid is one or more types selected from the<br><br>
group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosinet L-lysine, L-arginine, L-histidine and L-glutamate.<br>
19.	A method for producing a dipeptide, comprising: producing the dipeptide from an amino acid ester and an amino acid using a culture of a microbe belonging to the genus Corynebacterium, Pseudomonas or Bacillus and having the ability to produce the dipeptide from the amino acid ester and the amino acid, microbial cells isolated from the culture or a treated microbial product of the microbe.<br>
20.	The method for producing a dipeptide according to claim 19, wherein the L-amino acid ester is one or more types selected from the group consisting of an L-alanine ester, a glycine ester, an L-valine ester, an L-isoleucine ester, an L-methionine ester, an L-phenylalanine ester, an L-serine ester, an L-threonine ester, an L-glutamine ester, an L-tyrosine ester, an L-arginine ester, an L-aspartic acid-cc-ester, an L-aspartic acid-p-ester, an L-leucine ester, an L-asparagine ester, an L-lysine ester, an L-aspartic-a,p-dimethyl ester and an L-glutamine-y-ester.<br>
21.	The method for producing a dipeptide according to claim 19, wherein the L-amino acid is one or more types selected from the group consisting of L-glutamine, L-asparagine, glycine, L-alanine, L-leucine, L-methionine, L-proline, L-phenylalanine, L-tryptophan, L-serine, L-threonine, L-tyrosine, L-lysine, L-arginine, L-histidine and L-glutamate.<br><br>
22.	A protein substantially as herein above described with reference to the accompanying drawing.<br>
23.	A method for producing a dipeptide substantially as herein above described with reference to the accompanying drawing.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">136-chenp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">136-chenp-2004-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtY2xhaW1zIG9yaWdpbmFsLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">136-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">136-chenp-2004-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpIG9yaWdpbmFsLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">136-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">136-chenp-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTM2LWNoZW5wLTIwMDQtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">136-chenp-2004-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="202920-a-printing-cartridge-with-switch-array-identification.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="202922-image-printing-apparatus-including-a-microcontroller.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>202921</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>136/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Feb-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>03-Nov-2006</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jan-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. AJINOMOTO CO., INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>15-1, Kyobashi 1-chome, Chuo-ku, Tokyo 104-8315</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TONOUCHI, Naoto</td>
											<td>c/o Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, Kanagawa 210-8681</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SUZUKI, Sonoko</td>
											<td>c/o Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, Kanagawa 210-8681</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YOKOZEKI, Kenzo</td>
											<td>c/o Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, Kanagawa 210-8681</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NOZAKI, Hiroyuki</td>
											<td>c/o Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, Kanagawa 210-8681</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SUGIYAMA, Masakazu</td>
											<td>c/o Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, Kanagawa 210-8681</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/48</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2002/007635</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-07-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2001-310547</td>
									<td>2001-10-05</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2001-226568</td>
									<td>2001-07-26</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/202921-a-method-producing-a-dipeptide-forming-enzyme-and-a-method-of-producing-a-dipeptide-therewith by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:19:27 GMT -->
</html>
