<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/204279-a-method-of-production-of-a-cyclic-l-amino-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:33:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 204279:A METHOD OF PRODUCTION OF A CYCLIC L-AMINO ACID</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PRODUCTION OF A CYCLIC L-AMINO ACID</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention concerns a method for producing a cyclic L-amino acid of formula (I), characterised in that it consists in reacting a L-diamino acid of formula (II) or an enantiomeric mixture comprising such a L-diamino acid and a corresponding D-diamino acid in variable proportions, in the presence of an ornithine cyclodeaminase or a polypeptide homologous to the ornithine cyclodeaminase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br><br><br><br>
	The present invention relates to the stereoselective preparation of cyclic L-amino acids or salts and derivatives thereof.<br>
There is an increasing demand for cyclic amino acids, particularly L-proline, L-pipecolic acid or L-piperazine-2-carboxylic acid and salts thereof because of their use for the synthesis of new medicaments. Given that these are chiral molecules, synthesis thereof is difficult.<br>
From the biosynthetic point of view, only the formation of the ring of L-proline has been explained and described. It can be done in two ways:<br>
either by cyclisation of glutamic semialdehyde acid to form Al-pyrroline-5-carboxylic acid, followed by a reduction of the unsaturated ring;<br>
or directly from L-ornithine by cyclodeamination.<br>
However, these processes do not provide a satisfactory solution for the synthesis of chiral cyclic amino acids on an industrial scale.<br>
Moreover, an enzymatic activity of direct conversion of L-ornithine into L-proline has been described for the first time in Clostridium (R.N. Costilow et al., J. Biol. Chem., 246 (21), (1971), 6655-60; W. L. Muth et al., 1 Biol Chem., 249 (23), (1974), 7457-62, and a total conversion of 20 mM of L-ornithine into L-proline has been obtained in vitro in the presence of the partially purified enzyme. This ornithine cyclodeaminase (ocd) of Clostridium has the particular characteristic of being activated by NAD, a cofactor usually involved in the oxidation reactions, and seems ; to be quite instable when in a purified form. It does not react on D-ornithine, but is inhibited in the presence of 40 mM of D-ornithine, whilst in the presence of 40 mM<br><br>
of L-lysine its activity is not reduced. The possibility that the cyclodeaminase might act on the L-lysine has therefore not even been envisaged by these authors, probably because it is very often acknowledged that the enzymes of the metabolism are very specific.<br>
Some years later, the correlation between the expression of a gene of Agrobacterium involved in the degradation of nopaline and an ocd activity was established (N. Sans et al., Eur. J. Biochem., 173, (1988), 123-130), and the gene coding for ocd was then sequenced.<br>
A second very homologous gene was then found in another strain of Agrobacterium by the same authors (U. Schindler et al., J. Bacteriol, 171, (1989), 847-854). The properties of the two cyclodeaminases were studied and only an activity on L-ornithine was demonstrated. Thus, again, the possibility that L-lysine might be a substrate of the cyclodeaminase was not envisaged, only its inhibiting effect was examined (Schindler et al., ibid).<br>
Since then the total or partial sequencing of numerous organisms has permitted the identification of several new genes having a strong homology with this ocd gene of Agrobactehum. However, neither the enzymatic activity of the polypeptide encoded by each of these genes, nor its spectrum of activity have been studied, this enzyme being considered specific to ornithine. In particular, the specificity of these enzymes and their level of catalytic activity remain unknown even though these are determinant elements for the productivity of an enzymatic process.<br>
For three of these genes found in Streptomyces, producers of metabolites derived from L-pipecolic acid, a lysine cyclodeaminase activity for the encoded polypeptide has been postulated (WO-A1-96/01901; L.E. Khaw et al., J. Bacteriol., 180, (1998), 809-814; I. Molnar et al., Gene, 169, (1996), 1-7; H. Motamedi et al., Eur. J. Biochem.. 249, (1998), 528-534).<br><br>
Thus it has been suggested that a cyclodeaminase encoded by the gene rapL (Khaw et al., ibid.) or pipA (WO-A-96/01901) could be involved in the synthesis route of Rapamycin via transformation of lysine into pipecolic acid. To date, however, even if it has sometimes been suggested obiter dictum, no activity of cyclodeamination of L-lysine into L-pipecolic acid has been demonstrated, exemplified or quantified. This lack of knowledge of the reaction actually catalysed by these genes is demonstrated by the diversity of hypotheses concerning the biological route leading to pipecolic acid by certain of these authors who also relate in these publications that the conversion of lysine into pipecolic acid could be effected via D-lysine (Khaw et al., ibid.).<br>
Other biochemical studies have described the formation of L-pipecolic acid from a-aminoadipic acid by reduction of an unsaturated ring (A.J. Aspen et al., Biochemistry, /, (4), (1962), 606-612) in Aspergillus and, on the basis of isotopic marking experiments, such biosynthesis routes had also been proposed in a Streptomyces by persons skilled in the art (J.W. Reed et al., J. Org. Chem., 54 (5), (1989), 1161-65).<br>
The fact that the enzymes of biosynthesis of L-proline, and in particular ornithine cyclodeaminase, had not been proposed for the production of new cyclic amino acids constitutes additional proof that, in the general opinion of persons skilled in the art, it was not possible to achieve the biosynthesis of pipecolic acid in this manner. Thus several biotechnological processes of bioconversion leading to enantiomerically pure forms, and more particularly to the L form of pipecolic acid or of piperazine-2-carboxylic acid, have been protected by patents for several years. The examples appearing in these publications report a maximum production of 15 g/L of the cyclic amino acid; no mention has been found of the use of a cyclodeaminase.<br><br>
All of these documents, with the exception of only one, describe the resolution of a racemic substrate. They use activities of the N-acylase type (WO 95/10604, WO 99/07873), the amidase type (EP-A-0 686 698). the amino acid oxidase type (JP 06030789), the nitrilase type (JP 06038781, JP 11127885) or the esterase type (WO 00/12745) acting on suitable substrates.<br>
All of these methods have a certain number of drawbacks, amongst which mention may be made of a limitation of the maximum yield to 50% on the racemic mixture, the necessity of a separation of the product thus formed and of the substrate which is not transformable by the enzyme in order to recover the enantiomeric form sought, and the necessity of possible development of recycling of the other enantiomeric form. Only JP 06030781 describes the conversion of L-lysine, a chiral substrate which is commercially available and not very expensive, into L-pipecolic acid by various non-recombinant bacterial isolates, but without the sequence of chemical reactions borrowed in order to effect this bioconversion. The performances, in terms of yield and productivity, mentioned in this document - namely a production in 7 days of approximately 4.2 g/L of L-pipecolic acid from 10 g/L of L-lysine - remain quite insufficient for such a method to be advantageous and economically competitive. One of the objects of the invention is to obtain better performances than that known for the prior art, and in particular production of cyclic amino acids of the order of 5, 10 or even 20 times greater or more than those disclosed in the prior art.<br>
The present applicant has now shown that it is possible to convert with high yields concentrated solutions of diamino acids into solutions of a-imino-cyclic acids, particularly in aqueous solution of ammonium salts of a-imino-cyclic acids, by employing an enzyme encoded by the gene of ornithine cyclodeaminase (EC 4.3.1.12) of the strain of Agrobacterium C58 or by a gene homologous thereto, or recombinant microorganisms overproducing such enzymes.<br><br><br>
in which:<br>
R1 is chosen from amongst the hydrogen atom, a linear or branched alkyl radical having from 1 to 6 carbon atoms and a linear or branched acyl radical having from 1 to 6 carbon atoms; and<br>
*	X represents a saturated, or partially or totally unsaturated, linear or branched C1-<br>
C9, preferably C2-C4, hydrocarbon chain, optionally comprising in the chain and/or at<br>
the end of the chain one or several heteroatoms or heterogroups chosen from amongst<br>
O, S, P, NR2, R2 representing H or a C1-C4 alkyl or acyl group, the said chain also<br>
being optionally substituted by one or several identical or different radicals chosen<br>
from amongst -R, -OR, -SR, =0, -C(0)OR, -C(S)OR, -C(0)NR"R", -C(S)NR'Rv,<br>
-CN, -N02, -X, -MgX, -NR'R", -NR'C(0)R, -SiR and -SiOR, R, R' and RM, identical<br>
or different, representing hydrogen or a linear or branched, saturated, or totally or partially unsaturated, hydrocarbon radical having from 2 to 20 carbon atoms, it being understood that R' and R" can form a ring with the atom carrying them.<br><br><br>
in which X and R are as defined above;<br>
or a salt or derivative thereof,<br>
or an enantiomeric mixture comprising a L-diamino acid of formula (II) and a<br>
corresponding   D-diamino   acid,   the   salts   or   derivatives   thereof  in   variable<br>
proportions, preferably in an aqueous medium,<br>
is made to react in the presence of an ornithine cyclodeaminase, or a polypeptide<br>
homologous to ornithine cyclodeaminase, the enzyme or the homologous polypeptide<br>
being obtained from a recombinant expression vector expressing the said enzyme or<br>
the said homologous polypeptide,<br>
b) the cyclic L-amino acid of formula (I) or a salt or derivative thereof is recovered in an enantiomeric excess of at least 80%.<br>
"Amino acid derivative of formula (I) or (II)" is understood to mean an amide or ester thereof.<br><br>
in which Rt represents H or a C1-C6 alkyl group or a C1-C6 acyl group, and X represents a saturated linear or branched C2-C9, preferably C2-C4, hydrocarbon chain, optionally interrupted by one or several heteroatoms or heterogroups chosen from amongst 0, S, NR2, R2 representing H or a C1-C4 alkyl or acyl group, and/or optionally substituted by one or several hydroxy, amino or halogenated groups,<br>
characterised in that:<br><br><br>
or a salt or derivative thereof,<br>
or an enantiomeric mixture comprising a L-diamino acid of formula (II) and a<br>
corresponding   D-diamino   acid,   the   salts   or   derivatives   thereof   in   variable<br>
proportions, preferably in an aqueous medium,<br>
is made to react in the presence of an ornithine cyclodeaminase, or a polypeptide<br>
homologous to ornithine cyclodeaminase, the enzyme or the homologous polypeptide<br>
being obtained from a recombinant expression vector expressing the said enzyme or<br>
the said homologous polypeptide, and<br>
b) the cyclic L-amino acid of formula (I) or a salt or derivative thereof is recovered in an enantiomeric excess of at least 80%.<br>
When the hydrocarbon chain has one or several unsaturations of an ethylenic and/or acetylenic nature, these are preferably not carried by the carbon atoms situated in the a position of the nitrogen atoms forming the amine functions of the diamino acid of formula (II).<br>
It is also preferred that, when the hydrocarbon chain has one or several heteroatoms, the said heteroatoms are separated by at least two carbon atoms.<br>
The compounds of formula (I) obtained using the method according to the present<br>
invention most often include a ring of 3, 4, 5, 6 or 7 bonds, which are the rings most<br>
&gt; f<br>
frequently encountered in the field of organic chemistry. The compounds of formula (I) for which the ring has 5, 6, or 7 bonds are preferred and are the ones which seem the most accessible to the person skilled in the art.  However, the method according<br><br>
to the present invention should not be limited to the synthesis of these compounds with rings of 5, 6 or 7 bonds.<br>
The method is also preferred in which the compound of formula (I) comprises a ring of six bonds, X representing a hydrocarbon chain with four bonds. Even more specifically, the method is implemented in order to obtain a compound of formula (I) in which X is a linear or branched alkylene chain. In the present invention, a hydrocarbon chain should be understood to mean a chain having atoms of carbon and of hydrogen.<br>
For the method according to the present invention, it should be understood that the L-diamino acid of formula (II), as defined above, is placed in the presence of at least one enzyme and/or at least one polypeptide homologous to ornithine cyciodeaminase, obtained from a recombinant expression vector expressing this or these enzymc(s) or homologous polypeptide(s).<br>
Ornithine cyciodeaminase is understood to mean any enzyme capable of cyclising a diamino acid, more precisely an amino-a-amino acid and particularly ornithine and lysine. The ornithine cyciodeaminase to which reference is made in the rest of the text is preferably the ornithine cyciodeaminase (EC 4.3.1.12) of the strain of Agrobacterium C58.<br>
The ornithine cyciodeaminase of the strain of Agrobacterium C58, coded by a gene carried by the plasmid TiC58, is described by N. Sans et al., Eur. J. Biochem., 173, (1988), 123-130, and its polypeptide sequence is equally accessible on Genbank (gi:68365).<br>
"Polypeptide homologous to ornithine cyciodeaminase" is understood to mean the polypeptides having a sequence of amino acids homologous to that of an ornithine cyciodeaminase, particularly that of the strain of Agrobacterium C58.<br><br>
These homologous sequences can be defined as the sequences similar to at least 25% of the sequence of amino acids of the ocd gene of Agrobacterium and having an ornithine cyclodeaminase activity such as is described by R.N. Costilow et al„ J. Biol Chern., 246, 21, (1971), 6655-60.<br>
The term "similar" refers to the perfect resemblance or identity between the amino acids compared but also to the imperfect resemblance which is termed similarity. This search for similarities in a polypeptide sequence takes into account the conservative substitutions which are substitutions of amino acids of-the same class, such as substitutions of amino acids with uncharged side chains (such as asparagine, glutamine, serine, threonine and tyrosine), amino acids with basic side chains (such as lysine, arginine and histidine), amino acids with acidic side chains (such as aspartic acid and glutamic acid), amino acids with apolar side chains (such as glycine, alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan and cysteine).<br>
More generally, "homologous sequence of amino acids" is therefore understood to mean any sequence of amino acids which differs from the sequence of amino acids of ornithine cyclodeaminase coded by the ocd gene of Agrobacterium by substitution, deletion and/or insertion of an amino acid or a reduced number of amino acids, particularly by substitution of natural amino acids by non-natural amino acids or amino pseudoacids at positions such that these modifications do not significantly undermine the biological activity of the coded polypeptide.<br>
Advantageously such a homologous sequence of amino acids is similar to at least 35% of the sequence of the polypeptide coded by the ocd gene of Agrobacterium, preferably at least 45%.<br><br>
The homology is usually determined using a sequence analysis software package (for example. Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710 University Avenue, Madison, WI 53705. USA). Similar sequences of amino acids are aligned in order to obtain the maximum degree of homology (i.e. identity or similarity, as defined above). For this purpose it may be necessary to introduce gaps into the sequence in an artificial manner. Once the optimum alignment is achieved, the degree of homology is established by recording all the positions for which the amino acids of the two sequences compared are identical, relative to the total number of positions.<br>
These polypeptides have the feature in common that they have a a-5 or a-e diamino deaminase acid activity, without this necessarily having been established hitherto.<br>
Included amongst such homologous sequences are the sequences of polypeptides homologous to ornithine cyclodeaminase of Agrobactehum found in the microorganisms belonging to the genera Agrohacterium, Aeropyrum, Archaeoglobus, Brucella. Corynebacterium, Halobacterium, Mesorhizohium, Methcmohacterium* Pseudomonas, Rhizobium, Rhodobacfer, Sinorhizobium, Schizosaccharomyces, Sulfolobus, Thermoplasma, Staphylococcus and Streptomyces.<br>
Mention may be made in particular of the sequences of polypeptides homologous to ornithine cyclodeaminase of Agrobacterium found in the species or strains listed in Table 1 below:<br><br><br><br>
Mention may also be made of the polypeptide encoded by the gene pipA of Streptomyces pristinaespiralis (ATCC-25486) of which the sequence is described in WO-A1-96/01901 and the sequences of polypeptides homologous to the ornithine<br><br>
cyclodeaminase of Agrobacterium which may be found in the species or strains producing rapamycin, ascomycin and FK-506, as well as the Streptomyces or other microorganisms producing streptogramins, in particular Streptomyces ioidensis (ATCC-11415), Streptomyces miiakaensis (ATCC-15297), Streptomyces olivaceas (ATCC-12019), Streptomyces oslreogrisens (ATCC-27455), Streptomyces virginiae (ATCC-13161).<br>
Mention may also be made of the polypeptides homologous to the ornithine cyclodeaminase of Agrobacterium found in eukaryotic organisms such as A rabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Macropus fuliginosus, Mus musculus, and Rattus norvegicus and listed in the following Table 2:<br><br>
In a particular embodiment of the invention, these polypeptides are encoded by genes homologous to that of the ornithine cyclodeaminase of the strain of Agrobacterium C58.<br>
"Gene homologous to that of the ornithine cyclodeaminase of the strain of Agrobacterium C58" is understood to mean any gene having:<br><br>
a)	a nucleotide sequence similar to the coding sequence of the gene of the ornithine cyclodeaminase of the strain of Agrobacterium C58; or<br>
b)	a nucleotide sequence hybridising with the coding sequence of the gene of the ornithine cyclodeaminase of the strain of Agrobacterium C58 or its complementary sequence, under stringent hybridisation conditions; or<br>
c)	a nucleotide sequence coding for a polypeptide homologous to the ornithine cyclodeaminase of Agrobacterium as defined above or having an ornithine cyclodeaminase activity.<br>
Preferably, such a homologous nucleotide sequence according to the invention is similar to at least 75% of the sequence of the ocd gene of Agrobacterium or of the gene pipA, rapL, pipA* of sequence SEQ ID No. 1, rapL* of sequence SEQ ID No. 2 and rapL*L* of sequence SEQ ID No. 5 defined below, more preferably at least 85%, or at least 90%.<br>
In a preferred manner, such a homologous nucleotide sequence hybridises specifically to the complementary sequence of the sequence of the ocd gene of Agrobacterium or of the gene pipA, rapL, pipA* of sequence SEQ ID No. 1 rapL* of sequence SEQ ID No. 2 and rapL*L* of sequence SEQ ID No. 5 defined below, under stringent conditions. The parameters defining the conditions of stringency depend upon the temperature at which 50% of the matched strands separate (Tm).<br>
For the sequences comprising more than 30 bases, Tm is defined by the relation: Tm = 81.5 + 0.41 (% G+C) + 16.6 Log(concentration of cations) - 0.63 (% formamide) -(600/number of bases) (Sambrook et al., 1989).<br>
Under appropriately stringent conditions, at which the non-pecific sequences do not hybridise, the hybridisation temperature can preferably be from 5 to 10°C below Tm,<br><br>
and the hybridisation buffers used are preferably solutions with a high ionic strength such as a solution of 6xSSC for example.<br>
The term "similar sequences" used above refers to the perfect resemblance or identity between the nucleotides compared but also to the imperfect resemblance which is termed similarity. This search for similarities in the nucleic sequences distinguishes the purines and the pyrimidines for example.<br>
A homologous nucleotide sequence therefore includes any nucleotide sequence which differs from one of the identified sequences by mutation, insertion, deletion or substitution of one or several bases, or by the degeneracy of the genetic code.<br>
Such homologous sequences may be obtained from microorganisms belonging to the genera Agrobacterium, Aeropyrum, Archoeoglobus, Brucella, Corynebacterium, Halobacterium, Mesorhizobium, Methanohacterium, Pseudomonas, Rhizobium, Rhodobacter, Sinorhizobium, . Schizosaccharomyces, Sulfolobus, Thermoplasma, Staphylococcus and Strepiomyces.<br>
Mention may be made in particular of the sequences of genes coding for polypeptides homologous to the ornithine cyclodeaminase of Agrobacterium found in the species or strains listed in Table 1 above.<br>
Mention may also be made of the sequence of the gene pipA of Streptomyces pristinaespiralis (ATCC-25486) described in WO-A1-96/01901 and the sequences of the genes coding for polypeptides homologous to the ornithine cyclodeaminase of Agrobacterium which may be found in the species or strains producing rapamycin, ascomycin and FK-506, as well as the Streptomyces or other microorganisms producing streptogramins, in particular Streptomyces ioidensis (ATCC-11415), Streptomyces mitakaensis (ATCC-15297), Streptomyces olivaceus (ATCC-12019), Streptomyces ostreogriseus (ATCC-27455), Streptomyces virginiae (ATCC-13161).<br><br>
Mention may also be made of the sequences of eukaryotic genes of Arabidopsis thaliana, Drosophila melanogaster, Homo sapiens, Macropus fuliginosus, Mus muscuhis, and Ratius norvegicus which are described as encoding polypeptides homologous to the ornithine cyclodeaminase of Agrohacterium and listed in the preceding Table 2.<br>
Amongst the genes which are very suitable for the method according to the invention, mention may be made in particular of the genes rapL of Streptomyces hygroscopicus and pipA of Streptomyces pristinaespirales -of which the sequence is described respectively by Schwecke et al., Proc. Natl. Acad. Sci U.S.A., 92 (17), (1995), 7839-43 and in WO-A1-96/01901. According to an embodiment of the invention, the compound of general formula (II) or the enantiomeric mixture comprising an L-amino acid of formula (II) and a corresponding D-amino acid, salts or derivatives thereof in variable proportions, is placed in the presence of a suspension of the microorganism producing the polypeptide homologous to the ornithine cyclodeaminase.<br>
Preferred homologous genes as described above are synthetic genes obtained by directed mutagenesis as described below.<br>
According to an embodiment of the invention, the compound of formula (II) or the enantiomeric mixture comprising an L-amino acid of formula (II) and a corresponding D-amino acid, salts or derivatives thereof in variable proportions is placed in the presence of an enzyme isolated from the producing microorganism.<br>
According to another embodiment of the invention, the compound of formula (II) or the enantiomeric mixture comprising an L-diamino acid of formula (II) and a corresponding D-diamino acid in variable proportions is placed in the presence of an enzyme in the purified state.<br><br>
According to another embodiment of the invention, the compound of formula (II) or the enantiomeric mixture comprising an L-diamino acid of formula (II) and a corresponding D-diamino acid in variable proportions is placed in the presence of a recombinant enzyme, this latter embodiment being preferred.<br>
It was found surprisingly that under the conditions of the invention the polypeptides as described above permitted the stereospecific synthesis of the compounds of formula (I) of the invention without necessitating the exogenous addition of NAD to the reaction medium.<br>
In an embodiment of the invention, the enzymatic preparation or the cellular suspension which is optionally permeabilised has added to it a compound of formula (II) or a mixture of enantiomers of the compound of formula (II) and of a corresponding D-diamino acid in a concentration greater than approximately 0.05 M, preferably greater than approximately 01 M, this same concentration being less than approximately 3 M, and preferably less than 2.5 M, the salts or derivatives thereof, and this "is left to incubate at a temperature of between 10°C and 100°C, advantageously between 20 °C and 70 °C, preferably between 25 °C and 45 °C, for a period of between several hours and several days, with stirring.<br>
In this way it is possible to obtain molar yields of the compound of formula (I) of at least 20%, advantageously at least 80%, preferably at least 90% with an enantiomeric excess greater than 80%, advantageously greater than 90%, preferably greater than<br>
95%.<br>
Advantageously, the compound of formula (I) is in the form of an ammonium salt.<br>
As a variant, the enzymes extracted from the culture medium or purified with the compound of formula (II) or a mixture of enantiomers of the compound of formula<br><br>
(II) or of a corresponding D-diamino acid may also be put to incubate in a medium which may or may not be buffered at a pH of between 6 and 11 and preferably between 7 and 10.<br>
The product obtained can be collected by precipitation, crystallisation or ion exchange chromatography.<br>
Amongst the substrates preferred for the preparation of a compound of formula (I), mention may be made in particular of L-lysine or a mixture of L- and D-lysine, L-thialysine (S-2-aminoethyl-L-cysteine), L-ornithine, a mixture of L- and D-5-(R,S)-hydroxylysine, a mixture of L- and D-azalysine (y-N-2-aminoethyldiaminopropionic acid).<br>
In order to render the method more particularly efficient, the overproduction of the polypeptide with ornithine cyclodeaminase activity is particularly advantageous.<br>
With a view to this, the gene of interest coding for a polypeptide as defined above is introduced into a host microorganism, preferably overexpressing the polypeptide.<br>
To this end a vector containing a nucleic acid comprising one of the nucleotide sequences as defined above or a homologous sequence is transferred into a host cell which is cultured under conditions permitting the expression, preferably the overexpression, of the corresponding polypeptide.<br>
The nucleic acid sequence of interest can be inserted into an expression vector in which it is linked in an operative manner to elements permitting the regulation of the expression thereof, such as in particular promoters, activators and/or terminators of transcription.<br><br>
The signals controlling the expression of the nucleotide sequences (amongst which mention may be made of the promoters, the activators and the sequences of termination) are chosen as a function of the host cell used. For this purpose the nucleotide sequences according to the invention can be inserted into vectors with autonomous replication within the chosen host, or vectors which integrate into the chosen host. Such vectors will be prepared according to the methods currently used by the person skilled in the art, and the clones resulting therefrom can be introduced into an appropriate host by standard methods.<br>
Amongst the preferred vectors,-mention may be made of the plasmids pTZ18 (Sigma, St-Louis), pQE70 (Qiagen, Munich) and PQE60 (Qiagen, Munich).<br>
When they were used, the QIAGEN plasmids showed the major drawback of involving the formation of polyhistidines in the enzymes obtained. Therefore, these plasmids have to be modified in order to overcome this drawback. Thus the coding sequences must be modified or disrupted. These modifications did not bring about any difference in terms of level of expression of the gene encoding cyclodeaminase.<br>
The expression vectors are introduced into the host cells and these latter are cultured under conditions permitting the replication and/or the expression of the transfected nucleotide sequence.<br>
Examples of host cells include in particular E. coli, preferably the strains Escherichia coli DH5a, Escherichia coli (32033, and Escherichia coli K12 (MG1655). However, other host organisms belonging to different genera are equally suitable.<br>
Particularly interesting results are obtained when E. coli is transfected with genes modified relative to the corresponding native genes in suchf a way as to permit optimum expression of the gene of interest in E. coli.<br><br>
The heterologous gene is modified in such a way as to comprise a proportion of less than 65%, advantageously less than 55% of bases G + C, whilst encoding the same polypeptide as the native gene or a homologous polypeptide as defined previously.<br>
Thus for a given amino acid encoded by a codon of native DNA. if need be the third base of the codon is replaced, or the second base of the codon when this is G or C, by A or T, when the codon obtained corresponds to the same amino acid as that encoded by the native codon.<br>
The modified genes which have made it possible to obtain the best rates of expression and equally the highest yields of a-imino-cyclic acids are those in which the native codons have been replaced for a given amino acid of the polypeptide having amino acid cyclodeaminase activity by the following respective codons shown in Table 3.<br><br><br>
The modified genes were obtained by assembling sections obtained by PCR and directed mutagenesis from corresponding native genes.<br>
Amongst the modified genes which gave the best results, mention may be made of the modified genes pipA* and rapL** having respectively the sequences SEQ ID No. 1 and SEQ ID No. 5 shown in the annex to the present application.<br>
The polynucleotides corresponding to these modified genes constitute a further subject of the invention.<br>
The gene pipA* was obtained from the sequence of the polypeptide encoded by the gene pipA of Streptomyces pristinaespiralis shown in the annex as sequence SEQ ID<br>
No. 3.<br>
The gene rapL** was obtained from the sequence of the polypeptide encoded by the gene rapL of Streptomyces hygroscopicus shown in the annex as sequence SEQ ID No. 4.<br>
The modified genes are inserted into cloning vectors as described above for the native genes transfected into recombinant hosts.<br>
The host microorganisms which are, as the case may be, recombinant, and include a gene existing in the native state or modified as described above are cultivated, optionally in the presence of an expression inductor, for example IPTG.<br>
When the cells reach a density of the order of at least one unit of absorbance at 600 nm for standard cultures and of the order of several tens or more of such units for high-density cultures, they are separated from their culture medium and subjected to a permeabilisation treatment which may be physical (for example,  alternating<br><br>
freezing and thawing treatments, treatments with ultrasound or with a French press, mechanical crushing) or chemical (for example, addition of solvent or complexing agents such as EDTA), or enzymatic (for example, addition of lysozyme).<br>
The cellular suspension can then be used in order to achieve the preparation of the products of formula (II) as described above where the protein of interest which is produced can then be recovered and purified. By way of example, using a method of the "Fed-batchM type or equivalent, concentrations are obtained, expressed in grams of dry cells per litre of cellular suspension, greater than approximately 10 g/L, advantageously greater than 30 g/L. These same concentrations are, as a general rule, less than 60 g/L, or even less than 50 g/L, without this indicating an insuperable limit.<br>
The purification methods used are known to the person skilled in the art. The recombinant polypeptide obtained can be purified on the basis of lysates and cellular extracts, the supernatant of the culture medium, by methods used individually or in combination, such as fractionation, methods of chromatography, techniques of immuno-affmity with the aid of specific mono- or polyclonal antibodies, etc.<br>
However, it is not necessary to recover this protein.<br>
According to an advantageous embodiment, the host cells producing the polypeptide having the enzymatic activity required are destroyed so that their cytoplasmic content is released into the medium containing the diamino acid substrate of which the two amine groups are distant (by the shortest route when several routes are possible) by four or five bonds, in the L form or in the form of a mixture of enantiomers and in such a way as to enable the enzyme and its substrate to be brought together.<br>
The following examples illustrate the invention.<br><br>
Example 1:<br>
Amplification by PCR of the gene pipA of Streptomyces pristinaespiratis from the total DNA of the strain<br>
1.1	Extraction of the total DNA<br>
The strain Streptomyces pristinaespiralis ATCC 25486 was cultivated at 28 °C for 24 hours in the medium TSB (DIFCO), then 10 mL of the culture thus obtained were centrifuged for 5 minutes at 11000 g. The centrifugation pellet was taken up by 1 mL of a solution of 10 mM Tris pH 8.0 containing 2 mM of EDTA and 5 mg/mL of lysozyme. The suspension thus obtained was left with stirring for 30 minutes at 37 °C, then 100 μL of a 20% solution of SDS were added. After incubation at 30 °C for several minutes, 125 μL of a solution of proteinase K at 2 mg/mL were again added. Then the extraction of the DNA was carried out using the reagents of the kit DNeasy Tissue Kit (QIAGEN, Munich) following the supplier's instructions. The DNA thus extracted was purified by phenol/chloroform extraction, precipitated by ethanol and taken up in 100μl of water.<br>
1.2	Amplification of the gene pipA<br>
The gene pipA was amplified by PCR from the total DNA thus prepared using, for the synthesis of the primers, its sequence described in the patent WO-A1-96/01901. The reaction was carried out in a volume of 50 μL of buffer 20 mM Tris-HCl pH 8.8 containing 10 mM KC1, 10 mM (NH4)2S04, 2mM MgS04, 0.1% Triton X-100, 2 \xL of the total DNA solution, 250 ^M of each of the four dNTP, 1 unit of Vent DNA Polymerase (BioLabs) and 500 nM of each of the following two oligodeoxynucleotides:<br>
5' CCCGAATTCCGACACCACGCACGGACGAGAAG 5'CCCTGCAGGCATCTCTCTCCTCGCGAGGGC.<br><br>
The PCR procedure applied began by a step of denaturation at 97 °C for 4 minutes followed by incubation at 80 °C for 1 minute, and continued with 5 cycles characterised by a sequence of 1 minute of denaturation at 97 °C, followed by 1 minute of hybridisation at 55 °C, then 1 minute 30 seconds of elongation at 72 °C. Then 40 cycles characterised by a sequence of 1 minute of denaturation at 97 °C, followed by 1 minute of hybridisation at 50 °C, then 1 minute 30 seconds of elongation at 72 °C, were carried out. The procedure was completed by a step of elongation at 72°C for 10 minutes.<br>
The fragment thus amplified was purified on the column of the Qiaquick PCR purification kit (QIAGEN), then digested by EcoRI and PstI and cloned in the vector pTZl 8 (SIGMA, St-Louis, MO) previously cut. Two plasmid constructions (pKT 35, pKT 36) obtained following independent PCR amplifications were sequenced. The sequences of these two fragments thus cloned were identical. The specified protein, of sequence SEQ ID No. 3, differs however by an amino acid (alanine) at position 87 relative to the sequence published in WO-A1-96/01901 (glycine).<br>
Example 2:<br>
Amplification by PCR of the gene rapL of Streptomyces hygroscopic us from a<br>
suspension of cells of the strain.<br>
The strain Streptomyces hygroscopicus ATCC 29253 was cultivated at 28°C for 24 hours in the medium TSB (DIFCO). 50 μL of the cellular suspension thus obtained were subjected to the following steps of heating and cooling: 30 seconds at 65°C, 30 seconds at 8°C, 90 seconds at 65°C, 180 seconds at 97°C, 60 seconds at 8°C, 180 seconds at 65°C, 60 seconds at 97°C, 60 seconds at 65°C, 20 minutes at 80°C.<br>
The amplification of the gene rapL was carried out by PCR from a suspension of cells using its described sequence (Schwecke et al., 1995) for the synthesis of the<br><br>
primers. The reaction was carried out in a volume of 50 μL of buffer 20 mM Tris-HC1 pH 8.8 containing 10 mM KC1, 10 mM (NH4)2S04, 2 mM MgS04, 0.1% Triton X-100, 10 μL of the suspension of cells previously treated. 250μM of each of the four dNTP, 1 unit of Vent DNA Polymerase (BioLabs) and 500 nM of each of the following two oligodeoxynucleotides:<br>
5'CCCGAATTCGAGGTTGCATGCAGACCAAGGTTCTGTGCCAG 5' CCCAAGCTTAGATCTTTATTACAGCGAGTACGGATCGAGGACG '<br>
The PCR procedure applied began by a step of denaturation at 97 °C for 4 minutes followed by incubation at 80 °C for 1 minute, and continued first of all with 5 cycles characterised by a sequence of 1 minute of denaturation at 97 °C, followed by 1 minute of hybridisation at 60 °C, then 1 minute 30 seconds of elongation at 72 °C, then by 5 new cycles characterised by a sequence of 1 minute of denaturation at 97 °C, followed by 1 minute of hybridisation at 52 °C, then 1 minute 30 seconds of elongation at 72 °C. Then 30 cycles characterised by a sequence of 1 minute of denaturation at 97 °C, followed by 1 minute of hybridisation at 50 °C, then 1 minute 30 seconds of elongation at 72 °C, were carried out. The procedure was completed by a step of elongation at 72°C for 10 minutes.<br>
The fragment thus amplified was purified on the column of the Qiaquick PCR purification kit (QIAGEN), then digested by EcoRI and HindHI and cloned in the vector pTZ18 previously cut. Three plasmid constructions obtained (pKT 30, pKT 31, pKT 34) following independent PCR amplifications were sequenced. The sequences of these three fragments thus cloned were identical. The specified protein, of sequence SEQ ID No. 4, differs however by five amino acids relative to the sequence published by Schwecke et al., that is to say that the protein according to the invention contains in positions 50, 84, 102, 127 and 129 respectively a residue of threonine, glutamine, aspartic acid, alanine and alanine.<br><br>
Example 3: Total synthesis by PCR of the gene pipA*<br>
The gene pipA with a size of 1066 bp derived from Streptomyces pristinaespiralis and of which the sequence is described in the patent WO-A1-96/01901 is composed of a proportion of G + C of 69.6%. This gene has been rewritten according to the genetic code optimised for the expression in Escherichia coli described in Table 3. The rewritten gene pipA* of sequence SEQ ID No. 1 is henceforth composed of a proportion of G + C of 38%. In order to introduce a single restriction site Kpnl, the threonine residue 97 was encoded by ACC instead of ACT provided by the code of Table 3. Two codons for termination of the translation TAA were also added in position 15 and 1080 (1st nucleotide of the codon), respectively of the sequence SEQ ID No. 1, and the single restriction sites EcoRI and Ncol on the one hand, Hindlll on the other hand, were introduced respectively at 5' and 3' of the gene thus permitting its insertion into a cloning vector.<br>
The synthesis of the gene pipA* was carried out by assembling two sections EcoRI -Kpnl (section 1 of 305 bp - nucleotide 1 to nucleotide 304) and Kpnl - Hindlll (section 2 of 798 bp - nucleotide 305 to nucleotide 1092).<br>
Each section was synthesised by extension of single-strand oligonucleotides of a length of 50 to 60 nucleotides overlapping ("sense" and "antisense" strands), followed by an amplification by end oligonucleotides 5' and 3'.<br>
Synthesis of the section 1: The PCR reaction was carried out in a volume of 100 μL of buffer 20 mM Tris-HCl pH 8.8 containing 10 mM KC1, 10 mM (NH4)2S04, 2 mM MgS04, 0.1% Triton X-100, 0.01 μM of the oligonucleotides listed below, 0.3 mM of each of the four dNTP and 2 units of Vent DNA Polymerase (BioLabs). The PCR procedure applied was composed of 30 cycles characterised by a sequence of 30 seconds of denaturation at 96 °C, followed by 30 seconds of hybridisation at 55 °C, then 30 seconds of elongation at 72 °C At cycle number 10, the oligonucleotides of<br><br>
the ends 5' (PIPAUP) and 3' (PIPAD1) were added at a final concentration of 0.5 μM. At the end of cycle 30, a period of elongation of 2 minutes at 72 °C was added.<br>
The following oligonucleotides were used for the reaction: PipaVO, PipaV2, PipaV4, PipaV6, PipaV8, PipaD4, PipaD5, PipaD6 and PipaD7. The sequences of these oligonucleotides are described in Table 4.<br>
The PCR product thus obtained was purified on the column of the Qiaquick PCR purification kit (QIAGEN), then digested successively by the restriction enzymes Kpnl and EcoRI, and again purified on agarose gel. A band of apparent size of 300 bp was extracted from the gel using the Qiaquick gel extraction kit (QIAGEN). This purified PCR product was then ligated to the vector pTZ18, previously digested by the enzymes EcoRI and Kpnl and purified on agarose gel, by the T4 DNA ligase (BioLabs). The ligation reaction is carried out overnight at 16 °C in the ligation buffer supplied with the enzyme. The ligation product was then transformed in the strain E. Coli DH5a CIP104738 (D. Hanahan, I Moi Biol, 166, (1983), 557-580) by thermal shock and selected recombinant clones on agar LB medium (DIFCO) supplemented with ampicillin (100 mg/L).<br>
The plasmids contained in the transformants resistant to ampicillin were isolated using the plasmid preparation system QIAprep Spin Miniprep Kit (QIAGEN). For four of these plasmids, of which the analysis by appropriate restriction enzymes showed an insert of expected size, the fragments EcoRI-Kpnl were sequenced. Each of the sequences of the four constructions contained between one and three deviations relative to the expected sequence. The plasmid pSP23 for which the sequence of the fragment was correct with the exception of a deletion of 6 bp was subjected to a step of directed mutagenesis following a published method (Ansaldi et aL Anal Biochem., 234, (1996), 110-111) to correct this deletion. The phismid pSP39 was obtained in this way. The sequencing of the fragment EcoRI-Kpnl of the<br><br>
plasmid pSP39 was carried out and the sequence obtained was found to conform to the sequence expected.<br>
Synthesis of section 2: The PCR reaction for the synthesis of the section Kpnl-Hindlll was carried out as described for the section 1 with the exception of the period of elongation , the duration of which was extended to 1 minute for the 30 cycles and to 4 minutes at the end of the procedure. At cycle number 10, the oligonucleotides of the ends 5' (PIPAV29) and 3' (PIPAD3) were added at a final concentration of 0.5<br>
The following oligonucleotides were used for the reaction: PipaV8, PipaVIO, PipaV12. PipaV14, PipaV16, PipaV18. PipaV20, PipaV22, PipaV24, PipaV26, PipaV28, PipaD8, PipaD9, PipaDIO, PipaDll, PipaD12, PipaD13, PipaD14, PipaDlS, PipaD16 and PipaD17. The sequences of these oligonucleotides are described in Table 4.<br>
The PCR product was purified on the column of the Qiaquick PCR purification kit (QIAGEN), then digested by the restriction enzymes Kpnl and Hindlll and ligated to the vector pTZ18 previously digested by the same enzymes. The transformation of the strain £. coli DH5a, and the selection of the transformants and the analysis of the plasmids were carried out as described above for section 1. The plasmid pSP25 for which the sequence of the fragment Kpnl-Hindlll had five deviations relative to the expected sequence was subjected to several steps of directed mutagenesis (Ansaldi et al., 1996, ibid) to correct these errors. The plasmid pSP42 was obtained in this way.<br>
Assembly of the 2 sections: The entire gene pipA* was assembled by cloning of the fragment Kpnl-Hindlll extracted from the plasmid pSP42 in the plasmid pSP39 previously cut by the enzymes Kpnl and Hindlll.  One of the resulting plasmids, the<br><br><br><br><br><br>
a proportion of G + C of 68%. This gene has been rewritten according to the genetic code employed for the synthesis of the gene pip A* in Example 3. The rewritten gene rapL* of sequence SEQ ID No. 2 is henceforth composed of a proportion of G + C of 38%. In order to introduce a single restriction site Kgnl, the threonine residue 97 was encoded by ACC instead of ACT provided by the code shown in Table 3. Two codons TAA for termination of the translation were also added and single restriction sites EcoRI and Sphl on the one hand, HindHI on the other hand were introduced respectively at 5' and 3' of the gene thus permitting its insertion into different cloning vectors.<br>
The total synthesis of the gene rapL* was carried out by assembling the three sections EcoRI - Kgnl (section 1 of 305 bp), KpnI-PstI (section 2 of 316 bp), and PstI -HindHI (section 3 of 440 bp).<br>
Each section was synthesised by ligation of phosphorylated single-strand oligonucleotides at 5' of a length of 50 nucleotides on average, covering the totality of the sequence to be cloned ("sense" and "antisense" strands) and overlapping. The sequences and positions of these oligonucleotides are described in Table 5. The oligonucleotides, mixed in equimolar fashion (1 μM) in the overlapping buffer (20 mM Tris-HCl, pH 8; 0.08 M NaCl) in a final volume of 20 μl were heated for 10 minutes at 70 °C and kept in the heating block until they returned to ambient temperature. The product thus obtained was then ligated by the T4 DNA ligase overnight at 16 °C in the presence of the vector pTZ18 (molar concentration ratio vector/oligonucleotides 1/100) previously digested by the appropriate restriction enzymes.<br>
The ligation product was used to transform by electroporation the strain p2033 (Ee coli K12, pro thi, rpsL, hsdS, ∆lacZ, P(AlacZM15, laclq, traD36, proA+, eroB+) and the recombinant clones were selected on agar LB medium containing 100 μg/mL of ampicillin, 0.5 mM IPTG and 30 μg/mL X-Gal.<br><br>
The plasmids contained in several white transformants resistant to ampicillin were isolated using the plasmid preparation system QIAprep Spin Miniprep Kit (QIAGEN). For each section, the inserts of two to four plasmids, of which the analysis by appropriate restriction enzymes showed a fragment of the expected size, were sequenced.<br>
Synthesis of the section 1:<br>
The plasmid pTZ18 cut by EcoRI and Kpnl was ligated with the following oligonucleotides (of which the sequence is illustrated in Table 5): IcdV-1, IcdV-2, IcdV-3, IcdV-4, IcdV-5, IcdV-6, IcdVrev-16, IcdVrev-17, IcdVrev-18, IcdVrev-19, IcdVrev-20, IcdVrev-21, IcdVrev-22. The plasmid pSP3, of which the sequence of the insert had only two deviations relative to the expected sequence, was retained. The two errors were corrected by directed mutagenesis according to a published method (Ansaldi et al., 1996, ibid) and the corrections confirmed by sequencing of the insert of the resulting plasmid pSP14.<br>
Synthesis of the section 2:<br>
The plasmid pTZ18 cut by Kpnl and PstI was ligated with the following oligonucleotides (of which the sequence is illustrated in Table 5): IcdV-7, IcdV-8, IcdV-9, IcdV-10, IcdV-11, IcdV-12, IcdVrev-9, IcdVrev-10, IcdVrev-11, IcdVrev-12 IcdVrev-13, IcdVrev-14, IcdVrev-15. The plasmid pSP8, of which the sequence of the insert had only one deviation relative to the expected sequence, was retained. The error was corrected by directed mutagenesis (Ansaldi et al., 1996, ibid) and the correction confirmed by sequencing of the insert of the resulting plasmid pSP15.<br>
Synthesis, of the section 3:<br><br>
The plasmid pTZ18 cut by PstI and Hindlll was ligated with the following oligonucleotides (of which the sequence is illustrated in Table 5): IcdV-13, IcdV-14, IcdV-15, IcdV-16, IcdV-17, IcdV-18, IcdV-19, IcdV-20, IcdV-21, IcdVrev-1, IcdVrev-2, IcdVrev-3, IcdVrev-4, IcdVrev-5, IcdVrev-6, IcdVrev-7, IcdVrev-8. The plasmid pSP12, of which the sequence of the insert had only one deletion of 25 bp and one point deviation, was retained. The two errors were corrected by directed mutagenesis (Ansaldi et al., 1996, ibid) and the corrections confirmed by sequencing of the insert of the resulting plasmid pSP26.<br>
Assembly of the 3 sections: The corrected fragments KpnI-PstI, then Pstl-Hindlll are sub-cloned successively in the plasmid pSP14 already containing the fragment EcoRI-KpnL thus reconstituting the synthetic gene rapL*. The sequence of the gene rapL* of the resulting plasmid pSP33 was confirmed by sequencing.<br>
Obtaining the variant rapL** (Sequence SEQ ID No. 5): The variant rapL** was obtained from the gene rapL* of the plasmid pSP33 by introducing by directed mutagenesis (Ansaldi et al., 1996, ibid) the five changes necessary so that the protein coded by the gene rapL** is the sameas that coded by the amplified and sequenced gene described in Example 2 and corresponds to the sequence SEQ ID No. 4, whilst adhering to the genetic code described in Table 3. A plasmid pSP36 derived from the plasmid pTZ18 and containing the gene rapL** was thus obtained.<br>
Table 5: Synthesis of the gene rapL* sequence of the oligonucleotides<br>
Fragment EcoRI-Kpnl:<br>
"Sense" strand:<br>
lcdV-1 :	5' AATTCGAGGTTGCATGCAAACTAAAGTTTTATGTCAACGTGATATTAA-3,<br>
lcdV-2 :	5' p-ACGTATTTTATCTGTTGTT GGTCGTGATGTT	ATG      ATG      GAT<br>
CGT        TTA      ATT      T-3*<br><br><br><br><br><br><br>
Example 5: Overexpression of the gene pipA * in Escherichia coli<br>
The gene pipA* was cloned in the vector pQE60 (QIAGEN) (plasmid of which the<br>
coding sequences were previously modified as indicated above), between the<br>
restriction sites Ncol and HindlH on the basis of the plasmid pSP43 constructed in<br>
Example 3. The resulting plasmid pSP47 was introduced into a strain of E.coli K12<br>
MG1655 (Vidal et al., 1998) expressing the gene lad on the basis of the auxiliary<br>
plasmid pREP4 (QIAGEN). The strain +75 thus obtained was cultivated in the LB<br>
medium and the expression of the gene pipA* was induced by the addition of IPTG<br>
f according to the supplier's instructions. The total proteins of the cells were separated<br>
by polyacrylamide/SDS gel electrophoresis and stained with Coomassie blue. A<br><br>
protein with the molecular weight of 36 kD was detected and its rate of expression was estimated at approximately 5% of total proteins.<br>
Example 6: Overexpression of the genes pipA, rapL, rapL* and rapL** in Escherichia coli<br>
Proceeding as described in Example 5, a strain of £ coli overexpressing the gene pipA (strain +353) was obtained by introducing the plasmid pKT37. This plasmid was obtained from the plasmid pKT36 described in Example 1 by cloning of the gene of interest between the restriction sites Ncol and Hindlll of the vector pQE60.<br>
Similarly, strains of E. coli overexpressing the genes rapL (strain +38), rapL* (strain +60) or rapL** (strain +73) were obtained by introducing respectively the plasmids pSP32, pSP37 or pSP45. These plasmids pSP32, pSP37 or pSP45 were obtained respectively from the plasmids pKT30, pSP33 or pSP36, described in Examples 2 and 4, by cloning of the gene of interest between the restriction sites SphI and Hindlll of the vector pQE70.<br>
The rate of expression of these different proteins was determined as described in Example 5. For each strain, a protein with the expected molecular weight was detected with an expression rate of approximately 5% of total proteins.<br>
Example 7: Amplification by PCR and overexpression of the ocd gene of Agrobacterium tumefaciens in Eschericia coli<br>
A preparation of the plasmid Ti-C58 of Agrobacterium tumefaciens CIP 104333 was produced as described elsewhere (Hayman et al., Mol Gen. Genet., 223, (1990), 465-473). The ocd gene (Sans et al., (1988), ibid) was amplified by PCR from this preparation of the plasmid Ti-C58,<br><br>
The reaction was carried out in a volume of 50 μL of buffer 20 mM Tris-HCl pH 8.8 containing 10 mM KC1, 10 mM (NH4)2S04, 2 mM MgS04, 0.1% Triton X-100, 2 ng of the plasmid Ti-C58, 200 μM of each of the four dNTP, 1 unit of Vent DNA Polymerase (BioLabs) and 500 nM of each of the following two oligodeoxynucleotides:<br>
5' CCCGCATGCCTGCACTTGCCAACC<br>
5' CCCAGATCTTAACCACCCAAACGTCGGAAA<br>
The PCR procedure applied began by a step of denaturation at 94 °C for 2 minutes followed by 30 cycles characterised by a sequence of 30 seconds of denaturation at 94 °C, followed by 30 seconds of hybridisation at 52 °C, then 1 minute 30 seconds of elongation at 72 °C. The procedure is completed by a step of elongation at 72 °C for 10 minutes.<br>
The fragment thus amplified was digested by Ncol and Bglll and cloned in the previously cut vector pQE70 (QIAGEN) (plasmid of which the-coding sequences have been previously modified as indicated above). The fragment Ncol and Bglll of the plasmid pKR7 thus obtained was sequenced. The specified protein differs by two amino acids relative to the published sequence (Sans et al., (1988), ibid), namely the residue Gln212 replaced by lysine and the residue Ile297 replaced by leucine. The two deviations were confirmed by sequencing of a product of amplification by PCR obtained independently.<br>
By proceeding as described in Example 5, a strain of E. coli overexpressing the ocd gene (strain +78) was obtained by introducing the plasmid pKR7. The rate of expression of the protein coded by the ocd gene was determined as described in Example 5.    A protein with a molecular weight conforming to that which was<br><br>
expected was detected with a rate of expression of approximately 5% of the total proteins.<br>
Example 8: Production of L-pipecolic acid from L-Iysine by cellular suspensions of the recombinant strains of E. coli<br>
The strain +75 constructed in Example 5 is cultivated in the mineral medium MS (Richaud et al., J. Biol Chem., 268, (1993), 26827-35) containing 2 g/L glucose, 50 mg/L ampicillin and 30 mg/L kanamycin. At the end of 14 hours of culture, 400 mL of mineral medium MS containing 2 g/L glucose are inoculated at 1/10 with the preculture of the strain +75. This culture is placed at 37 °C with stirring until an OD6oonm of 0.7 is reached. The gene pipA* is then induced for 4 hours at 37 °C by the addition of 1 mM of IPTG to 200 mL of culture. A reference culture of 200 mL without the addition of IPTG is also produced. At the end of this step, the two cultures are centrifuged at 13000 g and the cellular pellets are taken up in mineral medium MS in such a way as to concentrate the cells 100 times. The cells are then permeabilised by two steps of freezing/thawing at -20 °C favouring the access of the substrate to the enzyme. L-lysine monohydrochloride (pH = 7) is then added at a final concentration of 1 M in the cellular suspensions (final volume = 2 mL). The enzymatic reaction is effected at 37 °C with stirring. At the end of 20 hours a sample of 300 |iL of each culture is taken which is centrifuged at 13000 g in such a way as to recover the supernatant. A dilute solution of the supernatants is then deposited on a thin layer of silica and simultaneously analysed with HPLC.<br>
A compound with chromatographic migration (Rf = 0.22; eluant: butanol/acetic acid/water 4/1/1 in CMC) and stained by ninhydrin which are indistinguishable from those of L-pipecolic acid is detected in the case of the culture induced with IPTG and absent in the case of the non-induced reference culture.<br><br>
The analysis with HPLC (conditions described below) indicates a concentration of 40 g/L of L-pipecolate at the end of 20 hours. The enantiomeric excess is greater than 95%.<br>
Tests of production of L-pipecolic acid from L-lysine were carried out with the strains +38, +60, +73, +78 and +353 constructed in Examples 6 and 7, under the same conditions as those applied above for the strain +75. The analysis by chromatography of the respective supernatants on a thin silica layer after staining with ninhydrin revealed a production of L-pipecolic acid only with the strains +73, +78 and+353.<br>
Description of the HPLC method used:<br>
Phenomenex Synergi Polar-RP-80 4Μ. (250 x 4.6 mm) column thermostat-controlled at 30 °C,<br>
UV Detection at 210 nm<br>
Volume of sample injected: 10 μL<br>
Isocratic elution by a solution of 0.05% of TFA in water for 5 minutes followed by washing of the column with 80% acetonitrile and 0.05% TFA in water,<br>
Retention time: lysine 2.98 minutes and pipecolic acid 4.62 minutes.<br>
Example 9: Production of L-thiomorphoIine-2-carboxylic acid from L-thialysine<br>
The production of L-thiomorpholine-2-carboxylic acid was carried out on the basis of a cellular suspension of the strain +75 prepared as described in Example 8. L-thialysine (S-(2-aminoethyl)-L-cysteine) (SIGMA) is added at a final concentration of 1 M. Thin silica layer chromatography of the supernatant of the incubated suspension is carried out and it reveals a compound with migration (Rf = 0.28; eluant: butanol/acetic acid/water 4/1/1) and stained by ninhydrin which are distinct from those of L-thialysine and L-pipecolic acid.<br><br>
Under the same conditions, the strain +78 described in Example 7 produces a compound identical to that produced by the strain +75.<br>
Example 10: Production of a mixture of 5-R- and 5-S-hydroxy-L-pipecoIic acids from a mixture of L- and D-5-(R,S)hydroxylysine<br>
The production of a mixture of 5-R- and 5-S-hydroxy-L-pipecolic acids was carried out on the basis of a cellular suspension of the strain +75 prepared as described in Example 8. 5-R,S-hydroxy-D,L-lysine (SIGMA) is added at a final concentration of 1 M. Thin silica layer chromatography of the supernatant of the incubated suspension is carried out and it reveals two compounds present in equivalent proportions with migration (Rfi = 0.14, Rf2 = 0.20; eluant: butanol/acetic acid/water 4/1/1) and stained by ninhydrin which are distinct from those of 5-R,S-hydroxy-D,L-lysine and L-pipecolic acid.<br>
Under the same conditions, the strain +78 described in Example 7 produces two compounds identical to those produced by the strain +75.<br>
Example 11; Preparation of other cyclic amino acids<br>
The great versatility of the method according to the present invention can be illustrated by the following cyclic amino acids, obtained using the mode of operation described in Example 8 by modifying the nature of the starting diamino acid:<br><br><br><br>
Claims<br><br>
in which:<br>
*	Ri is chosen from amongst the hydrogen atom, a linear or branched alkyl radical having from 1 to 6 carbon atoms and a linear or branched acyl radical having from 1 to 6 carbon atoms; and<br>
*	X represents a saturated, or partially or totally unsaturated, linear or branched Ci-C9, preferably C2-C4, hydrocarbon chain, optionally comprising in the chain and/or at the end of the chain one or several heteroatoms or heterogroups chosen from amongst O, S, P, NR2, R2 representing H or a C1-C4 alkyl or acyl group, the said chain optionally being substituted by one or several identical or different radicals chosen from amongst -R, -OR, -SR, =0, -C(0)OR, -C(S)OR, -C(0)NR'RM, -C(S)NR'RR'\ -CN, -N02, -X, -MgX, -NR'R", -NR'C(0)R, -SiR and -SiOR, R. R' and R", identical or different, representing hydrogen or a linear or branched, saturated, or totally or partially unsaturated, hydrocarbon radical and having from 2 to 20 carbon atoms, it being understood that R' and R" can form a ring with the atom carrying them,<br>
1<br>
characterised in that:<br><br><br>
in which X and R1 are as defined above;<br>
or a salt or derivative thereof,<br>
or an enantiomeric mixture comprising a L-diamino acid of formula (II) and a<br>
corresponding   D-diamino   acid,   the   salts   or   derivatives   thereof  in   variable<br>
proportions, preferably in an aqueous medium,<br>
is made to react in the presence of an ornithine cyclodeaminase, or a polypeptide<br>
homologous to ornithine cyclodeaminase, the enzyme or the homologous polypeptide<br>
being obtained from a recombinant expression vector expressing the said enzyme or<br>
the said homologous polypeptide,<br>
b) the cyclic L-amino acid of formula (I) or a salt or derivative thereof is recovered in an enantiomeric excess of at least 80%.<br>
2.	Method as claimed in Claim 1, characterised in that the compound of formula<br>
(I) comprises a ring with six bonds, X representing a hydrocarbon chain with four<br>
bonds.<br>
3.	Method as claimed in Claim 1 or Claim 2, characterised in that the<br>
hydrocarbon chain X is a linear or branched alkylene chain.<br>
4.	Method as claimed in one of the preceding claims, characterised in that the<br>
compound of formula (II) or the enantiomeric mixture comprising an L-diamino acid<br>
of formula (II) and a corresponding D-amino acid in variable proportions is placed in<br>
the presence of an enzyme in the purified state.<br><br>
5.	Method as claimed in one of the preceding claims, characterised in that the compound of general formula (I) is in the form of an ammonium salt in aqueous solution.<br>
6.	Method as claimed in one of the preceding claims, characterised in that L-piperidine-2-carboxylic acid or one of its salts is prepared from L-lysine.<br>
7.	Method as claimed in one of Claims 1, 2, 4 or 5, characterised in that L-piperazine-2-carboxylic acid or one of its salts is prepared from L-azalysine.<br>
8.	Method as claimed in one of Claims 1, 2, 4 or 5, characterised in that L-thiomorpholine-2-carboxylic acid or one of its salts is prepared from L-thialysine.<br>
9.	Method as claimed in one of the preceding claims, characterised in that the enzyme is an ornithine cyclodeaminase of Agrohacterium or a peptide having an activity homologous to that of the said ornithine cyclodeaminase.<br>
10.	Method as claimed in one of the preceding claims, characterised in that the ornithine cyclodeaminase is that of the strain of Agrohacterium C 58.<br>
11.	Method as claimed in one of Claims 1 to 9, characterised in that the enzyme is recombinant enzyme expressed by the ocd gene of the strain of Agrohacterium tumefaciens C58 or pipA of Streptomyces pristinaespiralis or rapL of Streptomyces hygroscopicus or a homologous gene in a host microorganism.<br>
12.	Method as claimed in Claim 11, characterised in that the recombinant enzyme is expressed in E. coli.<br>
13.	Method as claimed in Claim 12, characterised in that the recombinant enzyme is obtained by expression of a modified gene of a native ornithine cyclodeaminase, in<br><br>
which some bases G or C are replaced by bases A or T. in such a way as to obtain modified codons coding for the same amino acid as the native codon.<br>
14.	Method as claimed in Claim 13. characterised in that the modified genes have<br>
less than 65%. advantageously less than 55% of bases G + C.<br>
15.	Method as claimed in one of Claims 12 to 14, characterised in that the<br>
modified gene is the gene pipA* of sequence SEQ ID No. 1 or the gene rapL* of the<br>
sequence SEQ ID No. 2, or the gene rapL** of the sequence SEQ ID No. 5.<br>
16.	Method as claimed in one of the preceding claims, characterised in that no<br>
exogenous NAD is added to the reaction medium.<br>
17.	Polynucleotide comprising the sequence SEQ ID No. 1.<br>
18.	Polynucleotide comprising the sequence SEQ ID No. 2.<br>
19.	Polynucleotide comprising the sequence SEQ ID No. 5.<br>
20.	Compound of formula (I) or salt or derivative of the compound of formula (I):<br><br><br>
*	R1 is chosen from amongst the hydrogen atom, a linear or branched alkyl radical<br>
having from 1 to 6 carbon atoms and a linear or branched acyl radical having from 1<br>
to 6 carbon atoms; and<br>
	X represents a saturated, or partially or totally unsaturated, linear or branched C1-<br>
C4, preferably C2-C4, hydrocarbon chain, optionally comprising in the chain and/or at<br>
the end of the chain one or several heteroatoms or heterogroups chosen from amongst<br>
O. S, P. NR2, R2 representing H or a C1-C4 alkyl or acyl group, the said chain also<br>
being optionally substituted by one or several identical or different radicals chosen<br>
from amongst -R, -OR, -SR, ==0, -C(0)OR, -C(S)OR. -C(0)NR'R", -C(S)NR'R'<br>
-CN, -N02, -X, -MgX, -NR'R", -NR'C(0)R, -SiR and -SiOR, R, R' and R' identical<br>
or different, representing hydrogen or a linear or branched, saturated, or totally or<br>
partially unsaturated, hydrocarbon radical having from 2 to 20 carbon atoms, it being<br>
understood that R1 and R" can form a ring with the atom carrying them,<br>
substantially obtained by the method as claimed in one of Claims 1 to 16.<br><br>
21.	Method of production of a cyclic L-amino acid substantially as<br>
herein above described and exemplified.<br>
22.	Polynucleotide   substantially   as   herein   above   described   and<br>
exemplified.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkyOS1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1929-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="204278-block-and-block-station-automation-in-single-line-bidirectional-section.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="204280-liquid-seal-pump-of-the-helical-screw-type.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>204279</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1929/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>40/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>05-Oct-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Feb-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>05-Dec-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>RHODIA CHIMIE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>26 quai Alphonse Le Gallo, F-92512 Boulogne Billancourt Cedex</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THIBAUT, Denis</td>
											<td>15 Place Souham, F-75013 Paris</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DÖRING, Volker</td>
											<td>31 rue Saint-Amand, F-75015 Paris</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARLIERE, Philippe</td>
											<td>2 Allée Saint-Martin, F-91450 Etiolles</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/88</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/FR02/01983</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-06-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>01/07559</td>
									<td>2001-06-08</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/204279-a-method-of-production-of-a-cyclic-l-amino-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:33:01 GMT -->
</html>
