<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/204763-a-method-for-producing-a-protein-library-and-selection-of-a-desired-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:45:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 204763:A METHOD FOR PRODUCING A PROTEIN LIBRARY AND SELECTION OF A DESIRED PROTEIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR PRODUCING A PROTEIN LIBRARY AND SELECTION OF A DESIRED PROTEIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for producing a protein library, comprising the steps of: a) providing a population of RNA molecules, each of which comprises a translation initiation sequence and a start codon operably linked to a protein coding sequence and each of which is operably linked to a peptide acceptor at the 3&#x27; end of said protein coding sequence; b) in vitro translating said protein coding sequences to produce a population of RNA-protein fusions; and c) further incubating said population of RNA-protein fusions under high salt conditions, thereby producing a protein library, which protein library comprises RNA-protein fusions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
[See Section 10 RULE 13]  <br><br><br>
 " A METHOD FOR PRODUCING A PROTEIN LIBRARY AND SELECTION OF A DESIRED PROTEIN"<br><br><br><br><br><br><br><br><br><br><br>
THE GENERAL HOSPITAL CORPORATION, of 55 Fruit Street, Boston, Massachusetts 02114, United States of America,<br>
The following specification particularly describes and ascertain   the nature of the invention and the manner in which it is to be performed:-<br><br>
Background of the Invention This application is a continuation-in-part of co-pending application, Szostak et al., U.S.S.N. 09/007,005, filed January 14, 1998, which claims benefit from provisional applications, Szostak et al., U.S.S.N. 60/064,491, filed November 6, 1997, now abandoned, and U.S.S.N. 60/035,963, filed January 21,199.7, now abandoned. This invention relates to protein selection methods. The invention was made with government support under grant F32 GM17776-01 and F32 GM17776-02. The government has certain rights in the<br>
invention.<br>
Methods currently exist_for the isoalion of RNA and DNA molecules based on their functions. For example, experiments of Ellington and Szostak(Nature 346:818 (1990); and Nature 355:850 (1992)) and Tuerk and Gold (Science 249:505 (1990); and J. Mol. Biol 222:739 (1991)) have demonstrated that very rare (i.e., less than 1 in 1013) nucleic acid molecules with desired properties may be isolated out of complex pools of molecules by repeated rounds of selection and amplification. These methods offer advantages over traditional genetic selections in that (i) very large candidate pools may be screened ( &gt; 10'5), (ii) host viability and in vivo conditions are not concerns, and (iii) selections may be carried out even if an in vivo genetic screen does not exist. The power of in vitro selection has been demonstrated in defining novel RNA and DNA sequences with very specific protein binding functions (see, for example, Tuerk and Gold, Science 249:505 (1990); Irvine et al., J. Mol. Biol 222:739 (1991); Oliphant et al., Mol. Cell Biol. 9:2944 (1989); Blackwell et al., Science 250:1104 (1990); Pollock and Treisman, Nuc. Acids Res. 18:6197 (1990); Thiesen and Bach, Nuc. Acids Res. 18:3203 (1990); Bartel et al., Cell 57:529 (1991); Stormo<br><br>
and Yoshioka, Proc. Natl. Acad. Sci. USA 88:5699 (1991); and Bock et al., Nature 355:564 (1992)), small molecule binding functions (Ellington and Szostak, Nature 346:818 (1990); Ellington and Szostak, Nature 355:850 (1992)), and catalytic functions (Green et al., Nature 347:406 (1990); Robertson and Joyce, Nature 344:467 (1990); Beaudry and Joyce, Science 257:635 (1992); Bartel and Szostak, Science 261:1411 (1993); Lorsch and Szostak, Nature 371:31-36 (1994); Cuenoud and Szostak, Nature 375:611-614 (1995); Chapman and Szostak, Chemistry and Biology 2:325-333 (1995); and Lohse and Szostak, Nature 381:442-444 (1996)). A similar scheme for the selection and amplification of proteins has not been demonstrated.<br>
Summary of the Invention          The purpose of the present invention is to allow the principles of in vitro selection and in vitro evolution to be applied to proteins. The invention facilitates the isolation of proteins with desired properties from large pools of partially or completely random amino acid sequences. In addition, the invention solves the problem of recovering and amplifying the protein sequence information by covalently attaching the mRNA coding sequence to the protein molecule.<br>
In general, the inventive method consists of an in vitro or in situ   transcription/ translation protocol that generates protein covalently linked to the 3' end of its own mRNA, i.e., an RNA-protein fusion. This is accomplished by synthesis and in vitro or in situ translation of an mRNA molecule with a peptide acceptor attached to its 3' end. One preferred peptide acceptor is puromycin, a nucleoside analog that adds to the C-terminus of a growing peptide chain and terminates translation. In one preferred design, a DNA sequence is included between the end of the message and the peptide acceptor which is designed to cause the ribosome to pause at the end of the open reading frame, providing additional time for the peptide acceptor (for example, puromycin) to accept the nascent peptide chain before hydrolysis of the peptidyl-tRNA linkage.<br>
If desired, the resulting RNA-protein fusion allows repeated rounds of<br><br>
selection and amplification because the protein sequence information may be recovered by reverse transcription and amplification (for example, by PCR amplification as well as any other amplification technique, including RNA-based amplification techniques such as 3SR or TSA). The amplified nucleic acid may then be transcribed, modified, and jn vitro or in situ translated to generate mRNA-protein fusions for the next round of selection. The ability to carry out multiple rounds of selection and amplification enables the enrichment and isolation of very rare molecules, e.g., one desired molecule out of a pool of 1015 members. This in turn allows the isolation of new or improved proteins which specifically recognize virtually any target or which catalyze desired chemical reactions.<br>
Accordingly, in a first aspect, the invention features a method for selection of a desired protein, involving the steps of: (a) providing a population of candidate RNA molecules, each of which includes a translation initiation sequence and a start codon operably linked to a candidate protein coding sequence and each of which is operably linked to a peptide acceptor at the 3' end of the candidate protein coding sequence; (b) in vitro or iri situ translating the candidate protein coding sequences to produce a population of candidate RNA-protein fusions; and (c) selecting a desired RNA-protein fusion, thereby selecting the desired protein.<br>
In a related aspect, the invention features a method for selection of a DNA molecule which encodes a desired protein, involving the steps of: (a) providing a population of candidate RNA molecules, each of which includes a translation initiation sequence and a start codon operably linked to a candidate protein coding sequence and each of which is operably linked to a peptide acceptor at the 3' end of the candidate protein coding sequence; (b) in vitro or in situ translating the candidate protein coding sequences to produce a population of candidate RNA-protein fusions; (c) selecting a desired RNA-protein fusion; and (d) generating from the RNA portion of the fusion a DNA molecule which encodes the desired protein.<br>
In another related aspect, the invention features a method for selection of a<br><br>
protein having an altered function relative to a reference protein, involving the steps of: (a) producing a population of candidate RNA molecules from a population of DNA templates, the candidate DNA templates each having a candidate protein coding sequence which differs from the reference protein coding sequence, the RNA molecules each comprising a translation initiation sequence and a start codon operably linked to the candidate protein coding sequence and each being operably linked to a peptide acceptor at the 3' end; (b) in vitro or in situ translating the candidate protein coding sequences to produce a population of candidate RNA-protein fusions; and (c) selecting an RNA-protein fusion having an altered function, thereby selecting the protein having the altered function.<br>
In yet another related aspect, the invention features a method for selection of a DNA molecule which encodes a protein having an altered function relative to a reference protein, involving the steps of: (a) producing a population of candidate RNA molecules from a population of candidate DNA templates, the candidate DNA templates each having a candidate protein coding sequence which differs from the reference protein coding sequence, the RNA molecules each comprising a translation initiation sequence and a start codon operably linked to the candidate protein coding sequence and each being operably linked to a peptide acceptor at the 3' end; (b) in vitro or in situ, translating the candidate protein coding sequences to produce a population of RNA-protein fusions; (c) selecting an RNA-protein fusion having an altered function; and (d) generating from the RNA portion of the fusion a DNA molecule which encodes the protein having the altered function.<br>
In yet another related aspect, the invention features a method for selection of a desired RNA, involving the steps of: (a) providing a population of candidate RNA molecules, each of which includes a translation initiation sequence and a start codon operably linked to a candidate protein coding sequence and each of which is operably linked to a peptide acceptor at the 3' end of the candidate protein coding sequence; (b) in vitro or in situ translating the candidate protein coding sequences to produce a population of candidate RNA-protein fusions; and (c) selecting a desired<br><br>
RNA-protein fusion, thereby selecting the desired RNA.<br>
In preferred embodiments of the above methods, the peptide acceptor is puromycin; each of the candidate RNA molecules further includes a pause sequence or further includes a DNA or DNA analog sequence covalently bonded to the 3' end of the RNA; the population of candidate RNA molecules includes at least 109, preferably, at least 101°, more preferably, at least 1011 1012, or 1013, and, most preferably, at least 1014 different RNA molecules; the in vitro translation reaction is carried out in a lysate prepared from a eukaryotic cell or portion thereof (and is, for example, carried out in a reticulocyte lysate or wheat germ lysate); the in vitro translation reaction is carried out in an extract prepared from a prokaryotic cell (for example. E. coli) or portion thereof; the selection step involves binding of the desired protein to an immobilized binding partner; the selection step involves assaying for a functional activity of the desired protein; the DNA molecule is amplified; the method further involves repeating the steps of the above selection methods; the method further involves transcribing an RNA molecule from the DNA molecule and repeating steps (a) through (d); following the in vitro translating step, the method further involves an incubation step carried out in the presence of 50-100 mM Mg2+; and the RNA-protein fusion further includes a nucleic acid or nucleic acid analog sequence positioned proximal to the peptide acceptor which increases flexibility.<br>
In other related aspects, the invention features an RNA-protein fusion selected by any of the methods of the invention; a ribonucleic acid covalently bonded though an amide bond to an amino acid sequence, the amino acid sequence being encoded by the ribonucleic acid; and a ribonucleic acid which includes a translation initiation sequence and a start codon operably linked to a candidate protein coding sequence, the ribonucleic acid being operably linked to a peptide acceptor (for example, puromycin) at the 3' end of the candidate protein coding sequence.<br>
In a second aspect, the invention features a method for selection of a desired protein or desired RNA through enrichment of a sequence pool. This method involves the steps of: (a) providing a population of candidate RNA molecules, each<br><br>
of which includes a translation initiation sequence and a start codon operably linked to a candidate protein coding sequence and each of which is operably linked to a peptide acceptor at the 3' end of the candidate protein coding sequence; (b) in vitro or in situ translating the candidate protein coding sequences to produce a population of candidate RNA-protein fusions; (c) contacting the population of RNA-protein fusions with a binding partner specific for either the RNA portion or the protein portion of the RNA-protein fusion under conditions which substantially separate the binding partner-RNA-protein fusion complexes from unbound members of the population; (d) releasing the bound RNA-protein fusions from the complexes; and (e) contacting the population of RNA-protein fusions from step (d) with a binding partner specific for the protein portion of the desired RNA-protein fusion under conditions which substantially separate the binding partner-RNA-protein fusion complex from unbound members of said population, thereby selecting the desired protein and the desired RNA.<br>
In preferred embodiments, the method further involves repeating steps (a) through (e). In addition, for these repeated steps, the same or different binding partners may be used, in any order, for selective enrichment of the desired RNA-protein fusion. In another preferred embodiment, step (d) involves the use of a binding partner (for example, a monoclonal antibody) specific for the protein portion of the desired fusion. This step is preferably carried out following reverse transcription of the RNA portion of the fusion to generate a DNA which encodes the desired protein. If desired, this DNA may be isolated and/or PCR amplified. This enrichment technique may be used to select a desired protein or may be used to select a protein having an altered function relative to a reference protein.<br>
In other preferred embodiments of the enrichment methods, the peptide acceptor is puromycin; each of the candidate RNA molecules further includes a pause sequence or further includes a DNA or DNA analog sequence covalently bonded to the 3' end of the RNA; the population of candidate RNA molecules includes at least 109, preferably, at least 1010, more preferably, at least 1011", 1012, or 1013, and, most<br><br>
preferably, at least 1U" ditterent KMA molecules; the m vitro translation reaction is carried out in a lysate prepared from a eukaryotic cell or portion thereof (and is, for example, carried out in a reticulocyte lysate or wheat germ lysate); the in vitro translation reaction is carried out in an extract prepared from a prokaryotic cell or portion thereof (for example, E. coli): the DNA molecule is amplified; at least one of the binding partners is immobilized on a solid support; following the in vitro translating step, the method further involves an incubation step carried out in the presence of 50-100 mM Mg2+; and the RNA-protein fusion further includes a nucleic acid or nucleic acid analog sequence positioned proximal to the peptide acceptor which increases flexibility.<br>
In a related aspect, the invention features methods for producing libraries (for example, protein, DNA, or RNA-fusion libraries) or methods for selecting desired molecules (for example, protein, DNA, or RNA molecules or molecules having a particular function or altered function) which involve a step of post-translational incubation in the presence of high salt (including, without limitation, high salt which includes a monovalent cation, such as K+ NH4 or Na+, a divalent cation, such as Mg+2, or a combination thereof). This incubation may be carried out at approximately room temperature or approximately -20°C and preferred salt concentrations of between approximately 125 mM -1.5 M (more preferably, between approximately       300 mM - 600 mM) for monovalent cations and between approximately 25 mM - 200 mM for divalent cations.<br>
In another related aspect, the invention features kits for carrying out any of the selection methods described herein.<br>
In a third and final aspect, the invention features a microchip that includes       an array of immobilized single-stranded nucleic acids, the nucleic acids being<br>
hybridized to RNA-protein fusions. Preferably, the protein component of the RNA-protein fusion is encoded by the RNA.<br>
As used herein, by a "population" is meant more than one molecule (for example, more than one RNA, DNA, or RNA-protein fusion molecule). Because the<br><br>
methods of the invention facilitate selections which begin, if desired, with large numbers of candidate molecules, a "population" according to the invention preferably means more than 109 molecules, more preferably, more than 10", 1012, or 10'3 molecules, and, most preferably, more than 1013 molecules.<br>
By "selecting" is meant substantially partitioning a molecule from other molecules in a population. As used herein, a "selecting" step provides at least a 2-fold, preferably, a 30-fold, more preferably, a 100-fold, and, most preferably, a 1000-fold enrichment of a desired molecule relative to undesired molecules in a population following the selection step. As indicated herein, a selection step may be repeated any number of times, and different types of selection steps may be combined in a given approach.<br>
By a "protein" is meant any two or more naturally occurring or modified amino acids joined by one or more peptide bonds. "Protein" and "peptide" are used interchangeably herein.<br>
By "RNA" is meant a sequence of two or more covalently bonded, naturally occurring or modified ribonucleotides. One example of a modified RNA included within this term is phosphorothioate RNA.<br>
By a "translation initiation sequence" is meant any sequence which is capable of providing a functional ribosome entry site. In bacterial systems, this region is sometimes referred to as a Shine-Dalgarno sequence.<br>
By a "start codon" is meant three bases which signal the beginning of a protein coding sequence. Generally, these bases are AUG (or ATG); however, any other base triplet capable of being utilized in this manner may be substituted.<br>
By "covalently bonded" to a peptide acceptor is meant that the peptide acceptor is joined to a "protein coding sequence" either directly through a covalent bond or indirectly through another covalently bonded sequence (for example, DNA corresponding to a pause site).<br>
By a "peptide acceptor" is meant any molecule capable of being added to the C-terminus of a growing protein chain by the catalytic activity of the ribosomal<br><br>
peptidyl transferase function. Typically, such molecules contain (i) a nucleotide or nucleotide-like moiety (for example, adenosine or an adenosine analog (di-methylation at the N-6 amino position is acceptable)), (ii) an amino acid or amino acid-like moiety (for example, any of the 20 D- or L-amino acids or any amino acid analog thereof (for example, O-methyl tyrosine or any of the analogs described by Ellman et al., Meth. Enzymol. 202:301,1991), and (iii) a linkage between the two (for example, an ester, amide, or ketone linkage at the 3' position or, less preferably, the 2' position); preferably, this linkage does not significantly perturb the pucker of the ring from the natural ribonucleotide conformation. Peptide acceptors may also possess a nucleophile, which may be, without limitation, an amino group, a hydroxyl group, or a sulfhydryl group. In addition, peptide acceptors may be composed of nucleotide mimetics, amino acid mimetics, or mimetics of the combined nucleotide-amino acid structure.<br>
By a peptide acceptor being positioned "at the 3' end" of a protein coding sequence is meant that the peptide acceptor molecule is positioned after the final codon of that protein coding sequence. This term includes, without limitation, a peptide acceptor molecule that is positioned precisely at the 3' end of the protein coding sequence as well as one which is separated from the final codon by intervening coding or non-coding sequence (for example, a sequence corresponding to a pause site). This term also includes constructs in which coding or non-coding sequences follow (that is, are 3' to) the peptide acceptor molecule. In addition, this term encompasses, without limitation, a peptide acceptor molecule that is covalently bonded (either directly or indirectly through intervening nucleic acid sequence) to the protein coding sequence, as well as one that is joined to the protein coding sequence by some non-covalent means, for example, through hybridization using a second nucleic acid sequence that binds at or near the 3' end of the protein coding sequence and that itself is bound to a peptide acceptor molecule.<br>
By an "altered function" is meant any qualitative or quantitative change in the function of a molecule.<br><br>
By a "pause sequence  is meant a nucleic acid sequence which causes a ribosome to slow or stop its rate of translation.<br>
By "binding partner," as used herein, is meant any molecule which has a specific, covalent or non-covalent affinity for a portion of a desired RNA-protein fusion. Examples of binding partners include, without limitation, members of antigen/antibody pairs, protein/inhibitor pairs, receptor/ligand pairs (for example cell surface receptor/ligand pairs, such as hormone receptor/peptide hormone pairs), enzyme/substrate pairs (for example, kinase/substrate pairs), lectin/carbohydrate pairs, oligomeric or heterooligomeric protein aggregates, DNA binding protein/DNA binding site pairs, RNA/protein pairs, and nucleic acid duplexes, heteroduplexes, or ligated strands, as well as any molecule which is capable of forming one or more covalent or non-covalent bonds (for example, disulfide bonds) with any portion of an RNA-protein fusion. Binding partners include, without limitation, any of the "selection motifs" presented in Figure 2.<br>
By a "solid support" is meant, without limitation, any column (or column material), bead, test tube, microtiter dish, solid particle (for example, agarose or sepharose), microchip (for example, silicon, silicon-glass, or gold chip), or membrane (for example, the membrane of a liposome or vesicle) to which an affinity complex may be bound, either directly or indirectly (for example, through other binding partner intermediates such as other antibodies or Protein A), or in which an affinity complex may be embedded (for example, through a receptor or channel).<br>
By "high salt" is meant having a concentration of a monovalent cation of at least 200 mM, and, preferably, at least 500 mM or even 1 M, and/or a concentration of a divalent or higher valence cation of at least 25 mM, preferably, at least 50 mM, and, most preferably, at least 100 mM.<br>
The presently claimed invention provides a number of significant advantages. To begin with, it is the first example of this type of scheme for the selection and amplification of proteins. This technique overcomes the impasse<br><br>
created by the need to recover nucleotide sequences corresponding to desired, isolated proteins (since only nucleic acids can be replicated). In particular, many prior methods that allowed the isolation of proteins from partially or fully randomized pools did so through an in vivo step. Methods of this sort include monoclonal antibody technology (Milstein, Sci. Amer. 243:66 (1980); and Schultz et al., J. Chem. Engng. News 68:26 (1990)), phage display (Smith, Science 228:1315 (1985); Parmley and Smith, Gene 73:305 (1988); and McCafferty et al., Nature 348:552 (1990)), peptide-lac repressor fusions (Cull et al., Proc. Natl. Acad. Sci. USA 89:1865 (1992)), and classical genetic selections. Unlike the present technique, each of these methods relies on a topological link between the protein and the nucleic acid so that the information of the protein is retained and can be recovered in readable, nucleic acid form.<br>
In addition, the present invention provides advantages over the stalled translation method (Tuerk and Gold, Science 249:505 (1990); Irvine et al., J. Mol. Biol 222:739 (1991); Korman et al., Proc. Natl. Acad. Sci. USA 79:1844-1848 (1982); Mattheakis et al., Proc. Natl. Acad. Sci. USA 91:9022-9026 (1994); Mattheakis et al., Meth. Enzymol. 267:195 (1996); and Hanes and Pluckthun, Proc. Natl. Acad. Sci. USA 94:4937 (1997)), a technique in which selection is for some property of a nascent protein chain that is still complexed with the ribosome and its mRNA. Unlike the stalled translation technique, the present method does not rely on maintaining the integrity of an mRNA: ribosome: nascent chain ternary complex, a complex that is very fragile and is therefore limiting with respect to the types of selections which are technically feasible.<br>
The present method also provides advantages over the branched synthesis approach proposed by Brenner and Lemer (Proc. Natl. Acad. Sci. USA 89:5381-5383 (1992)), in which DNA-peptide fusions are generated, and genetic information is theoretically recovered following one round of selection. Unlike the branched synthesis approach, the present method does not require the regeneration of a peptide from the DNA portion of a fusion (which, in the branched synthesis approach, is<br><br>
generally accomplished by individual rounds of chemical synthesis). Accordingly, the present method allows for repeated rounds of selection using populations of candidate molecules. In addition, unlike the branched synthesis technique, which is generally limited to the selection of fairly short sequences, the present method is applicable to the selection of protein molecules of considerable length.<br>
In yet another advantage, the present selection and directed evolution technique can make use of very large and complex libraries of candidate sequences. In contrast, existing protein selection methods which rely on an in vivo step are typically limited to relatively small libraries of somewhat limited complexity. This advantage is particularly important when selecting functional protein sequences considering, for example, that 1013 possible sequences exist for a peptide of only 10 amino acids in length. In classical genetic techniques, lac repressor fusion approaches, and phage display methods, maximum complexities generally fall orders of magnitude below 1013 members. Large library size also provides an advantage for directed evolution applications, in that sequence space can be explored to a greater depth around any given starting sequence.<br>
The present technique also differs from prior approaches in that the selection step is context-independent. In many other selection schemes, the context in which, for example, an expressed protein is present can profoundly influence the nature of the library generated. For example, an expressed protein may not be properly expressed in a particular system or may not be properly displayed (for example, on the surface of a phage particle). Alternatively, the expression of a protein may actually interfere with one or more critical steps in a selection cycle, e.g., phage viability or infectivity, or lac repressor binding. These problems can result in the loss of functional molecules or in limitations on the nature of the selection procedures that may be applied.<br>
Finally, the present method is advantageous because it provides control over the repertoire of proteins that may be tested. In certain techniques (for example, antibody selection), there exists little or no control over the nature of the starting pool.<br><br>
In yet other techniques (for example, lac fusions and phage display), the candidate pool must be expressed in the context of a fusion protein. In contrast, RNA-protein fusion constructs provide control over the nature of the candidate pools available for screening. In addition, the candidate pool size has the potential to be as high as RNA or DNA pools (~ 1015 members), limited only by the size of the in vitro translation reaction performed. And the makeup of the candidate pool depends completely on experimental design; random regions may be screened in isolation or within the context of a desired fusion protein, and most if not all possible sequences may be expressed in candidate pools of RNA-protein fusions.<br>
Other features and advantages of the invention will be apparent from the following detailed description, and from the claims.<br>
Detailed Description The drawings will first briefly be described.<br>
Brief Description of the Drawings<br>
FIGURES 1A-1C are schematic representations of steps involved in the production of RNA-protein fusions. Figure 1A illustrates a sample DNA construct for generation of an RNA portion of a fusion. Figure IB illustrates the generation of an RNA/puromycin conjugate. And Figure 1C illustrates the generation of an RNA-protein fusion.<br>
FIGURE 2 is a schematic representation of a generalized selection protocol according to the invention.<br>
FIGURE 3 is a schematic representation of a synthesis protocol for minimal translation templates containing 3' puromycin. Step (A) shows the addition of protective groups to the reactive functional groups on puromycin (5'-OH and NH2); as modified, these groups are suitably protected for use in phosphoramidite based oligonucleotide synthesis. The protected puromycin was attached to aminohexyl controlled pore glass (CPG) through the 2'OH group using the standard protocol for<br><br>
attachment of DNA through its 3'OH (Gait, Oligonucleotide Synthesis, A Practical Approach, The Practical Approach Series (IRL Press, Oxford, 1984)). In step (B), a minimal translation template (termed "43-P"), which contained 43 nucleotides, was synthesized using standard RNA and DNA chemistry (MilHpore, Bedford, MA), deprotected using NH4OH and TBAF, and gel purified. The template contained 13 bases of RNA at the 5' end followed by 29 bases of DNA attached to the 3' puromycin at its 5' OH. The RNA sequence contained (i) a Shine-Dalgarno consensus sequence complementary to five bases of 16S rRNA (Stormo et al., Nucleic Acids Research 10:2971-2996 (1982); Shine and Dalgamo, Proc. Natl. Acad. Sci. USA 71:1342-1346 (1974); and Steitz and Jakes, Proc. Natl. Acad. Sci. USA 72:4734-4738 (1975)), (ii) a five base spacer, and (iii) a single AUG start codon. The DNA sequence was dA27dCdCP, where "P" is puromycin.<br>
FIGURE 4 is a schematic representation of a preferred method for the preparation of protected CPG-linked puromycin.<br>
FIGURE 5 is a schematic representation showing possible modes of methionine incorporation into a template of the invention. As shown in reaction (A), the template binds the ribosome, allowing formation of the 70S initiation complex. Fmet tRNA binds to the P site and is base paired to the template. The puromycin at the 3' end of the template enters the A site in an intramolecular fashion and forms an amide linkage to N-formyl methionine via the peptidyl transferase center, thereby deacylating the tRNA. Phenol/chloroform extraction of the reaction yields the template with methionine covalently attached. Shown in reaction (B) is an undesired intermolecular reaction of the template with puromycin containing oligonucleotides. As before, the minimal template stimulates formation of the 70S ribosome containing fmet tRNA bound to the P site. This is followed by entry of a second template in trans to give a covalently attached methionine.<br>
FIGURES 6A-6H are photographs showing the incorporation of 35S methionine (35S met) into translation templates. Figure 6A demonstrates magnesium (Mg2') dependence of the reaction. Figure 6B demonstrates base stability of the<br><br>
product; the change in mobility shown in this figure corresponds to a loss of the 5' RNA sequence of 43-P (also termed "Met template") to produce the DNA-puromycin portion, termed 30-P. The retention of the label following base treatment was consistent with the formation of a peptide bond between 35S methionine and the 3' puromycin of the template. Figure 6C demonstrates the inhibition of product formation in the presence of peptidyl transferase inhibitors. Figure 6D demonstrates the dependence of 35S methionine incorporation on a template coding sequence. Figure 6E demonstrates DNA template length dependence of 35S methionine incorporation. Figure 6F illustrates cis versus trans product formation using templates 43-P and 25-P. Figure 6G illustrates cis versus trans product formation using templates 43-P and 13-P. Figure 6H illustrates cis versus trans product formation using templates 43-P and 30-P in a reticulocyte lysate system.<br>
FIGURES 7A-7C are schematic illustrations of constructs for testing peptide fusion formation and selection. Figure 7A shows LP77 ("ligated-product," "77" nucleotides long) (also termed, "short myc template") (SEQ ID NO: 1). This sequence contains the c-myc monoclonal antibody epitope tag EQKLISEEDL (SEQ ID NO: 2) (Evan et al, Mol. Cell Biol. 5:3610-3616 (1985)) flanked by a 5' start codon and a 3' linker. The 5' region contains a bacterial Shine-Dalgarno sequence identical to that of 43-P. The coding sequence was optimized for translation in bacterial systems. In particular, the 5' UTRs of 43-P and LP77 contained a Shine-Dalgamo sequence complementary to five bases of 16S rRNA (Steitz and Jakes, Proc. Natl. Acad. Sci. USA 72:4734-4738 (1975)) and spaced similarly to ribosomal protein sequences (Stormo et al, Nucleic Acids Res. 10:2971-2996 (1982)). Figure 7B shows LP154 (ligated product, 154 nucleotides long) (also termed "long myc template") (SEQ ID NO: 3). This sequence contains the code for generation of the peptide used to isolate the c-myc antibody. The 5' end contains a truncated version of the TMV upstream sequence (designated "TE). This 5' UTR contained a 22 nucleotide sequence derived from the TMV 5' UTR encompassing two ACAAAUUAC direct repeats (Gallie et al., Nucl. Acids Res. 16:883 (1988)). Figure<br><br>
1C shows Pool #1 (SEQ ID NO: 4), an exemplary sequence to be used for peptide selection. The final seven amino acids from the original myc peptide were included in the template to serve as the 3' constant region required for PCR amplification of the template. This sequence is known not to be part of the antibody binding epitope.<br>
FIGURE 8 is a photograph demonstrating the synthesis of RNA-protein fusions using templates 43-P, LP77, and LP 154, and reticulocyte ("Retic") and wheat germ ("Wheat") translation systems. The left half of the figure illustrates 35S methionine incorporation in each of the three templates. The right half of the figure illustrates the resulting products after RNase A treatment of each of the three templates to remove the RNA coding region; shown are 35S methionine-labeled DNA-protein fusions. The DNA portion of each was identical to the oligo 30-P. Thus, differences in mobility were proportional to the length of the coding regions, consistent with the existence of proteins of different length in each case.<br>
FIGURE 9 is a photograph demonstrating protease sensitivity of an RNA-protein fusion synthesized from LP 154 and analyzed by denaturing polyacrylamide gel electrophoresis. Lane 1 contains 32P labeled 30-P. Lanes 2-4, 5-7, and 8-10 contain the 35S labeled translation templates recovered from reticulocyte lysate reactions either without treatment, with RNase A treatment, or with RNase A and proteinase K treatment, respectively.<br>
FIGURE 10 is a photograph showing the results of immunoprecipitation reactions using in vitro translated 33 amino acid myc-epitope protein. Lanes 1 and 2 show the translation products of the myc epitope protein and β-globin templates, respectively. Lanes 3-5 show the results of immunoprecipitation of the myc-epitope peptide using a c-myc monoclonal antibody and PBS, DB, and PBSTDS wash buffers, respectively. Lanes 6-8 show the same immunoprecipitation reactions, but using the p-globin translation product.<br>
FIGURE 11 is a photograph demonstrating immunoprecipitation of an RNA-protein fusion from an in vitro translation reaction. The picomoles of template used in the reaction are indicated. Lanes 1-4 show RNA124 (the RNA portion of<br><br>
fusion LP154), and lanes 5-7 show RNA-protein fusion LP154. After immunoprecipitation using a c-myc monoclonal antibody and protein G sepharose, the samples were treated with RNase A and T4 polynucleotide kinase, then loaded on a denaturing urea polyacrylamide gel to visualize the fusion. In ianes 1-4, with samples containing either no template or only the RNA portion of the long myc template (RNA124), no fusion was seen. In lanes 5-7, bands corresponding to the fusion were clearly visualized. The position of 32P labeled 30-P is indicated, and the amount of input template is indicated at the top of the figure.<br>
FIGURE 12 is a graph showing a quantitation of fusion material obtained from an in vitro translation reaction. The intensity of the fusion bands shown in lanes 5-7 of Figure 11 and the 30-P band (isolated in a parallel fashion on dT25, not shown) were quantitated on phosphorimager plates and plotted as a function of input LP 154 concentration. Recovered modified 30-P (left y axis) was linearly proportional to input template (x axis), whereas linker-peptide fusion (right y axis) was constant. From this analysis, it was calculated that ~I012 fusions were formed per ml of translation reaction sample.<br>
FIGURE 13 is a schematic representation of thiopropyl sepharose and dT25 agarose, and the ability of these substrates to interact with the RNA-protein fusions of the invention.<br>
FIGURE 14 is a photograph showing the results of sequential isolation of fusions of the invention. Lane 1 contains 32P labeled 30-P. Lanes 2 and 3 show LP 154 isolated from translation reactions and treated with RNase A. In lane 2, LP 154 was isolated sequentially, using thiopropyl sepharose followed by dT2S agarose. Lane 3 shows isolation using only dT25 agarose. The results indicated that the product contained a free thiol, likely the penultimate cysteine in the myc epitope coding sequence.<br>
FIGURES 15A and 15B are photographs showing the formation of fusion products using β-globin templates as assayed by SDS-tricine-PAGE (polyacrylamide gel electrophoresis). Figure 15A shows incorporation of 35S using either no template<br><br>
(lane 1), a syn-p-globin template (lanes 2-4), or an LP-P-globin template (lanes 5-7). Figure 15B (lanes labeled as in Fig. 15A) shows 35S-labeled material isolated by oligonucleotide affinity chromatography. No material was isolated in the absence of a 30-P tail (lanes 2-4).<br>
FIGURES 16A-16C are diagrams and photographs illustrating enrichment of myc dsDNA versus pool dsDNA by in vitro selection. Figure 16A is a schematic of the selection protocol. Four mixtures of the myc and pool templates were translated in vitro and isolated on dT25 agarose followed by TP sepharose to purify the template fusions from unmodified templates. The mRNA-peptide fusions were then reverse transcribed to suppress any secondary or tertiary structure present in the templates. Aliquots of each mixture were removed both before (Figure 16B) and after (Figure 16C) affinity selection, amplified by PCR in the presence of a labeled primer, and digested with a restriction enzyme that cleaved only the myc DNA. The input mixtures of templates were pure myc (lane 1), or a 1:20, 1:200, or 1:2000 myc:pool (lanes 2-4).   The unselected material deviated from the input ratios due to preferential translation and reverse transcription of the myc template. The enrichment of the myc template during the selective step was calculated from the change in the poolrmyc ratio before and after selection.<br>
FIGURE 17 is a photograph illustrating the translation of myc RNA templates. The following linkers were used: lanes 1-4, dA27dCdCP; lanes 5-8, dA27rCrCP; and lanes 9-12, dA21C9C9C9dAdCdCP. In each lane, the concentration of RNA template was 600 nM, and "S-Met was used for labeling. Reaction conditions were as follows: lanes 1,5, and 9,30°C for 1 hour; lanes 2, 6, and 10, 30°C for 2 hours; lane 3, 7, and 11, 30°C for 1 hour, -20oC for 16 hours; and lanes 4, 8, and 12, 30°C for 1 hour, -20°C for 16 hours with 50 mM Mg2+. In this Figure, "A" represents free peptide, and "B" represent mRNA-peptide fusion.<br>
FIGURE 18 is a photograph illustrating the translation of myc RNA templates labeled with 32P. The linker utilized was dA2lC9,C9C,dAdCdCP. Translation was performed at 30°C for 90 minutes, and incubations were carried out<br><br>
at -20 °C for 2 days without additional Mg2+. The concentrations of mRNA templates were 400 nM (lane 3), 200 nM (lane 4), 100 nM (lane 5), and 100 nM (lane 6). Lane 1 shows mRNA-peptide fusion labeled with 35S-Met. Lane 2 shows mRNA labeled with 32P. In lane 6, the reaction was carried out in the presence of 0.5 mM cap analog.<br>
FIGURE 19 is a photograph illustrating the translation of myc RNA template using lysate obtained from Ambion (lane 1), Novagen (lane 2), and Amersham (lane 3). The linker utilized was dA27dCdCP. The concentration of the template was 600 nM, and 35S-Met was used for labeling. Translations were performed at 30°C for 1 hour, and incubations were carried out at -20°C overnight in the presence of 50 mM Mg2+<br>
FIGURE 20 is a graph illustrating enrichment of RNA-peptide fusions bound by anti-myc monoclonal antibody 9E10 during six rounds of in vitro selection.<br>
FIGURE 21 is a graph showing competition assays with synthetic myc peptides.<br>
FIGURE 22 is a schematic representation illustrating the amino acid sequences of 12 selected peptides from a random 27-mer library.<br>
FIGURE 23 is a photograph illustrating the effect of linker length on fusion formation. In this figure, Myc templates containing linkers [N] = 13,19,25, 30, 35, 40,45, or 50 nucleotides long (dA1(M7dCdCP) were assayed for fusion formation by SDS-PAGE. The flexible linker F (dA21[C9]3dAdCdCP) is also shown. Translations were performed with 600 nM template at 30°C for 90 minutes, followed by addition of 50 mM Mg+2 and incubation at -20 °C for two days.<br>
FIGURE 24 is a photograph illustrating co-translation of myc and XPPase mRNA. In this figure, 200 nM of A.PPase RNA (RNA716) and/or 50 nM myc RNA (RNA152) containing the flexible linker F (dA2l[C9]3dAdCdCP) were translated with [35S]-Met. Mg+2 (75 mM) was added, followed by incubation at -20°C. No bands were observed from cross-products (myc templates fusion to XPPase protein).<br>
Described herein is a general method for the selection of proteins with<br><br>
desired functions using fusions in which these proteins are covalently linked to their own messenger RN As. These RNA-protein fusions are synthesized by in vitro or in situ translation of mRNA pools containing a peptide acceptor attached to their 3' ends (Figure 1B). In one preferred embodiment, after readthrough of the open reading frame of the message, the ribosome pauses when it reaches the designed pause site, and the acceptor moiety occupies the ribosomal A site and accepts the nascent peptide chain from the peptidyl-tRNA in the P site to generate the RNA-protein fusion (Figure 1C). The covalent link between the protein and the RNA (in the form of an amide bond between the 3' end of the mRNA and the C-terminus of the protein which it encodes) allows the genetic information in the protein to be recovered and amplified (e.g., by PCR) following selection by reverse transcription of the RNA. Once the fusion is generated, selection or enrichment is carried out based on the properties of the mRNA-protein fusion, or, alternatively, reverse transcription may be carried out using the mRNA template while it is attached to the protein to avoid any effect of the single-stranded RNA on the selection. When the mRNA-protein construct is used, selected fusions may be tested to determine which moiety (the protein, the RNA, or both) provides the desired function.<br>
In one preferred embodiment, puromycin (which resembles tyrosyl adenosine) acts as the acceptor to attach the growing peptide to its mRNA. Puromycin is an antibiotic that acts by terminating peptide elongation. As a mimetic of aminoacyl-tRNA, it acts as a universal inhibitor of protein synthesis by binding the A site, accepting the growing peptide chain, and falling off the ribosome (at a Kd = 104 M) (Traut and Monro, J. Mol. Biol. 10:63 (1964); Smith et al., J. Mol. Biol. 13:617 (1965)). One of the most attractive features of puromycin is the fact that it forms a stable amide bond to the growing peptide chain, thus allowing for more stable fusions than potential acceptors that form unstable ester linkages. In particular, the peptidyl-puromycin molecule contains a stable amide linkage between the peptide and the O-methyl tyrosine portion of the puromycin. The O-methyl tyrosine is in turn linked by a stable amide bond to the 3'-amino group of the modified adenosine<br><br>
portion of puromycin.<br>
Other possible choices for acceptors include tRNA-like structures at the 3' end of the mRNA, as well as other compounds that act in a manner similar to  juromycin. Such compounds include, without limitation, any compound which possesses an amino acid linked to an adenine or an adenine-like compound, such as he amino acid nucleotides, phenylalanyl-adenosine (A-Phe), tyrosyl adenosine (A-Fyr), and alanyl adenosine (A-Ala), as well as amide-linked structures, such as ohenylalanyl 3' deoxy 3' amino adenosine, alanyl 3' deoxy 3' amino adenosine, and yrosyl 3' deoxy 3' amino adenosine; in any of these compounds, any of the naturally-occurring L-amino acids or their analogs may be utilized. In addition, a combined RNA-like 3' structure-puromycin conjugate may also be used in the invention.<br>
Shown in Figure 2 is a preferred selection scheme according to the nvention. The steps involved in this selection are generally carried out as follows.<br>
Step 1.   Preparation of the DNA template. As a step toward generating :he RNA-protein fusions of the invention, the RNA portion of the fusion is synthesized. This may be accomplished by direct chemical RNA synthesis or, more ;ommonly, is accomplished by transcribing an appropriate double-stranded DNA template.<br>
Such DNA templates may be created by any standard technique (including any technique of recombinant DNA technology, chemical synthesis, or both). In principle, any method that allows production of one or more templates containing a known, random, randomized, or mutagenized sequence may be used for this purpose. In one particular approach, an oligonucleotide (for example, containing random bases) is synthesized and is amplified (for example, by PCR) prior to transcription. Chemical synthesis may also be used to produce a random cassette which is then inserted into the middle of a known protein coding sequence (see, for example, chapter 8.2, Ausubel et al., Current Protocols in Molecular Biology, John Wiley &amp; Sons and Greene Publishing Company, 1994). This latter approach produces a high density of mutations around a specific site of interest in the protein.<br><br>
An alternative to total randomization of a DNA template sequence is partial randomization, and a pool synthesized in this way is generally referred to as a "doped" pool. An example of this technique, performed on an RNA sequence, is described, for example, by Ekland et al. (Nucl. Acids Research 23:3231 (1995)). Partial randomization may be performed chemically by biasing the synthesis reactions such that each base addition reaction mixture contains an excess of one base and small amounts of each of the others; by careful control of the base concentrations, a desired mutation frequency may be achieved by this approach. Partially randomized pools may also be generated using error prone PCR techniques, for example, as described in Beaudry and Joyce (Science 257:635 (1992)) and Bartel and Szostak (Science 261:1411 (1993)).<br>
Numerous methods are also available for generating a DNA construct beginning with a known sequence and then creating a mutagenized DNA pool. Examples of such techniques are described in Ausubel et al. (supra, chapter 8); Sambrook et al. (Molecular Cloning: A Laboratory Manual, chapter 15, Cold Spring Harbor Press, New York, 2nd ed. (1989); Cadwell et al. (PCR Methods and Applications 2:28 (1992)); Tsang et al. (Meth. Enzymol. 267:410 (1996)); Reidhaar-Olsen et al. (Meth. Enzymol. 208:564 (1991)); and Ekland and Bartel (Nucl. Acids. Res. 23:3231 (1995)). Random sequences may also be generated by the "shuffling" technique outlined in Stemmer (Nature 370: 389 (1994)). Finally, a set of two or more homologous genes can be recombined in vitro to generate a starting library (Crameri et al. Nature 391:288-291 (1998)).<br>
ORFs may be constructed from random sequences in a variety of ways depending on the codons chosen. Stop codons in the open reading frame are preferably avoided. Totally random sequence libraries may be used (NNN coding) but contain a proportion of stop codons (3/64 = 4.7% per codon) that may be unacceptably high for all but the shortest libraries. Such libraries also contain rarely used codons that can sometimes result in poor translation. NNG/C codons provide a slightly reduced stop frequency (1/32 = 3.1% per codon) while providing access to the<br><br>
best codons for all 20 amino acids for mammalian translation systems. NNG/C codons are less optimal when applied in bacterial translation systems where the best codons end in A or T in 7 cases (AEGKRTV). Several solutions exist that provide for very low stop codon frequency (-1.0%), with amino acid content similar to globular proteins using three different nucleotide mixtures, N1,N2N3 codons (LaBean and Kauffman, Protein Science 2:1249-1254 (1993)) (and references therein). Finally, an almost infinite variety of semi-rational design strategies may be employed to pattern libraries according to amino acid type. For example, hydrophobic (h) or polar (p) amino acids can be chosen using NTN or NAN codons respectively (Beasley and Hecht, J. Biol. Chem. 272:2031-2034 (1997)). These can be patterned to give preference to α-helix (phpphhpp...) or p-sheet (phphph...) formation.<br>
ORFs constructed from synthetic sequences may also contain stop codons resulting from insertions or deletions in the synthetic DNA. These defects may have negative consequences due to alterations of the translation reading frame. Examination of a number of pools and synthetic genes constructed from synthetic oligonucleotides indicates that insertions and deletions occur with a frequency of -0.6% per position, or 1.8% per codon. The precise frequency of these occurrences is variable, and is thought to depend on the source and length of the synthetic DNA. In particular, longer sequences show a higher frequency of insertions and deletions (Haas et al., Current Biology 6:315-324 (1996)). A simple solution to reducing frame shifts within the ORF is to work with relatively short segments of synthetic DNA (80 nucleotides or less) that can be purified to homogeneity. Longer ORFs can then be generated by restriction and ligation of several shorter sequences.<br>
To optimize a selection scheme of the invention, the sequences and structures at the 5' and 3' ends of a template may also be altered. Preferably, this is carried out in two separate selections, each involving the insertion of random domains into the template proximal to the appropriate end, followed by selection. These selections may serve (i) to maximize the amount of fusion made (and thus to maximize the complexity of a library) or (ii) to provide optimized translation<br><br>
sequences. Further, the method may be generally applicable, combined with mutagenic PCR, to the optimization of translation templates both in the coding and non-coding regions.<br>
Step 2. Generation of RNA. As noted above, the RNA portion of an RNA-protein fusion may be chemically synthesized using standard techniques of oligonucleotide synthesis. Alternatively, and particularly if longer RNA sequences are utilized, the RNA portion is generated by in vitro transcription of a DNA template. In one preferred approach, T7 polymerase is used to enzymatically generate the RNA strand. Transcription is generally performed in the same volume as the PCR reaction (PCR DNA derived from a 100 μl reaction is used for 100 μl of transcription). This RNA can be generated with a 5' cap if desired using a large molar excess of m7GpppG to GTP in the transcription reaction (Gray and Hentze, EMBO J. 13:3882-3891 (1994)). Other appropriate RNA polymerases for this use include, without limitation, the SP6, T3 and E^ coli RNA polymerases (described, for example, in Ausubel et al. (supra, chapter 3). In addition, the synthesized RNA may be, in whole or in part, modified RNA. In one particular example, phosphorothioate RNA may be produced (for example, by T7 transcription) using modified ribonucleotides and standard techniques. Such modified RNA provides the advantage of being nuclease stable. Full length RNA samples are then purified from transcription reactions as previously described using urea PAGE followed by desalting on NAP-25 (Pharmacia) (Roberts and Szostak, Proc. Natl. Acad. Sci. USA 94:12297-12302 (1997)).<br>
Step 3. Ligation of Puromycin to the Template. Next, puromycin (or any other appropriate peptide acceptor) is covalently bonded to the template sequence. This step, may be accomplished using T4 RNA ligase to attach the puromycin directly to the RNA sequence, or preferably the puromycin may be attached by way of a DNA "splint" using T4 DNA ligase or any other enzyme which is capable of joining together two nucleotide sequences (see Figure 1B) (see also, for example, Ausubel et al., supra, chapter 3, sections 14 and 15). tRNA synthetases may also be used to attach puromycin-like compounds to RNA. For example, phenylalanyl tRNA<br><br>
synthetase links phenylalanine to phenylalanyl-tRNA molecules containing a 3' amino group, generating RNA molecules with puromycin-like 3' ends (Fraser and Rich, Proc. Natl. Acad. Sci. USA 70:2671 (1973)). Other peptide acceptors which may be used include, without limitation, any compound which possesses an amino acid linked to an adenine or an adenine-like compound, such as the amino acid nucleotides, phenylalanyl-adenosine (A-Phe), tyrosyl adenosine (A-Tyr), and alanyl adenosine (A-Ala), as well as amide-linked structures, such as phenylalanyl 3' deoxy 3' amino adenosine, alanyl 3' deoxy 3' amino adenosine, and tyrosyl 3' deoxy 3' amino adenosine; in any of these compounds, any of the naturally-occurring L-amino acids or their analogs may be utilized. A number of peptide acceptors are described, for example, in Krayevsky and Kukhanova, Progress in Nucleic Acids Research and Molecular Biology 23:1 (1979).<br>
Step 4. Generation and Recovery of RNA-Protein Fusions. To generate RNA-protein fusions, any in vitro or in situ translation system may be utilized. As shown below, eukaryotic systems are preferred, and two particularly preferred systems include the wheat germ and reticulocyte lysate systems. In principle, however, any translation system which allows formation of an RNA-protein fusion and which does not significantly degrade the RNA portion of the fusion is useful in the invention. In addition, to reduce RNA degradation in any of these systems, degradation-blocking antisense oligonucleotides may be included in the translation reaction mixture; such oligonucleotides specifically hybridize to and cover sequences within the RNA portion of the molecule that trigger degradation (see, for example, Hanes and Pluckthun, Proc, Natl. Acad. Sci USA 94:4937 (1997)).<br>
As noted above, any number of eukaryotic translation systems are available for use in the invention. These include, without limitation, lysates from yeast, ascites, tumor cells (Leibowitz et al., Meth. Enzymol. 194:536 (1991)), and xenopus oocyte eggs. Useful in vitro translation systems from bacterial systems include, without limitation, those described in Zubay (Ann. Rev. Genet. 7:267 (1973)); Chen and Zubay (Meth. Enzymol. 101:44 (1983)); and Ellman (Meth.<br><br>
Enzymol. 202:301 (1991)).<br>
In addition, translation reactions may be carried out in situ. In one particular example, translation may be carried out by injecting mRNA into Xenopus eggs using standard techniques.<br>
Once generated, RNA-protein fusions may be recovered from the translation reaction mixture by any standard technique of protein or RNA purification. Typically, protein purification techniques are utilized. As shown below, for example, purification of a fusion may be facilitated by the use of suitable chromatographic reagents such as dT25 agarose or thiopropyl sepharose. Purification, however, may also or alternatively involve purification based upon the RNA portion of the fusion; techniques for such purification are described, for example in Ausubel et al. (supra, chapter 4).<br>
Step 5. Selection of the Desired RNA-Protein Fusion. Selection of a desired RNA-protein fusion may be accomplished by any means available to selectively partition or isolate a desired fusion from a population of candidate fusions. Examples of isolation techniques include, without limitation, selective binding, for example, to a binding partner which is directly or indirectly immobilized on a column, bead, membrane, or other solid support, and immunoprecipitation using an antibody specific for the protein moiety of the fusion. The first of these techniques makes use of an immobilized selection motif which can consist of any type of molecule to which binding is possible. A list of possible selection motif molecules is presented in Figure 2. Selection may also be based upon the use of substrate molecules attached to an affinity label (for example, substrate-biotin) which react with a candidate molecule, or upon any other type of interaction with a fusion molecule. In addition, proteins may be selected based upon their catalytic activity in a manner analogous to that described by Bartel and Szostak for the isolation of RNA enzymes (supra): according to that particular technique, desired molecules are selected based upon their ability to link a target molecule to themselves, and the functional molecules are then isolated based upon the presence of that target. Selection schemes for isolating novel or improved<br><br>
catalytic proteins using this same approach or any other functional selection are enabled by the present invention.<br>
In addition, as described herein, selection of a desired RNA-protein fusion (or its DNA copy) may be facilitated by enrichment for that fusion in a pool of candidate molecules. To carry out such an optional enrichment, a population of candidate RNA-protein fusions is contacted with a binding partner (for example, one of the binding partners described above) which is specific for either the RNA portion or the protein portion of the fusion, under conditions which substantially separate the binding partner-fusion complex from unbound members in the sample. This step may be repeated, and the technique preferably includes at least two sequential enrichment steps, one in which the fusions are selected using a binding partner specific for the RNA portion and another in which the fusions are selected using a binding partner specific for the protein portion. In addition, if enrichment steps targeting the same portion of the fusion (for example, the protein portion) are repeated, different binding partners are preferably utilized. In one particular example described herein, a population of molecules is enriched for desired fusions by first using a binding partner specific for the RNA portion of the fusion and then, in two sequential steps, using two different binding partners, both of which are specific for the protein portion of the fusion. Again, these complexes may be separated from sample components by any standard separation technique including, without limitation, column affinity chromatography, centrifugation, or immunoprecipitation.<br>
Moreover, elution of an RNA-protein fusion from an enrichment (or selection) complex may be accomplished by a number of approaches. For example, as described herein, one may utilize a denaturing or non-specific chemical elution step to isolate a desired RNA-protein fusion. Such a step facilitates the release of complex components from each other or from an associated solid support in a relatively non¬specific manner by breaking non-covalent bonds between the components and/or between the components and the solid support. As described herein, one exemplary denaturing or non-specific chemical elution reagent is 4% HOAc/H20. Other<br><br>
exemplary denaturing or non-specific chemical elution reagents include guanidine, urea, high salt, detergent, or any other means by which non-covalent adducts may generally be removed. Alternatively, one may utilize a specific chemical elution approach, in which a chemical is exploited that causes the specific release of a fusion molecule. In one particular example, if the linker arm of a desired fusion protein contains one or more disulfide bonds, bound fusion aptamers may be eluted by the addition, for example, of DTT, resulting in the reduction of the disulfide bond and release of the bound target.<br>
Alternatively, elution may be accomplished by specifically disrupting affinity complexes; such techniques selectively release complex components by the addition of an excess of one member of the complex. For example, in an ATP-binding selection, elution is performed by the addition of excess ATP to the incubation mixture. Finally, one may carry out a step of enzymatic elution. By this approach, a bound molecule itself or an exogenously added protease (or other appropriate hydrolytic enzyme) cleaves and releases either the target or the enzyme. In one particular example, a protease target site may be included in either of the complex components, and the bound molecules eluted by addition of the protease. Alternately, in a catalytic selection, elution may be used as a selection step for isolating molecules capable of releasing (for example, cleaving) themselves from a solid support.<br>
Step 6. Generation of a DNA Copy of the RNA Sequence using Reverse Transcriptase. If desired, a DNA copy of a selected RNA fusion sequence is readily available by reverse transcribing that RNA sequence using any standard technique (for example, using Superscript reverse transcriptase). This step may be carried out prior to the selection or enrichment step (for example, as described in Figure 16), or following that step. Alternatively, the reverse transcription process may be carried out prior to the isolation of the fusion from the in vitro or in situ translation mixture.<br>
Next, the DNA template is amplified, either as a partial or full-length double-stranded sequence. Preferably, in this step, full-length DNA templates are<br><br>
generated, using appropriate oligonucleotides and PCR amplification.<br>
These steps, and the reagents and techniques for carrying out these steps, are now described in detail using particular examples. These examples are provided for the purpose of illustrating the invention, and should not be construed as limiting.<br>
GENERATION OF TEMPLATES FOR RNA-PROTEIN FUSTONS As shown in Figures 1A and 2, the selection scheme of the present invention preferably makes use of double-stranded DNA templates which include a number of design elements. The first of these elements is a promoter to be used in conjunction with a desired RNA polymerase for mRNA synthesis. As shown in Figure 1A and described herein, the T7 promoter is preferred, although any promoter capable of directing synthesis from a linear double-stranded DNA may be used.<br>
The second element of the template shown in Figure 1A is termed the 5' untranslated region (or 5'UTR) and corresponds to the RNA upstream of the translation start site. Shown in Figure 1A is a preferred 5'UTR (termed "TE") which is a deletion mutant of the Tobacco Mosaic Virus 5' untranslated region and, in particular, corresponds to the bases directly 5' of the TMV translation start; the sequence of this UTR is as follows: rGrGrG rArCrA rArUrU rArCrU rArUrU rUrArC rArArU rUrArC rA (with the first 3 G nucleotides being inserted to augment transcription) (SEQ ID NO: 5). Any other appropriate 5' UTR may be utilized (see, for example, Kozak, Microbiol. Rev. 47:1 (1983); and Jobling et al., Nature 325:622 (1987)).<br>
The third element shown in Figure 1A is the translation start site. In general, this is an AUG codon. However, there are examples where codons other than AUG are utilized in naturally-occurring coding sequences, and these codons may also be used in the selection scheme of the invention. The precise sequence context surrounding this codon influences the efficiency of translation (Kozak, Microbiological Reviews 47:1-45 (1983); and Kozak, J. Biol. Chem. 266:19867-<br><br>
19870 (1991)). The sequence 5'RNNAUGR provides a good start context for most sequences, with a preference for A as the first purine (-3), and G as the second (+4) (Kozak, Microbiological Reviews 47:1-45 (1983); and Kozak, J. Mol. Biol. 196:947-950(1987)).<br>
The fourth element in Figure 1A is the open reading frame of the protein (termed ORF), which encodes the protein sequence. This open reading frame may encode any naturally-occurring, random, randomized, mutagenized, or totally synthetic protein sequence. The most important feature of the ORF and adjacent 3' constant region is that neither contain stop codons. The presence of stop codons would allow premature termination of the protein synthesis, preventing fusion formation.<br>
The fifth element shown in Figure 1A is the 3' constant region. This sequence facilitates PCR amplification of the pool sequences and ligation of the puromycin-containing oligonucleotide to the mRNA. If desired, this region may also include a pause site, a sequence which causes the ribosome to pause and thereby allows additional time for an acceptor moiety (for example, puromycin) to accept a nascent peptide chain from the peptidyl-tRNA; this pause site is discussed in more detail below.<br>
To develop the present methodology, RNA-protein fusions were initially generated using highly simplified mRNA templates containing 1-2 codons. This approach was taken for two reasons. First, templates of this size could readily be made by chemical synthesis. And, second, a small open reading frame allowed critical features of the reaction, including efficiency of linkage, end heterogeneity, template dependence, and accuracy of translation, to be readily assayed.<br>
Design of Construct. A basic construct was used for generating test RNA-protein fusions.   The molecule consisted of a mRNA containing a Shine-Dalgarno (SD) sequence for translation initiation which contained a 3 base deletion of the SD sequence from ribosomal protein LI and which was complementary to 5 bases of 16S rRNA (i.e., rGrGrA rGrGrA rCrGrA rA) (SEQ ID NO: 6) (Stormo et al., Nucleic<br><br>
Acids Research 10:2971-2996 (1982); Shine and Dalgarno, Proc. Natl. Acad. Sci. USA 71:1342-1346 (1974); and Steitz and Jakes, Proc. Natl. Acad. Sci. USA 72:4734-4738 (1975)), (ii) an AUG start codon, (iii) a DNA linker to act as a pause site (i.e., 5'-(dA)27), (iv) dCdC-3', and (v) a 3' puromycin (P), The poly dA sequence was chosen because it was known to template tRNA poorly in the A site (Morgan et al., J. Mol. Biol. 26:477-497 (1967); Ricker and Kaji, Nucleic Acid Research 19:6573-6578 (1991)) and was designed to act as a good pause site. The length of the oligo dA linker was chosen to span the -60-70 A distance between the decoding site and the peptidyl transfer center of the ribosome. The dCdCP mimicked the CCA end of a tRNA and was designed to facilitate binding of the puromycin to the A site of the ribosome.<br>
Chemical Synthesis of Minimal Template 43-P. To synthesize construct 43-P (shown in Figure 3), puromycin was first attached to a solid support in such a way that it would be compatible with standard phosphoramidite oligonucleotide synthesis chemistry. The synthesis protocol for this oligo is outlined schematically in Figure 3 and is described in more detail below. To attach puromycin to a controlled pore glass (CPG) solid support, the amino group was protected with a trifluoroacetyl group as described in Applied Biosystems User Bulletin #49 for DNA synthesizer model 380 (1988). Next, protection of the 5' OH was carried out using a standard DMT-C1 approach (Gait, Oligonucleotide Synthesis a practical approachThe Practical Approach Series (IRL Press, Oxford, 1984)), and attachment to aminohexyl CPG through the 2' OH was effected in exactly the same fashion as the 3' OH would be used for attachment of a deoxynucleoside (see Fig. 3 and Gait, supra, p. 47).   The 5' DMT-CPG-linked protected puromycin was then suitable for chain extension with phosphoramidite monomers. The synthesis of the oligo proceeded in the 3' -&gt; 5' direction in the order: (i) 3' puromycin, (ii) pdCpdC, (iii) -27 units of dA as a linker, (iv) AUG, and (v) the Shine-Dalgarno sequence. The sequence of the 43-P construct is shown below.<br>
Synthesis of CPG Puromycin. The synthesis of protected CPG puromycin<br><br>
followed the general path used for deoxynucleosides as previously outlined (Gait, Oligonucleotide Synthesis, A Practical Approach, The Practical Approach Series (IRL Press, Oxford, 1984)). Major departures included the selection of an appropriate N blocking group, attachment at the puromycin 2' OH to the solid support, and the linkage reaction to the solid support. In the case of the latter, the reaction was carried out at very low concentrations of activated nucleotide as this material was significantly more precious than the solid support. The resulting yield (~20 umol/g support) was quite satisfactory considering the dilute reaction conditions.<br>
Synthesis of N-Trifluoroacetyl Puromycin. 267 mg (0.490 mmol) Puromycin*HCl was first converted to the free base form by dissolving in water, adding pH 11 carbonate buffer, and extracting (3X) into chloroform. The organic phase was evaporated to dryness and weighed (242 mg, 0.513 mmol). The free base was then dissolved in 11 ml dry pyridine and 11 ml dry acetonitrile, and 139 ul (2.0 mmol) triethylamine (TEA; Fluka) and 139 ul (1.0 mmol) of trifluoroacetic anhydride (TFAA; Fluka) were added with stirring. TFAA was then added to the turbid solution in 20 μl aliquots until none of the starting material remained, as assayed by thin layer chromatography (tic) (93:7, Chloroform/MeOH) (a total of 280 u.1). The reaction was allowed to proceed for one hour. At this point, two bands were revealed by thin layer chromatography, both of higher mobility than the starting material. Workup of the reaction with NH4OH and water reduced the product to a single band. Silica chromatography (93:7 Chloroform/MeOH) yielded 293 mg (0.515 mmol) of the product, N-TFA-Pur. The product of this reaction is shown schematically in Figure 4.<br>
Synthesis of N-Trifluoroacetvl 5'-DMT Puromvcin. The product from the above reaction was aliquoted and coevaporated 2X with dry pyridine to remove water. Multiple tubes were prepared to test multiple reaction conditions. In a small scale reaction, 27.4 mg (48.2 pinoles) N-TFA-Pur was dissolved in 480 ul of pyridine containing 0.05 eq of DMAP and 1.4 eq TEA. To this mixture, 20.6 mg of di-methoxy trityl chloride (60 μmol) was added, and the reaction was allowed to proceed to completion with stirring. The reaction was stopped by addition of an equal volume<br><br>
of water (approximately 500 μl) to the solution. Because this reaction appeared successful, a large scale version was performed. In particular, 262 mg (0.467 mmol) N-TFA-Pur was dissolved in 2.4 ml pyridine followed by addition of 1.4 eq of TEA, 0.05 eq of DMAP, and 1.2 eq of di-methoxy trityl chloride (Sigma). After approximately two hours, an additional 50 mg (0.3 eq) dimethoxytriryI*Cl (DMT*C1) was added, and the reaction was allowed to proceed for 20 additional minutes. The reaction was stopped by the addition of 3 ml of water and coevaporated 3X with CH3CN. The reaction was purified by 95:5 Chloroform/MeOH on a 100 ml silica (dry) 2 mm diameter column. Due to incomplete purification, a second identical column was run with 97.5:2.5 Chloroform/MeOH. The total yield was 325 mg or 0.373 mmol (or a yield of 72%). The product of this reaction is shown schematically in Figure 4.<br>
Synthesis of N-Trifluoroacetvl. 5'-DMT. 2' Succinyl Puromycin. In a small scale reaction, 32 mg (37 umol) of the product synthesized above was combined with 1.2 eq of DMAP dissolved in 350 ul of pyridine. To this solution, 1.2 equivalents of succinic anhydride was added in 44 μl of dry CH3CN and allowed to stir overnight. Thin layer chromatography revealed little of the starting material remaining. In a large scale reaction, 292 mg (336 umol) of the previous product was combined with 1.2 eq DMAP in 3 ml of pyridine. To this, 403 ul of 1M succinic anhydride (Fluka) in dry CH3CN was added, and the mixture was allowed to stir overnight. Thin layer chromatography again revealed little of the starting material remaining. The two reactions were combined, and an additional 0.2 eq of DMAP and succinate were added. The product was coevaporated with toluene IX and dried to a yellow foam in high vacuum. CH2C12 was added (20 ml), and this solution was extracted twice with 15 ml of 10% ice cold citric acid and then twice with pure water. The product was dried, redissolved in 2 ml of CH2C12, and precipitated by addition of 50 ml of hexane with stirring. The product was then vortexed and centrifuged at 600 rpm for 10 minutes in the clinical centrifuge. The majority of the eluent was drawn off, and the rest of the product was dried, first at low vacuum, then at high vacuum in<br><br>
a dessicator. The yield of this reaction was approximately 260 μmol for a stepwise yield of-70 %.<br>
Synthesis of N-Trifluoroacetvl 5'-DMT. 2' Succinyl. CPG Puromvcin. The product from the previous step was next dissolved with 1 ml of dioxane (Fluka) followed by 0.2 ml dioxane/0.2 ml pyridine. To this solution, 40 mg of p-nitrophenol (Fluka) and 140 mg of dicyclohexylcarbodiimide (DCC; Sigma) was added, and the reaction was allowed to proceed for 2 hours. The insoluble cyclohexyl urea produced by the reaction was removed by centrifugation, and the product solution was added to 5 g of aminohexyl controlled pore glass (CPG) suspended in 22 ml of dry DMF and stirred overnight. The resin was then washed with DMF, methanol, and ether, and dried. The resulting resin was assayed as containing 22.6 umol of trityl per g, well within the acceptable range for this type of support. The support was then capped by incubation with 15 ml of pyridine, 1 ml of acetic anhydride, and 60 mg of DMAP for 30 minutes. The resulting column material produced a negative (no color) ninhydrin test, in contrast to the results obtained before blocking in which the material produced a dark blue color reaction. The product of this reaction is shown schematically in Figure 4. Alternatively, puromycin-CPG may be obtained commercially (Trilink).<br>
Synthesis of mRNA-Puromvcin Conjugate. As discussed above, a puromycin tethered oligo may be used in either of two ways to generate a mRNA-puromycin conjugate which acts as a translation template. For extremely short open reading frames, the puromycin oligo is typically extended chemically with RNA or DNA monomers to create a totally synthetic template. When longer open reading frames are desired, the RNA or DNA oligo is generally Hgated to the 3' end of an mRNA using a DNA splint and T4 DNA ligase as described by Moore and Sharp (Science 256:992 (1992)).<br>
IN VTTRO TRANSLATION AND TESTING OF RNA-PROTEIN FUSIONS The templates generated above were translated in. vitro using both bacterial<br><br>
and eukaryotic in vitro translation systems as follows.<br>
In Vitro Translation of Minimal Templates. 43-P and related RNA-puromycin conjugates were added to several different in vitro translation systems including: (i) the S30 system derived from E. coli MRE600 (Zubay, Ann. Rev. Genet. 7:267 (1973); Collins, Gene 6:29 (1979); Chen and Zubay, Methods Enzymol, 101:44 (1983); Pratt, in Transcription and Translation: A Practical Approach, B. D. Hammes, S. J. Higgins, Eds. (IRL Press, Oxford, 1984) pp. 179-209; and Ellman et al., Methods Enzymol. 202:301 (1991)) prepared as described by Ellman et. al. (Methods Enzymol. 202:301 (1991)); (ii) the ribosomal fraction derived from the same strain, prepared as described by Kudlicki et al. (Anal. Chem. 206:389 (1992)); and (iii) the S30 system derived from E coil  BL21, prepared as described by Lesley et al. (J. Biol. Chem. 266:2632 (1991)). In each case, the premix used was that of Lesley et al. (J. Biol. Chem. 266:2632 (1991)), and the incubations were 30 minutes in duration.<br>
Testing the Nature of the Fusion. The 43-P template was first tested using S30 translation extracts from E. Coil- Figure 5 (Reaction "A") demonstrates the desired intramolecular (cis) reaction wherein 43-P binds the ribosome and acts as a template for and an acceptor of fMet at the same time. The incorporation of 35S-methionine and its position in the template was first tested, and the results are shown in Figures 6 A and 6B. After extraction of the in vitro translation reaction mixture with phenol/chloroform and analysis of the products by SDS-PAGE, an 35S labeled band appeared with the same mobility as the 43-P template. The amount of this material synthesized was dependent upon the Mg2+ concentration (Figure 6A). The optimum Mg2+ concentration appeared to be between 9 and 18 mM, which was similar to the optimum for translation in this system (Zubay, Ann. Rev. Genet. 7:267 (1973); Collins, Gene 6:29 (1979); Chen and Zubay, Methods Enzymol, 101:44 (1983); Pratt, in Transcription and Translation; A Practical Approach, B. D. Hammes, S. J. Higgins, Eds. (IRL Press, Oxford, 1984) pp. 179-209; Ellman et al., Methods Enzymol. 202:301 (1991); Kudlicki et al., Anal. Chem. 206:389 (1992); and<br><br>
Lesley et ah, J. Biol. Chem. 266:2632 (1991)). Furthermore, the incorporated label was stable to treatment with NKL,OH (Figure 6B), indicating that the label was located on the 3* half of the molecule (the base-stable DNA portion) and was attached by a base-stable linkage, as expected for an amide bond between puromycin and fMet.<br>
Ribosome and Template Dependence. To demonstrate that the reaction observed above occurred on the ribosome, the effects of specific inhibitors of the peptidyl transferase function of the ribosome were tested (Figure 6C), and the effect of changing the sequence coding for methionine was examined (Figure 6D). Figure 6C demonstrates clearly that the reaction was strongly inhibited by the peptidyl transferase inhibitors, virginiamycin, gougerotin, and chloramphenicol (Monro and Vazquez, J. Mol. Biol. 28:161-165 (1967); and Vazquez and Monro, Biochemica et Biophysical Acta 142:155-173 (1967)). Figure 6D demonstrates that changing a single base in the template from A to C abolished incorporation of 35S methionine at 9 mM Mg2+, and greatly decreased it at 18 mM (consistent with the fact that high levels of Mg2+ allow misreading of the message). These experiments demonstrated that the reaction occurred on the ribosome in a template dependent fashion.<br>
Linker Length. Also tested was the dependence of the reaction on the length of the linker (Figure 6E). The original template was designed so that the linker spanned the distance from the decoding site (occupied by the AUG of the template) to the acceptor site (occupied by the puromycin moiety), a distance which was approximately the same length as the distance between the anticodon loop and the acceptor stem in a tRNA, or about 60-70 A. The first linker tested was 30 nucleotides in length, based upon a minimum of 3.4 A per base (2 102 A). In the range between 30 and 21 nucleotides (n = 27 -18; length ;&gt; 102 - 71 A), little change was seen in the efficiency of the reaction. Accordingly, linker length may be varied. While a linker of between 21 and 30 nucleotides represents a preferred length, linkers shorter than 80 nucleotides and, preferably, shorter than 45 nucleotides may also be utilized in the invention.<br>
Intramolecular vs. Intermolecular Reactions. Finally, we tested whether<br><br>
the reaction occurred in an intramolecular fashion (Figure 5, Reaction "A") as desired or intermolecularly (Figure 5, Reaction "B"). This was tested by adding oligonucleotides with 3' puromycin but no ribosome binding sequence (i.e., templates 25-P, 13-P, and 30-P) to the translation reactions containing the 43-P template (Figures 6F, 6G, and 6H). If the reaction occurred by an intermolecular mechanism, the shorter oligos would also be labeled. As demonstrated in Figures 6F-H, there was little incorporation of 35S methionine in the three shorter oligos, indicating that the reaction occurred primarily in an intramolecular fashion. The sequences of 25-P (SEQ ID NO: 10), 13-P (SEQ ID NO: 9), and 30-P (SEQ ID NO: 8) are shown below. Reticulocyte Lysate. Figure 6H demonstrates that 35S-methionine may be incorporated in the 43-P template using a rabbit reticulocyte lysate (see below) for in vitro translation, in addition to the E coil lysates used above. This reaction occurred primarily in an intramolecular mechanism, as desired.<br>
SYNTHESIS AND TESTING OF FUSIONS CONTAINING A C-MYC EPITOPE TAG<br>
Exemplary fusions were also generated which contained, within the protein portion, the epitope tag for the c-myc monoclonal antibody 9E10 (Evan et al., Mol. Cell Biol. 5:3610 (1985)).<br>
Design of Templates. Three initial epitope tag templates (i.e., LP77, LP 154, and Pool #1) were designed and are shown in Figures 7A-C. The first two templates contained the c-myc epitope tag sequence EQKLISEEDL (SEQ ID NO: 2), and the third template was the design used in the synthesis of a random selection pool. LP77 encoded a 12 amino acid sequence, with the codons optimized for bacterial translation. LP 154 and its derivatives contained a 33 amino acid mRNA sequence in which the codons were optimized for eukaryotic translation. The encoded amino acid sequence of MAEEQKLISEEDLLRKRREQKLKHKLEQLRNSCA (SEQ ID NO: 7) corresponded to the original peptide used to isolate the 9E10 antibody. Pool#l contained 27 codons of NNG/C (to generate random peptides) followed by a sequence<br><br>
corresponding to the last seven amino acids of the myc peptide (which were not part of the myc epitope sequence). These sequences are shown below.<br>
Reticulocyte vs. Wheat Germ In Vitro Translation Systems. The 43-P, LP77, and LP 154 templates were tested in both rabbit reticulocyte and wheat germ extract (Promega, Boehringer Mannheim) translation systems (Figure 8). Translations were performed at 30°C for 60 minutes. Templates were isolated using dT25 agarose at 4°C. Templates were eluted from the agarose using 15 mM NaOH, 1 mM EDTA, neutralized with NaOAc/HOAc buffer, immediately ethanol precipitated (2.5 - 3 vol), washed (with 100% ethanol), and dried on a speedvac concentrator. Figure 8 shows that 35S methionine was incorporated into all three templates, in both the wheat germ and reticulocyte systems. Less degradation of the template was observed in the fusion reactions from the reticulocyte system and, accordingly, this system is preferred for the generation of RNA-protein fusions. In addition, in general, eukaryotic systems are preferred over bacterial systems. Because eukaryotic cells tend to contain lower levels of nucleases, mRNA lifetimes are generally 10-100 times longer in these cells than in bacterial cells. In experiments using one particular &amp; coli translation system, generation of fusions was not observed using a template encoding the c-myc epitope; labeling the template in various places demonstrated that this was likely due to degradation of both the RNA and DNA portions of the template.<br>
To examine the peptide portion of these fusions, samples were treated with RNase to remove the coding sequences. Following this treatment, the 43-P product ran with almost identical mobility to the 32P labeled 30-P oligo, consistent with a very small peptide (perhaps only methionine) added to 30-P. For LP77, removal of the coding sequence produced a product with lower mobility than the 30-P oligo, consistent with the notion that a 12 amino acid peptide was added to the puromycin. Finally, for LP 154, removal of the coding sequence produced a product of yet lower mobility, consistent with a 33 amino acid sequence attached to the 30-P oligo. No oligo was seen in the RNase-treated LP 154 reticulocyte lane due to a loading error. In Figure 9, the mobility of this product was shown to be the same as the product<br><br>
generated in the wheat genu extract. In sum. these results indicated that RNase resistant products were added to the ends of the 30-P oligos, that the sizes of the products were proportional to the length of the coding sequences, and that the products were quite homogeneous in size. In addition, although both systems produced similar fusion products, the reticulocyte system appeared superior due to higher template stability.<br>
Sensitivity to RNase A and Proteinase K. In Figure 9, sensitivity to RNase A and proteinase K were tested using the LP154 fusion. As shown in lanes 2-4, incorporation of 35S methionine was demonstrated for the LP 154 template. When this product was treated with RNase A, the mobility of the fusion decreased, but was still significantly higher than the 32P labeled 30-P oligonucleotide, consistent with the addition of a 33 amino acid peptide to the 3' end. When this material was also treated with proteinase K, the 35S signal completely disappeared, again consistent with the notion that the label was present in a peptide at the 3' end of the 30-P fragment. Similar results have been obtained in equivalent experiments using the 43-P and LP77 fusions.<br>
To confirm that the template labeling by 35S Met was a consequence of translation, and more specifically resulted from the peptidyl transferase activity of the ribosome, the effect of various inhibitors on the labeling reaction was examined. The specific inhibitors of eukaryotic peptidyl transferase, anisomycin, gougerotin, and sparsomycin (Vazquez, Inhibitors of Protein Biosynthesis (Springer-Verlag, New York), pp. 312 (1979)), as well as the translocation inhibitors cycloheximide and emetine (Vazquez, Inhibitors of Protein Biosynthesis (Springer-Verlag, New York), pp. 312 (1979)) all decreased RNA-peptide fusion formation by ~95% using the long myc template and a reticulocyte lysate translation extract.<br>
Immunoprecipitation Experiments. In an experiment designed to illustrate the efficacy of immunoprecipitating an mRNA-peptide fusion, an attempt was made to immunoprecipitate a free c-myc peptide generated by in vitro translation. Figure 10 shows the results of these experiments assayed on an SDS PAGE peptide gel.<br><br>
Lanes 1 and 2 show the labeled material from translation reactions containing either RNA124 (the RNA portion of LP 154) or p-globin mRNA. Lanes 3-8 show the immunoprecipitation of these reaction samples using the c-myc monoclonal antibody 9E10, under several different buffer conditions (described below). Lanes 3-5 show that the peptide derived from RNA 124 was effectively immunoprecipitated, with the best case being lane 4 where -83% of the total TCA precipitable counts were isolated. Lanes 6-8 show little of the p-globin protein, indicating a purification of &gt; 100 fold. These results indicated that the peptide coded for by RNA124 (and by LP 154) can be quantitatively isolated by this immunoprecipitation protocol.<br>
Immunoprecipitation of the Fusion. We next tested the ability to immunoprecipitate a chimeric RNA-peptide product, using an LP 154 translation reaction and the c-myc monoclonal antibody 9E10 (Figure 11). The translation products from a reticulocyte reaction were isolated by immunoprecipitation (as described herein) and treated with 1 fig of RNase A at room temperature for 30 minutes to remove the coding sequence. This generated a 5'OH, which was 32P labeled with T4 polynucleotide kinase and assayed by denaturing PAGE. Figure 11 demonstrates that a product with a mobility similar to that seen for the fusion of the c-myc epitope with 30-P generated by RNase treatment of the LP 154 fusion (see above) was isolated, but no corresponding product was made when only the RNA portion of the template (RNA124) was translated. In Figure 12, the quantity of fusion protein isolated was determined and was plotted against the amount of unmodified 30-P (not shown in this figure). Quantitation of the ratio of unmodified linker to linker-myc peptide fusion shows that 0.2 - 0.7% of the input message was converted to fusion product. A higher fraction of the input RNA was converted to fiision product in the presence of a higher ribosome/template ratio; over the range of input mRNA concentrations that were tested, approximately 0.8 -1.0 x 1012 fusion molecules were made per ml of translation extract.<br>
In addition, our results indicated that the peptides attached to the RNA species were encoded by that mRNA, i.e. the nascent peptide was not transferred to<br><br>
the puromycin of some other mRNA. No indication of cross-transfer was seen when a linker (30-P) was coincubated with the long myc template in translation extracts in ratios as high as 20:1, nor did the presence of free linker significantly decrease the amount of long myc fusion produced. Similarly, co-translation of the short and long templates, 43-P and LP 154, produced only the fusion products seen when the templates were translated alone, and no products of intermediate mobility were observed, as would be expected for fusion of the short template with the long myc peptide. Both of these results suggested that fusion formation occurred primarily between a nascent peptide and mRNA bound to the same ribosome.<br>
Sequential Isolation. As a further confirmation of the nature of the in vitro translated LP 154 template product, we examined the behavior of this product on two different types of chromatography media. Thiopropyl (TP) sepharose allows the isolation of a product containing a free cysteine (for example, the LP 154 product which has a cysteine residue adjacent to the C terminus) (Figure 13). Similarly, dT2S agarose allows the isolation of templates containing a poly dA sequence (for example, 30-P) (Figure 13). Figure 14 demonstrates that sequential isolation on TP sepharose followed by dT25 agarose produced the same product as isolation on dT25 agarose alone. The fact that the in vitro translation product contained both a poly-A tract and a free thiol strongly indicated that the translation product was the desired RNA-peptide fusion.<br>
The above results are consistent with the ability to synthesize mRNA-peptide fusions and to recover them intact from in vitro translation extracts. The peptide portions of fusions so synthesized appeared to have the intended sequences as demonstrated by immunoprecipitation and isolation using appropriate chromatographic techniques. According to the results presented above, the reactions are intramolecular and occur in a template dependent fashion. Finally, even with a template modification of less than 1%, the present system facilitates selections based on candidate complexities of about 1013 molecules.<br>
C-Mvc Epitope Recovery Selection. To select additional c-myc epitopes, a<br><br><br>
large library of translation templates (for example. 1015 members) is generated containing a randomized region (see Figure 7C and below). This library is used to generate ~1012 - 1013 fusions (as described herein) which are treated with the anti-c-myc antibody (for example, by immunoprecipitation or using an antibody immobilized on a column or other solid support) to enrich for c-myc-encoding templates in repeated rounds of in vitro selection-Models for Fusion Formation. Without being bound to a particular theory, we propose a model for the mechanism of fusion formation in which translation initiates normally and elongation proceeds to the end of the open reading frame. When the ribosome reaches the DNA portion of the template, translation stalls. At this point, the complex can partition between two fates: dissociation of the nascent peptide, or transfer of the nascent peptide to the puromycin at the 3'-end of the template. The efficiency of the transfer reaction is likely to be controlled by a number of factors that influence the stability of the stalled translation complex and the entry of the 3'-puromycin residue into the A site of the peptidyl transferase center. After the transfer reaction, the mRNA-peptide fusion likely remains complexed with the ribosome since the known release factors cannot hydrolyze the stable amide linkage between the RNA and peptide domains.<br>
Both the classical model for elongation (Watson, Bull. Soc. Chim. Biol. 46:1399 (1964)) and the intermediate states model (Moazed and Noller, Nature 342:142 (1989)) require that the A site be empty for puromycin entry into the peptidyl transferase center. For the puromycin to enter the empty A site, the linker must either loop around the outside of the ribosome or pass directly from the decoding site through the A site to the peptidyl transferase center. The data described herein do not clearly distinguish between these alternatives because the shortest linker tested (21 nts) is still long enough to pass around the outside of the ribosome. In some models of ribosome structure (Frank et al., Nature 376:441 (1995)), the mRNA is threaded through a channel that extends on either side of the decoding site, in which case unthreading of the linker from the channel would be required to allow the puromycin<br><br>
to reach the peptidyl transferase center through the A site.<br>
Transfer of the nascent peptide to the puromycin appeared to be slow relative to the elongation process as demonstrated by the homogeneity and length of the peptide attached to the linker. If the puromycin competed effectively with aminoacyl tRN As during elongation, the linker-peptide fusions present in the fusion products would be expected to be heterogeneous in size. Furthermore, the ribosome did not appear to read into the linker region as indicated by the similarity in gel mobilities between the Met-template fusion and the unmodified linker. dA3n should code for (lysine),, which would certainly decrease the mobility of the linker. The slow rate of unthreading of the mRNA may explain the slow rate of fusion formation relative to the rate of translocation. Preliminary results suggest that the amount of fusion product formed increases markedly following extended post-translation incubation at low temperature, perhaps because of the increased time available for transfer of the nascent peptide to the puromycin.<br>
DETAILED MATERIALS AND METHODS Described below are detailed materials and methods relating to the in vitro<br>
translation and testing of RNA-protein fusions, including fusions having a myc<br>
epitope tag.<br>
Sequences. A number of oligonucleotides were used above for the<br>
generation of RNA-protein fusions. These oligonucleotides have the following<br>
sequences.<br>
NAME     SEQUENCE<br>
30-P	5*AAA AAA AAA AAA AAA AAA AAA AAA AAA CCP (SEQ ID<br>
NO:8)<br>
13-P	5'AAA AAA AAA ACC P (SEQ ID NO: 9)<br>
25-P	5'CGC GGT TTT TAT TTT TTT TTT TCC P (SEQ ID NO: 10)<br><br>
43-P	5'rGrGrA rGrGrA rCrGrA rArArU rGAA AAA AAA AAA AAA AAA<br>
AAA AAA AAA ACC P (SEQ ID NO: 11)<br>
43-P [CUG]	5'rGrGrA rGrGrA rCrGrA rArCrU rGAA AAA AAA AAA AAA<br>
AAA AAA AAA AAA ACC P (SEQ ID NO: 12)<br>
40-P	5'rGrGrA rGrGrA rCrGrA rArCrU rGAA AAA AAA AAA AAA AAA<br>
AAA AAA ACC P (SEQ ID NO: 13)<br>
37-P	5'rGrGrA rGrGrA rCrGrA rArCrU rGAA AAA AAA AAA AAA AAA<br>
AAA ACC P (SEQ ID NO: 14)<br>
34-P	5'rGrGrA rGrGrA rCrGrA rArCrU rGAA AAA AAA AAA AAA AAA<br>
ACC P (SEQ ID NO: 15)<br>
31 -P	5'rGrGrA rGrGrA rCrGrA rArCrU rGAA AAA AAA AAA AAA ACC P<br>
(SEQ ID NO: 16)<br>
LP77        5'rGrGrG rArGrG rArCrG rArArA rUrGrG rArArC rArGrA rArArC rUrGrA rUrCrU rCrUrG rArArG rArArG rArCrC rUrGrA rArC AAA AAA AAA AAA AAA AAA AAA AAA AAA CCP (SEQ ID NO: 1)<br>
LP 154      5'rGrGrG rArCrA rArUrU rArCrU rArUrU rUrArC rArArU rUrArC rA rArUrG rGrCrU rGrArA rGrArA rCrArG rArArA rCrUrG rArUrC rUrCrU rGrArA rGrArA rGrArC rCrUrG rCrUrG rCrGrU rArArA rCrGrU rCrGrU rGrArA rCrArG rCrUrG rArArA rCrArC rArArA rCrUrG rGrArA rCrArG rCrUrG rCrGrU rArArC rUrCrU rUrGrC rGrCrU AAA AAA AAA AAA AAA AAA AAA AAA AAA CCP (SEQ ID NO: 3)<br><br>
LP 160      5' 5'rGrGrG rArCrA rArUrU rArCrU rArUrU rUrArC rArArU rUrArC rA rArUrG rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rNrNrS rCrArG rCrUrG rCrGrU rArArC rUrCrU rUrGrC rGrCrU AAA AAA AAA AAA AAA AAA AAA AAA AAA CCP (SEQ ID NO: 17)<br>
All oligonucleotides are listed in the 5' to 3' direction. Ribonucleotide bases are indicated by lower case "r" prior to the nucleotide designation; P is puromycin; rN indicates equal amounts of rA, rG, rC, and rU; rS indicates equal amounts of rG and rC; and all other base designations indicate DNA oligonucleotides.<br>
Chemicals. Puromycin HC1, long chain alkylamine controlled pore glass, gougerotin, chloramphenicol, virginiamycin, DMAP, dimethyltrityl chloride, and acetic anhydride were obtained from Sigma Chemical (St. Louis, MO). Pyridine, dimethylformamide, toluene, succinic anhydride, and para-nitrophenol were obtained from Fluka Chemical (Ronkonkoma, NY). Beta-globin mRNA was obtained from Novagen (Madison, WI). TMV RNA was obtained from Boehringer Mannheim (Indianapolis, IN).<br>
Enzymes. Proteinase K was obtained from Promega (Madison, WI). DNase-free RNAase was either produced by the protocol of Sambrook et al. (supra) or purchased from Boehringer Mannheim. T7 polymerase was made by the published protocol of Grodberg and Dunn (J. Bacteriol. 170:1245 (1988)) with the modifications of Zawadzki and Gross (Nucl. Acids Res. 19:1948 (1991)). T4 DNA Iigase was obtained from New England Biolabs (Beverly, MA).<br>
Quantitation of Radiolabel Incorporation. For radioactive gels bands, the amount of radiolabel (35S or 32P) present in each band was determined by quantitation either on a Betagen 603 blot analyzer (Betagen, Waltham, MA) or using phosphorimager plates (Molecular Dynamics, Sunnyvale, CA). For liquid and solid samples, the amount of radiolabel (35S or 32P) present was determined by scintillation<br><br>
counting Beckman Corumbra MD).<br>
Gel Images. Images of gels were obtained by autoradiography (using Kodak XAR film) or using phosphorimager plates (Molecular Dynamics).<br>
Synthesis of CPG Puromycin. Detailed protocols for synthesis of CPG-puromycin are outlined above.<br>
Enzymatic Reactions. In general, the preparation of nucleic acids for kinase, transcription, PCR, and translation reactions using E coli extracts was the same. Each preparative protocol began with extraction using an equal volume of 1:1 phenol/chloroform, followed by centrifugation and isolation of the aqueous phase. Sodium acetate (pH 5.2) and spermidine were added to a final concentration of 300 mM and 1 mM respectively, and the sample was precipitated by addition of 3 volumes of 100% ethanol and incubation at -70°C for 20 minutes. Samples were centrifuged at &gt;12,000 g, the supernatant was removed, and the pellets were washed with an excess of 95% ethanol, at 0°C. The resulting pellets were then dried under vacuum and resuspended.<br>
Oligonucleotides. All synthetic DNA and RNA was synthesized on a Millipore Expedite synthesizer using standard chemistry for each as supplied from the manufacturer (Milligen, Bedford, MA). Oligonucleotides containing 31 puromycin were synthesized using CPG puromycin columns packed with 30-50 mg of solid support (~20 μmole puromycin/gram). Oligonucleotides containing a 3' biotin were synthesized using 1 μmole bioteg CPG columns from Glen Research (Sterling, VA). Oligonucleotides containing a 5' biotin were synthesized by addition of bioteg phosphoramidite (Glen Research) as the 5' base.  Oligonucleotides to be ligated to the 3' ends of RNA molecules were either chemically phosphorylated at the 5' end (using chemical phosphorylation reagent from Glen Research) prior to deprotection or enzymatically phosphorylated using ATP and T4 polynucleotide kinase (New England Biolabs) after deprotection. Samples containing only DNA (and 3' puromycin or 3' biotin) were deprotected by addition of 25% NH4OH followed by incubation for 12 hours at 55CC. Samples containing RNA monomers (e.g., 43-P)<br><br>
were deprotected by addition of ethanol (25% (v/v)) to the NH4OH solution and incubation for 12 hours at 55°C. The 2'OH was deprotected using IM TBAP in THF (Sigma) for 48 hours at room temperature. TBAF was removed using a NAP-25 Sephadex column (Pharmacia, Piscataway, NJ).<br>
If desired, to test for the presence of 3' hydroxyl groups, the puromycin oligonucleotide may be radiolabeled at the 5' end using T4 polynucleotide kinase and then used as a primer for extension with terminal deoxynucleotidyl transferase. The presence of the primary amine in the puromycin may be assayed by reaction with amine derivatizing reagents such as NHS-LC-biotin (Pierce). Oligonucleotides, such as 30-P, show a detectable mobility shift by denaturing PAGE upon reaction, indicating quantitative reaction with the reagent. Oligonucleotides lacking puromycin do not react with NHS-LC-biotin and show no change in mobility.<br>
Deprotected DNA and RNA samples were then purified using denaturing PAGE, followed by either soaking or electro-eluting from the gel using an Elutrap (Schleicher and Schuell, Keene, NH) and desalting using either a NAP-25 Sephadex column or ethanol precipitation as described above.<br>
Mvc DNA construction. Two DNA templates containing the c-myc epitope tag were constructed. The first template was made from a combination of the oligonucleotides 64.27 (5'-GTT CAG GTC TTC TTG AGA GAT CAG TTT CTG TTC CAT TTC GTC CTC CCT ATA GTG AGT CGT ATT A-3') (SEQ ID NO: 18) and 18.109 (5'-TAA TAC GAC TCA CTA TAG-3') (SEQ ID NO: 19). Transcription using this template produced RNA 47.1 which coded for the peptide MEQKLISEEDLN (SEQ ID NO: 20). Ligation of RNA 47.1 to 30-P yielded LP77 shown in Figure 7A.<br>
The second template was made first as a single oligonucleotide 99 bases in length, having the designation RWR 99.6 and the sequence 5'AGC GCA AGA GTT ACG CAG CTG TTC CAG TTT GTG TTT CAG CTG TTC ACG ACG TTT ACG CAG CAG GTC TTC TTC AGA GAT CAG TTT CTG TTC TTC AGC CAT-3' (SEQ ID NO: 21). Double stranded transcription templates containing this sequence<br><br>
were constructed by PCR with the oligos RWR 21.103 (5'-AGC GCA AGA GTT ACG CAG CTG-3") (SEQ ID NO: 22) and RWR 63.26 (5TAA TAC GAC TCA CTA TAG GGA CAA TTA CTA TTT ACA ATT ACA ATG GCT GAA GAA CAG AAA CTG-3') (SEQ ID NO: 23) according to published protocols (Ausubel et al., supra, chapter 15). Transcription using this template produced an RNA referred to as RNA 124 which coded for the peptide<br>
MAEEQKLISEEDLLRKRREQLKHKLEQLRNSCA (SEQ ID NO: 24). This peptide contained the sequence used to raise monoclonal antibody 9E10 when conjugated to a carrier protein (Oncogene Science Technical Bulletin). RNA 124 was 124 nucleotides in length, and ligation of RNA 124 to 30-P produced LP 154 shown in Figure 7B. The sequence of RNA 124 is as follows (SEQ ID NO: 32): 5'-rGrGrG rArCrA rArUrU rArCrU rArUrU rUrArC rArArU rUrArC rArArUrG rGrCrU rGrArA rGrArA rCrArG rArArA rCrUrG rArUrC rUrCrU rGrArA rGrArA rGrArC rCrUrG rCrUrG rCrGrU rArArA rCrGrU rCrGrU rGrArA rCrArG rCrUrG rArArA rCrArC rArArA rCrUrG rGrArA rCrArG rCrUrG rCrGrU rArArC rUrCrU rUrGrC rGrCrU-3'<br>
Randomized Pool Construction. The randomized pool was constructed as a single oligonucleotide 130 bases in length denoted RWR130.1. Beginning at the 3' end, the sequence was 3' CCCTGTTAATGATAAATGTTAATGTTAC (NNS)27 GTC GAC GCA TTG AGA TAC CGA-5' (SEQ ID NO: 25). N denotes a random position, and this sequence was generated according to the standard synthesizer protocol. S denotes an equal mix of dG and dC bases. PCR was performed with the oligonucleotides 42.108 (5'-TAA TAC GAC TCA CTA TAG GGA CAA TTA CTA TTT ACA ATT ACA) (SEQ ID NO: 26) and 21.103 (5'-AGC GCA AGA GTT ACG CAG CTG) (SEQ ID NO: 27). Transcription off this template produced an RNA denoted pool 130.1. Ligation of pool 130.1 to 30-P yielded Pool #1 (also referred to as LP 160) shown in Figure 7C.<br>
Seven cycles of PCR were performed according to published protocols (Ausubel et al., supra) with the following exceptions: (i) the starting concentration of<br><br>
RWR130.1 was 30 nanomolar, (ii) each primer was used at a concentration of 1.5 uM, (iii) the dNTP concentration was 400 uM for each base, and (iv) the Taq polymerase (Boeliringer Mannheim) was used at 5 units per 100 μl.   The double stranded product was purified on non-denaturing PAGE and isolated by electrocution. The amount of DNA was determined both by UV absorbance at 260 nm and ethidium bromide fluorescence comparison with known standards.<br>
Enzymatic Synthesis of RNA. Transcription reactions from double stranded PCR DNA and synthetic oligonucleotides were performed as described previously (Milligan and Uhlenbeck, Meth. Enzymol. 180:51 (1989)). Full length RNA was purified by denaturing PAGE, electroeluted, and desalted as described above. The pool RNA concentration was estimated using an extinction coefficient of 1300 O.D./umole; RNA124, 1250 O.D./umole; RNA 47.1, 480 O.D./umole. Transcription from the double stranded pool DNA produced ~ 90 nanomoles of pool RNA.<br>
Enzvmatic Synthesis of RNA-Puromvcin Conjugates.   Ligation of the myc and pool messenger RNA sequences to the puromycin contaimng oligonucleotide was performed using a DNA splint, termed 19.35 (5'-TTT TTT TTT TAG CGC AAG A) (SEQ ID NO: 28) using a procedure analogous to that described by Moore and Sharp (Science 250:992 (1992)). The reaction consisted of mRNA, splint, and puromycin oligonucleotide (30-P, dA27dCdCP) in a mole ratio of 0.8 : 0.9 : 1.0 and 1-2.5 units of DNA ligase per picomole of pool mRNA. Reactions were conducted for one hour at room temperature. For the construction of the pool RNA fusions, the mRNA concentration was ~ 6.6 μmolar. Following ligation, the RNA-puromycin conjugate was prepared as described above for enzymatic reactions. The precipitate was resuspended, and full length fusions were purified on denaturing PAGE and isolated by electroelution as described above. The pool RNA concentration was estimated using an extinction coefficient of 1650 O.DVμmole and the myc template 1600 O.D./μmole. In this way, 2.5 nanomoles of conjugate were generated.<br>
Preparation of dT25 Streptavidin Agarose. dT25 containing a 3' biotin<br><br>
(synthesized on bioteg phosphoramidite columns (Glen Research)) and desalted on a NAP-25 column (Pharmacia) was incubated at 1-10 uM or even 1-20 uM with a slurry of streptavidin agarose (50% agarose by volume, Pierce, Rockford, IL) for 1 hour at room temperature in TE (10 raM Tris Chloride pH 8.2, 1 mM EDTA) and washed. The binding capacity of the agarose was then estimated optically by the disappearance of biotin-dT25 from solution and/or by titration of the resin with known amounts of complementary oligonucleotide.<br>
Translation Reactions using E. coli Derived Extracts and Ribosomes. In general, translation reactions were performed with purchased kits (for example, IL coli S30 Extract for Linear Templates, Promega, Madison, WI). However, £. coji MRE600 (obtained from the ATCC, Rockville, MD) was also used to generate S30 extracts prepared according to published protocols (for example, Ellman et al., Meih. Enzymol. 202:301(1991)), as well as a ribosomal fraction prepared as described by Kudlicki et al. (Anal. Biochem. 206:389 (1992)).   The standard reaction was performed in a 50 ul volume with 20-40 uCi of 35S methionine as a marker. The reaction mixture consisted of 30% extract v/v, 9-18 mM MgCl2, 40% premix minus methionine (Promega) v/v, and 5 uM of template (e.g., 43-P). For coincubation experiments, the oligos 13-P and 25-P were added at a concentration of 5 uM. For experiments using ribosomes, 3 ul of ribosome solution was added per reaction in place of the lysate. All reactions were incubated at 37°C for 30 minutes. Templates were purified as described above under enzymatic reactions.<br>
Wheat Germ Translation Reactions. The translation reactions in Figure 8 were performed using purchased kits lacking methionine (Promega), according to the manufacturer's recommendations. Template concentrations were 4 uM for 43-P and 0.8 uM for LP77 and LP154. Reactions were performed at 25°C with 30 uCi 35S methionine in a total volume of 25 ul<br>
Reticulocyte Translation Reactions. Translation reactions were performed either with purchased kits (Novagen, Madison, WI) or using extract prepared according to published protocols (Jackson and Hunt, Meth. Enzymol. 96:50 (1983)).<br><br>
Reticulocyte-rich blood was obtained from Pel-Freez Biologicals (Rogers, AK). In both cases, the reaction conditions were those recommended for use with Red Nova Lysate (Novagen). Reactions consisted of 100 mM KG, 0.5 mM MgOAc, 2 mM DTT, 20 mM HEPES pH 7.6, 8 mM creatine phosphate, 25 uM in each amino acid (with the exception of methionine if 35S Met was used), and 40% v/v of lysate. Incubation was at 30°C for 1 hour. Template concentrations depended on the experiment but generally ranged from 50 nM to 1 uM with the exception of 43-P (Figure 6H) which was 4 uM.<br>
For generation of the randomized pool, 10 ml of translation reaction was performed at a template concentration of- 0.1μM (1.25 nanomoles of template). In addition, 32P labeled template was included in the reaction to allow determination of the amount of material present at each step of the purification and selection procedure. After translation at 30 °C for one hour, the reaction was cooled on ice for 30-60 minutes.<br>
Isolation of Fusion with dT25 Streptavidin Agarose or Oligo dT Cellulose. After incubation, the translation reaction was diluted approximately 150 fold into isolation buffer (1.0 M NaCl, 0.1 M Tris chloride pH 8.2, 10 mM EDTA, and either I mM DTT or 0.2% Triton X-100) containing greater than a 10X molar excess of dT25-biotin-streptavidin agarose whose dT2J concentration was ~ 10 uM (volume of slurry equal or greater than the volume of lysate) or oligo dT cellulose (Pharmacia), and incubated with agitation at 4°C for one hour. The agarose was then removed from the mixture either by filtration (Millipore ultrafree MC filters) or centrifugation and washed with cold isolation buffer 2-4 times. The template was then liberated from the dT25 streptavidin agarose by repeated washing with 50-100 μl aliquots of 15 mM NaOH, 1 mM EDTA at 4°C, or pure water at room temperature. The eluent was immediately neutralized in 3M NaOAc pH 5.2,10 mM spermidine, and was ethanol precipitated or used directly for the next step of purification. For the pool reaction, the total radioactivity recovered indicated approximately 50-70% of the input template was recovered.<br><br>
Isolation of Fusion with Thiopropvl Sepharose. Fusions containing cysteine can be purified using thiopropyl sepharose 6B as in Figure 13 (Pharmacia). In the experiments described herein, isolation was either carried out directly from the translation reaction or following initial isolation of the fusion (e.g., with streptavidin agarose). For samples purified directly, a ratio of 1:10 (v/v) lysate to sepharose was used. For the pool, 0.5 ml of sepharose slurry was used to isolate all of the fusion material from 5 ml of reaction mixture. Samples were diluted into a 50:50 (v/v) slurry of thiopropyl sepharose in IX TE 8.2 (10 mM Tris-Cl, 1 mM EDTA, pH 8.2) containing DNase free RNase (Boehringer Mannheim) and incubated with rotation for 1 -2 hours at 4°C to allow complete reaction. The excess liquid was removed, and the sepharose was washed repeatedly with isolation buffer containing 20 mM DTT and recovered by centrifugation or filtration. The fusions were eluted from the sepharose using a solution of 25-30 mM dithiothreitol (DTT) in 10 mM Tris chloride pH 8.2, 1 mM EDTA. The fusion was then concentrated by a combination of evaporation under high vacuum, ethanol precipitation as described above, and, if desired, analyzed by SDS-Tricine-PAGE. For the pool reaction, the total radioactivity recovered indicated approximately 1% of the template was converted to fusion.<br>
For certain applications, dT25 was added to this eluate and rotated for 1 hour at 4°C. The agarose was rinsed three times with cold isolation buffer, isolated via filtration, and the bound material eluted as above. Carrier tPvNA was added, and the fusion product was ethanol precipitated. The sample was resuspended in TE pH 8.2 containing DNase free RNase A to remove the RNA portion of the template.<br>
Immunoprecipitation Reactions.   Immunoprecipitations of peptides from translation reactions (Figure 10) were performed by mixing 4 \i\ of reticulocyte translation reaction, 2 ul normal mouse sera, and 20 ul Protein G + A agarose (Calbiochem, La Jolla, CA) with 200 ul of either PBS (58 mM Na2HP04,17 mM NaH2P04,68 mM NaCl), dilution buffer (10 mM Tris chloride pH 8.2,140 mM NaCl, 1% v/v Triton X-100), or PBSTDS (PBS + 1% Triton X-100,0.5% deoxycholate 0.1% SDS). Samples were then rotated for one hour at 4°C, followed by<br><br>
centrifugation at 2500 rpm for 15 minutes. The eluent was removed, and 10 μl of c-myc monoclonal antibody 9E10 (CalbiocUem, La JoUa, CA) and 15 μl of  Protein G + A agarose was added and rotated for 2 hours at 4°C. Samples were then washed with two 1 ml volumes of either PBS, dilution buffer, or PBSTDS.  40 ul of gel loading buffer (Calbiochem Product Bulletin) was added to the mixture, and 20 ul was loaded on a denaturing PAGE as described by Schagger and von Jagow (Anal. Biochem. 166:368 (1987)).<br>
Immunoprecipitations of fusions (as shown in Figure 11) were performed by mixing 8 ul of reticulocyte translation reaction with 300 μl of dilution buffer (10 mM Tris chloride pH 8.2,140 mM NaCl, 1% v/v Triton X-100), 15 μl protein G sepharose (Sigma), and 10 ul (1 ug) c-myc antibody 9E10 (Calbiochem), followed by rotation for several hours at 4°C. After isolation, samples were washed, treated with DNase free RNase A, labeled with polynucleotide kinase and 32P gamnia ATP, and separated by denaturing urea PAGE (Figure 11).<br>
Reverse Transcription of Fusion Pool. Reverse transcription reactions were performed according to the manufacturers recommendation for Superscript II, except that the template, water, and primer were incubated at 70 °C for only two minutes (Gibco BRL, Grand Island, NY). To monitor extension, 50 μCi alpha 32P dCTP was included in some reactions; in other reactions, reverse transcription was monitored using 5' 32P-labeled primers which were prepared using 32P #ATP (New England Nuclear, Boston, MA) and T4 polynucleotide kinase (New England Biolabs, Beverly, MA).<br>
Preparation of Protein G and Antibody Sepharose. Two aliquots of 50 ul Protein G sepharose slurry (50 % solid by volume) (Sigma) were washed with dilution buffer (10 mM Tris chloride pH 8.2, 140 mM NaCl, 0.025% NaN3,1% v/v Triton X-100) and isolated by centrifugation. The first aliquot was reserved for use as a precolumn prior to the selection matrix. After resuspension of the second aliquot in dilution buffer, 40 ug of c-myc AB-1 monoclonal antibody (Oncogene Science) was added, and the reaction incubated overnight at4°C with rotation. The antibody<br><br>
sepharose was then purified by centrifugation for 15 minutes at 1500-2500 rpm in a microcentrifuge and washed 1-2 times with dilution buffer.<br>
Selection. After isolation of the fusion and complementary strand synthesis, the entire reverse transcriptase reaction was used directly in the selection process. Two protocols are outlined here. For round one, the reverse transcriptase reaction was added directly to the antibody sepharose prepared as described above and incubated 2 hours. For subsequent rounds, the reaction is incubated ~2 hours with washed protein G sepharose prior to the antibody column to decrease the number of binders that interact with protein G rather than the immobilized antibody.<br>
To elute the pool from the matrix, several approaches may be taken. The first is washing the selection matrix with 4% acetic acid. This procedure liberates the peptide from the matrix. Alternatively, a more stringent washing (e.g., using urea or another denaturant) may be used instead or in addition to the acetic acid approach.<br>
PCR of Selected Fusions. Selected molecules are amplified by PCR using standard protocols (for example, Fitzwater and Polisky, Meth. Enzymol. 267:275 (1996); and Conrad et al., Meth. Enzymol. 267:336 (1996)), as described above for construction of the pool. Performing PCR controls ai this step may be desirable to assure that the amplified pool results from the selection performed. Primer purity is of central importance. The pairs should be amplified in the absence of input template, as contamination with pool sequences or control constructs can occur. New primers should be synthesized if contamination is found. The isolated fusions should also be subjected to PCR prior to the RT step to assure that they are not contaminated with cDNA. Finally, the number of cycles needed for PCR reactions before and after selection should be compared. Large numbers of cycles needed to amplify a given sequence (&gt;25-30 rounds of PCR) may indicate failure of the RT reaction or problems with primer pairs.<br>
SYNTHESIS ANT) TESTING OF BETA-GLOBIN FUSIONS<br>
To synthesize a p-globin fusion construct, p-globin cDNA was generated<br><br>
from 2.5 fig globin mRNA by reverse transcription with 200 pmoles of primer 18.155 (5' GTG GTA TTT GTG AGC CAG) (SEQ ID NO: 29) and Superscript reverse transcriptase (Gibco BRL) according to the manufacturer's protocol. The primer sequence was complementary to the 18 nucleotides of β-globin 5' of the stop codon. To add a T7 promoter, 20 ul of the reverse transcription reaction was removed and subjected to 6 cycles of PCR with primers 18.155 and 40.54 (5' TAA TAC GAC TCA CTA TAG GGA CAC TTG CTT TTG ACA CAA C) (SEQ ID NO: 30). The resulting "syn-P-globin" mRNA was then generated by T7 runoff transcription according to Milligan and Uhlenbeck (Methods Enzymol. 180:51 (1989)), and the RNA gel purified, electroeluted, and desalted as described herein. "LP-p-globin" was then generated from the syn-p-globin construct by ligation of that construct to 30-P according to the method of Moore and Sharp (Science 256:992 (1992)) using primer 20.262 (5' TTT TTT TTT T GTG GTA TTT G) (SEQ ID NO: 31) as the splint. The product of the ligation reaction was then gel purified, electroeluted, and desalted as above. The concentration of the final product was determined by absorbance at 260 run.<br>
These β-globin templates were then translated in vitro as described in Table 1 in a total volume of 25 ul each. Mg2+ was added from a 25 mM stock solution. All reactions were incubated at 30°C for one hour and placed at -20°C overnight. dT25 precipitable CPM's were then determined twice using 6 ul of lysate and averaged minus background.<br><br><br><br>
To prepare the samples for gel analysis, 6 ul of each translation reaction was mixed with 1000 ul of Isolation Buffer (1 M NaCl, 100 mM Tris-Cl pH 8.2, 10 mM EDTA, 0.1 mM DTT), 1 ul RNase A (DNase Free, Boehringer Mannheim), and 20 ul of 20 uM dT25 streptavidin agarose. Samples were incubated at 4°C for one hour with rotation. Excess Isolation Buffer was removed, and the samples were added to a Millipore MC filter to remove any remaining Isolation Buffer. Samples were then washed four times with 50 μl of H20, and twice with 50 μl of 15 mM NaOH, 1 mM EDTA. The sample (300 ul) was neutralized with 100 μl TE pH 6.8 (10 mM Tris-Cl, 1 mM EDTA), 1 ul of 1 mg/ml RNase A (as above) was added, and the samples were incubated at 37°C. 10 μl of 2X SDS loading buffer (125 mM Tris-Cl pH 6.8, 2% SDS, 2% p-mercaptoethanol 20% glycerol, 0.001% bromphenol blue) was then added, and the sample was lyophilized to dryness and resuspended in 20 ul H20 and 1% p-mercaptoethanol. Samples were then loaded onto a peptide resolving gel as described by Schagger and von Jagow (Analytical Biochemistry 166:368 (1987)) and visualized by autoradiography.<br>
The results of these experiments are shown in Figures 15A and 15B. As indicated in Figure 15 A, 35S-methionine was incorporated into the protein portion of the syn-P-globin and LP-P-globin fusions. The protein was heterogeneous, but one strong band exhibited the mobility expected for P-globin mRNA. Also, as shown in Figure 15B, after dT25 isolation and RNase A digestion, no 35S-labeled material remained in the syn-p-globin lanes (Figure 15B, lanes 2-4). In contrast, in the LP-P-globin lanes, a homogeneously sized 35S-labeled product was observed.<br><br>
These results indicated that, as above, a fusion product was isolated by oligonucleotide affinity chromatography only when the template contained a 3' puromycin. This was confirmed by scintillation counting (see Table 1). The material obtained is expected to contain the 30-P linker fused to some portion of p-globin. The fusion product appeared quite homogeneous in size as judged by gel analysis. However, since the product exhibited a mobility very similar to natural P-globin (Figures 15A and 15B, control lanes), it was difficult to determine the precise length of the protein portion of the fusion product.<br>
FURTHER OPTIMIZATION OF RNA-PROTEIN FUSION FORMATION Certain factors have been found to further increase the efficiency of formation of RNA-peptide fusions. Fusion formation, i.e., the transfer of the nascent peptide chain from its tRNA to the puromycin moiety at the 3' end of the mRNA, is a slow reaction that follows the initial, relatively rapid translation of the open reading frame to generate the nascent peptide. The extent of fusion formation may be substantially enhanced by a post-translational incubation in elevated Mg2* conditions (preferably, in a range of 50-100 mM) and/or by the use of a more flexible linker between the mRNA and the puromycin moiety. In addition, long incubations (12-48 hours) at low temperatures (preferably,<br>
-20°C) also result in increased yields of fusions with less mRNA degradation than that which occurs during incubation at 30°C. By combining these factors, up to 40% of the input mRNA may be converted to mRNA-peptide fusion products, as shown below.<br>
Synthesis of mRNA-Puromycin Conjugates. In these optimization experiments, puromycin-containing linker oligonucleotides were ligated to the 3' ends of mRNAs using bacteriophage T4 DNA ligase in the presence of complementary DNA splints, generally as described above. Since T4 DNA ligase prefers precise base-pairing near the ligation junction and run-off transcription products with T7, T3, or SP6 RNA polymerase are often heterogeneous at their 3" ends (Nucleic Acids<br><br>
Research 15:8783 (1987)), only those RNAs containing the correct 3'-terminal nucleotide were efficiently ligated. When a standard DNA splint was used, approximately 40% of runoff transcription products were ligated to the puromycin oligo. The amount of ligation product was increased by using excess RNA, but was not increased using excess puromycin oligonucleotide. Without being bound to a particular theory, it appeared that the limiting factor for ligation was the amount of RNA which was fully complementary to the corresponding region of the DNA splint.<br>
To allow ligation of those transcripts ending with an extra non-templated nucleotide at the 3' terminus (termed "N+1 products"), a mixture of the standard DNA splint with a new DNA splint containing an additional random base at the ligation junction was used. The ligation efficiency increased to more than 70% for an exemplary myc RNA template (that is, RNA 124) in the presence of such a mixed DNA splint.<br>
In addition to this modified DNA splint approach, the efficiency of mRNA-puromycin conjugate formation was also further optimized by taking into account the following three factors. First, mRNAs were preferably designed or utilized which lacked 3'-termini having any significant, stable secondary structure that would interfere with annealing to a splint oligonucleotide. In addition, because a high concentration of salt sometimes caused failure of the ligation reaction, thorough desalting of the oligonucleotides using NAP-25 columns was preferably included as a step in the procedure. Finally, because the ligation reaction was relatively rapid and was generally complete within 40 minutes at room temperature, significantly longer incubation periods were not generally utilized and often resulted in unnecessary degradation of the RNA.<br>
Using the above conditions, mRNA-puromycin conjugates were synthesized as follows. Ligation of the myc RNA sequence (RNA124) to the puromycin-comaining oligonucleotide was performed using either a standard DNA splint (e.g., 5'-TTTlTTTTTTAGCGCAAGA) (SEQ ID NO: 32) or a splint containing a random base (N) at the ligation junction (e.g., 5'-<br><br>
TTTTTTTTTNAGCGCAAGA) (SEQ ID NO: 33). The reactions consisted of 11RNA, the DNA splint, and the puromycin oligonucleotide in a molar ratio of 1.0 : .5-2.0: 1 .0. An alternative molar ratio of 1.0: 1.2 :1.4 may also be utilized. A nixture of these components was first heated at 94°C for 1 minute and then cooled on ce for 15 minutes. Ligation reactions were performed for one hour at room emperature in 50 mM Tris-HCl (pH 7.5), 10 mM MgCl2, 10 mM DTT, 1 mM ATP, :5 μg/ml BSA, 15 μM puromycin oligo, 15 μM mRNA, 22.5-30 uM DNA splint, INasin inhibitor (Promega) at 1 U/ul, and 1.6-2.5 units of T4 DNA ligase per 'icomole of puromycin oligo. Following incubation, EDTA was added to a final oncentration of 30 mM, and the reaction mixtures were extracted with  henol/chloroform. Full length conjugates were purified by denaturing PAGE, solated by electroelution, and desalted.<br>
General Reticulocyte Translation Conditions. In addition to improving the ynthesis of the mRNA-puromycin conjugate, translation reactions were also further 'ptimized as follows. Reactions were performed in rabbit reticulocyte lysates from ifferent commercial sources (Novagen, Madison, WI; Amersham, Arlington Heights, L; Boehringer Mannheim, Indianapolis, IN; Ambion, Austin, TX; and Promega, Madison, WI). A typical reaction mixture (25 ul final volume) consisted of 20 mM IEPES pH 7.6,2 mM DTT, 8 mM creatine phosphate, 100 mM KC1,0.75 mM dg(OAc)2) 1 mM ATP, 0.2 mM GTP, 25 uM of each amino acid (0.7 uM methionine f35S-Met was used), RNasin at 1 U/ul, and 60% (v/v) lysate. The final concentration f template was in the range of 50 nM to 800 nM. For each incubation, all omponents except lysate were mixed carefully on ice, and the frozen lysate was lawed immediately before use. After addition of lysate, the reaction mixture was rixed thoroughly by gentle pipetting and incubated at 30°C to start translation. The ptimal concentrations of Mg2* and K* varied within the ranges of 0.25 mM - 2 mM nd 75 mM - 200 mM, respectively, for different mRNAs and was preferably etermined in preliminary experiments. Particularly for poorly translated mRNAs, be concentrations of hemin, creatine phosphate, tRNA, and amino acids were also<br><br>
sometimes optimized. Potassium chloride was generally preferred over potassium acetate for fusion reactions, but a mixture of KG and KOAc sometimes produced better results.<br>
After translation at 30°C for 30 to 90 minutes, the reaction was cooled on ice for 40 minutes, and Mg2+ or K+ were added. The final concentration of Mg2^ added at this step was also optimized for different mRNA templates, but was generally in the range of 50 mM to 100 mM (with 50 mM being preferably used for pools of mixed templates). The amount of added K+ was generally in the range of 125 mM-1.5 M. For a Mg2+ reaction, the resulting mixture was preferably incubated at -20°C for 16 to 48 hours, but could be incubated for as little as 12 hours. If IT or Mg27K+ were added, the mixture was incubated at room temperature for .one hour.<br>
To visualize the labeled fusion products, 2 μl of the reaction mixture was mixed with 4 μl loading buffer, and the mixture was heated at 75 °C for 3 minutes. The resulting mixture was then loaded onto a 6% glycine SDS-polyacrylamide gel (for 32P-labeled templates) or an 8% tricine SDS-polyacrylamide gel (for 35S-Met-labeled templates). As an alternative to this approach, the fusion products may also be isolated using dT25 streptavidin agarose or thiopropyl sepharose (or both), generally as described herein.<br>
To remove the RNA portion of the RNA-linker-puromycin-peptide conjugate for subsequent analysis by SDS-PAGE, an appropriate amount of EDTA was added after post-translational incubation, and the reaction mixture was desalted using a microcon-10 (or microcon-30) column. 2 μl of the resulting mixture (approximately 25 μl total) was mixed with 18 μl of RNase H buffer (30 mM Tris-HC1, pH 7.8, 30 mM (NH4)2S04, 8 mM MgCl2, 1.5 mM β-mercaptoethanol, and an appropriate amount of complementary DNA splint), and the mixture was incubated at 4°C for 45 minutes. RNase H was then added, and digestion was performed at 37CC<br>
for 20 minutes.<br>
Quality of Puromvcin Oligo. The quality of the puromycin oligonucleotide was also important for the efficient generation of fusion products.<br><br>
The coupling of 5"-DMT, 2'-succinyl, N-trifluoroacetyl puromycin with CPG was not as efficient as the coupling of the standard nucleotides. As such, the coupling reaction was carefully monitored to avoid the formation of CPG with too low a concentration of coupled puromycin, and unreacted amino groups on the CPG were fully quenched to avoid subsequent synthesis of oligonucleotides lacking a 3'-terminal puromycin. It was also important to avoid the use of CPG containing very fine mesh particles, as these were capable of causing problems with valve clogging during subsequent automated oligonucleotide synthesis steps.<br>
In addition, the synthesized puromycin oligo was preferably tested before large scale use to ensure the presence of puromycin at the 3' end. In our experiments, no fusion was detected if puromycin was substituted with a deoxyadenosine containing a primary amino group at the 3' end. To test for the presence of 3' hydroxyl groups (i.e., the undesired synthesis of oligos lacking a 3'-terminal puromycin), the puromycin oligo may first be radiolabeled (e.g., by 5'-phosphorylation) and then used as a primer for extension with terminal deoxynucleotidyl transferase. In the presence of a 3'-terminal puromycin moiety, no extension product should be observed.<br>
Time Course of Translation and Post-Transtational Incubation. The translation reaction was relatively rapid and was generally completed within 25 minutes at 30°C. The fusion reaction, however, was slower. When a standard linker (dA27dCdCP) was used at 30°C, fusion synthesis reached its maximum level in an additional 45 minutes. The post-translational incubation could be carried out at lower temperatures, for example, room temperature, 0°C, or -20°C. Less degradation of the mRNA template was observed at -20 °C, and the best fusion results were obtained after incubation at -20°C for 2 days.<br>
The Effect of Me2* or K* Concentration. A high concentration of Mg2* or K+ in the post-translational incubation greatly stimulated fusion formation. For example, for the myc RNA template described above, a 3-4 fold stimulation of fusion formation was observed using a standard linker (dA27dCdCP) in the presence of 50<br><br>
mM Mg2+ during the 16 hour incubation at -20°C (Figure 17, compare lanes 3 and 4). Efficient fusion formation was also observed using a post-translational incubation in the presence of a 50-100 mM Mg2+ concentration when the reactions were carried out at room temperature for 30-45 minutes. Similarly, addition of 250 - 500 mM K* increased fusion formation by greater than 7 fold relative to the no added K+ control. Optimum K+ concentrations were generally between 300 mM and 600 mM (500 mM for pools). Post-translational addition of NH4C1 also increased fusion formation. The choice of OAc" vs. CI" as the anion did not have a profound effect on fusion formation.<br>
Linker Length and Sequence. The dependence of the fusion reaction on the length of the linker was also examined. In the range between 21 and 30 nucleotides (n= 18-27), little change was seen in the efficiency of the fusion reaction (as described above). Similar results were obtained for linkers of 19 and 30 nucleotides, and greatest fusion formation was observed for linkers of 25 nucleotides (Figure 23). Shorter linkers (e.g., 13 or 16 nucleotides in length) and longer linkers (e.g., linkers greater than 40 nucleotides in length) resulted in much lower fusion formation. In addition, although particular linkers of greater length (that is, of 45 nucleotides and 54 nucleotides) also resulted in somewhat lower fusion efficiences, it remains likely that yet longer linkers may also be used to optimize the efficiency of the fusion reaction.<br>
With respect to linker sequence, substitution of deoxyribonucleotide residues near the 3' end with ribonucleotide residues did not significantly change the fusion efficiency. The dCdCP (or rCrCP) sequence at the 3' end of the linker was, however, important to fusion formation. Substitution of dCdCP with dUdUP reduced the efficiency of fusion formation significantly.<br>
Linker Flexibility. The dependence of the fusion reaction on the flexibility of the linker was also tested. In these experiments, it was determined that the fusion efficiency was low if the rigidity of the linker was increased by annealing with a complementary oligonucleotide near the 3' end. Similarly, when a more flexible linker (for example, dA2,C,C,C,dAdCdCP, where C9 represents HO(CH2CH20)3P02)<br><br>
was used, the fusion efficiency was significantly improved. Compared to the standard linker (dA„dCdCP), use of the more flexible linker (dA21C9,C9,C9,dAdCdCP) improved the fusion efficiency for RNA124 more than 4-fold (Figure 17, compare lanes 1 and 9). In addition, in contrast to the template with the standard linker whose post-translation fusion proceeded poorly in the absence of a high concentration of Mg2+ (Figure 17, lane 3 and 4), the template with the flexible linker did not require elevated Mg2~ to produce a good yield of fusion product in an extended post-translational incubation at -20°C (Figure 17, compare lanes 11 and 12). This linker, therefore, was very useful if post-translational additions of high concentrations of Mg2+ were not desired. In addition, the flexible linker also produced optimal fusion yields in the presence of elevated Mg2+.<br>
Quantitation of Fusion Efficiency. Fusion efficiency may be expressed as either the fraction of translated peptide converted to fusion product, or the fraction of input template converted to fusion product. To determine the fraction of translated peptide converted to fusion product, 35S-Met labeling of the translated peptide was utilized. In these experiments, when a dA27dCdCP or dA27rCrCP linker was used, about 3.5% of the translated peptide was fused to its mRNA after a 1 hour translation incubation at 30°C. This value increased to 12% after overnight incubation at -20°C. When the post-translational incubation was carried out in the presence of a high concentration of Mg2+, more than 50% of the translated peptide was fused to the template.<br>
For a template with a flexible linker, approximately 25% of the translated peptide was fused to the template after 1 hour of translation at 30°C This value increased to over 50% after overnight incubation at -20°C and to more than 75% if the post-translational incubation was performed in the presence of 50 mM Mg2*.<br>
To determine the percentage of the input template converted to fusion product, the translations were performed using 32P-labeled mRNA-linker template. When the flexible linker was used and post-translational incubation was performed at -20°C without addition of Mg2*, about 20%, 40%, 40%, 35%, and 20% of the input<br><br>
template was converted to mRNA-peptide fusion when the concentration of the input RNA template was 800,400,200, 100, and 50 nM, respectively (Figure 18). Similar results were obtained when the post-translational incubation was performed in the presence of 50 mM Mg2+. The best results were achieved using lysates obtained from Novagen, Amersham, or Ambion (Figure 19).<br>
The mobility differences between mRNAs and mRNA-peptide fusions as measured by SDS-PAGE may be very small if the mRNA template is long. In such cases, the template may be labeled at the 5' end of the linker with 32P (for example, using [32P] yATP and T4 polynucleotide kinase prior to ligation of the mRNA-puromycin conjugate). The long RNA portion may then be digested with RNase H in the presence of a complementary DNA splint after translation/incubation, and the fusion efficiency determined by quantitation of the ratio of unmodified linker to linker-peptide fusion. Compared to RNase A digestion, which produces 3'-P and 5'-OH, this approach has the advantage that the32P at the 5' end of the linker is not removed.<br>
For RNase H treatment, EDTA was added after posttranslational incubation to disrupt ribosomes, and the reaction mixture was desalted using a microcon-10 (or microcon-30) column. 2 ul of the resulting mixture was combined with 18 ul of RNase H buffer (30 mM Tris-HCl, pH7.8,30 mM (NH4)2S04, 8 mM MgCl2,1.5 mM p-mercaptoethanol, and an excess of complementary DNA splint) and incubated at 4°C for 45 minutes. RNase H was then added, and digestion was performed at 37°C for 20 minutes.<br>
Intramolecular vs. Intermolecular Fusion During Post-Translational Incubation. In addition to the above experiments, we tested whether the fusion reaction that occurred at -20 °C in the presence of Mg2+ was intra- or intermolecular in nature. Free linker (dA27dCdCP or dA2lC9C9,C9,dAdCdCP, where C, is -0(CH2CH20)3PO-2) was coincubated with a template containing a DNA linker, but without puromycin at the 3' end, under the translation and post-translational incubation conditions described above. In these experiments, no detectable amount<br><br>
(that is less than 2% of the normal level) of JSS-Met was incorporated into linker-peptide product, suggesting that post-translational fusion occurred primarily between the nascent peptide and the mRNA bound to the same ribosome.<br>
In additional experiments, co-incubations were carried out with templates and puromycin oligonucleotides whose fusion products and cross-products (templates fused to the wrong protein) could be separated by electrophoresis. No cross-product formation was observed for any template and linker combination examined. In these experiments, fusion cross-products could form via two different trans mechanisms: (1) reaction of free templates or linkers with the peptide in a peptide-mRNA-ribosome complex or (2) reaction of the template of one complex with the peptide in another. One particular example of testing the latter possibility is shown in Figure 24. There, the lambda protein phosphatase (XPPase) template, which synthesizes a protein 221 amino acids long, was coincubated with the myc template, which generates a 33 amino acid peptide. By themselves, both templates demonstrate fusion formation after post-translation incubation. When mixed together, only the individual fusion products were observed. No cross-products resulting from fusion of the XPPase protein with the myc template were seen. Similar experiments showed no cross-product formation with several different combinations: the myc template + the single codon template, a 20:1 ratio of the standard linker + the myc template, and the flexible linker + the myc template. These experiments argued strongly against both possible mechanisms of trans fusion formation.<br>
The effect of linker length on fusion formation was also consistent with an in cis mechanism. Reduction of the linker length from 19 to 13 nucleotides resulted in an abrupt decrease in the amount of fusion product expected if the chain could no longer reach the peptidyl transferase center from the decoding site (Figure 23). However, this effect could also be due to occlusion of the puromycin within the ribosome if the trans mechanism dominated (e.g., if ribosome-bound templates formed fusion via a trans mechanism). The decrease in fusion formation with longer linkers again argues against this type of reaction, as no decrease should be seen for the<br><br>
trans reaction once the puromycin is free of the ribosome.<br>
Optimization Results. As illustrated above, by using the flexible linker and/or performing the post-translational incubation in the presence of a high concentration of Mg2, fusion efficiencies were increased to approximately 40% of input mRNA. These results indicated that as many as 1014 molecules of mRNA-peptide fusion could be generated per ml of in vitro translation reaction mix, producing pools of mRNA-peptide fusions of very high complexity for use in in vitro selection experiments.<br>
SELECTIVE ENRICHMENT OF RNA-PROTEIN FUSIONS<br>
 	We have demonstrated the feasibility of using RNA-peptide fusions in<br>
selection and evolution experiments by enriching a particular RNA-peptide fusion from a complex pool of random sequence fusions on the basis of the encoded peptide. In particular, we prepared a series of mixtures in which a small quantity of known sequence (in this case, the long myc template, LP154) was combined with some amount of random sequence pool (that is, LP 160). These mixtures were translated, and the RNA-peptide fusion products selected by oligonucleotide and disulfide affinity chromatography as described herein. The myc-template fusions were selectively immunoprecipitated with anti-myc monoclonal antibody (Figure 16A). To measure the enrichment obtained in this selective step, aliquots of the mixture of      cDNA/mRNA-peptide fusions from before and after the immunoprecipitation were amplified by PCR in the presence of a radiolabeled primer. The amplified DNA was digested with a restriction endonuclease that cut the myc template sequence but not the pool (Figures 16B and 16C). Quantitation of the ratio of cut and uncut DNA indicated that the myc sequence was enriched by 20-40 fold relative to the random       library by immunoprecipitation.<br>
These experiments were carried out as follows.<br>
Translation Reactions. Translation reactions were performed generally as described above. Specifically, reactions were performed at 30°C for one hour<br><br>
according to the manufacturer's specifications (Movagen) and frozen overnight at -20 °C.   Two versions of six samples were made, one containing 35S methionine and one containing cold methionine added to afinal concentration of 52 μM. Reactions 1 -6 contained the amounts of templates described in Table 2. All numbers in Table 2 represent picomoles of template per 25 μl reaction mixture.<br>
TABLE 2<br>
Template Ratios Used in Doped Selection<br><br><br><br>
was w<br>
Preparation of dT:25, Streptavidin Agarose. Streptavidin agarose (Pierce) ashed three times with TE 8.2 (10 mM Tris-Cl pH 8.2, 1 mM EDTA) and resuspended as a 1:1 (v/v) slurry in TE 8.2. 3' biotinyl T25 synthesized using Bioteg CPG (Glen Research) was then added to the desired final concentration (generally 10 or 20 μM), and incubation was carried out with agitation for 1 hour. The dT25 streptavidin agarose was then washed three times with TE 8.2 and stored at 4°C until<br>
use.<br>
Purification of Templates from Translation Reactions. To purify templates from translation reactions, 25 ul of each reaction was removed and added to 7.5 ml of Isolation Buffer (1 M NaCl, 100 mM Tris-Cl pH 8.2,10 mM EDTA, 0.1 mM DTT) and 125 μl of 20 uM dT25 streptavidin agarose. This solution was incubated at 4°C for one hour with rotation. The tubes were centrifuged and the eluent removed. One ml of Isolation Buffer was added, the slurry was resuspended, and the mixtures were transferred to 1.5 ml microcentrifuge tubes. The samples were then washed four times with 1 ml aliquots of ice cold Isolation Buffer. Hot and cold samples from identical reactions were then combined in a Millpore MC filter unit and were eluted<br><br>
from (he dT23 agarose by washing with 2 volumes of 100 ul FLO, 0.1 mM DTT, and 2 volumes of 15 mM NaOH, 1 mM EDTA (4°C) followed by neutralization.<br>
To this eluent was added 40 ul of a 50% slurry of washed thiopropyl sepharose (Pharmacia), and incubation was carried out at 4°C with rotation for 1 hour. The samples were then washed with three 1 ml volumes of TE 8.2 and the eluent removed. One ul of 1 M DTT was added to the solid (total volume approximately 20-30 ul), and the sample was incubated for several hours, removed, and washed four times with 20 μ1 H20 (total volume 90 ul). The eluent contained 2.5 mM thiopyridone as judged by UV absorbance. 50 μl of this sample was ethanol precipitated by adding 6 ul 3 M NaOAc pH 5.2, 10 mM spermine, 1 ul glycogen (10 mg/ml, Boehringer Mannheim), and 170 μl 100% EtOH, incubating for 30 minutes at -70°C, and centrifuging for 30 minutes at 13,000 rpm in a microcentrifuge.<br>
Reverse Transcriptase Reactions. Reverse transcription reactions were performed on both the ethanol precipitated and the thiopyridone eluent samples as follows. For the ethanol precipitated samples, 30 ul of resuspended template, H20 to 48 ul, and 200 picomoles of primer 21.103 (SEQ ID NO: 22) were annealed at 70°C for 5 minutes and cooled on ice. To this sample, 16 ul of first strand buffer (250 mM Tris-Cl pH 8.3, 375 mM KC1, and 15 mM MgCl2; available from Gibco BRL, Grand Island, NY), 8 ul 100 mM DTT, and 4 μl 10 mM NTP were added and equilibrated at 42°C, and 4 ul Superscript II reverse transcriptase (Gibco BRL, Grand Island, NY) was added. H20 (13 μl) was added to the TP sepharose eluent (35 μl), and reactions were performed as above. After incubation for one hour, like numbered samples were combined (total volume 160 μ1). 10 p.1 of sample was reserved for the PCR of each unselected sample, and 150 μl of sample was reserved for immunoprecipitation.<br>
Immunoprecipitation. To carry out immunoprecipitations, 170 ul of reverse transcription reaction was added to 1 ml of Dilution Buffer (10 mM Tris'Cl, pH 8.2, 140 mM NaCl, 1% v/v Triton X-100) and 20 μl of Protein G/A conjugate (Calbiochem, La Jolla, CA), and precleared by incubation at 4°C with rotation for 1 hour. The eluent was removed, and 20 μl G/A conjugate and 20 μl of monoclonal<br><br>
antibody (2 μg, 12 picomoles) were added, and the sample incubated with rotation for two hours at 4°C. The conjugate was precipitated by microcentrifugation at 2500 rpm for 5 minutes, the eluent removed, and the conjugate washed three times with 1 ml aliquots of ice cold Dilution Buffer. The sample was then washed with 1 ml ice cold 10 mM Tris-Cl, pH 8.2, 100 mM NaCl. The bound fragments were removed using 3 volumes of frozen 4% HOAc, and the samples were lyophilized to dryness.<br>
PCR of Selected and Unselected Samples. PCR reactions were carried out by adding 20 ul of concentrated NH4OH to 10 μl of the unselected material and the entirety of the selected material and incubating for 5 minutes each at 55 °C, 70°C, and 90°C to destroy any RNA present in the sample. The samples were then evaporated to dryness using a speedvac. 200 μl of PCR mixture (1 uM primers 21.103 and 42.108,200 μM dNTP in PCR buffer plus Mg2+ (Boehringer Mannheim), and 2 ul of Taq polymerase (Boehringer Mannheim)) were added to each sample. 16 cycles of PCR were performed on unselected sample number 2, and 19 cycles were performed on all other samples.<br>
Samples were then amplified in the presence of 5' 32P-labeled primer 21.103 according to Table 3, and purified twice individually using Wizard direct PCR purification kits (Promega) to remove all primer and shorter fragments.<br>
TABLE 3 Amplification of Selected and Unselected PCR Samples<br><br><br>
5	selected	20 μl	7<br>
6	selected	20μ1	7<br><br>
Restriction Digests. 32P labeled DNA prepared from each of the above PCR reactions was added in equal amounts (by cpm of sample) to restriction digest reactions according to Table 4. TJie toial volume of each reaction was 25 μl 0.5 μl of Alwnl (5 units, New England Biolabs) was added to each reaction. Samples were incubated at 37°C for 1 hour, and the enzyme was heat inactivated by a 20 minute incubation at 65 °C. The samples were then mixed with 10 μ1 denaturing loading buffer (1 ml ultrapure formamide (USB), 20 ul 0.5 M EDTA, and 20 ul 1 M NaOH), heated to 90°C for 1 minute, cooled, and loaded onto a 12% denaturing polyacrylamide gel containing 8M urea. Following electrophoresis, the gel was fixed with 10% (v/v) HO Ac, 10% (v/v) MeOH, H20.<br>
TABLE 4<br>
Restriction Digest Conditions w/ Alwnl<br><br>
Sample<br>
1 2 3 4 5 6 1 ?<br>
3 4 5 6<br>
Quantitation of Digest. The amount ofmyc versus pool DNA present in a sample was quantitated using a phosphorimager (Molecular Dynamics). The amount of material present in each band was determined as the integrated volume of identical<br><br>
rectangles drawn around the gel bands. The total cpm present in each band was calculated as the volume minus the background. Three values of background were used: (1) an average of identical squares outside the area where counts occurred on the gel; (2) the cpm present in the unselected pool lane where the myc band should appear (no band appears at this position on the gel); and (3) a normalized value that reproduced the closest value to the 10-fold template increments between unselected lanes. Lanes 2, 3, and 4 of Figures 16B and 16C demonstrate enrichment of the target versus the pool sequence. The demonstrable enrichment in lane 3 (unselected/selected) yielded the largest values (17, 43, and 27 fold using methods 1-3, respectively) due to the optimization of the signal to noise ratio for this sample. These results are summarized in Table 5.<br>
TABLE 5<br>
Enrichment of Myc Template vs. Pool<br>
Method	Lane 2 (20)	Lane 3 (200)	Lane 4 (2000)<br>
1	7.0	16.6	5.7<br>
2	10.4	43	39<br>
3	8.7	27	10.2<br>
In a second set of experiments, these same PCR products were purified once using Wizard direct PCR purification kits, and digests were quantitated by method (2) above. In these experiments, similar results were obtained; enrichments of 10.7,38, and 12 fold, respectively, were measured for samples equivalent to those in lanes 2, 3, and 4 above.<br>
IN VITRO SELECTION FROM A LARGE RNA-PEPTIDE FUSION LIBRARY In another experiment demonstrating selection of desired fusion molecules from large libraries, a repertoire of 2 x 1013 randomized RNA-peptide fusions was generated using a modification of the method described above, A DNA library was<br><br>
generated that contained 27 randomized codons based on the synthesis scheme 5'-(NNS)27-3' (where N represents equimolar A, G, C and T, and S either G or C). Each NNS codon was a mixture of 32 triplets that included codons for all 20 natural amino acids. The randomized region was flanked by two primer binding sites for reverse transcription and PCR, as well as sequences encoding the T7 promoter and an initiation site for translation. RNA, synthesized by in vitro transcription, was modified by template-directed ligation to an oligonucleotide linker containing puromycin on its 3' terminus, dA27dCdC-P.<br>
Purified ligated RNA was |n vitro translated in rabbit reticulocyte extract to generate RNA-protein fusions as follows: a 123-mer DNA PP.01 (5'-AGC TTT TGG TGC TTG TGC ATC (SNN)27 CTC CTC GCC CTT GCT CAC CAT-3', N = A, G, C, T; S = C, G) (SEQ ID NO: 34) was synthesized and purified on a 6% denaturing polyacrylamide gel. 1 nmol of the purified DNA (6 x 10u molecules) was amplified by 3 rounds of PCR (94°C, 1 minute; 65°C, 1 minute; 72°C, 2 minutes) using 1 uM primers P1F (5'-AGC TTT TGG TGC TTG TGC ATC-3') (SEQ ID NO: 35) and PT7 (5'-TAA TAC GAC TCA CTA TAG GGA CAA TTA CTA TTT ACA ATT ACA ATG GTG AGC AAG GGC GAG GAG-3') (SEQ ID NO: 36) in a total volume of 5 ml (50 mM K.C1, 10 mM Tris-HCl pH 9.0, 0.1 % Triton X-100, 2.5 mM MgCl2, 0.25 mM dNTPs, 500 Units Promega Taq Polymerase). After precipitation, the DNA was redissolved in 100 μl TE (10 mM Tris-HCl pH 7.6, 1 mM EDTA pH 8.0). DNA (60 pi) was transcribed into RNA in a reaction (1 ml) using the Megashortscript In vitro Transcription kit from Ambion. The reaction was extracted twice with phenol/CHCl3 and excess NTPs were removed by purification on a NAP-25 column (Pharmacia). The puromycin containing linker 30-P (5'-dA27dCdCP) was synthesized as described herein and added to the 3'-end of the RNA library by template-directed ligation. RNA (25 nmol) were incubated with equimolar amounts of linker and splint (5'-TTT TTT TTT TNA GCT TTT GGT GCT TG 3') (SEQ ID NO: 37) in a reaction (1.5 ml) containing T4 DNA ligase buffer (Promega) and 1200 Units T4 DNA ligase (Promega). After incubation at room temperature for 4 hours,<br><br>
ligated RNA was separated from unligated RNA on a 6 % denaturing polyacrylamide gel, eluted from the gel, and rcdissolved (200 μl ddH20). To generate mRNA-peptide fusion molecules, ligated RNA (1.25 nmol) was translated in a total volume of 7.5 ml using the Rabbit Reticulocyte IVT kit from Ambion in the presence of 3.7 μCi 35S-methionine. After incubation (30 minutes at 30°C), the reaction was brought to a final concentration of 530 mM KC1 and 150 mM MgCl2 and incubated for a further 1 hour at room temperature. Fusion formation was enhanced about 10-fold by this addition of 530 mM KC1 and 150 mM MgCl2 after the translation reaction was completed.<br>
Using this improved method, about 1013 purified fusion molecules per ml were obtained. RN A-peptide fusions were purified from the crude translation reaction by oligonucleotide affinity chromatography, and the RNA portion of the joint molecules was reverse transcribed prior to the selection step using RNase H-free reverse transcriptase as follows. Translated fusion products were incubated with dT25 cellulose (Pharmacia) in incubation buffer (100 mM Tris-HCl pH 8.0,10 mM EDTA pH 8.0, 1 M NaCl and 0.25 % Triton X-100; 1 hour at 4°C). The cellulose was isolated by filtration and washed with incubation buffer, followed by elution of the fusion products with ddH20. The RNA was reverse transcribed (25 mM Tris-HCl pH 8.3, 75 mM KC1, 3 mM MgCl2, 10 mM DTT, and 0.5 mM dNTPs with 2 Units of Superscript II Reverse Transcriptase (Gibco BRL)) using a 5-fold excess of splint as primer.<br>
To explore the power of the RNA-protein fusion selection technology, the library was used to select peptides that bound to a c-myc monoclonal antibody using immunoprecipitation as the selection tool. Five rounds of repeated selection and amplification resulted in increased binding of the population of fusion molecules to the anti-myc monoclonal antibody 9E10(Evan et al., Mol. Cell Biol. 5:3610 (1985)). Less than 1% of the library applied to the selection step was recovered by elution in each of the first three rounds of selection; however, about 10% of the library bound to the antibody and was eluted in the fourth selection round. The proportion of binding<br><br>
molecules increased to 34% in the fifth round of selection. This result agreed well with the percentage of a wild type c-myc fusion construct that bound to the anti-myc antibody under these conditions (35%). In the sixth round of selection, no further enrichment was observed, and fusion molecules from the fifth and sixth rounds were used for characterization and sequence determination of the selected peptides.<br>
To carry out these experiments, the starting library of 2 x 1013 molecules was incubated with a 12-fold excess of the c-myc binding antibody 9E10 (Chemicon) in selection buffer (IX PBS, 0.1 % BSA; 0.05 % Tween) for 1 hour at 4°C. The peptide fusion - antibody complexes were precipitated by adding protein A -sepharose. After additional incubation for 1 hour at 4°C, the sepharose was isolated by filtration, and the flow through (FT) was collected. The sepharose was washed with five volumes of selection buffer (Wl - W5) to remove non-specific binders and binding peptides were eluted with four volumes of 15 mM acetic acid (El - E4). The cDNA portion of the eluted fusion molecules was amplified by PCR, and the resulting DNA was used to generate an enriched population of fusion products, which was submitted to further rounds of selection. In order to remove peptides with affinity for protein A - sepharose from the pool, a pre-selection en protein A - sepharose was introduced in the second round of selection. The progress of the selection was monitored by determining the percentage of 35:,S-labeled RNA-peptide fusion that was eluted from the immunoprecipitate with acetic acid. These results are shown in Figure 20.<br>
The pool of selected peptides was demonstrated to specifically bind the anti-myc antibody used for selection. Binding experiments with round 6 unfused peptides showed similar binding to the antibody compared to fused peptide, indicating that the nucleic acid portion of the fusion molecules was not needed for binding (data not shown).<br>
Fusion products from the sixth round of selection were evaluated under three different immunopreciptation conditions, as follows: (1) without the anti-myc antibody, (2) with the anti-integrin monoclonal antibody ASC-3 which is of the same<br><br>
isotype, but does not bind the myc epitope, and (3) with the anti-myc antibody 9E10. Experiments were carried out by incubating 35S-labeled RN A-peptide fusion products from the sixth round of selection (0.2 pmol) in selection buffer (IX PBS, 0.1 % BSA, 0.05 % Tween) for 1 hour at 4°C either with anti-myc monoclonal Antibody 9E10 (100 pmol), with anti-integrin p4 monoclonal antibody ASC-3 (100 pmol; Chemicon), or without antibody. Peptide fusion-antibody complexes were precipitated with Protein A-sepharose. After washing the sepharose with five volumes of selection buffer, bound species were eluted by the addition of 15 mM acetic acid.<br>
No significant binding could be detected in the control experiment without antibody, showing that the selected peptides did not bind nonspecifically to protein A - agarose. In addition, no binding to the anti-integrin monoclonal antibody was observed, indicating that the selected peptides were specific for the anti-myc antibody. A competition experiment with synthetic myc peptide was performed to determine whether the selected peptide fusion molecules interacted with the antigen-binding site of the anti-myc antibody 9E10. When 35S-labeled fusion molecules from the sixth round of selection were incubated with anti-myc monoclonal antibody and increasing amounts of unlabeled myc peptide, the percentage of binding molecules decreased. These results are shown in Figure 21. In this figure, 0.2 pmol 35S-labeled RN A-peptide fusion products from the sixth round of selection were incubated with 100 pmol anti-myc monoclonal antibody 9E10 in the presence of 0,0.2,1,2, or 10 nmol synthetic myc peptide (Calbiochem). The peptide fusion - antibody complexes were precipitated by addition of protein A - sepharose. The values represent the average percentage of fusion molecules that bound to the antibody and could be eluted with 15 mM acetic acid determined in triplicate binding reactions. The competition data demonstrated that the majority of the isolated fusion molecules were specific for<br>
the myc binding site.<br>
Sequence analysis of 116 individual clones derived from the fifth and sixth rounds of selection identified one sequence that occurred twice and contained the wild type c-myc epitope EQKLISEEDL (SEQ ID NO: 2). A third sequence was almost<br><br>
dentical to the other two, but showed two point mutations at the nucleotide level, one of which caused a mutation from Ile to Val in the conserved myc epitope region. All equences contained a consensus motif, X(Q,E)XLISEXX(L,M) (SEQ ID NO: 38), vhich was very similar to the c-myc epitope. The core region of four amino acids, ISE, was most highly conserved. Figure 22 illustrates the amino acid sequences of 2 selected peptides isolated from the random 27-mer library. At the top of the figure, the amino acid sequence of the c-myc epitope is shown. Of the sequences shown,  nly the regions containing the consensus motif are included. Residues within the leptidcs that match the consensus have been highlighted. Clone R6-63 contained the fild type myc epitope. Consensus residues (&gt; 50 % frequency at a given position) ppear at the bottom of the figure.<br>
Taking into consideration that the conserved motif contained one amino cid that was coded for by the defined 5' primer region, we calculated that the known 0 amino epitope c-myc epitope was represented only about 60 times in the starting 100I of 2 x 10,13 molecules. The observed enrichment of the wild type epitope in five ounds of selection corresponded well with an enrichment factor of &gt; 200 per election round, a factor which was confirmed in a separate series of experiments.<br>
Immunoprecipitation assays performed on the twelve selected sequences hown in Figure 22 confirmed specific binding of the library-derived RNA-peptide usions to the antigen-binding site of the anti-myc monoclonal antibody. As INA-peptide fusions, all twelve sequences bound to the anti-myc antibody and exhibited no binding to protein A - sepharose. Competitive binding for the anti-myc tntibody was also compared using 35S-labeled fusion products (derived from the welve sequences) and unlabeled synthetic myc peptide. Under the conditions used, abeled wild type myc fusion bound at 9% in the presence of unlabeled myc peptide, and the percentage of binding varied between 0.4% and 12% for the twelve sequences ested. These data indicated that the sequences bound the myc antibody with an iffinity similar to that of the wild type myc fusion.<br>
PURIFICATION OF ARM MOTIF PEPTIDES<br><br>
AND FUSIONS WITH IMMOBILIZED RNA RNA binding sites for the A-boxBR (Cilley and Williamson, RNA 3:57-67 (1997)), BIV-TAR (Puglisi et ah, Science 270:1200-1203 (1995)), and HIV-RRE (Battiste el a!., Science 273:1547-1551 (1997)) were synthesized containing a 3' biotin moiety using standard phosphoramidite chemistry. The synthetic RNA samples were deprotected, desalted, and gel purified as described herein. The 3' biotinyl-RNA sites were then immobilized by mixing a concentrated stock of the RNA with a 50% v/v slurry of ImmunoPure streptavidin agarose (Pierce) in IX TE 8.2 at a final RNA concentration of 5 mM for one hour (25°C) with shaking. Two translation reactions were performed containing (1) the template coding for the IN peptide fragment or (2) globin mRNA (Novagen) as a control. Aliquots (50 μl of a 50% slurry v/v) of each immobilized RNA were washed and resuspended in 500 μl in binding buffer (100 mM KCl, 1 mM MgCl2,10 mM Hepes-KOH pH 7.5, 0.5 mM EDTA, 0.01% NP-40, 1 mM DTT, 50 μg/ml yeast tRNA). Binding reactions were performed by adding 15 μl of the translation reaction containing either the N peptide or globin templates to tubes containing one of the three immobilized binding sites followed by incubation at room temperature for one hour. The beads were precipitated by centrifugation, washed 2X with 100 μl of binding buffer. RNase A (DNase free, 1 u.1, 1 mg/ml) (Boehringer Mannheim) was added and incubated for one hour at 37°C to liberate bound molecules. The supernatant was removed and mixed with 30 ul of SDS loading buffer and analyzed by SDS'Tricine PAGE. The same protocol was used for isolation of N peptide fusions, with the exception that 35 mM MgCl2 was added after the translation reaction followed by incubation at room temperature for one hour to promote fusion formation.<br>
The results of these experiments demonstrated that the N peptide retained its normal binding specificity both when synthesized in vitro and when generated as an RNA-peptide fusion with its own mRNA. This result was of critical importance. The attachment of a long nucleic acid sequence to the C terminus of a peptide or protein (i.e., fusion formation) has the potential to disrupt the polypeptide function<br><br>
relative to the unfused sequence. Arginine rich motif (ARM) peptides represent a stringent functional test of the fusion system due to their relatively high nonspecific nucleic acid binding properties. The fact that the N peptide-mRNA fusion (prior to cDNA synthesis) retained the function of the free peptide indicates that specificity is maintained even when there is a likelihood of forming either self- or non-specific complexes.<br>
USE OF PROTEIN SELECTION SYSTEMS The selection systems of the present invention have commercial applications in any area where protein technology is used to solve therapeutic, diagnostic, or industrial problems. This selection technology is useful f0r improving or altering existing proteins as well as for isolating new proteins with desired functions. These proteins may be naturally-occurring sequences, may be altered forms of naturally-occurring sequences, or may be partly or fully synthetic sequences. In addition, these methods may also be used to isolate or identify useful nucleic acid or small molecule targets.<br>
Isolation of Novel Binding Reagents. In one particular application, the RNA-protein fusion technology described herein is useful for the isolation of proteins with specific binding (for example, ligand binding) properties. Proteins exhibiting highly specific binding interactions may be used as non-antibody recognition reagents, allowing RNA-protein fusion technology to circumvent traditional monoclonal antibody technology. Antibody-type reagents isolated by this method may be used in any area where traditional antibodies are utilized, including diagnostic and therapeutic applications.<br>
Improvement of Human Antibodies. The present invention may also be used to improve human or humanized antibodies for the treatment of any of a number of diseases. In this application, antibody libraries are developed and are screened in vitro, eliminating the need for techniques such as cell-fusion or phage display. In one important application, the invention is useful for improving single chain antibody<br><br>
libraries (Ward et al., Nature 341:544 (1989); and Goulot et al., J. Mol. Biol. 213:617 (1990)). For this application, the variable region may be constructed either from a human source (to minimize possible adverse immune reactions of the recipient) or may contain a totally randomized cassette (to maximize the complexity of the library). To screen for improved antibody molecules, a pool of candidate molecules are tested for binding to a target molecule (for example, an antigen immobilized as shown in Figure 2). Higher levels of stringency are then applied to the binding step as the selection progresses from one round to the next. To increase stringency, conditions such as number of wash steps, concentration of excess competitor, buffer conditions, length of binding reaction time, and choice of immobilization matrix are altered.<br>
Single chain antibodies may be used either directly for therapy or indirectly for the design of standard antibodies. Such antibodies have a number of potential applications, including the isolation of anti-autoimmune antibodies, immune suppression, and in the development of vaccines for viral diseases such as AIDS. Isolation of New Catalysts. The present invention may also be used to select new catalytic proteins, in vitro selection and evolution has been used previously for the isolation of novel catalytic RNAs and DNAs, and, in the present invention, is used for the isolation of novel protein enzymes. In one particular example of this approach, a catalyst may be isolated indirectly by selecting for binding to a chemical analog of the catalyst's transition state. In another particular example, direct isolation may be carried out by selecting for covalent bond formation with a substrate (for example, using a substrate linked to an affinity tag) or by cleavage (for example, by selecting for the ability to break a specific bond and thereby liberate catalytic members of a library from a solid support).<br>
This approach to the isolation of new catalysts has at least two important advantages over catalytic antibody technology (reviewed in Schultz et al., J. Chem. Engng. News 68:26 (1990)). First, in catalytic antibody technology, the initial pool is generally limited to the immunoglobulin fold; in contrast, the starting library of RNA-protein fusions may be either completely random or may consist, without<br><br>
limitation, of variants of known enzymatic structures or protein scaffolds. In addition, the isolation of catalytic antibodies generally relies on an initial selection for binding to transition state reaction analogs followed by laborious screening for active antibodies; again, in contrast, direct selection for catalysis is possible using an RNA-protein fusion library approach, as previously demonstrated using RNA libraries. In an alternative approach to isolating protein enzymes, the transition-state-analog and direct selection approaches may be combined.<br>
Enzymes obtained by this method are highly valuable. For example, there currently exists a pressing need for novel and effective industrial catalysts that allow improved chemical processes to be developed. A major advantage of the invention is that selections may be carried out in arbitrary conditions and are not limited, for example, to in vivo conditions. The invention therefore facilitates the isolation of novel enzymes or improved variants of existing enzymes that can carry out highly specific transformations (and thereby minimize the formation of undesired byproducts) while functioning in predetermined environments, for example, environments of elevated temperature, pressure, or solvent concentration.<br>
An In Vitro Interaction Trap. The RNA-protein fusion technology is also useful for screening cDNA libraries and cloning new genes on the basis of protein-protein interactions. By this method, a cDNA library is generated from a desired source (for example, by the method of Ausubel et al., supra, chapter 5). To each of the candidate cDNAs, a peptide acceptor (for example, as a puromycin tail) is Iigated (for example, using the techniques described above for the generation of LP77, LP154, and LP160). RNA-protein fusions are then generated as described herein, and the ability of these fusions (or improved versions of the fusions) to interact with particular molecules is then tested as described above. If desired, stop codons and 3' UTR regions may be avoided in this process by either (i) adding suppressor tRNA to allow readthrough of the stop regions, (ii) removing the release factor from the translation reaction by immunoprecipitation, (iii) a combination of (i) and (ii), or (iv) removal of the stop codons and 3' UTR from the DNA sequences.<br><br>
the Tact that the interaction step takes place in vitro allows careful control of the reaction stringency, using nonspecific competitor, temperature, and ionic conditions. Alteration of normal small molecules with non-hydrolyzable analogs (e.g., ATP vs. ATPgS) provides for selections that discriminate between different conformers of the same molecule. This approach is useful for both the cloning and functional identification of many proteins since the RNA sequence of the selected binding partner is covalently attached and may therefore be readily isolated. In addition, the technique is useful for identifying functions and interactions of the -50-100,000 human genes, whose sequences are currently being determined by the Human Genome project.<br>
USE OF RNA-PROTEIN FUSIONS IN A MICROCHIP FORMAT "DNA chips" consist of spatially defined arrays of immobilized oligonucleotides or cloned fragments of cDNA or genomic DNA, and have applications such as rapid sequencing and transcript profiling. By annealing a mixture of RNA-protein fusions (for example, generated from a cellular DNA or RNA pool), to such a DNA chip, it is possible to generate a "protein display chip," in which each spot corresponding to one immobilized sequence is capable of annealing to its corresponding RNA sequence in the pool of RNA-protein fusions. By this approach, the corresponding protein is immobilized in a spatially defined manner because of its linkage to its own mRNA, and chips containing sets of DNA sequences display the corresponding set of proteins. Alternatively, peptide fragments of these proteins may be displayed if the fusion library is generated from smaller fragments of cDNAs or genomic DNAs.<br>
Such ordered displays of proteins and peptides have many uses. For example, they represent powerful tools for the identification of previously unknown protein-protein interactions. In one specific format, a probe protein is detectably labeled (for example, with a fluorescent dye), and the labeled protein is incubated with a protein display chip. By this approach, the identity of proteins that are able to<br><br>
bind the probe protein are determined from the location of the spots on the chip that become labeled due to binding of the probe. Another application is the rapid determination of proteins that are chemically modified through the action of modifying enzymes (for example, protein kinases, acyl transferases, and methyl transferases). By incubating the protein display chip with the enzyme of interest and a radioactiveiy labeled substrate, followed by washing and autoradiography, the location and hence the identity of those proteins that are substrates for the modifying enzyme may be readily determined. In addition, the use of this approach with ordered displays of small peptides allows the further localization of such modification sites.<br>
Protein display technology may be carried out using arrays of nucleic acids (including RNA, but preferably DNA) immobilized on any appropriate solid support. Exemplary solid supports may be made of materials such as glass (e.g., glass plates), silicon or silicon-glass (e.g., microchips), or gold (e.g., gold plates). Methods for attaching nucleic acids to precise regions on such solid surfaces, e.g., photolithographic methods, are well known in the art, and may be used to generate solid supports (such as DNA chips) for use in the invention. Exemplary methods for this purpose include, without limitation, Schena et al., Science 270:467-470 (1995); Kozal et al., Nature Medicine 2:753-759 (1996); Cheng et al., Nucleic Acids Research 24:380-385 (1996); Lipshutz et al., BioTechniques 19:442-447 (1995); Pease et al., Proc. Natl. Acad. Sci. USA 91:5022-5026 (1994); Fodor et al, Nature 364:555-556 (1993); Pirrung et al., U.S. Patent No. 5,143,854; and Fodor et al., WO 92/10092.<br><br>
Claim:<br>
1.	A method for producing a protein library, comprising the steps of:<br>
a)	providing a population of RNA molecules, each of which comprises a translation initiation sequence and a start codon operably linked to a protein coding sequence and each of which is operably linked to a peptide acceptor at the 3' end of said protein coding sequence;<br>
b)	in vitro translating said protein coding sequences to produce a population of RNA-protein fusions; and<br>
c)	further incubating said population of RNA-protein fusions under high salt conditions, thereby producing a protein library, which protein library comprises RNA-protein fusions.<br>
2.	A method for producing a DNA library, comprising the steps of:<br>
a)	providing a protein library produced according to the method as claimed in claim 1, which protein library comprises RNA-protein fusions, and<br>
b)	generating from each of the RNA portions of said fusions a DNA molecule, thereby producing a DNA library.<br>
3.	A method for the selection of a desired protein and nucleic acid<br>
encoding said protein, comprising the steps of:<br>
a)	providing a protein library produced as claimed in claim 1, which protein library comprises RNA-protein fusions, and<br>
b)	selecting a desired RNA-protein fusion, thereby selecting said desired protein and said nucleic acid encoding said protein.<br>
4.	The method as claimed in any of claims 1-3, wherein said high salt<br>
comprises a monovalent cation.<br><br>
The method as claimed in claim wherein said monovalent cation is at a concentration of between approximately 125 mM-1.5 mM.<br>
The method as claimed in claim 5, wherein said monovalent cation is at a Concentration of between approximately 300 mM - 600 mM.<br>
The method as claimed in claim 4 wherein said monovalent cation is K* or NH4+-<br>
The method as claimed in claim 4 wherein said monovalent cation is Na+.<br>
The method as claimed in claim 7, wherein said incubating step is carried out at approximately room temperature.<br>
The method as claimed in any of claims 1-3, wherein said high salt comprises a divalent cation.<br>
The method as claimed in claim 10, wherein said divalent cation is at a concentration of between approximately 25 mM - 200 mM.<br>
The method as claimed in claim 10, wherein said divalent cation is Mg+2.<br>
The method as claimed in any of claims 1-3, wherein said high salt comprises both a monovalent and a divalent cation.<br>
The method as claimed in any of claims 1-3, wherein each of said RNA molecules optionally comprises a pause sequence or optionally comprises a DNA or DNA analog sequence covalently bonded to the 3' end of said RNA molecule.<br><br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence is of a length sufficient to span the distance between the decoding site and the peptidyl transfer center of a ribosome.<br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence is approximately 60-70 A° in length.<br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence is less than approximately 80 nucleotides in length.<br>
The method as claimed in claim 14, wherein said Inure sequence or said DNA or DNA analog sequence is less than approximately 45 nucleotides in length.<br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence is between approximately 21-30 nucleotides in length.<br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence is joined to said RNA molecule using a DNA splint.<br>
The method as claimed in claim 14, wherein said pause sequence or said DNA or DNA analog sequence comprises a non-nucleotide moiety.<br><br>
22.	The method as claimed in claim 14, wherein said non-nucleotide moiety is one or more HO(CH2CH20)3P02 moieties.<br>
23.	The method as claimed in any of claims 1-3, wherein said RNA-protein fusion optionally comprises a nucleic acid or nucleic acid analog sequence positioned proximal to said peptide^ acceptor which increases flexibility.	 <br>
Dated this 10th day of August, 2001.	<br>
[RANJNA MEHTA-DUTT]<br>
of Remfry &amp; Sagar<br>
Attorney for the Applicants</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWNhbmNlbGxlZCBwYWdlcygxMC0wOC0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-cancelled pages(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWNsYWltcyhncmFudGVkKS0oMTAtMDgtMjAwMSkuZG9j" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-claims(granted)-(10-08-2001).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWNsYWltcyhncmFudGVkKS0oMTAtMDgtMjAwMSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-claims(granted)-(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWNvcnJlc3BvbmRlbmNlKDIyLTAzLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-correspondence(22-03-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWNvcnJlc3BvbmRlbmNlKGlwbyktKDA2LTAzLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-correspondence(ipo)-(06-03-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWRyYXdpbmcoMTAtMDgtMjAwMSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-drawing(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMSgxMC0wOC0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 1(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMTkoMjktMDQtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 19(29-04-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMWEoMDgtMDctMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 1a(08-07-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMihncmFudGVkKS0oMTAtMDgtMjAwMSkuZG9j" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 2(granted)-(10-08-2001).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMihncmFudGVkKS0oMTAtMDgtMjAwMSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 2(granted)-(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMygwOC0wNy0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 3(08-07-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gMygxMC0wOC0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 3(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0gNSgxMC0wOC0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form 5(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLWZvcm0tcGN0LWlwZWEtNDA5KDEwLTA4LTIwMDEpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-form-pct-ipea-409(10-08-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM4KDExLTA3LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-petition under rule 138(11-07-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgwOC0wNy0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-power of authority(08-07-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDA5NjQtbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgxMC0wOC0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00964-mum-power of authority(10-08-2001).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="204762-a-method-for-manufacturing-container-shaped-or-cup-shaped-products-with-a-coating.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="204764-a-fluid-dispenser-device-for-dispensing-a-fluid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>204763</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00964/MUM</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>25/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>22-Jun-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Mar-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Aug-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GENERAL HOSPITAL CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>55 FRUIT STREET, BOSTON, MASSACHUSETTS 02114, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JACK W. SZOSTAK</td>
											<td>308 COMMONWEALTH AVENUE, BOSTON, MA 02114, USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RICHARD W. ROBERTS</td>
											<td>APARTMENT #C, 514 GARFIELD AVENUE, SOUTH PASADENA, CA 91030, USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RIHE LIU</td>
											<td>APARTMENT 102, 2456 MASSACHUSETTS AVENUE, CAMBRIDGE, MA 02140, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/02589</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-02-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/247,190</td>
									<td>1999-02-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/204763-a-method-for-producing-a-protein-library-and-selection-of-a-desired-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:45:46 GMT -->
</html>
