<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/205472-in-vitro-method-for-transfection-of-eukaryotic-cell-for-producing-drug-for-non-viral-gene-therapy by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:26:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 205472:IN VITRO METHOD FOR TRANSFECTION OF EUKARYOTIC CELL FOR PRODUCING DRUG FOR NON-VIRAL GENE THERAPY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">IN VITRO METHOD FOR TRANSFECTION OF EUKARYOTIC CELL FOR PRODUCING DRUG FOR NON-VIRAL GENE THERAPY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An in vitro method tor transiection of culkaryotic cells for producing a cell as a drug for noivviral gone therapy using at least one protein capable of forming nucleoprotein filaments, wherein the protein .is initially modified with at least one functional component, which influences one or more steps of the transfection a nucleic acid to be transfected is then loaded with the modified protein, the nucleic acid and the protein forming a filament-like complex, and this complex is finally added to the cells to be transfected; wherein functional component causes, association of the complex to the cellular surface, non-endosomal passage of the complex through the cell membrane, release of the complex from endosomes/lysosomes and transport of the complex into the nucleus.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM   2<br>
THE PATENTS ACT, 1970 (39 Of 1970)<br>
COMPLETE  SPECIFICATION	<br>
[see Section  10, Rule 13]<br>
IN VITRO METHOD FOR TRANSFECTION OF EUKARYOTIC CELL FOR PRODUCINK DRUK <br>
FOR PRODUCINK DRUK FOR NON-VIRAL GENE THERAPY.<br>
We AMAXA GMBH of Nattermannallee 1,  50829 Koln, GERMANY, GERMAN Company<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed : -<br>
GRANTED<br>
29-3-2005<br><br>
Modular Transfection Systems<br>
Background of the Invention<br>
The present invention relates to the field of non-viral transfection of nucleic acids. The term transfection generally means the introduction of foreign substances into cells. The present invention relates to a method for the transfection of cells with the help of at least one protein which is capable of forming nucleoprotein filaments. The invention also relates to a transfection agent which contains a nucleoprotein filament (NPF) which is formed from at least one nucleic acid to be transfected and at least one protein which is capable of forming nucleoprotein filaments. In addition, the present invention relates to the use of the transfection agent according to the invention, a corresponding pharmaceutical formulation, in particular for use in gene therapy, a kit for the transfection of cells with nucleic acids and particular methods which use the transfection agent according to the invention.<br>
State of the Art<br>
The known non-viral transfection agents are subject to a series of restrictions. In non-viral transfection of nucleic acids, both the size of the globular nucleic acid complexes, which are often unilaterally charged, and the lack of controllability of one or several transfection steps are usually a major problem. One cause of the latter is the inadequate ability to penetrate the external cell membrane or the membranes of internal compartments, inadequate protection from enzymatic degradation, low bioavailability and biological effects of inadequate controllability caused by non-biological molecules in the cell. Nucleic acids mostly contain highly expanded steric structures, are easily degradable by enzymes, and their unilateral charge excess causes ready association with basic cell structures. The associated inadequate ability to pass into the nucleus has the result that the current nucleic acid transfection technologies almost exclusively employ cancer-like transformed cells, such as cell lines, as the nuclear membrane in these cells is temporarily disintegrated during cell<br><br><br>
division and entry into the nucleus is possible. However, transformed cells are not comparable to the original physiological state of primary cells, so that conclusions about the behavior of primary transfected cells cannot be reliably based on studies using transformed cells. ,<br>
A series of transfection agents is already known, almost all of which form globular complexes with nucleic acids by electrostatic interaction, often with a diameter of more than 50 nm (Tang and Szoka, 1997). These complexes mostly associate at the cell surface with a large excess of charge and are taken up by endocytosis. They leave the endosomes, either by buffering the acidification of the endosomes until these burst (e.g. with polyethylenimine, starburst dendrimers (Kukowska-Latello et al. 1996) or addition of chloroquine), or by the action of membrane-active groups, lipids or lipophilic peptides. However, the complexes can only reach the cell nucleus during the next cell division to show there the desired effects after the nucleic acid has been released.<br>
The problem of the division-dependent nuclear import can be avoided by the use of NLS-peptides (NLS = nuclear localization signal), particularly when attention is paid to the problems of signal masking and non-specific protein binding (WO 00/40742, Amaxa).<br>
US-Patent No. 5,468,629 describes the use of the RecA protein and the possible use of other proteins with functional homology to RecA in the transfection of cells with ssDNA (single-strand DNA). The protein RecA supports, evidently by catalysis, the process of homologous recombination of the transported ssDNA with the cell DNA by influencing strand pairing and the subsequent strand exchange. The complexes used here contain ssDNA of maximally 700 nucleotides as well as RecA protein and are described as "RecA-coated" complexes. Such ssDNA-protein complexes are formed from DNA in the presence of RecA and ATP-y-S, with the formation of stable helical presynaptic filaments. These complexes are used with little success, for example, for<br><br><br>
the transfection of cell lines, i.e. for actively dividing cells, in particular transformed cells.<br>
Summary of the Invention<br>
It is the object of the present invention to provide a method and a transfection agent for the transfection of nucleic acids of any sort into cells of any sort, which permit improved uptake in cells and which at the same time allows the control of the transfection process. The transfection agent for this purpose should both be adequately stable during the transfection process and also guarantee adequate release of the nucleic acids to be transfected in the target cell compartment.<br>
This object is solved according to the invention with a method of the type mentioned in the introduction, wherein the protein is initially modified with at least one functional component which influences one or more steps of the transfection. The nucleic acid to be transfected is then loaded with the modified protein, the nucleic acid and the protein forming a filament-shaped complex, and this complex is finally added to the cells to be transfected.<br>
In addition, this object is solved by a transfection agent of the type mentioned in the introduction, wherein the protein that is capable of forming nucleoprotein filaments is modified with at least one component influencing the transfection.<br>
As a result of the modification of one or several proteins which form nucleoprotein filaments (NPF) with one or several of the same or different additional functional components, individual steps of the complex transfection process can be controlled in a particularly advantageous manner which is specific for the nucleic acid to be transfected and for the target cell. In addition, the method and transfection agent according to the invention permit a clear increase in transfection efficiency in comparison with known methods.<br><br><br>
According to the invention, the NPF-forming proteins are used in an advantageous manner, particularly as modular carrier system for functional groups, which goes beyond pure protection of DNA. The complex exhibits very low extension in space in two respects: it is filamentous - not globular - and it contains only one nucleic acid molecule per filament. Because of the assembly which leads to a stochiometric ratio of a few nucleotides per carrier protein, an extremely high density of functional signals for transfection is possible, much more than with globular complexes. The signal density and also the combination of different signals can be individually adjusted by the mixture of different functionalized carrier proteins and proteins without functional groups, so that the transfection agent according to the invention can be designed as a modular system for the widest variety of transfection conditions. The low extension in space and the high signal density also make it possible to use in addition endogenous transport systems which are specific for small molecules. Because of its filamentary character, its simple structure and its adjustable and extremely high signal density, the method according to the invention makes possible, or the transfection agent according to the invention is capable of, specifically transporting of larger nucleic acid molecules, such as expression vectors, using endogenous mechanisms.<br>
The additional functional components can for example be bound directly to the NPF proteins or via a spacer, a non-functional separating unit. Spacers of this kind give rise to a greater distance between the NPF and the functional components, which avoids mutual steric hindrance and which guarantees better spatial availability of the NPF protein and of the functional components. The structure of the agents according to the invention thereby then largely avoids masking of the functional components.<br>
NPF-forming proteins form nucleoprotein filaments with nucleic acids mostly by cooperative binding, in which proteins and nucleic acids form a complex having a size or diameter that is much smaller than that of the known globular transfection agents. An NPF of this kind can for example be formed by proteins of the RecA family, which are DNA-dependent ATPases, in the presence of nucleoside triphosphates such as<br><br><br>
ATP (adenosine triphosphate), with total loading of a density of, for example, one protein per three bases in double-stranded DNA (Bianco et al., 1998). This high protein density offers excellent protection against enzymatic hydrolysis of the nucleic acids as the possible points of enzymatic attack are greatly reduced. The NPF not only causes the complex to be adequately stable but also allows adequate release of the transported nucleic acids, for example, in the nucleus. Use of non-hydrolyzable or poorly hydrolyzable analogues of nucleoside triphosphates, for example of ATP and/or GTP, such as ATP-y-S or GTP-y-S, also offers the possibility of providing extra stability to NPFs which are formed with ATP-forming proteins, such as the RecA family.<br>
According to the invention, the proteins capable of forming NPFs also include derivatized NPF proteins. For example, fusion proteins can be produced. In addition, NPF-forming proteins can be truncated or elongated, individual sections or amino acids can be deleted, introduced or chemically modified as long as their function which is essential to the invention, the structural formation of NPFs with nucleic acids, is maintained.<br>
A particular advantage of the invention is the spatial structure of the NPFs which makes it possible to exploit the natural mechanisms of nuclear transport. The maximal diameter of the transfection agent is determined by the size of the nuclear pores and is not exceeded even with very long nucleic acids, i.e. NPFs can be used as transfecting agents independently of the length of the nucleic acid to be transfected. It has been shown that the size limit for transport through the nuclear pore is approx. 25 nm (Feldherr and Akin, 1997) or 50 nm (SV40-Virus) (Yamada and Kasamatsu, 1993). Nucleic acids, of which the import through the nuclear membrane with conventional transfection agents having globular structure and/or non-specific binding of several nucleic acid molecules per transfection component is barred, can easily lie under such limit using transfection agents according to the invention. The structure of the transfection agents according to the invention even makes it possible to use for the first time diameters of 
<br><br>
used. The filamentous structure assembled by the use of NPF proteins is therefore also suitable for the transport of longer nucleic acids of several kilobases in length. The method according to the invention and the transfection agents according to the invention are consequently particularly well suited for the transfection of cells with larger nucleic acid sequences. Transfection agents according to the invention are preferred which contain a nucleic acid to be transfected including at least 700 nucleotides.<br>
The present invention can advantageously be used as such for the transfection of nucleic acids or in combination with other transfection methods and materials. The high degree of loading with NPF-forming proteins increases the stability to hydrolysis and the low diameter of the NPF allows the exploitation of endogenous cellular transport mechanisms which are only available to molecules which are small enough, for example, those of the nuclear transport system. In addition, adequate release of the nucleic acids to be transfected in the cell compartments, preferably in the nucleus, is guaranteed.<br>
The term "transfection agent" according to the invention is to be understood as transport vehicle for nucleic acids or their derivatives which already contain the nucleic acid to be transfected. A transfection agent in the sense of the invention performs at least a single step of the complex process of transfection.<br>
In the context of the present invention, the term "nucleoprotein filament" (NPF) means a molecular structure consisting of nucleic acid(s) or nucleic acid derivatives and proteins which, as the result of non-covalent, mostly cooperative binding, form a filamentary or thread-like complex which preferably contains only a single nucleic acid molecule or derivative of this. Particularly preferred are helical nucleoprotein filaments, for example, those formed from RecA with single or double-stranded DNA (Di Capua et al., 1982).<br><br><br>
The "nucleic acid to be transfected" can be either a double or a single-stranded DNA, or a double- or single-stranded RNA, or a double-stranded DNA with single-stranded ends, a DNA/RNA hybrid, an antisense DNA, antisense RNA or chemically modified nucleic acid derivatives, in which, for example, the resistance to hydrolysis is increased (pepide nucleic acid, PNA), or in which reactive molecular groups have been introduced for the covalent binding andJor the modification of target nucieic acids. Derivatives of transfectable nucleic acids are understood to include the modification of the nucleic acid used with a sequence-specific or covalently bound protein. The preferred nucleic acid of the present invention is DNA, in particular, double-stranded DNA. Until now, NPF-forming proteins have mostly been used together with single-stranded DNA, for example, for recombination. Surprisingly, it has turned out in the context of the invention that many NPF-forming proteins also form stable complexes with double-stranded DNA under suitable conditions and can be used for the transfection of double-stranded DNA in accordance with the invention. It is therefore a further advantage of the invention that efficent transfection with double-stranded DNA is possible too.<br>
In an advantageous embodiment of the invention, NPFs modified with functional components are produced by the modification of original NPF-forming proteins or their derivatives. This modification can be the deletion or insertion of amino acids and/or protein domains. The modification can also be provided by chemical alteration of amino acids and/or other molecular groups and/or by chemical coupling of peptides, proteins, carbohydrates, lipids or other molecules to the NPF-forming protein or its derivative. These modifications may occur with the use of a spacer.<br>
The first hurdle for transfection consists in the association of the transfection complex to the cell surface. A preferred embodiment of the method and transfection agent according to the invention therefore uses at least one protein capable of forming NPF which is modified with at least one functional group which causes the association of the complex or agent to the cell surface. This can be performed specifically for the cell type by binding to surface structures specific to the cell type which are expressed<br><br><br>
on one or only a few cell types, or non-specifically, for example, by an electrostatic interaction. All naturally occurring or synthetically produced substances can be used for the cell type-specific binding which bind to receptors on the cell surface, such as receptors which are used by viruses or bacteria for cell entry, e.g. the Epstein Barr Virus receptor CD21 (Delcayre, 1991) or Listeria monocytogenes receptor E-cadherin (Mengaud, 1996). Other examples for the use of ligand-receptor pairs include the transferrin receptor/transferrin system for cells which need a lot of iron, asialoglycoprotein receptor/galactose for hepatocytes, integrin/integrin-binding peptides, such as RGD (Harbottle 1998) or molossin (Collins 2000) or hormones which bind to hormone receptors, such as insulin (Rosenkranz 1992), EGF (epidermal growth factor) or insulin-like growth factor I (Feero, 1997) and oligosaccharides which bind to lectins (Midoux et al., 1993). It is also possible to use cell-specific antibodies (Fominaya, 1996) or protein A or its IgG-binding domain (Ohno, 1997) as well as biotinylated proteins (either proteins biotinylated on the cell surface or biotinylated monoclonal antibodies) in combination with streptavidin (Schoeman, 1995). It is particularly suitable in the context of the present invention to use "epitope tagging", i.e. cell type specific transfection using biospecific antibodies which, on the one hand can recognize an epitope, namely short peptides such as from the influenza hemagglutinin (Surdej, 1994) or from the c-myc protein (Evan, 1985) which are either coupled or fused by genetic engineering to the NPF-forming proteins or their derivatives and, on the other hand, to specific cell surface structures. For non-cell specific interactions of the transfection complex by electrostatic forces it is possible, for example, to introduce positive charges with additional amino acids (lysine, arginine, histidine).<br>
In another particularly preferred embodiment of the method and transfection agent according to the invention, it is intended that the functional components cause the non-endosomal passage of the complex or agent through the cell membrane. The non-endosomal membrane passage has the advantage that the agent is immediately available in the cytosol and is not exposed to the hydrolytically active environment of the lysosomes. Membrane passage of this kind can be attained with membrane-<br><br><br>
active molecules. These can be naturally occurring, modified or synthetic peptides which are mostly aliphatic or amphiphilic, such as viral peptides, for example HIV tat, VP22, HBV surface antigen; peptides from transcription factors such as, for example, the homeodomain of antennapedia (Thoren et al 2000), engrailed, HOXA-5; peptides from cytokines, e.g. IL-1 FGF-1, FGF-2; peptides from cellular signal sequences, e.g. the Kaposi fibroblast growth factor, monoclonal antibodies which penetrate living cells, e.g. mab 3E10 (Weisbart et al 2000), synthetic or chimeric peptides, e.g. amphiphilic model peptides or transportane (Pooga et al. 1998).<br>
In addition, an advantageous embodiment of the invention uses at least one functional component according to the invention which causes the release of the complex or agent from endosomes or lysosomes. For example, passage through the cell membrane can occur through endocytosis. After endocytosis, the nucleoprotein complexes must be released from the endosomes. For this purpose, all substances with endosomolytic activity can be used. These can, for example, be peptides, their derivatives or synthetic analogues from bacteria or viruses, or other synthetic substances known to the person skilled in the art. Endosomolytic substances from bacteria include, for example, streptolysin O, pneumolysin, staphylococcal -toxin, listeriolysin O (Provoda, 2000). Viral peptides include, for example, the N-terminal hemagglutinin HA-2 peptide of influenza virus (Steinhauer et al., 1995), the N-terminus of the VP-1 protein of rhinovirus HRV2 (Zauner et al., 1995) or the capsid component Ad2 of adenovirus (Hong, 1999). Synthetic substances include, for example, amphipathic peptides (GALA, KALA, EGLA, JTS1) (Wagner, 1999) or imidazole- (Pack, 2000) or polyamidoamine-modified polymers (Richardson, 1999).<br>
Passage (import) into the nucleus occurring usually in dependence on cell division and possibly after stimulation of the cell is a particular hurdle for all transfection agents. It is therefore intended in a particularly advantageous embodiment of the invention to use a functional component which causes the transport of the complex or agent into the cell nucleus. The essentials in nuclear transport are firstly the limiting diameter of the nuclear pores and secondly the signal molecules used for transport.<br><br><br>
The signals for nuclear transport in the context of the invention are nuclear ligands which bind to a nuclear receptor. Particularly suitable nuclear ligands are NLS (nuclear localization signals) or other components of the nuclear transport machinery. The nuclear localization signals which are particularly preferred for use are those signals which either them'selves and/or together with their flanking regions exhibit little or no positive charge excess, as a charge excess can lead to non-specific nucleic acid binding and thus to masking of the signal. Well suited are extended sequences of so-called classical NLS when the total charge of the peptide can be at least approximately balanced by flanking negatively charged amino acids. These amino acids can occur naturally in the peptide/protein in these positions or be introduced there on the basis of structural considerations. The so-called non-classical NLS can also be used, for example, an NLS from the influenza virus "nucleoprotein" (Wang et al., 1997, Neumann et al., 1997) or the sequence M9 from the heterogenous nuclear RNP (hnRNP) A1 protein which have no great excess of positive charges or which pass into the nucleus by a non-classical transport route. An incomplete but good review of the NLS which can be used in the embodiment of the present invention is given by T. Boulikas (1993, 1996, 1997). Approximately charge-neutral sequences include, for example, the large T-antigen from simian virus 40 having flanking negative charges which are either naturally occurring or artificially introduced (see also WO 00/40742 Amaxa).<br>
A particular advantage of the invention is therefore the clear improvement of the transfection of nucleic acids of eukaryotic cells which are either not dividing or only dividing weakly, in paticular primary eukaryotic cells. It is exactly these cells which have not yet forfeited their ability to provide biological or medical information, as they can, for example, be taken directly from the body by blood sampling or tissue biopsy and are of decisive importance to the skilled person. With the expected analysis of the almost completely decoded human genome, it is only the specific expression of the gene being examined in cell systems which will finally give clear evdience of possible technical applicability with adequate physiological relevance. In addition, the<br><br><br>
transfection of primary cells is an essential precondition for non-viral gene therapy, both ex vivo and in vivo.<br>
In an advantageous embodiment of the invention, the method and transfection agent according to the invention can each include at least one protein being capable of forming nucleoprotein filaments which is modified with several functional components of different function and/or different proteins which are modified with proteins of different function, respectively. The transfection agents according to the invention can, as described above, be realized with a plurality of proteins which can form NPFs or derivatives thereof which may be present in their original or in a functionally modified form. Either a single or several different proteins which can form NPFs may be used and these can be modified with one or several functional components of the same or different functions, depending on the specfic requirements of the transfection. Depending on the intended goal, different modules can be assembled by the user in this way. This gives rise to a modular system from which unmodified or modified NPF proteins of the same or different function can be selected in order to achieve an optimal adaptation of the transfection strategy to the nucleic acid to be transfected and the target cell.<br>
A particularly advantageous embodiment of the invention is intended, in which the protein is loaded with a plurality of functional components. In this way, the total signal density can be still further increased, so that improved transfection efficiency and better control of transfection are possible.<br>
Many NPF-forming proteins form the NPF structure in the presence of nucleoside triphosphates, such as ATP, as cofactor. The addition of nucleoside triphosphates hereby stabilizes the NPF. In the method according to the invention, the NPF structure can therefore be advantageously formed or stabilized by nucleoside triphosphates and /or non-hydrolyzable analogues of these, in particular with ATP (adenosine triphosphate) and/or GTP (guanosine triphosphate) and/or their non-hydrolyzable   analogues.   This   can   also   occur   by   covalent   binding   after   a<br><br><br>
photochemical reaction. Non-hydrolyzable nucleoside triphosphate analogues include, for example, ATP- S (adenosine 5'-0-3-thiotriphosphate) and GTF S (guanosine 5'-0-3-thiotriphosphate) (Ellouze, 1999). Possible ATP analogues which modify NPF-ATPases after a photochemical reaction include 8N3ATP (8-azidoadenosine 5'-triphosphate) and 5'FSBA (5'-p-fluorosulfonylbenzoyladenosine) (Knight, 1985).<br>
In an advantageous embodiment of the method according to the invention, this is used in combination with other biological and/or chemical and/or physical transfection methods for biologically active molecules, such as liposome-mediated transfer, microinjection, electroporation, immunoporation, the ballistic method, transfer with the help of cationic lipids, calcium phosphate, DEAE-dextran, polyethylenimine or pH-sensitive hydrogel. Transfer methods of this sort, in particular electroporation, can advantageously support individual steps in the transfection process, for example, such as the import into the cytoplasm of the cell.<br>
Preferred transfection agents according to the invention include as NPF-forming protein a protein selected from the group of proteins containing RecA, RadA, ScRad.51, RAD51, hDmc1, SASP, ICP8, preferably UvsX, more preferably hRAD51 or a mixture of at least 2 of the proteins in this group or one or several derivatives of these proteins. NPF-forming proteins for the embodiment of the present invention include for example RecA from Escherichia coli and its functional homologues from viruses, prokaryotes and eukaryotes, such as UvsX from the bacteriophage T4 (Mosig, 1987), RadA from archebacteria (Seitz et al., 1998), ScRad51 from Saccharomyces cerevisiae, RAD51 from mammals and, in particular, hRad51 from man. Homologous proteins to RecA have been detected in at least 60 different sorts of bacteria (Roca and Cox, 1990, Karlin and Brocchieri, 1996, Karlin et al., 1995), in archaea (Sandler et al. 1996), in all eukaryotes which have been examined (Ogawa et al. 1993), in mitochondria (Thyagarajan et al., 1996) and in plastids (Cerutti et al., 1992). With single- or double-stranded DNA, RecA and its homologues form helical NPF with a diameter of about 11 nm. The binding of RecA is cooperative and leads to<br><br><br>
a partial unwinding of the DNA helix. RecA, UvsX, ScRad51 and hRad51 form NPF with a binding stoichiometry of three base pairs per monomer (double-stranded DNA) or 3-6 nucleotides per monomer (single-stranded DNA) (Bianco et al. 1998, Baumann and West 1998). The meiosis-specific recombinase hDMC1 is preferred; this forms filaments with double-stranded DNA which consist of a linear row of stacked protein rings (Masson et al., 1999). In addition, the group of SASP proteins (small acid-soluble spore proteins) from the spores of Bacillus and Clostridium species is also preferred. The SASPs also bind to double-stranded DNA forming helical NPF with a diameter of about 6.6 nm (Griffith et al. 1994). Viral proteins which can form filaments with single- and/or double-stranded DNA, such as protein ICP8 from Herpes simplex, are also preferred as NPF-forming proteins (Lee &amp; Knipe, 1985).<br>
A binding stochiometry of one monomer protein per 3 base pairs has been demonstrated for complete binding of the NPF-forming proteins UvsX, Rad51 and RecA to double-stranded DNA (Bianco et al 1998). Complete binding of this kindcan be attained with UvsX, for example, by using a 3- to 5-fold excess of the protein, depending on the binding buffer used, the conformation of the nucleic acid and the temperature (Yu and Egelman 1993); with a/p type SASP from Bacillus subtilis the corresponding protein:DNA ratio is about 5:1 (Griffith et al 1994). It is particularly preferred in the context of this invention for the loading of the nucleic acid with protein to be as complete as at all possible; this is defined separately for each of the preferred NPF-forming proteins. However, incomplete loading of the nucleic acids below the absolutely highest degree of loading for a specific NPF-forming protein is possible in the context of the invention. Lower loading of the nucleic acids with NPF-forming proteins is, for example, preferred when additional DNA-binding proteins are to be used for defined steps in the transfection or when buffer conditions have to be selected for specific applications which are not optimal for complete loading of the nucleic acid with NPF-forming proteins.<br>
According to the invention, NPF-forming proteins are understood to include both natural NPF-forming proteins and their derivatives. Under derivatives of NPF-forming<br><br>
proteins, the skilled person understands firstly modification by deletion or insertion of additional amino acid sequences or protein domains in recombinant proteins and/or the introduction of functional groups by the chemical modification of molecular groups which are already present and/or the chemical coupling of proteins, peptides, carbohydrates, lipids or other molecules.<br>
The transfection agent according to the invention can also be advantageously used for producing drugs for the gene therapeutic treatment of humans and animals. The therapeutically useful nucleic acids can be made accessible to primary cells and complex transfection methods in the form of transfection agents according to the invention.<br>
A further aspect of the present invention concerns pharmaceutical preparations which contain a transfection agent according to the invention, possibly together with conventional adjuvants and carriers.<br>
A further aspect of the present invention concerns kits which are suitable for the transfection of cells with nucleic acids and which include at least one protein capable of forming NPFs according to the invention and at least one functional component according to the invention as well as at least one of the following components:<br>
a)	nucleoside triphosphate and/or nucleoside triphosphate analogues<br>
b)	at least one nucleic acid to be transfected<br>
c)	adjuvants and additives<br>
Such kits can be specifically designed for the different requirements of experts, for example, they can be optimized for certain nucleic acids, target cells or stability requirements, by the selection of specific NPFs, specific NPF modifications or individual adjuvants or additives. The protein can hereby already be loaded with the functional component(s) or the proteins or functional components can be contained separately in the kit.<br><br>
A further aspect of the present invention concerns the use of the transfection agents according to the invention for cell screening, specifically for the identification of activators or inhibitors of the expression product(s) of the transfection agent in mitotically inactive or weakly active cells, primary cells and other cells of limited life span. Screening of this kind is a fundamental method for the identification of activators or inhibitors of validated or non-validated target proteins in the identification of active substances in the pharmaceutical industry. These screening methods are usually planned so that cells which have been stably transfected with foreign nucleic acids are exposed to potential inhibitors and/or activators and the influence of these on the physiology of the cells is determined, possibly in comparison with comparator cells. The transfection agent according to the invention is suitable for the insertion of the foreign nucleic acid shortly before the addition of activators or inhibitors, even in mitotically inactive or only weakly active cells, primary cells and other cells of limited life span, and thus to make these cells accessible as test cells.<br>
A further aspect of the present invention concerns the use of transfection agents according to the invention in the identification of physiologically active nucleic acids. This is of particular importance for the rapid and physiologically relevant evaluation of genomic data which are available to the scientific and pharmaceutical community. As transfection with the transfection agents according to the invention is independent of cell division and/or endocytosis, the time between transfection and analysis is much shortened. This makes a much higher sample throughput possible. Even cells which are difficult to transfect and even non-dividing cells are accessible with the transfection agents according to the invention. The resulting transfection and the physiological evaluation of changes in comparison to control cells make the identification of physiologically active nucleic acids possible.<br><br><br><br>
Abbreviations:<br>
Aside from the abbreviations usual in Duden1, the following abbreviations are used:<br><br>
AMP-PCP<br>
AMP-PNP<br>
DEAE<br>
DNA<br>
dYT<br>
FACScan<br>
FCS<br>
FL<br>
FSC<br>
GMP-PNP<br>
GTP<br>
H6<br>
Ig<br>
kb<br>
ml<br>
mM<br>
msec<br>
NCBI<br>
ng<br>
nm<br>
PBS<br>
PCR<br>
Pi<br>
RNA<br>
RPMI<br>
SDS<br><br><br>
S'-Adenylylimidodiphosphate<br>
Diethylaminoethane<br>
Desoxyribonucleic acid<br>
Double yeast trypton<br>
Fluorescence activated cell scanning<br>
Fetal calf serum<br>
Fluorescence<br>
Forward scatter<br>
S '-Guanylylimidodiphosphate<br>
Guanosine triphosphate<br>
Histidine hexamer<br>
Immunoglobulin<br>
Kilobases<br>
Milliliter<br>
Millimolar<br>
Millisecond<br>
National Center for Biology Information<br>
Nanogram<br>
Nanomolar<br>
Phosphate buffered salt solution<br>
Polymerase chain reaction<br>
Sodium dihydrogen phosphate/Disodium hydrogen phosphate<br>
Ribonucleic acid<br>
Roswell Park Memorial Institute<br>
Sodium dodecylsulfate<br><br>
SV40	Simian virus 40<br>
TAE	Tris-acetate/ethylendiaminetetraacetate<br>
U/mg	Units/Milligram<br>
rpm	Revolutions per minute<br>
Cl-Puffer	Cell injection buffer<br>
ug	Microgram<br>
  Microliter<br>
Short Description of the Figures<br>
Figurel:	Schematic Representation of the Expression Plasmids<br>
Fig. 1 shows a schematic representation of the structure and production of the NPF-forming proteins described in the examples.<br>
Figure 2:	Binding of UvsX to Double-Stranded DNA<br>
Figure 2 shows reaction mixtures of 200 ng (in each case) of a purified 1 kb PCR fragment with the given quantities of H6UvsX in 76 mM K2HP04, 17 mM KH2P04l 14 mM NaH2P04, pH=7.2, 5 mM MgCI2 and 0 or 1 mM ATP S, which was incubated in a final volume of 15 for 30 min at room temperature and then applied to a 0.8 % TAE/agarose gel, which was afterwards stained with ethidium bromide.<br>
Figure 3:	Binding of NLS-modified UvsX to double-stranded DNA in the<br>
presence of different ATP analogues<br>
250 ng of a purified 1 kb PCR fragment were incubated with 15 ug UvsXH6N2 in 76 mM K2HP04, 17mM KH2P04, 14 mM NaH2P04, pH=7.2, 5 mM MgCI2 and 0.5 or 2 mM of the given nucleotide analogue in a final volume of 20 ul for 30 min at room temperature. The reaction mixture was then split and 220 ng of a 1.7 kb PCR<br>
' Translator's Note: German Standard dictionary<br><br>
fragment was added to one half (B), incubated for a further 30 min at room temperature; all reaction mixtures were then applied to a 0.8% TAE/agarose gel, which was afterwards stained with ethidium bromide.<br>
Figure 4:	Binding of a mixture of UvsX and modified UvsX to double-<br>
stranded DNA<br>
200 ng (in each case) of a purified 1 kb PCR fragment was incubated in 76 mM K2HP04, 17 mM KH2P04, 14 mM NaH2P04, pH=7.2, 5 mM MgCI2 and 1 mM ATP-y-S with the given quantities of purified H6UvsX or UvsXH6N2 for 30 min at RT and then all reaction mixtures were applied to a 0.8% TAE/agarose gel, which was afterwards stained with ethidium bromide. To exclude the possibility that the electrophoretic behavior of DNA is artificial as a result of salt, traces of imidazole or glycerine, samples of elution or dialysis buffer were included in tracks 8 and 9. (Final concentration: 1/10 vol elution buffer with 500 mM imidazole, 3/20 volumes dialysis buffer with 50% glycerine).<br>
Figure 5:	FACScan  Analysis  of the  Transfection  of  NIH3T3  Cells  in<br>
Combination with Electroporation<br>
Fig. 5 a-d shows an FACScan analysis: 5a) electroporation without DNA, 5b) electroporation of vector DNA without UvsX, 5c) with vector DNA packaged in UvsX, 5d) with vector DNA packaged in UvsX-NLS<br>
Figure 6:	A Further FACScan Analysis of the Transfection of NIH3T3 Cells<br>
in Combination with Electroporation<br>
Fig. 6 shows a bar diagram of the results of an FACSscan analysis of transfection with vector DNA packaged in UvsX-NLS (UvsXH6N2-2) and vector DNA packaged in UvsX "scrambled" NLS (UvsXH6N2sc).<br><br><br>
Figure 7:	Fluorescence Microscopic Analysis of the Transfection of NIH3T3<br>
Cells in Combination with Microinjection.<br>
The expression of a fluorescent marker protein (left side) is shown in microinjected NIH3T3 cells. BSA-Cy5 serves as injection marker, which is visualized in the corresponding fluorescence filter (right side). Pictures 1 and 2 show cells which were injected with DNA and UvsXH6N2sc in the cytoplasm; Pictures 3 and 4 show cells which were injected with DNA and UvsXH6N2-2 ; Pictures 5 and 6 show cells that were only injected with DNA.<br>
Figure 8:	Schematic Representation of the Expression Plasmids for SASP<br>
Proteins<br>
Fig. 8 shows a schematic representation of the structure and production of the SASP proteins described in Examples 7 and 8.<br>
Figure 9:	(A) Purification of SASP Protein<br>
(B) DNA Binding Ability of SASP Protein<br>
Fig. 9 A shows an SDS gel with purified SASP protein. Fig. 9 B shows the binding of DNA by SASP protein in a DNA shift analysis.<br>
Figure 10:	(A) Purification of SASP-NLS-Protein<br>
(B) DNA Binding Ability of SASP-NLS-Protein<br>
Figure. 10 A shows an SDS gel with purified SASP-NLS-Protein (SASP-H6N2). Fig. 10 B shows the binding of DNA by SASP-NLS-Protein in a DNA shift analysis.<br>
Figure 11:     Coupling of a Peptide with RGD-Motive to H6UvsX<br><br>
Fig. 11 shows the electrophoretic separation of H6UvsX protein with and without the chemically coupled peptide RGD2. Two independently prepared samples of H6UvsX*(RGD2) are shown. 1 ug protein was applied in each track.<br>
Figure 12:	Binding of Differently Modified UvsX Proteins to Double-Stranded<br>
DNA<br>
Fig. 12 shows the separation in agarose gels of the NPFs consisting of a double-stranded DNA fragment and a mixture of H6UvsX*(RGD2) and UvsXH6N2-2 (Track 1), or of UvsXH6N2-2 alone (Track 2) or of H6UvsX*(RGD2) alone (Track 3).<br>
Figure 13:	DNA Shift Analysis of the Binding of UvsXH6N2 and<br>
UvsXH6N2NIT-2 to Fluorescein-Labeled DNA<br>
Fig. 13 shows a DNA shift of DNA-UvsXH6N2 and DNA-UvsXH6N2NIT-2. The proteins were incubated with a AlexaFluor488-labeled DNA fragment, as described in Example 10.<br>
Figure 14:	Fluorescence Microscopy Analysis of the Uptake of NPFs in<br>
NIH3T3 Cells by Endocytosis<br>
Figs. 14a and 14b show fluorescence microscopy pictures of NIH3T3 cells. In each case, one picture shows a bright field image (bottom) and one in reflected light fluorescence (top). In the bright field, vesicular intracellular compartments are recognizable, which emit fluorescent light because of the endocytosed fluorescein-labeled DNA.<br>
Figure 15:     Percentage of Cells with Endocytosed NPFs<br>
Fig. 15 is a bar diagram which shows the percentage of the cells which inhibit at least one fluorescent vesicular compartment.<br><br><br>
Figure 16:     Schematic Representation of the Expression Plasmids<br>
Fig. 16 shows the schematic representation of the structure of the expression plasmids for the production of hRad51 fusion proteins.<br>
Figure 17:	Binding  of NLS-Modified  hRad51   (hRad51H6N2) to  Double-<br>
Stranded DNA<br>
Fig. 17 (A) shows an SDS/Coomassie gel with hRad51H6N2 (12.7 ug) and hRad51H6 (11 ug) after purification over nickel-chelate affinity chromatography. Fig. 17 (B) shows reaction mixtures of 100 ng (in each case) of a purified 0.9 kb PCR fragment with the given quantities of hRad51H6N2 in 38 mM K2HP04, 8.5 mM KH2P04, 7 mM NaH2PO4, pH=7.2, 15 mM MgCI2, 2.5 mM ATP and 25% glycerine, which were incubated in a final volume of 30 ul for 10 min at 37Â°C and then applied to a 1% TAE/agarose gel which was afterwards stained with ethidium bromide.<br>
Figure 18	Binding of a Mixture of Differently Modified UvsX (UvsX-NLS-<br>
VP22 and UvsX-NLS) to Double-Stranded DNA<br>
Figure. 18 (A) shows a SDS/Coomassie gel with UvsXH6N2VP22c50.<br>
Fig. 18 (B): 140 ng (in each case) of a purified 1.7 kb PCR fragment were incubated in 96 mM K2HPO4, 21.5 mM KH2PO4, 18 mM NaH2PO4l pH=7.2, 5 mM MgCI2 and 1.3 mM ATP-y-S with the indicated quantities of purifed UvsXH6N2VP22c50 or UvsXH6N2-2 for 30 min at RT. All reaction mixtures were then applied to a 0.8% TAE/agarose gel, which was afterwards stained with ethidium bromide.<br>
Figure 19:	Transfected NIH3T3 Cells after Treatment with Complexes of<br>
DNA and a Mixture of UvsX-NLS-VP22 and UvsX-NLS<br><br><br>
Fluorescence microscopy picture of NIH3T3 cells, 24 h after treatment with complexes of 1 ug expression vector DNA and a mixture of 19 ug UvsXH6N2VP22c50 and 39 pg UvsXH6N2-2.<br>
Figure 20:	Schematic Representation of the Basic Structure of the Method<br>
and  Transfection  Agent  based  on   Nucleoprotein   Filaments, according to the Invention.<br>
Description of the invention<br>
The following examples are intended to describe the invention in detail, without limiting it to exemplary disclosed substances and methods.<br>
Example 1: Generation of recombinant UvsX as NPF forming proteins<br>
The proteins UvsXH6, UvsXH6-2, H6UvsX und H6UvsX-2 were used as NPF forming proteins (see Fig. 1).<br>
Structure of the proteins:<br>
UvsXH6 (400 amino acids):<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 1-391), amino acids 392-394: linker consisting of the amino acids G392GS394, amino acids 395-400: H395HHHHH400 for purification by nickel chelate affinity chromatography, amino acid exchange: <br>
UvsXH6-2 (403 amino acids):<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no: AAD42669,<br>
amino acids 1-391), amino acids 392-394: linker consisting of the s392YG394, amino<br><br><br>
acids 395-400: H395HHHHH400, amino acids 401-403: C-terminus consisting of the amino acids M401YS403.<br>
H6UvsX (404 amino acids):<br>
Amino acids 1-4: N-terminus consisting of the amino acids M1SYS4, amino acids 5-10: H5HHHHH10, amino acids 11-13: linker consisting of the amino acids S11YG13, amino acids 14-404: UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 1-391), amino acid exchange: <br>
H6UvsX-2 (404 amino acids):<br>
Amino acids 1-4: N-terminus consisting of the amino acids M1GYS4, amino acids 5-10: H5HHHHH10, amino acids 11-13: linker consisting of the amino acids S11YG13, amino acids 14-404: UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 1-391).<br>
Cloning of the expression plasmids<br>
For the expression of the aforementioned proteins in suitable Escherichia coli cells, plasmids were constructed containing a coding sequence for UvsXH6, UvsXH6-2, H6UvsX or H6UvsX-2 under the control of the lac promotor (pExH-UvsXH6, pExH-UvsXH6-2, pExH-H6UvsX or pExH-H6UvsX-2, see Fig. 1). The plasmids were generated by ligation of two PCR products. The first PCR product was amplified from pMCS5 (MoBiTec, Gottingen, Germany): pMCS5 is constructed in a similar way as pBluescript SK(-) (Stratagene) and is only different in the 5' region of the coding sequence of the lacZa fragment. PMCS5 therefore contains the lac promotor followed by the lac operator, by which the expression of an inserted coding sequence is based on the absence of active lac repressor. In order to achieve constitutive expression in any case, the amplification primers were selected in such way, that the lac operator is not present anymore in the PCR product. The resulting PCR product corresponded to pMCS5 from position 992 to 664 plus restriction overhangs. The nucleotides GAATTC (EcoRI restriction site) as well as TGTGTG were added before<br><br><br>
position 992 (3' to the lac promotor), and the nucleotides ACTAGT (Spe I restriction<br>
site) and CACACA were added behind position 664 to enable the ligation after<br>
digestion with the restrictions enzymes EcoRI and Spel. The coding sequence for<br>
UvsXH6, UvsXH6-2 or H6UvsX and H6UvsX-2 was obtained by PCR amplification of<br>
T4 DNA using primers which contain the desired restriction sites, a ribosomal binding<br>
site and the additional codons. At their 5' end before the start codon, the PCR<br>
products	contained	the	additional	nucleotides	5'-<br>
CACACAGAATTCATAAAGGAAGATATCAT-3', as well as the additional nucleotides 5'-ACTAGTTGTGTG-3' at their 3' end after the stop codon.<br>
Purification:<br>
An overnight culture of pExH-H6UvsX in DH5 was inoculated with 1.5-3 I dYT/ampicillin (200Î¼g/ml) 1:1000, and was grown over night at 37Â° C with 250 UPM. The cultures were harvested at 7000 x g, and yielded approx. 5-15 g bacteria sediment. This was frozen for 1-3 days at -20Â° C. The sediment was thawed on ice and resuspended in 10-20 ml cold starting buffer. The lysis was carried out under slow stirring for 1 h at 4Â° C using 10 ml lysozym (Serva, 190.000 u/mg) and approx. 4 g glass beads (Sigma, G-8893). Then, 50 Î¼l DNAse I (Serva, 2 mg/ml) were added and further incubated for another 30 min. After centrifugation of the lysate (45 min, 11.000 x g, 4Â° C), the supernatant was filtered through a sterile filter (pore size 0.45 urn) and loaded on an equilibrated 1 ml HiTrapTM Chelating column (Pharmacia) preloaded with Ni++ ions. The further purification steps were carried out according to the respective Pharmacia protocol for proteins containing a histidin hexamer. Aliquots of the various elution fractions were added to SDS/Coomassie gels. The purest fractions were combined and further concentrated in Centriplus YM30 columns (Millipore) according to the respective protocol. Then, it was dialyzed twice (dialysis tube: Spectra/Por, MWCO: 25.000) against an at least one thousand-fold volume of Zl buffer for at least one hour at 4Â° C, then over night at 4Â° C against Zl buffer/50 % glycerine. The dialysate was aliquoted in 30-50 Î¼l fractions and stored at -80Â° C.<br><br><br>
Used buffers:<br><br>
Determination of concentration:<br>
The concentration of the UvsX proteins was determined by measuring the OD28o using the extinction coefficient calculated with the Gene Inspectorâ¢ (Textco, Inc.) software. For H6UvsX it was 2.5-3.5 Î¼g/ul.<br>
Description of the experiment:<br>
H6UvsX, purified over Ni++ sepharose, was incubated with 200 ng of a 1 kb PCR<br>
fragment (with and without 1 mM ATP-y-S).<br>
A shift of the DNA in the agarose gel caused by protein binding shows, that H6UvsX<br>
binds double-stranded DNA based on the concentration, and that this binding is<br>
enhanced by ATP-y-S.<br>
With ATP-y-S, protein-DNA complexes are formed which can be stained more<br>
intensely with ethidium bromide than without ATP-y-S, which indicates a topological<br>
change of the nucleoprotein filament by the nucleotide analog.<br>
Example 2: Generation of a transfecting agent, based on UvsX as NPF forming protein with a nuclear localization signal as a functional component.<br>
As in example 1, plasmids were generated which permit the expression of the fusion proteins UvsXH6N2, UvsXH6N2-2, N2H6UvsX and N2H6UvsX-2, which are based on<br><br><br>
the proteins described in examplel, and in addition contain a nuclear localization signal (see Fig. 1).<br>
Structure of the proteins:<br>
UvsXH6N2 (426 amino acids):<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no.: AAD42669, amino acids 1-391), amino acids 392-394: linker consisting of the amino acids G392GS394, amino acids 395-400: H395HHHHH400, amino acids 401-403: linker consisting of the amino acids G401GS403, amino acids 404-417: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 418-421: C-terminus of UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 388-391), amino acids 422-426: C-terminus consisting of the amino acids K422LVTG426, amino acid exchange: Y238âV.<br>
UvsXH6N2-2 (420 amino acids):<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no: AAD42669,<br>
amino acids 1-391), amino acids 392-394: linker consisting of the amino acids<br>
S392YG394 , amino acids M401YG403 , amino acids 404-417: nuclear localization<br>
consisting of the amino acids M401YS403, amino acids 404-417: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 418-420: C-terminus consisting of the amino acids G418YP420.<br>
N2H6UvsX (420 amino acids):<br>
Amino acids 1-3: N-terminus consisting of the amino acids M1SY3, amino acids 4-17: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 18-20: linker consisting of the amino acids L18YS20, amino acids 21-26: H21HHHHH26, amino acids 27-29: linker consisting of the amino acids S27YG29, amino acids 30-420: UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 1-391), amino acid exchange: Q356âL<br><br><br>
N2H6UvsX-2 (421 amino acids):<br>
Amino acids 1-4: N-terminus consisting of the amino acids M1GYP4, amino acids 5-18: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 19-21: linker consisting of the amino acids S19YS21, amino acids 22-27: H22HHHHH27, amino acids 28-30: linker consisting of the amino acids S28YG30, amino acids 31-421: UvsX from the phage T4 (NCBI protein accession no: AAD42669, amino acids 1-391).<br>
Cloning of the expression plasmids:<br>
For the expression in suitable Escherichia coli cells, plasmids were constructed<br>
containing   a   coding   sequence   for   UvsXH6N2,   UvsXH6N2-2,   N2H6UvsX   or<br>
N2H6UvsX-2  under the  control  of the  lac promotor (pExH-UvsXH6N2,  pExH-<br>
UvsXH6N2-2, pExH-N2H6UvsX or pExH-N2H6UvsX-2, see Fig. 1). The plasmids<br>
were generated as described in example 1 by ligation of two PCR products (see Fig.<br>
1).<br>
Purification:<br>
The purification of UvsXH6N2 was carried out as described in example 1) for<br>
H6UvsX.<br>
Concentration: 10-20Î¼g/Î¼l.<br>
Description of the experiment:<br>
UvsXH6N2 binds to double-stranded DNA. The binding is stabilized by ATP-Î³-S: In order to examine the influence of various ATP analogues on the binding performance of UvsXH6N2 to DNA, the protein was first incubated with a 1 kb DNA fragment and various ATP analogues. Then, a 1.7 kb DNA fragment was added for competition. If the binding of the protein to the DNA is stabilized by addition of an ATP analog, it can be expected that UvsXH6N2 less likely binds a competing DNA fragment as long as no equilibrium is present. As can be seen in Fig. 3, the protein-<br><br><br>
DNA complex which was generated with the 1 kb fragment and UvsXH6N2, remains stable in absence of ATP-Î³-S and the 1.7 kb fragment, compared to all other used ATP analogues, i.e. the 1.7 kb fragment apparently is not or only marginally occupied by liberated UvsXH6N2 molecules within the observation time.<br>
Example 3: Generation of a transfection agent with a mixture of modified and unmodified NPF-forming protein<br>
Description of the experiment:<br>
Various ratios of H6UvsX and UvsxH6N2 were incubated with a 1 kb DNA fragment. The two proteins retarded the DNA in different degrees, due to their different molecular weight (see column 2 and 3 of Figure 4). If the proteins are mixed before addition of DNA, intermediate complexes are formed based on the ratio of H6UvsX to UvsXH6N2, which yield a sharp band and therefore have an equal mean molecular weight. This shows that the DNA is statistically equally occupied by both proteins.<br>
Example 4: Transfection of a ceil line (NIH3T3) with UvsX-NLS in combination with electroporation<br>
NIH3T3-Zellen (adherent, cultivated until 70 - 80 % confluent) were transfected with a vector coding for the heavy chain of the murine MHC class I proteins H-2KK. 1 x 106 cells were electroporated with 25 ng vector DNA which has been preincubated in binding buffer (76 mM K2HP04, 17 mM KH2P04, 14 mM NaH2P04, 5 mM MgCl2 pH 7.21) with 14 Î¼g UvsX or UvsX-NLS as well as with or without 1 mM ATP-Î³-S for 30 minutes at room temperature. For this, the cells were added to a total volume of 100 pi electroporation buffer (103 mM NaCI, 5.36 mM KCI, 0.41 mM MgCI2, 23.8 mM NaHC03, 5.64 mM Na2HP04) 11.2 mM glucose, 0.42 mM Ca(N03)2, 20 mM HEPES, 3.25 nM gluthathione) and electroporated in a cuvette with 2 mm electrode spacing. The electroporation was carried out by an exponential discharge at a voltage of 240 V<br><br><br>
and a capacity of 450 uF. The half-life of the voltage drop was typically 12 msec. Immediately after the electroporation, the cells were flushed out of the cuvettes with culture medium (RPMI with 10% FCS), incubated for 10 min at 37Â° C, and then transferred to a culture dish with prewarmed culture medium. After 6 h incubation, the cells were harvested and washed twice with PBS, and then incubated with a Cy5-coupled anti-H-2KK antibody and analyzed flow-cytometrically (FACScan). The number of dead cells was determined by staining with propidium iodine. Six hours after the electroporation, 7.4% or 8.7% of the cells transfected with free vector DNA express the H-2KK protein (minus the background of 0.25% in average). In comparison, the expression rate of the cells that have been transfected with vector UvsX was 2.9% or 3.8%. The expression rate after a transfection with vector UvsX-NLS was 19.2% or 18.9% (see Figures 5a-5d).<br>
For examination of the nuclear transport, the physical procedure of electroporation was chosen so that no other biochemical components except UvsX influence the transfection. The tight binding of UvsX with ATP-y-S to DNA, however, impairs the mobility of the complex in the electric field and therefore reduces the efficiency of the electroporation by approx. 60%. The attachment of a nuclear transport signal alone results in an increase of the expression shortly after the transfection by a factor of averagely 5.7 in this system. The increase of the expression rate by UvsX-NLS compared to UvsX therefore demonstrated that using a nuclear transport signal as functional component, DNA is transported into the nucleus by UvsX. An analysis performed shortly after the transfection is significantly improved since even cells become accessible that have not divided between transfection and analysis.<br>
Example 5: Transfection of a cell line (NIH3T3) with UvsX-scrambled-NLS or UvsX-NLS in combination with electroporation<br>
Generation of UvsX-scrambled-NLS<br><br><br>
In order to test the influence of the nuclear localization signal on the transfection, an UvsX derivative was generated for comparison purposes which corresponds in its net charge to an UvsX protein with a NILS, but itself does not contain a functional NLS.<br>
Using partly homologous oligonucleotide primers, the UvsX gene was amplified from plasmid pExH-UvsXH6N2-2 (compare Fig. 1), so that the amino acid sequence SEQ ID NO:1 (,,scrambled", i.e. a mixed NLS sequence) is expressed at the C-terminus of the resulting protein UvsXH6N2sc instead of the amino acid sequence EEDTPPKKKRKVED ("nls-2", corresponding to the amino acids 2-15 from SEQ ID No:9 from WO 00/40742). The scrambled amino acids correspond to the described nls-2 with respect to their composition but not with respect to their order. The net charges of UvsXH6N2-2 and UvsXH6N2sc are therefore equal, but only UvsXH6N2-2 contains an intact nuclear localization signal.<br>
The protein UvsXH6N2sc was purified as described for the proteins in example 1.<br>
NIH3T3 cells (adherent, cultivated until 70 - 80 % confluent) were transfected with this vector coding for a fluorescent marker protein. For this purpose, 25 ng vector DNA in binding buffer (see example 1) were initially incubated with 1.5 mM ATP-y-S and 16 -18 ug of the described proteins for 30 min at room temperature. The protein-DNA complexes were each added to 3 x 105 NIH3T3-Zellen, resuspended in 80 ul electroporation buffer (140 mM Na2HP4/NaH2PO4, 10 mM MgCI2, 5 mM KCI, pH 7.2). The electroporation was carried out in a cuvette with 2 mm electrode spacing by an exponential discharge at a voltage of 240 V and a capacity of 450 uF, The half life of the voltage drop was typically 12 msec. After addition of 400 Î¼l medium, composed of RPMI 1640, Gibco company, 5 % FCS, 2 mM glutamax (L-alanyl-L-glutamine, Invitrogen), 100 U/ml penicillin/streptomycin, 0.5 mM Ã-mercaptoethanol, the cells were added to culture dishes (6-well plates) with 1 ml prewarmed medium and were incubated at 37Â° C and 5% C02. After 6 h, the flow-cytometric analysis was carried out (FACScan).<br><br><br>
The result is graphically shown in Figure 6: 9 % of the cells transfected with free vector DNA express the marker protein. The expression rate of the cells that have been transfected with vector-UvsX-NLS (DNA+UvsXH6N2-2), however, was 21%. In comparison, the expression rate of the cells after transfection with vector-UvsX-scrambled-NLS (DNA+UvsXH6N2sc) was only 4 %.<br>
UvsX, modified with a nuclear localization signal therefore results in a markedly increased efficiency compared to free DNA and also compared to the modification by a non-functional nuclear localization signal. Since the latter transfection represents the control, this means that the transfection efficiency could be increased to 5-fold by modification of the UvsX. Therefore, the transfection efficiency can be markedly increased by the method according to the invention or the transfection agent. Furthermore a targeted control of the transfection method, here, for example, a directing into the nucleus, is possible in an advantageous way when the NPF-forming protein is modified (see also example 6).<br>
Example 6: Transfection of a cell line (NIH3T3) in combination with microinjektion<br>
140 ng of a 1.7 kb expression vector DNA fragment were incubated with 9 ug modified UvsX protein as described above in binding buffer and 1 mM ATP-y-S in a final volume of 20 Î¼l for 30 min at RT. BSA-Cy5 was used as injection marker immediately before the injection in a concentration of approx. 1 Î¼g/Î¼l. NIH3T3 cells that had been seeded to subconfluency the day before on CELLocate coverslips (Eppendorf) were microinjected through samples loaded onto Femtotipps (Eppendorf) using a micromanipulator and transjector (Eppendorf) under an inverse fluorescence microscope (Leica DMIL).<br>
The analysis was carried out in a fluorescence microscope (Olympus BX 60 fluorescence microscope, digital b/w camera SPOT-RT from Diagnostic Instruments Inc., analysis software: Metaview Imaging System from Universal Imaging Corporation) after 5 hours of further incubation of the cells at 37Â° C and 5% C02.<br><br><br>
Figure 7 shows microinjected NIH3T3 cells. The images 1 and 2 show cells that were injected with DNA and UvsXH6N2sc into the cytoplasm, the images 3 and 4 show cells that were injected with DNA and UvsXH6N2-2, and images 5 and 6 show cells that were injected only with DNA. Expression was only observed when the protein-DNA complexes contained UvsXH6N2-2, i.e. an UvsX protein modified with a nuclear localization signal (Fig. 7, image 3). In the cells of the controls (Fig. 7, image 5: only DNA, or Fig. 7, image 1: DNA with UvsX-scrambled-NLS) that had been clearly injected only in the cytoplasm and not in the nucleus during microinjection, no expression was observed, not even using a very long exposure.<br>
Example 7: Generation of recombinant SASP as NPF-forming proteins<br>
Cloning, expression and purification of the SASP protein from B. subtilis<br>
The SspC gene from Bacillus subtiliSi which codes for a SASP (âsmall acid-soluble spore p_rotein"), was synthesized from 8 oligonucleotides according to the Khorana method (described in Bertram and Gassen: Gentechnische Methoden, Gustav Fischer Verlag, 1991, p. 212-213) and ligated between the Ncol and Bglll restriction sites of the plasmid pARA13 (Cagnon et al., 1991; Protein Engng. 4: 843-847). This was based on the protein sequence with the NCBI protein accession no: NP_389876. The reverse transcription to DNA was carried out using the codon preferences of genes, strongly expressed in E. coli and described in (Andersson and Kurland 1990; Microbiol. Rev. 54: 98-210). The resulting plasmid pARA13-SASP served as a matrix for the cloning of two other plasmids, which code for SASP proteins, carrying a polyhistidine sequence of 6 histidines either at the N-terminus (H6-SASP) or at the C-terminus (SASP-H6) (Fig. 8). The plasmids were transformed into the E.coli strain BL21 (DE3) pLysS (Novagen, Madison) and plated out on LB/ampicillin/glucose (0.2%).<br><br>
20 ml M9 minimal medium (0.2% glucose) each was inoculated with single colonies and grown over night at 37Â° C and 220 rpm. The following day, the cultures were induced with 0.2% arabinose at an OD600 of approx. 1.0 and were grown for another 6 hours. Raw extracts (0.5 ml pelleted culture in PBS/loading buffer) were applied to high resolution SDS gels (according to Schagger and von Jagow 1987; Anal. Biochem. 166:368-79) and stained with Coomassie Blue (Fig. 9 A). For preparative purification, 2 L M9/glucose were inoculated 1:200 with an overnight culture. Again, the culture was induced with 0.2% arabinose at an OD600of approx. 1,0, and further grown for 6 h. The bacteria were then pelleted and frozen at -20Â° C. In the following, the purification of SASP-H6 is described exemplary. Approx. 7 g of the pellet was thawed and resuspended in 14 ml starting buffer (see example 1), supplemented with a tablet of complete EDTA-free protease inhibitor cocktail, Roche, Mannheim, and sonificated on ice for 3 min at 280 Watts (Labsonic U, Braun Biotech, Melsungen repeating duty puis; 0.5 sec). The extract was centrifuged at 4Â° C. Apart from this, the purification was carried out as described for the UvsX proteins over HiTrap chelating columns (Amersham Pharmacia, Uppsala). The fractions between 200 mM and 500 mM imidazole, containing the protein in high concentration, were combined and concentrated to approx. 3 ml using centriplus columns (YM-3, Millipore, Eschborn). The SASP protein was then dialyzed three times against 1 x Zl buffer (see example 1), and aliquots were frozen at -80Â° C in a concentration of approx. 5 ug/ul.<br>
Testing of the DNA-binding capacity of the SASP protein<br>
125 ng (in each case) of a 1.7 kb DNA fragment were preincubated with different amounts of SASP-H6 protein in 1 x Zl buffer or 1/10 x Zl buffer for 30 min at room temperature, and then applied to an 0.8% TAE-agarose gel.<br>
Figure 9 B shows that the DNA is completely bound by the protein. The diffuse appearance of the DNA-protein bands may be due to a dissociation of the proteins from the DNA during the gel run.<br><br>
Example 8: Generation of a transfection agent based on SASP as NPF-forming protein with a nuclear localization signal as modification<br>
For generation of an SASP with a nuclear localization signal (NLS) as functional component, a DNA sequence coding for a nuclear localization signal (ânls-2", amino acids 2-15, SEQ ID NO: 9 from WO 00/40742) was added to the C-terminus of the already available clone pARA13-SASP-H6 (see Fig. 8) by PCR amplification. The aforementioned plasmid served as PCR template. The resulting plasmid pARA13-SASP-H6N2 (see Fig. 8) was transformed as described in example 7, and the protein SASP-H6N2 was purified accordingly.<br>
Testing of the binding ability of the SASP-NLS protein<br>
125 ng each of a 1.7 kb DNA fragment were preincubated with different amounts of SASP-H6N2 protein in 1 x Zl-Puffer for 30 min at room temperature, and then applied to a 0.8 % TAE-agarose gel.<br>
Figure 10 B shows that the DNA is held back by the NLS-modified SASP as well. The DNA protein bands appear diffuse.<br>
Example 9: Generation of a transfection agent with a integrin binding motif for the association of the complex to the cell surface as functional component<br>
Integrins are membrane-anchored adhesion proteins on the cell surface some of which recognize a peptide motif of three amino acids (arginine-glycine-aspartic acid or âRGD" motif) as binding partner. The binding results in clustering of several integrin molecules on the cell surface and in endocytosis (Plow et al., 2001). By modification of UvsX as NPF-binding protein with an RGD motif, it can be achieved that the transfection agens can be taken up specifically via integrins into the endosomal compartments of the cells.<br><br>
Structure of the proteins<br>
The used proteins H6UvsX and UvsXH6N2-2 are identical to the ones shown in Figure 1 and described in example 1 and 2.<br>
The protein H6UvsX*(RGD2) was generated by chemical coupling. The protein named RGD2 is a nonapeptide (NITRGDTYI) consisting of the penton base protein of adenovirus type 7 (Bal et al., 2000), which has been synthesized in such way that a chemically active group is present at the N-terminal amino group (SMCC, succinimidyl 4[N-maleimidomethyl]-cyclohexane-1-carboxylat) which permits the coupling to free cysteine SH groups in the UvsX protein.<br>
For the coupling, 6 nmol H6UvsX were incubated with 60 nmol peptide in 76 mM K2HP04, 17mM KH2P04, 14mM NaH2P04, pH=7.2 for one hour at 37Â° C, and was then purified by several washing steps with incubation buffer over a MicroCon filter (10 kDa cut off) to remove excess peptide. The successful coupling was detected by an altered running performance in an SDS polyacrylamide gel electrophoresis. Figure 11 shows two independently generated preparations of H6UvsX*(RGD2) on an SDS polyacrylamide gel. The coupled peptide results in an increase of the molecular weight and thus in an alteration of the running performance in SDS gels.<br>
Description of the experiment:<br>
Binding of UvsX-NLS and UvsX-RGD to double-stranded DNA and formation of mixed NPFs:<br>
Reactions with 140 ng (in each case) of a purified 1.6 kb PCR DNA fragment with a mixture of 15 ug H6UvsX*(RGD2) and 4 ug UvsXH6N2-2, with 4 ug UvsXH6N2-2 alone and with 15 ug H6UvsX*(RGD2) alone were incubated in 76 mM K2HP04, 17 mM KH2P04, 14 mM NaH2P04, pH=7.2, 5 mM MgCI2 and 1 mM ATP-y-S in a final volume of 20 ul for 30 min at room temperature and were then transferred to a 0.8%<br><br><br>
TAE/agarose gel which was afterwards stained with ethidium bromide. The electrophoresis was carried out at 100 V for 1 h.<br>
Figure 12 shows that the two differently modified proteins form NPFs with the DNA which markedly differ in their running performance. NPFs consisting of a double-stranded DNA fragment and a mixture of H6UvsX*(RGD2) and UvsXH6N2-2 (lane 1), or of UvsXH6N2-2 alone (lane 2) or of H6UvsX*(RGD2) alone (lane 3), were separated electrophoretically in an agarose gel. Since protein is present in low amounts, free DNA fragment is present as well. Both proteins in a reaction bind together to the DNA fragment and result in a mixed NPF that has a molecular weight that lies between that of the NPFs of the pure proteins (lane 1). From this it can be concluded that UvsX-NLS and UvsX-RGD bind to double-stranded DNA and can form mixed NPFs.<br>
Example 10: Specific uptake of NPFs into cells by integrin-mediated endocytosis<br>
Structure of the proteins<br>
As in example 1, plasmid UvsXH6N2NIT-2 (Fig. 1) which permits the expression of a fusion protein of UvsX and an integrin-binding RGD motif, was generated recombinantly.<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no.: AAD42669, amino acids 1-391), amino acids 392-394: linker consisting of the amino acids s392YG394, amino acids 395-400: H395HHHHH400, amino acids 401-403: linker consisting of the amino acids M401YS403, amino acids 404-417: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 418-420: C-termfnus consisting of the amino acids G418YP42D and amino acids 421-432: RGD motif âNIT": N421IT RGDTYIPYP432.<br><br><br>
Description of the experiment:<br>
Generation of NPFs from fluorescence-labeled DNA and UvsX derivatives:<br>
1 ug each of a purified 1.6 kb PCR DNA fragment containing AlexaFluor488-labeled dUTP (Molecular Probes, Eugene, Oregon, USA) instead of dTTP, was incubated in 76 mM K2HP04, 17 mM KH2P04, 14 mM NaH2P04l pH=7.2, 5 mM MgCI2 and 1 mM ATP-y-S withe 100 ug purified UvsXH6N2 or UvsXH6N2NIT-2 in a final volume of 200 ul for 30 min at room temparature. Then, 10 ul of each NPF reaction was applied to a 0.8% TAE/agarose gel which was subsequenty stained with ethidium bromide. The electrophoresis was carried out for 1 hour at 100 volts. Figure 13 shows that the DNA was completely retarded. Thus, the DNA binding of the proteins is not hampered by the fluorescein labelling of the DNA.<br>
Uptake of the NPFs in NIH3T3 cells by endocytosis:<br>
NIH3T3 cells were plated in 6 well plates (3x105 per well), incubated over night at 37Â° C and 5 % CO2, and were washed the next morning with prewarmed FCS-free medium; afterwards 2 ml FCS-free medium was added. 190 ul of the NPF reactions were added (see above), incubated for 30 min at room temperature, the supernatant was removed, the cells were washed and covered with 3 ml medium (with 10% FCS). After another incubation for 1 hour at 37 Â°C in the incubator, the analysis was carried out under the fluorescence microscope. In Figure 14a and 14b, one picture each is shown in bright field (lower) and reflected light fluorescence (upper)<br>
In the bright field, vesicular intracellular compartments are visible which glow in the fluorescent light due to the endocytosed DNA (Fig. 14a, b, upper).<br>
From each well, several pictures were taken. The cells were counted and the proportion of cells containing at least one fluorescent vesicular compartment was determined in percent (shown in Figure 15). Each picture shows a mean of 35 cells.<br><br><br>
Of the reactions with UvsXH6N2NIT-2, nine pictures were analyzed, and of the reactions with UvsXH6N2-2, five pictures were analyzed.<br>
The result shows that the modification of UvsX with an integrin-binding motif as a functional component (UvsXH6N2-NIT-2) results in a markedly increased endocytotic uptake of the transfection agents in the cells compared to the control (UvsXH6N2-2).<br>
Example 11:    Generation of a transfection agent based on hRad51 as NPF-forming protein<br>
The proteins hRad51H6 and hRad51H6N2 were used as NPF-forming proteins (see Figure 16).<br>
Structure of the proteins<br>
hRad51H6 (352 amino acids):<br>
Amino acids 1-339: human Rad51 (NCBI protein accession no: Q06609, amino acids 1-339), amino acids 340-343: linker consisting of the amino acids Y340SYG343, amino acids 344-349: H344HHHHH349 for purification by nickel chelate affinity chromatography, amino acids 350-352: C-terminus consisting of the amino acids<br>
M350YS352<br>
hRad51H6N2 (369 amino acids):<br>
Amino acids 1-339: human Rad51 (NCBI protein accession no: Q06609, amino acids 1-339), amino acids 340-343: linker consisting of the amino acids Y340SYG343, amino acids 344-349: H344HHHHH349 for purification by nickel chelate affinity chromatography, amino acids 350-352: linker consisting of the amino acids M350YS352, , amino acids 353-366: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 from WO 00/40742), amino acids 367-369: C-terminus consisting of the amino acids G367YP369.<br><br><br>
Cloning of the expression plasmids<br>
For expression of the above mentioned proteins in suitable Escherichia coli cells, plasmids were constructed that contain a coding sequence for hRad51H6 or hRad51H6N2 under the control of the lac promoter (pExH-hRad51H6 or pExH-hRad51H6N2, see Fig. 16). pExH-UvsXH6-2 and pExH-UvsXH6N2-2 were used as source plasmids (see Fig. 16). The coding region for UvsX was cut out with EcoR V and BsiW I, and replaced by a PCR fragment with the coding region for hRad51 which had been cut out in the same way. This was amplified from a human cDNA library using hRad51-specific primers that contain the desired restriction sites. At the 5'-end before the start codon, the PCR products contained the additional nucleotides 5'-CACACATCTAGACGTACGGATATCAT-3', and at their 3'-end they contained the additional nucleotides 5"-TACTCGTACGGAGGTGGCGGCCGCTGTGTG-3' instead of the stop codon.<br>
Purification:<br>
A preculture of 5 ml dYT/ampicillin (100ug/ml) was inoculated with a colony of pExH-Rad51H6 or pExH-Rad51H6N2 in DH5, and was grown for 5 hours at 37Â° C with 250 rpm. 10 I dYT/ampicillin (100ug/ml) was inoculated with this preculture, and was allowed to grow for another 24 hours at 37Â° C with 210 UPM. The cultures were harvested at 7000 x g, and yielded approx. 30-50 g bacterial pellet. This was frozen for 1-3 days at -20Â° C. The pellet was thawed on ice, and resuspended in 100 ml cold starting buffer. The cells were then solubilized by ultrasound using a B. Braun Labsonic U (large probe, parameters: 300 watts, 0.5 sec pulse duration per second, 8 min sonification). Then it was incubated with 10 mg lysozyme (Serva, 190.000 u/mg) for 1 hours at 4Â° C and for another 30 min after addition of 50 ul DNAse I (Serva, 2 mg/ml) with slow stirring. After removing the lysate by centrifugation (45 min, 18000 x g, 4 Â°C), the supernatant was filtered through sterile filters (pore sizes 0.45 urn and 0.2 urn) and loaded on an equilibrated 1 ml HiTrapTM chelating column (Pharmacia) preloaded with Ni++ ions. The further purification steps were carried out according to<br><br><br>
the respective Pharmacia protocol for proteins that have been provided with a histidine hexamer. Aliquots of the various elution fractions were applied to SDS/Coomassie gels. The purest fractions were combined and were further concentrated over Centriplus YM30 columns (Millipore) according to the respective protocol. Then it was dialyzed twice (dialysis tubing: Spectra/Por, MWCO: 25.000) against an at least one thousand-fold volume of Zl buffer for 1 hour each at 4Â° C, then over night at 4Â° C against ZI buffer/50 % glycerine. The dialysis product was aliquoted in 30-50 pi fractions and stored at -80Â° C. Fig. 17 A shows the purified hRad51-H6 and hRad51-H6N2 proteins.<br>
Used buffer:<br>
As in example 1, but different elution buffer: 20 mM Pi, 0.5 M NaCI, 100-1000 mM imidazole, pH=7.4<br>
Determination of concentration:<br>
The concentrations of the hRad51 proteins was determined by measurement of the OD28o using the extinction coefficient calculated with the Gene InspectorTM software (Textco, Inc.). With hRad51 H6 and hRad51 H6N2 it was 11 -13 Î¼g/Î¼l.<br>
Description of the experiment:<br>
Binding of NLS-modified hRad51 to double-stranded DNA:<br>
hRad51H6N2purified over Ni++-sepharose was incubated with 100 ng each of a 0.9 kb PCR fragment.<br>
A DNA shift in the agarose gel caused by the protein binding shows that hRad51H6N2 cooperatively binds double-stranded DNA based on the concentration (Fig.  17B). Even with low amounts of hRad51H6N2, single DNA molecules are<br><br><br>
completely bound by hRad51H6N2, and are therefore retarded maximally, so that the retardation of the DNA does not increase any more by increasing the amount of protein. It is concluded that hRad51H6N2 binds dsDNA.<br>
Example 12: Generation of a transfection agent based on UvsX with a signal for the non-endosomal membrane permeation and a nuclear localization signal as functional component<br>
As in example 1, a plasmid was generated that permits the expression of the fusion protein UvsXH6N2VP22c50 (see Fig. 1), which additionally contains a part of the tegument protein VP22 (gene UL49) of the human herpes virus 1 and is based on the protein UvsXH6N2-2 described in example 2. The here used VP22 peptide acts as a signal for the non-endosomal permeation through the cell membrane. Thus, the fusion protein UvsXH6N2VP22c50 contains a membrane transduction signal in addition to a nuclear localization signal (NSL).<br>
Structure of the protein<br>
UvsXH6N2VP22c50 (474 amino acids):<br>
Amino acids 1-391: UvsX from the phage T4 (NCBI protein accession no: AAD42669,<br>
amino acids 1-391), amino acids 392-394: linker consisting of the amino acids<br>
s392YG394, amino acids M401YS403, amino acods 401-403: linker<br>
sconsisting of the amino acids M401YS403, amino acids 404-417: nuclear localization signal nls-2 (amino acids 2-15, SEQ ID NO: 9 aus WO 00/40742), amino acids 418-422: linker consisting of the amino acids G418YPGS422, amino acids 423-472: part of the tegument protein VP22 (Gen UL49) of the human herpes virus 1 (NCBI protein accession no: NP_044651, amino acids 252-301), amino acids 473-474: C-terminus consisting of the amino acids P473R474.<br>
Cloning of the expression plasmid:<br><br>
For the expression in suitable Escherichia coli cells, pExHUvsXH6N2-2 (see example 1 and Fig. 1) was opened by restriction enzyme digestion with Acc65 I and Spe I, and was ligated with a PCT product cut with Acc65 I and Nhe I, which contains at the 5'-end the additional nucleotides 5'-CACACAGGTACCCGGGATCC-3' and at its 3'-end the additional nucleotides 5'-CCTAGGTAATAATAAGCGGCCGCGCTAGCTGTGTG-3', in addition to the coding sequence for the last 50 amino acids of the tegument protein VP22 (gene UL49) of the human herpes virus 1 (NCBI nucleotide accession no: NC_001806, complementary sequence of the nucleotides 105486-106391) (see Figure 1).<br>
Purification:<br>
The purification of UvsXH6N2VP22c50 was carried out as described in example 1 for H6UvsX (see Fig. 18A). Concentration: 1.8Î¼g/Î¼l.<br>
Description of the experiment:<br>
Binding of a mixture of various modified UvsX (UvsX-NLS-VP22 and UvsX-NLS) to double-stranded DNA:<br>
140 ng (in each case) of a purified 1.7 kb PCR fragments were incubated in 96 mM K2HP04, 21.5mM KH2PO4, l8mM NaH2P04l pH=7.2, 5mM MgCI2 and 1.3 mM ATP-y-S with the amounts according to Fig. 18B of purified UvsXH6N2VP22c50 or UvsXH6N2-2 for 30 min at room temperature, then, all reactions were applied to a 0.8% TAE/agarose gel which was afterwards stained with ethidium bromide. The two proteins were different regarding their molecular weight and net charge, and therefore retarded the DNA differently during electrophoresis, with the complex with UvsXH6N2VP22c50 remaining stuck in the gel pocket and not migrating any more (see lanes 1 and 7 of Figure 18B). When the proteins are mixed before they are added to the DNA, intermediate complexes are formed depending on the ratio of<br><br><br>
UvsXH6N2-2 and UvsXH6N2VP22c50, the migration performance of which is between those of the unmixed complexes (Fig. 18B). This shows that the DNA is occupied by both proteins. Also possible is a mixture of differently modified or one- or two-times modified NPF-forming proteins.<br>
Example 13: Transfection of a cell line (NIH3T3) with complexes of DNA and a mixture of UvsX-NLS-VP22 and UvsX-NLS<br>
Description of the experiment<br>
2.5 x 105 cells (NIH3T3) were plated out in each well of a 6-well plate, and transfected on the following day with a vector containing a gene for the expression of a fluorescent reporter protein. For this, 0 ug - 1 ug linear or 1 ug circular DNA was preincubated with 36 ug UvsXH6N2VP22c50 or a mixture of 19 ug UvsXH6N2VP22c50 and 39 ug UvsXH6N2-2 in binding buffer (76 mM K2HP04, 17 mM KH2P04, 14 mM NaH2P04, 5 mM MgCI2, 1 mM ATP-y-S, pH 7.21) for 30 min at room temperature, and together with 1 ml RPMI was added to cells that have before been washed once with PBS/BSA. After a 1 h incubation at 37Â°C, 5% C02 in the incubator, 1 ml RPMI/20% FCS was added respectively and further incubated in the incubator. 4 h or 24 h later, the cells were analyzed in the fluorescence microscope. Cells were observed that expressed the reporter gene after treatment with complexes of linear or circular DNA and the mixture of UvsXH6N2VP22c50 and UvsXH6N2-2 (see Fig. 19). The number of the transfected cells increased with the amount of used DNA or DNA-protein complexes.<br>
However, no expression of the reporter gene was achieved with DNA complexes containing only UvsXH6N2VP22c50, or in absence of DNA.<br>
Therefore it appears that by modification of an NPF-forming protein, here UvsX, with a membrane-active peptide, here VP22, the transfection of cells and here especially the membrane permeation is facilitated. The modular character of the method or<br><br><br>
transfection agent according to the invention is underlined by the combination of differently modified proteins, here modification with VP22 and NLS (see also Fig. 20). Whereas the VP22-modified UvsX permits the non-endosomal membrane permeation, the NLS-modified UvsX directs the transfected DNA from the cytoplasma to the nucleus; the transfected DNA can then be expressed there.<br>
Thus, individual steps of the complex transfection procedure can be controlled specifically, flexible and with high efficiency in especially advantageous manner.<br><br><br><br>
References<br>
Andersson, K. (1990). Codon Preferences in Free-Living Microorganisms. Micobiol Rev 54, 98-210.<br>
Bal, H. P., Chroboczek, J., Schoehn, G., Ruigrok, R. W., and Dewhurst, S. (2000). Adenovirus type 7 penton purification of soluble pentamers from Escherichia coli and development of an integrin-dependent gene delivery system. Eur J Biochem 267, 6074-6081.<br>
Baumann, P., and West, S. C. (1998). Role of the human RAD51 protein in homologous recombination and double-stranded-break repair. Trends Biochem Sci 23, 247-251.<br>
Bianco, P. R., Tracy, R. B., and Kowalczykowski, S. C. (1998). DNA strand exchange proteins: a biochemical and physical comparison. Front Biosci 3, D570-603.<br>
Boulikas, T. (1993). Nuclear localization signals (NLS). Crit Rev Eukaryot Gene Expr 3, 193-227.<br>
Boulikas, T. (1996). Nuclear import of protein kinases and cyclins. J Cell Biochem 60, 61-82.<br>
Boulikas, T. (1997). Nuclear import of DNA repair proteins. Anticancer Res 17, 843-863.<br>
Cagnon, C, Valverde, V., and Masson, J. M. (1991). A new family of sugar-inducible expression vectors for Escherichia coli. Protein Eng 4, 843-847.<br>
Cerutti, H., Osman, M., Grandoni, P., and Jagendorf, A. T. (1992). A homolog of Escherichia coli RecA protein in plastids of higher plants. Proc Natl Acad Sci U S A 89, 8068-8072.<br>
Collins, L, Sawyer, G. J., Zhang, X. H., Gustafsson, K., and Fabre, J. W. (2000). In vitro investigation of factors important for the delivery of an integrin-targeted nonviral DNA vector in organ transplantation. Transplantation 69, 1168-1176.<br>
Delcayre, A. X., Salas, F., Mathur, S., Kovats, K., Lotz, Mâ and Lernhardt, W. (1991). Epstein Barr virus/complement C3d receptor is an interferon alpha receptor. Embo J 10, 919-926.<br>
Di Capua, E., Engel, A., Stasiak, A., and Koller, T. (1982). Characterization of complexes between recA protein and duplex DNA by electron microscopy. J Mol Biol 157, 87-103.<br><br><br>
Ellouze, C, Selmane, T., Kim, H. K., Tuite, E., Norden, B., Mortensen, K., and Takahashi, M. (1999). Difference between active and inactive nucleotide cofactors in the effect on the DNA binding and the helical structure of RecA filament dissociation of RecA--DNA complex by.inactive nucleotides. Eur J Biochem 262, 88-94.<br>
Evan, G. I., Lewis, G. K., Ramsay, G., and Bishop, J. M. (1985). Isolation of monoclonal antibodies specific for human c-myc proto- oncogene product. Mol Cell Biol 5, 3610-3616.<br>
Feero, W. G., Li, S., Rosenblatt, J. D., Sirianni, N., Morgan, J. E., Partridge, T. A., Huang, L., and Hoffman, E. P. (1997). Selection and use of ligands for receptor-mediated gene delivery to myogenic cells. Gene Ther 4, 664-674.<br>
Feldherr, C. M., and Akin, D. (1997). The location of the transport gate in the nuclear pore complex. J Cell Sci 110(Pt 24), 3065-3070.<br>
Fominaya, J., and Wels, W. (1996). Target cell-specific DNA transfer mediated by a chimeric multidomain protein. Novel non-viral gene delivery system. J Biol Chem 271, 10560-10568.<br>
Griffith, J., Makhov, A., Santiago-Lara, L, and Setlow, P. (1994). Electron microscopic studies of the interaction between a Bacillus subtilis alpha/beta-type small, acid-soluble spore protein with DNA: protein binding is cooperative, stiffens the DNA, and induces negative supercoiling. Proc Natl Acad Sci U S A 91, 8224-8228.<br>
Harbottle, R. P., Cooper, R. G., Hart, S. L, Ladhoff, A., McKay, T., Knight, A. M., Wagner, E., Miller, A. D., and Coutelle, C. (1998). An RGD-oligolysine peptide: a prototype construct for integrin-mediated gene delivery. Hum Gene Ther 9, 1037-1047.<br>
Hong, S. S., Gay, B., Karayan, L., Dabauvalle, M. C, and Boulanger, P. (1999). Cellular uptake and nuclear delivery of recombinant adenovirus penton base. Virology 262, 163-177.<br>
Karlin, S., and Brocchieri, L. (1996). Evolutionary conservation of RecA genes in relation to protein structure and function. J Bacteriol 178, 1881-1894.<br>
Karlin, S., Weinstock, G. M., and Brendel, V. (1995). Bacterial classifications derived from recA protein sequence comparisons. J Bacteriol 177, 6881-6893.<br>
Knight, K. L., and McEntee, K. (1985). Affinity labeling of a tyrosine residue in the ATP binding site of the recA protein from Escherichia coli with 5'-p-fluorosulfonylbenzoyladenosine. J Biol Chem 260, 10177-10184.<br><br><br>
Kukowska-Latallo, J. F., Bielinska, A. U., Johnson, J., Spindler, R., Tomalia, D. A., and Baker, J. R., Jr. (1996). Efficient transfer of genetic material into mammalian cells using Starburst polyamidoamine dendrimers. Proc Natl Acad Sci U S A 93, 4897-4902.<br>
Lee, C. K., and Knipe, D. M. (1985). An immunoassay for the study of DNA-binding activities of herpes simplex virus protein ICP8. J Virol 54, 731-738.<br>
Masson, J. Y., Davies, A. A., Hajibagheri, N., Van Dyck, E., Benson, F. E., Stasiak, A. Z., Stasiak, A., and West, S. C. (1999). The meiosis-specific recombinase hDmd forms ring structures and interacts with hRad51. Embo J 18, 6552-6560.<br>
Mengaud, J., Ohayon, H., Gounon, P., Mege, R. M., and Cossart, P. (1996). E-cadherin is the receptor for internalin, a surface protein required for entry of L. monocytogenes into epithelial cells. Cell 84, 923-932.<br>
Midoux, P., Mendes, C, Legrand, A., Raimond, J., Mayer, R., Monsigny, M., and Roche, A. C. (1993). Specific gene transfer mediated by lactosylated poly-L-lysine into hepatoma cells. Nucleic Acids Res 21, 871-878.<br>
Mosig, G. (1987). The essential role of recombination in phage T4 growth. Annu Rev Genet 21, 347-371.<br>
Neumann, G., Castrucci, M. R., and Kawaoka, Y. (1997). Nuclear import and export of influenza virus nucleoprotein. J Virol 71, 9690-9700.<br>
Ogawa, T., Shinohara, A., Nabetani, A., Ikeya, T., Yu, X., Egelman, E. H., and Ogawa, H. (1993). RecA-like recombination proteins in eukaryotes: functions and structures of RAD51 genes. Cold Spring Harb Symp Quant Biol 58, 567-576.<br>
Ohno, K., Sawai, K., lijima, Y., Levin, B., and Meruelo, D. (1997). Cell-specific targeting of Sindbis virus vectors displaying IgG-binding domains of protein A. Nat Biotechnol 15, 763-767.<br>
Pack, D. W., Putnam, D., and Langer, R. (2000). Design of imidazole-containing endosomolytic biopolymers for gene delivery. Biotechnol Bioeng 67, 217-223.<br>
Plow, E. F., Haas, T. A., Zhang, L, Loftus, J., and Smith, J. W7(2000). Ligand binding to integrins. J Biol Chem 275, 21785-21788.<br>
Pooga, M., Soomets, U., Hallbrink, M., Valkna, A., Saar, K., Rezaei, K., Kahl, U., Hao, J. X., Xu, X. Jâ Wiesenfeld-Hallin, Z., etal. (1998). Cell penetrating PNA constructs regulate galanin receptor levels and modify pain transmission in vivo. Nat Biotechnol 16, 857-861.<br><br><br>
[<br>
Provoda, C. J., and Lee, K. D. (2000). Bacterial pore-forming hemolysins and thei use in the cytosolic delivery of macromolecules. Adv Drug Deliv Rev 41, 209-221.<br>
Richardson, S., Ferruti, P., and Duncan, R. (1999). Poly(amidoamine)s as potential endosomolytic polymers: evaluation in vitro and body distribution in normal and tumour-bearing animals. J Drug Target 6, 391-404.<br>
Roca, A. I., and Cox, M. M. (1990). The RecA protein: structure and function. Crit Rev Biochem Mol Biol 25, 415-456.<br>
Rosenkranz, A. A., Yachmenev, S. V., Jans, D. A., Serebryakova, N. V., Murav'ev, V. I., Peters, R., and Sobolev, A. S. (1992). Receptor-mediated endocytosis and nuclear transport of a transfecting DNA construct. Exp Cell Res 199, 323-329.<br>
Sandler, S. J., Satin, L. H., Samra, H. S., and Clark, A. J. (1996). recA-like genes from three archaean species with putative protein products similar to Rad51 and Dmc1 proteins of the yeast Saccharomyces cerevisiae. Nucleic Acids Res 24, 2125-2132.<br>
Schagger, H., and von Jagow, G. (1987). Tricine-Sodium Dodecyl Sulfate-Polyacrylamide Gel Electrophoresis for the Separation of Proteins in the Range from 1 to 100 kDa. Anal Biochem 166, 368-379.<br>
Schoeman, R., Joubert, D., Ariatti, M., and Hawtrey, A. O. (1995). Further studies on targeted DNA transfer to cells using a highly efficient delivery system of biotinylated transferrin and biotinylated polylysine complexed to streptavidin. J Drug Target 2, 509-516.<br>
Seitz, E. M., Brockman, J. P., Sandler, S. J., Clark, A. J., and Kowalczykowski, S. C. (1998). RadA protein is an archaeal RecA protein homolog that catalyzes DNA strand exchange. Genes Dev 12, 1248-1253.<br>
Steinhauer, D. A., Wharton, S. A., Skehel, J. J., and Wiley, D. C. (1995). Studies of the membrane fusion activities of fusion peptide mutants of influenza virus hemagglutinin. J Virol 69, 6643-6651.<br>
Surdej, P., and Jacobs-Lorena, M. (1994). Strategy for epitope tagging the protein-coding region of any gene. Biotechniques 17, 560-565.<br>
Tang, M. X., and Szoka, F. C. (1997). The influence of polymer structure on the interactions of cationic polymers with DNA and morphology of the resulting complexes. Gene Ther 4, 823-832.<br>
Thoren, P. E., Persson, D., Karlsson, M., and Norden, B. (2000). The antennapedia peptide penetratin translocates across lipid bilayers - the first direct observation. FEBS Lett 482, 265-268.<br><br><br>
Thyagarajan, B., Padua, R. A., and Campbell, C. (1996). Mammalian mitochondria possess homologous DNA recombination activity. J Biol Chem 271, 27536-27543.<br>
Wagner, E. (1999). Application of membrane-active peptides for nonviral gene delivery. Adv Drug Deliv Rev 38, 279-289.<br>
Wang, P., Palese, P., and O'Neill, R. E. (1997). The NPI-1/NPI-3 (karyopherin alpha) binding site on the influenza a virus nucleoprotein NP is a nonconventional nuclear localization signal. J Virol 71, 1850-1856.<br>
Weisbart, R. H., Baldwin, R., Huh, B., Zack, D. J., and Nishimura, R. (2000). Novel protein transfection of primary rat cortical neurons using an antibody that penetrates living cells. J Immunol 164, 6020-6026.<br>
Yamada, M., and Kasamatsu, H. (1993). Role of nuclear pore complex in simian virus 40 nuclear targeting. J Virol 67, 119-130.<br>
Yu, X., and Egelman, E. H. (1993). DNA confon T4 UvsX protein appears identical to the confon RecA protein. J Mol Biol 232, 1-4.<br>
Zauner, W., Blaas, D., Kuechler, E., and Wagn
<br><br>
WE CLAIM:<br>
1.	An in vitro method tor transiection of culkaryotic cells for producing a cell as a drug for noivviral gone therapy using at least one protein capable of forming nucleoprotein filaments, wherein the protein .is initially modified with at least one functional component, which influences one or more steps of the transfection a nucleic acid to be transfected is then loaded with the modified protein, the nucleic acid and the protein forming a filament-like complex, and this complex is finally added to the cells to be transfected; wherein functional component causes, association of the complex to the cellular surface,  non-endosomal passage of the complex through the cell membrane, release of the complex from endosomes/lysosomes and transport of the complex into the nucleus.<br>
2.	The method as claimed in claim 1, wherein the protein is modified by deletion or insertion of amino acids and/or protein domains.<br>
3.	The method as claimed in claim 1 oder 2, wherein the protein is modified by chemical alteration of amino acids and/or other molecule groups.<br>
4.	The method as claimed in claim 1, 2 oder 3, wherein the protein is modified by chemical coupling of peptides, proteins, carbohydrates, lipids or other molecules.<br>
5.	The method as claimed in any one of the claims 1 to 4, wherein the protein is modified with several functional components of different function, and/or wherein different proteins are modified with components of different function, respectively.<br>
6.	The method as claimed in anv one of the claims 1 to 5, wherein the protein is loaded with a plurality of functional components.<br><br><br>
The method as claimed in anv one of the claims 1 to 6, wherein the complex is stabilized by addition of nucleoside triphosphates and/or non-hydrolyzab'e analogues thereof, especially by ATP (adenosintriphosphate) and/or GT (guanosintriphosphate) and/or their non-hydrolyzable analogues.<br>
The method as claimed in claims 1 to 4, wherein the protein selected from the group of the proteins RecA, RadA, RAD51, hDrnd, SASP, ICP8, preferably UvsX, more preferably hRAD51 or a mixture of at least 2 of the listed proteins.<br>
The method as claimed in anv one of the claims 1 to 8 being employed in combination with other biological and/or chemical and/or physical transfection methods for nucleic acids, especially in combination with electroporation.<br>
Dated this 27th day of June;, 2003.<br>
HIRAL CHANDRAKANT JOSHI AGENT FOR                      AMAXA GMBH</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtY2FuY2VsbGVkIHBhZ2UoMjktMy0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-cancelled page(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgyOS0zLTIwMDUpLmRvYw==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-claims(granted)-(29-3-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgyOS0zLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-claims(granted)-(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoMy00LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-correspondence(3-4-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoaXBvKS0oMTgtNS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-correspondence(ipo)-(18-5-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZHJhd2luZygyOS0zLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-drawing(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAxOSgxNy0yLTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 19(17-2-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAxYSgyOC02LTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 1a(28-6-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgyOS0zLTIwMDUpLmRvYw==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 2(granted)-(29-3-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgyOS0zLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 2(granted)-(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAzKDE3LTMtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 3(17-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAzKDI4LTYtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 3(28-6-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSAzKDI5LTMtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 3(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybSA1KDI1LTYtMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form 5(25-6-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtZm9ybS1wY3QtaXNhLTIxMCgxMC05LTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-form-pct-isa-210(10-9-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtcG93ZXIgb2YgYXR0b3JuZXkoMjgtNy0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-power of attorney(28-7-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU3LW11bW5wLTIwMDMtcG93ZXIgb2YgYXR0b3JuZXkoMjktMy0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">657-mumnp-2003-power of attorney(29-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="205471-a-device-for-controlled-release-of-fluid-water-under-gravity-flow-and-an-improved-purification-sanitzing-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="205473-a-method-for-production-of-pharmaceutical-composition-conatining-azithromycin-for-treatment-of-non-infective-inflammatory-diseases.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>205472</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>657/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Jun-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>03-Apr-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Jun-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AMAXA GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NATTERMANNALLEE 1, 50829 KOLN, GERMANY.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SCHMIDT, HANNS-MARTIN</td>
											<td>SIEBACHSTRASSE 36 50733 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALTROGGE LUDGER</td>
											<td>AMSTERDAMER STRASSE 13 50259 PULHEIM, GERMANY.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LENZ DIETMAR</td>
											<td>AN DER MALSMUHLE, 2, 50676 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RIEMEN GUDULA</td>
											<td>PATER-KOLBE-WEG, 7, 40764 LANGENFELD, GERMANY,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>BROSTERHUS HELMUT</td>
											<td>IN DER RAHRBACH, 20 57399 KIRCHHUNDERN, GERMANY.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>LORBACH ELKE</td>
											<td>LOTHRINGER STRASSE, 85, 50677 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>HELFRICH JULIANA</td>
											<td>HOFESTRASSE 27, 51103 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>HEIN KATHARINA</td>
											<td>ACHTERSTRASSE 54, 50678 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>GREMSE MARION</td>
											<td>MEISENWEG 41, 50829 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>10</td>
											<td>MALES TATJANA</td>
											<td>DAGOBERTSTRASSE 4, 40237 HILDEN, GERMANY.</td>
										</tr>
										<tr>
											<td>11</td>
											<td>CHRISTINE RAINER</td>
											<td>LANDGRAFENSTRASSE 6, 50931 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>12</td>
											<td>SIEBENKOTTEN GREGOR</td>
											<td>SANKT-MAGDALENEN-STRASSE 69, 50226 FRECHEN-KONIGSDORT, GERMANY.</td>
										</tr>
										<tr>
											<td>13</td>
											<td>ORTMANN BODO</td>
											<td>SEVERINSTRASSE 54, B 50678 KOLN, GERMANY.</td>
										</tr>
										<tr>
											<td>14</td>
											<td>TURBANSKI TAMARA</td>
											<td>HUCKJAUSENER STRASSE, 43, 51674 WIEHL, GERMANY.</td>
										</tr>
										<tr>
											<td>15</td>
											<td>KLAES ANDREA</td>
											<td>DECHSTEINER STRASSE, 10, 50935 KOLN, GERMANY.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 N 15/87</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DE02/00060</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-01-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>101 00 996.8</td>
									<td>2001-01-10</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/205472-in-vitro-method-for-transfection-of-eukaryotic-cell-for-producing-drug-for-non-viral-gene-therapy by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:26:21 GMT -->
</html>
