<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/205451-a-method-for-the-preparation-of-a-kit-for-detecting-hepatitis-b-virus-hbv-nucleic-acid-in-biological-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:29:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 205451:A METHOD FOR THE PREPARATION OF A KIT FOR DETECTING HEPATITIS B VIRUS (HBV) NUCLEIC ACID IN BIOLOGICAL SAMPLES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR THE PREPARATION OF A KIT FOR DETECTING HEPATITIS B VIRUS (HBV) NUCLEIC ACID IN BIOLOGICAL SAMPLES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for preparation of a kit for capturing amplimers of Hepatitis B Virus (HBV) genome using a sense primer consisting of the sequence 5-atactgcggaactcctagc-3 [SEQ.ID. NO.l] and a anti-sense primer consisting the sequence 5- gttcacggtggtctccatgcgacgtgc-3 [ SEQ.ID. No. 2] with oligonucleotide probe having a sequence 5&#x27;-gggcgcacctctttacgcgg-3&#x27; [ SEQ. ID. NO. 3] comprising steps of i) designing oligonucleotide probe capable of capturing amplimers of HBV genome and labeled at 5&#x27; end; ii) adding biotinylated oligonucleotide probe about lµl to about 10 µl dilution buffer about 50µI to about about 100µl; iii) immobilizing the probe on a solid medium such as microwell plate, coated with streptavidin by incubating at temperature of about 37°C for about 30 minutes to about 60 minutes and washing off excess probe with wash buffer; iv) adding hybridization buffer comprising of sodium phosphate, sodium thiocyanate and Denhardts solution and incubating for period of about 15 minutes to about 30 minutes at temperature of about 37° C; v) adding amplimers of HBV genome preferably in an amount about 25µl in denatured form and incubating for period of about 30 minutes to about 60 minutes at temperature of about 42° C and washing off excess of unhybridized product by washing preferably about 5 times with wash buffer; vi) adding an enzyme conjugate preferably anti-fluorescein and incubating for the period of about 30 minutes to about 45 minutes at temperature of about 37° C and washing with wash buffer; 311/ MUM/ vii) adding substrate capable of changing colour in presence of enzyme conjugate preferably in an amount about 100 µl in absence of light, incubating at an ambient temperature for the period of about 15 minutes to about 30 minutes and adding stop solution; viii) measuring the colour developed with a suitable instrument at an appropriate wavelength preferably at 403 nm.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM   2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
COMPLETE SPECIFICATION<br>
SECTION 10: rule 13<br>
""A METHOD FOR DETECT1NGAMPLIMERS OF HEPATITIS B VIRUS (HBV) GENOME WlTH OLIFONUCLEOTIDE  PROBE"<br>
' A METHOD   FOR   THE   PREPARATION    OF   A KIT     FOR   CAPTURING<br>
AMPLlMERS    OF    HEPATITIS, BVRUS, ( HBv)   GENOME .'<br>
GRANTED<br>
20-1-2005<br>
RELIANCE LIFE SCIENCES PVT.LTD. an Indian Company having its Registered office at Chitrakoot, 2nd Floor, Shree Ram Mills Compound, Ganpath Rao Kadam Marg, Worli, Mumbai - 400 013, Maharashtra, India.<br><br><br>
This application has been divided out of our application number 67/MUM/2002 filed on 28/01/2002.<br>
Field of the Invention<br>
The present invention relates to a method for preparation of a kit for capturing amplimers of Hepatitis B Virus (HBV) genome using a sense primer consisting of the sequence 57-atactgcggaactcctagc-3 [SEQ.ID. NO.l] and a anti-sense primer consisting the sequence 5'-gttcacggtggtctccatgcgacgtgc-3 [ SEQ.ID. No. 2] with oligonucleotide probe having a sequence 5'-gggcgcacctctttacgcgg-3' [ SEQ. ID. NO. 3].<br>
Description of the Prior Art<br>
Hepatitis B virus (HBV) is an enveloped hepatotropic DNA virus. Acute and chronic HBV infection causes significant liver diseases and it is estimated that more than 300 million individuals world wide are chronically infected with HBV. The HBV genome is unique in the world of viruses due to its compact nature, use of overlapping reading frames, and dependence on a reverse-transcriptional step, though the viron contains primarily DNA. The human hepatitis B virus is a member of the Hepadna Viridae family which includes duck hepatitis virus (DHBV), Ground Squirrel hepatitis virus (GSHV), snow goose hepatitis B virus (sgHBV), woodchuck hepatitis virus (WHV) and wooly monkey hepatitis virus. Furthermore, persistent viral infection leads to chronic active hepatitis, liver cirrhosis and the development of hepatocellular carcinoma. It has recently been appreciated that individuals who recover from HBV infection have a broad based cellular immune response to HBV structural proteins. Indeed, cytotxic lymphocyte activity (CTL) may be critical for promoting viral clearance from the liver and CTL activity has been detected many years after resolution of acute infection. The presence of CTL activity may be due to persistence<br><br><br>
of low level HBV infection in the liver that can be identified only by molecular techniques such as PCR. Thus, the concept has arisen that even if individuals serologically recover from HBV infection, the virus, in most instances is never completely irradicated from the liver. Hepatitis B is of great medical importance because Hepatocellular carcinomas(HCC), one of the most common cancers afflicting humans, is primarily caused by chronic HBV infection. In the last few decades, the correlation between HBV and the development of HCC has been well established. However, the mechanism by which HBV transforms hepatocytes remains elusive. It is noticed that before HBV can transform a cell, the virus first infects it. However, the mechanism through which HBV enters hepatocyte has not been resolved despite further understanding of the viral protein involved. Much more research is needed before it is fully understood by the scientist and the spread of this infectious agent is controlled.<br>
It is noticed that in individuals who become persistently infected with HBV is due to lack of a broad based cellular immune response for unclear reasons. In this context, there are often deletions and mutations within the envelope and core genes that may allow for persistent viral infection to occur. Alternatively, these mutant viral strains may evolve as the result of immune selection pressure by the host. In specific instances, mutations in the viral genome can lead to or contribute to the generation of latent viral infection. Furthermore, specific mutations in the precore region that includes the regulatory elements may lead to more severe disease such as fulminant hepatitis.<br><br>
Cellular and humoral immune responses to HBV antigens are believed to play an essential role in the elimination of virus by the infected host. The activity of a broad-based cellular immune response to different HBV antigens has been demonstrated to be one of the most important factors contributing to virus clearance from infected hepatocytes. One hypothesis to explain the development of persistent viral infection is that HBV-specific CTLs are unable to clear virus from the liver because of substantially decreased intrahepatic levels or qualitative changes in CTL activity. Cellular immune responses against HBV may therefore play an important role in the pathogenesis of viral hepatitis as well as determine the development, severity and outcome of chronic liver disease. The cellular immune response to HBV is strong and multispecific in acutely infected patients, and these T lymphocytes typically secrete TH1-like antiviral cytokines such as interferon-a (IFN-a) and tumor necrosis factor-a (TNF-a) upon antigen stimulation. In contrast, the cellular immune response in patients persistently infected with HBV is weak and narrowly focused.<br>
However, it is known that some chronically infected individuals spontaneously clear HBV from serum, and this phenomenon is often accompanied by increased CD4+ T lymphocyte responses and acute exacerbation of liver disease as manifested by increased serum alanine aminotransferase levels. The observation of spontaneous HBV clearance in some patients implies that the suboptimal cellular immune response may be reversible. Therefore, strategies to enhance the HBV-specific immune response or to alter the balance between certain components of the response may be able to terminate persistent infection.<br><br><br>
[0002] Early and rapid diagnosis of HBV infection is of great importance. Yet, conventional methods for detection of HBV from serum, plasma are inaccurate and/or slow. Serologic markers are commonly used as diagnostic and/or prognostic indicators of acute or chronic HBV infection. The most common marker of HBV infection is the presence of HBV surface antigen (HBs Ag). Although carriers may clear HBs Ag and develop antibody to HBs Ag, there appears to still be a risk of serious liver complications later in life. HBs Ag is generally used as a secondary marker to indicate active HBV replication associated with progressive live disease. Failure to clear HBs Ag appears to increase the risk of end stage liver disease.<br>
Various strains of HBV can either produce HBs Ag that is not detectable in serum or the strain can lose the ability to make HBs Ag even when an active infection is present. The ability to detect HBV DNA in serum has been reported to have prognostic value for the outcome of acute and chronic HBV infections. The methodology can allow the detection of HBV DNA after HBs Ag clearance or detection of HBV lacking serologic makers.<br>
[0003] The following requirements need to be fulfilled for an optimal assay for HBV diagnosis.<br>
•	High sensitivity and specificity;<br>
•	Rapid results; and<br>
•	High reproducibility.<br><br><br>
[0004] Kits for detection of HBV are commercially available. One such kit is produced by Hoffmann-La Roche and sold under the tradename Amplicor. This kit makes use of amplification by Polymerase Chain Reaction (PCR) to create amplicons specific to HBV followed by Enzyme Linked Immunosorbent Assay (ELISA) to detect the amplicons. Furthermore, the test is an in vitro amplification test for the quantification of Hepatitis B Viral DNA in human serum or plasma. The test is not intended for use in screening blood or blood products for the presence of Hepatitis B Virus.<br>
In light of the foregoing, there is a need for a more sensitive and specific detection protocol for clinical samples. The inventors of the present invention have been successful in developing a kit and a method for detecting HBV in a more sensitive, specific and rapid manner. The present invention obviates the problems associated with the conventional kits.<br>
[0005]       Several terms used in the invention are defined as follows.<br>
[0006] The term "primer" refers to a synthetic oligonucleotide sequence synthesized for annealing to a specific nucleotide sequence of interest. The primer initiates DNA synthesis to occur using thermostable DNA dependent DNA polymerase. Selecting the proper primer is one of the most important steps in designing a PCR kit. The primer set must hybridize to the target sequence with little or no hybridization to other sequences that are also present in the sample.<br><br><br>
[0007] The term "probe" refers to a synthetic oligonucleotide sequence which lies internal to the + strand of the amplified product resulting from a PCR reaction.<br>
[0008] The term "hybridization" refers to annealing of nucleotide sequences to each other under optimal conditions. Typically, a nucleotide A binds to nucleotide T and nucleotide G binds to nucleotide C.<br>
[0009] The term "biological samples" refers to the samples selected from serum, plasma, whole blood, urine, paraffin embedded tissue, and combinations thereof. When selected, plasma and whole blood samples will preferably include an anticoagulant, such as EDTA and/or ACD.<br>
Summary of the Invention<br>
[0010]       It is an object of the present invention to develop an improved method for the<br>
preparation of a kit for detecting Hepatitis B Virus Viremia (HBV) nucleic acid in Biological<br>
Samples.<br>
[0011] It is an object of the present invention to develop an improved method for rapid clinical diagnosis of HBV infection.<br>
[0012] It is still an object of the invention is to develop a method using specific primer sets and detection probe having higher sensitivity and specificity as compared to the conventional assays.<br><br>
[0013] These and other objects of the present invention are substantially achieved by a method and kit for rapid clinical diagnosis of HBV in which the amplimers are transcripts of a pre-core or envelop region gene of HBV. The amplicons are hybridized to a specific oligonucleotide probe, which allows the amplicons to be detected.<br>
Description of the Invention.<br>
[0014]       Selecting the target DNA sequence, Polymerase gene large envelop core protein (Sgene) for HBV depends upon identification of regions within the HBV genome that show maximum sequence conservation among all the serotypes of HBV.<br>
[0015]       The present invention has primers complementary to sequences flanking a segment of the Polymerase gene large envelop core protein (Sgene) to be amplified. The primers of the present invention are selected on their ability to specifically recognize the Polymerase gene large envelop core protein (Sgene) with a low mutation frequency.<br>
[0016]       The primers of the present invention are so designed to avoid hairpin loop structure formation. In addition, the selected primers have been subjected to a gene bank search to identify homologies and percent similarities to the target of interest. The blast search results are as follows :<br>
For SEQ. ID. No.1 , SEQ. ID. No. 2 and SEQ. \D. No.3 the accession No: NC- 001707 yielded 100% homology<br><br>
[0017]       A diagnostic kit for detection of HBV in biological samples according to the present invention includes two components. The first component is the amplification component, which is used to amplify the target sequence. The second component of the kit is the detection component, which is used to detect the amplicons produced by the amplification component.<br>
[0018] The amplification component amplifies the target sequence via PCR and, therefore, will include a pair of amplification oligonucleotide primers, a DNA-dependent DNA polymerase; and deoxyribonucleoside triphosphates. The primers are labeled at their 51 end. Labels are preferably selected from the group consisting of fluorescein, biotin, digoxigenin, and radioactive labels (e.g., 32P). A more preferable label is fluorescein.<br>
[0019] The first oligonucleotide primer for use in the amplification component has the following nucleic acid sequence (SEQ. ID. NO. 1):<br>
5' -atactgcggaactcctagc-3' SEQ. ID. 1 contains nineteen (19) continuous bases selected from the pre-core or envelop region sense strand. The Nucleotide Sequence position of SEQ. ID. 1 is 1142-1160. SEQ. ID. NO. 1 is preferably present in the amplification component in an amount of about 1 µL to about 10 µL in a concentration of 10 -100 pM.<br>
[0020] The second oligonucleotide primer for use in the amplification component has the following nucleic acid sequence (SEQ. ID. NO. 2):<br><br>
5 -gttcacggtggtctccatgcgacgtgc-3<br>
SEQ. ID. 2 contains twenty seven (27) continuous bases selected from the pre-core or envelop region antisense strand. The Nucleotide Sequence position of SEQ. ID. NO. 2 is 1499-1473. SEQ. ID. NO. 2 is preferably present in the amplification component in an amount of about 1 µl to about 10 µl in a concentration of 10-100 pM.<br>
[0021] The annealing temperature of primer pair is generally calculated as 5° C lower than the estimated melting temperature. The annealing temperature for primers that are less than 20 bases is calculated using the following formula: [4(G+C) + 2 (A+T)]-.5°C. Ideally the annealing temperature of each primer should match and be within the 55° C and 75° C range. If the annealing temperature difference between the two primers is high, the lower annealing temperature can be increased adding to the length of that primer at either the 31 end (this can also keep the size of the amplified locus constant) or the 51 end. The annealing temperature for the primers of the present invention is about 68° C.<br>
[0022] The thermostable DNA-dependent DNA polymerase may be any suitable polymerase. Preferably, the polymerase is derived from Thermus aquaticus (Taq) bacteria. The polymerase is preferably present in the amplification component in an amount of about 1 Unit to about 2.5 Units.<br>
The deoxyribonucleoside triphosphates (dNTPs) useful in the present invention include: dATP, dCTP, 5MedCTP, dGTP, dITP, TTP, and dUTP. Preferably, the dNTPs are selected from dATP, dCTP, dGTP, dTTP, and combinations thereof. Preferably, each<br><br>
dNTP is present in the amplification component in an amount of about 100 µM to about 200 µM.<br>
[0023] The amplification component may contain any other suitable additional ingredient and/or component, such as an amplification buffer. For example, suitable 10x amplification buffers for use in the present invention include 100 mM Tris HCI (pH 8.3), 500 mM KCI, and MgCI2.<br>
[0024]       A preferred amplification component includes:<br>
(1)	an amplification buffer having 10 mM Tris HCI ( pH 8.3) and 500mM KCI;<br>
(2)	about 100 to about 200 µl each of dATP, dCTP, dTTP and dGTP;<br>
(3)	sterile distilled water(nuclease free);<br>
(4)	about 1 unit to about 2.5 units of thermostable DNA-dependent DNA polymerase;<br>
(5)	about 10 to about 100 pM of the first oligonucleotide primer (SEQ. ID NO. 1);<br>
(6)	about 10 to about 100 pM of the second oligonucleotide primer (SEQ. ID NO. 2);<br>
(7)	about 1.5 to about 2.5 mM MgCb; and<br>
(8)	the template to be amplified.<br>
[0025]       Preferably, the volume of the amplification buffer is about 25 to about 50 µl and the volume of the extracted sample is about 25 to about 50 µl. The final volume of the amplification component is about 50 to about 100 µL.<br><br>
[0026] The detection component detects the amplified target sequence via ELISA and, therefore, will include a oligonucleotide probe immobilized upon a solid medium, a conjugate that is adapted to bind to a label present on the amplicons, and a complex that changes color in the presence of the conjugate.<br>
[0027]       The oligonucleotide probe has the following a nucleic acid sequence (SEQ. ID. NO. 3):<br>
5 -gggcgcacctctctttacgcgg-3<br>
The Nucleotide Sequence position of SEQ.ID.NO.3 is 1396-1417 and contains twenty two (22) continuous bases. SEQ.ID.NO.3 is a region internal to the amplimers created using the primers of SEQ.ID.NO.1 and SEQ.ID.NO.2<br>
SEQ. ID. NO. 3 is specifically designed for capturing the amplification product. The oligonucleotide probe is preferably immobilized on a solid medium, such as a microwell plate. For example, the oligonucleotide probe is labeled at their 5'end with biotin, which is substantially irreversibly bound to streptavidin coating the microwell plate.<br>
The oligonucleotide probe is preferably present in the detection component in an amount of about 10 µL to about 100 µL in a concentration of 10-100 pM.<br>
[0028] Since the label on the amplicon is preferably fluorescein, the conjugate is preferably an anti-fluorescein/ horseradish peroxidase (HRP) conjugate present in an<br><br><br>
amount of about 1 Unit- 4 Units. However, any suitable conjugate may be used, depending only upon the label present on the amplicon.<br>
[0029] The enzyme on the conjugate may be any enzyme, depending only upon the selected substrate. For example, the preferred enzyme for the present invention is horseradish peroxidase (HRP). However, other enzymes such as and alkaline phosphatase may be used.<br>
The substrate changes color in the presence of the enzyme conjugate to visibly show the presence of the amplimer bound to the oligonucleotide probe. Thus, a change in the color of the detection solution positively indicates the presence of the amplimer and, by extension, the presence of HBV in the original biological sample. The selection of the substrate is dependent upon the selected enzyme on the conjugate. A preferred enzyme and substrate combination for use in the present invention is peroxidase and a mixture of hydrogen peroxide (H202) and 3,3',5,5'-Tetra methybenzidine hydrochloride(TMB), present in an amount of about 100 µl, which is oxidized by H2O2 in the presence of peroxidase and, thus, the detection solution changes from colorless to blue. Other suitable enzyme and substrate combinations are as follows:<br>
Alkaline Phosphatase and 5-Bromo-4Chloro-3lndolyl Phosphate (BCIP)<br>
Alkaline Phosphatase and Fast Red RC<br>
Alkaline Phosphatase and Naphthol AS-TR Phosphate<br>
Alkaline Phosphatase and Nitro BlueTetrazolium (NBT)<br>
Alkaline Phosphatase and p-Nitrophenyl Phosphate (pNPP)<br><br><br>
Peroxidase and 3-Amino-9-Ethylcarbazole (AEC)<br>
Peroxidase and 5-Aminosalicyclic acid (5AS)<br>
Peroxidase and 2,2'-Azino-bis(3-Ethylbenzthiazoline-6-Sulfonic acid)<br>
Peroxidase and 4-Chloro-1-Naphthol (4CIN)<br>
Peroxidase and 3-3'DiaminobenzidineTetrahydro-chloride (DAB)<br>
Peroxidase and o-Dianisidine<br>
Peroxidase and o-Phenylenediamine Freebase (OPD)<br>
[0030] A method for rapid clinical diagnosis HBV according to the present invention uses the first primer (SEQ. ID. NO. 1) and the second primer (SEQ. ID. NO. 2) in an amplification step, and the oligonucleotide probe (SEQ. ID. NO. 3) in a detection step.<br>
[0031] In accordance with a second aspect of this invention, a method according to the present invention includes the steps of sample extraction, amplification (preferably by PCR), and detection by enzyme immunoassay (preferably ELISA).<br>
[0032] HBV nucleic acid is extracted from a biological sample, preferably using chaotropic agents such as urea, diethylamine, guanidium hydrochloride, potassium iodide, sodium dodecyl sulphate (SDS), Formamide and combinations thereof. Any suitable and/or known technique for extraction of nucleic acid may be used.<br>
[0033] The extracted specimen is then added to an amplification component. As discussed above, the amplification component contains the primers of the present invention<br><br><br>
(SEQ. ID. Nos. 1 and 2), having a label at their 5 ends, deoxyribonucleoside triphosphates, and a thermostable DNA dependent DNA polymerase.<br>
[0034] Amplification is accomplished by repeated cycles of: DNA denaturation, primer annealing, and extension of the primed DNA sequence by the DNA polymerase in the presence of added purine and pyrimidine bases. In general, each cycle will double the amount of the target DNA sequence. The amplification cycle is repeated until a detectable amount of the DNA sequence has been created. Further details of the PCR method are provided in U.S. Patent Nos. 4,683,195; 4,683,202; 4,965,188; and 5,075,216, which are incorporated herein by reference in their entirety.<br>
[0035] The amplimers are preferably detected using ELISA. Denatured and labeled amplimers are added to a microwell containing the immobilized oligonucleotide probe of the present invention (SEQ. ID. NO. 3) and a hybridization solution, thereby immobilizing the amplimers by hybridization with the oligonucleotide probe. An anti-fluorescein conjugate with a selected enzyme is added to the microwell after excess unbound amplimers are washed away. Finally, a substrate is added to the microwell, which changes color in the presence of the anti- fluorescein conjugate. Thus, the detection solution changes color if any amplimers are present in the microwell, which denotes a positive result for the presence of HBV in the original biological sample.<br>
[0036] This method is rapid, automatable (ELISA type solid phase formats do not require gel electrophoresis) and applicable to large scale screening programs.<br><br>
[0037] The following example illustrates the process according to the invention without limitation.<br>
Example 1<br>
[0038] An experiment was conducted to demonstrate the method of present invention. The initial sample taken was plasma EDTA that was pre-determined as positive for the presence of HBV mediated antibodies (IgG, IgM) and Hepatitis B Surface Antigen (HbSAg). The sample (0.2 ml) was extracted using the a DNA isolation system sold by Qiagen Inc., Venlo, The Netherlands, under the tradename QIAamp®.<br>
[0039] The amplification reaction was set up using 25 µL of the above extracted nucleic acid and 25 µL of 10x amplification buffer including 100 mM Tris HCI (pH 8.3), 500 mM KCI, and MgCb was added to.make a final concentration of about 1.5 mM to about 2.5 mM. The primers (SEQ. ID. NOs, 1 and 2) were labeled with fluorescein and provided in a concentration of about 10 pM. Each dNTP (dATP, dCTP, dGTP, and dTTP) was provided in a concentration of 100- 200 µl. Taq polymerase was provided in an amount of about 1-2-5 units.<br>
This reaction mixture was heated to 94°C for 10 min followed by 30 cycles. Each cycle comprises of 94°C for 30 sec, 62°C for 45 sec, and 72°C for 30 sec. Following the 30 cycles, the final incubation was performed for 10 min at 72°C. The resulting amplimers were denatured using a solution of 0.4 M NaOH.<br><br>
[0040] The detection of the fluorescein labeled amplimers was done in the following manner:<br>
[0041] (1) 50-100 µL of dilution buffer was pipetted into a microwell plate coated with streptavidin, to which was added 1-10 µL of the biotinylated oligonucleotide probe (SEQ. ID. NO. 3). The solution was incubated at 37°C for 30-60 min. Thereafter, the microwell plate was washed with PBS-T wash buffer, and 100 µL of the hybridization buffer was added and incubated for 15-30 min at 37°C.The hybridization buffer included sodium phosphate, sodium thiocyanade and Denhardts solution which included Polyvinylpyrollidine (PVP), Ficoll, Bovine Serum Albumin (BSA)]<br>
25 µL of denatured amplification product was added to the microwell plate and incubated for 30-60 min at 42°C. Following incubation, the microwell plate was washed 5 times with PBS-T wash buffer. 100 uL of diluted anti-fluorescein-HRP conjugate was added to the washed plate and incubated for 30-45 min at 37°C. Following incubation, the microwell plate was washed 5 times with PBS. 100 µL of TMB was added to the microwell plate and incubated substantially without light at room temperature for 15-30 minutes.<br>
A color change from colorless to blue is observed after incubating for 15 min. 50 (µL of a stop solution containing 0.1 N H2SO4 was added and the change in color from blue to yellow was noticed and the plate was read at 403 nm in a colorimetric plate reader, and read at 403 nm<br><br><br>
[0042] In view of the foregoing descriptions and example, it will become apparent to those of ordinary skill in the art that equivalent modifications thereof may be made without departing from the spirit and scope of this invention. Various features are set forth in the following claims.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWNhbmNlbGxlZCBwYWdlcygyMC0xLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">311-mum-2003-cancelled pages(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWNsYWltcyhncmFudGVkKS0oMjAtMS0yMDA1KS5kb2M=" target="_blank" style="word-wrap:break-word;">311-mum-2003-claims(granted)-(20-1-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWNsYWltcyhncmFudGVkKS0oMjAtMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">311-mum-2003-claims(granted)-(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWNvcnJlc3BvbmRlbmNlKDIwLTUtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">311-mum-2003-correspondence(20-5-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWNvcnJlc3BvbmRlbmNlKGlwbyktKDE4LTctMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">311-mum-2003-correspondence(ipo)-(18-7-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMSgxOC0zLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 1(18-3-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMSgyMC0xLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 1(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMTMoMi00LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 13(2-4-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMTMoMjAtMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 13(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMTkoMTktMTEtMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 19(19-11-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMihncmFudGVkKS0oMjAtMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 2(granted)-(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0gMygyMC0xLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">311-mum-2003-form 3(20-1-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLWZvcm0tMi0oZ3JhbnRlZCktKDIwLTEtMjAwNSkuZG9j" target="_blank" style="word-wrap:break-word;">311-mum-2003-form-2-(granted)-(20-1-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzExLW11bS0yMDAzLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM4KDIwLTEtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">311-mum-2003-petition under rule 138(20-1-2005).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="205450-a-system-for-method-of-generating-power.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="205452-device-for-protecting-a-syringe-needle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>205451</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>311/MUM/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Apr-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>RELIANCE LIFE SCIENCES PRIVATE LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHITRAKOOT, 2ND FLOOR, SHREE RAM MILLS COMPUND, GANPATRAO KADAM MARG, WORLI, MUMBAI - 400 013,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VIJAY SHARMA</td>
											<td>FLAT NO. 197, 19TH FLOOR NAPEROL TOWERS &quot;B&quot; WING, R.A.KIDWAI MARG, WADALA, MUMBAI - 400005,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DR VENKATA RAMANA KONDIBOYINA</td>
											<td>307, SAHYADRI, NEELKANTH VALLEY, GHATKOPOAR (E)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A 61 K 35/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/205451-a-method-for-the-preparation-of-a-kit-for-detecting-hepatitis-b-virus-hbv-nucleic-acid-in-biological-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:29:51 GMT -->
</html>
