<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/205677-improved-method-for-the-recombinant-production-of-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:57:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 205677:IMPROVED METHOD FOR THE RECOMBINANT PRODUCTION OF POLYPEPTIDES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">IMPROVED METHOD FOR THE RECOMBINANT PRODUCTION OF POLYPEPTIDES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for the recombinant production of a desired polypeptide by expressing a nucleic acid encoding said polypeptide in a microbial host cell, forming in the cytoplasm of said host cell inclusion bodies containing said polypeptide,and isolating ,solubilizing and naturing said polypeptide, charaxterized in that after fermentation the host cell or the host cell content is incubated at a tempereture of 40C higher for at least 10 minutes and subsequently insoluble polypeptide is isolated from the host cell, this method providing an improved yield in inclusion bodies containing the desired polypeptide.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The invention relates to an improved method for the recombinant production of polypeptides in prokaryotes via inclusion bodies by post-incubating the host cells at an elevated temperature.<br>
In prokaryotic organisms, the protein synthesis, also referred to as translation, takes place on the ribosomes in the cytoplasm. In expressing recombinant DNA in prokaryotic host organisms, such as, e.g., E.coli, it is often desirable that the resultant recombinant gene product/protein should be precipitated in the cytoplasm in the form of insoluble "inclusion bodies". After completion of fermentation and lysis of the cells, the inclusion bodies are isolated and optionally purified and the recombinant protein contained therein is solubilized by adding denaturants such as urea or guanidinium hydrochloride and naturation of said protein is accomplished by reducing the denaturing conditions. Such methods are well-known and have long been used successfully also for the industrial manufacture of recombinant proteins (cf., e.g., Lee, S.Y., Trends Biotechnol. 14 (1996) 98-105; Panda, A.K., et al., J. Biotechnol 75 (1999) 161-172; Mattes, R., Semin. Thromb. Hemost. 27 (2001) 325-336; Clark, E.D., Curr. Opin. Biotechnol. 12 (2001) 202-207; Misawa, S., and Kumagai, L, Biopolymers 51 (1999) 297-307; and Lilie, H., Current Opinion Biotechnol 9 (1998) 497-501).<br>
The extent to which the proteins expressed in the cytoplasm are obtained as insoluble protein aggregates (inclusion bodies) or in soluble active or inactive form is determined essentially by the primary structure (amino acid sequence) of the protein. The primary structure of the protein determines the intrinsic biophysical/biochemical properties of the protein, said properties thus determining whether the protein will fold in the cytoplasmic environment of a microorganism to form a soluble biologically active or inactive protein or whether preferably high molecular weight protein aggregates will be formed. The folding equilibrium (formation of the soluble structured protein in relation to insoluble protein aggregates) can be influenced by the selection of the fermentation and expression conditions. For instance, by growing the prokaryotic cells at reduced cultivation temperatures and/or non-optimal induction conditions (limited inductor concentration) the recombinant plasmid-encoded protein biosynthesis rate can be reduced, thereby preventing or reducing an accumulation of the protein to form insoluble protein aggregates (cf., e.g., Kopetzki, E,, et al., Mol. Gen. Genet. 216 (1989) 149-155; and EP 0 300 425). In other instances, it is desired, however, to<br><br>
prepare the recombinant protein via the route of inclusion bodies. Accordingly, an objective of such methods is to obtain a high yield of recombinant protein in the inclusion bodies. In this connection, it should be kept in mind, however, that other parameters, too, are of high importance in recombinant gene expression in prokaryotes, such other parameters being, for example, to achieve a recombinant gene expression as high as possible and to prevent undesired post-translational modification.<br>
It was the object of the invention to increase the yield of insoluble polypeptides in the form of inclusion bodies after recombinant expression.<br>
Surprisingly, it was found that the yield of insoluble denatured desired recombinant polypeptide can be increased considerably by the additional step of an incubation at an unphysiologically elevated temperature following fermentation.<br>
The object of the invention therefore is a method for the recombinant production of a desired polypeptide by expressing a nucleic acid encoding said polypeptide in a microbial host cell, preferably prokaryotic host cell, forming in the cytoplasm of said host cell inclusion bodies containing said polypeptide, isolating the inclusion bodies, solubilizing and naturing the polypeptide, said method being characterized in that after fermentation the host cell or the host cell content is incubated at a temperature of 40°C or higher, preferably 45°C or higher, for at least 10 minutes, preferably for 10 to 180 minutes, and subsequent isolation of insoluble polypeptide from the host cell.<br>
The method according to the invention is performed in aqueous media and usually carried out at temperatures between 40 and 60°C, preferably at 45°C. Higher temperatures contribute only very little to the effect according to the invention and may result in irreversible denaturation of the polypeptide.<br>
Insoluble inclusion bodies are formed during recombinant expression of polypeptides in microbial host cells. Inclusion bodies are refractile aggregates of protease-resistant misfolded desired protein that occur upon over-expression of the encoding gene (Misawa, S., and Kumagai, L, Biopolymers 51 (1999) 297-307).<br><br>
Surprisingly, by the incubation of the host cells containing the desired recombinant polypeptide at such unphysiologically elevated temperatures the desired polypeptide accumulates as insoluble inclusion bodies. Therefore, the yield of desired polypeptide in the insoluble portion of the fermentation batch can be enhanced. The duration of incubation and the incubation temperature are uncritical per se. With increasing incubation time and at higher incubation temperatures, at first a larger amount of inclusion bodies is formed. A very long duration of incubation and very high incubation temperatures, however, result in the polypeptide being changed irreversibly, for instance, by irreversible denaturation by heat. The person skilled in the art can now easily find out the optimum conditions for the recombinant production of the polypeptide of interest by carrying out simple experiments. In such experiments, it has been found that a considerable increase in the yield of recombinant polypeptide -containing inclusion bodies can be achieved, for instance, by post-incubation at 45°C for one hour.<br>
Suitable prokaryotic host cells for recombinant gene expression are, for example, gram-negative or gram-positive organisms, such as, e.g., E.coli and Bacillus subtilis. Suitable E.coli strains are, for instance, E.coli K12 strains such as UT5600, HB101, XL 1, XI776 and W3110. However, other enterobacteriaceae as well as microorganisms such as Klebsiella, Salmonella or Bacillus subtilis, Pseudomonas or Streptomyces are also suitable as host cells. Also suitable as host cells are yeast strains, such as, e.g., Saccharomyces, Pichia, Hansenula, Kluyveromyces and Schizosaccharomyces.<br>
The nucleic acid coding for the polypeptide is usually inserted in an expression vector. Suitable vectors are well-known to one skilled in the art and are, for example, plasmids or phages.<br>
The fermentation of the host cells is also accomplished according to methods known to one skilled in the art. Usually, the cells are first incubated at the fermentation temperature or up to 37° C. After a predetermined number of cells has been reached (measured via the optical density of the fermentation broth / cell suspension), the expression of the recombinant polypeptide is induced and cultivation is performed until the stationary phase is reached (in the case of batch cultures). After completion of cell growth, the cells are harvested and the inclusion bodies are isolated and processed by solubilization and naturation according to<br><br>
known methods. The additional incubation step according to the invention is appropriately inserted in this procedure. This implies that, for example, after completion of fermentation, simply the temperature of the fermentation batch is increased and subsequently the cells are harvested. Likewise, the cells can, of course, also be harvested and incubated according to the invention in suspension before or after lysis. Therefore fermentation is complete if the cell suspension is harvested and removed from the fermenter, the cell growth is inhibited otherwise or if the temperature of the cell suspension is raised according to the invention to 40° C and more which inhibits also the further growth of the host cell.<br>
The fermentation temperature can be in the usual range for recombinant production of polypeptides in microbial host cells. The fermentation temperature is preferably 30°C or lower, especially if an N-terminal processing (e.g., N-terminal cleavage of methionine of the desired polypeptide) is desired. Such an N-terminal processing is performed enzymatically by endogenous enzymes of the host cell during fermentation. Particularly preferably, the range of temperature is between 18 and 28°C, and most preferably, between 22 and 26°C.<br>
The following examples, references, sequence listing and figure are provided to aid the understanding of the present invention, the true scope of which is set forth in the appended claims. It is understood that modifications can be made in the procedures set forth without departing from the spirit of the invention.<br>
Starting material<br>
The E.coli host/vector system (E.coli host strain and basic vector) employed for the expression of the genes/polypeptides according to the invention is described in US<br>
Patent No. 6,291,245.<br>
General Methods<br>
Recombinant DNA technique<br>
Standard methods were used to manipulate DNA as described in Sambrook, J., et al., Molecular cloning: A laboratory manual; Cold Spring Harbor Laboratory Press,<br><br>
Cold Spring Harbor, New York, 1989. The molecular biological reagents were used according to the manufacturer's instructions.<br>
Polypeptide determination<br>
The polypeptide concentration for the multimeric T-repeat fusion polypeptide and interferon-a-2a was estimated by determining the optical density (OD) at 280 nm, using the molar extinction coefficient calculated on the basis of the amino acid sequence [T-repeat: e = 148680 cm2/mol; IFN-a-2a: £ = 18020 cm2/mol].<br>
Example 1<br>
Synthesis of the T-repeat fusion gene<br>
LI       Gene design of the T-repeat fusion gene<br>
The artificial T-repeat fusion gene encodes a fusion polypeptide of 217 amino acids having a molecular weight of 27,242 D. The fusion polypeptide is composed of the N-terminal 13 amino acids of human interferon-oc-2a (MCDLPQTHSLGSR (SEQ ID NO:l); carrier peptide) and 5 copies of the inhibitory HIV peptide T-1357 (WQEWEQKITALLEQAQIQQEKNEYELQKLDKWASLWEWF (SEQIDNO:2), target peptide), which are connected via trypsin-cleavable peptide linkers (GR).<br>
In order to insert the T-repeat structural gene into the E.coli expression plasmid pBRori-URA3-LacI-RFN-Edel (production and description: see Example 2), a synthetic ribosomal RBSII binding site and a singular EcoRI cleavage site were inserted upstream at the 5' end and a singular Celll restriction endonuclease cleavage site upstream at the 3' end.<br>
1.2       Gene synthesis of the T-repeat fusion gene<br>
The RBSII T-repeat gene which is about 690 bp long and flanked by a singular EcoRI and Celll restriction endonuclease cleavage site was prepared from oligonucleotides by chemical synthesis. The double-stranded RBSII T-repeat gene was assembled by annealing and ligation of the oligonucleotides and subsequently cloned as an EcoRI/CeIII fragment of a length of 691 bp into an E.coli plasmid. The<br><br>
desired plasmid was designated pT-repeat. The predetermined DNA sequence of the cloned RBSII T-repeat gene was confirmed by DNA sequencing.<br>
Example 2<br>
Construction of the E.coli expression plasmids<br>
2.1       Construction of the starting plasmid pBRori-URA3-LacI-RFN-Edel<br>
The plasmid pBRori-URA3-Lacl-RFN-Edel is a vector for the expression of interferon-α-2a (IFN-oc-2a) in E.coli. It is based on the IFN-a-2b expression plasmid OripBR-URA3-EK-IFN (U.S. Patent No. 6,291,245). pBRori-URA3-LacI-RFN-Edel differs from OripBR-URA3-EK-IFN by an additionally present lad repressor gene and an IFN-ot-2a gene instead of an IFN-ct-2b gene. IFN-oc-2a and IFN-α-2b differ only by one amino acid at position 21 (Lys21Arg exchange). The lad repressor gene comes from plasmid pUHAl (Stuber, D., et al., System for high-level production in Escherichia coli and rapid application to epitope mapping, preparation of antibodies, and structure-function analysis; In: Immunological Methods IV (1990) 121-152). It was amplified by polymerase chain reaction (PCR) according to the method described by Mullis, K.B., and Faloona, F.A., In: Methods Enzymol. 155 (1987) 335-350, using the primers Nl (SEQ ID NO:3) and N2 (SEQ ID NO:4)<br>
Not I<br>
Nl:  5'-AAAAAAGCGGCCGCGACAATTCGCGCGCGAAGGCG-3'<br>
Not I<br>
N2:  5'-AAAAAAGCGGCCGCTCACTGCCCGCTTTCCAGTCGG-3'<br>
and subsequently ligated as a NotI fragment of a length of about 1210 bp into the singular NotI cleavage site of OripBR-URA3-EK-IFN.<br>
2.2       Construction of the expression vector pBRori-URA3-Lad-T-repeat<br>
The T-repeat gene was isolated as an EcoRI/Celll fragment of a length of about 690 bp from the plasmid pT-repeat (see Example 1.2) and subsequently ligated into the ca. 3.1 kbp long pBRori-URA3-LacI-RFN vector fragment digested with EcoRI and<br><br>
CeIII	 The desired plasmid pBRori-URA3-LacI-T-repeat was identified by restriction mapping and the subcloned T-repeat gene was verified again by DNA sequencing.<br>
Example 3<br>
3.1	Expression of the T-repeat gene in E.coli<br>
For the expression of the fusion gene according to the invention there was employed an E.coli host/vector system which enables an antibiotic-free plasmid selection by complementation of an E.coli auxotrophy (PyrF) (U.S. Patent No. 6,291,245).<br>
3.2	Transformation and cell production by complementation of a pyrF<br>
auxotrophy in selective medium<br>
In order to produce cells useful for expressing the T-repeat gene, an E.coli K12 strain [designated UT5600 (ApyrF)] was transformed with the expression plasmid pBRori-URA3-LacI-T-repeat described in Example 2.2. The transformed UT5600(ApyrF)/pBRori-URA3-LacI-T-repeat cells were first grown on agar plates and subsequently in a shaking culture in M9 minimal medium containing 0.5% casamino acids (Difco) up to an optical density at 550 nm (OD550) of 0.6-0.9 and subsequently induced with IPTG (l-5mmol/l final concentration). After an induction phase of 4-16 hours at 22-37°C, the cells were harvested by centrifugation, washed with 50 mmol/1 potassium phosphate buffer, pH 6.5, and stored at -20°C until further processing.<br>
Example 4<br>
Transformation and cell production for the expression of IFN-a-2a<br>
For expression of the IFN-(X-2a gene there was used an E.coli host/vector system enabling an antibiotic-free plasmid selection by complementation of an E.coli auxotrophy (PyrF) (US Patent No. 6,291,245).<br>
For the production of cells for expressing IFN-α-2a, E.coli K12 strain UT5600 (ApyrF) was transformed with the expression plasmid pBRori-URA3-LacI-RFN-<br><br>
Edel described in Example 2.1. The transformed UT5600(ApyrF)/pBRori-URA3-LacI-RFN-Edel cells were first grown on agar plates and subsequently in a shaking culture in M9 minimal medium containing 0.5% casamino acids (Difco) at 22-37°C up to an optical density at 550 nm (OD550) of 0.6-0.9 and subsequently induced with IPTG (1-5 mmol/1 final concentration). After an induction phase of 4-16 hours at 22-37°C, the cells were harvested by centrifugation and washed with 10 mmol/1 potassium phosphate buffer, pH 6.8.<br>
Example 5 Expression analysis<br>
The portion of soluble and insoluble IFN-oc-2a and T-repeat fusion polypeptide present after the completion of cell growth, fermentation and post-fermentation was analyzed in cell lysates by SDS polyacrylamide gel electrophoresis (PAGE), staining with Coomassie Brilliant Blue R dye and densitometric analysis. Separation of the cell lysates into a soluble (supernatant) and an insoluble fraction (IB's and insoluble cell components) was achieved by means of centrifugation.<br>
For this purpose, cell pellets from 3 OD550 units each (1 OD550 = 1 ml cell suspension with an OD550 of 1) of centrifiiged culture medium were resuspended in 0.25 ml 10 mmol/1 potassium phosphate buffer, pH 6.8, and the cells were lysed by ultrasonic treatment (2 pulses of 30 s at 50% intensity). The insoluble cell components were sedimented (14,000 rpm, 5 min) and the supernatant was admixed with 1/5 volumes (vol) 5xSDS sample buffer (IxSDS sample buffer: 50 mmol/1 Tris-HCl, pH 6.8, 1% SDS, 50 mmol/1 DTT, 10% glycerol, 0.001% bromophenol blue). The insoluble cell debris fraction (pellet) was resuspended in 0.3 ml 1 x SDS sample buffer and the samples were incubated for 5 min at 95°C and again centrifiiged. Subsequently, the polypeptides were separated by SDS polyacrylamide gel electrophoresis (PAGE) (Laemmli, U.K., Nature 227 (1970) 680-685), stained with Coomassie Brilliant Blue R dye and subsequently the relevant gel bands were analyzed densitometrically.<br><br>
Example 6<br>
6.1       101 high cell density fermentation of E.coli for the recombinant production of T-repeat<br>
Preculture<br>
In order to prepare the preculture, 300 ml M9plus medium (M9 medium supplemented with 0.5% casamino acids and 0.9 g/1 Trp, Pro and Leu each) was inocculated with 1 ml of a glycerol stock of E.coli UT5600ApyrF/pBRori-URA3-lacI-T-repeat in a 1000 ml Erlenmeyer flask. The culture was incubated for about 6 hours at 37°C on an excenter shaker with 150 rpm until an OD578nm of 3.0 was reached.<br>
101 Fed-batch main fermentation<br>
At the beginning of fermentation, the preculture was transferred into a 10 liter fermenter. The main culture was grown in defined M9 salt medium containing 1.4% glycerol instead of glucose, 2% casamino acids and 0.1% of the amino acids Trp, Leu and Pro each, at 25°C up to an OD578nm of 20. Subsequently, feeding of the culture with a glycerol yeast dosage (stock solution: 30% yeast extract and 33% glycerol) was started, the flow rate of which was varied between 0.8 and 3.5 ml/min depending on the development of the pH value of the culture, thereby avoiding any further addition of correction aids (H3PO4, KOH). The pH was maintained at pH 7.0, the p02 value was held at 50% by controlling the rpm.<br>
At an OD578nm of 70 1.5 mmol/1 IPTG was added and the gene/polypeptide expression was induced. The fermentation was terminated at an OD578nm of 160-180.<br><br>
6.2       101 high cell density fermentation of E.coli for the recombinant production of interferon-α-2a<br>
Preculture<br>
In order to prepare the preculture, 500 ml M9 medium supplemented with 0.5% caseamino acids and 0.9 g/1 Trp, Pro and Leu each was inocculated with about 1.5 ml of a glycerol stock of E.coli UT5600ApyrF/pBRori-URA3-lacI-RFN in a 21 Erlenmeyer flask. The culture was incubated for about 10 hours at 37°C and 130 rpm (shaker) until an OD546nm &gt; 2 was reached.<br>
Fermentation<br>
For fermentation, a complex medium on a yeast-glycerol basis was employed. The culture was carried out at 25°C and a pH value of 7.0. When an OD456nm of 10 was reached, a glycerol yeast dosage was started, which additionally contained the amino acid L-methionine. Induction with 0.5 mM IPTG took place at an OD546nm of 50. Then the fermentation temperature was increased from 25°C to 45°C for a period of two hours. As a result of the high temperature, the expression product which had so far been present almost entirely in soluble form was converted into inclusion bodies. Subsequently, the biomass was harvested and the polypeptide was isolated.<br>
Example 7<br>
Incubation of the cells after fermentation<br>
After completion of fermentation, the cells were incubated at 45°C for 1 h in the fermenter, while stirring, before being harvested by centrifugation.<br>
The cells were analyzed in regard to the expressed polypeptides (soluble portion versus insoluble portion) as described in Example 5.<br><br><br><br>
List of References<br>
Clark, E.D., Curr. Opin. Biotechnol. 12 (2001) 202-207<br>
EP 0 300 425<br>
EP-A 0 368 342<br>
Kopetzki, E., et al., Mol. Gen. Genet. 216 (1989) 149-155<br>
Laemmli, U.K., Nature 227 (1970) 680-685<br>
Lee, S.Y., Trends Biotechnol. 14 (1996) 98-105<br>
Lilie, H., Current Opinion Biotechnol. 9 (1998) 497-501<br>
Mattes, R., Semin. Thromb. Hemost. 27 (2001) 325-336<br>
Misawa, S., and Kumagai, I., Biopolymers 51 (1999) 297-307<br>
Mullis, K.B., and Faloona, F.A., Methods Enzymol. 155 (1987) 335-350<br>
Panda, A.K., et al., J. Biotechnol. 75 (1999) 161-172<br>
Sambrook, J., et al., Molecular cloning: A laboratory manual; Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, New York, 1989 Stuber, D., et al., System for high-level production in Escherichia coli and rapid<br>
application   to   epitope   mapping,   preparation   of antibodies,   and<br>
structure-function analysis; In: Immunological Methods IV  (1990)<br>
121-152 US Patent No. 6,291,245<br><br><br><br>
WE CLAIM:<br>
1.	A method for the recombinant production of a desired polypeptide by<br>
expressing a nucleic acid encoding said polypeptide in a microbial host cell, forming<br>
in the cytoplasm of said host cell inclusion bodies containing said polypeptide, and<br>
isolating, solubilizing and naturing said polypeptide, characterized in that after<br>
fermentation the host cell or the host cell content is incubated at a temperature of 40°C<br>
or higher for at least 10 minutes and subsequently insoluble polypeptide is isolated<br>
from the host cell.<br>
2.	The method as claimed in claim 1, wherein the fermentation is performed at a<br>
temperature of 30°C or lower.<br>
3.	The method as claimed in claim 1 or 2, wherein the incubation is carried out for<br>
10 to 180 minutes.<br>
4.	The method as claimed in claim 1 to 3, wherein the incubation is performed at<br>
a temperature of 40°C to 60°C.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">530-che-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">530-che-2004-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWNsYWltcyBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">530-che-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">530-che-2004-description(complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWRlc2NyaXB0aW9uKGNvbXBsZXRlKSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-description(complete) original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">530-che-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">530-che-2004-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">530-che-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTMwLWNoZS0yMDA0LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">530-che-2004-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="205676-nicotin-or-isonicotin-benzothiazole-derivatives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="205678-method-for-assessing-fowl-sperm-quality.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>205677</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>530/CHE/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Jun-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Apr-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F. HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>124 GRENZACHERSTRASSE, CH-4070 BASEL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RAINER GRUENBECK</td>
											<td>BIRKENSTRASSE 12, D-82377 PENZBERG</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ERHARD KOPETZKI</td>
											<td>KASTNERHOFSTRASSE 21 ,D-82377 PENZBERG</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FRIEDRICH POPP,</td>
											<td>FLOHBUEHLWEG7, D-82377 PENZBERG</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N11/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03 012 295.6</td>
									<td>2003-06-12</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/205677-improved-method-for-the-recombinant-production-of-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:57:12 GMT -->
</html>
