<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/206613-non-invasive-in-vivo-tissue-classification-using-near-infrared-measurements by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:12:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 206613:NON-INVASIVE IN-VIVO TISSUE CLASSIFICATION USING NEAR-INFRARED MEASUREMENTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NON-INVASIVE IN-VIVO TISSUE CLASSIFICATION USING NEAR-INFRARED MEASUREMENTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>There is disclosed a method of developing a classification model for classification of in vivo skin tissue samples comprising the steps of: providing a set of in vivo NIR spectral absorbance measurements (11) of skin tissue samples from a population of exemplary subjects; selecting (12) one or more features of interest within said spectral absorbance measurements wherein variation according to tissue type may be found; enhancing (13) said features of interest; extracting (14) at least one feature of interest relevant to classification; selecting factors (15) of said at least one extracted feature of interest related to structural and chemical properties and physiological state of said samples; defining classes (16, 17) for said tissue samples on the basis of structural and state similarity, wherein variation within a class is small compared to variation between classes; and assigning (18) class membership.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Non-Invasive In Vivo Tissue Classification Using Near -Infrared Measurements<br>
BACKGROUND OF THE INVENTION<br>
TECHNICAL FIELD<br>
The invention relates to the classification of biological tissue. More particularly the invention relates to a method of classifying tissue using non-invasive, in-vivo near-infrared measurements.<br>
DESCRIPTION OF THE PRIOR ART<br>
Within the biomedical field, examination of the structure and state of an individual's tissue may yield important information about the individual - for example, the presence or absence of disease, age, or the effect of environmental influences. Tissue biopsy has been an extremely important diagnostic procedure for decades. Additionally, tissue studies are often used to segregate individuals into classes based on the structural and chemical properties of their tissue. For example, transgenic mice have come to play an important role in biomedical research because desired phenotypic and genotypic characteristics can be readily induced by the insertion of foreign genes into their genotype to provide an animal model optimized to the study of a specific scientific problem (see E. Wolfe, R. Wanke, "Growth hormone over- production in transgenic mice: phenotypic alterations and deduced animal models," Welfare of Transgenic Animals, Springer-Verlag, Heidelberg (1996).<br>
-1-<br><br>
It may be difficult to distinguish the transgenic mice from their non-transgenic littermates, and it would be desirable to have a simple, reliable and noninvasive way to do so. The current practice is to sever a portion of each animal's tail in order to obtain enough of the animal's genetic material for use in the various chemical analytical stechnrques used to study the genome directly (see R. Wanke, E. Wolf, W. Hermanns, S. Folger, T. Buchmuller, G. Brem, The GH-Transgenic Mouse as an Experimental Model for Growth Research: Clinical and Pathological Studies, Hormone Research, vol. 37, pp. 74-87 (1992)). Such a procedure injures and traumatizes the animal. In addition, the biopsy procedure can be awkward and require the participation of several people. The chemical analytical techniques are costly and time-consuming, and obtaining the desired information can require several different laboratory procedures and take several days or even weeks. It is also impossible to obtain completely accurate information about the in-vivo structure and state of a tissue when the sample has been subjected to the insult inherent in the biopsy procedure. It would be a technological breakthrough to be able to assess the state and structure of a tissue in-vivo rapidly without relying on tissue biopsy and chemical analytical techniques.<br>
Near infrared (NIR) specfroscopy is a promising non-invasive technology that bases measurements on the absorbance of low energy NIR light that is transmitted into a subject. The light is focused onto a small area of the skin and propagated through subcutaneous tissue. The reflected or transmitted light that escapes and is detected by a spectrometer provides information about the structural and chemical properties of the tissue it has penetrated. The absorbance of light at each wavelength is a function of the structural and chemical properties of the tissue. However, appiication of NIR spectroscopy to perform accurate, noninvasive tissue typing is presently limited by the inability of current models to compensate for the complexity and dimensionality introduced by dramatic changes in the optical properties that occur in a sample the skin and living tissue of a subject - as a result of chemical, structural, and physiological variations. Tissue layers, each - containing a unique heterogeneous particulate distribution affect light absorbance through scattering and absorbance. Chemical components, such as water, protein, fat, and blood analytes, absorb light proportionally to their concentration through unique absorption profiles, or signatures.<br>
-2-<br><br>
The parent application to the current application, U.S. Patent Application Serial No. 09/359,191, An Intelligent System for Noninvasive Blood Analyte Prediction, filed July 522, 1999, by S. Malin, T Ruchti, discloses a method and apparatus for the use of NIR spectral measurements for predicting blood analyte levels that compensates for covariation of spectrally interfering species, sample heterogeneity, state variations, and structural variations through an intelligent pattern recognition system. The invention herein disclosed provides a non-invasive method of classifying tissue samples according to chemical and structural properties.<br>
SUMMARY OF THE INVENTION<br>
The invention disclosed herein provides a non-invasive, in vivo method of tissue classification according to chemicaf and structural properties that employs NIR spectral measurements. A tissue classification model is developed by taking NIR spectral absorbance measurements from an exemplary population of individuals. The spectra! measurements are assessed to identify features of interest most likely to represent variation between tissue types. Statistical and analytical techniques are used to enhance the features of interest and extract those features representing variation within a tissue. A classification routine determines the best model to define classes within the exemplary population based on variation between tissue types, such that the variation within a class is small compared to the variation between classes. A decision rule assigns class membership to individual members of the representative population based on the structural and chemical properties of each individual's tissue.<br>
The disclosed tissue classification model is applied in a non-invasive, in-vivo tissue classification procedure using NIR spectral measurements to classify individual tissue samples. The classification model defines classes and provides a set of exemplary data that enable the segregation of test subjects into any of the classes previously defined by the classification model.<br>
-3-<br><br>
A preferred embodiment of the invention is disclosed in which samples of transgenic mice are distinguished from non-transgenic samples based on variation in fat composition of the subcutaneous tissue. The disclosed method correctly classified the samples with an accuracy of 90%.<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
Figure 1 is a flow diagram showing the steps of a general procedure for tissue sample classification according to the invention;<br>
Figure 2 provides a graph plotting the mean NIR spectra of a population of transgenic growth hormone mice and one of non-transgenic mice, where a first feature of interest in the wavelength region 1100 to 1350 and a second feature of interest in the 1800 to 1880 region are identified, according to the invention;<br>
Figure 3 provides a graph comparing the absorbance bands at the features of interest within the mean NIR spectra of Figure 2 with the absorbance bands of a sample of animal fat according to the invention;<br>
Figure 4 provides a graph showing the features of interest of Figure 2 enhanced by scatter correction according to the invention;<br>
Figure 5 provides a scatter plot showing factor scores of a second principal component from Principal Component Analysis of the second feature of interest of Figure 2 according to the invention; and<br>
Figure 6 provides a graph of factor loadings of the second principal component of Figure 5 with the animal fat spectrum of Figure 3 according to the invention.<br>
DETAILED DESCRIPTION<br>
-4-<br><br>
Various features of biological tissue can be measured using NIR spectroscopy because these features often have unique signatures in the NIR wavelength region (700 to 2500nm) as a result of their absorbance and scattering properties. Many of these features vary according to tissue type and are thus useful for classifying tissue into separate types.    Useful features that can be measured using NIR absorbance and scattering patterns include, but are not limited to, thickness of adipose tissue, tissue hydration, magnitude of protein absorbance, scattering properties of the tissue, skin thickness, temperature related  effects,  age-related  effects,  spectral characteristics related to sex, path length estimates, volume fraction of blood in tissue, spectral characteristics related to environmental influences, and hematocrit levels. The features that vary according to tissue type may be isolated from tissue sample spectra using statistical techniques and can then be used to classify the sample accordingly.<br>
DEVELOPMENT OF A TISSUE CLASSIFICATION MODEL<br>
A non-invasive, in-vivo method for the classification of tissue samples according to chemical, physiological, and structural differences is described herein. The classification model employs the use of NIR measurements to quantify chemical, structural, or physiological properties of the tissue sample. Figure 1 provides a flow diagram of a general procedure used to develop a classification model. In general, the algorithm for developing a classification model comprises the following steps:<br>
1.	Providing exemplary NIR measurements (11)<br>
2.	Spectral feature selection (12)<br>
3.	Feature enhancement (13)<br>
4.	Feature extraction (14)<br>
5.	Factor selection (15)<br>
6.	Classification calibration (16)<br>
7.	Application of a Decision Rule (17)<br>
8.	Assignment to a group (18)<br>
MEASUREMENT<br>
-5-<br><br>
NIR measurements (11) are first taken from a population of exemplary subjects. The individuals are prepared for the procedure using commonly known methods. In the case of animal subjects, the subjects may need to be anesthetized or immobilized.  It is also desirabie to prepare the sample surface to be scanned, such that spectral interference due to surface irregularities is minimized. For example, surface hair or fur may need to be removed from animal subjects.    A spectrometer detecting light in the near-IR wavelength region (700 to 2500nm) is employed to collect the NIR measurements. The NIR measurements may be expressed in a variety of units, among them reflectance units and the negative base ten logarithm of reflectance units. While the method of the invention can be employed with various commercially available NIR spectrometers such as the Nicolet Magna-IR 760 or the Perstorp Analytical NIRS 5000, reference is made to figure  19 of the parent to this application,  U.S.  Patent Application Serial  No. 09/359,191, An Intelligent System for Noninvasive Blood Analyte Prediction, filed July 22, 1999, by S Malin, T. Ruchti, for a more complete description of an instrument well-suited to the practice of the invention. The sample measurement or tissue absorbance spectrum is the vector me meRN of absorbance values pertaining to a set of N wavelengths ?eRN that span the near IR wavelength region (700 to 2500nm).<br>
FEATURE SELECTION<br>
Spectral feature selection (12) can comprise a qualitative assessment of the sample spectra to identify the spectral region, or feature of interest in which variation according to tissue type is to be found. The spectra are truncated at the boundaries of the wavelength regions where the tissue-specific variation is present to reduce the complexity and dimensionality of the data set. The data reduction achieved improves the predictive ability of the classification model by reducing the dimensionality of the dataset. Feature selection is aided by comparing the sample spectra with known spectral absorbance patterns, those of protein or fat for example. After one or more<br>
-6-<br><br>
wavelength regions containing tissue-specific information have been identified, only these selected regions are used in the classification model.<br>
FEATURE ENHANCEMENT<br>
Once the feature of interest has been identified, a method of feature enhancement (13)<br>
is applied to the spectral measurements. Although NIR reflectance measurements are<br>
related to chemical and structural features of the tissue samples; the correlation is non<br>
linear, due in large part to the dilution of the spectral measurements by light scatter<br>
(see P. Geladi, D. MacDougall, H. Martens, Linearization and Scatter Correction for<br>
Near-infrared Reflectance Spectra of Meat Applied Spectroscopy. vol. 39, pp. 491-500<br>
(1985)). Feature enhancement, employing a technique such as multiplicative scatter<br>
correction (MSC), corrects interfering spectral variation resulting from changes in the<br>
scattering properties of the target tissue volume.	In the parent to the present<br>
application, U.S. Patent Application Serial No. 09/359,191, An intelligent System for Noninvasive Blood Analyte Prediction, filed July 22, 1999, by S. Malm, T. Ruchti, a method of feature enhancement employing MSC is disclosed. The scatter for each sample is estimated by rotating the sample spectrum and fitting it to a reference<br>
spectrum m according to <br>
where   a and b are the slope and intercept and e represents the error in fit. The spectrum is then corrected through<br><br>
where x is the processed absorbance spectrum. Thus, each sample's spectrum is standardized such that all samples are normalized with respect to a reference spectrum (see also B. Wise, and N. Gallagher, PLS Toolbox 2.0, Eigenvector Research, Inc. Manson (1998). When the feature of interest is due to an absorbing species in the tissue, or if the tissue-specific parameter has distinct spectral shapes, MSC can also be applied to wavelength regions adjacent the feature of interest.<br>
-7-<br><br>
Correcting for scatter in adjacent regions causes the characteristic shapes in the uncorrected wavelength regions to be exaggerated. Thus, the targeted chemical and structural features are effectively enhanced .<br>
FEATURE EXTRACTION<br>
Following feature enhancement, a method of feature extraction is applied to extract those features that are relevant to classification. Feature extraction is accomplished by any mathematical transformation that enhances a quality or aspect of the sample measurement for interpretation. The purpose of feature extraction is to represent concisely the chemical and structural properties and physiological state of the tissue measurement site.<br>
The preferred method of feature extraction, Principal Component Analysis (PCA) (14), iis performed on the feature-enhanced data. PCA decomposes the multi-dimensional data into ordered factors that represent the underlying variation within the data set (see R. Johnson and D. Wichern Applied Multivariate Statistical Analysis, 3rd. ed.; Prentice-Hall, New Jersey, (1992)). These factors are called principal components    The first principal component represents the most variation present in the data, the second principal   component   represents  the   second   most  variation,   the   third   principal component represents the third most variation and so on.   Principal components or factors represent abstract or simple features present in the data.   Abstract features represented by the factors do not necessarily have a specific interpretation related to the physical system.   Specifically, the scores of a principal component analysis are useful features although their physical interpretation is not always known. However, simple features are derived from an a priori understanding of the sample and can be related directly to a physical phenomenon. Protein and fat, for example, have known absorbance signatures that can be used to determine their contribution to the tissue spectral absorbance. The measured contribution is used as a feature and represents the underlying variable through a single value.<br>
-8-<br><br>
An a priori understanding of the tissue sample aids in the selection of features directly-related to physical phenomena of the tissue. The features are represented directly in a vector, zeRM from the scatter-corrected measurements through<br><br>
where f. Rn?Rm is a mapping from the measurement space to the feature space. Decomposing f(•) yields specific transformations, f.(•):Rn?Rm for determining a specific feature. The dimension M1, indicates whether the  feature is a scalar or a vector and the aggregation of all features is the vector z. The features, obtained by reducing the dimensionality of the complex spectral variation, are immediately useful to the tissue classification process. Therefore, feature extraction allows the identification of features representing variation directly attributable to physical phenomena and the exclusion of all others.    Thus, a highly dimensional problem is further reduced to a few easily managed dimensions.<br>
FACTOR SELECTION<br>
Selection of the factors (15) relevant to the classification model is done by visually inspecting the factor loadings and scores from the factor analysis. As described above, a PCA on the scatter-corrected data set, represented as a data matrix, captures the variation in the data into factors or principal components.  For the factors or principal components  representing tissue-specific features, the corresponding factor scores separate into groups according to tissue type and the corresponding factor loadings are indicative of the feature responsible for the tissue-specific variation. When the factor scores are represented as scatter plots, the relevant factors or principal components are readily identified through a simple visual inspection.<br>
CLASSIFICATION CALIBRATION<br>
-9-<br><br>
A classification routine (16) with a decision rule {i.e. Fisher's linear discriminant analysis) is applied to the scores of the tissue-specific factor. Classification routines are based on a criterion function that, when maximized, finds the best model to represent the separation between populations (see R. Duda and P. Hart, Pattern Classification and Scene Analysis, John Wiley and Sons, New York (1973)). Fisher's linear discriminant analysis (LDA) is the preferred routine used for this classification model. LDA finds the line between the two groups that maximizes the between-class variation and minimizes the within class variation. The LDA criterion function is<br><br>
where w is a directional unit vector, Sb is the between-class scatter matrix, and Sw is the within-class scatter matrix. The vector w is selected such that it maximizes the between-class variation/within-class variation ratio. The samples are projected onto w, further reducing the dimensionality of the problem (see R. Duda and P Hart, Pattern Classification and Scene Analysis. John Wiley and Sons, New York (1973)). The criterion function is applied to classify the samples on the basis of the first M scores. In the preferred realization described below, M=3. One skilled in the art can appreciate other methods of classification are readily applicable.<br>
DECISION RULE<br>
A decision rule (17) is developed to determine to which class a sample belongs. The criterion the decision rule employs to determine the class membership of the sample is whether the sample's projection (scalar) onto w is greater than or less than the mean of the two population means (see R. Johnson and  D. Wichern Applied  Multivariate Statistical Analysis,  3rd.  ed.,  Prentice-Hall,  New Jersey (1992)). The scalar L  is compared with L, the class boundary. If L&gt;L. the sample is assigned to population one. (18). If not, the sample is assigned to population two (18).<br>
CLASSIFICATION OF TISSUE SAMPLES<br>
-10-<br><br>
Implementation of the disclosed classification model for classification of actual tissue samples is described in detail in the parent application to the current application, U.S. Patent Application Serial No. 09/359,191, An Intelligent System for Noninvasive Blood Analyte Prediction, filed July 22, 1999 by S.Malin, T. Ruchti. In general, the steps of a 5procedure for tissue classification are:<br>
1.	NIR measurements<br>
2.	Feature extraction<br>
3.       Pattern classification<br>
4.       Assignment of class membership<br>
A set of absorbance values pertaining to a set of wavelengths spanning the near infrared (700 to 2500nm) is obtained for a new sample.   The spectral measurements are pre-processed to attenuate light scatter, noise, and error due to instrument variation without  affecting the  signal of interest.   Feature extraction,  previously  described, comprises the use of any mathematical transformation that isolates a particular aspect or quality of the  sample that is  useful  for interpretation.  Feature  extraction  is accomplished through the application of one or more multivariate analysis methods, such as principal component analysis, partial least squares, or artificial neural networks. The extracted features are compared to data from a set of predefined classes and the similarity is measured. A decision rule applies a criterion function to assign class membership.<br>
EXPERIMENT RESULTS<br>
INTRODUCTION<br>
In a preferred embodiment of the invention, the algorithm herein disclosed has been adapted to separate a population of growth hormone transgenic mice from a population of non-transgenic control mice. The embodiment described is specific to transgenic<br>
-11-<br><br>
mice.   However, the invention is appropriate for the classification of other tissues in other species wherein accurate spectral measurements are possible.<br>
Growth hormone transgenic mice have foreign growth hormone genes inserted into their genome under the control of a promoter gene. The control mice are the non-transgenic littermates of the transgenic mice, and are thus genetically identical to the growth hormone mice except for the presence of the growth hormone gene in the transgenic individuals. The presence of the foreign growth hormone gene causes the subject animal to produce excessive amounts of growth hormone. Compared to non-transgenic mice, transgenic growth hormone mice exhibit a multitude of phenotypic effects secondary to the overproduction of growth hormone, e.g. skeletal gigantism, oversized internal organs, increased body weight, and various characteristic tissue abnormalities (see Wolf et al., see also R. Wanke, E. Wolf, W. Hermanns, S. Folger, T. Buchmuller, G. Brem; The GH-Transgenic Mouse as an Experimental Model for Growth Research: Clinical and Pathological Studies. Hormone Research, vol. 37, pp. 74-87 (1992)).<br>
METHOD<br>
Using diffuse reflectance spectroscopy, NIR measurements were taken from a population of fourteen mice, the population comprising eight growth hormone mice and six controls. Four identical scans were collected from the abdomen of each mouse, once per day for a period of three days. Prior to measurement, the mice were anesthetized using an injectable anesthetic, and shaved using animal trimmers to avoid contamination of the spectral measurements by the subjects' body fur. On each of the three days the animals were selected for scanning in random order from each of their respective populations. A spectrometer setup as previously described was employed for the measurements.<br>
Using commonly known statistical methods, a mean spectrum for each of the two subject populations was calculated from the original spectral data. Figure 2 shows the mean spectra of the growth hormone mice (21) and the control mice (22). Visual<br>
-12-<br><br>
inspection of the spectra (21 and 22) revealed a divergence in two wavelength regions. A first feature of interest (23) shows that the control mice exhibited an absorbance band in the 1100 to 1350nm-wavelength region. A second feature of interest (24) revealed an absorbance band in the 1600 to 1880nm-wavelength region of the spectrum from the control mice. Unlike the controls, the transgenic subjects did not exhibit such absorbance bands.<br>
Figure 3 shows a comparison of the mean spectra of the two mouse populations (31 and 32) with the spectrum of a sample of pure animal fat (33), within the two identified wavelength regions (features of interest). The animal fat spectrum clearly shows a first absorbance band (34) in the 110 to 1350nm-wavelength region and second (35) and third (36) absorbance bands in the 1600 to 1880nm-wave!ength region. A first absorbance band (37) and second and third absorbance bands (38, 39) in the mean spectrum of the control population mirrors the absorbance bands (34, 35, 36) of the fat spectrum. At the same time, analogous absorbance bands are absent from the mean spectrum of the transgenic GH population, suggesting that the difference in the spectra of the two mouse populations was due to a difference in the fat composition of the tissues.<br>
Multiplicative Scatter Correction (MSC), as described earlier was applied to enhance the features of interest. Figure 4 provides a graph of the scatter-corrected spectral data. It is clear from the scatter-corrected spectra that the observed absorbance bands are expressed to varying degrees within the population of samples. A large number of the samples exhibit no noticeable absorbance bands, while other exhibit pronounced absorbance bands.<br>
Principal Component Analysis was performed on the scatter-corrected data from the wavelength regions of 1100 to 1350nm and 1600 to 1850nm. Visual inspection of the resulting factor scores revealed the second principal component to be significant. As Figure 5 shows, the factor scores for the second principal component separate neatly into two groups (51, 52) corresponding to the two mouse populations. The graph of Figure 6 shows a clear correlation between the absorbance bands of the animal fat<br>
-13-<br><br>
spectrum and the principal component 2 loadings, further evidence that the separation between the two mouse populations is due to the fat composition of the tissues.<br>
Fisher's linear discriminant analysis was applied to the first three principal components scores to define an appropriate model for the classes such that within class variation was minimized and between-class variation is maximized.<br>
"Leave-one-out" cross-validation, a calibration routine particularly well suited for application with small data sets, was used in the validation of the classification calibration. For this analysis, an iterative process was employed in which one sample was left out of the data set, the remaining samples were used to develop a calibration, and the resulting calibration was used to predict the sample left out. The process is repeated until all the samples have been left out and predicted.<br>
RESULTS The results of the calibration routine are presented in Table 1 below.<br>
Table 1: Cross Validation Classification Results<br><br>
Misclassifie     Correc    Tota Type                      d                     t              i          % Correct Growth<br>
Hormone               0                     99           99       100% Control                    17                    52           72       72%<br>
Total                        17                    154         171      90%<br>
A total of 171 spectral samples were examined, 99 spectral samples from GH transgenic individuals, and 72 samples from the controls. Of the samples, 99 or 100% of the transgenic samples were correctly classified; 54 or 72% of the control samples were correctly classified. Thus the algorithm correctly classified each of this population of samples with an accuracy of 90%. The two mouse populations were classified according to the chemical and structural differences in their tissue, without resorting to<br>
-14-<br><br>
invasive biopsy procedures. Data collection was readily performed by one person and an online classification could be developed that would require a few minutes as opposed to the days or weeks that may be required with conventional methods.<br>
DISCUSSION<br>
Although the current embodiment is directed to the classification of mice based on structural and chemical differences in their tissues, those skilled in the art will readily appreciate that the use of NIR measurements to indicate chemical, structural, and iphysiological variation in tissue has application in other fields of endeavor, for example non-invasive blood analyte prediction, wound healing research and dermatological disease therapy.<br>
The class definitions described here have been defined for a specific population of isubjects, and cannot be generalized to all subjects. A classification model suitable for all subjects includes a sufficient number of samples to model the total range of variation in the population for a specific tissue parameter. While the present embodiment employs a specific number of features, classes, decision rules, and classification models, the invention may use an arbitrary number of each in the configuration shown to classify tissue samples. While the experimental results demonstrate the validity and benefit of the invention, accuracy of the results is directly dependent on the accuracy of the spectral measurements. Further improvement to results accuracy is achieved through improvements in the noise level and the resolution of the spectrometer<br>
Although the invention is described herein with reference to certain embodiments, one skilled in the art will readily appreciate that other applications may be substituted for those set forth herein without departing from the spirit and scope of the present invention. Accordingly, the invention should only be limited by the Claims included below.<br>
-15-<br><br>
WE CLAIM:<br>
1.	A method of developing a classification model for classification of in vivo<br>
skin tissue samples comprising the steps of:<br>
providing a set of in vivo NIR spectral absorbance measurements of skin tissue samples from a population of exemplary subjects;<br>
selecting one or more features of interest within said spectral absorbance measurements wherein variation according to tissue type may be found;<br>
enhancing said features of interest,<br>
extracting at least one feature of interest relevant to classification;<br>
selecting factors of said at least one extracted feature of interest related to structural and chemical properties and physiological state of said samples;<br>
defining classes for said tissue samples on the basis of structural and state similarity, wherein variation within a class is small compared to variation between classes; and<br>
assigning class membership.<br>
2.	The method  as claimed  in  claim   1,  wherein each  of said spectral<br>
absorbance measurements is represented as a vector meRN of absorbance<br>
values pertaining to a set of N wavelengths ?eRN spanning the wavelength<br>
region of 700 to 2500nm.<br>
3 The method as claimed in claim 1, wherein said feature selection involves the steps of<br>
visually inspecting said spectral absorbance measurements;<br>
comparing said spectral absorbance measurements with known spectral absorbance patterns;<br>
selecting said one or more features of interest based on observed similarities between said spectral absorbance measurements and said known spectral absorbance patterns; and<br>
-16-<br><br>
truncating said spectral absorbance measurements at the boundaries of said selected one or more features of interest.<br>
4. The method as claimed in claim 1, wherein said step of enhancing features comprises correcting said spectral absorbance measurements for scatter.<br>
5 The method as claimed in claim 4, wherein correcting said spectral absorbance measurements for scatter comprises:<br>
correcting said spectral absorbance measurements for scatter using multiplicative scatter correction.<br>
6.	The method as claimed in claim 5, wherein scatter for each of said sample<br>
spectra is estimating by rotating said sample spectra to a reference spectrum m according to.<br><br><br>
where a and b are the slope and intercept and e is the error in fit, and wherein each of said spectra are corrected through. <br>
where x is the corrected absorbance spectrum.<br>
7.	The  method  as  claimed  in  claim   1,  wherein  said  feature extraction<br>
involves the steps of<br>
applying a mathematical transformation, wherein said feature-enhanced sample spectra are decomposed to distinct factors that represent underlying variation within the data set;<br>
employing factor-based methods to determine which of said factors -are attributable to a known spectral absorbance pattern; and<br>
-17-<br><br>
including the measured contribution of said known spectral absorbance pattern to the sample spectral absorbance as features.<br><br>
8.	The method as claimed in claim 7, wherein said extracted features are<br>
represented   from   said   scatter-corrected   measurements   in   a   vector zeRM<br>
through:<br>
f:Rn?Rm being a mapping from a measurement space to a feature space, wherein decomposing f(•) yields specific transformations if,(•): Rn?Rm whereby specific factors are determined; wherein the dimension M, indicates whether the 1th factor is any of a scalar and a vector, and wherein the aggregation of all of said factors is the vector z, and wherein x is a corrected absorbance spectrum.<br>
9.	The method as claimed in claim 8, which involves the steps of:<br>
including factors represented as vectors in the data set; and excluding<br>
those factors represented as scalars from the data set.<br>
10 The method as claimed in claim 1, wherein said factor selection involves the steps of:<br>
representing variation within said spectral absorbance measurements as factor loadings; and<br>
representing the weight of a particular sample on said variation within said spectral absorbance measurements as factor scores corresponding to said factor loadings.<br>
11.	The method as claimed in claim 10, wherein a clustering of said factor scores represents a variation according to tissue type and said factor loadings represent a feature responsible for tissue-specific variation.<br>
12.	The method as claimed in claim 1, wherein said step of defining classes<br>
-18-<br><br>
comprises defining classes wherein within-class variation is minimized and between-class variation is maximized.<br><br>
13.	The method as claimed in claim 12, wherein defining classes is based on linear discriminant analysis.<br>
14.	The method as claimed in claim 13, wherein defining classes based on linear discriminant analysis comprises defining classes according to a criterion function:<br>
where w is a directional unit vector, Sb is the between class scatter matrix, Sw is the within-class scatter matrix, wherein a vector w is selected such that the between-class variation/within-class variation ratio is maximized, wherein said vector w represents the separation between classes.<br>
15.	The method as claimed in claim 14, which involves the step of<br>
applying said criterion function on the basis of the first M scores, where M is an arbitrary number, wherein each of said samples are projected onto said vector w\ wherein said projections onto said vector w are scalars.<br>
16.	The method as claimed in claim 15, wherein said decision rule represents<br>
said scalars of said samples as L, and 1 represents a boundary between said<br>
classes; and<br>
wherein assignment to a first class is based on the condition L&gt;L and wherein assignment to a second class is based on the condition L<l.></l.>
17.	A method for classifying tissue comprising the steps of.<br>
providing a set of in vivo NIR spectral absorbance measurements of skin tissue from a test subject;<br>
-19-<br><br>
extracting features corresponding to tissue-specific variation; comparing said extracted features to data from a set of predefined classes and the similarity measured according to a classification model; and assigning class membership through application of a decision rule.<br>
18.	The method as claimed in claim 17, wherein said feature extraction step<br>
comprises any mathematical transformation that enhances a quality or aspect of<br>
sample   measurement  for   interpretation   to   represent   concisely   structural<br>
properties and physiological state of a tissue measurement site, wherein a<br>
resulting set of features is used to classify a subject.<br>
19,	The method as claimed in claim 17, wherein said classification model<br>
comprises means for determining a set of similarity measures with predefined<br>
classes.<br>
20.	The method as claimed in claim 17 wherein said decision rule comprises<br>
means for assigning class membership on the basis of a set of measures<br>
calculated by a decision engine.<br>
21,	The method as claimed in claim 19, which involves the step of:<br>
providing a classification system that assumes that said classes are<br>
mutually exclusive and that forces each measurement to be assigned to a single class.<br>
22      The method as claimed in claim 21, wherein said features are represented in a vector zeRM that is determined through:<br><br>
where f: RN?RM is a mapping from a measurement space to a feature space,<br>
-20-<br><br>
wherein decomposing f(•) yields specific transformations, f:RN?RM, for determining a specific feature, wherein the dimension M indicates whether an 1th feature is a scalar or vector and an aggregation of all features is the vector z, and wherein a feature exhibits a certain structure indicative of an underlying physical phenomenon when said feature is represented as a vector or a pattern.<br>
23 The method as claimed in claim 17, wherein said step of comparing extracted features comprises the steps of:<br>
measuring the similarity of a feature to predefined classes; and<br>
assigning class membership.<br>
24. The method as claimed in claim 23, wherein said measuring step uses mutually exclusive classes and assigns each measurement to one class.<br>
25      The method as claimed in claim 24, which involves the step of:<br>
using measurements and class assignments to determine a mapping from features to class assignments.<br>
26, A method of developing a classification model for distinguishing transgenic mice from non-transgenic mice based on fat composition in the tissue comprising the steps of:<br>
providing a set of spectral absorbance measurements from an exemplary population of subject animals;<br>
selecting one or more features of interest within said spectral absorbance measurements wherein variation according to tissue type may be found;<br>
enhancing said one or more features of interest;<br>
extracting at one or more features of interest relevant to classification,<br>
selecting factors of said extracted one or more features of interest related to structural and chemical properties of said samples;<br>
-21-<br><br>
defining classes for said tissue samples on the basis of structural and state similarity, wherein variation within a class is small compared to variation between classes; and<br>
assigning class membership.<br>
27.	The method as claimed in claim 26, wherein said transgenic mice are foreign growth hormone transgenic mice.<br>
28.	The method as claimed in claim 26, wherein each of said measured absorbance spectra are represented as a vector meRN  of absorbance values pertaining to a set of N wavelengths ?eRN spanning the wavelength region of 700 to 2500nm.<br>
29 The method as claimed in claim 26, wherein said feature extraction involves the steps of:<br>
visually inspecting said spectral absorbance measurements;<br>
comparing said spectral absorbance measurements with the spectral absorbance pattern of animal fat,<br>
selecting said one or more features of interest based on observed similarities between said spectra! absorbance measurements and said spectral absorbance pattern of animal fat; and<br>
truncating said spectral absorbance measurements at the boundaries of said selected one or more features of interest<br>
30 The method as claimed in claim 26, wherein said step of enhancing features comprises correcting said spectral absorbance measurements for light scatter.<br>
31. The method as claimed in claim 30, wherein correcting said spectra] absorbance measurements for scatter comprises<br>
-22-<br><br>
correcting said spectra! absorbance measurements for scatter using multiplicative scatter correction.<br>
32.	The method as claimed in claim 31, wherein scatter for each of said<br>
sample spectra is estimating by rotating said sample spectra to a reference<br>
spectrum m according to<br><br>
where a and b are the slope and intercept and e is the error in fit; and wherein each of said spectra are corrected through:<br><br><br>
where x is the corrected absorbance spectrum.<br>
33.	The method as claimed in claim  17, wherein said feature extraction<br>
involves the steps of:<br>
applying a mathematical transformation, wherein said feature-enhanced sample spectra are decomposed to distinct factors that represent underlying variation within the data set;<br>
employing factor-based methods to determine which of said factors are attributable to said spectral absorbance pattern of animal fat; and<br>
including the measured contribution of said animal fat spectral absorbance pattern to the sample spectral absorbance as features.<br><br>
34.	The method as claimed in claim 33, wherein said extracted features are<br>
represented  from  said   scatter-corrected   measurements   in   a  vector  zeRM<br>
through:<br>
f: Rn?Rm being a mapping from a measurement space to a feature space; wherein decomposing f(•) yields specific transformations 'f,(•): Rn?Rm whereby<br>
-23-<br><br>
specific factors are determined; wherein the dimension M, indicates whether the 1th factor is any of a scalar and a vector; and wherein the aggregation of all of said factors is the vector z, and wherein x is a corrected absorbance spectrum.<br>
35.	The method as claimed in claim 34, wherein loading of said factors represent variation according to fat composition of tissue, and scores of said factors identify different subject classes based on fat composition in the tissue.<br>
36.	The method as claimed in claim 26, wherein said class definition step defines classes for a subject population based on variation according to fat composition in the tissue, such that within-class variation is minimized and between-class variation is maximized.<br><br>
37	The method as claimed in claim 26, wherein said class assignment step employs a decision rule to assign membership to individuals from said sample population.<br>
38	A method of distinguishing transgenic mice from non-transgenic mice based on variation according to fat composition in the tissue comprising the steps of:<br>
providing a new set of spectral absorbance measurements from a subject; extracting features corresponding to tissue-specific variation; comparing said extracted features with a set of exemplary measurements according to a classification model; and<br>
assigning class membership through application of a decision rule.<br>
39.     The method as claimed in claim 38, wherein said transgenic mice are foreign growth hormone transgenic mice<br>
-24-<br><br>
40.	The method as claimed in claim 38, wherein said feature extraction step comprises any mathematical transformation that enhances a quality or aspect of sample   measurement   for   interpretation   to   represent   concisely   structural properties and physiological state of a tissue measurement site, wherein a resulting set of features is used to classify said subject according to fat composition in the tissue.<br>
41.	The method as claimed in claim 38, wherein said classification model comprises means for determining a set of similarity measures with predefined classes.<br>
42.	The method as claimed in claim 41, wherein said classes are defined on the basis of fat composition in the tissue.<br>
43.	The method as claimed in claim 38, wherein said decision rule comprises means for assigning class membership on the basis of a set of measures calculated by a decision engine.<br>
44.	The method as claimed in claim 41, which involves the step of:<br>
providing a classification system that assumes that said classes are<br>
mutually exclusive and that forces each measurement to be assigned to a single class.<br><br>
45.	The method as claimed in claim 44, wherein said features are represented<br>
in a vectorized that is determined through:<br>
where f:RN ?RM is a mapping from a measurement space to a feature space, wherein decomposing f(•)' yields specific transformations,<br>
f: RN?RM for determining a specific feature, wherein the dimension M, indicates<br>
-25-<br><br>
whether an 1th feature is a scalar or vector and an aggregation of all features is the vector z, and wherein a feature exhibits a certain structure indicative of variation according to fat composition in the tissue when said feature is represented as a vector or a pattern.<br>
46.	The method as claimed in claim 38, wherein said pattern classification<br>
step further comprises the steps of:<br>
measuring the similarity of a feature to predefined classes; and assigning class membership<br>
47.	The method as claimed in claim 46, wherein said predefined classes are any of transgenic mice and non-transgenic mice.<br>
48.	The method as claimed in claim 47, wherein said classes are mutually exclusive.<br>
49.	The method as claimed in claim 48, which involves the step of.<br>
using measurements and class assignments to determine a mapping from features to class assignments.<br>
-26-<br>
There is disclosed a method of developing a classification model for classification of in vivo skin tissue samples comprising the steps of:<br>
providing a set of in vivo NIR spectral absorbance measurements (11) of skin tissue samples from a population of exemplary subjects;<br>
selecting (12) one or more features of interest within said spectral absorbance measurements wherein variation according to tissue type may be found;<br>
enhancing (13) said features of interest;<br>
extracting (14) at least one feature of interest relevant to classification;<br>
selecting factors (15) of said at least one extracted feature of interest related to structural and chemical properties and physiological state of said samples;<br>
defining classes (16, 17) for said tissue samples on the basis of structural and state similarity, wherein variation within a class is small compared to variation between classes; and<br>
assigning (18) class membership.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="206612-pressure-relief-apparatus.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="206614-a-gas-and-steam-turbine-unit.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>206613</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00940/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-May-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>03-May-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Jul-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SENSYS MEDICAL, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>7470 WEST CHANDLER BLVD., CHANDLER, AZ 85226, UNITED STATES OF AMERICA, A CORPROATION ORGANIZED AND EXISTING UNDER THE LAWS OF THE STATE OF DELAWARE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MALIN, STEPHEN F.</td>
											<td>16228 S., 4TH STREET PHOENIX, AZ 85048</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RUCHTI, TIMOTHY, L.</td>
											<td>1501, WEST SEA HAZE DRIVE TEMPE, AZ 85282, UNITED STATES OF AMERICA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RENNERT, JESSICA</td>
											<td>8253 E.MC. DONALD SCOTTSDALE, AZ 85250 UNITED STATES OF AMERICA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 23/25</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/32976</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-12-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/487, 547</td>
									<td>2000-01-19</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/206613-non-invasive-in-vivo-tissue-classification-using-near-infrared-measurements by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:12:14 GMT -->
</html>
