<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/207712-a-method-for-production-of-one-or-more-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:33:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 207712:A METHOD FOR PRODUCTION OF ONE OR MORE PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR PRODUCTION OF ONE OR MORE PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>(EN) The invention relates to nucleic acid molecules comprising a heat-inducible promoter and to expression vectors and host cells containing at least one inventive nucleic acid molecule. The invention further relates to kits and methods for producing one or more proteins using the inventive nucleic acid molecules and to various uses of the same. The aim of the invention is to provide an as selective as possible heat-inducible promoter, especially a promoter that is active in yeasts and that is suitable for the protein expression at high temperatures. To this end, the inventive nucleic acid molecule comprises a heat-inducible promoter and is selected from the following nucleic acids: (a) a nucleic acid the sequence of which comprises the promoter sequence of a gene of $i(Hansenula polymorpha) coding for a protein with trehalose-6-phosphate-synthase activity; (b) a nucleic acid with the sequence indicated in SEQ ID NO: 1; (c) a nucleic acid with a sequence that is at least 40 % identical with any of the sequences indicated in (a) or (b) over a stretch of 300 bp; (d) a nucleic acid that hybridizes to an antisense strand of any of the sequences indicated in (a), (b) or (c); (e) a derivative of any of the nucleic acid acids indicated in (a), (b) or (c) obtained by substitution, addition and/or deletion of one or more nucleotides; (f) a fragment of any of the nucleic acids indicated in (a) to (e) that maintains the function of the heat-inducible promoter; (g) a combination of several of the nucleic acids indicated in (a) to (f), with the sequences of the nucleic acids being the same or different; or a nucleic acid molecule with a sequence that is complementary to the sequence of any of the nucleic acids indicated in (a) to.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Heat-inducible promoter<br>
The present invention relates to nucleic acid molecules comprising a heat-inducible promoter, as well as to expression vectors and host cells containing at least one nucleic acid molecule according to the invention. The present invention further relates to kits and methods for the production of one or more proteins using the nucleic acid molecules according to the invention and to various uses of the same.<br>
Microorganisms are able to respond to a number of stress situations, such as heat or cold shock, ethanol, heavy-metal ions, oxygen deprivation, or nutrient deprivation, in particular glucose deprivation. Yeasts and other fungi are known to accumulate trehalose during phases of reduced growth. These are generally the stages of development which, for example, are tolerant of water deprivation and heat, such as spores, conidiae, sclerotia, or cells in the stationary growth phase. It is also already known that Saccharomyces cerevisiae cells accumulate trehalose during a one-hour heat shock from 27 °C to 40 °C and that the trehalose accumulation correlates to an increased thermotolerance. Selective mutations have been used to demonstrate that trehalose is indeed a necessary factor for the induction of thermotolerance.<br>
HSEs (heat shock elements) and STREs (stress responsive elements) are present in the promoter regions of stress-induced genes, such as the genes of S. cerevisiae responsible for the trehalose synthesis. These elements appear to mediate activation of stress genes by stress induction, including heat shock induction. It is now generally accepted that phosphorylation of Msn2p and Msn4p via the Ras/cAMP pathway inhibits the Msn2p and Msn4p transcription factors. In the absence of this inhibition (e.g. under stress conditions) Msn2p and Msn4p become active. STREs with the sequence CCCCT are attributed with a role in the response to the stress conditions.<br>
Owing to their ability to perform cotranslational and posttranslational modifications which are similar to the human modifications, fungi, and in particular yeasts, are attractive systems for the production of recombinant proteins. For the production of recombinant proteins the coding sequence of a gene which encodes a protein of interest is often expressed under the control of a suitable heterologous promoter. The so-called inducible<br><br>
promoters which can be induced by particular environmental conditions have proved particularly advantageous for this purpose. The promoters of genes which encode key enzymes in the methylotrophic metabolism, such as the MOX (methanol oxidase) or the FMD (formate dehydrogenase) promoter, for example, offer widely exploitable possibilities for an heterologous gene expression which is regulated strongly by the carbon source.<br>
Expression vectors have been produced for research in molecular biology which comprise a heat-inducible promoter, for example that of the hsp70 gene from Drosophila. The promoters employed in the past for heat shock induction in fungal cells and in particular in yeasts have the drawback that they do not respond selectively to heat shock. Their mechanism of activation and deactivation cannot therefore be controlled sufficiently well, which can cause problems in particular during the production of proteins which are damaging to cells. The TPS1 promoter from S. cerevisiae, for example, exhibits several sequences known to be general stress elements (STRE elements), namely CCCCT and AGGGG, but no more than one sequence acting as a heat-shock element (HSE), namely GGAACAGAACAATCG. In addition, owing to their wide stress response, the promoters currently known are activated by a stress factor to a degree which is not satisfactory for many applications.<br>
The object of the invention is therefore to provide a promoter the heat-inducible characteristic of which is as selective as possible, specifically a promoter which is active in fungi and in particular in yeasts, and which is suitable for protein expression at high temperatures.<br>
According to the invention, this object is achieved by a nucleic acid molecule comprising a heat-inducible promoter and which is selected from the following nucleic acids:<br>
(a)	a nucleic acid the sequence of which comprises the promoter sequence of a Hansenula polymorpha gene coding for a protein with trehalose-6-phosphate synthase activity;<br>
(b)	a nucleic acid with the sequence indicated in SEQ ID NO:1;<br><br>
(c)	a nucleic acid with a sequence which exhibits at least 40% identity over a length of 300 bp with one of the sequences indicated in (a) or (b);<br>
(d)	a nucleic acid which hybridizes to the complementary strand of one of the nucleic acids indicated in (a), (b) or (c);<br>
(e)	a derivative of one of the nucleic acids indicated in (a), (b) or (c) obtained by substitution, addition and/or deletion of one or more nucleotides;<br>
(f)	a fragment of one of the nucleic acids indicated in (a) to (e) which retains the function of the heat-inducible promoter;<br>
(g)	a combination of several of the nucleic acids indicated in (a) to (f), wherein the sequences of the nucleic acids may be different or the same;<br>
or<br>
by a nucleic acid molecule the sequence of which is complementary to the sequence of one of the nucleic acids indicated in (a) to (g).<br>
The term "heat-inducible promoter, as employed in this context, refers to a nucleic acid sequence which, at a temperature rise in the culture medium from 25°C to at least 37°C, preferably to 47°C, brings about an increase of at least 50% in the transcription (RNA synthesis) of a gene under the transcriptional control of the promoter.<br>
"Trehalose-6-phosphate synthase activity" refers to the conversion of glucose-6-phosphate (Glu6P) and UDP-glucose (UDPG) to trehalose-6-phosphate and UDP, which is catalyzed by the enzyme trehalose-6-phosphate synthase (TPS). The trehalose-6-phosphate synthase activity of a protein or polypeptide can be measured for example by the method described below under "Materials and Methods".<br>
The feature "sequence which hybridizes to the complementary strand of one of the nucleic acids indicated in (a), (b) or (c)" refers to a sequence which hybridizes under stringent conditions with the complementary strand of a nucleic acid having the features<br><br>
indicated in (a), (b) or (c). For example, hybridization may be performed at 68°C in 2 x SSC or according to the protocol of the Dioxygenin labelling kit manufactured by Boehringer (Mannheim). A further example of stringent hybridization conditions is incubation at 65°C overnight in 7% SDS, 1% BSA, 1 mM EDTA, 250 mM sodium phosphate buffer (pH 7.2) followed by washing at 65°C with 2 x SSC, 0.1% SDS.<br>
The term"% identity", as known in the art, refers to the degree of similarity between the sequences of two or more DNA molecules or of two or more polypeptide molecules, as determined by a comparison of the sequences. The percentage of the "identity" results from the percentage of identical regions in two or more sequences in consideration of gaps or other particular sequence features.<br>
The identity of related DNA molecules or polypeptides can be determined by means of known procedures. In the main, dedicated computer programs are employed using algorithms which make allowance for the particular requirements. Preferred methods for determination of the identity first generate the greatest matches between the sequences studied. Computer programs for determining the identity between two sequences include, but are not limited to, the GCG program package, including GAP (Devereux, J., et a/.f Nucleic Acids Research 12 (12): 387 (1984); Genetics Computer Group University of Wisconsin, Madison, (Wl)); BLASTP, BLASTN and FASTA (Altschul, S. etal, J. Molec Biol 215:403/410 (1990)). The BLAST X program can be obtained from the National Centre for Biotechnology Information (NCBI) and from other sources (BLAST Manual, Altschul S., et a/., NCB NLM NIH Bethesda MD 20894; Altschul, S., et a/., J. Mol. Biol. 215:403/410 (1990)). The well known Smith Waterman algorithm may also be used to determine identity.<br>
Preferred parameters for sequence comparison comprise the following:<br>
Algorithm:	Needleman and Wunsch, J. Mol. Biol 48:443-453 (1970)<br>
Comparison matrix:	Matches        = +10,<br>
Mismatches  =     0<br>
Gap penalty:	50<br>
Gap length penalty:	3<br><br>
The GAP program is also suitable for use with the above parameters. The above parameters are the default parameters for nucleic acid sequence comparisons.<br>
Other algorithms, gap opening penalties, gap extension penalties, comparison matrices including those set forth in the Program Manual, Wisconsin Package, Version 9, September 1997, may be employed. The choices to be made will depend upon the specific comparison to be made, and additionally whether the comparison is between pairs of sequences, in which case GAP or Best Fit are preferred, or between one sequence and a large database of sequences, in which case FASTA or BLAST are preferred.<br>
Surprisingly, the nucleic acid molecules according to the invention, and in particular the promoter of the trehalose-6 phosphate synthase (TSP1) gene of Hansenula polymorpha, have now been found to contain, at least in the first 300 bp upstream of the coding sequence, none of the STRE elements which were found in S. cerevisiae and which were assumed to be primarily responsible for the stress response including heat-shock induction of this gene. This promoter was further found to respond well and very selectively to heat.<br>
The nucleic acid molecules according to the invention may either be prepared synthetically by conventional methods, or isolated from suitable DNA libraries and subsequently mutated as required. The preparation of such libraries is also known in the art. Isolation is preferably performed by preparing a probe with a length of at least 200 - 400 bp of the coding sequence of the TPS1 gene of H. polymorpha (see Figure 6), which is used to screen a DNA library, in particular a genomic DNA library. A probe of this kind can be prepared by means of PCR (polymerase chain reaction) using suitable primers, each of which should preferably be at least 20 - 21 bp in length and possess suitable sequences according to Figure 6 (or the corresponding complementary sequence), and genomic DNA or cDNA from H. polymorpha as a "template".<br>
Probes may either be synthesized, or prepared by fragmentation of available TPS1 DNA where applicable. It is of course also possible to screen directly by means of probes that correspond to parts of the promoter sequence; this procedure is less preferable,<br><br>
however, owing to the at best incomplete conservation of the sequence within non-coding parts.<br>
In an embodiment of the nucleic acid molecules according to the invention, the sequence of the nucleic acid exhibits at least 60%, preferably at least 80% identity over a length of 300 bp with one of the sequences indicated above under (a) or (b).<br>
Nucleic acid molecules which comprise a heat-inducible promoter and which exhibit at least 90% identity over a length of 300 bp with one of the sequences indicated above under (a) or (b) are particularly preferred. Most preferred are however nucleic acid molecules which exhibit at least 95% identity over a length of 300 bp with one of the sequences indicated above under (a) or (b).<br>
Nucleic acid molecules preferred for carrying out the invention exhibit at least one heat shock element with the sequence NGAANNNNNNNGAAN (SEQ ID NO:2) or the complementary sequence thereof, wherein the nucleotides denoted by N may be A, T, C and G independent of each other. The nucleic acid molecules according to the invention preferably exhibit at least one heat shock element with the sequence NGAANNBWMNNGAAN (SEQ ID NO:3) or the complementary sequence thereof, wherein BisaG, C orT, Wan AorT, andMaCorA.<br>
In a particularly preferred embodiment of the invention, the heat shock element is selected from TGAAGCCTCTTGAAA (SEQ ID NO:4) and/or TGAATATAAAGGAAA (SEQ ID NO:5) and/or the complementary sequences thereof, wherein two or more heat shock elements, where present, may exhibit the same or different sequences. A preferred nucleic acid molecule according to the invention exhibits at least two different heat shock elements.<br>
In a preferred embodiment of the invention, the nucleic acid molecules according to the invention do not contain an STRE element having the sequence CCCCT or AGGGG.<br>
The invention also provides fragments of the nucleic acid molecules according to the invention as stated above which retain the function of the heat-inducible promoter. A fragment comprising the sequence from nucleotide 228 to nucleotide 792 in the SEQ ID<br><br>
N0:1 is particularly preferred. A further preferred fragment comprises the sequence from nucleotide 493 to nucleotide 792 in the SEQ ID NO:1. A fragment comprising the sequence from nucleotide 627 to nucleotide 713 in the SEQ ID NO:1 may also be used.<br>
The nucleic acid molecules according to the invention may further comprise at least one nucleic acid sequence for a heterologous gene under the transcriptional control of the heat-inducible promoter.<br>
A "heterologous gene" shall refer to the coding part of a structural gene which is either not expressed under control of its own (homologous) promoter, or is not expressed in the organism from which the gene derives, or is expressed neither under the control of the original promoter nor in the original organism.<br>
In a further embodiment of the invention, the nucleic acid molecules according to the invention comprise a nucleic acid sequence under the transcriptional control of the heat-inducible promoter which is selected from the following sequences:<br>
(i)       a nucleic acid sequence which encodes a polypeptide with the amino acid sequence of the trehalose-6-phosphate synthase of Hansenula polymorphs]<br>
(ii)      a nucleic acid sequence as indicated in SEQ ID NO:6;<br>
(iii)      a nucleic acid sequence which exhibits at least 80% identity with the sequence indicated in SEQ ID NO:6;<br>
(iv)     a nucleic acid sequence which encodes a polypeptide with the amino acid sequence indicated in SEQ ID NO:7 or with a partial sequence thereof, wherein the polypeptide exhibits trehalose-6-phosphate synthase activity;<br>
(v)      a nucleic acid sequence which in consideration of the degeneration of the genetic code would code a polypeptide with the amino acid sequence indicated in SEQ ID NO:7 or with a partial sequence thereof, wherein the polypeptide exhibits trehalose-6-phosphate synthase activity;<br><br>
(vi)     a nucleic acid sequence which encodes a polypeptide the amino acid sequence of which exhibits at least 80% identity with the amino acid sequence indicated in SEQ ID NO:7.<br>
The nucleic acid sequence-indicated under (iii) preferably exhibits at least 90% identity with the sequence indicated in SEQ ID NO:6. In an alternative form of the nucleic acid molecules according to the invention, the nucleic acid sequence indicated under (vi) encodes a polypeptide the amino acid sequence of which exhibits at least 90% identity with the amino acid sequence indicated in SEQ ID NO:7.<br>
The nucleic acid molecule according to the invention may further comprise a nucleic acid sequence encoding a signal peptide which ensures export of the expressed protein, wherein the nucleic acid sequence encoding the signal peptide is preferably bound directly to the heterologous gene to be expressed. The secretion and modification of many eukaryotic proteins requires that the N-terminus of the protein sequence be fused with a signal sequence, in order to direct the polypeptides into the secretion apparatus. Components from the S. occidentalis gene GAM1 and from a hormonal gene of the crab Carcinus maenas, which have been used successfully for the secretion of hirudin (Weydemann et al., 1995), may for example be considered here. The nucleic acid molecule according to the invention may further comprise a terminator element containing signal structures for the RNA polymerase which lead to termination of the transcription. Examples of terminator elements which may be employed are the MOXor the PH01 terminator of H. polymorpha.<br>
A further subject matter of the invention is a host cell containing at least one nucleic acid molecule according to the invention, wherein the host cell is a prokaryotic or eukaryotic cell. The eukaryotic cell may for example be a plant cell. The eukaryotic cell is preferably a fungal cell, a yeast cell is particularly preferred. Fungi are given particular consideration as host cells for carrying out the present invention, for example filamentous fungi such as Aspergillus, Neurospora, Mucor, Trichoderma, Acremonium, Sordaria and Penicillium or yeasts such as Saccharomyces, Hansenula, Pichia, Kluyveromyces, Schwanniomyces, Yarrowia, Arxula, Trichosporon and Candida.<br><br>
In the most preferred embodiment of the invention the yeast cell is a facultative methylotrophic Hansenula yeast, preferably Hansenula polymorpha. H. polymorpha is a thermotolerant yeast cell and belongs to the small group of the so-called methylotrophic yeasts which are capable of using methanol as carbon and energy source. H. polymorpha was isolated from soil samples by incubation at 37°C (Levine and Cooney, 1973). The high temperature at which H. polymorpha continues to grow and produce protein enables other undesired organisms to be eliminated. The reason for this is that H. polymorpha has been shown not only to possess a very high optimum growth temperature, in the region of 37°C, but also to be able to survive temperatures of approximately 50°C unharmed (see Figure 1). The vitality of H. polymorpha following entry into the stationary phase does not fall for some 50 hours even at 47°C (Figure 2).<br>
A further subject matter of the present invention is an expression vector comprising at least one nucleic acid molecule according to the invention. Such expression vector may also contain other nucleic acid sequences in addition to the heat-inducible promoter, for example a sequence which encodes a polypeptide, a selection marker gene, an origin of replication for E. colif etc.<br>
The present invention also provides a kit comprising:<br>
(a)	an expression vector according to the invention which is suitable for having cloned into it a nucleic acid which encodes a recombinant protein, and<br>
(b)	a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein.<br>
"Cloning" is to comprise all cloning methods known in the art which could be employed for this purpose. These methods are not all described here individually, being familiar to a person skilled in the art.<br>
The invention further provides a kit comprising<br>
(a) an expression vector, and<br><br>
(b) a host cell suitable for induction of the heat-inducible promoter and for<br>
production of a protein encoded by a coding sequence under the transcriptional control of the heat-inducible promoter.<br>
The nucleic acid molecules, host cells, expression vectors and kits according to the invention may be used for recombinant expression of a gene under the control of the heat-inducible promoter, or for production of one or more proteins.<br>
"Recombinant expression in a suitable host cell" shall refer to all expression methods known in the state of the art in known expression systems which could be used for this purpose. These methods are not all described here individually, being familiar to a person skilled in the art.<br>
A further subject matter of the invention is a method for the production of one or more proteins, said method comprising:<br>
(i)       cloning at least one nucleic acid encoding a recombinant protein into an<br>
expression vector according to the invention, such that the nucleic acid thus cloned is under the transcriptional control of the heat-inducible promoter;<br>
(ii)      introduction of the expression vector obtained in (i) into a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein;<br>
(iii)      cultivation of the host cell obtained in (ii);<br>
(iv)     induction of the heat-inducible promoter by methods known per se.<br>
Should the expression vector according to the invention contain a sequence encoding a polypeptide and being under the transcriptional control of the heat-inducible promoter, the method according to the invention for production of one or more proteins comprises the following steps:<br><br>
(i)       introduction of an expression vector into a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein;<br>
(ii)      cultivation of the host cell obtained in (i);<br>
(iii)      induction of the heat-inducible promoter by methods known per se.<br>
The invention is now described in closer detail with reference to the figures, which show the following:<br>
Figure 1 shows growth curves of H. polymorphs at 27°C, 37°C and 47°C.<br>
Figure 2 shows the vitality following entry into the stationary phase at 27°C, 37°C and 47°C.<br>
Figure 3A shows a Northern blot of RNA from wild-type H. polymorphs following a heat shock from 27°C to 47°C and subsequent cooling to 27°C. The cells were cultivated in YDP medium at 27°C to the early exponential phase; the temperature was then increased to 47°C (time zero), and reduced again to 27°C after 120 minutes.<br>
Figure 3B shows a Western blot for the Tps1 protein (Tps1 p) from H. polymorphs following a heat shock from 27°C to 47°C and subsequent cooling to 27°C (see Figure 3A), from which a correlation can be seen between in the increase of TPS1 mRNA and the increase in Tps1 protein (Tpslp).<br>
Figure 3C shows the intracellular trehalose concentration and the trehalose-6-phosphate synthase activity plotted against time for H. polymorphs following a heat shock from 27°C to 47°C and subsequent cooling to 27°C (see Figure 3A). The open circles represent the intracellular trehalose concentration, the solid squares the trehalose-6-phosphate synthase activity. A correlation is evident from the figure between the increase in TPS1 mRNA, and the increase in trehalose-6-phosphate synthase activity and the intracellular trehalose concentration.<br><br>
Figure 4 shows three bar charts representing the trehalose-6-phosphate synthase activity (white bars) and the intracellular trehalose concentration (black bars) in cells of Hansenula polymorpha cultivated at 27°C (A), 37°C (B) and 47°C (C) and under glucose deprivation after 7,10,17 and 36 hours. The trehalose accumulation correlates to the increase in trehalose-6-phosphate synthase activity (Figure 4A), to that of the TPS1 mRNA (Figure 4B) and to that of the Tps1 protein (Tpslp) (Figure 4C).<br>
Figure 5 shows the homology of certain DNA sequence regions of trehalose-6-phosphate synthase from a number of organisms.<br>
Figure 6 shows the DNA sequence of the TPS1 gene of H. polymorpha (SEQ ID NO:8) and the derived amino acid sequence (SEQ ID NO:6). The heat shock elements in the promoter sequence are underlined.<br>
Figure 7 shows the plasmid pC11, a derivative of pM1 (M. Suckow, personal communication), which was obtained by insertion of the lacZ gene into the polylinker of pM1. The plasmid contains the HARS1 sequence (H. polymorpha Autonomously Replicating Sequences), the on (origin of replication) from pBR322, an ampicillin-resistance gene, the URA3 gene for propagation and selection in H. polymorpha and in E. colit and a MOX terminator behind the lacZ gene for termination of the transcription process.<br>
Figure 8 shows the plasmid pC11-FMD obtained by insertion of the FMD promoter in front of the lacZ reporter gene of pC11.<br>
Figure 9 shows the plasmid pC11-TPS1 obtained by insertion of the TPS1 promoter in front of the lacZ reporter gene of pC11.<br>
Figure 10 shows a comparison between the activity of FMD (A) and TPS1 promoters (B) at 30, 37 and 44°C in three different carbon sources (2% glucose, 2% glycerine or 2% methanol).<br>
Figure 11 shows the plasmid pTPS1 ConphysMT used in Example 4. MOX-T = MOX terminator, Conphys = Conphys3 gene, TPS1 = TPS1 promoter of Hansenula<br><br>
polymorphs, HARS = H- polymorphs Autonomously Replicating Sequences, tet = tetracycline-resistance gene, URA3 = URA3 from S. cerevisiae, amp = ampicillin-resistance gene<br>
Examples<br>
Materials and Methods:<br>
Special reagents and materials<br><br>
Bio 101, Vista, USA<br>
BioRad Lab., Munich, Germany<br>
Boehringer, Mannheim, Germany<br>
Fluka Chemie AG, Buchs, Switzerland<br>
ICN Biochemicals, Ohio, USA<br>
Kodak, New York, USA Mediatech, Herndon, USA Perkin Elmer Applied Biosystems,<br>
Pharmacia Biotech, Sweden<br>
Qiagen GmbH, Germany<br>
Schleicher + Schuell, Dassel, Germany<br>
Sigma, St. Louis, USA<br><br>
Geneclean II Kit<br>
BioRad Protein Assay (Bradford)<br>
GOD/POD kit for glucose measurement,<br>
ethanol kit, "COMPLETE" proteinase inhibitor<br>
cocktail tablets<br>
Cycloheximide (Actidion), SDS, D+trehalose,<br>
PEP, TRICIN, NADH, Folin-Ciocalteu phenol<br>
reagent<br>
"Liquigel" 40% acrylamide/N'N'-methylene-<br>
bisacrylamide (37.5:1)<br>
BIOMAX MR scientific imaging film<br>
Geneticin G418 sulphate (antibiotic)<br>
DNA sequencing kit<br>
Forest City, USA<br>
Nap-10 columns (with Sephadex G-25), all<br>
restriction enzymes used, Taq polymerase<br>
Plasmide Midi Kit (50)<br>
Protran BA 83 0.2 ^im/ 0 82 mm (cellulose<br>
nitrate round filter), Protran BA 83 0.2 urn<br>
(transfer membrane for blots)<br>
Monoclonal goat anti-rabbit immunoglobulins<br>
(alkaline phosphatase conjugate), trehalase<br>
from pig kidneys (Cat. No. T-8778), UDPG,<br>
glucose-6-P, LDH, pyruvate kinase<br><br>
Stratagene, La Jolla, USA<br>
US Biological, Swampscott, USA<br><br>
Prime-It II kit (random primer labelling kit), NucTrap columns (probe purification columns incl. push column beta shield device) Bacteriological Agar, YPD broth enhanced formulation W/Peptone X, LB broth Miller<br><br>
Apparatus used<br><br>
Electroporation unit<br>
HPLC<br>
Cooling centrifuges<br>
PCR apparatus Phosphoimager<br>
Photometer<br>
Sequencer<br><br>
E. coli pulser, BioRad Laboratories, Hercules<br>
USA<br>
DIONEX DX-300, DIONEX, Sunnyvale, USA<br>
Centrikon H-401, Kontron Instr. AG, Zurich,<br>
Switzerland<br>
IEC Centra GP8R, Brouwer, Lucerne,<br>
Switzerland<br>
Biofuge 17RS, Heraeus Sepatech, Germany<br>
Progene, Techne, Cambridge, United Kingdom<br>
GS 250 Molecular Imager (including<br>
associated equipment), BioRad Laboratories,<br>
Hercules, USA<br>
Anthos 2001 (for microtiter plates), Anthos<br>
Labtec Instruments, Salzburg, Austria<br>
Shimadzu UV-160A, Japan<br>
ABI PRISM 301 Genetic Analyzer, Perkin<br>
Elmer, Applied Biosystems, Foster City, USA<br><br>
Bacterial strain and culture conditions<br>
The E. coli strain DH5a (F'endA1/?sc/R17rkmk+ supE44thi-1recA1gyra relA(lacZYA-argF) U169(80A(lacZ)M15) (Gibco BRL, Gaithersburg MD, USA) was employed for cloning of the TPS1 gene of H. polymorpha, the standard protocols (Sambrook et a/., 1989) being followed. The medium for E. coli was also produced in accordance with a standard recipe (Sambrook et a/., 1989).<br><br>
Isolation of plasmid DNA from E. coll (STET prep)<br>
Plasmid DNA was isolated in accordance with a modified protocol according to Sambrook etal. (1989). A spatula was used to scrape cell material from a plate. This material was then added to 500 pi STET (8% [w/v] sucrose, 5% [v/v] Triton X-100, 50 mM EDTA, 50 mM Tris-HCIf pH 8.0) with 35 pi lysozyme (10 mg/ml) and mixed. The samples were then boiled for 1 min 40 s at 100°C and centrifuged for 10 minutes at 20,000 g and 4°C. Approx. 400 pi of supernatant was drawn by means of a pipette, and the DNA precipitated using 400 pi isopropanol. Following centrifugation for 10 minutes at 20,000 g and 4°C, the entire supernatant was discarded and the DNA pellet washed once with ice-cold 70% [v/v] ethanol. Finally, the DNA was dried at room temperature and suspended in 50-70 pi TE (10 mM Tris-HCI, pH 8.0,1 mM EDTA, pH 8.0).<br>
Yeast strain and culture conditions<br>
The yeast strain employed was a wild-type Hansenula polymorphs (made available by P. Piper, London (1994)). Stock cultures were grown on YPD Agar (2% [w/v] glucose, 2% [w/v] bactopeptone, 1% [w/v] yeast extract, 2% [w/v] agar) and re-stocked every six weeks. They served as inoculum for YPD liquid cultures (composition the same as YPD agar, but without 2% [w/v] agar).<br>
The strain H. polymorphs RB11 (odd orotidine-5-phosphate-decarboxylase-deficient (uracil-auxotrophic) H. polymorphs strain (Weydemann et si, 1995)) was used for the experiments in Examples 3 and 4. The full medium employed contained 2% glucose or glycerine, 1% yeast extract and 2% bactopeptone; the selection medium contained 0.17% yeast nitrogen base, 0.5% ammonium sulphate, 2% glucose or glycerine, 38.4 mg/l arginine, 57.6 mg/l isoleucine, 48 mg/I phenylalanine, 57.6 mg/l valine, 6 mg/ml threonine, 50 mg/l inositol, 40 mg/l tryptophan, 15 mg/l tyrosine, 60 mg/l leucine, 4 mg/l histidine. Uracil is not present in the selective medium.<br>
For the cultivation of cell cultures, autoclaved liquid media were inoculated with stock culture and incubated overnight in shaking incubators at 27°C, 37°C or 47°C, depending upon the experiment.<br><br>
Determination of the optical density of the ft polymorpha cell cultures<br>
In order to determine the optical density (OD), 200 nl (suitably diluted where applicable with YPD) cell culture was placed in a vial of a microtitre plate and measured at 620 nm using an Anthos 2001 photospectrometer. 200 ^l YPD was employed as the blank.<br>
Growth and heat shock experiments with ft polymorpha<br>
Overnight cultures were used to inoculate YPD medium in Erlenmeyer flasks. Care was taken to inoculate this preculture at the temperature at which the experiment itself was later begun (27°C for heat shocks, 27°C, 37°C or 47°C for growth experiments).<br>
The cultures were inoculated to an initial OD62o of 0.2 for each growth experiment, and maintained continuously in shaking incubators (Multitron). Conversely, the cultures were inoculated to an initial OD of 0.05 for heat shock experiments. The culture was allowed to grow at 27°C up to an OD62o of 0.4 (approx. 1-1.5 x 108 cells per ml of culture) before performance of the heat shock to 47°C in a water bath with shaking function (Aquatron). Samples were then taken over a further two hours. The cell culture was then cooled in a second water bath for one hour to 27°C.<br>
Transformation of ft polymorpha by electroporation<br>
100 ml of YPD was inoculated with 5 ml of a densely grown overnight culture. The culture was shaken at 37°C for approximately three hours to an ODeoo of 0.8-1.2. The cells were harvested by centrifugation at 3,000 rpm and resuspended in 20 ml Kpi buffer (50 mM/pH 7.5). Following addition of 0.5 ml DTT and shaking for 15 minutes at 37°C, the cells were sedimented by centrifugation at 2,500 rpm and washed twice with STM buffer (270 mM sucrose, 10 mM TrisCI, 1 mM MgCI2, pH 7.5). They were then suspended in 0.25 ml STM buffer, and 60 JJI aliquots stored at -70°C. For transformation with rDNA integrative vectors, the plasmid DNA was first linearized with Xho\ or Sac\. 0.1-1 |jg of the linearized plasmid DNA was mixed with fresh competent cells defrosted on ice. These preparations were then placed in a 2 mm cuvette. Transformation was performed in a Gene Pulser (Bio-Rad, Munich) at 2.0 kV, 25 MF and 200 Ohm. The cells were then incubated in 1 ml YPD for one hour at 37°C for recovery before being plated out on<br><br>
selective medium. Macroscopic colonies were visible following incubation for two to four days at 37°C.<br>
Determination of the glucose concentration in the medium<br>
The glucose concentration in the medium was determined by means of the GOD method (GOD/POD Kit, Bohringer). Samples were diluted 1:200 with water. 190 (j| 1% (w/v) GOD enzyme solution (supplied in powder form with the kit) was added to 10 pi of each sample and the mixture was incubated for approximately 25 minutes at 27°C. The glucose solution supplied in the kit was used as the standard, 10 pi (0.91 pg glucose) also being employed here. The absorption was measured in the Anthos 2001 spectrophotometer at 405 nm.<br>
Extraction and quantitative detection of trehalose<br>
Extraction of trehalose<br>
1-10 ml of cell culture was filtered through a glass-fibre filter (Whatman GF/C) and washed three times with water. The filter was placed in an Eppendorf tube with 1 ml of water and vortexed for 30 seconds before being carefully squeezed out and removed. The cell suspension was then boiled for 10 minutes in the water bath. In order to separate the supernatant completely from the cell material, it was centrifuged three times at 20,000 g.<br>
Determination of trehalose bv HPLC<br>
The extracted sugars were separated by means of a anion exchanger column (DIONEX<br>
CarboPac PA1 column, 4 x 250 mm) and detected amperometrically on a gold electrode<br>
(PED = pulsed electrochemical detector). The composition of the eluting gradientisas<br>
follows:<br><br><br>
These conditions resulted in a retention time for trehalose of approximately 3.7 minutes. 20 pi of sample was injected in each case. A 0.1 mg/ml trehalose solution was employed as the standard.<br>
Determination of trehalose by enzymatic assay<br>
An equally reliable enzymatic assay method was used in some cases as an alternative to the more expensive HPLC method (Parrou and Frangois, 1997, modified): 25 pi of trehalose extract was mixed with 12.5 pi of trehalase (Sigma) and 37.5 pi buffer solution (0.2 M sodium acetate, 0.03 M CaCl2, pH 5.7) and incubated for five hours at 37°C in a<br>
water bath. This resulted in complete breakdown of trehalose to two units of glucose. Following brief centrifugation, the samples were incubated for three minutes at 95°C and then centrifuged again for a further five minutes at 20,000 g. The trehalose concentration was determined indirectly by determination of the glucose concentration (GOD/POD kit, see above). 10 pi of this supernatant was used for this purpose.<br>
Protein determination<br>
Protein determination according to Peterson (slightly modified) (Peterson (1997) In order to determine the total protein concentration of a cell culture 1 ml of cell suspension was precipitated in 1 ml 10% (w/v) TCA and centrifuged for 10 minutes at 3,000 g . The supernatant was drawn by means of a Pasteur pipette connected to a water-jet pump, and the sediment washed in 1 ml 1 N PCA. The pellet was then suspended in 5-12 ml (depending upon the OD of the cell culture to be studied) of a solution of 0.8 N NaOH:10% (w/v) SDS (1:1) and incubated for at least one hourat60°C.<br><br>
200 |jl of this suspension was added 600 pi 6X dilution of CTC reagent (10% Na2C03,<br>
0.1% CuS04 • 5H20, 0.2% KNa tartrate). After exactly 10 minutes, 200 |jl 6x dilution of<br>
Folin-Ciocalteu reagent was added and mixed briefly. The samples were left in the dark for 30 minutes, after which the absorption was measured at 750 nm, BSA serving as the standard.<br>
Protein determination according to Bradford (1976)<br>
In order to determine the protein concentration in cell-free extract, 100 μl of a suitably diluted extract was mixed with 700 pi of water. 200 pi of BioRad protein assay reagent (Bradford) was then added and briefly shaken (Vortex). The absorption was measured at 595 nm, BSA serving as the standard.<br>
Enzyme activity measurements<br>
Preparation of permeabilized cells<br>
The enzymatic activity of the trehaIose-6-phosphate synthase (Tre-6-P synthase) was measured in permeabilized cells (De Virgilio et a/., 1991). For this purpose, 1-6 ml of cells was filtered (on GF/C glass-fibre filters, Whatman), washed twice using ice-cold water, and resuspended by vortexing in 1 ml lyse buffer (0.2 M TRICIN, pH 7.0, 0.5% [v/v] Triton X-100). The filters were removed and the Eppendorf tubes frozen in liquid nitrogen and stored at -20°C. Prior to performance of the measurement, the cells were defrosted in a water bath for three minutes at 30°C. They were then washed twice in 0.2 M TRICIN (pH 7.0), and centrifuged for 20 s at 4°C and 8,000 rpm (Biofuge 17RS) after each wash. Finally, the ceils were resuspended in 600 pi 0.2 M TRICIN (pH 7.0).<br>
Trehalose-6-phosphate synthase activity<br>
The Tre6P synthase activity was determined by the coupled enzymatic assay according to Hottiger et al. (1987) at 50°C, 60 pi permeabilized cells always being employed. Both substrate (without glucose-6-P) and enzyme blanks (without permeabilized cells) were processed as controls.<br><br>
Western Blot analyses<br>
Protein extraction bv cell disruption<br>
5-15 ml of cell culture was centrifuged for 5 minutes at 4°C and 3,000 rpm (EC Centra GP8R), and the supernatant then decanted. The pellet was suspended in 1 ml of water and transferred in a Sarstedt tube (with screw closure). Following centrifugation for 10 seconds, the supernatant was drawn by means of a pipette and the pellet weighed, an empty tube serving as a dead weight. 1 μl 0.2 M TRICIN buffer (pH 7.0; with proteinase inhibitors [2 tabs/ 25 ml]) was added per mg pellet and the pellet resuspended. Glass beads were added until just below the liquid meniscus, after which the Sarstedt tubes were mounted firmly in a cell homogenizer (Fastprep FP120) in the cold store. The cell homogenizer was run twice for 30 seconds at a setting of 6.0, resulting in &gt;90% cell disruption. From this point onwards, strict attention was paid to maintaining the samples well cooled at all times. A small hole was produced in the Sarstedt tube by means of a needle. The tube was placed upon a glass tube and centrifuged at 4°C and 100 g, thereby separating the extract from the glass beads. The quantity of TRICIN buffer used for cell disruption was then added once to the Sarstedt tubes, which were centrifuged again. The cloudy extract was then transferred to Eppendorf tubes and centrifuged three times for ten minutes at 25,000 g and 4°C (Biofuge 17RS), the supernatant containing the soluble proteins (including the Tre6P synthase) being subsequently used each time.<br>
Sample preparation<br>
The protein concentration of these extracts was then determined with the Bradford method (see above). According to the values obtained, they were diluted with water to 2.5 ng protein/ul, and one volume of 5x sample buffer was added to four volumes of this protein solution. The samples were then denatured for five minutes at 95°C and either used immediately for SDS gel electrophoresis, or frozen. 10 jil, i.e. 20 jxg protein, was used for the analysis.<br>
Sample buffer:	1 ml 0.5 M Tris-HCI, pH 6.8, 0.8 ml glycerine, 1.6 ml 10%<br>
[w/v] SDS, 0.2 ml 0.05% [w/v] bromophenol blue, 4 ml water. 19 volumes sample buffer were added to one volume 2-p-mercaptomethanol immediately prior to use.<br><br>
SDS polvacrvlamide gel electrophoresis (SDS-PAGE)<br>
The system according to Laemmli etal. (1970) was employed for separation of the proteins according to their molecular weight. A 10% and a 4% acrylamide gel (total dimensions 10X10 cm) with the following composition were prepared for use as the resolving gel and stacking gel respectively:<br><br>
20 \ig of protein was loaded onto each gel. The "Kaleidoscope prestained standard" from BioRad, the composition of which is as follows, was employed as the standard: myosin (204 kDa), p-galactosidase (121kDa), BSA (78 kDa), carboanhydrase (39 kDa)t soy trypsin inhibitor (30 kDa). The gel electrophoresis was performed for approximately one hour (but no longer than for the sample front to reach the lower edge of the gel) at a constant voltage of 200 V. These gels were then either stained with 0.1% (w/v) Coomassie Blue R250 in 10% (v/v) acetic acid/50% (v/v) ethanol (and destained after approximately one hour with 10% (v/v) acetic acid, 20% (v/v) ethanol) or blotted on nitrocellulose (refer to next section).<br>
ImmunoblottinQ<br>
The SDS-PAGE gels were then blotted on nitrocellulose in a blotting unit (Sciepias) with transblot buffer (250 mM Tris, 1250 mM glycine, 15% (v/v) methanol) for 1 hour 15 minutes at 40 V and 4°C.<br><br>
Immune staining<br>
The nitrocellulose membrane was first held for at least one hour in a saturation solution comprising 3% (w/v) BSA in TBS (TBS: 20 mM Tris, 500 mM NaCI, pH adjusted to 7.5 with HCI), followed by washing for 5 minutes using TTBS (TTBS: as TBS, but with 0.05% Tween-20). Polyclonal anti-Tps1p rabbit antibody (diluted 1:50 with 1% [w/v] BSA in TTBS) (Eurogentec, Belgium) was then added overnight at 4°C, the purpose of which was to bind to the Tps1 protein (Tpslp) from H. polymorphs present on the nitrocellulose.<br>
The nitrocellulose blot was subsequently washed twice for 5 minutes with TTBS and incubated for 1 hour 30 minutes with a monoclonal anti-rabbit antibody coupled with alkaline phosphatase (diluted 1:10.000 with 1% [w/v] BSA in TTBS). Washing twice for 5 minutes with TTBS and once followed this for 5 minutes with TBS. In order to develop the staining of the bands, 1 ml 10x colour development buffer (100 mM Tris-HCI, pH 9.5,1 mM MgCI) was diluted 1:10 with water and 45 \L\ NBT (75 mg/ ml 70% [v/v] DMF) and 35 y\ X-phosphate (50 mg 5-bromo-4-chloro-3-indolyI phosphate, toluidinium salt/ml DMF) were added. The membranes were incubated in the dark with this mixture for 20 minutes (or until the bands became clearly visible) before being washed with water in order to stop the reaction.<br>
Colony PCR with H. polymorpha cells<br>
Colony PCR was performed according to a protocol by Huxley et a/. (1990, modified): individual colonies were collected by means of a yellow pipette tip and scraped off in a PCR tube. The tubes were then heated for 2 minutes at full power in a microwave oven. Finally, 25 pi PCR mix (0.2 pi Taq polymerase, 2.5 pi 10x PCR buffer, 2.5 μl 25 mM MgCl2, 0.5 μl 10 mM dNTP, 0.5 pM per final concentration of each primer and water<br>
added to bring the volume up to 25 pi) was added to each tube, and the cells resuspended. The tubes were then immediately placed in the PCR unit, which was preheated to 920C, and the program started.<br><br>
Northern blot analysis<br>
RNA was extracted from H. polymorphs according to a protocol by Piper (1994, adapted). For this purpose, 40 ml logarithmic or 20 ml stationary cell culture was collected and (in heat shock experiments) cooled immediately by the addition of ice-cold, sterile DEPC water. The cells were then sedimented by centrifugation and washed again with sterile DEPC water. The pellet obtained following centrifugation and discarding of the supernatant was stored at -20°C. Following defrosting, 1-2 g glass beads, 2 ml RNA extraction buffer (20 mM Tris-HCI, pH 8.5,10 mM Na2-EDTA, 1% [w/v] SDS) and 2 ml phenol were added to the pellet This mixture was then vortexed without interruption for 5 minutes at room temperature, before being centrifuged for 5 minutes at 3,500 rpm (IEC Centra GP8R). The upper, aqueous phase was transferred to a new tube containing an equal volume of phenol/chloroform (1:1). The suspension was vortexed for 1 minute and centrifuged for 5 minutes at 3,500 rpm, and the supernatant placed in a new tube containing an equal volume of chloroform. Vortexing was repeated for 1 minute, centrifuging at 3,500 rpm for 2 minutes, and the supernatant transferred to 15 ml Corex tubes. 6 M ammonium acetate was added to a final concentration of 1 M ammonium acetate, followed by 2 volumes ethanol (ice-cooled), and the tubes were kept in the freezer compartment at -20°C for at least 20 minutes. The RNA was then sedimented by centrifugation for 15 minutes at 7,500 g and 4°C. The supernatant was decanted and the tubes dried on absorbent paper. The pellets were then suspended in 1 ml TE and the RNA precipitated by the addition of 3 M sodium acetate (to a final concentration of 0.2 M) and 2.5 volumes of ice-cold ethanol. Following centrifugation for 15 minutes at 7,500 g and 4°C, the pellet was washed with ice-cold 70% (v/v) ethanol and dried at room temperature. Finally, the RNA was resuspended in 400 μlTE.<br>
Sample preparation<br>
50 (xg RNA per sample was dried in the SpeedVac for 10-15 minutes for the Northern blot analysis (according to Sambrook et al., 1989). The RNA was then resuspended in 50 nl sample buffer (final concentrations: 20 mM MOPS, pH 7.0, 0.5 mM sodium acetate, 1 mM EDTA, pH 8.0, 2.2 M formaldehyde, 50% [v/vl] formamide) and heated for 10 minutes at 55°C. Finally, 5.5 RNA loading buffer (10x) and 1 μl ethidium bromide solution (1 (μ μml) were added to each sample.<br><br>
Pre-gel and main gel<br>
A pre-gel (1% [w/v] agarose and 0.65 M formaldehyde in a MOPS buffer containing<br>
40 mM MOPS, pH 7.0, 10 mM sodium acetate, 2 mM EDTA, pH 8.0) was used to test the<br>
integrity of the extracted RNA and to calibrate visually the loaded quantity. The main gel<br>
electrophoresis (composition identical to that of the pre-gel) was performed for 34 hours<br>
at 80 V with MOPS buffer serving as the running buffer.<br>
Blotting<br>
The gels were first washed twice for 20 minutes in 10xSSC(1.5 M NaCI, 170 mM<br>
sodium citrate). The RNA was then blotted overnight by capillary transfer (with 20 x SSC<br>
as the transfer buffer) onto a nitrocellulose membrane (BA 83). The membrane was then<br>
washed in 6x SSC, placed between 3MM filter papers (Whatman) and baked in a<br>
vacuum oven for 2 hours at 80°C, which enabled the RNA to be fixed to the<br>
nitrocellulose.<br>
Hybridization<br>
The nitrocellulose membrane was pre-hybridized in a special oven (Hybaid) in 10 ml RNA hybridization solution (0.5 M NaHP04l pH 7.2,1 mM EDTA, 1% [w/v] BSA. 7% [w/v] SDS) for 5 hours at 60°C. For the main hybridization stage, 150 pi of the radioactive probe (approximately 1 x 107 cpm in total) was added to 10 ml RNA hybridization solution and the membrane incubated in it overnight at 60°C. Finally, the surplus radioactivity was washed twice for 15 minutes at 60°C with 300 ml washing buffer (1 mM EDTA, 40 mM Na2HP04, pH 7.2,1% [w/v] SDS). The nitrocellulose membrane was exposed on BioMax film.<br>
Phytase detection<br>
The H. polymorpha cells were harvested from 3 ml overnight cultures and suspended in 200 |jl YNB medium and 1 ml 5% glycerine. Following growth over 1-2 days, the ODeoo was first determined. The cells were then sedimented by centrifugation and 25 |jl of the supernatant was subsequently used. 25 |j| 5 M NaAc and 50 pi 4-nitrophenyl phosphate were added to this aliquot. The mixture was incubated for 30 minutes at 37°C. The enzymatic conversion of the substrate was halted by the addition of 100 pi 15% trichloroacetic acid. Following the addition of 100 pi 1 M NaOH, supernatant samples of<br><br>
positive cultures were deep yellow coloured. The yellow colour was quantified by OD405 measurement in the photometer.<br>
X-gal overlay assay - detection of B-galactosidase<br>
The strains to be tested were cultivated in selective medium for 4-6 hours at 37°C. A 4 |jl drop of each culture was placed on a selective plate and incubated overnight at 37°C. The plate was coated with fresh top layer agar (0.5% agarose, 0.5 M Na2HP04/NaH2P04 (pH 7); 0.2% SDS; 2% DMF (dimethyl formamide) 2 mg/ml X-gal (o-nitrophenyl-p-D-galactopyranoside) at 70°C. After a few minutes, the clones with lacZ expression exhibited blueness.<br><br>
Preparation of a radioactive TPS1 probe<br>
Based upon a sequence comparison of the known TPS1 genes of S. cerevisiae,<br>
S. pombe, K. lactis, Candida albicans and A niger(see Figure 6), two degenerated<br>
primers could be prepared from two highly conserved regions which amplified a fragment<br>
of approximately 650 bp during PCR (consisting of 30 cycles each comprising 1 minute<br>
at 92°C, 30 seconds at 52°Ct 1 minute at 72°C) with genomic DNA from H. polymorpha.<br>
The sequences of the two primers were as follows:<br><br>
The PCR product was then loaded onto a preparative 1% (w/v) agarose gel and separated electrophoretically. The 650 bp band was cut out, extracted using the Geneclean II kit (Bio 101, Vista, USA), and marked with radioactive [a-32P]-dCTP. The Prime-It II kit was employed for this purpose, and the NucTrap columns for cleaning. This radioactive probe was used for the TPS1 screen of H. polymorpha and for the Northern blot analysis.<br><br>
Genomic DNA library of H. polymorphs:<br>
The genomic DNA library used was made available by R. Hilbrands (University of Groningen, Netherlands). Preparation of the genomic DNA library is not critical, provided the fragments are &gt; approximately 2 kb. Genomic DNA fragments of H. polymorphs 2-5 kb in length (possibly several times this length) were cloned into the BamH\ restriction site of pHRP2 (7813 bp). This plasmid (Faber et al., 1992) contains an oh (replication origin) and an ampicillin-resistance gene for replication and selection in £ coli. For transformation of H. polymorpha the HARS1 sequence (/-/. polymorpha autonomously replicating sequence) and the S. cerevisiae LEU2 gene acting as a marker which also functions in H. polymorpha are present. This library contains some 20,000 different clones.<br>
Transformation of £ coli<br>
Transformation of £ coli with the genomic DNA library was performed by electroporation (Sambrook etaL, 1989) and cells were plated out onto 50 LB+Amp (75 mg/l) plates (2,000-4,000 colonies per plate). The plates were incubated overnight at 37°C.<br>
Screening for the TPS1 gene of H. polymorphs<br>
In order to permit analysis of the DNA of the individual colonies, nitrocellulose membranes were carefully placed on the plates (according to Sambrook et a/., 1989). A thin needle was used to produce four asymmetrically distributed holes through the membrane and gel. These acted as markers in order to enable the orientation of the membranes on the plates to be reproduced at a later stage. When the membranes were drawn, the colonies present on the plate were replicated.<br>
Four plastic dishes containing 3MM absorbent paper (Whatman) were then laid out, and each dish moistened with one of four different solutions. Surplus liquid was discarded. The nitrocellulose membranes were first placed (with the colonies facing upwards) on absorbent paper soaked in 10% (w/v) SDS for 3 minutes. They were then placed in the second dish containing denaturing solution (0.5 N NaOH, 1.5 M NaCI) and held there for 5 minutes. Then they were held in turn on absorbent paper with neutralizing solution (1.5 M NaCI, 0.5M Tris-HCI, pH 7.4) and with 2 x SSC (10 x SSC 1.5 M NaCI, 170 mM sodium citrate), for 5 minutes each. In order to fix the DNA to the nitrocellulose, each<br><br>
membrane was placed between two 3MM absorbent papers and baked in a vacuum oven at 80°C for 2 hours. The membranes were then moistened for 5 minutes in 2 x SSC, before being dipped for 30 minutes in a prewash solution at 50°C (5 x SSC, 0.5% [w/v] SDS, 1 mM EDTA, pH 8.0). A wet Kleenex was used to wipe away surplus bacterial material before the membranes were placed for 2 hours in pre-hybridization solution (6x SSC, 0.25% [w/v] skim-milk powder) at 68°C. For the main hybridization process, approximately 1 x 107 cpm of radioactive TPS1 probe (refer to "preparation of a radioactive TPS1 probe") was placed in 40 ml pre-hybridization solution, and the membranes incubated in it overnight at 68°C. Following brief rinsing three times in 2 x SSC, 0.1% (w/v) SDS and washing for 1 hour at 68°C in 1 x SSC, 0.1% (w/v) SDS, the membranes were dried and exposed on BioMax film. The signals on the developed films enabled 8 positive colonies to be picked on the plates and stocks created from them. The plasmids were extracted from these colonies. PCR was employed to test whether the 650 bp fragment was in fact present.<br>
Example 2 Sequencing of the TPS1 gene of H. polymorphs<br>
Plasmid isolation<br>
For sequencing, two colonies were selected which, by means of PCR with primers from within the 650 bp fragment outwards (F4 and R4, see Table 1) and from the plasmid towards the insert (Plasm. F and Plasm. R, see Table 1) yielded the largest possible bands. Pure plasmid extracts were prepared from these two colonies (Nos. 20.1 and 21.3) by means of the Plasmid Midi Kit (Qiagen).<br>
Sequencing<br>
Sequences were produced by means of a cyclical sequencing program (PCR apparatus: Progene) and the ABI 301 automatic sequencer (Perkin Elmer). 0.5 |jl (0.5 jjg) plasmid DNA, 1 pi primer (final concentration 0.5 pM), 4 pi reaction mixture (DNA sequencing kit) and 4 pi water were used for this purpose. The sequencing program employed involved 27 cycles comprising 30 seconds at 96°C, 15 seconds at 50°C, and 4 minutes at 60°C. Upon completion of the program, 10 pi water was added to the reaction, and the DNA precipitated with sodium acetate and ethanol. The pellet was washed twice using 1 ml ice-cold 70% (v/v) ethanol. The DNA was then dried briefly and resuspended in 25 pi<br><br>
TSR (template suppressing reagent, DNA Sequencing Kit). Following incubation for two minutes, the samples were then ready for sequencing in the ABI301.<br>
The primers employed for sequencing the plasmid from clone No. 21.3 are listed in Table 1. They were prepared at the FMI on "Expedite™ Nucleic Acid Synthesis" equipment. The sequences were analyzed by means of the GCG program (Devereux et a/., 1984).<br><br><br><br>
A promoter isolated from H. polymorphs and its mode of action are described in greater detail below. This promoter, which controls the expression of TPS1, was studied by measurement of the increase in TPS1 mRNA under certain conditions. It was found that whilst this promoter expressed small quantities of TPS1 at temperatures very low for H. polymorphs, the expression increased very strongly at high temperatures, i.e. much more strongly than is the case with heat shock-induced promoters previously described (see Figure 3A, Northern blot of the heat shock). The heat-induced increase in TPS1 mRNA correlates with the increase in Tps1 protein (Figure 3B), with the increase in trehalose-6-phosphate synthase activity, and with the increase in the intracellular trehalose concentration (Figure 3C). In order to optimize the thermal influence, the promoter can for example be selectively shortened and coupled with further segments containing HSE.<br>
In addition to heat induction, a trehalose accumulation dependent upon the glucose deprivation was also observed, as anticipated owing to the close biological relationship between these two stress factors (see Figure 4A). This trehalose accumulation correlates with the increase in trehalose-6-phosphate synthase activity, the increase in TPS1 mRNA (Figure 4B), and the increase in trehalose accumulation observed with the increase in Tps1 protein during glucose deprivation (Figure 4C).<br>
The extremely high accumulation of TPS1 mRNA indicates that the TPS1 mRNA is highly stable, which makes it (and the cDNA based upon it or information obtainable from it) not only a valuable tool for isolation of the promoter, but also a particularly valuable means for protecting other organisms against a range of stress conditions, such as heat or drought. TPS1 DNA provided with suitable promoters and vectors (for example as described in WO 93/17093 and WO 96/00789) can for example be employed to protect<br><br>
plants against water deprivation, thus enabling them to be cultivated in warmer regions and regions with lower precipitation. Not only TPS1 DNA, but also DNA related to it can of course also be employed for this purpose.<br>
Example 3 Comparative expression of a bacterial lacZ gene under the control of the FMD and<br>
the TPS1 promoter<br>
Based upon the integrative H. polymorphs vector pC11 (Figure 7), two derivatives were constructed which differ only'in the respective promoter in front of the lacZ reporter gene. In the case of pC11-FMD (Figure 8), the lacZ gene is under the control of the FMD promoter, which has already been well characterized. In the case of pC11-TPS1 (Figure 9), it is under control of the heat-inducible promoter to be tested. For the purpose of this experiment, the fragment between nucleotides 228 and 792 of the sequence indicated under SEQ ID NO:1 (referred to below as the TPS1 promoter) was used as the heat-inducible promoter.<br>
H. polymorphs RB11 was transformed with pC11 -FMD and pC11-TPS1 (refer to Materials and Methods). Stable strains in which the respective plasmid was present in a genomically stable integrated state were produced separately from approximately 1,000 uracil-prototrophic cell clones for each transformation. The procedure in this case was as follows: following transformation, the cells were plated out onto plates containing selective media. Macroscopic discrete colonies were visible after three days. In both cases, 1,000 discrete separate colonies were transferred under sterile conditions to new selective plates, which were then incubated for two days at 37°C. This procedure was repeated a further two times (passaging). The cell clones were then transferred to full medium plates and incubated again for two days at 37°C (stabilization). Finally, the cell clones were transferred again to selective plates, in order to eliminate any remaining free plasmids. Following incubation of these plates for two days at 37°C, production of the strains was complete. The exact number of copies and the integration loci of the plasmids in the individual strains were not determined; according to Gatzke et a/. (1995), however, the various strains produced should differ clearly from each other in this respect.<br><br>
Since both the copy number and the genomic environment have a major influence upon the transcription rate of a gene, it had to be assumed that the individual cell clones would also differ considerably from each other with regard to their p-galactosidase activity. This was experimentally confirmed (data not shown). It was not therefore possible to compare promoter strengths directly by means of individual strains. To permit objective promoter studies despite this, 500 individual strains which had been produced separately were combined, the objective being to create representative strain mixtures with regard to the copy number and integration loci. Since the plasmids pC11-FMD and pC11-TPS1 used for strain production are identical with the exception of the respective promoter located in front of the lacZ gene, it can be assumed that they are integrated into the host genome in a homologous manner. This assumption was confirmed by the observation that various strain mixtures from the same transformation differ from each other only slightly in their p-galactosidase activity (data not shown). Determination of the p-galactosidase activity of strain mixtures produced by transformation with plasmids which are largely identical should therefore permit objective promoter comparisons in /-/. polymorphs.<br>
The lacZ activities under the control of FMD or TPS1 promoters were performed at three different temperatures in three different carbon sources (see Figure 10). To this end, the strain mixtures described above were cultivated to an OD6oo of 5 in 10 ml selective medium at the temperatures and with the carbon sources indicated, after which cell extracts were prepared, the p-galactosidase activities of which were determined by means of ONPG measurements in liquid media. The procedure was as follows: upon attainment of the desired density, the cultures were centrifuged for 10 minutes at 4°C, the cell pellets washed in 10 ml lacZ buffer (50 mM sodium phosphate buffer, pH 7; 10 mM KCI; 1 mM MgS04), resuspended in 500 nl lacZ buffer, and transferred to 1.5 ml Eppendorf tubes. Glass beads 0.45 mm in diameter were added to the suspensions (up to the meniscus of the liquid), after which the cells were disrupted in a Vibrax (Janke &amp; Kunkel; 6 minutes; 4°C; 2,200 rpm). The cell lysates were removed and centrifuged (bench centrifuge; 4°C; 10 minutes). The soluble fractions were used both for determination of the p-galactosidase activities and for measurement of the total protein content. For the p-galactosidase activity measurements, 1 ml ONPG solution (4 mg ONPG/ml lacZ buffer) was added to various dilutions of the soluble fractions, and each mixture was then transferred to a 1 cm plastic cuvette. The OD42o was then measured at 30-second intervals over a period of 3 minutes in order to permit measurement of the AE.<br><br>
To determine the total protein content of the cell extracts, 790 nl H20 was mixed with 10 \i\ of the respective soluble fraction (diluted 1:10,1:5,1:2 or undiluted, according to the protein content) and 200 ^l Bradford reagent (Biorad) was added. Following incubation for 10 minutes at room temperature, the OD490 was determined photometrically and adjusted to a control sample containing lacZ buffer instead of cell extract. The protein concentration in the cell extract was then determined from the absorption values by means of a BSA calibration curve. The specific p-galactosidase activities were calculated according to the following formula:<br><br>
The FMD promoter is known to be controlled primarily by the type of the carbon source; a temperature dependency has not yet been described (EP Patent No. 299108). This was confirmed by the measurements performed here (see Figure 10A). The p-galactosidase activities were shown to be low under glucose conditions (glucose repression), whereas substantially higher values were measured under glycerine or methanol conditions (derepression or induction). Temperature changes did not lead to dramatic changes in the measured values obtained (see Figure 10A). This was also observed in the test system employed here. The p-galactosidase activities were low at 30°C or 37°C, but rose dramatically at44°C (see Figure 10B). This temperature-dependent rise in promoter activity did not occur under methanol conditions (Figure 10B), a phenomenon which has not yet been described. Surprisingly, the highest p-galactosidase activities measured for TPS1 promoters were substantially higher than those for FMD promoters (see Figs. 10A.B).<br>
Example 4 Comparative expression of a phytase gene under the control of the FMD and the<br>
TPS1 promoter<br>
Recombinant strains were generated by transformation with the vectors pTPSIConphysMT and pFMTConphysMT in accordance with standard procedures. With<br><br>
the exception of the promoter element in the expression cassette, the two vectors employed for transformation are identical. The heat-inducible promoter contained in pTPSIConphysMT is the fragment corresponding to the sequence between nucleotides 228 and 792 in SEQ ID NO: 1, the 3' terminus of which possesses an EcoRI restriction site (referred to below as the TPS1 promoter), whereas pFMTConPhysMT contains the FMD promoter. The plasmid map and the nucleotide sequence of the vector pTPSIConphysMT are shown in Figure 11. A mutein of a phytase was used as the reporter gene.<br>
Following transformation by electroporation, recombinant H. polymorpha strains were obtained by growing the uracil-prototrophic clones produced by transformation on selective medium over at least 80 generations (Gatzke et a/.f 1995). Representative transformants of the two strain collections produced were cultivated comparatively under different conditions in 3 ml liquid cultures. Cultivation was performed in a YNB medium buffered with 0.1 M phosphate buffer pH 5.0 supplemented with 2% glucose or 5% glycerine. After 48 hours the secreted phytase was quantified in the aliquots of the culture supernatant with the aid of the method described under Materials and Methods.<br><br><br>
In this study, the TPS1 promoter was compared with the promoter most widely used to date, the FMD promoter. Use of the TPS1 promoter resulted in slightly increased expression values at 37°C when compared to the FMD promoter. An expression two to three times higher than that observed with the FMD promoter was observed at 40°C and 44°C when the TPS1 promoter was employed.<br><br>
Bibliography<br>
Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 72: 248-254.<br>
Devereux, J., Haeberli, P. and Smithies, 0. (1984) A comparative set of sequence analysis programs for the VAX. Nucl Acids Res 12: 387-395.<br>
De Virgilio, C, Burckert, N., Boiler, T. and Wiemken, A. (1991) A method to study the rapid phosphorylation-related modulation of neutral trehalase activity by temperature shifts in yeast. FEBS Left 291: 355-358.<br>
Faber, K.N., Swaving, G.J., Faber, F., Ab, G., Harder, W., Veenhuis, M. and Haima, P. (1992) Chromosomal targeting of replicating plasmids in the yeast Hansenula polymorpha. J Gen Microbiol 138: 2405-2416.<br>
Gatzke, R., Weydemann, U., Janowicz, Z. A. &amp; Hollenberg, C. P. (1995) Stable multicopy integration of vector sequences in Hansenula polymorpha. Appl. Microbiol. Biotechnol 43, 844-849.)<br>
Hottiger, T., Schmutz, P. and Wiemken, A. (1987) Heat-induced accumulation and futile cycling of trehalose in Saccharomyces cerevisiae. J Bacteriol 169: 5518-5522.<br>
Huxley, C, Green, E.D. and Dunham I. (1990) Rapid assessment of Saccharomyces cerevisiae mating type by PCR. Trends Genet 6 (8): p. 236.<br>
Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227:680-685.<br>
Levine, D.W. and Cooney, C.L. (1973) Isolation and characterization of a thermotolerant methanol-utilizing yeast. Appl Microbiol 26: 982-990.<br><br>
Parrou, J.L. and Francois, J. (1997) A simplified procedure for a rapid and reliable assay of both glycogen and trehalose in whole yeast cells. Anal Biochem 248; 186-188.<br>
Peterson, G.C. (1977) A simplification of the protein assay method of Lowry et ai which is more generally applicable. Anal Biochem 83: 346-356.<br>
Piper, P.W. (1994) Measurement of transcription. In: Molecular Genetics of yeast A practical approach, J.R. Johnston (ed.). IRL Press, Oxford.<br>
Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, Second edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor.<br>
Weydemann U, Keup P, Piontek M, Strasser AWM, Schweden J, Gellissen G, Janowicz ZA (1995) High-level secretion of hirudin by Hansenula polymorpha - authentic processing of three different preprohirudins. Appl Microbiol Biotechnol 44:844-849<br><br><br><br>
New Claim 1:<br>
1.   Nucleic acid molecule comprising a heat-inducible promoter and selected from the following nucleic acids:<br>
(a)	a nucleic acid the sequence of which comprises the promoter sequence of a Hansenula polymorphs gene coding for a protein with trehalose-6-phosphate synthase activity;<br>
(b)	a nucleic acid with the sequence indicated in SEQ ID NO:1;<br>
(c)	a nucleic acid having a sequence which exhibits at least 40% identity over a length of 300 bp with one of the sequences indicated in (a) or (b);<br>
(d)	a nucleic acid which hybridizes with the complementary strand of one of the nucleic acids indicated in (a)t (b) or (c);<br>
(e)	a derivative of one of the nucleic acids indicated in (a), (b) or (c) obtained by substitutbn, addition and/or deletion of one or more nucleotides;<br>
(f)	a fragment of one of the nucleic acids indicated in (a) to (e) which retains the function of the heat-inducible promoter;<br>
(g)	a combination of several of the nucleic acids indicated in (a) to (f), wherein the sequences of the nucleic acids may be different or the same;<br><br>
a nucleic acid molecule having a sequence complementary to the sequence of one of the nucleic acids indicated in (a) to (g),<br>
with the proviso that the nucleic acid molecule does not exhibit the promoter sequence of the trehalose-6-phosphate synthase gene of Saccharomyces cerevisiae or Schizosaccharomyces pombe.<br>
2. Nucleic acid molecule according to claim 1? characterized in that the nucleic acid indicated under c exhibits at least 60% identity with one of the sequences indicated in (a) or (b), or the complementary sequence thereof.<br><br>
3.	Nucleic acid molecule according to Claim 1, characterized in that the nucleic acid indicated under (c) exhibits at least 80% identity with one of the sequences indicated in (a) or (b), or the complementary sequence thereof.<br>
4.	Nucleic acid molecule according to Claim 1, characterized in that the nucleic acid indicated under (c) exhibits at least 90% identity with one of the sequences indicated in (a) or (b), or the complementary sequence thereof.<br>
5.	Nucleic acid molecule according to Claim 1, characterized in that the nucleic acid indicated under (c) exhibits at least 95% identity with one of the sequences indicated in (a) or (b), or the complementary sequence thereof.<br>
6.	Nucleic acid molecule according to one of the preceding claims, characterized in that it exhibits at least one heat shock element having the sequence NGAANNNNNNNGAAN (SEQ ID NO:2) or the complementary sequence thereof, wherein the nucleotides denoted by N may be A,T, C or G independent of each other.<br>
7.	Nucleic acid molecule according to claim 6, characterized in that it exhibits at least one heat shock element with the sequence NGAANNBWMNNGAAN (SEQ ID NO:3) or the complementary sequence thereof, wherein B is a G, C or T, W an A or T, and MaCorA.<br>
8.	Nucleic acid molecule according to Claim 7, characterized in that the heat shock element is selected from TGAAGCCTCTTGAAA (SEQ ID NO:4) and/or TGAATATAAAGGAAA (SEQ ID NO:5) and/or the complementary sequences thereof, wherein two or more heat shock elements, where present, may exhibit the same or different sequences.<br>
9.	Nucleic acid molecule according to claims 6, 7 or 8, characterized in that it exhibits at least two different heat shock elements.<br><br>
10.	Nucleic acid molecule according to one of the preceding claims, characterized in that it contains no STRE element having the sequence CCCCT or AGGGG.<br>
11.	Nucleic acid molecule according to Claim 1, characterized in that the fragment indicated under (f) comprises the sequence from nucleotide 228 to nucleotide 792 in SEQIDNO:1.<br>
12.	Nucleic acid molecule according to Claim 1, characterized in that the fragment indicated under (f) comprises the sequence from nucleotide 492 to nucleotide 792 in SEQIDNO:!<br>
13.	Nucleic acid molecule according to Claim 1, characterized in that the fragment indicated under (f) comprises the sequence from nucleotide 627 to nucleotide 713 in SEQIDNO:1.<br>
14.	Nucleic acid molecule according to one of the preceding claims, characterized in that it further comprises at least one nucleic acid sequence for a heterologous gene under the transcriptional control of the heat-inducible promoter.<br>
15.	Nucleic acid molecule according to one of Claims 1 to 13, characterized in that it further comprises a nucleic acid sequence under the transcriptional control of the heat-inducible promoter which is selected from the following sequences:<br>
(i)       a nucleic acid sequence which encodes a polypeptide with the amino acid sequence of the trehalose-6-phosphate synthase of Hansenula polymorphs]<br>
(ii)      a nucleic acid sequence as indicated in SEQ ID NO:6;<br>
(iii)      a nucleic acid sequence which exhibits at least 80% identity with the sequence indicated in SEQ ID NO:6;<br>
(iv)     a nucleic acid sequence which encodes a polypeptide with the amino acid sequence indicated in SEQ ID NO:7 or with a partial sequence thereof, wherein the polypeptide exhibits trehalose-6-phosphate synthase activity;<br><br>
(v)      a nucleic acid sequence which in consideration of the degeneration of the genetic code would encode a polypeptide with the amino acid sequence indicated in SEQ ID NO:7 or with a partial sequence thereof, wherein the polypeptide exhibits trehalose-6-phosphate synthase activity;<br>
(vi)     a nucleic acid sequence which encodes a polypeptide the amino acid sequence of which exhibits at least 80% identity with the amino acid sequence indicated in SEQ ID NO:7.<br>
16.	Nucleic acid molecule according to Claim 15, characterized in that the nucleic acid sequence indicated under (iii) exhibits at least 90% identity with the sequence indicated in SEQ ID NO:6.<br>
17.	Nucleic acid molecule according to Claim 15, characterized in that the nucleic acid sequence indicated under (vi) encodes a polypeptide the amino acid sequence of which exhibits at least 90% identity with the amino acid sequence indicated in SEQ ID NO:7.<br><br>
New Claim 18:<br>
18.     Host cell not occurring naturally and containing a nucleic acid molecule according to one of Claims 1 to 17, the host cell being a prokaryotic or eukaryotic cell.<br>
19.	Host cell according to claim 18, characterized in that the eukaryotic cell is fungal cell.<br>
20.	Host cell according to claim 19, characterized in that the fungal cell is a yeast cell.<br>
21.	Host cell according to claim 20, characterized in that the yeast cell is Hansenula polymorphs<br>
22.	Expression vector comprising at least  one  nucleic  acid molecule according to one of claims 1 to 13.<br><br>
23.	Expression vector comprising at least one nucleic acid molecule according to one of Claims 14 to 17.<br>
24.	Kit, comprising:<br><br>
(a)	an expression vector according to Claim 22, which is suitable for having cloned into it a nucleic acid which encodes a recombinant protein, and<br>
(b)	a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein.<br>
25.	Kit, comprising:<br>
(a)	an expression vector according to Claim 23 and<br>
(b)	a host cell which is suitable for induction of the heat-inducible promoter and for production of a protein encoded by a coding sequence under the transcriptional control of the heat-inducible promoter.<br><br>
26.	Use of a nucleic acid molecule according to one of Claims 1 to 17 or of a host cell according to one of Claims 18 to 21 or of an expression vector according to Claims 22 or 23 or of a kit according to Claim 24 or 25 for expression of a gene under the control of the heat-inducible promoter.<br>
27.	Use of a nucleic acid molecule according to one of Claims 1 to 17 or of a host cell according to one of Claims 18 to 21 or of an expression vector according to Claims 22 or 23 or of a kit according to Claim 24 or 25 for the production of one or more proteins.<br>
28.	Method for the production of one or more proteins, comprising<br>
(i)       Cloning of at least one nucleic acid which encodes a recombinant protein into an expression vector according to Claim 22, such that the nucleic acid thus cloned is under the transcriptional control of the heat-inducible promoter;<br><br>
(ii)      introduction of the expression vector obtained in (i) into a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein;<br>
(iii)      cultivation of the host cell obtained in (ii);<br>
(iv)     induction of the heat-inducible promoter by methods known per se.<br>
29. Method for the production of one or more proteins, comprising<br>
(i)       introduction of an expression vector according to Claim 23 into a host cell suitable for induction of the heat-inducible promoter and for production of the recombinant protein;<br>
(ii)      cultivation of the host cell obtained in (i);<br>
(iii)      induction of the heat-inducible promoter by methods known per se.<br><br>
30.	Nucleic acid molecule substantially as hereinbefore described<br>
31.	Host cell containing a nucleic acid molecule substantially as hereinbefore described<br>
32.	Kit substantially as hereinbefore described.<br>
33.	Method for the production of one or more proteins substantially as<br>
hereinbefore described.<br><br><br>
Summary<br>
Heat-inducible promoter<br>
The invention relates to nucleic acid molecules comprising a heat-inducible promoter and to expression vectors and host cells containing at least one nucleic acid molecule according to the invention. The present invention further relates to kits and methods for producing one or more proteins using the nucleic acid molecules according to the invention, and to various uses of the same. The object of the invention is to provide a promoter the heat-inducible characteristic of which is as selective as possible, in particular a promoter which is active in yeasts and which is suitable for protein expression at high temperatures. This object is fulfilled by a nucleic acid molecule comprising a heat-inducible promoter and which is selected from the following nucleic acids: (a) a nucleic acid the sequence of which comprises the promoter sequence of a Hansenula polymorpha gene coding for a protein with trehalose-6-phosphate synthase activity; (b) a nucleic acid with the sequence indicated in SEQ ID NO:1; (c) a nucleic acid having a sequence which exhibits at least 40% identity over a length of 300 bp with one of the sequences indicated in (a) or (b); (d) a nucleic acid which hybridizes with the complementary strand of one of the nucleic acids indicated in (a), (b) or (c); (e) a derivative of one of the nucleic acids indicated in (a), (b) or (c) obtained by substitution, addition and/or deletion of one or more nucleotides; (f) a fragment of one of the nucleic acids indicated in (a) to (e) which retains the function of the heat-inducible promoter; (g) a combination of several of the nucleic acids indicated in (a) to (f), wherein the sequences of the nucleic acids may be different or the same; or of a nucleic acid molecule having a sequence complementary to the sequence of one of the nucleic acids indicated in (a) to (g).<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1jbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1jbGFpbXMgb3JpZ2luYWwucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-description complete duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-description complete original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA4MC1pbi1wY3QtMjAwMS0xMDgwLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">1080-in-pct-2001-1080-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="207711-a-projectile-for-piercing-armor-and-a-method-for-piercing-armor-of-a-target.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="207713-a-sealant-containing-tire.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>207712</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1080/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Nov-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Jun-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Jul-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. RHEIN BIOTECH GESELLSCHAFT FÜR NEUE BIOTECHNOLOGISCHE PROZESSE UND PRODUKTE MBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Eichsfelder Strasse 11 D-40595 Düsseldorf</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ROMANO, Ivano</td>
											<td>Wasenhaldenstrasse 12 D-5080 Laufenburg</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/39</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2000/001144</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-02-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>279/99</td>
									<td>1999-02-11</td>
								    <td>Switzerland</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/207712-a-method-for-production-of-one-or-more-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:33:20 GMT -->
</html>
