<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208063-a-method-for-preparing-a-yeast-strain by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:24:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208063:A METHOD FOR PREPARING A YEAST STRAIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR PREPARING A YEAST STRAIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for preparing a yeast strain stably transformed with an extrachromosomal yeast expression plasmid, said method comprising: (i) providing an isolated first yeast expression plasmid comprising a functional antibiotic resistance marker gene and a DNA sequence .encoding a desired protein, such as herein described; (ii) modifying the antibiotic resistance marker gene in vitro to produce a ~&#x27;l,t second yeast expression plasmid, wherein said second plasmid</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Method of making Proteins in transformed Yeast Ceils<br>
FIELD OF INVENTION<br>
The present invention relates to expression of proteins in transformed yeast cells, DNA construct and vectors for use in such process and yeast cells transformed with the vectors.<br>
BACKGROUND OF THE INVENTION<br>
It is well known to use transformed yeast strains for the expression of proteins see for example European patent applications Nos. 0088632A, 0116201A, 0123294A, 0123544A, 0163529A, 0123289A , 0100561A, 0189998A and 0195986A, PCT patent applications Nos. WO 95/01421, 95/02059 and WO 90/10075, and US Patent No. 4,546,082.<br>
It is a common feature of the above methods that the yeast production plasmid contains a gene for an antibiotic marker. Such marker gene stems from the initial cloning steps in E. coli where it was used to screen for transformed cells or to maintain plasmids used as vectors. The antibiotic marker genes are not believed to have any adverse impact on the culturing of the transformed yeast cell and it has therefore been common practice not to take any steps to delete such DNA. In addition, characterization of the plasmid construct is usually done by isolation of plasmids from the transformed yeast cells and transformation of the isolated plasmid into E. coli followed by antibiotic selection. It has thus been convenient for practical purposes to retain the antibiotic resistance marker gene.<br>
Although both research laboratories and industrial production plants are controlled by very severe safety regulations there is always a small risk that a few cells by accident will be released to the environment. Due to their highly sophisticated nature such ge-<br><br>
netically engineered microorganisms will only survive for a very short period and the risk of harming the environment is extremely low. This is of course the reason why such transformed microorganisms have been approved for use in both research and in large scale operations.<br>
Even if the cells die quickly the plasmids containing the antibiotic resistant gene may still accidentally be disposed to the environment and there is a theoretical risk of introduction of resistance to antibiotics in bacteria if the plasmid is taken up spontaneously.<br>
Antibiotics are of great importance for treatment of human and animal bacterial infections. Any risk of a potential environmental contamination with a gene that confers resistance to antibiotics should be minimized, if possible.<br>
There is therefore a need to develop even more safer methods than the methods used up to this day and it is the object of the present invention to provide such improved methods.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to a method for expressing heterologous proteins or polypeptides in yeast wherein the yeast transformant strain used for production contains an expression vector in which an antibiotic marker gene used in the initial cloning steps has been made non functional by in vitro modification before transformation of the yeast host. The present invention also relates to DNA sequences and expression vectors for use in such method and to transformed yeast cells.<br>
According to one aspect the present invention is related to a recombinant yeast expression vector being unable to confer antibiotic resistance to bacteria cells and comprising a gene coding for a heterologous gene and an antibiotic resistance marker gene which has been made non functional by in vitro modification.<br><br>
According to a further aspect the present invention is related to a method for making a desired polypeptide or protein said method comprises culturing a yeast strain comprising a vector being unable to confer antibiotic resistance to bacteria cells and comprising a gene coding for a heterologous gene and an antibiotic resistance marker gene, which marker gene has been made non functional by in vitro modification before transformation of the yeast host, and isolating the desired product from the culture medium.<br>
The method according to the invention will typically comprise culturing a yeast strain containing a yeast expression plasmid in which plasmid a functional antibiotic marker gene used for initial cloning steps in bacteria has been made non-functional by in vitro deletion of part of the marker gene or the whole marker gene before insertion into the yeast host to be used for expression and secretion of the desired polypeptide or protein.<br>
The deletion of the antibiotic marker gene is preferably made by insertion of suitable restriction cleavage sites on each side of the antibiotic resistance marker gene whereupon the marker gene is deleted by in vitro treatment with suitable restriction enzymes.<br>
The present invention is also related to transformed yeast strains comprising a vector being unable to confer antibiotic resistance to bacteria cells and comprising a gene coding for a heterologous gene and an antibiotic resistance marker gene which has been made non functional by in vitro modification before transformation of the yeast host.<br>
The yeast strain is preferable a Saccharomyces strain, and in particular a Sac-charomyces cerevisiae strain.<br><br>
As used herein the expression "antibiotic marker gene" or "antibiotic resistance marker gene" means a gene that allows phenotypic selection of transformed bacterial cells and plasmid amplification.<br>
The most commonly used antibiotic resistance marker genes in E. coli are the am-picillin (AMP), chloramphenicol, neomycin, kanamycin and tetracylin resistance conferring marker genes.<br>
As used herein the expression "non functional marker gene" means that the marker gene has been either deleted or made non functional by deletion of part of the gene. It is preferred that the gene is completely deleted.<br>
As used herein "in vitro modification" means modification steps performed on the vector outside the cell environment.<br>
As used herein "unable to confer antibiotic resistance to bacteria cells" means that the antibiotic resistance genes are non functional in any organism due to the described gene manipulation.<br>
As used herein the "yeast host" means a yeast organism to be transformed or trans-fected with the expression plasmid or vector.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The present invention is further illustrated with reference to the appended drawings, wherein<br>
Fig. 1 shows the expression plasmid pAK729 which contains a gene expressing an insulin precursor under expression control of a TPI promoter and a TPI terminator sequence from S. cerevisiae and a signal leader sequence consisting of the YAP3 signal peptide and a synthetic LA19 leader peptide. The construction of pAK729 is<br><br>
described in WO 97/22706. The plasmid also contains the AMP-R sequence from pBR322 /pUC13 including the ampicillin resistance gene and an origin of DNA replication in E. coli;<br>
Fig. 2 shows the plasmid map of the pAK729.5 plasmid used for generation of the NN729.5 strain lacking the AMP gene prior to the deletion of the Amp gene;<br>
Fig. 3 shows the plasmid map of the pAK729.6 plasmid used for generation of the NN729.6 strain lacking the AMP gene prior to the deletion of the Amp gene;<br>
Fig. 4 shows the plasmid map of the pAK729.6-Aamp plasmid in which the AMP gene has been deleted;<br>
Fig. 5 shows the plasmid map of the pAK729.7 plasmid used for generation of the NN729.7 strain lacking the AMP gene prior to the deletion of the Amp gene; and<br>
Fig. 6 shows the plasmid map of the pKV228 plasmid modified by replacing the<br>
EcoRI (940) - Xbal (1403) coding sequence in pAK729 (Fig.1) with a MFaipha*-Arg34GLP-1 ^.3^ coding sequence.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The in vitro deletion of the antibiotic resistance marker gene is done either by use of available restriction sites or by introduction of suitable restriction sites byuse of PCR, site specific mutagenesis or other well know techniques for manipulation of DNA sequences followed by treatment with the suitable restriction enzymes.<br>
Four modified NN729 strains were constructed to evaluate whether various deletions in the plasmid might influence the insulin precursor fermentation yield or strain stability during long-term fermentation (Table I). The strains were compared with the origi-<br><br>
nal NN729 strain with respect to fermentation yield and fermentation stability (Table fl). In addition three modified yeast strain producing a GLP-1 variant Arg34GLP-l) were constructed to evaluate whether various deletions in plasmid pKV228 containing the AMP gene might influence the Arg34GLP-l  fermentation yield (Table III).<br>
Plasmids and strains wherein the AMP gene and possibly surrounding sequences have been deleted are all denoted "AAMP".<br>
The modified yeast strains were prepared by transformation of the pAK729 or pKV228 modified plasmids in which the AMP marker gene and possible other DNA sequences from the original plasmid had been deleted into S. cerevisiae strain MT663 (E2-7B XE11-36 a/a, AtpiAtpi.pep 4-3/pep 4-3) or ME1719 (MATa/aAyap3::ura3/Ayap3::URA3pep4-3/pep4-3Atpi::LEU2/Atpi::LEU2 eu2/leu2 Aura3/ Aura3).<br>
The modified plasmids were prepared by used of suitable restriction enzyme sites already present in the plasmid or by insertion of suitable restriction enzyme sites in such a way that the AMP gene can be deleted. The modified plasmids can be manipulated in vitro before transformation into S. cerevisiae (strain MT663) in such a way that the AMP gene is deleted or made non-functional and consequently the resulting yeast strain lacks the AMP gene. Thus, potential risk for environmental contamination with the AMP gene during disposal of the yeast cells is eliminated.<br>
The modified pAK729 or pKV228 plasmids were digested with the appropriate restriction enzymes, subjected to agarose electrophoresis, isolated, re-ligated and subsequently transformed into competent MT663 and competent ME1719 WO98/01535 S. cerevisiae cells respectively.<br><br>
The protein or polypeptide produced by the method of the invention may be any heterologous protein or polypeptide which may advantageously be produced in a yeast cell. Examples of such proteins are aprotinin, tissue factor pathway inhibitor or other protease inhibitors, insulin, insulin precursors or insulin analogues, insulin-like growth factor I or II, human or bovine growth hormone, interleukin, tissue plasminogen activator, transforming growth factor a or b, glucagon, glucagon-like peptide 1 (GLP-1), glucagon-like peptide 2 (GLP-2), GRPP, Factor VII, Factor VIII, Factor XIII, platelet-derived growth factor, and enzymes, such as lipases.<br>
By "a precursor of insulin" or "a precursor of an insulin analogue" is to be understood a single-chain polypeptide, including proinsulin, which by one or more subsequent chemical and/or enzymatic processes can be converted to a two-chain insulin or insulin analogue molecule having the correct establishment of the three disulphide bridges as found in natural human insulin. The insulin precursors will typically contain a modified C-peptide bridging the A- and B chain of insulin. In addition the preferred insulin precursors will lack the B(30) amino acid residue. Most preferred insulin precursors are those described in e.g. EP 163529 and PCT application Nos. 95/00550 and 95/07931.Examples of insulins are human insulin, preferably des(B30) human insulin, and porcine insulin. Preferred insulin analogues are such wherein one or more of the natural amino acid residues, preferably one, two, or three have been substituted by another codable amino acid residue. Thus in position A21 a parent insulin may instead of Asn have an amino acid residue selected from the group comprising Ala, Gin, Glu, Giy, His, lie, Leu, Met, Ser, Thr, Trp, Tyr or Val, in particular an amino acid residue selected from the group comprising Gly, Ala, Ser, and Thr. Likewise, in position B28 a parent insulin may instead of Pro have an amino acid residue selected from the group comprising Asp, Lys etc., and in position B29 a parent insulin may instead of Lys have the amino acid Pro.<br><br>
The expression "a codable amino acid residue" as used herein designates an amino acid residue which can be coded for by the genetic code, i. e. a triplet ("codon") of nucleotides.<br>
The DNA constructs used may be prepared synthetically by established standard methods, e.g. the phosphoamidite method described by S.L. Beaucage and M.H. Caruthers, Tetrahedron Letters 22. 1981, pp. 1859-1869, or the method described by Matthes et al., EMBQ Journal 3. 1984, pp. 801-805. According to the phosphoamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, duplexed and ligated to form the synthetic DNA construct. A currently preferred way of preparing the DNA construct is by polymerase chain reaction (PCR), e.g. as described in Sambrook et al., Molecular Cloning: A Laboratory Manual. Cold Spring Harbor, NY, 1989).<br>
The DNA coding for the desired protein may also be of genomic or cDNA origin, for instance obtained by preparing a genomic or cDNA library and screening for DNA sequences coding for all or part of the polypeptide or the invention by hybridization using synthetic oligonucleotide probes in accordance with standard techniques (cf. Sambrook et al., Molecular Cloning: A Laboratory Manual. Cold Spring Harbor, 1989).<br>
Finally, the DNA encoding the desired protein may be of mixed synthetic and genomic, mixed synthetic and cDNA or mixed genomic and cDNA origin prepared by annealing fragments of synthetic, genomic or cDNA origin (as appropriate), the fragments corresponding to various parts of the entire DNA construct, in accordance with standard techniques.<br>
The recombinant expression vector may be an autonomously replicating vector, i.e., a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g., a plasmid, an extrachromosomal element,<br><br>
a minichromosome, or an artificial chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which it has been integrated. The vector system may be a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be introduced into the genome of the host cell, or a transposon.<br>
The recombinant expression vector may will contain a DNA sequence encoding the desired protein or polypeptide operably connected to a suitable promoter sequence. The promoter may be any DNA sequence which shows transcriptional activity in yeast and may be derived from genes encoding proteins either homologous or heterologous to yeast. The promoter is preferably derived from a gene encoding a protein homologous to yeast. Examples of suitable promoters are the Saccharomyces cerevisiae Ma1( TPI, ADH or PGK promoters.<br>
The DNA sequence encoding the desired protein or polypeptide may also be operably connected to a suitable terminator, e.g. the TPI terminator (cf. T. Alber and G. Kawasaki, J. Mol. Appl. Genet. 1. 1982, pp. 419-434).<br>
The recombinant expression vector of the invention will also comprise a DNA sequence enabling the vector to replicate in yeast. Examples of such sequences are the yeast plasmid 2μ replication genes REP 1-3 and origin of replication. The vector may also comprise a selectable marker, e.g. the Schizosaccharomyces pompe TPI gene as described by P.R. Russell, Gene 40. 1985, pp. 125-130.<br>
Finally, the expression vector will preferably contain a signal/leader sequence to ensure secretion of the desired protein or polypeptide to the culture medium. A signal sequence is a DNA sequence that encodes a polypeptide (a "secretory peptide") that, as a component of a larger polypeptide, directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger polypeptide is com-<br><br>
manly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
The secretory signal sequence may encode any signal peptide which ensures efficient direction of the expressed polypeptide into the secretory pathway of the cell. The signal peptide may be naturally occurring signal peptide, or a functional part thereof, or it may be a synthetic peptide. Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae a-factor and Saccharo-myces cerevisiae invertase, the signal peptide of mouse salivary amylase (cf. O. Ha-genbuchle et al.,Nature 289. 1981, pp. 643-646), a modified carboxypeptidase signal peptide (cf. LA. Vails et al., Cell 48, 1987, pp. 887-897), the yeast BAR1 signal peptide (cf. WO 87/02670), or the yeast aspartic protease 3 (YAP3) signal peptide (cf. M. Egel-Mitani et al.. Yeast 6. 1990, pp. 127-137).<br>
For efficient secretion in yeast, a sequence encoding a leader peptide may also be inserted downstream of the signal sequence and upstream of the DNA sequence encoding the polypeptide. The function of the leader peptide is to allow the expressed polypeptide to be directed from the endoplasmic reticulum to the Golgi apparatus and further to a secretory vesicle for secretion into the culture medium (i.e. exportation of the polypeptide across the cell wall or at least through the cellular membrane into the periplasmic space of the yeast cell). The leader peptide may be the yeast a-factor leader (the use of which is described in e.g. US 4,546,082, EP 16 201, EP 123 294, EP 123 544 and EP 163 529). Alternatively, the leader peptide may be a synthetic leader peptide that is a leader peptide not found in nature. Synthetic leader peptides may be constructed as described in WO 89/02463 or WO 92/11378 and by Kjeldsen et al in "Protein Expression and Purification 9, 331-336 (1997).<br>
The expression "leader peptide" is understood to indicate a peptide in the form of a propeptide sequence whose function is to allow the heterologous protein to be secreted to be directed from the endoplasmatic reticulum to the Golgi apparatus and<br><br>
further to a secretory vesicle for secretion into the medium, (i.e. exportation of the expressed protein or polypeptide across the cellular membrane and cell wall, if present, or at least through the cellular membrane into the periplasmic space of a cell having a cell wall).<br>
The procedures used to ligate the DNA sequences coding for the desired protein or polypeptide, the promoter and the terminator, respectively, and to insert them into suitable yeast vectors containing the informaffon necessary for yeast replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit.). It will be understood that the vector may be constructed either by first preparing a DNA construct containing the entire DNA sequence coding for the polypeptide of the invention and subsequently inserting this fragment into a suitable expression vector, or by sequentially inserting DNA fragments containing genetic information for the individual elements (such as the signal, leader or heterologous protein) followed by ligation.<br>
The yeast organism used in the process of the invention may be any suitable yeast organism which, on cultivation, produces satisfactory amounts of the desired protein or polypeptide. Examples of suitable yeast organisms may be strains selected from the yeast species Saccharomyces cerevisiae, Saccharomyces kluyveri, Schizosac-charomyces pombe, Saccharomyces uvarum, Kluyveromyces lactis, Hansenula polymorpha, Pichia pastoris, Pichia methanolica, Pichia kluyveri, Yarrowia lipolytica, Candida sp., Candida utilis, Candida cacaoi, Geotrichum spM and Geotrichum fer-mentans, preferably the yeast species Saccharomyces cerevisiae.<br>
The transformation of the yeast cells may for instance be effected by protoplast formation followed by transformation in a manner known per se. The medium used to cultivate the cells may be any conventional medium suitable for growing yeast organisms. The secreted heterologous protein, a significant proportion of which will be present in the medium in correctly processed form, may be recovered from the me-<br><br>
dium by conventional procedures including separating the yeast cells from the medium by centrifugation or filtration, precipitating the proteinaceous components of the supernatant or filtrate by means of a salt, e.g. ammonium sulphate, followed by purification by a variety of chromatographic procedures, e.g. ion exchange chromatography, affinity chromatography, or the like. When the protein is secreted to the perip-lasmic space, the cells are disrupted enzymatically or mechanically.<br>
The desired protein or polypeptide may be expressed and secreted as an N-terminal extended fusion protein as described in WO 97/22706. The N-terminal extension may then be removed from the recovered protein in vitro by either chemical or enzymatic cleavage as well known in the art. It is preferred to conduct the cleavage by use of an enzyme. Examples of such enzymes are trypsin or Achromobacter iyticus protease I.<br>
The present invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention as claimed.<br>
Example 1<br>
Yeast Plasmid pAK729 constructed for expression of an insulin precursor (an N-terminal extended B(1-29)-Ala-Ala-Lys-A(1-21) insulin precursor, see WO 97/22706) contains two ApaLI enzyme restriction sites ApaLI (4477) and ApaLI (5723) (see Fig. 1). These restriction sites are situated on each side of the AMP marker gene. Removal of the 1246 nucleotides between the two ApaLI sites in pAK729 will remove the AMP marker gene and some additional E. coli derived plasmid DNA.<br>
The pAK729 plasmid was digested with ApaLI restriction enzyme, subjected to agarose electrophoresis, isolated, re-ligated and subsequently transformed into competent S. cerevisiae cells (MT663, see EP B0163529) to give transformed yeast strain NN729.1-AAMP. The modified expression plasmid was re-isolated from the yeast strain NN729.1-AAMP and DNA sequences were verified after PCR generation fol-<br><br>
lowed by subcioning of the DNA region featuring the deletion. Likewise, the DNA sequences encoding the insulin precursor were verified on plasmid DNA re-isolated from yeast strain NN729.1-AAMP.<br>
Yeast strain NN729.1-AAMP was cultured in YPD medium for at 30°C for 72 hours. The fermentation yield of the insulin precursor was determined by RP-HPLC.<br>
Example 2<br>
Enzyme restriction sites, Xhol (5676) and Xhol (5720) were introduced in the pAK729 plasmid by PCR. Selected DNA sequences of the resulting pAK729.5 plasmid were subsequently verified. The restriction plasmid map of pAK729.5 is shown in Fig. 2. The DNA fragment between the restriction enzyme sites Xhol (5676) and Xhol (5720) can be deleted from plasmid pAK729.5 deleting 44 nucleotides localized within the AMP gene.<br>
Plasmid pAK729.5 was digested with Xhol restriction enzymes, subjected to agarose electrophoresis, isolated, re-ligated and subsequently transformed into competent MT663 S. cerevisiae cells giving the yeast transformant NN729.5-AAMP. The modified expression plasmid was re-isolated from the yeast strain NN729.5-AAMP and DNA sequences were verified after PCR generation followed by subcioning of the DNA region featuring the deletion. Likewise, the DNA sequences encoding the insulin precursor were verified on plasmid DNA re-isolated from yeast strain NN729.5-AAMP. The 44 nucleotide deletion in pAK729.5-AAMP turned out to be as efficient as a complete deletion of the AMP gene with respect to destroying p-lactamase activity.<br>
Yeast strain NN729.5-AAMP was cultured in YPD medium for at 30°C for 72 hours. The fermentation yield of the insulin precursor was determined by RP-HPLC.<br><br>
Example 3<br>
Enzyme restriction enzyme site, Aatll (4982), was introduced into the the pAK729 plasmid by PCR. Selected DNA sequences of the resulting pAK729.6 plasmid were subsequently verified. The restriction plasmid map of pAK729.6 is shown in Fig. 3. In pAK729.6 the DNA fragment between the restriction enzyme sites Aatll (4982) and Aatll (5978) can be deleted, removing 996 nucleotides from the plasmid. This will remove all of the AMP gene and the promoter.<br>
The pAK729.6 plasmid was digested with the DNA restriction enzyme Aatll, subjected to agarose electrophoresis, isolated, re-ligated and subsequently transformed into competent MT663 S. cerevisiae cells. The modified expression plasmid was re-isolated from the yeast strain NN729.6-AAMP and DNA sequences were verified after PCR generation followed by subcloning of the DNA region featuring the deletion. Likewise, the DNA sequences encoding the insulin precursor were verified on plasmid DNA re-isolated from yeast strain NN729.6-AAMP. Plasmid pAK729.6-AAMP Lacking the AMP gene is shown in Fig. 4.<br>
Yeast strain NN729.6-AAMP was cultured in YPD medium for at 30°C for 72 hours. The fermentation yield of the insulin precursor was determined by RP-HPLC.<br>
Example 4<br>
The new enzyme restriction enzyme site, Aatll (3801), in the pAK729.7 plasmid was introduced into the original pAK729 plasmid by PCR. Selected DNA sequences of pAK729.7 plasmid were subsequently verified. In pAK729.7 the DNA fragment between the restriction enzyme sites Aatll (3801) and Aatll (5978) can be deleted, removing 2177 nucleotides from the expression plasmid. The pAK729.7 plasmid was designed so both the AMP gene and the E. coli origin of replication can be deleted. The restriction plasmid map of pAK729.7 is shown in Fig. 5.<br><br>
The pAK729.7 plasmid was digested with the DNA restriction enzyme Aattt, subjected to agarose electrophoresis, isolated, re-ligated and subsequently transformed into competent MT663 S. cerevisiae cells. The modified expression plasmid was re-isolated from the yeast strain NN729.7-AAMP and DNA sequences were verified after PCR generation followed by subcloning of the DNA region featuring the deletion. Likewise, the DNA sequences encoding the insulin precursor were verified on plasmid DNA re-isolated from yeast strain NN729.7-AAMP. Yeast strain NN729.7-AAMP was cultured in YPD medium for at 30°C for 72 hours. The fermentation yield of the insulin precursor was determined by RP-HPLC.<br><br>
The new NN729-AAMP strains were compared with the original NN729 strain with respect to fermentation yield of the insulin precursor (Table II).<br><br><br>
It appears from the above that yeast strains comprising an expression plasmid with a partly or fully deleted AMP marker gene express 10 - 20% more of the insulin precursor compared to the original yeast strain comprising an expression plasmid containing the AMP gene.<br>
Example 5<br>
Arg34GLP-l expression in yeast using plasmids with non-functionai or deleted AMP resistance gene<br>
The EcoRI (940) - Xbal (1403) sequence of the pAK729 constructs encoding LA19 X5 MI3 illustrated in Figures 1- 5 were replaced with an MFalpha Arg34 GLP-l coding sequence for the present example (Fig. 6). A modification of the MFa1 pre-pro leader peptide (Kurjan &amp; Herskowitz, Cell 30. 1982. pp. 933.) in which Leu in position 82 and Asp in position 83 have been substituted with Met and Ala respectively introducing the A/col cleavage site in the DNA sequence was applied in this constructs. The leader sequence was designated MFa1* (Kjeldsen T. et. ai 1996). The MFa1 signal MFa1* leader peptide sequence includes the dibasic Kex2p recognition motif (Lys-Arg) separating the leader from the coding sequence for Arg34 GLP-I^j. The peptide Arg^GLP-l^j is a human GLP-1{7.37) variant (S. Mojsov, et ai., J Biol.<br><br>
Chem. 261. 1986, pp. 11880-11889) wherein the natural amino acid residue in position 34 is substituted with an Arg residue.<br>
Three Arg34GLP-l expression plasmids were constructed with AMP resistance gene disruptions as described for NN729.1 (Example 1), NN729.5 (Example 2) and NN729.6 (Example 3) and subsequently transformed into competent ME1719 (see WO98/01535) S. cerevisiae cells giving the yeast transformants YES2076, YES2079 and YES2085, respectively.<br>
The host strain, which was been used to express Arg34GLP-1(7-37), is a diploid strain and has phenotypes which lack two asparatyl proteases, i.e., (1) yeast aspartyl protease 3 (YAP3) which cleaves C-terminal side of mono- or dibasic amino acid residues (Egel-Mitani, et al., YEAST 6: 127-137, 1990) and (2) vacuolar protease A responsible for activation of other proteases such as protease B, carboxypeptidase Y, aminopeptidase I, RNase, alkaline phosphatase, acid trehaiase and exopolyphos-phatase. Moreover the those phosphate isomerase gene (TPI) has been disrupted, which phenotype makes it possible to utilize glucose in transformants grown on glucose containing medium. The genetic background of ME1719 is MATa/a Dyap3::ura3/Dyap3::URA3 pep4-3/pep4-3 tpi::LEU2/Dtpi::LEU2 Ieu2/leu2 Dura3/ Dura3.<br>
The modified expression plasmids pKV301, pKV307 and pKV304 were re-isolated from the yeast strains and DNA sequences were verified after PCR generation followed by subcloning of the DNA region featuring the deletion. Likewise, the DNA sequences encoding Arg34GLP-l, was verified on plasmid DNA re-isolated from the yeast strains. Table III shows a comparison between modified and non-modified strains.<br><br><br><br><br><br><br>
CLAIMS<br>
1.	Recombinant yeast expression vector being unable to confer antibiotic resistance<br>
to bacteria cells said vector comprising a gene coding for a heterologous gene<br>
and a antibiotic resistance marker gene which has been made non functional by in<br>
vitro modification.<br>
2.	Recombinant yeast expression vector according to claim 1 wherein the antibiotic<br>
marker gene is the AMP gene.<br>
3.	Method for making a desired polypeptide or protein comprising culturing a yeast strain comprising a vector.according to claim 1-2 under suitable conditions and isolating the desired product from the culture medium.<br>
4.	Method for making a desired polypeptide or protein comprising culturing under suitable conditions a yeast strain containing a yeast expression plasmid in which plasmid a functional antibiotic marker gene used for initial cloning steps in bacteria has been made non-functional by in vitro deletion of part of the marker gene or the whole marker gene before insertion into the yeast host.<br>
5.	Method according to claim 4, comprising inserting suitable restriction sites around the antibiotic resistance marker gene and deleting the marker gene by in vitro treatment with suitable restriction enzymes.<br>
6.	Method according to claim 3-5, wherein the yeast strain is a Saccharomyces strain, preferably a Saccharomyces cerevisiae strain.<br>
7.	Transformed yeast cell comprising a vector according to claim 1- 2.<br><br>
•8. Transformed yeast cell according to claim 7 being a Saccharomyces strain, preferable a Saccharomyces cerevisiae strain.<br><br>
* 9.Recombinant  yeast  expression   vector   substantially   as   hereinbefore described with reference to the accompanying drawings.<br>
10. Method for making a desired polypeptide or protein substantially as hereinbefore described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1jbGFpbXMgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1jbGFpbXMgb3JpZ2luYWwucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-claims original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-description complete duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBvcmlnaW5hbC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-description complete original.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDU4LWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-058-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="208062-a-device-for-delaying-the-pyloric-passage-of-orally-assimilable-pharmaceutical-forms.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208064-system-and-method-for-wireless-communication-supporting-link-adaptation-and-incremental-redundancy.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208063</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/58/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Jul-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Jan-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVO NORDISK A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NOVO ALLE, DK-2880 BAGSVAERD.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAD KNUD</td>
											<td>HVIDTJOERNEVEJ 22, DK-2720 VANLOSE.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KJELDSEN THOMAS</td>
											<td>KVAEDEVEJ 32, DK-2830 VIRUM.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/81</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK99/00380</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-07-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 1998 00945</td>
									<td>1998-07-16</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208063-a-method-for-preparing-a-yeast-strain by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:24:28 GMT -->
</html>
