<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208188-a-process-for-producing-carboxylic-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:03:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208188:&quot; A PROCESS FOR PRODUCING CARBOXYLIC ACID&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot; A PROCESS FOR PRODUCING CARBOXYLIC ACID&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a novel Rhodococcus bacterium and to a process of hydrolyzing a cyano group of a nitrile compound using a novel Rhodococcus bacterium to produce the corresponding carboxylic acid. The present invention also relates to a process of producing carboxylic acids, in particular cyano carboxylic acids using a transformant transformed with a plasmid containing a nitrilase gene, a nitrile hydratase gene and an amidase gene derived from Rhodococcus bacteria capable of exhibiting particularly excellent position selectivity for the cyano group of aromatic polynitrile compounds, to such a transformant, such a plasmid, to such genes, to a process of producing an enzyme using the transformant, and to enzymes obtained by the process. The carboxylic acids, in particular cyano carboxylic acids obtained by the present invention are useful as starting materials for the synthesis of drugs, agrochemicals, dyestuff and other chemicals</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> A      PROCESS FOR PRODUCING CARBOXYLIC ACIDS<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
This application is based on the provisions of 35 U.S.C. Article 111(a)to claim the benefit of filing dates of U.S. provisional application Serial No. 60/183,754 filed on February 22, 2000 and U.S. provisional application Serial No.60/183,821 filed February 22, 2000 under the provisions of 35 U.S.C. 111(b), pursuant to the provision of 35 U.S.C. Article 119(e) (i).<br>
TECHNICAL FIELD<br>
The present invention relates to a novel Rhodococcus bacterium and to a process for hydrolyzing a cyano group of a nitrile compound using a novel Rhodococcus bacterium to produce the corresponding carboxylic acid.    The present invention also relates to a process of producing carboxylic acids, in particular cyano carboxylic acids using a transformant transformed with a plasmid containing a nitrilase gene, a nitrile hydratase gene, and an amidase gene derived from a Rhodococcus bacterium capable of exhibiting particularly excellent position selectivity for the cyano group of aromatic polynitrile compounds, to such a transformant, such a plasmid, to such genes, to a process of producing an enzyme using the transformant, and to enzymes obtained by the process.    The carboxylic acids, in particular cyano carboxylic acids obtained by the present invention are useful as starting materials for the synthesis<br><br><br>
2<br>
of drugs, agrochemicals, dyestuff and other chemicals.<br>
BACKGROUND ART<br>
Many studies have been made on the reaction of hydrolyzing a cyano group of a nitrile compound to obtain the corresponding carboxyl acids, because this is a simple and easy process for obtaining carboxylic acids.<br>
With respect to the bioreaction of hydrolyzing only a part of the cyano groups of a polynitrile compound having a plurality of cyano groups in one molecule to obtain the corresponding cyano carboxylic acid, in particular, with respect to the process for obtaining aromatic cyano carboxylic acids by selectively hydrolyzing only a specific cyano group of an aromatic polynitrile compound, many reports have been published on the reaction utilizing the specificity to the reaction of the microorganism.    For example, U.S. Patent 4,629,700 discloses a process of producing cyanobenzoic acids from phthalonitriles using a Rhodococcus bacterium. Also, for example, European Patent 178,106 discloses a process for producing cyano carboxylic acids and cyano carboxylic acid amides by the selective hydrolysis of a cyano group from a polynitrile compound using four genera of gram positive bacteria including the genus Rhodococcus.<br>
Selective hydrolysis reactions of a cyano group in a chemical synthesis are generally not suitable for the practical use because in order to perform the reaction, a complicated procedure, such as protection of a specific cyano group, is necessary.<br>
Bioreactions are generally admitted to have high selectivity.    However, when strictly inspected, they are in many cases accompanied by production of impurities due to side reaction.    For example, according to the above-described process for producing a cyano benzoic acid from a phthalonitrile using a Rhodococcus bacterium, the<br><br>
3<br>
selectivity is not 100%, but the reaction is accompanied by from 1.0% to a few % of byproducts originating in the phthalonitrile.    From the standpoint of the conversion from a starting material, this may be said to be an excellent process.    However, in the synthesis of medicaments or fine chemicals, the behaviors of a slight amount of byproducts greatly affect the capability or safety of a substance synthesized using a starting material containing the by-products.    Therefore, the above-described selectivity is not sufficiently high for the starting material in this field.<br>
In order to elevate the purity of the products, a method of obtaining a product and thereafter further purifying it may be considered.    However, for example, various by-products produced in the process of biologically producing a cyano carboxylic acid from an aromatic polynitrile are very close to each other in the physical properties such as boiling point and hydrophobicity, and complete separation thereof cannot be attained by commonly used purification methods such as distillation, extraction, and salting out.<br>
As such, in conventional processes for producing carboxylic acids by a hydrolysis reaction of a nitrile compound using microorganisms, the selectivity of the hydrolysis reaction itself is not high and the production of by-products is not sufficiently reduced.<br>
An alternative method for producing carboxylic acids by the hydrolysis of a nitrile compound includes enzymatic reaction methods using nitrilase, or nitrile hydratase and amidase.<br>
The nitrilase is an enzyme which catalyzes a reaction of converting a nitrile compound into a carboxylic acid and this is useful means for obtaining a carboxylic acid useful as a raw material for medical and agrochemical preparations.    Examples of the microorganisms which produces this enzyme include Fusarium solani (see Biochem.<br><br>
4<br>
J. 167, 685-692 (1977)), Nocardia sp. (see, Int. J. Biochem., 17, 677-683 (1985)), Arthrobacter sp. (see, Appl Environ. Microbiol., 51, 302-306 (1986)), Rhodococcus rhodochrous J1 (see, Eur. J. Biochem., 182, 349-356 (1989)), Rhodococcus rhodochrous K-22 (see, J. BacterioL, 172, 4807-4815 (1990)), Rhodococcus rhodochrous PA-34 (see, Appl. Microbiol. Biotechnol., 37, 184-190 (1992)) and Rhodococcus sp. ATCC39484 (see, Biotechnol. Appl. Biochem., 15, 283-302 sp. (1992)).<br>
From these microorganisms, nitrilase, nitrile hydratase, or amidase is produced. In order to use these enzymes in the genetic engineering, genes of some of these enzymes have been isolated and their primary structure has been determined.    With respect to the nitrilase gene, genes from Rhodococcus bacteria are disclosed, for example, in JP-A-7-99980 (the term "JP-A" as used herein denotes an unexamined Japanese patent application, first publication) and JP-A-9-28382.<br>
In recent years, attempts have been made to utilize the capability of converting a nitrile compound these microorganisms have.    Particularly, for the production of compounds having a high added value, an enzyme having excellent steric selectivity or position selectivity is required.    For example, JP-A-2-84198 discloses microorganisms for use in the production of an optically active  -substituted organic acid, JP-A-4-341185 discloses microorganisms for use in the production of an optically active 2-hydroxycarboxylic acid, and EPO433117 discloses microorganisms for use in the production of optically active ketoprofen.<br>
Among these microorganisms, the Rhodococcus sp. ATCC39484 strain has been reported to have a capacity to hydrolyze aromatic polynitrile compounds having a plurality of nitrile groups with excellent position selectivity (see, U S. Patent 556,625). The compounds having a nitrile group and a carboxyl group in the molecule, which are<br><br>
5<br>
produced by this selective nitrile degrading enzymatic system, are very effective as a synthesis block in the production of medical or agrochemical preparations.    However, the nitrilase of this microorganism is relatively low in the activity on aromatic polynitrile compounds and for utilizing this property in industry, it is an essential matter to improve the productivity of the enzyme which catalyzes the reaction.    However, the nitrilase gene of this microorganism, which is indispensable in the intended modification, has not yet been elucidated.<br>
Nitrile hydratase and amidase are enzymes which catalyze the reactions of converting a nitrile compound to an amide and an amide to a carboxylic acid, respectively.    By using nitrile hydratase and amidase, amides and carboxylic acids useful as starting materials for medicines, agrochemicals, etc. can be obtained from nitrile compounds.    Methods for converting nitrile compounds to corresponding amides or carboxylic acids have been developed by utilizing biocatalysts, and many microorganisms having such catalytic activity have been reported (see, JP-B-56-17918 (the term "JP-B" as used herein means an examined Japanese patent application, second publication), JP-B-59-037951, JP-B-61-162193, JP-B-61-021519, JP-B-64-086889, JP-B-4-197189, JP-B-2-000470, EP0444640, etc.).<br>
From these microorganisms, nitrile hydratase and amidase or nitrilase have been purified, and further, in order to utilize these genes in genetic engineering, the genes have been isolated and their primary structures have been determined.    With respect to the nitrile hydratase gene, for example, genes derived from Rhodococcus bacteria are disclosed in U.S. Patent 2,840,253 and EP0445646 (JP-A-40211379), genes derived from Pseudomonas bacteria are disclosed in JP-A-30251184, genes derived from Rhizobium bacteria are disclosed in JP-A-6-025296 and JP-A-6-303971.    Also,<br><br>
6<br>
with respect to the amidase gene, for example, genes derived from Brevibacterium bacteria and genes derived from Rhodococcus are disclosed in EP0433117.    Further, genes derived from Rhodococcus erythropolis are reported in Eur. J. Biochem. 217(1), 327-336 (1993) and genes derived from Pseudomonas bacteria are reported in FEBS Lett 367,275-279(1995).<br>
Further, an invention relating to a recombinant plasmid containing both a nitrile hydratase gene and an amidase gene derived from Rhodococcus bacteria is disclosed in JP-A-5-068566.<br>
In recent years, attempts have been made to utilize the capacity to convert a nitrile compound that these microorganisms have.    Particularly, for the production of compounds having a high added value, an enzyme having excellent steric selectivity or position selectivity is required.    For example, JP-A-2-84198 discloses microorganisms for use in the production of an optically active a-substituted organic acid, JP-A-4-341185 discloses microorganisms for use in the production of an optically active 2-hydroxycarboxylic acid, and EPO433117 discloses microorganisms for use in the production of optically active ketoprofen.<br>
Among these microorganisms, the Rhodococcus sp. ATCC39484 strain has been reported to have a capacity to hydrolyze aromatic polynitrile compounds having a plurality of nitrile groups with excellent position selectivity (see, U.S. Patent 556,625). The compounds having a cyano group and an amide group in the molecule or those compounds having a cyano group and a carboxyl group in the molecule, which are produced by this selective nitrile degrading enzymatic system, are very effective as a synthesis block in the production of medical or agrochemical preparations.    However, the nitrilase of this microorganism is relatively low in the activity on aromatic<br><br>
7<br>
polynitrile compounds and for utilizing this property in industry, it is an essential matter to improve the productivity of the enzyme which catalyzes the reaction.    However, the related enzyme genes of this microorganism, which are indispensable in the intended modification, have not yet been elucidated for either nitrile hydratase and amidase.<br>
DISCLOSURE OF THE INVENTION<br>
In consideration of the above-described problems, an object of the present invention is to provide a process for producing carboxylic acid, in which the hydrolysis reaction is favored with a higher yield than those in conventional processes and reduced in the amount of by-products, and also to provide a process for producing cyano carboxylic acids, comprising selectively hydrolyzing only a specific cyano group of a polynitrile compound to produce the corresponding cyano carboxylic acid, in which the hydrolysis reaction is favored with a higher yield than those in conventional processes and a reduced amount of by-products, and a mutant microorganism which catalyzes the above-described reactions.<br>
Another object of the present invention is to provide a novel nitrilase gene, a nitrile hydratase gene, and an amidase gene derived from a bacterium Rhodococcus. Still another object of the present invention is to provide a process for producing carboxylic acids from a nitrile compound, using a transformant transformed with a plasmid having incorporated therein these genes by using genetic engineering techniques.    Yet another object of the present invention is to provide such a transformant, such a plasmid, such genes, a process of producing an enzyme using the transformant, and an enzyme obtained by the process.<br>
The present inventors have made extensive investigations to substantially<br><br>
8<br>
reduce the by-products due to side reactions in the conventional hydrolysis reactions of a cyano group by microorganisms.    In particular, the by-products produced in various known techniques for producing cyanobenzoic acids from phthalonitriles were precisely analyzed, and as a result it has been found that the by-products produced in this reaction are mainly cyanobenzamide and phthalic acid monoamide further hydrolyzed from the cyanobenzamide.    Also, it has been found that when a microorganism defective or reduced in the activity of converting nitrile into amide is used in the reaction, those byproducts can be greatly decreased.<br>
For example, from the report by Kobayashi et al. (Nippon Nogeikagaku Kaishi (Japan Society for Bioscience, Biotechnology, and Agrochemistry), Vol. 71, No. 12 (1997)), and the like, it is known that two routes are present for the reaction by a microorganism to hydrolyze a cyano group of a nitrile compound into a carboxylic acid, (1) a one-stage reaction route by a nitrilase and (2) a two-stage reaction route of once passing through an amide form by two enzymes of nitrile hydratase and amidase.<br>
The present inventors have particularly studied the reaction route used in the conversion from a polynitrile compound into a cyano carboxylic acid by Rhodococcus sp. ATCC39484, which is a known nitrile converting bacterium.    This strain is confirmed to cause a reaction in a cell suspension to thereby produce a cyanobenzoic acid as a main product from phthalonitrile and at the same time produce cyanobenzamide and phthalic acid monoamide as by-products.    This was further studied and as a result it is estimated that the above-described two kinds of routes both competitively function in the hydrolysis of phthalonitrile by this microorganism.    From these, the present inventors have come to a conclusion that although the activity of the amide route has been considered useful for the production of a carboxylic acid by the<br><br>
9<br>
hydrolysis of nitrile, by rather making the activity defective or reduced, the by-products in question, specifically, cyanobenzamide and phthalic acid monoamide hydrolyzed from the cyanobenzamide, can be removed or reduced at the same time.<br>
From a parent strain ATCC39484, variant strain groups were formed using NTG (N-methyl-N'-nitro-N-nitrosoguanidine) in the ordinary manner.    Based on the assumption that the two-stage route passing through an amide form described above is under a series of controls, these variant groups were subjected to screening having as a target the inability to grow using benzamide as a sole carbon/nitrogen source.    As a result, many non-growing strains were acquired and actually subjected to the above-described reaction.    Then, a microorganism capable of extremely reducing the production of cyanobenzamide and phthalic acid monoamide in the reaction with phthalonitrile was acquired and designated as SD826 strain.    This acquisition has led to the accomplishment of the present invention.<br>
An embodiment of the present invention provides a process for producing carboxylic acids, comprising converting at least one cyano group of a nitrile compound into a carboxyl group using a microorganism, wherein a variant microorganism defective or reduced in the activity to convert a cyano group into an amide group is used.<br>
The variant microorganism may be a variant strain of a bacterium belonging to the genus Rhodococcus.    Further, the variant strain of a Rhodococcus bacterium may be a variant strain of a parent strain Rhodococcus sp. ATCC39484.    Further, in a preferred embodiment, the variant strain of a parent strain Rhodococcus sp ATCC39484 may be Rhodococcus sp. SD826 (FERM BP-7305).<br>
In the above process, the nitrile compound may be a polynitrile compound<br><br>
10<br>
having a plurality of cyano groups in the molecule and the carboxylic acid may be a cyano carboxylic acid.    In a preferred embodiment, the polynitrile compound is an aromatic polynitrile compound and the cyano carboxylic acid is an aromatic cyano carboxylic acid. More preferably, the aromatic polynitrile compound is o-phthalonitrile, isophthalonitrile, or terephthalonitrile, and the aromatic cyano carboxylic acid is o-cyanobenzoic acid, m-cyanobenzoic acid, or p-cyanobenzoic acid.<br>
Another embodiment of the present invention provides a variant microorganism having an activity to covert a cyano group into a carboxyl group and being defective or reduced in the activity of converting a cyano group into an amide group.    The variant may be a variant strain of a microorganism belonging to the genus Rhodococcus.    In a preferred embodiment, the variant microorganism is a variant strain of Rhodococcus sp. ATCC39484.<br>
Another embodiment of the present invention provides a Rhodococcus sp. SD826 (FERM BP-7305) strain.    The Rhodococcus sp. SD826 has been deposited on October 12, 1999 at National Institute of Bioscience and Human-Technology, Agency of Industrial Science and Technology, Ministry of International Trade and Industry (1-3, Higashi 1-chome Tsukuba-shi Ibaraki-ken, Japan)(Accession Number: FERM BP-7305).<br>
Another embodiment of the present invention provides a process for producing carboxylic acids, comprising converting a cyano group of a nitrile compound into a carboxyl group using a transformant transformed with a plasmid containing a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 2 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing<br><br>
11<br>
carboxylic acids, comprising converting a cyano group of a nitrile compound into a carboxyl group using a transformant transformed with a plasmid containing a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO lin the sequence list.<br>
Another embodiment of the present invention provides a process for producing cyano carboxylic acids, comprising converting at least one nitrile group of a polynitrile compound into a carboxyl group using the above-described transformant.<br>
In these production processes, the polynitrile compound may be an aromatic polynitrile compound.    Preferably, the aromatic polynitrile compound may be phthalonitrile, isophthalonitrile, or terephthalonitrile, and the cyano carboxylic acid may be o- cyanobenzoic acid, m- cyanobenzoic acid or p-cyanobenzoic acid.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 2 of the sequence list, for use in the process described above.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 1 of the sequence list, for use in the above-described production processes.<br>
Another embodiment of the present invention provides a plasmid containing a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 2 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing a<br><br>
12<br>
nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 1 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 2 of the sequence list.<br>
Another embodiment of the present invention provides a nitrilase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 1 of the sequence list,<br>
The Rhodococcus bacterium may be a Rhodococcus sp. ATCC39484 strain.<br>
Another embodiment of the present invention provides a process for producing nitrilase, comprising culturing the above-described transformant in a culture medium and collecting nitrilase from the culture.<br>
Another embodiment of the present invention provides nitrilase prepared by the above-described process.<br>
Another embodiment of the present invention provides a process for producing amide compounds, comprising converting a cyano group of a nitrile compound into an amide group using a transformant transformed with a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and 5 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing amide compounds, comprising converting a cyano group of a nitrile compound into an amide group using a transformant transformed with a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence<br><br>
13<br>
shown by SEQ ID NO 3 in the sequence list.<br>
Another embodiment of the present invention provides a process for producing carboxylic acids, comprising converting an amide group of an amide compound into a carboxyl group using a transformant transformed with a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing carboxylic acids, comprising converting an amide group of an amide compound into a carboxyl group using a transformant transformed with a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing carboxylic acids, comprising converting a cyano group of a nitrile compound into a carboxyl group using a transformant transformed with a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and/or 5 of the sequence list and an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing carboxylic acids, comprising converting a cyano group of a nitrile compound into a carboxyl group using a transformant transformed with a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list and an amidase gene derived<br><br>
14<br>
from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list.<br>
Another embodiment of the present invention provides a process for producing amide compounds, wherein the nitrile is orthophthalonitrile, isophthalonitrile, or terephthalonitrile, and the amide compound is o-cyanobenzamide, m-cyanobenzamide, or p-cyanobenzamide.<br>
In the above-described production, the amide compound may be o-cyanobenzamide, m-cyanobenzamide, or p-cyanobenzamide and the carboxylic acid may be o-, m-, or p-cyanobenzoic acid.    The nitrile may be orthophthalonitrile, isophthalonitrile, or terephthalonitrile, and the carboxylic acid may be o-, m-, or p-cyanobenzoic acid.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and/or 5 of the sequence list, for use in the above-described process.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list, for use in the above-described process.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NO 7 of the sequence list, for use in the above-described process.<br><br>
15<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list, for use in the above-described process.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and 5 of the sequence list and an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list, for use in the above-described process.<br>
Another embodiment of the present invention provides a transformant transformed with a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list and an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ED NO 6 of the sequence list, for use in the above-described process.<br>
Another embodiment of the present invention provides a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and 5 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list, for use in the preparation of the<br><br>
16<br>
above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequences shown by SEQ ID NOs 4 and/or 5 of the sequence list and an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a plasmid containing both a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list and an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list, for use in the preparation of the above-described transformant.<br>
Another embodiment of the present invention provides a nitrile hydratase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NOs 4 and 5 of the sequence list.<br>
Another embodiment of the present invention provides a nitrile hydratase gene<br><br>
17<br>
derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 3 of the sequence list.<br>
The Rhodococcus bacterium may be a Rhodococcus sp. ATCC39484 strain.<br>
Another embodiment of the present invention provides an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence encoding the amino acid sequence shown by SEQ ID NO 7 of the sequence list.<br>
Another embodiment of the present invention provides an amidase gene derived from Rhodococcus bacterium consisting of a DNA sequence shown by SEQ ID NO 6 of the sequence list.<br>
The Rhodococcus bacterium may be a Rhodococcus sp. ATCC39484 strain.<br>
Another embodiment of the present invention provides a process for producing nitrile hydratase, comprising culturing the above-described transformant in a culture medium and collecting nitrile hydratase from the culture.<br>
Another embodiment of the present invention provides a process for producing amidase, comprising culturing the above-described transformant in a culture medium and collecting amidase from the culture.<br>
Another embodiment of the present invention provides a process for producing nitrile hydratase and/or amidase, comprising culturing the above-described transformant in a culture medium and collecting nitrile hydratase and/or amidase from the culture.<br>
Another embodiment of the present invention provides nitrile hydratase prepared by the above production process.<br>
Another embodiment of the present invention provides amidase prepared by the above-described process.<br>
Another embodiment of the present invention provides nitrile hydratase and/or<br><br>
18<br>
amidase prepared by the above-described process.<br>
According to the present invention, high purity carboxylic acids can be simply and easily obtained from nitrile compounds as starting materials using a novel variant belonging to the genus Rhodococcus.    Also, high purity cyanocarboxylic acids can be simply and easily obtained from polynitrile compounds, particularly aromatic polynitrile compounds as starting materials.<br>
Further, the present invention provides a nitrilase gene, a nitrile hydratase gene and an amidase gene derived from Rhodococcus bacteria capable of exhibiting particularly excellent position selectivity for the cyano group of aromatic polynitrile compounds.    The DNA sequences of these genes derived from the Rhodococcus bacterium are indispensable for efficient production of nitrile hydratase and amidase using genetic engineering techniques and improvement of enzymes using protein engineering techniques.    As expected, the enzymes thus obtained are applicable to the industrial production of useful compounds.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 is a schematic diagram illustrating the structure of a plasmid prepared from a positive clone obtained by the colony hybridization (Example 4).<br>
Fig. 2 is a schematic diagram illustrating the construction of a nitrilase gene expression plasmid.<br>
Fig. 3 is a graph showing a comparison of accumulation curves of p-cyanobenzoic acid in the conversion reaction from terephthalonitrile to p-cyanobenzoic acid.<br>
Fig. 4 is a schematic diagram illustrating the structure of a plasmid prepared<br><br>
19<br>
from a cloned strain.<br>
Fig. 5 is a schematic diagram illustrating the construction (1) of expression plasmid.<br>
Fig. 6 is a schematic diagram illustrating the construction (2) of expression plasmid.<br>
Preferred Embodiments of the Invention<br>
1. Variant microorganisms for preparing carboxylic acids<br>
As the parent strain used in the present invention for producing a variant microorganism defective or reduced in the activity of hydrolyzing a cyano group into an amide group, various generally known microorganisms having an activity to hydrolyze a cyano group of a nitrile compound may be used.    In particular, microorganisms which have a nitrilase activity and allow a nitrile compound to undergo a carboxylic acid-generating reaction by hydration such that the by-products are an amide form, may be used.    Examples of the microorganisms known to have an activity of hydrolyzing nitrile include the microorganisms belonging to genera such as Rhodococcus, Rhodotorulla, Fusarium, Psendomonas, Acinetobacter, Bacillus, Brevibacterium, Klebsiella, Micrococcus, Burkholderia, Corynebacterium, Noccardia, Aeromonas, Agrobacterium, Achromobacter, Aspergillus and Rhizobium.<br>
For example, the Rhodococcus sp. ATCC39484 strain is cultured by a commonly known method for culturing microorganisms, and a generally known variation inductive compound or an ultraviolet ray is provided to act on the cells obtained to prepare a variant microorganism group.    Examples of the variation inductive compound include alkylating agents such as NTG (N-methyl-N'-nitro-N-<br><br>
20<br>
nitrosoguanidine) and EMS (ethyl methanesulfonate), base analogs such as 5-bromouracil, and intercalation agents such as azaserine and acridine orange.<br>
From the variant microorganism group prepared, variant strains reduced or defective in the activity of producing an amide compound from a nitrile compound are selected.    The fact that the variant strain is reduced or defective in the activity of producing an amide compound may be demonstrated as follows. Culture cells of the variant microorganism strain are allowed to act on a nitrile compound, the product obtained is analyzed by a method of analysis such as HPLC, and the state how the corresponding carboxylic acid amide form is produced accompanying the degradation of the nitrile compound is observed.<br>
At this time, in order to effectively concentrate the objective variant strains from a huge variant microorganism group, based on the assumption that the two-stage route passing through the amide form is subject to a series of control, the inventors considered using the loss or reduction of the activity of growth by assimilating an amide compound which can nourish and grow the parent strain microorganism, for example, benzamide or the like when ATCC39484 is the parent strain, as an index for the absence or reduction of a series of the reaction route to a carboxylic acid through an amide compound.    By various methods using this index, an efficient concentration of the objective variant microorganism form a variant microorganism group can be realized.<br>
The term "the loss or reduction of the activity of growth by assimilating an amide compound" as used in the present invention means that in the culture using the same amide compound as a nutrient source, the doubling time of the microorganisms is almost 2 times longer than that of the parent strain, or that the microorganisms cannot grow at all.    For example, a variant microorganism group is spread on an ordinary<br><br>
21<br>
nutritive agar culture medium, for example, LB agar culture medium, and then, the colonies formed are individually transplanted on an agar culture medium which contains benzamide as a sole carbon/nitrogen source, and by visually observing the presence or absence of the growth, the variant strains changed in the activity to assimilate benzamide can be detected.    Also, by applying what is termed the penicillin screening method, the variant microorganisms defective or reduced in the activity of growing using benzamide can be concentrated.    More specifically, a drug which acts on microorganisms in the process of their fission and growth and kills the microorganisms, for example, penicillin, is added to the culture medium using benzamide as a sole carbon/nitrogen source, and a variant microorganism group is inoculated thereon and cultured, where the strains capable of growing well with benzamide are killed and the strains defective or reduced in the activity of assimilating benzamide and growling are concentrated.    Thus, a concentrated group of variant microorganisms is obtained.<br>
The cultured cells of each variant microorganism are allowed to act on a nitrile compound, for example, phthalonitrile when the parent strain is ATCC39484, and the product obtained is analyzed by a method of analysis such as HPLC to search for the strains reduced in the accumulation of the carboxylic acid amide form. In the group of variant microorganisms thus concentrated, the strain reduced in the accumulation of the carboxylic acid amide form is found together with strains increased in the accumulation.<br>
One example of the variant strain thus created is Rhodococcus sp. SD826. The Rhodococcus sp, SD826 is a strain created by the present inventors from Rhodococcus sp. ATCC39484 (divided from American Type Culture Collection in U. S. A.) which is a known microorganism, and this new variant strain has been deposited at National Institute of Bioscience and Human-Technology, Agency of Industrial Science<br><br>
22<br>
and Technology, Ministry of International Trade and Industry, Japan, as FERM BP-7305.<br>
For example, the variant microorganism strain reduced or defective in the ability to produce amide compounds applied to the present invention may be acquired by destroying or deleting enzymes or factors regulating the enzymes participating in the production of amide compounds and regions of gene encoding these using genetic engineering techniques.    More specifically, it is realized as follows.    Related genes of enzymes contributing to the reaction are isolated and analyzed, a gene fragment having incorporated therein is introduced, a sequence having homology to the base sequence is introduced into a microorganism, and homologous recombination between enzyme-related genes on the chromosome is induced to cause insertion or deletion of the base sequence.<br>
The microorganisms which are applied to the present invention are those microorganisms which are reduced or defective in the ability to produce an amide compound.    The modification operation may affect other properties of the microorganism, in particular the ability to produce carboxylic acids which would be considered to be closely related to each other.    However, according to the object of the present invention, it may be sufficient that the production of amide compounds be relatively reduced as compared with the resulting carboxylic acid.    Although it is desirable that such a modification will cause no decrease in the ability of a variant strain to produce carboxylic acids as compared with that of the parent strain before the modification, the ability to produce carboxylic acids may vary within the ranges where the production of amide compounds is relatively decreased.<br>
The reaction of the present invention uses the thus-created variant<br><br>
23<br>
microorganism and can be performed in the same manner as in the conversion reaction using a general microorganism in an ordinary carboxylic acid-generating reaction by a microorganism having the activity of hydrolyzing a cyano group.    For example, the SD826 strain is cultured in a nutritive culture medium of 1% peptone or the like at a temperature of from 20 to 40°C, preferably from 25 to 30°C, for about 24 hours.    To the resulting culture, a nitrile compound is added in an amount of from 1 ppm to 50%, preferably from 10 ppm to 10%, and then the solution is continuously stirred at the temperature mentioned above for from 1 to 200 hours, and thereby the reaction is carried out. The entire amount of the added nitrile compound is not be dissolved. However, a solvent, a surfactant, or the like which improves the solubility or dispersibility in the reaction solution may be added.    In proportion with the amount of the nitrile compound consumed as the reaction proceeds, a nitrile compound may be added continuously or intermittently.    At this time, the concentration of the nitrile compound in the reaction solution is not limited to the above-described range.<br>
Examples of the carbon source which can be used in the culture medium for culturing microorganisms include saccharides such as glucose, sucrose, fructose and molasses, organic materials such as ethanol, acetic acid, citric acid, succinic acid, lactic acid, benzoic acid and fatty acid, alkali metal salts thereof, aliphatic hydrocarbons such as n-paraffin, aromatic hydrocarbons, and naturally occurring organic materials such as peptone, meat extract, fish extract, soybean powder and bran. These are used individually or in combination usually in a concentration of from 0.01 to 30%, preferably on the order of from 0.1 to 10%.<br>
Examples of the nitrogen source which can be used in the culture medium for culturing microorganisms include inorganic nitrogen compounds such as ammonium<br><br>
24<br>
sulfate, ammonium phosphate, sodium nitrate and potassium nitrate, nitrogen-containing organic materials such as urea and uric acid, and naturally occurring organic materials such as peptone, meat extract, fish extract and soybean powder.    These are used individually or in combination usually in a concentration of from 0.01 to 30%, preferably from 0.1 to 10%.    These starting materials for the reaction, of which cyano group is hydrolyzed by the strain into a carboxyl acid, are preferably added in advance during the culture, so that ammonium ion isolated by the hydrolysis with the progress of the reaction can serve as the nitrogen source for microorganisms.<br>
Furthermore, in order to improve the growth of the cells, a phosphate such as potassium dihydrogenphosphate or metal salt such as magnesium sulfate, ferrous sulfate, calcium acetate, manganese chloride, copper sulfate, zinc sulfate, cobalt sulfate and nickel sulfate, may be added, if desired.    The concentration in this addition varies depending on the culture conditions.    However, it is usually from 0.01 to 5% for the phosphate, from 10 ppm to 1% for the magnesium salt, and approximately from 0.1 to 1,000 ppm for other compounds.    In addition, depending on the culture medium selected, a source for supplying vitamins, amino acid, nucleic acid or the like, such as yeast extract, casamino acid, and yeast nucleic acid, may be added in an amount of approximately from 1 to 100 ppm, so that the growth of cells can be improved.<br>
In order to improve the reactivity of cells with the cyano group, a nitrile compound such as benzonitrile is preferably added during the culture in an amount of from 10 ppm to 1% as a source for inducing a cyano group hydrolase.    Furthermore, a nitrile compound which can serve both as a starting material for the reaction and an inducing source is preferably added during the culture.<br>
In using any ingredient, the pH of the culture medium is preferably adjusted to<br><br>
25<br>
from 5 to 9, more preferably from 6 to 8.    Also, the reaction is preferably performed after collecting the microorganism cells previously cultured in the medium described above from the culture solution by centrifugation or filtration through a membrane and re-suspending them in water containing a nitrile compound as a reaction starting material, in a physiological saline, or in a buffer solution which is adjusted to have the same pH as the culture medium and comprises phosphoric acid, acetic acid, boric acid, tris(hydroxymethyl)aminomethane, or a salt thereof, because the impurities in the reaction solution can be reduced and afterward the product can be easily collected. The pH can be usually maintained during the reaction when a buffer solution having sufficiently high concentration is used.    However, in the case where the pH departs from the above-described range with the progress of the reaction, the pH is preferably appropriately adjusted using sodium hydroxide, ammonia or the like.<br>
The cyano carboxylic acid produced in the reaction solution is collected by a commonly used method such as centrifugation, filtration through a membrane, drying under reduced pressure, distillation, extraction with a solvent, salting out, ion exchange, and various kinds of chromatography.    The collecting method is selected according to the status of the cyano carboxylic acid in the reaction solution.    Most simply and easily, the cyano carboxylic acid is precipitated by adjusting the reaction solution to be acidic, and the precipitate is centrifuged or filtered to recover the cyano carboxylic acid.    In the case where the reaction product is obtained as an aqueous solution, the microorganism cells are preferably removed by centrifugation, filtration through a membrane, or the like under the condition that the product is in the dissolved state.    In the case where the reaction product is obtained as a solid and when the crystal is sufficiently large, the product may be collected using a mesh formed of stainless steel,<br><br>
26<br>
nylon, or the like.    When the crystal is small and cannot be fractionated from microorganisms, a method of once forming the reaction product into an aqueous solution by setting a condition where the solid can dissolve, for example, an alkali condition, removing the cells by centrifugation, filtration through a membrane or the like, recovering the condition, re-precipitating the solid, and collecting the reaction product, is preferably used.    However, this is not an exclusive method if the microorganisms can be removed by means of ordinary art, such as direct distillation of the reaction solution.<br>
Depending on the properties of the reaction product, the product may accumulate in the reaction solution to decrease the reaction rate.    In this case, a method of adding water, physiological saline, or a reaction buffer solution to the reaction solution and continuously diluting the reaction solution according to the concentration of the product is suitably used.    Also, the reaction rate can be recovered by collecting the cells at the time when the reaction rate has decreased, recovering the supernatant as a product solution, and returning the collected cells to the solution or suspension containing the reaction starting material.    These methods each may be repeated on any number of occasions as long as the microorganisms maintain the activity of hydrolyzing nitrile.<br>
The present invention may also be similarly performed even using a cell-free extract of the microorganisms applied to the present invention or using an ingredient which catalyzes the above-described reaction and which is concentrated or extracted from the cell-free extract.    Furthermore, the present invention may be achieved by immobilizing a microorganism which can be applied to the present invention, or an extract solution or extracted ingredient thereof to a sparingly soluble supporter and<br><br>
27<br>
bringing this immobilized matter into contact with a starting material solution. Examples of the supporter which can be used for the immobilization include compounds capable of forming a sparingly water-soluble solid containing the microorganism or an extracted ingredient thereof, such as polyacrylamide, polyvinyl alcohol, poly-N-vinylformamide, polyallylamine, polyethyleneimine, methyl cellulose, glucomannan, alginate, carrageenan, and a polymer or cross-linked product thereof. These compounds may be used individually or in combination.    In addition, those obtained by bearing the microorganism or an extract solution or extracted ingredient thereof on a material previously formed as a solid, such as activated carbon, porous ceramic, glass fiber, porous polymer compact, and nitrocellulose membrane may be used.<br>
According to the process of the present invention, the specificity of the substrate used for the hydrolysis reaction of cyano groups is broad, the object includes various commonly known nitrile compounds such as aliphatic nitrile, aromatic nitrile, and heterocyclic nitrile, and the corresponding carboxylic acid can be obtained with high selectivity.<br>
Examples of the aliphatic nitrile include acetonitrile, propionitrile, n-butyronitrile, isobutyronitrile, n-valeronitrile, isovaleronitrile, capronitrile, malononitrile, glucononitrile, adiponitrile, succinonitrile, acrylonitrile and methacrylonitrile.<br>
Examples of the aromatic nitrile include benzonitrile, terephthalonitrile, orthophthalonitrile, tolunitrile, isophthalonitrile and substitution products of these aromatic nitrile compounds, such as chlorinated product, fluorinated product, nitrated product and aminated product.<br><br>
28<br>
Examples of the heterocyclic nitrile include 3-cyanopyridine, 4-cyanopyridine and cyanoindoles<br>
Also, according to the present invention, the object is preferably a polynitrile compound having a plurality of cyano groups in one molecule among the compounds described defective or reduced in the above and the corresponding cyano carboxylic acid can be obtained with high selectivity.    Examples of the polynitrile compound include aliphatic nitriles such as malononitrile, succinonitrile, adiponitrile and glucononitrile, aromatic nitriles such as orthophthalonitrile, terephthalonitrile and isophthalonitrile and substitution products of these aromatic nitrile compounds such as chlorinated product, fluorinated product, nitrated product and aminated product.<br>
According to the process of the present invention, carboxylic acids reduced in the amount of by-products can be obtained, specifically, so as to contain the by-products originating in the starting material nitrile compound in a total amount of 0 5 (mol)% or less in the product carboxylic acid.    In recent years, there is a great concern about the effects of trace chemical substances on human bodies and therefore, the present invention is made based on the concept that the substantial reduction of side reactions in the chemical reaction and high-purity chemicals obtained by such a reaction can create new possibilities in industry.<br>
In the process of the present invention, the production route of a carboxylic acid through an amide is fundamentally defective or reduced and therefore amide-form impurities ascribable to the partial hydrolysis of nitrile, and derivatives thereof, are not produced.    The carboxylic acids obtained by the present invention are suitable as a starting material for the synthesis in the field where high purity is particularly required, for example, in the field of medicaments or fine chemicals.<br><br>
29<br>
2. Nitrilase gene derived from Rhodococcus bacteria<br>
The method for determining the DNA sequence of the nitrilase gene of Rhodococcus sp. is described below.    The chromosomal DNA, for example, of the Rhodococcus sp. ATCC39484 strain can be prepared by applying the method of Saito et al. (see, Biochem. Biophys. Acta., 72, 619 (1963)).    The chromosomal DNA library for use in the cloning of the gene can be manufactured using, for example, a plasmid vector pUC18.    The cloning of the nitrilase gene can be performed using, for example, the polymerase chain reaction (PCR) by Saiki et al. (see, Science 230, 1350 (1985)).    At this time, a universal primer (forward or reverse) is used as one primer in PCR and for another primer, an appropriate sequence is selected from the DNA sequence coding for an enzyme N terminal sequence.    By combining these primers, an anchor PCR is performed using the chromosomal DNA library as a template and thereby the coding sequence fragment of the objective enzyme can be obtained.    By using the nitrilase coding sequence DNA partial fragment as a probe for the screening of all gene regions, a recombinant DNA containing a nitrilase gene can be obtained from the chromosomal DNA library of the Rhodococcus sp ATCC39484 strain.    The DNA sequence of the nitrilase coding sequence fragment can be determined using known method such as the dideoxy method described by Sanger et al. (see, Proc. Natl. Acad. Sci. U.S.A., 74, 5463 (1997)).<br>
In order to produce a nitrilase enzyme using the thus-obtained enzyme structural gene, the enzyme structural gene is ligated with an appropriate expression vector, for example, downstream from the lac promoter of pUC18.    Using the thus-obtained plasmid, a host such as Eschenchia coli JM101 is transformed     By culturing the obtained transformant, the objective nitrilase is produced in a very large amount<br><br>
30<br>
within the host cells.    The intact nitrilase cells may be used for the conversion reaction but a cell-free extract or purified enzyme obtained from the extract may also be used.<br>
In order for enzyme genes derived from different microorganisms to be expressed in a host microorganism in a manner such that it actually functions therein, it is well known that various requirements must be satisfied, for example, that the gene is actually retained and divided in the host microorganism, that the gene is transcribed by the transcription function of the host, that the transcribed information is translated into a protein, that the polypeptide produced by the translation is folded into a higher dimensional structure so that it can have a function, that an enzyme is secreted in the same manner as in the donor microorganism so that the enzyme can contact a substrate, or if the enzyme is an intracellular enzyme, that the host microorganism has a permeation/transportation system similar to those of the donor microorganism, and so on.    Further, in order for the expression to be in some degree useful for industrial application, each of the requirements must be satisfied at high levels.    To solve these problems, usually, operations such as analysis and modification of regulating regions, e g., promoters, adaptation of transcription/expression system and various cofactors to the body of the target gene by constructing a complicated shuttle vector and returning a cloned gene to the donor microorganism and the like become necessary.    These methods have the problems that it is difficult to obtain information on the target of analysis and to modify the method of modification for the regulating mechanism , which must rely on a trial and error method, and that they are limited by the ability of the donor in performing the expression by returning the cloned gene to the donor.    As far as the present inventors are aware, although some cases exist where the expression of nitrilase from a different microorganism has been confirmed, no case is known where<br><br>
31<br>
a nitrilase gene is obtained which catalyzes selective hydrolysis of polynitrile and which can exhibit high activity far exceeding the ability of the donor microorganism simply and easily by incorporating it into a well-known Escherichia coli vector system in order to transform it.    Also, no case is known where a selective and high level reaction is performed targeting nitrile compounds, in particular, aromatic polynitrile compounds, by a recombinant using such a nitrilase gene.<br>
The nitrile compound used as a starting material in the present invention is an aliphatic or aromatic compound having one nitrile group, or an aliphatic or aromatic polynitrile compound having a plurality of nitrile groups.    When the starting material used is orthophthalonitrile, isophthalonitrile or terephthalonitrile, the corresponding o-, m-, or p-cyanobenzoic acid can be preferably obtained in high purity.<br>
In the present invention, the conversion reaction may be performed by adding a starting material substance and cells, cell-free extract or enzyme having the conversion activity, to a dilute aqueous solution, such as a phosphate buffer solution, at a pH of from 5 to 10, preferably from 6 to 8, and a temperature of from 15 to 45°C, preferably from 30 to 42°C.<br>
The method for collecting the product produced in the reaction solution is not particularly limited but, for example, the supernatant of the reaction solution is separated and recovered, and thereafter the product may be obtained using a method such as precipitation formation, extraction, distillation, or combinations thereof, according to the properties of the product.    Also, the product can be obtained in high purity by performing separation and purification using column chromatography or the like 3  Nitrile hydratase gene and amidase gene derived from Rhodococcus bacteria<br><br>
32<br>
The chromosomal DNA, for example, of the Rhodococcus sp. ATCC39484 strain can be prepared by applying the method of Saito et al. (see, Biochem. Biophys. Acta., 72, 619 (1963)).    The chromosomal DNA library for use in the cloning of the gene can be manufactured using, for example, a plasmid vector pUCl 8.    The cloning of the nitrile hydratase gene and amidase gene can be performed by colony hybridization using a partial fragment prepared, for example, by the polymerase chain reaction (PCR) method by Saiki et al. (see, Science 230, 1350 (1985)) as a probe.    At this time, a universal primer (forward or reverse) is used as one primer in PCR and for another primer, an appropriate sequence is selected from the DNA sequence coding for an object enzyme protein N terminal sequence being analyzed.    By combining these primers, an anchor PCR is performed using the chromosomal DNA library as a template and thereby the coding sequence fragment of the objective enzyme can be obtained. By using the nitrile hydratase gene encoding sequence DNA fragment or amidase encoding sequence DNA fragment as a probe for the screening of all gene regions, a recombinant DNA containing a nitrile hydratase gene and/or amidase gene can be obtained from the chromosomal DNA library of the Rhodococcus sp. ATCC39484 strain.    The DNA sequences of the nitrile hydratase encoding sequence fragment and amidase encoding sequence fragment can be determined using known method such as dideoxy method described by Sanger et al. (see, Proc. Nail. Acad. Sci. U.S.A., 74, 5463 (1997)).<br>
In order to produce an enzyme using the thus-obtained enzyme structural gene, the enzyme structural gene is ligated with an appropriate expression vector, for example, downstream from the lac promoter of pUCl8.    Using the thus-obtained plasmid, a host such as Eschehchia coli JM101 is transformed.    By culturing the obtained<br><br>
33<br>
transform ants, the objective nitrile hydratase and/or amidase are or is produced in a very large amount within the host cells.    The enzyme or enzymes may be used in the form of intact cells for the conversion reaction but a cell-free extract or purified enzyme obtained from the extract may also be used.<br>
In order for enzyme genes derived from different microorganisms to be expressed in a host microorganism in a manner such that it actually functions therein, it is well known that various requirements must be satisfied, for example, that the gene is actually retained and divided in the host microorganism, that the gene is transcribed by the transcription function of the host, that the transcribed information is translated into a protein, that the polypeptide produced by the translation is folded into a higher dimensional structure so that it can have a function, that an enzyme is secreted in the same manner as in the donor microorganism so that the enzyme can contact a substrate, or if the enzyme is an intracellular enzyme, the host microorganism has a permeation/transportation system similar to those of the donor microorganism, and so on.    Further, in order for the expression to be in some degree useful for industrial application, each of the requirements must be satisfied at high levels.    To solve these problems, usually, operations such as analysis and modification of regulating regions, e.g., promoters, adaptation of transcription/expression system and various cofactors to the body of the target gene by constructing a complicated shuttle vector and returning cloned gene to the donor microorganism and the like become necessary.    These methods have the problems in that it is difficult to obtain information on the target of analysis and to modify method of modification for the regulating mechanism, which must rely on a trial and error method, and that they are limited by the ability of the donor in performing the expression by returning the cloned gene to the donor     As far<br><br>
34<br>
as the present inventors are aware, although some cases exist where nitrile hydratase gene and amidase gene of different microorganisms have been obtained and some cases where the expression of such nitrile hydratase and amidase has been confirmed, no case is known where a nitrile hydratase gene and amidase gene are obtained which catalyze selective hydrolysis of polynitrile and which can exhibit high activity highly exceeding the ability of the donor microorganism simply and easily by incorporating them into a well-known Escherichia coli vector system to transform it.    Also, no case is known where selective and high level reaction is performed targeting nitrile compounds, in particular, aromatic polynitrile compounds by a recombinant using such nitrile hydratase gene and amidase gene.<br>
The process for producing the carboxylic acids or amides conversion reaction of the present invention may be performed by adding a starting material substance and cells, cell-free extract or enzyme having the conversion activity, to a dilute an aqueous solution such as a phosphate buffer solution, at a pH of from 5 to 10, preferably from 6 to 8, and a temperature of from 15 to 45°C, preferably from 30 to 42°C. The product produced in the reaction solution may be obtained using a precipitation formation or column chromatography, depending on the property of the product.<br>
The nitrile used as a starting material in the present invention is an aliphatic or aromatic compound having at least one nitrile group in the molecule.    Preferred examples thereof include aromatic polynitrile compounds such orthophthalonitrile, isophthalonitrile, and terephthalonitrile.<br>
The amide used as a starting material of the process for producing carboxylic acids according to the present invention is an aliphatic or aromatic compound having an amide group     Preferred examples thereof include aromatic amide compounds having a<br><br>
35<br>
cyano group such as o-, m-, or p-cyanobenzamide.<br>
Hereinafter, the present invention will be described more specifically by examples.    However, the present invention should not be construed as being limited thereto.<br>
[Example 1]    Acquisition of Variant microorganism<br>
Rhodococcus sp. ATCC39484 (obtained from American Type Culture Collection in U.S.A.) was streaked in an LB agar culture medium and cultured for 24 hours in a constant temperature bath at 30°C.    From the colonies generated, one loopful of cells was picked up and inoculated in 5 ml of LB liquid medium and cultured under shaking in a shaker at 30°C for 6 hours.    The cells were recovered by the centrifugation of 10,000 g, washed three times with isovolume 50 mM potassium/sodium phosphate buffer solution (pH: 7.0), and again suspended in the same isovolume buffer solution.<br>
To the cell suspension, 2,000 ppm of NTG (N-methyl-N'-nitro-N-nitrosoguanidine) solution was added to have a final concentration of 100 ppm.    After thoroughly stirring, the solution was left standing at room temperature for 30 minutes. Then, the cells were recovered by the centrifugation of 10,000 g, washed once with the same buffer solution and again suspended in a slight amount of the same buffer solution. Thereafter, the entire amount of the cells were inoculated in 5 ml of an inorganic salt liquid medium containing 0.1% of benzamide.    The composition of the inorganic salt medium is shown below.<br>
(Inorganic Salt Culture Medium)<br>
KH2PO4	1.5 g/l<br><br>
36<br><br>
After the shaking culture at 30°C for 15 hours, ampicillin was added so as to have a concentration of 1 mg/L, and further cultured under shaking at 30°C for 12 hours. The culture solution obtained was 500-fold diluted and the dilution solution was spread on 300 plates of LB solid media (each on a Petri dish having a diameter of 90 mm) in an amount of 0.1 ml per plate.    Then the cells were cultured at 30°C for 48 hours and when colonies were generated, the colonies were copied to an autoclaved velvet, each Petri dish as a whole was transcribed to an inorganic salt solid medium (diameter 90 mm) having the above-described composition and containing 0.1% of benzamide and 1.5% of agar, and the cells were cultured at 30°C for 48 hours.<br>
The colony formation was compared between the solid medium as a transcription original and the inorganic salt solid medium as a transcription target, and about 400 strains which grew well in the LB and not in the inorganic salt solid medium were selected.    These selected strains were transplanted from the transcription original LB to a new LB solid medium by means of sterilized toothpick and cultured at 30°C for 24 hours.    All colonies generated were inoculated in 5 ml of the above-described inorganic salt culture medium and 0 1% isophthalonitrile was added thereto and reacted at 30°C for 48 hours.    The parent strain ATCC39484 was also cultured, inoculated, and reacted in the same manner.    The supernatant of the reaction solution obtained with<br><br>
37<br>
each strain was diluted 100-fold and subjected to reverse phase HPLC (column; Shodex DS-613, eluant: 50% acetonitrile/5 mM potassium-phosphate buffer solution, pH: 3.0, flow rate: 1 mL/min, detection: UV 210 nm).    In one strain (SD826 strain), it was verified that a large amount of 3-cyano benzoic acid was detected similarly to the parent strain ATCC39484 and 3-cyanobenzamide and phthalic acid monoamide, which were detected in the reaction solution of the parent strain, were extremely reduced.    The strain obtained was considered to be an objective strain defective in the side reaction route.<br>
Table 1 below describes the mycological properties of novel Rhodococcus bacterium, Rhodococcus sp. SD826 (FERM BP-3705).<br><br>
38<br>
Table 1<br><br><br>
39<br>
[Example 2]<br>
Rhodococcus sp. SD826 was streaked in an LB agar culture medium and cultured in a constant temperature bath at 30°C for 24 hours.    From the colonies generated, one loopful of cells were picked up and suspended in 100 mL of an LB liquid medium placed in a 500 mL-volume baffled flask.    The flask was placed in a constant temperature rotary shaker at 30°C and cultured with 120 revolutions per minute for 24 hours.    The microorganism cells obtained were recovered by the centrifugation of 10,000 g and suspended in a 50 mM sodium/potassium phosphate buffer solution (pH: 7) isovolume to the culture solution.    To the cell suspension, isophthalonitrile corresponding to 5% (mass/volume) was added, the suspension was placed in a constant temperature rotary shaker at 30°C, and the reaction was performed with 120 revolutions per minute for 72 hours.<br>
The reaction solution obtained was adjusted to a pH of 2 using 2 mol/1 of hydrochloric acid, ethyl acetate isovolume to the reaction solution was added, and the resulting solution was stirred and extracted. The ethyl acetate layer obtained was appropriately diluted and analyzed by reverse phase HPLC (column: Shodex DS-613: eluant  50% acetonitrile/5 mM potassium phosphate buffer solution, pH: 3.0, flow rate: 1 mL/min, detection: UV 210 nm).    In the reaction solution, the main ingredients having a peak whose retention time was coincident with the 3-cyanobenzoic acid sample was found     The peak ingredients were collected and subjected to GC-mass spectral analysis.    As a result, each was verified to delineate a fragment pattern suggesting the same structure as the sample.<br>
As a comparative example, the reaction, extraction, and analysis were performed in the same manner as above except for using a parent strain ATCC39484<br><br>
40<br>
The main ingredients in the reaction solution of the parent strain were subjected to LC-MS analysis and identified.<br>
The comparison between the reaction solution of the parent strain and the reaction solution of SD826 with respect to the concentration of each main component and estimated conversion ratio from isophthalonitrile which was a reaction starting material, and the reduction ratio of by-products owing to the use of SD826 strain based on ATCC39484 are shown in Table 2 below.<br><br><br>
Table 2<br><br>
41<br><br>
42<br>
[Example 3]<br>
Rhodococcus sp. SD826 was streaked in an LB agar culture medium and cultured in a constant temperature bath at 30°C for 24 hours.    From the colonies generated, one loopful of cells were picked up and suspended in 100 ml of an LB liquid medium in a 500 mL-volume baffled flask.    The flask was placed in a constant temperature rotary shaker at 30°C and cultured with 120 revolutions per minute for 24 hours.    The microorganism cells obtained were recovered by the centrifugation of 10,000 g and then suspended in a 50 mM sodium/potassium phosphate buffer solution (pH: 7) isovolume to the culture solution.<br>
To the cell suspension, isophthalonitrile in an amount corresponding to 1% (mass /volume) and 0.1% (mass/volume) of benzonitrile as an inductive substrate were added.    Then, the suspension was placed in a constant temperature rotary shaker at 30°C and the reaction was performed with 120 revolutions per minute for 72 hours The resulting reaction solution was adjusted to a pH of 2 using 2 mol/1 of hydrochloric acid, added to ethyl acetate isovolume to the reaction solution, stirred, and then extracted.    The ethyl acetate layer obtained was appropriately diluted and analyzed by reverse phase HPLC (column: Shodex DS-613, eluant: 50% acetonitrile/5 mM potassium phosphate buffer solution, pH: 3 0, flow rate: 1 mL/min, detection: UV 240 nm). In the reaction solution, main ingredients having a peak of which retention time was coincident with the 4-cyanobenzoic acid sample were found.    The peak ingredients were collected and subjected to GC-mass spectral analysis and as a result, each was verified to delineate a fragment pattern suggesting the same structure as the sample.<br>
As a comparative example, the reaction, extraction, and analysis were<br><br>
43<br>
performed in the same manner as above except for using a parent strain ATCC39484. The main product in the reaction solution of the parent strain under the above-described HPLC conditions was subjected to LC-MS analysis and then identified and quantified.<br>
The comparison between the reaction solution of the parent strain and the reaction solution of SD826 with respect to the concentration of each main ingredient and the estimated conversion ratio from isophthalonitrile, which was a reaction starting material, and the reduction ratio of by-products owing to the use of SD826 strain based on ATCC39484 are shown in Table 3 below.<br><br><br>
nd.: Not detected<br><br>
Table 3<br><br>
44<br><br>
45<br>
[Example 4] Preparation of Chromosomal DNA for Preparing Nitrilase Gene<br>
Rodococcus sp. ATCC39484 strain (hereinafter, referred to "R. sp.") was cultured a while day and night in an agar plate culture medium prepared by adding 2% of agar to an L broth (polypeptone: 1%, NaCl: 0.5%, yeast extract: 0.5%, pH: 7.0), and one loopful of cells thereof was cultured at 30°C for 24 hours in 300 ml of a culture medium prepared by adding 5 g/1 of glucose and 2 g/1 of urea to a base culture medium (KH2PO4. 1.5 g/1, Na2HPO42H20: 0.75 g/1, MgSO47H2O: 0.2 g/1, CaSO42H2O: 10 mg/1, FeSO47H2O: 5 mg/1, yeast extract: 20 mg/1).    The incubated cells were harvested and washed with 100 ml of 5 mM EDTA solution.    The resulting cells were suspended in 30 ml of a buffer solution (20 mM Tris hydrochloric acid buffer solution (pH: 7.1)), 60 mg of lysozyme was added thereto, and the suspension was incubated at 37°C for 2 hours.    This suspension solution was centrifuged (5,000 rpm, 7 minutes) to recover the cells.    The recovered cells were re-suspended in 11.34 mL of TE buffer, 0.6 ml of 10% SDS was added, proteinase R (produced by Merck) was added, and the mixture was gently shaken at 55°C for 1 hour.    This solution was extracted with phenol and precipitated with ethanol to prepare chromosomal DNA.<br>
[Example 5]    Construction of DNA library<br>
20  g of the chromosomal DNA obtained in Example 4 was subjected to partial digestion using a restriction enzyme Sau 3A1.    More specifically, the chromosomal DNA was charged into 5 tubes in an amount of 4  g per tube and the restriction enzyme Sau 3 AT (produced by Takara Shuzo Co., Ltd., from 4 to 12 U/ l) was added to each individual tube and reacted at 37°C in a reaction volume of 100  l<br><br>
46<br>
Every 10 seconds, one tube was taken up and the reaction was stopped by adding EDTA so as to have a final concentration of 20 mM.    The thus-prepared partial digestion fragment solution of chromosomal DNA was electrophoresed with agarose gel, and from 1 to 2 kb of the DNA fragment was recovered through electrophoresis extraction and precipitation with ethanol     The DNA fragment recovered was then dissolved in 30  l of TE solution.    9  l of this sample and 1  g of pUC18 (produced by Takara Shuzo Co., Ltd ) subjected to digestion with BamHI and to a BAP treatment were ligated using T4DNA ligase (ligation kit ver. 2, produced by Takara Shuzo Co., Ltd.) to make 20  l and thereafter Escherichia coli JM101 strain was transformed.    In order to prepare an amplified library from the library obtained, the Escherichia coli transformants were implanted by every 20 colonies on an L broth containing 50 ppm of ampicillin and cultured a whole day and night.    From the cells, a piasmid was extracted by an alkali-SDS method<br>
[Example 6]    Anchor pCR method<br>
In advance of cloning, anchor pCR was performed to obtain an enzyme gene partial fragment for use as a probe.    One primer which was derived from the enzyme sequence was prepared by selecting a sequence having an appropriate Tm from known N-terminal sequences of this enzyme.    That is, 5'-gct gcg gtg cag gca-3' (and complementary strand thereof) Tm  52°C<br>
The PCR was performed under the following reaction conditions. Composition of Reaction Solution:<br><br>
47<br><br>
Shuzo Co., Ltd.)<br>
Total 50  l Reaction Conditions:<br>
Denaturing: 94°C, 45 seconds<br>
Annealing: 37 to 55°C, 60 seconds<br>
Elongation: 72°C, 60 to 90 seconds<br>
Number of cycles: 24 times<br>
In the thus-performed reaction, a reaction solution found to have a fragment specifically amplified was subjected to 2% agarose gel electrophoresis and the region containing the fragment was cut off and purified using EASYTRAP ver. 2 (produced by Takara Shuzo Co., Ltd.).    The DNA sequence of each of the DNA fragments obtained was determined by the dideoxy method.    Those having a translated amino acid sequence homologous to the nitrilase N-terminal sequence of the R. sp. ATCC39484 strain were sought.    As a result, an about 900 bp fragment containing the DNA sequence coding for nitrilase 287 amino acid was found in the fragments obtained<br><br>
48<br>
[Example 7]    Colony Hybridization<br>
By using as a probe the PCR fragment obtained in Example 3 containing a part of the nitrilase gene, all genes were cloned by a colony hybridization method.    The partial digestion fragment solution of chromosomal DNA degraded by Sau 3Al according to the method of Example 6 was subjected to 1% agarose gel electrophoresis, and a 4 to 8 kb DNA fragment was recovered through electrophoresis extraction and precipitation with ethanol.    This fragment was dried and dissolved in 30  l of TE solution,    9  1 of this sample solution and pUCl8 (produced by Takara Shuzo Co., Ltd., 100 ng) subjected to digestion with 1 g of BamHI and to a BAP treatment were ligated using T4DNA ligase (ligation kit ver 2, produced by Takara Shuzo Co , Ltd ) and thereafter Escherichia coli JM101 strain was transformed.    The transformants were spread on an agar plate culture medium prepared by adding 2% of agar to an L broth containing 0.1 mM of isopropyl- -D-thiogalactopyranoside (IPTG), 0.004% of 5-bromo-4-chIoro-3-indolyl- -D-galactopyranoside (X-gal) and 50 ppm of ampicillin, and cultured at 37°C a whole day and night.<br>
The white colonies generated were picked up onto an agar plate culture medium prepared by adding 2% of agar to an L broth containing 50 ppm of ampicillin, and cultured at 37°C a while day and night.    After full growing, the agar plate culture medium was placed at 4°C for about 2 hours to become chilled,    A dry nylon membrane (Hybond-N+, produced by Amersham Pharmacia Biotech) was marked at the top, down, left, and right and then carefully placed on to a surface of the agar in contact with the colonies.    After the membrane was entirely wetted, the membrane was removed from the agar surface in a single continuous movement to transfer the colonies<br><br>
49<br>
on the plate to the membrane.    When the number of cells transferred is small, the membrane was placed on an agar plate culture medium prepared by adding 2% of agar to L broth containing 50 ppm of ampicillin, and cultured at 37°C a whole day and night.<br>
The membrane having the cells transferred thereon was floated on 3 ml of an alkali solution (0.5M NaOH) to dissolve the cells.    The undissolved residual cells were washed out from the membrane with 5xSSC for 20 minutes x 2 times.    To this membrane, colony hybridization was applied using Random prime DNA labeling and detection system (produced by Amersham Pharmacia Biotech).    The detection by hybridization was performed under standard conditions according to the specification attached to the kit.    As a result of hybridization performed on about 4,000 colonies, two strains were obtained as a positive clone.<br>
From these positive clones, plasmids were extracted by an alkali-SDS method. The position of the cleavage site by the restriction enzyme in the partial fragment used as the probe was compared with the restriction digestion pattern of each plasmid, and therefrom the position and the direction of genes in the insertion fragment were estimated.    As a result, the plasmids pNL06 and pNL09 prepared from two clone strains both were found to contain total nitrilase genes (see, Fig. 1).    Using the P09 strain plasmid (pNL09) having a larger insertion fragment length, the DNA sequence of the insertion fragment of about 2.6 kb was determined.    A portion homologous to the partial fragment sequence used as the probe was sought and as a result, it was found that a nitrilase gene was present in the positive direction with respect to the lac promoter from a portion about 300 bp downstream from the insertion fragment end.    The thus-found direction and position agreed with the position and direction of the gene estimated from the cleavage site by the restriction enzyme.    The amino acid sequence<br><br>
50<br>
translated from this nitrilase gene sequence was novel and different from the amino acid sequence of any known nitrilase.<br>
[Example 8]    Measurement of Nitrilase Activity<br>
The nitrilase activity was measured as follows.    The cells were added to a reaction solution obtained by suspending from 1 to 10 mass% of terephthalonitrile (TPN) as a substrate in 10 ml of 20 mM phosphate buffer solution (pH: 7.0) and reacted at 30°C while shaking, and the p-cyanobenzoic acid produced in the reaction solution was quantitated by HPLC at fixed intervals.    The solid matter was removed from the reaction solution by centrifugation and the supernatant 100-fold diluted with the eluant was used as the HPLC sample.    The apparatus and the conditions for the quantification of p-cyanobenzoic acid are shown below. Apparatus:<br>
Pump:	DS-2 (Shodex)<br>
Detector:	SPD-6AV UV-VIS spectrophotometer<br>
(Shimadzu)<br>
Introduction of sample:     Autosampler Model 23 (SIC)<br>
with 20 ml sample tube<br>
Recording:	Chromatocoder 12 (SIC)<br>
Column:	ODSpakF-411 (Shodex), 4.6x150 mm, 40°C<br>
Separation conditions:<br>
AcCN/H2O = 50 50, 0.1% TFA, 1 ml/min.<br>
The activity was shown by the mass of p-cyanobenzoic acid when cells in a dry mass of 1 g were produced in 1 I of the reaction solution within 1 hour (unit g/l/hr/g dry<br><br>
51<br>
cells).<br>
[Example 9]    Preparation of High Expression Strain<br>
The positive clone P09 strain obtained in Example 4 was cultured in an L broth containing 50 ppm of ampicillin and as a result, nitrilase activity was confirmed irrespective of the presence or absence of isopropyl- -D-thioga!actopyranoside (IPTG). However, this activity was as low as a few tenths of the Rhodococcus microorganism that was a donor     In the P06 strain, the nitrilase activity was not observed at all.<br>
In order to increase the production of the enzyme, two kinds of fragments, one containing only the enzyme structural gene portion and another containing the enzyme structural gene and the region of about 1.3 kb downstream there from were prepared by PCR.    Using these, plasmids pUNLEl and pUNLE2 each ligated immediately after the lac promoter of pUC18 were prepared.    The primers and the reaction conditions used for the preparation of pCR fragments are shown below. pUNLEl (forward)<br>
5'-aac atg gtc gaa tac aca aac-3' (reverse)<br>
5'-cc aag ctt tea gag ggt ggc tgt-3' Hindlll site pUNLE 2<br>
(forward) the same as pUNLEl (reverse) Ml3 primer M4 Composition of Reaction Solution<br><br>
52<br>
Plasmid DNA	0.8 to 1  g<br>
Primers	each 100 pmol<br>
dNTP Solutions	each 1 mM<br>
10x Reaction buffer	10  l<br>
EXTaqDNA Polymerase (produced by         2.5 U<br>
Takara Shuzo Co., Ltd.)<br>
Total 50  1 Reaction conditions:<br>
Denaturing:	94°C, 60 seconds<br>
Annealing:	55°C 60 seconds<br>
Elongation:	72°C, 120 seconds<br>
Number of cycles:	24 times<br>
The fragments produced were subjected to agarose gel electrophoresis and recovered by extraction.    Each of the fragments was cut at the Hindlll and Ncol site, ligated with EcoRINcol linker, and then ligated with pUCl 8 cleaved at EcoRI and Hindlll (see, Fig. 2).    With these plasmids, Eschehchia coji JM109 strain was transformed.    The transformants obtained each was cultured in an L broth containing 50 ppm of ampicillin a whole day and night, and after adding isopropyl- -D-thiogalacto-pyranoside (IPTG) to the culture solution to a concentration of 0.1 mM, was further cultured for 2 hours.    The transformants obtained were measured on the nitrile conversion activity by the method described in Example 5     As a result, the transformants obtained by the transformation with any plasmid were verified to have a nitrilase activity as high as about 500 times the pUNL09 transformant and about 80<br><br>
53<br>
times the Rhodococcus microorganism which was a donor (see, Table 4)<br>
Table 4<br><br>
Activity unit: g/l/hr/g dry cells<br>
[Example 10]    Production of p-Cyanobenzoic Acid Using High Activity Strain<br>
The pUNLEl transformant obtained in Example 9 was cultured a whole day and night in an agar plate culture medium prepared by adding 2 % of agar to an L broth containing 50 ppm of ampicillin, and the grown cells were inoculated with an inoculating loop in 100 ml of an L broth containing 100 ppm of ampicillin and cultured under shaking at 37°C.    This culture solution was subcultured in a 5 L-volume jar fermenter filled with 2 I of L broth containing 100 ppm of ampicillin and cultured with aeration and stirring a whole day and night under conditions of 37°C, 800 rpm agitation, and an aeration rate of 1 1/min.    To the cell culture solution at the initial stage of stationary phase or at the final stage of logarithmic growth phase, isopropyl- -D-<br><br>
54<br>
thiogalactopyranoside (IPTG) was added so as to have a final concentration of 0 1 mM, and the culturing was further continued for 4 hours.<br>
The culture solution was centrifuged and the cells obtained were again suspended in 1 I of 20 mM phosphate buffer solution (pH: 7.0).    Thereto, 100 g of terephthalonitrile (TPN) was added and reacted at 35°C while stirring.    A part of the reaction solution was sampled at intervals of one hour and the p-cyanobenzoic acid produced in the reaction solution was quantified by the method described in Example 5. The p-cyanobenzoic acid was quickly produced by the transformant and accumulated in a proportion of 3% in the reaction solution within about 3 hours (see, Fig. 3).    After the completion of reaction, concentrated hydrochloric acid was added to the reaction solution to adjust the pH to 1 and thereby precipitate the p-cyanobenzoic acid.    The precipitate was filtered through a filter paper, washed with dilute hydrochloric acid (0.1 mol/1) , and then vacuum dried.    The thus-obtained dry sample had a purity of 99.9% or more.    The impurity detected was the starting material terephthalonitrile.<br>
[Example 11]    Preparation of Chromosomal DNA for Preparing Nitrile Hydratase Gene and Amidase Gene<br>
R. sp. ATCC 39484 strain was cultured a whole day and night in a nutrient (L broth) agar plate culture medium, and a loopful of cells thereof was cultured at 30°C for 24 hours in 300 ml of a culture medium prepared by adding 5 g/1 of glucose and 2 g/1 of urea to a base culture medium (KH2PO4: 1.5 g/1, Na2HPO42H20. 0 75 g/1, MgSO47H2O: 0.2 g/1, CaSO42H2O: 10 mg/1, FeSO47H2O: 5 mg/1, yeast extract: 20 mg/1)     The incubated cells were harvested and washed with 100 ml of 5 mM EDTA solution.    The resulting cells were suspended in 30 ml of a buffer solution (20 mM Tris hydrochloric<br><br>
55<br>
acid buffer solution (pH: 7.1)), 60 mg of lysozyme was added thereto, and the suspension was incubated at 37°C for 2 hours.    This suspension solution was centrifuged (5,000 rpm, 7 minutes) to recover the cells.    The recovered cells were re-suspended in 11.34 mL of TE buffer, 0.6 ml of 10% SDS was added proteinase R (produced by Merck) was added to a concentration of 100 u.g/ml, and the mixture was gently shaken at 55°C for 1 hour    This solution was extracted with phenol and precipitated with ethanol to prepare chromosomal DNA.<br>
[Example 12]    Construction of a DNA library<br>
The 20  g of the chromosomal DNA obtained was subjected to partial digestion using a restriction enzyme Sau 3AT.    More specifically, the chromosomal DNA was charged into 5 tubes in an amount of 4  g per tube, and the restriction enzyme Sau 3AI (produced by Takara Shuzo Co., Ltd., from 4 to 12 U/ 1) was added to each individual tube and reacted at 37°C in a reaction volume of 100  l.    Every 10 seconds, one tube was taken up and the reaction was stopped by adding EDTA to have a final concentration of 20 mM.    The thus-prepared partially digested fragment solution of chromosomal DNA was electrophoresed with agarose gel, and from 5 to 10 kb of the DNA fragment was recovered through electrophoresis extraction and precipitation with ethanol.    The recovered DNA fragment was then dissolved in 30  1 of TE solution.    9  l of this sample and 1 fig of pUC18 (produced by Takara Shuzo Co , Ltd ) subjected to digestion with BamHI and BAP treatment were ligated using T4DNA ligase (ligation kit ver. 2, produced by Takara Shuzo Co , Ltd ) to yield 20  l and thereafter Eschehchia coli JM 101 strain was transformed     In order to prepare an amplified library from the<br><br>
56<br>
library obtained, the Escherichia coli transformants were implanted every 20 colonies on an L broth (pH 7.0) containing 50 ppm of ampicillin and cultured a whole day and night.    From the cells, a plasmid was extracted by an alkali-SDS method.<br>
[Example 13]    Purification of Nitrile Hydratase and Amidase<br>
One primer derived from the enzyme sequence necessary for anchor PCR was prepared from the N-terminal sequence of the enzyme peptide prepared as follows by selecting the sequence such that the primer has a suitable Tm.<br>
The nitrile hydratase activity or amidase activity were each qualitatively determined by allowing 1 ml of a reaction mixture containing 10 mM benzonitrile or 10 mM benzamide, 30 mM potassium phosphate buffer (pH 7.0), and a predetermined amount of cell extract to react at 25°C for 30 minutes and then detecting the produced benzamide or benzoic acid by HPLC (the HPLC separation conditions were the same as those in Example 8 described above)<br>
R. sp. ATCC39484 strain was inoculated in 600 ml of a nitrile decomposition enzymes inducing medium consisting of the basic medium of Example 1 and 1 g/t of benzonitrile as an induction substrate, and cultured with shaking at 30°C     The culture solution cultured a whole day and night was subjected to centrifugal ion (8,000 rpm, 15 minutes) to recover the cells, and 3 2 g in wet weight of the obtained cells were washed with 50 ml of 100 mM potassium phosphate buffer (pH 7.0, containing 1 mM EDTA and 2 mM DTT), and thereafter, suspended in 200 ml of the same buffer     This was subjected super sonicator to destroy the cells, followed by centrifugation (12,000 rpm, 20 minutes) to obtain 180 ml of a supernatant (crude enzyme extract solution)<br>
Ammonium sulfate was added to this cell-free extract solution to a 45%<br><br>
57<br>
saturation concentration, the mixture was stirred at 4°C for 1 hour, and then the generated precipitates were removed by centrifugation.    Further, ammonium sulfate was added to the separated supernatant to a 60% saturation concentration, the mixture was stirred at 4°C for 1 hour, and thereafter the precipitates were recovered by centrifugation.    The generated precipitates were confirmed to exhibit nitrile hydratase activity and amidase activity     The obtained precipitates were dissolved in 10 ml of a 100 mM potassium phosphate buffer (pH 7.0, containing 1 mM EDTA and 2 mM DTT), and the solution was dialyzed against the same buffer.<br>
The dialyzed crude enzyme solution was charged in a DEAE-Sepharose column (2 cm x 20 cm) equilibrated with a 100 mM potassium phosphate buffer (pH 7.0, containing 1 mM EDTA and 2 mM DTT) and washed with the equilibrated buffer until the UV absorption at 280 nm of the eluate decreased.    Subsequently, it was further washed with the same buffer but supplemented with 0.1 M KC1 until the UV absorption at 280 nm of the eluate decreased     Thereafter, nitrile hydratase and amidase were eluted with a 100 mM potassium phosphate buffer (pH 7.0, containing 1 mM EDTA and 2 mM DTT) with KC1 concentration being increased to 0.3 M.    Fractions showing the activity were collected and the enzyme protein was concentrated using an ultrafiltration membrane (molecular weight 30,000 cut).<br>
In Phenyl Sepharose CL-4B column (2 cm x 40 cm) equilibrated with a 100 mM potassium phosphate buffer (pH 7.0, containing 10% saturation concentration of ammonium sulfate), and a mixture of the concentrated active fraction and 10% saturation concentration of ammonium sulfate was charged to allow the enzyme to be adsorbed thereon.    Then, the column was washed with the equilibration buffer until the UV absorption at 280 nm of the eluate decreased.    Thereafter, nitrile hydratase and<br><br>
58<br>
amidase were eluted with the elution buffer (100 mM potassium phosphate buffer (pH 7.0)).    The active fractions were collected and the enzyme protein was concentrated using ultrafiltration membrane (molecular weight 30,000 cut).<br>
The concentrated nitrile hydratase active fraction was charged in a Sepharcryl S-300 Superfine Column (2 cm x 60 cm), equilibrated with a 100 mM potassium phosphate buffer (pH 7.0, containing 0.5 M NaCl), and separation was performed using the same buffer, thus fractionating the eluate into about 0.5 ml fractions.    In this stage, different fractions indicated the maximal nitrile hydratase and amidase activities, respectively so that the fraction showing the highest enzyme activity and the neighboring fractions were recovered for each enzyme.    About 1.5 ml each of fraction was concentrated using an ultrafiltration membrane (molecular weight 30,000 cut).<br>
[Example 14]    Determination of Peptide Terminal Sequence<br>
Determination of the N-terminal sequences of the obtained nitrile hydratase and amidase was tried, but both enzymes showed low signal intensity in Edman decomposition so that the determination of sequences was unsuccessful.    Accordingly, the enzyme protein was hydrolyzed by a cyanogen bromide (BrCN) and the produced peptides were separated under the following liquid chromatography conditions.<br>
Body;	LC 9A (Shimadzu Seisakusho)<br>
Column;	Asahipak ODP 50 6D (Shodex)<br>
Column temperature;         25°C<br>
Eluant,	Acetonitrile 0 to 80% (linear<br>
Concentration gradient, 60 minutes) 0.1% Trifluoroacetic acid<br><br>
59<br>
Flow rate: 0.5 ml/min.<br>
Detection,	SPD-6AV UV VIS Spectro Photometer<br>
(Shimadzu Seisakusho) 215 nm<br>
Of the plurality of peptides obtained from the nitrile hydratase active fractions, those samples which showed relatively good separation were selected and subjected again to N-terminal sequence analysis by Edman decomposition.    As a result, the following sequence having a high homology with the existing nitrile hydratase sequence was confirmed. Glu(E) o Tyr(Y) o Arg(R) o Ser(S) o Arg(R) o Val(V) o Val(V)<br>
Taking into consideration this sequence and the codon usage of Rhodococcus bacteria, a primer for nitrile hydratase was prepared. 5'-GAG TAC CGG TCC CGA-3' (and complementary strand thereof)<br>
Similarly, of the plurality of peptides obtained from the amidase active fractions, those fractions showing relatively good separation were selected and subjected again to N-terminal sequence analysis by Edman decomposition.    As a result, the following sequence having a high homology with the existing amidase sequence was confirmed. Ala(A) o Val(V) ¦ Gly(G) o Gly(G) o Asp(D) o Gln(Q) o Gly(G)<br>
Taking into consideration this sequence and the codon usage of Rhodococcus bacteria, a primer for amidase was prepared. 5'-GCA GTC GGC GGC GAC-3' (and complementary strand thereof)<br><br>
60<br>
[Example 15]    Anchor PCR method<br>
The PCR method was performed under the following reaction conditions: Composition of Reaction Solution:<br>
R. sp. ATCC39484 chromosomal DNA<br>
library	1  g<br>
Universal primer	100 pmol<br>
Enzyme peptide N-terminal primer	100 pmol<br>
dNTP Solution	each      1 mM<br>
10x Reaction buffer	10  1<br>
EXTaqDNA Polymerase (produced by	2.5 U<br>
Takara Shuzo Co., Ltd )<br>
Total 50  l Reaction Conditions:<br>
Denaturing: 94°C, 45 seconds Annealing: 37 to 60°C, 60 seconds Elongation 72°C, 60 to 90 seconds Number of cycles: 24 times<br>
In the thus-performed reaction, a reaction solution found to have a fragment specifically amplified was subjected to 2% agarose gel electrophoresis and the region containing the fragment was cut off and purified using EASYTRAP ver. 2 (produced by Takara Shuzo Co., Ltd.).    Each of the DNA fragments obtained was determined on the DNA sequence by the dideoxy method to confirm that the translated amino acid sequence have homology to the known nitrile hydratase or amidase     As a result, it was<br><br>
61<br>
revealed that obtained fragments 4 and 14 contained sequences having a high homology with the known nitrile hydratase and amidase, respectively.    Fragment 4 contained an about 500 bp a nitrile hydratase homologous sequence and fragment 14 contained an about 900 pb amidase homologous sequence.    Both had sufficient lengths for serving as a probe for use in the subsequent colony hybridization.<br>
[Example 16]    Colony Hybridization<br>
By using as a probe the PCR fragments containing a part of the nitrile hydratase gene and the PCR fragments containing a part of amidase gene, obtained in Example 15, full genes were cloned by a colony hybridization method.    The partial digestion fragment solution of chromosomal DNA degraded by Sau 3AI according to the method of Example 1 was subjected to 1% agarose gel electrophoresis, and a 4 to 8 kb DNA fragment was recovered through electrophoresis extraction and precipitation with ethanol.    This fragment was dried and dissolved in 30  l of TE solution.    9  l of this sample solution and 1  1 of pUC18 (produced by Takara Shuzo Co., Ltd., 100 ng) subjected to digestion with BamHI and BAP treatment were ligated using T4DNA ligase (ligation kit ver. 2, produced by Takara Shuzo Co., Ltd ), and thereafter Escherichia coli JM101 strain was transformed.    The transformants were spread on an agar plate culture medium prepared by adding 2% of agar to an L broth containing 0.1 mM of isopropyl-p-D-thiogalactopyranoside (IPTG), 0.004% of 5-bromo-4-chloro-3-indolyl- -D-galactopyranoside (X-gal) and 50 ppm of ampicillin, and cultured at 37°C a<br>
whole day and night.<br>
The white colonies generated were picked up onto an agar plate culture medium prepared by adding 2% of agar to an L broth containing 50 ppm of ampicillin,<br><br>
62<br>
and cultured at 37°C a whole day and night.    After full growing, the agar plate culture medium was placed at 4°C for about 2 hours so as to be chilled.<br>
A dry nylon membrane (Hybond-N+, produced by Amersham Pharmacia Biotech) was marked with a pencil at the top, down, left and right, and then carefully placed on to a surface of the agar in contact with the colonies.    After the membrane was entirely wetted, the membrane was gently removed from the agar surface, and in a single continuous movement the colonies on the plate were transferred to the membrane. When the number of cells transferred is small, the membrane was placed on an agar plate culture medium prepared by adding 2% of agar to L broth containing 50 ppm of ampicillin, and cultured at 37°C a whole day and night.<br>
The membrane having transferred thereon the celts was floated on 3 ml of an alkaline solution (0.5M NaOH) to dissolve the cells.    The undissolved residual cells were washed out from the membrane with 5xSSC for 20 minutes x 2 times.    To this membrane, colony hybridization was applied using Random prime DNA labeling and detection system (produced by Amersham Pharmacia Biotech).    The detection by hybridization was performed under standard conditions according to the specifications attached to the kit.    As a result of hybridization performed on about 8,000 colonies, one strain for each gene was obtained as a positive clone.<br>
From these positive clones, plasmids were extracted by an alkali-SDS method. The position of the cleavage site by the restriction enzyme in the partial fragment used as the probe was compared with the restriction digestion pattern of each plasmid, and therefrom the position and the direction of genes in the insertion fragment were estimated.    As a result, the plasmid UNH11 prepared from cloned strain Pll of nitrile hydratase was revealed to contain the whole nitrile hydratase gene (see, Fig. 4), while<br><br>
63<br>
the plasmid pUAMD12 prepared from the cloned strain P12 of amidase showed complex restriction enzyme treatment pattern and the position and direction of the gene could not be determined.    Accordingly, pUAMD12 alone was further treated with restriction enzyme and the obtained fragments were subjected to Southern hybridization. From the results, it was presumed that this also contained the whole structural gene regions (Fig. 4).<br>
[Example 17]    Preparation of Deletion Mutant and Determination of Base Sequence<br>
pUNHll and pUAMD12 are plasmids each containing an inserted fragment of 3 kb to 4 kb, and their base sequence was difficult to determine.    Thus, preparation of deletion mutants from which the inserted fragment was deleted from the terminal using exonuclease HI was tried.    For preparing the mutants, Deletion Kilo-Sequence Kit (Takara Shuzo Co., Ltd.) was used.    That is, 25  l (about 16  g for 0.4 mg/ml) of pUNHll or pUAMD 12 solution was fully digested with Sse83871 and Xbal (37°C, 24 hours), purified by extraction with phenol, and precipitated by addition of 1/10 volume of 3 M Na acetate and 2 5 volumes of ethanol.    The precipitates were centrifuged and recovered and once washed with 70% cold ethanol.    Thereafter, the precipitates were dried under vacuum.    The vacuum dried precipitates were dissolved in 100  1 of Exo III buffer.     1  1 of Exonuclease III was added to the DNA solution and the mixture was stirred using a vortex, and thereafter incubated at 37°C.    After 10 seconds and 30 seconds, each 50  l of the reaction mixtures was sampled (mixed with 50  l of a MB nuclease buffer prepared in advance to stop the reaction).<br>
2  l of MB nuclease was added to the reaction mixture and incubated at 37°C<br><br>
64<br>
for 20 minutes.    After completion of the reaction, the reaction mixture was extracted with phenol for purification, and 1/10 volume of 3 M Na acetate and 2 5 volumes of ethanol were-added to form precipitates.    The precipitates were recovered by centrifugation, washed once with 70% cold ethanol, and then dried under vacuum. The thus-obtained precipitates were dissolved in 50  l of Klenow buffer and 1  l of Klenow fragment was added thereto, followed by incubation at 37°C for 15 minutes The reaction mixture was subjected to agarose gel electrophoresis to fractionate into three strand length ranges (each was cut out from the gel, extracted and recovered).<br>
10  1 of recovery solution of the cut out fragment was mixed with 100  l of ligation solution A, 12  l of ligation solution B was added thereto, and the mixture was stirred at 16°C for 24 hours using a vortex to cause self ligation.    The obtained plasmid was used to transform the Escherichia coli JM109 strain.<br>
By the above operation, 20 or more deletion mutants could be obtained for pUNH11.    From these, 7 mutants containing insertion fragments with suitable lengths were selected and used for the determination of sequence.    However, for pUAMD12, it was revealed that suitable deletion mutants could not be obtained because it generated plasmids greater than the original plasmid or plasmids smaller than the vector used. Accordingly, for pUAMD12, sequence determination by a gene walking method, which determines the sequence while the primer is sequentially synthesized, was performed. The determination of a base sequence was performed according to a dideoxy method by using an about a 2 8 kb DNA sequence corresponding to the whole range of the inserted fragment for pUNH11, and an about 2.8 kb DNA sequence corresponding to about 2/3 of the inserted fragment for pUAMD12     Portions identical with partial fragment sequences used as a probe were searched and as a result it was revealed that<br><br>
65<br>
about 0.2 kb and about 1 1 kb downstream from the insertion fragments of pUNHll and pUAMD12 on the EcoRI site, the nitrile hydratase gene was present in a reverse direction to the lac promoter while the amidase gene was present in a forward direction thereto.    The results of analysis of base sequences are shown by SEQ ID NO 3 and 6. The thus-found direction and position agreed with the position and direction of the gene estimated from the cleavage pattern by the restriction enzyme with respect to pUNH11, and agreed with the position and direction of the gene estimated from the cleavage pattern by the restriction enzyme, and the results of Southern hybridization with respect to pUAMD12.    The amino acid sequences (SEQ ID NOs 4, 5 and 7) translated from these gene sequences were novel and different from the amino acid sequences of any known nitrile hydratase and amidase.<br>
[Example 18]    Measurement of Nitrile Hydratase and Amidase Activities<br>
The nitrile hydratase activity was measured as follows.    The cells (about 1 g by wet mass) were added to a reaction solution obtained by suspending from 1 to 10 mass% of terephthalonitrile (TPN) as a substrate in 10 ml of 20 mM phosphate buffer solution (pH: 7.0) and reacted at 30°C while shaking, and the p-cyanobenzoic acid amide produced in the reaction solution was quantitated by HPLC at fixed intervals. The solid matter was removed from the reaction solution by centrifugation, and the supernatant 100-fold diluted with the eluant was used as the HPLC sample.    The amidase activity was measured by using p-cyanobenzamide or benzamide serving as a substrate, performing the reaction under the same conditions as above, and determining the generated p-cyanobenzoic acid or benzoic acid by HPLC<br>
The products were determined using the apparatus and the conditions below:<br><br>
66<br>
Apparatus<br>
Pump:	DS-2 (Shodex)<br>
Detector.	SPD-6AV UV-VIS spectrophotometer<br>
(Shimadzu)<br>
Introduction        Autosampler Model 23 (SIC) with<br>
of sample	20  1 sample tube<br>
Recording:	Chromatocoder 12 (SIC)<br>
Column:	ODSpakF-411 (Shodex), 4.6x150 mm, 40°C<br>
Separation conditions:<br>
AcCN/H2O = 50:50, 0 1% TFA, 1 ml/min.<br>
The activity was shown by the mass of p-cyanobenzoic acid amide, p-cyanobenzoic acid, or benzoic acid when cells in a dry mass of 1 g were produced in 1 1 of the reaction solution within 1 hour (unit: g/l/hr/g dry cells).<br>
[Example 19]    Preparation of High Expression Strain<br>
The positive clone P11 strain or P12 strain obtained in Example 16 was cultured in an L broth containing 50 ppm of ampicillin, and as a result, nitrile hydratase activity was confirmed irrespective of the presence or absence of isopropyl- -D-thiogalactopyranoside (IPTG).    However, this activity was as low as a few tenths of the Rhodococcus microorganism that was a donor.    In the P12 strain, the amidase activity was not observed at all.<br>
In order to increase the production of the enzyme, fragments of only the enzyme structural gene portions were prepared by PCR and ligated immediately after the lac promoter of pUC18 to prepare plasmids pUNHEl and pUAMDEl.    Further,<br><br>
67<br>
plasmid pUNHAMDEl having the both fragments on the same plasmid was prepared.<br>
The primers and the reaction conditions used for the preparation of PCR fragments are shown below: pUNLEl (forward)<br>
5'-acc atg gat ggt atc cac gac-3' ((3 subuit initiation codon)(NcoI site) (reverse)<br>
5'-cc aag ctt tca tac gat cac ttc-3' (a subuit stop codon)(HindIII site) pUAMDEl (forward)<br>
5'-acc atg gct tcg ttg act cc-3' (NcoI site, mutation of amino acid 3 Ser--&gt;Ala) (reverse)<br>
5'-cc aag ctt tca gga cgg cac cga-3' (HindIII site)<br><br><br>
68<br>
By Takara Shuzo Co., Ltd.)<br>
Total 50 \x\ Reaction conditions:<br>
Denaturing	94°C, 60 seconds<br>
Annealing:	55°C. 60 seconds<br>
Elongation:	72°C, 120 seconds<br>
Number of cycles	24 times<br>
For both the nitrile hydratase gene and amidase gene, the fragments produced were subjected to agarose gel electrophoresis and recovered by extraction.    The fragments each was cut at Ncol and HindIII sites, ligated with EcoRINcol linker, and then ligated with pUC18 cleaved at EcoRI and Hindlll (see, Fig. 5).<br>
The plasmid on which the both the nitrile hydratase gene and the amidase gene are present was first digested with restriction enzymes Ncol and Hindlll to cleave the nitrile hydratase fragment, the fragments were ligated with EcoRINcol linker, HindIII-Ncol linker in order, thereafter ligated with the amidase fragment cleaved with Ncol and Hindlll, and finally the resulting fragment was ligated with pUC18 cleaved with EcoIR-HindIII (Fig. 6).<br>
With these plasmids, Eschehchia coli JM109 strain was transformed.    Each of the    transformants obtained was cultured in an L broth containing 50 ppm of ampicillin over a twenty-four our period, and after adding isopropyl- -D-thiogalactopyranoside (IPTG) to the culture solution to a concentration of 0.1 mM, further cultured for 2 hours. The transformants obtained were measured on the nitrile conversion activity by the method described in Example 8     As a result, the transformants obtained by the transformation with any plasmid were verified to have a nitrilase activity higher than<br><br>
69<br>
that of the Rhodococcus bacterium, the donor.    Only the transformant transformed with a plasmid having the both genes thereon showed activity as high as that of the donor. The results are summarized in Table 5 below.<br>
Table 5<br><br>
Activity unit: g/l/hr/g dry cells<br>
1)	The activity of donor measured was amide generation rate only (acid<br>
generation rate was impossible to accurately measure due to the influence of nitrilase.)<br>
2)	For pUNHAMDEl, the generation rate of acid from nitrile was measured<br>
DDESCR1PTI0N OF DEPOSIT<br>
The following microorganism has been deposited at National Institute of Bioscience and Human-Technology, Agency of Industrial Science and Technology,<br><br>
70<br>
Ministry of International Trade and Industry (1-3, Higashi 1-chome Tsukuba-shi Ibaraki-ken, Japan).<br>
Microorganism	Accession Number	Date of Deposition<br>
Rhodococcus sp  SD826   PERM BP-7305	October 12, 1999<br>
The deposited microorganism has been deposited under the provisions of Budapest Treaty on International Recognition of the Deposit of Microorganisms for the Purposes of Patent Procedure and Rules based thereon.<br><br>
71<br>
WE CLAIM :<br>
1.	A process for producing carboxylic acid, comprising converting at least one<br>
cyano group of a nitrile compound in an amount of from lppm to 50% into a carboxyl group using a microorganism, under a temperature of 20 to 40°C, wherein a variant microorganism defective or reduced in the activity of converting a cyano group into an<br>
amide group is used.<br>
2.	The process for producing carboxylic acid as claimed in claim 1, wherein said<br>
variant microorganism is a variant strain of a bacterium belonging to the genus Rhodococcus.<br>
3.	The process for producing carboxylic acid as claimed in claim 2, wherein said<br>
variant strain of  a Rhodococcus bacterium is a variant strain of a parent strain<br>
Rhodococcus sp. ATCC39484.<br>
4.	The process for producing carboxylic acid as claimed in claim 3, wherein the<br>
variant strain of a parent strain Rhodococcus sp. ATCC39484 is Rhodococcus sp.<br>
SD826(FERMBP-7305).<br>
5.	The process for producing carboxylic acid as claimed in claim 1, wherein the<br>
nitrile compound is a polynitrile compound having a plurality of cyano groups in the<br>
molecule and the carboxylic acid is a cyano carboxylic acid.<br>
6.	The process for producing carboxylic acid as claimed in claim 5, wherein the<br>
polynitrile compound is an aromatic polynitrile compound and the cyano carboxylic<br>
acid is an aromatic cyano carboxylic acid.<br>
7.	The process for producing carboxylic acid as claimed in claim 6, wherein the<br>
aromatic polynitrile compound is selected from the group consisting of o-phthalonitrile,<br>
isophthalonitrile, and terephthalonitrile, and the aromatic cyano carboxylic acid is<br><br>
72<br>
corresponding o-cyanobenzoic acid, m-cyanobenzoic acid, or p-cyanobenzoic acid.<br>
8.	A variant microorganism having the activity of converting a cyano group into a<br>
carboxyl group and being defective or reduced in the activity of converting a cyano<br>
group into an amide group.<br>
9.	The variant microorganism as claimed in claim 8, which is a variant strain of a<br>
microorganism belonging to the genus Rhodococcus.<br><br>
10.	The variant microorganism as claimed in claim 9, which is a variant strain of<br>
Rhodococcus sp. ATCC39484.<br>
11.	A Rhodococcus sp. SD826 (FERM BP-7305) strain.<br>
The present invention relates to a novel Rhodococcus bacterium and to a process of hydrolyzing a cyano group of a nitrile compound using a novel Rhodococcus bacterium to produce the corresponding carboxylic acid.    The present invention also relates to a process of producing carboxylic acids, in particular cyano carboxylic acids using a transformant transformed with a plasmid containing a nitrilase gene, a nitrile hydratase gene and an amidase gene derived from Rhodococcus bacteria capable of exhibiting particularly excellent position selectivity for the cyano group of aromatic polynitrile compounds, to such a transformant, such a plasmid, to such genes, to a process of producing an enzyme using the transformant, and to enzymes obtained by the process.    The carboxylic acids, in particular cyano carboxylic acids obtained by the present invention are useful as starting materials for the synthesis of drugs, agrochemicals, dyestuff and other chemicals</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="208187-a-method-of-making-a-vitreous-bonded-abrasive-article-vitreous-bond-abrasive-admixture-and-the-article.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208189-a-modular-shelving-unit-and-components-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208188</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00608/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Jul-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Jul-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Jun-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SHOWA DENKO KK</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>13-9, SHIBADAIMON 1-CHOME, MINATO-KU, TOKYO,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>AOKI HIROBUMI</td>
											<td>C/O.SHOWA DENKO KK ,CENTRAL RESEARCH LABORATORY, 1-1, OHNODAI 1-CHOME, MIDORI-KU, CHIBA-SHI,CHIBA-KEN, JAPAN. CHIBA-KEN,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KAMACHI HARUMI</td>
											<td>C/O. SHOWA DENKO KK ,CENTRAL RESEARCH LABORATORY, 1-1, OHNODAI 1-CHOME, MIDORI-KU, CHIBA-SHI, CHIBA-KEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP00/07464</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-10-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>11/303212</td>
									<td>1999-10-26</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2000-107855</td>
									<td>2000-04-10</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>3</td>
									<td>2000-021797</td>
									<td>2000-01-26</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208188-a-process-for-producing-carboxylic-acid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:03:18 GMT -->
</html>
