<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208332-reducing-immunogenicities-of-immunoglobulins-by-framework-patching by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:45:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208332:REDUCING IMMUNOGENICITIES OF IMMUNOGLOBULINS BY FRAMEWORK-PATCHING</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">REDUCING IMMUNOGENICITIES OF IMMUNOGLOBULINS BY FRAMEWORK-PATCHING</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A re-engineered, or framework (FR)-patched immunoglobulin containing the heavy and/or light chain variable region sequences from a parent antibody, in which at least one of the compartmentalized framework sequences, defined as FR1, FR2, FR3 and FR4 are replaced, or patched by the corresponding framework sequences from the heavy and light chain immunoglobulin variable region of a different species, wherein said re-engineered immunoglobulin chain comprises framework sequences derived from at least two different sources of immunoglobulin chains, wherein said different immunoglobulin chains can be sourced from different immunoglobulins of the same species or from different immunoglobulins of different species, and such FR-patched immunoglobulin binds specifically to an antigen with affinity comparable to, or within 3-fold of, that of the parent immunoglobulin.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM - 2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
COMPLETE SPECIFICATION<br>
[Section 10; Rule 13]<br>
1.	"REDUCING    IMMUNOGENICITIES    OF    IMMUNOGLOBULINS    BY<br>
FRAMEWORK-PATCHING"<br>
2.       (a)      LEUNG, Shawn, Shui-on<br>
(b)	Rm. 305, 40 Consititution Way, Porte Liberte, Jersey City, NJ 07930 United States of America<br>
(c)	United States of America<br>
The following specification particularly describes the nature of the invention and the	 <br>
manner in which it is to be performed.<br><br><br>
24-10-2004<br><br>
1<br><br>
GRANTED<br><br>
The present invention relates to novel methods in re-engineering, or reshaping antibodies'with clinical indications (both therapeutic and diagnostic). The method combines the use of recombinant technology and, stepwise and systemic approaches in re-designing antibody sequences. The invention particularly provides for antibodies which are modified to be less immunogenic than the unmodified counterpart when used in vivo.<br>
BACKGROUND OF THE INDENTION<br>
Monoclonal antibodies (Mabs) have become the most successful protein drugs being used for the treatment of a variety of diseases, including cancers, transplantation, viral infection, etc. However, the concept of magic bullet took more than 25 years to realize, because there were problems associated with the use of monoclonal antibodies. One of the main problems stems from the original source of most monoclonal antibodies, which are of rodent and murine origin. Repeated injections of these foreign proteins into human would inevitably result in the elicitation of host immune responses against the antibodies: the so-called human anti-mouse antibody (HAMA) responses. Although earlier attempts to use the techniques of molecular engineering to construct chimeric antibodies (for example, mouse variable regions joined to human constant regions) were somewhat effective in mitigating HAMA responses, there remains a large stretch of murine variable sequences constituting 1/3 of the total antibody sequence that could be sufficiently immunogenic in eliciting human anti-chimeric antibody (HACA) responses. A more advanced improvement in antibody engineering has recently been utilized to generate humanized antibodies in which the complementarity determining regions (CDR's) from a donor mouse or rat immunoglobulin are grafted onto human framework regions (for example, EPO Publication No. 0239400, incorporated herein by reference). The process is called "humanization", or "CDR-grafting". The original concept of humanization describes the direct grafting of CDR's onto human frameworks, reducing the non-human sequences to less than 5%, and thereby the HAMA and HACA responses. However, direct replacement of framework sequences without further modifications can result in the loss of affinity for the antigen, sometimes as much as 10-fold or more (Jones et al., Nature, 321:522-525, 1986; Verhoyen et al., Science, 239:1534-1536, 1988).    To<br><br><br>
2<br><br>
maintain the affinity of the CDR-grafted or humanized antibody, substitutions of a human framework amino acid of the acceptor immunoglobulin with the corresponding amino acid from a donor immunoglobulin at selected positions will be required. The positions where the substitution takes place are determined by a set of published criteria (US 5,85,089; US 5,693,762; US 5,693,761; incorporated herein by reference). However, the presence of murine amino acids within stretches of human framework sequences can be immunogenic in the generation of new T- and B-cell epitopes. Moreover, the identification of the proper framework amino acids to be replaced can sometimes be difficult, further reducing the chances of success in humanization without significant impacts on the specificity and affinity of the humanized antibody.<br>
New and improved means for producing re-engineered immunoglobulin with reduced or eliminated immunogenicity while maintaining the specificity and affinity of the parent antibody are therefore needed. Preferably, the re-engineered immunoglobulin should contain no FR amino acid substitutions from the parent antibody, which can be a likely source of immunogenic epitopes for T- or B-cells. However, the approach also offers flexibility in the sequence design where few murine residues or a stretch of murine sequences can be included in the final design, with the ultimate goal of reducing immunogenicity while maintaining specificity and affinity of the resultant antibody for human uses. The present invention describes the methods and approaches in fulfilling these goals.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to novel methods for re-engineering immunoglobulin chains having generally one or more complementarity determining regions (CDR's) from a donor immunoglobulin and portions of framework sequences from one or more human, or primate immunoglobulins. The preferred methods comprise first dividing the framework sequences from immunoglobulins of all species into compartmentalized subregions of FR1, FR2, FR3, and FR4, according to the Kabat Database (Kabat et al. Sequences of proteins of immunological interest. Maryland: US Department of Health and Human Services, NIH,  1991), and comparing the<br><br>
3<br><br>
individual FR's, instead of the whole framework, in the variable region amino acid sequence subregions of the parent immunoglobulin to corresponding sequences in a collection of human, or primate immunoglobulin chains, and selecting the appropriate human or primate FR's with the highest degree of homology to replace the original FR's of the parent immunoglobulin (framework- or FR-patching). The human FR's can be selected from more than one human or primate immunoglobulin sequences. A collection of human or primate immunoglobulin sequences can be obtained from different databases (for example, Kabat database, National Biomedical Research Foundation Protein Identification Resource, Brookhaven Protein Data Bank, internet, etc.). The individual FR sequences selected from human or primate immunoglobulins will typically have more than 60% homology to the corresponding parent FR sequences. Although high overall homology will be an important criteria for selecting the FR's for patching, lesser homology FR's will be used if the homology of sequences directly flanking the CDR's or at loop positions where contact(s) with the antigen binding site is (are) determined experimentally or predicted via computer modeling. The parent immunoglobulin whose FR's are to be patched may be either a heavy chain or light chain. A patched light and heavy chain can be used to form a complete FR-patched immunoglobulin or antibody, having two light/heavy chain pairs, with or without partial or full-length human constant regions.<br>
The individual FR's chosen for patching a parent immunoglobulin chain (applies to both heavy and light chains) should:<br>
(1)	preferably have amino acid sequences immediately adjacent to the CDR's identical to that of the parent immunoglobulin chain;<br>
(2)	have amino acid, sequences immediately adjacent to the CDR's conservatively similar in structure to, if not completely identical to, that of the parent immunoglobulin chain;<br>
4<br><br>
(3)	preferably have identical amino acid at corresponding FR position of the parent immunoglobulin predicted to be within about 3 A of the CDR's (or the effective antigen-binding site) in a three-dimensional immunoglobulin model and capable of interacting with the antigen or with the CDR's of the parent or FR-patched immunoglobulin;<br>
(4)	have amino acid conservatively similar in structure to amino acid at corresponding FR position of the parent immunoglobluin predicted to be within about 3A of the CDR's (or-the effective antigen-binding site) in a three-dimensional immunoglobulin model and capable of interacting with the antigen or with the CDR's of the parent or FR-patched immunoglobulin.<br>
Each of the heavy and light chains of the FR-patched immunoglobulin will typically comprise PR's sourced from one or more human or primate immunoglobulins according to any one or all of the selection criteria. The FR-patched heavy and light chains of the present invention, when combined into an intact antibody, antibody fragment, or antibody-based derivatives (for example single-chain antibody, diabodies, etc.), will be substantially non-immunogenic in humans and retain substantially the same affinity and properties (for example internalization upon binding) as the parent immunoglobulin to the antigen. These affinity levels should vary within the range of 4-fold, and preferably within about 2 fold of the parent immunoglobulin's original affinity to the antigen.<br>
Similar principles apply to re-engineer, or patch, parent immunoglobulins of one species with the FR's from a different species, People skilled in the art of protein and/or molecular engineering will be able to adopt the design and principle of the present invention to produce FR-patched immunoglobulins, or derivatives thereof. Once designed, there exist a variety of techniques in constructing the FR-patched immunoglobulin sequence, for example, by site-directed mutagenesis, and gene-synthesis. The assembled FR-patched sequences will be subcloned into expression vectors containing the appropriate immunoglobulin constant heavy and light chains<br><br>
5<br><br>
for transfection in producer cell lines. Diffrent cell systems can be used for the<br>
production of the FR-patched immunoglobulins, including bacterial, yeast, insect, and<br>
mammalian cells.		<br>
Alternatively, the immunoglobulins can be produced in the milks of transgenic or transomatic animals, or as stored proteins in transgenic plants. The present invention offers an improved and novel methods, that are relatively easy (no need to identify important FR amino acid interacting with the CDR's) and highly flexible (freedom to match, and change if necessary, individual FR's) in generating immunoglobulins with reduced or eliminated immunogenicities without sacrificing binding affinity and the likelihood of introducing new T- and B-cell epitopes resulting from the introduction of parent immunoglobulin's framework amino acids into the human FR's. The FR-patched antibodies will be suitable for human use in treating a variety of disease, either used singly or repeatedly, at low (less than 10 mg/m2) or high (more than 100 mg/m ) doses, in. naked forms or as fusion or chemical conjugates, used alone, or in conjunction with other immunoglobulins or treatment modalities.<br>
DETAILED DESCRIPTION OF THE FIGURES<br>
Fig. 1A and Fig. IB. Amino acid sequences (single-letter code) of the heavy chain (VH)(A) and light chain (VL)(B) variable regions of the murine anti-CD22 antibody, RFB4. CDR's are boxed.<br>
Fig. 2A and Fig. 2B. Comparison of the compartmentalized framework sequences (FR's) of the heavy chain (A) and light chain (B) variable regions of RFB4, with the different human FR's of the highest homology. The FR1, FR2, FR3, and FR4 are indicated. The CDR's are boxed. The bracketed italic next on the left of the FR sequence indicates the source of the human FR. Amino acids in the human FR's that are different from that of the corresponding murine FR's are in bold.<br><br>
6<br><br>
Fig. 3A and Fig. 3B. The final designed sequences (single-letter code) of the heavy chain (A) and light chain (B) variable regions of the FR-patched antibody, hpRFB4. CDR's are boxed. Amino acids in the human FR's that are different from that of the original murine FR's are in bold.<br>
Fig. 4. SDS-PAGE analysis of purified cRFB4 and hpRFB4 under both reducing and non-reducing conditions.<br>
Fig. 5. Flow cytometry analyses on the binding specificity and affinity of cRFB4 and hpRFB4 on Raji cells. An irrelevant antibody was used as a control.<br>
Fig. 6. Competition binding assay comparing the binding affinity between cRFB4 and hpRFB4.<br>
Fig. 7A and Fig. 7B. Amino acid sequences (single-letter code) of the heavy chain (A) and light chain (B) variable regions of the murine anti-CD20 antibody, 1F5. CDR's are boxed.<br>
Fig. 8A and Fig. 8B. Comparison of the compartmentalized framework sequences (FR's) of the heavy chain (A) and light chain (B) variable regions of 1F5 with the different human FR's of the highest homology. The FR1, FR2, FR3, and FR4 are indicated. The CDR's are boxed. The bracketed italic next on the left of the FR sequence indicates the source of the human FR. Amino acids in the human FR's that are different from that of the corresponding murine FR's are in bold.<br>
Fig. 9A and Fig. 9B. The final designed sequences (single-letter code) of the heavy chain (A) and light chain (B) variable regions of the FR-patched antibody, hplF5. CPR's are boxe4. Amino acids in the human FR's that &amp;re different from that of the original murine FR's are in bold. Murine FR's not replaced by human sequences are underlined.<br><br>
7<br><br><br><br>
Fig. 10. Amino acid sequence of an alternative design of FR-patched variable regions for 1F5 (Alternative Design). CDR's are boxed. Human framework amino acids that differ from that of the corresponding murine frameworks are in bold.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention aims to establish novel approaches in the design of immunoglobulin with high degree of homology to human or primate sequences through a process named "framework (FR) patching". The FR-patched immunoglobulin (patched immunoglobulin thereafter) will have substantially reduced, or eliminated immunogenicity when used in human, and carry most or all of the characteristics of a human immunoglobulin such as the ability to target specific antigens, and effector functions (for example, complement fixation, ADCC, etc.), while maintaining the specificity and affinity of the parent immunoglobulin against a specific antigen. The patched immunoglobulin will comprise a heavy and light chain, of which, the respective variable region will contain sequences representing FR1, CDR1, FR2, CDR2, FR3, CDR3, and FR4, according to Rabat's classification (Kabat et al., op. cit). At least one of the four FR's of each parent immunoglobulin chain containing one or more complementary determining regions (CDR's) will be replaced, or "patched" with, a corresponding human or primate FR. When two or more FR's of the parent immunoglobulin chain are to be replaced, they can be patched with corresponding FR's either from the same human or primate immunoglobulin, or from different human or primate immunoglobulin within the same subgroup or in different subgroups, or from a combination of human and primate immunoglobulins. The patched immunoglobulins will be expressed in appropriate host system for large-scale production at typical pharmaceutical margins, and used in humans at appropriate formats or combinations to treat or detect a wide range of human diseases.<br>
To ensure a better understanding of the present invention, several definitions are set forth. As used herein, an "immunoglobulin" refers to a protein consisting of one or more polypeptides substantially encoded by immunoglobulin genes. A typical immunoglobulin protein contains two heavy chains paired with two light chains.   A<br><br><br><br>
8<br><br>
full-length immunoglobulin heavy chain is about 50 kD in size (approximately 446 amino acids in length), and is encoded by a heavy chain variable region gene (about 116 amino acids) and a constant region gene. There are different constant region genes encoding heavy chain constant region of different isotypes such as alpha, gamma (IgGl, IgG2, IgG3, IgG4), delta, epsilon, and mu sequences. A full-length immunoglobulin light chain is about 25 Kd in size (approximately 214 amino acids in length), and is encoded by a light chain variable region gene (about 110 amino acids) and a kappa or lambda constant region gene. Naturally occurring immunoglobulin is known as antibody, usually in the form of a tetramer consisting of two identical pairs of immunoglobulin chains, each, pair having one light and one heavy chain. In each pair, the light and heavy chain variable regions are together responsible for binding to an antigen, and the constant regions are responsible for the effector functions typical of an antibody.<br>
Immunoglobulin may be in different forms, either naturally occurring, chemically modified, or genetically-engineered, such as Fv (Huston et al.„ Proc. Natl. Acad. Sci. USA. 85:5879-5833; Bird et al., Science 242:423-426, 1988), diabodies, mini-antibodies, Fab, Fab', F(ab')2, bifunctional hybrid antibodies (Lanzavecchia et al, Eur. J. Immunol. 17:105, 1987) (See, generally, Hood et al., "Immunology", Benjamin, NY, 2nd ed. 1984; Hunkapiller and Hood, Nature 323:15-16, 1986).<br>
The variable region of both the heavy and light chain is divided into segments comprising four framework sub-regions (FR1, FR2, FR3, and FR4), interrupted by three stretches of hypervariable sequences, or the complementary determining regions (CDR's), as defined in Rabat's database (Kabat et al., op. cit), with the CDR1 positioned between FR1 and FR2, CDR2 between FR2 and FR3, and CDR3 between FR3 and FR4. Without specifying the particular sub-regions as FR1, FR2, FR3 or FR4, a framework region, as referred by others, represents the combined FR's within the variable region of a single, naturally occurring immunoglobulin chain. As used herein, a FR represents one of the four sub-regions, and FR's represents two or more of the four sub-regions constituting a framework region.    The sequences of the<br><br><br>
9<br><br>
framework regions of different light or heavy chains are relatively conserved within a species. The framework region of an antibody is the combined framework regions of the constituent light and heavy chains and serves to position and align the CDR's. The CDR's are primarily responsible for forming the binding site of an antibody conferring binding specificity and affinity to an epitope of an antigen.<br>
Parent antibody is an antibody of a particular species, for example, murine, which is to be re-engineered, re-shaped, or in this invention, FR-patched into a form, sequence, or structure, appropriate for use in a different species, for example, human, with reduced or minimized immuribgenicity.<br>
Chimeric antibodies are antibodies whose variable regions are linked, without significant sequence modifications from the parent V-region sequences, to the corresponding heavy and light chain constant regions of a different species. Construction of a chimeric antibody is usually accomplished by ligating the DNA sequences encoding the variable regions to the DNA sequences encoding the corresponding constant chains. The most common types of chimeric antibodies are those containing murine variable regions and human constant regions. In this case, the expressed hybrid molecule will have the binding specificity and affinity of the parent murine antibody, and the effector functions of a human antibody. Most importantly, 2/3 of the amino acids of the recombinant protein are of human origin, a reduced or insignificant immunogenicity is therefore expected when used in human, as in the case of the therapeutic chimeric antibody C2B8 (or Rituxan) (Davis et al, J. Clin. Oncol. 17:1851-1857, 1999; Coiffier et al., Blood 92:1927-1932, 1998; McLaughlin et al, J. Clin. Oncol. 16:2825-2833, 1998).<br>
A "humanized" immunoglobulin is generally accepted as an immunglobuhn comprising a human framework region and one or more CDR's from a non-human immunoglobulin (Jones et al., op. cit; Verhoeyen et al., op. cit; Riechmann et al., op. cit). The non-human immunoglobulin providing the CDR's is called the "donor" and the human  immunoglobulin providing  the  framework is  called  the "acceptor".<br><br><br>
10<br><br>
modeling in identifying the appropriate framework amino acid to mutate, and lack flexibility in the choice of acceptor human frameworks in an all-or-none mode. Most importantly, it is likely to introduce potential new immunogenic epitopes by retaining parent "donor" residues in the human "acceptor" framework.<br>
The present invention addresses these problems and creates a novel approach with increased flexibility and simplicity in generating a FR-patched antibody that is not immunogeniL or is low in immunogenicity, yet having retained most or all of the original affinity against a specific antigen, as in the parent antibody. Since most of the immune responses against a chimeric or humanized immunoglobulin will be directed against epitopes in the variable regions, without intending to be bound by theory, the principle by which the invention comes about will be illustrated by, but not limited to, using the variable region as the example.<br>
There exist at least two kinds of epitopes contributing to the immunogenicity against a protein. The so-called "T cell epitopes" are short peptide sequences released during the degradation of proteins within cells and subsequently presented by molecules of the major histocompatabihty complexes (MHC) in order to trigger the activation of T cells. For peptides presented by MHC class II, such activation of T cells can then give rise to an antibody response by direct stimulation of B cells to produce such antibodies. A detailed analysis of the structure of a humanized variable region reveals the unavoidable existence of stretches of potentially immunogenic CDR's. These CDR's physically and functionally compartmentalize the rest of the framework sequences into four sub-regions, namely, the FR1, FR2, FR3, and FR4 (Kabat et al., op. cit). Since T cell epitopes are linear continuous short peptides, the presence or absence of such epitopes in each FR compartments should have no correlation to each other, whether the different FR's are derived from the same or different frameworks. The introduction of donor framework residues to the acceptor framework region using the humanization approach of Queen et al. (US 5,85,089; US 5,693,762; US 5,693,761; incorporated herein by reference) will have the possibility of generating new, immunogenic T cell epitopes, resulting in the elicitation of immune responses<br><br><br>
11<br><br>
Usually, as has been used and referred to by others, an acceptor is derived from a single human immunoglobulin species. To maintain the affinity of the "humanized" immunoglobulin, donor amino acid residues may have to be incorporated in the framework region of the acceptor immunoglobulin. There is a set of criteria for selecting a limited number of amino acids within the acceptor immunoglobulin for conversion into donor sequences, as published in a series of publications (US 5,85,089; US 5,693,762; US 5,693,761; incorporated herein by reference). The humanized immunoglobulins may or may not contain constant regions. A humanized heavy chain immunoglobulin is a humanized immunoglobulin comprising a corresponding human heavy chain constant region, and a humanized light chain immunoglobulin is a humanized immunoglobulin comprising a corresponding human light chain constant region. A humanized antibody is an antibody comprising a humanized light chain and a humanized heavy chain immunoglobulin.<br>
A successful humanized antibody will have to have the following characteristics:<br>
(1)	significantly reduced, and preferably eliminated, immunogenicity resulting from the humanized sequences, allowing multiple injection of the humanized antibody for human uses;<br>
(2)	minimally perturbed immunoreactivity including specificity and affinity (within 3-fold) against the original antigen;<br>
(3)	capable of inducing human effector functions such as complement fixation, complement-mediated cytotoxicity, antibody-dependent cell cytotoxicity, etc.<br>
Direct grafting of donor CDR's onto human acceptor framework without further sequence modifications, will likely result in substantial loss of antigen affinity. Although the introduction of selected donor amino acids to acceptor framework regions will somehow rectify the problem, and most of the time, improve affinity, however, the approach is tedious, requiring sometimes the assistance of computer<br><br>
12<br><br>
against the humanized antibody, particularly antibody responses against the idiotypic region formed by the donor CDR loops. It is uncommon to have between 3 to 7 donor amino acids incorporated into each humanized immunoglobulin chain, greatly increasing the chances of emergency for new T cell epitopes.<br>
Similarly, these donor-derived residues embedded within the human framework can form new immunogenic B-cell epitopes recognizable by antibodies. While it is well-established that re-introduction of donor residues to the acceptor framework is important in maintaining the original antigen affinity of the humanized immunolobulin, ideally, it would be preferable if humanization can be accomplished by direct grafting of donor CDR's onto acceptor framework without additional modification and loss of affinity.<br>
The present invention provides a new approach in reducing or eliminating the immunogenicity of immunoglobulins whose affinity against the specific antigen is maintained within three fold of its original level. The approach is flexible, versatile, simple, and does not usually require sophisticated computer modeling analysis (although it does not preclude its being used). It deals with the problem of reciprocal relation between reducing immunogenicity and maintaining affinity in humanizing an antibody with the previous and available methodologies. Using an immunoglobulin variable region as example, a set of criteria and principles will be followed in FR-patching the sequence. The criteria may be used singly, or when necessary in combination, to achieve reduced or eliminated immunogenicity, and the desired affinity or other characteristics.<br>
In humanizing an immunoglobulin variable region by FR-patching, the parent immunoglobulin amino acid sequences are compartmentalized into FR1, CDR1, FR2, CDR2, FR3, CDR3, and FR4 according to the classification of Kabat et al. (opt. cit). Each of the compartmentalized FR's will be dealt with separately and used to align with the corresponding FR segments found in all databases, available either in the public domain, commercial entities, or private possession (for example the Kabat<br><br>
13<br><br>
database, opt.cit; the National Biomedical Research Foundation Protein Identification Resource). An immunoglobulin can be patched with FR's from more than one immunoglobulin sources. Preferably, human FR segments with the highest degree of homology (&gt;60%) to the corresponding parent FR's will be used. However, amino acids in the FR's adjacent to one or more of the 3 CDR's in the primary sequence of the immunoglobulin chain may interact directly with the antigen (Amit et al., Science, 233:747-753, 1986, which is incorporated herein by reference) and selecting these amino acids identical to the human FR's with lesser homology will be used according to the criteria set forth below.<br>
A human FR1 will be used when it has the highest homology to the parent FR1, preferably 100%, at three or more amino acids immediately adjacent to CDR1.<br>
A human FR2 will be used when it has the highest homology to the parent FR2, preferably 100%, at three or more amino acids at both ends immediately adjacent to the flanking CDR1 and CDR2.<br>
A human FR3 will be used when it has the highest homology to the parent FR3, preferably 100%, at three or more amino acids at both ends immediately adjacent to the flanking CDR2 and CDR3.<br>
A human FR4 will be used when it has the highest homology to the parent FR4, preferably 100%, at three or more amino acids immediately adjacent to CDR3.<br>
In case human FR's with 100% homology at three or more amino acids adjacent to the CDRs cannot be identified, FR's with the closest homology at these positions containing conservatively similar amino acids, such as, gly, ala; val, ile, leu; asp, glu; asn, gin; ser, thr; lys, arg; and phe, tyr; will be selected.<br>
Preferably, human FR's whose amino acids at positions known to be close to, or have interactions with the CDR's/antigen binding site (Chothia and Lesk, J. Mol. Biol.<br><br><br><br>
14<br><br>
196:901, 1987; Chothia et al., Nature 342:877, 1989, and Tramontano et al., J. Mol. Biol. 215:175, 1990, all of which are incorporated herein by reference), either based on computer modeling (see Levy et al., Biochemistry 28:7168-7175, 1989; Bruccolcri et al., Nature 335:564-568, 1988; Chothia et al., Science 233:755-758, 1986, all of which are incorporated herein by reference), crystal structure, published information, or prior experience, which are identical, or conservatively similar to that of the parent FR's will be selected.<br>
FR-patching does not preclude the introduction of parent amino acids at corresponding positions of a patched FR where necessary, or the inclusion of FR's in the immunoglobulin from different species such as different primates, murine, etc, when available databases fail to produce a satisfactory FR that meet the above criteria. The primary goal is to produce antibodies with reduced, preferably, eliminated immunogenicity without substantial loss of affinity. The approach increases the chances of success in this regard, with significant improvements over other methods in terms of flexibility, simplicity, and ease of operation.<br>
FR-patched antibodies carrying human constant sequences will be able to induce human immune effector functions such as complement-mediated cytotoxicity (CM) or anti-body-dependent cellular cytotoxicity (ADCC), upon binding to the target antigens. Moreover, when injected in human for therapeutic or diagnostic purposes, antibodies patched with human FR's are expected to be non-immunogenic, i.e., will not elicit antibody responses against the injected protein, allowing for multiple injections into human patients if necessary for achieving maximum clinical benefits. Non-human antibodies have been reported to have significantly shorter circulation half-lives than that of human antibodies (Shaw et al., J Immunol. 138:4534-4538, 1987). The patched antibodies, carrying mostly human sequences, will presumably have an extended half-life reminiscent to naturally occurring human antibodies.<br>
In the construction of a FR-patched immunoglobulin, sequence design for the variable regions of the immunoglobulin, will be done using the criteria and principles illustrated<br><br><br>
15<br><br>
above. The designed FR-patched variable region sequence will be assembled using a variety of standard recombinant techniques well known to those skilled in the art. Preferably, the designed sequence, usually of a size of about 350 base pairs, will be gene-synthesized (Leung et ah, Molecular Immunol. 32:1413-1427, 1995; Daugherty et al.,'Nucl, Acid Res. 19:2471-2476; DeMartino et al., Antibody Immunoconj. Radiopharrnaceut. 4:829, 1991; Jones et al., op. cit, all of which are incorporated herein by reference), or the individual FR's can be introduced to replace the parent FR's by methods of site- or oligonucleotide-directed mutagenesis (Gillman and Smith, Gene 8:81-97, 1979; and Roberts et al., Nature 328:731-734; both of which are incorporated herein by reference).<br>
The DNA segment encoding the FR-patched immunoglobulin will be joined to DNA sequences encoding the human heavy and light chain regions in DNA expression vectors suitable for bacterial propagation and expression in different host cells. There are a variety of DNA vectors suitable for expression in a variety of host cell systems. Appropriate DNA vectors can be chosen for the expression of the FR-patched immunoglobulins. Typically, a suitable expression control DNA sequence is linked operably to DNA segments encoding the immunoglobulin chains. Preferably, the expression control sequences will be eukaryotic promoter systems in vectors capable of transforming or transfecting eukaryotic host cells, but control sequences for prokaryotic hosts may also be used. The sequence encoding the FR-patched heavy and light immunoglobulin chains can be incorporated into one single DNA expression vector, or into separate heavy and light chain expression vectors. In the latter case, host cells will have to be simultaneously incorporated with both vectors in order to produce a FR-patched antibody with the properly paired heavy and light chain polypeptides. In general, a leader sequence allowing the transportation of the immunoglobulin polypeptide into the Golgi apparaUis for later secretion is included at the N-terminal end of each immunoglobulin chain for expression in eukaryotic host cells. Once the vector has been incorporated into the appropriate host, the host is maintained under conditions suitable for high level expression of the nucleotide sequences, and, as desired, the collection and purification of the humanized light<br><br><br>
16<br><br>
chains, heavy chains, light/heavy chain dimers or intact antibodies, binding fragments, single chain antibody (sFv), diabodies, or derivatives thereof, or other immunoglobulin forms may follow (see Beychok, Cells of Immunoglobulin Synthesis, Academic Press, NY, 1979, which is incorporated herein by reference).<br>
It is a well-known fact that there are different human constant regions for the heavy and light chains. A particular isotype will have specific effector characteristics that can be chosen for use for different purposes. Human constant region DNA sequences can be isolated in accordance with well known procedures from a variety of human cells, but preferably immortalized B-cells (see, Kabat op. cit. and WP87/02671). The CDR's for producing the immunoglobulins of the present invention will be similarly derived from monoclonal antibodies capable of binding to the predetermined antigens, such as CD22 and CD20, for example, and produced by well known methods in any convenient mammalian source including, mice, rat, rabbits, or other vertebrate, capable of producing antibodies. Suitable source cells for the constant region and framework DNA and secretion, can be obtained from a number of sources such as the American Type Culture Collection ("Catalogue of Cell Lines and Hybridomas," sixth edition, 1988, Rockville, MD, USA, which is incorporated herein by reference).<br>
DNA expression vectors containing the coding sequences for the FR-patched immunoglobulin chains operably linked to an expression control sequence (including promoter and enhancers) are typically replicable in the host organisms either as episomes or as an integral part of the host chromosomal DNA. Selectable markers such as tetracycline, neomycin, beta-lactamase, etc., are included in the vector to allow detection of cells transformed with the DNA vectors (see, for example, US Pat. No. 4,704,362, which is incorporated herein by reference).<br>
Bacterial hosts are suitable for propagating the DNA vectors as well as expressing the incorporated immunoglobulin DNA. For example, E. coli is the most commonly used prokaryotic host used for cloning the DNA sequence for the present invention. Other microbial hosts that are useful for the same purposes include, as examples, bacilli (for<br><br>
17<br><br>
example Bacillu subtilus), and other enterobacteriaceae (for example Salmonella, Serratia), and various Pseudomonas species. Expression of cloned sequences in these hosts require the presence of expression control sequences compatible with the host cell (for example an origin of replication), and functional promoters to be included in the DNA vector. Example of well-known promoter system include, but not limited to, tryptophan (trp) promoter system, beta-lactamase promoter system, phage lambda promoter system, etc. These promoters are responsible for controlling expression, or transcription, of the functional gene sequence downstream of the promoter system, which contains, in addition to all necessary motifs, and optionally with an operator sequence, ribosome'binding site sequences and the like, necessary for transcription initiation and translation.<br>
Similarly, other microbes, such as yeast, may also be used for expression. For example, a preferred host will be Saccharomyces, which is a suitable host for expression vectors containing the appropriate expression control elements, such as promoters, including 3-phosphoglycerate kinase or other glycolytic enzymes, and an origin of replication, termination sequences and the like as desired.<br>
Eukaryotic host cells of invertebrate origin can be used. For example, insect cells, such as hi-5, SF9, SF2L Appropriate expression vectors containing convenient cloning sites, promoters, termination sequences, etc., that are important for high-level expression in the host cells are available commercially (Invitrogen, San Diego, CA).<br>
Preferably, mammalian tissue cell culture may be used to express and produce the polypeptides of the present invention (see, Winnacker, "From Genes to Clones," VCH Publisher, NY, NY, 1987, which is incorporated herein by reference). The most commonly used mammalian host cells are Chinese Hamster Ovary (CHO) cell lines, various COS cell lines, HeLa cells, and myeloma cell lines such as SP2/0 cell lines, NSO cell lines, YB2/0 cell lines, etc, and transformed B-cells or hybridomas. These cell lines are capable of conferring the right glycosylation at appropriate site such as amino   acid  297   at  the  heavy   chain   CH2   domain,   and   secreting  full-length<br><br><br>
18<br><br>
immunoglobulins, and are the host cell system of choice for this particular invention. Similar to expression vectors for other host cells, a eukaryotic cell expression vector will contain the appropriate expression control sequences including promoter (for example, those derived from immunoglobulin genes, metallothionine gene, SV40, Adenovirus, cytomegalovirus, Bovine Papilloma Virus, and the like), enhancers, usually with a leader sequence for directing the expressed polypeptide to the Golgi apparatus for glycosylation and export, the DNA segments of interest (for example, the heavy and light chain encoding sequences and expression control sequences), a termination codon, other necessary processing information sites (such as ribosome binding sites, RNA splice sites, a polyadenylation sequence, and transcriptional terminator sequences), and a selection marker (such as mutant Dhfr, glutamine synthetase (GS), hygromycin, neomycin)(see Kellems, "Gene Amplification in mammalian cells", Marcel Dekker Inc., NY, NY, 1993; which is incorporated herein by reference).<br>
There exist a plethora of established and well-known methods for introducing the vectors containing the DNA segments of interest into the host cell, either transiently or stably integrated into the host cell genome. They include, but not limited to, calcium chloride transfection, calcium phosphate treatment, electroporation, lipofection, etc. (See, Maniatis et al., Molelcular Cloning: A Laboratory Manual, Cold Spring Harbor Press, 1982; which is incorporated herein by reference). Identification of host cells incorporated with the appropriate expression vector will be achievable typically by first growing cells under selection pressure in accordance with the selectable marker used in the vector, and detection of secreted proteins, for example, the whole antibodies containing two pairs of heavy and light chains, or other immunoglobulin forms of the present invention, by standard procedures such as ELISA and Western analysis... Purification of the expressed immunoglobulin can be purified according to standard procedures of the art, including ammonium sulfate precipitation, affinity columns, column chromatography, gel electrophorsis and the like (see, generally, R. Scopes, "Protein Purification", Springer-Verlag, NY, 1982). Substantially pure immunoglobulins of at least about 90 to 95% homogeneity are preferred, and 98 to<br><br><br>
19<br><br>
99% or more homogeneity most preferred, for pharmaceutical uses. Once purified, partially or to homogeneity as desired, the polypeptides may then by used therapeutically (including extracorporeally) or in developing and performing assay procedures, immunofluorescent stainings, and the like (See, generally, Immunological Methods Vols I and II, Lefkovits and Pernis, eds., Academic Press, New York, NY, 1979 and 1981).<br>
The antibodies of the present invention will typically find use individually, or in combination with other treatment modalities, in treating diseases susceptible to antibody-based therapy.  For example, the immunoglobulins can be used for passive<br>
immunization, or the removal of unwanted cells or antigens, such as by complement<br><br>
mediated lysis, all without substantial adverse immune reactions (for example anaphylactic shock) associated with many prior antibodies.<br>
A preferable usage of the antibodies of the present invention will be the treatment of diseases using their naked forms (naked antibodies) at dosages ranging from 50 mg to 400 mg/m2 , administered either locally at the lesion site, subcutaneousely, intraveneousely, and intramuscularly, etc. Multiple dosing at different intervals will be performed to achieve optimal therapeutic or diagnostic responses, for example, at weekly intervals, once a week, for four weeks. Usage of the antibodies derived from the present invention can be combined with different treatment modalities, such as chemomerapeutic drugs (for example CHOP, Dox, 5-Fu, ..etc), radiotherpy radioimmunotherapy, vaccines, enzymes, toxins/immunotoxins or other_antibodies derived from the present invention or others. The antibodies of the present invention, if specific for the idiotype of an anti-tumor antibody, can be used as tumor vaccines for the elicitation of Ab3 against the tumor antigen. Numerous additional agents, or combinations of agents, well-known to those skilled in the art may also be utilized.<br>
Additionally, the antibodies of the present invention can be utilized in different pharmaceutical compositions. The antibodies can be used in their naked forms, or as conjugated proteins with drugs, radionuclides, toxins, cytokines, soluble factors,<br><br><br>
20<br><br>
-^<br><br>
normans enzymes (for example carboxylesterase, ribonuclease), peptides, antigens (as tumor vaccine), DNA, RNA, or any other effector molecules having a specific therapeutic function with the antibody moiety serving as the targeting agents or delivery vehicles. Moreover, the antibodies or antibody derivatives, such as antibody fragments, single-chain Fv, diabodies, etc. of the present invention can be used as fusion proteins to other functional moieties, such as, antibodies or antibody derivatives of a different invention (for example as bispecific antibodies), toxins, cytokines, soluble factors, hormones, enzymes, peptides, etc. Different combinations of pharmaceutical composition, well-known to those skilled in the art may also be utilized.<br>
FR-patched antibodies of the present invention can also be used for in vitro purposes, for example, as diagnostic tools for the detection of specific antigens, or the like.<br>
The following examples are offered by way of illustration, not by limitation.<br>
EXPERIMENTAL<br>
In designing the amino acid sequence of the FR-patched immunoglobulin chain, the murine variable region sequence (applies to both VH and VL) was compartmentalized into FR1, CDR1, FR2, CDR2, FR3, CDR3, and FR4, according to Kabal's classification (Kabat et al., op. cit). Selection of the individual FR's for patching was in accordance to the guidelines as described previously.<br>
A human FR1 will be used when it has the highest homology to the parent FR1, preferably 100%, at three or more amino acids immediately adjacent to CDR1.<br>
A human FR2 will be used when it has the highest homology to the parent FR2, preferably 100%), at three or more amino acids at both ends immediately adjacent to the flanking CDR1 and CDR2.<br><br><br>
21<br><br>
A human FR3 will be used when it has the highest homology to the parent FR3, preferably 100%, at three or more amino acids at both ends immediately adjacent to the flanking CDR2 and CDR3;<br>
A human FR4 will be used when it has the highest homology to the parent FR4, preferably 100%, at three or more amino acids immediately adjacent to CDR3.<br>
In case human FR's with 100% homology at three or more amino acids adjacent to the CDRs cannot be identified, FR's with the closest homology at these positions containing conservatively similar amino acids, such as, gly, ala; val, ile, leu; asp, glu; asn, gin; ser, thr; lys, arg; and phe, tyr; will be selected.<br>
Preferably, human FR's whose amino acids at positions known to be close to, or have interactions with the CDR's/antigen binding site (Chothia and Lesk, op. cit; Chothia et al., op. cit., and Tramontano et al., op. cit.), either based on computer modeling (sec Levy et al., Biochemistry op. cit.; Bruccoleri et al., op. cit.; Chothia et al., op. cit.), crystal structure, published information, or prior experience, which are identical, or conservatively similar to that of the parent FR's will be selected..<br>
In case where a particular human FR satisfying all the above is unavailable, and direct patching results in the loss of affinity or specificity, murine residues considered to have interactions with the antigen binding site, or contribute to the final affinity of the antibody, will be introduced back to the best available FR. Alternatively, the particular FR with no matching human counterpart will be retained and used in its murine composition without modification; the final FR-patched sequence will contain a mixture of liuman and murine FR's. For the purpose of illustration, two murine anti-B cell antibodies will be FR-patched using the approach as described herein in this invention.<br>
22<br><br>
EXAMPLE 1<br>
FR-PATCHED ANTI-CD22 ANTIBODY<br>
Design of genes for FR-patched anti-CD22 light and heavy chain<br>
The heavy and light chain sequence of a murine anti-CD22 antibody, RFB4 (Li et al., Cell Immunol. 118:85, 1989; Mansfield et al., Blood 90:2020-2026, 1997) is used as an example to illustrate the approach of using FR-patching to reduce or eliminate immunogenicity of the re-engineered antibody. The sequences of the heavy (a) and light chain (b) variable region for the murine antibody are shown in Figure 1.<br>
Patching of the individual FRs for the heavy chain variable region for RFB4 was done as follows:<br>
a. FR1: the FR1 sequence of the murine VH was compared with the FR1 sequences of human VH from the Kabat's database (Kabat et al., op. cil.). Although human FR1 of the highest sequence homology is preferred, particular emphasis on the sequence closest to the CDR1 was taken. There are three FR1 sequences that are of high homology to the murine FR1. They are, namely, EIK, RF-SJ1, and WAS. The FR1 with the highest overall homology with the five residues closest to the CDR1 identical to the murine parent is EIK, however, there is a missing residue in position 12, which can create potential problems affecting the immunoreactivity of the resultant antibody. The preferred FR1 picked for the patching was therefore from WAS. First, except at position 28, the third closest residue to GDR1, a whole stretch of 11 amino acids next to the CDR1 is identical to the murine parent. In position 28, a serine residue is found instead of an alanine in the murine sequence. Since serine is considered as a hydroxylated version of alanine, the change is conservative. Moreover, residues that are different between the human and murine are relatively similar in characteristics. For example, valine and leucine in position 5, lysine and glutamine at position 13, lysine and arginine at position 19, and alanine and serine at position 28. Therefore, human sequence from WAS was chosen for patching the FR1 of the anti-CD22 antibody (Figure 2 A).<br>
23<br><br>
b.	FR2: by the same token and based on the degree of homology, the human WAS<br>
sequence is chosen for patching the FR2 of the anti-CD22 antibody (Figure 2 A).<br>
c.	FR3: with the sequences closest to the CDR2, and CDR3 being identical, and the<br>
high degree of homology, the human GAL sequence was selected for patching the<br>
FR3 of the anti-CD22 antibody (Figure 2A).<br>
d.	FR4; there are many human FR4 with the sequence closest to the CDR3 being<br>
identical, and a high degree of homology to the murine parent. In this example,<br>
the human DOB sequence was selected for patching the FR4 of the anti-CD22<br>
antibody (Figure 2A).<br>
The final design of the FR-patched VH sequence (Figure 3A) for the anti-CD22 antibody is composed of the human WAS FR1 and FR2, and the GAL FR3 and DOB FR4, replacing the original VH PR's of the anti-CD22 antibody. There is no single mutation or re-introduction of murine FR residues in the final design of the FR-patched sequence.<br>
Using a similar strategy, the sequence design for the FR-patched light chain (VL) was done as follows:<br>
a.	FR1: human JOH was chosen for patching the FR1 of the murine VL. It has a high<br>
degree of sequence homology and the stretch of 8 amino acids adjacent to the<br>
CDR1 being identical to the parent sequence (Figure 2B).<br>
b.	FR2: human Vd'CL was chosen for patching the FR2 of the murine VL, for<br>
similar reasons. More than 4 identical sequences are adjacent to the CDR1, and<br>
CDR2 (Figure 2B).<br>
c.	FR3: human WES was chosen for patching the FR3 of the murine VL. FR3 has<br>
the longest sequence, and the sequence homology between WES and the murine<br>
24<br><br>
FR3 is high, with the sequences flanking the CDR2 and CDR3 being identical (Figure 2B).<br>
d. FR4: human RZ was chosen for patching the FR4 of the murine VL, for similar reasons (Figure 2B).<br>
The final design of the FR-patched VL sequence (Figure 3B) for the anti-CD22 antibody is composed of the human JOH FR1, Vd'CL FR2, WES FR3, and RZ FR4, replacing the original VL FR's of the anti-CD22 antibody. Once again, tliere is no single mutation or re-introduction of murine FR residues in the final design of the FR-patched sequence.<br>
Construction of the FR-patched heavy and light chain genes<br>
The designed heavy and light chain variable region sequences of the FR-patched antibody are assembled by a combination of oligonucleotide synthesis and PCR using a variety of published methods (Leung et al, op. cit.; Daugherty et al., op. cit.; DeMartino et al., op. cit.; Jones et al., op. cit.).<br>
To construct the FR-patched heavy chain variable region sequence (SEQ ID no. 1), the full DNA sequence is divided into two halves: the N-terminal half and the C-terminal halves. Both were constructed separately by PCR and the complete variable region sequence was formed by joining the N- and C-terminal halves at the Kpnl site.<br>
The N-terminal half was constructed as follows; a N-template (SEQ 3) is a synthetic sense-strand oligonucleotide (111-mer) encoding amino acid 14 - 50 of the VH region (SEQ ID no. 2). The template is PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 4) is a synthetic sense-strand oligonucleotide (57-mer) encoding amino acid 1 - 19 of the VH region. The 3' end of the primer overlaps with the 5' end of the template by 18 nucleotides.<br>
25<br><br>
The 3' Primer (SEQ ID no. 5) is a synthetic anti-sense-strand oligonucleotide (48-mer) encoding amino acid 43 - 59. The primer overlaps with the template by 21 nucleotides.<br>
The N-template (SEQ ID no. 3) was PCR-amplified using the 5' and 3' primer set (SEQ ID no. 4 &amp; 5) using standard techniques and procedures.<br>
The C-terminal half was constructed as follows: a C-template (SEQ ID no. 6) is a synthetic sense-strand oligonucleotide (132-mer) encoding amino acid 68 - 111 of the VH region (SEQ ID no. 2). The template was PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 7) is a synthetic sense-strand oligonucleotide (60-mer) encoding amino acid 55 - 74 of the VH region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br>
The 3' Primer (SEQ ID no. 8) is a synthetic anti-sense-strand oligonucleotide (58-mer) encoding amino acid 105 - 123 of the VH region. The primer and the template overlaps by 21 nucleotides.<br>
The C-template (SEQ ID no. 6) is PCR-amplified using the 5' and 3' primer set (SEQ ID no. 7 &amp; 8) using standard techniques and procedures.<br>
For the construction of the full-length FR-patchcd RFB4 VH domain, the N-template (SEQ ID no. 3, 111-mer), C-template (SEQ ID no. 6, 132-mer), and their respective 5'- and 3' primers (SEQ ID no. 4 &amp; 5 for N-template, and SEQ ID no.7 &amp; 8 for C-template), were synthesized on an automated Applied Biosystem 3 8 OB DNA synthesizer (Foster City, CA). The oligonucleotides were desalted by passing through a CHROMOSPIN-10™ column (Clonetech, Palo Alto, CA). The oligonucleotides were adjusted to a final concentration of 20 uM. One u.1 of template oligonucleotides at various dilutions (10X, 100X, 1000X and lOOOOX, etc.) were mixed with 5 pi of their corresponding flanking primers in the presence of 10 pi of lOx PCR Buffer (500<br>
26<br><br>
mM KCI, 100 mM Tris.HCl buffer, pH 8.3, 15 mM MgC12) and 5 units of AMPLITAQ™ DNA polymerase (Perkin Elmer, Applied Biosystems Division, Foster City, CA). This reaction mixture was adjusted to a final volume of 100 u.1 and subjected to 30 cycles of PCR reaction consisting of denaturation at 94°C for 1 minute, annealing at 50°C for 1 minutes, and polymerization at 72°C for 1 minute. The PCR reaction mixture were analyzed under 2% agarose gel electrophoresis. The highest template dilution that gave rise to sufficiently abundant product of the right size would be chosen for further processing.<br>
Double-stranded PCR-amplified products for the N- and C-templates were gel-purified, restriction-digested with KpnI. The restricted N- and C-double stranded DNA were ligated at the KpnJ site, and the ligated products were subjected to another round of PCR amplification using the 5' primer for the N-template (SEQ ID no.4) and the 3' primer for the C-template (SEQ ID no. 8). The PCR product with a size of ~350 was directly cloned into the TA cloning vector (Invitrogen, San Diego, CA). The sequence of the cloned fragment was confirmed by Sanger's method (Sanger et al., PNAS 74:5463-5467, 1977) to be identical to the designed VH sequence. The confirmed sequence was used to replace the VH sequence of a heavy chain expression vector containing an lgH promoter, an Ig enhancer, a human IgGl constant region genomic sequence, and a selectable marker, gpt. The final heavy chain expression vector is designated as hpRFB4pSMh.<br>
To construct the FR-patched light chain variable region sequence (SEQ ID no. 9), the full length VL variable region sequence was divided into two halves. The N-terminal and C-terminal halves were assembled separately by PCR and joined together via the Spel site.<br>
The N-terminal half was constructed as follows: a N-template (SEQ ID no. 11) is a synthetic sense-strand oligonucleotide (108-mer) encoding amino acid 11-46 of the VL region (SEQ ID no. 10). The template was PCR-amplified by two primers:<br>
27<br><br>
The 5' Primer (SEQ ID no. 12) is a synthetic sense-strand oligonucleotide (51-mer) encoding amino acid 1-17 of the VL region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br>
The 3' Primer (SEQ ID no. 13) is a synthetic anti-sense-strand oligonucleotide (40-mer) encoding amino acid 40 - 53. The primer overlaps with the template by 18 nucleotides.<br>
The N-template (SEQ ID no. 11) is PCR-amplified using the 5' and 3' primer set (SEQ ID no. 12 &amp; 13) using standard techniques and procedures.<br>
The C-terminal half was constructed as follows: a C-template (SEQ ID no. 14) is a synthetic sense-strand oligonucleotide (120-mer) encoding amino acid 59 - 98 of the VL region (SEQ ID no. 10). The template was PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 15) is a synthetic sense-strand oligonucleotide (49-mer) encoding amino acid 50-65 of the VL region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br>
The 3' Primer (SEQ ID no. 16) is a synthetic antisense-strand oligonucleotide (48-mer) encoding amino acid 92 - 107 of the VL region. The primer and the template overlaps by 21 nucleotides.<br>
The C-template (SEQ ID no, 14) is PCR-amplified using the 5' and 3' primer set (SEQ ID no. 15 &amp; 16) using standard techniques and procedures.<br>
For the construction of the FR-patched RFB4 VL domain, the N-tcmplate (SEQ ID no. 11, 108-mer), C-template (SEQ ID no. 14, 120-mer), and their respective 5'- and 3' primers (SEQ ID no. 12 &amp; 13 for N-template, and SEQ ID no. 15 &amp; 16 for C-template), were synthesized on an automated Applied Biosystem 380B DNA synthesizer.        The    oligonucleotides    were    desalted    by    passing    through    a<br>
28<br><br>
CHROMOSPIN-10 column (Clonetech). The oligonucleotides were adjusted to a final concentration of 20 uM. One u.1 of template oligonucleotides at various dilutions (IOX, 100X, 1000X and 10000X, etc.) were mixed with 5 ul of their corresponding flanking primers in the presence of 10 ul of lOx PCR Buffer (500 mM KC1, 100 mM Tris.HCl buffer, pH 8.3, 15 mM MgC12) and 5 units of AMPLITAQ™ DNA polymerase (Perkin Elmer). This reaction mixture was adjusted to a final volume of 100 \x\ and subjected to 30 cycles of PCR reaction consisting of denaturation at 94°C for 1 minute, annealing at 50°C for 1 minutes, and polymerization at 72°C for 1 minute. The PCR reaction mixtures were analyzed under 2% agarose gel electrophoresis. The highest template dilution that gave rise to sufficiently abundant product of tiie right size would be chosen for further processing.<br>
Double-stranded PCR-amplified products for the N- and C-templates were gel-purified, restriction-digested with Spel. The restricted N- and C-double stranded DNA were ligated at the Spel site, and the ligatged products were subjected to another round of PCR-amplification using the 5' primer for the N-template (SEQ ID no. 12) and the 3' primer for the C-template (SEQ ID no. 16). The PCR product with a size of -320 was directly cloned into the TA cloning vector (Invitrogen). The sequence of the cloned fragment was confirmed by Sanger's method (Sanger op. cit.) to be identical to the designed VL sequence. The confirmed sequence was used to replace the VL sequence of a light chain expression vector containing an IgH promoter, an Ig enhancer, a human kappa constant region genomic sequence, and a selectable marker, hyg. The final light chain expression vector is designated as hpRFB4pSMk.<br>
Expression and affinity of FR-patched antibody<br>
The expression plasmids hpRFB4pSMh and hpRFB4pSMk were linearized and co-transfected into mouse Sp2/0 cells. Cells transfected with the plasmids were selected in the presence of mycophenolic acid and/or hygromycin B conferred by the gpt and hyg genes on the plasmids by standard methods. Cells surviving selection were tested for human antibody secretion using ELISA methods. Clones that were identified to be<br>
29<br><br>
secreting human antibody were expanded for production in 500 ml roller bottles. Antibodies were purified using standard protein A columns. The purified antibody was analyzed in a SDS-PAGE gel under both reducing and non-reducing conditions (Figure 4). The affinity of the FR-patched antibody (hpRFB4) was first evaluated by flow cytometry. Raji cells (5 x 105) were incubated with 1 ug of either purified hpRFB4 or chimeric RFB4 (cRFB4) in a final volume of 100 p.l of PBS supplemented with 1% FCS and 0.01% (w/v) sodium azide (PBS-FA). cRFB4 differs from hpRFB4 in the variable region sequences which were derived directly from the murine parent without modifications. The mixtures were incubated for 30 minutes at 4°C and washed three times with PBS to remove unbound antibodies. The binding levels of the antibodies to Raji cells were assessed by the addition of a 20 x diluted FITC-labeled, goat anti-human IgGl, Fc fragment-specific antibodies (Jackson IrnmunoResearch, West Grove, PA) in a final volume of 100 u,l in PBS-FA, and incubating for 30 minutes at 4°C. The mixture was washed three times with PBS and fluorescence intensities were measured by a FACSCAN fluorescence-activated cell sorter (Becton Dickinson, Bedford, MA) (Figure 5) The results indicated that both antibodies bound well to Raji cells with similar affinity.<br>
To compare the affinity of the antibody before and after re-engineering the VH and VL sequences of RFB4, a competitive binding assay was performed. Fixed amount (10 x dilution from stock) of FITC-conjugated RFB4 (Ancell Corporation, Bayport, MN) was mixed with varying concentrations of either cRFB4 or hpRFB4. The mixtures were added to Raji cells in a final volume of 100 ul in PBS-FA, and incubated for 30 minutes at 4°C. After washing three times with PBS, the fluorescence intensities of Raji cells bound with the FITC-RFB4 were measured by FASCAN (Becton Dickinson, Bedford, MA). The results indicated that FR-patching of the RFB4 sequence did not have significant effects on the affinity of the re-engineered antibody (Figure 6).<br>
30<br><br>
EXAMPLE 2<br>
FR-PATCHED ANTI-CD20 ANTIBODY<br>
Design of genes for FR-patched anti-CD20 light and heavy chain.<br>
The heavy and light chain sequence of a murine anti-CD20 antibody, 1F5 (Ref.) is used as an example to illustrate the approach of using FR-patching to reduce or eliminate immunogenicity of the re-engineered antibody. The sequences of the heavy and light chain variable region for the murine antibody are shown in Figure (7).<br>
In designing the amino acid sequence of the FR-patched immunoglobulin for 1F5, the same set of rules as described previousely applies. However, there are always situations when no appropriate FR's fulfill all the above-mentioned requirements. The FR-patching approach offers a great degree of flexibility allowing the introduction of murine residues in the problematic FR's, or alternatively, inclusion of the original murine FR.'s without modifications. The resultant FR-patched antibody will presumably have significantly reduced immunogenicity compared to a murine or chimeric antibody. An anti-CD20 antibody, 1F5, is used as an example for FR-patching to illustrate these points.<br>
Patching of the individual FR's of the 1F5 VH sequence was done as follows:<br>
a. FR1: the FR1 sequence of the murine VH was compared with the FR1 sequences of human VH from the Kabat's database (Kabat et al., op. cit.). Human FR1 of the highest sequence homology is preferred, particularly at the sequences closest to the CDR1. The human FR1 sequence from LS2'CL has close to 80% of sequence homology to that of the murine anti-CD20 antibody, and the 10 residues adjacent to the CDR1 are identical to the murine parent sequence. Therefore, the human FR1 sequence from. LS2'CL was chosen for patching the FR1 of the anti-CD20 antibody (Figure 8A)<br>
31<br><br>
b.	FR2: the FR2 sequence of the human NEWM was chosen for patching the FR2<br>
sequence of the anti-CD20 antibody. It should be noted that although the third<br>
residue of the NEWM FR2 closest to the CDR1 is not identical to that of the<br>
murine parent sequence, it is a conserved K to R conversion (Figure 8A).<br>
c.	FR3: human heavy chain FR3 sequences with satisfactorily high sequence<br>
homology and identical sequences adjacent to the CDR2 and CDR3 could not be<br>
identified. Although the human FR3 from 783CCL exhibited 78% of sequence<br>
homology, the residues flanking the CDR2 were drastically different, despite the<br>
differences being conserved. For example, the K, A, and L at positions 57, 58 and<br>
60 (Rabat's numbering, Kabat et al., op. cit.) which are the 1st, 2nd, and 4lh residues<br>
closest to the CDR2 are replaced by the conserved human residues R, V and I,<br>
respectively. Nevertheless, the high number of changes in proximity to the CDR2,<br>
albeit conservative, could result in significant conformational changes at the<br>
antigen binding site. Without risking the loss of affinity, and as an illustration on<br>
the flexibility of the FR-patching approach, the FR3 would not be patched with<br>
any of the human FR's. Instead, the murine FR3 sequence was retained in this<br>
particular antibody (Figure 8A).<br>
d.	FR4: there are many human FR4 with the sequence closest to the CDR3 being<br>
identical, and a high degree of homology to the murine parent. In this example,<br>
the hum; n 4G12'CL sequence was selected for patching the FR4 of the anti-CD20<br>
antibody (Figure 8A).<br>
The final design of the FR-patched VH sequence (Figure 9A) for the anti-CD20 antibody is composed of the human LS2'CL FRl, NEWM FR2, murine 1F5 FR3 and 4G12'CL FR4, replacing the original VH FR's of the murine anti-CD20 antibody.<br>
An alternative design would be a patched VH containing the murine CDRs embedded in human LS2'CL FRl, NEWM FR2, 783CCL FR3, and 4G12'CL FR4 (Figure<br>
32<br><br>
10A). For the purpose of illustration, the construction of the former version would be described below.<br>
Using a similar strategy, the sequence design for the FR-patched light chain was constructed as follows:<br>
a.	FRl: human BJ19 was chosen for patching the FRl of the murine VL. This is the<br>
human FRl sequence with the highest homology to the murine parent (61%).<br>
Moreover, some of the human residues that are different from that of the murine<br>
are conserved. For example, the E to D, and K to R conversions at positions 18<br>
and 19, respectively, are conserved changes (Figure 8B).<br>
b.	FR2: although there is a human FR2, MOT, that was found to be of high sequence<br>
homology (73%) to the murine FR2, the Tryptophan at position 32 (Rabat's<br>
numbering, Kabat et al., op. cit), the 3rd residues closest to the CDR2, was<br>
replaced by a non-conservative Valine in the MOT FR2 sequence. This<br>
replacement potentially might have a significant effect on the final conformation<br>
of the antigen binding site. It was therefore determined thai the murine FR2 of" the<br>
VL domain would remain in the design of the FR-patched antibody (Figure 8B).<br>
c.	FR3: human WES was chosen for patching the FR3 of the murine VL. FR3 has<br>
the longest sequence, and the sequence homology between WES and the murine<br>
FR3 is 71%, with the three human residues flanking the CDR2 and CDR3 being<br>
identical to that of the murine (Figure 8B).<br>
d.	FR4: human X FR4 sequence from NIG-58 was chosen for patching the FR4 of the<br>
murine VL, for similar reasons. The sequences are 72% homologous to the stretch<br>
of 7 resid' :es adjacent to the CDR3 being identical between the human and murine<br>
(Figure 8B).<br>
33<br><br>
The final design of the FR-patched VL sequence (Figure 9B) for the anti-CD20 antibody is composed of the human BJ19 FR1, murine 1F5 FR2, WHS FR3, and NTG-58 FR4, replacing the original VL PR's of the anti-CD20 antibody. An alternative design of FR-patched VL would be composed of the human BJ19 FR1, MOT FR2, WES FR3, and NIG-58 FR4, forming the scaffold supporting the CDR loops (Figure 10B). For the purpose of illustration in this application, only the construction of the former FR-patched VL would be described below.<br>
Construction of the FR-patched heavy and light chain genes<br>
The designed heavy and light chain variable region sequences of the FR-patched antibody were assembled by a combination of oligonucleotide synthesis and PCR using a variety of published methods.<br>
To construct the FR-patched heavy chain variable region sequence (SEQ ID no. 17), the full DNA sequence was divided into two halves. The N-terminal half and the C-terminal half were constructed separately by PCR and the complete variable region sequence is formed by joining the N- and C-terminal halves at the Spet site.<br>
The N-terminal half was constructed as follows: a N-template (SEQ ID no. 19) is a synthetic sense-strand oligonucleotide (114-mer) encoding amino acid 12-49 of the VH region (SEQ ID no. 18). The template is PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 20) is a synthetic sense-strand oligonucleotide (57-mer) encoding amino acid 1-19 of the VII region. The 3" end of the primer overlaps with the 5' end of the template by 24 nucleotides.<br>
The 3' Primer (SEQ ID no. 21) is a synthetic anti-sense-strand oligonucleotide (55-mer) encoding amino acid 43 - 60. The primer overlaps with the template by 21 nucleotides.<br>
34<br><br>
were analyzed under 2% agarose gel electrophoresis. The highest template dilution that gave rise to sufficiently abundant product of the right size would be chosen for further processing.<br>
Double-stranded PCR-amplified products for the N- and C-templates were gel-purified, restriction-digested with Kpnl site. The N- and C-doubie stranded DNA were ligated at the Spel site, and the ligated products were subjected to another round of PCR amplification using the 5' primer for the N-template (SEQ ID no. 19) and the 3' primer for the C-template (SEQ ID no. 22). The PCR product with a size of-350 was directly cloned into the TA cloning vector (Invitrogen). The sequence of die cloned fragment was confirmed by Sanger's method (Sanger et al. op. cit.) to be identical to the designed VH sequence. The confirmed sequence was used to replace the VH sequence of a heavy chain expression vector containing an IgH promoter, an Ig enhancer, a human IgGl constant region genomic sequence, and a selectable marker, gpt. The final heavy chain expression vector is designated as hplF5pSMh.<br>
To construct the FR-patched light chain variable region sequence (SEQ ID no. 25), the full length VL variable region sequence is divided into two halves. The N-terminal and C-terminal halves are assembled separately by PCR and joined together via the BspEI site.<br>
The N-terminal half was constructed as follows: a N-template (SEQ ID no. 27) is a synthetic sense-strand oligonucleotide (129-mer) encoding amino acid 9-51 of the VL region (SEQ ID no. 26). The template was PCR-amplificd by Iwo primers:<br>
The 5' Primer (SEQ ID no. 28) is a synthetic sense-strand oligonucleotide (45-mer) encoding amino acid 1 - 15 of the VH region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br><br><br>
35<br><br><br>
The N-template (SEQ ID no. 19) is PCR-amplified using the 5' and 3* primer set (SEQ ID no. 20 &amp; 21) using standard techniques and procedures.<br>
The C-terminal half was constructed as follows: a C-lemplate (SEQ ID no. 22) is a synthetic sense-strand oligonucleotide (126-mer) encoding amino acid 70 - 111 of the VH region (SEQ ID no. 18). The template was PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 23) is a synthetic sense-strand oligonucleotide (61-mer) encoding amino acid 57 - 76 of the VH region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br>
The 3' Primer (SEQ ID no. 24) is a synthetic antisense-strand oligonucleotide (59-mer) encoding amino acid 105 - 123 of the VH region. The primer and the template overlaps by 21 nucleotides.<br>
The C-template (SEQ ID no. 22) was PCR-amplified using the 5' and 3' primer set (SEQ ID no. 23 &amp; 24) using standard techniques and procedures.<br>
For the construction of the FR-patched 1F5 VH domain, the N-template (SEQ ID no. 19, 114-mer), C-template (SEQ ID no. 22, 126-mer), and their respective 5'- and 3' primers (SEQ ID no.20 &amp; 21 for N-template, and SEQ ID no.23 &amp; 24 for C-template), were synthesized on an automated Applied Biosystem 380B DNA synthesizer. The oligonucleotides were desalted by passing through a CHROMOSPIN-10™ column (Clonetech). The oligonucleotides were adjusted to a final concentration of 20 u.M. One ul of template oligonucleotides at various dilutions (10X, 100X, 1000X and 10000X, etc.) were mixed with 5 ui of their corresponding flanking primers in the presence of 10 jd of lOx PCR Buffer (500 mM KC1, 100 mM Tris.HCl buffer, pH 8.3, 15 mM MgCl2) and 5 units of AMPLITAQ™ DNA polymerase (Perkin Elmer). This reaction mixture was adjusted to a final volume of 100 u.1 and subjected to 30 cycles of PCR reaction consisting of denaturation at 94°C for 1 minute, annealing at 50°C for 1.5 minutes, and polymerization at 72°C for 1 minute.   The PCR reaction mixtures<br>
36<br><br>
The 3' Primer (SEQ ID no. 29) is a synthetic anti-sense-strand oligonucleotide (40-mer) encoding amino acid 45 - 57. The primer overlaps with the template by 21 nucleotides.<br>
The N-template (SEQ ID no. 27) was PCR-amplified using the 51 and 3' primer set (SEQ ID no. 28 &amp; 29) using standard techniques and procedures.<br>
The C-terminal half was constructed as follows: a C-template (SEQ ID no. 30) is a synthetic sense-strand oligonucleotide (120-mer) encoding amino acid 61 - 100 of the VH region (SEQ ID no. 26). The template is PCR-amplified by two primers:<br>
The 5' Primer (SEQ ID no. 31) is a synthetic sense-strand oligonucleotide (43-mer) encoding amino acid 54 - 67 of the VH region. The 3' end of the primer overlaps with the 5' end of the template by 21 nucleotides.<br>
The 3' Primer (SEQ ID no. 32) is a synthetic antisense-strand oligonucleotide (42-mer) encoding amino acids 94-107 of the VH region. The primer and the template overlap by 21 nucleotides.<br>
The C-template (SEQ ID no. 30) was PCR-amplified using the 5' and 3' primer set (SEQ ID no. 31 &amp; 32) using standard techniques and procedures.<br>
For the construction of the FR-patchcd 1F5 VL domain, (he N-template (SEQ ID no. 27, 129-mer), C-template (SEQ ID no. 30, 120-mer), and their respective 5'- and 3' primers (SEQ ID no.28 &amp; 29 for N-template, and SEQ ID no.3I &amp; 32 for C-template), were synthesized on an automated Applied Biosystem 380B DNA synthesizer. The oligonucleotides were desalted by passing through a CHROMOSPIN-10™ column (Clonetech). The oligonucleotides were adjusted to a final concentration of 20 uM. One pi of template oligonucleotides at various dilutions (10X, 100X, 1000X and 10000X, etc.) were mixed with 5 pi of their corresponding flanking primers in the presence of 10 pi of lOx PCR Buffer (500 raM KC1. 100 mM Tris.HCl buffer. pH 8.3.<br>
37<br><br>
15 mM MgCl2) and 5 units of AMPLITAQ™ DNA polymerase (Perkin Elmer). This reaction mixture was adjusted to a final volume of 100 u.1 and subjected to 30 cycles of PCR reaction consisting of denaturation at 94°C for 1 minute, annealing at 50°C for 1.5 minutes, and polymerization at 72°C for 1 minute. The PCR reaction mixture were analyzed under 2% agarose gel electrophoresis. The highest template dilution that gave rise to sufficiently abundant product of the right size would be chosen for further processing.<br>
Double-stranded PCR-amplified products for the N- and C-templates were gel-purified, restriction-digested with Spel site. The N- and C-double stranded DMA were ligated at the BspEI site, and amplified using the 5' primer for the N-template (SEQ ID no. 12) and the 3' primer for the C-template (SEQ ID no. 16). The PCR producl with a size of ~320 was directly cloned into the TA cloning vector (Invi(rogen). The sequence of the cloned fragment was confirmed by Sanger's method (Sanger et al., op. cit.) to be identical to the designed VL sequence. The confirmed sequence was used to replace the VL sequence of a light chain expression vector containing an Igl I promoter, an Ig enhancer, a human kappa constant region genomic sequence, and a selectable marker, hyg. The final light chain expression vector is designated as hplF5pSMk.<br>
SEQUENCE LISTING<br>
(1) GENERAL INFORMATION:<br>
(2) INFORMATION FOR SEQ ID NO: 1:<br>
(i)SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH:<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single (D)TOPOLOGY:linear<br>
(ii)     MOLECULE TYPE: DNA (oligonucleotide) (xi) SEQUENCE DESCRIPTION: SEQ ID NO:l:<br>
38<br><br>
FR-patched RFB4<br>
VH:<br>
Full length cDNA sequence (SEQ ID no. 1):<br>
GAAGTGCAGCTGCTGGAGTCTGGGGGAGGCTTAGTGCAGCCTGGAGGGTC<br>
CCTGAGGCTCTCCTGTGCAGCCTCTGGATTCTCCTTCAGTATCTATGACAT<br>
GTCTTGGGTTCGCCAGGCACCGGGAAAGGGGCTGGAGTGGGTCGCATACA<br>
TTAGTAGTGGTGGTGGTACCACCTACTATCCAGACACTGTGAAGGGCCGA<br>
TTCACCATCTCCAGAGACAATGCCAAGAACTCCCTGTACCTGCAAATGAA<br>
CAGTCTGAGGGTGGAGGACACAGCCTTATATTACTGTGCAAGACATAGTG<br>
GCTACGGTAGTAGCTACGGGGTTTTGTTTGCTTACTGGGGCCAAGGGACTC<br>
TGGTCACTGTCTCTTCA<br>
Full length amino acid sequence (SEQ ID no. 2):<br>
EVQLLESGGGLVQPGGSLRLSCAASGFSFSIYDMSWVRQAPGKGLEWVAYIS SGGGTTYYPDTVKGRFTISRDNAKNSLYLQMNSLRVEDTALYYCARHSGYGS SYGVLFAYWGQGTLVTVSS<br>
N-terminal sense strand template DNA sequence (SEQ ID no. 3): CCTGGAGGGTCCCTGAGGCTCTCCTGTGCAGCCTCTGGATTCTCCTTCAGT ATCTA7TJACATGTCTTGGGTTCGCCAGGCACCGGGAAAGGGGCTGGAGTG GGTCGCATAC<br>
5' Primer for N-template (SEQ ID no. 4).<br>
GAAGTGCAGCTGCTGGAGTCTGGGGGAGGCTTAGTGCAGCCTGGAGGGTC<br>
CCTGAGG<br>
3' Primer for N-template (SEQ ID no. 5) GTAGGTGGTACCACCACCACTACTAATGTATGCGACCCACTCCAGCCC<br>
C-terminal sense strand template DNA sequence (SEQ ID no. 6): TTCACCATCTCCAGAGACAATGCCAAGAACTCCCTGTACCTGCAAATGAA CAGTCTGAGGGTGGAGGACACAGCCTTATATTACTGTGCAAGACATAGTG GCTACGGTAGTAGCTACGGGGTTTTGTTTGCT<br>
5' Primer for C-template (SEQ ID no. 7) GGTGGTACCACCTACTATCCAGACACTGTGAAGGGCCGATTCACCATCTCC<br>
AGAGACAAT<br>
3' Primer for C-template (SEQ ID no. 8) TGAAGAGACAGTGACCAGAGTCCCTTGGCCCCAGTAAGCAAACAAAACCC<br>
CGTAGCT Joining site: Kpnl<br>
39<br><br>
VK:<br>
Full length cDNA sequence (SEQ ID no. 9)<br>
GATATCCAGATGACCCAGTCTCCATCCTCCCTGTCTGCCTCTGTGGGAGAC<br>
AGAGTCACCATTAGTTGCAGGGCAAGTCAGGACATTAGCAATTATTTAAA<br>
CTGGTATCAGCAGAAACGAGGTAAGGCTCCGAAACTCCTGATCTACTACA<br>
CTAGTATATTACACTCAGGAGTCCCATCAAGGTTCAGTGGCAGTGGGTCTG<br>
GAACAGAATTTACTCTCACCATTAGCTCCCTGCAGCCAGAAGATTTTGCCA<br>
CTTACTTTTGCCAACAGGGTAATACGC1TCCGTGGACGTTCGGTGGAGGCA<br>
CCAAGGTGGAAATCAAA<br>
Full length amino acid sequence (SEQ ID no. 10)<br>
DIQMTQSPSSLSASVGDRVTISCRASQDISNYLNWYQQKPGKAPKLLIYYTSIL<br>
HSGVPSRFSGSGSGTEFTLTISSLQPEDFATYFCQQGNTLPWTFGGGTKVEIK<br>
N-terminal sense strand template DNA sequence (SEQ ID no. 11) CTGTCTGCCTCTGTGGGAGACAGAGTCACCATTAGTTGCAGGGCAAGTCA GGACATTAGCAATTATTTAAACTGGTATCAGCAGAAACCAGGTAAGGCTC CGAAACTC<br>
5' Primer for N template (SEQ ID no. 12) GATATCCAGATGACCCAGTCTCCATCCTCCCTGTCTGCCTCTGTGGGAGAC<br>
3' Primer for N-template (SEQ ID no. 13) ATATACTAGTGTAGTAGATCAGGAGTTTCGGAGCCTTACC<br>
C-terminal sense strand template DNA sequence (SEQ ID no. 14) CCATCAAGGTTCAGTGGCAGTGGGTCTGGAACAGAATTTACTCTCACCATT AGCTCCCTGCAGCCAGAAGATTTTGCCACTTACTTTTGCCAACAGGGTAAT ACGCTTCCGTGGACGTTC<br>
5' Primer for C-template (SEQ ID no. 15) CTACACTAGTATATTACACTCAGGAGTCCCATCAAGGTTCAGTGGCAGT<br>
3' Primer for C-template (SEQ ID no. 16) TTTGATTTCCACCTTGGTGCCTCCACCGAACGTCCACGGAAGCGTATT<br>
Joining site: Spel AVCTAGAT<br>
40<br><br>
FR-Patched chimeric 1F5<br>
VH:<br>
Full length cDNA sequence (SEQ ID no. 17):<br>
CAGGTGCAACTGGTGGCTTCCGGGGCTGAGGTAAATAAGCCTGGGGCCTC<br>
AGTGAAGGTCTCCTGCAAGGCTTCTGGCTACACATTTACCAGTTACAATAT<br>
GCACTGGGTACGGCAGCCTCCTGGAAGGGGCCTGGAATGGATTGGAGCTA<br>
TTTATCCAGGAAATGGTGATACTAGTTACAATCAGAAATTCAAGGGCAAG<br>
GCCACATTGACTGCAGACAAATCCTCCAGCACAGCCTACATGCAGCTCAG<br>
CAGTCTGACATCTGAGGACTCTGCGGTCTATTACTGTGCAAGATCGCACTA<br>
CGGTAGTAACTACGTAGACTACTTTGACTACTGGGGCCAAGGCACCACTG<br>
TTACAGTCTCCTCTGATCA<br>
Full-length amino acid sequence (SEQ ID no. 18):<br>
QVQLVASGAEVNKPGASVKVSCKASGYTFTSYNMHWVRQPPGRGLEWIGAI YPGNGDTSYNQKFKGKATLTADKSSSTAYMQLSSLTSEDSAVYYCARSHYGS NYVDYFDYWGQGTTVTVSSD<br>
N-terminal sense strand template DNA sequence (SEQ ID no. 19): AATAAGCCTGGGGCCTCAGTGAAGGTCTCCTGCAAGGCTTCTGGCTACAC ATTTACCAGTTACAATATGCACTGGGTACGGCAGCCTCCTGGAAGGGGCC TGGAATGGATTGGA<br>
5' Primer for N-template (SEQ ID no. 20)<br>
CAGGTGCAACTGGTGGCTTCCGGGGCTGAGGTAAATAAGCCTGGGGCCTC<br>
AGTGAAG<br>
3' Primer for N-template (SEQ ID no. 21)<br>
TGTAACTAGTATCACCATTTCCTGGATAAATAGCTCCAATCCATTCCAGGC<br>
CCCT<br>
C-terminal sense strand template DNA sequence (SEQ ID no. 22): TTGACTGCAGACAAATCCTCCAGCACAGCCTACATGCAGCTCAGCAGTCT GACATCTGAGGACTCTGCGGTCTATTACTGTGCAAGATCGCACTACGGTAG TAACTACGTAGACTACTTTGACTAC<br>
5' Primer for C-template (SEQ ID no. 23)<br>
TGATACTAGTTACAATCAGAAATTCAAGGGCAAGGCCACATTGACTGCAG<br>
ACAAATCCTCC<br>
3' Primer for C-template (SEQ ID no. 24) TGATCAGAGGAGACTGTAACAGTGGTGCCTTGGCCCCAGTAGTCAAAGTA<br>
GTCTACGTA<br>
41<br><br>
Joining site: Spel<br>
VK:<br>
Full-length cDNA sequence (SEQ ID no. 25):<br>
GATATTCAACTCACACAGTCTCCATCAAGTCTTTCTGCATCTGTGGGGGAC<br>
AGAGTCACAATTACTTGCAGGGCCAGCTCAAGTTTAAGTTTCATGCACTGG<br>
TACCAGCAGAAGCCAGGATCCTCCCCCAAACCCTGGATTTATGCCACATCC<br>
AACCTGGCTTCCGGAGTCCCTAGTCGCTTCAGTGGCAGTGGGTCTGGGACC<br>
GAGTTCACTCTCACAATCAGCAGTTTGCAGCCTGAAGATTTCGCCACTTAT<br>
TTCTGCCATCAGTGGAGTAGTAACCCGCTCACGTTCGGTGCTGGGACCAAG<br>
CTGACCGTTCTACGG<br>
Full-length amino acid sequence (SEQ ID no. 26):<br>
DIQLTQSPSSLSASVGDRVTITCRASSSLSFMHWYQQKPGSSPKPWIYATSNLA<br>
SGVPSRFSGSGSGTEFTLTISSLQPEDFATYFCHQWSSNPLTFGAGTKLTVLR<br>
N-teraiinal sense strand template DNA sequence (SEQ ID no. 27): TCAAGTCTTTCTGCATCTGTGGGGGACAGAGTCACAATTACTTGCAGGGCC AGCTCAAGTTTAAGTTTCATGCACTGGTACCAGCAGAAGCCAGGATCCTCC CCCAAACCCTGGATTTATGCCACATCC<br>
5' Primer for N-template (SEQ ID no. 28): GATATTCAACTCACACAGTCTCCATCAAGTCTTTCTGCATCTGTG<br>
3' Primer for N-template (SEQ ID no. 29): GGACTCCGGAAGCCAGGTTGGATGTGGCATAAATCCAGGG<br>
C-terminal sense strand template DNA sequence (SEQ ID no. 30): TTCAGTGGCAGTGGGTCTGGGACCGAGTTCACTCTCAGAATCAGCAGTTTG CAGCCTGAAGATTTCGCGACTTATTTCTGCCATCAGTGGAGTAGTAACCCG CTCACGTTCGGTGCTGGG<br>
5' Primer for C-template (SEQ ID no. 31): GGCTTCCGGAGTCCCTAGTCGCTTCAGTGGCAGTGGGTCTGGG<br>
3' Primer for C-template (SEQ ID no. 32): CCGTAGAACGGTCAGCTTGGTCCCAGCACCGAACGTGAGCGG<br>
Joining site: BspEI<br>
42<br><br>
We claim:<br>
1.	A re-engineered, or framework (FR)-patched immunoglobulin containing the heavy and/or light chain variable region sequences from a parent antibody, in which at least one of the compartmentalized framework sequences, defined as FR1, FR2, FR3 and FR4 are replaced, or patched by the corresponding framework sequences from the heavy and light chain immunoglobulin variable region of a different species, wherein said re-engineered immunoglobulin chain comprises framework sequences derived from at least two different sources of immunoglobulin chains, wherein said different immunoglobulin chains can be sourced from different immunoglobulins of the same species or from different immunoglobulins of different species, and such FR-patched immunoglobulin binds specifically to an antigen with affinity comparable to, or within 3-fold of, that of the parent immunoglobulin.<br>
2.	A re-engineered, or FR-patched immunoglobulin according to claim 1, in which the particular FR chosen for patching or replacing each corresponding FR in the parent immunoglobulin:<br>
a.	exhibits the highest degree of homology, or at least 60%, to the<br>
corresponding parent FR;<br>
b.	exhibits identical sequence homology to the corresponding parent FR at<br>
the three amino acids immediately adjacent to the flanking CDR's; and<br>
c.	contains identical amino acid to the corresponding parent FR at positions<br>
known to be close to, or have interactions with the CDR's/antigen binding<br>
site, as evaluated by computer modeling, crystal structure, published<br>
information, or prior experience<br>
3.	A re-engineered, or FR-patched immunoglobulin according to claim 1, in which<br>
the particular FR chosen for patching or replacing each corresponding FR in the<br>
parent immunoglobulin:<br>
43<br>
/<br><br>
a.	exhibits the highest degree of homology, or at least 60%, to the<br>
corresponding parent FR;<br>
b.	exhibits identical sequence homology to the corresponding parent FR at<br>
the four amino acids immediately adjacent to the flanking CDR's; and<br>
c.	contains identical amino acid to the corresponding parent FR at positions<br>
known to be close to, or have interactions with the CDR's/antigen binding<br>
site, as evaluated by computer modeling, crystal structure, published<br>
information, or prior experience.<br>
4.	A re-engineered, or FR-patched immunoglobulin according to claim (1), in which<br>
the particular FR chosen for patching or replacing each corresponding FR in the<br>
parent immunoglobulin:<br>
a.	exhibits the highest degree of homology, or at least 60%, to the<br>
corresponding parent FR;<br>
b.	exhibits the highest degree of sequence homology to the corresponding<br>
parent FR, preferably 100%, or contains conservatively similar amino<br>
acids, such as, gly, ala; val, ile, leu; asp, glu; asn, gin; ser, thr; lys, arg; and<br>
phe, tyr, at the three amino acids immediately adjacent to the flanking<br>
CDR's; and<br>
c.	contains identical, or conservatively similar amino acids (as listed in claim<br>
4b) to the corresponding parent FR at positions known to be close to, or<br>
have interactions with the CDR's/antigen binding site, as evaluated by<br>
computer modeling, crystal structure, published information, or prior<br>
experience<br>
5.	A re-engineered, or FR-patched immunoglobulin according to claim (1), in which<br>
the particular FR chosen for patching each corresponding FR in the parent<br>
immunoglobulin:<br>
44<br><br>
a.	exhibits the highest degree of homology, or at least 60%, to the<br>
corresponding parent FR;<br>
b.	exhibits the highest degree of sequence homology to the corresponding<br>
parent FR, preferably 100%, or contains conservatively similar amino<br>
acids (as listed in claim 4b) at the four amino acids immediately adjacent<br>
to the flanking CDR's; and<br>
c.	contains identical, or conservatively similar amino acids (as listed in claim<br>
4b) to the corresponding parent FR at positions known to be close to, or<br>
have interactions with the CDR's/antigen binding site, as evaluated by<br>
computer modeling, crystal structure, published information, or prior<br>
experience.<br>
A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, 3, 4 and 5 containing the heavy and/or light chain variable region sequences from a parent antibody, in which the particular FR chosen for patching each corresponding FR in the parent immunoglobulin comprises re-introduced amino acids from the parent immunoglobulin framework outside the Kabat and Chothia CDRs, wherein the back mutated amino acids replace corresponding amino acids in the patching FR , which is the particular FR derived from a different source used for patching, or that replaces the original FR of, the parent immunoglobulin, and each of said back mutated amino acids:<br>
a.	is adjacent to a CDR in the donor immunoglobulin sequence, or<br>
b.	contains an atom within a distance of 4 A of a CDR in said re-engineered<br>
immunoglobulin<br>
A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, 3,4 and 5 containing the heavy and/or light chain variable region sequences from a parent antibody, in which the particular FR chosen for patching each corresponding FR in the parent immunoglobulin comprises re-introduced amino acids from the parent immunoglobulin framework outside the Kabat and Chothia CDRs, wherein<br>
45<br><br>
the back mutated amino acids replace corresponding amino acids in the patching FR, which is the particular FR derived from a different source used for patching, or that replaces the original FR of, the parent immunoglobulin, and each of said back mutated amino acids:<br>
a.	is adjacent to a CDR in the donor immunoglobulin sequence, or<br>
b.	contains an atom within a distance of 5 A of a CDR in said re-engineered<br>
immunoglobulin.<br>
A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, 3, 4 and 5 containing the heavy and/or light chain variable region sequences from a parent antibody, in which the particular FR chosen for patching each corresponding FR in the parent immunoglobulin comprises re-introduced amino acids from the parent immunoglobulin framework outside the Kabat and Chothia CDRs, wherein the back mutated amino acids replace corresponding amino acids in the patching FR, which is the particular FR derived from a different source used for patching, or that replaces the original FR of, the parent immunoglobulin, and each of said back mutated amino acids:<br>
a.	is adjacent to a CDR in the donor immunoglobulin sequence, or<br>
b.	contains an atom within a distance of 6 A of a CDR in said re-engineered<br>
immunoglobulin.<br>
A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, .3,4 and 5 containing the heavy and/or light chain variable region sequences from a parent antibody, in which the particular FR chosen for patching each corresponding FR in the parent immunoglobulin comprises re-introduced amino acids from the parent immunoglobulin framework outside the Kabat and Chothia CDRs, wherein the back mutated amino acids replace corresponding amino acids in the patching FR, which is the particular FR derived from a different source used for patching,<br>
46<br><br>
or that replaces the original FR of, the parent immunoglobulin, and each of said back mutated amino acids:<br>
a.	is adjacent to a CDR in the donor immunoglobulin sequence, or<br>
b.	is capable of interacting with amino acids in the CDRs, or<br>
c.	is typical at its position for the species of the particular FR chosen for the<br>
patching, and the replaced amino acid in the said FR is rare at its position<br>
for the species from where the FR is derived.<br>
10.	A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, 3, 4, 5, 6, 7, 8, and 9, which specifically binds to an antigen with an affinity of between lO'M-'andlO^M-1.<br>
11.	A ^engineered, or FR-patched immunoglobulin according to claim 1, 2, 3, 4, 5, 6, 7, 8, and 9, which specifically binds to an antigen with an affinity of between lO8 M and lO10M-1.<br>
12.	A re-engineered, or FR-patched immunoglobulin according to claim 1, 2, 3, 4, 5, 6,7, 8, and 9 which is substantially pure.	<br>
13.	A re-engineered immunoglobulin of claim 1, designated hpRFB4.<br>
14.	Are-engineered immunoglobulin of claim 1, designated hplF5.<br>
15.	A pharmaceutical composition comprising a re-engineered, or FR-patched immunoglobulin according to claim 1,2, 3,4, 5, 6, 7, 8, 9 and claim 13 or 14 in a pharmaceutically acceptable carrier.<br><br><br><br>
47<br><br>
16.      A composition comprising the re-engineered immunoglobin of claim 13 or 14.<br>
Dated this 11th Day of December 2003.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWNhbmNlbGxlZCBwYWdlcygyNC0xMC0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-cancelled pages(24-10-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWNsYWltcyhncmFudGVkKS0oMjQtMTAtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-claims(granted)-(24-10-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlKDUtOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-correspondence(5-9-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWNvcnJlc3BvbmRlbmNlKGlwbyktKDI0LTEwLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-correspondence(ipo)-(24-10-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWRyYXdpbmcoMjQtMTAtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-drawing(24-10-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gMTgoMTYtNS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 18(16-5-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gMWEoMTUtMTItMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 1a(15-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gMihncmFudGVkKS0oMjQtMTAtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 2(granted)-(24-10-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gMygxNS0xMi0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 3(15-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gNCgxMi00LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 4(12-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0gNSgxMS0xMi0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form 5(11-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0tcGN0LWlwZWEtNDA5KDI2LTEyLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form-pct-ipea-409(26-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE0MC1tdW1ucC0yMDAzLWZvcm0tcGN0LWlzYS0yMTAoMjYtMTItMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">1140-mumnp-2003-form-pct-isa-210(26-12-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="208331-a-d-c-arc-lamp-assembly-and-a-method-of-cooling-a-d-c-lamp.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208333-process-of-preparation-of-modified-release-pharmaceutical-compositions-of-cefadroxil-and-cephalexin.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208332</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1140/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-Oct-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Jul-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Dec-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LEUNG, SHAWN, SHUI-ON</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RM. 305,40 CONSTITUTION WAY, PORTE LIBERTE, JERSEY CITY, NJ07930,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LEUNG, SHAWN, SHUI-ON</td>
											<td>RM. 305,40 CONSTITUTION WAY, PORTE LIBERTE, JERSEY CITY, NJ07930,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/18512</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-06-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/892,613</td>
									<td>2001-06-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208332-reducing-immunogenicities-of-immunoglobulins-by-framework-patching by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:45:11 GMT -->
</html>
