<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208348-an-e-coli-strain-bl21-de3 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:49:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208348:AN E. COLI STRAIN BL21 [DE3]</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN E. COLI STRAIN BL21 [DE3]</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An E. coli strain BL21[DE3] which harbors a vector such as herein described, which consist of a DNA molecule having the sequence for the luciferase LuAL, Lul64, Ltfl6, Lu39, Lu45, Lu52 or Lu22 or a functional equivalent thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
THE PATENTS RULES, 2003 COMPLETE SPECIFICATION<br>
[See Section 10; rule 13] <br>
"AN E. COLI STRAIN BL21[DE3]"<br>
BAYER HEALTHCARE AG, a body corporate organized under the laws of Germany, D-51368 Leverkusen, Germany,<br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br><br><br>
Coelenterazine-independent eukaryotic luciferase<br><br>
Tab. 3 :	Coelenterazine-independent eukaryotic luciferases<br>
Luciferases can also be distinguished from each other on the basis of their substrate specificity. The most important substrates include coelenterazine (Jones et al., 1999) and luciferin, and also derivatives of the two substances. Diagrams of the substrates, and their transformation by luciferase, are shown below:<br>
Luciferase substrates<br>
Some luciferase substrates, and their transformation, are depicted below by way of example. All the substrates which are shown here are transformed enzymically with the release of light and carbon dioxide (CO2) and consumption of oxygen (O2). The dependence of the reaction on cofactors or energy carriers (e.g. ATP in the case of Firefly Luciferase) is enzyme-specific.<br><br><br><br>
Reporter systems<br>
A reporter gene or indicator gene is the term which is generally given to genes whose gene products can readily be detected using simple biochemical or histochemical methods. At least 2 types of reporter gene are distinguished.<br>
1.	Resistance genes. Resistance genes is the term given to genes whose expression confers on a cell resistance to antibiotics or other substances whose presence in the growth medium leads to cell death when the resistance gene is absent.<br>
2.	Reporter gene. In recombinant DNA technology, the products of reporter genes are used as fused or unfused indicators. The most common reporter genes include beta-galactosidase (Alam et al., 1990), alkaline phosphatase (Yang et al., 1997; Cullen et al., 1992), luciferases and other photoproteins (Shinomura, 1985; Phillips GN, 1997; Snowdowne et al., 1984).<br>
Classification of the species Metridia longa<br>
Arthropoda -&gt;-» Crustacea -+-» Copepoda<br>
The species Metridia longa belongs to the Crustacea, especially the copepoda or zooplancton.<br>
Isolating the cDNA<br>
In order to investigate the bioluminescence activity of the species Metridia longa, specimens were caught in the White Sea (Kartesh Biological Station4 Russia) and stored in liquid nitrogen. In order to prepare cDNA libraries of Metridia longa, the RNA was isolated by the method of Krieg (Krieg et al., 1996) using isothiocyanate.<br><br>
RT-PCR was carried out in order to prepare the cDNA. For this, 10 ug of RNA were incubated with reverse transcriptase (Superscript Gold II) in accordance with the following scheme:<br><br>
In order to inactivate the polymerase, the reaction products were incubated with proteinase K at 37°C for 30 minutes, and the cDNA was precipitated with ethanol. The cDNA was dissolved in water and incubated at 37°C for one hour with Sfil. The reaction products were subjected to gel filtration in order to separate off small fragments. The fractionated cDNA was then ligated into the Sfil-cut and dephosphorylated A,TriplEx2 vector. In order to prepare a X phage expression library, the cloned cDNA fragments were subsequently packaged into X phages using the SMART cDNA Library Construction Kit (Clontech) in-vitro packaging system.<br>
The recombinant phages which contained a cDNA insertion with potential for expressing coelenterazine-dependent luciferases were identified by carrying out a library screening.<br>
For this, bacterial lawns composed of E. coli XL 1-Blue were plated out on 90 mm culture dishes and incubated at 37°C for 10 hours. They were then infected with 2500 phages per culture dish, with this then being followed by an incubation phase of 8 hours at 37°C to enable plaques to be formed. The culture dishes were subsequently stored at 4°C for one hour in order to harden the soft agar.<br><br>
In order to carry out a replica plating, nitrocellulose membranes were saturated with E. coli XLl-Blue suspensions and dried. The dry membranes were laid for 60 seconds on the phage plaques and then laid on fresh agar plates. The agar plates were then incubated at 37°C for 2 hours and 4 ml of SOB medium (+10 mM MgS04, 0.2% maltose) were added. The bacterial lawn was detached, resuspended in LB medium (+20 mM IPTG) and incubated at 37°C for one hour. The bacteria were harvested by cenirifugation and disrupted by ultrasonication, and the bioluminescence activity was determined in a luminometer after having added coelenterazine.<br>
Culture plates giving a positive bioluminescence signal were divided into sectors and a fresh replica plating was carried out. The replica plating was continued until active individual plaques had been identified. In order to subclone the cDNA insertions in the phages in positive plaques [lacuna] took place into the pTriplEX2 vector in E. coli BM25.8 in accordance with the manufacturer's protocol for the SMART cDNA library construction kit. The pTriplEx2 cDNA-transfected E. coli were incubated overnight, at 37°C, in LB medium containing an ampicillin concentration<br>
y<br>
of 100 ug/ml. In order to achieve overexpression, the overnight culture was diluted 1:150 with LB medium and incubated at 37°C for 1 hour. Induction was then effected by adding IPTG (isopropylthiogalactoside) to a final concentration of 20 mM. The induced culture was incubated at 37°C for 1 hour and the bacteria were harvested by centrifugation. The cells were disrupted by ultrasonication in 0.5 ml of SM buffer. The chemiluminescence was measured in a luminometer after adding 10 pi of coelenterazine (10'4 M in methanol).<br>
Three luciferases which exhibited coelenterazine-dependent luciferase activity were<br>
identified. The luciferases were designated Lul64, LuAL and Lu22. The luciferases<br>
-<br>
are described in detail below.<br>
The  invention  also  relates  to  functional  equivalents  of the  three  luciferases. Functional equivalents are those luciferases which have a comparable substrate<br><br>
spectrum, which are secreted and which are at least 70% homologous. A homology of 80% or 90% is preferred. A homology of 95% is particularly preferred.<br>
The luciferases are suitable for use as reporter genes for cellular systems, especially for receptors, for ion channels, for transporters, for transcription factors or for inducible systems.<br>
The luciferases can be used in bacterial systems, for example for titer determination or as substrates for biochemical systems, especially for proteinases.<br>
The luciferases can also be used as reporter enzymes which are coupled to antibodies or other proteins, e.g. for ELISA, for immunohistochemistry or for Western blotting.<br>
The luciferases can be used in BRET (Bioluminescence Resonance Energy Transfer) systems.<br>
The luciferases are also suitable for use as fusion proteins for confocal microscopy or for analyzing protein-protein interactions.<br>
The luciferases can be used as reporter enzymes which are coupled to biotin, NHS, CN-Br or other coupling mediators, e.g. for ELISA or for immobilization.<br>
The luciferases can furthermore be used as reporter enzymes which are coupled to DNA or RNA oligonucleotides, e.g. for Northern and Southern blotting or for real time PCR.<br>
The invention also relates to the purification of the luciferases as wild-type or tag proteins, and to the use of the luciferases in in-vitro translation systems.<br><br><br><br><br><br><br><br>
Enzymic activity and biochemical characterization of the luciferases<br>
The proteins LuAL, Lul64 and Lu22 are enzymes which release light while transforming coelenterazme. They therefore belong to the luciferases. The luciferases can be actively expressed in both bacterial and eukaryotic cells. The luciferases LuAl, Lul64 and Lu22 which are expressed in eukaryotic cells are secreted. No secretion takes place in connection with bacterial expression.<br>
The activity of the luciferases is temperature-dependent. Temperature optima of 22°C (for LuAL) and 27°C (for Lul64) were determined for the luciferases LuAL and Lul64, respectively. The temperature optimum for luciferase Lu22 activity is 4°C or lower.<br><br>
Examples<br>
Plasmids/Constructs<br>
The vectors employed for preparing the constructs which are descibed below were the vectors pcDNA3.1(+) and pTriplEx2 from Clontech and the vector pASMplr (in-house construct possessing cAMP-sensitive promoter elements; ere). The derivatives of the vectors were designated pcDNA3-x, pTriplEx2-x and pASM-x.<br>
LuAL<br>
Fig. 1 shows the plasmid maps of the vectors pTriplEX2-LuAL, pcDNA3-LuAL and<br>
pASM-LuAL<br>
Fig. 2 shows the plasmid maps of the vectors pTriplEX2-Lul64, pcDNA3-Lul64 andpASM-Lul64<br>
Fig. 3 shows the plasmid maps of the vectors pTriplEX2-Lu22, pcDNA3-Lu22 and pASM-Lu22<br>
Coelenterazine derivates as substrates of Lul64<br>
In order to identify substrates for Lul64, 10 /J.1 solutions of different coelenterazine derivatives (10"4M) were in each case incubated with 10 ul of supernatant from CHO-pcDNA3-Lul64 cell lines and the luminescence was measured. The coelenterazines were obtained from Molecular Probes (USA). No differences as compared with luciferase Lul64 were seen in the case of luciferases LuAL and Lu22. Unmodified coelenterazine (Fig. B, coelenterazine a) was identified as being the optimal substrate for Lul64, LuAl and Lu22.<br>
Fig. 4 shows coelenterazine derivatives as potential substrates for Lul64 and a graph of the measurement of luminescence for 30 seconds at 8.7 kV in a luminometer<br><br>
(RLU, relative light units); and also a diagram of the molecular structures of the coelenterazine derivatives.<br>
Enzymic activity of the luciferases Lul64, LuAL and Lu22 in dependence on coelenterazine<br>
Bacterial Expression<br>
The bacterial expression took place in the E. coli strain BL21(DE3) as a result of transforming the bacteria with the expression plasmids pTriplEX2-Lul64, pTriplEX2-LuAL and pTriplEX2-Lu22. The transformed bacteria were incubated at 37°C for 3 hours in LB medium and expression was induced for 4 hours by adding IPTG to a final concentration of 1 mM. The induced bacteria were harvested by centrifugation, resuspended in PBS and disrupted by ultrasonication. Coelenterazine (104 M in methanol) or mciferin (Firefly Luciferin) was added to 5 \i\ of the lysate (5 mg/ml) and the chemiluminescence was measured.<br>
The measurement, in RLU (relative light units)^ took place at 9.5 kV for 30 seconds. Values of 230 000, 320 000 and 260 000 RLU were measured in the case of Lul64, LuAL and Lu22, respectively. The enzymes were expressed in E. coli BL21(DE3) using the vectors pTriplEx2-Lul64, pTriplEx2-LuAL and pTriplEx2-Lu22.<br>
Eukaryotic Expression<br>
Constitutive eukaryotic expression was affected in CHO cells by transfecting the cells with the expression plasmids pcDNA3-Lul64, pcDNA3-LuAL and pcDNA3-Lu22 in transient experiments. For this, 10 000 cells in DMEM-F12 medium were plated, per well, in 96-well microtiter plates and incubated overnight at 37°C. The transfection was effected using the Fugene 6 kit (Roche) in accordance with the manufacturer's instructions. The transfected cells were incubated overnight at 37°C in DMEM-F12 medium. The chemiluminescence in the medium (5 uf) and the cell<br><br>
lysate (5 u.1) was measured for 30 seconds at 9.5 kV in a lummometer, at room temperature, after adding coelenterazine (10"4 M in methanol).<br>
Values of 680 000, 670 000 and 510 000 RLU (relative light units) were measured in the case of Lul64, LuAL and Lu22, respectively. The expression was effected in CHO cells using the vectors pcDNA3-Lul64, pcDNA3-LuAL and pcDNA3-Lu22.<br>
Emission spectra of the luciferases Lul64, LuAL and Lu22<br>
In order to measure the emission spectra, E. coli BL21(DE3) cells were transformed with the plasmids pTriplEx2-Lul64, pTriplEx2-LuAL and pTriplEx2-Lu22 and overexpressed as described under 3.1. 50 ul of coelenterazine (10"4 M) were added to 100 ul volumes of the bacterial lysates and the emission spectra were measured. Graphs of the emission spectra of the luciferases are shown below.<br>
In the case of the luciferases LuAL, Lul64 and Lu22, maximum emission resulting from the substrate transformation takes place at a wavelength of about 490 nm.<br>
Fig. 5 shows the emission spectra of the luciferases Lul64 (A), LuAL.(B) and Lu22 (C) following bacterial expression (RLU, relative light units)<br>
Secretion of the luciferases Lul64, LuAL and Lu22 from CHO cells, taking as examples Lul64 and LuAL<br>
In order to characterize the expression of the luciferases LuAl, Lul64 and Lu22 in eukaryotic cells, CHO cells were stably transfected with the plasmids pcDNA3-LuAl, pcDNA3-Lul64, pcDNA3-Fireluc and pcDNA3.1(+). The resulting clones were cultured in DMEM-F12 medium. Firefly luciferase was used as a positive control for nonsecreted luciferase. The plasmid pcDNA3.1(+) was used as a control plasmid for detecting potential endogenous activity in the CHO parent cell.<br><br>
In order to detect the secretion of the luciferases, 2000 cells were plated on 384-well microtiter plates. After 24 hours, the medium was removed and the cells were washed with Tyrode solution and 30 ul of fresh medium were added. The first measurement (0 h) then took place, in a luminometer at 9.5 kV for 30 seconds, after adding 5 ul of coelenterazine (10'4M) or luciferin in the case of the Firefly luciferase. The 1 h to 5 h measurements took place after one to five hours.<br>
Figure 6 depicts the increase in luciferase activity in the medium in dependence on the time. The Firefly luciferase was not secreted. The luciferases LuAL, Lul64 and Lu22 are secretory luciferases.<br>
Fig. 6 shows the luciferase activity in the CHO cell medium (5 (il) after the CHO cells have been transfected with pcDNA3-LuAL, pcDNA3-Firefly, pcDNA3-Lul64 or pcDNA3 as the control vector without any cDNA insertion. (RLU, relative light units; h, hours; Firefly: Firefly luciferase)<br>
Dependence of the luciferase activity on the temperature<br>
In order to determine the temperature dependence of the luciferases Lu22, Lul64 and LuAL, CHO cells were transiently transfected with the vectors pcDNA3-Lu22, pcDNA3-Lul64 and pcDNA3-LuAl and the luciferase activity in the supernatants was determined at temperatures of between 0 and 47°C. In order to do this, the cell supernatant and the coelenterazine solution were adapted to the measurement temperature for 5 minutes. The measurement took place at 9.5 kV for 30 seconds in a luminometer.<br>
Figure 7 shows the luminescence which was measured, in dependence on the temperature, in the case of the luciferases LuAl, Lul64 and Lu22. The temperature optimum for the luciferase activity of LuAL is 27°C. A temperature optimum of 22°C and of 4°C or lower was determined in the case of Lul64 and Lu22, respectively.<br><br>
Fig. 7 shows the temperature-dependent luciferase activity in CHO cell medium (5 |il) following transfection with pcDNA3-LuAL, pcDNA3-Firefly and pcDNA3-Lul64. (RLU: relative light units; medium: DMEM-F12+10% FCS)<br>
Induced expression of the luciferases Lul64, LuAL and Lu22 in CHO cells taking as an example LuAL<br>
Eukaryotic expression was induced in CHO cells by transfecting the cells with the expression plasmid pASM-LuAL in transient experiments. For this, 10 000 cells in DMEM-F12 medium were plated per well in 96-well microtiter plates and incubated overnight at 37°C. The transfection was effected using the Fugene 6 kit (Roche) in accordance with the manufacturer's instructions. The transfected cells were incubated overnight, at 37°C, in DMEM-F12 medium. They were then induced with Forkolin (10"5M) for 5 hours. The chemiluminescence in the medium and in the cell lysate was then measured, at 9.5 kV for 30 seconds, in a luminometer after-having added coelenterazine (104 M in methanol).<br>
Fig. 8 shows the induced expression of LuAL in CHO cells. The expression was induced for 5 hours with Forskolin (10~5M) at 37°C. The activity was measured in 10 ul of cell supernatant (RLU: relative light units; induction factor: ratio of induced RLU to uninduced RLU)<br>
Use of the luciferases Lul64, LuAL and Lu22 as reporter genes in cellular systems taking as examples the receptors NPY2 and A2A and using LuAL as the reporter gene<br>
In order to be able to analyze the activation of G protein-coupled receptors by receptor-specific ligand in cell-based systems, the cDNA sequence for luciferase LuAL was cloned into the expression vector pASMplr. The expression vector pASMplr contains cAMP-sensitive promoter elements (CRE) which enable the<br><br>
intracellular concentration of cAMP to be measured indirectly. The luciferase serves as the reporter gene in the system.<br>
The use of the luciferases Lu22, Lul64 and Lu22 as reporter genes in cellular systems was demonstrated by taking as an example the G protein-coupled receptors NPY2 (neuropeptide receptor 2) and A2A (adenosine receptor 2a). To do this, the stable clone CHO-pASM-LuAL was transiently transfected with the vector pcDNA3-NPY2 or the vector pcDNA3-A2A. The receptor NPY2 is a Gi-coupled receptor, while the A2A receptor is a Gs-coupled receptor.<br>
The A2A receptor was activated for 4 h by adding 1 uM NECA. The NPY2 receptor was activated by adding 10 uM NPY2 peptide in the presence of 10"5 M Forskolin. The luciferase activity in the medium (30 ul) was measured, at 9.5 kV and for 30 seconds in a luminometer, after having added coelenterazine (10' M).<br>
Fig. 9 shows the use of the luciferases as reporter genes for cellular systems taking as an example the G protein-coupled receptors A2A and NPY2. (RLU: relative light units)<br><br>
References/Patents<br>
Apley EC, Wagner R, Engelbrecht S., Rapid procedure for the preparation of ferredoxin-NADP+ oxidoreductase in molecularly pure form at 36 kDa., Anal Biochem. 1985 Oct;150(l):145-54.<br>
Berestovskaya NG, Shaloiko LA, Gorokhovatsky AY, Bondar VS, Vysotski ES, Maximov JE, von Doehren H, Alakhov YB. Cotranslational formation of active photoprotein obelin in a cell-free translation system: direct ultrahigh sensitive measure of the translation course. Anal Biochem. 1999 Mar l;268(l):72-8.<br>
Bonin et al. Photinus pyralis luciferase. Gene. 1994 141:75-77<br>
Alam J, Cook JL. Reporter genes: application to the study of mammalian gene transcription. Anal Biochem. 1990 Aug l;188(2):245-54<br>
Cullen Bryan R., Malim Michael H., Secreted placental alkaline phosphatase as a eukaryotic reporter gene. Methods in Enzymology. 216:362ff<br>
Hastings, Cell Physiology : Source book, Sperelakis N. (ed.), Academic press, pp665-681,1995<br>
Head JF, Inouye S, Teranishi K, Shimomura O., The crystal structure of the photoprotein aequorin at 2.3 A resolution. Nature. 2000 May 18;405(6784):372-6.<br>
Gould S., Suresh S., Firefly luciferase as a tool in molecular an dcell biology. Anal. Biochem. 1988, 161:501-507<br>
Idelgaufts H., Gentechnologie von A-Z [Recombinant DNA Technology from A-Z]. VCHVerlag, Weinheim, 1993<br><br>
Inouye S, Noguchi M, Sakaki Y, Takagi Y, Miyata T, Iwanaga S, Miyata T, Tsuji FI. Cloning and sequence analysis of cDNA for the luminescent protein aequorin. Proc Natl Acad Sci USA 1985 May;82(10):3154-8<br>
Inouye, S., Kakoi, H., Goto, T., Tetrahedron Lett., 34, 2971-2974 (1976)<br>
Inouye, S., Watanebe, H., Nakamura, H. and Shimomura, O., FEBS letters 481 (2000) 19-25<br>
Jones Keith, Hibbert Frank, Keenan Martine. Glowing jellyfish, luminescence and a molecule called coelenterazine. Tibtech, 1999. 17: 477ff<br>
Kendall Jonathan M., Badminton Michael N. Aequoria victoria bioluminescence moves into an exciting new era. Tibtech. 1998 16:216<br>
Krieg, S., Castles, C, Allred, D., Benedix, M., Fuqua S., RNA from air-dried frozen sections for RT-PCR and differential display. Biotechniques. 1996 Sep;21(3):425-8.<br>
Lorenz WW, McCann RO, Longiaru M, Cormier MJ., Isolation and expression of a cDNA encoding Renilla reniformis luciferase. Proc Natl Acad Sci USA. 1991 May 15;88(10):4438-42.<br>
Lorenz WW, Cormier MJ, O'Kane DJ, Hua D, Escher AA, Szalay AA.,<br>
Expression of the Renilla reniformis luciferase gene in mammalian cells. J Biolumin Qhemilumin. 1996 Jan-Feb;ll(l):31-7.<br>
Mathews J. C. et al. Purification and propeties of rernilla reniformis luciferase. Biochemistry. 1977, 16(1): 85-91<br><br>
Matveev SV, Lewis JC, Daunert S., Genetically engineered obelin as a bioluminescent label in an assay for a peptide. Anal Biochem. 1999 May 15;270(l):69-74.<br>
Phillips GN. Structure and dynamics of green fluorescent protein. Curr Opin Struct Biol. 1997 Dec;7(6):821-7<br>
Sala-Newby et al., Expression of recombinant firefly luciferase. Biochem Soc. Transact. 1992,20:143S<br>
Shimomura O, Johnson FH., Properties of the bioluminescent protein aequorin. Biochemistry. 1969 Oct;8(10):3991-7.<br>
Shimomura O., Bioluminescence in the sea: photoprotein systems. Symp Soc Exp Biol 1985;39:351-72.<br>
Shimomura O, Teranishi K., Light-emitters involved in the luminescence of coelenterazine. Luminescence. 2000 Jan-Feb;15(l):51-8.<br>
Snowdowne KW, Borle AB. Measurement of cytosolic free calcium in mammalian cells with aequorin. Am J Physiol. 1984 Nov;247(5 Pt l):C396-408.<br>
Thompson EM, Nagata S, Tsuji FL, Cloning and expression of cDNA for the luciferase from the marine ostracod Vargula hilgendorfii. Proc Natl Acad Sci USA. 1989Sep;86(17):6567-71.<br>
Thompson EM, Nagata S, Tsuji FL, Vargula hilgendorfii luciferase: a secreted reporter enzyme for monitoring gene expression in mammalian cells. Gene. 1990 Dec 15;96(2):257-62.<br><br>
Ungrin MD, Singh LM, Stocco R, Sas DE, Abramovitz M., An automated aequorin luminescence-based functional calcium assay for G-protein-coupled receptors. Anal Biochem. 1999 Jul 15;272(l):34-42.<br>
Webster JJ, Chang JC, Manley ER, Spivey HO, Leach FR., Buffer effects on ATP analysis by firefly hiciferase. Anal Biochem. 1980 Jul 15; 106(1):7-11.<br>
Yang Te-Tuan, Sinai Pflrisa, Kitts Paul A. Kain Seven R., Quantification of gene expresssion with a secreted alkaline phosphatase reporter system. Biotechnique. 1997 23(6)1110ff<br>
JP 05064583, Tora, (TORAY fND Inc.), Luciferase modified with antigen, antibody, hapten or hormone, etc. - is isolated from Vargula hilgendorfii, useful in bio-luminescent analysis<br>
JP 08027200, Tora, (TORAY IND Inc.), Protein comprising luciferase fused to IgG binding domain - useful as diagnostic reagent in chemiluminescent immunoassays<br>
JP 09187281, Kikk, (KJjCKOMAN Corp.), Antibody-firefly luciferase fused protein - and related products i.&amp;- firefly luciferase fused gene, recombinant DNA and its preparation<br>
US 5196524, Gustafson G.; Ingolia T.; Kirchner G; Roberts J., (Lilly &amp; Co), Recombinant DNA encoding fusion protein with bacterial luciferase activity -comprises coding regions of lux A and B genes isolated from specific naturally bio luminescent bacteria, and DNA linker of specified restriction enzyme recognition site<br>
US 5418155, Milton J., Cornier; William W. Lorenz, Isolated Renilla Luciferase and Method of use \nexeoi<br><br>
US 5422266, Cormier M. J.; Prasher D., (UNTV GEORGIA RES FOUND INC), DNA coding for apo:aequorin - useful for recombinant prodn. of apo:aequorin which is a bio:luminescent marker in chemical and biochemical assays<br>
US 5670356, SherfB.; Wood K. (Promega Corp), Modified firefly luciferase gene -encoding cytoplasmic form of luciferase enzyme<br>
US 5798441 Cormier M. J.; Prasher D., (UNIV GEORGIA RES FOUND INC), New isolated apoaquorin polypeptide - useful as a luminescent marker in bio/chemical assays<br>
WO 0020619, Alan P.; Liu Jingxue, Secreted renilla luciferase<br>
WO 9520653, Renard J P; Thompson E. M.; Thompson K, (INRA INST NAT RECH AGRONOMIQUE), Detecting transgenic embryos using Vargula luciferase gene as a marker - allows identification of such embryos at the blastocyst stage, also transgenic animals, embryos and cell line contg. this gene<br>
WO 9938999, Roelant C (Packard Instr. Co. Inc.), Detecting the presence of renilla luciferase in a sample by detecting luminescence of the sample<br><br>
We claim:<br>
1. An E. coli strain BL21[DE3] which harbors a vector such as herein described, which consist of a DNA molecule having the sequence for the luciferase LuAL, Lul64, Ltfl6, Lu39, Lu45, Lu52 or Lu22 or a functional equivalent thereof.<br>
Dated   this	9th     day of   May, 2003.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="208346-a-nucleoside-compound-of-formula-i-and-the-composition-of-the-said-nucleoside-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208349-a-tuneable-phase-shifter-and-or-attenuator.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208348</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>494/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>32/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jul-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-May-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BAYER HEALTHCARE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-51368 LEVERKUSEN, GERMANY.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STEFAN GOLZ</td>
											<td>BUCKMANNSMUHLE 46, D-45326 ESSEN, GERMANY.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BERND KALTHOF</td>
											<td>CLAUDIUSWEG 9, D-42115 WUPPERTAL, GERMANY.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SVELTANA MARKOVA</td>
											<td>AKADEMGORODOK 20 AP. 48, 660036 KRASNOYARSK, RUSSIAN FEDERATION.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LUDMILA FRANK</td>
											<td>AKADEMGORODOK 19 AP. 61, 660036 KRASNOYARSK, RUSSIAN FEDERATION.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>EUGENE VYSOTSKI</td>
											<td>LADO KECOVELI ST. 35, APT. 108, 660100 KRASNOYARSK, RUSSIAN FEDERATION.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/53, C12N 9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP01/13597</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>100 58 091.2</td>
									<td>2000-11-23</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208348-an-e-coli-strain-bl21-de3 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:49:41 GMT -->
</html>
