<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208698-subtilase-enzymes-having-an-additional-amino-acid-residue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:19:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208698:SUBTILASE ENZYMES HAVING AN ADDITIONAL AMINO ACID RESIDUE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">SUBTILASE ENZYMES HAVING AN ADDITIONAL AMINO ACID RESIDUE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a subtilase enzymes of the 1-81 and 1-82 sub-groups having at least one additional amino acid residue in position 102 of the active site loop (b) region from position 95 to 103, whereby said additional amino acid residues correspond to the insertion of at least one amino acid residue between positions 102 and 103.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title:  Subtilase enzymes of the I-Sl and I-S2 sub-groups having an additional amino acid residue in an active site loop region.<br>
TECHNICAL FIELD<br>
This invention relates to novel subtilase enzymes of the I-Sl and I-S2 sub-groups having at least One additional amino acid residue in position 102 of the active site loop (b) region from position 95 to 103. These proteases are useful exhibiting excellent or improved wash performance when used in detergents; cleaning and detergent compositions. The invention further relates to genes coding for the expression of said enzymes when inserted into a suitable host cell or organism; and such host cells transformed therewith and capable of expressing said enzyme variants, and methods for producing the novel enzymes.<br>
BACKGROUND OF THE INVENTION<br>
In the detergent industry enzymes have for more than 30 years been implemented in washing formulations. Enzymes used in such formulations comprise proteases, lipases, amylases, cellulases, as well as other enzymes, or mixtures thereof. Commercially most important enzymes are proteases.<br>
An increasing number of commercially used proteases are protein engineered variants of naturally occurring wild type proteases, e.g. DURAZYM* (Novo Nordisk A/S), RELASE* (Novo Nordisk A/S), MAXAPEM* (Gist-Brocades N.V.), PURAFECT* (Genencor International, Inc.).<br>
Further a number of protease variants are described in the art, such as in EP 130756 (GENENTECH) (corresponding to US Reissue Patent No. 34,606 (GENENCOR));  EP 214435 (HENKEL); WO 87/04461 (AMGEN); WO 87/05050 (GENEX); EP 260105 (GENENCOR); Thomas, Russell, and Fersht (1985) Nature  318 375-376; Thomas,<br><br>
Russell, and Fersht (1987) J,   Mol.   Biol.   193 803-813; Russel and Fersht Nature 328 496-500 (1987); WO 88/08028 (Genex);   WO 88/08033 (Amgen); WO 95/27049 (SOLVAY S.A.); WO 95/30011 (PROCTER Sc  GAMBLE COMPANY); WO 95/30010 (PROCTER &amp; GAMBLE COMPANY); WO 95/29979 (PROCTER &amp; GAMBLE COMPANY); US 5.543.302 (SOLVAY S.A.); EP 251 446 (GENENCOR) ; WO 89/06279 (NOVO NORDISK A/S) ; WO 91/00345 (NOVO NORDISK A/S) ; EP 525 510 Al (SOLVAY); and WO 94/02618 (GIST-BROCADES N.V.),<br>
However, even though a number of useful protease variants have been described, there is still a need for new improved proteases or protease variants for a number of industrial uses.   )<br>
Therefore, an object of the present invention, is to provide improved proteases or protein engineered protease variants, especially for use in the detergent industry.<br>
SUMMARY OF THE INVENTION<br>
The present inventors have found that subtilisins wherein at least one of the active site loops are longer than those presently known, exhibit improved wash performance properties in detergent compositions. The identification thereof was done in constructing subtilisin variants, especially of the subtilisin<br>
3 09 (BLSAVI or Savinase®) , exhibiting improved wash performance properties in detergent compositions relative to the parent wild type enzyme. This has been described in our earlier application DK1332/97.<br>
It has now been found that certain subtilases or variants thereof of the I-Sl (true " subtilisins" ) and I-S2 (high alkaline subtilisins) sub-groups having at least one additional amino acid residue in position 102 (or rather between positions 102 and 103) of the active site loop (b) region from position 95 to 103, exhibit surprisingly improved wash performance in<br><br>
comparison to those presently known and those described in said application.<br>
The improved proteases according to the invention may be obtained by isolation from natural resources or by the introduction of at least one further amino acid residue (an insertion) in the active site loop (b) between positions 102 and 103 in a wild type subtilase (for a definition of the active site loops and the numbering of positions see below) .<br>
Although this finding was done in subtilisin 3 09 it is predicted that it will be possible to produce or isolate similar advantageous subtilases or subtilase variants.<br>
Furthermore it will be possible to specifically screen natural isolates to identify novel wild type subtilases comprising an active site loop (b) which is longer than the corresponding active site loop in known wild type subtilases, such as subtilisin 309, which subtilases can be considered to have an inserted amino acid residue between positions 102 and 103, and exhibiting excellent wash performance in a detergent, in comparison to their closest related known subtilisin, such as subtilisin 309.<br>
Concerning alignment and numbering reference is made to Figs. 1, la, 2 and 2a below showing alignments between sxibtilisin BPN' (BASBPN) (a) and subtilisin 309 (BLSAVI) (b) , and alignments between subtilisin BPN' (a) (BASBPN) and subtilisin Carlsberg (g) . In Figs. 1 and 2 the alignments were established by the use of the GAP routine of the GCG package as indicated below, whereas the alignments of Figs, la and 2a are the same as shown in WO 91/00345. These alignments are in this patent application used as a reference for numbering the residues.<br>
The seven active site loops (a) to (g) (including both the end amino acid residues indicated) are here defined to encompass the amino acid residues in the segments given below<br><br>
(a)	the	region between amino acid residue	3 3 and 43;<br>
(b)	the	region between amino acid residue	95 and 103;<br>
(c)	the	region between amino acid residue	125 and 132;<br>
(d)	the	region between amino acid residue	153 and 173;<br>
(e)	the	region between amino acid residue	181 and 195;<br>
(f)	the	region between amino acid residue	202 and 204;<br>
(g)	the	region between amino acid residue	218 and 219.<br>
Accordingly, in a first aspect the invention relates to an isolated (i.e. greater than 10 % pure) subtilase enzymes of the I-Sl and I-S2 sub-groups having at least one additional amino acid residue in position 102 of the active site loop (b) region from position 95 to 103, whereby said additional amino acid residue (s) correspond to the insertion of at least one amino acid residue between positions 102 and 103 .<br>
In a second aspect the invention relates to an isolated DNA sequence encoding a subtilase variant of the invention.<br>
In a third aspect the invention relates to an expressioi vector comprising an isolated DNA sequence encoding a subtilase variant of the invention.<br>
In a fourth aspect the invention relates to a microbial lost cell transformed with an expression vector according to th€ fourth aspect.<br>
In a further aspect the invention relates to the production of the subtilisin enzymes of the invention.<br>
The enzymes of the invention can generally be produced k either cultivation of a microbial strain from which the enzyme was isolated and recovering the enzyme in substantially pure form; or by inserting an expression vector according to the fourth aspect of the invention into a suitable microbial host.<br><br>
cultivating the host to-express the desired subtilase enzyme, and recovering the enzyme product.<br>
Further the invention relates to a composition comprising a subtilase or subtilase variant of the invention.<br>
Even further the invention relates to the use of the enzymes of the invention for a number of industrial relevant uses, in particular for use in cleaning compositions and cleaning compositions comprising the mutant enzymes, especially detergent compositions comprising the mutant subtilisin enzymes.<br>
DEFINITONS<br>
Prior to discussing this invention in further detail, the following terms and conventions will first be defined.<br><br>
NOMENCLATURE OF AMINO ACIDS<br>
A	=	Ala    =   Alanine<br>
V	=	Val    =   Valine<br>
L	=	Leu    =   Leucine<br>
 I	=	lie    =    Isoleucine<br>
P	=	Pro    =   Proline<br>
F	=	Phe    =    Phenylalanine<br>
W	=	Trp    =   Tryptophan<br>
M	=	Met    =   Methionine<br>
) G	=	Gly    =   Glycine<br>
S	=	Ser    =   Serine<br>
T	=	Thr    =   Threonine<br>
C	=	Cys    =   Cysteine<br>
Y	=	Tyr    =   Tyrosine<br>
 N	=	Asn    =   Asparagine<br>
Q	=	Gin    =   Glutamine<br>
D	=	Asp    =   Aspartic Acid<br>
E	=	Glu    =   Glutamic Acid<br>
K	-	Lys    =   Lysine<br>
R	=	Arg    =   Arginine<br>
H	=	His    =   Histidine<br>
X	=	Xaa    =   Any amino acid<br>
NOMENCLATURE OF NUCLEIC ACIDS<br>
A	=	Adenine<br>
G	=	Guanine<br>
C	=	Cytosine<br>
T	=:	Thymine (only in DNA)<br>
U	=	Uracil (only in RNA)<br>
NOMENCLATURE AND CONVENTIONS FOR DESIGNATION OF VARIANTS<br><br>
In describing the various enzyme variants produced or contemplated according to the invention, the following nomenclatures and conventions have been adapted for ease of reference:<br>
A frame of reference is first defined by aligning the isolated or parent wild type enzyme with subtilisin BPN' (BASBPN),<br>
The alignment can be obtained by the GAP routine of the GCG package version 9-1 to number the variants using the following parameters: gap creation penalty = 8 and gap extension penalty = 8 and all other parameters kept at their default values.<br>
Another method is to use known recognised alignments between subtilases, such as the alignment indicated in WO 91/00345. In most cases the differences will not be of any importance.<br>
Such alignments between subtilisin BPN' (BASBPN) and subtilisin 3 09 (BLSAVI) and subtilisin Carlsberg (BLSCAR), respectively are indicated in Figs. 1,   la, 2,   and 2a. By this a number of deletions and insertions will be defined in relation to BASBPN. In Fig. 1 subtilisin 309 has 6 deletions in positions 36, 58, 158, 162, 163, and 164 in comparison to BASBPN, whereas in Fig. la subtilisin 309 has the same deletions in positions 36, 56, 159, 164, 165, and 166 in comparison to BASBPN. In Fig. 2 subtilisin Carlsberg has one deletion in position 58 in comparison to BASBPN, whereas in Fig. 2a subtilisin Carlsberg has the one deletion in position 56 in comparison to BASBPN. These deletions are in Figs. 1, la, 2, and 2a indicated by asterixes {*) .<br>
The various modifications performed in a wild type enzyme is indicated in general using three elements as follows:<br><br>
Original amino acid position substituted amino acid<br>
The notation G195E thus means a substitution of a glycine in position 195 with a glutamic acid.         In the case when the original amino acid residue may be any amino acid residue, a short hand notation may at times be used indicating only the position and substituted amino acid,<br>
Position substituted amino acid<br>
Such a notation is particular relevant in connection witt modification(s) in homologous subtilases {vide  infra) .<br>
Similarly when the identity of the substituting amino acid residue(s) is immaterial,<br>
Original amino acid position<br>
When both the original amino acid(s) and substituted amino acid(s) may comprise any amino acid, then only the  position is indicated, e.g.:     170.<br>
When the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), then the selected amino acids are indicated inside brackets { },<br>
Original amino acid position (substituted amino acidi , , . . , substituted amino acidn}<br>
For specific variants the specific three or one letter codes are used, including the codes Xaa and X to indicate any amino acid residue.<br>
SUBSTITUTIONS:<br><br>
The substitution of Glutamic acid for glycine in position 195 is designated as:<br>
Glyl95Glu   or  G195E or the substitution of any amino acid residue acid for glycine in position 195 is designated as:<br>
Glyl95Xaa   or  G195X or<br>
Glyl95      or  G195<br>
The substitution of serine for any amino acid residue in position 170 would thus be designated<br>
XaalVOSer   or  X170S • or<br>
IVOSer      or  170S<br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases {vide infra).   170Ser is thus meant to comprise e.g.  both a Lysl70Ser modification in BASBPN and ArglTOSer modification in BLSAVI (cf. Fig. 1) ,<br>
For a modification where the original amino acid(s) and/or substituted amino acid{s) may comprise more than one, but not all amino acid(s), the substitution of glycine, alanine, serine or threonine for arginine in position 170 would be indicated by<br>
Argl70{Gly,Ala,Ser,Thr}   or R170{G,A,S,T} to indicate the variants<br>
R170G, R170A, R170S, and R170T.<br>
DELETIONS:<br>
A deletion of glycine in position 195 will be indicated by:<br>
Glyl95*     or  G195*<br><br>
Correspondingly the deletion of more than one amino acid residue, such as the deletion of glycine and leucine in positions 195 and 196 will be designated<br>
Glyl95*+Leul96*  or  G195*+L196*<br>
INSERTIONS:<br>
The insertion of an additional amino acid residue such as e.g.   a lysine after G195 is :<br>
Glyl95GlyLys     or  G195GK; or when more than one amino acid residue is inserted, such as e.g. a Lys, Ala and Ser after G195 this is :<br>
Glyl95GlyLysAlaSer   or   G195GKAS<br>
In such cases the inserted amino acid residue (s) are numbered by the addition of lower case letters to the position number of the amino acid residue preceding the inserted amino acid residue (s). In the above example the sequences 194 to 196 would thus be:<br>
194 195 196 BLSAVI      A - G - L<br>
194 195 195a 195b 195c 196 Variant    A-G-K- A-  S-  L<br>
In cases where an amino acid residue identical to the existing amino acid residue is inserted it is clear that a degeneracy in the nomenclature arises. If for example a glycine is inserted after the glycine in the above example this would be indicated by G195GG. The same actual change could just as well be indicated as A194AG for the change from<br>
194 195 196 BLSAVI      A - G - L<br><br>
to<br>
194	195	195a 196<br>
Variant    A -	G -	G -  L<br>
194	194a	195  196<br>
Such instances will be apparent to the skilled person, and the indication G195GG and corresponding indications for this type of insertions are thus meant to comprise such equivalent degenerate indications,<br>
FILLING A GAP:<br>
Where a deletion in an enzyme exists in the reference comparison with the subtilisin BPN' sequence used for the numbering, an insertion in such a position is indicated as:<br>
*36Asp     or   *36D for the insertion of an aspartic acid in position 36<br>
MULTIPLE MODIFICATIONS<br>
Variants comprising multiple modifications are separated<br>
by pluses, e.gr.:<br>
Argl70Tyr+Glyl95Glu       or       R170Y+G195E<br>
representing modifications in positions  170  and 195  substituting<br>
tyrosine and glutamic acid for arginine and glycine,   respec-<br>
tively.<br>
Dr  e.g.   Tyrl67{Gly,Ala,Ser,Thr}+Argl70(Gly,Ala,Ser,Thr}<br>
iesignates the variants<br>
Tyrl67Gly+Argl70Gly,	Tyrl67Gly+Argl70Ala,<br>
Tyrl67Gly+Argl70Ser,	Tyrl67Gly+Argl70Thr,<br>
Tyrl67Ala+Argl70Gly,	Tyrl67Ala+Argl70Ala,<br>
Tyrl67Ala+Argl70Ser,	Tyrl67Ala+Argl70Thr,<br>
Tyrl67Ser+Argl70Gly,	Tyrl67Ser+Argl70Ala,<br>
Tyrl67Ser+Argl70Ser,	Tyrl67Ser+Argl70Thr,<br><br>
Tyrl67Thr+Argl70Gly,       Tyrl67Thr+Argl70Ala, Tyrl67Thr+Argl70Ser,   and Tyrl67Thr+Argl70Thr. This nomenclature is particular relevant relating to modifications aimed at substituting, replacing, inserting or deleting amino acid residues having specific common properties, such as residues of positive charge (K, R, H) , negative charge (D, E) , or conservative amino acid modification (s) of e.g. Tyrl67{Gly,Ala,Ser,Thr}+Argl70{Gly,Ala,Ser,Thr} , which signifies substituting a small amino acid for another small amino acid. See section 'Detailed description of the invention" for further details.<br>
Proteases<br>
Enzymes cleaving the amide linkages in protein substrates are classified as proteases, or (interchangeably) peptidases (see Walsh, 1979, Enzymatic Reaction Mechanisms,   W.H. Freeman and Company, San Francisco, Chapter 3),<br>
Numbering of amino acid positions/residues<br>
If nothing else is mentioned the amino acid numbering used herein correspond to that of the subtilase BPN" (BASBPN) sequence. For further description of the BPN" sequence see Figs. 1 and 2, or Siezen et al.,   Protein Engng.  4 (1991) 719-737.<br>
Serine proteases<br>
A serine protease is an enzyme which catalyzes the hydrolysis of peptide bonds, and in which there is an essential serine residue at the active site (White, Handler and Smith, 1973 "Principles of Biochemistry, " Fifth Edition, McGraw-Hill Book Company, NY, pp. 271-272).<br>
The bacterial serine proteases have molecular weights in the 20,000 to 45,000 Dalton range. They are inhibited by<br><br>
diisopropylfluorophosphate. They hydrolyze simple terminal esters and are similar in activity to eukaryotic chymotrypsin, also a serine protease. A more narrow term, alkaline protease, covering a sub-group, reflects the high pH optimum of some of the serine proteases, from pH 9.0 to 11.0 (for review, see Priest (1977) BacterioiogricaJ Rev.   41 711-753) .<br>
Subtilases<br>
A sub-group of the serine proteases tentatively designated subtilases has been proposed by Siezen et al. ,   Protein Engng.   4 (1991) 719-737 and Siezen et al. Protein Science  6   (1997) 501-523. They are defined by homology analysis of more than 170 amino acid sequences of serine proteases previously referred to as subtilisin-like proteases. A subtilisin was previously often defined as a serine protease produced by Gram-positive bacteria or fungi, and according to Siezen et al.   now is a subgroup of the subtilases. A wide variety of subtilases have been identified, and the amino acid sequence of a number of subtilases has been detexTnined. For a more detailed description of such subtilases and their amino acid sequences reference is made to Siezen et al.(1997).<br>
One subgroup of the subtilases, I-SI or " true# subtilisins, comprises the "classical" subtilisins, such as subtilisin 168 (BSS168) , subtilisin BPN* , subtilisin Carlsberg (ALCALASE% NOVO NORDISK A/S) ,   and subtilisin DY (BSSDY) .<br>
A further subgroup of the subtilases, I-S2 or high alkaline subtilisins, is recognised by Siezen et al.   (supra) . Sub-group I-S2 proteases are described as highly alkaline subtilisins and comprises enzymes such as subtilisin PB92 (BAALKP) (MAXACAL*, Gist-Brocades NV) , subtilisin 309 (SAVINASE*, NOVO NORDISK A/S),   subtilisin 147 (BLS147) (ESPERASE*, NOVO NORDISK A/S), and alkaline elastase YaB (BSEYAB).<br><br>
List  of  acronyms  for subtilases:<br>
I-Sl<br>
Subtilisin 168,   BSS168   (BSSAS   (Subtilisin amylosacchariticus),<br>
BSAPRJ   (Subtilisin J) ,   BSAPRN   (Subtilisin NAT),   BMSAMP   (Mes-<br>
entericopeptidase) ,<br>
Subtilisin BPN' ,   BASBPN,<br>
Subtilisin DY,   BSSDY,<br>
Subtilisin  Carlsberg,   BLSCAR   (BLKERA   (Keratinase),   BLSCAl,<br>
BLSCA2,   BLSCA3) ,<br>
BSSPRC,   Serine  protease   C<br>
BSSPRD,   Serine protease D<br>
'I-S2<br>
Subtilisin Sendai, BSAPRS Subtilisin ALP 1, BSAPRQ,<br>
Subtilisin 147, Esperase® BLS147 (BSAPRM (SubtilisinAprM), BAHlOl) ,<br>
Subtilisin 309, Savinase®, BLS309/BLSAVI (BSKSMK (M-protease) , BAALKP(Subtilisin PB92, Bacillus alkalophilic alkaline protease) , BLSUBL (Subtilisin BL)), Alkaline elastase YaB, BYSYAB,<br>
^^SAVINASE®^^<br>
SAVINASE® is marketed by NOVO NORDISK A/S.   It  is subtilisin 309  from B.   Lentus and differs  from BAALKP only in one position   (N87S,   see  Fig.   1 herein) .   SAVINASE® has  the  amino acid sequence designated b)   in Fig.   1.<br>
Parent  subtilase<br>
The term  "parent  subtilase"   describes  a  subtilase defined according to Siezen et al.    (1991  and 1997).   For  further<br><br>
details see description, of "SUBTILASES" immediately above. A parent subtilase may also be a subtilase isolated from a natural source, wherein subsequent modification have been made while retaining the characteristic of a subtilase. Alternatively the  term "parent subtilase" may be termed "wild type subtilase".<br>
Modification(s) of a subtilase variant<br>
The term "modification (s) " used herein is defined to include chemical modification of a subtilase as well as genetic  manipulation of the DNA encoding a subtilase. The modification (s) can be replacement (s) of the amino acid side chain(s) , substitution(s) , deletion(s) and/or insertions in or at the amino acid(s) of interest.<br>
 Subtilase variant<br>
In the context of this invention, the term subtilase variant or mutated subtilase means a subtilase that has been produced by an organism which is expressing a mutant gene derived from a parent microorganism which possessed an original<br>
 or parent gene and which produced a corresponding parent enzyme, the parent gene having been mutated in order to produce the mutant gene from which said mutated subtilase protease is produced when expressed in a suitable host.<br>
Homologous subtilase sequences<br>
Specific active site loop regions, and amino acid<br>
insertions in said loops of SAVINASE® subtilase are identified for modification herein to obtain a subtilase variant of the invention.<br>
However, the invention is not limited to modifications of this particular subtilase, but extend to other parent (wild-type) subtilases, which have a homologous primary structure to<br><br>
that of SAVINASE®. The homology between two amino acid sequences is in this context described by the parameter "identity".<br>
In order to determine the degree of identity between two subtilases the GAP routine of the GCG package version 9.1 can be applied (infra)   using the same settings. The output from the routine is besides the amino acid alignment the calculation of the "Percent Identity" between the two sequences.<br>
Based on this description it is routine for a person skilled in the art to identify suitable homologous subtilases and corresponding homologous active site loop regions, which can be modified according to the invention.<br>
Wash performance<br>
The ability of an enzyme to catalyze the degradation of various naturally occurring substrates present on the objects to be cleaned during e.g.   wash or hard surface cleaning is often referred to as its washing ability, wash-ability, detergency, or wash performance. Throughout this application the term wash performance will be used to encompass this property.<br>
Isolated DNA sequence<br>
The term "isolated", when applied to a DNA sequence molecule, denotes that the DNA sequence has been removed from its natural genetic milieu and is thus free of other extraneous or unwanted coding sequences, and is in a form suitable for use within genetically engineered protein production systems. Such isolated molecules are those that are separated from their natural environment and include cDNA and genomic clones. Isolated DNA molecules of the present invention are free of  other genes with which they are ordinarily associated, but may include naturally occurring 5' and 3 * untranslated regions such as promoters and terminators. The identification of associated<br><br>
regions will be evident, to one of ordinary skill in the art (see for example, Dynan and Tijan, Nature 316 ;774-78, 1985). The term "an isolated DNA sequence" may alternatively be termed "a cloned DNA sequence".<br>
Isolated protein<br>
When applied to a protein, the term "isolated" indicates that the protein has been removed from its native environment.<br>
In a preferred form, the isolated protein is substantially free of other proteins, particularly other homologous proteins (i,e. "homologous impurities" (see below)).<br>
An isolated protein is greater than 10 % pure, preferably greater than 20 % pure, more preferably greater than 30 % pure, as determined by SDS-PAGE. Further it is preferred to provide the protein in a highly purified form, i.e., greater than 40% pure, greater than 60% pure, greater than 80% pure, more preferably greater than 95% pure, and even more preferably greater than 99% pure, as determined by SDS-PAGE.<br>
The term "isolated protein" may alternatively be termed "purified protein" ,<br>
Homologous impurities<br>
The term "homologous impurities" means any impurity (e.g. another polypeptide than the polypeptide of the invention) which originate from the homologous cell where the polypeptide of the invention is originally obtained from.<br>
Obtained from<br>
The term "obtained from" as used herein in connection with a specific microbial source, means that the polynucleotide and/or polypeptide produced by the specific source, or by a cell in which a gene from the source has been inserted.<br><br>
Substrate<br>
The term "Substrate" used in connection with a substrate for a protease is should be interpreted in its broadest form as comprising a compound containing at least one peptide bond susceptible to hydrolysis by a subtilisin protease.<br>
Product<br>
The term "product" used in connection with a product derived from a protease enzymatic reaction should in the context of this invention be interpreted to include the products of a hydrolysis reaction involving a subtilase protease. A product may be the substrate in a subsequent hydrolysis reaction.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Fig. 1 shows an alignment between subtilisin BPN' (a) and<br>
Savinase" (b) using the GAP routine mentioned above.<br>
Fig. la shows the alignment between subtilisin BPN' and<br>
Savinase® as taken from WO 91/00345.<br>
Fig. 2 shows an alignment between subtilisin BPN' and subtilisin Carlsberg using the GAP routine mentioned above.<br>
Fig. 2a shows the alignment between subtilisin BPN' and" subtilisin Carlsberg as taken from WO 91/00345.<br>
Fig. 3 shows the three dimensional structure of Savinase (Protein data bank (PDB) entry ISVN). In the Figure the active site loop (b) is indicated.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The subtilases of the invention in a first aspect relates to an isolated (i.e. greater than 10 % pure) subtilase enzyme of the I-Sl and I-S2 sub-groups having at least one additional amino acid residue in position 102 of the active site loop (b)<br><br>
region from position 95 .to 103, whereby said additional amino acid residue (s) correspond to the insertion of at least one amino acid residue between positions 102 and 103.<br>
In other words the subtilases of the invention are characterized by comprising an active site loop (b) region of more than 9 amino acid residue and wherein* the additional amino acid residue is or can be considered as being inserted between positions 102 and 103 as compared to the parent or a known wild type subtilase.<br>
A subtilase of the first aspect of the invention may be a parent or wildtype subtilase identified and isolated from nature.<br>
Such a parent wildtype subtilase may be specifically screened for by standard techniques known in the art.<br>
One preferred way of doing this may be by specifically PCR amplify DNA regions known to encode active site loops in subtilases from numerous different microorganism, preferably different Bacillus strains,<br>
Subtilases are a group of conserved enzymes, in the sense that their DNA and amino acid sequences are homologous. Accordingly it is possible to construct relatively specific primers flanking active site loops.<br>
One way of doing this is by investigating an alignment of different subtilases (see e.g. Siezen et al. Protein Science 6 (1997) 501-523) . It is from this routine work for a person skilled in the art to construct PCR primers flanking the active site loop corresponding to the active site loop (b) between amino acid residue 95 to 103 in any of the group I-SI or I-S2 groups, such as from BLSAVI, Using such PCR primers to amplify DNA from a number of different microorganism, preferably different Bacillus strains, followed by DNA sequencing of said amplified PCR fragments, it will be possible to identify strains<br><br>
which produce subtilases of these groups comprising a longer, as compared to e.g. BLSAVI, active site region corresponding to the active site loop region from positions 95 to 103, and where an insertion can be considered to exist between positions 102 and 103. Having identified the strain and a partial DNA sequence of such a subtilase of interest, it is routine work for a person skilled in the art to complete cloning, expression and purification of such a subtilase of the invention.<br>
However, it is envisaged that a subtilase enzyme of the invention predominantly is a variant of a parent subtilase.<br>
Accordingly, in one embodiment the invention relates to an isolated subtilase enzyme according to the first aspect of the invention, wherein said subtilase enzyme is a constructed variant having a longer active site loop (b) than its parent enzyme by having at least one amino acid insertion between amino acid residues 102 and 103.<br>
The subtilases of the invention exhibit excellent wash performance in a detergent, and if the enzyme is a constructed variant an improved wash performance in a detergent in comparison to its closest related subtilase, such as subtilisin 309.<br>
Different subtilase products will exhibit a different wash performance in different types of detergent compositions. A subtilase of the invention has improved wash performance, as compared to its closest relative in a majority of such different types of detergent compositions.<br>
Preferably a subtilase enzyme of the invention has improved wash performance, as compared to its closest relative in the detergent composition shown in Example 3 herein (vide infra) .<br>
In order to determine if a given subtilase amino acid sequence (irrelevant whether said subtilase sequence is a parent<br><br>
wildtype subtilase sequence or a subtilase variant sequence produced by any other method than by site directed mutagenesis) is within the scope of the invention, the following procedure may be used:<br>
i)   align said subtilase sequence to the amino acid sequence of subtilisin BPN' (see section "D'efinitions" herein (vide supra); ii)  based on the alignment performed in step i) identify the active site loop (b), in said subtilase sequence corresponding to the active site loop (b) region of subtilisin BPN' comprising the region (both of the end amino acids included) between amino acid residue from 95 to 103; iii) determine if the active site loop (b) in said subtilase sequence, identified in step ii) is longer than the corresponding active site loop in BLSAVI and if said prolongation corresponds to the insertion of at least one amino acid residue between positions 102 and 103, If this is the case the subtilase investigated is a subtilase within the scope of the present invention.<br>
The alignment performed in step i) above is performed as described above by using the GAP routine.<br>
Based on this description it is routine for a person skilled in the art to identify the active site loop (b) in a subtilase and determine if the subtilase in question is within the scope of the invention. If a variant is constructed by site directed mutagenesis, it is of course known beforehand if the subtilase variant is within the scope of the invention.<br>
A subtilase variant of the invention may be constructed by standard techniques known in the art such as by site-directed/random mutagenesis or by DNA shuffling of different s\ibtilase sequences. See section '*PRODUCING A SUBTILASE<br><br>
VARIANT" and Material and methods herein (vide infra) for<br>
further details.<br>
In further embodiments the invention relates to<br>
l.an isolated subtilase enzyme according to the invention,<br>
wherein said at least one inserted amino acid residue is chosen from the group comprising: T;G,A, and S;<br>
2. an isolated subtilase enzyme according to the invention,<br>
wherein said at least one inserted amino acid residue is chosen from the group of charged amino acid residues comprising: D,E,H,K, and R, more preferably D,E,K and R;<br>
3.an isolated subtilase enzyme according to the invention,<br>
wherein said at least one inserted amino acid residue is chosen from the group of hydrophilic amino acid residues comprising: C,N,Q,S and T, more preferably N,Q,S and T;<br>
4-an isolated subtilase enzyme according to the invention,<br>
wherein said at least one inserted amino acid residue is chosen from the group of small hydrophobic amino acid residues comprising: A,G and V; or<br>
5.an isolated subtilase enzyme according to the invention,<br>
wherein said at least one inserted amino acid residue is chosen from the group of large hydrophilic amino acid residues comprising: F,I,L,M,P,W and Y, more preferably F,I,L,M, and Y, In a further embodiment, the invention relates to an<br>
isolated subtilase enzyme according to the invention, wherein<br>
said insertion between positions 102 and 103 comprises at least<br>
two amino acids, as compared to the corresponding active site<br>
loop in BLSAVI.<br>
In further embodiments the invention relates to an<br>
isolated subtilase enzyme comprising at least one insertion,<br>
chosen from the group comprising (in BASBPN numbering) :<br>
X102X{T,G,A,S}<br><br>
X102X{D,E,K,R}<br>
X102X{H,V,C,N,Q}<br>
X102X{F,I,L,M,P,W,Y}<br>
or more specific for subtilisin 3 09 and closely related<br>
subtilases, such as BAALKP, BLSUBL, and BSKSMK<br>
G102GA<br>
G102GT<br>
G102GG<br>
G102GS<br>
G102GD G102GE G102GK G102GR<br>
G102GH G102GV G102GC G102GN G102GQ<br>
G102GF G102GI G102GL G102GM G102GP G102GW G102GY<br>
Furthermore the invention relates to subtilases comprising multiple insertions in position 102, or any of the following combinations<br><br>
G102GT+Y167A<br>
It is well known in the art .that a so-called conservative substitution of one amino acid residue to a similar amino acid residue is expected to produce only a minor change in the characteristic of the enzyme.<br>
Table III below list groups of conservative amino acid substitutions.<br>
Table III<br>
Conservative amino acid substitutions<br>
Common Property	Amino Acid<br>
Basic (positive charge)        K = lysine<br>
H = histidine Acidic (negative charge)        E = glutamic acid<br>
D = aspartic acid<br>
Polar	Q = glutamine<br>
N = asparagine<br>
Hydrophobic	L = leucine<br>
I = isoleucine<br>
V	= valine<br>
M = methionine<br>
Aromatic	F = phenylalanine<br>
W = tryptophan<br>
Y	= tyrosine<br>
Small                        G = glycine<br>
A = alanine S = serine T = threonine<br>
According to this principle subtilase variants comprising conservative substitutions, such as G97A+A98AS+S99G,<br><br>
G97S+A98AT+S99A are expected to exhibit characteristics that are not drastically different from each other.<br>
Based on the disclosed and/or exemplified subtilase variants herein, it is routine work for a person skilled in the art to identify suitable conservative modification(s) to these variants in order to obtain other subtilase variants exhibiting similarly improved wash-performance.<br>
According to the invention it the subtilases of the invention belong to the subgroups I-Sl and I-S2, especially subgroup 1-S2,   both for isolating novel enzymes of the invention from nature or from the artificial creation of diversity, and for designing and producing variants from a parent subtilase.<br>
In relation to variants from subgroup I-SI, it is preferred to choose a parent subtilase from the group comprising BSS168 (BSSAS, BSAPRJ, BSAPRN, BMSAMP) , BASBPN, BSSDY, BLSCAR (BLKERA, BLSCAl, BLSCA2, BLSCA3) , BSSPRC, and BSSPRD, or functional variants thereof having retained the characteristic of sub-group I-Sl.<br>
In relation to variants from subgroup I-52 it is preferred to choose a parent subtilase from the group comprising BSAPRQ, BLS147 (BSAPRM, BAHlOl) , BLSAVI (BSKSMK, BAALKP, BLSUBL) , BYSYAB, and BSAPRS, or functional variants thereof having retained the characteristic of sub-group I-S2.<br>
In particular said parent subtilase is BLSAVI (SAVINASE® NOVO NORDISK A/S) , and a preferred subtilase variant of the invention is accordingly a variant of SAVINASE®.<br>
The present invention also comprises any of the above mentioned subtilases of the invention in combination with any other modification to the amino acid sequence thereof. Especially combinations with other modifications known in the art to provide improved properties to the enzyme are envisaged. The art describe a number of subtilase variants with different improved<br><br>
properties and a number, of those are mentioned in the "Background of the invention" section herein (vide supra).   Those references are disclosed here as references to identify a subtilase variant, which advantageously can be combined with a subtilase variant of the invention.<br>
Such combinations comprise the positions: 222 (improve oxidation stability),   218 (improves thermal stability), substitutions in the Ca-binding sites stabilizing the enzyme, e.g. position 76, and many other apparent from the prior art.<br>
In further embodiments a subtilase variant of the invention may advantageously be combined with one or more modification (s) in any of the positions:<br>
27, 36, 57, 76, 87, 97, 101, 104, 120, 123, 167, 170, 206, 218, 222, 224, 235 and 274.<br>
Specifically the following BLSAVI, BLSUBL, BSKSMK, and BAALKP variants are considered appropriate for combination: K27R, *36D, S57P, N76D, S87N, G97N, SIOIG, S103A, V104A, V104I, V104N, V104Y, H120D, N123S, Y167, R170, Q206E, N218S, M222S, M222A, T224S, K235L and T274A.<br>
Furthermore variants comprising any of the variants S101G+V104N, S87N+S101G+V104N, K27R+V104Y+N123S+T274A, N76D+S103A+V104I or N76D+V104A or other combinations of these mutations (V104N, SIOIG, K27R, V104Y, N123S, T274A, N76D, V104A) in combination with any one or more of the modification(s) mentioned above exhibit improved properties-<br>
Even further subtilase variants of the main aspect(s) of the invention are preferably combined with one or more modification (s) in any of the positions 129, 131, 133 and 194, preferably as 129K, 131H, 133P, 133D and 194P modifications, and most preferably as P129K, P131H, A133P, A133D and A194P modifications. Any of those modification(s) are expected to provide a<br><br>
higher expression level of a subtilase variant of the invention in the production thereof.<br>
Accordingly, an even further embodiment of the invention relates to a variant according to the invention, wherein said modification is chosen from the group comprising:<br>
PRODUCING A SUBTILASE VARIANT<br>
Many methods for cloning a subtilase of the invention and for introducing insertions into genes (e.g. subtilase genes) are well known in the art, cf. the references cited in the ^^BACKGROUND OF THE INVENTION" section.<br>
In general standard procedures for cloning of genes and introducing insertions (random and/or site directed) into said genes may be used in order to obtain a subtilase variant of the invention. For further description of suitable techniques reference is made to Examples herein {vide infra)   and (Sambrook et al. (1989) Molecular cloning: A laboratory manual. Cold Spring Harbor lab.. Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R,, and Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990); and WO 96/34946,<br>
Further a subtilase variant of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (WO 95/22625; Stemmer WPC, Nature 370:389-91 (1994)). DNA<br>
shuffling of e.g. the gene encoding Savinase® with one or more partial subtilase sequences identified in nature to comprise an active site (b) loop regions longer than the active site (b) loop of Savinase®, will after subsequent screening for improved wash performance variants, provide subtilase variants according to the invention.<br><br>
EXPRESSION VECTORS<br>
A recombinant expression vector comprising a DNA construct encoding the enzyme of the invention may be any vector which may conveniently be subjected to recombinant DNA procedures •<br>
The choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector, which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid. Alternatively, the vector may be one that on introduction into a host cell is integrated into the host cell genome in part or in its entirety and replicated together with the chromosome (s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA sequence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from plasmid or viral DNA, or may contain elements of both. The term, ^'operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The promoter may be any DNA sequence which shows transcriptional activity in.the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for use in bacterial host cells include the promoter of the Bacillus stearothermophilus maltogenic amylase gene, the Bacillus licheniformis  alpha-amylase gene, the Bacillus amyloliguefaciens  alpha-amylase gene.<br><br>
the Bacillus subtilis  alkaline protease gen, or the Bacillus pumilus  xylosidase gene, or the phage Lambda PR or PL promoters or the E. coli lac, trp or tac promoters.<br>
The DNA sequence encoding the enzyme of the invention may also, if necessary, be operably connected to a suitable terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, or a gene encoding resistance to e.g. antibiotics like kanamycin, chloramphenicol, erythromycin, tetracycline, spectinomycine, or the like, or resistance to heavy metals or herbicides.<br>
To direct an enzyme of the present invention into the secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is joined to the DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 5' to the DNA sequence encoding the enzyme. The secretory signal sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br>
The procedures used to ligate the DNA sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, or to assemble these sequences by suitable PCR amplification schemes,  and to insert them into suitable vectors containing the information necessary for replication or integration, are well<br><br>
known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit.).<br>
HOST CELL<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question. If homologous to the host cell, i.e. produced by the host cell in nature, it will typically be operably connected to another promoter sequence or, if applicable, another secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a DNA sequence encoding an enzyme native to the host organism in question. The term "heterologous" is intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may be from another organism, or it may be a synthetic sequence.<br>
The host cell into which the DNA constmct or the recombinant vector of the invention is introduced may be any cell which is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells including plants.<br>
Examples of bacterial host cells which, on cultivation, are capable of producing the enzyme of the invention are gram-positive bacteria such as strains of Bacillus,   such as strains of B. suJbtalis, B. lichenifoxmis,   B. lentus,   B,   brevis,   B. stearothermophilus,   B.   alkalophilus,   B. amyloliquefaciens,  B. coagulans,   B.   circulans,   B.   lautus,   B,  megatherium  or B. thuringiensis,   or strains of Streptomyces, such as 5. lividans or S.  murinus,   or gram-negative bacteria such as Echerichia coli.<br>
The transformation of the bacteria may be effected by protoplast transformation, electroporation, conjugation, or by<br><br>
using competent cells in a manner known per se (of. Sambrook et al., supra).<br>
When expressing the enzyme in bacteria such as E.   coli, the enzyme may be retained in the cytoplasm, typically as insol-<br>
 uble granules (known as inclusion bodies), or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be<br>
 recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme.<br>
When expressing the enzyme in gram-positive bacteria such as Bacillus or Streptomyces strains, the enzyme may be retained<br>
 in the cytoplasm, or may be directed to the extracellular medium by a bacterial secretion sequence. In the latter case, the enzyme may be recovered from the medium as described below.<br>
METHOD OF PRODUCING SUBTILASE<br>
The present invention provides a method of producing an isolated enzyme according to the invention, wherein a suitable host cell, which has been transformed with a DNA sequence encoding the enzyme, is cultured under conditions permitting the production of the enzyme, and the resulting enzyme is recovered from the culture.<br>
When an expression vector comprising a DNA sequence encoding the enzyme is transformed into a heterologous host cell it is possible to enable heterologous recombinant production of the enzyme of the invention.<br>
Thereby it is possible to make a highly purified subtilase composition, characterized in being free from homologous impurities.<br><br>
In this context homologous impurities means any impurities (e.g. other polypeptides than the enzyme of the invention) which originate from the homologous cell where the enzyme of the invention is originally obtained from.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed subtilase may conveniently be secreted into the culture medium and may be recovered therefrom by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulfate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
USE OF A SUBTIIASE VARIANT OF THE INVENTION<br>
A subtilase protease variant of the invention may be used for a number of industrial applications, in particular within the detergent industry.<br>
Further the invention relates to an enzyme composition, which comprise a subtilase variant of the invention.<br>
An summary of preferred industrial applications and corresponding preferred enzyme compositions are described below.<br>
This summary is not in any way intended to be a complete list of suitable applications of a subtilase variant of the invention. A subtilase variants of the invention may be used in other industrial applications known in the art to include use of a protease, in particular a subtilase.<br>
DETERGENT COMPOSITIONS COMPRISING THE MUTANT ENZYMES<br>
The present invention comprises the use of the mutant enzymes of the invention in cleaning and detergent compositions<br><br>
and such compositions comprising the mutant subtilisin enzymes. Such cleaning and detergent compositions are well described in the art and reference is made to WO 96/34946; WO 97/07202; WO 95/30011 for further description of suitable cleaning and  detergent compositions.<br>
Furthermore the example(s) below demonstrate the improvements in wash performance for a number of subtilase variants of the invention.<br>
Detergent Compositions<br>
The enzyme of the invention may be added to and thus become a component of a detergent composition-<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations, or be formulated for hand or machine dishwashing operations.<br>
In a specific aspect, the invention provides a detergent additive comprising the enzyme of the invention. The detergent additive as well as the detergent composition may comprise one or more other enzymes such as a protease, a lipase, a cutinase, an amylase, a carbohydrase, a cellulase, a pectinase, a man-nanase, an arabinase, a galactanase, a xylanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme(s) should be compatible with the selected detergent, (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.), and the enzyme(s) should be present in effective amounts.<br><br>
Proteases: Suitable proteases include those of animal, vegetable or microbial origin. Microbial origin is preferred. Chemically modified or protein engineered . mutants are included. The protease may be a serine protease or a met alio protease, preferably an alkaline microbial protease or a trypsin-like protease. Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carlsberg, subtilisin 3 09, subtilisin 147 and subtilisin 168 (described in WO 89/06279) . Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine origin) and the Fusarium protease described in WO 8 9/06270 and WO 94/25583.<br>
Examples of useful proteases are the variants described in WO  92/19729,  WO  98/20115,  WO  98/20116,  and  WO  98/34946, especially the variants with substitutions in one or more of the following positions: 27, 36, 57, 76, 87, 97, 101, 104, 120, 123, 167, 170, 194, 206, 218, 222, 224, 235 and 274.<br>
Preferred commercially available protease enzymes include AlcalaseTM, SavinaseTM, Primase™, Duralase*™, EsperaseTM, and KannaseTM (Novo Nordisk A/S) , MaxataseTM, MaxacalTM, Maxapem™,Properase™, Purafect™, Purafect OxP™, FN2™, and FN3™ (Genencor International Inc.).<br>
Lipases; Suitable lipases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful lipases include lipases from Humicola (synonym Thermomyces) , e.g. from H. lanuginosa (T. lanuginosus) as described in EP 258 068 and EP 3 05 216 or from H. insolens as described in WO 96/13580, a Pseudomonas lipase, e.g. from P. alcaligenes or P. pseudoalcaligenes (EP 218 272), P. cepacia (EP 331 376), P. stutzeri (GB 1,372,034), P. fluorescens, Pseudomonas sp. strain SD 705 (WO 95/06720 and WO 96/27002), P. wisconsinensis    (WO 96/12012), a Bacillus   lipase.<br><br>
e.g. from B. subtilis . (Dartois et al. (1993), Biochemica et Biophysica Acta, 1131, 253-360), B. stearothermophilus (JP 64/744992) or B.   pumilus   (WO 91/16422) .<br>
Other examples are lipase variants such as those described in WO 92/05249, WO 94/01541, EP 4 07 225, EP 260 105, WO 95/35381, WO 96/00292, WO 95/30744, WO 94/25578, WO 95/14783, WO 95/22615, WO 97/04079 and WO 97/07202.<br>
Preferred commercially available lipase enzymes include Lipolase™ and Lipolase Ultra™ (Novo Nordisk A/S),<br>
Amylases: Suitable amylases (a and/or P) include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Amylases include, for example, a-amylases obtained from Bacillus, e.g. a special strain of B. licheniformis,   described in more detail in GB 1,296,839.<br>
Examples of useful amylases are the variants described in WO 94/02597, WO 94/18314, WO 96/23873, and WO 97/43424, especially the variants with substitutions in one or more of the following positions: 15, 23, 105, 106, 124, 128, 133, 154, 156, 181, 188, 190, 197, 202, 208, 209, 243, 264, 304, 305, 391, 408, and 444.<br>
Commercially available amylases are Duramyl™, Termamyl™, Fungamyl™ and BAN™ (Novo Nordisk A/S) , Rapidase"™ and Purastar™ (from Genencor International Inc.).<br>
Cellulases; Suitable cellulases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Suitable cellulases include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonium, e.g. the fungal cellulases produced from Humicola insolens, Myceliophthora thejrmophila and Fusarium oxysporum disclosed in US 4,435,307, US 5,648,263, US 5,691,178, US 5,776,757 and WO 89/09259.<br><br>
Especially suitable cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellulases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/29397; WO 98/08940. Other examples are cellulase variants such as those described in WO 94/07998, EP 0 531 315, US 5,457,046, US 5,686,593, US 5,763,'254, WO 95/24471, WO 98/12307 and PCT/DK98/00299.<br>
Commercially available cellulases include Celluzyme™, and Carezyme"^^ (Novo Nordisk A/S) , Clazinase'"^,. and Puradax HA"^ (Genencor International Inc.) , and KAC-500 (B)™ (Kao Corporation).<br>
Peroxidases/Oxidases: Suitable peroxidases/oxidases include those of plant, bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g. from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257.<br>
Commercially available peroxidases include Guardzyme*™ (Novo Nordisk A/S).<br>
The detergent enzyme (s) may be included in a detergent composition by adding separate additives containing one or more enzymes, or by adding a combined additive comprising all of these enzymes. A detergent additive of the invention, i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Preferred detergent additive formulations are granulates, in particular non-dusting granulates, liquids, in particular stabilized liquids, or slurries.<br>
Non-dusting granulates may be produced, e.g., as disclosed in US 4,106,991 and 4,661,452 and may optionally be coated by methods known in the art. Examples of waxy coating materials are poly (ethylene oxide) products (polyethyleneglycol,<br><br>
PEG) with mean molar weights of 1000 to 20000; ethoxylated non-ylphenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or I sugar alcohol, lactic acid or boric acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any convenient form, e.g., a bar, a tablet, a powder, a granule, a paste or a liquid. A liquid detergent may be aqueous, typically containing up to 70 % water and 0-3 0 % organic solvent, or nonaqueous .<br>
The detergent composition comprises one or more surfactants, which may be non- ionic including semi -polar and/or ani -onic and/or cationic and/or zwitterionic. The surfactants are typically present at a level of from 0.1% to 60% by weight.<br>
When included therein the detergent will usually contain from about 1% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsulfonate, alkyl sulfate (fatty alcohol sulfate), alcohol ethoxysulfate, secondary al-kanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuccinic acid or soap.<br>
When included therein the detergent will usually contain from about 0.2% to about 40% of a non-ionic surfactant such as alcohol ethoxylate, nonylphenol ethoxylate, alkylpolyglycoside, alkyldimethylamineoxide,  ethoxylated  fatty  acid  monoethanol-amide, fatty acid monoethanolamide, polyhydroxy alkyl fatty acid<br><br>
amide,   or   N-acyl   N-alkyl   derivatives   of   glucosamine ("glucamides" ) •<br>
The detergent may contain 0-65 % of a detergent builder or complexing agent such as zeolite, diphosphate, triphosphate,  phosphonate, carbonate, citrate, nitrilotriacetic acid, ethyl-enediaminetetraacetic acid, diethylenetriaminepentaacetic acid, alkyl- or alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst).<br>
The detergent may comprise one or more polymers. Examples  are carboxymethylcellulose, poly (vinylpyrrolidone) , poly (ethylene glycol) , poly (vinyl alcohol) , poly (vinylpyridine-N-oxide) , poly (vinylimidazole) , polycarboxylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.  The detergent may contain a bleaching system which may comprise a H2O2 source such as perborate or percarbonate which may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxybenzenesulfonate. Alternatively, the bleaching system may comprise peroxyacids of e.g. the amide, imide, or sulfone type.<br>
The enzyme(s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g., a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g., an aromatic borate ester, or a phenyl boronic acid derivative such as 4-formylphenyl boronic acid, and the composition may be foirmulated as described in e.g. WO 92/19709 and WO 92/19708,<br>
The detergent may also contain other conventional detergent ingredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents,  anti-soil redeposition agents,  dyes,<br><br>
bactericides, optical brighteners, hydrotropes, tarnish inhibitors, or perfumes.<br>
It is at present contemplated that in the detergent compositions any enzyme, in particular the enzyme of the invention, may be added in an amount corresponding to 0.01-100 mg of enzyme protein per liter of wash liquor, preferably 0.05-5 mg of enzyme protein per liter of wash liquor, in particular 0.1-1 mg of enzyme protein per liter of wash liquor.<br>
The enzyme of the invention may additionally be incorporated in the detergent formulations disclosed in WO 97/07202 which is hereby incorporated as reference.<br>
LEATHER INDUSTRY APPLICATIONS<br>
A subtilase of the invention may be used in the leather industry, in particular for use in depilation of skins.<br>
In said application a subtilase variant of the invention is preferably used in an enzyme composition which further comprises another protease.<br>
For a more detailed description of suitable other proteases see section relating to suitable enzymes for use in a detergent composition {vide supra).<br>
WOOL INDUSTRY APPLICATIONS<br>
A subtilase of the invention may be used in the wool industry, in particular for use in cleaning of clothes comprising wool.<br>
In said application a subtilase variant of the invention is preferably used in an enzyme composition which further comprises another protease.<br>
For a more detailed description of suitable other proteases see section relating to suitable enzymes for use in a detergent composition {vide supra).<br><br>
The invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention as claimed.<br>
MATERIALS AND METHODS<br>
STRAINS:<br>
B. subtilis  DN1885 (Diderichsen et al., 1990).<br>
JS. lentus  309 and 147 are specific strains of Bacillus lentus,   deposited with the NCIB and accorded the accession numbers NCIB  10309 and 10147, and described in US Patent No. 3,723,250 incorporated by reference herein.<br>
E.   coll  MC 1000 (M.J. Casadaban and S.N. Cohen (1980); L<br>
Mol.   Biol.   138 179-207), was made r",m+ by conventional methods and is also described in US Patent Application Serial No. 039,298.<br>
PLASMIDS:<br>
pJS3: E,   coli   " B.   subtilis  shuttle vector containing a synthetic gene encoding for subtilase 309. (Described by Jacob Schiodt et al. in Protein and Peptide letters 3:39-44 (1996)),<br>
pSX222: B.   subtilis  expression vector (Described in WO 96/34946) .<br>
GENERAL MOLECULAR BIOLOGY METHODS:<br>
Unless otherwise mentioned the DNA manipulations and transformations were performed using standard methods of molecular biology (Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab.. Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R., and<br><br>
Cutting, S. M. (eds.) "Molecular Biological Methods for Bacillus". John Wiley and Sons, 1990).<br>
Enzymes for DNA manipulations were used according to the specifications of the suppliers.<br>
ENZYMES FOR DNA MANIPULATIONS<br>
Unless otherwise mentioned all enzymes for DNA manipulations, such as e.g. restiction endonucleases, ligases etc., are obtained from New England Biolabs, Inc.<br>
PROTEOLYTIC ACTIVITY<br>
In the context of this invention proteolytic activity is expressed in Kilo NOVO Protease Units (KNPU) . The activity is<br>
determined relatively to an enzyme standard (SAVINASE*), and the determination is based on the digestion of a dimethyl casein (DMC) solution by the proteolytic enzyme at standard conditions, i.e. 50°C, pH 8.3, 9 min. reaction time, 3 min. measuring time. A folder AF 220/1 is available upon request to Novo Nordisk A/S, Denmark, which folder is hereby included by reference.<br>
A GU is a Glycine Unit, defined as the proteolytic enzyme activity which, under standard conditions, during a 15 minutes' incubation at 40°C, with N-acetyl casein as substrate, produces an amount of NHj-group equivalent to 1 mmole of glycine.<br>
Enzyme activity can also be measured using the PNA assay, according to reaction with the soluble substrate succinyl-alanine-alanine-proline-phenyl-alanine-para-nitro-phenol, which is described in the Journal of American Oil Chemists Society, Rothgeb, T.M., Goodlander, B.D., Garrison, P.H., and Smith, L.A., (1988).<br>
FERMENTATION;<br><br>
Fermentations for the production of subti/ase enzymes were performed at 30°C on a rotary shaking table (300  r.p.m.) in 500 ml baffled Erlenmeyer flasks containing 100 BPX medium for 5 days.<br>
Consequently in order to make an e.g.   2  1/ter broth 20 Erlenmeyer flasks were fermented simultaneously.<br>
MEDIA:<br>
BPX Medium Composition	(per liter)<br>
Potato starch	100 g<br>
Ground barley	50 g<br>
Soybean flour	20 g<br>
Na2HP04 x 12 H2O	9 g<br>
Pluronic	0 .1 g<br>
Sodium caseinate	10 g<br>
The starch in the medium is liquefied with a-amylase and the medium is sterilized by heating at 120°C for 45 minutes. After sterilization the pH of the medium is adjusted to 9 by addition of NaHC03 to 0.1 M,<br>
EXAMPLE 1<br>
CONSTRUCTION AND EXPRESSION OP ENZYME VARIANTS: SITE-DIRECTED MUTAGENESIS:<br>
Subtilase 309 site-directed variants of t/^^ invention comprising specific insertions in the active site loop (b) between positions 102 and 103 were made by traditional cloning of DNA fragments (Sambrook et al., Molecular Clcning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor, 1989) produced by PCR of oligos containing the desired insertion. (See below) <br><br>
The  template plasmid DNA was pJS3,   or an analogue of  this containing  a variant of  Subtilase  309._<br>
Insertions were  introduced by oligo directed mutagenesis to the  construction of G102GX  insertion variants   (X =  any amino acid residue   inserted between positions  102  and 103)   resulting in G102GX  subtilase  309 variants.<br>
The  Subtilase  309 variants  were transformed into E.   coli, DNA purified  from a over night  culture of  these  transformants were  transformed into B.   subtilis by<br>
restriction  endonuclease digestion,   purification of DNA fragments,   ligation,   transformation of B.   sujbtilis. Transformation of B.   subtilis was performed as  described by Dubnau  et   al. ,   1971,   J.   Mol.   Biol.   56,   pp.   209-221.<br>
LOCALIZED  RANDOM MUTAGENESIS   IN ORDER TO   INSERT RANDOM INSERTIONS   IN A LOCALIZED REGION:<br>
The  overall strategy to used to perform localized random mutagenesis was:<br>
a mutagenic primer   (oligonucleotide)   was  synthesized, that  corresponds  to the DNA sequence  flanking the  site of insertion,   separated by the DNA base pairs defining the insertion.<br>
Subsequently,   the resulting mutagenic primer was used in a PCR reaction with a  suitable opposite primer.   The resulting PCR fragment  was purified and extended in a  second PCR-reaction, before being digested by endonucleases and cloned into the E. coli   - B.   subtilis shuttle vector   (see below) ,<br>
Alternatively,   and if necessary,   the resulting PCR fragment  is  used in a  second PCR reaction as a primer with a second suitable opposite primer to allow digestion and cloning of the mutagenized region into the  shuttle vector •   The PCR reactions  are performed under normal conditions.<br><br>
Following this strategy a localized random library was constructed in SAVINASE wherein insertions were introduced in the active site loop region between positions 102 and 103,<br>
The mutations were introduced by mutagenic primers (see  below) , so that all 20 amino acids are represented (N = 25% of A, T, C, and G; whereas S = 50% C and G. The produced PCR fragment were extended towards the N-terminal of Savinase by another round of PCR by combination of a overlapping sequence with a PCR-fragment produced by PCR-amplification with primers;  5 CTA AAT ATT CGT GGTGGC GC 3' (sense)   and 5' GAG TTT AAC AGC GTA TAG CTC AGC 3' (antisense) . The extended DNA-fragments were cloned into the Hind III- and Mlu I- sites of the modified plasmid pJS3 (see above) ,   and ten randomly chosen E.   coli colonies were sequenced to confirm the mutations designed._<br>
The mutagenic primer (5' CTA GGG GCG AGC GGT TCA GGT NNS TCG GTC AGC TCG ATT GCC CAA G 3' (sense) )   were used in a PCR reaction with a suitable anti-sense opposite primer, situated downstream of the Mlu I site in pJS3 (e.g.   5'- CCC TTT AAC CGC ACA GCG TTT -3' (anti-sense)) and the plasmid pJS3 as template. This resulting PCR product was cloned into the pJS3 shuttle vector by using the restriction enzymes Hind III and Mlu I.<br>
The random library was transformed into E.coli by well known techniques.<br>
The library prepared contained approximately 100,000 individual clones/library.<br>
Ten randomly chosen colonies were sequenced to confirm the mutations designed.<br>
In order to purify a subtilase variant of the invention, the B. sujbtilis pJS3 expression plasmid comprising a variant of the invention was transformed into a competent B.   subtilis strain and was fermented as described above in a medium containing 10 jig/ml Chloramphenicol (CAM) ,<br><br>
EXAMPLE 2<br>
PURIFICATION OF ENZYME VARIANTS:<br>
This procedure relates to purification of a 2 liter scale fermentation for the production of the subtilases of the invention in a Bacillus  host cell.<br>
Approximately 1.6 liters of fermentation broth were centrifuged at 5000 rpm for 35 minutes in 1 liter beakers. The supernatants were adjusted to pH 6.5 using 10% acetic acid and filtered on Seitz Supra SlOO filter plates.<br>
The filtrates were concentrated to approximately 400 ml using an Amicon CH2A UF unit equipped with an Amicon SIYIO UF cartridge. The UF concentrate was centrifuged and filtered prior to absorption at room temperature on a Bacitracin affinity column at pH 7. The protease was eluted from the Bacitracin column at room temperature using 25% 2-propanol and 1 M sodium chloride in a buffer solution with 0.01 dimethylglutaric acid, 0.1 M boric acid and 0.002 M calcium chloride adjusted to pH 7.<br>
The fractions with protease activity from the Bacitracin purification step were combined and applied to a 750 ml Sephadex G25 column (5 cm dia.) equilibrated with a buffer containing 0.01 dimethylglutaric acid, 0.2 M boric acid and 0,002 m calcium chloride adjusted to pH 6.5.<br>
Fractions with proteolytic activity from the Sephadex G25 column were combined and applied to a 150 ml CM Sepharose CL 6B cation exchange column (5 cm dia.) equilibrated with a buffer containing 0.01 M dimethylglutaric acid, 0.2 M boric acid, and 0.002 M calcium chloride adjusted to pH 6.5.<br>
The protease was eluted using a linear gradient of 0-0.1 M sodium chloride in 2 litres of the same buffer (0-0.2 M sodium chloride in case of Subtilisin 147).<br><br>
In a final purification step protease containing fractions from the CM Sepharose column were combined and concentrated in an Amicon ultrafiltration cell equipped with a GR81PP membrane (from the Danish Sugar Factories Inc.).<br>
By using the techniques of Example 1 for the construction and fermentation, and the above isolation procedure the following subtilisin 309 variants were produced and isolated:<br><br>
These variants exhibited better wash performance than Savinase in a preliminary assay.<br>
EXAMPLE 3<br>
WASH PERFORMANCE OF DETERGENT COMPOSITIONS COMPRISING ENZYME VARIANTS<br>
The following examples provide results from a number of washing tests that were conducted under the conditions indicated<br>
MINI WASH<br><br><br>
DETERGENTS:<br>
The detergents used were obtained from supermarkets in Denmark (OMO, datasheet ED-9745105) and the USA (Wisk, datasheet 3D-9711893), respectively. Prior to use all enzymatic activity In the detergents was inactivated by'micro wave treatment.<br>
SWATCHES;<br>
The swatches used were EMPA116  and EMPA117,   obtained from 3MPA Testmaterialen,   Movenstrasse  12,   CH-9015 St.   Gall, Switzerland.<br>
REFLECTANCE<br>
Measurement  of  reflectance   (R)   on the test  material was ione at  460  nm using a Macbeth ColorEye  7000 photometer.   The ineasurements were  done according to the manufacturers protocol.<br><br>
EVALUATION<br>
The evaluation of the wash performance of a subtilase is determined by either the improvement factor or the performance factor for the subtilase iinvestigated.<br>
The improvement factor, IFoose/response' is defined as the ratio between the slopes of the wash performance curves for a detergent containing the subtilase investigated and the same detergent containing a reference subtilase at the asymptotic concentration of the subtilase goes to zero<br>
^^Dose/response^ ^/^ref<br>
The wash performance is calculated according to the formula I:<br><br>
where<br>
R is the wash performance in reflectance units; RQ  is the intercept of the fitted curve with y-axis (blind) ; a is the slope of the fitted curve as c —&gt; 0; c is the enzyme concentration; and ARmax is the theoretical maximal wash effect<br>
as c -&gt; 00.<br>
The performance factor, P, is calculated according to the formula II<br><br>
where<br>
Rvariant is the reflectance of test material washed with lOnM<br>
variant; Rsavinase is the reflectance of test material washed<br><br>
with lOnM Savinase; Rblank is the reflectance of test material washed with no enzyme<br><br>
The subtilases of the inventions are thus seen to exhibit improved wash performance in comparison to Savinase*.<br><br>
SEQUENCE LISTING  Novo Nordisk A/S<br> Subtilase enzymes of the I-Sl and 1-S2  sub-groups<br>
having an ad-ditional amino acid residue in an active site loop region.<br> 5797.204<br><br> 2<br> Patentin Ver. 2.1<br>	1<br>	275<br>	PRT<br>	Bacillus licheniformis<br><br><br>
130	135 ,	±'±u<br>
Arg Gly Val Val Val Val Ala Ala Ala Gly Asn Ser Gly Ser Ser Gly<br>
145	150	155	160<br>
Asn Thr Asn Thr lie Gly Tyr Pro Ala Lys Tyr Asp Ser Val lie Ala<br>
165	170	175<br>
Val Gly Ala Val Asp Ser Asn Ser Asn Arg Ala Ser Phe Ser Ser Val<br>
180	185	190<br>
Gly Ala Glu Leu Glu Val Met Ala Pro Gly Ala Gly Val Tyr Ser Thr<br>
195	200	205<br>
Tyr Pro Thr Ser Thr Tyr Ala Thr Leu Asn Gly Thr Ser Met Ala Ser<br>
210	215	220<br>
Pro His Val Ala Gly Ala Ala Ala Leu He Leu Ser Lys His Pro Asn<br>
225	230	235	240<br>
Leu Ser Ala Ser Gin Val Arg Asn Arg Leu Ser Ser Thr Ala Thr Tyr<br>
245	250	255<br>
Leu Gly Ser Ser Phe Tyr Tyr Gly Lys Gly Leu He Asn Val Glu Ala<br>
260	265	270<br>
Ala Ala Gin 275<br> 2<br> 270<br> PRT<br> Bacillus lentus<br> 2<br>
Ala Gin Ser Val Pro Trp Gly He Ser Arg Val Gin Ala Pro Ala Ala<br>
15	10	15<br>
His Asn Arg Gly Leu Thr Gly Ser Gly Val Lys Val Ala Val Leu Asp<br>
20	25	30<br>
Thr Gly He Ser Thr His Pro Asp Leu Asn He Arg Gly Gly Ala Ser<br>
35	40	45<br>
Phe Val Pro Gly Glu Pro Ser Thr Gin Asp Gly Asn Gly His Gly Thr<br>
50	55	60<br><br>
His Val Ala Gly Thr He Ala Ala Leu Asn Asn Ser He Gly Val Leu<br>
65	70	75	80<br>
Gly Val Ala Pro Ser Ala Glu Leu Tyr Ala Val Lys Val Leu Gly Ala<br>
85	90	95<br>
Ser Gly Ser Gly Xaa Ser Val Ser Ser He Ala Gin Gly Leu Glu Trp<br>
100	105	110<br>
Ala Gly Asn Asn Gly Met His Val Ala Asn Leu Ser Leu Gly Ser Pro<br>
115	120	125<br>
Ser Pro Ser Ala Thr Leu Glu Gin Ala Val Asn Ser Ala Thr Ser Arg<br>
130	135	140<br>
Gly Val Leu Val Val Ala Ala Ser Gly Asn Ser Gly Ala Gly Ser He<br>
145	150	155	160<br>
Ser Tyr Pro Ala Arg Tyr Ala Asn Ala Met Ala Val Gly Ala Thr Asp<br>
165	170	175<br>
Gin Asn Asn Asn Arg Ala Ser Phe Ser Gin Tyr Gly Ala Gly Leu Asp<br>
180	185	190<br>
He Val Ala Pro Gly Val Asn Val Gin Ser Thr Tyr Pro Gly Ser Thr<br>
195	200	205<br>
Tyr Ala Ser Leu Asn Gly Thr Ser Met Ala Thr Pro His Val Ala Gly<br>
210	215	220<br>
Ala Ala Ala Leu Val Lys Gin Lys Asn Pro Ser Trp Ser Asn Val Gin<br>
225	230	235	240<br>
He Arg Asn His Leu Lys Asn Thr Ala Thr Ser Leu Gly Ser Thr Asn<br>
245	250	255<br>
Leu Tyr Gly Ser Gly Leu Val Asn Ala Glu Ala Ala Thr Arg<br>
260	265	270<br><br><br><br><br>
PATENT CLAIMS<br>
1.  A Subtilase enzymes of the I-S1 and I-S2 sub-groups having at least one additional amino acid residue in position 102 of the  active site loop (b) region from position 95 to 103, whereby-said additional amino acid residue(s) correspond to the insertion of at least one amino acid residue between positions 102 and 103.<br>
  2.  The isolated subtilase enzyme of claim 1, wherein said subtilase enzyme is a constructed variant having at least one inserted amino acid residue between positions 102 and 103 of a precursor subtilase.<br>
 3-  The isolated subtilase enzyme of claim 1 or 2 selected from the group comprising<br>
X102X{A,T,G,S}, X102X{D,E,K,R}; X102X{H,V,C,N,Q}, and X102X{F,I,L,M,P,W,Y}<br>
4.	The isolated subtilase enzyme of claim 3, wherein said at least one additional or inserted amino acid residue is chosen from the group comprising: T, G, A, and S,<br>
5.	The isolated subtilase enzyme of claim 3, wherein said at least one additional or inserted amino acid residue is chosen from the group of charged amino acid residues comprising: D,E,H,K, and R, more preferably D,E,K and R.<br>
6.	The isolated subtilase enzyme of claim 3, wherein said at least one additional or inserted amino acid residue is chosen<br><br>
from the group of hydrophilic amino acid residues comprising: C,N,Q,S and T, more preferably N,Q,S and T.<br>
7.	The isolated subtilase enzyme of claim 3, wherein said at least one additional or inserted amino acid residue is chosen from the group of small hydrophobic amino acid residues comprising: A,G and V.<br>
8.	The isolated subtilase enzyme of claim 3, wherein said at least one additional or inserted amino acid residue is chosen from the group of large hydrophobic amino acid residues comprising: F,I,L,M,P,W and Y, more preferably F,I,L,M, and Y.<br>
9.	The isolated subtilase enzyme according to any of the preceding claims, wherein said at least one additional or inserted amino acid residue, comprises more than one additional or inserted amino acid residue in the active site loop (b) .<br>
10.	The subtilase variant of any of the preceding claims, wherein said insertion (s) between positions 102 and 103 are combined with one or more further modification(s) in any other position(s).<br>
11.	The subtilase variant of claim 15, wherein said further modification(s) are in one or more of the positions 27, 36, 57, 76, 87, 97, 101, 104, 120, 123, 167, 170, 206, 218, 222, 224, 235 and 274.<br>
12- The subtilase variant of any of the preceding claims, wherein said modification(s) is/are combined with modification(s) in one or more of the positions 129, 131, 133 and 194.<br><br>
13.	The subtilase of any of the preceding claims, wherein the subtilase, or if the subtilase is a variant the parent subtilase, belongs to the sub-group I-S1.<br>
14.	The subtilase of claim 13, wherein the parent subtilase is chosen from the group comprising ABSS168, BASBPN, BSSDY, and BLSCAR, or functional variants thereof having retained the characteristic of sub-group I-Sl.<br>
15.	The subtilase according to any of claims 1-14, wherein the subtilase, or if the subtilase is a variant, the parent subtilase belongs to the sub-group I-S2,<br>
16.	The subtilase of claim 15, wherein the parent subtilase is chosen from the group comprising BLS147, BLS309, BAPB92, TVTHER and BYSYAB, or functional variants thereof having retained the characteristic of sub-group I-S2.<br>
17.	The isolated subtilase enzyme of claim 3, 15 or 16 selected from the group comprising<br>
G102GA, G102GT, G102GG, G102GS, G102GD, G102GE, G102GK, G102GR, G102GH, G102GV, G102GC, G102GN,<br><br>
G102GQ,<br>
G102GF,<br>
G102GI,<br>
G102GL,<br>
G102GM,<br>
G102GP,<br>
G102GW, and<br>
G102GY.<br>
10 18. The subtilase variant of any of the claims 15 to 17, wherein said further modification(s) are chosen from the group comprising K27R, *36D, S57P, N76D, S87N, G97N, SIOIG, V104A, V104N, V104Y, H120D, N123S, Y167X, R170X, Q206E, N218S, M222S, M222A, T224S, K235L, and T274A.<br>
19- The subtilase variant of any of the claims 15 to 17, wherein said further tnodification(s) are chosen from the group comprising S101G+V104N, S87N+S101G+V104N,<br>
K27R+V104Y+N123S+T274A, N76D+S103A+V104I or N76D+V104A, or other combinations of these mutations (V104N, SIOIG, K27R, V104Y, N123S, T274A, N76D; V104A) , in combination with any one or more of the substitutions, deletions and/or insertions mentioned in any of claims 1 to 14.<br>
20.	The subtilase variant of any of the claims 15 to 17, wherein said further modification (s) are chosen from the group further comprising P129K, P131H, A133P, A133D and A194P,<br>
21.	The variant according to any of the preceding claims comprising the modification chosen from the group comprising: G102GT+Y167A<br><br>
22.	A subtilase belonging to the I-Sl sub-group having the amino<br>
acid sequence:<br>
1	10	20	30<br>
A-Q-T-V-P-Y-G-I-P-L-I-K-A-D-K-V-Q-A-Q-G-F-K-G-A-N-V-K-V-A-V<br>
40	50	60<br>
L-D-T-G-1 -Q-A-S-H-P-D-L-N-V-V-G-G-A-S-F-V-A-G-E-A- * -Y-N-T-D<br>
70	80	90<br>
G-N-G-H-G-T-H-V-A-G-T-V-A-A-L-D-N-T-T-G-V-L-G-V-A-P-S-V-S-L<br>
102a	110	120<br>
Y-A-V-K-V-L-N-S-S-G-S-G-X-T-Y-S-G-I-V-S-G-I-E-W-A-T-T-N-G-M-D<br>
130	140	150<br>
V-I-N-M-S-L-G-G-P-S-G-S-T-A-M-K-Q-A-V-D-N-A-Y-A-R-G-V-V-V-V<br>
160	170	180<br>
A-A-A-G-N-S-G-S-S-G-N-T-N-T-I-G-Y-P-A-K-Y-D-S-V-I-A-V-G-A-V<br>
190	200	210<br>
D-S-N-S-N-R-A-S-F-S-S-V-G-A-E-L-E-V-M-A-P-G-A-G-V-Y-S-T-Y-P<br>
220	230	240<br>
T-S-T-Y-A-T-L-N-G-T-S-M-A-S-P-H-V-A-G-A-A-A-L-I-L-S-K-H-P-N<br>
250	260	270<br>
L-S-A-S-Q-V-R-N-R-L-S-S-T-A-T-Y-L-G-S-S-F-Y-Y-G-K-G-L-I-N-V<br>
275 E-A-A-A-Q<br>
or a homologous subtilase having an amino acid sequence comprising a position 102a amino acid residue and exhibiting an identity of more than 70%, 75%, 80%, 85%, 90%, or 95% therewith.<br>
23.	A subtilase belonging to the I-S2 sub-group having the amino<br>
acid sequence:<br>
.1	10	20	30<br>
•A-Q-S-V-P-W-G-I-S-R-V-Q-A-P-A-A-H-N-R-G-L-T-G-S-G-V-K-V-A-V-<br>
40	50	60<br>
. L-D-T-G-1 -* -S-T-H-P-D-L-N-1 -R-G-G-A-S-F-V-P-G-E-P- * -S-T-Q-D-<br><br>
70        .80	90	.<br>
. G-N-G-H-G-T-H-V-A-G-T-1 -A-A-L-N-N-S-1 -G-V-L-G-V-A-P-S-A-E-L-<br>
102a	110	120<br>
.Y-A-V-K-V-L-G-A-S-G-S-G-X-S-V-S-S-I-A-Q-G-L-E-W-A-G-N-N-G-M-H-<br>
130	140	150<br>
.V-A-N-L-S-L-G-S-P-S-P-S-A-T-L-E-Q-A-V-N-S-A-T-S-R-G-V-L-V-V-<br>
160	170	180<br>
.A-A-S-G-N-S-G-A-*-G-S-I-S-*-*-*-Y-P-A-R-Y-A-N-A-M-A-V-G-A-T-<br>
190	200	210<br>
• D-Q-N-N-N-R-A-S-F-S-Q-Y-G-A-G-L-D-I-V-A-P-G-V-N-V-Q-S-T-Y-P-<br>
220	230	240<br>
.G-S-T-Y-A-S-L-N-G-T-S-M-A-T-P-H-V-A-G-A-A-A-L-V-K-Q-K-N-P-S-<br>
250	260	270<br>
.W-S-N-V-Q-I-R-N-H-L-K-N-T-A-T-S-L-G-S-T-N-L-Y-G-S-G-L-V-N-A-<br>
275 . E-A-A-T-R<br>
or a homologous subtilase having an amino acid sequence comprising a position 102a amino acid residue and exhibiting an identity of more than 70%, 75%, 80%, 85%, 90%, or 95% therewith.<br>
24.	The subtilase variant of claims 22 or 23, wherein X in position 102a is chosen from the group comprising T, A, G, S, and P.<br>
25.	An isolated DNA sequence encoding a subtilase or a subtilase variant of any of the claims 1 to 24.<br>
26.	An expression vector comprising an isolated DNA sequence of claim 25.<br>
27.	A microbial host cell transformed with an expression vector of claim 26.<br><br>
28.	The microbial host of claim 27, which is a bacterium, preferably a Bacillus, especially B.   lentus.<br>
29.	The microbial host of claim 21,   which is a fungus or yeast, preferably a filamentous fungus, especially an Aspergillus.<br>
30.	A method for producing a subtilase or a subtilase variant of any of claims 1 to 24, wherein a host of any of claims 27 to 29 is cultured under conditions conducive to the expression and secretion of said variant, and the variant is recovered.<br>
31.	A composition comprising a subtilase or a subtilase variant according to any of claims 1 to 24.<br>
32.	The composition according to claim 31, which additionally comprises a cellulase, lipase, cutinase, oxidoreductase, another protease, or an amylase.<br>
33.	The composition according to claim 31 or 32, wherein the composition is a detergent composition.<br>
34.	Use of a subtilase or a subtilase variant according to any of claims 1 to 24 or an enzyme composition according to claims 31 or 32 in a laundry and/or a dishwash detergent.<br><br>
35. A subtilase enzyraes of the I-Sl and I-S2 sub-groups substantially as hereinbefore described with reference to the accompanying drawings.<br>
36.An isolated DNA sequence substantially as hereinbefore described with reference to the accompanying drawings.<br>
37.An expression vector substantially as hereinbefore described with reference to the accompanying drawings.<br>
38. A method for producing a subtilase substantially as hereinbefore described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1jbGFpbXMgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1jbGFpbXMgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1jb3JyZXNwb25kbmVjZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1kZXNjcmlwdGlvbihjb21wbGV0ZSkgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-description(complete) filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1vdGhlciBkb2N1bWVudHMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtODIxLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-821-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="208697-subtilase-enzymes-having-an-additional-amino-acid-residue.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208699-a-process-for-purification-of-terephthalic-acid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208698</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/821/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-Jun-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsvaerd</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDERSEN VILBOUR, Kim</td>
											<td>Tasingegade 31, 4.th. DK-2100 Copenhagen</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MIKKELSEN, Frank</td>
											<td>Novo Nordisk A/S, Novo Alle, DK-2880 Bagsvaerd</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HANSEN KAMP, Peter</td>
											<td>Novo Nordisk A/S, Novo Alla, DK-2880 Bagsvaerd</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ANDERSEN, Carsten</td>
											<td>Novo Nordisk A/S, Novo Alla, DK-2880 Bagsvaerd</td>
										</tr>
										<tr>
											<td>5</td>
											<td>NORREGAARD-MADSEN, Mads</td>
											<td>Novo Nordisk A/S, Novo Alla, DK-2880 Bagsvaerd</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK99/00717</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-12-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 1998 01671</td>
									<td>1998-12-18</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208698-subtilase-enzymes-having-an-additional-amino-acid-residue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:19:06 GMT -->
</html>
