<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208673-a-recombinant-host-microorganism-and-a-method-of-making-a-recombinant-microorganism by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:13:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208673:A RECOMBINANT HOST MICROORGANISM AND A METHOD OF MAKING A RECOMBINANT MICROORGANISM</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT HOST MICROORGANISM AND A METHOD OF MAKING A RECOMBINANT MICROORGANISM</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a recombinant host microorganism comprising a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase polypeptide under the transcriptional control of a surrogate promoter, said promoter capable of causing increased expression of said polysaccharase polypeptide; and a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein expression of said fIrst and second polynucleotide segments results in the increased production of a polysaccharase by the recombinant host microorganism. This invention also relates to a method of making a recombinant microorganism.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
RECOMBINANT HOSTS SUITABLE FOR SIMULTANEOUS SACCHARIFICATION AND FERMENTATION<br>
Related Information<br>
This application claims priority to U.S. provisional Application No. 60/136.376, entitled "RECOMBINANT HOSTS SUITABLE FOR SIMULTANEOUS SACCHARIFICATION AND FERMENTATION," filed on may 26, 1999, incorporated herein in its entirety by this reference. The contents of all patents, patent applications. and references cited throughout this specification are hereby incorporated by reference in their entireties.<br>
Government Sponsored Research<br>
This work was supported, in part, by grants from the United States Department of Agriculture, National Research Initiative (95-37308-1843; 98-35504-6] 77). and United States Department of Energy (DE-FG02-96ER20222)<br>
Background of the Invention<br>
Many environmental and societal benefits would result from the replacement of petroleum-based automotive fuels with renewable fuels obtained from plant materials (Lynd et al, {\99\) Science 251:1318-1323; Olson et al, (1996) Enzyme Microh TechnoL 18:1-17; Wyman et al., (1995)Amer. Chem. Soc. Symp. 618:272-290)   Each year, the United States bums over 120 billion gallons of automotive fuel, roughly equivalent to the total amount of imported petroleum. The development of ethanol as a renewable alternative fiiel has the potential to eliminate United States dependence on imported oil, improve the environment, and provide new employment (Sheehan, (1994) ACS Symposium Series No. 566, ACS Press, pp 1-53).<br>
In theory, the solution to the problem of imported oil for automotive fuel appears quite simple. Rather than using petroleum, a finite resource, the ethanol can be produced efficiently by the fermentation of plant material, a renewable resource '    Indeed, Brazil has demonstrated the feasibility of producing ethanol and the use of ethanol as a primary automotive fuel for more than 20 years. Similarly, the United States produces over 1.2 billion gallons of fuel ethanol each year. Currently, fuel<br><br>
ethanol is produced from com starch or cane syrup utilizing either Saccharomyccs cerevisiae or Zymomonas mobilis (Z mobilis). However, neither of these sugar sources can supply the volumes needed to realize a replacement of petroleum-based automotive fuels. In addition, both cane sugar and com starch are relatively expensive starting materials which have competing uses as food products.<br>
Moreover, these sugar substrates represent only a fraction of the total carbohydrates in plants. Indeed, the majority of the carbohydrates in plants is in the form of lignocellulose, a complex structural polymer containing cellulose, hemicellulose, pectin, and lignin. Lignocellulose is found in, for example, the stems. leaves, hulls, husks, and cobs of plants. Hydrolysis of these polymers releases a mixture of neutral sugars including glucose, xylose, mannose, galactose, and arabinosc. No known natural organism can rapidly and efficiently metabolize all these sugars into ethanol.<br>
Nonetheless, in an effort to exploit this substrate source, the Gulf Oil Company developed a method for the production of ethanol from cellulose using a yeast-based process termed simultaneous saccharification and fermentation (SSF) (Gauss et al (1976) U.S.P.N. 3,990,944). Fungal cellulase preparations and yeasts were added to a slurry of the cellulosic substrate in a single vessel. Ethanol was produced concurrently during cellulose hydrolysis. However, Gulf's SSF process has some shortcomings. I or example, fungal cellulases have been considered, thus far, to be too expensive for use in large scale bioethanol processes (Himmel et ai, (1997) Amer. Chem. Soc. pp. 2-45: Ingram et at, {\9%l)Appl Environ. Microbiol 53:2420-2425; Okamoto et al. (1994) Appl Microbiol Biotechnol 42:563-568; Philippidis, G., (1994) Amer. Chem. Soc. pp. 188-217; Saito et al, (1990) J. Ferment Bioeng. 69:282-286; Sheehan, J., (1994) Amer. Chem. Soc. pp 1-52; Su et al, (1993) Biotechnol Lett. 15:979-984).<br>
Summary of the Invention<br>
The development of inexpensive enzymatic methods for cellulose hydrolysis has great potential for improving the efficiency of substrate utilization and the economics of the saccharification and fermentation process. Accordingly, developing a biocatalyst which can be used for the efficient depolymerization of a complex cellulosic substrate<br><br>
and subsequent rapid fermentation of the substrate into ethanol, would be of great benefit.<br>
The present invention provides a recombinant host cell engineered for increased expression and secretion of a polysaccharase suitable for depolymerizing complex carbohydrates. Specifically exemplified are two recombinant enteric bacteria. Escherichia coli and Klebsiella oxytoca, which express a polysaccharase at high  levels under the transcriptional control of a surrogate promoter. The invention provides for the further modification of these hosts to include a secretory protein/s which allows for the increased production of polysaccharase in cell. In a preferred embodiment, the polysaccharase is produced in either increased amounts, with increased activity, or a combination thereof. In a preferred embodiment, the invention provides for the further modification of these hosts to include exogenous ethanologenic genes derived from an efficient ethanol producer, such as Zymomonas mohilis. Accordingly, these hosts are capable of expressing high levels of proteins that may be used alone or in combination with other enzymes or recombinant hosts for the efficient production of ethanol from complex carbohydrates.<br>
More particularly, in a first aspect, the present invention features a recombmant host cell having increased production of a polysaccharase. The host cell of this aspect contains a heterologous polynucleotide segment containing a sequence that encodes a polysaccharase where the sequence is imder the transcriptional control of a surrogate promoter and this promoter is capable of causing increased production of the polysaccharase. In addition, this aspect features a host cell that also contains a second heterologous polynucleotide segment containing a sequence that encodes a secretory polypeptide. The expression of the first and second heterologous polynucleotide segments results in the increased production of polysaccharase amounts, activ ity, or a combination thereof, by the recombinant host cell.<br>
In a preferred embodiment, the polysaccharase polypeptide is secreted.<br>
In another embodiment, the host cell is a bacterial cell, preferably Gram-negative, facultatively anaerobic, and from the family Enterobacteriaceae. In another related embodiment, the recombinant host cell is of the genus Escherichia or Klebsiella and, preferably, is the strain E, coli B, £ coli DH5a, E. coli K04 (ATCC 55123). E<br><br><br>
In even another embodiment, the host cell of the above aspect and foregoing embodiments is ethanologenic.<br>
In a second aspect, the present invention provides a recombinant ethanologenic host cell containing a heterologous polynucleotide segment that encodes a polysaccharase and this segment is under the transcriptional control of an exogenous surrogate promoter.<br>
In one embodiment, the host cell is a bacterial cell, preferably Gram-negative, facultatively anaerobic, and from the family Enterobacteriaceae. In a related embodiment, the recombinant ethanologenic host cell is of the genus Escherichia or Klebsiella and, preferably, is the strain E, coli B, £. coli DH5a, E. coli K04 (ATCC 55123),E. coli KOll (ATCC 55124),£. co//K012 (ATCC 55125),E. coli LYO1 K oxytoca M5A1, or K. oxytoca P2 (ATCC 55307).<br><br>
In another embodiment, the recombinant host cell contains a polynucieoute segment that encodes a polysaccharase that is a glucanase, endoglucanase, exoglucanase, cellobiohydrolase, a-glucosidase, endo-l,4-a-xylanase, P-xylosidasc. (V glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase<br><br><br>
glucoamylase, pullulanase, p-glucanase, hemicellulase, arabinosidase, mannanasc. pectin hydrolase, pectate lyase, or a combination of these polysaccharases.<br>
In a preferred embodiment, the first polypeptide of the recombinant host is the polysaccharase glucanase, preferably an expression product of the celZ gene, and more preferably, is derived from Erwinia chrysanthemi.<br><br><br>
production of the gene-of-interest such that the increased expression indicates that the fragment is a surrogate promoter.<br>
In a sixth aspect, the invention provides a method of making a recombinant host cell for use in simultaneous saccharification and fermentation. In particular, the method involves introducing into the host cell a first heterologous polynucleotide segment containing a sequence encoding a polysaccharase polypeptide under the transctrinonal control of a surrogate promoter, the promoter being capable of causing increased expression of the polysaccharase polypeptide. In addition, the method further includes introducing into the host cell a second heterologous polynucleotide segment containing a sequence encoding a secretory polypeptide/s such that the expression of the fust and second polynucleotide segments results in the increased production of a pol) saccharase polypeptide by the recombinant host cell. In a preferred embodiment, the increased production of the polysaccharase polypeptide is an increase in activity, amount, or a combination thereof. In another preferred embodiment, the polysaccharase polypeptide is secreted. In a more preferred embodiment, the host cell is ethanologenic.<br>
In a seventh aspect, the invention features a vector comprising the sequence of pLOI2306(SEQIDNO:12).<br>
In an eighth aspect, the invention features a host cell comprising the foregoing vector.<br>
In a ninth aspect, the invention features a method of making a recombinant host cell integrant including the steps of introducing into the host a vector comprising the sequence of pLOI2306 and identifying a host cell having the vector stably integrated<br>
In a tenth aspect, the invention features a method for expressing a poly saccharase in a host cell encompassing the steps of introducing into the host cell a vector containing the polynucleotide sequence of pLOI2306 and identifying a host cell expressing the polysaccharase. In a preferred embodiment, each of the above aspects features a host cell that is ethanologenic.<br>
In an eleventh aspect, the invention provides a method for producing ethanol from an oligosaccharide source by contacting said oligosaccharide source with a ethanologenic host cell containing a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase under the transcriptional control of a sunogate promoter. Moreover, the promoter is capable of causing increased expression<br><br>
of the polysaccharase. In addition, the ethanologenic Host contains a second heterologous polynucleotide segment comprising a sequence encoding a secreton polypeptide. The expression of said first and second polynucleotide segments of the-ethanologenic host cell resuh in the increased production of polysaccharase activity by the host cell such that the oligosaccharide source is enzymatically degraded and fermented into ethanol.<br>
In one embodiment, the first polypeptide of the recombinant host is a polysaccharase, and, preferably the polypeptide is of increased activity. In a related embodiment, the polysaccharase is a glucanase, endoglucanase, exoglucanase. cellobiohydrolase, a-giucosidase, endo-l,4-a-xylanase, p-xylosidase, p-glucuronidase. a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase. glucoamylase, pulluianase, β-glucanase, hemicellulase, arabinosidase, mannanase. pectin hydrolase, pectate lyase, or a combination of these polysaccharases.<br>
In a preferred embodiment, the first polypeptide of the recombinant host is the polysaccharase glucanase, preferably an expression product of the cell gene, and more preferably, is derived from Envinia chrysanthemi.<br>
In another embodiment, the second heterologous polynucleotide segment of the recombinant host cell contains at least onepul gene or out gene, preferably derived from a bacterial cell from the family Enterobacteriaceae and more preferably, from K. oxytoca, E, carotovora, E, carotovora subspecies carotovora, E. carotovora subspecies atroseptica, or E. chrysanthemi.<br>
In another embodiment, the recombinant host cell is a facultatively anaerobic bacterial cell. In a related embodiment, the host cell is from the family Enterobacteriaceae, preferably Escherichia or Klebsiella, and more preferably, is the strain E. coli K04 (ATCC 55123), £ coli KOI 1 (ATCC 55124), E coli KO12 (ATCC 55125), K. oxytoca M5A1, or K oxytoca P2 (ATCC 55307).<br>
In another embodiment, the method is carried out in an aqueous solution. In even another embodiment, the method is used for simultaneous saccharification and fermentation. In still another embodiment, the oligosaccharide is preferably lignocellulose, hemicellulose, cellulose, pectin, or any combination of these oligosaccharides.<br><br>
In yet another embodiment, the method uses a nucleic acid construct that is of is derived from, a piasmid selected from the group consisting of pLOI2306.<br>
Other features and advantages of the invention will be apparent from the following detailed description and claims.<br>
Brief Description of the Drawings<br>
Figure 1 shows fermentation rates for the ethanologenic recombinant host E coli KOI 1 using rice hull substrates pretreated with dilute acid and supplemented with two different medias.<br>
Figure 2 shows simultaneous saccharification and fermentation (SSF) rates for the ethanologenic recombinant host strain K. oxytoca P2 using mixed waste office paper. Insoluble residues from SSF were recycled as a source of bound cellul ase enzymes and substrate during subsequent fermentations.<br>
Figure 3 shows the structure of the piasmid pLOI2171, a low copy promoter probe vector showing the orientation of the kanamycin resistance gene {kan) for selection, the temperature sensitive pSClOl replicon (Rep(ts)) for episomal maintenance of the piasmid, and the promoterless polysaccharase gene ce/Z encoding glueanase (EGZ).<br>
Figure 4 is a graph showing the high correspondence between the size of the zone of clearance on CMC indicator plates (x-axis) measured for a transformed bacterial colony and the amotint of glueanase activity expressed (y-axis).<br>
Figure 5 shows the partial nucleotide sequence (SEQ ID NO: 1) of the Z mohilis DNA fragment in the pL0I2183 piasmid that functions as a surrogate promoter. The full sequence has been assigned GenBank accession number AF109242 (SEQ ID NO 2). Indicated are two transcriptional start sites (#), -35 and -10 regions, the Shine-Delgamo site (bold), partial vector and celZ sequence (lowercase), and the celZ start codon (atg indicated in bold).<br>
Figure 6 represents electron micrographs of E. coli B.cells harboring different plasmids expressing little if any (pUC19; panel A), moderate (pLOI2164; panel B), and high levels (pLOI2307; panel C) of glueanase in the form of periplasmic inclusion bodies (pib) localized between the outer cell wall and the inner membrane (im). The bar shown represents 0.1 µm.<br><br>
Figure /shows a schematic detailing the cloning strategy used to construet  th celZ integration vector pLOH306, a genetic construct capable of being introduced into the genome of a recombinant host and conferring stable glucanase expression activity to the host.<br>
Figure 8 shows a schematic representation of the celZ integration vector pLOI2306 (SEQ ID NO: 12) with the locations of the surrogate promoter from Z mobilis, the ce/Z gene from E. chrysanthemi, resistance markers {bla and tet) and A oxytoca target sequence indicated.<br>
Detailed Description of the Invention<br>
In order for the full scope of the invention to be clearly understood, the follou ing definitions are provided.<br>
/, Definitions<br>
As used herein the term "recombinant host" is intended to include a cell suitable for genetic manipulation, e.g,, which can incorporate heterologous polynucleotide sequences, e.g., which can be transfected. The cell can be a microorganism or a higher eukaryotic cell. The term is intended to include progeny of the cell originally transfected. In preferred embodiments, the cell is a bacterial cell, e.g., a Granvnegati\ e bacterial cell, and this term is intended to include all facultatively anaerobic Gram-negative cells of the family Enterobacteriaceae such as Escherichia, Shigella, Citrobacter, Salmonella, Klebsiella, Enterobacter, Erwinia, Kluyvera, Senatia, Cedecea, Morganella, Hafnia, Edwardsiella, Providencia, Proteus, and Yersinia. Particularly preferred recombinant hosts are Escherichia coli or Klebsiella axyioca cells.<br>
The term "heterologous polynucleotide segment" is intended to include a polynucleotide segment that encodes one or more polypeptides or portions or fragments of polypeptides. A heterologous polynucleotide segment may be derived from any source, e.g., eukaryotes, prokaryotes, virii, or synthetic polynucleotide fragments<br>
The terms "polysaccharase" or "cellulase" are used interchangeably herem and are intended to include a polypeptide capable of catalyzing the degradation or depolymerization of any linked sugar moiety, e.g., disaccharides, trisaccharides, oligosaccharides, including, complex carbohydrates, e.g., lignocellulose, which<br><br>
comprises cellulose, hemicellulose, and pectin. The terms are intended to include cellulases such as glucanases, including both endoglucanases and exoglucanascs. and P-glucosidase. More particularly, the terms are intended to include, e.g., cellobiohydrolase. endo-l,4-β-xylanase, P-xylosidase, a-glucuronidase, a-L-arabinofiiranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase. glucoamylase, puUulanase, β-glucanase. hemicellulase, arabinosidase, mannanasc. pectin hydrolase, pectate lyase, or a combination of any of these cellulases.<br>
The term "surrogate promoter" is intended to include a polynucleotide seemeni that can transcriptionally control a gene-of-interest that it does not transcriptional I \ control in nature. In a preferred embodiment, the transcriptional control of a surrogate promoter results in an increase in expression of the gene-of-interest. In a preierred embodiment, a surrogate promoter is placed 5' to the gene-of-interest. A surrogate promoter may be used to replace the natural promoter, or may be used in addition to the natural promoter. A surrogate promoter may be endogenous with regard to the host ceil in which it is used or it may be a heterologous polynucleotide sequence introduced into the host cell, e.g., exogenous with regard to the host cell in which it is used.<br>
The terms "oligosaccharide source," "oligosaccharide," "complex cellulose/ "complex carbohydrate," and "polysaccharide" are used essentially interchangeably and are intended to include any carbohydrate source comprising more than one sugar molecule. These carbohydrates may be derived from any unprocessed plant niatenal or any processed plant material. Examples are wood, paper, pulp, plant derived fiber, or synthetic fiber comprising more than one linked carbohydrate moiety, i.e.. one sugar residue. One particular oligosaccharide source is lignocellulose which represents approximately 90% of the dry weight of most plant material and contains carbohydrates, e.g., cellulose, hemicellulose, pectin, and aromatic polymers, e.g., lignin. Cellulose, makes up 30%-50% of the dry weight of lignocellulose and is a homopolymer of cellobiose (a dimer of glucose). Similarly, hemicellulose, makes up 20%-505 of the dry weight of lignocellulose and is a complex polymer containing a mixture of pentose (xylose, arabinose) and hexose (glucose, mannose, galactose) sugars which contain acetyl and glucuronyl side chains. Pectin makes up l%-20% of the dry weight of lignocellulose and is a methylated homopolymer of glucuronic acid. Any one or a combination of the above carbohydrate polymers are potential sources of sugars for<br><br>
depolymerization and subsequent bioconversion to ethanol by fermentation according to the products and methods of the present invention.<br>
The term "gene/s" is intended to include nucleic acid molecules, e.g., polynucleotides which include an open reading frame encoding a polypeptide, and can further include non-coding regulatory sequences, and introns. In addition, the term gene/s is intended to include one or more genes that map to a functional locus, e.g . the out or pul genes of Erwinia and Klebsiella, respectively, that encode more than one gene product, e.g., a secretory polypeptide.<br>
The term "gene-of-interest" is intended to include a specific gene for a selecied purpose. The gene may be endogenous to the host cell or may be recombinanth introduced into the host cell. In a preferred embodiment, a gene-of-interest is involved in at least one step in the bioconversion of a carbohydrate to ethanol. Accordingly, the term is intended to include any gene encoding a polypeptide such as an alcohol dehydrogenase, a pyruvate decarboxylase, a secretory protein/s, or a polysaccharase, e.g., a glucanase, such as an endoglucanase or exoglucanase, a cellobiohydrolase, β-glucosidase, endo-l54-β-xylanase, β-xylosidase, a-glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase, glucoamylase, pullulanase, P-glucanase. hemicellulase, arabinosidase, mannanasc. pectin hydrolase, pectate lyase, or a combination thereof<br>
The term "fragmenting a genomic polynucleotide from an organism" is intended to include the disruption of the genomic polynucleotide belonging to an organism into one or more segments using either mechanical, e.g., shearing, sonication, trituration, or enzymatic methods, e.g., a nuclease. Preferably, a restriction enzyme is used in order to facilitate the cloning of genomic fragments into a test vector for subsequent identification as a candidate promoter element. A genomic polynucleotide may be derived from any source, e.g.. eukaryotes, prokaryotes, virii, or synthetic polynucleotide fragments.<br>
The term "simultaneous saccharification and fermentation' or "SSF' is intended to include the use of one or more recombinant hosts for the contemporaneous     degradation or depolymerization of a complex sugar and bioconversion of that sugar residue into ethanol by fermentation.<br><br>
The term "transcriptional control" is intended to include the ability to moduiate gene expression at the level of transcription. In a preferred embodiment, transcription. and thus gene expression, is modulated by replacing or adding a surrogate promoter near the 5' end of the coding region of a gene-of-interest thereby resulting in altered gene expression.<br>
The term '"expression" is intended to include the expression of a gene at least ati the level of RNA production.<br>
The term "expression product" is intended to include the resultant product of an expressed gene, e.g., a polypeptide.<br>
The term "increased expression" is intended to include an alteration in gene expression at least at the level of increased RNA production and preferably, at the le\ ej of polypeptide expression.<br>
The term "increased production" is intended to include an increase in the amount of a polypeptide expressed, in the level of the enzymatic activity of the polypeptide, or a combination thereof.<br>
The terms "activity" and "enzymatic activity" are used interchangeably and are intended to include any functional activity normally attributed to a selected polypeptide when produced under favorable conditions. The activity of a polysaccharase would be, for example, the ability of the polypeptide to enzymatically depolymerize a complex saccharide. Typically, the activity of a selected polypeptide encompasses the total enzymatic activity associated with the produced polypeptide. The polypeptide produced by a host cell and having enzymatic activity may be located in the intracellular space of the cell, cell-associated, secreted into the extracellular milieu, or a combination thereof Techniques for determining total activity as compared to secreted activity are described herein and are known in the art.<br>
The term "secreted" is intended to include an increase in the secretion of a polypeptide, e.g., a heterologous polypeptide, preferably a polysaccharase. Typical)), the polypeptide is secreted at an increased level that is in excess of the naturally-occurring amount of secretion. More preferably, the term 'secreted" refers to an increase in secretion of a given polypeptide that is at least 10% and more preferably, at least 100%, 200%, 300,%, 400%, 500%, 600%, 700%, 800%, 900%, 1000%, or more. as compared to the naturally-occurring level of secretion.<br><br>
The term "secretory polypeptide" is intended to include any polypepude s. aione or in combination with other polypeptides, that facilitate the transport of another polypeptide from the intracellular space of a cell to the extracellular milieu. In one embodiment, the secretory polypeptide/s encompass all the necessary secretor\ polypeptides sufficient to impart secretory activity to a Gram-negative host cell Typically, secretory proteins are encoded in a single region or locus that ma&gt; he isolated from one host cell and transferred to another host cell using genetic engineermg   In a preferred embodiment, the secretory polypeptide/s are derived from any bacterial cell having secretory activity. In a more preferred embodiment, the secretory polypeptide s are derived from a host cell having Type II secretory activity. In another more preferred embodiment, the host cell is selected from the family Enterobacteriaceae. In a most preferred embodiment, the secretory polypeptide/s are one or more gene products of the out or pul genes derived from, respectively, Erwinia or Klebsiella. Moreover, the skilled artisan will appreciate that any secretory protein/s derived from a related host that is sufficiently homologous to the out or pul gene/s described herein may also he employed (Pugsley et al, (1993) Microbiological Reviews 57:50-108; Lindeberg ct al, (1996) Mo/. Mcro.20:175-190; Lindeberg et al., (1992)y. of Bacteriology, 1747385-7397; He et a/., (1991) Proc. Natl Acad. Set USA, 88:1079-1083).<br>
The term "derived from" is intended to include the isolation (in whole or in part) of a polynucleotide segment from an indicated source. The term is intended to include. for example, direct cloning, PCR amplification, or artificial synthesis from, or based on, a sequence associated with the indicated polynucleotide source.<br>
The term "ethanologenic" is intended to include the ability of a microorganism to produce ethanol from a carbohydrate as a primary fermentation product. The term is intended to include naturally occurring ethanologenic organisms, ethanologenic organisms with naturally occurring or induced mutations, and ethanologenic organisms which have been genetically modified.<br>
The term "Gram-negative bacteria" is intended to include the art recognized definition of this term. Typically, Gram-negative bacteria include, for example, the family Enterobacteriaceae which comprises, among others, the species Escherichia and Klebsiella.<br><br>
The term "sufficiently homologous" is intended to include a first ammo acid or nucleotide sequence which contains a sufficient or minimum number of identical or equivalent amino acid residues or nucleotides, e.g., an amino acid residue which has a similar side chain, to a second amino acid or nucleotide sequence such that the fust and second amino acid or nucleotide sequences share common structural domains and or a common functional activity. For example, amino acid or nucleotide sequences whihc share common structural domains have at least about 40% homology, preferably 50% homology, more preferably 60%, 70%, 80%, or 90% homology across the ammo acid sequences of the domains and contain at least one, preferably two, more prefcrably three, and even more preferably four, five, or six structural domains, are defined  herein as sufficiently homologous. Furthermore, amino acid or nucleotide sequences which share at least 40%, preferably 50%, more preferably 60%, 70%, 80%, or 9if/u homology and share a common functional activity are defined herein as sufficiently homologous<br>
In one embodiment, two polynucleotide segments, e.g., promoters, are "sufficiently homologous" if they have substantially the same regulatory effect as a result of a substantial identity in nucleotide sequence. Typically, "sufficiently homologous" sequences are at least 50%, more preferably at least 60%, 70%.80%, or 90% identical, at least in regions known to be involved in the desired regulation)   More preferably, no more than five bases differ. Most preferably, no more than five consecutive bases differ.<br>
To determine the percent identity of two polynucleotide segments, or two ammo acid sequences, the sequences are aligned for optimal comparison purposes (c.g gaps can be introduced in one or both of a first and a second amino acid or nucleic acid sequence for optimal alignment and non-homologous sequences can be disregarded for comparison purposes). In a preferred embodiment, the length of a reference sequence aligned for comparison purposes is at least 30%, preferably at least 40%, more preferably at least 50%, even more preferably at least 60%, and even more preferably at least 70%, 80%, or 90% of the length of the reference sequence. The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in the first sequence is occupied by the same amino acid residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position (as used herein amino acid or nucleic acid<br><br>
"identity" is equivalent to amino acid or nucleic acid "homology")- The percent identity between the two sequences is a function of the number of identical positions shared by the sequences, taking into account the number of gaps, and the length of each gap, which need to be introduced for optimal alignment of the two sequences<br>
The comparison of sequences and determination of percent identity between two sequences can be accomplished using a mathematical algorithm. In a preferred embodiment, the percent identity between two amino acid sequences is determuieci using the Needleman and Wunsch {J. Mol Biol (48):444-453 (1970)) algorithm which has been incorporated into the GAP program in the GCG software package (avaliable at http://www.gcg.com), using either a Blossom 62 matrix or a PAM250 matrix, and a gap weight of 16, 14, 12, 10, 8, 6, or 4 and alength weight of 1, 2, 3, 4, 5, or 6   In yct another preferred embodiment, the percent identity between two nucleotide sequences is determined using the GAP program in the GCG software package (available at http://www.gcg.com), using a NWSgapdna.CMP matrix and a gap weight of 4(1 50. 60, 70, or 80 and a length weight of 1, 2, 3, 4, 5, or 6. In another embodiment, the percent identity between two amino acid or nucleotide sequences is determined using the algorithm of E. Meyers and W. Miller (CABIOS, 4:11-17 (1989)) which has been incorporated into the ALIGN program (version 2.0), using a PAM120 weight residue table, a gap length penalty of 12 and a gap penalty of 4.<br>
The polynucleotide and amino acid sequences of the present invention can further be used as a "query sequence" to perform a search against public databases to. for example, identify other family members or related sequences, e.g., promoter sequences. Such searches can be performed using the NBLAST and XBLASI programs (version 2.0) of Altschul, et al (1990) J. Mol Biol 215:403-10. BLAST nueieotide searches can be performed with the NBLAST program, score = 100, wordlength = 12 to obtain nucleotide sequences homologous to polynucleotide molecules of the invention. BLAST protein searches can be performed with the XBLAST program, score = 50, wordlength = 3 to obtain amino acid sequences homologous to polypeptide moleeules of the invention. To obtain gapped alignments for comparison purposes, Gapped EM.AS 1 can be utilized as described in Altschul et al, (1997) Nucleic Acids Res. 25( 17);3389-3402, When utilizing BLAST and Gapped BLAST programs, the default parameters of<br><br>
the respective programs (e.g., XBLAST and NBLAST) can be used. See http://www.ncbi.nlm,nih.gov.<br>
//.       Recombinant Hosts<br>
The present invention relates to recombinant host cells that are suitable for use in the production of ethanol. In one embodiment, the cell comprises a heterologous-polynucleotide segment encoding a polypeptide under the transcriptional control of a surrogate promoter. The heterologous polynucleotide and surrogate promoter may be plasmid-based or integrated into the genome of the organism (as described in the examples). In a preferred embodiment, the host cell is used as a source of a desired polypeptide for use in the bioconversion of a complex sugar to ethanol, or a step thereof.<br>
In a preferred embodiment, the heterologous polynucleotide segment encodes a polysaccharase polypeptide which is expressed at higher levels than are naturally occurring in the host. The polysaccharase may be a p-glucosidase, a glucanase. either an endoglucanase or a exoglucanase, cellobiohydrolase, endo-l,4-p-xylanase. β-xylosidase, a-glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase, (i-amylase, glucoamylase, pullulanase, p-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination thereof<br>
In one embodiment, the polysaccharase is derived from E. chrysanthemi and is the glucanase (EGZ) polypeptide encoded by the celZ gene. However, other polysaccharases from E. chrysanthemi may be used including, e.g., the glucohydroiases encoded by ce/Y(Guiseppi et al, (1991) Gene 106:109-114) or bgxA (Vroeman rt al. (1995) Mol. Gen. Genet. 246:465-477). The celY gene product (EGY) is an endoglucanase. The bgxA gene encodes p-glucosidase and P-xylosidase acti\ ities (Vroeman et al, (1995) Mol Gen. Genet. 246:465-477). Preferably, an increase in polysaccharase activity of at least 10%, more preferably 20%, 30%, 40%, or 50% is observed. Most preferably, an increase in polysaccharase activity of several fold is obtained, e.g., 200%, 300%, 400%, 500%, 600%, 700%, or 800%.<br>
Alternatively, a desired polysaccharase may be encoded by a polynucleotide segment from another species, e.g., a yeast, an insect, an animal, or a plant. Any one or more of these genes may be introduced and expressed in the host cell of the invention in<br><br>
order to give rise to elevated levels of a polysaccharase suitable for depolymcrizing a complex sugar substrate. The techniques for introducing and expressing one of these genes in a recombinant host, are presented in the examples.<br>
In another embodiment of the invention, the host cell has been engineered \to express a secretory protein/s to facilitate the export of a desired polypeptide from the cell. In one embodiment, the secretory protein or proteins are derived from a drain negative bacterial cell, e.g., a cell from the family Enterobacteriaceae. In another embodiment, the secretory protein/s are from Erwinia and are encoded by the out genes. In another embodiment, the secretory proteins are ihtpul genes derived from Klebsiella. The introduction of one or more of these secretory proteins is especially desirable if the host cell is an enteric bacterium, e.g., a Gram-negative bacterium having a cell wall Representative Gram-negative host cells of the invention are from the family Enterobacteriaceae and include, e.g., Escherichia and Klebsiella. In one embodiment. the introduction of one or more secretory proteins into the host results in an increase in the secretion of the selected protein, e.g., a polysaccharase, as compared to naturally occurring levels of secretion. Preferably, the increase in secretion is at least 10% and more preferably, 100%, 200%, 300%, 400%, 500%, 600%, 700%, 800%, 900%,, 1000%, or more, as compared to naturally-occurring levels of secretion. In a preferred embodiment, the addition of secretion genes allows for the polysaccharase polypeptide to be produced at higher levels. In a preferred embodiment, the addition of secretion genes allows for the polysaccharase polypeptide to be produced with higher enz\ matic activity. In a most preferred embodiment, the polysaccharase is produced at higher levels and with higher enzymatic activity. Preferably, an increase in polysaccharase activity of at least 10%, more preferably 20%, 30%, 40%, 50%, 60%, 70%, 80%, 90%, or 100% is observed. Most preferably, an increase in polysaccharase activity of se\ eral fold is obtained, e.g, 200%, 300%, 400%, 500%, 600%, 700%, 800%, 900%. or 1000%, as compared to cells without secretion genes (e.g., cells that either lack or do not express secretion genes at a sufficient level). The techniques and methods for introducing such genes and measuring increased output of a desired polypeptide such as, e.g., a. polysaccharase, are described in further detail in the examples. Other equivalent methods are known to those skilled in the art.<br><br>
In preferred embodiments, the invention makes use of a recombinant host that is also ethanologenic. In one embodiment, the recombinant host is a Gram-negative bacterium. In another embodiment, the recombinant host is from the family Enterobacteriaceae. The ethanologenic hosts of U.S.P.N. 5,821,093, hereby incorporated by reference, for example, are suitable hosts and include, in partcular.E coil strains K04 (ATCC 55123), KOll (ATCC 55124), and K012 (ATCC 55125. and Klebsiella oxytoca strain M5A1. Alternatively, a non-ethanologenic host of ihe present invention may be converted into an ethanologenic host (such as the above-mentioned strains) by introducing, for example, ethanologenic genes from an efficient ethanol producer like Zymomonas mobilis. This type of genetic engineering, using standard techniques, results in a recombinant host capable of efficiently fermenting sugar into<br>
ethanol. In addition, the LYOl ethanol tolerant strain (ATCC	) may be employed<br>
as described in published PCT international application WO 98/45425 and this published application is hereby incorporated by reference (see also, e.g., Yomano et el. (1998) J. of Ind Micro. &amp; Bio. 20:132-138).<br>
In another preferred embodiment, the invention makes use of a non-ethanologenic recombinant host, e.g., E. coli strain B, E. coli strain DH5a, or Klebsiella oxytoca strain M5A1. These strains may be used to express a desired polypeptide, eg., a polysaccharase using techniques describe herein. In addition, these recombinant host may be used in conjunction with another recombinant host that expresses, yet another desirable polypeptide, e.g., a different polysaccharase. In addition, the non-ethanologenic host cell may be used in conjunction with an ethanologenic host eell   For example, the use of a non-ethanologenic host/s for carrying out, e.g., the depolymerization of a complex sugar may be followed by the use of an ethanologenie host for fermenting the depolymerized sugar. Accordingly, it will be appreciated that these reactions may be carried out serially or contemporaneously using, e.g.. homogeneous or mixed cultures of non-ethanologenic and ethanologenic recombinant<br>
hosts.<br>
In a preferred embodiment, one or more genes necessary for fermenting a sugar substrate into ethanol are provided on a plasmid or integrated into the host chromosome. More preferably, essential genes for fermenting a sugar substrate into ethanol, e.g., pyruvate decarboxylase {e.g.,pdc) and/or alcohol dehydrogenase (e.g., adh) are<br><br>
introduced into the host of the invention using an artificial operon such as  operon as described in U.S.P.N. 5.82L093, hereby incorporated by reference  will be appreciated that the present invention, in combination with what is know n in the art, provides techniques and vectors for introducing multiple genes into a  (see, e.g., Current Protocols in Molecular Biology, eds. Ausubel et ai, John  Sons (1992), Sambrook, J. et aL, Molecular Cloning: A Laboratory Manual.  Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989), and Bergey's Manual of Determinative Bacteriology. Krcig atal. Williams and Wilkins (1984), hereby incorporated by reference). Accordingly using the methods of the invention, a single genetic construct could encode all of the necessary gene products {e.g., a glucanase, an endoglucanase. an exoglucanase, a secretory protein/s, pyruvate decarboxylase, alcohol dehydrogenase) for performing  saccharification and fermentation (SSF). In addition, it will also be appreciated that such a host may be further manipulated, using methods known in the art. to hane mutations in any endogenous gene/s {e.g., recombinase genes) that would interfere with the stability, expression, and function of the introduced genes. Further, it \AWILL also be appreciated that the invention is intended to encompass any regulatory elements, gene/s, or gene products, i.e., polypeptides, that are sufficiently homologous to the ones described herein.<br>
Methods for screening strains having the introduced genes are routine and may be facilitated by visual screens that can identify cells expressing either the alcohol dehydrogenase (ADH) or glucanase (EGZ) gene product. The ADH gene product produces acetaldehyde that reacts with the leucosulfonic acid derivative of  to produce an intensely red product. Thus, ADH-positive clones can be easily screened and identified as bleeding red colonies. Methods for screening for  polysaccharase activity, also results in a clear visual phenotype as described below and in the examples.<br>
Recombinant bacteria expressing, for example, the PET operon typically grow to higher cell densities in liquid culture than the unmodified parent organisms due to the production of neutral rather than acidic fermentation products (Ingram et al., (1988) Appl Environ, Microbiol 54:397-404). On plates, ethanologenic clones are readily apparent as large, raised colonies which appear much like yeast. These traits have been<br><br>
very useful during the construction of new strains and can provide a prelimmary indication of the utility of new constructs. Rapid evaluations of ethanol producing potential can also be made by testing the speed of red spot development on aldchyde indicator plates (Conway et al, (1987) J. Bacteriol 169:2591-2597). Typically, strains which prove to be efficient in sugar conversion to ethanol can be recognized by the production of red spots on aldehyde indicator plates within minutes of transfer<br>
In a most preferred embodiment of the invention, a single host cell is ethanologenic, that is, has all the necessary genes, either naturally occurring or artificially introduced or enhanced (e.g., using a surrogate promoter and/or genes from a different species or strain), such that the host cell has the ability to produce and sccrete a polysaccharase/s, degrade a complex sugar, and ferment the degraded sugar into ethanol. Accordingly, such a host is suitable for simultaneous saccharification and fermentation.<br>
Moreover, the present invention takes into account that the native E coli fermentation pathways produce a mixture of acidic and neutral products (in order of abundance): lactic acid, hydrogen + carbon dioxide (from formate), acetic acid, ethanol, and succinate. However, the Z mobilis PDC (pyruvate decarboxylase) has a lower Km for pyruvate than any of the competing E. coli enzymes. By expressing higli actiMties of PDC, carbon flow is effectively redirected from lactic acid and acetyl-CoA into acetylaldehyde and ethanol. Small amounts of phosphoenolpyruvate can be eliminated by deleting the fumarate reductase gene {frd) (Ingram et al, (1991) U.S.P.N, 5,000.000; Ohta et a., {\99\)Appl Environ. Microbiol 57:893-900). Additional mutations (e.g. in the pfl or Idh genes) may be made to completely eliminate other competing pathvva\ s (Ingram et ah, (1991) U.S.P.N, 5,000,000). Additional mutations to remove enzymes (e.g., recombinases, such as recA) that may compromise the stability of the introduced genes (either plasmid-based or integrated into the genome) may also be introduced, selected for, or chosen from a particular background.<br>
In addition, it should be readily apparent to one skilled in the art that the ability conferred by the present invention, to transform genes coding for a protein or an entire metabolic pathway into a single manipulable construct, is extremely useful. Envisioned in this regard, for example, is the application of the present invention to a variety of situations where genes from different genetic loci are placed on a chromosome. This<br><br>
may be a multi-cistronic cassette under the control of a single promoter or separate promoters may be used.<br>
Exemplary E. coli strains that are ethanologenic and suitable for further improvement according to the methods of the invention include, for example, K04, KOI 1, and K012 strains, as well as the LYOl strain, an ethanol-tolerant mutant of the E. coli strain KOI 1. Ideally, these strains may be derived from the E. coli strain ATCC 11303, which is hardy to environmental stresses and can be engineered to be ethanologenic and secrete a polysaccharase/s. In addition, recent PCR investigaions have confirmed that the ATCC 11303 strain lacks all genes knovm to be associated with the pathogenicity of £. coli (Kuhnert et al, (1997) Appl. Environ, Microhiol 63:703-709).<br>
Another preferred ethanologenic host for improvement according lo the methods of the invention is the E. coli KOI 1 strain which is capable of fermenting heniicellulose hydrolysates from many different lignocellulosic materials and other substrates (Asghari et al, (1996) J, Ind. Microbiol 16:42-47; Barbosa et al, (1992) Current Microbjol 28:279-282; Beall et al, (1991) Biotechnol Bioeng, 38:296-303; Beall et el,(1992) Biotechnol Lett. 14:857-862; Hahn-Hagerdal et al, {1994) Appl. Microbiol. Biotchnol 41:62-72; Moniruzzaman et al, (1996) Biotechnol Lett. 18:955-990; Moniruzzaman et al, (1998) Biotechnol Lett. 20:943-947; Grohmann et al, (1994) Biotechnol Lett 16:281-286; Guimaraes et al, (1992) Biotechnol Bioeng, 40:41-45; Guimaraes et al.. (1992) Biotechnol Lett. 14:415-420; Moniruzzaman et al, (1997) J. Bacterial 179:1880-1886). In Figure 1, the kinetics of bioconversion for this strain are shown   hi particular, this strain is able to rapidly ferment a hemicellulose hydrolysate from rice hulls (which contained 58.5 g/L of pentose sugars and 37 g/L of hexose sugars) mio ethanol (Moniruzzaman et al, (1998) Biotechnol Lett. 20:943-947). It was noted that this strain was capable of fermenting a hemicellulose hydrolysate to completion w ithin 48 to 72 hours, and under ideal conditions, within 24 hours.<br>
Another preferred host cell of the invention is the bacterium Klebsiella. In particular, Klebsiella oxytoca is preferred because, like E. coli, this enteric bacterium has the native ability to metabolize monomeric sugars, which are the constituents of more complex sugars. Moreover, K. oxytoca has the added advantage of being able to transport and metabolize cellobiose and cellotriose, the soluble intermediates from the<br><br>
enzymatic hydrolysis of cellulose (Lai et al, (1996) Appl Environ. Microbial 63 355-363; Moniruzzaman et al, {\991)Appl Environ. Microbiol 63:4633-4637; Wood et al (\992) Appl, Environ. Microbiol. 58:2103-2110). The invention provides eenetically engineered ethanologenic derivatives of K:. oxytoca, e.g.. strain M5A1 having the Z mobilispdc and adhB genes encoded within the PET operon (as described herein and in U.S.P.N. 5,821,093; Wood etal, (1992) Appl. Environ. Microbiol. 58:2103-:i lO)<br>
Accordingly, the resulting organism, strain P2, produces ethanol efficienily liorn monomer sugars and from a variety of saccharides including raffinose, stachyosc. sucrose, cellobiose, cellotriose, xylobiose, xylotriose, maltose, etc. (Burchhaidt cf ui , (1992) Appl. Environ. Microbiol. 58:1128-1133; Moniruzzaman et al, (1997) Appi Environ. Microbiol 63:4633-4637; Moniruzzaman et al, (1997) J. Bacterial 179:1880-1886; Wood et al. (1992) Appl Environ. Microbiol 58:2103-2110). These strains may be further modified according to the methods of the invention to express and secrete a polysaccharase. Accordingly, this strain is suitable for use in the bioconversion of a complex saccharide in an SSF process (Doran et al, (1993) Biotechnol Progress 9:533-538; Doran et al, (1994) Biotechnol Bioeng. 44:240-247; Wood et al  (1 992) Appl Environ. Microbiol 58:2103-2110). In particular, the use of this ethanologenic P2 strain eliminates the need to add supplemental cellobiase, and this is one of the least stable components of commercial fungal cellulases (Grohmann, (1994) Biotechnol. Lett. 16:281-286),<br>
Screen for Promoters Suitable for Use in Heterologous Gene Expression<br>
While in one embodiment, the surrogate promoter of the invention is used to improve the expression of a heterologous gene, e.g., a polysaccharase, it will be appreciated that the invention also allows for the screening of surrogate promoters suitable for enhancing the expression of any desirable gene product. In general, the screening method makes use of the cloning vector described in Example 1 and depicted in Figure 3 that allows for candidate promoter fragments to be conveniently ligated and operably-linked to a reporter gene. In one embodiment, the celZ gene encoding glucanase serves as a convenient reporter gene because a strong colorimetric change results from the expression of this enzyme (glucanase) when cells bearing the plasmid are grown on a particular media (CMC plates). Accordingly, candidate promoters, e.g.,<br><br>
a particular promoter sequence or, alternatively, random sequences that can be 'shotgun" cloned and operably linked to the vector, can be introduced into a host cell and resultant colonies are scanned, visually, for having increased gene expression as evidenced by a phenotypic glucanase-mediated colorimetric change on a CMC plate. Colonies having the desired phenotype are then processed to yield the transforming DNA and the promoter is sequenced using appropriate primers (see Example 1 for more details).<br>
The high correspondence between the glucanase-mediated colorimetric change on a CMC plate and expression levels of the enzyme is an excellent indication of the strength of a candidate promoter (Fig. 4). Hence, the methods of invention provide a rapid visual test for rating the strength of candidate surrogate promoters. Accordingly, depending on the desired expression level needed for a specific gene product, a particular identified surrogate promoter can be selected using this assay. For example, if simply the highest expression level is desired, then the candidate promoter that produces the largest colorimetric change may be selected. If a lower level of expression is desired, for example, because the intended product to be expressed is toxic at high levels or must be expressed at equivalent levels with another product, a weaker surrogate promoter can be identified, selected, and used as described.<br>
///.      Methods of Use<br>
Degrading or Depolymerizing a Complex Saccharide<br>
In one embodiment, the host cell of the invention is used to degrade or depolymerize a complex sugar e.g., lignocellulose or an oligosaccharide into a smaller sugar moiety. To accomplish this, the host cell of the invention preferably expresses one or more polysaccharases, e.g., a glucanase, and these polysaccharases may be liberated naturally from the producer organism. Alternatively, the polysaccharase is liberated from the producer cell by physically disrupting the cell. Various methods for mechanically (e.g., shearing, sonication), enzymatically (e.g., lysozyme), or chemically disrupting cells, are known in the art, and any of these methods may be employed. Once the desired polypeptide is liberated from the inner cell space it may be used to degrade a complex saccharide substrate into smaller sugar moieties for subsequent bioconversion into ethanol. The liberated cellulase may be purified using standard biochemical<br><br>
techniques known in the art. Altematively, the liberated polysaccharide need not b
In another embodiment, a host cell is employed that coexpresses a polysaccharase and a secretory protein/s such that the polysaccharase is secreted into the growth medium. This eliminates the above-mentioned step of having to liberate the polysaccharase from the host cell. When employing this type of host, the host may be used directly in an aqueous solution containing a complex saccharide.<br>
In another embodiment, a host cell of the invention is designed to express more than one polysaccharase or is mixed with another host expressing a different polysaccharase. For example, one host cell could express a heterologous p-glucosidase while another host cell could express an endoglucanase and yet another host cell could express an exoglucanase, and these cells could be combined to form a heterogeneous culture having multiple polysaccharase activities. Alternatively, in a preferred embodiment, a single host strain is engineered to produce all of the above polysaccharases. In either case, a culture of recombinant host/s is produced having high expression of the desired polysaccharases for application to a sugar substrate. If desired, this mixture can be combined with an additional cellulase, e.g., an exogenous cellulase, such as a fungal cellulase. This mixture is then used to degrade a complex substrate. Alternatively, prior to the addition of the complex sugar substrate, the polysaccharase/s are purified from the cells and/or media using standard biochemical techniques and used as a pure enzyme source for depolymerizing a sugar substrate.<br>
Finally, it will be appreciated by the skilled artisan, that the ethanol-producing bacterial strains of the invention are superior hosts for the production of recombinant proteins because, under anaerobic conditions (e.g., in the absence of oxygen), there is less opportunity for improper folding of the protein {e.g., due to inappropriate disulfide bond formation). Thus, the hosts and culture conditions of the invention potentially result in the greater recovery of a biologically active product.<br>
Fermenting a Complex Saccharide<br>
In a preferred embodiment of the present invention, the host cell having the above mentioned attributes is also ethanologenic. Accordingly, such a host cell can be<br><br>
applied in degrading or depolymerizing a complex saccharide into a monosaccharide. Subsequently, the cell can catabolize the simpler sugar into ethanol by fermentation. This process of concurrent complex saccharide depolymerization into smaller sugar residues followed by fermentation is referred to as simultaneous saccharification and fermentation.<br>
Typically, fermentation conditions are selected that provide an optimal pH and temperature for promoting the best growth kinetics of the producer host cell strain and catalytic conditions for the enzymes produced by the culture (Doran et al, (1993) Biotechnol Progress. 9:533-538). For example, for Klebsiella, e.g., the P2 strain, optimal conditions were determined to be between 35-37° C and pH 5.0- pH 5.4. Under these conditions, even exogenously added fungal endoglucanases and exoglucanases are quite stable and continue to function for long periods of time. Other conditions are discussed in the Examples. Moreover, it will be appreciated by the skilled artisan, that only routine experimentation is needed, using techniques known in the art, for optimizing a given fermentation reaction of the invention.<br>
Currently, the conversion of a complex saccharide such as lignocellulose, is a very involved, multi-step process. For example, the lignocellulose must first be degraded or depolymerized using acid hydrolysis. This is then followed by steps that separate liquids from solids and these products are subsequently washed and detoxified to result in cellulose and hemicellulose that can be further depolymerized (using added cellulases) and finally, fermented by a suitable ethanologenic host cell. In contrast, the fermenting of com is much simpler in that amylases can be used to break down the com starch for immediate bioconversion by an ethanologenic host in essentially a one-step process. Accordingly, it will be appreciated by the skilled artisan that the recombinant hosts and methods of the invention afford the use of a similarly simpler and more efficient process for fermenting lignocellulose. For example, the method of the invention is intended to encompass a method that avoids acid hydrolysis altogether. Moreover, the hosts of the invention have the following advantages, 1) efficiency of pentose and hexose co-fermentation; 2) resistance to toxins; 3) production of enzymes for complex saccharide depolymerization; and 4) environmental hardiness. Accordingly, the complexity of depolymerizing lignocellulose can be simplified using an improved biocatalyst of the invention. Indeed, in one preferred embodiment of the<br><br>
invention, the reaction can be conducted in a single reaction vessel and in the absence of acid hydrolysis, e.g., as an SSF process.<br>
Potential Substrates for Bioconversion into Ethanol<br>
One advantage of the invention is the ability to use a saccharide source that has been, heretofore, underutilized.<br>
A number of complex saccharide substrates may be used as a starting source for depolymerization and subsequent fermentation using the host cells and methods of the invention. Ideally, a recyclable resource may be used in the SSF process. Mixed waste office paper is a preferred substrate (Brooks et al, (1995) Biotechnol Progress. 11:619-625; Ingram et al, (1995) U.S.P.N. 5,424.202), and is much more readily digested than acid pretreated bagasse (Doran et al, (1994) Biotech. Bioeng. 44:240-247) or highly purified crystalline cellulose (Doran et al (1993) Biotechnol Progress, 9:533-538). Since glucanases, both endoglucanases and exoglucanases, contain a cellulose binding domain, and these enzymes can be readily recycled for subsequent fermentations by harvesting the undigested cellulose residue using centrifugation (Brooks et al., (1995) Biotechnol. Progress. 11:619-625). By adding this residue with bound enzyme as a starter, ethanol yields (per unit substrate) were increased to over 80% of the theoretical yield with a concurrent 60% reduction in fungal enzyme usage (Figure 2). Such approaches work well with purified cellulose, although the number of recycling steps may be limited with substrates with a higher lignin content. Other substrate sources that are within the scope of the invention include any type of processed or unprocessed plant material, e.g., lawn clippings, husks, cobs, stems, leaves, fibers, pulp, hemp, sawdust, newspapers, etc.<br>
This invention is further illustrated by the following examples which should not be construed as limiting.<br>
EXAMPLE 1<br>
Methods for Making Recombinant Escherichia Hosts Suitable for Fermenting<br>
	Oligosaccharides into Ethanol<br>
In this example, methods for developing and using Escherichia hosts suitable for fermenting oligosaccharides into ethanol are described. In particular, a strong promoter<br><br>
is identified which can be used to increase the expression of a polysaccharase {e.g., glucanase). In addition, genes from Erwinia chrysanthemi are employed to facilitate polysaccharase secretion thereby eliminating the need for cell disruption in order to release the desired polysaccharase activity.<br>
Throughout this example, the following materials and methods are used unless otherwise stated.<br>
Materials and Methods<br>
Organisms and Culture Conditions<br>
The bacterial strains and plasmids used in this example are listed in Table 1, below.<br>
For plasmid constructions, the host cell E. coli DH5a was used. The particular gene employed encoding a polysaccharase (e.g., glucanase) was the celZ gene derived from Erwinia chrysanthemi P86021 (Beall, (1995) Ph.D. Dissertation, University of Florida; Wood et al, (1997) Biotech Bioeng. 55:547-555). The particular genes used for improving secretion were the out genes derived from E. chrysanthemi EC 16 (He et al, (1991) Proc. Natl Acad Sci. USA. 88:1079-1083).<br>
Typically, host cell cultures were grown in Luria-Bertani broth (LB) (10 g L'* Difco® tryptone, 5 g L-1 Difco® yeast extract, 5 g L-1 sodium chloride) or on Luria agar (LB supplemented with 15 g L-1 of agar). For screening host cells having glucanase celZ activity (EGZ), CMC-plates (Luria agar plates containing carboxymethyl cellulose (3 g L-1)) were used (Wood et al, (1988) Methods in Enzymology 160:87-112). When appropriate, the antibiotics ampicillin (50 mg L-1), spectinomycin (100 g L-1), kanamycin (50 g L-1) were added to the media for selection of recombinant or integrant host cells containing resistance markers. Constructs containing plasmids with a temperature conditional pSClOl replicon (Posfai et al, (1997) J. Bacteriol 179:4426-4428) were grown at 30°C and, unless stated otherwise, constructs with pUC-based plasmids were grown at 37°C.<br><br><br><br><br>
Genetic Methods<br>
Standard techniques were used for all plasmid constructions (Ausubel et al, (1987) Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc.; Sambrook et al, (1989) Molecular cloning: a laboratory manual 2nd ed. C.S.H.L., Cold Spring Harbor, N, Y).   For conducting small-scale plasmid isolation, the TELT procedure was performed. For large-scale plasmid isolation, the Promega® Wizard Kit was used. For isolating DNA fragments from gels, the Qiaquick® Gel Extraction Kit from Qiagen* was employed. To isolate chromosomal DNA from £ coli and Z mobilis the methods of Cutting and Yomano were used (Cutting et al, (1990). Genetic analysis, pp. 61-74, In, Molecular biological methods for Bacillus. John Wiley &amp; Sons, Inc.; Yomano et al. (1993) J Bacteriol 175:3926-3933).<br>
To isolate the two glycolytic gene promoters {e.g.gap and eno) described herein, purified chromosomal DNA from E, coli DH5a was used as a template for the PCR (polymerase chain reaction) amplification of these nucleic acids using the following primer pairs: gap promoter, 5' -CGAATTCCTGCCGAAGTTTATTAGCCA-3 ' (SEQ ID NO: 3) and 5' -AAGGATCCTTCCACCAGCTATTTGTTAGTGA-3' (SEQ ID NO: 4); eno promoter, 5' -AGAATTCTGCCAGTTGGTTGACGATAG-3 ' (SEQ ID NO: 5)and5'-CAGGATCCCCTCAAGTCACTAGTTAAACTG-3' (SEQ ID NO: 6). The out genes encoding secretory proteins derived from E. chysanthemi (pCPP2006) were<br><br>
conjugated into E. coli using pRK2013 for mobilization (Figurski et ai, (1979) Proc. Natl. Acad ScL USA. 76: 1648-1652; Murata et al., (1990)7. Bactehol 172:2970-2978).<br>
To detennine the sequence of various DNAs of interest, the dideoxy sequencing method using fluorescent primers was performed on a LI-COR Model 4000-L DNA Sequencer. The pST76-K-based plasmids were sequenced in one direction using a T7 primer (5 ^-TAATACGACTCACTATAGGG-3'  (SEQ ID NO: 7)). The pUC18-and pUCI9-based plasmids were sequenced in two directions using either a forward primer (5 ' -CACGACGTTGTAAAACGAC-3 '   (SEQ ID NO: 8)) or a reverse primer (5 ' -TAACAATTTCACACAGGA-3' (SEQ ID NO: 9)). The extension reactions of the sequencing method were performed using a Perkin Elmer GeneAmp® PCR 9600 and SequiTherm Long-Read Sequencing Kit-LC . Resultant sequences were subsequently analyzed using the Wisconsin Genetic Computer Group (GCG) software package (Devereux et al, (1984) Nucleic Acids Rev. 12:387-395).<br>
To determine the start of transcriptional initiation in the above-mentioned promoters, primer extension analysis was performed using standard techniques. In particular, promoter regions were identified by mapping the transcriptional start sites using a primer finding correspondence within the celZ gene RNA that was isolated from cells in late exponential phase using a Qiagen RNeasy® kit. Briefly, cells were treated with lysozyme (400 µg/ml) in TE (Tris-HCk EDTA) containing 0.2 M sucrose and incubated at 25° C for 5 min prior to lysis. Liberated RNA was subjected to ethanol precipitation and subsequently dissolved in 20 |il of Promega® AMV reverse transcriptase buffer (50 mM Tris-HCl, pH 8.3, 50 mM KCl, 10 mM MgCb, 0.5 mM spermadine, 10 mM DTT). An IRD41-labeled primer (5' -<br>
GACTGGATGGTTATCCGAATAAGAGAGAGG-3 *   (SEQ ID NO: 10)) from Ll-Cor Inc. was then added and the sample was denatured at 80° C for 5 min, annealed at 55° C for 1 hr, and purified by alcohol precipitation. Annealed samples were dissolved in 19 µl of AMV reverse transcriptase buffer containing 500 µM dNTPs and 10 units AMV reverse transcriptase, and incubated for extension (1 h at 42°C). Products were treated with 0.5 µg/ml DNase-free RNase A, precipitated, dissolved in loading buffer, and compared to parallel dideoxy promoter sequences obtained using the LI-COR Model 4000-L DNA sequencer.<br><br>
Polysaccharase Activity<br>
To determine the amount of polysaccharase activity (e.g., glucanase activity) resulting from expression of the cell gene, a Congo Red procedure was used (Wood et al, (1988) Methods in Enzymology 160:87-112). In particular, selected clones were transferred to gridded CMC plates and incubated for 18 h at 30°C and then stained and recombinant host cells expressing glucanase formed yellow zones on a red background. Accordingly, the diameters of these colorimetric zones were recorded as a relative measure of celZ expression.<br>
Glucanase activity (EGZ) was also measured using carboxymethyl cellulose as a substrate. In this test, appropriate dilutions of cell-free culture broth (extracellular activity) or broth containing cells treated with ultrasound (total activity) were assayed at 35°C in 50 mM citrate buffer (pH 5.2) containing carboxymethyl cellulose (20 g L'*). Conditions for optimal enzyme release for 3-4 ml samples were determined to be 4 pulses at full power for 1 second each using a cell disrupter (Model W-220F, Heat System-Ultrasonics Inc., Plainview, NY). To stop the enzyme reactions of the assay, samples were heated in a boiling water bath for 10 min. To measure reducing sugars liberated enzymatically by the glucanase, a dinitrosalicylic acid reagent was employed using glucose as a standard (Wood et al, (1988) Methods in Enzymology 160:87-112). The amount of enzyme activity (lU) was expressed as jimols of reducing sugar released per min or as a percentage of total activity from an average of two or more determinations.<br>
UUrastructural Analysis<br>
To determine the ultrastructure of various recombinant host cells, fresh colonies from Luria agar plates were prepared for analysis by fixing in 2% glutaraldehyde in 0.2 M sodium cacodylate buffer (pH 7) followed by incubation in 1% osmium tetroxide and followed by 1% uranyl acetate in distilled water. Samples were dehydrated in ethanol, embedded in Spurr's plastic, and ultrathin sections were prepared and examined using a Zeiss® EM-IOC A electron microscope (Spur (1969) J. UltrastrucL Res. 26:31).<br><br>
Construction of a Low Copy Promoter Probe Vector Using celZ as the Reporter Gene<br>
To facilitate the isolation of strong promoters, a low copy vector was constructed with a pSClOl repiicon and a BamRl site immediately preceding a promoterless celZ gene (pL012171). Accordingly, this promoterless plasmid was used as a negative control. The plasmid pLOI1620 was used as a source of celZ and is a pUCl 8 derivative with expression from consecutive lac and ce/Z promoters. The BamUl site in this plasmid was eliminated by digestion and Klenow treatment (pL0I2164). The celZ gene was isolated as a promoterless Ndel fragment after Klenow treatment. The resulting blunt fragment was digested with Hindlll to remove downstream DNA and ligated into pUCl 9 (Hindlll to Hindi) to produce pL0I2170. In this plasmid, celZ is oriented opposite to the direction of/acZ transcription and was only weakly expressed. The BamHl (amino terminus)-Sp/2l (carboxyl terminus) fragment from pLOI2170 containing celZ was then cloned into the corresponding sites of pST76-K, a low copy vector with a temperature sensitive repiicon, to produce pLOI2171 (Fig. 3). Expression of celZ in this vector was extremely low facilitating its use as a probe for candidate strong promoters.<br>
Analysis of celZ Expression from Two E. coli Glycolytic Promoters (gap and eno)<br>
Two exemplary promoters driving glycolytic genes {gap and eno) in E. coli were examined for their ability to drive the expression of the heterologous celZ gene encoding glucanase. Chromosomal DNA from the E. coli DH5a strain was used as a template to amplify the gap and eno promoter regions by the polymerase chain reaction. The resulting fragments of approximately 400 bp each were digested with EcoRl and BamUl and cloned into the corresponding sites in front of a promoterless celZ gene in pL0I2171 to produce pLOI2174 (gap promoter) and pLOI2175 (eno promoter). As a control, the EcoKl-Sphl fragment from pLOI2164 containing the complete celZ gene and native E. chrysanthemi promoter was cloned into the corresponding sites of pST76-K to produce pL0I2173. These three plasmids were transformed into E, coli strains B and DH5a and glucanase activity (EGZ) was compared. For both strains of £. coli, glucanase activities were lower on CMC plates wdth E. coli glycolytic promoters than with pLOI2173 containing the native E. chrysanthemi promoter (Table 2). Assuming activity is related to the square of the radius of each zone (Fick's Law of diffusion), EGZ production with glycolytic promoters (pLOI2174 and pLOI2175) was<br><br>
esrimated to be 33% to 65% lower than in the original construct. Accordingly, other        ' candidate promoters for driving high levels of celZ gene expression were investigated.<br>
Identifying and Cloning Random DNA Fragments Suitable for Use as Promoters for Heterologous Gene Expression<br>
Random fragments derived from Z mobilis can be an effective source of surrogate promoters for the high level expression of heterologous genes in E. colL (Conway et al, (1987)7. Bacterial 169:2327-2335; Ingram et al, {\9%%)Appl. Environ. Micro. 54:397-404). Accordingly, to identify surrogate promoters for Erwinia celZ expression, Z mobilis chromosomal DNA was extensively digested with Sau3A\ and resulting fragments were ligated into pL0I2171 at the BamHI site and transformed into £ coli DH5a to generate a library' of potential candidate promoters. To rapidly identify superior candidate promoters capable of driving celZ gene expression in £ coli, the following biological screen was employed. Colonies transformed with celZ plasmids having different random candidate promoters were transferred to gridded CMC plates and stained for glucanase activity after incubation (Table 2). Approximately 20% of the 18,000 clones tested were CMC positive. The 75 clones which produced larger zones than the control, pLOI2173, were examined further using another strain, £ coli B.<br>
TABLE 2. Evaluation of promoter strength for celZ expression in £. coli using CMC indicator plates*<br><br><br><br>
 The number of clones which the indicated range of activities.<br>
 The average size of the diameters from three CMC digestion zones.<br>
 R'x is the square of the radius of the clear zone with the test plasm id: R2c is the square of the radius of the clear zone for the control (pLOI2173).<br>
Thus, promoter strength for selected candidate promoters was confirmed in two different strains with, in general, recombinants of DH5a producing larger zones (e.g., more glucanase) than recombinants of strain B. However, relative promoter strength in each host was similar for most clones. Based on these analyses of glucanase production as measured by zone size using CMC plates, four clones appeared to express celZ at approximately 6-fold higher levels than the construct with the original E. chrysanthemi celZ gene (pLOI2173). and at 10-fold higher levels than either of the E. coli glycolytic promoters. Accordingly, these and similarly strong candidate promoters were selected for further study.<br>
Production and Secretion of Glucanase<br>
Eight plasmid derivatives of pST76-K (pLOI2177 to pLOI2184) were selected from the above-described screen (see Group I and Group II (Table 2)) and assayed for total glucanase activity in £. coli strain B (Table 3). The four plasmids giving rise to the largest zones on CMC plates were also confirmed to have the highest glucanase activities (pLOI2177, pLOI2180, pLOI2182, and pLOI2183). The activities were approximately 6-fold higher than that of the unmodified celZ (pLOI2173), in excellent agreement with our estimate using the square of the radius of the cleared zone on CMC plates. Figure 4 shows a comparison of activity estimates from CMC plates and in vitro enzyme assays for strain B containing a variety of different promoters, with and without the addition of out genes encoding secretory proteins. Although there is some scatter, a<br><br>
direct relationship is clearly evident which validates the plate method for estimating relative activity. The original construct in pUCl 8, a high copy plasmid, was also included for comparison (pLOI2164). This construct with consecutive lac and celZ promoters produced less EGZ activity than three of the low copy plasmids with surrogate promoters (pLOI2177, pLOI2182, and pLOI2183). Thus, to increase celZ expression of glucanase even more, the DNA fragment containing celZ and the most effective surrogate promoter was isolated from pL0I2183 (as a EcoKl-Sphl fragment) and inserted into pUC19 with transcription oriented opposite to that of the lac promoter (pLOI2307). Accordingly, the above-identified strong surrogate promoter when incorporated into a high copy plasmid, further increased glucanase activity by 2-fold.<br>
Engineering Increased Secretion of Glucanase<br>
To further improve on the above-described results for increasing expression of celZ encoded glucanase, the above host cells were engineered for increased secretion. Genes encoding secretory proteins (e.g., the out genes) derived from E. chrysanlhemi EC 16 were used for improving the export of the glucanase using the plasmid as described in He et al that contains out genes (pCPP2006) (He et a/., (1991) Proc, Natl Acad. Set USA. 88:1079-1083). The increased secretion of EGZ in E. coll B was    investigated and results are presented in Table 3.<br><br><br>
Plasmids pL0I2164 and pLOI2307 are pUC-based plasmids (high copy number). All other plasmids are<br>
derivatives of pST76-K (low copy number).<br>
Glucanase activities were determined after 16 h of growth at 30°C.  Extracellular activity (secreted or released).<br>
Recombinant hosts with low copy plasmids produced only 7- 17% of the total EGZ extracellularly (after 16 h of growth) without the additional heterologous secretory proteins {out proteins encoded by plasmid pCPP2006), A larger fraction of EGZ (20-28%) was found in the extracellular broth surrounding host cells with the high-copy pUC-based plasmids than with the low copy pST76-based plasmids containing the same promoters. However, in either case^ the addition of out genes encoding secretory proteins (e.g., pCPP2006) increased the total level of expression by up to 2-fold and<br><br>
increased the fraction of extracellular enzyme (38-74%) by approximately 4-fold. The highest activity, 13,000 lU/L of total glucanase of which 7,800 lU/L was found in the cell-free supernatant was produced by strain B having both pLOI2307 encoding celZ driven by a strong surrogate promoter and pCPP2006 encoding out secretory proteins).<br>
It has been reported that under certain conditions (pH 7, 37° C), the specific activity for pure EGZ enzyme is 419 lU (Py et al, {\99\) Protein Engineering 4:325-333) and it has been determined that EGZ produced under these conditions is 25% more active than under the above-mentioned conditions (pH 5.2 citrate buffer, 35° C). Accordingly, assuming a specific activity of 316 lU for pure enzyme at pH 5.2 (35°C), the cultures of £ coli B (containing pLOI2307 and pCPP2006, e.g., plasmids encoding glucanase and secretory proteins), produced approximately 41 mg of active EGZ per liter or 4-6% of the total host cell protein was active glucanase.<br>
Sequence Analysis of the Strongest Promoter Derived from Z. mobilis<br>
The sequences of the four strongest surrogate promoters (pLOI2177, pLOI2I80, pL0I2182, and pLOI2183) were determined. To facilitate this process, each was fiised with pUC19 at the Pstl site. The resulting plasmids, pL0I2196, pLOI2197, pLOI2198, and pLOI2199, were produced at high copy numbers (ColEI replicon) and could be sequenced in both directions using Ml3 and T7 sequencing primers. All four plasmids contained identical pieces of Z mobilis DNA and were siblings. Each was 1417 bp in length and contained 4 internal Sau3 AI sites. DNA and translated protein sequences (six reading frames) of each piece were compared to the current data base. Only one fragment (281 bp internal fragment) exhibited a strong match in a Blast search (National Center for Biotechnology Information; http://www.ncbi.nlm.nih.gov/BLAST/) and this fragment was 99% identical in DNA sequence to part of the Z mobilis hpnB gene which is proposed to function in cell envelope biosynthesis (Reipen et al, (1995) Microbiology 141:155-161). Primer extension analysis revealed a single major start site, 67 bp upstream from the Sau3AI/BamHI junction site with celZ, and a second minor start site further upstream (Fig. 5). Sequences in the -10 and -35 regions were compared to the conserved sequences for E. coli sigma factors (Wang et al, (1989) 1 Bacteriol 180:5626-5631; Wise er el.,(1996)J. Bacteriol 178:2785-2793). The dominant<br><br>
promoter region (approximately 85% of total start site) appears similar to a sigma70<br>
38<br>
promoter while the secondary promoter site resembles a sigma   promoter.<br>
Microscopic Analysis of Recombinant Host Cells Producing Glucanase<br>
Little difference in cell morphology was observed between recombinants and the parental organism by light microscopy. Under the electron microscope, however, small polar inclusion bodies were clearly evident in the periplasm of strain B (pLOI2164) expressing high amounts of glucanase and these inclusion bodies were presumed to contain EGZ (Fig. 6). In the strain B (pLOI2307) that produced 2-fold higher glucanase activity the inclusion bodies were even larger and occupied up to 20% of the total cell volume. The large size of these polar bodies suggests that glucanase activity measurements may underestimate the total EGZ production. Typically, polar inclusion bodies were smaller in host cells also having constructs encoding the out secretory proteins which allow for increased secretion of proteins from the periplasmic space. As expected, no periplasmic inclusion bodies were evident in the negative control strain B (pUC19) which does not produce glucanase.<br>
EXAMPLE 2 Recombinant Klebsuella Hosts Suitable for Fermenting Oligosaccharides into<br>
Ethanol<br>
In this example, a recombinant Klebsiella host, suitable for use as a biocatalyst for depolymerizing and fermenting oligosaccharides into ethanol, is described.<br>
Materials and Methods Used in this Example<br>
Unless otherwise stated, the following materials and methods were used in the example that follows.<br>
Bacteria, Plasmids, and Culture Conditions<br>
The strains and plasmids that were used in this exemplification are summarized     in Table 4 below.<br><br><br><br><br><br><br><br>
The culture conditions used for cultivating £. coli and K oxytoca M5A] typically employed Luria-Bertani broth (LB) containing per liter: 10 g Difco® tryptone, 5 g yeast extract, and 5 g sodium chloride, or, alternatively, Luria agar (LB supplemented with 15 g of agar) (Sambrook et al, (1989), Molecular Cloning: A Laboratory Manual, C.S.H.L., Cold Spring Harbor, N.Y.).<br>
For screening bacterial colonies under selective conditions, CMC-plates (Luria agar plates containing 3 g L-1 carboxymethyl cellulose) were used to determine levels of glucanase activity expressed by a given bacterial strain (Wood et al (1988) Enzymology,, 160:87-112). For cultivating ethanologenic strains, glucose was added to solid media (20 g L-1) and broth (50 g L-1). In determining glucanase activity, the glucose in the growth media was replaced with sorbitol (50 g L-1), a non-reducing sugar. For cultivating various strains or cultures in preparation for introducing nucleic acids by electroporation, a modified SOC medium was used (e.g., 20 g L 1 Difco® tryptone, 5 g L"' , Difco® yeast extract, 10 mM NaCl, 2.5 mM KCl, 10 mM MgS04, 10 mM MgCb, and 50 g L-1 glucose). The antibiotics ampicillin (50 mg L-1), spectinomycin (100 mg L-1), kanamycin (50 mg L-1), tetracycline (6 or 12 mg L-1, and chloramphenicol (40, 200, or 600 mg L-1) were added when appropriate for selection of recombinant hosts bearing antibiotic resistance markers. Unless stated otherwise, cultures were grown at 37° C. Ethanologenic strains and strains containing plasmids with a temperature-sensitive pSC101 replicon were grown at 30° C.<br>
Genetic Methods<br>
For plasmid construction, cloning, and transformations, standard methods and E. coli DH5a hosts were used (Ausubel et al (1987) Current Protocols in Molecular Biology. John Wiley &amp; Sons, Inc.; Sambrook et al, (1989) Molecular Cloning: A Laboratory Manual, C.S.HX., Cold Spring Harbor, N.Y.). Construction of the celZ integration vector, pLOI2306, was performed as shown in Figure 7. A circular DNA fragment lacking a replicon from pLOI2306 (see Figure 7) was electroporated into the ethanologenic K, oxytoca P2 using a Bio-Rad Gene Pulser using the following conditions: 2.5 kV and 25 µF with a measured time constant of 3,8-4.0 msec (Comaduran et al (1998) Biotechnol Lett. 20:489-493). The E chrysanthemi EC 16 secretion system (pCPP2006) was conjugated into K oxytoca using pRK2013 for<br><br><br>
scale plasmid isolations were performed using the TELT procedure and a Promega Wizard Kit, respectively. DNA fragments were isolated from gels using a Qiaquick® Gel Extraction Kit from Qiagen® (Qiagen Inc., Chatsworth, CA). Chromosomal DNA from K. oxytoca M5A1 and Z mobilis CP4 were isolated as described by Cutting and Yomano (see Example 1). The DNAs of interest were sequenced using a LI-COR Model 4000-L DNA sequencer (Wood et al (1997) Biotech Bioeng. 55:547-555).<br>
Chromosomal Integration of celZ<br>
Two approaches were employed for chromosomal integration of ce/Z, using selection with a temperature-conditional plasmid (pLOI2183) using a procedure previously described for E. coli (Hamilton et al., (1989)J. Bacteriol 171:4617-4622) and direct integration of circular DNA fragments lacking a functional replicon. This same method was employed for chromosomal integration of Z mobilis genes encoding the ethanol pathway in E. coli B (Ohta K et al, {\99\)Appl Environ. Microbiol 57:893-900)and K. oxytoca M5A1 (Wood et al. (1992)Appl Environ. Microbiol 58:2103-2110). Typically, circular DNA was transformed into P2 by electroporation using a Bio-Rad Gene Pulser. Next, transformants were selected on solid medium containing tetracycline (6 mg L-1) and grown on CMC plates to determine levels of glucanase activity.<br>
Glucanase Activity<br>
Glucanase activity resulting from expression of celZ gene product (i.e., glucanase) under the control of different test promoters was evaluated by staining CMC plates as described in Example 1. This colorimetric assay results in yellow zones indicating glucanase activity and the diameter of the zone was used as a relative measure of celZ polypeptide expression. Clones that exhibited the largest zones of yellow color were further evaluated for glucanase activity at 35° C using carboxymethyl cellulose as the substrate (20 g L-1 dissolved in 50 mM citrate buffer, pH 5.2) (Wood et al (1988) Methods in Enzymology 160: 87-112). In order to measure the amount of intracellular glucanase, enzymatic activity was released from cultures by treatment with ultra-sound for 4 seconds (Model W-290F cell disruptor, Heat System-Ultrasonics Inc., Plainview,<br><br>
NY), The amount of glucanase activity expressed was measured and is presented here as ^imol of reducing sugar released per min (lU). Reducing sugar was measured as described by Wood (Wood et al (1988) Methods in Enzymology 160: 87-112) using a glucose standard.<br>
Substrate Depolymerization<br>
To further determine the amount of glucanase activity produced by various host cells, different carbohydrate substrates were incubated with various cell extracts (20 g L-1 suspended in 50 mM citrate buffer, pH 5.2). In one example, test substrates comprising acid-swollen and ball-milled cellulose were prepared as described by Wood (Wood et al (1988) Methods in Enzymology 160: 87-112). A typical polysaccharase extract {i.e., EGZ (glucanase) from K. oxytoca SZ6 (pCPP2006)) was prepared by cultivating the host cells at 30°C for 16 h in LB supplemented with sorbitol, a nonreducing sugar. Dilutions of cell-free broth were added to substrates and incubated at 35°C for 16 h.   Several drops of chloroform were added to prevent the growth of adventitious contaminants during incubation. Samples were removed before and after incubation to measure reducing sugars by the DNS method (see, Wood et al (1988) Methods in Enzymology 160: 87-112). The degree of polymerization (DP) was estimated by dividing the total calculated sugar residues present in the polymer by the     number of reducing ends.<br>
Fermentation Conditions<br>
Fermentations were carried out in 250 ml flasks containing 100 ml of Luria broth supplemented with 50 g L"' of carbohydrate. Test carbohydrates were sterilized<br>
\    separately and added after cooling. To minimize substrate changes, acid-swollen cellulose, ball-milled cellulose and xylan were not autoclaved. The antibiotic chloramphenicol (200 mg L-1) was added to prevent the growth of contaminating organisms. Flasks were inoculated (10% v/v) with 24-h broth cultures (50 g L-1 glucose) and incubated at 35°C with agitation (100 rpm) for 24-96 h. To monitor<br>
    cultures, samples were removed daily to determine the ethanol concentrations by gas chromatography (Dombek et al {1986Appl Environ. Microbiol 52:975-981).<br><br>
Methods for Isolating and Identifying a Surrogate Promoter<br>
In order to identify random fragments of Z mobilis that would serve as surrogate promoters for the expression of heterologous genes in Klebsiella and other host cells, a vector for the efficient cloning of candidate promoters was constructed as described in Example 1 (see also, Ingram et al {\9%%)Appl Environ, Microbiol 54:397-404).<br>
Next, Saw3AI digested Z mobilis DNA fragments were ligated into the BamYil site of pLOI2171 to generate a library of potential promoters. These plasmids were transformed into E. coli DH5a for initial screening. Of the 18,000 colonies individually tested on CMC plates, 75 clones produced larger yellow zones than the control (pLOI2173). Plasmids from these 75 clones were then transformed into K. oxytoca M5A1, retested, and found to express high levels of cell in this second host.<br>
Recombinant Klebsiella Hosts for Producing Polysaccharase<br>
The high expressing clones (pLOI2177 to pLOI2194) with the largest zones on CMC plates indicating celZ expression were grown in LB broth and assayed for glucanase activity (Table 5).<br><br><br><br>
■ pLOI2173 contains the celZ gene with the original promoter, all others contain the celZ gene with a Z mobilis DNA fragment which serves as a surrogate promoter.<br>
 Glucanase (CMCase) activities were detennined after 16 h of growth at 30°C.<br>
Activities with these plasmids were up to 8-fold higher than with the control plasmid containing a native celZ promoter (pL0I2173). The four plasmids which produced the largest zones (pLOI2177, pLOI2180, pLOI2182 and pLOI2183) also produced the highest total glucanase activities (approximately 20,000 lU L-1) released into the broth. One of these plasmids, pL0I2183, was selected for chromosomal integration.<br>
Chromosomal Integration of a Polysaccharase Gene<br>
To stably incorporate a desirable polysaccharase gene into a suitable host cell, e.g., Klebsiella P2 strain, a novel vector (pLOI2306) was constructed to facilitate the isolation of a DNA fragment which lacked all replication functions but contained the celZ gene with surrogate promoter, a selectable marker, and a homologous DNA fragment for integration (Figure 7). Two Asc\ sites were added to pUC19 by inserting a linker (GGCGCGCC; SEQ ID NO: 11) into Klenow-treated Ndel and Sapl sites which flank the polylinker region to produce pLOI2302. A blunt fragment containing the tet<br><br>
resistance marker gene from pBR322 (excised with EcoRl and Aval, followed by Klenow treatment) was cloned into the Psil site of pLOI2302 (cut with Pstl, followed by Klenow treatment) to produce pLOI2303. To this plasmid was ligated a blunt fragment of K, oxytoca M5A1 chromosomal DNA (cut with EcoRI and made blunt with Klenow treatment) into the Smal site of pLOI2303 to produce (pLO12305). The EcoRI - Sphl fragment (Klenow treated) containing the surrogate Z mobilis promoter and cell gene from pLOI2183 was ligated into the £coRI site of pLOI2305 (EcdBl, Klenow treatment) to produce pLOI2306. Digestion of pLOI2306 with Ascl produced two fragments, the larger of which contained the celZ gene with a surrogate promoter, tet gene, and chromosomal DNA fragment for homologous recombination. This larger fragment (10 kbp) was purified by agarose gel electrophoresis, circularized by self-ligation, and electroporated into the Klebsiella strain P2 and subsequently grown under selection for tetracycline resistance. The resulting 21 tetracycline-resistant colonies were purified and tested on CMC plates for glucanase activity. All were positive with large zones indicating functional expression of the celZ gene product.<br>
Clones used to produce the recombinant strains were tested for the presence of unwanted plasmids by transforming DH5a with plasmid DNA preparations and by gel electrophoresis. No transformants were obtained with 12 clones tested. However, two of these strains were subsequently found to contain large plasmid bands which may contain celZ and these were discarded. Both strains with large plasmids contained DNA which could be sequenced with T7 and Ml3 primers confirming the presence of multicopy plasmids. The remaining ten strains contain integrated celZ genes and could not be sequenced with either primer.<br>
The structural features of the novel vector pLOI2306 are schematically shown in Fig. 8 and the nucleotide sequence of the vector, including various coding regions {i.e., of the genes celZ, bla, and tet), are indicated in SEQ ID NO: 12 of the sequence listing. Nucleotide base pairs 3282-4281, which represent non-coding sequence downstream of the celZ gene (obtained from E. chrysanthemi), and base pairs 9476-11544 which represent a portion of the non-coding target sequence obtained from K. oxytoca M5A1, remain to be sequenced using standard techniques {e.g., as described in Sambrook, J. et ai, T. Molecular Cloning: A Laboratory Manual. 2nd, ed, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, (1989);<br><br>
Current Protocols in Molecular Biology, eds. Ausubel et al, John Wiley &amp; Sons (1992)). For example, sufficient flanking sequence on either side of the aforementioned unsequenced regions of the pLOI2306 plasmid is provided such that sequencing primers that correspond to these known sequences can be synthesized and used to carry out standard sequencing reactions using the pLOI2306 plasmid as a template.<br>
Alternatively, it will be understood by the skilled artisan that these unsequenced regions can also be determined even in the absence of the pLOI2306 plasmid for use as a template. For example, the remaining celZ sequence can be determined by using the sequence provided herein (e.g., nucleotides 1452-2735 of SEQ ID NO: 12) for synthesizing probes and primers for, respectively, isolating a celZ containing clone from a library comprising E. chrysanthemi sequences and sequencing the isolated clone using a standard DNA sequencing reaction. Similarly, the remaining target sequence can be determined by using the sequence provided herein (e.g., nucleotides 8426-9475 of SEQ ID NO: 12) for synthesizing probes and primers for, respectively, isolating a clone containing target sequence from a library comprising K, oxytoca M5A1 EcoRl fragments {e.g., of the appropriate size) and sequencing the isolated clone using a standard DNA sequencing reaction (a source of K. oxytoca M5A1 would be, e.g., ATCC 68564 cured free of any plasmid using standard techniques). The skilled artisan will further recognize that the making of libraries representative of the cDNA or genomic<br>
    sequences of a bacterium and the isolation of a desired nucleic acid fragment from such a library (e.g., a cDNA or genomic library), are well known in the art and are typically carried out using, e.g,, hybridization techniques or the polymerase chain reaction (PCR) and all of these techniques are standard in the art (see, e.g., Sambrook, J. et al, T. Molecular Cloning: A Laboratory Manual 2nd, ed, Cold Spring Harbor Laboratory,<br>
    Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, (1989); Current Protocols in Molecular Biology, eds. Ausubel et al, John Wiley &amp; Sons (1992); Oligonucleotide Synthesis (M.J. Gait, Ed. 1984); and PCR Handbook Current Protocols in Nucleic Acid Chemistry, Beaucage, Ed. John Wiley &amp; Sons (1999) (Editor)).<br><br>
Heterologous Gene Expression Using a Surrogate Promoter and Integrated or Plasmid'Based Constructs<br>
The ten integrated strains (SZI-SZIO) were investigated for glucanase production in LB sorbitol broth (Table 6). All produced 5,000-7,000 lUL-1 of active enzyme. Although this represents twice the activity expressed from plasmid pLOI2173 containing the native celZ promoter, the integrated strains produced only 1/3 the glucanase activity achieved by P2 (pLOI2183) containing the same surrogate Z mobilis promoter (Table 5). The reduction in glucanase expression upon integration may be attributed to a decrease in copy number {i.e., multiple copy plasmid versus a single integrated copy).<br>
Secretion of Glucanase EGZ<br>
K. oxytoca contains a native Type II secretion system for puUulanase secretion (Pugsley (1993) Microbiol Rev. 57:50-108), analogous to the secretion system encoded by the out genes in Erwinia chrysanthemi which secrete pectate lyases and glucanase (EGZ) (Ban-as et al. {\99A)Annu, Rev. Phytopathol 32:201-234; He et al (1991) Proc, Natl. Acad. Sci. USA. 88: 1079-1083). Type II secretion systems are typically very specific and function poorly with heterologous proteins (He et al. (1991) Proc. Natl. Acad Sci. USA. 88: 1079- 1083;Py et al (1991)FEMSMicrobiol Lett. 79:315022; Sauvonnet et al (1996) Mol Microbiol 22: 1-7). Thus as expected, recombinant celZ was expressed primarily as a cell associated product with either M5A1 (Table 5) or P2 (Table 6) as the host. About 1/4 (12-26%) of the total recombinant EGZ activity was recovered in the broth. With E. coll DH5a, about 8-12% of the total extracellular EGZ was present. Thus the native secretion system in K. oxytoca may facilitate partial secretion of recombinant EGZ.<br>
To further improve secretion of the desired products, type II secretion genes (out genes) from E. chrysanthemi EC16 were introduced {e.g., using pCPP2006) to facilitate secretion of the recombinant EGZ from strain P86021 in ethanologenic strains of K. oxytoca (Table 5 and Table 6). For most strains containing plasmids with celZ, addition of the out genes resulted in a 5-fold increase in extracellular EGZ and a 2-fold increase in total glucanase activity. For strains with integrated celZ, addition of the out genes resulted in a 10-fold increase in extracellular EGZ and a 4-fold increase in total<br><br>
glucanase activity. In both cases, the out genes facilitated secretion of approximately half the total glucanase activity. The increase in EGZ activity resulting from addition of the out genes may reflect improved folding of the secreted product in both plasmid and integrated celZ constructs. The smaller increase observed with the pUC-based derivatives may result from plasmid burden and competition for export machinery during the production of periplasmic |3-lactamase from the bla gene on this high copy plasmid.<br>
Two criteria were used to identify the best integrated strains of P2, growth on solid medium containing high levels of chloramphenicol (a marker for high level expression of the upstream pdc and adhB genes) and effective secretion of glucanase with the out genes. Two recombinant strains were selected for further study, SZ2 and SZ6. Both produced 24,000 lU L-1 of glucanase activity, equivalent to approximately 5% of the total cellular protein (Py et al (1991) Protein Engirt, 4:325-333).<br>
Substrate Depolymerization<br>
The substrate depolymerization of the recombinant EGZ was determined to be excellent when applied to a CMC source (Table 7). When applied to acid swollen cellulose, the activity of the glucanase was less than 10% of the activity measured for CMC activity. Little activity was noted when the polysaccharase was applied to Avicei or xylan. However, when allowed to digest overnight, the EGZ polysaccharase resulted in a measurable reduction in average polymer length for all substrates. CMC and acid-swollen cellulose were depolymerized to an average length of 7 sugar residues. These cellulose polymers of 7 residues are marginally soluble and, ideally, may be further digested for efficient metabolization (Wood et al (1992) Appl Environ. Microbiol 58:2103-2110). The average chain length of ball-milled cellulose and Avicei was reduced to 1/3 of the original length while less than a single cut was observed per xylan polymer.<br><br><br><br><br>
Fermentation<br>
To be useful, addition of celZ and out genes to strain P2 must not reduce the fermentative ability of the resulting biocatalyst. A comparison was made using glucose and cellobiose (Table 8). All strains were equivalent in their ability to ferment these sugars indicating a lack of detrimental effects from the integration of celZ or addition of pCPP2006. These strains were also examined for their ability to convert acid-swollen cellulose directly into ethanol. The most active construct SZ6 (pCPP2006) produced a small amount of ethanol (3.9 g L -1) from amorphous cellulose. Approximately 1.5 g L-1 ethanol was present initially at the time of inoculation for all strains. This decreased with time to zero for all strains except SZ6 (pCPP2006). Thus the production of 3.9 g L-1 ethanol observed with SZ6 (pCPP2006) may represent an underestimate of total ethanol production. However, at best, this represents conversion of only a fraction of the polymer present. It is likely that low levels of glucose, cellobiose, and cellotriose were produced by EGZ hydrolysis of acid swollen cellulose and fermented. These compounds can be metabolized by the native phosphoenolpyruvate-dependent phosphotransferase system in K. oxytoca (Ohta K et al, {\99\) Appl Environ. Microbiol 57:893-900; Wood et al. {1992)Appl Environ, Microbiol 58:2103-2110).<br><br><br>
Equivalents<br>
Those skilled in the art will recognize, or be able to ascertain using no more than routine experimentation, many equivalents to the specific embodiments of the invention described herein. Such equivalents are intended to be encompassed by the following claims. Moreover, any number of genetic constructs, host cells, and methods described in United States Patent Nos. 5,821,093; 5,482,846; 5,424,202; 5,028,539; 5,000,000; 5,487,989, 5,554,520, and 5,162,516, may be employed in carrying out the present invention and are hereby incorporated by reference.<br><br><br><br><br>
WE CLAIM:<br>
1.	A recombinant host microorganism comprising a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase polypeptide under the transcriptional control of a surrogate promoter, said promoter capable of causing increased expression of said polysaccharase polypeptide; and a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein expression of said first and second polynucleotide segments results in the increased production of a polysaccharase by the recombinant host microorganism.<br>
2.	The recombinant host microorganism as claimed in claim 1 wherein production of polysaccharase is selected from the group consisting of the activity of said polysaccharase polypeptide, the amount of said polysaccharase polypeptide produced by said recombinant microorganism, and a combination thereof<br>
3.	The recombinant host microorganism as claimed in claim 2 wherein said polysaccharase polypeptide is secreted.<br>
4.	The recombinant host microorganism as claimed in claim 2 wherein said microorganism is a bacterial cell.<br>
5.	The recombinant host microorganism as claimed in claim 4 wherein said microorganism is a Gram-negative bacterial cell.<br>
6.	The recombinant host microorganism as claimed in claim 5 wherein said microorganism is a facultatively anaerobic bacterial cell.<br><br>
7.	The recombinant host microorganism as claimed in claim 6 wherein said microorganism is selected from the family Enterobacteriaceae.<br>
8.	The recombinant host microorganism as claimed in claim 7 wherein said host is selected from the group consisting of Escherichia and Klebsiella,<br>
9.	The recombinant host microorganism as claimed in claim 8 wherein said Escherichia is selected from the group consisting of E. coli B, E. coli DHSa, E. coli K04 (ATCC 55123), E. coli KOll (ATCC 55124), E, coli K012 (ATCC 55125) and E, coli LYOl, K. oxytoca M5A1, and K, oxytoca P2 (ATCC<br>
55307).<br>
10.	The recombinant host microorganism as claimed in claim 2 wherein said polysaccharase is selected from the group consisting of glucanase, endoglucanase, exoglucanase, cellobiohydrolase, β-glucosidase, endo-l,4-P" xylanase, a-xylosidase, a-glucuronidase, a-L-arabinoftiranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase, glucoamylase, pullulanase, β-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination thereof.<br>
11.	The recombinant host microorganism as claimed in claim 10 wherein said polysaccharase is glucanase.<br>
12.	The recombinant host microorganism as claimed in claim 10, wherein said polysaccharase is an expression product of a celZ gene.<br>
13.	The recombinant host microorganism as claimed in claim 12 wherein said celZ gene is derived from Erwinia chrysanthemi.<br><br>
14.	The recombinant host microorganism as claimed in claim 2 wherein said second heterologous polynucleotide segment comprises at least one pul gene or out gene.<br>
15.	The recombinant host microorganism as claimed in claim 2 wherein said second heterologous polynucleotide segment is derived from a bacterial cell selected from the family Enterobacteriaceae.<br>
16.	The recombinant host microorganism as claimed in claim 15 wherein said bacterial cell is selected from the group consisting of K. oxytoca, E. carotovora, E. carotovora subspecies carotovora, E. carotovora subspecies atroseptica, and E. chrysanthemi<br>
17.	The recombinant host microorganism as claimed in claim 2 wherein said surrogate promoter comprises a polynucleotide fragment derived from Zymomonas mobilis.<br>
18.	The recombinant host microorganism as claimed in claim 17 wherein said surrogate promoter comprises a nucleic acid having the sequence provided in SEQ ID NO: 1, or a fragment thereof<br>
19.	The recombinant host microorganism as claimed in any one of claims 1 to 18 wherein said host cell is ethanologenic.<br>
20.	A recombinant ethanologenic microorganism comprising a heterologous polynucleotide segment encoding a polysaccharase under the transcriptional control of an exogenous surrogate promoter.<br><br>
21.	The recombinant host microorganism as claimed in claim 20 wherein said host cell is a bacterial cell.<br>
22.	The recombinant host microorganism as claimed in claim 21 wherein said host cell is a Gram-negative bacterial cell.<br>
23.	The recombinant host microorganism as claimed in claim 22 wherein said host cell is a facultatively anaerobic bacterial cell.<br>
24.	The recombinant host microorganism as claimed in claim 23 wherein said host cell is selected from the family Enterobacteriaceae.<br>
25.	The recombinant host microorganism as claimed in claim 24 wherein said host is selected from the group consisting of Escherichia and Klebsiella.<br>
26.	The recombinant host microorganism as claimed in claim 25 wherein said Escherichia and Klebsiella are selected from the group consisting of E. coli B, E, coli DH5a, E, coli K04 (ATCC 55123), E. coli KOll (ATCC 55124), E. coli K012 (ATCC 55125), E. coli LYOl, K oxytoca M5A1 and K oxytoca P2 (ATCC 55307).<br>
27.	The recombinant host microorganism as claimed in claim 20 wherein said polysaccharase is selected from the group consisting of glucanase, endoglucanase, exoglucanase, cellobiohydrolase, a-glucosidase, endo-1,4-a-xylanase, β-xylosidase, β-glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase, glucoamylase, pullulanase, β-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination thereof.<br><br>
28.	The recombinant host microorganism as claimed in claim 27 wherein said polysaccharase is glucanase.<br>
29.	The recombinant host microorganism as claimed in claim 28 wherein said polysaccharase is an expression product of a celZ gene.<br>
30.	The recombinant host microorganism as claimed in claim 29 wherein said celZ gene is derived from Erwinia chrysanthemi.<br>
31.	The recombinant host microorganism as claimed in claim 20 wherein said surrogate promoter comprises a polynucleotide fragment derived from Zymomonas mobilis,<br>
32.	The recombinant host microorganism as claimed in claim 31 wherein said surrogate promoter comprises a polynucleotide segment having the sequence provided in SEQ ID NO: 1, or a fragment thereof.<br>
33.	A recombinant ethanologenic Gram-negative bacterial microorganism comprising a first heterologous polynucleotide segment comprising a sequence encoding a first polypeptide; and a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein production of the first polypeptide by the host microorganism is increased.<br>
34.	The recombinant host microorganism as claimed in claim 33 wherein said first polypeptide is secreted.<br>
35.	The recombinant host microorganism as claimed in claim 33 wherein said host cell is a facultatively anaerobic bacterial cell.<br><br>
36.	The recombinant host microorganism as claimed in claim 35 wherein said host cell is selected from the family Enterobacteriaceae.<br>
37.	The recombinant bacterial microorganism as claimed in claim 36 wherein said host cell is selected from the group consisting of Escherichia and Klebsiella.<br>
38.	The recombinant bacterial microorganism as claimed in claim 37 wherein said Escherichia and Klebsiella are selected from die group consisting of E. coli B, E, coli DH5a, E. coli K04 (ATCC 55123), E. coli KOll (ATCC 55124), E. coli K012 (ATCC 55125) E. coli LYOl, K oxytoca M5A1, and K oxytoca P2 (ATCC 55307).<br>
39.	The recombinant bacterial microorganism as claimed in claim 33 wherein said first polypeptide is a polysaccharase.<br>
40.	The recombinant bacterial microorganism as claimed in claim 39 wherein said polysaccharase is of increased activity.<br>
41.	The recombinant host microorganism as claimed in claim 39 wherein said polysaccharase is selected from the group consisting of glucanase, endoglucanase, exoglucanase, cellobiohydrolase, a-glucosidase, endo-l,4-a-xylanase, β-xylosidase, β-glucuronidase, a-L-arabinoftiranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase, glucoamylase, pullulanase, β-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination thereof.<br>
42.	The recombinant host microorganism as claimed in claim 41 wherein said polysaccharase is glucanase.<br><br>
43.	The recombinant host microorganism as claimed in claim 42 wherein said glucanase is an expression product of a celZ gene.<br>
44.	The recombinant host microorganism as claimed in claim 43 wherein said celZ gene is derived from Erwinia chrysanthemi,<br>
45.	The recombinant host microorganism as claimed in claim 33 wherein said second heterologous polynucleotide segment comprises at least one pul gene or out gene.<br>
46.	The recombinant host microorganism as claimed in claim 45 wherein said second heterologous polynucleotide segment is derived from a bacterial cell selected from the family Enterobacteriaceae.<br>
47.	The recombinant host microorganism as claimed in claim 46 wherein said bacterial cell is selected from the group consisting of K, oxytoca, E. carotovora, E. carotovora subspecies carotovora, E. carotovora subspecies atroseptica, and E. chrysanthemi,<br>
48.	A method for enzymatically degrading an oligosaccharide comprising the steps of providing a oligosaccharide; and contacting said oligosaccharide with a host microorganism comprising a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase under the transcriptional control of a surrogate promoter, said promoter capable of causing increased expression of said polysaccharase; and a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein expression of said first and second polynucleotide segments results in the increased production of polysaccharase activity by the recombinant host microorganism such that the oligosaccharide is enzymatically degraded.<br><br>
49.	The method as claimed in claim 48 wherein said polysaccharase is secreted.<br>
50.	The method as claimed in claim 48 wherein said host cell is ethanologenic.<br>
51.	The method as claimed in claim 48 wherein said method is conducted in an aqueous solution.<br>
52.	The method as claimed in claim 48 wherein said method is used for simultaneous saccharification and fermentation.<br>
53.	The method as claimed in claim 48 wherein said ohgosaccharide is selected from the group consisting of lignocellulose, hemicellulose, cellulose, pectin, and any combination thereof<br>
54.	A method of making a recombinant microorganism for use in simultaneous saccharification and fermentation comprising introducing into said microorgansim a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase under the transcriptional control of a surrogate promoter, said promoter capable of causing increased expression of said polysaccharase; and introducing into said host microorganism a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein expression of said first and second polynucleotide segments results in the increased production of a polysaccharase by the recombinant host microorganism.<br><br>
55.	The recombinant microorganism as claimed in claim 54 wherein production is selected from the group consisting of activity, amount, and a combination thereof<br>
56.	The recombinant microorganism as claimed in claim 54 or 55 wherein said polysaccharase polypeptide is secreted.<br>
57.	The method as claimed in claim 54, 55, or 56 wherein said microorgansim is ethanologenic.<br>
58.	A vector comprising the polynucleotide sequence of pLOI2306 (SEQ ID NO: 12).<br>
59.	A microorganism having a vector comprising the polynucleotide sequence of pLOI2306 (SEQ ID NO: 12).<br>
60.	A method of making a recombinant microorganism integrant comprising introducing into said host microorganism a vector comprising the polynucleotide sequence of pLOI2306 (SEQ ID NO: 12); and identifying a host microorganism having said vector stably integrated.<br>
61.	The method for expressing a polysaccharase in a microorganism comprising introducing into said host microorganism a vector comprising the polynucleotide sequence of pLOI2306 (SEQ ID NO: 12); and identifying a microorganism expressing said polysaccharase.<br>
62.	The method as claimed in any one of claims 59 to 61 wherein said host microorganism is ethanologenic.<br><br>
63.	The method for producing ethanol from an oligosaccharide source comprising, contacting said oligosaccharide source with a ethanologenic host microorganism comprising a first heterologous polynucleotide segment comprising a sequence encoding a polysaccharase under the transcriptional control of a surrogate promoter, said promoter capable of causing increased expression of said polysaccharase; and a second heterologous polynucleotide segment comprising a sequence encoding a secretory polypeptide, wherein expression of said first and second polynucleotide segments results in the increased production of polysaccharase activity by the ethanologenic cell such that the oligosaccharide source is enzymatically degraded and fermented into ethanol.<br>
64.	The microorganism as claimed in claim 63 wherein said polysaccharase is selected from the group consisting of glucanase, endoglucanase, exoglucanase, cellobiohydrolase, a-glucosidase, endo-l,4-a-xylanase, β-xylosidase, β-glucuronidase, a-L-arabinofuranosidase, acetylesterase, acetylxylanesterase, a-amylase, β-amylase, glucoamylase, puUulanase, β-glucanase, hemicellulase, arabinosidase, mannanase, pectin hydrolase, pectate lyase, or a combination thereof<br>
65.	The microorganism as claimed in claim 64 wherein said polysaccharase is glucanase.<br>
66.	The microorganism as claimed in claim 65 wherein said glucanase is an expression product of a celZ gene.<br>
67.	The microorganism as claimed in claim 66 wherein said celZ gene is derived from Erwinia chrysanthemi.<br><br>
68.	The microorganism as claimed in claim 63 wherein said second heterologous<br>
polynucleotide segment comprises at least one pul gene or out gene.<br>
69.	The microorganism as claimed in claim 63 wherein said microorganism is<br>
selected from the family Enterobacteriaceae.<br>
70.	The microorganism as claimed in claim 63 wherein said microorganism is<br>
selected from the group consisting of Escherichia and Klebsiella.<br>
71.	The microorganism as claimed in claim 63, wherein said host microorganism is<br>
selected from the group consisting of E. coli K04 (ATCC 55123), E. coli<br>
KOll (ATCC 55124), E. co/z KOI2 (ATCC 55125), K. oxytoca M5a1,AND K.<br>
oxytoca P2 (ATCC 55307).<br>
72.	The microorganism as claimed in claim 63, wherein said polysaccharase is of<br>
increased activity.<br>
73.	The method as claimed in claim 63, wherein said method is conducted in an aqueous solution.<br>
74.	The method as claimed in claim 63, wherein said oligosaccharide is selected from the group consisting of lignocellulose, hemicellulose, cellulose, pectin, and any combination thereof<br>
75.	The method as claimed in claim 63, wherein said first heterologous<br>
polynucleotide segment is, or derived from, pLOI2306 (SEQ ID NO: 12).<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtYXNzaWduZW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtY2xhaW1zIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-description(complete) filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-description(complete) granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTgwNC1jaGUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1804-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="208672-tuned-patch-cable.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208674-a-process-for-manufacture-of-quaternary-carboxylic-acid-from-linear-olefins.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208673</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1804/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Dec-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. THE UNIVERSITY OF FLORIDA RESEARCH FOUNDATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>223 Grinter Hall, P.O. Box 115500, Gainesville, FL 32606</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>INGRAM, Lonnie, O.</td>
											<td>3132 N.W. 57th Terrace, Gainesville, FL 32606</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ZHOU, Shengde</td>
											<td>Apartment 46, 328 S.W. 34th Street, Gainesville, FL 32607</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/56</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/14773</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-05-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/136,376</td>
									<td>1999-09-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208673-a-recombinant-host-microorganism-and-a-method-of-making-a-recombinant-microorganism by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:13:17 GMT -->
</html>
