<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208981-a-nucleic-acid-construct-comprising-an-expression-regulating-nucleic-acid-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:20:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208981:A NUCLEIC ACID CONSTRUCT COMPRISING AN EXPRESSION-REGULATING NUCLEIC ACID SEQUENCE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A NUCLEIC ACID CONSTRUCT COMPRISING AN EXPRESSION-REGULATING NUCLEIC ACID SEQUENCE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a nucleic acid construct comprising an expression- regulating nucleic acid sequence operably linked to nucleic acid sequence encoding a polypeptide of interest, wherein said expression-regulating nucleic acid sequence has at least 70% nucleotide sequence identity over the entire length to nucleotides 1 to 1779 of SEQ ID No 1 or wherein said expression regulating sequence comprises at least 40 contiguous nucleotides of nucleotides 1 to 1779 of SEQ ID No 1.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL EXPRESSION-REGULATING SEQUENCES AND EXPRESSION PRODUCTS IN THE FIELD OF FILAMENTOUS FUNGI<br>
Field of the invention<br>
The subject invention relates to novel enzymes derived from filamentous fungi, especially from strains of the genus Chrysosporium, and to coding sequences and expression-regulating sequences for these enzymes. The present invention is an extension on the invention disclosed in WO 00/20555 (PCT/NL99/00618), filed October 6, 1999, not published prior to the present priority date. The latter disclosure is herein incorporated by reference.<br>
Background to the invention.<br>
A number of hosts for gene expression and methods of transformation have been disclosed in the prior art. Bacteria are often mentioned e.g. Escherichia coli. E. coli is however a microorganism incapable of secretion of a number of proteins or polypeptides and as such is undesirable as host cell for production of protein or polypeptide at the industrial level. An additional disadvantage for E. coli, which is valid also for bacteria in general, is that prokaryotes cannot provide additional modifications required for numerous eukaryotic proteins or polypeptides to be produced in an active form. Glycosylation of proteins and proper folding of proteins are examples of processing required to ensure an active protein oV polypeptide is produced. To ensure such processing one can sometimes use mammalian cells; however, the disadvantage of such cells is that they are often difficult to maintain and require expensive media. Such transformation systems are therefore not practical for production of proteins or polypeptides at the industrial level. They may be cost efficient for highly priced pharmaceutical compounds requiring relatively low amounts, but certainly not for industrial enzymes.<br>
A number of fungal expression systems have been developed e.g. Aspergillus niger, Aspergillus awamori Aspergillus nidulans, Trichoderma reesei. A number of others have been suggested but for various reasons have not found wide-spread acceptance or use. In general terms the ideal host must fulfil a large number of criteria:<br>
The ideal host must be readily fermented using inexpensive medium.<br>
The ideal host should use the medium efficiently.<br>
The ideal host must produce the polypeptide or protein in high yield, i.e. must exhibit high protein to biomass ratio.<br>
The ideal host should be capable of efficient secretion of the protein or polypeptide.<br>
The ideal host must enable ease of isolation and purification of the desired protein or polypeptide.<br>
The ideal host must process the desired protein or polypeptide such that it is produced in an<br><br>
active form not requiring additional activation or modification steps.<br>
The ideal host should be readily transformed.<br>
The ideal host should allow a wide range of expression regulatory elements to be used thus ensuring ease of application and versatility.<br>
The ideal host should allow use of easily selectable markers that are cheap to use.<br>
The ideal host should produce stable transformants.<br>
The ideal host should allow cultivation under conditions not detrimental to the protein or polypeptide product, e.g. low viscosity, low shear.<br>
WO 96/02563 and US patents 5,602,004, 5,604,129 and 5,695,985 to Novo Nordisk describe the drawbacks of Aspergillus and Trichoderma systems and suggest that cultivation conditions for other fungi may be more suited to large scale protein production. The only examples provided for any transformed cultures are those of Myceliophthora ihermophila, Acremonium alabamense, Thielavia terrestris and Sporotrichum cellulophilum strains. The Sporotrichum strain is reported to lyse and produce green p1gment under fermentation conditions not leading to such results for the other strains. A non-sporulating mutant of Thielavia terrestris is described as being the organism of choice by virtue of its morphology. However it is also stated that the protoplasting efficiency of Thielavia and Acremonium (whereby the Acremonium strain used was the imperfect state of the Thielavia strain used) is low and that hygromycin was not useful as a selection marker. A large number of others are suggested as being potentially useful by virtue of their morphology but no transformation thereof is described. The suggested strains are Corynascus, Thermoascus, Chaetomium, Ctenomyces, Scytalidium and Talaromyces. The transformed hosts are mentioned as only producing low levels of the introduced Humicola xylanase with Thielavia producing the lowest amount; however, the information is ambiguous and could actually infer Thielavia was the best embodiment. The nomenclature of this reference is based on the ATCC names of Industrial Fungi of 1994. Thus it is apparent that no high degree of heterologous expression was achieved and in fact no positive correlation could be derived between the postulated morphology and the degree of expression. If any correlation could be made, it was more likely to be negative. According to the 1996 ATCC fungal classification Sporotrichum thermophilum ATCC 20493 is a Myceliophthora thermophila strain. Currently the strain is still identified as Myceliophthora ihermophila. The unpredictability of the art is apparent from these recent disclosures.<br>
WO 97/26330 of Novo Nordisk suggests a method of obtaining mutants of filamentous fungal parent cells having an improved property for production of heterologous polypeptide. The method comprises first finding a specific altered morphology followed by assessing whether a transformant produces more heterologous polypeptide than the parent. The method is illustrated only for strains of Fusarium A3/5 and Aspergillus oryzae. The method is suggested to be applicable for Aspergillus,  Trichoderma,  Thielavia, Fusarium, Neurospora, Acremonium,  Tolyplocadium,<br><br>
Humicola, SCytalidium, Myceliophthora or Mucor. As stated above, the unpredictability in the art and also the unpredictability of the method of the cited application do not provide a generally applicable teaching with a reasonable expectation of success.<br>
In WO 00/20555, we have described an alternative fungal expression system with the simplicity of use of the above-mentioned Aspergilli and Trichoderma fulfilling the above requirements. The new system provides the additional advantages that transformation rates are higher than those for the frequently used Trichoderma reesei system. In addition the culture conditions offer the additional bonus of being advantageous for the polypeptide product.<br>
Detailed description of the invention.<br>
We now describe a number of industrially interesting enzymes derived from Chrysosporium strains, together with full sequence information. We also describe novel promoter systems derived from Chrysosporium strains and useful for expressing homologous and heterologous genes.<br>
The present invention is in particular concerned with glycosyl hydrolases of the families 7 (e.g. cellobiohydrolases) and 10 (e.g. xylanases), and glyceraldehyde phosphate dehydrogenases, as identified by their amino acid sequence, as well as peptides derived from these enzymatic proteins, and with nucleic acid sequences encoding these peptides and proteins, as well as, in particular, with regulating sequences related to these genes.<br>
In particular, the present invention pertains to isolated or recombinant enzymic proteins or active parts thereof of the three classes referred to above, including mutants thereof having at least a certain degree of sequence identity as specified in the further disclosure and in the claims, as well as nucleic acid sequences encoding these proteins or parts thereof, and/or nucleic acid sequences regulating their expression. These enzymes are especially: (1) a glycosyl hydrolase of family 7 (cellobiohydrolase, CBHl) having at least 75%, preferably at least 80% or even at least 85% amino acid identity with the sequence of SEQ ID No 2; (2) a glycosyl hydrolase of family 10 (endo-xylanase XYLl) having at least 70%, preferably at least 75% or even at least 80% amino acid identity with the sequence of SEQ ID No 4; and (3) a glyceraldehyde phosphate dehydrogenase (GPDI) having at least 86%, preferably at least 90% or even at least 1 93% amino acid identity with the sequence of SEQ ID No 6. Polypeptides and nucleic acid sequences encoding these polypeptides, having at least 20, preferably at least 30 contiguous amino acids of SEQ ID No*s 2, 4 and 6 are also a preferred part of the invention. The corresponding nucleotide sequences are dep1cted in SEQ ID No's 1 (cbhl), 3 (xyll) and 5 (gpdl), respectively.<br>
The recombinant enzymes may comprise essentially the complete protein, or a truncated protein having at least part of the enzymatic activity. Such truncated part may be the catalytic domain, or at least about 75%) of the amino acids thereof. By way of example, the catalytic domain of the CBHl according to the invention comprises the amino acids 20-495 of the aminoacid<br><br>
sequence of SEQ ID No. 2, and the catalytic domain of the XYLl according to the invention comprises the aminoacids 54-384 of the aminoacid sequence of SEQ ID No. 4. The catalytic domain may or may not be combined with a signal sequence originating from another protein and/or with a carbohydrate-binding domain from another enzymic protein. Alternatively, the<br>
J cellulose-binding domain of the enzymes of the invention (CBHl and XYLl) may be fused to catalytic domains of other enzymic proteins.<br>
The nucleic acid sequences according to the invention may be complete protein-encoding regions or oligonucleotides or, preferentially, expression-regulating sequences. Oligonucleotides may be used also as probes for identifying genes corresponding to, but not identical to the genes of<br>
I SEQ ID No.'s 1, 3 and 5; these genes, when fulfilling the percentage identity criteria defined herein, as well as encoding and non-encoding parts thereof and their expression products are also part of the invention. Oligonucletoides are preferably 15-75, most preferably 20-50 nucleotides in length.<br>
The invention also pertains to expression systems (cassettes) comprising either an expression-regulating region (including a promoter) of any of the three protein classes fused to a gene encoding another protein of interest, or an encoding region of any of these proteins fused to another expression regulating region, or both the expression-regulating region and the protein-encoding region of these novel proteins. The expression-regulating region comprises at least 60%, preferably at least 70%, more preferably at least 75% or even 80% of the 5'-non-coding region of SEQ ID No.'s 1, 3 and 5, and/or at least 20, especially at least 40 contiguous nucleotides from these 5' non-coding regions. Terminating sequences similarly derived from the 3' non-coding regions of the genes of the invention are also useful in expressing cassettes, whether combined with homologous or heterologous genes.<br>
The polynucleotides and oligonucleotides of the invention can have the minimum sequnece identity with the corresponding sequences of SEQ ID NO's 1, 3 or 5, or, alternatively hybridise under stringent conditions with these given sequences. Stringent hybridisation conditions are those as understood in the art, e.g. hybridisation in 6 x SSC (20xSSC per 1000 ml : 175.3 g NaCl, 107.1 g sodium citrate.5H20, pH 7.0), 0.1% SDS, 0.05% sodium pyrophosphate, 5 * Denhardt's solution and 20 µg/ml denatured herring sperm DNA at 56°C for 18-24 hrs followed by two 30 min. washes in 5 x SSC, 0.1 % SDS at 56°C and two 30 min. washes in 2 x SSC, 0.1% SSC at<br>
56°C.<br>
These expression systems may be contained in a Chrysosporium host, such as a Chrysosporium lucknowense host, or in another non-fungal or, preferably, fungal host. Examples of<br><br>
Where reference is made in this specification and in the appending claims to "polypeptides" or "peptides" or "polypeptides of interest" or "peptides of interest" as the products of the expression system of the invention, this term also comprise proteins, i.e. polypeptides having a particular function and/or secondary and/or tertiary structure. Where reference is made to a percentage amino acid identity, such identity relates to a complete protein or to a specific part defined by initial and final amino acid number, as determined by the conventionally used BLAST algorithm.<br>
In the fungal expression system described in WO 00/20555, the pH of the culture medium can be neutral or alkaline thus no longer subjecting the produced protein or polypeptide to aggressive and potentially inactivating acid pH. It is also possible to culture at acid pH such as pH 4 for cases where the protein or polypeptide is better suited to an acidic environment. Suitably culture can occur at a pH between 4.0-10.0. A preference however exists for neutral to alkaline pH as the host strain exhibits better growth at such pH, e.g. between 6 and 9. Growth at alkaline pH which can be from pH 8 up and can even be as high as 10 is also a good alternative for some cases. Also the cultivation temperature of such host strains is advantageous to the stability of some types of produced polypeptide. The cultivation temperature is suitably at a temperature of 23-43°C. Clearly such conditions are of particular interest for production of mammalian polypeptides. The selected temperature will depend on cost effectiveness of the cultivation and sensitivity of the polypeptide or cultivation strain.<br>
It has also been ascertained that the biomass to viscosity relation and the amount of protein produced is exceedingly favourable for the Chrysosporium host. Comparisons have been carried out with Trichoderma longibrachiatum (formerly also known as Trichoderma reesei) and with Aspergillus niger. Trichoderma longibrachiatum gave 2.5-5 g/1 biomass, Aspergillus niger gave 5-10 g/1 biomass and the Chrysosporium host gave 0.5-1 g/l biomass under their respective optimised conditions. This thus offers 5-10 fold improvement over the commercially used strains. The subject invention is directed at expression systems comprising a nucleic acid sequence encoding a heterologous protein or polypeptide, said nucleic acid sequence being operably linked to an expression regulating region described below and optionally a secretion signal encoding sequence and/or a carrier protein encoding sequence. Preferably a recombinant strain according to the invention will secrete the polypeptide of interest. This will avoid the necessity of disrupting the cell in order to isolate the polypeptide of interest and also minimise the risk of degradation of the expressed product by other components of the host cell.<br>
Chrysosporium can be defined by morphology consistent with that disclosed in Bamett and Hunter 1972, Illustrated Genera of Imperfect Fungi, 3rd Edition of Burgess Publishing Company. Other sources providing details concerning classification of fungi of the genus Chrysosporium are known e.g. Sutton Classification (Van Oorschot, C.A.N. (1980) "A revision of Chrysosporium and allied genera" in Studies in Mycology No. 20 of the CBS in Baarn, The Netherlands pl-36). CBS is<br><br>
one of the depository institutes of the Budapest Treaty. According to these teachings the genus Chrysosporium falls within the family Moniliaceae which belongs to the order Hyphomycetales. The following strains are defined as Chrysosporium but the definition of Chrysosporium is not limited to these strains: C botryoides, C carmichaelii, C. crassitunicatum, C. europae, C evolceannui, C. farinicola, C fastidium, C. filiforme, C georgiae, C globiferum, C globiferum var. articulatum, C. globiferum var. niveum, C. hirundo, C hispanicum, C holmii, C indicum, C inops, C keratinophilum, C kreiselii, C kiaurovianum, C lignorum, C. lobatum, C. lucknowense, C lucknowense Garg 27K, C medium, C. medium var. sp1ssescens, C mephiticum, C merdarium, C. merdarium var. roseum, C minor, C pannicola, C. parvum, C parvum var. crescens, C p1losum, C. pseudomerdarium, C pyriformis, C queenslandicum, C sigleri, C. sulfureum, C. synchronum, C. trop1cum, C. undulatum. C vallenarense, C. vespertilium, C zonatum.<br>
C. lucknowense forms one of the species of Chrysosporium that have raised particular interest as it has provided a natural high producer of cellulase proteins (WO 98/15633 and related US 5,811,381). The characteristics of this Chrysosporium lucknowense are:<br>
Colonies attain 55 mm diameter on Sabouraud glucose agar in 14 days, are cream-coloured, felty and fluffy; dense and 3-5 mm high; margins are defined, regular, and fimbriate; reverse pale yellow to cream-coloured. Hyphae are hyaline, smooth- and thin-walled, little branched. Aerial hyphae are mostly fertile and closely septate, about 1-3.5 µm wide. Submerged hyphae are infertile, about 1-4.5 µm wide, with the thinner hyphae often being contorted. Conidia are terminal and lateral, mostly sessile or on short, frequently conical protrusions or short side branches. Conidia are solitary but in close proximity to one another, 1-4 conidia develop1ng on one hyphal cell, subhyaline, fairly thin- and smooth-walled, mostly subglobose, also clavate orobovoid, 1-celled, 2.5-11 X 1.5-6µm, with broad basal scars (1-2 µm). Intercalary conidia are absent. Chlamydospores are absent. ATCC 44006, CBS 251.72, CBS 143.77 and CBS 272.77 are examples of Chrysosporium lucknowense strains and other examples are provided in WO 98/15633 (US 5,811,381).<br>
A further strain was isolated from this species with an even higher production capacity for cellulases. This strain is called CI by its internal notation and was deposited with the international Depository of the All Russian Collection of micro-organisms of the Russian Academy of Sciences Bakrushina Street 8, Moscow, Russia 113184 on August 29, 1996, as a deposit according to the Budapest Treaty and was assigned Accession Number VKM F-3500D. It is called Chrysosporium lucknowense Garg 27K. The characteristics of the C) strain are as follows:<br>
Colonies grow to about 55-66 mm diameter in size on potato-dextrose agar in about 7 days; are white-cream-coloured, felty, 2-3 µm high at the centre; margins are defined, regular, fimbriate; reverse pale, cream-coloured. Hyphae are hyaline, smooth- and thin-walled, little branched. Aerial hyphae are fertile, septate, 2-3 mm wide. Submerged hyphae are infertile. Conidia are terminal and lateral; sessile or on short side branches; absent; solitary, but in close proximity to one another.<br><br>
hyaline, thirf- and smooth-walled, subglobose, clavate or obovoid, 1-celled, 4-10 µm. Chlamydo-spores are absent. Intercalary conidia are absent.<br>
The method of isolation of the CI strain is described in WO 98/15633, US 5,811,381, and<br>
US 6,015,707. Also included within the definition of Chrysosporium are strains derived from<br>
      Chrysosporium predecessors including those that have mutated somewhat either naturally or by<br>
induced mutagenesis.  Mutants of Chrysosporium can  be  obtained  by  induced  mutagenesis,<br>
especially by a combination of irradiation and chemical mutagenesis.<br>
For example strain CI was mutagenised by subjecting it to ultraviolet light to generate strain UV13-6. This strain was subsequently further mutated with N-methyl-N'-nitro-N-nitrosoguanidine to generate strain NG7C-19. The latter strain in turn was subjected to mutation by ultraviolet light resulting in strain UV18-25. During this mutation process the morphological characteristics have varied somewhat in culture in liquid or on plates as well as under the microscope. With each successive mutagenesis the cultures showed less of the fluffy and felty appearance on plates that are described as being characteristic o{ Chrysosporium, until the colonies attained a flat and matted appearance. A brown p1gment observed with the wild type strain in some media was also less prevalent in mutant strains. In liquid culture the mutant UV18-25 was noticeably less viscous than the wild type strain CI and the mutants UV13-6 and NG7C-19. While all strains maintained the gross microscop1c characteristics of Chrysosporium, the mycelia became narrower with each successive mutation and with UV 18-25 distinct fragmentation of the mycelia could be observed. This mycelial fragmentation is likely to be the cause of the lower viscosity associated with cultures of UV 18-25. The ability of the strains to sporulate decreased with each mutagenic step. The above illustrates that for a strain to belong to the genus Chrysosporium there is some leeway from the above morphological definition. At each mutation step production of cellulase and extracellular proteins has in addition also increased, while several mutations resulted in decrease of protease expression. Criteria with which fungal taxonomy can be determined are available from CBS, VKMF and ATCC for example. The strains internally designated as Chrysosporium strain CI, strain UV13-6, strain NG7C-19 and strain UV 18-25, have been deposited in accordance with the Budapest Treaty with the All Russian Collection (VKM) depository institute in Moscow. Wild type CI strain was deposited with number VKM F-3500 D, deposit date 29-08-1996, CI UV13-6 mutant was deposited as VKM F-3632 D (02-09-1998), CI NG7c-19 mutant was deposited as VKM F-3633 D (02-09-1998) and CI UVl 8-25 mutant was deposited as VKM F-363 1 D (02-09-1998),<br>
It is preferable to use non-toxic Chrysosporium strains of which a number are known in the art as this will reduce risks to the environment upon large scale production and simplify production procedures with the concomitant reduction in costs.<br>
An expression-regulating region is a DNA sequence recognised by the host Chrysosporium strain for expression. It comprises a promoter sequence operably linked to a nucleic acid sequence<br><br>
\   ,<br>
encoding the polypeptide to be expressed. The promoter is linked such that the positioning vis-a-vis the initiation codon of the sequence to be expressed allows expression. The promoter sequence can be constitutive or inducible. Any expression regulating sequence or combination thereof capable of permitting expression of a polypeptide from a Chrysosporium strain is envisaged. The expression regulating sequence is suitably a fungal expression-regulating region e.g. an ascomycete regulating region. Suitably the fungal expression regulating region is a regulating region from any of the following genera of fungi: Aspergillus, Trichoderma, Chrysosporium, Hansenula, Mucor, p1chia, Neurospora, Tolypocladium, Rhizomucor, Fusarium, Penicillium, Saccharomyces, Talaromyces or alternative sexual forms thereof like Emericella, Hypocrea e.g. the cellobiohydrolase promoter from Trichoderma, glucoamylase promoter from Aspergillus^ glyceraldehyde phosphate dehydrogenase promoter from Aspergillus, alcohol dehydrogenase A and alcohol dehydrogenase R promoter of Aspergillus, TAKA amylase promoter from Aspergillus, phosphoglycerate and cross-pathway control promoters of Neurospora, aspartic proteinase promoter of Rhizomucor miehei, lipase promoter of Rhizomucor miehei and beta-gaiactosidase promoter of Penicillium canescens. An expression regulating sequence from the same genus as the host strain is extremely suitable, as it is most likely to be specifically adapted to the specific host. Thus preferably the expression regulating sequence is one from a Chrysosporium strain.<br>
Preferably an expression-regulating region enabling high expression in the selected host is applied. This is preferably an expression-regulating region derived from Chrysosporium according to the invention. It can also be a high expression-regulating region derived from a heterologous host, such as are well known in the art. Specific examples of proteins known to be expressed in large quantities and thus providing suitable expression regulating sequences for the invention are without being limited thereto hydrophobin, protease, amylase, xylanase, pectinase, esterase, beta-galactosidase, cellulase (e.g. endo-glucanase, cellobiohydrolase) and polygalacturonase. The high production has been ascertained in both solid state and submerged fermentation conditions. Assays for assessing the presence or production of such proteins are well known in the art. The catalogues of Sigma and Megazyme for example provide numerous examples. Megazyme is located at Bray Business Park, Bray, County Wicklow in Ireland. Sigma Aldrich has many affiliates world wide e.g. USA P.O. Box 14508 St. Louis Missouri. For cellulase we refer to commercially available assays such as CMCase assays, endoviscometric assays, Avicelase assays, beta-glucanase assays, RBBCMCase assays, Cellazyme C assays. Xylanase assays are also commercially available (e.g. DNS and Megazyme). Alternatives are well known to a person skilled in the art and can be found<br><br>
We have found that heterologous expression-regulating sequences work as efficiently in Chrysosporium as native Chrysosporium sequences. This allows well known constructs and vectors to be used in transformation of Chrysosporium as well as offering numerous other possibilities for constructing vectors enabling good rates of expression in this novel expression and secretion host. For example standard Aspergillus transformation techniques can be used as described for example by Christiansen et al in Bio/Technol. 6:1419-1422 (1988). Other documents providing details of Aspergillus transformation vectors, e.g. US patents 4,816,405, 5,198,345, 5,503,991, 5,364,770 and 5,578,463, EP-B-215.594 (also for Trichoderma) and their contents are incorporated by reference. As extremely high expression rates for cellulase have been ascertained for Chrysosporium strains, the expression regulating regions of such proteins are particularly preferred. We refer for specific examples to the previously mentioned deposited Chrysosporium strains.<br>
A nucleic acid construct comprising a nucleic acid expression regulatory region from Chrysosporium, preferably from Chrysosporium lucknowense or a derivative thereof forms a preferred embodiment of the invention, as does the mutant Chrysosporium strain comprising such operably linked to a gene encoding a polypeptide to be expressed. Such a nucleic acid construct will be an expression regulatory region from Chrysosporium associated with cellulase or xylanase expression, preferably cellobiohydrolase expression, or glyceraldehyde phosphate dehydrogenase expression, as detailed below. The nuclejc acid sequence according to the invention can suitably be obtained from a Chrysosporium strain, such strain being defined elsewhere in the description. The manner in which promoter sequences can be determined are numerous and well known in the art. Nuclease deletion experiments of the region upstream of the ATG codon at the beginning of the relevant gene will provide such sequence. Also for example analysis of consensus sequences can lead to finding a gene of interest. Using hybridisation and amplification techniques one skilled in the art can readily arrive at the corresponding promoter sequences.<br>
The promoter sequences of CI endogiucanases were identified in this manner, by cloning the corresponding genes. Preferred promoters according to the invention are the 55 kDa cellobiohydrolase (CBHl) promoter, the 30 kDa xylanase (Xyll) promoters, and the glyceraldehyde phosphate dehydrogenase promoter, as the enzymes are expressed at high level by their own promoters. The corresponding promoter sequences are identified in a straightforward manner by cloning as described in WO 00/20555, using the sequence information given in SEQ ID No. 1 (for CBHl) and SEQ ID No. 3 (for Xyll), respectively. The promoters of the carbohydrate-degrading<br><br>
For particular embodiments of the recombinant strain and the nucleic acid sequence according to the invention we also refer to the examples. We also refer for the recombinant strains to prior art describing high expression promoter sequences in particular those providing high expression in fungi e.g. such as are disclosed for Aspergillus and Trichoderma. The prior art provides a number of expression regulating regions for use in Aspergillus e.g. US 5,252,726 of Novo and US 5,705,358 of Unilever. The contents of such prior art are hereby incorporated by<br>
reference.<br>
The hydrophobin gene is a fungal gene that is highly expressed. It is thus suggested that the promoter sequence of a hydrophobin gene, preferably from Chrysosporium, may be suitably applied as expression regulating sequence in a suitable embodiment of the invention. Trichoderma reesei and Trichoderma harzianum gene sequences for hydrophobin have been disclosed for example in the prior art as well as a gene sequence for Aspergillus fumigatus and Aspergillus nidulans and the relevant sequence information is hereby incorporated by reference (Munoz et al, Curr. Genet. 1997, 32(3):225-230; Nakari-Setala T. et al, Eur. J. Biochem. 1996 15:235 (l-2):248-255, M. Parta et al, Infect. Immun. 1994 62 (10): 4389-4395 and Stringer M.A. et al. Moi Microbiol 1995 16(l):33-44). Using this sequence information a person skilled in the art can obtain the expression regulating sequences of Chrysosporium hydrophobin genes without undue experimentation following standard techniques as suggested already above. A recombinant Chrysosporium strain according to the invention can comprise a hydrophobin-regulating region operably linked to the sequence encoding the polypeptide of interest.<br>
An expression regulating sequence can also additionally comprise an enhancer or silencer. These are also well known in the prior art and are usually located some distance away from the promoter. The expression regulating sequences can also comprise promoters with activator binding sites and repressor binding sites. In some cases such sites may also be modified to eliminate this type of regulation. Filamentous fungal promoters in which creA sites are present have been described. Such creA sites can be mutated to ensure the glucose repression normally resulting from the presence of the non-mutated sites is eliminated. Gist-Brocades' WO 94/13820 illustrates this principle. Use of such a promoter enables production of the polypeptide encoded by the nucleic acid sequence regulated by the promoter in the presence of glucose. The same principle is also apparent from WO 97/09438. These promoters can be used either with or without their creA sites. Mutants in which the creA sites have been mutated can be used as expression regulating sequences in a recombinant strain according to the invention and the nucleic acid sequence it regulates can then be expressed in the presence of glucose. Such Chrysosporium promoters ensure derepression in an analogous manner to that illustrated in WO 97/09438. The identity of creA sites is known<br><br>
creA gene itself, so that the strain can, notwithstanding the presence of creA binding sites, produce the protein or polypeptide in the presence of glucose.<br>
Terminator sequences are also expression-regulating sequences and these are operably linked to the 3' terminus of the sequence to be expressed. Any fungal terminator is likely to be functional in the host Chrysosporium strain according to the invention. Examples are A. nidulans trpC terminator (1), A. niger alpha-glucosidase terminator (2), A. niger glucoamylase terminator (3), Mucor miehei carboxyl protease terminator (US 5,578,463) and the Trichoderma reesei cellobiohydrolase terminator. Naturally Chrysosporium terminator sequences will function in Chrysosporium and are suitable e.g. CBHl terminator.<br>
A suitable recombinant Chrysosporium strain to be used according to the invention has the nucleic acid sequence to be expressed operably linked to a sequence encoding the amino acid sequence defined as signal sequence. A signal sequence is an amino acid sequence which when operably linked to the amino acid sequence of the expressed polypeptide allows secretion thereof from the host fungus. Such a signal sequence may be one normally associated with the heterologous polypeptide or may be one native to the host. It can also be foreign to both host and the polypeptide. The nucleic acid sequence encoding the signal sequence must be positioned in frame to permit translation of the signal sequence and the heterologous polypeptide. Any signal sequence capable of permitting secretion of a polypeptide from a Chrysosporium strain is envisaged. Such a signal sequence is suitably a fungal signal sequence, preferably an ascomycete signal sequence.<br>
Suitable examples of signal sequences can be derived from yeasts in general or any of the following specific genera of fungi: Aspergillus, Trichoderma, Chrysosporium, p1chia, Neurospora, Rhizomucor, Hansenula, Humicola, Mucor, Tolypocladium, Fidsarium. Penicillium, Saccharo-myces, Talaromyces or alternative sexual forms thereof like Emericella, Hypocrea. Signal sequences that are particularly useful are often natively associated with the following proteins a cellobiohydrolase, an endoglucanase, a beta-galactosidase, a xylanase, a pectinase, an esterase, a hydrophobin, a protease or an amylase. Examples include amylase or glucoamylase oi Aspergillus or Humicola (4), TAKA amylase of Aspergillus oryzae, alpha-amylase of Aspergillus niger, carboxyl peptidase of Mucor (US 5,578,463), a lipase or proteinase from Rhizomucor miehei, cellobiohydrolase of Trichoderma (5), beta-galactosidase of Penicillium canescens and alpha mating factor of Saccharomyces.<br>
Alternatively the signal sequence can be from an amylase or subtilisin gene of a strain of Bacillus. A signal sequence from the same genus as the host strain is extremely suitable as it is most likely to be specifically adapted to the specific host thus preferably the signal sequence is a signal sequence of Chrysosporium, especially of Chrysosporium strain CI, strain UV13-6, strain NG7C-19 and strain UV18-25, referred to above. Signal sequences from filamentous fungi, yeast<br><br>
and bacteria are useful. Signal sequences of non-fungal origin are also considered useful, particularly bacterial, plant and mammalian.<br>
A recombinant host to be used according to any of the embodiments of the invention can further comprise a selectable marker. Such a selectable marker will permit easy selection of transformed or transfected cells. A selectable marker often encodes a gene product providing a specific type of resistance foreign to the non-transformed strain. This can be resistance to heavy metals, antibiotics and biocides in general. Prototrophy is also a useful selectable marker of the non-antibiotic variety. Non-antibiotic selectable markers can be preferred where the protein or polypeptide of interest is to be used in food or pharmaceuticals with a view to speedier or less complicated regulatory approval of such a product. Very often the GRAS indication is used for such markers. A number of such markers are available to the person skilled in the art. The FDA e.g. provides a list of such. Most commonly used are selectable markers selected from the group conferring resistance to a drug or relieving a nutritional defect e.g. the group comprising amdS (acetamidase), hph (hygromycin phosphotransferase), pyrG (orotidine-5'-phosphate decarboxylase), trpC (anthranilate synthase), argB (ornithine carbamoyltransferase), sC (sulphate adenyl-transferase), bar (phosphinothricin acetyltransferase), glufosinate resistance, niaD (nitrate reductase), a bleomycin resistance gene, more specifically Sh ble, sulfonylurea resistance e.g. acetolactate synthase mutation ilvl. Selection can also be carried out by virtue of cotransformation<br>
■<br>
where the selection marker is on a separate vector or where the selection marker is on the same nucleic acid fragment as the polypeptide-encoding sequence for the polypeptide of interest.<br>
As used herein the term heterologous polypeptide is a protein or polypeptide not normally expressed and secreted by the Chrysosporium host strain used for expression according to the invention. The polypeptide can be of plant or animal (vertebrate or invertebrate) origin e.g. mammalian, fish, insect, or micro-organism origin, with the proviso it does not occur in the host strain. A mammal can include a human. A micro-organism comprises viruses, bacteria, archae-bacteria and fungi i.e. filamentous fungi and yeasts. Bergey's Manual for Bacterial Determinology provides adequate lists of bacteria and archaebacteria. For pharmaceutical purposes quite often a preference will exist for human proteins thus a recombinant host according to the invention forming a preferred embodiment will be a host wherein the polypeptide is of human origin. For purposes such as food production suitably the heterologous polypeptide will be of animal, plant or algal origin. Such embodiments are therefore also considered suitable examples of the invention. Alternative embodiments that are useful also include a heterologous polypeptide of any of bacterial, yeast, viral, archaebacterial and fungal origin. Fungal origin is most preferred.<br>
A suitable embodiment of the invention will comprise a heterologous nucleic acid sequence with adapted codon usage. Such a sequence encodes the native amino acid sequence of the host from which it is derived, but has a different nucleic acid sequence, i.e. a nucleic acid sequence in<br><br>
which certain codons have been replaced by other codons encoding the same amino acid but which are more readily used by the host strain being used for expression. This can lead to better expression of the heterologous nucleic acid sequence. This is common practice to a person skilled in the art. This adapted codon usage can be carried out on the basis of known codon usage of fungal vis-a-vis non-fungal codon usage. It can also be even more specifically adapted to codon usage of Chrysosporium itself. The similarities are such that codon usage as observed in Trichoderma, Humicola and Aspergillus should enable exchange of sequences of such organisms without adaptation of codon usage. Details are available to the skilled person concerning the codon usage of these fungi and are incorporated herein by reference.<br>
The invention is not restricted to the above-mentioned recombinant Chrysosporium strains, but also covers a recombinant Chrysosporium strain comprising a nucleic acid sequence encoding a homologous protein for a Chrysosporium strain, said nucleic acid sequence being operably linked to an expression-regulating region and said recombinant strain expressing more of said protein than the corresponding non-recombinant strain under the same conditions. In the case of homologous polypeptide of interest such is preferably a neutral or alkaline enzyme like a hydrolase, a protease or a carbohydrate degrading enzyme as already described elsewhere. The polypeptide may also be acidic. Preferably the recombinant strain will express the polypeptide in greater amounts than the non-recombinant strain. All comments mentioned vis-a-vis the heterologous polypeptide are also valid (mutatis mutandis) for the homologous polypeptide cellulase.<br>
Thus the invention also covers genetically engineered microbial strains wherein the sequence that is introduced can be of Chrysosporium origin. Such a strain can, however, be distinguished from natively occurring strains by virtue of for example heterologous sequences being present in the nucleic acid sequence used to transform or transfect the Chrysosporium, by virtue of the fact that multiple cop1es of the sequence encoding the polypeptide of interest are present or by virtue of the fact that these are expressed in an amount exceeding that of the non-engineered strain under identical conditions or by virtue of the fact that expression occurs under normally non-expressing conditions. The latter can be the case if an inducible promoter regulates the sequence of interest contrary to the non-recombinant situation or if another factor induces the expression than is the case in the non-engineered strain. The invention is directed at strains derived through engineering either using classical genetic technologies or genetic engineering methodologies.<br>
The expression systems and host strains containing them according to the invention can comprise a nucleic acid sequence encoding a heterologous protein selected from carbohydrate-degrading enzymes (cellulases, xylanases, mannanases, mannosidases, pectinases, amylases, e.g. glucoamylases, a-amylases, a- and p-galactosidases, a- and p-glucosidases, β-glucanases, chitinases, chitanases), proteases (endoproteases, amino-proteases, amino-and carboxy-peptidases, keratinases), other hydrolases (lipases, esterases, phytases), oxidoreductases (catalases, glucose-<br><br>
oxidases) and transferases (transglycosylases, transglutaminases, isomerases and invertases).<br>
The most interesting products to be produced according to invention are cellulases, xylanases, pectinases, lipases and proteases, wherein cellulases and xylanases cleave beta-1,4-bonds, and cellulases comprise endoglucanases, cellobiohydrolases and beta-glucosidases. These proteins are extremely useful in various industrial processes known in the art. Specifically for cellulases we refer e.g. to WO 98/15633 describing cellobiohydrolases and-endoglucanases of use. The contents of said application are hereby incorporated by reference.<br>
A recombinant according to the invention may have a nucleic acid sequence encoding the polypeptide of interest encodes a polypeptide that is inactivated or unstable at acid pH i.e. pH below 6, even below pH 5,5, more suitably even below pH 5 and even as low as or lower than pH 4. This is a particularly interesting embodiment, as the generally disclosed fungal expression systems are not cultured under conditions that are neutral to alkaline, but are cultured at acidic pH. Thus the system according to the invention provides a safe fungal expression system for proteins or polypeptides that are susceptible to being inactivated or are unstable at acid pH.<br>
Quite specifically a recombinant strain as defined in any of the embodiments according to the invention, wherein the nucleic acid sequence encoding the polypeptide of interest encodes a protein or polypeptide exhibiting optimal activity and/or stability at a pH above 5, preferably at neutral or alkaline pH (i.e. above 7) and/or at a pH higher than 6, is considered a preferred embodiment of the invention. More than 50%, more than 70% and even more than 90% of optimal activities at such pH values are anticipated as being particularly useful embodiments. A polypeptide expressed under the cultivation conditions does not necessarily have to be active at the cultivation conditions, in fact it can be advantageous for it to be cultured under conditions under which it is inactive as its active form could be detrimental to the host. What is however required is for the protein or polypeptide to be stable under the cultivation conditions. The stability can be thermal stability. It can also be stability against specific compositions or chemicals, such as are present for example in compositions or processes of production or application of the polypeptide or protein of interest. LAS in detergent compositions comprising cellulases or lipases, etc. is an example of a chemical often detrimental to proteins. The time periods of use in applications can vary from short to long exposure so stability can be over a varying length of time varying per application. The skilled person will be able to ascertain the correct conditions on a case by case basis. One can use a number of commercially available assays to determine the optimal activities of the various enzymatic products. The catalogues of Sigma and Megazyme for example show such. Specific examples of tests are mentioned elsewhere in the description. The manufacturers provide guidance on the application.<br>
A Chrysosporium strain can be suitably used to transform or transfect with the sequence of interest to be expressed and such a strain exhibits a relatively low biomass. We have found that<br><br>
Chrysosporium strains having a biomass two to five times lower than that of Trichoderma reesei when cultured to a viscosity of 200-600 cP at the end of fermentation and exhibiting a biomass of 10 to 20 times lower than that oi Aspergill-us niger when cultured to a viscosity of 1500-2000 cP under corresponding conditions, i.e. their respective optimal cultivation conditions can provide a high level of expression. This level of expression far exceeds that of the two commercial reference strains at a much lower biomass and at much lower viscosity. This means that the yield of expression of such Chrysosporium strains will be appreciably higher than from Aspergillus niger and Trichoderma reesei. Such a transformed or transfected Chrysosporium strain forms a suitable embodiment of the invention.<br>
We fmd a biomass of 0,5-1,0 g/1 for Chrysosporium strain Cl(18-25) as opposed to 2,5-5,0 g/1 for Trichoderma reesei and 5-10 g/1 of Aspergillus niger under the above described conditions. In the Examples we provide details of this process.<br>
In a suitable embodiment a recombinant Chrysosporium strain produces protein or polypeptide in at least the amount equivalent to the production in moles per liter of cellulase by the strain UV13-6 or C-19, and most preferably at least equivalent to or higher than that of the strain UV18-25 under the corresponding or identical conditions, i.e. their respective optimal cultivation conditions.<br>
We have also found that expression and secretion rates are exceedingly high when using a Chrysosporium strain exhibiting the mycelial morphology of strain UV 18-25 i.e. fragmented short mycelia. Thus a recombinant strain according to the invention will preferably exhibit such morphology. The invention however also covers non-recombinant strains or otherwise engineered strains of Chrysosporium exhibiting this novel and inventive characteristic. Also covered by the invention is a recombinant Chrysosporium strain in any of the embodiments described according to the invention further exhibiting reduced sporulation in comparison to CI, preferably below that of strain UV13-6, preferably below that of NG7C-I9, preferably below that of UV 18-25 under equivalent fermenter conditions. Also covered by the invention is a recombinant Chrysosporium strain in any of the embodiments described according to the invention further exhibiting at least the amount of protein production ratio to biomass in comparison to CI, preferably in comparison to that of any of strains UVI3-6, NG7C-19 and UV 18-25 under equivalent fermenter conditions. The invention however also covers non-recombinant strains or otherwise engineered strains of Chrysosporium exhibiting this novel and inventive characteristic as such or in combination with any of the other embodiments.<br>
Another attractive embodiment of the invention also covers a recombin'ant Chrysosporium strain exhibiting a viscosity below that of strain NG7C-] 9, preferably below that of UV18-25 under corresponding or identical fermenter conditions. The invention however also covers non-recombinant strains or otherwise engineered strains of Chrysosporium exhibiting this novel and<br><br>
inventive characteristic as such or in combination with any of the other embodiments. We have determined that the viscosity of a culture of UV18-25 is below 10 cP opposed to that of Trichoderma reesei being of the order 200-600 cP, with that of Aspergillus niger being of the order 1500-2000 cP under their respective optimal culture conditions at the end of fermentation. The        process used for such determination is provided in the examples.<br>
Viscosity can be assessed in many cases by visual monitoring. The fluidity of the substance can vary to such a large extent that it can be nearly solid, sauce-like or liquid. Viscosity can also readily be ascertained by Brookfield rotational viscometry, use of kinematic viscosity tubes, falling ball viscometer or cup type viscometer. The yields from such a low viscosity culture are higher than from the commercial known higher viscosity cultures per time unit and per cell.<br>
The processing of such low viscosity cultures according to the invention is advantageous in particular when the cultures are scaled up. The subject Chrysosporium strains with the low viscosity perform very well in cultures as large as up to 150,000 liter cultures. Thus any culture size up to 150,000 litres provides a useful embodiment of the invention. Any other conventional size of fermentation should be carried out well with the strains according to the invention. The reasoning behind this is that problems can arise in large scale production with the formation of aggregates that have mycelia that are too dense and/or are unevenly distributed. The media as a result cannot be effectively utilised during the culture thus leading to an inefficient production process in particular in large scale fermentations i.e. over 150,000 liters. Aeration and mixing become problematic leading to oxygen and nutrient starvation and thus reduced concentration of productive biomass and reduced yield of polypeptide during the culture and/or can result in longer fermentation times. In addition high viscosity and high shear are not desirable in commercial fermentation processes and in current commercial processes they are the production limiting factors. All these negative aspects can be overcome by the Chrysosporium host according to the invention which exhibits much better characteristics than Trichoderma reesei, Aspergillus niger and Aspergillus oryzae that are commercially used in this respect i.e. exhibits better protein production levels and viscosity properties and biomass figures.<br>
A Chrysosporium strain according to any of the above-mentioned embodiments of the invention, said strain further exhibiting production of one or more of the fungal enzymes selected from the carbohydrate-degrading enzymes, proteases, other hydrolases, oxidoreductase, and transferases mentioned above, is considered a particularly useful embodiment of the invention. The most interesting products are specifically cellulases, xylanases, pectinases, lipases and proteases. Also useful as embodiment of the invention however is a Chrysosporium strain exhibiting production of one or more fungal enzymes that exhibit neutral or alkaline optimal stability and/or activity, preferably alkaline optimal stability and/or activity, said enzyme being selected from carbohydrate-degrading    enzymes,    hydrolases    and    proteases,    preferably    hydrolases    and<br><br>
carbohydrate-degrading enzymes. In the case of non-recombinant Chrysosporium, such enzymes are suitably other than celluiase as disclosed in WO 98/15633. Enzymes of particular interest are xylanases, proteases, esterases, alpha galactosidases, beta-galactosidases, beta-glucanases and pectinases. The enzymes are not limited to the aforementioned. The comments vis-a-vis stability and activity elsewhere in the description are valid here also.<br>
The invention also covers a method of producing a polypeptide of interest, said method comprising culturing a host strain (e.g. fungal such as of the genera Chrysosporium, Aspergillus, Trichoderma, Hansenula, Mucor, p1chia, Neurospora, Tolypocladium, Rhitomucor, Fusarium, Penicillium or bacterial or other microbial) in any of the embodiments according to the invention under conditions permitting expression and preferably secretion of the polypeptide and recovering the subsequently produced polypeptide of interest.<br>
Where protein or polypeptide is mentioned, variants and mutants e.g. substitution, insertion or deletion mutants of naturally occurring proteins are intended to be included that exhibit the activity of the non-mutant. The same is valid vis-a-vis the corresponding nucleic acid sequences. Processes such as gene shuffling, protein engineering and directed evolution site directed mutagenesis and random mutagenesis are processes through which such polypeptides, variants or mutants can be obtained. US 5,223,409, US 5,780,279 and US 5,770,356 provide teaching of<br>
directed evolution. Using this process a library of randomly mutated gene sequences created for<br>
■<br>
example by gene shuffling via error prone PCR occurs in any cell type. Each gene has a secretion region and an immobilising region attached to it such that the resulting protein is secreted and stays fixed to the host surface. Subsequently conditions are created that necessitate the biological activity of the particular polypeptide. This occurs for a number of cycles ultimately leading to a final gene with the desired characteristics. In other words a speeded up directed process of evolution. US 5,763,192 also describes a process for obtaining DNA, RNA, peptides, polypeptides or protein by way of synthetic polynucleotide coupling stochastically generated sequences, introduction thereof into a host followed by selection of the host cell with the corresponding predetermined characteristic.<br>
Another application of the method of the present invention is in the process of "directed evolution", wherein novel protein-encoding DNA sequences are generated, the encoded proteins are expressed in a host cell, and those sequences encoding proteins having a desired characteristic are mutated and expressed again. The process is repeated for a number of cycles until a protein with the desired characteristics is obtained. Gene shuffling, protein engineering, error-prone PCR, site-directed mutagenesis, and combinatorial and random mutagenesis are examples of processes through which novel DNA sequences encoding exogenous proteins can be generated. U.S. patents 5,223,409, 5,780,279 and 5,770,356 provide teaching of directed evolution. See also Kuchner and Arnold, Trends in Biotechnology, 15:523-530 (1997); Schmidt-Dannert and Arnold, Trends in<br><br>
Biotech., 17:135-136 (1999); Arnold and Volkov, Curr. Op1rt. Chem. Biol., 3:54-59 (1999); Zhao et al-. Manual of Industrial Microbiology and Biotechnology, 2nd Ed., (Demain and Davies, eds.) pp. 597-604, ASM Press, Washington DC, 1999; Arnold and Wintrode, Encyclopedia of Bioprocess Technology: Fermentation, Biocatalysis, and Bioseparation, (Flickinger and Drew, eds.) pp. 971-987, John Wiley &amp; Sons, New York, 1999; and Minshull and Stemmer, Curr. Op1n, Chem, Biol. 3:284-290.<br>
An application of combinatorial mutagenesis is disclosed in Hu et a!.. Biochemistry. 1998 37:10006-10015. US 5,763,192 describes a process for obtaining novel protein-encoding DNA sequences by stochastically generating synthetic sequences, introducing them into a host, and selecting host cells with the desired characteristic. Methods for effecting artificial gene recombination (DNA shuffling) include random priming recombination (Z. Shao, et a!., Nucleic Acids Res., 26:681-683 (1998)), the staggered extension process (H. Zhao et al., Nature Biotech., 16:258-262 (1998)), and heteroduplex recombination (A. Volkov et al.. Nucleic Acids Res., 27:eI8 (1999)). Error-prone PCR is yet another approach (Song and Rhee, Appl. Environ. Microbiol. 66:890-894 (2000)).<br>
There are two widely-practised methods of carrying out the selection step in a directed evolution process. In one method, the protein activity of interest is somehow made essential to the survival of the host cells. For example, if the activity desired is a cellulase active at pH 8, a celtulase gene could be mutated and introduced into the host cells. The transformants are grown with cellulose as the sole carbon source, and the pH raised gradually until only a few survivors remain. The mutated cellulase gene from the survivors, which presumably encodes a cellulase active at relatively high pH, is subjected to another round of mutation, and the process is repeated until transformants that can grow on cellulose at pH 8 are obtained. Thermostable variants of enzymes can likewise be evolved, by cycles of gene mutation and high-temperature culturing of host cells (Liao et al., Proc. Natl. Acad Sci. USA 83:576-580 (1986); Giver et al., Proc. Natl. Acad. Sci. USA 95:12809-12813 (1998).<br>
An alternative to the massively parallel "survival of the fittest" approach is serial screening. In this approach, individual transformants are screened by traditional methods, such as observation of cleared or coloured zones around colonies growing on indicator media, colorimetric or fluorometric enzyme assays, immunoassays, binding assays, etc. See for example Joo et al., Nature 399:670-673 (1999), where a cytochrome P450 monooxygenase not requiring NADH as a cofactor was evolved by cycles of mutation and screening; May et al., Nature Biotech: 18:317-320 (2000), where a hydantoinase of reversed stereoselectivity was evolved in a similar fashion; and Miyazaki et al.,y. Mol. Biol. 297:1015-1026 (2000), where a thermostable subtilisin was evolved.<br>
Standard cloning and protein or polypeptide isolation techniques can be used to arrive at the required sequence information. Parts of known sequences can be used as probes to isolate other<br><br>
homologues"in other genera and strains. The nucleic acid sequence encoding a particular enzyme activity can be used to screen a Chrysosporium library for example. A person skilled in the art will realise which hybridisation conditions are appropriate. Conventional methods for nucleic acid hybridisation construction of libraries and cloning techniques are described in Sambrook et al (Eeds) (1989) In "Molecular Cloning. A Laboratory Manual" Cold Spring Harbor, Press Plainview, New York, and Ausubel et al (Eds) "Current Protocols in Molecular Biology" (1987) John Wiley and Sons, New York. The relevant information can also be derived from later handbooks and patents, as well as from various commercially available kits in the field.<br>
In an alternative embodiment, said method comprises culturing a strain according to the invention under conditions permitting expression and preferably secretion of the protein or polypeptide or precursor thereof and recovering the subsequently produced polypeptide and optionally subjecting the precursor to additional isolation and purification steps to obtain the polypeptide of interest. Such a method may suitably comprise a cleavage step of the precursor into the polypeptide or precursor of interest. The cleavage step can be cleavage with a Kex-2 like protease, any basic amino acid paired protease or Kex-2 for example when a protease cleavage site links a well secreted protein carrier and the polypeptide of interest. A person skilled in the art can readily find Kex-2-like protease sequences as consensus sequence details for such are available and a number of alternatives have already been disclosed e.g. furin.<br>
Suitably in a method for production of the polypeptide according to any of the embodiments of the invention the cultivation occurs at pH higher than 5, preferably 5-10, more preferably 6-9. Suitably in such a method the cultivation occurs at a temperature between 25-43 °C, preferably 30-40°C. The strain used in the method according to the invention is quite suitably a recombinant Chrysosporium strain or other fungal or non-fungal strain. The method according to the invention in such a case can further be preceded by the step of production of a recombinant strain according to the invention. The selection of the appropriate conditions will depend on the nature of the polypeptide to be expressed and such selection lies well within the realm of normal activity of a person skilled in the art.<br>
The method of production of a recombinant strain according to the invention is also part of the subject invention. The method comprises stably introducing a nucleic acid sequence encoding a heterologous or homologous polypeptide into a suitable host strain, said nucleic acid sequence being operably linked to an expression regulating region, said introduction occurring in a manner known per se for transforming filamentous fungi. As stated above numerous references hereof are available and a small selection has been cited. The information provided is sufficient to enable the skilled person to cany out the method without undue burden. The method comprises introduction of a nucleic acid sequence comprising any of the nucleic acid elements described in the various embodiments of the recombinant strain according to the invention as such or in combination.<br><br>
By way of example the introduction can occur using the protoplast transformation method. The method is described in the examples. Alternative protoplast or spheroplast transformation methods are known and can be used as have been described in the prior art for other filamentous fungi. Details of such methods can be found in many of the cited references and are thus incorporated by reference. A method according to the invention suitably comprises using a non-recombinant strain as starting material for introduction of the desired sequence encoding the polypeptide of interest.<br>
The subject invention also covers a method of producing Chrysosporium enzyme, said method comprising culturing a Chrysosporium or other strain in or on a cultivation medium at pH higher than 5, preferably 5-10, more preferably 6-9, suitably 6-7.5, 7.5-9 as examples of neutral and alkaline pH ranges.<br>
More in general the invention further covers a method of producing enzymes exhibiting<br>
neutral or alkaline optimal activity and/or stability, preferably alkaline optimal activity and/or<br>
stability. The preferred ranges vis-a-vis pH and optimal activity as well as assays with which to<br>
determine such have been provided elsewhere in the description. The enzyme should be selected<br>
from   carbohydrate-degrading   enzymes,   proteases,   other   hydrolases,   oxidoreductases,   and<br>
transferases, as described above, said method comprising cultivating a host cell transformed or<br>
transfected with the corresponding enzyme-encoding nucleic acid sequence. Suitably such an<br>
enzyme will be a Chrysosporium enzyme. A suitable method suth as this comprises production<br>
specifically of cellulase, xylanase, pectinase, lipase and protease, wherein cellulase and xylanase cleave P-l,4-bonds and cellulase comprises endoglucanase, cellobiohydrolase and p-glucosidase.<br>
The method according to the invention can comprise cultivating any Chrysosporium host according to the invention comprising nucleic acid encoding such aforementioned enzymes. Suitably the production of non-recombinant Chrysosporium hosts according to the invention is directed at production of carbohydrate degrading enzymes, hydrolases and proteases. In such a case the enzyme is suitably other than a cellulase. Methods of isolating are analogous to those described in WO 98/15633 and are incorporated by reference.<br>
The enzymes produced according to the invention are also covered by the invention. Enzymes of Chrysosporium origin as can be isolated from non-recombinant Chrysosporium strains according to the invention are also covered. They exhibit the aforementioned stability, activity characteristics. Suitably they are stable in the presence of LAS. In particular proteases with p1 4-9.5, proteases with a MW of 25-95 kD, xylanases with p1 between 4.0 and 9.5, xylanases with MW between 25 and 65 kD, endoglucanases with a p1 between 3.5 and 6.5, endoglucanases with MW of 25-55 kDa, β-glucosidases, a,β-galactosidases with a p1 of 4-4.5, β-glucosidases, a,β-galactosidases with a MW of 45-50 kDa, cellobiohydrolases of p1 4-5, cellobiohydrolases of MW 45-75 kDa, e.g. a MW of 55 kD and p1 4.4, polygalacturonases, with a p1 of 4.0-5.0 polygalacturonase of 60-70<br><br>
kDa, e.g. 65-kDa, esterases with a pi 4-5, and esterases with a MW of 95-105 kDa with the aforementioned stability, activity characteristics are claimed. The molecular weights (MW) are those determined by SDS-PAGE. The non-recombinant i.e. natively occurring enzyme is other than cellulase as disclosed in WO 98/15633. Enzymes with combinations of the pi values and molecular weights mentioned above are also covered.<br>
The invention is also concerned with the (over)production of non-protein products by the mutant (recombinant) strains of the invention. Such non-protein products include primary metabolites such as organic acids, amino acids, and secondary such as antibiotics, e.g. penicillins and cephalosporins and other therapeutics. These products are the result of combinations of biochemical pathways, involving several fungal genes of interest. Fungal primary and secondary metabolites and procedures for producing these metabolites in fungal organisms are well known in the art. Examples of the production of primary metabolites have been described by Mattey M., The Production of Organic Acids, Current Reviews in Biotechnology, 12, 87-132 (1992). Examples of the production of secondary metabolites have been described by Penalva et al. The Optimization of Penicillin Biosynthesis in Fungi, Trends in Biotechnology 16, 483-489 (1998).<br>
EXAMPLES<br>
EXAMPLES OF TRANSFORMATION COMPARING CHRYSOSPORIUM. TRICHODERMA AND TOLYPOCLADIUM GEODES<br>
Two untransformed Chrysosporium CI strains and one Trichoderma reesei reference strain were tested on two media (Gs pH 6,8 and Pridham agar, PA, pH 6,8), To test the antibiotic resistance level spores were collected from 7 day old PDA plates. Selective plates were incubated at 32°C and scored after 2,4 and 5 days. It followed that the C-1 strains NG7C-19 and UV18-25 clearly have a low basal resistance level both to phleomycin and hygromycin. This level is comparable to that for a reference T reesei commonly used laboratory strain. Thus there is clear indication these two standard fungal selectable markers can be used well in Chrysosporium strains. Problems with other standard fungal selectable markers should not be expected.<br>
Selection of Sh-ble (phleomycin-resistance) transformed Chrysosporium strains was succesfully carried out at 50 µg/ml. This was also the selection level used for T reesei thus showing that differential selection can be easily achieved in Chrysosporium, The same comments are valid for transformed strains with hygromycin resistance at a level of 150 µg/ml.<br>
The protoplast transformation technique was used on Chrysosporium based on the most generally applied fungal transformation technology. All spores from one 90mm PDA plate were recovered in 8ml ICl and transferred into a shake flask of 50ml ICl medium for incubation for 15 hours at 35°C and 200 rpm. After this the culture was centrifuged, the pellet was washed in MnP, brought back into solution in 10ml MnP and lOmg/ml Caylase C3 and incubated for 30 minutes at<br><br>
kDa, e.g. 65'kDa, esterases with a p1 4-5, and esterases with a MW of 95-105 kDa with the aforementioned stability, activity characteristics are claimed. The molecular weights (MW) are those determined by SDS-PAGE. The non-recombinant i.e. natively occurring enzyme is other than cellulase as disclosed in WO 98/15633. Enzymes with combinations of the p1 values and molecular weights mentioned above are also covered.<br>
The invention is also concerned with the (over)production of non-protein products by the mutant (recombinant) strains of the invention. Such non-protein products include primary metabolites such as organic acids, amino acids, and secondary such as antibiotics, e.g. penicillins and cephalosporins and other therapeutics. These products are the result of combinations of biochemical pathways, involving several fungal genes of interest. Fungal primary and secondary metabolites and procedures for producing these metabolites in fungal organisms are well known in the art. Examples of the production of primary metabolites have been described by Mattey M., The Production of Organic Acids, Current Reviews in Biotechnology, 12, 87-132 (1992). Examples of the production of secondary metabolites have been described by Penalva et al. The Optimization of Penicillin Biosynthesis in Fungi, Trends in Biotechnology 16, 483-489 (1998).<br>
EXAMPLES<br>
EXAMPLES OF TRANSFORMATION COMPARING CHRYSOSPORIUM, TRICH0DERM4 AND<br>
«<br>
TOLYPOCLADIUM GEODES<br>
Two untransformed Chrysosporium CI strains and one Trichoderma reesei reference strain were tested on two media (Gs pH 6,8 and Pridham agar, PA, pH 6,8). To test the antibiotic resistance level spores were collected from 7 day old PDA plates. Selective plates were incubated at 32°C and scored after 2,4 and 5 days. It followed that the C-1 strains NG7C-19 and UV18-25 clearly have a low basal resistance level both to phleomycin and hygromycin. This level is comparable to that for a reference T reesei commonly used laboratory strain. Thus there is clear indication these two standard fungal selectable markers can be used well in Chrysosporium strains. Problems with other standard fungal selectable markers should not be expected.<br>
Selection of Sh-ble (phleomycin-resistance) transformed Chrysosporium strains was succesfully carried out at 50 µg/ml. This was also the selection level used for T reesei thus showing that differential selection can be easily achieved in Chrysosporium, The same comments are valid for transformed strains with hygromycin resistance at a level of 150µg/ml.<br>
The protoplast transformation technique was used on Chrysosporium based on the most generally applied fungal transformation technology. All spores from one 90mm PDA plate were recovered in 8ml ICl and transferred into a shake flask of 50ml ICl medium for incubation for 15 hours at 35°C and 200 rpm. After this the culture was centrifuged, the pellet was washed in MnP, brought back into solution in 10ml MnP and lOmg/ml Caylase C3 and incubated for 30 minutes at<br><br>
35°C with agitation (150 rpm).<br>
The solution was filtered and the filtrate was subjected to centrifugation for 10 minutes at 3500 rpm. The pellet was washed with 10 ml MnPCa2+. This was centrifuged for 10 minutes at 25°C. Then 50 microlitres of cold MPC was added. The mixture was kept on ice for 30 minutes whereupon 2,5 ml PMC was added. After 15 minutes at room temperature 500 microlitres of the treated protoplasts were mixed to 3 ml of MnR Soft and immediately plated out on a MnR plate containing phleomycin or hygromycin as selection agent. After incubation for five days at 30°C transformants were analysed (clones become visible after 48 hours). Transformation efficiency was determined using 10 microgrammes of reference plasmid pAN8-l . The results are presented in the following Table 1.<br><br>
The Chrysosporium transformant's viability is superior to that of Trichoderma. The transformability of the strains is comparable and thus the number of transformanls obtained in one experiment lies 4 times higher for Chrysosporium than for 7. reesei. Thus the Chrysosporium transformation system not only equals the commonly used T. reesei system, but even outperforms it. This improvement can prove especially useftil for vectors that are less transformation efficient than pAN8-l. Examples of such less efficient transformation vectors are protein carrier vectors for production of non-fungal proteins which generally yield 10 times fewer transformants.<br>
A number of other transformation and expression vectors were constructed with homologous Chrysosporium protein encoding sequences and also with heterologous protein encoding sequences for use in transformation experiments with Chrysosporium,<br>
Examples of expression systems include a Chrysosporium xylanase Xyll promoter fragment linked to a xylanase signal sequence in frame with a xylanase open reading frame followed by a xylanase terminator sequence. Transformant selection is carried out by using cotransformation with a selectable vector.<br>
Another example is a Chrysosporium lucknowense cellobiohydrolase promoter linked to Penicillium endoglucanase 3 signal sequence in frame with the Penicillium endoglucanase 3 open reading frame  followed by the Chrysosporium cellobiohydrolase terminator sequence. In addition<br><br>
this vector carries a second expression cassette with a selection marker i.e. the aceetamidase S gene (AmdS gene).<br>
A further example comprises Chrysosporium glyceraldehyde-3-phosphate dehydrogenase 1 promoter linked to the Aspergillus niger glucoamylase signal sequence and the glucoamylase open reading frame fused to the human Interleukine 6 open reading frame. In adddition this vector carries a second expression cassette with a selection marker i.e. the AmdS gene.<br>
A still further example is a Aspergillus nidulans glyceraldehyde-3-phosphate dehydrogenase A promoter linked to the endoglucanase 5 open reading frame followed by a Aspergillus nidulans terminator sequence.<br>
EXAMPLES OF HETEROLOGOUS AND HOMOLOGOUS EXPRESSION OF CHRYSOSPORIUM TRANSFORMANTS<br>
CI strains (NG7C-19 and/or UV18-25) have been tested for their ability to secrete various heterologous proteins: a bacterial protein {Streptoalloteichus hindustanus phleomycin-resistance protein, Sh ble), a fungal protein {Trichoderma reesei xylanase II, XYN2) and a human protein (the human lysozyme, HLZ),<br>
CI secretion of Trichoderma reesei xylanase II (XYN2).<br>
CI strain UV18-25 has been transformed by the plasmids pUT1064 and pUT1065. pUT1064 presents the two following fungal expression cassettes:<br>
The first cassette allows the selection of phleomycin-resistant transformants:<br>
-	Neurospora crassa cross-pathway control gene 1 (cpc-I) promoter<br>
-	StreptoalloteichiuS hindustanus phleomycin-resistance gene Sh ble 'Aspergillus nidulans tryptophan-synthase (trpC) terminator<br>
The second cassette is the xylanase production cassette: 'T. reesei strain TR2 cbhl promoter 15<br>
'T. reesei strain TR2 xyn2 gene (including its signal sequence) 16 -T. reesei strain TR2 cbhl terminator 15<br>
The vector also carries an E. coli replication origin from plasmid pUC19   . The plasmid detailed map is provided in figure 1. pUT1065 presents the following fungal expression cassette:<br>
-A. nidulans glyceraldehyde-3-phosphate dehydrogenase (gpdA) promoter 2<br>
-	A synthetic T. reesei cellobiohydrolase I {cbhl) signal sequence 1,3<br>
-	S. hindustanus phleomycin-resistance gene Sh ble   used as carrier-protein<br>
-	A linker peptide (SGERX) featuring a KEX2-like protease cleavage site 1<br>
-	T reesei strain TKlxyn2 gene (without signal sequence) 16<br><br>
- A, nidulans tryptophan-synthase {trpC) terminator<br>
The vector also carries the beta-lactamase gene {bid) and an E. coli replication origin from plasmid pUC18 6. The plasmid detailed map is provided in figure 2. CI protoplasts were transformed with plasmid pUT1064 or pUT1065 following the same procedure already described in the above example. The fusion protein in plasmid pUT1065 (Sh ble :: XYN2) is functional with respect to the phleomycin resistance thus allowing easy selection of the CI transformants. Moreover, the level of phleomycin resistance correlates roughly with the level of xyn2 expression. In pUT1064, xyn2 was cloned with its own signal sequence.<br>
The xylanase production of CI transformants (phleomycin-resistant clones) was analysed by xylanase activity assay as follows: Primary transformants were toothpicked to GS+phleomycin (5µg/ml) plates (resistance verification) and also on XYLAN plates (xylanase activity detection by clearing zone visualisation 17). Plates were grown for 5 days at 32°C. Each validated clone was subcloned onto XYLAN plates. Two subclones per transformant were used to inoculate PDA plates in order to get spores for liquid culture initiation. The liquid cultures in 1C1+ 5g/l KPhtalate were grown 5 days at 27°C (shaking 180 rpm). Then, the cultures were centrifuged (5000g, lOmin.). From these samples, xylanase activity was measured by DNS technique according to Miller et al.<br><br>
These data show that:<br>
1)	Points 1 to 4 from example 2 are confirmed.<br>
2)	C1 can be used as host for the secretion of a heterologous fungal protein.<br>
Appendix to the Examples: Media Transformation media:<br>
Mandels Base:	MnP Medium :<br>
KH2PO4                                 2.0 g/l	Mandels Base with<br>
(NH4)2S04                             1.4 g/l	Peptone   1 g/l<br>
MgS04.7H20                       0.3 g/l	MES    2 g/l<br><br><br>
The regeneration media (MnR) supplemented with 50 |ig/ml phleomycin or 100-150 jig/ml hygromycin is used to select transformants. GS medium, supplemented with 5 µg/ml phleomycin is used to confirm antibiotic resistance.<br>
PDA is a complete medium for fast growth and good sporulation. Liquid media are inoculated with l/20th of spore suspension (all spores from one 90mm PDA plate in 5ml 0.1% Tween). Such cultures are grown at 27°C in shake flasks (200 rpm).<br>
ISOUTIONAND CHARACTERISATION OF CI GENES AND GENE EXPRESSION SEQUENCES<br>
OF CBHl XYLl, AND GPD<br>
Construction of a BlueSTAR gene library of UV18-25<br>
Chromosomal DNA of UV18-25 was partially digested with Saw3 A, fragments of 12-15 kb were<br><br>
isolated and ligated in a BamHl site of cloning vector BlueSTAR. Packaging of 20% of the ligation mixture resulted in a gene library of 4.6 xlO independent clones. This library was multiplied and stored at 4 °C and -80°C. The rest of the ligation mixture was also stored at 4°C.<br>
Screening the gene library of 17718-25 for isolation of the genes for cbhl, xyll and gpdl For the isolation of the different genes, in total ±7.5X10 individual BlueSTAR phages per probe were hybridised in duplo. Hybridisation was carried out with the PCR fragments of cbhl, and xyll (as described in WO 00/20555) at homologous conditions {65°C; 0.2xSSC) and with the gpdl gene of. niger at heterologous conditions (53°C; 0.5xSSC). The number of positive signals is given in Table 3. The positive clones were rescreened and for each clone two individual phages were used for further experiments. DNA of the different clones was analysed by restriction analysis to determine the number of different clones isolated from each gene (results are given in Table 3). As for each of the 3 genes, 4-6 different clones were isolated, we conclude that the primary gene library ( ±4-5 x104 clones) represents about 5x genome of UV18-25. From this result we conclude that the complete genome of UV 18-25 is represented in 9x103 clones. Based on an average genomic insert of 13 kb, this would indicate a genome size of ±120 Mb, which is 3 times the size of the Aspergillus genome.<br>
PCR reactions with specific primers for the gene present on the plasmid (based on previous sequence determination from the isolated PCR fragments) and the T7 and T3 primer present in the polylinker of pBlueSTAR we were able to determine the location of the genes in a number of clones. From each gene a plasmid was used for sequence determination of the gene.<br><br>
Table 3: Screening of 7.5x104 phages of the gene library of UV 18-25 with PCR fragments of UVI8-25 for the cbhJ gene and the xyJl gene (homologous conditions) and with the gpdA gene of A. niger (heterologous conditions). DNA isolation and restriction analysis was used to determine the number of different clones.<br>
Sequence analysis of the cloned genes<br>
For the cbhl, xyll, and the gpdJ gene, the results of the sequence determination are represented in SEQ ID No's 1, 3 and 5 respectively. Also the deduced amino acid sequences of the proteins are represented in these SEQ ID No's 2, 4 and 6. Some properties of the proteins are given in Table 4. It should be mentioned that the position of the start of the translation and the introns is based on homology with genes from the same family (i.e. paper genetics).<br><br>
CBHl<br>
From the amino acid sequences of CBHl, we concluded that the protein is about 63 kD in size and that a cellulose-binding domain (CBD) is present at the C-terminal part of the protein. Interestingly, no evidence was found for the presence of a CBD in the isolated 55 kD major protein. However, the presence of the isolated peptides from this 55 kD major protein in the encoded CBHl protein (SEQ ID No.l, 2), confirms that the 55 kD protein is encoded by the cloned gene. A possible explanation of these results is that the 55 kD protein is a truncated version of the CBHl protein<br>
lacking the CBD.<br>
The cellobiohydrolase CBHl has activity against MUF-cellobioside, MUF lactoside, FP and avicel,<br>
also against p-nitrophenyl β-glucoside, cellobiose and p-nitrophenyl lactoside. Its activity toward<br>
MUF cellobioside is inhibited by cellobiose with inhibition constant of 0.4 mM. The Michaelis constant toward MUF cellobioside was 0,14 mM, toward MUF lactoside was 4 mM and toward CMC was 3.6 g/1. The pH optimum is from 4.5 to 7. 50% of maximum activity toward CMC and 80% activity toward RBB-CMC is kept at pH 8. 70-80% activity within pH 5-8 is kept during 25 hours of incubation. The temperature optimum is 60-70°C. CBHl is a member of the cellobiohydrolase family 7. The corresponding CBH promoter, which is a preferred embodiment of the invention, is indicated in SEQ ID No. 1.<br>
Xvll<br>
From the amino acid sequences of Xyll we conclude that also here a CBD is present, in this protein at the N-terminal side (i.e. directly attached to or at less than 5 amino acids distance from the signal sequence). In the literature only few more xylanases with a CBD are known {Fusarium oxysporum, Humicola grisea and Neocallimastix patriciarum). The estimated size of the Xyll protein is 43 kD and several peptides isolated from a 30 kD xylanase originate from this protein (SEQ ID No. 3, 4). Several isolated peptides could not be found in the encoded sequence. This could indicate that alternative xylanase proteins are present in UV18-25. In previous analyses, no evidence was found for the presence of CBD in this 30 kD protein. Also from these results we hypothesized that the CBD of the protein is cleaved off by proteolysis. This hypothesis will be analysed further (by determination of activities, N-terminal sequences and sizes of the different proteins in the different CI strains: CI wild type, NG7C, UV13-6, UV18-25 and protease" mutants of UV18-25). Also the effect of the presence or absence of the CBD on enzymatic activities is analysed in more detail. Overexpression of the full length genes in various CI hosts may be considered. The presence of a cellulose-binding domain (CBD) is a particular feature of this enzyme. The only other known family 10 glycolytic enzyme (xylanase) having an N-terminal CBD is the Fusarium oxysporum XylF. The CBD of the Chrysosporium lucknowense Xyll protein has the sequence: WGQCG GQGWT GPTTC VSGAV COFVN DWYSQ CV (amino acids 22-53 of SEQ ID No. 4). This sequence does not comply to the CBD consensus sequence described in US 5,763,254 (Novo).<br><br>
This consensus sequence of US 5,763,254 is: WA^-G/A-Q-C-G G-Q/l/N-G/N-W/FA^-SAT/N/Q G-P/A/C-T/R/K-T/C/N-C X-X-G/P-SH^/F/L/A/- -T/K C-V/T/R/E/K-K/Q/A-Q/I-N Q/D/A-W/FA^-Y-Y/S/H/A-Q C-IA/Q/V/T, wherein WA" means either W or Y etc., X means any amino acid, and -means absent. Four differences with the most degenerate consensus sequence are present in Xyll, which are underlined in sequence 7 above. The invention thus pertains to xylanases having an N-terminal CBD different from this consensus CBD and other than the Fusarium oxysporum xylanase. More particularly the xylanase of the invention has a CBD having at least 55%, especially at least 65%, preferably at least 75% sequence identity with the sequence 7 above. Preferably the CBD contains one of the amino acids Phe, Tyr and Trp at position 23, or at least one of the four amino acids Val at position 20, Gin at position 22, Phe at position 23, and Val at position 24. Preferred sequences comprise Cys-Xaa-Phe, Xaa-Phe-Val, Cys-Xaa-Phe-Val, Cys-Gln-Phe, Val-Cys-Xaa-Phe, Gln-Phe-Val, Gln-Trp-Val, Gln-Tyr-Val, Val-Cys-Gln, Val-Cys-Gln-Phe, and Val-Cys-Xaa -Phe-Val, wherein Xaa is any amino acid or preferably Val, Thr, Arg, Glu, Gin or Lys, or most preferably Gin or Glu.<br>
The xylanase does not possess MUF cellobiase activity and is thus a true xylanase. It possesses high activity within a broad pH range from 5-8 maintaining 65% of maximum activity at pH 9-10; it is a member of the xylanase F family. The corresponding xylanase promoter, which is a preferred embodiment of the invention, is shown in SEQ ID No. 3. The Michaelis constant towards birch xylan is 4,2 g/1 for the 30kD xylanase. Temperature optimum was high and equal to 70°C for the xylanase.<br>
Gpdl<br>
The DNA sequence of the C-terminal part of the gpdl gene is not determined. The promoter sequences of this gene is a preferred embodiment of the present invention and is depicted in SEQ ID No. 5.<br>
The expression level of four Chrysosporium genes was studied by Northern analysis. Various strains of C. lucknowense were grown in rich medium containing pharmedia with cellulose and lactose (medium 1) or rich medium containing pharmedia and glucose (medium 2) at 33°C. After 48 h, mycelium was harvested and RNA was isolated. The RNA was hybridised with 4 different probes: EG5, EG6, Xyll and CBH. After exposure, the Northern blots were stripped and hybridised again with a probe for ribosomal L3 as a control for the amount of mRNA on the blot. Most strains showed very high response for CBH and high response for Xyll in medium 1; in medium 2, half of the strain showed high response for all genes, and the other half showed low response. The order of expression strength was deducted from these data as CBH &gt; Xyll &gt; EG5 &gt; EG6.<br>
The protein Xyll of C. lucknowense is 67% identical (72% homologous) to its closest homoiogue in the Genbank DATABASE (Table 4). The strong homology of the CBHl protein to its related Humicola grisea homoiogue (74% identical / 82% homologous) is noteworthy. It is also noted that in<br><br>
all cases th6 closest homologues originate from Fusarium, Humicola or other Pyrenomycetous (Sordariamycetous) fungi (Table 4), whereas Chrysosporium belongs to the Plectomycetous (Eurotio-mycetous) ftingi according to the NCBI database (Table 4). Thus the invention also pertains to glycanolytic enzymes, especially cellobiohydrolases and xylanases comprising a CBD, derived from plectomycetous fungi.<br><br>
Table 4: Structural and comparative data of CBHl, Xyll, and GPDl of the invention.<br>
Description of the Figures Figure 1 is a pUT1064 map Figure 2 is a pUTI065 map<br>
References (the contents hereof are incorporated)<br>
1.	Calmels T.P., Martin F., Durand H., and Tiraby G. {\99\) Proteolytic events in the processing<br>
of secreted proteins in fungi. J Biotechnol 17(1): p. 51-66.<br>
2.	Punt P.J., Dingemanse M.A., Jacobs-Meijsing B.J., Pouwels P.H., and van den Honde! C.A.<br>
(1988) Isolation and characterization of the glyceraldehyde-3-phosphate dehydrogenase gene of Aspergillus nidulans. Gene 69(1): p. 49-57.<br>
3.	Shoemaker S., Schweickart V., Ladner M., Gelfand D., Kwok S., Myambo K., and Innis M,<br>
(1983) Molecular cloning of exo-cellobiohydrolase I derived from Trichoderma reesei strain 127. Bio/Technology Oct:691-696.<br>
4.	Drocourt D., Calmels T., Reynes J.P., Baron M., and Tiraby G. (1990) Cassettes of the<br>
Streptoalloteichus hindustanus ble gene for transformation of lower and higher eukaryotes<br><br>
tophleomycin resistance. Nucleic Acids Res 18(13): p. 4009.<br>
5.	Mullaney E.J., Hamer J.E., Roberti K.A., Yelton M.M., and Timberlake W.E. (1985) Primary<br>
structure of the trpC gene Jrom Aspergillus nidulans. Mol Gen Genet 199(1): p. 37-45.<br>
6.	Yanisch-Perron C, Vieira J., and Messing J. (1987) Improved Ml3 phage cloning vectors and<br>
host strains: nucleotide sequences of the M13mpl8 andpUC19 vectors. Gene 33:103-119.<br>
7.	Durand H., Baron M., Calmels T., and Tiraby G. (1988) Classical and molecular genetics<br>
applied to Trichoderma reeseifor the selection of improved cellulolytic industrial strains, in Biochemistry and genetics of cellulose degradation, J.P. Aubert, Editor. Academic Press, p. 135-151.<br>
8.	Lowry O.H., Rosebrough N.J., Fair A.L., and Randall R.J. (1951) Protein measurements with<br>
the folinphenol reagent. J. Biol. Chem. 193: 265-275.<br>
9.	Parriche M., Bousson J.C., Baron M., and Tiraby G. Development of heterologous protein<br>
secretion systems in filamentous fungi, in 3rd European Conference on Fungal Genetics. 1996. Miinster, Germany.<br>
10.	Baron M., Tiraby G., Calmels T., Parriche M., and Durand H. (1992) Efficient secretion of<br>
human lysozyme fused to the Sh ble phleomycin resistance protein by the fungus Tolypocladium geodes. J Biotechnol 24(3): p. 253-266.<br>
11.	Jeenes D.J., Marczinke B., MacKenzie D.A., and Archer D.B. (1993)yi truncatedgluco-<br>
amylase gene fusion for heterologous protein secretion from Aspergillus niger. FEMS Microbiol. Lett. 107(2-3): p. 267-271. .12.  Stone P.J., Makoff A.J., Parish J.H., and Radford A. (1993) Cloning and sequence-analysis of the glucoamylase gene of neurospora-crassa. Current Genetics 24(3): p. 205-211.<br>
13.	Morsky P. (1983) Turbidimetric determination of lysozyme with Micrococcus lysodeikticus<br>
cells: Reexamination of reaction conditions. Analytical Biochem. 128:77-85.<br>
14.	Paluh J.L., Orbach M.J., Legerton T.L., and Yanofsky C. (1988) The cross-pathway control<br>
gene of Neurospora crassa, cpc-1, encodes a protein similar to GCN4 of yeast and the DNA-binding domain of the oncogene v-jun-encoded protein. Proc. Natl. Acad. Sci. USA 85(11): p. 3728-32.<br>
15.	Nakari T., Onnela M.L., Ilmen M., Nevalainen K., and Penttila M. (1994) Fungal promoters<br>
active in the presence of glucose, WO 94/04673, Alko.<br>
16.	Jorronen A., Mach R.L., Messner R., Gonzalez R., Kalkkinen N., Harkki A., and Kubicek C.P.<br>
(1992) The two major xylanases from Trichoderma reesei: characterization of both enzymes and genes. Biotechnology (N Y) 10(11): p. 1461-5.<br>
17.	Farkas V. (1985) Novel media for detection of microbial producers of cellulose and xylanase.<br>
FEMS Microbiol. Letters 28:137-140.<br>
18.	Miller G.L. (1959) Use of dinitrosalicylic acid reagent for determination of reducing sugar.<br>
Anal. Chem. 31:426-428.<br>
19.	Punt P.J., Mattem I.E., van den Hondel C.A.MJ.J. (1988)A vector for Aspergillus<br>
transformation conferring phleomycin resistance. Fungal Genetics Newsletter 35, 25-30.<br><br>
SEQ ID No. 1:  -<br>
DNA sequence and amino acid of complete Chrysosporium CBHl gene including promoter and terminator sequences. Promoter sequence (1-1779), terminator sequence (3427-4451) and intron sequence (2179-2256) are given in small case.<br>
aaggtatccgattcggggaacgtcgatgaaagtattgcaaaagtgacgagagttgcgcaa   60<br>
ctaactcgctgccgaagaagctgcggaagaaagagaacaccgaaagtggaataacgttac	120<br>
ggatgtcctgacctcaaagttgaaaccagcccttcctgctctatttgggaaagcggcttg	180<br>
cccttgaatgcgctgcactgtggcacgactaccagtgatcgggaggagcaaactaccctg	240<br>
gtccgttccttggtggggcggcactaggcccaacttagggtgatcggaggtcgatgccgc	300<br>
ggtcctcgttggtctgggctcttctcatttcccggtttgcaccccccgttgcacctgctg	360<br>
atcgcccgccaacgccgatgaggttgcgcccagaccgacaatcaccgcggctgcattccc	420<br>
aagtatattgaagatggcaccaggtacccggttttgcgtcccagtcgtttggtgccaaat	4 80<br>
ttgggagtttttgagcctcaagatctggggaaatcgacctcaacttccatacaagttaaa	540<br>
gtcgcacacacggcgagttccacgaagagacacatttttttctgaaggcctctctccccg	600<br>
cacatcagaaaccaccaaataccaagactgcagaagccggggtaagtgggccaccgggac	660<br>
tacactaaaatgcggggagaagcgagatccgttgcgaagggaagggatggggtgtgctgc	720<br>
•	•<br>
ggctttctccgctctcgtgcgccttttgcttgaatctagtgtacaccagggtaggctccg	780<br>
aaggagtatctacggcagcgctgttcgtgctgcgttgagagtcagggcggagacgagcag	8 40<br>
gcgacaggagcctcgcaccggcacttcggatcgcatttgcgcggagcgtcaaatacgctc	900<br>
ttctgcggtcatcagagagcatcgtgaaccaaggttcttccgcagggcggcctgggcttc	960<br>
gcagagtcgcactcggcggacgccttccgtgtcacccctgataacctggctgccgcgccc	1020<br>
agactcctccaatgaggtgtgtggttgccctcgccgacccttcagcaaccttaatcgctt	1080<br>
ccatcgcacggctccacgtcctcgaacgatgccctcagtccgtgcccggccgtggcaacc	114 0<br>
ataacgtgacatcgccgcccagcctactagccgctatcgaccggttaggcttgtcaccgc	1200<br>
agcgcccattctccatcgggcctctactctgatccacctcacccaccgcaagcactagcg	12 60<br>
agcctcaccagagtgcaagcgacacgacccgcttggcccttcgtccttgactatctccca	1320<br>
gacctcttgccatcttgccgacgccgcccccttttttttctcctccccctgccggcaggt	1380<br>
cggtggccccagtcccgagatggcattgctccgttgtccatgacgacccatcattcgatg	144 0<br>
gctgactggcacactcgtcttgtttgagcatcgacggcccgcggcccgtctcccacggta	1500<br>
cggaacctcgttgtacagtacctctcgtaatgatacccaacaccggggccgagcgctggg	1560<br>
agggcggcgttcccgagaagccgggaaggcggctggccggctgacctttgtgacttggcg	1620<br>
atggatgcggccatggagaatgtccgtccgaagcgacgcgacaattagcctggctaccat	1680<br><br>
cgatataaattgggtgattcccagctcttgatgggcgtgtcttctgcctggcagccctcg  1740<br>
tcttcagatcaagcaactgtgtgctgatcctcttccgccATGTACGCCAAGTTCGCGACC  1800<br>
M  Y  A  K  F  A  T<br>
CTCGCCGCCCTTGTGGCTGGCGCCGCTGCTCAGAACGCCTGCACTCTGACCGCTGAGAAC     18 60 LAALVAGAAAQNACTLTAEN<br>
CACCCCTCGCTGACGTGGTCCAAGTGCACGTCTGGCGGCAGCTGCACCAGCGTCCAGGGT     1920 HPSLTWSKCTSGGSCTSVQG<br>
TCCATCACCATCGACGCCAACTGGCGGTGGACTCACCGGACCGATAGCGCCACCAACTGC     1980 S      ITI      DANWRWTHRTDSATNC<br>
TACGAGGGCAACAAGTGGGATACTTCGTACTGCAGCGATGGTCCTTCTTGCGCCTCCAAG     204 0 YEGNKWDTSYCSDGPSCASK<br>
TGCTGCATCGACGGCGCTGACTACTCGAGCACCTATGGCATCACCACGAGCGGTAACTCC     2100 CCIDGADYSSTYGITTSGNS<br>
CTGAACCTCAAGTTCGTCACCAAGGGCCAGTACTCGACCAACATCGGCTCGCGTACCTAC     2160 LNLKFVTKGQYSTNIGSRTY<br>
CTGATGGAGAGCGACACCAAGTACCAGAgtaagttcctctcgcacccggccgccgggaga     2220 LMESDTKYQM<br>
tgatggcgcccagcccgctgacgcgaatgacacaGTGTTCCAGCTCCTCGGCAACGAGTT     2280<br>
FQLLGNEF<br>
CACCTTCGATGTCGACGTCTCCAACCTCGGCTGCGGCCTCAATGGCGCCCTCTACTTCGT  234 0 TFDVDVSNLGCGLNGALYFV<br>
GTCCATGGATGCCGATGGTGGCATGTCCAAGTACTCGGGCAACAAGGCAGGTGCCAAGTA  2 4 00 SMDADGGMSKYSGNKAGAKY<br>
CGGTACCGGCTACTGTGATTCTCAGTGCCCCCGCGACCTCAAGTTCATCAACGGCGAGGC  24 60 GTGYCDSQCPRDLKFINGEA<br>
CAACGTAGAGAACTGGCAGAGCTCGACCAACGATGCCAACGCCGGCACGGGCAAGTACGG  2 520 NVENWQSSTNDANAGTGKYG<br>
CAGCTGGTGCTCCGAGATGGACGTCTGGGAGGCCAACAACATGGCCGCCGCCTTCACTCC  2 58 0 SCCSEMDVWEANNMAAAFTP<br>
CCACCCTTGCACCGTGATCGGCCAGTCGCGCTGCGAGGGCGACTCGTGCGGCGGTACCTA  2 64 0 HPCWVIGQSRCEGDSCGGTY<br>
CAGCACCGACCGCTATGCCGGCATCTGCGACCCCGACGGATGCGACTTCAACTCGTACCG  2700 STDRYAGICDPDGCD  FNSY R<br>
CCAGGGCAACAAGACCTTCTACGGCAAGGGCATGACGGTCGACACGACCAAGAAGATCAC  27 60 QGNKTFYGKGMTVDTTKKIT<br>
GGTCGTCACCCAGTTCCTCAAGAACTCGGCCGGCGAGCTCTCCGAGATCAAGCGGTTCTA  2820 VVTQFLKNSAGELSEIKRFY<br>
CGTCCAGAACGGCAAGGTCATCCCCAACTCCGAGTCCACCATCCCGGGCGTCGAGGGCAA  28 80 VQNGKVI  PNSESTIPGVEGN<br>
CTCCATCACCCAGGACTGGTGCGACCGCCAGAAGGCCGCCTTCGGCGACGTGACCGACTT  294 0 S  ITQDWCDRQKAAFGDVTD?<br><br>
SEQ ID No-. 2:<br>
Amino acid of complete Chrysosporium CBHl protein. The putative signal peptide (1-19) is shown in italic letters and the cellulose binding domain (496-526) is shown in bold underlined letters.<br>
MYAKFATLAA  LVAGAAAQNA  CTLTAENHPS LTWSKCTSGG SCTSVQGSIT	50<br>
IDANWRWTHR TDSATNCYEG NKWDTSYCSD GPSCASKCCI DGADYSSTYG	100<br>
ITTSGNSLNL KFVTKGQYST NIGSRTYLME SDTKYQMFQL LGNEFTFDVD	150<br>
VSNLGCGLNG ALYFVSMDAD GGMSKYSGNK AGAKYGTGYC DSQCPRDLKF	200<br>
INGEANVENW QSSTNDANAG TGKYGSCCSE MDVWEANNMA AAFTPHPC7V	250<br>
IGQSRCEGDS CGGTYSTDRY AGICDPDGCD FNSYRQGNKT FYGKGMTVDT	300<br>
TKKITWTQF LKNSAGELSE IKRFYVQNGK VIPNSESTIP GVEGNSITQD	350<br>
WCDRQKAAFG DVTD7QDKGG MVQMGKALAG PMVLVMSIWD DHAVNMLWLD	400<br>
STWPIDGAGK PGAERGACPT TSGVPAEVEA EAPNSNVIFS NIRFGPIGST	450<br>
VSGLPDGGSG NPNPPVSSST PVPSSSTTSS GSSGPTGGTG VAKHYEQCGG	500<br>
IGFTGPTQCE SPYTCTKLND WYSQCL  *  526<br>
SEQ ID No. 3<br>
DNA sequence sequence of complete Chrysosporium Xyll gene including promoter and terminator sequences. Promoter sequence (1-969), terminator sequence (2428-3030(3028)) and intron sequences (1043-1116, 1181-1332(1331), 1596 (1595)-1674 (1672) are given in small case.<br>
tcatcaacttggcgtttggatgtactaatattacacgtcgtttgcnnagcggagtctgtg	60<br>
tcatctccgtggggtcgggtgctccagacgacgcttcgggccgatcctgaattcgggaag	12 0<br>
gaaacggttcggctaatcaggtcctctaaaatataacgaagcactacagagggagttcct	180<br>
cagaggacatcgtatcaaccgaagaacgaagcgccgaaaggactgatcaaaacaggagta	24 0<br>
ggtagggatgtgtgagtacctaaactttccatacctgacataaaatcatcatggtgcttc	300<br>
agacctgtttgatgaggcgagggcggaggccgcattgtattttcgttccttccttctttt	360<br>
tgttagtatatctnagggttccatcgtaaaatggaatcttccagctctactagtaattag	420<br>
aacaatagttctgatgtcgtgcgccaagctttttcagatgactgccaaaaacccatcatg	48 0<br>
ggtatggacaaaagcagtaatcggagtcacaacgccgcattttccttcatgatttccgtc	540<br>
aaccggagaggtcggaggaggactccggccacatgtgatgcgaagaagtacatggcgcca	600<br>
tggttctaacctcttatagtctgaaaatgcgcggaggccagcgaagccaagcccgggaac	660<br>
cgttcttgtcatggtttcagtattgtttcgctaaacattctatccgattcgcgataggtg	720<br>
cggctgccaccgaaggttgtatccttaaag.ctttggtaagtacggagtacggaaatggaa	7 80<br><br>
acgcgccgcagtcctggttccatcggtatcctccgcatgctccgccaaaaaaagaaaacc 84 0 cgggtatgtttacaaaggatataagagacaagatgcaccacccgcccccttcccatctgc 900 cggttgcccacgtcgccgtcgactgcttgtccgcttcctacctgcagcctctttcagaga  960<br>
ccatcaaacATGCGTACTCTTACGTTCGTGCTGGCAGCCGCCCCGGTGGCTGTGCTTGCC  1020<br>
MRTLTFVLAAAPVAVLA<br>
CAATCTCCTCTGTGGGGCCAGTgtatgtaattgccttactcggaaaatagtcaccactag  1080 QSPLWGQC<br>
agggacttaagctcactacttcctgtttcacaatagGCGGCGGTCAAGGCTGGACAGGTC  1140<br>
G     G     Q     G     ^/i     T     G<br>
CCACGACCTGCGTTTCtGGCGCAGTATGCCAATTCGTCAAgtcagtaactgcttttatt   1200 PTTCVSGAVCQFVN<br>
tcttttctctctgggattacgatttcgttttgcacttagcttggttctgcatttcattgt  12 60<br>
tgtattgttctctttttgtgtgtgagaggttttattaccacctaaaggccatttgctaac  1320<br>
aaatctccccagTGACTGGTACTCCCAATGCGTGCCCGGATCGAGCAACCCTCCTACGGG  138 0<br>
DWYSQCVPGSSNPPTG<br>
CACCACCAGCAGCACCACTGGAAGCACCCCGGCTCCTACTGGCGGCGGCGGCAGCGGAAC  14 4 0 TTSSTTGSTPAPTGGGGSGT<br>
CGGCCTCCACGACAAATTCAAGGCCAAGGGCAAGCTCTACTTCGGAACCGAGATCGATCA  1500 GLHDKFKAKGKLYFGTEIDH<br>
CTACCATCTCAACAACAATGCCTTGACCAACATTGTCAAGAAAGACTTTGGTCAAGTCAC  1560 YHLNNNALTNIVKKDFGQVT<br>
TCACGAGAACAGCTTGAAGTGGGATGCTACTGAGCgtgagtgacctctcctccttctccc  1620 HENSLKWDATEP<br>
gacaataatagataattacgagccggttcgaggctgacattgcgcgattctagCGAGCC  1680<br>
S  R<br>
GCAATCAATTCAACTTTGCCAACGCCGACGCGGTTGTCAACTTTGCCCAGGCCAACGGCA  17 4 0 NQFN  FANADAVVN  FAQANGK<br>
AGCTCATCCGCGGCCACACCCTCCTCTGGCACTCTCAGCTGCCGCAGTGGGTGCAGAACA  1800 LIRGHTLLWHSQLPQWVQNI<br>
TCAACGACCGCAACACCTTGACCCAGGTCATCGAGAACCACGTCACCACCCTTGTCACTC  18 60 NDRNTLTQVIENHVTTLVTR<br>
GCTACAAGGGCAAGATCCTCCACTGGGACGTCGTTAACGAGATCTTTGCCGAGGACGGCT  1920 YKGKILHWDVVNEI  FAEDG-S<br>
CGCTCCGCGACAGCGTCTTCAGCCGCGTCCTCGGCGAGGACTTTGTCGGCATCGCCTTCC  1980 LRDSVFSRVLGEDFVGIAFR<br>
GCGCCGCCCGCGCCGCCGATCCCAACGCCAAGCTCTACATCAACGACTACAACCTCGACA  2040 AARAADPNAKLYINDYNLDI<br>
TTGCCAACTACGCCAAGGTGACCCGGGGCATGGTCGAGAAGGTCAACAAGTGGATCGCCC  2100 ANYAKVTRGMVEKVNKWIAQ<br><br>
AGGGCATCCCGATCGACGGCATCGGCACCCAGTGCCACCTGGCCGGGCCCGGCGGGTGGA  2160 GIPI  DGIGTQCHLAGPGGWN<br>
ACACGGCCGCCGGCGTCCCCGACGCCCTCAAGGCCCTCGCCGCGGCCAACGTCAAGGAGA  2220 TAAGVPDALKALAAANVKE  I<br>
TCGCCATCACCGAGCTCGACATCGCCGGCGCCTCCGCCAACGACTACCTCACCGTCATGA  2280 AITELDIAGASANDYLTVMN<br>
ACGCCTGCCTCCAGGTCTCCAAGTGCGTCGGCATCACCGTCTGGGGCGTCTCTGACAAGG  2340 ACLQVSKCVGITVWGVSDKD<br>
ACAGCTGGAGGTCGAGCAGCAACCCGCTCCTCTTCGACAGCAACTACCAGCCAAAGGCGG  2400 SWRS  SSNPLLFDSNYQPKAA<br>
CATACAATGCTCTGATTAATGCCTTGTAAgaggaggtatattatttttagaggcaatgaa  24 60 YNALINAL*<br>
gctaggaggaaagaggggaagtgaggtaattagctaggacaggcaaatctagcagcaatt	2520<br>
ataagtcaacactatataaaatattcctataatggcttgtgcttcggtgtgcaaaaaaaa	2580<br>
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaactcaaaaacaaaaatgatccaacatgatt	2 640<br>
cgaaatggcgaccttgcaaatgcacacctcagataataccactatacaatacaccttaaa	2700<br>
tggcacctaaatccatttgtctgcggtcatagacggggcttaagaagcctgggatgcagg	2760<br>
tgtcgatgcaagggttacgtcagtgtatgatatgagtatgaaccatgctgtctgggtaat	2820<br>
tctccactttccctccccttacgactcttcgggtgtgcctctctagaaagtcgactcctg	2880<br>
gcgcctcagatcgccctttggctctgttcggtacaatgacgtccgctggtttcttccaaa	2 940<br>
gaccaggtatttctcccgtggcaacaaagaataccaaatacctatatcgaaccgtagtct	3000<br>
tctgataattagatgtctctcaaggcgcgg  3030<br>
SEQ ID No. 4<br>
Amino acid sequence of complete Chrysosporium Xyll protein. The signal peptide (1-20) is shown in italic letters and the cellulose binding domain (22-53) is shown in bold underlined letters,<br>
1	MRTLTFVLAA	APVAVLAQSP  LWCgCGGgGW TGPTTCVSGA VCQFVNDWYS<br>
51	QCVPGSSNPP	TGTTSSTTGS TPAPTGGGGS GTGLHDKFKA KGKLYFGTEI<br>
101	DHYHLNNNAL	TNIVKKDFGQ VTENSLKWDA TEPSRNQFNF ANADAVVNFA<br>
151	QANGKLIRGH	TLLWHSQLPQ WVQNINDRNT LTQVIENHVT TLVTRYKGKI<br>
201	LHWDWNEIF	AEDGSLRDSV FSRVLGEDFV GIAFRAARAA DPNAKLYIND<br>
251	YNLDIANYAK	VTRGMVEKVN KWIAQGIPID GIGTQCHLAG PGGWNTAAGV<br>
301	PDALKALAAA	NVKEIAITEL DIAGASANDY LTVMNACLQV SKCVGITVWG<br>
351	VSDKDSWRSS	SNPLLFDSNY QPKAAYNALI NAL*<br><br>
SEQ ID No-. 5<br>
DNA sequence of partial Chrysosporium GPDl gene including promoter sequences. Promoter sequence (1-1555) and intron sequence (1682-1781) are given in small case. The 3* end of the gene is lacking.<br>
tgagcagcaatgagcagcaatgagcattcctgggccaccgagtctgagtgccagtacgga  60<br>
gtatcgtacttcgtaccggggtttgatttggtgacggtgcttttcacctctcgatgcccg 120<br>
aaatcgggtctaagctgagtttgatcaaatatgtgactccaacatcgcccccttcggcaa 180<br>
accccgtcgacacgtgtgtcatccttccattgcaagcgatcactcgcagggcgtgacgat 2 40<br>
gaacgagatttttgcccggaccgattcgcggatatagcggcagccgaccagccctaccac 300<br>
actgatggccgtgtcactagtgtatgctcccagaaccgcaagcatacactgggcaatgct 3 60<br>
tggtatgcagttgaggcagctttatgtttccatacccttccacttcggctcggggactcg 420<br>
gcggggtcgcggaagtttgacggcagccgtcgggccttaggccgagattaccgtggttgt 4 80<br>
ggcccagttttagccgttcccgtccgtttcctaccggaccatgattttcgtgaaccattg 54 0<br>
caatcccgaagcgcatttccgacgttaaggagttacctccgctgcccagaattcatgatc 600<br>
gtggccggctcaaggcagcgtggcggggcatccgtgtcaagctcccaggaggaggtgcgc 660<br>
gatttcaaatccgggccaaaacaggccaagactggctggccaaaaaaaggagcgtagacg 720<br>
gcccgggacatcggacgtcagctcgcagccacccaaaaccggtccgatctactcgcttac 780<br>
tgtggtagttcaggtacttttgagtagtaaaaacgctacggcagggccggggggttcccc 840<br>
ggtgacggaggtgcctctgcggtggcgaacatcccacgcactctcgagctacggtgacac 900<br>
ctcgtgtcctgttggtcttgcaatgctggggcggcaggaaatgcgtcgcgctcctcccgg 960<br>
ccaagacctaaaacagacagcgccgcaaagtcgctcactagcaccgcgaaacgaagatgc 1020<br>
cccacctcaacgcaatctgtgatgcaagcaattgggaaggctcaccccacctcagcgagg 1080<br>
ggctcaaccatttttattatcagctcatgccaccacaacatgactgttttctttccttgc 1140<br>
tcatcccacatttgacaaaaatcgtcgattaatctctttccatacaggccgtccgcgctc 1200<br>
tgataaccacataaaagtctcttcagtcaacagctcaaagctccctcatccctccaggta 12 60<br>
agcagccaaagagctcccccacggaccccgcactgcctcatcccgcctgtatcggacctg 1320<br>
cgcgacccagcagagaatcccaaacctttgctgcttgctgcccggttccggactgagctg 1380<br>
caacccaagcctttaaaaagcttttcccttctcccacggtgtcaactctgtcctatccct 14 40<br>
ccgacatccgttgagctcaacaactccccgaaccttttaccccgcgccgagctacccctc 1500<br>
catcaaaccaccctgacagctcgctcactcacctccccacatcacagaaatcaaaATGAC 1560<br>
M  T  -<br>
TATCAAGGTCGGCATCAACGGTTTCGGCCGTATCGGCCGTATCGTCTTCCGCAACTCCAT 1620 IKVGINGFGRIGRIVFRNSI-<br>
CGAGCACTCGGATGTCGAGATCGTTGCCGTCAACGACCCCTTCATTGAGCCCAAGTACGC 1680<br><br>
EHS-DVEIVAVNDPFIEPKYA-<br>
TgtaagtagttttttttttccttcGtcgcgttctttcctgttccatcgacagtacgagat 1740<br>
GatcttgcaggcggatcggagctaaccgcgattgtcgtacagGAGTACATGCTCAAGTAT 1800<br>
E  Y  M  L  K  Y<br>
GACTCGACCCACGGTATCTTCAACGGCACCATCGCCGTCGAGGGCAACGACCTCATTGTC 18 60 DSTHGIFNGTIAVEGNDLIV<br>
AACGGCAAGAGGGTCAAGTTCTACACTGAGCGGGMCCCCGCCAACATTCCCTGGARGGAA 1920 NGKRVKFYTER?PANIPW?E<br>
ACTGGTGCCGAGTACATMRTCGAGTCGACCGGTGTGTTCACCAMCACCSAGAAGGCTAGC 1980 TGAEYI?ESTGVFT?T?KAS<br>
GCCCACCTCAAGGGCGGCGCCAAGCGCGTCATCATCTCTGCTCCCTCGGCCGATGCCCCC 2 04 0 AHLKGGAKRVIISAPSADAP<br>
ATGTACGTCATGGGCGTCAACGAGAAGACCTACGACGGCAAGGCCCAGGTCATCTCTAAC 210 0 MYVMGVNEKTYDGKAQVISN<br>
GCCTCGTGCACCACCAACTGCCTGGCTCCCCTCGCCAAGGTCATCCACGACAAGTTCGGC 2160 ASCTTNCLAPLAKVIHDKFG<br>
CTCGTTGAGGGTCTCATGACCACCGTCCACTCCTACACTGCCACCCAGAAGACCGTCGAT 2220 LVEGLMTTVHSYTATQKTVD<br>
GGTCCCTCTGCCAAGGACTGGCGTGGTGGCCGTGGTGCTGCTCAGAACATCATCCCCAGC 2280 GPSAKDWRGGRGAAQNIIPS<br>
AGCACTGGCGCCGCCAAGGCCGTCGGCAAGGTCATCCCTGAGCTCAACGGCAAGCTCACC 2 34 0 STGAAKAVGKVIPELNGKLT<br>
GGCATGTCCCTCCGTGTCCCCACCCCCAACGTTTCCGTTGTCGACCTCACCTGCCGCCTC 24 00 GMSLRVPTPNVSVVDLTCRL<br>
GAGAAGGAGGCTACCTACGACGACATCAAGGCCGCCATCAAGGAGGCCGCCGCCGGCCCC 24 60 EKEATYDDIKAAIKEAAAGP<br>
CTCAAGGgtgagttatctggttcctttttttttttttggagaacgacacatgctgataaa 2520 L  K  G<br>
acccagGCATCCTCGACTACACTGAGG 2547 I  L  D ■ Y  T  E<br>
SEQ ID No. 6<br>
Amino acid of partial Chrysosporium GPDl protein (the C-terminus is lacking in the sequence available),<br>
MTIKVGINGF GRIGRIVFRN	SIEHSDVEIV AVNDPFIEPK YAEYMLKYDS<br>
THGIFNGTIA VEGNDLIVNG	KRVKFYTER? PANIPW?ETG AEYI?ESTGV<br>
FT?T?KASAH LKGGAKRVII	SAPSADAPMY VMGVNEKTYD GKAQVISNAS<br>
CTTNCLAPLA KVIHDKFGLV	EGLMTTVHSY TATQKTVDGP SAKDWRGGRG<br>
AAQNIIPSST GAAKAVGKVI	PELNGKLTGM SLRVPTPNVS VVDLTCRLEK<br>
EATYDDIKAA IKEAAAGPLK	GILDYTE<br><br><br><br><br>
CLAIMS<br>
1.	A protein corresponding to a Chrysosporium glycosyl hydrolase family 7, exhibiting at least 75% aminoacid identity [ as determined by BLAST algorithm ] with the amino acid sequence of SEQ ID No. 2 or a part thereof having at least 20 contiguous amino acids which are identical to the corresponding part of the amino acid sequence 1-246 or 394-526 of SEQ ID No. 2.<br>
2.	A protein corresponding to a Chrysosporium glycosyl hydrolase family 10, exhibiting at least 70% aminoacid identity [ as determined by BLAST algorithm ] with the amino acid sequence of SEQ ID No. 4 or a part thereof having at least 20 contiguous amino acids which are identical to the corresponding part of the amino acid sequence 1-133 or 252-383 of SEQ ID No. 4.<br>
3.	A protein corresponding to a Chrysosporium glycosyl hydrolase family 10, exhibiting at least 65% aminoacid identity [ as determined by BLAST algorithm ] with the amino acid sequence of SEQ ID No. 4 or a part thereof having at least 20 contiguous amino acids which are identical to the corresponding part of the amino acid sequence 1-383 of SEQ ID No. 4.<br>
4.	A a protein corresponding to a glycosyl hydrolase family 10 and comprising a cellulose-binding domain having at least 55% aminoacid identity with the amino acid sequence 22-53 and/or having at least 20 contiguous amino acids from amino acid sequence 22-53 of SEQ ID No. 4, and/or having one of the aminoacids F, W and Y at relative position 44.<br>
5.	A protein corresponding to a Chrysosporium glyceraldehyde phosphate dehydrogenase, exhibiting at least 86% amino acid identity [ as determined by BLAST algorithm ] with the partial amino acid sequence of SEQ ID No. 6 or a part thereof having at least 20 contiguous amino acids which are identical to the corresponding part of the amino acid sequence 1-277 of SEQ ID No. 6.<br>
6.	A process of hydrolysing p-glucosidic bonds, comprising the use of an enzyme according to claim 1.<br><br>
7.	A process of hydrolysing β-xylosidic bonds, comprising the use of an enzyme according to claim 2, 3 or 4.<br>
8.	A nucleic acid sequence encoding a protein according to any one of claims 1-5.<br>
9.	A nucleic acid sequence comprising at least 70% of the nucleotides contained in the 5'-non-coding region of the nucleic acid sequence of any one of SEQ ID No's 1, 3 and 5.<br>
10.	A nucleic acid construct comprising a nucleic acid expression-regulatory region derived from Chrysosporium, contained in the 5'-noncoding region of the nucleic acid sequence of any one of SEQ ID No's I, 3 and 5, operationally linked to a nucleic acid sequence encoding a polypeptide of interest.<br>
11.	A recombinant microbial strain, preferably a fungal strain, containing a nucleic acid sequence according to any one of claims 8-10, and capable of expressing the polypeptide encoded by the coding nucleic acid sequence.<br>
12.	A process of producing a polypeptide using a construct according to claim 10 or a microbial strain according to claim 11.<br>
13.	An oligonucleotide probe comprising at least 15 contiguous nucleotides of the nucleic acid sequence of any one of SEQ ID No's 1, 3 and 5, or its complement.<br>
14.	An oligonucleotide probe according to claim 13, wherein the probe is 20-50 nucleotides in length.<br>
15.	An oligonucleotide probe according to claim 13 or 14, wherein the probe is labelled with a detectable label.<br><br>
16.    A protein substantially as herein described with reference to the<br>
accompanying drawings. 17-    A process of producing a polypeptide substantially as herein<br>
described with reference to the accompanying drawings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtY2xhaW1zIGdyYW5kLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-claims grand.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-description(complete) filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpIGdyYW5kLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-description(complete) grand.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTgzNy1jaGUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1837-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="208980-method-and-apparatus-for-compensating-local-oscillator-frequency-error.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208982-digital-document-processing-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208981</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/1837/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>38/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Sep-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Nov-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SHRI. EMALFARB, Mark, Aaron</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>193 Spyglass Court Jupiter FL 33477.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>EMALFARB, Mark, Aaron</td>
											<td>193 Spyglass Court Jupiter FL 33477.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL2001/000301</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-04-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>00201343.1</td>
									<td>2000-04-13</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208981-a-nucleic-acid-construct-comprising-an-expression-regulating-nucleic-acid-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:20:41 GMT -->
</html>
