<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/208915-an-isolated-or-recombinant-polypeptide-having-endoprotease-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:05:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 208915:AN ISOLATED OR RECOMBINANT POLYPEPTIDE HAVING ENDOPROTEASE ACTIVITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED OR RECOMBINANT POLYPEPTIDE HAVING ENDOPROTEASE ACTIVITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>New subtilisin homologues (both nucleic acids and proteins) are provided. Compositions which include these new proteins, recombinant cells, shuffling methods involving the new homologues, antibodies to the new homologues, and methods of using the homologues are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Subdivision Variants<br>
COPYRIGHT NOTIFICATION<br>
Pursuant to 37 C.F.R. 1.71(e), Applicants note that a portion of this<br>
disclosure contains material which is subject to copyright protection. The copyright owner has no objection to the facsimile reproduction by anyone of the patent document or patent disclosure, as it appears in the Patent and Trademark Office patent file or records, but otherwise reserves all copyright rights whatsoever.<br>
CROSS REFERENCE TO RELATED APPLICATIONS<br>
The present application claims priority to and benefit of United States<br>
Provisional Application 60/194,143, filed April 3, 2000, the disclosure of which is incorporated herein in its entirety for all purposes.<br>
BACKGROUND OF THE INVENTION<br>
Sales of the serine protease subtilisin exceed $ 300 million annually,<br>
accounting for approximately 40% of the industrial enzyme market. For more than 30 years, proteases, including subtilisin, have been used as additives in laundry and other detergents. Subtilisin has a broad specificity for proteins that commonly soil clothing, including proteins found in blood, grass, soil and many food products.<br>
Initially isolated from the bacteria Bacillus subtilis, subtilisin has become one of the most intensively studied and extensively engineered proteins known to date. A wide variety of subtilisins have been identified, and the amino acid sequences of a number of these subtilisins have been determined.  In addition, structural investigations, including more than 100 crystal structures, have revealed that subtilisins share a common active site with other serine proteases, the Ser-His-Asp catalytic triad.<br>
Despite such studies, structural features correlating with specific functional properties remain to be elucidated. Indeed, due both to the lack of structural predictabihty and to the need to optimize multiple characteristics simultaneously, the task of protein engineering remains difficult.<br>
For example, in detergent applications, subtilisins are not only active under a variety of washing conditions, they are also stable in the presence of other detergent<br><br>
components and additives. Such additives may include, among other things, other enzymes such as cellulases, lipases and the like. Subtilisin should be stable in the presence.of effective concentrations of such enzymes, and at the same time must not result in the degradation (proteolysis) of these enzymes. The subtilisin selected for such an application should also be active under a variety of specific conditions such as high or low temperature, acid, neutral or alkaline pH, or the presence of such additives as bleaching agents. Mutations or alterations in the nucleotide or amino acid sequences which would provide these benefits are difficult to predict, and therefore difficult to engineer.<br>
Nonetheless, both random mutagenesis and targeted mutagenesis approaches have been applied to the goal of producing improved subtilisin homologues. However, attempts to develop proteases that are improved for multiple properties are hampered by the fact that random mutations are often deleterious, and attempts to rationally alter one property of an enzyme often disrupt other important existing characteristics (Patkar et al. (1998) Chem PHs Lipids 93:95; Smooched et al. (1995) Proc Natal AcadSci US A. (1995) 92:452).<br>
The present invention provides novel subtilisin homologues that are improved for a variety of specific properties including thermal stability, activity at low temperature, alkaline satiety as well as other desirable properties and combinations of properties. These subtihsins are useful in a variety of detergent and other industrial and commercial applications.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides novel submission homologues with improved<br>
characteristics and combinations of characteristics, including thermo tolerance (thermal stability), activity at alkaline, acid and/or neutral pH, activity at ambient temperatures and activity in organic solvents. In one aspect, the invention relates to isolated and recombinant nucleic acids corresponding to polynucleotides that are novel subtilisin homologues, encode novel subtilisin proteins, hybridize under highly stringent conditions to such novel subtilisin homologues or polynucleotides encoding novel subtilisin proteins, or are fragments thereof, encoding polypeptides with endow-protease activity.<br>
Embodiments of the invention include polynucleotides which include a subsequence corresponding to one or more sequence selected from SEQ ED N0:1 to SEQ<br><br><br><br>
active in organic solvents. Nucleic acids that encode endo-proteases with combinations of such desirable properties are also embodiments.<br>
Nucleic acids encoding thermotolerant endo-proteases incorporating SEQ ID NOs: 3, 7, 8,10, 12,14, 15,16, 18, 21 and 25 are embodiments of the invention. Sinailarly, nucleic acids encoding alkaline active endo-proteases incorporating the SEQ ID NOs: 1,17, 19, 22, 23, 24, 25, 26, 27 and 32 are embodiments of the invention. Nucleic acids encoding endo-proteases that are active in organic solvents, such as dimethylformamide (DMF) incorporating SEQ ID NOs: 2, 4, 5, 6, 11,13, 20, 29, 30 and 33 are also embodiments of the invention.<br>
Compositions containing two or more such nucleic acids or encoded polypeptides are a feature of the invention. In some cases, these compositions are libraries of nucleic acids, preferably containing at least 10 such nucleic acids. Compositions produced by digesting the nucleic acids of the invention with a restriction endonuclease, a DNAse or an RNAse are also a feature of the invention, as are compositions produced by incubating a nucleic acid of the invention with deoxyribonucleotide triphosphates and a nucleic acid polymerase, including thermostable nucleic acid polymerases.<br>
Another aspect of the invention is vectors incorporating a nucleic acid of the invention. Such vectors include plasmids, cosmids, phage, viruses, including chromosome integration vectors. In preferred embodiments, the vector is an expression vector. Cells transduced by such vectors, or which otherwise incorporate the nucleic acid of the invention are an aspect of the invention. In a preferred embodiment, the cells express a polypeptide encoded by the nucleic acid.<br>
Isolated or recombinant polypeptides encoded by the nucleic acids of the invention are another aspect of the invention. Similarly, polypeptides comprising the sequence elements of SEQ ID NO: 131 to SEQ ID NO:260 are an aspect of the invention. Such polypeptides are endo-proteases. Preferred embodiments include polypeptides that are endo-proteases with one or more properties selected from among: activity at ambient temperature, psychrophilic activity, thermotolerance or thermostability, activity at alkaline, acid and/or neutral pH, and activity in the presence of organic solvents, such as dimethylformamide (DMF). Certain embodiments are endo-proteases with combinations of desired properties. Other embodiments are endo-protease polypeptides with desired conditional properties, such as pH dependence, temperature dependence, dependence on<br><br>
ionic strength, activation by ligand binding, and inactivation by ligand binding. In some embodiments, the polypeptide has at least 70% sequence identity to at least one of SEQ ID NO: 131 to SEQ ID NO:260 over a comparison window of at least 20 contiguous amino acids. In other embodiments, the polypeptide as at least 80%, at least 90%, at least 95%, 96%, 97%, 98% or 99% sequence identity to at least one of SEQ ID N0:131 to SEQ ID NO:260. In other embodiments the polypeptide maintains sequence identity over a comparison window of at least 30, at least about 50, at least about 100, or at least about 150 amino acids of one or more of SEQ ID NO: 131 to SEQ ID NO:260.<br>
In some embodiments, the polypeptide has an improved endo-protease activity selected from among increased thermotolerance, increased activity at ambient temperature, increased activity at alkaline pH, increased activity at acid pH, increased activity at neutral pH, and increased activity in the presence of organic solvents, relative to the subtilisin homologue polypeptide corresponding to SEQ ID NO:261, which polypeptide has at least 70% sequence identity to at least one of SEQ ID NO: 131 to SEQ ID NO:260, over a comparison window of at least 20 contiguous amino acids. In some embodiments, the polypeptide has at least 80%, at least 90%, at least 95%, 96%, 97%, 98%, or 99% sequence identity to at least one of SEQ ID N0:131 to SEQ ID NO:260. In some embodiments, the polypeptide maintains sequence identity over a comparison window of at least about 30, at least about 50, at least about 100, or more amino acids. In an embodiment the polypeptide with an improved endo-protease activity comprises a sequence element selected from among SEQ ID NO: 131 to SEQ ID NO:260.<br>
Polypeptides 150 contiguous amino acids or greater in length that are encoded by a polynucleotide comprising SEQ ID N0:1 to SEQ ID NO:130, a polynucleotide encoding SEQ ID N0:131 to SEQ ID NO:260, or a polynucleotide sequence that hybridizes under highly stringent conditions to such a polynucleotide are embodiments of the invention. Such polypeptides exhibit endo-protease activity. In some embodiments, such polypeptides are at least about 250 amino acids, e.g., about 269 amino acids in length. Alternatively such polypeptides are at least about 350 amino acids in length, e.g., pre-pro peptides of about 380 amino acids in length.<br>
Furthermore, polypeptides of the invention with secretion and/or localization sequences are a feature of the invention, as are such polypeptides with purification sequences, including epitope tags, FLAG tags, polyhistidine tags, and GST fusions. Similarly, the polypeptides of the invention bearing a methionine at the N-<br><br>
terminus or having one or more modified amino acids, e.g., glycosylated, Legislated, famesylated, acetylated or biotinylated amino acids, are features of the invention.<br>
Compositions that include one or more polypeptide of the invention and a detergent are an aspect of the invention.<br>
Methods of producing the polypeptides of the invention by introducing the nucleic acids encoding them into cells and then expressing and recovering them from the cells or culture medium are a feature of the invention. In preferred embodiments, the cells expressing the polypeptides of the invention are grown in a bulk fermentation vessel. Polypeptides that are specifically bound by a polyclonal antisera that reacts against an antigen derived from SEQ ID NO: 131 to SEQ E) NO:260, but not to a naturally cocooning subtilisin polypeptide or a previously described the sequence of which was available in Embank as of April 3, 2000, as exemplified by P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146 as well as antibodies which are produced by administering an antigen derived from any one of SEQ ID NO: 131 to SEQ ID NO:260 and/or which bind specifically to such antigens and which do not specifically bind to a naturally occurring subtilisin polypeptide or a subtilisin polypeptide corresponding to one or more of, e.g., P29600, P41362, P29599, P27693. P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146 are all features of the invention.<br>
Another aspect of the invention relates to methods of producing novel subtilisin homologues by mutating or recombining, e.g., recursively recombining, the nucleic acids of the invention in vitro or in vivo. In an embodiment, the recursive recombination produces at least one library of recombinant subtilisin homologue nucleic acids. The libraries so produced are embodiments of the invention, as are cells comprising the libraries. Furthermore, methods of producing a modified subtilisin nucleic acid homologue by mutating a nucleic acid of the invention are embodiments of the invention. Recombinant and mutant subtilisin homologue nucleic acids produced by the methods of the invention are also embodiments of the invention.<br>
In addition, nucleic acids which are unique subsequences of SEQ ID NO:l to SEQ ID NO: 130, (as compared to anv subtilisin nucleic acid sequences available in<br><br>
GenBank, as of April 3,2000, as exemplified by, e.g., M65086, D13I57, S48754, ' AB005792, D29688, and M28537), or are unique subsequences of polypeptides selected from among SEQ ID NO: 131 to SEQ ID NO:260, (as compared to any subtilisin protein sequences available in GenBank, as of April 3, 2000, as exemplified by: P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29i42, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P4.0903, P23314, P23653, P33295, P42780, and P80146), or are target nucleic acids that hybridize to unique coding oligonucleotides that encode a unique subsequence in a polypeptide selected from SEQ ID NO: 131 to SEQ ID NO:260, and that are unique as compared to a polypeptide encoded by a sequence available in GenBank as of April 3, 2000 and extemporized by M65086, D13157, S48754, AB005792, D29688, and M28537, are all embodiments of the invention.<br>
The invention also provides computers, computer readable medium and integrated systems, including databases that are composed of sequence records including character strings corresponding to SEQ ID NOs: 1-260. Such integrated systems optionally include, one or more instruction set for selecting, aligning, translating, reverse-translating or viewing any one or more character strings corresponding to SEQ ID NOs:l-260, with each other and/or with any additional nucleic acid or amino acid sequence.<br>
BRIEF DESCRIPTION ON THE FIGURES<br>
Figure 1. The Amino Acid Sequences of Savinase®.<br>
Figure 2 A-C. Sequence diagrams illustrating putative motifs.<br>
DETAILED DISCUSSION<br>
Subtilisins (Butt et al. (1996) Adv Exp Med Boil 379:277; Rao et al.<br>
(1998)'J Biome Strict Den 15:1053) are commercially important serine endo-proteases whose broad specificity for peptide bonds and relative ease of production makes them highly valued for a range of applications including food and leather processing and as additives to laundry detergents for stain hydrolysis and volatilization. Because of their high value, subtilisins have been extensively studied, with over 100 crystal structures solved (Siezen et al. (1991) Protein Eng 4:719).<br>
The present invention provides novel subtilisin homologues with improved properties as well as combinations of properties. Among these properties are enhanced<br><br>
thermostability in high or low temperatures, stability and activity at high and low pH, and stability in organic solvents.<br>
DEFTNITIONS<br>
A "polynucleotide sequence" is a nucleic acid, e.g., DNA, RNA (which is a polymer of nucleotides (A,C,T,U,G, etc. or naturally occurring or artificial nucleotide analogues) or a character string representing a nucleic acid, depending on context. Either the given nucleic acid or the complementary nucleic acid can be determined from any specified polynucleotide sequence.<br>
Similarly, an "amino acid sequence" is a polymer of amino acids (a protein, polypeptide, etc.) or a character string representing an amino acid polymer, depending on context.<br>
A nucleic acid, protein or other component is "isolated" when it is partially or completely separated from components with which it is normally associated (other proteins, nucleic acids, cells, synthetic reagents, etc.), A nucleic acid or polypeptide is "recombinant" when it is artificial or engineered, or derived from an artificial or engineered protein or nucleic acid.<br>
A "subsequence" or "fragment" is any portion of an entire sequence, up to and including the complete sequence.<br>
Numbering of an amino acid or nucleotide polymer corresponds to numbering of a selected amino acid polymer or nucleic acid when the position of a given monomer component (amino acid residue, incorporated nucleotide, etc.) of the polymer corresponds to the same residue position in a selected reference polypeptide or polynucleotide. Unless otherwise specified, numbering is given with reference to the sequence of Savinase®, as provided in Figure 1.<br>
A vector is a composition for facilitating cell transduction by a selected nucleic acid, and/or expression of the nucleic acid in the cell. Vectors include, e.g., plasmids, cosmids, viruses, YACs, bacteria, poly-lysine, chromosome integration vectors, episomal vectors, etc.<br>
"Substantially an entire length of a polynucleotide or amino acid sequence" refers to at least 70%, generally at least 80%, or typically 90% or more of a sequence.<br><br>
As used herein, an "antibody" refers to a protein comprising one or more polypeptides substantially or partially encoded by immunoglobulin genes or fragments of immunoglobulin genes. The recognized immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon and mu constant region genes, as well as myriad immunoglobulin variable region genes. Light chains are classified as either kappa or lambda. Heavy chains are classified as gamma, mu, alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG, IgM, Ig, Ig and Age, respectively. A typical immunoglobulin (antibody) structural unit comprises a tetramer. Each tetramer is composed of two identical pairs of polypeptide chains, each pair having one "light" (about 25 kD) and one "heavy" chain (about 50-70 kD). The N-terminus of each chain defines a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. The terms variable light chain (VL) and variable heavy chain (VH) refer to these tight and heavy chains respectively. Antibodies exist as intact immunoglobulins or as a number of well characterized fragments produced by digestion with various peptidases. Thus, for example, pepsin digests an antibody below the disulfide linkages in the hinge region to produce F(ab)'2, a dimer of Fab which itself is a light chain joined to VH-Chi by a disulfide bond. The F(ab)'2 may be reduced under mild conditions to break the disulfide encage in the hinge region thereby converting the (Fab')2 dimer into an Fab' monomer. The Fab' monomer is essentially an Fab with part of the hinge region {see. Fundamental Immunology, 4  Edition,W.E. Paul (ed.), Raven Press, N.Y. (1998), for a more detailed description of other antibody fragments). While various antibody fragments are defined in terms of the digestion of an intact antibody, one of skill will appreciate that such Fab' fragments may be synthesized de novo either chemically or by utilizing recombinant DNA methodology. Thus, the term antibody, as used herein also includes antibody fragments either produced by the modification of whole antibodies or synthesized de novo using recombinant DNA methodologies. Antibodies include single chain antibodies, including single chain Fv (sofa) antibodies in which a variable heavy and a variable light chain are joined together (directly or through a peptide linker) to form a continuous polypeptide.<br>
A variety of additional terms are defined or otherwise characterized herein.<br><br>
POLYNUCLEOTIDES<br>
Subtilisin Homologue Sequences<br>
The invention provides isolated or recombinant subtilisin homologue<br>
polypeptides, and isolated or recombinant polynucleotides encoding the polypeptides.<br>
For convenience, comparisons are made to the subtilisin Savinase® and/or the<br>
polynucleotide encoding it.   The 380 amino acid Savinase® polypeptide consists of an<br>
111 amino acid pre-pro-peptide and the 269 amino acid mature subtilisin, which is<br>
released by autolytic cleavage following secretion and folding. The primary structure of<br>
the Savinase® polypeptide [GenBank accession no. P29600] is illustrated in Figure 1<br>
(and in sequence Hastings 261).<br>
Polynucleotides encoding the polypeptides of the invention were discovered in libraries of subtilisin related sequences. DNA fragments were cloned into a Bacillus expression vector to generate a library of "diversified" region clones, corresponding to amino acids 55 through 227 of the mature protein (as indicated in Fig. 1 in bole:J. Library members were screened for protease activity, and assayed for a variety of desirable characteristics, including thermal stability, alkaline stability and activity in orgarjc solvents.<br>
Briefly, small libraries, e.g., of 654 active clones in one exemplary trial, were tested for four properties: activity at 23'C, thermostability, solvent stability, and pH dependence. To characterize the library, colonies were grown on casein plates and protease activity was evaluated by the production of clearing halos. Active colonies were grower. to stationary phase in LB medium, and the secreted protease was recovered from the medium and diluted 100-200 fold for assay procedures. The protease samples were assayed under five different conditions: pHlO; pH5.5, pH7.5; pH7.5 with 35% DMF; and pHlO following heat treatment. .<br>
In each condition tested, clones were obtained that outperformed the commercially available subtilisin, Savinase®. The most dramatic increase in total activity was at pH 5.5, where progeny were obtained with a 2-4-fold greater activity than Saver- -.e®. More significant than improvements in single properties, however, are the   of desirable properties provided by the proteases of the present invention.<br>
In one set of assays, seventy-seven clones (12%) that poisoned as well or better r.an Savinase ® at 23°C and pH 10 were assayed for the additional properties of residue: activity in organic solvent and stability to heat treatment. Nucleic acids encoding<br><br>
proteases with up to three times more residual activity after heat treatment or up to 50% greater residual activity in 35% dimethylformamide (DMF) were obtained. In addition, many clones that produced proteases that were both more heat-stable and more active in organic solvent than Savinase® were also obtained. It will be appreciated that in addition to the properties described above, desirable properties such as psychrophilic activity (i.e., activity at low temperature), activity in the presence of compounds such as hypochlorite, supercritical carbon dioxide, etc., can be isolated from the present library.<br>
Thus, the present invention provides polynucleotide sequences encoding and polypeptide sequences corresponding to subtilisin homologues with one or more desirable properties such as increased thermotolerance, increased activity at ambient temperature, increased activity at alkaline pH, increased activity at acid pH, increased activity at neutral pH, increased activity in the presence of organic solvents, and the like, relative to Savinase®. In some instances, the improved property is a conditional activity, or conditional property. For example, properties that facilitate large scale preparation and/or purification often can be described as conditional activities. Subtilisin homologues with high activity at, e.g., pH 10 relative to pH 7, or with high activity at pH 7 relative to pHlO can be purified at the inactive pH, and then provided in compositions, e.g., detergents, cleaning fluids, with a pH permissive of the high activity, reducing autoproteolysis in the preparation process. Similarly, heat activated or cold activated subtilisin homologues, as well as subtilisin homologues activated by, e.g., reduced ionic strength (as by dilution of a composition of high ionic strength containing a subtilisin homologue) or by binding of a ligand, e.g., a component of a detergent, cleaning solution or cosmetic, can be isolated from among the sequences described herein, or derived therefrom according to the methods described herein.<br>
Exemplary recombinant, e.g., shuffled, nucleic acids which encode the diversified region of subtilisin homologue polypeptides having desirable properties or combinations of properties, or which can be screened to provide additional subtilisin homologues with these or other desirable properties, are provided in SEQ ID N0:1 to SEQ ID NO: 130, which encode the diversified region polypeptides identified herein as SEQ ID NO: 131 to SEQ ID NO:260. Under many circumstances, including the expression and screening procedures described herein, the diversified regions indicated in the sequence listings are expressed in the context of a mature subtilisin or pre-pro peptide.<br><br>
When expressed in the context of the mature subtilisin protein SEQ DD NO: 131 to SEQ ED NO:260 correspond to amino acids 55 -227, inclusive.<br>
Making Polynucleotides<br>
Polynucleotides and oligonucleotides of the invention can be prepared by<br>
standard solid-phase methods, according to known synthetic methods. Typically,<br>
fragments of up to about 100 bases are individually synthesized, then joined (e.g., by<br>
enzymatic or chemical ligation methods, or polymerase mediated recombination methods)<br>
to form essentially any desired continuous sequence. For example, the polynucleotides<br>
and oligonucleotides of the invention can be prepared by chemical synthesis using, e.g.,<br>
the classical phosphoramidite method described by Beaucage et al. (1981) Tetrahedron<br>
Letters 22:1859-69, or the method described by Matthews et al. (1984) EMBO J. 3: 801-<br>
05., e.g., as is typically practiced in automated synthetic methods. According to the<br>
phosphoramidite method, oligonucleotides are synthesized, e.g., in an automatic DNA<br>
synthesizer, purified, annealed, ligated and cloned in appropriate vectors.<br>
In addition, essentially any nucleic acid can be custom ordered from any of a variety of commercial sources, such as The Midland Certified Reagent Company (<a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="1a777968795a7576737d756934797577">[email&#160;protected]</a>), The Great American Gene Company (http://www.genco.com), Expressage Inc. (www.expressgen.com), Oberon Technologies Inc. (Alameda, CA) and many others. Similarly, peptides and antibodies can be custom ordered from any of a variety of sources, such as PeptidoGenic (<a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="05756e6c684566666b60712b666a68">[email&#160;protected]</a>), HTI Bio-products, inc. (http://www.htibio.com), BMA Biomedicals Ltd (U.K.), Bio.Synthesis, Inc., and many others.<br>
Certain polynucleotides of the invention may also be obtained by screening cDNA libraries (e.g., libraries generated by recombining homologous nucleic acids as in typical shuffling methods) using oligonucleotide probes which can hybridize to or PCR-amplify polynucleotides which encode the subtilisin homologue polypeptides and fragments of those polypeptides. Procedures for screening and isolating cDNA clones are well-known to those of skill in the art. Such techniques are described in, for example, Sambrook et al. (1989) infra, and Ausubel FM et al. (1989; supplemented through 1999 a. Some polynucleotides of the invention can be obtained by altering a naturally occurring backbone, e.g., by autogenesis or oligonucleotide shuffling. In other cases, such polynucleotides can be made by in silicone or oligonucleotide shuffling methods as described in the references cited below.<br><br>
As described in more detail herein, the polynucleotides of the invention include sequences which encode novel mature subtilisin homologues and sequences complementary to the coding sequences, and novel fragments of coding sequence and complements thereof. The polynucleotides can be in the form of RNA or in the fonn of DNA, and include mRNA, cRNA, synthetic RNA and DNA, and cDNA. The polynucleotides can be double-stranded or single-stranded, and if single-stranded, can be the coding strand or the non-coding (anti-sense, complementary) strand.   The polynucleotides optionally include the coding sequence of a subtilisin homologue (i) in isolation, (ii) in combination with additional coding sequence, so as to encode, e.g., a fusion protein, a pre-protein, a prepare-protein, or the like, (iii) in combination with non-coding. sequences, such as introns, control elements such as a promoter, a terminator element, or 5' and/or 3' untranslated regions effective for expression of the coding sequence in a suitable host, and/or (iv) in a vector or host environment in which the subtilisin homologue coding sequence is a heterogonous gene. Sequences can also be found in combination with typical compositional formulations of nucleic acids, including in the presence of carriers, buffers, adjuvants, excipients and the like.<br>
Using Polynucleotides<br>
The polynucleotides of the invention have a variety of uses in, for<br>
example: recombinant production (i.e., expression) of the subtilisin homologue<br>
polypeptides of the invention; as detergent components; in food processing; as<br>
immunizes; as diagnostic probes for the presence of complementary or partially<br>
complementary nucleic acids (including for detection of natural subtilisin coding nucleic<br>
acids; as substrates for further diversity generation, e.g., diversity generating reactions,<br>
such as shuffling reactions or mutation reactions, to produce new and/or improved<br>
subtilisin homologues, and the like.<br>
EXPRESSION OF POLYPEPTIDES<br>
In accordance with the present invention, polynucleotide sequences which encode novel mature subtilisin homologues, fragments of subtilisin proteins, related fusion proteins, or functional equivalents thereof, collectively referred to herein as "subtilisin homologue polypeptides," or, simply, ^'subtilisin homologues," are used in recombinant DNA molecules that direct the expression of the subtilisin homologue polypeptides in appropriate host cells, such as bacterial cells. Due to the inherent<br><br>
degeneracy of the genetic code, other nucleic acid sequences which encode substantially the same or a functionally equivalent amino acid sequence are also used to clone and express the subtilisin homologues.<br>
Modified Coding Sequences:<br>
As will be understood by those of skill in the art, it can be advantageous to<br>
modify a coding sequence to enhance its expression in a particular host. The genetic code<br>
is redundant with 64 possible codons, but most organisms preferentially use a subset of<br>
these codons. The codons that are utilized most often in a species are called optimal<br>
codons, and those not utilized very often are classified as rare or low-usage codons {see,<br>
e.g., Zhang SP et al. (1991) Gene 105:61-72).   Codons can be substituted to reflect the<br>
preferred codon usage of the host, a process sometimes called "codon optimization" or<br>
. "contouring for species codon bias."<br>
Optimized coding sequence containing codons preferred by a particular<br>
prokaryotic or eucharistic host {see also, Murray, E. et al. (1989) Nuc. Acids Res. leally-<br>
508) can be prepared, for example, to increase the rate of translation or to produce<br>
recombinant RNA transcripts having desirable properties, such as a longer half-life, as<br>
compared with transcripts produced from a non-optimized sequence.   Translation stop<br>
codons can also be modified to reflect host preference. For example, preferred stop<br>
codons for S. cerevisiae and mammals are UAA and UGA respectively. The preferred<br>
stop codon for monocotyledonous plants is UGA, whereas insects and E. coli prefer to<br>
use UAA as the stop codon (Dolphins ME et al. (1996) Nuc. Acids Res. 24: 216-218).<br>
The polynucleotide sequences of the present invention can be engineered<br>
in order to alter a subtilisin homologue coding sequence for a variety of reasons,<br>
including but not limited to, alterations which modify the cloning, processing and/or<br>
expression of the gene product. For example, alterations may be introduced using<br>
techniques that are well known in the art, e.g., site-directed mutagenesis, to insert new<br>
restriction sites, alter glycosylation patterns, change codon preference, introduce splice<br>
sites, etc.<br>
Vectors. Promoters and Expression Systems,<br>
The present invention also includes recombinant constructs comprising<br>
one or more of the nucleic acid sequences as broadly described above. The constructs<br>
comprise a vector, such as, a plasmids, a cosmic, a phage, a virus, a bacterial artificial<br>
chromosome (BAC), a yeast artificial chromosome (YAC), or the like, into which a<br><br>
nucleic acid sequence of the invention has been inserted, in a forward or reverse orientation. In a preferred aspect of this embodiment, the construct further comprises regulatory sequences, including, for example, a promoter, operably honked to the sequence. Large numbers of suitable vectors and promoters are known to those of skill in the art, and are commercially available.<br>
General texts which describe molecular biological techniques useful herein, including the use of vectors, promoters and many other relevant topics, include Berger and Kimmel, Guide to Molecular Cloning Techniques. Methods in Enzymolosv volume 152 Academic Press, Inc., San Diego, CA (Berger); Sambrook et al., Molecular Cloning - A Laboratory Manual (2nd Ed.), Vol. 1-3, Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, 1989 ("Sambrook") and Current Protocols in Molecular Biology, F.M. Ausubel et al., eds., Current Protocols, a joint venture between Greene Publishing Associates, Inc. and John Wiley &amp; Sons, Inc., (supplemented through 1999) ("Ausubel"). Examples of protocols sufficient to direct persons of skill through in vitro amplification methods, including the polymerase chain reaction (PCR) the Hags chain reaction (LCR), Qp-replicase amplification and other RNA polymerase mediated techniques (e.g., NASBA), e.g., for the production of the homologous nucleic acids of the invention are found in Berger, Sambrook, and Ausubel, as well as Mules et al. (1987) U.S. Patent No. 4,683,202; PCR Protocols A Guide to Methods and Applications Obtains et al. eds) Academic Press Inc. San Diego, CA (1990) (Innis); Amheim &amp; Levinson (October 1, 1990) C&amp;EN 36-47; The Journal Of NIH Reecho (1991) 3:81-94; Kwoh et al. (1989) Proc. Natl. Acad. Sci. USA 86:1173; Gauchely et al. (1990) Proc. Natl. Acad. Sci. USA 87:1874; Lomell et al, (1989) J. Clin. Chem 35:1826; Landegren et al. (1988) Science 241:1077-1080; Van Brunt (1990) Biotechnology 8:291-294; Wu and Wallace (1989) Gene 4:560; Arranger et al. (1990) Gene 89:117, and Spokane and Maize (1995) Biotechnology 13:563-564. Improved methods for cloning in vitro amplified nucleic acids are described in Wallace et al., U.S. Pat. No. 5,426,039. Improved methods for amplifying large nucleic acids by PCR are summarized in Cheng et al. (1994) Nature 369:684-685 and the references cited therein, in which PCR amp icons of up to 40kb are generated. One of skill will appreciate that essentially any RNA can be converted into a double stranded DNA suitable for restriction digestion, PCR expansion and sequencing using reverse transcripts and a polymerase. See, e.g., Ausubel, Sambrook and Berger, all supra.<br><br>
The present invention also relates to engineered host cells that are transduced (transformed or transfected) with a vector of the invention (e.g., an invention cloning vector or an invention expression vector), as well as the production of polypeptides of the invention by recombinant techniques. The vector may be, for example, a plasmid, a viral particle, a phage, etc., or a non-replicating vector, such as liposome’s, naked or conjugated DNA, DNA-microparticles, etc. The engineered host cells can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants, or amplifying the subtilisin homologue gene. Culture conditions, such as temperature, pH and the like, are those previously used with the host cell selected for expression, and will be apparent to those skilled in the art and in the references cited herein, including, e.g., Sambrook, Ausubel and Berger, as well as e.g., Freshney (1994) Culture of Animal Cells, a Manual of Basic Technique, third edition, Wiley- Liss, New York and the references cited therein.<br>
Subtilisin homologue proteins of the invention can be produced in non-animal cells such as plants, yeast, fungi, bacteria and the like. In addition to Sambrook, Berger and Ausubel, details regarding non-animal cell culture can be found in Payne et al. (1992) Plant Cell and Tissue Culture in Liquid Systems John Wiley &amp; Sons, Inc. New York, NY; Gamborg and Phillips (eds) (1995) Plant Cell Tissue and Oren Culture; Fundamental Methods Springer Lab Manual, Springer-Vela (Berlin Heidelberg New York) and Atlas and Parks (eds) The Handbook of Microbiological Media (1993) CRC Press, Boca Raton, FL.<br>
Polynucleotides of the present invention can be incorporated into any one of a variety of expression vectors suitable for expressing a polypeptide. Suitable vectors include chromosomal, no chromosomal and synthetic DNA sequences, e.g., derivatives of SV40; bacterial plasmids; phage DNA; baculovirus; yeast plasmids; vectors derived from combinations of plasmids and phage DNA, viral DNA such as vaccine, adenovirus, fowl pox virus, pseudorabies, adenovirus, admen-associated virus, retroviruses and many others. Any vector that transduces genetic material into a cell, and, if replication is desired, which is replicable and viable in the relevant host can be used.<br>
When incorporated into an expression vector, the invention polynucleotide is operatively linked to an appropriate transcription control sequence (promoter) to direct mRNA synthesis. Examples of such transcription control sequences include: LTR or SV40 promoter, E. coli lac or tip promoter, phage lambda PL promoter, and other<br><br>
promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses. This invention expression vector, optionally contains a ribosome binding site for translation initiation, and a transcription terminator. The vector also optionally includes appropriate sequences for amplifying expression, e.g., an enhancer. In addition, the expression vectors of the present invention optionally contain one or more selectable marker genes to provide a phenotypic trait for selection of transformed host cells, such as dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in E. coil<br>
Vectors of the present invention can be employed to transform an appropriate host to permit the host to express an invention protein or polypeptide. Examples of appropriate expression hosts include: bacterial cells, such as E. coli, 5. subtilis, Streptomyces, and Salmonella typhimurium; fungal cells, such as Saccharomyces cerevisiae, Picric pastoris, sad Neuj-ospora crass\ insect cells such as Drosophila’s and Spodopterafrugiperda', mammalian cells such as CHO, COS, BHK, HEK 293 or Bowes melanoma; plant cells, etc. It is understood that not all cells or cell Tunes need to be capable of producing fully functional subtilisin homologues; for example, antigenic fragments of an submission homologue may be produced. The invention is not limited by the host cells employed.<br>
In bacterial systems, a number of expression vectors may be selected depending upon the use intended for the subtilisin homologue. For example, when large quantities of subtilisin homologue or fragments thereof are needed for commercial production or for induction of antibodies, vectors which direct high level expression of fusion proteins that are readily purified can be desirable. Such vectors include, but are not limited to, multifunctional E, coli cloning and expression vectors such as BLUESCRIPT (Stratagene), in which the subtihsin homologue coding sequence may be ligated into the vector in-frame with sequences for the amino-terminal Met and the subsequent 7 residues of beta-galactosidase so that a hybrid protein is produced; pen vectors (Van Heike &amp; Schuster (1989) J Boil Chem 264:5503-5509); pet vectors (Novae, Madison WI); and the like.<br>
Similarly, in the yeast Saccharomyces cerevisiae a number of vectors containing constitutive or inducible promoters such as alpha factor, alcohol oxidase and PGH may be used for production of the subtilisin homologue polypeptides of the<br><br>
invention. For reviews, see Ausubel et al. {supra) and Grant et al. (1987) Methods in Enzymologist 153:516-544).<br>
In mammalian host cells, a variety of expression systems, including viral-based systems, may be utilized. In cases where an adenovirus is used as an expression vector, a coding sequence, e.g., of a subtilisin homologue polypeptide, is optionally ligated into an adenovirus transcription/translation complex consisting of the late promoter and tripartite leader sequence. Insertion of a subtilisin polypeptide coding region into a nonessential El or E3 region of the viral genome will result in a viable virus capable of expressing subtilisin homologue in infected host cells (Logan and Sheen (1984) Proc NaCl Acids Sci USA 81:3655-3659). In addition, transcription enhancers, such as the routs sarcoma virus (RSV) enhancer, may be used to increase expression in mammalian host cells.<br>
Additional Expression Elements<br>
Specific initiation signals can aid in efficient translation of a subtilisin<br>
homologue coding sequence of the present invention. These signals can include, e.g., the<br>
ATG initiation codon and adjacent sequences. In cases where a subtilisin homologue<br>
coding sequence, its initiation codon and upstream sequences are inserted into an<br>
appropriate expression vector, no additional translational control signals may be needed.<br>
However, in cases where only coding sequence (e.g., a mature protein coding sequence),<br>
or a portion thereof, is inserted, exogenous transcriptional control signals including the<br>
ATG initiation codon must be provided. Furthermore, the initiation codon must be in the<br>
con-ect reading frame to ensure transcription of the entire insert. Exogenous<br>
transcriptional elements and initiation codons can be of various origins, both natural and<br>
synthetic. The efficiency of expression may be enhanced by the inclusion of enhancers<br>
appropriate to the cell system in use (Scharf et al. (1994) Results Probl CeU Differ<br>
20:125-62; Bittner et al. (1987) Methods in Enzyme. 153:516-544).<br>
Secretion/Localization Sequences<br>
Polynucleotides of the invention can also be fused, for example, in-frame<br>
to nucleic acids encoding a secretion/localization sequence, to target polypeptide<br>
expression to a desired cellular compartment, membrane, or organelle of a mammalian<br>
cell, or to direct polypeptide secretion to the periplasmic space or into the cell culture<br>
media. Such sequences are known to those of skill, and include secretion leader peptides,<br>
organelle targeting sequences (e.g., nuclear localization sequences, ER retention signals,<br><br>
mitochondrial transit sequences, chloroplast transit sequences), membrane localization/anchor sequences (e.g., stop transfer sequences, GPI anchor sequences), and the like.<br>
Expression Hosts<br>
In a further embodiment, the present invention relates to host cells<br>
containing the above-described constructs. The host cell can be a eukaryotic cell, such as<br>
a mammalian cell, a yeast cell, or a plant cell, or the host cell can be a prokaryotic cell,<br>
such as a bacterial cell. Introduction of the construct into the host cell can be effected by<br>
calcium phosphate transfection, DEAE-Dextran mediated transfection, electro oration, or<br>
other common techniques (Davis et al. (1986) Basic Methods in Molecular Biology).<br>
A host cell strain is optionally chosen for its ability to modulate the<br>
expression of the inserted sequences or to process the expressed protein in the desired<br>
fashion. Such modifications of the protein include, but are not limited to, acetylating,<br>
carboxy ation, glycosylation, phosphorylation, limitation and acylation. Post-translational<br>
processing which cleaves a "pre" or a "prepro" form of the protein may also be important<br>
for correct insertion, folding and/or function. Different host cells such as E. coli, Bacillus<br>
sp., yeast or mammalian cells such as CHO, HeLa, BHK, MDCK, 293, WI38, etc. have<br>
specific cellular machinery and characteristic mechanisms, e.g., for post-translational<br>
activities and may be chosen to ensure the desired modification and processing of the<br>
introduced, foreign protein.<br>
For long-term, high-yield production of recombinant proteins, stable<br>
expression systems can be used. For example, cell lines which stably express a<br>
polypeptide of the invention are transduced using expression vectors which contain viral<br>
origins of replication or endogenous expression elements and a selectable marker gene.<br>
Following the introduction of the vector, cells may be allowed to grow for a period<br>
determined to be appropriate for the cell type, e.g., 1-2 days for mammalian cell, 1 or<br>
more hours for bacterial cells, in an enriched media before they are switched to selective<br>
media. The purpose of the selectable marker is to confer resistance to selection, and its<br>
presence allows growth and recovery of cells which successfully express the introduced<br>
sequences. For example, resistant clumps of stably transformed cells can be proliferated<br>
using tissue culture techniques appropriate to the cell type.<br>
Host cells transformed with a nucleotide sequence encoding a polypeptide<br>
of the invention are optionally cultured under conditions suitable for the expression and<br><br>
recovery of the encoded protein from cell culture. The protein or fragment thereof produced by a recombinant cell may be secreted, membrane-bound, or contained intracellularly, depending on the sequence and/or the vector used. As will be understood by those of skill in the art, expression vectors containing polynucleotides encoding mature subtilisin homologues of the invention can be designed with signal sequences which direct secretion of the mature polypeptides through a prokaryotic or eukaryotic cell membrane.<br>
Additional Polypeptide Sequences<br>
Polynucleotides of the present invention may also comprise a coding<br>
sequence fused in-frame to a marker sequence which, e.g., facihtates purification of the<br>
encoded polypeptide. Such purification facilitating domains include, but are not limited<br>
to, metal chelating peptides such as histidine-tryptophan modules that allow purification<br>
on immobilized metals, a sequence which binds glutathione (e.g., GST), a hemagglutinin<br>
(HA) tag (corresponding to an epitope derived from the influenza hemagglutinin protein;<br>
' Wilson et al. (1984) Cell 37:767), maltose binding protein sequences, the FLAG epitope<br>
utilized in the FLAGS extension/affinity purification system (Immunex Corp, Seattle,<br>
WA), and the like. The inclusion of a protease-cleavable polypeptide linker sequence<br>
between the purification domain and the subtilisin homologue sequence is useful to<br>
facilitate purification. One expression vector contemplated for use in the compositions<br>
and methods described herein provides for expression of a fusion protein comprising a<br>
polypeptide of the invention fused to a polyhistidine region separated by an enterokinase<br>
cleavage site. The histidine residues facilitate purification on IlVnAC (immobilized metal<br>
ion affinity chromatography, as described in Porath et al. (1992) Protein Expression and<br>
Purification 3:263-281) while the enterokinase cleavage site provides a means for<br>
separating the subtilisin homologue polypeptide from the fusion protein. pGEX vectors<br>
(Promega; Madison, WI) may also be used to express foreign polypeptides as fusion<br>
proteins with glutathione S-transferase (GST). In general, such fusion proteins are<br>
soluble and can easily be purified from lysed cells by adsorption to ligand-agarose beads<br>
(e.g., glutathione-agarose in the case of GST-fusions) followed by elution in the presence<br>
of freeload.<br>
Polypeptide Production and Recovery<br>
Following transduction of a suitable host strain and growth of the host<br>
strain to an appropriate cell density, the selected promoter is induced by appropriate<br><br>
means (e.g., temperature shift or chemical induction) and cells are cultured for an additional period. Cells are typically harvested by centrifugation, disrupted by physical or chemical means, and the resulting crude extract retained for further purification. Microbial cells employed in expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical disruption, or use of cell lysing agents, or other methods,   which are well known to those skilled in the art.<br>
As noted, many references are available for the culture and production of many cells, including cells of bacterial, plant, animal (especially mammahan) and archebacterial origin. See, e.g., Sambrook, Ausubel, and Berger {all supra), as well as Freshney (1994) Culture of Animal Cells, a Manual of Basic Technique, third edition, Wiley- Liss, New York and the references cited therein; Doyle and Griffiths (1997) Mammalian Cell Culture: Essential Techniques John Wiley and Sons, NY; Humason (1979) Animal Tissue Techniques, fourth edition W.H. Freeman and Company; and Rjcciardelli, et al. (1989) In vitro Cell Dev. Biol. 25:1016-1024. For plant cell culture and regeneration, Payne et al. (1992) Plant Cell and Tissue Culture in Liquid Systems John Wiley &amp; Sons, Inc. New York, NY; Gamborg and Phillips (eds) (1995) Plant Cell Tissue and Organ Culture; Fundamental Methods Springer Lab Manual, Springer-Overflag (Berlin Heidelberg New York) and Plant Molecular Boiled (1993) R.R.D.Croy, Ed. Bios Scientific Publishers, Oxford, U.K. ISBN 0 12 198370 6. Cell culture media in general are set forth in Atlas and Parks (eds) The Handbook of Microbiological Media (1993) CRC Press, Boca Raton, EL. Additional information for cell culture is found in available commercial literature such as the Life Science Research Cell Culture Catalogue (1998) from Sigma-Aldrich, Inc (St Louis, MO) ("Sigma-LSRCCC") and, e.g., the Plant Culture Catalogue and supplement (1997) also from Sigma-Aldrich, Inc (St Louis, MO) ("Sigma-<br>
PCCS").<br>
Polypeptides of the invention can be recovered and purified from recombinant cell cultures by any of a number of methods well known in the art, including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography (e.g., using any of the tagging systems noted herein), hydroxylapatite chromatography, and lection chromatography. Protein refolding steps can be used, as desired, in completing the configuration of the mature protein. Finally, high performance Hquid chromatography (HPLC) can be employed in the final<br><br>
purification steps. Li addition to the references noted supra, a variety of purification methods are well known in the art, including, e.g., those set forth in Sandana (1997) Bioseparation of Proteins, Academic Press, Inc.; and Bourg et al. (1996) Protein Methods, 2°"  Edition Wiley-Liss, NY; Walker (1996) The Protein Protocols Handbook Humana Press, NJ, Harris and Angel (1990) Protein Purification Applications: A Practical Approach IRL Press at Oxford, Oxford, England; Harris and Angal Protein Purification Methods: A Practical Approach IRL Press at Oxford, Oxford, England; Scopes (1993) Protein Purification: Principles and Practice 3  Edition Springer Verlag, NY; Janson and Ryden (1998) Protein Purification: Principles. High Resolution Methods and Applications, Second Edition Wiley-VCH, NY; and Walker (1998) Protein Protocols on CD-ROM Humana Press, NJ.<br>
In some cases it is desirable to produce the subtilisin homologues of the invention in a large scale suitable for industrial and/or commercial applications. In such cases bulk fermentation procedui-es are employed. Briefly, polynucleotides comprising any one of SEQ ID N0:1 to SEQ ID NO: 130, or other nucleic acids encoding subtilisin homologues of the invention can be cloned into an expression vector. For example, U.S. Patent No. 5,955,310 to Wider et al. "METHODS FOR PRODUCING A POLYPEPTIDE IN A BACILLUS CELL," describes a vector with tandem promoters, and stabilizing sequences operably linked to a polypeptide encoding sequence. After inserting the polynucleotide of interest into a vector, the vector is transformed into a bacterial, e.g., a Bacillus subtilis strain PLISOIDE (maybe, apron, nor, sops]IE::Tn917) host. The introduction of an expression vector into a Bacillus cell may, for instance, be effected by protoplast transformation (see, e.g., Chang and Cohen (1979) Molecular General Genetics 168:111\ by using competent cells (see, e.g.. Young and Spizizin (1961) Journal of Bacteriology 81:823, or Dubai and Davidoff-Abelson’s (1971) Joum of Molecular Biology 56:209), by electroporation (see, e.g., Shigekava and Dower (1988) Biotechniques 6:742), or by conjugation (see, e.g., Koehler and Tome (1987) Joumal of Bacteriology 169:5271'). also Ausubel, Sarabrook and Berger, all supra.<br>
The transformed cells are cultivated in a nutrient medium suitable for production of the polypeptide using methods that are known in the art. For example, the cell may be cultivated by shake flask cultivation, small-scale or large-scale fermentation (including continuous, batch, fed-batch, or solid state femaentations) in laboratory or industrial feretory performed in a suitable medium and under conditions allowing the<br><br>
polypeptide to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium comprising carbon and nitrogen sources and inorganic salts, using procedures known in the art. Suitable media are available from commercial suppliers or may be prepared according to published compositions (e.g., in catalogues of the American Type Culture Collection). The secreted polypeptide can be recovered directly from the medium.<br>
The resulting polypeptide may be isolated by methods known in the art. For example, the polypeptide may be isolated from the nutrient medium by conventional procedures including, but not limited to, centrifugation, filtration, extraction, spray-drying, evaporation, or precipitation. The isolated polypeptide may then be further purified by a variety of procedures known in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures (e.g., preparative bioelectric focusing), differential solubility (e.g., ammonium sulfate precipitation), or extraction (see, e.g., Bollag et al. ri996) Protein Methods. T Edition Willed-Liss, NY; Walker (1996) The Protein Protocols Handbook Humana Press, NJ; Bollag et al, (1996) Protein Methods. 2""^ Edition Wiley-Liss, NY; Walker (1996) The Protein Protocols Handbook Humana Press, NJ).<br>
In vitro Expression Systems<br>
Cell-free transcription/ti-anslation systems can also be employed to<br>
produce polypeptides using DNAs or RNAs of the present invention. Several such<br>
systems are commercially available. A general guide to in vitro transcription and<br>
translation protocols is found in Tymms (1995) In vitro Transcription and Translation<br>
Protocols: Methods in Molecular Biology Volume 37, Garland Publishing, NY.<br>
(ix) Modified Amino Acids: Polypeptides of the invention may contain one or more modified amino acid. The presence of modified amino acids may be advantageous in, for example, (a) increasing polypeptide serum half-life, (b) reducing polypeptide antigenicity, (c) increasing polypeptide storage stability. Amino acid(s) are modified, for example, co-translationally or post-translation ally during recombinant production (e,g., N-linked glycosylation at N-X-S/T motifs during expression in mammalian cells) or modified by synthetic means.<br>
Non-limiting examples of a modified amino acid include a glycosylated amino acid, a sulfated amino acid, aprenlyated (e.g., famesylated, geranylgeranylated) amino acid, an acetylated amino acid, an acylated amino acid, a PEG-platted amino acid.<br><br>
a biotinylated amino acid, a carboxylated amino acid, a phosphorylated amino acid, and the like. References adequate to guide one of skill in the modification of amino acids are replete throughout the literature. Example protocols are found in Walker (1998) Protein Protocols on CD-ROM Human Press, Towpath, NJ.<br>
Use as Probes<br>
Also contemplated are uses of polynucleotides, also referred to herein as<br>
oligonucleotides, typically having at least 12 bases, preferably at least 15, more<br>
preferably at least 20, 30, or 50 or more bases, which hybridize under highly stringent<br>
conditions to an subtilisin homologue polynucleotide sequence described above. The<br>
polynucleotides may be used as probes, primers, sense and antisepses agents, and the like,<br>
according to methods as noted supra.<br>
SEQUENCE VARIATIONS<br>
Silent Variations<br>
It will be appreciated by those skilled in the art that due to the degeneracy<br>
of the genetic code, a multitude of nucleic acids sequences encoding subtilisin homologue<br>
polypeptides of the invention may be produced, some of which bear substantial identity to<br>
the nucleic acid sequences explicitly disclosed herein.<br><br><br>
For instance, inspection of the codon table (Table 1) shows that ■ codons AGA, AGG, CGA, CGC, CGG, and CGU all encode the amino acid arginine. Thus, at every position in the nucleic acids of the invention where an arginine is specified by a codon, the codon can be altered to any of the corresponding codons described above without altering the encoded polypeptide. It is understood that U in an RNA sequence corresponds to T in a DNA sequence.<br>
Using, as an example, the nucleic acid sequence corresponding to nucleotides 2-16 of SEQ ID N0:1, TCG ACT CAA GAT GGG, a silent variation of this sequence includes AGT ACC CAG GAG GGA, boo± sequences which encode the amino acid sequence STQDG, corresponding to amino acids 1-5 of SEQ ID NO: 131.<br>
Such "silent variations" are one species of "conservatively modified variations", discussed below. One of skill will recognize that each codon in a nucleic acid (except AUG, which is ordinarily the only codon for methionine) can be modified by standard techniques to encode a functionally identical polypeptide. Accordingly, each silent variation of a nucleic acid which encodes a polypeptide is implicit in any described<br><br>
sequence. The invention provides each and every possible variation of nucleic acid sequence encoding a polypeptide of the invention that could be made by selecting combinations based on possible codon choices. These combinations are made in ccordance with the standard triplet genetic code (e.g., as set forth in Table 1) as applied 3 the nucleic acid sequence encoding an subtilisin homologue polypeptide of the invention. All such variations of every nucleic acid herein are specifically provided and described by consideration of the sequence in combination with the genetic code. Any 'Arian can be produced as noted herein.<br>
Conservative Variations "Conservatively modified variations" or, simply, "conservative<br>
variations" of a particular nucleic acid sequence refers to those nucleic acids which<br>
;node identical or essentially identical amino acid sequences, or, where the nucleic acid<br>
[odes not encode an amino acid sequence, to essentially identical sequences. One of skill<br>
villa recognize that individual substitutions, deletions or additions which alter, add or<br>
delete a single amino acid or a small percentage of amino acids (typically less than about<br>
 , more typically less than about 4%, about 2% or about 1%) in an encoded sequence<br>
ire "conservatively modified variations" where the alterations result in the deletion of an<br>
amino acid, addition of an amino acid, or substitution of an amino acid with a chemically<br>
similar amino acid.<br>
Conservative substitution tables providing functionally similar amino acids<br>
ire well known in the art. Table 2 sets forth six groups which contain amino acids that<br>
ire "conservative substitutions" for one another.<br><br>
Thus, "conservatively substituted variations" of a listed polypeptide sequence of the present invention include substitutions of a small percentage, typically<br><br>
less than about 5%, more typically less than about 2% and often less than about 1%, of the amino acids of the polypeptide sequence, with a conservatively selected amino acid of the same conservative substitution group.<br>
For example, a conservatively substituted variation of the polypeptide identified herein as SEQ ID N0:131 will contain "conservative substitutions", according to the six groups defined above, in up to 8 residues (i,e., about 5% of the amino acids) in the 169 amino acid polypeptide.<br>
In a further example, if four conservative substitutions were localized in the region corresponding to amino acids 25 to 35 of SEQ ID NO: 131, examples of conservatively substituted variations of this region, AAL NNS IGV L, include: AAL QNA LGV V and<br>
AAL QNT VGV M and the like, in accordance with the conservative substitutions listed in Table 2 (in the above example, conservative substitutions are underlined). Listing of a protein sequence herein, in conjunction with the above substitution table, provides an express listing of all conservatively substituted proteins.<br>
Finally, the addition of sequences which do not alter the encoded activity of a nucleic acid molecule, such as the addition of a non-functional or non-coding sequence, is a conservative variation of the basic nucleic acid.<br>
One of skill will appreciate that many conservative variations of the nucleic acid constructs which are disclosed yield a functionally identical construct. For example, as discussed above, owing to the degeneracy of the genetic code, "silent substitutions" (i.e., substitutions in a nucleic acid sequence which do not result in an alteration in an encoded polypeptide) are an implied feature of every nucleic acid sequence which encodes an amino acid. Similarly, "conservative amino acid-substitutions," in one or a few amino acids in an amino acid sequence are substituted with different amino acids with highly similar properties, are also readily identified as being highly similar to a disclosed construct. Such conservative variations of each disclosed sequence are a feature of the present invention.<br>
Non Conservative Variations<br>
Non-conservative modifications of a particular nucleic acid are those<br>
which substitute any amino acid not characterized as a conservative substitution. For<br>
example, any substitution which crosses the bounds of the six groups set forth in Table 2.<br><br>
These include substitutions of basic or acidic amino acids for neutral amino acids, (e.g., Asp, Glu, Asn, or Gin for Val, lie, Leu or Met), aromatic amino acid for basic or acidic amino acids (e.g., Phe, Tyr or Trp for Asp, Asn, (p-lu or Gin) or any other substitution not replacing an amino acid with a like amino acid.  Percent Sequence Identitv-Sequeqce Si:niilaritY<br>
As noted above, the polypeptides and nucleic acids employed in the<br>
subject invention need not be identical, but can be substantially identical (or substantially<br>
similar), to the corresponding sequence of a subliming homologue molecule or related<br>
molecule. The polypeptides (and peptides) can tie subject to various changes, such as<br>
insertions, deletions, and substitutions, either conservative or non-conservative, where<br>
such changes might provide for certain advantages in their use. The polypeptides of the<br>
invention can be modified in a number of ways so long as they comprise a sequence<br>
substantially similar or substantially identical (aL defined below) to a sequence in a<br>
subtilisin homologue molecule.<br>
Alignment and comparison of relatively short amino acid sequences (less<br>
than about 30 residues) is typically straightforwatd. Comparison of longer sequences can<br>
require more sophisticated methods to achieve optimal alignment of two sequences.<br>
Optimal alignment of sequences for aligning a comparison window can be conducted by<br>
the local homology algorithm of Smith and Waterman (1981) Adv Appl Math 2:482, by<br>
the homology alignment algorithm of Needlemah and Wunsch (1970) J Mol Boil 48:443,<br>
by the search for similarity method of Pearson and Lip man (1988) Proc Nati Aced Sci<br>
USA 85:2444, by computerized implementations of these algorithms (GAP, BESTFTT,<br>
FASTA and TFASTA in the Wisconsin Genetics Software Package Release 7.0, Genetics<br>
ComputerGroup, 575 Science Dr., Madison, WI; and BLAST, see, e.g., Latches et al.,<br>
(1977) Nuc Acids Rees 25:3389-3402 and Altschil et aJ., (1990) J Mol Boil 215:403-410),<br>
or by inspection, with the best ahgnment (i.e., resulting in die highest percentage of<br>
sequence similarity over the comparison window) generated by the various methods<br>
being selected.<br>
The term "sequence identity" meds that two polynucleotide sequences are<br>
identical (i.e., on a nucleotide-by-nucleotide basils) over a window of comparison. The<br>
term "percentage of sequence identity" or "percentage bf sequence similarity" is<br>
calculated by comparing two optimally aligned sequences over the window of<br>
comparison, determining the number of positions at which the identical residues occur in<br><br>
both nucleotide sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity (or percentage of sequence similarity). With regard to polypeptide sequences, the term sequence identity likewise means that two polypeptide sequences are identical (on an amino acid-by-amino acid basis) over a window of comparison, and a percentage of amino acid residue sequence identity (or percentage of amino acid residue sequence similarity), also can be calculated. Maximum correspondence can be determined by using one of the sequence algorithms described herein (or other algorithms available to those of ordinary skill in the art) or by visual inspection.<br>
As applied to polypeptides, the term substantial identity or substantial similarity means that two peptide sequences, when optimally aligned, such as by the programs BLAST, GAP or BESTFTT using default gap weights (described in detail below) or by visual inspection, share at least about 60 percent, 70 percent, or 80 percent sequence identity or sequence similarity, preferably at least about 90 percent amino acid residue sequence identity or sequence similarity, more preferably at least about 95 percent sequence identity or sequence similarity, or more (including, e.g., about 96, 97, 98, 98.5, 99, 99.5 or more percent amino acid residue sequence identity or sequence similarity). Similarly, as applied in the context of two nucleic acids, the term substantial identity or substantial similarity means that the two nucleic acid sequences, when optimally aligned, such as by the programs BLAST, GAP or BESTFIT using default gap weights (described in detail below) or by visual inspection, share at least about 60 percent, 70 percent, or 80 percent sequence identity or sequence similarity, preferably at least about 90 percent amino acid residue sequence identity or sequence similarity, more preferably at least about 95 percent sequence identity or sequence similarity, or more (including, e.g., about 96, 97, 98, 98.5, 99, 99.5 or more percent nucleotide sequence identity or sequence similarity).<br>
In one aspect, the present invention provides subtilisin homologue nucleic acids having at least about 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 98.5%, 99%, 99.5% or more percent sequence identity or sequence similarity with the nucleic acid sequences of any of SEQ ID NOs: 1-130 or fragments thereof. In another aspect, the present invention provides subtilisin homologue polypeptides having at least about 70%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 98.5, 99%, 99.5% or more percent sequence<br><br>
identity or sequence similarity with the amino acid sequences of any of SEQ ID N0s:131-260, or fragments thereof that exhibit endo-protease activity. In yet another aspect, the present invention provides subtilisin homologue polypeptides that are substantially identical or substantially similar over at least about 20 (or about 30, 40, 60, 80,100 or more) contiguous amino acids of at least one of SEQ ID NOs: 131-260; some such polypeptides may exhibit improved properties such as thermostability, activity at low or neutral pH, or activity in organic solvents, and the like.<br>
Alternatively, parameters are set such that one or more sequences of the invention are identified by alignment to a query sequence selected from among SEQ ID N0:1 to SEQ K) NO: 130, while sequences corresponding to unrelated polypeptides, e.g., those encoded by nucleic acid sequence represented by GenBank accession numbers: M65086, D13157, S48754, AB005792, D29688, and M28537, are not identified.<br>
Preferably, residue positions which are not identical differ by conservative amino acid substitutions. Conservative amino acid substitution refers to the interchangeability of residues having similar side chains. For example, a group of amino acids having aliphatic side chains is glycine, alumina, vainer, Lucien, and isoleucine; a group of amino acids having aliphatic-hydroxyl side chains is serine and heroine; a group of amino acids having amide-containing side chains is asparagines and glutamine; a group of amino acids having aromatic side chains is phenylalanine, tyrosine, and tryptophan; a group of amino acids having basic side chains is lysine, arginine, and histidine; and a group of amino acids having sulfur-containing side chains is cysteine and methionine. Preferred conservative amino acids substitution groups are: valine-leucine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, and asparagines-glutamine.<br>
A preferred example of an algorithm that is suitable for determining " percent sequence identity or sequence similarity is the FASTA algorithm, which is described in Pearson, W.R. &amp; Lip man, D. J., (1988) Proc Nati Aced Sci USA 85:2444. See also, W. R..Pearson, (1996) Methods Enzvmologv 266:227-258. Preferred parameters used in a FASTA alignment of DNA sequences to calculate percent identity or percent similarity are optimized, BL50 Matrix 15: -5, k-tuples = 2; joining penalty = 40, optimization = 28; gap penalty -12, gap length penalty --2; and width = 16.<br>
Other preferred examples of algorithms that are suitable for determining percent sequence identity or sequence similarity are the BLAST and BLAST 2.0<br><br>
algorithms, which are*described in Latched et al., (1977) Nuc Acids Rees 25:3389-3402 and Altschul et al., (1990) J Mol Boil 215:403-410, respectively. BLAST and BLAST 2.0 are used, with the parameters described herein, to determine percent sequence identity or percent sequence similarity for the nucleic acids and polypeptides and proteins of the invention. Software for performing BLAST analyses is pushily available through the National Center for Biotechnology Information (http: //www.ncbi.nlm.nih.gov/). This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al., supra). These initial neighborhood word hits act as seeds for initiating searches to find longer HSPs containing them. The word hits are extended in.both directions along each sequence for as far as the cumulative ahgnment score can be increased. Cumulative scores are calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always &gt; 0) and N (penalty score for mismatching residues; always 
 <br>
molecules found in, e.g., GenBank. In other words, the stringency of comparison of the algorithms can be increased so that all known prior art (e.g., those represented by GenBank accession numbers: M65086, D13157, S48754, AB005792, D29688, and M28537 or other similar molecules found in, e.g., GenBank, as well as sequences represented by GenBank accession numbers: P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146) is excluded.<br>
The BLAST algorithm also performs a statistical analysis of the similarity or identity between two sequences {see, e.g., Karlin &amp; Altschul, (1993) Proc Nat] Acad Sci USA 90:5873-5787). One measure of similarity or identity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar to a reference sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.2, more preferably less than about 0.01, and most preferably less than about 0.001.<br>
Another example of a useful algorithm is PILEUP. PILEUP creates a multiple sequence alignment from a group of related sequences using progressive, pair wise alignments to show relationship and percent sequence identity or percent sequence similarity. It also plots a tree or endogamy showing the clustering relationships used to create the alignment. PILEUP uses a simplification of the progressive alignment method of Fangs &amp; Doolittle, (1987) J Mol Evol 35:351-360. The motion used is similar to the method described by Higgins &amp; Sharp, (1989) CABIOS 5:151-153. The program can align up to 300 sequences, each of a maximum length of 5,000 nucleotides or amino . acids. The multiple alignment procedure begins with the pairwise alignment of the two most similar sequences, producing a cluster of two aligned sequences. This cluster is then aligned to the next most related sequence or cluster of aligned sequences. Two clusters of sequences are aligned by a simple extension of the pairwise alignment of two individual sequences. The final alignment is achieved by a series of progressive, pairwise alignments. The program is run by designating specific sequences and their amino acid or nucleotide coordinates for regions of sequence comparison and by designating the program parameters. Using PILEUP, a reference sequence is compared to other test<br><br>
sequences to determine the percent sequence identity (or percent sequence similarity) relationship using the following parameters: default gap weight (3.00), default gap length weight (0.10), and weighted end gaps. PE.EUP can be obtained from the GCG sequence analysis software package, e.g., version 7.0 (Devereaux et al., (1984) Nuc Acids Res 12:387-395).<br>
Another preferred example of an algorithm that is suitable for multiple DNA and amino acid sequence ahgnment is the CLUSTALW program (Thompson, J. D. et al, (1994) Nuc Acids Res 22:4673-4680). CLUST.\LW performs multiple pairwise comparisons between groups of sequences and assembles them into a multiple alignment based on homology. Gap open and Gap extension penalties were 10 and 0.05 respectively. For amino acid alignments, the BLOSUM algorithm can be used as a protein weight matrix (Henikoff and Henikoff, (1992) Proc Nati Acad Sci USA 89:10915-10919).<br>
It will be understood by one of ordinary skill in the art, that the above discussion of search and alignment algorithms also applies to identification and evaluation of polynucleotide sequences, with the substitution of query sequences comprising nucleotide sequences, and where appropriate, selection of nucleic acid databases.<br>
Nucleic Acid Hvbridization<br>
Nucleic acids "hybridize" when they associate, typically in solution.<br>
Nucleic acids hybridize due to a variety of well characterized physico-chemical forces,<br>
such as hydrogen bonding, solvent exclusion, base stacking and the like. An extensive<br>
guide to the hybridization of nucleic acids is found in Tijssen (1993) Laboratory<br>
Techniques in Biochemistry and Molecular Biology—Hybridization with Nucleic Acid<br>
Probes part I chapter 2, "Overview of principles of hybridization and the strategy of<br>
nucleic acid probe assays," (Elsevier, New York), as well as in Ausubel, supra, Hames<br>
and Higgins (1995) Gene Probes 1IRL Press at Oxford University Press, Oxford,<br>
England, (Hames and Higgins 1) and Hames and High (1995) Gene Probes 2 IRL<br>
Press at Oxford University Press, Oxford, England (Hames and Higgins 2) provide details<br>
on the synthesis, lambent, detection and quantification of DNA and RNA, including<br>
oligonucleotides.<br>
"Stringent hybridization wash conditions" in the context of nucleic acid<br>
hybridization experiments such as Southern and northern hybridizations are sequence<br><br>
dependent, and are different under different environmental parameters. An extensive guide to the hybridization of nucleic acids is found in Tijssen (1993), supra, and in Hames and Higgins, 1 and 2.<br>
For purposes of the present invention, generally, "highly stringent" hybridization and wash conditions are selected to be about 5° C lower than the hennas melting point (Tm) for the specific sequence at a defined ionic strength and pH. The Tm is the temperature (under defined ionic strength and pH) at which 50% of the test sequence hybridizes to a perfectly matched probe. Very stringent conditions are selected to be equal to the Tm for a particular probe.<br>
An example of stringent hybridization conditions for hybridization of complementary nucleic acids which have more than 100 complementary residues on a filter in a Southern or northern blot is 50% formally with 1 mg of heparin at Alco, with the hybridization being carried out overnight. An example of stringent wash conditions is a 0.2x SSC wash at 65°C for 15 minutes {see, Sambrook, supra for a description of SSC buffer). Often the high stringency wash is preceded by a low stringency wash to remove background probe signal. An example low stringency wash is 2x SSC at 40°C for 15 minutes. In general, a signal to noise ratio of 5x (or higher) than that observed for an unmatched probe (e.g., a publically available subtilisin coding nucleic acid with a sequence found in Greenback prior to the filing of the present application) in the particular hybridization assay indicates detection of a specific hybridization.<br>
Comparative hybridization can be used to identify nucleic acids of the invention, and this comparative hybridization method is a preteen  method of distinguishing nucleic acids of the invention.<br>
In particular, detection of highly stringent hybridization in the context of the present invention indicates strong structural similaiity to, e.g., the nucleic acids provided in the sequence listing herein. For example, it is desirable to identify test nucleic acids which hybridize to the exemplar nucleic acids herein under stringent conditions. One measure of stringent hybridization is the ability to hybridize to one of the listed nucleic acids (e.g., nucleic acid sequences SEQ ID Noël to SEQ ID NO: 130, and complementary polynucleotide sequences thereof, or a subsequence thereof, (e.g., subsequences encoding amino acid positions 71-95, 86-110, 111-135, and/or 196-230).<br><br>
under highly stringent conditions. Stringent hybridization and wash conditions can easily be determined empirically for any test nucleic acid.<br>
For example, in determining highly stringent hybridization and wash conditions, the hybridization and wash conditions are gradually increased (e.g., by increasing temperature, decreasing salt concentration, increasing detergent concentration and/or increasing the concentration of organic solvents such as formal in in the hybridization or wash), until a selected set of criteria are met. For example, the hybridization and wash conditions are gradually increased until a probe comprising one or more nucleic acid sequences selected from SEQ ID N0:1 to SEQ ID NO: 130, or complementary polynucleotide sequences thereof, binds to a perfectly matched complementary target (again, a nucleic acid comprising one or more nucleic acid sequences selected from SEQ ID N0:1 to SEQ ID NO: 130, and complementary polynucleotide sequences thereof), with a signal to noise ratio that is at least 5x as high as that observed for hybridization of the probe to an unmatched target, and is sometimes lOx, 20x, 50x or even higher, depending on the desired discrimination. In this case, the unmatched target is a nucleic acid corresponding to a known subtilisin homologue, e.g., an subtilisin homologue nucleic acid (other than those in the accompanying sequence listing) that is present in a public database such as GenBank"^^ at the time of fang of the subject application. Examples of such unmatched target nucleic acids include, e.g., those with the following Genial accession numbers: M65086, D13157, S48754, AB005792, D29688, andM28537. Additional such sequences can be identified in GenBank by one of skill.<br>
A test nucleic acid is said to specifically hybridize to a probe nucleic acid when it hybridizes at least V2 as well to the probe as to the perfectly matched complementary target, i.e., with a signal to noise ratio at least Vi as high as hybridization of the probe to the target under conditions in which the perfectly matched probe binds to the perfectly matched complementary target with a signal to noise ratio that is at least about 5x-10x, and occasionally 20x, 5Gx or greater than that observed for hybridization to any of the unmatched target nucleic acids M65086, D13157, S48754, AB005792, D29688, andM28537.<br>
Ultra high-stringency hybridization and wash conditions are those in which the stringency of hybridization and wash conditions are increased until the signal to noise ratio for binding of the probe to the perfectly matched complementary target<br><br>
nucleic acid is at least lOx, sometimes 20x, and occasionally 50x as high as that observed for hybridization to any of the unmatched target nucleic acids M65086, D13157, S48754, AB005792, D29688, and M28537. A target nucleic acid which hybridizes to a probe under such conditions, with a signal to noise ratio of at least V2 that of the perfectly matched complementary target nucleic acid is said to bind to the probe under ultra-high stringency conditions.<br>
Similarly, even higher levels of stringency can be determined by gradually increasing the hybridization and/or wash conditions of the relevant hybridization assay. For example, those in which the stringency of hybridization and wash conditions are increased until the signal to noise ratio for binding of the probe to the perfectly matched complementary target nucleic acid is at least lOx, 20X, 50X, lOx, or 500X or more as high as that observed for hybridization to any of the unmatched target nucleic acids M65086, D13157, S48754, AB005792, D29688, and M28537can be identified. A target nucleic acid which hybridizes to a probe under such conditions, with a signal to noise ratio of at least V2 that of the perfectly matched complementary target nucleic acid is said to bind to the probe under ultra-ultra-high stringency conditions. For example, the most similar sequences selected from among those available in GenBank, as of the filing date, can be used as the control sequences.<br>
Target nucleic acids which hybridize to the nucleic acids represented by SEQ K) NO: 1 to SEQ ID NO: 130, under high, ultra-high and ultra-ultra high stringency conditions are a feature of the invention. Examples of such nucleic acids include those with one or a few silent or conservative nucleic acid substitutions as compared to a given nucleic acid sequence.<br>
Nucleic acids which do not hybridize to each other under stringent conditions are still substantially identical if the polypeptides which they encode are substantially identical. This occurs, e.g., when a copy of a nucleic acid is created using the maximum codon degeneracy permitted by the genetic code, or when antisera generated against one or more of SEQ ID NO: 131 to SEQ ID NO:260 which has been subtracted using the polypeptides encoded by the following subtilisin sequences in GenBank: M65086, D13157, S48754, AB005792, D29688, and M28537. Further details on immunological identification of polypeptides of the invention are found below.<br>
In one aspect, the invention provides a nucleic acid which comprises a unique subsequence in a nucleic acid selected from SEQ ID N0:1 to SEQ ID NO: 130.<br><br>
The unique subsequence is unique as compared to a nucleic acid corresponding to any of: M65086,D13157,S48754,AB005792,D29688, andM28537. Such unique subsequences can be determined by aligning any of SEQ ED N0:1 to SEQ E) NO: 130 against the complete set of nucleic acids corresponding to M65086, D13157, S48754, AB005792, D29688, andM28537. Alignment can be performed using the BLAST algorithm set to default parameters. Any unique subsequence is useful, e.g., as a probe to identify the nucleic acids of the invention.<br>
Similarly, the invention includes a polypeptide which comprises a unique subsequence in a polypeptide selected from: SEQ ID NO: 131 to SEQ ID NO:260. Here, the unique subsequence is unique as compared to a polypeptide corresponding to any of (GenBank accession numbers): P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146. Here again, the polypeptide is aligned against the complete set of polypeptides corresponding to P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146 (note that where the sequence coixesponds to a non-translated sequence such as a pseudo gene, the corresponding polypeptide is generated simply by in silico translation of the nucleic acid sequence into an amino acid sequence, where the reading frame is selected to correspond to the reading frame of homologous subtilisin nucleic acids.<br>
The invention also provides for target nucleic acids which hybridizes under stringent conditions to a unique coding oligonucleotide which encodes a unique subsequence in a polypeptide selected from: SEQ ID NO: 131 to SEQ ID NO:260, wherein the unique subsequence is unique as compared to a polypeptide corresponding to any of the control polypeptides. Unique sequences are determined as noted above.<br>
In one example, the stringent conditions are selected such that a perfectly complementary oligonucleotide to the coding oligonucleotide hybridizes to the coding oligonucleotide with at least about a 5-lOx higher signal to noise ratio than for hybridization of the perfectly complementary oligonucleotide to a control nucleic acid corresponding to any of the control polypeptides. Conditions can be selected such that higher ratios of signal to noise are observed in the particular assay which is used, e.g.,<br><br>
about 15x, 20x, 30x, 50x or more. In this example, the target nucleic acid hybridizes to the unique coding oligonucleotide with at least a 2x higher signal to noise ratio as compared to hybridization of the control nucleic acid to the coding oligonucleotide. Again, higher signal to noise ratios can be selected, e.g., about 5x, lOx, 20x, 30x, 50x or more. The particular signal will depend on the label used in the relevant assay, e.g., a fluorescent label, a colorimetric label, a radio active label, or the like.<br>
SUBSTRATES AND FORMATS FOR SEQUENCE I^COMBINATION<br>
The polynucleotides of the invention are optionally used as substrates for a variety of diversity generating procedures, including recombination and recursive recombination (e.g., DNA shuffling) reactions, i.e., to produce additional subtilisin homologues with desired properties. In addition to standard cloning methods as set for the in, e.g., Sambrook, Ausubel and Berger, all supra, a variety of diversity generating protocols are available and described in the art. The procedures can be used separately, and/or in combination to produce one or more variants of a nucleic acid or set of nucleic acids, as well variants of encoded proteins. Individually and collectively, these procedures provide robust, widely applicable ways of generating diversified nucleic acids and sets of nucleic acids (including, e.g., nucleic acid libraries) useful, e.g., for the engineering or rapid evolution of nucleic acids, proteins, pathways, cells and/or organisms with new and/or improved characteristics.<br>
While distinctions and classifications are made in the course of the ensuing discussion for clarity, it will be appreciated that the techniques are often not mutually exclusive. Indeed, the various methods can be used singly or in combination, in parallel or in series, to access diverse sequence variants.<br>
The result of any of the diversity generating procedures described herein can be the generation of one or more nucleic acids, which can be selected or screened for nucleic acids with or which confer desirable properties, or that encode proteins with or which confer desirable properties. Following diversification by one or more of the methods herein, or otherwise available to one of skill, any nucleic acids that are produced can be selected for a desired activity or property, e.g. subtilisin homologues with improved thermostability, increased activity at neutral or low pH, increased activity in organic solvents, and the like. This can include identifying any activity that can be detected, for example, in an automated or automat able format, by any of the assays in the<br><br>
art, including the various methods for assessing protease activity described herein, and known in the art. A variety of related (or even unrelated) properties can be evaluated, in serial or in parallel, at the discretion of the practitioner.<br>
Descriptions of a variety of diversity generating procedures suitable for generating modified nucleic acid sequences encoding subtilisin homologues with desired properties are found in the following publications and the references cited therein: Soong, N. et al. (2000) "Molecular breeding of viruses" Nat Genet 25(4):436-439; Stemmer, et al. (1999) "Molecular breeding of viruses for targeting and other clinical properties" Tumor Targeting 4:1-4; Ness et al. (1999) "DNA Shuffling of subgenomic sequences of subtilisin" Nature Eiotechnoloev 17:893-896; Chang et al. (1999) "Evolution of a cytokine using DNA family shuffling" Nature Biotechnologv 17:793-797; Minshull and Stemmer (1999) "Protein evolution by molecular breeding" Current Opinion in Chemical Biologv 3:284-290; Christians et al. (1999) "Directed evolution of thymidine kinase for AZT phosphorylation using DNA family shuffling" Nature Biotechnologv 17:259-264; Crameri et al. (1998) 'T)NA shuffling of a family of genes from diverse species accelerates directed evolution" Nature 391:288-291; Crameri et al. (1997) "Molecular evolution of an arsenate detoxification pathway by DNA shuffling," Nature Biotechnologv 15:436-438; Zhang et al. (1997) "Directed evolution of an effective fucosidase from a galactosidase by DNA shuffling and screening" Proc. Natl. Acad Sci. USA 94:4504-4509; Patten et al. (1997) "Applications of DNA Shuffling to Pharmaceuticals and Vaccines" Current Opinion in Biotechnology 8:724-733; Crameri et al. (1996) "Construction and evolution of antibody-phage libraries by DNA shuffling" Nature Medicine 2:100-103; Crameri et ah (1996) "Improved green fluorescent protein by molecular evolution using DNA shuffling" Nature Biotechnologv 14:315-319; Gates et al. (1996) "Affinity selective isolation of ligand from peptide libraries through display on a lac repressor 'headpiece dimer'" Journal of Molecular Biologv 255:373-386; Stemmer (1996) "Sexual PCR and Assembly PCR" In: The Encvclopedia of Molecular Biology. VCH Publishers, New York, pp.447-457; Crameri and Stemmer (1995) "Combinatorial multiple cassette mutagenesis creates all the permutations of mutant and wildtype cassettes" BioTechniques 18:194-195; Stemmer et al., (1995) "Single-step assembly of a gene and entire plasmid form large numbers of ohgodeoxy-ribonucleotides" Gene. 164:49-53; Stemmer (1995) "The Evolution of Molecular Computation" Science 270: 1510; Stemmer (1995) "Searching Sequence Space"<br><br>
Bio/Technology 13:549-553; Stemmer (1994) "Rapid evolution of a protein in vitro by DNA shuffling" Nature 370:389-391; and Stemmer (1994) "DNA snuffing by random fragmentation and reassembly: In vitro recombination for molecular evolution." Proc, Natl, Acad. Sci. USA 91:10747-10751.<br>
Mutational methods of generating diversity include, for example, site-directed mutagenesis (Ling et al. (1997) "Approaches to DNA mutagenesis: an overview" Anal Biochem. 254(2): 157-178; Dale et al. (1996) "Oligonucleotide-directed random mutagenesis using the phosphorothioate method" Methods Mol. Biol. 57:369-374; Smith<br>
(1985)	"In vitro mutagenesis" Ann. Rev. Genet. 19:423-462; Botstein &amp; Shortle (1985)<br>
"Strategies and applications of in vitro mutagenesis" Science 229:1193-1201; Carter<br>
(1986)	"Site-directed mutagenesis" Biochem. J. 237:1-7; and Kunkel (1987) "The<br>
efficiency of oligonucleotide directed mutagenesis" in Nucleic Acids &amp; Molecular<br>
Biology (Eckstein, F. and Lilley, D.M.J. eds., Springer Verlag, Berlin)); mutagenesis<br>
using uracil containing templates (Kunkel (1985) "Rapid and efficient site-specific<br>
mutagenesis without phenotypic selection" Proc. Natl. Acad. Sci. USA 82:488-492;<br>
Kunkel et al. (1987) "Rapid and efficient site-specific mutagenesis without phenotypic<br>
selection" Methods in Enzvmol. 154, 367-382; and Bass et al. (1988) "Mutant Trp<br>
repressors with new DNA-binding specificities" Science 242:240-245); oligonucleotide-<br>
directed mutagenesis (Methods in Enzvmol 100:468-500 (1983); Methods in Enzvmol.<br>
154: 329-350 (1987); Zoller &amp; Simi (1982) "Oligonucleotide-directed mutagenesis<br>
using M13-derived vectors: an efficient and general procedure for the production of point<br>
mutations in any DNA fragment" Nucleic Acids Res. 10:6487-6500; Zoller &amp; Smith<br>
(1983) "Oligonucleotide-directed mutagenesis of DNA fragments cloned into M13<br>
vectors" Methods in Enzvmol. 100:468-500; and Zoller &amp; Smith (1987)<br>
"Oligonucleotide-directed mutagenesis; a simple method using two oligonucleotide<br>
primers and a single-stranded DNA template" Methods in Enzvmol. 154:329-350);<br>
phosphorothioate-modified mutagenesis (Taylor et al. (1985) "The use of<br>
phosphorothioate-modified DNA in restriction enzyme reactions to prepare nicked DNA"<br>
Nucl. Acids Res. 13: 8749-8764; Taylor et al. (1985) "The rapid generation of<br>
oligonucleotide-directed mutations at high frequency using phosphorothioate-modified<br>
DNA" Nucl. Acids Res. 13: 8765-8787 (1985);   &amp; Eckstein (1986) "Inhibition<br>
of restriction endonuclease Nci I cleavage by phosphorothioate groups and its application<br>
to oligonucleotide-directed mutagenesis" Nucl. Acids Res. 14: 9679-9698; Sayers et al.<br><br>
(1988) "Y-T Exonucleases in phosphorothioate-based oligonucleotide-directed mutagenesis" Nucl. Acids Res. 16:791-802; and Sayers et al. (1988) "Strand specific cleavage of phosphorothioate-containing DNA by reaction with restriction endonucleases in the presence of ethidium bromide" Nucl. Acids Res. 16: 803-314); mutagenesis using gapped duplex DNA (Kramer et al. (1984) 'The gapped duplex DNA approach to oligonucleotide-directed mutation construction" Nucl. Acids Res. 12: 9441-9456; Kramer &amp; Fritz (1987) Methods in Enzvmol "Oligonucleotide-directed construction of mutations via gapped duplex DNA" 154:350-367; Kramer et al. (1988) "Improved enzymatic in vitro reactions in the gapped duplex DNA approach to oligonucleotide-directed construction of mutations" Nucl. Acids Res. 16: 7207; and Fritz et al. (1988) "Oligonucleotide-directed construction of mutations: a gapped duplex DNA procedure without enzymatic reactions in vitro" Nucl. Acids Res. 16: 6987-6999).<br>
Additional suitable methods include point mismatch repair (Kramer et al. (1984) "Point Mismatch Repair" Cell 38:879-887), mutagenesis using repair-deficient host strains (Carter et al. (1985) "Improved oligonucleotide site-directed mutagenesis using M13 vectors" Nucl. Acids Res. 13: 4431-4443; and Carter (1987) "Improved oligonucleotide-directed mutagenesis using M13 vectors" Methods in EnzvmoL 154: 382-403), deletion mutagenesis (Eghtedarzadeh &amp; HenikofI' (1986) "Use of oligonucleotides to generate large deletions" Nucl. Acids Res. 14: 5115), restriction-selection and restriction-purification (Wells et al. (1986) "Importance of hydrogen-bond formation in stabilizing the transition state of subtilisin" Phil. Trans. R. Soc. Lond. A 317: 415-423), mutagenesis by total gene synthesis (Zambia et al. (1984) "Total synthesis and cloning of a gene coding for the ribonuclease S protein" Science 223: 1299-1301; Salami and Shoran (1988) "Total synthesis and expression of a gene for the a-subunit of bovine rod outer segment guanine nucleotide-binding protein (transducin)" Nucl. Acids Res. 14:  . 6361-6372; Wells et al. (1985) "Cassette mutagenesis: an efficient method for generation of multiple mutations at defined sites" Gene 34:315-323; and Grundstrom et al. (1985) "Ohgonucleotide-directed mutagenesis by micro scale 'shot-gun' gene synthesis" Nucl. Acids Res. 13: 3305-3316), double-strand break repair (LANDesk (1986) "Oligonucleotide-directed double-strand break repair in plasmids of Escherichia coli: a method for site-specific mutagenesis" Proc. Natl. Acad. Sci. USA, 83:7177-7181; and Arnold (1993) "Protein engineering for unusual environments" Current Opinion in Biotechnologv 4:450-455). Additional details on many of the above methods can be<br><br>
found in Methods in Enzvmologv Volume 154, which also describes useful controls for trouble-shooting problems with various mutagenesis methods.<br>
Additional details regarding various diversity generating methods, e.g., DNA shuffling methods, can be found in the following U.S. patents, PCT publications and appositions, and EPO publications: U.S. Pat. No. 5,605,793 to Stemmer (February 25,1997), "Methods for In Vitro Recombination;" U.S. Pat. No. 5,811.238 to Stemmer et al. (September 22,1998) "Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;" U.S. Pat. No. 5,830,721 to Stemmer et al. (November 3,1998), "DNA Mutagenesis by Random Fragmentation and Reassembly;" U.S. Pat. No. 5,834,252 to Stemmer, et ;il. (November 10,1998) "End-Complementary Polymerase Reaction;" U.S. Pat. No. 5,837,458 to Minshull, et al. (November 17,1998), "Methods and Compositions for Cellular and Metabolic Engineering;" WO 95/22625, Stemmer and Crameri, "Mutagenesis by Random Fragmentation and Reassembly;" WO 96/33207 by Stemmer and Lipschutz "End Complementary Polymerase Chain Reaction;" WO 97/20078 by Stemmer and Crameri "Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;" WO 97/35966 by Minshull and Stemmer, "Methods and Compositions for Cellular and Metabolic Engineering;" WO 99/41402 by Punnonen et al. "Targeting of Genetic Vaccine Vectors;" WO 99/41383 by Pennon et al. "Antigen Library Immunization;" WO 99/41369 by Punnonen et al. "Genetic Vaccine Vector Engineering;" WO 99/41368 by Punnonen et al. "Optimization of Immunomodulatory Properties of Genetic Vaccines;" EP 752008 by Stemmer and Crameri, "DNA Mutagenesis by Random Fragmentation and Reassembly;" EP 0932670 by Stemmer "Evolving Cellular DNA Uptake by Recursive Sequence Recombination;" WO 99/23107 by Stemmer et al., "Modification of Virus Tropism and Host Range by Viral Genome Shuffling;" WO 99/21979 by Apt et al., "Human Papillomavirus Vectors;" WO 98/31837 by del Cardayre et al. "Evolution of Whole Cells and Organisms by Recursive Sequence Recombination;" WO 98/27230 by Patten and Stemmer, "Methods and Compositions for Polypeptide Engineering;" WO 98/27230 by Stemmer et al., "Methods for Optimization of Gene Therapy by Recursive Sequence Shuffling and Selection," WO 00/00632, "Methods for Generating Highly Diverse Libraries," WO 00/09679, "Methods for Obtaining in Vitro Recombined Polynucleotide Sequence Banks and Resulting Sequences," WO 98/42832 by Arnold et al., "Recombination of Polynucleotide<br><br>
Sequences Using Random or Defined Primers," WO 99/29902 by Arnold et al., "Method for Creating Polynucleotide and Polypeptide Sequences," WO 98/41653 by Vind, "An in Vitro Method for Construction of a DNA Library," WO 98/41622 by Borchert et al., "Method for Constructing a Library Using DNA Chuffing," and WO 98/42727 by Pati and Zarling, "Sequence Alterations using Homologous Recombination;" WO 00/18906 by Patten et al., "Shuffling of Codon-Altered Genes;" WO 00/04190 by del Cardayre et al. "Evolution of Whole Cells and Organisms by Recursive Recombination;" WO 00/42561 by Crameri et al., "Oligonucleotide Mediated Nucleic Acid Recombination;" WO 00/42559 by Selifonov and Stemmer "Methods of Populating Data Structures for Use in Evolutionary Simulations;" WO 00/42560 by Selifonov et al, "Methods for Making Character Strings, Polynucleotides &amp; Polypeptides Having Desired Characteristics;" PCT/USOO/26708 by Welch et al., "Use of Codon-Varied OKgonucleotide Synthesis for Synthetic Shuffling;" and PCT/USOl/06775 "Single-Stranded Nucleic Acid Template-Mediated Recombination and Nucleic Acid Fragment Isolation" by Affronter.<br>
In brief, several different general classes of sequence modification methods, such as mutation, recombination, etc. are applicable to the present invention and set forth, e.g., in the references above. That is, any of the methods cited above can be adapted to the present invention to evolve the subtilisin homologues discussed herein to produce new endo-proteases with improved properties,. Both the methods of making such subtilisins and the subtilisins produced by these methods are a feature of the invention.<br>
The following exemplify some of the different types of preferred formats for diversity generation in the context of the present invention, including, e.g., certain recombination based diversity generation formats.<br>
Nucleic acids can be recombined in vitro by any of a variety of techniques discussed in the references above, including e.g., DNAse digestion of nucleic acids to be recombined followed by ligation and/or PCR reassembly of the nucleic acids. For example, sexual PCR mutagenesis can be used in which random (or pseudo random, or even non-random) fragmentation of the DNA molecule is followed by recombination,   . based on sequence similarity, between DNA molecules with different but related DNA sequences, in vitro, followed by fixation of the crossover by extension in a polymerase chain reaction. This process and many process variants is described in several of the references above, e.g., in Stemmer (1994) Proc. Natl. Acad. Sci. USA 91:10747-10751.<br><br>
Thus, any of the subtilisin homologue nucleic acids described herein can be recombined in vitro to generate additional subtilisin homologues with desired properties.<br>
Similarly, nucleic acids can be recursively recombined in vivo, e.g., by allowing recombination to occur between nucleic acids in cells. Many such in vivo recombination formats are set forth in the references noted above. Such formats optionally provide direct recombination between nucleic acids of interest, or provide recombination between vectors, viruses, plasmids, etc., comprising the nucleic acids of interest, as well as other formats. Details regarding such procedures are found in the references noted above. Accordingly, any of the subtilisin homologue encoding nucleic acids can be recombined in vivo to produce novel subtilisin homologues with desired properties.<br>
■ Whole genome recombination methods can also be used in which whole genomes of cells or other organisms are recombined, optionally including spiking of the genomic recombination mixtures with desired library components (e.g., genes corresponding to the pathways of the present invention). These methods have many applications, including those in which the identity of a target gene is not known. Details on such methods are found, e.g., in WO 98/31837 by del Cardayre et al. "Evolution of Whole Cells and Organisms by Recursive Sequence Recombination;" and in, e.g., PCT/US99/15972 by del Cardayre et al., also entitled "Evolution of Whole Cells and Organisms by Recursive Sequence Recombination.'* Any of the subtilisin homologue nucleic acids of the invention can, thus, be recombined using whole genome recombination methods to generate additional subtilisin homologues with advantageous characteristics.<br>
Synthetic recombination methods can also be used, in which oligonucleotides corresponding to targets of interest, e.g., the subtilisin homologues provided herein, are synthesized and reassembled in PCR or ligation reactions which include, for example, oligonucleotides which correspond to more than one parental nucleic acid, oligonucleotides corresponding to consensus sequences for a plurality of parental nucleic acids, (optionally incorporating one or more variable nucleotide positions), oligonucleotides incorporating proven or putative functional motifs, etc., thereby generating new recombined nucleic acids. Oligonucleotides can be made by standard nucleotide addition methods, or can be made, e.g., by tri-nucleotide synthetic approaches. Details regarding such approaches are found in the references noted above,<br><br>
including, e.g., WO 00/42561 by Crameri et al., "Oligonucleotides Mediated Nucleic Acid Recombination;" PCT/USOO/26708 by Welch et al., "Use of Codon-Varied Oligonucleotide Synthesis for Synthetic Shuffling;" WO 00/42560 by Selifonov et al., "Methods for Making Character Strings, Polynucleotides and Polypeptides Having Desired Characteristics;" and WO 00/42559 by Selifonov and Stemmer "Methods of Populating Data Structures for Use in Evolutionary Simulations."<br>
In silicon methods of recombination can be effected in which genetic algorithms are used in a computer to recombine sequence strings which correspond to homologous (or even non-homologous) nucleic acids. The resulting recombined sequence strings are optionally converted into nucleic acids by synthesis of nucleic acids which correspond to the recombined sequences, e.g., in concert with oligonucleotide synthesis/ gene reassembly techniques. This approach can generate random, partially random or designed variants. Many details regarding in silico recombination, including the use of genetic algorithms, genetic operators and the like in computer systems, combined with generation of corresponding nucleic acids (and/or proteins), as well as combinations of designed nucleic acids and/or proteins (e.g., based on cross-over site selection) as well as designed, pseudo-random or random recombination methods are described in WO 00/42560 by Selifonov et al., "Methods for Making Character Strings, Polynucleotides and Polypeptides Having Desired Chiiiracteristics" and WO 00/42559 by Selifonov and Stemmer "Methods of Populating Data Structures for Use in Evolutionary Simulations." Extensive details regarding in silico recombination methods are found in these applications. This methodology is generally applicable to the present invention in providing for recombination of subtilisin homologues in silico and/ or the generation of corresponding nucleic acids or proteins.<br>
Many methods of accessing natural diversity, e.g., by hybridization of diverse nucleic acids or nucleic acid fragments to single-stranded templates, followed by polymerization and/or ligation to regenerate full-length sequences, optionally followed by degradation of the templates and recovery of the resulting modified nucleic acids can be similarly used. In one method employing a single-stranded template, the fragment population derived from the genomic library(ies) is annealed with partial, or, often approximately full length ssDNA or RNA corresponding to the opposite strand. Assembly of complex chimeric genes from this population is then mediated by nuclease-base removal of non-hybridizing fragment ends, polymerization to fill gaps between such<br><br>
fragments and subsequent single stranded ligation. The parental polynucleotide strand can be removed by digestion (e.g., if RNA or uracil-containing), magnetic separation under denaturing conditions (if labeled in a inaner conducive to such separation) and other available separation/purification methods. Alternatively, the parental strand is optionally co-purified with the chimeric strands and removed during subsequent screening and processing steps. Additional details regarding this approach are found, e.g., in "Single-Stranded Nucleic Acid Template-Mediated Recombination and Nucleic Acid Fragment Isolation" by Affronters, PCT/USOl/06775.<br>
In another approach, single-stranded molecules are converted to double-stranded DNA (dsDNA) and the dsDNA molecules are bound to a solid support by ligand-mediated binding. After separation of unbound DNA, the selected DNA molecules are released from the support and introduced into a suitable host cell to generate a library enriched sequences which hybridize to the probe. A library produced in this manner provides a desirable substrate for further diversification using any of the procedures described herein.<br>
Any of the preceding general recombination formats can be practiced in a reiterative fashion (e.g., one or more cycles of mutation/recombination or other diversity generation methods, optionally followed by one or more selection methods) to generate a more diverse set of recombinant nucleic acids.<br>
Mutagenesis employing polynucleotide chain termination methods have also been proposed {see e.g., U.S. Patent No. 5,965,408, "Method of DNA reassembly by interrupting synthesis" to Short, and the references above), and can be applied to the present invention. In this approach, double stranded DNAs corresponding to one or more genes sharing regions of sequence similarity are combined and denatured, in the presence or absence of primers specific for the gene. The single stranded polynucleotides are then annealed and incubated in the presence of a polymerase and a chain terminating reagent (e.g., ultraviolet, gamma or X-ray irradiation; ethidium bromide or other intercalators; DNA binding proteins, such as single strand binding proteins, transcription activating factors, or histones; polycyclic aromatic hydrocarbons; trivalent chromium or a trivalent ' chromium salt; or abbreviated polymerization mediated by rapid thennocycling; and the like), resulting in the production of partial duplex molecules. The partial duplex molecules, e.g., containing partially extended chains, are then denatured and reinvaded in subsequent rounds of replication or partial replication resulting in polynucleotides<br><br>
which share varying degrees of sequence similarity and which are diversified with respect to the starting population of DNA molecules. Optionally, the products, or partial pools of the products, can be amplified at one or more stages in the process. Polynucleotides produced by a chain termination method, such as described above, are suitable substrates for any other described recombination format.<br>
Diversity also can be generated in nucleic acids or populations of nucleic acids using a recombinational procedure termed "incremental truncation for the creation of hybrid enzymes" Cultch") described in Ostermeier et al. (1999) "A combinatorial approach to hybrid enzymes independent of DNA homology" Nature Biotech 17:1205. This approach can be used to generate an initial a librai-y of variants which can optionally serve as a substrate for one or more in vitro or in vivo recombination methods. See, also, Ostermeier et al. (1999) "Combinatorial Protein Engineering by Incremental Truncation," Proc. Natl. Acad. Sci. USA, 96: 3562-67; Ostermeier et al. (1999), "Incremental Truncation as a Strategy in the Engineering of Novel Biocatalysts," Biological and Medicinal Chemistry, 7: 2139-44.<br>
Mutational methods which result in the alteration of individual nucleotides or groups of contiguous or non-contiguous nucleotides can be favorably employed to introduce nucleotide diversity into one or more parental subtilisin homologues. Many mutagenesis methods are found in the above-cited references; additional details regarding mutagenesis methods can be found in following, which can also be applied to the present invention.<br>
For example, error-prone PCR can be used to generate nucleic acid variants. Using this technique, PCR is performed under conditions where the copying fidelity of the DNA polymerase is low, such that a high rate of point mutations is obtained along the entire length of the PCR product. Examples of such techniques are found in the references above and, e.g., in Leung et al. (1989) Technique 1:11-15 and Caldwell et al. (1992) PCR Methods Alpha. 2:28-33. Similarly, assembly PCR can be used, in a process which involves the assembly of a PCR product from a mixture of small DNA fragments. A large number of different PCR reactions can occur in parallel in the same reaction mixture, with the products of one reaction priming the products of another reaction.<br>
Oligonucleotide directed mutagenesis can be used to introduce site-specific mutations in a nucleic acid sequence of interest. Examples of such t-techniques<br><br>
are found in the references above and, e.g., in Reinhardt-Olson et al. (1988) Science, 241:53-57. Similarly, cassette mutagenesis can be used in a process that replaces a small region of a double stranded DNA molecule with a synthetic oligonucleotide cassette that differs from the native sequence. The oligonucleotide can contain, e.g., completely and/or partially randomized native sequence(s).<br>
Recursive ensemble mutagenesis is a process in which an algorithm for protein mutagenesis is used to produce diverse populations of phenotypically related mutants, members of which differ in amino acid sequence. This method uses a feedback mechanism to monitor successive rounds of combinatorial cassette mutagenesis. Examples of this approach are found in Arcing &amp; You van (1992) Proc. Natl. Acad. Sci. USA 89:7811-7815.<br>
Exponential ensemble mutagenesis can be used for generating combinatorial libraries with a high percentage of unique and functional mutants. Small groups of residues in a sequence of interest are randomized in parallel to identify, at each altered position, amino acids which lead to functional proteins. Examples of such procedures are found in Delegate &amp; You van (1993) Biotechnology Research 11:1548-1552.<br>
In vivo mutagenesis can be used to generate random mutations in any cloned DNA of interest by propagating the DNA, e.g., in a strain of £. coli that carries mutations in one or more of the DNA repair pathways. These "mutator" strains have a higher random mutation rate than that of a wild-type pimento. Propagating the DNA in one of these strains will eventually generate random mutations within the DNA. Such procedures are described in the references noted above.<br>
Other procedures for introducing diversity into a genome, e.g. a bacterial, fungal, animal or plant genome can be used in conjunction with the above described and/or referenced methods. For example, in addition to the methods above, techniques have been proposed which produce nucleic acid multimers suitable for transformation into a variety of species {see, e.g., Schellenberger U.S. Patent No. 5,756,316 and the references above). Transformation of a suitable host with such multimers, consisting of genes that are divergent with respect to one another, (e.g., derived from natural diversity or through application of site directed mutagenesis, editor prone PCR, passage through mutagenic bacterial strains, and the like), provides a source of nucleic acid diversity for DNA diversification, e.g., by an in vivo recombination process as indicated above.<br><br>
Alternatively, a multiplicity of monomelic polynucleotides sharing regions of partial sequence similarity can be transformed into a host species and recombined in vivo by the host cell. Subsequent rounds of cell division can be used to generate libraries, members of which, include a single, homogenous population, or pool of monomeric polynucleotides. Alternatively, the monomeric nucleic acid can be recovered by standard techniques, e.g., PCR and/or cloning, and recombined in any of the recombination formats, including recursive recombination formats, described above.<br>
Methods for generating multispecies expression libraries have been' described (in addition to the reference noted above, see, e.g., Peterson et al. (1998) U.S. Pat. No. 5,783,431 "Methods for Generating and Screening Novel Metabolic Pathways" and Thompson, et al. (1998) U.S. Pat. No. 5,824,485 "Methods for Generating and Screening Novel Marabou Pathways) and their use to identify protein activities of interest has been proposed (In addition to the references noted above, see. Short (1999) U.S. Pat. No. 5,958,672 "Protein Activity Screening of Clones Having DNA from Uncultivated Microorganisms"). Multispecies expression libraries include, in general, libraries comprising cDNA or genomic sequences from a plurality of species or strains, operably Nuked to appropriate regulatory sequences, in an expression cassette. The cDNA and/or genomic sequences are optionally randomly ligated to further enhance diversity. The vector can be a shuttle vector suitable for transformation and expression in more than one species of host organism, e.g., bacterial species, eukaryotic cells. In some cases, the library is biased by reselecting sequences which encode a protein of interest, or which hybridize to a nucleic acid of interest. Any such libraries can be provided as substrates for any of the methods herein described.<br>
The above described procedures have been largely directed to increasing nucleic acid and/ or encoded protein diversity. However, in many cases, not all of the • diversity is useful, e.g., functional, and contributes merely to increasing the background of variants that must be screened or selected to identify the few favorable variants. In some appKcations, it is desirable to reselect or prescreem libraries (e.g., an amplified library, a genomic library, a cDNA library, a normalized Horary, etc.) or other substrate nucleic acids prior to diversification, e.g., by recombination-based mutagenesis procedures, or to otherwise bias the substrates towards nucleic acids that encode functional products. For example, in the case of antibody engineering, it is possible to bias the diversity generating process toward antibodies with functional antigen binding<br><br>
sites by taking advantage of in vivo recombination events prior to manipulation by any of the described methods. For example, recombined Cars derived from B cell cDNA Harries can be muffed and assembled into framework regions (e.g., Jerold et al. (1998) 'Exploiting sequence space: shuffling in vivo formed complementarily determining regions into a master framework" Gene 215: 471) prior to diversifying according to any of the methods described herein.<br>
Libraries can be biased towards nucleic acids which encode proteins with desirable enzyme activities. For example, after identifying a clone from a library which exhibits a specified activity, the clone can be mutagenized using any known method for introducing DNA alterations. A library comprising the mutagenized homologues is then screened for a desired activity, which can be the same as or different from the initially specified activity. An example of such a procedure is proposed in Short (1999) U.S. Patent No. 5,939,250 for "Production of Enzymes Having Desired Activities by Mutagenesis." Desired activities can be identified by any method known in the art. For example, WO 99/10539 proposes that gene libraries cam be screened by combining extracts from the gene library with components obtained from metabolically rich cells and identifying combinations which exhibit the desired activity. It has also been proposed (e.g., WO 98/58085) that clones with desired activities can be identified by inserting bioactive substrates into samples of the Hbrary, and detecting bioactive fluorescence corresponding to the product of a desired activity using a fluorescent analyzer, e.g., a flow cytometry device, a CCD, a fluorometer, or a spectrophotometer.<br>
Libraries can also be biased towards nucleic acids which have specified characteristics, e.g., hybridization to a selected nucleic acid probe. For example, application WO 99/10539 proposes that polynucleotides encoding a desired activity (e.g., an enzymatic "activity, for example: a lipase, an esterase, a protease, a glycosidase, a glycosyl transferase, a phosphatase, a kinase, an oxygenated, a peroxides, a hydrolase, a hydratase, a nitrilase, a transaminase, an amidst or an acylated) can be identified from among genomic DNA sequences in the following manner. Single stranded DNA molecules from a population of genomic DNA are hybridized to a ligand-conjugated probe. The genomic DNA can be derived from either a cultivated or uncultivated microorganism, or from an environmental sample. Alternatively, the genomic DNA can be derived from a multicultural organism, or a tissue derived therefrom. Second strand synthesis can be conducted directly from the hybridization probe used in the capture, with<br><br>
or without prior release from the capture medium or by a wide variety of other strategies known in the art. Alternatively, the isolated single-stranded genomic DNA population can be fragmented without further cloning and used directly in, e.g., a recombination-based approach, that employs a single-stranded template, as described above.<br>
"Non-Stochastic" methods of generating nucleic acids and polypeptides are alleged in Short "Non-Stochastic Generation of Genetic Vaccines and Enzymes" WO 00/46344. These methods, including proposed non-stochastic polynucleotide reassembly and site-saturation mutagenesis methods be applied to the present invention as well. Random or semi-random mutagenesis using doped or degenerate oligonucleotides is also described in, e.g., Arlcin and Youvan (1992) "Optindzing nucleotide mixtures to encode specific subsets of amino acids for semi-random mutagenesis" Biotechnology 10:297-300; Reidhaar-Olson et al. (1991)."Random mutagenesis of protein sequences using oligonucleotide cassettes" Methods Enzvmol. 208:564-86; Lim and Sauer( 1991) "The role of internal packing interactions in determining the structure and stability of a protein" J. Mol. Biol. 219:359-76; Breyer and Sauer (1989) "Mutational analysis of the fine specificity of binding of monoclonal antibody 5 IF to Lambda repressor" J. Biol. Chem. 264:13355-60); and "Walk-Through Mutagenesis" (Crea, R; US Patents 5,830,650 and 5,798,208, and EP Patent 0527809 Bl.<br>
It will readily be appreciated that any of the above described techniques suitable for enriching a library prior to diversification can also be used to screen the products, or libraries of products, produced by the diversity generating methods.<br>
Kits for mutagenesis, library construction and other diversity generation methods are also commercially available. For example, kits are available from, e.g., Stratagene (e.g., Quick Change'™ site-directed mutagenesis kit; and Chameleon^^ double-stranded, site-directed mutagenesis kit), Bio/Can Scientific, Bio-Red (e.g., using the Kunkel method described above), Boehringer Mannheim Corp., Contac Laboratories, DNA Technologies, Epicentre Technologies (e.g., 5 prime 3 prime kit); Genpak Inc, Lemargo Inc, Life Technologies (Gibson BRL), New England Biolabs, Pharmacia Biotech, Promega Corp., Quantum Biotechnologies, Amerces Intemational pig (e.g., using the Eckstein method above), and Anglian Biotechnology Ltd (e.g., using the CarterAVinter method above).<br>
The above references provide many mutational formats, including recombination, recursive recombination, recursive mutation and combinations or<br><br>
recombination with other forms of mutagenesis, as well as many modifications of these formats. Regardless of the diversity generation format that is used, the nucleic acids of the invention can be recombined (with each other, or with related (or even unrelated) sequences) to produce a diverse set of recombinant nucleic acids, including, e.g., sets of homologous nucleic acids, as well as corresponding polypeptides.<br>
OTHER POLYNUCLEOTIDE COMPOSITIONS<br>
The invention also includes compositions comprising two or more polynucleotides of the invention (e.g., as substrates for recombination). The composition can comprise a library of recombinant nucleic acids, where the library contains at least 2, 3, 5,10, 20, or 50 or more, e.g., at least about 100, at least about 1000, at least about 10,000, or more, nucleic acids. The nucleic acids are optionally cloned into expression vectors, providing expression libraries.<br>
The invention also includes compositions produced by digesting one or more polynucleotide of the invention with a restriction endonuclease, an RNAse, or a DNAse (e.g., as is performed in certain of the recombination formats noted above); and compositions produced by fragmenting or shearing one or more polynucleotide of the invention by mechanical means (e.g., sonication, overtaxing, and the like), which can also be used to provide substrates for recombination in the methods above. Similarly, compositions comprising sets of oligonucleotides corresponding to more than one nucleic acid of the invention are useful as recombination substrates and are a feature of the invention. For convenience, these fragmented, sheared,, or oligonucleotide synthesized mixtures are referred to as fragmented nucleic acid sets.<br>
Also included in the invention are compositions produced by incubating one or more of the fragmented nucleic acid sets in the presence of ribonucleotide- or deoxyribonucleotide triphosphates and a nucleic acid polymerase. This resulting composition forms a recombination mixture for many of the recombination formats noted above. The nucleic acid polymerase may be an RNA polymerase, a DNA polymerase, or an RNA-directed DNA polymerase (e.g., a "reverse transcriptase"); the polymerase can be, e.g., a thermostable DNA polymerase (such as, VENT, TAQ, or the like).<br>
SUBTILISIN HOMOLOGUE POLYPEPTIDES<br>
The invention provides isolated or recombinant subtilisin homologue polypeptides, referred to herein as "subtilisin homologue polypeptides" or simply<br><br>
"subtilisin homologues." An isolated or recombinant subtilisin homologue polypeptide of the invention includes a polypeptide comprising a sequence selected from SEQ ID NO: 131 to SEQ ID NO:260, and conservatively modified variations therof.<br>
Several conclusions may be drawn from comparison of exemplary sequences exhibiting desirable functional attributes to the subtilisin homologue, Savinase®. While the amino acids substituted demonstrate a certain amount of variability, and while the same amino acids are not universally substituted in all the homologues sharing a functional characteristic, patterns of substitutions, or motifs, corresponding to functional attributes can be discerned. For example, distinct but overlapping amino acid substitutions are correlated with the selected properties of thermal stability, alkaline stability and stability in organic sheets, e.g., dimethylformamide (DMF). Exemplary sequence alignments are illustrated in Figure 2 A-C.<br>
Thermal Stability<br>
A comparison of exemplary subtilisin homologues with enhanced thermal<br>
stability reveals a number of variable amino acid positions. In comparison to Savinase®,<br>
several features are remarkable (Fig. 2A). The vast majority of novel subtilisin<br>
homologues with enhanced thermal satiety have substituted Rag for Ser99 (all amino<br>
acid comparisons are made relative to the mature Savinase® protein), Ala for Asnll4,<br>
Asn for Ser 206, and Arg for Thr207. In addition a cluster of variable residues is<br>
observed at positions 209-212. Notably, the amino acid substitutions at positions 99, 114,<br>
206 and 207 are non-conservative substitutions.<br>
pH Shifting<br>
Again, a number of variable positions are observed among exemplary<br>
subtilisin homologues with activity at shifted pH, and these there are striking<br>
substitutions relative to Savinase® (Fig. 2B). For example. Asp for Asn74, Glu for Ile77,<br>
Asn for Ser85, Asp for Glu87, Ser or Asp for Prol27, Ala or Tyr for Serl39 and Gly for<br>
Asnl98 are found in the majority of subtilisin homologues with activities at altered pH.<br>
Substitutions at amino acid positions 74,77, 85,127 198 are non-conservative<br>
substitutions.<br>
Activity in Organic Solvents<br>
Exemplary subtilisins demonstrating improved residual activity in the<br>
organic solvent, dimethylformamide (DMF), typically also have a number of notable<br>
amino acid substitutions (Fig. 2C). For example. Asp for Glul32, Asn for Ser97, Ala for<br><br>
Glyl 13, Ala or Thr for Asnl 14, Asn for Glyl 16, Asp or Ser for Prol27, Ala for Serl28, Tyr for Serl39, Asn for Serl54 and Ser for Alal56.<br>
Amino acid comparisons, such as those listed above, provide rational grounds for subsequent attempts at protein engineering of subtilisin homologues.<br>
Making Polypeptides<br>
Recombinant methods for producing and isolating subtilisin homologue<br>
polypeptides of the invention are described above. In addition to recombinant production,<br>
the polypeptides may be produced by direct peptide synthesis using solid-phase<br>
techniques (Stewart et al. (1969) Solid-Phase Peptide Synthesis, WH Freeman Co, San<br>
Francisco; Merrifield (1963) J. Am. Chem. Soc. 85:2149-2154). Peptide synthesis may<br>
be performed using manual techniques or by automation. Automated synthesis may be<br>
achieved, for example, using Apposed Bios stems 431A Peptide Synthesizer (Per kin<br>
Elmer, Foster City, Calif.) in accordance with the instructions provided by the<br>
manufacturer. For example, subsequences may be chemically synthesized separately and<br>
combined using chemical methods to provide full-length subtilisin homologues. Peptides<br>
can also be ordered from a variety of sources.<br>
Using Polypeptides<br>
Antibodies<br>
In another aspect of the invention, a subtilisin homologue polypeptide of<br>
the invention is used to produce antibodies which have, for example, diagnostic uses, e.g.,<br>
related to the activity, distribution, and expression of subtilisin homologues.<br>
Antibodies to subtilisin homologues of the invention may be generated by methods well known in the art. Such antibodies may include, but are not limited to, polyclonal, monoclonal, chimeric, humanized, single chain, Fab fragments and fragments produced by an Fab expression library,<br>
Subtilisin homologue polypeptides for jmtibody induction do not require biological activity; however, the polypeptide or oligopeptide must be antigenic. Peptides used to induce specific antibodies may have an amino acid sequence consisting of at least 10 amino acids, preferably at least 15 or 20 amino acids. Short stretches of a subtihsin homologue polypeptide may be fused with another protein, such as keyhole limpet hemocyanin, and antibody produced against the chimeric molecule.<br>
Methods of producing polyclonal and monoclonal antibodies are known to those of skill in the art, and many antibodies are available. See, e.g., Coalinga (1991)<br><br>
Current Protocols in Immunology Wiley/Greene, NY; and Harlow and Lane (1989) Antibodies: A Laboratory Manual Cold Spring Harbor Press, NY; Stites et al. (eds.) Basic and Clinical Immunology (4th ed.) Lange Medical Publications, Los Altos, CA, and references cited therein; Coding (1986) Monoclonal Antibodies: Principles and Practice (2d ed.) Academic Press, New York, NY; and Kohler and Milstein (1975) Nature 256: 495-497. Other suitable techniques for antibody preparation include selection of libraries of recombinant antibodies in phage or similar vectors. See, Hues et al. (1989) Science 246: 1275-1281; and Ward, et al. (1989) Nature 341: 544-546. Specific monoclonal and polyclonal antibodies and antisera will usually bind with a KD of at least about 0.1 |AM, preferably at least about 0.01 ^iM or better, and most typically and preferably, 0.001 ppm or better.<br>
Additional details antibody production and engineering techniques can be found in Borrebaeck (ed) (1995) Antibody Engineering, 2^^ Edition Freeman and Company, NY (Bareback); McCafferty et al. (1996) Antibody Engineering, A Practical Approach IRL at Oxford Press, Oxford, England (McCafferty), and Paul (1995) Antibody Engineering Protocols Humana Press, Towata, NJ (Paul).<br>
SEQUENCE VARIATIONS<br>
Conservatively Modified Variations<br>
Subtilisin homologue polypeptides of the present invention include<br>
conservatively modified variations of the sequences disclosed herein as SEQ ID N0:131<br>
to SEQ ID NO:260. Such conservatively modified validations comprise substitutions,<br>
additions or deletions which alter, add or delete a single amino acid or a small percentage<br>
of amino acids (typically less than about 5%, more typically less than about 4%, about<br>
2%, or about 1%) in any of SEQ ID N0:131 to SEQ H) NO:260.<br>
For example, a conservatively modified variation (e.g., deletion) of the 173 amino acid polypeptide identified herein as SEQ ID NO: 131 will have a length of at least 164 amino acids, preferably at least 166 amino acids, more preferably at least 170 amino acids, and still more preferably at least 171 amino acids, corresponding to a deletion of less than about 5%, about 4%, about 2% or about 1%, or less of the polypeptide sequence.<br>
Another example of a conservatively modified variation (e.g., a "conservatively substituted variation") of the polypeptide identified herein as SEQ ID NO:131 will contain "conservative substitutions", according to the six substitution groups<br><br>
set forth in Table 2 (supra), in up to about 9 residues (i.e., less than about 5%) of the 173 amino acid polypeptide.<br>
The subtilisin polypeptide sequence homologues of the invention, including conservatively substituted sequences, can be present as part of larger polypeptide sequences such as occur in a mature subtilisin protease, in a pre-pro subtilisin peptide or upon the addition of one of more domains for purification of the protein (e.g., poly Histidine (His) segments, FLAG tag segments, etc.). In the latter case, the additional functional domains have httle or no effect on the activity of the subtilisin portion of the protein, or where the additional domains can be removed by post synthesis processing steps such as by treatment with a protease.<br><br>
Because the polypeptides of the invention provide a variety of new polypeptide sequences as compared to other subtihsin homologues, the polypeptides also provide new structural features which can be recognized, e.g., in immunological assays. The generation of antisera which specifically binds the polypeptides of the invention, as well as the polypeptides which are bound by such antisera, are a feature of the invention.<br>
The invention includes subtilisin homologue proteins that specifically bind to or that are specifically immunoreactive with an antibody or antisera generated against an immunogen comprising an amino acid sequence selected from one or more of SEQ ID NO: SEQ ID N0:131 to SEQ ID NO:260. To eliminate cross-reactivity with other subtilisin homologues, the antibody or antisera is subtracted with available subtilisins, such as those represented by the proteins or peptides corresponding to GenBank accession numbers available as of April 3, 2000 and exemplified by P29600, P41362, P29599, P27693, P20724, P41363, P00780, P00781, P35835, P00783, P29142, P04189, P07518, P00782, P04072, P16396, P29140, P29139, P08594, P16588, P11018, P54423, P40903, P23314, P23653, P33295, P42780, and P80146. Where the accession number corresponds to a nucleic acid, a polypeptide encoded by the nucleic acid is generated and used for antibody/antisera subtraction purposes.<br>
In one typical format, the iminunoassay uses a polyclonal antiserum which was raised against one or more polypeptide comprising one or more of the sequences corresponding to one or more of: SEQ ID NO: 131 to SEQ ID NO:260, or a substantial subsequence thereof (i.e., at least about 30% of the full length sequence provided). The<br><br>
full set of potential polypeptide immunogens derived from SEQ ID N0:131 to SEQ ID NO:260 are collectively referred to below as "the immunogenic polypeptides." The resulting antisera is optionally selected to have low cross-reactivity against the control subtilisin homologues, other known subtilisin homologues and any such cross-reactivity is removed by immunoabsorbtion with one or more of the control subtilisin homologues, prior to use of the polyclonal antiserum in the immunoassay.<br>
In order to produce antisera for use in an immunoassay, one or more of the immunogenic polypeptides is produced and purified as described herein. For example, recombinant protein may be produced in a bacterial cell line. An inbred strain of mice (used in this assay because results are more reproducible due to the virtual genetic identity of the mice) is immunized with the immunogenic protein(s) in combination with a standard adjuvant, such as Freund's adjuvant, and a standard mouse immunization protocol {see, Harlow and Lane (1988) Antibodies, A Laboratory Manual Cold Spring Harbor Publications, New York, for a standee’s description of antibody generation, immunoassay formats and conditions that can be used to determine specific immunoreactivity). Alternatively, one or more synthetic or recombinant polypeptide derived from the sequences disclosed herein is conjugated to a carrier protein and used as an immunogen.<br>
Polyclonal sera are collected and tattered against the immunogenic polypeptide in an immunoassay, for example, a solid phase immunoassay with one or more of the immunogenic proteins immobilized on a solid support. Polyclonal antisera with a titer of 10*^ or greater are selected, pooled and subtracted with the control subtilisin polypeptides, e.g., those identified from GenBank as noted, to produce subtracted pooled titered polyclonal antisera.<br>
The subtracted pooled titered polyclonial antisera are tested for cross reactivity against the control subtilisin homologues. Preferably at least two of the immunogenic subtilisins are used in this determination, preferably in conjunction with at least two of the control subtilisin homologues, to identify antibodies which are specifically bound by the immunogenic protein(s).<br>
In this comparative assay, discriminatory binding conditions are determined for the subtracted titered polyclonal antisera which result in at least about a 5-10 fold higher signal to noise ratio for binding of the titered polyclonal antisera to the immunogenic subtilisin homologues as compared to binding to the control subtilisin<br><br>
homologues. That is, the stringency of the binding reaction is adjusted by the addition of non-specific competitors such as albumin or non-fat dry milk, or by adjusting salt conditions, temperature, or the like. These binding conditions are used in subsequent assays for determining whether a test polypeptide is specifically bound by the pooled subtracted polyclonal antisera. In particular, test polypeptides which show at least a 2-5x higher signal to noise ratio than the control polypeptides under discriminatory binding conditions, and at least about a Vz signal to noise ratio as compared to the immunogenic polypeptide(s), shares substantial structural similarity with the immunogenic polypeptide as compared to known subtilisin, and is, therefore a polypeptide of the invention.<br>
In another example, immunoassays in the competitive binding format are used for detection of a test polypeptide. For example, as noted, cross-reacting antibodies are removed from the pooled antisera mixture by immunoabsorbtion with the control subtilisin polypeptides. The immunogenic polypeptide(s) are then immobilized to a solid support which is exposed to the subtracted pooled antisera. Test proteins are added to the assay to compete for binding to the pooled subtracted antisera. The ability of the test protein(s) to compete for binding to the pooled subtracted antisera as compared to the immobilized protein(s) is compared to the ability of the immunogenic polypeptide(s) added to the assay to compete for binding (the immunogenic polypeptides compete effectively with the immobilized immunogenic polypeptides for binding to the pooled antisera). The percent cross-reactivity for the test proteins is calculated, using standard calculations.<br>
In a parallel assay, the ability of the control proteins to compete for binding to the pooled subtracted antisera is determined as compared to the ability of the immunogenic polypeptide(s) to compete for binding to the antisera. Again, the percent cross-reactivity for the control polypeptides is calculated, using standard calculations. Where the percent cross-reactivity is at least 5-lOx as high for the test polypeptides, the test polypeptides are said to specifically bind the pooled subtracted antisera.<br>
In general, the immunoabsorbed and pooled antisera can be used in a competitive binding immunoassay as described herein to compare any test polypeptide to the immunogenic polypeptide(s). In order to make this comparison, the two polypeptides are each assayed at a wide range of concentrations and the amount of each polypeptide required to inhibit 50% of the binding of the subtracted antisera to the immobilized protein is determined using standard techniques. If the amount of the test polypeptide<br><br>
required is less than twice the amount of the immunogenic polypeptide that is required, then the test polypeptide is said to specifically bind to an antibody generated to the immunogenic protein, provided the amount is at least about 5-lOx as high as for a control polypeptide.<br>
As a final determination of specificity, the pooled antisera is optionally fully immunosorbed with the immunogenic polypeptide(s) (rather than the control polypeptides) until litde or no binding of the resulting immunogenic polypeptide subtracted pooled antisera to the immunogenic polypeptide(s) used in the immunosorbtion is detectable. This fully immunosorbed antisera is then tested for reactivity with the test polypeptide. If little or no reactivity is observed (i.e., no more than 2x the signal to noise ratio observed for binding of the fully immunosorbed antisera to the immunogenic polypeptide), then the test polypeptide is specifically bound by the antisera elicited by the immunogenic protein.<br>
CLEANING SOLUTIONS<br>
The subtilisin homologues of the invention are favorably used in compositions that serve as cleaning solutions in wide variety of applications, including laundry detergents, contact lens cleansing solutions, and dry cleaning, among others.<br>
For example, the present invention provides the use of the novel subtilisin homologues of the invention in cleaning and detergent compositions, as well as such compositions containing mutant subtilisin enzymes. Such cleaning and detergent compositions can in principle have any physical form, but the subtilisin homologues are preferably incorporated in liquid detergent compositions or in detergent compositions in the form of bars, tablets, sticks and the like for direct application, wherein they exhibit improved enzyme stability or performance.<br>
Among the liquid compositions of the present invention are aqueous liquid detergents having for example a homogeneous physical character, e.g. they can consist of a micellar solution of surfactants in a continuous aqueous phase, so-called isotropic liquids. Alternatively, they can have a heterogeneous physical phase and they can be structured, containing suspended solid particles such as particles of builder materials e.g. of the kinds mentioned below. In addition, the liquid detergents according to the present invention can include an enzyme stabilization system, comprising calcium ion, boric acid,<br><br>
propylene glycol and/or short chain carboxylic acids. Optionally, the detergents include additional enzyme components including cellulase, lipases, or proteases.<br>
In addition, powder detergent compositions can include, in addition to any one or more of the subtilisin homologues of the invention as described herein, such components as builders (such as phosphate or zeolite builders), surfactants (such as anionic, cationic, non-ionic or zwitterionic type surfactants), polymers (such as acrylic or equivalent polymers), bleach systems (such as perborate- or amino-containing bleach precursors or activators), structurants (such as silicate structurants), alkali or acid to adjust pH, humectants, and/or neutral inorganic salts. Furthermore, a number of other ingredients are normally present in the compositions of the invention, such as co surfactants, tart rate succinate builder, neutralization system, suds suppressor, other enzymes and other optional components.<br>
INTEGRATED SYSTEMS<br>
The present invention provides computers, computer readable media and integrated systems comprising character strings corresponding to the sequence information herein for the polypeptides and nucleic acids herein, including, e.g., those sequences listed herein and the various silent substitutions and conservative substitutions thereof.<br>
Various methods and genetic algorithms (Goes) known in the art can be used to detect homology or similarity between different character strings, or can be used to perform other desirable functions such as to control output files, provide the basis for making presentations of information including the sequences and the like. Examples include BLAST, discussed supra.<br>
Thus, different types of homology and similarity of various stringency and length can be detected and recognized in the integrated systems herein. For example, many homology determination methods have been designed for comparative analysis of sequences of biopolymers, for spell-checking in word processing, and for data retrieval from various databases. With an understanding of double-helix pair-wise complement interactions among 4 principal nucleobases in natural polynucleotides, models that simulate annealing of complementary homologous polynucleotide strings can also be used as a foundation of sequence alignment or other operations typically performed on the character strings corresponding to the sequences herein (e.g., word-processing<br><br>
manipulations, construction of figures comprising sequence or subsequence character strings, output tables, etc.). An example of a software package with Goes for calculating sequence similarity is BLAST, which can be adapted to the present invention by inputting character strings corresponding to the sequences herein.<br>
Similarly, standard desktop applications such as word processing software (e.g., Microsoft Word or Corel WordPerfect" ) and database software (e.g., spreadsheet software such as Microsoft Excel"™, Corel Quattro Pro" or database programs such as Microsoft Access"™ or Paradox"™) can be adapted to the present invention by inputting a character string corresponding to the subtilisin homologues of the invention (either nucleic acids or proteins, or both). For example, the integrated systems can include the foregoing softwood-e having the appropriate character string information, e.g., used in conjunction with a user interface (e.g., a GUI in a standard operating system such as a Windows, Macintosh or LINUX system) to manipulate strings of characters. As noted, specialized alignment programs such as BLAST can also be incorporated into the systems of the invention for alignment of nucleic acids or proteins (or corresponding character strings).<br>
Integrated systems for analysis in the present invention typically include a digital computer with GO software for aligning sequences, as well as data sets entered into the software system comprising any of the sequences herein. The computer can be, e.g., a PC (Intel x86 or Pentium chip- compatible DOS™, 0S2™ WINDOWS™ WINDOWS NT™, WINDOWS95™, WINDOWS98™ LINUX based machine, a MACINTOSH™, Power PC, or a UNIX based (e.g., SUN™ work station) machine) or other commercially common computer which is known to one of skill. Software for aligning or otherwise manipulating sequences is available, or can easily be constructed by one of skill using a standard programming language such as Visual basic, Fortran, Basic, lava, or the like.<br>
Any controller or computer optionally includes a monitor which is often a cathode ray tube ("CRT") display, a flat panel display (e.g., active matrix liquid crystal display, Hquid crystal display), or others. Computer circuitry is often placed in a box which includes numerous integrated circuit chips, such as a microprocessor, memory, interface circuits, and others. The box also optionally includes a hard disk drive, a floppy disk drive, a high capacity removable drive such as a write able CD-ROM, and other common peripheral elements. Inputting devices such as a keyboard or mouse optionally<br><br>
provide for input from a user and for user selection of sequences to be compared or otherwise manipulated in the relevant computer system.<br>
The computer typically includes appropriate software for receiving user instructions, either in the form of user input into a set parameter fields, e.g., in a GUI, or in the form of preprogrammed instructions, e.g., preprogrammed for a variety of different specific operations. The software then converts these instructions to appropriate language for instructing the operation of the fluid direction and transport controller to carry out the desired operation.<br>
The software can also include output elements for controlling nucleic acid synthesis (e.g., based upon a sequence or an alignment of a sequences herein) or other operations which occur downstream from an alignment or other operation performed using a character string corresponding to a sequence herein.<br>
In an additional aspect, the present invention provides kits embodying the methods, composition, systems and apparatus herein. Kits of the invention optionally comprise one or more of the following: (1) an apparatus, system, system component or apparatus component as described herein; (2) instructions for practicing the methods described herein, and/or for operating the apparatus or apparatus components herein and/or for using the compositions herein; (3) one or more subtilisin composition or component; (4) a container for holding components or compositions, and, (5) packaging materials.<br>
In a further aspect, the present invention provides for the use of any apparatus, apparatus component, composition or Icit herein, for the practice of any method or assay herein, and/or for the use of any apparatus or Icit to practice any assay or method herein.<br>
EXAMPLES<br>
Recombinant, (e.g., shuffled) library sequences corresponding to the<br>
diversified region (amino acids 55-227) in the context of Savinase® protease in an expression vector were cloned into a Bacillus 168 apr nprB strain (Harwood and Cutting (1990) Molecular Biological Methods for Bacillus. J. Wiley and Sons, New York) for expression and screening. Activity was compared to plait of Savinase®. Genes were sequenced using an Applied Bios stems 310 Sequencer according to the manufacturers directions.<br><br>
Bacillus colonies comprising library produced clearing halos on casein plates were grown to stationary phase in LB medium. The supernatant from this medium contained secreted protease and was diluted 100-fold (for pH 5.5 and pH 10 reactions) or 200-fold (for pH 7.5 reactions) into the reaction mixture. Protease activities in the culture supematants were assayed using BODEPY FL casein as a substrate (Jones et al. (1997) Anal Biochem 251: 144). Huorescence of this multi-fluorophore casein derivative is internally quenched when the protein is intact. Proteolysis causes separation of neighboring fluorophores, relieving quenching, so activity is measured as an increase in fluorescence with time. The reaction mixture contained 5  /ml BODIPY EL casein, 1 mM CaCl2, and either 50 in sodium borate (pH 10), 50 mM Tris/HCl (pH 7.5), or 50 mM MES (pH 5.5). All reactions were performed at room temperature for 40-70 minutes. Fluorescence was monitored at-535nm using an excitation wavelength of 485nm (BMG Flusters). The cavy(%) observed for independent determinations with the Savinase® strain was 
The pH dependence of the exemplary comes was determined by measuring activity at pH's 5.5, 7.5, and 10. ThermostabiUty was measured as the residual activity at pH 10 after incubation at 70°C for 5 minutes. Function in organic solvent was assayed as activity in 35% DMF at pH 7.5. Representative values are given in Table 3. Assay values obtained for additional clones are provided in Table 4.<br>
The most dramatic increase in activity v/as at pH 5.5, where clones encoding subtilisin homologues with between 2 and 4-fold greater activity than Savinase® were obtained.<br>
Combinations of properties were evaluated by simultaneously comparing the activities of the recovered clones for pairs of propellers. Seventy-seven of the clones demonstrating the highest activity at 23 °C and pHlO were evaluated for the additional properties of residual activity in organic solvent and stability to heat treatment. The seventy-seven clones that were highly active at pH 10 show a broad distribution of properties under these two additional reaction conditions. Enzymes with up to nearly four times more residual after heat treatment or up to 50% greater residual activity in 35% DMF (at pH 7.5) were obtained. Many individuals were also obtained that were both<br><br>
more heat-stable and more active in organic solvent than Savinase® or any of the naturally occurring subtilisins.<br>
The submission homologue library was tested for combinations of properties by plotting the activities of a large number (i.e., greater than 650) active clones for pairs of properties. Activities are expressed relative to Savinase®. In every case, proteases with higher activities that Savinase® were obtained. For example, Clones 3A3,3B3 and 4C6 possess activity levels significantly higher than Savinase® at pHlO, while maintaining heat stability. Other clones show novel activities: 7C6 is active at both pH 10 and pH 5.5; 6A4, 7A2,4D7 and 5E1 all showed a much greater activity at pH 5.5 than at pHlO as compared to Savinase®.<br>
While the foregoing invention has been described in some detail for purposes of clarity and understanding, it will be clear to one skilled in the art from a reading of this disclosure that various changes in form and detail can be made without departing from the true scope of the invention. For  , all the techniques, methods, compositions, apparatus and systems described above may be used in various combinations. All pubhcations, patents, patent applications, or other documents cited in this appKcations are incorporated by reference in their entirety for all purposes to the same extent as if each individual publication, patent, patent supplication, or other document were individually indicated to be incorporated by reference for all purposes.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
1.     An isolated or recombinant polypeptide with an improved endow-<br>
I :   selected from among increased thermo tolerance, increased activity at ;  increased activity at alkaline pH, increased activity at acid pH,<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
(iii) Displaying the selected character strings, or aligning the selected character strings with an additional character string.<br>
108.	The method of claim 107, further comprising displaying an augment<br>
of the selected character string with the additional character string.<br>
109,	The method of claim 108, further comprising displaying the list.<br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtY2xhaW1zIGdyYW5kLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-claims grand.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-description(complete) filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMTU5MC1jaGUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-1590-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="208914-a-method-of-changing-the-colour-of-a-diamond.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="208916-a-colour-chart-and-a-method-of-preparing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>208915</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/1590/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>38/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Sep-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Oct-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. MAXYGEN INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>515 Galveston Drive, Redwood City, CALIFORNIA 94063</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NESS, Jon, E</td>
											<td>1220 N. Fairoaks Avenue #2115, Sunnyvale, CA 94089</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2001/010781</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-04-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/194,143</td>
									<td>2000-04-03</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script data-cfasync="false" src="../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/208915-an-isolated-or-recombinant-polypeptide-having-endoprotease-activity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:05:50 GMT -->
</html>
