<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209120-streptomyces-avermitilis-gene-directing-the-ratio-of-b2-b1-avermectins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:51:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209120:STREPTOMYCES AVERMITILIS GENE DIRECTING THE RATIO OF B2:B1 AVERMECTINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">STREPTOMYCES AVERMITILIS GENE DIRECTING THE RATIO OF B2:B1 AVERMECTINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>. A polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avernitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:l), or a degenerate variant thereof, but which nucleotide sequence comprises one or more mutations encoding an amino acid substitution at one or more amino acid residues corresponding to amino acid .position 38, 48, 89, 99,111,&#x27; 136,154, 179, 228, 238, 289, or 298 of SEQ ID No:2, such that cells of S. avernitilis strain ATCC 53692 in which the wild-type aLyeG allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence produce a class 2:1 ratio of avermectins that is different from the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
    [See Section 10, Rule 13]<br>
"STREPTOMYCES AVERMITILIS GENE DIRECTING THE RATIO OF B2:B1<br>
AYERMECTINS".<br>
PFIZER PRODUCTS, INC., A Corporation Organized Under The Laws Of The State Of Connecticut, United States Of America, Of Eastern Point Road, Groton, Connecticut 06340, United States Of America,<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed:-<br><br>
The present invention relates to a polynucleotide molecule<br>
1. FIELD OF THE INVENTION<br>
The present invention is directed to compositions and methods for producing avermectins, and is primarily in the field of animal health. More particularly, the present invention relates to polynucleotide molecules comprising nucleotide sequences encoding an AveC gene product, which can be used to modulate the ratio of class 2:1 avermectins produced by fermentation of cultures of Streptomyces avermitilis, and to compositions and methods for screening for such polynucleotide molecules. The present invention further relates to vectors, transformed host cells, and novel mutant strains of S. avermitilis in which the aveC gene has been mutated so as to modulate the ratio of class 2:1 avermectins produced.<br>
2. BACKGROUND OF THE INVENTION 2.1. Avermectins Streptomyces species produce a wide variety of secondary metabolites, including the avermectins, which comprise a series of eight related sixteen-membered macrocyclic lactones having potent anthelmintic and insecticidal activity.   The eight distinct but closely related compounds are referred to as A1a, A1b, A2a, A2b, B1a, B1b, B2a and B2b. The "a" series of compounds refers to the natural avermectin where the substituent at the C25 position is (S&gt;-sec-butyl, and the "b" series refers to those compounds where the substituent at the C25 position is isopropyl. The designations "A" and "B" refer to avermectins where the substituent at the C5 position is methoxy and hydroxy, respectively.    The numeral "1" refers to avermectins where a double bond is present at the C22.23 position, and the numeral "2" refers to avermectins having a hydrogen at the C22 position and a hydroxy at the C23 position. Among the related avermectins, the B1 type of avermectin is recognized as having the most effective antiparasitic and pesticidal activity, and is therefore the most commercially desirable avermectin.<br>
The avermectins and their production by aerobic fermentation of strains of S. avermitilis are described in United States Patents 4,310,519 and 4,429,042. The biosynthesis of natural avermectins is believed to be initiated endogenously from the CpA thioester analogs of isobutyric acid and S-(+)-2-methyl butyric acid.<br>
A combination of both strain improvement through random mutagenesis and the use of exogenously supplied fatty acids has led to the efficient production of avermectin analogs. Mutants of S. avermitilis that are deficient in branched-chain 2-oxo acid dehydrogenase (bkd<br><br>
deficient'mutants) can only produce avermectins when fermentations are supplemented with fatty acids. Screening and isolation of mutants deficient in branched-chain dehydrogenase activity (e.g., S. avermitilis, ATCC 53567) are described in European Patent (EP) 276103. Fermentation of such mutants in the presence of exogenously supplied fatty acids results in production of only the four avermectins corresponding to the fatty acid employed. Thus, supplementing fermentations of S. avermitilis (ATCC 53567) with S-(+)-2-methylbutyric acid results in production of the natural avermectins A1a, A2a, B1a and B2a; supplementing fermentations with isobutyric acid results in production of the natural avermectins A1 b, A2b, B1b, and B2b; and supplementing fermentations with cyclopentanecarboxylic acid results in the production of the four novel cyclopentylavemnectins A1, A2, B1, and B2.<br>
If supplemented with other fatty acids, novel avermectins are produced. By screening over 800 potential precursors, more than 60 other novel avermectins have been identified. (See, e.g., Dutton et al, 1991, J. Antibiot 44:357-365; and Banks et a/., 1994, Roy. Soc. Chem. 147:16-26). In addition, mutants of S. avermitilis deficient in 5-O-methyltransferase activity produce essentially only the B analog avermectins. Consequently, S. avermitilis mutants lacking both branched-chain 2-oxo acid dehydrogenase and 5-O-methyltransferase activity produce only the B avermectins corresponding to the fatty acid employed to supplement the fermentation. Thus, supplementing such double mutants with S-(+)-2-methylbutyric acid results in production of only the natural avermectins B1a and B2a, while supplementing with isobutyric acid or cyclopentanecarboxylic acid results in production of the natural avermectins B1b and B2b or the novel cyclopentyl B1 and B2 avermectins, respectively. Supplementation of the double mutant strain with cyclohexane carboxylic acid is a preferred method for producing the commercially important novel avermectin, cyclohexylavermectin B1 (doramectin). The isolation and characteristics of such double mutants, e.g., S. avermitilis (ATCC 53692), is described in EP 276103.<br>
2.2. Genes Involved In Avermectin Biosynthesis In many cases, genes involved in production of secondary metabolites and genes encoding a particular antibiotic are found clustered together on the chromosome. Such is the  case, e.g., with the Streptomyces polyketide synthase gene cluster (PKS) (see Hopwood and Sherman, 1990, Ann. Rev. Genet. 24:37-66). Thus, one strategy for cloning genes in a  piosynthetic pathway has been to isolate a drug resistance gene and then test adjacent egions of the chromosome for other genes related to the biosynthesis of that particular intibiotic. Another strategy for cloning genes involved in the biosynthesis of important netabolites has been complementation of mutants.   For example, portions of a DNA library<br><br>
from an organism capable of producing a particular metabolite are introduced into a non-producing mutant and transformants screened for production of the metabolite. Additionally, hybridization of a library using probes derived from other Streptomyces species has been used to identify and clone genes in biosynthetic pathways.<br>
Genes involved in avermectin biosynthesis (ave genes), like the genes required for biosynthesis of other Streptomyces secondary metabolites (e.g., PKS), are found clustered on the chromosome. A number of ave genes have been successfully cloned using vectors to complement S. avermitilis mutants blocked in avermectin biosynthesis. The cloning of such genes is described in U.S. Patent 5,252,474. In addition, Ikeda et al., 1995, J. Antibiot. 48:532-534, describes the localization of a chromosomal region involving the C22.23 dehydration step (aveC) to a 4.82 Kb BamHI fragment of S. avermitilis, as well as mutations in the aveC gene that result in the production of a single component B2a producer. Since ivermectin, a potent anthelmintic compound, can be produced chemically from avermectin B2a, such a single component producer of avermectin B2a is considered particularly useful for commercial production of ivermectin.<br>
Identification of mutations in the aveC gene that minimize the complexity of avermectin production, such as, e.g., mutations that decrease the B2:B1 ratio of avermectins, would simplify production and purification of commercially important avermectins.<br>
3. SUMMARY OF THE INVENTION The present invention provides an isolated polynucleotide molecule comprising the complete aveC ORF of S. avermitilis or a substantial portion thereof, which isolated polynucleotide molecule lacks the next complete ORF that is located downstream from the aveC ORF in situ in the S. avermitilis chromosome. The isolated polynucleotide molecule of the present invention preferably comprises a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC2096D4), or that is the same as the nucleotide sequence of the aveC ORF of FIGURE 1 (SEQ ID NO:1), or substantial portion thereof. The present invention further provides an isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:1 or a degenerate variant thereof.<br>
The present invention further provides an isolated polynucleotide molecule having a nucleotide sequence that is homologous to the S. avermitilis AveC gene product-encoding sequence of pJasmid pSE186 (ATCC 2£96Q4)or to the nucJeotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof.<br><br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide having an amino acid sequence that is homologous to the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or substantial portion thereof.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence encoding an AveC homolog gene product. In a preferred embodiment, the isolated polynucleotide molecule comprises a nucleotide sequence encoding the AveC homolog gene product from S. hygroscopicus, which homolog gene product comprises the amino acid sequence of SEQ ID NO:4 or a substantial portion thereof. In a preferred embodiment, the isolated polynucleotide molecule of the present invention that encodes the S. hygroscopicus AveC homolog gene product comprises the nucleotide sequence of SEQ ID NO:3 or a substantial portion thereof.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus nucleotide sequence of SEQ ID NO:3. The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide that is homologous to the S. hygroscopicus AveC homolog gene product having the amino acid sequence of SEQ ID NO:4. The present invention further provides oligonucleotides that hybridize to a polynucleotide molecule having the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3, or to a polynucleotide molecule having a nucleotide sequence which is the complement of the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3.<br>
The present invention further provides recombinant cloning vectors and expression vectors that are useful in cloning or expressing a polynucleotide of the present invention including polynucleotide molecules comprising the aveC ORF of S. avermitilis or an aveC homolog ORF. In a non-limiting embodiment, the present invention provides plasmid pSE186 (ATCC 209604), which comprises the entire ORF of the aveC gene of S. avermitilis. The present invention further provides transformed host cells comprising a polynucleotide molecule or recombinant vector of the invention, and novel strains or cell lines derived therefrom.<br>
The present invention further provides a recombinantly expressed AveC gene product or AveC homplog gene product, or a substantial portion thereof, that has been substantially purified or isolated, as well as homologs thereof. The present invention further provides a method for producing a recombinant AveC gene product, comprising culturing a host cell<br><br>
transformed with a recombinant expression vector, said recombinant expression vector comprising a polynucleotide molecule having a nucleotide sequence encoding an AveC gene product or AveC homolog gene product, which polynucleotide molecule is in operative association with one or more regulatory elements that control expression of the polynucleotide molecule in the host cell, under conditions conducive to the production of the recombinant AveC gene product or AveC homolog gene product, and recovering the AveC gene product or AveC homolog gene product from the cell culture.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the S. avermitilis AveC allele, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprises one or more mutations, so that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence produce a different ratio or amount of avermectins than are produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele. According to the present invention, such polynucleotide molecules can be used to produce novel strains of 5. avermitilis that exhibit a detectable change in avenmectin production compared to the same strain that instead expresses only the wild-type aveC allele. In a preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce avermectins in a reduced class 2:1 ratio compared to that from the same strain that instead expresses only the wild-type aveC allele.  In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce increased levels of avermectins compared to the same strain that instead expresses only a single wild-type aveC allele.   In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis in which the aveC gene has been inactivated.<br>
The present invention provides methods for identifying mutations of the aveC ORF of S. avermitilis capable of altering the ratio and/or amount of avermectins produced. In a preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF capable of altering the class 2:1 ratio of avermectins produced, comprising: (a) determining the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product has been<br><br>
introduced and is being expressed; (b) determining the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only the wild-type aveC allele or the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) to the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b); such that if the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) is different from the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b), then a mutation of the aveC ORF capable of altering the class 2:1 ratio of avermectins has been identified. In a preferred embodiment, the class 2:1 ratio of avermectins is reduced by the mutation.<br>
In a further preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF or genetic constructs comprising the aveC ORF capable of altering the amount of avermectins produced, comprising: (a) determining the amount of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product or comprising a genetic construct comprising a nucleotide sequence encoding an AveC gene product has been introduced and is being expressed; (b) determining the amount of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a single aveC allele having the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the amount of avermectins produced by the S. avermitilis cells of step (a) to the amount of avermectins produced by the S. avermitilis cells of step (b); such that if the amount of avermectins produced by the S. avermitilis cells of step (a) is different from the amount of avermectins produced by. the S. avermitilis cells of step (b), then a mutation of the aveC ORF or a genetic construct capable of altering the amount of avermectins has been identified. In a preferred embodiment, the amount of avermectins produced is increased by the mutation.<br>
The present invention further provides recombinant vectors that are useful for making novel strains of S. avermitilis having altered avermectin production. For example, the present invention provides vectors that can be used to target any of the polynucleotide molecules comprising the mutated nucleotide sequences of the present invention .to the site of the aveC gene of the S. avermitilis chromosome to either insert into or replace the aveC allele or ORF or a portion thereof by homologous recombination. According to the present invention, however, a polynucleotide molecule comprising a mutated nucleotide sequence of the present<br><br>
invention provided herewith can also function to modulate avermectin biosynthesis when<br>
inserted into the S. avermitilis chromosome at a site other than at the aveC gene, or when<br>
maintained episomally in S. avermitilis cells.   Thus, the present invention also provides<br>
vectors comprising a polynucleotide molecule comprising a mutated nucleotide sequence of       the present invention, which vectors can be used to insert the polynucleotide molecule at a site in the S. avermitilis chromosdme other than at the aveC gene, or to be maintained episomally.   In a preferred embodiment, the present invention provides gene replacement vectors that can be used to insert a mutated aveC allele into the S. avermitilis chromosome to generate novel strains of cells that produce avermectins in a reduced class 2:1  ratio       compared to the cells of the same strain which instead express only the wild-type aveC allele. The present invention further provides methods for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce altered ratios and/or amounts of avermectins compared to cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele.   In a preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce an altered class 2:1 ratio of avermectins compared to cells of the same strain of S. avermitilis that instead express only a wild-type aveC allele, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele that encodes a gene product that alters the class 2:1  ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated allelecompared to cells of the same strain that instead express only the wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered class 2:1 ratio compared to the class 2:1 ratio produced by cells of the strain that instead express the wild-type aveC<br>
allele. In a preferred embodiment, the class 2:1 ratio of avermectins produced is reduced in<br>
cells of the novel strain.	 <br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that produce altered amounts of avermectin, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele or a genetic construct comprising the aveC allele, the expression of which results in an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct as compared to cells of the same strain that instead express only the wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered amount compared to the amount of avermectins produced by cells of the strain that instead express only the wild-type aveC allele.   In a<br><br>
preferred embodiment, the amount of avermectins produced is increased in cells of the novel strain.<br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis, the cells of which comprise an inactivated aveC allele, comprising transforming cells of a strain of S. avermitilis that express any aveC allele with a vector that inactivates the aveC allele, and selecting transformed cells in which the aveC allele has been inactivated.<br>
The present invention further provides novel strains of S. avermitilis comprising cells that have been transformed with any of the polynucleotide molecules or vectors comprising a mutated nucleotide sequence of the present invention. In a preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain produce avermectins in an altered class 2:1 ratio compared to cells of the same strain that instead express only the wild-type aveC allele. In a more preferred embodiment, the cells of the novel strain produce avermectins in a reduced class 2:1 ratio compared to cells of the same strain that instead express only the wild-type aveC allele. Such novel strains are useful in the large-scale production of commercially desirable avermectins such as doramectin.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele, or a genetic construct comprising the aveC allele, in place of, or in addition to, the aveC allele native thereto, which results in the production by the cells of an altered amount of avermectins compared to the amount of avermectins produced by cells of the same strain that instead express only the wild-type aveC allele. In a preferred embodiment, the novel cells produce an increased amount of avermectins.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells in which the aveC gene has been inactivated.  Such strains are useful both for the different spectrum of avermectins that they produce compared to the wild-type strain, and in complementation screening assays as described herein, to determine whether targeted or random mutagenesis of the aveC gene affects avermectin production.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele compared to cells of the<br><br>
same strain which do not express the mutated aveC allele but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the class 2:1 ratio of avermectins produced by cells expressing the mutation is reduced. This process provides increased efficiency in the production of commercially valuable avermectins such as doramectin.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele or a genetic construct comprising an aveC allele that results in the production of an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct compared to cells of the same strain which do not express the mutated aveC allele or genetic construct but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the amount of avermectins produced by cells expressing the mutation or genetic construct is increased.<br>
The present invention further provides a novel composition of avermectins produced by a strain of S. avermitilis expressing a mutated aveC allele of the present invention, wherein the avermectins are produced in a reduced class 2:1 ratio as compared to the class 2:1 ratio of avermectins produced by cells of the same strain of S. .avermitilis that do not express the mutated aveC allele 6ut instead express only the wild-type aveC allele. The novel avermectin<br>
r<br>
composition can be present as produced in fermentation culture fluid, or can be harvested therefrom, and can be partially or substantially purified therefrom.<br>
4. BRIEF DESCRIPTION OF THE FIGURES<br>
FIGURE 1. DNA sequence (SEQ ID NO:1) comprising the S. avermitilis aveC ORF, and deduced amino acid sequence (SEQ ID NO;2).<br>
FIGURE 2. Plasmid vector pSE186 (ATCC 209604) comprising the entire ORF of the aveC gene of S. avermitilis.<br>
FIGURE 3. Gene replacement vector pSE180 (ATCC 209605) comprising the ermE gene of Sacc. erythraea inserted into the aveC ORF of S. avermitilis.<br>
FIGURE 4. BamHl restriction map of the avermectin polyketide synthase gene cluster from S. avermitilis with five overlapping cosmid clones identified (i.e., pSE65, pSE66, pSE67, pSE68, pSE69). The relationship of pSE118 and pSE119 is also indicated.<br><br>
FIGURE 5.   HPLC analysis of fermentation products produced by S. avermitilis strains. Peak quantitation was performed by comparison to standard quantities of cyclohexyl B1. Cyclohexyl B2 retention time was 7.4-7.7 min; cyclohexyl B1 retention time was 11.9-12.3! min.   FIG. 5A.   S. avermitilis strain SE180-11 with an inactivated aveC ORF.   FIG. 5B.   S.       avermitilis strain SE180-11 transformed with pSE186 (ATCC 209604). FIG. 5C.  S. avermitilis strain SE180-11 transformed with pSE187.    FtG. 5D.    S. avermitilis strain SE180-11 transformed with pSE188.<br>
FIGURE 6.   Comparison of deduced amino acid sequences encoded by the aveC ORF   of   S.   avermitilis   (SEQ   ID   NO:2),   an   aveC   homolog   partial   ORF   from   S.       griseochromogenes (SEQ ID NO:5), and the aveC homolog ORF from S. hygroscopicus (SEQ ID NO:4). The valine residue in bold is the putative start site for the protein. Conserved residues are shown in capital letters for homology in all three sequences and in lower case letters for homology in 2 of the 3 sequences.     The amino acid sequences  contain<br>
approximately 50% sequence identity. FIGURE 7.  Hybrid plasmid construct containing a 564 bp BsaA\/Kpn\ fragment from the S. hygroscopicus aveC homolog gene inserted into the BsaAUKpn) site in the S. avermitilis aveC ORF.<br>
5. DETAILED DESCRIPTION OF THE INVENTION<br>
The   present   invention   relates   to   the   identification   and   characterization   of polynucleotide molecules having nucleotide sequences that encode the AveC gene product from Streptomyces avermitilis, the construction of novel strains of S. avermitilis that can be used to screen mutated AveC gene products for their effect on avermectin production, and the discovery that certain mutated AveC gene products can reduce the ratio of B2:B1 avermectins produced by S. avermitilis.   By way of example, the invention is described in the sections<br>
below for a polynucleotide molecule having either a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1), and for polynucleotides molecules having mutated nucleotide sequences derived therefrom and degenerate variants thereof. However, the principles set forth in the present invention can be analogously applied to other polynucleotide molecules, including aveC homolog genes from other Streptomyces species including, e.g., S. hygroscopicus and S. griseochromogenes, among others.<br>
5.1.       Polynucleotide Molecules Encoding The S. avermitilis AveC Gene Product<br>
The present invention provides an isolated polynucleotide molecule comprising the<br>
complete aveC ORF of S. avermitilis or a substantial portion thereof, which  isolated<br><br>
polynucleotide molecule lacks the next complete ORF that is located downstream from the aveC ORF in situ in the S. avermitilis chromosome.<br>
The isolated polynucleotide molecule of the present invention preferably comprises a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that is the same as the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or substantial portion thereof. As used herein, a "substantial portion" of an isolated polynucleotide molecule comprising a nucleotide sequence encoding the S. avermitilis AveC gene product means an isolated polynucleotide molecule comprising at least about 70% of the complete aveC ORF sequence shown in FIGURE 1 (SEQ ID NO:1), and that encodes a functionally equivalent AveC gene product. In this regard, a "functionally equivalent" AveC gene product is defined as a gene product that, when expressed in S. avermitilis strain ATCC 53692 in which the native aveC allele has been inactivated, results in the production of substantially the same ratio and amount of avermectins as produced by S. avermitilis strain ATCC 53692 which instead expresses only      the wild-type, functional aveC allele native to S. avermitilis strain ATCC 53692.<br>
In addition to the nucleotide sequence of the aveC ORF, the isolated polynucleotide molecule of the present invention can further comprise nucleotide sequences that naturally flank the aveC gene in situ in S. avermitilis, such as those flanking nucleotide sequences shown in FIGURE 1 (SEQ ID NO:1).<br>
The present invention further provides an isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:1 or a degenerate variant thereof.<br>
As used herein, the terms "polynucleotide molecule," "polynucleotide sequence," "coding sequence," "open-reading frame," and "ORP are intended to refer to both DNA and RNA molecules, which can either be single-stranded or double-stranded, and that can be transcribed and translated (DNA), or translated (RNA), into an AveC gene product or, as described below, into an AveC homolog gene product, or into a polypeptide that is homologous to an AveC gene product or AveC homolog gene product in an appropriate host cell expression system when placed under the control of appropriate regulatory elements. A coding sequence can include but is not limited to prokaryotic sequences, cDNA sequences, genomic DNA sequences, and chemically synthesized DNA and RNA sequences.<br>
The nucleotide sequence shown in FIGURE 1 (SEQ ID NO:1) comprises four different GTG codons at bp positions 42,174,177 and 180. As described in Section 9 below, multiple deletions of the 5' region of the aveC ORF (FIGURE 1; SEQ ID NO:1) were constructed to help define which of these codons could function in the aveC ORF as start sites for protein<br><br>
expression. Deletion of the first GTG site at bp 42 did not eliminate AveC activity. Additional deletion of all of the GTG codons at bp positions 174, 177 and 180 together eliminated AveC activity, indicating that this region is necessary for protein expression. The present invention thus encompasses variable length aveC ORFs.<br>
The present invention further provides a polynucleotide molecule having a nucleotide sequence that is homologous to the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or to the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof.   The term "homologous" when used to refer to a polynucleotide molecule that is homologous to an S. avermitilis AveC gene product-encoding sequence means a polynucleotide molecule having a nucleotide sequence: (a) that encodes the same AveC gene product as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that encodes the same AveC gene product as the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1), but that includes one or more silent changes to the nucleotide sequence according to the degeneracy of the genetic code (i.e., a degenerate variant); or (b) that hybridizes to the complement of a polynucleotide molecule having a nucleotide sequence that encodes the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or that encodes the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2) under moderately stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7%  sodium dodecyl sulfate (SDS),  1   mM  EDTA at 65°C,  and washing  in 0.2xSSC/0.1% SDS at 42°C (see Ausubel et al. (eds.), 1989, Current Protocols in Molecular Biology, Vol. I, Green Publishing Associates, Inc., and John Wiley &amp; Sons, Inc., New York, at p. 2.10.3), and encodes a functionally equivalent AveC gene product as defined above.  In a preferred   embodiment,   the   homologous   polynucleotide   molecule   hybridizes   to   the complement of the AveC gene product-encoding nucleotide sequence of plasmid pSE186 (ATCC 209604) or to the complement of the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof under highly stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04. 7% SDS, 1 mM EDTA at 65°C, and washing in 0.1xSSC/0.1% SDS at 68°C (Ausubel et al., 1989, above), and encodes a functionally equivalent AveC gene product as defined above.<br>
The activity of an AveC gene product and potential functional equivalents thereof can be determined through HPLC analysis of fermentation products, as described in the examples below. Polynucleotide molecules having nucleotide sequences that encode functional equivalents of the S. avermitilis AveC gene product include naturally occurring aveC genes<br><br>
present in other strains of S. avermitilis, aveC homolog genes present in other species of Streptomyces, and mutated aveC alleles, whether naturally occurring or engineered.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide having an amino acid sequence that is homologous to the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or substantial portion thereof. As used herein, a "substantial portion" of the amino acid sequence of FIGURE 1 (SEQ ID NO:2) means a polypeptide comprising at least about 70% of the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2), and that constitutes a functionally equivalent AveC gene product, as defined above.<br>
As used herein to refer to amino acid sequences that are homologous to the amino acid sequence of an AveC gene product from S. avermitilis, the term "homologous' refers to a polypeptide which otherwise has the amino acid sequence of FIGURE 1 (SEQ ID NO:2), but in which one or more amino acid residues has been conservatively substituted with a different amino acid residue, wherein said amino acid sequence has at least about 70%, more preferably at least about 80%, and most preferably at least about 90% amino acid sequence identity to the polypeptide encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the amino acid sequence of Figure 1  (SEQ ID NO:2) as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm  (GENBANK,  NCBI),  and where such  conservative substitution results  in  a functionally  equivalent  gene  product,   as   defined   above.     Conservative  amino   acid substitutions are well known in the art.   Rules for making such substitutions include those described by Dayhof, M.D., 1978, Nat. Biomed. Res. Found.. Washington, D.C., Vol. 5, Sup. 3, among others.   More specifically, conservative amino acid substitutions are those that generally take place within a family of amino acids that are related in the acidity or polarity. Genetically encoded amino acids are generally divided into four groups:    (1) acidic = aspartate, glutamate; (2) basic = lysine, arginine, histidine; (3) non-polar = alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar = glycine,   asparagine,   glutamine,   cysteine,   serine,  threonine,  tyrosine.     Phenylalanine,      tryptophan and tyrosine are also jointly classified as aromatic amino acids.   One or more replacements within any particular group, e.g., of a leucine with an isoleucine or valine, or of an aspartate with a glutamate, or of a threonine with a serine, or of any other amino acid . residue with a structurally related amino acid residue, e.g., an amino acid residue with similar<br><br>
acidity or polarity, or with similarity in some combination thereof, will generally have an insignificant effect on the function of the polypeptide.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence encoding an AveC homolog gene product. As used herein, an "AveC homolog gene product" is defined as a gene product having at least about 50% amino acid sequence identity to an AveC gene product of S. avermitilis comprising the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2), as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm (GENBANK, NCBI). In a non-limiting embodiment the AveC homolog gene product is from S. hygroscopicus,   (described  in   EP  application  0298423;  deposit  FERM   BP-1901)  and comprises the amino acid sequence of SEQ ID NO:4, or a substantial portion thereof.   A "substantial portion" of the amino acid sequence of SEQ ID NO:4 means a polypeptide comprising at least about 70% of the amino acid sequence of SEQ ID NO:4, and that constitutes a functionally equivalent AveC homolog gene product. A "functionally equivalenf AveC homolog gene product is defined as a gene product that, when expressed in S. hygroscopicus strain FERM BP-1901 in which the native aveC homolog allele has been inactivated, results in the production of substantially the same ratio and amount of milbemycins as produced by S. hygroscopicus strain FERM BP-1901 expressing instead only the wild-type, functional aveC homolog allele native to S. hygroscopicus strain FERM BP-1901.   In a non-limiting embodiment, the isolated polynucleotide molecule of the present invention that encodes the S. hygroscopicus AveC homolog gene product comprises the nucleotide sequence of SEQ ID NO:3 or a substantial portion thereof.   In this regard, a "substantial portion" of the isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:3 means an isolated polynucleotide molecule comprising at least about 70% of the nucleotide sequence of SEQ ID NO:3, and that encodes a functionally equivalent AveC homolog gene product, as defined immediately above.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus nucleotide sequence of SEQ ID NO:3. The term "homologous" when used to refer to a polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus AveC homolog gene product-encoding sequence of SEQ ID NO:3 means a polynucleotide molecule having a nucleotide sequence: (a) that encodes the same gene product as the nucleotide sequence of SEQ ID NO:3, but that includes one or more silent changes to the nucleotide sequence<br><br>
according to the degeneracy of the genetic code {i.e., a degenerate variant); or (b) that hybridizes to the complement of a polynucleotide molecule having a nucleotide sequence that encodes the amino acid sequence of SEQ ID NO:4, under moderately stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7% SDS, 1 mM EDTA at 65°C, and washing in 0.2xSSC/0.1% SDS at 42°C (see Ausubel et al. above), and encodes a functionally equivalent AveC homolog gene product as defined above. In a preferred embodiment, the homologous polynucleotide molecule hybridizes to the complement of the AveC homolog gene product-encoding nucleotide sequence of SEQ ID NO:3, under highly stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7% SDS, 1 mM EDTA at 65°C, and washing in 0.1xSSC/0.1% SDS at 68°C (Ausubel et al., 1989, above), and encodes a functionally equivalent AveC homolog gene product as defined above.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide that is homologous to the S. hygroscopicus AveC homolog gene product. As used herein to refer to polypeptides that are homologous to the AveC homolog gene product of SEQ ID NO:4 from S. hygroscopicus, the term "homologous" refers to a polypeptide which otherwise has the amino acid sequence of SEQ ID NO:4, but in which one or more amino acid residues has been conservatively substituted with a different amino acid residue as defined above, wherein said amino acid sequence has at least about 70%, more preferably at least about 80%, and most preferably at least about 90% amino acid sequence identity to the polypeptide of SEQ ID NO:4, as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm (GENBANK, NCBI), and where such conservative substitution results in a functionally equivalent AveC homolog gene product, as defined above.<br>
The present invention further provides oligonucleotides that hybridize to a polynucleotide molecule having the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3, or to a polynucleotide molecule having a nucleotide sequence which is the complement of the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3. Such oligonucleotides are at least about 10 nucleotides in length, and preferably from about 15 to about 30 nucleotides in length, and hybridize to one of the aforementioned polynucleotide molecules under highly stringent conditions, i.e., washing in 6xSSC/0.5% sodium pyrophosphate at ~37°C for ~14-base oligos, at ~48°C for -17-base oligos, at ~55°C for -20-base oligos, and at ~60°C for -23-base oligos. In a preferred embodiment, the oligonucleotides are complementary to a portion of one of the aforementioned polynucleotide molecules. These oligonucleotides are useful for a variety of purposes including to encode or<br><br>
act as antisense molecules useful in gene regulation, or as primers in amplification of aveC-or aveC homolog-encoding polynucleotide molecules.<br>
Additional aveC homolog genes can be identified in other species or strains of Sireptomyces using the polynucleotide molecules or oligonucleotides disclosed herein in conjunction with known techniques. For example, an oligonucleotide molecule comprising a portion of the S. avermitilis nucleotide sequence of FIGURE 1 (SEQ ID NO.I) or a portion of the S. hygroscopicus nucleotide sequence of SEQ ID NO:3 can be detectably labeled and used to screen a genomic library constructed from DNA derived from the organism of interest. The stringency of the hybridization conditions is selected based on the relationship of the reference organism, in this example S. avermitilis or S. hygroscopicus, to the organism of interest. Requirements for different stringency conditions are well known to those of skill in the art, and such conditions will vary predictably depending on the specific organisms from which the library and the labeled sequences are derived. Such oligonucleotides are preferably at feast about 15 nucleotides in length and include, e.g., those described in the examples below. Amplification of homolog genes can be carried out using these and other oligonucleotides by applying standard techniques such as the polymerase chain reaction (PCR), although other amplification techniques known in the art, e.g., the ligase chain reaction, can also be used.<br>
Clones identified as containing aveC homolog nucleotide sequences can be tested for &gt; their ability to encode a functional AveC homolog gene product. For this purpose, the clones can be subjected to sequence analysis in order to identify a suitable reading frame, as well as initiation and termination signals. Alternatively or additionally, the cloned DNA sequence can be inserted into an appropriate expression vector, i.e., a vector that contains the necessary elements for the transcription and translation of the inserted protein-coding sequence. Any of a variety of host/vector systems can be used as described below, including but not limited to bacterial systems such as plasmid, bacteriophage, or cosmid expression vectors. Appropriate host cells transformed with such vectors comprising potential aveC homolog coding sequences can then be analyzed for AveC-type activity using methods such as HPLC analysis of fermentation products, as described, e.g., in Section 7, below.<br>
Production and manipulation of the polynucleotide molecules disclosed herein are within the skill in the art and can be carried out according to recombinant techniques described, e.g., in Maniatis, etal., 1989, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press. Cold Spring Harbor, NY; Ausubel, et al., 1989, Current Protocols In Molecular Biology, Greene Publishing Associates &amp; Wiley Interscience, NY; Sambrook, ef al.,<br><br>
1989, Molecular Cloning: A Laboratory Manual, 2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Innis et al. (eds), 1995, PCR Strategies, Academic Press, Inc., San Diego; and Erlich (ed), 1992, PCR Technology, Oxford University Press, New York, all of which are incorporated herein by reference.   Polynucleotide clones encoding AveC gene products or AveC homolog gene products can be identified using any method known in the art, including but not limited to the methods set forth in Section 7, below.   Genomic DNA libraries can be screened for aveC and aveC homolog coding sequences using techniques such as the methods set forth in Benton and Davis, 1977, Science 196:180, for bacteriophage libraries, and in Grunstein and Hogness, 1975, Proc. Natl. Acad. Sci. USA, 72:3961-3965, for<br>
plasmid libraries.   Polynucleotide molecules having nucleotide sequences known to include the aveC ORF, as present, e.g., in plasmid pSE186 (ATCC 209604), or in plasmid pSE119 (described in Section 7, below), can be used as probes in these screening experiments. Alternatively,. oligonucleotide probes can be synthesized that correspond to nucleotide sequences deduced from partial or complete amino acid sequences of the purified AveC homolog gene product.<br>
5.2. Recombinant Systems 5.2.1. Cloning And Expression Vectors The present invention further provides recombinant cloning vectors and expression vectors which are useful in cloning or expressing polynucleotide molecules of the present invention comprising, e.g., the aveC ORF of S. avermitilis or any aveC homolog ORFs. In a non-limiting embodiment, the present invention provides plasmid pSE186 (ATCC 209604), which comprises the complete ORF of the aveC gene of S. avermitilis.<br>
All of the following description regarding the aveC ORF from S. avermitilis, or a polynucleotide molecule comprising the aveC ORF from S. avermitilis or portion thereof, or an S. avermitilis AveC gene product, also refers to aveC homologs and AveC homolog gene products, unless indicated explicitly or by context.<br>
A variety of different vectors have been developed for specific use in Streptomyces, including phage, high copy number plasmids, low copy number plasmids, and E.coli-Streptomyces shuttle vectors, among others, and any of these can be used to practice the present invention. A number of drug resistance genes have also been cloned from Streptomyces, and several of these genes have been incorporated into vectors as selectable markers. Examples of current vectors for use in Streptomyces are presented, among other places, in Hutchinson, 1980, Applied Biochem. Biotech. 16:169-190.<br><br>
Recombinant vectors of the present invention, particularly expression vectors, are preferably constructed so that the coding sequence for the polynucleotide molecule of the invention is in operative association with one or more regulatory elements necessary for transcription and translation of the coding sequence to produce a polypeptide. As used herein, the term "regulatory element" includes but is not limited to nucleotide sequences that encode inducible and non-inducible promoters, enhancers, operators and other elements known in the art that serve to drive and/or regulate expression of polynucleotide coding sequences. Also, as used herein, the coding sequence is in "operative association" with one or more regulatory elements where the regulatory elements effectively regulate and allow for the transcription of the coding sequence or the translation of its mRNA, or both.<br>
Typical plasmid vectors that can be engineered to contain a polynucleotide molecule of the present invention include pCR-Blunt, pCR2.1 (Invitrogen), pGEM32f (Promega), and the shuttle vector pWHM3 (Vara ef at., 1989, J. Bact. 171:5872-5881), among many others.<br>
Methods are well-known in the art for constructing recombinant vectors containing particular coding sequences in operative association with appropriate regulatory elements, and these can be used to practice the present invention. These methods include in vitro recombinant techniques, synthetic techniques, and in vivo genetic recombination. See, e.g., the techniques described in Maniatis et a/., 1989, above; Ausubel et at., 1989, above; Sambrook etal., 1989, above; Innis et al., 1995, above; and Erlich, 1992, above.<br>
The regulatory elements of these vectors can vary in their strength and specificities. Depending on the host/vector system utilized, any of a number of suitable transcription and translation elements can be used. Non-limiting examples of transcriptional regulatory regions or promoters for bacteria include the β-gal promoter, the T7 promoter, the TAC promoter, A. left and right promoters, tap and lac promoters, trp-lac fusion promoters and, more specifically for Streptomyces, the promoters ermE, meIC, and tipA, etc. In a specific embodiment described in Section 11 below, an expression vector was generated that contained the aveC ORF cloned adjacent to the strong constitutive ermE promoter from Saccharopolyspora erythraea. The vector was transformed into S. avermitilis, and subsequent HPLC analysis of fermentation products indicated an increased titer of avermectins produced compared to production by the same strain but which instead expresses the wild-type aveC allele.<br>
Fusion protein expression vectors can be used to express an AveC gene product-fusion protein. The purified fusion protein can be used to raise antisera against the AveC gene product, to study the biochemical properties of the AveC gene product, to engineer AveC fusion proteins with different biochemical activities, or to aid in the identification or<br><br>
purification of the expressed AveC gene product. Possible fusion protein expression vectors include but are not limited to vectors incorporating sequences that encode p-galactosidase and trpE fusions, maltose-binding protein fusions, glutathione-S-transferase fusions and polyhistidine fusions (carrier regions). In an alternative embodiment, an AveC gene product or a portion thereof can be fused to an AveC homolog gene product, or portion thereof, derived from another species or strain of Streptomyces, such as, e.g., S. hygroscopicus or S. griseochromogenes. In a particular embodiment described in Section 12, below, and depicted in FIGURE 7, a chimeric plasmid was constructed that contains a 564 bp region of the S. hygroscopicus aveC homolog ORF replacing a homologous 564 bp region of the S. avermitilis aveC ORF. Such hybrid vectors can be transformed into S. avermitilis cells and tested to determine their effect, e.g., on the ratio of class 2:1 avermectin produced.<br>
AveC fusion proteins can be engineered to comprise a region useful for purification. For example, AveC-maltose-binding protein fusions can be purified using amylose resin; AveC-glutathione-S-transferase fusion proteins can be purified using glutathione-agarose beads; and AveC-polyhistidine fusions can be purified using divalent nickel resin. Alternatively, antibodies against a carrier protein or peptide can be used for affinity chromatography purification of the fusion protein. For example, a nucleotide sequence coding for the target epitope of a monoclonal antibody can be engineered into the expression vector in operative association with the regulatory elements and situated so that the expressed epitope is fused to the AveC polypeptide. For example, a nucleotide sequence coding for the FLAG™ epitope tag (International Biotechnologies Inc.), which is a hydrophilic marker peptide, can be inserted by standard techniques into the expression vector at a point corresponding, e.g., to the carboxyl terminus of the AveC polypeptide. The expressed AveC polypeptide-FLAG™ epitope fusion product can then be detected and affinity-purified using commercially available anti-FLAG™ antibodies.<br>
The expression vector encoding the AveC fusion protein can also be engineered to contain polylinker sequences that encode specific protease cleavage sites so that the expressed AveC polypeptide can be released from the carrier region or fusion partner by treatment with a specific protease. For example, the fusion protein vector can include DNA sequences encoding thrombin or factor Xa cleavage sites, among others.<br>
A signal sequence upstream from, and in reading frame with, the aveC ORF can be engineered into the expression vector by known methods to direct the trafficking and secretion of the expressed gene product.   Non-limiting examples of signal sequences include those<br><br>
from α-factor, immunoglobulins, outer membrane proteins, penicillinase, and T-cell receptors, among others.<br>
To aid in the selection of host cells transformed or transfected with cloning or expression vectors of the present invention, the vector can be engineered to further comprise a coding sequence for a reporter gene product or other selectable marker. Such a coding sequence is preferably in operative association with the regulatory element coding sequences, as described above. Reporter genes that are useful in the invention are well-known in the art and include those encoding green fluorescent protein, luciferase, xylE, and tyrosinase, among others. Nucleotide sequences encoding selectable markers are well known in the art, and include those that encode gene products conferring resistance to antibiotics or anti-metabolites, or that supply an auxotrophic requirement. Examples of such sequences include those that encode resistance to erythromycin, thiostrepton or kanamycin, among many others.<br>
5.2.2. Transformation Of Host Cells The present invention further provides transformed host cells comprising a polynucleotide molecule or recombinant vector of the invention, and novel strains or cell lines derived therefrom. Host cells useful in the practice of the invention are preferably Streptomyces cells, although other prokaryotic cells or eukaryotic cells can also be used. Such transformed host cells typically include but are not limited to microorganisms, such as bacteria transformed with recombinant bacteriophage DNA, plasmid DNA or cosmid DNA vectors, or yeast transformed with recombinant vectors, among others.<br>
The polynucleotide molecules of the present invention are intended to function in Streptomyces cells, but can also be transformed into other bacterial or eukaryotic cells, e.g., for cloning or expression purposes. A strain of E. coli can typically be used, such as, e.g., the DH5a strain, available from the American Type Culture Collection (ATCC), Rockville, MD, USA (Accession No. 31343), and from commercial sources (Stratagene). Preferred eukaryotic host cells include yeast cells, although mammalian cells or insect cells can also be utilized effectively.<br>
The recombinant expression vector of the invention is preferably transformed or transfected into one or more host cells of a substantially homogeneous culture of cells. The expression vector is generally introduced into host cells in accordance with known techniques, such as, e.g., by protoplast transformation, calcium phosphate precipitation, calcium chloride treatment, microinjection, electroporation, transfection by contact with a recombined virus, liposome-mediated transfection,  DEAE-dextran transfection, transduction, conjugation, or<br><br>
microprojectile bombardment.   Selection of transformants can be conducted by standard procedures, such as by selecting for cells expressing a selectable marker, e.g., antibiotic resistance, associated with the recombinant vector, as described above.<br>
Once the expression vector is introduced into the host cell, the integration and<br>
maintenance of the aveC coding sequence either in the host cell chromosome or episomally can be confirmed by standard techniques, e.g., by Southern hybridization analysis, restriction enzyme analysis, PCR analysis, including reverse transcriptase PCR (rt-PCR), or by immunological assay to detect the expected gene product.   Host cells containing and/or expressing the recombinant aveC coding sequence can be identified by any of at least four general approaches which are well-known in the art, including: (j) DNA-DNA, DNA-RNA, or RNA-antisense RNA hybridization; (ii) detecting the presence of "marker" gene functions; (iii) assessing the level of transcription as measured by the expression of aveC-specific mRNA<br>
transcripts in the host cell; and (iv) detecting the presence of mature polypeptide product as measured, e.g., by immunoassay or by the presence of AveC biological activity (e.g., the production of specific ratios and amounts of avermectins indicative of AveC activity in, e.g., S. avermitilis host cells).<br>
5.2.3.   Expression And Characterization<br>
Of A Recombinant AveC Gene Product<br>
Once the aveC coding sequence has been stably introduced into an appropriate host cell, the transformed host cell is clonally propagated, and the resulting cells can be grown under conditions conducive to the maximum production of the AveC gene product. Such conditions typically include growing cells to high density. Where the expression vector comprises an inducible promoter, appropriate induction conditions such as, e.g., temperature shift, exhaustion of nutrients, addition of gratuitous inducers (e.g., analogs of carbohydrates, such as isopropyl-p-D-thiogalactopyranoside (IPTG)), accumulation of excess metabolic by products, or the like, are employed as needed to induce expression.<br>
Where the expressed AveC gene product is retained inside the host cells, the cells are harvested and lysed, and the product isolated and purified from the lysate under extraction conditions known in the art to minimize protein degradation such as, e.g., at4°C, or in the presence of protease inhibitors, or both. Where the expressed AveC gene product is secreted from the host cells, the exhausted nutrient medium can simply be collected and the product isolated therefrom.<br>
The expressed AveC gene product can be isolated or substantially purified from cell lysates or culture medium, as appropriate, using standard methods, including but not limited<br><br>
to any combination of the following methods: ammonium sulfate precipitation, size fractionation, ion exchange chromatography, HPLC, density centrifugation, and affinity chromatography. Where the expressed AveC gene product exhibits biological activity, increasing purity of the preparation can be monitored at each step of the purification procedure by use of an appropriate assay. Whether or not the expressed AveC gene product exhibits biological activity, it can be detected as based, e.g., on size, or reactivity with an antibody otherwise specific for AveC, or by the presence of a fusion tag. As used herein, an AveC gene product is "substantially purified" where the product constitutes more than about 20 wt% of the protein in a particular preparation. Also, as used herein, an AveC gene product is "isolated" where the product constitutes at least about 80 wt% of the protein in a particular preparation.<br>
The present invention thus provides a recombinantly-expressed isolated or substantially purified S. avermitilis AveC gene product comprising the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or a substantial portion thereof, and homologs thereof.<br>
The present invention further provides a recombinantly-expressed isolated or substantially purified S. hygroscopicus AveC homolog gene product comprising the amino acid sequence of SEQ ID NO:4 or a substantial portion thereof, and homologs thereof.<br>
The present invention further provides a method for producing an AveC gene product, comprising culturing a host cell transformed with a recombinant expression vector, said vector comprising a polynucleotide molecule having a nucleotide sequence encoding the AveC gene product, which polynucleotide molecule is in operative association with one or more regulatory elements that control expression of the polynucleotide molecule in the host cell, under conditions conducive to the production of the recombinant AveC gene product, and recovering the AveC gene product from the cell culture.<br>
The recombinantly expressed S. avermitilis AveC gene product is useful for a variety of purposes, including for screening compounds that alter AveC gene product function and thereby modulate avermectin biosynthesis, and for raising antibodies directed against the AveC gene product.<br>
Once an AveC gene product of sufficient purity has been obtained, it can be characterized by standard methods, including by SDS-PAGE, size exclusion chromatography, amino acid sequence analysis, biological activity in producing appropriate products in the avermectin biosynthetic pathway, ete.   For example, the amino acid sequence of the AveC<br><br>
gene product can be determined using standard.peptide sequencing techniques. The AveC gene product can be further characterized using hydrophilicity analysis (see, e.g., Hopp and Woods, 1981, Proc. Natl. Acad. Sci. USA  78:3824), or analogous software algorithms, to identify hydrophobic and hydrophilic regions of the AveC gene product.  Structural analysis can be carried out to identify regions of the AveC gene product that assume specific secondary structures.  Biophysical methods such as X-ray crystallography (Engstrom, 1974, Biochem. Exp. Biol. 11: 7-13), computer modelling (Fletterick and Zoller (eds), 1986, in: Current Communications in Molecular Biology, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY), and nuclear magnetic resonance (NMR) can be used to map and study sites of interaction between the AveC gene product and its substrate.   Information obtained from these studies can be used to select new sites for mutation in the aveC ORF to help develop new strains of S. avermitUis having more desirable avermectin production characteristics . 5.3. Construction And Use Of AveC Mutants The present invention provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the S. avermitilis aveC allele or a degenerate variant thereof, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprises one or more mutations, so that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence or the degenerate variant thereof produce a different ratio or amount of avermectins than are produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele.<br>
According to the present invention, such polynucleotide molecules can be used to produce novel strains of S. avermitilis that exhibit a detectable change in avermectin production compared to the same strain which instead expresses only the wild-type aveC allele. In a preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce avermectins in a reduced class 2:1 ratio compared to the same strain which instead expresses only the wild-type aveC allele. In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce increased levels of avermectins compared to the same strain which instead expresses only a single wild-type aveC allele. In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis in which the aveC gene has been inactivated.<br><br>
Mutations to the aveC allele or coding sequence include any mutations that introduce<br>
one or more amino acid deletions, additions, or substitutions into the AveC gene product, or that result in truncation of the AveC gene product, or any combination thereof, and that produce the desired result. Such mutated aveC allele sequences are also intended to include any degenerate variants thereof. For example, the present invention provides polynucleotide molecules comprising the nucleotide sequence of the aveC allele or a degenerate variant thereof, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as present in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprise one or more mutations that encode the substitution of an amino acid residue with a different amino acid residue at selected positions in the AveC gene product.   In several non-limiting embodiments, several of which are exemplified below, such substitutions can be carried out at any amino acid positions of the AveC gene product which correspond to amino acid positions<br>
38, 48, 55, 89, 99, 111, 136, 138, 139, 154, 179, 228. 230, 238, 266, 275. 289 or 298 of SEQ ID NO:2. or some combination thereof.<br>
Mutations to the aveC coding sequence are carried out by any of a variety of known methods, including by use of error-prone PCR, or by cassette mutagenesis. For example, oligonucleotide-directed mutagenesis can be employed to alter the sequence of the aveC allele or ORF in a defined way such as, e.g., to introduce one or more restriction sites, or a termination codon, into specific regions within the aveC allele or ORF. Methods such as those described in U.S. Patent 5,605,793, U.S. Patent 5,830,721 and U.S. Patent 5,837,458, which involve random fragmentation, repeated cycles of mutagenesis, and nucleotide shuffling, can also be used to generate large libraries of polynucleotides having nucleotide sequences encoding aveC mutations.<br>
Targeted mutations can be useful, particularly where they serve to alter one or more conserved amino acid residues in the AveC gene product. For example, a comparison of deduced amino acid sequences of AveC gene products and AveC homolog gene products from S. avermitilis (SEQ ID NO:2), S. griseochromogenes (SEQ ID NO:5), and S. hygroscopicus (SEQ ID NO:4), as presented in FIGURE 6, indicates sites of significant conservation of amino acid residues between these species. Targeted mutagenesis vat leads to a change in one or more of these conserved amino acid residues can be particularly effective in producing novel mutant strains that exhibit desirable alterations in avermectin production.<br><br>
Random mutagenesis can also be useful, and can be carried out by exposing cells of S. avermitilis to ultraviolet radiation or x-rays, or to chemical mutagens such as N-methyl-N'-nitrosoguanidine, ethyl methane sulfonate, nitrous acid or nitrogen mustards. See, e.g., Ausubel, 1989, above, for a review of mutagenesis techniques.<br>
Once mutated  polynucleotide molecules are generated,  they are screened  to determine whether they can modulate avermectin biosynthesis in S. avermitilis. In a preferred embodiment, a polynucleotide molecule having a mutated nucleotide sequence is tested by complementing a strain of S. avermitilis in which the aveC gene has been inactivated to give an aveC negative {aveC) background.  In a non-limiting method, the mutated polynucleotide molecule is spliced into an expression plasmid in operative association with one or more regulatory elements, which plasmid also preferably comprises one or more drug resistance genes to allow for selection of transformed cells. This vector is then transformed into aveC host cells using known techniques, and transformed cells are selected and cultured in appropriate fermentation media under conditions that permit or induce avermectin production. Fermentation products are then analyzed by HPLC to determine the ability of the mutated polynucleotide molecule to complement the host cell.    Several vectors bearing mutated polynucleotide molecules capable of reducing the B2:B1  ratio of avermectins, including pSE188, pSE199, pSE231, pSE239, and pSE290 through pSE297, are exemplified in Section 8.3, below.<br>
The present invention provides methods for identifying mutations of the S. avermitilis aveC ORF capable of altering the ratio and/or amount of avermectins produced. In a preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF capable of altering the class 2:1 ratio of avermectins produced, comprising: (a) determining the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product has been introduced and is being expressed; (b) determining the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a wild-type aveC allele or an aveC allele having the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) to the class 2:1 ratio of avermectins produced by the S. avermMis cells of step (b); such thai if the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) is different from the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b), then a<br><br>
mutation of the aveC ORF capable of altering the class 2:1 ratio of avermectins has been identified. In a preferred embodiment, the class 2:1 ratio of avermectins is reduced by the mutation.<br>
In a further preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF or genetic constructs comprising the aveC ORF capable of altering the amount of avermectins produced, comprising:   (a) determining the amount of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product or comprising a genetic construct comprising a nucleotide sequence encoding an AveC gene product has been introduced and is being expressed; (b) determining the amount of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a wild-type aveC allele or a nucleotide sequence that is homologous thereto; and (c) comparing the amount of avermectins produced by the S. avermitilis cells of step (a) to the amount of avermectins produced by the S. avermitilis cells of step (b); such that if the amount of avermectins produced by the S. avermitilis cells of step (a) is different from the amount of avermectins produced by the S. avermitilis cells of step (b), then a mutation of the aveC ORF or a genetic construct capable of altering the amount of avermectins has been identified. In a preferred embodiment, the amount of avermectins produced is increased by the mutation.<br>
Any of the aforementioned methods for identifying mutations are be carried out using fermentation culture media preferably supplemented with cyclohexane carboxylic acid, although other appropriate fatty acid precursors, such as any one of the fatty acid precursors listed in TABLE 1, can also used.<br>
Once a mutated polynucleotide molecule that modulates avermectin production in a desirable direction has been identified, the location of the mutation in the nucleotide sequence can be determined. For example, a polynucleotide molecule having a nucleotide sequence encoding a mutated AveC gene product can be isolated by PCR and subjected to DNA sequence analysis using known methods. By comparing the DNA sequence of the mutated aveC allele to that of the wild-type aveC allele, the mutation(s) responsible for the alteration in avermectin production can be determined. In specific though non-limiting embodiments of the present invention, S. avermitilis AveC gene products comprising either single amino acid substitutions at any of residues 55 (S55F), 138 (S138T), 139 (A139T), or 230 (G230D), or double substitutions at positions 138 (S138T) and 139 (A139T or A139F), yielded changes in AveC gene product function such that the ratio of class 2:1 avermectins produced was altered<br><br>
(see Section 8, below), wherein the recited amino acid positions correspond to those<br>
presented in FIGURE 1 (SEQ ID NO:2). In addition, the following seven combinations of<br>
mutations have each been shown to effectively reduce the class 2:1 ratio of avermectins: (1)<br>
D48E/A89T;	(2)	S138T/A139T/G179S;	(3)	Q38P/L136P/E238D;	(4)<br>
F99S/S138T/A139T/G179S; (5) A139T7 M228T; (6) G111V/P289L; (7) A139T/K154E/Q298H. As used herein, the aforementioned designations, such as A139T, indicate the original amino acid residue by single letter designation, which in this example is alanine (A), at the indicated position, which in this example is position 139 (referring to SEQ ID NO:2) of the polypeptide, followed by the amino acid residue which replaces the original amino acid residue, which in this example is threonine (T). Accordingly, polynucleotide molecules having nucleotide sequences that encode mutated S. avermitilis AveC gene products comprising amino acid substitutions or deletions at one or more of amino acid positions 38, 48, 55, 89, 99, 111,136, 138,139,154, 179, 228, 230, 238, 266, 275, 289 or 298 (see FIGURE 1), or any combination thereof, are encompassed by the present invention.<br>
In a preferred embodiment, such mutations encode amino acid substitutions selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(b)	amino acid residue D at position 48 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a<br>
. conservative substitution for P;<br>
(g)	amino acid residue S at position 138 replaced by T or by an amino acid that is a<br>
conservative substitution for T;<br>
(h) amino acid residue A at position 139 replaced by T or F, or by an amino acid that<br>
is a conservative substitution for T or F; (i)   amino acid residue K at position 154 replaced by E or by an amino acid that is a<br>
conservative substitution for E;<br><br>
(j)   amino acid residue G at position 179 replaced by S or by an amino acid that is a<br>
conservative substitution for S; (k) amino acid residue M at position 228 replaced by T or by an amino acid that<br>
is a conservative substitution for T; (I)   amino acid residue E at position 238 replaced by 0 or by an amino acid that is a<br>
conservative substitution for D; (m) amino acid, residue P at position 289 replaced by L or by an amino acid that is a<br>
conservative substitution for L; and (n) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H; wherein conservative amino acid substitutions are as defined above in Section 5.1.<br>
In a further preferred embodiment, such mutations encode a combination of amino acid substitutions, wherein the combination of amino acid residues substituted is selected from the group consisting of: .<br>
(a)	amino acid residues S138 and A139;<br>
(b)	amino acid residues D48 and A89;<br>
(c)	amino acid residues S138, A139 and G179;<br>
(d)	amino acid residues Q38, L136 and E238;<br>
(e)	amino acid residues F99, S138, A139 and G179;<br>
(f)	amino acid residues A139 and M228;<br>
(g)	amino acid residues G111 and P289; and (h) amino acid residues A139, K154 and Q298.<br>
Jn a further preferred embodiment, specific combinations of mutations in the aveC allele useful in effectively reducing the class 2:1 ratio of avermectins according to the present invention are selected from one or more of the group consisting of:<br>
(a)	S138T/A139T<br>
(b)	S138T7A139F<br>
(c)	D48E/A89T;<br>
(d)	S138T7A139T/G179S;<br>
(e)	Q38P/L136P/E238D;<br>
(f)	F99S/S138T/A139T/G179S;<br>
(g)	A139T/M228T;<br>
(h) G111V/P289L;and (i)   A139T/K154E/Q298H.<br><br>
The present invention further provides compositions for making novel strains of S. avermitilis, the cells of which contain a mutated aveC allele that results in the alteration of avermectin production. For example, the present invention provides recombinant vectors that can be used to target any of the polynucleotide molecules comprising mutated nucleotide sequences of the present invention to the site of the aveC gene of the S. avermitilis chromosome to either insert into or replace the aveC ORF or a portion thereof by homologous recombination.   According to the present invention, however, a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention provided herewith can also function to modulate avermectin biosynthesis when inserted into the S. avermitilis chromosome at a site other than at the aveC gene, or when maintained episomally in S. avermitilis  cells.     Thus,   the   present   invention   also  provides  vectors   comprising   a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention, which vectors can be used to insert the polynucleotide molecule at a site in the S. avermitilis chromosome other than at the aveC gene, or to be maintained episomally.<br>
In a preferred embodiment, the present invention provides gene replacement vectors that can be used to insert a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis, thereby generating novel strains of S. avermitilis, the cells of which produce avermectins in an altered class 2:1 ratio compared to cells of the same strain which instead express only the wild-type aveC allele. In a preferred embodiment, the class 2:1 ratio of avermectins produced by the cells is reduced. Such gene replacement vectors can be constructed using mutated polynucleotide molecules present in expression vectors provided herewith, such as, e.g., pSE188, pSE199, and pSE231, which expression vectors are exemplified in Section 8 below.<br>
In a further preferred embodiment, the present invention provides vectors that can be used to insert a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis to generate novel strains of cells that produce altered amounts of avermectins compared to cells of the same strain which instead express only the wild-type aveC allele. In a preferred embodiment, the amount of avermectins produced by the cells is increased. In a specific though non-limiting embodiment, such a vector further comprises a strong promoter as known in the art, such as, e.g., the strong constitutive ermE promoter from Saccharopolyspora erythraea, that is situated upstream from, and in operative association with, the aveC allele. Such a vector can be plasmid pSE189, described in Example 11 below, or can be constructed using the mutated aveC allele of plasmid pSE189.<br><br>
In a further preferred embodiment, the present invention provides gene replacement vectors that are useful to inactivate the aveC gene in a wild-type strain of S. avermitilis. In a non-limiting embodiment, such gene replacement vectors can be constructed using the mutated polynucleotide molecule present in plasmid pSE180 (ATCC 209605), which is exemplified in Section 8.1, below (FIGURE 3). The present invention further provides gene replacement vectors that comprise a polynucleotide molecule comprising or consisting of nucleotide sequences that naturally flank the aveC gene in situ in the S. avermitilis chromosome, including, e.g., those flanking nucleotide sequences shown in FIGURE 1 (SEQ ID NO:1), which vectors can be used to delete the S. avermitilis aveC ORF.<br>
The present invention further provides methods for making novel strains of S.<br>
avermitilis comprising cells that express a mutated aveC allele and that produce an altered ratio and/or amount of avermectins compared to cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele.   In a preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce an altered class 2:1 ratio of avermectins compared to cells of the same strain of S. avermitilis that instead express only a wild-type aveC allele, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele thereof compared to cells of the same strain that instead express only a wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered class 2:1  ratio compared to the class 2:1 ratio produced by cells of the strain that instead express only the wild-type aveC allele.   In a more preferred embodiment, the present invention provides a method for making a novel strain of S. avermitilis, comprising transforming cells of a strain of S. avermitilis with a vector capable of introducing a mutation into the aveC allele of such cells, wherein the mutation to the aveC allele results in the substitution in the encoded AveC gene product of a different amino acid residue at one or more amino acid positions corresponding to amino acid residues 38, 48, 55, 89, 99, 111, 136, 138, 139, 154, 179, 228. 230, 238, 266, 275, 289 or 298 of SEQ ID NO:2, such that cells of the S. avermitilis strain in which the aveC allele has been so mutated produce a class 2:1 ratio of avermectins that is different from the ratio produced by cells of the same S. avermitilis strain that instead express only the wild-type aveC allele. In a preferred embodiment, the altered class 2:1 ratio of avermectins is reduced. As used herein, where an amino acid residue encoded by an aveC allele in the S. avermitilis chromosome, or in a vector or isolated polynucleotide molecule of the present<br><br>
invention is referred to as "corresponding to" a particular amino acid residue of SEQ ID NO:2, or where an amino acid substitution is referred to as occurring at a particular position "corresponding to" that of a specific numbered amino acid residue of SEQ ID NO:2, this is intended to refer to the amino acid residue at the same relative location in the AveC gene product, which the skilled artisan can quickly determine by reference to the amino acid sequence presented herein as SEQ ID NO:2.<br>
The present invention further provides methods of making novel strains wherein<br>
specific mutations in the aveC allele encoding particular mutations are recited as base<br>
changes at specific nucleotide positions in the aveC allele "corresponding to" particular<br>
nucleotide positions as shown in SEQ ID NO:1. As above with regard to corresponding amino acid positions, where a nucleotide position in the aveC allele is referred to as "corresponding to" a particular nucleotide position in SEQ ID NO:1, this is intended to refer to the nucleotide at the same relative location in the aveC nucleotide sequence, which the skilled artisan can quickly determine by reference to the nucleotide sequence presented herein as SEQ ID NO:1. In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that produce altered amounts of avermectin, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele or a genetic construct comprising the aveC allele, the expression of which results in an alteration in the amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct as compared to cells of the same strain that instead express only a  single wild-type  aveC allele,  and  selecting transformed cells that produce avermectins in an altered amount compared to the amount of avermectins   produced by cells of the strain that instead express only the single wild-type aveC allele.    In a preferred embodiment, the amount of avermectins produced in the transformed cells is increased.<br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis, the cells of which comprise an inactivated aveC allele, comprising transforming cells of a strain of S. avermitilis that express any aveC allele with a vector that inactivates the aveC allele, and selecting transformed cells in which the aveC allele has been inactivated. In a preferred though non-limiting embodiment, cells of a strain of S. avermitilis are transformed with a gene replacement vector that carries an aveC allele that has been inactivated by mutation or by replacement of a portion of the aveC allele with a heterologous gene sequence, and transformed cells are selected in which the aveC allele otherwise native thereto has been replaced with the inactivated aveC allele.   Inactivation of<br><br>
the aveC allele can be determined by HPLC analysis of fermentation products, as described below. In a specific though non-limiting embodiment described in Section 8.1 below, the aveC allele is inactivated by insertion of the ermE gene from Saccharopolyspora erythraea into the aveC ORF.<br>
The present invention further provides novel strains of S. avermitilis comprising cells that have been transformed with any of the polynucleotide mofecufes or vectors of (he present invention.   In a preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele or degenerate variant thereof in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain produce avermectins in an altered class 2:1  ratio compared to the class 2:1  ratio of avermectins produced by cells of the same strain that instead express only the wild-type aveC allele.  In a preferred embodiment, the altered class 2:1 ratio produced by the novel cells is reduced.    Such novel strains are useful in the large-scale production of commercially desirable avermectins such as doramectin.   In a more preferred embodiment, the present invention provides cells of S. avermitilis comprising any of the aforementioned mutations or combinations of mutations in the aveC allele at nucleotide positions corresponding to those presented hereinabove or which otherwise encode any of the aforementioned amino acid substitutions in the AveC gene product. Although such mutations can be present in such cells on an extrachromosomal element such as a plasmid, it is preferred that such mutations are present in the aveC allele located on the S. avermitilis chromosome.    In a preferred embodiment, the present invention provides a strain of Streptomyces avermitilis comprising cells having a mutation in the aveC allele that encodes an AveC gene product having a substitution at one or more amino acid positions corresponding to amino acid residues 38,48, 55, 89, 99. 111, 136, 138, 139. 154, 179, 228, 230. 238, 266. 275, 289. or 298 of SEQ ID NO:2, wherein the cell produces a class 2:1 ratio of avermectins that is different from the ratio produced by a cell of the same S. avermitilis strain which express the wild-type aveC allele.<br>
It is a primary objective of the screening assays described herein to identify mutated alleles of the aveC gene the expression of which, in S. avermitilis cells, alters and, more particularly, reduces the ratio of class 2:1 avermectins produced. In a preferred embodiment, the ratio of B2:B1 avermectins produced by cells of a novel S. avermitilis strain of the present invention expressing a mutated aveC allele, or degenerate variant thereof, of the present invention is about 1.6:1 or less. In a more preferred embodiment, the ratio is about 1:1 or less. In a more preferred embodiment, the ratio is about 0.84:1 or less. In a more preferred embodiment, the ratio is about 0.80:1 or less.   In a more preferred embodiment, the ratio is<br><br>
about 0.75:1 or less. In a more preferred embodiment, the ratio is about 0:73:1 or less. In a more preferred embodiment, the ratio is about 0.68:1 or less. In an even more preferred embodiment, the ratio, is about 0.67:1 or less. In a more preferred embodiment, the ratio is about 0.57:1 or less. In an even more preferred embodiment, the ratio is about 0.53:1 or less. In an even more preferred embodiment, the ratio is about 0.42:1 or less. In an even more preferred embodiment, the ratio is about 0.40:1 or less.<br>
In a specific embodiment described below, novel cells of the present invention<br>
produce cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of less than 1.6:1.  In a different<br>
specific embodiment described below, novel cells of the present invention produce cyclohexyl<br>
B2:cyclohexyl B1 avermectins in a ratio of about 0.94:1.    In a further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
B2:cyclohexyl B1 avermectins in a ratio of about 0.88:1.    In a further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.84:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.75:1.   In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl Bi avermectins in a ratio of about 0.73:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1-avermectins in a ratio of about 0.68:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.67:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.57:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.53:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.42:1.   In yet a further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.40:1.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele or a degenerate variant thereof, or a genetic construct comprising an aveC allele or a degenerate variant thereof, in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain<br><br>
produce an altered amount of avermectins compared to cells of the same strain that instead express only the wild-type aveC allele. In a preferred embodiment, the novel strain produces an increased amount of avermectins. In a non-limiting embodiment, the genetic construct further comprises a strong promoter, such as the strong constitutive ermE promoter from Saccharopolyspora erythraea, upstream from and in operative association with the aveC ORF.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells in which the aveC gene has been inactivated. Such strains are useful both for the different spectrum of avermectins that they produce compared to the wild-type strain, and in complementation screening assays as described herein, to determine whether targeted or random mutagenesis of the aveC gene affects avermectin production. In a specific embodiment described below, S. avermitilis host cells were genetically engineered to contain an inactivated aveC gene. For example, strain SE180-11, described in the examples below, was generated using the gene replacement plasmid pSE180 (ATCC 209605) (FIGURE 3), which was constructed to inactivate the S. avermitilis aveC gene by insertion of the ermE resistance gene into the aveC coding region.<br>
The present invention further provides recombinantly expressed mutated S. avermitilis AveC gene products encoded by any of the aforementioned polynucleotide molecules of the invention, and methods of preparing the same.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele compared to cells of the same strain that instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the class 2:1 ratio of avermectins produced in the culture by cells expressing the mutated aveC allele is reduced. This process provides increased efficiency in the production of commercially valuable avermectins such as doramectin.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele or a genetic construct comprising an aveC allele that results in the production of an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct compared to cells of the same strain which do not<br><br>
express the mutated aveC allele or genetic construct but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the amount of avermectins produced in culture by cells expressing the mutated aveC allele, degenerate variant or genetic construct is increased.<br>
The present invention further provides a novel composition of avermectins produced by a strain of S. avermitilis expressing a mutated aveC allele or degenerate variant thereof that encodes a gene product that reduces the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or degenerate variant compared to cells of the same strain that instead express only the wild-type aveC allele, wherein the avermectins in the novel composition.are produced in a reduced class 2:1 ratio as compared to the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele. The novel avermectin composition can be present as produced in exhausted fermentation culture fluid, or can be harvested therefrom. The novel avermectin composition can be partially or substantially purified from the culture fluid by known biochemical techniques of purification, such as by ammonium sulfate precipitation, dialysis, size fractionation, ion exchange chromatography, HPLC, etc.<br>
5.4. Uses Of Avermectins<br>
Avermectins are highly active antiparasitic agents having particular utility as anthelmintics, ectoparasiticides, insecticides and acaricides. Avermectin compounds produced according to the methods of the present invention are useful for any of these purposes. For example, avermectin compounds produced according to the present invention are useful to treat various diseases or conditions in humans, particularly where those diseases or conditions are caused by parasitic infections, as known in the art. See, e.g., Ikeda and Omura, 1997, Chem. Rev. 97(7):2591-2609. More particularly, avermectin compounds produced according to the present invention are effective in treating a variety of diseases or conditions caused by endoparasites, such as parasitic nematodes, which can infect humans, domestic animals, swine, sheep, poultry, horses or cattle.<br>
More specifically, avermectin compounds produced according to the present invention, are effective against nematodes that infect humans, as well as those that infect various species of animals. Suchnematodes include gastrointestinal parasites such as Ancylostoma, Necator, Ascaris, Strongyloides, Trichinella, Capillaria, Trichuris, Enterobius, Dirofilaria, and<br><br>
parasites that are found in the blood or other tissues or organs, such as filarial worms and the extract intestinal states of Strongyloides and Trichinella.<br>
The avermectin compounds produced according to the present invention are also useful in treating ectoparasitic infections including, e.g., arthropod infestations of mammals and birds, caused by ticks, mites, lice, fleas, blowflies, biting insects, or migrating dipterous farvae that can affect cattle and horses, among others.<br>
The avermectin compounds produced according to the present invention are also useful as insecticides against household pests such as, e.g., the cockroach, clothes moth, carpet beetle and the housefly among others, as well as insect pests of stored grain and of agricultural plants, which pests include spider mites, aphids, caterpillars, and orthopterans such as locusts, among others.<br>
Animals that can be treated with the avermectin compounds produced according to the present invention include sheep, cattle, horses, deer, goats, swine, birds including poultry, and dogs and cats.<br>
An avermectin compound produced according to the' present invention is administered in a formulation appropriate to the specific intended use, the particular species of host animal being treated, and the parasite or insect involved. For use as a parasiticide, an avermectin compound produced according to the present invention can be administered orally in the form of a capsule, bolus, tablet or liquid drench or, alternatively, can be administered as a pour-on, or by injection, or as an implant. Such formulations are prepared in a conventional manner in accordance with standard veterinary practice. Thus, capsules, boluses or tablets can be prepared by mixing the active ingredient with a suitable finely divided diluent or carrier additionally containing a disintegrating agent and/or binder such as starch, lactose, talc, magnesium stearate, etc. A drench formulation can be prepared by dispersing the active ingredient in an aqueous solution together with a dispersing or wetting agent, etc. Injectable formulations can be prepared in the form of a sterile solution, which can contain other substances such as, e.g., sufficient salts and/or glucose to make the solution isotonic with blood.<br>
Such formulations will vary with regard to the weight of active compound depending on the patient, or species of host animal to be treated, the severity and type of infection, and the body weight of the host. Generally, for oral administration a dose of active compound of from about 0.001 to 10 mg per kg of patient or animal body weight given as a single dose or in divided doses for a period of from 1 to 5 days will be satisfactory.   However, there can be<br><br>
instances where higher or lower dosage ranges are indicated, as determined, e.g., by. a physician or veterinarian, as based on clinical symptoms.<br>
As an alternative, an avermectin compound produced according to the present invention can be administered in combination with animal feedstuff, and for this purpose a concentrated feed additive or premix can be prepared for mixing with the normal animal feed. For use as an insecticide, and for treating agricultural pests, an avermectin compound produced according to the present invention can be applied as a spray, dust, emulsion and the like in accordance with standard agricultural practice.<br>
6. EXAMPLE: FERMENTATION OF STREPTOMYCES AVERMITILIS AND B2:B1 AVERMECTIN ANALYSIS<br>
Strains   lacking   both   branched-chain   2-oxo   acid   dehydrogenase   and   5-0- ethyltransferase activities produce no avermectins if the fermentation medium is pot upplemented with fatty acids. This example demonstrates that in such mutants a wide range f B2:B1 ratios of avermectins can be obtained when biosynthesis is initiated in the presence of different fatty acids.<br>
6.1. Materials And Methods<br>
Streptomyces avermitilis ATCC 53692 was stored at -70°C as a whole broth prepared in seed medium consisting of: Starch (Nadex, Laing National) - 20g;
g/liter.   The fermentation broth was incubated for 14 days at 28°C, then the broth was<br>
centrifuged (2,500 rpm for 2 min) and the supernatant discarded.   The mycelial pellet was extracted with acetone (15 ml), then with dichloromethane (30 ml), and the organic phase<br><br>
separated, filtered, then evaporated to dryness. The residue was taken up in methanol (1 ml) and analyzed by HPLC with a Hewlett-Packard 1090A liquid chromatograph equipped with a scanning diode-array detector set at 240 nm. The column used was a Beckman Ultrasphere C-18, 5 nm, 4.6 mm x 25 cm column maintained at 40°C.   Twenty-five μl of the above<br>
5 methanol solution was injected onto the column. Elution was performed with a linear gradient of mefhanol-water from 80:20 to 95:5 over 40 min at 0.85/ml min. Two standard concentrations of cyclohexyl B1 were used to calibrate the detector response, and the area under the curves for B2 and B1 avermectins was measured.<br>
6.2. Results<br>
 	The, HPLC retention times observed for the B2 and B1 avermectins, and the 2:1<br>
ratios, are shown in TABLE 1.<br>
TABLE 1<br><br>
	HPLC Retention Time (min)	Ratio<br>
Substrate	B2	B1	B2.B1<br>
4-Tetrahydropyran parboxylic acid	8.1	.    14.5	0.25<br>
Isobutyric acid	10.8	18.9	0.5<br>
3-Furoic acid	7.6	14.6	0.62<br>
S-(+)-2-methylbutyric acid	12.8	21.6	10<br>
Cyclohexanecarboxyiic acid	16.9	26.0	1.6<br>
3-Thiophenecarboxylic acid	8.8	16.0	1.8<br>
Cyclopentanecarboxylic acid	14.2	23.0	2.0<br>
3-Trifluoromethylbutyric acid	10.9	18.8	3.9<br>
2-Methylpentanoic acid	14.5	24.9	4.2<br>
Cycloheptanecarboxylic acid	18.6	29.0	15.0<br>
The data presented in TABLE 1 demonstrates an extremely wide range of B2:B1 avermectin product ratios, indicating a considerable difference 'in the results of detiydrative conversion of class 2 compounds to class 1 compounds, depending on the nature of the fatty acid side chain starter unit supplied.   This indicates that changes in B2:B1 ratios resulting<br><br>
from alterations to the AveC protein may be specific to particular substrates. Consequently, screening for mutants exhibiting changes in the B2:B1 ratio obtained with a particular substrate needs to be done in the presence of that substrate. The subsequent examples described below use cyclohexanecarboxylic acid as the screening substrate. However, this substrate is used merely to exemplify the potential, and is not intended to limit the applicability, of the present invention.<br>
7. EXAMPLE: ISOLATION OF THE aveC GENE This example describes the isolation and characterization of a region of the Streptomyces   avermitilis  chromosome   that   encodes   the   AveC   gene   product.      As demonstrated below, the aveC gene was identified as capable of modifying the ratio of cyclohexyl-B2 to cyclohexyl-B1 (B2:B1) avermectins produced.<br>
7.1. Materials And Methods 7.1.1. Growth Of Streptomyces For DNA Isolation The following method was followed for growing Streptomyces. Single colonies of S. avermitilis ATCC 31272 (single colony isolate #2) were isolated on 1/2 strength YPD-6 containing: Difco Yeast Extract - 5 g; Difco Bacto-peptone - 5 g; dextrose - 2.5 g; MOPS . 5 g; Difco Bacto agar -15 g. Final volume was adjusted to 1 liter with dH20, pH was adjusted to 7.0, and the medium was autoclaved at 121 °C for 25 min.<br>
The mycelia grown in the above medium were used to inoculate 10 ml of TSB medium (Difco Tryptic Soy Broth - 30 g, in 1 liter dH20, autoclaved at 121 °C for 25 min) in a 25 mm x 150 mm tube which was maintained with shaking (300 rpm) at 28°C for 48-72 hr§. 7.1.2. Chromosomal DNA Isolation From Streptomyces Aliquots (0.25 mi or 0.5 ml) of mycelia grown as described above were placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded and the cells were resuspended in 0.25 ml TSE buffer (2o ml 1.5 M sucrose, 2.5 ml 1 M Tris-HCI, pH 8.0, 2.5 ml 1 M EDTA, pH 8.0, and 75 ml dH20) containing 2 mg/ml lysozyme. The samples were incubated at 37°C for 20 min with shaking, loaded into an AutoGen 540™ automated nucleic acid isolation instrument (Integrated Separation Systems, Natick, MA), and genomic DNA isolated using Cycle 159 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 5 ml of mycelia were placed in a 17 mm. x 100 mm tube, the cells concentrated by centrifugation at 3,000 rpm for 5 min, and the supernatant removed. Cells were resuspended in 1 ml TSE buffer, concentrated by centrifugation at 3,000 rpm for 5 min, and the supernatant removed. Cells were resuspended in 1 ml TSE buffer containing 2 mg/ml<br><br>
lysozyme, and incubated at 37°C with shaking for 30-60 min. After incubation, 0.5 ml 10% sodium dodecyl sulfate (SDS) was added and the cells incubated at 37°C until lysis was complete. The lysate was incubated at 65°C for 10 min, cooled to rm temp, split into two 1.5 ml Eppendorf tubes, and extracted 1x with 0.5 ml phenol/chloroform (50% phenol previously equilibrated with 0.5 M Tris, pH 8.0; 50% chloroform). The aqueous phase was removed and extracted 2 to 5x with chlorofprm:isoamyl alcohol (24:1). The DNA was precipitated by adding 1/10 volume 3M sodium acetate, pH 4.8, incubating the mixture on ice for 10 min, centrifuging the mixture at 15,000 rpm at 5°C for 10 min, and removing the supernatant to a clean tube to which 1 volume of isopropanol was added. The supernatant plus isopropanol mixture was then incubated on ice for 20 min, centrifuged at 15,000 rpm for 20 min at 5°C, the supernatant removed, and the DNA pellet washed 1x with 70% ethanol. After the pellet was dry, the DNA was resuspended in TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0).<br>
7.1.3. Plasmid DNA Isolation From Streptomyces An aliquot (1.0 ml) of mycelia was placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded, the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at 12,000 x g for 60 sec, and the supernatant discarded. The cells were then resuspended in 0.25 ml TSE buffer containing 2 mg/ml lysozyme, and incubated at 37?C for 20 min with shaking and loaded into the AutoGen 540™ automated nucleic acid isolation instrument. Plasmid DNA was isolated using Cycle 106 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 1.5 ml of mycelia were placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded, the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at 12,000 x g for 60 sec, and the supernatant discarded. The cells were resuspended in 0.5 ml TSE buffer containing 2 mg/ml lysozyme, and incubated at 37°C for 15-30 min. After incubation, 0.25 ml alkaline SDS (0.3N NaOH, 2% SDS) was added and the cells incubated at 55°C for 15-30 min or until the solution was clear. Sodium acetate (0.1 ml, 3M, pH 4.8) was added to the DNA solution, which was then incubated on ice for 10 min. The DNA samples were centrifuged at 14,000 rpm for 10 min at 5°C. The supernatant was removed to a clean tube, and 0.2 ml phenol :chloroform (50% phenol :50% chloroform) was added and gently mixed. The DNA solution was centrifuged at 14,000 rpm for 10 min at 5°C and the upper layer removed to a clean Eppendorf tube. Isopropanol (0.75 ml) was added, and the solution was gently mixed and then incubated at rm temp for 20 min.    The DNA solution was<br><br>
centrifuged at 14,000 rpm for 15 min at 5°C, the supernatant removed, and the DNA pellet was washed with 70% ethanol, dried, and resuspended in TE buffer.<br>
7.1.4. Plasmid DNA Isolation From E. coli A single transformed E. coli colony was inoculated into 5 ml Luria-Bertani (LB)<br>
E medium (Bacto-Tryptone -10 g, Bacto-yeast extract - 5 g, and NaCI -10 g in 1 liter dH20, pH 7.0, autociaved -at 121°C for 25 min, and supplemented with 100 [ig/ml ampicillin). The culture was incubated overnight, and a 1 ml aliquot placed in a 1.5 ml microcentrifuge tube. The culture samples were loaded into the AutoGen 540™ automated nucleic acid isolation instrument and plasmid DNA was isolated using Cycle 3 (equipment software) according to<br>
     manufacturer's instructions.<br>
7.1.5.    Preparation And Transformation Of S. avermitilis Protoplasts<br>
Single colonies of s. avermitilis were isolated on 1/2 strength YPD-6.   The mycelia<br>
were used to inoculate 10 ml of TSB medium in a 25 mm x 150 mm tube, which was then   incubated with shaking (300 rpm) at 28°C for 48 hrs. One ml of mycelia was used to inoculate 50 ml YEME medium.   YEME medium contains per liter:   Difco Yeast Extract - 3 g; Difco Bacto-peptone - 5 g; Difco Malt Extract - 3 g; Sucrose - 300 g. After autoclaving at 121°C for 25 min, the following were added: 2.5 M MgCI2   6H20 (separately autoclaved at 121°C for 25 min) - 2 ml; and glycine (20%) (filter-sterilized)- 25 ml. The mycelia were grown at 30°C for 48-72 hrs and harvested by centrifugation in a 50 ml centrifuge tube (Falcon) at 3,000 rpm for 20 min. The supernatant was discarded and the mycelia were resuspended in P buffer, which contains:   sucrose - 205 g; K2S04 - 0.25 g; MgCI2   6H20 - 2.02 g; H2o - 600 ml; K2P04 (0.5%) - 10 ml; trace element solution* - 20 ml; CaCI2   2H20 (3.68%) - 1Qo ml; and MES buffer (i.O M, pH 6.5) - 10 ml.   (*Trace element solution contains per liter: ZnCI2 - 40 mg; FeCI3 • 6H20 - 200 mg; CuCI2 • 2H20 - 10 mg; MnCI2 4H20 - 10 mg; Na2B407    10H20 - 10 mg; (NH4)6 Mo^     4H20 - 10 mg).   The pH was adjusted to 6.5, final volume was adjusted to 1 liter, and the medium was filtered hot through a 0.45 micron filter.<br>
The mycelia were pelleted at 3,000 rpm for 20 min, the supernatant was iscarded,<br>
and the mycelia were resuspended in 20 ml P buffer containing 2 mg/ml lysozyme.   The<br>
mycelia were incubated at 35°C for 15 min with shaking, and checked microscopically to determine extent of protoplast formation. When protoplast formation was complete, the protoplasts were centrifuged at 8,000 rpm for 10 min. The supernatant was removed and the protoplasts were resuspended in 10 ml P buffer.  The protoplasts were centrifuged at 8,000<br><br>
as described therein. Plasmid DNA was isolated from S. lividans transformants as described in Section 7.1.3, above.<br>
7.1.7. Fermentation Analysis Of S. avermitilis Strains S. avermitilis mycelia grown on 1/2 strength YPD-6 for 4-7 days were inoculated into 1x6 inch tubes containing 8 ml of preform medium and two 5 mm glass beads. Preform medium contains: soluble starch (either thin boiled starch or KOSO, Japan Corn Starch Co.. Nagoya) • 20 g/L; Pharmamedia - 15 g/L; Ardamine pH - 5 g/L (Champlain Ind., Clifton, NJ); CaC03 - 2 g/L; 2x bcfa ("bcfa" refers to branched chain fatty acids) containing a final concentration in the medium of 50 ppm 2-(+/-)-methyl butyric acid, 60 ppm isobutyric acid, and 20 ppm isovaleric acid. The pH was adjusted to 7.2, and the medium was autoclaved at 121°Cfor 25min.<br>
The tube was shaken at a 17° angle at 215 rpm at 29°C for 3 days. A 2-ml aliquot of the seed culture was used to inoculate a 300 ml Erlenmeyer flask containing 25 ml of production medium which contains: starch (either thin boiled starch or KOSO) - 160 g/L; Nutrisoy (Archer Daniels Midland, Decatur, IL) -10 g/L; Ardamine pH -10 g/L; K2HP04 - 2 g/L; MgS04.4H20 - 2 g/L; FeS04.7H20 - 0.02 g/L; MnCI2 - 0.002 g/L; ZnS04.7H20 - 0.002 g/L; CaC03 -14 g/L; 2x bcfa (as above); and cyclohexane carboxylic acid (CHC) (made up as a 20% solution at pH 7.0) - 800 ppm. The pH was adjusted to 6.9, and the medium was autoclaved at 121 °C for 25 min.<br>
After inoculation, the flask was incubated at 29°C for 12 days with shaking at 200 rpm. After incubation, a 2 ml sample was withdrawn from the flask, diluted with 8 ml of methanol, mixed, and the mixture centrifuged at 1,250 x g for 10 min to pellet debris. The supernatant was then assayed by HPLC using a Beckman Ultrasphere ODS column (25 cm x 4.6 mm ID) with a flow rate of 0.75 ml/min and detection by absorbance at 240 nm. The mobile phase was 86/8.9/5.1 methanol/water/ acetonitrile.<br>
7.1.8. Isolation Of S. avermitilis PKS Genes<br>
A cosmid library of S. avermitilis (ATCC 31272, SC-2) chromosomal DNA was prepared and hybridized with a ketosynthase (KS) probe made from a fragment of the Saccharopolyspora erythraea polyketide synthase (PKS) gene. A detailed description of the preparation of cosmid libraries can be found in Sambrook et al., 1989, above. A detailed description of the preparation of Streptomyces chromosomal DNA libraries is presented in Hopwood et al., 1985, above. Cosmid clones containing ketosynthase-hybridizing regions were identified by hybridization to a 2.7 Kb Nodel/Eco47lll fragment from pEX26 (kindly supplied by Dr. P. Leadlay, Cambridge, UK).   Approximately 5 ng of pEX26 were digested<br><br>
using A/del and Eco47lll.   The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts, Rockland, ME).   The 2.7 Kb A/del/Eco47lll fragment was excised from the gel after electrophoresis and the DNA recovered from the gel using GELase™ from Epicentre Technologies using the Fast Protocol.  The 2.7 Kb A/del/Eco47lll fragment    was    labeled    with     [α-32PJdCTP     (deoxycytidine     5-triphosphate,     tetra (triethylammonium) salt,  [alpha-32?]-)  (NEN-Dupont,  Boston,  MA)  using the  BRL  Nick Translation System (BRL Life Technologies, Inc., Gaithersburg, MD) following the supplier's instructions. A typical reaction was performed in 0.05 ml volume. After addition of 5 μl Stop buffer, the labeled DNA was separated from unincorporated nucleotides using a G-25 Sephadex Quick Spin™ Column (Boehringer Mannheim) following supplier's instructions. Approximately 1,800 cosmid clones were screened by colony hybridization.   Ten clones were identified that hybridized strongly to the Sacc. erythraea KS probe.   E. coli<br>
colonies containing cosmid DNA were grown in LB liquid medium and cosmid DNA was isolated from each culture in the AutoGen 540™ automated nucleic acid isolation instrument using Cycle 3 (equipment software) according to manufacturer's instructions.   Restrict on endonuclease mapping and Southern blot hybridization analyses revealed that five of the clones contained overlapping chromosomal regions.    An  S. avermitilis genomic SamHl restriction map of the five cosmids (i.e., pSE65, pSE66, pSE67, pSE68, pSE69) was constructed by analysis of overlapping cosmids and hybridizations (FIGURE 4).<br>
7.1.9.    identification Of DNA That Modulates Avermectin B2:B1 Ratios And Identification Of An aveC ORF<br>
The following methods were used to test subcloned fragments derived from the<br>
pSE66 cosmid clone, for their ability to modulate avermectin B2.B1 ratios in AveC mutants. pSE66 (5 μg) was digested with Sacl and SamHl.   The reaction mixture was loaded on a 0.8% SeaPlaque™ GTG agarose gel (FMC BioProducts), a 2.9 Kb Sacl/BamHI fragment was excised from the gebafter electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol.   Approximately 5 μg of the shuttle vector pWHM3 (Vara et al., 1989, J. Bacteriol. 171:5872-5881) was digested with Sacl and SamHl.  About 0.5 μg of the 2.9 Kb insert and 0.5 ug of digested pWHM3 were mixed together and incubated overnight with 1 unit of ligase (New England Biolabs, Inc., Beverly, MA) at 15°C, in a total volume of 20 μl, according to supplier's instructions. After incubation, 5 μl of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to<br>
transform competent E. coli DH5a cells (BRL) according to manufacturer's instructions.<br><br>
Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the 2.9 Kb Sacl/SamHI insert was confirmed by restriction analysis. This plasmid was designated as pSE119.<br>
Protoplasts of S. avermitilis strain 1100-SC38 (Pfizer in-house strain) were prepared and transformed with pSE119 as described in Section 7.1.5 above. Strain 1100-SC38 is a mutant that produces significantly more of the avermectin cyclohexyl-B2 form compared to avermectin cyclohexyl-B1 form when supplemented with cyclohexane carboxylic acid (B2:B1 of about 30:1). pSE119 used to transform S. avermitilis protoplasts was isolated from either E. co//strain GM2163. (obtained from Dr. B. J. Bachmann, Curator, E. co//Genetic Stock Center, Yale University), E. coli strain DM1 (BRL), or S. lividans strain TK64. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE119 produced an altered ratio of avermectin cyclohexyl-B2:cyclohexyl-B1 of about 3.7:1 (TABLE 2).<br>
Having established that pSE119 was able to modulate avermectin B2:B1 ratios in an AveC mutant, the insert DNA was sequenced. Approximately 10 μg of pSE119 were isolated using a plasmid DNA isolation kit (Qiagen, Valencia, CA) following manufacturer's instructions, and sequenced using an ABI 373A Automated DNA Sequencer (Perkin Elmer, Foster City, CA). Sequence data was assembled and edited using Genetic Computer Group programs (GCG, Madison, Wl). The DNA sequence and the aveC ORF are presented in FIGURE 1 (SEQ ID NO:1).<br>
A new plasmid, designated as pSE118, was constructed as follows. Approximately 5 jig of pSE66 was digested with Sph\ and BamHI. The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts), a 2.8 Kb Sphl/BamHI fragment was excised from the gel after electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol. Approximately 5 μg of the shuttle vector pWHM3 was digested with Sph\ and BamHI. About 0.5 μg of the 2.8 Kb insert and 0.5 μg of digested pWHM3 were mixed together and incubated overnight with 1 unit of ligase (New England Biolabs) at 15°C in a total volume of 20 μl according to supplier's instructions. After incubation, 5 μl of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to transform competent E. coli DH5cc cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the 2.8 Kb SphllBamH\ insert was confirmed by restriction analysis. This plasmid was designated as<br><br>
pSE118.  The insert DNA in pSE118 and pSE119 overlap by approximately 838 nucleotides (FIGURE 4).<br>
Protoplasts of S. avermitilis strain 1100-SC38 were transformed with pSE118 as above. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE118 were not altered in the ratios of avermectin cyclohexyl-B2: avermectin<br>
cyclohexyl-B1 compared to strain 1100-SC38 (TABLE 2).<br>
7.1.10.   PCR Amplification Of The aveC Gene From S. avermitilis Chromosomal DNA<br>
A -1.2 Kb fragment containing the aveC ORF was isolated from S. avermitilis chromosomal DNA by PCR amplification using primers designed on the basis of the aveC nucleotide sequence obtained above. The PCR primers were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward primer was: 5- CACGAAACCGGACACAC-3' (SEQ ID NO:6); and the leftward primer was: 5'- CATGATCGCTGAACCGAG-3' (SEQ ID NO:7). The PCR reaction was carried out with Deep Vent™ polymerase (New England Biolabs) in buffer provided by the manufacturer, and in the presence of 300μM dNTP, 10% glycerol, 200 pmol of each primer, 0.1 μg template, and 2.5 units enzyme in a final volume of 100 μl, using a Perkin-Elmer Cetus thermal cycler. The thermal profile of the first cycle was 95°C for 5 min (denaturation step), 60°C for 2 min (annealing step), and 72°C for 2 min (extension step). The subsequent 24 cycles had a similar thermal profile except that the denaturation step was shortened to 45 sec and the annealing step was shortened to 1 min.<br>
The PCR product was electrophoresed in a 1 % agarose gel and a single DNA band of -1.2 Kb was detected. This DNA was purified from the gel, and ligated with 25 ng of linearized, blunt pCR-Blunt vector (Invitrogen) in a 1:10 molar vector-to-insert ratio following manufacturer's instructions. The ligation mixture was used to transform One Shot™ Competent E. coli cells (Invitrogen) following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.2 Kb insert was confirmed by restriction analysis. This plasmid was designated as pSE179.<br>
The insert DNA from pSE179 was isolated by digestion with BamHI/Xbal, separated by electrophoresis, purified from the gel, and ligated with shuttle vector pWHM3, which had also been digested with BamH\IXba\, in a total DNA concentration of 1 μg in a 1:5 molar vector-to-insert ratio. The ligation mixture was used to transform competent E. coli DH5ct cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the -1.2 Kb insert was confirmed by restriction<br><br>
analysis. This plasmid, which was designated as pSE186 (FIGURE 2, ATCC 209604), was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transformants.<br>
7.2. Results<br>
A 2.9 Kb Sac\IBamH\ fragment from pSE119 was identified that, when transformed into S. avermitilis strain 1100-SC38, significantly altered the ratio of B2:B1 avermectin production. S. avermitilis strain 1100-SC38 normally has a B2:B1 ratio of about 30:1, but when transformed with a vector comprising the 2.9 Kb Sacl/BamHI fragment, the ratio of B2.B1 avermectin decreased to about 3.7:1. Post-fermentation analysis of transformant cultures verified the presence of the transforming DNA.<br>
The 2.9 Kb pSE119 fragment was sequenced and a -0.9 Kb ORF was identified (FIGURE 1) (SEQ ID NO:1), which encompasses a Pst\ISph\ fragment that had previously been mutated elsewhere to produce B2 products only (Ikeda et a/., 1995, above). A comparison of this ORF, or its corresponding deduced polypeptide, against known databases (GenEMBL, SWISS-PROT) did not show any strong homology with known DNA or protein sequences.<br>
TABLE 2 presents the fermentation analysis of S. avermitilis strain 1100-SC38 transformed with various plasmids.<br>
TABLE 2<br><br>
S. avermitilis strain (transforming plasmid)	No. Transformants Tested	Avg. B2:B1<br>
Ratio<br>
1100-SC38(none)	9	30.66<br>
1100-SC38(pWHM3)	21	31.3<br>
1100-SC38(pSE119)	12	3.7<br><br>
1100-SC38(pSE118)	12	30.4<br>
1100-SC38(pSE185)	14	27.9<br>
8. EXAMPLE: CONSTRUCTION OF S. AVERMITILIS AveC MUTANTS<br>
This example describes the construction of several different S. avermitilis AveC<br>
mutants using the compositions and methods described above.   A general description of<br>
techniques for introducing mutations into a gene in Streptomyces is described by Kieser and<br><br>
Hopwood, 1991, Meth. Enzym. 204:430-458. A more detailed description is provided by Anzai et a/., 1988, J. Antibiot. XLI(2):226-233, and by Stutzman-Engwall et a/.. 1992, J. Bacterid. 174(1):144-154. These references are incorporated herein by reference in their entirety.<br>
8.1. Inactivation Of The S. avermitilis aveC Gene AveC mutants containing inactivated aveC genes were constructed using several methods, as detailed below.<br>
in the first method, a 640 bp Sph\lPst] fragment internal to the aveC gene in pSE119 (plasmid described in Section 7.1.9, above) was replaced with the ermE gene (for erythromycin resistance) from Sacc. erythraea. The ermE gene was isolated from plJ4026 (provided by the John Innes Institute, Norwich, U.K.; see also Bibb et a/., 1985, Gene 41:357-368) by restriction enzyme digestion with Bg/ll and EcoRI, followed by electrophoresis, and was purified from the gel. This -1.7 Kb fragment was ligated into pGEM7Zf (Promega) which had been digested with BamHl and EcoRI, and the ligation mixture transformed into competent E. coli DH5a cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.7 Kb insert was confirmed by restriction analysis. This plasmid was designated as pSE27.<br>
pSE118 (described in Section 7.1.9, above) was digested with Sphl and BamHl, the digest electrophoresed, and the -2.8 Kb Sphl/BamHI insert purified from the gel. pSE119 was digested with Psti and EcoRI, the digest electrophoresed, and the -1.5 Kb Psfl/EcoRI insert purified from the gel. Shuttle vector pWHM3 was digested with BamHl and EcoRI. pSE27 was digested with Pst\ and Sphl, the digest electrophoresed, and the -1.7 Kb Psf/Sp/?l insert purified from the gel. All four fragments (;.e., -2.8 Kb, -1.5Kb, -7.2Kb, -1.7 Kb) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. coli DH5oc cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE180 (FIGURE 3; ATCC 209605).<br>
pSE180 was transformed into S. Iividans TK64 and transformed colonies identified by resistance to thiostrepton and erythromycin. pSE180 was isolated from S. Iividans and used to transform S. avermitilis protoplasts. Four thiostrepton resistant S. avermitilis transformants were identified, and protoplasts were prepared and plated under non-selective conditions on RM14 media. After the protoplasts had regenerated; single colonies were screened for the presence of erythromycin resistance and the absence of thiostrepton resistance, indicating<br><br>
chromosomal integration of the inactivated aveC gene and loss of the free replicon. One Ermr Thios transformant was identified and designated as strain SE180-11. Total chromosomal DNA was isolated from strain SE180-11, digested with restriction enzymes SamHI, HindWU Pst\, or Sphl, resolved by electrophoresis on a 0.8% agarose gel, transferred to nylon membranes, and hybridized to the ermE probe. These analyses showed that chromosomal integration of the ermE resistance gene, and concomitant deletion of the 640 bp Pstt/Sphl fragment had occurred by a double crossover event. HPLC analysis of fermentation products of strain SE180-11 showed that normal avermectins were no longer produced (FIGURE 5A).<br>
In a second method for inactivating the aveC gene, the 1.7 Kb ermE gene was<br>
removed from the chromosome of S. avermitilis strain SE180-11, leaving a 640 bp PstllSphl deletion in the aveC gene. A gene replacement plasmid was constructed as follows: pSE180 was partially digested with Xbal and an -11.4 Kb fragment purified from the gel.  The ~11.4 Kb band lacks the 1.7 Kb ermE resistance gene. The DNA was then ligated and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE184, was transformed into £ coli DM1, and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis  strain   SE180-11.      Protoplasts   were   prepared   from   thiostrepton   resistant transformants of strain SE180-11 and were plated as single colonies on RM14.   After the protoplasts had regenerated,  single colonies were screened for the absence of both erythromycin resistance and thiostrepton resistance, indicating chromosomal integration of the<br>
inactivated aveC gene and loss of the free replicon containing the ermE gene.   One Erms Thios transformant was identified and designated as SE184-1-13.  Fermentation analysis of SE184-1-13 showed that normal avermectins were not produced and that SE184-1-13 had the same fermentation profile as SE180-11.<br>
In a third method for inactivating the aveC gene, a frameshift was introduced into the chromosomal aveC gene by adding two G's after the C at nt position 471 using PCR, thereby creating a BspE1 site. The presence of the engineered BspE1 site was useful in detecting the gene replacement event. The PCR primers were designed to introduce a frameshift mutation into the aveC gene, and were supplied by Genosys Biotechnologies, Inc. The rightward primer was: 5'-GGTTCCGGATGCCGTTCTCG-3' (SEQ ID NO:8) and the leftward primer was: 5'-AACTCCGGTCGACTCCCCTTC-3" (SEQ ID NO:9). The PCR conditions were as described in Section 7.1.10 above. The 666 bp PCR product was digested with Sphl to give<br><br>
two fragments of 278 bp and 388 bp, respectively. The 388 bp fragment was purified from the gel.<br>
The gene replacement plasmid was constructed as follows: shuttle vector pWHM3 was digested with EcoRl and BamHl. pSE119 was digested with SamHI and Sphl, the digest electrophoresed, and a -840 bp fragment was purified from the gel. pSE119 was digested with EcoRl and Xmn\, the digest was resolved by electrophoresis, and a -1.7 Kb fragment was purified from the gel. All four fragments (i.e., -7.2 Kb, -840 bp, -1.7 Kb, and 388 bp) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. coli DH5cx cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid, which was designated as pSE185, was transformed into E. coli DM1 and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis strain 1100-SC38. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. pSE185 did not significantly alter the B2:B1 avermectin ratios when transformed into S. avermitilis strain 1100-SC38 (TABLE 2).pSE185 was used to transform protoplasts of S. avermitilis to generate a frameshift<br>
mutation in the chromosomal aveC gene. Protoplasts were prepared from thiostrepton<br>
resistant transformants and plated as single colonies on RM14. After the protoplasts had<br>
regenerated, single colonies were screened for the absence of thiostrepton resistance.<br>
Chromosomal DNA from thiostrepton sensitive colonies was isolated and screened by PCRfor the presence of the frameshift mutation integrated into the chromosome. The PCR primers were designed based on the aveC nucleotide sequence, and were supplied by Genosys Biotechnologies,     Inc.     (Texas).	The     rightward     PCR     primer     was:	5'- GCAAGGATACGGGGACTAC-3' (SEQ ID NO:10) and the leftward PCR primer was: 5-GAACCGACCGCCTGATAC-3* (SEQ ID NO:11), and the PCR conditions were as described in Section 7.1.10 above. The PCR product obtained was 543 bp and, when digested with BspE1, three fragments of 368 bp, 96 bp, and 79 bp were observed, indicating chromosomal integration of the inactivated aveC gene and loss of the free replicon.<br>
Fermentation analysis of S. avermitilis mutants containing the frameshift mutation in the aveC gene showed that normal avermectins were no longer produced, and that these mutants had the same fermentation HPLC profile as strains SE180-11 and SE184-1-13. One Thios transformant was identified and designated as strain SE185-5a.<br><br>
Additionally, a mutation in the aveC gene that changes nt position 520 from G to A, which results in changing the codon encoding a tryptophan (W) at position 116 to a termination codon, was produced. An S. avermitilis strain with this mutation did hot produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, and SE185-5a.<br>
Additionally, mutations in the aveC gene that change both: (i) nt position 970 from G to A, which changes the amino acid at position 266 from a glycine (G) to an aspartate (D), and (ii) nt position 996 from T to C, which changes the amino acid at position 275 from tyrosine (Y) to histidine (H), were produced. An S. avermitilis strain with these mutations (G256D/Y275H) did not produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, and SE185-5a.<br>
The S. avermitilis aveC inactivation mutant strains SE180-11, SE184-1-13, SE185- 5a, and others provided herewith, provide screening tools to assess the impact of other mutations in the aveC gene. pSE186, which contains a wild-type copy of the aveC gene, was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transformants.  This pSE186 DNA was used to transform protoplasts of S. avermitilis strain SE180-11.    Thiostrepton resistant transformants of strain SE180-11 were isolated, the<br>
presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products.  The presence of the functional aveC gene in trans was able to restore normal avermectin production to strain SE180-11 (FIGURE 5B).<br>
8.2. Analysis Of Mutations In The aveC Gene That Alter Class B2:B1 Ratios<br>
As described above, S. avermitilis strain SE180-11 containing an inactive aveC gene was complemented by transformation with a plasmid containing a functional aveC gene (pSE186). Strain SE180-11 was also utilized as a host strain to characterize other mutations in the aveC gene, as described below.<br>
Chromosomal DNA was isolated from strain 1100-SC38, and used as a template for PCR amplification of the aveC gene. The 1.2 Kb ORF was isolated by PCR amplification using primers designed on the basis of the aveC nucleotide sequence. The rightward primer was SEQ ID NO:6 and the leftward primer was SEQ ID NO:7 (see Section 7.1.10, above). The PCR and subcloning conditions were as described in Section 7.1.10. DNA sequence analysis of the 1.2 Kb ORF shows a mutation in the aveC gene that changes nt position 337 from C to T, which changes the amino acid at position 55 from serine (S) to phenylalanine (F). The aveC gene containing the S55F mutation was subcloned into pWHM3 to produce a<br><br>
plasmid which was designated as pSE187, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the aveC gene encoding a change at amino acid residue 55 (S55F) was able to restore norma/ avermectin production to strain SE180-11 (Fig. 5C); however, the cylohexyl   B2:cyclohexyl B1 ratio was about 26:1, as compared to strain SE180-11 transformed with pSE186, which had a ratio of B2:B1 of about 1.6:1 (TABLE 3), indicating that the single mutation (S55F) modulates the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1.<br>
Another mutation in the aveC gene was identified that changes nt position 862 from G to A, which changes the amino acid at position 230 from glycine (G) to aspartate (D). An S. avermitilis strain having this mutation (G230D) produces avermectins at a B2:B1 ratio of about 30:1.<br>
8.3. Mutations That Reduce The B2:B1 Ratio Several  mutations were constructed that reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1, as follows.<br>
A mutation in the aveC gene was identified that changes nt position 588 from G to A, which changes the amino acid at position 139 from alanine (A) to threonine (T). The aveC gene containing the A139T mutation was subcloned into pWHM3 to produce a plasmid which was designated pSE188, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 139 (A139T) was able to restore avermectin production to strain SE180-11 (FIGURE 5D); however, the B2:B1 ratio was about 0.94:1, indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1. This result was unexpected because published results, as well as the results of mutations described above, have only demonstrated either inactivation of the aveC gene or increased production of the B2 form of avermectin relative to the B1 form (TABLE 3).<br>
Because the A139T mutation altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed that encoded a threonine instead of a serine at amino acid position 138. Thus, pSE186 was digested with EcoRI and cloned into pGEM3Zf (Promega) which had been digested with EcoRI. This plasmid, which was designated as pSE186a, was digested with Apa\ and Kpn\, the DNA fragments separated on an agarose gel,<br><br>
and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel.  The -1.2 Kb insert DNA from pSE186 was used as a PCR template to introduce a single base change at nt position 585. The PCR primers were designed to introduce a mutation at nt position 585, and were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward PCR primer was: 5'- GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCCCTGGCGACG-3' (SEQ ID NO: 12); and the leftward PCR primer was:   5'-GGAACCGACCGCCTGATACA-3, (SEQ ID NO:13). The PCR reaction was carried out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 μM dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.0 M GC-Melt and 1 unit KlenTaq Polymerase Mix in a final volume of 50 μl The thermal profile of the first cycle was 94CC for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 295 bp was digested with Apal and Kpnl to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel.  All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation. The ligation mixture was transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE198. pSE198 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE199, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 138 (S138T) was able to restore normal avermectin production to strain SE1 so¬μl; however, the B2:B1 ratio was 0.88:1 indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3). This B2:B1 ratio is even lower than the 0.94:1 ratio observed with the A139T mutation produced by transformation of strain SE180-11 with pSE188, as described above.<br>
Another mutation was constructed to introduce a threonine at both amino acid positions 138 and 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template.<br><br>
The PCR primers were designed to introduce mutations at nt positions 585 and 588, and were supplied by Genosys Biotechnologies, Inc. (Texas).   The rightward PCR primer was:   5'-GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGCTGGCGACGACC-3'     (SEQ      ID NO:14); and the leftward PCR primer was:   5'-GAACATCACGGCATTCACC-3' (SEQ ID<br>
NO:15). The PCR reaction was performed using the conditions described immediately above in this Section. A PCR product of 449 bp was digested with Apa\ and Kpn\ to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. pSE186a was digested with Apa\ and Kpn\, the DNA fragments separated on an agarose gel, and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel.  All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation, and the ligation mixture was transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE230.pSE230 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE231, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of SE180-11 were, isolated, the presence of erythromycin resistance was determined, and Thior Ermr transfonnants were analyzed by fermentation. The presence of the double mutated aveC gene, encoding S138T/A139T, was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.84:1 showing that this mutation further reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3), over the reductions provided by transformation of strain SE180-11 with pSE188 or pSE199, as described above.<br>
Another mutation was constructed to further reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1. Because the S138T/A139T mutations altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed to introduce a threonine at amino acid position 138 and a phenylalanine at amino acid position 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template. The PCR primers were designed to introduce mutations at nt positions 585 (changing a T to A), 588 (changing a G to T), and 589 (changing a C to T), and were supplied by Genosys Biotechnologies, Inc. (Texas).   The<br><br>
rightward PCR primer was:    5'-GGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGC TGGCGACGTTC-3"   (SEQ   ID   NO:25);   and   the   leftward   PCR   primer   was:       5'-<br>
GGAACATCACGGCATTCACC-3' (SEQ ID N0:15). The PCR reaction was carried    out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 |iM dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.1 mM Mg acetate, 1.0 M GC-Melt and 1 unit Tth DNA Polymerase in a final volume of 50 \A.  The thermal profile of the first cycle was 94°C for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 449 bp was digested with Apa\ and Kpn\ to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation.   The ligation mixture was transformed into competent E.  coli DH5a cells.     Plasmid  DNA was  isolated from ampicillin  resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE238. pSE238 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI. and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE239, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the double mutated aveC gene encoding S138T/A139F was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.75:1 showing that this mutation further reduced the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3) over the reductions provided by transformation of strain SE180-11 with pSE188, pSE199, or pSE231 as described above.<br><br>
TABLE 3<br><br>
S. avermitilis strain (transforming plasmid)	No.<br>
Transformants<br>
Tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	30	0	0	0<br>
SE180-11(pWHM3)	30	0	0	0<br>
SE180-11 (pSE186)	26	222	140	1.59<br>
SE180-11(pSE187)	12	283	11	26.3<br>
SE180-11 (pSE188)	24	193	206	0.94<br>
SE180-11 (pSE199)	18	155	171	0.88<br>
SE180-11(pSE231)	6	259	309	0.84<br>
SE180-11(pSE239) .	20                         184	242	0.75<br>
Additional mutations were constructed to further reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 using the technique of DNA shuffling as described in Stemmer, 1994, Nature 370:389-391; and Stemmer, 1994, Proc. Natl. Acad. Sci. USA 91:10747-10751; and further described in United States Patents 5605793, 5811238, 5830721, and 5837458.<br>
DNA shuffled plasmids containing mutated aveC genes were transformed into competent dam dcm E. coli cells. Plasmid DNA was isolated from ampicillin resistant transformants, and used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated and screened for the production of avermectins with a cyclohexyl-B2:cyclohexyl-B1 ratio of 1:1 or less. The DNA sequence of plasmid DNA from SE180-11 transformants producing avermectins with a B2:B1 ratio of 1:1 or less was determined.<br>
Eight transformants were identified that produced reduced amounts of cyclohexyl-B2 relative to cyclohexyl-B1. The lowest B2:B1 ratio achieved among these transformants was 04:1 (TABLE 4). Plasmid DNA was isolated from each of the eight transformants and the DNA sequence determined to identify the mutations in the aveC gene. The mutations are as follows.<br>
pSE290 contains 4 nucleotide mutations at nt position 317 from T to A, at nt position 353 from C to A, at nt position 438 from G to A, and at nt position 1155 from T to A. The nucleotide change at nt position 317 changes the amino acid at AA position 48 from D to E and the nucleotide change at nt position 438 changes the amino acid at AA position 89 from A to T. The B2:B1 ratio produced by cells carrying this plasmid was 0.42:1 (TABLE 4).<br><br>
pSE291 contains 4 nucleotide mutations at nt position 272 from G to A, at nt position 585 from T to A, at nt position 588 from G to A, and at nt position 708 from G to A. The nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, and the nucleotide change at nt position 708 changes the amino acid at AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.57:1 (TABLE 4).<br>
pSE292 contains the same four nucleotide mutations as pSE290. The B2:B1 ratio produced by cells carrying this plasmid was 0.40:1 (TABLE 4).<br>
pSE293 contains 6 nucleotide mutations at nt 24 from A to G, at nt position 286 from A to C, at nt position 497 from T to C, at nt position 554 from C to T, at nt position 580 from T to C, and at nt position 886 from A to T. The nucleotide change at nt position 286 changes the amino acid at AA position 38 from Q to P, the nucleotide change at nt position 580 changes the amino acid at AA position 136 from L to P, and the nucleotide change at nt position 886 changes the amino acid at AA position 238 from E to D. The B2:B1 ratio produced by cells carrying this plasmid was 0.68:1 (TABLE 4).<br>
pSE294 contains 6 nucleotide mutations at nt 469 from T to C, at nt position 585 from T to A, at nt position 588 from G to A, at nt position 708 from G to A. at nt position 833 from C to T, and at nt position 1184 from G to A. In addition, nts at positions 173, 174, and 175 are deleted. The nucleotide change at nt position 469 changes the amino acid at AA position 99 from F to S, the nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T. and the nucleotide change at nt position 708 changes the amino acid from AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.53:1 (TABLE 4).<br>
pSE295 contains 2 nucleotide mutations at nt 588 from G to A and at nt 856 from T to C. The nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T and the nucleotide change at nt position 856 changes the amino acid at AA position 228 from M to T. The B2.B1 ratio produced by cells carrying this plasmid was 0.80:1 (TABLE 4).<br>
pSE296 contains 5 nucleotide mutations at nt position 155 from T to C, at nt position 505 from G to T, at nt position 1039 from C to T, at nt position 1202 from C to T, and at nt position 1210 from T to C. The nucleotide change at nt position 505 changes the amino acid at AA position 111 from G to V and the nucleotide change at nt position 1039 changes the<br><br>
amino acid at AA position 289 from P to L. The B2:B1 ratio produced by cetls carrying this plasmid was 0.73:1 (TABLE 4).<br>
pSE297 contains.4 nucleotide mutations at nt position 377 from G to T, at nt position 588 from G to A, at nt position 633 from A to G, and at nt position 1067 from A to T. The nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, the nucleotide change at nt position 633 changes the amino acid at AA position 154 from K to E, and the nucleotide change at nt position 1067 changes the amino acid at AA position 298 from Q to H. The B2:B1 ratio produced by cells carrying this plasmid was 0.67:1 (TABLE 4).<br>
TABLE 4<br><br>
S. avermitilis strain (transforming piasmid)	No.<br>
Transformants<br>
Tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	4	0	0	0<br>
SE180-11(pWHM3)	4	0	0	0<br>
SE180-11 (pSE290)	4	87	208	0.42<br>
SE180-11 (pSE291)	4	106	185	0.57<br>
SE180-11 (pSE292)	4	91	231	0.40<br>
SE180-11 (pSE293)	4	123	180	0.68<br>
SE180-11 (pSE294)	4	68	129	0.53<br>
SE180-11 (pSE295)	4	217	271	0.80<br>
SE180-11 (pSE296)	1	135	186	0.73<br>
SE180-11 (pSE297)<br>
 	1	148	221	0.67<br>
9. EXAMPLE: CONSTRUCTION OF 5* DELETION MUTANTS<br>
  As explained in Section 5.1, above, the S. avermitilis nucleotide sequence shown in FIGURE 1 (SEQ ID NO:1) comprises four different GTG codons at bp positions 42, 174, 177 and 180 which are potential start sites. This section describes the construction of multiple deletions of the 5' region of the aveC ORF (FIGURE 1; SEQ ID NO:1) to help define which of these codons could function as start sites in the aveC ORF for protein expression.<br>
Fragments of the aveC gene variously deleted at the 5" end were isolated from S. avermitilis chromosomal DNA by PCR amplification. The PCR primers were designed based<br><br>
on the aveC DNA sequence, and were supplied by Genosys Biotechnologies, Inc. The rightward primers were 5'-AACCCATCCGAGCCGCTC-3' (SEQ ID NO:16) (D1F1); 5'-TCGGCCTGCCAACGAAC-3' (SEQ ID NO:17) (D1F2); 5'-CCAACGAACGTGTAGTAG-3' (SEQ ID NO:18) (D1F3); and 3'-TGCAGGCGTACGTGTTCAGC-3' (SEQ ID NO:19) (D2F2). The leftward primers were 5'-CATGATCGCTGAACCGA-3' (SEQ ID NO:20); 5'-CATGAT CGCTGAACCGAGGA-3 iSEQ )D ND:21); and P-AGGAGTGTGGTGCGTCTGGA-3 {SEQ. ID NO:22). The PCR reaction was carried out as described in Section 8.3, above.<br>
The PCR products were separated by electrophoresis in a 1 % agarose gel and single DNA bands of either -1.0 Kb or -1.1 Kb were detected. The PCR products were purified from the gel and ligated with 25 ng of linearized pCR2.1 vector (Invitrogen) in a 1:10 molar vector-to-insert ratio following the manufacturer's instructions. The ligation mixtures were used to transform One Shot™ Competent E. coli cells (Invitrogen) following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the insert was confirmed by restriction analysis and DNA sequence analysis. These plasmids were designated as pSEl90 (obtained with primer D1F1), pSE191 (obtained with primer D1F2), pSE193 (obtained with primer D1F3), and pSE193 (obtained with primer D2F2).<br>
The insert DNAs were each digested with BamHUXbal, separated by electrophoresis, purified from the gel, and separately ligated with shuttle vector pWHM3, which had been digested with BamHI/Xbal, in a total DNA concentration of 1 μg in a 1:5 molar vector-to-insert ratio. The ligation mixtures were used to transform competent E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the insert was confirmed by restriction analysis. These plasmids, which were designated as pSE194 (D1F1), pSE195 (D1F2). PSE196 (D1F3), and pSE197 (D2F2), were each separately transformed into E. coli strain DM1, plasmid DNA isolated from ampicillin resistant transformants, and the presence of the correct insert confirmed by restriction analysis. This DNA was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 'were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products to determine which GTG sites were necessary for aveC expression. The results indicate that the GTG codon at position 42 can be eliminated without affecting aveC expression, since pSE194 PSE195, and pSE196, each of which lack the GTG site at position 42, but which all certain the three GTG sites at positions 174, 177, and 180, were each able to restore normal avermectin production when transformed into SE180-11. Nomal<br><br>
avermectin production was not restored when strain SE180-11 was transformed with pSE197, which lacks all four of the GTG sites (TABLE 5).<br>
TABLE 5<br><br>
S. avermitilis strain (transforming plasmid)	No. transformants tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	6	0	0	0<br>
SE180-11 (pWHM3),	6	0	0	0<br>
SE180-11 (pSE186)	6	241	152	1.58<br>
SE180-11 (pSE194)	6	35	15	2.43<br>
SE180-11 (pSE195)	6	74	38	1.97<br>
SE180-11 (pSE196)	6	328	208	1.58<br>
SE180-11(pSE197)	12	0	0	0<br>
10. EXAMPLE: CLONING OF aveC HOMOLOGS FROM S. HYGROSCOPICUS AND S. GRISEOCHROMOGENES<br>
The present invention allows aveC homolog genes from other avermectin- or milbemycin-producing species of Streptomyces to be identified and cloned. For example, a cosmid library of S. hygroscopicus (FERM BP-1901) genomic DNA was hybridized with the 1.2 Kb aveC probe from S. avermitilis described above. Several cosmid clones were identified that hybridized strongly. Chromosomal DNA was isolated from these cosmids, and a 4.9 Kb Kpn\ fragment was identified that hybridized with the aveC probe. This DNA was sequenced and an ORF (SEQ ID NO:3) was identified having significant homology to the aveC ORF of S. avermitilis. An amino acid sequence (SEQ ID NO:4) deduced from the S. hygroscopicus aveC homolog ORF is presented in FIGURE 6.<br>
In addition, a cosmid library of S. griseochromogenes genomic DNA was hybridized with the 1.2 Kb aveC probe from S. avermitilis described above. Several cosmid clones were identified that hybridized strongly. Chromosomal DNA was isolated from these cosmids, and a 5.4 Kb Psfl fragment was identified that hybridized with the aveC probe. This DNA was sequenced and an aveC homolog partial ORF was identified having significant homology to the aveC ORF of S. avermitilis. A deduced partial amino acid sequence (SEQ ID NO:5) is' presented in FIGURE 6.<br><br>
DNA and amino acid sequence analysis of the aveC homoiogs from S. hygroscopicus and S. griseochromogenes indicates that these regions share significant homology (-50% sequence identity at the amino acid level) both to each other and to the S. avermitilis aveC ORF and AveC gene product FIGURE 6).<br>
11. EXAMPLE: CONSTRUCTION OF A PLASMID WITH THE aveC GENE BEHIND THE ermE PROMOTER<br>
The 1.2 Kb aveC ORF from pSE186 was subcioned in pSE34, which is the shuttle vector pWHM3 having the 300 bp ermE promoter inserted as a Kpn\IBamH\ fragment in the Kpn\IBamH\ site of pWHM3 (see Ward et al, 1986, Mol. Gen. Genet. 203:468-478). pSE186 was digested with BamHl and Hindlll, the digest resolved by electrophoresis, and the 1.2 Kb fragment was isolated from the agarose gel and ligated with pSE34, which had been digested with fiamHI and H/ndlll. The ligation mixture was transformed into competent E coli DH5a cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the 1.2 Kb insert was confirmed by restriction analysis. This plasmid, which was designated as pSE189, was transformed into E. coli DM1, and plasmid DNA isolated from ampicillin resistant transformants. Protoplasts of S. avermitilis strain 1100-SC38 were transformed with pSE189. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products.<br>
S. avermitilis strain 1100-SC38 transformants containing pSE189 were altered in the ratios of avermectin.cyclohexyl-B2:avermectin cyclohexyl-B1 produced (about 3:1) compared to strain 1100-SC38 (about 34:1), and total avermectin production was increased approximately 2.4-fold compared to strain 1100-SC38 transformed with pSE119 (TABLE 6).<br>
pSE189 was also transformed into protoplasts of a wild-type S. avermitilis strain. Thiostrepton resistant transformants were isolated and analyzed by HPLC analysis of fermentation products. Total avermectins produced by S. avermitilis wild-type transformed with pSE189 were increased approximately 2.2-fold compared to wild-type S. avermitilis transformed with pSE119 (TABLE 6).<br>
TABLE 6<br><br>
S. avermitilis strain<br>
(transforming plasmid)	No. Trans¬formants Tested	Relative [B2]	Relative [B1]	Relative Total Avermectins	Avg. B2:B1 Ratio<br>
1100-SC38	6	155	4.8	176	33.9<br>
1100-SC38 (PSE119)	9	239	50.3	357	4.7<br><br>
S. avermitiiis strain<br>
(transforming plasmid)	No. Trans-<br>
forrnants<br>
Tested	Relative [B2]	Relative [B1]	Relative Total Avermectins	Avg. B2:B1 Ratio<br>
1100-SC38 (pSE189)	16	546	166	849	3.3<br>
wild type	6	59	42	113	1.41<br>
wild type (PSE119)	6	248	151	481	1.64<br>
wild type (PSE189)	5	545	345	1,071	1.58<br>
12. EXAMPLE: CHIMERIC PLASMID CONTAINING<br>
	SEQUENCES FROM BOTH S. AVERMITILIS aveC<br>
QRF AND S. HYGROSCOPICUS aveC HOMOLOG<br>
A hybrid plasmid designated as pSE350 was constructed that contains a 564 bp<br>
portion of the S. hygroscopicus aveC homolog replacing a 564 bp homologous portion of the S. avermitiiis aveC ORF (FIGURE 7), as follows.   pSE350 was constructed using a BsaAl i     restriction site that is conserved in both sequences (aveC position 225), and a Kpnl restriction site that is present in the S- avermitiiis aveC gene (aveC position 810).  The Kpnl site was introduced  into the  S.  hygroscopicus DNA by PCR using the  rightward  primer 5'- CTTCAGGTGTACGTGTTCG-3'    (SEQ    ID    NO:23)    and    the    leftward    primer    5- GAACTGGTACCAGTGCCC-3' (SEQ ID NO:24) (supplied by Genosys Biotechnologies) using PCR conditions described in Section 7.1.10, above.   The PCR product was digested with BsaAl and Kpnl, the fragments were separated by electrophoresis in a 1% agarose gel, and the 564 bp BsaAUKpnl fragment was isolated from the gel. pSE179 (described in Section 7.1.10,   above)  was   digested   with   Kpnl   and   HindlU,   the   fragments   separated   by electrophoresis in a 1% agarose gel, and a fragment of -4.5 Kb was isolated from the gel. pSE179 was digested with Hindlll and BsaAl, the fragments separated by electrophoresis in a<br>
1% agarose gel, and a -0.2 Kb BsaAI/H/ndlll fragment isolated from the gel.  The -4.5 Kb<br>
H/ndlll/Kpnl fragment, the -0.2 Kb BsaAI/W/ndlll fragment, and the 564 bp BsaAUKpnl<br>
fragment from S. hygroscopicus were ligated together in a 3-way ligation and the ligation<br>
mixture transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from<br>
ampicillin resistant transforniants and the presence of the correct insert was confirmed by<br>
restriction analysis using Kpnl and Aval. This plasmid was digested with Hindlll and Xoal to<br><br>
release the 1.2 Kb insert, which was then ligated with pWHM3 which had been digested with Hindlll and Xoal. The ligation mixture was transformed into competent £. coli DH5a cells, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis using H/ndlll and Aval. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid was designated as pSE350 and used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. Results show that transformants containing the S. avermitilis/S. hygroscopicus hybrid plasmid have an average B2:B1 ratio of about 109:1 (TABLE 7).<br>
TABLE 7<br><br>
S. avermitilis strain (transforming plasmid)	No. transformants tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	8	0	0	0<br>
SE180-11(pWHM3)	8	0	0	0<br>
SE180-11 (pSE350)	16	233	2	109<br>
DEPOSIT OF BIOLOGICAL MATERIALS<br>
The following biological material was deposited with the American Type Culture Collection (ATCC) at 12301 Parklawn Drive, Rockville, MD, 20852, USA, on January 29, 1998, and was assigned the following accession numbers:<br>
Plasmid	Accession No.<br>
plasmid pSE 180	209605<br>
plasmid pSE186	209604<br>
All patents, patent applications, and publications cited above are incorporated herein by reference in their entirety.<br>
The present invention is not to be limited in scope by the specific embodiment described herein, which are intended as single illustrations of individual aspects of the<br><br>
invention, and functionally equivalent methods and components are within the scope of the invention. Indeed, various modifications of the invention, in addition to those shown and described herein will become apparent to those skilled in the art .from the foregoing description and accompanying drawings. Such modifications are intended to fall within the scope of the appended claims.<br>
Reference   to   co-pending   application   IP:   732/MUM/2000   is   made therewith.<br><br>
WE CLAIM:<br>
1.        A polynucleotide molecule comprising a nucleotide  sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avernitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:l), or a degenerate variant thereof, but which nucleotide sequence      comprises   one   or   more   mutations   encoding   an   amino   acid substitution at one or more amino acid residues corresponding to amino acid .position 38, 48, 89, 99,111,' 136,154, 179, 228, 238, 289, or 298 of SEQ ID No:2, such that cells of S. avernitilis strain ATCC 53692 in which the wild-type aLyeG allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide  sequence produce  a class 2:1  ratio of avermectins that is different from the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele.<br>
2.	The polynucleotide molecule as claimed in claim 1, wherein the class 2:1 avermectins are cyclohexyl B2:'cyclohexyl Bl avermectins.<br>
3.	The polynucleotide molecule as claimed in claim 2, wherein the different class 2:1 ratio of avermectins is a reduced ratio compared to the class 2:1 ratio produced by cells of S. avermililis strain ATCC 53692 that instead express only the wild-type aveC allele.<br><br>
4.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.8:1 or less.<br>
5.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.68:1 or less.<br>
6.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.53:1 or less.<br><br>
7.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 arermectins is about 0.42:1 or less.<br>
8.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.40:1 or less.<br>
9.	The polynucleotide molecule as claimed in claim 1, wherein the<br>
nucleotide sequence comprises one or more mutations encoding an amino acid<br>
substitution at one or both amino acid residues corresponding to amino acid<br>
positions 138 and 139 of SEQ ID NO:2.<br>
10.	The polynucleotide molecule as claimed in claim 2, wherein the amino<br>
acid substitutions are selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino acid<br>
that is a conservative substitution for P;<br>
(b)	amino' acid residue D at position 48 replaced by E or by an amino acid<br>
that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(g)	amino acid residue K at position 154 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(h)       amino acid residue G at position 179 replaced by S or by an amino acid<br>
that is a conservative substitution for S;<br>
(i)        amino acid residue M at position 228 replaced by T or by an amino acid<br>
that is a conservative substitution for T;<br>
(j)	amino acid residue E at position 238 replaced by D or by an amino acid<br>
that is a conservative substitution for D;<br><br>
(k) amino acid residue P at position 289 replaced by L or by an amino acid that is a conservative substitution for L; and<br>
(1) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H.<br>
11. The polynucleotide molecule as claimed in claim 2, wherein a combination of amino acid residues is mutated, and wherein the combination is selected from one or more of the group consisting of:<br>
(a)	amino acid residues D48 and A89;<br>
(b)	amino acid residues Q38, L136 and E238;<br>
(c)	amino acid residues Gill and P289.<br>
12.	The polynucleotide molecule as claimed in claim 11, wherein the<br>
combination of amino acid substitutions is selected from one or more of<br>
the group consisting of:<br>
(a)	D48E/A89T;<br>
(b)	Q38P/L136P/E238D;<br>
(c)	G1llV/P289L, and<br><br>
13.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding D48E/A89T comprise a base change from T to A at a nucleotide position corresponding to nt 317 of SEQ ID NO: 1, and a base change from G to A at a nucleotide position corresponding to nt 438 of SEQ ID NO: 1.<br>
14.	The polynucleotide molecule as claimed in claim 13, comprising a base change from C to A at a nucleotide position corresponding to nt 353 of SEQ ID.NO: 1, and a base change from T to A at a nucleotide position corresponding to nt 1155 of SEQ ID NO: 1.<br>
15.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding Q38P/L136P/E238D comprise a base change from A to C at a nucleotide position corresponding to nt 285 of SEQ ID NO; 1, a base change from T to C at a nucleotide position corresponding to nt 580 of SEQ ID NO: 1, and a base change from A to T at a nucleotide position corresponding to nt 886 of SEQ ID NO: 1.<br>
16.	The polynucleotide molecule as claimed in claim 15, comprising a<br><br>
base change from A to C at a nucleotide position corresponding to nt 24 of SEQ ID NO: 1, a base change from T to C at a nucleotide position corresponding to nt 497 of SEQ ID NQ: 1, and a base change from C to T at a nucleotide position corresponding to nt 554 of SEQ ID NO: 1.<br>
17.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding G1l1V/P289L comprise a base change from G to T at a nucleotide position corresponding to nt 505 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position corresponding to nt 1039 of SEQ ID NO:l.<br>
18.	The polynucleotide molecule as claimed in claim 22, comprising a base change from T to C at a nucleotide position corresponding to nt 155 of SEQ ID NO: 1, a base change from C to T at a nucleotide position corresponding to nt 1202 of SEQ ID NO: 1, and a base change from T to C at a nucleotide position corresponding to nt 1210 of SEQ ID NO:l.<br>
19.	A recombinant vector comprising the polynucleotide molecule as claimed in claim 1.<br><br>
20.	A method for making a novel strain of S. avermitilis, comprising mutating the aveC allele in cells of a strain of S. avemitilis, which mutation results in the substitution in the AveC gene product of a different amino acid residue at one or more amino acid positions corresponding to amino acid residues 38, 48, 89, 99, 111, 136, 154, 179, 228, 238, 289 or 298 of SEQ ID NO 2, such that cells of the S. avermitilis strain in which the aveC allele has been so mutated produce a class 2:1 ratio of avermectins that is different from the ratio produced by cells of the same S. avermitilis strain that instead express only the wild-type aveC allele.<br>
21.	The   method   as   claimed   in   claim   20,   wherein  the   class   2:1 avermectins are cyclohexyl B2: cyclohexyl B 1 avermectins.<br>
22.	The method as claimed in claim 21, wherein the different class 2:1 ratio of avermectins is a reduced ratio.<br>
23.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the avec allele has been so mutated is about 0.8:1 or less.<br>
24.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC<br><br>
allele has been so mutated is about 0.68:1 or less.<br>
25.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC allele has been so mutated is about 0.53:1 or less.<br>
26.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC allele has been, so mutated is about 0.42:1 or less.<br>
27.	The method as claimed in claim 22, wherein the ratio of class 2:1<br>
avermectins produced by cells of the S. avermitilis strain in which the<br>
aveC allele has been so mutated is about 0.40:1 or less.<br>
28.	The method as claimed in claim 21, wherein the amino acid<br>
substitutions are selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino' acid . that is a conservative substitution for P;<br>
(b)	amino acid residue D at position 48 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid<br>
that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(g)	amino acid residue K at position 154 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(h) amino acid residue G at position 179 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(i) amino acid residue M at position 228 replaced by T or by an amino acid that is a conservative substitution for T;<br><br>
(j) amino acid residue E at position 238 replaced by D or by an amino acid that is a conservative substitution for D;<br>
(k) amino acid residue P at position 289 replaced by L or by an amino acid that is a conservative substitution for L; and<br>
(1) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H.<br>
29.	The method as claimed in claim 21, wherein the aveC allele is<br>
mutated to encode amino acid substitutions at a combination of amino<br>
acid positions, and wherein the combination is selected from one or more<br>
of the group consisting of:<br>
(a)	amino	acid	residues	D48	and	A89;<br>
(b)	amino acid residues Q38, L136 and E238;<br>
(c)	amino acid residues G1ll and P289.<br>
30.	The method as claimed in claim 29, wherein the combination of<br>
amino acid substitutions is selected from one or more of the group<br>
consisting of:<br>
(a)	D48E/A89T;<br>
(b)	Q38P/L136P/E238D;<br>
(c)	G111V/P289L.<br><br>
31.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding D48E/A89T comprise a base change from T to A at a nucleotide position in the aveC allele corresponding to nt 317 of SEQ ID NO: 1, and a base change from G to A at a nucleotide position in the aveC allele corresponding to nt 438 of SEQ ID NO: 1.<br>
32.	The method as claimed in claim 31 comprising introducing a base change from C to A at a nucleotide position in the aveC allele corresponding to nt 353 of SEQ ID NO: 1, and a base change from T to A at a nucleotide position in the aveC allele corresponding to nt 1155 of SEQ	ID	NO:	1.<br>
33.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding Q38P/L136P/E238D comprise a base change from<br><br>
A to C at a nucleotide position in the aveC allele corresponding to nt 286 of SEQ ID NO:l, a base change from T to C at a nucleotide position in the aveG allele corresponding to nt 580 of SEQ ID NQ: 1, and a base change from A to T at a nucleotide position in the aveC allele corresponding to nt 886 of SEQ ID NO: 1.<br>
34.	The method as claimed in claim 33, comprising introducing a base change from A to G at a nucleotide position in the aveC allele corresponding to nt 24 of SEQ ID NO:l, a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 497 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position in the avec allele corresponding to 554 of SEQ ID NO:l.<br>
35.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding Gl11V/P289L comprise a base change from G to T at	a nucleotide position in the aveC allele corresponding to nt 505 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position in the aveC  allele corresponding to nt 1039 of SEQ ID NO: 1.<br>
36.	The method as claimed in claim 35, comprising introducing a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 155 of SEQ ID NO: 1, a base change from C to T at a nucleotide position in the avec allele corresponding to nt 1202 of SEQ ID NO: 1. and a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 1210 of SEQ ID NO: 1.<br>
37.	A process for producing avermectins, comprising culturing the cells of S. Avermitilis in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture.<br>
Dated this January 30, 2002.	 <br>
[DEEPAK KUMAR]<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPLICANT[S]<br>
FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
    [See Section 10, Rule 13]<br>
"STREPTOMYCES AVERMITILIS GENE DIRECTING THE RATIO OF B2:B1<br>
AYERMECTINS".<br>
PFIZER PRODUCTS, INC., A Corporation Organized Under The Laws Of The State Of Connecticut, United States Of America, Of Eastern Point Road, Groton, Connecticut 06340, United States Of America,<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed:-<br><br>
The present invention relates to a polynucleotide molecule<br>
1. FIELD OF THE INVENTION<br>
The present invention is directed to compositions and methods for producing avermectins, and is primarily in the field of animal health. More particularly, the present invention relates to polynucleotide molecules comprising nucleotide sequences encoding an AveC gene product, which can be used to modulate the ratio of class 2:1 avermectins produced by fermentation of cultures of Streptomyces avermitilis, and to compositions and methods for screening for such polynucleotide molecules. The present invention further relates to vectors, transformed host cells, and novel mutant strains of S. avermitilis in which the aveC gene has been mutated so as to modulate the ratio of class 2:1 avermectins produced.<br>
2. BACKGROUND OF THE INVENTION 2.1. Avermectins Streptomyces species produce a wide variety of secondary metabolites, including the avermectins, which comprise a series of eight related sixteen-membered macrocyclic lactones having potent anthelmintic and insecticidal activity.   The eight distinct but closely related compounds are referred to as A1a, A1b, A2a, A2b, B1a, B1b, B2a and B2b. The "a" series of compounds refers to the natural avermectin where the substituent at the C25 position is (S&gt;-sec-butyl, and the "b" series refers to those compounds where the substituent at the C25 position is isopropyl. The designations "A" and "B" refer to avermectins where the substituent at the C5 position is methoxy and hydroxy, respectively.    The numeral "1" refers to avermectins where a double bond is present at the C22.23 position, and the numeral "2" refers to avermectins having a hydrogen at the C22 position and a hydroxy at the C23 position. Among the related avermectins, the B1 type of avermectin is recognized as having the most effective antiparasitic and pesticidal activity, and is therefore the most commercially desirable avermectin.<br>
The avermectins and their production by aerobic fermentation of strains of S. avermitilis are described in United States Patents 4,310,519 and 4,429,042. The biosynthesis of natural avermectins is believed to be initiated endogenously from the CpA thioester analogs of isobutyric acid and S-(+)-2-methyl butyric acid.<br>
A combination of both strain improvement through random mutagenesis and the use of exogenously supplied fatty acids has led to the efficient production of avermectin analogs. Mutants of S. avermitilis that are deficient in branched-chain 2-oxo acid dehydrogenase (bkd<br><br>
deficient'mutants) can only produce avermectins when fermentations are supplemented with fatty acids. Screening and isolation of mutants deficient in branched-chain dehydrogenase activity (e.g., S. avermitilis, ATCC 53567) are described in European Patent (EP) 276103. Fermentation of such mutants in the presence of exogenously supplied fatty acids results in production of only the four avermectins corresponding to the fatty acid employed. Thus, supplementing fermentations of S. avermitilis (ATCC 53567) with S-(+)-2-methylbutyric acid results in production of the natural avermectins A1a, A2a, B1a and B2a; supplementing fermentations with isobutyric acid results in production of the natural avermectins A1 b, A2b, B1b, and B2b; and supplementing fermentations with cyclopentanecarboxylic acid results in the production of the four novel cyclopentylavemnectins A1, A2, B1, and B2.<br>
If supplemented with other fatty acids, novel avermectins are produced. By screening over 800 potential precursors, more than 60 other novel avermectins have been identified. (See, e.g., Dutton et al, 1991, J. Antibiot 44:357-365; and Banks et a/., 1994, Roy. Soc. Chem. 147:16-26). In addition, mutants of S. avermitilis deficient in 5-O-methyltransferase activity produce essentially only the B analog avermectins. Consequently, S. avermitilis mutants lacking both branched-chain 2-oxo acid dehydrogenase and 5-O-methyltransferase activity produce only the B avermectins corresponding to the fatty acid employed to supplement the fermentation. Thus, supplementing such double mutants with S-(+)-2-methylbutyric acid results in production of only the natural avermectins B1a and B2a, while supplementing with isobutyric acid or cyclopentanecarboxylic acid results in production of the natural avermectins B1b and B2b or the novel cyclopentyl B1 and B2 avermectins, respectively. Supplementation of the double mutant strain with cyclohexane carboxylic acid is a preferred method for producing the commercially important novel avermectin, cyclohexylavermectin B1 (doramectin). The isolation and characteristics of such double mutants, e.g., S. avermitilis (ATCC 53692), is described in EP 276103.<br>
2.2. Genes Involved In Avermectin Biosynthesis In many cases, genes involved in production of secondary metabolites and genes encoding a particular antibiotic are found clustered together on the chromosome. Such is the  case, e.g., with the Streptomyces polyketide synthase gene cluster (PKS) (see Hopwood and Sherman, 1990, Ann. Rev. Genet. 24:37-66). Thus, one strategy for cloning genes in a  piosynthetic pathway has been to isolate a drug resistance gene and then test adjacent egions of the chromosome for other genes related to the biosynthesis of that particular intibiotic. Another strategy for cloning genes involved in the biosynthesis of important netabolites has been complementation of mutants.   For example, portions of a DNA library<br><br>
from an organism capable of producing a particular metabolite are introduced into a non-producing mutant and transformants screened for production of the metabolite. Additionally, hybridization of a library using probes derived from other Streptomyces species has been used to identify and clone genes in biosynthetic pathways.<br>
Genes involved in avermectin biosynthesis (ave genes), like the genes required for biosynthesis of other Streptomyces secondary metabolites (e.g., PKS), are found clustered on the chromosome. A number of ave genes have been successfully cloned using vectors to complement S. avermitilis mutants blocked in avermectin biosynthesis. The cloning of such genes is described in U.S. Patent 5,252,474. In addition, Ikeda et al., 1995, J. Antibiot. 48:532-534, describes the localization of a chromosomal region involving the C22.23 dehydration step (aveC) to a 4.82 Kb BamHI fragment of S. avermitilis, as well as mutations in the aveC gene that result in the production of a single component B2a producer. Since ivermectin, a potent anthelmintic compound, can be produced chemically from avermectin B2a, such a single component producer of avermectin B2a is considered particularly useful for commercial production of ivermectin.<br>
Identification of mutations in the aveC gene that minimize the complexity of avermectin production, such as, e.g., mutations that decrease the B2:B1 ratio of avermectins, would simplify production and purification of commercially important avermectins.<br>
3. SUMMARY OF THE INVENTION The present invention provides an isolated polynucleotide molecule comprising the complete aveC ORF of S. avermitilis or a substantial portion thereof, which isolated polynucleotide molecule lacks the next complete ORF that is located downstream from the aveC ORF in situ in the S. avermitilis chromosome. The isolated polynucleotide molecule of the present invention preferably comprises a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC2096D4), or that is the same as the nucleotide sequence of the aveC ORF of FIGURE 1 (SEQ ID NO:1), or substantial portion thereof. The present invention further provides an isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:1 or a degenerate variant thereof.<br>
The present invention further provides an isolated polynucleotide molecule having a nucleotide sequence that is homologous to the S. avermitilis AveC gene product-encoding sequence of pJasmid pSE186 (ATCC 2£96Q4)or to the nucJeotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof.<br><br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide having an amino acid sequence that is homologous to the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or substantial portion thereof.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence encoding an AveC homolog gene product. In a preferred embodiment, the isolated polynucleotide molecule comprises a nucleotide sequence encoding the AveC homolog gene product from S. hygroscopicus, which homolog gene product comprises the amino acid sequence of SEQ ID NO:4 or a substantial portion thereof. In a preferred embodiment, the isolated polynucleotide molecule of the present invention that encodes the S. hygroscopicus AveC homolog gene product comprises the nucleotide sequence of SEQ ID NO:3 or a substantial portion thereof.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus nucleotide sequence of SEQ ID NO:3. The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide that is homologous to the S. hygroscopicus AveC homolog gene product having the amino acid sequence of SEQ ID NO:4. The present invention further provides oligonucleotides that hybridize to a polynucleotide molecule having the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3, or to a polynucleotide molecule having a nucleotide sequence which is the complement of the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3.<br>
The present invention further provides recombinant cloning vectors and expression vectors that are useful in cloning or expressing a polynucleotide of the present invention including polynucleotide molecules comprising the aveC ORF of S. avermitilis or an aveC homolog ORF. In a non-limiting embodiment, the present invention provides plasmid pSE186 (ATCC 209604), which comprises the entire ORF of the aveC gene of S. avermitilis. The present invention further provides transformed host cells comprising a polynucleotide molecule or recombinant vector of the invention, and novel strains or cell lines derived therefrom.<br>
The present invention further provides a recombinantly expressed AveC gene product or AveC homplog gene product, or a substantial portion thereof, that has been substantially purified or isolated, as well as homologs thereof. The present invention further provides a method for producing a recombinant AveC gene product, comprising culturing a host cell<br><br>
transformed with a recombinant expression vector, said recombinant expression vector comprising a polynucleotide molecule having a nucleotide sequence encoding an AveC gene product or AveC homolog gene product, which polynucleotide molecule is in operative association with one or more regulatory elements that control expression of the polynucleotide molecule in the host cell, under conditions conducive to the production of the recombinant AveC gene product or AveC homolog gene product, and recovering the AveC gene product or AveC homolog gene product from the cell culture.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the S. avermitilis AveC allele, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprises one or more mutations, so that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence produce a different ratio or amount of avermectins than are produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele. According to the present invention, such polynucleotide molecules can be used to produce novel strains of 5. avermitilis that exhibit a detectable change in avenmectin production compared to the same strain that instead expresses only the wild-type aveC allele. In a preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce avermectins in a reduced class 2:1 ratio compared to that from the same strain that instead expresses only the wild-type aveC allele.  In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce increased levels of avermectins compared to the same strain that instead expresses only a single wild-type aveC allele.   In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis in which the aveC gene has been inactivated.<br>
The present invention provides methods for identifying mutations of the aveC ORF of S. avermitilis capable of altering the ratio and/or amount of avermectins produced. In a preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF capable of altering the class 2:1 ratio of avermectins produced, comprising: (a) determining the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product has been<br><br>
introduced and is being expressed; (b) determining the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only the wild-type aveC allele or the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) to the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b); such that if the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) is different from the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b), then a mutation of the aveC ORF capable of altering the class 2:1 ratio of avermectins has been identified. In a preferred embodiment, the class 2:1 ratio of avermectins is reduced by the mutation.<br>
In a further preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF or genetic constructs comprising the aveC ORF capable of altering the amount of avermectins produced, comprising: (a) determining the amount of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product or comprising a genetic construct comprising a nucleotide sequence encoding an AveC gene product has been introduced and is being expressed; (b) determining the amount of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a single aveC allele having the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the amount of avermectins produced by the S. avermitilis cells of step (a) to the amount of avermectins produced by the S. avermitilis cells of step (b); such that if the amount of avermectins produced by the S. avermitilis cells of step (a) is different from the amount of avermectins produced by. the S. avermitilis cells of step (b), then a mutation of the aveC ORF or a genetic construct capable of altering the amount of avermectins has been identified. In a preferred embodiment, the amount of avermectins produced is increased by the mutation.<br>
The present invention further provides recombinant vectors that are useful for making novel strains of S. avermitilis having altered avermectin production. For example, the present invention provides vectors that can be used to target any of the polynucleotide molecules comprising the mutated nucleotide sequences of the present invention .to the site of the aveC gene of the S. avermitilis chromosome to either insert into or replace the aveC allele or ORF or a portion thereof by homologous recombination. According to the present invention, however, a polynucleotide molecule comprising a mutated nucleotide sequence of the present<br><br>
invention provided herewith can also function to modulate avermectin biosynthesis when<br>
inserted into the S. avermitilis chromosome at a site other than at the aveC gene, or when<br>
maintained episomally in S. avermitilis cells.   Thus, the present invention also provides<br>
vectors comprising a polynucleotide molecule comprising a mutated nucleotide sequence of       the present invention, which vectors can be used to insert the polynucleotide molecule at a site in the S. avermitilis chromosdme other than at the aveC gene, or to be maintained episomally.   In a preferred embodiment, the present invention provides gene replacement vectors that can be used to insert a mutated aveC allele into the S. avermitilis chromosome to generate novel strains of cells that produce avermectins in a reduced class 2:1  ratio       compared to the cells of the same strain which instead express only the wild-type aveC allele. The present invention further provides methods for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce altered ratios and/or amounts of avermectins compared to cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele.   In a preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce an altered class 2:1 ratio of avermectins compared to cells of the same strain of S. avermitilis that instead express only a wild-type aveC allele, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele that encodes a gene product that alters the class 2:1  ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated allelecompared to cells of the same strain that instead express only the wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered class 2:1 ratio compared to the class 2:1 ratio produced by cells of the strain that instead express the wild-type aveC<br>
allele. In a preferred embodiment, the class 2:1 ratio of avermectins produced is reduced in<br>
cells of the novel strain.	 <br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that produce altered amounts of avermectin, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele or a genetic construct comprising the aveC allele, the expression of which results in an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct as compared to cells of the same strain that instead express only the wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered amount compared to the amount of avermectins produced by cells of the strain that instead express only the wild-type aveC allele.   In a<br><br>
preferred embodiment, the amount of avermectins produced is increased in cells of the novel strain.<br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis, the cells of which comprise an inactivated aveC allele, comprising transforming cells of a strain of S. avermitilis that express any aveC allele with a vector that inactivates the aveC allele, and selecting transformed cells in which the aveC allele has been inactivated.<br>
The present invention further provides novel strains of S. avermitilis comprising cells that have been transformed with any of the polynucleotide molecules or vectors comprising a mutated nucleotide sequence of the present invention. In a preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain produce avermectins in an altered class 2:1 ratio compared to cells of the same strain that instead express only the wild-type aveC allele. In a more preferred embodiment, the cells of the novel strain produce avermectins in a reduced class 2:1 ratio compared to cells of the same strain that instead express only the wild-type aveC allele. Such novel strains are useful in the large-scale production of commercially desirable avermectins such as doramectin.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele, or a genetic construct comprising the aveC allele, in place of, or in addition to, the aveC allele native thereto, which results in the production by the cells of an altered amount of avermectins compared to the amount of avermectins produced by cells of the same strain that instead express only the wild-type aveC allele. In a preferred embodiment, the novel cells produce an increased amount of avermectins.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells in which the aveC gene has been inactivated.  Such strains are useful both for the different spectrum of avermectins that they produce compared to the wild-type strain, and in complementation screening assays as described herein, to determine whether targeted or random mutagenesis of the aveC gene affects avermectin production.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele compared to cells of the<br><br>
same strain which do not express the mutated aveC allele but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the class 2:1 ratio of avermectins produced by cells expressing the mutation is reduced. This process provides increased efficiency in the production of commercially valuable avermectins such as doramectin.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele or a genetic construct comprising an aveC allele that results in the production of an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct compared to cells of the same strain which do not express the mutated aveC allele or genetic construct but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the amount of avermectins produced by cells expressing the mutation or genetic construct is increased.<br>
The present invention further provides a novel composition of avermectins produced by a strain of S. avermitilis expressing a mutated aveC allele of the present invention, wherein the avermectins are produced in a reduced class 2:1 ratio as compared to the class 2:1 ratio of avermectins produced by cells of the same strain of S. .avermitilis that do not express the mutated aveC allele 6ut instead express only the wild-type aveC allele. The novel avermectin<br>
r<br>
composition can be present as produced in fermentation culture fluid, or can be harvested therefrom, and can be partially or substantially purified therefrom.<br>
4. BRIEF DESCRIPTION OF THE FIGURES<br>
FIGURE 1. DNA sequence (SEQ ID NO:1) comprising the S. avermitilis aveC ORF, and deduced amino acid sequence (SEQ ID NO;2).<br>
FIGURE 2. Plasmid vector pSE186 (ATCC 209604) comprising the entire ORF of the aveC gene of S. avermitilis.<br>
FIGURE 3. Gene replacement vector pSE180 (ATCC 209605) comprising the ermE gene of Sacc. erythraea inserted into the aveC ORF of S. avermitilis.<br>
FIGURE 4. BamHl restriction map of the avermectin polyketide synthase gene cluster from S. avermitilis with five overlapping cosmid clones identified (i.e., pSE65, pSE66, pSE67, pSE68, pSE69). The relationship of pSE118 and pSE119 is also indicated.<br><br>
FIGURE 5.   HPLC analysis of fermentation products produced by S. avermitilis strains. Peak quantitation was performed by comparison to standard quantities of cyclohexyl B1. Cyclohexyl B2 retention time was 7.4-7.7 min; cyclohexyl B1 retention time was 11.9-12.3! min.   FIG. 5A.   S. avermitilis strain SE180-11 with an inactivated aveC ORF.   FIG. 5B.   S.       avermitilis strain SE180-11 transformed with pSE186 (ATCC 209604). FIG. 5C.  S. avermitilis strain SE180-11 transformed with pSE187.    FtG. 5D.    S. avermitilis strain SE180-11 transformed with pSE188.<br>
FIGURE 6.   Comparison of deduced amino acid sequences encoded by the aveC ORF   of   S.   avermitilis   (SEQ   ID   NO:2),   an   aveC   homolog   partial   ORF   from   S.       griseochromogenes (SEQ ID NO:5), and the aveC homolog ORF from S. hygroscopicus (SEQ ID NO:4). The valine residue in bold is the putative start site for the protein. Conserved residues are shown in capital letters for homology in all three sequences and in lower case letters for homology in 2 of the 3 sequences.     The amino acid sequences  contain<br>
approximately 50% sequence identity. FIGURE 7.  Hybrid plasmid construct containing a 564 bp BsaA\/Kpn\ fragment from the S. hygroscopicus aveC homolog gene inserted into the BsaAUKpn) site in the S. avermitilis aveC ORF.<br>
5. DETAILED DESCRIPTION OF THE INVENTION<br>
The   present   invention   relates   to   the   identification   and   characterization   of polynucleotide molecules having nucleotide sequences that encode the AveC gene product from Streptomyces avermitilis, the construction of novel strains of S. avermitilis that can be used to screen mutated AveC gene products for their effect on avermectin production, and the discovery that certain mutated AveC gene products can reduce the ratio of B2:B1 avermectins produced by S. avermitilis.   By way of example, the invention is described in the sections<br>
below for a polynucleotide molecule having either a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1), and for polynucleotides molecules having mutated nucleotide sequences derived therefrom and degenerate variants thereof. However, the principles set forth in the present invention can be analogously applied to other polynucleotide molecules, including aveC homolog genes from other Streptomyces species including, e.g., S. hygroscopicus and S. griseochromogenes, among others.<br>
5.1.       Polynucleotide Molecules Encoding The S. avermitilis AveC Gene Product<br>
The present invention provides an isolated polynucleotide molecule comprising the<br>
complete aveC ORF of S. avermitilis or a substantial portion thereof, which  isolated<br><br>
polynucleotide molecule lacks the next complete ORF that is located downstream from the aveC ORF in situ in the S. avermitilis chromosome.<br>
The isolated polynucleotide molecule of the present invention preferably comprises a nucleotide sequence that is the same as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that is the same as the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or substantial portion thereof. As used herein, a "substantial portion" of an isolated polynucleotide molecule comprising a nucleotide sequence encoding the S. avermitilis AveC gene product means an isolated polynucleotide molecule comprising at least about 70% of the complete aveC ORF sequence shown in FIGURE 1 (SEQ ID NO:1), and that encodes a functionally equivalent AveC gene product. In this regard, a "functionally equivalent" AveC gene product is defined as a gene product that, when expressed in S. avermitilis strain ATCC 53692 in which the native aveC allele has been inactivated, results in the production of substantially the same ratio and amount of avermectins as produced by S. avermitilis strain ATCC 53692 which instead expresses only      the wild-type, functional aveC allele native to S. avermitilis strain ATCC 53692.<br>
In addition to the nucleotide sequence of the aveC ORF, the isolated polynucleotide molecule of the present invention can further comprise nucleotide sequences that naturally flank the aveC gene in situ in S. avermitilis, such as those flanking nucleotide sequences shown in FIGURE 1 (SEQ ID NO:1).<br>
The present invention further provides an isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:1 or a degenerate variant thereof.<br>
As used herein, the terms "polynucleotide molecule," "polynucleotide sequence," "coding sequence," "open-reading frame," and "ORP are intended to refer to both DNA and RNA molecules, which can either be single-stranded or double-stranded, and that can be transcribed and translated (DNA), or translated (RNA), into an AveC gene product or, as described below, into an AveC homolog gene product, or into a polypeptide that is homologous to an AveC gene product or AveC homolog gene product in an appropriate host cell expression system when placed under the control of appropriate regulatory elements. A coding sequence can include but is not limited to prokaryotic sequences, cDNA sequences, genomic DNA sequences, and chemically synthesized DNA and RNA sequences.<br>
The nucleotide sequence shown in FIGURE 1 (SEQ ID NO:1) comprises four different GTG codons at bp positions 42,174,177 and 180. As described in Section 9 below, multiple deletions of the 5' region of the aveC ORF (FIGURE 1; SEQ ID NO:1) were constructed to help define which of these codons could function in the aveC ORF as start sites for protein<br><br>
expression. Deletion of the first GTG site at bp 42 did not eliminate AveC activity. Additional deletion of all of the GTG codons at bp positions 174, 177 and 180 together eliminated AveC activity, indicating that this region is necessary for protein expression. The present invention thus encompasses variable length aveC ORFs.<br>
The present invention further provides a polynucleotide molecule having a nucleotide sequence that is homologous to the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or to the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof.   The term "homologous" when used to refer to a polynucleotide molecule that is homologous to an S. avermitilis AveC gene product-encoding sequence means a polynucleotide molecule having a nucleotide sequence: (a) that encodes the same AveC gene product as the S. avermitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or that encodes the same AveC gene product as the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1), but that includes one or more silent changes to the nucleotide sequence according to the degeneracy of the genetic code (i.e., a degenerate variant); or (b) that hybridizes to the complement of a polynucleotide molecule having a nucleotide sequence that encodes the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or that encodes the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2) under moderately stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7%  sodium dodecyl sulfate (SDS),  1   mM  EDTA at 65°C,  and washing  in 0.2xSSC/0.1% SDS at 42°C (see Ausubel et al. (eds.), 1989, Current Protocols in Molecular Biology, Vol. I, Green Publishing Associates, Inc., and John Wiley &amp; Sons, Inc., New York, at p. 2.10.3), and encodes a functionally equivalent AveC gene product as defined above.  In a preferred   embodiment,   the   homologous   polynucleotide   molecule   hybridizes   to   the complement of the AveC gene product-encoding nucleotide sequence of plasmid pSE186 (ATCC 209604) or to the complement of the nucleotide sequence of the aveC ORF presented in FIGURE 1 (SEQ ID NO:1) or substantial portion thereof under highly stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04. 7% SDS, 1 mM EDTA at 65°C, and washing in 0.1xSSC/0.1% SDS at 68°C (Ausubel et al., 1989, above), and encodes a functionally equivalent AveC gene product as defined above.<br>
The activity of an AveC gene product and potential functional equivalents thereof can be determined through HPLC analysis of fermentation products, as described in the examples below. Polynucleotide molecules having nucleotide sequences that encode functional equivalents of the S. avermitilis AveC gene product include naturally occurring aveC genes<br><br>
present in other strains of S. avermitilis, aveC homolog genes present in other species of Streptomyces, and mutated aveC alleles, whether naturally occurring or engineered.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide having an amino acid sequence that is homologous to the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or substantial portion thereof. As used herein, a "substantial portion" of the amino acid sequence of FIGURE 1 (SEQ ID NO:2) means a polypeptide comprising at least about 70% of the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2), and that constitutes a functionally equivalent AveC gene product, as defined above.<br>
As used herein to refer to amino acid sequences that are homologous to the amino acid sequence of an AveC gene product from S. avermitilis, the term "homologous' refers to a polypeptide which otherwise has the amino acid sequence of FIGURE 1 (SEQ ID NO:2), but in which one or more amino acid residues has been conservatively substituted with a different amino acid residue, wherein said amino acid sequence has at least about 70%, more preferably at least about 80%, and most preferably at least about 90% amino acid sequence identity to the polypeptide encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the amino acid sequence of Figure 1  (SEQ ID NO:2) as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm  (GENBANK,  NCBI),  and where such  conservative substitution results  in  a functionally  equivalent  gene  product,   as   defined   above.     Conservative  amino   acid substitutions are well known in the art.   Rules for making such substitutions include those described by Dayhof, M.D., 1978, Nat. Biomed. Res. Found.. Washington, D.C., Vol. 5, Sup. 3, among others.   More specifically, conservative amino acid substitutions are those that generally take place within a family of amino acids that are related in the acidity or polarity. Genetically encoded amino acids are generally divided into four groups:    (1) acidic = aspartate, glutamate; (2) basic = lysine, arginine, histidine; (3) non-polar = alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar = glycine,   asparagine,   glutamine,   cysteine,   serine,  threonine,  tyrosine.     Phenylalanine,      tryptophan and tyrosine are also jointly classified as aromatic amino acids.   One or more replacements within any particular group, e.g., of a leucine with an isoleucine or valine, or of an aspartate with a glutamate, or of a threonine with a serine, or of any other amino acid . residue with a structurally related amino acid residue, e.g., an amino acid residue with similar<br><br>
acidity or polarity, or with similarity in some combination thereof, will generally have an insignificant effect on the function of the polypeptide.<br>
The present invention further provides an isolated polynucleotide molecule comprising a nucleotide sequence encoding an AveC homolog gene product. As used herein, an "AveC homolog gene product" is defined as a gene product having at least about 50% amino acid sequence identity to an AveC gene product of S. avermitilis comprising the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence shown in FIGURE 1 (SEQ ID NO:2), as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm (GENBANK, NCBI). In a non-limiting embodiment the AveC homolog gene product is from S. hygroscopicus,   (described  in   EP  application  0298423;  deposit  FERM   BP-1901)  and comprises the amino acid sequence of SEQ ID NO:4, or a substantial portion thereof.   A "substantial portion" of the amino acid sequence of SEQ ID NO:4 means a polypeptide comprising at least about 70% of the amino acid sequence of SEQ ID NO:4, and that constitutes a functionally equivalent AveC homolog gene product. A "functionally equivalenf AveC homolog gene product is defined as a gene product that, when expressed in S. hygroscopicus strain FERM BP-1901 in which the native aveC homolog allele has been inactivated, results in the production of substantially the same ratio and amount of milbemycins as produced by S. hygroscopicus strain FERM BP-1901 expressing instead only the wild-type, functional aveC homolog allele native to S. hygroscopicus strain FERM BP-1901.   In a non-limiting embodiment, the isolated polynucleotide molecule of the present invention that encodes the S. hygroscopicus AveC homolog gene product comprises the nucleotide sequence of SEQ ID NO:3 or a substantial portion thereof.   In this regard, a "substantial portion" of the isolated polynucleotide molecule comprising the nucleotide sequence of SEQ ID NO:3 means an isolated polynucleotide molecule comprising at least about 70% of the nucleotide sequence of SEQ ID NO:3, and that encodes a functionally equivalent AveC homolog gene product, as defined immediately above.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus nucleotide sequence of SEQ ID NO:3. The term "homologous" when used to refer to a polynucleotide molecule comprising a nucleotide sequence that is homologous to the S. hygroscopicus AveC homolog gene product-encoding sequence of SEQ ID NO:3 means a polynucleotide molecule having a nucleotide sequence: (a) that encodes the same gene product as the nucleotide sequence of SEQ ID NO:3, but that includes one or more silent changes to the nucleotide sequence<br><br>
according to the degeneracy of the genetic code {i.e., a degenerate variant); or (b) that hybridizes to the complement of a polynucleotide molecule having a nucleotide sequence that encodes the amino acid sequence of SEQ ID NO:4, under moderately stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7% SDS, 1 mM EDTA at 65°C, and washing in 0.2xSSC/0.1% SDS at 42°C (see Ausubel et al. above), and encodes a functionally equivalent AveC homolog gene product as defined above. In a preferred embodiment, the homologous polynucleotide molecule hybridizes to the complement of the AveC homolog gene product-encoding nucleotide sequence of SEQ ID NO:3, under highly stringent conditions, i.e., hybridization to filter-bound DNA in 0.5 M NaHP04, 7% SDS, 1 mM EDTA at 65°C, and washing in 0.1xSSC/0.1% SDS at 68°C (Ausubel et al., 1989, above), and encodes a functionally equivalent AveC homolog gene product as defined above.<br>
The present invention further provides a polynucleotide molecule comprising a nucleotide sequence that encodes a polypeptide that is homologous to the S. hygroscopicus AveC homolog gene product. As used herein to refer to polypeptides that are homologous to the AveC homolog gene product of SEQ ID NO:4 from S. hygroscopicus, the term "homologous" refers to a polypeptide which otherwise has the amino acid sequence of SEQ ID NO:4, but in which one or more amino acid residues has been conservatively substituted with a different amino acid residue as defined above, wherein said amino acid sequence has at least about 70%, more preferably at least about 80%, and most preferably at least about 90% amino acid sequence identity to the polypeptide of SEQ ID NO:4, as determined by any standard amino acid sequence identity algorithm, such as the BLASTP algorithm (GENBANK, NCBI), and where such conservative substitution results in a functionally equivalent AveC homolog gene product, as defined above.<br>
The present invention further provides oligonucleotides that hybridize to a polynucleotide molecule having the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3, or to a polynucleotide molecule having a nucleotide sequence which is the complement of the nucleotide sequence of FIGURE 1 (SEQ ID NO:1) or SEQ ID NO:3. Such oligonucleotides are at least about 10 nucleotides in length, and preferably from about 15 to about 30 nucleotides in length, and hybridize to one of the aforementioned polynucleotide molecules under highly stringent conditions, i.e., washing in 6xSSC/0.5% sodium pyrophosphate at ~37°C for ~14-base oligos, at ~48°C for -17-base oligos, at ~55°C for -20-base oligos, and at ~60°C for -23-base oligos. In a preferred embodiment, the oligonucleotides are complementary to a portion of one of the aforementioned polynucleotide molecules. These oligonucleotides are useful for a variety of purposes including to encode or<br><br>
act as antisense molecules useful in gene regulation, or as primers in amplification of aveC-or aveC homolog-encoding polynucleotide molecules.<br>
Additional aveC homolog genes can be identified in other species or strains of Sireptomyces using the polynucleotide molecules or oligonucleotides disclosed herein in conjunction with known techniques. For example, an oligonucleotide molecule comprising a portion of the S. avermitilis nucleotide sequence of FIGURE 1 (SEQ ID NO.I) or a portion of the S. hygroscopicus nucleotide sequence of SEQ ID NO:3 can be detectably labeled and used to screen a genomic library constructed from DNA derived from the organism of interest. The stringency of the hybridization conditions is selected based on the relationship of the reference organism, in this example S. avermitilis or S. hygroscopicus, to the organism of interest. Requirements for different stringency conditions are well known to those of skill in the art, and such conditions will vary predictably depending on the specific organisms from which the library and the labeled sequences are derived. Such oligonucleotides are preferably at feast about 15 nucleotides in length and include, e.g., those described in the examples below. Amplification of homolog genes can be carried out using these and other oligonucleotides by applying standard techniques such as the polymerase chain reaction (PCR), although other amplification techniques known in the art, e.g., the ligase chain reaction, can also be used.<br>
Clones identified as containing aveC homolog nucleotide sequences can be tested for &gt; their ability to encode a functional AveC homolog gene product. For this purpose, the clones can be subjected to sequence analysis in order to identify a suitable reading frame, as well as initiation and termination signals. Alternatively or additionally, the cloned DNA sequence can be inserted into an appropriate expression vector, i.e., a vector that contains the necessary elements for the transcription and translation of the inserted protein-coding sequence. Any of a variety of host/vector systems can be used as described below, including but not limited to bacterial systems such as plasmid, bacteriophage, or cosmid expression vectors. Appropriate host cells transformed with such vectors comprising potential aveC homolog coding sequences can then be analyzed for AveC-type activity using methods such as HPLC analysis of fermentation products, as described, e.g., in Section 7, below.<br>
Production and manipulation of the polynucleotide molecules disclosed herein are within the skill in the art and can be carried out according to recombinant techniques described, e.g., in Maniatis, etal., 1989, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press. Cold Spring Harbor, NY; Ausubel, et al., 1989, Current Protocols In Molecular Biology, Greene Publishing Associates &amp; Wiley Interscience, NY; Sambrook, ef al.,<br><br>
1989, Molecular Cloning: A Laboratory Manual, 2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY; Innis et al. (eds), 1995, PCR Strategies, Academic Press, Inc., San Diego; and Erlich (ed), 1992, PCR Technology, Oxford University Press, New York, all of which are incorporated herein by reference.   Polynucleotide clones encoding AveC gene products or AveC homolog gene products can be identified using any method known in the art, including but not limited to the methods set forth in Section 7, below.   Genomic DNA libraries can be screened for aveC and aveC homolog coding sequences using techniques such as the methods set forth in Benton and Davis, 1977, Science 196:180, for bacteriophage libraries, and in Grunstein and Hogness, 1975, Proc. Natl. Acad. Sci. USA, 72:3961-3965, for<br>
plasmid libraries.   Polynucleotide molecules having nucleotide sequences known to include the aveC ORF, as present, e.g., in plasmid pSE186 (ATCC 209604), or in plasmid pSE119 (described in Section 7, below), can be used as probes in these screening experiments. Alternatively,. oligonucleotide probes can be synthesized that correspond to nucleotide sequences deduced from partial or complete amino acid sequences of the purified AveC homolog gene product.<br>
5.2. Recombinant Systems 5.2.1. Cloning And Expression Vectors The present invention further provides recombinant cloning vectors and expression vectors which are useful in cloning or expressing polynucleotide molecules of the present invention comprising, e.g., the aveC ORF of S. avermitilis or any aveC homolog ORFs. In a non-limiting embodiment, the present invention provides plasmid pSE186 (ATCC 209604), which comprises the complete ORF of the aveC gene of S. avermitilis.<br>
All of the following description regarding the aveC ORF from S. avermitilis, or a polynucleotide molecule comprising the aveC ORF from S. avermitilis or portion thereof, or an S. avermitilis AveC gene product, also refers to aveC homologs and AveC homolog gene products, unless indicated explicitly or by context.<br>
A variety of different vectors have been developed for specific use in Streptomyces, including phage, high copy number plasmids, low copy number plasmids, and E.coli-Streptomyces shuttle vectors, among others, and any of these can be used to practice the present invention. A number of drug resistance genes have also been cloned from Streptomyces, and several of these genes have been incorporated into vectors as selectable markers. Examples of current vectors for use in Streptomyces are presented, among other places, in Hutchinson, 1980, Applied Biochem. Biotech. 16:169-190.<br><br>
Recombinant vectors of the present invention, particularly expression vectors, are preferably constructed so that the coding sequence for the polynucleotide molecule of the invention is in operative association with one or more regulatory elements necessary for transcription and translation of the coding sequence to produce a polypeptide. As used herein, the term "regulatory element" includes but is not limited to nucleotide sequences that encode inducible and non-inducible promoters, enhancers, operators and other elements known in the art that serve to drive and/or regulate expression of polynucleotide coding sequences. Also, as used herein, the coding sequence is in "operative association" with one or more regulatory elements where the regulatory elements effectively regulate and allow for the transcription of the coding sequence or the translation of its mRNA, or both.<br>
Typical plasmid vectors that can be engineered to contain a polynucleotide molecule of the present invention include pCR-Blunt, pCR2.1 (Invitrogen), pGEM32f (Promega), and the shuttle vector pWHM3 (Vara ef at., 1989, J. Bact. 171:5872-5881), among many others.<br>
Methods are well-known in the art for constructing recombinant vectors containing particular coding sequences in operative association with appropriate regulatory elements, and these can be used to practice the present invention. These methods include in vitro recombinant techniques, synthetic techniques, and in vivo genetic recombination. See, e.g., the techniques described in Maniatis et a/., 1989, above; Ausubel et at., 1989, above; Sambrook etal., 1989, above; Innis et al., 1995, above; and Erlich, 1992, above.<br>
The regulatory elements of these vectors can vary in their strength and specificities. Depending on the host/vector system utilized, any of a number of suitable transcription and translation elements can be used. Non-limiting examples of transcriptional regulatory regions or promoters for bacteria include the β-gal promoter, the T7 promoter, the TAC promoter, A. left and right promoters, tap and lac promoters, trp-lac fusion promoters and, more specifically for Streptomyces, the promoters ermE, meIC, and tipA, etc. In a specific embodiment described in Section 11 below, an expression vector was generated that contained the aveC ORF cloned adjacent to the strong constitutive ermE promoter from Saccharopolyspora erythraea. The vector was transformed into S. avermitilis, and subsequent HPLC analysis of fermentation products indicated an increased titer of avermectins produced compared to production by the same strain but which instead expresses the wild-type aveC allele.<br>
Fusion protein expression vectors can be used to express an AveC gene product-fusion protein. The purified fusion protein can be used to raise antisera against the AveC gene product, to study the biochemical properties of the AveC gene product, to engineer AveC fusion proteins with different biochemical activities, or to aid in the identification or<br><br>
purification of the expressed AveC gene product. Possible fusion protein expression vectors include but are not limited to vectors incorporating sequences that encode p-galactosidase and trpE fusions, maltose-binding protein fusions, glutathione-S-transferase fusions and polyhistidine fusions (carrier regions). In an alternative embodiment, an AveC gene product or a portion thereof can be fused to an AveC homolog gene product, or portion thereof, derived from another species or strain of Streptomyces, such as, e.g., S. hygroscopicus or S. griseochromogenes. In a particular embodiment described in Section 12, below, and depicted in FIGURE 7, a chimeric plasmid was constructed that contains a 564 bp region of the S. hygroscopicus aveC homolog ORF replacing a homologous 564 bp region of the S. avermitilis aveC ORF. Such hybrid vectors can be transformed into S. avermitilis cells and tested to determine their effect, e.g., on the ratio of class 2:1 avermectin produced.<br>
AveC fusion proteins can be engineered to comprise a region useful for purification. For example, AveC-maltose-binding protein fusions can be purified using amylose resin; AveC-glutathione-S-transferase fusion proteins can be purified using glutathione-agarose beads; and AveC-polyhistidine fusions can be purified using divalent nickel resin. Alternatively, antibodies against a carrier protein or peptide can be used for affinity chromatography purification of the fusion protein. For example, a nucleotide sequence coding for the target epitope of a monoclonal antibody can be engineered into the expression vector in operative association with the regulatory elements and situated so that the expressed epitope is fused to the AveC polypeptide. For example, a nucleotide sequence coding for the FLAG™ epitope tag (International Biotechnologies Inc.), which is a hydrophilic marker peptide, can be inserted by standard techniques into the expression vector at a point corresponding, e.g., to the carboxyl terminus of the AveC polypeptide. The expressed AveC polypeptide-FLAG™ epitope fusion product can then be detected and affinity-purified using commercially available anti-FLAG™ antibodies.<br>
The expression vector encoding the AveC fusion protein can also be engineered to contain polylinker sequences that encode specific protease cleavage sites so that the expressed AveC polypeptide can be released from the carrier region or fusion partner by treatment with a specific protease. For example, the fusion protein vector can include DNA sequences encoding thrombin or factor Xa cleavage sites, among others.<br>
A signal sequence upstream from, and in reading frame with, the aveC ORF can be engineered into the expression vector by known methods to direct the trafficking and secretion of the expressed gene product.   Non-limiting examples of signal sequences include those<br><br>
from α-factor, immunoglobulins, outer membrane proteins, penicillinase, and T-cell receptors, among others.<br>
To aid in the selection of host cells transformed or transfected with cloning or expression vectors of the present invention, the vector can be engineered to further comprise a coding sequence for a reporter gene product or other selectable marker. Such a coding sequence is preferably in operative association with the regulatory element coding sequences, as described above. Reporter genes that are useful in the invention are well-known in the art and include those encoding green fluorescent protein, luciferase, xylE, and tyrosinase, among others. Nucleotide sequences encoding selectable markers are well known in the art, and include those that encode gene products conferring resistance to antibiotics or anti-metabolites, or that supply an auxotrophic requirement. Examples of such sequences include those that encode resistance to erythromycin, thiostrepton or kanamycin, among many others.<br>
5.2.2. Transformation Of Host Cells The present invention further provides transformed host cells comprising a polynucleotide molecule or recombinant vector of the invention, and novel strains or cell lines derived therefrom. Host cells useful in the practice of the invention are preferably Streptomyces cells, although other prokaryotic cells or eukaryotic cells can also be used. Such transformed host cells typically include but are not limited to microorganisms, such as bacteria transformed with recombinant bacteriophage DNA, plasmid DNA or cosmid DNA vectors, or yeast transformed with recombinant vectors, among others.<br>
The polynucleotide molecules of the present invention are intended to function in Streptomyces cells, but can also be transformed into other bacterial or eukaryotic cells, e.g., for cloning or expression purposes. A strain of E. coli can typically be used, such as, e.g., the DH5a strain, available from the American Type Culture Collection (ATCC), Rockville, MD, USA (Accession No. 31343), and from commercial sources (Stratagene). Preferred eukaryotic host cells include yeast cells, although mammalian cells or insect cells can also be utilized effectively.<br>
The recombinant expression vector of the invention is preferably transformed or transfected into one or more host cells of a substantially homogeneous culture of cells. The expression vector is generally introduced into host cells in accordance with known techniques, such as, e.g., by protoplast transformation, calcium phosphate precipitation, calcium chloride treatment, microinjection, electroporation, transfection by contact with a recombined virus, liposome-mediated transfection,  DEAE-dextran transfection, transduction, conjugation, or<br><br>
microprojectile bombardment.   Selection of transformants can be conducted by standard procedures, such as by selecting for cells expressing a selectable marker, e.g., antibiotic resistance, associated with the recombinant vector, as described above.<br>
Once the expression vector is introduced into the host cell, the integration and<br>
maintenance of the aveC coding sequence either in the host cell chromosome or episomally can be confirmed by standard techniques, e.g., by Southern hybridization analysis, restriction enzyme analysis, PCR analysis, including reverse transcriptase PCR (rt-PCR), or by immunological assay to detect the expected gene product.   Host cells containing and/or expressing the recombinant aveC coding sequence can be identified by any of at least four general approaches which are well-known in the art, including: (j) DNA-DNA, DNA-RNA, or RNA-antisense RNA hybridization; (ii) detecting the presence of "marker" gene functions; (iii) assessing the level of transcription as measured by the expression of aveC-specific mRNA<br>
transcripts in the host cell; and (iv) detecting the presence of mature polypeptide product as measured, e.g., by immunoassay or by the presence of AveC biological activity (e.g., the production of specific ratios and amounts of avermectins indicative of AveC activity in, e.g., S. avermitilis host cells).<br>
5.2.3.   Expression And Characterization<br>
Of A Recombinant AveC Gene Product<br>
Once the aveC coding sequence has been stably introduced into an appropriate host cell, the transformed host cell is clonally propagated, and the resulting cells can be grown under conditions conducive to the maximum production of the AveC gene product. Such conditions typically include growing cells to high density. Where the expression vector comprises an inducible promoter, appropriate induction conditions such as, e.g., temperature shift, exhaustion of nutrients, addition of gratuitous inducers (e.g., analogs of carbohydrates, such as isopropyl-p-D-thiogalactopyranoside (IPTG)), accumulation of excess metabolic by products, or the like, are employed as needed to induce expression.<br>
Where the expressed AveC gene product is retained inside the host cells, the cells are harvested and lysed, and the product isolated and purified from the lysate under extraction conditions known in the art to minimize protein degradation such as, e.g., at4°C, or in the presence of protease inhibitors, or both. Where the expressed AveC gene product is secreted from the host cells, the exhausted nutrient medium can simply be collected and the product isolated therefrom.<br>
The expressed AveC gene product can be isolated or substantially purified from cell lysates or culture medium, as appropriate, using standard methods, including but not limited<br><br>
to any combination of the following methods: ammonium sulfate precipitation, size fractionation, ion exchange chromatography, HPLC, density centrifugation, and affinity chromatography. Where the expressed AveC gene product exhibits biological activity, increasing purity of the preparation can be monitored at each step of the purification procedure by use of an appropriate assay. Whether or not the expressed AveC gene product exhibits biological activity, it can be detected as based, e.g., on size, or reactivity with an antibody otherwise specific for AveC, or by the presence of a fusion tag. As used herein, an AveC gene product is "substantially purified" where the product constitutes more than about 20 wt% of the protein in a particular preparation. Also, as used herein, an AveC gene product is "isolated" where the product constitutes at least about 80 wt% of the protein in a particular preparation.<br>
The present invention thus provides a recombinantly-expressed isolated or substantially purified S. avermitilis AveC gene product comprising the amino acid sequence encoded by the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604), or the amino acid sequence of FIGURE 1 (SEQ ID NO:2) or a substantial portion thereof, and homologs thereof.<br>
The present invention further provides a recombinantly-expressed isolated or substantially purified S. hygroscopicus AveC homolog gene product comprising the amino acid sequence of SEQ ID NO:4 or a substantial portion thereof, and homologs thereof.<br>
The present invention further provides a method for producing an AveC gene product, comprising culturing a host cell transformed with a recombinant expression vector, said vector comprising a polynucleotide molecule having a nucleotide sequence encoding the AveC gene product, which polynucleotide molecule is in operative association with one or more regulatory elements that control expression of the polynucleotide molecule in the host cell, under conditions conducive to the production of the recombinant AveC gene product, and recovering the AveC gene product from the cell culture.<br>
The recombinantly expressed S. avermitilis AveC gene product is useful for a variety of purposes, including for screening compounds that alter AveC gene product function and thereby modulate avermectin biosynthesis, and for raising antibodies directed against the AveC gene product.<br>
Once an AveC gene product of sufficient purity has been obtained, it can be characterized by standard methods, including by SDS-PAGE, size exclusion chromatography, amino acid sequence analysis, biological activity in producing appropriate products in the avermectin biosynthetic pathway, ete.   For example, the amino acid sequence of the AveC<br><br>
gene product can be determined using standard.peptide sequencing techniques. The AveC gene product can be further characterized using hydrophilicity analysis (see, e.g., Hopp and Woods, 1981, Proc. Natl. Acad. Sci. USA  78:3824), or analogous software algorithms, to identify hydrophobic and hydrophilic regions of the AveC gene product.  Structural analysis can be carried out to identify regions of the AveC gene product that assume specific secondary structures.  Biophysical methods such as X-ray crystallography (Engstrom, 1974, Biochem. Exp. Biol. 11: 7-13), computer modelling (Fletterick and Zoller (eds), 1986, in: Current Communications in Molecular Biology, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY), and nuclear magnetic resonance (NMR) can be used to map and study sites of interaction between the AveC gene product and its substrate.   Information obtained from these studies can be used to select new sites for mutation in the aveC ORF to help develop new strains of S. avermitUis having more desirable avermectin production characteristics . 5.3. Construction And Use Of AveC Mutants The present invention provides a polynucleotide molecule comprising a nucleotide sequence that is otherwise the same as the S. avermitilis aveC allele or a degenerate variant thereof, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprises one or more mutations, so that cells of S. avermitilis strain ATCC 53692 in which the wild-type aveC allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide sequence or the degenerate variant thereof produce a different ratio or amount of avermectins than are produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele.<br>
According to the present invention, such polynucleotide molecules can be used to produce novel strains of S. avermitilis that exhibit a detectable change in avermectin production compared to the same strain which instead expresses only the wild-type aveC allele. In a preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce avermectins in a reduced class 2:1 ratio compared to the same strain which instead expresses only the wild-type aveC allele. In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis that produce increased levels of avermectins compared to the same strain which instead expresses only a single wild-type aveC allele. In a further preferred embodiment, such polynucleotide molecules are useful to produce novel strains of S. avermitilis in which the aveC gene has been inactivated.<br><br>
Mutations to the aveC allele or coding sequence include any mutations that introduce<br>
one or more amino acid deletions, additions, or substitutions into the AveC gene product, or that result in truncation of the AveC gene product, or any combination thereof, and that produce the desired result. Such mutated aveC allele sequences are also intended to include any degenerate variants thereof. For example, the present invention provides polynucleotide molecules comprising the nucleotide sequence of the aveC allele or a degenerate variant thereof, or the AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or a degenerate variant thereof, or the nucleotide sequence of the aveC ORF of S. avermitilis as present in FIGURE 1 (SEQ ID NO:1) or a degenerate variant thereof, but that further comprise one or more mutations that encode the substitution of an amino acid residue with a different amino acid residue at selected positions in the AveC gene product.   In several non-limiting embodiments, several of which are exemplified below, such substitutions can be carried out at any amino acid positions of the AveC gene product which correspond to amino acid positions<br>
38, 48, 55, 89, 99, 111, 136, 138, 139, 154, 179, 228. 230, 238, 266, 275. 289 or 298 of SEQ ID NO:2. or some combination thereof.<br>
Mutations to the aveC coding sequence are carried out by any of a variety of known methods, including by use of error-prone PCR, or by cassette mutagenesis. For example, oligonucleotide-directed mutagenesis can be employed to alter the sequence of the aveC allele or ORF in a defined way such as, e.g., to introduce one or more restriction sites, or a termination codon, into specific regions within the aveC allele or ORF. Methods such as those described in U.S. Patent 5,605,793, U.S. Patent 5,830,721 and U.S. Patent 5,837,458, which involve random fragmentation, repeated cycles of mutagenesis, and nucleotide shuffling, can also be used to generate large libraries of polynucleotides having nucleotide sequences encoding aveC mutations.<br>
Targeted mutations can be useful, particularly where they serve to alter one or more conserved amino acid residues in the AveC gene product. For example, a comparison of deduced amino acid sequences of AveC gene products and AveC homolog gene products from S. avermitilis (SEQ ID NO:2), S. griseochromogenes (SEQ ID NO:5), and S. hygroscopicus (SEQ ID NO:4), as presented in FIGURE 6, indicates sites of significant conservation of amino acid residues between these species. Targeted mutagenesis vat leads to a change in one or more of these conserved amino acid residues can be particularly effective in producing novel mutant strains that exhibit desirable alterations in avermectin production.<br><br>
Random mutagenesis can also be useful, and can be carried out by exposing cells of S. avermitilis to ultraviolet radiation or x-rays, or to chemical mutagens such as N-methyl-N'-nitrosoguanidine, ethyl methane sulfonate, nitrous acid or nitrogen mustards. See, e.g., Ausubel, 1989, above, for a review of mutagenesis techniques.<br>
Once mutated  polynucleotide molecules are generated,  they are screened  to determine whether they can modulate avermectin biosynthesis in S. avermitilis. In a preferred embodiment, a polynucleotide molecule having a mutated nucleotide sequence is tested by complementing a strain of S. avermitilis in which the aveC gene has been inactivated to give an aveC negative {aveC) background.  In a non-limiting method, the mutated polynucleotide molecule is spliced into an expression plasmid in operative association with one or more regulatory elements, which plasmid also preferably comprises one or more drug resistance genes to allow for selection of transformed cells. This vector is then transformed into aveC host cells using known techniques, and transformed cells are selected and cultured in appropriate fermentation media under conditions that permit or induce avermectin production. Fermentation products are then analyzed by HPLC to determine the ability of the mutated polynucleotide molecule to complement the host cell.    Several vectors bearing mutated polynucleotide molecules capable of reducing the B2:B1  ratio of avermectins, including pSE188, pSE199, pSE231, pSE239, and pSE290 through pSE297, are exemplified in Section 8.3, below.<br>
The present invention provides methods for identifying mutations of the S. avermitilis aveC ORF capable of altering the ratio and/or amount of avermectins produced. In a preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF capable of altering the class 2:1 ratio of avermectins produced, comprising: (a) determining the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product has been introduced and is being expressed; (b) determining the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a wild-type aveC allele or an aveC allele having the nucleotide sequence of the ORF of FIGURE 1 (SEQ ID NO:1) or a nucleotide sequence that is homologous thereto; and (c) comparing the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) to the class 2:1 ratio of avermectins produced by the S. avermMis cells of step (b); such thai if the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (a) is different from the class 2:1 ratio of avermectins produced by the S. avermitilis cells of step (b), then a<br><br>
mutation of the aveC ORF capable of altering the class 2:1 ratio of avermectins has been identified. In a preferred embodiment, the class 2:1 ratio of avermectins is reduced by the mutation.<br>
In a further preferred embodiment, the present invention provides a method for identifying mutations of the aveC ORF or genetic constructs comprising the aveC ORF capable of altering the amount of avermectins produced, comprising:   (a) determining the amount of avermectins produced by cells of a strain of S. avermitilis in which the aveC allele native thereto has been inactivated, and into which a polynucleotide molecule comprising a nucleotide sequence encoding a mutated AveC gene product or comprising a genetic construct comprising a nucleotide sequence encoding an AveC gene product has been introduced and is being expressed; (b) determining the amount of avermectins produced by cells of the same strain of S. avermitilis as in step (a) but which instead express only a wild-type aveC allele or a nucleotide sequence that is homologous thereto; and (c) comparing the amount of avermectins produced by the S. avermitilis cells of step (a) to the amount of avermectins produced by the S. avermitilis cells of step (b); such that if the amount of avermectins produced by the S. avermitilis cells of step (a) is different from the amount of avermectins produced by the S. avermitilis cells of step (b), then a mutation of the aveC ORF or a genetic construct capable of altering the amount of avermectins has been identified. In a preferred embodiment, the amount of avermectins produced is increased by the mutation.<br>
Any of the aforementioned methods for identifying mutations are be carried out using fermentation culture media preferably supplemented with cyclohexane carboxylic acid, although other appropriate fatty acid precursors, such as any one of the fatty acid precursors listed in TABLE 1, can also used.<br>
Once a mutated polynucleotide molecule that modulates avermectin production in a desirable direction has been identified, the location of the mutation in the nucleotide sequence can be determined. For example, a polynucleotide molecule having a nucleotide sequence encoding a mutated AveC gene product can be isolated by PCR and subjected to DNA sequence analysis using known methods. By comparing the DNA sequence of the mutated aveC allele to that of the wild-type aveC allele, the mutation(s) responsible for the alteration in avermectin production can be determined. In specific though non-limiting embodiments of the present invention, S. avermitilis AveC gene products comprising either single amino acid substitutions at any of residues 55 (S55F), 138 (S138T), 139 (A139T), or 230 (G230D), or double substitutions at positions 138 (S138T) and 139 (A139T or A139F), yielded changes in AveC gene product function such that the ratio of class 2:1 avermectins produced was altered<br><br>
(see Section 8, below), wherein the recited amino acid positions correspond to those<br>
presented in FIGURE 1 (SEQ ID NO:2). In addition, the following seven combinations of<br>
mutations have each been shown to effectively reduce the class 2:1 ratio of avermectins: (1)<br>
D48E/A89T;	(2)	S138T/A139T/G179S;	(3)	Q38P/L136P/E238D;	(4)<br>
F99S/S138T/A139T/G179S; (5) A139T7 M228T; (6) G111V/P289L; (7) A139T/K154E/Q298H. As used herein, the aforementioned designations, such as A139T, indicate the original amino acid residue by single letter designation, which in this example is alanine (A), at the indicated position, which in this example is position 139 (referring to SEQ ID NO:2) of the polypeptide, followed by the amino acid residue which replaces the original amino acid residue, which in this example is threonine (T). Accordingly, polynucleotide molecules having nucleotide sequences that encode mutated S. avermitilis AveC gene products comprising amino acid substitutions or deletions at one or more of amino acid positions 38, 48, 55, 89, 99, 111,136, 138,139,154, 179, 228, 230, 238, 266, 275, 289 or 298 (see FIGURE 1), or any combination thereof, are encompassed by the present invention.<br>
In a preferred embodiment, such mutations encode amino acid substitutions selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(b)	amino acid residue D at position 48 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a<br>
. conservative substitution for P;<br>
(g)	amino acid residue S at position 138 replaced by T or by an amino acid that is a<br>
conservative substitution for T;<br>
(h) amino acid residue A at position 139 replaced by T or F, or by an amino acid that<br>
is a conservative substitution for T or F; (i)   amino acid residue K at position 154 replaced by E or by an amino acid that is a<br>
conservative substitution for E;<br><br>
(j)   amino acid residue G at position 179 replaced by S or by an amino acid that is a<br>
conservative substitution for S; (k) amino acid residue M at position 228 replaced by T or by an amino acid that<br>
is a conservative substitution for T; (I)   amino acid residue E at position 238 replaced by 0 or by an amino acid that is a<br>
conservative substitution for D; (m) amino acid, residue P at position 289 replaced by L or by an amino acid that is a<br>
conservative substitution for L; and (n) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H; wherein conservative amino acid substitutions are as defined above in Section 5.1.<br>
In a further preferred embodiment, such mutations encode a combination of amino acid substitutions, wherein the combination of amino acid residues substituted is selected from the group consisting of: .<br>
(a)	amino acid residues S138 and A139;<br>
(b)	amino acid residues D48 and A89;<br>
(c)	amino acid residues S138, A139 and G179;<br>
(d)	amino acid residues Q38, L136 and E238;<br>
(e)	amino acid residues F99, S138, A139 and G179;<br>
(f)	amino acid residues A139 and M228;<br>
(g)	amino acid residues G111 and P289; and (h) amino acid residues A139, K154 and Q298.<br>
Jn a further preferred embodiment, specific combinations of mutations in the aveC allele useful in effectively reducing the class 2:1 ratio of avermectins according to the present invention are selected from one or more of the group consisting of:<br>
(a)	S138T/A139T<br>
(b)	S138T7A139F<br>
(c)	D48E/A89T;<br>
(d)	S138T7A139T/G179S;<br>
(e)	Q38P/L136P/E238D;<br>
(f)	F99S/S138T/A139T/G179S;<br>
(g)	A139T/M228T;<br>
(h) G111V/P289L;and (i)   A139T/K154E/Q298H.<br><br>
The present invention further provides compositions for making novel strains of S. avermitilis, the cells of which contain a mutated aveC allele that results in the alteration of avermectin production. For example, the present invention provides recombinant vectors that can be used to target any of the polynucleotide molecules comprising mutated nucleotide sequences of the present invention to the site of the aveC gene of the S. avermitilis chromosome to either insert into or replace the aveC ORF or a portion thereof by homologous recombination.   According to the present invention, however, a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention provided herewith can also function to modulate avermectin biosynthesis when inserted into the S. avermitilis chromosome at a site other than at the aveC gene, or when maintained episomally in S. avermitilis  cells.     Thus,   the   present   invention   also  provides  vectors   comprising   a polynucleotide molecule comprising a mutated nucleotide sequence of the present invention, which vectors can be used to insert the polynucleotide molecule at a site in the S. avermitilis chromosome other than at the aveC gene, or to be maintained episomally.<br>
In a preferred embodiment, the present invention provides gene replacement vectors that can be used to insert a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis, thereby generating novel strains of S. avermitilis, the cells of which produce avermectins in an altered class 2:1 ratio compared to cells of the same strain which instead express only the wild-type aveC allele. In a preferred embodiment, the class 2:1 ratio of avermectins produced by the cells is reduced. Such gene replacement vectors can be constructed using mutated polynucleotide molecules present in expression vectors provided herewith, such as, e.g., pSE188, pSE199, and pSE231, which expression vectors are exemplified in Section 8 below.<br>
In a further preferred embodiment, the present invention provides vectors that can be used to insert a mutated aveC allele or degenerate variant thereof into cells of a strain of S. avermitilis to generate novel strains of cells that produce altered amounts of avermectins compared to cells of the same strain which instead express only the wild-type aveC allele. In a preferred embodiment, the amount of avermectins produced by the cells is increased. In a specific though non-limiting embodiment, such a vector further comprises a strong promoter as known in the art, such as, e.g., the strong constitutive ermE promoter from Saccharopolyspora erythraea, that is situated upstream from, and in operative association with, the aveC allele. Such a vector can be plasmid pSE189, described in Example 11 below, or can be constructed using the mutated aveC allele of plasmid pSE189.<br><br>
In a further preferred embodiment, the present invention provides gene replacement vectors that are useful to inactivate the aveC gene in a wild-type strain of S. avermitilis. In a non-limiting embodiment, such gene replacement vectors can be constructed using the mutated polynucleotide molecule present in plasmid pSE180 (ATCC 209605), which is exemplified in Section 8.1, below (FIGURE 3). The present invention further provides gene replacement vectors that comprise a polynucleotide molecule comprising or consisting of nucleotide sequences that naturally flank the aveC gene in situ in the S. avermitilis chromosome, including, e.g., those flanking nucleotide sequences shown in FIGURE 1 (SEQ ID NO:1), which vectors can be used to delete the S. avermitilis aveC ORF.<br>
The present invention further provides methods for making novel strains of S.<br>
avermitilis comprising cells that express a mutated aveC allele and that produce an altered ratio and/or amount of avermectins compared to cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele.   In a preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that express a mutated aveC allele and that produce an altered class 2:1 ratio of avermectins compared to cells of the same strain of S. avermitilis that instead express only a wild-type aveC allele, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele thereof compared to cells of the same strain that instead express only a wild-type aveC allele, and selecting transformed cells that produce avermectins in an altered class 2:1  ratio compared to the class 2:1 ratio produced by cells of the strain that instead express only the wild-type aveC allele.   In a more preferred embodiment, the present invention provides a method for making a novel strain of S. avermitilis, comprising transforming cells of a strain of S. avermitilis with a vector capable of introducing a mutation into the aveC allele of such cells, wherein the mutation to the aveC allele results in the substitution in the encoded AveC gene product of a different amino acid residue at one or more amino acid positions corresponding to amino acid residues 38, 48, 55, 89, 99, 111, 136, 138, 139, 154, 179, 228. 230, 238, 266, 275, 289 or 298 of SEQ ID NO:2, such that cells of the S. avermitilis strain in which the aveC allele has been so mutated produce a class 2:1 ratio of avermectins that is different from the ratio produced by cells of the same S. avermitilis strain that instead express only the wild-type aveC allele. In a preferred embodiment, the altered class 2:1 ratio of avermectins is reduced. As used herein, where an amino acid residue encoded by an aveC allele in the S. avermitilis chromosome, or in a vector or isolated polynucleotide molecule of the present<br><br>
invention is referred to as "corresponding to" a particular amino acid residue of SEQ ID NO:2, or where an amino acid substitution is referred to as occurring at a particular position "corresponding to" that of a specific numbered amino acid residue of SEQ ID NO:2, this is intended to refer to the amino acid residue at the same relative location in the AveC gene product, which the skilled artisan can quickly determine by reference to the amino acid sequence presented herein as SEQ ID NO:2.<br>
The present invention further provides methods of making novel strains wherein<br>
specific mutations in the aveC allele encoding particular mutations are recited as base<br>
changes at specific nucleotide positions in the aveC allele "corresponding to" particular<br>
nucleotide positions as shown in SEQ ID NO:1. As above with regard to corresponding amino acid positions, where a nucleotide position in the aveC allele is referred to as "corresponding to" a particular nucleotide position in SEQ ID NO:1, this is intended to refer to the nucleotide at the same relative location in the aveC nucleotide sequence, which the skilled artisan can quickly determine by reference to the nucleotide sequence presented herein as SEQ ID NO:1. In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis comprising cells that produce altered amounts of avermectin, comprising transforming cells of a strain of S. avermitilis with a vector that carries a mutated aveC allele or a genetic construct comprising the aveC allele, the expression of which results in an alteration in the amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct as compared to cells of the same strain that instead express only a  single wild-type  aveC allele,  and  selecting transformed cells that produce avermectins in an altered amount compared to the amount of avermectins   produced by cells of the strain that instead express only the single wild-type aveC allele.    In a preferred embodiment, the amount of avermectins produced in the transformed cells is increased.<br>
In a further preferred embodiment, the present invention provides a method for making novel strains of S. avermitilis, the cells of which comprise an inactivated aveC allele, comprising transforming cells of a strain of S. avermitilis that express any aveC allele with a vector that inactivates the aveC allele, and selecting transformed cells in which the aveC allele has been inactivated. In a preferred though non-limiting embodiment, cells of a strain of S. avermitilis are transformed with a gene replacement vector that carries an aveC allele that has been inactivated by mutation or by replacement of a portion of the aveC allele with a heterologous gene sequence, and transformed cells are selected in which the aveC allele otherwise native thereto has been replaced with the inactivated aveC allele.   Inactivation of<br><br>
the aveC allele can be determined by HPLC analysis of fermentation products, as described below. In a specific though non-limiting embodiment described in Section 8.1 below, the aveC allele is inactivated by insertion of the ermE gene from Saccharopolyspora erythraea into the aveC ORF.<br>
The present invention further provides novel strains of S. avermitilis comprising cells that have been transformed with any of the polynucleotide mofecufes or vectors of (he present invention.   In a preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele or degenerate variant thereof in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain produce avermectins in an altered class 2:1  ratio compared to the class 2:1  ratio of avermectins produced by cells of the same strain that instead express only the wild-type aveC allele.  In a preferred embodiment, the altered class 2:1 ratio produced by the novel cells is reduced.    Such novel strains are useful in the large-scale production of commercially desirable avermectins such as doramectin.   In a more preferred embodiment, the present invention provides cells of S. avermitilis comprising any of the aforementioned mutations or combinations of mutations in the aveC allele at nucleotide positions corresponding to those presented hereinabove or which otherwise encode any of the aforementioned amino acid substitutions in the AveC gene product. Although such mutations can be present in such cells on an extrachromosomal element such as a plasmid, it is preferred that such mutations are present in the aveC allele located on the S. avermitilis chromosome.    In a preferred embodiment, the present invention provides a strain of Streptomyces avermitilis comprising cells having a mutation in the aveC allele that encodes an AveC gene product having a substitution at one or more amino acid positions corresponding to amino acid residues 38,48, 55, 89, 99. 111, 136, 138, 139. 154, 179, 228, 230. 238, 266. 275, 289. or 298 of SEQ ID NO:2, wherein the cell produces a class 2:1 ratio of avermectins that is different from the ratio produced by a cell of the same S. avermitilis strain which express the wild-type aveC allele.<br>
It is a primary objective of the screening assays described herein to identify mutated alleles of the aveC gene the expression of which, in S. avermitilis cells, alters and, more particularly, reduces the ratio of class 2:1 avermectins produced. In a preferred embodiment, the ratio of B2:B1 avermectins produced by cells of a novel S. avermitilis strain of the present invention expressing a mutated aveC allele, or degenerate variant thereof, of the present invention is about 1.6:1 or less. In a more preferred embodiment, the ratio is about 1:1 or less. In a more preferred embodiment, the ratio is about 0.84:1 or less. In a more preferred embodiment, the ratio is about 0.80:1 or less.   In a more preferred embodiment, the ratio is<br><br>
about 0.75:1 or less. In a more preferred embodiment, the ratio is about 0:73:1 or less. In a more preferred embodiment, the ratio is about 0.68:1 or less. In an even more preferred embodiment, the ratio, is about 0.67:1 or less. In a more preferred embodiment, the ratio is about 0.57:1 or less. In an even more preferred embodiment, the ratio is about 0.53:1 or less. In an even more preferred embodiment, the ratio is about 0.42:1 or less. In an even more preferred embodiment, the ratio is about 0.40:1 or less.<br>
In a specific embodiment described below, novel cells of the present invention<br>
produce cyclohexyl B2:cyclohexyl B1 avermectins in a ratio of less than 1.6:1.  In a different<br>
specific embodiment described below, novel cells of the present invention produce cyclohexyl<br>
B2:cyclohexyl B1 avermectins in a ratio of about 0.94:1.    In a further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
B2:cyclohexyl B1 avermectins in a ratio of about 0.88:1.    In a further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.84:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.75:1.   In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl Bi avermectins in a ratio of about 0.73:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1-avermectins in a ratio of about 0.68:1. In a still further different specific<br>
embodiment described below, novel cells of the present invention produce cyclohexyl<br>
2:cyclohexyl B1 avermectins in a ratio of about 0.67:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.57:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.53:1. In a still further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.42:1.   In yet a further different specific embodiment described below, novel cells of the present invention produce cyclohexyl 2:cyclohexyl B1 avermectins in a ratio of about 0.40:1.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells which express a mutated aveC allele or a degenerate variant thereof, or a genetic construct comprising an aveC allele or a degenerate variant thereof, in place of, or in addition to, the wild-type aveC allele, wherein the cells of the novel strain<br><br>
produce an altered amount of avermectins compared to cells of the same strain that instead express only the wild-type aveC allele. In a preferred embodiment, the novel strain produces an increased amount of avermectins. In a non-limiting embodiment, the genetic construct further comprises a strong promoter, such as the strong constitutive ermE promoter from Saccharopolyspora erythraea, upstream from and in operative association with the aveC ORF.<br>
In a further preferred embodiment, the present invention provides novel strains of S. avermitilis comprising cells in which the aveC gene has been inactivated. Such strains are useful both for the different spectrum of avermectins that they produce compared to the wild-type strain, and in complementation screening assays as described herein, to determine whether targeted or random mutagenesis of the aveC gene affects avermectin production. In a specific embodiment described below, S. avermitilis host cells were genetically engineered to contain an inactivated aveC gene. For example, strain SE180-11, described in the examples below, was generated using the gene replacement plasmid pSE180 (ATCC 209605) (FIGURE 3), which was constructed to inactivate the S. avermitilis aveC gene by insertion of the ermE resistance gene into the aveC coding region.<br>
The present invention further provides recombinantly expressed mutated S. avermitilis AveC gene products encoded by any of the aforementioned polynucleotide molecules of the invention, and methods of preparing the same.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele that encodes a gene product that alters the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele compared to cells of the same strain that instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the class 2:1 ratio of avermectins produced in the culture by cells expressing the mutated aveC allele is reduced. This process provides increased efficiency in the production of commercially valuable avermectins such as doramectin.<br>
The present invention further provides a process for producing avermectins, comprising culturing cells of a strain of S. avermitilis, which cells express a mutated aveC allele or a genetic construct comprising an aveC allele that results in the production of an altered amount of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or genetic construct compared to cells of the same strain which do not<br><br>
express the mutated aveC allele or genetic construct but instead express only the wild-type aveC allele, in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture. In a preferred embodiment, the amount of avermectins produced in culture by cells expressing the mutated aveC allele, degenerate variant or genetic construct is increased.<br>
The present invention further provides a novel composition of avermectins produced by a strain of S. avermitilis expressing a mutated aveC allele or degenerate variant thereof that encodes a gene product that reduces the class 2:1 ratio of avermectins produced by cells of a strain of S. avermitilis expressing the mutated aveC allele or degenerate variant compared to cells of the same strain that instead express only the wild-type aveC allele, wherein the avermectins in the novel composition.are produced in a reduced class 2:1 ratio as compared to the class 2:1 ratio of avermectins produced by cells of the same strain of S. avermitilis that instead express only the wild-type aveC allele. The novel avermectin composition can be present as produced in exhausted fermentation culture fluid, or can be harvested therefrom. The novel avermectin composition can be partially or substantially purified from the culture fluid by known biochemical techniques of purification, such as by ammonium sulfate precipitation, dialysis, size fractionation, ion exchange chromatography, HPLC, etc.<br>
5.4. Uses Of Avermectins<br>
Avermectins are highly active antiparasitic agents having particular utility as anthelmintics, ectoparasiticides, insecticides and acaricides. Avermectin compounds produced according to the methods of the present invention are useful for any of these purposes. For example, avermectin compounds produced according to the present invention are useful to treat various diseases or conditions in humans, particularly where those diseases or conditions are caused by parasitic infections, as known in the art. See, e.g., Ikeda and Omura, 1997, Chem. Rev. 97(7):2591-2609. More particularly, avermectin compounds produced according to the present invention are effective in treating a variety of diseases or conditions caused by endoparasites, such as parasitic nematodes, which can infect humans, domestic animals, swine, sheep, poultry, horses or cattle.<br>
More specifically, avermectin compounds produced according to the present invention, are effective against nematodes that infect humans, as well as those that infect various species of animals. Suchnematodes include gastrointestinal parasites such as Ancylostoma, Necator, Ascaris, Strongyloides, Trichinella, Capillaria, Trichuris, Enterobius, Dirofilaria, and<br><br>
parasites that are found in the blood or other tissues or organs, such as filarial worms and the extract intestinal states of Strongyloides and Trichinella.<br>
The avermectin compounds produced according to the present invention are also useful in treating ectoparasitic infections including, e.g., arthropod infestations of mammals and birds, caused by ticks, mites, lice, fleas, blowflies, biting insects, or migrating dipterous farvae that can affect cattle and horses, among others.<br>
The avermectin compounds produced according to the present invention are also useful as insecticides against household pests such as, e.g., the cockroach, clothes moth, carpet beetle and the housefly among others, as well as insect pests of stored grain and of agricultural plants, which pests include spider mites, aphids, caterpillars, and orthopterans such as locusts, among others.<br>
Animals that can be treated with the avermectin compounds produced according to the present invention include sheep, cattle, horses, deer, goats, swine, birds including poultry, and dogs and cats.<br>
An avermectin compound produced according to the' present invention is administered in a formulation appropriate to the specific intended use, the particular species of host animal being treated, and the parasite or insect involved. For use as a parasiticide, an avermectin compound produced according to the present invention can be administered orally in the form of a capsule, bolus, tablet or liquid drench or, alternatively, can be administered as a pour-on, or by injection, or as an implant. Such formulations are prepared in a conventional manner in accordance with standard veterinary practice. Thus, capsules, boluses or tablets can be prepared by mixing the active ingredient with a suitable finely divided diluent or carrier additionally containing a disintegrating agent and/or binder such as starch, lactose, talc, magnesium stearate, etc. A drench formulation can be prepared by dispersing the active ingredient in an aqueous solution together with a dispersing or wetting agent, etc. Injectable formulations can be prepared in the form of a sterile solution, which can contain other substances such as, e.g., sufficient salts and/or glucose to make the solution isotonic with blood.<br>
Such formulations will vary with regard to the weight of active compound depending on the patient, or species of host animal to be treated, the severity and type of infection, and the body weight of the host. Generally, for oral administration a dose of active compound of from about 0.001 to 10 mg per kg of patient or animal body weight given as a single dose or in divided doses for a period of from 1 to 5 days will be satisfactory.   However, there can be<br><br>
instances where higher or lower dosage ranges are indicated, as determined, e.g., by. a physician or veterinarian, as based on clinical symptoms.<br>
As an alternative, an avermectin compound produced according to the present invention can be administered in combination with animal feedstuff, and for this purpose a concentrated feed additive or premix can be prepared for mixing with the normal animal feed. For use as an insecticide, and for treating agricultural pests, an avermectin compound produced according to the present invention can be applied as a spray, dust, emulsion and the like in accordance with standard agricultural practice.<br>
6. EXAMPLE: FERMENTATION OF STREPTOMYCES AVERMITILIS AND B2:B1 AVERMECTIN ANALYSIS<br>
Strains   lacking   both   branched-chain   2-oxo   acid   dehydrogenase   and   5-0- ethyltransferase activities produce no avermectins if the fermentation medium is pot upplemented with fatty acids. This example demonstrates that in such mutants a wide range f B2:B1 ratios of avermectins can be obtained when biosynthesis is initiated in the presence of different fatty acids.<br>
6.1. Materials And Methods<br>
Streptomyces avermitilis ATCC 53692 was stored at -70°C as a whole broth prepared in seed medium consisting of: Starch (Nadex, Laing National) - 20g;
g/liter.   The fermentation broth was incubated for 14 days at 28°C, then the broth was<br>
centrifuged (2,500 rpm for 2 min) and the supernatant discarded.   The mycelial pellet was extracted with acetone (15 ml), then with dichloromethane (30 ml), and the organic phase<br><br>
separated, filtered, then evaporated to dryness. The residue was taken up in methanol (1 ml) and analyzed by HPLC with a Hewlett-Packard 1090A liquid chromatograph equipped with a scanning diode-array detector set at 240 nm. The column used was a Beckman Ultrasphere C-18, 5 nm, 4.6 mm x 25 cm column maintained at 40°C.   Twenty-five μl of the above<br>
5 methanol solution was injected onto the column. Elution was performed with a linear gradient of mefhanol-water from 80:20 to 95:5 over 40 min at 0.85/ml min. Two standard concentrations of cyclohexyl B1 were used to calibrate the detector response, and the area under the curves for B2 and B1 avermectins was measured.<br>
6.2. Results<br>
 	The, HPLC retention times observed for the B2 and B1 avermectins, and the 2:1<br>
ratios, are shown in TABLE 1.<br>
TABLE 1<br><br>
	HPLC Retention Time (min)	Ratio<br>
Substrate	B2	B1	B2.B1<br>
4-Tetrahydropyran parboxylic acid	8.1	.    14.5	0.25<br>
Isobutyric acid	10.8	18.9	0.5<br>
3-Furoic acid	7.6	14.6	0.62<br>
S-(+)-2-methylbutyric acid	12.8	21.6	10<br>
Cyclohexanecarboxyiic acid	16.9	26.0	1.6<br>
3-Thiophenecarboxylic acid	8.8	16.0	1.8<br>
Cyclopentanecarboxylic acid	14.2	23.0	2.0<br>
3-Trifluoromethylbutyric acid	10.9	18.8	3.9<br>
2-Methylpentanoic acid	14.5	24.9	4.2<br>
Cycloheptanecarboxylic acid	18.6	29.0	15.0<br>
The data presented in TABLE 1 demonstrates an extremely wide range of B2:B1 avermectin product ratios, indicating a considerable difference 'in the results of detiydrative conversion of class 2 compounds to class 1 compounds, depending on the nature of the fatty acid side chain starter unit supplied.   This indicates that changes in B2:B1 ratios resulting<br><br>
from alterations to the AveC protein may be specific to particular substrates. Consequently, screening for mutants exhibiting changes in the B2:B1 ratio obtained with a particular substrate needs to be done in the presence of that substrate. The subsequent examples described below use cyclohexanecarboxylic acid as the screening substrate. However, this substrate is used merely to exemplify the potential, and is not intended to limit the applicability, of the present invention.<br>
7. EXAMPLE: ISOLATION OF THE aveC GENE This example describes the isolation and characterization of a region of the Streptomyces   avermitilis  chromosome   that   encodes   the   AveC   gene   product.      As demonstrated below, the aveC gene was identified as capable of modifying the ratio of cyclohexyl-B2 to cyclohexyl-B1 (B2:B1) avermectins produced.<br>
7.1. Materials And Methods 7.1.1. Growth Of Streptomyces For DNA Isolation The following method was followed for growing Streptomyces. Single colonies of S. avermitilis ATCC 31272 (single colony isolate #2) were isolated on 1/2 strength YPD-6 containing: Difco Yeast Extract - 5 g; Difco Bacto-peptone - 5 g; dextrose - 2.5 g; MOPS . 5 g; Difco Bacto agar -15 g. Final volume was adjusted to 1 liter with dH20, pH was adjusted to 7.0, and the medium was autoclaved at 121 °C for 25 min.<br>
The mycelia grown in the above medium were used to inoculate 10 ml of TSB medium (Difco Tryptic Soy Broth - 30 g, in 1 liter dH20, autoclaved at 121 °C for 25 min) in a 25 mm x 150 mm tube which was maintained with shaking (300 rpm) at 28°C for 48-72 hr§. 7.1.2. Chromosomal DNA Isolation From Streptomyces Aliquots (0.25 mi or 0.5 ml) of mycelia grown as described above were placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded and the cells were resuspended in 0.25 ml TSE buffer (2o ml 1.5 M sucrose, 2.5 ml 1 M Tris-HCI, pH 8.0, 2.5 ml 1 M EDTA, pH 8.0, and 75 ml dH20) containing 2 mg/ml lysozyme. The samples were incubated at 37°C for 20 min with shaking, loaded into an AutoGen 540™ automated nucleic acid isolation instrument (Integrated Separation Systems, Natick, MA), and genomic DNA isolated using Cycle 159 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 5 ml of mycelia were placed in a 17 mm. x 100 mm tube, the cells concentrated by centrifugation at 3,000 rpm for 5 min, and the supernatant removed. Cells were resuspended in 1 ml TSE buffer, concentrated by centrifugation at 3,000 rpm for 5 min, and the supernatant removed. Cells were resuspended in 1 ml TSE buffer containing 2 mg/ml<br><br>
lysozyme, and incubated at 37°C with shaking for 30-60 min. After incubation, 0.5 ml 10% sodium dodecyl sulfate (SDS) was added and the cells incubated at 37°C until lysis was complete. The lysate was incubated at 65°C for 10 min, cooled to rm temp, split into two 1.5 ml Eppendorf tubes, and extracted 1x with 0.5 ml phenol/chloroform (50% phenol previously equilibrated with 0.5 M Tris, pH 8.0; 50% chloroform). The aqueous phase was removed and extracted 2 to 5x with chlorofprm:isoamyl alcohol (24:1). The DNA was precipitated by adding 1/10 volume 3M sodium acetate, pH 4.8, incubating the mixture on ice for 10 min, centrifuging the mixture at 15,000 rpm at 5°C for 10 min, and removing the supernatant to a clean tube to which 1 volume of isopropanol was added. The supernatant plus isopropanol mixture was then incubated on ice for 20 min, centrifuged at 15,000 rpm for 20 min at 5°C, the supernatant removed, and the DNA pellet washed 1x with 70% ethanol. After the pellet was dry, the DNA was resuspended in TE buffer (10 mM Tris, 1 mM EDTA, pH 8.0).<br>
7.1.3. Plasmid DNA Isolation From Streptomyces An aliquot (1.0 ml) of mycelia was placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded, the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at 12,000 x g for 60 sec, and the supernatant discarded. The cells were then resuspended in 0.25 ml TSE buffer containing 2 mg/ml lysozyme, and incubated at 37?C for 20 min with shaking and loaded into the AutoGen 540™ automated nucleic acid isolation instrument. Plasmid DNA was isolated using Cycle 106 (equipment software) according to manufacturer's instructions.<br>
Alternatively, 1.5 ml of mycelia were placed in 1.5 ml microcentrifuge tubes and the cells concentrated by centrifugation at 12,000 x g for 60 sec. The supernatant was discarded, the cells were resuspended in 1.0 ml 10.3% sucrose and concentrated by centrifugation at 12,000 x g for 60 sec, and the supernatant discarded. The cells were resuspended in 0.5 ml TSE buffer containing 2 mg/ml lysozyme, and incubated at 37°C for 15-30 min. After incubation, 0.25 ml alkaline SDS (0.3N NaOH, 2% SDS) was added and the cells incubated at 55°C for 15-30 min or until the solution was clear. Sodium acetate (0.1 ml, 3M, pH 4.8) was added to the DNA solution, which was then incubated on ice for 10 min. The DNA samples were centrifuged at 14,000 rpm for 10 min at 5°C. The supernatant was removed to a clean tube, and 0.2 ml phenol :chloroform (50% phenol :50% chloroform) was added and gently mixed. The DNA solution was centrifuged at 14,000 rpm for 10 min at 5°C and the upper layer removed to a clean Eppendorf tube. Isopropanol (0.75 ml) was added, and the solution was gently mixed and then incubated at rm temp for 20 min.    The DNA solution was<br><br>
centrifuged at 14,000 rpm for 15 min at 5°C, the supernatant removed, and the DNA pellet was washed with 70% ethanol, dried, and resuspended in TE buffer.<br>
7.1.4. Plasmid DNA Isolation From E. coli A single transformed E. coli colony was inoculated into 5 ml Luria-Bertani (LB)<br>
E medium (Bacto-Tryptone -10 g, Bacto-yeast extract - 5 g, and NaCI -10 g in 1 liter dH20, pH 7.0, autociaved -at 121°C for 25 min, and supplemented with 100 [ig/ml ampicillin). The culture was incubated overnight, and a 1 ml aliquot placed in a 1.5 ml microcentrifuge tube. The culture samples were loaded into the AutoGen 540™ automated nucleic acid isolation instrument and plasmid DNA was isolated using Cycle 3 (equipment software) according to<br>
     manufacturer's instructions.<br>
7.1.5.    Preparation And Transformation Of S. avermitilis Protoplasts<br>
Single colonies of s. avermitilis were isolated on 1/2 strength YPD-6.   The mycelia<br>
were used to inoculate 10 ml of TSB medium in a 25 mm x 150 mm tube, which was then   incubated with shaking (300 rpm) at 28°C for 48 hrs. One ml of mycelia was used to inoculate 50 ml YEME medium.   YEME medium contains per liter:   Difco Yeast Extract - 3 g; Difco Bacto-peptone - 5 g; Difco Malt Extract - 3 g; Sucrose - 300 g. After autoclaving at 121°C for 25 min, the following were added: 2.5 M MgCI2   6H20 (separately autoclaved at 121°C for 25 min) - 2 ml; and glycine (20%) (filter-sterilized)- 25 ml. The mycelia were grown at 30°C for 48-72 hrs and harvested by centrifugation in a 50 ml centrifuge tube (Falcon) at 3,000 rpm for 20 min. The supernatant was discarded and the mycelia were resuspended in P buffer, which contains:   sucrose - 205 g; K2S04 - 0.25 g; MgCI2   6H20 - 2.02 g; H2o - 600 ml; K2P04 (0.5%) - 10 ml; trace element solution* - 20 ml; CaCI2   2H20 (3.68%) - 1Qo ml; and MES buffer (i.O M, pH 6.5) - 10 ml.   (*Trace element solution contains per liter: ZnCI2 - 40 mg; FeCI3 • 6H20 - 200 mg; CuCI2 • 2H20 - 10 mg; MnCI2 4H20 - 10 mg; Na2B407    10H20 - 10 mg; (NH4)6 Mo^     4H20 - 10 mg).   The pH was adjusted to 6.5, final volume was adjusted to 1 liter, and the medium was filtered hot through a 0.45 micron filter.<br>
The mycelia were pelleted at 3,000 rpm for 20 min, the supernatant was iscarded,<br>
and the mycelia were resuspended in 20 ml P buffer containing 2 mg/ml lysozyme.   The<br>
mycelia were incubated at 35°C for 15 min with shaking, and checked microscopically to determine extent of protoplast formation. When protoplast formation was complete, the protoplasts were centrifuged at 8,000 rpm for 10 min. The supernatant was removed and the protoplasts were resuspended in 10 ml P buffer.  The protoplasts were centrifuged at 8,000<br><br>
as described therein. Plasmid DNA was isolated from S. lividans transformants as described in Section 7.1.3, above.<br>
7.1.7. Fermentation Analysis Of S. avermitilis Strains S. avermitilis mycelia grown on 1/2 strength YPD-6 for 4-7 days were inoculated into 1x6 inch tubes containing 8 ml of preform medium and two 5 mm glass beads. Preform medium contains: soluble starch (either thin boiled starch or KOSO, Japan Corn Starch Co.. Nagoya) • 20 g/L; Pharmamedia - 15 g/L; Ardamine pH - 5 g/L (Champlain Ind., Clifton, NJ); CaC03 - 2 g/L; 2x bcfa ("bcfa" refers to branched chain fatty acids) containing a final concentration in the medium of 50 ppm 2-(+/-)-methyl butyric acid, 60 ppm isobutyric acid, and 20 ppm isovaleric acid. The pH was adjusted to 7.2, and the medium was autoclaved at 121°Cfor 25min.<br>
The tube was shaken at a 17° angle at 215 rpm at 29°C for 3 days. A 2-ml aliquot of the seed culture was used to inoculate a 300 ml Erlenmeyer flask containing 25 ml of production medium which contains: starch (either thin boiled starch or KOSO) - 160 g/L; Nutrisoy (Archer Daniels Midland, Decatur, IL) -10 g/L; Ardamine pH -10 g/L; K2HP04 - 2 g/L; MgS04.4H20 - 2 g/L; FeS04.7H20 - 0.02 g/L; MnCI2 - 0.002 g/L; ZnS04.7H20 - 0.002 g/L; CaC03 -14 g/L; 2x bcfa (as above); and cyclohexane carboxylic acid (CHC) (made up as a 20% solution at pH 7.0) - 800 ppm. The pH was adjusted to 6.9, and the medium was autoclaved at 121 °C for 25 min.<br>
After inoculation, the flask was incubated at 29°C for 12 days with shaking at 200 rpm. After incubation, a 2 ml sample was withdrawn from the flask, diluted with 8 ml of methanol, mixed, and the mixture centrifuged at 1,250 x g for 10 min to pellet debris. The supernatant was then assayed by HPLC using a Beckman Ultrasphere ODS column (25 cm x 4.6 mm ID) with a flow rate of 0.75 ml/min and detection by absorbance at 240 nm. The mobile phase was 86/8.9/5.1 methanol/water/ acetonitrile.<br>
7.1.8. Isolation Of S. avermitilis PKS Genes<br>
A cosmid library of S. avermitilis (ATCC 31272, SC-2) chromosomal DNA was prepared and hybridized with a ketosynthase (KS) probe made from a fragment of the Saccharopolyspora erythraea polyketide synthase (PKS) gene. A detailed description of the preparation of cosmid libraries can be found in Sambrook et al., 1989, above. A detailed description of the preparation of Streptomyces chromosomal DNA libraries is presented in Hopwood et al., 1985, above. Cosmid clones containing ketosynthase-hybridizing regions were identified by hybridization to a 2.7 Kb Nodel/Eco47lll fragment from pEX26 (kindly supplied by Dr. P. Leadlay, Cambridge, UK).   Approximately 5 ng of pEX26 were digested<br><br>
using A/del and Eco47lll.   The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts, Rockland, ME).   The 2.7 Kb A/del/Eco47lll fragment was excised from the gel after electrophoresis and the DNA recovered from the gel using GELase™ from Epicentre Technologies using the Fast Protocol.  The 2.7 Kb A/del/Eco47lll fragment    was    labeled    with     [α-32PJdCTP     (deoxycytidine     5-triphosphate,     tetra (triethylammonium) salt,  [alpha-32?]-)  (NEN-Dupont,  Boston,  MA)  using the  BRL  Nick Translation System (BRL Life Technologies, Inc., Gaithersburg, MD) following the supplier's instructions. A typical reaction was performed in 0.05 ml volume. After addition of 5 μl Stop buffer, the labeled DNA was separated from unincorporated nucleotides using a G-25 Sephadex Quick Spin™ Column (Boehringer Mannheim) following supplier's instructions. Approximately 1,800 cosmid clones were screened by colony hybridization.   Ten clones were identified that hybridized strongly to the Sacc. erythraea KS probe.   E. coli<br>
colonies containing cosmid DNA were grown in LB liquid medium and cosmid DNA was isolated from each culture in the AutoGen 540™ automated nucleic acid isolation instrument using Cycle 3 (equipment software) according to manufacturer's instructions.   Restrict on endonuclease mapping and Southern blot hybridization analyses revealed that five of the clones contained overlapping chromosomal regions.    An  S. avermitilis genomic SamHl restriction map of the five cosmids (i.e., pSE65, pSE66, pSE67, pSE68, pSE69) was constructed by analysis of overlapping cosmids and hybridizations (FIGURE 4).<br>
7.1.9.    identification Of DNA That Modulates Avermectin B2:B1 Ratios And Identification Of An aveC ORF<br>
The following methods were used to test subcloned fragments derived from the<br>
pSE66 cosmid clone, for their ability to modulate avermectin B2.B1 ratios in AveC mutants. pSE66 (5 μg) was digested with Sacl and SamHl.   The reaction mixture was loaded on a 0.8% SeaPlaque™ GTG agarose gel (FMC BioProducts), a 2.9 Kb Sacl/BamHI fragment was excised from the gebafter electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol.   Approximately 5 μg of the shuttle vector pWHM3 (Vara et al., 1989, J. Bacteriol. 171:5872-5881) was digested with Sacl and SamHl.  About 0.5 μg of the 2.9 Kb insert and 0.5 ug of digested pWHM3 were mixed together and incubated overnight with 1 unit of ligase (New England Biolabs, Inc., Beverly, MA) at 15°C, in a total volume of 20 μl, according to supplier's instructions. After incubation, 5 μl of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to<br>
transform competent E. coli DH5a cells (BRL) according to manufacturer's instructions.<br><br>
Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the 2.9 Kb Sacl/SamHI insert was confirmed by restriction analysis. This plasmid was designated as pSE119.<br>
Protoplasts of S. avermitilis strain 1100-SC38 (Pfizer in-house strain) were prepared and transformed with pSE119 as described in Section 7.1.5 above. Strain 1100-SC38 is a mutant that produces significantly more of the avermectin cyclohexyl-B2 form compared to avermectin cyclohexyl-B1 form when supplemented with cyclohexane carboxylic acid (B2:B1 of about 30:1). pSE119 used to transform S. avermitilis protoplasts was isolated from either E. co//strain GM2163. (obtained from Dr. B. J. Bachmann, Curator, E. co//Genetic Stock Center, Yale University), E. coli strain DM1 (BRL), or S. lividans strain TK64. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE119 produced an altered ratio of avermectin cyclohexyl-B2:cyclohexyl-B1 of about 3.7:1 (TABLE 2).<br>
Having established that pSE119 was able to modulate avermectin B2:B1 ratios in an AveC mutant, the insert DNA was sequenced. Approximately 10 μg of pSE119 were isolated using a plasmid DNA isolation kit (Qiagen, Valencia, CA) following manufacturer's instructions, and sequenced using an ABI 373A Automated DNA Sequencer (Perkin Elmer, Foster City, CA). Sequence data was assembled and edited using Genetic Computer Group programs (GCG, Madison, Wl). The DNA sequence and the aveC ORF are presented in FIGURE 1 (SEQ ID NO:1).<br>
A new plasmid, designated as pSE118, was constructed as follows. Approximately 5 jig of pSE66 was digested with Sph\ and BamHI. The reaction mixture was loaded on a 0.8% SeaPlaque GTG agarose gel (FMC BioProducts), a 2.8 Kb Sphl/BamHI fragment was excised from the gel after electrophoresis, and the DNA was recovered from the gel using GELase™ (Epicentre Technologies) using the Fast Protocol. Approximately 5 μg of the shuttle vector pWHM3 was digested with Sph\ and BamHI. About 0.5 μg of the 2.8 Kb insert and 0.5 μg of digested pWHM3 were mixed together and incubated overnight with 1 unit of ligase (New England Biolabs) at 15°C in a total volume of 20 μl according to supplier's instructions. After incubation, 5 μl of the ligation mixture was incubated at 70°C for 10 min, cooled to rm temp, and used to transform competent E. coli DH5cc cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the 2.8 Kb SphllBamH\ insert was confirmed by restriction analysis. This plasmid was designated as<br><br>
pSE118.  The insert DNA in pSE118 and pSE119 overlap by approximately 838 nucleotides (FIGURE 4).<br>
Protoplasts of S. avermitilis strain 1100-SC38 were transformed with pSE118 as above. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. Transformants of S. avermitilis strain 1100-SC38 containing pSE118 were not altered in the ratios of avermectin cyclohexyl-B2: avermectin<br>
cyclohexyl-B1 compared to strain 1100-SC38 (TABLE 2).<br>
7.1.10.   PCR Amplification Of The aveC Gene From S. avermitilis Chromosomal DNA<br>
A -1.2 Kb fragment containing the aveC ORF was isolated from S. avermitilis chromosomal DNA by PCR amplification using primers designed on the basis of the aveC nucleotide sequence obtained above. The PCR primers were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward primer was: 5- CACGAAACCGGACACAC-3' (SEQ ID NO:6); and the leftward primer was: 5'- CATGATCGCTGAACCGAG-3' (SEQ ID NO:7). The PCR reaction was carried out with Deep Vent™ polymerase (New England Biolabs) in buffer provided by the manufacturer, and in the presence of 300μM dNTP, 10% glycerol, 200 pmol of each primer, 0.1 μg template, and 2.5 units enzyme in a final volume of 100 μl, using a Perkin-Elmer Cetus thermal cycler. The thermal profile of the first cycle was 95°C for 5 min (denaturation step), 60°C for 2 min (annealing step), and 72°C for 2 min (extension step). The subsequent 24 cycles had a similar thermal profile except that the denaturation step was shortened to 45 sec and the annealing step was shortened to 1 min.<br>
The PCR product was electrophoresed in a 1 % agarose gel and a single DNA band of -1.2 Kb was detected. This DNA was purified from the gel, and ligated with 25 ng of linearized, blunt pCR-Blunt vector (Invitrogen) in a 1:10 molar vector-to-insert ratio following manufacturer's instructions. The ligation mixture was used to transform One Shot™ Competent E. coli cells (Invitrogen) following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.2 Kb insert was confirmed by restriction analysis. This plasmid was designated as pSE179.<br>
The insert DNA from pSE179 was isolated by digestion with BamHI/Xbal, separated by electrophoresis, purified from the gel, and ligated with shuttle vector pWHM3, which had also been digested with BamH\IXba\, in a total DNA concentration of 1 μg in a 1:5 molar vector-to-insert ratio. The ligation mixture was used to transform competent E. coli DH5ct cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the -1.2 Kb insert was confirmed by restriction<br><br>
analysis. This plasmid, which was designated as pSE186 (FIGURE 2, ATCC 209604), was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transformants.<br>
7.2. Results<br>
A 2.9 Kb Sac\IBamH\ fragment from pSE119 was identified that, when transformed into S. avermitilis strain 1100-SC38, significantly altered the ratio of B2:B1 avermectin production. S. avermitilis strain 1100-SC38 normally has a B2:B1 ratio of about 30:1, but when transformed with a vector comprising the 2.9 Kb Sacl/BamHI fragment, the ratio of B2.B1 avermectin decreased to about 3.7:1. Post-fermentation analysis of transformant cultures verified the presence of the transforming DNA.<br>
The 2.9 Kb pSE119 fragment was sequenced and a -0.9 Kb ORF was identified (FIGURE 1) (SEQ ID NO:1), which encompasses a Pst\ISph\ fragment that had previously been mutated elsewhere to produce B2 products only (Ikeda et a/., 1995, above). A comparison of this ORF, or its corresponding deduced polypeptide, against known databases (GenEMBL, SWISS-PROT) did not show any strong homology with known DNA or protein sequences.<br>
TABLE 2 presents the fermentation analysis of S. avermitilis strain 1100-SC38 transformed with various plasmids.<br>
TABLE 2<br><br>
S. avermitilis strain (transforming plasmid)	No. Transformants Tested	Avg. B2:B1<br>
Ratio<br>
1100-SC38(none)	9	30.66<br>
1100-SC38(pWHM3)	21	31.3<br>
1100-SC38(pSE119)	12	3.7<br><br>
1100-SC38(pSE118)	12	30.4<br>
1100-SC38(pSE185)	14	27.9<br>
8. EXAMPLE: CONSTRUCTION OF S. AVERMITILIS AveC MUTANTS<br>
This example describes the construction of several different S. avermitilis AveC<br>
mutants using the compositions and methods described above.   A general description of<br>
techniques for introducing mutations into a gene in Streptomyces is described by Kieser and<br><br>
Hopwood, 1991, Meth. Enzym. 204:430-458. A more detailed description is provided by Anzai et a/., 1988, J. Antibiot. XLI(2):226-233, and by Stutzman-Engwall et a/.. 1992, J. Bacterid. 174(1):144-154. These references are incorporated herein by reference in their entirety.<br>
8.1. Inactivation Of The S. avermitilis aveC Gene AveC mutants containing inactivated aveC genes were constructed using several methods, as detailed below.<br>
in the first method, a 640 bp Sph\lPst] fragment internal to the aveC gene in pSE119 (plasmid described in Section 7.1.9, above) was replaced with the ermE gene (for erythromycin resistance) from Sacc. erythraea. The ermE gene was isolated from plJ4026 (provided by the John Innes Institute, Norwich, U.K.; see also Bibb et a/., 1985, Gene 41:357-368) by restriction enzyme digestion with Bg/ll and EcoRI, followed by electrophoresis, and was purified from the gel. This -1.7 Kb fragment was ligated into pGEM7Zf (Promega) which had been digested with BamHl and EcoRI, and the ligation mixture transformed into competent E. coli DH5a cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the -1.7 Kb insert was confirmed by restriction analysis. This plasmid was designated as pSE27.<br>
pSE118 (described in Section 7.1.9, above) was digested with Sphl and BamHl, the digest electrophoresed, and the -2.8 Kb Sphl/BamHI insert purified from the gel. pSE119 was digested with Psti and EcoRI, the digest electrophoresed, and the -1.5 Kb Psfl/EcoRI insert purified from the gel. Shuttle vector pWHM3 was digested with BamHl and EcoRI. pSE27 was digested with Pst\ and Sphl, the digest electrophoresed, and the -1.7 Kb Psf/Sp/?l insert purified from the gel. All four fragments (;.e., -2.8 Kb, -1.5Kb, -7.2Kb, -1.7 Kb) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. coli DH5oc cells following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE180 (FIGURE 3; ATCC 209605).<br>
pSE180 was transformed into S. Iividans TK64 and transformed colonies identified by resistance to thiostrepton and erythromycin. pSE180 was isolated from S. Iividans and used to transform S. avermitilis protoplasts. Four thiostrepton resistant S. avermitilis transformants were identified, and protoplasts were prepared and plated under non-selective conditions on RM14 media. After the protoplasts had regenerated; single colonies were screened for the presence of erythromycin resistance and the absence of thiostrepton resistance, indicating<br><br>
chromosomal integration of the inactivated aveC gene and loss of the free replicon. One Ermr Thios transformant was identified and designated as strain SE180-11. Total chromosomal DNA was isolated from strain SE180-11, digested with restriction enzymes SamHI, HindWU Pst\, or Sphl, resolved by electrophoresis on a 0.8% agarose gel, transferred to nylon membranes, and hybridized to the ermE probe. These analyses showed that chromosomal integration of the ermE resistance gene, and concomitant deletion of the 640 bp Pstt/Sphl fragment had occurred by a double crossover event. HPLC analysis of fermentation products of strain SE180-11 showed that normal avermectins were no longer produced (FIGURE 5A).<br>
In a second method for inactivating the aveC gene, the 1.7 Kb ermE gene was<br>
removed from the chromosome of S. avermitilis strain SE180-11, leaving a 640 bp PstllSphl deletion in the aveC gene. A gene replacement plasmid was constructed as follows: pSE180 was partially digested with Xbal and an -11.4 Kb fragment purified from the gel.  The ~11.4 Kb band lacks the 1.7 Kb ermE resistance gene. The DNA was then ligated and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE184, was transformed into £ coli DM1, and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis  strain   SE180-11.      Protoplasts   were   prepared   from   thiostrepton   resistant transformants of strain SE180-11 and were plated as single colonies on RM14.   After the protoplasts had regenerated,  single colonies were screened for the absence of both erythromycin resistance and thiostrepton resistance, indicating chromosomal integration of the<br>
inactivated aveC gene and loss of the free replicon containing the ermE gene.   One Erms Thios transformant was identified and designated as SE184-1-13.  Fermentation analysis of SE184-1-13 showed that normal avermectins were not produced and that SE184-1-13 had the same fermentation profile as SE180-11.<br>
In a third method for inactivating the aveC gene, a frameshift was introduced into the chromosomal aveC gene by adding two G's after the C at nt position 471 using PCR, thereby creating a BspE1 site. The presence of the engineered BspE1 site was useful in detecting the gene replacement event. The PCR primers were designed to introduce a frameshift mutation into the aveC gene, and were supplied by Genosys Biotechnologies, Inc. The rightward primer was: 5'-GGTTCCGGATGCCGTTCTCG-3' (SEQ ID NO:8) and the leftward primer was: 5'-AACTCCGGTCGACTCCCCTTC-3" (SEQ ID NO:9). The PCR conditions were as described in Section 7.1.10 above. The 666 bp PCR product was digested with Sphl to give<br><br>
two fragments of 278 bp and 388 bp, respectively. The 388 bp fragment was purified from the gel.<br>
The gene replacement plasmid was constructed as follows: shuttle vector pWHM3 was digested with EcoRl and BamHl. pSE119 was digested with SamHI and Sphl, the digest electrophoresed, and a -840 bp fragment was purified from the gel. pSE119 was digested with EcoRl and Xmn\, the digest was resolved by electrophoresis, and a -1.7 Kb fragment was purified from the gel. All four fragments (i.e., -7.2 Kb, -840 bp, -1.7 Kb, and 388 bp) were ligated together in a 4-way ligation. The ligation mixture was transformed into competent E. coli DH5cx cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid, which was designated as pSE185, was transformed into E. coli DM1 and plasmid DNA isolated from ampicillin resistant transformants. This plasmid was used to transform protoplasts of S. avermitilis strain 1100-SC38. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products. pSE185 did not significantly alter the B2:B1 avermectin ratios when transformed into S. avermitilis strain 1100-SC38 (TABLE 2).pSE185 was used to transform protoplasts of S. avermitilis to generate a frameshift<br>
mutation in the chromosomal aveC gene. Protoplasts were prepared from thiostrepton<br>
resistant transformants and plated as single colonies on RM14. After the protoplasts had<br>
regenerated, single colonies were screened for the absence of thiostrepton resistance.<br>
Chromosomal DNA from thiostrepton sensitive colonies was isolated and screened by PCRfor the presence of the frameshift mutation integrated into the chromosome. The PCR primers were designed based on the aveC nucleotide sequence, and were supplied by Genosys Biotechnologies,     Inc.     (Texas).	The     rightward     PCR     primer     was:	5'- GCAAGGATACGGGGACTAC-3' (SEQ ID NO:10) and the leftward PCR primer was: 5-GAACCGACCGCCTGATAC-3* (SEQ ID NO:11), and the PCR conditions were as described in Section 7.1.10 above. The PCR product obtained was 543 bp and, when digested with BspE1, three fragments of 368 bp, 96 bp, and 79 bp were observed, indicating chromosomal integration of the inactivated aveC gene and loss of the free replicon.<br>
Fermentation analysis of S. avermitilis mutants containing the frameshift mutation in the aveC gene showed that normal avermectins were no longer produced, and that these mutants had the same fermentation HPLC profile as strains SE180-11 and SE184-1-13. One Thios transformant was identified and designated as strain SE185-5a.<br><br>
Additionally, a mutation in the aveC gene that changes nt position 520 from G to A, which results in changing the codon encoding a tryptophan (W) at position 116 to a termination codon, was produced. An S. avermitilis strain with this mutation did hot produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, and SE185-5a.<br>
Additionally, mutations in the aveC gene that change both: (i) nt position 970 from G to A, which changes the amino acid at position 266 from a glycine (G) to an aspartate (D), and (ii) nt position 996 from T to C, which changes the amino acid at position 275 from tyrosine (Y) to histidine (H), were produced. An S. avermitilis strain with these mutations (G256D/Y275H) did not produce normal avermectins and had the same fermentation profile as strains SE180-11, SE184-1-13, and SE185-5a.<br>
The S. avermitilis aveC inactivation mutant strains SE180-11, SE184-1-13, SE185- 5a, and others provided herewith, provide screening tools to assess the impact of other mutations in the aveC gene. pSE186, which contains a wild-type copy of the aveC gene, was transformed into E. coli DM1, and plasmid DNA was isolated from ampicillin resistant transformants.  This pSE186 DNA was used to transform protoplasts of S. avermitilis strain SE180-11.    Thiostrepton resistant transformants of strain SE180-11 were isolated, the<br>
presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products.  The presence of the functional aveC gene in trans was able to restore normal avermectin production to strain SE180-11 (FIGURE 5B).<br>
8.2. Analysis Of Mutations In The aveC Gene That Alter Class B2:B1 Ratios<br>
As described above, S. avermitilis strain SE180-11 containing an inactive aveC gene was complemented by transformation with a plasmid containing a functional aveC gene (pSE186). Strain SE180-11 was also utilized as a host strain to characterize other mutations in the aveC gene, as described below.<br>
Chromosomal DNA was isolated from strain 1100-SC38, and used as a template for PCR amplification of the aveC gene. The 1.2 Kb ORF was isolated by PCR amplification using primers designed on the basis of the aveC nucleotide sequence. The rightward primer was SEQ ID NO:6 and the leftward primer was SEQ ID NO:7 (see Section 7.1.10, above). The PCR and subcloning conditions were as described in Section 7.1.10. DNA sequence analysis of the 1.2 Kb ORF shows a mutation in the aveC gene that changes nt position 337 from C to T, which changes the amino acid at position 55 from serine (S) to phenylalanine (F). The aveC gene containing the S55F mutation was subcloned into pWHM3 to produce a<br><br>
plasmid which was designated as pSE187, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the aveC gene encoding a change at amino acid residue 55 (S55F) was able to restore norma/ avermectin production to strain SE180-11 (Fig. 5C); however, the cylohexyl   B2:cyclohexyl B1 ratio was about 26:1, as compared to strain SE180-11 transformed with pSE186, which had a ratio of B2:B1 of about 1.6:1 (TABLE 3), indicating that the single mutation (S55F) modulates the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1.<br>
Another mutation in the aveC gene was identified that changes nt position 862 from G to A, which changes the amino acid at position 230 from glycine (G) to aspartate (D). An S. avermitilis strain having this mutation (G230D) produces avermectins at a B2:B1 ratio of about 30:1.<br>
8.3. Mutations That Reduce The B2:B1 Ratio Several  mutations were constructed that reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1, as follows.<br>
A mutation in the aveC gene was identified that changes nt position 588 from G to A, which changes the amino acid at position 139 from alanine (A) to threonine (T). The aveC gene containing the A139T mutation was subcloned into pWHM3 to produce a plasmid which was designated pSE188, and which was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 139 (A139T) was able to restore avermectin production to strain SE180-11 (FIGURE 5D); however, the B2:B1 ratio was about 0.94:1, indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1. This result was unexpected because published results, as well as the results of mutations described above, have only demonstrated either inactivation of the aveC gene or increased production of the B2 form of avermectin relative to the B1 form (TABLE 3).<br>
Because the A139T mutation altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed that encoded a threonine instead of a serine at amino acid position 138. Thus, pSE186 was digested with EcoRI and cloned into pGEM3Zf (Promega) which had been digested with EcoRI. This plasmid, which was designated as pSE186a, was digested with Apa\ and Kpn\, the DNA fragments separated on an agarose gel,<br><br>
and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel.  The -1.2 Kb insert DNA from pSE186 was used as a PCR template to introduce a single base change at nt position 585. The PCR primers were designed to introduce a mutation at nt position 585, and were supplied by Genosys Biotechnologies, Inc. (Texas). The rightward PCR primer was: 5'- GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCCCTGGCGACG-3' (SEQ ID NO: 12); and the leftward PCR primer was:   5'-GGAACCGACCGCCTGATACA-3, (SEQ ID NO:13). The PCR reaction was carried out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 μM dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.0 M GC-Melt and 1 unit KlenTaq Polymerase Mix in a final volume of 50 μl The thermal profile of the first cycle was 94CC for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 295 bp was digested with Apal and Kpnl to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel.  All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation. The ligation mixture was transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE198. pSE198 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE199, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the mutated aveC gene encoding a change at amino acid residue 138 (S138T) was able to restore normal avermectin production to strain SE1 so¬μl; however, the B2:B1 ratio was 0.88:1 indicating that this mutation reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3). This B2:B1 ratio is even lower than the 0.94:1 ratio observed with the A139T mutation produced by transformation of strain SE180-11 with pSE188, as described above.<br>
Another mutation was constructed to introduce a threonine at both amino acid positions 138 and 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template.<br><br>
The PCR primers were designed to introduce mutations at nt positions 585 and 588, and were supplied by Genosys Biotechnologies, Inc. (Texas).   The rightward PCR primer was:   5'-GGGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGCTGGCGACGACC-3'     (SEQ      ID NO:14); and the leftward PCR primer was:   5'-GAACATCACGGCATTCACC-3' (SEQ ID<br>
NO:15). The PCR reaction was performed using the conditions described immediately above in this Section. A PCR product of 449 bp was digested with Apa\ and Kpn\ to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. pSE186a was digested with Apa\ and Kpn\, the DNA fragments separated on an agarose gel, and two fragments of -3.8 Kb and -0.4 Kb were purified from the gel.  All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation, and the ligation mixture was transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE230.pSE230 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI, and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE231, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of SE180-11 were, isolated, the presence of erythromycin resistance was determined, and Thior Ermr transfonnants were analyzed by fermentation. The presence of the double mutated aveC gene, encoding S138T/A139T, was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.84:1 showing that this mutation further reduces the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3), over the reductions provided by transformation of strain SE180-11 with pSE188 or pSE199, as described above.<br>
Another mutation was constructed to further reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1. Because the S138T/A139T mutations altered the B2:B1 ratios in the more favorable B1 direction, a mutation was constructed to introduce a threonine at amino acid position 138 and a phenylalanine at amino acid position 139. The -1.2 Kb insert DNA from pSE186 was used as a PCR template. The PCR primers were designed to introduce mutations at nt positions 585 (changing a T to A), 588 (changing a G to T), and 589 (changing a C to T), and were supplied by Genosys Biotechnologies, Inc. (Texas).   The<br><br>
rightward PCR primer was:    5'-GGGGCGGGCCCGGGTGCGGAGGCGGAAATGCCGC TGGCGACGTTC-3"   (SEQ   ID   NO:25);   and   the   leftward   PCR   primer   was:       5'-<br>
GGAACATCACGGCATTCACC-3' (SEQ ID N0:15). The PCR reaction was carried    out using an Advantage GC genomic PCR kit (Clonetech Laboratories, Palo Alto, CA) in buffer provided by the manufacturer in the presence of 200 |iM dNTPs, 200 pmol of each primer, 50 ng template DNA, 1.1 mM Mg acetate, 1.0 M GC-Melt and 1 unit Tth DNA Polymerase in a final volume of 50 \A.  The thermal profile of the first cycle was 94°C for 1 min; followed by 25 cycles of 94°C for 30 sec and 68°C for 2 min; and 1 cycle at 68°C for 3 min. A PCR product of 449 bp was digested with Apa\ and Kpn\ to release a 254 bp fragment, which was resolved by electrophoresis and purified from the gel. All three fragments (-3.8 Kb, -0.4 Kb and 254 bp) were ligated together in a 3-way ligation.   The ligation mixture was transformed into competent E.  coli DH5a cells.     Plasmid  DNA was  isolated from ampicillin  resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid was designated as pSE238. pSE238 was digested with EcoRI, cloned into pWHM3, which had been digested with EcoRI. and transformed into E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis. This plasmid, which was designated as pSE239, was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. The presence of the double mutated aveC gene encoding S138T/A139F was able to restore normal avermectin production to strain SE180-11; however, the B2:B1 ratio was 0.75:1 showing that this mutation further reduced the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 (TABLE 3) over the reductions provided by transformation of strain SE180-11 with pSE188, pSE199, or pSE231 as described above.<br><br>
TABLE 3<br><br>
S. avermitilis strain (transforming plasmid)	No.<br>
Transformants<br>
Tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	30	0	0	0<br>
SE180-11(pWHM3)	30	0	0	0<br>
SE180-11 (pSE186)	26	222	140	1.59<br>
SE180-11(pSE187)	12	283	11	26.3<br>
SE180-11 (pSE188)	24	193	206	0.94<br>
SE180-11 (pSE199)	18	155	171	0.88<br>
SE180-11(pSE231)	6	259	309	0.84<br>
SE180-11(pSE239) .	20                         184	242	0.75<br>
Additional mutations were constructed to further reduce the amount of cyclohexyl-B2 produced relative to cyclohexyl-B1 using the technique of DNA shuffling as described in Stemmer, 1994, Nature 370:389-391; and Stemmer, 1994, Proc. Natl. Acad. Sci. USA 91:10747-10751; and further described in United States Patents 5605793, 5811238, 5830721, and 5837458.<br>
DNA shuffled plasmids containing mutated aveC genes were transformed into competent dam dcm E. coli cells. Plasmid DNA was isolated from ampicillin resistant transformants, and used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated and screened for the production of avermectins with a cyclohexyl-B2:cyclohexyl-B1 ratio of 1:1 or less. The DNA sequence of plasmid DNA from SE180-11 transformants producing avermectins with a B2:B1 ratio of 1:1 or less was determined.<br>
Eight transformants were identified that produced reduced amounts of cyclohexyl-B2 relative to cyclohexyl-B1. The lowest B2:B1 ratio achieved among these transformants was 04:1 (TABLE 4). Plasmid DNA was isolated from each of the eight transformants and the DNA sequence determined to identify the mutations in the aveC gene. The mutations are as follows.<br>
pSE290 contains 4 nucleotide mutations at nt position 317 from T to A, at nt position 353 from C to A, at nt position 438 from G to A, and at nt position 1155 from T to A. The nucleotide change at nt position 317 changes the amino acid at AA position 48 from D to E and the nucleotide change at nt position 438 changes the amino acid at AA position 89 from A to T. The B2:B1 ratio produced by cells carrying this plasmid was 0.42:1 (TABLE 4).<br><br>
pSE291 contains 4 nucleotide mutations at nt position 272 from G to A, at nt position 585 from T to A, at nt position 588 from G to A, and at nt position 708 from G to A. The nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, and the nucleotide change at nt position 708 changes the amino acid at AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.57:1 (TABLE 4).<br>
pSE292 contains the same four nucleotide mutations as pSE290. The B2:B1 ratio produced by cells carrying this plasmid was 0.40:1 (TABLE 4).<br>
pSE293 contains 6 nucleotide mutations at nt 24 from A to G, at nt position 286 from A to C, at nt position 497 from T to C, at nt position 554 from C to T, at nt position 580 from T to C, and at nt position 886 from A to T. The nucleotide change at nt position 286 changes the amino acid at AA position 38 from Q to P, the nucleotide change at nt position 580 changes the amino acid at AA position 136 from L to P, and the nucleotide change at nt position 886 changes the amino acid at AA position 238 from E to D. The B2:B1 ratio produced by cells carrying this plasmid was 0.68:1 (TABLE 4).<br>
pSE294 contains 6 nucleotide mutations at nt 469 from T to C, at nt position 585 from T to A, at nt position 588 from G to A, at nt position 708 from G to A. at nt position 833 from C to T, and at nt position 1184 from G to A. In addition, nts at positions 173, 174, and 175 are deleted. The nucleotide change at nt position 469 changes the amino acid at AA position 99 from F to S, the nucleotide change at nt position 585 changes the amino acid at AA position 138 from S to T, the nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T. and the nucleotide change at nt position 708 changes the amino acid from AA position 179 from G to S. The B2:B1 ratio produced by cells carrying this plasmid was 0.53:1 (TABLE 4).<br>
pSE295 contains 2 nucleotide mutations at nt 588 from G to A and at nt 856 from T to C. The nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T and the nucleotide change at nt position 856 changes the amino acid at AA position 228 from M to T. The B2.B1 ratio produced by cells carrying this plasmid was 0.80:1 (TABLE 4).<br>
pSE296 contains 5 nucleotide mutations at nt position 155 from T to C, at nt position 505 from G to T, at nt position 1039 from C to T, at nt position 1202 from C to T, and at nt position 1210 from T to C. The nucleotide change at nt position 505 changes the amino acid at AA position 111 from G to V and the nucleotide change at nt position 1039 changes the<br><br>
amino acid at AA position 289 from P to L. The B2:B1 ratio produced by cetls carrying this plasmid was 0.73:1 (TABLE 4).<br>
pSE297 contains.4 nucleotide mutations at nt position 377 from G to T, at nt position 588 from G to A, at nt position 633 from A to G, and at nt position 1067 from A to T. The nucleotide change at nt position 588 changes the amino acid at AA position 139 from A to T, the nucleotide change at nt position 633 changes the amino acid at AA position 154 from K to E, and the nucleotide change at nt position 1067 changes the amino acid at AA position 298 from Q to H. The B2:B1 ratio produced by cells carrying this plasmid was 0.67:1 (TABLE 4).<br>
TABLE 4<br><br>
S. avermitilis strain (transforming piasmid)	No.<br>
Transformants<br>
Tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	4	0	0	0<br>
SE180-11(pWHM3)	4	0	0	0<br>
SE180-11 (pSE290)	4	87	208	0.42<br>
SE180-11 (pSE291)	4	106	185	0.57<br>
SE180-11 (pSE292)	4	91	231	0.40<br>
SE180-11 (pSE293)	4	123	180	0.68<br>
SE180-11 (pSE294)	4	68	129	0.53<br>
SE180-11 (pSE295)	4	217	271	0.80<br>
SE180-11 (pSE296)	1	135	186	0.73<br>
SE180-11 (pSE297)<br>
 	1	148	221	0.67<br>
9. EXAMPLE: CONSTRUCTION OF 5* DELETION MUTANTS<br>
  As explained in Section 5.1, above, the S. avermitilis nucleotide sequence shown in FIGURE 1 (SEQ ID NO:1) comprises four different GTG codons at bp positions 42, 174, 177 and 180 which are potential start sites. This section describes the construction of multiple deletions of the 5' region of the aveC ORF (FIGURE 1; SEQ ID NO:1) to help define which of these codons could function as start sites in the aveC ORF for protein expression.<br>
Fragments of the aveC gene variously deleted at the 5" end were isolated from S. avermitilis chromosomal DNA by PCR amplification. The PCR primers were designed based<br><br>
on the aveC DNA sequence, and were supplied by Genosys Biotechnologies, Inc. The rightward primers were 5'-AACCCATCCGAGCCGCTC-3' (SEQ ID NO:16) (D1F1); 5'-TCGGCCTGCCAACGAAC-3' (SEQ ID NO:17) (D1F2); 5'-CCAACGAACGTGTAGTAG-3' (SEQ ID NO:18) (D1F3); and 3'-TGCAGGCGTACGTGTTCAGC-3' (SEQ ID NO:19) (D2F2). The leftward primers were 5'-CATGATCGCTGAACCGA-3' (SEQ ID NO:20); 5'-CATGAT CGCTGAACCGAGGA-3 iSEQ )D ND:21); and P-AGGAGTGTGGTGCGTCTGGA-3 {SEQ. ID NO:22). The PCR reaction was carried out as described in Section 8.3, above.<br>
The PCR products were separated by electrophoresis in a 1 % agarose gel and single DNA bands of either -1.0 Kb or -1.1 Kb were detected. The PCR products were purified from the gel and ligated with 25 ng of linearized pCR2.1 vector (Invitrogen) in a 1:10 molar vector-to-insert ratio following the manufacturer's instructions. The ligation mixtures were used to transform One Shot™ Competent E. coli cells (Invitrogen) following manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the insert was confirmed by restriction analysis and DNA sequence analysis. These plasmids were designated as pSEl90 (obtained with primer D1F1), pSE191 (obtained with primer D1F2), pSE193 (obtained with primer D1F3), and pSE193 (obtained with primer D2F2).<br>
The insert DNAs were each digested with BamHUXbal, separated by electrophoresis, purified from the gel, and separately ligated with shuttle vector pWHM3, which had been digested with BamHI/Xbal, in a total DNA concentration of 1 μg in a 1:5 molar vector-to-insert ratio. The ligation mixtures were used to transform competent E. coli DH5a cells. Plasmid DNA was isolated from ampicillin resistant transformants and the presence of the insert was confirmed by restriction analysis. These plasmids, which were designated as pSE194 (D1F1), pSE195 (D1F2). PSE196 (D1F3), and pSE197 (D2F2), were each separately transformed into E. coli strain DM1, plasmid DNA isolated from ampicillin resistant transformants, and the presence of the correct insert confirmed by restriction analysis. This DNA was used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 'were isolated, the presence of erythromycin resistance was determined, and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products to determine which GTG sites were necessary for aveC expression. The results indicate that the GTG codon at position 42 can be eliminated without affecting aveC expression, since pSE194 PSE195, and pSE196, each of which lack the GTG site at position 42, but which all certain the three GTG sites at positions 174, 177, and 180, were each able to restore normal avermectin production when transformed into SE180-11. Nomal<br><br>
avermectin production was not restored when strain SE180-11 was transformed with pSE197, which lacks all four of the GTG sites (TABLE 5).<br>
TABLE 5<br><br>
S. avermitilis strain (transforming plasmid)	No. transformants tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	6	0	0	0<br>
SE180-11 (pWHM3),	6	0	0	0<br>
SE180-11 (pSE186)	6	241	152	1.58<br>
SE180-11 (pSE194)	6	35	15	2.43<br>
SE180-11 (pSE195)	6	74	38	1.97<br>
SE180-11 (pSE196)	6	328	208	1.58<br>
SE180-11(pSE197)	12	0	0	0<br>
10. EXAMPLE: CLONING OF aveC HOMOLOGS FROM S. HYGROSCOPICUS AND S. GRISEOCHROMOGENES<br>
The present invention allows aveC homolog genes from other avermectin- or milbemycin-producing species of Streptomyces to be identified and cloned. For example, a cosmid library of S. hygroscopicus (FERM BP-1901) genomic DNA was hybridized with the 1.2 Kb aveC probe from S. avermitilis described above. Several cosmid clones were identified that hybridized strongly. Chromosomal DNA was isolated from these cosmids, and a 4.9 Kb Kpn\ fragment was identified that hybridized with the aveC probe. This DNA was sequenced and an ORF (SEQ ID NO:3) was identified having significant homology to the aveC ORF of S. avermitilis. An amino acid sequence (SEQ ID NO:4) deduced from the S. hygroscopicus aveC homolog ORF is presented in FIGURE 6.<br>
In addition, a cosmid library of S. griseochromogenes genomic DNA was hybridized with the 1.2 Kb aveC probe from S. avermitilis described above. Several cosmid clones were identified that hybridized strongly. Chromosomal DNA was isolated from these cosmids, and a 5.4 Kb Psfl fragment was identified that hybridized with the aveC probe. This DNA was sequenced and an aveC homolog partial ORF was identified having significant homology to the aveC ORF of S. avermitilis. A deduced partial amino acid sequence (SEQ ID NO:5) is' presented in FIGURE 6.<br><br>
DNA and amino acid sequence analysis of the aveC homoiogs from S. hygroscopicus and S. griseochromogenes indicates that these regions share significant homology (-50% sequence identity at the amino acid level) both to each other and to the S. avermitilis aveC ORF and AveC gene product FIGURE 6).<br>
11. EXAMPLE: CONSTRUCTION OF A PLASMID WITH THE aveC GENE BEHIND THE ermE PROMOTER<br>
The 1.2 Kb aveC ORF from pSE186 was subcioned in pSE34, which is the shuttle vector pWHM3 having the 300 bp ermE promoter inserted as a Kpn\IBamH\ fragment in the Kpn\IBamH\ site of pWHM3 (see Ward et al, 1986, Mol. Gen. Genet. 203:468-478). pSE186 was digested with BamHl and Hindlll, the digest resolved by electrophoresis, and the 1.2 Kb fragment was isolated from the agarose gel and ligated with pSE34, which had been digested with fiamHI and H/ndlll. The ligation mixture was transformed into competent E coli DH5a cells according to manufacturer's instructions. Plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the 1.2 Kb insert was confirmed by restriction analysis. This plasmid, which was designated as pSE189, was transformed into E. coli DM1, and plasmid DNA isolated from ampicillin resistant transformants. Protoplasts of S. avermitilis strain 1100-SC38 were transformed with pSE189. Thiostrepton resistant transformants of strain 1100-SC38 were isolated and analyzed by HPLC analysis of fermentation products.<br>
S. avermitilis strain 1100-SC38 transformants containing pSE189 were altered in the ratios of avermectin.cyclohexyl-B2:avermectin cyclohexyl-B1 produced (about 3:1) compared to strain 1100-SC38 (about 34:1), and total avermectin production was increased approximately 2.4-fold compared to strain 1100-SC38 transformed with pSE119 (TABLE 6).<br>
pSE189 was also transformed into protoplasts of a wild-type S. avermitilis strain. Thiostrepton resistant transformants were isolated and analyzed by HPLC analysis of fermentation products. Total avermectins produced by S. avermitilis wild-type transformed with pSE189 were increased approximately 2.2-fold compared to wild-type S. avermitilis transformed with pSE119 (TABLE 6).<br>
TABLE 6<br><br>
S. avermitilis strain<br>
(transforming plasmid)	No. Trans¬formants Tested	Relative [B2]	Relative [B1]	Relative Total Avermectins	Avg. B2:B1 Ratio<br>
1100-SC38	6	155	4.8	176	33.9<br>
1100-SC38 (PSE119)	9	239	50.3	357	4.7<br><br>
S. avermitiiis strain<br>
(transforming plasmid)	No. Trans-<br>
forrnants<br>
Tested	Relative [B2]	Relative [B1]	Relative Total Avermectins	Avg. B2:B1 Ratio<br>
1100-SC38 (pSE189)	16	546	166	849	3.3<br>
wild type	6	59	42	113	1.41<br>
wild type (PSE119)	6	248	151	481	1.64<br>
wild type (PSE189)	5	545	345	1,071	1.58<br>
12. EXAMPLE: CHIMERIC PLASMID CONTAINING<br>
	SEQUENCES FROM BOTH S. AVERMITILIS aveC<br>
QRF AND S. HYGROSCOPICUS aveC HOMOLOG<br>
A hybrid plasmid designated as pSE350 was constructed that contains a 564 bp<br>
portion of the S. hygroscopicus aveC homolog replacing a 564 bp homologous portion of the S. avermitiiis aveC ORF (FIGURE 7), as follows.   pSE350 was constructed using a BsaAl i     restriction site that is conserved in both sequences (aveC position 225), and a Kpnl restriction site that is present in the S- avermitiiis aveC gene (aveC position 810).  The Kpnl site was introduced  into the  S.  hygroscopicus DNA by PCR using the  rightward  primer 5'- CTTCAGGTGTACGTGTTCG-3'    (SEQ    ID    NO:23)    and    the    leftward    primer    5- GAACTGGTACCAGTGCCC-3' (SEQ ID NO:24) (supplied by Genosys Biotechnologies) using PCR conditions described in Section 7.1.10, above.   The PCR product was digested with BsaAl and Kpnl, the fragments were separated by electrophoresis in a 1% agarose gel, and the 564 bp BsaAUKpnl fragment was isolated from the gel. pSE179 (described in Section 7.1.10,   above)  was   digested   with   Kpnl   and   HindlU,   the   fragments   separated   by electrophoresis in a 1% agarose gel, and a fragment of -4.5 Kb was isolated from the gel. pSE179 was digested with Hindlll and BsaAl, the fragments separated by electrophoresis in a<br>
1% agarose gel, and a -0.2 Kb BsaAI/H/ndlll fragment isolated from the gel.  The -4.5 Kb<br>
H/ndlll/Kpnl fragment, the -0.2 Kb BsaAI/W/ndlll fragment, and the 564 bp BsaAUKpnl<br>
fragment from S. hygroscopicus were ligated together in a 3-way ligation and the ligation<br>
mixture transformed into competent E. coli DH5a cells.   Plasmid DNA was isolated from<br>
ampicillin resistant transforniants and the presence of the correct insert was confirmed by<br>
restriction analysis using Kpnl and Aval. This plasmid was digested with Hindlll and Xoal to<br><br>
release the 1.2 Kb insert, which was then ligated with pWHM3 which had been digested with Hindlll and Xoal. The ligation mixture was transformed into competent £. coli DH5a cells, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis using H/ndlll and Aval. This plasmid DNA was transformed into E. coli DM1, plasmid DNA was isolated from ampicillin resistant transformants, and the presence of the correct insert was confirmed by restriction analysis and DNA sequence analysis. This plasmid was designated as pSE350 and used to transform protoplasts of S. avermitilis strain SE180-11. Thiostrepton resistant transformants of strain SE180-11 were isolated, the presence of erythromycin resistance was determined and Thior Ermr transformants were analyzed by HPLC analysis of fermentation products. Results show that transformants containing the S. avermitilis/S. hygroscopicus hybrid plasmid have an average B2:B1 ratio of about 109:1 (TABLE 7).<br>
TABLE 7<br><br>
S. avermitilis strain (transforming plasmid)	No. transformants tested	Relative B2 Cone.	Relative B1 Cone.	Avg. B2:B1 Ratio<br>
SE180-11 (none)	8	0	0	0<br>
SE180-11(pWHM3)	8	0	0	0<br>
SE180-11 (pSE350)	16	233	2	109<br>
DEPOSIT OF BIOLOGICAL MATERIALS<br>
The following biological material was deposited with the American Type Culture Collection (ATCC) at 12301 Parklawn Drive, Rockville, MD, 20852, USA, on January 29, 1998, and was assigned the following accession numbers:<br>
Plasmid	Accession No.<br>
plasmid pSE 180	209605<br>
plasmid pSE186	209604<br>
All patents, patent applications, and publications cited above are incorporated herein by reference in their entirety.<br>
The present invention is not to be limited in scope by the specific embodiment described herein, which are intended as single illustrations of individual aspects of the<br><br>
invention, and functionally equivalent methods and components are within the scope of the invention. Indeed, various modifications of the invention, in addition to those shown and described herein will become apparent to those skilled in the art .from the foregoing description and accompanying drawings. Such modifications are intended to fall within the scope of the appended claims.<br>
Reference   to   co-pending   application   IP:   732/MUM/2000   is   made therewith.<br><br>
WE CLAIM:<br>
1.        A polynucleotide molecule comprising a nucleotide  sequence that is otherwise the same as the Streptomyces avermitilis aveC allele, the S. avernitilis AveC gene product-encoding sequence of plasmid pSE186 (ATCC 209604) or the nucleotide sequence of the aveC ORF of S. avermitilis as presented in FIGURE 1 (SEQ ID NO:l), or a degenerate variant thereof, but which nucleotide sequence      comprises   one   or   more   mutations   encoding   an   amino   acid substitution at one or more amino acid residues corresponding to amino acid .position 38, 48, 89, 99,111,' 136,154, 179, 228, 238, 289, or 298 of SEQ ID No:2, such that cells of S. avernitilis strain ATCC 53692 in which the wild-type aLyeG allele has been inactivated and that express the polynucleotide molecule comprising the mutated nucleotide  sequence produce  a class 2:1  ratio of avermectins that is different from the ratio produced by cells of S. avermitilis strain ATCC 53692 that instead express only the wild-type aveC allele.<br>
2.	The polynucleotide molecule as claimed in claim 1, wherein the class 2:1 avermectins are cyclohexyl B2:'cyclohexyl Bl avermectins.<br>
3.	The polynucleotide molecule as claimed in claim 2, wherein the different class 2:1 ratio of avermectins is a reduced ratio compared to the class 2:1 ratio produced by cells of S. avermililis strain ATCC 53692 that instead express only the wild-type aveC allele.<br><br>
4.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.8:1 or less.<br>
5.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.68:1 or less.<br>
6.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.53:1 or less.<br><br>
7.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 arermectins is about 0.42:1 or less.<br>
8.	The polynucleotide molecule as claimed in claim 3, wherein the ratio of class 2:1 avermectins is about 0.40:1 or less.<br>
9.	The polynucleotide molecule as claimed in claim 1, wherein the<br>
nucleotide sequence comprises one or more mutations encoding an amino acid<br>
substitution at one or both amino acid residues corresponding to amino acid<br>
positions 138 and 139 of SEQ ID NO:2.<br>
10.	The polynucleotide molecule as claimed in claim 2, wherein the amino<br>
acid substitutions are selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino acid<br>
that is a conservative substitution for P;<br>
(b)	amino' acid residue D at position 48 replaced by E or by an amino acid<br>
that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(g)	amino acid residue K at position 154 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(h)       amino acid residue G at position 179 replaced by S or by an amino acid<br>
that is a conservative substitution for S;<br>
(i)        amino acid residue M at position 228 replaced by T or by an amino acid<br>
that is a conservative substitution for T;<br>
(j)	amino acid residue E at position 238 replaced by D or by an amino acid<br>
that is a conservative substitution for D;<br><br>
(k) amino acid residue P at position 289 replaced by L or by an amino acid that is a conservative substitution for L; and<br>
(1) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H.<br>
11. The polynucleotide molecule as claimed in claim 2, wherein a combination of amino acid residues is mutated, and wherein the combination is selected from one or more of the group consisting of:<br>
(a)	amino acid residues D48 and A89;<br>
(b)	amino acid residues Q38, L136 and E238;<br>
(c)	amino acid residues Gill and P289.<br>
12.	The polynucleotide molecule as claimed in claim 11, wherein the<br>
combination of amino acid substitutions is selected from one or more of<br>
the group consisting of:<br>
(a)	D48E/A89T;<br>
(b)	Q38P/L136P/E238D;<br>
(c)	G1llV/P289L, and<br><br>
13.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding D48E/A89T comprise a base change from T to A at a nucleotide position corresponding to nt 317 of SEQ ID NO: 1, and a base change from G to A at a nucleotide position corresponding to nt 438 of SEQ ID NO: 1.<br>
14.	The polynucleotide molecule as claimed in claim 13, comprising a base change from C to A at a nucleotide position corresponding to nt 353 of SEQ ID.NO: 1, and a base change from T to A at a nucleotide position corresponding to nt 1155 of SEQ ID NO: 1.<br>
15.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding Q38P/L136P/E238D comprise a base change from A to C at a nucleotide position corresponding to nt 285 of SEQ ID NO; 1, a base change from T to C at a nucleotide position corresponding to nt 580 of SEQ ID NO: 1, and a base change from A to T at a nucleotide position corresponding to nt 886 of SEQ ID NO: 1.<br>
16.	The polynucleotide molecule as claimed in claim 15, comprising a<br><br>
base change from A to C at a nucleotide position corresponding to nt 24 of SEQ ID NO: 1, a base change from T to C at a nucleotide position corresponding to nt 497 of SEQ ID NQ: 1, and a base change from C to T at a nucleotide position corresponding to nt 554 of SEQ ID NO: 1.<br>
17.	The polynucleotide molecule as claimed in claim 12, wherein the mutations in the aveC sequence encoding G1l1V/P289L comprise a base change from G to T at a nucleotide position corresponding to nt 505 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position corresponding to nt 1039 of SEQ ID NO:l.<br>
18.	The polynucleotide molecule as claimed in claim 22, comprising a base change from T to C at a nucleotide position corresponding to nt 155 of SEQ ID NO: 1, a base change from C to T at a nucleotide position corresponding to nt 1202 of SEQ ID NO: 1, and a base change from T to C at a nucleotide position corresponding to nt 1210 of SEQ ID NO:l.<br>
19.	A recombinant vector comprising the polynucleotide molecule as claimed in claim 1.<br><br>
20.	A method for making a novel strain of S. avermitilis, comprising mutating the aveC allele in cells of a strain of S. avemitilis, which mutation results in the substitution in the AveC gene product of a different amino acid residue at one or more amino acid positions corresponding to amino acid residues 38, 48, 89, 99, 111, 136, 154, 179, 228, 238, 289 or 298 of SEQ ID NO 2, such that cells of the S. avermitilis strain in which the aveC allele has been so mutated produce a class 2:1 ratio of avermectins that is different from the ratio produced by cells of the same S. avermitilis strain that instead express only the wild-type aveC allele.<br>
21.	The   method   as   claimed   in   claim   20,   wherein  the   class   2:1 avermectins are cyclohexyl B2: cyclohexyl B 1 avermectins.<br>
22.	The method as claimed in claim 21, wherein the different class 2:1 ratio of avermectins is a reduced ratio.<br>
23.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the avec allele has been so mutated is about 0.8:1 or less.<br>
24.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC<br><br>
allele has been so mutated is about 0.68:1 or less.<br>
25.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC allele has been so mutated is about 0.53:1 or less.<br>
26.	The method as claimed in claim 22, wherein the ratio of class 2:1 avermectins produced by cells of the S. avermitilis strain in which the aveC allele has been, so mutated is about 0.42:1 or less.<br>
27.	The method as claimed in claim 22, wherein the ratio of class 2:1<br>
avermectins produced by cells of the S. avermitilis strain in which the<br>
aveC allele has been so mutated is about 0.40:1 or less.<br>
28.	The method as claimed in claim 21, wherein the amino acid<br>
substitutions are selected from one or more of the group consisting of:<br>
(a)	amino acid residue Q at position 38 replaced by P or by an amino' acid . that is a conservative substitution for P;<br>
(b)	amino acid residue D at position 48 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(c)	amino acid residue A at position 89 replaced by T or by an amino acid that is a conservative substitution for T;<br>
(d)	amino acid residue F at position 99 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(e)	amino acid residue G at position 111 replaced by V or by an amino acid<br>
that is a conservative substitution for V;<br>
(f)	amino acid residue L at position 136 replaced by P or by an amino acid that is a conservative substitution for P;<br>
(g)	amino acid residue K at position 154 replaced by E or by an amino acid that is a conservative substitution for E;<br>
(h) amino acid residue G at position 179 replaced by S or by an amino acid that is a conservative substitution for S;<br>
(i) amino acid residue M at position 228 replaced by T or by an amino acid that is a conservative substitution for T;<br><br>
(j) amino acid residue E at position 238 replaced by D or by an amino acid that is a conservative substitution for D;<br>
(k) amino acid residue P at position 289 replaced by L or by an amino acid that is a conservative substitution for L; and<br>
(1) amino acid residue Q at position 298 replaced by H or by an amino acid that is a conservative substitution for H.<br>
29.	The method as claimed in claim 21, wherein the aveC allele is<br>
mutated to encode amino acid substitutions at a combination of amino<br>
acid positions, and wherein the combination is selected from one or more<br>
of the group consisting of:<br>
(a)	amino	acid	residues	D48	and	A89;<br>
(b)	amino acid residues Q38, L136 and E238;<br>
(c)	amino acid residues G1ll and P289.<br>
30.	The method as claimed in claim 29, wherein the combination of<br>
amino acid substitutions is selected from one or more of the group<br>
consisting of:<br>
(a)	D48E/A89T;<br>
(b)	Q38P/L136P/E238D;<br>
(c)	G111V/P289L.<br><br>
31.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding D48E/A89T comprise a base change from T to A at a nucleotide position in the aveC allele corresponding to nt 317 of SEQ ID NO: 1, and a base change from G to A at a nucleotide position in the aveC allele corresponding to nt 438 of SEQ ID NO: 1.<br>
32.	The method as claimed in claim 31 comprising introducing a base change from C to A at a nucleotide position in the aveC allele corresponding to nt 353 of SEQ ID NO: 1, and a base change from T to A at a nucleotide position in the aveC allele corresponding to nt 1155 of SEQ	ID	NO:	1.<br>
33.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding Q38P/L136P/E238D comprise a base change from<br><br>
A to C at a nucleotide position in the aveC allele corresponding to nt 286 of SEQ ID NO:l, a base change from T to C at a nucleotide position in the aveG allele corresponding to nt 580 of SEQ ID NQ: 1, and a base change from A to T at a nucleotide position in the aveC allele corresponding to nt 886 of SEQ ID NO: 1.<br>
34.	The method as claimed in claim 33, comprising introducing a base change from A to G at a nucleotide position in the aveC allele corresponding to nt 24 of SEQ ID NO:l, a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 497 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position in the avec allele corresponding to 554 of SEQ ID NO:l.<br>
35.	The method as claimed in claim 30, wherein the mutations in the aveC allele encoding Gl11V/P289L comprise a base change from G to T at	a nucleotide position in the aveC allele corresponding to nt 505 of SEQ ID NO: 1, and a base change from C to T at a nucleotide position in the aveC  allele corresponding to nt 1039 of SEQ ID NO: 1.<br>
36.	The method as claimed in claim 35, comprising introducing a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 155 of SEQ ID NO: 1, a base change from C to T at a nucleotide position in the avec allele corresponding to nt 1202 of SEQ ID NO: 1. and a base change from T to C at a nucleotide position in the aveC allele corresponding to nt 1210 of SEQ ID NO: 1.<br>
37.	A process for producing avermectins, comprising culturing the cells of S. Avermitilis in culture media under conditions that permit or induce the production of avermectins therefrom, and recovering said avermectins from the culture.<br>
Dated this January 30, 2002.	 <br>
[DEEPAK KUMAR]<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPLICANT[S]<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWNhbmNlbGxlZCBwYWdlcyg4LTgtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-cancelled pages(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWNsYWltcyhncmFudGVkKS0oOC04LTIwMDYpLmRvYw==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-claims(granted)-(8-8-2006).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWNsYWltcyhncmFudGVkKS0oOC04LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-claims(granted)-(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWNvcnJlc3BvbmRlbmNlKDItOC0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-correspondence(2-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWNvcnJlc3BvbmRlbmNlKGlwbyktKDIxLTgtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-correspondence(ipo)-(21-8-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMSgyOC0xLTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 1(28-1-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMSgzMC0xLTIwMDIpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 1(30-1-2002).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMTgoNy0zLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 18(7-3-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMWEoMjYtNC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 1a(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMihncmFudGVkKS0oOC04LTIwMDYpLmRvYw==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 2(granted)-(8-8-2006).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMihncmFudGVkKS0oOC04LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 2(granted)-(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMygyNC00LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 3(24-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gMygyNi00LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 3(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gNCgyOC0yLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 4(28-2-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0gNSgyNi00LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form 5(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLWZvcm0tcGN0LWlzYS0yMTAoOC04LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-form-pct-isa-210(8-8-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM3KDI2LTQtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-petition under rule 137(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM4KDI2LTQtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-petition under rule 138(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgyNi00LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-power of authority(26-4-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDAxMzItbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgyNi05LTIwMDApLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-00132-mum-power of authority(26-9-2000).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="209119-a-process-for-manufacture-of-water-soluble-or-water-dispersible-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209121-a-dispensing-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209120</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/00132/MUM</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Jan-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PFIZER PRODUCTS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>EASTERN POINT ROAD, GROTON, CONNECTICUT 06340, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>YAN CHEN</td>
											<td>1765 BOWERS AVENUE, SANTA CLARA, CALIFORNIA 95051, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CLAES GUSTAFSSON</td>
											<td>1813 BAYVIEW AVENUE, BELMONT, CALIFORNIA 94002, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ANKE KREBBER</td>
											<td>1935 ROCK STREET, # 1, MOUNTAIN VIEW, CALIFORNIA 94043, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JEREMY STEPHEN MINSHULL</td>
											<td>842 HERMOSA WAY, MENLO PARK, CALIFORNIA 94025 USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SUN AI RAILLARD</td>
											<td>964 TROPHY DRIVE, MOUNTAIN VIEW, CALIFORNIA 94043, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KIM JONELLE STUTZMAN-ENGWALL</td>
											<td>PFIZER CENTRAL RESEARCH, EASTERN POINT ROAD, GROTON, CONNECTICUT 06340, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/60</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB00/01017</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-07-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/148,645</td>
									<td>1999-08-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209120-streptomyces-avermitilis-gene-directing-the-ratio-of-b2-b1-avermectins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:51:24 GMT -->
</html>
