<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209228-a-polynucleotide-that-is-suitable-for-accelerating-a-biosynthesis-of-ml236b-and-the-method-for-producing-ml236b-by-using-the-polynucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:13:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209228:&quot;A POLYNUCLEOTIDE THAT IS SUITABLE FOR ACCELERATING A BIOSYNTHESIS OF ML236B AND THE METHOD FOR PRODUCING ML236B BY USING THE POLYNUCLEOTIDE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A POLYNUCLEOTIDE THAT IS SUITABLE FOR ACCELERATING A BIOSYNTHESIS OF ML236B AND THE METHOD FOR PRODUCING ML236B BY USING THE POLYNUCLEOTIDE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A polynucleotide selected from the group consisting of: (a) potymicletide encoding a protein having the amino acid sequence of SEQ ID NO 38, or a polynucleotide variant thereof encoding a modified amino acid sequence having at least one deletion addition substitution or alteration said variant polynucleotide being suitable for use in accelerating the blosynthesis of ML-236B; and (b) a polynucleotide encoding a protein having the amino acid sequence of SEQ ID NO 42 or a polynucleotide variant thereof encoding a modified amino acid having at least one deletion substitution or alteration said variant polynucleotide being suitable for use in accelerating the biosynthesis of ML-236B.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to a polynucleotide and a method for preparing the same.<br>
FIELD OF THE INVENTION<br>
The present invention relates to a gene cluster, and more particularly to genes from a gene cluster.<br>
More particularly the invention relates to polynucleotides. such as DNA, which accelerate the biosynthesis of a HMG-CoA reductaseinhibitor. ML-236B. in an ML-236B producing micro-organism when introduced in the ML-236B producing micro-organism.  The invention further relates-to vectors into which said polynucleotides are incorporated, host cells transformed by said vectors, proteins expressed by said vectors, a method for producing ML-236B using said polynucleotides and/or proteins where the method comprises recovering ML-236B from the culture of said host cell, and the invention further relates to other associated aspects.<br>
BACKGROUND OF THE INVENTION<br>
Pravastalin is an HMG-CoA reductase inhibitor. Pravastatin sodium has been used in the treatment of hyperlipemia or hyperlipidaemia and has the useful pharmacological effect of being able to reduce serum cholesterol. Pravastatin can be obtained using Streptomyces carbophilus by microbial conversion of ML-236B produced by Penicillium cifrinum [described in Endo, A., etal,J. Antibiot.,29 1346(1976): Matsuoka, T. etal. Eur. J. Biochem., 184, 707 (1989), and in Japanese Patent Application Publication No 57-2240].<br>
It has been shown that both ML-236B. a precursor of pravastatin. and lovastatin. a HMG-CoA inhibitor, share the same partial structure.   They are synthesized biologically via polyketides [described in Moore, R.N., etal, J.Am.Chem.Soc.,107, 3694(1985); Shiao,<br><br>
2 M.and Don, H.S., Proc. Natl. Sci. Counc. Repub. China B. 11. 223(1987)].<br>
Polyketides are compounds derived from a p-keto carbon chains that result from a continuous condensation reaction of low-molecular weight carboxylic acids, such as acetic acid, propionic acid, butyric acid or the like.   Various structures may be derived depending on the pathway of condensation or reduction of each of the p-keto carbonyi groups [described in Hopwood, D.A. and Sherman. D.H.. Annu. Rev.Genet.. 24. 37-66 (1990): Hutchinson. C. R. and Fujii, I., Annu. Rev. Microbiol.. 49. 201-238(1995)].<br>
Polyketide Synthases (hereinafter referred to as PKSs) that contribute to the synthesis of polyketides are enzymes known to be present in filamentous fungi and bacteria.   The enzymes of filamentous fungi have been studied using molecular biological techniques [as described in Feng. G.H. and Leonard. T. J.. J. Bacteriol.. 177. 6246 (1995): Takano. Y., et al Mol. Gen. Genet. 249. 162 (1995)].   In Aspergillus terreus. which is a lovastatin producing micro-organism, a PKS gene related to the biosynthesis of lovastatin has been analyzed [described in International application laid-open in Japan (KOHYO) No.9-504436. and see corresponding WO 9512661 which claims DNA encoding a triol polyketide synthase].<br>
Genes related to biosynthesis of secondary metabolites of filamentous fungi often form a cluster on the genome.   In the pathways of the biosynthesis of polyketides. gene clusters participating in said pathway are known to exist.   In the biosynthesis of Aflatoxin. which is a polyketide produced by Aspergillus flavus and Aspergillus parasiiicus. genes encoding enzyme proteins participating in said biosynthesis (such as PKS) have been known to form a cluster structure.   Genomic analysis and a comparison of the genes participating in the biosynthesis of Aflatoxin in each of the micro-organisms has been carried out [see Yu. J.7 et al., Appl. Environ. Microbiol.. 61, 2365 (1995)].   It has been reported that genes participating in biosynthesis of Sterigmatocystin produced by Aspergillus nidulans form a cluster structure in about 60 kb of a continuous region on its genome [described in Brown, D. W. etal.9 Proc. Natl. Acad. Sci. USA, 93, 1418 (1996)].<br>
The modulation of polyketide synthase activity by accessory proteins during<br><br>
3 lovastatin synthesis has been investigated, [see Kennedy. J. et ai. Science Vol 284. 1368<br>
(1999)].<br>
However, to date, there has been insufficient molecular biological analysis into the biosynthesis of ML-236B. and factors regulating it.   The present invention sets out to address this problem.<br>
SUMMARY OF INVENTION<br>
According to the present invention, there is provided a polynucleotide which is suitable for use in accelerating the biosynthesis of ML-236B.<br>
The polynucleotide is typically a polynucleotide encoding a protein including or consisting of the amino acid sequence of SEQ ID NO38. 42.44. 46. 48 or 50.   Polynucleotide variants thereof are also provided which encode a modified amino acid sequence having at least one deletion, addition, substitution or alteration.<br>
TABULATION FOR SEQUENCE LISTING<br>
A sequence listing forms part of this patent specification.   As an aid to understanding, we give the following tabulation of the listed sequences.<br><br><br>
4<br>
7	oligonucleotide DNA (1) for 5'-RACE. Example 8<br>
8	oligonucleotide DNA (1) for 5'-RACE. Example 8<br>
9	oligonucleotide DNA (1) for 5'-RACE. Example 8<br>
10	oligonucleotide DNA (1) for 5'-RACE. Example 8<br>
11	oligonucleotide DNA (2) for 5'-RACE. Example 8<br>
12	oligonucleotide DNA (2) for 5'-RACE. Example 8<br>
13	oligonucleotide DNA (2) for 5'-RACE. Example 8<br>
14	oligonucleotide DNA (2) for 5'-RACE. Example 8<br>
15	oligonucleotide DNA (2) for 5'-RACE. Example 8<br>
16	oligonucleotide DNA (2) for 5 '-RACE, Example 8<br>
17	5'-end cDNA fragment. Example 8<br>
18	5'-end cDNA fragment. Example 8<br>
19	5'-end cDNA fragment. Example 8<br>
20	5'-end cDNA fragment. Example 8<br>
21	5'-end cDNA fragment. Example 8<br>
22	5'-end cDNA fragment. Example 8<br>
23	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
24	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
25	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
26	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
27	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
28	oligonucleotide DNA (3) for 3'-RACE. Example 8<br>
29	3'-end cDNA fragment. Example 8<br>
30	3'-end cDNA fragment. Example 8<br>
31	3'-end cDNA fragment. Example 8<br>
32	3'-end cDNA fragment. Example 8<br>
33	3'-end cDNA fragment. Example 8<br><br>
34	3'-end cDNA fragment, Example 8<br>
35	RT-PCR primer, Example 9<br>
36	RT-PCR primer, Example 9<br>
37	mlcE; cDNA nucleotide sequence and deduced amino acid<br><br>
5 sequence<br>
38	deduced mlcE polypeptide<br>
39	RT-PCR primer. Example 12<br>
40	RT-PCR primer. Example 12<br>
41	mlcK: cDNA nucleotide sequence and deduced amino acid<br>
sequence<br>
42	deduced mlcR polypeptide<br>
43	mlcA: cDNA nucleotide sequence and deduced amino acid<br>
sequence<br>
44	deduced mlcA polypeptide<br>
45	m/cB: cDNA nucleotide sequence and deduced amino acid<br>
sequence<br>
46	deduced mlcB polypeptide<br>
47	mlcQ: cDNA nucleotide sequence and deduced amino acid<br>
sequence<br>
48	deduced mlcC polypeptide<br>
49	mlcD: cDNA nucleotide sequence and deduced amino acid<br>
sequence<br>
50	deduced mlcD polypeptide<br>
51	RT-PCR primer. Example 17<br>
52	RT-PCR primer. Example 17<br>
53	RT-PCR primer. Example 17<br>
54	RT-PCR primer. Example 17<br>
55	RT-PCR primer. Example 17<br>
56	RT-PCR primer. Example 17<br>
57	RT-PCR primer. Example 17<br>
58	RT-PCR primer. Example 17<br>
59	RT-PCR primer, Example 17<br>
60	RT-PCR primer, Example 17<br>
61	RT-PCR primer. Example 17<br>
62	RT-PCR primer, Example 17<br><br>
6<br>
PREFERRED EMBODIMENTS<br>
The polynucleotides encoding the amino acid sequences of SEQ ID NO 38 .42. 44 46. 48 or 50 can be cDNA. genomic DNA or mRNA.   The genomic DNA encoding each of these six sequences are referred ,o as structural genes m/cE. mlcR. w/cA. mlcB. mlc and mlcd. respectively.   Without being tied to these assignments, we believe that the structural genes encode proteins with the following functions:<br>
mlcA	polyketide synthase<br>
mlcB	polyketide synthase<br>
mlcC	P450 monooxygenase<br>
mlcD	HMG-CoA reductase<br>
rnlcE	efflux pump<br>
mlcR	transcriptional factor<br>
We have discovered that the incorporation of m/cE or cDNA corresponding to m/cE can accelerate the biosynthesis of ML-236B. and the incorporation of mlcR or cDNA corresponding to mlcK can accelerate the biosynthesis of ML-236B.   Furthermore. m/cR stimulates transcriptional expression of m/cA to D.   mlcA. B. C and D are involved in the production of ML-236B, independently or in combination, as shown by gene disruption studies-Variants of mlcA. B and/or C obtainably by natural or artificial change will be useful to produce derivatives of ML-236B. including statins such as pravastatin or lovastatin.   In this respect, it may be possible to produce pravastatin directly by using such variants with only the one fermentation step and without the need for microbial conversion of ML-236B to pravastatin currently performed with Sireptomyces carbophilus.<br>
A preferred polynucleotide includes a sequence comprising SEQ ID NO 37, or<br><br>
7<br>
comprising a mutant or variant thereof capable of accelerating the biosynthesis of ML-236B Such a DNA polynucleotide is obtainable from transformed Exchcnchia coft pSAKexpE SANK 72499 (FERM BP-7005).<br>
Another preferred polynucleotide includes a sequence comprising SEQ ID NO 41. or comprising a mutant or variant thereof capable of accelerating the biosynthesis of ML-236B. Such a DNA polynucleotide is obtainable from transformed Eschenchia coli pSAKexpR SANK 72599 (PERM BP-7006).<br>
The polynucleotidcs of this invention can be employed in operative combination with one or more polynucleotides.   Preferred combinations are suitable for use in enhancing the production of ML236B in an ML-236B producing micro-organism.<br>
Examples of such combinations include the polynucleotide of SEQ ID NO 37, or variant thereof having similar function, in combination with one or more sequences selected from SEQ ID NO 37 itself 41,43,45.47 or 49;as well as the polynucleotide of SEQ ID NO 41, or variant thereof having similar function, m combination with one or more sequences selected from SEQ ID NO 37,41 itself 43, 45, 47 or 49.<br>
In one aspect, the polynucleotide is preferably a polynucleotide encoding a protein including .of consisting of the ammo acid sequence of SEQ ID NO 38,42,44,46,48 or 50 and capable of accelerating the biosynthesis of ML-236B alone or in conjunction with the polypucieotidc of SEQ IB NO 37, SEQ ID NO 41 or a variant thereof having a similar 'function.<br>
Tne present invention further extends to polynucleotides which are capable of hybridizing under stringent conditions with a polynucleotide of this invention.   Such polynucleotidcs extend to polynucleotides suitable for accelerating the biosynthesis of ML-236B in a ML-236B producing micro-organism when introduced in the ML-236B producing micro-organism.<br>
The polynucleotide is typically DNA, cDNA-or genomic DNA, or RNA, and can be sense or antisense.  The polynucleotide is typically a purified polyimcleotide, such as a<br><br>
8 polynueleotide free from other cellular components.<br>
The present invention extends to polynucleotide variants encoding arnino acid sequences of the indicated SEQ ID NO 38. 42. 44.46.48 or 50. where one or more nucleotides has been changed.   The changes may be naturally occuring. and can be made within the redundancy or degeneracy of the triplets of the genetic code.   Such degeneratively changed polvnucleotides thus encode the same amino acid sequence.   Within these polynucleotide variants, we include genomic DNA having extrons exon and introns. rather than simply the cDNA sequence.<br>
The present invention further extends to polynucleotide variants encoding amino acid sequences of the indicated SEQ ID NO 38. 42. 44. 46. 48 or 50. which encode a modified amino acid sequence having at least one deletion, addition, substitution or alteration.   Thus, the invention extends to polynucleotide variants of the indicated sequences which encode amino acid sequences which are shorter, longer or the same length as that encoded by the indicated sequences.   Preferably the variant polypeptides retain an ability to accelerate the synthesis of ML-236B. and preferably have activity substantially similar to or better than the parent sequence giving rise to the variant sequence.<br>
The polynucleotide variants retain a degree of identity with the parent sequence. Suitably the degree of identity is at least 60%. at least 80%. at least 90% or at least 95% or 100%.   The degree of identity of a variant is preferably assessed by computer software, such as the BLAST program which uses an algorithm for performing homology searches.<br>
In one aspect, the preferred polynucleotide of this invention is DNA selected from the group consisting of:<br>
(a)	DNA which comprises one or more of nucleotide sequence shown in nucleotide No. 1 to<br>
1662 of SEQ ID No. 37 of the Sequence Listing, and which is characterized in accelerating<br>
the biosynthesis of ML-236B in a ML-236B producing micro-organism when being<br>
introduced in said ML-236B producing micro-organism;<br>
(b)	DNA which hybridizes with the DNA described in (a) under the stringent condition, and<br><br>
9 which is characterized in accelerating the biosynthesis of ML-236B in a ML-236B producing<br>
micro-organism when being introduced in said ML-236B producing micro-organism;<br>
(c)	DNA which comprises one or more of nucleotide sequence shown in nucleotide No. 1 to<br>
1380 of SEQ ID No. 41 of the Sequence Listing, and which is characterized in accelerating<br>
the biosynthesis of ML-236B in a ML-236B producing micro-organism when being<br>
introduced in said ML-236B producing micro-organism:<br>
(d)	DNA which hybridizes with the DNA described in (c) under the stringent condition, and<br>
which is characterized in accelerating the biosynthesis of ML-236B in a ML-236B producing<br>
micro-organism when being introduced in said ML-236B producing micro-organism.<br>
The polynucleotides of this invention accelerate the biosynthesis of ML-236B in a micro-organism which produces ML-236B.   Examples of ML-236B producing micro-orgamsms include PemcilHum species, such as Penicillium citrinum. Peniallium brevicompactum [described in Brown. A.G.. et at. J. Chem. Soc. Perkin-1.. 1165(1976)]. PemcilHum cyclopium [described in Doss. S.L.. et al. J. Natl. Prod..49. 357 (1986)] or the like.   Other examples include: Eupenicillium sp.M6603 [described in Endo. A., et al. J. Antibiot.-Tokyo, 39. 1609(1986)], Paecilomyces viridis FERM P-6236 [described in Japanese Patent Application Publication No.58-98092]. Paecilomyces sp.M2016 [described in Endo. A., el al. J. Antibiot. -Tokyo. 39. 1609 (1986)]. Trichoderma longibrachiatum M6735 [described in Endo, A., et al. J. Antibiot.-Tokyo. 39. 1609(1986)]. Hypomyces chrysospermus IFO 7798 [described in Endo. A., et al. J. Antibiot.-Tokyo. 39. 1609(1986)]. Gliocladium sp. YJ-9515 [described in WO 9806867]. Trichorderma viride IFO 5836 [described in Japanese Patent Publication No.62-19159]. Eupenicillium reticulisporum IFO 9022 [described in Japanese Patent Publication No. 62-19159]. or any other suitable organism.<br>
Among these ML-236B producing micro-organisms, PemcilHum citrinum is preferred, and the Penicillium citrinum strain SANK 13380 is more preferred.   Penicillium citrinum SANK 13380 strain was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on December 22, 1992 under the deposit Nos. FERM BP-4129, in accordance with the Budapest Treaty on the Deposition of Micro-organisms.   Examples of ML-236B producing micro-organisms also include those<br><br>
10 isolated from natural sources and those mutated naturally or artifically.<br>
The invention further provides vectors comprising a polynucleotide of this invention, such as the vector obtainable from Escherichia coli pSAKexpE SANK 72499 (FERM BP-7005) or Escherichia coli pSAKexpR SANK 72599 (FERM BP-7006).   Such vectors of this invention include expression vectors.<br>
Host cells transformed by a vector of this invention are also provided, including ML-236B producing micro-organisms.   Host cells of this invention include Penicillhtm citrinum and Escherichia coli, such as Escherichia coli pSAKexpE SANK 72499 (FERM BP-7005) or Escherichia coli pSAKexpR SANK 72599 (FERM BP-7006).<br>
Additionally the invention extends to polypeptides encoded by a polynucleotide of this invention.   Examples of polypeptides of this invention include the sequence of SEQ ID NO 38 or 42. or a variant thereof which has at a specified degree of identity to SEQ ID NO 38 or 42 and which is capable of accelerating ML236B production in an ML236B producing organism.   Other polypeptides are those encoded by the other polynucleotide sequences of this invention, and variants which retain a degree of identity.<br>
Suitably the degree of identity of polypeptide variants to SEQ ID NO 38 or 42 is at least 80%, at least 90% or at least 95% or 100%.   The degree of identity of a variant is preferably assessed by computer software, such as the BLAST program which uses an algorithm for performing homology searches.<br>
The polypeptides of this invention include shorter or longer sequences of SEQ ID NO 38 or 42 or variants.   Shorter poiypeptides comprise partial amino acid sequences of SEQ ID NO 38, 42 or variants thereof and preferably retain the ability to accelerate the biosynthesis of ML236B.   Longer polypeptides comprise all or partial amino acid sequences of SEQ ID NO 38, 42 or variants thereof and preferably retain the ability to accelerate the biosynthesis of ML236B.   Longer polypeptides include fusion proteins such as Fc-fused protein.<br><br>
11<br>
Polypeptides of this invention include one having the sequence of SEQ ID NO 38, SEQ ID NO 42, SEQ ID NO 44. SEQ ID NO 46, SEQ ID NO 48, or a varinat thereof having the similar function.   Antibody to polypeptides of this invention arc also provided.   Both polyclonal antibody and monoclonal antibody are provided by this invention. Said antibody is useful for regulating ML-236B production and for producing derivatives of ML-236B such as statins including pravastatin and lovastatin. Furthermore, said antibody can be preferably used for analysis of ML-236B biosynthesis and regulatory mechanisms thereof. Such analysis is useful for modulating ML-236B production and for producing derivatives of ML-236B.<br>
The host cells of this invention which have a vector of this invention can be used in a method for producing ML-236B, comprising culturing such a host cell and then recovering ML-236B from the culture.   In one method, the vector comprises mlcE or micR, and no additional genes such as nilcA, mlcB, mlcC or mlcD.<br>
Production by a method of this invention can occur in the absence of recombinant mlcA, mlcB. mlcC and/or mlcD (polypeptides) corresponding 10 SEQ ID NO 44. SEQ ID MU 46, SEQ ID NO 48 or SEQ ID NO 50.<br>
DESCRIPTION OF SPECIFIC EMBODIMENTS<br>
The present invention will be hereinafter described in more detail.<br>
The inventors of the present invention have cloned genomic DNA comprising genes participating in the biosynthesis of ML-236B in PeniciUium citrinum.   The genomic DNA is hereinafter referred to as ML-236B biosynthesis related genomic DNA, and was cloned from a genomic DNA library of a ML-236B producing micro-organism. The genomic DNA was analyzed to find structural genes on said genomic DNA, then cDNAs corresponding to said structural genes were obtained by reverse transcription - polymerase chain reaction (hereinafter referred to as a "RT-PCR") using total RNA which contains mRNA of Penicillium citrinum as a template.   It was found that the biosynthesis of ML-236B in a ML-236B producing micro-organism was accelerated when the ML-236B producing microorganism was transformed by a recombined DNA vector containing said cDNAs.<br>
The present invention relates particularly to cDNAs (hereinafter referred to as.ML-<br><br>
12 236B biosynthesis accelerating cDNA) that accelerate the biosynthesis of ML-236B in a ML-<br>
236B producing micro-organism when introduced into said ML-236B producing microorganism.<br>
An ML-236B biosynthesis accelerating poiynucleotide of the present invention, such as ML-236B biosynthesis acclerating cDNA. includes, by way of example:<br>
(I)	DNA obtainable by synthesis using, as a template, a transcribed product<br>
(messenger RNA, hereinafter referred to as mRNA) of a structural gene which participates in<br>
the biosynthesis of ML-236B and which exists in the genomic DNA of a ML-236B-producing<br>
micro-organism;<br>
(II)	double stranded DNA formed as a result of association of a DNA (I) and the<br>
second strand DNA synthesized using the DNA (I) as a first strand;<br>
(III)	double stranded DNA formed by replicating or amplifying the double stranded<br>
DNA (II), for example, by a method of cloning or the like;<br>
(IV)	DNA which can hybridize with one of the above DNA's or mRNA under<br>
stringent conditions.<br>
The DNA (IV) can be those shown in any of the structural gene sequences herein, for example nucleotide No. 1 to 1662 of SEQ ID No. 37 of the Sequence Listing or nucleotide numbers 1 to 1380 of SEQ ID No 41. wherein one or more nucleotides is optionally substituted, deleted and/or added, and which can accelerate the biosynthesis of ML-236B in an ML-236B producing micro-organism when introduced in the ML-236B producing microorganism.<br>
When two single stranded nucleic acids hybridize they form a double-stranded molecule in a region in which they are complementary or highly complementary with each other, and "stringent conditions" suitably refers to the case in which the hybridization solution is 6 x SSC [1 x SSC has a composition of 150 mM NaCl, 15 mM of sodium citrate], and the temperature for the hybridization is 55°C.<br>
ML-236B biosynthesis accelerating cDNA can be obtained, for example, by isolating<br><br>
13 a clone containing the cONA from a cONA library of a ML-236B producing micro-organism.<br>
As an alternative, RT-PCR can be used employing a pair of primers designed on the basis of the nucleotide sequence of an ML-236B biosynthesis-related genomic DNA together with mRNA or total RNA of a ML-236B producing micro-organism.<br>
An ML-236B producing micro-organism is a micro-organism inherently having an ability to produce ML-236B.   As indicated previously, examples of ML-236B producing micro-organisms include Penicillium species. Such as Penicillium citrinitm, Penicillium brevicompacium, Penicillium cyclopiwn or the like, and other examples include: Eupeniciliiurn sp.M6603. Paecilomyces viridis FERM P-6236, Paecilomyces sp.M2016, TrichoiUtma lonphrachiaiiim M6735, Hypomyces chrysospermus IFO 7798. Gliocladivm sp. YJ-9515, Trichordcrma viride IFO 5836, Eupenicilhum reiiculisporum IFO 9022, and any other suitable organisms.<br>
Among these ML-236B producing micro-organisms.- PgmciliiufriCitrinum is preferred, and the Penicillium citrinum strain SANK 13380 is more preferred.   Penicillium citrimim SANK 13380 strain was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on December 22, 1992 under the deposit Nos. FERM BP-4129. in accordance with the Budapest Treaty on the Deposition of Micro-organisms.   Examples of ML-236B producing micro-organisms also include both those isolated from natural sources and those mutated naturally or artificially.<br>
ML-236B biosynthesis related genomic DNA can be obtained by screening a genomic DNA library of an ML-236B producing micro-organism with a suitable probe. Suitably the probe is designed on the basis of a DNA sequence predicted to have a role in ML-236B biosynthesis, suitably originating from a filamentous fungus.<br>
The choice of methods for creating a genomic DNA library are not limited, and any suitable method may be used, preferably being a general method for constructing o genomic DNA library of a eukaryotic organism.   Examples thereof include the method of Maniatis ct al. [Maniatis, T., et al.. Molecular cloning, a laboratory manual, 2nd cd., Cold Spring Harbor<br><br>
Laboratory. Cold Spring Harbor. N.Y. (1989)].   Other suitable methods are known in the art.<br>
In outline, genomic DNA from an ML-236B producing micro-organism can be obtained by recovering cells from a culture of said ML-236B producing micro-organism, physically breaking the cells, extracting DNA present in the nuclei thereof and purifying said DNA.<br>
Culturing of a ML-236B producing micro-organism can be performed under conditions suitable for the particular ML-236B producing micro-organisms.   For example, culturing of Penicillium citrinum. a preferred ML-236B producing micro-organism, can be performed by inoculating the cells in MBG3-8 medium [composition: 7 %(w/v) glycerin. 3 %(w/v) glucose. 1 %(w/v) soybean powder. 1 %(w/v) peptone (manufactured by Kyokuto Seiyaku Kogyo corporation). 1 %(w/v) Corn steep liqueur (manufactured by Honen corporation), 0.5 %(w/v) sodium nitrate. 0.1 %(w/v) magnesium sulfate heptahydrate (pH 6.5)], and incubating at 22 to 28°C with shaking for 3 to 7 days.   A slant for storage of the bacterium can be prepared by pouring melted PGA agar medium [composition: 200g/L potato extract, 15 %(w/v) glycerin. 2% (w/v) agar] into a test tube, and allowing the agar to solidify at an angle.   Penicillium citrinum may then be inoculating into the slant using a platinum needle, followed by incubation at 22 to 28°C for 7 to 15 days.   Micro-organisms or bacteria grown in this way can be continuously maintained on the slant by reserving the slant at 0 to 4°C.<br>
Cells of an ML-236B producing micro-organism cultured in a liquid medium can be recovered by centrifugation. and those cultured on a solid medium can be recovered by scraping from the solid media with a cell scraper or the like,<br>
Physical breaking of cells can be performed by grinding the cells using a pestle and a mortar, after freezing them with liquid nitrogen or the like.   DNA in the nuclei of the broken cell can be extracted using a surfactant such as sodium dodecylsulfate (hereinafter referred to as SDS) or other suitable surfactant.   The extracted genomic DNA is suitably treated with phenol - chloroform to remove protein, and recovered as a precipitate by performing an<br><br>
15 ethanol precipitation.<br>
The resulting genomic DNA is fragmented by digestion with a suitable restriction enzyme.   There is no limitation on the restriction enzymes that can be used for the restriction digest, with generally available restriction enzymes preferred.   Examples thereof include Sau3AI.   Other suitable enzymes are known in the art. Digested DNA is then subjected to gel electrophoresis. and genomic DNA having a suitable size is recovered from the gel.   The size of DNA fragment is not particularly limited, but is preferably 20 kb or more.<br>
There is likewise no limitation on the choice of DNA vector used in construction of the genomic DNA library, as long as the vector has a DNA sequence necessary for replication in the host cell which is to be transformed by the vector.   Examples of suitable vectors include a plasmid vector, a phage vector, a cosmid vector, a BAC vector or the like, with a cosmid vector being preferred.   The DNA vector is preferably an expression vector.   More preferably, the DNA vector comprises a DNA or nucleotide sequence which confers a selective phenotype onto the host cell transformed by the vector.<br>
The DNA vector is suitably a vector that can be used in both cloning and expression. Preferably the vector is a shuttle vector which can be used for transformation of more than one micro-organism host.   The shuttle vector suitably has a DNA sequence which permits replication in a host cell, and preferably a sequence or sequences which permit replication in a number of different host cells from different micro-organism groups such as bacteria and fungi.   Furthermore, the shuttle vector preferably comprises a DNA sequence which can provides a selectable phenotype for a range of different host cells, such as cells from different micro-organism groups.<br>
The choice of combination of a micro-organism groups and host cells transformed by the shuttle vector is not particularly limited, provided that one of the micro-organism groups can be used in cloning and the other has ML-236B producing ability.   Such combination can be. for example, a combination of a bacterium and filamentous fungi, a combination of yeast and filamentous fungi, with a combination of a bacterium and filamentous fungi being<br><br>
16 preferred.   The choice of bacterium is not particularly limited as long as it can be generally<br>
used in biotechnology, such as for example Eschehchia colL Bacillus subtilis or the like. Escherichia coli is preferred, and Escherichia coli XL 1 -Blue MR is more preferred. Similarly there is no restriction on yeast species as long as it can be generally used in biotechnology, such as for example, Saccharomyces cerevisiae or the like.   Examples of filamentous fungi include ML-236B producing micro-organisms described above. Other suitable examples of micro-organisms are known in the art.<br>
In the present invention, the micro-organism group can be selected from bacteria, filamentous fungi and yeast.<br>
Examples of the above-mentioned shuttle vector include a cosmid vector having a suitable marker gene for selecting a phenotype and a cos site. Other suitable vectors are known in the art.   The preferred vector is pSAKcosl. constructed by inserting a cos site from cosmid vector pWE15 (manufactured by STRATAGENE) into plasmid pSAK.333. which comprises the sequence of Escherichia coli hygromycin B phosphotransferase gene [described in Japanese Patent Application Publication No.3-262486].   A method for constructing pSAKcosl is shown in Figure 1.   The present invention is not limited to this vector.<br>
A genomic DNA library can be prepared by introducing a shuttle vector into a host cell, the vector containing a genomic DNA fragment from an ML-236B producing microorganism.   The host cell to be used is preferably Escherichia coli. more preferably Escherichia coli XLl-Blue MR. When the host cell is Escherichia coli. introduction can be performed by in vitro packaging.   In the present invention, transformation also covers the introduction of foreign DNA by in vitro packaging, and a transformed cell also covers a cell to which foreign DNA is introduced by in vitro packaging.<br>
A genomic library can be screened to identify a desired clone using an antibody or a nucleic acid probe, with a nucleic acid probe being preferred.   Preferably the nucleic acid probe is prepared based on the nucleotide sequence of a gene or DNA related to polyketide biosynthesis, preferably being a sequence derived from a filamentous fungus.   The choice of<br><br>
particular gene is not limited, as long as it is involved in biosynthesis of polyketides and the nucleotide sequence thereof is known.   Examples of such genes include the Aflatoxin PKS gene of Aspergillus flavus and Aspergillus parasitic the Sterigmatocystin PKS gene of Aspergillus nidulans or the like.<br>
Suitable nucleic acid probes can be obtained, for example, by synthesizing an oligonucleotide probe comprising part of a known genomic DNA sequence as described above, or by preparing oligonucleotide primers and amplifying the target DNA using the polymerase chain reaction [hereinafter referred to as "PCR'\ described in Saiki. R. K.. et a/.. Science. 239, 487 (1988)] and genomic DNA as a template, or by RT-PCR using mRNA as a template.   Other suitable methods for obtaining such probes are well known in the art.<br>
A nucleic acid probe can be obtained from a ML-236B producing micro-organism using, for example. PCR or RT-PCR.   Design of the primers used for PCR or RT-PCR (hereinafter referred to as "primer for PCR") is preferably carried out based on the nucleotide sequence of a gene related to polyketide biosynthesis for which the nucleotide sequence is known.   Preferably the gene is the aflatoxin PKS gene of Aspergillus flavus. Aspergillus parasiticus, or the Sterigmatocystin PKS gene of Aspergillus nidulans.<br>
The primer for PCR are suitably designed to comprise nucleotide sequences which encode amino acid sequences that are highly conserved within PKS genes.   Methods to identify nucleotide sequences corresponding to a given amino acid sequence include deduction on the basis of the codon usage of the host cell, and methods of making mixed oligonucleotide sequences using multiple codons (hereinafter referred to as a 'degenerate oligonucleotides').   In the latter case, the multiplicity of oligonucleotides can be reduced by introducing hypoxanthine to their nucleotide sequences.<br>
Primer for PCR may comprise a nucleotide sequence designed to anneal with a template chain, the primer being joined to an additional 5' sequence.   The choice of such an additional 5' nucleotide sequence is not particularly limited, as long as the primer can be used for PCR or RT-PCR.   Such an additional 5' sequence can be, for example, a nucleotide<br><br>
18<br>
sequence convenient for the cloning operation of a PCR product.   Such a nucieotide sequence can be, for example, a restriction enzyme cleavage site or a nucleotide sequence containing a restriction enzyme cleavage site.<br>
Furthermore, in designing of the primer for PCR. it is preferred that the sum of the number of guanine (G) and the number of cyiosine (C) bases is 40 to 60 % of the total number of bases.   Furthermore, preferably there is little or no self-annealling for a given primer and. in the case of a pair of primers, preferably little or no annealing between the primers.<br>
The number of nucleotides making up the primer for PCR is not particularly limited, as long as it can be used for PCR.   The lower limit of the number is generally 10 to 14 nucleotides, with the upper limit 40 to 60 nucleotides.   Preferably, primers are 14 to 40 oligonucleotides in length.<br>
The primer for PCR is preferably DNA.   Nucleosides in the primer can be deoxy adenosine, deoxy cytidine. deoxy thymidine. and deoxy guanosine. and additionally deoxy inosine.   The 5'-position of the nucleoside at the 5'-end of the primer for PCR is suitably a hydroxyl group or a hydroxy group to which one phosphoric acid is bonded by an ester link.<br>
Synthesis of primer for PCR can be performed by methods generally used for synthesis of nucleic acids, for example, the phosphoamidite method.   An automated DNA synthesizer can be preferably used in such a method.<br>
Genomic DNA and mRNA from an ML-236B producing micro-organism can be used as a template for PCR or RT-PCR respectively.   Total RNA can also be used as a template for RT-PCR instead of mRNA.<br>
The PCR product or RT-PCR product can be cloned by incorporation into a suitable DNA vector.   The choice of DNA vector used for the cloning step is not generally limited. Kits for the easy cloning of PCR and RT-PCR products are commercially available.   By way of example, the Original TA Cloning Kit (manufactured by Invitrogen: using pCR2.1 as DNA<br><br>
19 vector) is suitable for such cloning.<br>
In order to obtain a cloned PCR product, transformed host cells containing plasmids comprising the desired PCR product are cultured, and then the plasmids extracted from the cells and purified.   The inserted DNA fragment is then recovered from the resulting plasmid.<br>
Culturing of the transformed host cells is suitably performed under conditions appropriate for the host cells.   A preferred host cell. Escherichia coli. can be cultured in LB medium [l%(w/v) trypton. 0.5% (w/v) yeast extract. 0.5% (w/v) sodium chloride] at 30 to 37°C for 18 hours to two days with shaking.<br>
Preparation of plasmids from a culture of the transformed host cells can be performed by recovering the host cells and isolating plasmids free from other cellular components such as genomic DNA or host protein.   Preparation of plasmid DNA from a culture of Escherichia coli can be performed according to the alkaline method of Maniatis [described in Maniatis. T.. et al.t Molecular cloning, a laboratory manual. 2nd ed.. Cold Spring Harbor Laboratory. Cold Spring Harbor. N.Y. (1989)]. Kits for obtaining a plasmid having higher purity are commercially available.   The Plasmid Mini Kit [manufactured by QIAGEN AG] is preferred. Furthermore, a kit for mass-production of a piasmid is commercially available.   The Plasmid Maxi Kit (manufactured by QIAGEN AG) is preferred.<br>
The concentration of the resulting plasmid DNA can be determined by measuring absorbance at a wavelength of 260 nm after adequate dilution of DNA sample, and calculating on the basis that a solution with an absorbance OD260 of 1 contains 50 ?/ml DNA (described in Maniatis, T.. et al. supra).<br>
Purity of DNA can be calculated from a ratio of absorbance at a wavelength of 280 and 260 nm (described in Maniatis. T.. et al, supra).<br>
Methods for labeling of nucleic acid probes can be generally classified as radiolabeling<and non-radio the choice of radionucleotide for radio-labeling is not></and><br>
20 generally limited, and can be. for example. 32P. 35S. 14C or the like.   The use of 32P in labeling<br>
is preferred.   The choice of agent for non-radiolabeling is also not generally limited, so long as it may be generally used for labeling nucleic acid, and can be for example, digoxgenin. biotin. or the like, with digoxigenin preferred.<br>
Methods for the labeling of nucleic acid probe are also not generally limited. Preferred are commonly used methods, such as. for example, methods incorporating the label into the product by PCR or RT-PCR using labeled nucleotide substrates, nick translation, use of random primers, terminal labeling, and methods for synthesizing oligonucleotide DNA using labeled nucleotide substrates.   A suitable method can be selected from these methods depending on the kind of nucleic acid probe.<br>
The presence in the genome of a ML-236B producing micro-organism of a nucleotide sequence that is the same as the nucleotide sequence of a particular nucleic acid probe can be confirmed by Southern blot hybridization with the genomic DNA of said ML-236B producing micro-organism.<br>
Southern blot hybridization can be performed according to the method of Maniatis [described in Maniatis. T.. et ai. supra].<br>
A labeled nucleic acid probe, prepared as described above, can be used to screen a genomic DNA library.   The choice of screening method is not particularly limited, as long as it is generally appropriate for gene cloning, but it is preferably the colony hybridization method [described in Maniatis. T.. et ai. supra].<br>
Culturing of the colonies used for colony hybridization is suitably performed under conditions appropriate for the host cells.   Culturing of Escherichia coli. a preferred host, can be performed by incubation in LB agar medium [l%(w/v) trypton. 0.5% (w/v) yeast extract, 0.5% (w/v) sodium chloride. 1.5 % (w/v) agarose] at 30 to 37°C for 18 hours to two days.<br>
Preparation of recombinant DNA vector from the positive clone obtained by colony<br><br>
21 hybridization is generally performed by extracting the plasmid from the culture of the positive<br>
clone and purifying it.<br>
A transformed Escherichia coli strain. Escherichia coli pML48 SANK 71199 representing a positive clone obtained according to the present invention, was deposited at the Research Institute of Life Science and Technology of the Agency of industrial Science and Technology on July 7, 1999, in accordance with the Budapest Treaty on the Deposition of Micro-organisms, and was accorded the accession number FERM BP-6780.<br>
A typical DNA vector carried by Escherichia coli pML48 SANK71199 was designated as pML48.<br>
Confirmation that the recombinam DNA vector present in the positive Cone contains ML-236B biosynthesis-related genomic DNA can be suitably assessed by determining the nucleotide sequence of the recombinam DNA vector insert. Southern blot hybridization or expression of the insert to determine function.<br>
The nucleotide sequence of DNA can be determined according to the Maxam and Gilbert chemical modification technique [described in Maxam. A. M. M. and Gilbert. W.. Methods in Enzymology. 65. 499 (1980)] or the dideoxy chain termination method [described in Messing, J. and Vieira, J.. Gene. 19. 269 (1982)].   Other suitable methods are well known in the art.   Plasmid DNA used for determination of nucleotide sequence is preferably a high purity sample, as described above.<br>
The nucleotide sequence of the pML48 insert is shown in SEQ ID No. 1 of the Sequence Listing.   The nucleotide sequence shown in SEQ ID No. 2 of the Sequence Listing is completely complementary to the nucleotide sequence shown in SEQ ID No. 1.   Generally, a nucleotide sequence of a genomic DNA can have genetic polymorphisms within a species, that is, allogenic differences.   Furthermore, in the process of DNA cloning and sequencing, it is known that nucleotide substitutions, or other alterations, can occur at a certain frequency. Accordingly, the ML-236B biosynthesis-related genomic DNA of the present invention also<br><br>
22 includes genomic and other DNAs that can be hybridized to DNA of nucleotide No. 1 to<br>
34203 of SEQ ID No. 1 or 2 of the Sequence Listing.   Preferred are genomic or other DNAs that can be hybridized under stringent condition to DNA of nucleotide No. 1 to 34203 of SEQ ID No. 1 or 2 of the Sequence Listing.  These DNAs include the DNA of nucleotide No. 1 to 34203 of SEQ ID No. 1 or 2 of the Sequence Listing, wherein one or more nucleotides are substituted, deleted and/or added.   Additionally these hybridizing genomic or other DNAs can include DNA originating from ML-236B producing micro-organisms other than Penicillium citrinum SANK 133 80. preferably being those capable of improving the production of ML-236B when introduced into an ML-236B producing micro-organism.<br>
ML-236B biosynthesis related genomic DNA is suitably analyzed in accordance with the following methods 1) to 3).<br>
1) Analysis with gene analyzing software<br>
Genes within genomic DNA can be located using a program for finding genes (hereinafter referred to as "GRAIL"), and a program for searching homologous sequences (BLASTN and BLASTX).<br>
GRAIL is a program which searches for structural genes in genomic DNA by separating the genomic sequence into seven parameters for evaluation of the appearance of a gene sequence, and integration of the results using a neural net method [described in Uberbachen E.C. &amp; Mural. R.J.. Proc. Natl. Acad. Sci. USA.. 88, 11261 (1991)].   By way of example, the ApoCom GRAIL Toolkit [produced by Apocom corporation] can be used.<br>
BLAST is a program using an algorithm for performing homology searches of nucleotide sequences and amino acid sequences [described in Altschul, S.F., Madden, T. L., et al., Nucl. Acids Res., 25, 3389 (1997)].<br>
The position and direction of a structural gene on a sample genomic DNA sequence can be predicted by dividing the DNA sequence into suitable lengths and performing a<br><br>
23 homology search of a genetic data base using BLASTN.   The position and direction of<br>
structural gene on a DNA sequence to be tested can also be predicted by translating the divided genomic DNA sequences into the six translation frames (three on the sense strand and the other three on the antisense strand) and performing a homology search of the derived amino acid sequences in a peptide data base using BLASTX.<br>
Coding regions for structural genes in genomic DNA are sometimes split with introns in eukaryotic organisms.   For analysis of structural genes having such gaps, the BLAST program for sequences containing gaps is more effective, with Gapped-BLAST program (installed in BLAST2: WISCONSIN GCG package ver. 10.0) being preferred.<br>
2) Analysis according to Northern blot hybridization method<br>
Expression of a structural gene predicted by the analysis methods described in paragraph 1) can be studied using the Northern blot hybridization method.<br>
Suitably, total RNA from a ML-236B producing micro-organism is obtained from a culture of the micro-organism.   A culture of the preferred ML-236B producing microorganism Penicillium citrinum can be obtained by inoculating said micro-organism from a slant into MGB3-8 medium, followed by incubation with shaking, incubating at 22 to 28 °C for one to four days.<br>
The choice of method of extraction of RNA from an ML-236B producing microorganism is not limited, and preferred is the guanidine thiocyanate-hot phenol method, guanidine thiocyanate-guanidine hydrochloric acid method or the like.   Examples of a commercially available kit for preparing higher purity total RNA include RNeasy Plant Mini Kit (manufactured by Qiagen AG).   Furthermore. mRNA can be obtained by applying total RNA to an oligo (dT) column, and recovering the fraction adsorbed in the column.<br>
Transfer of RNA to a membrane, preparation of a probe, hybridization and detection of a signal can be performed in a similar manner to the above mentioned Southern blot<br><br>
24 hybridization method.<br>
3) Analysis of 5'-end and 3'-end of transcript.<br>
Analysis of the 5'-end and 3'-end of each transcript can be performed according to the 'RACE' (rapid amplification of cDNA ends) method.   RACE is a method for obtaining a cDNA comprising a known nucleotide region and an unknown region at the 5'-end or 3'-end of a gene, using RT-PCR with mRNA as a template [described in Frohman. M. A.. Methods Enzymoi. 218. 340 (1998)].<br>
5'-RACE can be performed according to the following method.   The first strand of a cDNA is synthesized according to a reverse transcriptase reaction using mRNA as a template. As a primer, antisense oligonucleotides (1) are used which are designed to a known part of a nucleotide sequence. A homopolymeric nucleotide chain (consisting of one kind of base) is added to the 3'-end of the first strand of the cDNA using terminal deoxynucleotidyl transferase.   Then, double stranded cDNA in 5'-end region is amplified by PCR using the first strand of the cDNA as a template.   For amplification. 2 primers are used: a DNA oligonucleotide from the sense strand containing a sequence complementary to the homopolymeric sequence, and an oligonucleotide (2) on the antisense strand and on the 3'-end side of the oligonucleotide DNA (1) [described in Frohman. M.A.. Methods in Enzymol.. 218, 340 (1993].   A kit for 5' RACE is commercially available, suitably the 5' RACE System for Rapid Amplification of cDNA ends. Version 2.0 (manufactured by GIBCO corporation).<br>
3' RACE is a method using the polyA region existing at the 3'-end of mRNA. Specifically, the first strand of cDNA is synthesized through a reverse transcriptase reaction using mRNA as a template and an oligo d(T) adapter as a primer.   Then, double stranded cDNA in 3'-end region is amplified by PCR using the first strand of the cDNA as a template. As primers, a DNA oligonucleotide (3) on the sense strand designed to a known part of the nucleotide sequence of the sense strand, and the oligo d(T) adapter on the antisense strand are used.   A kit for 3' RACE is commercially available, suitably the Ready-To-Go T-primed First-Strand Kit (Phramacia corporation).<br><br>
25<br>
The results of analysis 1) and 2) above are preferably used in the RACE procedure, in the design of the primers based upon a known part of the nucleotide sequence of interest.<br>
Using the methods of the analysis described in 1) to 3) above, the direction of a structural gene on a genomic DNA sequence, the location of transcription initiation site in the structural gene, the position of the translation initiation codon. and translation termination codon and position thereof can be deduced.   Based on the above information, each structural gene, and cDNA thereof, namely. ML-236B biosynthesis accelerating cDNAs can be obtained.<br>
Six structural genes are assumed to be present on the incorporated sequence in a recombinant DNA vector pML48 obtained according to the present invention.   They are named mlcA mlcB. mlcC. mlcD. mlcE, and mlcR. respectively.   Among them. mlcA m!cB. mlcE and mlcR are assumed to have a coding region on the nucleotide sequence shown in SEQ ID No. 2 of the Sequence Listing.   mlcC and mlcD are assumed to have a coding region on the nucleotide sequence shown in SEQ ID No. 1 of the Sequence Listing.<br>
Examples of a method for obtaining the specific ML-236B biosynthesis accelerating cDNAs corresponding to the above-mentioned structural genes include; cloning with RT-PCR using primers designed to the sequence of each of the structural genes and flanking DNA thereof and cloning from a cDNA library using appropriate DNA probes designed to known nucleotide sequences. Other suitable methods are well known in the art.   In order lo express functionally the cDNA obtained according to these methods, it is preferable to obtain a full length cDNA.<br>
A method for obtaining ML-236B biosynthesis accelerating cDNA using RT-PCR is explained below.<br>
A pair of primers for RT-PCR and for obtaining ML-236B biosynthesis accelerating cDNA needs to be designed so that it selectively anneals with each template chain, to allow cDNA to be obtained.   However, it is not essential that the primers for RT-PCR are<br><br>
26 completely complementary to a part of each template chain, provided that they satisfy the<br>
condition described above.   Suitable primers for RT-PCR that can anneal with the antisense chain (hereinafter referred to as 'sense primer") are sense primers that are completely complementary to a part of the antisense chain (hereinafter referred to as "unsubstituted sense primer") or sense primers that are not completely complementary to a part of the antisense chain (hereinafter referred to as "partially substituted sense primer").   The other suitable primers for RT-PCR that can anneal with the sense chain (hereinafter referred to as "antisense primer') are antisense primers that are completely complementary to a part of the sense chain (hereinafter referred to as "unsubstituted antisense primer") or antisense primers that are not completely complementary to a part of the sense chain (hereinafter referred to as "partially substituted antisense primer").<br>
A sense primer is suitably designed so that the RT-PCR product obtained using it contains the codon ATG at the original position of translation initiation.   Suitably the RT-PCR product also only contains the correct translation termination codon in the reading frame having the original ATG start site, and no additional (spurious) translational stop sites.   The position of the translation initiation codon of those structural genes predicted in the present invention is shown in Table 5 for genes located in SEQ ID No. 1 and SEQ ID No. 2 of the Sequence Listing.<br>
The 5'-end of the unsubstituted sense primer is suitably the nucleotide "A" of the translation initiation codon ATG. or a base existing on the 5'-end side thereof.<br>
A partially substituted sense primer selectively anneals with a specific region in SEQ ID No. 1 or SEQ ID No. 2 of the Sequence Listing, the nucleotide sequence of SEQ ID No. 2 of the Sequence Listing being completely complementary to SEQ ID No. 1 of the Sequence Listing.<br>
When a partially substituted sense primer contains a nucleotide sequence present on the 3'- side of the translation initiation codon ATG, it suitably does not contain nucleotide sequences in this region that are termination codons (TAA, TAG or TGA) in the same reading<br><br>
27 frame as the ATG.<br>
A partially substituted sense primer may contain nucieotide "A", nucleotide sequence "AT" or "ATG" (hereinafter referred to as Nucieotide or nucieotide sequence m) which correspond to nucieotide "A". nucieotide sequence "AT" or "ATG" of the translation initiation codon (hereinafter referred to as "nucieotide or nucieotide sequence m).   Where the nucieotide m is "A", corresponding to the "A" of sequence iVm", we prefer that the m "A" is located at 3'-end of the partially substituted sense primer.   Similarly, where m is "AT", we prefer that this m" "AT" sequence is located at 3'-end of the partially substituted sense primer. When the nucieotide or nucieotide sequence m is "ATG", corresponding to the m* "ATG". we prefer that those trinucleotides which are 3' to the ATG in the primer are not stop codons.   In other words, for trinucleotides whose 5"-end nucieotide is the (3 x n +l)th nucieotide (n represents an integer of one or more) counted from A of the m "ATG" in the direction of the 3'-end? the nucieotide sequence of the trinucleotide is preferably neither TAA, TAG nor TGA. Primers described above can be used to obtain cDNA having a methionine codon at the position corresponding to the translational initiation codon of mRNA used as an RT-PCT template.<br>
Where the 3'-end of a partially substituted sense primer is nucieotide position (3 x n + 1), preferably the trinucleotide which begins at this position is not TAA. TAG or TGA in the RT-PCR product obtained using the partially substituted sense primer as one of the primers, and RNA or mRNA of the ML-236B producing micro-organism as a template, or in the PCR products obtained by using genomic DNA or cDNA as a template.   The nucieotide position is counted from the 'A* of the translation initiation codon "ATG" in the direction of 3'-end, and where 'n' represents an integer of one or more.<br>
Where the 3'-end of a partially substituted sense primer is nucieotide position (3 x n + 2), the triplet for which position 3 x n+2 is the central nucieotide is preferably none of the sequences TAA, TAG or TGA for a PCR or RT-PCR product obtained as above.<br>
Where the 3'-end of a partially substituted sense primer is nucieotide position (3 x n+<br><br>
28 3). the triplet for which position (3 x n+ 3) is the 3' nucleotide is preferably none of the<br>
sequences TAA. TAG or TGA.<br>
The requirements for the sense primer are as discussed above.<br>
An antisense primer is designed so that, when paired together with the sense primer. cDNA encoding each of structural genes (mlcA. mlcB. mlcC. mlcD. mlcE and micR) can be amplified using RT-PCR in a direction equivalent to the N-terminus to C-terminus of the corresponding peptides.<br>
The choice of unsubstituted antisense primer is not limited, as long as it is an antisense primer having a nucleotide sequence complementary to a nucleotide sequence located in the region of the translational termination site of the cDNA.   However, a primer having a 5'-end base which is complementary to the base at 3'-end of translation termination codon. or having a base on the 5'-end side of said primer base, is preferred.   A primer containing three bases complementary to a translation termination codon is more preferred. Tables 8 to 10 show the translation termination codon of each structural gene, the sequence complementary to the translation termination codon. an amino acid residue at C-terminal of the peptide encoded by each structural gene, the nucleotide sequence encoding the amino acid residue, and position thereof in SEQ ID No. 1 or SEQ ID No. 2.<br>
Partially substituted antisense primers selectively anneal with a specific region in the nucleotide sequence of SEQ ID No. 1 or SEQ ID No. 2 of the Sequence Listing.<br>
The above are requirements for an antisense primer.<br>
It is possible to add suitable nucleotide sequences to the 5'-end of the partially substituted sense primers and the partially substituted antisense primers, as long as the above-mentioned requirements are satisfied.   The choice of such a nucleotide sequence is not particularly limited, as long as the primer can be used for PCR.   Examples of suitable sequences include nucleotide sequences convenient for the cloning of PCR products, such as<br><br>
29 restriction enzyme cleavage sites and nucleotide sequence containing suitable restriction<br>
enzyme cleavage sites.<br>
In addition, the sense primer and the antisense primer are suitably designed according to the above description and in accordance with the general design of primer for PCR.<br>
As described above. mRNA or total RNA from a ML-236B producing microorganism may be used as a template for RT-PCR.   In the present invention an ML-236B biosynthesis-accelerating cDNA corresponding to the structural gene mlcE was obtained by designing and synthesizing a pair of primers suitable to amplify all of coding region of the structural gene mlcE in the pML48 insert sequence and then performing RT-PCR using total RNA of SANK13380 as a template [primers represented by nucleotide sequences SEQ ID Nos. 35 and 36 of the Sequence Listing respectively].<br>
An ML-236B biosynthesis-accelerating cDNA corresponding to the structural gene mlcR was obtained in a similar way using primers represented by nucleotide sequences SEQ ID Nos. 39 and 40 of the Sequence Listing respectively.<br>
As described above, the RT-PCR product can be cloned by incorporation into a suitable DNA vector.   The choice of DNA vector used for such cloning is not limited, and is suitably a DNA vector generally used for cloning of DNA fragments. Kits for easily performing cloning of an RT-PCR product are commercially available, and the Original TA Cloning Kit [manufactured by Invitrogen: using pCR2.1 as DNA vector] is preferred.<br>
Confirmation of functional expression of the ML-236B biosynthesis accelerating cDNAs obtained using the above methods in an ML-236B producing micro-organism can be obtained by cloning the cDNA into a DNA vector suitable for functional expression in an ML-236B producing micro-organism.   Suitable cells are then transformed with the recombinant DNA vector, and the ML-236B biosynthesis ability of the transformed cells and non transformed host cells compared.   If ML-236B biosynthesis accelerating cDNA is functionally expressed in the transformed cell, then the ML-236B biosynthesis ability of the<br><br>
30 transformed cell is improved compared with that of a host cell.<br>
The choice of DNA vector suitable for expression in an ML-236B producing microorganism [hereinafter referred to as a functional expression vector] is not particularly limited, as long as it can be used to transform the ML-236B producing micro-organism and can functionally express the polypeptide encoded by the ML-236B biosynthesis accelerating cDNA in that organism.   Preferably the vector is stable in the host cell, and has a nucleotide sequence which allows replication in the host ceil.<br>
The vector for functional expression can contain one or more than one of ML-236B biosynthesis accelerating cDNAs. for example cDNAs corresponding to the structural genes mlcE and/or mlcR.<br>
A vector for functional expression may contain one or more than one kind of DNA. other than cDNA corresponding to the structural genes mlcE and/or m/cR. that accelerate biosynthesis of ML-236B when introduced into ML-236B producing micro-organism. Examples of such DNA include: cDNAs corresponding to structural genes mlcA, mlcB. mlcC, or mlcD, ML-236B biosynthesis related genomic DNA. DNA encoding expression regulatory factors of ML-236B biosynthesis accelerating cDNA of the present invention, or the like.<br>
A vector for functional expression preferably comprises a nucleotide sequence providing a selective phenotype for the plasmid in a host cell and is preferably a shuttle vector.<br>
Furthermore, the selective phenotype may be a drug resistance phenotype or the like, is preferably antibiotic resistance, and more preferably resistance to ampicillin or resistance to hydromycin B.<br>
In the case that the expression vector is a shuttle vector, the vector suitably comprises a nucleotide sequence which allows the vector to replicate in a host cell of one of the micro-organism groups, and a nucleotide sequence necessary for the expression of<br><br>
31 polypeptide encoded by the vector insert in another host cell type.   It is preferable that the<br>
vector affords a different selective phenotype for each host cell of the different microorganism groups transformed.   The requirements for combinations of micro-organism groups is similar to the requirement for the shuttle vector used for cioning and expression of ML-236B biosynthesis related genomic DMA described in the present specification.<br>
In the present invention, a suitable shuttle vector DNA vector is pSAK.700. constructed by combining the 3-phosphoglycerate kinase (hereinafter referred to as "pgk") promoter originating from Aspergillus nidulam existing in the DNA vector pSAK333 (described in Japanese Patent Application Publication No.3-262486), an adapter for incorporating a foreign gene, and pgk terminator existing in the DNA. in this order (see Figure<br>
4).<br>
A polypeptide can be expressed in an ML-236B producing micro-organism by incorporating the cDNA corresponding to the strutural gene m/cE. described above, into the expression vector described above.   In the present invention, a recombinant cDNA expression vector pSAKexpE has been obtained by incorporating cDNA corresponding to the strutural gene mlcE into an adopter site of pSAK700.   The incorporated sequence in pSAKexpE. namely the nucleotide sequence of cDNA corresponding to the strutural gene mlcE is shown in SEQ ID No. 37 of the Sequence Listing.   Similarly a recombinant cDNA expression vector pSAKexpR has been obtained by incorporating cDNA corresponding to the strutural gene mlcR into an adapter site of pSAK700.   The incorporated sequence in pSAKexpR. namely the nucleotide sequence of cDNA corresponding to the strutural gene mlcR is shown in SEQ ID No. 41 of the Sequence Listing.<br>
Escherichia coli pSAKexpE SANK 72499 that is Escherichia coli strain transformed by pSAKexpE was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on January 25, 2000 under the Deposit Nos FERM BP-7005, in accordance with the Budapest Treaty on the Deposition of Microorganisms.   Escherichia coli pSAKexpR SANK 72599 that is Escherichia coli strain transformed by pSAKexpR was deposited at the Research Institute of Life Science and<br><br>
32 Technology of the Agency of Industrial Science and Technology on January 25. 2000 under<br>
the Deposit Nos FERM BP-7006. in accordance with the Budapest Treaty on the Deposition of Micro-organisms.<br>
Suitable methods of transformation can be appropriately selected- depending on the host cell, to obtain expression of ML-236B biosynthesis accelerating cDNA. ML-236B biosynthesis related genomic DNA or fragments thereof.   Transformation of Penicillium cithnum, a preferred ML-236B producing micro-organism, can be performed by preparing protoplasts from spores of Penicillium citrinum. then introducing recombinant DNA vector into the protoplast [described in Nara. R. et al.. Curr. Genet 23.28 (1993)].<br>
Suitably spores from a slant of culture of Penicillium citrinum are inoculated on a plate of PGA agar medium and incubated at 22 to 28 °C. for 10 to 14 days.   The spores are then harvested from the plate and 1 x 107 - 1 x 109 spores inoculated into 50 to 100 ml of YPL-20 culture medium [composition: 0.1% (w/v) yeast extract (manufactured by Difco corporation), 0.5 % (w/v) polypeptone (manufactured by Nihon Seiyaku corporation). 20 % (w/v) of lactose, pH5.0]. then incubated at 22 to 28 °C for 18 hours to two days.   The germinating spores are recovered from the culture, and treated with cell wall degrading enzymes to yield protoplasts.   The choice of cell wall degrading enzyme is not particularly limited, as long as it can degrade the cell wall of Penicillium citrinum and does not have a harmful effect on the micro-organism.   Example thereof include: zymolyase, chitinase or the like.<br>
Mixing of a recombinant DNA vector comprising an ML-236B biosynthesis accelerating cDNA and ML-236B producing micro-organism, or the protoplast thereof, under suitable conditions allows introduction of the recombinant DNA vector into said protoplast, to provide a transformant.<br>
Cuituring of transformants of ML-236B producing micro-organism is suitably performed under conditions suitable for each of the host-cell.   Cuituring of a transformant of Penicillium citrinum, a preferred ML-236B producing micro-organism, can be performed by<br><br>
33 culturing the previous transformed protoplast under conditions appropriate to regenerate a cell<br>
wall, and then culturing.   Namely, the transformed protoplast of PeniciUium citrinum may be introduced into VGS middle layer agar medium [composition: Vogel minimum medium. 2 %(w/v) glucose. 1M glucitol. 2 %(w/v) agar]. the VGS middle layer agar then sandwiched between VGS lower layer agar medium [composition: Vogel minimum medium. 2 %(w/v) glucose, 1M glucitol. 2.7 %(w/v) agar] and VGS upper layer agar medium [composition: Vogel minimum medium. 2 %(w/v) glucose. 1M glucitol. 1.5 %(w/v) agar] containing 800 ?g/ml hygromycin B. then incubated at 22 to 28°C for 7 to 15 days.   The resultant strain is subcultured with incubation at 22 to 28°C on PGA medium.   The strain is inoculated with a platinum needle to a slant prepared of PGA medium, incubated at 22 to 28°C for 10 to 14 days, and then reserved at 0 to 4°C.<br>
As described above. ML-236B can be efficiently produced by inoculating a PeniciUium ciirimtm transformant obtained from a slant as above, and having a regenerated cell wall, into MBG 3-8 medium, followed by incubation at 22 to 28°C for 7 to 12 days with shaking.    PeniciUium ciirinum as a host can be cultured in liquid medium as well to produce ML-236B.<br>
Purification of ML-236B from culture of a transformant of ML-236B producing micro-organism can be performed by combining various methods generally used for purification of natural products.   The choice of such methods is not particularly limited, and can be, for example, by centrifugation. separation of solids and liquids by filtration, treatment with alkali or acid, extraction with organic solvents, dissolution, chromatography methods such as adsorption chromatography. partition chromatography or the like, and crystallization or the like.   ML-236B can be in either hydroxy acid or lactone form, which may be reciprocally converted.   The hydroxy acid is convertible to a salt thereof that is more stable. Using such physical properties, the ML-236B hydroxy acid form (hereinafter referred to as free hydroxy acid), salts of ML-236B hydroxy acid (hereinafter referred to as a salt of hydroxy acid), or the ML236B lactone form (hereinafter referred to as lactone) can be obtained.<br>
The culture is subjected to alkaline hydrolysis at raised temperature or room<br><br>
34 temperature for ring opening and conversion to a salt of hydroxy acid, and then the reaction<br>
solution is acidified, followed by filtration.   The filtrate is extracted with an organic solvent that separates from water to provide an intended product as a free hydroxy acid.   The choice of organic solvent is not particularly limited.   Examples thereof include: aliphatic hydrocarbons such as hexane. heptane or the like; aromatic hydrocarbons such as benzene, toluene or the like; haiogenated hydrocarbons such as methylene chloride, chloroform or the like: ethers such as diethyl ether or the like; esters such as ethyl formate, ethyl acetate or the like; or a mixture consisting of two or more solvents.<br>
The intended compound can be obtained as a hydroxy acid salt by dissolving the free hydroxy acid in an aqueous solution of an alkaline metal salt such as sodium hydroxide.<br>
Furthermore, the intended compound can be obtained as iactone through ring closure by heating the free hydroxy acid in an organic solvent to be dehydrated, or by other suitable methods.<br>
It is possible to purify and isolate the free hydroxy acid, hydroxy acid or Iactone thus obtained using column chromatography or the like.   The support for the column used in chromatography is not particularly limited.   Examples thereof include: Sephadex LH-20 (produced by Pharmacia corporation). Diaion HP-20 (produced by Mitsubishi Kagaku corporation), silica gel. reversed phase supports or the like, with supports of the C18 series preferred.<br>
The choice of a method for quantification of ML-236B are not particularly limited, preferably being a method generally used for quantification of organic compounds.   Examples thereof include: reversed phase high performance liquid chromatography (hereinafter referred to as "reverse phase HPLC') or the like.   Quantification according to the reverse phase HPLC can be performed by subjecting a culture of an ML-236B producing micro-organism to alkaline hydrolysis, subjecting the soluble fraction to reverse phase HPLC using a C18 column, measuring UV absorption, and converting the absorption value to an amount of ML-236B.   Choice of C18 column is not particularly limited, preferably being a C18 column used<br><br>
35 for general reverse phase HPLC.   Examples thereof include: SSC-ODS-262 (diameter of 6<br>
mm. length of 100 mm. manufactured by Senshu Kagaku corporation) or the like.   The choice of solvent for the moving phase is not particularly limited, so long as it is a solvent generally used for reverse phase HPLC.   It is. for example. 75%(v/v) methanol - 0.1 %(v/v) triethyl amine - 0.1%(v/v) acetic acid or the like.   When ML-236B is added at room temperature to an SSC-ODS-262 column, where 75%(v/v) methanol - 0.1%(v/v) triethyi amine -0.1%(v/v) acetic acid is used as moving phase at a rate of 2 ml/minute. ML-236B is eluted after 4.0 minutes.   ML-236B can be detected using a UV detector for HPLC.   The absorbed wave length for UV detection is 220 to 280 nm. preferably 220 to 260 nm. more preferably 236 nm.<br>
Pharmaceutical compositions are provided containing ML-236B obtained using the present invention, together with a pharmaceutical carrier.<br>
Pharmaceutical compositions are also provided containing pravastatin prepared from ML-236B obtained using the present invention, together with a pharmaceutical carrier.<br>
The pharmaceutical compositions of this invention can be conventional and the same as those employed for existing formulations of ML-236B or pravastatin.<br>
Methods of treatment are also part of this invention and employ the compounds or compositions to treat hyperlipemia and other conditions.<br>
The invention is now illustrated in more detail with reference to the following Figures and Examples. The Examples are illustrative of. but not binding upon, the present invention.<br>
DESCRIPTION OF TUB FIGURES<br>
Figure 1 is a diagram depicting the construction of DNA vector pSAKcosl;<br>
Figure 2 is the results of structural gene analysis of the inserted sequence of pML48;<br><br>
36 Figure 3 shows Northern blot hybridization of the inserted sequence of pML48:<br>
Figure 4 is a diagram depicting the construction of cDNA expression vector pSAK.700: and<br>
Figure 5 shows RT-PCR analysis for transcription of mlc A-E and R in a pSAKexpR transformant.<br>
Figure 6 shows RT-PCR analysis for transcription of mlcE in a pSAKexpE transformant.<br>
EXAMPLES OF THE INVENTION<br>
Example 1: Construction of pSAKcosl vector<br>
Plasmid pSAK333 containing the hygromycin B phosphotransferase gene (hereinafter referred to as "HPT") originating from Escherichia coli (Japanese Patent Application Publication No. 3-262486) was digested with restriction enzyme BamHI (manufactured by Takara Shuzo Co., Ltd.. Japan), and was treated to form blunt ends with T4DNA polymerase (manufactured by Takara Shuzo Co.. Ltd.. Japan).<br>
The DNA fragment obtained as above was self-ligated into a circular form using DNA ligation kit Ver.2 (manufactured by Takara Shuzo Co.. Ltd.. Japan), and competent cells JM 109 of Escherichia coli (manufactured by Takara Shuzo Co.. Ltd.. Japan) were then transformed therewith.   A strain having a plasmid in which the BamHI site was deleted was selected from the transformed Escherichia coli. and was designated pSAK.360.<br>
pSAK 360 was digested with restriction enzyme PvuII, and then treated with alkaline phosphatase to produce a fragment dephosphorylated at 5'-end.   A Sall-Scal'fragment (about 3kb) containing a cos site was obtained from a cosmid vector pWE15 (manufactured by STRATAGENE) and was treated to form blunt ends with T4 DNA polymerase.   It was<br><br>
37 subsequently ligated to the PvuII site of pSAK360. JMI09 was transformed with this DNA.<br>
Those strains having a plasmid into which Sall-Scal fragment (about 3kb) was inserted at the PvuII site were selected from the transformed Eschehchia coli. and the plasmid carried by the strain was designated pSAKcosl.   pSAKcosl contains a cleavage site for the restriction enzymes BamHI. EcoRI and Notl. each site originating from pWEl 5.   The pSAKcosl has an ampicillin resistance gene and a hygromycin resistance gene as selection markers.<br>
In the following examples, where Eschehchia coli was used as a host, selection of pSAKcosl transformants. or transformants of pSAKcosl comprising a foreign gene-insert, was performed by adding 40 ?g/ml ampicillin (Ampicillin: manufactured by Sigma corporation) to the relevant medium.   Where Penicillium ciirinum SANK 13380 was used as a host, selection of pSAKcosl transformants. or transformants of pSAKcosl comprising a foreign gene-insert, was performed by adding 200 jig/ml hygromycin (Hygromycin B: manufactured by Sigma corporation) to the relevant medium.<br>
The method of construction of pSAKcosl is shown in Fig.l<br>
Example 2: Preparation of genomic DNA of Penicillium citrinum SANK 13380<br>
1)	Culture of Penicillium cithnum SANK. 13380<br>
A seed culture of Penicillium cithnum SANK 13380 was made on a slant of PGA agar medium. Namely; the agar was inoculated with Penicillium citrinum SANK 13380 using a platinum needle, and kept at 26°C for 14 days.   The slant was kept at 4°C.<br>
Main culturing was performed by liquid aeration culture. Cells from a 5 mm square of the above-mentioned slant were inoculated in 50 ml of MBG3-8 medium in 500 ml conical flask, and incubated at 26°C with shaking at 210 rpm for five days.<br>
2)	Preparation of genomic DNA from Penicillium citrinum SANK 13380<br><br>
38<br>
The culture obtained in step 1) was centrifuged at 10000 x G at room temperature for 10 minutes, and cells were harvested.   3g (wet weight) of ceils were broken in a mortar cooled with dry ice so as to be in the form of a powder.   The broken cells were put in a centrifuge tube filled with 20 ml of 62.5mM EDTA-2Na (manufactured by Wako Pure Chemical Industries. Ltd.) - 5% (w/v) SDS - 50mM Tris hydrochloric acid (manufactured by Wako Pure Chemical Industries. Ltd.) buffer (pHS.O). and were mixed gently, then allowed to stand at 0°C for one hour. 10 ml of phenol saturated with 10 mM Tris hydrochloric acid - 0.1 mM EDTA-2Na (pH 8.0. hereinafter referred to as "TE") were added thereto, and the mix stirred gently at 50°C for one hour.<br>
After centrifugation at room temperature at 10000 x G for 10 minutes. 15 ml of the upper layer (water phase) was placed into another centrifuge tube. To the solution were added 0.5 times by volume of TE saturated phenol and 0.5 times by volume of chloroform solution. The mixture was stirred for two minutes and centrifuged at room temperature at 10000 x G for 10 minutes (hereinafter referred to as "phenol chloroform extraction").   To 10 ml of the upper layer (water phase) was added 10 ml of 8M ammonium acetate (pH 7.5) and 25 ml of 2-propanol (manufactured by Wako Pure Chemical Industries. Ltd.), followed by cooling at -80°C for 15 minutes, and centrifugation at 4 °C at 10000 x G for 10 minutes.<br>
After precipitation, the precipitates were dissolved in 5 ml of TE. after which 20 (il of 10 mg/ml ribonuclease A (manufactured by Sigma corporation) and 250 units of ribonuclease Tl (manufactured by GIBCO corporation) were added thereto, followed by incubation at 37°C for 20 minutes.   20 ml of 2-propanol was added thereto, and mixed gently.   Subsequently, threads of genomic DNA were spooled at the tip of a Pasteur pipette, and dissolved in one ml of TE.<br>
Next 0.1 times by volume of 3 M sodium acetate (pH6.5) and 2.5 times by volume of ethanol were added'to the DNA solution.   The solution was cooled at - 80°C for 15 minutes, and then, centrifuged.at 4 °C, at 10000 x G for five minutes (herein after referred to as "ethanol-precipitation").   The resultant precipitate was dissolved in 200 ?l of TE, and was a<br><br>
39 genomic DNA fraction.<br>
Example 3: Preparation of genomic DNA library of Penicillium citrinum SANKB380<br>
1) Preparation of genomic DNA fragment<br>
0.25 units of Sau3Al (Takara Shuzo Co.. Ltd.. Japan) were added to 100 ?l of an aqueous solution of genomic DNA (50 ?g) of Penicillium citrinum SANK13380 obtained in Example 2.   After intervals of 10.30. 60. 90 and 120 seconds. 20 ?l samples of the mixture were taken, and 0.5 M EDTA (pH 8.0) was added to each sample to terminate the restriction enzyme reaction.  The resulting partially digested DNA fragments were separated by agarose gel electrophoresis, and agarose gel was recovered containing DNA fragments of 30 kb or more<br>
The recovered gel was finely crushed, and placed into Ultra Free C3 Centrifuged Filtration Unit (manufactured by Japan Milipore corporation).   The gel was cooled at -80°C for 15 minutes until frozen, and then the gel was melted by incubating it at 37°C for 10 minutes.   It was centrifuged at 5000 x G for 5 minutes, to extract DNA.   The DNA was subjected to phenol - chloroform extraction and ethanol precipitation.   The resulting precipitates were dissolved in a small, appropriate amount of TE.<br>
2)	Pretreatment of DNA vector pSAKcosl<br>
pSAKcosl was digested with restriction enzyme BamHI (Takara Shuzo Co.. Ltd.. Japan), and then subjected to alkaline phosphatase (Takara Shuzo Co., Ltd.. Japan) treated at 65°C for 30 minutes.   The resultant reaction solution was subjected to phenol - chloroform extraction and ethanol precipitation.   The resulting precipitation was dissolved in a small amount of TE.<br>
3)	Ligation and in vitro packaging<br><br>
40 The genomic DNA fragment (2 fig) described in the above step 1) and pSAKcosl (1<br>
?g) subjected to pretreatmem as above were mixed, and then Iigated at 16°C for 16 hours using DNA ligation kit Ver.2 (Takara Shuzo Co.. Ltd.. Japan).   The resultant reaction solution was subjected to phenol - chloroform extraction and ethanol precipitation.   The resulting precipitates were dissolved in 5 ?l of TE.   The ligation product solution was subjected to in vitro packaging using the G1GAPAK II Gold kit (manufactured by STRATAGENE corporation) to provide Escherichia coli transformants containing a recombinant DNA vector. 3 ml of LB medium were poured onto a plate on which colonies of Escherichia coli transformant had formed, and then the colonies on the plate were recovered using a cell scraper (referred to as "recovered solution 1"}.   The plate was washed with a further 3 ml of LB medium, and cells recovered (referred to as "recovered solution 2").   Glycerol was added to a mixture of recovered solution 1 and 2. to achieve a final concentration of 18 % (referred to as Escherichia coli cell solution), which was kept at - 80°C as a genomic DNA library of Penicitlium citrinum SANK.13380.<br>
Example 4: Amplification of PKS gene fragment by PCR using genomic DNA of Penicitlium citrinum SANK13380 as a template<br>
1) Design and synthesis of primers for PCR.<br>
Based on the amino acid sequence of a PKS gene of Aspergillus flavus (described in Brown, D.W.. et al., Proc. Natl. Acad. Sci. USA. 93. 1418 (1996)). degenerate primers shown in SEQ ID Nos. 3 and 4 of the Sequence Listing were designed and synthesized. The synthesis was performed according to the phosphoamidite method.<br>
SEQ ID No. 3 of the Sequence Listing: gayacngcntgyasttc<br>
SEQ ID No. 4 of the Sequence Listing: tcnccnknrcwgtgncc<br><br>
41 In the nucieotide sequence of SEQ ID Nos. 3 and 4. n represents inosine<br>
(hypoxanthine), y represents t or c. s represents g or c. k represents g or t. r represents g or a. and w represents a or t.<br>
2} Amplification of DNA segment by PCR<br>
50 ?l of reaction solution was prepared containing the primers for PCR described in the above step l)(each 100 pmol). genomic DNA of Pemcillium citrirmm SANK13380 obtained in Example 2 (500 ng). 0.2 mM of dATP. 0.2 mM of dCTR 0.2 mM of dGTP. 0.2 mM of dTTP. 50 mM of potassium chloride. 2 mM of magnesium chloride and 1.25 units of Ex. Tac DNA polymerase (Takara Shuzo Co.. Ltd.. Japan).   The solution was subjected to a reaction cycle consisting of three consecutive steps as follows: one minute at 94°C two minutes at 58°C and 3 minutes at 70°C.  The cycle was repeated 30 times to amplify- the DNA fragment.  PCR was performed using TaKaRa PCR Thermal Cycler MP TP 3000 (manufactured by Takara Shuzo Co.. Ltd.. Japan).<br>
The amplified DNA fragments were subjected to agarose gel electrophoresis. and then agarose containing DNA fragments having a size of about 1.0 to 2.0 kb were recovered. DNA was recovered from the gel. and subjected to phenol-chloroform extraction and ethanol precipitation.   The resulting precipitate was dissolved in a small amount of TE.<br>
3) Ligation and transformation<br>
The DNA fragment obtained in step 2) was ligated to the plasmid pCR2.1 using the TA cloning system pCR 2.1 (manufactured by lnvitrogen corporation), the plasmid being provided as part of the kit.   The plasmid was transformed into Escherichia coli JM109 to provide transformants.<br>
Several colonies were selected from the resulting transformants. and were cultured according to the method of Maniatis? et a!., [described in Maniatis, T., et al., Molecular cloning, a laboratory manual, 2nd ed., Cold Spring Harbor Laboratory, Cold Spring Harbor,<br><br>
42 NY. (1989)].   Namely, each of the colonies was inoculated into a 24 ml test tube containing 2<br>
ml of LB medium, and was incubated at 37°C for 18 hours with shaking.<br>
A recombinant DNA vector was prepared from the culture according to the alkaline method (described in Maniatis. T.. et al., supra).   Namely. 1.5 ml of the culture solution was centrifuged at room temperature at 10000 x G for two minutes.   Cells were then recovered from the precipitate.   To the cells were added 100 ?l of a solution of 50 mM glucose. 25 mM Tris - hydrochloric acid. 10 mM EDTA (pH 8.0). to form a suspension.   Thereto was added 200 \x\ of 0.2 N sodium hydroxide - l%(w/v) SDS.   The suspension was stirred gently, to lyse the micro-organisms. 150 ul of 3 M potassium acetate - 11.5%(w/v) acetic acid was then added to denature any protein, followed by centrifugation at room temperature at 10000 x G for 10 minutes.   The supernatant was recovered.   The supernatant was subjected to phenol -chloroform extraction and ethanol precipitation.   The resulting precipitate was dissolved in 50 [il of TE containing 40 jig/ml of ribonuclease A (manufactured by Sigma corporation).<br>
Each of the recombinant DNA vectors was digested with restriction enzymes, and subjected to electrophoresis.   The nucleotide sequences of the DNA inserts in the recombinant DNA vectors were determined using a DNA sequencer (model 377: manufactured by Perkin Elmer Japan) for all inserts having different digestion patterns on electrophoresis.<br>
In this way a strain was identified having a recombinant DNA vector containing a PKS fragment derived from Penicillium citrinum.<br>
Example 5: Genomic Southern blotting hybridization of Penicillium citrinum SANK13380<br>
1) Electrophoresis and transfer to membrane<br>
The genomic DNA (10 ?g) of Penicillium citrinum SANK13380 obtained in Example 2 was digested with restriction enzymes EcoRI, SaII, HindIII or Sacl (all manufactured by<br><br>
43 Takara Shuzo Co.. Ltd.. Japan), and then subjected to agarose gel electrophoresis.   The gel<br>
was made using agarose L03 "TAKARA" (Takara Shuzo Co.. Ltd.. Japan).   After eleetrophoresis, the gel was soaked in 0.25 N hydrochloric acid (manufactured by Wako Pure Chemical industries. Ltd.). and incubated at room temperature for 10 minutes with gentle shaking.   The gel was transferred to 0.4 N sodium hydroxide (manufactured by Wako Pure Chemical Industries, Ltd.). and gently incubated at room temperature for 30 minutes.   Using the alkaline transfer method of Mamatis et al (supra). DNA in the gel was transferred onio a nylon membrane Hybond ™-N+ (manufactured by Amersham corporation), and fixed thereon.   The membrane was washed with 2 x SSC (1 x SSC contains 150 mM NaCl. 15 mM sodium citrate), and then air-dried.<br>
2) Hybridization and detection of signal<br>
The membrane obtained in step 1) was hybridized with the PKS gene fragment obtained in Example 4 as a probe.<br>
For the probe. 1 ?g of the PKS gene insert fragment DNA obtained in Example 4 was labeled with a DIG DNA Labeling Kit (manufactured by Boeringer-Mannheim) and was boiled for 10 minutes and then rapidly cooled just prior to use.<br>
The membrane described in step 1) was soaked in hybridization liquid (DIG Easy Hyb: manufactured by Boeringer-Mannheim). and then subjected to prehybridization with shaking at 20 rpm at 42°C for 2 hours.   Then, the above-mentioned labeled probe was added to the hybridization liquid, and hybridization was performed with shaking at 20 rpm at 42°C for 18 hours using Multishaker Oven HB (manufactured by TAITEC corporation).   The membrane subjected to hybridization was then subjected to three washes using 2 x SSC at room temperature for 20 minutes, and two washes using 0.1 x SSC at 55°C for 30 minutes.<br>
The washed membrane was treated with DIG Luminescent Detection Kit for Nucleic Acids (manufactured by Boeringer-Mannheim) and exposed to X ray film (Lumifilm, manufactured by Boeringer-Mannheim).   Exposure was performed using Fuji medical film<br><br>
44 processor FPM 800A (manufactured by Fuji Film Corporation).<br>
As a result, it was confirmed that the PKS gene fragment obtained in Example 4 existed on the genome of Penicillium citrinum .<br>
Example 6: Screening of genomic DNA library of Penicillium citrinum SANK13380 using PKS gene fragment as a probe<br>
Cloning of a genomic DNA fragment containing a PKS gene was performed using a colony hybridization method.<br>
1)	Preparation of membrane<br>
The Escherichia coli cell solution kept as a genomic DNA library of Penicillium citrinum SANK13380 (described in Example 3) was diluted and spread on a LB agar medium plate, such that 5000 to 10000 colonies might grow per plate.   The plate was kept at 26°C for 18 hours, and cooled at 4°C for one hour.   Hybond ™-N+ (manufactured by Amasham corporation) was placed on the plate, and brought into contact therewith for one minute.   The membrane on which the colony was adhered was carefully removed from the plate.   The surface which had been in contact with the colonies was turned upward and soaked in 200 ml of a solution of 1.5 M sodium chloride. 0.5 N sodium hydroxide for 7 minutes, and then soaked in 200 ml of a solution of 1.5 M sodium chloride. 0.5 M Tris hydrochloric acid, 1 mM EDTA (pH 7.5) for three minutes twice, and then washed with 400 ml of 2 x SSC.   The washed membrane was air-dried for 30 minutes.<br>
2)	Hybridization<br>
The PKS gene insert DNA obtained in Example 4(1 ?g) was used as a probe.   The DNA was labeled with using a DIG DNA Labeling Kit (manufactured by Boeringer-Mannheim) and was boiled for 10 minutes and rapidly cooled just prior to use.<br><br>
45 The membrane described in step 1) was soaked in hybridization liquid (DIG Easy Hyb:<br>
manufactured by Boeringer-Mannheim). and then subjected to a prehybridization wash at 20 rprn at 42°C for 2 hours.   Then, the above-mentioned labeled probe was added to the hybridization liquid, and hybridization was performed at 20 rpm at 42°C for 18 hours using Multishaker Oven HB (manufactured by TAITEC corporation).   The membrane subjected to hybridization was subjected to three washes using 2 x SSC at room temperature for 20 minutes, and two washes using 0.1 x SSC at 68°C for 30 minutes.<br>
The washed membrane was treated with DIG Luminescent Detection Kit for Nucleic Acids (manufactured by Boeringer-Mannheim). and exposed to X ray film (Lumifilm. manufactured by Boeringer-Mannheim).   Exposure was performed using Fuji medical film processor FPM 800A (manufactured by Fuji Film Corporation).<br>
The above steps 1) and 2) are referred to as Screening.<br>
Colonies on the plate where the positive signal was detected at the first screening was scraped and recovered cells suspended in LB medium.   Then, cells were diluted adequately and spread on a suitable plate.   Subsequently, a second screening was performed to purify the positive clone.<br>
The positive clone obtained in the present example, namely transformed Escherichia coli Escherichia coii pML48 SANK71199 strain was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on July 7. 1999 under the Deposit Nos FERM BP-6780. in accordance with the Budapest Treaty on the Deposition of Micro-organisms.<br>
Example 7: Analysis of the inserted sequence of a recombinant DNA vector pML48 (1)<br>
Culturing of Escherichia coli pML48 SANK7U99 strain obtained in Example 6 and preparation of a recombinant DNA vector from the culture were performed in a similar manner to that described in Example 4.<br><br>
46<br>
ML-.6B bosynthesis related genimic DNA, was digested with various restriction enzymes and resulting subcloned into pUC119 (manufactured by Takara Shuzo<br>
CO  Ltd  Japan).   Using the result, Subclone as probes. Souths blot hybridization was performed by a similar method to that described in Example 5.   Namely, the products obtained by digesting pML48 with various restriction enzymes were subjected to decuophorcsis, and the DNAs were transferred to a membrane, and subjected to hybridization.   As a result, a restriction enzyme cleavage map of the inserted sequence of pML48 was made using techniques standard in the art.<br>
The nucleotide sequence of the inserted sequence of each of the subclones was determined using DNA sequencer model 377 (manufactured by Perkin Elmer Japan Co.Ltd.), followed by determination of the entire nucleotide sequence of pML8<br>
The insened sequence of pML4S consisted of 34203 bases in total.<br>
The nucleotide sequence of the inserted sequence of pML48 is described in SEQ ID Nos- 1 and 2 of the Sequence Listing. Tlie sequences described in SEQ ID Nos. 1 and 2 of the Sequence Listing are completely complementary with each other.<br>
Existence of structural genes on the PML48 insert sequence was analyzed using a gene searching program GRAIL (ApoCom GRAIL Toolkit: produced by Apocom Corporation) and a homology searching program BLAST (Gapped-BLAST (BLAST2): installed in WISCONSIN GOG package ver.10.0).<br>
As a result, six different structural genes were predicted to exist in the inserted sequence of pML48, and were designated mlcA, rnlcB, mlcC, mlcD, mlcE and mlcR respectively.   Furthermore, it was predicted that mlcA.. mlcB, mlcE and mlcR have a coding region in the nucleotide sequence of SEQ ID NO. 2 of the Sequence L.sting. and mlcC and mlcD have a coding region in the nucleotide sequence of SEQ ID NO. 1 of the Sequence<br><br>
47<br>
Listing,   The relative position and length of each of the presumed structure genes of the inserted sequence were also predicted.<br>
The results of the present example are shown in Figure 2.   Each arrow indicates localization, direction and relative size of each structural gene on the pML48 insert.   An arrow which points to the left indicates that the coding region of a structural gene exists (mlcA, B, E or K) exists on ID SEQ NO 2" An arrow which'points to the right indicates that the coding region of a structural gene (mlcC or D) exists on ID SEQ NO 1.<br>
Example 8; Analysis of the inserted sequence of a recornbinant DNA vector pML4$ (2)<br>
Analvsis of expression of the structural genes whose existence was predicted in Example 7 was carried out by Northern blot hybridization and RACE. Analysis of 5'- and 3'-end regions was performed.<br>
1) Preparation of total RNA of Penicillium citrinum SANK13380<br>
Cells from a 5 nun square in the Pemcillium citrinum SANK1338O slant culture (described in Example 2) were inoculated into 10 ml MGB3-8 medium in a 100 ml conical flask, and incubated at 26oC for 3 days with shaking.<br>
Preparation of total RNA from the culture was performed with the RNeasy Plant Mini Kit (manufactured by Qiagen AG) which uses the guanidine.- isothiocyanate method, Namely, the culture was centrifuged at room temperature at 5000.x G for 10 minutes to recover cells-   Subsequently, 2g (wet weight) of the cells were frozen with liquid nitrogen and then crushed in a mortar to form a powder.   The crushed cells were suspended in 4 ml of buffer for lysis (comprised in the kit).   150 ?l of the suspension was poured into each of 10 of QIAshredder spin columns contained in the kit, and then centrifuged at room temperature at 1000 x G for 10 minutes.   Each of the resultant eluents was recovered, and 225 ?l of ethanol was added thereto, which was then added to an RNA mini spin column contained in the kit.<br><br>
The column was washed with buffer for washing contained in the kit. followed by elation of adsorbs in each column with 50 ?l of ribonuclease free distilled water.   The eluent was used as total RNA fraction.<br>
2) Northern blot hybridization<br>
An RNA sample was produced by adding 2.25 ?l of an aqueous solution containing 20 ug of total RNA of Pemcillium citrinum SANK 13380 to: one ?l of 10 x MOPS (composition: 200 mM 3-morpholino propane sulfonic acid. 50 mM sodium acetate. 10 mM EDTA-2Na: pH 7.0; used after sterilization at 121 °C for 20 minutes in an autoclave: manufactured by Dojinkagaku Laboratory Co.Ltd.). 1.75 ?l of formaldehyde and 5 ?l of formamide. followed by mixing.   The RNA sample was kept at 65°C for 10 minutes, then rapidly cooled in ice water, and subjected to agarose gel electrophoresis.   The gel for the electrophoresis was prepared by mixing 10 ml of 10 x MOPS and one gram of Agarose L03 "TAKARA" (manufactured by Takara Shuzo Co.. Ltd.. Japan) with 72 ml of pyrocarbonic acid diethyl ester treatment water (manufactured by Sigma Corporation), heated to dissolve the agarose. and then cooled, followed by addition of 18 ml of formaldehyde.   As the sample buffer. 1 x MOPS (prepared by diluting 10 x MOPS with 10 times water) was used.   RNA in the gel was transferred to Hybond ™-N+ (manufactured by Amasham corporation) in 10 X SSC.<br>
DNA fragments a. b. c. d and e. obtained by digesting the inserted sequence of pML48 with the restriction enzymes 1 and 2 shown in the following Table 1. were used as probes. Localization of each probe on the pML48 insert is shown in the upper panel of Figure 3.<br>
Table 1 Probe for Northern blot hybridization<br><br><br><br>
49<br>
* Each nucieotide No. exists on SEQ ID  NO. 1 of the sequence Listing<br>
Labeling of probes, hybridization and detection of signal were performed according to Southern blot hybridization described in Example 5.<br>
The results of the Example are shown in the lower panel of Figure 3.<br>
Each signal shows the existence of a transcription product homologous to the nucieotide sequence of each probe.<br>
The results suggest that the structural genes predicted to exist in the inserted sequence of pML48 in the present example, namely mlcA mlcB. mlcC mlcD, mlcE and mlcR were transcribed in Pemcillium citrinwn SANK13380.<br>
The position of each signal does not show the relative size of the transcription product. 3) Determination of 5'-end sequence according to 5'RACE<br>
cDNA containing the 5'-end region of each structural gene was obtained using 5' RACE System for Rapid Amplification of cDNA ends, Version 2.0 (manufactured by GIBCO corporation).<br><br>
50 Two kinds of antisense oligonucleotide DNAs were produced. The design was based on the<br>
nucleotide sequence presumed to be in the coding region and near the 5'-end of each structural gene in the inserted sequence of pML48, as predicted by the results of Example 7 and the item 2) of the present example.<br>
The nucleotide sequence of the antisense oligonucleotide DNA (1) designed based on the nucleotide sequence on the 3'-end side of each structural gene is shown in Table 2.   The nucleotide sequence of the antisense oligonucleotide DNA (2). designed based on the nucleotide sequence on the 5'-end side of the each structural gene was shown in Table 3.<br>
Table 2: Oligonucleot.de DNA (1) used for determination of 5'-end sequence according to 5'RACE<br><br><br>
51<br><br>
The first strand of cDNA was synthesized according to a reverse transcription reaction using the oiigonucleotide DNA (1) as a primer, and total RNA of PeniciUimn citrmum SANK J 3380 as a template.  Namely. 24 ?l of the reaction mixture comprising one ?g of total RNA, 2.5 pmoi of oligonucleotide DNA (1) and one ?l of SUPER SCR1PTTM U reverse transcriptase (contained in the kit) was incubated at 16°C for one hour, and the reaction product was added to GLASSMAX spin cartridge contained in the kit. to purify the first strand of cDNA.<br>
A poly C chain was added to the 3'-end of the cDNA first strand using terminal deoxyribonucleotidyl transferase contained in the kit.<br>
50 ul of the reaction mixture comprising the first strand of cDNA to which the 3r-end poly C chain had been added, was mixed with 40 pmol of oligonucleotide DNA (2) and 40 pmol of Abriged Anchor Primer (contained in the kit), followed by incubation at 94 °C for two minutes.   The incubation cycle of 30 seconds at 94 °C. 30 seconds at 55 °C and two minutes at 72 °C was then repeated 35 times, followed by incubation at 72°C for five minutes and at 4°C for 18 hours.   The resulting product was subjected to agarose gel electrophoresis. and DNA was recovered from the gel.   The product was purified by phenol - chloroform extraction and ethanol precipitation, and cloned in the similar manner to a method described in Example 4 using pCR 2.1.<br>
The operation described above is 5'-RACE.<br>
The nucleotide sequence of cDNA fragment containing 5'-end was determined, and position of transcription initiation point and translation initiation codon were predicted.<br>
Table 4 shows the SEQ ID No. in which the nucleotide sequence of the 5'-end cDNA fragment corresponding to each structural gene obtained by 5' RACE was described.   Table 5<br><br>
52 shows the SEQ ID No. in which the transcription initiation point and translation initiation<br>
point of each structural gene exist, and the position of the transcription initiation point and translation initiation point.<br>
Table 4: SEQ 10 Nos in which nucleotide sequence of 5'-end cDNA fragment is shown<br><br>
Table 5: Position of transcription initiation point and translation initiation point of each gene<br><br><br>
53<br>
* nucleotide sequence shown in SEQ ID No. 1 and 2 of Sequence Listing are completely complementary with each other.<br>
4) Determination of 3'-end sequence according to 3' RACE<br>
cDNA containing the 3'-end region of each structural gene was obtained using the Ready To Go: T-Primed First-Strand kit (manufactured by Pharmacia corporation).<br>
One kind of sense oligonucleotide DNA (3) presumed to be in coding region and near the 3'-end in each structural gene in the inserted sequence of pML48 was produced, predicte from the results of Example 7 and the item 2) of the present example.<br>
The nucleotide sequence of the oligonucleotide DNA (3) produced for each structural gene is shown in Table 6.<br>
Table 6: Oligonucleotide DNA (3) used for determination of 3'-end sequence according to 3' RACE<br><br>
The first strand of cDNA was synthesized by a reverse transcription reaction using the NotI-d(T)18 primer (contained in the kit), and total RNA of Penicillium citrinum SANK13380 (one ?g) as a template.<br><br>
54 100 ?l of the reaction mixture comprising the first strand of cDNA, 40 pmol of<br>
oltgormcfeotide DNA (3) and Notl-d(T) 18 primer (contained in the kit) was kept at 94X for two minutes.   An incubation cycle of 30 seconds at 94°C, 30 seconds at 55°C and two minutes at 72°C was repeated 35 times, followed by incubation at 72°C for five minutes and at 4°C for 18 hours.   The resulting product was subjected to agarose gel electrophoresis, and then DNA was recovered from the gel.   The product was purified by phenol - chloroform extraction and ethanol precipitation, and cloned in the similar manner to a method described in Example 4 using pCR 2.1.<br>
The operation described above is 3'-RACE.<br>
The nucleotide sequence of cDNA at the 3'-end was determined, and the position of the translation termination codon was predicted.<br>
Table 7 shows the SEQ ID No. of the Sequence Listing in which the nucleotide sequence of the 3'-end cDNA fragment corresponding to each structural gene obtained by 3' RACE is described.   Table 8 shows the translation termination codon and position of the codon based on SEQ ID Nos, I and 2 of Sequence Listing.<br>
Table 8: Translation termination codon and position of the translation termination codon of<br>
Table 7: SEQ IDNos in which nucleotide sequence of 3'-end cDNA fragment<br><br><br>
55 each structural gene<br><br>
* nuclcotidc sequence shown in SEQ ID No.l and 2 of Sequence Listing are completely complementary with each other.<br>
Table 9 shows the C-terminal amino acid residue of the polypcptide predicted to be encoded by each structural gene, the nucleotide sequence of the trinucleotide encoding the amino acid residue and the position of the trinucleotide.<br>
Table 9: C-terminal amino acid residue of the polypeptide encoded by each structural gene<br><br><br><br><br><br><br>
56<br>
* the nucleotide sequence shown in SEQ ID No.1 and 2 of Sequence Listing are completely complementary with each other.<br>
Table ] 0 summarizes the sequence complementary to the translation termination codon shown in Table 8. the SEQ ID where the complementary sequence exists and the position of the complementary sequence<br>
Table 10 Sequence complementary to translation termination codon of each structural gene<br><br>
* the nucleotide sequence shown in SEQ ID No.l and 2 of Sequence Listing are completely complementary with each other.<br>
As described above, the position of each structural gene, the direction thereof and position thereof were ascertained.   Based on the above information, the transcription product and translation product of each structural gene can be obtained.<br>
Example 9: Obtaining cDNA corresponding to the structural gene mlcE<br>
1) Preparation of total RNA<br><br>
57<br>
Total RNA of Penicillium cithnum was prepared according to the method of Example 8.<br>
2)	Design of primer<br>
In order to obtain a full length cDNA corresponding to structural gene mlcE determined in Example 8. the following primers were designed and synthesized:<br>
sense primer 5'-gttaacatgtcagaacctctaccccc-3' (See SEQ ID 35 of Sequence Listing): and antisense primer 5'-aatatttcaagcatcagtctcaggcac-3': (See SEQ ID 36 of Sequence Listing).<br>
The primers are derived from the sequence on the 5'-end upstream region of structural gene m/cE and from the sequence at the 3'-end downstream region respectively.   Synthesis was performed according to the phosphoamidite method.<br>
3)	RT-PCR<br>
In order to obtain a full-length cDNA encoding the gene product of mlcE. the Takara RNA LA PCR kit (AMV) Ver. 1.1 was used.<br>
Specifically. 20 pi of a reaction mixture comprising one u.g of total RNA. 2.5 pmol of Random 9 mers primer (contained in the kit), and one ?l of reverse transcription enzyme (contained in the kit) was incubated at 42°C for 30 minutes to produce the first strand of cDNA.   The reverse transcription enzyme was then deactivated by heating at 99°C for five minutes.<br>
100 ?l of a second reaction mixture comprising the total amount of the reaction mixture of the first strand of cDNA (above), 40 pmol of sense primer and 40 pmol of antisense primer was incubated at 94 °C for two minutes.   An incubation cycle of 30 seconds at 94°C. 30 seconds at 60°C and two minutes at 72°C was repeated 30 times, followed by<br><br>
58 incubation at 72°C for five minutes and at 4°C for 18 hours.   The resulting product was<br>
subjected to agarose gel electrophoresis. and DNA was recovered from the gel.   The product was purified by phenol - chloroform extraction and ethanol precipitation, and used to transform Eschehchia coli competent cell JM109 strain (manufactured by Takara Shuzo Co.. Ltd., Japan) in the similar manner to a method described in Example 4 using pCR 2.1.   A strain carrying a plasmid having the DNA fragment was selected from the transformed Escherichia coli. and the plasmid carried by the strain was designated as pCRexpE.<br>
The nucleotide sequence of the inserted DNA of the resulting recombinant DNA vector pCRexpE was determined.   The inserted DNA contained full-length cDNA corresponding to structural gene mlcE.   The nucleotide sequence thereof and an amino acid sequence of the peptide deduced from the nucleotide sequence are shown in SEQ ID NO.3 7 and/or SEQ ID NO 38 of the Sequence Listing.<br>
The nearest known sequence for mlc E (polypeptide) was ORF10 on the gene cluster related to biosynthesis of lovastatin. with 70% identity.<br>
Example 10: Construction of the expression vector pSAK 700<br>
cDNA expression vector pSAK.700 was constructed using the vector pSAK333 and pSAK360 described in Example 1.<br>
pSAK333 was digested with both restriction enzymes BamH I and Hind III (manufactured by Takara Shuzo Co.. Ltd.. Japan), and then subjected to agarose gel electrophoresis.   A 4.1kb fragment was recovered from the gel. and the end of the DNA fragment was blunt-ended with T4-DNA polymerase (manufactured by Takara Shuzo Co., Ltd., Japan).<br>
An EcoRI-Notl-BamHI adapter (manufactured by Takara Shuzo Co., Ltd., Japan) was linked to the above-mentioned DNA fragment using DNA Hgation kit Ver.2 (manufactured by Takara Shuzo Co., Ltd., Japan). Escherichia coli competent cell JM109 strain (manufactured<br><br>
59 by Takara Shuzo Co., Ltd.. Japan) was transformed with the ligated DNA.   A strain carrying<br>
the plasmid having the adapter was selected from the transformed Eschehchia colt, and the plasmid carried by the strain was designated as pSAK410.<br>
pSAK360 was digested with both restriction enzymes Pvu II and Ssp I. and subjected to electrophoresis.   A DNA fragment (about 2.9 kb) containing the promoter and terminator of 3-phosphoglycerate kinase (hereinafter referred to as "pgk") gene and HPT originating from Escherichia coli was recovered from the gel.<br>
The recovered above-mentioned DNA fragment was linked to the Pvu II site of pSAK410 using DNA ligation kit Ver.2 (manufactured by Takara Shuzo Co.. Ltd.. Japan). Escherichia coli competent cell JM109 strain was transformed with the ligated DNA.   A strain carrying the plasmid having the DNA fragment was selected from the transformed Escherichia coli. and the plasmid carried by the strain was designated as pSAK.700.<br>
The construction of pSAK700 is shown in Figure 4.<br>
pSAK 700 has one restriction enzyme site for each of the enzymes BamHI and Notl. pSAK700 also has an ampicillin resistant gene (hereinafter referred to as "Ampr") and hygromycin resistance gene HTP as a selection marker.   In the following examples, when Escherichia coli is used as a host, selection of cells transformed by pSAK700 or by pSAK700 comprising a foreign DNA insert was performed by adding 40 ug/ml of ampicillin to the relevant medium.   When Penicillium citrinum SANK 13380 is used as a host, selection of cells transformed by pSAK700 or by pSAK700 comprising a foreign DNA insert was performed by adding 200 ug/ml of hygromycin to the relevant medium.<br>
Example 11: Construction of cDNA expression vector pSAKexpE<br>
Recombinant DNA vector pCRexpE obtained in Example 9 was reacted at 37°C for 2 hours in the presence of the restriction enzymes Hpal and Sspl (manufactured by Takara Shuzo Co., Ltd., Japan), and the reaction product was subjected to agarose gel electrophoresis.<br><br>
60 A band containing a full-length cDNA of mlcE around K7kb was recovered from the gel.<br>
After reacting pSAK700 with the restriction enzyme Notl (manufactured by Takara Shuzo Co.. Ltd.. Japan) at 37°C for one hour, the end of the vector was blunt-ended with T4 DNA polymerase (Takara Shuzo Co.. Ltd.. Japan) at 37°C for 5 minutes.   Then, the vector was subjected to phenol chloroform extraction and ethanol precipitation.   The precipitate DNA was dissolved in a small amount of TE.   Alkaline phosphatase was added thereto and was incubated at 65°C for 30 minutes,   pSAK.700, prepared as described above, was iigated to 1.7 kb of DNA fragment obtained in the step 1) using DNA ligation kit Ver.2 (manufactured by Takara Shuzo Co.. Ltd.. Japan). Escherichia coli competent cell JM109 strain was transformed using the iigated DNA.   An Escherichia coli strain transformed by cDNA expression vector was obtained.<br>
The transformed Escherichia coli. termed Escherichia coli pSAKexpE SANK 72499. obtained in the present example was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on January 25. 2000 under the Deposit Nos. FERM BP-7005. in accordance with the Budapest Treaty on the Deposition of Micro-organisms.<br>
Example 12: Obtaining cDNA corresponding to the structural gene mlcR<br>
1)	Preparation of total RNA<br>
Total RNA of Penicillium citrinum was prepared according to the method of Example 8.<br>
2)	Design of primer<br>
In order to obtain a full length cDNA corresponding to the structural gene rnlcR determined in Example 8, the following primers were designed and synthesized:<br><br>
61 sense primer : 5'-ggatccatgtccctgccgcatgcaacgattc-3': (See SEQ ID 39 of Sequence Listing):<br>
and<br>
antisense primer 5'-ggatccctaagcaatattgtgtttcttcgc-3': (See SEQ ID 40 of Sequence Listing).<br>
The primers were designed from the sequence on 5'-end upstream region of structural gene mlcR and from the sequence at the 3r-end downstream region, respectively.   Synthesis was performed according to the phosphoamidite method.<br>
3) RT-PCR<br>
In order to obtain a full-length cDNA encoding the gene product of mlcR. a Takara RNA LA PCR kit (AMV) Ver. 1.1 was used.<br>
Specifically. 20 ul of a reaction mixture comprising one ?g of total RNA. 2.5 pmol of Random 9 mers primer (contained in the kit), and one ?l of reverse transcription enzyme (contained in the kit) was incubated at 42°C for 30 minutes to produce the first strand of cDNA.   The reverse transcription enzyme was then deactivated by heating at 99°C for five minutes.<br>
100 ?l of a second reaction mixture comprising the total amount of the reaction mixture of the first strand of cDNA (above). 40 pmol of sense primer and 40 pmol of antisense primer was incubated at 94 °C for two minutes.   An incubation cycle of 30 seconds at 94°C, 30 seconds at 60°C and two minutes at 72°C was repeated 30 times, followed by incubation at 72°C for five minutes and at 4°C for 18 hours.   The resulting product was subjected to agarose gel electrophoresis. and DNA was recovered from the gel.   The product was purified by phenol - chloroform extraction and ethanol precipitation, and used to transform Escherichia coli competent cell JM109 strain (manufactured by Takara Shuzo Co., Ltd., Japan) in the similar manner to a method described in Example 4 using pCR 2.1.   A strain carrying a plasmid having the DNA fragment was selected from the transformed Escherichia coli, and the plasmid carried by the strain was designated as pCRexpR.<br><br>
62 The nucleotide sequence of the inserted DNA of the resulting recombinant DNA<br>
vector pCRexpR was determined.   The inserted DNA contained full-length cDNA corresponding to structural gene mlcR.   The nucleotide sequence thereof and an amino acid sequence of the peptide deduced from the nucleotide sequence are shown in SEQ ID NO 41 and/or SEQ ID NO 42 of the Sequence Listing.<br>
The nearest known sequence for mlc R (polypeptide) was lovE on the gene cluster related to biosynthesis of lovastatin. with 34% identity.<br>
Example 13: Construction of cDNA expression vector pSAKexpR<br>
Recombinant DNA vector pCRexpR obtained in Example 12 was reacted at 37°C for 2 hours in the presence of restriction enzyme BamHI (manufactured by Takara Shuzo Co.. Ltd.. Japan), and the reaction product was subjected to agarose gel electrophoresis.   A band containing a full-length cDNA of mlcR around 1.4kb was recovered from the gel.<br>
After reacting pSAK700 with the restriction enzyme BamHI (manufactured by Takara Shuzo Co.. Ltd., Japan) at 37°C for one hour, alkaline phosphatase (manufactured by Takara Shuzo Co.. Ltd.. Japan) was added and reacted at 65°C for 30 minutes.   pSAK700 digested with BamHI as described above was ligated to 1.4 kb of DNA fragment obtained in the step 1) using DNA ligation kit Ver.2 (manufactured by Takara Shuzo Co., Ltd.. Japan). Escherichia coli competent cell JM109 strain was transformed with the ligated DNA.   An Escherichia coli strain transformed by cDNA expression vector was obtained.<br>
The transformed Escherichia coli. termed Escherichia coli pSAKexpR SANK 72599, obtained in the present example was deposited at the Research Institute of Life Science and Technology of the Agency of Industrial Science and Technology on January 25, 2000 under the Deposit Nos. FERM BP-7006, in accordance with the Budapest Treaty on the Deposition of Micro-organisms.<br>
Example 14: Transformation of IV1L-236B producing micro-organisms<br><br>
63 1) Preparation of protoplasts<br>
Spores from a slant of a culture ofPeniciUium citrinum SANK 13380 strain were inoculated on a PGA agar medium, then incubated at 26 °C for 14 days.   The spores of PeniciUium citrinum SANK 13380 strain were then recovered from the culture, and 1 x 108 of the spore were inoculated into 80 ml of YPL-20 culture medium, incubated at 26°C for one day.   After confirming germination of the spores by observation under a microscope, the germinating spores were centrifuged at room temperature at 5000 x G for ten minutes, and recovered as a precipitate.<br>
The spores were washed with sterilized water three times, to form protoplasts. Namely, 200 mg of zymolyase 20 T (manufactured by Seikagaku Kogyo corporation) and 100 mg of chitinase (manufactured by Sigma corporation) were dissolved in 10 ml of 0.55 M magnesium chloride solution, and centrifuged at room temperature at 5000 x G for 10 minutes.   The resultant supernatant was used as an enzyme solution.   20 ml of the enzyme solution and 0.5 g (wet weight) of germinating spores were put into 100 ml conical flask and incubated with gently shaking at 30°C for 60 minutes.   After confirming that the germinating spores became protoplasts using a microscope, the reaction solution was filtered through 3G-2 glass filter (manufactured by HARIO corporation).   The filtrate was centrifuged at room temperature at 1000 x G for 10 minutes, and then the protoplasts were recovered as a precipitate.<br>
2) Transformation<br>
The protoplasts obtained in step 1) were washed twice with 30 ml of 0.55 M magnesium chloride and once with 30 ml of a solution consisting of 0.55 M magnesium chloride, 50 mM calcium chloride and 10 mM 3-morpholino propane sulfonate (pH 6.3 or lower, hereinafter referred to as MCM solution).   Protoplasts were then suspended in 100 ?l of a solution of 4 % (w/v) polyethylene glycol 8000, 10 mM 3-morpholino propane sulfonate, 0.0025% (w/v) heparin (manufactured by sigma corporation), 50 mM magnesium chloride<br><br>
64 (pH 6.3 or less, hereinafter referred to as "transformation solution").<br>
96 ?l of transformation solution containing about 5 x 107 protoplasts and 10 ?l of TE containing 120 ug of pSAKexpE. or pSAKexpR. were mixed, and allowed to stand on ice for 30 minutes.   Thereto was added 1.2 ml of a solution of 20 % (w/v) polyethylene glycol. 50 mM of magnesium chloride, 10 mM of 3-morpholino propane sulfonic acid (pH 6.3).   The liquid was then gently pipetted and then allowed to stand at room temperature for 20 minutes. Thereto was added 10 ml of MCM solution, followed by gentle mixing, and centrifugation at room temperature at 1000 x G for 10 minutes.   The transformed protoplasts were recovered from the precipitate.<br>
3) Regeneration of the cell wall of transformed protoplasts<br>
The transformed protoplasts obtained in step 2) were suspended in 5 ml of liquid VGS middle layer agar medium, and layered on 10 ml of a solidified VGS lower agar medium plate.   The plate was incubated at 26°C for one day. after which 10 ml of liquid VGS upper agar medium containing 5 mg hygromycin B per plate (final concentration of hygromycin of 200 fig/ml) was layered on top.   After incubation at 26°C for 14 days, both strains (i.e. those strains derived from protoplasts transformed with pSAKexpE, or pSAKexpR ) were subcultured on PGA agar medium containing 200 ug/ml of hygromycin B, and subcultured on a slant prepared with PGA agar medium, incubated at 26°C for 14 days.<br>
The slants were reserved at 4°C.<br>
Test Example 1: Comparison of ML-236B biosynthesis ability in transformed and original strains<br>
The transformed strains obtained in Example 14 and Penicillium citrinum SANK 13380 were cultured and the amount of ML-236B in each culture was measured.<br>
A 5 mm square inoculum of spores was cultured from the slants in which the<br><br>
65 transformed strains were cultured, as described in Example 14. and from the slant described in<br>
Example 2, relating to Peniciliium citrinum SANK 13380,   Cells were inoculated in 10 ml of MBG3-8 medium in a 100 ml conical flask, then incubated at 24°C for two days with shaking, followed by the addition of 3.5 ml of 50 % (w/v) glycerin solution.   Then, culturing was continued at 24°C for 10 days with shaking.<br>
To 10 ml of the culture was added 50 ml of 0.2 N sodium hydroxide, followed by incubation at 26°C for one hour with shaking.  The culture was centrifiiged at room temperature at 3000 x G for two minutes.  One ml of the supernatant was recovered, mixed with 9 ml of 75 % methanol and subjected to HPLC<br>
SSC-ODS-262 (having a diameter of 6 mm. length of 100 mm. manufactured by Senshu Kagaku Co.Ltd.) was used as HPLC column, and 75 % (v/v) methanoi - 0.1 % (v/v) triethylamine - 0.1 % (v/v) acetic acid was used as the mobile phase.   Elution was carried out at room temperature at a flow rate of 2 ml/minute. Under these conditions. ML-236B was eluted 4 minutes after addition to the column.   Detection was performed with a UV detector at absorption wavelength of 236 nm.<br>
ML-236B biosynthesis ability was increased in three strains among the eight pSAKexpE transformed strains.   ML-236B biosynthesis ability of these strains was 10 % higher on average compared with the original strain.   ML-236B biosynthesis ability of these three strains was also maintained stably after subculture, such as monospore treatment or the like.   These results indicate that the insert of pSAKexpE is an ML-236B biosynthesis accelerating cDNA.<br>
ML-236B biosynthesis ability was increased in five strains among the pSAKexpR transformed strains.   ML-236B biosynthesis ability of these strains was 15 % higher on average compared with the original strain.   ML-236B biosynthesis ability of these five strains was also maintained stably after subculture such as monospore treatment or the like.   These results indicate that the insert of pSAKexpE is an ML-236B biosynthesis accelerating cDNA.<br><br>
66 Thus, ML-236B biosynthesis accelerating cDNA obtained from an ML-236B<br>
producing micro-organism according to the present invention accelerates ML-236B biosynthesis in the ML-236B producing micro-organism when introduced in the ML-236B producing micro-organism.<br>
Example 15: Determination of the sequence of cDNAs corresponding to the structural genes mlc A-D.<br>
The sequence of the cDNA corresponding to the structural gene mlc A was determined.<br>
The first strand cDNA was synthesized with TAKARA LA PCR kit verl. 1 (Takara Shuzo Co.. Ltd.)- Several PCRs were carried out for amplification of the full or partial region of the cDNA using the first strand cDNA as a template and several distinct pairs of oligonucleotides as primers.<br>
The cycle of 30 seconds at 94°C. 30 seconds at 60°C and five minutes at 72°C was repeated 30 times using The Big Dye Primer/Terminator Cycle Sequencing Kit and The ABI Prism 377 sequence (PE Applied Biosystems).<br>
The product of each reaction was inserted into plasmid pCR2.1 individually. Escherichia coli transformants of each recombinant plasmid were obtained.<br>
The nucleotide sequences of each insert of the recombinant plasmids obtained from said transformants were determined.<br>
The sequences of exons and introns were determined on the basis of a comparison between the nucleotide sequence of several RT-PCR products mentioned above and that of the structural gene mlc A.<br><br>
6? Then, the sequence of cDNA corresponding to the structural gene mlc A was<br>
determined (SEQ ID NO 43). The corresponding amino acid sequence of polypeptide encoded by said cDNA was predicted (SEQ ID NO 44) and a function of the polypeptide was assumed on the basis of a hornology search using the amino acid sequence.<br>
The nearest known sequence for mlc A (polypeptide) was LNKS(ZovB) on the gene cluster related to biosynthesis of lovastatin, with 60% identity.<br>
In a simlar way, the sequence of cDNA corresponding to the structural gene mlc B was determined (SEQ ID NO 45).   The corresponding amino acid sequence of polypeptide encoded by said cDNA was predicted (SEQ ID NO 46) and a function of the polypeptide was assumed on the basis of a homology search using the amino acid sequence.<br>
The nearest known sequence for mlc B (polypeptide) was LDKS(fovF) on the gene cluster related to biosynthesis of lovastatin. with 61% identity.<br>
Similarly, the sequence of cDNA corresponding to the structural gene mlc C was determined (SEQ ID NO 47). The corresponding amino acid sequence of polypeptide encoded by said cDNA was predicted (SEQ ID NO 48) and a function of the polypeptide was assumed on the basis of a homology search using the amino acid sequence.<br>
The nearest known sequence for mlc C (polypeptide) was lovA on the gene cluster related to biosynthesis of lovastatin, with 72% identity.<br>
Furthermore, the sequence of cDNA corresponding to the structural gene mlc D was determined (SEQ ID NO 49).   The corresponding amino acid sequence of polypeptide encoded by said cDNA was predicted (SEQID NO 50) and a function of the polypeptide was assumed on the basis of a homology search using the amino acid sequence.<br>
The nearest known sequence for mlc D (polypeptide) was ORF8 on the gene cluster related to biosynthesis of lovastatin, with 63% identity.<br><br>
68<br>
The positions of exons of each structural gene on SEQ ID NO 1 or SEQ ID NO 2 were<br>
determined, as follows:	<br>
Table 11: The positions of exons of mlcA-D in pML48 inserts<br><br>
Thr positions of transriptionl termination site of each structural gene on SEQ ID NO 1 or SEQ ID NO 2 were determined as follows:<br>
Table 12: The positions of transcription termination site of structural genes mlc A-E and R in pML48 inserts<br><br><br>
69<br><br>
Example 16: Studies of gene disruption.<br>
The structural genes mlc A. B or D of p, citrinum were disrupted via site directed mutagenics using homologous recombination.<br>
The recombinant plasmid for obtaining the structural gene m/cA-disrupted mutant of P. citrimtm was constructed using a plasrnid. pSAK333.<br>
A 4.1-kb internal Kpn\ fragment of the mlcA locus on the pML48 insert was recovered, purified, blunt ended with a DMA Blunting Kit (Takara Shuzo Co.. Ltd,) was ligated to fuull-digested pSAK333. The resultant plasmid was designated as pdism/cA<br>
P. citrinum SANK13380 was transformed by p&amp;ismlcA,<br>
Southern hybridization of genomic DNA of pdismlcA transformant was carried out to confirm the disruption of the structural gene mlcA,<br>
The resultant mlcA-disrupted mutant did not produce ML-236B or its precursor at all.<br>
The recombinant plasrnid for obtaining the structural gene mlcB-disrupted mutant of P. citrinum was constructed using a plasmid, pSAK333.<br>
1.4-Kb PsI-BamHI fragment of the mlcB locus on the pML48 insert was recovered, purified, blunt ended with a DNA Blunting Kit (Takara Shuzo Co., Ltd.) and ligated to PuuII-digested pSAK333.   The resultant plasmid was designated as pdismlcB.<br><br>
70 P. citrinum SANK13380 was transformed by pdismlcB.<br>
Southern hybridization of genomic DNA of pdismlcB transformant was carried out to confirm the disruption of the structural gene mlcB.<br>
The resultant mlcB-disrupted mutant produced not ML-236B but ML-236A. the precursor of ML-236B.<br>
The recombinant plasmid for obtaining the structural gene m/cD-disrupted mutant of P. citrinum was constructed using a plasmid. pSAK333.<br>
1.4-Kb Kpnl-BamHI fragment of the mlcD locus on the pML48 insert was recovered, purified, blunt ended with a DNA Blunting Kit (Takara Shuzo Co.. Ltd.) and ligated to Puull-digested pSAK333. The resultant plasmid was designated as pdismlcD.<br>
P. citrinum SANK13380 was transformed by pdismlcD.<br>
Southern hybridization of genomic DNA of pdismlcD transformant was carried out to confirm the disruption of the structural gene mlcD.<br>
The amount of ML-236B produced by resultant mlcD-disrupted mutant was about 30% of that of the untransformed control host.<br>
Example 17: Functional Analysis of mlc R in pSAKexpR transform ants.<br>
Two of the pSAKexpR transformants which were obtained in Example 12, designated as TR1 and TR2 respectively, and untransformed host cells. Penicillium citrinum SANK13380, were inoculated in MBG3-8 medium and incubated individually as described in Example 8.<br>
Total RNA was extracted from each of the cultures described in Example 8.<br><br>
71<br>
RT-PCR was carried out using said total RNA as a template and a pair of oligonucleotides designed on the basis of nucleotide sequence of the structural genes mlc A. B. C D E or R as primers.<br>
Table 13: Nucleotide sequences of pairs of primers for RT-PCR.<br><br>
The results of RT-PCR analysis are shown in Figure 5 for the untransformed Pencillium citrinum 13380. and for the two transformants designated TR1. TR2.<br>
The structural genes mlc A. B. C. D and R were expressed at the first, second and third day of cultivation in pSAKexpR transformants.<br>
In contrast, all these structural genes were expressed only at the third day of cultivation in untransformed host cells.<br>
There was no difference in the expression of the structural gene mlcE between pSAKexpR transformants and untranformed host cells.<br>
The results suggests that a protein encoded by cDNA corresponding to a structural gene mlc R induces transcriptions of some of the other structural genes (for example, mlc A, B, C, D) located in the ML-236B biosynthesis related gene cluster.<br><br>
72 Example 18: Functional Analysis of mlc E in pSAKexpE transformants.<br>
A pSAKexpE transformant designated as TE1 which was obtained in Example 12. and its untransformed host cells. Penicillum citrinum SANK 13380. were inoculated in MBG3-8 medium and incubated individually as described in Example 8.<br>
Total RNA was extracted from each of the cultures described in Example 8.<br>
RT-PCR was carried out using said total RNA as a template and a pair of oligonucleotides designed on the basis of nucleotide sequence of the structural genes mlc A.<br>
B,	C, D, E or R as primers.   Primers used for the present example were identical with those in<br>
the table of the previous Example.<br>
The results of RT-PCR analysis are shown in Figure 6 for the untransformed Pencillium citrinum 13380. and for a transformant designated TE1.<br>
The structural gene mlc E was expressed at the first, second and third day of cultivation in pSAKexpE transformants.<br>
In contrast, the structural gene mlc E was expressed only at the third day of cultivation in untransformed host cells.<br>
On the other hand, there was no difference in the expression of the structural genes mlc A. B, C , D and R between pSAKexpE transformant and untranformed host cells (data not shown).<br>
The results suggests that a protein encoded by cDNA corresponding to a structural gene mlc E accelerates ML-236B biosynthesis independently of the structural genes mlc A, B,<br>
C,	D and R.<br><br>
73 WE CLAIM<br>
1. A polynucleotide selected from the group consisting of:<br>
(a) a polynucleotide encoding a protein having the amino acid sequence of SEQ 10 NO 38. or a polynucleotide variant thereof encoding a modified amino acid sequence having at least one deletion addition nubstlution or attention said variant polynucleotide being sultable for use in accelerating the biosynthesis of ML-23CB; and<br>
(b)a polynucleotide encoding a protein having the amino acid sequence of SEQ ID NO 42 or a polynucteotide variant thereof encoding a modified amino acid having at least one deletion substitution or alteration said variant potynucteotide being suflabte for use In accelerating tlhe biosynthesis of ML-236B.<br>
2.	A polynucleotide as claimed in claim 1 comprising SEQ ID NO 37 or<br>
comprising a mutant or variant thereof suitable for use m accelerating the<br>
biosynthesis of ML-236B.<br>
3.	A polynucleotide as claimed in claim 1 to 2 comprising DMA obtainable<br>
from transformed Escherichia coll pSAKexpE SANK 72499 (FERM BP-<br>
7005).<br>
4.	A polynucleotide as claimed tn claim 1 comprising SEQ ID NO 41, or<br>
comprising  a  variant  thereof,   suitable  for  use  in accelerating the<br>
biosynthesis of ML-236B.<br>
5.	A polynucleotide as claimed In claim 1 or 6 comprising DMA obtainable<br>
from transformed Escherichia coll pSAKexpR SANK 72599 (FERM BP-<br>
7006).<br>
6.	A   polynucleotide   as   claimed   in  any  pecedlng   claim   in  operative<br>
combination with one or more polynucleottdes, said combination being<br>
suitable for use in enhancing the production of ML236B m an ML-236B<br>
producing micro-organism.<br><br>
74<br>
7.	A polynucteotide as claimed in claim 6 comprising (ha polynucbotide of<br>
SEQ ID NO 37 or variant thereof having similar function in combination<br>
with one or more sequences sobcted from SEQ ID NO 37,41,43,49,47 or<br>
49 or variant thereof having similar function.<br>
8.	A polynucleotide as claimed In cbbn 6 comprising the polynucleotide of<br>
SEQ ID NO 41 or variant thereof having simillar function in combination<br>
with one or more sequences selected from SEQ ID NO 37,41,43,45,47 or<br>
49 or variant thereof having simillar function.<br>
9.	A polynucleotide capable of hybridizing under stringent conditions with a<br>
polynucteotide according to any preceding claim.<br>
10.A polynucleotide as claimed In cbbn 9 that is suitable for accelerating the<br>
biosynthesis of ML-236B in an ML-236B producing micro-organism when<br>
introduced in the ML-2368 producing micro-organism. 11.A polynucteotide as claim In claim 9 or 10 which is RNA. 12.A vector comprising a polynucleotide os ctobn In any preceding claim. 13.A  vector as  claimed  in claim   12  obtainable from Escherichia col<br>
pSAKexpE SANK 72499 (PERM BP-7005) or Escbertchfia col PSAKexpR<br>
SANK 725999 (VferM BP-7006).<br>
14.A vector as claimed In cbbn 12 or 13 which Is an expression vector. 15 A host cell transformed by a vector os cbbn In any of Cbims 12 to 14. 16.A host cell as claimed m cbtm 15 wherein that 1 is an ML-236B producing<br>
micro-organism.<br>
17.A host eel os claimed in claim 16 wherein that it is Penicillium clrinum. 18.A host cell as claimed In claim 15 wherein that M Is EscherlcMa col 19.A host col! as claim in ctatan  18 wherein that H Is Eschorichb col<br>
pSAKexpE SANK 72499 (FERM BP-7005). r2o. A host as claimed m claim 18 wherein that N Is Escherlchto coH pSAKexpR.<br>
SANK 72499 (FERM BP-7006). 21.A polypeptlde encoded by a polynucleotide as claimed in any of claims 1<br>
to 11.<br><br>
76<br>
22.A polypeptide comprising the sequence of SEQ ID NO .38, or a valiant thereof which has at toast 80% Identity to SEQ ID NO 38 and wMch fs capable of accelerating ML236B production *n an ML238B producing organism.<br>
23.A polypeptide as claimed fri claim 22, having the sequence of SEQ ID NO<br>
38. 24.A polypeptide comprising the sequence of SEQ ID NO 42, or a variant<br>
thereof which has at least 80% Identity with SEQ ID NO 42 and which ft<br>
capable of accelerating ML236B production ton an ML236B producing<br>
organism. 25.A potypeptide as claimed in claim 24, having the sequence of SEQ ID NO<br>
42. 26.A method for producing ML236B, comprising cutturlng a host cell as<br>
claimed In any of claims 15 to 17 and then recovering ML236B from the<br>
culture. 27.A method as claimed In claim 26, wherein the host ceR is transformed with<br>
a vector comprising SEQ ID NO 41. 28.A mothod ti claimed In cHalm 27, wherein the vector comprising no<br>
additional genes. 29.A method as claimed In any of claims 26 to 28, wherein production occurs<br>
in the absence of recombtoart mlc ABC or D corresponding to SEQ ID<br>
NO 44,46,48 or 50.<br>
30.ML-236B produced by the method of any claims 26 to 29. 31 .A method of manufacturing pravastatln. which comprises carrying out a<br>
method as claimed in claims 26 to 29 and converting the ML-236B to<br>
pravastattn. 32. A antibody reactive with the protein of SEQ ID NO 38 or SEQ ID NO 42.<br><br>
76<br>
33.A polynucleotide encode a protein having the amtno acid sequence selected from SEQ ID NO 44, 46,48 or 50 or a variant polynucleotide encoding a modification of eaid ammo acid sequence having a deletion, substitution, addition or alteration, said variant being suitable for use In accelerating the biosynthesis of ML-236B.<br>
34.A polynucleotide as claimed in claim 33 selected from the group consisting of SEQ ID NO. 43,45,47 or 49.<br>
35.A polynucleotide as claimed in claim 33 or 34, said polynucleotide being capable of accelerating the biosynthesis of ML-236B alone or in conjunction with the polynucleotide of SEQ ID NO 37 or SEQ ID NO 41.<br>
36.A vector comprising a polynudeotide as claimed in any of claims 33 to 35.<br>
37.A host cell comprising a vector as claimed in claim 36.<br>
38.A potypeptlde encoded by a polynucleotkle as claimed In any of claims 33 to 35.<br>
39.A method for the production of ML236B comprising cutturing a host cell as claimed in claim 37 and then recovering ML-236B from the culture.<br>
A polynucleotide selected from the group consisting of: (a) potymicletide encoding a protein having the amino acid sequence of SEQ ID NO 38, or a polynucleotide variant thereof encoding a modified amino acid sequence having at least one deletion addition substitution or alteration said variant polynucleotide being suitable for use in accelerating the blosynthesis of ML-236B; and (b) a polynucleotide encoding a protein having the amino acid sequence of SEQ ID NO 42 or a polynucleotide variant thereof encoding a modified amino acid having at least one deletion substitution or alteration said variant polynucleotide being suitable for use in accelerating the biosynthesis of ML-236B.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="209227-iontophoretic-delivery-of-an-antimigraine-drug.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209229-a-receptacle-holding-a-plurality-of-sanitary-absorbent-article.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209228</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00218/CAL/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>34/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Aug-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>23-Aug-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>16-Apr-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANKYO COMPANY, LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>5-1, NIHONBASHI 3-CHOME, CHUO-KU, TOKYO,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ABE, YUKI</td>
											<td>C/O.SANKYO COMPANY LIMITED, 389-4,AZA OHTSURUGI, IZUMIMACHISHIMOKAWA, IWAKI-SHI, FUKUSHIMA,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YOSHIKAWA, HIROJI</td>
											<td>C/O.SANKYO COMPANY LIMITED, 389-4,AZA OHTSURUGI, IZUMIMACHISHIMOKAWA, IWAKI-SHI, FUKUSHIMA,JAPAN</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ONO,CHIHO</td>
											<td>C/O.SANKYO COMPANY LIMITED, 389-4,AZA OHTSURUGI, IZUMIMACHISHIMOKAWA, IWAKI-SHI, FUKUSHIMA,JAPAN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-116591</td>
									<td>2000-04-18</td>
								    <td>Japan</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2000-117458</td>
									<td>2000-04-19</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209228-a-polynucleotide-that-is-suitable-for-accelerating-a-biosynthesis-of-ml236b-and-the-method-for-producing-ml236b-by-using-the-polynucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:13:39 GMT -->
</html>
