<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209531-a-vaccine-composition-comprising-novel-hiv-protein-construct by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:13:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209531:A VACCINE COMPOSITION COMPRISING NOVEL HIV PROTEIN CONSTRUCT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A VACCINE COMPOSITION COMPRISING NOVEL HIV PROTEIN CONSTRUCT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Pharmaceutical composi~~~ntaining a .,~ proanthocyanidin polymer composition which are useful for the treatment and prevention of secretory diarrhea are provided. The invention specifically relates to pharmaceutical formulations of a proanthocyanidin polymer composition which has been isolated from a Croton spp. or a Calophyllum spp. In particular, the invention relates to a formulation of a proanthocyanidin polymer composition which protects the composition from the effects of stomach acid after oral administration, particularly to those formulations which are enteric coated. Methods for use of the formulations as well as methods for use of the proanthocyanidin polymer composition in combination with an effective amount of a compound effective either to inhibit secretion of stomach acid or to neutralize stomach acid are disclosed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to novel HIV protein constructs, to their use in medicine, to pharmaceutical compositions containing them and to methods of their manufacture.<br>
In particular, the invention relates to fusion proteins comprising HIV-1 Tat and/or Nef proteins.<br>
HIV- 1 is the primary cause of the acquired immune deficiency syndrome (AIDS), which is regarded as one of the world's major health problems. Although extensive research throughout the world has been conducted to produce a vaccine, such efforts thus far, have not been successful.<br>
Non-envelope proteins of HIV- 1 have been described and include for example internal structural proteins such as the products of the gag and pol genes and, other non-structural proteins such as Rev, Nef, Vif and Tat (Greene et al., New England J. Med, 324, 5, 308 et seq (1991) and Bryant et al. (Ed. Pizzo), Pediatr. Infect. Dis. J., 11,5, 390 et seq (1992).<br>
HIV Nef and Tat proteins are early proteins, that is, they are expressed early in infection and in the absence of structural proteins.<br>
Accordingly the present invention provides a vaccine comprising a: fusion protein comprising<br>
a)	HIV Nef protein,<br>
b)	HIV tat protein, and<br>
c)	optionally a lipoprotein fusion partner.<br>
By 'fusion partner' is meant any protein sequence that is not Tat or Nef.<br>
Preferably the fusion partner is protein D or its' lipidated derivative Lipoprotein D,<br>
from Haemophilius influenzae B. In particular, it is preferred that the N-terminal<br><br>
unito, i.e. approximately me first 100-130 amino acids are utilised. This is represented herein as Lipo D 1/3. In a preferred embodiment of the invention the Nef protein or derivative thereof may be linked to the Tat protein or derivative thereof. Such Nef-Tat fusions may optionally also be linked to an fusion partner, such as protein D.<br>
The fusion partner is normally linked to the N-terminus of the Nef or Tat protein.<br>
Derivatives encompassed within the present invention include molecules with a C terminal Histidine tail which preferably comprises between 5-10 Histidine residues. Generally, a histidine tail containing n residues is represented herein as His (n). The presence of an histidine (or 'His') tail aids purification. More specifically, the invention provides proteins with the following structure<br>
Lipo D1/3      -	Nef	-	His Q<br>
Lipo D 1/3      -	Nef-Tat	-	His Q)<br>
ProtDl/3       -	Nef	-	His Q<br>
ProtDl/3       -	Nef-Tat	-	His G)<br>
Nef-Tat	-	His (6)<br>
Figure 1 provides the amino-acid (Seq. ID. No. 7) and DNA sequence (Seq. ID. No. 6) of the fusion partner for such constructs.<br>
In a preferred embodiment the proteins are expressed with a Histidine tail comprising between 5 to 10 and preferably six Histidine residues. These are advantageous in aiding purification. Separate expression, in yeast (Saccharomyces cerevisiae), of Nef (Macreadie I.G. et aL, 1993, Yeast 9 (6) 565-573) and Tat (Braddock M et al., 1989, Cell 58 (2) 269-79) has already been reported. Nef protein only is myristilated. The present invention provides for the first time the expression of Nef and Tat separately<br><br>
in a Pichia expression system (Nef-His and Tat-His constructs), and the successful expression of a fusion construct Nef-Tat-His. The DNA and amino acid sequences of representative Nef-His (Seq, ID. No.s 8 and 9), Tat-His (Seq. ID. No.s 10 and 1 l)and of Nef-Tat-His fusion proteins (Seq. ID. No.s 12 and 13)are set forth in Figure 2.<br>
Derivatives encompassed within the present invention also include mutated proteins. The term 'mutated' is used herein to mean a molecule which has undergone deletion, addition or substitution of one or more amino acids using well known techniques for site directed mutagenesis or any other conventional method.<br>
A mutated Tat is illustrated in Figure 2 (Seq. ID. No.s 22 and 23) as is a Nef-Tat Mutant-His (Seq. ID. No.s 24 and 25).<br>
The present invention also provides a DNA encoding the proteins of the present invention. Such sequences can be inserted into a suitable expression vector and expressed in a suitable host.<br>
A DNA sequence encoding the proteins of the present invention can be synthesized using standard DNA synthesis techniques, such as by enzymatic ligation as described by D.M. Roberts et al in Biochemistry 1985, 24, 5090-5098, by chemical synthesis, by in vitro enzymatic polymerization, or by PCR technology utilising for example a heat stable polymerase, or by a combination of these techniques.<br>
Enzymatic polymerisation of DNA may be carried out in vitro using a DNA polymerase such as DNA polymerase I (Klenow fragment) in an appropriate buffer containing the nucleoside triphosphates dATP, dCTP, dGTP and dTTP as required at a temperature of 10°-37°C, generally in a volume of 50µl or less. Enzymatic ligation of DNA fragments may be carried out using a DNA ligase such as T4 DNA ligase in an appropriate buffer, such as 0.05M Tris (pH 7.4), O.OIM MgCU, O.OIM dithiothreitol, ImM spermidine, ImM ATP and O.lmg/ml bovine serum albumin, at a temperature of 4°C to ambient, generally in a volume of 50ml or less. The chemical synthesis of the DNA polymer or fragments may be carried out by conventional<br><br>
phosphotriester, phosphite or phosphoramidite chemistry, using solid phase techniques such as those described in 'Chemical and Enzymatic Synthesis of Gene Fragments - A Laboratory Manual' (ed. H.G. Gassen and A. Lang), Verlag Chemie, Weinheim (1982), or in other scientific publications, for example M.J. Gait, H.W.D. Matthes, M. Singh, B.S. Sproat, and R.C. Titmas, Nucleic Acids Research, 1982,10, 6243; B.S. Sproat, and W. Bannwarth, Tetrahedron Letters, 1983, 24, 5771; M.D. Matteucci and M.H. Caruthers, Tetrahedron Letters, 1980, 21,719; M.D. Matteucci and M.H. Caruthers, Journal of the American Chemical Society, 1981, 103, 3185; S.P. Adams et ai, Journal of the American Chemical Society, 1983, 105, 661; N.D. Sinha, J. Biemat, J. McMannus, and H. Koester, Nucleic Acids Research, 1984,12, 4539; and H.W.D. Matthes et al, EMBO Journal, 1984, 3, 801.<br>
The invention also provides a process for preparing a protein of the invention, the process comprising the steps of:<br>
i)	preparing a replicable or integrating expression vector<br>
capable, in a host cell, of expressing a DNA polymer comprising a nucleotide sequence that encodes the protein or a derivative thereof<br>
ii)        transforming a host cell with said vector<br>
iii)       culturing said transformed host cell under conditions<br>
permitting expression of said DNA polymer to produce said protein; and<br>
iv)       recovering said protein<br>
The process of the invention may be performed by conventional recombinant techniques such as described in Maniatis et al, Molecular Cloning - A Laboratory Manual; Cold Spring Harbor, 1982-1989.<br>
The term 'transforming' is used herein to mean the introduction of foreign DNA into a host cell. This can be achieved for example by transformation, transfection or<br><br>
infection with an appropriate plasmid or viral vector using e.g. conventional techniques as described in Genetic Engineering; Eds. S.M. Kingsman and A.J. Kingsman; Blackwell Scientific Publications; Oxford, England, 1988. The term 'transformed' or 'transformant' will hereafter apply to the resulting host cell containing and expressing the foreign gene of interest.<br>
The expression vectors are novel and also form part of the invention.<br>
The replicable expression vectors may be prepared in accordance with the invention, by cleaving a vector compatible with the host cell to provide a linear DNA segment having an intact replicon, and combining said linear segment with one or more DNA molecules which, together with said linear segment encode the desired product, such as the DNA polymer encoding the protein of the invention, or derivative thereof, under ligating conditions.<br>
Thus, the DNA polymer may be preformed or formed during the construction of the vector, as desired.<br>
The choice of vector will be determined in part by the host cell, which may be prokaryotic or eukaryotic but preferably is E. coli or yeast. Suitable vectors include plasmids, bacteriophages, cosmids and recombinant viruses.<br>
The preparation of the replicable expression vector may be carried out conventionally with appropriate enzymes for restriction, polymerisation and ligation of the DNA, by procedures described in, for example, Maniatis et al cited above.<br>
The recombinant host cell is prepared, in accordance with the invention, by transforming a host cell with a replicable expression vector of the invention under transforming conditions. Suitable transforming conditions are conventional and are described in, for example, Maniatis et al cited above, or "DNA Cloning" Vol. II, D.M. Glover ed., IRL Press Ltd, 1985.<br><br>
The choice of transforming conditions is determined by the host cell. Thus, a bacterial host such as E. coli may be treated with a solution of CaCl2 (Cohen ei al., Proc. Nat. Acad. Sci., 1973, 69, 2110) or with a solution comprising a mixture of RbCl, MnCl2, potassium acetate and glycerol, and then with 3-[N-morpholino]-propane-sulphonic acid, RbCl and glycerol. Mammalian cells in culture may be transformed by calcium co-precipitation of the vector DNA onto the cells. The invention also extends to a host cell transformed with a replicable expression vector of the invention.<br>
Culturing the transformed host cell under conditions permitting expression of the DNA polymer is carried out conventionally, as described in, for example, Maniatis et al and "DNA Cloning" cited above. Thus, preferably the cell is supplied with nutrient and cultured at a temperature below 50°C.<br>
The product is recovered by conventional methods according to the host cell. Thus, where the host cell is bacterial, such as E. coli - or yeast such as Pichia; it may be lysed physically, chemically or enzymatically and the protein product isolated from the resulting lysate. Where the host cell is mammalian, the product may generally be isolated from the nutrient medium or from cell free extracts. Conventional protein isolation techniques include selective precipitation, adsorption chromatography, and affinity chromatography including a monoclonal antibody affinity column.<br>
For proteins of the present invention provided with Histidine tails, purification can easily be achieved by the use of a metal ion affinity column. In a preferred embodiment, the protein is further purified by subjecting it to cation ion exchange chromatography and/or Gel filtration chromatography. The protein is then sterilised by passing through a 0.22 µm membrane.<br>
The proteins of the invention can then be formulated as a vaccine, or the Histidine residues enzymatically cleared.<br><br>
The proteins of the present invention are provided preferably at least 80% pure more preferably 90% pure as visualised by SDS PAGE. Preferably the proteins appear as a single band by SDS PAGE.<br>
The present invention also provides pharmaceutical composition comprising a protein of the present invention in a pharmaceutically acceptable excipient.<br>
Vaccine preparation is generally described in New Trends and Developments in Vaccines, Voller et aL (eds.), University Park Press, Baltimore, Maryland, 1978. Encapsulation within liposomes is described by FuUerton, US Patent 4,235,877.<br>
The proteins of the present invention are preferably adjuvanted in the vaccine formulation of the invention. Suitable adjuvants include an aluminium salt such as aluminium hydroxide gel (alum) or aluminium phosphate, but may also be a salt of calcium, iron or zinc, or may be an insoluble suspension of acylated tyrosine, or acylated sugars, cationically or anionically derivatised polysaccharides, or polyphosphazenes.<br>
In the formulation of the inventions it is preferred that the adjuvant composition induces a preferential THl response. Suitable adjuvant systems include, for example, a combination of monophosphoryl lipid A or derivative thereof, preferably 3-de-O-acylated monophosphoryl lipid A (3D-MPL) together with an aluminium salt.<br>
An enhanced system involves the combination of a monophosphoryl lipid A and a saponin derivative particularly the combination of QS21 and 3D- MPL as disclosed in WO 94/00153, or a less reactogenic composition where the QS21 is quenched with cholesterol as disclosed in WO 96/33739.<br>
A particularly potent adjuvant formulation involving QS21, 3D-MPL &amp; tocopherol in an oil in water emulsion is described in WO 95/17210 and is a preferred formulation.<br><br>
Accordingly in one embodiment of the present invention there is provided a vaccine comprising a protein according to the invention adjuvanted with a monophosphoryl lipid A or derivative thereof, especially 3D-MPL.<br>
     Preferably the vaccine additionally comprises a saponin, more preferably QS21.<br>
Preferably the formulation additional comprises an oil in water emulsion and tocopherol. The present invention also provides a method for producing a vaccine formulation comprising mixing a protein of the present invention together with a pharmaceutically acceptable excipient, such as 3D-MPL.<br>
The vaccine of the present invention may additional comprise further HIV proteins, such as the envelope glycoprotein gpl60 or its derivative gp 120.<br>
In another aspect, the invention relates to an HIV Nef or an HIV Tat protein or derivative thereof expressed in Pichia pastor is.<br>
The invention will be further described by reference to the following examples:<br>
EXAMPLES:<br>
GJeneral<br>
Nef and Tat proteins, two regulatory proteins encoded by the human immunodeficiency virus (HIV-1) were produced in E.coli and in the methylotrophic yeast Pichia pastor is.<br>
The we/gene from the Bru/Lai isolate (Cell 40: 9-17,1985) was selected for these constructs since this gene is among those that are most closely related to the consensus Nef.<br><br>
The starting material for the Bru/Lai ne/gene was a 1170bp DNA fragment cloned on the mammalian expression vector pcDNA3 (pcDNA3/nef).<br>
The tat gene originates from the BH10 molecular clone. This gene was received as an HTLV III cDNA clone named pCVl and described in Science, 229, p69-73, 1985.<br>
1.	EXPRESSION OF HIV-1 nef AND tat SEQUENCES IN E.COLL<br>
Sequences encoding the Nef protein as well as a fusion of nef and tat sequences were placed in plasmids vectors: pRIT14586 and pRIT14589 (see figure 1).<br>
Nef and the Nef-Tat fusion were produced as fusion proteins using as fusion partner a part of the protein D. Protein D is an immunoglobulin D binding protein exposed at the surface of the gram-negative bacterium Haemophilus influenzae.<br>
pRIT14586 contains, under the control of a A.PL promoter, a DNA sequence derived from the bacterium Haemophilus influenzae which codes for the first 127 amino acids of the protein D (Infect. Immun. 60 : 1336-1342, 1992), immediately followed by a multiple cloning site region plus a DNA sequence coding for one glycine, 6 histidines residues and a stop codon (Fig. 1 A).<br>
This vector is designed to express a processed lipidated His tailed fusion protein (LipoD fusion protein). The fusion protein is synthesised as a precursor with an 18 amino acid residues long signal sequence and after processing, the cysteine at position 19 in the precursor molecule becomes the amino terminal residue which is then modified by covalently bound fatty acids (Fig.IB).<br>
pRIT14589 is almost identical to pRIT14586 except that the protD derived sequence starts immediately after the cysteine 19 codon.<br>
Expression from this vector resuhs in a His tailed, non lipidated fusion protein (Prot D fusion protein).<br><br>
Four constructs were made: LipoD-nef-His, LipoD-nef-tat-His, ProtD-nef-His, and ProtD-nef/-tat-His.<br>
The first two constructs were made using the expression vector pRIT14586, the last two constructs used pRIT14589.<br>
1.1       CONSTRUCTION OF THE RECOMBINANT STRAIN ECLD-Nl PRODUCING THE LIPOD-Nef-HIS FUSION PROTEIN.<br>
1.1.1    Construction of the lipoD-nef-His expression plasmid pRIT14595<br>
The nef gene(Bru/Lai isolate) was amplified by PCR from pcDNA3/Nef plasmid with primers 01 and 02.<br>
Ncol PRIMER 01 (Seq ID NO 1): 5'ATCGTCCATG.GGT.GGC.AAG.TGG.T 3'<br>
Spel PRIMER 02 (Seq ID NO 2): 5' CGGCTACTAGTGCAGTTCTTGAA 3'<br>
The nef DNA region amplified starts at nucleotide 8357 and terminates at nucleotide 8971 (Cell, 40: 9-17, 1985).<br>
An Ncol restriction site (which carries the ATG codon of the nef gene) was introduced at the 5'end of the PCR fragment while a Spel site was introduced at the 3' end.<br>
The PCR fragment obtained and the expression plasmid pRIT14586 were both restricted by Ncol and Spel, purified on an agarose gel, ligated and transformed in the<br><br>
appropriate E.coli host cell, strain AR58.This strain is a cryptic λ lysogen derived from N99 that is galE::TnlO, A-8 (chlD-pgl), A-Hl (cro-cMA), N+ and cI857.<br>
The resulting recombinant plasmid received, after verification of the we/amplified region by automatic sequencing,(see section 1.1.2 below) the pRIT14595 denomination.<br>
1.1,2    Selection of transformants of E1. Coli strain AR58 with pRIT14595<br>
When transformed in AR58 E.coli host strain, the recombinant plasmid directs the heat-inducible production of the heterologous protein.<br>
Heat inducible protein production of several recombinant lipoD-Nef-His transformants was analysed by Coomassie Blue stained SDS-PAGE. AH the transformants analysed showed an heat inducible heterologous protein production. The abundance of the recombinant Lipo D-Nef-Tat-His fusion protein was estimated at 10% of total protein.<br>
One of the transformants was selected and given the laboratory accession number ECLD-NL<br>
The recombinant plasmid was reisolated from strain ECLD-Nl, and the sequence of the nef-His coding region was confirmed by automated sequencing .This plasmid received the official designation pRIT 14595.<br>
The fully processed and acylated recombinant Lipo D-nef-His fusion protein produced by strain ECLD-Nl is composed of:<br>
°Fatty acids<br>
°109 a.a. of proteinD (starting at a.a.l9 and extending to a.a.l27).<br><br>
°A niethionine, created by the use of Ncol cloning site of pRIT14586<br>
(Fig.l).<br>
°205a.a. of Nef protein (starting at a.a.2 and extending to a.a.206).<br>
°A threonine and a serine created by the cloning procedure (cloning at Spel<br>
site of pRIT14586).<br>
°One glycine and six histidines.<br>
1.2       CONSTRUCTION OF RECOMBINANT STRAIN ECD-Nl PRODUCING PROT D-Nef-HIS FUSION PROTEIN.<br>
Construction of expression plasmid pRIT14600 encoding the Prot D-Nef-His fusion protein was identical to the plasmid construction described in example 1.1.1 with the exception that pRIT14589 was used as receptor plasmid for the PCR amplified nef fragment.<br>
E.coli AR58 strain was transformed with pRIT14600 and transformants were analysed as described in example 1.1.2. The transformant selected received laboratory accession number ECD-Nl.<br><br>
1.3       CONSTRUCTION OF RECOMBINANT STRAIN ECLD-NT6 PRODUCING THE LIPO D-Nef-Tat-HIS FUSION PROTEIN.<br>
1.3.1	Construction of the lipo D-Nef-Tat-His expression plasmid pRIT14596<br>
The tat gene(BH10 isolate) was amplified by PCR fi-om a derivative of the pCVl plasmid with primers 03 and 04. Spel restriction sites were introduced at both ends of the PCR fragment.<br>
Spel PRIMER 03 (Seq ID NO 3): 5' ATCGTACTAGT.GAG.CCA.GTA.GAT.C 3'<br>
Spel PRIMER 04 (Seq ID NO 4): 5' CGGCTACTAGTTTCCTTCGGGCCT 3'<br>
The nucleotide sequence of the amplified tat gene is illustrated in the pCVl clone (Science 229 : 69-73, 1985) and covers nucleotide 5414 till nucleotide 7998.<br>
The PCR fragment obtained and the plasmid pRIT14595 (expressing lipoD-Nef-His protein) were both digested by Spel restriction enzyme, purified on an agarose gel, ligated and transformed in competent AR58 cells. The resulting recombinant plasmid received, after verification of the tat amplified sequence by automatic sequencing (see section 1.3.2 below), the pRIT14596 denomination.<br>
1.3.2	Selection of transformants of strain AR58 with pRIT14596<br>
Transformants were grown, heat induced and their proteins were analysed by Coomassie Blue stained gels. The production level of the recombinant protein was estimated at 1% of total protein. One recombinant strain was selected and received the laboratory denomination ECLD-NT6.<br><br>
The lipoD-nef- tat -His recombinant plasmid was reisolated from ECLD-NT6 strain, sequenced and received the official designation pRIT14596.<br>
The fully processed and acylated recombinant Lipo D-Nef-Tat-His fusion protein produced by strain ECLD-N6 is composed of:<br>
°Fatty acids<br>
°109 a.a. of proteinD (starting at a.a.l9 and extending to a.a.l27). °A methionine, created by the use of Ncol cloning site of pRIT14586. °205a.a. of the Nef protein (starting at a.a.2 and extending to a.a.206) °A threonine and a serine created by the cloning procedure °85a.a. of the Tat protein (starting at a.a.2 and extending to a.a.86) °A threonine and a serine introduced by cloning procedure °One glycine and six histidines.<br>
1.4      CONSTRUCTION OF RECOMBINANT STRAIN ECD-NTl PRODUCING PROT D-Nef-Tat-HIS FUSION PROTEIN.<br>
Construction of expression plasmid pRIT 14601 encoding the Prot D-Nef-Tat-His fusion protein was identical to the plasmid construction described in example 1.3.1 with the exception that pRIT 14600 was used as receptor plasmid for the PCR amplified nef fragment.<br>
E.coli AR58 strain was transformed with pRIT 14601 and transformants were analysed as described previously. The transformant selected received laboratory accession number ECD-NTl.<br><br>
2.        EXPRESSION OF HIV-1 nef ANB tat SEQUENCES IN PICHIA PASTORIS.<br>
Nef protein, Tat protein and the fusion Nef-Tat were expressed in the methylotrophic yeast Pichia past oris under the control of the inducible alcohol oxidase (AOXl) promoter.<br>
To express these HIV-1 genes a modified version of the integrative vector PHIL-D2 (INVITROGEN) was used. This vector was modified in such a way that expression of heterologous protein starts immediately after the native ATG codon of the AOXl gene and will produce recombinant protein with a tail of one glycine and six histidines residues . This PHIL-D2-M0D vector was constructed by cloning an oligonucleotide linker between the adjacent AsuII and EcoRI sites of PHIL-D2 vector (see Figure 3). In addition to the His tail, this linker carries Ncol, Spel and Xbal restriction sites between which nef, tat and nef-tat fusion were inserted.<br>
2.1       CONSTRUCTION OF THE INTEGRATIVE VECTORS pRIT14597 (encoding Nef-His protein), pRIT14598 (encoding Tat-His protein) and pRIT14599 (encoding fusion Nef-Tat-His).<br>
The nef gene was amplified by PCR from the pcDNA3/Nef plasmid with primers 01 and 02(see section 1.1.1 construction of pRIT14595).The PCR fragment obtained and the integrative PHIL-D2-M0D vector were both restricted by Ncol and Spel, purified on agarose gel and ligated to create the integrative plasmid pRIT14597 (see Figure 3).<br>
The tat gene was amplified by PCR from a derivative of the pCVl plasmid with primers 05 and 04(see section 1.3.1 construction of pRIT14596):<br>
Ncol PRIMER 05 (Seq ID NO 5): 5'ATCGTCCATGGAGCCAGTAGATC 3'<br><br>
An Ncol restriction site was introduced at the 5' end of the PCR fragment while a Spel site was introduced at the 3' end with primer 04. The PCR fragment obtained and the PHIL-D2-M0D vector were both restricted by Ncol and Spel, purified on agarose gel and ligated to create the integrative plasmid pRIT14598.<br>
To construct pRIT 14599, a 910bp DNA fragment corresponding to the nef-tat-His coding sequence was ligated between the EcoRI blunted(T4 polymerase) and Ncol sites of the PHIL-D2-M0D vector. The nef-tat-His coding fragment was obtained by Xbal blunted(T4 polymerase) and Ncol digestions of pRIT14596.<br>
2.2       TRANSFORMATION OF PICHIA PASTORIS STRAIN GS115(his4).<br>
To obtain Pichia pastoris strains expressing Nef-His, Tat-His and the fusion Nef-Tat-His, strain GS115 was transformed with linear NotI fragments carrying the respective expression cassettes plus the HIS4 gene to complement his4 in the host genome.Transformation of GS115 with Notl-linear fragments favors recombination at the AOXI locus.<br>
Multicopy integrant clones were selected by quantitative dot blot analysis and the type of integration, insertion (Mufphenotype) or transplacement (Mut'phenotype), was determined.<br>
From each transformation, one transformant showing a high production level for the recombinant protein was selected :<br>
Strain Y1738 (Muf phenotype) producing the recombinant Nef-His protein, a myristylated 215 amino acids protein which is composed of:<br>
°Myristic acid<br>
°A methionine, created by the use of Ncol cloning site of PHIL-D2-M0D vector<br>
°205 a.a. of Nef protein(starting at a.a.2 and extending to a.a.206)<br><br>
°A threonine and a serine created by the cloning procedure'(cloning at Spel site of PHIL-D2-M0D vector. °One glycine and six histidines.<br>
Strain Y1739 (Mut+ phenotype) producing the Tat-His protein, a 95 amino acid protein which is composed of:<br>
°A methionine created by the use of Ncol cloning site<br>
°85 a.a. of the Tat protein(starting at a.a.2 and extending to a.a.86)<br>
°A threonine and a serine introduced by cloning procedure °One glycine and six histidines<br>
Strain Y1737(Muf phenotype) producing the recombinant Nef-Tat-His fusion protein, a myristylated 302 amino acids protein which is composed of:<br>
°Myristic acid<br>
°A methionine, created by the use of Ncol cloning site<br>
°205a.a. of Nef protein(starting at a.a.2 and extending to a.a.206)<br>
°A threonine and a serine created by the cloning procedure<br>
°85a.a. of the Tat protein(starting at a.a.2 and extending to a.a.86)<br>
°A threonine and a serine introduced by the cloning procedure<br>
°One glycine and six histidines<br><br>
3.	EXPRESSION OF HIV-1 Tat-MUTANT IN PICHIA PASTORIS<br>
As well as a Nef-Tat mutant fusion protein, a mutant recombinant Tat protein has also been expressed. The mutant Tat protein must be biologically inactive while maintaining its immunogenic epitopes.<br>
A double mutant tat gene, constructed by D.Clements (Tulane University) was selected for these constructs.<br>
This tat gene (originates from BH10 molecular clone) bears mutations in the active site region (Lys41-&gt;Ala)and in RGD motif (Arg78--&gt;Lys and Asp80--&gt;Glu) (<br>
Virology 235: 48-64, 1997).<br>
The mutant tat gene was received as a cDNA fragment subcloned between the EcoRI and Hindlll sites within a CMV expression plasmid (pCMVLys41/KGE)<br>
3.1       CONSTRUCTION OF THE INTEGRATIVE VECTORS pRIT14912(encoding Tat mutant-His protein) and pRIT14913(encoding fusion Nef-Tat mutant-His).<br>
The tat mutant gene was amplified by PCR from the pCMVLys41/KGE plasmid with primers 05 and 04 (see section 2.1 construction of pRIT 14598)<br>
An Ncol restriction site was introduced at the 5' end of the PCR fragment while a Spel site was introduced at the 3' end with primer 04. The PCR fragment obtained and the PHIL-D2-M0D vector were both restricted by Ncol and Spel, purified on agarose gel and ligated to create the integrative plasmid pRIT14912<br><br>
To construct pRIT14913, the tat mutant gene was amplified by PCR from the pCMVLys41/KGE plasmid with primers 03 and 04 (see section 1.3.1 construction of pRIT14596).<br>
The PCR fragment obtained and the plasmid pRIT14597 (expressing Nef-His protein) were both digested by Spel restriction enzyme, purified on agarose gel and ligated to create the integrative plasmid pRIT14913<br>
3.2      TRANSFORMATION OF PICHIA PASTORIS STRAIN GS115.<br>
Pichia pastoris strains expressing Tat mutant-His protein and the fusion Nef-Tat mutant-His were obtained, by applying integration and recombinant strain selection strategies previously described in section 2.2 .<br>
Two recombinant strains producing Tat mutant-His protein ,a 95 amino-acids protein, were selected:   Y1775 (Mut+ phenotype) and Y1776(Muts phenotype).<br>
One recombinant strain expressing Nef-Tat mutant-His fusion protein, a 302 amino-acids protein was selected:   Y1774(Mut+ phenotype).<br><br>
4.        PURIFICATION OF Nef-Tat-His FUSION PROTEIN (PICHIA PASTORIS)<br>
The purification scheme has been developed from 146g of recombinant Pichia pastoris cells (wet weight) or 2L Dyno-mill homogenate OD 55. The chromatographic steps are performed at room temperature. Between steps , Nef-Tat positive fractions are kept overnight in the cold room (+4°C); for longer time, samples are frozen at -20°C.<br>
146g of Pichia pastoris cells<br>
Homogenization	Buffer: 2L 50 mM PO4 pH 7.0<br>
final OD:50<br>
Dyno-mill disruption (4 passes)<br>
Centrifugation	JAl 0 rotor / 9500 rpm/ 30 min / room temperature<br>
Dyno-mill Pellet<br>
Wash	Buffer: +2L 10 mM PO4 pH 7.5 - 150mM - NaCl<br>
(lh-4°C)	0,5%empigen<br>
Cenfrifugation	JAIO rotor / 9500 rpm/ 30 min / room temperatur<br><br><br><br><br>
Purity<br>
The level of purity as estimated by SDS-PAGE is shown in Figure 4 by Daiichi Silver Staining and in Figure 5 by Coomassie blue G250.<br><br>
After Superdex200 step:	&gt; 95%<br>
After dialysis and sterile filtration steps:	&gt; 95%<br>
Recovery<br>
51mg of Nef-Tat-his protein are purified fi:om 146g of recombinant Pichia pastoris cells (= 2L of Dyno-mill homogenate OD 55)<br>
5. VACCINE PREPARATION<br>
A vaccine prepared in accordance with the invention comprises the expression product of a DNA recombinant encoding an antigen as exemplified in example 1 or 2 and as adjuvant, the formulation comprising a mixture of 3 de -0-acylated monophosphoryl lipid A 3D-MPL and QS21 in an oil/water emulsion.<br>
3D-MPL: is a chemically detoxified form of the lipopoly saccharide (LPS) of the Gram-negative bacteria Salmonella minnesota.<br>
Experiments performed at Smith Kline Beecham Biologicals have shown that 3D-MPL combined with various vehicles strongly enhances both the humoral and a THl type of cellular immunity.<br>
QS21: is one saponin purified from a crude extract of the bark of the Quillaja Saponaria Molina tree, which has a strong adjuvant activity: it activates both antigen-specific lymphoproliferation and CTLs to several antigens. Experiments performed at Smith Kline Beecham Biologicals have demonstrated a clear synergistic effect of combinations of 3D-MPL and QS21 in the induction of both humoral and THl type cellular immune responses.<br>
The oil/water emulsion is composed of 2 oils (a tocopherol and squalene), and of PBS containing Tween 80 as emulsifier. The emulsion comprised 5% squalene 5%<br><br>
tocopherol 0.4% Tween 80 and had an average particle size of 180 nm (see WO 95/17210).<br>
Experiments performed at Smith Kline Beecham Biologicals have proven that the adjunction of this 0/W emulsion to 3D-MPL/QS21 further increases their immunostimulant properties.<br>
Preparation of the oil/water emulsion (2 fold concentrate)<br>
Tween 80 is dissolved in phosphate buffered saline (PBS) to give a 2% solution in the PBS. To provide 100ml two fold concentrate emulsion 5g of DL alpha tocopherol and 5ml of squalene are vortexed to mix thoroughly. 90ml of PBS/Tween solution is added and mixed thoroughly. The resulting emulsion is then passed through a syringe and finally microfluidised by using an Ml 1 OS microfluidics machine. The resulting oil droplets have a size of approximately 180 nm.<br>
Preparation of oil in water formulation.<br>
Antigen prepared in accordance with example 1 or 2 (5µg) was diluted in 10 fold concentrated PBS pH 6.8 and H2O before consecutive addition of SB62, 3D-MPL (5 µg), QS21 (5|ig) and 50 |ig/ml thiomersal as preservative at 5 min interval. The emulsion volume is equal to 50% of the total volume (50µl for a dose of 100µl).<br>
All incubations were carried out at room temperature with agitation.<br>
6.        IMMUNOGENICITY OF Tat AND Nef-Tat IN RODENTS<br>
Characterization of the immune response induced after immunization with Tat and NefTat was carried out. To obtain information on isotype profiles and cell-mediated immunity (CMI) two immunization experiments in mice were conducted. In the first experiment mice were immunized twice two weeks apart into the footpad with Tat or<br><br>
NefTat in the oxydized or reduced form, respectively. Antigens were formulated in an oil in water emulsion comprising squalene, tween 80 Tm (polyoxyethylene sorbitan monooleate) QS21, 3D-MPL and a-tocopherol, and a control group received the adjuvant alone. Two weeks after the last immunization sera were obtained and subjected to Tat-specific ELISA (using reduced Tat for coating) for the determination of antibody titers and isotypes (Figure 6a). The antibody titers were highest in the mice having received oxydized Tat. In general, the oxydized molecules induced higher antibody titers than the reduced forms, and Tat alone induced higher antibody titers than NefTat. The latter observation was confirmed in the second experiment. Most interestingly, the isotype profile of Tat-specific antibodies differed depending on the antigens used for immunization. Tat alone elicited a balanced IgGland IgG2a profile, while NefTat induced a much stronger TH2 bias (Figure 6b). This was again confirmed in the second experiment.<br>
In the second mouse experiment animals received only the reduced forms of the molecules or the adjuvant alone. Besides serological analysis (see above) lymphoproliferative responses from lymph node cells were evaluated. After restimulation of those cells in vitro with Tat or NefTat ^H-thymidine incorporation was measured after 4 days of culture. Presentation of the results as stimulation indices indicates that very strong responses were induced in both groups of mice having received antigen (Figure 7).<br>
In conclusion, the mice studies indicate that Tat as well as Nef-Tat are highly immunogenic candidate vaccine antigens. The immune response directed against the two molecules is characterized by high antibody responses with at least 50% IgGl. Furthermore, strong CMI responses (as measured by lymphoproliferation) were observed.<br>
7.	FUNCTIONAL PROPERTIES OF THE Tat AND Nef-Tat PROTEINS<br>
The Tat and NefTat molecules in oxydized or reduced form were investigated for their ability to bind to human T cell lines. Furthermore, the effect on growth of<br><br>
those cell lines was assessed. ELISA plates were coated overnight with different concentration of the Tat and NefTat proteins, the irrelevant gD from herpes simplex virus type II, or with a buffer control alone. After removal of the coating solution HUT-78 cells were added to the wells. After two hours of incubation the wells were washed and binding of cells to the bottom of the wells was assessed microscopically. As a quantitative measure cells were stained with toluidine blue, lysed by SDS, and the toluidine blue concentration in the supernatant was determined with an ELISA plate reader. The results indicate that all four proteins, Tat and NefTat in oxydized or reduced form mediated binding of the cells to the ELISA plate (Figure 8). The irrelevant protein (data not shown) and the buffer did not fix the cells. This indicates that the recombinantly expressed Tat-containing proteins bind specifically to human T cell lines.<br>
In a second experiment HUT-78 cells were left in contact with the proteins for 16 hours. At the end of the incubation period the cells were labeled with [^H]-thymidine and the incorporation rate was determined as a measure of cell growth. All four proteins included in this assay inhibited cell growth as judged by diminished radioactivity incorporation (Figure 9). The buffer control did not mediate this effect. These results demonstrate that the recombinant Tat-containing proteins are capable of inhibiting growth of a human T cell line.<br>
In summary the functional characterization of the Tat and NefTat proteins reveals that these proteins are able to bind to human Tcell lines. Furthermore, the proteins are able to inhibit growth of such cell lines.<br><br><br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID N0:1: ATCGTCCATG .GGT.GGC.A AG.TGG.T<br>
(2) INFORMATION FOR SEQ ID NO:2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 23 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2: CGGCTACTAG TGCAGTTCTT GAA<br>
(2) INFORMATION FOR SEQ ID NO:3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 29 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3: ATCGTACTAG T.GAG.CCA. GTA.GAT.C<br>
(2) INFORMATION FOR SEQ ID NO:4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 24 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4: CGGCTACTAG TTTCCTTCGG GCCT<br>
(2) INFORMATION FOR SEQ ID NO:5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 23 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5: ATCGTCCATG GAGCCAGTAG ATC<br><br>
(2) INFORMATION FOR SEQ ID NO: 6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 441 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:<br>
ATGGATCCAA AAACTTTAGC CCTTTCTTTA TTAGCAGCTG GCGTACTAGC AGGTTGTAGC	60<br>
AGCCATTCAT CAAATATGGC GAATACCCAA ATGAAATCAG ACAAAATCAT TATTGCTCAC	120<br>
CGTGGTGCTA GCGGTTATTT ACCAGAGCAT ACGTTAGAAT CTAAAGCACT TGCTTTTGCA	180<br>
CAACAGGCTG ATTATTTAGA GCAAGATTTA GCAATGACTA AGGATGGTCG TTTAGTGGTT	240<br>
ATTCACGATC ACTTTTTAGA TGGCTTGACT GATGTTGCGA AAAAATTCCC ACATCGTCAT	300<br>
CGTAAP.GATG GCCGTTACTA TGTCATCGAC TTTACCTTAA AAGAAATTCA AAGTTTAGAA	360<br>
ATGACAGAAA ACTTTGAAAC CATGGCCACG TGTGATCAGA GCTCAACTAG TGGCCACCAT	420<br>
CACCATCACC ATTAATCTAG A	441<br>
(2) INFORMATION FOR SEQ ID NO:7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 144 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:<br>
Met Asp Pro Lys Thr Leu Ala Leu Ser Leu Leu Ala Ala G-ly Val Leu<br>
15	10	15<br>
Ala Gly Cys Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lys<br>
20	25	30<br>
Ser Asp Lys Ile Ile Ile Ala His Arg Gly Ala Ser Gly Tyr Leu Pro<br>
35	40	45<br>
Glu His Thr Leu Glu Ser Lys Ala Leu Ala Phe Ala Gin Gln Ala Asp<br>
50	55	60<br>
Tyr Leu Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val<br>
65	70	75	80<br>
Ile His Asp His Phe Leu Asp Gly Leu Thr Asp Val Ala Lys Lys Phe<br>
85	90	95<br>
Pro His Arg His Arg Lys Asp Gly Arg Tyr Tyr Val Ile Asp Phe Thr<br>
100	105	110<br>
Leu Lys Glu Ile Gln Ser Leu Glu Met Thr Glu Asn Phe Glu Thr Met<br>
115	120	125<br>
Ala Thr Cys Asp Gln Ser Ser Thr Ser Gly His His His His His His<br>
130	135	140<br>
(2) INFORMATION FOR SEQ ID NO:8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 648 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br><br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:8:<br>
ATGGGTGGCA AGTGGTCAAA AAGTAGTGTG GTTGGATGGC CTACTGTAAG GGAAAGAATG	60<br>
AGACGAGCTG AGCCAGCAGC AGATGGGGTG GGAGCAGCAT CTCGAGACCT GGAAAAACAT	120<br>
GGAGCAATCA CAAGTAGCAA TACAGCAGCT ACCAATGCTG CTTGTGCCTG GCTAGAAGCA	180<br>
CAAGAGGAGG AGGAGGTGGG TTTTCCAGTC ACACCTCAGG TACCTTTAAG ACCAATGACT	24 0<br>
TACAAGGCAG CTGTAGATCT TAGCCACTTT TTAAAAGAAA AGGGGGGACT GGAAGGGCTA	300<br>
ATTCACTCCC AACGAAGACA AGATATCCTT GATCTGTGGA TCTACCACAC ACAAGGCTAC	3 60<br>
TTCCCTGATT GGCAGAACTA CACACCAGGG CCAGGGGTCA GATATCCACT GACCTTTGGA	4 20<br>
TGGTGCTACA AGCTAGTACC AGTTGAGCCA GATAAGGTAG AAGAGGCCAA TAAAGGAGAG	4 80<br>
AACACCAGCT TGTTACACCC TGTGAGCCTG CATGGAATGG ATGACCCTGA GAGAGAAGTG	54 0<br>
TTAGAGTGGA GGTTTGACAG CCGCCTAGCA TTTCATCACG TGGCCCGAGA GCTGCATCCG	600<br>
GAGTACTTCA AGAACTGCAC TAGTGGCCAC CATCACCATC ACCATTAA	648<br>
(2) INFORMATION FOR SEQ ID NO:9:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 216 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:<br>
Met Gly Giy Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val<br>
15	10	15<br>
Arg Glu Arg Met Arg Arg Ala Giu Pro Ala Ala Asp Giy Val Gly Ala<br>
20	25	30<br>
Ala Ser Arg Asp Leu Glu Lys His Gly Ala Ile Thr Ser Ser Asn Thr<br>
35	40	45<br>
Ala Ala Thr Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu<br>
50	55	60<br>
Glu Val Gly Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr<br>
65	70	75	80<br>
Tyr Lys Ala Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly<br>
85	90	95<br>
Leu Glu Gly Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu<br>
100	105	110<br>
Trp Ile Tyr His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr<br>
115	120	125<br>
Pro Gly Pro Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys<br>
130	135	140<br>
Leu Val Pro Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu<br>
145	150	155	160<br>
Asn Thr Ser Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro<br>
165	170	175<br>
Glu Arg Glu Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His<br>
180	185	190<br>
His Val Ala Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser<br>
195	200	205<br>
Gly His His His His His His<br>
210	215<br>
(2) INFORMATION FOR SEQ ID NO:10:<br><br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 288 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:<br>
ATGGAGCCAG TAGATCCTAG ACTAGAGCCC TGGAAGCATC CAGGAAGTCA GCCTAAAACT	60<br>
GCTTGTACCA ATTGCTATTG TAAAAAGTGT TGCTTTCATT GCCAAGTTTG TTTCATAACA	120<br>
AAAGCCTTAG GCATCTCCTA TGGCAGGAAG AAGCGGAGAC AGCGACGAAG ACCTCCTCAA	180<br>
GGCAGTCAGA CTCATCAAGT TTCTCTATCA AAGCAACCCA CCTCCCAATC CCGAGGGGAC	240<br>
CCGACAGGCC CGAAGGAAAC TAGTGGCCAC CATCACCATC ACCATTAA	288<br>
(2) INFORMATION FOR SEQ ID NO:11:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 96 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:<br>
Met Glu Pro Val Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser<br>
15	10	15<br>
Gin Pro Lys Thr Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe<br>
20	25	30<br>
His Cys Gln Val Cys Phe He Thr Lys Ala Leu Gly He Ser Tyr Gly<br>
35	40	45<br>
Arg Lys Lys Arg Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr<br>
50	55	60<br>
His Gln Val Ser Leu Ser Lys Gln Pro Thr Ser Gln Ser Arg Gly Asp<br>
65	70	75	80<br>
Pro Thr Gly Pro Lys Glu Thr Ser Gly His His His His His His<br>
85	90	95<br>
(2) INFORMATION FOR SEQ ID NO:12:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 909 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:<br>
ATGGGTGGCA AGTGGTCAAA AAGTAGTGTG GTTGGATGGC CTACTGTAAG GGAAAGAATG	60<br>
AGACGAGCTG AGCCAGCAGC AGATGGGGTG GGAGCAGCAT CTCGAGACCT GGAAAAACAT	120<br>
GGAGCAATCA CAAGTAGCAA TACAGCAGCT ACCAATGCTG CTTGTGCCTG GCTAGAAGCA	180<br>
CAAGAGGAGG AGGAGGTGGG TTTTCCAGTC ACACCTCAGG TACCTTTAAG ACCAATGACT	240<br>
TACAAGGCAG CTGTAGATCT TAGCCACTTT TTAAAAGAAA AGGGGGGACT GGAAGGGCTA	300<br>
ATTCACTCCC AACGAAGACA AGATATCCTT GATCTGTGGA TCTACCACAC ACAAGGCTAC	360<br><br>
TTCCCTGATT GGCAGAACTA CACACCAGGG CCAGGGGTCA GATATCCACT GACCTTTGGA	4 20<br>
TGGTGCTACA AGCTAGTACC AGTTGAGCCA GATAAGGTAG AAGAGGCCAA TAAAGGAGAG	4 80<br>
AACACCAGCT TGTTACACCC TGTGAGCCTG CATGGAATGG ATGACCCTGA GAGAGAAGTG	540<br>
TTAGAGTGGA GGTTTGACAG CCGCCTAGCA TTTCATCACG TGGCCCGAGA GCTGCATCCG	600<br>
GAGTACTTCA AGAACTGCAC TAGTGAGCCA GTAGATCCTA GACTAGAGCC CTGGAAGCAT	660<br>
CCAGGAAGTC AGCCTAAAAC TGCTTGTACC AATTGCTATT GTAAAAAGTG TTGCTTTCAT	720<br>
TGCCAAGTTT GTTTCATAAC AAAAGCCTTA GGCATCTCCT ATGGCAGGAA GAAGCGGAGA	7 80<br>
CAGCGACGAA GACCTCCTCA AGGCAGTCAG ACTCATCAAG TTTCTCTATC AAAGCAACCC	84 0<br>
ACCTCCCAAT CCCGAGGGGA CCCGACAGGC CCGAAGGAAA CTAGTGGCCA CCATCACCAT	900<br>
CACCATTAA	909<br>
(2) INFORMATION FOR SEQ ID NO:13:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 303 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:<br>
Met Gly Gly Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val<br>
15	10	15<br>
Arg Glu Arg Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala<br>
20	25	30<br>
Ala Ser Arg Asp Leu Glu Lys His Gly Ala Ile Thr Ser Ser Asn Thr<br>
35	40	45<br>
Ala Ala Thr Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu<br>
50	55	60<br>
Glu Val Gly Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr<br>
65	70	75	80<br>
Tyr Lys Ala Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly<br>
85	90	95<br>
Leu Glu Gly Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu<br>
100	105	110<br>
Trp Ile Tyr His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr<br>
115	120	125<br>
Pro Gly Pro Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys<br>
130	135	140<br>
Leu Val Pro Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu<br>
145	150	155	160<br>
Asn Thr Ser Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro<br>
165	170	175<br>
Glu Arg Glu Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His<br>
180	185	190<br>
His Val Ala Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser<br>
195	200	205<br>
Glu Pro Val Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser Gin<br>
210	215	220<br>
Pro Lys Thr Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe His<br>
225	230	235	240<br>
Cys Gln Val Cys Phe Ile Thr Lys Ala Leu Gly Ile Ser T-yr Gly Arg<br>
245	250	255<br>
Lys Lys Arg Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr His<br>
260	265	270<br>
Gin Val Ser Leu Ser Lys Gln Pro Thr Ser Gln Ser Arg Gly Asp Pro<br><br>
275	280	285<br>
Thr Gly Pro Lys Glu Thr Ser Gly His His His His His His<br>
290	295	300<br>
(2) INFORMATION FOR SEQ ID NO:14:<br>
(i) SEQUENCE CHARACTERISTICS: (A) LENGTH: 1029 base pairs -(B) TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:<br>
ATGGATCCAA AAACTTTAGC CCTTTCTTTA TTAGCAGCTG GCGTACTAGC AGGTTGTAGC	60<br>
AGCCATTCAT CAAATATGGG GAATACCCAA ATGAAATCAG ACAAAATCAT TATTGCTCAC	120<br>
CGTGGTGCTA GCGGTTATTT ACCAGAGCAT ACGTTAGAAT CTAAAGCACT TGCTTTTGCA	180<br>
CAACAGGCTG ATTATTTAGA GCAAGATTTA GCAATGACTA AGGATGGTCG TTTAGTGGTT	2 40<br>
ATTCACGATC ACTTTTTAGA TGGCTTGACT GATGTTGCGA AAAAATTCCC ACATCGTCAT	300<br>
CGTAAAGATG GCCGTTACTA TGTCATCGAC TTTACCTTAA AAGAAATTCA AAGTTTAGAA	360<br>
ATGACAGAAA ACTTTGAAAC CATGGGTGGC AAGTGGTCAA AAAGTAGTGT GGTTGGATGG	420<br>
CCTACTGTAA GGGAAAGAAT GAGACGAGCT GAGCCAGCAG CAGATGGGGT GGGAGCAGCA	4 80<br>
TCTCGAGACC TGGAAAAACA TGGAGCAATC ACAAGTAGCA ATACAGCAGC TACCAATGCT	540<br>
GCTTGTGCCT GGCTAGAAGC ACAAGAGGAG GAGGAGGTGG GTTTTCCAGT CACACCTCAG	600<br>
GTACCTTTAA GACCAATGAC TTACAAGGCA GCTGTAGATC TTAGCCACTT TTTAAAAGAA	660<br>
AAGGGGGGAC TGGAAGGGCT AATTCACTCC CAACGAAGAC AAGATATCCT TGATCTGTGG	720<br>
ATCTACCACA CACAAGGCTA CTTCCCTGAT TGGCAGAACT ACACACCAGG GCCAGGGGTC	780<br>
AGATATCCAC TGACCTTTGG ATGGTGCTAC AAGCTAGTAC CAGTTGAGCC AGATAAGGTA	8 40<br>
GAAGAGGCCA ATAAAGGAGA GAACACCAGC TTGTTACACC CTGTGAGCCT GCATGGAATG	900<br>
GATGACCCTG AGAGAGAAGT GTTAGAGTGG AGGTTTGACA GCCGCCTAGC ATTTCATCAC	960<br>
GTGGCCCGAG AGCTGCATCC GGAGTACTTC AAGAACTGCA CTAGTGGCCA CCATCACCAT	1020<br>
CACCATTAA	1029<br>
(2) INFORMATION FOR SEQ ID NO:15:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 325 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:<br>
Cys Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lys Ser Asp<br>
15	10	15<br>
Lys Ile Ile Ile Ala His Arg Gly Ala Ser Gly Tyr Leu Pro Glu His<br>
20	25	30<br>
Thr Leu Glu Ser Lys Ala Leu Ala Phe Ala Gin Gln Ala Asp Tyr Leu<br>
35	40	45<br>
Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val Ile His<br>
50	55	60<br>
Asp His Phe Leu Asp Gly Leu Thr Asp Val Ala Lys Lys Phe Pro His<br>
65	70	75	80<br>
Arg His Arg Lys Asp Gly Arg Tyr Tyr Val Ile Asp Phe Thr Leu Lys<br>
85	90	95<br><br>
Glu Ile Gln Ser Leu Glu Met Thr Glu Asn Phe Glu Thr Met Gly Gly<br>
100	105	110<br>
Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val Arg Glu Arg<br>
115	120	125<br>
Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala Ala Ser Arg<br>
130	135	140<br>
Asp Leu Glu Lys His Gly Ala Ile Thr Ser ,Ser Asn Thr Ala Ala Thr<br>
145	150	155	160<br>
Asn Ala Ala Cys Ala  Trp Leu Glu Ala Gln Glu Glu Glu Glu Val Gly<br>
165	170	175<br>
Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr Tyr Lys Ala<br>
180	185	190<br>
Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly Leu Glu Gly<br>
195	200	205<br>
Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu Trp Ile Tyr<br>
210	215	220<br>
His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr Pro Gly Pro<br>
225	230	235	240<br>
Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys Leu Val Pro<br>
245	250	255<br>
Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu Asn Thr Ser<br>
260	265	270<br>
Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro Glu Arg Glu<br>
275	280	285<br>
Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His His Val Ala<br>
290	295	300<br>
Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser Gly His His<br>
305	310	315	320<br>
His His His His<br>
(2) INFORMATION FOR SEQ ID NO:16:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1290 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:<br>
ATGGATCCAA AAACTTTAGC CCTTTCTTTA TTAGCAGCTG GCGTACTAGC AGGTTGTAGC	60<br>
AGCCATTCAT CAAATATGGC GAATACCCAA ATGAAATCAG ACAAAATCAT TATTGCTCAC	120<br>
CGTGGTGCTA GCGGTTATTT ACCAGAGCAT ACGTTAGAAT CTAAAGCACT TGCGTTTGCA	180<br>
CAACAGGCTG ATTATTTAGA GCAAGATTTA GCAATGACTA AGGATGGTCG TTTAGTGGTT	240<br>
ATTCACGATC ACTTTTTAGA TGGCTTGACT GATGTTGCGA AAAAATTCCC ACATCGTCAT	300<br>
CGTAAAGATG GCCGTTACTA TGTCATCGAC TTTACCTTAA AAGAAATTCA AAGTTTAGAA	360<br>
ATGACAGAAA ACTTTGAAAC CATGGGTGGC AAGTGGTCAA AAAGTAGTGT GGTTGGATGG	4 20<br>
CCTACTGTAA GGGAAAGAAT GAGACGAGCT GAGCCAGCAG CAGATGGGGT GGGAGCAGCA	4 80<br>
TCTCGAGACC TGGAAAAACA TGGAGCAATC ACAAGTAGCA ATACAGCAGC TACCAATGCT	540<br>
GCTTGTGCCT GGCTAGAAGC ACAAGAGGAG GAGGAGGTGG GTTTTCCAGT CACACCTCAG	600<br>
GTACCTTTAA GACCAATGAC TTACAAGGCA GCTGTAGATC TTAGCCACTT TTTAAAAGAA	660<br>
AAGGGGGGAC TGGAAGGGCT AATTCACTCC CAACGAAGAC AAGATATCCT TGATCTGTGG	720<br>
ATCTACCACA CACAAGGCTA CTTCCCTGAT TGGCAGAACT ACACACCAGG GCCAGGGGTC	7 80<br>
AGATATCCAC TGACCTTTGG ATGGTGCTAC AAGCTAGTAC CAGTTGAGCC AGATAAGGTA	840<br>
GAAGAGGCCA ATAAAGGAGA GAACACCAGC TTGTTACACC CTGTGAGCCT GCATGGAATG	900<br><br>
GATGACCCTG AGAGAGAAGT GTTAGAGTGG AGGTTTGACA GCCGCCTAGC ATTTCATCAC	960<br>
GTGGCCCGAG AGCTGCATCC GGAGTACTTC AAGAACTGCA CTAGTGAGCC AGTAGATCCT	1020<br>
AGACTAGAGC CCTGGAAGCA TCCAGGAAGT CAGCCTAAAA CTGCTTGTAC CAATTGCTAT	1080<br>
TGTAAAAAGT GTTGCTTTCA TTGCCAAGTT TGTTTCATAA CAAAAGCCTT AGGCATCTCC	1140<br>
TATGGCAGGA AGAAGCGGAG ACAGCGACGA AGACCTCCTC AAGGCAGTCA GACTCATCAA	1200<br>
GTTTCTCTAT CAAAGCAACC CACCTCCCAA TCCCGAGGGG ACCCGACAGG CCCGAAGGAA	12 60<br>
ACTAGTGGCC ACCATCACCA TCACCATTAA	1290<br>
(2) INFORMATION FOR SEQ ID NO:17:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 412 amino acids<br>
(B)	TYPE:   amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:<br>
Cys Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lys Ser Asp<br>
15	10	15<br>
Lys He He He Ala His Arg Giy Ala Ser Gly Tyr Leu Pro Glu His<br>
20	25	30<br>
Thr Leu Glu Ser Lys Ala Leu Ala Phe Ala Gin Gln Ala Asp Tyr Leu<br>
35	40	45<br>
Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val He His<br>
50	55	60<br>
Asp His Phe Leu Asp Gly Leu Thr Asp Val Ala Lys Lys Phe Pro His<br>
65	70	75	80<br>
Arg His Arg Lys Asp Gly Arg Tyr Tyr Val He Asp Phe Thr Leu Lys<br>
85	90	95<br>
Glu He Gln Ser Leu Glu Met Thr Glu Asn Phe Glu Thr Met Gly Gly<br>
100	105	110<br>
Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val Arg Glu Arg<br>
115	120	125<br>
Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala Ala Ser Arg<br>
130	135	140<br>
Asp Leu Glu Lys His Gly Ala He Thr Ser Ser Asn Thr Ala Ala Thr<br>
145	150	155	160<br>
Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu Glu Val Gly<br>
165	170	175<br>
Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr Tyr Lys Ala<br>
180	185	190<br>
Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly Leu Glu Gly<br>
195	200	205<br>
Leu He His Ser Gln Arg Arg Gln Asp He Leu Asp Leu Trp He Tyr<br>
210	215	220<br>
His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr Pro Gly Pro<br>
225	230	235	240<br>
Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys Leu Val Pro<br>
245	250	255<br>
Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu Asn Thr Ser<br>
260	265	270<br>
Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro Glu Arg Glu<br>
275	280	285  -<br>
Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His His Val Ala<br>
290	295	300<br><br>
Arg Giu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser Glu Pro Val<br>
305	310	315	320<br>
Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser Gln Pro Lys Thr<br>
325	330	335<br>
Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe His Cys Gln Val<br>
340	345	350<br>
Cys Phe Ile Thr Lys Ala Leu Gly Ile Ser Tyr Gly Arg Lys Lys Arg<br>
355	360	365<br>
Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr His Gln Val Ser<br>
370	375	380<br>
Leu Ser Lys Gln Pro Thr Ser Gln Ser Arg Gly Asp Pro Thr Gly Pro<br>
385	390	395	400<br>
Lys Glu Thr Ser Gly His His His His His His<br>
405	410<br>
(2) INFORMATION FOR SEQ ID NO:18:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 981 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18:<br>
ATGGATCCAA GCAGCCATTC ATCAAATATG GCGAATACCC AAATGAAATC AGACAAAATC	60<br>
ATTATTGCTC ACCGTGGTGC TAGCGGTTAT TTACCAGAGC ATACGTTAGA ATCTAAAGCA	120<br>
CTTGCGTTTG CACAACAGGC TGATTATTTA GAGCAAGATT TAGCAATGAC TAAGGATGGT	180<br>
CGTTTAGTGG TTATTCACGA TCACTTTTTA GATGGCTTGA CTGATGTTGC GAAAAAATTC	240<br>
CCACATCGTC ATCGTAAAGA TGGCCGTTAC TATGTCATCG ACTTTACCTT AAAAGAAATT	300<br>
CAAAGTTTAG AAATGACAGA AAACTTTGAA ACCATGGGTG GCAAGTGGTC AAAAAGTAGT	360<br>
GTGGTTGGAT GGCCTACTGT AAGGGAAAGA ATGAGACGAG CTGAGCCAGC AGCAGATGGG	420<br>
GTGGGAGCAG CATCTCGAGA CCTGGAAAAA CATGGAGCAA TCACAAGTAG CAATACAGCA	480<br>
GCTACCAATG CTGCTTGTGC CTGGCTAGAA GCACAAGAGG AGGAGGAGGT GGGTTTTCCA	540<br>
GTCACACCTC AGGTACCTTT AAGACCAATG ACTTACAAGG CAGCTGTAGA TCTTAGCCAC	600<br>
TTTTTAAAAG AAAAGGGGGG ACTGGAAGGG CTAATTCACT CCCAACGAAG ACAAGATATC	660<br>
CTTGATCTGT GGATCTACCA CACACAAGGC TACTTCCCTG ATTGGCAGAA CTACACACCA	720<br>
GGGCCAGGGG TCAGATATCC ACTGACCTTT GGATGGTGCT ACAAGCTAGT ACCAGTTGAG	780<br>
CCAGATAAGG TAGAAGAGGC CAATAAAGGA GAGAACACCA GCTTGTTACA CCCTGTGAGC	840<br>
CTGCATGGAA TGGATGACCC TGAGAGAGAA GTGTTAGAGT GGAGGTTTGA CAGCCGCCTA	900<br>
GCATTTCATC ACGTGGCCCG AGAGCTGCAT CCGGAGTACT TCAAGAACTG CACTAGTGGC	960<br>
CACCATCACC ATCACCATTA A	981<br>
(2) INFORMATION FOR SEQ ID NO:19:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 327 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID N0:19:<br>
Met Asp Pro Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lys<br>
15	10	15<br><br>
Ser Asp Lys Ile Ile Ile Ala His Arg Gly Ala Ser Gly Tyr Leu Pro<br>
20	25	30<br>
Glu His Thr Leu Glu Ser Lys Ala Leu Ala Phe Ala Gin Gln Ala Asp<br>
35	40	45<br>
Tyr Leu Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val<br>
50	55	60<br>
Ile His Asp His Phe Leu Asp Gly Leu Thr Asp Val Ala Lys Lys Phe<br>
65	70	75	80<br>
Pro His Arg His Arg Lys Asp Gly Arg Tyr Tyr Val Ile Asp Phe Thr<br>
85	90	95<br>
Leu Lys Glu Ile Gln Ser Leu Glu Met Thr Glu Asn Phe Glu Thr Met<br>
100	105	no<br>
Gly Gly Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val Arg<br>
115	120	125<br>
Glu Arg Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala Ala<br>
130	135	140<br>
Ser Arg Asp Leu Glu Lys His Gly Ala Ile Thr Ser Ser Asn Thr Ala<br>
145	150	155	160<br>
Ala Thr Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu Glu<br>
165	170	175<br>
Val Gly Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr Tyr<br>
180	185	190<br>
Lys Ala Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly Leu<br>
195	200	205<br>
Glu Gly Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu Trp<br>
210	215 .	220<br>
Ile Tyr His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr Pro<br>
225	230	235	240<br>
Gly Pro Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys Leu<br>
245	250	255<br>
Val Pro Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu Asn<br>
260	265	270<br>
Thr Ser Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro Glu<br>
275	280	285<br>
Arg Glu Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His His<br>
290	295	300<br>
Val Ala Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser Gly<br>
305	310	315	320<br>
His His His His His His<br>
325<br>
(2) INFORMATION FOR SEQ ID NO:20:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 1242 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:20:<br>
ATGGATCCAA GCAGCCATTC ATCAAATATG GCGAATACCC AAATGAAATC AGACAAAATC	60<br>
ATTATTGCTC ACCGTGGTGC TAGCGGTTAT TTACCAGAGC ATACGTTAGA ATCTAAAGCA	120<br>
CTTGCGTTTG CACAACAGGC TGATTATTTA GAGCAAGATT TAGCAATGAC TAAGGATGGT	180<br>
CGTTTAGTGG TTATTCACGA TCACTTTTTA GATGGCTTGA CTGATGTTGC GAAAAAATTC	24 0<br>
CCACATCGTC ATCGTAAAGA TGGCCGTTAC TATGTCATCG ACTTTACCTT AAAAGAAATT	300<br><br>
CAAAGTTTAG AAATGACAGA AAACTTTGAA ACCATGGGTG GCAAGTGGTC AAAAAGTAGT	360<br>
GTGGTTGGAT GGCCTACTGT AAGGGAAAGA ATGAGACGAG CTGAGCCAGC AGCAGATGGG	420<br>
GTGGGAGCAG CATCTCGAGA CCTGGAAAAA CATGGAGCAA TCACAAGTAG CAATACAGCA	480<br>
GCTACCAATG CTGCTTGTGC CTGGCTAGAA GCACAAGAGG AGGAGGAGGT GGGTTTTCCA	540<br>
GTCACACCTC AGGTACCTTT AAGACCAATG ACTTACAAGG CAGCTGTAGA TCTTAGCCAC	600<br>
TTTTTAAAAG AAAAGGGGGG ACTGGAAGGG CTAATTCACT CCCAACGAAG ACAAGATATC	660<br>
CTTGATCTGT GGATCTACCA CACACAAGGC TACTTCCCTG ATTGGCAGAA CTACACACCA	720<br>
GGGCCAGGGG  TCAGATATCC ACTGACCTTT GGATGGTGCT ACAAGCTAGT ACCAGTTGAG	780<br>
CCAGATAAGG TAGAAGAGGC CAATAAAGGA GAGAACACCA GCTTGTTACA CCCTGTGAGC	840<br>
CTGCATGGAA TGGATGACCC TGAGAGAGAA GTGTTAGAGT GGAGGTTTGA CAGCCGCCTA	900<br>
GCATTTCATC ACGTGGCCCG AGAGCTGCAT CCGGAGTACT TCAAGAACTG CACTAGTGAG	960<br>
CCAGTAGATC CTAGACTAGA GCCCTGGAAG CATCCAGGAA GTCAGCCTAA AACTGCTTGT	1020<br>
ACCAATTGCT ATTGTAAAAA GTGTTGCTTT CATTGCCAAG TTTGTTTCAT AACAAAAGCC	1080<br>
TTAGGCATCT CCTATGGCAG GAAGAAGCGG AGACAGCGAC GAAGACCTCC TCAAGGCAGT	1140<br>
CAGACTCATC AAGTTTCTCT ATCAAAGCAA CCCACCTCCC AATCCCGAGG GGACCCGACA	1200<br>
GGCCCGAAGG AAACTAGTGG CCACCATCAC CATCACCATT AA	1242<br>
(2) INFORMATION FOR SEQ ID NO:21:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 414 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:<br>
Met Asp Pro Ser Ser His Ser Ser Asn Met Ala Asn Thr Gln Met Lys<br>
15	10	15<br>
Ser Asp Lys Ile Ile Ile Ala His Arg Gly Ala Ser Gly Tyr Leu Pro<br>
20	25	30<br>
Glu His Thr Leu Glu Ser Lys Ala Leu Ala Phe Ala Gin Gln Ala Asp<br>
35	40	45<br>
Tyr Leu Glu Gln Asp Leu Ala Met Thr Lys Asp Gly Arg Leu Val Val<br>
50	55	60<br>
Ile His Asp His Phe Leu Asp Gly Leu Thr Asp Val Ala Lys Lys Phe<br>
65	70	75	80<br>
Pro His Arg His Arg Lys Asp Gly Arg Tyr Tyr Val Ile Asp Phe Thr<br>
85	90	95<br>
Leu Lys Glu Ile Gln Ser Leu Glu Met Thr Glu Asn Phe Glu Thr Met<br>
100	105	110<br>
Gly Gly Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val Arg<br>
115	120	125<br>
Glu Arg Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala Ala<br>
130	135	140<br>
Ser Arg Asp Leu Glu Lys His Gly Ala Ile Thr Ser Ser Asn Thr Ala<br>
145	150	155	160<br>
Ala Thr Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu Glu<br>
165	170	175<br>
Val Gly Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr Tyr<br>
180	185	190<br>
Lys Ala Ala Val Asp Leu Ser His Phe Leu Lys Glu Lys Gly Gly Leu<br>
195	200	205<br>
Glu Gly Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu Trp<br>
210	215	220<br>
Ile Tyr His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr Pro<br><br>
225	230	235	240<br>
Gly Pro Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys Leu<br>
245	250	255<br>
Val Pro Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu Asn<br>
260	265	270<br>
Thr Ser Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro Glu<br>
275	280	285<br>
Arg Glu Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His His<br>
290	295	300<br>
Val Ala Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser Glu<br>
305	310	315	320<br>
Pro Val Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser Gln Pro<br>
325	330	335<br>
Lys Thr Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe His Cys<br>
340	345	350<br>
Gin Val Cys   Phe Ile Thr Lys Ala Leu Gly Ile Ser Tyr Gly Arg Lys<br>
355	360	365<br>
Lys Arg Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr His Gin<br>
370	375	380<br>
Val Ser Leu Ser Lys Gln Pro Thr Ser Gln Ser Arg Gly Asp Pro Thr<br>
385	390	395	400<br>
Gly Pro Lys Glu Thr Ser Gly His His His His His His<br>
405	410<br>
(2) INFORMATION FOR SEQ ID NO:22:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 288 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:<br>
ATGGAGCCAG TAGATCCTAG ACTAGAGCCC TGGAAGCATC CAGGAAGTCA GCCTAAAACT	60<br>
GCTTGTACCA ATTGCTATTG TAAAAAGTGT TGCTTTCATT GCCAAGTTTG TTTCATAACA	120<br>
GCTGCCTTAG GCATCTCCTA TGGCAGGAAG AAGCGGAGAC AGCGACGAAG ACCTCCTCAA	180<br>
GGCAGTCAGA CTCATCAAGT TTCTCTATCA AAGCAACCCA CCTCCCAATC CAAAGGGGAG	240<br>
CCGACAGGCC CGAAGGAAAC TAGTGGCCAC CATCACCATC ACCATTAA	288<br>
(2) INFORMATION FOR SEQ ID NO:23:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 96 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:<br>
Met Glu Pro Val Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser<br>
15	10	15<br>
Gin Pro Lys Thr Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe<br>
20	25	30<br>
His Cys Gln Val Cys Phe He Thr Ala Ala Leu Gly He Ser Tyr Gly<br><br>
35	40	45<br>
Arg Lys Lys Arg Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr<br>
50	55	60<br>
His Gln Val Ser Leu Ser Lys Gln Pro Thr Ser Gln Ser Lys Gly Glu<br>
65	70	75	80<br>
Pro Thr Gly Pro Lys Glu Thr Ser Gly His His His His His His<br>
85	90	95<br>
(2) INFORMATION FOR SEQ ID NO:24:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 909 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:<br>
ATGGGTGGCA AGTGGTCAAA AAGTAGTGTG GTTGGATGGC CTACTGTAAG GGAAAGAATG	60<br>
AGACGAGCTG AGCCAGCAGC AGATGGGGTG GGAGCAGCAT CTCGAGACCT GGAAAAACAT	120<br>
GGAGCAATCA CAAGTAGCAA TACAGCAGCT ACCAATGCTG CTTGTGCCTG GCTAGAAGCA	180<br>
CAAGAGGAGG AGGAGGTGGG TTTTCCAGTC ACACCTCAGG TACCTTTAAG ACCAATGACT	240<br>
TACAAGGCAG CTGTAGATCT TAGCCACTTT TTAAAAGAAA AGGGGGGACT GGAAGGGCTA	300<br>
ATTCACTCCC AACGAAGACA AGATATCCTT GATCTGTGGA TCTACCACAC ACAAGGCTAC	360<br>
TTCCCTGATT GGCAGAACTA CACACCAGGG CCAGGGGTCA GATATCCACT GACCTTTGGA	420<br>
TGGTGCTACA AGCTAGTACC AGTTGAGCCA GATAAGGTAG AAGAGGCCAA TAAAGGAGAG	480<br>
AACACCAGCT TGTTACACCC TGTGAGCCTG CATGGAATGG ATGACCCTGA GAGAGAAGTG	540<br>
TTAGAGTGGA GGTTTGACAG CCGCCTAGCA TTTCATCACG TGGCCCGAGA GCTGCATCCG	600<br>
GAGTACTTCA AGAACTGCAC TAGTGAGCCA GTAGATCCTA GACTAGAGCC CTGGAAGCAT	660<br>
CCAGGAAGTC AGCCTAAAAC TGCTTGTACC AATTGCTATT GTAAAAAGTG TTGCTTTCAT	720<br>
TGCCAAGTTT GTTTCATAAC AGCTGCCTTA GGCATCTCCT ATGGCAGGAA GAAGCGGAGA	780<br>
CAGCGACGAA GACCTCCTCA AGGCAGTCAG ACTCATCAAG TTTCTCTATC AAAGCAACCC	840<br>
ACCTCCCAAT CCAAAGGGGA GCCGACAGGC CCGAAGGAAA CTAGTGGCCA CCATCACCAT	900<br>
CACCATTAA	909<br>
(2) INFORMATION FOR SEQ ID NO:25:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 303 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:<br>
Met Gly Gly Lys Trp Ser Lys Ser Ser Val Val Gly Trp Pro Thr Val<br>
15	10	15<br>
Arg Glu Arg Met Arg Arg Ala Glu Pro Ala Ala Asp Gly Val Gly Ala<br>
20	25	30<br>
Ala Ser Arg Asp Leu Glu Lys His Gly Ala Ile Thr Ser Ser Asn Thr<br>
35	40	45<br>
Ala Ala Thr Asn Ala Ala Cys Ala Trp Leu Glu Ala Gln Glu Glu Glu<br>
50	55	60<br>
Glu Val Gly Phe Pro Val Thr Pro Gln Val Pro Leu Arg Pro Met Thr<br>
65	70	75	80<br><br>
Tyr Lys Ala Ala Val Asp Leu Ser His Phe Leu Lys Glu L-ys Gly Gly<br>
85	90	95<br>
Leu Glu Gly Leu Ile His Ser Gln Arg Arg Gln Asp Ile Leu Asp Leu<br>
100	105	110<br>
Trp Ile Tyr His Thr Gln Gly Tyr Phe Pro Asp Trp Gln Asn Tyr Thr<br>
115	, 120	125<br>
Pro Gly Pro Gly Val Arg Tyr Pro Leu Thr Phe Gly Trp Cys Tyr Lys<br>
130	135	140<br>
Leu Val Pro Val Glu Pro Asp Lys Val Glu Glu Ala Asn Lys Gly Glu<br>
145	150	155	160<br>
Asn Thr Ser Leu Leu His Pro Val Ser Leu His Gly Met Asp Asp Pro<br>
165	170	175<br>
Glu Arg Glu Val Leu Glu Trp Arg Phe Asp Ser Arg Leu Ala Phe His<br>
180	185	190<br>
His Val Ala Arg Glu Leu His Pro Glu Tyr Phe Lys Asn Cys Thr Ser<br>
195	200	205<br>
Glu Pro Val Asp Pro Arg Leu Glu Pro Trp Lys His Pro Gly Ser Gin<br>
210	215	220<br>
Pro Lys Thr Ala Cys Thr Asn Cys Tyr Cys Lys Lys Cys Cys Phe His<br>
225	230	235	240<br>
Cys Gln Val Cys Phe Ile Thr Ala Ala Leu Gly Ile Ser Tyr Gly Arg<br>
245	250	255<br>
Lys Lys Arg Arg Gln Arg Arg Arg Pro Pro Gln Gly Ser Gln Thr His<br>
260	265	270<br>
Gin Val Ser Leu Ser Lys Gln Pro Thr Ser Gln Ser Lys Gly Glu Pro<br>
275	280	285<br>
Thr Gly Pro Lys Glu Thr Ser Gly His His His His His His<br>
290	295	300<br>
(2) INFORMATION FOR SEQ ID NO:26:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 57 base pairs<br>
(B)	TYPE: nucleic acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26: TTCGAAACCA TGGCCGCGGA CTAGTGGCCA CCATCACCAT CACCATTAAC GGAATTC        57 (2) INFORMATION FOR SEQ ID NO:27:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A)	LENGTH: 17 amino acids<br>
(B)	TYPE: amino acid<br>
(C)	STRANDEDNESS: single<br>
(D)	TOPOLOGY: linear<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:<br>
Thr Ser Gly His His His His His His<br>
1	5<br><br><br><br><br>
WE CLAIM:<br>
1.	A vaccine comprising a:<br>
fusion protein comprising<br>
a)	HIV Nef protein,<br>
b)	HIV tat protein, and<br>
c)	optionally a lipoprotein fusion partner.<br><br>
2.	The vaccine as claimed in claim 1, where the lipoprotein is Haemophilus influenza B protein D or a derivative thereof.<br>
3.	The vaccine as claimed in claim 2, wherein the protein D comprises between 100 - 130 amino acids from the N terminal of said protein.<br>
4.	The vaccine as claim in any one of claims 1 to 3 wherein the tat protein is fused to an HIV Nef protein.<br>
5.	The vaccine as claimed in any one of claims 1-4 wherein the fusion protein has a histidine tail.<br>
6.	The vaccine as claimed in any one of claims 1-5 wherein the fusion protein is carboxymethylated.<br>
7.	The vaccine as claimed in any one of claims 1-6 wherein the HIV tat protein is the entire protein.<br>
8.	The vaccine as claimed in any one of claims 1-6 wherein the HIV tat is a mutated protein.<br>
9.	The vaccine as claimed in claim 8, wherein the mutated protein has the sequence identified in Sequence Id No. 23.<br>
10.	The vaccine as claimed in any one of claims 1-9 wherein the HIV Nef protein is the entire protein.<br><br>
11.	The vaccine as claimed in any one of claims 1 - 9, wherein the HIV Nef protein<br>
is a mutated nef.<br>
12.	The vaccine as claimed in any one of 1 - 11, which additionally comprises HIV gp 160 or its derivative gp 20.<br>
13.	The vaccine as claimed in any one of claims 1-12, wherein the composition is a vaccine composition, which additionally comprises a pharmaceutically acceptable<br>
excipient.<br>
14.	The vaccine as claimed in claim 13, which additionally comprises an adjuvant.<br>
15.	The vaccine as claimed in claim 14, wherein the adjuvant is a THl inducing<br>
adjuvant.<br>
16.	The vaccine as claimed in claim 13 or 14, wherein the adjuvant comprises monophosphoryl lipid A or a derivative thereof<br>
17.	The vaccine as claimed in claim 16 wherein the adjuvant is 3 de-O-acylated monophosphoryl lipid A.<br>
18.	The vaccine as claimed in any one of claims 14 to 17 additionally comprising a<br>
saponin adjuvant.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1jbGFpbXMgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1jbGFpbXMgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">2172-mas-1998-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1jb3JyZXNwb25kbmVjZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">2172-mas-1998-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1kZXNjcmlwdGlvbihjb21wbGV0ZSlmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">2172-mas-1998-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1kZXNjcmlwdGlvbihjb21wbGV0ZSlncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">2172-mas-1998-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">2172-mas-1998-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2172-mas-1998-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1mb3JtIDQucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-form 4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE3Mi1tYXMtMTk5OC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">2172-mas-1998-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="209530-herbicidal-compositions-of-tetrazolinone-herbicides-and-antidotes-therefor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209532-pharmaceutical-composition-comprising-entric-coated-proanthocyanidin-polymer.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209531</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2172/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Sep-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Sep-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. SMITHKLINE BEECHAM BIOLOGICALS SA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RUE DE L&#x27;INSTITUT 89, B-1330 RIXENSART</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CLAUDINE BRUCK</td>
											<td>INSTITUT 89, B-1330 RIXENSART.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A 61K 039 /015</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9720585.0</td>
									<td>1997-09-26</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209531-a-vaccine-composition-comprising-novel-hiv-protein-construct by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:13:16 GMT -->
</html>
