<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209870-invltro-method-for-evaluation-of-antibody-treatment-response by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:50:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209870:INVLTRO METHOD FOR EVALUATION OF ANTIBODY TREATMENT RESPONSE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">INVLTRO METHOD FOR EVALUATION OF ANTIBODY TREATMENT RESPONSE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>1. A method of assessing the response of a subject to a therapeutic antibody treatment, comprising determining in vitro the FCGR3A158 genotype of said subject wherein it determines amino acid residue at position 158 of FcyRIIIa receptor, a Valine at position 158 being indicative of a better response to said treatment and a phenylalanine at position 158 being indicative of a lower response to said treatment by sequencing, amplifying, allele-specific restriction enzyme digestion and hybridization</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM   2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
COMPLETE SPECIFICATION (See Section 10, rule 13)<br>
INVETRO METHQD FOR EVALUATION OF ANTIBODY, TREATMENT. RESPONSE.<br>
CENTRE HOSPITALIER REGIONAL ET UNIVERSITAIRE DE TOURS of 2, Boulevard Tonnelle F-37044 TOURS CEDEX, FRANCE, FRENCH National AND INNATE PHARMA of Immeuble Grand Pre, 121 ancien Chemin de Cassis, 13009 MARSEILLE, FRANCE, FRENCH Company<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed : -<br><br><br>
ORIGINAL<br>
234/MUMNP/2004<br><br>
GRANTED<br>
1-6-2005<br><br>
METHODS AND COMPOSITIONS TO EVALUATE ANTIBODY TREATMENT RESPONSE<br>
The present invention relates to methods and compositions to evaluate or assess the response of a subject to particular therapeutic treatment. More particularly, the invention provides methods to determine the response of subjects, or to adapt the treatment protocol of subjects treated with therapeutic antibodies. The invention can be Used for patients with malignancies, particularly lymphoma, and is suited to select best responders and/or adjust treatment condition or protocol for low responders.<br>
INTRODUCTION<br>
Various therapeutic strategies in human beings are based on the use of therapeutic antibodies. This includes, for instance, the use of therapeutic antibodies developed to deplete target cells, particularly diseased cells such as virally-infected cells, tumor cells or other pathogenic cells, including allogenic immunocompetent cells. Such antibodies are typically monoclonal antibodies, of IgG species, typically JgGl and IgG3. These antibodies can be recombinant antibodies and humanized antibodies, comprising functional domains from various species or origin or specificity. A particular example of such therapeutic antibodies is rituximab (Mabthera , Rituxan' which is a chimeric anti-CD20IgGl monoclonal antibody made with human y\ and K constant regions linked to murine variable domains1. For a few years, rituximab has been considerably modifying the therapeutical strategy against B lymphoproliferative malignancies, particularly non-Hodgkin's lymphomas (NHL). Other examples of intact humanized IgGl antibodies include alemtuzumab (Campath-IH8), which is<br>
1        .<br>
used in the treatment of B cell malignancies or trastuzumab (Herceptin), which is used in the treatment of breast cancer. Additional examples of therapeutic antibodies under development are disclosed in the art.<br>
While these antibodies represent a novel efficient approach to human therapy, particularly for treatment of tumors, they do not always exhibit a strong efficacyi and,their use could be<br><br>
improved by evaluating the response of subjects thereto. For instance, while rituximab, alone or in combination with chemotherapy was shown to be effective in the treatment of both low-intermediate2-8 and high-grade NHl 6,9 30% to 50% of patients with low grade NHL have no clinical response to rituximab4, 5. It has been suggested that the level of CD20 expression on lymphoma cells2, the presence of high tumor burden at the time of treatment6 or low serum rituximab concentrations2 may explain the lack of efficacy of rituximab in some patients. Nevertheless, the actual causes of treatment failure remain largely unknown.<br>
The availability of methods allowing the evaluation of patient response to antibody treatment would greatly enhance the therapeutic efficacy of these products. However, the precise mode of action in vivo of such therapeutic antibodies is not clearly documented. Indeed, while in vitro studies suggest various possible modes of action of rituximab (antibody-dependant cell-mediated cytotoxicity (ADCC)10,11 complement-dependant cytotoxicity10,12,13, direct signalling leading to apoptosis14,15, etc.), the clear action of these target cell-depleting antibodies in vivo is not documented in humans. Furthermore, while ADCC is an important effector mechanism in the eradication of intracellular pathogens and tumor cells, the role of an ADCC is still controversia12,13.<br>
The present invention now proposes novel methods and compositions to assess the therapeutic response of a subject to a therapeutic antibody. The invention also proposes methods to select patients having best responding profile to therapeutic antibody treatment. The invention also relates to methods of treating patients with therapeutic antibodies, comprising a prior step of evaluating the patient's response. The invention also relates to compositions and kits suitable to perform the invention. The invention may as well be used in clinical trials or experimental settings, to assess or monitor a subject's response, or to verify the mode of action of an antibody.<br>
The invention is based, in part, on the demonstration of a correlation between the genotype of a subject and its ability to respond to therapeutic antibody treatment. More specifically, the<br><br><br>
invention shows that the genotype of the FcyRIIIa receptor directly correlates with the subject's response to therapeutic antibody treatment.<br>
Three classes of FcyR (FcyRI, FcyRII and FcyRIII) and their subclasses are encoded by eight genes in humans, all located on the long arm of chromosome 1. Some of these genes display a functional allelic polymorphism generating allotypes with different receptor properties. These polymorphisms have been identified as genetic factors increasing the susceptibility to<br>
autoimmune or infectious diseases19-21 One of these genetic factors is a gene dimorphism in FCGR3A, which encodes FcyRIIIa with either a phenylalanine (F) or a valine (V) at amino-acid position 15822,23. This residue directly interacts with the lower hinge region of IgG 1 as recently shown by IgGl-FcyRIII co-cristallization24. It has been clearly demonstrated that human IgGl binds more strongly to homozygous FcyRIUa-158V natural killer cells (NK) than to homozygous FcyRIIIa-158F or heterozygous NK cells22, 23<br>
We undertook to evaluate a possible correlation between the FCGR3A genotype and a patient response, to therapeutic antibody treatment in vivo. Our invention stems in part from the unexpected discovery that a very strong correlation exists between said genotype and said response profile, the presence of a valine residue at position 158 being, indicative of a high response rate. More specifically, the genotyping of FCGR3A was performed in patients with previously untreated follicular NHL who had received rituximab alone, a particular situation in<br>
which the response rate is very high5. The FCGi?2/l-13lH/R was also determined as control since this gene co-localizes with FCGR3A on chromosome lq22 and encodes the macrophage FcyRIIa receptor.<br>
The FCGR3A-158V/F genotype was determined in 47 patients having received rituximab for a previously untreated follicular non-Hodgkin's lymphoma. Clinical and molecular response were evaluated at two months (M2) and at one year (M12). Positive molecular response was defined as a disappearance of the BCL2-]Hgene rearrangement in both peripheral blood and bone marrow. FCGR3A-158V homozygous patients were 21%/ whcreas .FCGRJ3A-158F<br><br><br>
i homozygous and heterozygous patients (FCGR3A-158F carriers) were 34% and 45%,<br>
respectively. The objective response rates at M2 and M12 were 100% and 90% in FCGR3A-<br>
158V homozygous patients compared with 65% (p = 0.02) and 51% (p-0.03) in FCGR3A-<br>
158F carriers. A positive molecular response was observed at M12 in 5/6 of homozygous<br>
FCGR3A-158V patients  compared with  5/16  of FCGR3A-158F  carriers   (p-0.04).<br>
Furthermore, the homozygous FCGR3A-158V genotype was confirmed to be the single<br>
parameters associated with clinical and molecular responses in multivariate analysis and was<br>
also associated with a lower rate of disease progression (p = 0.05)<br>
r<br>
Accordingly, the present invention establishes, for the first time, an association between the FCGR3A genotype and clinical and molecular responses to therapeutic antibodies. The invention thus provides a first unique marker that can be used to monitor, evaluate or select a patient's response. This invention thus introduces new pharmacogenetical approaches in the management of patients with malignancies, viral infections or other diseases related to the presence of pathological cells in a subject, particularly non-Hodgkin's lymphoma.<br>
&gt;	<br>
An object of this invention resides in a method of assessing the response of a subject to a<br>
therapeutic antibody treatment, comprising determining in vitro the FCGR3A genotype<br>
and/or the presence of a polymorphism in the FcyRIIIa receptor of said subject. More<br>
specifically, the method comprises determining in vitro the FCGR3A158 genotype of said<br>
subject.	<br>
A further object of this invention is a method of selecting patients for therapeutic antibody treatment, the method comprising determining in vitro the FCGR3A genotype and/or the presence of a polymorphism in the Fcy-REIII a receptor of said subject. More specifically, the method comprises determining in vitro the FCGR3A158 genotype of said subject.<br>
An other object of this invention is a method of improving the efficacy or treatment condition or protocol of a therapeutic antibody treatment in a subject, comprising determining in vitro the FCGR3A genotype and/or the presence of a polymorphism in the FcyRIIIa receptor of<br><br><br>
said subject. More specifically, the method comprises determining' in vitro the FCGR3A158<br>
genotype of said subject.	<br>
More specifically, determining in vitro the FCGR3A158 genotype of a subject comprises determining amino acid residue at position 158 of Fq'RIII a receptor (or corresponding codon in the FCGR3A gene), a valine at position 158 being indicative of a better response to said treatment and a phenylalanine at position 158 being indicative of a lower response to said treatment.<br>
Within the context of this invention, the term "therapeutic antibody or antibodies" designates more specifically any antibody that functions to deplete target cells in a patient. Specific examples of such target cells include tumor cells, virus-infected cells, allogenic cells, pathological immunocompetent cells (e.g., B lymphocytes, T lymphocytes, antigen-presenting cells, etc.) involved in allergies, autoimmune diseases, allogenic reactions, etc., or even healthy cells (e.g., endothelial cells in an anti-angiogenic therapeutic strategy). Most preferred target cells within the context of this invention are tumor cells and virus-infected cells. The therapeutic antibodies may, for instance, mediate a cytotoxic effect 6r a cell lysis, panicularly by antibody-dependant cell-mediated cytotoxicity (ADCC). ADCC requires leukocyte receptors for the Fc portion of IgG (FcyR) whose function is to link the IgG-sensitized antigens to FcyR-bearing cytotoxic cells and to trigger the cell activation machinery. While this mechanism of action has not been evidenced in vivo in humans, it may account for the efficacy of such target cell-depleting therapeutic antibodies. The therapeutic antibodies may by polyclonal or, preferably, monoclonal. They may be produced by hybridomas or by recombinant cells engineered to express the desired variable and"consiant domains. The antibodies may by single chain antibodies or other antibody derivatives retaining the antigen specificity and the lower hinge region or a variant thereof. These may be polyfunctional antibodies, recombinant antibodies, ScFv, humanized antibodies, or variants thereof. Therapeutic antibodies are specific for surface antigens, e.g., membrane antigens. Most preferred therapeutic antibodies are specific for tumor antigens (e.g., molecules specifically<br><br><br>
expressed by tumor cells), such as CD20, CD52, ErbB2 (or HER2/Neu), CD33, CD22, CD25,<br>
MUC-l, CEA, KDR  aVB3, etc., particularly lymphoma antigens (e.g., CD20). The therapeutic<br>
tntibodiesare preferably IgGl or IgG3, more preferably IgGl.<br>
Typical examples of therapeutic antibodies of this invention are rituximab, alemtuzumnb and rastuzumab. Such antibodies may be used according to clinical protocols that have been ruthorized for use in human subjects. Additional specific examples of therapeutic antibodies nclude, for instance, epratuzumab, basiliximab, daclizumab, cetuximab, labetuzumab, ;evirumab, tuvurimab, palivizumab, infliximab, omalizumab, eializumab, natalizumab, :lenoliximab, etc., as listed in the following table:<br><br>
Ab specificity	DCI	Commercial name	 Typical Indications<br>
Anti-CD20	rituximab	MabThera®, Rituxan®	LNH B<br>
Anti-CD52	alemtuzumab	CAMPATH-1H®	LLC, allograft<br>
Anti-CD33		Zamyl™	Acute myeloid Leukemia<br>
Anti-HLA-DR		Remitogcn™	LNH B<br>
Anti-CD22	epratuzumab	LymphoCide™	LNH B         <br>
Anti-erbB2 (HER-2/ncu)	trastuzumab	Herceptin®,	Metastatic breast cancer<br>
Anti-EGFR<br>
(HER-1,<br>
erbBl)	cetuximab		ORL and colorectal Cancers<br>
Anti-MUC-1		Therex®	Breast and epithelial cancers<br>
Anti-CEA	labetuzumab	CEA-Cide™	<br>
Anti-aVp3		Vitaxin	Cancers (atiti-angiogenic)<br>
Anti-KDR (VEGFR2) aiiti-VRS fusion protein			Cancers (anti-angiogenic)<br><br>
	palivizumab	Synagis®	Viral diseases<br>
Idem		Numax™	Idem<br>
CMV	sevirumab	Protovir	CMV Infection<br>
HBs	tuvirumab	Oslavir™	Hepatitis B<br>
Anti-CD25	basiliximab	Simulcct®	Prevention/treatment-       allograft rejection          <br>
Anti-CD25	daclizumab	Zenapax®	Prevention/treatment.        allograft<br><br><br><br>
			rejection<br>
anti-TNF-a	inlliximab	Remicade™	C'folu ' disease,     polyarlhrite rhumatoid<br>
anti-lgE	omalizumab	Xolair™	Asthma.<br>
anti-intcgrin aL    (CD 11 a, LFA-1)	efalizumab	Xanelim™	psoriasis<br>
alnti-CD4	keliximab		<br>
anti-CD2	siplizumab		<br>
Anti-CD64			anemia<br>
anli-CD147			GvH<br>
anti-integrin a4(a4B1-a4B7)	natalizumab	Antegren®	Sclerosis, Crohn<br>
Anli-integrin B7			Crohn, RCH<br>
anti-CD4*	clenoliximab		<br>
"Within the context of the present invention, a subject or patient includes any mammalian subject or patient, more preferably a human subject or patient, i<br>
According to the invention the term FCGR3A gene refers to any nucleic acid molecule encoding a FcyRHIa polypeptide in a subject. This term includes, in particular, genomic DNA, cDNA, RNA (pre-rRNA, messenger RNA, etc.), etc. or any synthetic nucleic acid comprising all or part of the sequence thereof. Synthetic nucleic acid includes cDNA, prepared from RNAs, and containing at least a portion of a sequence of the FCGR3A genomic DNA as for example one or more introns or a portion containing one or more mutations. Most preferably, the term FCGR3A gene refers to genomic DNA, cDNA or mRNA, typically genomic DNA or mRNA. The FCGR3A gene is preferably a human FCGRIIIa gene or nucleic acid, i.e., comprises the sequence of a nucleic acid encoding all or part of aFcyFJUa polypeptide having the sequence of human FcyRIlIa polypeptide. Such nucleic acids can be isolated or prepared according to known techniques. For instance, they may be isolated from gene libraries or banks, by hybridization techniques. They can also be genetically or chemically synthesized. The genetic organization of a human FCGRIIIa gene is depicted on Figure 2. The amino acid sequence of human FcyRIHa is represented figure 3. Amino acid position 158 is numbered<br><br><br>
from residue 1 of the mature protein. It corresponds to residue 176 of  the pre-protein having a signal peptide. The sequence of a wild type FCGR3A gene is represented on figure 4 (see also Gcnbank accession Number AL590385 or NM 000569 for partial sequence).<br>
Within the context of this invention, a portion or part means at least 3 nucleotides (e.g., a codon), preferably at least 9 nucleotides, even more preferably at least 15 nucleotides, and can contain as much as 1000 nucleotides. Such a portion can be obtained by any technique well known in the art, e.g., enzymatic and/or chemical cleavage, chemical synthesis or a combination thereof. The sequence of a portion of a FCGR3A gene encoding amino acid position 158 is represented below, for sake of clarity:<br>
cDNA	540       550       560       570      580<br>
genomic DNA	4970      4980      4990      5000<br>
158F allele     tcctacttctgcagggggctttttgggagtaaaaatgtgtcttca SYFCRGLFGSKNVSS<br>
158V allele     tcctacttctgcagggggcttgttgggagtaaaaatgtgtcttca<br>
SYPCRGLVGSK|NVSS<br>
As indicated above, the invention comprises a method of determining in vitro the<br>
FCGR3A158 genotype of said subject. This more particularly comprises determining the<br>
3 nature of amino acid residue present (or encoded) at position 158 of the FcyRIII a polypeptide.<br>
Genotyping the FCGR3A gene or corresponding polypeptide in said subject may be achieved by various techniques, comprising analysing the coding nucleic acid molecules or the encoded polypeptide. Analysis may comprise sequencing, migration, electrophoresis, immuno-techniques, amplifications, specific digestions or hybridisations, etc.<br>
In a particular embodiment, determining amino acid residue at position I58 of Fcy RIIII a receptor comprises a step of sequencing the FCGR3A receptor gene or R'NA or a portion thereof comprising the nucleotides encoding amino acid residue 158.<br><br><br>
In an other particular embodiment, detennining amino acid residue at position 158 of FcyRIII a receptor comprises a step of amplifying the FCGR3A receptor gene or RNA or a portion thereof comprising the nucleotides encoding amino acid residue 158. Amplification may be performed by polymerase chain reaction (PCR), such as simple PCR,RT-PCR or nested PCR, for instance, using conventional methods and primers.<br>
In this regard, amplification primers for use in this invention more preferably contain less than<br>
about 50 nucleotides even more preferably less than 30 nucleotides, typically less than about 25<br>
or 20 nucleotides. Also, preferred primers usually contain at least 5, preferably at least 8<br>
nucleotides, to ensure specificity. The sequence of the primer can be prepared based on the<br>
sequence of the FCGR3A gene, to allow full complementarity therewith, preferably. The probe<br>
may be labelled using any known techniques such as radioactivity, fluorescence, enzymatic,<br>
chemical, etc. This labeling can use for example Phosphor 32, biotin (16-dUTP), digoxygenin<br>
(11-dUTP). It should be understood that the present invention shall not be bound or limited<br>
by particular detection or labelling techniques. The primers may further comprise restriction<br>
sites to introduce allele-specific restriction sites in the amplified nucleic acids, as disclosed<br>
below.	<br>
Specific examples of such amplification primers are, for instance, SEQ ID NO: 1-4.<br>
It should be understood that other primers can be designed by the skilled artisan, such as any fragment of the FCGR3A gene, for use in the amplification step and especially a pair of primers comprising a forward sequence and a reverse sequence wherein said primers of said pair hybridize with a region of a FCGR3A gene and allow amplification of at least a portion of the FCGR3A gene containing codon 158. In a preferred embodiment, each pair of primers comprises at least one primer that is complementary, and overlaps with codon 158, and allows to discriminate between 158V (gtt) and 158F (ttt). The amplification conditions may also be adjusted by the skilled person, based on common general knowledge and the guidance contained in the specification.<br><br><br>
In a particular embodiment, the method of the present invention thus comprises a PCR<br>
i	■;<br>
amplification of a portion of the FCGR3a mRNA or gDNA with specific oligonucleotide<br>
primers, in the cell or in the biological sample, said portion comprising codon 158, and a direct<br>
or indirect analysis of PCR products, e.g., by electrophoresis, particularly Denaturing Gel<br>
 Gradient Electrophoresis (DGGE).	<br>
In an other particular embodiment, determining amino acid residue at position 158 of FcyRHIa receptor comprises a step of allele-specific restriction enzyme digestion. This can be done by using restriction enzymes that cleave the coding sequence of a particular allele (e.g., the 158V allele) and that do not cleave the other allele (e.g., the 158Fallele, or vice versa). Where such allele-specific restriction enzyme sites are not present naturally in the sequence, they may be introduced therein artificially, by amplifying the nucleic acid with a'lclc-specific amplification primers containing such a site in their sequence. Upon amplification, determining the presence of an allele may be carried out by analyzing the digestion products, for instance by electrophoresis. This technique also allows to discriminate subjects that are homozygous or<br>
heterozygous for the selected allele.<br>
i<br>
Examples of allele-specific amplification primers include for instance SEQ ID NO: 3. SEQ ID NO:3 introduces the first 3 nucleotides of the NlaIII site (5'-CATG-3) Cleavage occurs after G. This primer comprises 11 bases that do not hybridise with FCGR3A, that extend the primer in order to facilitate electrophoretic analysis of the amplification products) and 21 bases that hybridise to FCGR3A, except for nucleotide 31 (A) which creates the restriction site.<br>
In a further particular embodiment, determining amino acid residue at position 158 of FcyRHIa receptor comprises a step of hybridization of the FCGR3A receptor gene or RNA or a portion thereof comprising the nucleotides encoding amino acid residue 158, with a nucleic acid probe specific for the genotype Valine or Phenylalanine, and determining the presence or absence of hybrids.<br><br><br>
It should be understood that the above methods can be used either alone or in various combinations. Furthermore, other techniques known to the skilled person may be used as well to determine the FCGR3A158 genotype, such as any method employing amplification (e.g. PCR), specific primers, specific probes, migration, etc., typically quantitative RT-PCR, LCR  (Ligase Chain Reaction), TMA (Transcription Mediated Amplification), PCE: (an enzyme amplified immunoassay) and bDNA (branched DNA signal amplification) assays.<br>
In a preferred embodiment of this invention, determining amino acid residue at. position 158<br>
of FcyRIII a receptor comprises:	<br>
Obtaining genomic DNA from a biological sample,<br>
-	Amplifying the FcyRIII a receptor gene or a portion thereof comprising the<br>
nucleotides encoding amino acid residue 158, and<br>
determining amino acid residue at position 158 of said FcyRIII a receptor gene.<br>
Amplification can be accomplished with any specific technique such as PCR, including nested PCR', using specific primers as described above. In a most preferred embodiment, determining amino acid residue at position 158 is performed by allele-specific restriction enzyme digestion. In that case, the method comprises:<br>
Obtaining genomic DNA from a biological sample,<br>
-	Amplifying the FcyRIIIa receptor gene or a portion thereof comprising the nucleotides encoding amino acid residue 158,<br>
-	Introducing an allele-specific restriction site,<br>
-	Digesting the nucleic acids with the enzyme specific for said restriction site and,	<br>
-	Analysing the digestion products, i.e., by electrophoresis, the presence of digestion products being indicative of the presence of the allele.<br>
In an other particular embodiment, the genotype is determined by.a method comprising: total (or messenger) RNA extraction from cell or biological sample or biological fluid in vitro or ex<br><br><br>
vivo, optionally cDNA'synthesis, (PCR) amplification with FCGR3A-specific oligonucleotide primers, and analysis of PCR products.<br>
The method of this invention may also comprise determining amino acid residue at position 158. of FcyRIIIa receptor directly by sequencing the FcyRIIIa receptor polypeptide or a portion thereof comprising amino acid residue 158 or by using reagcnts'specific for each allele of the FcyRIIIa polypeptide. This can be determined by any suitable technique known to the skilled artisan, including by immuno-assay (ELISA, EIA, RIA, etc.). This can be made using any affinity reagent specific for a FcyRinal58 polypeptide, more preferably any antibody or fragment or derivative thereof. In a particular embodiment, the FcyRIII a.158 polypeptide is detected with an anti- FcyRIIIal58 antibody (or a fragment thereof) that discriminates between FcyRIIIal58V and FcyRIHal58F, more preferably a monoclonal antibody. The antibody (or affinity reagent) may be labelled by any suitable method (radioactivity, fluorescence, enzymatic, chemical, etc.). Alternatively, Fc7RIIIal58 antibody immune complexes may be revealed (and/or quantified) using a second reagent (e.g., antibody), labelled, that binds to the anti-FcyRIIIal58 antibody, for instance.<br>
The above methods are based on the genotyping of FCGR3 A158 in a biological sample of the subject. The biological sample may be any sample containing a FCGR3A gene or corresponding polypeptide, particularly blood, bone marrow, lymph node or a fluid, particularly blood or urine, that contains a FCGR3A158 gene or polypeptide. Furthermore, because the FCGR3A 158 gene is generally present within the cells, tissues or fluids mentioned above, the method of this invention usually uses a sample treatedto reader the gene or polypeptide available for detection or analysis. Treatment may comprise any conventional fixation techniques, cell lysis (mechanical or chemical or physical), or any other conventional method used in imrnunohistology or biology, for instance.<br>
The method is particularly suited to determine the response of a subject to an anti-tumor therapeutic antibody treatment. In this regard, in a particular embodiment, the subject has a tumor and the therapeutic antibody treatment aims at reducing the tumor burden, particularly<br><br><br>
at depleting the tumor cells. More preferably, the tumor is a lymphoma, such as more<br>
preferably a B lymphoma, particularly a NHL. As indicated above, the antibody is preferably<br>
an IgGl or an IgG3, particularly an anti-CD20IgGl or IgG3, further preferably a humanized<br>
antibody, for instance rituximab.	<br>
The invention also relates to a bispecific antibody, wherein said bispecific antibody specifically binds CD16 and a tumor antigen, for instance a CD20 antigen. The invention also encompasses pharmaceutical compositions comprising such a bispecific antibody and a pharmaceutically acceptable excipient or adjuvant.<br>
Further aspects and advantages of this invention will be disclosed in the following examples, which should be regarded as illustrative and not limiting the scope of this application.<br>
FIGURE LEGENDS<br><br>
FIGURE 1: Adjusted KAPLAN-MEIER estimates of progression-free survival after rituximab treatment according to FcyR3a-158V/F' genotype (p = 0,05). FIGURE 2: Genetic organization of the human FCGR3A gene FIGURE 3: Amino acid sequences of human FcyRIIIIa158F (SEQ ID NO:7) FIGURE 4: Nucleic acid sequence of human FCGR3A158F (SEQ I'D NO:8)<br>
MATERIALS AND METHODS<br>
Patients and treatment<br>
Clinical trial design, eligibility criteria and end-point assessment have been previously<br>
reported.5 In brief, patients were eligible for inclusion in this study if they had previously<br>
untreated follicular CD20 positive NHL according to the REAL classification.26 Patients were required to present with stage II to IV disease according to Ann-Arbor classification and at least one measurable disease site. All patients were required to have low tumor burden according to the GELF criteria. A total of four 375 mg/m2 doses of rituximab (Roche,<br><br>
Neuilly, France) were administered by intravenous infusion (clays 1, 8, 15, 22). The<br>
management of infusion and adverse events has already been reported5' The study protocol was approved by an ethics committee, and all patients gave their informed consent.<br>
Monitoring and endpoints<br>
Baseline evaluation included clinical examination, chest X-ray, computed tomography (CT) of the chest, abdomen and pelvis, and unilateral bone marrow biopsy. Response was assessed by an independent panel of radiologists who reviewed all the CT scans of the included patients. The primary efficacy endpoint was the objective response rate, i.e the proportion of patients<br>
achieving either complete remission (CR), unconfirmed CR (CRu) or partial response (PR)<br>
according to the criteria recently proposed by an international expert committee28 Clinical response was evaluated at days 50 and 78. Only the maximum response was taken into account and that assessment time point named M2. All patients were evaluated for progression at one year (M12). Patients in CR or CRu with disappearance of bone marrow infiltration at M2 and  reappearance of lymphoma cells in bone marrow at M12 were considered " progressiA'e "; patients in PR with negative bone marrow biopsy at M2 and positive biopsy at M12 were considered in PR. Molecular analysis of BCL2JH gene rearrangement was performed by PCR, as previously<br>
described,5 on a lymph node obtained at diagnosis and on both peripheral blood and bone     marrow at diagnosis, M2 and M12.<br>
FCGR3A-158V/F genotyping<br>
Out of the 50 patients included in the clinical trial, one patient was excluded after histological review and DNA was not available for two other patients. Forty seven patients were therefore     available for FCGR3A genotype analysis. All samples were analysed in the same laboratory and DNA was extracted using standard procedures including precautions to, avoid cross-contamination. DNA was isolated from peripheral blood (n=43), bone marrow (n-3) or lymph node (n-1). Genotyping of FCGR3A-15SV/F' polymorphism was performed as<br>
described by Koeneetal with a nested PCR followed by an aJlele-specific restriction enzyme    digestion. Briefly, two FCGR3A specific primers (5-ATATTT ACAGAATGGCACAGG-3',<br><br><br>
SEQ ID NO: 1; 5'-GACTTGGTACCCAGGTTG AA-3; SEQ ID NO: 2) (Eurobio, Les Ulis, France) were used to.amplify a 1.2 kb fragment containing the polymorphic site. The PCR assay was performed with 1.25 ug of genomic DNA, 200 ng of each primer, 200 pmol/L of each dNTP (MBI Fermentas, Vilnius, Lithuania) and 1U of Taq DNA polymerase (Promega,   Charbonniere, France) as recommended by the manufacturer. This first PCR consisted in 10 min at 95°C, then 35 cycles (each consisting in 3 steps at 95°C for 1 rhin, 57°C for 1.5 min, 72°C for 1.5 min) and 8 min at 72°C to achieve complete extension. The second PCR used primers (5-ATCAGATTCGATCCTACTTCTGCAGGGGGCAT-3' SEQ ID NO: 3; 5-ACGTGCTGAGCTTGAGTGATGGTGATGTTCAC-3' SEQ ID NO: 4) (Eurobio) amplifying a 94 bp fragment and creating a NlalIIIrestriction site only in the FCCR3A-158V allele. This nested PCR was performed with 1 pL of the amplified DNA, 150 ng of each primer, 200 pmol/L of each dNTP and 1 U of Taq DNA polymerase. The first cycle consisted in 5 min at 95°C then 35 cycles (each consisting in 3 steps at 9.5°C for 1 min, 64°C for 1 min, 72°C for 1 min) and 9.5 min at 72°C to complete extension. The amplified DNA (10 uL)was then digested with 10 U of NaIII (New England Biolabs, Hitchin, England) for 12 h at 37°C<br>
i,<br>
and separated by electrophoresis on a 8% polyacrylamide gel. After staining with ethidium<br>
bromide, DNA bands were visualized with UV light. For homozygous FCGR3A-158F<br>
. patients, only one undigested band (94 bp) was visible. Three bands (94 bp, 61 bp and 33 bp)<br>
were seen in heterozygous individuals whereas for homozygous FCGR3A-\S%V patients, only<br>
    two digested bands (61,bp and 33 bp) were obtained.<br>
FCGR2A-131H/R genotyping<br>
Genotyping of FCGR2A-131FI/R was done by PCR followed by an allele-specific restriction<br>
enzyme digestion according to Liang et al28 The sense primer (5'-GGAAAATCCCAGAAATTCTCGC-3' SEQ ID NO: 5) (Eurobio) has been modified to create a BstUI  restriction site in case of R allele whereas the antisense primer (5'-CAACAGCCTGACTACCTATTACGCGGGO1 SEQ ID NO: G) (Eurobio) has been modified to carry a second BscUl restriction site that served as an internal'control. PCR amplification was performed in a 50 uL reaction with 1.25 ug genomicDN A, 170 ng of each primer, 200 umol/L of each dNTP, 0.5 U of Taq DNA polymerase, and the manufacturer's<br><br>
t<br>
buffer. The first cycle consisted of 3 minutes at 94°C followed by 35 cycles (each consisting in 3 steps at 94°C for 15 seconds, 55°C for 30 seconds, 72°C for 40 seconds) and 7 min at 72°C to complete extension. The amplified DNA (7pL) were then digested with 20 U of BstUl (New England Biolabs) for 12 h at 60°C. Further analysis was performed as described for FCGR3A genotyping. The FCGR2A-13IH and -131R alleles were visualized as a 337 bp and  316 bp DNA fragments, respectively.<br>
Statistical analysis<br>
Clinical and biological characteristics as well as clinical and molecular responses of the patients in the different genotypic groups were compared using a Chi-squared test or by Fisher's exact test when appropriated. A logistic regression analysis including: sex, age (&gt; or  or 
calculated according to the method of Kaplan and Meier29 and was measured from the start<br>
of treatment until progression/relapse or death. Comparison of the progression-free survival<br>
by FCGR3A genotype was performed using the log-rank test. P 
statistically significant.	<br>
RESULTS<br>
Clinical response<br>
Out of the 49 patients tested for the FCGR3A-158 V/F polymorphism', 10 (20%) and 17 (35%) were homozygous for FCGR3A-15SV and FCGR3A-158F, respectively, and 22 (45%) were heterozygous. The three groups were not different in terms of sex, disease stage, bone marrow involvement, number of extra-nodal sites involved or presence of BCL2-JH rearrangement in peripheral blood and bone marrow at diagnosis (Table 1). No difference was found when homozygous FCGR A-158V patients were compared with FCGR3A-158F carriers (FCGR3A-158F homozygous and heterozygous patients) or when homozygous FCGR3A-158F patients were compared with FCGR3A-158V carriers (FCGR3A-158V homozygous and heterozygous<br><br><br>
patients). The objective] response rate at M2 was 100% (CR + CRu-40%), 70% (CR+CRu-29%) andl64%'(CR+CRu-18%) in /CGR3A-158V homozygous, FCGR3A-158F homozygous and heterozygous patients respectively (P-0.09). A:significant difference in objective response rate was observed between FCGR3A-158V homozygous patients and FCGR3A-158F carriers with 67% (CR + CRu=23%) objective response rate for this latter group (relative risk = 1.5; 95% CI, 1.2-1.9; P-0.03) (Table 2). No difference was observed between FCGR3A-158F homozygous patients and FCGR3A-158V carriers. At M12, the objective response rate was 90% (CR + CRu-70%), 59% (CRT+CRu-35%) and 45% (CR + CRu=32%) in FCGR3A-15ZV homozygous, FCGR3A-158F homozygous and heterozygous patients respectively (P-0.06). The difference in objective response rate was still present one year after treatment between FCGR3A-158V homozygous group and FCGR3A-158F carriers with 51% (CR + CRu = 33%) objective response rate for this latter group (relative risk-1.7; 95% CI, 1.2-2.5;p=0.03). The logistic regression analysis showed that the homozygous FCGR3A-158V genotype was the only predictive factor for clinical response both at M2 (P=»0.02) and at M12 (P~0.01). The progression-free survival at 3 years (median follow-up: 35 months; 31-41)(Figure 1) was 56% in FCGR3A-158V homozygous patients and 35% in FCGR3A-158F carriers (ns). Out of the 45 patients analyzed for FCGR2A-131H/R polymorphism, 9 (20%) and 13 (29%) were homozygous for FCGR2A-131R and FCGR2A-131H, respectively, while 23 (51%) were heterozygous. There was no difference in the characteristics at inclusion or clinical response to rituximab treatment for these three groups or for homozygous FCGR2A-131H patients and FCGR2A-131R carriers, or for homozygous FCGR2A-131R patients and FCGR2A-131H carriers (data not shown).<br>
Molecular response<br>
At diagnosis, BCL2-]H rearrangement was detected in both peripheral blood and in bone marrow in 30 (64%) patients, enabling further follow-up. Twenty-five patients (six FCGR3A-158V homozygous patients and 19FCGR3A-158F carriers) and 23 patients (six FCGR3A-158V homozygous patients and 17 FCGR3A-158F carriers) were analysed for BCL2-JH rear angement in both peripheral blood and bone marrow at M2 and at M12 (Table 3). At M2, a cleaning of BCL2-JH rearrangement was observed in 3/6'of the FCGR3A-158V homozygous patients and in 5/19 of the FCGR3A-158F carriers (ns). In contrast, the rate of<br><br><br>
BCL2-JH rearrangement cleaning at M12 was higher (5/6) in the FCGR3A-158V homozygous patients than in the FCGR3A-158F carriers (5/17) (relative risk = 2.8;95% CI, 1.2-6.4; P=0.03). The logistic regression analysis showed that the FCGR3A-158V homozygous genotype was the only factor associated with a greater probability of exhibiting BCL2-JH rearrangement - cleaning at M12 (P-0.04). The single homozygous FCGR3A-158V patient still presenting with DCL2-JHrearrangement in peripheral blood and bone marrow at M12 was in CR 23 months after rituximab treatment. In contrast, the molecular responses at M2 and M12 were not influenced by the FCG.R2A-131H/R polymorphism (data not shown).  <br>
DISCUSSION<br>
Because of the increasing use of rituximab in B cell lymphoproliferative malignancies, enhanced understanding of treatment failures and of the mode of action of rituximab is required. In this regard, we genotypcd FCGR3A  in follicular NHL patients with well-defined<br>
clinical and laboratory characteristics and treated with rituximab alone.5 In particular, all the patients included in this study had a low tumor burden NHL and a molecular analysis of BCL2-JHat diagnosis and during follow-up.The FCGR3A allele frequencies in this population were similar to those of a general Caucasian population.23,24 Our results show an association between the FCGR3A genotype and the response to rituximab. Indeed, homozygous FCGR3A-158V patients, who account for one fifth of the population; had a greater probability of experiencing clinical response, with 100% and 90% objective response rates at M2 and M12, respectively. Moreover, five of the six FCGR3A-158V homozygous patients analysed for BCL2-JH rearrangement showed molecular response at M12, compared to 5 of the 17 .FCGR3A-158F carriers. FCGR3A-158V homozygosity was the only factor associated with the clinical and molecular responses. However, these higher clinical and molecular responses were still unsufficient to significantly improve the progression-free survival in homozygous FCGR3A-158V patients.<br>
This is the first report of an easily assessable genetic predictive factor for both clinical and molecular  responses to rituximab. However, the genetic association does not demonstrate the mode of action of rituximab involves FcyRIIIa. The association observed between FCGR3A genotype and response to rituximab might be due to another genetic polymorphism<br><br><br>
in linkage disequilibrium. Those polymorphisms could be located in FCGR3A itself like the triallelic I'CGR3A-48L/H/R polymorphism31 or in other FcyR-coding genes, since FCGR3A is located on the long arm of chromosome 1, which includes the three FCGR2 genes and<br>
FCGR3B.32 A linkage disequilibrium has been reported between FCGR2A and FCGR3B.33 However, the fact that FCGR2A-131H/R polymorphism was not associated with a better response to rituximab strongly supports the fact that a gene very close to FCGR3A or FCGR3A itself is directly involved.<br>
Several in vitro studies argue in favor of direct involvement -of FCGR3A-158V/F<br>
polymorphism. First, Koeneetal23 shown that the previously reported differences in IgG<br>
binding among the three FcyRllIa-48L/H/R isoforms31 arc a consequence of the linked<br>
FcyRIHa-158V/F polymorphism and several teams have demonstrated that NK cells from<br>
individuals homozygous for the FCGR3A-158V allotype have a higher affinity for human<br>
complexed IgGl and are more cytotoxic towards IgGl-sensitized targets.23,24,34 Our present<br>
results establish that FCGR3A-158V homozygous patients have a better response to<br>
rituximab, which is probably due to a better in vivo binding of that chimeric human IgGl to<br>
FcyRIHa. Secondly, NK cell- and macrophage-mediated ADCC is one of the mechanisms<br>
triggered by anti-CD20 antibodies in vitro 8,11,12 as well as in murine models in vivo,17-19and<br>
rituximab-mediated apoptosis is amplified by FcyR-expressing cells. 15,16 out of all FcyR,<br>
FcyRIII a is the only receptor shared by NK cells and macrophages. We thus postulate that<br>
FCGR3A-1S8V patients show a better response to rituximab because they have better ADCC<br>
activity against lymphoma cells. The fact that more than 50% of the FCGR3A-158F carriers<br>
nonetheless present a clinical response to rituximab could be explained by lower, but still<br>
sufficient, ADCC activity or, more likely, by other mechanism's operating in vivo such as<br>
complement-dependent	cytotoxicity,	complement-dependent        cell-mediated<br>
cytotoxicity11,13,14 and/or apoptosis. 15,16 ADCC could then be viewed as an additional mechanism in the response to rituximab that is particularly effective in FCGR3A-158V homozygous patients.<br>
The vitro studies suggest a" gene-dose " effect with a level of IgGl binding to NK cells from FCGR3A heterozygous donors intermediate between that observed with NK cells<br><br><br>
from FCGR3A-158V and FCGR3A-158F homozygotes23. However, the clinical response of<br>
heterozygous patients appears similar to tht of FCGR3A-1581  homozygous patients, Further<br>
studies with larger groups of patients will be required to conclude against a " gene-dose "<br>
effect in vivo.<br>
	Since FcyRlIIa is strongly associated with a better response to rituxiinab, it needs to be<br>
taken into account in the development of new drugs targetting the CD20 antigen. For example, it may be possible to use engineered rituximab to treat ,FCG3A.-158F-carrier patients with B cell lymphomas. Indeed, by modifying various residues in the IgGl lower hinge region, Shields et al have recently obtained IgGl mutants which bind more strongly to FcyRIIIa-158F than<br>
     native IgGl34<br>
Taken together, these results allow to set up new therapeutic strategies against B<br>
lymphoproliferative disorders based upon prior determination of the patients FCGR3A<br>
genotype. Since this polymorphism has the same distribution in various ethnic population,<br>
including blacks and Japanese, such a strategy may be applied worldwide.23,35,36<br>
Furthermore, such a pharmacogenelic approach may also be applied to other intact humanized<br>
IgGl antibodies used in the treatment of B cell malignancies, such as Campath-IH, or those<br>
used in the treatment of other malignancies, such as trastuzumab' (Herceptin). Even more<br>
generally, this approach may apply to other intact (humanized) therapeutic (IgGl) antibodies<br>
developed to deplete target cells.	<br><br><br>
REFERENCES<br>
1.	Maloncy DG, Lilcs T'M, Czcrwinski DK, ct al,: Phase I clinical trial usingescalating single-dosc infusion of chimeric anti-CD20 monoclonaJ antibody (IDEC-C2B8) in patients with recurrent B-ccll lymphoma. Blood. 1994;84:2457-2466.	<br>
2.	McLaughlin P, Grillo-Lopez. AJ, Link UK, et al.: Riluximab chimeric anti-CD20 monoclonal antibody therapy for relapsed indolent lymphoma: half of patients respond to a four-close treatment program. J Clin Oncol. 1998;16:2825-2833.<br>
3.	Maloney DG, Grillo-Lopez AJ, White CA, et al.: IDEC-C2B8 (Riluximab) anti-CD20 monoclonal antibody therapy in patients with relapsed low-grade non-Hodgkin's lymphoma. Blood. 1997;90:2188-2195.<br>
4.	Hainsworth JD, Burns HA, 3rd, Momssey LH, et al.: Rituiximab monoclonal antibody as initial systemic therapy for patients with low-grade non-Hodgkin lymphoma. Blood. 2000;95:3052-3056.<br>
5.	Colombat P, Salles G, Brousse N, et al.: Rituximab (anti-CD20 monoclonal antibody) as first-line therapy of follicular lymphoma patients with low tumor burden: clinical and molecular evaluation. Blood. 2001;97:101-106.<br>
6.	Coiffier B, Haioun C, KettererN, et a].: Riluximab (anti-CD20monoc.lonal antibody) for the treatment of patients with relapsing or refractory aggressive lymphoma: a multicenter phase II study. Blood. 1998;92:1927-1932.	'<br>
7.	Foran JM, Rohatiner AZ, Cunningham D, et al.: European phase II study of rituximab (chimeric anti-CD20 monoclonaJ antibody) for patients with newly diagnosed mantle-cell lymphoma and previously treated mantle-cell lymphoma, iinmunocyioma, and small B-cell lymphocytic lymphoma. J Clin Oncol. 2000;18:317-324.	;;<br>
8.	Anderson DR, Grillo-Lopez. A, Varns C, Chambers KS, HannaN: Targeted anti-cancer therapy using rituximab, a chimaeric anti-CD20 antibody (IDEC-C2B8) in the treatment of .nqn-Hodgkin's B-cell lymphoma. Biochcm Soc Trans. 1997;25:705-708.<br>
.  9.	VoseJ.LinkB, GrossbardM, et al.: Phase II study of rituximab in combination with CHOP<br>
chemotherapy in patients with prciously untreated intermediate or high-grade non-Hodgkin's lymphoma (NHL).<br>
i<br>
Ann Oncol. 1999;10:58.<br>
10.	Berinstein NL, Grillo-Lopez AJ, White CA, et al.: Association of serum Rituximab (1DEC-C2B8) concentration and anti-tumor response in the treatment of recurrent low-grade or follicular uon-Hodgkin's lymphoma. Ann Oncol. 1998;9:995-1001.<br>
11.	Harjunpaa A, Junnikkala S, Meri S: Rituximab (anti-CD20) therapy of B-cell lymphomas: direct complement killing is superior to cellular effector mechanisms. Scand J Immunol. 2000;51:634-64 1.<br>
12.	Reff ME, Carner K, Chambers KS, et aJ.: Depletion ol B cells 'in vivo by a chimeric mouse human monoclonaJ antibody to CD20. Blood. 1994;83:435-445.<br><br><br><br><br>
28.	Cheson BD, Horning SJ, Coiffier B, et al.: Report of an international workshop to standardize response criteria for non-Hodgkin's lymphomas. NCI Sponsored International Working. J Clin Oncol. 1999;17:1244.<br>
29.	Jiang XM, Arepally G, Poncz M, McKenzie SE: Rapid detection of the Fc y RI1A-H/R 131 ligand-binfling polymorphism using an allele-specific restriction  enzyme digestion (ASRED). J Immunol Methods. 1996;199:55-59.<br>
30.	Kaplan E, Meier P: Nonparametric. estimation from incomplete observations. J Am Stat Assoc. 1958;53:457-481.<br>
31.	de Haas M, Koene HR, KleijerM, et al.: A triallelic Fcy receptor type IIIA polymorphism influences the binding of human IgG by NK cell Fcy RIJI. J Immunol. 1996;156:394X-3955.<br>
32.	Pcltz GA, Grundy HO, Lcbo R V, Ysscl H, Barsh GS, Moore KW: Human Fey RIII cloning, expression, and identification of the chromosomal locus of two Fc receptors forlgG. Proc Natl Acad Sci USA. 1989;86:1013-1017.	<br>
33.	Schnackenbcrg L, Flesch BK.NeppeitJ: Linkage disequilibria between Duffy blood groups, Fc y IIa and Fcy Illb allotypes. Exp Clin Immunogenct. 1997;14:235-242.	<br>
34.	Shields RL, Namenuk AK, Hong K, et al.: High resolution mapping of the binding site on human IgGl for Fc y RI, Fc y R1I, Fc y Rill, and FcRn and design of IgGl variants with improved binding to the. Fcy R.J Biol Chem. 2001;276:6591-6604.	<br>
35.	Leppers-van de Straat FG, van der Pol \V, Jansen MD, et al.: A novel PCR-based method for direct Fey receptor IIIa (CD16) allotyping. J Immunol Methods. 2000;242:127-132.<br>
36.	LehrnbecherT, Foster CB, ZhuS, et al.: Variant genotypes of the low-affinity Fey receptors in two control populations and a review of low-affinity Fcy receptor polymorphisms in control and disease populations. Blood. 1999;94:4220-4232.<br><br><br>
TABLE 1. CHARACTERISTICS OF PATIENTS ACCORDING; 10 THE FCGR3A-158F POLYMORPHISM.<br>
FCGR3A-  FCGR3A- FCGR3A-    p*<br>
158W	158VF	158FF<br><br><br>
10	ns<br>
7	<br>
6	ns<br>
11	<br>
9	ns<br>
8	<br>
13	ns<br>
4	<br>
11	ns<br>
11	ns<br>
10 (20%)      22 (45%)   .  17 (35%)<br>
12<br>
'   10<br>
6<br>
   16<br>
16<br>
    6<br>
   20<br>
   2   12<br>
n (%)<br>
Sex<br>
M F<br>
Disease stage<br>
II-III<br>
BCL2-JH rearrangement in peripheral blood	8<br>
BCL2-JH rearrangement in bone marrow	7<br>
IV Bone marrow involvement yes no Extra-nodal sites involved 2<br><br>
 Satiscical comparisons of the three groups of homozygous FCGR3A-158V patients vs FCGR3A-158F carriers and of homozygous FCGR3A-158F patients against FCGR3AA58V carriers.<br><br><br>
TABLE   2.   CLINICAL   RESPONSE   TO   RITUXIMAB   BY   FCGR3AA58V/F POLYMORPHISM.<br><br>
	FCGR3A-158VV	FCGR3A-158F carriers	p&gt;<br>
Clinical response at M2			<br>
Objective response	10 (100%)	   26 (67%)	0.03<br>
complete remission	3	7	<br>
complete remission unconfirmed	1	2	<br>
partial response	.6	17	<br>
No response	0 (0%)	   13(33%)	<br>
no change	0	10	<br>
progressive disease	0	3	<br><br>
Clinical response at M12<br>
Objective response	9 (90%)<br>
complete remission	6<br>
complete remission unconfirmed	1 -<br>
partial response	2<br>
No response	1 (10%)<br>
no change	0<br>
progressive disease	1<br><br>
20 (51 %)<br>
11<br>
2<br>
7 19(49%)<br>
2<br>
17<br><br>
0.03<br><br>
* Satistical comparison of homozygous FCGR3A-158V patients, against FCGR3A-158F carriers. Data concerning the three genotype subgroups arc given in the text.<br><br><br>
TABLE 3. MOLECULAR RESPONSE TO RITUXIMAB AT M2 AND AT M12 BY THE<br>
i<br>
FCGR3A-158V/F POLYMORPHISM.<br>
FCGR3A-158W      FCGR3A-158F	p<br>
carriers<br>
Molecular response at M2	ns<br>
Cleaning ofBCL2-JH rearrangement<br>
Persistent BCL2-JH rearrangement<br>
Molecular response at M12<br>
Cleanig of BCL2]H rearrangement<br>
Persistent BCL2-JH rearrangement <br><br>
WE CLAIM :<br>
1.	A method of assessing the response of a subject to a therapeutic antibody treatment, comprising determining in vitro the FCGR3A158 genotype of said subject wherein it determines amino acid residue at position 158 of FcyRIIIa receptor, a Valine at position 158 being indicative of a better response to said treatment and a phenylalanine at position 158 being indicative of a lower response to said treatment by sequencing, amplifying, allele-specific restriction enzyme digestion and hybridization.<br>
2.	The method as claimed in claim 1 wherein the step of sequencing the FcyRIIIa receptor gene or RNA or a portion thereof comprising the nucleotides encoding amino acid residue 158.<br>
3.	The method as claimed in claim 1 wherein the step of amplifying the FcyRIIIa receptor gene or RNA or a portion thereof comprising the nucleotides encoding amino acid residue 158.<br>
4.	The method as claimed in claim 1, wherein amplificatin is performed by polymerase chain reaction (PCR), such as PCR, RT-PCR and nested PCR.<br>
5.	The method as claimed in claims 1 to 3, wherein determining amino acid residue at position 158 of FcyRIIIa receptor comprises a step of allele-specific restriction enzymedigestion.<br>
6.	The method as claimed in claim 1, wherein determining amino acid residue at position 158 of FcyRIIIa receptor comprises a step of hybridization of the FcyRIIIa receptor gene or RNA or a portion thereof comprising the nucleotides encoding amino acid residue 158, with a nucleic acid probe specific for the genotype Valine or Phenylalanine.<br>
7.	The method as claimed in claims 1 to 6, wherein determining amino acid residue at position 158 of FcyRIIla receptor comprises;<br>
Obtaining genomic  DNA from a biological sample for in vitro method<br>
assessing;	<br>
Amplifying the FcyREa receptor gene or a portion thereof comprising the<br>
nucleotides encoding amino acid residue 158;<br>
and - determining amino acid residue at position 158 of said FcyRIIIa receptor<br>
gene.<br>
8.	The method as claimed in claims 1 to 7, wherein determining amino acid<br>
residue at position 158 of FcyRIIIa receptor comprises:<br>
Obtaining genomic  DNA from a biological  sample for  in  vitro  method<br>
assessing:     	<br><br>
Amplifying the FcyRIHa receptor gene or a  portion thereof comprising the<br>
nucleotides encoding amino acid residue 158;<br>
Introducing an allele-specific restriction site;     <br>
Digesting the nucleic acids with the enzyme specific for said restriction site;<br>
And Analysing the digestion products, i. e., by electrophoresis, the presence of<br>
digestion   products  being   indicative   of the   presence   of the   allele.<br>
9.	The method of any one of claims I to 9, wherein determining amino acid residue at position 158 of FcyREa receptor comprises: total (or messenger) RNA extraction from cell or biological sample or biological fluid in vitro or ex vivo, optionally cDNA synthesis, (PCR) amplification with specific FCGRIIIa oligonucleotide primers, and analysis of PCR products.<br>
10.	The method as claimed in claims 1 to 4, wherein determining amino acid residue at position 158 of FcyRIIIa receptor comprises a step of sequencing the FcyRIHa receptor polypeptide or a portion thereof comprising amino acid residue 158.<br>
i<br>
11.	The method as claimed in claims 1 to 10, wherein the genomic DNA sample was obtained from human being.<br>
12.	The method of claim 11, wherein the said DNA sample has a tumor, a viral infection, or a disease condition associated with allogenic or pathological immunocompetent cells.<br>
13.	The method of claim 12, wherein the tumor is a lymphoma, particularly a NHL.<br>
14.	The method as claimed in claim 1, wherein the antibody is an IgGI or an IgG3.<br>
15.	The method of claim 14, wherein the antibody is an anti-CD20 antibody, particularly rituximab.<br>
Dated this 15th day of April, 2004.	<br>
HIRAL CHANDRAKANT JOSHI AGENT FOR :ENTRE HOSPITALIER REGIONAL ET UNIVERSITAIRE DE TOURS AND INNATE PHARMA</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtY2FuY2VsbGVkIHBhZ2VzKDAxLTA2LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-cancelled pages(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtY2xhaW1zKGdyYW50ZWQpLSgwMS0wNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-claims(granted)-(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtY2xhaW1zKGdyYW50ZWQpLSgxLTYtMjAwNSkuZG9j" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-claims(granted)-(1-6-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtY29ycmVzcG9uZGVuY2UoMDEtMDYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-correspondence(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtY29ycmVzcG9uZGVuY2UoaXBvKS0oMzAtMDUtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-correspondence(ipo)-(30-05-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZHJhd2luZygwMS0wNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-drawing(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZm9ybSAxKDAxLTA2LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-form 1(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZm9ybSAyKGdyYW50ZWQpLSgwMS0wNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-form 2(granted)-(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZm9ybSAyKGdyYW50ZWQpLSgxLTYtMjAwNSkuZG9j" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-form 2(granted)-(1-6-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZm9ybS1wY3QtaXBlYS00MDkoMDEtMDYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-form-pct-ipea-409(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM0LW11bW5wLTIwMDQtZm9ybS1wY3QtaXNhLTIxMCgwMS0wNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">234-mumnp-2004-form-pct-isa-210(01-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="209869-composite-pressure-sintered-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209871-hydrochloride-salts-of-5-4-2-n-methyl-n-2-pyridyl-amino-ethoxy-benzyl-thiazolidine-2-4-dione.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209870</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>234/MUMNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Oct-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Sep-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Apr-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTRE HOSPITALIER REGIONAL ET UNIVERSITAIRE DE TOURS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2, BOULEVARD TONNELLE F-37044 TOURS CEDEX, FRANCE.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WATIER HERVE</td>
											<td>58 RUE DU MARECHAL FOCH, 37510 BALLAN MIRE, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CARTRON GUILLAUME</td>
											<td>68 ROUTE DU PETIT BOIS, 37510 SAVONNIERES FRANCE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>COLOMBAT PHILIPPE</td>
											<td>ROCHECAVE 37270 LARCAY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP02/11397</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-10-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>01 402718.9</td>
									<td>2001-10-19</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209870-invltro-method-for-evaluation-of-antibody-treatment-response by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:50:52 GMT -->
</html>
