<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209924-a-method-of-alleviating-infestation-of-plant-pests by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:40:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209924:&quot;A METHOD OF ALLEVIATING INFESTATION OF PLANT PESTS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF ALLEVIATING INFESTATION OF PLANT PESTS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of alleviating infestation of plant pests, which method comprises a) identifying a DNA sequence from said pest which is critical for its survival, growth, proliferation, b) cloning said sequence from step a) or a fragment thereof in a suitable vector in an orientation relative to promoter(s) such that said promoter(s) is capable of initiating transcription of said DNA sequence to RNA or dsRNA upon binding of an appropriate transcription factor to said promoter(s), and c) introducing said vector into the plant.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>CHARACTERISATION OF GENE FUNCTION USING DOUBLE STRANDED RNA INHIBITION<br>
The present invention is concerned with characterization or identification of gene function using double stranded RNA inhibition (dsRNAi) and methods of identifying DNA responsible for inducing a specific phenotype in a cell and a method of assigning function to known gene sequences.<br>
It has recently been described in Nature Vol 391, pp.806-811, February 98, that introducing double stranded RNA into a cell results in potent and specific interference with expression of endogenous genes in the cell and which interference is substantially more effective than providing either RNA strand individually as proposed in antisense technology. This specific reduction of the activity of the gene was also found to occur in the nematode worm Caenorhabditis elegans (C. elegans) when the RNA was introduced into the genome or body cavity of the worm.<br>
The present inventors have utilized this technique and applied it further to devise novel and inventive methods of assigning functions to genes or DNA fragments, which have been sequenced in various projects, such as, for example, the human genome project and which have yet to be accorded a particular function and for use in identifying DNA responsible for conferring a particular phenotype.<br>
Therefore, according to a first aspect of the present invention there is provided a method of identifying DNA responsible for conferring a phenotype in a cell which method comprises a) constructing a cDNA or genomic library of the DNA of said cell in an orientation relative to a promoter(s) capable of<br>
promoting transcription of said cDNA or DNA to double stranded (ds) RNA upon binding of an appropriate transcription factor to said promoter(s), b) introducing said library into one or more of said cells comprising said transcription factor, and c) identifying and isolating a desired phenotype of said cell comprising said library and identifying the DNA or cDNA fragment from said library responsible for conferring said phenotype.<br>
In a preferred embodiment of the invention the library may be organised into hierarchical pools as described in more detail in the examples provided, prior to step b) such as to include, for example, gene families.<br>
According to a further aspect of the invention there is also provided a method of assigning function to a known DNA sequence which method comprises a)identifying a homologue(s) of said DNA in a cell, b) isolating the relevant DNA homologue(s) or a fragment thereof from said cell, c) cloning said homologue or fragment into an appropriate vector in an orientation relative to a promoter(s) capable of promoting transcription of dsRNA upon binding of an appropriate transcription factor to said promoters, d) introducing said vector into said cell from step a) comprising said transcription factor, and e) identifying the phenotype of said cell compared to wild type.<br>
In each aspect of the invention, the nucleotide or DNA sequence may either be provided in a sense and an antisense orientation relative to a single promoter which has the properties defined above, or alternatively it may be provided between two identical promoters.  In both embodiments dsRNA is provided from the transcription initiated from the promoter following binding of its appropriate transcription<br>
factor.<br>
The cell according to the invention may be derived from or contained in an organism. Where the cell is contained within an organism, the organism may be adapted to express the appropriate transcription factor. The organism may be any of a plant, animal, fungus or yeast but preferably may be the nematode worm C. elegans, which may be any of a wild type, a nuc-1 or pha-ts mutant of C. elegans or a combination of said mutations. In an alternative embodiment the DNA or cDNA library or the DNA homologue or fragment thereof may, advantageously, be transfected or transformed into a microorganism, such as a bacterial or yeast cell, which may be fed to the organism, which is preferably the nematode worm C. elegans. In this embodiment of the invention the microorganism may be adapted to express the appropriate transcription factor. Preferably, the microorganism is E. coli.<br>
In each aspect of the invention, the DNA library, DNA homologue or DNA fragment may be constructed in a suitable DNA vector which comprises a sequence of nucleotides which encode said transcription factor. Alternatively, said transcription factor is encoded by a further vector.  In an even further alternative, the cell or organism may express or be adapted to express said transcription factor. Preferably, any of the vectors used in the method according to the invention comprises a selectable marker which may be, for example, a nucleotide sequence encoding sup-35 or a fragment thereof.  The nucleotide sequence may be orientated relative to a promoter such that binding of a transcription factor to the promoter initiates transcription of the DNA into double stranded RNA. Figure 10 illustrates the vectors and the orientation of the DNA sequence which enable double stranded RNA<br>
production in C. elegans.  Thus in one embodiment the DNA is located between two promoters on a vector capable of expressing dsRNA upon binding of an appropriate transcription factor to said promoters. Alternatively, the vector comprises two copies of the DNA sequence organised in a sense and antisense orientation relative to the promoter and which marker is selectable when contained in a pha-1 mutant C. elegans. Preferably, the promoters are any of T7, T3 or SP6 promoters and the transcription factor comprises the appropriate polymerase.<br>
Preferably, the selectable marker comprises a nucleotide sequence capable of inhibiting or preventing expression of a gene in said cell and which gene is responsible for conferring a known phenotype. This nucleotide sequence may be part of or identical to said gene conferring said phenotype, and which nucleotide sequence is itself oriented relative to a suitable promoter(s) capable of initiating transcription of double stranded RNA upon binding of an appropriate transcription factor to said promoter(s).  Alternatively, the nucleotide sequence may be a part of or identical to said gene sequence conferring said phenotype, and which nucleotide sequence is such as to permit integration of said suitable or further vector by homologous recombination in the genome of said cell and following said integration said nucleotide sequence is capable of inhibiting expression of said gene sequence conferring said phenotype.  In this embodiment said nucleotide sequence comprises stop codons sufficient to prevent translation of said nucleotide sequence following its integration into said genome.<br>
Compounds can, advantageously, in said method be added to said cell or organism for the purposes of<br>
screening for desired phenotypes, such as for example, resistance or sensitivity to the compound when compared to wild type. The promoters are preferably inducible. The transcription factor may in some embodiments be phage derived, such as for example, a T7 polymerase driven by a phage promoter. However, when C. elegans is utilised a worm specific or tissue specific promoter can be used, such as for example, Iet858, SERCA, UL6, myo-2 or tnyo-3. Preferably, the E. coil strain is an RNAaselll and even more preferably an Rnase negative strain.<br>
A further aspect of the present invention provides a method of generating a transgenic non-human organism comprising an exogenous transcription factor and a transgene comprising a promoter operably linked to DNA fragment which is expressed upon binding of said transcription factor thereto, the method comprising a) providing a first transgenic organism comprising a first construct incorporating DNA encoding an exogenous transcription factor and a second transgenic organism comprising a second construct including at least one promoter operably linked to a desired DNA sequence which is expressed upon binding of the transcription factor of said first transgenic organism thereto b) crossing said first and second transgenic organisms and selecting offspring expressing said desired DNA sequence.  In one embodiment said first and second transgenic organisms are generated by transforming said first and second constructs into respective microorganisms for subsequent feeding to the respective organism. Preferably, said second construct comprises said desired DNA sequence in an orientation relative to said promoter so as to be capable of initiating transcription of said DNA to dsRNA upon binding of<br>
said transcription factor thereto.  In this embodiment said second construct comprises two promoters flanking said desired DNA sequence which promoters can initiate transcription of said DNA sequence to dsRNA upon binding of said transcription factor to said promoters. Alternatively, said DNA sequence is provided in a sense and an antisense orientation relative to said promoter so as to produce dsRNA upon binding of the transcription factor to the promoters.. In each of these embodiments the first and/or second constructs may preferably be provided with a reporter gene operably linked to a promoter which is capable of initiating transcription of said reporter upon binding of said transcription factor thereto.  Preferably, the reporter gene encodes any of Luciferase, Green Fluorescent protein, (3 galactosidase or 3-lactamase. The present invention also includes a method of validating clones identified in yeast two hybrid vector experiments which experiments are well known to those skilled in the art and which experiments were first proposed by Chien et al. (1991) to detect protein - protein interactions. The method according to the invention comprises providing a construct including the DNA encoding a protein identified in a two hybrid vector experiment, which construct is such that said DNA is provided in an orientation relative to one or more promoters capable of promoting transcription of said DNA to double stranded RNA upon binding of an appropriate transcription factor to said promoters, transforming a cell, such as a bacterial cell or alternatively transforming an organism comprising said transcription factor with said constructs and identifying a phenotypic change in said cell or organism, which may be C. elegans or the like, compared to wild type. Preferably, the transcription<br>
factor is inducible in the cell or organism.  Once again the DNA sequence may be located between two promoters or in both a sense and antisense orientation relative to a single promoter, as described above. Preferably, the promoter is a phage polymerase promoter and said transcription factor is a RNA polymerase, and preferably T7 polymerases. Also encompassed with the scope of the present invention are vectors used to transform said cells or organisms and the cells or organisms themselves.<br>
In a further aspect of the present invention there is provided a method of alleviating pest infestation of plants, which method comprises a) identifying a DNA sequence from said pest which is critical either for its survival, growth, proliferation or reproduction, b) cloning said sequence from step a) or a fragment thereof in a suitable vector relative to one or more promoters capable of transcribing said sequence to RNA or dsRNA upon binding of an appropriate transcription factor to said promoters, and c) introducing said vector into the plant.<br>
Thus, advantageously, the method according to the invention provides a particularly selective mechanism for alleviating pest infestation, and in some cases parasitic infestation of plants, such that when the pest feeds on the plant it will digest the expressed dsRNA in the plant thus inhibiting the expression of the DNA within the pest which is critical for its growth, survival, proliferation or reproduction. In a preferred embodiment, the pest may be any of Tylenchulus ssp. Radopholus ssp., Rhadinaphelenchus ssp., Heterodera ssp., Rotylenchulus ssp., Pratylenchus ssp., Belonolaimus ssp., Canjanus ssp., Meloidogyne ssp., Globodera ssp., Nacobbus ssp.,<br>
Ditylenchus ssp., Aphelenchoides sap., Hirschmenniella ssp., Anguina ssp., Hoplolaimus ssp., Heliotylenchus ssp., Criconemellassp., Xiphinemassp., Longidorus ssp., Trichodorus ssp., Paratrichodorus ssp., Aphelenchs ssp. The DNA sequence or fragment thereof according to this aspect of the invention may be cloned between two tissue specific promoters, such as two root specific promoters.<br>
A further aspect of the invention concerns the vector used in each of the methods of the invention for constructing said library, which vector comprises two identical promoters oriented such that they are capable of initiating transcription of DNA sequence located between said promoters to dsRNA upon binding of an appropriate transcription factor to said promoters.  The DNA sequence may, for example, include a multiple cloning site.  Preferably, the expression vector comprises a nucleotide sequence encoding a selectable marker. In one embodiment the nucleotide sequence encoding said selectable marker is located between two identical promoters oriented such that they are capable of initiating transcription of DNA located between said promoters to double stranded RNA upon binding of an appropriate transcription factor to said promoters. Preferably, the selectable marker comprises a nucleotide sequence encoding sup-35, for introduction into C. elegans having a pha-1 mutation.<br>
Preferably, the transcription factor comprises either a phage polymerase which binds to its corresponding promoter or a C. elegans specific promoter and even more preferably T7 polymerase. Preferably, the vector includes a multiple cloning site between said identical promoters.<br>
In a further aspect of the invention there is provided an expression vector for expressing an<br>
appropriate transcription factor for use in a method according to the invention which vector comprises a sequence of nucleotides encoding said transcription factor operably linked to suitable expression control sequences. Preferably, the expression control sequences include promoters which are inducible, constitutive, general or tissue specific promoters, or combinations thereof. Preferably, the transcription factor comprises a phage polymerase, and preferably T7, T3 or SP6, RNA polymerase.<br>
A further aspect of the invention provides a selection system for identifying transformation of a cell or organism with a vector according to the invention which system comprises a vector according to the invention wherein said selectable marker comprises a nucleotide sequence capable of inhibiting or preventing expression of a gene in said cell or organism which gene is responsible for conferring a known phenotype.  Preferably said nucleotide sequence corresponds to a part of or is identical to said gene conferring said known phenotype, and which nucleotide sequence is itself located between two identical promoters capable of initiating transcription of double stranded RNA upon binding of an appropriate transcription factor thereto. Alternatively, the nucleotide sequence comprises a nucleotide sequence which is a part of or identical to said gene sequence which confers a known phenotype on said cell or organism, and which is such that following integration of said vector by homologous recombination in the chromosome of said cell or organism said sequence inhibits expression of said gene sequence conferring said known phenotype. Preferably, according to this embodiment the nucleotide sequence comprises stop codons sufficient<br>
to prevent translation of the nucleotide sequence following integration into said chromosome. Preferably, the known gene sequence comprises a sup-35 gene or a fragment thereof which is selectable by identifying offspring growing at a temperature above 25°C following introduction in a pha-1 et!23ts mutant C. elegans worm.<br>
In a further aspect of the invention provides said known gene sequence comprises a sup-35 gene or a fragment thereof which is selectable by identifying offspring growing at a temperature above 25°C following introduction of said vector in a pha-1 et123ts mutant C. elegans worm. An even further aspect comprises a method of assigning function to a DNA sequence of a multicellular organism which method comprises a) providing i) a construct comprising said DNA fragment cloned between two promoters capable of promoting transcription in said multicellular organism, in a multicellular organism capable of initiating transcription from said promoter; b) identifying the phenotype of said multicellular organism compared to wild type.<br>
The present invention may be more clearly understood by the following examples which are purely exemplary with reference to the accompanying figures, wherein:<br>
Figure 1  is a nucleotide sequence of plasmid PGN1 in accordance with the present invention.<br>
Figure 2  is a nucleotide sequence of plasmid PGN100 in accordance with the present invention.<br>
Figure 3  is a schematic representation of the vectors used and the transformation regime used in the methods according to the present<br>
invention.<br>
Figure 4 is an illustration of an expression vector used in accordance with the invention.<br>
Figure 5 is a schematic illustration of the T7 RNA polymerase expression vectors used for transforming C. elegans.<br>
Figure 6  is an illustration of plasmid PGN1.<br>
Figure 7 is a diagrammatic representation of an enhanced vector for dsRNA inhibition encoding sup-35 dsRNA.<br>
Figure 8 is an illustration of a vector for<br>
integration into the genome of C. elegans.<br>
Figure 9 is an illustration of the position of a DNA sequence(s) relative to a suitable promoter to initiate expression of dsRNA from the DNA sequence(s).<br>
Figure 10	is	a representation of plasmid pGN108.<br>
Figure 11	is	a representation of plasmid pGN105.<br>
Figure 12	is	a representation of plasmid pGN400.<br>
Figure 13	is	a representation of plasmid pGN401.<br>
Figure 14	is	a representation of plasmid pGNHO.<br>
Figure 15 is a representation of plasmid pAS2 with forward and reverse T7/T3/SP6 promoters.<br>
Figure 16 is a representation of plasmid pGAD424 with forward and reverse T7/T3/SP6 promoters.<br>
Figure 17 is a representation of plasmid pAS2-cyh2-HA+, both TV-final.<br>
Figure 18 is a representation of plasmid pGAD424-without-FULL-ICE-BOTH-T7.<br>
Figure 19 (a) is a representation of plasmid pGN205 and(b) is a representation of plasmid pGN207.<br>
Example A: Construction of an ordered and hierarchical pooled cDNA library and applications thereof.<br>
A random ordered and pooled library;<br>
The vector is an E. coli vector harboring two T7 promoters, with a multiple cloning site (MCS) in between. The two promoters are orientated towards each other, and towards the MCS. In the presence of T7 RNA polymerase, expressed in E. coli, C. elegans or any other organism, RNA will be produced, starting from the two T7 promoters. As these are oriented in the opposite sense, both strands of RNA will be produced from the DNA inserted (cloned) into the MCS in between the two promoters which results in the generation of double stranded RNA (dsRNA) upon binding of the T7 RNA polymerase thereto.<br>
A C. elegans cDNA library is constructed in the MCS using standard molecular biological techniques. The library is transformed into E. coli, and the resulting E. coli are grown in culture and stored in 96 multi-well plates. At this stage, plasmid DNA can<br>
be isolated and stored in 96-multi-well plates corresponding to those of the E. coli colonies. Approximately 100,000 colonies are scored. In this way, the library will harbor approximately 5 times the total expressed cDNA variation of C. elegans, which gives the opportunity for low expressed sequences to be present in the library. This will result in approximately 1041 96-well plates. The plates are hierarchical pooled as necessary. For the present pooling of the clones is arranged in a range of 10 to 100. If the hierarchical pooling is per 8 or 12 (numbers are more convenient as 96-well plates have a 8 to 12 grid), this will result in approximately 87 multi-well plates and approximately 8352 wells. If hierarchical pooling is per 96 wells, which is a full plate, this results in approximately 11 plates and approximately 1041 wells. At any stage of the hierarchical pooling, plasmid DNA can be isolated, which would be less elaborate as less plates are used, but will result in a loss of complexity although this should not be the case in the pooling per 12. The pooling of the DNA can also be carried out with the original DNA.<br>
The experiments below describe how the hierarchical pooling should be performed, both for the DNA and for the E. coli library.<br>
An ordered library for RNAi technology, harboring every gene of the C. elegans genome, with applications thereof<br>
As the genome-sequencing project is coming to an end, this information can be used in the application of T7 RNA inhibition technology. Every gene of the C. elegans genome can be cloned using PCR technology. In<br><br>
preference, exons will be cloned with a minimal length of 500 bp. If the exons are too small, smaller fragments will be isolated with PCR, or even parts of introns and neighboring exons will be isolated with PCR technology so that at least a sufficient part of the translated region of the gene is cloned. For this, at least 17000 PCR reactions need to be performed. This collection of PCR products will be cloned in a T7 vector as described (two T7 promoters oriented towards each other with a multiple cloning site in between). Every PCR product is cloned independently, or can be used to generate a random library, analogous to the described cDNA library. If every PCR product is cloned individually, the resulting bacteria and plasmid DNA can be pooled in various ways. Firstly, this collection of individually cloned PCR products in the T7 RNAi vector can be pooled randomly, as described in the random library. This pooling can also be done in a more rational way. For instance, the genes of the C. elegans genome can be analyzed using bioinformatic tools (in silico biology). Various genes of the<br>
genome will belong to a gene family, or will have homologues in the genome. These members of the gene family will be pooled, or the members, being homologues will be pooled. In this way the total number of about 17000 clones is reduced to a more useable quantity. This library can be used to screen for phenotypes in the methods according to the invention. The resulting phenotype gives a functional description to the gene or gene family or gene homologues of the C. elegans genome. As the library consists of a part of every gene in the genome, this method enables description of the full genome in functional-phenotypic terms. For this the double<br>
stranded RNA (dsRNA) needs to be introduced in the worm. This introduction of clones alone, or pooled clones, being random pooling or rational pooling can be achieved in several ways as described.<br>
Example of a vector for the expression of double stranded RNAi<br>
Any vector containing a T7 promoter may be used, and which contains a multiple cloning site (there are many commercially available). Primers containing the T7 promoter and a primer with the reverse complementary strand, both with the appropriate ends are designed. These primers can be hybridized, and if well designed, cloned in the vector of choice. The minimal sequence for a T7 promoter is TAATACGACTCACTATAGGGCGA. Although any vector can be used for the construction of a T7 expression vector there follows an example of how to achieve this with the vector pGEM-3zf(-).<br>
-	Vector pGEM-3zf(+) (PROMEGA) was digested with<br>
HindiII and Sail<br>
-	Primers oGNl and oGN2 were mixed together at a final<br>
concentration of 1 ywg/30 /zl boiled and cooled slowly<br>
to room temperature.<br>
-	The primer was ligated into the vector using<br>
standard ligation procedures. The resulting vector is<br>
pGNl (shown in Figure 1) and contains two T7 promoters<br>
oriented towards each other, and harbors a multiple<br>
cloning site in between.<br>
Sequences of oGNl and oGN2 are:<br>
-	oGNl: AGC TGT AAT ACG ACT CAC TAT AGG GCG AGA AGC TT<br>
-	oGN2: TCG AAA GCT TCT CGC ATA ATA GTG ACT CGT ATT AC<br>
Example of the construction of a library<br>
RNA may be isolated from every organism that is sensitive to RNAi. In general the isolated RNA is then copied into double stranded cDNA, and subsequently prepared in suitable vectors for cloning. Several procedures exist and molecular biology kits can be purchased from various firms including promega, clontech, boehringer Mannheim, BRL, etc which enable:<br>
-	isolation of RNA,<br>
-	eventually polyA RNA can be isolated (several<br>
techniques and kits available)<br>
-	first strand synthesis with AMV reverse<br>
transcriptase, random hexameric primers and/or<br>
oligo (dT) primer<br>
-	second strand synthesis with Rnase H, DNA<br>
Polymerasel,<br>
-	flush ends with T4 DNA Polymerase<br>
-	addition of an adaptor with T4 DNA ligase.<br>
-	eventually treatment with T4 polynucleotide Kinase<br>
-	cloning 'of the cDNA into the vector.<br>
The resulting ligation mixture can be considered as the cDNA library. The ligation contains all cDNA of the procedure ligated into the vector of interest. To order the library, the ligation needs to be transformed into E. coli strains.<br>
Application of this E. coli or DNA library<br>
T7 RNA producing strain:<br>
-	a standard strain is BL21 (DE3): F-ompT[Ion]hsds(r-<br>
m-; and E. coli B strain) A (DE3).  Eventually<br>
variants of BL21 (DE3) can be used, although BL21<br>
 (DE3)pLysS is used.<br>
-	any other E. coli strain which produces the T7 RNA<br>
polymerase, which may be available needs to be<br>
constructed.  This can be generated easily using a<br>
phage, which is commercially available, in this case,<br>
the XCE6 vector (provided by Promega) is used.  Almost<br>
every E. cola strain can be transfected with this<br>
phage and will produce T7 RNA polymerase.<br>
-	a RNAselll mutant E. coli:<br>
Various strains are in principle available, we chose in a first experiment to use strain AB301-105: rna-19, suc-11, bio-3, gdhA2, his95, rnc-105, relAl, spoTl, metBl.  (Kinder et al. 1973 Mol. Gen. Genet 126:53), but other strains may suit better.  This strain is infected with XCE6 and so a T7 producing variant will be constructed.<br>
Wild type C. elegans worms can be grown on the bacteria pools.  The bacteria is expressing the T7 RNA polymerase.  This results in large quantities of dsRNA in the gut of the C. elegans, which will diffuse in the organism and results in the inhibition of expression.  This library can now be used for the screening of several phenotypes.  This technique has the advantage that it is a much faster to detect relevant genes in certain pathways, than the known C. elegans technology.  Moreover, if an interesting phenotype is found, the responsible gene can be cloned easily.<br>
Using the hierarchical pooling one can easily find in a second screen the relevant clone of the pool.  The inserted DNA of this clone can then be sequenced. This experiment results in genetic and biochemical DATA in one step.<br>
Wild type C. elegans strains can be combined with compounds to screen for phenotype, drug resistance and<br>
or drug sensibility.  The C. elegans strain can be a mutant strain, screening for an enhanced phenotype, reduced phenotype, or a new phenotype.  The C. elegans strain can be a mutant strain, and the library screen can be combined with compounds.  So one can screen for drug resistance, drug sensibility, enhanced phenotype, reduced phenotype, or a new phenotype.  The E. coli strain may be any T7 RNA polymerase expressing strain, like BL21 (DE3), for example, but the formation of double strand RNA may be enhanced by using a special E. coli strain that is RNAseIII negative.  RNAseIII recognizes specific loops in dsRNA.  Eventually, an E. coli strain can be used that is deleted in RNAses other than RNAselll or an E, coli can be used that is deleted in one or more RNAses.  The expression of the T7 RNA polymerase in most known E. coli strains and constructs which are available to generate T7 RNA polymerase producing E. coli strains, generally comprise an inducible promoter.  In this way the production of the T7 RNA polymerase is regulated, and thus the production of the dsRNA.  Advantageously, this feature can be used to "pulse" feed the C. elegans worms at specific stages of growth.  The worms are grown on the non-induced E. coli strains.  When the worm has reached the stage of interest, the T7 RNA production in the bacteria is induced.  This allows the studying of the function of any gene at any point in the life cycle of the animal.<br>
Screening the library for homologues of putative interesting human genes, and assign function to these genes<br>
Hundreds of genes have been isolated in various projects, being genomic projects, differential<br>
expressed arrays, hybridization studies, etc.  The described cDNA library can provide a way to validate and or assign function to these genes in a fast and efficient manner.  First of all the worm homologue or homologues or the genes need to be identified by bioinformatic tools (in silico biology).  PCR primers are developed and the cDNA fragment is isolated using PCR technology.  PCR can be performed on the hierarchical pools.  The positive pool or individual wells harboring the bacteria that has the appropriate cDNA is fed to C. elegans and the phenotype is scored.<br>
PCR can be performed on cDNA isolated from C. elegans.  The resulting DNA can be cloned in the T7 vector and transformed in the dsRNA producing E. coli on which the C. elegans worms are then fed.  Depending on which way is faster and more reliable a choice needs to be made.<br>
If the gene belongs to a gene family, the worm may need to be fed on a mixture of bacteria.  Each of them harboring a part of the member of the gene family.  E. coli strains, growth conditions, combinations with compounds can be performed as described above.<br>
If the library rational is used, in which all the genes of C. elegans are cloned in a organized and structured way, the C. elegans homologue and eventually the other homologues, orthologues, and members of the gene family can be traced back easily in the library using a silico biology.  No PCR is involved in this step, and the bacteria and or DNA can be isolated on which the worm will be grown.<br>
Examples<br>
The idea of the series of experiments was to test<br>
both the RNAi vector and the various E. coli strains that were constructed.<br>
1)	Construction of a test plasmid<br>
Any cDNA that gives a clear phenotype in the worm when knocked-out, or used in a RNAi experiment can be used.  It is known that unc-22 is a good candidate, but may other genes are possible.  We opted for a sensitive system that can be used at a later stage. The system was tested with sup-35 in a pha-1 background.  Exon 5 of the sup-35 was isolated by PCR and cloned in the T7 promoter vector pGNl.  The resulting vector was designated pGN2.  pha-1 (e2123) mutant worms cannot produce offspring at temperatures higher than 25°C.  This is due to a developmental problem in embryogenesis.  When sup-35 is knocked-out, or inhibited in this strain, offspring may grow at this temperature.  Combination of pha-1 mutant worms and sup-35 RNAi is a good system to validate the various options.<br>
2)	Testing the RNAi using an E. coli strain<br>
that produces dsRNA.<br>
-	pGN2 was introduced in E. coli strain BL21(DE3) and<br>
T7 RNA polymerase was induced with IPTG. C, elegans<br>
worms (pha-1 (e2123)) were inoculated on this<br>
bacteria, and grown at the restricted temperature of<br>
25°C.  As this mutant is an embryonic mutant at this<br>
temperature, no offspring will be observed.  If the<br>
sup-35 gene is efficiently inhibited by the dsRNA<br>
present in the E. coli, offspring will be observed.<br>
-	pGN2 was introduced in E. coli strain AB301-105(DE3)<br>
and T7 RNA polymerase was induced with IPTG.  C.<br>
elegans worms (pha-1 (e2123)) were inoculated on this<br>
bacteria, and grown at the restricted temperature of<br>
25°C.  As this mutant is an embryonic mutant at this temperature, no offspring will be observed.  If the sup-35 gene is efficiently inhibited by the dsRNA present in the E. coli, offspring will be observed.<br>
3)   Improving the worm strain for better uptake of dsRNA.<br>
Before plating the pha-1 C. elegans on the E. coli strain that produce the double stranded sup-35 RNA.  The worm was mutagenised with EMS (Methane sulfonic Acid Ethyl).  The offspring of this mutagenised worm is then plated on the bacteria.  The worm that feed on this bacteria give larger offspring which has a mutation that results in an improvement of dsRNA uptake, and can be used for further experiments.<br>
Stable integration of the dsRNA producing vector into the genome of the T7 RNA polymerase producing worm<br>
An E. coli vector can be constructed harboring the following features; Two T7 promoters directed towards each other, with a restriction site or a multiple cloning site in between.  Furthermore, the vector may contain the C. elegans sup35 genomic DNA, engineered in such a way that it contains several stopcodons at various intervals, so that no full length protein can be expressed form the sup35 genomic DNA fragment as illustrated in Figure 8.  Any cDNA or cDNA fragment can be cloned in the multiple cloning site between the two T7 promoters.  When this vector is introduced in a C. elegans strain which expresses T7 RNA polymerase, the cDNA or DNA fragment cloned between the two T7 promoters will be transcribed, generating dsRNA from the cloned fragment.<br>
The vector is designed to be used in pha-1 (e2123) mutant worms expressing T7 RNA polymerase. The expression of the T7 RNA polymerase may be constitutive or regulated, general or tissue specific. These pha-1 (e2123) worms cannot produce offspring at temperatures higher than 25°C, which is due to a development problem in embryogenesis. When sup-35 is inhibited or knocked-out in this stain, offspring may grow at this temperature.<br>
When the vector is introduced in the worm, the vector may integrate by homologous recombination (Campbell-like integration). It has been shown that homologous recombination occurs in C. elegans, although at low frequencies (Plasterk and Groenen, EMBO J. 11:287-290, 1992). Homologous recombination at the sup35 gene will result in a knock-out of the gene as the two resulting sup-35 genes will harbor the stopcodons. The resulting worm, and its offspring, if this recombination happens in the eggs, will have a copy of the vector integrated in the genome. This can be selected as only the worms for which the sup-35 has been knocked-out will have offspring at temperatures higher than 25°C. Furthermore, the resulting worm will stably produce double stranded RNA from the DNA fragment cloned between the two T7 promoters. This worm can now be considered as a stable transgenic worm strain with a reduction of function of the gene, from which a fragment has been cloned between the two T7 promoters.<br>
The DNA may be delivered to the worm by several techniques, including injection, ballistic transformation, soaking in the DNA solution, feeding with bacteria. New and other methods that increase the transformation efficiencies can be considered.<br>
The target C. elegans strain may in addition,<br>
have other mutations than the pha-1 (e2123) mutation, and may express other genes than T7 RNA polytnerase.<br>
EXAMPLE B: a Yeast two-hybrid-RNAi vector<br>
A yeast two hybrid vector can be constructed harboring the two T7 promoters. The vectors can be designed to replicate both in yeast and in E. coil. In general cDNA libraries for the yeast two hybrid system are made in the Gal4 or LexA vectors. The library is constructed in vectors having the activation domain of one of these genes. A vector can be constructed that can still perform in the yeast two hybrid screen but which also contains two T7 promoters orientated towards each other, with a cloning site therein between. The order of the sequences in the plasmid will then be "plasmid backbone, (GAL4-T7) , MCS, T7, backbone".  A C. elegans cDNA library constructed in this vector can be used as a standard yeast two hybrid library in an experiment to isolate interacting proteins with a given protein. Once a clone is isolated, the plasmid can be introduced in an E. coil strain expressing the T7 RNA polymerase, and hence will produce dsRNA of the cloned fragment. The bacteria producing this dsRNA can be fed to the worm and phenotypes can be scored. As in the previous example, this validation procedure for a newly isolated yeast two hybrid clone is remarkably shorter than the standard procedure, which requires PCR and/or cloning steps, RNA experiments and/or knock-out experiments. In most cases isolated clones are sequenced first, and on the basis of the sequence, a decision is made to continue with further experiments. In the present invention every isolated clone can easily be introduced into the appropriate E.<br>
cola and fed to the worm. Validation is then performed by phenotype analysis.<br>
To apply this procedure a yeast two hybrid was performed using a known gene as bait and the newly constructed library as the target. Proteins coded by the clones in the target that interact with the bait protein, will result in positive yeast clones expressing the reporter molecule such as can be observed by LacZ staining with X-gal. The plasmid coding for the target protein is isolated directly from the yeast strain and introduced in E. coli. The E. coli is T7 RNA polymerase producing E. coli. In this case, double stranded RNA is produced from the DNA cloned in the multiple cloning site of the vector. When this dsRNA is fed to the worm using the methods described previously, the gene has inhibited in the worm, resulting in a particular phenotype.<br>
-	This yeast two hybrid vector can advantageously be<br>
used to construct an ordered and hierarchically pooled<br>
library as described in the previous example.<br>
-	A yeast strain can also be constructed that<br>
conditionally produces T7 RNA polymerase. After yeast<br>
two hybrid experiments, the expression of the T7<br>
polymerase could be induced, resulting in the<br>
production of dsRNA in the yeast cell. Consequently<br>
the yeast could be fed to the worm. Evidence is<br>
available showing that the C. elegans worms can feed<br>
on yeast.<br>
Construction of a T7 RNA polymerase producing strain, and applications thereof<br>
A C. elegans strain can be constructed that expresses T7 RNA polymerase. The expression can be general and constitutive, but could also be regulated<br>
under a tissue specific promoter, an inducible promoter, or a temporal promoter or a promoter that harbors one of these characteristics or combination of characteristics. DNA can be introduced in this C. elegans strain. This is done either by injection, by shooting with particles, by electroporation or as aforementioned by feeding. If the DNA is a plasmid as described in the previous examples, i.e. a plasmid harboring a cloned cDNA fragment or a PCR fragment between two flanking T7 promoters, then dsRNA of this cDNA or PCR fragment is formed in the cell or whole organism resulting in down regulation of the corresponding gene. The introduced DNA can have an efficient transient down regulation. The introduced DNA can form an extrachromosomal array, which array might result in a more catalytic knock-out or reduction of function phenotype. The plasmid might also integrate into the genome of the organism, resulting in the same catalytic knock out or reduction of function phenotype, but which is stably transmittable.<br>
-	Plasmid DNA harboring a cDNA or a part of a cDNA or<br>
an EST or an PCR fragment of C. elegans cloned between<br>
two T7 promoters as described in Examples A) and B)<br>
can be introduced in the T7 RNA polymerase worm, by<br>
standard techniques. Phenotypes can be analysed -DNA<br>
from an ordered and pooled library as in Example A)<br>
can be introduced in the T7 RNA polymerase worm, by<br>
standard techniques (injection, shooting) . Phenotypes<br>
can be analysed. With the hierarchical pool, the<br>
original clone can be found easily.<br>
-	The same procedure can be performed with a mutant<br>
worm expressing the T7 RNA polymerase. Screening for<br>
enhanced, reduced or new phenotypes.<br>
-	The procedure can be used to enable screening of<br><br>
compounds. Screening with either a wild-type strain or a mutant strain for enhanced or new phenotypes.<br>
-	The DNA could be introduced in the worm by new<br>
methods. One of which is the delivery of DNA by E.<br>
coli. In this case the hierarchical pooled library is<br>
fed to the animal. To prevent digestion of the E. coli<br>
DNA in the gut of the nematode, preferentially a DNAse<br>
deficient C. elegans will be used, such as nuc-1<br>
(e1392). This procedure would be one of the most<br>
interesting as it would be independent of<br>
transformation efficiencies of other techniques, and<br>
generally faster and less labourious.<br>
2) Putative enhancements of the method.<br>
-	A vector is designed, so that it harbors the sup-35<br>
cDNA or a part of this cDNA, cloned in between two T7<br>
promoters. The rest of the vector is as described in<br>
Examples A) and B). This vector can be introduced into<br>
a pha-lts mutant C. elegans. A temperature selection<br>
system exists in this case and only those worms which<br>
have taken up the DNA and express the double stranded<br>
sup-35 RNA will survive at restricted temperatures.<br>
The hierarchical pooled library can be delivered by<br>
any method described above.<br>
-	The vector can be used to construct a library that<br>
is introduced in a T7 RNA polymerase expressing E.<br>
coli. In this case we have an analogous screening as<br>
in part A)with an additional screening for worms where<br>
the dsRNA of sup-35 is active.<br>
-	The DNA and or dsRNA of sup-35 could be delivered on<br>
a different plasmid. For the feeding, both DNA feeding<br>
(Example C) or dsRNA feeding Example A) and B), this<br>
means that the two plasmids could be present in one<br>
bacterium, or that the worm is fed on a mixture of<br>
bacteria, one of which harbors the sup-35 construct.<br>
Example of the construction of a T7 RNA producing C. elegans<br>
To produce T7 RNA polymerase in the worm, several possibilities are possible. The T7 polymerase can be expressed under various promoters, being inducible promoters, constitutive promoters, general promoters and tissue (cell) specific promoters, or combinations of those. Examples of these promoters are the heatshock promoter hsp-16, the gut promoter ges 1, the promoter from cet858, but also the promoter of dpy 7 and the promoter element GATA1. In this example the T7 RNA polymerase is expressed under the control of the hsp-16 promoter that is available in the pPD49.78 vector. The T7 RNA polymerase is isolated as a PCR product using the primers of GN3 an GN4.<br>
The resulting PCR product is digested with Nhel and Ncol, as is the vector in which we want to clone, being the Fire vector pPD49.78. The resulting vector is pGNlOO illustrated in Figure 2 oGN3: CAT GGC AGG ATG AAC ACG ATT AAC ATC GC OGN4: ATG GCC CCA TGG TTA CGG GAA CGC GAA GTC CG pGNlOO is included.<br>
The vector is introduced into the worm using standard techniques, such as micro injection, for example.<br>
The following strains were then constructed:<br>
-Wild-type (pGNlOO) -nuc-1 (e!392) (pGNlOO) -pha-1 (e2123) (pGNlOO) -pha-1; nuc-1  (pGNlOO)<br>
All of these strains are able to produce T7 RNA polymerase when temperature induced or alternatively<br>
by metals such as application of heavy cadmium or<br>
mercury. The procedure for temperature induction is to<br>
shift the animal to a temperature of 30-33°C for at<br>
least one hour, then the animal can be shifted back to<br>
standard<br>
temperatures (15-25°C).<br>
The wild type strain producing T7 RNA polymerase can be used for the production of any RNA in the worm. More specifically, the plasmids from the described libraries can be introduced in these worms, and phenotypes can be scored.<br>
The nuc-1 mutant worm will be used to introduce DNA via bacteria on which the worm feed. As the nuc-1 worm does not digest the DNA, the plasmid DNA can cross the gut wall. If taken up by the cells that produce the T7 RNA polymerase, dsRNA will be produced thus inhibiting the gene from which the RNA was transcribed.<br>
The pha-1 mutant strain that produced T7 RNA polymerase can be used to enhance the procedures as described above.  DNA can be introduced by shooting, micro injection or feeding. More specifically this strain can be used for the vectors that produce dsRNA from sup-35 and from the gene of interest, the latter can be a PCR product, a cDNA, or a library as described.<br>
The pha-1; nuc-1 mutant producing T7 RNA polymerase can be used for the bacterial delivery of the DNA. DNA will preferentially be the plasmid that produce dsRNA from both sup-35 and the gene of interest. The worm strain will preferentially produce the T7 RNA polymerase in the gut. Delivery will preferentially happen by feeding the worm on bacteria harboring the plasmid.<br>
Application of the RNAi technology in plants<br>
Nematodes are responsible a large part of the damage inflicted on plants and more particularly to plants used in the agricultural industry. The RNAi procedures according to the invention can be applied to plants to prevent these parasitic nematodes from feeding longer. In a first step, a DNA fragment is isolated from the parasitic plant nematode that is critical for the animals survival or growth, or to feed or to proliferate. Any gene from which the expression is essential is suitable for this purpose.<br>
A part of this gene, an exon or cDNA is cloned. This DNA fragment can be cloned under the influence of a tissue specific promoter preferably a root specific promoter even more preferably between two root specific promoters. The DNA of the cloned gene under the control of the root specific promoter can be introduced in the plant of interest, using plant transgenic technology. For every parasitic nematode, a different piece of DNA may be required and likewise for every plant race, a different promoter will be needed.<br>
The root will produce RNA or dsRNA from the introduced piece of DNA when root specific promoter is utilised. As the nematode feeds on the plant, the RNA and/or dsRNA will be consumed or ingested by the nematode. The RNA and/or dsRNA can enter the cells of the nematode and perform its inhibitory action on the target DNA. Depending on the nature of the cloned DNA piece of worm, the nematode will not be able to survive, to eat, proliferate, etc in any case preventing the animal of feeding longer on the plant, and thus protecting the plant.<br>
Construction of a T7 RNA polymerase producing C. elegans<br>
To produce a T7 RNA polymerase or other RNA polymerases in animals, and more particularly in nematodes and most particularly in C. elegans, several possibilities can be envisaged. The T7 RNA polymerase can be expressed under various promoters. These promoters may be inducible promoters, constitutive promoters, general promoters, tissue specific promoters, or combinations of those.<br>
Examplel;<br>
Construction of a basic vector for expression of<br>
T7 polvmerase in C. elegans<br>
The T7 polymerase coding sequence was PCR amplified from_X CE6 (Novagen, Madison, USA)using the primers oGN26 (ATGGAATTCTTACGCGAACGCGAAGTCCG) and oGN46(CTCACCGGTAATGAACACGATTAACATCGC), using standard procedures (PCR, A practical A practical approach, 1993, Ed. J. McPherson, et al, IRL Press). The resulting DNA fragment encoding for the T7 RNA polymerase was digested with Agel and EcoRI and inserted into the Fire vector pPD97.82 digested with Agel and EcoRI. The resulting construct encodes for an open reading frame of T7 RNA polymerase in fusion with the SV40 large T antigen nuclear localization signal (NLS) with amino acid sequence MTAPKKKRKVPV. This nuclear localization signal sequence is required to translocate the T7 RNA polymerase from the cytoplasm to the nucleus, where it is able to bind to its specific promoters, designated T7 promoters. Upstream of the coding sequence for the T7polymerasefusion protein is a minimal promoter (myo-2) preceded by a<br>
multiple cloning site (MCS) in which several C. elegans promoters can be inserted. This plasmid (pGNlO5 shown in Figure 11) is a basic T7 RNA polymerase plasmid which enables the expression of T7polymerase in C. elegans. Derivatives of this plasmid wherein promoters are cloned into the multiple cloning site, allow for the inducible, constitutive, general and tissue specific expression of T7 RNA polymerase in C. elegans, as expression will be regulated by the promoter cloned in the multiple cloning site.<br>
Although not restricted to these examples, for the following promoters it is known that they induce expression in the following tissues. let-858 (ubiquitous expression), myo-2 (pharynx expression), myo-3 (body wall muscles), egl-15 (vulval muscles), unc-119 (pan-neuron).<br>
Example 2:<br>
Construction of a vector for expression of T7 RNA<br>
polymerase in C. elegans muscle tissue.<br>
The T7 RNA polymerase coding sequence was PCR amplified from_X CE6 using the primers oGN43 (GCCACCGGTGCGAGCTCATGAACACGATTAACATCGC) and oGN44 (CACTAGTGGGCCCTTACGCGAACGCGAAGTCCG) digested with Agel/Spel and inserted in the pGK13 vector digested with Agel/Spel. (This vector contains the strong SERCA promoter which drives expression in the pharynx, the vulval muscle, the tail and the body wall muscle). A nuclear localization signal (NLS) of SV40 large T antigen was inserted in front of the T7 polymerase coding sequence by insertion of two overlapping oligo's OGN45 (CCGGATGACTGCTCCAAAGAAGAAGCGTAAGCT) and OGN46 (CTCACCGGTAATGAACACGATTAACATCGC) into the<br>
SacI/Agel restriction sites. The resulting construct was called pGN108 as shown in Figure 10. Introduction of this plasmid into C. elegans results in the expression of T7 RNA polymerase in the pharynx, vulva muscle, tail and body wall muscles.<br>
To test expression and functionality of T7 RNA polymerase in C. elegans under the regulation of the SERCA promoter,  pGNlO8, which encodes the T7RNA polymerase under the control of the SERCA promoter was injected into C. elegans. A test vector was co-injected. This test vector encodes for GFP under the control of a T7 promoter (pGN401 in Figure 13).  The plasmid pGN401 was constructed by inserting two overlapping oligo's oGN41 (CCCGGGATTAATACGACTCACTATA) and OGN42 (CCGGTATAGTGAGTCGTATTAATCCCGGGAGCT) in the SacI/Agel opened Fire vector pPD97.82. generating a T7 promoter. Furthermore a selection marker was co-injected to select for transformants (ro!6, pRF4). The latter selection vector pRF4 is well known to person skilled in the art. Transgenic Fl could easy be isolated as they display the rol 6 phenotype. These transgenic C. elegans all expressed GFP in the pharynx, the vulval muscle, the tail and the body wall muscle. This data show clearly that theT7 RNA polymerase is functionally expressed under the regulation of the SERCA promoter, and that the expressed T7 RNA polymerase binds to the T7 promoter present in pGN401 and initiates transcription of the GFP gene, which is then functionally expressed, resulting in fluorescence in the muscle tissues where SERCA is inducing the expression of the T7 RNA polymerase.<br>
Example 3:<br>
Construction of a vector for ubiquitous<br>
expression of T7 polymerase in C. eleaans<br>
The NLS-T7 RNA polymerase fusion gene was isolated from pGN108 with XmaI/Bspl201 and cloned into the Fire vector pPD103.05 digested with XmaI/Bspl20I.This results in a vector wherein the T7 RNA polymerase is cloned under the regulation of the 1et858 promoter. This specific promoter enables the expression of T7 RNA polymerase in all tissues. The resulting plasmid was named pGN110 (Figure 14) .<br>
Example 4;<br>
Construction of a vector for T7 RNA polymerase mediated expression of DNA fragments, genes, and cDNA's under the control of a T7 promoter.<br>
The Fire vector pPD97.82 was digested with SacI / Agel and a T7 promoter sequence was generated by insertion of two overlapping oligo's oGN41<br>
(CCCGGGATTAATACGACTCACTATA) and oGN42<br>
(CCGGTATAGTGAGTCGTATTAATCCCGGGAGCT) into the Sad/Age/ restriction endonuclease sites. This construct (pGN400 Figure 12) contains a GFP open reading frame cloned between SacI and EcoRI restriction endonuclease sites under the regulation of the T7 promoter. Any gene, cDNA, or DNA fragment can be cloned in this vector by deleting the GFP gene as a Agel/ SacI fragment and cloning the DNA fragment of interest into the vector. Preferentially the DNA fragment of interest can be obtained by PCR amplification, inserting the SacI/Afel sites in the primers. The resulting DNA fragment after PCR amplification is the digested and the GFP gene in pGN400 is replaced by the amplified DNA fragment.<br>
Every vector that contains a T7 promoter could be used for the purpose of T7 RNA polymerase induced expression in C. elegans, such as the commercially available pGEM vectors and the pBluescript vectors. This is clearly shown by the pGN401 vector which expresses GFP under the regulation of the T7 promoter in a transgenic C. elegans which expresses T7 RNA polymerase.<br>
The use of pGN400 has the advantage that the vector includes a 3'UTR fragment from unc-54 which enhances the transcription or stability of the RNA.<br>
Generation of permanent, tissue specific  "pseudo knock-out" RNAi C. elegans lines<br>
At present, gene knock outs in C. elegans are obtained after random, large scale mutagenesis and PCR base sib-selection. This method is bulky, very time consuming and tedious. It has been described that introducing double stranded RNA into a cell results in potent and specific interference of expression of endogenous genes. In C. elegans gene expression can be down regulated by injection of RNA into the body cavity of the worm, soaking the worm in a solution containing dsRNA or feeding E. coli that express dsRNA corresponding to the gene of interest. C. elegans cells have the ability to take in dsRNA from their extracellular environment. It has been reported that mRNA is the target of this ds RNA mediated genetic interference (Montgomery and Fire 1998). It is also suggested that the targeted RNA is degraded in the nucleus before translation can occur. Although the RNAi mediated reduction of gene expression can be passed on to the next generations, heritability is poor and the effect is rapidly lost during further<br>
offspring. This is probably due to a continued decrease of the dsRNA pool. We propose here a method to construct C. elegans lines with a permanent, inheritable, RNAi phenotype. The method encompasses the generation of transgenic C. elegans lines by introducing plasmids containing cDNA fragments of the target gene in the sense and antisense orientation under control of a worm promoter or by transcription of an inverted repeat of the cDNA from a single construct. Alternatively, ds RNA can be transcribed from a vector harboring a cDNA flanked by two T7 promoters in a C. elegans strain that expresses T7 polymerase. The result is a transgenic worm with an heritable stable "pseudo knock-out" phenotype. The expression of the cDNA or the T7 polymerase can be general and constitutive but could also be regulated under a tissue specific promoter. In contrast to RNAi induced by external ds RNAi (injected, soaked or feeded) this method would enable to obtain conditional, tissue specific inhibition of gene expression.<br>
Inhibition of unc-22 expression by RNA interference results in a "twitching" phenotype.<br>
Unc 22 cDNA (exon 22) was cloned in sense and antisense orientation in pPD103.05. (A. Fire nr L2865) containing the let 858 promoter that is capable of expressing RNA sequences in all tissues. The resulting plasmids were named pGN205 (Figure 19a) and pGN207 (Figure 19 b). These constructs were introduced into C. elegans together with a selectable marker (rol-6; GFP). Transgenic Fl individuals (expressing rol-6 or GFP) showed a "twitching" phenotype indicating that RNAi could be mediated by endogenous transcription of<br>
RNA from transgenic DNA. The RNAi phenotype co-segregated with the selectable marker into further offspring. This resulted in the generation of C. elegans lines with permanent RNAi phenotype.<br>
Generation, of stable lines T7 RNA polymerase lines and generation of dual transgenic worms.<br>
An expression system in C. elegans based on an exogenous RNA polymerase demands two plasmids. One is encoded for the RNA polymerase under the control of a specific promoter, while the other plasmid encodes for the DNA fragment to be expressed, under the regulation of the T7 promoter. In the case of semi stable RNAi also designated pseudo stable knockouts, the DNA of interest is cloned between two T7 promoters so that dsRNA can be produced.<br>
As the T7 RNA polymerase expression system is known to be a high expression system this will result in problems to generate dually transgenic animals. If the gene to be expressed in the C. elegans nematode is toxic, this will result in lethal effects and hence in the construction of a C. elegans without highly regulated stable expression of the gene of interest. If the gene of interest is essential for the survival of the organism, RNAi with a DNA fragment from this gene will also result in lethal effects, so that pseudo- stable knockouts are not possible.<br>
To overcome this problem the present inventors have designed a system consisting of two transgenic animals. The first animal is transgenic for the T7 RNA polymerase, This T7 RNA polymerase can be expressed in all cells or specific cells or tissues as has been shown in previous examples. The second transgenic animal is transgenic for the DNA fragment of interest.<br><br>
This can be a gene or cDNA linked to a. T7 promoter, or if one wants to perform RNAi a DMA fragment of such gene cloned between two T7 promoters.<br>
Both transgenic animals are viable and do not show any aberrant phenotypes. This is because the T7RNA polymerase expressed in the first tranagenic organism is not toxic for the organism, even if expressed at relative high levels. In the second transgenic organism, the gene of interest is not expressed or the dsRNA is not produced as these transgenic animals do not contain the T7 RNA polymerase.<br>
Expression of the gene or cDNA of interest or RNAi with a DNA fragment can now be obtained by mating the two transgenic animals. The offspring of these are dually transgenic and express the gene of interest or express dsRNa of the DNA fragment of interest. To generate sufficient males in such a mating, one of the transgenic animals males can be a C. elegans mutant with a phenotype favouring generation of males. An Example of such a mutant is him-5.  Preferentially such a mutant will be used to make a C. elegans transgenic for T7 RNA polymerase, while the hermaphrodite harbors the DNA fragment under the regulation of the T7 promoter.<br>
To select efficiently for the dual transgenic offspring a second transgene can be introduced in the second transgenic animal. This transgene contains a reporter gene under the regulation of the T7 promoter. The reporter gene can be GFP, luciferase, Beta galmactosidase, or beta-lactamase. an example of such a ttransgene are the vectors pGN400 and pGN401.<br>
To obtain inducible, tissue specific expression of a transgene in C. elegans we can make male stock (i.e. him-5) carrying the T7 polymerase construct<br>
under the control of different C. elegans promoters that enable tissue specific expression such as). This males can be crossed with hermaphrodites carrying the gene of interest under the control of a T7 promoter.<br>
Furthermore, the transgenes can be integrated into the genome of the animal. Methods to generate stable integration of a plasmid into the genome of the animal have been described (Methods in cell biology, Vol. 48, 1995, ed. by epstein and Shakes, academic press) and involve radiation of the animal. This can be done for both animals animals, but preferentially, the animals expressing the T7 RNA polymerase are subject to such traetment. This result in a collection of C. elegans nematodes that stably express T7 RNA polymerase under the control of various promoters, examples of such promoters are the myo-2 (pharynx expression), myo-3 (body wall muscles), egl-15 (vulval muscles), unc-119 (pan-neuron), SERCA (muscles), Iet858 (all cells) ges-1 (gut).<br>
Construction of RNAi T7 promoter yeast two hybrid vectors<br>
pGAD424 with forward and reverse T7/T3 and or Sp6 In most two-hybrid experiments a cDNA library is cloned in plasmid pGAD424 (Figure 16) which has been engineered with additional restriction sites in the polylinker such as a Ncol site (Clontech). This library allows for screening of binding proteins in a yeast two hybrid experiment.  We constructed a new yeast two hybrid vector with the same possibilities to perform yeast two hybrid, but which contain two addditional T7 promoters, so that the vector can be used for T7 RNA polymerase induced pseudo-stable knock-outs. For this we inserted a forward T7 by using<br>
a T7- linker (consisting of the following primers aattcttaatacgactcactatagggcc and<br>
catgggccctatagtgagtcgtattaag) into the EcoRI-Ncol site of pGAD424. The resulting vector was designated pGAD424-without-FULL-ICE-both-T7. Care was taken to eliminate stop codons and using maximal polylinker compatible amino acids. We adopted the same strategy for the reverse T7 (consisting of both primers gatccgtcgacagatctccctatagtgagtcgtattactgca and gtaatacgactcactatagggagatctgtcgacg) with BamHl and Pstl. To avoid loss of SalI,  we included this site in the primer.<br>
The Sail site is important as most libraries are cloned in this site, adapters are available. This makes the newly constructed vector compatible with existing vectors.<br>
pAS2 with with forward and reverse T7/T3 and or Sp6<br>
An analogous yeast two hybrid vector was constructed based on pAS2 (Clontech). By partial EcoRV digestion we were able to remove a significant part of the cyh2 gene. The right construct can be isolated and checked by a restriction digest with Bglll. this restriction site is present in the EcoRV fragment of PAS2 to be eliminated.  This climates the cyh2 gene which is slightly toxic gene and involved in growth retardation. This gene is non-essential for the performing of RNAi and Yeast two hybrid experiments. After the elimination of the EcoRV fragment,  The EcoRI restriction site which is located between the DNA sequence encoding for GAL4DB and HA (epitope) becomes unique for the plasmid, and can be used to subsitute HA with a T7 promoter containing linker.<br>
This ensures persistence of all restriction sites, allowing both in frame cloning and compatibility with previous vectors and pGAD424. We used the following linker (primers: aattcttaatacgactcactatagggca and tatgccctatagtgagtcgtattaag) using Ecorl and Ndel cloning sites. We adopted the same strategy for the reverse T7 (primers:<br>
gatccgtcgacagatctccctatagtgagtcgtattactgca catgggccctatagtgagtcgtattaag and<br>
gtaatacgactcactatagggagatctgtcgacg) with BamHl and Pstl. To avoid loss of Sail we included it in the primer.The resulting vector was designated pAS2-cyh2-HA+both T7-final.<br>
Having the T7 promoter (or alternatively the T3, or SP6 promoter) in pGAD424 allows to go quickly from interacting protein to RNAi and assigning function to the isolated DNA fragment. An additional advantage is the ability to make by in vitro transcription coupled to in vitro translation (There is an ATG in frame with either GAL4DB or GAL4AD) labeled protein which can be used for in vitro controls (e.g. pull down assays) of the actual protein-protein interaction.<br>
The sequences of the plasmids produced and the SP6 and T3 polymerase are identified in the Sequence Listing provided below-<br><br><br><br><br><br><br><br><br><br><br>
Claims<br>
1.	A method of identifying DNA responsible for<br>
conferring a particular phenotype in a cell which<br>
method comprises<br>
a)	constructing a cDNA or genomic library of<br>
the DNA of said cell in a suitable vector in an<br>
orientation relative to a promoter(s) capable of<br>
initiating transcription of said cDNA or DNA to double<br>
stranded (ds) RNA upon binding of an appropriate<br>
transcription factor to said promoter(s),<br>
b)	introducing said library into one or more of<br>
said cells comprising said transcription factor, and<br>
c)	identifying and isolating a particular<br>
phenotype of said cell comprising said library and<br>
identifying the DNA or cDNA fragment from said library<br>
responsible for conferring said phenotype.<br><br>
2.	A method according to claim 1 wherein said<br>
library is organised into hierarchical pools prior to<br>
step b).<br>
3.	A method of assigning function to a known<br>
DNA sequence which method comprises<br><br>
a)	identifying a homologue(s) of said DNA<br>
sequence in a cell,<br>
b)	isolating the relevant DNA homologue(s) or a<br>
fragment thereof from said cell,<br>
c)	cloning said homologue or fragment thereof<br>
into an appropriate vector in an orientation relative<br>
to a suitable promoter(s) capable of initiating<br>
transcription of dsRNA from said DNA homologue or<br>
fragment upon binding of an appropriate transcription<br>
factor to said promoter(s),<br>
d)	introducing said vector into said cell from<br>
step a) comprising said transcription factor, and<br>
e)   identifying the phenotype of said cell compared to wild type.<br>
4.	A method according to any of claims 1 to 3<br>
wherein said DNA library, homologue or fragment is<br>
cloned in a sense and an antisense direction relative<br>
to said promoter.<br>
5.	A method according to any of claims 1 to 3<br>
wherein said DNA library, homologue or fragment is<br>
cloned between two promoters capable of producing<br>
dsRNA from said DNA library, homologue or fragment<br>
upon binding of said transcription factor to said<br>
promoters.<br>
6.	A method according to any of claims 1 to 5<br>
wherein said cell is adapted to express said<br>
transcription factor.<br>
7.	A method according to any of claims 1 to 6<br>
wherein said DNA library, homologue or fragment is<br>
constructed in a suitable vector which comprises a<br>
sequence of nucleotides encoding said transcription<br>
factor operably linked to a suitable promoter.<br>
8.	A method according to any of claims 1 to 6<br>
wherein said transcription factor is encoded by a<br>
further vector independent of the vector including<br>
said DNA library, DNA homologue or fragment and which<br>
sequence encoding said transcription factor is<br>
operably linked to a suitable promoter.<br>
9.   A method according to claim 7 or 8 wherein said transcription factor comprises any of T7,<br>
T3 or SP6 polymerase.<br>
10.	A method according to claim 7 or 8 wherein<br>
said suitable promoter comprises any of let 858,<br>
SERCA, UL6, myo-2 or myo-3.<br>
11.	A method according to any of claims 7 to 10,<br>
wherein said suitable vector or said further vector<br>
comprises a selectable marker.<br>
12.	A method according to claim 11 wherein said<br>
selectable marker comprises a nucleotide sequence<br>
capable of inhibiting or preventing expression of a<br>
gene in said cell and which gene is responsible for<br>
conferring a known phenotype.<br>
13.	A method according to claim 12 wherein said<br>
nucleotide sequence comprises a sequence which is a<br>
part of or identical to said gene conferring said<br>
phenotype, and which nucleotide sequence is itself<br>
oriented relative to a suitable promoter(s) capable of<br>
initiating transcription of double stranded RNA upon<br>
binding of an appropriate transcription factor to said<br>
promoters.<br>
14.	A method according to claim 12 wherein said<br>
nucleotide sequence is a part of or identical to said<br>
gene sequence conferring said phenotype, and which<br>
nucleotide sequence is such as to permit integration<br>
of said suitable or further vector by homologous<br>
recombination in the genome of said cell and following<br>
said integration said nucleotide sequence is capable<br>
of inhibiting expression of said gene sequence<br>
conferring said phenotype.<br>
15.	A method according to claim 14 wherein said<br>
nucleotide sequence comprises stop codons sufficient<br>
to prevent translation of said nucleotide sequence<br>
following its integration into said genome.<br>
16.	A method according to any preceding claim<br>
wherein said cell is a microorganism suitable for<br>
feeding to, transforming or infecting an organism.<br>
17.	A method according to any of claims 1 to 14<br>
wherein said cell is contained in an organism or an<br>
embryo thereof.<br>
18.	A method according to any of claims 1 to 17<br>
wherein said promoters are T7 promoters.<br>
19.	A method according to any of claims 12 to 18<br>
wherein said known gene sequence comprises a sup-35<br>
gene or a fragment thereof which is selectable by<br>
identifying offspring growing at a temperature above<br>
25°C following introduction of said vector in the<br>
genome of a pha-1 et!23ts mutant C. elegans worm.<br>
20.	A method according to any of claims 1 to 19<br>
wherein said cell or organism is contacted with a<br>
specified compound for screening for a desired<br>
phenotype, such as resistance or sensitivity to said<br>
compound when compared to the wild type cell or<br>
organism.<br>
21.	A method according to any preceding claim<br>
wherein said transcription factor is inducible.<br>
22.	A method according to claim 16 wherein said<br>
microorganism is an E. coli strain which is an RNAase<br>
III and preferably an RNAase negative strain.<br>
23.	A method according to any of claims 17<br>
wherein said organism is selected from the class<br>
nematoda.<br>
24.	A method according to claim 23 wherein said<br>
organism is Caenorhabditis elegans.<br>
25.	A method of generating a transgenic non-<br>
human organism comprising an exogenous transcription<br>
factor and a transgene comprising a promoter operably<br>
linked to DNA fragment which is expressed upon binding<br>
of said transcription factor thereto, the method<br>
comprising<br><br>
a)	providing a first transgenic organism<br>
comprising a first construct incorporating DNA<br>
encoding an exogenous transcription factor and a<br>
second transgenic organism comprising a second<br>
construct including at least one promoter operably<br>
linked to a desired DNA sequence which is expressed<br>
upon binding of the transcription factor of said first<br>
transgenic organism thereto,<br>
b)	crossing said first and second transgenic<br>
organisms and selecting offspring expressing said<br>
desired DNA sequence.<br><br>
26.	A method according to claim 25 wherein said<br>
first and second transgenic organisms are generated by<br>
transforming said first and second constructs into<br>
respective microorganisms for subsequent feeding to<br>
the respective organism.<br>
27.	A method according to claim 25 or 26 wherein<br>
said second construct comprises said desired DNA<br>
sequence in an orientation relative to said promoter so as to be capable of initiating transcription of said DNA to dsRNA upon binding of said transcription factor thereto.<br>
28.	A method according to claim 27 wherein said<br>
second construct comprises two promoters flanking said<br>
desired DNA sequence which promoters can initiate<br>
transcription of said DNA sequence to dsRNA upon<br>
binding of said transcription factor to said<br>
promoters.<br>
29.	A method according to claim 27 wherein said<br>
DNA sequence is provided in a sense and an antisense<br>
orientation relative to said promoter so as to produce<br>
dsRNA upon binding of the transcription factor to the<br>
promoter.<br>
30.	A method according to any of claims 25 to 29<br>
wherein said second transgenic organism further<br>
comprises a reporter gene operably linked to a<br>
promoter which is capable of initiating transcription<br>
of said reporter upon binding of said transcription<br>
factor thereto.<br>
31.	A method according to any of claims 25 to 30<br>
wherein said transcription factor comprises a<br>
polymerase.<br>
32.	A method according to claim 31 wherein said<br>
polymerase comprises any of T7, T3 or SP6 polymerase.<br>
33.	A method according to any of claims 25 to 31<br>
wherein said promoters comprises any of T7, T3 or SP6<br>
promoters.<br>
34.	A method according to claim 30 wherein said<br>
reporter gene comprises any of those sequence encoding<br>
Luciferase, Green Fluorescent protein, P galactosidase<br>
or 3-lactamase.<br>
35.	A method according to any of claims 25 to 30<br>
wherein said organism is of the species nematoda.<br>
36.	A method according to claim 35 wherein said<br>
nematoda species is C. elegans.<br>
37.	A transgenic non-human multicellular<br>
organism obtainable according to the methods of any<br>
one of claims 25 to 34.<br>
38.	A method of validating clones identified in<br>
yeast two hybrid vector experiments which method<br>
comprises<br><br>
a)	providing a construct including the DNA<br>
encoding the protein identified in the two hybrid<br>
vector experiment, which construct is such that said<br>
DNA is orientated relative to a promoter(s) that is<br>
capable of initiating transcription of said DNA to<br>
double stranded RNA upon binding of an appropriate<br>
transcription factor to said promoter(s),<br>
b)	transforming a cell comprising said<br>
transcription factor with said construct, and<br>
c)	identifying a phenotypic change in said cell<br>
or organism when compared to a wild type.<br>
39.	A method according to claim 38 wherein said<br>
DNA sequence is provided between two<br>
promoters capable of initiating<br>
transcription of the DNA sequence to dsRNA<br>
upon binding of the transcription factor to<br>
said promoters.<br>
40.A method according to claim 38 wherein said DNA is provided in a sense and an antisense orientation relative to said promoter such that binding of the transcription factor to said promoter initiates transcription of dsRNA from said DNA.<br>
41.	A method according to any of claims 38 to 40<br>
wherein said transcription factor is inducible in said<br>
cell.<br>
42.	A method according to any of claims 38 to 41<br>
wherein said promoter is a phage polymerase promoter<br>
and said transcription factor is a RNA polymerase.<br>
43.	A method according to claim 42 wherein said<br>
polymerase is any of T7 RNA polymerase, T3 RNA<br>
polymerase or SP6 RNA polymerase.<br>
44.	A method according to claim 43 wherein said<br>
promoters comprise any of T7, T3 or SP6 promoter<br>
45.	A method according to any of claims 38 to 44<br>
wherein said construct is such that it may be used in<br>
yeast two hybrid experiments.<br>
46.	A method according to any of claims 38 to 45<br>
wherein said cell is an E. coli cell.<br>
47.	A method according to any of claims 38 to 45<br>
wherein said cell is part of an organism or an embryo<br>
thereof.<br>
48.	A method according to claim 47 wherein said<br>
organism is of the species nematoda and preferably C. elegans.<br>
49.	Plasmid pGNl as illustrated in Figure 1.<br>
50.	Plasmid pGNlOO as illustrated in Figure 2.<br>
51.	The yeast two hybrid vector plasmid<br>
illustrated in any of Figures 4, 15 or 16(Seq ID Nos 8<br>
and 9}.<br>
52.	A plasmid as illustrated in Figure 7.<br>
53.	A plasmid as illustrated in Figure 8.<br>
54.	A method of alleviating infestation of plant<br>
pests, which method comprises<br><br>
a)	identifying a DNA sequence from said pest<br>
which is critical for its survival, growth,<br>
proliferation,<br>
b)	cloning said sequence from step a) or a<br>
fragment thereof in a suitable vector in an<br>
orientation relative to promoter(s) such that said<br>
promoter(s) is capable of initiating transcription of<br>
said DNA sequence to RNA or dsRNA upon binding of an<br>
appropriate transcription factor to said promoter(s),<br>
and<br>
c)	introducing said vector into the plant.<br>
55.	A method according to claim 54 wherein said<br>
DNA sequence is provided between two promoters such<br>
that binding of the transcription factor to the<br>
promoters results in transcription of the DNA to<br>
dsRNA.<br>
56.	A method according to claim 54 wherein said<br>
DNA sequence is provided in a sense and an antisense<br>
orientation relative to said promoter such that<br>
binding of the transcription factor to the promoter<br>
results in transcription of the DNA to dsRNA.<br>
57.	A method according to claim 54 wherein said<br>
pest is a nematode worm.<br>
58.	A method according to claim 57 wherein said<br>
nematode comprises any of Tylenchulus ssp. Radopholus<br>
ssp., Rhadinaphelenchus ssp., Heterodera ssp.,<br>
Rotylenchulus ssp., Pratylenchus ssp., Belonolaimus<br>
ssp., Canjanus ssp., Meloidogyne ssp., Globodera ssp.,<br>
Nacobbus ssp., Ditylenchus ssp., Aphelenchoides ssp.,<br>
Hirschmenniella ssp., Anguina ssp., Hoplolaimus ssp.,<br>
Heliotylenchus ssp., Criconemellassp., Xiphinemassp.,<br>
Longidorus ssp., Trichodorus ssp., Paratrichodorus<br>
ssp., Aphelenchs ssp.<br>
59.	A method according to claim 55 wherein said<br>
DNA sequence or fragment thereof is cloned between two<br>
tissue, preferably root specific promoters.<br>
60.	An expression vector for use in a method<br>
according to any proceeding claim comprising a<br>
promoter or promoters oriented relative to a DNA<br>
sequence such that they are capable of initiating<br>
transcription of said DNA sequence to double stranded<br>
RNA upon binding of an appropriate transcription<br>
factor to said promoter or promoters.<br>
61.	An expression vector according to claim 60<br>
comprising two identical promoters flanking the DNA<br>
sequence.<br>
62.	An expression vector according to claim 60<br>
comprising said DNA sequence in a sense and an<br>
antisense orientation relative to said promoter.<br>
63.	An expression vector according to any of<br>
claims 60 to 62 which further comprises a nucleotide<br>
sequence encoding a selectable marker.<br>
64.	An expression vector according to claim 63<br>
wherein said nucleotide sequence encoding said<br>
selectable marker is orientated relative to the<br>
promoter(s) such that transcription of the nucleotide<br>
sequence to double stranded RNA occurs upon binding of<br>
an appropriate transcription factor to said<br>
promoter(s).<br>
65.	An expression vector according to claim 64<br>
wherein said nucleotide sequence encoding the<br>
selectable marker is provided between the identical<br>
promoters capable of initiating transcription of the<br>
nucleotide sequence to dsRNA upon binding of the<br>
transcription factor to the promoters.<br>
66.	An expression vector according to claim 64<br>
wherein said nucleotide sequence encoding the<br>
selectable marker is provided in a sense and an<br>
antisense orientation relative to the promoter such<br>
that transcription of the nucleotide sequence to dsRNA<br>
upon binding of the transcription factor to said<br>
promoter occurs.<br>
67.	An expression vector according to claim 63<br>
or 64 wherein said selectable marker comprises a<br>
nucleotide sequence encoding sup-35, for introduction<br>
into C. elegans having a pha-1 mutation.<br>
68.	An expression vector for expressing an<br>
appropriate transcription factor for use in a method<br>
according to any of claims 1 to 48 and 54 to 59 which<br>
vector comprises a sequence of nucleotides encoding<br>
said transcription factor operably linked to suitable<br>
expression control sequences.<br>
69.	An expression vector according to claim 68<br>
wherein said expression control sequences include<br>
promoters which are inducible, constitutive, general<br>
or tissue specific promoters, or combinations thereof.<br>
70.	An expression vector according to any of<br>
claims 68 to 69 wherein said transcription factor<br>
comprises a phage polymerase, and preferably T7 RNA<br>
polymerase.<br>
71.	An organism or cell transformed or<br>
transfected with a plasmid according to any of claims<br>
49 to 53 or an expression vector according to any of<br>
claims 60 to 70.<br>
72.	An organism according to claim 71, which is<br>
of the species nematoda and preferentially C. elegans.<br>
73.	A method of introducing dsRNA or DNA capable<br>
of producing dsRNA into an organism which method<br>
comprises feeding said organism with a suitable<br>
microorganism comprising an expression vector<br>
according to any of claims 60 to 67 or feeding said<br>
organism directly with an expression vector according<br>
to any of claims 60 to 67.<br>
74.	A method according to claim 73 wherein said<br>
microorganism or said organism is adapted to express<br><br>
said transcription factor.<br>
75.	A method according to claim 74 wherein<br>
either said microorganism or said organism comprises<br>
an expressing vector according to any of claims 66 to<br>
69.<br>
76.	A method according to any of claims 73 to 75<br>
wherein said organism is C. elegans and said<br>
microorganism is E. coli.<br>
77.	A method according to claim 76 wherein said<br>
E. coli strain is an RNAaseIII negative strain.<br>
78.	A method according to any of claims 73 to 75<br>
wherein said organism is a C. elegans nuc-1 mutant.<br>
79.	A selection system for identifying<br>
transformation of a cell or organism with a vector<br>
according to claims 60 to 63 which system comprises a<br>
vector according to claims 60 to 63 and said<br>
selectable marker comprises a nucleotide sequence<br>
capable of inhibiting or preventing expression of a<br>
gene in said cell or organism which gene is<br>
responsible for conferring a known phenotype.<br>
80.	A selection system according to claim 79<br>
wherein said nucleotide sequence comprises a sequence<br>
which is a part of or identical to said gene<br>
conferring said known phenotype, and which nucleotide<br>
sequence is itself located between two identical<br>
promoters capable of initiating transcription of<br>
double stranded RNA upon binding of an appropriate<br>
transcription factor to said promoters.<br>
81.	A selection system according to claim 79<br>
wherein said nucleotide sequence is a part of or<br>
identical to said gene sequence which confers a known<br>
phenotype on said cell or organism, and which<br>
nucleotide sequence permits integration of said vector<br>
by homologous recombination in the chromosome of said<br>
cell or organism and following said integration said<br>
sequence inhibits expression of said gene sequence<br>
conferring said known phenotype.<br>
82.	A selection system according to claim 81<br>
wherein said nucleotide sequence comprises stop codons<br>
sufficient to prevent translation of said nucleotide<br>
sequence following integration into said genome.<br>
83.	A selection system according to claim 79<br>
wherein said known gene sequence comprises a sup-35<br>
gene or a fragment thereof which is selectable by<br>
identifying offspring growing at a temperature above<br>
25°C following introduction of said vector in a pha-1<br>
et123ts mutant C. elegans worm.<br>
84.	A method according to claim 74 wherein said<br>
transcription factor is T7 RNA polymerase.<br>
85.	A method of assigning function to a DNA<br>
sequence of a multicellular organism which method<br>
comprises:<br>
a) providing:<br>
(i)  a first construct comprising said DNA sequence cloned in a sense direction under the regulation of a suitable promoter<br>
(ii)   a second construct comprising said DNA sequence cloned in an anti-sense direction under the regulation of the same promoter as in step (i) ,<br>
in a multicellular organism which is adapted to initiate transcription of said DNA fragment from said promoter,<br>
b)   identifying the phenotype of said multicellular organism compared to wild type.<br>
86.	A method of assigning function to a DNA<br>
sequence of a multicellular organism which method<br>
comprises:<br>
a)	providing<br>
i)   a construct comprising said DNA fragment cloned between two promoters capable of promoting transcription in said multicellular organism,<br>
in a multicellular organism capable of initiating transcription from said promoter;<br>
b)	identifying the phenotype of said<br>
multicellular organism compared to wild type.<br>
87.	A method for expressing a gene, cDNA or<br>
DNA fragment in C.  elegans which method comprises :<br>
i) providing a transgenic C.  elegans expressing an exogenous transcription factor,<br>
ii) cloning said gene or cDNA or DNA fragment into an appropriate vector operably linked to a promoter capable of initiating transcription of said gene, cDNA or DNA fragment upon binding of said exogenous transcription factor thereto,<br>
iii) introducing said vector into said transgenic C.  elegans from step i).<br>
88.	A method according to claim 87 wherein<br>
said exogenous transcription factor is a phage<br>
polymerase.<br>
89.	A method according to claim 88 wherein<br>
said phage polymerase is T3, SP6 or T7 RNA polymerase.<br>
90.  A method according to any of claims 87 to 89 wherein said promoter is any of T3, T7 or SP6 promoter.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tNC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDAwMDYtZGVsLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-00006-del-pct-409.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="209922-a-method-for-coating-a-fiber-with-a-water-blocking-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209925-a-granular-detergent-composition-having-encapsulated-perfume-particle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209924</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00006/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Sep-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jan-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DEVGEN NV,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 30, 9052 GENT-ZWIJNAARDE, BELGIUM,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KATHERINE MORTIER</td>
											<td>DEVEGEN N.V., TECHNOLOGIEPARL, ZWIJNAARDE 9, B-9052 ZWIJNAARDE, BELGIUM,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>THIERRY BOGAERT</td>
											<td>DEVEGEN N.V., TECHNOLOGIEPARL, ZWIJNAARDE 9, B-9052 ZWIJNAARDE, BELGIUM,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHRIST PLATTEEUW</td>
											<td>DEVEGEN N.V., TECHNOLOGIEPARL, ZWIJNAARDE 9, B-9052 ZWIJNAARDE, BELGIUM,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GEERT PLAETINCK</td>
											<td>C/O. DEVEGEN N.V., TECHNOLOGIEPARL, ZWIJNAARDE 9, B-9052 ZWIJNAARDE, BELGIUM,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP99/04718</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-07-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9814536.0</td>
									<td>1998-07-03</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>9827152.1</td>
									<td>1998-12-09</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209924-a-method-of-alleviating-infestation-of-plant-pests by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:40:41 GMT -->
</html>
