<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/209963-method-for-determining-the-degree-of-methylation-of-defined-cytosines-in-genomic-dna-in-the-sequence-context-5-cpg-3 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:33:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 209963:METHOD FOR DETERMINING THE DEGREE OF METHYLATION OF DEFINED CYTOSINES IN GENOMIC DNA IN THE SEQUENCE CONTEXT 5&#x27; -CPG-3&#x27;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR DETERMINING THE DEGREE OF METHYLATION OF DEFINED CYTOSINES IN GENOMIC DNA IN THE SEQUENCE CONTEXT 5&#x27; -CPG-3&#x27;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method is described for the detection of the degree of methylation of a specific cutosine in the sequence context 5&#x27;-CpG-3&#x27; of a genomic DNA sample. In the first step, the genomic DNA which contain the said specific cytosine are amplified, whereby the amplified products are given a detectable label and in the following steps the extent of hybridization of the amplified products on two classes of oligonucleotides is determined by detection of the label of the amplified products, and a conclusion is made on the extent of methylation of said specific cytosine in the genomic DNA sample from the ration of the labels detected on the two classes of oligonucleotides as a consequence of the hybridization. To The Controller of Patents, The Patent Office, Mumbai</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM   .2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
COMPLETE SPECIFICATION (See Section 10, rule 13)<br><br><br><br><br><br>
                            AN IN VITRO METHOD FOR THE DETECTION OF THE DEGREE OF<br>
                                             METHYLATION OF A SPECIFIC CYTOSINE IN THE SEQUNCE CONTEXT 5"- 4K-3<br><br><br><br><br><br>
                EPIGENOMICS AG of KASTANIENALLEE 24, D-10435 BERLIN,      <br>
                GERMANY,GERMAN Company<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed : -<br>
ORIGINAL<br>
155/MUMNP/03<br>
   31/01/03<br><br>
The invention concerns a method for the detection of the degree of methylation of a specific cytosine in the sequence context 5"-CpG-3" of a genomic DNA sample.<br>
The levels of observation that have been well studied in molecular biology according to developments in methods in recent yearslnclude the genes themselves, the transcription of these genes into RNA and the translation to proteins therefrom. During the course of development of an individual, which gene is turned on and how the activation and inhibition of certain genes in certain cells and tissues are controlled can be correlated with the extent and nature of the methylation of the genes or of the genome. In this regard, pathogenic states are also expressed by a modified methylation pattern of individual genes or of the genome.<br>
The present invention describes a method with which many cytosine bases in a given DNA sample can be investigated simultaneously for the presence of a methyl group at position 5 by means of hybridization. It also describes nucleic acids, oligonucleotides and PNA oligomers which are useful in order to employ the method for the diagnosis of existing diseases or of predisposition for [certain] diseases.<br>
5-Methylcytosine is the most frequent covalently modified base in the DNA of gukaryotic cells. For example, it plays a role in the regulation of transcription, in genetic imprinting and in tumorigenesis. The identification oi-5-methylcytosine as a component of genetic information is thus of considerable interest. 5-Methylcytosine positions, however, cannot be identified by sequencing, since 5-methylcytosine has the same base-pairing behavior as cytosine. In addition, in the case of a PCR amplification, the epigenetic information which is borne by the<br><br>
5-methylcytosines is completely lost.<br>
The methylation of CpG islands is often equated with transcription inactivity. Although there is clear evidence that CDG islands are to be found in promoters of genes, not all CDG islands and methylation sites are localized in known promoters. In various tissue-specific and imprinting genes, the CpG islands are localized at considerable distance downstream of the start of transcription, and also many genes possess multiple promoters. Methylation of CpG dinucleotides has been detected as a causal factor for a number of diseases. In contrast to classical mutations, DNA methylation involves a mechanism, which describes a substitution on the base without modifying the coding function of a gene. This interplay between epigenetic modification and classical mutations plays an important role in tumorigenesis. For example, focal hypermethylation and generalized genomic demethylation are features of many different tumor types. It is assumed that tumorigenesis and tumor progression are caused, first of all, by hypermethylation of induced mutation events, and secondly, by the turning off of genes which control cellular proliferation, and/or the induced reactivation of genes, which are [normally] used only for embryological development, via demethylation.<br>
In hereditable non-polyposis colorectal cancer, e.g., the majority of mutation-negative cases of colon cancer are based rather on the hypermethylation of the hMLH1 promoter and the associated non-expression of hMLH1, a repair gene for erroneous base pairings (Bevilacqua RA, Simpson AJ, Methylation of the hMLH1, promoter but no hMLH1 mutations in sporadic gastric carcinomas with high-level microsatellite instability. Int J Cancer. 2000 Jul 15;87(2):200-3). In the pathogenesis of lung cancer, the loss of expression is correlated with the. methylation of CpG islands in the promoter sequence of an RAS effector homolog. (Dammann R, Li C, Yoon JH, Chin PL, Bates S, Pfeifer GP, Nucleotide. Epigenetic inactivation of a RAS association domain family protein from the lung tumour suppressor locus 3p21.3. Nat Genet. 2000 Jul;25(3):315-9). An<br><br><br>
epigenetic inactivation of the LKB1 tumor supressor gene, including the hypermethylation of the promoter, is associated with the Peutz-Jeghers syndrome (Esteller M, Avizienyte E, Corn PG, Lothe RA, Baylin SB, Aaltonen LA, Herman JG, Epigenetic inactivation of LKB1 in primary tumors associated with the Peutz-Jeghers syndrome. Oncogene. 2000 Jan 6; 19(1): 164-8).<br>
A plurality of diseases, which are associated with methylation, have in their etiology a close connection with the tumor suppressor genes p16 or p15. Thus a relationship between Mycosis fungoides and hypermethylation of the p16(INK4a gene is assumed (Navas IC, Ortiz-Romero PL, Villuendas R, Martinez P, Garcia C, Gomez E, Rodriguez JL, Garcia D, Vanaclocha F, Iglesias L, Piris MA, Algara P, p16(INK4a) gene alterations are frequent in lesions of mycosis fungoides. Am J Pathol. 2000 May; 156(5): 1565-72). Also, there is a strong correlation between the turning off of the transcription of the p16 gene in gastric carcinoma and the de novo methylation of a few specific CpG sites (Song SH, Jong HS, Choi HH, Kang SH, Ryu MH, Kim NK, Kim WH, Bang YJ, Methylation of specific CpG sites in the promoter region could significantly down-regulate p16(INK4a) expression in gastric adenocarcinoma. Int J Cancer. 2000 Jul 15;87(2):236-40). The pathogenesis of cholangiocarcinoma, which is associated with primary sclerosing cholangitis, has been related to the inactivation of the p16 tumorsuppressor gene, which is again dependent on the methylation of the p16 promoter (Ahrendt SA, Eisenberger CF, Yip L, Rashid A, Chow JT, Pitt HA, Sidransky D, Chromosome 9p21 loss and p16 inactivation in primary sclerosing cholangitis-associated cholangiocarcinoma. J Surg Res. 1999 Jun 1;84(i):88-93). The inactivation of the p16 gene by hypermethylation plays a role in the genesis of leukemia and in the progression of acute lymphoblastic leukemia (Nakamura M, Sugita K, Inukai T, Goi K, lijima K, Tezuka T, Kojika S, Shicaishi K, Miyamoto N, Karakida N, Kagami K, O-Koyama T, Mori T, Nakazawa S, p16/MTS1/INK4A gene is frequently inactivated by hypermethylation in childhood acute lymphoblastic leukemia with 11q23 translocation. Leukemia. 1999 Jun;13(6):884-90). tn addition, it is postulated that the hypermethylation of the p16 and p15<br><br><br>
genes plays a decisive role in the tumorigenesis of multiple myeloma (Ng MH, Wong IH, Lo KW, DNA methylation changes and multiple myeloma. Leuk Lymphoma. 1999 Aug;34(5-6):463-72). The VHL gene which is inactivated by methylation, appears to participate in predisposition to renal carcinoma (Glavac D, Ravnik-Glavac M, Ovcak Z, Masera A, Genetic changes in the origin and development of renal cell carcinoma (RCC). Pflugers Arch. 1996;431(6 Suppl 2):R193-4). A divergent methylation of the 5"-CpG island may participate in nasopharyngeal carcinoma, possibly by the inactivation of transcription of the p16 gene (Lo KW, Cheung ST, Leung SF, van Hasselt A, Tsang YS, Mak KF, Chung YF, Woo JK, Lee JC, Huang DP, Hypermethylation of the p16 gene in nasopharyngeal carcinoma. Cancer Res. 1996 Jun 15;56(12):2721-5). An inactivation of the p16 protein was detected in liver cell carcinoma. Promoter hypermethylation and homozygous deletions are the most frequent mechanisms here (Jin M, Piao Z, Kim NG, Park C, Shin EC, Park JH, Jung HJ, Kim CG, Kim H, p16 is a major inactivation target in hepatocellular carcinoma. Cancer. 2000 Jul 1;89(1):60-8). DNA methylation as a control of gene expression was detected for the BRCA1 gene for breast cancer (Magdinier F, Billard LM, Wittmann G, Frappart L, Benchaib M, Lenoir GM, Guerin JF, Dante R, Regional methylation of the 5" end CpG island of BRCA1 is associated with reduced gene expression in human somatic cells FASEB J. 2000 Aug;14(11):1585-94). A correlation between methylation and non-Hodgkin"s lymphoma is also assumed (Martinez-Delgado B, Richart A, Garcia MJ, Robledo M, Qsorio A, Cebrian A, Rivas C, Benitez J, Hypermethylation of P16ink4a and P15ink4b genes as a marker of disease in the follow-up of non-Hodgkin"s lymphomas. Br J Haematol. 2000 Apr;109(1):97-103). CpG methylation also brings about the progression of T-cell leukemia, which is related to a decreased expression of the CDKN2A gene (Nosaka K, Maeda M, Tamiya S, Sakai T, Mitsuya H, Matsuoka M, Increasing raetbylation of the CDKN2A gene is associated with the progression of"adult T-cell leukemia. Cancer Res. 2000 Feb 15;60(4): 1043-8). An increased methylation of the CpG islands was established in bladder cancer (Salem C, Liang G, Tsai YC, Coulter J, Knowles MA, Feng AC, Groshen S, Nichols PW, Jones PA, Progressive<br><br>
increases in de novo methylation of CpG islands in bladder cancer. Cancer Res. 2000 May 1;60(9):2473-6). Transcription inactivation by esophageal squamous cell carcinomas has been related to the methylation of the FHIT gene, which is associated with the progression of the disease (Shimada Y, Sato F, Watanabe G, Yamasaki S, Kato M, Maeda M, Imamura M, Loss of fragile histidine triad gene expression is associated with progression of esophageal squamous cell carcinoma, but not with the patient"s prognosis and smoking history. Cancer. 2000 Jul 1;89(1):5-11). Neutral endopeptidase 24.11 (NEP) inactivates the increase of neuropeptides which participate in the growth of androgen-independent prostate cancer. A loss of NEP expression by hypermethylation of the NEP promotors may contribute to the development of neuropeptide-stimulated, androgen-independent prostate cancer (Usmani BA, Shen R, Janeczko M, Papandreou CN, Lee WH, Nelson WG, Nelson JB, Nanus DM, Methylation of the neutral endopeptidase gene promoter in human prostate cancers. Clin Cancer Res. 2000 May;6(5): 1664-70). Adrenocortical tumors in adults display structural abnormalities in the tumor DNA. Among other things, these abnormalities contain an overexpression of the IGF2 gene in correlation with a demethylation of the DNA at this locus (Wilkin F, Gagne N, Paquette J, Oligny LL, Deal C, Pediatric adrenocortical tumors: molecular events leading to insulin-like growth factor II gene overexpression. J Clin Endocrinol Metab. 2000 May;85(5):2048-56. Review). It is assumed that DNA methylations in several exons in the retinoblastoma gene contribute to the disease (Mancini D, Singh S, Ainsworth P, Rodenhiser D, Constitutively methylated CpG dinucleotides as mutation hot spots in the retinoblastoma gene (RB1)-Am J Hum Genet. 1997 Jul;61(1):80-7). In chronic myeloid leukemia, a relationship is suspected between the deregulation of the p53 gene and a change in the methylation pattern with progression of the disease (Guinn BA, Mills Kl, p53 mutations, methylation and genomic instability in the progression of chronic myeloid leukaemia. Leuk Lymphoma.. 1997 Jul;26(3-4):211-26). A connection with methylation has also been detected for acute myeloid leukemia (Melki JR, Vincent PC, Clark SJ. Concurrent DNA hypermethylation of multiple genes in acute myeloid leukemia.<br>
6<br><br>
Cancer Res. 1999 Aug 1;59(l5):3730-40). A tumor-specific methylation site in the Wilms tumor suppressor gene has been identified (Kleymenova EV, Yuan X, LaBate ME, Walker CL, Identification of a tumor-specific methylation site in the Wilms tumor suppressor gene. Oncogene. 1998 Feb 12;16(6):713-20). In Burkitt"s lymphoma, several promotors have a complete CpG methylation (Tao Q, Robertson KD, Manns A, Hildesheim A, Ambinder RF, Epstein-Barr virus (EBV) in endemic Burkitt"s lymphoma: molecular analysis of primary tumor tissue. Blood. 1998 Feb 15;91 (4): 1373-81). It is assumed that DNA methylation plays a role in thyroid carcinoma (Venkataraman GM, Yatin M, Marcinek R, Ain KB, Restoration of iodide uptake in dedifferentiated thyroid carcinoma: relationship to human Na+/l-symporter gene methylation status. J Clin Endocrinol Metab. 1999 Jul;84(7):2449-57).<br>
Not only are many cancer diseases associated with methylation, but there are also many other diseasesjbat are related to methylation. Investigations of, inflammatoiy arthritis have indicated that this disease is associated with a hypomethylation of genomic DNA (Kim Yl, Logan JW, Mason JB, Roubenoff R, DNA hypomethylation in inflammatory arthritis: reversal with methotrexate. J Lab Clin Med. 1996 Aug; 128(2): 165-72). A methylation-regulated expression has been detected for the ICF syndrome (Kondo T, Bobek MP, Kuick R, Lamb B, Zhu X, Narayan A, Bourc"his D, Viegas-Pequignot E, Ehrlich M, Hanash SM, Whole-genome methylation scan in ICF syndrome: hypomethylation of nonsatellite DNA repeats D4Z4 and NBL2). The participation of methylation is suspected in systemic lupus erythematosus (Vallin H, Perers A, Aim GV, Ronnblom L, Anti-double-stranded DNA antibodies and immunostimulatory plasmid DNA in combination mimic the endogenous IFN-alpha inducer in systemic lupus erythematosus. J Immunol. 1999 Dec; 163(11):6306-13) and there may.also be a relationship between the Duchenne muscular dystrophy gene and a CpG-rich island (Banerjee S, Singh PB, Rasberry C, Cattanach BM, Embryonic inheritance of  the chromatin organisation of the imprinted H19 domain in mouse spermatozoa. Mech Dev. 2000 Feb;9Q(2y.217-2e-, Burmeister M, Letoach H,<br><br>
Long-range restriction map around the Duchenne muscular dystrophy gene. Nature. 1986 Dec 11-17;324(6097):582-5). An epigenetic effect, which involves the hypomethylation of the amyloid precursor protein [gene], which is related to the development of the disease, is suspected in Alzheimer"s  disease (West RL, Lee JM, Maroun LE, Hypomethylation of the amyloid precursor protein gene in the brain of an Alzheimer"s disease patient. J Mol Neurosci. 1995;6(2):141-6). The methylation state also plays an important role at the chromosomal level. For example, in mental retardation syndromes, which are coupled with the fragility of the X chromosome, the degree of chromosomal fragility is determined by the methylation (de Muniain AL, Cobo AM, Poza JJ, Saenz A, [Diseases due to instability of DNA]. Neurologia. 1995 Dec; 10 Suppl 1:12-9).<br>
A relatively new method that in the meantime has become the most widely used method, for investigating DNA for 5-methylcytosine is based on the specific reaction,of bisulfite with cytosine, which, after subsequent alkaline hydrolysis, is then converted to uracil, which corresponds in its base-pairing behavior to thymidine. In contrast, 5-methylcytosine is not modified under these conditions. Thus, the original DNA is converted so that methylcytosine, which originally cannot be distinguished from cytosine by its hybridization behavior, can now be detected by "standard" molecular biology techniques as the only remaining cytosine, for example, by amplification and hybridization or sequencing. All of these techniques are based on base pairing, which is now fully utilized. The prior art, which concerns sensitivity, is defined by a method that incorporates the DNA to be investigated in an agarose matrix, so that the diffusion and renaturation of the DNA is prevented (bisulfite reacts only on single-stranded DNA) and all precipitation and purification steps are replaced by rapid dialysis. (Olek, A. et al., Nucl. Acids Res. 1996, 24, 5064-5066). Individual cells can be investigated by this method, which illustrates the potential of the method. Of course, up until now, only individual regions of up to approximately 3000 base pairs long have been investigated; a global investigation of cells for thousands of possible methylation analyses is not possible. Of course, this method also cannot reliably analyze very<br><br><br>
small fragments of small quantities of sample. These are lost despite the protection from diffusion through the matrix.<br>
An overview of other known possibilities for detecting 5-methylcytosines can be derived from the following review article: Rein, T., DePamphilis, M. L, Zorbas, H., Nucleic Acids Res. 1998, 26, 2255.<br>
The bisulfite technique has been previously applied only in research, with a few exceptions (e.g., Zechnigk, M. et al., Eur. J. Hum. Gen. 1997, 5, 94-98). However, short, specific segments of a known gene have always been amplified after a bisulfite treatment, and either completely sequenced (OIek, A. and Walter, J., Nat. Genet. 1997, 17, 275-276) or individual cytosine positions have been detected by a primer extension reaction (Gonzalgo, M. L. and Jones, P. A., Nucl. Acids Res. 1997, 25, 2529-2531, WO-Patent 95-00669) or an enzyme step (Xiong, Z. and Laird, P. W., Nucl. Acids. Res. 1997, 25, 2532-2534). Detection by hybridization has also been described (OIek et al., WO-A 99/28,498).<br>
Other publications which are concerned with the application of the bisulfite technique for the detection of methylation in the case of individual genes are: Xiong, Z, and Laird, P. W. (1997), Nucl. Acids Res. 25, 2532; Gonzalgo, M. L and Jones, P. A. (1997), Nucl. Acids Res. 25, 2529; Grigg, S. and Clark, S. (1994), Bioassays 16, 431; Zeschnik, M. et al. (1997), Human Molecular Genetics 6, 387; Teil, R. et al. (1994), Nucl. Acids Res. 22, 695; Martin, V. et al. (1995), Gene 157, 261; WO-A 97/46,705, WO-A 95/15,373 and WO-A 95/45,560.<br>
A review of the prior art in oligomer array production can-be taken from the special edition of Nature Genetics that appeared iri January 1999 (Nature Genetics Supplement, Volume 21, January 1999) and the literature cited therein.<br>
Probes with multiple fluorescent labels have been used for scanning an<br><br><br>
immobilized DNA array. Particularly suitable for fluorescent labels is the simple introduction of Cv3 and Cy5 dyes at the 5"-OH of the respective probe. The fluorescence of the hybridized probes is detected, for example, by means of a confocal microscope. The dyes Cy3 and Cy5, among many others, are commercially available.<br>
Matrix-assisted laser desorptions/ionization mass spectrometry (MALDI-TOF) is a very powerful development for the analysis of biomolecules (Karas, M. and Hillenkamp, F. (1988), Laser desorption ionization of proteins with molecular masses exceeding 10,000 daltons. Anal. Chem. 60: 2299-2301). An analyte is embedded in a light-absorbing matrix. The matrix is vaporized by a short laser pulse and the analyte molecule is transported unfragmented into the gaseous phase. The analyte is ionized by collisions with matrix molecules. An applied voltage accelerates the ions in a field-free flight tube. Ions are accelerated to varying degrees based on their different masses. Smaller ions reach the detector sooner than large ions.<br>
MALDI-TOF spectrometry is excellently suitable for the analysis of peptides and proteins. The analysis of nucleic acids is somewhat more difficult (Gut, I. G. and Beck, S. (1995)), DNA and Matrix Assisted Laser Desorption Ionization Mass Spectrometry. Molecular Biology: Current Innovations and Future Trends 1: 147-157). For nucleic acids, the sensitivity is approximately 100 times poorer than for peptides and decreases overproportionally with increasing fragment size. For nucleic acids, which have a backbone with a multiple negative charge, the ionization process via the matrix is basically less efficient. In MALDI-TOF spectrometry, the choice of matrix plays an imminently important role. Several very powerful matrices, which produce a very fine crystallization, have been found for the desorption of peptides. In the meantime, several effective matrices have been developed for DNA, but the difference in sensitivity has not been reduced thereby. The difference in sensitivity can be reduced by modifying the DNA chemically in such a way that it resembles a peptide. Phosphorothioate<br><br><br>
nucleic acids, in which the usual phosphates of the backbone are substituted by thiophosphates, can be converted by simple alkylation chemistry to a charge-neutral DNA (Gut, I. G. und Beck, S. (1995), A procedure for selective DNA alkylation and detection by mass spectrometry. Nucleic Acids Res. 23: 1367-1373). The coupling of a charge tag to this modified DNA results in an increase in sensitivity of the same order of magnitude as is found for peptides. Another advantage of charge tagging is the increased stability of the analysis in the presence of impurities, which make the detection of unmodified substrates very difficult.<br>
Genomic DNAls obtained from DNA of cells, tissue or other test samples by standard methods. This standard methodology is found in references such as Fritsch and Maniatis, eds., Molecular Cloning: A Laboratory Manual, 1989.<br>
The present invention will present a particularly efficient and reliable method, which permits investigating many cytosine bases in a given DNA sample simultaneously for the presence of a methyl group at position 5 by means of hybridization. Oligonucleotides and PNA oligomers are also presented for this purpose, which are particularly suitable for using the method for the diagnosis of existing diseases and of predisposition for diseases by analysis of a set of genetic and/or epigenetic parameters.<br>
Genetic parameters in the sense of this invention are mutations and polymorphisms of the claimed nucleic acids (Seq. ID 1 to Seq. ID 40712) and additional sequences necessary for their regulation. Particularly designated as mutations are insertions, deletions, point mutations, inversions and polymorphisms and particularly preferred are SNPs (single nucleotide polymorphisms). Polymorphisms, however, can also be insertions, deletions or inversions.<br>
Epigenetic parameters in the sense of this invention are particularly cytosine<br><br>
methylations and other chemical modifications of DNA bases of the claimed nucleic acids (Seq. ID 1 to Seq. ID 40712) and additional sequences necessary for their regulation. Other epigenetic parameters, for example, are the acetylation of histones, although this cannot be directly analyzed with the described method; however, it is correlated in turn with DNA methylation.<br>
The present method serves for the detection of the degree of methylation of at least one specific cytosine in the sequence context 5"-CpG-3" of a genomic DNA sample. The method is particularly preferably used for the simultaneous detection of many different methylation positions.<br>
The object is solved according to the invention by a method for the detection of the degree of methylation of a specific cytosine in the sequence context 5"- CpG-3" of a genomic DNA sample, which is characterized in that a).       the genomic DNA is treated, whereby the cytosine bases are converted to uracil, but not the 5-methylcytosine bases;<br>
b)	segments of the genomic DNA, which contain said specific cytosine, are amplified, whereby the amplified products are given a detectable label;<br>
c)	the amplified products are hybridized to two classes of oligonucleotides and/or PNA oligomers, each of which [class] has at least one member;<br>
d)	the extent of hybridization of the amplified products on the two classes of<br>
oligonucleotides is determined by detection of the label of the amplified products;<br>
e)	a conclusion is made on the extent of methylation of said specific cytosine<br>
in the genomic DNA sample from the ratio of the labels detected on the two<br>
classes of oligonucleotides as a consequence of the hybridization.<br>
A method is particularly preferred in which a hybridization of the amplified products is conducted in step c) on two classes of Oligomers (oligonucleotides and/or PNA oligomers), each of which [class] has at least one member, whereby the oligomers of the first class preferably hybridize to the sequence which arises from the chemical treatment of the genomic DNA, if said specific cytosine was<br><br>
present in the methylated state in the genomic DNA, and whereby the oligomers of the second class preferably hybridize to the sequence which arises from the chemical treatment of the genomic DNA if said specific cytosine was present in the unmethylated state in the genomic DNA. One of these two classes of oligomers, for example, can be formed by oligonucleotides which contain a CG in the middle, and the other class can be formed by oligonucleotides which have a TG (or a CA, in the counterstrand) in the middle. The remaining segments of the oligomer sequences should preferably be the same in the two classes. In this case, oligonucleotides of the first class hybridize to the sequence (around the specific cytosine to be investigated) if it was present in the methylated state before bisulfite conversion, and vice versa, those of the second class would hybridize to the sequence if it was present in the unmethylated state before the bisulfite conversion.<br>
A method is also particularly preferred in which a hybridization of the amplified products is conducted in step c) on two classes of oligomers (oligonucleotides and/or PNA oligomers), each of which [class] has at least one member, whereby the oligomers of the first class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA if said specific cytosine was present in the methylated state in the genomic DNA and less preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA if said specific cytosine was present in the unmethylated state in the genomic DNA, and whereby the oligomers of the second class hybridize to the amplified product to be investigated essentially independently of the degree of methylation of said specific cytosine in the genomic DNA.<br>
Correspondingly, a method is also particularly preferred jn which a hybridization of the amplified products is conducted in step c) on two classes of oligomers (oligonucleotides and/or PNA oligomers), each of which [class] has at least one member, whereby the oligomers of the first class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA if said<br><br><br>
specific cytosine was present in the unmethylated state in the genomic DNA and less preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA if said specific cytosine was present in the methylated state in the genomic DNA, and whereby the oligomers of the second class hybridize to the amplified product to be investigated essentially independently of the degree of methylation of said specific cytosine in the genomic DNA.<br>
Thus, in these cases, the second class of oligomers hybridizes to the amplified product without producing an essential methylation specificity, and thus accordingly, preferably to a position of the amplified product which does not correspond to methylatable cytosine positions. Thus, only the concentration of the amplified product is determined by the intensity of hybridization. In this regard, hybridization to the first class of oligonucleotides results as a function of the degree of methylation of the specific cytosine to be investigated.<br>
It is preferred that the method is conducted not only with the genomic DNA sample, but also logically with standard DNA in which it is known whether the cytosine at said specific position is present in methylated or unmethylated state, whereby the ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the unmethylated standard DNA, serve as a calibration value for a degree of methylation of 0, and correspondingly the ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the methylated standard DNA, serve as a calibration value for a degree of methylation of 1, and these calibration values are used for the determination of the degree of methylation of the genomic DNA samples.<br><br>
It is particularly preferred that additional known standard DNA samples, each of which has any known degree of methylation of said specific cytosine, are used for calibration.<br><br>
The standard DNA samples used and the samples (the amplied product prepared from a genomic DNA) are each preferably given a different label. The standard DNA samples used are each preferably labeled in turn with different labels.<br>
A method is also particularly preferred in which amplified products originating from different genomic DNA samples are provided with different labels. In this case, it is possible to measure different samples simultaneously with one set of oligonucleotides of the two classes, for example, on an oligomer array which contains oligonucleotides of the two classes.<br>
A method is also particularly preferred, in which amplified products originating from the same genomic DNA samples are provided with different labels in order to achieve an increase of measurement accuracy by an averaging of the values obtained from different detection methods. For example, this can be carried out by labeling with different fluorescent dyes. In this case, the measurement is conducted with a fluorescence scanner, which provides several channels for the measurement of individual emission wavelengths of the fluorescent dyes.<br>
Accordingly, a method is also particularly preferred, in which the labels are fluorescent labels.<br>
According to the invention, it is further preferred that said label is a fluorescent label. It is preferred that said label is detected by chemiluminescence, its UV absorption or fluorescence polarization.<br>
It is particularly preferred according to the invention that the DNA treatment in step a) is conducted with a solution of a bisulfite (= Hydrogen sulfite, disulfite). A method is also particularly preferred, in which oligonucleotides are used for the amplification, which comprise a sequence segment of a chemically pretreated DNA which is at least 18 bases long according to one of the [sequences] Seq. ID<br><br><br>
1 to Seq. ID 40712^ It is assured that primers complementary to the bisulfite-treated DNA are used, which can amplify regulatory regions (CpG islands) which can then be investigated with respect to methylation.<br>
A method is also particularly preferred, in which, in a hybridization step, oligonucleotides and/or peptide nucleic acid (PNA) oligomers are used, which hybridize to a sequence segment that is at least 9 bases long of a chemically pretreated DNA according to one of the [sequences] Seq. ID 1 to Seq. ID 40712 or correspond to this segment, whereby the base sequence contains at least one CpG dinucleotide and the CpG dinucleotide is found in approximately the middle third of the oligomer. These oligonucleotides are suitable for investigating specific CpG positions with respect to their degree of methylation according to the method of the invention. They preferably bind to the amplified products of treated DNA, which originates from a genomic DNA sample methylated at the respective cytosine positions.<br>
It is also preferred that the labels are radionuclides.<br>
It is further preferred that the labels.are removable mass labels, which are detected in a mass spectrometer. It is particularly preferred according to the invention that the PCR products as a whole or their characteristic fragments are detected in the mass spectrometer and thus are clearly characterized by their mass.<br>
It is particularly preferred according to the invention that the oligomers (oligonucleotides and/or PNA oligomers) of one class contain the sequence 5"-CG-3It is particularly preferred according to the invention that the oligomers (oligonucleotides and/or PNA oligomers) of one class contain the sequence 5"-TG-3" and/or the sequence 5"-CA-3 <br><br>
It is also particularly preferred that the oligonucleotides of the first class contain the sequence 5"-CG-3" and the oligonucleotides of the second class contain the sequence 5"-TG-3" and/or the sequence 5"-CA-3"<br>
It is also preferred that the oligonucleotides of the first and of the second classes are immobilized on a common solid phase. It is also preferred that the oligonucleotides are arranged on a planar solid phase in a rectangular or hexagonal grid and the site of specific oligonucleotides on the solid phase is correlated with their respective sequence.<br>
It is also particularly preferred that the oligomers of the first and second classes are immobilized on beads, which are coded with a set of separately detectable labels. The latter serve for identifying the bead, i.e., the sequence bound to the bead in question. The amplified products bound to the beads are then identified by means of other labels, which are bound to the amplified products. Instruments for conducting such measurements based on beads are offered, for example, by the Luminex company.<br>
A method is most particularly preferred according to the invention, wherein step b) is conducted in two sub-steps as follows:<br>
a)	a PCR pre-amplification with at least one pair of primers of different sequence which hybridize nonspecifically to a DNA sample pretreated according to claim 1 and thus produce more than one amplified product in the PCR step;<br>
b)	a PCR amplification of the product formed in the pre-amplification, with primers of different sequence, which are each identical or inversely complementary to a segment of the DNA sample [(+) strandor (-) strand] that has undergone pretreatment according to claim 1, and hybridize specifically to the DNA to be amplified.<br>
In connection with this invention, hybridization is understood as a hybridization of<br><br><br>
two single DNA strands that are completely inversely complementary to each other according to Watson-Crick rules without the occurrence of an erroneous base pairing. Uracil is considered in this respect as thymine.<br>
It is also preferred according to the invention that the amplification of several DNA segments is conducted in one reaction vessel.<br>
It is further preferred according to the invention that a heat-stable DNA polymerase is used for the amplification. It is also particularly preferred that the primer oligonucleotides used for the amplification contain either only the bases T, A and C or the bases T, A and G.<br>
It is also preferred that at least 10 CpG positions in different sequence context are analyzed simultaneously. It is particularly preferred that at least 50 CpG positions in different sequence context are analyzed simultaneously. It is even more particularly preferred that at least 100 CpG positions in different sequence context are analyzed simultaneously. It is even more preferred that at least 500 CpG positions in different sequence context are analyzed simultaneously. It is most preferable that at least 1000 CpG positions in different sequence context are analyzed simultaneously.<br>
The method is preferred according to the invention, whereby the genomic DNA sample has been obtained from cell lines, blood, sputum, stool, urine, cerebrospinal fluid, tissue embedded in paraffin for example, tissue from eyes,<br>
intestine, kidney, brain, heartr rostate, lung, breast or liver, histological slides or<br>
all other possible combinations thereof.	<br>
The use of a method according to the invention is preferred for the diagnosis and/or prognosis of adverse events for patients or individuals, whereby these adverse events belong to at least one of the following categories: undesired drug interactions; cancer diseases; CNS malfunctions, damage or disease; symptoms<br><br><br>
of aggression or behavioral disturbances; clinical, psychological and social consequences of brain damage; psychotic disturbances and personality disorders; dementia and/or associated syndromes; cardiovascular disease, malfunction and damage; malfunction, damage or disease of the gastrointestinal tract; malfunction, damage or disease of the respiratory system; lesion, inflammation, infection, immunity and/or convalescence; malfunction, damage or disease of the body as an abnormality in the development process; malfunction, damage or disorder of the skin, the muscles, the connective tissue or the bones; endocrine and metabolic malfunction, damage or disease; headaches or sexual malfunctions.<br>
The use of a method according to the invention is also preferred for distinguishing cell types or tissues or for investigating cell differentiation.<br>
The subject of the present invention is [also] a kit comprising a reagent containing bisulfite, primer oligonucleotides for the production of the amplified products and/or preferably oligonucleotides immobilized to a solid phase as well as instructions for conducting the method according to the invention. The primer oligonucleotides and the immobilized oligonucleotides, as described above, are derived from the [sequences] Seq. IDs 1 to 40712.<br>
This genomic DNA sample has been obtained preferably from cell lines, blood, sputum, stool, urine, cerebrospinal fluid, tissue embedded in paraffin, for example, tissue from eyes, intestine, kidney, brain, heart, prostate, lung, breast or liver, histological slides or alfother possible combinations thereof.<br>
In this method in the first step, a genomic DNA samplejs treated in such a way that except for the 5-methylcytosine bases, all cytosine bases are converted to uracil. This chemical treatment is preferably conducted with the solution of a bisulfite (= hydrogen sulfite, disulfite). This step of the method can be conducted not only with the genomic DNA sample, but also preferably and logically with<br><br>
standard DNA in which it is known whether the cytosine at said specific position is present in methylated or unmethylated state.<br>
In the second step of the method, the segments of the genomic DNA that contain said specific cytosine are amplified. This step can be particularly preferably conducted in two sub-steps:<br>
1.	First, a PCR pre-amplification is conducted with at least one pair of primers of different sequence, which hybridize to a chemically pretreated DNA. This treatment was chemically conducted in such a way that the cytosine bases were converted to uracil, but not the 5-methylcytosine bases.<br>
2.	A PCR amplification of the product formed in the pre-amplification is conducted with primers of different sequence, These primers are identical or inversely complementary to a segment of the chemically pretreated DNA [(+) strand or (-) strand] and specifically hybridize to the DNA to be amplified. The amplified products preferably contain a detectable label.<br>
In the following third step of the method, a hybridization of the amplified products<br>
takes place on preferably two classes of oligonucleotides, each of which [class]<br>
has at least one member. In a particularly preferred variant of the method, the<br>
oligonucleotides of the first class contain the sequence 5"-CG-3" and the<br>
oligonucleotides of the second class contain the sequence 5-TG-3" and/or the<br>
sequence 5"-CA-3". The oligonucleotides of the first and the second classes are<br>
preferably immobilized on a common solid phase. The oligonucleotides are<br>
arranged on a planar solid phase in a rectangular or hexagonal grid and the site<br>
of specific oligonucleotides on the solid phase is correlated with their respective<br>
sequence.	 <br>
The oligonucleotides of the first class preferably hybridize to the sequence which arises from the chemical treatment of the genomic DNA if said specific cytosine<br><br><br>
was present in the methylated state in the genomic DNA. The oligonucleotides of the second class preferably hybridize to the sequence which arises from the chemical treatment of the genomic DNA if said specific cytosine was present in the unmethylated state in the genomic DNA.<br>
The amplification of several DNA segments is particularly preferably conducted in one reaction vessel. The amplification is preferably conducted with the polymerase chain reaction (PCR), wherein a heat-stable DNA polymerase is preferably used.<br>
The primer oligonucleotides used for the amplification contain preferably either only the bases T, A and C or the bases T, A and G.<br>
In the fourth step of the method, the extent of hybridization of the amplified products on the two classes of oligonucleotides is determined by detection,of the labels of the amplified products. The labels are particularly preferably fluorescent labels, radionuclides, or removable mass labels, which are detected in a mass spectrometer. The labels are preferably also detected by chemiluminescence, UV absorption or fluorescence polarization. The PCR products can also preferably be detected as a whole or as their characteristic fragments in the mass spectrometer. Thus the PCR products are clearly characterized by their mass.<br>
In the last step of the method, a conclusion is made on the extent of methylation of said specific cytosine in the genomic DNA sample from the ratio of the labels detected on the two classes of oligonucleotides as a consequence of the hybridization.<br>
The ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the unmethylated standard DNA, preferably serve as a calibration value for a degree of methylation of 0.<br><br><br>
Correspondingly, the ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the methylated standard DNA, preferably serve as a calibration value for a degree of methylation of 1. The calibration values are particularly preferably used for the determination of the degree of methylation of the genomic DNA samples.<br>
Preferably, additional known standard DNA samples, each of which has any known degree of methylation of said specific cytosine, are also used for the calibration.<br>
The method is further characterized in that preferably at least 10 CpG positions in different sequence context are analyzed simultaneously. In addition, preferably at least 50 CpG positions in different sequence context can be analyzed simultaneously. It is also preferred that at least 100 CpG positions in different sequence context are analyzed simultaneously. The simultaneous analysis of at.least 500 CpG positions in different sequence context is very much preferred. The simultaneous analysis of at least 1000 CpG positions in different sequence context is finally particularly preferred.<br>
The described method is preferably used for the diagnosis and/or prognosis of adverse events for patients or individuals, whereby thase adverse events belong to at least one of the following categories ndesired drug interactions; cancer diseases; CNS malfunctions, damage or disease; symptoms of aggression or behavioral disturbances; clinical, psychological and social consequences of brain damage; psychotic disturbances and personality disorders; dementia and/or associated syndromes; cardiovascular disease, malfunction and damage; malfunction, damage or disease of the gastrointestinal tract; malfunction, damage or disease of the respiratory system; lesion, inflammation, infection, immunity and/or convalescence; malfunction, damage or disease of the body as an abnormality in the development process; malfunction, damage or disorder of<br><br>
the skin, the muscles, the connective tissue or the bones; endocrine and metabolic malfunction, damage or disease; headaches or sexual malfunctions.<br>
The present method is particularly preferably used for distinguishing cell types or tissues or for investigating cell differentiation.<br>
By determining the hybridization ratios between the two classes of oligonucleotides utilized (e.g., containing CG/TG), the method is not dependent on the intensity of the total hybridization of unknown tissue samples.<br>
Unmethylated and methylated reference samples are utilized as standards for calibrating unknown tissue samples.<br>
A component of this method is also a kit, which comprises a reagent containing bisulfite, primer oligonucleotides for the production of amplified products and/or preferably oligonucleotides immobilized on a solid phase. The oligonucleotides (first class) comprise the sequence 5"-CG-3  The oligonucleotides (second class) comprise the sequence 5"-TG-3" and/or the sequence 5"-CA-3". Instructions for conducting the method are also included in the kit.<br>
The subject of the present invention is also nucleic acids that are particularly suitable for conducting the method.<br>
The subject of the invention is also a set of at least 10 oligomer probes (oligonucleotides and/or PNA oligomers), which serve for the detection of the cytosine methylation,state in chemically pretreated genomic DNA (Seq. ID 1 to Seq. ID 40712). The analysis of a set of genetic and/or epigenetic parameters for the diagnosis of existing diseases or for the diagnosts of predisposition to specific diseases is possible with these probes.<br>
The subject of the present invention is also a sequence segment of a treated<br><br><br>
DNA which is at least 18 bases long according to one of the [sequences] Seq. ID 1 to Seq. ID 40712. These segments of 18 base pairs in length comprised of Seq. ID 1 to Seq. ID 40712 are utilized for the amplification of the treated genomic DNA. Oligomers with a length of at least 9 nucleotides are used as detectors of these segments.<br>
The oligomers preferably contain at least one CpG dinucleotjde. The cytosine of the corresponding CpG dinucleotide is found in approximately the middle third of the oligomer. It is a deciding factor that at least one oligonucleotide from Seq. ID 1 to Seq. ID 40712 is present in the respective set of oligomers for at least each of the CpG dinucleotides.<br>
The oligomers are preferably produced on a support material in a fixed arrangement, whereby at least one oligomer is coupled to a solid phase. Methods for binding oligomer probes to solid phases are known to the person of average skill in the art.<br>
It is also important in this connection that it is not individual CpG dinucleotides, but the large number of CpG dinucleotides present in the sequences, which must be analyzed for the diagnosis of genetic and/or epigenetic parameters of the claimed nucleic acids (Seq. ID 1 to Seq. ID 40712). In a particularly preferred variant of the method, all of the CpG dinucleotides present in the sequences are to be investigated.<br>
It is further preferred that all oligomer probes have the same length. In addition,<br>
all 18-mer [segments] which have a CpG dinucleotide in the center and which<br>
hybridize to one of the Seq. ID 1 to Seq. ID 40712 without erroneous base<br>
pairing are particularly preferred.	 <br>
In another preferred variant of the method, at least ten of the oligomers are used for the detection of the cytosine methylation state and/or of single nucleotide<br><br><br>
polymorphisms (SNPs) in chemically pretreated genomic DNA .<br>
The oligomers are preferably used for the diagnosis of undesired drug interactions; cancer diseases; CNS malfunctions, damage or diseases; symptoms of aggression or behavioral disturbances; clinical, psychological and social consequences of brain lesions; psychotic disturbances and personality disorders; dementia and/or associated syndromes; cardiovascular disease; malfunction, damage or disorder of the gastrointestinal tract; malfunction, damage or disorder of the respiratory system; lesion, inflammation, infection, immunity and/or convalescence; malfunction, damage or disorder of the body as an abnormality in the development process; malfunction, damage or disorder of the skin, the muscles, the connective tissue or the bones; endocrine and metabolic malfunction, damage or disorder; headaches and sexual malfunctions, by analysis of methylation patterns.<br>
Also, of the nucleic acids or considerable segments thereof listed in the sequence protocol (Seq. ID 1 to Seq. ID 40712), preferably at least one will be used for the analysis of a set of genetic and/or epigenetic parameters for the diagnosis of existing disorders or for the diagnosis of predisposition for specific disorders.<br>
The person of average skill in the art understands that the oligomers fulfill the same objective when thymine is exchanged for uracil.<br>
The genomic DNA to be analyzed is obtained preferably from the usual sources, for DNA, such as, e.g., cell lines, blood, sputum, stool, urine, cerebrospinal fluid, tissue embedded in paraffin, for example, tissue from eyes intestine, kidney, brain, heart, prostate, lung, breast or liver, histological slides and all other possible combinations thereof.<br>
The subject of the present invention is also nucleic acids containing a sequence<br><br><br>
segment which is at least 18 bases long of a chemically pretreated DNA, according to one of the [sequences] Seq. ID 1 to Seq. ID 40712.<br>
The subject of the present invention is also an oligomer (oligonucleotide or peptide nucleic acid (PNA) oligomer) for the detection of the cytosine methylation state in chemically pretreated DNA, each containing at least one base sequence with a length of at least 9 nucleotides, which hybridizes to a chemically pretreated DNA (Seq. ID 1 to Seq. ID 40712). It is also preferred according to the invention that the base sequence contains at least one dinucleotide. It is also preferred that the cytosine of the CpG dinucleotide is found in approximately the middle third of the oligomer.<br>
The subject of the invention is also a set of oligomers according to the invention, containing at least one oligomer for at least one of the CpG dinucleotides of one of the sequences of Seq. ID 1 to Seq. ID 40712. A set of oligomers containing at least one oligomer for each of the CpG dinucleotides of one of the sequences of Seq. ID 1 to Seq. ID 40712 is preferred.<br>
The subject of the present invention is also a set of at least two nucleic acids, which are utilized as primer oligonucleotides for the amplification according to the invention of at least one of the [sequences] Seq. ID 1 to Seq. ID 40712 or segments thereof. It is preferred that at least one oligonucleotide is bound to a solid phase.<br>
The subject of the present invention is also a set of oligomer probes for the detection of the cytosine methylation state and/or of single nucleotide polymorphisms (SNPs) in chemically pretreated genomic- DNA according to one of the [sequences] Seq. ID 1 to Seq. ID 40712, containing at least ten of the above-named oligomers according to the invention.<br>
The subject of the present invention is also a method for the production of an<br><br><br>
arrangement of different oligomers (an array) fixed on a support material for the analysis of disorders related to the methylation state of the CpG dinucleotides of one of the [sequences] Seq. ID 1 to Seq. ID 40712, in which at least one oligomer according to the invention is coupled to a solid phase.<br>
The subject of the invention is also arrangements of different oligomers (array) bound to a solid phase.<br>
The subject of the present invention is also an array of different oligonucleotide and/or PNA oligomer sequences whereby these are arranged on a planar solid phase in the form of a rectangular or hexagonal grid. It is preferred that the solid phase surface is comprised of silicon, glass, polystyrene, aluminum, steel, iron, copper, nickel, silver, or gold.<br>
According to the invention, a DNA and/or PNA array is also [included] for the analysis of disorders related to the methylation state of genes, which contains at least one nucleic acid as described above according to the invention.<br>
The following examples explain the invention.<br>
Example 1:<br>
Production of unmethylated and methylated DNA and bisulfite treatment<br>
For the production of methylated DNA, human genomic DNA was treated with S-<br>
adenosylmethionine und CpG methylase (Sssl, New England Biolabs,)<br>
according to the information of the manufacturer. For the production of<br>
unmethylated DNA, the gene fragment ELK-1 (Accession numnber ep5901-1) was<br>
amplified by means of PCR with the primers	 <br>
GCTCTATGGTCTTGTCTAACCGTA and AGGTGGTGGTGGCGGTGG, starting from human genomic DNA. The unmethylated and methylated DNA, which was prepared in this way, as well as also the human genomic DNA was treated with<br><br><br>
the use of bisulfite (hydrogen sulfite, disulfite), such that all cytosines unmethylated at the 5-position of the base are changed so that a base that is different with respect to base pairing behavior is formed, whereas the cytosines that are methylated in the 5-position remain unchanged. If bisulfite in the concentration range between 0.1 M and 6 M is used for the reaction, then an addition occurs at the unmethylated cytosine bases. Also, a denaturing reagent or solvent as well as a radical trap must be present. A subsequent alkaline hydrolysis then leads to the conversion of unmethylated cytosine nucleobases to uracil. This converted DNA serves for the detection of methylated cytosines.<br>
Example 2:<br>
Production of Cy5-labeled gene probes<br>
Starting with DNA samples treated with bisulfite, a defined fragment of 529 bp in length from the promoter region of the ELK-1 gene (Accession number ep59011) was amplified. The amplification is conducted with the primer oligonucleotides ATGGTTTTGTTTAATYGTAGAGTTGTTT and<br>
TAAACCCRAAAAAAAAAAACCCAATAT. By using primer oligonucleotides that   are labeled with the fluorescent dye Cy5, the fragment is directly labeled in the PCR. (1) Unmethylated DNA, (2) methylated DNA and (3) human genomic DNA treated with bisulfite (hydrogen sulfite, disulfite) are used as the matrix DNA. Then these three different DNA fragments are investigated in separate hybridizations for their degree of methylation at a specific CpG position.<br>
Example 3:<br>
Conducting the hybridization and evaluating a hybridized DNA chip.<br>
The gene probes prepared in Example 2 are hybrfdized to a DNA chip. First, oligonucleotides are immobilized on the chip. The oligonucleotide sequences are derived from the amplified fragment of the ELK-1 gene named in Example 2, and represent the CG dinucleotide, including the immediate surroundings. The length<br><br><br>
of the oligonucleotides amounts to 14-22 nucleotides; the position of the CG dinucleotide within the oligonucleotide is variable. After the hybridization, the DNA chip is scanned (see Fig. 1) and the hybridization signals are numerically evaluated (data not shown). The result of the hybridization for the oligonucleotides CTACTCAACGAAAACAAA and CTACTCAACAAAAACAAA is shown in Fig. 1 and Fig. 2. CTACTCAACGAAAACAAA preferably hybridizes if the cytosine of the ELK-1 fragment, which is found at position 103 of the amplified product, is methylated; CTACTCAACAAAAACAAA hybridizes if this cytosine is unmethylated.<br>
A DNA chip is shown in Figure 1 after hybridization with the ELK-1 fragment. The pseudo-color image as it is produced after scanning is shown. Unlike the black-and-white illustration shown here, a color image is produced by the scanner. The intensity of the different colors represent the degree of hybridization, whereby the degree of hybridization decreases from red (this can be recognized as light spots in Figure 1) to blue (recognized as dark spots in Figure 1).<br>
Figure 2 A shows an excerpted image from Figure 1. The spotted pairs of oligonucleotides are circled in white to clarify the hybridization diagram: ctactcaacaaaaacaaa (left) and ctactcaacgaaaacaaa (right).<br>
The excerpted image of Figure 2 B shows the spotting pattern for an unknown methylation state of the tissue sample and the excerpted image of Figure 2 C shows the methylation state for the methylated reference sample.<br>
*s <br><br>
The mean is indicated each time for a wavelength of 635 nm. Column A gives the values for the unmethylated [reference] sample and column B for an unknown methylation state of a tissue sample and column C gives the values for the methylated reference sample. The CG/CA ratios represent the methylatioq state of the respective sample. The value of 0.74 shows that the sample is basically present in methylated form.<br>
Example 4:<br>
The following example relates to a fragment of the hMLH1 gene associated with hereditable non-polyposis colorectal cancer, in which a specific CG position is investigated for methylation.<br>
In the first step, a genomic sequence is treated with the use of bisulfite (hydrogen sulfite, disulfite) in such a way that all of the unmethylated cytosines at the 5-position of the base are modified such that a base that is different in its base pairing behavior is formed, while the cytosines that are rpejiylated in the.5-pdsition remain unchanged. If bisulfite in the concentration range between 0.1 M and 6 M is used for the reaction, then an addition occurs at the unmethylated cytosine bases. Also, a denaturing reagent or solvent as well as a radical trap must be present. A subsequent alkaline hydrolysis then leads to the conversion<br><br><br>
of unmethylated cytosine nucleobases to uracil. This converted DNA serves for the detection of methylated cytosines. In the second step of the method, the treated DNA sample is diluted with water or an aqueous solution. A desulfonation of the DNA (10-30 min, 90-100 C) at alkaline pH is then preferably conducted. In the third step of the method, the DNA sample is amplified in a polymerase chain reaction, preferably with a heat-stable DNA polymerase. In the present Example, cytosines of the hMLH1 gene, here from a 1551-bp-long 5"-flanking region, are investigated. A defined fragment of 719-bp length is amplified for this purpose with the specific primer oligonucleotides AGCAACACCTCCATGCACTG and TTGATTGGACAGCTTGAATGC. This amplified product serves as a sample, which hybridizes to an oligonucleotide that has been previously bound to a solid phase, with the formation of a duplex structure, for example, GAAGAGCGGACAG, whereby the cytosine to be detected is found at position 588 of the amplified product. The detection of the hybridization product is based on primer oligonucleotides fluorescently labeled with Cy3 and Cy5, which were used for the amplification. A hybridization reaction of the amplified DNA with the oligonucleotide occurs only if a methylated cytosine was present at this site in the bisulfite-treated DNA. Thus the methylation state of the respective cytosine to be investigated decides the hybridization product.<br>
Example 5;<br>
Production of bisulfite-modified DNA with agarose beads<br>
In the present experiment, starting with DNA treated with bisulfite, a defined<br>
fragment of 529 bp in length from the promoter region of the ELK-1 gene<br>
(Accession number ep59011) is amplified. By using primer oligonucleotides<br>
ATGGTTTTGTTTAATYGTAGAGTTGTTT and	 <br>
TAAACCCRAAAAAAAAAAACCCAATAT, which are" labeled with the fluorescent dye ALEXA 488, the fragment is directly labeled in the PCR. Oligonucleotides of the first class (here, for example, ATTAATAGCGTTTTGGTT) and of the second class (here: for example, ATTAATAGTGTTTTGGTT) are immobilized at the<br><br><br>
surface of beads, which are distinguished by an individual color coding. In a following step, the amplified products of the ELK-1 gene, which were prepared with the above-named primer oligonukleotides, are combined with a mixture of both classes of beads, whereby the amplified products hybridize to the immobilized oligonucleotides, here, for example, ATTAATAGCGTTTTGGTT and ATTAATAGTGTTTTGGTT, whereby the C or T to be detected is found each time at position 476 of the amplified product. Then the beads are separated, identified by fluorescence measurement based on their color coding and the degree of hybridization is determined by measurement of the fluorescent intensities, which are specific to the fluorescent dye ALEXA 488.<br>
Example 6:<br>
Use of multiple dyes for internal calibration<br>
Starting with bisulfite-treated DNA, a defined fragment of 529-bp length from the promoter region of the ELK-1 gene is amplified. The fragment is labeled directly in the PCR by use of the fluorescently labeled primer oligonucleotides ATGGTTTTGTTTAATYGTAGAGTTGTTT and<br>
TAAACCCRAAAAAAAAAAACCCAATAT. For the PCR reaction, which is conducted on a thermocycler (Eppendorf GmbH), 10 ng of bisulfite-treated DNA," 6 pmol of each primer, 200 uM of each dNTP, 1.5 mM MgC12 and 1 U of HotstartTaq (Qiagen AG) are used. The other conditions are selected following the manufacturer"s information. For the amplification, a denaturing is conducted for 14 min at 96C, followed by 39 cycles with the conditions: 60 sec at 96C, 45 sec at 55 C and 75 sec at 72 C. In conclusion, an elongation is conducted for 10 min at 72C. In the present case, the samples to be investigated, here the tissue of healthy and sick persons, are labeled with they2-dye. For the internal calibration of the methylation state, primer oligonucleotides are used for the amplification of samples for the calibration, which are labeled with the fluorescent dyes Cy3 and Cy5. The amplified products from the samples for the calibration represent a known methylation state, on the one hand, a state of one-hundred<br><br><br>
percent methylation, and, on the other hand, an unmethylated state. Since in this method, the ratios of the color intensities of the fluoresent dyes, which are measured with the use of the ScanArray 4000XL, Packard BioScience-BioChip Technologies, are calculated for two classes of oligonucleotides, here, for example, ATTAATAGCGTTTTGGTT and ATTAATAGTGTTTTGGTT, wherein the C or T to be detected is found each time at position 476 of the amplified product, the ratio of methylated to unmethylated state of the unknown sample can be determined.<br>
Example 7:<br>
Use of multiple dyes for increasing the sample throughput volume and for<br>
increasing the complexity of the analysis.<br>
The present experiment serves for the purpose of analyzing different samples in a single hybridization step and in this way increasing the sample throughput volume. For this purpose, a defined fragment of 529-bp length from the promoter region of the ELK-1 gene taken from each of four individuals and, starting with the bis"ulfitertreated DNA, is amplified with the primer oligonucleotides ATGGTTTTGTTTAATYGTAGAGTTGTTT and<br>
TAAACCCRAAAAAAAAAAACCCAATAT. These fragments originating from four individuals are labeled with the four different fluorescent dyes Cy3, Cy5, Cy2 und Cy7 and hybridized to immobilized oligonucleotides, here, for example, ATTAATAGCGTTTTGGTT and ATTAATAGTGTTTTGGTT, whereby the C or T to be detected is found each time at position 476 of the amplified product The samples with different fluorescent labels are then analyzed at different wavelengths without a mutual interference based on the fluorescent dye.<br>
On the other hand, it is also possible to produce different sets of fragments from one DNA sample, which [fragments] are labeled with different dyes, here Cy2, Cy3 and Cy5. If a set of oligonucleotide probes for 64 genes (set 1) is immobilized on a chip, then the specificity is sufficient in order to analyze 64<br><br><br>
fragments, e.g., labeled with Cy3, independent of one another. Due to the fact that samples of set 1 are labeled here with the fluorescent dye Cy3 and samples of set 2 are labeled with the fluorescent dye Cy5 (and set 3 with Cy2), the detection of the methylation state via measuring the fluoresent intensities of the fragments of set 1 is not influenced by the fluorescently labeled amplified products of sets 2 and 3 (and vice versa). Therefore, despite the increased complexity of the amplified products, it is possible to produce data that are equally reliable to those for a [lesser] complexity of 64 amplified products.<br>
Example 8:<br>
Use of multiple dyes for verifying experimental reproducibility<br>
In the present experiment, the same PCR amplified products are labeled with four different fluorescent dyes and these are verified [for reliability] by a 4X redundancy. For this purpose, starting from bisulfite-treated DNA, a a defined fragment of 529-bp length from the promoter region of the ELK-1 gene is amplified with the primer oligonucleotides ATGGTTTTGTTTAATYGTAGAGTTGTTT and<br>
TAAACCCRAAAAAAAAAAACCCAATAT, and hybridized to immobilized oligonucleotides, here, for example, ATTAATAGTGTTTTGGTT und ATTAATAGTG I I I I GGTT, whereby the C or T to be detected is found each time at position 476 of the amplified product. The ratios of samples with different fluorescent labels are compared by employing primer oligonucleotides fluorescently labeled with Cy3, Cy5, Cy2 and Cy7, and in this way a higher experimental reliability is achieved.<br><br><br>
WE CLAIM:<br>
1.	An in vitro method for the detection of the degree of methylation of a specific cytosine<br>
in the sequence context 5"-CpG-3" of a genomic DNA sample, is hereby characterized in<br>
that<br>
a)	the genomic DNA is chemically treated, whereby the cytosine bases are converted to uracil, but not the 5-methylcytosine bases;<br>
b)	segments of the genomic DNA, which contain said specific cytosine, are amplified, whereby the amplified products are given a detectable label;<br>
c)	the amplified products are hybridized to two classes of oligonucleotides and/or PNA oligomers, each of which class has at least one member;	 <br>
d)	the extent of hybridization of the amplified products to the two classes of<br>
oligonucleotides and/or PNA oligomers is determined by detection of the label of the<br>
amplified products;<br>
e)	the extent of methylation of said specific cytosine in the genomic DNA sample from<br>
the ratio of the labels detected for the two classes of oligonucleotides and/or PNA<br>
oligomers are as a consequence of the hybridization.<br>
2.	The method as claimed in claim 1, optionally in that a hybridization of the amplified products is conducted in step c) on two classes of oligomers (oligonucleotides and/ or PNA oligomers), each of which class has at least one member, whereby the oligomers of the first class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the methylated state in the genomic DNA and whereby the oligomers of the second class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the unmethylated state in the genomic DNA.<br>
3.	The method as claimed in claim 1, optionally in that a hybridization of the amplified products is conducted in step c) on two classes of oligomers (oligonucleotides and/ or PNA oligomers), each of which class has at least one member, whereby the oligomers<br><br><br>
of the first class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the methylated state in the genomic DNA and less preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the unmethylated state in the genomic DNA, and whereby the oligomers of the second class hybridize to the amplified product to be investigated essentially independently of the degree of methylation of said specific cytosine in the genomic DNA.<br>
4.	The method as claimed in claim 1, optionally in that a hybridization of the amplified products is conducted in step c) on two classes of oligomers (oligonucleotides and/or PNA oligomers), each of which class has at least one member, whereby the oligomers of the first class preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the unmethylated state in the genomic PNA and less preferably hybridize to the sequence which arises after the chemical treatment of the genomic DNA, if said specific cytosine was present in the methylated state in the genomic DNA, and whereby the oligomers of the second class hybridize to the amplified product to be investigated essentially independently of the degree of methylation of said specific cytosine in the genomic DNA.<br>
5.	The method as claimed in claim 1, optionally in that the method is conducted not only with the genomic DNA sample, but also logically with standard DNA in which it is known whether the cytosine at said specific position is present in methylated or unmethylated state, whereby the ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the unmethylated standard DNA according to claim 1, serve as a calibration value for a degree of methylation of 0, and correspondingly, the ratios of the labels detected on the two classes of oligonucleotides, which are measured each time with the methylated standard DNA according to claim 1, serve as a calibration value for a degree of methylation of 1, and these calibration<br><br><br>
values are used for the determination of the degree of methylation of the genomic DNA sample.<br>
6.	The method as claimed in claim 5, optionally in that additional known standard DNA samples, each of which has any known degree of methylation of said specific cytosine, are used for calibration.<br>
7.	The method as claimed in claim 5, optionally in that the DNAs used as the standard are each labeled differently and the amplified product from the genomic DNA is provided in turn with a label that is different therefrom.<br>
8.	The method as claimed in claim 1, optionally in that amplified products that are derived from different genomic DNA samples are provided with different labels.<br>
9.	The method as claimed in claim 1, optionally in that amplified products originating from the same genomic DNA samples are provided with different labels in order to achieve an increase of measurement accuracy by an averaging of the values obtained from different detection methods.<br>
10.	The method as claimed in claim 1, optionally in that said labels are fluorescent labels.<br>
11.	The method as claimed in claim 1, Optionally in that the chemical treatment is<br>
conducted with a solution of a bisulfite (=hydrogen sulfite, disulfite).<br>
12.	The method as claimed in claim 1, optionally in that oligonucleotides are used for the<br>
amplification, which comprise a sequence segment of a chemically pretreated DNA<br>
which is at least 18 bases long of one of the sequences Seq. ID 1 to Seq. ID 40712.<br>
13.	The method as claimed in claim 1, optionally in that in a hybridization step,<br>
oligonucleotides and/or peptide nucleic acid (PNA) oligomers are used, which<br>
hybridize to a sequence segment that is at least 9 bases long of a chemically pretreated<br><br><br>
DNA according to one of the sequences Seq. ID 1 to Seq. ID 40712 or correspond to this sequence, whereby the base sequence contains at least one CpG dinucleotide and the CpG dinucleotide is found in approximately the middle third of the oligomer.<br>
14.	The method as claimed in claim 1, optionally in that said label is detected by its<br>
hemiluminescence, its UV absorption or fluorescence polarization.<br>
 15.	The method as claimed in claim 1, optionally in that the labels are radionuclides.<br>
16.	The method as claimed in claim 1, optionally in that the labels are removable mass labels, which are detected in a mass spectrometer.<br>
17.	The method as claimed in claim 1, optionally in that the PCR products as a whole or their characteristic fragments are detected in the mass spectrometer and thus are clearly characterized by their mass.<br>
18.	The method as claimed in claim 1, optionally in that the oligomers (oligonucleotides and/or PNA oligomers) of one class contain the sequence 5"-CG-3".<br>
19.	The method as claimed in claim 1, optionally in that the oligomers (oligonucleotides and/or PNA oligomers) of one class contain the sequence 5"-TG-3" and/or the sequence<br>
5"-CA-3".<br>
20.	The method as claimed in claims 18 or 19, optionally in that the oligomers<br>
(oligonucleotides and/or PNA oligomers) of the first class contain the sequence 5"-CG-<br>
3" and the oligomers of the second class contain the sequence 5"-TG-3" and/or the<br>
sequence 5"-CA-3".<br>
21.	The method as claimed in claim 1, optionally in that the oligomers of the first and the<br>
second classes are immobilized on a common solid phase.<br><br><br>
22.	The method as claimed in claim 11, optionally in that the oligonucleotides are arranged on a planar solid phase in a rectangular or hexagonal grid and the site of specific oligonucleotides on the solid phase is correlated with their respective sequence.<br>
23.	The method as claimed in claim 1, optionally in that the oligomers of the first and second classes are immobilized on beads, which are coded with a set of separately detectable labels.<br>
24.	The method as claimed in claim 1, optionally in that step b) is conducted in two sub-steps as follows:<br><br>
a)	a PCR pre-amplification with at least one pair of primers of different sequence which hybridize nonspecifically to a DNA sample pretreated according to claim 1 and thus produce more than one amplified product in the PCR step;<br>
b)	a PCR amplification of the product formed in the pre-amplification, with primers of different sequence, which are each identical or inversely complementary to a segment of the DNA sample ((+) strand or (-) strand) that has been pretreated according to claim 1, and hybridize specifically to the DNA to be amplified.<br><br>
25.	The method as claimed in claim 1, optionally in that the amplification of several DNA segments is conducted in one reaction vessel.<br>
26.	The method as claimed in claim 1, optionally in that a heat-stable DNA polymerase is used for the amplification.<br>
27.	The method as claimed in claim 1, optionally in that the primer oligonucleotides used for the amplification contain either only the bases T, A and C or the bases T, A and G.<br>
28.	The method as claimed in claim 1, optionally in that at least 10 CpG positions in different sequence context are analyzed simultaneously.<br>
39<br><br>
29.	The method as claimed in claim 1, Optionally in that at least 50 CpG positions in<br>
different sequence context are analyzed simultaneously.<br>
6<br>
30.	The method as claimed in claim 1, Optionally in that at least 100 CpG positions in different sequence context are analyzed simultaneously.<br>
31.	The method as claimed in claim 1, optionally in that at least 500 CpG positions in different sequence context are analyzed simultaneously.<br>
r<br>
32.	The method as claimed in claim 1, optionally in that at least 1000 CpG positions in<br>
different sequence context are analyzed simultaneously.<br>
33.	The method as claimed in claim 1, whereby the genomic DNA sample has been<br>
obtained from cell lines, blood, sputum, stool, urine, cerebrospinal fluid, tissue<br>
embedded in paraffin, for example, tissue from eyes, intestine, kidney, brain, heart,<br>
prostate, lung, breast or liver, histological slides or all other possible combinations<br>
thereof.<br><br>
HIRAL CHANDRAKANT JOSHl AGENT FOR EPIGENOMICS AG<br>
Dated this 30th day of January, 2003.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtYWJzdHJhY3QoMjktMDctMjAwNCkuZG9j" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-abstract(29-07-2004).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtYWJzdHJhY3QoMjktMDctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-abstract(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtY2FuY2VsbGVkIHBhZ2UoMjktMDctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-cancelled page(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtY2xhaW1zIChncmFudGVkKS0oMjktMDctMjAwNCkuZG9j" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-claims (granted)-(29-07-2004).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtY2xhaW1zIChncmFudGVkKS0oMjktMDctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-claims (granted)-(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoMjMtMDMtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-correspondence(23-03-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoaXBvKS0oMjItMDItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-correspondence(ipo)-(22-02-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZHJhd2luZygyOS0wNy0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-drawing(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSAxOSgxNy0wMi0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 19(17-02-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSAxYSgyOS0wNy0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 1a(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgyOS0wNy0yMDA0KS5kb2M=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 2(granted)-(29-07-2004).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgyOS0wNy0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 2(granted)-(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSAzKDI5LTA3LTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 3(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybSA1KDI5LTA3LTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form 5(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtZm9ybS1wY3QtaXNhLTIxMCgzMS0wMS0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-form-pct-isa-210(31-01-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU1LW11bW5wLTIwMDMtcG93ZXIgb2YgYXR0b3JuZXkoMjktMDctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">155-mumnp-2003-power of attorney(29-07-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="209962-large-size-manipulator-especially-for-truch-mounted-concrete-pumps-and-a-method-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="209964-hmg-c0a-reductase-inhibiting-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>209963</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>155/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>38/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Sep-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Jan-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EPIGENOMICS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KASTANIENALLEE 24, D-10435 BERLIN, GERMANY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OLEK, ALEXANDER</td>
											<td>SCHRODERSTRASSE 13, D-10115 BERLIN, GERMANY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PIEPENBROCK, CHRISTIAN</td>
											<td>SCHWARTZKOPFFSTRASSE 7B, 10115 BERLIN, GERMANY</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BERLIN, KURT</td>
											<td>MARIENKAFFRWEG 4, 14532 STAHNSDORF, GERMANY</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GUTIG, DAVID</td>
											<td>KASTANIENALLEE 74, 10435 BERLIN, GERMANY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP01/10074</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-09-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>100 43 826.1</td>
									<td>2000-09-01</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/209963-method-for-determining-the-degree-of-methylation-of-defined-cytosines-in-genomic-dna-in-the-sequence-context-5-cpg-3 by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:33:07 GMT -->
</html>
