<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/210084-an-antibody-molecule-capable-of-recognizing-two-regions-of-a4-peptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 210084:AN ANTIBODY MOLECULE CAPABLE OF RECOGNIZING TWO REGIONS OF β-A4 PEPTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ANTIBODY MOLECULE CAPABLE OF RECOGNIZING TWO REGIONS OF β-A4 PEPTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to antibody molecules capable of specifically recognizing two regions of the R-A4 peptide, wherein the first region comprises the amino acid sequence AEFRHDSGY as shown in SEQ ID NO: 1 or a fragment thereof and wherein the second region comprises the amino acid sequence VHHQKLVFFAEDVG as shown in SEQ ID NO: 2 or a fragment thereof. Furthermore, nucleic acid molecules encoding the inventive antibody molecules and vectors and hosts comprising said nucleic acid molecules are disclosed. In addition, the present invention provides for compositions, preferably pharmaceutical or diagnostic compositions, comprising the compounds of the invention as well as for specific uses of the antibody molecules, nucleic acid molecules, vectors or hosts of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Anti-Aβ antibodies and their use<br>
The present invention relates to antibody molecules capable of specifically recognizing two regions of the P-A4 peptide, wherein the first region comprises the amino acid sequence AEFRHDSGY as shown in SEQ ID NO: 1 or a fragment thereof and wherein the second region comprises the amino acid sequence VHHQKLVFFAEDVG as shown in SEQ ID NO: 2 or a fragment thereof. Furthermore, nucleic acid molecules encoding the inventive antibody molecules and vectors and hosts comprising said nucleic acid molecules are disclosed. In addition, the present invention provides for compositions, preferably pharmaceutical or diagnostic compositions, comprising the compounds of the invention as well as for specific uses of the antibody molecules, nucleic acid molecules, vectors or hosts of the invention.<br>
Several documents are cited throughout the text of this specification. Each of the documents cited herein (including any manufacturers specifications, instructions, etc.) are hereby incorporated by reference.<br>
About 70% of all cases of dementia are due to Alzheimer's disease which is associated with selective damage of brain regions and neural circuits critical for cognition. Alzheimer's disease is characterized by neurofibrillary tangles in particular in pyramidal neurons of the hippocampus and numerous amyloid plaques containing mostly a dense core of amyloid deposits and defused halos.<br>
The extracellular neuritic plaques contain large amounts of a pre-dominantly fibrillar peptide termed "amyloid β", "A-beta", "Aβ4", "β-A4" or "Aβ"; see Selkoe (1994), Ann. Rev.  Cell  Biol.  10,  373-403,  Koo (1999),   PNAS Vol.  96,  pp.  9989-9990,  US<br><br>
4,666,829 or Glenner (1984), BBRC 12, 1131. This amyloid β is derived from "Alzheimer precursor protein/β-amyloid precursor protein" (APP). APPs are integral membrane glycoproteins (see Sisodia (1992), PNAS Vol. 89, pp. 6075) and are endoproteolytically cleaved within the Aβ sequence by a plasma membrane protease, a-secretase (see Sisodia (1992), loc. cit). Furthermore, further secretase activity, in particular β-secretase and y-secretase activity leads to the extracellular release of amyloid-P (A|3) comprising either 39 amino acids (Ap39), 40 amino acids (Aβ40), 42 amino acids (Aβ42) or 43 amino acids (Aβ43); see Sinha (1999), PNAS 96, 11094-1053; Price (1998), Science 282, 1078 to 1083; WO 00/72880 or Hardy (1997), TINS 20, 154.<br>
It is of note that Aβ has several naturally occurring forms, whereby the human forms are referred to as the above mentioned Ap39, AP40, Ap41, Ap42 and Ap43. The most prominent form, Ap42, has the amino acid sequence (starting from the N-terminus): DAEFRHDSGYEVHHQKLVFFAEDVGSNKGAIIGLMVGGWIA (SEQ ID NO: 27). In Ap41, Ap40, Ap39, the C-terminal amino acids A, lA and VIA are missing, respectively. In the Ap43-form an additional threonine residue is comprised at the C-terminus of the above depicted sequence (SEQ ID NO: 27).<br>
The time required to nucleate Ap40 fibrils was shown to be significantly longer than that to nucleate Ap42 fibrils; see Koo, loc. cit. and Harper (1997), Ann. Rev. Biochem. 66, 385-407. As reviewed in Wagner (1999), J. Clin. Invest. 104, 1239-1332, the Ap42 is more frequently found associated with neuritic plaques and is considered to be more fibrillogenic in vitro. It was also suggested that Ap42 serves as a "seed" in the nucleation-dependent polymerization of ordered non-crystalline Ap peptides; Jarrett (1993), Cell 93, 1055-1058.<br>
It has to be stressed that modified APP processing and/or the generation of extracellular plaques containing proteinaceous depositions are not only known from Alzheimer's pathology but also from subjects suffering from other neurological and/or neurodegenerative disorders. These disorders comprise, inter alia, Down's syndrome, Hereditary cerebral hemorrhage with amyloidosis Dutch type, Parkinson's<br><br>
disease, ALS (amyotrophic lateral sclerosis), Creutzfeld Jacob disease, HIV-related dementia and motor neuropathy.<br>
In order to prevent, treat and/or ameliorate disorders and/or diseases related to the pathological deposition of amyloid plaques, means and methods have to be developed which either interfere with (3-amyloid plaque formation, which are capable of preventing Aβ aggregation and/or are useful in de-polymerization of already formed amyloid deposits or amyloid-(3 aggregates.<br>
Accordingly, and considering the severe defects of modified and/or pathological amyloid biology, means and methods for treating amyloid related disorders are highly desirable. In particular, efficient drugs which either interfere with pathological amyloid aggregation or which are capable of de-polymerization of aggregated Ap are desired. Furthermore, diagnostic means are desirable to detect, inter alia, amyloid plaques.<br>
Thus, the technical problem of the present invention is to comply with the needs described herein above.<br>
Accordingly, the present invention relates to an antibody molecule capable of specifically recognizing two regions of the β-A4/Aβ4 peptide, wherein the first region comprises the amino acid sequence AEFRHDSGY (SEQ ID NO; 1) or a fragment thereof and wherein the second region comprises the amino acid sequence VHHQKLVFFAEDVG (SEQ ID NO: 2) or a fragment thereof.<br>
In context of the present invention, the term "antibody molecule" relates to full immunoglobulin molecules, preferably IgMs, IgDs, IgEs, IgAs or IgGs, more preferably lgG1, lgG2a, lgG2b, lgG3 or !gG4 as well as to parts of such immunoglobulin molecules, like Fab-fragments or VL-, VH- or CDR-regions. Furthermore, the term relates to modified and/or altered antibody molecules, like chimeric and humanized antibodies. The term also relates to modified or altered monoclonal or polyclonal antibodies as well as to recombinantly or synthetically generated/synthesized antibodies. The term also relates to intact antibodies as well<br><br>
as to antibody fragments/parts thereof, like, separated light and heavy chains, Fab, Fab/c, Fv, Fab', F(ab')2. The ternn "antibody molecule" also connprises antibody derivatives, the bifunctional antibodies and antibody constructs, like single chain Fvs (scFv), bispecific scFvs or antibody-fusion proteins. Further details on the term "antibody molecule" of the invention are provided herein below.<br>
The term "specifically recognizing" means in accordance with this invention that the antibody molecule is capable of specifically interacting with and/or binding to at least two amino acids of each of the two regions of p-A4 as defined herein. Said term relates to the specificity of the antibody molecule, i.e. to its ability to discriminate between the specific regions of the β-A4 peptide as defined herein and another, not related region of the β-A4 peptide or another, not APP-related protein/peptide/(unrelated) tests-peptide. Accordingly, specificity can be determined experimentally by methods known in the art and methods as disclosed and described herein. Such methods comprise, but are not limited to Western blots, ELISA-, R1A-, ECL-, IRMA-tests and peptide scans. Such methods also comprise the determination of Ko-values as, inter alia, illustrated in the appended examples. The peptide scan (pepspot assay) is routinely employed to map linear epitopes in a polypeptide antigen. The primary sequence of the polypeptide is synthesized successively on activated cellulose with peptides overlapping one another. The recognition of certain peptides by the antibody to be tested for its ability to detect or recognize a specific antigen/epitope is scored by routine colour development (secondary antibody with horseradish peroxidase and 4-chloronaphthol and hydrogenperoxide), by a chemoluminescence reaction or similar means known in the art. In the case of, inter alia, chemoluminescence reactions, the reaction can be quantified. If the antibody reacts with a certain set of overlapping peptides one can deduce the minimum sequence of amino acids that are necessary for reaction; see illustrative Example 6 and appended Table 2.<br>
The same assay can reveal two distant clusters of reactive peptides, which indicate the recognition of a discontinuous, i. e. conformational epitope in the antigenic polypeptide (Geysen (1986), Mol. Immunol. 23, 709-715).<br><br>
In addition to the pepspot assay, standard ELISA assay can be carried out. As demonstrated in the appended examples small hexapeptides may be coupled to a protein and coated to an immunoplate and reacted with antibodies to be tested. The scoring may be carried out by standard colour development (e.g. secondary antibody with horseradish peroxidase and tetram^ethyl benzidine with hydrogenperoxide). The reaction in certain wells is scored by the optical density, for example at 450 nm. Typical background (=negative reaction) may be 0.1 OD, typical positive reaction may be 1 OD. This means the difference (ratio) positive/negative can be more than 10 fold. Further details are given in the appended examples. Additional, quantitative methods for determining the specificity and the ability of "specifically recognizing" the herein defined two regions of the p-A4 peptide are given herein below.<br>
The term "two regions of the β-A4 peptide" relates to two regions as defined by their amino acid sequences shown in SEQ ID NOs: 1 and 2, relating to the N-terminal amino acids 2 to 10 and to the central amino acids 12 to 25 of (3-A4 peptide. The term "β-A4 peptide" in context of this invention relates to the herein above described Aβ39, Aβ41, Aβ43, preferably to A(340 and A|342. Aβ42 is also depicted in appended SEQ ID NO: 27. It is of note that the term "two regions of the [3-A4 peptide" also relates to an "epitope" and/or an "antigenic determinant" which comprises the herein defined two regions of the β-A4 peptide or parts thereof. In accordance with this invention, said two regions of the β-A4 peptide are separated (on the level of the amino acid sequence) in the primary structure of the β-A4 peptide by at least one amino acid, preferably by at least two amino acids, more preferably by at least three amino acids, more preferably by at least four amino acids, more preferably by at least five amino acids, more preferably at least six amino acids, more preferably at least nine amino acids and most preferably at least twelve amino acids. As shown herein and as documented in the appended examples, the inventive antibodies/antibody molecules detect/interact with and/or bind to two regions of the (3-A4 peptide as defined herein, whereby said two regions are separated (on the primary structure level of the amino acid sequence) by at least one amino acid and wherein the sequence separating said two regions/"epitope" may comprise more then ten amino adds, preferably 14 amino acids, more preferably 15 amino acids or<br><br>
16 amino acids. For example, MSR-3 Fab (as an inventive antibody molecule) recognizes detects/interacts with two regions on the β-A4 peptide, wherein said first region comprises amino acids 3 and 4 (EF) and said second regions comprises amino acids 18 to 23 (VFFAED). Accordingly, the separating sequence between the region/epitopes to be detected/recognized has a length of 13 amino acids on the primary amino acid sequence structure. Similarly, MSR #3.4H7 lgG1, an optimized and matured antibody molecules derived from MSR-3 and comprised in an IgG1-framework, detects/interacts with/binds to two epitopes/regions of [3-A4 which comprise in the first region positions 1 to 4 (DAEF) and in the second region positions 19 to 24 (FFAEDV) of β-A4 as defined herein. Accordingly, MSR #3.4H7 lgG1 recognizes/detects/interacts with/binds to two epitopes/regions which are, on the primary amino acid sequence level, separated by 14 amino acids. As detailed in the appended examples, affinity maturation and conversion of monovalent inventive Fab fragments to full-length lgG1 antibodies may result in a certain broadening of the epitopes/regions detected in pepspot, ELISA assays and the like. Therefore, the antibody molecules of the invention are capable of simultaneously and independently recognizing two regions of the P-A4 peptide/Ap4 wherein said regions comprise the amino acid sequence as shown in SEQ ID NO; 1 (or parts thereof) and the amino acid sequence as shown in SEQ ID NO: 2 (or (a) part(s) thereof). Due to the potential broadening of epitopes as detailed herein it is, however, also envisaged that amino acids in close proximity to the sequences of SEQ ID NO: 1 and 2 are detected/recognized, i.e. that additional amino acids are part of the two regions to be detected/recognized. Accordingly, it is also envisaged that, e.g. the first amino acid of Ap (1-42) as defined herein, namely D (Aspartic acid) in part of one epitope to be detected/recognized or that amino acids located after the region of Aβ (1-42) as defined in SEQ ID NO: 2 are detected/recognized. Said additional amino acid may, e.g., be the amino acid on position 26 of SEQ ID NO: 27 (I3A4/AP (1-42)), namely S (Serine).<br>
The term may also relate to a conformational epitope, a structural epitope or a discountinuous epitope consisting of said two regions or parts thereof; see also Geysen (1986), loc. cit In context of this invention, a conformational epitope is defined by two or more discrete amino acid sequences separated in the primary<br><br>
sequence which come together on the surface when the polypeptide folds to the native protein (Sela, (1969) Science 166, 1365 and Laver, (1990) Cell 61, 553-6). The antibody molecules of the present invention are envisaged to specifically bind to/interact with a conformational/structural epitope(s) composed of and/or comprising the two regions of [3-A4 described herein or parts thereof as disclosed herein below. The "antibody molecules" of the present invention are thought to comprise a simultaneous and independent dual specificity to (a) an amino acid stretch comprising amino acids 2 to 10 (or (a) part(s) thereof) of β-A4 and (b) an amino acid stretch comprising amino acids 12 to 25 (or (a) part(s) thereof) of p-A4 (SEQ ID NO. 27). Fragments or parts of these stretches comprise at least two, more preferably at least three amino acids. Preferred fragments or parts are in the first region/stretch of SEQ ID NO: 27 the amino acid sequences AEFRHD, EF, EFR, FR, EFRHDSG, EFRHD or HDSG and in the second region/stretch of SEQ ID NO: 27 the amino acid sequences HHQKL, LV, LVFFAE, VFFAED VFFA, or FFAEDV. As mentioned above, said fragments may also comprise additional amino acids or may be parts of the fragments defined herein. Specific examples are DAE, DAEF, FRH or RHDSG.<br>
A number of antibodies specifically recognizing Ap peptides have been described in the art. These antibodies have mainly been obtained by immunizing animals with Aβi-40 or Aβi-42 or fragments thereof using standard technologies. According to published data monoclonal antibodies that were generated by immunization with the complete Aβ peptide (1-40 or 1-42) recognize exclusively an epitope close to the N-terminus of Aβ. Further, examples are the antibodies BAP-1 and BAP-2 (Brockhaus, unpublished) which were generated by immunization of mice with Apl-40 and which recognize the amino acids 4-6 in the context of larger Ap peptides; see appended Example 7, Table 2 and Example 12, Table 7. Antibodies that recognize the middle part of Aβ derive from immunizations with smaller peptides. For example, the antibody 4G8 was generated by immunization with the Ap peptide 1-24 and recognizes exclusively the sequence 17-24 (Kim, (1988) Neuroscience Research Communications 2, 121-130). Many other monoclonal antibodies have been generated by immunizing mice with Ap-derived fragments, and antibodies recognizing the C-terminal end of Apl-40 and Ap1-42 are widely used to distinguish<br><br>
and quantitate the corresponding Aβ peptides in biological fluids and tissues by ELISA, Western blot and immunohistochemistry analysis (Ida at al, (1996) J. Biol. Chem. 271, 22908-22914; Johnson-Wood et a!., (1997), Proc. Natl. Acad. Sci. USA (1994), 1550-1555; Suzuki et a!.. (1994), Science 264, 1336-1340; Brockhaus (1998), Neuro Rep. 9, 1481-1486). BAP-17 is a nnouse nnonoclonal antibody which has been generated by innmunizing mice with Aβ fragment 35-40. It specifically recognizes the C-terminal end of Aβ1-40 (Brockhaus (1998) Neuroreport 9, 1481-1486).<br>
It is believed that the immunization with T-cell dependent antigens (often poor immunogens) requires a proteolytic cleavage of the antigen in the endosomes of antigen presenting cells. The in vivo selection of high affinity antibodies after immunization is driven by the contact of helper T cells to antigen presenting cells. The antigen presenting cells only present short peptides and not polypeptides of large size. Accordingly, these cells have a complicated (but well known) machinery to endocytose antigen(s), degrade the antigen(s) in endosomes, combine selected peptides with suitable MHC class II molecules, and to export the peptide-MHC complex to the cell surface. This is where the antigen specific recognition by T cells occurs, with the aim to provide help to maturing B cells. The B cells which receive most T cell help have the best chance to develop into antibody secreting cells and to proliferate. This shows that antigen processing by proteolysis is an important step for the generation of an high affinity antibody response in vivo and may explain the dominance of the N-terminal Aβ epitope in prior art monoclonal and polyclonal antibodies derived by immunization.<br>
In contrast, the selection of antibodies/antibody molecules of the present invention is driven by the physical adherence of Fab expressing phages to the antigen. There is no degradation of the antigen involved in this in vitro selection process. The phages which express the Fab with the highest affinity towards the antigen are selected and propagated. A synthetic library as employed in the appended examples to select for specific antibody molecules according to this invention is particularly suited for avoiding any bias for single, continuous epitopes that is often found in libraries derived from immunized B cells.<br><br>
It is of note that the prior art has not described antibody molecules recognizing two, independent regions of Aβ4 which specifically recognizes (a) discontinuous/structural/confornnational epitope(s) and/or which are capable of simultaneously and independently recognizing two regions/epitopes of Ap4. Vaccination of transgenic mice overexpressing mutant human APP V717F (PDAPP mice) with A(31-42 resulted in an almost complete prevention of amyloid deposition in the brain when treatment was initiated in young animals, i. e. before the onset of neuropathologies, whereas in older animals a reduction of already formed plaques was observed suggesting antibody-mediated clearance of plaques (Schenk et al., (1999), Nature 400,173-177). The antibodies generated by this immunization procedure were reactive against the N-terminus of A[34 covering an epitope around amino acids 3-7 (Schenk et al., (1999), loc. cit.; WO 00/72880). Active immunization with Apl-42 also reduced behavioural impairment and memory loss in different transgenic models for Alzheimer's Disease (Janus et al., (2000) Nature 408, 979-982; Morgan et al., (2000) Nature 408, 982-985). Subsequent studies with peripherally administered antibodies, i. e. passive immunization, have confirmed that antibodies can enter the central nervous system, decorate plaques and induce clearance of preexisting amyloid plaques in APP transgenic mice (PDAPP mice) (Bard et al., (2000) Nat. Med. 6, 916-919; WO 00/72880). In these studies, the monoclonal antibodies with the most potent in vivo and ex vivo efficacy (triggering of phagocytosis in exogenous microglial cells) were those which recognized A[34 N-terminal epitopes 1-5 (mab 306, lgG2b) or 3-6 (mab 10D5, lgG1). Likewise, polyclonal antibodies isolated from mice, rabbits or monkeys after immunization with Aβ1-42 displayed a similar N-terminal epitope specificity and were also efficacious in triggering phagocytosis and in vivo plaque clearing, in contrast, C-terminal specific antibodies binding to A|31-40 or A|31-42 with high affinity did not induce phagocytosis in the ex vivo assay and were not efficacious in vivo (WO 00/72880). Monoclonal antibody m266 (WO 00/72880) was raised against Aβ 13-28 (central domain of Aβ) and epitope mapping confirmed the antibody specificity to cover amino acids 16-24 in the Ap sequence. This antibody does not bind well to aggregated Aβ and amyloid deposits and merely reacts with soluble (monomeric) Aβ, i. e. properties which are similar to another well-known and commercially available monoclonal antibody (4G8;<br><br>
Kim, (1988) Neuroscience Research Communications 2, 121-130; commercially available from Signet Laboratories Inc. Dedham, MA USA) which recognizes the same epitope.<br>
In vivo, the m266 antibody was recently found to markedly reduce Aβ deposition in PDAPP mice after peripheral administration (DeMattos, (2001) Proc. Natl. Acad. Sci. USA 98, 8850-8855). However, and in contrast to N-terminal specific antibodies, m266 did not decorate amyloid plaques in vivo, and it was therefore hyothesized that the brain A[3 burden was reduced by an antibody-induced shift in equilibrium between CNS and plasma Aj3 resulting in the accumulation of brain-derived Aβ in the periphery, firmly complexed to m266 (DeMattos, (2001) loo. cit.).<br>
The antibodies/antibody molecules of the present invention, by simultaneously (for example in a structural/conformational epitope formed by the N-terminal and central region of f2.A4 as described herein) and independently (for example in pepspot assays as documented in the appended experimental part) binding to the N-terminal and central epitopes, combine the properties of an N-terminal-specific antibody and a central epitope-specific antibody in a single molecule. Antibodies with the dual epitope specificity, as described in the present invention, are considered to be more efficacious in vivo, in particular in medical and diagnostic settings for, e.g., reducing amyloid plaque burden or amyloidogenesis or for the detection of amyloid deposits and plaques.It is well known that in the process of Aβ4 aggregation and amyloid deposition conformational changes occur, and while the central epitope is easily accessible in soluble Aβ4 it appears to be hidden and less reactive in aggregated or fibrillar Aβ4. The fact that the central/middle epitope-specific antibody m266 is efficacious in vivo indicates that neutralization of soluble Aβ4 may also be a critical parameter. The antibodies/antibody molecules of the present invention, due to the dual epitope specificity, can bind to both fibrillar and soluble Aβ4 with similar efficacy, thus allowing interaction with amyloid plaques as well as neutralization of soluble Aβ4. The term "simultaneously and independently binding to the N-terminal and central/middle epitopes of p-A4' as employed herein in context of the inventive antibody molecules relates to the fact that the antibodies/antibody molecules<br><br>
described herein may detect and/or bind to both epitopes simultaneously, i.e. at the same time (for example on conformational/structural epitopes formed by the N-terminal epitope (or (a) part(s) thereof) and central epitopes (or (a) part(s) thereof) of BA4 as defined herein) and that the same antibody molecules, however, are also capable of detecting/binding to each of the defined epitopes in an independent fashion, as inter alia, demonstrated in the pepspot analysis shown in the examples.<br>
Clearance of amyloid plaques in vivo in PDAPP mice after direct application of the antibodies to the brain is not dependent on the IgG subtype and may also involve a mechanism which is not Fc-mediated, i. e. no involvement of activated microglia in plaque clearance (Bacskai, (2001), Abstract Society for Neuroscience 31^^ Annual Meeting, November 10-15, 2001, San Diego). This observation is in contrast to what has been postulated in an earlier study by Bard (2000), loc. cit.<br>
In another study antibodies raised against Aβi-28 and Aβ1-16 peptides were found to be effective in disaggregating Aβ fibrils in vitro, whereas an antibody specific for Aβ 13-28 was much less active in this assay (Solomon, (1997) Proc. Natl. Acad. Sci. USA 94, 4109-4112). Prevention of Ap aggregation by an anti-Api-28 antibody (AMY-33) has also been reported (Solomon, (1996) Proc. Natl. Acad. Sci. USA 93, 452-455). In the same study, antibody 6F/3D which has been raised against Ap fragment 8-17 slightly interfered with Zn2+-induced Ap aggregation but had no effect on the self aggregation induced by other aggregation-inducing agents. The efficacy of the various antibodies in these in vitro assays correlates with the accessibility of their epitopes in Ap4 aggregates. The N-terminus is exposed and N-terminal specific antibodies clearly induce de-polymerization, whereas the central region and the C-terminus are hidden and not easily accessible and thus antibodies against these epitope are much less effective.<br>
Investigations with respect to epitope accessibilty for antibodies have shown that in aggregated Ap the N-terminal epitope is exposed and reacts with the BAP-1 antibody, whereas the middle or central epitope indeed remains cryptic, i. e. no binding of the 4G8 antibody was observed. However, in monomericAp both epitopes are overt and are equally recognized by both prior art antibodies.<br><br>
In contrast, in the present invention, it was surprisingly found that the herein described antibody molecules recognize two discontinuous amino acid sequences, e.g. a conformational/structural epitope on the Aβ peptide. Two "discontinuous amino acid sequences" in accordance with this invention means that said two amino acid sequences forming the N-terminal and central/middle epitopes, respectively, are separated on p-A4 in its primary structure by at least two amino acids which are not part of either epitope.<br>
The binding area of an antibody Fab (=paratope) occupies a molecular surface of approximately 30 x 30 A in size (Laver, Cell 61 (1990), 553-556). This is enough to contact 15 to 22 amino acid residues which may be present on several surface loops. The discontinuous epitope recognized by the inventive antibody molecules resembles a conformation in which the N-terminal (residues 2 to 10 or parts thereof) and middle Aβ peptide sequences (residues 12 to 25 or parts thereof) are in close proximity. Only within this conformation, the maximum number of antigen-antibody contacts and the lowest free energy state are obtained.<br>
Based on energetic calculations it has been suggested that a smaller subset of 5-6 residues, which are not arranged in a linear sequence but are scattered over the epitope surface, contributes most of the binding energy while surrounding residues may merely constitute a complementary array (Laver (1990) loc. cit.).<br>
The inventive antibodies/antibody molecules are capable of binding to aggregated A(3 and strongly react with amyloid plaques in the brain of AD patients (as documented in the appended examples). In addition, they are capable of de-polymerizing/disintegrating amyloid aggregates.<br>
Without being bound by theory, the conformational/structural epitope (composed by the two regions of Aβ4 or (a) part(s) of said regions as described herein) is believed to be partially exposed in aggregated Aβ. However, it is known that major part of the middle/second epitope/region alone is not freely accessible in these Aβ aggregates (based on the poor reactivities of middle epitope-specific antibodies 4G8 and m266). On the other hand, and in view of the considerations mentioned above, it is likely<br><br>
that one or several residues of the middle region are components of the conformational epitope and, in conjunction with the residues from the N-terminal region, are accessible to the antibodies of the present invention, thereby significantly contributing to the binding energy of the antibody-Ap4 interaction. The reactivity of the inventive antibody molecules with the conformational epitope in aggregated Ap is therefore unique and clearly distinct from a-Aβ4 antibodies described in the prior art. Yet, as pointed out herein above, a further unique feature of the inventive antibodies/antibody molecules is their capacity to simultaneously and independently binding to/recognizing two separate epitopes on β-A4, as defined herein and in the appended examples.<br>
In a preferred embodiment of the invention, the inventive antibody molecule is an antibody molecule wherein the least two regions of the |3-A4 to be specifically recognized by said antibody form a conformational/structural epitope ^or a discontinuous epitope; see Geysen (1986), loc. cit.; Ghoshal (2001), J. Neurochem. 77, 1372-1385; Hochleitner (2000), J. Imm. 164, 4156-4161; Laver (1990), loc. cit.. The term "discontinuous epitope" means in context of the invention non-linear epitopes that are assembled from residues from distant portions of the polypeptide chain. These residues come together on the surface when the polypeptide chain folds into a three-dimensional structure to constitute a conformational/structural epitope. The present invention provides for preferred, unexpected epitopes within p-A4, which result in the inventive generation of specific antibody molecules, capable of specifically interacting with these epitopes. These inventive antibodies/antibody molecules provide the basis for increased efficacy, and a reduced potential for side effects. As pointed out above, the inventive antibodies, however, were also capable of independently interacting with each of the defined two regions/epitopes of β-A4, for example in Pepspot assays as documented in the appended examples.<br>
The present invention, accordingly, provides for unique tools which may be employed to de-polymerize aggregated Ap-fibrils in vivo and in vitro and/or which are capable of stabilizing and/or neutralizing a conformational epitope of monomeric Aβ and thereby capable of preventing the pathological Aj3 aggregation.<br><br>
It is furthermore envisaged that the inventive antibodies bind to Aβ deposits at the rim of amyloid plaques in, inter alia, Alzheimer's brain and efficiently dissolve the pathological protofibrils and fibrils.<br>
In a preferred embodiment, the antibody molecule of the invention recognizes at least two consecutive amino acids within the two regions of Aβ4 defined herein, more preferably said antibody molecule recognizes in the first region an amino acid sequence comprising the amino acids: AEFRHD, EF, EFR, FR, EFRHDSG, EFRHD or HDSG and in the second region an amino acid sequence comprising the amino acids: HHQKL, LV, LVFFAE, VFFAED, VFFA or FFAEDV. Further fragments or broadened parts comprise: DAE. DAEF, FRH or RHDSG.<br>
It is particularly preferred that the antibody molecule of the invention comprises a variable Vn-region as encoded by a nucleic acid molecule as shown in SEQ ID NO: 3, 5 or 7 or a variable Vn-region as shown in the amino acid sequences depicted in SEQ ID NOs: 4, 6 or 8, The sequences as shown in SEQ ID NOs: 3 and 4 depict the coding region and the amino acid sequence, respectively, of the Vn-region of the inventive, parental antibody MSR-3 (MS-Roche 3), the sequences in SEQ ID NOs: 5 and 6 depict the coding region and the amino acid sequence, respectively, of the VH-region of the inventive, parental antibody MSR-7 (MS-Roche 7) and SEQ ID NOs: 7 and 8 depict the coding region and the amino acid sequence, respectively, of the VH-region of the inventive, parental antibody MSR-8 (MS-Roche 8). Accordingly, the invention also provides for antibody molecules which comprise a variable VL-region as encoded by a nucleic acid molecule as shown in a SEQ ID NO selected from the group consisting of SEQ ID NO: 9, 11 or 13 or a variable Vt-region as shown in the amino acid sequences depicted in SEQ ID NOs: 10, 12 or 14. SEQ ID NOs: 9 and 10 correspond to the Vt-region of MSR-3, SEQ ID NOs: 11 and 12 correspond to the Vt-region of MSR-7 and SEQ ID NOs: 13 and 14 correspond to the VL-region of MSR-8. As illustrated in the appended examples, the parental antibodies MSR-3, -7 and -8, are employed to further generate optimized antibody molecules with even better properties and/or binding affinities. Some of the corresponding and possible strategies are exemplified and shown in the appended examples.<br><br>
The optimization strategy as illustrated in the appended examples lead to a plurality of inventive, optimized antibodies. These optimized antibodies share with their parental antibodies the CDR-3 domain of the Vw-region, Whereas the original framework region (as shown in appended Figure 1) remains the same, in the matured/optimized antibody molecules, CDR1, CDR2 and/or VL CDRS-regions are changed. Illustrative, modified sequence motives for optimized antibody molecules are shown in appended table 1. Accordingly, within the scope of the present invention are also optimized antibody molecules which are derived from the herein disclosed MSR-3, -7 and -8 and which are capable of specifically reacting with/specifically recognizing the two regions of the p-A4 peptide as defined herein. In particular, CDR-regions, preferably CDR1s, more preferably CDR1s and CDR2s, most preferably CDR1s, CDR2s and CDR3s as defined herein may be employed to generate further inventive antibodies/antibody molecules, inter alia, by CDR-grafting methods known in the art; see Jones (1986), Nature 321, 522-515 or Riechmann (1988), Nature 332, 323-327. Most preferably the inventive antibodies/antibody molecules as well as antibody fragments or derivatives are derived from the parental antibodies as disclosed herein and share, as disclosed above, the CDR-3 domain of the Vn-region with at least one of said parental antibodies. As illustrated below, it is also envisaged that cross-cloned antibodies are generated which are to be considered as optimized/maturated antibodies/antibody molecules of the present invention. Accordingly, preferred antibody molecules may also comprise or may also be derived from antibodies/antibody molecules which are characterized by VH-regions as shown in any of SEQ ID NOs: 32 to 45 or Vt-regions as shown in SEQ ID NOs: 46 to 59 or which may comprise a CDR-3 region as defined in any of SEQ ID NOs: 60 to 87. In a particular preferred embodiment, the optimized antibody molecule of the present invention comprises Vn-regions and Vu-regions as depicted in SEQ ID NOs: 88/89 and 90/91, respectively, or parts thereof. Apart thereof may be (a) CDR-region(s), preferably (a) CDR3-region(s). A particularly preferred antibody molecule of the optimized type comprises a H-CDR3 as characterized in SEQ ID NOs: 92 or 93 and/or a L-CDR3 as characterized in SEQ ID NOs: 94 or 95. It is preferred that the antibodies/antibody molecules of the invention are characterized by their specific reactivity with β-A4 and/or peptides derived from said β-A4. For example, optical densities in ELISA-tests, as illustrated in the appended<br><br>
examples, may be established and the ratio of optical densities may be employed to define the specific reactivity of the parental or the optimized antibodies. Accordingly, a preferred antibody of the invention is an antibody which reacts in an ELISA-test with {3-A4 to arrive at an optical density measured at 450 nm that is 10 times higher than the optical density measured without β-A4, i. e. 10 times over background. Preferably the measurement of the optical density is performed a few minutes (e.g. 1, 2, 3, 4, 5, 6, or 7 minutes) after initiation of the color developing reaction in order to optimize signal to background ratio.<br>
In a particular preferred embodiment, the inventive antibody molecule comprises at least one CDR3 of an Vt-region as encoded by a nucleic acid molecule as shown in SEQ ID NOs: 15, 17 or 19 or at least one CDR3 amino acid sequence of an VL-region as shown in SEQ ID NOs: 16, 18 or 20 and/or said antibody molecule comprises at least one CDR3 of an Vn-region as encoded by a nucleic acid molecule as shown in SEQ ID NOs: 21, 23 or 25 or at least one CDR3 amino acid sequence of an Vw-region as shown in SEQ ID NOs: 22, 24 or 26. Most preferred are antibodies comprising at least one CDR3 of an Vn-region as defined herein. The CDR-3 domains mentioned herein above relate to the inventive, illustrative parental antibody molecules MSR-3, -7, or-8. However, as illustrated in the appended tables 1, 8 or 10, matured and/or optimized antibody molecules obtainable by the methods disclosed in the appended examples may comprise modified VH-, VL-, CDR1, CDR2 and CDR3 regions. Accordingly, the antibody molecule of the invention is preferably selected from the group consisting of MSR-3, -7 and -8 or an affinity-matured version of MSR-3, -7 or -8. Affinity-matured as well as cross-cloned versions of MSR-3, -1 and -8 comprise, inter alia, antibody molecules comprising CDR1, CDR2 and/or CDR3 regions as shown in table 1 or 8 or characterized \n any of SEQ ID NOs: 15 to 20, 21 to 26, 60 to 74, 75 to 87, 92 and 93 or 94 and 95 as well as in SEQ ID NOs: 354 to 413. Most preferably, the antibody of the invention comprises at least one CDR, preferably a CDR1, more preferably a. CDR2, most preferably a CDR3 as shown in the appended table 1, 8 or as documented in appended table 10.<br>
It is of note that affinity-maturation techniques are known in the art, described in the appended examples and, inter alia, in Knappik (2000), J. Mol. Biol. 296, 55; Krebs<br><br>
(2000), J. Imm. Meth. 254, 67-84; WO 01/87337; WO 01/87338; US 6,300,064; EP 96 92 92 78.8 and further references cited herein below.<br>
In a more preferred embodiment of the invention, the antibody molecule is a full antibody (immunoglobulin, like an IgGI, an lgG2, an lgG2b, an lgG3, an lgG4, an IgA, an IgM, an IgD or an IgE), an F(ab)-, Fabc-, Fv-, Fab'-, F(ab')2- fragment, a single-chain antibody, a chimeric antibody, a CDR-grafted antibody, a bivalent antibody-construct, an antibody-fusion protein, a cross-cloned antibody or a synthetic antibody. Also envisaged are genetic variants of immunoglobulin genes. Genetic variants of, e.g., immunoglobulin heavy G chain subclass 1 (lgG1) may comprise the G1m(17) or G1m(3) allotypic markers in the CH1 domain, or the G1m(1) or the G1m(non-1) allotypic marker in the CH3 domain. The antibody molecule of the invention also comprises modified or mutant antibodies, like mutant IgG with enhanced or attenuated Fc-receptor binding or complement activation. It is also envisaged that the antibodies of the invention are produced by conventional means, e.g. the production of specific monoclonal antibodies generated by immunization of mammals, preferably mice, with peptides comprising the two regions of (SA4 as defined herein, e.g. the N-terminai and central region/epitope comprising (a) amino acids 2 to 10 (or (a) part(s) thereof) of p-A4 and (b) an amino acid stretch comprising amino acids 12 to 25 (or (a) part(s) thereof) of p-A4 (SEQ ID NO. 27). Accordingly, the person skilled in the art may generate monoclonal antibodies against such a peptide and may screen the obtained antibodies for the capacity to simultaneously and independently binding to/reacting with the N-terminai and central region/epitope of EA4 as defined herein. Corresponding screening m'ethods are disclosed in the appended examples.<br>
As illustrated in the appended examples, the inventive antibodies/antibody molecules can readily and preferably be recombinantiy constructed and expressed. Preferably, the antibody molecule of the invention comprises at least one, more preferably at least two, preferably at least three, more preferably at least four, more preferably at least five and most preferably six CDRs of the herein defined MSR-3, MSR-7 or MSR-8 parental antibodies or of affinity-matured/optimized antibodies derived from said parental antibodies, it is of note that also more than six CDRs may<br><br>
be comprised in recombinantly produced antibodies of the invention. The person skilled in the art can readily employ the information given in the appended examples to deduce corresponding CDRs of the parental as well as the affinity optimized antibodies. Examples of optimized antibodies which have been obtained by maturation/optimization of the parental antibodies are, inter alia, shown in appended table 1. An maturated/optimized antibody molecule of the invention is, e.g. MSR 7.9H7 which is also characterized by sequences appended herein, which comprise SEQ ID NOs: 88 to 95 and which depict the Vn-region of MSR 7.9H7 (SEQ ID NOs: 88 and 89), the Vu-region of MSR 7.9H7 (SEQ ID NOs: 90 and 91), the H-CDR3 of MSR 7.9H7 (SEQ ID NOs: 92 and 93) as well as the L-CDR3 of MSR 7,9H7 (SEQ ID NOs: 94 and 95). Illustrative antibody molecule 7.9H7 is derived from parental antibody MSR7 and is a particular preferred inventive example of an optimized/matured antibody molecule of the present invention. This antibody molecule may be further modified in accordance with this invention, for example in form of cross-cloning, see herein below and appended examples.<br>
As documented in the appended examples, the antibodies of the invention also comprise cross-cloned antibodies, i.e. antibodies comprising different antibody regions (e.g. CDR-regions) from one or more parental or affinity-optimized antibody(ies) as described herein. These cross-cloned antibodies may be antibodies in several, different frameworks, whereby the most preferred framework is an IgG-framework, even more preferred in an lgG1-, lgG2a or an lgG2b-framework. It is particularly preferred that said antibody framework is a mammalian, most preferably a human framework. The domains on the light and heavy chains have the same general structure and each domain comprises four framework regions, whose sequences are relatively conserved, joined by three hypervariabie domains known as complementarity determining regions (CDR1-3).<br>
As used herein, a "human framework region" relates to a framework region that is substantially identical (about 85% or more, usually 90-95% or more) to the framework region of a naturally occurring human immunoglobulin. The framework region of an antibody, that is the combined framework regions of the constituent light and heavy chains, serves to position and align the CDR's. The CDR's are primarily<br><br>
responsible for binding to an epitope of an antigen. It is of note that not only cross-cloned antibodies described herein may be presented in a preferred (human) antibody framework, but also antibody molecules comprising CDRs from, inter alia, the parental antibodies MSR-3, -7 or -8 as described herein or of matured antibodies derived from said parental antibodies, may be introduced in an immunoglobulin framework. Preferred frameworks are lgG1, lgG2a and lgG2b. Most preferred are human frameworks and human lgG1 frameworks.<br>
As shown in the appended examples, it is, inter alia possible, to transfer, by genetic engineering known in the art whole light chains from an optimized donor clone to an optimized recipient clone. Example for an optimized donor clone is, e.g. L-CDR1 (L1) and an example for an optimized recipient clone is H-CDR2 (H2)- Epitope specificity may be conserved by combining clones which possess the same H-CDR-3 regions. Further details are given in illustrative Example 13.<br>
Preferred cross-cloned antibody molecules of the invention are selected from the group consisting of MS-R#3.3H1x3.4L9, MS-R #3.4H1x3.4L9, MS-R #3.4H3x3.4L7, MS-R #3.4H3x3.4L9. MS-R #3.4H7x3.4L9, MS-R #3.4H7x3.4L7, MS-R #3.6H5x3.6L1, MS-R #3.6H5x3.6L2, MS-R #3.6.H8x3.6.L2, MS-R #7.2H2x7.2L1, MS-R #7.4H2x7.2L1, MS-R #7.4H2x7,12L2, MS-R #7.9H2x7.2L1(L1), MS-R #7.9H2x7.12L1, MS-R #7.9H2x7.12L2, MS-R #7.9H2x7.12L2(L1+2), MS-R #7.9H4x7.12.L2, MS-R #7.11H1x7.2L1, MS-R #7.11H1x7.11L1, MS-R #7.11H2x7.2L1(L1), MS-R #7.11H2x7.9L1 (L1), MS-R #7.11H2x7.12L1 or MS-R #8.1H1x8.2L1.<br>
The generation of cross-cloned antibodies is also illustrated in the appended examples. The, above mentioned preferred cross-cloned antibodies/antibody molecules are optimized/matured antibody molecules derived from parental antibodies disclosed herein, in particular from MSR-3 and MSR-7. in addition, further characterizing CDR-sequences and V-regions of the cross-cloned antibody molecules/antibodies are given in appended SEQ ID NOs: 32, 33, 46 and 47 (MSR 3.6H5x3.6.L2; VH-, VL-region); 34, 35, 48 and 49 (MSR 3.6H8x3.6.L2; VH-, VL-regions); 36, 37, 50 and 51 (MSR 7.4H2x7.2.L1; VH-, Vt-regions); 38, 39, 52 and 53 (MSR 7.9H2X7.12.L2; VH-, Vu-regions); 40, 41, 54 and 55 (MSR # 7.9H4x7.12.L2;<br><br>
VH-, Vt-regions); 42, 43, 56 and 57 (MSR#7.11H1x7.11.L1; VH-. VL-regions;; ana 44, 45, 58 and 59 (MSR # 7.11H1x7.2.L1; VH-, VL-regions). Corresponding CDR3 regions of these particular preferred cross-cloned antibody molecules are depicted in SEQ ID NOs: 60 to 87. For further MSR antibody nnolecules, VH-, VL-, CDR-regions can be deduced from appended Tables 8 or 10 and from the appended sequence listing, in particular SEQ ID NOS: 32 to 95*for MS-R antibodies/antibody molecules #3.6H5 X 3.6L2, #3.6H8 x 3.6L2, #7.4H2 x 7.2L1, #7.9H2 x 7.12L2, #7.9H4 x 7.12L2, #7.11 H1 X 7.11 LI, #7.11 HI X 7.2L1 and #7.9H7 or SEQ ID NOS: 294 to 413 for MSR-R antibodies/antibody molecules MS-R #3,3H1x3.4L9, #3.4H1 x 3.4L9, #3.4H3 X 3.4L7, #3,4H3 x 3.4L9, #3.4H7 x 3.4L9, #3.4H7 x 3.4L7, #3.6H5 x 3.6L1, #7.2H2 x 7.2L1, #7.4H2 x 7.12L2, #7.9H2 x 7.2L1, #7.9H2 x 7.12L1, #7,11H2 x 7.2L1, #7.11H2 X 7.9L1, #7.11H2 x 7.12L1 or #8.1H1 x 8.2L1. Accordingly, besides VH-regions defined above, preferred antibody molecules of the invention may comprise Vn-regions as defined in any one of SEQ ID NOs: 294 to 323. Similarly, SEQ ID NOs: 324 to 353 depict preferred Vt-regions which, besides to Vu-regions defined above which may be comprised in the inventive antibody molecules. Corresponding CDR-3 regions are defined above, as well as in additional sequences shown in SEQ ID NOs: 354 to 413.<br>
inventive antibody molecules can easily be produced in sufficient quantities, inter alia, by recombinant methods known in the art,' see, e.g. Bentley, Hybridoma 17 (1998), 559-567; Racher, Appl. Microbiol. Biotechnol. 40 (1994), 851-856; Samuelsson, Eur. J. Immunol. 26 (1996), 3029-3034.<br>
Theoretically, in soluble |3-A4 (monomeric/oiigomeric) both the N-terminal and the middle epitopes are accessible for antibody interaction and antibody molecules of the present invention may either bind to the N-terminal or middle epitope separately, but under these conditions maximum affinity will not be obtained. However, it is more likely that an optimal contact to the antibody paratope will be attained by simultaneous binding to both epitopes, i.e. similar to the interaction with aggregated β-A4. Thus, antibodies of the present invention are unique anti-Aβ antibodies in that they bind to aggregated β-A4 (via interaction with the N-terminal and middle epitope), and at the same time are also able to stabilize and neutralize the<br><br>
conformational epitope in soluble β-A4. These antibodies are distinct to prior art antibodies.<br>
Most preferred are antibody molecules of the invention which have an affinity to Aβ or defined fragments thereof with a KD value lower than 2000 nM, preferably lower than 100 nM, more preferably lower than 10 nM, most preferably lower than 1 nM. The measurement of such affinity/affinities may be carried out by methods illustrated in the examples and known in the art. Such methods comprise, but are not limited to BIACORE™-assays (www.biacore.com; Malmquist (1999), Biochem.Soc. Trans 27, 335-340) and solid phase assays using labeled antibodies or labeled Aβ.<br>
Preferably, the antibody molecule of the invention is capable of decorating/reacting with/binding to amyloid plaques in in vitro (post-mortem) brain sections from patients suffering from amyloid-related disorders, like Alzheimer's disease. Yet, it is also preferred that the inventive antibody/antibody molecules prevent Ap-aggregation in vivo as well as in in vitro assays, as illustrated in the appended examples. Similarly, the antibody molecules of the present invention are preferred to de-polymerize Ap-aggregate in vivo and/or in in vitro assays shown in the examples. This capacity of the inventive antibodies/antibody molecules is, inter alia, to be employed in medical settings, in particular in pharmaceutical compositions described herein below.<br>
The invention also provides for a nucleic acid molecule encoding an inventive antibody molecule as defined herein.<br>
Said nucleic acid molecule may be a naturally nucleic acid molecule as well as a recombinant nucleic acid molecule. The nucleic acid molecule of the invention may, therefore, be of natural origin, synthetic or semi-synthetic. It may comprise DNA, RNA as well as PNA and it may be a hybrid thereof.<br>
It is evident to the person skilled in the art that regulatory sequences may be added to the nucleic acid molecule of the invention. For example, promoters, transcriptional enhancers and/or sequences which allow for induced expression of the polynucleotide of the invention may be employed. A suitable inducible system is for<br><br>
example tetracycline-regulated gene expression as described, e.g., by Gossen and Bujard (Proc. Natl. Acad. Sci. USA 89 (1992), 5547-5551) and Gossen et al. (Trends Biotech. 12 (1994), 58-62), or a dexamethasone-inducibie gene expression system as described, e.g. by Crook (1989) EMBO J. 8, 513-519.<br>
Furthermore, it is envisaged for further purposes that nucleic acid molecule may contain, for example, thioester bonds and/or nucleotide analogues. Said modifications may be useful for the stabilization of the nucleic acid molecule against endo- and/or exonucleases in the cell. Said nucleic acid molecules may be transcribed by an appropriate vector containing a chimeric gene which allows for the transcription of said nucleic acid molecule in the cell. In this respect, it is also to be understood that the polynucleotide of the invention can be used for "gene targeting" or "gene therapeutic" approaches. In another embodiment said nucleic acid molecules are labeled. Methods for the detection of nucleic acids are well known in the art, e.g., Southern and Northern blotting, PCR or primer extension. This embodiment may be useful for screening methods for verifying successful introduction of the inventive nucleic acid molecules during gene therapy approaches.<br>
The nucleic acid molecule(s) of the invention may be a recombinantly produced chimeric nucleic acid molecule comprising any of the aforementioned nucleic acid molecules either alone or in combination. Preferably, the nucleic acid molecule of the invention is part of a vector.<br>
The present invention therefore also relates to a vector comprising the nucleic acid<br>
molecule of the present invention.<br>
The vector of the present invention may be,  e.g.,  a plasmid, cosmid, virus,<br>
bacteriophage or another vector used e.g. conventionally in genetic engineering, and<br>
may comprise further genes such as marker genes which allow for the selection of<br>
said vector in a suitable host cell and under suitable conditions.<br>
Furthermore, the vector of the present invention may, in addition to the nucleic acid<br>
sequences of the invention, comprise expression control elements, allowing proper<br>
expression of the coding regions in suitable hosts. Such control elements are known<br>
to the artisan and may include a promoter, a splice cassette, translation initiation<br><br>
codon, translation and insertion site for introducing an insert into the vector. Preferably, the nucleic acid molecule of the invention is operatively linked to said expression control sequences allowing expression in eukaryotic or prokaryotic cells.<br>
Control elements ensuring expression in eukaryotic and prokaryotic cells are well known to those skilled in the art. As mentioned herein above, they usually comprise regulatory sequences ensuring initiation of transcription and optionally poly-A signals ensuring termination of transcription and stabilization of the transcript. Additional regulatory elements may include transcriptional as well as translational enhancers, and/or naturally-associated or heterologous promoter regions. Possible regulatory elements permitting expression in for example mammalian host cells comprise the CMV- HSV thymidine kinase promoter, SV40, RSV-promoter (Rous Sarcoma Virus), human elongation factor 1a-promoter, the glucocorticoid-inducible MMTV-promoter (7vloloney Mouse Tumor Virus), metailothionein- or tetracyclin-inducible promoters, or enhancers, like CMV enhancer or SV40-enhancer. For expression in neurai cells, it is envisaged that neurofilament-, PGDF-, NSE-, PrP-, or thy-1-promoters can be employed. Said promoters are known in the art and, inter alia, described in Charron (1S95), J. Biol. Chem. 270, 25739-25745. For the expression in prokaryotic cells, a m.uititude of promoters including, for example, the tac-lac-promoter or the trp promoter, has been described. Besides elements which are responsible for the initiation of transcription such regulatory elements may also comprise transcription termination signals, such as SV40-poly-A site or the tk-poly-A site, downstream of the polynucleotide. In this context, suitable expression vectors are known in the art such as Okayama-Berg cDNA expression vector pcDV1 (Pharmacia), pRc/CMV, pcDNAI, pcDNA3 (In-vitrogene), pSPORTI (GIBCO BRL), pX (Pagano (1992) Science 255, 1144-1147), yeast two-hybrid vectors, such as pEG202 and dpJG4-5 (Gyuris (1995) Cell 75, 791-803), or prokaryotic expression vectors, such as lambda gi11 or pGEX (Amersham-Pharmacia). Beside the nucleic acid molecules of the present invention, the vector may further comprise nucleic acid sequences encoding for secretion signals. Such sequences are well known to the person skilled in the art. Furthermore, depending on the expression system used leader sequences capable of directing the peptides of the invention to a cellular compartment may be added to the coding sequence of the nucleic acid molecules of the invention and are well<br><br>
known in the art. The leader sequence(s) is (are) assembled in appropriate phase with translation, initiation, and termination sequences, and preferably, a leader sequence capable of directing secretion of translated protein, or a protein thereof, into the periplasmic space or extracellular medium. Optionally, the heterologous sequence can encode a fusionprotein including an C- or N-terminal identification peptide imparting desired characteristics, e.g., stabilization or simplified purification of expressed recombinant product. Once the vector has been incorporated into the appropriate host, the host is maintained under conditions suitable for high level expression of the nucleotide sequences, and, as desired, the collection and purification of the antibody molecules or fragments thereof of the invention may follow. The invention also relates, accordingly, to hosts/host cells which comprise a vector as defined herein. Such hosts may be useful for in processes for obtaining antibodies/antibody molecules of the invention as well as in medical/pharmaceutical settings. Said host cells may also comprise transduced ortransfected neuronal cells, like neuronal stem cells, preferably adult neuronal stem cells. Such host cells may be useful in transplantation therapies.<br>
Furthermore, the vector of the present invention may also be an expression, a gene transfer or gene targeting vector. Gene therapy, which is based on introducing therapeutic genes into cells by ex-vivo or in-vivo techniques is one of the most important applications of gene transfer. Transgenic mice expressing a neutralizing antibody directed against nerve growth factor have been generated using the 'neuroantibody" technique; Capsoni, Proc. Natl. Acad. Sci. USA 97 (2000), 6826-6831 and Biocca, Embo J. 9 (1990), 101-108. Suitable vectors, methods or gene-delivering systems for in-vitro or in-vivo gene therapy are described in the literature and are known to the person skilled in the art; see, e.g., Giordano, Nature Medicine 2 (1996), 534-539; Schaper, Circ. Res. 79 (1996), 911-919; Anderson, Science 256 (1992), 808-813, Isner, Lancet 348 (1996), 370-374; Muhlhauser, Circ. Res. 77 (1995), 1077-1086; Onodua, Blood 91 (1998), 30-36; Verzeletti, Hum. Gene Ther. 9 (1998), 2243-2251; Verma, Nature 389 (1997), 239-242; Anderson, Nature 392 (Supp. 1998), 25-30; Wang, Gene Therapy 4 (1997), 393-400; Wang, Nature Medicine 2 (1996), 714-716; WO 94/29469; WO 97/00957; US 5,580,859; US 5,559,466; US 4,394,448 or Schaper, Current Opinion in Biotechnology 7 (1996),<br><br>
635-640, and references cited therein. In particular, said vectors a/id/or gene delivery systems are also described in gene therapy approaches in neuroiogical tissue/ceils (see, inter aiia Blomer, J. Virology 71 (1997) 6641-6649) or in the hypothalamus (see, inter alia, Geddes, Front Neuroendocrinol. 20 (1999), 296-316 or Geddes, Nat. Med. 3 (1997), 1402-1404). Further suitable gene therapy constructs for use in neurological cells/tissues are known in the art, for example in Meier (1999), J. Neuropathol. Exp. Neurol. 58, 1099-1110. The nucleic acid molecules and vectors of the invention may be designed for direct introduction or for introduction via liposomes, viral vectors (e.g. adenoviral, retroviral), electroporation, ballistic (e.g. gene gun) or other delivery systems into the ceil. Additionally, a baculoviral system can be used as eukaryotic expression system for the nucleic acid molecules of the invention. The introduction and gene therapeutic approach should, preferably, lead to the expression of a functional antibody molecule of the invention, whereby said expressed antibody molecule is particularly useful in the treatment, amelioration and/or prevention of neurological disorders related to abnormal amyloid synthesis, assembly and/or aggregation, like, Alzheimer's disease and the like.<br>
Accordingly, the nucleic acid molecule of the present invention and/or the above described vectors/hosts of the present invention may be particularly useful as pharmaceutical compositions. Said pharmaceutical compositions may be employed in gene therapy approaches. In this context, it is envisaged that the nucleic acid molecules and/or vectors of the present invention may be employed to modulate, alter and/or modify the (cellular) expression and/or concentration of the antibody molecules of the invention or of (a) fragment(s) thereof.<br>
For gene therapy applications, nucleic acids encoding the peptide(s) of the invention or fragments thereof may be cloned into a gene delivering system, such as a virus and the virus used for infection and conferring disease ameliorating or curing effects in the infected cells or organism.<br>
The present invention also relates to a host cell transfected or transformed with the vector of the invention or a non-human host carrying the vector of the present invention, i.e. to a host cell or host which is genetically modified with a nucleic acid<br><br>
molecule according to the invention or with a vector connpnsing such a nucleic acid molecule. The term "genetically modified" means that the host cell or host comprises in addition to its natural genome a nucleic acid molecule or vector according to the invention which was introduced into the cell or host or into one of its predecessors/parents. The nucleic acid molecule or vector may be present in the genetically modified host cell or host either as an independent molecule outside the genome, preferably as a molecule which is capable of replication, or it may be stably integrated into the genome of the host cell or host.<br>
The host cell of the present invention may be any prokaryotic or eukaryotic cell. Suitable prokaryotic cells are those generally used for cloning like E. coli or Bacillus subtilis. Furthermore, eukaryotic cells comprise, for example, fungal or animal cells. Examples for suitable fungal cells are yeast cells, preferably those of the genus Saccharomyces and most preferably those of the species Saccharomyces cerevisiae. Suitable animal cells are, for instance, insect cells, vertebrate cells, preferably mammalian cells, such as e.g. HEK293, NSO. CHO, MDCK, U2-0SHela, NIH3T3, MOLT-4, Jurkat, PC-12, PC-3, IMR, NT2N, Sk-n-sh, CaSki, C33A. These host cells, e.g. CHO-cells, may provide post-translational modifications to the antibody molecules of the invention, including leader peptide removal, folding and assembly of H (heavy) and L (light) chains, glycosylation of the molecule at correct sides and secretion of the functional molecule. Further suitable cell lines known in the art are obtainable from cell line depositories, like the American Type Culture Collection (ATCC). In accordance with the present invention, it is furthermore envisaged that primary cells/cell cultures may function as host celJs. Said cells are in particular derived from insects (like insects of the species Drosophila or Blatta) or mammals (like human, swine, mouse or rat). Said host cells may also comprise cells from and/or derived from cell lines like neuroblastoma cell lines. The above mentioned primary cells are well known in the art and comprise, inter alia, primary astrocytes, (mixed) spinal cultures or hippocampal cultures.<br>
In a more preferred embodiment the host cell which is transformed with the vector of the invention is a neuronal cell, a neuronal stem cell (e.g. an adult neuronal stem cell), a brain cell or a cell (line) derived therefrom. However, also a CHO-cell<br><br>
comprising the nucleic acid molecule of the present invetion may be particularly useful as host. Such cells may provide for correct secondary modifications on the expressed molecules, i.e. the antibody molecules of the present invention. These modifications comprise, inter alia, glycosylations and phosphorylations.<br>
Hosts may be non-human mammals, most preferably mice, rats, sheep, calves, dogs, monkeys or apes. Said mamimais may be indispensable for developing a cure, preferably a cure for neurological and/or neurodegenerative disorders mentioned herein. Furthermore, the hosts of the present invention may be particularly useful in producing the antibody molecules (or fragments thereof) of the invention. It is envisaged that said antibody molecules (or fragments thereof) be isolated from said host. It is, inter alia, envisaged that the nucleic acid molecules and or vectors described herein are incorporated in sequences for transgenic expression. The introduction of the inventive nucleic acid molecules as transgenes into non-human hosts and their subsequent expression may be employed for the production of the inventive antibodies. For example, the expression of such (a) transgene(s) in the milk of the transgenic animal provide for means of obtaining the inventive antibody molecules in quantitative amounts; see inter alia, US 5,741,957, US 5,304,489 or US 5,849,992. Useful transgenes in this respect comprise the nucleic acid molecules of the invention, for example, coding sequences for the light and heavy chains of the antibody molecules described herein, operatively linked to promoter and/or enhancer structures from a mammary gland specific gene, like casein or beta-lactoglobulin.<br>
The invention also provides for a method for the preparation of an antibody molecule of the invention comprising culturing the host cell described herein above under conditions that allow synthesis of said antibody molecule and recovering said antibody molecule from said culture.<br>
The invention also relates to a composition comprising an antibody molecule of the invention or produced by the method described herein above, a nucleic acid molecule encoding the antibody molecule of the invention, a vector comprising said nucleic acid molecule or a host-cell as defined herein above and optionally, further molecules, either alone or in combination, like e.g. molecules which are capable of<br><br>
interfering with the formation of amyloid plaques or which are capable of depolymerizing already formed amyloid-plaques. The term "composition" as employed herein comprises at least one compound of the invention. Preferably, such a composition is a pharmaceutical or a diagnostic composition.<br>
The composition may be in solid or liquid form and may be, inter alia, in a form of (a) powder(s), (a) tablet(s), (a) solution(s) or (an) aerosol(s). Said composition may comprise on or more antibodies/antibody molecules of the invention or nucleic acid molecules, vector or hosts of the invention. It is also envisaged that said composition comprises at least two, preferably three, more preferably four, most preferably five antibody molecules of the invention or nucleic acid molecule(s) encoding said antibody molecule(s). Said composition may also comprise optimized, inventive antibodies/antibody molecules obtainable by the methods described herein below and in the appended examples.<br>
It is preferred that said pharmaceutical composition, optionally comprises a pharmaceutically acceptable carrier and/or diluent. The herein disclosed pharmaceutical composition may be particularly useful for the treatment of neurological and/or neurodegenerative disorders. Said disorders comprise, but are not limited to Alzheimer's disease, amyothrophic lateral sclerosis (ALS), hereditary cerebral hemorrhage with amyloidosis Dutch type, Down's syndrome, HIV-dementia, Parkinson's disease and neuronal disorders related to aging. The pharmaceutical composition of the invention is, inter alia, envisaged as potent inhibitors of amyloid plaque formation or as a potent stimulator for the de-polymerization of amyloid plaques. Therefore, the present invention provides for pharmaceutical compositions comprising the compounds of the invention to be used for the treatment of diseases/disorders associated with pathological APR proteolysis and/or amyloid plaque formation.<br>
Examples of suitable pharmaceutical carriers, excipients and/or diluents are well known in the art and include phosphate buffered saline solutions, water, emulsions, such as oil/water emulsions, various types of wetting agents, sterile solutions etc. Compositions   comprising   such   carriers   can   be   formulated   by   well   known<br><br>
conventional methods. These pharmaceutical compositions can be administered to the subject at a suitable dose. Administration of the suitable compositions may be effected by different ways, e.g., by intravenous, intraperitoneal, subcutaneous, intramuscular, topical, intradermal, intranasal or intrabronchial administration. It is particularly preferred that said administration is carried out by injection and/or delivery, e.g., to a site in a brain artery or directly into brain tissue. The compositions of the invention may also be administered directly to the target site, e.g., by biolistic delivery to an external or internal target site, like the brain. The dosage regimen will be determined by the attending physician and clinical factors. As is well known in the medical arts, dosages for any one patient depends upon many factors, including the patient's size, body surface area, age, the particular compound to be administered, sex, time and route of administration, general health, and other drugs being administered concurrently. Proteinaceous pharmaceutically active matter may be present in amounts between 1 ng and 10 mg/kg body weight per dose; however, doses below or above this exemplary range are envisioned, especially considering the aforementioned factors. If the regimen is a continuous infusion, it should also be in the range of 1 [jg to 10 mg units per kilogram of body weight per minute. Progress can be monitored by periodic assessment. The compositions of the invention may be administered locally or systemically. It is of note that peripherally administered antibodies can enter the central nen/ous system, see, inter alia, Bard (2000), Nature Med. 6, 916-919. Preparations for parenteral administration include sterile aqueous or non-aqueous solutions, suspensions, and emulsions. Examples of non-aqueous solvents are propylene glycol, polyethylene glycol, vegetable oils such as olive oil, and injectable organic esters such as ethyl oieate. Aqueous carriers include water, alcoholic/aqueous solutions, emulsions or suspensions, including saline and buffered media. Parenteral vehicles inciude sodium chloride solution. Ringer's dextrose, dextrose and sodium chloride, lactated Ringer's, or fixed oils. Intravenous vehicles include fluid and nutrient repienishers, electrolyte replenishers (such as those based on Ringer's dextrose), and the like. Preservatives and other additives may also be present such as, for example, antimicrobials, anti-oxidants, chelating agents, and inert gases and the like. Furthermore, the pharmaceutical composition of the invention may comprise further agents depending on the intended use of the pharmaceutical composition. Said agents may be drugs acting on the<br><br>
central nervous system, like, neuroprotective factors, cholinesterase inhibitors, agonists of M1 nnuscarinic receptor, hormones, antioxidants, inhibitors of inflammation etc. It is particularly preferred that said pharmaceutical composition comprises further agents like, e.g. neurotransmitters and/or substitution molecules for neurotransmitters, vitamin E, or alpha-lipoic acid.<br>
The pharmaceutical compositions, as well as the methods of the invention or the uses of the invention described infra can be used for the treatment of all kinds of diseases hitherto unknown or being related to or dependent on pathological APP aggregation or pathological APP processing. They may be particularly useful for the treatment of Alzheimer's disease and other diseases where extracellular deposits of amyloid-p, appear to play a role. They may be desirably employed in humans, although animal treatment is also encompassed by the methods, uses and compositions described herein.<br>
In a preferred embodiment of the invention, the composition of the present invention as disclosed herein above is a diagnostic composition further comprising, optionally, suitable means for detection. The diagnostic composition comprises at least one of the aforementioned compounds of the invention.<br>
Said diagnostic composition may comprise the compounds of the invention, in particular and preferably the antibody molecules of the present invention, in soluble form/liquid phase but it is also envisaged that said compounds are bound to/attached to and/or linked to a solid support.<br>
Solid supports may be used in combination with the diagnostic composition as defined herein or the compounds of the present invention may be directly bound to said solid supports. Such supports are well known in the art and comprise, inter alia, commercially available column materials, polystyrene beads, latex beads, magnetic beads, colloid metal particles, glass and/or silicon chips and surfaces, nitrocellulose strips, membranes, sheets, duracytes, wells and walls of reaction trays, plastic tubes etc. The compound(s) of the invention, in particular the antibodies of the present invention, may be bound to many different carriers. Examples of well-known carriers include    glass,    polystyrene,    polyvinyl   chloride,    polypropylene,    polyethylene.<br><br>
polycarbonate, dextran, nylon, amyloses, natural and modified celluloses, polyacrylamides, agaroses, and magnetite. The nature of the carrier can be either soluble or insoluble for the purposes of the invention. Appropriate labels and methods for labeling have been identified above and are furthermore mentioned herein below. Suitable methods for fixing/immobilizing said compound(s) of the invention are well known and include, but are not limited to ionic, hydrophobic, covalent interactions and the like.<br>
It is particularly preferred that the diagnostic composition of the invention is employed for the detection and/or quantification of APP and/or APP-processing products, like amy!oid-p or for the detection and/or quantification of pathological and/or (genetically) modified APP-cleavage sides.<br>
As illustrated in the appended examples, the compounds of the present invention, in particular the inventive antibody molecules are particularly useful as diagnostic reagents in the detection of genuine human amyloid plaques in brain sections of Alzheimer's Disease patients by indirect immunofluorescence.<br>
It is preferred that said compounds of the present invention to be employed in a diagnostic composition are detectably labeled. A variety of techniques are available for labeling biomolecules, are well known to the person skilled in the art and are considered to be within the scope of the present invention. Such techniques are, e.g., described in Tijssen, "Practice and theory of enzyme immuno assays", Burden, RH and von Knippenburg (Eds), Volume 15 (1985), "Basic methods in molecular biology"; Davis LG, Dibmer MD; Battey Elsevier (1990), Mayer et aL, (Eds) "Im.munochemical methods in cell and molecular biology" Academic Press, London (1987), or in the series "Methods in Enzymology", Academic Press, Inc.<br>
There are many different labels and methods of labeling known to those of ordinary skill in the art. Examples of the types of labels which can be used in the present invention include enzymes, radioisotopes, colloidal metals, fluorescent compounds, chemiluminescent compounds, and bioluminescent compounds.<br><br>
Commonly used labels comprise, inter alia, fluorochromes (like fluorescein, rhodamine, Texas Red, etc.), enzymes (like horse radish peroxidase, p-galactosidase, alkaline phosphatase), radioactive isotopes (like 32P or 125l), biotin, digoxygenin, colloidal metals, chemi- or bioluminescent compounds (like dioxetanes, luminol or acridiniums). Labeling procedures, like covalent coupling of enzymes or biotinyl groups, iodinations, phosphorylations, biotinylations, etc. are well known in the art.<br>
Detection methods comprise, but are not limited to, autoradiography, fluorescence microscopy, direct and indirect enzymatic reactions, etc. Commonly used detection assays comprise radioisotopic or non-radioisotopic methods. These comprise, inter alia, Westernblotting, overlay-assays, RIA (Radioimmuno Assay) and IRMA (Immune Radioimmunometric Assay), ElA (Enzyme Immune Assay), ELISA (Enzyme Linked Immune Sorbent Assay), FIA (Fluorescent Immuno Assay), and CLIA (Chemioluminescent Immune Assay).<br>
Furthermore, the present invention provides for the use of an antibody molecule of invention, or an antibody molecule produced by the method of the invention, of a nucleic acid molecule, vector of or a host of the invention for the preparation of a pharmaceutical or a diagnostic composition for the prevention, treatment and/or diagnosis of a disease associated with amyloidogenesis and/or amyloid-plaque formation. It is further preferred that the compounds described herein, in particular the antibody molecules of the invention, be employed in the prevention and/or treatment of neuropathologies associated with modified or abnormal APP-processing and/or amyloidogenesis. The antibody molecules, e.g in format of (engineered) immunoglobulins, like antibodies in a IgG framework, in particular in an lgG1-framework, or in the format of chimeric antibodies, bispecific antibodies, single chain Fvs (scFvs) or bispecific scFvs and the like are to employed in the preparation of the pharmaceutical compositions provided herein. Yet, the antibody molecules are also useful in diagnostic settings as documented in the appended examples, since the ariibody molecules of the invention specifically interact with/detect A(3.4 and/or arryloid deposits/plaques.<br><br>
Therefore an inventive use of the compounds of the present invention is the use for the preparation of a pharmaceutical composition for a neurological disorder which calls for amelioration, for example by disintegration of p-amyloid plaques, by amyloid (plaque) clearance or by passive immunization against p-amyloid plaque formation. As illustrated in the appended examples, the inventive antibody molecules are particularly useful in preventing A|3 aggregation and in de-polymerization of already formed amyloid aggregates. Accordingly, the inventive antibodies are to be em,ployed in the reduction of pathological amyloid deposits/plaques, in the clearance of amyloid plaques/plaque precursors as well as in neuronal protection. It is in particular envisaged that the antibody molecules of the invention be employed in the in vivo prevention of amyloid plaques as well as in in vivo clearance of pre-existing amyloid plaques/deposits. Furthermore, the antibody molecules of the invention may be employed in passive immunization approaches against AR4. Clearance of Afi4/Aft4 deposits may, inter alia, be achieved by the medical use of antibodies of the present invention which comprise an Fc-part. Said Fc-part of an antibody may be particularly useful in Fc-receptor mediated immune responses, e.g. the attraction of macrophages (phagocytic cells and/or microglia) and/or helper cells. For the mediation of Fc-part-related immunoresponses, the antibody molecule of the invention is preferably in an (human) lgG1- framework. As discussed herein, the preferred subject to be treated with the inventive antibody molecules, the nucleic acid molecules encoding the same or parts thereof, the vectors of the invention or the host cells of this invention is a human subject. Other frameworks, like lgG2a- or lgG2b-frameworks for the inventive antibody molecules are also envisaged. Imimunoglobulin frameworks in lgG2a und lgG2b fomnat are particular envisaged in mouse settings, for example in scientific uses of the inventive antibody molecules, e.g. in tests on transgenic mice expressing (human) wildtype or mutated APP, APP-fragments and/or AM.<br>
The above recited diseases associated with amyloidogenesis and/or amyloid-plaque formation comprise, but are not limited to dementia, Alzheimer's disease, motor neuropathy, Parkinson's disease, ALS (amyotrophic lateral sclerosis), scrapie, HIV-related dementia as well as Creutzfeld-Jakob disease, hereditary cerebral hemorrhage, with amyloidis Dutch type,Down's syndrome and neuronal disorders<br><br>
related to aging. The antibody molecules of the invention and	the connpositions<br>
provided  herein may also be useful  in the amelioration and	or prevention  of<br>
inflammatory processes relating to amyloidogenesis and/or	amyloid plaque<br>
formation.<br>
Accordingly, the present invention also provides for a method for treating, preventing and/or delaying neurological and/or neurodegenerative disorders comprising the step of administering to a subject suffering from said neurological and/or neurodegenerative disorder and/or to a subject susceptible to said neurological and/or neurodegenerative disorder an effective amount of a antibody molecule of the invention, a nucleic acid molecule of invention and/or a composition as defined herein above.<br>
In yet another embodiment, the present invention provides for a kit comprising at least one antibody molecule, at least one nucleic acid molecule, at least one vector or at least one host cell of the invention. Advantageously, the kit of the present invention further comprises, optionally (a) buffer(s), storage solutions and/or remaining reagents or materials required for the conduct of medical, scientific or diagnostic assays and purposes. Furthermore, parts of the kit of the invention can be packaged individually in vials or bottles or in combination in containers or multicontainer units.<br>
The kit of the present invention may be advantageously used, inter alia, for carrying out the method of the invention and could be employed in a variety of applications referred herein, e.g., as diagnostic kits, as research tools or medical tools. Additionally, the kit of the invention may contain means for detection suitable for scientific, medical and/or diagnostic purposes. The manufacture of the kits follows preferably standard procedures which are known to the person skilled in the art.<br>
The invention also provides for a method for the optimization of an antibody molecule as defined herein above comprising the steps of<br>
(a; constructing a library of diversified Fab antibody fragments derived from an antibody comprising at least one CDR3 of an Vh-region as encoded by a nucleic acid molecule as<br><br>
shown in SEQ ID NOs: 21, 23 or 25 or at least one CDR3 amino acid sequence of an VH-region as shown in SEQ ID NOs: 22, 24 or 26;<br>
(b)	testing the resulting Fab optimization library by panning against Ap/Aβ4;<br>
(c)	identifying optimized clones; and<br>
(d)	expressing of selected, optimized clones.<br>
Optimization of the antibodies/antibody molecules of the invention is also documented in the appended examples and may comprise the selection for, e.g. higher affinity for one or both regions/epitopes of β-A4 as defined herein or selection for improved expression and the like. In one embodiment, said selection for to higher affinity for one or both regions/epitopes of β-A4 comprises the selection for high affinity to (a) an amino acid stretch comprising amino acids 2 to 10 (or (a) part(s) thereof) of p-A4 and/or (b) an amino acid stretch comprising amino acids 12 to 25 (or (a) part(s) thereof) of β-A4 (SEQ (D NO. 27).<br>
* The person skilled in the art can readily carry out the inventive method employing<br>
the teachings of the present invention. Optimization protocols for antibodies are<br>
known in the art. These optimization protocols comprise, inter alia, CDR walking<br>
mutagenesis as disclosed and illustrated herein and described in Yang (1995), J.<br>
MoL Biol. 25, 392-403; Schier (1996), J. Mol. Biol. 263, 551-567; Barbas (1996),<br>
Trends. Biotech 14, 230-34 or Wu (1998), PNAS 95, 6037-6042; Schier (1996),<br>
Human Antibodies Hybridomas 7, 97; Moore (1997), J. Mol, Biol. 272, 336.<br>
"Panning"-techniques are also known in the aft, see, e.g. Kay (1993), Gene 128, 59-<br>
65.   Furthermore,   publications  like  Borrebaeck  (1995),   "Antibody  Engineering",<br>
Oxford  University,  229-266;  McCafferty (1996),  "Antibody Engineering",  Oxford<br>
University Press; Kay (1996), A Laboratory Manual, Academic Press provide for<br>
optimization protocols which may be modified in accordance with this invention.<br>
The optimization method may further comprise a step (ca), whereby the optimized clones are further optimized by cassette mutagenesis, as illustrated in the appended examples.<br>
The method for the optimization of an antibody molecule described herein is further illustrated in the appended examples as affinity maturation of parental antibodies<br><br>
/antibody molecules capable of specifically recognizing two regions of the beta -A4 peptide/ Abeta4/ Ap/Aβ4/BA4.<br>
Preferably, said Aβ/Aβ4 (also designated as (3,A4 in context of this invention) in step (b) of the method described herein above is aggregated Aβ/Aβ4. Said panning may be carried out (as described in the appended examples) with increased stringency of binding. Stringency may be increased, inter alia, by reducing the Ap/Aβ4 concentration or by elevating the (assay) temperature. The testing of the optimized library by panning is known to the skilled artisan and described in Kay (1993), loc. cit. Preferably, the identification in step (c) is carried out by ranking according to the lowest Kc-values.<br>
Most preferably said identification in step (c) is carried out by koff-ranking. Koff-ranking is known to the skilled artisan and described in Schier (1996), loc. cit.; Schier (1996), J. Mol. Biol. 255, 28-43 or Duenas (1996), MoL Immunol. 33, 279-286. Furthermore, koff-ranking is illustrated in the appended examples. The off-rate constant may be measured as described in the appended examples.<br>
As mentioned herein above, the identified clones may, for further evaluation, be expressed. The expression may be carried out by known methods, inter alia, illustrated in the appended examples. The expression may, inter alia, lead to expressed Fab-fragments, scFvs, bispecific immunoglobulins, bispecific antibody molecules, Fab- and/or Fv fusion proteins, or full antibodies, like IgGs, in particular IgGI.<br>
Optimized antibodies, in particular optimized Fabs or optimized IgGs, preferably IgGIs, may be tested by methods as illustrated in the appended examples. Such methods comprise, but are not limited to, the testing of binding affinities, the determination of KD values, pepspot anaylysis, ELISA-assays, RIA-assays, CLIA-<br>
assays, (immune-) histological studies (for example staining of amyloid plaques), de-polymerization assays or antibody-dependent β-A4 phagocytoses.<br>
In a further embodiment of the present invention, a method is provided wherein optimized antibodies are generated by cross-cloning. This method is also illustrated in the appended examples and comprises the step of combining independently<br><br>
optimized CDR-regions, for example, by combining independently optimized H-CDR2 and L-CDR2. from matured clones with H-CDR3, preferably the same H-CDR3.<br>
In a preferred embodiment, the invention relates to a method for the preparation of a pharmaceutical composition comprising the steps of<br>
(a)	optimization of an antibody according to the method described herein and<br>
illustrated in the appended examples; and<br>
(b)	formulating the optimized antibody/antibody molecule with an physiologically<br>
acceptable carrier, as described herein above.<br>
Accordingly, the invention also provides for a pharmaceutical composition prepared by the method disclosed herein and comprising further optimized antibody molecules capable of specifically recognizing two regions of the beta-A4 petide/Abeta4/A(i/A4(i/f2,A4, as described herein above.<br><br><br><br>
SEQiDN0:5<br>
VH-region of MS-Roche#7 (amino acid sequence)<br>
QVQLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEWVSAISGS GGSTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARGKGNTHKPYGY VRYFDVWGQGTLVTVSS (SEQ ID NO: 6)<br>
SEQ ID NO; 7<br>
VH-region of MS-Roche#8 (nucleic acid sequence)<br>
CAGGTGCAATTGGTGGAAAGCGGCGGCGGCCTGGTGCAACCGGGCGGCAGC<br>
CTGCGTCTGAGCTGCGCGGCCTCCGGATTTACCTTTAGCAGCTATGCGATGAG<br>
CTGGGTGCGCCAAGCCCCTGGGAAGGGTCTCGAGTGGGTGAGCGCGATTAGC<br>
GGTAGCGGCGGCAGCACCTATTATGCGGATAGCGTGAAAGGCCGTTTTACCAT<br>
TTCACGTGATAATTCGAAAAACACCCTGTATCTGCAAATGAACAGCCTGCGTGC<br>
GGAAGATACGGCCGTGTATTATTGCGCGCGTCTTCTTTCTCGTGGTTATAATGG<br>
TTATTATCATAAGTTTGATGTTTGGGGCCAAGGCACCCTGGTGACGGTTAGCTC<br>
AGC (SEQ ID NO: 7)<br>
SEQ ID NO: 8<br>
VH-region of MS-Roche#8 (amino acid sequence)<br>
QVQLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEWVSAISGS GGSTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARLLSRGYNGYYH KFDVWGQGTLVTVSS (SEQ ID NO: 8)<br>
SEQ ID NO: 9<br>
VL-region of MS-Roche#3 (nucleic acid sequence)<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTGTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGAGCGTGAGCAGCAGCTATCTGGG<br>
GTGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGA<br>
GCAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCAC<br>
GGATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGGTTTATTA<br>
TTGCCAGCAGGTTTATAATCCTCCTGTTACCTTTGGCCAGGGTACGAAAGTTGA<br>
AATTAAACGTACG (SEQ ID NO: 9)<br>
SEQ ID NO: 10<br>
VL-region of MS-Roche #3 (amino acid sequence)<br>
DIVLTQSPATLSLSPGERATLSCRASQSVSSSYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTiSSLEPEDFAVYYCQQVYNPPVTFGQGTKVEIKRT (SEQ ID NO: 10)<br>
SEQ ID NO: 11<br>
VL-region of MS-Roche#7 (nucleic acid sequence)<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGAGCGTGAGCAGCAGCTATCTGGC<br><br>
GTGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGA GCAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCAC GGATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGACTTATTA TTGCTTTCAGCTTTATTCTGATCCTTTTACCTTTGGCCAGGGTACGAAAGTTGAA ATTAAACGTACG (SEQ ID NO. 11)<br>
SEQIDNO: 12<br>
VL-region of MS-Roche#7 (amino acid sequence)<br>
DIVLTQSPATLSLSPGERATLSCRASQSVSSSYLAWYQQKPGQAPRLLIYGASSRA<br>
TGVPARFSGSGSGTDFTLTISSLEPEDFATYYCFQLYSDPFTFGQGTKVEIKRT<br>
(SEQ ID NO: 12)<br>
SEQ IDNO: 13<br>
VL-region of MS-Roche#8 (nucleic acid sequence)<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAG<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGAGCGTGAGCAGCAGCTATCTGGC<br>
GTGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGA<br>
GCAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCAC<br>
GGATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGACTTATTA<br>
TTGCCAGCAGCTTTCTTCTTTTCCTCCTACGTTTGGCCAGGGTACGAAAGTTGA<br>
AATTAAACGTACG (SEQ ID NO: 13)<br>
SEQ ID NO: 14<br>
VL-region of MS-Roclie#8 (amino acid sequence)<br>
DIVLTQSPATLSLSPGERATLSCRASQSVSSSYLAWVYQQKPGQAPRLLIYGASSRA<br>
TGVPARFSGSGSGTDFTLTISSLEPEDFATYYCQQLSSFPPTFGQGTKVEIKRT<br>
(SEQ ID NO: 14)<br>
SEQ IDNO: 15<br>
CDR3 of Vt-region of MSR-3 (nucleic acid sequence)<br>
ICAGCAGGTTTATAATCCTCCTGTTl (SEQ IDNO: 15)<br>
SEQ ID NO: 16<br>
CDR3 of Vt-region of MSR-3 (amino acid sequence)<br>
QQVYNPPV (SEQ ID NO: 16)<br>
SEQIDNO: 17<br>
CDR3 of Vt-region of MSR-7 (nucleic acid sequence)<br>
TTTCAG CTTTATTCTGATCCTTTTi<br>
(SEQ ID NO: 17)<br><br>
SEQ ID NO: 18<br>
CDR3 of VL-region of MSR-7 (amino acid sequence)<br>
FQLYSDPF(SEQ IDNO. 18)<br>
SEQ ID NO: 19<br>
CDR3 of Vi-region of MSR-8 (nucleic acid sequence)<br>
CAG C AG CTT TCTTCT TTT CCT CCT<br>
(SEQ ID NO. 19)<br>
SEQ ID NO: 20<br>
CDR3 of VL-region of MSR-8 (amino acid sequence)<br>
QQLSSFPP (SEQ ID NO: 20)<br>
SEQ ID NO: 21<br>
CDR of Vn-region of MSR-3 (nucleic acid sequence)<br>
CTTACTCATTATGCTCGTTATTATGGTTATTTTGATGTT<br>
(SEQ ID NO: 21)<br>
SEQ ID NO: 22<br>
CDR of VH-region of MSR-3 (amino acid sequence)<br>
LTHYARYYRYFDV (SEQ ID NO: 22)<br>
SEQ ID NO: 23<br>
CDR of VH-region of MSR-7 (nucleic acid sequence)<br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT<br>
(SEQ ID NO: 23)<br>
SEQ ID NO: 24<br>
CDR of Vh-region of MSR-7 (amino acid sequence)<br>
GKGNTHKPYGYVRYFDV (SEQ ID NO: 24)<br>
SEQ ID NO: 25<br>
CDR of VH-region of MSR-8 (nucleic acid sequence)<br><br>
CTT CTTTCT CGT GGTTAT AAT GGTTATTAT CATAAG TTT GAT GTT<br>
(SEQIDNO. 25)<br>
SEQ1DN0:26<br>
CDR ofVH-region of MSR-8 (amino acid sequence)<br>
LLSRGYNGYYHKFDV (SEQ ID NO: 26)<br>
SEQ ID NO: 27 A[34 (amino acids 1 to 42) DAEFRHDSGYEVHHQKLVFFAEDVGSNKGAIIGLMVGGWIA (SEQ ID NO: 27)<br>
SEQ ID NO: 28 primer 5-GTGGTGGTTCCGATATC-3' (SEQ ID NO: 28)<br>
SEQ ID NO: 29 primer<br>
5-- AGCGTCACACTCGGTGCGGCTTTCGGCTGGCCAAGAACGGTTA-3' (SEQ ID<br>
NO: 29)<br>
SEQ ID NO: 30 primer 5'-CAGGAAACAGCTATGAC-3' (SEQ ID NO: 30)<br>
SEQIDNO: 31 primer 5'-TACCGTTGCTCTTCACCCC-3' (SEQ ID NO: 31)<br>
SEQ ID NO: 32 VH of MS-Roche#3.6H5 x 3.6L2; DMA; artificial sequence<br>
CAATTGGTGGAAAGCGGCGGCGGCCTGGTGCAACCGGGCGGCAGCCTGCGTC<br>
TGAGCTGCGCGGCCTCCGGATTTACCTTTAGCAGCTATGCGATGAGCTGGGTG<br>
CGCCAAGCCCCTGGGAAGGGTCTCGAGTGGGTGAGCGCTATTTCTGAGTCTG<br>
GTAAGACTAAGTATTATGCTGATTCTGTTAAGGGTCGTTTTACCATTTCACGTGA<br>
TAATTCGAAAAACACCCTGTATCTGCAAATGAACAGCCTGCGTGCGGAAGATAC<br>
GGCCGTGTATTATTGCGCGCGTCTTACTCATTATGCTCGTTATTATCGTTATTTT<br>
GATGTTTGGGGCCAAGGCACCCTGGTGACGGTTAGGTCA (SEQ ID NO: 32)<br><br><br><br><br><br>
QLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEWVSAINAVGM KKFYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARGKGNTHKPYGYVR YFDVWGQGTLVTVSS (SEQ ID NO: 41)<br>
SEQ ID NO: 42 VH region of MS-Roche#7.11H1 x 7.11L1; DNA; artificial sequence<br>
CAATTGGTGGAAAGCGGCGGCGGCCTGGTGCAACCGGGCGGCAGCCTGCGTC<br>
TGAGCTGCGCGGCCTCCGGATTTACCTTTAGCAGCTATGCGATGAGCTGGGTG<br>
CGCCAAGCCCCTGGGAAGGGTCTCGAGTGGGTGAGCGGTATTAATGCTGCTG<br>
GTTTTCGTACTTATTATGCTGATTCTGTTAAGGGTCGTTTTACCATTTCACGTGA<br>
TAATTCGAAAAACACCCTGTATCTGCAAATGAACAGCCTGCGTGCGGAAGATAC<br>
GGCCGTGTATTATTGCGCGCGTGGTAAGGGTAATACTCATAAGCCTTATGGTTA<br>
TGTTCGTTATTTTGATGTTTGGGGCCAAGGCACCCTGGTGACGGTTAGCTCA<br>
(SEQ ID NO: 42)<br>
SEQ ID NO. 43 prot VH region of MS-Roche#7.11H1 x 7.11 LI; protein/1; artificial sequence<br>
QLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEWVSGINAAGF RTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARGKGNTHKPYGYVR YFDVWGQGTLVTVSS (SEQ ID NO: 43)<br>
SEQ ID NO: 44 VH region of MS-Roche#7.11 H1 x 7.2L1; DNA; artificial sequence<br>
CAATTGGTGGAAAGCGGCGGCGGCCTGGTGCAACCGGGCGGCAGCCTGCGTC<br>
TGAGCTGCGCGGCCTCCGGATTTACCTTTAGCAGCTATGCGATGAGCTGGGTG<br>
CGCCAAGCCCCTGGGAAGGGTCTCGAGTGGGTGAGCGGTATTAATGCTGCTG<br>
GTTTTCGTACTTATTATGCTGATTCTGTTAAGGGTCGTTTTACCATTTCACGTGA<br>
TAATTCGAAAAACACCCTGTATCTGCAAATGAACAGCCTGCGTGCGGAAGATAC<br>
GGCCGTGTATTATTGCGCGCGTGGTAAGGGTAATACTCATAAGCCTTATGGTTA<br>
TGTTCGTTATTTTGATGTTTGGGGCCAAGGCACCCTGGTGACGGTTAGCTCA<br>
(SEQ ID NO: 44)<br>
SEQ ID NO: 45 prot VH region of MS-Roche#7.11H1 x 7.2L1; protein/1; artificial sequence<br><br>
QLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEVWSGINAAGF RTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARGKGNTHKPYGYVR YFDVWGQGTLVTVSS (SEQ ID NO: 45)<br>
SEQ ID NO: 46 VL region of MS-Roche#3.6H5 x 3.6L2; DNA; artificial sequence<br>
GATATCGTGCTGACCCAGAGCCCGGCGACGCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGTTTCTTTCTCGTTATTATCTGGCGT<br>
GGTACCAGCAGAAACCAGGTCAAGCACCGGGTCTATTAATTTATGGCGCGAGC<br>
AGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCACGG<br>
ATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGGTTTATTATTG<br>
CCAGCAGACTTATAATTATCCTCCTACCTTTGGCCAGGGTACGAAAGTTGAAAT<br>
TAAACGTACG (SEQ ID NO: 46)<br>
SEQ ID NO:47 prot VL region of MS-Roche#3.6H5 x 3.6L2; protein/1; artificial sequence<br>
DIVLTQSPATLSLSPGERATLSCRASQFLSRYYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTISSLEPEDFAVYYCQQTYNYPPTFGQGTKVEIKRT<br>
(SEQ ID NO: 47)<br>
SEQ ID NO: 48 VL region of MS-Roche#3.6H8 x 3.6L2; DNA; artificial sequence GATATCGTGCTGACCGAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC GTGCGACCCTGAGCTGCAGAGCGAGCCAGTTTCTTTCTCGTTATTATCTGGCGT GGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGAGC AGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGGACGG •      ATTTTACCCTGACCATTAGGAGCCTGGAACCTGAAGACTTTGCGGTTTATTATTG CCAGCAGACTTATAATTATCCTCCTACCTTTGGCCAGGGTACGAAAGTTGAAAT TAAACGTACG (SEQ ID NO: 48)<br>
SEQ ID NO: 49 prot VL region of MS-Roche#3.6H8 x 3.6L2; protein/1; artificial sequence<br>
DiVLTQSPATLSLSPGERATLSCRA.SQFLSRYYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTISSLEPEDFAVYYCQQTYNYPPTFGQGTKVEIKRT (SEQ ID NO: 49)<br><br>
SEQ ID NO: 50 VL region of MS-Roche#7.4H2 x 7.2L1; DNA; artificial sequence<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGTATGTTGATCGTACTTATCTGGCG<br>
TGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGAG<br>
CAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGGACG<br>
GATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGACTTATTATT<br>
GCCAGCAGATTTATTCTTTTCCTCATACCTTTGGCCAGGGTAGGAAAGTTGAAAT<br>
TAAACGTACG (SEQ ID NO: 50)<br>
SEQ ID NO: 51  prot VL region of MS-Roche#7.4H2 x 7.2L1; protein/1; artificial sequence<br>
DIVLTQSPATLSLSPGERATLSCRASQYVDRTYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTISSLEPEDFATYYCQQIYSFPHTFGQGTKVEIKRT<br>
(SEQ ID NO: 51)<br>
SEQ ID NO: 52 VL region of MS-Roche#7.9H2 x 7.12 L2; DNA; artificial sequence<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGCGTTTTTTTTATAAGTATCTGGCGT<br>
GGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTCTGGTTCTTCTA<br>
ACCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCACGGA<br>
TTTTACCCTGACCATTAGCAGCGTGGAACCTGAAGACTTTGCGGTTTATTATTGC<br>
CTTCAGCTTTATAATATTCCTAATACCTTTGGCCAGGGTACGAAAGTTGAAATTA<br>
AACGTACG (SEQ ID NO: 52)<br>
SEQ ID NO: 53 prot VL region of MS-Roche#7.9H2 x 7.12 L2; protein/1; artificial sequence<br>
DIVLTQSPATLSLSPGERATLSCRASQRFFYKYLAWYQQKPGQAPRLLISGSSNRA TGVPARFSGSGSGTDFTLTiSSLEPEDFAVYYCLQLYNlPNTFGQGTKVEIKRT (SEQ ID NO: 53)<br>
SEQ ID NO: 54 VL region of MS-Roche#7.9H4 x 7.12L2; DNA; artificial sequence<br><br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGCGTTTTTTTTATAAGTATCTGGCGT<br>
GGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTCTGGTTCTTCTA<br>
ACCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCACGGA<br>
TTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGGTTTATTATTGC<br>
CTTCAGCTTTATAATATTCCTAATACCTTTGGCCAGGGTACGAAAGTTGAAATTA<br>
AACGTACG (SEQ ID NO: 54)<br>
SEQ ID NO: 55 prot VL region of MS-Roche#7.9H4 x 7.12L2; protein/1; artificial sequence<br>
DiVLTOSPATLSLSPGERATLSCRASQRFFYKYLAWYQQKPGQAPRLLISGSSNRA TGVPARFSGSGSGTDFTLTISSLEPEDFAVYYCLQLYNIPNTFGQGTKVEIKRT<br>
(SEQ ID NO: 55)<br>
SEQ ID NO: 56 VL region of MS-Roche#7.11 HI x 7.11 L1; DNA; artificial sequence<br>
GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC<br>
GTGCGACCCTGAGCTGCAGAGCGAGCCAGCGTATTCTTCGTATTTATCTGGCG<br>
TGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGAG<br>
CAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCACG<br>
GATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGACTTATTATT<br>
GCCAGCAGGTTTATTCTCCTCCTCATACCTTTGGCCAGGGTACGAAAGTTGAAA<br>
TTAAACGTACG (SEQ ID NO: 56)<br>
SEQ ID NO: 57 prot VL region of MS-Roche#7.11H1 x 7.11 LI; protein/1; artificial sequence<br>
DiVLTQSPATLSLSPGERATLSCRASQRILRIYIJWVYQQKPGQAPRLLIYGASSRAT GVPARFSGSGSGTDFTLTISSLEPEDFATYYCQQVYSPPHTFGQGTKVEIKRT (SEQ ID NO: 57)<br>
SEQ ID NO: 58 VL region of MS-Roche#7.11H1 x7.2L1; DNA; artificial sequence GATATCGTGCTGACCCAGAGCCCGGCGACCCTGAGCCTGTCTCCGGGCGAAC GTGCGACCCTGAGCTGCAGAGCGAGCCAGTATGTTGATCGTAGTTATCTGGCG TGGTACCAGCAGAAACCAGGTCAAGCACCGCGTCTATTAATTTATGGCGCGAG<br><br>
CAGCCGTGCAACTGGGGTCCCGGCGCGTTTTAGCGGCTCTGGATCCGGCACG GATTTTACCCTGACCATTAGCAGCCTGGAACCTGAAGACTTTGCGACTTATTATT GCCAGCAGATTTATTCTTTTCCTCATACCTTTGGCCAGGGTACGAAAGTTGAAAT TAAACGTACG (SEQ ID NO: 58)<br>
SEQ ID NO: 59 prot VL region of MS-Roche#7.11H1 x 7.2L1; protein/1; artificial sequence<br>
DIVLTQSPATLSLSPGERATLSCRASQYVDRTYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTISSLEPEDFATYYCQQIYSFPHTFGQGTKVEIKRT<br>
(SEQ ID NO: 59)<br>
SEQ   ID  NO:  60  HCDR3  region  of IVlS-Roche#3.6H5 x 3.6L2;  DNA;   artificial<br>
sequence<br>
CTTACTCATTATGCTCGTTATTATCGTTATTTTGATGTT (SEQ ID NO: 60)<br>
SEQ ID NO: 61 prot HCDR3 region of MS-Roche#3.6H5 x 3.6L2; protein/1; artificial<br>
sequence<br>
LTHYARYYRYFDV (SEQ ID NO: 61)<br>
SEQ  ID  NO:  62  HCDR3  region  of MS-Roche#3.6H8 x 3.6L2;   DNA;  artificial<br>
sequence<br>
CTTACTCATTATGCTCGTTATTATCGTTATTTTGATGTT (SEQ ID NO: 62)<br>
SEQ ID NO: 63 prot HCDR3 region of MS-Roche#3.6H8 x 3.6L2; protein/1; artificial<br>
sequence<br>
LTHYARYYRYFDV (SEQ ID NO: 63)<br>
SEQ  ID  NO:  64  HCDR3 region  of MS-Roche#7.4H2 x 7.2L1;  DNA;  artificial sequence<br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT   (SEQ ID NO: 64)<br><br>
SEQ ID NO; 65 prot HCDR3 region of MS-Roche#7.4H2 x 7.2L1; protein/1; artificial<br>
sequence<br>
GKGNTHKPYGYVRYFDV (SEQ ID NO: 65)<br>
SEQ ID NO; 56 HCDR3 region of MS-Roche#7.9H2 x 7.12 L2, DNA; artificial sequence<br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT   (SEQ ID NO: 66)<br>
SEQ  ID NO: 67 prot HCDR3 region of#i\/lS-Roche 7.9H2 x 7.12 L2;  protein/1; artificial sequence GKGNTHKPYGYVRYFDV (SEQ ID NO: 67)<br>
SEQ  ID  NO:  68  HCDR3  region  of MS-Roche#7.9H4 x 7.12L2;  DNA;   artificial sequence<br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT   (SEQ ID NO: 68)<br>
SEQ ID NO: 69 prot HCDR3 region of MS-Roche#7.9H4 x 7.12L2; protein/1; artificial<br>
sequence<br>
GKGNTHKPYGYVRYFDV (SEQ ID NO: 69)<br>
SEQ ID NO; 70 HCDR3 region of MS-Roche#7.11H1  x 7.11L1; DNA; artificial sequence<br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT  (SEQ ID NO: 70)<br>
SEQ ID NO: 71  prot HCDR3 region of MS-Roche#7.11H1 x 7.11L1; protein/1; artificial sequence GKGNTHKPYGYVRYFDV (SEQ ID NO: 71)<br>
SEQ  ID NO:  72 HCDR3 region of MS-Roche#7.11H1  x 7.2L1;  DNA;  artificial sequence<br><br>
GGTAAGGGTAATACTCATAAGCCTTATGGTTATGTTCGTTATTTTGATGTT   (SEQ<br>
ID NO: 72)<br>
SEQ ID NO: 73 prot HCDR3 region of MS-Roche#7.11H1 x7.2L1; protein/1; artificial<br>
sequence<br>
GKGNTHKPYGYVRYFDV (SEQ ID NO: 73)<br>
SEQ   ID  NO:  74  LCDR3  region  of MS-Roche#3.6H5  x 3.6L2;   DNA;   artificial<br>
sequence<br>
CAGCAGACTTATAATTATCCTCCT (SEQ ID NO: 74)<br>
SEQ ID NO: 75 prot LCDR3 region of MS-Roche#3.6H5 x 3.6L2; protein/1; artificial<br>
sequence<br>
QQTYNYPP (SEQ ID NO: 75)<br>
SEQ   ID  NO:   76   LCDR3  region  of MS-Roche#3.6H8  x  3.6L2;   DNA;   artificial<br>
sequence<br>
CAGCAGACTTATAATTATCCTCCT (SEQ ID NO: 76)<br>
SEQ ID NO: 77 prot LCDR3 region of MS-Roche#3.6H8 x 3.6L2; protein/1; artificial<br>
sequence<br>
QQTYNYPP (SEQ ID NO: 77)<br>
SEQ  ID  NO:  78  LCDR3  region  of MS-Roche#7.4H2 x 7.2L1;  DNA;  artificial<br>
sequence<br>
CAGCAGATTTATTCTTTTCCTCAT (SEQ ID NO: 78)<br>
SEQ ID NO: 79 prot LCDR3 region of MS-Roche#7.4H2 x 7.2L1; protein/1; artificial<br>
sequence<br>
QQIYSFPH (SEQ ID NO: 79)<br>
SEQ ID NO: 80 LCDR3 region of MS-Roche#7.9H2 x 7.12 L2; DNA; artificial sequence<br><br>
CTTCAGCTTTATAATATTCCTAAT (SEQ ID NO: 80)<br>
SEQ ID NO: 81  prot LCDR3 region of MS-Roche#7.9H2 x 7.12 L2; protein/1;<br>
artificial sequence<br>
LQLYNIPN(SEQIDN0:81)<br>
SEQ  ID NO:  82  LCDR3 region of MS-Roche#7.9H4 x 7.12L2;  DMA;  artificial<br>
sequence<br>
CTTCAGCTTTATAATATTCCTAAT (SEQ ID NO: 82)<br>
SEQ ID NO: 83 prot LCDR3 region of MS-Roche#7.9H4 x 7.12L2; protein/1; artificial<br>
sequence<br>
LQLYNIPN (SEQ ID NO: 83)<br>
SEQ ID NO: 84 LCDR3 region of MS-Roche#7.11H1  x 7.11 LI;  DNA; artificial<br>
sequence<br>
CAGCAGGTTTATTCTCCTCCTCAT (SEQ ID NO: 84)<br>
SEQ ID NO: 85 prot LCDR3 region of MS-Roche#7.11H1  x 7.11 LI; protein/1;<br>
t<br>
artificial sequence QQVYSPPH (SEQ ID NO: 85)<br>
SEQ ID NO:  86  LCDR3 region of MS-Roche#7.11H1   x 7.2L1;  DNA;  artificial<br>
sequence<br>
CAGCAGATTTATTCTTTTCCTCAT (SEQ ID NO: 86)<br>
SEQ ID NO: 87 prot LCDR3 region of MS-Roche#7.11 HI x 7.2L1; protein/1; artificial<br>
sequence<br>
QQIYSFPH (SEQ ID NO: 87)<br>
SEQ ID NO: 88 VH region of MS-Roche#7.9H7; DNA; artificial sequence<br>
Caggtgcaattggtggaaagcggcggcggcctggtgcaaccgggcggcagcctgcgtctgagctgcgcggcctc<br>
cggatttacctttagcagctatgcgatgagctgggtgcgccaagcccctgggaagggtctcgagtgggtgagcgctat<br><br>
taatgcttctggtactcgtacttattatgctgattctgttaagggtcgttttaccatttcacgtgataattcgaaaaacaccctg tatctgcaaatgaacagcctgcgtgcggaagatacggccgtgtattattgcgcgcgtggtaagggtaatactcataag ccttatggttatgttcgttattttgatgtttggggccaaggcaccctggtgacggttagctca (SEQ ID NO: 88)<br>
SEQ ID NO: 89 prot VH region of MS-Roche#7.9H7; protein/1; artificial sequence QVQLVESGGGLVQPGGSLRLSCAASGFTFSSYAMSWVRQAPGKGLEWVSAINAS GTRTYYADSVKGRFTISRDNSKNTLYLQMNSLRAEDTAVYYCARGKGNTHKPYGY VRYFDVWGQGTLVTVSS (SEQ ID NO: 89)<br>
SEQ ID NO: 90 VL region of MS-Roche#7.9H7; DNA; artificial sequence<br>
Gatatcgtgctgacccagagcccggcgaccctgagcctgtctccgggcgaacgtgcgaccctgagctgcagagcg<br>
agccagagcgtgagcagcagctatctggcgtggtaccagcagaaaccaggtcaagcaccgcgtctattaatttatg<br>
gcgcgagcagccgtgcaactggggtcccggcgcgttttagcggctctggatccggcacggattttaccctgaccatta<br>
gcagcctggaacctgaagactttgcgacttattattgccttcagatttataatatgcctattacctttggccagggtacgaa<br>
agttgaaattaaacgtacg (SEQ ID NO: 90)<br>
SEQ ID NO: 91 prot VL region of MS-Roche#7.9H7; protein/1; artificial sequence DIVLTQSPATLSLSPGERATLSCRASQSVSSSYLAWYQQKPGQAPRLLIYGASSRA TGVPARFSGSGSGTDFTLTISSLEPEDFATYYCLQIYNMPITFGQGTKVEIKRT (SEQ ID NO: 91)<br>
SEQ ID NO: 92 HCDR3 region of MS-Roche#7.9H7; DNA; artificial sequence Ggtaagggtaatactcataagccttatggttatgttcgttattttgatgtt (SEQ ID NO: 92)<br>
SEQ  ID  NO:  93  prot HCDR3  region  of MS-Roche#7.9H7;  protein/1;  artificial<br>
sequence<br>
GKGNTHKPYGYVRYFDV (SEQ ID NO: 93)<br>
SEQ ID NO: 94 LCDR3 region of MS-Roche#7.9H7; DNA; artificial sequence Cttcagatttataatatgcctatt (SEQ ID NO: 94)<br>
SEQ  ID  NO:  95  prot  LCDR3  region  of MS-Roche#7.9H7;  protein/1;  artificial sequence<br><br>
LQIYNMPI (SEQ ID NO: 95)<br>
Further illustrative sequences are depicted in the appended sequence listing and are also shown in the appended tables, in particular tables 1, 8 and 10.<br>
The Figures show:<br>
Figure 1       Sequence summary of HuCAL®-Fab1 Library<br>
The numbering is according .to VBASE except the gap in VLA position 9, In VBASE the gap is set at position 10 {Chothia et ai, 1992). In the sequence summary all CDR3 residues which were kept constant are indicated. Corresponding sequences employed for the HuCAL-Fab1 library can be found in the appended sequence listing. A: amino acid sequence B: DNA sequence<br>
Figure 2       Fab display vector pM0RPH®18_Fab<br>
Vector map and DNA sequence including restriction sites<br>
Figure 3       Fab expression vector pM0RPH®x9_Fab<br>
Vector map and DNA sequence including restriction sites<br>
Figure 4       Sequences of the parental Fab fragments MS-Roche-3, MS-Roche-7 and MS-Roche 8<br>
A: amino acid sequence B: DNA sequence<br>
Figure 5;      Indirect   immunofluorescence   of  amyloid-plaques   from   a   cryostat<br>
section of human temporal cortex. The plaques were labeled with MS-R # 3.2 Fab (upper panels) and MS-R # 7.4 Fab (lower panels) at 20 jig/ml (left panels) and 5 jig/ml (right panels) under stringent blocking conditions. Bound MS-R Fab was revealed by goat anti-human-Cy3.<br><br>
Figure 6: Indirect immunofluorescence of amyloid-plaques trom a cryostat section of human temporal cortex. The plaques were labeled with MS-R # 3.3 lgG1 (upper panels) and MS-R # 7.12 lgG1 (lower panels) at 0.05 Lig/ml (left panels) and 0.01 jig/ml (right panels) under stringent blocking conditions. Bound MS-R lgG1 antibody was revealed by goat anti-human (H+L)-Cy3.<br>
Figure 7: Indirect immunofluorescence of amyloid-plaques from a cryostat section of human temporal cortex using antibodies after final affinity maturation. The plaques were labeled with MS-R # 7.9.H7 IgGI (MAB 31, top panel), MS-R # 7.11.H1x7.2-L1 IgGI (MAB 11, middle panel) and MS-R # 3.4.H7, bottom panel). Antibodies were used at 0.05 (ig/ml (left panels) and 0.01 ug/ml (right panels) under stringent blocking conditions. Bound MS-R IgGI antibody was revealed by goat anti-human (H+L)-Cy3. Scale: 8,5 mm = 150 ym.<br>
Figures: Polymerization Assay. Anti-Aβ antibodies prevent incorporation of biotinylated Aβ into preformed Aβ aggregates.<br>
Figure 9: De-polymerization Assay. Anti-Aβ antibodies induce release of biotinylated Aβ from aggregated Ap.<br>
Figure 10: In vivo decoration of amyloid plaques in an APP/PS2 double transgenic mouse after intravenous injection of 1mg MS-Roche IgG #7.9.H2 x 7.12.L2. After three days the mouse was perfused with phosphate-buffered saline and sacrificed. The presence of human IgG bound to amyloid plaques was revealed by confocal microscopy after labelling cryostat sections from the frontal cortex with a goat anti-human IgG-Cy3 conjugate (panel B). The same section was counterstained with an anti-Abeta mouse monoclonal antibody (BAP-2-Alexa488 conjugate, panel A) to visualize the position of amyloid plaques. Individual red<br><br>
{panel B) and green (panel A) channels, merged image (panel D) and colocalized (pancel C) signals are shown. Scale: 1 cm = 50 pm<br>
Figure 11;    In vivo decoration of amyloid plaques in an APP/PS2 double transgenic mouse after intravenous injection of 1mg MS-Roche IgG #7.9.H4 x 7.12.L2. Experimental conditions and staining procedure were identical to those described in the legend of figure 10. Scale: 1.6 cm = 50 pm<br>
Figure 12:    In vivo decoration of amyloid plaques in an APP/PS2 double transgenic mouse after intravenous injection of 1mg MS-Roche IgG #7.11.HI x 7.2.LI (MAB 11). Experimental conditions and staining procedure were identical to those described in the legend of figure 10. Scale: 1.4 cm = 70 pm<br>
Figure 13: In vivo decoration of amyloid plaques in an APP/PS2 double transgenic mouse after intravenous injection of 2 mg MS-Roche IgG #7.9.H7 (MAB 31) at day 0, 3, and 6. After nine days the mouse was perfused with phosphate-buffered saline and sacrificed. The presence of human IgG bound to amyloid plaques was revealed by confocal microscopy after labelling cryostat sections from the frontal cortex with a goat anti-human lgG-Cy3 conjugate (panel B). The same section was counterstained with an anti-Abeta mouse monoclonal antibody (BAP-2-Alexa488 conjugate, panel A) to visualize the position of amyloid plaques. Individual red (panel B) and green (panel A) channels, merged image (panel D) and colocalized (panel C) signals and are shown. Scale: 1.6 cm = 80 pm (panels A, B, C); 1.0 cm = 50 ym (panel D)<br>
Figure 14:    In vivo decoration of amyloid plaques in an APP/PS2 double transgenic<br>
mouse after intravenous injection of 2 nng MS-Roche IgG #7.11.HI x 7.2.L1  (MAB 11) at day 0, 3 and 6.  Experimental conditions and<br><br>
staining procedure were identical to those described in the legend of<br>
figure 13.<br>
Scale: 1.6 cm = 80 \im<br>
Figure 15: Binding analysis of anti-A[3 antibodies to cell surface APP. Antibody binding to human APP-transfected HEK293 cells and non-transfected control cells was analyzed ^y flow cytometry.<br>
The examples illustrate the invention.<br>
Example 1: Construction and Screening of a Human Combinatorial Antibody Library (HuCAL®-Fab1)<br>
Cloning of HuCAL®-Fab 1<br>
HuCAL®-Fab 1  is a fully synthetic, modular human antibody library in the Fab<br>
antibody fragment format. HuCAL-Fab 1 was assembled starting from an antibody<br>
library in the single-chain format (HuCAL-scFv; Knappik,(2000), J. Mol. Biol. 296,<br>
57-86).<br>
VA positions 1 and 2. The original HuCAL® master genes were constructed with their<br>
authentic N-termini: VU1: QS (CAGAGC), VU2: QS (CAGAGG), and VU3: SY<br>
(AGCTAT). Sequences containing these amino acids are shown in WO 97/08320.<br>
During HuCAL® library construction, the first two amino acids were changed to Dl to<br>
facilitate library cloning {EcoRI site). All HuCAL® libraries contain VLX genes with the<br>
Ecof?\/site GATATC (Dl) at the 5'-end. All HuCAL® kappa genes (master genes and<br>
all genes in the library) contain Dl at the 5'-end (figure 1 A and B).<br>
VH position 1. The original HuCAL® master genes were constructed with their<br>
authentic N-termini: VH1A, VH1B, VH2, VH4, and VH6 with Q (=CAG) as the first<br>
amino acid and VH3 and VH5 with E (=GAA) as the first amino acid. Sequences<br>
containing these amino acids are shown in WO 97/08320. During cloning of the<br>
HuCAL®-Fab1 library, amino acid at position 1 of VH was changed to Q (CAG) in all<br>
VH genes (figure 1 A and B).<br>
Design of the CDR libraries<br><br>
VK1/VK3 position 85. Because of the cassette mutagenesis procedure used to introduce the CDR3 library {Knappik. (2000), loc, cit), position 85 of VK1 and VK3 can be either! or V. Thus, during HuCAL®-scFv1 library construction, position 85 of VK1 and VK3 was varied as follows: VK1 original, 85T (codon ACC); VK1 library, 85T or 85V (TRIM codons ACT or GTT); VK3 original, 85V (codon GTG); VK3 library, 85T or 85V (TRIM codons ACT or GTT); the same applies to HuCAL®-Fab1. CDR3 design. All CDR3 residues, which were kept constant, are indicated in figure 1 A and B.<br>
CDR3 length. The designed CDR3 length distribution is as follows. Residues, which were varied are shown in brackets (x) in figure 1. V kappa CDR3, 8 amino acid residues (position 89 to 96) (occasionally 7-10 residues), with Q89, S90, and D92 fixed; and VH CDR3, 5 to 28 amino acid residues (position 95 to 102) (occasionally 4-28), with D101 fixed.<br>
HuCAL®-Fab 1 was cloned into a phagemid expression vector pM0RPH®18_Fab1 (figure 2). This vector comprises the Fd fragment with a phoA signal sequence fused at the C-terminus to a truncated gene III protein of filamentous phage, and further comprises the light chain VL-CL with an ompA signal sequence. Both chains are under the control of the lac operon. The constant domains C^ CK and CH1 are synthetic genes fully compatible with the modular system of HuCAL® {Knappik, (2000), loa cit).<br>
The whole VH-chain ('Mu/7//S(y'/-fragment) was replaced by a 1205 bp dummy fragment containing the p-lactamase transcription unit (bla), thereby facilitating subsequent steps for vector fragment preparation and allowing for selection of complete VH removal.<br>
After VH-replacement, VU was removed by EcoRI/Dralll and VLK by EcoRI/BslWI and replaced with bacterial alkaline phosphatase (bap) gene fragment (1420 bp). As the variability of the light chains is lower than that of the heavy chains, cloning was started with the light chain libraries. The VU and VUlight chain libraries diversified in L-CDR3, which were generated for the HuCAL®-scFv library (Knappik, (2000), loc. cit.) were also used for cloning of HuCAL®-Fab1. In case of Xthey consisted of the 511-, X2- and X,3-HuCAL®-frannework and had a total variability of 5.7<br><br>
As quality control the light chain and heavy chain of single clones was sequenced with 5'"CAGGAAACAGCTATGAC-3' (SEQ ID NO: 30) and 5'-TAGCGTTGCTCTTCACCCC'3' (SEQ ID NO: 31), respectively.<br>
Phagemid rescue, phage amplification and purification<br>
HuCAL®-Fab 1 was amplified in 2 x TY medium containing 34 pg/ml chloramphenicol, 10 pg/ml tetracycline and 1 % glucose (2 x TY-CG). After helper phage infection (VCSM13) at 37'C at an ODeoo of about 0,5, centrifugation and resuspension in 2 x TY / 34 pg/ml chloramphenicol / 50 pg/ml kanamycin cells were grown overnight at 30'C. Phage were PEG-precipitated from the supernatant (Ausubel, (1998), Current protocols in molecular biology. John Wiley &amp; Sons, Inc., New York, USA), resuspended in PBS/20% glycerol and stored at -80°C. Phage amplification between two panning rounds was conducted as follows: mid-log phase TGI-cells were infected with eluted phage and plated onto LB-agar supplemented with 1% of glucose and 34 pg/ml of chloramphenicol. After overnight incubation at 30'C colonies were scraped off, adjusted to an ODQOO of 0.5 and helper phage added as described above.<br>
Example 2: Solid phase panning<br>
Wells of MaxiSorp^'^ microtiterplates F96 (Nunc) were coated with 100 jii 2.5 jiM human Aj3 (1-40) peptide (Bachem) dissolved in TBS containing NaHz (0.05% v/v) and the sealed plate was incubated for 3 days at 37 °C where the peptide is prone to aggregate on the plate. After blocking with 5% non-fat dried milk in TBS, 1-5 x 10^^ HuCAL®-Fab phage purified as above were added for 1h at 20X. After several washing steps, bound phages were eluted by pH-el.ution with 500 mM NaCI, 100 mM glycin pH 2.2 and subsequent neutralisation with 1M TRIS-Cl pH 7. Three rounds of panning were performed with phage amplification conducted between each round as described above, the washing stringency was increased from round to round.<br>
Example 3; Subcloning of selected Fab fragments for expression<br><br>
The Fab-encoding inserts of the selected HuCAL-Fab fragments were subcloned into the expression vector pM0RPH®x7_FS to facilitate rapid expression of soluble Fab. The DNA preparation of the selected HuCAL®-Fab clones was digested with Xbal/EcoRI, thus cutting out the Fab encoding insert (ompA-VL and phoA-Fd). Subcloning of the purified inserts into the Xbal/EcoRI cut vector pM0RPH®x7, previously carrying a scFv insert, leads to a Fab expression vector designated pi\/IORPH®x9_Fab1 (figure 3). Fabs expressed in this vector carry two C-terminal tags (FLAG and Strep) for detection and purification.<br>
Example 4: Identification of Ap-binding Fab fragments by ELISA<br>
Wells of MaxisorpTM microtiterplates F384 (Nunc) were coated with 20ul 2.5 uM human Ap (1-40) peptide (Bachem) dissolved in TBS containing NaNa (0.05% v/v) and the sealed plate was incubated for 3 days at 37 .C, where the peptide is prone to aggregate on the plate. Expression of individual Fab was induced with 1 mM IPTG for 16 h at 22X. Soluble Fab was extracted from E. coli by BEL lysis (boric acid, NaCI, EDTA and lysozyme containing buffer pH 8) and used in an ELISA. The Fab fragment was detected with an alkaline phosphatase-conjugated goat anti-Fab antibody (Dianova/Jackson immune Research). After excitation at 340 nm the emission at 535 nm was read out after addition of AttoPhos fluorescence substrate (Roche Diagnostics).<br>
Example 5: Optimization of antibody fragments<br>
In order to optimize the binding affinity of the selected Aβ binding antibody fragments, some of the Fab fragments, MS-Roche-3 (MSR-3), MS-Roche-7 (MSR-7) and MS-Roche-8 (MSR-8) (figure 4), were used to construct a library of Fab antibody fragments by replacing the parental VL K3 chain by the pool of all kappa chains K1-3 diversified in CDR3 from the HuCAL® library (Knappik et aL, 2000). The Fab fragments MS-Roche-3, 7 and 8 were cloned via Xbal/EcoRI from pM0RPH®x9 JFS into pM0RPH®18, a phagemid-based vector for phage display of<br><br>
The Fab-encoding inserts of the selected HuCAL®-Fab fragments were subcloned into the expression vector pM0RPH"x7_FS to facilitate rapid expression of soluble Fab. The DNA preparation of the selected HuCAL®-Fab clones was digested with Xbal/EcoRI, thus cutting out the Fab encoding insert (ompA-VL and phoA-Fd). Subcloning of the purified inserts into the Xbal/EcoRI cut vector pM0RPH®x7, previously carrying a scFv insert, leads to a Fab expression vector designated pM0RPH®x9_Fab1 (figure 3). Fabs expressed in this vector carry two C-terminal tags (FLAG and Strep) for detection and purification.<br>
Example 4: Identification of Ap-binding Fab fragments by ELISA<br>
Wells of Maxisorp™ microtiterplates F384 (Nunc) were coated with 20jil 2.5 uM human Aβ (1-40) peptide (Bachem) dissolved in TBS containing NaNa (0.05% v/v) and the sealed plate was incubated for 3 days at 37 X, where the peptide is prone to aggregate on the plate. Expression of individual Fab was induced with 1 mM IPTG for 16 h at 22°C. Soluble Fab was extracted from E coli by BEL lysis (boric acid; NaCI, EDTA and lysozyme containing buffer pH 8) and used in an ELISA. The Fab fragment was detected with an alkaline phosphatase-conjugated goat anti-Fab antibody (Dianova/Jackson Immune Research). After excitation at 340 nm the emission at 535 nm was read out after addition of AttoPhos fluorescence substrate (Roche Diagnostics).<br>
Example 5: Optimization of antibody fragments<br>
in order to optimize the binding affinity of the selected Aβ binding antibody fragments, some of the Fab fragments, MS-Roche-3 (MSR-3), MS-Roche-7 (MSR-7) and MS-Roche-8 (MSR-8) (figure 4), were used to construct a library of Fab antibody fragments by replacing the parental VL K3 chain by the pool of all kappa chains K1-3 diversified in CDR3 from the HuCAL® library {Knappik et ai, 2000). The Fab fragments MS-Roche-3, 7 and 8 were cloned via Xbal/EcoRI from pM0RPH®x9 _FS into pM0RPH18. a phagemid-based vector for phage display of<br><br>
Fab fragments, to generate pM0RPH®18_Fab1 (figure 2). A kappa chain pool was cloned into pM0RPH®18_Fab1 via Xbal/SphI restriction sites.<br>
The resulting Fab optimization library was screened by panning against aggregated human Ap (1-40) peptide coated to a solid support as described in example 2. Optimized clones were identified by koff-ranking in a Biacore assay as described in Example 8. The optimized clones MS-Roche-3.2. 3.3. 3.4. 3.6. 7.2. 7.3. 7.4. 7.9. 7.11. 7.12. 8.1, 8.2. were further characterized and showed improved affinity and biological activity compared to the starting fragment MS-Roche-3, MS-Roche-7 and MS-Roche-8 (figure 4). The CDRs listed refer to the HuCAL® consensus-based antibody gene VH3kappa3. The Fab fragment MS-Roche-7.12 was obtained by cloning the HCDR3 of parental clone MS-R 7 into a HuCAL®-Fab library, carrying diversity in all 6 CDR regions using a design procedure identical with that for CDR3 cassettes described in Knappik et al., 2000. The library cassettes were designed strongly b^iased for the known natural distribution of amino acids and following the concept of canonical CDR conformations established by Allazikani (Allazikani et al., 1997). However in contrast to the HuCAL® master genes, the clone MS-Roche 7.12" contains amino acid S at position 49 of the VL chain (see appended table 1).<br>
The optimized Fabs after the first affinity maturation round showed improved characteristics over the starting l\/lS-Roche-3, MS-Roche-7 and MS-Roche-8 clones (Figure 4). The binding affinities of the maturated Fabs to Ap1-40 and A|31-42 were significantly increased yielding Ko values in the range of 22 - 240 nM in comparison to 850 - 1714 nM of the parental clones (Table 3). Immunohistochemistry analysis of amyloid plaques in human AD brain sections also showed a significantly increased staining profile of the maturated clones, 1. e. better signal to background ratios were obtained and positive plaque staining was detected at relatively low concentrations of the maturated Fabs (Figure 5).<br>
For further optimization, the VH CDR2 regions and the VL CDR1 regions of a set of antibody fragments derived from L-CDR3 optimized MS-Roche-3, -7 and -8 (table 1; figure 4) were optimized by cassette mutagenesis using trinudeotide-directed mutagenesis (Virnekas et al.,   1994).  Therefore,  a trinucleotide-based  HCDR2<br><br>
cassette and a trinucleotide-based LCDR1 cassette were constructed using a design procedure identical with that for CDR3 cassettes described in Knappik et ai, 2000, The library cassettes were designed strongly biased for the known natural distribution of amino acids and following the concept of canonical CDR conformations established by Allazikani (Allazikani et al., 1997). The protocol used for the optimization of the initial selected antibody fragments would mimic the process of affinity maturation by somatic hypermutation observed during the natural immune response.<br>
The resulting libraries were screened separately as described above leading to optimized clones either in the H-CDR2 or in the L-CDR1 region. All clones were identified as above by an improved koff towards Aβi-40-fibers after a koff-ranking in the Biacore and showed improved affinity either to Aβ1-40 or Aβ-42 or both when compared to the corresponding parent clone (Table 3). Table 1 contains the sequence characteristics of the parental as well as sequences of the optimized clones. The CDRs listed refer to the HuCAL® consensus-based antibody gene VH3kappa3.<br>
For example, the affinity of the MS-Roche-7 parental Fab towards Ab1-40 was improved over 35-fold from 1100 nM to 31 nM after L-CDR3 optimization (MS-Roche-7.9) and further improved to 5 nM after H-CDR2 optimization (MS-Roche-7.9H2) as illustrated in Table 3.<br>
The H-CDR2 and L-CDR1 optimization procedure not only increased the affinity but also resulted for some of the clones in a significantly improved staining of amyloid plaques in AD brain section, as particularly seen with MS-Roche 7.9H2 and 7.9H3.<br><br><br><br><br><br><br><br><br><br><br><br>
Example 6<br>
Construction of HuCAL immunoglobulin expression vectors Heavy chain cloning. The multiple cloning site of pcDNA3.1+ (invitrogen) was removed (Nhel/Apal), and a stuffer compatible with the restriction sites used for HuCAL® design was inserted for the ligation of the leader sequences (Nhel/EcoRI), VH-domains (Muni/), and the immunoglobulin constant regions {BIpl/Apal), The leader sequence (EMBL 83133) was equipped with a Kozak sequence (Kozak, 1987). The constant regions of human IgG (PIR A02146), lgG4 (EMBL KOI 316), and serum lgA1 (EMBL J00220) were dissected into overlapping oligonucleotides with length of about 70 bases. Silent mutations were introduced to remove restriction sites non-compatible with the HuCAL® design. The oligonucleotides were spliced by overlap extension-PCR.<br>
During sub-cloning from Fab into IgG, the VH DNA sequence of the Fab is cut out via Mfe I / BIp I and ligated into the IgG vector opened via EcoR I / BIp 1. EcoR I (g/aattc) and Mfe I (c/aattg) share compatible cohesive ends {aatt) and the DNA sequence of the original Mfe I site in the Fab changes from: c/aattg to: g/aattg after ligation into the IgG expression vector, thereby destroying both Mfe I and EcoR I site, and thus also leading to an amino acid change from Q (codon: caa) to E (codon: gaa).<br>
Light chain cloning. The multiple cloning site of pcDNA3.1/Zeo+ (Invitrogen) was replaced by two different stuffers. The K-stuffer provided restriction sites for insertion of a K-leader (Nhel/EcoRV), HuCAL®-scFv VK-domains {EcoRV/BsiWI), and the K-chain constant region (BsiWI/Apal), The corresponding restriction sites in the X-stuffer were Nhel/EcoRV (X,-leader), EcoRV/Hpal (VX-domains), and Hpal/Apal {X-chain constant region). The K-Ieader (EMBL Z00022) as well as the X-leader (EMBL J00241) were both equipped with Kozak sequences. The constant regions of the human K- (EMBL L00241) and /.-chain (EMBL Ml 8645) were assembled by overlap extension-PCR as described above.<br>
Generation of IgG-expressing CHO-cells. CH0-K1 cells weYe co-transfected with an equimolar mixture of IgG heavy and light chain expression vectors. Double-resistant<br><br>
transtectants were seiectea witn 600 ug/ml G418 ana 300 ug/mi zeocm (invitrogen) followed by limiting dilution. The supernatant of single clones was assessed for IgG expression by capture-ELISA. Positive clones were expanded in RPMl-1640 nnedium supplennented with 10% ultra-low IgG-FCS (Life Technologies). After adjusting the pH of the supernatant to 8.0 and sterile filtration, the solution was subjected to standard protein A column chromatography (Poros 20 A, PE Biosystems).<br>
Example 7: Pepspot analysis with decapeptides<br>
The following aminoacid sequence encompassing Aβ (1-42) was divided into 43<br>
overlapping decapeptides with a frameshift of 1 aminoacid.<br>
ISEVKM1DAEF RHDSGYEVHH QKLVFFAEDV GSNKGAllGL MVGGWl42ATV IV<br>
(SEQ    ID    NO:    414).    Accordingly,    DAEF    RHDSGYEVHH    QKLVFFAEDV<br>
GSNKGAllGL MVGGWIA (SEQ ID NO: 27) as enclosed represents amino acids 1<br>
to 42 of Aβ4/β-A4 peptide.<br>
The 43 decapeptides were synthesized with N-terminal acetylation and C-terminal<br>
covalent attachment to a cellulose sheet f pepspot") by a commercial supplier (Jerini<br>
BioTools, Berlin). The cellulose sheet is incubated for 2 hours on a rocking platform<br>
with monoclonal antibody (2 pg/ml) in blocking buffer (50 mM TrisHCI, 140 mM<br>
NaCI, 5 mM NaEDTA, 0.05% NP40 (Fluka), 0.25% gelatine (Sigma), 1% bovine<br>
serum albumine fraction V (Sigma), pH 7.4). The sheet is washed 3 times 3 minutes<br>
on a rocking platform with TBS (10 mM Tris.HCI, 150 mM NaCI, pH 7.5). It is then<br>
wetted with cathode buffer (25 mM Tris base, 40 mM 6-Aminohexane acid, 0.01%<br>
SDS, 20% methanol) and transfered to a semi-dry blotting stack with the peptide<br>
side facing a PVDF membrane (Biorad) of equal size.<br>
The semi-dry blotting stack consists out of freshly wetted filter papers (Whatman<br>
No.3) slightly larger than the peptide sheet:<br>
3 papers wetted with Cathode buffer<br>
the peptide sheet<br>
a sheet of PVDF membrane wetted with methanol<br>
3 papers wetted with Anode buffer 1 (30mM Tris base, 20% methanol)<br>
3 papers wetted with Anode buffer 2 (0.3 mM Tris base, 20% methanol)<br><br>
The transfer is conducted at a current density between Cathode and Anode of 0.8 nnA/cm2 for 40 minutes which is sufficient to elute nnost of the antibody from the cellulose sheet and deposit it on the PVDF membrane. The PVDF membrane is then exchanged for a 2nd PVDF membrane and transferred for another 40 minutes to ensure complete elution from the cellulose sheet.<br>
The PVDF membrane is immersed in blocking buffer for 10 minutes. Then HRP-labeied anti-human Ig H+L (Pierce) is added at 1:1000 dilution and the membrane is incubated on a rocking platform for 1 hour. It is washed 3x10 minutes with TBST (TBS with 0.005% Tween20). Color is developed by immersing the membrane into a solution made of 3 mg 4-chloronaphthol dissolved in 9 ml methanol with 41 ml PBS (20 mM Na-phosphate, 150 mM NaCI, pH 7.2) an 10 pi 30% hydrogen peroxide (Merck). After the development of blue-black spots the membrane is washed extensively with water and dried.<br>
The assignment of antibody-reactive pepspots is made by visual inspection through a transparent spot matrix. The epitopes of the antibody in question is defined as the minimal aminoacid sequence in reactive peptides. For comparison mouse monoclonal antibodies (BAP-2, BAP-1, BAP-17 BAP-21, BAP-24, and 4G8) are analyzed in the same way, except using HRP-labeled anti-mouse Ig instead of anti-human Ig.<br>
it is of note that affinity maturation and conversion of the monovalent Fab fragments into full-length lgG1 antibodies results usually in some broadening of the epitope recognition sequence as indicated by pepspot and ELISA analyses. This may be related to the recruitment of more contact points in the antibody-antigen interaction area as a consequence of the affinity maturation or to a stronger binding to the minimal epitope such that also weak interactions with adjacent amino acid can be detected. The latter may be the case when Ap-derived peptides are probed with full-length IgG antibodies. As illustrated in Table 2 for the pepspot analysis, the recognition sequences of the N-terminal and middle epitopes are extended by up to three amino acids when parent Fabs and corresponding fully maturated IgG antibodies are compared. However, it has to be kept in mind that the decapeptides are modified for covalent attachment at the C-terminal amino acid and this amino acid may therefore not easily be accessible to the full-length antibody due to steric<br><br><br><br><br><br>
active Fab in each preparation was deternnined in the Biacore according to a method described by Christensen, Analytical Biochemistry (1997) 249, 153 -164. Briefly, tinne-dependent protein binding to Ap1-40 fibers immobilized on the Biacore chip was measured during the association phase under mass-limited conditions at different flow rates of the analyte solution. The conditions of mass limitation were realized by immobilizing high amounts of Aβ fibers (2300 response units) on the chip surface of a measuring channel and by working at relatively low analyte concentrations, i. e. 160 nM (based on the total Fab protein concentration). A summary of the KD values of selected MS-Roche clones identified in the primary screen of the HuCAL library and their corresponding maturated derivatives after the 1st and 2nd affinity maturation cycle is shown in Table 3. In the 1®* affinity maturation cycle the heavy chain CDR3 (VH-CDR3) was kept constant and optimization was focussed on diversification of the light chain CDR3 (VL-CDR3). In the 2nd affinity cycle diversification of VL-CDR1 and VH-CDR2 was performed. Some of the binders from the 1st maturation cycle were converted to full-length human IgGI antibodies according to the technology developed by MorphoSys as described in Example 6 and KD values determined in the Biacore as described above. The KD values for full-length lgG1 binding to Ap1-40 and A|31-42 fibers are shown in Table 4.<br>
Matured derivatives from both the L-CDR1 as well as H-CDR2 library after the 2nd maturation cycle were identified and allowed combination of light and heavy chains. The cross-cloning strategy is described in Example 13. Either whole light chains, LCDR1 or L-CDR1+2 were exchanged. Ko values of selected cross-cloned Fabs are shown in Table 8.<br>
Some of the Fabs from the 1®* and 2nd maturation cycles and from the cross-cloned binders were converted to full-length human igG1 antibodies according to the technology developed by MorphoSys as described in Example 6. KD values of IgG binding to Ap1-40 and Aβi-42 fibers were determined in the Biacore. Briefly, a kinetic model for the stepwise formation of a bivalent complex was used, and KD values were calculated by Scatchard type analysis of equilibrium binding. Due to the very slow association process at low antibody concentration (several hours to reach equilibrium) equilibrium binding data were obtained by extrapolation of the association curves to long time intervals. The on- and off rates for the formation of<br><br>
the monovalent and bivalent complex were determined via the curve fit procedure and used for the extrapolation. Based on these Req values a Scatchard analysis was performed and KD values for the formation of the monovalent and the bivalent complex were determined. The data are summarized in Table 5. From the curvilinear Scatchard plot a higher (bivalent) and lower (monovalent) affinity interaction was derived for the MS-R IgGs derived from the 2nd affinity maturation cycle and cross-clones. These two affinities represent the lower and upper KD values of the range indicated in Table 5.<br><br><br><br>
Table 3: KD values forMS-R Fab binding to Aβ31-40 and Aβi-42 fibers as determined in tile Biacore. For tine ciones derived from the 1st and 2nd affinity maturation cycle the values are corrected for the content of active Fab present in each sample as described in the text. , values were calculated from the concentration-dependent sensor responses at equilibrium; n.d., not determined.<br><br>
Table 4: KD values for MS-R lgG1 binding to Aβ1-40 and Aβ1-42 fibers as determined in the Biacore. The IgGs were derived from MS-R Fabs selected after the 1st affinity maturation cycle. The values are corrected for the content of active MS-R IgGs present in each sample as described in the text.<br><br><br>
Table 5: KD values for MS-R lgG1 binding to AI31-40 and A|31-42 fibers as determined in the Biacore. The IgGs were derived from MS-R Fabs selected after the 1st and 2nd affinity maturation cycle and from crosscloned Fabs. The values are corrected for the content of active MS-R IgGs present in each sample as described in the text. The two KQ values given for MS-R IgGs derived from the 2nd affinity maturation step and cross-cloned binders represent higher and lower affinity interaction as calculated from the curvilinear Scatchard plots. With a number of additional MS-R IgGs (for example MS-R IgG 7.9,H2x7.12.L2 and MS-R IgG 7.9.H4x7.12.L2), complex curvilinear Scatchard blots were obtained and determination of Ko-values was therefore not possible.<br>
Example 9: Staining of genuine human amyloid plaques in brain sections of an Alzheimer's Disease patient by indirect immunofluorescence<br><br>
Selected MS-Roche Fabs and full-length lgG1 were tested for binding to p-amyloid plaques by immunohistochemistry analysis. Cryostat sections of unfixed tissue from human temporal cortex (obtained postmortem from a patient that was positively diagnosed for Alzheimer's disease) were labeled by indirect immunofluorescence using MS-Roche Fabs or full-length human lgG1 antibodies at various concentrations. Fabs and lgG1 antibodies were revealed by goat anti-human affinity-purified F(ab')2 fragment conjugated to Cy3 and goat anti-human (H+L) conjugated to Cy3, respectively. Both secondary reagents were obtained from Jackson Immuno Research. Controls included an unrelated Fab and the secondary antibodies alone, which all gave negative results. Typical examples of plaque stainings with selected MS-Roche Fabs and MS-Roche lgG1 antibodies are shown in Figures 5 to 7.<br>
Example 10: Polymerization Assay: Prevention of Ap aggregation<br>
Synthetic Aβ when incubated in aqueous buffer over several days spontaneously aggregates and forms fibrillar structures which are similar to those seen in amyloid deposits in the brains of Alzheimer's Disease patients. We have developed an in vitro assay to measure incorporation of biotinylated Ap into preformed Ap aggregates in order to analyze the Ap-neutralizing potential of anti-Ap antibodies and other Ap-binding proteins such as albumin (Bohrmann et al., 1999, J. Biol. Chem. 274, 15990-15995). The effect of small molecules on Ap aggregation can also be analyzed in this assay.<br>
Experimental procedure:<br>
NUNC Maxisorb microtiter plates (MTP) are coated with a 1:1 mixture of Ap1-40 and Ap1-42 (2 [iM each, 100 u\ per well) at 37°C for three days. Under these conditions highly aggregated, fibrillar Ap is adsorbed and immobilized on the surface of the well. The coating solution is then removed and the plates are dried at room temperature for 2-4 hours. (The dried plates can be stored at -20°C). Residual binding sites are blocked by adding 300 ^il/well phosphate-buffered saline containing 0.05 % Tween 20 (T-PBS) and 1 % bovine serum albumin (BSA). After 1-2 hours incubation at room temperature the plates are washed 1 x with 300 |j.l T-PBS. A<br><br>
solution of 20 nM biotinylated Ap1 -40 in 20 mM Tris-HCI. 150 mM NaCI pH 7.2 (TBS) containing 0.05 % NaNs and serially diluted antibody is added (100 µl/well) and the plate incubated at 37.C overnight. After washing 3 x with 300 µl T-PBS a streptavidin-POD conjugate (Roche Molecular Biochemicals), diluted 1:1000 in T-PBS containing 1% BSA, is added (100 µl/well) and incubated at room temperature for 2 hours. The wells are washed 3 x with T-PBS and 100 )µl/well of a freshly prepared tetramethyl-benzidine (TMB) solution are added. [Preparation of the TMB solution: 10 ml 30 mM citric acid pH 4.1 (adjusted with KOH) + 0.5 ml TMB (12 mg TMB in 1 ml acetone + 9 ml methanol) + 0.01 ml 35 % H2O2]. The reaction is stopped by adding 100 .µl/well 1 N H2SO4 and absorbance is read at 450 nm in a microtiter plate reader.<br>
Result:<br>
Figure 8 shows that MS-Roche IgGI antibodies prevented inorporation of biotinylated Ap1-40 into preformed Ap1-40/Ap1-42 aggregates. The Ap-neutralizing capacity of these full-length human IgGs was similar to that of the mouse monoclonal antibody BAP-1 which had been generated by a standard immunization procedure and specifically recognizes amino acid residues 4-6 of the Appeptide when analyzed by the Pepspot technique as described in example 7. Mouse monoclonal antibody BAP-2 which also reacts exclusively with amino acids 4-6 (Brockhaus, unpublished) was significantly less active in this assay. An even lower activity was found with the Api-40 C-terminal specific antibody BAP-17 (Brockhaus, Neuroreport 9 (1998), 1481-1486) and the monoclonal antibody 4G8 which recognizes an epitope between position 17 and 24 in the Ap sequence (Kim, 1988, Neuroscience Research Communication Vol. 2, 121-130). BSA at a concentration of up to 10 |ig/ml did not affect incorporation of biotinylated Ap and served as a negative control. However, at higher concentrations; i. e. &gt; 100 µg/ml, BSA has been reported to inhibit binding of biotinylated Ap into preformed Ap fibers (Bohrmann, (1999) J Biol Chem 21A (23), 15990-5) indicating that the interaction of BSA with Ap is not of high affinity.<br><br>
Example 11: De-polymerization Assay: Release of biotinylated Aβ from aggregated Ap<br>
In a similar experimental setup we have tested the potential of MS-Roche IgG antibodies to induce depolymerization of aggregated Aβ. Biotinylated Aβ1-40 was first incorporated into preformed Aβ1-40/Aβ1-42 fibers before treatment with various anti-Aβ antibodies. Liberation of biotinylated Aβ was measured using the same assay as described in the polymerization assay.<br>
Experimental procedure:<br>
NUNC Maxisorb microtiter plates (MTP) are coated with a 1:1 mixture of Ap1-40 and Api-42 as described in the polymerization assay. For incorporation of biotinylated Ap the coated plates are incubated with 200 |il/well 20 nM biotinylated Aβ1-40 in TBS containing 0.05 % NaNa at 37°C overnight. After washing the plate with 3 x 300 ul/well T-PBS, antibodies serially diluted in TBS containing 0.05 % NaNa were added and incubated at 37°C for 3 hours. The plate was washed and analyzed for the presence of biotinylated Ap1-40 as described above.<br>
Result:<br>
Figures 9A to D shows that the inventive antibodies induced de-polymerization of aggregated Ap as measured by the release of incorporated biotinylated Ap1-40. The MS-R antibodies and the mouse monoclonal antibody BAP-1 were similarly active whereas the BAP-2, BAP-17 and 4G8 antibodies were clearly less efficient in liberating biotinylated Ap from the bulk of immobilized Ap aggregates. BAP-1 can clearly be differentiated from the MS-R antibodies by its reactivity with cell surface full-length APP (see Figure 15), and antibodies like BAP-1 with such properties are not useful for therapeutic applications as potential autoimmunological reactions may ' be induced. It is interesting to note that BAP-2, despite its specificity for amino acid residue 4-6 which is exposed in aggregated Ap has a clearly lower activity in this assay indicating that not all N-terminus specific antibodies a priori are equally efficient in releasing Ap from preformed aggregates. The MS-Roche IgGs are clearly<br><br><br><br><br><br>
antibodies were analysed in the same way, except using as revealing agent HRP-labelled anti-mouse Ig instead of anti-human Ig.<br>
Employing specific of the above described heptapeptides derived from Ap, specific ELlSA-tests as described herein above were carried out. Preferably, inventive antibodies comprise antibodies which show, as measured by of optical densities, a signal to background ratio above "10" when their reactivity with an A-beta derived peptide (AEFRHD; amino acid 2 to 7 of A-beta) is compared to an non-related protein/peptide like BSA. Most preferably, the ratio of optical densities is above "5" for a corresponding reaction with at least one of the following three Aβ derived peptides: (VFFAED; amino acid 18 to 23 of Ap) or (FFAEDV; amino acid 19 to 24 of Ap) or (LVFFAE; amino acid 17 to 22 of Ap).<br>
Corresponding results for the inventive parental and/or maturated antibodies are shown in the following two tables:<br><br><br><br><br><br><br><br>
step. For a further improvement of affinity the independently optimized H-CDR2 and L-CDR1 from matured clones with the same H-CDR3 were combined, because there was a high probability that this combination would lead to a further gain of affinity (Yang et al., 1995, J.Mol.Bioi. 254, 392-403; Schier et al., 1996b, J.Mol.Biol. 263, 551-567; Chen et al., 1999. J.Mol.Biol. 293, 865-881). Whole light chains, or fragments thereof, were transferred from an L-CDR1 optimized donor clone to a H-CDR2 optimized recipient clone. Donor and recipient clones were only combined, if both carried identical H-CDR3 sequences. All donor and recipient clones carried the VH3-VK3 framework.<br>
This was accomplished by transferring whole light chains from the L-CDR1-optimized donor clone to the H-CDR2-optimized recipient clone. Epitope specificity was conserved by only combining clones with the same H-CDR3. By light chain exchange a H-CDR2-optimiz5d clone obtained only an optimized L-CDR1, if the exchange occured between clones with the same L-CDR3. If the L-CDR3 of the clones to be combined was different, the H-CDR2-optimized clone acquired in addition to the optimized L-CDR1 another L-CDR3 (L-CDR2 remained the HuCAL consensus sequence (Knappik et al., 2000)) and when derivatives of MS-Roche #7.12 were used as donors of the light chain L-CDR1, 2 and 3 were exchanged in the H-CDR2-optimized acceptor clone. Three different cloning strategies were employed:<br>
1)	Using restriction endonudeases Xbal and SphI the whole antibody light chain fragment was excised from plasmid 1 (e.g. pMx9_Fab_MS-Roche#7.11.H1_FS) and the thereby obtained vector backbone was then ligated to the light chain fragment of plasmid 2 (e.g. pMx9_Fab_MS"Roche#7.2.L1_FS) generated by Xbal and SphI digest. Thereby a new plasmid (nomenclature: pMx9_Fab_MS-Roche#7.11.H1x7.2.L1_FS) was created encoding L-CDR1,2,3 of parental clone #7.2.L1 and H-CDR1,2.3 of parental clone #7.11 .H1,<br>
2)	Using restriction endonudeases Xbal and Acc65l an L-CDR1 coding fragment was excised from plasmid 1 (e.g. pMx9_Fab_MS-Roche#7.11.H2_FS) and the thereby obtained vector backbone was then ligated to the L-CDR1 fragment of plasmid 2 (e.g. pMx9_Fab_MS-Roche#7.12.L1_FS) generated by Xbal and Acc65l.     Thereby     a     new     plasmid     (nomenclature:     pMx9_Fab_MS-<br><br>
Roche#7.11.H2x7,12.L1(L-CDR1LFS) was created encoding L-CDR1 of parental clone #7.12.L1 while L"CDR2,3 and H-CDR1,2,3 are derived from parental clone #7.11.H2.<br>
3) Using restriction endonucleases Xbal and BamHI an L-CDR1 and L-CDR2<br>
coding fragment was excised from plasmid 1 (e.g. pMx9_Fab_MS-<br>
Roche#7.11 .H2_FS) and the thereby obtained vector backbone was then ligated<br>
to the L-CDR1 and L-CDR2 fragment of plasmid 2 (e.g. pMx9_Fab_MS-<br>
Roche#7.12.L1_FS) generated by Xbal and BamHI digest. Thereby a new<br>
plasmid	(nomenclature:	pl\/lx9_Fab_MS-Roche#7.11.H2x7.12.L1(L-<br>
CDR1+2LFS) was created encoding L-CDR1 and L-CDR2 of parental clone #7.12.LI while L-CDR3 and H-CDR1,2,3 are derived from parental clone #7.11.H2.<br>
Illustrative examples for the different cloning strategies as well as for sequences<br>
donor and recipient clones are given in table 8.<br>
After large scale expression and purification their affinities were determined on Aβ<br>
(1-40) fibers. Furthermore, Ko values for selected cross-cloned MS-R Fab/antibodies<br>
are given in appended Table 9.<br><br><br><br><br><br><br><br><br><br><br><br><br>
Table 9: KD values for crosscloned MS-R Fab binding to A|31-40 and Ap1-42 fibers as determined in the Biacore. The preparation of crosscloned Fabs is described in example 13. The KD values were determined by kinetic curve fittings and corrected for the content of active Fab present in each sample as described in the text. Some of the Fabs were additionally purified by size exclusion chromatography or preparative ultracentrifugation to remove aggregated material. (L1), the H-CDR2-matured acceptor clone received only L-CDR1 from the L-CDR1 improved donor clone; (L1+2), the H-CDR2-matured acceptor done received L-CDR1+2 from the L-CDR1 improved donor clone.<br><br>
Example 14: In vivo amyloid plaque decoration in a mouse model of Alzheimer's disease as revealed by confocal laser scanning microscopy and coiocalization analysis.<br>
Selected MS-R lgG1 antibodies were tested in APP/PS2 double transgenic mice (Reference: Richards et al., Soc. Neurosci. Abstr., Vol. 27, Program No, 5467, 2001) for amyloid plaque decoration in vivo. The antibodies (1 mg/mouse) were administered i.v. and after 3 days the brains were perfused with saline and prepared for cryosection. In another study the mice were exposed to higher concentrations of the antibodies, i.e. 2 mg injected i.v. at day 0, 3, and 6, and sacrificed at day nine. The presence of the antibodies bound to amyloid plaques was assessed on unfixed cryostat sections by double-labeled indirect immunofluorescence using goat anti-human IgG (H+L) conjugated to either Cy3 (#109-165-003, Jackson Immuno Research) followed by BAP-2-Alexa488 immunoconjugate. Imaging was done by confocal laser microscopy and image processing for quantitative detection of colocalizations by IMARIS and COIOCALIZATION software (Bitplane, Switzerland). Typical examples are shown in Figures 10-14, All of the MS-R antibodies tested were found positive in immunodecoration of amyloid plaques in vivo, although some variability was noted.<br>
Example 15: Investigation of binding of different monoclonal antibodies to amyloid precursor protein (APR) on the surface of HEK293 cells:<br>
APP is widely expressed in the central nervous system. Binding of antibody to cell surface APP may lead to complement activation and cell destruction in healthy brain areas. Therefore, it is mandatory for therapeutic A-beta antibodies to be devoid of reactivity towards APP. High affinity antibodies against the N-terminal domain of A-beta (e.g. BAP-1, BAP-2) recognize the respective epitope also in the framework of APP. In contrast, the antibodies against the middle epitope (e.g. 4G8), and the antibodies of the invention are surprisingly unable to recognize to cell surface APP. Thus, antibodies of the invention which decorate A-beta, but not APP in vivo, are superior to non-selective antibodies.<br><br>
The method of flow cytometry is well known in the art. Relative units of fluorescence (FL1-H) measured by flow cytometry indicate cell surface binding of the respective antibody. A fluorescence shift on APP transfected HEK293 compared to untransfected HEK293 cells indicates the unwanted reaction with cell surface APP. As an example, antibodies BAP-1 and BAP-2 against the N-terminai domain show a significant shift of FL-1 signal in HEK293/APP (thick line) compared to untransfected HEK293 cells (dotted line). The 4G8 antibody (specific for the middle A-beta epitope) and all antibodies of the invention (specific for N-terminal and middle A-beta epitopes) show no significant shift in fluorescence. Differences in basal fluorescence between HE293/APP ad HEK293 cells are due to different cell size. A FACScan instrument was used in combination with the Cellquest Pro Software package (both Becton Dickinson).<br>
Example 16: List of identified SEQ ID NOs relating to inventive antibody molecules<br>
The appended table 10 relates to sequences as defined herein for some specific inventive antibody molecules.<br><br><br><br><br><br><br><br>
Claims<br>
1.	An antibody molecule capable of specifically recognizing two regions of the β-A4 peptide/Aβ4, wherein the first region comprises the amino acid sequence AEFRHDSGY as shown in SEQ ID NO: 1 or a fragment thereof and wherein the second region comprises the amino acid sequence VHHQKLVFFAEDVG as shown in SEQ ID NO: 2 or a fragment thereof.<br>
2.	The antibody molecule of claim 1, wherein said antibody molecule recognizes at least two consecutive amino acids within the two regions of |3-A4.<br>
3.	The antibody molecule of claim 1 or 2, wherein said antibody molecule recognizes in the first region an amino acid sequence comprising: AEFRHD, EF, EFR, FR, EFRHDSG, EFRHD or HDSG and in the second region an amino acid sequence comprising: HHQKL, LV, LVFFAE, VFFAED or VFFA, FFAEDV.<br>
4.	The antibody molecule of any one of claims 1 to 3, wherein said antibody molecule comprises a variable Vh-region as encoded by a nucleic acid molecule as shown in a SEQ ID NO selected from the group consisting of SEQ ID NO: 3, 5 or 7 or a variable Vn-region as shown in a SEQ ID NO: selected from the group consisting of SEQ ID NOs: 4, 6 and 8.<br>
5.	The antibody molecule of any one of claims 1 to 3, wherein said antibody molecule comprises a variable Vt-region as encoded by a nucleic acid molecule as shown in a SEQ ID NO selected from the group consisting of SEQ ID NO: 9, 11 and 13 or a variable Vt-region as shown in a SEQ ID NO selected from the group consisting of SEQ ID NOs: 10, 12 and 14.<br><br>
6.	The antibody molecule of any one of claims 1 to 5, wherein said antibody molecule comprises at least one CDR3 of an VL-region as encoded by a nucleic acid molecule as shown in SEQ ID NOs: 15, 17 or 19 or at least one CDR3 amino acid sequence of an Vi-region as shown in SEQ ID NOs: 16, 18 or 20 and/or wherein said antibody molecule comprises at least one CDR3 of an VH-region as encoded by a nucleic acid molecule as shown in SEQ ID NOs: 21, 23 or 25 or at least one CDR3 amino acid sequence of an VH-region as shown in SEQ ID NOs: 22, 24 or 26.<br>
7.	The antibody molecule of any one of claims 1 to 6, wherein said antibody is selected from the group consisting of MSR-3, -7 and -8 or an affinity-matured version of MSR-3, -7 or -8.<br>
8.	The antibody molecule of any one of claims 1 to 7, wherein said antibody molecule is a full antibody (immunoglobulin), a F(ab)-fragment, a F(ab)2-fragment, a single-chain antibody, a chimeric antibody, a CDR-grafted antibody, a bivalent antibody-construct, a synthetic antibody or a cross-cloned antibody.<br>
9.	The antibody molecule of any one of claims 1 to 8, wherein said at least two regions of β-A4 form a conformational epitope or a discontinuous epitope.<br>
10.	The antibody molecule of claim 8 or 9, wherein said cross-cloned antibody is selected from the group consisting of<br>
MS-R#3.3H1x3.4L9; MS-R #3.6H5 X 3.6L2; MS-R#3.6H8x3.6L2; MS-R#7.4H2x7.'2L1; MS-R#7.9H2x7.12L2; MS-R#7.9H4x7.12L2; MS-R #7.11 HI X 7.11 L'1; MS-R #7.11H1 X7.2L1; MS-R#3.4H1 X3.4L9;<br><br>
MS-R#3.4H3x3.4L7; MS-R #3.4H3 X 3.4L9; MS-R #3.4H7 X 3.4L9; MS-R #3.4H7 X 3.4L7; MS-R#3.6H5x3.6L1; MS-R#7.2H2x7.2L1; IVlS-R#7.4H2x7.12L2; MS-R#7.9H2x7.2L1; MS-R#7.9H2x7.12L1; MS-R#7.11H2x7.2L1; MS-R #7.11 H2X7.9L1; MS-R#7.11H2x7.12L1 or MS-R #8.1 HI X8.2L1.<br>
11.	A nucleic acid molecule encoding an antibody molecule of any one of claims 1 to 10.<br>
12.	A vector comprising the nucleic acid molecule of claim 11.<br>
13.	A host cell comprising the vector of claim 12.<br>
14.	A method for the preparation of an antibody molecule of any one of claims 1 to 10 comprising culturing the host cell of claim 13 under conditions that allow synthesis of said antibody molecule and recovering said antibody molecule from said culture.<br>
15.	A composition comprising an antibody molecule of any one of claims 1 to 10 or an antibody molecule produced by the method of claim 14. ,<br>
16.	The composition of claim 15, which is a pharmaceutical or a diagnostic composition.<br><br>
17.	Use of an antibody molecule of any one of claims 1 to 10 or an antibody molecule produced by the method of claim 14, of a nucleic acid molecule of claim 11, of a vector of claim 12 or a host of claim 13 for the preparation of a pharmaceutical composition for the prevention and/or treatment of a disease associated with amyloidogenesis and/or amyloid-plaque formation.<br>
18.	Use of an antibody molecule of any one of claims 1 to 10 or an antibody molecule produced by the method of claim 14 for the preparation of a diagnostic composition for the detection of a disease associated with amyloidogenesis and/or amyloid-plaque formation.<br>
19.	Use of an antibody molecule of any one of claims 1 to 10 or an antibody molecule produced by the method of claim 14 for the preparation of a pharmaceutical composition for the disintegration of p-amyloid plaques.<br>
20.	Use of an antibody molecule of any one of claims 1 to 10 or an antibody molecule produced by the method of claim 14 for the preparation of a pharmaceutical composition for passive immunization against p-amyloid plaque formation.<br>
21.	The use of claims 17 or 18, wherein said disease is dementia, Alzheimer's disease, motor neuropathy, Down's syndrome, Creutzfeld Jacob disease, hereditary cerebralhemorrhage with amyloidosis Dutch type, Parkinson^s disease, HlV-related dementia, ALS or neuronal disorders related to aging.<br>
22.	Kit comprising an antibody molecule of any one of claims 1 to 10, a nucleic acid molecule of claim 16, a vector of claim 17 or a host cell of claim 18.<br>
23.	A method for the optimization of an antibody molecule as defined in any one of claims 1 to 10 comprising the steps of<br>
(a) constructing a library of diversified Fab antibody fragments derived from an antibody comprising at least one CDR3 of an Vn-region as encoded by a nucleic acid molecule as shown in SEQ ID NOs: 21,23 or<br><br>
25 or at least one CDR3 amino acid sequence of an VH-region as shown in SEQ ID NOs: 22, 24 or 26;<br>
(b)	testing  the  resulting  Fab  optimization  library by  panning  against<br>
Aβ/Aβ4;<br>
(c)	identifying optimized clones; and<br>
(d)	expressing of selected, optimized clones.<br>
2I4.     The method of claim 23 further comprising a step (ca), whereby the optimized clones are further optimized by cassette mutagenesis<br>
25.     The method of claim 23 or 24, wherein said Ap/Ap4 in step (b) is aggregated A|3/Ap4.<br>
26,     The method of any one of claims 23 to 25, wherein said identification in step (c) is carried out by koff-ranking.<br>
27.	A method for the preparation of a pharmaceutical composition comprising the<br>
steps of<br>
(a)	optimization of an antibody according to the method of any one of claims 23 to 26; and<br>
(b)	formulating   the   optimized   antibody/antibody   molecule   with   an physiologically acceptable carrier.<br>
28.	A pharmaceutical composition prepared by tlie metJiod of claim 27.<br><br>
29.	An antibody molecule substantially as herein described with reference to the accompanying, drawings.<br>
30.	A pharmaceutical composition substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWNsYWltcyBmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWRlc2NyaXB0aW9uKGNvbXBsZXRlKWZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWRlc2NyaXB0aW9uKGNvbXBsZXRlKWdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg0MS1jaGVucC0yMDA0LXByaW9yaXR5IGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1841-chenp-2004-priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="210083-disk-cartridge.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="210085-a-process-for-the-preparation-of-a-composition-useful-for-treatment-in-chronic-respiratory-disorders.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>210084</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1841/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Sep-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Aug-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. F. HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>124, Grenzacherstrasse, CH-4070 Basel,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BARDROFF, Michael (ET. AL.)</td>
											<td>Schietweg 2a, D-81375 Munchen,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/001759</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-02-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02003844.4</td>
									<td>2002-02-20</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/210084-an-antibody-molecule-capable-of-recognizing-two-regions-of-a4-peptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 05:09:09 GMT -->
</html>
