<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/210475-an-expression-vector-useful-in-producing-safracinor-its-analogue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:53:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 210475:AN EXPRESSION VECTOR USEFUL IN PRODUCING SAFRACINOR ITS ANALOGUE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN EXPRESSION VECTOR USEFUL IN PRODUCING SAFRACINOR ITS ANALOGUE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>1. An expression vector comprising isolated nucleic acid sequence represented by SEQ ID NO 1 or variants or portions thereof, the said sequence encoding for a polypeptide useful in bio-synthesis of safracin or its analogue.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
(See section 10, rule 13)<br>
"AN EXPRESSION VECTOR USEFUL IN PRODUCING SAFRACIN OR ITS<br>
ANALOGUE"<br>
PHARMA MAR, S.A., of Calle de la Calera 3, Poligono Industrial de  Ires Cantos, Tres Cantos, E-28760 Madrid, Spain,<br>
The following specification particularly describes the invention and the manner in which it is to be performed.<br>
GRANTED<br><br>
19-6-2007<br><br><br>
FIELD OF THE INVENTION<br>
The present invention relates to an expression vector comprising nucleic acid sequence represented by SEQ ID NO 1 or variants or portions thereof, the said sequence encoding for a polypeptide useful in synthesis of safracin or its analogue. Further, the present invention relates to a gene cluster having open reading frames which encode polypeptides directing the synthesis of a safracin molecule. More particularly, the present invention relates to the gene cluster responsible for the biosynthesis of safracin, its uses for genetic enGlneering and new safracins obtained by manipulation of biosynthesis mechanism.<br>
BACKGROUND OF THE INVENTION<br>
Safracins, a family of new compounds with a potent broad-spectrum antibacterial activity, were discovered in a culture broth of Pseudomonas sp. Safracin occurs in two Pseudomonas sp. strains, Pseudomonas fluorescens A2-2 isolated from a soil sample collected in Tagawagun, Fukuoka, Japan (Ikeda et al. /. Antibiotics 1983,36,1279-1283; WO 82 00146 and JP 58113192) and Pseudomonas fluorescens SC 12695 isolated from water samples taken from the Raritan-Delaware Canal, near New Jersey (Meyers et al. /. Antibiot. 1983,36 (2), 190-193). Safracins A and B, produced by Pseudomonas fluorescens A2-2, have been examined against different tumor cell lines and has been found to possess antitumor activity in addition to antibacterial activity.<br><br><br>
SafracinsA R=H	ET-743<br>
B R=OH<br>
Due to the structural similarities between safracin B and ET-743 safracin offers the possibility of hemi-synthesis of the highly promising potent new antitumor agent ET-743, isolated from the marine tunicate Ecteinascidia turbinata and which is currently in Phase II clinical trials in Europe and the United States. A hemisynthesis of ET-743 has been achieved starting from safracin B (Cuevas et al. Organic Lett. 2000, 10, 2545-2548; WO 00 69862 and WO 01 87895).<br>
As an alternative of making safracins or its structural analogs by chemical synthesis, manipulating genes of governing secondary metabolism offer a promising alternative and allows for preparation of these compounds bio synthetically. Additionally, safracin structure offers exciting possibilities for combinatorial biosynthesis.<br>
In view of the complex structure of the safracins and the limitations in their obtention from Pseudomonas fluorescens A2-2, it would be highly desirable to understand the genetic basis of their synthesis in order to create the means to influence them in a targeted manner. This could increase   the   amounts   of safracins   being  produced,   because   natural<br><br><br>
production strains generally yield only low concentrations of the secondary metabolites that are of interest. It could also allow the production of safracins in hosts that otherwise do not produce these compounds. Additionally, the genetic manipulation could be used for combinatorial creation of novel safracin analogs that could exhibit improved properties and that could be used in the hemi-synthesis of new ecteinascidins compounds.<br>
However, the success of a biosynthetic approach depends critically on the availability of novel genetic systems and on genes encoding novel enzyme activities. Elucidation of the safracin gene cluster contributes to the general field of combinatorial biosynthesis by expanding the repertoire of genes uniquely associated with safracin biosynthesis, leading to the possibility of making novel precursors and safracins via combinatorial biosynthesis.<br>
SUMMARY OF THE INVENTION<br>
We have now been able to identify and clone the genes of safracin ) Jbiosynthesis providing the genetic basis for improving and manipulating : in a targeted manner the productivity of Pseudomonas sp., and using genetic methods, for synthesising safracin analogues. Additionally, these genes encode enzymes that are involved in biosynthetic processes to produce structures, such as safracin precursors, that can form the basis of combinatorial chemistry to produce a wide variety of compounds. These compounds can be screened for a variety of bioactivities including anticancer activity.<br><br><br>
Therefore in a first aspect the present invention provides a nucleic acid, suitably an isolated nucleic acid, which includes a DNA sequence (including mutations or variants thereof), that encodes non-ribosomal peptide synthetases which are responsible for the biosynthesis of safracins. This invention provides a gene cluster, suitably an isolated i gene cluster, with open reading frames encoding polypeptides to direct the ' assembly of a safracin molecule.<br>
One aspect of the present invention is a composition including at least one nucleic acid sequence, suitably an isolated nucleic acid molecule, that encodes at least one polypeptide that catalyses at least one step of the biosynthesis of safracins. Two or more such nucleic acid sequences can be present in the composition. DNA or corresponding RNA is also provided.<br>
In particular the present invention is directed to a nucleic acid sequence, suitably an isolated nucleic acid sequence, from a safracin gene cluster comprising said nucleic acid sequence, a portion or portions of said nucleic acid sequence wherein said portion or portions encode a polypeptide or polypeptides or a biologically active fragment of a polypeptide or polypeptides, a single-stranded nucleic acid sequence derived from said nucleic acid sequence, or a single stranded nucleic acid sequence derived from a portion or portions of said nucleic acid sequence, or a double-stranded nucleic acid sequence derived from the single-stranded nucleic acid sequence (such as cDNA from mRNA). The nucleic acid sequence can be DNA or RNA.<br>
More particularly, the present invention is directed to a nucleic acid sequence, suitably an isolated nucleic acid sequence, which includes or comprises at least SEQ ID 1, variants or portions thereof, or at least one of<br><br><br>
the sacA, sacB, sacC, sacC, sacD, sacE, sacF, sacG, sacH, sacH, sad, sacJ, orfl, orf2, orf3 or orf4 genes, including variants or portions. Portions can be at least 10, 15, 20, 25, 50, 100, 1000, 2500, 5000, 10000, 20000, 25000 or more nucleotides in length. Typically the portions are in the range 100 to 5000, or 100 to 2500 nucleotides in length, and are biologically functional.<br>
Mutants or variants include polynucleotide molecules in which at least one nucleotide residue is altered, substituted, deleted or inserted. Multiple changes are possible, with a different nucleotide at 1, 2, 3, 4, 5, 10, 15, 25, 50, 100, 200, 500 or more positions. Degenerate variants are envisaged which encode the same polypeptide, as well as non-degenerate variants which encode a different polypeptide. The portion, mutant or variant nucleic acid sequence suitably encodes a polypeptide which retains a biological activity of the respective polypeptide encoded by any of the open reading frames of the safracin gene cluster. Allelic forms and polymorphisms are embraced.<br>
The invention is also directed to an isolated nucleic acid sequence capable of hybridizing under stringent conditions with a nucleic acid sequence of this invention. Particularly preferred is hybridisation with a translatable length of a nucleic acid sequence of this invention.<br>
The invention is also directed to a nucleic acid encoding a polypeptide which is at least 30%, preferably 50%, preferably 60%, more preferably 70%, in particular 80%, 90%, 95% or more identical in amino acid sequence to a polypeptide encoded by any of the safracin gene cluster open reading frames sacA to sacJ and orfl to orf4 (SEQ ID 1 and genes encoded in SEQ ID 1) or encoded by a variant or portion thereof. The polypeptide suitably retains a biological activity of the respective polypeptide encoded by any of the safracin gene cluster open reading<br><br><br><br>
frames.<br>
In particular, the invention is directed to an isolated nucleic acid sequence encoding for any of SacA, SacB, SacC, SacD, SacE, SacF, SacG, SacH, Sad, SacJ, Orfl, 0rf2, Orf3 or 0rf4 proteins (SEQ ID 2-15), and variants, mutants or portions thereof.<br>
In one aspect, an isolated nucleic acid sequence of this invention encodes a peptide synthetase, a L-Tyr derivative hidroxylase, a L-Tyr derivative methylase, a L-Tyr O-methylase, a methyl-transferase or a monooxygenase or a safracin resistance protein.<br>
The invention also provides a hybridization probe which is a nucleic acid sequence as defined above or a portion thereof. Probes suitably comprise a sequence of at least 5, 10, 15, 20, 25, 30, 40, 50, 60, or more nucleotide residues. Sequences with a length on the range 25 to 60 are preferred. The invention is also directed to the use of a probe as defined for the detection of a safracin or ecteinascidin gene. In particular, the probe is used for the detection of genes in Ecteinascidia turbinata.<br>
In a related aspect the invention is directed to a polypeptide encoded by a nucleic acid sequence as defined above. Full sequence, variant, mutant or fragment polypeptides are envisaged.<br>
In a further aspect the invention is directed to a vector, preferably an expression vector, preferably a cosmid, comprising a nucleic acid sequence encoding a protein or biologically active fragment of a protein, wherein said nucleic acid is as defined above.<br>
In another aspect the invention is directed to a host cell transformed with one or more of the nucleic acid sequences as defined above, or a<br><br>
vector, an expression vector or cosmid as defined above. A preferred host cell is transformed with an exogenous nucleic acid comprising a gene cluster encoding polypeptides sufficient to direct the assembly of a safracin or safracin analog. Preferably the host cell is a microorganism, more preferably a bacteria.<br>
The invention is also directed to a recombinant bacterial host cell in which at least a portion of a nucleic acid sequence as defined above is disrupted to result in a recombinant host cell that produces altered levels of safracin compound or safracin analogue, relative to a corresponding nonrecombinant bacterial host cell.<br>
The invention is also directed to a method of producing a safracin compound or safracin analogue comprising fermenting, under conditions and in a medium suitable for producing such a compound or analogue, an organism such as Pseudomonas sp, in which the copy number of the safracin genes/cluster encoding polypeptides sufficient to direct the assembly of a safracin or safracin analog has been increased.<br>
The invention is also directed to a method of producing a safracin compound or analogue comprising fermenting, under conditions and in a medium suitable for producing such compound or analogue, an organism such as Pseudomonas sp in which expression of the genes encoding polypeptides sufficient to direct the assembly of a safracin or safracin analogue has been modulated by manipulation or replacement of one or more genes or sequence responsible for regulating such expression. Preferably expression of the genes is enhanced.<br>
The invention is also directed to the use of a composition including at least one isolated nucleic acid sequence as defined above or a modification thereof for the combinatorial biosynthesis of non-ribosomal<br><br>
peptides, diketopiperazine rings and safracins.<br>
In particular the method involves contacting a compound that is a substrate for a polypeptide encoded by one or more of the safracin biosynthesis gene cluster open reading frames as defined above with the polypeptide encoded by one or more safracin biosynthesis gene cluster open reading frames, whereby the polypeptide chemically modifies the compound.<br>
In still another embodiment, this invention provides a method of producing a safracin or safracin analog. The method involves providing a microorganism transformed with an exogenous nucleic acid comprising a safracin gene cluster encoding polypeptides sufficient to direct the assembly of said safracin or safracin analog; culturing the bacteria junder conditions permitting the biosynthesis of safracin or safracin analog; and isolating said safracin or safracin analog from said cell.<br>
The invention is also directed to any of the precursor compounds P2, P14, analogs and derivatives thereof and their use in the combinatorial biosynthesis non-ribosomal peptides, diketopiperazine rings and safracins.<br>
Additionally, the invention is also directed to the new safracins obtained by knock out safracin P19B, safracin P22A, safracin P22B, safracin D and safracin E, and their use as antimicrobial or antitumor agents, as well as their use in the synthesis of ecteinascidin compounds.<br>
The invention is also directed to new safracins obtained by directed biosynthesis as defined_above, and their use as antimicrobial or antitumor agents, as well as their use in the synthesis of ecteinascidin compounds. In particular the invention is directed to safracin B-ethoxy and safracin A-ethoxy and their use.<br><br><br>
In one aspect, the present invention enables the preparation of structures related to safracins and ecteinascidins which cannot or are difficult to prepare by chemical synthesis. Another aspect is to use the knowledge to gain access to the biosynthesis of ecteinascidins in Ecteinasddia turbinata, for example using these sequences or parts as probes in this organism or a putative symbiont.<br>
More fundamentally, the invention opens a broad field and gives access to ecteinascidins by genetic enGlneering.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Fig. 1: Structural organization of the chromosomal DNA region cloned in pL30p cosmid. The region of P. fluorescens A2-2 DNA, containing the safracin gene cluster, is shown. Both, sacABCDEFGH and sacIJ, gene operons and the modular organization of the peptide synthetases deduced from sacA, sacB and sacC are illustrated. The following domains are indicated: C: condensation; T: thiolation; A: adenylation and Re: reductase. Location of other genes present in pL30p cosmid (orf 1 to orf4) as well as their proposed function is shown.<br>
Fig. 2: Conserved core motifs between NRPSs. Conserved amino acid sequences in SacA, SacB and SacC proteins and their comparison with its homologous sequences from Myxococcus xanthus DM50415.<br>
Figure 3. NRPS biosynthesis mechanism proposed for the formation of the Ala-Gly dipeptide. Step a*, adenylation of Ala; b*, transfer to the 4'-phosphopantetheinyl arm; c*, transfer to the waiting/elongation site; d*, adenylation of the Gly; e*, transfer to the 4'-phosphopantetheinyl arm; f*,<br><br>
condensation of the elongation chain on the 4'-phosphopantetheinyl arm with the starter chain at the waiting/elongation site; g*, Ala-Gly dipeptide attached to the phosphopantetheinyl arm of SacA and h*, transfer of the elongated chain to the following waiting/elongation site.<br>
Fig. 4: Cross-feeding experiments. A. Scheme of A2-2 DNA fragments cloned in pBBRl-MCS2 vector and products obtained in the heterologous host. B. HPLC profile of safracin production in wild type strain versus sacF mutant. The addition of P2 precursor to the sacF mutant, provided both in trans and synthetically, yield safracin B production. SfcA, safracin A and SfcB, safracin B.<br>
Fig. 5: Scheme of the safracin biosynthesis mechanism and biosynthetic intermediates. Single enzymatic steps are indicated by a continuous arrow and multiple reactions steps are indicated by discontinuous arrows.<br>
Fig. 6: Safracin gene disruptions and compounds produced. A. Gene disruption and precursor molecules synthesized by the mutants constructed. Gene marked with an asterisk does not belong to the safracin cluster. Inactivation of genes orf 1, orf 2, orf 3 and orf 4 has demonstrated to have no effect over safracin production. B. HPLC profile of safracin production in wild type strain and in sacA, sad and sacJ mutants. Structure of the different molecules obtained is shown.<br>
Fig. 7: Structure of the different molecules obtained by gene disruption. Inactivation of SacJ protein (a) yields P22B, P22A and P19 molecules, whereas gene disruption of sacl (b), produces only P19 compound. The sad disruption, together with the sacJ reconstructed expression, produces two new safracins: safracin D (possible precursor for ET-729 hemi-synthesis) and safracin E (c).<br><br><br>
Fig. 8: Addition of specific designed 'unnatural' precursors (P3). Chemical structure of the two molecules obtained by addition of P3 compound to the sacF mutant.<br>
Fig. 9: Scheme of the gene disruption event through simple recombination, using an homologous DNA fragment cloned into pK18:MOB (an integrative plasmid in Pseudomonas).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Non ribosomal peptide synthetases (NRPS) are en2ymes responsible for the biosynthesis of a family of compounds that include a large number of structurally and functionally diverse natural products. For example, peptides with biological activities provide the structural backbone for compounds that exhibit a variety of biological activities such as, antibiotics, antiviral, antitumor, and immunosuppressive agents (Zuber et al. Biotechiology of Antibiotics 1997 (W. Strohl, ed.), 187-216 Marcel dekker, Inc., N.Y; Marahiel et al. Chem. Rev. 1997, 97, 2651-2673).<br>
Although structurally diverse, most of these biologically active peptides share a common mechanistic scheme of biosynthesis. According to this model, peptide bond formation takes place on multienzymes designated peptides synthetases, on which amino acid substrates are activated by ATP hydrolysis to the corresponding adenylate. This unstable intermediate is subsequently transferred to another site of the multienzymes where it is bound as a thioester to the cysteamine group of an enzyme-bound 4'-phosphopantetheninyl (4'-PP) cofactor. At this stage, the thiol-activated substrates can undergo modifications such as epimerisation or N-methylation. Thioesterified substrate amino acids are then integrated into the peptide product through a step-by-step elongation by a series of<br><br><br><br>
transpeptidation reactions. With this template arrangement in peptide synthetases, the modules seem to operate independently of one another, but they act in concert to catalyse the formation of successive peptide bonds (Stachelhaus et al. Science 1995, 269, 69-72; Stachelhaus et al. Chem. Biol. 1996, 3, 913-921). The general scheme for non-ribosomal peptide biosynthesis has been widely reviewed (Marahiel et al. Chem. Rev. 1997, 97, 2651-2673; Konz and Marahiel, Chem, and Biol 1999, 6, R39-R48; Moffit and Neilan, FEMS Microbiol. Letters 2000, 191, 159-167).<br>
A large number of bacterial operons and fungal genes encoding peptide synthetases have recently been cloned, sequenced and partially characterized, providing valuables insights into their molecule architecture (Marahiel, Chem and Biol 1997, 4, 561-567). Different cloning strategies were used, including probing of expression libraries by antibodies raised against peptide synthetases, complementation of deficient mutants, and the use of designed oligonucleotides derived from amino acid sequences of peptide synthetase fragments.<br>
Analysis of the primary structure of these genes revealed the presence of distinct homologous domains of about 600 amino acids. This specific functional domains consist of at least six highly conserved core sequences of about three to eight amino acids in length, whose order and location within all known domains are very similar (Kusard and Marahiel, Peptide Research 1994, 7, 238-241). The used of degenerated oligonucleotides derived from the conserved cores opens the possibility of identifying and cloning peptide synthetases from genomic DNA, by using the polymerase chain reaction (PCR) technology (Kusard and Marahiel, Peptide Research 1994, 7, 238-241; Borchert et al. FEMS Microbiol Letters 1992,92,175-180).<br>
The structure of safracin suggests that this compound is synthesized<br><br><br>
by a NRPS mechanism. The cloning and expression of the non-ribosomal peptide synthetases and the associated tailoring enzymes from Pseudomonas fluorescens A2-2 safracin cluster would allow production of unlimited amounts of safracin. In addition, the cloned genes could be used for combinatorial creation of novel safracin analogs that could exhibit improved properties and that could be used in the hemi-synthesis of new ecteinascidins. Moreover, cloning and expressing the safracin gene cluster in heterologous systems or the combination of safracin gene cluster with other NRPS genes could result in the creation of novel drugs with improved activities.<br>
The present invention provides, in particular, the DNA sequence encoding NRPS responsible for biosynthesis of safracin, i.e., safracin synthetases. We have characterized a 26,705 bp region (SEQ ID NO:l) from Pseudomonas fluorescens A2-2 genome, cloned in pL30P cosmid and demonstrated, by knockout experiments and heterologous expression, that this region is responsible for the safracin biosynthesis. We expressed the pL30P cosmid in two strains of Pseudomonas sp., which do not produce safracin, and the result was a production of safracin A and B at levels of a 22%, for P. fluorescens (CECT 378), and 2%, for P. aeruGlnosa (CECT 110), in comparison with P. fluorescens A2-2 production. The predicted amino acids sequences of the various peptides encoded by this DNA sequence is shown in SEQ ID NO:2 through SEQ ID NO: 15 respectively.<br>
The gene cluster for safracin biosynthesis derived from P. fluorescens A2-2, is characterized by the presence of several open reading frames (ORF) that are organized in two divergent operons (Fig. 1), an eight genes operdh {sacABCDEFGH) and a two genes operon {sacIJ), preceded by well-conserved putative promoters regions that overlap. The safracin biosynthesis gene cluster is present in only one copy in P. fluorescens A2-2 genome.<br><br><br>
propose that the Ala would be first charged on the phosphopantetheinyl arm of SacA (Fig. 3 a* and b*) before being transferred to a waiting position, a condensation domain, located in N-terminal of sacA (Fig. 3, c*). The Gly adenylate would then be charged on the same phosphopantetheinyl arm (Fig. 3, d* and e*), positioned to the elongation site, and elongation would occur (Fig. 3, f*). The arm of the first module would at this stage be charged with a Ala-Gly dipeptide (Fig. 3, g*). We proposed that the dipeptide would then be transferred on a waiting position in the second phosphopantetheinyl arm (Fig. 3, h*), located in SacB, to continue the synthesis of the safracin tetrapeptide basic skeleton. An alternative biosynthesis mechanism could be the direct incorporation of a dipeptide Ala-Gly into SacA. In this case, the dipeptide could be oriGlnated from the activity of highly active peptidyl transferase ribozyme family (Sun et al, Chem. and Biol 2002, 9, 619-626) or from the activity of bacterial proteolysis.<br>
And fourthly, although in most of the prokaryotic peptide synthetases the thioesterase moiety, which appears to be responsible for the release of the mature peptide chain from the enzyme, is fused to the C-terminal end of the last amino acid binding module (Marahiel et al. Chem. Rev. 1997, 97, 2651-2673), in the case of safracin synthetases, the TE domain is missing. Probably, in the safracin synthesis after the last elongation step, the tetrapeptide could be released by an alternative strategy for peptide-chain termination that also occurs in the saframycin synthesis (Pospiech et al. Microbiol 1996, 142, 741-746). This particular termination strategy is catalysed by a reductase domain at the carboxy-terminal end of the SacC peptide synthetase which catalyses the reductive cleavage of the associated T-domain-tethered acyl group, releasing a linear aldehyde.<br>
Our cross feeding experiments indicate that the last two amino acids incorporated into the safracin molecule are two L-Tyr derivatives called P2<br><br><br>
(3-hydroxy-5-methyl-0-methyltyrosine) (Fig. 4, S), instead of two L-Tyr as it is proposed to occur in saframycin synthesis. First, the products of two genes (sacF and sacG), similar to bacterial methyltransferases, have shown to be involved in the O, C-methylation of L-Tyr to produce P14 (3-methyl-Omethyltyrosine), precursor of P2. A possible mechanism could envisage that the O-methylation occurs first and then the C-methylation of the amino acid derivative is produced. Secondly, P2, the substrate for the peptide synthetases SacB and SacC, is formed by the hydroxylation of P14 by SacD (Fig. 4, 5).<br><br>
P-14	P-2<br>
Apart from the safracin biosynthetic genes, in the sacABCDEFGH operon there are also found two genes, sacE and sacH, involved in an unknown function and in the safracin resistance mechanism, respectively. We have demonstrated that sacH gene codes for a protein that when is heterologous expressed, in different Pseudomonas strains, a highly increase of the safracin B resistance is produced. SacH is a putative transmembrane protein, that transforms the C21-OH group of safracin B into a C21-H group, to produce safracin A, a compound with less antibiotic and antitumoral activity. Finally, even though still is unknown about the putative function of SacE, homologous of this gene have been found close<br><br><br>
to various secondary metabolites biosynthetic gene clusters in some microorganisms genomes, suggesting a conserved function of this genes in secondary metabolite formation or regulation.<br>
In the sacIJ operon, the deduced amino acid sequences encoded by sacl and sacJ strongly resemble gene products of methyltransferase and hydroxylase/monoxygenase, respectively. Our data reveals that Sacl is the enzyme responsible for the iV-methylation present in the safracin structure, and that SacJ is the protein which makes an additional hydroxylation on one of the L-Tyr derivative incorporated into the tetrapeptide to produce the quinone structure present in all safracin molecules. N-N-Methylation is one of the modifications of nonribosomally synthesized peptides that significantly contributes to their biological activity. Except for saframycin (Pospiech et al. Microbiol 1996, 142, 741-746), that is produced by bacteria and is N-methylated, all the N-methylated nonribosomal peptides .known are produced by fungi or actinomycetes and, in most of the cases, the responsible for the N-methylation is a domain which reside in the nonribosomal peptide synthetase.<br>
Table I. Summary of safracin biosynthetic and resistance genes identified in cosmid pL30P.<br><br>
ORF	Protein      Proposed function	Position	Amino acids      Molecular<br>
nam	name	start-stop bp	weight<br>
Peptide synthetase	3052-6063	1004	110.4<br>
Peptide synthetase	6068-9268	1063	117.5<br>
Peptide synthetase	9275-13570	1432	157.3<br>
L-Tyr derivative	13602-14651	350	39.2<br>
hidroxylase<br>
e	<br>
sacA      SacA<br>
sacB SacB sacC SacC sacD     SacD<br><br><br>
sacE	SacE	Unknown	14719-14901	61	6.7<br>
sacF	SacF	L-Tyr derivative methylase	14962-16026	355	39.8<br>
sacG	SacG	L-Tyr O-methylase '	16115-17155	347	38.3<br>
sacH	SacH	Resistance protein	17244-17783	180	19.6<br>
sacl	Sad	methyl-transferase	2513-1854	220	24.2<br>
sacJ	SacJ	monooxygenase	1861-355	509	55.3<br>
The safracin putative synthetic pathway, with indications of the specific amino acid substrates used for each condensation reaction and the various post-condensation activities, is shown in Fig. S.<br>
To further evaluate the role of safracin biosynthetic genes, we constructed knock out mutants of each of the genes of the safracin cluster (Fig. 6). The disruption of the NRPSs genes {sacA, sacB and sacQ as well as sacD, sacF and sacG, resulted in safracin and P2 non producing mutants. Our results indicate that the genes from sacA to sacH are part of the same genetic operon. As a consequence of the sacl and sacJ gene disruptions three new molecules were oriGlnated, P19B, P22A and P22B (Fig. 6).<br><br><br>
The production of P22A and P22B (Fig. 7a*) by sacJ mutant demonstrated that the role of the SacJ protein is to produce the additional hydroxylation of the left L-Tyr derivatives amino acid of the safracin, the one involved in the quinone ring. The production of P19B (Pig. 7b*) by sad mutant, a safracin like molecule where the N-methylation and the quinone ring are missing, confirms that Sacl is the N-methyltransferase enzyme and suggests that sadJ is a transcriptional operon. The production of P19B also by sacJ mutant (Fig. 7a*) suggests that probably the N-methylation occurs after the quinone ring has been formed. Even though these new structures have no interesting antimicrobial activity on B. subtilis or no high citotoxic activity on cancer cells, they can serve as interesting new precursors for the hemisynthesis of new active molecules. As far as structure activity is concerned, the observation that P19B, P22A and P22B appear to loose their activity, suggests that the lost of the quinone ring from the safracin structure is directly related with the lost of activity of the safracin family molecules.<br>
The safracin D and safracin E are safracin B and safracin A like<br>
The disruption of sad gene with the reconstitution of the sacJ gene expression resulted in the production of two new safracins. The two antibiotics produced, at levels of production as high as the levels of safracin A/safracin B production in the wild type strain, have been named as safracin D and safracin E (Fig. 7c*).<br><br><br>
molecules, respectively, where the N-methylation is missing. Both, safracin D and safracin E have been shown to possess the same antibacterial and antitumoral activities as safracin B and safracin A, respectively. Apart from its high activities properties, antibacterial and antitumoral, safracin D could be used in the hemi-synthesis of the ecteinascidin ET-729, a potent antitumoral agent, as well as in the hemi-synthesis of new ecteinascidins.<br>
A question arises concerning the role of the aminopeptidase-like protein coded by a gene located at 3'site of the safracin operon. The insertional inactivation of orfl (PM-S1-14) showed no effect on safracin A/safracin B production. Because of its functionality properties it remains unclear if this protein could play some role in the safracin metabolism. The other genes present in the pL30P cosmid (or/2 to orf4) will have to be studied in more detail.<br>
Another aspect of the invention is that provides the tools necessary for the production of new specific designed "unnatural" molecules. The addition of a specific modified P2 derivative precursor named P3, a 3-hydroxy-5-methyl-O-methylryrosine, to the sacE mutant yields two "unnatural" safracins that incorporated this specific modified precursor, safracin A(OEt) and safracin B(OEt) (Fig. 8).<br><br><br><br><br>
The two new safracins are potent antibiotic and antitumoral compounds. The biological activities of safracin A(OEt) and Safracin B(OEt) are as potent as the activities of safracin A and safracin B, respectively. These new safracins could be the source for new potent antitumoral agents, as well as a source of molecules for the hemi-synthesis of new ecteinascidins.<br>
In addition, the genes involved in safracin synthesis could be combined with other non ribosomal peptide synthetases genes to result in the creation of novel "unnatural" drugs and analogs with improved activities.<br>
EXAMPLES<br>
Example 1: Extraction of nucleic acid molecules from Pseudomonas fluorescens A2-2<br>
Bacterial strains<br>
Strains of Pseudomonas sp. were grown at 27°C in Luria-Bertani (LB) broth (Ausubel et al. 1995, J. Wiley and Sons, New York, N.Y). E. coli strains were grown at 37°C in LB medium. Antibiotics were used at the following concentrations: ampicillin (50 ng/ml), tetracycline (20 p-g/ml) and kanamycin (50 ug/ml).<br>
Table II. Strains used in this invention.<br><br><br>
Code	Genotype	<br>
PM-S1 -001	P. fluorescens A2-2 wild type<br>
PM-S1-002	sacA-<br>
PM-S1-003	sacB<br>
PM-S 1-004	sacC<br>
PM-S 1-005	sacJ-<br>
PM-S1-006	sacf<br>
PM-S1-007	sad- with sacj expression reconstitution<br>
PM-S 1-008	sacF-<br>
PM-S1-009	sacG-<br>
PM-S1- 010	sacD-<br>
PM-S1- 014	orfl-<br>
PM-S1-015	A2-2 + pLAFR3<br>
PM-S1- 016	A2-2 + pL30p<br>
PM-19- 001	P. fluorescens CECT378 + pLAFR3<br>
PM-19- 002	P. fluorescens CECT378 + pL30p<br>
PM-19- 003	P. fluorescens CECT378 + pBBRl-MCS2<br>
PM-19- 004	P. fluorescens CECT378 + pB5H83<br>
PM-19- 005	P. fluorescens CECT378 + pB7983<br>
PM-19- 006	P. fluorescens CECT378 + pBHPT3<br>
PM-16- 001	P. aeruGlnosa CECT110 + pLAFR3<br>
PM-16- 002	P. aeruGlnosa CECT110 + pL30p<br>
PM-17- 003	P. putida ATCC12633+ pBBRl-MCS2<br>
PM-17- 004	P. putida ATCC 12633+ pB5H83<br>
PM-17- 005	P. putida ATCC12633+ pB7983<br>
PM-18- 003	P. stutzeri ATCC 17588+ pBBRl-MCS2<br>
PM-18- 004	P. stutzeri ATCC 17588+ pB5H83<br>
PM-18- 005	P. stutzeri ATCC 17588+ pB7983<br>
DNA manipulation<br>
Unless otherwise noted, standard molecular biology techniques for in vitro DNA manipulations and cloning were used (Sambrook et al. 1989, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory).<br><br><br>
DNA extraction<br>
Total DNA from Pseudomonas fluorescens A2-2 cultures was prepared as reported (Sambrook et al. 1989, Cold Spring Harbor, NY: Cold Spring Harbor Laboratory).<br>
Computer analysis<br>
Sequence data were compiled and analysed using DNA-Star software package.<br>
Example  2:  Identification of NRPS  genes  responsible  for  safracin production in Pseudomonas fluorescens A2-2.<br>
Primer design<br>
Marahiel et al (Marahiel et al Chem. Rev. 1997, 97, 2651-2673) previously reported highly conserved core motifs of the catalytic domains of cyclic and branched peptide synthetases. Based on multiple sequence alignments of several reported peptide synthetases the conserved regions A2, A3, A5, A6, A7 and A8 of adenylation and T of thiolation modules were targeted for the degenerate primer design (Turgay and Marahiel, Peptide Res. 1994, 7, 238-241). The wobble positions were designed in respect to codon preferences within the selected modules and the expected high G/C content of Pseudomonas sp. All oligonucleotides were obtained from ISOGEN (Bioscience BV). A PCR fragment was obtained when degenerate oligonucleotides derived from the YGPTE (A5 core) and LGGXS (T core) sequences were used. These oligonucleotides were denoted PS34-YG and PS6-FF, respectively.<br><br>
Table III. PCJR primers designed for this study.<br><br>
Primer designation<br>
and<br>
orientation	Sequence	Length<br>
PS34-YG (forward)	5'- TAYGGNCCNACNGA -3'	14-mer<br>
PS6-FF (reverse)	5'-TSNCCNCCNADNTCRAARAA-3'	20-mer<br>
PCR conditions for amplification of DNA from P. fluorescens A2-2<br>
A fragment internal to nonribosomal peptide synthetases (NRPS) was amplified using PS-34-YG and PS6-FF oligonucleotides and P. fluorescens A2-2 chromosomal DNA as template. Reaction buffer and Taq polymerase from Promega were used. The cycling profile performed in a Personal thermocycler (Eppendorf) consists on: 30 cycles of 1 min at 95°C, 1 min at 50°C, 2 min at 72°C. PCR products were on the expected size (750 bp aprox.) based on the location of the primers within the NRPS domains of other synthetase genes.<br>
DNA cloning<br>
PCR amplification fragments were cloned into pGEM-Teasy vector according to the manufacturer (Qiagen, Inc., Valencia, CA). In this way, cloned fragments are flanked by two EcoRI restriction sites, in order to facilitate subsequent subclonig in other plasmids (see below). Since NRPSs enzymes are modular, clones from the degenerated PCR primers represents a pool of fragments from different domains.<br>
DNA sequencing<br>
All sequencing was performed using primers directed against the cloning vector, with an ABI Automated sequencer (Perkin-Elmer). Cloned DNA sequences were identified using the BLAST server of the National<br><br><br>
Center for Biotechnology Information accessed over the Internet (Altschul et al., Nucleic Acids Res. 1997, 25, 3389-3521). All of the sequences have signature regions for NRPSs and show high similarity in BLAST searches to bacterial NRPS showing that they are in fact of peptide oriGln. Moreover, a probable domain similarity search was performed using the PROSITE (European Molecular Biology Laboratory, Heidelberg, Germany) web server.<br>
Gene disruption ofPseudomonas fluorescens A2-2<br>
In order to analyse the function of the genes cloned, these genes were disrupted through homologous recombination (Fig. 9). For this purpose, recombinant plasmids (pG-PS derivatives) harbouring the NRPS gene fragment were digested with EcoKL restriction enzyme. The resulting fragments belonGlng to the gene to be mutated were cloned into the pK18mob mobilizable plasmid (Schafer et al Gene 1994, 145, 69-73), a chromosomal integrative plasmid able to replicate in B. coli but not in Pseudomonas strains. Recombinant plasmids were introduced first in E. coli S17-XPIR strain by transformation and then in P. fluorescens A2-2 through biparental conjugation (Herrero et al, J Bacteriol 1990, 172, 6557-6567). Different dilutions of the conjugation were plated onto LB solid medium containing ampicillim plus kanamycin and incubated overnight at 27°C. Kanamycin-resistant transconjugants, containing plasmids integrated into the genome via homologous recombination, were selected.<br>
Biological assay (biotest)for safracin production<br>
Strains P. fluorescens A2-2 and its derivatives were incubated in 50 ml baffled erlenmeyer flasks containing fermentation medium with the corresponding antibiotics. Initially, SA3 fermentation medium was used (Ikeda Y. J. Ferment Technol 1985, 63, 283-286). In order to increase the productivity of the fermentation process statistical-mathematical methods like Plackett-Burman designed was used to select nutrients and response surface optimisation techniques were tested (Hendrix C. Chemtech 1980,<br><br><br>
10, 488-497) in order to determine the optimum level of each key independent variable. Experiments to improve the culture conditions like incubation temperature and agitation have also been done. Finally a highly safracin B producer medium named 16B (152 g/1 of mannitol, 35g/l of G20-25 yeast, 26 g/1 of CaC03 , 14 g/1 of ammonium sulphate, 0.18 g/1 of ferric chloride, pH 6.5) was selected.<br>
The safracin production was assay testing the capacity of inhibition a Bacillus subtilis solid culture by 10 μl of the supernatant of a 3 days Pseudomonas sp. culture incubated at 27°C (Alijah et al. Appl Microbiol Biotechnol 1991, 34, 749-755). P. fluorescens A2-2 cultures produce inhibition zones of 10-14 mm diameter while non-producing mutants did not inhibit B. subtilis growth. Three isolated clones had the safracin biosynthetic pathway affected. In order to confirm the results, HPLC analysis of safracin production was performed.<br>
HPLC analysis of safracin production.<br>
The supernatant was analysed by using HPLC Symmetry C-18. 300A, 5 urn , 250 x 4.6 mm column (Waters) with guard-column (Symmetry C-18, 5jim 3.9 x 20 mm, Waters). An ammonium acetate buffer (10 mM, 1% Diethanolamine, pH 4.0)- acetonitrile gradient was the mobile phase. Safracin was detected by absorption at 268 nm. In Fig. 6, HPLC profile of safracin and safracin precursors produce by P. fluorescens A2-2 strain and different safracin-like structures produced by P. fluorescens mutants are shown.<br>
Example 3. Cloning and sequence analysis of safracin cluster<br>
Inverse PCR and phage library hybridisation<br><br><br>
Southern hybridisation on mutant chromosomal DNAs verified the correct gene disruption and demonstrated that the peptide synthethase fragment cloned into pK18mob plasmid was essential for the production of safracin. Analysis of the non safracin producers mutants obtained demonstrated that all of them had a gene disruption into the same gene, sacA.<br>
Icverse, PCR from genomic DNA. and screening of a. phage, library of P. fluorescens A2-2 genomic DNA revealed the presence of additional genes flanking sacA gene, probably involved in safracin biosynthesis.<br>
The GenBank accession number for the nucleotide sequence data of the P. fluorescens A2-2 safracin biosynthetic cluster is AY061859.<br>
Cosmid library construction and heterologous expression<br>
To determine whether safracin cluster was able to confer safracin biosynthetic capability to a non producer strain, it was cloned into a wide range cosmid vector (pLAFR3, Staskawicz B. et ah J Bacteriol 1987, 169, 5789-5794) and conjugated to a different Pseudomonas sp collection strains.<br>
To obtain a clone containing the whole cluster, a cosmid library was constructed and screened. For this purpose, chromosomal DNA was partially digested with the restriction enzyme Psfl, the fragments were dephosphorylated an4 ligated into the PstI site of cosmid vector pLAFR3. The cosmids were packaged with Gigapack III gold packaGlng extracts (Stratagene) as manufacturer's recommendations. Infected cells of strain XL 1-Blue were plated on LB-agar supplemented with 50 μg/ml of tetracycline. Positives clones were selected using colony hybridization with a DIG-labeled DNA fragment belonGlng to the 3'-end of the safracin cluster. In order to ensure the cloning of the whole cluster, a new colony hybridization with a 5'-end DNA fragment was done. Only cosmid pL30p showed multiple hybridizations with DNA probes. To confirm the accurate cloning, PCR amplification and DNA-sequencing with DNA oligonucleotides<br><br><br>
belonGlng to the safracin sequence were carried out. The size of the insert of pL30P was 26,705 bp . The pL30p clone DNA was transformed into E. coli S17XPIR and the resulting strain were conjugated with the heterologous Pseudomonas sp. strains. The pL30p cosmid was introduced into P. fluorescens CECT378 and P. aeruGlnosa CECT110 by biparental conjugation as described above. Once a clone encoding the whole cluster was identified, it was determined whether the candidate was capable of producing safracin. Safracin production in the conjugated strains was assessed by HPLC analysis and biological assay of broth cultures supernatants as previously described.<br>
The strain P. fluorescens CECT378 expressing the pL30p cosmid (PM-19-002) was able to produce safracin in considerable amounts, whereas safracin production in P. aeruGlnosa CECT110 strain expressing pL30P (PM-16-002) was 10 times less than the CECT378. Safracin production in these strains was about 22 % and 2 % of the total production in comparison with the natural producer strain.<br>
Genes   involved   in   the formation   of safracin.   Sequence   analysis   of sacABCDEFGH and sacIJ operons<br>
Computer analyses of the DNA sequence of pL30P revealed 14 ORFs (Fig. 1). A potential ribosome binding site precedes each of the ATG start codons.<br>
In the sacABCDEFGH operon, three very large ORFs, sacA, sacB and sacC (positions 3052 to 6063, 6080 to 9268 and 9275 to 13570 of the P. fluorescens A2-2 safracin sequence SEQ ID NO:l, respectively) can be read in the same direction and encode the putative safracin NRPSs: SacA (1004 amino acids, Mr 110452), SacB (1063 amino acids, Mr 117539) and SacC (1432 amino acids, Mr 157331). The three NRPSs genes contain the domains resembling amino acid activating domains of known peptide synthetases. Specifically, the amino acid activating domains from these NRPS genes are very similar to three of the four amino acid activating<br><br>
domains (Gly, Tyr and Tyr) found in the Myxococcus xanthus saframycin NRPSs (Pospiech et al. Microbiology 1995, 141, 1793-803; Pospiech et al. Microbiol 1996, 142, 741-746). In particular, SacA (SEQ ID NO:2) shows 33% identity with saframycin Mxl synthetase B protein (SafB) from M. xanthus (NCBI accession number U24657), whereas SacB (SEQ ID NO:3) and SacC (SEQ ID NO:4) share, respectively, 39% and 41% identity with saframycin Mxl synthetase A (SafA) from M. xanthus (NCBI accession number U24657). The Fig. 2 shows a comparison among SacA, SacB y SacC and the different amino acid activating domains of saframycin NRPS. Downstream sacC five small ORFs reading in the same direction as the NRPSs genes exist (Fig.l). The first one, sacD (position 13602 to 14651 of P. fluorescens A2-2 safracin sequence), encodes a putative protein, SacD (350 amino acids, Mr 39187; SEQ ID NO:5), with no similarities in the GeneBank DB. The next one, sacE (position 14719 to 14901 of P. fluorescens A2-2 safracin sequence), encodes a small putative protein called SacE (61 amino acids, Mr 6729; (SEQ ID NO:6)), which shows some similarity with proteins of unknown function in the databases (ORF1 from Streptomyces viridochromogenes (NCBI accession number Y17268; 44% identity) and MbtH from Mycobacterium, tuberculosis (NCBI accession number Z95208; 36% identity). The third ORF, sacF (position 14962 to 16026 of P. fluorescens A2-2 safracin sequence), encodes a 355-residue protein with a molecular weigh calculated of 39,834 (SEQ ID NO:7). This protein most closely resembles hydroxyneurosporene methyltransferase (CrtF) from Chloroflexus aurantiacus (NCBI accession number AF288602; 25% identity). The nucleotide sequence of the fourth ORF, sacG (position 16115 to 17155 of P. fluorescens A2-2 safracin sequence), predicted a gene product of 347 amino acids having a molecular mass of 38,22 kDa (SEQ ID NO: 8). The protein, called SacG, is similar to bacterial O-methyltransferases, including O-dimethylpuromycin-O-methyltransferase (DmpM) from Streptomyces anulatus (NCBI accession number P42712; 31% identity). A computer search also shows that this protein contains the<br><br><br><br>
three sequence motifs found in diverse S-adenosylmethionine-dependent methytransferases (Kagan and Clarke, Arch. Biochem. Biophys. 1994, 310, 417-427). The fifth gene, sacH (position 17244 to 17783 of P. fluorescens A2-2 safracin sequence), encodes a putative protein SacH (180 amino acids, Mr 19632; (SEQ ID NO:9). A computer search for similarities, between the deduced amino acid sequence of SacH and other protein sequences, revealed identity with some conserved hypothetical proteins of unknown function, which contains a well conserved transmembrane motif and a dihydrofolate reductase-like active site (Conserved hypothetical protein from Pseudomonas aeruGlnosa PAOl, NCBI accession number P3469; 35% identity).<br>
Upstream sacABCDEFGH operon, reading in opposite sense, a two genes operon, sacIJ, is located. The sad gene (position 2513 to 1854) encodes a 220-amino acids protein (Mr 24219; (SEQ ID NO: 10) that most closely resembles ubiquinone/manequinone methyltrasnferase from Thermotoga maritime (NCBI accession number AE001745; 32% identity). The sacJ gene (position 1861 to 335) encodes a 509-amino acid protein (SEQ ID NO: 11), with a molecular mass of 55341 Da, similar to bacterial monooxygenases/hydroxylases, including putative monooxygenase from Bacillus subtilis (NCBI accession number Y14081; 33% identity) and Streptomyces coelicolor (NCBI accession number AL109972; 29% identity).<br>
SacABCDEFGH and sacIJ operons are transcribed divergently and are separated by 450 bp approximately. Both operons are flanked by residual transposase fragments.<br>
Related safracin cluster genes<br>
A putative ORF (orfl; position 18322 to 19365 of P. fluorescens A2-2 safracin sequence) located at the 3'-end of the safracin sequence has been<br><br><br>
found {Fig. 1). ORFl protein (SEQ ID NO: 12) shows similarity with aminopeptidases from the Gene Bank DataBase (peptidase M20/M25/M40 family from Caulobacter crescentus CB15; NCBI accession number NP422131; 30% identity). Using the strategy described in Example 2, the gene disruption of orfl do not affect safracin production in P. fluorescens A2-2.<br>
At the 3'-end of the safracin sequence cloned in pL30p cosmid, three putative ORFs (orf2, orf3 and orf4), were found. Reading in opposite direction than sacABCDEFGH operon, or/2 gene (position 22885 to 21169 of SEQ ID NO:l) codes for a protein, ORF2 (SEQ ID NO: 13), with similarities to Aquifex aeolicus HoxX sensor protein (NCBI accession number NCO00918.1; 35% identity), whereas or/3 gene (position 23730 to 23041 of SEQ ID NO:l) codes for ORF3 protein (SEQ ID NO: 14) which shares 44% identity with a glycosil transferase related protein from Xanthomonas axonopodis pv. Citri str. 306 (NCBI accession number NP642442).<br>
The third gene is located at the 3'-end of SEQ ID NO:l (position 25037 to 26095). This gene, named orf4 (position 2513 to 1854), encodes a protein, ORF4 (SEQ ID NO: 15), that most closely resembles to a hypothetical isochorismatase family protein YcdL from Escherichia coli. (NCBI accession number P75897; 32% identity).<br>
Presumably, these three genes would not be involve in the safracin biosynthetic pathway, however, future gene disruption of these genes will confirm this assumption.<br>
The different DNA sequences found are listed at the end of the description.<br>
Example 4. Functional analysis of the safracin loci and search for<br><br>
possible precursors<br>
Since the pathway for synthesis of safracin in P. fluorescens A2-2 is at present unknown, the inactivation of each of the genes described in Example 3 would permit fundamental studies on the mechanism of safracin biosynthesis in this strain.<br>
In order to analyze the functionality of each particular protein in the safracin production pathway, disruption of each particular gene of the cluster, but sacE, was performed. All of the genetic mutants were obtained following the disruption strategy previously described.<br>
Figure 6 is a summary of the different mutants constructed in this invention as well as a summary of the compounds produced by the mutants as a consequence of the gene disruption. In the wild type strain both safracin A and B and other compounds, P2 and P14, were clearly detected by HPLC (see Fig. 6,WT). The gene disruption of the sacA (PM-S1-002), sacB (PM-S1-003), sacC (PM-S1-004), sacD (PM-S1-010), sacF (PM-Sl-008), and sacG (PM-S 1-009), genes generated mutants that were unable to produce neither safracin A and safracin B, nor the precursor compounds with retention times beneath 15 min, P2 and P14 respectively. The structure elucidation of P14 and P2 revealed that P14 is a 3-methyl-O methyl tyrosine, where as P2 is a 3-hydroxy-5~methyl-0-methyl tyrosine. Because of the small size of the sacE gene, the sacE- mutant was not possible to be obtained by gene disruption, but deletion of this gene is in process. The overexpression of SacE protein, in trans, had no effect on safracin B/A production. The sacl- mutants (PM-S1-006) produced P2, P14 and significant amount of a compound called P19B (Fig. 6; Fig7b*). Structure elucidation of P19B revealed that this compound is a safracin-like molecule in which the N-Met and one of the OH from the quinone ring are missing. In the sacJ- mutants (PM-S1-005), P2, P14, P19B and two new compounds called P22A and P22B were obtained (Fig. 6; Fig. 7a*).<br><br>
Structure elucidation of P22A and P22B revealed that they are safracin A and safracin B like molecules, respectively, without one of the -OH group from the quinone ring. The biological assay of the sac? and the sacJ-mutants extracts revealed very low activity against Bacillus subtilis.<br>
The disruption of sacl gene with the reconstitution of the sacJ gene expression resulted in a new safracins producer mutant, PM-S1-007. The two antibiotics produced, at levels of production as high as the levels of safracin A and safracin B in the wild type strain, have been named as safracin D and safracin E (Fig. 7c*). The safracin D and safracin E are safracin B and safracin A like molecules, respectively, where the N-methylation is missing.<br>
These results strongly suggest that i) sacA, sacB and sacC genes encode for the safracin NRPSs; ii) sacD, sacF and sacG genes are responsible for the transformation of L-Tyr into the L-Tyr derivative P2 and iii) sacl and sacJ are responsible for the tailoring modifications that convert P19 and P22 into safracin.<br>
Characterization of Natural Precursors: p. 14<br><br><br><br>
Strain:<br>
Pseudomonas fluorescens A2-2 (wild type) (PM-S1-001)<br>
Fermentation conditions:<br>
Seed medium YMP3 containing 1% glucose; 0.25% beef extract; 0.5% bacto-peptone; 0.25% NaCl; 0,8% CaC03 was inoculated with 0.1% of a frozen vegetative stock of the microorganism, and incubated on a rotary-shaker (250 rpm) at 27°C. After 30h of incubation, the 2% (v/v) seed culture was transferred into 2000 ml Erlenmeyer flasks containing 250 ml of the M-16B production medium, composed of 15.2 % mannitol; 3.5 % Dried brewer's yeast; 1.4 % (NH4)2 S04; 0.001%; FeCb; 2.6 % COaCa. The temperature of the incubation was 27°C from the inoculation till 40 hours and then, 24°C to final process (71 hours). The pH was not controlled. The agitation of the rotatory shaker was 220 rpm with 5 cm eccentricity.<br>
Isolation:<br>
After 71 hours of incubation, 2 Erlenmeyer flasks were pooled and the 500 ml of fermentation broth was clarified by 7.500 rpm centrifugation during 15 minutes. 50 grams of the resin XAD-16 (Amberlite) were added to the supernatant and mixed during 30 minutes at room temperature. Then, the resin was recovered from the clarified broth by filtration. The resin was washed twice with distilled water and extracted with 250 ml of isopropanol (2-PrOH). The alcohol extract was dried under high vacuum till obtention of 500 mg crude extract. This crude was dissolved in methanol and purified by chromatographic column using Sephadex LH-20 and methanol as mobile phase. The P-14 compound was eluted and dried as a 15 mg yellowish solid. The purity was tested by analytical HPLC and 1H NMR.<br><br><br>
P-14 was also isolated in a similar way from cultures of the sacJ- mutant (PM-S 1-005), using semipreparative HPLC as the last step in the purification process.<br>
Biological activities: NO ACTIVE<br>
Spectroscopic data:<br>
ESMS m/z254 (C11H14N03Na2+), 232 (CnHisNOsNa-), 210 (M+H+). 1H RMN (300 MHz, CD3OD): 7.07 (d, J=8.1 Hz, H-9), 7.06 (s, H-5), 6.84 (d, J=8.1 Hz, H-8), 3.79 (s, H-ll), 3.72 (dd, J=8.7, 3.9 Hz, H-2), 3.20 (dd, J=14.4, 3.9 Hz, H-3a), 2.91 (dd, J=14.4, 8.9 Hz, H-3b), 2.16 (s, H-10). 13C RMN (75 MHz, CD3OD): 174.1 (C-l), 158.6 (C-7), 132.5 (C-5), 128.9 (C-9), 128.5 (C-4), 128.0 (C-6), 111.4 (C-8), 57.6 (C-2), 55.8 (C-ll), 37.4 (C-3), 16.3 (C-10)<br>
P-2<br><br>
Strain:<br>
Pseudomonas fluorescens A2-2 (wild type) (PM-S 1-001)<br><br><br>
Fermentation conditions:<br>
The same process than P-14<br>
Isolation:<br>
Similar procedure as the P-14, except in the Sephadex chromatography, where the fractions containing P-2 have eluted later. A semi-preparative HPLC step (Symmetry Prep C-18 column, 7.8 x 150 mm, AcONH4 10 mM pH=3/CH3CN 95:5 held for 5 min and then gradient from 5 to 6.8 % of CH3CN in 3 min) has been necessary to purify the P-2. Also this compound has been isolated from the fermentation broth of the Pseudomonas putida ATCC12633+pB5H83 (PM-17-004) as result of heterologous expression.<br>
Biological activities: NO ACTIVE<br>
Spectroscopic data:<br>
ESMS m/z 226 [M+H]+; 1H RMN (CD3OD, 300 MHz): 6.65 (d, J- 1.8 Hz, H-5), 6.59 (d, J= 1.8 Hz, H-9), 3.72 (s, H-ll), 3.71 (dd, J= 9.0, 4.2 Hz, H-2), 3.16 (dd, J= 14.4, 4.2 Hz, H-3a), 2.83 (dd, J= 14.4, 9.0 Hz, H-3b), 2.22 (s, H-10); we RMN (DMSO, 75 MHz): 170.88 (s, C-l), 150.025 (s, C-7), 144.56 (s, C-8), 132.28 (s, C-4), 130.36 (s, C-6), 121.73 (d, C-5), 115.55 (d, C-9), 59.06 (q, 7-OMe), 55.40 (d, C-2), 36.21 (t, C-3), 15.86 (q, 6-Me).<br>
Characterization of Safracins like compounds obtained by knock out COMPOUND P-22B<br><br><br><br>
Strain:<br>
sac J- mutant from P.fluorescens A2-2 (PM-S1 -005)<br>
Fermentation conditions:<br>
50 liters of the SAM-7 medium (50 1) composed of dextrose (3.2%), mannitol (9.6%), dry brewer's yeast (2%), ammonium sulphate (1.4%), potassium secondary phosphate (0.03%), potassium chloride (0.8%), Iron (III) chloride 6-hydrtate (0.001%), L-tyrosine (0.1%), calcium carbonate (0.8%), poly- (propylene glycol) 2000 (0.05%) and antifoam ASSAF 1000 (0.2%) was poured into a jar-fermentor (BioenGlneering LP-351) with 75 1 total capacity and, after sterilization, sterile antibiotics (amplicillin 0.05 g/1 and kanamycin 0.05 g/1) were added. Then, it was inoculated with seed culture (2%) of the mutant strain PM-S 1-005. The fermentation was carried out during 71 h. under aerated and agitated conditions (1.0 1/1/min and 500 rpm). The temperature was controlled from 27°C (from the inoculation till 24 hours) to 25°C (from 24h to final process). The pH<br><br>
was controlled at pH 6.0 by automatic feeding of diluted sulphuric acid from 22 hours to final process.<br>
Isolation<br>
The whole broth was clarified (Sharpies centrifuge). The pH of the clarified broth was adjusted to pH 9.0 by addition of NaOH 10% and extracted with 25 litres of ethyl acetate. After 20' mixing, the two phases were separated. The organic phase was frozen overnight and then, filtered for removing ice and evaporated to a greasy dark green extract (65.8 g). This extract was mixed with 500 ml hexane (250 ml two times) and filtered for removing hexane soluble impurities. The remaining solid, after drying, gave a 27.4 g of a dry green-beige extract.<br>
This new extract was dissolved in methanol and purified by a Sephadex LH-20 chromatography (using methanol as mobile solvent) and the safracins-like compounds were eluted in the central fractions {Analyzed on TLC conditions: Silica normal phase, mobile phase: EtOAaMeOH 5:3. Aprox. Rf valor: 0.3forP-22B, 0.25 P-22A and 0.1 for P-19).<br>
The pooled fractions, (7,6g) containing the three safracin-like compound were purified by a Silica column using a mixture of EtOAc:MeOH from 50:1 to 0:1. and other chromatographic system (isocratic CHCl3:MeOH:H20:AcOH 50:45:5:0.1). Compounds P22-A, P22-B and P19-B were purified by reversed-phase HPLC (SymmetryPrep C-18 column 150 x 7.8 mm, 4 mL/min, mobile phase: 5 min MeOH:H20 (0.02 % TFA) 5:95 and gradient from MeOH:H20 (0.02 % TFA) 5:95 to MeOH 100 % in 30 min).<br>
Biological activities ofsafracin P-22B<br><br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis. lO μg/disk (6mm diameter): 10 mm inhibition zone<br>
Spectroscopic data:<br>
HRFABMS m/z 509.275351 [M-H20+H]+ (calcd for C28H37N4O5 509.276396 A 1.0 mmu); LRFABMS using m-NBA as matrix m/z (rel intensity) 509 [M-H20+H]+ (5), 460 (2.7), 391 (3).<br>
1H NMR (CD3OD, 500 MHz): 6.70 (s, H-15), 6.52 (s, H-5), 4.72 (bs, H-ll), 4.66 (d, J= 2.0 Hz, H-21), 4.62 (dd, J= 8.4, 3.7 Hz, H-1), 3.98 (bd, J= 7.6 Hz, H-13), 3.74 (s, 7-OMe), 3.71 (s, 17-OMe), 3.63 (m, overlapped signal, H-25), 3.62 (m, overlapped signal, H-3), 3.30 (m, H-22a), 3.29 (m, H-14a), 3.18 (d, J= 18.6 Hz, H-14b), 2.90 (m, H-4a), 2.88 (m, H-22b), 2.76 (s, 12-NMe), 2.30 (s, 16-Me), 2.22 (m, H-4b), 1.16 (d, J= 7.4 Hz, H-26);<br>
13C NMR (CD3OD, 125 MHz): 170.75 (s, C-24), 149.24 (s, C-18), 147.54 (s, C-8), 145.95 (s, C-7), 145.82 (s, C17), 133.93 (s, C-16), 132.31 (s, C-9), 131.30 (s, C-6), 128.95 (s, C-20), 121.93 (d, C-15), 121.76 (d, C-5), 121.44 (s, C-10), 112.45 (s, C-19), 92.87 (d, C-21), 60.86 (q, 7-OMe), 60.76 (q, 17-OMe), 59.39 (d, C-ll), 57.96 (d, C-13), 55.51 (d, C-l), 54.29 (d, C-3), 50.08 (d, C-25), 45.55 (t, C-22), 40.43 (q, 12-NMe), 32.56 (t, C-4), 25.84 (t, C-14), 17.20 (q, C-26), 16.00 (q, 16-Me), 15.81 (q, 6-Me).<br><br>
COMPOUND P-22A<br><br>
Strain:<br>
The same as for P-22B<br>
Fermentation conditions: The same as for P-22B<br>
Isolation:<br>
The same as for P-22B<br>
Biological activities ofsafracin P-22A Antitumor activities<br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis. lO ug/disk (6mm diameter): NO ACTIVE<br><br>
Spectroscopic data:<br>
HRFABMS m/z 511.290345 [M+H]+ (calcd for C28H39N4O5 511.292046 A 1.7 mmu); LRFABMS using m-NBA as matrix m/z (rel intensity) 511 [M+H]+ (61), 409 (25), 391 (4); 1H NMR (CD3OD, 500 MHz): 6.68 (s, H-15), 6.44 (s, H-5), 3.71 (s, 7-OMe), 3.67 (s, 17-OMe), 2.72 (s, 12-NMe), 2.28 (s, 16-Me), 2.20 (s, 6-Me), 0.87 (d, J- 7.1 Hz, H-26);<br>
COMPOUND P-19B<br><br>
Strain:<br>
The same as for P-22B<br>
Fermentation conditions: The same as for P-22B<br>
Isolation<br>
The same as for P-22B<br><br><br><br>
Biological activities ofsafracin P-19B Antitumor activities<br>
Antimicrobial activity:  On solid medium<br>
Bacillus subtilis. lOpg/disk (6mm diameter): NO ACTIVE<br>
Spectroscopic data:<br>
HRFABMS m/z 495.260410 [M-H20+H]+ (calcd for C27H35N4O5 495.260746 A 0.3 mmu); LRFABMS using m-NBA as matrix m/z (rel intensity) 495 [M-H20+H]+ (13), 460 (3), 391 (2); *H NMR (CD3OD, 500 MHz): 6.67 (s, H-15), 6.5 (s, H-5), 3.73 (s, 7-OMe), 3.71 (s, 17-OMe), 2.29 (s, 16-Me), 2.24 (s, 6-Me), 1.13 (d, J = 7.1 Hz, H-26);<br><br>
New Safracin compounds obtained by knock out SAFRACIN D<br><br>
Strain:<br>
sac F with sacJ expression reconstitution from P.fluorescens A2-2 (PM-S1-<br>
007)<br>
Fermentation conditions:<br>
50 litres of the SAM-7 medium (50 1) composed of dextrose (3.2%), mannitol (9.6%), dry brewer's yeast (2%), ammonium sulphate (1.4%), potassium secondary phosphate (0.03%), potassium chloride (0.8%), Iron (III) chloride 6-hydrtate (0.001%), L-tyrosine (0.1%), calcium carbonate (0.8%), poly- (propylene glycol) 2000 (0.05%) and antifoam ASSAF 1000 (0.2%) was poured into a jar-fermentor (BioenGlneering LP-351) with 75 1 total capacity and, after sterilization, sterile antibiotics (arnplicillin 0.05 g/1 and kanamycin 0.05 g/1) were added. Then, it was inoculated with seed culture (2%) of the mutant strain PM-S1-007. The fermentation was carried out during 89  h.  under aerated and agitated  conditions  (1.0<br><br>
1/1/min and 500 rpm). The temperature was controlled from 27°C (from the inoculation till 24 hours) to 25°C (from 24h to final process). The pH was controlled at pH 6.0 by automatic feeding of diluted sulphuric acid from 27 hours to final process.<br>
Isolation:<br>
The cultured medium (45 1) thus obtained was, after removal of cells by centrifugation, adjusted to pH 9.5 with diluted sodium hydroxide, extracted with 25 liter of ethyl acetate twice. The mixture was carried out into an agitated-vessel at room temperature for 20 minutes. The two phases were separated by a liquid-liquid centrifuge. The organic phases were frozen at -20°C and filtered for removing ice and evaporated until obtention of a 35g. oil-dark-crude extract. After a 5 1. hexane triturating, the extract (12.6g) was purified by a flash-chromatographic column (5.5 cm diameter, 20 cm length) on silica-normal phase, mobile phase: Ethyl acetate: MeOH: 1 L of each 1:0; 20:1; 10:1; 5:1 and 7:3, 250 ml- fractions were eluted and pooled depending of the TLC (Silica-Normal, EtOAc.MeOH 5:2, Safracin D Rf 0.2, safracin E 0.05). The fraction containing impure safracin D and E was evaporated under high vacuum (2.2 g). An additional purification step was necessary to separate D and E on similar conditions (EtOAc:MeOH from 1:0 to 5:1), from this, the fractions containing safracin D and E are separate and evaporated and further purification by Sephadex LH-20 column chromatography eluted with methanol.<br>
The safracins D and E obtained were independent precipitated from CH2CI2 (80 ml) and Hexane (1500 ml) as a green/yellowish-dried solid (800 mg safracin D) and (250 mg safracin E).<br>
Biological activities Safracin D Antitumor screening:<br><br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis.   lO μg/disk  (6mm  diameter):   Inhibition zone:   15  mm<br>
diameter<br>
Spectroscopic data<br>
ESMS: m/z 509 [M-H20+H]+; 1H NMR (CDCb, 300 MHz): 6.50 (s, C-15), 4.02 (s, OMe), 3.73 (s, OMe), 2.22 (s, Me), 1.85 (s, Me), 0.80 (d, J= 7.2 Hz); "C NMR (CDCb, 75 MHz): 186.51, 181.15, 175.83, 156.59, 145.09, 142.59, 140.78, 137.84, 131.20, 129.01, 126.88, 121.57 (2 x C), 82.59, 60.92, 60.69, 53.12, 21.40, 50.68, 50.22, 48.68, 40.57, 29.60, 25.01, 21.46, 15.64, 8.44.<br><br><br>
Strain:<br>
The same than safracin D<br>
Fermentation conditions: The same batch as safracin D<br>
Isolation:<br>
See safracin D conditions<br>
Biological activities Safracin E Antitumor screening:<br><br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis. lO μg/disk (6mm diameter): 9.5 mm inhibition zone<br>
Spectroscopic data<br>
ESMS: m/z 511 [M+H]+; 1H NMR (CDCl3, 300 MHz): 6.51 (s, C-15), 4.04 (s, OMe), 3.75 (s, OMe), 2.23 (s, Me), 1.89 (s, Me), 0.84 (d, J = 6.6 Hz); ™C NMR (CDCb, 75 MHz): 186.32, 181.28, 175.83, 156.43, 145.27, 142.75, 141.05, 137.00, 132.63, 128.67, 126.64, 122.00, 120.69, 60.69, 60.21, 59.12, 58.04, 57.89, 50.12, 49.20, 46.72, 39.88, 32.22, 25.33, 21.29, 15.44, 8.23.<br><br><br>
Example 5. Cross-feeding experiments<br>
Heterologous expression of safracin biosynthetic precursors genes for P2 and PI 4 production<br>
In the attempt to shed light on the mechanism of the P2 and PI4 biosynthesis we have cloned and expressed the downstream NRPS genes to determine their biochemical activity.<br>
To overproduce P14, sacEFGH genes were cloned (pB7983) (Fig. 4).<br>
To overproduce P2 in a heterologous system, sacD to sacH genes were<br>
cloned (pB5H83)(Fig. 4). For this purpose we PCR amplified fragments<br>
harboring the genes of interest using oligonucleotides that contain a Xbal<br>
restriction site at the 5' end. Oligonucleotides PFSC79 (5'-<br>
CGTCTAGACACCGGCTTCATGG-3')	and	PFSC83	(5'-<br>
GGTCTAGATAACAGCCAACAAACATA-3') were used to amplify sacE to<br>
sacH	genes;	and	oligonucleotides	5HPT1-XB	(5'-<br>
CATCTAGACCGGACTGATATTCG-3')	and	PFSC83	(5'-<br>
GGTCTAGATAACAGCCAACAAACATA-3') were used to amplify sacD to sacH genes. The PCR fragments digested with Xbal were cloned into the Xbal restriction site of the pBBRl-MCS2 plasmid (Kovach et al, Gene 1994, 166, 175-176). The two plasmids, pB7983 and pB5H83, were introduce separately into three heterologous bacteria P. fluorescens (CECT 378), P. putida (ATCC12633) and P. stutzeri (ATCC 17588) by conjugation (see table II). When culture broth of the fermentation of the transconjugant strains was checked by HPLC analysis, big amounts of PI4 compound was visualized in the three strains containing pB7983 plasmid, whereas big amounts of P2 and some P14 product were observed when pB5H83 plasmid was expressed in the heterologa bacteria.<br><br><br>
Cross-feedmg<br>
As it was shown in Example 4, the sacF- (PM-S1-008) and sacG-(PM-S1-009) mutants were not able to produce neither safracins nor P2 and P14 compounds. The addition of chemically synthesized P2 to these mutants during their fermentation yields safracin production.<br>
Moreover, the co-cultivation of an heterologous strain of P. stutzeri (ATCC 17588) harboring plasmid pB5H83 (PM-18-004), which expression produces P2 and PI4, with either one of the two mutants sacF- and sacG-resulted in safracin production. The co-cultivation of an heterologous strain P. stutzeri (ATCC 17588) harboring plasmid pB7983 (PM-18-005), which expression produces only P14, with either one of the two P. fluorescens A2-2 mutants mentioned before resulted in no safracin production at all. These results suggest that P14 is transformed into P2, a molecule that can easily be transported in and out through the Pseudomonas sp. cell wall and which presence it is absolutely necessary for the biosynthesis of safracin.<br>
Example 6. Biological production of new "unnatural" molecules<br>
The addition of 2g/L of an specific modified P2 derivative precursor, P3, a 3-hydroxy-5-methyl-0-methyltyrosine, to the sacF mutant (PM-S1-008) fermentation yielded two "unnatural" safracins that incorporated the modified precursor P3 in its structure, Safracin A(OEt) and Safracin B(OEt).<br>
SAFRACIN B-Etoxi (Safracin B (OEt))<br><br><br>
Strain<br>
saf F - mutant from P.fluorescens A2-2 (PM-S 1-008)<br>
Fermentation conditions:<br>
Seed medium containing 1% glucose; 0.25% beef extract; 0.5% bacto-peptone; 0.25% NaCl; 0,8% CaC03 was inoculated with 0.1% of a frozen vegetative stock of the microorganism, and incubated on a rotary shaker (250 rpm) at 27°C. After 30h of incubation, the 2% (v/v) seed culture of the mutant PM-S 1-008 was transferred into 2000 ml Erlenmeyer flasks containing 250 ml of the M-16 B production medium, composed of 15.2 % mannitol; 3.5 % Dried brewer's yeast; 1.4 % (NH4)2 0.001%; FeCl3; 2.6 % C03Ca and 0.2% P3 (3-hydroxy-5-methyl-0-methyltyrosine) The temperature of the incubation was 27°C from the inoculation till 40 hours and then, 24°C to final process (71 hours). The pH was not controlled. The agitation of the rotatory shaker was 220 rpm with 5 cm eccentricity.<br>
Isolation<br>
4  x  2000/250  ml  Erlenmeyer flasks  were joined  together   (970   ml),<br>
centrifuged (12.000 rpm, 4°C, 10', J2-21 Centrifuge BECKMAN) to remove<br><br>
cells. The clarified broth (765 ml) was adjusted to pH 9.0 by NaOH 10%. Then, the alkali-clarified broth was extracted with 1:1 (v/v) EtOAc (x2). The organic phase was evaporated under high vacuum and a greasy-dark extract was obtained (302 mg).<br>
This extract was washed by an hexane trituration for removing impurities and the solids were purified by a chromatography column using Silica normal-phase and a mixture of Ethyl Acetate: Methanol (from 12:1 to 1:1). The fractions were analyzed under UV on TLC (Silica 60, mobile phase EtOAc:MeOH 5:4. Rf 0.3 (Safracin B-OEt and 0.15 Safracin A-OEt). From this, safracins B OEt (25 mg) and safracin A OEt (20 mg) were obtained.<br>
Biological activities of safracin B (OEt) Antitumor activities<br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis. lOpig/disk (6 mm diameter): 17,5 mm inhibition zone<br>
Spectroscopic data:<br>
ESMS: m/z 551 [M-H20+H]+; 1H NMR (CDCL3 300 MHz): 6.48 (s, H-15), 2.31 (s, 16-Mc), 2.22 (s, 12-NMe), 1.88 (s, 6-Me), 1.43 (t, J= 6.9 Hz, Me-Etoxy), 1.35 (t, J= 6.9 Hz, Me-Etoxy), 0.81 (d, J= 7.2 Hz, H-26)<br><br><br>
SAFRACIN A-Etoxi (Safracin A (OEt))<br><br>
Strain:<br>
The same as for Safracin B (OEt)<br>
Fermentation conditions:<br>
The same as for Safracin B (OEt)<br>
Isolation:<br>
4 x 2000/250  ml Erlenmeyer  flasks  were joined  together  (970  ml),<br>
centrifuged (12.000 rpm, 4°C, 10', J2-21 Centrifuge BECKMAN) to remove<br>
cells. The clarified broth (765 ml) was adjusted to pH 9,0 by NaOH 10%.<br>
Then, the alkali-clarified broth was extracted with 1:1 (v/v) EtOAc (x2). The<br>
organic phase was evaporated under high vacuum and a greasy-dark<br>
extract was obtained (302 mg).<br>
This extract was washed by an hexane trituration for removing impurities<br>
and the solids were purified by a chromatography column using Silica<br>
normal-phase and a mixture of Ethyl Acetate: Methanol (from 12:1 to 1:1).<br>
The fractions were analysed under UV on TLC (Silica. 60, mobile phase<br><br><br>
EtOAc:MeOH 5:4. Rf 0.3 Safracin B-OEt and 0.15 Safracin A-OEt). From this, safracins B OEt (25 mg) and safracin A OEt (20 mg) were obtained.<br>
Biological activities of safracin A (OEt): Antitumor activities<br><br>
Antimicrobial activity: On solid medium<br>
Bacillus subtilis. lOjjg/disk (6 mm diameter): 10 mm inhibition zone<br>
Spectroscopic data:<br>
ESMS: m/z 553 [M+H]+; 1H NMR (CDC13, 300 MHz): 6.48 (s, H-15), 2.33 (s, 16-Me), 2.21 (s, 12-NMe), 1.88 (s, 6-Me), 1.42 (t, J = 6.9 Hz, Me-Etoxy), 1.34 (t, J= 6.9 Hz, Me-Etoxy), 0.8 (d, J= 6.9 Hz, H-26)<br><br><br>
Example 7. Enzymatic transformation of Safracin B into Safracin A<br>
In order to assay the enzymatic activity of conversion of safracin B into safracin A, a 120 hours fermentation cultures (see conditions in Example.2.Biological assay (biotest) for safracin production) of different strains were collected and centrifuged (9.000 rpm x 20 min.). The strains assayed were P. fhiorescens A2-2, as wild type strain, and P.fluorescens CECT378 + pBHPT3 (PM-19-006), as heterologous expression host. Supernatant were discarded and cells were washed (NaCl 0.9 %) twice and resuspended in 60 ml phosphate buffer 100 mM pH 7.2. 20 ml from the cell suspension was distributed into three Erlenmeyer flask:<br>
A.	Cell suspension + Safracin B (400 mg/L)<br>
B.	Cell suspension heated at 100 °C during 10 min. + Safracin B<br>
(400 mg/L)  (negative control)<br>
C.	Cell suspension without Safracin B (negative control)<br>
The biochemical reaction was incubated at 27 °C at 220 rpm and samples were taken every 10 min. Transformation of safracin B into safracin A was followed by HPLC. The results clearly demonstrated that the gene cloned in pBHPT3, sacH, codes for a protein responsible for the transformation of safracin B into safracin A.<br>
Based on this results we did an assay to find out if this same enzyme was able to recognize a different substrate such as ecteinascidin 743 (ET-743) and transform this compound into Et-745 (with the C-21 hydroxy missing). The experiment above was repeated to obtain Erlenmeyer flasks containing:<br>
A. Cell suspension + ET-743 (567 mg/L aprox.)<br><br>
, <br>
B.	Cell suspension heated at 100 °C during 10 min. + ET-743(567<br>
mg/L) (negative control)<br>
C.	Cell suspension without ET-743 (negative control)<br>
The biochemical reaction was incubated at 27 °C at 220 rpm and samples were taken at o, 10 min, lh, 2h, 3h, 4h, 20h, 40h, 44h, 48h. Transformation of ET-743 into ET-745 was followed by HPLC. The results clearly demonstrated that the gene cloned in pBHPT3, sacH, codes for a protein responsible for the transformation of Et-743 into Et-745. This demonstrates that this enzymes recognizes ecteinascidin as substrate and that it can be used in the biotransformation of a broad range of structures.<br>
SEQUENCES<br> Pharma Mar<br>  The gene cluster involved in safracin biosynthesis and its uses for genetic enGlneering  aaa   15<br> Patentln version 3.1<br> SEQ ID 1<br> Lenght: 26705<br> Type DNA<br> Organism:Pseudomonas fluorescens A2-2<br> Feature:<br> NAME/KEY: sacB<br> LOCATION: (6080)..(9268)<br> OTHER INFORMATION: non ribosomal peptide synthetase gene<br><br>	sacC<br>	(9275)..(13570)<br>	non ribosomal peptide synthetase gene<br><br><br><br> sacD<br> (13602)..(14651)<br> Hypothetical protein gene<br><br>	sacE<br>	(14719)..(14901)<br>	Hypothetical protein gene<br><br> sacF<br> (14962)..(16026)<br> methyl-transferase protein gene<br><br> sacG<br> (16115)..(17155)<br> methyl-transferase protein gene<br><br> sacH<br> (17244)..(17783)<br> hypothetical protein gene<br><br>	sad<br>	(1854)..(2513)<br>	Complementary, methyl-transferase protein gene<br><br>	sacJ<br>	(335)..(1861)<br>	Complementary, mono-oxygenase protein gene<br><br> orfl<br> (18322)..(19365)<br> aminoacid peptidase-like protein<br> orf2<br><br> (21169)..(22885)<br> complementary; hox-like regulator protein<br><br> orf3<br> (23041)..(23730)<br> complementary; glycosil transferase-like protein<br><br> orf4<br> (25037)..(26095)<br> isochorismatase-like protein<br><br>  SEQUENCE 1 ctgcaggtgg tttgcgcgcg	gaagacccgc	cactgccggt	gcgctcgttt	gaattgcaca	60<br>
tggcgtggcg	tgggtcgcag gataatgatc	cgggggagcg gtggttgcgg	tcgcggattc	120<br>
agatgttttt	tggcgacccc	gatagccttt	aattaaactc	cactaaaatc	ggcgattgca	180<br>
gagcctgagt	acaacacggc	tactggactg	aagtgggcgc	atcgtgccgc	atagccatag	240<br>
tgatctcggt	gtgtctcgcc	atgtcccggc	ccaggtcgta	ggtcatgctc	ttgcgcattg	300<br>
ccagcatctt	cgtgctccct	tgccagctgt	ttcaggtcag	gctctgacgc	gcggatttag	360<br>
aatcgtccag	cagccactca	cccaagcgct	ccttggccaa	ggtcgatttt	ttaccgaccc	420<br>
agatcaccac	gccatccggc	ctgacgatga	ctccctcgcc	agcggacaag	ctgttattgt	480<br>
gcgaatcctc	gcagatagat	gccgtttgca	accccggaaa	atcacgtcga	agatcggcgg	540<br>
ccagtgcttt	ggcacgatga	tgtaccagga	caaaccgccc	tgctcgcagc	aactgggtca	600<br>
agctctgtcg	tggtaatcgc	tcaccctcgg	gaagcaggct	caacaggggt	aaacgtcggc	660<br>
ccaccaagcg	atgatcgcct	cggcgccgca	cactgtcata	gcgcacgccc	tctcccgcca	720<br>
gtgcactgac	caccttctgc	gccacatacg	gggcccgtgt	cgcctgcaag	ccgatccagt	780<br>
gaatcagacg	gcctataggg	ccggaagccg	tattgaatcg	gaaaagcaga	tccgtgttgc	840<br>
gcaaggctgc	cgccgcaata	ggcctacgct	cggcctcgta	actctccaga	agatccatcg	900<br>
gcaatgtggc	ctgtatcaca	cccgccagct	tccaggcgag	gttcgccgcg	tcaccgatcc	960<br>
ccatctgcaa	accttgcccc	ccggcgggga	cgtgggtgtg	agcagcatct	cccagcagaa	1020<br>
ataccctccc	ctggcgataa	tgagtcgcca	ggcgctgctg	gctgcggtaa	cgagcgctcc	1080<br><br><br>
acagcacctg cgccaatccg aaatcggttc ccagaatatc tttcatccct ccggcaattt	1140<br>
cctcgtgggt gaccggctgt ttgaccggag tatccatacg ttcgttgtct tcgatactga	12 00<br>
cgcgataact gccatcgggt aatggaaaca gggcaaccag acccctggag accgaccttg	1260<br>
catggactgc aggtgaaggc ggatttctca agacaacgtc cgccaccacc aacgaatgct	132 0<br>
tgtagtcctg cccgacaaac gaaatattga ggagttggcg gacagaactg ttgaccccat	1380<br>
cagcccccag cacccagtcg tagcggcttt gctgcacgct gccggtctcg ctgtgttcca	144 0<br>
gggttacctc aacgtgtaaa tcgccggcat ccagagcctt tagcgcatac ccacgcttca	1500<br>
gattcacccc cttgcgattg acccaatcag tcaacaccga ctcggtctga gactgtggga	1560<br>
tgatcaccat gtggggatac tcacaaggga gtttggaaaa tgagagcgtt cggcccgcct	1620<br>
tgtcacccaa cggcgcgctt gcccagacga tcccccgacg tatcatctca tctgccacgc	1680<br>
cccaggcatt gagcaactcc agggtcacgg gctccaagcc aaaggcccgg gaatggggag	1740<br>
aggcagccgg tcttttatcg atgagatcaa ccgctatacc caactcggcc agagctgcag	1800<br>
ccacagccaa cccgacgggc cccgccccca cgaccaggac ctgtttattt ttaacgacca	1860<br>
tgatcaccca cctctccaca gcaggggcgg aacgtggcga caatgacgtg ctggtcggtg	1920<br>
acgctcgttt ccatgctctg cagcccggcc gtttgcgcga gctggattac ctcatgctct	1980<br>
gatcgataat ctgccagtga gcccagcagc cagttgatca atgtggtcaa tccccagccc	2040<br>
gcaggcaggt tctccaccag gcagaaaagg ccctgaggct tgagcacgcg acggacctca	2100<br>
ttcaggctga cagacttgtc agcccaatgg ccgaacgaca tcgagcacac caccagatcc	2160<br>
atgctttgcg aggggaatgg caaggcttcg gcaactcctt tgacgaacga ggcgaaggga	222 0<br>
cgtcgtttgg cggcctcgtc gaccatgccc tgagccgggt cgacgccttc gaagcgcgcc	22 80<br>
tcaggccaca gagcgaacat gcgttcgatc aatgcgccgg taccacaacc gatgtccaga	2340<br>
acacgctccg gtctcgaggt gcacatccat cggctcaaca ttcgcagaca gtcgtcatgg	2400<br>
gcctggctca gtttggtacc gtacttctct tcatacgtcg gcgcgatacg gctgaacgtc	2460<br>
cgcacaaaac ctggatttcc attgcctttc ccgccagaaa atacgttaga cattattgaa	2520<br>
catccatata tcaacagtta tccgccaagg accatagtag agaaaatcca tcccatccaa	258 0<br>
ataaaaatta aataagtggg gctaaccgca atccagggaa actctgaaaa ggcccgctac	2 640<br>
ttgtcgacgc ggctgtctgg aggccgcata gttactgaac ttactattaa aagactgggc	2 700<br>
tttttcagag ccccaccgga tgttggctcc ttgtccatca tttcgggggc actgtaacat	2760<br>
tctgttacct ggctatcgct tgacttttaa tctgaacggg caattatagg tctaaccgca	282 0<br><br><br>
acagccccac	ggcgcttaag	ttcgaaaaaa<br>
aataaggggc	actttacaag	ccgcataaga<br>
cacgacaagt	aaaaaacact	tgtccaatac<br>
cgtcaaaaag	ttctctgctt	acagcagtct<br>
gaagtcgctt	ttcatgtgat	cacccatctt<br>
agagtggtcg agcgacatgc gtctttacgg<br>
tggattgaac	aagccccacc	gcaacaacga<br>
gcatcgaccg	atgcactgct	ggcgccgagc<br>
ttgttccgcg	ccgaagtcgt	tgagcgcagc<br>
catcacatca	tcgccgacct	gtggtctgtc<br>
tgtatggacc	gctccagcat	caccctggcg<br>
gagttctggc	ggcaccaaat	gtcacaggac<br>
gaacagcaca	cggaccgccg	catggtgctg<br>
gctgacctgg	cccgcctggc	cacagcctgc<br>
gcacaagtat	tggcgctgtc	cagaatcggc<br>
ttccatggcc	gcaacagggg	caacaaggat<br>
gtgcctttcg	atgtcagcga	atgcagcgtg<br>
ctggatgagg	cctcaaaagc	cagcgtcggt<br>
acgccgctgg	gatgggctgc	gaccgccccg<br>
ccaggcatgc	caagaggatt	ggcggcggct<br>
gagatggcgc	tgaccgcgga	acaggcaccg<br>
ctgctgacgc	gccacgacgg	caagctgcat<br>
cccggttggc	tggcagaggc	gttagccaga<br>
cgtgatccac	aggccagact	gtcagccttg<br>
ccgagccaag	cgcggccggc	gcctgcgtca<br>
gtcgccatca	cgccggacaa	gcccgcgctg<br>
gaattggcca	gtcgagtcgc	caggctctcg<br>
gaacagaccc	tggcaatact	cctgcctcgc<br><br><br>
gtagctgcac	cttgctcaac	tgcatcttgt	2880<br>
cataaatttt	atgctgactc	cccaaacaag	2940<br>
caaggagggt	atacggtgca agcttcttta	3000<br>
atcgatccca	gccagccagg	catgttactt	3060<br>
tctacttcgc	agttggtatc	gcgtatcgag	3120<br>
cagcgctttg tcatgcgcaa tggcacttac	3180<br>
cgctactgcg	tggtacgcac	ctatgatgaa	3240<br>
cgcgagcaca	tcggggttga	gtctgagcgt	3300<br>
gacggacaac	gctacttggt	cttccgaatt	3360<br>
ggcctcctga	ttcgagactt	tgccgaagac	3420<br>
tcaagaccga	ttgccccgtt	gatcgaccct	3480<br>
actccgtttt	ccttgcccat	ggcctccctg	3540<br>
tcttcgttcg	ttattgatca	ggagagcagc	3600<br>
gcggtaaccc	cgtacaccgt	aatgctcgcc	3660<br>
cagagtggcc	gtctgtcact	tgcggtgacg	3720<br>
gcggtaggtt	acttcgccaa	tacgcttgcc	3780<br>
ggcgagtttg	tcaaacgcac	cgccaagcgc	3840<br>
gccggttatc	ccgaattggc	agagttcatg	3900<br>
accaatgcgg	tgatttacca	gcaggatatg	3960<br>
ctgctgggat	tgggcacggt	gcagttgggc	4020<br>
cccagcatcg gcccgtttgc	cactgcgctg	4080<br>
ggccgggtcg aggtcgatcc	tgcgcagcat	4140<br>
cagttcgctg	tgatcctgcg ggaaatggtg	4200<br>
ccagcgtgcc	tgttacacca	accaaaatac	4260<br>
gaaacattgg	tagccacctt	tctccggcaa	4320<br>
cgtacgccgc	aggccagcat	cagctatagc	4380<br>
gcagccttgc	gcgtacgcgg	cttcaaacct	4440<br>
gatatcaatc	tggtacccgc	tctgctggcg	4500<br><br><br><br><br>
atcatggcct gcggtggcag ttatgtgcca ctcagtgacg cgaaccccgc cgaactcaac	4560<br>
cgttcgattc tgaccagggc ccgttgccgc gcgattctca cggatcagga gggtttgacc	4520<br>
cgtttcgctc acttggcgcc ctgctggtcc ttgagcgacc tgctgtcgat gcccgacgcc	46 80<br>
ccgctgcagg accagtccaa gcttcaagcc aaggcctata tcctatttac cagcggctcc	4740<br>
accggtgaac caaaaggcgt ggcgatcacc catgctaatg ccgccaacct gctgcgttgg	4 800<br>
gcggctctcg attgtggccc cgagtacctg gcgcaaacac tggcggcaac ccccactacg	4860<br>
ttcgatcttt cgattttcga gatgtttgct ccccttatgg tcggtggctg cgtacagccc	4920<br>
gtttcctcgg tcatggcgct gatcgacaat ccggccctgc taaagggcac aacactgatc	4980<br>
aatacggtgc cgtcggtggc cgacgctttg ttgcagcatg atgtactggt gccttccttg	504 0<br>
cgcatgctca acctcgcggg agaacccctg aaccgggatc tttacctgcg gcttcaggca	5100<br>
aaactgaccg ccacacgcat cgtcaacctc tacggcccga cggaaacaac aacctattcc	5160<br>
accgccctgg tgatcgagcc cgcacaacaa gagatcacca tcggttttcc actgtatggc	522 0<br>
acctgggtgg atgtcgttga tcaaaacatg caaagcgtcg gtatcggtgt acctggcgag	5280<br>
ttgatcattc atggacacgg cgtggcgcaa ggctatgtca gcgaccccgt gcgtagcgcc	5340<br>
gcttctttcc tgccggcatc cgatggcttg cgttgctacc gcacgggaga ccgtgtccgc	5400<br>
tggttgcccg atggccgcct ggactttatc ggtcgagagg atgatcaggt caaagttcgc	5460<br>
ggtttccggg tcgagttggg gcctgttcag gcggcactgc atgccattga gacgattcat	5520<br>
gaatccgcag tagtcgttgt gccgaaaggg cagcagcgca gcatcgtggc gttcatcgtc	5580<br>
ctcaaagcgc cgagcgaaga tgaagcggtg cagcgcaata acatcaaaca acacttactc	5640<br>
ggcgtactcc cctattacgc actaccggac aagtttattt ttgttaaagc actgccaaga	5700<br>
aacacacatg gaaaaatcga cagaacgctg ctcttgcaac atgagcccca gactgagcaa	5760<br>
gaaagcgcca tgcgagatgc gaccgacgtc gaacatcgca tcgccaactg ctggcaaacc	5820<br>
atcatcggac accccgtcca actccacgaa aacttcctgg acattggcgg ccactcgctt	5880<br>
tcgcttacgc atttaacggg cctactgaga aaagaattta atattcatat ttctctacac	5940<br>
gacctctgga tcaggccaac catagaacaa caggccgact tcattcataa gttgcaaaat	6000<br>
tcggtattga caaaacctgc cgccgcgcca atcccgcgac ttgaccgaaa gatctctcat	6060<br>
cattaatcag gagtaccgca tgagcgtcga tacatgcagg actgcaactt tccctgcgtc	612 0<br>
atacggccag gaacagatct ggtttctgaa cgaactaaac ccgcactctc aactggctta	618 0<br>
taccctggcg atgaaagtat ctatcgccgg gaaattgaac acactgcggt tgcagcgtgc	6240<br><br><br>
ggtcaaccaa gtggtggcct cccaggaaat tttgagaaca tcattcgcct ataaaaacca	6300<br>
gaagttgagc caggtcattt caccctccgc gacactgccc attcgcagcg cgcactgcat	6360<br>
tgacgatgta cctgggctgc aacgcctgat caacatggaa gcccagcgtg gctggtcgct	642 0<br>
gagcagcgcg ccactgtacc gcttgctgct gataaaaacc ggcgaccagc aacatgagct	6480<br>
ggtcatctgc acccaccata tcgtctgcga tggcatctcg ctgcaactgc tgctgcaaaa	6540<br>
aatagtcagc gcctatcaag gccaaagcga tgggcgggtg ctcacaagtc cggatgaaga	6600<br>
gaccctgcaa ttcgtcgatt atgcggcctg gtcaaggcag cacgaatatg ccggtctcga	6660<br>
gtactggcgc cagcaactgg ccgacgcccc gacaatcctg gatatttcga caaaaaccgg	6720<br>
ccgaagtgag caacagacat ttctcggcgc gcgaattccc gtcgagttca gccaccacca	6780<br>
atggcaagca ttgcgccaga tattcagacc ccagggtatc tcctgcgcgg cggtgttcct	6840<br>
ggcggcctac tgcgtcgtcc tgcaccgcct ggccgagcag gacgacattc tgatcgggct	6900<br>
gccaacttca aatcgcctgc gtccggagtt ggcacaggtg atcggctacc tgtccaatct	6960<br>
gtgcgtgttt cgcagccagt atgctcacga ccagagcgtc acagactttc ttcaacaggt	7020<br>
tcaattgacc ttacccaact tgatcgagca cggggagacg cctttccagc aagtactgga	7 080<br>
aagtgttgag catacccggc aagccggtgt gacgccgttg tgccaagtac tgtttggtta	7140<br>
tgagcaggac gttcgacgca cgctggatat cggcgacctg caattgacgg tctcggatgt	7200<br>
ggacacgggg gccgcacgcc tggatctatc gctgttcttg ttcgaggacc acgaactcaa	7260<br>
cgtttgcggt tttctggaat atgccacgga ccgtatcgac gccgcatctg cgcaaaacat	7320<br>
ggtgcgcatg ctcagcagcg tgctacgcga gttcgttgcg gcgccgcagg cgccgctcag	73 80<br>
cgaagtacag ctgggggcgg cggattccca agcccagaca cctgcgatcg caccagcatt	744 0<br>
cccaagcgtg ccggctcgtc tgttcgcctt ggcagacagt caccccaatg cgaccgcgct	7500<br>
gcgtgacgag caaggtgaac tgacctatgc gcaagtttgc caacagattc tgcaggcagc	7560<br>
ggccactctg cgagcccagg gggcgaaacc tggaaccctg atcgcggtca tcggcgagcg	762 0<br>
cggtaacccc tggttgatcg ccatgttggc gatctggcaa gtcggcggta tctatgtgcc	7680<br>
attgtccaag gacctgcccg aacagcgcct gcaaggcatc ctggcggaac tcgaaggggc	7740<br>
catactgatt accgacgaca ccacgccgga acgcttccgg caacgtgtga cgctgcccat	7800<br>
gcacgcctta tgggccgatg gcgcaacgca fccacgagcgg cagacgacgg acgccagccg	7860<br>
gctgtctggc tacatgatgt acacctcggg atcgaccggt aaaccgaaag gcgtgcatgt	7920<br><br>
cagccaggcc aacctggtcg cgaccctgag cgcattcggc cagctgctgc aggtgaaacc	7980<br>
cagcgatcgg atgctcgcac tgacgacctt ctccttcgac atttcgctgc tcgagctgct	8040<br>
gcttcccctg gtccagggcg ccagcgtgca aatcgctgtc gcacaggctc aacgcgatgc	8100<br>
ggaaaagctc gcgggctatc tcgcagaccc tcggatcacg cttgttcagg ccacaccggt	8160<br>
gacctggaga ctattactgt cgacaggctg gcagccacgg gaaagcctga ccctgctgtg	8220<br>
cggtggcgaa gcgctgccac aggatctggc ggacaggttg tgcttgccgg gcatgacctt	8280<br>
gtggaacctc tacggcccca ccgaaacaac aatctggtcc acggcctgcc gcctgcaacc	8340<br>
gggtgcgccg gtgcaactgg gccatcccat tgcaggtacg caaatagccc tggtggatcg	84 00<br>
gaacctgcgc agcgtgccca gaggggttat cggtgaactg ctgatttgcg gccccggcgt	8460<br>
cagccagggc tactatcgca acccggttga aacagccaag cggttcgtac cggacccgca	8520<br>
tggttcaggt aagcgcgcct atctgaccgg cgaccggatg cgcatgcagc aggatggttc	85 80<br>
gctggcctat atcggccgac gtgacgacca gatcaagctg cgcggccacc gtatcgagct	8640<br>
gggagagatc gagacagcgt tgcgaaaact gcccggcgta cgggatgctg ccgcccaact	8700<br>
ccatgaccag gacccaagtc gaggcataca ggcctttgtc cagctttgcg caacggtcga	8760<br>
tgagagcctc atcgatatag gccagtggct ggaaacactg cgccaaacgc tgcctgaggc	8 820<br>
gtggctgcct actgagtatt acaggatcga tggcatccct cttacctaca acggcaaacg	8 880<br>
cgacaggaag cgcctcctgc accaggccgt caggctgcaa acactcagtc tgagggtggc	8940<br>
tcccagcagt gacaccgaga cccgggtgca gcagatctgg tgcgagctgc tcggtctcga	9000<br>
ggatatcggc gttacggatg attttttcca gttaggcggc cactccattc tggtggcgcg	9060<br>
catggtcgag cgcatcgaaa ccgcgtttgg acggcgcgta cctatcgcag atatctattt	912 0<br>
ttctccgacg atcgcccgtg tggcggcgac gctggactcc atgacatttg aacaaggact	9180<br>
ggccgcacac agcgtgaaag gcgattggga gttcaccgcc atcagccttc aacacaacgc	924 0<br>
cgacagcaca gccgccgctc aggagagatg aatcatgcac agccccacta tcgatacttt	9300<br>
cgaggccgca ctgcgctcat tgcccgctgc ccgcgacgca cttggtgcct atcccttgtc	93 60<br>
cagcgaacaa aagcgcctct ggttactggc ccaactggcg ggcacggcaa cgttgccggt	9420<br>
aacggtgcgt tatgcattca ccggcacggt ggaccttgct gtcgtgcagc agaacctgag	948 0<br>
cgcgtggatc gcacacagcg agtccttacg cagccttttc gtcgaagtac tggaacgccc	954 0<br>
cgtcaggctt ctgatgccta cgggcctggt gaaactggag tacttcgatc gcccgccatc	9600<br>
cgatgccgat atggccgagc tcataggcgc cgcctttgaa ctcgacaaag ggccgttgct	9660<br><br>
gcgtgcgttc atcactcgaa ccgctgcaca acagcatgaa ttgcatctgg tcggccatcc $720 tattgtcgtg gacgaacctt ccctgcagcg cattgcccaa accctcttcc agaccgaacc 978 0 cgatcatcag taccccgccg tcggtgcgat cgccgaggta ttccagcgcg aacagacact 9840 ggcacaggat gcgcaaatca ccgaacaatg gcagcaatgg ggaataggcc ttcaggcgcc 9900 tgcggcaacc gaaattccga ccgaaaaccc ccgccccgct atcaagggct cagatcgtca  9960<br>
agtacatgaa gcccttactg catggggcga ccaacccgta gcagaggccg aaattgtcag	10020<br>
cagttggctg accgtgctga tgcgctggca gggatcgcaa* tcggcgcttt gcgcaatcaa	10080<br>
ggtgcgcgac aaggcgcatg ccaacttgat cggcccactg caaacctacc tgccggtccg	10140<br>
cgttgatatg ccggatggca gcaccctggc acaactgcga ctccaggtgg aggaacagct	102 00<br>
caatggcaac gaccatccgt ccttttccac gctgctggaa gtttgcccac caaagcggga	10260<br>
cctgagtcgc accccctact tccaaaccgg cctgcagttc attgcgcacg atgttgaaca	10320<br>
gcgcgacttc catgccggca acttgacacg cctgccaacg aagcagccaa gcagcgacct	103 80<br>
tgacctgttc atttcctgct gggtaagcga cggcacgctt ggcctgacgc tggattatga	10440<br>
ttgcgccgtg ctgaattcga gccaggtcga ggttctggcc caggcgctca tcagcgtatt	10500<br>
gtcagcgccc ggtgaacagc caatcgcaac cgttgcgctg atgggccagc aaatgcagca	10560<br>
aaccgtcctg gctcaggccc acggcccccg cacgacgccg ccgcaactga cactgaccga	10620<br>
atgggtcgcc gccagcacgg aaaaatcccc gctggcggtt gcggtgatcg accacggcca	10680<br>
gcagctcagc tatgcagagt tatgggcaag agctgcactg gtagcggcga acatcagcca	1074 0<br>
gcatgtggca aagcctcgga gcatcatcgc tgtagcactg cccagatcgg ctgaatttat	10800<br>
tgcagcgctg ctgggggtag tgcgagcagg tcatgcgttc ttgcccatcg atccccgcct	10860<br>
gcccaccgac cgcatccagt tcctgattga aaacagtggc tgtgagttgg tcattacctc	10920<br>
tgatcagcaa tccgtggagg gttggccgca ggtcgccagg atacgaatgg aggcgcttga	10980<br>
tccagacatt cgctgggtgg cgccgacggg gctcagccac agcgatgccg cctacctgat	11040<br>
ctatacctcc ggcagcaccg gcgttccgaa gggagtcgtt gtcgagcacc ggcaagtagt	11100<br>
gaataacatc ttgtggcggc aacgaacctg gccgctgacg gcacaggaca acgtgctgca	11160<br>
taaccattcg ttcagcttcg atcccagcgt ctgggcgttg ttctggccgc tgctgaccgg	11220<br>
tggcaccata gtgctggcgg atgtcagaac catggaggac agcaccgccc tcctcgacct	11280<br>
gatgatccgc catgatgtca gcgttctggg tggcgtaccg agccttctcg gtacgctgat	11340<br><br><br>
gctggtggcg	ctacggggtt	tcttgcccat	gggcatcgtg	ccggattacc	cacgcatctt	13140<br>
cgacatcgtg	cccgtggact	aygtcgccgc	ggcgatcgtg	cacatatcga	tgcaaccgca	13200<br>
gggcagggac	aaattcttcc	acctgttcaa	cccggcgccg	gtcaccatcc	gccagttctg	13260<br>
cgactggatt	cgcgaattcg gttacgagtt	caagttggtc	gacttcgaac	acggtcggca	13320<br>
gcaggcattg	agcgtaccgc	ccgggcacct	gctgtacccg	ttggtccccc	tgatcaggga	13380<br>
tgccgatccg	ctgccccacc	gcgcgctgga	ccctgactac	atccatgaag	tgaaccccgc	13440<br>
actggaatgc	aagcaaacct	tagagctgct	ggcctcctcg gacatcaccc	tgtcgaaaac	13500<br>
cacaaaggct	tacgcgcaca	caattttgcg	ctacctgata	gacaccggct	tcatggccaa	13560<br>
gcctggcgtg	tagcggattg	agcacaaaca	ggacgaatat	catggaatcg	atagcctttc	13620<br>
ccattgcaca	taagcccttc	atcctgggct	gtccggaaaa	cctgccggcc	accgagcggg	13680<br>
cgcttgcccc	ttctgcggcg	atggcgcggc	aggttttgga	gtacctcgaa	gcgtgccccc	13740<br>
aggcgaaaaa	cctcgagcag tacctcggga	cgctgcgtga	agtcctggcg	cacctgcctt	13800<br>
gtgcttccac	cggactgatg	accgatgatc	cacgggaaaa	ccaggaaaac	cgcgacaacg	13860<br>
atttcgcctt	cggtattgaa	cgacaccagg	gcgacactgt	gaccctgatg gtcaaggcca	13920<br>
cccttgatgc	agcgattcaa	acgggcgagt	tggtccaacg	cagcggcact	agcctggatc	13980<br>
actcggagtg gagcgacatg	atgtcagtcg	cacaggtgat	tctgcagacg	attgccgacc	14040<br>
ctcgggttat	gcccgaatcc	cgtttgacgt	tccaggcacc	gaaaagcaag	gtcgaagaag	14100<br>
atgaccagga	cccgctgcga	cgctgggtgc	gtggccacct	gctgttcatg gtcctgtgcc	14160<br>
aaggcatgag	cctgtgtacc	aacctcctga	tcagcgcggc	ccacgacaag	gacctcgaac	14220<br>
tggcgtgtgc	acaggccaat	cgcctgattc	aactgatgaa	catctcgcgc	atcacgcttg	14280<br>
agtttgcaac	cgacctgaac	tcacaacagt	acgtcagcca	gattcgcccg	acgctcatgc	14340<br>
cgscgatcgc	gccgcccaag	atgagtggca	tcaactggcg tgaccatgtg gtgatgattc	14400<br>
gttggatgcg	ccagtccacc	gatgcctgga	acttcattga gcaggcctac	cctcaactgg	14460<br>
ctgaacgtat	gcgaaccaca	ttggcgcagg	tctacagcgc	tcatcggggg gtctgcgaaa	14520<br>
agttcgtagg	cgaagaaaac	accagtttgt	tggccaagga	aaacgccact	aatacggccg	14580<br>
gccaggtgtt	ggaaaacctg	aagaaatcga gattgaaata	cctc'aagaca	aaaggttgcg	14640<br>
ccggtgcggg	ataagccctg	actgcgcctc	gcccccatca	aaaccggact	gatattcggg	14700<br>
aaaacaaagg	agagaagcat	gccgacattt	ctgggagacg	acgacgcagt	gccatgcgtg	14760<br><br>
 gtcgtcgtta acgccgacaa acactattcg atttggccaa gcgcgagaga cattccatca	                                   14 820<br>
ggttggtccg aagaaggatt caaggggtca cgttcagact gcttggaaca tatcgcgcaa	14 8 80<br>
atctggccag agccgacggc atagatacaa cgtgatgcaa aaaatgcggg aaacatcaac	1494 0<br>
taaccaaagc aaggaagaaa aatgacttca actcatcgca ccactgatca agtcaagcct	15000<br>
gctgttctgg atatgccagg cctgtcgggc attcttttcg gccacgccgc attccaatac	15060<br>
ctgcgggcca gctgcgaatt ggatctgttc gagcatgtcc gcgacctgcg cgaagccacc	15120<br>
aaggagagca tcagcagccg actgaagttg caggaacgcg ccgccgatat tctgctgctg	151B0<br>
ggcgcgacct ccctgggcat gctggtcaag gaaaacggca tctaccgcaa tgccgatgtg	15240<br>
gttgaggatt tgatggccac ggacgactgg caacgtttca aggataccgt ggcctttgaa	15300<br>
aactatatcg tctatgaagg gcagctggac tttaccgagt ccctgcagaa aaacactaac	15360<br>
gtcggccttc agcgtttccc gggcgaaggg cgggacctct atcaccgcct gcaccagaat	15420<br>
cctaagctgg aaaacgtgtt ctaccgctac atgcgctcgt ggtctgaact ggccaaccag	15480<br>
gacctggtca agcacctcga cctgtcgcgc gtgaaaaaat tgctcgacgc gggtggcggt	15540<br>
gatgcggtca acgccatcgc cctggccaaa cacaatgagc aactgaacgt aacggtactg	15600<br>
gatatcgaca actccattcc ggtcactcag ggcaaaatca atgattccgg gctcagccac	15660<br>
cgggtgaaag cccaggcatt ggatatcctg caccaatcct tccctgaagg ttacgactgc	15720<br>
attctcttcg cccaccaatt ggtgatatgg accctcgaag aaaacaccca catgctgcgc	15780<br>
aaggcctacg atgcgctacc agaaggcgga cgcgtggtca tcttcaactc catgtccaac	15840<br>
gatgaaggcg acggcccggt catggccgca ctggacagcg tctactttgc ctgtctaccc	15900<br>
gccgagggcg gcatgatcta ttcctggaaa cagtatgagg tctgcctggc ggaagccggc	1596 0<br>
ttcaaaaacc ccgtacgcac cgcgattcca ggctggaccc cacacggcat catcgtggcc	16020<br>
tacaagtaat tttgcctcct ccgcccctac tggggccgga ggagtcattt caacatttgc	16080<br>
gtcattgacg ccacctggcg atagggacac ccacatggca cgttcacccg agacaaatag	16140<br>
tgcgatgccg caacagataa gacagctttt atacagccaa ctgatttcgc aatcgattca	16200<br>
aaccttctgt gaactgcgcc tgcctgatgt tctgcaagca gctggccagc ctacctccat	16260<br>
cgaacggctt gctgagcaga cacacactca tatcagcgcc ctgtcacgct tgttgaaagc	1632 0<br>
gttgaaacca ttcgggctag tgaaagaaac cgacgaaggt ttttccttga ccgatctcgg	163 80<br>
cgccagtctg acccacgacg cctttgcttc cgctcaaccc agtgctttgt tgatcaatgg	16440<br>
tgaaatgggc caagcctggc gtggcatggc gcagacaatc cgaaccggtg aatccagctt	16500<br><br><br>
caagatgtac tatggcatca gcctgttcga gtattttgaa cagcacccgg aacgccgggc	16560<br>
catttttgac cgttcccaag acatgggact ggacctggag atcccggaaa tcctggagaa	1662 0<br>
catcaacctg aatgacggtg agaacattgt cgatgtaggg ggtggttcag ggcatttgct	16680<br>
gatgcacatg ctggacaagt ggccagaaag cacaggcata cttttcgact tacccgtcgc	1674 0<br>
ggcaaaaatc gcgcagcaac atctgcacaa atctggaaaa gcaggctgct ttgaaatcgt	16800<br>
cgcaggggat tttttcaaga gcctgccgga cagtggcagc gtttacttgc tgtcccatgt	16860<br>
cttgcacgac tggggcgacg aagactgcaa ggccattttg gccacctgcc ggcggagcat	16920<br>
gccggacaat gcgctgttgg ttgtagtgga cttggtgatt gaccagagtg aaagtgccca	16980<br>
gcccaacccc acgggcgcaa tgatggatct ttacatgctg tccttgttcg gtatcgccgg	1704 0<br>
aggcaaagag cgcaacgagg atgaattcag aaccctcatt gaaaacagcg gcttcaacgt	17100<br>
caaacaggtg aagcgcctgc caagtggaaa cggcatcatc ttcgcctacc caaaataaat	17160<br>
gatcctcatt gcccctcgcc actttccagg ggggctattt tattctcggg tgattccccc	17220<br>
cctaatgatt acaaggaaga cacatgtcga cgctggttta ctacgtagca gcaaccctgg	17280<br>
atggttatat cgccactcaa caacacaaac tggattggct ggagaacttt gccctggggg	17340<br>
atgacgcaac ggcctatgay gatttttatc agacgatcgg agcagtggtc atgggatcgc	17400<br>
agacctatga atggatcatg tcgaacgctc ccgatgactg gccctaccag gacgtacccg	17460<br>
cctttgtcat gagcaaccgg gatctgtcag cccccgccaa tttggatatc accttcttac	1752 0<br>
gcggcgatgc cagtgccatc gcggtcaggg ccaggcaagc ggcgaagggc aagaatgtct	17580<br>
ggctggtcgg tggcggcaaa acggcggcct gttttgccaa cgcaggggaa ttacagcagc	17640<br>
tgttcatcac cactattcca acctttatcg gcaccggcgt tccggtactg cccgtagacc	17700<br>
gcgcgcttga agtggttctc agagaacaac gcacgctgca gagcggtgcc atggaatgca	17760<br>
tcctggacgt gaaaaaagcg gattaacgtc tacaagacaa tcgtgtatcg aaactcgcaa	17820<br>
cgtccaaacc caagggaaaa accagtgaag cgattggtat tgagtttatg tttgttggct	17880<br>
gttatcgctc tcgccagtgt tcaaggaata aggatggtga aacccgccgc cctgacagcc	17940<br>
gccgatgctc gcgatatcgg ctatctgaat gtacgcgata gcctttccgt cattgccgcc	18000<br>
gccccccacc ccaccgcctc acctcgccag gccgttgtca ggcattattt gcgggaaacg	18 060<br>
attgcgggca tgggttacca ggtggttgag caaccctttc tatttaccat cgagagcatg	1812 0<br>
gtgaaccggc agaaaaccct ctatgccgag ttgaacgagc agcagcgcca agcgttcgat	18180<br><br>
gctgagctgg cccgggtggg cgcggacagt tttgaaaaag aagtgcggat tcgctccggc	18240<br>
ctactggaag gcgacagcgg ccagggaacc aacttgatag cctcacaccg cgtaccggga	18300<br>
gcgaccgcga cggtcctgtt catggcgcat tacgacagcg tcggcaccgc tcccggtgcc	18360<br>
agtgacgatg gcatggccgt cgcctcgata ctccaactga tgcgggaaac cataacccgc	18420<br>
agcgatgcca aaaataacgt tgtctttcta ctcrccgatg gcgaagaact gggcttgctc	184 80<br>
ggagcggagc actacgtctc gcagctcagt acgcctgaac gtgaagccat ccgcctggtg	18540<br>
ttgaactttg aagcccgggg taaccagggc atccctttac tgttsgagac atcccagaag	18600<br>
gactacgccc tgatcaggac tgttaacgca ggggttcggg acatcatatc cttctcattc	18660<br>
acgcccttga tttacaatat gctacaaaac gacaccgact ttacggtgtt caggaaaaag	18720<br>
aacatsgcgg ggttgaattt tgcagtcgtg gagggttttc agcactacca ccacatgags	18780<br>
gacaccgtgg agaaccttgs gccagagacc ttgtttcgct accaaaagac agtgcgtgaa	18 840<br>
gtgggcaacc actttatcca gggtatcgac ctctcctccc tgagtgctga tgaggacgca	18900<br>
acctatttcc cactgccagg cggcacgctg ttggtactca acttacccac cctgtatgcg	18960<br>
ctgggcatgg gctcgttcgt gctctgcggt ctttgggcgc aacgctgccg cactcgccga	19020<br>
cagcatcagg gcaagaattg cgtactgcgc cccatggcta ttgccctgct cggcattgcc	19080<br>
tgcgcagcac ttgtattcta cgtcccgagc attgcctatc tattcgtcat ccccagtctg	1914 0<br>
cttctggctt gcgccatgtt gtcgcgaagc ctctttatct cctattcgat catgctgctg	192 00<br>
ggcgcttatg cctgcgggat actctacgcg cctatcgtct acctgatttc atcaggcctt	192 60<br>
aaaatgccgt tcattgccgg ggtcattgca ctactcccgc tctgcctgct ggccgtggga	1932 0<br>
ctggccggcg tcatcgcacg atcgagagac tgtcgaacct gcgactagca agacccgata	193 80<br>
aaacgtcgct tcaaacgcca gatgacgtgc ctcgtcagcc aggcgtggaa ccatctggtg	1944 0<br>
gcggcaaatg tgcataaggt gggaacgcag agcgcccgct gcaacacgcc caccccaagc	19500<br>
accgcgcctc aacggataat caggctcaag ggaattccac cttgcaacct gaaagagcaa	19560<br>
tcgagcgccc gtcggacaca acaaactgat caccgtcaat tcgggcaagg agcaatccac	19620<br>
gggcttttgc tccaacctca actccctttg aaaaatcagc cggccacaat ttgcccctac	196 8 0<br>
cctttcagga tatcctcgat aagcgtttta tcagaacagc gaaaaaccac ttcaagttcg	19740<br>
tgtacttttt actgcgatct gcgatcgctc ccatggtaca araatgacag atgggaagat	19800<br>
cgctttaata cctactctct cacctgagaa aaagtaacca ccgggccgta ttcctgatca	19860<br>
gacactatcg cctgcacaca aaatttcttc tctggaaact tactcagcaa caccatcctc	19920<br><br><br>
catgactgag	caataaggtt	gcacagttgt<br>
cagcctgcaa	aaacatcata	caggtgcaca<br>
tcacgattac	aagactgcat	ccaactggag<br>
aacacgcccg	actcgtaaac	aacaaagtct<br>
tcttcaggat	gtacaacatc	actaataaaa<br>
cattgggact	tccagcgctc	gtatttggga<br>
ttcatagcac	tgttcctgca	ctgatgcctt<br>
tcgatggctc	attgttcatc	aagaaaaaaa<br>
cggcgcgttg	agaacagtga	aaagtcactc<br>
acaagatacc	cagtattcag	cttttcaatg<br>
tcgacctctt	cagcaaactc	atcgccgtag<br>
ggtttcaaag	ccgaattgaa	tgagccgatc<br>
agttcgggtt	cactgaacag	gtgcagcatt<br>
gccttgaaga	gcgaatacca	atcgacgctc<br>
ccaggaacca	cctgcccccg gacattgcgc<br>
accgcctttt	ttctgcgcca	ggcaaagtcc<br>
cgtatggccg	cctttgatgc	ccagcacgct<br>
ccgaatttat	ccgcattgtg cgacacctgc<br>
aatgacgaaa	tgaacgcctc	tccaacaccc<br>
gtaatttgca	catagggctt	cataaatcgt<br>
gtatatccaa	agagctctcc	cacgttactc<br>
cgcctcacgc	gccaccgccc	acggcgccat<br>
cttgagcacg	aaattgcggc	atttctcggc<br>
ctgcatctgc	gccagctcgg	cttcacggca<br>
tttacgggcg	cgagccacgg	cgtacttttc<br>
cagccaacgg	ctgaaggcgt	gcgggcagcg<br>
ctcggcttgc	aaggcactga	tcggcaggca<br>
gccaacggcg	cgtggcaggc	tgtaggtcca<br><br><br>
tcataaacag	cttcatcgac	tctatcgctg	19980<br>
tgattcaaaa	ccttctcaac	cgccaccaca	20040<br>
aatgcttcaa	cagtaaatcc	gctttccaga	20100<br>
ggaaacaaca	cggtggaaaa	caccaggaca	20160<br>
caagactcaa	ccaactcgga	ttgatccttc	20220<br>
gacctgacag	ggtctgtcgt	cctgattatt	20280<br>
ttggcgattt	tttgtttgag	cgagaatcga	20340<br>
ttctctcgaa	acagactctt	aagctcggcc	20400<br>
gccacatcct	gaatattctc	tgtgaccctc	20460<br>
ggaaagcctg	acgcagcaat	ccgctcaaca	20520<br>
aacatagccc	aaccgatatt	gggcacgtcc	20580<br>
tgaaaacttt	tatatttctc	atgcgggcta	20640<br>
ccgagttgag	gaggaaaaat	ttcacaccat	20700<br>
ttgctactga	ccgagcggta	tgtgattgcg	20760<br>
gccgtgtgga	tgacgctccc	acagccttta	20820<br>
aggtagaagt	ccgataatgc	gccgttaaac	20880<br>
tcactcgcgg	ccacgcccat	gaaaggctct	20940<br>
tctggaacca	gtaactcccc	ttcccgactt	21000<br>
caaccgatag	tttcagactt	cgtcctgatt	21060<br>
caaagtctcg tcaattcacg ggtgacacaa	21120<br>
atcgcatcga gtctatcaac	caaccaacgc	21180<br>
caaccgctgc	ggggtctggc	aggttttgcg	21240<br>
gaattggtgg cggttgtgca	ccatgtccaa	21300<br>
gcgtgtaatc	tggtcgatgt	ccaacgccgc	21360<br>
atcggttaac	gcgctgcttg	cctgctgcat	21420<br>
cgggccgatg	ccctgaacca	gcccgtattg	21480<br>
ggcgtcggtg	agttgatggg	cgacttcact	21540<br>
gtattcggag	ccgtacaggc	ccatggtttt	21600<br><br>
gtaatgcggg	ttgagtacca	cgctctcgcg	ggccaatacg	atgtctgcgg	ccagcgccag	21660<br>
cattacacca	ccggcgccgg	cgctgccggt	caggccgctg	atcaccagtt	gccgggccgt	21720<br>
gagcagttcg	tggcacacat	cgtacgatgg	cctgaatgtt	ggcccaggct	tccagccccg	21780<br>
gcactggggc	ggcctggatg	acgttgaggt	gcacaccatt	ggaaaagctg	ccgcgcccgc	21840<br>
ccttgatcac	cagcacttgg gtgtcccgcg	tcttggccca	gcgcaacgcc	gccaccagtc	21900<br>
gctggcactg	ctcggtgctc	atggcgccgt	tgtagaactc	aaaggtgagt	tcaccgacat	21960<br>
ggccggcttc	gcgatagcga	atcggttgat	aggcttgctc	atcgaacatt	tgattggcga	22020<br>
tcgagctgtc	cagcacggga	atatccgcca	gtgcttccgc	cagcacgtgg	cgggccggca	22080<br>
gcttgaaggt	ctcctccccc	ggccgggctt	tgcgtttgag	cgagccgatc	cacaggctct	22140<br>
gatcaccggc	cgccaccagc	accgcgtcgt	cctgcaccgc	gaggatctca	cccggtgtgc	22200<br>
cgtggcgcgc	atccaggtgc gcgtcgtaca	ggtaatactg	cccgccctgg	atactggcca	22260<br>
gcacaccggg	ctggccatcg gctgcgtcga tgcagcgttt gatgaagcgt	gcgcaatcgt	22320<br>
accaactgaa	ggtgcgatca	gcctgtgtca	tgttcggctg	caaacgcccg	attacgtggg	22380<br>
cttgggtgta	atcgagcggc	accgggacga	aaacccgggc	gaacttttcc	accacgtcgc	22440<br>
ggatgcaata	gagggcggcg	tcactcaccg	cgccgttgta	cagctcggat	ttgcgcacat	22500<br>
cggcaggcat	gtcgaattca	caggtcgacc	agatcggccc	ggcgtccatt	tcctccaccg	22560<br>
cctgcaaagc	cgtgacgccc	cagcggccga	cctgctggct	gatggcccag	tccagcgcgc	22620<br>
tggcaccacg	gtcgccgacg	atgcccggat	ggataatcac	cacagggcgc	tcaaggttgc	22680<br>
tccaaagttg	ctgtggcaca	cggtctttca	gaaaggggca	gatcaccagg	tcggcgtctg	22740<br>
aatcctcgat	ctgctggcac	accaaggctg	gatcggtgaa	cagaacaacg	ctgggcgcgt	22800<br>
gccccgactg	gcgtaaatcc	agccaggccc	gctgggtcaa	accgttgaac	gccgacgcta	22860<br>
acacgatgat	cttcaatgac	cgcatggctg	actcatcctt	gagaatgcgc	ggccagaggt	22920<br>
gctccttgag	ccctccctgg	cctttgatgg aagtacaagg	atagttggcg	tgccaggcag	22980<br>
gctacctgat	caggatcaat	cttgtgtcag	cgagtgcttg	aacgtaggcg	cctgcgttca	23040<br>
accaataggc	gcatggsctg gcgagtgctc	ccgcgtgccc	tctgccacaa	gggacgccag	23100<br>
gtattcgcca	aacccgccgc	gacacttgta	gtcccgacgc gcactggtga	ccacgggatt	23160<br>
ggtcgccgtc	cacacgatcc	gcgcgccaat	cctctcgagg	tcggccacca	actgcacatc	23220<br>
ttcatgggca accaaatgct ggaacccacc cgcgtttcga taggcatccg	cactcaagcc	23280<br>
caggttggca	ccgtgtatat	ggcggtggtt	ctcggtgaac	tgatacaact	caaggtagcg	23340<br><br>
cgaacgaacc	gattcaccgt	actcgctcca	gctgtccacc	tcgacggttc	cgcacaccgc	23400<br>
atcggcgcca	aagccgatct	gacgcaccag	ccagtcgrcg	ggcacaactg	tgtcagcgtc	23460<br>
ggtgaatgcc	agccactggg	cgccgacttc	aagcaatcgc	tccgcgccca	aggccctggc	23520<br>
cttgcccaca tttcgaacgc	tcacctcaag	cgtkgcgaca	cccatggccg	acacgcgcgt	23580<br>
ggcggtctcg	tccgaacacg	catccagcac	caccagcaat	tggacctgtt	ggtgtgccag	23640<br>
agccggatga	gcaatggcgc	gctggatgga	ggcgaggcag	gcactgatgt	gccgttcttc	23700<br>
gttatgggca	ggtatcacta	tccctatcat	tgacgttccc	tctaccaggc	aaagtgtcta	23760<br>
cagctatcga	ccgggccgtg	aggcagaagg	tttaaacaat	ctgaaggcgc	cgccaaacaa	23820<br>
tgacgtgaga	caggtcgcag	tgattaaacg gaacgtcaca	ggcgccacag	gctcagatgg	23880<br>
tttacgtgtt	tgatgcacgg	atgaacccgc	cattcctaca	aacaggtcag	ccatcatgtc	23940<br>
taacgattat	caaggtatcg	ccagtgtcat	cacggcttct	cgtcacatgg	gtacagactc	24000<br>
ggatgaacgc	cttaatgaga	cggtaaatat	tcaattgacc	tgcagcggta	aaccaacgat	24060<br>
tgcgcggttg	agtttcgaca	ccccgcttca	atggcccggc	caccccaact	ttgtgctgat	24120<br>
caacctgccg gacggttcat	cggtgggtgg tgtgattgcc	gaaattgaaa	agtcgaccga	24180<br>
tggcccgggt	tgggtgacgt	ttacggtgga	tgactgaggt	cttcccaaca	ggcttcaaat	24240<br>
cacctccagg	cggctgcctc	gaatgagaca	cacaggccag	taatcgagac	gcacagacaa	24300<br>
gcctattttc	gcagatacat	tttgtaacgt	cctatgattg	acgcttgctc	gaatcaccgc	24360<br>
agggattggg	tggcgtgtgt	ttatcacgcc	cttgaatccg	cagcgaaaaa	tgattcgagt	24420<br>
tcagcgaaca	attcgattgg	gacaaacaaa	aggatgcggg	ctatgtcatt	gcgtaattta	244B0<br>
tctttattgg	tcaccacact	ggcgctgttt	aagtggggtg	taatgcgctc	gcggggcaaa	24540<br>
acccaacatg	ctcagtgatg	atgacgtgaa	atcccaaagc	gccggcgcac	tgggctatgc	24600<br>
cccgacagac	ctgagcatcg tcaaccgtcg	aaccgaaggc	accaacacct	acgtgctgct	24660<br>
taaaaccaac	gacaacaagc	agttcaactg	cattatcaac	ggaggcaata	tcctgacctt	24720<br>
cggtatgtcc	aacccgcctt	cgtgtgcgaa gaaaggtgaa	cagatcaaga gtggcccgtt	24780<br>
cgggagctga	tctgtcgctg	gaaaaaaggg	ccaggccacc	tctaagaacg	gaggcctggc	24840<br>
ccttttttat	tcgctcagat	gagtttaaaa	gacaagatat	cgggcagctg	ggctccggcc	24900<br>
cgttcagtct	gggcacccca	cacaaaatgc	tcagcgacta	cttggccgtc	gccgcacacc	24960<br>
gtttaacggg	tgcgacctac	agcgtcrccc	tggttgaagg	cagcaacgaa	taaaccctat	25020<br><br>
  tgatcggaga gcgaccatgc acccgcataa aaccgcgatt gtcttgattg aataccagaa	25080<br>
cgacttcacc acccccggcg gcgtgttcca tgacgctgtg aaagacgtca tgcaaacgtc	2514 0<br>
caacatgctg gcgaataccg ccaccacgat tgagcaggcc cgcaagctgg gcgtgaagat	252 00<br>
catccactta cccatccgct ttgccgacgg ctacccagag ctgaccctgc gctcatacgg	25260<br>
cattctcaaa ggcgtcgccg acggcagcgc gtttcgtgcc ggcagctggg gcgccgagat	2532 0<br>
caccgacgcg ctgaaacgcg accccaccga tattgtgatc gaaggcaaac gcggcctgga	25380<br>
tgctttcgcc accaccgggc tggacctggt gctgcgcaac aatggcatcc agaacctggt	25440<br>
tgtcgcaggt ttcctgacta actgctgcgt tgaaggcacg gttcgatccg gttacgagaa	25500<br>
aggttatgac gtggtgacct tgaccgactg caccgcgaca ttcagtgatg aacaacagcg	25560<br>
cgcagccgag cagtttacgt tgccgatgtt tttcgcaaac cctgcaacac accgcgtttc	2562 0<br>
tgcaagcact gaacgccgga taaaaaaagc ggcggactcc tgccgagtcg ccgctttttt	25680<br>
agtgcttggg tcattcggtt ggcgcgtact gcatttcgcc gttcccaaac gaccagtctt	25740<br>
cgcgcttcac gtccaccagg ctgatccaca cgtcttcctt gcgcagcccg gtcttggcat	25800<br>
ggatgccgtc ggcgatgaac ttatagaaag cctttttcac gtcaatgctg cgcccggcgt	25860<br>
tccacgtgac ttggataaac acgatcttgg gtgtgtaagt gacgccaaga tacccggccg	2592 0<br>
ccgggtaaac cagctcatcc ttggcatggc ggttgatgat ctggaatttg tcgtgctcag	25980<br>
gcacgttggc ccacactggt catcgcggcg tacacgacat caccgatggc cgtcgcggtt	26040<br>
tcagtggaag tgtcggcggc gaggtcgatt cgaactaaag gcatggacaa atccttagtg	26100<br>
attttcagct gaaaatgggc gtgtggctca cacactcgcg ccaaccgggc aacttgcgcc	26160<br>
aggccaacga gttgctggcc cagggagttg ccgacggttt gcgctagtgc gccgcgaaac	26220<br>
ttcggcattt gacgcatcgg tgaatggctg accggatgtc agtgcttatt gacctgaata	262 80<br>
tagactgccg tgcacagacc aatcaaacaa ataccggcga tgtagtaagc ggcgcccatc	26340<br>
tgactgtatt gaagcaatag agtaacgacc atcggcgtca ggccaccgaa tacggcgtac	26400<br>
gacaagttgt aggaaaatga caagccggaa aaccgcacta ccggtggaaa ggcacgcacc	26460<br>
atcacagcag gggctgcgcc tatcgcgccg acaaaaaaac cggtaagtga atagagtgga	26520<br>
accagccatt gcgggtgcgt ttcaagcgtc ttgaacaaga gcagtgcgct gaacagaagc	26580<br>
atgacgctgc cgatcatcaa tacccarccc gcactgaaat gatcggccag tttcccggcg	2664 0<br>
atcacgcaac caacactcaa rrcacrcaat agcgagactg ttggcctgca aggcttgcgc	26700<br>
tgcag	26705<br><br> SEQID 2<br> Lenght: 1004<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE 2<br>
Met Leu Leu Glu Val Ala Phe His Val He Thr His Leu Ser Thr Ser<br>
15	10	15<br>
Gln Leu Val Ser Arg He Glu Arg Val Val Glu Arg His Ala Ser Leu<br>
20	25	30<br>
Arg Gln Arg Phe Val Met Arg Asn Gly Thr Tyr Trp He Glu Gln Ala<br>
35	40	45<br>
Pro Pro Gln Gln Arg Arg Tyr Cys Val Val Arg Thr Tyr Asp Glu Ala<br>
50	55	60<br>
Ser Thr Asp Ala Leu Leu Ala Pro Ser Arg Glu His He Gly Val Glu<br>
65	70	75	80<br>
Ser Glu Arg Leu Phe Arg Ala Glu Val Val Glu Arg Ser Asp Gly Gln<br>
85	90	95<br>
Arg Tyr Leu Val Phe Arg He His His He He Ala Asp Leu Trp Ser<br>
100	105	110<br>
Val Gly Leu Leu He Arg Asp Phe Ala Glu Asp Cys Met Asp Arg Ser<br>
115	120	125<br>
Ser He Thr Leu Ala Ser Arg Pro He Ala Pro Leu He Asp Pro Glu<br>
130	135	140<br>
Phe Trp Arg His Gln Met Ser Gln Asp Thr Pro Phe Ser Leu Pro Met<br>
145	150	155	160<br>
Ala Ser Leu Glu Gln His Thr Asp Arg Arg Met Val Leu Ser Ser Phe<br>
165	170	175<br>
Val He Asp Gln Glu Ser Ser Ala Asp Leu Ala Arg Leu Ala Thr Ala<br>
180	185	190<br><br>
Cys Ala Val Thr 195<br><br>
Pro Tyr Thr Val 200<br><br>
Met Leu Ala Ala<br><br>
Gln Val Leu Ala 205<br><br><br><br>
Leu Ser Arg Ilee 210<br><br>
Gly Gln Ser Gly<br>
215<br><br>
Arg Leu Ser Leu 220<br><br>
Ala Val Thr Phe<br><br><br><br>
His Gly Arg Asn 225<br><br>
Arg Gly Asn Lys 230<br><br>
Asp Ala Val Gly 235<br><br>
Tyr Phe Ala Asn 240<br><br><br><br>
Thr Leu Ala Val<br><br>
Pro Phe Asp Val 245<br><br>
Ser Glu Cys Ser 250<br><br>
Val Gly Glu Phe 255<br><br><br><br>
Val Lys Arg Thr 260<br><br>
Ala Lys Arg Leu<br><br>
Asp Glu Ala Ser 265<br><br>
Lys Ala Ser Val 270<br><br><br><br>
Gly Ala Gly Tyr 275<br><br>
Pro Glu Leu Ala 280<br><br>
Glu Phe Met Thr<br><br>
Pro Leu Gly Trp 285<br><br><br><br>
Ala Ala Thr Ala 290<br><br>
Pro Thr Asn Ala 295<br><br>
Val He Tyr Gln 300<br><br>
Gln Asp Met Pro<br><br><br><br>
Gly Met Pro Arg 305<br><br>
Gly Leu Ala Ala 310<br><br>
Ala Leu Leu Gly 315<br><br>
Leu Gly Thr Val 320<br><br><br><br>
Gln Leu Gly Glu<br><br>
Met Ala Leu Thr 325<br><br>
Ala Glu Gln Ala 330<br><br>
Pro Pro Ser He 335<br><br><br><br>
Gly Pro Phe Ala 340<br><br>
Thr Ala Leu Leu<br><br>
Leu Thr Arg His 345<br><br>
Asp Gly Lys Leu 350<br><br><br><br>
His Gly Arg Val 355<br><br>
Glu Val Asp Pro 360<br><br>
Ala Gln His Pro<br><br>
Gly Trp Leu Ala 365<br><br><br><br>
Glu Ala Leu Ala 370<br><br>
Arg Gln Phe Ala 375<br><br>
Val He Leu Arg 380<br><br>
Glu Met Val Arg<br><br><br><br>
Asp Pro Gln Ala 385<br><br>
Arg Leu Ser Ala 390<br><br>
Leu Pro Ala Cys 395<br><br>
Leu Leu His Gln 400<br><br><br><br>
Pro Lys Tyr Pro<br><br>
Ser Gln Ala Arg 405<br><br>
Pro Ala Pro Ala 410<br><br>
Ser Glu Thr Leu 415<br><br><br>
  Thr Leu Ile Asn Thr Val Pro Ser Val Ala Asp Ala Leu Leu Gln His<br>
645	650	655<br>
Asp Val Leu Val Pro Ser Leu Arg Met Leu Asn Leu Ala Gly Glu Pro<br>
660	665	670<br>
Leu Asn Arg Asp Leu Tyr Leu Arg Leu Gln Ala Lys Leu Thr Ala Thr<br>
675	680	685<br>
Arg Ile Val Asn Leu Tyr Gly Pro Thr Glu Thr Thr Thr Tyr Ser Thr<br>
690	695	700<br>
Ala Leu Val He Glu Pro Ala Gln Gln Glu Ile Thr Ile Gly Phe Pro<br>
705	710	715	720<br>
Leu Tyr Gly Thr Trp Val Asp Val Val Asp Gln Asn Met Gln Ser Val<br>
725	730	735<br>
Gly Ile Gly Val Pro Gly Glu Leu Ile Ile His Gly His Gly Val Ala<br>
740	745	750<br>
Gln Gly Tyr Val Ser Asp Pro Val Arg Ser Ala Ala Ser Phe Leu Pro<br>
755	760	765<br>
Ala Ser Asp Gly Leu Arg Cys Tyr Arg Thr Gly Asp Arg Val Arg Trp<br>
770	775	780<br>
Leu Pro Asp Gly Arg Leu Asp Phe Ile Gly Arg Glu Asp Asp Gln Val<br>
785	790	795	800<br>
Lys Val Arg Gly Phe Arg Val Glu Leu Gly Pro Val Gln Ala Ala Leu<br>
805	810	815<br>
His Ala Ile Glu Thr Ile His Glu Ser Ala Val Val Val Val Pro Lys<br>
820	825	830<br>
Gly Gln Gln Arg Ser Ile Val Ala Phe He Val Leu Lys Ala Pro Ser<br>
835	840	845<br>
Glu Asp Glu Ala Val Gln Arg Asn Asn Ile Lys Gln His Leu Leu Gly<br>
850	855	860Val Leu Pro Tyr Tyr Ala Leu Pro Asp Lys Phe Ile Phe Val Lys Ala<br>
865	870	875	880<br>
Leu Pro Arg Asn Thr His Gly Lys Ile Asp Arg Thr Leu Leu Leu Gln<br>
885	890	895<br>
His Glu Pro Gl Thr Glu Gl Glu Ser Ala Met Arg Asp Ala Thr Asp<br>
900	905	910<br>
Val Glu His Arg Ila Asn Cys Trp Gln Thr Ile Ile Gly His Pro<br>
915	920	925<br>
Val Gln Leu His Glu Asn Phe Leu Asp Ile Gly Gly His Ser Leu Ser<br>
930	935	940<br>
Leu Thr His Leu Thr Gly Leu Leu Arg Lys Glu Phe Asn Ile His Ile<br>
945	950	955	960<br>
Ser Leu His Asp Leu Trp Ile Arg Pro Thr Ile Glu Gln Gln Ala Asp<br>
965	970	975<br>
Phe Ile His Lys Leu Gln Asn Ser Val Leu Thr Lys Pro Ala Ala Ala<br>
980	965	990<br>
Pro Ile Pro Arg Leu Asp Arg Lys  Ile Ser His His<br>
995	1000<br>	SEQ ID 3<br>	Lenght: 1062<br>	Type: PRT<br>	Organism: Pseudomonas fluorescens A2-2<br>	SEQUENCE 3<br>
Met Ser Val Asp Thr Cys Arg Thr Ala Thr Phe Pro Ala Ser Tyr Gly<br>
15	10	15<br>
Gln Glu Gln ILe Trp Phe Leu Asn Glu Leu Asn Pro His Ser Gln Leu<br>
20	25	30<br>
Ala Tyr Thr Leu Ala Met Lys Val Ser Ile Ala Gly Lys Leu Asn Thr<br>
35	40	45<br>
Leu Arg Leu Gln Arg Ala Val Asn Gln Val Val Ala Ser Gln Glu Ile<br>
50	55	60<br><br>
Leu Arg Thr Ser Phe Ala Tyr Lys Asn Gln Lys Leu Ser Gln Val Ile<br>
65	70	75	80<br>
Ser Pro Ser Ala Thr Leu Pro Ile Arg Ser Ala His Cys Ile Asp Asp<br>
85	90	95<br>
Val Pro Gly Leu Gln Arg Leu Ile Asn Met Glu Ala Gln Arg Gly Trp<br>
100	105	110<br>
Ser Leu Ser Ser Ala Pro Leu Tyr Arg Leu Leu Leu Ile Lys Thr Gly<br>
115	120	125<br>
Asp Gln Gln His Glu Leu Val Ile Cys Thr His His Ile Val Cys Asp<br>
130	135	140<br>
Gly Ile Ser Leu Gln Leu Leu Leu Gln Lys Ile Val Ser Ala Tyr Gln<br>
145	150	155	160<br>
Gly Gln Ser Asp Gly Arg Val Leu Thr Ser Pro Asp Glu Glu Thr Leu<br>
165	170	175<br>
Gln PIle Val Asp Tyr Ala Ala Trp Ser Arg Gln His Glu Tyr Ala Gly<br>
180	185	190<br>
Leu Glu Tyr Trp Arg Gln Gln Leu Ala Asp Ala Pro Thr Ile Leu Asp<br>
195	200	205<br>
Ile Ser Thr Lys Thr Gly Arg Ser Glu Gin Gin Thr Phe Leu Gly Ala<br>
210	215	220<br>
Arg Ile Pro Val Glu Phe Ser His His Gin Trp Gin Ala Leu Arg Gin<br>
225	230	235	240<br>
Ile Phe Arg Pro Gln Gly Ile Ser Cys Ala Ala Val Phe Leu Ala Ala<br>
245	250	255<br>
Tyr Cys Val Val Leu His Arg Leu Ala Glu Gln Asp Asp Ile Leu Ile<br>
260	265	270<br>
Gly Leu Pro Thr Ser Asn Arg Leu Arg Pro Glu Leu Ala Gln Val Ile<br>
275	280	285<br>
3°<br><br>
IGly Tyr Leu Ser 290<br><br>
Asn Leu Cys Val<br>
295<br><br>
Phe Arg Ser Gln 300<br><br>
Tyr Ala His Asp<br><br><br><br>
Gln Ser Val Thr 305<br><br>
Asp Phe Leu Gln 310<br><br>
Gln Val Gln Leu 315<br><br>
Thr Leu Pro Asn 320<br><br><br><br>
Leu lie Glu His<br><br>
Gly Glu Thr Pro 325<br><br>
Phe Gln Gln Val 330<br><br>
Leu Glu Ser Val 335<br><br><br><br>
Glu His Thr Arg 340<br><br>
Gln Ala Gly Val<br><br>
Thr Pro Leu Cys 345<br><br>
Gln Val Leu Phe 350<br><br><br><br>
Gly Tyr Glu Gln 355<br><br>
Asp Val Arg Arg 360<br><br>
Thr Leu Asp lie<br><br>
Gly Asp Leu Gln 365<br><br><br><br>
Leu Thr Val Ser 370<br><br>
Asp Val Asp Thr 375<br><br>
Gly Ala Ala Arg 380<br><br>
Leu Asp Leu Ser<br><br><br><br>
Leu Phe Leu Phe 385<br><br>
Glu Asp Glu Leu 390<br><br>
Asn Val Cys Gly 395<br><br>
Phe Leu Glu Tyr 400<br><br><br><br>
Ala Thr Asp Arg<br><br>
lie Asp Ala Ala 405<br><br>
Ser Ala Gln Asn 410<br><br>
Met Val Arg Met 415<br><br><br><br>
Leu Ser Ser Val 420<br><br>
Leu Arg Glu Phe<br><br>
Val Ala Ala Pro 425<br><br>
Gln Ala Pro Leu 430<br><br><br><br>
Ser Glu Val Gln 435<br><br>
Leu Gly Ala Ala 440<br><br>
Asp Ser Gln Ala<br><br>
Gln Thr Pro Ala 445<br><br><br><br>
lie Ala Pro Ala 450<br><br>
Phe Pro Ser Val 455<br><br>
Pro Ala Arg Leu 460<br><br>
Phe Ala Leu Ala<br><br><br><br>
Asp Ser His Pro 465<br><br>
Asn Ala Thr Ala 470<br><br>
Leu Arg Asp Glu 475<br><br>
Gln Gly Glu Leu 4B0<br><br><br><br>
Thr Tyr Ala Gln<br><br>
Val Cys Gln Gln 4B5<br><br>
lie Leu Gln Ala 490<br><br>
Ala Ala Thr Leu 495<br><br><br><br>
Arg Ala Gln Gly 500<br><br>
Ala Lys Pro Gly<br><br>
Thr Leu lie Ala 505<br><br>
Val lie Gly Glu 510<br><br><br><br>
Arg Gly Asn Pro<br><br>
Trp Leu lie Ala<br><br>
Met Leu Ala lie<br><br>
Trp Gln Val Gly<br><br>
 515                                                520	525<br>
Gly lie Tyr Val Pro Leu Ser Lys Asp Leu Pro Glu Gln Arg Leu Gln<br>
530	535	540<br>
Gly lie Leu Ala Glu Leu Glu Gly Ala lie Leu lie Thr Asp Asp Thr<br>
545	550	555	560<br>
Thr Pro Glu Arg Phe Arg Gln Arg Val Thr Leu Pro Met His Ala Leu<br>
565	570	575<br>
Trp Ala Asp Gly Ala Thr His His Glu Arg Gln Thr Thr Asp Ala Ser<br>
580	585	590<br>
Arg Leu Ser Gly Tyr Met Met Tyr Thr Ser Gly Ser Thr Gly Lys Pro<br>
595	600	605<br>
Lys Gly Val His Val Ser Gln Ala Asn Leu Val Ala Thr Leu Ser Ala<br>
610	615	620<br>
Phe Gly Gln Leu Leu Gln Val Lys Pro Ser Asp Arg Met Leu Ala Leu<br>
625	630	635	640<br>
Thr Thr Phe Ser Phe Asp lie Ser Leu Leu Glu Leu Leu Leu Pro Leu<br>
645	650	655<br>
Val Gln Gly Ala Ser Val Gln lie Ala Val Ala Gln Ala Gln Arg Asp<br>
660	665	670<br>
Ala Glu Lys Leu Ala Gly Tyr Leu Ala Asp Pro Arg lie Thr Leu Val<br>
675	680	685<br>
Gln Ala Thr Pro Val Thr Trp Arg Leu Leu Leu Ser Thr Gly Trp Gln<br>
690	695	700<br>
Pro Arg Glu Ser Leu Thr Leu Leu Cys Gly Gly Glu Ala Leu Pro Gln<br>
705	710	715	720<br>
Asp Leu Ala Asp Arg Leu Cys Leu Pro Gly Met Thr Leu Trp Asn Leu<br>
725	730	735<br>
Tyr Gly Pro Thr Glu Thr Thr lie Trp Ser Thr Ala Cys Arg Leu Gln<br>
740	745	750<br><br><br><br>
Pro Gly Ala Pro 755<br><br>
Val Gln Leu Gly 760<br><br>
His Pro Ile Ala<br><br>
Gly Thr Gln Ile 765<br><br><br><br>
Ala Leu Val Asp 770<br><br>
Arg Asn Leu Arg 775<br><br>
Ser Val Pro Arg 780<br><br>
Gly Val Ile Gly<br><br><br><br>
Glu Leu Leu He 785<br><br>
Cys Gly Pro Gly 790<br><br>
Val Ser Gln Gly<br>
795<br><br>
Tyr Tyr Arg Asn BOO<br><br><br><br>
Pro Val Glu Thr<br><br>
Ala Lys Arg Phe 805<br><br>
Val Pro Asp Pro 810<br><br>
His Gly Ser Gly 815<br><br><br><br>
Lys Arg Ala Tyr 820<br><br>
Leu Thr Gly Asp<br><br>
Arg Met Arg Met 825<br><br>
Gln Gln Asp Gly 830<br><br><br><br>
Ser Leu Ala Tyr 835<br><br>
Ile Gly Arg Arg 840<br><br>
Asp Asp Gln Ile<br><br>
Lys Leu Arg Gly 845<br><br><br><br>
His Arg Ile Glu 850<br><br>
Leu Gly Glu Ile 855<br><br>
Glu Thr Ala Leu 860<br><br>
Arg Lys Leu Pro<br><br><br><br>
Gly Val Arg Asp 865<br><br>
Ala Ala Ala Gln B70<br><br>
Leu His Asp Gln 875<br><br>
Asp Pro Ser Arg 880<br><br><br><br>
Gly Ile Gln Ala<br><br>
Phe Val Gln Leu 885<br><br>
Cys Ala Thr Val 890<br><br>
Asp Glu Ser Leu 895<br><br><br><br>
Ile Asp Ile Gly 900<br><br>
Gln Trp Leu Glu<br><br>
Thr Leu Arg Gln 905<br><br>
Thr Leu Pro Glu 910<br><br><br><br>
Ala Trp Leu Pro 915<br><br>
Thr Glu Tyr Tyr 920<br><br>
Arg Ile Asp Gly<br><br>
Ile Pro Leu Thr 925<br><br><br><br>
Tyr Asn Gly Lys 930<br><br>
Arg Asp Arg Lys 935<br><br>
Arg Leu Leu His 940<br><br>
Gln Ala Val Arg<br><br><br><br>
Leu Gln Thr Leu 945<br><br>
Ser Leu Arg Val 950<br><br>
Ala Pro Ser Ser 955<br><br>
Asp Thr Glu Thr 960<br><br><br><br>
Arg Val Gln Gln<br><br>
Ile Trp Cys Glu 965<br><br>
Leu Leu Gly Leu 970<br><br>
Glu Asp Ile Gly 975<br><br>
Val Thr Asp Asp PIle Phe Gln Leu Gly Gly His Ser Ile Leu Val Ala<br>
980	985	990<br>
Arg Met Val Glu Arg Ile Glu Thr Ala Phe Gly Arg Arg Val Pro Ile<br>
995	1000	1005<br>
Ala Asp Ile Tyr Phe Ser Pro Thr Ile Ala Arg Val Ala Ala Thr<br>
1010	1015	1020<br>
Leu Asp Ser Met Thr Phe Glu Gln Gly Leu Ala Ala His Ser Val<br>
1025	1030	1035<br>
Lys Gly Asp Trp Glu Phe Thr Ala Ile Ser Leu Gln His Asn Ala<br>
1040	1045	1050<br>
Asp Ser Thr Ala Ala Ala Gln Glu Arg<br>
1055	1060<br> SEQ ID 4<br> Length: 1432<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE 4<br>
Met His Ser Pro Thr Ile Asp Thr Phe Glu Ala Ala Leu Arg Ser Leu<br>
15	10	15<br>
Pro Ala Ala Arg Asp Ala Leu Gly Ala Tyr Pro Leu Ser Ser Glu Gln<br>
20	25	30<br>
Lys Arg Leu Trp Leu Leu Ala Gln Leu Ala Gly Thr Ala Thr Leu Pro<br>
35	40	45<br>
Val Thr Val Arg Tyr Ala Phe Thr Gly Thr Val Asp Leu Ala Val Val<br>
50	55	60<br>
Gln Gln Asn Leu Ser Ala Trp Ile Ala His Ser Glu Ser Leu Arg Ser<br>
65	70	75	80<br>
Leu Phe Val Glu Val Leu Glu Arg Pro Val Arg Leu Leu Met Pro Thr<br><br>
85<br><br>
90<br><br>
95<br><br>
Gly Leu Val Lys Leu Glu Tyr Phe Asp Arg Pro Pro Ser Asp Ala Asp<br>
100	105	110<br>
Met Ala Glu Leu Ile Gly Ala Ala Phe Glu Leu Asp Lys Gly Pro Leu<br>
115	120	125<br>
Leu Arg Ala Phe He Thr Arg Thr Ala Ala Gln Gln His Glu Leu His<br>
130	135	140<br>
Leu Val Gly His Pro Ile Val Val Asp Glu Pro Ser Leu Gln Arg Ile<br>
145	150	155	160<br>
Ala Gln Thr Leu Phe Gln Thr Glu Pro Asp His Gln Tyr Pro Ala Val<br>
165	170	175<br>
Gly Ala Ile Ala Glu Val Phe Gln Arg Glu Gln Thr Leu Ala Gln Asp<br>
180	185	190<br>
Ala Gln Ile Thr Glu Gln Trp Gln Gln Trp Gly Ile Gly Leu Gln Ala<br>
195	200	205<br>
Pro Ala Ala Thr Glu Ile Pro Thr Glu Asn Pro Arg Pro Ala Ile Lys<br>
210	215	220<br>
Gly Ser Asp Arg Gln Val His Glu Ala Leu Thr Ala Trp Gly Asp Gln<br>
225	230	235	240<br>
Pro Val Ala Glu Ala Glu Ile Val Ser Ser Trp Leu Thr Val Leu Met<br>
245	250	255<br>
Arg Trp Gln Gly Ser Gln Ser Ala Leu Cys Ala Ile Lys Val Arg Asp<br>
260	265	270<br>
Lys Ala His Ala Asn Leu Ile Gly Pro Leu Gln Thr Tyr Leu Pro Val<br>
275	280	285<br>
Arg Val Asp Met Pro Asp Gly Ser Thr Leu Ala Gln Leu Arg Leu Gln<br>
290	295	300<br>
Val Glu Glu Gln Leu Asn Gly Asn Asp His Pro Ser Phe Ser Thr Leu<br>
305	310	315	320<br><br>
Leu Glu Val  Cys<br><br>
Pro Pro Lys Arg 325<br><br>
Asp Leu Ser Arg 330<br><br>
Thr Pro Tyr Phe 335<br><br><br><br>
Gln Thr Gly Leu 340<br><br>
Gln Phe Ile Ala<br><br>
His Asp Val Glu 345<br><br>
Gln Arg Asp Phe 350<br><br><br><br>
His Ala Gly Asn 355<br><br>
Leu Thr Arg Leu 360<br><br>
Pro Thr Lys Gln<br><br>
Pro Ser Ser Asp 365<br><br><br><br>
Leu Asp Leu Phe 370<br><br>
Ile Ser Cys Trp 375<br><br>
Val Ser Asp Gly 380<br><br>
Thr Leu Gly Leu<br><br><br><br>
Thr Leu Asp Tyr 385<br><br>
Asp Cys Ala Val 390<br><br>
Leu Asn Ser Ser 395<br><br>
Gln Val Glu Val 400<br><br><br><br>
Leu Ala Gln Ala<br><br>
Leu Ile Ser Val 405<br><br>
Leu Ser Ala Pro 410<br><br>
Gly Glu Gln Pro 415<br><br><br><br>
Ile Ala Thr Val 420<br><br>
Ala Leu Met Gly<br><br>
Gln Gln Met Gln 425<br><br>
Gln Thr Val Leu 430<br><br><br><br>
Ala Gln Ala His<br>
435<br><br>
Gly Pro Arg Thr 440<br><br>
Thr Pro Pro Gln<br><br>
Leu Thr Leu Thr 445<br><br><br><br>
Glu Trp Val Ala 450<br><br>
Ala Ser Thr Glu 455<br><br>
Lys Ser Pro Leu 460<br><br>
Ala Val Ala Val<br><br><br><br>
lie Asp His Gly 465<br><br>
Gln Gln Leu Ser 470<br><br>
Tyr Ala Glu Leu 475<br><br>
Trp Ala Arg Ala 480<br><br><br><br>
Ala Leu Val Ala<br><br>
Ala Asn Ile Ser<br>
485<br><br>
Gln His Val Ala 490<br><br>
Lys Pro Arg Ser 495<br><br><br><br>
Ile Ile Ala Val 500<br><br>
Ala Leu Pro Arg<br><br>
Ser Ala Glu Phe 505<br><br>
Ile Ala Ala Leu 510<br><br><br><br>
Leu Gly Val Val 515<br><br>
Arg Ala Gly His 520<br><br>
Ala Phe Leu Pro<br><br>
Ile Asp Pro Arg 525<br><br><br><br>
Leu Pro Thr Asp 530<br><br>
Arg Ile Gln PIle 535<br><br>
Leu Ile Glu Asn 540<br><br>
Ser Gly Cys Glu<br><br>
Leu Val Ile Thr Ser Asp Gln Gln Ser Val Glu Gly Trp Pro Gln Val<br>
545	550	555	560<br>
Ala Arg Ile Arg Met Glu Ala Leu Asp Pro Asp Ile Arg Trp Val Ala<br>
565	570	575<br>
Pro Thr Gly Leu Ser His Ser Asp Ala Ala Tyr Leu Ile Tyr Thr Ser<br>
580	585	590<br>
Gly Ser Thr Gly Val Pro Lys Gly Val Val Val Glu His Arg Gln Val<br>
595	600	605<br>
Val Asn Asn Ile Leu Trp Arg Gln Arg Thr Trp Pro Leu Thr Ala Gln<br>
610	615	620<br>
Asp Asn Val Leu His Asn His Ser Phe Ser Phe Asp Pro Ser Val Trp<br>
625	630	635	640<br>
Ala Leu Phe Trp Pro Leu Leu Thr Gly Gly Thr lle Val Leu Ala Asp<br>
645	650	655<br>
Val Arg Thr Met Glu Asp Ser Thr Ala Leu Leu Asp Leu Met Ile Arg<br>
660	665	670<br>
His Asp Val Ser Val Leu Gly Gly Val Pro Ser Leu Leu Gly Thr Leu<br>
675	680	685<br>
Ile Asp His Pro Phe Ala Asn Asp Cys Arg Ala Val Lys Leu Val Leu<br>
690	695	700<br>
Ser Gly Gly Glu Val Leu Asn Pro Glu Leu Ala His Lys Ile Gln Lys<br>
705	710	715	720<br>
Val Trp Gln Ala Asp Val Ala Asn Leu Tyr Gly Pro Thr Glu Ala Thr<br>
725	730	735<br>
lle Asp Ala Leu Tyr PIle Ser Ile Asp Lys Asn Ala Ala Gly Ala Ile<br>
740	745	750<br>
Pro Ile Gly Tyr Pro lle Asp Asn Thr Asp Ala Tyr Ile Val Asp Leu<br>
755	760	765<br><br>
Asn Leu Asn Pro Val Pro Pro Gly Val Pro Gly Glu Ile Met Leu Ala<br>
770	775	780<br>
Gly Gln Asn Leu Ala Arg Gly Tyr Leu Gly Lys Pro Ala Gln Thr Ala<br>
785	790	795	600<br>
Gln Arg Phe Leu Pro Asn Pro Phe Gly Asn Gly Arg Val Tyr Ala Thr<br>
805	810	815<br>
Gly Asp Leu Gly Arg Arg Trp Ser Ser Gly Ala Ile Ser Tyr Leu Gly<br>
820	825	830<br>
Arg Arg Asp Gln Gln Val Lys Ile Arg Gly His Arg Ile Glu Leu Asn<br>
835	840	845<br>
Glu Val Ala His Leu Leu Cys Gln Ala Leu Glu Leu Lys Glu Ala Ile<br>
850	855	860<br>
Val Phe Ala Gln His Ala Gly Thr Glu Gln Ala Arg Leu Val Ala Ala<br>
865	870	875	880<br>
Ile Glu Gln Gln Pro Gly Leu His Ser Glu Gly Ile Lys Gln Glu Leu<br>
885	890	895<br>
Leu Arg His Leu Pro Ala Tyr Leu Ile Pro Ser Gln Leu Leu Leu Leu<br>
900	905	910<br>
Asp Glu Leu Pro Arg Thr Ala Thr Gly Lys Val Asp Met Leu Lys Leu<br>
915	920	925<br>
Asp Gln Leu Ala Ala Pro Gln Leu Asn Asp Ala Gly Gly Thr Glu Cys<br>
930	935	940<br>
Arg Ala Pro Arg Thr Asp Leu Glu Gln Ser Val Met Thr Asp Phe Ala<br>
945	950	955	960<br>
Gln Val Leu Gly Leu Thr Ala Val Thr Pro Asp Thr Asp Phe Phe Glu<br>
965	970	975<br>
Gln Gly Gly Asn Ser Ile Leu Leu Thr Arg Leu Ala Gly Thr Leu Ser<br>
980	985	990<br>
Ala Lys Tyr Gln Val Gln lle Pro Leu His Glu PIle Phe Leu Thr Pro<br><br>
995<br><br>
1000<br><br>
1005<br><br>
Thr Pro Ala Ala Val Ala Gln Ala Ile Glu Ile Tyr Arg Arg Glu<br>
1010	1015	1020<br>
Gly Leu Thr Ala Leu Leu Ser Arg Gln His Ala Gln Thr Leu Glu<br>
1025	1030	1035<br>
Gln Asp Ile Tyr Leu Glu Glu His Ile Arg Pro Asp Gly Leu Pro<br>
1040	1045	1050<br>
His Ala Asn Trp Tyr Gln Pro Ser Val Val Phe Leu Thr Gly Ala<br>
1055	1060	1065<br>
Thr Gly Tyr Leu Gly Leu Tyr Leu Ile Glu Gln Leu Leu Lys Arg<br>
1070	1075	1080<br>
Thr Thr Ser Arg Val Ile Cys Leu Cys Arg Ala Lys Asp Ala Glu<br>
1085	1090	1095<br>
His Ala Lys Ala Arg Ile Leu Glu Gly Leu Lys Thr Tyr Arg Ile<br>
1100	1105	1110<br>
Asp Val Gly Ser Glu Leu His Arg Val Glu Tyr Leu Thr Gly Asp<br>
1115	1120	1125<br>
Leu Ala Leu Pro His Leu Gly Leu Ser Glu His Gln Trp Gln Thr<br>
1130	1135	1140<br>
Leu Ala Glu Glu Val Asp Val  Ile Tyr His Asn Gly Ala Leu Val<br>
1145	1150	1155<br>
Asn Phe Val Tyr Pro Tyr Ser Ala Leu Lys Ala Thr Asn Val Gly<br>
1160	1165	1170<br>
Gly Thr Gln Ala Ile Leu Glu Leu Ala Cys Thr Ala Arg Leu Lys<br>
1175	1180	1185<br>
Ser Val Gln Tyr Val Ser Thr Val Asp Thr Leu Leu Ala Thr His<br>
1190	1195	1200<br>
Val Pro Arg Pro Phe Ile Glu Asp Asp Ala Pro Leu Arg Ser Ala<br>
1205	1210	1215<br><br><br>
Val Gly Val Pro Val Gly Tyr Thr Gly Ser Lys Trp Val Ala Glu<br>
1220	1225	1230<br>
Gly Val Ala Asn Leu Gly Leu Arg Arg Gly lle Pro Val Ser lle<br>
1235	1240	1245<br>
Phe Arg Pro Gly Leu lle Leu Gly His Thr Glu Thr Gly Ala Ser<br>
1250	1255	1260<br>
Gln Ser lle Asp Tyr Leu Leu Val Ala Leu Arg Gly Phe Leu Pro<br>
1265	1270	1275<br>
Met Gly lle Val Pro Asp Tyr Pro Arg lle Phe Asp lle Val Pro<br>
1280	1285	1290<br>
Val Asp Tyr Val Ala Ala Ala lie Val His lie Ser Met Gln Pro<br>
1295	1300	1305<br>
Gln Gly Arg Asp Lys Phe Phe His Leu Phe Asn Pro Ala Pro Val<br>
1310	1315	1320<br>
Thr lle Arg Gln Phe Cys Asp Trp lle Arg Glu Phe Gly Tyr Glu<br>
1325	1330	1335<br>
Phe Lys Leu Val Asp Phe Glu His Gly Arg Gln Gln Ala Leu Ser<br>
1340	1345	1350<br>
Val Pro Pro Gly His Leu Leu Tyr Pro Leu Val Pro Leu Ile Arg<br>
1355	1360	1365<br>
Asp Ala Asp Pro Leu Pro His Arg Ala Leu Asp Pro Asp Tyr Ile<br>
1370	1375	1380<br>
His Glu Val Asn Pro Ala Leu Glu Cys Lys Gln Thr Leu Glu Leu<br>
1385	1390	1395<br>
Leu Ala Ser Ser Asp Ile Thr Leu Ser Lys Thr Thr Lys Ala Tyr<br>
1400	1405	1410<br>
Ala His Thr Ile Leu Arg Tyr Leu Ile Asp Thr Gly PIle Met Ala<br>
1415	1420	1425<br><br>
Lys Pro Gly Val 1430<br> SEQID5<br> Lenght: 350<br> TYpe: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE 5<br>
Met Glu Ser Ile Ala Phe Pro Ile Ala His Lys Pro Phe Ile Leu Gly<br>
15	10	15<br>
Cys Pro Glu Asn Leu Pro Ala Thr Glu Arg Ala Leu Ala Pro Ser Ala<br>
20	25	30<br>
Ala Met Ala Arg Gln Val Leu Glu Tyr Leu Glu Ala Cys Pro Gln Ala<br>
35	40	45<br>
Lys Asn Leu Glu Gln Tyr Leu Gly Thr Leu Arg Glu Val Leu Ala His<br>
50	55	60<br>
Leu Pro Cys Ala Ser Thr Gly Leu Met Thr Asp Asp Pro Arg Glu Asn<br>
65	70	75	80<br>
Gln Glu Asn Arg Asp Asn Asp Phe Ala Phe Gly Ile Glu Arg His Gln<br>
85	90	95<br>
Gly Asp Thr Val Thr Leu Met Val Lys Ala Thr Leu Asp Ala Ala Ile<br>
100	105	110<br>
Gln Thr Gly Glu Leu Val Gln Arg Ser Gly Thr Ser Leu Asp His Ser<br>
115	120	125<br>
Glu Trp Ser Asp Met Met Ser Val Ala Gln Val Ile Leu Gln Thr Ile<br>
130	135	140<br>
Ala Asp Pro Arg Val Met Pro Glu Ser Arg Leu Thr Phe Gln Ala Pro<br>
145	150	155	160<br>
Lys Ser Lys Val Glu Glu Asp Asp Gln Asp Pro Leu Arg Arg Trp Val<br>
165	170	175<br><br>
Arg Gly His Leu Leu Phe Met Val Leu Cys Gln Gly Met Ser Leu Cys<br>
180	185	150<br>
Thr Asn Leu Leu lle Ser Ala Ala His Asp Lys Asp Leu Glu Leu Ala<br>
195	200	205<br>
Cys Ala Gln Ala Asn Arg Leu lle Gln Leu Met Asn Ile Ser Arg Ile<br>
210	215	220<br>
Thr Leu Glu Phe Ala Thr Asp Leu Asn Ser Gln Gln Tyr Val Ser Gln<br>
225	230	235	240<br>
Ile Arg Pro Thr Leu Met Pro Ala Ile Ala Pro Pro Lys Met Ser Gly<br>
245	250	255<br>
Ile Asn Trp Arg Asp His Val Val Met Ile Arg Trp Met Arg Gln Ser<br>
260	255	270<br>
Thr Asp Ala Trp Asn Phe Ile Glu Gln Ala Tyr Pro Gln Leu Ala Glu<br>
275	280	285<br>
Arg Met Arg Thr Thr Leu Ala Gln Val Tyr Ser Ala His Arg Gly Val<br>
290	295	300<br>
Cys Glu Lys Phe Val Gly Glu Glu Asn Thr Ser Leu Leu Ala Lys Glu<br>
305	310	315	320<br>
Asn Ala Thr Asn Thr Ala Gly Gln Val Leu Glu Asn Leu Lys Lys Ser<br>
325	330	335<br>
Arg Leu Lys Tyr Leu Lys Thr Lys Gly Cys Ala Gly Ala Gly<br>
340	345	350<br>  SEQID6<br> Lenght:61<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 6<br>
Met Pro Thr Phe Leu Gly Asp Asp Asp Ala Val Pro Cys Val Val Val<br>
15	10	15<br><br>
{<br><br>
Val Asn Ala Asp Lys His Tyr Ser Ile Trp Pro Ser Ala Arg Asp Ile<br>
20	25	30<br>
Pro Ser Gly Trp Ser Glu Glu Gly Phe Lys Gly Ser Arg Ser Asp Cys<br>
35	40	45<br>
Leu Glu His Ile Ala Gln Ile Trp Pro Glu Pro Thr Ala<br>
50	55	60<br> SEQ ID 7<br> Lenght: 355<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br>  SEQUENCE 7<br>
Met Thr Ser Thr His Arg Thr Thr Asp Gln Val Lys Pro Ala Val Leu<br>
15	10	15<br>
Asp Met Pro Gly Leu Ser Gly Ile Leu Phe Gly His Ala Ala Phe Gln<br>
20	25	30<br>
Tyr Leu Arg Ala Ser Cys Glu Leu Asp Leu Phe Glu His Val Arg Asp<br>
35	40	45<br>
Leu Arg Glu Ala Thr Lys Glu Ser Ile Ser Ser Arg Leu Lys Leu Gln<br>
50	55	60<br>
Glu Arg Ala Ala Asp Ile Leu Leu Leu Gly Ala Thr Ser Leu Gly Met<br>
65	70	75	80<br>
Leu Val Lys Glu Asn Gly lle Tyr Arg Asn Ala Asp Val Val Glu Asp<br>
85	90	95<br>
Leu Met Ala Thr Asp Asp Trp Gln Arg Phe Lys Asp Thr Val Ala Phe<br>
100	105	110<br>
Glu Asn Tyr Ile Val Tyr Glu Gly Gln Leu Asp Phe Thr Glu Ser Leu<br>
115	120	125<br>
Gln Lys Asn Thr Asn Val Gly Leu Gln Arg Phe Pro Gly Glu Gly Arg<br>
130	135	140<br><br><br>
Asp Leu Tyr His Arg Leu His Gln Asn Pro Lys Leu Glu Asn Val Phe<br>
145	150	155	160<br>
Tyr Arg Tyr Met Arg Ser Trp Ser Glu Leu Ala Asn Gln Asp Leu Val<br>
165	170	175<br>
Lys His Leu Asp Leu Ser Arg Val Lys Lys Leu Leu Asp Ala Gly Gly<br>
180	185	190<br>
Gly Asp Ala Val Asn Ala lle Ala Leu Ala Lys His Asn Glu Gln Leu<br>
195	200	205<br>
Asn Val Thr Val Leu Asp lle Asp Asn Ser lle Pro Val Thr Gln Gly<br>
210	215	220<br>
Lys lle Asn Asp Ser Gly Leu Ser His Arg Val Lys Ala Gln Ala Leu<br>
225	230	235	240<br>
Asp Ile Leu His Gln Ser Phe Pro Glu Gly Tyr Asp Cys Ile Leu Phe<br>
245	250	255<br>
Ala His Gln Leu Val Ile Trp Thr Leu Glu Glu Asn Thr His Met Leu<br>
260	265	270<br>
Arg Lys Ala Tyr Asp Ala Leu Pro Glu Gly Gly Arg Val Val Ile Phe<br>
275	280	285<br>
Asn Ser Met Ser Asn Asp Glu Gly Asp Gly Pro Val Met Ala Ala Leu<br>
290	295	300<br>
Asp Ser Val Tyr Phe Ala Cys Leu Pro Ala Glu Gly Gly Met Ile Tyr<br>
305	310	315	320<br>
Ser Trp Lys Gln Tyr Glu Val Cys Leu Ala Glu Ala Gly Phe Lys Asn<br>
325	330	335<br>
Pro Val Arg Thr Ala Ile Pro Gly Trp Thr Pro His Gly Ile Ile Val<br>
340	345	350<br>
Ala Tyr Lys 355<br><br><br> SEQID8<br> Lenght347<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE 8<br>
Met Ala Arg Ser Pro Glu Thr Asn Ser Ala Met Pro Gln Gln lle Arg<br>
15	10	15<br>
Gln Leu Leu Tyr Ser Gln Leu lle Ser Gln Ser Ile Gln Thr Phe Cys<br>
20	25	30<br>
Glu Leu Arg Leu Pro Asp Val Leu Gln Ala Ala Gly Gln Pro Thr Ser<br>
35	40	45<br>
Ile Glu Arg Leu Ala Glu Gln Thr His Thr His Ile Ser Ala Leu Ser<br>
50	55	60<br>
Arg Leu Leu Lys Ala Leu Lys Pro Phe Gly Leu Val Lys Glu Thr Asp<br>
65	70	75	80<br>
Glu Gly Phe Ser Leu Thr Asp Leu Gly Ala Ser Leu Thr His Asp Ala<br>
85	90	95<br>
Phe Ala Ser Ala Gln Pro Ser Ala Leu Leu Ile Asn Gly Glu Met Gly<br>
100	105	110<br>
Gln Ala Trp Arg Gly Met Ala Gln Thr Ile Arg Thr Gly Glu Ser Ser<br>
115	120	125<br>
Phe Lys Met Tyr Tyr Gly Ile Ser Leu Phe Glu Tyr Phe Glu Gln His<br>
130	135	140<br>
Pro Glu Arg Arg Ala Ile Phe Asp Arg Ser Gln Asp Met Gly Leu Asp<br>
145	150	155	160<br>
Leu Glu lle Pro Glu Ile Leu Glu Asn Ile Asn Leu Asn Asp Gly Glu<br>
165	170	175<br>
Asn Ile Val Asp Val Gly Gly Gly Ser Gly His Leu Leu Met His Met<br>
180	185	190<br><br>
Leu Asp Lys Trp Pro Glu Ser Thr Gly Ile Leu Phe Asp Leu Pro Val<br>
195	200	205<br>
Ala Ala Lys Ile Ala Gln Oln His Leu His Lys Ser Gly Lys Ala Gly<br>
210	215	220<br>
Cys Phe Glu Ile Val Ala Gly Asp Phe Phe Lys Ser Leu Pro Asp Ser<br>
225	230	235	240<br>
Gly Ser Val Tyr Leu Leu Ser His Val Leu His Asp Trp Gly Asp Glu<br>
245	250	255<br>
Asp Cys Lys Ala Ile Leu Ala Thr Cys Arg Arg Ser Met Pro Asp Asn<br>
260	265	270<br>
Ala Leu Leu Val Val Val Asp Leu Val Ile Asp Gln Ser Glu Ser Ala<br>
275	280	285<br>
Gln Pro Asn Pro Thr Gly Ala Met Met Asp Leu Tyr Met Leu Ser Leu<br>
290	295	300<br>
Phe Gly Ile Ala Gly Gly Lys Glu Arg Asn Glu Asp Glu PIle Arg Thr<br>
305	310	315	320<br>
Leu Ile Glu Asn Ser Gly Phe Asn Val Lys Gln Val Lys Arg Leu Pro<br>
325	330	335<br>
Ser Gly Asn Gly Ile Ile Phe Ala Tyr Pro Lys<br>
340	345<br> SEQID <br> Lenght: 180<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 9<br>
Met Ser Thr Leu Val Tyr Tyr Val Ala Ala Thr Leu Asp Gly Tyr Ile<br>
15	10	15<br>
Ala Thr Gln Gln His Lys Leu Asp Trp Leu Glu Asn Phe Ala Leu Gly<br>
20	25	30<br><br><br>
Asp Asp Ala Thr Ala Tyr Asp Asp Phe Tyr Gln Thr lle Gly Ala Val<br>
35	40	45<br>
Val Met Gly Ser Gln Thr Tyr Glu Trp Ile Met Ser Asn Ala Pro Asp<br>
50	55	SO<br>
Asp Trp Pro Tyr Gln Asp Val Pro Ala Phe Val Met Ser Asn Arg Asp<br>
65	70	75	80<br>
Leu Ser Ala Pro Ala Asn Leu Asp Ile Thr PIle Leu Arg Gly Asp Ala<br>
85	90	95<br>
Ser Ala Ile Ala Val Arg Ala Arg Gln Ala Ala Lys Gly Lys Asn Val<br>
100	105	110<br>
Trp Leu Val Gly Gly Gly Lys Thr Ala Ala Cys Phe Ala Asn Ala Gly<br>
115	120	125<br>
Glu Leu Gln Gln Leu Phe Ile Thr Thr Ile Pro Thr Phe Ile Gly Thr<br>
130	135	140<br>
Gly Val Pro Val Leu Pro Val Asp Arg Ala Leu Glu Val Val Leu Arg<br>
145	150	155	160<br>
Glu Gln Arg Thr Leu Gln Ser Gly Ala Met Glu Cys Ile Leu Asp Val<br>
165	170	175<br>
Lys Lys Ala Asp 180<br> SEQID 10<br> Length: 220<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 10<br>
Met Ser Asn Val Phe Ser Gly Gly Lys Gly Asn Gly Asn Pro Gly Phe<br>
15	10	15<br>
Val Arg Thr Phe Ser Arg Ile Ala Pro Thr Tyr Glu Glu Lys Tyr Gly<br>
20	25	30<br>
Thr Lys Leu Ser Gln Ala His Asp Asp Cys Leu Arg Met Leu Ser Arg<br><br>
35<br><br>
40<br><br>
45<br><br>
Trp Met Cys Thr Ser Arg Pro Glu Arg Val Leu Asp Ile Gly Cys Gly<br>
50	55	60<br>
Thr Gly Ala Leu Ile Glu Arg Met Phe Ala Leu Trp Pro Glu Ala Arg<br>
65	70	75	80<br>
Phe Glu Gly Val Asp Pro Ala Gln Gly Met Val Asp Glu Ala Ala Lys<br>
85	90	95<br>
Arg Arg Pro Phe Ala Ser Phe Val Lys Gly Val Ala Glu Ala Leu Pro<br>
100	105	110<br>
Phe Pro Ser Gln Ser Met Asp Leu Val Val Cys Ser Met Ser Phe Gly<br>
115	120	125<br>
His Trp Ala Asp Lys Ser Val Ser Leu Asn Glu Val Arg Arg Val Leu<br>
130	135	140<br>
Lys Pro Gln Gly Leu Phe Cys Leu Val Glu Asn Leu Pro Ala Gly Trp<br>
145	150	155	160<br>
Gly Leu Thr Thr Leu Ile Asn Trp Leu Leu Gly Ser Leu Ala Asp Tyr<br>
165	170	175<br>
Arg Ser Glu His Glu Val Ile Gln Leu Ala Gln Thr Ala Gly Leu Gln<br>
180	185	190<br>
Ser Met Glu Thr Ser Val Thr Asp Gln His Val Ile Val Ala Thr Phe<br>
195	200	205<br>
Arg Pro Cys Cys Gly Glu Val Gly Asp His Gly Arg<br>
210	215	220<br> SEQID 11<br> Length: 509<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 11<br>
Met Val Val Lys Asn Lys Gln Val Leu Val Val Gly Ala Gly Pro Val<br><br>
10	15<br>
Gly Leu Ala Val Ala Ala Ala Leu Ala Glu Leu Gly lle Ala Val Asp<br>
20	25	30<br>
Leu lle Asp Lys Arg Pro Ala Ala Ser Pro His Ser Arg Ala Phe Gly<br>
35	40	45<br>
Leu Glu Pro Val Thr Leu Glu Leu Leu Asn Ala Trp Gly Val Ala Asp<br>
50	55	60<br>
Glu Met lle Arg Arg Gly Ile Val Trp Ala Ser Ala Pro Leu Gly Asp<br>
65	70	75	80<br>
Lys Ala Gly Arg Thr Leu Ser Phe Ser Lys Leu Pro Cys Glu Tyr Pro<br>
85	90	95<br>
His Met Val Ile Ile Pro Gln Ser Gln Thr Glu Ser Val Leu Thr Asp<br>
100	105	110<br>
Trp Val Asn Arg Lys Gly Val Asn Leu Lys Arg Gly Tyr Ala Leu Lys<br>
115	120	125<br>
Ala Leu Asp Ala Gly Asp Leu His Val Glu Val Thr Leu Glu His Ser<br>
130	135	140<br>
Glu Thr Gly Ser Val Gln Gln Ser Arg Tyr Asp Trp Val Leu Gly Ala<br>
145	150	155	160<br>
Asp Gly Val Asn Ser Ser Val Arg Gln Leu Leu Asn Ile Ser Phe Val<br>
165	170	175<br>
Gly Gln Asp Tyr Lys His Ser Leu Val Val Ala Asp Val Val Leu Arg<br>
180	185	190<br>
Asn Pro Pro Ser Pro Ala Val His Ala Arg Ser Val Ser Arg Gly Leu<br>
195	200	205<br>
Val Ala Leu Phe Pro Leu Pro Asp Gly Ser Tyr Arg Val Ser Ile Glu<br>
210	215	220<br>
Asp Asn Glu Arg Met Asp Thr Pro Val Lys Gln Pro Val Thr His Glu<br>
225	230	235	240<br><br>
Glu lle Ala Gly<br><br>
Gly Met Lys Asp 245<br><br>
Ile Leu Gly Thr 250<br><br>
Asp Phe Gly Leu 255<br><br><br><br>
Ala Gln Val Leu 260<br><br>
Trp Ser Ala Arg<br><br>
Tyr Arg Ser Gln 265<br><br>
Gln Arg Leu Ala 270<br><br><br><br>
Thr His Tyr Arg 275<br><br>
Gln Gly Arg Val 280<br><br>
Phe Leu Leu Gly<br><br>
Asp Ala Ala His 285<br><br><br><br>
Thr His Val Pro 290<br><br>
Ala Gly Gly Gln 295<br><br>
Gly Leu Gln Met 300<br><br>
Gly Ile Gly Asp<br><br><br><br>
Ala Ala Asn Leu 305<br><br>
Ala Trp Lys Leu 310<br><br>
Ala Gly Val Ile 315<br><br>
Gln Ala Thr Leu 320<br><br><br><br>
Pro Met Asp Leu<br><br>
Leu Glu Ser Tyr 325<br><br>
Glu Ala Glu Arg 330<br><br>
Arg Pro Ile Ala 335<br><br><br><br>
Ala Ala Ala Leu 340<br><br>
Arg Asn Thr Asp<br><br>
Leu Leu Phe Arg 345<br><br>
Phe Asn Thr Ala 350<br><br><br><br>
Ser Gly Pro lle 355<br><br>
Gly Arg Leu lle 360<br><br>
His Trp Ile Gly<br><br>
Leu Gln Ala Thr 365<br><br><br><br>
Arg Ala Pro Tyr 370<br><br>
Val Ala Gln Lys 375<br><br>
Val Val Ser Ala 380<br><br>
Leu Ala Gly Glu<br><br><br><br>
Gly Val Arg Tyr 385<br><br>
Asp Ser Val Arg 390<br><br>
Arg Arg Gly Asp 395<br><br>
His Arg Leu Val 400<br><br><br><br>
Gly Arg Arg Leu<br><br>
Pro Leu Leu Ser<br>
405<br><br>
Leu Leu Pro Glu 410<br><br>
Gly Glu Arg Leu 415<br><br><br><br>
Pro Arg Gln Ser 420<br><br>
Leu Thr Gln Leu<br><br>
Leu Arg Ala Gly 425<br><br>
Arg Phe Val Leu 430<br><br><br><br>
Val His His Arg 435<br><br>
Ala Lys Ala Leu 440<br><br>
Ala Ala Asp Leu<br><br>
Arg Arg Asp Phe 445<br><br><br><br>
Pro Gly Leu Gln 450<br><br>
Thr Ala Ser Ile 455<br><br>
Cys Glu Asp Ser 460<br><br>
His Asn Asn Ser<br><br>
Leu Ser Ala Gly Glu Gly Val  Ile Val Arg Pro Asp Gly Val Val Ile<br>
465	470	475	480<br>
Trp Val Gly Lys Lys  Ser Thr Leu Ala Lys Glu Arg Leu Gly Glu Trp<br>
485	490	495<br>
Leu Leu Asp Asp Ser Lys  Ser Ala Arg Gln Ser Leu Thr<br>
500	505<br> SEQID 12<br> LENGHT:348<br> TYPE: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 12<br>
Met Ala His Tyr Asp Ser Val Gly Thr Ala Pro Gly Ala Ser Asp Asp<br>
15	10	15<br>
Gly Met Ala Val Ala Ser Ile Leu Gln Leu Met Arg Glu Thr Ile Thr<br>
20	25	30<br>
Arg Ser Asp Ala Lys Asn Asn Val Val Phe Leu Leu Ala Asp Gly Glu<br>
35	40	45<br>
Glu Leu Gly Leu Leu Gly Ala Glu His Tyr Val Ser Gln Leu Ser Thr<br>
50	55	60<br>
Pro Glu Arg Glu Ala Ile Arg Leu Val Leu Asn Phe Glu Ala Arg Gly<br>
65	70	75	80<br>
Asn Gln Gly Ile Pro Leu Leu Phe Glu Thr Ser Gln Lys Asp Tyr Ala<br>
85	90	95<br>
Leu Ile Arg Thr Val Asn Ala Gly Val Arg Asp Ile Ile Ser Phe Ser<br>
100	105	110<br>
Phe Thr Pro Leu Ile Tyr Asn Met Leu Gln Asn Asp Thr Asp Phe Thr<br>
115	120	125<br>
Val Phe Arg Lys Lys Asn Ile Ala Gly Leu Asn Phe Ala Val Val Glu<br>
130	135	140<br><br>
Gly Phe Gln His Tyr His His Met Ser Asp Thr Val Glu Asn Leu Gly<br>
145	150	155	160<br>
Pro Glu Thr Leu Phe Arg Tyr Gln Lys Thr Val Arg Glu Val Gly Asn<br>
165	170	175<br>
His Phe Ile Gln Gly Ile Asp Leu Ser Ser Leu Ser Ala Asp Glu Asp<br>
180	185	190<br>
Ala Thr Tyr Phe Pro Leu Pro Gly Gly Thr Leu Leu Val Leu Asn Leu<br>
195	200	205<br>
Pro Thr Leu Tyr Ala Leu Gly Met Gly Ser Phe Val Leu Cys Gly Leu<br>
210	215	220<br>
Trp Ala Gln Arg Cys Arg Thr Arg Arg Gln His Gln Gly Lys Asn Cys<br>
225	230	235	240<br>
Val Leu Arg Pro Met Ala Ile Ala Leu Leu Gly Ile Ala Cys Ala Ala<br>
245	250	255<br>
Leu Val Phe Tyr Val Pro Ser Ile Ala Tyr Leu Phe Val Ile Pro Ser<br>
260	265	270<br>
Leu Leu Leu Ala Cys Ala Met Leu Ser Arg Ser Leu Phe Ile Ser Tyr<br>
275	280	285<br>
Ser Ile Met Leu Leu Gly Ala Tyr Ala Cys Gly Ile Leu Tyr Ala Pro<br>
290	295	300<br>
Ile Val Tyr Leu Ile Ser Ser Gly Leu Lys Met Pro Phe Ile Ala Gly<br>
305	310	315	320<br>
Val Ile Ala Leu Leu Pro Leu Cys Leu Leu Ala Val Gly Leu Ala Gly<br>
325	330	335<br>
Val Ile Ala Arg Ser Arg Asp Cys Arg Thr Cys Asp<br>
340	345<br><br><br>  SEQ ID 13<br> Lenght: 572<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br> SEQUENCE: 13<br>
Met Arg Ser Leu Lys ILe ILe Val Leu Ala Ser Ala Phe Asn Gly Leu<br>
15	10	15<br>
Thr Gln Arg Ala Trp Leu Asp Leu Arg Gln Ser Gly His Ala Pro Ser<br>
20	25	30<br>
Val Val Leu Phe Thr Asp Pro Ala Leu Val Cys Gln Gln ILe Glu Asp<br>
35	40	45<br>
Ser Asp Ala Asp Leu Val Ile Cys Pro Phe Leu Lys Asp Arg Val Pro<br>
50	55	60<br>
Gln Gln Leu Trp Ser Asn Leu Glu Arg Pro Val Val Ile Ile His Pro<br>
65	70	75	80<br>
Gly Ile Val Gly Asp Arg Gly Ala Ser Ala Leu Asp Trp Ala Ile Ser<br>
85	90	95<br>
Gln Gln Val Gly Arg Trp Gly Val Thr Ala Leu Gln Ala Val Glu Glu<br>
100	105	110<br>
Met Asp Ala Gly Pro Ile Trp Ser Thr Cys Glu PIle Asp Met Pro Ala<br>
115	120	125<br>
Asp Val Arg Lys Ser Glu Leu Tyr Asn Gly Ala Val Ser Asp Ala Ala<br>
130	135	140<br>
Leu Tyr Cys Ile Arg Asp Val Val Glu Lys Phe Ala Arg Val Phe Val<br>
145	150	155	160<br>
Pro Val Pro Leu Asp Tyr Thr Gln Ala His Val Ile Gly Arg Leu Gln<br>
165	170	175<br>
Pro Asn Met Thr Gln Ala Asp Arg Thr Phe Ser Trp Tyr Asp Cys Ala<br>
180	185	190<br>
Arg Phe Ile Lys Arg Cys Ile Asp Ala Ala Asp Gly Gln Pro Gly Val<br><br>
195<br><br>
200<br><br>
205<br><br><br><br>
Leu Ala Ser Ile 210<br><br>
Gln Gly Gly Gln 215<br><br>
Tyr Tyr Leu Tyr 220<br><br>
Asp Ala His Leu<br><br><br><br>
Asp Ala Arg His 225<br><br>
Gly Thr Pro Gly 230<br><br>
Glu Ile Leu Ala 235<br><br>
Val Gln Asp Asp 240<br><br><br><br>
Ala Val Leu Val<br><br>
Ala Ala Gly Asp 245<br><br>
Gln Ser Leu Trp 250<br><br>
Ile Gly Ser Leu 255<br><br><br><br>
Lys Arg Lys Ala 260<br><br>
Arg Pro Gly Glu<br><br>
Glu Thr Phe Lys 265<br><br>
Leu Pro Ala Arg 270<br><br><br><br>
His Val Leu Ala 275<br><br>
Glu Ala Leu Ala 280<br><br>
Asp Ile Pro Val<br><br>
Leu Asp Ser Ser 285<br><br><br><br>
Ile Ala Asn Gln 290<br><br>
Met Phe Asp Glu 295<br><br>
Gln Ala Tyr Gln 300<br><br>
Pro Ile Arg Tyr<br><br><br><br>
Arg Glu Ala Gly 305<br><br>
His Val Gly Glu 310<br><br>
Leu Thr Phe Glu 315<br><br>
Phe Tyr Asn Gly 320<br><br><br><br>
Ala Met Ser Thr<br><br>
Glu Gln Cys Gln 325<br><br>
Arg Leu Val Ala 330<br><br>
Ala Leu Arg Trp 335<br><br><br><br>
Ala Lys Thr Arg 340<br><br>
Asp Thr Gln Val<br><br>
Leu Val Ile Lys 345<br><br>
Gly Gly Arg Gly 350<br><br><br><br>
Ser Phe Ser Asn 355<br><br>
Gly Val His Leu 360<br><br>
Asn Val Ile Gln<br><br>
Ala Ala Pro Val 365<br><br><br><br>
Pro Gly Leu Glu 370<br><br>
Ala Trp Ala Asn 375<br><br>
Ile Gln Ala Ile 380<br><br>
Tyr Asp Val Cys<br><br><br><br>
His Glu Leu Leu 385<br><br>
Thr Ala Arg Gln 390<br><br>
Leu Val Ile Ser 395<br><br>
Gly Leu Thr Gly 400<br><br><br><br>
Ser Ala Gly Ala<br><br>
Gly Gly Val Met 405<br><br>
Leu Ala Leu Ala 410<br><br>
Ala Asp Ile Val 415<br><br><br><br>
Leu Ala Arg Glu 420<br><br>
Ser Val Val Leu<br><br>
Asn Pro His Tyr 425<br><br>
Lys Thr Met Gly 430<br><br>
Leu Tyr Gly Ser Glu Tyr Trp Thr Tyr Ser Leu Pro Arg Ala Val Gly<br>
435	440	445<br>
Ser Glu Val Ala His Gln Leu Thr Asp Ala Cys Leu Pro Ile Ser Ala<br>
450	455	460<br>
Leu Gln Ala Glu Gln Tyr Gly Leu Val Gln Gly Ile Gly Pro Arg Cys<br>
465	470	475	480<br>
Pro His Ala Phe Ser Arg Trp Leu Met Gln Gln Ala Ser Ser Ala Leu<br>
485	490	495<br>
Thr Asp Glu Lys Tyr Ala Val Ala Arg Ala Arg Lys Ala Ala Leu Asp<br>
500	505	510<br>
Ile Asp Gln Ile Thr Arg Cys Arg Glu Ala Glu Leu Ala Gln Met Gln<br>
515	520	525<br>
Leu Asp Met Val His Asn Arg His Gln Phe Ala Glu Lys Cys Arg Asn<br>
530	535	540<br>
Phe Val Leu Lys Arg Lys Thr Cys Gln Thr Pro Gln Arg Leu Met Ala<br>
545	550	555	560<br>
Pro Trp Ala Val Ala Arg Glu Ala Ala Leu Val Gly<br>
565	570<br>SEQID14<br> Lenght: 230<br> Type: PRT<br> Organism: Pseudomonas fluorescens A2-2<br>  SEQUENCE: 14<br>
Met Ile Gly Ile Val  Ile Pro Ala His Asn Glu Glu Arg His  Ile Ser<br>
15	10	15<br>
Ala Cys Leu Ala Ser Ile Gln Arg Ala Ile Ala His  Pro Ala Leu Ala<br>
20	25	30<br>
His Gln Gln Val  Gln Leu Leu Val Val  Leu Asp Ala Cys  Ser Asp Glu<br>
35	40	45<br><br>
Thr Ala Thr Arg Val Ser Ala Met Gly Val Ala Thr Leu Glu Val Ser<br>
50	55	60<br>
Val Arg Asn Val Gly Lys Ala Arg Ala Leu Gly Ala Glu Arg Leu Leu<br>
65	70	75	80<br>
Glu Val Gly Ala Gln Trp Leu Ala Phe Thr Asp Ala Asp Thr Val Val<br>
85	90	95<br>
Pro Ala Asp Trp Leu Val Arg Gln ILe Gly Phe Gly Ala Asp Ala Val<br>
100	105	110<br>
Cys Gly Thr Val Glu Val Asp Ser Trp Ser Glu Tyr Gly Glu Ser Val<br>
115	120	125<br>
Arg Ser Arg Tyr Leu Glu Leu Tyr Gln Phe Thr Glu Asn His Arg His<br>
130	135	140<br>
ILe His Gly Ala Asn Leu Gly Leu Ser Ala Asp Ala Tyr Arg Asn Ala<br>
145	150	155	160<br>
Gly Gly Phe Gln His Leu Val Ala His Glu Asp Val Gln Leu Val Ala<br>
165	170	175<br>
Asp Leu Glu Arg ILe Gly Ala Arg ILe Val Trp Thr Ala Thr Asn Pro<br>
180	185	190<br>
Val Val Thr Ser Ala Arg Arg Asp Tyr Lys Cys Arg Gly Gly Phe Gly<br>
195	200	205<br>
Glu Tyr Leu Ala Ser Leu Val Ala Glu Gly Thr Arg Glu His Ser Pro<br>
210	215	220<br>
Ala His Ala Pro ILe Gly<br>
225	230<br>  SEQ ID 15<br> Lenght: 348<br> Type. PRT<br> Organism: Pseudomonas fTuorescens A2-2<br> SEQUENCE: 15<br><br>
Met His Pro His Lys Thr Ala ILe Val Leu Ile Glu Tyr Gln Asn Asp<br>
15	10	15<br>
PIle Thr Thr Pro Gly Gly Val Phe His Asp Ala Val Lys Asp Val Met<br>
20	25	30<br>
Gln Thr Ser Asn Met Leu Ala Asn Thr Ala Thr Thr Ile Glu Gln Ala<br>
35	40	45<br>
Arg Lys Leu Gly Val Lys Ile Ile His Leu Pro Ile Arg Phe Ala Asp<br>
50	55	60<br>
Gly Tyr Pro Glu Leu Thr Leu Arg Ser Tyr Gly Ile Leu Lys Gly Val<br>
65	70	75	80<br>
Ala Asp Gly Ser Ala Phe Arg Ala Gly Ser Trp Gly Ala Glu Ile Thr<br>
85	90	95<br>
Asp Ala Leu Lys Arg Asp Pro Thr Asp Ile Val Ile Glu Gly Lys Arg<br>
100	105	110<br>
Gly Leu Asp Ala Phe Ala Thr Thr Gly Leu Asp Leu Val Leu Arg Asn<br>
115	120	125<br>
Asn Gly Ile Gln Asn Leu Val Val Ala Gly Phe Leu Thr Asn Cys Cys<br>
130	135	140<br>
Val Glu Gly Thr Val Arg Ser Gly Tyr Glu Lys Gly Tyr Asp Val Val<br>
145	150	155	160<br>
Thr Leu Thr Asp Cys Thr Ala Thr Phe Ser Asp.Glu Gln Gln Arg Ala<br>
165	170	175<br>
Ala Glu Gln Phe Thr Leu Pro Met Phe Phe Ala Asn Pro Ala Thr His<br>
180	185	190<br>
Arg Val Ser Ala Ser Thr Glu Arg Arg Ile Lys Lys Ala Ala Thr Pro<br>
195	200	205<br>
Ala Glu Ser Pro Leu Phe Cys Leu Gly His Ser Val Gly Ala Tyr Cys<br>
210	215	220<br><br><br>
Ile Ser Pro Phe Pro Asn Asp Gln Ser Ser Arg Phe Thr Ser Thr Arg<br>
225	230	235	240<br>
Leu Ile His Thr Ser Ser Leu Arg Ser Pro Val Leu Ala Trp Met Pro<br>
245	250	255<br>
Ser Ala Met Asn Leu Lys Ala Phe Phe Thr Ser Met Leu Arg Pro Ala<br>
260	265	270<br>
Phe His Val Thr Trp Ile Asn Thr Ile Leu Gly Val Val Thr Pro Arg<br>
275	280	285<br>
Tyr Pro Ala Ala Gly Thr Ser Ser Ser Leu Ala Trp Arg Leu Met Ile<br>
290	295	300<br>
Trp Asn Leu Ser Cys Ser Gly Thr Leu Ala Thr Leu Val Ile Ala Ala<br>
305	310	315	320<br>
Tyr Thr Thr Ser Pro Met Ala Val Ala Val Ser Val Glu Val Ser Ala<br>
325	330	335<br>
Ala Arg Ser Ile Arg Thr Lys Gly Met Asp Lys Ser<br>
340	345<br><br>
Comparison of new dehydroxysafracins with safracin A.<br><br><br><br><br>
We Claim:<br>
1.	An expression vector comprising isolated nucleic acid sequence represented by SEQ ID NO 1 or variants or portions thereof, the said sequence encoding for a polypeptide useful in bio-synthesis of safracin or its analogue.<br>
2.	An expression vector as claimed in claim 1, wherein the vector is a cosmid.<br>
3.	A host cell transformed with an expression vector encoding polypeptides sufficient to direct the synthesis of a safracin or safracin analogue wherein the expression vector comprising nucleic acid sequence represented by SEQ ID NO 1 or variants or portions thereof.<br>
4.	A host cell as claimed in claim 3, wherein the host cell is a microorganism<br>
5.	A host cell as claimed in claim 4, wherein the micro-organism is bacteria.<br>
6.	A recombinant bacterial cell wherein a nucleic acid sequence represented by SEQ ID NO 1 or a portion thereof is disrupted to enable the cell to produce altered levels of safracin or its analogues, relative to corresponding non-recombinant bacterial cell.<br>
7.	A recombinant cell as claimed in claim 6, wherein the disrupted nucleic acid sequence is endogenous.<br>
8.	A method of producing a bio-systhesized safracin compound or safracin analogue, said method comprising fermenting by an organism in which the copy number of the gene cluster as represented by SEQ ID NO. 1 has been increased and wherein the method comprises of:<br>
a.	inoculating a mutant strain in a suitable medium as herein described,<br>
b.	incubating the medium for a period in the range of 50 to 1 00 hours at a<br>
controlled temperature such as herein described,<br>
c.	transferring the culture obtained from step (b) to a fermentor, and<br>
d.	fermenting under agitated and aerated conditions for a period of 20 to<br>
75 hours to produce said safracin.<br>
9.	A method as claimed in claim 8, wherein mutant strain is a mutated Pseudomonas sp.<br>
10.	A method as claimed in claim 8, wherein the temperature from inoculation till 24 hours is about 27°C.<br>
11.	A method as claimed in claim 8, wherein the temperature for incubation is maintained at about 25°C beyond 24 hours.<br><br>
12.	A bio-synthesized safracin compound oblained by a method as claimed in claim 8.<br>
13.	A   bio-synthesized   safracin   compound   as   claimed   in   claim   12,   wherein   the compound is selected from any of:<br><br><br><br>
MeO<br><br><br><br><br><br><br><br><br><br>
14. A pharmaceutical composition comprising a compound as claimed in claims 12 or 1 3 and a pharmaceutically acceptable diluent, carrier or excipient.<br><br><br>
Dated this 9th   day of   June,  2005<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MDAtbXVtbnAtMjAwNS1zZXF1ZW5jZSBsaXN0aW5nLmRvYw==" target="_blank" style="word-wrap:break-word;">00600-mumnp-2005-sequence listing.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtY2FuY2VsbGVkIHBhZ2VzKDE5LTA2LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-cancelled pages(19-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxOS0wNi0yMDA3KS5kb2M=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-claims(granted)-(19-06-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtY2xhaW1zKGdyYW50ZWQpLSgxOS0wNi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-claims(granted)-(19-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoMzEtMDgtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-correspondence(31-08-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtY29ycmVzcG9uZGVuY2UoaXBvKS0oMTUtMDYtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-correspondence(ipo)-(15-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZHJhd2luZygxOS0wNi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-drawing(19-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAxKDEzLTA2LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 1(13-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAxKDE5LTA2LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 1(19-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAxKDIwLTA5LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 1(20-09-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAxOCgxOS0xMi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 18(19-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxOS0wNi0yMDA3KS5kb2M=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 2(granted)-(19-06-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAyKGdyYW50ZWQpLSgxOS0wNi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 2(granted)-(19-06-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAyNigxMy0wNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 26(13-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAzKDA5LTA2LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 3(09-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSAzKDEzLTA5LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 3(13-09-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjAwLW11bW5wLTIwMDUtZm9ybSA1KDEzLTA2LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">600-mumnp-2005-form 5(13-06-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="210474-device-for-conveying-electrodes-used-in-the-electrolytic-refining-or-electrowinning-of-metals.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="210476-a-pharmaceutical-composition-for-controlled-release-of-a-beta-lactam-antibiotic.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>210475</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>600/MUMNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>43/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Oct-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Oct-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Jun-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PHARMA MAR S.A</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CALLE DE LA CALERA 3, POLIGONO INDUSTRIAL DE TRES CANTOS, TRES CANTOS, E-28760 MADRID</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>APARICIO PEREZ</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VELASCO IGLESIAS ANA</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DE LA CALLE FERNANDO</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SCHLEISSNER SANCHEZ</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ACEBO PAIS PALOMA</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>6</td>
											<td>RODRIGUEZ RAMOS PILAR</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>7</td>
											<td>REYES BENITEZ FERNANDO</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
										<tr>
											<td>8</td>
											<td>HENRIQUEZ PELAEZ RUBEN</td>
											<td>POLLGONO INDUSTRIAL LA MINA, AVDA.DE LOS REYES,1, COLMENAR VIEJO, E-28770 MADRIED</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/52, 9/00 9/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2003/005563</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-12-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0229793.5</td>
									<td>2002-12-20</td>
								    <td>GB</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/210475-an-expression-vector-useful-in-producing-safracinor-its-analogue by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:53:19 GMT -->
</html>
