<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/210754-a-method-of-producing-a-lipolytic-enzyme-variant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:51:01 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 210754:A METHOD OF PRODUCING A LIPOLYTIC ENZYME VARIANT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PRODUCING A LIPOLYTIC ENZYME VARIANT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a substrate specificity of a lipolytic enzyme can be modified by making alterations to the amino acid sequence in a defined region of the lipolytic enzyme, so as to increase the level of a desired activity or to decrease the level of an undesired activity. Thus, the inventors have developed lipolytic enzyme variants with a modified amino acid sequence with a substrate specificity which can be tailored for specific uses.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
LIPOLYTIC ENZYME VARIANTS<br>
FIELD OF THE INVENTION<br>
The present invention relates to a method of altering the substrate specificity of a lipolytic enzyme by modifying the amino acid sequence, and to lipolytic enzyme variants obtained by such modification. The invention also relates to a screening method for lipolytic enzymes,<br>
BACKGROUND OF THE INVENTION<br>
Lipolytic enzymes (such as lipases and phospholipases) are capable of hy-drolyzing carboxylic ester bonds in a substrate to release carboxylic acids. The hy-drolytic activity on different ester bonds is important for the usefulness of the lipolytic enzyme in various industrial applications.<br>
Thus, enzymes with a high phospholipase activity are useful in a wide range of applications such as baking (US 4,567,046), filtration of wheat starch hydrolysate (US 5,264,367) and treatment of vegetable oil to reduce the content of phospholipid (US 5,264,367). For the treatment of vegetable oil, the enzyme should have a low lipase activity, i.e. a low hydrolytic activity towards ester bonds in triglycerides..<br>
WO 98/45453 indicates that an enzyme with a high hydrolytic activity on di-galactosyl diglyceride (DGDG) is useful in baking.<br>
It is well known to add a lipase to laundry detergents to aid in the removal of greasy soils (e.g. EP 258,068).<br>
The release of short-chain fatty acids as free fatty acids (FFA) may be desirable for flavor development in food products, e.g. in cheese ripening (M. Hanson, ZFL, 41 (10), 664-666 (1990)).<br>
The three-dimensional (3D) structure of several lipolytic enzymes is known, and several structures are known to contain a so-called "lid" which may be in an open or closed state covering the active site. Brady et al., Nature. 343, 767-770 (1990). Brzozowski A M et al., Nature, 351, 491 (1991). Derewenda et al., Biochemistry, 31 (5). 1532-1541 (1992).<br>
F. Hara et al., JAGGS, 74 (9), 1129-32 (1997) indicates that some lipases have a certain phospholipase activity, whereas most lipases have little or no activity on phospholipids. Thus, phospholipase activity has been described in the lipases from guinea pig pancreas, Fusarium oxysporum and Staphylococcus hyicus, and attempts have been made to relate the phospholipase activity to the structure of the<br><br>
lipase. WO 98/26057; M.D. van Kampen et al., Chemistry and Physics of Lipids, 93 (1998), 39-45; A. Hjorth et a!.. Bio- chemistry 1993, 32, 4702-4707.<br>
The prior art has described the effect on chain-length selectivity by amino acid substitutions in a lipase from Rhizopus delemar. Thus, R. D. Joerger et al.. Lipids, 29 (6). 377-384 (1994) indicates that the variants F95D. F112W and V209W have an altered preference to C4 and C8 acids. R.R. Klein et al., JAOCS, 74 (11), 1401-1407 (1997) shows that the variant V206T+F95D has a higher selectivity for C8 acid. R. R. Klein et al.. Lipids, 32 (2), 123-130 (1997) indicates that the variants V209W+F112W, V94W and F95D+F214R have a higher hydrolytic activity towards C4 and C8 acids, and suggests that structural determinants for medium-chain length specificity may reside In the distal end of the acyl binding groove.<br>
SUMMARY OF THE INVENTION<br>
The inventors have found that the substrate specificity of a lipolytic enzyme can be modified by making alterations to the amino acid sequence in a defined region of the lipolytic enzyme, so as to increase the level of a desired activity or to de-crease the level of an undesired activity. Thus, the inventors have developed lipolytic enzymes with a modified amino acid sequence (hereinafter called lipolytic enzyme variants, or variants for short) with a substrate specificity which can be tailored for specific uses.<br>
Accordingly,the invention provides a method of producing a lipolytic enzyme variant and lipolytic enzyme variants prepared by the method. The method comprises:<br>
a)	selecting a substrate and an ester bond of interest,<br>
b)	selecting a parent lipolytic enzyme,<br>
c)	selecting at least one amino acid residue in a region near the active site, near the C-terminal or in the lid region of the parent lipolytic enzyme as described below,<br>
d)	making alterations each of which is an insertion, a deletion or a substitution of the amino acid residue,<br>
e)	optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than c),<br>
f)	preparing the resulting variant,<br>
g)	testing the activity of the variant on the ester bond in the substrate, and<br>
h) selecting a variant having an altered activity on the ester bond.<br><br>
lipase. WO 98/26057; M.D, van Kampen et al., Chemistry and Physics of Lipids, 93 (1998), 39-45; A. Hjorth et a!.. Bio- chemistry 1993. 32, 4702-4707.<br>
The prior art has described the effect on chain-length selectivity by amino acid substitutions in a lipase from Rhizopus delemar. Thus, R. D. Joerger et al., Lipids, 29 (6). 377-384 (1994) indicates that the variants F95D, F112W and V209W have an altered preference to C4 and C8 acids. R. R. Klein et al., JAOCS, 74 (11), 1401.-1407 (1997) shows that the variant V206T+F95D has a higher selectivity for C8 acid. R. R. Klein et al., Lipids. 32 (2), 123-130 (1997) indicates that the variants V209W+F112W, V94W and F95D+F214R have a higher hydrolytic activity towards C4 and C8 acids, and suggests that structural determinants for medium-chain length specificity may reside In the distal end of the acyl binding groove.<br>
SUMMARY OF THE INVENTION<br>
The inventors have found that the substrate specificity of a lipolytic enzyme can be modified by making alterations to the amino acid sequence in a defined region of the lipolytic enzyme, so as to increase the level of a desired activity or to decrease the level of an undesired activity. Thus, the inventors have developed lipolytic enzymes with a modified amino acid sequence (hereinafter called lipolytic enzyme variants, or variants for short) with a substrate specificity which can be tailored for specific uses.<br>
Accordingly.the invention provides a method of producing a lipolytic enzyme variant and lipolytic enzyme variants prepared by the method. The method comprises:<br>
a)	selecting a substrate and an ester bond of interest,<br>
b)	selecting a parent lipolytic enzyme.<br>
c)	selecting at least one amino acid residue in a region near the active site, near the C-terminal or in the lid region of the parent lipolytic enzyme as described below,<br>
t<br>
d)	making alterations each of which is an Insertion, a deletion or a substitution of the amino acid residue,<br>
e)	optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than c),<br>
f)	preparing the resulting variant,<br>
g)	testing the activity of the variant on the ester bond In the substrate, and<br>
h) selecting a variant having an altered activity on the ester bond.<br><br>
Thus, in one aspect, the parent lipolytic enzyme has an alcohol binding site having a glycerol part with an sn2 position, and the amino acid alteration is within 10 A of the C atom at the sn2 posi- tion of the glycerol part of a substrate triglyceride.<br>
In another aspect, the parent lipolytic enzyme has a structure comprising a catalytic triad consisting of an active Ser, an active Asp and an active His residue, and the amino acid to be altered is either located between the active His residue of the catalytic residue and the C-terminal, or belongs to a set E defined by the following steps:<br>
i) aligning the structure of the lipolytic enzyme with Rhizomucor miehei lipase structure 4TGL comprising a catalytic triad and an inhibitor phosphorus atom (4TGL-inhP), so as to minimize the sum of squares of deviation between atoms of the catalytic triads of the two structures,<br>
ii) defining a set A consisting of atoms of the lipolytic enzyme inside a sphere of radius 18 A with center at 4TGL-inhP.<br>
iii) forming a first plane defined by 4TGL-inhP, the Ca atom of the active Ser residue of the parent lipolytic enzyme, and the Ca atom of the active Asp residue of the parent lipolytic enzyme and defining a set B as a subset of set A consisting of atoms on the same side of the first plane as the Ca atom of the active His residue of the parent lipolytic enzyme,<br>
iv) forming a second plane defined by 4TGL-inhP, the Ca atom of the active Ser residue of the parent lipolytic enzyme, and the Ca atom of the active His residue of the parent lipolytic enzyme and defining a set C as a subset of set A consisting of<br>
■<br>
atoms on the opposite side of the second plane from the Ca atom of the active Asp residue of the parent lipolytic enzyme,<br>
v) forming a set D consisting of atoms belonging to the union of sets B and C, and having a solvent accessibility of 15 or higher, and<br>
vi) forming set E consisting of amino acid residues in the structure which comprise an atom belonging to set D or an atom belonging to the union of sets B and C and located less than 3.5 A from an atom belonging to set D,<br>
In a third aspect, the lipolytic enzyme has an active site comprising an active His residue, and the alteration Is made in the amino acid sequence between the active His residue and the C-temninal.<br>
In yet another aspect of the invention, the amino acid alteration is made among the 10 amino acid residues at the C-terminal.<br>
In a further aspect, the parent lipolytic enzyme has a lid, and the alteration is made in the lid.<br><br>
The invention also provides a DNA sequence encoding the variant, an expression vector comprising the DNA sequence, a transformed host cell harboring the DNA sequence or the expression vector, and to a method of producing the variant by cultivating the transformed host cell so as to produce the variant and recovering the variant from the resulting broth. Further, the invention provides uses of the variants.<br>
The Inventors have also found that a lipolytic enzyme which has lipase and phospholipase activity as well as activity on digalactosyl diglyceride is particularly effective for use in baking, and they designed a screening method for lipolytic enzymes by testing for these activities.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
Fig. 1 shows an alignment of lipase sequences.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Altered activity on selected ester bond in substrate<br>
Compared to the parent lipolytic enzyme, the invention aims to alter the<br>
activity on at least one selected ester bond in at least one substrate, i.e. to increase<br>
a desired activity, decrease an undesired activity or to change the substrate<br>
specificity by decreasing the ratio of an undesired activity to a desired activity.	^<br>
Thus, an enzyme with increased phospholipase activity may be useful, e.g., in baking or In purification of vegetable oil. It may be desired to increase the hydro-lytic activity on digalactosyl-diglyceride (DGDG) for use in baking.<br>
It may be desired to increase the lipase activity for any industrial use where lipases are used. For use in detergents or baking it may be desired to increase the activity on long-chain (C16-C20) triglycerides, and It may be desired to increase the specificity for long-chain fatty acids by decreasing the ratio of activity on short-chain or medium-chain (C4-C8) fatty acids to the activity on long-chain fatty acids.<br>
For use in, or for use in flavor development in food products (such as cheese ripening) it may be desired to increase the lipase activity on short-chain or medium-chain (C4-C8) triglycerides.<br><br>
For use as a phospholipase in purification of vegetable oil, it may be desired to decrease the ratio of lipase activity on long-chain (C16-C20) triglycerides to the phospholipase activity.<br>
Parent lipolytic Enzyme<br>
The lipolytic enzyme to be used in the present invention is one that can hy-drolyze ester bonds. Such enzymes include, for example, lipases, such as triacyl-glycerol lipase (EC 3.1.1.3), lipoprotein lipase (EC 3.1.1.34), monoglyceride lipase (EC 3.1.1.23), lysophospholipase, ferulic acid esterase and esterase (EC 3.1.1.1, EC 3.1.1.2). The numbers in parentheses are the systematic numbers assigned by the Enzyme Commission of the Intemational Union of Biochemistry in accordance with the type of the enzymatic reactivity of the enzyme.<br>
The parent lipolytic enzyme may be prokaryotic, particularly a bacterial enzyme, e.g. from Pseudomonas. Examples are Pseudomonas lipases, e.g. from P. cepacia (US 5,290,694, pdb file 10IL), P. glumae (N Frenken et al. (1992), Appl. En-vir. Microbiol. 58 3787-3791, pdb files 1TAH and 1QGE), P. pseudoalcaligenes (EP 334 462) and Pseudomonas sp. strain SD 705 (FERM BP-4772) (WO 95/06720, EP 721 981, WO 96/27002, EP 812 910). The P. glumae lipase sequence is identical to the amino acid sequence of Chromobacterium viscosum (DE 3908131 Al). Other examples are bacterial cutinases, e.g. from Pseudomonas such as P. mendocina (US 5,389,536) or P. putida (WO 88/09367).<br>
Altematively, the parent lipolytic enzyme may be eukaryotic, e.g. a fungal lipolytic enzyme such as lipolytic enzymes of the Humicola family and the Zygomycetes family and fungal cutinases.<br>
Examples of fungal cutinases are the cutinases of Fusarium solani pisi (S. Longhi et al.. Journal of Molecular Biology, 268 (4), 779-799 (1997)) and Humicola /nso/ens (US 5.827,719V<br><br>
terase from A. niger (Y09330), li- pase/ferullc acid esterase from A.<br>
tubingensis   (Y09331),   lipase   from   A. tubingensis	(WO .       98/45453),<br>
lysophosphoiipase from A. niger (WO 98/31790), lipase from F. so/an//liaving an isoelectric point of 6.9 and an apparent molecular weight of 30 kDa (WO 96/18729).<br>
The Zygomycetes family comprises lipases having at least 50 % homology with the lipase of Rhizomucor miehei (PI 9515). This family also includes the lipases from Absidia reflexa, A. sporophora, A. corymbifem, A. blakesleeana, A. griseola (ail described in WO 96/13578 and WO 97/27276) and Rhizopus oryzae (P21811). Numbers in parentheses indicate publication or accession to the EMBL, GenBank, GeneSeqp or Swiss-Prot databases.<br>
It is of particular interest to derive a variant with phospholipase activity from a parent lipolytic enzyme having no or very little phospholipase activity, e.g. corresponding to a ratio of phospholipase activity to lipase activity below 0.1 PHLU/LU or below 50 PHLU/mg.<br>
Alteration near alcohol binding site<br>
As already stated, the amino acid sequence of the parent lipolytic enzyme may be modified at a position which near the glycerol part of a substrate triglyceride. This region will be refen-ed to as the "alcohol binding site" of the lipase; it is described in Brzozowski A M et al., Nature, 351: 491 (1991); Uppenberg et al., Biochemistry, 1995, 34, 16838-16851; A. Svendsen. Inform, 5(5), 619-623 (1994):<br>
For the Rhizomucor miehei lipase, the extent of the alcohol binding site can be found from the PDB file "5tgl.pdb" available in Structural Classification of Proteins (SCOP) on the Internet, at http://www.rcsb.org/pdb/, showing the complex with the inhibitor n-hexylphosphonate ethyl ester which mimics the substrate. It is descrihed<br><br>
267. The following are within 8 A: 81-83, 144,  257-258,  265-267.  and  the fol<br>
lowing within 6 A: 82. 144. 257, 266.	<br>
In the Humicola lanuginosa lipase, the following positions are within 10 A of the sn2 position: 18. 21, 81-85. 89, 145-148, 172, 201, 203, 255-256. 258-260, 264-267, The following are within 8 A: 82-84. 89, 146. 258-259, 265-267, and the following within 6 A: 83.146, 258, 266.<br>
Alteration near catalytic triad<br>
As already stated, in one aspect the parent lipolytic enzyme has a structure comprising a catalytic triad consisting of an active Ser, an active Asp and an active His residue, and the amino acid to be altered belongs to a set defined by a certain procedure described above. The structure may be an open or a closed structure, and it may or may not include a substrate or an inhibitor.<br>
The procedure is conveniently performed by use of software such as MSPs Insight l(. It involves alignment with 4TGL, a crystal structure of the lipase from Rhi-zomucor miehei inhibited irreversibly by diethyl p-nitrophenyl phosphate. This is available in Structural Classification of Proteins (SCOP) on the Internet, at http://www.rcsb.org/pdb/, and is described in Derewenda et al. (supra). The Rhi-zomucor miehei lipase comprises a catalytic triad consisting of the amino acid residues S144, D203 and H 257.<br>
For the Humicola lanuginosa lipase, the structure Itib may be used; it is available in Structural Classification of Proteins (SCOP) on the Internet. Using this structure, the set defined by the procedure includes the following positions: 10-23. 26, 40, 55-64, 80-87, 116-117. 119, 145-149, 151, 168. 170.194,196-201, 220-222. 224-227, and 254-269.<br><br>
Alteration near C-terminal<br>
As stated above, one or more alterationsmay be made within 10 amino acid positions from the C-terminal of the mature protein,<br>
or at positions corresponding to such positions in the H. lanuginosa lipase, i.e. positions 260-269 of the H. lanuginosa lipase. Corresponding positions may be found by alignment of the two sequences as described later in this specification.<br>
The lipolytic enzyme variant may be truncated by deleting amino acid residues corresponding to the first 1. 2, 3, 4, 5 or 6 positions at the C-terminal. A truncated variant may have improved thermostabllity.<br>
Alternatively, the variant may carry a peptide extension at the C-terminal and/or the N-terminal. The C-terminal extension may consist of 1-10 amino acid residues, e.g. A, P, AG, DG, PG, AGG. PVGF, AGRF, PRGF, AGGF or AGGFS; or it may consist of 40-50 residues, e.g., consisting of the 48 C-terminal residues of the Fusarium oxysporum lipase AGGFSWRRYRSAESVDKRATMTDAELEKKLNSY-VQMDKEYVKNlNQARS. The C-terminal extension may increase the phospholipase activity.<br>
Some alterations in the region overiapping with the alcohol binding site are described below.<br>
A specific alteration is a substitution at a position corresponding to G266 in the Humicola lanuginosa lipase, specifically with an amino acid of intermediate size, e.g. A, C, D, N, L, I, S, T, P or V. Such alteration alone has been found sufficient to increase the phospholipase activity.<br>
Other specific alterations are such that alter the tertiary structure, e.g. by introducing bulky side chains or by disrupting the bond angles, e.g. by introducing Pro. Such alterations may be made at positions con-esponding to positions G263, L264, 1265. T267 or L269 In the Humicola lanuainosa lipase. Some specific substitutions<br><br>
ginosa lipase with a neutral or positively charged amino acid, e.g. a substitution .corresponding to G91A,T, L93K, N94D, D96S,W,G. L97Q, K98D,F.E and/or E99K,D.<br>
Specifically, a variant with an alteration in the lid region also contains one or more alterations near the catalytic triad, near the substrate binding site or near the C-terminal.<br>
Lipolytic enzyme variants<br>
The lipolytic enzyme variant of the invention comprises one or more alterations of an amino acid residue in any of the regions described above. Each alteration may be a deletion or a substitution of the amino acid residue, or it may be an Insertion before or after the amino acid residue. If the amino acid residue is at the C-terminal, the insertion may be a C-terminal extension. An insertion typically consists of 1-5 amino acid residues, e.g. 1-2, and a C-terminal extension may consist of 1-50 or 2-10 amino acid residues.<br>
The total number of alterations in the above regions is typically not more than 20, e.g. not more than 10 or not more than 5, and there may be as little as 1 or 2 alterations In the above regions.<br>
In addition, the lipolytic enzyme variant of the invention may optionally include other modifications of the parent enzyme, typically not more than 10, e.g. not more than 5 such modifications.<br>
The variant generally has a homology with the parent lipolytic enzyme of at least 80 %, e.g. at least 85 %, typically at least 90 % or at least 95 %.<br>
The variant of the invention may further comprise a peptide extension at the N-terminal, e.g. consisting of 1-15 (particularly 4-10) amino acid residues, and specifically comprising 1, 2 or 3 positively charged amino acids. Some specific N-<br><br><br>
Some specific alterations are substitutions coo'esponding to thie<br>
following     In    H.    lanuginosa    lipase:	Y21V/I/L/A/G/M/W/P/F/N/Q/S/T,<br>
V60V/I/L/A/G/M/W/P/F/N/Q/S/T.	G61V/l/L/A/G/M/W/P/F/N/Q/S/r,<br>
D62E/A/V, S83T, R84K/L/W. P256A. G263E,Q,R,F. L264A,C,P.F.G,I, I265L.N,F G266D/E or T267A,Q,P.S,E, or an insertion corresponding to T267GS or T267GL.<br>
To alter the activity towards short-chain (C4-C8) fatty acids in triglycerides, alterations may be made at positions con-esponding to Y21, E56, D57, V60, G61, D62, R81, S83, R84, L259, Y261 or G266, e.g. a substitution con-esponding to Y21V/I, V60G, D62E/AA/, S83T, R84KIIAN orG266D/E.<br>
To Increase the activity for DGDG, alterations may be made at positions corresponding to Y21, G23, N26, D57, D62, R81, S83, R84, S85. G266, T267 or L269; e.g., two or more such alterations may be made, e.g. together with one or more alterations in the lid region. To increase the phospholipase activity, alterations may be made at positions corresponding to R81, R84, S85, or 263-267, e.g. G266 orT267.<br>
To prepare variants of a Pseudomonas lipase, amino acid modifications may be made at positions con-esponding to 12-13, 16-34, 45-52, 59-66, 68, 86-87, 107-109, 111, 143-153, 155, 157-158. 207-212, 228, 230, 242-249, 264, 279-280, 282-297, 301-302, 304-305, 307-308 in the P. cepacia lipase, particulariy L17/L17, T18/A18, Y29A'29, L287/L286, E289/E288, 1290/1289, Q292/Q291 or L293/L292 In the P. cepacia/P. glumae lipase.<br>
Specific variants of the H. lanuginosa lipase are disclosed In the examples.<br>
Con-esponding alterations may be made In other parent lipolytic enzymes. Further<br>
variants may be derived from these by omitting amino acid modifications at positions<br>
1, 106, 186, 225, 232, 237, 239 or 274. Variants with 274S may optionally have a<br>
further	C-terminal	extension	of	WRRYR-<br>
SAESVDKRATMTDAELEKKLNSYVQMDKEYVKNNQARS (corresponding to the C-<br><br>
is   uncertain.   SPIRR   indicates   an   N- terminal extension, D266 may refer to the position or to substitution with any amino acid (except D).<br>
E1SPPCGRRP or SPPCGRRP(- E) indicates a substitution of E1 with SPPCGRRP, I.e. a peptide addition at the N-terminal  T267GS indicates a substitution of T267 with GS, or in other words the substitution T267G and an insertion of S between G267 and 0268.<br>
Homology and alignment<br>
For purposes of the present invention, the degree of homology may be suitably determined by means of computer programs known in the art, such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970). Journal of Molecular Biology, 48, 443-45), using GAP with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1.<br>
In the present invention, corresponding (or homologous) positions in the lipase sequences of Rhizomucor miehei (rhimi), Rhizopus delemar (rhidi), Thermomy-ces lanuginosa (former; Humicola lanuginosa) (SP400). Penicillium camembertii (Pol) and Fusarium oxysporum (FoLnpll), are defined by the alignment shown in Figure 1.<br>
To find, the homologous positions In lipase sequences not shown in the alignment, the sequence of interest is aligned to the sequences shown in Figure 1. The new sequence is aligned to the present alignment In Fig. 1 by using the GAP alignment to the most homologous sequence found by the GAP program. GAP is provided in the GCG program package (Program Manual for the Wisconsin Package,<br><br>
By the PHLU method described later in this specification, the variant may have a phospholipase activity of at least 100 PHLU/mg (nig of pure enzyme protein), particularly at least 500 PHLU/mg. The variant has a ratio of phospholipase activity to lipase activity (both measured at pH 7) of at least 0.1 PHLU/LU, e.g. at least 0.5, particularly at least 2.<br>
The variants of the invention may have the ability to hydrolyze intact phospholipid, as demonstrated by the PHLU method. They may have A1 and/or A2 activity, so they may be able to hydrolyze one or both fatty acyl groups in the phospholipid.<br>
pH optimum<br>
Many variants of the Humicola lanuginosa lipase have an alkaline pH optimum for lipase activity and an acid pH optimum for phospholipase activity (e.g. pH 9-10 for lipase and pH 4-6 for phospholipase). Such variants can be used at acid pH (e.g. in oil degumming, described later), as phospholipases with very low concomitant lipase activity.<br>
However, some variants of the Humicola lanuginosa lipase which include the substitution G266D,E have pH optima for both lipase and phospholipase activities around pH 5-6. Such variants may be used at acid pH when both lipase and phospholipase activities are desired, e.g. in baking.<br>
Themiostabiiity<br>
The thermostability of the variant can conveniently be evaluated by means of Differential Scanning Calorimetry (DSC). Depending on exact mutations, the variants of the invention generally have similar or slightly lower thermostability than the parent lipolytic enzyme.<br><br>
Improvement of filtration<br>
A variant with lysophospholipase activity can be used to improve tiie filterabllity of an aqueous solution or slurry of carbohydrate origin by treating it with the variant. This is particularly- applicable to a solution or slurry containing a starch hydrolysate, especially a wheat starch hydrolysate since this tends to be difficult to filter and to give cloudy filtrates. The treatment can be done In analogy with EP 219,269 (CPC International).<br>
Vegetable oil treatment<br>
A variant with phospholipase activity can be used in a process for reducing the content of phospholipid In an edible oil, comprising treating the oil with the variant so as to hydrolyze a major part of the phospholipid, and separating an aqueous phase containing the hydrolyzed phospholipid from the oil. This process is applicable to the purification of any edible oil which contains phospholipid, e.g. vegetable oil such as soy bean oil, rape seed oil and sunflower oil. The treatment may be carried out at acid pH, e.g. pH 3-5, Advantageously, a variant can be selected so as to have a high phospholipase activity and a low lipase activity at low plrl, due to different pH optima of the two activities.<br>
The process for oil treatment can be conducted according to principles known in the art, e.g. in analogy with US 5,264,367 (Metallgesellschaft, Rohm); K. Dahike &amp; H. Buchold, INFORM, 6 (12), 1284-91 (1995); H. Buchold, Fat Sci. Tech-nol., 95 (8), 300-304 (1993); JP-A 2-153997 (Showa Sangyo); or EP 654,527 (Metallgesellschaft, Rohm).<br>
Miscellaneous uses of phospholipase<br>
A variant with phospholipase activity can be used to prepare lysophosphol-ipid (e.g. lyso-lecithin) by treating the con-esponding phospholipid with the variant,<br><br><br>
Baking<br>
A variant with phospholipase arid/or DGDGase activity can be used in the preparation of dough, bread and cal<es e.g. to increase dough stability and handling properties or improve the elasticity of bread cake. thus variant can be used in a process for making comprising adding ingredients kneading baking make bread. this done analogy with us hakko jp-a corp. ep wo nordisk></es>
It is particulariy advantageous to use the variant together with an. anti-staling endo-amylase and optionally also to add a phospholipid, to reduce-staling of the bread and particulariy to improve softness of the bread in the first 24 hours after baking. The endo-amylase may be a maltogenic a-amylase (e.g. from Bacillus sp., such as Novamyl® from Novo Nordisk) or a fungal or bacterial a-amylase, e.g. from As-perglllus-or Bacillus, particulariy A. oryzae, B. licheniformis or S. amyloliquefaclens.<br>
In baking, the variant may have a low activity on short-chain.or medium-chain (C4-C8), e.g. corresponding to a SLU/LU ratio above 3. The use of such a variant may avoid or suppress the development of an undesired flavor due to the release of short-chain fatty acids. The variant may have activity on triglycerides and phospholipid as well as DGDG.<br>
Cheese flavor<br>
A variant with activity towards short-chain fatty acyl groups may be used to release free fatty acids (FFA) for flavor development in food products, e.g. in cheese ripening, e.g. as described in M. Hanson, ZFL. 41 (10), 664-666 (1990)).<br>
Lipolytic enzyme variants with increased release of short chain fatty com-<br>
pared to long chain fatty acids from milk fat are useful in cheese production e.g for<br><br>
Use in detergent<br>
The variant nnay be used as a detergent additive, e.g. at a concentration (expressed as pure enzyme protein) of 0.001-10 (e.g. 0.01-1) mg per gram of detergent or 0.001-100 (e.g. 0.01-10) mg per liter of wash liquor.<br>
In detergents, the variant may have a high activity on long-chain triglycerides (C16-C20) to improve the removal of fatty soiling. The variant may have phospholipase activity. The variant may have low activity towards short-chain (C4-C8) fatty acids in triglycerides, e.g. corresponding to a SLU/LU ratio above 10. The use of such a variant may avoid or suppress the development of an undesired odor due to the release of short-chain fatty acids.<br>
Variants having both lipase and phospholipase activity at alkaline pH may be used in detergents.<br>
Detergent composition<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations. In a laundry detergent, the variant may be effective for the removal of fatty stains, for whiteness maintenance and for dingy cleanup. A laundry detergent composition may be formulated as described in WO 97/04079. WO 97/07202, WO 97/41212, PCT/DK WO 98/08939 and WO 97/43375.<br>
The detergent composition of the invention may particulariy be formulated for hand or machine dishwashing operations, e.g. as described in GB 2,247,025 (Unilever) or WO 99/01531 (Procter &amp; Gamble). In a dishwashing composition, the variant may be effective for removal of greasy/oily stains, for prevention of the staining /discoloration of the dishware and plastic components of the dishwasher by highly<br><br>
When included therein the detergent will usually contain from about 1% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsuifonate, alkyl sulfate (fatty alcohol sulfate), alcohol ethoxysuifate, secondary alkanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuc-cinic acid or soap.<br>
When included therein the detergent will usually contain from about 0.2% to about 40% of a non-Ionic surfactant such as alcohol ethoxylate, nonyl-phenol ethoxy-late, alkylpolyglycoside, alkyldimethylamine-oxide, ethoxylated fatty acid monoetha-nol-amide, fatty acid monoethanolamlde, polyhydroxy alkyl fatty acid amide, or N-acyl N-aikyI derivatives of glucosamine ("glucamides").<br>
The invention also provides a detergent additive comprising the variant of the invention. The detergent additive as well as the detergent composition may comprise one or more other enzymes such as a protease, a lipase, a cutinase, an amylase, a carbohydrase, a ceilulase, a pectinase, a mannanase, an arabinase, a galactanase, a xylanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme(s) should be compatible with the selected detergent, (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.), and the enzyme(s) should be present in effective amounts.<br>
Proteases: Suitable proteases Include those of animal, vegetable or microbial origin. Microbial origin is prefenred. Chemically modified or protein engineered mutants are included. The protease may be a serine protease or a metallo protease, e.g. an alkaline microbial protease or a trypsin-like protease. Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carisberg, subtilisin 309, subtilisin 147 and subtilisin 168 (described in WO 89/06279). Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine<br><br>
include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonium, e.g. the fungal cellulases produced from Humicola insolens, Myceliophthora thermophila and Fusarium oxysporum disclosed in US 4.435.307, US 5.648,263; US 5,691.178. US 5.776.757 and WO 89/09259.<br>
Especially suitable.cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellulases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/29397, WO 98/08940. Other examples are cellulase variants such as those described in WO 94/07998, EP 0 531 315. US 5,457.046. US 5,686,593, US 5,763,254. WO 95/24471, WO 98/12307 and PCT/DK98/00299.<br>
Commercially available cellulases include Celluzyme ®, and Carezyme ® (Novo Nordisk A/S), Clazinase ®, and Puradax HA ® (Genencor International Inc.), and KAC-500(B)® (Kao Corporation).<br>
Peroxidases/Oxidases: Suitable per-oxidases/oxidases include those of plant, bac-terial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g, from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257.<br>
Commercially available peroxidases include Guardzyme ® (Novo Nordisk A/S).<br>
The detergent enzyme(s) may be included in a detergent composition by adding separate additives containing one or more enzymes, or by adding a combined additive comprising ail of these enzymes. A detergent additive of the invention, i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Specific detergent additive formulations are granulates, in particular non-dusting granulates liquids in particular stabilized liquids, or slurries.<br><br>
tic acid or boric acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent may contain 0-65 % of a detergent builder or complexing agent such as zeolite, diphosphate, tripho-sphate, phosphonate, carbonate, citrate, nitrilotriacetic acid, ethylenediaminetetraacetic acid, diethylenetri-aminepen-taacetic acid, alkyl- or alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst).<br>
The detergent may comprise one or more polymers. Examples are car-boxymethylcellulose, poly(vinyl-pyn'olidone), poly (ethylene glycol), poly(vinyl alcohol), poly(vinylpyridine-N-oxide), poly(vinylimidazole), polycarboxylates such as poly-acrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br>
The detergent may contain a bleaching system which may comprise a H202 source such as perborate or percarbonate which may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxyben-zenesul-fonate. Alternatively, the bleaching system may comprise peroxyacids of e.g.<br>
*<br>
the amide, imide, or sulfone type.<br>
The enzyme(s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g., a poiyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g., an aromatic borate ester, or a phenyl boronic acid derivative such as 4-formyjphenyl boronic acid, and  he composition may be formulated as described in e.g. WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional detergent ingredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil redeposition agents, dyes,<br><br>
Methods for preparing enzyme variants<br>
The enzyme variant of the invention can be prepared by methods known in the art, e.g. as described in WO 97/04079 (Novo Nordisk). The following describes methods for the cloning of enzyme-encoding DNA sequences, followed by methods for generating mutations at specific sites within the enzyme-encoding sequence.<br>
Cloning a DNA sequence encoding a enzyme<br>
The DNA sequence encoding a parent enzyme may be Isolated from any cell or microorganism producing the enzyme in question, using various methods well known in the art. First, a genomic DNA and/or cDNA library should be constructed using chromosomal DNA or messenger RNA from the organism that produces the enzyme to be studied. Then, if the amino acid sequence of the enzyme is known, labeled oligonucleotide probes may be synthesized and used to identify enzyme-encoding clones from a genomic library prepared from the organism in question. Alternatively, a labeled oligonucleotide probe containing sequences homologous to another known enzyme gene could be used as a probe to identify enzyme-encoding clones, using hybridization and washing conditions of lower stringency.<br>
Yet another method for identifying enzyme-encoding clones would involve Inserting fragments of genomic DNA into an expression vector, such as a plasmid, transfomiing enzyme-negative bacteria with the resulting genomic DNA library, and then plating the transformed bacteria onto agar containing a substrate for enzyme (/.e. maltose), thereby allowing clones expressing the enzyme to be identified.<br>
Alternatively, the DNA sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamidite method described S.L Beaucage and M.H. Caruthers, (1981), tetrahedron Letters 22, p. ip^Q-iP«Q  nrthP mpthnrj described bv Matthes et al.. M984V EMBO J. 3, p. 801-<br><br>
Site-directed mutagenesis<br>
Once a enzyme-encoding DNA sequence has been isolated, and desirable sites for mutation identified, muta- tions may be introduced using syntlietic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites. In a specific method, a single-stranded gap of DNA, the enzyme-encoding sequence, is created in a vector carrying the enzyme gene. Then the synthetic nucleotide, bearing the desired mutation, is annealed to a homologous portion of the single-stranded DNA. The remaining gap is then filled in with DNA polymerase I (Klenow fragment) and the construct is ligated using T4 ligase, A specific exampje of this method is described in Morinaga et al.,'(1984), Biotechnology 2, p. 646-639. US 4,760,025 discloses the introduction of oligonucleotides encoding multiple mutations by performing minor alterations of the cassette. However, an even greater variety of mutations can be introduced at any one time by the Morinaga method, because a multitude of oligonucleotides, of various lengths, can be introduced.   '<br>
Another method for introducing mutations into enzyme-encoding DNA se-quences is described in Nelson and Long, (1989), Analytical Biochemistry 180, p. 147-151. It involves the 3-step generation of a PCR fragment containing the desired mutation introduced by using a chemically synthesized DNA strand as one of the primers in the PCR reactions. From the PCR-generated fragment, a DNA fragment carrying the mutation may be isolated by cleavage with restriction endonucleases and reinserted into an expression plasmid.   •<br>
Further, Sierks. et al., (1989) "Site-directed mutagenesis at the active site Trp120 oi Aspeiyillus awamori glucoamylase. Protein Eng., 2, 621-625; Sierks et al., (1990), "Catalytic mechanism of fungal glucoamylase as defined by mutagenesis of Asp176, Glu179 and Glu180 in the enzyme from Aspergillus awamori. Protein Eng.<br><br>
Expression vector<br>
The recombinant expression vector carrying the DNA sequence ^ encoding a enzyme variant of the invention may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which it is to be introduced. The vector may be one which, when introduced into a host cell, is integrated into the host ceil genome and replicated together with the chromosome(s) into which it has been integrated. Examples of suitable expression vectors include pMT838.<br>
Promoter<br>
In the vector, the DNA sequence should be operably connected to a suitable promoter sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the DNA sequence encoding a enzyme variant of the invention, especially in a bacterial host, are the promoter of the lac operon of Ecoli, the Streptomyces coelicolor agarase gene dagA promoters, the promoters of the Bacillus licheniformis a-amylase gene (amyL), the promoters of the Bacillus stearothermophllus maltogenic amylase gene (amyM), the promoters of the Bacillus amyloliquefaciens a-amylase (amyQ), the promoters of the Bacillus subtilis xylA and xylB genes etc. For transcription in a fungal host, examples of useful promoters are those derived from the gene encoding A. oryzae TAKA amylase, the TPI (triose phosphate isomerase) promoter from S. cerevisiae (Alber et al. (1982), J. Mol. Appl. Genet 1. p. 419-434. Rhizomucor miehei aspartic proteinase. A. n/ger neutral a-amylase, A. niger acid stable a-amylase, A. niger glu-coamyiase, Rhizomucor miehei lipase, A. oryzae alkaline protease, A. oryzae triose phosphate isomerase or A nidulans acetamidase.<br><br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the dal genes from 6. subtHis or B. licheniformis, or one which confers antibiotic resistance such as am-picillin, kanamycin, chloramphenicol or tetracyclin resistance. Furthermore, the vector may comprise Aspergillus selection markers such as amdS, argB, niaD and sC, a marker giving rise to hygromycin resistance, or the selection may be accomplished by co-transformation, e.g. as described in WO 91/17243.<br>
The procedures used to ligate the DNA construct of the invention encoding a enzyme variant, the promoter, terminator and other elements, respectively, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al.. Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor, 1989).<br>
Host Ceils<br>
The cell of the invention, either comprising a DNA construct or an expression vector of the invention as defined above, is advantageously used as a host cell in the<br>
*<br>
recombinant production of a enzyme variant of the invention. The cell may be transformed with the DNA construct of the invention encoding the variant, conveniently by integrating the DNA construct (in one or more copies) in the host chromosome. This integration is generally considered to be an advantage as the DNA sequence is more likely to be stably maintained in the cell. Integration of the DNA constructs into the host chromosome may be performed according to conventional methods, e.g. by homologous or heterologous recombination. Alternatively, the cell may be transformed with an expression vector as described above in connection with the different types of host ceils.<br>
The cell of the invention may be a cell of a higher organism such as a mam-<br><br>
The host cell may also be a filamentous fungus e.g. a strain belonging to a species of Aspergillus, such as Aspergillus oryzae or Aspergillus niger, or a strain of Fusarium, such as a strain of Fusarium oxysporium, Fusarium graminearum (in the perfect state named Gribberella zeae, previously Sphaena zeae, synonym with Gibberella roseum and Gibberella roseum f. sp. cerealis), or Fusarium sulphureum (in the prefect state named Gibberella puricaris, synonym with Fusarium trichothecioides, Fusarium bactridioides, Fusarium sambucium, Fusarium roseum, and Fusarium roseum van graminearum), Fusarium cerealis (synonym with Fusarium crokkwellnse), or Fusarium venenatum.<br>
In a specific embodiment of the invention the host cell is a protease deficient of protease minus strain.<br>
This may for instance be the protease deficient strain Aspergillus oryzae JaL 125 having the alkaline protease gene named "alp" deleted. This strain is described in WO 97/35956 (Novo Nordisk).<br>
Filamentous fungi cells may be transformed by a process involving protoplast formation and transfonnation of the protoplasts followed by regeneration of the cell wall in a manner known per se. The use of Aspergillus as a host micro-organism is described in EP 238 023 (Novo Nordisk A/S), the contents of which are hereby incorporated by reference.<br>
Method of producing the enzyme variant of the invention<br>
The enzyme variant of the invention may be produced by a method qompris-ing cultivating a host cell under conditions conducive to the production of the variant and recovering the variant from the cells and/or culture medium.<br>
The medium used to cultivate the cells may be any conventional medium suitable for growing the host cell In question and obtaining expression of the enzyme<br>
variant of the invention. Suitable media are available from commercial suppliers or<br><br>
Expression of variant in plants<br>
The present invention also relates to a transgenic plant, plant part or plant cell which has been transformed with a DNA sequence encoding the variant of the Invention so as to express and produce this enzyme in recoverable quantities. The enzyme may be recovered from the plant or plant part. Alternatively, the plant or plant part containing the recombinant enzyme may be used as such.<br>
The transgenic plant can be dicotyledonous or monocotyledonous, for short a dicot or a monocot. Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa), forage grass such as festuca, lolium, temperate grass, such as Agrostis, and cereals, e.g. wheat, oats, rye, barley, rice, sorghum and maize (com).<br>
Examples of dicot plants are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis thaliana.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, and tubers. In the present context, also specific plant tissues, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes and cytoplasm are considered to be a plant part. Furthemiore, any plant cell, whatever the tissue origin, is considered to be a plant part.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing the variant of the invention may be constructed in accordance with methods known in the art. In short the plant or plant cell is constructed by incorporating one or more expression constructs encoding the variant of the invention into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell.<br><br>
ducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific tissue or plant part such as seeds or leaves. Regulatory sequences are eg described by Tague et al. Plant, Phys., 86, 506, 1988.<br>
For constitutive expression the 35S-CaMV promoter may be used (Franck et a!., 1980. Cell 21: 285-294). Organ-specific promoters may eg be a promoter from storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet. 24:275-303), or from metabolic sink tissues such as meris-tems (Ito et a!., 1994. Plant Mol. Biol. 24: 863-878), a seed specific promoter such as the glutelin, prolamin, globulin or albumin promoter from rice (Wu et al.. Plant and Cell Physiology Vol. 39, No. 8 pp. 885-889 (1998)), a Vicia faba promoter from the legumin B4 and the unknown seed protein gene from Vicia faba described by Conrad U. et al, Joumal of Plant Physiology Vol. 152, No. 6 pp. 708-711 (1998). a promoter from a seed oil body protein (Chen et al., Plant and cell physiology vol. 39, No. 9 pp. 935-941 (1998); the storage protein napA promoter from Brassica napus, or any other seed specific promoter known in the art, eg as described in WO 91/14772. Furthermore, the promoter may be a leaf specific promoter such as the rbcs promoter from rice or tomato (Kyozuka et al.. Plant Physiology Vol. 102, No. 3 pp. 991-1000 (1993), the chlorella vinjs adenine methyltransferase gene promoter (Mitra, A. and Higgins, DW, Plant Molecular Biology Vol. 26, No. 1 pp. 85-93 (1994), or the aldP gene promoter from rice (Kagaya et al.. Molecular and General Genetics Vol. 248, No. 6 pp. 668-674 (1995), or a wound inducible promoter such as the potato pin2 promoter (Xu et al, Plant Molecular Biology Vol. 22, No. 4 pp. 573-588 (1993).<br>
A promoter enhancer element may be used to achieve higher expression of the enzyme in the plant. For instance, the promoter enhancer element may be an in-tron which is placed between the promoter and the nucleotide sequence encoding<br>
the enzyme. For instance Xu et al on cit disclose the use of the first introp of the<br><br>
Plant Mol. Biol. 19: 15-38), however it can also be used for transforming monocots, although other transformation methods are generally used for these plants. Presently, the method of choice for generating transgenic monocots is particle bombardment (microscopic gold or tungsten particles coated with the trans-forming DNA) of embryonic calli or developing embryos (Christou, 1992. Plant J. 2: 275-281; Shimamoto, 1994. Cun-. Opin. Biotechnol. 5: 158-162; Vasil et al., 1992. Blo/Technology 10: 667-674). An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh S, et al., Plant Molecular biology Vol. 21, No. 3 pp. 415-428 (1993).<br>
Following transformation, the transfomriants having incorporated the expression construct are selected and regenerated into whole plants according to methods well-knovwi in the art.<br>
MATERIALS AND METHODS<br>
Lipase activity on tributyrin (LU)<br>
A substrate for lipase is prepared by emulsifying tributyrin (glycerin tribu-tyrate) using gum Arabic as emuisifier. The hydrolysis of tributyrin at 30 "C at pH 7 is followed in a pH-stat titration experiment. One unit of lipase activity (1 LU) equals the amount of enzyme capable of releasing 1 μmol butyric acid/min at the standard conditions.<br>
Lipase activity on triolein (SLU)<br>
The lipolytic activity may be determined using olive oil as substrate. In this SLU method, the lipase activity is measured at 30  C and pH 9 with a stabilized olive oil emulsion (Sigma catalog No. 800-1) as the substrate, in a 5 mM<br><br>
Phospholipase activity fPHLU)<br>
Phospholipase activity (PHLU) is measured as the release of free fatty acids from lecithin. 50 ΜI 4% L-alpha- phosphatidylcholine (plant lecithin from Avanti). 4 % Triton X-100, 5 mM CaCb In 50 mM HEPES. pH 7 is added 50 μl enzyme solution diluted to an appropriate concentration in 50 mM HEPES, pH 7. The samples are incubated for 10 min at 30 °C and the reaction stopped at 95 °C for 5 min prior to centrifugation (5 min at 7000 rpm). Free fatty acids are determined using the NEFA C kit from Wako Chemicals GmbH; 25 μl reaction mixture is added 250 pi Reagent A and incubated 10 min at 37 °C. Then 500 μl Reagent B is added and the sample is incubated again, 10 min at 37 °C. The absorption at 550 nm is measured using an HP 8452A diode array spectrophotometer. Samples are run in at least in duplicates. Substrate and enzyme blinds (preheated enzyme samples (10 min at 95 °C) + substrate) are included. Oleic acid is used as a fatty acid standard. 1 PHLU equals the amount of enzyme capable of releasing 1 pmol of free fatty acid/min at these conditions.<br>
Phospholipase activitv (LEU)<br>
Lecithin is hydrolyzed under constant pH and temperature, and the phospholipase activity is determined as the rate of titrant (0.1 N NaOH) consumption during neutralization of the liberated fatty acid.<br>
The substrate is soy lecithin (L-a-Phosphotidyl-Choline), and the conditions are pH 8.00,40.0°C, reaction time 2 min. The unit is defined relative to a standard.<br>
»<br>
Phospholipase monolayer assay<br>
On a thoroughly purified surface of a buffer solution (either 10 mM Glycin, pH 9.0 or 10 mM NaOAc, pH 5.0; 1 mM CaCI2. 25°C) a monolayer of Di-Decanoyl-Phosphatidyl Choline (DDPC) is spread from a chloroform solution. After relaxation<br><br>
ute by the enzyme is estimated from the mean molecular area (MMA) of DDPC. The results are calculated on basis of average banier speed over the first 5 minutes of hydrolysis.<br>
The result Is considered positive for phospholipase if the barrier moves at more than 2 mm/min.<br>
Plate assav 1<br>
A)	50 ml 2% agarose in purified water is melted/stirred for 5 minutes and cooled to 60 - 63°C.<br>
B)	50 ml 2% plant L-alpha-Phosphatidylcholine 95% in 0,2M NaOAc, 10 mM CaCl2. pH 5,5 at 60°C in 30 min. is blended in 15 sec. with ultrathorax.<br>
Equal volumes of 2% agarose and 2% Lecithin (A and B) are mixed, and an equal volume of 1 % Triton X-100 is added to this mixture. 250 pi 4 mg/ml crystal violet in purified water is added as indicator. The mixture is poured into appropriate petri dishes (e.g. 30 ml in 14cm 0 dish), and appropriate holes are made in the agar (3-5 mm) for application of enzyme solution.<br>
The enzyme sample is diluted to a concentration corresponding to OD2B0 = 0.5 and 10 microliter is applied into holes in the agarose/lecithin-matrix. Plates are incubated at 30°C and reaction zones in the plates are identified after approx. 4-5 hours and/or after approx. 20 hours incubation. The Humicola lanuginosa lipase is used as a control, and the presence of a larger clearing zone than the control is taken as a positive result for phospholipase activity.<br>
In a variation of this assay, the addition of Triton X-100 is omitted.<br>
Plate assav 2<br>
10 g agarose is melted in 550 ml H20 by boiling in a microwave oven. After cooling to 60-70°C the following ingredients are added:<br><br>
Digalactosyl   digiyceride   hydroiyzing (DGDGase) activity<br>
Monolayer assay 1<br>
On a thoroughly purified surface of a buffer solution (10 mM NaOAc, pH 5.5; 1 mM CaCI2, 25-0; 10 mM beta-cyclodextrin (Sigma C-4767)) a monolayer of DGDG (Sigma (D4651)) is spread from a chloroform solution. After relaxation of the monolayer (evaporation of chlorofom) the surface pressure is adjusted to 15 mN/m. A solution containing approximately 60 μg (micro gram) enzyme is injected through the monolayer into the subphase of the re-action compartment (cylinder with surface area 2230 mm2 and reaction volume 56570 mm3) in the "zero-order trough". Enzymatic activity is manifested through increased speed of a mobile barrier compressing the monolayer in order to maintain constant surface pressure as insoluble substrate molecules are hydrolyzed into more water soluble reaction products (in presence of beta cyclodextrin).<br>
The result is considered positive for DGDGase if the barrier moves at more than 1 mm/min.<br>
Monolayer 2<br>
On a thoroughly purified surface of a buffer solution (approx. 75 ml, 10 mM NaOAc, pH 5.5; 1 mM CaCI2, 25°C; 10 mM beta-cyclodextrin (Sigma C-4767)) a monolayer of DGDG (Sigma (D4651)) is spread from a chloroform solution to a surface pressure of about 30 mN/m. After relaxation of the monolayer (evaporation of chlorofom) a solution containing approximately 30 μg (micro gram) purified enzyme is injected through the monolayer into the 75 ml subphase while surface pressure is measured continuously. Enzymatic activity is manifested through increased rate of decrease in surface pressure as DGDG is hydrolyzed into water soluble reaction products (in presence of beta cyclodextrin).<br><br><br>
Transformation of yeast strain<br>
The DNA fragments and the opened vectors are mixed and transformed Into the yeast Sac- charomyces cerevisiae YNG318 by standard methods.<br>
Vector for yeast transformation<br>
pJSO026 (S. cerevisiae expression plasmid) is described in WO 97/07205 and in J.S.OIckeis, (1996) "A URA3-promoter deletion in a pYES vector increases the expression level of a fungal lipase in Saccharomyces cerevisiae. Recombinant DNA Biotechnology III: The Integration of Biological and Engineering Sciences, vol. 782 of the Annals of the New/ York Academy of Sciences). It is derived from pYES 2.0 by replacing the inducible GAL1-promoter of pYES 2.0 with the constitutively expressed TPI (those phosphate isomerase)-promoter from Saccharomyces cerevisiae (Albert and Kanwasaki, (1982), J. Mol. AppI Genet, 1, 419-434), and deleting a part of the URA3 promoter"<br>
Site-directed mutagenesis<br>
For the construction of variants of a hi. lanuginosa lipolytic enzyme the commercial kit, Chameleon double-stranded, site-directed mutagenesis kit can be used according to the manufacturer's instructions.<br>
The gene encoding the lipolytic enzyme in question is inserted into the plas-• mid pHD414. In accord-ance with the manufacturer's Instructions the Seal-site of the Amplcillin gene of pHD414 is changed to a Mlul site by use of the following primer:<br>
Primer 3: AGAAATCGGGTATCCTTTCAG.<br>
The pHD414 vector comprising the lipolytic gene in question is then used as a template for DNA polymerase and oligos 7258 and 7770.<br><br>
Screening method<br>
Tbe yeast libraries are spread on cellulose filters on SC-ura agar plates and incubated for 3-4 days at 30°C.<br>
The filters are then transfen-ed to the lecithin plates and incubated at 37°C for 2-6 hours. Yeast cells harboring active phospholipases will develop white clearing zones around the colonies. The positive variants can then be further purified and tested.<br><br>
Media<br>
SC-ura medium<br>
Yeast Nitrogen (without amino aicds)    7.5 g<br>
Succinic acid	11.3 g<br>
NaOH	6.8 g<br>
Casaminoacid {without vitamins)	5.6 g<br>
Tryptophan	0.1 g<br>
Agar, IVIerck	20 g<br>
Distilled water	ad 1000 ml<br>
Autoclaved for 20 minutes at 121°C.<br>
From a sterile stock solution of 5% Threonine 4 ml is added to a volume of 900 ml together.with 100 ml of a sterile 20% glucose.<br>
EXAMPLES<br>
Example 1: Construction of variants with the backbone from Humicola lanuginosa lipase and C-terminal from Fusarium oxysporum phosphoiipase by PCR reaction<br>
The fojiowing variants were used as templates for the backbone from the Humicola lanuginosa lipase: E1A +G91A +D96W +E99K +Q249R and SPIRR +G91A +D96W +E99K +Q249R. The parent lipase was used for generating a fragment in the C-temiinal without Q249R. The template for the C-terminal phosphoiipase was the Fusarium oxyspomm phosphoiipase, cloned in the same vector as the variants of Humicola lanuainnsi^ linasp<br><br>
(SEQ ID NO: 3) as 5' primer and AP (SEQ ID NO: 2) as 3' primer and PCR reaction 2 and 3 as template.<br>
The final PCR was made with 4244 (SEQ ID NO: 1) as 5' primer and KBoj14 (SEQ ID NO: 7) as 3' primer and PGR re-action 1 and 4 as template (by using Humicola lanuginosa lipase as template in reaction 2 a possibility to omit the mutation in position 249 was created).<br>
The final PCR fragment was used in an in vivo recombination in yeast together with pJSO026 cut with the restriction enzymes. Smal(or BamHI) and Xbal (to remove the coding region and at the same time create an overiap of about 75 bp in each end to make a recombination event possible). This final treatment was also used in the following examples.<br>
Primer F0L14 (SEQ ID NO: 3) and primer 15/16 are mixed oligoes to give the possibility to bind both with Humicola lanuginosa lipase and phospholipase templates and at the same time give possibilities for introducing the amino acids from both templates in the different positions. For some of the positions new amino acids could be introduced as well.<br>
Primer F0L14 (SEQ ID NO: 3)<br>
Position 205 in the H. lanuginosa lipase: 75% R, 25% S<br>
Primer F0L15 (SEQ ID NO: 4) /F0L16 (SEQ ID NO: 5)<br>
Position 256 in the H. lanuginosa lipase: 50% P, 50% A<br>
Position 260 in the H. lanuginosa lipase: 25% R, 12.5% Q, 12.5% H, 12.5% C, 12.5% Y, 12.5% W, 12.5% stop.<br>
The sequences of the resulting variants were determined, and were found to correspond to Humicola lanuginosa lipase with the following alterations. Alterations in parentheses are uncertain.<br>
E1A, G91A, D96W, E99K, P256A, W260H, G263Q. L264A. I265T, G266D. T9R7A I9RPN PTOA ?71G 272G. 273F. ^ 274S&gt;<br><br>
Example  2:  Production  of truncated sequences<br>
Variants were made .with stop after amino acid 269, 270, 271, 272,( 273 and 274)<br>
The following PCR reactions were made with the following template: E1A, G91A, D96W, E99K, P256A, W260H, G263Q. L264A, I265T. G266D, T267A, L269N, 270A, 271G. 272G. 273F, {274S).<br>
Reaction 1: 5' primer 4244 (SEQ ID NO: 1) and 3' primer KBoi36 (stop after 269)<br>
Reaction 2: 5' primer 4244 (SEQ ID NO: 1) and 3' primer KBoj37 (stop after 270)<br>
Reaction 3: 5' primer 4244 (SEQ ID NO: 1) and 3' primer KBoj38 (stop after 271)<br>
Reaction 4: 5' primer 4244 (SEQ ID NO: 1) and 3' primer KBoj39 (stop after 272)<br>
The sequences of the resulting variants were determined, and were found to correspond to Humicola lanuginosa lipase with the following alterations:<br>
E1A, G91A, D96W. E99K, P256A. W260H, G263Q. L264A, I265T, G266D. T267A,L269N<br>
E1A, G91A, D96W, E99K, P256A. W260H, G263Q. L264A. I265T, G266D, T267A, L269N, 270A<br>
E1A, G91A. D96W, E99K, P256A, W260H. G263Q, L264A, I265T, G266D, T267A, L269N. 270A, 271G<br>
E1A, G91A, D96W. E99K, P256A, W260H, G263Q. L264A, I265T, G266D, T267A. L269N. 270A, 271G, 272G<br>
Example 3: Removal of mutations in the lid region<br><br>
Example 4: Doping in the C-terminal region of Humicola lanuginosa lipase to introduce phospholipase activity<br>
Three different libraries were constructed with possibilities for mutations in position 256 and position 263-269. At the same time possibilities for extension of the C-terminal with either 1,2,3 or 4 amino acids were included.<br>
Doping, the wt sequences are underlined:<br>
256: PJ4, A 3. T 3<br>
263: G87. E 4.8, A 3.8, R 3.6, Q 0.2, P 0.2<br>
264: L87, P4.8, Q 3.8, V3.6, A0.2, E0.2<br>
265: L85, T 5.6, L 2.2, S 1.6, N 1.5. F 1.4. R 0.4, K 0.4 A,P 0.1, G,D,C,H,Y 0.03, Q,E 0.01, stop 0.016<br>
266: G 86. D 5.9, R 2, S 1.7, C 1.6, A 0.9, V 0.9, E 0.7, W 0.2, H,Y 0.1, l,L,T,F.P 0.02. Q,K 0.01, stop 0.014<br>
267: £86, A 6.6, S 1.9, R 0.9, N 0.9, I 0.9, K 0.9, M 0.9, P 0.9, P 0.9, G,V 0.14. D,E 0.07;L 0.03, C,Q,H,F,W,Y 0.01, stop 0.01<br>
268: C91. S 1.9, R 1.0, G 1.0, F 0.9, Y 0.9, L 0.04. A.N.D.H.I.P.T.V 0.01, stop 2.8<br>
269: LJ2, stop 8 (KBoj 32 (SEQ ID NO: 8) and KBoj33)/ N 86. K 2.7, D 1.8, H 1.8, I 1.8, S 1.8, T 1.9, Y 1.8, R 0.1, Q,M,E 0.06, A,C.G,L.F.P.V 0.04. stop 0.06{KBoj34)<br>
270: stop 100 (KBoj33)/A 44. P 44, S 1.9. T 1.8. R 1.5. L 1.5, G 1.4, V 1.4, D 0.7. Q 0.7, E 0.7. H 0.7. N.C.I.K.M.F.W.Y 0.03. stop. 0.03 (KBoj 32 (SEQ ip NO: 8) and KBoj 34)<br>
271: G 72. R 4.5. V 3.2. E 3.0. C 2.9. A 1.6. S 1.2, D 1.0, L 0.5, l,K,Y 0.15, Q,T 0.08, N,P 0.05, stop 9.2<br>
272: G 72, R 4.5, V 3.2, E 3.0, C 2.9, A 1.6, S 1.2, D 1.0, L 0.5, I.K,Y 0.15, Q.T0.08. N.P 0.05. stOD 9.2<br><br>
Library C: PCR reaction with 4244 (SEQ ID NO: 1) as 5' primer and KBoj 34 as 3' primer and E1A +G91A +D96W +E99K +Q249R or E1A +G225R as template. Variants from this library will most probably contain mutations in position 269 and a C-terminal extension but can contain stop codons before the extension.<br>
The following variants were obtained:<br>
Library A:<br>
E1A +G91A +D96W +E99K +Q249R +G266D<br>
Library B:<br>
E1A +G91A +D96W +E99K +(R232L) +Q249R +G266S +270A<br>
E1A +G91A +D96W +E99K +Q249R +G266S +270D +271G<br>
E1A+ G91A+ D96W+ E99K+ Q249R+ L264G+ I265G+ G266F+ T267stop<br>
E1A +G91A +D96W +E99K +Q249R +G266A +270P +271G<br>
E1A +G91A +D96W +E99K +Q249R +L264P +I265F +L269stop<br>
Library C.-<br>
E1A +G91A +D96W +E99K +Q249R +G263E +G266D +L269N +270P +271V +272G +273F<br>
El A +G91A +D96W +E99K +Q249R +G263A +G266S +L269N +270A +271G+272R+273F<br>
El A +G91A +D96W +E99K +Q249R +L264P -G266 +L269I +270P +271R +272G +273F<br>
E1A +G91A +D96W +E99K +Q249R +G266D +L269S +270A +271G +272G<br>
+273F<br>
El A +D27G +G91A +D96W +E99K +Q249R +G266S +L269N +270A +271G +272G +?7:^F<br><br>
{Humicola lanuginosa lipase) showed no activity at pH 5 or 9. Depending on the variant, the activity at pH 5 was higher or lower than at pH 9.<br>
A prior-art variant of Humicola lanuginosa lipase was found to have no phosphollpase activity at pH 5: SPIRR +N94K +F95L +D96H +N101S +F181L +D234Y +I252L +P256T +G263A +L264Q.<br>
Example 5: Variants of Humicola lipase with phosphoiipase activity<br>
Variants of the parent lipase from Humicola lanuginosa were prepared and tested for phosphollpase activity as described above. The following variants were found to have phosphollpase activity, where as the parent had no phosphollpase activity by the same method.<br><br><br><br><br><br><br><br><br><br><br><br>
In the table above, (+274S) indicates that the presence of this amino acid residue at the C-terminal is uncertain. For one such variant, it was found that only a minor fraction contained this residue<br>
Several of the above variants had a higher ratio of phospholipase (PHLU) to lipase (LU) than a prior-art enzyme from F. oxysporum known to have both lipase and phospholipase activity.<br>
For some of the above variants, the pH optimum for lipase and phospholipase was determined by using the LU and PHLU methods at various pH values. The results showed that the pH optimum phospholipase activity was in the range 4-6. The optimum for lipase activity varied from about pH 6 to about pH 10.<br>
8 variants listed in Example 5 were analyzed for phospholipase activity by the mono layer assay described above at pH 5 and 9. The results showed that all the variants have phospholipase activity at pH 5 and 9, whereas the parent lipase {Humicola lanuginosa lipase) showed no activity at pH 5 or 9. Depending on the variant, the activity at pH 5 was higher or lower than at pH 9.<br>
A prior-art variant oi Humicola lanuginosa lipase was found to have no phospholipase activity at pH 5: SPIRR +N94K +F95L +D96H +N101S +F181L +D234Y +I252L +P256T +G263A +L264Q.<br>
The following variants of the parent lipase from Humicola lanuginosa may also have phospholipase activity:<br><br><br><br>
G266V<br>
at<br>
Example 7: Variants of Humicola lipase witii  increased specificity for long-chain fetty acids<br>
Variants of the parent lipase from Humicola lanuginosa were prepared and tested for their hydrolytic activity on two triglyceride substrates with different chain length: tributyrin (C4:o) and triolein (Ci8:i). The tests were done at pH 9 by the LU and SLU methods described above. The following variants were found to have a higher ratio of triolein activity to tributyrin activity than the parent enzyme {Humicola lanuginosa lipase):<br><br><br><br><br><br><br><br>
Example 8: Variants of Fusarium lipase with increased specificity for long-chain fatty acids<br>
Variants of the parent lipase from Fusarium oxysporum were prepared and tested as in the previous example. The following variants were found to have a higher ratio of triolein activity to tributyrin activity than the parent enzyme:<br>
Y23S<br>
Y260L<br>
The following variants of the parent lipase from Fusarium oxysporum may also have an increased specificity for long-chain fatty acids:<br>
R80H +S82T<br>
S82T+A129T<br>
Example 9: Variants of Rhizomucor lipase with increased specificity for long-chain fatty acids<br>
The following variants of the parent lipase from Rhizomucor miehei may have an increased specificity for long-chain fatty acids:<br><br><br>
t<br>
Example 11: Variants of Fusarium lipase with increased specificity for short-chain fatty acids<br>
Variants of the parent lipase from Fusarium oxysporum were prepared and tested as in the previous example. The following variants were found to have a hiaher ratio of tributvrin actlvitv to triolein activitv than the oarent enzvme:<br><br>
Y260C Y260G Y260V<br>
Example 13: Variants of Humicola lipase with OGDGase activity<br>
Variants of the parent lipase from Humicola lanuginosa were prepared, and the hydrolytic activity towards DGDG (di-galactosyl-di-glyceride) was detennined as described above. The following variants were found to have DGDGase activity, whereas the parent lipase gave a negative result.<br>
Example 14: Variants of Humicola lipase with increased pH optimum<br>
Variants of the parent lipase from Humicola lanuginosa were prepared, and the lipase activity was measured by the LU method at pH 7 and 9. The following variants were found to have a higher ratio of activity at pH 9 to activity at pH 7 than the parent lipase:<br>
■R84L<br>
R84W<br>
Y21I<br>
Y21V<br>
Y261I<br><br>
Example 16: Use of Humicola lipase variants in degumming of vegetable<br>
oil	<br>
Rapeseed oil was treated with two variants of the lipase from Humicola lanuginosa, essentially as described in Example 6 of WO 98/18912 (Novo Nordisk).<br>
One variant was tested at an enzyme dosage of 0.6 mg of enzyme protein per kg of oil. Results of tests at various pH and temperatures showed optimum performance at pH 5.7, 35-45°C, where a final P content of 4 ppm was reached. A separate experiment at 45°C, pH 6 showed that a final P content of 4 ppm could be reached at an enzyme dosage as low as 0.15 mg/kg.<br>
A similar experiment with another Humicola lanuginosa lipase variant showed optimum performance at 40°C, pH 5.0-5.5. The enzyme dosage was 0.3 mg/kg.<br>
A degumming experiment was made with a third Humicola lanuginosa lipase variant, using rape seed oil at 45°C, pH 5, 1.8 mg enzyme/kg oil. For comparison, a similar experiment was made with the parent lipase {Humicola lanuginosa lipase) at 18 mg/kg. The results showed that good degumming (
The parent lipase {Humicola lanuginosa lipase) was found to have very little degumming effect, even at 10 times higher enzyme dosage.<br>
Example 17: Use of lipase variants in baking<br>
A variant of the lipase from Humicola lanuginosa was evaluated in baking tests as follows.<br>
Doughs were prepared from Meneba flour according to the European straight dough method (ABF-SP-1201.01) with 40 ppm of ascorbic acid. Various combinations of additives at the following dosages were used: the lipase variant at 0,<br><br>
Firmness of crumb was measured using a texture analyzer TA-XT2 from Stable Micro Systems (probe diameter 40 mm).<br>
Softness in gram was measured as the force needed to press a probe 6,25 mm into a crumb of a 25 mm thick slice of bread (26 % penetration).<br>
The results showed that the addition of 1.5 mg of the variant increased the loaf volume. The results for firmness and elasticity show that the variant gives significantly softer crumb and significantly better elasticity from day 0 to day 7.<br>
Example 18: Use of lipase variants for dough stability in baking<br>
A variant of the Humicola lanuginosa lipase was evaluated in a baking trial to evaluate its tolerance towards extended proofing of the dough.<br>
Doughs were prepared from Pelikan flour according to the European straight dough method (347-SP-1217) with 30 ppm ascorbic acid, fungal a-amylase (10 FAU of Fungamyl), and pentosanase (100 FXU of Pentopan Mono). Dosages of 0.2, 0.4, and 0.6 mg entyme protein/kg flour of the variant were compared with 1000 LU of the parent lipase.<br>
The doughs were made into rolls. Half of the rolls were proofed for 45 minutes (normal proofing) and the other half for 70 minutes (over proofing).<br>
After baking the bread was cooled, and the volume and the standing of the rolls were evaluated after approximately 2 hours. The standing is a measure of the shape of the rolls and is defined as the height of 10 rolls divided by the width of 10 roils, which means that nice round loaves have a high standing value, whereas fiat rolls have a low standing value.<br>
The results showed that at normal proofing time the volume of 0.4 and 0.6 mg of the variant were better than that of the parent lipase, and the standing of the rolls were better for the variant at all dosages than for the parent lipase. When the rolls were over proofed. both volume and standing was better for the variant at all<br><br>
The lipases tested were Humicola lanuginosa lipase and variants thereof. For each lipase, the chain-length specificity is expressed as the ratio of activities on triolein (SLU) and tributyrin (LU).<br>
Three persons evaluated the samples and agreed on the ranking shown below<br>
+	Detectable smell<br>
++       Clear and characteristic butyric acid and/or sour odor<br>
+++      Strong butyric acid and/or sour odor<br>
Three variants of Humicola lanuginosa lipase having a higher SLU/LU ratio than Humicola lanuginosa lipase were found to have less malodor than the parent lipase.<br>
Example 20: Effect of lipase variants on malodour on textile after washing<br>
Soiling:<br>
Cotton textile was soiled with a dairy product as described here. 50 mg of butter was applied over an area of approximately 30 cm2 in an even spot. The soiled textile was aged for 24 hours at ambient conditions.<br>
Washing procedure:<br>
Washing of the soiled textile was done in a Terg-0-tometer using a commercial detergent(5 g/l) with and without lipase(1250 and 5000 LU/I). The washing was done at 30°C for 20 min at 100 rpm. After washing the swatches were left overnight to dry at ambient conditions.<br>
Sensorv analysis:<br>
Next day, the malodour was assessed by a sensory panel consisting of at<br><br>
Variants of Humicola lanuginosa lipase with an increased ratio of tri-oleln/tributyrin activities (increased SLU/LU ratio) were found to give a fainter smell from butter stains than the parent enzyme {Humicola lanuginosa lipase). A separate washing experiment showed that the variants, like the parent enzyme, were effective In the removal of lard stains.<br>
Alternative methods<br>
The intensity of butyric acid from dairy stains on fabric can also be evaluated by instrumental analysis:<br>
1.	By Head Space Gas Chromatography, or<br>
2,	By extraction of the odours from fabric followed by Gas Chromatography<br>
Example 21: Effect of lipase variants on odour of bread baked with butter<br>
Six variants of the lipase from Humicola lanuginosa were prepared and were evaluated in bread baked by the European straight dough procedure (347-SP-1217) with addition of 3% butter. 0.2 mg enzyme protein/kg flour was used for each of the variants.<br>
The chain-length specificity of the variants was also determined by measuring the triolein/tributyrin activity ratio (SLU/LU described above). The parent lipase from Humicola lanuginosa and a prior-art lipase with phospholipase acivity from Fusarium oxysporum were also tested for comparison.<br>
The results are summarized below:<br>
+	detectable smell<br>
++        clear and characteristic butyric acid and/or sour odour<br>
+++      strong butyric acid and/or sour odour<br><br><br>
The results, indicate that that lipase variants with a SLU/LU ratio at 3 or above (i.e. a high specificity for long-chain fatty acids) give no unpleasant odour in bread baking even with butter in the recipe.<br><br><br><br><br><br>
AMENDED CLAIMS<br>
In response to first written opinion<br>
1.	A method of producing a lipolytic enzyme variant comprising:<br>
a)	selecting a parent lipolytic enzyme having an alcohol binding site having a glycerol part with an sn2 position,<br>
b)	in the parent lipolytic enzyme selecting at least one amino acid residue which comprises at least one atom within 10 A of the C atom at the sn2 position of the glycerol part of a substrate triglyceride in a three-dimensi6nal structure of the parent lipolytic enzyme and the substrate,<br>
c)	making alterations each of which is an insertion, a deletion or a substitution of the amino acid residue,<br>
d)	optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than b),<br>
e)	preparing the variant resulting from steps a)-d),<br>
f)	testing substrate specificity of the variant,<br>
g)	selecting a variant having an altered substrate specificity, and h) producing the selected variant.<br>
2.	The method of any of claim 1 wherein the parent lipolytic enzyme is native to an<br>
eukaryote, preferably to a fungus, and most preferably belongs to the Humicola fam<br>
ily or the Zygomycetes family.<br><br>
e)	preparing the variant resulting from steps a-d,<br>
f)	testing substrate specificity of the variant, and<br>
g) selecting a variant having an altered substrate specificity.<br>
4.	The method of claim 2 or 3 wherein the parent lipolytic enzyme is the lipase of<br>
Humicola lanuginosa strain DSi\/14109.<br>
5.	The method of any of claims 1-4 wherein the altered substrate specificity is a<br>
lower ratio of activity towards a C4-C8 acyl bond in a triglyceride and a C16-C20 acyl<br>
bond in a triglyceride.<br>
6.	The method of claim 5 wherein the parent lipolytic enzyme belongs to the Humicola family or the Zygomycetes family, preferably the lipase of Humicola lanuginosa strain DSM 4109, and the selected amino acid residues comprise an amino acid corresponding to Y21, E56, D57, V60, G61, D62. R81, S83, R84, L259. Y261 or G266 in the Humicola lanuginosa lipase.<br>
7.	The method of any of claims 1-4 wherein the altered substrate specificity Is a lower ratio of activity towards a C16-C20 acyl bond in a triglyceride and a C4-C8 acyl<br>
bond in a triglyceride.<br>
8.	The method of any of claims 1-4 wherein the altered substrate specificity is a<br>
higher phospholipase activity.<br>
9.	The method of claim 8 wherein the parent lipolytic enzyme has a phospholipase<br>
activity below 50 PHLU/mg and/or a ratio of phospholipase activity to lipase activity<br><br>
11. The method of any of claims 8-10 wherein the alterations comprise insertion of a peptide extension at the C-termlnal, preferably comprising 1-5 amino acid residues, the first preferably being A. P or D, the second (if present) preferably being V, G or R, the third (if present) preferably being V, G or R, the fourth (If present) preferably being F, and the fifth (if present) preferably being S,<br>
12.	The method of any of claims 1-4 wherein the altered activity Is a higher hydro-<br>
lytic activity on a digalactosyi-diglyceride,<br>
13.	The method of claim 12 wherein the parent lipolytic enzyme belongs to the<br>
Humicola family or the Zygomycetes family, preferably the lipase of Humicola lanugh<br>
nosa strain DSM 4109, and the selected amino acid residues comprise an amino acid<br>
corresponding to 21, 23. 26, 57, 62. 81. 83, 84, 85, 266. 267 or 269 in the Humicola<br>
lanuginosa lipase.<br>
14.	A lipolytic enzyme which is a variant of a parent lipolytic enzyme having an al<br>
cohol binding site having a glycerol part with an sn2 position, which variant:<br>
a)	comprises 1-20 alterations each of which is an Insertion, a deletion or a substitution of an amino acid residue at a position which in a three-dimensional structure of the parent lipolytic enzynne and a substrate is within 10 A of the C atom at the sn2 position of the glycerol part of a substrate triglyceride,<br>
b)	optionally comprises up tolO other amino acid alterations, and<br>
c)	has an altered substrate specificity.<br><br>
a) has an amino acid sequence that is at least 80 % homologous with the amino acid sequence of a reference lipolytic enzyme of the Humicola family or the Zygomycetes family;<br>
b)	has an amino acid sequence that differs from the amino acid sequence of<br>
the reference lipolytic enzyme wherein the difference comprises:<br>
i) an amino acid substitution, deletion or insertion at a position corresponding to A20. Y21, G23, K24, N25, V63, R81, G82, R84. A257, W260, Y261, F262 or G266 in the Humicola lanuginosa DSM 4109 lipase;<br>
ii) a substitution of an amino acid corresponding to C268 or L269 in the Humicola lanuginosa DSM 4109 lipase;<br>
iii)a substitution corresponding to V60G, D62E, L93K, L97Q. K98E,F, E99D, P256A, G263E.Q,R,F,N, L264A,C,P,F.G,VJ, I265L,N,F or T267A,Q,P,S.V,E in the Humicola lanuginosa DSM 4109 lipase;<br>
iv) an insertion corresponding to T267GS or T267GL in the Humicola lanuginosa DSM 4109 lipase;<br>
v) a peptide extension at the C-terminal which is A, P, MD, OP, AG, DG, AGG, PVGF. AGRF, PRGF. AGGF or AGGFS; vi) a peptide extension at the C-terminal of 40-50 amino acids; or vii) a truncation of 1, 2, 3,4, 5 or 6 amino acids at the C-terminal; and<br>
c)	has a different substrate specificity from the reference lipolytic enzyme.<br>
8.   The lipolytic enzyme of claim 17 wherein the amino acid substitution corre-ponds   to    R84K,L,W,   W260H,Q.C,   G266A,C,D,N,UI.S.T.P.V,F,W.E,K,R,Y   or<br><br>
21,   The lipolytic enzyme of any of claims 17-20 which comprises a peptide extension at the N-terminai compared to the lipolytic enzyme..<br>
22.	The lipolytic enzyme of any of claims 17-21 wherein the reference lipolytic enzyme is the lipase from Humicola lanuginosa,<br>
23.	The lipolytic enzyme of any of claims 17-21 wherein the reference lipolytic enzyme is the lipase from Rhizomucor mieheL<br>
24.	The lipolytic enzyme of any of claims 17-21 wherein the reference lipolytic en-zyme is the lipase from Fusarium oxysporum.<br>
25.	The lipolytic enzyme of any of claims 14-24 wherein the altered substrate specificity is a lower ratio of activity towards a C4-C8 acyl bond in a triglyceride and a C16-<br>
C20 acyl bond in a triglyceride.<br>
26.	The lipolytic enzyme of claim 25 which comprises an amino acid alteration at a position corresponding to Y21, E56, D57, V60. G61, D62, R81, S83. .R84, L259, Y261 or G266 in the Humicola lanuginosa lipase.<br>
27.	The lipolytic enzyme of any of claims 14-24 wherein the altered substrate sped-ficity is a lower ratio of activity towards a C16-C20 acyl bond in a triglyceride and a C4-<br>
C8 acyl bond in a triglyceride.<br>
28.	The lipolytic enzyme of any of claims 14-24 wherein the altered substrate speci-<br><br>
phospholipase activity greater than 100 PHLU/mg (preferably greater than 500 PHLU/mg) and/or a ratio of phospholipase activity to lipase activity greater than 0.1 PHLU/LU (preferably greater than 0.5 PHLU/LU).<br>
31.	The lipolytic enzyme of any of claims 28-30 wherein the parent lipolytic enzyme has a phospholipase activity below 50 PHLU/mg and/or a ratio of phospholipase activity to lipase activity below 0,1 PHLU/LU,<br>
32.	The lipolytic enzyme of any of claims 28-31 which comprises a peptide extension at the C-terminal, preferably comprising 1-5 amino acid residues, the first preferably being A, P or D, the second (if present) preferably being V, G or R, the third (if present) preferably being V, G or R, the fourth (if present) preferably being F, and the<br>
fifth (if present) preferably being S.<br>
33.	The lipolytic enzyme of any of claims 28-32 which comprises a deletion of amino acid residues at positions corresponding to positions C268 and L269 in the lipase of Humicola lanuginosa strain DSM 4109.<br>
34.	The lipolytic enzyme of any of claims 14-24 whereinthe altered substrate specificity is a higher hydrolytic activity on a digalactosyl-digiyceride.<br>
35.	The lipolytic enzyme of claim 34 which comprises an-amino acid alteration at a position corresponding to Y21. G23, N26, D57, D62. R81, S83, R84. S85, G266, T267 or L269 in the Humicola lanuginosa lipase, preferably comprising two or more such alterations, most preferably further comprising at least one alteration in the lid<br>
region<br><br>
38.   A DNA sequence encoding the lipolytic enzyme of any of claims 14-37.<br>
39.	A vector comprising the DNA sequence of claim 38.<br>
40.	A transformed host cell harboring the DNA sequence of claim 38 or the vector<br>
of claim 39.<br>
41.	A method of producing the lipolytic enzyme of any of claims 14-37 comprising<br>
a)	cultivating the cell of claim 55 so as to express and preferably secrete the lipolytic enzyme, and<br>
b)	recovering the lipolytic enzyme.<br><br>
42.	A process for preparing a dough or a baked product prepared from the dough which comprises adding the lipolytic enzyme of any of claims 14-37 to the dough, wherein the lipolytic enzyme preferably has phospholipase and/or dlgalactosyl di-glyceride activity.<br>
43.	The process of claim 42 which further comprises adding to the dough an endo-<br>
amylase and/or a phospholipid.<br>
44.	The process of claim 42 or 43 wherein the endo-amylase is from Bacillus, and is preferably a maltogenic amylase from B. stearothermophilus,<br>
45.	A process for reducing the content of phospholipid in an edible oil, comprising treating the oil with the lipolytic enzyme of any of claims 28-33 so as to hydrolyze a<br><br>
47. A detergent composition comprising a surfactant and the lipolytic enzyme of any of claims 14-37, wherein the lipolytic enzyme preferably has a specificity for long-chain fatty acids corresponding to a ratio of SLU to LU above 3.<br>
48.	A method of enhancing the flavor of a food product containing milk fat, comprising treating the food product with the lipolytic enzyme of any of claims 27-52 so as to release free fatty acids, wherein the lipolytic enzyme preferably has a specificity for short-chain fatty acids corresponding to a ratio of SLU to LU below 0,5, more preferably beiow 0.2, e.g. below 0.1.<br>
49.	A method of producing a lipolytic enzyme variant comprising:<br><br>
a)	selecting a parent lipolytic enzyme having a structure comprising a catalytic triad consisting of an active Ser. an active Asp and an active His residue,<br>
b)	in the parent lipolytic enzyme selecting at least one amino acid residue comprising at least one atom belonging to a set E defined by the following steps:<br>
i) aligning the structure of the lipolytic enzyme with Rhizomucor mie-hei lipase structure 4TGL comprising a catalytic triad and an inhibitor phosphorus atom (4TGL-inhP), so as to minimize the sum of squares of deviation between atoms of the catalytic triads of the two structures,<br>
ii) defining a set A consisting of atoms of the lipolytic enzyme inside a sphere of radius 18 A with center at 4TGL-inhP, iii) forming a first plane defined by 4TGL-inhP, the Ca atom of the active Ser residue  of the parent lipolytic enzvme, and the Ca atom of<br><br>
the second plane from the Cα atom of the active Asp residue of the parent lipolytic enzyme,<br>
v) forming a set D consisting of atoms belonging to the union of sets B and C, and having a solvent accessibility of 15 or higher, and vi) forming set E consisting of amino acid residues in the structure which comprise an atom belonging to set D or an atom belonging to the union of sets B and C and located less than 3.5 A from an atom belonging to set D, c) making alterations each of which is an insertion, a deletion or a substitution of the selected amino acid residues, . d) optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than d).<br>
e)	preparing the variant resulting from steps a) -d), and<br>
f)	testing substrate specificity of the variant,<br>
g)	selecting a variant having an altered substrate specificity, and h) producing the selected variant.<br>
50.   A method of producing a lipolytic enzyme variant comprising:<br>
a)	selecting a parent lipolytic enzyme having an active site comprising an active His residue,<br>
b)	in the amino acid sequence of the parent lipolytic enzyme selecting at least one amino acid residue at the C-terminal side of the active His residue,<br>
c)	making alterations each of which is an insertion, a deletion or a substitution of the amino acid residue,<br>
d)	optionally, making alterations each of which is an insertion, a deletion or a<br><br>
b) selecting at least one amino acid residue among 10 amino acid residues at the C-terminal,<br>
c)	making alterations each of which is an insertion, a deletion or a substitution of the selected amino acid residues,<br>
d)	optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than c),<br>
e)	preparing the variant resulting from steps a) - d),<br>
f)	testing substrate specificity of the variant,    ,<br>
g)	selecting a variant having an altered substrate specificity, and<br>
h) producing the selected variant.<br>
«<br>
52.	A method of producing a lipolytic enzyme variant comprising:<br>
a)	selecting a parent lipolytic enzyme having a lid,<br>
b)	selecting at least one amino acid residue in the lid,<br>
c)	making alterations each of which is an insertion, a deletion or a substitution of the selected amino acid residues,<br>
d)	optionally, making alterations each of which is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than b),<br>
e)	preparing the variant resulting from steps a) - d),<br>
f)	testing substrate specificity of the variant,<br>
g)	selecting a variant having an altered substrate specificity, and h) producing the selected variant<br>
53.	A method of preparing a lipolytic enzyme variant for use in baking, which<br>
method comprises:<br><br>
d) preparing the lipolytic enzyme variant.<br>
54.   The method of claim 53 wherein the parent lipolytic enzyme and the amino  acid to be altered are selected as defined in any of claims 1-4 or 49-52.<br>
55.	A method of preparing a lipolytic enzyme variant for use in baking, which<br>
method comprises<br>
a)	subjecting a DNA sequence encoding a lipolytic enzyme to random mutagenesis,<br>
b)	expressing the mutated DNA sequence obtained in step (a) in a host cell, and<br>
c)	screening for host cells expressing a lipolytic enzyme variant which compared to the parent lipolytic enzyme has:<br>
i) a higher ratio selectivity for long-chain fatty acyl groups, ii) a higher activity on digalactosyl diglyceride, and iii) a higher phospholipase activity, and<br>
d)	preparing the lipolytic enzyme expressed by the host cells,<br>
56.	A lipolytic enzyme which is a variant of a parent lipolytic enzyme having a lid,<br>
which variant:<br>
a)	comprises an alteration which is an insertion, a deletion or a substitution of an amino acid residue in the lid,<br>
b)	has an altered substrate specificity.<br>
57.	A lipolytic enzyme which is a variant of a parent lipolytic enzyme having an ac-<br><br>
59.   A lipolytic enzyme which:<br>
a)	is a polypeptide having an amino acid sequence which has at least 80 % homology with a reference enzyme which is the lysophosphoiipase from Aspergillus foetidus, the ferulic acid esterase from Aspergillus niger, the ferulic acid esterase from Aspergillus tubigensis or phospholipase A1 from Aspergillus oryzae,<br>
b)	compared to said reference enzyme comprises an amino acid alteration which is a substitution, deletion or insertion at a position corresponding to 20-25, 56-64, 81-85, 91-98, 255-257 or 259-269 in the Humicola lanuginosa lipase, and<br>
«<br>
c)	has an altered substrate specificity compared with the reference enzyme.<br>
60.	The lipolytic enzyme of claim 59 wherein the alteration is at a position corre<br>
sponding to Y21, E56, D57, V60, G61, D62, S83. R84, G91. L93, N94. D96, L97,<br>
K98, E99, P256, W260, Y261, G263, L264, 1265, G266, T267 or L269 in the Humh<br>
cola lanuginosa lipase<br>
61.	The lipolytic enzyme of claim 59 or 60 wherein the alteration is an extension or a truncation at the C-terminal, preferably by 1-5 amino acids.<br>
62.	A lipolytic enzyme which is a variant of a parent lipase derived from Humicola lanuginosa strain DSM 4109 comprising the alterations E1E,D,A+ G91GAS,T+ N94N,D+ D96D,G,F,W+ E99E.K+ G225G,R,K+ G263Q.N+ L264LAV+ I2651,T.S+<br>
G266G,A,V,S,D,E+ T267TAV+ L269L,I.N,Q.<br><br>
b) selecting a lipolytic enzyme having hydrolytic activity towards digalac-tosyi diglyceride and the phospholipid, and having a ratio of activity towards the C16-C20 acyl bond and the C4-C8 acyl bond which corresponds to a SLU/LU ratio of at least 3. and c) adding the selected lipolytic enzyme to the dough.<br><br>
65.     A method of producing a lipolytic enzyme variant substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1jbGFpbXMgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1jbGFpbXMgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1jb3JyZXNwb25kbmVjZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1kZXNjcmlwdGlvbihjb21wbGV0ZSlmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1kZXNjcmlwdGlvbihjb21wbGV0ZSlncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1vdGhlciBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-other document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzMyLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-732-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="210753-fast-slewing-pseudorandom-noise-sequence-generator.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="210755-dosing-machine-for-hard-gelatin-capsules.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>210754</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/732/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Oct-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-May-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsvaerd,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FUGLSANG, Klaus, Crone</td>
											<td>Poppelhøj 43, DK-2990 Niva,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHAMKANT, Anant, Patkar</td>
											<td>Christoffers Alle 91, DK-2800 Lyngby,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BORCH, Kim</td>
											<td>Klerkegade 12, 2tv, DK-1808 Copenhagen,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VIND, Jesper</td>
											<td>Bagsvaerdvej 115, DK-2800 Lyngby,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>PETRI, Andreas</td>
											<td>Tagensvej 81, 3 DK-2200 Copenhagen,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>GLAD, Sanne, Schroder</td>
											<td>Viggo Barfoeds Alle 59, DK-2840 Holte,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>BUDOLFSEN, Gitte</td>
											<td>Lyovej 3, 3.tv., DK-2000 Frederiksberg,</td>
										</tr>
										<tr>
											<td>8</td>
											<td>BOJSEN, Kirsten</td>
											<td>Bakkedal 5, DK-2900 Hellerup,</td>
										</tr>
										<tr>
											<td>9</td>
											<td>SVENDSEN, Allan</td>
											<td>Bakkeledet 28, DK-3460 Birkerod,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/20</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK1999/000664</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-11-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 1999 00391</td>
									<td>1999-03-22</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 1998 01572</td>
									<td>1998-11-27</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>3</td>
									<td>PA 1999 01481</td>
									<td>1999-10-15</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/210754-a-method-of-producing-a-lipolytic-enzyme-variant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:51:02 GMT -->
</html>
