<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/210841-an-invitro-produced-polypeptide-of-not-more-than-80-amino-acids-in-length-comprising-at-least-one-c35-peptide-epitope by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:30:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 210841:&quot;AN INVITRO PRODUCED POLYPEPTIDE OF NOT MORE THAN 80 AMINO ACIDS IN LENGTH COMPRISING AT LEAST ONE C35 PEPTIDE EPITOPE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN INVITRO PRODUCED POLYPEPTIDE OF NOT MORE THAN 80 AMINO ACIDS IN LENGTH COMPRISING AT LEAST ONE C35 PEPTIDE EPITOPE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention discloses an in vitro produced polypeptide of not more than 80 amino acids in length comprising at least one C35 peptide epitope selected from the group consisting of: amino acids S-9 to V-17, amino acids V-10 to V-l 7, amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids S-21 to F-35, amino acids C-30 to T-38, amino acids E-31 to Y-39, amino acids E-36 to A-43, amino acids A-37 to A-45, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to I-53; amino acids G-52 to L-59, amino acids E-54 to T-62, amino acids S-57 to F-75, amino acids R-58 to 1-67, amino acids G-61 to 1-69, amino acids G-63 to F-83, amino acids E-66 to L-73, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1-105 to 1-114, each of SEQ ID NO:2.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>AN   INVITRO   PRODUCED   POLYPEPTIDE   OF   NOT   MORE   THAN   80<br>
AMINO   ACIDS   IN   LENGTH   COMPRISING   AT   LEAST   ONE   C35   PEPTIDE   EPITOPE<br>
BACKGROUND OF THE INVENTION<br><br><br>
Field of the Invention<br><br>
The present invention relates to a novel human gene that is differentially express ed in human breast and bladder carcinoma. More specifically, the present invention relates to apolynucleotide encoding a novel human polypeptide named C35. This invention also relates to C35 polypeptides, as well as vectors, host cells, antibodies directed to C35 polypeptides, and the recombinant methods for producing the same. The present invention further relates to diagnostic methods for detecting carcinomas, including human breast and bladder carcinomas. The present invention further relates to the formulation and use of the C35 gene and polypeptides in immunogenic compositions or vaccines, to induce antibody and cell-mediated immunity against target cells, such as tumor cells, that express the C35 gene. The invention further relates to screening methods for identifying agonists and antagonists of C35 activity.<br>
Background Art<br>
Cancer afflicts approximately 1.2 million people in the United States each year. About 50% of these cancers are curable with surgery, radiation therapy, and chemotherapy. Despite significant technical advances in these three types of treatments, each year more than 500,000 people will die of cancer in the United States alone. (Jaffee, E. M., Ann. N. Y. Acad. Sci. 886:61-12 (1999)). Because most recurrences are at distant sites such as the liver, brain, bone, and lung, there is an urgent need for improved systemic therapies.<br>
The goal of cancer treatment is to develop modalities that specifically target tumor cells, thereby avoiding unnecessary side effects to normal tissue. Immunotherapy has the potential to provide an alternative systemic treatment for<br><br>
-2-<br>
most types of cancer. The advantage of immunotherapy over radiation and chemotherapy is that it can act specifically against the tumor without causing normal tissue damage. One form of immunotherapy, vaccines, is particularly attractive because they can also provide for active immunization, which allows for amplification of the immune response. In addition, vaccines can generate a memory immune response.<br>
The possibility that altered features of a tumor cell are recognized by the immune system as non-self and may induce protective immunity is the basis for attempts to develop cancer vaccines. Whether or not this is a viable strategy depends on how the features of a transformed cell are altered. Appreciation of the central role of mutation in tumor transformation gave rise to the hypothesis that tumor antigens arise as a result of random mutation in genetically unstable cells. Although random mutations might prove immunogenic, it would be predicted that these would induce specific immunity unique for each tumor. This would be unfavorable for development of broadly effective tumor vaccines. An alternate hypothesis, however, is that a tumor antigen may arise as a result of systematic and reproducible tissue specific gene deregulation that is associated with the transformation process. This could give rise to qualitatively or quantitatively different expression of shared antigens in certain types of tumors that might be suitable targets for immunotherapy. Early results, demonstrating that the immunogenicity of some experimental tumors could be traced to random mutations (De Plaen, et al., Proc. Natl. Acad. Sci USA 85: 2274-2278 (1988); Srivastava, &amp; Old, Immunol. Today P:78 (1989)), clearly supported the first hypothesis. There is, however, no a priori reason why random mutation and systematic gene deregulation could not both give rise to new immunogenic expression in tumors. Indeed, more recent studies in both experimental tumors (Sahasrabudhe et al., J. Immunol. 151:6202-6310 (1993); Torigoe et al., J. Immunol 147:3251 (1991)) and human melanoma (van Der Bruggen et al., Science -254:1643-1647 (1991); Brichard et al., J. Exp. Med. 775:489-495 (1993);Kawakami et al., Proc. Natl.Acad.Sci USA 91:3515-3519(1994);Boel<br><br>
-3-<br>
et al., Immunity 2:167-175 (1995); Van den Bynde et al., J. Exp. Med. 182: 689-698 (1995)) have clearly demonstrated expression of shared tumor antigens encoded by deregulatednormal genes. The identification of MAGE-1 and other antigens common to different human melanoma holds great promise for the future development of multiple tumor vaccines.<br>
In spite of theprogress in melanoma, very few shared antigens recognized by cytotoxic T cells have not been described for other human tumors. The major challenge is technological. The most widespread and to date most successful approach to identify immunogenic molecules uniquely expressed in tumor cells is  to  screen  a  cDNA  library with tumor-specific  CTLs   (cytotoxic  T lymphocytes). Application of this strategy has led to identification of several gene families expressed predominantly in human melanoma.   Two major limitations of this approach, however, are that (1) screening requires labor intensive transfection of numerous small pools of recombinant DNA into separate target populations, which themselves often need to be modified to express one or more MHC molecules required for antigen presentation, in order to assay T cell stimulation by a minor component of some pool; and (2) with the possible exception of renal cell carcinoma, tumor-specific CTLs have been very difficult to isolate from either tumor infiltrating lymphocytes (TIL) or PBL of patients with other types of tumors, especially the epithelial cell carcinomas that comprise greater than 80% of human tumors. It appears that there may be tissue specific properties that result in tumor-specific CTLs being sequestered in melanoma.<br>
Direct immunization with tumor-specific gene products may be essential to elicit an immune response against some shared tumor antigens. It has been argued that, if a tumor expressed strong antigens, it should have been eradicated prior to clinical manifestation. Perhaps then, tumors express only weak antigens. Immunologists have long been interested in the issue of what makes an antigen weak or strong. There have been two major hypotheses. Weak antigens may be poorly processed and fail to be presented effectively to T cells. Alternatively, the<br><br>
-4-<br>
number of   T cells in the organism with appropriate specificity might be inadequate for a vigorous response (a so-called "hole in the repertoire"). Elucidation of the complex cellular process whereby antigenic peptides associate with MHC molecules for transport to the cell surface and presentation to T cells has been one of the triumphs of modern immunology. These experiments have clearly established that failure of presentation due to processing defects or competition from other peptides could render a particular peptide less immunogenic. In contrast, it has, for technical reasons, been more difficult to establish that the frequency of clonal representation in the T cell repertoire is an important mechanism of low responsiveness. Recent studies demonstrating that the relationship between immunodominant and cryptic peptides of a protein antigen change in T cell receptor transgenic mice suggest, however, that the relative frequency of peptide-specific T cells can, indeed, be a determining factor in whether a particular peptide is cryptic or dominant in a T cell response. This has encouraging implications for development of vaccines. With present day methods, it would be a complex and difficult undertaking to modify the way in which antigenic peptides of a tumor are processed and presented to T cells. The relative frequency of a specific T cell population can, however, be directly and effectively increased by prior vaccination.  This could, therefore, be the key manipulation required to render an otherwise crypticresponse immunoprotective. These considerations of cryptic or sub-dominant antigens have special relevance in relation to possible immune evasion by tumors through tolerance induction. Evidence has been presented to suggest that tumor-specific T cells in the tumor-bearing host are anergic, possibly as a result of antigen presentation on nonprofessional APC (Morgan, D.J. et al., J. Immunol 163:723-27 (1999); Sotomayor, E.M. et al., Proc. Natl Acad. Sci. U.S.A. 96:\1476-81 (1999); Lee, P.P. et al., Nature Medicine 5:677-85 (1999)).  Prior tolerization of T cells specific for immunodominant antigens of a tumor may, therefore, account for the difficulty in developing successful strategies for immunotherapy of cancer. These observations suggest that T cells specific for immunodominant tumor<br><br>
-5-<br>
antigens are less likely to be effective for immunotherapy of established tumors because they are most likely to have been tolerized. It may, therefore, be that T cells specific for sub-dominant antigens or T cells that are initially present at a lower frequency would prove more effective because they have escaped the tolerizing influence of a growing tumor.<br>
Another major concern for the development of broadly effective human vaccines is the extreme polymorphism of HLA class I molecules. Class I MHC: cellular peptide complexes are the target antigens for specific CD8+ CTLs. The cellular peptides, derived by degradation of endogenously synthesized proteins, are translocated into a pre-Golgi compartment where they bind to class I MHC molecules for transport to the cell surface. The CD8 molecule contributes to the avidity of the interaction between T cell and target by binding to the a3 domain of the class I heavy chain. Since all endogenous proteins turn over, peptides derived from any cytoplasmic or nuclear protein may bind to an MHC molecule and be transported for presentation at the cell surface. This allows T cells to survey a much larger representation of cellular proteins than antibodies which are restricted to recognize conformational determinants of only those proteins that are either secreted or integrated at the cell membrane.<br>
The T cell receptor antigen binding site interacts with determinants of both the peptide and the surrounding MHC. T cell specificity must, therefore, be defined in terms of an MHC: peptide complex. The specificity of peptide binding to MHC molecules is very broad and of relatively low affinity in comparison to the antigen binding sites of specific antibodies. Class I-bound peptides are generally 8-10 residues in length and accommodate amino acid side chains of restricted diversity at certain key positions that match pockets in the MHC peptide binding site. These key features of peptides that bind to a particular MHC molecule constitute a peptide binding motif.<br>
Hence, there exists a need for methods to facilitate the induction and isolation of T cells specific for human tumors, cancers and infected cells and for<br><br>
-6-<br>
methods to efficiently select the genes that encode the major target antigens recognized by these T cells in the proper MHC-context.<br>
BRIEF SUMMARY OF THE INVENTION<br>
The present invention relates to a novel polynucleotide, C35, that is differentially expressed in human breast and bladder carcinoma, and to the encoded polypeptide of C35. Moreover, the present invention relates to vectors, host cells, antibodies, and recombinant methods for producing C35 polypeptides and polynucleotides. The present invention further relates to the formulation and use of C35 polypeptides and polynucleotides in immunogenic compositions to induce antibodies and cell-mediated immunity against target cells, such as tumor cells, that express the C35 gene products. Also provided are diagnostic methods for detecting disorders relating to the C35 genes and polypeptides, including use as a prognostic marker for carcinomas, such as human breast carcinoma, and therapeutic methods for treating such disorders. The invention further relates to screening methods for identifying binding partners of C35.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 (Panels A-B). Panel A shows the DNA coding sequence (SEQ ID NO: 1) of C35. The sequence immediately upstream of the predicted ATG start codon is shown in lower case and conforms to the expected features described by Kozak, M., J. Biol Chem. 266(30)-19867-19870 (1991). Panel B shows the deduced amino acid sequence (SEQ ID NO:2) of C35.<br>
Figure 2. (Panel A to C). Panel A: C35 is over expressed in Breast tumor cell lines. Upper Panel: 300ng of poly-A RNA from 3 week old human thymus, normal breast epithelial cell line H16N2 from patient 21, and 4 breast tumor cell lines derived one year apart from primary or metastatic nodules of the same patient 21; 21NT, 21PT 21MT1, and 21MT2, was resolved on a 1% agarose/formaldehyde gel and transferred to a Gene Screen membrane. This blot<br><br>
-7-<br>
was hybridized with a 32P labeled C35 probe. Hybridization was detected by exposing the blot to film for 15 hours.   Lower Panel:   To quarititate RNA loading, the same blotwas stripped andre-hybridizedwith a 32P labeled probe for Glyceraldehyde-3 Phosphate Dehydrogenase (GAPDH). For each sample the C35 signal was normalized to the GAPDH signal.  The numbers represent the fold expression of C35 in each sample relative to H16N2.   Panel B: C35 is expressed at low levels in normal tissues. A Blot containing 1 microgram of poly-A RNA from each of the indicated adult normal tissues (Clontech) was hybridized with a 32P labeled C35 probe.   Hybridization was detected by exposing the blot to film for 15 hours (upper panel), or 96 hours (lower panel). Panel C. C35 is overexpressed in primary Breast tumors. A blot containing 2 micrograms of poly-A RNA from 3 primary infiltrating ductal mammary carcinoma, T1, T2, T3 and 1 normal breast epithelium, N (Invitrogen) was hybridized with a 32P labeled C35 probe. To normalize loading a 32P labeled beta-Actin probe was included in the hybridization mix.  Hybridization was detected by exposing the blot to film for 6 hours. The numbers represent the fold expression of C35 in each sample relative to normal breast epithelium.<br>
Figure 3. Expression of C35 in Breast Tumor Cell Lines. C35 is overexpressed in different breast tumor cell lines. Upper Panel: 300ng of poly-A RNA from BT474 (ATCC HYB-20, mammary ductal carcinoma), SKBR3 (ATCC HTB-30, mammary adenocarcinoma), T47D (ATCC HTB-133, mammary ductal carcinoma), normal breast epithelial cell line H16N2 from patient 21, and 21 -NT breast tumor cell line derived from primary tumor nodule of the same patient 21 was resolved on a 1% agarose/formaldehyde gel and transferred to a GeneScreen membrane. This blot was hybridized with a 32P labeled C35 probe. Hybridization was detected by exposing the blot to film for 15 hours. Lower Panel: To quantitate RNA loading, the same blot was stripped and re-hybridized with a 32P labeled probe for beta-actin. For each sample the C35 signal was normalized to the actin signal. The numbers represent the fold expression of C35 in each sample relative to H16N2.<br><br>
-8-<br>
Figure 4 (Panels A-C): Surface Expression of C35 Protein Detected by Flow Cytometry. 1 x 105 breast tumor cells were stained with 3.5 microliters of antiserum raised in BALB/c mice against Line 1 mouse tumor cells transduced with retrovirus encoding human C35 or control, pre-bleed BALB/c serum. After a 30 minute incubation, cells were washed twice with staining buffer (PAB) and incubated with FITC-goat anti-mouse IgG (1 ug/sample) for 30 minutes. Samples were washed and analyzed on an EPICS Elite flow cytometer. Panel A: 21NT Panel B: SKBR3. Panel C: MDA-MB-231. These three breast tumor lines were selected to represent tumor cells that express high, intermediate and low levels of C35 RNA on Northern blots (see FIG. 3). Abbreviations: nms, ns; normal mouse serum; C35; C35 immune serum.<br>
Figure 5 (Panels A and B). CML Selected Recombinant Vaccinia cDNA Clones Stimulate Tumor Specific CTL. Panel A: CML Selected vaccinia clones were assayed for the ability, following infection of B/C.N, to stimulate tumor specific CTL to secrete interferon gamma. The amount of cytokine was measured by ELISA, and is represented as OD490 (14). An OD490 of 1.4 is approximately equal to 4ng/ml of IFNg, and an OD490 of 0.65 is approximately equal to 1ng/ml of IFNg. Panel B: CML selected clones sensitize host cells to lysis by tumor specific CTL. Monolayers of B/C.N in wells of a 6 well plate were infected with moi=1 of the indicated vaccinia virus clones. After 14 hours of infection the infected cells were harvested and along with, the indicated control targets labeled with 51Cr. Target cells were incubated with the indicated ratios of tumor specific Cytotoxic T Lymphocytes for 4 hours at 37°C and percentage specific lysis was determined (15). This experiment was repeated at least three times with similar results.<br>
Figure 6 (Panels A and B). The Tumor Antigen Is Encoded by a Ribosomal Protein L3 Gene. Sequence of H2.16 and rpL3 from amino acid position 45 to 56. Panel A: The amino acid (in single letter code) and nucleotide sequence of cDNA clone rpL3 (GenBank Accession no. Y00225). Panel B: A single nucleotide substitution at C170T of the H2.16 tumor cDNA is the only<br><br>
-9-<br>
sequence change relative to the published L3 ribosomal allele. This substitution results in a T54I amino acid substitution in the protein.<br>
Figure 7 (Panels A and B).     Identification of the Peptide Epitope Recognized by the Tumor Specific CTL. Panel A: CML assay to identify the peptide recognized by tumor specific CTL. Target cells were labeled with 51Cr (15).  During the 51Cr incubation samples of B/C.N cells were incubated with 1µM peptide L348_56(154), 100 µM L348_56(T54) or 100µM peptide L345.54(154). Target cells were incubated with the indicated ratios of tumor specific Cytotoxic T Lymphocytes for 4 hours at 37°C and percentage specific lysis was determined.   This experiment was repeated at least three times with similar results. Panel B: Titration of peptide L348_56 (154). Target cells were labeled with 51Cr. During the 51Cr incubation samples of B/C.N cells were incubated either with no peptide addition (D) or with the indicated concentrations (1µM, 1 OnM, 1nM) of L348.56(154) (¦), BCA 39 cells were included as apositive control (?). Target cells were incubated with the indicated ratios of Tumor Specific Cytotoxic T Lymphocytes for 4 hours at 37°C and percentage specific lysis was determined. The experiment was repeated twice with similar results.<br>
Figure 8 (Panels A-C). Analysis of L3 Expressed by Each Cell Line. Panel A: Sau3AI map of published rpL3 and H2.16. Shown above is the Sau3AI restriction map for the published ribosomal protein L3 gene (Top), and for H2.16 (Bottom). Digestion of cDNA for the published L3 sequence generates fragments of 200, 355, 348, 289, and 84bp. The pattern for H2.16 is identical except for an extra Sau3AI site at position 168 caused by the C170T. This results in a 168bp digestion product in place of the 200bp fragment. Panel B: The BCA tumors express both L3 alleles. RT-PCR products generated from each cell line or from vH2.16 were generated using L3 specific primers and then digested with Sau3AI, and resolved on a 3% agarose gel for 2 hours at 80 volts. Panel C: The Immunogenic L3 allele is expressed at greatly reduced levels in B/C.N, BCB13, and Thymus. L3 specific RT-PCR products from each indicated sample were generated using a 32P end labeled 5 prime PCR primer. No PCR<br><br>
-10-<br>
product was observed when RNA for each s ample was used as template for PCR without cDNA synthesis, indicating that no sample was contaminated with genomic DNA. The PCR products were gel purified to ensure purity, digested with Sau3AI, and resolved on a 3% agarose gel for 15 hours at 60 volts. No PCR product was observed in a control PCR sample that had no template added to it. This result has been reproduced a total of 3 times.<br>
Figure 9 (Panels A-C). Immunization with iL3 is Immunoprotective. Panel A: Immunization with H2.16 induces tumor specific CTL. Balb/c mice (2/group) were immunized by subcutaneous inj ection with 5X106 pfu of vH2.16, or control vector v7.5/tk. Seven days later splenocytes were harvested and restimulated with peptide L348-56(I54) (26). Five days following the second restimulation the lymphocytes were tested in a chromium release assay as described in Figure 11. The L348-56(I54) peptide was used at a 1 micromolar concentration, and the L348-56(T54) peptide was used at a 100 micromolar concentration. Similar results were obtained when the immunization experiment was repeated. Panels B and C: Female Balb/cByJ mice were immunized as indicated (27). The mice were challenged by SC injection with 200,000 viable BCA 34 tumor cells into the abdominal wall. Data is from day 35 post challenge. These data are representative of 4 independent experiments.<br>
Figure 10 (Panels A and B). Panel A: C35 coding sequence with translation; 5' and 3' untranslated regions are shown in lowercase letters. The predicted prenylation site, CVIL, at the 3' terminus is boxed Panel B: Genomic alignment of C35 gene on chromosome 17.<br>
Figure 11 (Panels A and B). C35 Expression in Breast Carcinoma. C35 was labeled with 32P in a random priming reaction and hybridized to Northern blots at 10s cpm/ml. Each blot was stripped andre-probed with GAPDH or Beta-actin to normalize mRNA loads. The numbers indicate densitometry ratios normalized against GAPDH/Beta-actin. A value of 1 has been assigned to normal cell line H16N2, and all values are relative to the level of expression in the normal cell line.  Panel A: C35 expression in breast epithelial cell lines.<br><br>
-11-<br>
Panel B: C35 expression in primary breast tissue/tumors. 300 ng mRNA was electrophoresed on 0.8% alkaline agarose gels, then blotted to Genescreen Plus, except leftmost panel of B loaded with 1 µg mRNA from 3 primary tumors and<br>
1	normal tissue control (Real Tumor Blots, Invitrogen). Similar exposures are<br>
shown for all blots.<br>
Figure 12. C35 Expression in Bladder Carcinoma. C35 was labeled with 32P in a random priming reaction and hybridized to a Northern blot of tumor and normal RRA at 106cpm/ml. The blot was stripped and re-probed with Beta-actin to normalize mRNA loads. The numbers indicate densitometry ratios normalized against Beta-actin. Values are relative to the level of expression in the normal bladder samples. 300 ng mRNA was electrophoresed on 0.8% alkaline agarose gels, then blotted to Genescreen Plus.<br>
Figure 13 (Panels A and B). FACS Analysis with Anti-C35 Antibodies. Panel A: Breast cell lines were stained with (top panel) sera from mice immunized with Line 1 cells infected with C35 recombinant retrovirus, and (bottom panel) 2C3 purified monoclonal antibody or isotype control. Panel B; Bladder cell lines stained with 2C3 purified monoclonal antibody or isotype control.<br>
Figure 14. Inhibition of Tumor Growth in Presence of 2C3 Antibody.<br>
21NT breast tumor cells or H1 6N2 normal breast epithelial cells were incubated with the indicated concentrations of 2C3 anti-C35 monoclonal antibody or a nonspecific isotype control antibody. Cell growth was measured by XTT assay following 72 hour incubation in the presence or absence of antibodies.<br>
Figure 15 (Panels A and B). CTL stimulated with C35 expressing dendritic cells specifically lyse C35+ Breast (21NT) and Bladder (ppT11A3) tumor cell lines, with rninimal activity against normal breast (MEC), irnmortalizednon-tumorigenic breast (H16N2) andbladder (SV-HUC) cell lines, or an NK sensitive cell line (K562). Panel A: T cell line 4 was generated from normal human PBL. Panel B: T cell clone 10G3 was selected from line 4 for C35-specific activity.   Target cell lines MEC, ppT11A3 and SV-HUC are<br><br>
-12-<br>
naturally HLA-A2 positive. Target cell lines 21NT and H16N2 were transected with HLA-A2 to provide a required MHC restriction element.<br>
Figure 16 (Panels A and B). Cytokine Release from T Cell Clone 10G3 upon Stimulation with Targets. Panel A: IFN-gamma secretion. Panel B: TNF-alpha secretion. Breast and bladder target cell lines were distinguished by the presence or absence of expression of HLA-A2 and C35 tumor antigen, an amino terminal 50 amino acid fragment of C35 (C35-50aa), or the irrelevant mouse L3 ribosomal protein. Each marker was either endogenously expressed or introduced by transfection of an HLA-A2.1 construct (pS V2. A2), or by infection with a vaccinia recombinant of C35 (vv.C35, vv.C55-55aa), L3 (vv.L3), or HLA-A2 (w.A2)<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Definitions<br>
The following definitions are provided to facilitate understanding of certain terms used throughout this specification.<br>
In the present invention, "isolated" refers to material removed from its native environment (e.g., the natural environment if it is naturally occurring), and thus is altered "by the hand of man" from its natural state. For example, an isolated polynucleotide could be part of a vector or a composition of matter, or could be contained within a cell, and still be "isolated" because that vector, composition of matter, or particular cell is not the original environment of the polynucleotide.<br>
In the present invention, a "membrane" C35 protein is one expressed on the cell surface through either direct or indirect association with the lipid bilayer, including, in particular, through prenylation of a carboxyl-terminal amino acid motif. Prenylation involves the covalent modification of a protein by the addition of either a farnesyl or geranylgeranyl isoprenoid. Prenylation occurs on<br><br>
-13-<br>
a cysteine residue located near the carboxyl-terminus of a protein.  The C35 polypeptide contains the amino acids Cys-Val-He-Leu at positions 112-115, with the Leu being the C terminal residue of the polypeptide. The motif Cys-X-X-Leu, where "X" represents any aliphatic amino acid, results in the addition of a 20 carbon geranylgeranyl group onto the Cys residue.   Generally, following addition of this lipid the three terminal amino acid residues are cleaved off the polypeptide, and the lipid group is methylated.   Prenylation promotes the membrane localization of most proteins, with sequence motifs in the polypeptide being involved in directing the prenylated protein to the plasma, nuclear, or golgi membranes. Prenylation plays a role in protein-protein interactions, and many prenylated proteins are involved in signal transduction. Examples of prenylated proteins include Ras and the nuclear lamin B. (Zhang, F.L. and Casey, P.J., Ann. Rev. Biochem. 65:241-269 (1996)). The C35 protein has been detected on the surface of two breast tumor cell lines by fluorescence analysis employing as a primary reagent a mouse anti-human C35 antiserum (Figure 4).<br>
In the present invention, a "secreted" C35 protein refers to a protein capable of being directed to the ER, secretory vesicles, or the extracellular space as a result of a signal sequence, as well as a C35 protein released into the extracellular space without necessarily containing a signal sequence. If the C35 secreted protein is released into the extracellular space, the C35 secreted protein can undergo extracellular processing to produce a "mature" C35 protein. Release into the extracellular space can occur by many mechanisms, including exocytosis and proteolytic cleavage.<br>
As used herein, a C35 "polynucleotide" refers to a molecule having a nucleic acid sequence contained in SEQ ID NO:1. For example, the C35 polynucleotide can contain the nucleotide sequence of the full length cDNA sequence, including the 5' and 3' untranslated sequences, the coding region, with or without the signal sequence, the secreted protein coding region, as well as fragments, epitopes, domains, and variants of the nucleic acid sequence. Moreover, as used herein, a C35 "polypeptide" refers to a molecule having the<br><br>
-14-<br>
translated amino acid sequence generated from the polynucleotide as broadly defined.<br>
In specific embodiments, the polynucleotides of the invention are less than 300 nt, 200 nt, 100 nt, 50 nt, 15 nt, 10 nt, or 7 nt in length. In a further embodiment, polynucleotides of the invention comprise at least 15 contiguous nucleotides of C35 coding sequence, but do not comprise all or a portion of any C35 intron. In another embodiment, the nucleic acid comprising C35 coding sequence does not contain coding sequences of a genomic flanking gene (i.e., 5' or 3' to the C35 gene in the genome).<br>
In the present invention, the full length C35 coding sequence is identified as SEQ ID NO: 1.<br>
A C35 "polynucleotide" also refers to isolated polynucleotides which encode the C35 polypeptides, and polynucleotides closely related thereto.<br>
A C35 "polynucleotide" also refers to isolated polynucleotides which encode the amino acid sequence shown in SEQ ID NO: 2, or a biologically active fragment thereof.<br>
A C35 "polynucleotide" also includes those polynucleotides capable of hybridizing, under stringent hybridization conditions, to sequences contained in SEQ ID NO: 1, the complement thereof, or the cDNA within the deposited clone. "Stringent hybridization conditions" refers to an overnight incubation at 42° C in a solution comprising 50% formamide, 5x SSC (750 mM NaCl, 75 mM sodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardt's solution, 10% dextran sulfate, and 20 µg/ml denatured, sheared salmon sperm DNA, followed by washing the filters in 0.1x SSC at about 65°C.<br>
Of course, a polynucleotide which hybridizes only to polyA+ sequences (such as any 3' terminal polyA+ tract of a cDNA), or to a complementary stretch of T (or U) residues, would not be included in the definition of "polynucleotide," since such a polynucleotide would hybridize to any nucleic acid molecule containing a poly (A) stretch or the complement thereof (e.g., practically any double-stranded cDNA clone).<br><br>
-15-<br>
The C35 polynucleotide can be composed of any polyribonucleotide or polydeoxyribonucleotide, which may be unmodified RNA or DNA or modified RNA or DNA. For example, C35 polynucleotides can be composed of single-and double-stranded DNA, DNA that is a mixture of single- and double-stranded regions, single- and double-stranded RNA, and RNA that is mixture of single-and double-stranded regions, hybrid molecules comprising DNA and RNA that may be single-stranded or, more typically, double-stranded or a mixture of single- and double-stranded regions. In addition, the C35 polynucleotides can be composed of triple-stranded regions comprising RNA or DNA or both RNA and DNA. C35 polynucleotides may also contain one or more modified bases or DNA or RNA backbones modified for stability or for other reasons. "Modified" bases include, for example, tritylated bases and unusual bases such as inosine. A variety of modifications can be made to DNA and RNA; thus, "polynucleotide" embraces chemically, enzymatically, or metabolically modified forms.<br>
C35 polypeptides can be composed of amino acids joined to each other by peptide bonds or modified peptide bonds, i.e., peptide isosteres, and may contain amino acids other than the 20 gene-encoded amino acids. The C35 polypeptides may be modified by either natural processes, such as posttranslational processing, or by chemical modification techniques which are well known in the art. Such modifications are well described in basic texts and in more detailed monographs, as well as in a voluminous research literature. Modifications can occur anywhere in the C35 polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. It will be appreciated that the same type of modification may be present in the same or varying degrees at several sites in a given C35 polypeptide. Also, a given C35 polypeptide may contain many types of modifications. C35 polypeptides may be branched, for example, as a result of ubiqiutination, and they may be cyclic, with or without branching. Cyclic, branched, and branched cyclic C35 polypeptides may result from posttranslation natural processes or may be made<br><br>
-16-<br>
by   synthetic   methods.       Modifications   include   acetylation,   acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of phosphotidylinositol, cross-linking, cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxy lation, iodination, methylation, myristoylation, oxidation, pegylation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, transfer-RNA mediated addition of amino acids to proteins such as arginylation, and ubiquitination.  (See, for instance, Proteins - Structure And Molecular Properties, 2nd Ed., T. B.  Creighton, W. H.   Freeman and Company, New York (1993); Posttranslational Covalent Modification of Proteins, B. C. Johnson, Ed., Academic Press, New York, pgs. 1-12 (1983); Seifter et al., Meth Enzymol 182:626-646 (1990); Rattan et al., Ann NY Acad Sci 663:48-62 (1992).)<br>
"SEQ ID NO: 1" refers to a C35 polynucleotide sequence while "SEQ ID NO: 2" refers to a C35 polypeptide sequence.<br>
A C35 polypeptide "having biological activity" refers to polypeptides exhibiting activity similar to, but not necessarily identical to, an activity of a C35 polypeptide, including mature forms, as measured in a particular biological assay, with or without dose dependency. In the case where dose dependency does exist, it need not be identical to that of the C35 polypeptide, but rather substantially similar to the dose-dependence in a given activity as compared to the C35 polypeptide (i.e., the candidate polypeptide will exhibit greater activity or not more than about 25-fold less and, preferably, not more than about tenfold less activity, and most preferably, not more than about three-fold less activity relative to the C35 polypeptide.)<br><br>
-17-<br>
C35 Polynucleotides and Polypeptides<br>
A 348 base pair fragment of C35 was initially isolated by subtractive hybridization of poly-A RNA from tumor and normal mammary epithelial cell lines derived from the same patient with primary and infiltrating intraductal mammary carcinoma.   Band, V. et al., Cancer Res. 50:7351-7357 (1990). Employing primers based on this sequence and that of an overlapping EST sequence (Accession No. W57569), a cDNA that includes the full-length C35 coding sequence was then amplified and cloned from the BT-20 breast tumor cell line (ATCC, HTB-19).   This C35 cDNA contains the entire coding region identified as SEQ ID NO: 1. The C35 clone includes, in addition to the 348 bp coding sequence, 167 bp of 3' untranslated region.  The open reading frame begins at an N-terrninal methionine located at nucleotide position 1, and ends at a stop codon at nucleotide position 348 (FIG. 1).   A representative clone containing all or most of the sequence for SEQ ID NO: 1 was deposited with the American Type Culture Collection ("ATCC") on August 1,2000, and was given the ATCC Deposit Number PTA-2310.    The ATCC is located at 10801 University Boulevard, Manassas, VA 20110-2209, USA. The ATCC deposit was made pursuant to the terms of the Budapest Treaty on the international recognition of the deposit of microorganisms for purposes of patent procedure.<br>
Therefore, SEQ ID NO: 1 and the translated SEQ ID NO: 2 are sufficiently accurate and otherwise suitable for a variety of uses well known in the art and described further below. For instance, SEQ ID NO: 1 is useful for designing nucleic acid hybridization probes that will detect nucleic acid sequences contained in SEQ ID NO: 1 or the cDNA contained in the deposited clone. These probes will also hybridize to nucleic acid molecules in biological samples, thereby enabling a variety of forensic and diagnostic methods of the invention. Similarly, polypeptides identified from SEQ ID NO:2 may be used to generate antibodies which bind specifically to C35, or to stimulate T cells<br><br>
-18-<br>
which are specific for C35 derived peptides in association with MHC molecules on the cell surface.<br>
Nevertheless, DNA sequences generated by sequencing reactions can contain sequencing errors. The errors exist as misidentified nucleotides, or as insertions or deletions of nucleotides in the generated DNA sequence. The erroneously inserted or deleted nucleotides cause frame shifts in the reading frames of the predicted amino acid sequence. In these cases, the predicted amino acid sequence diverges from the actual amino acid sequence, even though the generated DNA sequence may be greater than 99.9% identical to the actual DNA sequence (for example, one base insertion or deletion in an open reading frame of over 1000 bases).<br>
Accordingly, for those applications requiring precision in the nucleotide sequence or the amino acid sequence, the present invention provides not only the generated nucleotide sequence identified as SEQ ID NO:1 and the predicted translated amino acid sequence identified as SBQ ID NO:2. The nucleotide sequence of the deposited C35 clone can readily be determined by sequencing the deposited clone in accordance with known methods. The predicted C35 amino acid sequence can then be verified from such deposits. Moreover, the amino acid sequence of the protein encoded by the deposited clone can also be directly determined by peptide sequencing or by expressing the protein in a suitable host cell containing the deposited human C35 cDNA, collecting the protein, and determining its sequence.<br>
The present invention also relates to the C35 gene corresponding to SBQ ID NO: 1, or the deposited clone. The C35 gene can be isolated in accordance with known methods using the sequence information disclosed herein. Such methods include preparing probes or primers from the disclosed sequence and identifying or amplifying the C35 gene from appropriate sources of genomic material.<br>
Also provided in the present invention are species homologs of C35. Species homologs may be isolated and identified by making suitable probes or<br><br>
-19-<br>
primers from the sequences provided herein and screening a suitable nucleic acid source for the desired homologue.<br>
By "C35 polypeptide(s)" is meant all forms of C35 proteins and polypeptides described herein. The C35 polypeptides can be prepared in any suitable manner. Such polypeptides include isolated naturally occurring polypeptides, recombinantly produced polypeptides, synthetically produced polypeptides, or polypeptides produced by a combination of these methods. Means for preparing such polypeptides are well understood in the art.<br>
The C35 polypeptides may be in the form of the membrane protein or a secreted protein, including the mature form, or may be a part of a larger protein, such as a fusion protein (see below). It is often advantageous to include an additional amino acid sequence which contains secretory or leader sequences, pro-sequences, sequences which aid in purification, such as multiple histidine residues, or an additional sequence for stability during recombinant production.<br>
C35 polypeptides are preferably provided in an isolated form, and preferably are substantially purified. A recombinantly produced version of a C35 polypeptide, including the secreted polypeptide, can be substantially purified by the one-step method described in Smith and Johnson, Gene 67:31-40 (1988), C35 polypeptides also can be purified from natural or recombinant sources using antibodies of the invention raised against the C35 protein in methods which are well known in the art.<br>
Polynucleotide and Polypeptide Variants<br>
"Variant" refers to apolynucleotide orpolypeptide differing from the C35 polynucleotide or polypeptide, but retaining essential properties thereof. Generally, variants are overall closely similar, and, in many regions, identical to the C35 polynucleotide or polypeptide.<br>
By a polynucleotide having a nucleotide sequence at least, for example, 95% "identical" to a reference nucleotide sequence of the present invention, it is<br><br>
-20-<br>
intended that the nucleotide sequence of the polynucleotide is identical to the reference sequence except that the polynucleotide sequence may include up to five point mutations per each 100 nucleotides of the reference nucleotide sequence encoding the C35 polypeptide. In other words, to obtain a polynucleotide having a nucleotide sequence at least 95% identical to a reference nucleotide sequence, up to 5% of the nucleotides in the reference sequence may be deleted or substituted with another nucleotide, or a number of nucleotides up to 5% of the total nucleotides in the reference sequence may be inserted into the reference sequence. The query sequence may be an entire sequence shown of SEQ ID NO:1, the ORF (open reading frame), or any fragment specified as described herein.<br>
As a practical matter, whether any particular nucleic acid molecule or polypeptide is at least 90%, 95%, 96%, 97%, 98% or 99% identical to a nucleotide sequence of the presence invention can be determined conventionally using known computer programs. A preferred method for determining the best overall match between a query sequence (a sequence of the present invention) and a subject sequence, also referred to as a global sequence alignment, can be determined using the FASTDB computer program based on the algorithm of Brutlag et al., Comp. App. Biosci 6:237-245 (1990). la a sequence alignment the query and subject sequences are both DNA sequences. An RNA sequence can be compared by converting U's to T's. The result of said global sequence alignment is in percent identity.   Preferred parameters used in a FASTDB alignment of DNA sequences to calculate percent identity are: Matrix=Unitary, k-tuple=4, Mismatch Penalty=1, Joining Penalty=30, Randomization Group Length=0, Cutoff Score=1, Gap Penalty=5, Gap Size Penalty 0.05, Window Size=500 or the length of the subject nucleotide sequence, whichever is shorter.<br>
If the subject sequence is shorter man the query sequence because of 5' or 3' deletions, not because of internal deletions, a manual correction must be made to the results. This is because the FASTDB program does not account for 5' and 3' truncations of the subject sequence when calculating percent identity.<br><br>
-21-<br>
For subject sequences truncated at the 5' or 3' ends, relative to the query sequence, the percent identity is corrected by calculating the number of bases of the query sequence that are 5' and 3' of the subject sequence, which are not matched/aligned, as a percent of the total bases of the query sequence. Whether a nucleotide is matched/aligned is determined by results of the FASTDB sequence alignment. This percentage is then subtracted from the percent identity, calculated by the above FASTDB program using the specified parameters, to arrive at a final percent identity score. This corrected score is what is used for the purposes of the present invention. Only bases outside the 5' and 3' bases of the subject sequence, as displayed by the FASTDB alignment, which are not matched/aligned with the query sequence, are calculated for the purposes of manually adjusting the percent identity score.<br>
For example, a 90 base subject sequence is aligned to a 100 base query sequence to determine percent identity. The deletions occur at the 5' end of the subject sequence and therefore, the FASTDB alignment does not show a matched/alignment of the first 10 bases at 5' end. The 10 unpaired bases represent 10% of the sequence (number of bases at the 5' and 3' ends not matched/total number of basesin the query sequence) so 10% is subtracted from the percent identity score calculated by the FASTDB program. If the remaining 90 bases were perfectly matched the final percent identity would be 90%. In<br>
i<br>
another example, a 90 base subject sequence is compared with a 100 base query sequence. This time the deletions are internal deletions so that there are no bases on the 5' or 3' of the subject sequence which are not matched/aligned with the query. In this case the percent identity calculated by FASTDB is not manually corrected. Once again, only bases 5' and 3' of the subject sequence which are not matched/aligned with the query sequence are manually corrected for. No other manual corrections are to made for the purposes of the present invention.<br>
By a polypeptide having an amino acid sequence at least, for example, 95% "identical" to a query amino acid sequence of the present invention, it is intended that the amino acid sequence of the subject polypeptide is identical to<br><br>
-22-<br>
the query sequence except that the subject polypeptide sequence may include up to five amino acid alterations per each 100 amino acids of the query amino acid sequence. In other words, to obtain a polypeptide having an amino acid sequence at least 95% identical to a query amino acid sequence, up to 5% of the amino acid residues in the subject sequence may be inserted, deleted, or substituted with another amino acid. These alterations of the reference sequence may occur at the amino or carboxy terminal positions of the reference amino acid sequence or anywhere between those terminal positions, interspersed either individually among residues in the reference sequence or in one-or more contiguous groups within the reference sequence.<br>
As a practical matter, whether any particular polypeptide is at least 90%, 95%, 96%, 97%, 98% or 99% identical to, for instance, the amino acid sequence shown in SEQ ID NO:2 or to the amino acid sequence encoded by deposited DNA clone, can be determined conventionally using known computer programs. A preferred method for determining the best overall match between a query sequence (a sequence of the present invention) and a subject sequence, also referred to as a global sequence alignment, can be determined using the FASTDB computer program based on the algorithm of Brutlag et al., Comp. App. Bio sci. 6:237-245 (1990). In a sequence alignment the query and subject sequences are either both nucleotide sequences or both amino acid sequences. The result of said global sequence alignment is in percent identity. Preferred parameters used in a FASTDB amino acid alignment are: Matrix=PAM 0, k-tuple=2, Mismatch Penalty=1, Joining Penalty=20, Randomization Group Length=0, Cutoff Score=l, WindowSize=sequencelength, Gap Penalty=5, Gap Size Penalty=0.05, Window Size=500 or the length of the subject amino acid sequence, whichever is shorter.<br>
If the subject sequence is shorter than the query sequence due to N- or C-terminal deletions, not because of internal deletions, a manual correction must be made to the results. This is because the FASTDB program does not account for N- and C-terminal truncations of the subject sequence when calculating<br><br>
-23-<br>
global percent identity. For subject sequences truncated at the N- and C-termini, relative to the query sequence, the percent identity is corrected by calculating the number of residues of the query sequence that are N- and C-terminal of the subject sequence, which are not matched/aligned with a corresponding subject residue, as a percent of the total bases of the query sequence. Whether a residue is matched/aligned is determined by results of the FASTDB sequence alignment. This percentage is then subtracted from the percent identity, calculated by the above FASTDB program using the specified parameters, to arrive at a final percent identity score. This final percent identity score is what is used for the purposes of the present invention. Only residues to the N- and C-termini of the subject sequence, which are not matched/aligned with the query sequence, are considered for the purposes of manually adjusting the percent identity score. That is, only query residue positions outside the farthest N- and C-terminal residues of the subject sequence.<br>
For example, a 90 amino acid residue subject sequence is aligned with a 100 residue query sequence to determine percent identity. The deletion occurs at the N-terminus of the subject sequence and therefore, the FASTDB alignment does not show a matching/alignment of the first 10 residues at the N-terminus. The 10 unpaired residues represent 10% of the sequence (number of residues at the N- and C- termini not matched/total number of residues in the query sequence) so 10% is subtracted from the percent identity score calculated by the FASTDB program. If the remaining 90 residues were perfectly matched the final percent identity would be 90%. In another example, a 90 residue subject sequence is compared with a 100 residue query sequence. This time the deletions are internal deletions so there are no residues at the N- or C-termini of the subject sequence which are not matched/aligned with the query. In this case the percent identity calculated by FASTDB is not manually corrected. Once again, only residue positions outside the N- and C-terminal ends of the subject sequence, as displayed in the FASTDB alignment, which are not matched/aligned with the<br><br>
-24-<br>
query sequence are manually corrected for. No other manual corrections are to be made for the purposes of the present invention.<br>
The C35 variants may contain alterations in the coding regions, non-coding regions, or both. Especially preferred are polynucleotide variants containing alterations which produce silent substitutions, additions, or deletions, but do not alter the properties or activities of the encoded polypeptide. Nucleotide variants produced by silent substitutions due to the degeneracy of the genetic code are preferred. Moreover, variants in which 5-10,1-5, or 1-2 amino acids are substituted, deleted, or added in any combination are also preferred. C35 polynucleotide variants can be produced for a variety of reasons, e.g., to optimize codon expression for a particular host (change codons in the human mRNA to those preferred by a bacterial host such as E. coli).<br>
Naturally occurring C35 variants are called "allelic variants," and refer to one of several alternate forms of a gene occupying a given locus on a chromosome of an organism. (Genes II, Lewin, B., ed, John Wiley &amp; Sons, New York (1985).) Also, allelic variants can occur as "tandem alleles" which are highly homologous sequences that occur at different loci on chromosomes of an organism. These allelic variants can vary at either the polynucleotide and/or polypeptide level. Alternatively, non-naturally occurring variants may be produced by mutagenesis techniques or by direct synthesis.<br>
Using known methods of protein engineering and recombinant DNA technology, variants may be generated to improve or alter the characteristics of the C35 polypeptides. For instance, one or more amino acids can be deleted from the N-terminus or C-terminus of the secreted protein without substantial loss of biological function. The authors of Ron et al., J. Biol. Chem. 268: 2984-2988 (1993),reportedvariant KGF protems having heparin binding activity even after deleting 3, 8, or 27 ammo-terminal amino acid residues. Similarly, Interferon gamma exhibited up to ten times higher activity after deleting 8-10 amino acid residues from the carboxy terminus of this protein (Dobeli et al., J. Biotechnology 7:199-216 (1988)).<br><br>
-25-<br>
Moreover, ample evidence demonstrates that variants often retain a biological activity similar to that of the naturally occurring protein. For example, Gayle and coworkers (J. Biol. Chem 268:22105-22111 (1993)) conducted extensive mutational analysis of human cytokine IL-1a.   They used random mutagenesis to generate over 3,500 individual IL-1a mutants that averaged 2.5 amino acid changes per variant over the entire length of the molecule. Multiple mutations were examined at every possible amino acid position.     The investigators found that" [m]ost of the molecule could be altered with little effect on either [binding or biological activity]."   (See, Abstract.) In fact, only 23 unique amino acid sequences, out of more than 3,500 nucleotide sequences examined, produced a protein that significantly differed in activity from wild-type.<br>
Furthermore, even if deleting one or more amino acids from the N-terminus or C-terminus of a polypeptide results in modification or loss of one or more biological functions, other biological activities may still be retained. For example, the ability of a deletion, variant to induce and/or to bind antibodies which recognize the secreted form will likely be retained when less than the majority of the residues of the secreted form are removed from the N-tenninus or C-terrninus. Whether a particular polypeptide lacking N- or C-terminal residues of a protein retains such immunogenic activities can readily be determined by routine methods described herein and otherwise known in the art.<br>
Thus, the invention further includes C35 polypeptide variants which show substantial biological activity. Such variants include deletions, insertions, inversions, repeats, and substitutions selected according to general rules known in the art so as to have little effect on activity. For example, guidance concerning how to make phenotypically silent amino acid substitutions is providedin Bowie, J. U. et al., Science 247:1306-1310 (1990), wherein the authors indicate that there are two main strategies for studying the tolerance of an amino acid sequence to change.<br><br>
-26-<br>
The first strategy exploits the tolerance of amino acid substitutions by natural selection during the process of evolution. By comparing amino acid sequences in different species, conserved amino acids can be identified. These conserved amino acids are likely important for protein function. In contrast, the amino acid positions where substitutions have been tolerated by natural selection indicates that these positions are not critical for protein function. Thus, positions tolerating amino acid substitution could be modified while still maintaining biological activity of the protein.<br>
The second strategy uses genetic engineering to introduce amino acid changes at specific positions of a cloned gene to identify regions critical for protein function. For example, site directed mutagenesis or alanine-scanning mutagenesis (introduction of single alanine mutations at every residue in the molecule) can be used. (Cunningham and Wells, Science 244:1081-1085 (1989).) The resulting mutant molecules can then be tested for biological activity.<br>
As the authors state, these two strategies have revealed that proteins are surprisingly tolerant of amino acid substitutions. The authors further indicate which amino acid changes are likely to be permissive at certain amino acid positions in the protein. For example, most buried (within the tertiary structure of the protein) amino acid residues require nonpolar side chains, whereas few features of surface side chains are generally conserved. Moreover, tolerated conservative amino acid substitutions involve replacement of the aliphatic or hydrophobic amino acids Ala, Val, Leu and IIe; replacement of the hydroxyl residues Ser and Thr, replacement of the acidic residues Asp and Glu; replacement of the amide residues Asn and Gin, replacement of the basic residues Lys, Arg, and His; replacement of the aromatic residues Phe, Tyr, and Trp, and replacement of the small-sized amino acids Ala, Ser, Thr, Met, and Gly.<br>
Besides conservative amino acid substitution, variants of C35 include (i) substitutions with one or more of the non-conserved amino acid residues, where the substituted amino acid residues may or may not be one encoded by the<br><br>
-27-<br>
genetic code, or (ii) substitution with one or more of amino acid residues having a substituent group, or (iii) fusion of the mature polypeptide with another compound, such as a compound to increase the stability and/or solubility of the polypeptide (for example, polyethylene glycol), or (iv) fusion of the polypeptide with additional amino acids, such as an IgG Fc fusion region peptide, or leader or secretory sequence, or a sequence facilitating purification. Such variant polypeptides are deemed to be within the scope of those skilled in the art from the teachings herein.<br>
For example, C35 polypeptide variants containing amino acid substitutions of charged amino acids with other charged or neutral amino acids may produce proteins with improved characteristics, such as less aggregation. Aggregation of pharmaceutical formulations both reduces activity and increases clearance due to the aggregate's immunogenic activity. (Pinckard et al., Clin. Exp. Immunol. 2:331-340 (1967); Robbins et al., Diabetes 36: 838-845 (1987); Cleland et al., Crit. Rev. Therapeutic Drug Carrier Systems 10:201-311 (1993).)<br>
Polynucleotide and Polypeptide Fragments<br>
In the present invention, a "polynucleotide fragment" refers to a short polynucleotide having a nucleic acid sequence contained in the deposited clone or shown in SEQ ID NO:1. The short nucleotide fragments are preferably at least about 15 nt, and more preferably at least about 20 nt, still more preferably at least about 30 nt, and even more preferably, at least about 40 nt in length. A fragment "at least 20 nt in length," for example, is intended to include 20 or more contiguous bases from the cDNA sequence contained in the deposited clone or the nucleotide sequence shown in SEQ ID NO: 1. These nucleotide fragments are useful as diagnostic probes and primers as discussed herein. Of course, larger fragments (e.g., at least 50, 100, 150, 200, 250, 300 nucleotides) are preferred<br>
Moreover, representative examples of C35 polynucleotide fragments include, for example, fragments having a sequence from about nucleotide number<br><br>
-28-<br>
1-50, 51-100, 101-150, 151-200, 201-250, 251-300, or 301 to the end of SEQ ID NO: 1 or the cDNA contained in the deposited clone. In this context "about" includes the particularly recited ranges, larger or smaller by several (5, 4, 3, 2, or 1) nucleotides, at either terminus or at both termini. Preferably, these fragments encode a polypeptide which has biological activity. More preferably, these polynucleotides can be used as probes or primers as discussed herein.<br>
In the present invention, a "polypeptide fragment" refers to a short amino acid sequence contained in SEQ ID NO:2 or encoded by the cDNA contained in the deposited clone. Protein fragments may be "free-standing," or comprised within a larger polypeptide of which the fragment forms a part or region, most preferably as a single continuous region. Representative examples of polypeptide fragments of the invention, include, for example, fragments from about amino acidnumber 1-20, 21-40, 41-60, 61-80, 81-100, or 101 to the end of the coding region. Moreover, polypeptide fragments can comprise 9, 15, 20, 30, 40, 50, 60, 70, 80, 90, or 100 amino acids in length. In this context "about" includes the particularly recited ranges, larger or smaller by several .(5, 4, 3, 2, or 1) amino acids, at either extreme or at both extremes.<br>
Preferred polypeptide fragments include the secreted C35 protein as well as the mature form. Further preferred polypeptide fragments include the secreted C35 protein or the mature form having a continuous series of deleted residues from the amino or the carboxy terminus, or both.<br>
As mentioned above, even if deletion of one or more amino acids from the N-terminus of a protein results in modification or loss of one or more biological functions of the protein, other biological activities may still be retained. Thus, the ability of shortened C35 muteins to induce and/or bind to antibodies which recognize the complete or mature forms of the polypeptides generally will be retained when less than the majority of the residues of the complete or mature polypeptide are removed from the N-terminus. Whether a particular polypeptide lacking N-terminal residues of a complete polypeptide retains such immunologic activities can readily be determined by routine<br><br>
-29-<br>
methods described herein and otherwise known in the art. It is not unlikely that a C35 mutein with a large number of deleted N-terminal amino acidresidues may retain some biological or immunogenic activities. In fact, peptides composed of as few as 9 C35 amino acid residues may often evoke an immune response.<br>
Accordingly, the present invention further provides polypeptides having one or more residues deleted from the amino terminus of the C35 amino acid sequence shown in SEQ ID NO:2, up to the Threonine residue at position number 105 and polynucleotides encoding such polypeptides.<br>
Also as mentioned above, even if deletion of one or more amino acids from the C-terminus of a protein results in modification or loss of one or more biological functions of the protein, other biological activities may still be retained. Thus, the ability of the shortened C35 mutein to induce and/or bind to antibodies which recognize the complete or mature forms of the polypeptide generally will be retained when less than the majority of the residues of the<br>
complete or mature polypeptide are removed from the C-terminus. Whether a<br>
i<br>
particular polypeptide lacking C-terminal residues of a complete polypeptide retains such immunologic activities can readily be determined by routine methods described herein and otherwise known in the art. It is not unlikely that a C35 mutein with alargenumber of deleted C-terminal amino acidresidues may retain some biological or immunogenic activities.<br>
Accordingly, the present invention further provides polypeptides having one or more residues deleted from the carboxy terminus of the amino acid sequence of the C35 polypeptide shown in SEQ ID NO:2, up to the valine residue at position number 10, and polynucleotides encoding such polypeptides<br>
Moreover, the invention also provides polypeptides having one or more amino acids deleted from both the amino and the carboxyl termini. In preferred embodiments, the invention is directed to polypeptides having residues: S-9 to V-17; V-10 to V-17; E-16 to V-23; E-16 to R-24; E-16 to I-25; S-21 to F-35; C-30 to T-38; E-31 to Y-39; E-36 to A-43; A-37 to A-45; A-37 to V-46; Y-39 to y-46; S-44 to I-53; A-45 to I-53; G-52 to I-59; E-54 to T-62; S-57 to F-75; R-<br><br>
-30-<br>
58 to I-67; G-61 to I-69; G-63 to F-83; E-66 to L-73; E-66 to V-74; F-83 to E-103; D-88 to A-96; L-89 to A-96; A-92 to T-101; R-95 to L-102; A-96 to K-104; K-104 to V-113; I-105 to V-113; I-105 to I-114 of SEQ ID NO:2, and polynucleotides encoding such polypeptides.<br>
Many polynucleotide sequences, such as EST sequences, are publicly available and accessible through sequence databases.<br>
The human EST sequences referred to below were identified in a BLAST search of the EST database. These sequences are believed to be partial sequences of the cDNA inserts identified in the recited GenBank accession numbers. No homologous sequences were identified in a search of the annotated GenBank database. The Expect value (E) is a parameter that describes the number of hits one can "expect" to see just by chance when searching a database of a particular size. It decreases exponentially with the Score (S) that is assigned to a match between two sequences. Essentially, the E value describes the random background noise that exists for matches between sequences. In BLAST 2.0, the Expect value is also used instead of the P value (probability) to Teport the significance of matches. For example, an B value of 1 assigned to a hit can be interpreted as meaning that in a database of the current size one might expect to see 1 match with a similar score simply by chance.<br>
For example, the following sequences are related to SEQ ID NO:1, GenBank Accession Nos.: AA971857 (SEQ ID NO:3); W57569 (SEQ ED NO:4); AI288765 (SEQ ID NO: 5 ); W65390 (SEQ ID NO: 6 ); W37432 (SEQ ID NO: 7); N42748 (SEQ ID NO:8 ); AA971638 (SEQ ID NO:9 ); R22331 (SEQ ID NO: 10); AA308370 (SEQ ID NO: 11); AA285089 (SEQ ID NO: 12 ); R68901 (SEQ ID NO: 13 ); AA037285 (SEQ ID NO: 14 ); H94832 (SEQ ID NO: 15); H96058 (SEQ ID NO: 16); H56522 (SEQ ID NO: 17); AA935328 (SEQ ID NO: 18); AW327450 (SEQ ID NO: 19); AW406075 (SEQ ID NO:20); AW406223 (SEQ ID NO:21); AI909652 (SEQ ID NO:22); AA026773 (SEQ ID NO: 23); H96055(SEQ ID NO:24); H12836(SEQ ID NO:25); R22401 (SEQ ID NO:26); N34596 (SEQ ID NO:27); W32121 (SEQ ID NO:28); T84927 (SEQ<br><br>
-31-<br>
ID NO:29); R63575 (SEQ ID NO:30); R23139 (SEQ ID N0:31); AA337071 (SEQ ED NO:32); AA813244 (SEQ ID NO:33); AA313422 (SEQ ID NO:34); N31910 (SEQ ID NO:35);N42693 (SEQ ID NO:36); N32532 (SEQ ID NO: 37); AA375119 (SEQ ID NO:38); R32153 (SEQ ID NO:39); R23369 (SEQ ID NO:40); AA393628 (SEQ ID N0:41); H12779 (SEQ ID NO:42); AI083674 (SEQ ID NO:43); AA284919 (SEQ ID NO:44); AA375286 (SEQ ID NO:45); AA830592 (SEQ ID NO:46); H95363 (SEQ ID NO:47); T92052 (SEQ ID NO:48); AI336555 (SEQ ID NO:49); AI285284 (SEQ ID NO:50); AA568537 (SEQ ID NO:51); A1041967 (SEQ ID NO.S2); W44577 (SEQ ID NO:53); R22332(SEQ ID NO:54); N27088 (SEQ ID NO:55); H96418 (SEQ ID NO:56); AI025384 (SEQ ED NO: 57); AA707623 (SEQ ID NO: 58); A1051009 (SEQ ID NO:59); AA026774 (SEQ ID NO:60); W51792 (SEQ ID NO:61); AI362693 (SEQ ID NO:62); AA911823 (SEQ ID NO:63); H96422 (SEQ ID NO:64); AI800991 (SEQ ID NO:65); A1525314 (SEQ ID NO:66); AI934846 (SEQ ID NO:67); AI937133 (SEQ ID NO:68); AW006797 (SEQ ID NO:69); AI914716 (SEQ ID NO:70); AI672936 (SEQ ID NO:71); W6I294 (SEQ ID NO.72); AI199227 (SEQ ID NO:73); AI499727 (SEQ ID NO:74); R32154 (SEQ ID NO:75); AI439771 (SEQ ID NO:76); AA872671 (SEQ ID NO:77); AA502178 (SEQ ID NO:78); N26715 (SEQ ID NO:79); AA704668 (SEQ ID NO:80); R68799 (SEQ ID NO:81); H56704 (SEQ ID NO:82); AI360416 (SEQ ID NO: 83).<br>
Thus, in one embodiment the present invention is directed to polynucleotides comprising the polynucleotide fragments and full-length polynucleotide (e.g. the coding region) describedherein exclusive of one or more of the above-recited ESTs.<br>
Also preferred are C35 polypeptide and polynucleotide fragments characterized by stractural or functional domains. Preferred embodiments of the invention include fragments that comprise MHC binding epitopes and prenylation sites.<br><br>
-32-<br>
Other preferred fragments are biologically active C35 fragments. Biologically active fragments are those exhibiting activity similar, but not necessarily identical, to an activity of the C35 polypeptide. The biological activity of the fragments may include an improved desired activity, or a decreased undesirable activity.<br>
Epitopes &amp; Antibodies<br>
Cellular peptides derived by degradation of endogenously synthesized proteins are translocated into a pre-Golgi compartment where they bind to Class IMHC molecules for transport to the cell surface. These class I MHC peptide complexes are the target antigens for specific CD8+ cytotoxic T cells. Since all endogenous proteins "turn over," peptides derived from any cytoplasmic or nuclear protein may bind to an MHC molecule and be transported for presentation at the cell surface. This allows T cells to survey a much larger representation of cellular proteins than antibodies which are restricted to recognize conformational determinants of only those proteins that are either secreted or integrated at the cell membrane. The T cell receptor antigen binding site interacts with determinants of both the peptide and the surrounding MHC. T cell specificity must, therefore, he defined in terms of an MHC: peptide complex. The specificity of peptide binding to MHC molecules is very broad and of relatively low affinity in comparison to the antigen binding site of specific antibodies. Class I-bound peptides are generally 8-10 residues in length that accommodate amino acid side chains of restricted diversity at certain key positions that match pockets in the MHC peptide binding site. These key features of peptides that bind to a particular MHC molecule constitute a peptide binding motif.<br>
A number of computer algorithms have been described for identification of peptides in a larger protein that may satisfy the requirements of peptide binding motifs for specific MHC class I or MHC class II molecules. Because of<br><br>
-33-<br><br>
the extensive polymorphism of MHC molecules, different peptides will often bind to different MHC molecules. Tables 1-3 list C35 peptides predicted to be MHC binding peptides using three different algorithms. Specifically, Tables 1 and 5 list C35 HLA Class I and II epitopes predicted using the rules found at the SYFPEITHI website (wvsiwvg://35/http://134.2.96.221/scripts/hlaserver.dll/ EpPredict.htm) and are based on the book "MHC Ligands and Peptide Motifs" by Rammensee, H.G., Bachmann, J. and Stevanovic, S. (Chapman &amp; Hall, New York 1997). Table 2 lists predicted MHC binding peptides derived from the C35 sequence    using   the   NIH   BIMAS   program   available   on   the   web (http://bimas.dcrt.nih.gov/cgi-bin/molbio/ken parker comboform)     Finally, Tables 3 and 6 list predicted C35 peptides identified by the Tepitope program, a program for prediction of peptides that may bind to multiple different MHC class II molecules. Using Tepitope, four C35 peptides were identified as likely candidates for binding to a variety of HLA class II molecules. These peptides are, in general, longer than those binding to HLA class I and more degenerate in terms of binding to multiple HLA class II molecules.<br><br>
-34-<br>
37<br>
ATYLELASA<br>
19 97<br>
SNGETLEKI<br>
18 105<br>
ITNSRPPCV<br>
18 2<br>
SGEPGQTSV<br>
17 45<br>
AVKEQYPGI<br>
17 38<br>
TYLELASAV<br>
16 61<br>
GTGAFEIEI<br>
16 85<br>
YEKDLIEAI<br>
16 65<br>
FEIEINGQL<br>
15 107<br>
NSRPPCVIL<br>
15 41<br>
ELASAVKEQ<br>
14 58<br>
RLGGTGAFE<br>
14 59<br>
LGGTGAFEI<br>
14 66<br>
EIEINGQLV<br>
14<br><br>
-35-<br>
68<br>
EINGQLVFS<br>
14 81<br>
GGFPYEKDL<br>
14 94<br>
RRASNGETL<br>
14<br>
HLA-A*0201<br>
decanters<br>
Pos                                                  Score<br>
	1   234567890	<br>
58<br>
RLGGTGAFEI<br>
22 96<br>
ASNGETLEKI<br>
19 104<br>
KITNSRPPCV<br>
19 37<br>
ATYLELASAV<br>
18 17<br>
VEPGSGVRIV<br>
17 33<br>
CGFEATYLEL<br>
16 44<br>
SAVKEQYPGI<br>
16 92<br>
AIRRASNGET<br>
16<br>
39<br>
YLELASAVKE<br>
15<br><br>
-36-<br>
53<br>
IEIESRLGGT<br>
15 65<br>
FEIEINGQLV<br>
15 105<br>
ITNSRPPCVI<br>
15 1<br>
MSGEPGQTSV<br>
14 63<br>
GAFEIEINGQ<br>
14 68<br>
EINGQLVFSK<br>
14 69<br>
INGQLVFSKL<br>
14 83<br>
FPYEKDLIEA<br>
14 88<br>
DLIEAIRRAS<br>
14 93<br>
IRRASNGETL<br>
14 72<br>
QLVFSKLENG<br>
13 89<br>
LIEAIRRASN<br>
13 8<br>
TSVAPPPEEV<br>
12 16<br>
EVEPGSGVRI<br>
12<br><br>
-37-<br>
50<br>
YPGIEIESRL<br>
12 60<br>
GGTGAFEIEI<br>
12 81<br>
GGFPYEKDLI<br>
12 106<br>
TNSRPPCVIL<br>
12<br>
HLA-A*0203 nonamers<br>
Pos                                               Score<br>
	1   23456789	<br>
35<br>
FEATYLELA<br>
12<br>
HLA-A*0203 decamers<br>
Pos                                                  Score<br>
	1   234567890	<br>
36<br>
EATYLELASA<br>
18<br>
HLA-A1 nonamers<br>
Pos                                                Score<br>
	1   23456789	<br>
77<br>
KLENGGFPY<br>
29<br><br>
-38-<br>
2<br>
SGEPGQTSV<br>
18 21<br>
SGVRIVVEY<br>
18 16<br>
EVEPGSGVR<br>
17 29<br>
YCEPCGFEA<br>
17 42<br>
LASAVKEQY<br>
17 31<br>
EPCGFEATY<br>
16 34<br>
GFEATYLEL<br>
16 39<br>
YLELASAVK<br>
14 -84<br>
P YEKDLIEA<br>
14 66<br>
EIEINGQLV<br>
13 13<br>
PPEEVEPGS<br>
12 46<br>
VKEQYPGIE<br>
12 52<br>
GIEIESRLG<br>
12 96<br>
ASNGBTLEK<br>
12<br><br>
-39-<br>
HLA-A1 de earners<br>
Pos                                                  Score<br>
	1   234567890	<br>
20<br>
GSGVRIVVEY<br>
20 29<br>
YCEPCGFEAT<br>
19 76<br>
SKLENGGFPY<br>
18 2<br>
SGEPGQTSVA<br>
17 52<br>
GIEIESRLGG<br>
17 66<br>
EIEINGQLVF<br>
17 41<br>
ELASAVKEQY<br>
16 46<br>
VKEQYPGIEI<br>
16 16<br>
EVEPGSGVRI<br>
15 30<br>
CEPCGFEATY<br>
15 39<br>
YLELASAVKE<br>
15 77<br>
KLENGGFPYE<br>
14 86<br>
EKDLIEAIRR<br>
14<br><br>
-40-<br>
98<br>
NGETLEKITN<br>
14 34<br>
GFEATYLELA<br>
12 64<br>
AFEIEINGQL<br>
12 NO:1   101<br>
TLEKITNSRP<br>
12<br>
HLA-A26 nonamers<br>
Pos                                               Score<br>
	1   23456789	<br>
68<br>
EINGQLVFS<br>
24 100<br>
ETLEKITNS<br>
24 88<br>
DLIEAIRRA<br>
23 54<br>
EIESRLGGT<br>
22 41<br>
ELASAVKEQ<br>
21 45<br>
AVKEQYPGI<br>
20 31<br>
EPCGFEATY<br>
19 34<br>
GFEATYLEL<br>
19<br><br>
-41-<br>
73<br>
LVFSKLENG<br>
19 16<br>
EVEPGSGVR<br>
18 77<br>
KLENGGFPY<br>
18 66<br>
EIEINGQLV<br>
17 21<br>
SGVRIVVEY<br>
16 37<br>
ATYLELASA<br>
16 24<br>
RIVVBYCEP<br>
15<br>
9<br>
SVAPPPEEV<br>
14 22<br>
GVRIVVEYC<br>
14 51<br>
PGIEIESRL<br>
14 70<br>
NGQLVFSKL<br>
14 57<br>
SRLGGTGAF<br>
13 65<br>
FEIEINGQL<br>
13<br>
25<br>
IVVEYCEPC<br>
12<br><br>
-42-<br>
48<br>
EQYPGIEIE<br>
12 67<br>
IEINGQLVF<br>
12 75<br>
FSKLENGGF<br>
12 81<br>
GGFPYBKDL<br>
12 104<br>
KITNSRPPC<br>
12 105<br>
ITNSRPPCV<br>
12<br>
HLA-A26 decanters<br>
Pos	Score<br>
	1234567890<br>
41<br>
ELASAVKEQY<br>
27 66<br>
EIEINGQLVF<br>
26 68<br>
EINGQLVFSK<br>
23 26<br>
VVBYCBPCGF<br>
21 16<br>
EVEPGSGVRI<br>
20 88<br>
DLIEAIRRAS<br>
19<br><br>
-43-<br>
100<br>
ETLEKITNSR<br>
19 74<br>
VFSKLENGGF<br>
18 33<br>
CGFEATYLEL<br>
17 54<br>
EIESRLGGTG<br>
17 56<br>
ESRLGGTGAF<br>
17 20<br>
GSGVRIVVEY<br>
16 31<br>
EPCGFEATYL<br>
16 64<br>
AFEIEINGQL<br>
15 69<br>
INGQLVFSKL<br>
15 61<br>
GTGAFEIEIN<br>
14 73<br>
LVFSKLENGG<br>
14 9<br>
SVAPPPEEVE<br>
13 25<br>
IVVEYCEPCG<br>
13 45<br>
AVKEQYPGIE<br>
13<br><br>
-44-<br>
72<br>
QLVFSKLENG<br>
13 77<br>
KLENGGFPYE<br>
13 79<br>
ENGGFPYEKD<br>
13 4<br>
EPGQTSVAPP<br>
12 7<br>
QTSVAPPPEE<br>
12 30<br>
CEPCGFEATY<br>
12 36<br>
EATYLELASA<br>
12 37<br>
ATYLELASAV<br>
12 76<br>
SKLENGGFPY<br>
12 89<br>
LIEAIRRASN<br>
12<br>
HLA-A3 nonaniers<br>
Pos	Score<br>
	1   23456789	<br>
39<br>
YLELASAVK<br>
28 77<br>
KLENGGFPY<br>
25<br><br>
-45-<br>
16<br>
EVEPGSGVR<br>
24 58<br>
RLGGTGAFE<br>
22 67<br>
IEINGQLVF<br>
19 96<br>
ASNGETLEK<br>
18 92<br>
AIRRASNGE<br>
17 9<br>
SVAPPPEEV<br>
16 101<br>
TLEKITNSR<br>
16 22<br>
GVRIVVEYC<br>
15 31<br>
EPCGFEATY<br>
15 45<br>
AVKEQYPGI<br>
15 72<br>
QLVFSKLEN<br>
15 21<br>
SGVRIVVEY<br>
14 68<br>
EINGQLVFS<br>
14<br>
69<br>
INGQLVFSK<br>
14<br><br>
-46-<br>
88<br>
DLIEAIRRA<br>
14 91<br>
EAIRRASNG<br>
14 25<br>
IVVEYCEPC<br>
13 37<br>
ATYLELASA<br>
13 55<br>
IESRLGGTG<br>
13 51<br>
SRLGGTGAF<br>
13<br>
79<br>
ENGGFPYEK<br>
13 87<br>
KDLIEAIRR<br>
13 104<br>
KITNSRPPC<br>
13 24<br>
RIVVEYCEP<br>
12 42<br>
LASAVKEQY<br>
12 66<br>
EIEINGQLV<br>
12 89<br>
LIEAIRRAS<br>
12 90<br>
IEAIRRASN<br>
12<br><br>
-47-<br>
94<br>
RRASNGETL<br>
12<br>
HLA-A3 decamers<br>
Pos	Score<br>
	1234567890<br>
68<br>
EINGQLVFSK<br>
22 16<br>
EVEPGSGVRI<br>
20 38<br>
TYLELASAVK<br>
20 41<br>
ELASAVKEQY<br>
20 66<br>
EIEINGQLVF<br>
20 9<br>
SVAPPPEEVE<br>
19 58<br>
RLGGTGAFEI<br>
19 39<br>
YLBLASAVKE<br>
18 92<br>
AIRRASNGET<br>
18 95<br>
RASNGETLEK<br>
18 45<br>
AVKEQYPGIE<br>
17<br><br>
-48-<br>
54<br>
EIESRLGGTG<br>
16 88<br>
DLIEAIRRAS<br>
16 89<br>
LIEAIRRASN<br>
16 26<br>
VVEYCEPCGF<br>
15 37<br>
ATYLELASAV<br>
15 22<br>
GVRIVVEYCE<br>
14 77<br>
KLENGGFPYE<br>
14 93<br>
IRRASNGETL<br>
14 25<br>
IVVEYCEPCG<br>
13 30<br>
CEPCGFEATY<br>
13 52<br>
GIEIESRLGG<br>
13 76<br>
SKLENGGFPY<br>
13 78<br>
LENGGFPYEK<br>
13 101<br>
TLBKITNSRP<br>
13<br><br>
-49-<br>
104<br>
KITNSRPPCV<br>
13 24<br>
RIVVEYCEPC<br>
12 72<br>
QLVFSKLENG<br>
12<br>
HLA-B*0702 nonamers<br>
Pos                                               Score<br>
	1   23456789	<br>
18<br>
EPGSGVRIV<br>
19 107<br>
NSRPPCVIL<br>
18 4<br>
EPGQTSVAP<br>
15 11<br>
APPPEEVEP<br>
15 31<br>
EPCGFEATY<br>
14 34<br>
GFEATYLEL<br>
13 94<br>
RRASNGETL<br>
13 12<br>
PPPEEVEPG<br>
12 19<br>
PGSGVRIVV<br>
12<br><br>
-50-<br>
32<br>
PCGFEATYL<br>
12 83<br>
FPYEKDLIE<br>
12 106<br>
TNSRPPCVI<br>
12<br>
HLA-B*0702 decamers<br>
Pos                                                  Score<br>
	1   234567890	<br>
31<br>
EPCGFEATYL<br>
24<br>
50<br>
YPGIEIESRL<br>
21<br>
18<br>
EPGSGVRIVV<br>
20<br>
83<br>
FPYEKDLIEA<br>
16<br>
4<br>
EPGQTSVAPP<br>
15<br>
11<br>
APPPEEVEPG<br>
15<br>
93<br>
IRRASNGETL<br>
14<br>
106<br>
TNSRPPCVIL<br>
14<br>
69<br>
INGQLVFSKL<br>
13<br><br>
-51-<br>
33<br>
CGFEATYLEL<br>
12 64<br>
AFEIEINGQL<br>
12<br>
HLA-B*08 octamers<br>
Pos	Score<br>
1   2 3 4 5 6 7 8	<br>
83<br>
FPYEKDLI<br>
25 66<br>
EIEINGQL<br>
16 52<br>
GIEIESRL<br>
15 18<br>
EPGSGVRI<br>
14 54<br>
EIESRLGG<br>
14 91<br>
EAIRRASN<br>
14 95<br>
RASNGETL<br>
14 100<br>
ETLEKITN<br>
14 33<br>
CGFEATYL<br>
12 45<br>
AVKEQYPG<br>
12<br><br>
-52-<br>
58<br>
RLGGTGAF<br>
12 68<br>
EINGQLVF<br>
12 71<br>
GQLVFSKL<br>
12 75<br>
FSKLENGG<br>
12 82<br>
GFPYEKDL<br>
12 107<br>
NSRPPCVI<br>
12 108<br>
SRPPCVIL<br>
12<br>
HLA-B*08 nonamers<br>
Pos                                               Score<br>
	1   23456789	<br>
75<br>
FSKLENGGF<br>
19 83<br>
FPYEKDLIE<br>
19 45<br>
AVKEQYPGI<br>
18 85<br>
YEKDLIEAI<br>
18 107<br>
NSRPPCVIL<br>
17<br><br>
-53-<br>
100<br>
ETLEKITNS<br>
15 54<br>
EIESRLGGT<br>
14 65<br>
FEIEINGQL<br>
14 91<br>
EAIRRASNG<br>
14 20<br>
GSGVRIVVE<br>
12 34<br>
GFEATYLEL<br>
12 51<br>
PGIEIESRL<br>
12 81<br>
GGFPYEKDL<br>
12<br>
HLA-B*1510<br>
nonamers<br>
Pos	Score<br>
	123456789<br>
107<br>
NSRPPCVIL<br>
15 34<br>
GFEATYLEL<br>
13 51<br>
PGIEIESRL<br>
13-81<br>
GGFPYEKDL<br>
13<br><br>
-54-<br>
94<br>
RRASNGETL<br>
13<br>
HLA-B*2705<br>
nonamers<br>
Pos                                               Score<br>
	1   23456789	<br>
57<br>
SRLGGTGAF<br>
26 94<br>
RRASNGETL<br>
25 61<br>
IEINGQLVF<br>
19 87<br>
KDLIEAIRR<br>
19 51<br>
PGIEIESRL<br>
17 81<br>
GGFPYEKDL<br>
17 65<br>
FEIEINGQL<br>
16 69<br>
INGQLVFSK<br>
16 96<br>
ASNGETLEK<br>
16 16<br>
EVEPGSGVR<br>
15 34<br>
GFEATYLEL<br>
15<br><br>
-55-<br>
50<br>
YPGIEIESR<br>
15 70<br>
NGQLVFSKL<br>
15 101<br>
TLEKITNSR<br>
15 23<br>
VRIVVEYCE<br>
14 32<br>
PCGFEATYL<br>
14 39<br>
YLELASAVK<br>
14 79<br>
ENGGFPYEK<br>
14 93<br>
IRRASNGET<br>
14 21<br>
SGVRIVVEY<br>
13 27<br>
VEYCEPCGF<br>
13 75<br>
FSKLENGGF<br>
13 86<br>
EKDLIEAIR<br>
13 107<br>
NSRPPCVIL<br>
13 17<br>
VEPGSGVRI<br>
12<br><br>
-56-<br>
31<br>
EPCGFEATY<br>
12 77<br>
KLENGGFPY<br>
12<br>
HLA-B*2709<br>
nonamers<br>
Pos                                             Score<br>
	1   23456789	<br>
94<br>
RRASNGETL<br>
25 57<br>
SRLGGTGAF<br>
20 81<br>
GGFPYEKDL<br>
16 34<br>
GFEATYLEL<br>
14 51<br>
PGIEIESRL<br>
13 65<br>
FEIEINGQL<br>
13 23<br>
VRIVVEYCE<br>
12 107<br>
NSRPPCVIL<br>
12<br><br>
-57-<br>
HLA-B*5101 nonamers<br>
Pos                                            Score<br>
	1   23456789	<br>
18<br>
EPGSGVRIV<br>
21 81<br>
GGFPYEKDL<br>
21 51<br>
PGIEIESRL<br>
20 70<br>
NGQLVFSKL<br>
20 19<br>
PGSGVRIVV<br>
19 31<br>
EPCGFEATY<br>
19 2<br>
SGEPGQTSV<br>
18 42<br>
LASAVKEQY<br>
18 59<br>
LGGTGAFEI<br>
18 21<br>
SGVRIVVEY<br>
14 83<br>
FPYEKDLIE<br>
14 97<br>
SNGETLEKI<br>
14<br><br>
-58-<br>
13<br>
PPEEVEPGS<br>
13 38<br>
TYLELASAV<br>
13 45<br>
AVKEQYPGI<br>
13 63<br>
GAFEIEING<br>
13 94<br>
RRASNGETL<br>
13 12<br>
PPPEEVEPG<br>
12 33<br>
CGFEATYLE<br>
12 50<br>
YPGIEIESR<br>
12 66<br>
EIEINGQLV<br>
12 85<br>
YEKDLIEAI<br>
12 95<br>
RASNGETLE<br>
12 105<br>
ITNSRPPCV<br>
12<br><br>
-59-<br>
HLA-B*5101 octamers<br>
Pos                                               Score<br>
	1   2 3 4 5 6 7 8	<br>
83<br>
FPYEKDLI<br>
25 95<br>
RASNGETL<br>
23 10<br>
VAPPPEEV<br>
21 18<br>
EPGSGVRI<br>
21 33<br>
CGFEATYL<br>
21 98<br>
NGETLEKI<br>
19 19<br>
PGSGVRIV<br>
18 60<br>
GGTGAFEI<br>
18 62<br>
TGAFEIEI<br>
18 63<br>
GAFBIEIN<br>
14 71<br>
GQLVFSKL<br>
14 48<br>
EQYPGIEI<br>
13 67<br>
IEINGQLV<br>
13<br><br>
-60-<br>
106<br>
TNSRPPCV<br>
12<br>
Class II MHC<br>
HLA-DRB1*0101 15-mers<br>
Pos	Score<br>
	1   234567 8-9012345<br>
72<br>
QLVFSKLENGGFPYE<br>
29 37<br>
ATYLELASAVKEQYP<br>
26 26<br>
VVEYCEPCGFEATYL<br>
25 63<br>
GAFEIEINGQLVFSK<br>
25 24<br>
RIVVEYCEPCGFEAT<br>
24 36<br>
EATYLELASAVKEQY<br>
24 39<br>
YLELASAVKEQYPGI<br>
24 53<br>
IEIESRLGGTGAFEI<br>
24 56 ESRLGGTGAFEIEIN<br>
24 14 PEEVEPGSGVRIVVE<br>
23<br><br>
-61-<br>
43<br>
ASAVKEQYPGIEIES<br>
23<br>
20<br>
GSGVRIVVEYCEPCG<br>
20<br>
62<br>
TGAFBIEINGQLVFS<br>
20<br>
32<br>
PCGFEATYLELASAV<br>
19<br>
47<br>
KEQYPGIEIESRLGG<br>
19<br>
64 AFEIEINGQLVFSKL<br>
19<br>
82<br>
GFPYEKDLIEAIRRA<br>
19<br>
34<br>
GFEATYLELASAVKE<br>
18<br>
54 EIESRLGGTGAFEIE<br>
18<br>
90<br>
IEAIRRASNGETLEK<br>
18<br>
99<br>
GETLEKITNSRPPCV<br>
18<br>
31 EPCGFEATYLELASA<br>
17<br>
49<br>
QYFGIEIESRLGGTG<br>
17<br>
58<br>
RLGGTGAFEIEINGQ<br>
17<br><br>
-62-<br>
66<br>
EIEINGQLVFSKLEN<br>
17 61<br>
IEINGQLVFSKLENG<br>
17 68<br>
EINGQLVFSKLENGG<br>
17 84 PYEKDLIEAIRRASN<br>
17 86 EKDLIEAIRRASNGE<br>
17 35<br>
FEATYLELASAVKEQ<br>
16 74<br>
VFSKLENGGFPYEKD<br>
16 87<br>
KDLIEAIRRASNGET<br>
16 91<br>
EAIRRASNGETLEKI<br>
16 1 MSGEPGQTSVAPPPE<br>
15 4 EPGQTSVAPPPEEVE<br>
15 11 APPPEEVEPGSGVRI<br>
15 12 PPPEEVEPGSGVRIV<br>
15 29<br>
YCEPCGFEATYLELA<br>
15<br><br>
-63-<br>
5 PGQTSVAPPPEEVEP<br>
14 6 GQTSVAPPPEEVEPG<br>
14 44<br>
SAVKEQYPGIEIESR<br>
14 52<br>
GIEIESRLGGTGAFE<br>
14 61<br>
GTGAFEIEINGQLVF<br>
13 50 YPGIEIESRLGGTGA<br>
12<br>
HLA-DRB1*0301 (DR17)15-mers<br>
Pos	Score<br>
	1   23456789012345<br>
64<br>
AFEIEINGQLVFSKL<br>
26 39<br>
YLELASAVKEQYPGI<br>
25 72<br>
QLVFSKLENGGFPYE<br>
23 62 TGAFEIEINGQLVFS<br>
22 24<br>
RIVVEYCEPCGFEAT<br>
19 71 GQLVFSKLENGGFPY<br>
19<br><br>
-64-<br>
86<br>
EKDLIEAIRRASNGE<br>
19 7 QTSVAPPPEEVEPGS<br>
18 23<br>
VRIVVEYCEPCGFEA<br>
18 50 YPGIEIESRLGGTGA<br>
18 90<br>
IEAIRRASNGETLEK<br>
18 20<br>
GSGVRIVVEYCEPCG<br>
17 87<br>
KDLIEAIRRASNGET<br>
17 99<br>
GETLEKITNSRPPCV<br>
16 28<br>
EYCEPCGFEATYLEL<br>
15 37 ATYLELASAVKEQYP<br>
14 48<br>
EQYPGIEIESRLGGT<br>
14 78<br>
LENGGFPYEKDLIEA<br>
14 14 PEEVEPGSGVRIVVE<br>
13 70 NGQLVFSKLENGGFP<br>
13<br><br>
-65-<br>
43<br>
ASAVKEQYPGIEIES<br>
12 52<br>
GIEIESRLGGTGAFE<br>
12 54 EIESRLGGTGAFEIE<br>
12 74<br>
VFSKLENGGFPYEKD<br>
12 82<br>
GFPYEKDLIEAIRRA<br>
12<br>
HLA-DRB1*0401 (DR4Dw4) 15 - mers<br>
Pos	Score<br>
	1   23456789012345<br>
36<br>
EATYLELASAVKEQY<br>
28 62<br>
TGAFEIEINGQLVFS<br>
28 86 EKDLIEAIRRASNGE<br>
26 87<br>
KDLIEAIRRASNGET<br>
26 90<br>
IEAIRRASNGETLEK<br>
26 72<br>
QLVFSKLENGGFPYE<br>
22 82<br>
GFPYEKDLIEAIRRA<br>
22<br><br>
-66-<br>
50 YPGIEIESRLGGTGA<br>
20 99<br>
GETLEKITNSRPPCV<br>
20 26<br>
VVEYCEPCGFEATYL<br>
16 32<br>
PCGFEATYLELASAV<br>
16 47<br>
KEQYPGIEIESRLGG<br>
16 80 NGGFPYEKDLIEAIR<br>
16 14 PEEVEPGSGVRIVVE<br>
14 20<br>
GSGVRIVVEYCEPCG<br>
14 22<br>
GVRIVVEYCEPCGFE<br>
14 37 ATYLELASAVKEQYP<br>
14 39<br>
YLELASAVKEQYPGI<br>
14 56 ESRLGGTGAFEIEIN<br>
14 64<br>
AFEIEINGQLVFSKL<br>
14 66<br>
EIEINGQLVFSKLEN<br>
14<br><br>
-67-<br>
10<br>
VAPPPEEVEPGSGVR<br>
12 12<br>
PPPEEVEPGSGVRIV<br>
12 16<br>
EVEPGSGVRIVVEYC<br>
12<br>
29<br>
YCEPCGFEATYLELA<br>
12 30<br>
CEPCGFEATYLELAS<br>
12 31<br>
EPCGFEATYLELASA<br>
12 34<br>
GFEATYLELASAVKE<br>
12 35<br>
FEATYLELASAVKEQ<br>
12 42<br>
LASAVKEQYPGIEIE<br>
12 48<br>
EQYPGIEIESRLGGT<br>
12 49<br>
QYPGIEIESRLGGTG<br>
12 53 IEIESRLGGTGAFEI<br>
12 58 RLGGTGAFEIEINGQ<br>
12<br>
59 LGGTGAFEIEINGQL<br>
12<br><br>
-68-<br>
61<br>
GTGAFEIEINGQLVF<br>
12 63<br>
GAFEIEINGQLVFSK<br>
12 67<br>
IEINGQLVFSKLENG<br>
12 68<br>
EINGQLVFSKLENGG<br>
12 69<br>
INGQLVFSKLENGGF<br>
12 85<br>
YEKDLIEAIRRASNG<br>
12 93<br>
IRRASNGETLEKITN<br>
12 94<br>
RRASNGETLEKITNS<br>
12 96<br>
ASNGETLEKITNSRP<br>
12 91<br>
SNGETLEKITNSRPP<br>
12<br><br>
-69-<br><br><br>
-70-<br><br><br>
-71-<br><br><br>
-72-<br><br><br>
-73-<br><br><br>
-74-<br><br><br>
-75-<br><br><br>
-76-<br><br><br>
-77-<br><br><br>
-78-<br><br><br>
-79-<br><br><br>
-80-<br><br><br>
-81-<br><br><br>
-82-<br><br><br>
-83-<br><br><br>
84-<br><br><br>
-85-<br><br><br>
-86-<br><br><br>
-87-<br><br><br>
-88-<br><br><br>
-89-<br><br><br>
-90-<br><br><br>
-91-<br><br><br>
-92-<br><br><br>
-93-<br><br><br>
-94-<br><br><br>
-95-<br><br><br>
-96-<br><br><br>
-97-<br><br><br>
-98-<br><br><br>
-99-<br><br><br>
-100-<br><br><br>
-101-<br><br><br>
-102-<br><br><br>
-103-<br><br><br>
-104-<br><br><br>
-105-<br><br><br>
-106-<br><br><br>
-107-<br><br><br>
-108-<br><br><br>
-109-<br><br><br>
-110-<br><br><br>
-111-<br><br><br>
-112-<br><br><br>
-113-<br><br><br>
-114-<br><br><br>
-115-<br><br><br>
-116-<br><br><br>
-117-<br><br><br>
-118-<br><br><br>
-219-<br><br><br>
-120-<br><br><br>
-121-<br><br><br>
-122-<br><br><br>
-123-<br><br><br>
-124-<br><br><br>
-125-<br><br><br>
-126-<br><br><br>
-127-<br><br><br>
-128-<br><br><br>
-129-<br><br><br>
-130-<br><br><br>
-131-<br><br><br>
-132-<br><br><br>
-133-<br><br><br>
-134-<br><br><br>
-135-<br><br><br>
-136-<br><br><br>
-137-<br><br><br>
-138-<br><br><br>
-139-<br>
Altered Peptide Ligands<br>
Identification of immunodominant epitopes of C35 for MHC class I antigens using specific human T cell lines is a key step toward their successful use in cancer vaccines. Modified C35 peptide epitomes containing amino acid substitutions at MHC binding residues have the potential to be used for enhancement of immune function. Such altered peptide ligand, or heteroclitic peptides,  can become  strong T  cell  agonists  even  at   100-fold lower concentrations that the original peptide (Dressel, A.  et al., "Autoantigen recognition by human CDS T Cell clones: enhanced agonist response induced by altered peptide ligand," J. Immunol 159:4943-51(1997). These altered peptide ligand can be of two forms: those modifications that enhance T cell receptor contact with the peptide (must be determined experimentally) and those that enhance HLA binding of the peptide by improving the anchor residues. Table 4 specifies modifications that enhance HLA Class I binding by introducing favorable anchor residues or replacing deleterious residues.<br>
TABLE 4<br>
Modifications that Enhance HLA Class I Binding<br>
(Unless otherwise indicated, examples apply to peptides of 9 ammo acids; for 10-mers the amino acid at position 5 is disregarded and the resultant 9-mer is evaluated (http://bhnas.dcrt.nih.gov/cgi-bin/molbio/hla_coefficient viewing page. The modifications listed below are provided by way of example based on current data in existing databases and are not intended in any way to be an inclusive list of all potential alterations of peptides binding all potential HLA molecules, both known and unknown to date.)<br>
HLAA*0101<br>
Any altered peptide that has S or T at position 2 Any altered peptide that has D or E at position 3 Any altered peptide that has P at position 4<br><br>
-140-<br>
Any altered peptide that has A, F, I, L, M, P, V, or Y at position 7<br>
Any altered peptide that has F, K, R, or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D,E, F,G, H, K, M, N, P, Q, R, W, Y<br>
P3: E,K,R,W<br>
P4: K,R<br>
P7: D, E, G, R<br>
P9: D,E,P<br>
HLAA*0201<br>
Any altered peptide that has F, I, K, L, M, V, W, or Y at position 1<br>
Any altered peptide that has I, L, M, Q, or V at anchor position 2<br>
Any altered peptide that has F, L, M, W, or Y at position 3<br>
Any altered peptide that has D or E at position 4<br>
Any altered peptide that has F at position 5<br>
Any altered peptide that has F, I, L, M, V, W or Y at auxiliary anchor position 6<br>
Any altered peptide that has F, or W at position 7<br>
Any altered peptide that has F, W, or Y at position 8<br>
Any altered peptide that has I, L, T or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D,E,H,P<br>
P2: C, F, H, K, N, P, R, S, W,Y<br>
P3: D, E, K, R<br>
P7: D, E, G, R<br>
P8: I,V<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-A*0205<br>
Any altered peptide that has F, I, K, L, M, V, W, ox Y at position 1 Any altered peptide that has E, I, L, M, Q, or V at anchor position 2 Any altered peptide that has F, L, M, W, or Y at position 3 Any altered peptide that has D or E at position 4<br><br>
-141-<br>
Any altered peptide that has F, Y at position 5<br>
Any altered peptide that has F, I, L, M, V, W or Y at auxiliary anchor position 6<br>
Any altered peptide that has F, or W at position 7<br>
Any altered peptide that has F, W, or Y at position 8<br>
Any altered peptide that has I, L, T or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1:	D, E, P<br>
P2:	Q, D, F, G, H, K, N, P, R, S, W, Y<br>
P3:	D, E, K, R<br>
P7:	D, B, R<br>
P9:	D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-A*03<br>
Any altered peptide that has G or K at position 1<br>
Any altered peptide that has I, L, M, Q, T or V at anchor position 2<br>
Any altered peptide that has F, I, L, M, V, W, or Y at position 3<br>
Any altered peptide that has E, G or P at position 4<br>
Any altered peptide that has F, I, P, V, W, Y at position 5<br>
Any altered peptide that has F, I, L, M, or V at position 6<br>
Any altered peptide that has F, I, L, M, W, or Y at position 7<br>
Any altered peptide that has F, I, K, L, Q or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D,E, P<br>
P2: D, E, F, G, H, K, N, B, S, W, Y<br>
P7: G,K,R<br>
P9: D, E, G, H, N, P, Q, S, T<br>
HLA-A*1101<br>
Any altered peptide that has G, K or R at position 1<br>
Any altered peptide that has I, L, M, Q, T, V, Y at anchor position 2<br>
Any altered peptide that has F, I, L, M, V, W, Y at position 3<br><br>
-142-<br>
Any altered peptide that has F, I, L, M, W or Y at position 7<br>
Any altered peptide that has K or R at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D,E,P<br>
P2: D, E, G, H, K, N, R, S, W<br>
P7: K, R<br>
P9: C, D, E, G, N, P, Q, S, T<br>
HLA-A24<br>
Any altered peptide that has K or R at position 1<br>
Any altered peptide that has F or Y at anchor position 2<br>
Any altered peptide that has E, I, L, M, N, P, Q, or Vat position 3<br>
Any altered peptide that has D, E, or P at position 4<br>
Any altered peptide that has I, L, or V at position 5<br>
Any altered peptide that has F at position 6<br>
Any altered peptide that has N or Q at position 7<br>
Any altered peptide that has E or K at position 8<br>
Any altered peptide that has F, I, L, or M at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, H, K, R<br>
P9: D, E, H, P, Q, R<br>
HLA-A*3101<br>
Any altered peptide that has K or R at position 1<br>
Any altered peptide that has F, I, L, M, Q, T, V; or Y at anchor position 2<br>
Any altered peptide that has F, I, L, M, V W, or Y at position 3<br>
Any altered peptide that has F, I, L, M, or V at position 6<br>
Any altered peptide that has F, I, L, M, W, or Y at position 7<br>
Any altered peptide that has K or R at anchor position 9<br><br>
-143-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D,E,P<br>
P2: D, E, G, H, K, N, R, S<br>
P7: K, R<br>
P9: C, G, N, P, Q, S, T<br>
HLA-A*3302<br>
Any altered peptide that has D or E at position 1<br>
Any altered peptide that has I, L, M, S, V or Y at anchor position 2<br>
Any altered peptide that has R at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: K, P, R<br>
P2: D, E, K, R<br>
P9: D, E, F, G, N, P, W, Y<br>
HLA-B7<br>
Any altered peptide that has A at position. 1<br>
Any altered peptide that has A, P or V at anchor position 2<br>
Any altered peptide that has M or R at position 3<br>
Any altered peptide that has P at position 5<br>
Any altered peptide that has R at position 6<br>
Any altered peptide that has I, L, M or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, F, H, K, R, W, Y<br>
P3; D, E<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B8<br>
Any altered peptide that has D or E at position 1<br>
Any altered peptide that has A, C, L, or P at anchor position 2<br>
Any altered peptide that has K or R at position 3<br><br>
-144-<br>
Any altered peptide that has D or E at position. 4<br>
Any altered peptide that has K or R at position 5<br>
Any altered peptide that has I, L, M, or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: K, P, R<br>
P2: D, E, F, G, H, K, Q, R, W, or Y<br>
P3: D, E<br>
P5: D, E<br>
P9: D, B, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B8 (8-mer peptides)<br>
Any altered peptide that has D or E at position 1<br>
Any altered peptide that has A, C, L, or P at anchor position 2<br>
Any altered peptide that has K or R at position 3<br>
Any altered peptide that has D or E at position 4<br>
Any altered peptide that has K or R at position 5<br>
Any altered peptide that has I, L, M, or V at anchor position 8<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: K, P, R<br>
P2: D, E, F, G, H, K, Q, R, W, or Y<br>
P3: D, E<br>
P5: D, E<br>
P8: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HA-B14<br>
Any altered peptide that has D or E at position 1<br>
Any altered peptide that has K or R at anchor position 1<br>
Any altered peptide that has F, I, L, M, P, V, W, Y at position 3<br>
Any altered peptide that has H or R at position 5<br>
Any altered peptide that has I, L, M, R, or V at position 6<br>
Any alered peptide the has T at position 7<br>
Any altered peptide that has I, L, M, or V at anchor position 9<br><br>
-145-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, F, W or Y<br>
P3: E, R<br>
P5: E, W, Y<br>
P9: D, E, G, H, K, N, P, Q, R<br>
HLA-B*2702<br>
Any altered peptide that has K or R at position 1<br>
Any altered peptide that has E, L, M, N, Q or R at anchor position 2<br>
Any altered peptide that has F, W, or Y at position 3<br>
Any altered peptide that has V, I, L, W or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1:	D, E, P<br>
P2:	D, F, G, H, K, W, or Y<br>
P7:	K<br>
P9:	D, E, G, K, N, P, Q, R, S<br>
HLA-B27*05 (8-mer peptides)<br>
Any altered peptide that has K or R at position 1<br>
Any altered peptide that has E, L, M, N, Q or R at anchor position 2<br>
Any altered peptide that has F, W, or Y at position 3<br>
Any altered peptide that has F, I, K, L, M, R, V or Y at anchor position 8<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D, E, P<br>
P2: D, F, G, K, or Y<br>
P7: K<br>
P9: D, B, G, K, N, P, Q, R, S<br>
HLA-B*3501 (8-mer peptides)<br>
Any altered peptide that has K or R at position 1<br>
Any altered peptide that has A, P, or S at anchor position 2<br>
Any altered peptide that has K or R at position 3<br><br>
-146-<br>
Any altered peptide that has D or E at position 4<br>
Any altered peptide that has D or E at position 5<br>
Any altered peptide that has F, I, L, M, V, W or Y at anchor position 8<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, F, H, K, R, W, Y<br>
P3: D, E<br>
P8: D, E, F, G, H, K, P, Q, R<br>
HLA-B*3701<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has I or V at position 5<br>
Any altered peptide that has F, L, or M at position 8<br>
Any altered peptide that has F, I, L, M, V or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P9: D, E, G, H, K, P, Q, R<br>
HLA-B*3801<br>
Any altered peptide that has F, H, P, W or Y at anchor position 2<br>
Any altered peptide that has D or E at position 3<br>
Any altered peptide that has D, E, or G at position 4<br>
Any altered peptide that has A, I, L, M, or V at position 5<br>
Any altered peptide that has K or Y at position 8<br>
Any altered peptide that has F, I, L, M, or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, K, R<br>
P3: K, R<br>
P9: D, E, G, H, K, P, Q, R<br><br>
-147-<br>
HLA-B*3901 (8-mer peptides)<br>
Any altered peptide that has H or R at anchor position 2<br>
Any altered peptide that has D, E, F, I, L, M, V, W, or W at position 3<br>
Any altered peptide that has D or E at position 4<br>
Any altered peptide that has I, L, M, or V at position 6<br>
Any altered peptide that has I, L, M or V at anchor position 8<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1:	P<br>
P2:	D, E<br>
P3:	K. R<br>
P6:	D, E, K, R<br>
P8:	D, B, G, H, K, P, Q, R<br>
HLA-B*3902<br>
Any altered peptide that has K or Q at anchor position 2<br>
Any altered peptide that has F, I, L, M, V, W, or Y at position 5<br>
Any altered peptide that has F, L, or M at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E<br>
P3: K, R<br>
P9: D, E, G, B, K, P, Q, R<br>
HLA-B40<br>
Any altered peptide that has A or G at position. 1<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has A, F, I, L, M, V, W, or Y at position 3<br>
Any altered peptide that has P at position 4<br>
Any altered peptide that has P at position 5<br>
Any altered peptide that has A, L, M, or W at anchor position 9<br><br>
-148-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1:	P<br>
P2:	F, H, I, K, L, M, Q, R, V, W, or Y<br>
P3:	D, E, K, R<br>
P9:	D, E, G, H, K, N, P, Q, R<br>
HLA-B44*03<br>
Any altered peptide that has A, D, or S at position 1<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has A, I, L, M, or V at position 3<br>
Any altered peptide that has F, I, or P at position 4<br>
Any altered peptide that has A, K, or V at position 5<br>
Any altered peptide that has A, L, T, or V at position 6<br>
Any altered peptide that has F, K, or T at position 7<br>
Any altered peptide that has K at position 8<br>
Any altered peptide that has F, W or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: F, H, I, K, L, M, Q, R, V, W, Y<br>
P9: D, E, G, H, K, N, P, Q, R<br>
HLA-B*5101 (8-mer peptides)<br>
Any altered peptide that has D, E, F, I, L, M, V, or Y at position 1<br>
Any altered peptide that has A, G or P at anchor position 2<br>
Any altered peptide that has F, W or Y at position 3<br>
Any altered peptide that has D, E, G, I, K, or V at position 4<br>
Any altered peptide that has A, G, I, S, T, or V at position 5<br>
Any altered peptide that has I, K, L, N, or Q at position 6<br>
Any altered peptide that has D, K, Q, or R at position 7<br>
Any altered peptide that has I. L, M. or V at anchor position 8<br><br>
-149-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: K, P, R<br>
P2: D, E, H, K<br>
P8: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*5102<br>
Any altered peptide that has F or Y at position 1<br>
Any altered peptide that has A, G, or P at anchor position. 2<br>
Any altered peptide that has F, I, L, V, W, or Y at position 3<br>
Any altered peptide that has E, G, H, K, L, N, Q, R, or T at position 4<br>
Any altered peptide that has G, N, Q, T, or V at position 5<br>
Any altered peptide that has I, N, Q, or T at position 6<br>
Any altered peptide that has E, K, Q, or R at position 7<br>
Any altered peptide that has K, R, T, or Y at position 8<br>
Any altered peptide that has I, L, M, or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, H, K, R<br>
P3: D, E, K, R<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*5102 (8-mer peptides)<br>
Any altered peptide that has F or Y at position 1<br>
Any altered peptide that has A, G, or P at anchor position 2<br>
Any altered peptide that has F, I, L, V, W, or Y at position 3<br>
Any altered peptide that has E, G, H, K, L, V, W, or Y at position 4<br>
Any altered peptide that has G, N, Q, T, V at position 5<br>
Any altered peptide that has I, N, or Q at position 6<br>
Any altered peptide that has Q, or R at position 7<br>
Any altered peptide that has I, L, M, or V at position 8<br><br>
-150-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, H, K, R<br>
P3: D, E, K, R<br>
P8: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*5103<br>
Any altered peptide that has D, T, or V at position 1<br>
Any altered peptide that has A, G, or P at anchor position 2<br>
Any altered peptide that has D, F, L, or Y at position 3<br>
Any altered peptide that has E, G, L, N, Q, R, T, or V at position 4<br>
Any altered peptide that has A, G, M, N, Q, R, K or V at position 5<br>
Any altered peptide that has I, K, or T at position 6<br>
Any altered peptide that has M or V at position 7<br>
Any altered peptide that has I, L, M, or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, H, K, R<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*5201 (8-mer peptides)<br>
Any altered peptide that has I, L, M, or V at position 1 Any altered peptide that has G, P, or Q at anchor position 2 Any altered peptide that has D, F, I, L, P, W, or Y at position 3 Any altered peptide that has A, E, I, K, L, P, or V at position 4 Any altered peptide that has A, F, G, I, L, M, T or V at position 5 Any altered peptide that has K, L, N, S or T at position 6 Any altered peptide that has E, K, Q, or Y at position 7 Any altered peptide that has F, I, L, M, or V at anchor position 8<br><br>
-151-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: H, K, R<br>
P3: R<br>
P8: D, E, G, H, K, N, P, Q, R, S<br>
HLA-B*5801<br>
Any altered peptide that has I, K, or R at position 1<br>
Any altered peptide that has A, S, or T at anchor position 2<br>
Any altered peptide that has D at position 3<br>
Any altered peptide that has E, K, or P at position 4<br>
Any altered peptide that has F, I, L, M, or V at position 5<br>
Any altered peptide that has F, I, L, or V at position 6<br>
Any altered peptide that has L, M, N, or Y at position 7<br>
Any altered peptide that has K, N, R, or T at position 8<br>
Any altered peptide that has F, W, or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: D,E,P<br>
P2: D, E, F, H, I, K, L, M, N, Q, R, V, W, Y<br>
P9: D, S, G, H, K, N, P, Q, R, S<br>
HLA-B*60<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has A, 1, L, M, S, or V at position 3<br>
Any altered peptide that has L, I, or V at position 5<br>
Any altered peptide that has I, L, M, V, or Y at position 7<br>
Any altered peptide that has K, Q, or R at position 8<br>
Any altered peptide that has I, L, M, or V at anchor position 9<br><br>
-152-<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: F, H, I, K, L, M, Q, R, V, W, Y<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*61<br>
Any altered peptide that has G or R at position 1<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has A, F, I, L, M, T, V, W, or Y at position 3<br>
Any altered peptide that has I at position 6<br>
Any altered peptide that has Y at position 7<br>
Any altered peptide that has A, I, L, M, or V at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: F, H, I, K, L, M, Q, R, V, W, Y<br>
P9: D, E, F, G, H, K, N, P, Q, R, S, W, Y<br>
HLA-B*61 (S-mer peptides)<br>
Any altered peptide that has G or R at position 1<br>
Any altered peptide that has D or E at anchor position 2<br>
Any altered peptide that has A, F, I, L, M, T, V, W, or Y at position 3<br>
Any altered peptide that has I at position 6<br>
Any altered peptide that has Y at position 7<br>
Any altered peptide that has A, I, L, M, or V at anchor position 8<br>
Any altered peptide where deleterious residues at file following positions are replaced:<br>
P1: P<br>
P2: F, H, I, K, L, M, Q, R, V, W, Y<br>
P8: D, F, G, H, K, N, P, Q, R, S, W, Y<br><br>
-153-<br>
HLA-B*62<br>
Any altered peptide that has I at position 1<br>
Any altered peptide that has I, L, Q at anchor position 2<br>
Any altered peptide that has G, K, R at position 3<br>
Any altered peptide that has D, E, G, or P at position 4<br>
Any altered peptide that has F, G, I, L, or V at position 5<br>
Any altered peptide that has I, L, T, V at position 6<br>
Any altered peptide that has T, V, or Y at position 7<br>
Any altered peptide that has F, W, Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, F, H, K, N, R, S, W, Y<br>
P3: D, E<br>
P6: D, E, K, R<br>
P9: D, E, G, H, K, N, P, Q, R, S<br>
HLA-Cw0301<br>
Any altered peptide that has A or R at anchor position 2<br>
Any altered peptide that has F, I, L, M, V, or Y at position 3<br>
Any altered peptide that has E, P, or R at position 4<br>
Any altered peptide that has N at position 5<br>
Any altered peptide that has F, M, or Y at position 6<br>
Any altered peptide that has K, M, R, or S at position 7<br>
Any altered peptide that has T at position 8<br>
Any altered peptide that has F, I, L, M at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P3: D, K, R<br>
P6: D, E, K, R<br>
P9: D, E, G, H, K, N, P, Q, R, S,<br><br>
-154-<br>
HLA-Cw0401<br>
Any altered peptide that has F, P, W, or Y at anchor position 2<br>
Any altered peptide that has D, or H at position 3<br>
Any altered peptide that has D or E at position 4<br>
Any altered peptide that has A, H, M, R, or T at position 5<br>
Any altered peptide that has I, L, M, or V at position 6<br>
Any altered peptide that has A at position 7<br>
Any altered peptide that has H, K, or S at position 8<br>
Any altered peptide that has F, I, L, M, V or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P2: D, E, H, K, R<br>
P9: D, E, G, H, K, N, P, Q, R, S<br>
HLA-Cw0602<br>
Any altered peptide that has F, I, K, or Y at position 1<br>
Any altered peptide that has A, P, Q, or R at anchor position 2<br>
Any altered peptide that has F, I, K, L, or M at position 5<br>
Any altered peptide that has I, L, or V at position 6<br>
Any altered peptide that has K, N, Q, or R at position 7<br>
Any altered peptide that has I, L, M, V, or Y at anchor position 9<br>
Any altered peptide where deleterious residues at the following positions are replaced:<br>
P1: P<br>
P9: D, E, G, H, K, N, P, Q, R, S<br><br>
-155-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
Examples of predicted human Class I MHC binding peptides from the C35 aa sequence and how they might be changed to improve binding:<br>
HLA-A*0101<br>
Rank	Start position      Subsequence       Score (estimated half time of dissociation)<br>
1	77	KLENGGRPY	225.000<br>
2	16	EVEPGSGVR	90.000<br>
3	29	YCEPCGFEA	45.000<br>
4	39	YLELASAVK	36.000<br>
5	2	SGEPGQTSV	2.250 G is deleterious at P2<br>
example of<br>
improved peptide	STEPGQTSV	22.S0 G replaced with T @ P2<br>
example of<br>
improved peptide	STEPGQISY	5625.00<br>
V at P9 replaced with Y, P7 enhanced<br>
HLA-A*0101 (10-mer peptides)<br>
1	66	EIENGQLVF	45.000<br>
2	16	EVEPGSGVRI	18.000<br>
3	29	YCEPCGFEAT	9.000<br>
4	26	WEYCEPCGF	9.000<br>
5	52	GIEIESRLGG	2.250<br>
example of<br>
improved peptide	GTEPSRLGY	1125.000 replace I with T @P2<br>
replace G with Y@P9 P5 enhanced with P<br>
HLA-A*0201 (9-mer peptides)<br>
1	9	SVAPPPEEV	2.982<br>
2	104	KITNSRPPC	2.391<br>
3	105	ITNSRPPCV	1.642<br>
4	25	IVVEYCEPC	1.485<br><br>
-156-<br>
Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
5	65	FEIEINGQL	1.018<br>
example of<br>
improved peptide	FUEINWYL	16619.000<br>
HLA-A*0201 (10-mer peptides)<br>
1	58	RLGGTGAFEI	60.510<br>
2	104	KITNSRPPCV	33.472<br>
3	65	FEIEINGQLV	25.506<br>
4	83	FPYWDUEA	4.502     P is deleterious at P2<br>
example of<br>
improved peptide	FLYEKDLIEA	689.606    replace P with L @ P2<br>
example of<br>
improved peptide	FLYEKDLIEV	9654.485 replace A with V @ P9<br>
5	33	CGFEATYLEL	3.173<br>
HLA-A*0205<br>
1	65	FEIEINGQL	8.820<br>
2	NO:1	25	IWEYCEPC	3.060<br>
3	9	SVAPPPEEV	2.000<br>
4	.104	KITNSRPPC	1.500<br>
5	81	GGFPYEKDL	1.260 G is deleterious at P2<br>
example of<br>
improved peptide	GVFPYEKDL	50.400 replace G with V @ P2<br>
HLA-A*0205 (10-mer peptides)<br>
1	33	CGEFATYLEL	6.300 G is deleterious at P2<br>
example of<br>
improved peptide	CVEFATYLEL	11.200 replace G with V @ P2<br>
2	104	KITNSRPPCV	6.000<br>
3	65	FEIEINGQLV	2.520<br><br>
-157-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence      Score (estimated half time of dissociation)<br>
4	53	IEIESRLGGT                   1.428<br>
5	83      	FPYEKDLIEA                 1.350 P is deleterious at P2<br>
example of<br>
improved peptide	FVYEKDLIEA	54.000 replace P with V @ P2<br>
HLA-A24<br>
1	34	GFEATYLEL	33.000<br>
2	49	QYPGIEIES	11.550<br>
example of<br>
improved peptide	QYPGJEIEL	462.000 enhance P9<br>
3	70	NGQLZFSKL	11.088<br>
4	38	TYLELASAV	10.800<br>
5	82	GFPYEKDU	7.500<br>
HLA-A24 (10-mer peptides)<br>
1	64	AFEIEINGQL	42.000<br>
2	74	VFSKLENGGF	10.000<br>
3	84	PYEKDLIEAI	9.000<br>
4	69	INGQLVFSKL	7.392<br>
example of	<br>
improved peptide	IYGQLVFSKL	369.6 enhance P2<br>
5	28	EYCEPCGFEA	6.600<br>
HLA-A3<br>
1	77	KLENGGFPY	36.000<br>
example of<br>
improved peptide	KLENGGFPK	180.000   enhance P9<br>
2	39	YLELASAVK	20.000<br>
3	101	TLEKITNSR	6.000<br>
4	61	GTGAFEIEI	0.540<br><br>
-158-Examples of predicted human Class IMHC binding peptides - continued<br>
Rank     Start position	Subsequence      Score (estimated half time of dissociation)<br>
5	69	INGQLVFSK	0.360    N is deleterious @P2<br>
example of<br>
improved peptide	DLGQLVFSK	180.000 replace N with L @ P2<br>
HLA-A3 (10-mer peptides)<br>
1	68	EINGQLWSK	8.100<br>
2	58	RLGGTGAFEI	2.700<br>
3	41	ELASAVKEQY	1.800<br>
4	78	LENGGFPYEK	0.810     E is deleterious @ P2<br>
example of<br>
improved peptide	LLNGGFPYEK	270.000 replace E with L @ P2<br>
5	95	RASNGETLEK	0.400<br>
HLA- A*1101<br>
1	39	YLELASAVK	0.400<br>
2	69	INGQLVFSK	0.120 N is deleterious @ P2<br>
example of<br>
improved peptide	IVGQLVFSK	6.000 replace N with V @ P2<br>
3	16	EVEPGSGVR	0.120<br>
4	101	TLEKITNSR	0.080<br>
5	61	GTGAFEEEI	0.060<br>
HLA-A*1101 (10-mer peptides)<br>
1	95	RASNGETLEK	1.200<br>
2	38	TYLELASAVK	0.600<br>
3	68	EINGGLVFSK	0.360<br>
4	78	LENGGFPYEK	0.1-20 E is deleterious @ P2<br>
example of<br>
improved peptide	LVNGGFPYEK	4.000 replace E with V @ P2<br>
5	100	ETLEKTTNSR	0.090<br><br>
-159-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subseqnence       Score (estimated half time of dissociation)<br>
HLA-A*3101<br>
1	101	TLEKITNSR	2.000<br>
2	16	EVEPGSGVR	0.600<br>
3	50	YPGIEIESR	0.400<br>
4	87	KDLDEAIRR	0.240 D is deleterious @ P2<br>
example of<br>
improved peptide	KILIEAIRR	12.000 replace D with I @ P2<br>
5	39	YLELASAVK	0.200<br>
HLA-A*3302<br>
1	16	EVEPGSGVR	45.000<br>
2	101	TLEKTTNSR	9.000<br>
3	50	YPGIEIESR	3.000<br>
4	66	EIEINGQLV	1.500<br>
5	56	ESRLGGTGA	1.500<br>
HLA-A*3302 (10-mer peptides)<br>
1	49	QYPGIEIESR	15.000<br>
2	100	ETLEKITNSR	9.000<br>
3	16	EVEPGSGVRI	1.500<br>
4	28	EYCEPCGFEA	1.500<br>
5	68	EINGQLVFSK	1.500<br>
HLA-A68.1<br>
1	16	EVEPGSGVR	900.000<br>
2	9	SVAPPPEEV	12.000<br>
3	50	YPGIEIESR	10.000<br><br>
-160-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
example of<br>
improved peptide	YVGIEIESR	400.000 enhance P2<br>
4	96	ASNGETLEK	9.000<br>
5	101	TLEKITNSR	5.000<br>
H1A-A68.1 (10-mer peptides)<br>
1	100	ETLEKITNSR	300.000<br>
2	16	EVEPGSGVRI	18.000<br>
3	68	EINGGLWSK	9.000<br>
4	15	EEVEPGSGVR	9.000      E is deleterious @ P2<br>
example of<br>
improved peptide	EWEPGSGR	1200.00 replace E with V @ P2<br>
5	95	RASNGETLEK	3.000<br>
HLA-B14<br>
1	94	RRASNGETL	20.000<br>
2	57	SRLGGTGAF	5.000<br>
example of<br>
improved peptide	SRLGGTGAL	100.000 enhance P9<br>
3	100	ETLEKTTNS	3.375<br>
4	105	ITNSRPPCV	2.000<br>
5	88	DLIEAIRRA	1.350<br>
HLA-B14 (10-mer peptides)<br>
1	103	EKITNSRPPC	6.750<br>
example of<br>
improved peptide	ERITNSRPPL	900.000 enhance F10<br>
2	33	CGFEATYLEL	5.000<br>
3	93	IRRASNGETL	4.000<br>
4	18	EPGSGVRIVV	3.000<br><br>
-161-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence      Score (estimated halftone of dissociation)<br>
5	88	DLIEAIRRAS	2.250<br>
HLA-B40<br>
1	65	FEIEINGQL	80.000<br>
2	3	GEPGQTSVA	40.000<br>
3	35	FEATYLELA	40.000<br>
4	15	EEVEPGSGV	24.000<br>
example of<br>
improved peptide	EEVEPGSGL	120.000   enhance P9<br>
5	67	IEINGQLVF	16.000<br>
HLA-B40 (10-raer peptides)<br>
1	55		IESRLGGTGA	20.000<br>
2	53	IEIESRLGGT	16.000<br>
example of<br>
improved peptide	IEIESKLGGL	80.000  enhance P10<br>
3	65	FEDSINGQLV	16.000<br>
4	67	IEINGQLVFS	16.000<br>
5	99	GETLEKITNS	8.000<br>
HLA-B60<br>
1	65	FEIEFNGQL	387.200<br>
2	17	VEPGSGVRI	17.600<br>
example of<br>
improved peptide	VEPGSGVRL	352.000   enhance P9<br>
3	15	EEVEPGSGV	16.000<br>
4	47	KEQYPGIEI	16.000<br>
5	85	YEKDLIEAI	8.800<br><br>
-162-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-B60 (10-mer peptides)<br>
1	65	FEIEINGQLV	16.000<br>
example of<br>
improved peptide	FEIEINGQLL	320.000   enhance P10<br>
2	106	TNSRPPCVIL	16.000<br>
3	53	IEIESRLGGT	8.000<br>
4	33	CGFEATYLEL	8.000<br>
5	17	VEPGSGVRIV	8.000<br>
HLA-B61<br>
1	15	EEVEPGSGV	80.000<br>
2	35	FEATYLELA	40.000<br>
example of<br>
improved peptide	FEATYLELV	160.000  enhance P9<br>
3	3	GEPGQTSVA	22.000<br>
4	65	FEIEINGQL	16.000<br>
5	85	YEKDLIEAI	16.000<br>
HLA-R61 (10-mer peptides)<br>
1	65	FEIEINGQLV	80.000<br>
2	17	VEPGSGVRIV	40.000<br>
3	55	IESRLGGTGA	20.000<br>
4	87	KDLIEAIRRA	10.000<br>
example of<br>
improved peptide	KELIEAIRRV	160.000  enhance P2, P10<br>
5	53	IEIESRLGGT	8.000<br><br>
-163-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B62<br>
1	77	KLENGGFPY	24.000<br>
2	21	SGVRIVVEY	4.800<br>
3	75	FSKLENGGF	3.000<br>
4	31	EPCGFEATY	2.640   P is deleterious @ P2<br>
example of<br>
improved peptide	EQCGFEATY	105.6   replace P with Q @ P2<br>
5	88	DUEAIRRA	2.200<br>
HLA-B62 (10-mer peptides)<br>
1	41	ELASAVKEQY	40.000<br>
2	58	RLGGTGAFEI	9.600<br>
3	66	E1EINGQLVF	7.920<br>
4	56	ESRLGGTGAF	6.000     Sis deleterious @ P2<br>
example of<br>
improved peptide	EQRLGGTGAF	480.000 replace S with Q @ P2<br>
5	20	GSGVRIWEY	4.800     S is deleterious @ P2<br>
example of<br>
improved peptide	GQGVRTWEY	384.000 replace S with Q @P2<br>
HLA-B7<br>
1	107	NSRPPCVIL	60.000<br>
example of<br>
improved peptide	NPRPPCVIL	1200.000   enhance P2<br>
2	45	AVKEQYPGI	6.000<br>
3	22	GVPIVVEYC	5.000<br>
4	70	NGQLVFSKL	4.000<br>
5	81	GGFPYEKDL	4.000<br><br>
-164-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-B7 (10-mer peptides)<br>
1	50	YPGIEIESRL	80.000<br>
2	31	EPCGFEATYL	80.000<br>
3	18	EPGSGVRIVV	6.000<br>
example of<br>
improved peptide	EPGSGVRIVL	120.000 enhance P10<br>
4	106	TNSRPPCVIL	6.000<br>
5	80	NGGFPYEKDL	4.000<br>
HLA-B8<br>
1	107	NSRPPCVIL	4.000<br>
2	45	AVKEQYPGI	1.500<br>
3	105	ITNSRPPCV	0.600<br>
4	56	ESRLGGTGA	0.400<br>
5	100	ETLEKITNS	0.300   S is deleterious @ P9<br>
example of<br>
improved peptide	ETLEKITNL	12.000 replace S with L @ ¥9<br>
HLA-B8 (8-mer peptides)<br>
1	83	FPYEKDLI	6.000<br>
2	107	NSRPPCVI	1.000<br>
3	91	EAIRRASW	0.800   N is deleterious @ P8<br>
example of<br>
improved peptide	EAIRRASL	32.000 replace N with L @ P9<br>
4	20	GSGVRIW	0.600<br>
5	18	EPGSGVRI	0.400<br><br>
-165-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-B8 (10-mer peptides)<br>
1	50	YPGIEIESRL	0.800<br>
2	93	IRRASNGETL	0.400<br>
example of<br>
improved peptide	IA RASNGETL	16.000 replace R with A @ P2<br>
3	31	EPCGFEATYL	0.320<br>
4	104	KITNSRPPCV	0.300<br>
5	18	EPGSGVRIVV	0.240<br>
HLA-B*2702<br>
1	57	SRLGGTGAF	200.000<br>
2	94	RRASNGETL	180.000<br>
example of<br>
improved peptide	RRASNGETF	600.000   enhance P9<br>
3	93	IRRASNGET	20.000<br>
4	27	VEYCEPCGF	15.000<br>
5	77	KLENGGFPY	9.000<br>
HLA-B*2702 (10-mer peptides)<br>
1	93	IRRASNGETL	60.000<br>
2	94	RRASNGETLE	6.000<br>
3	30	CEPCGFEATY	3.000<br>
4	58	RLGGTGAFEI	2.700<br>
5	23	VRIWEYCEP	2.000     P is deleterious @ P10<br>
example of<br>
improved peptide	VRIWEYCEY	200.000 replace P with Y@ P10<br><br>
-166-Examples of predicted human Class I MHC binding peptides - continued Rank     Startposition      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*2705<br>
1	94	RRASNGETL	6000.000<br>
2	57	SRLGGTGAF	1000.000<br>
3	93	IRRASNGET	200.000<br>
example of<br>
improved peptide	IRRA5NGEL	2000.000  enhance P9<br>
4	27	VEYCEPCGF	75.000<br>
5	77	KLENGGFPY	45.000<br>
HLA-B*2705 (10-mer peptides)<br>
1	93	IRRASNGETL	2000.000<br>
2	94	RRASNGETLE	60.000 E is deleterious @P2<br>
example of<br>
improved peptide	RRASNGETLL	6000.000 replace E with JL @ P2<br>
3	78	LENGGFPYEK	30.000<br>
4	95	RASNGETLEK	30.000<br>
5	58	RLGGTGAFEI	27.000<br>
H1A-B*3501<br>
1	31	EPCGFEATY	40.000<br>
2	75	FSKLENGGF	22.500<br>
example of<br>
improved peptide	FPKLENGGM	120.000 enhance P2, P9<br>
3	107	NSRPPCVTL	15.000<br>
4	42	LASAVKEQY	6.000<br>
5	18	EPGSGVRTV	4.000<br><br>
-167-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*3501 (10-mer peptides)<br>
1	31	EPCGFEATYL	30.000<br>
2	50	YPGIEIESRL	20.000<br>
3	56	ESRLGGTGAF	15.000<br>
4	20	GSGVRIWEY	10.000<br>
5	83	FPYEKDL1EA	6.000<br>
example of<br>
improved peptide	FPYEKDLIEM	120.000  enhance P10<br>
HLA-B*3701<br>
1	65	FEIEINGQL	15.000<br>
example of<br>
improved peptide	FDEINGQL	60.000 enhance P2<br>
2	47	KEQYPGIEI	10.000<br>
3	85	YEKDLTEAI	10.000<br>
4	17	VEPGSGVRI	10.000<br>
5	35	FEATYLELA	5.000<br>
HLA-B*3701 (10-mer peptides)<br>
1	65	FEIEINGQLV	10.000<br>
example of<br>
improved peptide	FDIEINGQLI	200.000  enhance P2, P10<br>
2	67	IEINGQLVFS	5.000<br>
3	81	GGFPYEKDLI	5.000<br>
4	87	KDLIEAIRRA	4.000<br>
5	30	CEPCGFEATY	2.000<br><br>
-168-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*3801<br>
1	34	GFEATYLEL	6.000<br>
example of<br>
improved peptide	GHEATYLEL	90.000 enhance P2<br>
2	70	NGQLWSKL	1.560<br>
3	38	TYLELASAV	1.040<br>
4	81	GGFPYEKDL	1.000<br>
5	97	SNGETLEKI	0.720<br>
HLA-B*3801 (10-mer peptides)<br>
1	64	AFEIEINGQL	7.800<br>
example of<br>
improved peptide	AHEIEINGQL	117.000 enhance P2<br>
2	31	EPCGFEATYL	 4.800<br>
3	66	EIEINGQLVF	3.000<br>
4	26	WEYCEPCGF	3.000<br>
5	50	YPGIEIESRL	2.600<br>
HLA-B*3901<br>
1	94	RRASNGETL	15.000<br>
example of<br>
improved peptide	RHASNGETL	90.000   enhance P2<br>
2	34	GFEATYLEL	9.000<br>
3	38	TYLELASAV	4.000<br>
4	66	EIEINGQLV	3.000<br>
5	2	SGEPGQTSV	3.000<br><br>
-169-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*3901 (10-mer peptides)<br>
1	33	CGFEATYLEL	12.000<br>
example of<br>
improved peptide	CHFEATYLEL	360.000   enhance P2<br>
2	64	AFEIEINGQL	9.000<br>
3	93	IRRASNGETL	4.500<br>
4	46	VKEQYPGIEI	3.000<br>
5	16	EVEPGSGVRI	3.000<br>
HLA-B*3902<br>
1	70	NGQLVFSKL	2.400<br>
example of<br>
improved peptide	NKQLVFSKL	24.000 enhance P2<br>
2	81	GGFPYEKDL	2.400<br>
3	94	RRASNGETL	2.000<br>
4	34	GFEATYLEL	2.000<br>
5	107	NSRPPCVTL	0.600<br>
HLA-B*3902 (10-mer peptides)<br>
1	69	INGQLVFSKL	2.400<br>
2	64	AFEIEINGQL	2.400<br>
3	       50	YPGIEIESRL	2.400<br>
4	80	NGGFPYEKDL	2.400<br>
5	106	TNSRPPCVIL	2.000<br><br>
-170-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*4403<br>
1	67	IEINGQLVF	200.000<br>
example of<br>
improved peptide	IEINGQLVY	900.000 enhance P9<br>
2	27	VEYCEPCGF	40.000<br>
3	21	SGVRIWEY	36.000<br>
4	65	FEIEINGQL	20.000<br>
5	35	FEATYLELA	12.000<br>
HLA-B*4403 (10-mer peptides)<br>
1	30	CEPCGFEATY	120.000<br>
2	53	IEIESRLGGT	30.000<br>
example of<br>
improved peptide	IEIESRLGGY	900.000 enhance F10<br>
3	67	IEINGQLVFS	30.000<br>
4	65	FEIEINGQLV	20.000<br>
5	17	VEPGSGVRIV	18.000<br>
HLA-B*5101<br>
1	18	EPGSGVRIV	484.000<br>
2	59	LGGTGAFEI	114.400<br>
example of<br>
improved peptide	LPGTGAFEI	572.000   enhance P2<br>
3	2	SGEPGQTSV	48.400<br>
4	81	GGFPYEKDL	44.000<br>
5	70	NGQLVFSKL	22.000<br><br>
-171-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*5101 (10-mer peptides)<br>
1	18	EPGSGVRIVV	440.000<br>
2	44	SAVKEQYPGI	220.000<br>
example of<br>
improved peptide	SPVKEQYPGI	440.000   enhance P2<br>
3	31	EPCGFEATYL	220.000<br>
4	81	GGFPYEKDLI	176.000<br>
5	50	YPGIEIESRL	157.300<br>
HLA-B*5102<br>
1	18	EPGSGVRIV	242.000<br>
2	81	GGFPYEKDL	110.000<br>
example of<br>
improved peptide	GPFPYEKDI	2200.000 enhance P2, P9<br>
3	59	LGGTGAFEI	96.800<br>
4	70	NGQLVFSKL	48.400<br>
5	2	SGEPGQTSV	24.200<br>
HLA-B*5102 (10-raer peptide)<br>
1	44	SAVKEQYPGI	726.000<br>
example of<br>
improved peptide	SPVKEQYPGI	1452.000 enhance P2<br>
2	50	YPGIEIESRL	400.000<br>
3	81	GGFPYEKDLI	400.000<br>
4	18	EPGSGVRIVV	220.000<br>
5	31	EPCGFEATYL	121.000<br><br>
-172-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
HLA-B*5103<br>
1	59	LGGTGAFEI	48.400<br>
example of<br>
improved peptide	LAFTGAFEI	145.200   enhance P2<br>
2	2	SGEPGQTSV	44.000<br>
3	18	EPGSGVRTV	44.000<br>
4	70	NGQLWSKL	7.260<br>
5	81	GGFPYEKDL	7.200<br>
BLA-B*5103 (10-mer peptide)<br>
1	44	SAVKEQYPGI	110.000<br>
2	81	GGFPYEKDLI	52.800<br>
3	18	EPGSGVRIVV	44.000<br>
example of<br>
improved peptide	EAGSGVRIVV	110.000 enhance P2<br>
4	60	GGTGAFEIEI	44.000<br>
5	33	CGFEATYLEL	7.920<br>
HLA-B*S201<br>
1	18	WPGSGVRIV	75.000<br>
2	67	LEINGQLVF	22.500<br>
example of<br>
improved peptide	LQINGQLVI	450.000 enhance P2, P9<br>
3	59	LGGTGAFEI	11.250<br>
4	98	NGETLEKIT	11.000<br>
5	19	PGSGVRIVV	10.000<br><br>
-173-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subseqnence       Score (estimated half time of dissociation)<br>
HLA-B*5201 (10-mer peptides)<br>
1	18	EPGSGVRIVV	100.000<br>
2	17	VEPGSGVRIV	45.000<br>
example of<br>
improved peptide	VQPGSGVRIV	450.000 enhance P2<br>
3	81	GGFPYEKDLI	33.000<br>
4	105	ITNSRPPCVI	15.000<br>
5	37	ATYLELASAV	12.000<br>
HLA-B*5801<br>
1	75	FSKLENGGF	40.000<br>
example of<br>
improved peptide	FSKLENGGW	80.000   enhance P9<br>
2	42	LASAVKEQY	4.500<br>
3	107	NSRPPCVTL	4.000<br>
4	61	GTGAFEIEI	3.000<br>
5	105	ITNSRPPCV	3.000<br>
HLA-B*5801 (10-mer peptides)<br>
1	56	ESRLGGTGAF	12.000<br>
2	20	GSGVRIWEY	10.800<br>
example of<br>
improved peptide	GSGVRIWEW	144.000 enhance P10<br>
3	1	MSGEPGQTSV	4.000<br>
4	105	ITNSRPPCVI	3.000<br>
5	37	ATYLELASAV	3.000<br><br>
-174-Examples of predicted human Class I MHC binding peptides - continued<br>
Rank     Start position	Subsequence       Score (estimated half time of dissociation)<br>
HLA-Cw*0301<br>
1	65	FEIEINGQL	30.000<br>
2	81	GGFPYEKDL	18.000<br>
3	70	NGQLVFSKL	12.000<br>
4	57	SRLGGTGAF	10.000<br>
5	34	GFEATYLEL	10.000<br>
HLA-Cw*0301 (10-mer peptides)<br>
1	44	SAVKEQYPGI	50.000<br>
example of<br>
improved peptide	SAVKEQYPGL	100.000 enhance P10<br>
2	33	CGFEATYLEL	45.000<br>
3	69	INGQLVFSKL	12.000<br>
4	81	GGFPYEKDLI	3.750<br>
5	106	TNSRPPCVIL	3.000<br>
HLA-Cw*0401<br>
1	34	GFEATYLEL	240,000<br>
2	38	TYLELASAV	30.000<br>
3	82	GFPYEKDLI	25.000<br>
4	18	EPGSGVRIV	20.000<br>
5	31	EPCGFEATY	12.000<br>
example of<br>
improved peptide	EFCGFEATL	200.000 enhance P2,P9<br>
HLA-Cw*0401 (10-mer peptides)<br>
1	64	AFEIEINGQL	200.000<br>
2	74	VFSKLENGGF	100.000<br><br>
-175-Examples of predicted human Class I MHC binding peptides - continued Rank     Start position      Subsequence       Score (estimated half time of dissociation)<br>
example of<br>
improved peptide	VFSKLENGGL	200.000 enhance P10<br>
3	50	YPGIBIESRL	80.000<br>
4	31	EPCGFEATYL	80.000<br>
5	18	EPGSGVRTW	10.000<br>
HLA-Cw*0602<br>
1	85	YEKDLIEAI	6.600<br>
2	65	FEIEINGQL	6.600<br>
3	21	SGVRIWEY	6.000<br>
4	31	EPCGFEATY	3.300<br>
5	61	GTGAGEIEI	3.000<br>
HLA-Cw*0702<br>
1	       31	EPCGFEATY	24.000'<br>
2	21	SGVRIWEY	19.200<br>
3	42	LASAVKEQY	8.800<br>
4	77	KLENGGFFY	4.000<br>
5	49	QYPGIEIES	2.880<br>
KLA-Cw*0702 (10-mer peptides)<br>
1	20	GSGVRIWEY	38.400<br>
2	30	CEPCGFEATY	16.000<br>
3	41	ELASAVKEQY	16.000<br>
4	50	YPGIEIESRL	7.920<br>
5	76	SKLENGGEPY	4.000<br><br>
-176-<br><br><br>
-177-<br><br><br>
-178-<br><br><br>
-179-<br>
DRB1*1107<br>
DRB1*1305<br>
DRB1*1307<br>
DRB1*1311<br>
DRB1*1321<br>
DRB1*1501<br>
DRB1*1502<br>
DRB5*0101<br>
*Class II MHC epitopes predicted using TEPITOPE software. Sturniolo, T., et al. 1999. Generation of tissue-specific and promiscuous HLA ligand databases using DNA microarrays and virtual HLA class II matrices. Nature Biotechnology 17:555-571.<br>
In the present invention, "epitopes" refer to C35 polypeptide fragments<br>
having antigenic or immunogenic activity in an animal, especially in a human,<br>
or that are capable of eliciting a T lymphocyte response in an animal, preferably<br>
a human. A preferred embodiment of the present invention relates to a C35<br>
polypeptide fragment comprising an epitope, as well as the polynucleotide<br>
encoding this fragment. A further preferred embodiment of the present invention<br>
relates to a C35 polypeptide fragment consisting of an epitope, as well as the<br>
polynucleotide encoding this fragment. In specific preferred embodiments of the<br>
present invention, the epitope comprises a C35 fragment listed in any of Tables<br>
1-6.  In another preferred embodiment of the present invention, the epitope<br>
consists of a C35 fragment listed in any of Tables 1-6. A region of a protein<br>
molecule to which an antibody can bind is defined as an "antigenic epitope." In<br>
contrast, an "immunogenic epitope" is defined as a part of a protein that elicits<br>
an antibody response. (See, for instance, Gey sen et al., Proc. Natl Acad. Sci<br>
USA 81 : 3998- 4002 (1983)).   Thus, a further preferred embodiment of the<br>
present invention is an immunogenic C35 peptide fragment mat is capable of<br>
eliciting a T cell response when bound to the peptide binding cleft of an MHC<br>
molecule. In a specific preferred embodiment, the immunogenic C35 peptide<br><br>
-180-fragment comprises an epitope listed in any of Tables 1-6. In another preferred embodiment, the immunogenic C35 peptide fragment consists of an epitope listed in any of Tables 1-6. Further embodiments of the invention are directed to pharmaceutical formulations and vaccine compositions comprising said immunogenic C35 peptide fragments or the polynucleotides encoding them.<br>
Fragments which function as epitopes may be produced by any conventional means. (See, e.g., Houghten5 R. A., Proc. Natl Acad. Sci. USA 82:5131-5135 (1985) further described in U.S. Patent No. 4,631,211.)<br>
The sequence of peptide epitopes known to bind to specific MHC molecules can be modified at the known peptide anchor positions in predictable ways that act to increaseMHC binding affinity. Such "epitope enhancement" has been employed to improve the immunogenicity of a number of different MHC class I or MHC class II binding peptide epitopes (Berzofsky, J. A, et al., Immunol. Rev. 170:151-72 (1999); Ahlers, J.D. et al., Proc. Natl. Acad. Sci U.S.A. 24:10856-61 (1997); Overwijk, et al., J. Exp. Med. 188:211-86 (1998); Parkhurst, M.R. et al., J. Immunol 757:2539-48 (1996)). Accordingly, a further embodiment of the invention is directed to such enhanced C35 epitopes, and to the polynucleotides encoding such enhanced epitomes.<br>
In the present invention, antigenic epitopes preferably contain a sequence of at least seven, more preferably at least nine, and most preferably between about 15 to about 30 amino acids. Antigenic epitopes are useful to raise antibodies, including monoclonal antibodies, that specifically bind the epitope. (See, for instance, Wilson et al., Cell 37:767-778 (1984); Sutcliffe, J. G. et al., Science 219:660-666 (1983).)<br>
Similarly, immunogenic epitopes can be used to induce B cells and T cells according to methods well known in the art. (See Sutcliffe et al., supra; Wilson et al., supra; Chow, M. et al., Proc. Natl Acad. Sci USA 82:910-914; and Bittle, F. J. et al., J. Gen. Virol 66:2341-2354 (1985).) A preferred immunogenic epitope includes the secreted protein. The immunogenic epitopes may be presented together with a carrier protein, such as an albumin, to an<br><br>
-181-<br>
anirnal system (such as rabbit or mouse) or, if it is long enough (at least about 2 5 amino acids), without a carrier. However, immunogenic epitopes comprising as few as 9 amino acids have been shown to be sufficient to raise antibodies capable of binding to, at the very least, linear epitopes in a denatured polypeptide (e.g., in Western blotting.)<br>
As used herein, the term "antibody" (Ab) or "monoclonal antibody" (Mab) is meant to include intact molecules as well as antibody fragments (such as, for example, Fab and F(ab)2 fragments) which are capable of specifically binding to protein. Fab and F(ab)2 fragments lack the Fc fragment of intact antibody, clear more rapidly from the circulation, and may have less non-specific tissue binding than an intact antibody. (Wahl et at, J. Nucl. Med. 24:316-325 (1983).) Thus, for some applications these fragments are preferred, as well as the products of a Fab or other immunoglobulin expression library. Moreover, antibodies of the present invention include chimeric, single chain, andhumanized antibodies.<br>
Diagnostic and Therapeutic Uses of Antibodies<br>
The present invention further relates to C35 antibodies, C35 antibody fragments and antibody conjugates and single-chain immunotoxins reactive with human carcinoma cells, particularly human breast and bladder carcinoma cells.<br>
Table 7 provides a list of C35-specific monoclonal antibodies that have been isolated and characterized for use in different applications.<br>
TABLE 7: C35 -Specific Murine Monoclonal Antibodies<br><br>
Fusion	Hybridoma	ELISA	Isotype	...<br>
Western Blot	Flow Cytometry	Immunohisto-chemistry<br>
alpha	1F5	positive	IgM		positive	positive<br>
	1F7	positive	IgM		positive	<br><br>
-182-<br><br>
Fusion	Hybridoma	ELISA	Isotype	Western Blot	Flow Cytometry	Immunohisto-chemistry<br>
	1F11	positive	IgM		positive	<br>
	2D9	positive	IgM	positive	positive	positive<br>
beta   .	2G3	positive	IgG1			<br>
	2G8	positive				<br>
	2G10	positive	IgG3			<br>
	2G11	positive	IgG3			<br>
	3F9	positive	IgG1			<br>
	4D11	positive	IgG1			<br>
	4G3	positive	IgG3			<br>
	7C2	positive	IgM			<br>
	8B11	positive	IgM			<br>
	8G2	positive	IgM			<br>
	10F4	positive	IgG1			<br>
	11B10	positive	IgM	positive		<br>
	12B10	positive				<br>
	16C10	positive	IgM			<br>
	16F10	positive				<br>
ELISA assay on bacterially-synthesized C35 blank = not determined<br>
As used in this example, the following words or phrases have the meanings specified.<br>
As used in this example, "joined" means to couple directly or indirectly one molecule with another by whatever means, e.g., by covalent bonding, by non-covalent bonding, by ionic bonding, or by non-ionic bonding. Covalent bonding includes bonding by various linkers such as thioether linkers or thioester linkers. Direct coupling involves one molecule attached to the molecule of interest. Indirect coupling involves one molecule attached to another molecule not of interest which in turn is attached directly or indirectly to the molecule of interest.<br><br>
-183-<br>
As used in this example, "recombinant molecule" means a molecule produced by genetic engineering methods.<br>
As used in this example, "fragment" is defined as at least a portion of the variable region of the immunoglobulin molecule which binds to its target, i.e. the antigen binding region. Some of the constant region of the immunoglobulin may be included.<br>
As used in this example, an "immunoconjugate" means any molecule or ligand such as an antibody or growth factor chemically or biologically linked to a cytotoxin, a radioactive agent, an anti-tumor drug or a therapeutic agent. The antibody or growth factor may be linked to the cytotoxin, radioactive agent, anti-tumor drug or therapeutic agent at any location along the molecule so long as it is able to bind its target. Examples of immuoconjugates include immunotoxins and antibody conjugates.<br>
Asusedinthis example, "selectively killing" means killing those cells to which the antibody binds.<br>
As used in this example, examples of "carcinomas" include bladder, breast, colon, liver, lung, ovarian, and pancreatic carcinomas.<br>
As used in this example, "immunotoxin" means an antibody or growth factor chemically or biologically linked to a cytotoxin or cytotoxic agent.<br>
As used in this example, an "effective amount" is an amount of the antibody, immunoconjugate, recombinant molecule which kills cells or inhibits the proliferation thereof.<br>
As used in this example, "competitively inhibits" means being capable of binding to the same target as another molecule. With regard to an antibody, competitively inhibits mean that the antibody is capable, of recognizing and binding the same antigen binding region to which another antibody is directed.<br>
As used in this example, "antigen-binding region" means that part of the antibody, recombinant molecule, the fusion protein, or the immunoconjugate of the invention which recognizes the target or portions thereof.<br><br>
-184-<br>
As used in this example, "therapeutic agent" means any agent useful for therapy including anti-tumor drugs, cytotoxins, cytotoxin agents, and radioactive agents.<br>
As used in this example, "anti-tumor drug" means any agent useful to combat cancer including, but not limited to, cytotoxins and agents such as antimetabolites, alkylating agents, anthracyclines, antibiotics, antimitotic agents, procarbazine, hydroxyurea, asparaginase, corticosteroids, mytotane (0,P'-(DDD)), interferons and radioactive agents.<br>
As used in this example, "a cytotoxin or cytotoxic agent" means any agent that is detrimental to cells. Examples include taxol, cytochalasin B, gramicidin D, ethidium bromide, emetine, mitomycin, etoposide, tenoposide, vincristine, vinblastine, colchicin, doxorubicin, daunorubicin, dihydroxy anthracin dione, mitoxantrone, mithramycin, actinomycin D, 1-dehydrotestosterone, glucocorticoids, procaine, tetracaine, lidocaine, propranolol, and puromycin and analogs or homologs thereof.<br>
As used in this example, "radioisotope" includes any radioisotope which is effective in destroying a tumor. Examples include, but are not limited to, cobalt-60 and X-rays. Additionally, naturally occurring radioactive elements such as uranium, radium, and thorium which typically represent mixtures of radioisotopes, are suitable examples of a radioactive agent.<br>
As used in this example, "administering" means oral administration, administration as a suppository, topical contact, intravenous, intraperitoneal, intramuscular or subcutaneous administration, or the implantation of a slow-release device such as a miniosmotic pump, to the subject.<br>
As used in this example, "directly" means the use of antibodies coupled to a label. The specimen is incubated with the labeled antibody, unbound antibody is removed by washing, and the specimen may be examined.<br>
As used in this example, "indirectly" means incubating the specimen with an    unconjugated   antibody,   washing   and   incubating   with    a<br><br>
-185-<br>
fluorochrome-coujugated antibody. The second or "sandwich" antibody thus reveals the presence of the first.<br>
As used in this example "reacting" means to recognize and bind the target. The binding may be non-specific. Specific binding is preferred.<br>
As used in this example, "curmg" meanstoprovidesubstantially complete tumor regression so that the tumor is not palpable for a period of time, i.e., &gt;/= 10 tumor volume doubling delays (TVDD = the time in days that it takes for control tumors to double in size).<br>
As used in this example, "tumor targeted antibody" means any antibody NO:1which recognizes the C35 antigen on tumor (i.e., cancer) cells.<br>
As used in this example, "inhibit proliferation" means to interfere with cell growth by whatever means.<br>
As used in this example, "mammalian tumor cells" include cells from animals such as human, ovine, porcine, murine, bovine animals.<br>
As used in this example, "pharmaceutically acceptable carrier" includes any material which when combined with the antibody retains the antibody's immunogenicity and is non-reactive with the subject's immune systems. Examples include, but are not limited to, any of the standard pharmaceutical carriers such as a phosphate buffered saline solution, water, emulsions such as oil/water emulsion, and various types of wetting agents. Other carriers may also include sterile solutions, tablets including coated tablets and capsules.<br>
Typically such carriers contain excipients such as starch, milk, sugar, certain types of clay, gelatin, stearic acid or salts thereof, magnesium or calcium stearate, talc, vegetable fats or oils, gums, glycols, or other known excipients. Such carriers may also include flavor and color additives or other ingredients. Compositions comprising such carriers are formulated by well known conventional methods.<br>
The present invention relates to C35 antibodies that are highly specific for carcinoma cells. More particularly, the antibodies react with a range of carcinomas such as breast, bladder, lung, ovary and colon carcinomas, while<br><br>
-186-<br>
showing none or limited reactivity with normal human tissues or other types of tumors such as, for example, sarcomas or lymphomas.<br>
The term "C35 antibody" as used herein includes whole, intact polyclonal and monoclonal antibody materials, and chimeric antibody molecules. The C35 antibody described above includes any fragments thereof containing the active antigen-binding region of the antibody such as Fab, F(ab')2 and Fv fragments, using techniques well established in the art [see, e.g., Rousseaux et al., "Optimal Conditions For The Preparation of Proteolytic Fragments From Monoclonal IgG of Different Rat IgG Subclasses", in Methods Enzymol, 121:665-69 (Academic Press 1986)]. The C35 antibody of the invention also includes fusion proteins.<br>
Also included within the scope of the invention are anti-idiotypic antibodies to the C35 antibody of the invention. These anti-idiotypic antibodies can be produced using the C35 antibody and/or the fragments thereof as immunogen and are useful for diagnostic purposes in detecting humoral response to tumors and in therapeutic applications, e.g., in a vaccine, to induce an anti-tumor response in patients [see, e.g., Nepom et al., "Anti-idiotypic Antibodies And The Induction Of Specific Tumor Immunity", in Cancer And Metastasis Reviews, 5:487-501 (1987)].<br>
In addition, the present invention encompasses antibodies that are capable of binding to the same antigenic determinant as the C35 antibodies and competing with the antibodies for binding at that site. These include antibodies having the same antigenic specificity as the C35 antibodies but differing in species origin, isotype, binding affinity or biological functions (e.g., cytotoxicity). For example, class, isotype and other variants of the antibodies of the invention having the antigen-binding region of the C35 antibody can be constructed using recombinant class-switching and fusion techniques known in the art [see, e.g., Thammana et al., "Immunoglobulin Heavy Chain Class Switch From IgM to IgG In A Hybridoma", Eur. J. Immunol, 13:614 (1983); Spira et al., "The Identification Of Monoclonal Class Switch Variants By Subselection And ELISA Assav", J. Immunol Metk 74:301-15 (1984); Neuberger et al.,<br><br>
-187-<br>
"Recombinant Antibodies Possessing Novel Effector Functions", Nature 312: 614-608 (1984); and Oi et al., "Chimeric Antibodies", Biotechniques 4 (3):214-21 (1986)]. Thus, other chimeric antibodies or other recombinant antibodies (e.g., fusion proteins wherein the antibody is combined with a second protein such as a lymphokine or a tumor inhibitory growth factor) having the same binding specificity as the C35-specific antibodies fall within the scope of this invention.<br>
Genetic engineering techniques known in the art may be used as described herein to prepare recombinant immunotoxins produced by fusing antigen binding regions of antibody C35 to a therapeutic of cytotoxic agent at the DNA level and producing the cytotoxic molecule as a chimeric protein. Examples of therapeutic agents include, but are not limited to, antimetabolites, alkylating agents, anthracyclines, antibiotics, and anti-mitotic agents. Antimetabolites include methotrexate, 6-mercaptopurine, 6-thioguanine, cytarabine, 5-fluorouracil decarbazine. Alkylating agents include mechorethamine, thioepa chlorambucil, melphalan, carmustine (BSNU) and lomustine (CCNU), cyclothosphamide, busulfan, dibromomannitol, streptozotocin, mitomycin C, and cis-dichlorodiamine platinum (II) (DDP) cisplatin. Anthracyclines include daunorubicin (formerly daunomycin) and doxorubicin (also referred to herein as adriamycin). Additional examples include mitozantrone and bisantrene. Antibiotics include dactinomycin (formerly actinomycin), bleomycin, mithramycin, and anihramycin (AMC). Antimytotic agents include vincristine and vinblastine (which are commonly referred to as vinca alkaloids). Other cytotoxic agents include procarbazine, hydroxyurea, asparaginase, corticosteroids, mytotane (0,P'-(DDD)), interferons. Further examples of cytotoxic agents include, but are not limited to, ricin, doxorubicin, taxol, cytochalasin B, gramicidin D, ethidium bromide, etoposide, tenoposide, colchicin, dihydroxy anthracin dione, 1-dehydrotestosterone, andglucocorticoid<br>
Clearly analogs and homologs of such therapeutic and cytotoxic agents are encompassed by the present invention. For example, the chemotherapuetic<br><br>
-188-<br>
agent aminopterin has a correlative improved analog namely methotrexate. Further, the improved analog of doxorubicin is an Fe-chelate. Also, the improved analog for 1-methylnitrosourea is lomustine. Further, the improved analog of vinblastine is vincristine. Also, the improved analog of mechlorethamine is cyclophosphamide.<br>
Recombinant immunotoxins, particularly single-chain immunotoxins, have an advantage over drug/antibody conjugates in that they are more readily produced than these conjugates, and generate a population of homogenous molecules, i.e. single peptides composed of the same amino acid residues. The techniques for cloning and expressing DNA sequences encoding the amino acid sequences corresponding to C35 single-chain immunotoxins, e.g synthesis of oligonucleotides, PCR, transforming cells, constructing vectors, expression systems, and the like are well-established in the art, and most practitioners are familiar with the standard resource materials for specific conditions and procedures [see, e.g. Sambrook et al., eds., Molecular Cloning, A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press (1989)].<br>
The following include preferred embodiments of the immunoconjugates of the invention. Other embodiments which are known in the art are encompassed by the invention. The invention is not limited to these specific immunoconjugates, but also includes other immunoconjugates incorporating antibodies and/or antibody fragments according to the present invention.<br>
The conjugates comprise atleast one drug molecule connected by a linker of the invention to a targeting ligand molecule that is reactive with the desired target cell population. The ligand molecule can be an immunoreactive protein such as an antibody, or fragment thereof, a non-immunoreactive protein or peptide ligand such as bombesin or, a binding ligand recognizing a cell associated receptor such as a lectin or steroid molecule.<br>
Further, because the conjugates of the invention can be used for modifying a given biological response, the drug moiety is not to be construed as limited to classical chemical therapeutic agents. For example, the drug moiety<br><br>
-189-<br>
may be a protein or polypeptide possessing a desired biological activity. Such proteins may include, for example, a toxin such as abrin, ricin A, pseudomonas exotoxin, or diphtheria toxin; a protein such as tumor necrosis factor, alpha -interferon, beta -interferon, nerve growth factor, platelet derived growth factor, tissue plasminogen activator; or, biological response modifiers such as, for example,   lymphokines,   interleukin-1    ("IL-1"),   interleukin-2    ("IL-2"), interleukin-6 ("DL-6"), granulocyte macrophase colony stimulating factor ("GM-CSF"), granulocyte colony stimulating factor ("G-CSF"), or other growth factors.<br>
The preferred drugs for use in the present invention are cytotoxic drugs, particularly those which are used for cancer therapy. Such drugs include, in general, alkylating agents, anti-proliferative agents, tubulin binding agents and the like. Preferred classes of cytotoxic agents include, for example, the anthracycline family of drugs, the vinca drugs, the mitomycins, the bleomycins, the cytotoxic nucleosides, the pteridine family of drugs, diynenes, and the podophyllotoxins. Particularly useful members of those classes include, for example, adriamycin, carminomycin, daunorubicin, aminopterin, methotrexate, methopterin, dichloromethotrexate, mitomycin C, porfiromycin, 5-fluorouracil, 6-mercaptopurine, cytosine arabinoside, podophyllotoxin, or podophyllotoxin derivatives such as etoposide or etoposide phosphate, melphalan, vinblastine, vincristine, leurosidine, vindesine, leurosine and the like. As noted previously, one skilled in the art may make chemical modifications to the desired compound in order to make reactions of that compound more convenient for purposes of preparing conjugates of the invention.<br>
As noted, one skilled in the art will appreciate that the invention also encompasses the use of antigen recognizing immunoglobulin fragments. Such immunoglobutin fragments may include, for example, the Fab', F(ab')2, F[v ]or Fab fragments, or other antigen recognizing immunoglobulin fragments. Such immunoglobulin fragments can be prepared, for example, by proteolytic enzyme digestion, for example, by pepsin or papain digestion, reductive alkylation, or<br><br>
-190-<br>
recombinant techniques. The materials and methods for preparing such immunoglobulin fragments are well-known to those skilled in the art. See generally, Parham, J. Immunology, 131, 2895 (1983); Lamoyi et al., J. Immunological Methods, 56, 235 (1983); Parham, id, 53, 133 (1982); and Matthew et al., id., 50,239 (1982).<br>
The immunoglobulin can be a "chimeric antibody" as that term is recognized in the art. Also, the immunoglobulin may be a "bifunctional" or "hybrid" antibody, that is, an antibody which may have one arm having a specificity for one antigenic site, such as a tumor associated antigen while the other arm recognizes a different target, for example, a hapten which is, or to which is bound, an agent lethal to the antigen-bearing tumor cell. Alternatively, the bifunctional antibody may be one in which each arm has specificity for a different epitope of a tumor associated antigen of the cell to be therapeutically or biologically modified. In any case, the hybrid antibodies have a dual specificity, preferably with one or more binding sites specific for the hapten of choice or one or more binding sites specific for a target antigen, for example, an antigen associated with a tumor, an infectious organism, or other disease state.<br>
Biological bifunctional antibodies are described, for example, in European Patent Publication, EPA 0105360, to which those skilled in the art are referred. Such hybrid or bifunctional antibodies may be derived, as noted, either biologically, by cell fusion techniques, or chemically, especially with cross-linking agents or disulfidebridge-forming reagents, and may be comprised of whose antibodies and/or fragments thereof. Methods for obtaining such hybrid antibodies are disclosed, for example, in PCT application W083/03679, published Oct. 27,1983, and published European Application EPA 0 217 577, published Apr. 8,1987. Particularly preferred bifunctional antibodies are those biologically prepared from a "polydome" or "quadroma" or which are synthetically prepared with cross-linking agents such as bis-(maleimideo)-methyl ether ("BMME"), or with other cross-linking agents familiar to those skilled in the art.<br><br>
-191-<br>
In addition the immunoglobulin may be a single chain antibody ("SCA"). These may consist of single chain Fv fragments ("scFv") in which the variable light ("V[L]") and variable heavy ("V[H]") domains are linked by a peptide bridge or by disulfide bonds. Also, the immunoglobulin may consist of single V[H]domains (dAbs) which possess antigen-binding activity. See, e.g., G. Winter and C. Milstein, Nature 349:295 (1991); R. Glockshuber et al., Biochemistry 29: 1362 (1990); and, E. S. Ward et al., Nature 341: 544 (1989).<br>
Especially preferred for use in the present invention are chimeric monoclonal antibodies, preferably those chimeric antibodies having specificity toward a tumor associated antigen. As used in this example, the term "chimeric antibody" refers to a monoclonal antibody comprising a variable region, i.e. binding region, from one source or species and at least a portion of a constant region derived from a different source or species, usually prepared by recombinant DNA techniques. Chimeric antibodies comprising amurine variable region and a human constant region are preferred in certain applications of the invention, particularly human therapy, because such antibodies are readily prepared and may be less immunogenic than purely murine monoclonal antibodies. Such murine/human chimeric antibodies are the product of expressed immunoglobulin genes comprising DNA segments encoding murine immunoglobulin variable regions and DNA segments encoding human immunoglobulin constant regions. Other forms of chimeric antibodies encompassed by the invention are those in which the class or subclass has been modified or changed from that of the original antibody. Such "chimeric" antibodies are also referred to as "class-switched antibodies". Methods for producing chimeric antibodies involve conventional recombinant DNA and gene transfection techniques now well known in the art. See, e.g., Morrison, S. L. et al., Proc. Nat'l Acad. Sci., 81, 6851 (1984).<br>
Encompassed by the term "chimeric antibody" is the concept of "humanized antibody", that is those antibodies in which the framowork or "coplementarity" determimgregions ("CDR") have been modified to comprise<br><br>
-192-<br>
the CDR of an immunoglobulin of different specificity as compared to that of the parent immunoglobulin. In a preferred embodiment, a murine CDR is grafted into the framework region of a human antibody to prepare the "humanized antibody". See, e.g., L. Riechmann et al., Nature 332: 323 (1988); M. S. Neuberger et al., Nature 314: 268 (1985). Particularly preferred CDR'S correspond to those representing sequences recognizing the antigens noted above for the chimeric and bifunctional antibodies? The reader is referred to the teaching of EPA 0 239 400 (published Sep. 30,1987), for its teaching of CDR modified antibodies.<br>
One skilled in the art will recognize that a bifunctional-chimeric antibody can be prepared which would have the benefits of lower immunogenicity of the chimeric or humanized antibody, as well as the flexibility, especially for therapeutic treatment, of the bifunctional antibodies described above. Such bifunctional-chimeric antibodies can be synthesized, for instance, by chemical synthesis using cross-linking agents and/or recombinant methods of the type described above. In any event, the present invention should not be construed as limited in scope by any particular method of production of an antibody whether bifunctional, chimeric, bifunctional-chimeric, humanized, or an antigen-recognizing fragment or derivative thereof.<br>
In addition, the invention encompasses within its scope immunoglobulins (as defined above) or immunoglobulin fragments to which are fused active proteins, for example, an enzyme of the type disclosed in Neuberger, et al., PCT application, WO86/01533, published Mar. 13, 1986. The disclosure of such products is incorporated herein by reference.<br>
As noted, "bifunctional", "fused", "chimeric" (including humanized), and 'bifunctional-chimeric" (including humanized) antibody constructions also include, within their individual contexts constructions comprising antigen recognizing fragments. As one skilled in the art will recognize, such fragments could be prepared by traditional enzymatic cleavage of intact bifunctional, chimeric, humanized, or chimeric-bifunctional antibodies. If, however, intact<br><br>
-193-<br>
antibodies are not susceptible to such cleavage, because of the nature of the construction involved, the noted constructions can be prepared with immunoglobulin fragments used as the starting materials; or, if recombinant techniques are used, the DNA sequences, themselves, can be tailored to encode the desired "fragment" which, when expressed, can be combined in vivo or in vitro, by chemical or biological means, to prepare' the final desired intact immunoglobulin "fragment". It is in this context, therefore, that the term "fragment" is used.<br>
Furthermore, as noted above, the immunoglobulin (antibody), or fragment thereof, used in the present invention may be polyclonal or monoclonal in nature. Monoclonal antibodies are the preferred immunoglobulins, however. The preparation of such polyclonal or monoclonal antibodies now is well known to those skilled in the art who, of course, are fully capable of producing useful immunoglobulins which can be used in the invention. See, e.g., G. Kohler and C. Milstein, Nature256:495 (1975). In addition, hybridomes and/or monoclonal antibodies which are produced by such hybridomas and which are useful in the practice of the present invention are publicly available from sources such as the AmericanType Culture Collection ("ATCC") 10801 University Blvd., Manassas, VA. 20110.<br>
Particularly preferred monoclonal antibodies for use in the present invention are those which recognize tumor associated antigens-Diagnostic Techniques<br>
Serologic diagnostic techniques involve the detection and quantitiation of tumor-associated antigens that have been secreted or "shed" into the serum or other biological fluids of patients thought to be suffering from carcinoma. Such antigens can be detected in the body fluids using techniques known in the art such as radioimmunoassays (RIA) or enzyme-linked immunosorbent assays (ELISA) wherein an antibody reactive with the "shed" antigen is used to detect<br><br>
-194-<br>
the presence of the antigen in a fluid sample [see, e.g., Uotila et al., "Two-Site Sandwich ELISA With Monoclonal Antibodies To Human AFP", J. Immunol. Methods, 42:11 (1981) and Allum et al., supra at pp. 48-51], These assays, using the C35 antibodies disclosed herein, can therefore be used for the detection in biological fluids of the antigen with which the C35 antibodies react and thus the detection of human carcinoma in patients. Thus, it is apparent from the foregoing that the C35 antibodies of the invention can be used in most assays involving antigen-antibody reactions. These assays include, but are not limited to, standard RIA techniques, both liquid and solid phase, as well as ELISA assays, ELISPOT, immunofluorescence techniques, and other immunocytochemical assays [see, e.g., Sikoraet al. (eds), Monoclonal Antibodies, pp. 32-52 (Blackwell Scientific Publications 1984)].<br>
The invention also encompasses diagnostic kits for carrying out the assays described above. In one embodiment, the diagnostic kit comprises the C35 monoclonal antibody, fragments thereof, fusion proteins or chimeric antibody of the invention, and a conjugate comprising a specific binding partner for the C35 antibody and a label capable of producing a detectable signal. The reagents can also include ancillary agents such as buffering agents and protein stabilizing agents (e.g., polysaccharides). The diagnostic kit can further comprise, where necessary, other components of the signal-producing system including agents for reducing background interference, control reagents or an apparatus or container for conducting the test<br>
In another embodiment, the diagnostic kit comprises a conjugate of the C35 antibodies of the invention and a label capable of producing a detectable single. Ancillary agents as mentioned above can also be present.<br>
The C35 antibody of the invention is also useful for in vivo diagnostic applications for the detection of human carcinomas. One such approach involves the detection of tumors in vivo by tumor imaging techniques. According to this approach, the C35 antibody is labeled with an appropriate imaging reagent that produces a detectable signal. Examples of imaging reagents that can be used<br><br>
-195-<br>
include, but at not limited to, radiolabels such as  I,  In,  I,  Tc,  P,  I,  H, and  C, fluorescent labels such as fluorescein and rhodamine, and chemiluninescers such as luciferin. The antibody can be labeled with such reagents using techniques known in the art. For example, see Wensel and Meares, Radioimmunoimaging And Radioimmunotherapy, Elsevier, New York (1983) for techniques relating to the radiolabeling of antibodies [see also, Colcher et al., "Use Of Monoclonal Antibodies As Radiopharmaceuticals For The Localization Of Human Carcinoma Xenografts In Athymic Mice", Meth, Enzymol 727:802-16 (1986)].<br>
In the case of radiolabeled antibody, the antibody is administered to the patient, localizes to the tumor bearing the antigen with which the antibody reacts, and is detected or "imaged" in vivo using known techniques such as radionuclear scanning using, e.g., a gamma camera or emission tomography [see, e.g., Bradwell et al., "Developments In Antibody Imaging", inMonoclonal Antibodies For Cancer Detection And Therapy, Baldwin et al. (eds.), PP- 65-85 (Academic Press 1985)]. the antibody is administered to the patient in a pharmaceutically acceptable carrier such as water, saline, Ringer's solution, Hank's solution or nonaqueous carriers such as fixed oils. The carrier may also contain substances that enhance isotonicity and chemical stability of the antibody such as buffers or preservatives. The antibody formulation is administered, for example, intravenously, at a dosage sufficient to provide enough gamma emission to allow visualization of the tumor target site. Sufficient time should be allowed between administration of the antibody and detection to allow for localization to the tumor target For a general discussion of tumor imaging, see Allum et al., supra at pp. 51-55.<br><br>
-196-<br>
Therapeutic Applications of C35 Antibodies<br>
The properties of the C35 antibody suggest a number of in vivo therapeutic applications.<br>
First, the C35 antibody can be used alone to target and kill tumor cells in vivo. The antibody can also be used in conjunction with an appropriate therapeutic agent to treat human carcinoma. For example, the antibody can be used in combination with standard or conventional treatment methods such as chemotherapy, radiation therapy or can be conjugated or linked to a therapeutic drug, or toxin, as well as to a lymphokine or a tumor-inhibitory growth factor, for delivery of the therapeutic agent to the site of the carcinoma.<br>
Techniques for conjugating such therapeutic agents to antibodies are well known [see, e.g., Arnon et al., "Monoclonal Antibodies For Immunotargeting Of Drugs In Cancer Therapy", in Monoclonal Antibodies And Cancer Therapy, Reisfeld et al. (eds.), pp. 243-56 (Alan R. Liss, Inc. 1985); Hellstrom et al., "Antibodies For Drug Delivery", in Controlled Drug Delivery (2nd Ed.), Robinson et al. (eds.), pp. 623-53 (Marcel Dekker, Inc. 1987); Thorpe, "Antibody Carriers Of Cytotoxic Agents In Cancer Therapy: A Review", in Monoclonal Antibodies '84: Biological And Clinical Applications, Pinchera et al. (eds.), pp. 475-506(1985); and Thorpe et al., "The Preparation And Cytotoxic Properties Of Antibody-Toxin Conjugates", Immunol. Rev., 62:119-58 (1982)]. Alternatively, the C35 antibody can be coupled to high-energy radiation, e.g., a radioisotope such as  I, which, when localized at the tumor site, results in a killing of several cell diameters [see, e.g., Order, "Analysis, Results, And Future Prospective Of The Therapeutic Use Of Radiolabeled Antibody In Cancer Therapy", in Monoclonal Antibodies For Cancer Detection And Therapy, Baldwin et al. (eds.), pp. 303-16 (Academic Press 1985)]. According to yet another embodiment, the C35 antibody can be conjugated to a second antibody to form an antibody heteroconjugate for the treatment of tumor cells as described by Segal in U.S. Pat. No. 4,676,980.<br><br>
-197-<br>
Still other therapeutic applications for the C35 antibody of the invention include conjugation or linkage, e.g., by recombinant DNA techniques, to an enzyme capable of converting a prodrug into a cytotoxic drug and the use of that antibody-enzyme conjugate in combination with the prodrug to convert the prodrug to a cytotoxic agent at the tumor site [see, e.g., Senter et al., "Anti-Tumor Effects Of Antibody-alkaline Phosphatase", Proc. Natl. Acad. Sci. USA, 85:4842-46 (1988); "Enhancement of the in vitro and in vivo Antitumor Activites of Phosphorylated Mitocycin C and Etoposide Derivatives by Monoclonal Antibody-Alkaline Phosphatase Conjugates", Cancer Research 49:5789-5792 (1989); andSenter, "Activation of Prodrugs by Antibody-Enzyme Conjugates: A New Approach to Cancer Therapy," FASEB J. 4:188-193 (1990)].<br>
Still another therapeutic use for the C35 antibody involves use, either in the presence of complement or as part of an antibody-drug or antibody-toxin conjugate, to remove tumor cells from the bone marrow of cancer patients. According to this approach, autologous bone marrow may be purged ex vivo by treatment with the antibody and the marrow infused back into the patient [see, e.g., Ramsay et al., "Bone Marrow Purging Using Monoclonal Antibodies", J. Clin. Immunol., 8(2):81-88 (1988)].<br>
Furthermore, chimeric C35, recombinant irnmunotoxins and other recombinant constructs of the invention containing the specificity of the antigen-binding region of the C35 monoclonal antibody, as described earlier, may be used therapeutically. For example, the single-chain irnmunotoxins of the invention, may be used to treat human carcinoma in vivo.<br>
Similarly, a fusion protein comprising at least the antigen-binding region of the C35 antibody joined to at least a functionally active portion of a second protein having anti-tumor acitivty, e.g., a lymphokine or oncostatin can be used to treat human carcinoma in vivo. Furthermore, recombinant techniques known in the art can be used to construct bispecific antibodies wherein one of the binding specificities of the antibody is that of C35, while the other binding specificty of the antibody is that of a molecule other than C35.<br><br>
-198-<br>
Finally, anti-idiotypic antibodies of the C35 antibody may be used therapeutically in active tumor immunization and tumor therapy [see, e.g., Hellstrom et al., "Immunological Approaches To Tumor Therapy: Monoclonal Antibodies, Tumor Vaccines, And Anti-Idiotypes", in Covalently Modified Antigens And Antibodies In Diagnosis And Therapy, supra at pp. 35-41].<br>
The present invention provides a method for selectively killing tumor cells expressing the antigen that specifically binds to the C35 monoclonal antibody  or  functional  equivalent.   This  method  comprises  reacting  the immunoconjugate (e.g. the immunotoxin) of the invention with said tumor cells. These tumor cells may be from a human carcinoma.<br>
Additionally, this invention provides a method of treating carcinomas (for example human carcinomas) in vivo. This method comprises administering to a subject a pharmaceutically effective amount of a composition containing at least one of the immunoconjugates (e.g. the immunotoxin) of the invention.<br>
In accordance with the practice of this invention, the subject may be a human, equine, porcine, bovine, murine, canine, feline, and avian subjects. Other warm blooded animals are also included in this invention.<br>
The present invention also provides a method for curing a subject suffering from a cancer. The subject may be a human, dog, cat, mouse, rat, rabbit, horse, goat, sheep, cow, chicken. The cancer may be identified as abreast, bladder, retinoblastoma, papillary cystadenocartinoma of the ovary, Wilm's tumor, or small cell lung carcinoma and is generally characterized as a group of cells having tumor associated antigens on the cell surface. This method comprises administering to the subj ect a cancer killing amount of a tumor targeted antibody joined to a cytotoxic agent. Generally, the joining of the tumor targeted antibody with the cytotoxic agent is made under conditions which permit the antibody so joined to bind its target on the cell surface. By binding its target, the tumor targeted antibody acts directly or indirectly to cause or contribute to the killing of the cells so bound thereby curing the subject.<br><br>
-199-<br>
Also provided is a method of inhibiting the proliferation of mammalian tumor cells which comprises contacting the mammalian tumor cells with a sufficient concentration of the immunoconjugate of the invention so as to inhibit proliferation of the mammalian tumor cells.<br>
The subject invention further provides methods for inhibiting the growth of human tumor cells, treating a tumor in a subject, and treating a proliferative type disease in a subject. These methods comprise administering to the subject an effective amount of the composition of the invention.<br>
It is apparent therefore that the present invention encompasses pharmaceutical compositions, combinations and methods for treating human carcinomas. For example., the invention includes pharmaceutical compositions for use in the treatment of human carcinomas comprising a pharmaceutically effective amount of a C35 antibody and a pharmaceutically acceptable carrier.<br>
The compositions may contain the C35 antibody or antibody fragments, either unmodified, conjugated to a therapeutic agent (e.g., drug, toxin, enzyme or second antibody) or in are combinant form (e.g., chimeric C35, fragments of chimeric C35, bispecific C35 or single-chain immunotoxin C35). The compositions may additionally include other antibodies or conjugates fortreating carcinomas (e.g., an antibody cocktail).<br>
The antibody, antibody conjugate and immunotoxin compositions of the invention can be administered using conventional modes of administration including, but not limited to, intravenous, intraperitoneal, oral, intralymphatic or administration directly into the tumor. Intravenous administration is preferred.<br>
The compositions of the invention may be in a variety of dosage forms which include, but are not limited to, liquid solutions or suspension, tablets, pills, powders, suppositories, polymeric microcapsules or microvesicles, liposomes, and injectable or infusible solutions. The preferred form depends upon the mode of administration and the therapeutic application.<br>
The compositions of the invention also preferably include conventional pharmaceutically acceptable carriers and adjuvants known in the art such as<br><br>
-200-<br>
human semm albumin, ion exchangers, alumina, lecithin, buffer substances such as phosphates, glycine, sorbic acid, potassium sorbate, and salts or electrolytes such as protamine sulfate.<br>
The most effective mode of administration and dosage regimen for the compositions of this invention depends upon the severity and course of the disease, the patient's health and response to treatment and the judgment of the treating physician. Accordingly, the dosages of the compositions should be titrated to the individual patient. Nevertheless, an effective dose of the compositions of this invention may be in the range of from about 1 to about 2000 mg/kg.<br>
The molecules described herein may be in a variety of dosage forms which include, but are not limited to, liquid solutions or suspensions, tablets, pills, powders, suppositories, polymeric microcapsules or microvesicles, liposomes, and injectable or infusible solutions. The preferred form depends upon the mode of administration and the therapeutic application.<br>
The most effective mode of administration and dosage regimen for the molecules of the present invention depends upon the location of the tumor being treated, the severity and course of the cancer, the subject's health and response to treatment and the judgment of the treating physician. Accordingly, the dosages of the molecules should be titrated to the individual subject.<br>
The interrelationship of dosages for animals of various sizes and species and humans based on mg/kg of surface area is described by Freireich, E. J., et al. Cancer Chemother., Rep. 50 (4): 219-244 (1966). Adjustments in the dosage regimen may be made to optimize the tumor cell growth mhibiting and killing response, e.g., doses may be divided and administered on a daily basis or the dose reduced proportionally depending upon the situation (e.g., several divided doses may be administered daily or proportionally reduced depending on the specific therapeutic situation.<br>
It would be clear that the dose of the composition of the invention required to achieve cures may be further reduced with schedule optimization.<br><br>
-201-<br>
In accordance with the practice of the invention, the pharmaceutical carrier may be a lipid carrier. The lipid carrier may be a phospholipid. Further, the lipid carrier may be a fatty acid. Also, the lipid carrier may be a detergent. As used herein, a detergent is any substance that alters the surface tension of a liquid, generally lowering it.<br>
In one example of the invention, the detergent may be a nonionic detergent. Examples of nonionic detergents include, but are not limited to, polys orbate 80 (also known as Tween 80 or (poryoxyethylenesorbitan monooleate), Brij, and Triton (for example Triton WR-1339 and Triton A-20).<br>
Alternatively, the detergent may be an ionic detergent. An example of an ionic detergent includes, but is not limited to, alkyltrimethylarnmonium bromide.<br>
Additionally, in accordance with the invention, the lipid carrier may be a liposome. As used in this application, a "liposome" is any membrane bound vesicle which contains any molecules of the invention or combinations thereof.<br>
Vaccine Formulations<br>
The C35 epitopes can be produced in quantity by recombinant DNA methods and formulated with an adjuvant that promotes a cell-mediated immune response. The present invention encompasses the expression of the C35 polypeptides, or C35 epitopes (including cytotoxic or helper T cell eliciting epitopes), in either eucaryotic or procaryotic recombinant expression vectors^ and the formulation of the same as immunogenic and/or antigenic compositions. Such compositions are described in, for example, U.S. Patent Appl. No. 08/935,377, the entire contents of which are incorporated herein by reference. In accordance with the present invention, the recombinantly expressed C35 epitope may be expressed, purified and formulated as a subunit vaccine. Preferably, the DNA encoding the C35 epitopemay also be constructedinto viral vectors, preferably pox virus, adenovirus, herpesvirus, and alphavirus vectors, for use in vaccines. In this regard, either a live recombinant viral vaccine, an<br><br>
-202-<br>
inactivated recombinant viral vaccine, or a killed recombinant viral vaccine can be formulated.<br>
(i)        Expression of C35 in Procaryotic and Eucaryotic Expression Systems<br>
The present invention encompasses expression systems, both eucaryotic and procaryotic expression vectors, which may be used to express the C35 epitope. The C35 epitope may be expressed in both truncated or full-length forms, in particular for the formation of subunit vaccines.<br>
The present invention encompasses the expression of nucleotide sequences encoding the C35 polypeptide and immunologically equivalent fragments. Such immunologically equivalent fragments may be identified by making analogs of the nucleotide sequence encoding the identified epitopes that are truncated at the 5' and/or 3' ends of the sequence and/or have one or more internal deletions, expressing the analog nucleotide sequences, and determining whether the resulting fragments immunologically are recognized by the epitope-specific T lymphocytes andinduce a cell-mediated immune response, or epitope-specific B lymphocytes for inductions of a humoral imrnune response.<br>
The invention encompasses the DNA expression vectors that contain any of the foregoing coding sequences operatively associated with a regulatory element that directs expression of the coding sequences and genetically engineered host cells that contain any of the foregoing coding sequences operatively associated with aregulatory element that directs the expression of the coding sequences in the host cell. As used herein, regulatory elements include but are not limited to, inducible and non-inducible promoters, enhancers, operators and other elements known to those skilled in the art that drive and regulate expression.<br>
The C35 epitope gene products or peptide fragments thereof, may be produced by recombinant DNA technology using techniques well known in the<br><br>
-203-<br>
art. Thus, methods for preparing the C35 epitope gene polypeptides and peptides of the invention by expressing nucleic acid containing epitope gene sequences are described herein. Methods which are well known to those skilled in the art can be used to construct expression vectors containing epitope gene product coding sequences and appropriate transcriptional and translational control signals. These methods include, for example, in vitro recombinant DNA techniques, synthetic techniques, and in vivo genetic recombination. See, for example, the techniques described in Sambrook et al., Molecular Cloning: A Laboratory Manual, 2ndEd., (1989), Cold Spring Harbor Laboratory Press, and Ausubel et al., 1989, supra. Alternatively, RNA capable of encoding glycoprotein epitope gene product sequences may be chemically synthesized using, for example, synthesizers. See, for example, the techniques described in "Oligonucleotide Synthesis", 1984, Gait, M.J. ed., IRL Press, Oxford, which is incorporated by reference herein in its entirety.<br>
The invention also encompasses nucleotide sequences that encode peptide fragments of the C35 epitope gene products. For example, polypeptides or peptides corresponding to the extracellular domain of the C35 epitope may be useful as "soluble" protein which would facilitate secretion, particularly useful in the production of subunit vaccines. The C35 epitope gene product or peptide fragments thereof, can be linked to a heterologous epitope that is recognized by a commercially available antibody is also included in the invention. A durable fusion protein may also be engineered; i.e., a fusion protein which has a cleavage site located between the C35 epitope sequence and the heterologous protein sequence, so that the selected C35 can be cleaved away from the heterologous moiety. For example, a collagenase cleavage recognition consensus sequence may be engineered between the C35 epitope protein or peptide and the heterologous peptide or protein. The epitopic domain can be released from mis fusion protein by treatment with collagenase. hi apreferred embodiment of the invention, a fusion protein of glutathione-S-transferase and the C35 epitope protein may be engineered.<br><br>
-204-<br>
The C35 epitope proteins of the present invention for use in vaccine preparations, in particular subunit vaccine preparations, are substantially pure or homogeneous. The protein is considered substantially pure or homogeneous when at least 60 to 75% of the sample exhibits a single polypeptide sequence. A substantially pure protein will preferably comprise 60 to 90% of a protein sample, more preferably about 95% and most preferably 99%. Methods which are well known to those skilled in the art can be used to determine protein purity or homogeneity, such as polyacrylamide gel electrophoresis of a sample, followed by visualizing a single polypeptide band on a staining gel. Higher resolution may be determined using HPLC or other similar methods well known in the art.<br>
The present invention encompasses C35 polypeptides which are typically purified from host cells expressing recombinant nucleotide sequences encoding these proteins. Such protein purification can be accomplished by a variety of methods well known in the art. In a preferred embodiment, the C35 epitope protein of the present invention is expressed as a fusion protein with glutathiones-transferase. The resulting recombinant fusion proteins purified by affinity chromatography and the epitope protein domain is cleaved away from the heterologous moiety resulting in a substantially pure protein sample. Other methods known to those skilled in the art may be used; see for example, the techniques described in "Methods In Enzymology", 1990, Academic Press, Inc., San Diego, "Protein Purification: Principles and Practice", 1982, Springer-Verlag, New York, which are incorporated by reference herein in their entirety.<br>
(ii)      Eucaryotic and Procaryotic Expression Vectors<br>
The present invention encompasses expression systems, both eucaryotic and procaryotic expression vectors, which may be used to express the C35 epitope. A variety of host-expression vector systems may be utilized to express the C35 epitope gene of the invention. Such host-expression systems represent<br><br>
-205-<br>
vehicles by which the C35 coding sequence may be produced and subsequently purified, but also represent cells which may, when transformed or transfected with the C35 nucleotide coding sequences, exhibit the C35 epitope gene product of the invention in situ.  These include but are not limited to microorganisms such as bacteria (e.g., E. coli, B. subtilis) transformed with recombinant bacteriophage DNA, plasmid DNA or cosmid DNA expression vectors containing the C35  epitope gene product coding sequence;  yeast (e.g., Saccharomyces, Pichia) transformed with recombinant yeast expression vectors containing the C35 epitope gene product coding sequence; insect cell systems infected with recombinant virus expression vectors (e.g., baculovirus) containing the C35 epitope gene product coding sequence; plant cell systems infected with recombinant virus expression vectors (e.g., cauliflower mosaic virus, CaMV; tobacco mosaic virus, TMV) or transformed with recombinant plasmid expression vectors (e.g., Ti plasmid) containing C35 epitope gene product coding sequence; or mammalian cell systems (e.g., COS, CHO, BHK, 293, 3T3) harboring recombinant expression constructs containing promoters derived from the genome of mammalian cells (e.g., metallothionein promoter) or from mammalian viruses (e.g., the adenovirus late promoter; the vaccinia virus 7.5K promoter).<br>
(iii)     Host Cells<br>
The present invention encompasses the expression of the C35 epitope in animal andinsect cell lines. In apreferred embodiment of the present invention, the C35 epitope is expressed in a baculovirus vector in an insect cell line to produce an unglycosylated antigen. In another preferred embodiment of the invention, the C35 epitope is expressed in a stably transfected mammalian host cell, e.g., CHO cell line to produce a glycosylated antigen. The C35 epitopes which are expressed recombinantly by these cell lines may be formulated as subunit vaccines.  The present invention is further directed to host cells that<br><br>
-206-<br>
overexpress the C35 gene product. The cell may be a host cell transiently or stable transected or transformed with any suitable vector which includes a polynucleotide sequence encoding the C35 polypeptide or a fragment thereof and suitable promoter and enhancer sequences to direct overexpression of the C35 gene product. However, the overexpressing cell may also be a product of an insertion, for example via homologous recombination, of a heterologous promoter or enhancer which will direct overexpression of the endogenous C35 gene. The term "overexpression" refers to a level of expression which is higher than a basal level of expression typically characterizing a given cell under otherwise identical conditions.<br>
A host cell strain may be chosen which modulates the expression of the inserted sequences, or modifies and processes the C35 gene product in the specific fashion desired. Such modifications (e.g., glycosylation) andprocessing (e.g, cleavage) of protein products may be important for the function of the protein. Different host cells.have characteristic and specific mechanisms for the post-translational processing and modification of proteins and gene products. Appropriate cell lines or host systems can be chosen to ensure the correct modification of the foreign protein expressed. To this end, eucaryotic host cells which possess the cellular machinery for proper processing of the primary transcript, glycosylation, phosphorylation, and prenylation of the C35 gene product may be used. Such mammalian host cells include but are not limited to CHO, VBRO, BHK, HeLa, COS, MDCK, 293, 3T3 and W138 cell lines.<br>
For long term, high-yield production of recombinant proteins, stable expression is preferred. For example, cell lines which stably express the C35 target epitope may be engineered. Rather than using expression vectors which contain viral origins of replication, host cells can be transformed with DNA controlled by appropriate expression control elements (e.g., promoter, enhancer, sequences, transcription terminators, polyadenylation sites, etc.), and aselectable marker. Following the introduction of the foreign DNA, engineered cells may be allowed to grow for 1-2 days in an enriched media, and then are switched to<br><br>
-207-<br>
a selective media. The selectable marker in the recombinant plasmid confers resistance to the selection and allows cells to stably integrate the plasmid into their chromosomes and grow to form foci which in turn can be cloned and expanded into cell lines. This method may advantageously be used to engineer cell lines. This method may advantageously be used to engineer cell lines which express the C35 epitope gene products. Such cell lines would be particularly useful in screening and evaluation of compounds that affect the endogenous activity of the C35 epitope gene product.<br>
A number of selection systems may be used, including but not limited to the herpes simplex virus thymidine kinase (Wigler, et al., 1977, Cell 11:223), hypoxanthme-guanine phosphoribosyltransferase(Szybalska&amp; Szybalski, 1962, Proc. Natl Acad. Sci. USA 48:2026), and adenine phosphoribosyltransferase (Lowy, et al., 1980, Cell 22:817) genes can be employed in tk; hgprt or aprt cells, respectively. Also, antimetabolite resistance can be used as the basis of selection for the following genes: dhfr, which confers resistance to methotrexate (Wigler, et al., 1980, Proc. Natl. Acad. Sci. USA 77:3567; O'Hare, et al., 1981, Proc. Natl. Acad. Sci. USA 78:1527); gpt, which confers resistance to mycophenolic acid (Mulligan &amp; Berg, 1981, Proc. Natl Acad. Sci. USA 78:2072); neo, which confers resistance to the aminoglycoside G-418 (Colberre-Garapin, et al., 1981, J. Mol Biol 150: V); and hygro, which confers resistance to hygromycin (Santerre, et al., 1984, Gene 30:147).<br>
Alternatively, any fusion protein may be readily purified by utilizing an antibody specific for the fusion protein being expressed. For example, a system described by Janknecht et al allows for the ready purification of non-denatured fusion proteins expressed in human cell lines (Janknecht, et al., 1991, Proc. Natl. Acad. Sci. USA 88:8972-8976). In this system, the gene of interest is subcloned into a vaccinia recombination plasmid such that the gene's open reading frame is transkrionally fused to an ammo-terminal tag consisting of six histidine residues. Extracts from cells infected with recombinant vaccinia virus are loaded<br><br>
-208-<br>
onto Ni2+-nitriloacetic acid-agarose columns and histidine-tagged proteins are selectively eluted with imidazole-containing buffers.<br>
(iv)     Expression of C35 Epitope in Recombinant Viral Vaccines<br>
In another embodiment of the present invention, either a live recombinant viral vaccine or an inactivated recombinant viral vaccine expressing the C35 epitope can be engineered. A live vaccine may be preferred because multiplication in the host leads to a prolonged stimulus of similar kind and magnitude to that occurring in natural infections, and therefore, confers substantial, long-lasting immunity. Production of such live recombinant virus vaccine formulations may be accomplished using conventional methods involving propagation of the virus in cell culture or in the allantois of the chick embryo followed by purification.<br>
In this regard, a variety of viruses may be genetically engineered to express the C35 epitope. For vaccine purposes, it may be required that the recombinant viruses display attenuation characteristics. Current live virus vaccine candidates for use in humans are either cold adapted, temperature sensitive, or attenuated. The introduction of appropriate mutations (e.g., deletions) into the templates used for transfection may provide the novel viruses with attenuation characteristics. For example, specific multiple missense mutations that are associated with temperature sensitivity or cold adaptation can be made into deletion mutations and/or multiple mutations can be introduced into individual viral genes. These mutants should be more stable than the cold or temperature sensitive mutants containing single point mutations and reversion frequencies should be extremely low. Alternatively, recombinant viruses with "suicide" characteristics may be constructed. Such viruses go through only one or a few rounds of replication in the host.<br>
For purposes of the invention, any virus may be used in accordance with the present invention which:  (a) displays an attenuated phenotype or may be<br><br>
-209-<br>
engineered to display attenuated characteristics; (b) displays a tropism for mammals, in particular humans, or may be engineered to display such a tropism; and (c) may be engineered to express the C35 epitope of the present invention.<br>
Vaccinia viral vectors may be used in accordance with the present invention, as large fragments of DNA are easily cloned into its genome and recombinant attenuated vaccinia variants have been described (Meyer, et ah, 1991, J. Gen. Virol. 72:1031-1038).   Orthomyxoviruses, including influenza; NO:1 Paramyxoviruses, including respiratory syncytial virus and Sendai virus; and Rhabdoviruses may be engineered to express mutations which result in attenuated phenotypes (see U.S. Patent Serial No. 5,578,473, issued November 26, 1996). These viral genomes may also be engineered to express foreign nucleotide sequences, such as the C35 epitopes of the present invention (see U.S. Patent Serial No. 5,166,057, issued November 24, 1992, incorporated herein by reference in its entirety).    Reverse genetic techniques can be applied to manipulate negative and positive strand RNA viral genomes to introduce mutations which result in attenuated phenotypes, as demonstrated in influenza virus, Herpes Simplex virus, cytomegalovirus and Epstein-Barr virus, Sindbis virus and poliovirus (see Palese et al., 1996, Proc. Natl Acad. Sci. USA 93:11354-11358). These techniques may also be utilized to introduce foreign DNA, i.e., the C35 epitopes, to create recombinant viral vectors to be used as vaccines in accordance with the presentinvention. See, for instance, U.S. Patent Appl. No. 08/935,377, the entire contents of which are incorporated herein by reference.    In addition, attenuated adenoviruses and retroviruses may be engineered to express the C35 epitope. Therefore, a wide variety of viruses may be engineered to design the vaccines of the present invention, however, by way of example, and not by limitation, recombinant attenuated vaccinia vectors expressing the C35 epitope for use as vaccines are described herein.<br>
In one embodiment, a recombinant modified vaccinia variant, Modified Virus Ankara (MVA) is used in a vaccine formulation. This modified virus has been passaged for 500 cycles in avian cells and is unable to undergo a full<br><br>
-210-<br>
infectious cycle in mammalian cells (Meyer, et al., 1991, J. Gen. Virol. 72:1031-1038). When used as a vaccine, the recombinant virus goes through a single replication cycle and induces a sufficient level of immune response but does not go further in the human host and cause disease: Recombinant viruses lacking one or more of essential vaccinia virus genes are not able to undergo successive rounds of replication. Such defective viruses can be produced by co-transfecting vaccinia vectors lacking a specific gene(s) required for viral replication into cell lines which permanently express this gene(s). Viruses lacking an essential gene(s) will be replicated in these cell lines but when administered to the human host will not be able to complete a round of replication. Such preparations may transcribe and translate — in this abortive cycle — a sufficient number of genes to induce an immune response.<br>
Alternatively, larger quantities of the strains can be administered, so that these preparations serve as inactivated (killed) virus, vaccines. For inactivated vaccines, it is preferred that the heterologous C35 gene product be expressed as a viral component, so that the C35 gene product is associated with the virion. The advantage of such preparations is that they contain native proteins and do not undergo inactivation by treatment with formalin or other agents used in the manufacturing of killed virus vaccines'.<br>
In another embodiment of the invention, inactivated vaccine formulations are prepared using conventional techniques to "kill" the recombinant viruses. Inactivated vaccines are "dead" in the sense that their infectivity has been destroyed Ideally, the infectivity of the virus is destroyed without affecting immunogenicity. In orderto prepare inactivated vaccines, the recombinant virus may be grown in cell culture or in the allantois of the chick embryo, purified by zonal ultracentrifugation, inactivated by formaldehyde or p-propiolactone, and pooled. The resulting vaccine is usually inoculated intramuscularly.<br>
Inactivated viruses may be formulated with a suitable adjuvant in order to enhance the immunological response. Such adjuvants may include but are not limited to mineral gels, e.g., aluminumhydroxide, surface active substances such<br><br>
-211-<br>
as lysolecithin, pluronic polyols, polyanions; peptides; oligonucleotides, oil emulsions; and potentially useful human adjuvants such as BCG and Corynebacterium parvum.<br>
(v)       Methods of Treatment and/or Vaccination<br>
Since the C35 epitopes of the present invention can be produced in large amounts, the antigen thus produced and purified has use in vaccine preparations. The C35 epitope may be formulated into a subunit vaccine preparation, or may be engineered into viral vectors and formulated into vaccine preparations. Alternatively, the DNA encoding the C35 epitope may be administered directly as a vaccine formulation. The "naked" plasmid DNA once administered to a subject invades cells, is expressed, processed into peptide fragments, some of which can be presented in association with MHC molecules on the surface of the invaded cell, and elicits a cellular immune response so that T lymphocytes "will attack cells displaying the C35 epitope. The C35 epitope also has utility in diagnostics, e.g., to detect or measure in a sample of body fluid from a subject the presence of tumors that express C35 or the presence of antibodies or T cells that have been induced by C35 expressing tumor andthus to diagnose cancer and tumors and/or to monitor the cellular immune response of the subject subsequent to vaccination.<br>
The recombinant viruses of the invention can be used to treat tumor-bearingmammals, including humans, to generate an immuneresponse against the<br>
tumor cells. The generation of an adequate and appropriate immune response leads to tumor regression in vivo. Such "vaccines" can be used either alone or in combination with other therapeutic regimens, including but not limited to chemotherapy, radiation therapy, surgery, bone marrow transplantation, etc. for the treatment of tumors. For example, surgical or radiation techniques could be used to debulk the tumor mass, after which, the vaccine formulations of the invention can be administered to ensure the regression and prevent the<br><br>
-212-<br>
progression of remaining tumor masses or micrometastases in the body. Alternatively, administration of the "vaccine" can precede such surgical, radiation or chemotherapeutic treatment<br>
Alternatively, the recombinant viruses of the invention can be used to immunize or "vaccinate" tumor-free subjects to prevent tumor formation. With the advent of genetic testing, it is now possible to predict a subject's predisposition for certain cancers. Such subjects, therefore, may be immunized using a recombinant vaccinia virus expressing the C35 antigen.<br>
The immunopotency of the C35 epitope vaccine formulations can be determined by monitoring the immune response in test animals following immunization or by use of any immunoassay known in the art. Generation of a cell-mediated and/or humoral immune response may be taken as an indication of an immune response. Test animals may include mice, hamsters, dogs, cats, monkeys, rabbits, chimpanzees, etc., and eventually human subjects.<br>
Suitable preparations of such vaccines include injectables, either as liquid solutions or suspensions; solid forms suitable for solution in, suspension in, liquid prior to injection, may also be prepared. The preparation may also be emulsified, or the polypeptides encapsulated in liposomes. The active immunogenic ingredients are often mixed with excipients which are pharmaceutically acceptable and compatible with the active ingredient. Suitable excipients are, for example, water, saline, dextrose, glycerol, ethanol, or the like and combinations thereof. In addition, if desired, the vaccine preparation may also include minor amounts of auxiliary substances such as wetting or emulsifying agents, pH buffering agents, and/or adjuvants which enhance the effectiveness of the vaccine.<br>
Examples of adjuvants which may be effective, include, but are not limited to: aluminum hydroxide, N-acetyl-muramyl-L-threonyl-D-isoglutamine (thr-MDP), N-acetyl-nor-min-amyl-L-alanyl-D-isoglutamme, N-acetylmuramyl-L-alanyl-D-isoglutaminyl-L-alanine-2-(l'-2'-dipalrnitoyl-sn-glycero-3-hydroxyphosphoryloxy)-ethylarnine, GM-CSF, QS-21 (investigational drug,<br><br>
-213-<br>
Progenics    Pharmaceuticals,Inc.),   DETOX   (investigational    drug,   Ribi Pharmaceuticals), BCG, and CpG rich oligonucleotides.<br>
The effectiveness of an adjuvant may be determined by measuring the induction of the cellular immune response directed against the C35 epitome.<br>
The vaccines of the invention may be multivalent or univalent. Multivalent vaccines are made from recombinant viruses that direct the expression of more than one antigen. Multival ent vaccines compris ed of multiple T cell epitomes, both cytotoxic and helper, are preferred.<br>
The composition, if desired, can also contain minor amounts of wetting or emulsifying agents, or pH buffering agents. The composition can be a liquid solution, suspension, emulsion, tablet, pill, capsule, sustained release formulation, or powder. Oral formulation can include standard carriers such as pharmaceutical grades of mannitol, lactose, starch, magnesium stearate, sodium saccharine, cellulose, magnesium carbonate, etc.<br>
Generally, the ingredients are supplied either separately or mixed together in unit dosage form, for example, as a dry lyophilized powder or water free concentrate in a hermetically sealed container such as an ampoule or sachette indicating the quantity of active agent. Where the composition is administered by injection, an ampoule of sterile diluent can be provided so that the ingredients may be mixed prior to administration.<br>
In a specific embodiment, a lyophilized C35 epitope of the invention is provided in a first container; a second container comprises diluent consisting of an aqueous solution of 50% glycerin, 0.25% phenol, and an antiseptic (e.g., 0.005% brilliant green).<br>
Use of purified C35 antigens as vaccine preparations can be carried out by standard methods. For example, the purified C35 epitopes shouldbe adjusted to an appropriate concentration, formulated with any suitable vaccine adjuvant and packaged for use. Suitable adjuvants may include, but are not limited to: mineral gels, e.g., aluminum hydroxide; surface active substances such as lysoledthin, pluronic polyols; polyanions; peptides; oil emulsions; alum, and<br><br>
-214-<br>
MDP. The immunogen may also be incorporated into liposomes, or conjugated to polysaccharides and/or other polymers for use in a vaccine formulation. In instances where the recombinant antigen is a hapten, i.e., a molecule that is antigenic in that it can react selectively with cognate antibodies, but not immunogenic in that it cannot elicit an immune response, the hapten may be covalently bound to a carrier or immunogenic molecule; for instance, a large protein such as serum albumin will confer immunogenicity to the hapten coupled to it. The hapten-carrier may be formulated for use as a vaccine.<br>
Many methods may be used to introduce the vaccine formulations described above into a patient. These include, but are not limited to, oral, intradermal, intramuscular, intraperitoneal, intravenous, subcutaneous, intranasal, transdermal, epidural, pulmonary, gastric, intestinal, rectal, vaginal, or urethral routes. When the method of treatment uses a live recombinant vaccinia vaccine formulation of the invention, it may be preferable to introduce the formulation via the natural route of infection of the vaccinia virus, i.e., through a mucosal membrane or surface, such as an oral, nasal, gastric, intestinal, rectal, vaginal or urethral route, or through the skin. To induce a CTL response, the mucosal route of administration may be through an oral or nasal membrane. Alternatively, an intramuscular or intraperitoneal route of administration may be used. Preferably, a dose of 106 - 107 PFU (plaque forming units) of cold adapted recombinant vaccinia virus is given to a human patient.<br>
The precise dose of vaccine preparation to be employed in the formulation will also depend on the route of administration, and the nature of the patient, and should be decided according to the judgment of the practitioner and each patient's circumstances according to standard clinical techniques. An effective immunizing amount is that amount sufficient to produce an immune response to the antigen in the host to which the vaccine preparation is administered.<br>
Where subsequent or booster doses are required, a modified vaccinia virus such as MVA can be selected as the parental virus used to generate the<br><br>
-215-<br>
recombinant. Alternatively, another virus, e.g., adenovirus, canary pox virus, or a subunit preparation can be used to boost. Immunization and/or cancer immunotherapy may be accomplished using a combined immunization regimen, e.g., immunization with a recombinant vaccinia viral vaccine of the invention and a boost of a recombinant adenoviral vaccine. In such an embodiment, a strong secondary CD8+ T cell response is induced after priming and boosting with different viruses expressing the same epitope (for such methods of immunization and boosting, see, e.g., Murata et al., Cellular Immunol. 173:96-107). For example, a patient is first primed with a vaccine formulation of the invention comprising a recombinant vaccinia virus expressing an epitope, e.g., a selected tumor-associated antigen or fragment thereof. The patient is then boosted, e.g., 21 days later, with a vaccine formulation comprising arecombinant virus other than vaccinia expressing the same epitope. Such priming followed by boosting induces a strong secondary T cell response. Such a priming and boosting immunization regimen is preferably used to treat a patient with a tumor, metastasis or neoplastic growth expressingthe tumor associate, e.g., C35, antigen<br>
In yet another embodiment, the recombinant vaccinia viruses can be used<br>
as a booster immunization subsequent to a primary immunization with<br>
inactivated tumor cells, a subunit vaccine containing the C35 antigen or its<br>
epitope, or another recombinant viral vaccine, e.g., adenovirus, canary poxvirus,<br>
or MVA.<br>
In an alternate embodiment, recombinant vaccinia virus encoding C35 epitopes or fragment thereof may be used in adoptive irnmunotherapeutic methods for the activation of T lymphocytes that are histocompatible with the patient and specific for the C35 antigen (for methods of adoptive immunotherapy, see, e.g., Rosenberg, U.S. Patent No. 4,690,915, issued September 1, 1987; Zarling, et al., U.S. PatentNo. 5,081,029, issued January 14, 1992). Such T lymphocytes may be isolated from the patient or a histocompatible donor. The T lymphocytes are activated in vitro by exposure to the recombinant vaccinia virus of the invention. Activated T lymphocytes are<br><br>
-216-<br>
expanded and inoculated into the patient in order to transfer T cell immunity directed against the C35 antigen epitome.<br>
The invention also provides a pharmaceutical pack or kit comprising one or more containers comprising one or more of the ingredients of the vaccine formulations of the invention. Associated with such container(s) can be a notice in the form prescribed by a governmental agency regulating the manufacture, use or sale of pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human administration.<br>
Cancer Diagnosis and Prognosis<br>
There are two classes of genes affecting tumor development. Genes influencing the cancer phenotype that act directly as a result of changes ( e.g., mutation) at the DNA level, such as BRCA1, BRCA2, and p53, are one class of genes. Another class of genes affect the phenotype by modulation at the expression level. Development of breast cancer and subsequent malignant progression is associated with alterations of a variety of genes of both classes. Identification of quantitative changes in gene expression that occur in the malignant mammary gland, if sufficiently characterized, may yield novel molecular markers which may be useful in the diagnosis and treatment of human breast cancer.<br>
The present inventors have identified a new breast cancer marker, C35, that is differentially expressed in primary infiltrating intraductal mammary carcinoma cells. Low expression levels of C35 in normal mammary epithelial cells suggest that overexpression of C35 indicates breast cancer malignant progression. It is possible that C35 may also be overexpressed in tumors of certain other tissue types including bladder and lung.<br>
The present inventors have demonstrated that certain tissues in mammals with cancer express significantly enhanced levels of the C35 protein and mRNA encoding the C35 protein when compared to a corresponding "standard"<br><br>
-217-<br>
mammal, i.e., a mammal of the same species not having the cancer. Further, it is believed that enhanced levels of the C35 protein, or of antibodies or lymphocytes specific for the C35 protein, can be detected in certain body fluids (e.g., sera, plasma, urine, and spinal fluid) from mammals with cancer when compared to sera from mammals of the same species not having the cancer.<br>
Thus, the present invention provides a diagnostic method useful fortumor diagnosis, which involves assaying the expression level of the gene encoding the C35 protein in mammalian cells or body fluid and comparing the gene expression level with a standard C35 gene expression level, whereby an increase in the gene expression level over the standard is indicative of certain tumors. Alternatively, the expression levels of antibodies or lymphocytes specific for C35 protein or C35 polypeptides can be determined in blood or other body fluids and compared with a standard of expression of C35-specific antibodies or lymphocytes.<br>
Where a tumor diagnosis has already been made according to conventional methods, the present invention is useful as a prognostic indicator, whereby patients exhibiting enhanced C35 gene expression may experience a worse clinical outcome relative to patients expressing the gene at a lower level.<br>
By "assaying the expression level of the gene encoding the C35 protein" is intended qualitatively or quantitatively measuring or estimating the level of the C35 protein or the level of the mRNA encoding the C35 protein in a first biological sample either directly (e.g., by determining or estimating absolute protein level or mKNA level) or relatively (e. g., by comparing to the C35 protein level or mKNA level in a second biological sample).<br>
Preferably, the C35 protein level or mKNA level in the first biological sample is measured or estimated and compared to a standard C35 protein level or mKNA level, the standard being taken from a second biological sample obtained from an individual not having the cancer. As will be appreciated in the art, once a standard C35 protein level or mRNA level is known, it can be used repeatedly as a standard for comparison.<br><br>
-218-<br>
By "biological sample" is intended any biological sample obtained from an individual, cell line, tissue culture, or other source which contains C35protein or mRNA. Biological samples include mammalian body fluids (such as sera, plasma, urine, synovial fluid and spinal fluid) which contain secreted mature C35 protein, and ovarian, prostate, heart, placenta, pancreas, liver, spleen, lung, breast, bladder and umbilical tissue which may contain precursor or mature forms of C35.<br>
The present invention is useful for detecting cancer in mammals. In particular, the invention is useful during diagnosis of the following types of cancers in mammals: breast, bladder, ovarian, prostate, bone, liver, lung, pancreatic, and splenic. Preferred mammals include monkeys, apes, cats, dogs, cows, pigs, horses, rabbits and humans. Particularly preferred are humans.<br>
Total cellular RNA can be isolated from a biological sample using the single-step guanidinium-thiocyanate-phenol-chloroform method described in Chomczynski and Sacchi, Anal. Biochem. 162:156-159 (1987). Levels of mRNA encoding the C35 protein are then assayed using any appropriate method. These include Northern blot analysis (Harada et al., Cell 63:303-312 (1990)), S1 nuclease mapping (Fujita et al., Cell 49:357-367 (1987)), the polymerase chain reaction (PCR), reverse transcription in combination with the polymerase chain reaction (RT-PCR) (Makino et al., Technique 2:295-301 (1990)), and reverse transcription in combination with the Iigase chain reaction (RT-LCR).<br>
Assaying C35 protein levels in biological sample can occur using antibody-based techniques. For example, C35 protein expression in tissues can be studied with classical immunohistological methods (Jalkanen, M., et al., J. Cell Biol. 101:976-985 (1985); Jalkanen, M., et al., J. CellBiol 105:3087-3096 (1987)).<br>
Other antibody-based methods useful for detecting C35 protein expression include immunoassays, such as enzyme linked immunosorbent assay (ELISA), ELISPOT, and the radioimmunoassay (RIA).<br><br>
-219-<br>
Suitable labels are known in the art and include enzyme labels, such as, Glucose oxidase, and radioisotopes, such as iodine (125I, 121I), carbon (14C), sulfur (35S), tritium (3H), indium (112In), and technetium (99mTc), and fluorescent labels, such as fluorescein and rhodamine, and biotin.<br>
C35-specific T ceils may be detected in a variety of proliferation and lymphokine secretion assays following activation by C35 presented by antigen presenting cells according to methods known in the art. Tetrameric complexes of a C35 peptide epitope bound to soluble MHC molecules can be employed to directly stain and enumerate C35-specific T cells in a population of cells (Lee, P.P. et al., Nature Medicine 5:677-85 (1999) the entire contents of which is hereby incorporated by reference.<br>
In addition to assaying secreted protein levels in a biological sample, proteins can also be detected in vivo by imaging. Antibody labels or markers for in vivo imaging of protein include those detectable by X-radiograpny, NMR or ESR. For X-radiography, suitable labels include radioisotopes such as barium or cesium, which emit detectable radiation but are not overtly harmful to the subject. Suitable markers for NMR and ESR include those with a detectable characteristic spin, such as deuterium, which may be incorporated into the antibody by labeling of nutrients for the relevant hybridoma.<br>
A protein-specific antibody or antibody fragment which has been labeled with an appropriate detectable imaging moiety, such as a radioisotope (for example, 131I, 112In, 99mTc), a radio-opaque substance, or a material detectable by nuclear magnetic resonance, is introduced (for example, parenterally, subcutaneously, or intraperitoneally) into the mammal. It will be understood in the art that the size of the subject and the imaging system used will determine the quantity of imaging moiety needed to produce diagnostic images. In the case of a radioisotope moiety, for a human subject, the quantity of radioactivity injected will normally range from about 5 to 20 millicuries of 99mTc. The labeled antibody or antibody fragment will then preferentially accumulate at the location of cells which contain the specific protein. In vivo tumor imaging is described<br><br>
-220-<br>
in S.W. Burchiel et al., "Immunopharmacokinetics of Radiolabeled Antibodies and Their Fragments." (Chapter 13 in Tumor Imaging: The Radiochemical Detection of Cancer, S.W. Burchiel and B. A. Rhodes, eds., Mass on Publishing Inc. (1982).)<br>
Fusion Proteins<br>
Any C35 polypeptide can be used to generate fusion proteins. For example, the C35 polypeptide, when fused to a second protein, can be used as an antigenic tag. Antibodies raised against the C35 polypeptide can be used to indirectly detect the second protein by binding to the C35. Moreover, because secreted proteins target cellular locations based on trafficking signals, the C35 polypeptides can be used as a targeting molecule once fused to other proteins.<br>
Examples of domains that can be fused to C35 polypeptides include not only heterologous signal sequences, but also other heterologous functional regions. The fusion does not necessarily need to be direct, but may occur through linker sequences.<br>
In certain preferred embodiments, C35 fusion polypeptides may be constructed which include additional N-terrninal and/or C-tenninal amino acid residues. In particular, any N-terminally or C-terrninally deleted C35 polypeptide disclosedhereinmay be altered by inclusion of additional amino acid residues at me N-terrninus to produce a C35 fusion polypeptide. In addition, C35 fusionpolypeptides are contemplatedwhich include additional N-terrninal and/or C-terminal amino acid residues fused to a C35 polypeptide comprising any combination of N- and C-termmal deletions set forth above.<br>
Moreover, fusion proteins may also be engineered to improve characteristics of the C35 polypeptide. For instance, aregion of additional amino acids, particularly charged amino acids, may be added to the N-terminus of the C35 polypeptide to improve stability andpersistence during purification from the host cell or subsequent handling and storage. Also, peptide moieties may be<br><br>
-221-<br>
added to the C35 polypeptide to facilitate purification. Such regions may be removed prior to final preparation of the C35 polypeptide. The addition of peptide moieties to facilitate handling of polypeptides are familiar and routine techniques in the art.<br>
Moreover, C35 polypeptides, including fragments, and specifically epitopes, can be combined with parts of the constant domain of immunoglobulins (IgG), resulting in chimeric polypeptides. These fusion proteins facilitate purification and show an increased half-life in vivo. One reported example describes chimeric proteins consisting of the first two domains of the human CD4-polypeptide and various domains of the constant regions of the heavy or light chains of mammalian immunoglobulins. (EPA 394,827; Traunecker et al., Nature 337:84-86 (1988).) Fusion proteins having disulfide-linked dimeric structures (due to the IgG) can also be more efficient in binding and neutralizing other molecules, than the monomelic secreted protein or protein fragment alone. (Fountoulakis et al., J. Biochem. 270:3958-3964 (1995).)<br>
Similarly, EP-A-O 464 533 (Canadian counterpart 2045869) discloses fusion proteins comprising various portions of constant region of immunoglobulin molecules together with another human protein or part thereof. In many cases, the Fc part in a fusion protein is beneficial in therapy and diagnosis, and thus can result in, for example, improved pharmacokinetic properties. (EP-A 0232 262.) Alternatively, deleting the Fc part after the fusion protein has been expressed, detected, and purified, would be desired. For example, the Fc portion may binder therapy and diagnosis if the fusion protein is used as an antigen for irnmunizations. In drug discovery, for example, human proteins, such as hIL-5, have been fused with Fc portions for the purpose of high-throughput screening assays to identify antagonists of hIL-5. (See, D. Bennett et al., J. Molecular Recognition 5:52-58 (1995); K. Johanson et al., J. Biol Chem. 270:9459-9471 (1995).)<br>
Moreover, the C35 polypeptides can be fused to marker sequences, such as apeptide which facilitates purification of C35. In preferred embodiments, the<br><br>
-222-<br>
marker amino acid sequence is a hexa-histidine peptide, such as the tag provided in a pQE vector (QIAGEN, Inc., 9259 Eton Avenue, Chatsworth, CA, 91311), among others, many of which are commercially available. As described in Gentz et al., Proc. Natl Acad. Set USA 5(5:821-824 (1989), for instance, hexa-histidine provides for convenient purification of the fusion protein. Another peptide tag useful for purification, the "HA" tag, corresponds to an epitope derived from the influenza hemagglutinin protein. (Wilson et al., Cell 37:767(1984).)<br>
Thus, any of these above fusions can be engineered using the C35 polynucleotides or the C35 polypeptides.<br>
Vectors, Host Cells, and Protein Production<br>
The present invention also relates to vectors containing the C35 polynucleotide, host cells, and the production of C35 polypeptides by recombinant techniques. The vector may be, for example, a phage, plasmid, viral, or retroviral vector. Retroviral vectors may be replication competent or replication defective. In the latter case, viral propagation generally will occur only in complementing host cells.<br>
C35 polynucleotides may be joined to a vector containing a selectable marker for propagation in a host. Generally, a plasmid vector is introduced in a precipitate, such as a calcium phosphate precipitate, or in a complex with a charged lipid. If the vector is a virus, it may be packaged in vitro using an appropriate packaging cell line and then transduced into host cells.<br>
The C35 polynucleotide insert should be operatively linked to an appropriate promoter, such as the phage lambda PL promoter, the E. coli lac, trp, phoA and tac promoters, the SV40 early and late promoters and promoters of retroviral LTRs, to name a few. Other suitable promoters will be known to the skilled artisan. The expression constructs will further contain sites for transcription initiation, termination, and, in the transcribed region, a ribosome binding site for translation. The coding portion of the transcripts expressed by<br><br>
-223-<br>
the constructs will preferably include a translation initiating codon at the beginning and a termination codon (UAA, UGA or XJAG) appropriately positioned at the end of the polypeptide to be translated.<br>
. As indicated, the expression vectors will preferably include at least one selectable marker. Such markers include dihydrofolate reductase, G418 or neomycin resistance for eukaryotic cell culture and tetracycline, kanamycin or ampicillin resistance genes for culturing in E. coli and other bacteria. Representative examples of appropriate hosts include, but are not limited to, bacterial cells, such as E. coli, Streptomyces and Salmonella typhimurium cells; fungal cells, such as yeast cells; insect cells such as Drosopbila S2 and Spodoptera Sf9 cells; animal cells such as CHO, COS, 293, and Bowes melanoma cells; andplant cells. Appropriate culture mediums and conditions for the above-described host cells are known in the art.<br>
Among vectors preferred for use in bacteria include pHE-4 (and variants thereof); pQE70, pQE60 and pQE-9, available from QIAGEN, Inc.; pBluescript vectors, Phagescript vectors, pNH8A, pNH16a, pNH18A, pNH46A, available from Stratagene Cloning Systems, Inc.; and ptrc99a, pKK223-3, pKK233-3, pDR540, pRIT5 available from Pharmacia Biotech, Inc. Among preferred eukaryotic vectors are pWLNEO, pSV2CAT, pOG445 pXT1 and pSG available from Stratagene; and pSVK3, pBPV, pMSG and pSVL available from Pharmacia. Other suitable vectors will be readily apparent to the skilled artisan. Preferred vectors are poxvirus vectors, particularly vaccinia virus vectors such as those described in U.S. Patent Appl. No. 08/935,377, the entire contents of which are incorporated herein by reference.<br>
Introduction of the construct into the host cell can be effected by calcium phosphate transfection, DEAE-dextran mediated transfection, cationic lipid-mediated transfection, electroporation, transduction, infection, or other methods. Suchmethods are describedinmany standard laboratory manuals, such as Davis et al., Basic Methods In Molecular Biology (1986).<br><br>
-224-<br>
C35 polypeptides can be recovered and purified from recombinant cell cultures by well-known methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Most preferably, high performance liquid chromatography ("HPLC") is employed for purification.<br>
C35 polypeptides can also be recovered from: products purified from natural sources, including bodily fluids, tissues and cells, whether directly isolated or cultured; products of chemical synthetic procedures; and products produced by recombinant techniques from a prokaryotic or eukaryotic host, including, for example, bacterial, yeast, higher plant, insect, and mammalian cells. Depending upon the host employed in a recombinant production procedure, the C35 polypeptides may be glycosylated or may be non-glycosylated. In addition, C35 polypeptides may also include an initial modified methionine residue, in some cases as a result of host-mediated processes. Thus, it is well known in the art that the N-terminal methionine encoded by the translation initiation codon generally is removed with high efficiency from any protein after translation in all eukaryotic cells. While the N-terrninal methionine on most proteins also is efficiently removed in most prokaryotes, for some proteins, this prokaryotic removal process is inefficient, depending on the nature of the amino acid to which the N-terminal methionine is covalently linked.<br>
In addition to encompassing host cells containing the vector constructs discussed herein, the invention also encompasses primary, secondary, and immortalized host cells of vertebrate origin, particularly mammalian origin, that have been engineered to delete or replace endogenous genetic material (e.g. C35 coding sequence), and/or to include genetic material (e.g., heterologous polynucleotide sequences) that is operably associated with C35 polynucleotides of the invention, and which activates, alters, and/or amplifies endogenous C35<br><br>
-225-<br>
polynucleotides. For example, techniques known in the art may be used to operably associateheterologous control regions (e.g., promoter and/or enhancer) and endogenous C35 polynucleotide sequences via homologous recombination (see, e.g., U.S. Patent No. 5,641,670, issued June 24, 1997; International Publication No. WO 96/29411, published September 26, 1996; International Publication No. WO 94/12650, published August 4,1994; Koller et al., Proc. Natl. Acad. Sci USA 86:8932-8935 (1989); and Zijlstra et al., Nature 342:435-438 (1989), the disclosures of each of which are incorporated by reference in their entireties).<br>
Having generally described the invention, the same will be more readily understood by reference to the following examples, which are provided by way of illustration and are not intended as limiting.<br>
EXAMPLES<br>
EXAMPLE 1 Differential Expression of C35 in Human Breast Carcinoma<br>
The present inventors have characterized a full-length cDNA representing a gene, C35, that is differentially expressed in human breast and bladder cancer (Fig. 1). A 348 base pair DNA fragment of C35 was initially isolated by subtractive hybridization of poly-A RNA from tumor and normal mammary epithelial cell lines derived from the same patient with primary infiltrating intraductal mammary carcinoma. (Band, V. et al., Cancer Res. 50:7351-7357 (1990). Employing primers based on this sequence and that of an overlapping EST sequence (Accession No. W57569), a cDNA that includes the full-length C35 coding sequence was then amplified and cloned from the SKBR3 breast tumor cell line (ATCC, HTB-19). This C35 cDNA includes, in addition to the 348 bp coding sequence, 167 bp of 3' untranslated region.<br><br>
-226-<br>
Differential expression of the C35 sequence is demonstrated in Fig. 2 panel A which compares expression levels of clone CSS in poly-A RNA from cell lines derived from normal mammary epithelium, from two primary breast tumor nodules,  and from two metastatic lung tumor nodules  isolated approximately one year later from the same patient (Band, V. et al., Cancer Res. 50:7351-7357 (1990)).   Quantitative analysis indicates that the sequence is expressed at a more than 10 fold higher level in tumor cells than in normal mammary epithelium. Low expression levels in a panel of other normal tissues is demonstrated by the Northern hybridization results of Fig. 2 panel B. Even though three times as much poly-A RNA was loaded from normal tissues as from the tumor cell lines, little or no expression of RNA homologous to C35 was detected after a comparable 15 hour exposure. Only after an extended 96 hour exposure was low level expression of some homologous sequences detected in normal spleen and kidney tissues. Analysis of expression of C35 homologous sequences in poly-A RNA from three    primary infiltrating ductal breast carcinoma from different patients as w ell as a sample of normal breast epithelium is shown in Fig. 2 panel C.   In comparison to normal breast epithelium, sequences homologous to C35 are overexpressed as much as 45 and 25 fold in two of the three primary breast tumors.<br>
The present inventors previously conducted an analysis of an immunoprotective tumor antigen expressed in several independently derived murine tumors and, at much reduced levels, in normal mouse tissues. (See U.S. Patent Application filed March 28,2000, titled "Methods of Producing a Library and Methods of Directly Selecting Cells Expressing Inserts of Interest," fhe entire contents of which are hereby incorporated herein by reference). In this case, a factor of 9 difference between expression levels in tumor and normal tissues was associated with induction of an immunoprotective tumor-specific response. As discussed above, the expression level of C35 in some human breast cancers relative to normal tissue exceeds a factor of 9, suggesting that C35 might also be immunoprotective against breast cancer in these individuals.<br><br>
-227-<br>
EXAMPLE 2 C35 Specific CTL are Cytolytic for C35 Positive Breast Tumor Cells<br>
Although a gene product may be overexpressed in tumor cells, as is the case for C35, it is immunologically relevant only if peptides derived from that gene product can be processed andpresented in association with MHC molecules of the tumor cells. It is conceivable that for any given gene product either no peptides are produced during the cellular degradation process that satisfy the requirements for binding to the MHC molecules expressed by that tumor, or, even if such peptides are generated, that defects in transport or competition for MHC molecules by other tumor peptides would preclude presentation of any peptides from that specific gene product. Even if relevant tumor peptides are processed and presented in association with human MHC in the tumor cells, it must in all cases be determined whether human T cells reactive to these peptides are well-represented in the repertoire or whether T cells may have been rendered tolerant, perhaps due to expression of the same or a related antigen in some other non-homologous normal tissue. For both these reasons, therefore, it is essential to confirm that MHC-restricted, human tumor antigen-specific T cells can be induced by C35 and that they are indeed crossreactive on human tumor cells. Relevant information on this point can be obtained through in vitro stimulation of human T cell responses with recombinant C35 or C35 peptides presented by autologous antigen presenting cells.<br>
A major technical problem in evaluating T cell responses to recombinant gene products is that a strong immune response against the expression vector can block or obscure the recombinant specific response. This is particularly a problem with primary responses that may require multiple cycles of in vitro stimulation. To minimize vector specific responses, it is possible to alternate stimulation by antigen presenting cells infected with different viral vectors recombinant for the same gene product. Convenient vectors include: retroviruses, adenovirus, and pox viruses.<br><br>
-228-<br>
Human PBMC were purified using Ficoll-Paque and subject to resetting with neurarnioidase-treated sheep erythrocytes to isolate monocytes (erythrocyte rosette negative, ER-) and T lymphocytes (ER+). Dendritic cells were generated from the ER- fraction by culture for 7 days in rhGM-CSF (1000 U/ml) and rhIL-4 (1000 U/ml) with fresh medium and cytokines being added every other day.    At day 7, immature dendritic cells were transduced with retrovirus expressing human C35 in the presence of polybrene (1 ug/ml) for 6 hours. Cells were washed and incubated under maturation conditions for 4 days in the presence of 12.5% monocyte conditioned medium, 1000 U/ml rhGM-CSF and 1000 rhU/ml IL-4 and 1% autologous serum. At this point, the dendritic cells were incubated with autologous T lymphocytes (cryopreserved ER+ fraction) at a ratio of 1 DC:50 T cells for 14 days. Viable T cells were restimulated with autologous, irradiated EBV-B B cells infected at a multiplicity of infection of 1 overnight (16 hours) with a vaccinia recombinant expressing human C35 in the presence of cytokines IL-2 (20U/ml), IL-12 (20 U/ml) and IL-18 (10 ng/ml). Cells were restimulated two more times with autologous EBV-B cells infected with C35-bearing retrovirus in the presence of IL-2 and IL-7 (10 ng/ml). Cytotoxic activity was measured after a total of 4 stimulations by 51Cr release assay using 5000 targets/well in a 4 hour assay. The results shown in Table 8 below demonstrate specific cytotoxic activity of C35 stimulated T cells against 21NT breast tumor cells mat express relatively elevated levels of C35 but not against MDA-MB-231 tumor cells that express the same low levels of C35 as normal nontransformed epithelial cells.<br><br>
-229-<br><br>
EXAMPLE 3 C35 Expression on the Membrane of Breast Carcinoma Cells<br>
To determine whether the C35 polypeptide product is expressed on the surface of tumor cells, a C35 specific antiserum was prepared. BALB/c mice were immunized with syngeneic Line 1 mouse tumor cells that had been transduced with retrovirus encoding human C35. Mice were bled following a series of two or more immunizations. The immunesera were employed to detect surface expression of C35 protein by flow cytometry on three breast tumor cell lines representing high (2 INT), intermediate (SKBR3), and low (MDA-MB-231 levels of expressionof the C35 transcript in Northern blots (see Fig. 4), 1x105 breast tumor cells were stained with 3.5 microliters of C35 specific antiserum or control, pre-bleed BALB/c serum. After a 30 minute incubation, cells were washed twice with staining buffer (PAB) and incubated with FTTC-goat anti-mouse IgG (1 ug/sample) for 30 minutes, Samples were washed and analyzed on an EPICS Elite flow cytometer. The results presented in Figure 4 demonstrate membrane expression of the C3 5 antigen recognized by the specific immune serum at high levels on tumor line 21NT (panel A), intermediate levels<br><br>
-230-<br>
for tumor line SKBR3 (panel B), and undetectable levels in tumor line MDA-MB-231 (panel C). The high level ofreactivity of antibody to membranes of tumor cells that express elevated levels of C35 transcripts suggests that C35 specific antibodies may serve as effective immunotherapeutic agents for the large number of breast carcinoma that overexpress this gene product (see Figures 2 and 3).<br>
EXAMPLE 4<br>
A Deregulated Ribosomal Protein L3 Gene Encodes a Shared Murine Tumor<br>
Rejection Antigen<br>
The presentinventors have developednovel antigen discovery technology that allows for the selection of genes encoding CTL epitopes from a cDNA library constructed in a poxvirus. Using this technology the present inventors have determined that a shared murine tumor antigen is encoded by an alternate allele of the ribosomal protein L3 gene. The immunogenic L3 gene is expressed at significant albeit reduced levels in normal tissues including thymus. Immunization with a vaccinia recombinant of the immunogenic L3 cDNA induces protective immunity against tumor challenge. It is of particular interest that a deregulated allele of a housekeeping gene can serve as an immunoprotective antigen and that thymic expression does not preclude immunogenicity of an upregulated tumor product. These observations emphasize that tolerance to a self-protein is not absolute but must be defined in relation to quantitative levels of expression. The ribosomal protein described may be representative of a class of shared tumor antigens that arise as a result of deregulated expression of a self-protein without compromising immune tolerance to normal tissues. Such antigens would be suitable for immunotherapy of cancer in vital organs.<br><br>
-231-Methods<br>
Total RNA was isolated from BCA 39 tumor cells using the Perfect RNA Total RNA Isolation Kit (5 Prime 3 Prime, Inc., Boulder, CO). Poly A+ mRNA was isolated from the total RNA using Dynabeads (Dynal, Lake Success, NY). Two micrograms of poly A+ mRNA was converted to double stranded cDNA using the Great Lengths cDNA Synthesis Kit (Clontech, Palo Alto, CA). The double stranded cDNA was then inserted in vaccinia virus vector v7.5/tk.<br>
Balb/cByJ (Jackson Labs) mice were immunized intraperitoneally with 2 X 106 irradiated (6,500 cGy) BCA 34 cells. Two weeks later the mice were boosted by subcutaneous injection of 2X106 irradiated BCA 34 cells. Oneweek following the second immunization splenocytes were harvested, divided into 12 parts and cultured in 12 well plates with 6 X 105 irradiated (10,000 cGy), mitomycin C treated BCA 34 cells per well. At weekly intervals viable T cells were purified using Lympholyte-M (Accurate Chemical, Westbury, NY) and cultured in 12 well plates at 1.5 X 106 T cells per well. To each well was also added 4 X 106 irradiated (5000 cGy) Balb/c spleen, along with 6 X 105 irradiated, mitomycin C treated BCA 34 cells.<br>
A specific vaccinia recombinant that encodes the well characterized ovalbumin 257-264 peptide (SUNFEKL) that is immunodominant in association with H-2Kb was diluted with non-recombinant virus so that it initially constituted either 0.2%, 0.01%, or 0.001% of total viral pfu. An adherent monolayer of MC57Gcells (H-2b) were infected with this viral mix at m.o.i.=1 (approximately 5 x 105 cells/well). Following 12 hours infection, ovalbumin peptide-specific CTL, derived by repeated in vitro stimulation of ovalbumin primed splenic T cells with the immunodominant SENFEKL peptide, were added. During this incubation those adherent cells which were infected with a recombinant particle that expresses the ovalbumin peptide are targeted by specific cytotoxic T cells and undergo a lytic event which causes them to be released from the monolayer. Following incubation with CTL, the monolayer is gently washed, and both<br><br>
-232-<br>
floating ceils and the remaining adherent cells are separately harvested. Virus extracted from each cell population was titred for the frequency of recombinant (BRdU resistant) viral pfu. Virus extracted from floating cells was then used as input to another enrichment cycle with fresh adherent MC57G cells and ovalbumin peptide-specific CTL. It was observed that following enrichment of VVova to greater than 10% of total virus, further enrichment of the recombinant virus was accelerated if the m.o.i. in succeeding cycles was reduced from 1 to 0.1.<br>
Confluent monolayers of BCN in wells of a 12 well plate were infected with moi=l .0 vaccinia BCA39 cDNA library. At 12 hours post-infection the monolayers were washed 3X with media, and 2.5 X 106 CTL were added to the wells in a 250 ul volume. The T cells and targets were incubated at 37°C for 4 hours. Following the incubation the supernatant was harvested, and the monolayer gently washed 3X with 250µl media. Virus was released from the cells by freeze/thaw, and titers determined by plaque assay on BSCl cells. The selected virus population (floating cells in cultures that received specific T cells) was amplified on BSCl cells in one well of a 12 well plate for 2 days. The virus was then harvested and titered. This viral stock was subjected to three additional enrichment cycles. The selected virus population was not amplified prior to the next cycle.<br>
Virus from the fourth enrichment cycle was divided into 40 pools of 5 pfu each. Bach pool was amplified on BSCl cells in a 96 well plate, with 1 pool / well. After 4 days the virus was harvested (P1), and used to infect monolayers of BCN in a 96 well plate at moi=5, with 1 pool per well. As a control, a monolayer of BCN was infected with moi=5 vNotI/tk (Merschlinsky et ah, Virology 190:522 (1992)). At 5 hours post-infection, 2X104 washed CTL were added to each well. The final volume in each well was 225 µl The cells were incubated at 37°C for 18 hours. The cells were then pelleted by centrifugation, 150µl supernatant was harvested and tested for IFNg by ELISA. Twenty seven of the forty pools of 5 pfu were positive for the ability to stimulate CTL.<br><br>
-233-<br>
Suggesting, by Poisson analysis, that specific recombinants were enriched to greater than 20%. Individual clones were picked from 5 positive pools and assayed as above.<br>
Monolayers of B/C.N in a 6 well plate were infected with moi=10 of v7.5/tk, vF5.8, or vH2.16. At 14 hours post-infection cells were harvested along with the control targets: B/C.N, BCA 34, and BCA 39. The target cells were labeled with 100 microcuries 5lChrarmum (Dupont, Boston, MA) for 1 hour at 37°C, and 104 cells were added to wells of a 96 well round bottom plate in quadruplicate. Tumor specific CTL were added to target cells at the indicated ratios. Cells were incubated at 37°C for 4 hours. Supernaiants were harvested and 51Cr release determined. Spontaneous release was derived by incubating target cells with media alone. Maximal release was determined by incubating target cells with 5% Triton X 100. Percentage of specific lysis was calculated using the formula: % specific lysis= ((experimental release-spontaneous release) / (maximal release-spontaneous release)) X 100. In each case the mean of quadruplicate wells was used in the above formula.<br>
Two micrograms of total RNA was converted to cDNA using a dT primer and Superscript II Reverse Transcriptase (BRL, Gaithersburg, MD). cDN A was used as the template for a PCR using L3 specific primers; L3.F1.S (CGGCGAGATGTCTCACAGGA) and L3.F1.AS (ACCCCACCATCTGCACAAAG); and Klentaq DNA Polymerase Mix (Clontech) in a 20 microliter final volume. Reaction conditions included an initial denaturation step of 94°C for 3 minutes, followed by 30 cycles of: 94°C 30 seconds, 60°C for 30 seconds, 68°C for 2 minutes. These PCR products contained the region ofL3 between position 3 and 1252. The PCR products were purified using Centricon 100 columns (Amicon, Beverly, MA), digested with San3AI, and resolved on a 3% Agarose/ethidium bromide gel.<br>
Adult female Balb/cByJ mice (2 mice per group) were immunized by subcutaneous injection of 5X106 pfu of vH2.16, or v7.5/tk. Seven days following the immunization splenocytes were harvested and cultured in 12 well<br><br>
-234-<br>
plates along with 1 micromolar peptide L348_56(154). After seven days the viable T cells were purified using Lympholyte-M, and 1X106 T cells were added to wells of a 12 well plate along with 1 micromolar peptide and 4 X 106 irradiated (5000 cGy) Balb/c spleen cells per well.<br>
Adult female Balb/cBy J mice were immunized by subcutaneous injection of 10X106 pfu of vH2.16, vPKIa, v7.5/tk or Phosphate Buffered Saline. Secondary immunizations were given 21 days later. Mice were challenged with tumor by subcutaneous injection of 2X105 BCA 34 cells twenty one (primary immunization only) or fourteen days following immunization.<br>
Results and Discussion<br>
Prospects for development of broadly effective tumor vaccines have been advanced by evidence that several self-proteins can be recognized as tumor antigens by immune T cells (Van den Eynde et al., J. Exp. Med. 773:1373 (1991); M. B. Bloom et ah, J. Exp. Med. 755:453 (1997); Van Der Bruggen et<br>
al., Science 254:1643 (1991); Gaugler et al., J. Exp. Med. 179:921 (1994); Boel et al., Immunity 2:167 (1995); Van Den Eynde et al., J. Exp. Med. 752:689 (1995); Kawakami et al., Proc. Natl. Acad. Sci. U.S.A. 91:3515 (1994); Kawakami et al., Proc. Natl. Acad. Sci. U.SA. .97:6458 (1994); Brichard et al., J. Exp. Med. 775:489 (1993)). Such normal, nonmutated gene products may serve as common target antigens in tumors of certain types arising in different individuals. Clinical evidence for induction of protective immunity following vaccination with such shared tumor antigens is, currently, very limited (Marchand et al., Int. J. Cancer 80:219 (1999); Rosenberg et al., Nat. Med. 4:321 (1998); Overwijk et al., Proc. Natl. Acad. Set 96:29S2 (1999); Brandle et al., Eur. J. Immunol 25:4010 (1998)). It is, moreover, not at all clear whether the T cell responses to these self-proteins represent a surprising breakdown in immunological tolerance or are a consequence of qualitative or quantitative changes in the expression of the self-proteins in tumor cells. In the latter case,<br><br>
-235-<br>
normal tissue tolerance could be maintained and vaccine induced immunity to self-proteins whose expression is systematically altered in tumors might be applicable even to cancer of vital organs.<br>
The present inventors have shown that a ribosomal protein allele that is systematically deregulated in multiple murine tumors during the transformation process is a tumor rejection antigen and that the principal correlate of immunogenicity is a dramatic change in quantitative expression in tumors relative to normal tissues and thymus.<br>
Previously, the present inventors have reported that cross-protective immunity is induced among three independently derived murine tumor cell lines (Sahasrabudhe et al., J. Immunology 151:6302 (1993)). These tumors, BCA 22, BCA 34, and BCA 39 were derived by in vitro mutagenesis of independent subcultures of the B/C.N line, a cloned, immortalized, anchorage-dependent, contact inhibited, nontumorigenic fibroblast cell line derived from a Balb/c embryo (Conms et al., Nature299A69(1982); Lin et al., JNCI 74: 1025(1985)). Strikingly, immunization with any of these tumor cell lines, but not with B/C.N provided protection against challenge with not only homologous tumor cells, but also against challenge with the heterologous tumor cell lines. Following immunization with any of these three tumor cell lines, CD8+ cytolytic T lymphocyte (CTL) lines and clones could be generated which in vitro displayed crossreactive specificity for the same three tumors, but not for the nontumorigenic B/C.N cells from which they derived.<br>
In order to move from an immunological definition to a molecular definition of this shared tumor antigen(s), the present inventors developed a novel and efficient method for the identification of genes that encode CTL target epitopes. In this approach a cDNA library from the BCA 39 tumor cell line was constructed in a modified vaccinia virus expression vector (Merchlinsky et al., Virology 238:444 (1997); E. Smith et al., Manuscript in preparation). Five hundred thousand plaque forming units (pfu) of this library were used to infect a monolayer of antigen-negative B/C.N cells at a multiplicity of infection (moi)<br><br>
-236-<br>
of 1. Following 12 hours infection, BCA 34 tumor specific CTL were added to the target cell monolayer at an effector to target ratio that gives approximately 50% lysis in a standard 51Cr release assay.  CTL specific for the heterologous BCA 34 tumor cell line were used in order to facilitate the identification of antigen(s) which are shared between these two tumor cell lines. Since adherence is an energy dependent process, it was expected that cells that undergo a CTL mediated lytic event would come off of the monolayer and could be recovered in the supernatant.   By harvesting virus from floating cells following cell mediated lymphocytotoxicity (CML), it was possible to enrich for viral recombinants that had s ensitized the host cell to lysis. An essential feature of this procedure is that it lends itself to repetition. The virus harvested following one cycle of enrichment can be used as input for additional cycles of selection using fresh monolayers and fresh CTL until the desired level of enrichment has been achieved. In a model experiment with CTL specific for a known recombinant, it was possible to demonstrate that specific recombinants could be enriched from an initial dilution of 0.001% to approximately 20% in 6 cycles of selection (Table 9). At this level it is a simple matter to pick individual plaques for further characterization.<br><br><br>
-237-<br><br>
The poxvirus expression library was subjected to 4 cycles of selection with tumor-specific CTL. Individual plaques of the selected viral recombinants were expanded and used to infect separate cultures of B/C.N cells. These cells were assayed for the ability to stimulate specific CTL to secrete interferon gamma (IFN-gamma) (FIG. 5A), or for sensitization to lysis by the tumor-specific CTL (FIG. 5B). Ten viral clones were isolated, all of which conferred upon B/C.N the ability to stimulate a line of tumor-specific CTL to secrete IFNy. All 10 clones contained the same sized (1,300 bp) insert (Smith et al., unpublished data). Sequence analysis confirmed that clones F5.8 and H2.16 contained the same full-length cDNA. It appeared, therefore, that all ten clones were recombinant for the same cDNA. In all, 6 of 6 CTL lines that were generated by immunization with BCA 34 demonstrated specificity for this antigen.<br>
A search of GenBank revealed that this cDNA is highly homologous to the murine ribosomal protein L3 gene (Peckham et al., Genes and Development 3:2062 (1989)). Sequencing the entire H2.16 clone revealed only a single nucleotide substitution that coded for an amino acid change when compared to the published L3 gene sequence. This C170T substitution generates a Threonine to Isoleucine substitution at amino acid position 54. The F5.8 clone also contained this nucleotide substitution.<br><br>
-238-<br>
Since  CTL recognize  antigen  as  peptide presented by  a Major Histocompatibility Complex (MHC) molecule, it was of interest to identify the peptide epitope recognized by these class I MHC-restricted tumor-specific CD8+ T cells. It was considered likely that the altered amino acid (IIe 54) would be included in the peptide recognized by the CTL. This hypothesis was supported by the demonstration that a vaccinia virus clone recombinant for only the first 199 bp (63 amino acids) of H2.16 (vH2199) was able to sensitize B/C.N to lysis by tumor-specific CTL (Smith et al., unpublished data). A Computer screen of peptide-binding motifs suggested that there are two epitopes encoded within this region that could associate with high affinity to the class I MHC molecule Kd (FIG. 12) (Parker et al., J. Immunology 152:163 (1994))., These two peptides, L345_54 (154) and L348_56 (154) were synthesized and tested for the ability to sensitize B/C.N cells to lysis by tumor-specific CTL.  As shown in FIG. 7A, peptide L348.56 (154) sensitized B/C.N to lysis, while L345-54 (154), and the wild type L348-56 (T54) did not.   It was determined that 10 nM L348.36 (154) was sufficient to sensitize targets to lysis by CTL, whereas 100 mM L348.56 (T54) did not (FIG. 7B). These results demonstrate that peptide L348.56 (154) is a target epitope recognized by the-tumor-specific CTL.<br>
To analyze expression of the different L3 gene products, oligo-dT primed cDNA was synthesized from RNA of tumors and me B/C.N cell line from which they derived. The first strand cDNA was subjected to PCR amplification using a pair of primers which amplify nearly the entire mouse L3 mRNA. Sequence analysis of these PCR products showed that B/C.N and BCB13 L3 cDNA contained a C at position 170 (same as published sequence). BCB13 is a tumor cell line that was derived from the B/C.N cell line, but that is not immunologically cross-protective with the BCA tumor cell lines (Sahasrabudhe et al., J. Immunology 151:6302 (1993)). Sequence analysis of the PCR products from the crossreactive BCA 39, BCA 34, and BCA 22 tumors suggested that these cell lines express two different species of L3 mRNA. One species contains<br><br>
-239-<br>
a C at 170, and the other contains a T at 170, as in the H2.16 clone.   The sequence of all L3 cDNAs were identical except for this one base substitution.<br>
There are two possible ways to account for the origin of the new L3 RNA in tumor cells. Either the L3 (C170T) gene expressed in these tumors is a somatic mutant of the wild type gene or there are multiple germ line alleles of L3, at least one of which gives rise to an immunogenic product when deregulated during the process of tumor transformation. We considered the first hypothesis unlikely because the crossreactive BCA 39, BCA 34, and BCA 22 tumors were independently derived. It would be remarkable if the same mutant epitope was generated in all three tumors. On the other hand, Southern blots of different restriction digests of genomic DNA from BCA 3 9 and B/C.N suggested that there are multiple copies of the L3 gene in the mouse genome (Smith et at, unpublished data). The L3 gene has also been reported to be multi-allelic in both the rat and the cow (Kuwano et ah, Biochemical and Biophysical Research Communications 757:58 (1992); Simonic et al., Biochemica et Biophysica Acta 1219:706 (1994)). Further analysis was required to test the hypothesis that different L3 alleles in the germ line are subject to differential regulation in tumors and normal cells.<br>
The nucleotide sequence of the published L3 from position 168 to 171 is GACC. Thesequenceof H2.16inthissameregionisGATC(FIG. 8A). This new palindrome is the recognition sequence for a number of restriction endonucleases, including Sau3AI. As shown in the restriction map of FIG. 8A, a Sau3A I digest of L3 is expected to generate fragments of 200,355,348,289, and 84 base pairs, while a Sau 3A I digest of H2.16 would generate a 168bp fragment in place of the 200 bp fragment. This difference in the Sau 3AI digestion products was used to confirm that the three BCA cell lines express at least two different L3 alleles. The L3 RT-PCR products from all 5 cell lines and thymus RNA were digested with Sau 3AI and analyzed on an agarose gel. As shown in FIG. 8B all 3 BCA lines express both versions of L3. Remarkably, when this assay was repeated using greater amounts of starting material, the 168<br><br>
-240-<br>
bp fragment was also detectable in the digests of B/C.N, BCB13 and normal thymus cDNA (Smith et al., unpublished data). To enhance the sensitivity of this assay, the PCR was repeated using a P32 end-labeled 5' L3 specific primer. The radiolabeled PCR products were digested with Sau3AI and resolved on an agarose gel. As shown in FIG. 8C, B/C.N, BCB13 and thymus contain the 168bp fragment.    Quantitative analysis indicates that the ratio of 200bp: 168bp fragments in the BCA tumors is 2:1 while the ratio of the same fragments detected in B/C.N, BCB13, and thymus is approximately 20:1. Low levels of expression of this immunogenic L3 allele was also observed when RNA from kidney, heart, and skeletal muscle was analyzed (Smith et al., unpublished data). These results suggest that gene deregulation associated with the transformation process in the crossreactive tumors leads to the expression of higher levels of this germ line L3 (C170T) allele, and that this altered L3 gene was not generated by somatic mutation of the L3 gene that is predominantly expressed in normal tissues.   The present inventors have termed this new L3 allele (C170T), the immunogenic L3 allele (iL3).<br>
It is particularly intriguing that the immunogenic L3 allele is also expressed, albeit at a 10 fold reduced level, in normal thymus. This level of expression is evidently not sufficient to tolerize all T cells with functional avidity for the level of deregulated iL3 expressed in some tumors. The observation that although B/C.N and BCB 13 express low levels of iL3, they are not susceptible to lysis by the tumor specific CTL suggests, however, that higher affinity T cells have been tolerized. This appears to be the firstinstance in which atumor antigen has been reported to be expressed in the thymus. These observations emphasize that tolerance to a self-protein is not absolute but must be defined in relation to quantitative levels of expression (Targoni et al., J. Exp. Med. 187:2055 (1998); C. J. Harrington et al., Immunity 5:571 (1998)).<br>
If broadly effective vaccines are to be developed based on expression of shared tumor antigens, then it is critical to demonstrate that such antigens can be immunoprotective. The largest number of shared antigens have been identified<br><br>
-241-<br>
for human tumors, but clinical Immunotherapy trials employing these antigens have so far been inconclusive, in part because of uncertainty regarding optimal vaccination strategies (Pardoll, D.M., Nat. Med. 4:525 (1998)). In mice, where immunotherapeutic strategies could be more thoroughly investigated, very few shared tumor antigens have been identified. It was, therefore, of considerable interest to determine whether immunization with iL3 recombinant vaccinia virus would induce tumor specific CTL and protect mice from tumor challenge (Overwijk et al., Proc. Natl Acad. Sci 96.29S2 (1999); Moss, B., Science 252:1662(1991); Irvine et al., J. Immunology 154:4651 (1995); McCabe et al., Cancer Research 55:1741 (1995); Estin et al., Proc. Natl. Acad. Sci 85:1052 (1988); J. Kantor et al., JNCI 84:1084 (1992); V. Bronte et al., Proc. Natl Acad. Sci. 94:3183 (1997)).    Immunization of Balb/c mice with vaccinia virus recombinant for the iL3 gene (H2.16) generated CTL that were able to lyse both BCA 34 and BCA 39 tumor cells, but not B/C.N in vitro (FIG. 9A).   Mice immunized twice or even once with vaccinia virus recombinant for iL3 were able to reject challenge with BCA 34 tumor cells (FIGS. 9B and 9C).    Mice immunized with empty viral vector, or control vaccinia recombinant for the Inhibitor Protein of cAMP-dependent Protein Kinase (PKIa) were unable to reject this tumor challenge (Olsen, S.R. and Uhler, M.D., J. Biol. Chem. 266:1115% (1991); Mueller et al., Manuscript in Preparation).   These results demonstrate that the iL3 self-protein is an immunoprotective tumor antigen.<br>
The present inventors have developed a new strategy to identify genes that encode CTL epitopes based on CTL-mediated selection from atumor cDNA library in a modified vaccinia virus vector (Merchlinsky et al., Virology 238:444 (1997); E. Smith et al., manuscript in preparation). We have applied this strategy to identify a deregulated housekeeping gene that encodes a tumor rejection antigen shared by three independently derived murine tumors. This ribosomal protein may berepresentative of alarger class of immunoprotective sharedtumor antigens that become immunogenic as aresult of deregulated expression of self-<br><br>
-242-<br>
proteins without compromising immune tolerance to normal tissues.   Such antigens would be well suited for immunotherapy of cancer in vital organs.<br>
EXAMPLE 5 Expression and Immunogenicity of C35 Tumor Antigen<br>
RNA transcripts of the novel C35 tumor gene arc overexpressed in 70% (12/17) of primary human breast carcinomas examined and 50% (5/10) of bladder carcinomas examined when compared to expression in normal human tissues. The full-length gene encodes a novel 115 amino acid protein of unknown function. A monoclonal antibody, 2C3, has been selected that stains the surface membrane of cells expressing C35 by flow cytometric analysis. In addition, human cytotoxic T lymphocytes (CTL) have been generated in vitro that specifically lyse C35+ breast and bladder tumors. The ability to generate C35-specific CTL in vitro from normal human donors suggests the absence of tolerance to the overexpressed protein. Overexpression of C35 in tumors of different individuals and the ability to induce humoral and cellular immune responses make C35 a promising candidate for immunotherapy.<br>
Material and Methods<br>
Cell lines: Humanmammary carcinoma cell lines BT20, BT474, MCF7, MDA-MB231, SKBR3, T47D (suppliedby ATC) were grown in RPMI-1640 (BioWhitaker, Walkersville, MD) supplemented with 10% fetal bovine serum (Biofluids, Rockville, MD). An immortalized line derived from normal breast epithelium, H16N2, two metastastic tumors, 21-MT1 and 21-MT2, and two primary tumors, 21-NT and 21-PT all derived from the same patient, and grown in DFCI medium (Band, V. andSager,R., "Tumor ProgressioninBreastCancer" in Neoplastic Transformation in Human Cell Culture, J.S. Rhim and A. Dritschilo eds., The Human Press Inc., Totowa, NJ.(1991), pp. 169-78) were<br><br>
-243-<br>
generously provided by Dr. Vimla Band, New England-Tufts Medical Center. The bladder tumor cell line ppT11A3 was derived from the immortalized nontumorigenic cell line SV-HUC. These bladder cell lines were generously provided by Dr. Catherine Reznikoff, University of Wisconsin Clinical Cancer Center, and grown in F12 medium supplemented with 1% FBS, 0.025 units insulin, 1 ug hydrocortisone, 5 ug transferrin, 2.7 g dextrose, 0.1 uM nonessential amino acids, 2 mM L-glutamine, 100 units penicillin, and 100 ug streptomycin per 500 ml. Normal proliferating breast epithelial cells (MEC) were purchased from Clonetics (BioWhittaker) and maintained according to the supplier's directions.<br>
RNA extraction and Northern Blot Analysis: Cell lines were harvested for . RNA extraction at approximately 80% confluency. Cells were harvested and lysed in QG buffer from Qiagen RNAeasy kit. Total RNA was extracted as per manufacturer's protocol and stored at -80°C as precipitates with GITC and alcohol. Tissue samples were provided by the Cooperative Human Tissue Network as snap frozen samples, which were homogenized in lysis buffer for use in the RNAeasy protocol. For Northern blots, mRNA was extracted from total RNA (30 ug total RNA/well) using DynaTs (Lake Success, NY) oligo-dT25 magnetic beads and electrophoresed in 0.8% SeaKemLE (FMC Bioproducts) with 3% formaldehyde. The mRNA was blotted onto Genescreen Plus (NEN) in 10X SSC overnight by capillary blot, then baked for 2 hours at 80°C. Membranes were probed with random-primed 32P-labeled cDNA probes (Prime-It, Stratagene, LaJolla, CA) at 106 cpm/ml Quickhyb solution (Stratagene), at 68°C as per manufacturer's protocol. Blots were exposed to Xray film and/or phosphorimager screens overnight. Expression on all blots was normalized to a housekeeping gene, such as GAPDH or beta actin.<br>
Subtractive Hybridization: PCRSelectcDNASubtraction Kit(Clontech, Palo Alto, CA), based on Representational Difference Analysis as first described by Lisitsyn et al.(Lisitsyn, N. and Wigler, N.M., Science 259:946-51 (1993)), was employed as per manufacturer's protocol to generate cDNAs enriched for<br><br>
-244-<br>
genes overexpressed in tumor compared to normal breast cell lines.  Briefly, oligo-dT-primed double stranded cDNA was synthesized from 2 ug high quality, DNase-treated mRNA from tumor and normal cells. Adaptors were ligated to short blunt-end (Rsal digested) tumor sequences and hybridized with excess Rsal digested normal fragments. Following 32 hour hybridization, suppression PCR (Clontech) allowed preferential amplification of overexpressed tumor sequences using adaptor sequences as primers.   The products of the PCR amplification were cloned into pT7Blue3 (Novagen, Madison, WI) to generate a subtracted library. Clones were grown in LB/ampicillin (100ug/ml) in 96-well format, inserts were PCR amplified from the overnight cultures and PCR products were spotted on Genescreen Plus using BioDot manifold (BioRad, Hercules, CA). Duplicate dot blots were then probed with random-primed tumor or normal cDNA, or, alternatively, the PCR products of the forward and reverse subtractive hybridizations.   Clones that appeared to be overexpressed in the tumor cDNA and forward subtraction (tumor minus normal) were analyzed by Northern Blot (as described above) to confirm differential gene expression.<br>
cDNA library and full length gene:  Oligo-dT primed double stranded<br>
cDNA was generated from SKBR3 cell line using SMART cDNA Synthesis<br>
(Clontech  Laboratories),   followed by phenol: chloroform: isoamyl  alcohol<br>
extraction.	Primers    were    synthesized    (C35    sense:	5'-<br>
GCGATGACGGGGGAGCC,     and     C35     antisense:      5'-CCACGGAATCTTCTATTCTTTCT; Fisher Oligos, The Woodlands, TX) to amplify the coding region of C35, based on the open reading frame deduced from EST homologies, Accession # W57569, in particular. PCR products were cloned into pT7 Blue 3 (Novagen).<br>
Vaccinia and Retroviral CSS recombinants: The coding sequence of C35 was subcloned from the library into vaccinia transfer plasmid, pVTKO at BamHI/Sall sites in a defined orientation. Recombinant virus was generated by transfectian of pVTK0.C35 along with NotI and Apal digested V7.5/TK viral DNA into fowlpox virus infected BSC-1 cells. As described elsewhere (U.S.<br><br>
-245-<br>
Utility Patent Application No. 08/935,377; PCT/US98/24029; T Cells Specific for Target Antigens and Vaccines Based Thereon), this is an efficient method for construction of vaccinia virus recombinants. The C35 gene was also cloned into a retroviral vector pLXSN, and viral stocks were generated by co-transfection of 293-GP cells with pVSVg for pseudotyping. Supernatants including infectious virus were collected 48 hours later.<br>
Generation of C35-specific 2C3 monoclonal antibody and FACS analysis: Linel mouse small cell lung carcinoma cells were infected with C35-retrovirus, and 103 - 2 x 104 cells were injected into three BALB/cByJ mice. Following 21 days, serum was harvested from retro-orbital bleeds and checked for reactivity with human tumor cells known to express low (MDA-MB-231) or very high (21NT) levels of C35mRNA. Spleens were also harvested for the production of hybridomas by the fusion of spleen cells with P3 myeloma cells using standard mouse hybridoma technology. ELISA was used to screen HAT resistant clones for the presence of Ig. High producers were isotyped, quantitated, and used to screen C35+ and C35- cell lines by flow cytometry. Hybridoma clone supernatants containing 1 ug IgG were incubated with 106 cells inPAB (PBS, 1% BSA, 0.1% azide) for 30 min on ice, followed by 3 washes with PAB, and incubation with goat anti-mouse IgG conjugated to FTTC (Southern Biotechnology, Birmingham, AL) for 30 minutes on ice. One hybridoma clone, 2C3, recapitulated the surface staining seen with the immune serum (Figure 14) and was selected for expansion and antibody purification (BioBxpress, West Lebanon, NY).<br>
Generation of human C35-specific T cell line. Peripheral blood derived from a healthy female donor (HLA A2, 11, B35, 44) was separated into erythrocyte-rosette positive fraction (a source of total T lymphocytes) and negative fraction (a source of monocytes). The T lymphocytes were cryopreserved for later use while me monocyte were incubated under conditions to generate dendritic cells (DC). Maturation of DCs was induced as described by Bhardwaj and colleagues (Bender, A. et al., J. Immunol. Meth. 196:121-35<br><br>
-246-<br>
(1996); Reddy, A. et al., Blood 90:3640-46 (1991); Engelmayer, J. et ah, J. Immunology 163:6762-68 (1999))with some modifications. hGM-CSF and hIL-4 (l000U/ml) were added every other day. At day 7, non-adherent, immature DC were incubated with a retrovirus recombinant for C35 for 6 hours in the presence of GM-CSF and IL-4. At this point, the retroviral supernatant was washed out and immature dendritic cells were subjected to maturation conditions, which again included GM-CSF, IL-4 as well as 12.5% monocyte conditioned medium (MCM). After 4 days, these mature, C35-expressing DC were used to stimulate autologous T cells at a ratio of 1 DC:50 T cells for a period of 14 days. A fresh pool of autologous DC were generated for restimulation of the T cells, but this time they were infected after 48 hours of maturation in MCM with a vaccinia virus recombinant for C35. Cytokines IL-2 (20 U/ml), IL2 (20 U/ml) and IL-18 (10 ng/ml) were added and a 1:50 ratio of DC.T cells was maintained. Following 12 days culture, T cells were stimulated for 7 additional days with EBV-Bcells infected withC35recombinant retrovirus and with addition of IL-2 (20 U/ml) and IL-7 (10 ng/ml). Cytokines were all purchased from R&amp;D Systems (Minneapolis, MN). Atthis point, the cells were &gt;90% CD8+ and were tested for activity in a standard 51Cr Release assay. Briefly, one million target cells were incubated with 100 uCi 51Cr, washed, then incubated with CTL effectors for 4 hours in RPMI-1640, supplemented with 10% human AB serum (BioWhittaker). Activity of the CTL is expressed as the percent of specific lysis, measured as (51Cr released into the supernatant upon lysis of labeled targets by CTL - spontaneous release)/(maximal release - spontaneous release).<br>
Results<br>
Characterization of C35: The sequence of clone C35, differentially expressed in human breast tumor cells, is not homologous to any known gene in Genbank, but homologous BSTsequences (prototype Accession# W57569) were identified Homologous human EST fragments arepresentinNCICGAP (Cancer<br><br>
-247-<br>
Genome Anatomy Project) libraries, including tumors of brain, lung and kidney (A# AA954696), Soares ovary (A# AA285089) and parathyroid tumors (A# W37432), an endometrial tumor (A#AA337071), and colon carcinoma (A# AA313422). An open reading frame was identified that encodes a 115 amino acid protein (Figure 10A).  The full-length gene was isolated from a cDNA library of the breast adenocarcinoma cell line SKBR3. Sequencing of full-length transcripts from the cell lines SKBR3, 21MT2-D, and H16N2 confirmed that there were no point mutations in the cDNA; the transcript is 100% homologous in C35hi cell lines, as well as C35lo cell lines. The C35 gene aligns on human chromosome   17q12  (A# AC040933)  and mouse  chromosome  11   (A# AC064803). Exons were deduced from homologies with cDNA EST sequences, as well as GRAIL predictions. Interestingly, the gene for C35 is within 1000 base pairs of the Her2/neu oncogene and within 2000 bp of the gene for Growth Factor Receptor-Bound Protein 7 (GRB7), a tyrosine kinase that is involved in activating the cell cycle and that is overexpressed in esophageal carcinomas (Tanaka, S. et al., J. Clin. Invest. .702:821-27 (1998)) (Figure 10B). Her2/neu protein overexpression has been correlated with, gene amplification in 30% breast tumors and is associated with poor clinical prognosis (Slamon, D.J. et al., Science 235:177-82(1987)).<br>
Predicted protein motifs in the C35 amino acid sequence include: casein kinase II phosphorylation sites at amino acids 38 to 41 (TYLE), 76 to 79 (SKLE), and 97 to 100 (SNGE); an N-myristoylation site at amino acids 60 to 65 (GGTGAF); and a cAMP- and cGMP-dependent protein kinase phosphorylation site at amino acids 94 to 97 (RRAS). Finally, the C35 protein contains a prenylation motif at the COOH-terminus, amino acids 112 to 115 (CVTL). Prenylation, the covalent attachment of a hydrophobic isoprenoid moiety, is a post-translational modification that promotes membrane association and also appears to mediate protein-protein interactions (Fu, H.-W. and Casey, P. J., Recent Progress in Hormone Research 54:315-43 (1999)). Prenylation has been shown to be required for localization and transforming potential of the<br><br>
-248-<br>
oncogenic Ras family proteins to the cell surface (Jackson, J.H. et al., Proc. Natl Acad. Sci. U.S.A. 87:3042-46 (1990); Hancock, J.F. et al., Cell 57:1161-71 (1989)). Inhibitors of prenylation have been shown to possess anti-tumor activities, such as slowing tumor growth (Garcia, A.M. et al., J. Biol Chem. 268:18415-18 (1993)) and to promote rejection in animal models (Kohl, N.E. et al., Nature Med. 1:192-91 (1995)). Three O-glycosylation sites are predicted at or near the amino terminus - thr8, ser2, and ser9 using NetOGlyc2.0.<br>
CSS Transcript is Overexpressed in Breast and Bladder Carcinoma: An ideal target antigen for tumor immunotherapy would be abundantly expressed in multiple independent carcinomas, and would be absent or minimally expressed in normal proliferating and vital tissues. Differential expression of C35 was confirmed by Northern blot analysis. C35 is expressed in 7/10 human tumor cell lines at levels 10-25X higher than expression in a normal immortalized breast epithelial cell line, H16N2 (Figure 11A). Importantly, C35 expression is shared among lines derived from both primary (21NT, 21PT) and metastatic (21MT1, 21MT2) lesions of a single patient, suggesting its expression may be associated with early events in the process of tumor transformation. In addition, the overexpression of C35 is shared among independently derived human mammary carcinoma cell lines, including SKBR3, T47D, and BT474. Interestingly, the C35 expression pattern in SKBR3, MDA MB231, H16N2 and tumors derived from the same patient correlates with Her2/neu expression, which may be associated with the close genomic proximity of the two genes and the incidence of HER2/neu gene amplification.<br>
To investigate whether C35 expression in patient derived tumors is clinically relevant for development of a cancer vaccine, mRNA was extracted from snap frozen human tissue samples obtained from the Cooperative Human Tissue Network (CHTN). 70% of primary breast tumor samples overexpress C35 transcript (Figure 11B), and 35% (7/20) of these breast adenocarcinomas overexpress at levels 10-70 fold higher than normal breast Overexpression of C35 is also seen in 50% of bladder carcinoma primary specimens examined<br><br>
-249-<br>
(Figure 12), while 20% (3/14) of primary bladder carcinoma express at levels greater than 10 fold higher than normal bladder. Overexpression of C35, at levels 9X or greater, was not detected in panels of ovarian (0/7), prostate (0/5), or colon (0/15) carcinomas (data not shown).<br>
2C3 Monoclonal Antibody reacts with C35+ cells:   In order to confirm differential expression of the gene product encoded by C35, a monoclonal antibody against the shared tumor antigen was selected.   Hybridomas were produced by immunizing mice with a poorly immunogenic BALB/cByJ tumor cell line, which had been transduced with a retroviral human C35 recombinant. Hybridoma clones were screened for their ability to stain C35-H- breast and bladder tumor cell lines (Figures 13Aandl3B). Non-tumorigenicbreastH16N2 and bladder SV-HUC epithelial cell lines did not show a significant shift in fluorescence intensity when compared to the isotype control. In contrast, 2C3 monoclonal antibody specifically stained C35+ breast tumors, SKBR3 and 21-NT-D, and bladder tumor ppTHA3. The staining was carried out on cells that were neither fixed nor permeabilized, indicating that 2C3 antibody recognizes a surface molecule.<br>
Inhibition of tumor growth with C35 antibodies:<br>
Antibodies are useful tools to detect diagnostic markers of cancer, but they may also have potential use for therapeutic applications. Humanized Her2/neu specific antibody (Herceptin) has been successfully employed for treatment of some breast cancers. Herceptinbmds HER2/neuanddownregulates signal transduction from the growth factor receptor. Growth inhibition studies were performed with C35-specific 2C3 antibody. 21NT-D breast tumor and H16N2 "normal" breast cell lines were grown in vitro in the presence of various antibody concentrations. An XTT assay was performed to evaluate cell expansion at 72 hours. Results shown in Figure 14 indicate that 2C3 inhibits<br><br>
-250-<br>
growth of 21NT tumor cells by approximately 50% at concentrations as low as 1 ug/ml.<br>
A C35 Class I epitope is HLA-A2 restricted:<br>
Establishment of self-tolerance is a major obstacle to development of vaccines based on self proteins. Tolerance, however, must be defined in terms of quantitative levels of expression. It is possible that even while high affinity antigen-specific T cells are tolerized, T cells with lower affinity receptors that do not have functional avidity for a low concentration of antigen escape tolerance induction. These same T cells could, however, subsequently become functionally significant if there is markedly increased avidity associated with overexpression of the target antigen. Even if they are few in number, such T cells could be expanded by the most fundamental of immunological manipulations, vaccination.<br>
C35 is a self-protein expressed at low basal levels in normal human tissues. It was, therefore, necessary to determine if human T cells are tolerant to C35 at levels of expression characteristic of carcinomas. The only way to exclude tolerance is by demonstrating responsiveness, and the only way to demonstrate responsiveness short of a clinical trial is by in vitro stimulation. Human T cells and autologous dendritic cells were derived from PBL from a normal donor. The T cells were primed by alternate stimulation with autologous dendritic cells infected with retroviral or pox virus recombinants of the C35 cDNA. CTL recovered in vitro following several cycles of stimulation were analyzed for their ability to lyse C35+ target tumor cells (Figure 15) or to secrete cytokines in response to antigen induced activation (Figure 16). The targets either endogenously expressed C35 and/or HLA-A2.1, or were engineered to express these proteins via standard transfection with a C35-recombinant mammalian expression vector, or by infection with C35-recombinant vaccinia virus. Previous studies have demonstrated that protein expression by vaccinia<br><br>
-251-<br>
virus is an efficient means of targeting peptides to the MHC-I processing pathway (Moss, B., Science 252:1662-61 (1991).<br>
Following several rounds of stimulation, both a bulk T cell line and a T cell clone were selected that differentially lyse C35+ tumor cells relative to C35lo H16N2 normal breast epithelial cell line in a 51Cr release assay (Figure 15A and B). The HLA-A2 restricted C35-specific CTL clone 10G3 efficiently lysed the HLA-A2 transfected tumorigenic cell line, 21-NT.A2, which expresses C35 antigen at levels 15X greater than H16N2 and is stained with 2C3 monoclonal antibody. Specific lysis was also with the HLA-A2+ bladder tumor cell line ppT11A3 compared to the non-tumorigenic bladder cell line SV-HUC from which it was derived (Figure 15B). The data demonstrate CTL sensitivity of tumors that express high levels of C35 with minimal lysis of C35lo nontumorigenic immortalized cell lines. Importantly, the same CTL are not reactive with MEC, a primary culture of non-immortalized, non-transformed, HLA-A2+ breast epithelial cells that do not express C35 at significant levels. Further evidence to support C35+ tumor recognition by the T cells is shown in Figure 16A and B. The T cells secrete IFN-gamma and TNF-alpha in response to C35+, HLA-A2+ stimulator. Again, the non-tumorigenic, C35lo cell line H16N2.A2 did not induce cytokine secretion by C35-specific T cells. However, infection of this line with vaccinia virus recombinant for C35 confers the ability to activate the T cells. Since the T cells do not secrete IFN-gamma or TNF-alpha in response to H16N2 A2 transduced with an irrelevant protein L3, this indicates that the response is specific to C35 protein expression (Figure 16A and B).<br>
FoEowing several rounds of stimulation, both a bulk T cell line and a T cell clone were selected that differentially lyse C35+, HLA-A2+ tumor cells in a 51Cr release assay. The C35-specific CTL did not lyse the HLA-A2 transfected non-tumorigenic breast epithelial cell line, H16N2.A2 (Figure 15A and B), although this cell line does express C35 at low levels based on the Northern blot data shown in Figure 11A. However, C35-specific CTL efficiently lysed the HLA-A2 transfected tumorigenic cell line, 21-NT.A2, which expresses C35<br><br>
-252-<br>
antigen at levels 15X greater than H16N2 and is stained with 2C3 monoclonal antibody. C35+ tumor-specific lysis was also shown with the bladder tumor cell line ppT11A3 compared to the non-tumorigenic bladder cell line SV-HUC from which it was derived.   The data demonstrate CTL sensitivity of tumors that express high levels of C35 with minimal lysis of C35lo nontumorigenic immortalized cell lines. Importantly, the same CTL are not reactive with MEC, a primary culture of non-immortalized, non-transformed, HLA-A2+ breast epithelial cells that do not express C35 at significant levels. Further evidence to support C35+ tumor recognition by the T cells is shown in Figure 16A andB. The T cells secrete IFN-gamma and TNF-alpha in response to C35+, HLA-A2+ stimulator. Again, the non-tumorigenic, C35lo cell line H16N2. A2 did not induce cytokine secretion by C35-specific T cells. However, infection of this line with vaccinia virus recombinant for C35 confers the ability to activate the T cells. Since the T cells do not secrete IFN-gamma or TNF-alpha in response to H16N2.A2 transduced with an irrelevant protein L3, this indicates that the response is specific to C35 protein expression.<br>
The C35-specific T cells were generated from a donor with HLA haplotype A2, All, B8, B35. The bladder cell lines, SV-HUC and ppT11A3 derive from a donor with haplotype HLA-A2, B18, B44. However, since the H16N2 immortalized breast epithelial cell line and 21-NT and 21-MT breast tumor cell lines derived from the same HLA-A2 negative donor, these cell lines had to be transfected with HLA-A2.1 to provide a required MHC restriction element for recognition by HLA-A2 restricted 10G3 T cell clone (Figure 16A and B). The T cells were strongly stimulated to secrete these lymphokines by the breast lines that expressed both C35 and HLA-AZ (compare 21-MT2 with 21-MT2.wA2). The data indicate that there is at least one HLA-A2.1 defined epitope of C35.<br>
Deletion mutants of C35 coding region were constructed to identify cDNA segments that encode the peptide epitope recognized by the CTL. Figure 15A and B demonstrates almoste valent IFN-gammaand TNF-alphasecretion<br><br>
-253-<br>
by T cells stimulated with the full length C35 or a truncated mutant<br>
encompassing only the first 50 amino acids.<br>
Discussion<br>
C35 is a novel tumor antigen that is overexpressed in breast and bladder carcinoma. The gene has properties that make it a promising candidate fortumor immunotherapy. It is expressed in a significant number of tumors derived from different individuals. Expression in vital normal tissues is relatively low, reducing the risk of autoimmune reactions and, equally important, making it unlikely that immune cells have been rendered tolerant to the gene product. C35 is characterized as a "tumor antigen" since C35 expressing dendritic cells induce autologous tumor specific human cytotoxic T lymphocytes in vitro.<br>
C35 is a novel gene product of unknown function. However, our studies with monoclonal antibodies have provided some insight into the localization of the protein. Both serum and a monoclonal antibody derived from a C35-immunized mouse specifically stain unfixed cells that express C35. This suggests that the antibody recognizes a tumor surface membrane protein. Although the protein sequence does not conform with known transmembrane motifs based on hydrophobicity, the existence of a prenylation site at the COOH terminus suggests insertion into the membrane. Prenylation is apost-translational lipid modification that produces a substantially more hydrophobic protein with high affinity for the membrane (Fu, H.-W. and Casey, P. J., Recent Progress in Hormone Research 54:315-43 (1999)). Otherproteins that contain prenylation sites include the Ras oncogene family. Ras GTPases act in signal tranduction cascades with MAPK to induce cell division and proliferation. Ras proteins are anchored to the plasma membrane via prenylation, but the proteins remain in the cytoplasmic face of the membrane. Therefore, it is possible that C35 also remains on the cytoplasmic side of the membrane, but there may be sufficient transport to the outer surface to be detected with a specific antibody.<br><br>
-254-<br>
C35-specific antibodies are valuable tools for studying the protein<br>
expression of C35, to corroborate Northern blot analysis, and for use in assays such as Western blots and immunohistochemistry. In addition, these antibodies may have therapeutic benefits, such as has been recently been demonstrated for Herceptin (Baselga, J. et al., J. Clin. Oncol. 74:737-44 (1996); Pegram, M.D. et<br>
al., J. Clin. Oncol. 7(5:2659-71 (1998)), an antibody to the tumor-associated antigen HER2-new(c-erbB-2)(Schechter, A.L. et al., Nature 312:513-16 (1984). Herceptin's anti-tumor effects include binding the epidermal growth factor receptor, which inhibits tumor cell growth, and eliciting antibody dependent cell-mediated cytotoxicity (Dillman, R.O., Cancer Biotherapy &amp; Radiopharmaceuticals 14:5-10 (1999).<br>
EXAMPLE 6 Induction of Cytotoxic T Cells Specific for Target Antigens of Tumors<br>
Human tumor-specific T cells have been induced in vitro by stimulation of PEL with autologous tumors or autologous antigen presenting cells pulsed with tumor lysates (van DerBruggen, P. et al., Science 254: 1643-1647(1991); Yasumura, S. et al., Cancer Res. 53:1461-68 (1993); Yasumura, S. et al., Int. J. Cancer 57: 297-305 (1994); Simons, J.W. et al., Cancer Res. 57:1537-46 (1997); Jacob, L. et al., Int. J. Cancer 71:325-332 (1997); Chaux, P. et al., J. Immunol. 163:2928-2936(1999)). PBL have been derived from either patients deliberately immunized with tumor, with tumor modified to enhance its immunogenicity, or with tumor extracts, orpatients whose only prior stimulation was in the natural course of disease. T cells with reactivity for infectious agents could be similarly derived by in vitro stimulation of T cells with autologous cells that have been either infected in vitro or were infected in vivo during the natural course of exposure to the infectious agent. CD4+ and CD8+ T cells or antibody selected under these or other conditions to be specific for either tumor cells or cells infected with either a virus, fungus or mycobacteria or T cells or antibodies<br><br>
-255-<br>
specific for the target antigens of an autoimmune disease could be employed in the selection and screening methods of this invention to detect and isolate cDNA that encode these target antigens and that have been incorporated into a representative cDNA library.<br>
In spite of demonstrated success in the induction of human T cell responses in vitro against a number of antigens of tumors and infected cells, it is not certain that these represent the full repertoire of responses that might be induced in vivo. Because safety considerations limit the possibilities of experimental immunization in people, there is a need for an alternative animal model to explore immune responses to human disease antigens. The major obstacle to developing such a model is that numerous molecules expressed in normal human cells are strongly immunogenic in other species. It is, therefore, necessary to devise a means of inducing tolerance to normal human antigens in another species in order to reveal immune responses to any human disease-specific antigens. It is now recognized that activation of antigen-specific T lymphocytes requires two signals of which one involves presentation of a specific antigenic complex to the T cell antigen receptor and title second is an independent costimulator signal commonly mediated by interaction of the B7 family of molecules on the surface of the antigen presenting cell with the CD28 molecule on the T cell membrane. Delivery of an antigen-specific signal in the absence of a costimulator signal not only fails to induce T cell immunity but results in T cell unresponsiveness to subsequent stimulation (Lenschow, D.J. et<br>
al., Ann. Rev. Immunol. 14:233-258 (1996)). Additional studies have revealed a key role for another pair of interactions between the CD40 molecule on the antigen presenting cell and CD40 ligand on the T cell. This interaction results in upregulation of the B7 costimulator molecules (Roy, M. et al., Eur. J. Immunol 25:596-603 (1995)). In the presence of anti-CD40 ligand antibody either in vivo or in vitro, the interaction with CD40 is blocked, B7 costimulator is not up regulated, and stimulation with a specific antigenic complex results in T cell tolerance rather than T cell immunity (Bluestone, J.A. et al., Immunol<br><br>
-256-<br>
Rev. 165:5-12(1998)). Various protocols to block either or both CD40/CD40 ligand interactions and B7/CD28 interactions have been shown to effectively induce transplantation tolerance (Larsen, C. et al., Nature 381:434-438 (1996); Kirk et al., Nature Medicine 5:686-693 (1999)). An example of the effect of anti-CD40 ligand antibody (anti-CD 154) in blocking the reactivity of murine T cells to specific transplantation antigens is shown in FIG. 17. DBA/2 (H-2d) mice were immunized with 107 C57B1/6 (H-2b) spleen cells intraperitoneally and, in addition, were injected with either saline or 0.5 mg monoclonal anti-CD40 ligand antibody (MR1, anti-CD 154, Pharmingen 09021D) administered both at the time of immunization and two days later. On day 10 following immunisation, spleen cells from these mice were removed and stimulated in vitro with either C57B1/6 or control allogeneic C3H (H-2k) spleen cells that had been irradiated (20 Gy), After 5 days in vitro stimulation, C57B1/6 and C3H specific cytolytic responses were assayed at various effector:target ratios by 51Cr release assay from specific labeled targets, in this case, either C3H or C57B1/6 dendritic cells pulsed with syngeneic spleen cell lysates. The results in FIG. 17 show that significant cytotoxicity was induced against the control C3H alloantigens in both saline and anti-CD 154 treated mice whereas a cytotoxic response to C57B1/6 was induced in the saline treated mice but not the anti-CD 154 treated mice. This demonstrates specific tolerance induction to the antigen employed for immune stimulation at the time CD40/CD40 ligand interactions were blocked by anti-CD154.<br>
A tolerization protocol similar to the above employing either anti-CD 154 alone or a combination of anti-CD154 and anti-B7 or anti-CD28 could be employed to induce tolerance to normal human xenoantigens in mice prior to immunization with a human tumor. In one embodiment, the normal antigens would be expressed on immortalized normal cells derived from the same individual and tissue from which a tumor cell line is derived. In another embodiment, the normal and tumor antigens would derive from cell lysates of normal and tumor tissue of tile same individual each lysate pulsed onto antigen<br><br>
-257-<br>
presenting cells for presentation to syngeneic murine T cells both in vivo and in vitro. In a preferred embodiment, the tumors would derive by in vitro mutagenesis or oncogene transformation from an immortalized, contact-inhibited, anchorage-dependent, non-tumorigenic cell line so that very well-matched non-tumorigenic cells would be available for tolerance induction.<br>
An alternative to the tolerization protocols is depletion of T cells that are activated by normal antigens prior to immunization with tumor. Activated T cells transiently express CD69 and CD25 with peak expression between 24 and 48 hours post-stimulation. T cells expressing these markers following activation with normal cells or normal cell lysates can be depleted with anti-CD69 and anti-CD25 antibody coupled directly or indirectly to a matrix such as magnetic beads. Subsequent immunization of the remaining T cells with tumor cells or tumor cell lysates either in vitro or in vivo following adoptive transfer will preferentially give rise to a tumor-specific response.<br>
In one embodiment, the mice to be tolerized to normal human cells or lysates and subsequently immunized with tumor cells or lysates are any of a variety of commercially available inbred and outbred strains. Because murine T cells are restricted to recognize peptide antigens in association with murine MHC molecules which are not expressed by human cells, effective tolerization or stimulation requires either transfection of human cells with murine MHC molecules or re-presentation of human normal and tumor antigens by mouse antigen presenting cells. Dendritic cells are especially preferred as antigen presenting cells because of their ability to re-present antigenic peptides in both the class I and class II MHC pathways (Huang, et al., Science 264:961-965 (1994); Inaba, et al., J. Exp. Med. 176:1702 (1992); Inaba, et al., J. Exp. Med. 178:479-488(1993)). Inanother embodiment, mice double transgenic for human HLA and human CD8 or CD4 are employed. The HLA transgene permits selection of a high affinity, HLA-restricted T cell repertoire in the mouse thymus. In addition, a human CD8 or CD4 transgene is required because murine CDS and CD4 do not interact efficiently with the cognate human class I or class II MHC<br><br>
-258-<br>
molecules. The use of non-transgenic mice to generate human tumor-specific T cells would lead to identification of any human tumor antigens that can be processed in association with murine MHC molecules. Since multiple murine strains with diverse MHC molecules are available, this could encompass a wide range of antigens.   However, it would have to be separately determined by stimulation of human T cells with autologous antigen presenting cells whether these tumor-specific antigens also express peptides that can be processed and presented in association with human HLA.   Such peptides may or may not overlap with those initially detected in association with murine MHC molecules hut would derive from the same set of proteins. By employing HLA transgenic mice it is possible to more directly address the relevance of antigenic peptides to human MHC. There can, however, be no assurance that peptide processing will he identical in murine and human antigen presenting cells.   It is essential, therefore, to confirm that HLA-restricted, human tumor antigen-specific T cells are indeed also crossreactive on human tumor cells. Finally, no matter how the issue of processing and presentation in association with human HLA is addressed, it must in all cases be determined whether human T cells are reactive to the identified antigens or whether they have been rendered tolerant, perhaps due to expression of the same or arelated antigen in some other non-homologous normal tissue. Relevant information on this point can be obtained through in vitro stimulation of human T cell responses with the identified antigens or antigenic peptides presented by autologous antigen presenting cells. Ideally, it would be shown that patients with antigen positive tumors have an increased frequency of T cells reactive with the purported tumor-specific antigen. To demonstrate that the antigen-specific human T cells induced can be effective in eradicating tumors, the selected human T cells could be adoptively transferred into SCID mice bearing ahuman tumor xenograft as described by Renner, C. et<br>
al.,  Science 264:833-835 (1994). However, definitive evidence for clinical relevance would await the results of a human clinical trial.<br><br>
-259-<br>
Conditions for in vitro stimulation of primary human T cell responses are described in Example 2 and are applicable to both. CD4+ and CD8+ responses. The strategies described for induction of human T cell or antibody responses specific for human tumors are equally applicable to induction of T cell or antibody responses to target antigens of human cells infected with either a virus, fungus or mycobacteria. Indeed, in this case the same uninfected cell population affords an immediately available normal control population for tolerance induction and to confirm infectious specificity.<br>
The construction of transgenic mice is well known in the art and is described, for example, in Manipulating the Mouse Embroy: A laboratory Manual, Hogan, et al., Cold Spring Harbor Press, second edition (1994). Human CD8 transgenic mice may be constructed by the method of LaFace, et al., J. Exp. Med. 182:1315-25 (1995). Construction of new lines of transgenic mice expressing the human CD8alpha and CD8beta subunits may be made by insertion of the corresponding human cDNA into a human CD2 minigene based vector for T cell-specific expression in transgenic mice (Zhumabekov, et al., J. Immunol Methods 755:133-140(1995)). HLA class I transgenic mice may be constructed by the methods of Chamberlain, et al., Proc Natl. Acad. Sci. USA 55:7690-7694 (1988) or Bernhard, et al., J. Exp. Med. 168:1157-1162 (1988) orVitiello, et al., J. Exp. Med. J73:1007-1015 (1991) or Barra, et al., J. Immunol. 150:3681-3689 (1993).<br>
Construction of additional HLA class I transgenic mice may be achieved by construction of an H-2Kb cassette that includes 2 kb of upstream regulatory region together with the first two introns previously implicatedin gene regulation (Rralova, et al., 1992,BMBOJ 11:4591-4600). Endogenous translational start sites are eliminated from this region and restriction sites for insertion of HLA cDNA are introduced into the third exon followed by a polyA addition site. By including an additional 3kb of genomic H-2Kb sequence at the 3' end of this construct, the class I gene can be targeted for homologous recombination, at the H-2Kb locus in embryonic stem cells. This has the advantage that the transgene<br><br>
-260-<br>
is likely to be expressed at a defined locus known to be compatible with murine class I expression and that these mice are likely to be deficient for possible competition by H-2Kb expression at the cell membrane. It is believed that this will give relatively reproducible expression of diverse human HLA class I cDNA introduced in the same construct.<br>
EXAMPLE 7 Construction of N-Terminal and/or C-Terrninal Deletion Mutants<br>
The following general approach may be used to clone a N-terminal or C-terminal deletion C35 deletion mutant. Generally, two ohgonucleotideprirners of about 15-25 nucleotides are derived from the desired 5' and 3' positions of a polynucleotide of SBQ ID NO: 1. The 5' and 3' positions of the primers are determined based onihe desired C35 polynucleotide fragment. An initiation and stop codon are added to the 5' and 3' primers respectively, if necessary, to express the C35 polypeptide fragment encoded by the polynucleotide fragment. Preferred C35 polynucleotide fragments are those encoding the candidate MHC class I and MHC class II binding peptides disclosed above in the "Polynucleotide and Polypeptide Fragments" section of the Specification.<br>
Additional nucleotides containing restriction sites to facilitate cloning of the C35 polynucleotide fragment in a desired vector may also be added to the 5' and 3' primer sequences. The C35 polynucleotide fragment is amplified from F i genomic DNA or from fee cDNA clone using me appropriate PCR oligonucleotide primers and conditions discussedherein or known in the art. The C35 polypeptide fragments encoded by the C35 polynucleotide fragments of the present invention may be expressed and purified in the same general manner as the full length polypeptides, although routine modifications may be necessary due to the differences in chemical and physical properties between a particular fragment and full length polypeptide.<br><br>
-261-<br>
As a means of exemplifying but not limiting the present invention, the polynucleotide encoding the C35 polypeptide fragment is amplified and cloned as follows: A 5' primer is generated comprising a restriction enzyme site followed by an initiation codon in frame with the polynucleotide sequence encoding the N-tenninal portion of an MHC binding peptide epitope listedin any of Tables 1 through 6. A complementary 3' primer is generated comprising a restriction enzyme site followed by a stop codon in frame with the polynucleotide sequence encoding C-terminal portion of a C35 MHC binding peptide epitope listedin any of Tables 1 through 6.<br>
The amplified polynucleotide fragment and the expression vector are digested with restriction enzymes which recognise the sites in the primers. The digested polynucleotides are then ligated together. The C35 polynucleotide fragment is inserted into the restricted expression vector, preferably in a manner which places the C35 polypeptide fragment coding region downstream from the promoter. The ligation mixture is transformed into competent E. coli cells using standard procedures and as described in the Examples herein. Plasmid DNA is isolated from resistant colonies and the identity of the cloned DNA confirmed by restriction analysis, PCR and DNA sequencing.<br>
EXAMPLE 8 Protein Fusions of C35<br>
C35 polypeptides are preferably fused to other proteins. These fusion proteins can be used for a variety of applications. For example, fusion of C35 polypeptides to His-tag, HA-tag, protein A, IgG domains, and maltose binding protein facilitates purification. (See Example 5; see also EP A 394,827; Traunecker et al., Nature 357:84-86 (1988).) Similarly, fusion to IgG-1, IgG-3, and albumin increases the halfiife time in vivo. Nuclear localization signals fused to C35 polypeptides can target the protein to a specific subcellular localization, while covalentheterodimer or homodimers can increase or decrease<br><br>
-262-<br>
the activity of a fusion protein. Fusion proteins can also create chimeric molecules having more than one function. Finally, fusion proteins can increase solubility and/or stability of the fused protein compared to the non-fused, protein. All of the types of fusion proteins described above can be made by modifying the following protocol, which outlines the fusion of a polypeptide to an IgG molecule.<br>
Briefly, the human Fc portion of the IgG molecule can be PCR amplified, using primers that span the 5' and 3' ends of the sequence described helow. These primers also should have convenient restriction enzyme sites that will facilitate cloning into an expression vector, preferably a mammalian expression vector.<br>
For example, if pC4 (Accession No. 209646) is used, the human Fc portion can be ligated into the BamHI cloning site. Note that the 31 BamHI site should be destroyed. Next, the vector containing the human Fc portion is re-restricted with BamHI, linearizing the vector, andC35 polynucleotide, isolated by the PCR protocol described in Example 1, is ligated into this BamHI site. Note that the C35 polynucleotide is cloned without a stop codon, otherwise a fusion protein will not be produced.<br>
If the naturally occurring signal sequence is used to produce the secreted protein, pC4 does not need a second signal peptide. Alternatively, if the naturally occurring signal sequence is not used, the vector can be modified to include a heterologous signal sequence. (See, e.g., WO 96/34891.)<br>
Human IgG Fc region:<br>
GGGATCCGGAGCCCAAATCTTCTGACAAAACTCACACATGeeCACCGTGCCC AGCACCTGAATTCGAGGGTGCACCGTCAGTCTTCCTCTTCCCCCCAAAACCC AAGGACACCCTCATGATCTCCCGGACTCCTGAGGTCACATGCGTGGTGGTGG ACGTAAGCCACGAAGACCCTGAGGTCAAGTTCAACTGGTACGTGGACGGCGT GGAGGTGCATAATGCCAAGACAAAGCCGCGGGAGGAGCAGTACAACAGCACG TACCGTGTGGTCAGCGTCCTCACCGTCCTGCACCAGGACTGGCTGAATGGCA AGGAGTACAAGTGCAAGGTCTCCAACAAAGCCCTCCCAAGCCCCATCGAGAA<br><br>
-263-<br>
AACCATCTCCAAAGCCAAAGGGCAGCCCCGAGAACCACAGGTGTACACCCTG CCCCCATCCCGGGATGAGCTGACCAAGAACCAGGTCAGCCTGACCTGCCTGG TCAAAGGCTTCTATCCAAGCGACATCGCCGTGGAGTGGGAGAGCAATGGGCA GCCGGAGAACAACTACAAGACCACGCCTCCCGTGCTGGACTCCGACGGCTCC rTCTTCCTCTACAGCAAGCTCACCGTGGACAAGAGCAGGTGGCAGCAGGGGA ACGTCTTCTCATGCTCCGTGATGCATGAGGCTCTGCACAACCACTACACGCA GAAGAGCCTCTCCCTGTCTCCGGGTAAATGAGTGCGACGGCCGCGACTCTAG AGGAT      (SEQ   ID NO:[84])<br>
A preferred fusion product is fusion of a C35 peptide to the amino terminus of an MHC molecule in such fashion that the peptide will naturally occupy the MHC peptide binding groove. Kang, X. et al., Cancer Res. 57:202-5 (1997) have reported that such fusion proteins can be employed in vaccine compositions that are especially effective for stimulation of specific T cells.<br>
EXAMPLE 9 Method of Detecting Abnormal Levels of C35 in a Biological Sample<br>
C35 polypeptides can be detected in a biological sample, and if an increased or decreased level of C35 is detected, this polypeptide is a marker for a particular phenotype. Methods of detection are numerous, and thus, it is understood that one skilled in the art can modify the following assay to fit their particular needs.<br>
For example, antibody-sandwich ELTSAs are used to detect C35 in a sample, preferably a biological sample. Wells of a microtiter plate are coated with specific antibodies to C35, at a final concentration of 0.2 to 10 ug/ml. The antibodies are either monoclonal or polyclonal. The wells are blocked so that non-specific binding of C35 to the well is reduced.<br>
The coated wells are then incubated for &gt; 2 hours at RT with a sample containing C35. Preferably, serial dilutions of the sample should be used to<br><br>
-264-<br>
validate results. The plates are then washed three times with saline to remove unbounded C35.<br>
Next, 50 ul of specific antibody-alkaline phosphatase conjugate that recognizes a C35 antigenic determinant which does not overlap with that recognized by the plate bound antibody, at a concentration of 25-400 ng, is added and incubated for 2 hours at room temperature. The plates are again washed three times with deionized or distilled water to remove unbounded conjugate.<br>
Add 75 ul of 4-methylumbelliferyl phosphate (MUP) or p-nitrophenyl phosphate (NPP) substrate solution to each well and incubate 1 hour at room temperature. Measure the reaction by a microtiter plate reader. Prepare a standard curve, using serial dilutions of a control sample, and plot C35 polypeptide concentration on the X-axis (log scale) and fluorescence or absorbance on the Y-axis (linear scale). Interpolate the concentration of the C35 in the sample using the standard curve.<br>
EXAMPLE 10 Formulating a Polypeptide<br>
The C35 composition will be formulated anddosedin afashion consistent with good medical practice, taking into account the clinical condition of the individual patient (especially the side effects of treatment with the C35 polypeptide alone), the site of delivery, the method of administration, the scheduling of adrninistration, and other factors known to practitioners. The "effective amount" forpurposes herein is thus determined by such considerations.<br>
As a general proposition, the total pharmaceutically effective amount of C35 adrninistered parenterally per dose will be in the range of about 1 ug/kg/day to 10 mg/kg/day of patient body weight, although, as noted above, this will be subject to therapeutic discretion. More preferably, this dose is at least 0.01 mg/kg/day, and most preferably for humans between about 0.01 and 1 mg/kg/day. If given continuously, C35 is typically administered at a dose rate<br><br>
-265-<br>
of about 1 ug/kg/hour to about 50ug/kg/hour, either by 1-4 injections per day or by continuous subcutaneous infusions, for example, using a mini-pump. An intravenous bag solution may also be employed. The length of treatment needed to observe changes and the interval following treatment for responses to occur appears to vary depending on the desired effect.<br>
Pharmaceutical compositions containing C35 are administered orally, rectally, parenterally, intracistemally, intravaginally, intraperitoneally, topically (as by powders, ointments, gels, drops or transdermal patch), bucally; or as an oral or nasal spray. "Pharmaceutically acceptable carrier" refers to a non-toxic solid, semisolid or liquid filler, diluent, encapsulating material or formulation auxiliary of any type. The term "parenteral" as used herein refers to modes of administration which include intravenous, intramuscular, intraperitoneal, intrasternal, subcutaneous and intraarticular injection and infusion.<br>
C35 is also suitably administered by sustained-release systems. Suitable examples of sustained-release compositions include semi-permeable polymer matrices  in the  form of shaped articles,  e.g.,  films,  or mirocapsules. Sustained-release matrices include polylactides (U.S. Pat. No. 3,773,919, EP 58,481), copolymers of glutamic acid andgamma-ethyl-L-glutamate(Sidman, U. et al., Biopolyrners 22:547-556 (1983)), poly (2- hydroxyethyl methacrylate) (R. Langer et al., J. Biomed. Mater. Res. 75:167-277 (1981), and R. Langer, Chem. Tech. 72:98-105 (1982)), ethylene vinyl acetate (R. Langer et al) or poly-D-(-)-3-hydroxybutyricacid(EP 133,988). Sustained-release compositions alsoinclude liposomally entrapped C35 polypeptides. Liposomes containingthe C35 are prepared by methods known per se:DB 3,218,121; Epstein et al., Proc. Natl. Acad. Sci. USA 52:3688-3692 (1985); Hwang et al., Proc. Natl Acad. Sci. USA 77:4030-4034 (1980); EP 52,322; EP 36,676; BP 88,046; EP 143,949; EP 142,641; Japanese Pat. Appl. 83-118008; U.S. Pat. Nos. 4,485,045 and 4,544,545; and EP 102,324. Ordinarily, the liposomes are of the small (about 200-800 Angstroms) unilamellar type in which the lipid content is greater than<br><br>
-266-<br>
about 30 mol. percent cholesterol, the selected proportion being adjusted for the optimal secreted polypeptide therapy.<br>
For parenteral administration, in one embodiment, C35 is formulated generally by mixing it at the desired degree of purity, in a unit dosage injectable form (solution, suspension, or emulsion), with a pharmaceutically acceptable carrier, i.e., one that is non-toxic to recipients at the dosages and concentrations employed and is compatible with other ingredients of the formulation. For example, the formulation preferably does not include oxidizing agents and other compounds that are known to be deleterious to polypeptide.<br>
Generally, the formulations are prepared by contacting C35 uniformly and intimately with liquid carriers or finely divided solid carriers or both. Then, if necessary, the product is shaped into the desired formulation. Preferably the carrier is a parenteral carrier, more preferably a solution that is isotonic with the blood of the recipient. Examples of such carrier vehicles include water, saline, Ringer's solution, and dextrose solution. Non-aqueous vehicles such as fixed oils and ethyl oleate are also useful herein, as well as liposomes.<br>
The carrier suitably contains minor amounts of additives such as substances that enhance isotonicity and chemical stability. Such materials are non-toxic to recipients at the dosages and concentrations employed, and include buffers such as phosphate, citrate, succinate, acetic acid, and other organic acids or their salts; antioxidants such as ascorbic acid; low molecular weight (less than about ten residues) polypeptides, e.g., polyarginine or tripeptides; proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidone; amino acids, such as glycine, glutamic acid, aspartic acid, or argtnrne; monosaccharides, disaccharides, and other carbohydrates including cellulose or its derivatives, glucose, manose, or dextrins; chelating agents such as EDTA; sugar alcohols such as mannitol or sorbitol; counterions such as sodium; and/or nonionic surfactants such as polysorbates, poloxamers, or PEG.<br>
C35 is typically formulated in such vehicles at a concentration of about 0.1 mg/ml to 100 mg/ml, preferably 1-10 mg/ml, at a pH of about 3 to 8. It will<br><br>
-267-<br>
be understood that the use of certain of the foregoing excipients, carriers, or stabilizers will result in the formation of polypeptide salts.<br>
C35 used for therapeutic administration can be sterile. Sterility is readily accomplished by filtration through sterile filtration membranes (e.g., 0.2 micron membranes). Therapeutic polypeptide compositions generally are placed into a container having a sterile access port, for example, an intravenous solution bag or vial having a stopper pierceable by a hypodermic injection needle.<br>
C35 polypeptides ordinarily will be stored in unit or multi-dose containers, for example, sealed ampoules or vials, as an aqueous solution or as a lyophilized formulation for reconstitution. As an example of a lyophilized formulation, 10-ml vials are filled with 5 ml of sterile-filtered 1% (w/v) aqueous C35 polypeptide solution, and the resulting mixture is lyophilized. The infusion solution is prepared by reconstituting the lyophilized C35 polypeptide using bacteriostatic Water-for-Injection.<br>
The invention also provides a pharmaceutical pack or kit comprising one or more containers filled with one or more of the ingredients of the pharmaceutical compositions of the invention. Associated with such containers) can be a notice in the form prescribed by a governmental agency regulating the manufacture, use or sale of pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human adrninistration. In addition, C35 may be employed in conjunction with other therapeutic compounds.<br>
It will be clear that the invention may be practiced otherwise than as particularly described in the foregoing description and examples. Numerous modifications and variations of the present invention are possible in light of the above teachings and, therefore, within the scope of the appended claims, the invention may be practiced otherwise than as particularly described.<br>
The entire disclosure of each document cited (including patents, patent applications, journal articles, abstracts, laboratory manuals, books, or other<br><br>
-268-<br>
disclosures) in the Background of the Invention, Detailed Description, Examples, and Sequence Listing is hereby incorporated herein by reference.<br>
It will be clear that the invention may be practiced otherwise than as particularly described in the foregoing description and examples. Numerous modifications and variations of the present invention are possible in light of the above teachings and, therefore, within the scope of the appended claims, the invention may be practiced otherwise than as particularly described.<br>
The entire disclosure of each document cited (including patents, patent applications, journal articles, abstracts, laboratory manuals, books, or other disclosures) in the Background of the Invention, Detailed Description, Examples, and Sequence Listing is hereby incorporated herein by reference.<br><br>
WE CLAIM:<br>
1.	An in vitro produced polypeptide of not more than 80 amino acids in length comprising at least-one C35 peptide epitope selected from the group, consisting of: amino acids S-9 to V-17, amino acids V-10 to V-17, amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids S-21 to F-35, amino acids C-30 to T-38, amino acids E-31 to Y-39, amino acids E-36 to A-43, amino acids A-37 to A-45, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to 1-53; amino acids G-52 to L-59, amino acids E-54 to T-62, amino acids S-57 to F-75, amino acids R-58 to 1-67, amino acids G-61 to 1-69, amino acids G-63 to F-83, amino acids E-66 to L-73, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1-105 to 1-114, each of SEQ ID NO:2.<br>
2.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids S-9 to V-17 of SEQ ID NO:2.<br>
3.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids V-10 to V-17 of SEQ ID NO:2.<br>
4.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-16 to V-23 of SEQ ID NO:2.<br>
5.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-16 to R-24 of SEQ ID NO:2.<br>
6.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-16 to 1-25 of SEQ ID NO:2.<br>
7.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids S-21 to F-35 of SEQ ID NO:2.<br><br>
270<br>
8.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids C-30 to T-38 of SEQ ID NO:2.<br>
9.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-31 to Y-39 of SEQ ID NO:2.<br>
10.	The in vitro produced polypeptide as claimed in claim 1. wherein said C35 peptide epitope is amino acids E-36 to A-43 of SEQ ID NO:2.<br>
11.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids A-37 to A-45 of SEQ ID NO:2.<br>
12.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids A-37 to V-46 of SEQ ID NO:2.<br>
13.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids Y-39 to V-46 of SEQ ID NO:2.<br>
14.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids S-44 to 1-53 of SEQ ID NO:2.<br>
15.	The   in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids A-45 to 1-53 of SEQ ID NO:2.<br>
16.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids G-52 to L-59 of SEQ ID NO:2.<br>
17.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-54 to T-62 of SEQ ID NO:2.<br>
18.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids S-57 to F-75 of SEQ ID NO:2.<br><br>
271<br>
19.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids R-58 to 1-67 of SEQ ID NO:2.<br>
20.	The   in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids G-61 to 1-69 of SEQ ID NO:2.<br>
21.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids G-63 to F-83 of SEQ ID NO:2.<br>
22.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-66 to L-73 of SEQ ID NO:2.<br>
23.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids E-66 to V-74 of SEQ ID NO:2.<br>
24.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids F-83 to E-103 of SEQ ID NO:2.<br>
25.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids D-88 to A-96 of SEQ ID NO:2.<br>
26.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids L-89 to A-96 of SEQ ID NO:2.<br>
27.	The   in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids A-92 to T-101 of SEQ ID NO:2.<br>
28.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids R-95 to L-102 of SEQ ID NO:2.<br>
29.	The   in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids A-96 to K-104 of SEQ ID NO:2.<br><br>
272<br>
30.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids K-104 to V-113 of SEQ ID NO:2.<br>
31.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids 1-105 to V-113 of SEQ ID NO:2.<br>
32.	The in vitro produced polypeptide as claimed in claim 1, wherein said C35 peptide epitope is amino acids 1-105 to 1-114 of SEQ ID NO:2.<br>
33.	A fusion protein comprising at least one C35 peptide epitope, wherein said at least one C35 peptide epitope is a peptide epitope selected from the group consisting of: amino acids S-9 to V-17, amino acids V-10 to V-l7, amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids S-21 to F-35, amino acids C-30 to T-38, amino acids E-31 to Y-39, amino acids E-36 to A-43, amino acids A-37 to A-45, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to 1-53, amino acids G-52 to L-59, amino acids E-54 to T-62, amino acids S-57 to F-75, amino acids R-58 to 1-67, amino acids G-61 to 1-69, amino acids G-63 to F-83, amino acids E-66 to L-73, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1-105 to M14, each of SEQ ID NO:2.<br>
34.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids S-9 to V-l 7 of SEQ ID NO:2.<br>
35.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids V-10 to V-17 of SEQ ID NO:2.<br>
36.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-16 to V-23 of SEQ ID NO:2.<br>
37.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-16 to R-24 of SEQ ID NO:2.<br><br>
273<br>
38.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-l 6 to 1-25 of SEQ ID NO:2.<br>
39.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids S-21 to F-35 of SEQ ID NO:2.<br>
40.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids C-30 to T-38 of SEQ ID NO:2.<br>
41.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-31 to Y-39 of SEQ ID NO:2.<br>
42.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-36 to A-43 of SEQ ID NO:2.<br>
43.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids A-37 to A-45 of SEQ ID NO:2.<br>
44.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids A-37 to V-46 of SEQ ID NO:2.<br>
45.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids Y-39 to V-46 of SEQ ID NO:2.<br>
46.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids S-44 to 1-53 of SEQ ID NO:2.<br>
47.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids A-45 to 1-53 of SEQ ID NO:2.<br>
48.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids G-52 to L-59 of SEQ ID NO:2.<br><br>
274<br>
49.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-54 to T-62 of SEQ ID NO:2.<br>
50.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids S-57 to F-75 of SEQ ID NO:2.<br>
51.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids R-58 to 1-67 of SEQ ID NO:2.<br>
52.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids G-61 to 1-69 of SEQ ID NO:2.<br>
53.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids G-63 to F-83 of SEQ ID NO:2.<br>
54.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-66 to L-73 of SEQ ID NO:2.<br>
55.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids E-66 to V-74 of SEQ ID NO:2.<br>
56.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids F-83 to E-103 of SEQ ID NO:2.<br>
57.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids D-88 to A-96 of SEQ ID NO:2.<br>
58.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids L-89 to A-96 of SEQ ID NO:2.<br>
59.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids A-92 to T-101 of SEQ ID NO:2.<br><br>
275<br>
60.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids R-95 to L-102 of SEQ ID NO:2.<br>
61.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids A-96 to K-104 of SEQ ID NO:2..<br>
62.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids K-104 to V-113 of SEQ ID NO:2.<br>
63.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids 1-105 to V-113 of SEQ ID NO:2.<br>
64.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is amino acids 1-105 to 1-114 of SEQ ID NO:2.<br>
65.	The fusion protein as claimed in claim 33, wherein said C35 peptide epitope is linked to a<br>
heterologous epitope.<br>
66.	An   in vitro produced C35 peptide epitope selected from the group consisting of: amino acids S-9 to V-17, amino acids V-10 to V-17, amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids S-21 to F-35, amino acids C-30 to T-38, amino acids E-31 to Y-39, amino acids E-36 to A-43, amino acids A-37 to A-45, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to 1-53, amino acids G-52 to L-59, amino acids E-54 to T-62, amino acids S-57 to F-75, amino acids R-58 to 1-67, amino acids G-61 to 1-69, amino acids G-63 to F-83, amino acids E-66 to L-73, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1-105 to 1-114, each of SEQ ID NO:2.<br>
67.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids S-9 to V-17 of SEQ ID NO:2.<br><br>
276<br>
68.      The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids V-10 to V-17 of-SEQ ID NO:2.<br>
69.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-16 to V-23 of SEQ ID NO:2.<br>
70.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-16 to R-24 of SEQ ID NO:2.<br>
71.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-16 to 1-25 of SEQ ID NO:2.<br>
72.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids S-21 to F-35 of SEQ ID NO:2.<br>
73.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids C-30 to T-38 of SEQ ID NO:2.<br>
74.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-31 to Y-39 of SEQ ID NO:2.<br>
75.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-36 to A-43 of SEQ ID NO:2.<br>
76.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids A-37 to A-45 of SEQ ID NO:2.<br>
77.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids A-37 to V-46 of SEQ ID NO:2.<br>
78.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids Y-39 to V-46 of SEQ ID NO:2.<br><br>
277<br>
79.       The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids S-44 to 1-53 of SEQ ID NO:2.<br>
80.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids A-45 to 1-53 of SEQ ID NO:2.<br>
81.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids G-52 to L-59 of SEQ ID NO:2.<br>
82.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-54 to T-62 of SEQ ID NO:2.<br>
83.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids S-57 to F-75 of SEQ ID NO:2.<br>
84.	The   in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids R-58 to 1-67 of SEQ ID NO:2.<br>
85.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids G-61 to 1-69 of SEQ ID NO:2.<br>
86.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids G-63 to F-83 of SEQ ID NO:2.<br>
87.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-66 to L-73 of SEQ ID NO:2.<br>
88.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids E-66 to V-74 of SEQ ID NO:2.<br>
89.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids F-83 to E-103 of SEQ ID NO:2.<br><br>
278<br>
90.      The  in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids D-88 to A-96 of SEQ ID NO:2.<br>
91.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids L-89 to A-96 of SEQ ID NO:2.<br>
92.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids A-92 to T-101 of SEQ ID NO:2.<br>
93.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids R-95 to L-102 of SEQ ID NO:2.<br>
94.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids A-96 to K-104 of SEQ ID NO:2.<br>
95.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids K-104 to V-113 of SEQ ID NO :2.<br>
96.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids 1-105 to V-113 of SEQ ID NO:2.<br>
97.	The in vitro produced epitope as claimed in claim 66, wherein said C35 peptide epitope is amino acids 1-105 to 1-114 of SEQ ID NO:2.<br>
98.	An in vitro produced polypeptide of not more than 100 amino acids in length comprising at least one C35 peptide epitope selected from the group consisting of: amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to 1-53, amino acids S-57 to F-75, amino acids G-63 to F-83, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1.-105 to 1-114, each of SEQ ID NO:2.<br><br>
279<br>
99.	The in vitro produced polypeptide as claimed in claim 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 90 amino acids in length.<br>
100.	The in vitro produced polypeptide as claimed in claim 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 80 amino acids in length.<br>
101.	The in vitro produced polypeptide as claimed in any of claims 1-32 or 98-100, or the fusion protein as claimed in any of claims 33-65, which is not more than 70 amino acids in length.<br>
102.	The in vitro produced polypeptide as claimed in any of claims 1-32 or 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 60 amino acids in length.<br>
103.	The in vitro produced polypeptide as claimed in any of claims 11-32 or 98, or the fusion protein a as claimed in any of claims 33-65, which is not more than 50 amino acids in length.<br>
104.	The in vitro produced polypeptide as claimed in any of claims 1 -32 or 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 40 amino acids in length.<br>
105.	The in vitro produced polypeptide as claimed in any of claims 1-32, or 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 30 amino acids in length.<br>
106.	The   in vitro produced polypeptide as claimed in any of claims 1-32 or 98, or the fusion protein as claimed in any of claims 33-65, which is not more than 20 amino acids in length.<br>
107.	A composition comprising the in vitro produced polypeptide as claimed in any of claims 1-32 or 98, or the fusion protein as claimed in any of claims 33-65, or the in vitro produced epitope as claimed in any of claims 66-97, and a pharmaceutically acceptable carrier.<br>
108.	A recombinant nucleic acid encoding the in vitro produced polypeptide as claimed in any of claims 1-32 or 98, or the fusion protein as claimed in any of claims 33-65, or the in vitro produced epitope as claimed in any of claims 66-97.<br>
The present invention discloses an in vitro produced polypeptide of not more than 80 amino acids in length comprising at least one C35 peptide epitope selected from the group consisting of: amino acids S-9 to V-17, amino acids V-10 to V-l 7, amino acids E-16 to V-23, amino acids E-16 to R-24, amino acids E-16 to 1-25, amino acids S-21 to F-35, amino acids C-30 to T-38, amino acids E-31 to Y-39, amino acids E-36 to A-43, amino acids A-37 to A-45, amino acids A-37 to V-46, amino acids Y-39 to V-46, amino acids S-44 to 1-53, amino acids A-45 to I-53; amino acids G-52 to L-59, amino acids E-54 to T-62, amino acids S-57 to F-75, amino acids R-58 to 1-67, amino acids G-61 to 1-69, amino acids G-63 to F-83, amino acids E-66 to L-73, amino acids E-66 to V-74, amino acids F-83 to E-103, amino acids D-88 to A-96, amino acids L-89 to A-96, amino acids A-92 to T-101, amino acids R-95 to L-102, amino acids A-96 to K-104, amino acids K-104 to V-113, amino acids 1-105 to V-113, and amino acids 1-105 to 1-114, each of SEQ ID NO:2.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="210840-a-quinazolinone-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="210842-novel-non-psychotropic-cannabinoids.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>210841</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/01363/KOL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>41/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Oct-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Oct-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Nov-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>UNIVERSITY OF ROCHESTER</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>OFFICE OF THE TECHNOLOGY TRANSFER ,518 HYLAN BUILDING , ROCHESTER,NY 14627-0140,U.S.A</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ZAUDERER MAURICE,</td>
											<td>44 WOODLAND ROAD, PITTSFORD, NY 14534, U.S.A</td>
										</tr>
										<tr>
											<td>2</td>
											<td>EVANS ELIZABETHE,</td>
											<td>52 HOLCROFT ROAD,ROCHESTER,NY 14618, U.S.A</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BORRELLO MELIDAA,</td>
											<td>95 CHADWICK DRIVE,ROCHESTER, NY 14618, U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12 N 15/12</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/10855</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-04-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/194,463</td>
									<td>2000-04-04</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/210841-an-invitro-produced-polypeptide-of-not-more-than-80-amino-acids-in-length-comprising-at-least-one-c35-peptide-epitope by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:30:48 GMT -->
</html>
