<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/210988-chimeric-expression-promoters-originating-from-commelina-yellow-mottel-virus-and-cassava-vein-mosaic-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:59:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 210988:CHIMERIC EXPRESSION PROMOTERS ORIGINATING FROM COMMELINA YELLOW MOTTEL VIRUS AND CASSAVA VEIN MOSAIC VIRUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CHIMERIC EXPRESSION PROMOTERS ORIGINATING FROM COMMELINA YELLOW MOTTEL VIRUS AND CASSAVA VEIN MOSAIC VIRUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to chimeric promoters comprising at least one nucleic acid sequence, derived from a first plant promoter comprising a plant vascular expression promoter region, said plant vascular expression promoter region being replaced with a nucleic acid sequence, derived from a second plant promoter and comprising a plant green tissue expression promoter region. The invention also relates to a process for manufacturing these promoters, and expression cassettes, vectors, and transgenic plants and seeds containing them.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
COMPLETE SPECIFICATION<br>
[See Section 10]<br><br><br><br>
"CHIMERIC EXPRESSION PROMOTERS ORIGINATING FROM COMMELINA YELLOW    MOTTEL    VIRUS    AND    CASSAVA    VEIN    MOSAIC    VIRUS"<br>
MERISTEM   THERAPEUTICS,   of   8   Rue   des   Freres   Lumiere,   F-63100 Clermont Ferrand,  France<br>
The following specification particularly describes the nature of the invention and the manner in which it is to be performed :-<br><br><br><br>
[DESCRIPTION]<br><br>
The present invention relates to chimeric expression promoters, intended in particular for use in the field of plant biotechnology.<br>
              Expression promoters are in general known in the field of       biotechnology and genetic manipulation. Insofar as plant biotechnology is more particularly concerned, the rate of expression of a gene" coding for a polypeptide that it is desired to produce in a host cell is often dependent on the promoter used. The problem with this is that the various promoters commonly used       are often limited to specific applications or tissues, simply<br>
because of their tissue specificity or expression strength. It is possible for example to cite the cauliflower mosaic virus promoter 35S as relatively strong promoter, compared to that for example originating from the the nos  gene, these two promoters being more<br>
     particularly used in the field of plant biotechnology. There thus exists a need for new promoters that are able to overcome the disadvantages described above of the application of the promoters known today. One attempt at solving this problem has been reported in PCT<br>
    application published under the number WO 97/48819, which<br>
describes"promoters derived from the cassava vein mosaic virus (CsVMV) , all of which comprise a portion of a nucleic acid sequence having 18 sequential nucleotides of which the homology to a reference sequence cited in the application is at least 80%.<br>
    The expressions used in the present description and claims have the following meaning unless indicated otherwise :<br>
	"nucleic acid" means DNA or RNA ;<br>
	"nucleic acid sequence" means a single or double stranded<br>
  oligomer or polymer, of nucleotide bases read from the 5" end<br>
towards the 3" end, and comprises self-replicating plasmids,<br>
genes, DNA or RNA polymers, infectious or non-infectious, and<br>
functional or non-functional DKA or RNA. In the nucleotide<br><br>
2<br>
   Rotation used" in the present application, and  unless otherwise indicated, the left end of a single stranded nucleotide sequence simple is the 5" end ;<br>
-	"derived nucleic acid sequence" means that the sequence derives<br>
   directly or indirectly from the sequence to  which it refers, for<br>
example by substitution, deletion, addition, mutation, fragmentation, and/or synthesis of one or more nucleotides ;<br>
-	"promoter" or "promoter nucleic acid sequence" means a region of<br>
nucleic acid upstream of the start codon for translation which is<br>
          directly implicated in the recognition and binding of RNA polymerase and other proteins necessary for transcription ;<br>
-	"plant promoter" is a promoter capable of initiating transcription in plant cells ;<br>
-	"constitutive promoter" is a promoter capable of expressing nucleic acid sequences operably linked thereto, in all or substantially all of the tissues of the host organism throughout -the entire development of said organism ;<br>
-	"tissue specific promoter" is a promoter capable of selectively expressing, nucleic acid sequences operably linked to said<br>
     "promoter, in certain specific tissues of the host organism ;<br>
-	"operably linked to" means the linking of the promoter, to the<br>
nucleic acid sequence or gene, coding for a polypeptide to be<br>
produced, such that the promoter positively drives transcription<br>
of said linked nucleic acid sequence. It should be understood that<br>
    the promoter sequence also includes transcribed sequences situated between the transcription initiation site and the translation start codon ;<br>
-	"expression cassette" means nucleotide seqenees capable of<br>
directing the expression of a nucleic acid sequence, or a gene,<br>
    coding for a polypeptilde to be produced in a host organism<br>
compatible with such sequences. Such expression cassettes comprise at least one promoter and a transcription terminator signal, and optionally other factors necessary or useful for expression ;<br>
-	"vector" means expression systems, for example projectiles<br>
coated with DNA," nuucleic acid based transit vehicles, nucleic acid<br>
molecules adapted for delivery of nucleic acid, and circular<br>
self-replicating autonomous DNA, for example plasmids, cosmids.<br>
5<br><br>
WO 00/58485<br>
3<br>
  Aphagemids, etc. If a recombinant micro-organism or cell culture is described as a host for an expression vector, this can also include circular extrachromosomal DNA (such as for example mitochondrial or chloroplast DNA), where DNA has been integrated     into the chromosome(s) of the host(s), the vector either being stably replicated with the cells during mitosis as an autonornpus structure, integrated into the host genome, or maintained in the nucleus or the cytoplasm of the host ;<br>
-	"plasmid" means a molecule of circular autonomous DNA capable of<br>
   replication within a cell, and.comprises both plasmids called<br>
"expression plasmids" and plasmids called "non-expression plasmids". If a recombinant micro-organism or cell culture is described in the present application as the host to an "expression  plasmid", this means both molecules of circular^ extrachromosomal    DNA and DNA having been integrated. into the host chromosome. If the plasmid is maintained in a cell host, the plasmid is either stably replicated with the cells during mitosis as an autonomous structure, or integrated into the host genome ;<br>
-	"heterologous sequence" or "heterologous nucleic acid sequence"<br>
means a sequence originating from a source, or a species, that is<br>
foreign to its natural environment, or where it comes from the<br>
same environment, has been modified with respect to its original .<br>
native form. The modification of the nucleic acid sequence can<br>
occur for example by treating the nucleic acid with a restriction<br>
   enzyme to generate a nucleic acid fragment capable of being operably linked to a promoter. The modification can also be carried out using techniques such as site specific or site<br><br>
directed mutagenesis ;<br>
-	"box" means a nucleic acid sequence to which a regulatory<br>
   function is attributed <br><br>
-	"like"ineans that the box and/or nucleic acid sequence to which<br>
the term refers, comprises a certain sequence identicality or<br>
consensus with a known reference box and/or nucleic acid sequence,<br>
and preferably of at least 50% sequence identicality, even more<br>
preferably a sequence identicality of at least 75%, and most<br>
preferably a sequence identicality of at  least  90% with the<br>
reference sequence. The percentage of sequence identicality is<br>
4<br><br>
WO 00/58485	gCT/IBnO/0037fl<br>
 calculated on the basis of a comparison window of at-least 6 contiguous nucleotide bases. The determination of a comparison window can be made by using sequence alignment algorithms in order  to determine an homology with a reference sequence, for example the local homology algorithm, the homology alignment algorithm, and the similitude search algorithm, these algorithms also existing in electronic or computerised form, under the names GAP, BESTFIT, FASTA et TFASTA. The percentage of sequence identicality is obtained by comparing the reference sequence with the box     and/or nucleic acid sequence ;<br>
-	"situated" means the position on a nucleic acid sequence of an<br>
identified element, such as a "box", a restriction enzyme site, or<br>
a codon having a particular function. The position given is<br>
indicated with a number that refers to the position of the start<br>
    of the element in the nucleic acid sequence, in the reading fram direction of the latter, that is to say, most frequently, and unless otherwise indicated from 5" to 3" ;<br>
-	"transgenic plant" means a plant having been obtained using<br>
genetic manipulation techniques, and covers whole plants obtained<br>
    thereby, their progeny, as well as vital plant organs, for example<br>
    roots, stems and leaves, obtained using these techniques. The transgenic plants according to the present invention can have different levels of ploidy, and can in particular be""polyploid, diploid, and haploid ;         - "propagule" means a structured or unstructured collection or<br>
assembly or association of plant cells" from which it is possible to regenerate a whole plant,   for example explants,   calli,   stems, leaves, roots, cuttings, and even seeds. The applicant of the present invention has taken a different<br>
    approach to that taken by the applicant of the previously<br>
discussed PCT patent application. Indeed, the present applicant has serendipitously succeeded in producing chimeric promoters capable of satisfying and fulfilling the need described   previously, and especially capable of increasing the rate of<br>
   expression of a gene or nucleic acid sequence coding for a<br>
polypeptide to be produced, in a host cell, and more particularly in a plant cell or regenerated plant, with respect to the existing<br><br>
WO 00/58485<br>
5 <br>
 Promoters most often used. Furthermore, the applicant has also succeeded in producing a complete family of promoters in order to be able to choose the promoter most suited to the envisaged task or application and the environment in which it will be put to   work, and thus making it possible to control to a certain extent, the expression rate of a gene to be expressed coding for a polypeptide to be produced.<br>
Consequently, one of the objects of the present invention is_a chimeric expression promoter comprising at least one nucleic acid  sequence, derived from a first plant promoter comprising a plant vascular expression promoter region, said plant vascular expression promoter region being replaced with a nucleic acid sequence derived from a second plant promoter and comprising a plant green tissue expression promoter region.  Preferably, the first plant promoter originates from the Commelina Yellow Mottle Virus (CoYMV) and the second plant promoter originates from the Cassava Vein Mosaic Virus (CsVMV) . Even more preferably, the promoter nucleic acid sequences originate from the intergenic regions of the first and second promoters.<br>
In a particularly preferred embodiment, the chimeric expression promoter comprises at least a part of a nucleic acid sequence identified under the number SEQ.ID01 fused to at least a part of of a nucleic"acid sequence identified under  the number SEQ,IDD2, In an even more preferred embodiment, the chimeric promoter according to the invention is selected from the group consisting of the nucleic acid sequences identified under the numbers SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06 and SEQ.1D07. According to another object of the present invention, the applicant has discovered that it was possible  to produce particularly active chimeric expression promoters by starting from a base promoter of viral origin, of which a part consists of an exogenic element that is capable of promoting plant green, tissue expression (GT) . Preferably, the exogenic GT promoter element is also of viral origin. Furthermore, and according to a preferred embodiment of this aspect of the invention, the promoter of viral origin originates from the Commelina Yellow" Mottle Virus (CoYMV).<br><br>
WO 00/58485	PCJ3BOO/0037Q<br>
 Preferably, the exogenic promoter element originates from the Cassava Vein Mosaic Virus" (CsVMV). Even more preferably, the exogenic GT element replaces the endogenic element capable of promoting vascular tissue expression (VT) of the promoter of viral     origin.<br>
According to a preferred embodiment of the previously defined objects of the invention recited above, the chimeric promoters further comprise at least one "endosperm like" box, and more preferably from 4 to 10 "endosperm like" boxes, and most preferably 6 "endosperm like" boxes.<br>
According to another preferred embodiment, the promoters -further comprise at least one" "asl like" box operably linked to the plant green tissue GT promoter element.<br>
According to yet  another preferred embodiment, the promoters of<br>
the present invention comprise at least one "asl" box operably<br>
linked to the plant green tissue (GT) promoter element.<br>
In another preferred embodiment, the promoters further comprise at<br>
least one "as2" box operably linked to the plant green tissue<br>
(GT)promoter element.<br>
    Preferably, the one or more of the "asl like", "asl", and "as2"<br>
boxes are operably linked upstream or downstream of the plant<br>
green tissue expression GT  promoter element.	<br>
Even more preferably, the one or more of the., "asl like", "asl", and "as2" boxes are operably linked in normal (5"&gt;3") or inverse (3"&gt;5") orientation.<br><br>
Most preferably, the promoter comprises at least one "as2/as2/as2" box in normal (5"&gt;3") or inverse (5"&gt;3") orientation. Preferably, the previously described promoter is selected from the group consisting of the nucleic acid sequences identified under the numbers SEQ.ID01, SEQ.ID02, SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID19, SEQ.ID20, SEQ.ID21", SEQ.ID22, SEQID.23, SEQ.ID24, and SEQ.ID25.<br>
Another object of the present invention is an expression cassette comprising at least one nucleic acid sequence, der-ived from a first plant promoter comprising a plant vascular expression promoter region, said plant vascular expression promoter region being replaced with a nucleic acid sequence derived from a second<br>
7<br><br>
wo 00/58485	PCjwmmrno<br><br>
plant promoter and comprising a plant green tissue expression promoter region, the promoter nucleic acid sequences being operably linked to a nucleic acid sequence, or gene, to be expressed coding for a polypeptide to be produced, the latter<br>
r<br>
 nucleic acid sequence or gene being operably linked to a transcription terminator nucleic acid sequence. Preferably, and according to this embodiment, said first plant promoter originates from the Commelina Yellow Mottle Virus (CoYMV) and said second plant promoter originates from the Cassava Vein Mosaic Virus (CsVMV) . Even more preferably, the expression cassette comprises at least a part of a nucleic acid sequence identified under the number SEQ.ID01 fused to at least a part of a nucleic acid sequence identified under the number SEQ.ID02. Even more preferably, the nucleic acid sequence of the chimeric    promoter of the expression cassette is selected from the group consisting of the sequences identified under the numbers SEQ.ID03, SEQ.ID04, SEQ,ID05, SEQ.ID06, SEQ.ID07, SEQ.ID19, SEQ.ID20, SEQ.ID21, SEQ.ID22, SEQ1D.23, SEQ.ID24, and SEQ.ID25. In accordance with another object of the present invention, there     is provided an isolated promoter nucleic acid sequence, selected from the group consisting of the sequences identified under the<br>
numbers SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07, ■SEQ.ID19, SEQ.ID20, SEQ.ID21, SEQ.ID22, SEQ2D,23, SEQ.ID24, and<br>
SEQ.ID25. ft$~  Yet another object of the invention relates to desoxynucleotide<br>
building blocks for the production of promoters or promoter<br>
nucleic acid sequences as defined above. These building blocks can<br>
be :<br>
- "directional" building blocks, that is to say sequences 30^ that read in the same direction as the reading, frame of the final<br>
promoter sequence, usually from the 5" end to the 3" end;" and/or  "guide" blocks, that is to say sequences of whose ends<br>
comprise nucleotide bases that overlap with the ends of the<br>
directional building blocks.<br>
In this way, and preferably, the directional building block<br>
corresponds to at least one sequence selected from the group<br>
consisting of the sequences  identified under the numbers SEQ.ID08,<br>
8<br><br>
WO 00/58485<br>
SEQ.ID09, SEQ.ID1Q, SEQ.ID11, SEQ.ID13, and SEQ.ID14. Moreover, it is preferred"to use desoxynucleotide "guide" corresponding to at least one sequence selected from the group consisting of the sequences  identified under the numbers SEQ.ID15,   SEQ,ID16, SEQ.ID17, and SEQ.ID18. Yet another object of the present invention is a vector comprising a promoter, or a promoter nucleic acid sequence, capable  of initiating transcription of a nucleic acid sequence, or gene, coding form a polypeptide to be produced, wherein the promoter or promoter nucleic acid sequence corresponds to a chimeric expression promoter or to a promoter nucleic acid sequence as described previously above.<br>
Preferably, the vector is selected from the group consisting of the binary vectors pMRT1152, PMRTH71, pMRT1172, pMRT 11857      pMRTH86, pMRT1187, pMRTH88, pMRT1182, pMRT1245, pMRT1246, pMRT1247, pMRT1248, pMRT1249, pMRT125Q, pMRT1251, pMRT1252,<br>
PMRT1253 and pMRTl254.<br>
Finally, another object of" the present invention is a process for the manufacture of a chimeric expression promoter or an isolated promoter nucleic acid sequence as described previously, wherein said process comprises the steps consisting of :<br>
 carrying out a ligation chain reaction, called LCR, to produce single stranded continuous DNA from at least one desoxynucleotide building block selected from the group consisting  of the "directional" desoxynucleotide building blocks SI, S2, S3, S4, S5, S6, and S7  identified under"the numbers SEQ.ID08, SEQ.ID09, SEQ.ID1Q, SEQ. ID11.,SEQ...ID12, SEQ.ID13 and SEQ.ID14 respectively, and. at least one "guide" desoxynucleotide building block for said promoter nucleic acid sequence or promoter selected from the group consisting of the guide desoxynucleotides Gl, G2, G3  and G4  identified under the numbers SEQ.ID15, SEQ.ID16, SEQ.ID17 et SEQ.ID18 respectively ;<br>
-	carrying out PCR.amplification on the single stranded DNA<br>
Obtained from the previous step to produce a double stranded DNA<br>
corresponding to the chimeric expression promoter or the promoter nucleic acid sequence ;<br>
-	optionally isolating the promoter of promoter nucleic acid<br>
9<br><br>
WO 00/58485	PCT/IB00/00370<br>
Sequence.<br>
Advantageously, and preferably, the desoxynucleotide building blocks are phosphorylated before ligation. Even more preferably, the ligation is carried out in the presence of at least one DNA - ligase in a thermocycle, under the following conditions :<br>
-	a cycle of about one minute at about 94°C ;<br>
-	eight identical cycles, each one consisting of the following steps :<br>
-	one minute at 65 °C, one minute at 57°C, one minute at 52°C,<br>
one minute at 48°C, one minute at 43°C and ten minutes at 370C.<br>
Yet another object of the present invention is a transgenic plant<br>
having stably integrated into its genome at least one promoter or<br>
at least one promoter nucleic acid sequence as defined previously.<br>
Preferably, the transgenic plant is selected from dicotyledonous<br>
species, and preferably potato, tobacco, cotton, lettuce, tomato,<br>
melon, cucumber, pea, rape, canola, beetroot, or sunflower, or<br>
from monocotyledonous species, and preferably wheat, barley, oat,<br>
rice, or corn,<br>
_Yet still another object of the present .invention, is a propagule of a transgenic plant as defined previously, preferably the propagule is a seed.<br>
According to the present invention, another object is a cell containing a promoter or promoter nucleic acid sequence as defined above, and preferably, the cell is a plant cell.<br>
-According to another object of the present invention, a method of expression of a nucleic acid sequence, or gene, is provided, that codes for a polypeptide to be produced, by the cell, wherein the method comprises the steps consisting of :<br>
-	transforming a cell with a vector comprising at least one promoter or at least one  promoter nucleic acid sequence as defined previously ;<br>
-	culturing the cell under conditions enabling expression of the nucleic acid sequence, or gene, coding for said polypeptide and production thereof. Preferably, the cell is a prokaryotic or eukaryotic cell, and more preferably is selected from the group consisting of microbial cells, algal and microalgal cells, fungal cells, insect cells, animal cells, mammalian and human cells, and<br>
10<br><br>
WO 00/58485<br>
plant sells, and most preferably is a plant cell.<br>
According to yet another object of the present invention, a method of manufacture is supplied for manufacturing a transgenic plant or a propagule wherein the method comprises the  steps consisting of :<br>
-	transforming a plant cell with a vector comprising at least one promoter or at least one promoter nucleic acid sequence as defined previously ;<br>
-	selecting the plant cell having integrated the promoter or promoter nucleic acid sequence ;<br>
-	propagated the selected and transformed plant cell, by culture, or by regeneration of whole chimeric or transgenic plants.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
The invention"will be better understood through the following "detailed description of one or more preferred embodiments, given purely as non-limiting examples, and  with referral to the annexed drawing in which :<br>
-	Figures I, II and III schematically represent the structures of the comparative reference constructs, enabling a comparison of the chimeric promoters of the present invention with those already known and used. In Figure I, the construction concerned contains the reporter gene coding for B-glucuronidase in the total absence of any  promoter sequence as  such, and thus useful as a negative control.<br>
-	Figure II schematically represents a construct containing the B-glucuronidase gene under the control of the CaMV double 35S promoter, useful as a strong reference control ;<br>
-	Figure III represents a construct useful as an internal reference for the transient expression experiments, and includes" the reporter gene coding for a luciferase under the control of the CaMV 35S promoter ;<br>
-	Figure IV schematically, represents, the structure of several preferred embodiments of chimeric promoters produced in accordance with the present invention. The chimeric promoters" MPrlll6 and MPrlll7 were obtained using the technique called 1b-PCR. MPrll46 and MPrll47 were obtained by cloning the activator elements asl and as2 from the CaMV promoter at the restriction enzyme site<br>
11<br><br>
WO 00/58485<br>
DraISI. The promoter MPrll54 was obtained through deletion of the two "as-1 like" sequences" from the CoYMV promoter present in the 5" region of MPrll47. All of these promoters were cloned at the restriction sites PstI and BamHI into the vector pMRTH44 in order   to obtain transcriptional fusion with the reporter gene uidA ; -  Figure V represents histochemical staining of tobacco leaves transformed with different promoters in accordance with the present invention- The tobacco  leaves were transformed by a biolistic method using a «  PDS1000/He » gene gun available from<br>
-—	ir<br>
BIORAD, under the following conditions : split rupture disks at " 900 psi, 2 ug of DNA bombarded over two successive firings, projectiles consisting of gold beads or spheres of about 1 urn in ■ diameter, plant material positioned at 6 then 9 cm from the macrocarrier. After bombardment, the leaves were incubated in the dark in a culture chamber for 48 hours to enable expression of the reporter gene. The leaves were then. incubated in a 0.1 M phosphate buffer containing 2 mg/ml of X-Glu at 37°C for 24 to 48  hours, then bleached in a 70% ethanol bath.<br>
-	Figure VI represents a graph comparing the relative promoter activity of the different constructs after transient expression in tobacco leaves. Three days after bombardment the leaves were ground then the crude extract clarified by. centrifugation. The B-glucuronidase and luciferase activities were measured using fluorimetric methods on crude extract aliquots, then the ratio of GUS activity / LUC activity was determined. The histograms correspond to the  average of the ratios for a given construct +/- standard mean error ;<br>
-	Figure VII schematically represents other preferred embodiments of chimeric promoters according to"the present invention, where :<br>
the dark disk-shaped symbols represent the green tissue expression specific element ;<br>
the small white parallelepiped symbols represent the "endosperm like" boxes ;<br>
the small and large black hatched parallelepipeds represent respectively the "as-2" and "as-1" boxes from the CaMV promoter.<br>
-	Figure VIII represents a comparison of the relative"<br>
2<br><br>
WO 00/58485	PCKID00/00370<br>
"activity of the different promoters of the invention in transient "expression experiments in corn albumen, where the p-glucuronidase<br>
and luciferase activities were measured by  fluorimetry on an ■aliquot of crude extract., The histograms correspond to the average a given construction +/- standard mean error ; - Figure IX represents a comparison of the relative activity of the chimeric promoters MPrlll6, MPr1146, MPr1167 and reference promoter MPrl092, evaluated in stable tobacco expression. Samples were taken from each  primary transformant at 2,   4, 6, 8, and 10    weeks after transfer of the plants into the greenhouse. The<br>
p-glucuronidase activity was measured on each sample and weighted<br>
in relation to the total quantity of total protein. For each<br>
series of transformants, at a given time, the activities are<br>
classed in decreasing order and compared ;<br>
	-  Figure X-represents a comparison of the relative activity<br>
of the chimeric promoters MPrll62, MPrll64, MPrll65, MPril67"and the reference promoter MPrl092, evaluated in stable tobacco expression. Samples were taken from each primary transformant at 2, 4, 6, 8, and 10 weeks after transfer to"the greenhouse. The       p-glucuronidase was measured on each sample and weighted in relation to the total quantity of protein. For each series of transformants, at a given times, the activities were.classed in decreasing order and compared.<br>
In the various figures, certain terms have the following     meanings :<br>
-	uidA - the sequence coding for B-glucuronidase ;<br>
-	IV2 = the patatin gene intron ;<br>
-	nos term = the terminator from the Nopaline Synthase  gene ;<br>
-	35S term = the RNA 35S CaMV terminator ;    <br>
           -  CaMV - the cauliflower mosaic virus ;<br>
-	as-l = activating sequence 1 from the CaMV 35S promoter ;<br>
-	as-2 = activating sequence 2 from the CaMV 35S promoter ;<br>
-	B = the endonuclease restriction site BamHI ;<br>
  =  the endonuclease restriction site EcoRI ;<br>
35    -  H = the endonuclease restriction site HindIII ;<br>
-	P = the endonuclease restriction site PstI ;<br>
-	Sp = the endonuclease restriction site Sphl.<br>
13	<br><br>
wooo/58485	PCT/IB00/00370<br>
-	D = the endonuclease restriction site Drain ;  IN  = the endonuclease restriction site Ndel ;<br>
-	S = the endonuclease restriction site Spel ; -" CoYMV = the Commelina Yellow Mottle Virus ;<br>
CsVMV = the Cassava-Vein Mosaic Virus ;<br>
-	TATA = the TATA box;<br>
-	+1 = the transcription initiation site ;<br>
-	"like" means that the sequence is not 100% homologous to the sequence  to which it refers, as  defined previously.<br>
E3PMPLES<br>
Example 1<br>
Comparative Constructs (controls)<br>
In order to enable the comparison between the chimeric promoters    of the present invention, and those known and currently used, the<br>
uidA  gene coding for B-glucuronidase (Jefferson ef al.r   1986) and<br>
i<br>
containing the intron IV2 sequence from the potato patatin gene ST-LS1   (Vancanneyt et al.r   1990) (uidA-IV2) was placed under the control of one of the promoters and the terminator from the<br>
   nopaline synthase gene [nos  term) from AGROBACTERIUM tumefaciens into the plasmid pGEM3Z commercialised by Promega Corp. (Madison, USA) .<br>
1.1. Construction of negative control pMRT1144,  In order to facilitate cloning, a plasmid derived from pGEM3Z,<br>
25 containing only the sequences "uidA-IV2/nos  term" and  lacking any promoter sequence was produced. This plasmid was designated pMRTll44 and served as the negative control (Fig. 1). In order to insert the uidA/nos  term sequence into pGEM3Z,. the uidA  sequence under"the control of the entire promoter from the pea plastocyanin gene and the nopaline synthase terminator, were<br>
isolated from 5 jig of plasmid pGA492-PpetE, This plasmid had been obtained by cloning, into the plasmid pGA4 92-Pem2-uidA,   the petE promoter originating from the pea plastocyanin gene from the /plasmid pKHn2 (Pwee et Gray, 1993) instead of the em2  promoter ly    (Gaubier et al.t   1993), originating from the plasmid bp<br>
I221-Pem2. The plasmid bp I221-Pem2 was digested with 20 units of each of the enzymes HindIII and EcoRI for 1 hour at 37 °C. Then,<br>
14<br><br>
"WO 00/58485	J£CIZ1B0O#O37O<br>
the expression cassette nPem2/uidA/nos  term" was separated by electrophoresis on 0,8%  agarose gel, electroeluted, precipitated in the presence of 1/10 volume 3M sodium acetate pH 4.8 and 2.5 volumes of absolute ethanol at -80°C for 30 minutes, centrifuged at 12000 g for 30 min, washed in 70% ethanol, dried, resuspended in water and inserted at the HindIII and EcoRI sites of the ■ plasmid pGA492 (An, 1986) previously digested by these two enzymes for 1 h at 37°C, precipitated in the presence of 1/10 volume 3M sodium acetate pH 4,8 and 2.5  volumes absolute ethanol at -80°C     for 30 min, centrifuged at 12000- g for 30 min, washed in 70%<br>
ethanol, dried and then resuspended in water. Ligation was carried out in the presence of 1.0ul T4 10X DNA ligase buffer (Amersham) and 2.5 unites of T4 DNA ligase (Amersham) at 14°C for 16  h. Previously prepared viable and competent Escherichia  coli  DH5a,     bacteria were transformed (Hannahan, 1983). .The,plasmid DNA of the obtained clones, selected on Luria-Bertani media (LB, bactotryptone 10 g/1, yeast extract 5 g/1, NaCl 10 g/1/ Agar 15 g/1) supplemented with tetracycline {12 mg/1), was extracted according to the alkaline lysis method (Birnboim et Doly, 1983)    and analysed by enzymatic digestion.<br>
*       Starting from the obtained pGA492-Pem2-uidA plasmid, the promoter ?em2  was deleted by double digestion with Hindlll and XbaI. The plasmid fragment was separated by electrophoresis on 0.8% agarose gel, electroeluted, precipitated in the presence of 1/10 volume 3M  sodium acetate pH 4.8 and 2.5 volumes absolute ethanol at -80°C   for 30 min, centrifuged at 12000 g for 30 min, washed in 70%<br>
ethanol, dried, subjected to a Klenow fragment of DNA polymerase I (New England Biolabs) for 30 min at 37°C according to the manufacturer"s recommendations. Then, it was deproteinised by 30 extraction a volume of phenol, then a volume of phenol :<br>
chloroform : isoamyl alcohol {25:24:1 v/v/v) and finally a volume of chloroform : isoamyl alcohol (24:1 v/v), precipitated in the presence of 1/10 volume 3M sodium acetate pH 4,8 and 2,5 volumes absolute ethanol at -80°C for 30 min, then centrifuged at 12000 g for 30 min, washed in 70% ethanol, dried, resuspended in water. Then, it was dephosphorylated for 1 h at 37°C using 10 units of calf intestine alkaline phosphatase (Boehringer Mannheim)<br>
15<br><br>
WO 00/58485	PCT/IDO 0/0 0370-<br>
according to the manufacturer"s recommendations, deproteinized by fetraction with a volume of phenol, then a volume of phenol : chloroform : isoamyl alcohol (25:24:1 v/v/v) and finally a volume of chloroform : isoamyl alcohol (24:1 v/v), precipitated in the     presence of 1/10 volume 3M sodium acetate pH 4.8 and 2.5 volumes absolute ethanol at -80°C for 30 min then centrifuged at 12000 g for 30 min,   washed in 70% ethanol, dried then resuspended in . ,  water. The resulting plasmid was designated pGA492APem2.<br>
In.parallel, the promoter petE  (818 bp J which corresponds to the \v    promoter, of the pea plastocyanin gene, was obtained from the<br>
plasmid pKHn2 by digestion with. Ncol for 1 h at 37°C. The 828  bp promoter fragment was isolated on 0.8% agarose gel, electroeluted, precipitated in the presence of 1/10 volume 3M sodium acetate pH ,   4.8 and 2.5 volumes absolute ethanol at ^80°C for 30 min, ¥5    centrifuged at,12000 g for 30 min, washed in 70% ethanol, dried, resuspended in water, then subjected to the action of 5 units of Mung Bean nuclease (New England Biolabs) for 30 min at 30QC according to the manufacturer"s recommendations , deproteinized by extraction with, a "volume of phenol, then a-volume of phenol : chloroform :■ isoamyl alcohol (25:24:1 v/v/v) and finally a volume of chloroform : isoamyl alcohol (24:1 v/v), precipitated in the presence of 1/10 volume 3M sodium acetate pH 4.8 and 2.5 volumes absolute ethanol at -80°0  for 30 min then centrifuged at 12000 g for 30 min, washed in 70% ethanol, dried then resuspendu in water. 25 This promoter fragment was inserted into the plasmid pGA492APem2,<br>
described above, in the presence of 1.0 ul T4 10X DNA ligase buffer (Amersham) and 2.5 units of T4 DNA ligase (Amersham) at 14°C for 16 h. Previously prepared viable and competent Escherichia  coli DHSot bacteria were transformed. The plasmid DNA of the obtained clones, selected on LB media supplemented with tetracycline  (12 mg/1), was extracted according to the alkaline lysis method and analysed by enzymatic digestion . The resultant plasmid was designated pGA4 92-pet£ prom. In order to isolate the expression cassette "pet£ prom / vidA /<br>
Jo    nos  term", 5 u-g of plasmid pGA492-pet£ prom were digested with PstI (site situated in the 5" region of the" promoter of the pea<br>
16<br><br>
WO 00/58485	QZHWMQ37Q<br>
16<br>
plstocyanin gene) and EcoRl (site situated in the 3" region of   the terminator sequence) for 1 h at 37°C, subjected to 0.8 agarose gel electrophoresis and purified on a QIA quick affinity column (Qiagen, Hilden, Germany) according to the recommendation of the     supplier.Furthermore, 500 ng of the plasmid pGEM3Z  were, simultaneously digested for 1 h at 37°C with EcoRI and Pstl (restriction sites present in the multiple cloning site or polylinker), subjected to 0.8% agarose gel electrophoresis, then purified on a QIAquick affinity column.    The ligation was carried out with 50 ng of vector<br>
pGEM3Z-PstI/EcoRI and 50 ng of the expression cassette petE  prom /<br>
uidA / nos  term for 1 night at 18°C in a reaction volume of 12 jl in the presence of 1.2 u-1 T4 10X DNA ligase buffer (New England &gt; Biolabs) and 4 00 units of T4 DNA ligase (New England Biolabs) -   Previously prepared Escherichia coli DH5a  bacteria were<br>
transformed with the ligation reaction mixture. The  plasmid DNA of the obtained clones, selected on LB  media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and" analysed by enzymatic digestion. The plasmid KT~ obtained was designated pGEM3Z-pe£E prom.<br>
In order to insert the- 192 bp IV2 intron from the potato patatin gene into the coding uidA  sequence, an internal portion of this gene (fragment SnaBl / BstBI of 710 bp in pGEM3Z-petE prom) was excised then replaced with the equivalent" sequence containing the     iv2  intron (fragment SnaBI / BstBI of 902 bp ) . In order to<br>
achieve this, the plasmid pGEM3Z-pete prom was digested for 1 h at 37°C with SnaBI (restriction site situated at position -+383 bp downstream of the initiator ATG codon of the uidA gene) then for 1 h at 65°C with BstBI {site situated at position +1093 bp). The 30 plasmid deleted of this 710 bp fragment was isolated by 0.8% gel agarose electrophoresis, then purified on a QIAquick affinity column. The fragment BstBI/SnaBl of 902  bp corresponding to the IV2 intron sequence followed by the uidA  sequence stretching from .position +383 to +1093 bp, was1 isolated and purified from the 3D "plasmid pSCVl-2-GI. This plasmid derives from the plasmid pSCVl.2 which in turn derives from the plasmid pSCVl constructed by G.A. Edwards in 1990 according to the usual cloning methods. The binary<br>
17<br><br>
WO00/58485	rcxroggorat<br>
plasmid pSCVl.2 was obtained through cloning of the fragment jjfcnd"lll tearing the expression cassette "35S  prom / nptll / nos term" (Fromm et al.t   1986) at the Hindlll site of pSCVl,  The expression .cassette "35S prom / GUS-IV2 / 35S term" was obtained y^oy  digesting the plasmid p35S GUS INT with Hindlll for 1 h at 37 °C as described by Vancanneyt et 3.1.   (1990). The DNA fragment corresponding to the expression cassette was isolated on 0.8 % agarose gel, electroeluted then precipitated in the presence of 1/10 volume 3M sodium acetate pH 4.8 and 2.5 volumes absolute 10- ethanol at -80°C for 30 min then, centrifuged at 12000 g for 30 min, washed in 70 %  ethanol, dried and resuspended in water. The 51 protruding ends of this, fragment were blunted by the action of the DNA polymerase I Klenow fragment (New England Biolabsj for 30 min at 37°C according to the manufacturer"s recommendations , and l^~the fragment was deproteinized by extraction with a volume of phenol, then a volume of phenol : chloroform : isoamyl alcohol (25:24:1 v/v/v) and finally a volume of chloroform : isoamyl alcohol (24:1 v/v), precipitated in the presence of 1/10 volume 3M sodium acetate pH 4,8 and 2,5 volumes absolute ethanol at -80"C 20"~ for 30 min then, centrifuged at 12000 g for 30 min, washed in 70% . ethanol,. dried and finally ligated with 20 ng of plasmid pSCV1.2,<br>
digested-with Smal for 1 h at 25°C, in the presence of 1.0 \il  T4<br>
*<br>
10X DNA ligase buffer (Amersham) and 2.5 units of T4 DNA ligase (Amersham) at 14°C for 16 h. Previously prepared competent and viable Escherichia coli  DH5abacteria were transformed. The plasmid DNA of the obtained clones,   selected on LB media supplemented with ampicillin (50 mg/1), was  extracted according to the alkaline lysis method and analysed by enzymatic digestion . Five micrograms (5 jig) of the plasmid pSCV1.2-GI were digested for<br>
    1 h at 37°C with SnaBI (restriction site situated at position +383 bp  downstream of the initiator ATG codon of the gene uidA)   then for 1 h at 65°G with" BstBl (site situated at position +1285 bp) . The 902 bp fragment was  isolated by  1.0% agarose gel electrophoresis, then purified on a QIAquick affinity column .<br>
35 The  ligation was carried out with 20 ng of vector pGEM3z-pet£ prom BstBI/SnaBI and 80 ng of the 902 bp fragment BstBI/SnaBI, for 1 night at 18 °C in -a reaction volume of 10 u.1 in the presence of 1.0<br>
18<br><br>
WO 00/58485	PCT/IBOO/00370<br>
ul of T4 10X DNA ligase buffer (New England Biolabs) et 400 units 6f T4  DNA ligase (New England Biolabs). Previously prepared competent and viable Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The     plasmid DNA of the obtained clones, selected on LB media<br>
supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and. was analysed by enzymatic digestion. The plasmid obtained was  designated pGEM3Z-petE  prom / IV2. In order to eliminate the promoter sequence corresponding to the     818 bp fragment (petE)   from the plasmid pGEM3Z-petE prom / IV2, the latter was  digested for 1 .h at 37°C with BamHI then, for 1 h at 37°C with PstI, isolated by electrophoresis on 0.8% gel agarose, then purified on  a QIAguick affinity column. The protruding 5" ends of this, plasmid were blunted with Pfu DNA     polymerase (Stratagene, La Jolla, USA)  according to the<br>
supplier"s recommendations. The ligation was carried out with 10 ng of thus modified plasmid for 1 night at 18°C in a reaction volume of 12 ul   in the presence of 1.2 ul of T4 10X DNA ligase buffer [New England Biolabs) and 400.units"of T4 DNA ligase (New     England Biolabs). Previously prepared competent and  viable<br>
Escherichia coli DH5a bacteria were transformed with half the reaction mixture of ligation. The plasmid DNA of the. obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was" extracted according to the alkaline lysis method,      analysed by enzymatic digestion, et verified by sequencing<br>
according to the method described by Sanger et al. (1977). The plasmid obtained was designated pMRT1144 (Fig. I). 1.2. Construction of the positive control MPrl092. In order to obtain a reference promoter sequence, the "double 35S"     promoter from the cauliflower mosaic virus (CaMV D35S prom), was placed upstream of the sequence uidA-XV2/ nos  term. The plasmid pMRT1092 (Fig. I) resulted from the following cloning steps : First of all, the 192 bp IV2 intron from the potato patatin gene ras inserted into the coding sequence uidA  at position +383 bp as described in section 1.1. A one microgram amount (1 ug) of plasmid. pBI221 (Clontech, CA, USA) was digested for lh30 at 37°C with<br>
19<br><br>
WO00/5S485	i_P^aaflQZQfi370<br>
19<br>
SnaBI then for lh30 at 65 °C with BstBI, The plasmid deleted of the 71% bp fragment was isolated by 0.8% agarose gel electrophoresis , then purified on a QIAquick affinity column.<br>
A twenty nanogram amount (20 ng) of the vector pBI221 BstBI/SnaBI    and 80 ng of the 902 bp fragment BstBI/SnaBI originating from  pSCVl.2-GI as  described previously, were ligated for 1 night at 18 °C  in a reaction volume of 10 u1, in the presence 1 ul of T4 10X DNA ligase buffer (New England Biolabs) and 4 00 units T4 DNA ligase (New England Biolabs) Previously prepared competent and     viable Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The DNA Of the obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method and analysed by enzymatic digestion. The plasmid obtained was designated  pBI221/iudA-IV2.<br><br>
Next, the sequence of the simple 35S CaMV promoter present in the plasmid pBl221/uidA-IV2 was replaced with the sequence "CaMV D35S". In order to achieve, this, the plasmid pBI221/uidA-IV2 was digested for.l0h30 at 37°C with 10 units o"f Hindlll, then the sticky ends were blunted by the action of the Klenow fragment of DNA polymerase I {New England Biolabs) for 30  min at 37°C according to the manufacturer"s recommendations. After purification of the product of this reaction on a QIAquick affinity column, the DNA was digested for a night at 37°C with 10     units of BamHI, The plasmid fragment, corresponding to the vector deleted of the 828 bp CaMV 35S promoter fragment, was isolated by 0.8% agarose gel electrophoresis, then purified on a QIAquick affinity column .<br>
The CaMV D35S promoter was obtained from the  plasmid pJIT163A.     This plasmid derives from pJITl63 which in turn derives from the plasmid pJIT160 (Guerineau et Mullineaux, 1993). The plasmid pJIT163 possesses a ATG codon between the sites HindiII and Sall of the polylinker. in order to delete this ATG and obtain the /plasmid pJIT163A, the plasmid DNA of pJIT163 was digested with     Hindlll and Sail, purified by 0.8% agarose gel electrophoresis , electroeluted, precipitated in the presence of 1/10 volume 3M sodium acetate pH 4,8 and 2.5  volumes absolute ethanol at -80°C<br>
20<br><br>
WO 00/58485	PCT/JBQmf&amp;70<br>
gel, prified on a QIAquick affinity column and dephosphorylated for l h at 37°C in the presence of 12 ul  of 10X buffer 3 (New England Biolabs) and 5000 units of calf intestine alkaline phosphatase (CIP, New England Biolabs), and finally purified once    again on a QIAquick affinity column.<br>
The  fragment Spel/Dralll of 70 bp containing the two as-2 elements and the as-1 element was  obtained from the plasmid pMRTllll. This latter plasmid, which contains a 58 bp sequence corresponding to a duplication of the as-2 element {Lam et Chua, 10 1989) and an as-1 element (Lam et al.,   1989) originating from the 35S RNA CaMV promoter upstream of the minimal pea plastocyanin promoter modified by the addition of a "G" box, was obtained by lb-PCR in the following manner. The single stranded continuous DNA was generated using the following "directional" desoxynucleotides 15  :<br>
-	SI = 5"<br>
TTCCCTTCAAACACATACAAATTCAGTAGAGAA.GAAACTCATTACTCTTGAGAAACCTAGAGGATC CCCG 3"    (SEQ.ID08)<br>
       -  S2 = 51<br>
CACAAAAACCCAATCCACATCTTTATCATCCATTCTATAAAAAATCACCTTCTGTGTGTCTCTCTT-TCGA 3"    (SEQ-ID09)<br>
-	S5  =  5"<br>
CTGTGGCACATCTACATTATCTAAATCTAAGCCACGTCGGAGGATAACATATTCTTCCACACATCT          TAGCCA 31  (SEQ.ID12)<br>
-	S7  =  5"<br>
CATGCTGCAGACTAGTGATTGATGTGATATCAAGATTGATGTGATATCTCCACTGACGTAAGGGAT GACGCATGCCACT 31 (SEQ.ID14)<br>
One hundred picomoles (100 pmol) of the desoxynucleotides S1, S2 30 and S5 were 51 phosphorylated with 15 units of kinase (Amersham) in the presence of 5 ul 10X kinase buffer (Amersham) and 500 pmol of ATP (Sigma), for 30 min at 37°C,  The phosphorylated oligodesoxynucleotides were purified by extraction with a volume of phenol, then a v.olume of phenol : chloroform : isoamyl alcohol 35  (25:24:1 v/v/v) and finally a volume of chloroform : isoamyl<br>
alcohol (24:1 v/v), before being precipitated with 1/10 volume 3M sodium acetate pH. 4.8 and 2.5 volumes absolute ethanol at -8000<br>
21<br><br>
WO 00/58485	PCT/IB00/0D370<br>
27<br>
for 20 min then    centrifuged at  16060  g for  30 min.   The preci-pitated oligodesoxynucleotides were washed in 70%.  ethanol,<br>
dried,   then resuspended in water at a concentration of  10 pmol/ul. In order to link the  "directional"  oligodesoxynucleotides,   the following  "guide" oligodesoxynucleotides were used   :<br>
-	Gl- 5" TGTGTTTGAAGGGAATCGAAAGAGAGACACA 3" (SEQ.ID15)<br>
-	G2= 5" GATTGGGTTTTTGTGTGGCTAAGATGTGTG 3" [SEQ.ID16)<br>
-	G4= 5" TGTAGATGTGCCACAGAGTGGCATGCGT 3" (SEQ.ID18)<br>
In order to carry out the LCR reaction, 10 pmol of the phosphorylated directional desoxynucleotides si, S2, S5  and S7 were ligated in the presence of 10 pmol of the "guide"<br>
desoxynucleotides Gl, G2 and G4t   5 ul Taq 10X DNA ligase buffer (New England Biolabs) and 40  units Taq DNA ligase (New England Biolabs) , The ligation reaction was carried out in a GeneAmp PCR System 9700 thermocycle (Perkin Elmer, Norwalk, USA)". It consists of a cycle of 1 min at 94°C, and 8 identical cycles estch . consisting of the succession of the following steps : 1 min at 65°C, 1 min at 57°C, 1 min at 52°C, 1 min at 48°C, 1 min at 43&amp;C, and finally 10 min at 37°C. Then, the ligation reaction mixture was purified on a QIAquick affinity column according to the supplier"s recommendations.<br>
Finally, PCR amplification of the single stranded  continuous DNA obtained was  carried out in a GeneAmp PCR System 97 00 thermocycle in the presence of 100 pmol of each of the following oligodesoxynucleotide probes 5" CATGCTGCAGACTAGTGGATT 3", and 5" CGGGGATCCTCTAGGTTTCT 3", 50 nmol of each of the dNTP, 10 \ul  of Vent 10X DNA polymerase buffer (New England Biolabs), and 2 units of Vent DNA polymerase (New England Biolabs) . The DNA was denatured for 5 min at 94°C,   subjected to 25 cycles, each consisting of a 30 sec denaturing step at 95°C, a" 30 sec hybridisation step at 56°C, and a 1 min elongation step at 72°C, then further elongation at 72°C for 5 min.<br>
The DNA fragments of the reaction mixture were digested for 45 min at 37°C with 20 units of BamHI, then for 1 h at 370C with 20 units of PstI, and finally purified on a QIAquick column.  They were inserted into the plasmid pGEM3Z-petE prom, digested  for 1 h at 37°C with BamHI, then for 1 h at 37°C with PstI, subjected to 0.8%<br><br>
WO 00/58485	PCT/TB00/0037O<br>
28<br>
agatose gel electrophoresis,  purified on a QIAquick affinity Column, dephosphorylated for 1 h at 37°C in the presence of 12 ul of 10X buffer 3 {New England Biolabs) and 5000 units of calf intestine alkaline phosphatase (CIP, New England Biolabs), and 5 finally purified on a QIAquick affinity column. In order to carry out the ligation, 25 ng of plasmid treated as described above were contacted with 100 ng of the DNA fragments obtained by PCR,  in<br>
the presence of 1,2 ul of T4 10X DNA ligase buffer (New England Biolabs) and 400 units T4 DNA ligase {New England Biolabs) for 1 10 night at 18°C. Previously prepared viable and competent<br>
Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The DNA of the obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was extracted according to the aikaline lysis method and was analysed 15    by enzymatic digestion. The promoter sequence  MPrllll of  one of these clones was verified by  sequencing.<br>
The 70 bp fragment containing the two as-2 elements and the as-1 element was obtained by digesting 25 ug of plasmid pMRTllll with 40 units of Spel for 1 hat 37°C, then with 4 units of Drain for 20 1 h at 37 °c. The fragment was isolated by electrophoresis on Nu-Sieve 3% gel agarose (EMC, Rockland, USA) and finally purified on a,QIAquick affinity column.<br>
The ligation was carried out with 30 ng of dephosphorylated plasmid vector pMRT1116 Nhel/DraIII and 50 ng of the 70 bp 25 fragment for 15 h at 18°C in a reaction volume of 20. ul in the presence of 2.0 ul T4 10X DNA ligase buffer (New England Biolabs) and 800 units T4 DNA ligase (New England Biolabs). Previously prepared viable and competent Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The 30 plasmid DNA of the obtained clones, selected on LB media<br>
supplemented with ampicillin {50 mg/1), was extracted according to alkaline lysis method, and analysed by enzymatic digestion and gene amplification using the desoxynucleotides 5" -CATGCTGCAGACTAGTATCC 31 selected on the promoter and 5" 35 TTGATTTCACGGGTTGGG 31 selected on the uidA sequence. The promoter sequence MPrI14 6 (SEQ.ID05) of one of these clones was verified by<br><br>
WO 00/58485	PCT/IB00/00370<br>
"	29<br>
sequencing.<br>
2   Construction of the promoter MPrll47 :<br>
The promoter MPrll47 (Fig. IV) was obtained by inserting a sequence of 44 bp from the RNA 35S CaMV promoter  containing the 5 elements as-2 and as-1 (Lam, 1989 ; Lam"et al.   1989) and<br>
restriction sites adjacent to sites NheI and DraIII of MPrlll7 {SEQ.ID04).<br>
In order to achieve this, the plasmid pMRT1117 was digested with 25 units of Nhel for 1 h at 37 °C then purified on  a QIAquick 30 affinity column. The ends generated were blunted by the action of Pfu DNA polymerase (Stratagene, La Jolla, USA)  according to the supplier"s recommendations, then the plasmid obtained was digested for 1 h at 37°C with 4 units of Drain. The thus modified plasmid was  isolated on 0,8% agarose gel, " purified on a QIAquick 15 affinity column and dephosphorylated for 1 h at 37 °C  in the<br>
presence of 12 ul of 10X buffer 3 (New England Biolabs) and 5000 units of calf intestine alkaline phosphatase (CIP, New England Biolabs), and finally purified again on a QIAquick affinity column. 20 The 54 bp fragment Pstl/Dralll containing the 44 bp of the as-2 and as-1 elements of the CaMV 35S was obtained from plasmid pMRTlllO. The plasmid pMRTlllO, which contains a  sequence of 44 bp corresponding to the as-2 element (Lam et Chua, 1989) and the as-1 element (Lam et al.,  1989) of the 35S RNA CaMV promoter 25 upstream of the minimal pea plastocyanine promoter modified by the addition of a "G" box, was obtained by Ib-PCR in the following manner. The single stranded continuous DNA was formed using the "guide" desoxynucleotides :<br>
-	SI  =  5"<br>
30     TTCCCTTCAAACACATACAAATTCAGTAGAGAAGAAACTCATTACTCTTGAGAAACCTAGAGGATC CCCG 3"    (SEQ.IDG8)<br>
-	S2  =  51<br>
CACAAAAACCCAATCCACATCTTTATCATCCATTCTATAAAA7AATCACCTTCTGTGTGTCTCTCTT■ TCGA 3"    (SEQ.ID09)<br><br>
35    - S5 = 5"<br>
CTGTGGCACATCTACATTATCTAAATCTAAGCCACGTCGGAGGATAACATATTCTTCCACACATCT TAGCCA 3"    (SEQ.ID12)<br>
24<br><br>
WO 00/58485	PCT/IB00/00370<br>
30<br><br>
"CATGCTGCAGACTAGTGGATTGATGTGATATCTCCACTGACGTAAGGGATGACGCATGCCACT<br>
31 (SEQ.ID13)<br>
One hundred picomoles {100 pmol) of the desoxynucleotides SI, S2 5 and S5 were phosphorylated in 5" by 15 units, of kinase (Amersham)<br>
in the presence of 5 ul 10X kinase buffer (Amersham) and 500 pmol ATP (Sigma), for 30 min at"37°C.  The phosphorylated<br>
oligodesoxynucleotides were purified by extraction extraction with<br>
a volume of phenol, then a volume of phenol : chloroform :"- isoamyl 10 alcohol (25:24:1 v/v/v) and finally a volume of chloroform : isoamyl alcohol (24:1 v/v), before being precipitated with 1/10 volume 3M sodium acetate pH 4.8 et 2.5 volumes absolute ethanol at -80°C for 20 min then centrifuged at 16060 g for 30.min. The precipitated oligodesoxynucleotides were washed in 70% ethanol, 15 dried, then resuspended in water at a concentration of 10 pmol/ul In order to link the "directional" oligodesoxynucleotides , the following "guide" oligodesoxynucleotides were used :<br>
-	G l= 5" TGTGTTTGAAGGGAATCGAAAGAGAGACACA 3" (SEQ..ID15)<br>
-	G 2= 5" GATTGGGTTTTTGTGTGGCTAAGATGTGTG 3" (SEQ. ID16) 20  - G 4=   5" TGTAGATGTGCCACAGAGTGGCATGCGT 3" (SEQ.XD18)<br>
In order to carry out the LCR reaction, 10 pmol of the phosphorylated "directional" desoxynucleotides SI, S2, S5 and S6 were ligated in the presence of 10 pmol of the "guide"<br>
desoxynucleotides Gl, G2 and G4, 5 ul Taq 10X DNA ligase buffer 25 and 40 units  Taq DNA ligase (New England Biolabs) . The ligation reaction was carried out in a GeneAmp PCR System 97 00 thermocycle. It consists of a cycle of 1 min at 94°C, and 8 identical cycles each consisting of the succession of the following steps : 1 min at 65"C, 1 min at-. 57°C, 1 min at 52°C, 1 min at 48°C, 1 min at 30 43°C, and finally 10 min at 37°C,  Then, the ligation reaction mixture was  purified on a QIAquick column according-to the . supplier" s recommendations.<br>
Finally, PCR amplification of the single stranded DNA obtained was carried out in a thermocycleur GeneAmp PCR System 9700 in the 35 presence of 100 pmol of each of the oligodesoxynucleotide probes 5" CATGCTGCAGACTAGTGGATT 3", and 5* CGGGGATCCTCTAGGTTTCT 3", of 50 nmol of each of the dNTP, of 10 ul of 10X Vent DNA polymerase<br><br>
WO 00/58485	PCT/IBQ0/QQ37Q<br>
31<br>
bufor (New England Biolabs), and 2  units Vent DNA polymerase (New<br>
iigland Biolabs). The DNA"was denatured for 5 min at 94°C, subjected to 25 cycles each consisting of a 30 sec  denaturing step at 95°C, a 30 sec hybridisation step at 56°C, and of 1 min 5 elongation at 720C, then further elongation at 72°C for 5 min.<br>
The DNA fragments of the reaction mixture were digested for 45 min at 37°C with 20 units of BamHI then for 1 h at 37 °c with 20 units of PstI, and finally "purified on a QIAquick column.  They were inserted in the plasmid"pGEM3Z-pet£ prom digested  for 1 h at 37PC 0- with BamHI then for 1 h at 37°C-with PstI, subjected to 0.8% agarose gel electrophoresis, purified on a QIAquick affinity column, dephosphorylated for 1 h at 37°C in the presence  of 12<br>
ul of 10X buffer 3 (New England Biolabs) and 5000  units of calf intestine alkaline phosphatase (CIP, New England Biolabs), and<br>
5 finally purified, on a QIAquick affinity column* To carry out the ligation, 25 ng of plasmid treated as described above were contacted with 100 ng of the DNA fragments obtained by PCR, in the presence of 1.2 ul of-T4 10X DNA ligase buffer (New England Biolabs) and 400 units T4 DNA ligase (New England Biolabs) for 1<br>
) night at 18°C. Previously prepared viable and competent<br>
Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The DNA of the obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was extracted according to alkaline lysis method and analysed by <br>
 enzymatic digestion. The promoter sequence MPrlllO borne by this .plasmid pMRTlllO was verified by sequencing. The 54 bp fragment containing the sequences as-2 and as-1<br>
originating from the CaMV promoter was obtained by digesting 25 ug of the1 plasmid pMRTlllO for 1 h at 37°C with 80 units of PstI, then the generated end"s were blunted by the action of Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the supplier"s recommendations. The plasmid thus modified was digested for 1 h at 37°C with 4 units of DraIII, and the 54 bp fragment was<br>
isolated by electrophoresis on  3% Nu-Sieve agarose gel (FMC, Rockland, USA) and finally purified on a QIAquick affinity column. The ligation was carried out with 30 ng of-vector pMRT1117 prepared as described previously and 50 ng of the 54 bp fragment<br><br>
. WO«»«5	PCT/IBOO/00370<br>
32<br>
for 15 h  at 18°C in a reaction volume of 20 ul in the presence of 2.0 ul of T4 I0X DNA ligase buffer (New England Biolabs) and 800 units T4 DNA ligase (New England Biolabs).. Previously prepared viable and competent Escherichia coli DH5a  bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on  LB media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method , and analysed by enzymatic digestion<br>
and gene amplification using the desoxynucleotides 51 CATGCTGCAGACTAGTATCC 3" selected from the promoter and 5* TTGATTTCACGGGTTGGG 3" selected from the uidA  sequence. The promoter sequence MPrll47 (SEQ.ID06) of one of these clones was verified by sequencing.<br>
2,4. Construction of the promoter MPxll54 :<br>
The promoter MPrll54 (Fig. IV) was obtained by deleting a .44 bp . sequence containing the duplicated as-1-like element from CoYMV present in the promoter MPrll47 (SEQ.ID04).<br>
In order to achieve this, the plasmid pMRT1147 was digested with. 20 units of Spel for 1 h at 37°C, isolated on 0.8% gel agarose, purified on a QIAquick affinity column and religated for 15 h at 18 °C in a reaction volume of 10 ui in the presence of 1 p.1 of T4 10X DNA ligase buffer {New England Biolabs) and 400 units T4 DNA ligase (New England Biolabs). Previously prepared viable and<br>
competent Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones,   selected on LB  media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method, and analysed by  enzymatic digestion and gene amplification using the desoxynucleotides 5" ATTTAGGTGACACTATAG 3" selected from the plasmid and 5" TTGATTTCACGGGTTGGG 31 selected from the uidA sequence. The promoter sequence MPrll54 (SEQ.ID07) of one of these clones was verified by sequencing . 2.5. Construction of the promoters MPrll62, MPrll63, MPrll64, MPrll65 :<br>
The promoters MPrll62, MPrll63, MPrll64, and MPrll65 (Figure VII) were obtained by inserting one or multiple copies of the 70 bp<br>
-27-<br><br><br>
WO 00/58485	PCT/IB00/00370<br>
33 sequence corresponding to a duplication of the as-2 box (Lam.and Ghua, 1989) and the as-1 box (Lam et alt,  1989) of the 35S RNA CaMV promoter, bearing restriction sites on either side, into the site BstEII from MPrlllS (SEQ.ID03), in the 5"&gt;3" orientation or 5 inverted, that is to say, in the 3"&gt;5".orientation.<br>
In order to do this, the plasmid pMRT1116 was digested with 4 units of BstEII for 1 h at 37°C then purified on a QIAquick affinity column. The generated ends were blunted by the action of Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the 10 recommendations of the supplier,. then dephosphorylated for 1 h at 37°C in the presence of 12 ul of 10X "buffer 3" (New England Biolabs) and 25 units of calf intestine alkaline phosphatase (CIP, New England Biolabs). Finally, the thus obtained plasmid was purified again on a QIAquick affinity column. 15 The 70 bp SpeI/DraIII fragment containing the two "as-2" boxes and the "as-1" box was obtained from plasmid pMRTllll. The plasmid pMRTllll, which contains a 58 bp sequence corresponding to a duplication of the as-2 box (Lam et Chua, 1989) and the as-1  box (Lam et al.,   1989) from the 35S RNA-CaMV promoter 20 placed upstream of the "minimal pea plastocyanine promoter modified by the addition of a "G" box, was obtained by lb-PCR in the following way. The single stranded DNA was produced with the help of the following directional oligodesoxynucleotides :"<br>
-	S1 = 5"<br>
25 TTCCCTTCAAACACATACAAATTCAGTAGAGAAGAAACTCATTACTCTTGAGAAACCTAGAGGATC<br>
CCCG 3" (SEQ.ID08)<br>
-	S2 = 5"<br>
CACAAAAACCCAATCCACATCTTTATCATCCATTCTATAAAAAATCACCTTCTGTGTGTCTCTCTT TCGA 3" (SEQ.ID09)<br>
30 - S5 = 5"<br>
CTGTGGCACATCTACATTATCTAAATCTAAGCCACGTCGGAGGATAACATATTCTTCCACACATCT TAGCCA 3" (SEQ.ID12)<br>
-	S7 = 5"	<br>
CATGCTGCAGACTAGTGATTGATGTGATATCAAGATTGATGTGATATCTCCACTGACGTAAGGGAT<br>
35 GACGCATGCCACT 3" (SEQ.ID14)<br>
One hundred picomoles (100 pmol) of the SI, S2 et S5 oligodesoxynucleotides were 5" phosphorylated with 15 units of<br>
28<br><br>
  W° °°/58485	PCT/1BO0/00370<br>
34<br>
kinase (Amersham) in the presence of 5 ul 10X kinase buffer<br>
 (Amersham) and 500 pmol ATP (Sigma), for 30 min at 37°C. The<br>
phosphoryiated desoxynucleotides were purified by extraction with<br>
a volume of phenol, then a volume of phenol : chloroform : isoamyl<br>
5. alcohol (25:24:1 v/v/v) and finally a volume of chloroform :<br>
isoamyl alcohol (24:1 v/v), before being precipitated by 1/10<br>
volume 3M sodium acetate pH 4.8 and 2.5 volumes absolute ethanol<br>
at -80°C for 20 min then centrifuged at 16060 g for 30 min. The<br>
precipitated oligodesoxynucleotides were washed in 70% ethanol,<br>
10 dried, then resuspended in water at a concentration of 10 pmol/ml.<br>
In order to link the directional oligodesoxynucleotides, the<br>
following guide oligodesoxynucleotides were used :<br>
-	Gl=  5" TGTGTTTGAAGGGAATCGAAAGAGAGACACA 3" (SEQ.ID15)<br>
-	G2= 5" GATTGGGTTTTTGTGTGGCTAAGATGTGTG 3" (SEQ.ID16) 15  - G4= 5" TGTAGATGTGCCACAGAGTGGCATGCGT 3" (SEQ.ID18)<br>
In order to carry out the LCR reaction, 10 pmol of the phosphoryiated oligodesoxynucleotides S1, S2, S5 and S7 were ligated in the presence of 10 pmol of the guide<br>
oligodesoxynucleotides Gl, G2 and G4, " 5 ul of Taq 10X.DNA ligase 20 buffer (New England Biolabs) and 40 units Taq DNA ligase (New England Biolabs). The  ligation reaction was carried out in a "GeneAmp PCR System 9700" thermocycle (Perkin Elmer, Norwalk, USA). It consisted of a cycle of 1 min at 94°C,   and "8 identical cycles each consisting "of the succession of the following steps : 25 1 min at 65°C, 1 min at 57°C, 1 min at 52°C, 1 min at 48°C, 1 min at 43aC,   and finally 10 min at 37°C. Then, the ligation reaction mixture was purified on a QIAquick column according to the suppliers"s recommendations.<br>
Finally, PCR amplification of the the single stranded DNA obtained 30 was carried out in a "GeneAmp PCR System 9700" thermocycle in the presence of 100 pmol of  each of the oligodesoxynucleotide probes 5" CATGCTGCAGACTAGTGGATT 3", and 5" CGGGGATCCTCTAGGTTTCT 3", 50 nmol of each dNTP, 10 ul of Vent 10X DNA polymerase buffer (New England Biolabs), and 2 units Vent DNA polymerase- (New England 35 Biolabs). The DNA was denatured for 5 min at 94°C, subjected to 25 cycles each consisting of a 30 sec denaturing step at 95°C, of a 30 sec hybridisation step at 560C, and of 1 min elongation at<br><br>
WO 00/58485	PCT/IBOO/00370<br>
35 72°C then further elongation at 72°C for 5 min. The  DNA fragments from the reaction mixture were digested for 45 min at 37eC by 20 units of BamHI, then for 1 hat 37°C by 20 units  of PstI, and finally purified on a QlAquick column. They were 5 inserted into the plasmid pGEM3Z-petE prom  previously digested for   1 hat 37°C by BamHI, then for 1 h at 37°C by PstI, subjected to electrophoresis on 0.8%,agarose gel, purified on a QlAquick affinity column, dephosphorylated for 1 h at 37°C in the presence  of 12 ul of I0X "buffer 3" (New England Biolabs) and 25 units of 10 calf intestine alkaline phosphatase (CIP, New England Biolabs), and finally purified on a Qiaquick affinity column. In order to carry out ligation, 25 ng of the thus treated plasmid was contacted with 100 ng of the DNA fragments obtained by PCR, in the presence of 1.2 ul of T4 10X DNA ligase buffer (New England 15 Biolabs) and 400 units of T4 DNA ligase (New England Biolabs) for 1 night at 18°C. Previously prepared viable and competent Escherichia  coli  DH5a bacteria were transformed half of the ligation reaction mixture. The DNA of the obtained clones, selected on LB media supplemented with ampicillin {50 mg/1), was 20 extracted according to the alkaline lysis method and analysed y enzymatic digestion. The promoter sequence MPrllll of one of the obtained clones was verified by sequencing.<br>
The 70 bp fragment containing the two "as-2" boxes and the "as-1" box were obtained by digestin 25 mg of plasmid pMRTllll with 40 2-5 units of SpeI for 1 h at 37°C, then with 4 units of Drain for 1 h at 37°C. The fragment was isolated by electrophoresis on Nu-Sieve 3 % agarose gel (FMC,   Rockland, USA) and finally purified on a QlAquick affinity column. The ends of this fragment were blunted by the action of Pfu DNA polymerase (Stratagene, La Jolla, USA) 30 according to the recommandations of the supplier, then purified again on a QlAquick affinity column.<br>
The ligation was carried out with 30  ng of pMRT1116 vector prepared as described above and 50 ng of 70 bp fragment for 15 h at 18°C in a reaction mixture of 20 ul in the presence of 2.0 ul 35 of 10X T4 DNA ligase buffer (New England Biolabs) and 400 units of T4 DNA ligase (New England Biolabs), Previously prepared viable<br>
and competent Escherichia coli  DH5a bacteria, were transformed<br>
30<br><br>
WO 00/58485	PCT7IB00/00370<br>
36<br>
wite a third of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method, and analysed by enzymatic digestion and by gene 5 amplification with the help of the universal SP6<br>
oligodesoxynucleotide (5" TAAATCCACTGTGATATCTTATG 3") located in the 51 region of the promoter and the oligodesoxynucleotide 5" TTGATTTCACGGGTTGGG 3" selected on the uidA  sequence. This. cloning strategy enabled the production of 70 bp fragment inserts 10 into the vector in the 5"&gt;3" orientation {the sequences,<br>
as-2/as-2/as-l are cloned in the same orientation as in their native promoter), in 3"&gt;5" antisens orientation (an oriention inverse to that present in the CaMV 35S promoter) and in one or more copies. The following synthetic, and chimeric promoters could 15 be obtained using this strategy : MPrll62 (SEQ.ID19), which<br>
corresponds to the insertion of a single 70 bp sequence in normal<br>
5"&gt;3" orientation, MPrll63 (SEQ.ID20), which corresponds to the<br>
insertion of two 70 bp sequences in the normal 5"&gt;3" orientation,<br>
MPrll64 (SEQ.ID21), which corresponds to the insertion of a single<br>
20 70 bp sequence in inverse or antisens orientation, and MPrll65<br>
(SEQ.ID22), which corresponds to the insertion of four 70 bp<br>
sequences in normal 5"&gt;3" orientation* Each one of these clones<br>
was verified by sequencing,	<br>
25    2.6.   Construction of  promoters MPrll67, MPrll68f  and MPrll69  :<br>
The promoters MPrll67, MPrll68 and MPrll69 (Fig. VII) are derived from MPrll47 (SEQ.ID06). They were obtained by inserting, into the BstEII site of  MPrll47,   one or more normally 5"&gt;3" oriented (i.e. cloned in the same orientation as the native orientation of the<br>
30 sequences in the CaMV promoter) 70 bp sequences containing the 58 bp sequence coresponding to a duplication of the as-2 box (Lam et Chua, 1989) and the as-1 box (Lam et al.,  1989) of the 35S RNA CaMV promoter. In order to do this, the plasmid pMRT1147 was digested with 4<br>
35 units of BstEII for 1 h at 37°C then purified on a Qiaquick<br>
affinity column. The generated ends were blunted by the action of Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the<br>
31<br><br>
WO 00/58485	PCT/IB00/00370<br>
37	<br>
recommandations of  the supplier, then dephosphorylated for 1 h at 770C in the presence of 12 ul of 10X "buffer 3" (New England Biolabs) and 25 units of calf instestine alkaline phosphatase (CIV,   New England Biolabs), Finally the plasmid thus obtained was purified again on a Qiaquick affinity column.<br>
The 70 bp Spel/Dralll fragment containing the two "as-2" boxes and the "as-l" box was obtained from plasmid pMRTllll, the production of which was described previously. The ends of this fragment were blunted as described above.<br>
The ligation was carried out with 30 ng of vector pMRT114 7 prepared as described previously and 50  ng of the 70 bp fragment for 15 h at 18°C in a reaction mixture of 20 ul in the presence " of 2.0 ul 10X T4 DNA ligase buffer (New England Biolabs) and 400 units T4 DNA ligase {New England Biolabs). Previously prepared<br>
viable and competent Escherichia  coli  DH5a bacteria were transformed by reacting with a third of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was.extracted according to the alkaline lysis method, and analysed by  enzymatic digestion and by gene amplification using the universal SP6 oligodesoxynucleotide (51 TAAATCCACTGTGATATCTTATG 3") located in the 5" region of the promoter and the oligodesoxynucleotide 5" TTGATTTCACGGGTTGGG 3" selected from the uidA  sequence. The following synthetic promoters could be prepared by the preceding method : MPrll67 (SEQ.ID23), which corresponds to the insertion of three 70 bp sequences in normal 5"&gt;3" orientation, MPrll68 (SEQ.ID24), which corresponds to the insertion of two 70 bp sequences in normal 5"&gt;3" orientation and.MPrll69 (SEQ.ID25), which corresponds to the insertion of a single 70 bp sequence in normal 5"&gt;3" orientation. Each of these clones was verified by sequencing.<br>
2.7. Production of binary vector pMRT1218 containing promoter MPrl218 ;<br>
The reference promoter for expression of the GUS reporter gene in corn seed endosperm is the promoter of gene coding for a storage protein in corn s_eed of 28 kDa belonging to the family of<br>
32<br><br>
WO 00/58485	PCT/IB00/00370<br>
38<br>
ganha-zeins. This construct has been described in French patent Application number FR9912373, filed on September 30th, 1999, in the name of the applicants, and is incorporated herein ,for the purposes of the specific description of said reference promoter<br>
5 and vector. The 1.7 kb y-zein whole length promoter sequence<br>
{Pry-zein). contained in the plasmid p63 described by Reina et al. (1990) was placed upstream of the uidA-IV2/term-nos sequence in a vector designated pMRT1126. The vector pMRT1126 is described in full in the abovementioned French patent application, and its 10 description is incorporated herein for the purposes of the description of this vector. The promoter Mprll26 in vector PMRT1126 derives from the HMWG-Dx5 (High Molecular Weight Glutenin) promoter by deletion of the sequence situated upstream of nucleotide -142,   which sequence comprises two "prolamine like" 15    boxes, two "GATA" boxes, a "G" box"and and an activator element. The promoter fragment was amplified by PCR and isolated.,<br>
The 1.7 kb y-zein promoter was obtained by digesting 15 pg of plasmid p63 with restriction enzymes HindIII and BamHI for 1 h at 37°C. The thus liberated 1.7 kb Pry-zein fragment was isolated on<br>
20  0.8% agarose gel using a «  Concert Rapid Gel Extraction System » kit.<br>
In parallel, 10 ug of plasmid pMRT1126 were digested.by the restriction enzymes Hindlll and BamHI for 1 h at 37°C. The vector fragment was then isolated on 0,8% agarose gel using a « Concert<br>
25 Rapid Gel Extraction System » kit.and dephosphorylated with 40<br>
units of calf intestine alkaline phosphatase (New England Biolabs) in the presence of 10X "buffer 3" at 37°c for 1 h.<br>
The ligation reaction was carried out with 50 ng of y-zein promoter fragment and 100 ng of plasmid pMRT1126, in a reaction volume of<br>
30 10 ul, in the prese"nce of tampon T4 10X DNA ligase and 400 units of T4 DNA ligase {New England Biolabs), in a « GeneAmp PCR System 9700 » thermocycle as  described above.   Previously prepared viable and competent Escherichia coli  DH5a bacteria were transformed with all of the ligation reaction mixture. The plasmid DNA of the<br>
35 obtained clones, selected on LB media supplemented with ampicillin (50 mg/1), was extracted according to the alkaline lysis method<br>
-32-<br><br>
WO 00/58485	PCT/IBOO/00370<br>
	39<br>
and analysed by enzymatic digestion. The resulting plasmid was dignated pMRT1218.<br>
3. Construction of binary plasmids containing the promotera.<br>
5 Three types of binary vector were used for cloning the various expression cassettes. The vector pGA492 was used to prepare the cassettes containing MPrlll6, MPrll46, MPrll47 and MPrl092 to , create the expression cassettes or  binary vectors pMRTH52, pMRT1171, pMRT1172 and pGA492MPr1092 respectively. The vector 10 . pGA492 was prepared as follows. A 25 ug amount of plasmid pGA492 (An, 1986) was  digested with 80 units of Hindlil for 1 h at 37°C then purified on a QIAquick affinity column. The 5" protruding ends of this plasmid were blunted using P£u DNA polymerase {Stratagene, La Jolla, USA) according to the supplier"s 15    recommendations.. The thus modified plasmid was digested with 80 units of EcoRI for 1 h at 37°C, then the vector deleted of its-291 bp fragment was isolated on 0.7% agarose gel and purified on  a QIAquick affinity column. A 25 ug amount of plasmid pGA492 (An., 1986) was digested with 80 units of Hindlil" for 1 h at 37°C then . 20 purified on a QIAquick affinity column.<br>
The vector pMRTl118 was used to clone the cassettes under control of the promoters MPrll62, MPrll64, MPrll65, MPrll67 and MPrlD92 to give the vectors pMRTll85, pMRT1186, pMRT1187, pMRT1188 and pMRT1182 respectively. 25, The plasmid pMRTlllS. is described completely in French patent-application number FR9911112, filed on September 3, 1999, also.in.  the name of the present applicant, the specific description of which is incorporated herein by reference. The binary plasmid PMRT1118  (5971 pb)  results from the introduction of a T-DNA 30 fragment digested by Avrll enzyme into the Avrll site of another dephosphorylated plasmid also fully described in the previously mentioned prior application to same applicant, and designated pMRT1106, also specifically incorporated herein by reference. In order to carry out the insertion, the pMRTHQ6 plasmid DNA (5 35 ug) was digested with Avrll enzyme, purified with the aid of the «QIAquick PCR Purification&gt;&gt; kit, then dephosphorylated with 50 units of calf intestine alkaline phosphatase (New England Biolabs)<br>
-34-<br><br>
WO 00/58485	FCT/TB00/00370<br>
40<br>
in a final reaction mixture volume of  120 ul in the presence of 12 ui 3X  10 buffer (New England Biolabs) at 37 °C for 1 hour, isolated by electrophoresis on at 0.6% agarose gel in TBE buffer, purified with a «QIAguick Gel Extraction» kit," dephosphorylated a 5 second time with the calf intestine alkaline phosphatase under the conditions mentioned above,  -and .finally purified with a «QIAquick PCR Purification&gt;&gt; kit and transferred to 50  ul de H2o The PCR ligation reaction was carried out with 32,5 ng of digested dephosphorylated plasmid pMRT1106 and 50 ng of T-DNA fragments 10 digested in a reaction mixture volume of 10 ul in the presence of 1 ul T4 lOx XMA  ligase buffer (New England Biolabs) and.400 units of T4 DMA ligase (New England Biolabs). The ligation comprised 180 cycles each including 2 steps, the first one at 10°C for 30 seconds and the second step at 30°C for 30 seconds in a «GeneAmp 15 PCR System 9700» thermocycle.<br>
Previously prepared viable and competent Escherichia coli DH5a bacteria, were transformed (Hanahan, 1983) , The plasmid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method 20" {Birnboim et Doly, 1979) and verified by enzymatic digestion and sequencing. The resulting plasmid was designated pMRTlll8. This vector was  then prepared in the following manner :  a 25 ug amount of plasmid was digested with 80 units of HindIII for 1 h at 37°C then purified on a Qiaquick affinity column. The 5" 25 protruding ends of this plasmid were, blunted using Pfu DNA<br>
polymerase (Stratagene, La  Jolla, USA) according to the supplier"s" recommendations. The thus modified plasmid was digested with 80 units of EcqRI for 1 h at 37°C, then the vector was isolated on 0.7% agaros gel and purified on a Qiaquick affinity column. This 30 was then dephosphorylated for 1 h at 37°C using 10 units of calf intestine alkaline phosphatase (New England Biolabs) according to the manufacturer"s recommendations, then purified on  a QIAquick affinity column.<br>
The vector pMRT1195 was used to clone the cassettes under the 35 control of the promoters MPrlll6, MPrll54, MPrll62, MPrll63,<br>
MPrll64, MPrll65, MPrll67, MPrll68, MPrll69 and MPrl092 to give the vectors pMRTl245, pMRT1246, pMRT1247, pMRT1248, pMRT1249,<br><br>
WO 00/58485	PCT/IBOO/00370<br>
pMRT1250, pMRT1251, pMRT1252, pMRT1253 and pMRT1254 respectively, the vector pMRT1195, which is also described in the prior French application number FR 9911112, filed on September 3, 1999, also in the name of the present applicant, and incorporated herein by 5 reference to that particular aspect,  comprises in order : an ori RK2 region followed by an  ori ColEI region, followed by nptlll and an nptll regions and then a trfA region, subsequently followed by a transfer DNA left border region, a nos terminator region, a bar region (coding for phosphinotricine acetyl transferase, a protein 10 conferring herbicide resistance), a rice actin intron-1 region, a polyadenylation transcription termination signal region, a multiple cloning site region and then a transfer DNA right border region. The vector pMRT1195 was prepared in the following manner. A 20 ug amount of plasmid pMRT1195 was digested by 15 units of 15    Hpal for 1 h at 37°C then purified on a QIAquick affinity column. The thus opened vector was dephosphorylated for 1 h at 37 °c using 10 units of calf intestine alkaline phosphatase {New England Biolabs) according to the manufacturer"s recommendations, then purified on a QIAquick affinity column. 20 3.1. Production of pMRT1152.<br>
The expression cassette MPrlll6 / uidA-IV2 / nos  term was cloned at the modified H<br>
indIII site of the binary plasmid pGA492. It was obtained from the plasmid pMRT1116 digested with 30 units of PstI for 1 h at 37°C and purified on a QIAquick affinity 25 column. The protruding 5" ends of this plasmid were blunted using Pfu DNA polymerase (Stratagene, La Jolla, USA)  according to the supplier"s recommendations. The plasmid thus modified was digested simultaneously with 80 units of EcoRI and 40 units of XmnI for 1 h at 376C, then the 2.5 kb DMA fragment corresponding to the 30 expression cassette was separated on 1% agarose gel and purified on a QIAquick affinity column.<br>
The ligation was carried out by mixing 100 ng of binary plasmid pGA492 prepared as described above and 50 ng of expression cassette for 1 night at 18 °C in a reaction volume" of 20 u1  in 35 .the presence of 2 ul of T4 10X DNA ligase buffer  (New England Biolabs) and 400 units of T4 DNA ligase (New England Biolabs). Previously prepared viable and competent Escherichia coli DH5a<br>
-36-<br><br>
, WO 00/58485	PCT/TBOG/00370<br>
42 bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with tetracycline  (12 mg/1), was extracted according to,the alkaline lysis method and analysed by enzymatic 5 digestion as well as by gene amplification using the<br>
desoxynucleotides 5" ATATGAGACTCTAATTGGATACCGAGGGG 3" selected from the transfer DNA of the binary plasmid and 5 • TTGATTTCACGGGTTGGG 31 selected from the the expression cassette in the uidA  sequence. The resulting clone was designated pMRT1152. 10 3.2. Production of the binary plasmid pMRT1171. ■ The expression cassette MPrll46 / uidA-IV2 / nos  term was cloned at the, modified HindIII site of the binary plasmid pGA492 by following the same  protocol as  for plasmid pMRT1152, except that the expression cassette was isolated from the plasmid pMRT1146. 15 The resulting clone was designated pMRT1171.<br>
3.3.	Production of the binary plasmid pMRT1172,<br>
The expression cassette MPrll47 / uidA-IV2 / nos  term was cloned at the modified Hindlll site of the binary, plasmid pGA492 by following the same protocol as for "the plasmid pMRT1152, except 20 that the expression cassette was isolated from the plasmid pMRT1147. The  resulting clone was designated pMRT1172,<br>
3.4.	Production of the binary plasmid pGA492MPrl092.<br>
The promoter fragments MPrl092 and uidA-IV2 / nos  term sequence were inserted into the binary plasmid pGA492 prepared as described 25 above. The two fragments were prepared in the following manner : - CaMV D35S prom was isolated by digesting 10 ug. of plasmid<br>
pJIT163A with 40 units of Kpnl for 1 h at 37 °C. The ends of this linearised plasmid were blunted using 6 units of T4 DNA polymerase (New England Biolabs) for 30 min at 37°C according to the<br>
30 manufacturer"s recommendations. The plasmid thus modified was purified on a QIAquick affinity column , then redigested with 80 units of Hindlll for 1 h at 37°C. The 743 bp fragment corresponding to the promoter was separated on 0.8% agarose gel, then purified on  a QIAquick affinity column.<br>
35 - the cassette "uidA-IV2/nos  term" was obtained by digesting 4 ug of the plasmid pMRT1092 with 40 units of Hindlll and EcoRI for 1 h. The 2.2 kb fragment corresponding to the sequence uidA-IV2/ nos<br><br>
WO 00/58485	PCT/IB00/00370<br>
43 tere was separated on 0.8%  agarose gel, then purified on a  Aquick affinity column.<br>
The ligation between the three fragments was carried out by mixing 100 ng of binary plasmid, 50 ng  of promoter fragment and 50  ng of 5 the fragment corresponding to the nuidA-IV2/nos  term" sequence in a reaction volume of 20 ul, in the presence of 2 ul of T4 10X DNA ligase buffer (New England Biolabs) and 400  units of T4 DNA ligase (New England Biolabs) ".  The incubation was carried out in a thermocycle by subjecting the ligation mixture to 198 cycles each 10 consisting of a 30 sec  incubation at 30°C,   and  a 30  sec incubation at 10°C. Previously prepared viable and competent Escherichia coli DH5a bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with tetracycline (12 mg/1)/ was extracted 15 according to the alkaline lysis method and analysed by enzymatic digestion and gene amplification using the desoxynucleotides 5" ATATGAGACTCTAATTGGATACCGAGGGG 3" selected from the transfer DNA of the binary plasmid and 5" TTGATTTCACGGGTTGGG 3" selected from the expression cassette in the uidA  sequence. One  of the retained 20 clones was designated pGA492MPrl092.<br>
The plasmids pMRT1152, pMRT1171, pMRTH72 and pMRT1182 were transferred into a strain of Agrobacterium tumefaciens  LBA44Q4 according to the technique described by Holsters et a1.   (1978), The plasmid DNA of the obtained clones, selected on LB media 25 supplemented with rifampicin (50 mg/1) and with tetracycline  (5 mg/1], was. extracted according to the alkaline lysis method, modified by adding lysozyme (25 mg/ml) to the cell resuspension buffer. The plasmid DNA obtained was analysed by enzymatic digestion and gene amplification using the desoxynucleotides 5" 30 ATATGAGACTCTAATTGGATACCGAGGGG 3" selected from the plasmid and 5" . TTGATTTCACGGGTTGGG 3" selected from the expression cassette. The agrobacteria clones obtained were used to carry out plant genetic transformation.<br>
35 3.5. Production of the binary vectors pMRT1185, pMRT1186, pMRT1187 and pMRT1188 :<br>
The expression cassettes MPrll62/uidA-IV2/nos term,<br>
38<br><br>
WO 00/58485	PCT/IB00/00370<br>
44<br>
MPRT164/ uidA-IV2/nos  term, MPrll65/ vidA-lV2/nos  term and MPrll67/uidA-IV2/nos  term were produced from plasmids pMRT1162, pMRTU64, pMRTH65 and pMRT1167 respectively, and were cloned into the binary plasmid pMRTH18 described previously. 5 A 10 ug amount of each of the plasmids pMRT1162, pMRTH64,<br>
pMRT1165„ and pMRT1167 were digested by PstI for 1 h at 37°C then purified on an affinity column. The protruding 5" ends  of these various opened vectors were blunted using Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the supplier"s 10 recommendations. The thus treated vectors were digested again simultaneously by 20  units of EcoRI and 10 units of Xmnl for 1 h at 37°C, For each of these digestions, the DNA fragment corresponding to the expression cassette was isolated on 1% agarose gel and purified on a Qiaquick affinity column, 15 The ligation was carried out in a thermocycle (GeneAmp PCR Systems 9700) by mixing 100 ng of binary plasmid pMRT1118 prepared as described above and 50 ng of "expression cassette in a reaction volume of 12 ul in the presence of 1.2 ul T4 10X DNA ligase buffer (Epicentre Technologies), 1.2 ul  of 25 mM ATP solution and 3 units. 20 of 10X DNA ligase (Epicentre Technologies). The ligation reaction consisted of a series of 200 identical cycles each consisting of a 30 sec  step at 10°C and a 30 sec step at 30°C.   Previously prepared, viable and competent Escherichia  coli  DH5a bacteria were transformed with half of the ligation reaction mixture. The . 25 plasmid DNA from the clones obtained, selected on LB media<br>
supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed by enyzmatic digestion with the enzymes BamHl and EcoRI. The resulting clones were designated pMRT1185, pMRT1186, pMRT1187, and pMRT1188, and contain 30 respectively the promoters MPrll62, MPrll64, MPrll65 and MPrll67.<br>
3.6. Production of positive control binary vector pMRT1182.<br>
The binary vector pMRT1182 was obtained by insertion of the PrD35S CaMV promoter fragment and the uidA-IV2/term-nos sequence into the 35 binary plasmid pMRT1118 prepared as described above.<br>
The  PrD35S CaMV promoter was isolated by digesting 10 ug of the<br>
" plasmid pJIT163A successively with Kpnl and Hindlll for 1 h at<br><br>
WO 00/58485	PCT/IB 00/00370<br>
45<br>
37°C The "743 bp fragment corresponding to pD35S CaMV was isolated 0,8 % agarose gel, then purified on. a Qiaquick affinity column.<br>
The "uidA-IV2/nos "term" sequence was obtained by digesting 4 ug  of  5 plasmid pMRT1092 with 40 units of Hindlll and EcoRI for 1 h. The 2.2 kb fragment corresponding to the "uidA-IV2/nos  term" sequence was isolated on 0.8% agaros gel, then purified on a Qiaquick affinity column.<br>
The ligation was carried out in the presence of 100 ng of binary 10 plasmid, 50 ng of the PD35S CaMV fragment and 50 ng of the fragment corresponding to the uidA-IV2/term-nos sequence in a reaction volume of 20 ul, in the presence of T4 (IX) DNA ligase buffer and 400 units of T4 DNA ligase (New "England Biolabs) . The incubation was carried out by PCR cycles in a «GeneAmp PCR System 15 9700» thermocycle as described previously. Previously prepared,<br>
vaible and competent Escherichia  coli  DH5a bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to 20 the alkaline lysis method and analysed by enzymatic digestion. The resulting plasmid was designated pMRT1182-<br>
3.7. Production of binary vectors pMRT1245, pMRTl246, pMRTl247, PMRT1248, pMRTl249, pMRT1250, pMRTl251, pMRT1252, and pMRTl2S3 :<br>
25 The expression cassettes HPrlll6/uidA-IV2/nos term,<br>
MPrll54/uidA-IV2/nos term, MFrll62/uiciA-IV2/nos term,    MPrll63/uidA-IV2/nos term, MPrll64/uidA-IV2/nos term, MPrll65/uidA-IV2/nos term, MPrll67/uidA-IV2/nos term, MPril68/uidA-IV2/nos term and MPrll69/uidA-IV2/nos term were<br>
30    produced from plasmids pMRT1116, pMRT1154, pMRT1162, pMRT1163, pMRT1164, pMRT1165, pMRT1167, pMRT1168 and pMRTll69 respectively, and were cloned into the Hpal site of the binary vector pMRT1195. A 10 ug amount of each of the plasmids pMRT1116, pMRT1154, PMRT1162, pMRT1163, pMRT1164, pMRT1165, pMRT1167, pMRT1168,<br>
35 pMRT1169 was digested simultaneously by 20 units of Pstl, 20 units of EcoRI and 10  units of XmnI for 1 h at 37°C. For each of these digestions, the "corresponding DNA fragment to the expression<br><br>
WO 00/58485	PCMB00/00370<br>
46<br>
sassette was isolated on 1% agarose gel and purified on a Qiaquick affinity column. The 5" protruding ends of these different fragments were blunted using Pfu DNA polymerase (Stratagene, La Jolla, USA)according to the supplier"s recommendations. 5 Ligation was carried out in a thermocycle (GeneAmp PCR Systems "9700) by mixing 100 ng of binary vector pMRT1195 prepared as described above and 50 ng of expression cassette in a reaction volume of 12 ul in the presence of 1.2 ul of T4 10X DNA. ligase buffer (Epicentre Technologies), 1,2 ul of 25 mM ATP solution and 10 3 units of 10X DNA ligase (Epicentre Technologies). The ligation reaction consisted in a series of 200 identical cycles each consisting of a 30 sec step at 10°C and a 30 sec step at 30°C.<br>
Previously prepared, viable and competent Escherichia coli  DH5a bacteria were transformed with half of the ligation reaction<br>
15 mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed by enzymatic digestion by the enzymes BamHI and EcoRI. The resulting clones were designated pMRT1245, pMRT1246, pMRTl247, pMRT1248, pMRT1249,<br>
20  pMRTl250, pMRT1251, pMRT1252 and pMRT1253, and contain<br>
respectively the promoters MPrlll6, MPrll54, MPrll62, MPrll63, MPrll64, MPrll65, MPrll67, MPrll68 and MPrll69.<br>
3,8. Production of the positive control binary vectox pMRT1254.<br>
25 The binary vector pMRT1254 is used as a positive control during evaluation of the expression in plants.<br>
The expression cassette MPrl092/uidA-IV2/nos term was cloned into the Hpal site of the binary vector pMRT1195 prepared as described above. It was obtained by digesting 10 ug of plasmid pMRT1182 with<br>
30 40  units KpnI for 1 h at 37°C, purification of the digestion<br>
product on an affinity column and then digestion with 40 units of EcoRI. The 2,8 kb DNA fragment corresponding to the expression cassette was isolated on 1% agarose gel and purified on a Qiaquick affinity column. The 5" protruding ends sortantes "of this fragment<br>
35 were blunted with Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the supplier"s recommendations. Ligation was carried out in a thermocycle (GeneAmp PCR Systems<br><br>
WO 00/58485	PCT/tBfl 0/00370<br>
47 97000 by mixing 100 ng of the binary vector pMRT1195 prepared as scribed previously with 50 ng expression cassette in a reaction volume of 12 ul in the presence of 1.2 ul T4 10X DNA ligase buffer (Epicentre Technologies), 1.2 ul of 25 mM ATP solution and 3 units 5 of 10X DMA ligase"(Epicentre Technologies). The ligation reaction consisted of a series of 200 identical cycles each consisting of a 30 sec step at 10°C and a 30 sec step at 30°C. Previously prepared, viable and competent Escherichia  coll  DH5a bacteria were transformed with half of the ligation reaction mixture. The 10 plasmid DNA of the obtained clones, selected on LB media<br>
supplemented with kanamycin (50 mg/1) ,  was extracted according to the alkaline lysis method and analysed by enzymatic digestion with the enzymes BamHI and EcoRI. The resulting clone was designated PMRT1254, 15<br>
3.9. Production of the negative control binaxy vector pMRTl255. The binary vector pMRT1255 is used as a negative control during evalutation of the expression in plants.<br>
The expression cassette deprived of a promoter and corresponding 20 to the "uidA-IV2/nos term" sequence was cloned into the Hpal site of the binary vector pMRT1195, in order to use this plasmid as the negative control.<br>
This sequence  was obtained by digesting 10 ug of the plasmid PMRT1163 with.40 units of BamHI for 1 h at 37°C, purifiying the 25 digestion product on an affinity column and then digesting with 40 units of EcoRI at 370C for In. The 2.2 kb DNA fragment corresponding to the promoter-less expression cassette was isolated on 1% agaros gel and purified on a Qiaquick affinity column. The 5" protruding ends of this fragment were blunted using 30 Pfu DNA polymerase (Stratagene, La Jolla, USA) according to the supplier"s recommendations.<br>
Ligation was carried out in a thermocycle (GeneAmp PCR" Systems 9700) by mixing 100 ng of binary vector pMRTl195 prepared as described above and 50 ng of expression cassette in  a reaction 35 volume of 12 ul in the presence of 1.2 ul of T4 1 OX"DNA ligase buffer (Epicentre Technologies), 1.2 ul of 25  mM ATP solution and 3 units of 10X- DNA ligase {Epicentre Technologies). The ligation<br><br>
WO 00/58485	PCT/IB00/00370<br>
48<br>
reaction consisted of a series of 200 identical cycles each consisting of a 30 sec step at 10°C and a 30 sec step at 30°C. Previously prepared, viable and competent Escherichia  coli  DH5a bacteria were transformed with half of the ligation reaction mixture. The plasmid DNA of the obtained clones, selected on LB media supplemented with kanamycin (50 mg/1), was extracted according to the alkaline lysis method and analysed by enzymatic digestion with the enzymes BamHl and EcoRI. The resulting clone was designated pMRTl255.<br>
10<br>
4. Measurement and comparison of the levels of expression of the<br>
different promoters according to the invention through transient<br>
expression experiments.<br>
4.1. Culture and plant material production. 15 4.1.1 In vitro culture of tobacco, leaf preparation.<br>
The transient expression experiments were carried out on tobacco<br>
leaves (Nicotiaua tabacum  L.) of the cultivar PBD6 aged 6 weeks.<br>
Mature tobacco cv. PBD6 seeds were sterilised for 10 min in a<br>
saturated calcium hypochlorite solution (7-0 g/1), then rinsed<br>
20 thrice for 5 min in sterile deionized water. The sterile seeds<br>
were placed on MS20 media (Murashige et Skoog, 1962) and incubated<br>
for 6 weeks in aculture chamber (constant temperature of 24°C,<br>
photoperiod 16 h light / .8 h darkness, luminous intensity.of 200<br>
umol photons.m2. sec-;) .<br>
25 In order to avoid the foliar mesophyll cells splitting during<br>
transformation, the 2 main leaves of the tobacco plants PBD6 aged 6 weeks were excised from the plant 24 h before transformation with a gene gun, and placed, adlignous face up, on gentle plasmolysis BY3 media (Salts MS 4,4 g/1, myoinositol 100 mg/1,<br>
30 thiamine 1 mg/1, KH2P04 200 mg/1, Saccharose 30 g/1, Sorbitol 45,5 g/1, 2,4 D 1 mg/1, pH 5,8).<br>
4.1.2. Production and preparation of corn seed.<br>
Transient expression experiments were carried out on the endosperm 35 of L2 corn seeds (cultivar SN 87 165), taken from corn plants<br>
cultured in a phytotron at 24°c, at 60% relative humidity and with a ephotoperiod of 16 h light / 8 h darkness.<br><br>
WO 00/58485	PCT/IB00/00370<br>
49<br>
Twerve days after pollinisation [12  DAP), the corn seed was taken and sterilised in a 20% bath of Domestos ®, with agitation for 5 min. Following the elimination of the Domestos ® by successive rinsing with deionized sterilised water, the pericarp and the 5 aleurone cell layer were carefully removed under sterile<br>
conditions. Tangential cuts of the. now exposed endosperm were made and placed on filter paper soaked in the minimal Murashige et Skoog media (MS 5524, Sigma).<br>
10 4,1.3. Production of corn leaves.<br>
Transient expression experiments were carried out on leaves of L2 corn (cultivar SN 87 165), taken from the plant.after two weeks culture in a phytotron at 24°C, at 60% relative humidity and with a photoperiod of 16 h light / 8 h darkness. 15 Twelve days after germination, the youngest leaves were taken and sterilised in a 20% bath of Domestos ®, with agitation for 5 min". The Domestos ® was eliminated by successive rinsing with deionized sterilised water, then the leaves were placed for 24 h onto the weak plasmolysis media N6P6 0.4M (salts MS 3.98 g/1, vitamines N6 20 100 mg/1, L-proline 700 mg/1, casein hydrosylate 100 mg/1,<br>
saccharose 20 g/1, sorbitol ,36,4 g/1, mannitol 36,4 g/1, 2,4 D 1 mg/1, pH 5.8, phytagel 3g/l), adlignous face up, in order to avoid splitting of the foliar cells during transformation.<br>
2,5 4.2. Gold particle coating with the chimeric construction DNA.<br>
Biolistic transformation required prior deposition of DNA onto spherical gold beads of 0.6 um in diameter that had been sterilised for 10 min in absolute ethanol {99.98%, with less than 0.02% water), washed four times in sterile deionized water, and<br>
30 finally stored for a maximum of 4 weeks at -200C in a solution of 50% glycerol.<br>
The concentration of all of the control and test plasmids used during for the transformation experiments, was adjusted to 1 ug/ul. In each of the transformation experiments, an internal<br>
35    reference control (pCaMV35SIuc) was cotransformed in order to normalise the variations of GUS activity between the different experiments (Leckie et al.,   1994).<br><br>
, WO 00/58485	PCT/IB00/00370<br>
The  coating of DNA onto the  gold beads  prepared as  above was carried out  in a  sterile  chamber under laminar flux.  An aliquot<br>
fraction of 1.8 mg of sterile bead suspension in  30 ul of glycerol     50 %,   was mixed vigorously in a vortex for 1 min,   then for  10 sec 5    with 20  ul of DNA suspension  containing  4  ug of one of the plasmids  to be  tested" and 2  uq of the reference plasmid pCaMV35Sluc.  Then,   20  ul  of  2. 5M CaC12  were  added  an mixed<br>
vigorously for 10  sec.   Next,   20  u1  of 0. IM spermidine was  added to the mixture and the whole mixture was  agitated in  a vortex for a 10    further 30 sec.   The DNA coating of the beads was  continued by incubating the mixture  in  ice  for  15 min,   then the  coated beads were  centrifuged at  low velocity for 5  sec and washed twice  in absolute ethanol.<br>
After washing,   the coated beads were resuspended in 32 ul  absolute 15    ethanol,   subjected to ultrasound treatment  three  times  for a<br>
duration of 2 sec each time,   vigorously mixed in  a vortex for 15 sec,   then immediately divided into 4  identical  aliquot parts on sterile  "macrocarrier"  disks  of the  Biolistic  PDS-1000/He  system prepared according to the  supplier"s  recommendations   (BioRad, 20    Hercule,   USA).  The entire assembly    of  "macrocarrier  support  /<br>
macrocarrier bearing the bead deposit",   was  left   to dry for 5 min.<br>
4.3  Bombardment of tobacco foliar tissues and transient expression.<br>
25    The bombardment  of tobacco leaves was  carried out  using a<br>
Biolistic FDS-1OOO/He gene gun  system by following  the general recommendations of the  supplier   (BioRad,   Hercule,   USA)   relating to the manipulations and assembly of the various components  of the apparatus.  Each leaf was bombarded twice  successively under the<br>
30    following shooting conditions   :<br>
-	helium pressure selected for the acceleration  of the coated gold beads was  6200  kPa   (900 psi).<br>
-	the plant sample was placed at  9 cm from the bead acceleration zone.<br>
35    - the  shooting was  carried out  in a vacuum of 27  mm of mercury. After bombardment the  leaves were  left  in BY3 media and incubated<br>
-45-<br><br>
WO 00/58485	PCMBOO/00370<br>
51 for 18 h in the dark in a culture chamber at 24°C.   This incubation anabled transient expression of the transgenes.introduced into the cells to occur.<br>
5 4.3.1 Evaluation of the activity of the different promoters by histochemical staining.<br>
The revelation of the expression of β-glucuronidase was carried out by histochemical staining as described by Jefferson et al, (1987). After 48 h  in the culture chamber, each  leaf was cut in 10 two along the longitudinal axis of the central rib. Half of the leaf was incubated in staining buffer for p-glucuronidase (5-bromo, 4-chloro, 3-indolyl glucuronide (X-Gluc) 500 mg/1, Triton xlOO 0.05% in 0.1 M phosphate buffer, pH 7.0) for 48 h at 37°c, whereas the other half was frozen in liquid nitrogen, then 15    stored at -80°C.<br>
After staining, the leaves were bleached by dipping them in two 95% ethanol baths for respectively 3 and 12 h, then rinsed in distilled water and dried flat between two sheets of cellophane. The results of these histochemical stainings are presented in" 20 Figure V. The promoter activity of the various constructions was evaluated by the number of blue spots revealed on each leaf half after two shootings amounting to a total of 2 ug of DNA bearing the GUS reporter gene.<br>
Two categories of promoters were identified. The leaves bombarded 25   with the promoters MPrlll6 and MPrll46 showed on average a number of blue spots significantly greater than 150 (Figure VIII) compared to the number of blue spots obtained by bombarding the leaves in the same conditions and with the same amount of reference control plasmid pMRTl092. The leaves bombarded with the 30 promoters MPrlll7 and MPrll47 show an average number of blue spots comprised between 50 et 150 (cf. Figure VIII) .<br>
To sum up, the chimeric promoters MPrlll6 and MPrll46 enable or promote expression of β-glucuronidase to a level greater than or equal to that obtained using the strong constitutive reference 35 promoter D35S prom.<br><br>
WO 00/58485	PCT7IB00/00370<br>
52   3.2.   Quantification of  the expression of β-glucuronidase with the various promoters, by luminometric enzymatic activity determination.<br>
The frozen leaf halves were ground in a mortar, then the powder 5 was left to thaw in extraction buffer (Tris Phosphate 25 mM pH 7.8, Dithiothreitol 2 mM,  1,2-diaminocyclohexane, N,N,N",N"-tetracetic acid  2 mM, glycerol 10 %, Triton X100 1 %) in a ratio of 1 ml of buffer to 200 mg of tissue. The mixture was homogenised then incubated for 15 min in ice before being<br>
10 clarified by centrifugation tor 5 min at 16060 g.<br>
GUS  activity was measured on 20  ul of clarified crude leaf" extract using a "GUS-Light chemiluminescent reporter gene assay" detection kit (Tropix Inc., Bedford/ USA)  according to  the supplier"s recommendations. The measurement of light emission was carried out<br>
15 using a Lumat LB 9507 luminometer (EGG-Berthold, Bad Wildbad, Germany).<br>
The luciferase activity was measured on 20 ul crude leaf extract using a "Luciferase assay system" detection kit (Promega Corp., Madison, USA) according to the supplier"s recommendations. The 20 measurement of light emission was carried out  using a Lumat LB 9507 luminometer.<br>
The results are presented in Figure VI. For each experiment [one bombarded leaf = one crude extract), the ratio between β-glucuronidase and luciferase activities measured by the 25 luminometer, was calculated. The average of the different<br>
experiments for a given construction and the standard mean error were determined. The results obtained show that : The promoters MPrlll6 and MPrll46 (Fig. IV) seem to significantly increase expression over that of the double 35S CaMV reference 30    promoter (MPrl092, Fig. II). The promoter MPrll46 differs from promoter MPrlll6 by the insertion of a duplication of the "as-2" box preceding the "as-1" box of the 35S CaMV promoter located between the "as-1 like" box of the CoYMV promoter,and the green tissue specific element of the CsVMV promoter. This insertion of 35    elements from the 355 CaMV  promoter in MPrll4 6 seems to slightly increase the average degree of expression in relation to promoter<br>
47<br><br>
WO 00/58485	PCT/IB00/00370<br>
53 MPrlll6 (4.5 %).   These elements seem to be implicated in a   psitive synergy in "such a combination.<br>
The promoters MPrlll6 and MPrlll7 (Fig. IV) only differ with respect to the addition of an "as-1 like" box from the CoYMV 5 promoter in MPrlll7. The average expression conferred by the promoter MPrlll7 is significantly lower than that obtained with MPrlll6 (22 ft). This result suggests that the "as-1 like" element from CoYMV located in the 5" region of the chimeric promoter plays a repressor role in the promoter activity. 10 The promoter MPrll47 (Fig. IV) differs from promoter MPrlll7 by the insertion of the "as-1" and "as-2" boxes between the "as-1 like" element from the CoYMV promoter and the green tissue specific element from the CsVMV promoter. The  addition of these boxes and interaction with the other elements seems to lead to a 15 significant reduction in the average rate of expression with<br>
respect to MPrlll7 (43 %). The association of these elements seems to favour a negative synergy. This can be used in constructs where the level of expression required in the transformed cell is relatively low, for example when providing antibiotic or herbicide. 20 resistance for selection or marking purposes. Nevertheless, the weakest expression conferred by the promoter MPrll47 only differs by 51 % to that obtained by the promoter MPrl092. This negative effect could be explained in the case of the chimeric" promoter, by a competition with trans-activating elements or else by a steric 25 hindrance effect of these factors on the promoter, thus diminishing its activity.<br>
To conclude, the chimeric promoters MPrlll6 and MPril46 seem to cause an average expression of the GUS reporter gene in tobacco leaves that is noticeably better than that obtained by MPrl092. 30 This latter promoter is commonly reported in the literature as being the strongest chimeric promoter (of the order of 10 times greater than the strong constitutive promoter CaMV p35S (Kay et al.,   1987), and routinely used for transgenesis. MPrlll6 and MPrll46 can thus be classed among the strongest chimeric promoters 35 known to date.<br>
The weaker expressing promoters can be of interest as mentioned above as promoters used for conferring antibiotic resistance for<br>
-48-<br><br>
.  WO 00/58485	PCT/IBOO/00370<br>
54<br>
the purposes of selection, for example in the same way as promoters of the "nos"  type,<br>
"Figure IX illustrates complementary results. For each experiment, the ratio between β-glucuronidase activity and luciferase activity 5 measured with the luminometer was calculated. The mean  average of the different experiments for a given construction and standard mean error were determined. The results obtained show that :<br>
- the promoters MPrll63 and MPrll65 (cf. Figure VII), much as the promoters MPrlll6 and MPrll46 (cf. Figure VII) are responsible 10 for an average .expression of the reporter gene slightly greater, by 8%, 5%, 6.5% and 11.5% respectively, than that obtained by the reference promoter MPrl092. The promoters MPrll63 and MPrll65 differ from the promoter MPrlll6 by the insertion of respectively 2 and 4 series of as-2/as-2/as-l boxes immediately downstream of 15    the green tissue specific region from the CsVMV promoter. This insertion of multiple boxes from the CaHV 353 promoter into MPrlll6 does not enable an increase in the average degree of expression with respect to that of promoter MPrlll6.<br>
- the comparison of the average activity of the promoters 20 MPrll46 and MPrll62 reveal that cloning into MPrlll6 of the series of activating elements "as-2/as-2/as-l" from the CaMV promoter into the 5" region of the green tissue specific region of the CsVMV promoter, which is the case  for MPrll46, is more favorable than cloning these same elements into the 3" region of this 25 sequence, which is the case for MPrll62. This data indicates that the position of the activating elements with respect to each other is significantly related to the capacity of the promoter to transcribe efficiently. There is thus a synergy between the activating elements. 30     - the comparison of the average activity of the promoters MPrll62, MPrll63 and MPrll65 reveals that multiplication of the as-2/as-2/as-l boxes immediately downstream of the green tissue specific region of the CsVMV promoter does not provide a significant propotional increase in the activity of these 35 promoters in transient expression. This data tends to show that there is no positive synergy between the whole of these activating element boxes assembled in such combinations. Without wishing to<br><br>
WO 00/58485	PCTYIBOO/00370<br>
55 be Limited by theory, this could be explained by competition with tans-activating elements or else, by steric hindrance problems of these trans-activating elements on the promoter.<br>
- the reported negative effect of the "as-1 like" box from 5 CoYMV that had been mentioned above during comparison of the activity of the promoters MPrlll6, which has  only one such box, and MPrlll7, which has two such boxes, is confirmed. The deletion of these "as-1 like" boxes in promoter MPrll54 with respect to promoter MPrll47 enables a significant 65% increase in the 10 activity of the promoter.<br>
- the promoters MPrll62 and MPrl164  {Figure VII) only differ from the orientation of the series of as-2/as-2/as-l boxes from CaMV downstream of the CsVMV sequence.. No significant difference between these two promoters was observed. The orientation of these 15 boxes does  not therefore appear, at least in the current<br>
configuration, to have any repercussions on the activity of the chimeric promoter.<br>
To conclude, the chimeric promoters MPrll63 and MPrll65 seem to provoke an average expression of the GITS reported gene in tobacco 20 leaves at least as great as  that obtained with the promoter<br>
MPrlG92. The chimeric promoters MPrll63 and MPrll65, much as the promoters MPrlll6 and MPrll4 6 can thus be classed among the strongest chimeric promoters described to date and can therefore be used routinely in transgenesis programs for dicotyledonous 25    plants, as a substitute to the CaMV D35S (double 35S or enhanced)<br>
r<br>
promoter.<br>
4,4. Corn bombardment and transient expression.<br>
The bombardment of various corn tissues, and among others young 30 leaves and albumen, was carried out with a Biolistic PDS-1000/He gene gun system using the general recommendations of the supplier (BioRad, Hercule, USA) in relation to the manipulations and assembly of the different components of the apparatus. Each endosperm was bombarded twice in succession  with tungsten 35 particles of 0,6 urn diameter, using the following shooting conditions : - the helium pressure to accelerate the particles was 6200 kPa<br>
-50-<br><br>
WO 00/58485	PCT/IB00/00370<br>
56<br>
(900 psi),<br>
- the plant sample was placed at 6 cm from the particle<br>
acceleration zone,<br>
- the shooting was.carried out in a vacuum of 27 mm of mercury. 5 Following bombardment, the endosperm were left in position and<br>
incubated for 24 h in the dark in a culture chamber at 26°C, to enable transient expression of the transgenes introduced into the cells.<br>
10 4.5. Evaluation of the activity of different promoters in corn endosperm by histochemical staining.<br>
Revelation of the expression of p-glucuronidase was carried out by histochemical staining as described by Jeffersson et al.   (1987). After 24 h in a culture chamber, each portion of endosperm was 15 incubated for 48. h at  37°C in the presence of the substrate<br>
5-bromo, 4-chloro, 3-indolyl glucuronide, X-Gluc at 500 mg/1 in a 0.1 M phosphate buffer at pH 7.0 to- which Triton. xlOO 0.05% had been added.<br>
After staining, the portions of endosperm-were rinsed in water 20 then stored in a 96% ethanol bath.<br>
The promoter activity of the different constructions were evaluated by the number of blue spots revealed on each portion of endosperm after two bombardments totalling 2 ug of DNA bearing the GUS reporter gene. 25 Analysis of the results of the histochemical stainings revealed that two categories of promoters could be identified. The endosperm bombarded with the promoters MPrl092, MPrlllS, MPrll46 and MPrll47 showed on average relatively few blue spots of small diameter, ranging from 0 to 15 in number. The endosperm bombarded 30 with the promoters MPrll54, MPril62, MPrll63, MPrll64, MPrll67 and MPrll69, and the reference control promoter MPrl218 showed a number of blue spots comprised between 10 and 30, the diameter of which WAS  greater than those obtained with the above previously described promoters. 35<br>
4,6. Evaluation of β-glucuronidase expression in corn endosperm using luminometric enzyme assay measurement.<br>
-51-<br><br>
WO 00/5848S	PCT/IB00/00370<br>
51<br>
Frozen portions of endosperm were ground in a tube in extraction luffer (Tris Phosphate 25mM pH 7.8, Dithiothreitol 2  mM, 1,2-diaminocyclohexane N,N,N" ,N"-tetracetic acid 2 mM, glycerol 10%, Triton X10Q 1%) using 1 ml of buffer for 200 mg of tissue. 5 The mixture was homogenized then incubated for 15 min in ice before being clarified by centrifugation for 5 min at 16060  g. GUS activity was measured on 20 ul of clarified crude extract usig a "GUS-Light chemiluminescent reporter gene assay" detection kit (Tropix Inc., Bedford, USA) according to the supplier"s 10 recommendations. The measurement of light emission was carried out using a Lumat LB 9507 luminometer (EGG-Berthold, Bad Wildbad, Germany).<br>
Luciferase activity was measured on 20 ul crude extract using a "Luciferase assay system" detection kit (Promega Corp., Madison, 15 USA) according to the supplier"s recommendations. The measurement of emitted light was carried out using a Lumat LB 9507 luminometer. The results are presented in Figure VIII. For each crude extract,<br>
the relationship between p-glucuro"nidase activity and luciferase 20 activity as measured by the luminometer, was calculated. The<br>
average of the different experiments for a given construction and standard mean error were determined. The results obtained show that the original chimeric viral promoter MPrlll6 confers an activity 3.8 times greater than that obtained with the reference 25 promoter MPrl218. This shows  that the chimeric promoter, which contains "endosperm like" boxes possesses the necessary signals for expression of the GUS reporter gene in corn seed. The chimeric promoters MPrll62, MPrll63, MPrll64 and Mprll65, which derive from promoter Mprlll6, have an activity comprised between 4.7 and 6 30 times that obtained with MPrl218 and between 1.2 and 1,. 6 times that obtained with MPrlll6. Taking into account the fact that all of the chimeric promoters derived from MPrlH6 possess the same basic sequence comprising the same regulatory elements or boxes apart from the activating elements from the CaMV promoter, the 35 differences in activity observed seem to the due to the activating elements taken from CaMV. The promoters containing a duplication of the "as-1 like" box from<br>
- 52 -<br><br>
WO 00/58485	PCT/IB00/00370<br>
58<br>
"CoYMV, which is the case for promoters MPrll67, MPrll68 and MPrll69, all show on average an activity that is significantly less than the promoters that only contain a single "as-1 like" box, as is the case for promoters MPrll62, MPrll63, MPrll64 and  5 MPrll65. These results seem to confirm the repressor effect of<br>
this element from CoYMV, already observed for transient expression in tobacco. However, the comparison of average activities conferred by the promoters MPrlll6 and MPrll54 shows that the absence of  the "as-1 like" box and its replacement by  an 10    activiating sequence "as-2/as-l". from CaMV in MPrll54, does not lead to an increase in activity with respect to MPrlll6. It would therefore appear that it is the position of the boxes immediately in the 5" region of the 104 bp sequence of CsVMV which is unfavorable for the activity rather than the boxes themselves. 15 Without wishing to be bound by theory, it is likley that the attachment of trans-activators in this position provokes a conformational change in the whole of the "promoter sequence / protein" assembly, and that this is unfavorable, to the attachment of other activating elements and/or the transcriptional machinery. 20 The analysis of the results reveals that the number of repetitions of the combination of "as-2/as-2/as-l" boxes probably conditions the activity of the chimeric promoter. For example, an increase in the average activity between MPrll62, which possesses a single series of elements, MPrll63, which has two such series, and 25 MPrll65, which has 4 such series, was observed, even if the<br>
differences do not appear very significant. In the same way, the activity increases between MPrll69, MPrll68 and MPrll67 which respectively have 1, 2 and  3 series of boxes.<br>
Finally, the orientation of these activating boxes or elements 30 does not seem to have any repercussions on the activity since no significant difference was observed between the promoters MPrll62 and MPrl164.<br>
To conclude, the chimeric promoters created according to the present invention are functional and operational in 35 monocotyledonous plants and show in particular strong actiivity in corn albumen. The promoters MPrll63 and MPrll65 are the promoters that show the greatest activity of the GUS gene in transient<br>
53<br><br>
WO 00/58485	PCT7IB0O/0O37O<br>
59<br>
expression in corn endosperm 12 days after pollenisation. This activity is of a high level since it is about 6 times higher than that obtained with the strong promoter responsible for the active expression of gamma-zein, the major storage protein in corn 5 albumen. These promoters can therefore be routinely used in<br>
transgenesis programs for monocotyledonous plants as an efficient alternative to other heterologous promoteurs.<br>
5. Expression of the different promoters in tobacco after stable 10 transformation.<br>
5.1, Stable transformation of tobacco.<br>
Tobacco transformation (Nicotiana  tabacvm  L., cultivar P0D6) was carried out hy  infecting foliar disks isolated from tobacco plants  aged 6 weeks with recombinant Agrobacteria according to the method 15 described by Horsch et al.   (1985).<br>
During transformation, the Petri dishes were incubated in a culture chamber under the following conditions : temperature of 24°C, photbperiod of 16 h night / 8 h jour, luminous intensity<br><br>
of 200 umol photons,m2.sec-1 , and apart from the initial coculture 20 step, the whole of the callogenesis, regeneration and rooting steps were carried out on different selective media supplemented with Augmentin  (400 mg/1) and kanamycin (200 or  100 mg/ml) ; The various steps and media used are the following : - a coculture  step lasting three days, during .which the Agrobacteria infect the 25 plant cells, on a solid MS30  coculture media (media with MS base (Murashige et Skoog, 1962) supplemented with vitamins (Gamborg et al.,  1968) 4.4 g/1 (Sigma, M0404}, Saccharose,30 g/1, agar 8 g/1 (Merck), pH 5.7) supplemented with Benzyl Amino Purine at 1 mg/1 and Indol-3 Acetic Acid at 0.1 mg/1 . 30 - two bud formation steps lasting two weeks each in a. culture<br>
chamber on a solid MS20 regeneration media (Salts and vitamins, MS 4.4 g/1 (Sigma, MQ404), .Saccharose 20 g/1, agar 8 g/1 (Merck), pH 5.7), supplemented with Benzyl.Amino Purine at 1 mg/1, Indol-3 Acetic Acid at 0.1 mg/1, Augmentin * at 400 mg/1 et kanamycine at 35 200 mg/1.<br>
- a development and rooting step lasting 3 weeks in a culture chamber on a solid MS20 development media supplemented with<br><br>
WO 00/58485	PCTYIB00/00370<br>
60<br><br>
 lugmentin   at 400 mg/1  and  kanamycine at 100 mg/1.<br>
- a repotting step into glass pots in a culture chamber on a solid<br>
MS20 development media supplemented with Augmentin ® at 400 mg/l<br>
and kanamycine at 100 mg/1.<br>
5<br>
5.2. Measurement and comparison of β-glucuronidase activity in regenerated tobacco plants.<br>
β-glucuronidase activity was measured on samples of" first generation transgenic plants, taken at 2 weeks after their 10 acclimatisation in a greenhouse." For each plant, three leaf samples were taken, one from an "aged" leaf (located the basal foliar level),   one  from  a mature leaf (located at a median foliar level), and one from a young leaf (located at the plant apex). Each sample was ground in liquid nitrogen in a mortar and then the 15 powder was  resuspended in extraction buffer . (Tris Phosphate 25 mM pH 7.8, Dithiothreitol 2 mM, 1,2-diaminocyclohexane, N,N,N" ,N"-tetracetic acid 2 mM, glycerol 10 %, Triton X100 1 %)  in a ration of 1 ml of buffer for 200 mg of tissue. The mixture was homogenised then incubated for 15 min in ice before being 20 clarified by centrifugation for 5 min at 16060 g.<br>
GUS activity was measured on 20 u1 clarified crude leaf extract using a "GUS-Light chemiluminescent reporter gene assay" detection kit.(Tropix Inc., Bedford, USA) according to the supplier"s recommendations. The measurement of light emission was carried out 25 using a Lumat LB 9507 luminometer (EGG-Berthold, Bad Wildbad, Germany) .<br>
The quantity of total protein present in the crude extract was measured according to Bradford technique (1976), using a "BioRad protein assay" (BioRad, Munchen, Germany) .<br>
-55-<br><br>
WO 00/SS48S	PCT/IB00/00370<br>
61<br>
Cited_ References<br>
An G. (1986) . Development of plant promoter expression vector and their use for analysis of differential activity of nopaline 5 synthase promoter in transformed tobacco cells. Plant Physiol. 81, 86-91.<br>
Barany F. (1991). The ligase chain reaction in a PCR world. PCR Methods Appl, 1, 5-16..<br>
Birnboim H. C. et Doly J. (1979) . A rapid alkaline extraction 10 procedure for screening recombinant plasmid DNA. Nuc. Ac. Res, 7, 1513.<br>
Bradford M. (1976). A rapid and sensitive method for the detection of microgram quantities of proteins utilizing the principle of protein-dye binding. Anal. Biochem. 72, 24B-254. 15 Fromm M.E., Taylor L.P, et Walbot V. (1986). Stable transformation of maize after gene transfer by electropofation. Nature, 319, 791-793,<br>
Gamborg O.L., Miller R.A. et ojima K. (1966). Nutrient requirements of suspension cultures of soybean root cells. Exp. 20 Cell Res. 50, 151-158.<br>
Gaubier P., Raynal M., Hull G., Huestis G.M., Grellet F., Arenas C, Pages M. et Delseny M, (1993). Two different Em-like genes are expressed in Arabidopsis thaliana seeds during, germination. Mol. Gen". Genet. 238, 409-418. 25 Guexineau et Mullineaux (1993). In  Plant molecular biology labfax, Croy R.R.D. (Ed.), BioS Scientific Publishers, Black well Scientific Publications.<br>
Jefferson R.A., Burgess S.M. et Hirsh D (1986). b-Glucuronidase as a gene-fusion marker. Proc. Nat. Acad. sci. USA, 83, 8447-8451. 30 Jefferson R.A., Kavanagh T.A. et Sevan M.W. (1987). GUS fusions : β-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J. 6, 3901-390.7.<br>
Hanahan D. (1983). Studies on transformation of Escherichia coii with plasmids. J. Mol. Biol. 166, 557. 35 Holsters M., Dewaele D., Depicker A., Messenf E., Van Montagu M. etn Schell J. (1978). Transfection and transformation of Agrobacterium tumefaciens.  Mol. Gen. Genet." 136, 181-187.<br>
56<br><br>
WO 00/58485	PCT/IBOO/00370<br>
62<br>
brsch R.B., Fry J.E., Hoffmann N.L., Eiholtz D., Rogers S.G. et Fraley R.T. (1985) . A simple and general method for transfer ring genes into plants. Science 221,   129-1231.<br>
Kay R., Chan A., Daly M. et McPherson J. (1987). Duplication of 5    CaMV  35S promoter sequences creates a strong enhancer  for plant genes. Science 236, 1299-1302.<br>
Lam E.., Benfey P. N., Gilmartin P. M., Fang R. X. et Chua N. H. (1989). Site-specific mutations alter in vitro  factor binding and change  promoter expression pattern in transgenic plants. Proc. 10 Natl. Acad. Sci. USA, 86, 7890-7894.<br>
Lam E. et Chua N. H. (1989). ASF-2 : a factor that binds to the cauliflower mosaic virus 35S promoter and a conserved GATA box in Cab promoters. Plant Cell, 1, 1147-1156.<br>
Last D. I. et Gray J. C. (1989). Plastocyanin is encoded by. a" 15    single-copy gene in the pea haploid genome. Plant Mol, Biol. 12, 655-666.<br>
Leckie L, Devoto A. and Lorenzo G. (1994) ,  Normalization of GUS by LUC activity from the same cell extract reduces transformation variability. Biotechniques 17, 52-56. 20 Murashige T. et Skoog F. (1962) . A revised medium for rapid growth and bioassays with tobacco tissue cultures. Physiol. Plant. 15, 473-497.<br>
Medberry S. 1., Lockhart B. E. L. and Olszewski N. E.- (1992). The Commelina Yellow Mottle Virus"promoter is a strong promoter in 25 vascular and reproductive tissues. Plant Cell, 4, 185-192, Pwee K.H. et Gray J.C. (1993). The pea plastocyanin promoter directs cell-specific but not full light-regulated expression in transgenic tobacco plants. Plant J. 3, 437-449.<br>
Sanger F., Nicklen S. et Coulson A. R. (1977), DNA sequencing with 30 chain-terminating inhibitors. Proc. Nati. Acad. Sci. USA, 74, 5463-5467.<br>
Torrent M., Alvarez I., Geli M.I,, Dalcol I. et Ludevid D. (1997). Lysine-rich modified g-zein accumulate in protein bodies of transiently transformed maize endosperms. Plant Mol. Biol. 34, 35 139-149.<br>
Vancanneyt G., Schmidt R., O"Connor-Sanchez A., Willmitzer L. et Rocha-Sosa M. (1990), Construction of an intron-containing marker<br><br>
WO 00/58485<br><br>
PCT/IB00/0037Q<br><br>
63<br>
gene : Splicing of the intron in transgenic plants and its use in msnitoring early events in Agrobacterium-mediated plant transformation. Mol. Gen. Genet. 220, 245-250.<br>
Verdaguer B., of Kochko A., Beachy R. N. and Fauguet C. (1996). Isolation and expression in transgenic tobacco and rice plants of the Cassava  Vein Mosaic Virus (CsVMV) promoter. Plant Mol. Biol. 31, 1129-1139.<br>
58<br><br>
WE CLAIM:<br>
1)	Chimeric expression promoter comprising at least one nucleic acid sequence, consisting of a first plant promoter originating from the Commerlina Yellow Mottle Virus (CoYMV), comprising a plant vascular expression promoter region, said plant vascular expression promoter region being replaced with a nucleic acid sequence consisting of a second plant promoter originating from the Cassava Vein Mosaic Virus (CsVMV), and comprising a plant"green tissue expression promoter region.<br>
2)	Chimeric expression promoter as claimed in claim 1, wherein it comprises at least a part of a nucleic acid sequence identified in the sequence listing under the number SEQ. IDOl fused to at least a part of a nucleic identified in the sequence listing under the number SEQ.ID02.<br>
3)	Chimeric promoter as claimed in anyone of the preceding claims, wherein the nucleic acid sequence of the chimeric promoter consists of a sequence selected from the group consisting of the sequences identified in the sequence listing under the numbers SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.lD07, SEQ.ID19, SEQ.lD20, SEQ.ID21, SEQ.ID22, SEQ.ID23, SEQ.ID24 and SEQ.ID25.<br>
4)	Chimeric promoter as claimed in anyone of the preceding claims, wherein it further comprises at least one "endosperm like" box,<br>
5)	Chimeric promoter as claimed in anyone of the preceding claims, wherein it further comprises at least one "as l like" box operably linked to the plant green tissue expression prompter, nucleic acid sequence.<br>
6)	Chimeric promoter as claimed in anyone of the preceding claims, wherein it further comprises at least one "as2" box operably linked to the<br><br>
plant green tissue expression promoter nucleic acid sequence.<br>
7)	Chimeric promoter as claimed in anyone of the preceding claims,<br>
wherein the one or more of the "asl like", "asl", and "as2" boxes are<br>
operably linked upstream or downstream of the plant green tissue<br>
expression promoter nucleic acid sequence.<br>
8)	Chimeric promoter as claimed in anyone of the preceding claims,<br>
     wherein the one or more of the "asl like", "asl ", and "as2" boxes are<br>
operably linked in normal (5"&gt;3") or inverse (3"&gt;5") orientation.<br>
9)	Chimeric promoter as claimed in anyone of the preceding claims,<br>
wherein it comprises at least one "as2/as2/as2" box in normal (5"&gt;3") or<br>
        inverse (3"&gt;5") orientation.        <br>
10)	Chimeric promoter as claimed in anyone of claims 4 to 9 wherein it<br>
comprises at least a sequence selected from the group consisting of the<br>
sequences   identified   in   the   sequence   listing   under   the   numbers<br>
       SEQ.ID03,    SEQ.ID04,    SEQ.ID05,   SEQ.ID06,    SEQ.ID07,    SEQ.ID19, SEQ.ID20, SEQ.ID21, SEQ.ID22, SEQ.ID23, SEQ.ID24 and SEQ.ID25.<br>
11)	Expression cassette comprising at least one nucleic acid sequence<br>
consisting of a first plant promoter originating from the Commelia Yellow<br>
Mottle virus (CoYMV) comprising a plant vascular expression promoter<br>
region, said plant vascular expression promoter region being replaced<br>
with a nucleic acid sequence consisting of a second plant promoter,<br>
originating from the Cassava Vein Mosaic Virus (CsVMV), and comprising<br>
a plant green tissue expression promoter region, the sequences being<br>
operably (linked to a nucleic acid sequence or gene coding for a<br>
polypeptide to be produced, itself operably linked to a transcription<br>
termination nucleic acid sequence.<br><br>
12)	Expression cassette as claimed in claim 11, wherein it comprises<br>
at least a part of a nucleic acid sequence identified in the sequence<br>
listing under the number SEQ.IDOl fused to at least a part of a nucleic<br>
acid   sequence  identified  in  the  sequence  listing  under the  number<br>
SEQ.ID02.	<br><br>
13)	Expression cassette as claimed in claim 12, wherein the promoter<br>
nucleic acid sequence consists of a sequence selected from the group<br>
consisting of the sequences identified in the sequence listing under the<br>
numbers SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID06, SEQ.ID07,<br>
SEQ.ID19, SEQ.ID20, SEQJD21, SEQ.ID22, SEQ.ID23, SEQ.ID24 and<br>
SEQ.ID25.<br><br>
14)	Isolated promoter nucleic acid sequence, wherein it comprises, a<br>
fusion of at least a promoter part of each of the sequences identified in the sequence listing under the numbers SEQ.IDOl and SEQ.ID02.<br>
15)	Isolated promoter nucleic acid sequence, as claimed in claim 14,<br>
wherein it corresponds to anyone of the sequences identified in the<br>
■<br>
sequence listing under the numbers SEQ.ID03, SEQ.ID04, SEQ.ID05, SEQ.ID05, SEQ.ID06, SEQ.ID07, SEQ.ID19, SEQ.ID20, SEQ.ID21, SEQ;ID22, SEQ.ID23, SEQ.ID24 and SEQ.ID25.<br>
16)	Directional desoxynucleotide building block for the construction of a chimeric expression promoter or an isolated promoter nucleic acid sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID08.<br>
17)	Directional desoxynucleotide building block for the construction of<br><br>
a chimeric expression promoter or an isolated promoter nucleic acid sequence as claimed in anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID09.<br>
18)	 Directional desoxynucleotide building block for the construction of<br>
a chimeric expression promoter or an isolated promoter nucleic acid<br>
sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it<br>
corresponds to the sequence identified in the sequence listing under the<br>
  number SEQ.ID 10.<br>
19)	Directional desoxynucleotide building block for the construction of<br>
a chimeric expression promoter or an isolated promoter nucleic acid<br>
sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it<br>
corresponds to the sequence identified in the sequence listing under the<br>
       number SEQ.IDll.<br>
20)	Directional desoxynucleotide building block for the construction of<br>
a chimeric expression promoter or an isolated promoter nucleic acid<br>
sequence according to anyone of claims 1 to 10 or 14 or ,15, wherein it<br>
     corresponds to. the sequence identified in the sequence listing under the number SEQ.ID12.<br>
21)	Directional desoxynucleotide building block for the construction of<br>
       a chimeric expression promoter or an isolated promoter nucleic acid<br>
sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID13.<br>
 22)     Directional desoxynucleotide building block for the construction of a chimeric expression promoter or an isolated promoter nucleic acid<br><br>
sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID14. <br>
23)	Guide desoxynucleotide building block for the construction of a<br>
chimeric   expression  promoter  or  an  isolated   promoter  nucleic   acid<br>
   sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID15.<br>
24)	Guide desoxynucleotide building block for the construction of a<br>
        chimeric   expression  promoter  or  an  isolated   promoter  nucleic  acid<br>
sequence according to anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID16,<br>
25)	Guide desoxynucleotide building block for the construction of a<br>
chimeric expression promoter or an isolated promoter nucleic acid<br>
sequence as claimed in anyone of claims 1 to 10 or 14 or 15, wherein it<br>
,       corresponds to the sequence identified in the sequence listing under the number SEQ.ID 17,<br>
26)	Guide desoxynucleotide building block for the construction of a<br>
chimeric   expression  promoter  or  an  isolated   promoter  nucleic   acid<br>
   sequence as claimed in anyone of claims 1 to 10 or 14 or 15, wherein it corresponds to the sequence identified in the sequence listing under the number SEQ.ID18.<br><br>
27)	Vector   comprising   a   promoter,   or   a   promoter   nucleic   acid<br>
sequence, capable of initiating transcription of a nucleic acid sequence, or gene, coding for a polypeptide to produce, wherein the promoter or the<br><br>
isolated promoter nucleic acid sequence corresponds to a chimeric expression promoter or a promoter nucleic acid sequence as claimed in anyone of claims 1 to 10 or 14 or 15.<br>
28) Method for expressing a nucleic acid sequence, or gene of interest, coding for a polypeptide to produce, by a cell, wherein the method comprises the steps consisting of:<br>
transforming the cell with a vector comprising at least one promoter or at least one promoter nucleic acid sequence as claimed in anyone of claims 1 to 10 or 14 to 15,<br>
culturing the cell under conditions enabling the expression of the   nucleic acid sequence, or gene of interest, coding for the polypeptide.<br>
29]     Method as claimed in claim 28, wherein the cell is prokaryotic or        eukaryotic cell.<br>
30]     Method as claimed in anyone of claims 28 or 29, wherein the cell is a cell selected from the group consisting of bacterial cells, fungal cells, yeast cells, insect cells, human cells, animal cells, algal cells, mibroalgal        cells and plant cells.<br>
31]     Method as claimed in anyone of claims 28 to 30, wherein the cell is         a plant cell,<br>
32) Method for the manufacture of a transgenic plant as claimed in any of the preceding claims, wherein the method comprises the steps consisting of:<br>
transforming a plant cell with a vector comprising at least one<br><br>
promoter or at least one promoter nucleic acid sequence according to anyone of claim 1 to 10 or 14 or 15;<br>
selecting the  plant cell having integrated  the promoter  or the promoter nucleic acid sequence;<br>
propagating the transformed selected plant cell, either by culture or by regeneration of whole chimeric or transgenic plants.<br><br><br>
Dated	this	September	19,	2001.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWNhbmNlbGxlZCBwYWdlKDIxLTYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-cancelled page(21-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWNsYWltcyBncmFudGVkKDI3LTYtMjAwNSkuZG9j" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-claims granted(27-6-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWNsYWltcyBncmFudGVkKDI3LTYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-claims granted(27-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWNvcnJlc3BvbmRlbmNlKDEzLTMtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-correspondence(13-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWNvcnJlc3BvbmRlbmNlLWlwbygyNy02LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-correspondence-ipo(27-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWRyYXdpbmcoMjEtNi0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-drawing(21-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gMSgyMS03LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 1(21-7-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gMTkoMjgtNC0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 19(28-4-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gMiBncmFudGVkKDI3LTYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 2 granted(27-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gMygxOS05LTIwMDEpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 3(19-9-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gMygyMS02LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 3(21-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0gNSgxOS05LTIwMDEpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form 5(19-9-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLWZvcm0yIGdyYW50ZWQoMjctNi0yMDA1KS5kb2M=" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-form2 granted(27-6-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLXBjdC1pcGVhLTQwOSgxOS05LTIwMDEpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-pct-ipea-409(19-9-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM3KDIxLTYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-petition under rule 137(21-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM4KDIxLTYtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-petition under rule 138(21-6-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgxOS05LTIwMDEpNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-power of authority(19-9-2001)5).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMDExNDAtbXVtLXBvd2VyIG9mIGF1dGhvcml0eSgyMS02LTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-01140-mum-power of authority(21-6-2005).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="210987-method-of-manufacturing-optical-an-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="210989-process-for-the-preparation-of-a-hydroxyperoxyester.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>210988</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/01140/MUM</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>45/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>09-Nov-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Oct-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Sep-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERISTEM THERAPEUTICS</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>8 RUE DES FRERES LUMIERE, F-63100 CLERMONT FERRAND, FRANCE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>IAAN RANCE</td>
											<td>54 RUE PRODHOUN, F-63000 CLERMONT FERRANCE, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VERONIQUE GRUBER</td>
											<td>RESIDENCE STE MADELEINE, 44c AVENUE JEAN JAURES, F-63400 CHAMALIERES, FRANCE.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MANFRED THEISEN</td>
											<td>1 RUE DES ROSERAIES, F-63400 CHAMALIERES, FRANCE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82,C12Q1/68,A01H5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB00/00370</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-03-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>99/03925</td>
									<td>1999-03-29</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/210988-chimeric-expression-promoters-originating-from-commelina-yellow-mottel-virus-and-cassava-vein-mosaic-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:59:09 GMT -->
</html>
