<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/211149-an-avian-paramyxovirus-cdna-and-a-method-for-generating-infectious-copy-avian-paramyxovirus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:17:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 211149:AN AVIAN-PARAMYXOVIRUS cDNA AND A METHOD FOR GENERATING INFECTIOUS COPY AVIAN-PARAMYXOVIRUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN AVIAN-PARAMYXOVIRUS cDNA AND A METHOD FOR GENERATING INFECTIOUS COPY AVIAN-PARAMYXOVIRUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An avian-paramyxovirus cDNA at least comprising a nucleic acid sequence corresponding to the 5 &#x27;-terminal end as set forth in SEQ ill No.82, and, corresponding to the 3&#x27; terminal end, as set forth in SEQ ill No:80 or SEQ ill No.81, of the genome of avian-paramyxovirus, said nucleic acid sequence allowing generation of an infectious copy of avian-paramyxovirus, wherein said avian-paramyxovirus is selected from the group consisting of avian paramyxovius type-I, avian paramyxovirus type-2, avian paramyxovirus type-3, avian paramyxovirus type-4, avian paramyxovirus type- 5, avian paramyxovirus type-6, avian paramyxovirus type- 7, avian paramyxovirus type-8, avian paramyxovirus -9</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title:  Newcastle disease virus infectious clones, vaccines and diagnostic assays.<br>
The invention relates to Newcastle disease virus infections of poultry.<br>
Newcastle disease virus (NDV) is one of the most diverse and deadly avian pathogens. The almost simultaneous occurrence of Newcastle disease as an apparent new disease in several different geographical locations and the great variation in type and severity of the disease has caused some problems with nomenclature.<br>
The disease has been termed pseudo fowl pest, pseudo poultry plague, avian pest, avian distemper and avian pneumoencephalitis. The importance of the disease is primarily due to the development of the poultry industry during the 20th Century into a highly efficient international industry which is dependent on intensive trade between countries.<br>
It is generally assumed that the first outbreaks of Newcastle disease occurred in 1926 in Java, Indonesia, and in Newcastle-upon-Tyne, England (Kraneveld, 1926; Doyle, 1927) . The name Newcastle disease was coined by Doyle as a temporary name to avoid a descriptive name that might be confused with other diseases. It later became clear that other less severe diseases were caused by viruses indistinguishable from NDV. In the US a relatively mild respiratory disease was termed avian pneumoencephalitis and was shown to be caused by NDV (Beach, 1944). Within a few years, numerous NDV isolations that caused extremely mild or no disease in chickens were made around the world.<br>
The 'following methods have been implicated in the spread of the disease: 1) movement of live birds, feral birds, game birds, racing pigeons and commercial poultry; 2) movement of people and equipment; 3) movement of poultry products; 4) airborne spread; 5) contaminated poultry feed; 6) contaminated water; 7) incompletely inactivated or<br><br>
heterogeneous vaccines. According to the OIE, Newcastle disease is a disease of poultry caused by a virus of avian paramyxovirus serotype 1 (APMV-l) which has an intracerebral pathogenicity index (ICPI) in day-old chicks of O;7 or greater. Virulent virus can also be confirmed by the presence of multiple basic amino acids at the C-terminus of the F2 protein and F (phenylanine) at residue 117, the N-terminus of the Fl protein. Failure to demonstrate this amino acid sequence would require characterisation by ICPI tests. The word 'poultry' refers to domestic fowl, turkeys, guinea fowl, ducks, geese, quails, pigeons, pheasants, partridges and ratites that are reared or kept in captivity for breeding, the production of meat or eggs for consumption, or for restocking supplies of game.<br>
According to Alexander (1988) three panzootics of Newcastle disease have occurred since the first recognition of the disease- The first represented the initial outbreaks of the disease and appears to have arisen in South-East Asia. Isolated outbreaks, such as the one in England in 1926, were chance introductions ahead of the mainstream which slowly moved through Asia to Europe.<br>
A second panzootic appears to have begun in the Middle East in the late 1960's and reached most countries by 1973 The more rapid spread of the second panzootic was probably caused by the major revolution of the poultry industry with considerable international trade.<br>
A third panzootic primarily affected domesticated birds such as pigeons and doves (Vindevogel and Duchatel, 198 8). The disease apparently arose in the Middle East in the late 1970's. By 1981 it had reached Europe and then spread rapidly to all parts of the world, largely as a result of contact between birds at races and shows and the international trade in such birds.<br>
Nowadays, Newcastle disease is still widespread in many countries of Asia, Africa, the Americas, and Europe. Only the countries of Oceania appear to be relatively free from the disease (Spradbrow, 1988) .<br><br>
NDV belongs to the order Mononegavirales,   family Paramyxoviridae,   subfamily Paramyxovirinae,   genus Rubulavirus. Apart from NDV, generally called avian paramyxovirus type-1, eight other serotypes, designated avian paramyxovirus type-2 to' -9, can be distinguished on the basis of their antigenic relatedness in hemagglutination-inhibition tests and serum neutralisation tests (Alexander, 1993) .<br>
Despite the consistency of the serological grouping there are some cross-relationships between viruses of the different serotypes.<br>
The genome of NDV is a single-stranded RNA molecule of negative polarity, complementary to the messenger RNA's which code for the virus proteins. The RNA genome is approximately 15,200 nt in size and codes for the following gene products (listed from the 3' end to the 5' end of the genomic RNA) : nucleocapsid protein (NP), phosphoprotein (P), matrix protein (M), fusion protein (F) , hemagglutinin-neuraminidase (HN), and large polymerase protein (L) (Chambers et al., 1986).<br>
The RNA is complexed with the NP, P and L proteins to form a ribonucleocapsid particle (RNP) that is surrounded by an envelope that is lined at the inside by the M protein. The envelope contains the F and HN proteins which are involved in attachment and penetration of the host cell.<br>
Replication of NDV is similar to the strategy used by other paramyxovirinae. The initial step is attachment of the virus to the host cell receptors, mediated by the HN protein. Fusion of the viral envelope with the host cell membrane is dependent on the action of both the HN and F proteins and results in the release of the RNP into the cytoplasm where virus replication.takes place.     <br>
The viral RNA-dependent RNA polymerase (which is part of the RNP) produces complementary transcripts that act as mRNA's and are used by the cell's translation machinery for the synthesis of virus proteins. Due to the accumulation of NP protein, the RNA polymerase complex switches from<br><br>
transcription to replication, resulting in the synthesis of full-length genomic and antigenomic RNA molecules.<br>
Newly formed RNP's are encapsidated at the cellular membrane by the action of the M protein and the F'and HN proteins which have accumulated in the cellular plasma membrane. Newly formed virus particles are released from the infected cell by a budding mechanism. For more detailed information about NDV replication see Peeples (1988) . For a recent review of the molecular biology of paramyxovirinae see Lamb and Kolakofsky (1996).<br>
Apart from commercial domestic poultry (i.e. chickens, turkeys, pheasants, guinea fowl, ducks, geese, pigeons), a wide range of captive, semi-domestic and free-living birds, including migratory waterfowl, is susceptible to NDV and can be primary infection sources (Kaleta and Baldauf, 1988) ,<br>
The pathogenicity of NDV strains differs greatly with the host. The most resistant species appear to be aquatic birds while the most susceptible are gregarious birds forming temporary of permanent flocks. Chickens are highly susceptible but ducks and geese may be infected and show few or no clinical signs, even with strains which are lethal for chickens.<br>
Newcastle Disease is complicated in that different isolates and strains of the virus may induce enormous variation in the severity of the disease. Beard and Hanson (1984) grouped NDV strains and isolates into different pathotypes that relate to disease signs that may be seen in fully susceptible chickens: 1) viscerotropic velogenic NDV, which produces acute lethal infections in which hemorrhagic lesions are prominent in the gut; and neurotropic velogenic NDV, which produces high mortality preceded by respiratory and neurological signs, but no gut lesions; 2) mesogenic NDV, which produces low mortality, acute respiratory disease and nervous signs in some birds; 3) lentogenic NDV, which produces mild or inapparant respiratory infections or even asymptomatic enteric NDV, avirulent viruses that appear to replicate primarily in the intestinal tract. Some overlap<br><br>
between the signs associated with the different groups has been reported-<br>
The virus enters the body via the respiratory and the intestinal tract or via the eye. In the trachea, the virus is spread by ciliary action and by cell-to-cell spread. After initial multiplication at the introduction site, virus is carried during episodes of viraemia to spleen, liver, kidney and lungs. Viruses of some strains reach vital organs like liver and kidney very rapidly so that the birds may die before disease symptoms are overt.<br>
Most viruses reach the central nervous system via the blood before significant amounts of antibody exist. A long, asymptomatic carrier state presumed to occur in psittacines constitutes a potential threat to the poultry industry. A long term carrier state of both lentogenic and velogenic virus may also exist in chickens (Heuschele and Easterday, 1970).<br>
During the replication of NDV it is necessary for the precursor glycoprotein Fo to be cleaved to Fl and F2 for the progeny virus to be infectious (Rott and Klenk, 1988). This posttranslational cleavage is mediated by host cell proteases. If cleavage fails to take place, non-infectious virus particles are produced and viral replication cannot proceed. The Fo protein of virulent viruses can be cleaved by a wide range of proteases, but Fo proteins in viruses of low virulence are restricted in their sensitivity and these viruses can only grow in vivo  in certain host cell types and<br>
in general cannot be grown in  vitro,<br>
Lentogenic viruses only replicate in areas with trypsin-like enzymes such as the respiratory and intestinal tract, whereas virulent viruses can replicate in a 'range of tissues ' and organs resulting in fatal systemic infection.<br>
Amino acid sequencing of the Fo precursor has shown that low-virulence viruses have a single arginine (R) that links the F2 and Fl chains, whereas virulent strains possess additional basic amino acids forming two pairs such as<br><br>
K/R-X-K/R-R-F at the site of cleavage. Furthermore, the F2 chain of virulent strains generally starts with a phenylalanine residue whereas that of nonvirulent strains generally starts with a leucine.<br>
For a few strains of NDV the HN protein is also produced as a precursor that requires cleavage to be biologically active (Garten et al,, 1980; Millar et al., 1988}.<br>
Besides cleavability of the F and HN proteins, other viral factors may contribute to pathogenicity, Madansky and Bratt (1978, 1981a, 1981b) have shown that alterations in transcription and translation could modulate growth and cell-to-cell spread of the virus and/or cytopathogenicity.<br>
The initial immune response to infection with NDV is cell mediated and may be detectable as early as 2-3 days after infection with live vaccine strains. This presumably explains the early protection against challenge that has been recorded in vaccinated birds before a measurable antibody response is seen (Gough and Alexander, 1973).<br>
At about 1 week after infection, circulating antibodies may protect the host from re-infection. In the early phase IgM is involved, followed by IgG, Titres and protection peak after about 3 weeks and gradually decline if there is no boosting. This means that for older birds, re-vaccinations are necessary.<br>
Only live vaccines administered by the respiratory route stimulate antibody in all mucosal surfaces as well as in serum. Inactivated vaccine, even when applied via the intramuscular route, does not elicit local resistance in the respiratory tract, despite high concentIations of serum antibody.<br>
This stresses the importance of live vaccines capable of presenting viral antigen to the upper respiratory tract to induce both local and systemic immunity. Small droplets penetrate into the lower respiratory trace thereby provoking<br><br>
a mainly humoral immune response, while coarse droplets stimulate local immunity in the upper respiratory tract.<br>
Therefor, aerosols with a wide range of droplet sizes generate the best overall local and humoral immunity.<br>
It should be noted, however, that despite intensive vaccination with current vaccines creating high levels of antibody titers, virus can still be recovered from mucous surfaces.<br>
The identification of Newcastle disease in the USA led to the use of inactivated vaccines (Hofstad, 1953), The observation that some of the enzootic viruses produced only mild disease resulted first in the development of the mesogenic live vaccine Roakin (Beaudette et al., 1949) and, subsequently, in the development of the milder Hitchner Bl (Hitchner and Johnson, 1948) and LaSota (Goldhaft, 1980) strains, which are now the most widely used live vaccines.<br>
NDV live vaccines can be divided into two groups, lentogenic and mesogenic. Mesogenic strains are suitable only for secondary vaccination of birds due to their greater virulence. The immune response increases as the pathogenicity of the live vaccine increases. Therefore, to obtain the desired level of protection without serious reaction, currently vaccination programs are used that involve sequential use of progressively more virulent vaccines, or live vaccines followed by inactivated vaccines.<br>
On of the main advantages of live vaccines is that they may be administered by inexpensive mass application techniques. A common method of application is via drinking water. However, drinking water application must be carefully monitored as the virus may be inactivated by excessive heat and light and by virucidal impurities in the water.<br>
Mass application of live vaccines by sprays and aerosols is also very popular due to the ease with which large numbers of birds can be vaccinated in a short time. It is important to achieve the correct particle size by controlling the conditions under which the particles are generated.<br><br>
Currently used live vaccines have several disadvantages. The vaccine may still cause disease signs, depending upon environmental conditions and the presence of complicating infections. Therefore, it is important to use extremely mild virus for primary vaccination and, as a result, multiple vaccinations are usually needed. Furthermore, maternally derived antibodies may prevent successful primary vaccination with lentogenic live vaccines.<br>
Inactivated vaccines are usually produced from infectious allantoic fluid which is treated with formalin or betapropiolactone to kill the virus and mixed with a suitable adjuvant. Inactivated vaccines are administered by injection, either intramuscularly or subcutaneously. Inactivated vaccines are expensive to produce and to apply.<br>
However, inactivated oil-emulsion vaccines are not as adversely affected by maternal immunity as live vaccines and they can be used in day-old chicks. Advantages of inactivated vaccines are the low level of adverse reactions in vaccinated birds, the high level of protective antibodies, and the long duration of protection. None of the above vaccines can serologically be differentiated from wild-type NDV.<br>
The development of recombinant viral vaccines has been of interest to the poultry industry for a number of years. The concept is to insert genes of critical immunising epitopes of a disease agent of interest into a nonessential gene of a vector virus. Vaccination with the recombinant virus thus results in immunisation against both the vector virus as well as the disease agent of interest.<br>
Several types of viruses have been evaluated as potential live viral vaccines for poultry. Two avian viruses that have received most attention are fowlpox virus (FPV) and herpesvirus of turkeys (HVT). Fowlpox virus is a DNA virus that has a large genome and hence is considered to have ample room to carry foreign genes.<br>
When attenuated, FPV does not cause clinical disease and is commonly used as a vaccine in chickens. HVT is also a DNA<br><br>
virus and is classified as serotype III of the Marek's disease virus (MDV) family. HVT is non-pathogenic for chickens yet cross-protective against MDV and is commonly used to vaccinate chickens against Marek's disease.<br>
It has been shown that protection against Newcastle disease can be induced by using recombinant HVT or FPV vaccines {Morgan et al., 1992, 1993; Heckert et al., 1996; Boursnell et al., 1990; Taylor et al., 1990).<br>
However, the onset of protection against Newcastle disease following vaccination with such recombinant vaccines that express either the NDV F protein or both the F and HN proteins was severely delayed compared to that following vaccination with a conventional live or inactivated NDV vaccine, possibly because the recombinant vaccines do not provide a wide enough immunological spectre of antigenically relevant NDV epitopes other than those found on the NDV protein that is expressed by the recombinant vaccine or are not properly presented to the immune system.<br>
Furthermore, local (mucosal, respiratory or enteric) protection was not effectively induced in birds vaccinated with the recombinants. This is a serious drawback since vaccines used for primary vaccination against respiratory diseases must induce local immunity to prevent infection and spread of virulent viruses that infect chickens reared under field conditions.<br>
Antibodies against NDV which are capable of protecting the host can be measured in virus neutralisation tests. However, since the neutralisation response appears to parallel the haemagglutination inhibition (HI) response, the latter test is frequently used to assess the protective response, especially after vaccination.<br>
Antibodies against both the F and HN proteins can neutralise NDV. However, antibodies against the F protein appear to induce greater neutralisation than those directed against HN in in  vivo  and in  vitro  tests (Meulemans et al. ,<br>
1986).<br><br>
The presence of specific antibodies to NDV in the serum of a bird gives little information on the infecting strain of NDV and therefore has limited diagnostic value.<br>
The omnipresence of lentogenic NDV strains in birds in most countries and the almost universal use of live vaccines that can not be distinguished, at least not serologically from wild-type NDV, means that the mere demonstration of infection is rarely adequate cause for control measures to be imposed. Since field disease may be an unreliable measure of the true virulence of the virus, it is necessary to further characterise the virus that is found.<br>
At present, the only method of Newcastle disease diagnosis which allows characterisation of the infecting strain, is virus isolation followed by pathogenicity testing. At present, three in vivo  tests are used for this purpose: 1)<br>
mean death time (MDT) in eggs; 2) Intracerebral pathogenicity index (ICPI) in one-day-old chickens; 3} Intravenous pathogenicity index (IVPI) in 6-week-old birds.<br>
These tests suffer from a number of drawbacks, such as the availability of animals, poor reproducibility, and the relatively long duration of the tests. Last but not least, these tests do not allow a simple serological identification of poultry vaccinated with a vaccine or infected with a wild-type strain.<br>
As an alternative to in  vivo  tests, the polymerase chain<br>
reaction (PCR) has been successfully used to distinguish between virulent and non-virulent i and non-virulent isolates (Staiiber et al. , 1995; Kant et al., 1997), however, again<br>
serological differentiation is not possible.<br>
»<br>
The raising of poultry and trade of their products is now organised on a international basis,   frequently under management of multinational companies. The threat of Newcastle disease has proven a great restraint on such trade.<br>
Successful control of Newcastle disease will only be approached when all countries report outbreaks. However,<br><br>
international agreements are not simple due to enormous variation in the extent of disease surveillance in different countries. Some countries do not vaccinate and would not want any form of NDV introduced in domestic poultry because vaccinated poultry cannot be distinguished from those infected with wild-type NDV.<br>
Others only allow the use of specific live vaccines and consider other vaccines as unacceptably virulent. Yet other countries have the continued presence of circulating highly virulent virus, which is not recognised as such because overt disease is masked by vaccination.<br>
In many countries legislation exists to control Newcastle disease outbreaks that may occur. National control measures are directed at prevention of introduction and spread. Most countries have restrictions on  trade in poultry products, eggs, and live poultry. Most countries have established quarantine procedures for importation, especially for psittacine birds.<br>
Some countries have adopted eradication policies with compulsory slaughter of infected birds, their contacts, and products. Others require prophylactic vaccination of birds even in the absence of outbreaks, while some have a policy of ring vaccination around outbreaks to establish a buffer zone.<br>
Clearly, there is a need for better vaccines and for better diagnostic methods which can be used to control Newcastle disease. Due both to large differences in the dose that is received by individual birds during mass application of live vaccines and to variation in levels of maternal immunity in young chickens, post-vaccination reactions with live vaccines are inevitable. This is one of the main concerns of farmers in countries where vaccination is compulsory.<br>
Furthermore, many vaccines are mixtures of sub-populations. When cloned,' these sub-populations may differ significantly from each other in immunogenicity and pathogenicity (Hanson, 1988).<br><br>
However, the largest drawback of currently used live vaccines and inactivated vaccines is the fact that vaccinated animals cannot be distinguished from infected animals with currently used screening techniques such as heamagglutination-inhibition or virus neutralisation tests.<br>
Virulent field-virus may still spread in vaccinated flocks since disease symptoms are masked by vaccination. Since virus isolation and characterisation of virulence by in<br>
vivo  techniques is not feasible on a large scale, there is a<br>
great need for new and effective attenuated live vaccines which can be serologically discriminated from field-viruses.<br>
Such vaccines, called NDV marker vaccines (and accompanying diagnostic methods and kits) , which should provide the fullest possible immunological spectre of antigenically relevant NDV epitopes, and yet should be serologically distinct from wild-type NDV are not yet available.<br>
The invention provides a method to.modify an avian-paramyxovirus genome by genetic modification, provides genetically modified avian-paramyxovirus and an avian-paramyxovirus marker vaccine.<br>
The advent of modern molecular biological techniques has allowed the genetic modification of many RNA viruses, including negative-strand RNA viruses. This technique is often referred to as "reverse genetics". One first provides a (full-length) cDNA copy of the viral RNA, after which one transcribes this DNA in susceptible cells to produce infectious RNA which can again replicate to produce infectious virus particles.<br>
In general, by previous modification of the cDNA with standard molecular biological techniques, it is possible to obtain a genetically modified RNA virus. However, this has never materialised for NDV or other avian-paramyxoviruses, it has even not yet been possible to generate minigenome<br><br>
fragments or plasmids of avian-paramyxovirus genomic fragments to study replicative events of avian-paramyxovirus, thereby creating an understanding on how to construct infectious copy virus.<br>
Surprisingly, although in this description it has now been fully established that the genome of avian-paramyxovirus is the smallest of all paramyxovirus genomes sequenced up to now, especially the 5' terminal end sequence of the NDV genome is much longer than previously had been established and was expected by comparison with other Paramyxoviridae.<br>
The invention now for the first time provides a full sequence of an avian-paramyxovirus genome and provides full-length or minigenomic length cDNA of such a virus.<br>
The invention herewith provides avian-paramyxovirus cDNA at least comprising a nucleic acid sequence corresponding to the 5'-terminal end of the genome of avian-paramyxovirus allowing generating an infectious copy of avian-paramyxovirus, said cDNA preferably comprising a full-length cDNA. However, the invention also provides cDNA at least comprising a nucleic acid sequence corresponding to the 5'-terminal end of the genome of avian-paramyxovirus thereby allowing generating an replicating avian-paramyxovirus minigenome. Such minigenomes can advantageously be used to -transcribe RNA and/or express protein from modified nucleic acid sequences. The invention provides a cDNA according to the invention at least partly derived from Newcastle Disease Virus, for example wherein said Newcastle Disease Virus is a lentogenic virus, preferably derived from a vaccine strain, such as LaSota strain ATCC VR-699.<br>
The invention furthermore provides a cDNA according to the invention additionally provided with a modification, such as a deletion, insertion, mutation, reversion, or otherwise in a nucleic acid. For example a cDNA is provided wherein said modification comprises a nucleic acid encoding a modified protease cleavage site, for example wherein said<br><br>
cleavage site is a protease cleavage site of the fusion (F) protein.<br>
In yet another embodiment, the invention provides a cDNA according to the invention wherein said modification comprises a nucleic acid encoding a hybrid viral protein, such as a hybrid hemaglutinin-neuraminidase (HN) protein as described in the experimental part of the invention. The invention also provides a cDNA according to the invention wherein said modification comprises a deletion in a nucleic acid encoding a viral protein, such as a matrix (M) protein.<br>
The invention additionally provides a cDNA according to the- invention additionally provided with a nucleic acid encoding an heterologous antigen, preferably wherein said antigen is derived from a poultry pathogen, as for example described below. An RNA, and protein derived thereof, obtained from a cDNA according to the invention is also provided.<br>
In recent years, a number of non-segmented negative-strand RNA viruses has been fully characterised and fundamental work on the replication and expression of their genomes has culminated in the ability to generate infectious virus entirely by transfecting cells with cloned cDNA of said virus (reviewed by Conzelmann, 1996) .<br>
To date, infectious virus of non-segmented negative-strand RNA viruses has been generated from cloned cDNA of for example rabies virus (Schnell et al., 1994, Conzelmann; EP0702085A1) . (Schnell et al. , 1994; EP0702085A1) , vesicular stomatitis virus (Lawson et al . , 1995; Whelan et al., 1995), Sendai virus (Garcin et al., 1995), measles virus (Radecke et al., 1995; Schneider et al. , 1997; EP0780475A1) , human .respiratory syncytial virus (Collins et al. , 1995), rinderpest virus (Baron and Barrett, 1997), and human parainfluenza virus type 3 (Hoffman and Banerjee, 1997, Conzelmann; P0702085A1) , (Schnell et al. , 1994; EP0702085A1) .<br>
However, all of above infectious copy viruses are capable of growing both in  vivo  as well as in  vitro  in hosts.<br><br>
tissues or cells of various origin, allowing easy cDNA transfection and replication and generation of infectious virus particles on a suitable cell line.<br>
Such possibility does not exist for NDV, certainly not for lentogenic NDV strains which can provide a vaccine. Virulence of such an NDV strain is associated with its ability to replicate in a wide range of cells, reflected by the fact that virulent strains can easily replicate in  vitro<br>
and in  vivo,   whereas vaccine strains can only replicate in vivo.<br>
Thus, with NDV a catch 22 situation is apparent. While attempts to generate an infectious copy virus from for example infectious cDNA may possibly result in infectious virus, such virus is in general not suitable for use as a vaccine because the thus generated infectious virus is by default too virulent to be used as vaccine; the fact that it can be generated and replicated after transfection of cDNA on a cell line reflects its easy cleavability of the Fo protein into Fl and F2, as discussed above a hallmark of virulence of a NDV.<br>
Using a vaccine strain as parent material for the cDNA would not solve this problem; a vaccine strain, especially of a lentogenic type does not contain an easily cleavable Fo protein, rendering it impossible for first generation virus to continue to replicate. The cell used for transfection will simply not be susceptible to support one or more rounds of replication of vaccine-type virus with a non-cleaved Fo protein.<br>
The invention now elegantly provides a solution for this problem, and therewith provides infectious copy NDV, for example for use in a vaccine.<br>
The invention provides a method to generate infectious copy Newcastle Disease Virus comprising transfecting cells, capable of expressing viral NP, P and L proteins for complexing with viral RNA with cloned full-length or genomic-length cDNA of said virus and further comprising incubating<br><br>
said cells in growth medium comprising proteolytic activity allowing cleavage of the Fo protein of said virus.<br>
In our system, co-transfection of a plasmid expressing NP could be omitted. NP is probably expressed from the full length cDNA because the NP gene is the first gene after the 5' end of the antigenomic RNA. Since eukaryotic mRNA's are usually monocistronic, expression of distal genes is not expected. However it is possible to generate full-length cDNA in which the relative positions of the NDV genes are changed. If the first gene of such a cDNA is the P or L gene, it is not necessary to express the corresponding gene product from a co-transfected plasmid.<br>
As an alternative to using full-length cDNA, it is possible to use two or more subgenomic cDNA's which generate replication competent subgenomic RNA's and which together express the full complement of avian-paramixovirus proteins. Even if the RNA's are packaged separately, the resulting virus-like particles can be used for successive rounds of replication by means of co-infection and complementation of gene functions.<br>
In a preferred embodiment, the invention provides a method wherein said proteolytic activity is derived of an enzyme, such as a trypsin-like enzyme, or is derived of a composition comprising said proteolytic activity. In a much preferred embodiment, said growth medium comprises allantoic fluid comprising proteolytic activity. Cleavage of the Fo protein is required for the generation of infectious virus. It is possible to generate infectious virus from lentogenic strain without the addition of exogenous proteolytic activity. By inoculating the supernatant of transfected cells into the allantoic cavity of embryonated eggs, the proteolytic activity which is present in the allantoic fluid is able to cleave the FO protein to generate the fusion-competent F1-F2 complex. Virions with such an activated F protein are able to infect susceptible cells and replication in cells which express the desired proteolytic activity yields infectious progeny. As an alternative to providing the<br><br>
desired proteolytic activity to the supernatant of transfected cells, it is for example possibe to use a cell that is permissive for NDV and which already expresses said proteolytic activity. Such a cell line is used to produce infectious lentogenic NDV without the addition of exogenous proteolytic activity. Such a cell line can also be generated by stable transfecting a cell line with a gene that specifies said activity. Furthermore, it is possible to generate a stable transfected cell line that expresses the wild-type F protein in the virus envelope, thereby providing infectious particles (themselves not provided with genomic information encoding wild-type F protein) with means to enter a cell. Rescue of infectious lentogenic virus is also possible by infection of transfected cells with an NDV helpervirus. An essential requirement for such a helpervirus would be that it can be selected against, for instance by means of neutralizing antibodies which eliminate the helpervirus but which do not react with the lentogenic virus. Finally, one may construct a stably transfected cell line that expresses one, two, or all of the three essential NDV proteins, NP, P, and L. Such cell lines require the co-expression of only a subset of the three essential proteins or no co-expression at all for supporting generating infectious copy virus.<br>
In a preferred embodiment, the invention provides a method wherein said cells used for transfecting are derived of chicken primary or secondary cells or cell-lines. The description provides for example CER or CEF cells, which, as most in  vitro  cells in general, lack the appropriate<br>
proteases which are required to cleave the Fo protein of NDV, for example of strain LaSota. However, cells derived from for' example other birds can also be used.<br>
The invention further provides a method to generate infectious copy Newcastle Disease Virus comprising transfecting cells with cloned full-length or genomic-length cDNA of said virus as for example identified in figure 3 and<br><br>
further comprising incubating said cells in growth medium comprising proteolytic activity allowing cleavage of the Fo protein of said virus, further comprising recovering infectious virus by culturing said cells and inoculating material derived from said cultured cells into the allantoic cavity of embryonated eggs. Said material for example comprises (harvested or freeze-thawed) cells or cell debris -or supernatant derived from said cell culture.<br>
For example, the description describes a method to recover infectious virus, wherein the supernatant of transfected CEF monolayers was inoculated into the allantoic cavity of embryonated eggs. Four days later the allantoic fluid was harvested, analyzed in a haemagglutination assay, and passaged further in eggs.<br>
In addition, the invention provides a method further comprising passaging said infectious copy Newcastle Disease Virus by harvesting allantoic fluid and re-inoculating embryonated eggs.<br>
In a preferred embodiment of the invention, a method is provided wherein said virus is a lentogenic virus, for example derived from an avirulent field-case of NDV or from a vaccine strain of NDV, such as the LaSota strain of NDV. Furthermore, a method is provided to modify an avian-paramyxovirus genome by means of genetic modification which allows the introduction of one or more mutations, deletions, and/or insertions or other modifications. For example, method is provided to attenuate or modify the virulence of avian-paramyxovirus by modifing cDNA, for example encoding a viral protein, such as the V protein, and cloning said modified cDNA into full-length cDNA and generating infectious'copy virus from said full-length cDNA, thereby generating new  NDV strains or new attenuated live vaccines with improved properties.<br>
Apart from attenuation by modification of gene products it is also possible to attenuate avian-paramyxovirus by modification of nucleotide sequences which are involved in transcription and/or replication. Such modifications result<br><br>
in attenuated strains which express wild type like F proteins which are cleavable both in vitro and in vivo  in a wide range of cells and as a result are more immunogenic than the classical vaccine strains.<br>
In a preferred embodiment, the invention provides a method to attenuate or modify the virulence of an avian paramyxovirus such as a Newcastle Disease Virus, comprising modifying a protease cleavage site of a viral protein by modifying cDNA encoding said cleavage site, further comprising cloning said cDNA into genomic length cDNA of e.g. Newcastle disease virus and generating infectious copy Newcastle Disease virus. Said cleavage site is for example a protease cleavage site in the F or HN protein of Newcastle Disease Virus. Attenuation is in general restricted to reduction of virulence, however, it is now also possible to use a relatively a-virulent strain of NDV and provide the progeny of such a strain with increased virulence, for example by providing it with an increased tendency to replicate in a specified cell-type. It is now thus possible to assign distinct virulence attributes to NDV.<br>
The invention provides a method to antigenically modify avian paramyxovirus such as a Newcastle Disease Virus, comprising modifying cDNA encoding at least a part of a viral protein harbouring at least one immunodominant epitope, further comprising cloning said cDNA into genomic length cDNA of Newcastle disease virus and generating infectious copy Newcastle Disease virus.<br>
For example, the invention provides a method to (further) modify NDV, for example using a method to produce an infectious copy of NDV (vaccine) which has been provided, a method to produce a recombinant marker NDV vaccine is provided, a marker vaccine that contains the fullest possible or needed immunological spectrum of antigenically relevant NDV epitopes, and yet is serologically distinct from wild-type NDV because a distinct, serologically relevant epitope or marker has been removed by recombinant techniques. The<br><br>
invention provides a method to modify the antigenic make-up of avian paramyxovirus such as NDV, thus allowing the generation of e.g a live NDV marker vaccine which can be serologically distinguished from avian paramyxovirus field<br>
strains.<br>
In one embodiment, the invention provides infectious<br>
copy NDV wherein the HN protein of NDV has been modified by recombining cDNA encoding a part of said HN protein with cDNA encoding a part of HN protein derived from an  avian-paramyxovirus, for example type 2 or type 4. Said hybrid HN protein serves as a serological marker for the infectious copy NDV strain thus obtained or can serve to change the tropism of the avian paramyxovirus to other cells and/or tissues. These, so called, marker strains as provided by the invention allow the generation of vaccines which are an invaluable tool to assess the prevalence of NDV in commercial flocks around the world. Furthermore, the large-scale application of such marker vaccines will lead to the complete eradication of NDV by a process of intensive screening and stamping out of infected flocks.<br>
Furthermore, a method is provided to generate an infectious copy NDV strain which expresses one or more antigens from other pathogens and which can be used to vaccinate against multiple diseases. Such an infectious copy NDV virus for example comprises a heterologous cDNA encoding a heterologous protein obtained from for example Avian Influenza (AI) (Haemagglutinim (H5 and H7) and Neuraminidase), Avian leukosis virus (ALV) (env protein (gp85)), Chicken anemia virus (CAV) ( VP1+VP2), Marek's disease virus (MDV) (glycoprotein B (gB), gH), Infectious laringotracheitis virus (I-LT) (gB, gH, gD) , •Infectious bursal disease virus (IBDV) (VP2 and VP3), Turkey rhinotracheitis virus (TRT) (fusion (F) protein), Avian paramyxovirus-2,-3,-6 (PMV) (F-protein, Haemagglutinin neuraminidase (HN), or others. Infectious bronchitis virus (IBV) (peplomer protein, nucleoprotein), Reoviruses (sigma protein), Adenoviruses Pneumoviruses, Salmonella enteritidis, Campylobacter<br><br>
jejuni,Escherichia coli, Bordetella avium (formerly Alcaligenes faecalis) , Haemphilus paragallinarum, Pasteurella multocida, Ornithobacterium rhinotracheale, Riemerella (formerly Pasteurella) anatipestifer, Mycoplasmata (M. gallisepticum, M synoviae, M. mereagridis, M. iowae) , or Aspergilli (A, flavus, A. fumigatus).<br>
The invention herewith provides avian-paramyxovirus or strains derived thereof which can be used as a vaccine vector for the expression of antigens from other poultry pathogens. Several properties make NDV an ideal vaccine vector for vaccination against respiratory or intestinal diseases. 1) NDV can be easily cultured to very high titres in embryonated eggs, 2) Mass culture of NDV in embryonated eggs is relatively cheap. 3) NDV vaccines are relatively stable and can be simply administered by mass application methods such as by drinking water or by spraying or aerosol formation. 4) The natural route of infection of NDV is by the respiratory and/or intestinal tract which are also the major natural routes of infection of many other poultry pathogens. 5) NDV can induce local immunity despite the presence of circulating maternal antibody.<br>
It has been shown that NDV has potent antineoplastic, as well as immune-stimulating properties (for a review see Schirrmacher et al., 1998) [Schirrmacher, V., Ahlert, T., Steiner, H. -H. , Herold-Mende, C. , Gerhards, R. and Hagmuller E. (1998) Immunization with virus-modified tumor cells. Seminars in  Oncology  25: 677-696] . Although NDV does not seem<br>
to be able to replicate productively in normal human cells, a selective NDV-mediated killing of human cancer cells was noted. After local NDV therapy, viral oncolysis and complete remissions of human tumor xenografts were observed in nude mice. This has led to the use of NDV for tumor therapy. However, a problem is that such application may be restricted to local treatment.<br>
NDV infection induces interferons, chemokines, and other potentially important gene products, and introduces<br><br>
pleiotropic immune-stimulatory properties into tumor cells. This concept has been used for the production of autologous tumor cell vaccines consisting of fresh operative specimens that have been infected with NDV. This type of vaccine is called autologous tumor vaccine-NDV or ATV-NDV (Schirrmacher et al., 1998). The NDV-infected cells are inactivated by gamma-irradiation which prevents cell division but which still allows replication of NDV in the cytoplasm of infected cells. After inoculation of patients with ATV-NDV, T-cells are recruited through NDV-induced chemokines. Some of these T-cells may express a T-cell receptor that can interact with peptides from tumor-associated antigens in complex with major histocompatibility complex class I molecules at the cell surface. This interaction results in the induction of a cytotoxic T-cell response which results in specific killing of autologous tumor cells.<br>
The invention provides that the repertoire and amount of chemokines and immune stimulatory proteins induced by NDV infection are modulated- The present invention provides a method for generating recombinant NDV that has been modified to incorporate and express (a) heterologous gene{s). Such recombinant NDV may be used to modify the repertoire and amount of immune-stimulatory proteins in infected cells. In one embodiment, the invention provides a recombinant NDV that incorporates and expresses genes encoding human interferons, chemokines or other immune stimulatory proteins. Said recombinant NDV is used for the production of ATV-NDV which is more potent than conventional ATV-NDV. For example: cytokines IFN-a, -p, TNF-a, IL-1, IL-6; chemokines RANTES, IP-10; other genes such as HSP, ACTH, endorphin, iNOS, EPA/TIMP, NFKB . ) The pleiotropic immune-stimulatory properties of NDV may also be used as an adjuvant for vaccination of animals and humans against infectious diseases. In one embodiment of the invention, foreign genes encoding (a) relevant antigen(s) of (an) infectious agent(s) are introduced in the NDV genome and the simultaneous expression of the antigen(s) and the immune-stimulatory<br><br>
proteins by infected cells may induce a potent immune<br>
response against the infectious agent. In another embodiment<br>
of the invention, the immune-stimulating properties of NDV<br>
may be further enhanced by using NDV recombinants' that<br>
1<br>
simultaneaously express antigens and specific immune-stimulatory proteins. In a preferred embodiment, the invention is used to generate an AIDS (acquired immune-deficiency syndrome) vaccine by using NDV recombinants that express relevant antigens of human immune-deficiency virus (HIV), either alone or in combination with immune-stimulatory<br>
proteins.<br>
NDV are also used as an adjuvant for vaccination of animals and humans against infectious diseases. In one embodi-ment of the invention, heterologous or foreign genes encoding (a) relevant antigen(s) of (an) infectious agent(s) are introduced in the NDV genome and the simultaneous expression of the antigen(s) and the immune-stimulatory proteins by infected cells may induce a potent immune response against the infectious agent. In another embodiment of the invention, the immune-stimulating properties of NDV are further enhanced by using NDV recombinants that simultaneaously express antigens and specific immune-stimulatory proteins. In a preferred embodiment, the invention is used to generate an AIDS (acquired immune-deficiency syndrome) vaccine by using NDV recombinants that express relevant antigens of human immune-deficiency virus (HIV), either alone or in combination with immune-stimulatory proteins.<br>
Also, a method is provided to generate a conditional lethal NDV deletion mutant which can be used as self-restricted non-transmissible-(carrier)vaccine.• An NDV deletion mutant was generated which is unable to express the matrix (M) protein which is involved in budding of NDV at the inner cell membrane. The invention provides for example a phenotypically complemented NDV strain that is unable to express the M protein which is still able to  infect cells and spread by means of cell-to-cell transmission. However, the<br><br>
mutant virus is unable to generate infectious progeny on non-complementing cells. This shows that phenotypically complemented NDV deletion mutants can be used as safe self-restricted vaccines which are unable to spread into the environment- Such a non-transmissible vaccine combines the most important advantage of live vaccines, i.e., efficacy, with the most important advantage of killed vaccines, i.e.,<br>
safety.<br>
The invention provides Newcastle Disease Virus, or strains derived thereof, for example by passaging or further cultivation in embryonated eggs or appropriate cells, that is derived from infectious copy virus obtainable by a method provided by the invention.<br>
For example, NDV is provided that has been modified in at least one way to generate infectious copy Newcastle Disease Virus which is attenuated, modified in virulence, antigenically modified, expressing a heterologous antigen or are non-transmissible, or combinations thereof.<br>
Herewith the invention provides NDV vaccines, characterised for example by carrying distinct virulence attributes or distinct antigenic characteristics, be it for marker vaccine purposes and/or for expressing heterologous antigens derived from other pathogens, be it in transmissible and/or non-transmissible form.<br>
Such a vaccine can be a killed or a live vaccine. Preferably, such a vaccine is a live vaccine, however, killed vaccines as provided by the invention are beneficial under those circumstances where a live vaccine is not or only little applicable, for example because of trade restrictions or other conditions set by disease controlling authorities.<br>
The invention herewith also provides a diagnostic method, and corresponding test kit, to detect antibodies against said serologically relevant immunodominant epitope or marker, therewith providing methods and means to execute a method for control and/or eradication of NDV and/or other poultry diseases in poultry. The invention provides new and effective vaccines which can be serologically discriminated<br><br>
from field-viruses and old-type vaccines. Such new vaccines, called NDV marker vaccines, provide the fullest possible immunological spectrum of antigenically relevant NDV epitopes, and yet are serologically distinct from wild-type NDV by applying accompanying diagnostic methods and kits.<br>
The invention provides a method for distinguishing unvaccinated animals or animals vaccinated with a NDV vaccine according to the invention from animals infected with wild-type NDV or vaccinated with an unmodified mesogenic or lentogenic NDV-vaccine strain comprising taking a least one sample (such as serum, blood, eggs or eye fluid) from said animal and determining in said sample the presence of antibodies directed against an immunodominant epitope or marker expressed by said wild-type or unmodified NDV but not by a vaccine according to the invention.<br>
The invention provides a method wherein said antibodies are directed against the HN or F protein of NDV, for example a hybrid protein as described in the experimental part as this description.The invention provides for example a diagnostic method wherein said animal is selected from the group composed of poultry', preferably of chickens.<br>
The invention also provides a diagnostic kit for use in a method to serologically distinguish between animals. In one embodiment of the invention, a simple and rapid haemagglutination-inhibition (HI) test is used to distinguish between vaccinated animals and infected animals. Animals vaccinated with a marker vaccine in which the complete globular head of HN of NDV has been replaced with the corresponding part of HN of another serotype will not induce antibodies to HN of NDV and therefore will not inhibit haemagglutination of erythocytes by NDV virions.<br>
By using marker vaccine virions in the HI test, antibodies against the hybrid HN protein is detected and may used as  a measure for the efficacy of vaccination. As an alternative, an ELISA that detects antibodies against the F protein of NDV is used to measure the efficacy of vaccination.<br><br>
Apart from the HI test, an ELISA can be used to determine the presence of antibodies against HN of NDV. The antigen to be used in such a test is for example HN of NDV that is expressed by recombinant DNA techniques or. a conserved peptide from HN of NDV,<br>
A blocking ELISA may also be used. In this case one or more monoclonal antibodies against conserved epitopes of HN of NDV are used to determine whether competing antibodies are present in samples from vaccinated animals. The ELISA tests can advantagously be used if the marker vaccine contains a chimeric HN protein only or when a few epitopes of HN of NDV are replaced.<br>
The invention is further explained in the experimental part of this description without limiting the invention thereto.<br>
EXPERIMENTAL PART<br>
MATERIALS AND METHODS<br>
Standard cloning procedures were carried out according to Sambrook et al. (1989) unless stated otherwise. All constructions involving DNA fragments which were generated by means of the polymerase chain reaction (PCR) were verified by sequence analysis. In the primer sequences given below, the underlined nucleotides correspond to NDV sequences and the position within the NDV genome is indicated. The nucleotide sequence of restriction sites which were used for cloning are indicated in boldface.<br>
Cells and viruses<br>
CER cells (Smith et al. , 1976) were grown in GMEM/EMEM (1:1) containing 5% foetal calf serum and 2% of an antibiotic mix that contained 1000 U/ml Penicillin, 1000 µg/ml Streptamycin, 20 µg/ml  Fungizone, 500 µg/xnl  Polymixin B, and<br><br>
10 mg/ml Kanamycin. QT35 cells (Moscovici et al., 1977; Cho, 1982) were grown in medium supplied by GibcoBRL/Life Technologies (cat.no. 041-91536; proprietary composition of Fort Dodge) supplemented with 5% FCS and 2% antibiotic mix. QMS cells (Antin and Ordahl, 1991) were grown in M199 medium supplemented with 10 % tryptose phosphate broth, 10% FCS and 2% antibiotic mix,<br>
NDV strain LaSota was obtained from ATCC (ATCC VR-699) and was passaged two times in embryonated eggs. Before we started with the construction and cloning of cDNA, the virus was plaque purified by three rounds of plaque purification on primary chicken embryo fibroblasts (CEF). To this end the virus was titrated on CEF cells cultured in GMEM/EMEM (1:1) containing 5% foetal calf serum, 2% antibiotic mix, 5% allantoic fluid, 30 mM MgCl2, 200 µg/ml DEAE-dextran (Sigma) and 0.8% agar Nobel (Difco) . Virus from the third round of<br>
plaque purification (designated clone E13-1) was grown in embryonated eggs and four days after inoculation the allantoic fluid was harvested and stored in aliquots at -70°C. The fowlpox recombinant virus fpEFLT7pol (Britton et al. , 1996; hereafter called FPV-T7) , which expresses T7 RNA polymerase, was a kind gift of Dr. Michael Skinner and was grown on QT3 5 cells.<br>
Isolation of viral RNA<br>
All manipulations were carried out in RNase-free glassware or plastics and all solutions were made up with RNase-free water which was treated with 1% diethyl-pyrocarbonate (DEPC) and sterilized by autoclaving. Virus was pelleted from allantoic fluid by centrifugation at 21,000 rpm for 70 min in a Beckman SW40 rotor at 4°C. The pellet was resuspended in homogenization buffer (50 mM Tris-HCl pH 7.5, 50 mM NaCl, 5 mM EDTA, 0.5% SDS) and treated with Proteinase K (200 pig/ml) for 90 min at 37°C during constant agitation. The lysate was extracted two times with an equal<br><br>
volume of phenol/chloroform (1:1} pH 5.4 and once with an equal volume of chloroform. The viral RNA was precipitated from the aquous phase by the addition of 0.1 volume of 3M NaOAc pH 5,3 and 2.5 volumes of 100% ethanol. The;precipitate was collected by centrifugation, washed once with 70% ethanol, resuspended in water, and stored in aliquots at -70*^0.<br>
Reverse transcription<br>
Viral RNA (1.5 /µg) was mixed with 500 ng of primer in a volume of 12 /xl and incubated for 10 min at 70°C. Four µxl of 5x RT buffer (250mM Tris-HCl, pH 8.3, 375 mM KCl, 15 mM MgCl2' GibcoBRL/Life Technologies), 2 µl 0. IM DTT and 2 µ1 lOmM dNTP's (2.5 mM each) was added and the mixture was incubated for 2 min at 42°C. Reverse transcription was performed in a final volume of 20 µ1  by the addition of 200 Units of reverse transcriptase (Superscript II; GibcoBRL/Life Technologies) followed by incubation for 60 min at 42°C,<br>
Polymerase Chain Reaction (PCR)<br>
All PCR reactions which were used to determine the 3' and 5' end of the NDV genome (see below) were carried out by using Taq DNA Polymerase (Perkin Elmer) . For the cloning of individual NDV genes or large subgenomic cDNA's, either the proofreading DNA polymerase Pwo, or mixtures of Taq and Pwo (Expand High Fidelity Kit or Expand Long Template Kit) were used according to the instructions of the supplier (Boehringer Mannheim). All samples were incubated for 2 min at 94°C before the start of the indicated number of PCR cycles. After the indicated number of PCR cycles, the samples were incubated at the elongation temperature for at least 3x the duration of the elongation time of the PCR cycle. PCR fragments were purified directly by using the High Pure PCR Product Purification Kit (Boehringer Mannheim) or after<br><br>
agarose gelelectrophoresis by using the QiaexII extraction kit (Qiagen) essentially as described by the suppliers.<br>
Sequence analysis<br>
All sequences were determined by using the PRISM Ready Reaction Dye Deoxy Terminator Cycle Sequencing Kit (Perkin Elmer) . Reaction mixtures (5 µL)   were subjected to 25 cycles of linear amplification (10 sec at 94°C, 5 sec at 50°C, and 4 min at 60°C) in a GeneAmp2400 thermocycler. Subsequently, the reaction mixtures were precipitated with ethanol, washed once with 70% ethanol, resuspended in 15 µl TSR buffer (Perkin Elmer) and heated for 2 min at 94°C before being loaded on an Applied Biosystems AB310 automatic sequencer.<br>
The nucleotide sequences of the primers which were used to sequence the complete genome of NDV strain LaSota were either derived from published sequences or from sequences established during this sequencing project. The primers are shown in Table 1.<br>
Cloning and sequencing of the 3' and 5' termini of the genome of NDV strain LaSota<br>
The nucleotide sequence of the 3' and 5' termini of the NDV genome were determined by using RACE  procedures (rapid amplification of cDNA ends) . NDV RNA was used in a reverse transcription reaction in a final volume of 20µL  by using primer p360 (5 ' -GGCGATGTAATCAGCCTAGTGCTT-3 ' ; nt 14756-14779) which was derived from the published sequence of the L-gene of NDV (Yusoff et al. , 1987). The single-stranded cDNA (2*.5 µ1 of the RT mixture) was added to 8 pmol anchorprimer ALG3 (5'-CACGAATTCACTATCGATTCTGGATCCTTC-3') and ligated overnight at room temperature in 20 /il of a reaction mixture containing 50 mM Tris-HCl, pH 8.0, 10 mM MgCl2, 10 µg/ml  BSA, 25% PEG, 1 mM HCC, 20 uM ATP and 10 units of T4 RNA ligase (New England Biolabs) as described by Tessier et al. (1986), One µ1 of the<br><br>
ligation reaction was used as template in a PCR reaction by using primers p375 (5 ' -CAATGAATTCAAAGGATATTACAGTAACT-3'  : nt 14964-14983) and ALG4 (5' -GAAGGATCCAGAATCGATAG-3 ' ) . The<br>
latter primer is complementary to anchorprimer ALS3 . The PCR<br>
'i<br>
conditions (40 cycles) were as follows: 1 min at 94°C, 1 min at 55°C, and 2 min at 72°C, The PCR products were purified and cloned in T-vector pBluescriptll-TSK (Ichihara and Kurosawa, 1993) . Alternatively, the purified PCR products were treated with Klenow DNA polymerase I to create blunt ends and cloned in the HincII-site of plasmid pGEM4Z (Promega). Thirteen independent clones (8x pBleuscriptll-TSK and 5x pGEM4Z) were sequenced to determine the nucleotide sequence of the 5' end of the genome of NDV strain LaSota. The nucleotide sequence of the 3 ' end was determined by two independent methods. In method I, primer ALG3 was ligated to the 3 ' end of the viral RNA by using T4 RNA ligase as described by SchUtze et al. (1995). The reaction mixture (final volume 10 µl) contained 2.5  fxg  NDV RNA, 100 pmol ALG3, 1 µl lOx T4 RNA ligase buffer (SOOmM Tris-HCl, pH 7,8, 100 mM MgCl2, 100 mM DTT, 10 mM ATP), 1 µ1 DMSO, 1 µl 10 uM hexamine-cobaltchloride, 1 µl  RNasin (Promega) and 10 units of T4 RNA ligase (New England Biolabs) . The mixture was incubated overnight at room temperature and 5 µl  of the ligation reaction was used as template in a reverse transcription reaction by using ALG4 as primer. One µl  of the RT-reaction was used in a PCR reaction by using primers ALG4 and p3 76 (5 ' -GAGCCTTAAGGAGCTGCTCGTACTGATC-3 ' ; nt 137-164) which was derived from the published sequence of the 3 ' end of NDV (Ishida et al., 1986). The PCR conditions were as described above for the 5' RACE.. In method II, the 3' and 5' ends of the viral NDV RNA were ligated to each other by using T4 RNA ligase using the same conditions as described above for method I. Five /xl of the ligation mixture was used as template in a reverse transcription reaction by using primer p360. One /xl of the RT-reaction was used in a PCR reaction by using primers p375 and p376 and PCR conditions described above for the 5' RACE. The PCR products were treated with<br><br>
Klenow DNA polymerase I to create blunt ends and cloned in the Hindi-site of plasmid pGEM4Z (Promega) . Ten independent clones (4 from method I and 6 from method II) were sequenced to determine the nucleotide sequence of the 3'  end of the genome of NDV strain LaSota.<br>
Construction of transcription vector<br>
A low-copy-number transcription vector was constructed by using plasmid pOK12 (Vie.ira and Messing, 1991) as the basic replicon. Plasmid pOK12 was digested with PvuII and the DNA fragment containing the replication origin and the Kanamycin-resistance gene was isolated. This DNA fragment was ligated to an Eco47III-AflII fragment (the Aflll site was made blunt by using Klenow DNA plymerase I) from transcription vector 2.0 (a generous gift of Dr. Andrew Ball; Pattnaik et al., 1992). From the resulting plasmid an Xbal-Nhel fragment was deleted to eliminate as much unique restriction sites as possible. The resulting plasmid was designated pOLTVS (Fig, 1). Transcription vector pOLTVB contains the T7 DNA dependent RNA polymerase promoter followed by unique StuI and Smal restriction sites, the autocatalytic ribozyme from hepatitis delta virus (HDV) and the transcription termination signal from bacteriophage T7. DNA fragments cloned between the StuI and Smal restriction sites can be transcribed either in vitro or in vivo by using T7 RNA polymerase. After transcription, the 5' end of the resulting transcripts contains two G residues encoded by the plasmid. Due to the autocatalytic action of the HDV ribozyme, the 3' end of the transcripts corresponds to the exact terminal nucleotide of the cloned DNA fragment (Pattnaik et al., 1992).<br><br>
Construction of miniaenome plasmids<br>
In order to examine the requirements for replication and transcription of NDV, minigenome plasmids were constructed which contained the 3'  and 5• terminal regions of NDV flanking a reporter gene that replaced all NDV genes (Fig. 2) . DNA fragments corresponding to the 3' and 5' terminal regions of NDV were generated by means of PCR by using Pwo DNA polymerase (30 cyles; 15 sec at 94°C, 30 sec at 50°C, and 30. sec at 72°C and using plasmids containing the 3' and 5' RACE fragments as templates (see above).<br>
The 3*- region (nt 1-119) was generated by using primers 3UIT (5 ' -ACCAAACAGAGAATCCGTGAGTTACGA-3 ' . nt 1-27) and SEAP3 (5'-ArCG;ATACTGgTCAGCArGCTGGCAGAAGGCTTTCTCG-3 ' .   nt 102-119). The<br>
5' region (nt 14973-15186) was generated by using primers SEAP5 (5 ' -GCflrGCTGACCAGTATCGAJTTTACAGTAACTGTGACT-3' , nt 14973-14990) and 5NDV (5 ' -ACCAAACAAAGATTTGGTGAATGACGA-3 ' . nt 15158-15186) . The two DNA fragments were joined in an overlap PCR (the overlap is shown in italics in the primer sequences shown above) by using primers 3UIT and 5NDV. The resulting DNA fragment, which is a fusion of the 3' and 5' end of NDV separated by 2 0 nucleotides, was phosphorylated by treatment with T4 polynucleotide kinase and cloned in both orientations in transcription plasmid pOLTVS (Fig. 1) which was cleaved with StuI and Smal and dephosphorylated with calf intestinal phosphatase (Boehringer Mannheim), Finally, the SEAP-gene (encoding secreted alkaline phosphatase) was recovered from plasmid pSEAP-Basic (Clontech) by digestion with SphI and Clal, and cloned between the SphI and Clal sites between the 3' and 5' ends of NDV. The resulting plasmids were designated pOLTV53 5 and pOLTV553, respectively. In vivo or in vitro transcription using T7 RNA polymerase of plasmid pOLTV53 5 gives rise to antigenomic RNA ( [ + ]-RNA), whereas<br><br>
transcription of plasmid p0LTV553 gives rise to genomic RNA ([-]-RNA).<br>
Plasmids pOLTV53 5N0 to -N5 and pOLTV553N0 to -N5 were generated by inserting self-complementary oligonucleotides in the Clal-site located between the SEAP-gene and the 5' end of NDV in pOLTV53 5 and pOLTV553, respectively (see Fig, 2). The oligonucleotides used were: NO, 5'-CGCGAGCTCG-3'; Nl, 5'-CGCGAGSCTCG-3* ; N2 , 5 '-CGCGAGCGCTCG-3 • ; N3 , 5 '-CGCGAGCWGCTCG 3'; N4, 5'-CGCGAGCATGCTCG-3'; N5, 5'-CGCGAGCASTGCTCG-3 ' (W = A or T; S = C or G) .<br>
Modification of the T7 promoter in plasmids DOLTV535 and pQLTV553<br>
To generate in vitro or in vivo transcripts containing the authentic 5' and 3' terminal ends of NDV, the T7 promoter in plasmids pOLTV53 5 and pOLTV553 was modified such that transcription would start at the first nucleotide of the 3' or 5* terminal end of NDV.<br>
Primers were designed which contained, 1) a Bgll-restriction site, 2) the sequence of the T7 promoter (shown in italics) which was modified such that the two G residues at the end of the T7 promoter were replaced by an A residue, and 3) the 3.'<br>
(nt 1-21) or 5' end (nt 15164-15186) of NDV.  Primers BGL3F2 (5 ' -GATATGGCCATTCAGGCrTAAraCGACrCACTATAACCAAACAGAGAATCCGTGAG-<br>
3') and SEAP3 (see above) were used to generate a DNA fragment containing the modified T7 promoter and the entire 3' end of NDV up to the start of the SEAP gene in pOLTV535. Similarly, a DNA fragment was generated that containing the modified T7 promoter and the entire 5' end of NDV up to'the end of the SEAP gene in pOLTV553 by using primers BGL5F2 (5'-GATATnnrCATTCAGGCrrAATACGACTCACrATAACCAAACAAAGATTTGGTGAATG-<br>
3') and SEAP5. The resulting fragments were digested with Bgll and SphI (3' end) or Bgll and Clal (5' end), respectively, and used to replace the Bgll-SphI fragment in<br><br>
pOLTV535, or the Bgll-Clal fragment in pOLTV553, The resulting plasmids were designated pOLTV735 and pOLTV753, respectively, Plasmids pOLTV735N3 and pOLTV753N3 were generated by inserting a self-complementary oligonucleotide (5'-CGCGAGCWGCTCG-3'; W = A or T) in the Clal-site located between the SEAP-gene and the 5' end of NDV in pOLTV735 and pOLTV753, respectively.<br>
Construction of SEAP-reporter plasmids<br>
Plasmid pCIneoSEAP was constructed by cloning an Xhol-Clal fragment (Clal-site was made blunt by using Klenow DNA polymerase I) containing the SEAP-gene from plasmid pSEAP-Basic ' (Clontech) between the Xhol and Smal sites of the eukaryotic expression vector pCIneo (Promega). The latter plasmid contains the human cytomegalovirus (hCMV) promoter in addition to the bacteriophage T7 promoter. In order to examine and quantitate SEAP expression by transcripts generated from the T7 promoter only, another plasmid was constructed which lacked the hCMV promoter. To this end the hCMV promoter was deleted from pCIneo by partial digestion with Hindlll followed by complete digestion with Bglll. The DNA fragment (nt 756-5469 according to numbering of Clontech) from which the hCMV promoter was deleted was isolated, treated with T4 DNA polymerase to generate blunt ends and recircularized by using T4 DNA ligase. The resulting plasmid was designated pCIneoD. Finally, the SEAP gene was recovered from pSEAP-Bacis as an MluI-AccI fragment and cloned in pCIneoD between the Mlul and Clal sites. The resulting plasmid was designated pCIneoD SEAP.<br>
Transfections<br>
Cells were seeded in 24-well culture dishes, grown overnight to 60-80% confluency, and infected at a m.o.i. of 1 with FPV-T7 for Ih at 37°C. The cells were transfected with 0,5 	µg minigenome plasmid DNA by using 3 µ1 of LipofectAMINE"<br><br>
and OptiMem essentially as described by the supplier (GibcoBRL/Life Technologies) . After incubation for 4h (CER cells) or 16H (QM5 cells) at 37°C the cells were either infected with NDV (Dutch virulent isolate nr. 152608; 200 µl per well) for Ih at a m.o.i. of  5, or left uninfected. The inoculum was aspirated and replaced by 1ml of complete medium and the cells were further incubated at 3 7°C. For co-transfections, cells were grown in 6-well culture dishes and infected with FPV-T7 as described above. The cells were co-transfected with 0,25 /xg minigenome plasmid DNA, 0.4 µg  pCIneoNP, 0.2 µg pCIneoP and 0.2 µg pCIneoL(c) or pCIneo by -using either 8 µl of Lipof ectAMINE or 9 µl  of FuGene"6 (Boehringer Mannheim). In order to generate infectious virus, the minigenome plasmid was replaced by a transcription plasmid that contained the full-length NDV cDNA.<br>
Quantitation of SEAP activity<br>
The amount of SEAP which was secreted into the medium of transfected cells was measured in disposable 96-well plates by using the Phospha-Light " Chemiluminescent Reporter Assay for Secreted Alkaline Phosphatase kit essentially as described by the supplier (Tropix). Chemiluminescense was quantitated by using a liquid scintillation counter (Wallac' 1450 microbeta PLUS).<br>
Cloning and sequencing of cDNA's spanning the entire genome of NDV strain LaSota.<br>
To clone and sequence the entire genome of NDV strain LaSota, large subgenomic cDNA clones were generated by means of RT-PCR and cloned in pGEM-T. First strand cDNA synthesis was performed by using primer 3UIT as described above, and 1 µ1  of the RTreaction was used in a PCR reaction by using the Expand Long Template PCR kit (Boehringer Mannheim). The PCR consisted of 5 cycles of 10 sec at 94°C, 30 sec at 58°C, and 6 min at 68°C,   followed by 10 cycles of 10 sec at 94°C, 30<br><br>
sec at 58°C, and 6 min at 68°C, in which the elongation time at 68°C was increased by 20 sec per cycle. The PCR fragments were cloned in pGEM-T by using the pGEM-T cloning kit essentially as decribed by the supplier (Promega)\ Ligation mixtures were tranformed into E.coli strain SURE II (Stratagene). Two independent RT-PCR reactions (A and B) were performed and each yielded a similar set of cDNA clones. The nucleotide sequence of the subgenomic cDNA clones was determined by using NDV-specific primers (Table 1) and by primers flanking the inserts. After comparsion of the nucleotide sequence of the A and B series of clones, remaining ambiguities were resolved by sequencing relevant regions of a third independent series of cDNA's (C series). The nucleotide sequence of NDV strain LaSota is shown in Fig.3.<br>
Construction of a full length genomic cDNA clone of NDV.<br>
The full-length NDV cDNA was assembled in transcription plasmid pOLTVS by using pOLTV53 5 as the starting plasmid. The DNA fragments were joined at overlaps by using common restriction enzymes as detailed in Fig. 4B. In a series of cloning steps, a plasmid (designated p535-DI) was constructed containing nucleotides 1-3521 and 12355-15186 separated by'a Clal-site that was generated by joining the Clal-sites at position 3521 and 12355. In another series of cloning steps, a plasmid (designated pGEM-B) was constructed which contained part of the NDV genome including nucleotides 3521-12355 (Clal-fragment). To facilitate cloning, the latter Clal-fragment was tagged with the Chloramphenicol resistance (Cm) gene from plasmid pACYC184 (Chang and Cohen, 1978). To this end the Cm-gene was recovered from pACYC184 by means of PCR by using primers CAT-F (5'<br>
GCGTACGTCTAGACTGGTGTCCCTGTTGATACCGG-3') and CAT-R (5 ' -GCTCTAGACGTACGACCCTGCCCTGAACCGACG-3') . The PCR was carried out with Pwo DNA polymerase and consisted of 30 cycles of 30 sec at 94°C, 45 sec at 60°C,   and 60 sec at 72°C. The<br><br>
resulting PCR fragment was digested with BsiWI and cloned in the unique BsiWI site of pGEM-B, yielding pGEM-B(CAT), The Clal fragment from pGEM-B{CAT) was cloned in the unique Clal site of p535-DI, yielding pNDFL{CAT). Finally, the Cm-gene was removed from this plasmid by digestion with BsiWI followed by religation and transformation of E.coli strain DH5a. The resulting plasmid was designated pNDFL+ and contains the entire NDV cDNA sequence cloned between the T7 promoter and the HDV ribozyme in transcription plasmid pOLTVS.<br>
Cloning and expression of individual NDV genes.<br>
DNA fragments containing each of the NDV LaSota genes were generated by means of RT-PCR and cloned in pCIneo. After cloning, all fragments were sequenced by using primers flanking the inserts and by gene-specific primers. NP-gene: Primer 386 (5 ' -GAGCAATCGAAGTCGTACGGGTAGAAGGTG-3 ' . nt 40-69) was used for reverse transcription. Primers 365 (5'-GTGTGAATTCCGAGTGCGAGCCCGAAG-3 ' ; nt 77-94) and 892 (5'-TTGCATGCCTGCAGGTCAGTACCCCCAGTC-3 ' ; ht 1577-1593) were used for PCR by using Pwo DNA polymerase. The following PCR profile (30 cyles) was used; 30 sec at 95°C, 40 sec at 65°C, and 45 sec at 12°C.   The resulting DNA fragment was digested with EcoRI and cloned in pCIneo between the EcoRI and Smal sites. Expression of NP was verified in an immunoperoxidase monolayer assay (IPMA) as described by Peeters et al (1992) by using monoclonal antibody 38 (Russell et al. , 1983) . P-gene: Primer pRTl (5 ' -CAAAGAATTCAGAAAAAAGTACGGGTAGAA-3 ' : nt 1794-1814') was used for reverse transcription. Primers pRTl and p2 (5 ' -GCAGTCTAGATTAGCCATTCACTGCAAGGCGC-J ' ; nt 3053-3071) were used for PCR by using Pwo DNA polymerase. The following PCR profile (30 cyles) was used; 30 sec at 95 °C, 40 sec at 65°C, and 60 sec at 72°C. 'The resulting DNA fragment was digested with EcoRI and Xbal and cloned in pCIneo between the EcoRI and Xbal sites. Expression of P was verified in an IPMA by using monoclonal antibody 688 (Russell et al. , 1983).<br><br>
M-gene: Primer 3UIT (5' -ACCAAACAGAGAATCCGTGAGTTArna.3'   nt 1-27) was used for reverse transcription. Primers NDV5M (5*-nnnTGCTAGCGGAGTGCCCCAATTGTGCCAA-3' ; nt 3268-3288) and NDV3M ■ (5' 1 -TGTCCCCGQGGCAGCTTATTTCTTAAAAGGAT-3 ' ; nt 4368-4389) were used for PCR by using the Expand High Fidelity kit. The PCR consisted of 10 cycles of 15 sec at 95°C, 30 sec at 55°C, and " 2 min at 68°C, followed by 15 cycles in which the elongation time at 68°C was increased for 20 sec per cycle. The resulting DNA fragment was treated with T4 DNA polymerase to create blunt ends, digested with Nhel, and cloned in pCIneo between the Nhel and Smal sites.  Expression of the M protein was verified in an IPMA by using monoclonal antibody 424 (Russell et al., 1983).<br>
F-gene: Primer 3UIT (see above) was used for reverse transcription. Primers NDV5F (5'-<br>
AGGGGCTAGCGATTCTGGATCCCGGTTGG-3 ' ; nt 4508-4526) and NDV3F (5 ' -ACTACCCGGGAAACCTTCGTTCCTCAT-3 ' : nt 6212-31) were used for PCR by using the Expand High Fidelity kit using the conditions described above for the M-gene. The resulting DNA fragment was treated with T4 DNA polymerase to create blunt ends, digested with Nhel, and cloned in pCIneo between the Nhel and Smal sites. Expression of the F protein was verified in an IPMA by using monoclonal antibody 8E12A8C3 (ID-DLO, department of Avian Virology).<br>
HN-gene: Primer 3UIT was used for reverse transcription. Primers NDV5HN (5 ' -GTAGGCTAGCAAGAGAGGCCGCCCCTCAAT-2 ' r nt 6335-6354) and NDV3HN (5 ' CGAGCCCGGGCCGGCATTCGGTTTGATTCTTG-3 ' : nt 8205-8227) were used for PCR by using the Expand High Fidelity kit using the conditions described above for the M-gene. The resulting DNA fragment was treated with T4 DNA polymerase to create blunt ends and after digestion witK Xmal it was cloned in pCIneo between the blunted (Klenow DNA polymerase) Nhel site and the Xmal site. Expression of the HN protein was verified in an IPMA by using monoclonal antibody 86 (Russell et al., 1983).<br>
L-gene: The L-gene was recovered from cDNA clone pGEM-L7a (Figure 4A) by digestion with SacII and Sail. Before<br><br>
digestion with Sall, the SacII site was made blunt by treatment with T4 DNA polymerase. The resulting fragment was cloned in pCIneo between the blunted (Klenow DNA polymerase) Nhel site and the Sall site. The 5' untranslated region between the T7 promoter and the ATG startcodon of 'the L-gene contained 2 out-of-frame ATG codons which might interfere with correct expression of the L protein. Therefore, a new plasmid was constructed in which the first ATG was missing and in which the second ATG was changed to AAG by means of PCR mutagenesis, as follows. Primers 5LE(E) 5 * -CAATGGAATTCAAGGCAAAACAGCTCAAGGTAAATAATACGGG-3 ' ; nt 8332-8374)<br>
and 3LE(B) 5 ' -GTGAATCTAGAATGCCGGATCCGTACGAATGC-3 ' : nt 8847-8870) were used in a PCR reaction using plasmid pGEM-L7a (Fig. 4) as a template. The PCR was carried out by using Pwo DNA polymerase and consisted of 3 0 cycles of 3 0 sec at 94°C, 45 sec at 60°C, and 60 sec at 72°C. The resulting DNA fragment was digested with EcoRI and Xbal and cloned in pCIneo between the EcoRI and Xbal sites, generating plasmid pCIneoL(N). Subsequently, the BsiWI-Sall fragment from pGEM-L7a, which contains the remaining part of the L-gene (nt 8852-15046), was cloned in pCIneoL(N) between the BsiWI and Sail sites, generating plasmid pCIneoL(c). Since antibodies against the L-protein are not available, expression of L could not be checked by immunochemistry.<br>
Introduction of a genetic tag in the F-aene.<br>
To show unambiguously that infectious virus can be generated from cloned full-length cDNA, a genetic tag was introduced in the F gene by means of PCR mutagenesis. To this end, the F-gene was cloned by using two overlapping PCR fragments. The first PCR fragment was generated by using primers NDV5F (see above) and primer F5R (5'-AAAGCgCCGCTGTCTCCTCCCTCCAGATGTAGTCAC-3 ' ; nt 4859-4894). The<br>
residues shown in bold are changes which were introduced in the primer in order to change the amino acid sequence of the<br><br>
proteolytic cleavage site between Fl and F2 from that of the NDV LaSota strain (GGRQGR | L) to that of the consensus cleavage site for virulent NDV strains (GRRQRR | F) . The second PCR fragment was generated by using primers F3F (5'-GGAGGAGACAGCGGCGCTTTATAGGCGCCATTATTGG"3 ' ; nt 4875-4911) and IV09 (5 ' -CTCTGTCGACACAGACTACCAGAACTTTCAC-3 ' ; nt 6246-6266). The PCR was performed with Pwo DNA polymerase and consisted of 25 cycles of 15 sec at 94°C, 30 sec at 55°C, and 2 min at 72°C. The two overlapping PCR fragments (the overlap is shown in italics in the primer sequences) were joined in a second PCR by using primers NDV5F and IV0 9 and by using the same PCR conditions. The resulting fragment, which contains the entire ORF of the F gene and which encodes a virulent consensus cleavage site, was digested with Nhel and Sail and cloned in pCIneo between the Nhel and Sall sites, yielding pCIneoFwt. The StuI-NotI fragment (nt 4646-4952) from pCIneoFwt was used to replace the corresponding fragment in plasmid p53 5-S which had been constructed by inserting the Clal-Scal (nt 3521-10311) from pGEM-B in p535-DI between the Clal and Seal sites (see Fig. 4C). The resulting plasmid was designated p535-SLFwtc] . A PCR fragment containing the Cm-resistance gene from pACYC184 (see above) was cloned as an Xbal fragment into the unique Xbal site (position 6172 in the NDV sequence) of plasmid p535-S [Fwtc] , yielding plasmid p535-S [Fwtc] Cm. Subsequently, the Cm-tagged Apal-Spel fragment (nt 2285-8094) of this plasmid was used to replace the corresponding fragment of the full-length cDNA clone pNDFL+.  Finally, the Cm-gene was removed from this plasmid by digestion with Xbal followed by recircularization using T4 DNA ligase. The resulting plasmid, which contains the genetically tagged full-length NDV cDNA, was designated pNDFL+[Fwt] .<br><br>
resulting fragment contains the T7 RNA polymerase gene located behind the baculovirus plO promoter. The DNA fragment was cloned in plasmid pCIneoO between the EcoRI and Sall sites, generating plasmid pCIneolOV. Plasmid pCIneoO lacks the T7 promoter and was derived from pCIneo by cleavage with Nhel followed by partial cleavage with Seal, filling in the sticky ends with Klenow DNA polymerase and recircularization by using T4 DNA ligase. The baculovirus sequences were removed from pCIneolO7 by digestion with EcoRI and Pad, followed by T4 DNA polymerase treatment to generate blunt ends and recircularizaton. The resulting plasmid was designated pCIneoOO7. Expression of T7 RNA polymerase was verified by co-tranfection of cells with pCIneoOO7 and pPRhOl. The latter plasid contains the E2 protein of classical swine fever virus cloned behind a T7 promoter and containing an internal ribosome entry site (Rene van Gennip, personal communication). Expression of E2 was determined in an IPMA by using monoclonal antibody V4 (Wensvoort et al., 1986). Stably transformed CER cell lines expressing T7 RNA polymerase were generated and isolated as described above except that 10 cm culture dishes were used and the cells were transfected with 5 fig  of pCIneo007 DNA and 25 µl of LipofectAmine. To examine individual cell lines for the expression T7 RNA polymerase, they were transfected with plasmid pPRhOl and expression of E2 (which is dependent on T7 RNA polymerase) was determined in an IPMA by using monoclonal antibody V4. Several cell lines which expressed T7 RNA polymerase were identified. One cell line, designated CER-C9, was used for subsequent experiments.<br>
Cloning and expression of HN-genes and hybrid-HN genes<br>
Primer 3UIT was used to synthesize single-stranded cDNA of NDV and avian paramyxovirus serotype-2 and -4 (APMV2 and APMV4) as decribed above. All subsequent PCR reactions were performed by using 25 cyles of 15 sec at 94°C, 30 sec  at 55°C and 2 min at 72°C.<br><br>
The entire coding region of the HN-gene of APMV2 was recovered by means of PCR by using primers IV03 (5*-GGGGGAATTCCCCATTCAATGAAGGGTCTAC - 3 ' )   and IVO 5 (5'  -GATGGCCGGGTCTTAAACCAGGCTTCGCAATG-3' ) which were derived from the sequence of the HN gene of APMV2 (GenBank accession number D14030). The entire coding region of the HN-gene of APMV4 was recovered by means of PCR by using primers IV06<br>
(5 • - GGGGGAATTCTGGTAGGGTGGGGAAGGTAGC- 3 ' ) and IVO 8 (5 • -ATTGCCCGGGGGGTAACTAATCAGGATCTCAG-3 ' ) which were derived from the sequence of the HN-gene of APMV4 (GenBank accession number D14031) . The resulting PCR fragments were digested<br>
(either directly or after subcloning in pGEM-T), with EcoRI and Xmal and cloned in pCIneo between the EcoRI and Xmal sites. The resulting plasmids were designated pCIneoHN2 and pCIneoHN4, respectively.<br>
Hybrids between the HN-gene of NDV strain LaSota and the HN-genes of APMV2 and -4 were constructed by means of overlap PCR as follows. The N-terminal part (aa 1-141) of the HN-gene of NDV strain LaSota was amplified with Pwo DNA polymerase by using primers IVOIB (5 ' -GTAGGAATTCAAGAGAGGCCGCCCCTCAAT-3 ' ; nt 6325-6354) and IVIO (5' '-AATGAGTTCTTTGCCTATCCCCCC-3 ' : nt 6811-<br>
6834). The C-terminal part of the HN-gene of APMV2 (aa 142-580) was amplified with Pwo DNA polymerase by using primers IVllB (5 ' -GGGGGGATAGGCAAAGAACTCATTCAAGGAGATGCATCTGCAGGC-3 ' )"<br>
and IV05. The resulting PCR fragments were joined in an overlap PCR (overlap shown in italics) by using primers IVOIB and IV05 and by using the Expand High Fidelity enzyme mix. The resulting PCR fragment was digested (either directly or after subcloning in pGEM-T) with EcoRI and Xmal and cloned in pCIneo between the EcoRI and Xmal sites. The resulting plasmid which contains a hybrid HN-gene consisting of aa 1-141 of NDV and aa 142-580 of APMV2 was designated pCIneoHNl/2141.<br>
The C-terminal part of the HN-gene of APMV4 (aa 143-569) was amplified by using primers IV14B (5'-GGGGGGATAGGCAAAGAACTCATTGTAGATGATGCATCTGCAGGCCTAAATTTCC-3 ' )<br><br>
and IV08. This fragment was joined with the N-terminal part of the HN-gene of NDV (see above) in an overlap PCR by using primers IVOIB and IV08. The resulting PCR fragment was digested (either directly or after subcloning in pGEM-T) with EcoRI and Xmal and cloned in pCIneo between the EcoRI and Xmal sites. The resulting plasmid which contains a hybrid HN-gene consisting of aa 1-141 of NDV and aa 143-569 of APMV4 was designated pCIneoHNl/4141<br>
In analogy to the constructions described above, hybrid HN genes were constructed which consisted of aa 1-143 of NDV and aa 144-580 of APMV2, or aa 1-143 of NDV and aa 145-569 of APMV4. For these constructions PCR fragments were obtained by using the following pairs of primers; NDV aa 1-143, primer IVOIB -and IV13 (5 ' 'ATCTACAATGAGTTCTTTGCCTATC-2 ■ ; nt 6816-<br>
6840); APMV2 aa 144-580, primer IV14B (5'-GGQGGGATAGGCAAAGAACTCATTGTAGATGATGCAHCUGCAGGCCllPJ^TTHCC - 3 ' )<br>
and IV05; APMV4 aa 14 5-569, primer IV15B (5'-GGGGGGATAGGCAAAGAACTCATTGTAGATCAAACAGCTGACHACACAGCAG-3 ' ) and<br>
141<br>
IV08. The PCR fragments were digested (either directly or after subcloning in pGEM-T) with EcoRI and Xmal and cloned in pCIneo between the EcoRI and Xmal sites. The resulting plasmids were designated pCIneol/2143 and pCIneol/4143 respectively. To examine expression of the HN proteins,. CER cells or QM5 cells were infected with FPV-T7 for Ih at a m.o.i. of 1, transfected with plasmids pCIneoHN, pCIneoHN2, pCIneoHN4, pCIneoHNl/2143, pCIneoHNl/2143, pCIneoHNl/4143 and pCIneoHNl/4143 and 24h after transfection the monolayers were overlayed with a 1% suspension of chicken erythrocytes in PBS for 45 min at roomtemperature. Subsequently, the monolayers were carefully washed three times with PBS and adhesion of erythrocytes to transfected cells was examined microscopically. To examine induction of cell fusion after co-expression of the HN and F protein, CER cells or QMS cells were co-transfected with pCIneoFwt together with either pCIneo-HNl, pCneoHN2, pCIneoHN4, pCIneoHNl/2<br><br>
pClneoHNl/4141 pCIneoHNl/2143 or pCIneoHNl/4143, After incubation for 2 to 3 days, the monolayers were washed with PBS, stained for 15 min with a Giemsa solution (1:30 dilution in water), and examined microscopically-<br>
Cloning of hvbrid-HN genes in full length genomic NDV cDNA<br>
A synthetic linker, designated HN12, was inserted between the NotI and Spel sites of pGEM-T (Promega) by using oligonucleotides HN12a (5'- GGCCGCATATTCTAGAGTTAACGACTTA-3») and HN12b (5 '-CTAGTAAGTCGTTAACTCTAGAATATGC-3 ' ) . A synthetic linker, designated HN14, was inserted between the NotI and Spel sites of pGEM-T by using oligonucleotides HN14a (5'-GGCCGCATATTCTAGAGTTAACGA-3 ' ) and HN14b (5 ' -CTAGTCGTTAACTCTAGAATATGC-3 • ) . The resulting plasmids were designated pGEM-HN12 and pGEM-HN14, respectively.  These plamids were digested with NotI and Xbal and used to clone the Notl-Spel fragment (nt 3390-7488) from plasmid p535-S[Fwtc]Cm. The resulting plasmids were designated pGEM-HN1/2NS and pGEM-HNl/4NS, respectively. The HN genes of these plasmids were replaced by the hybrid HN genes from plasmids pCIneoHNl/2143 and pCIneoHNl/4143 respectively (see section: Cloning and expression of HN-genes and hybrid-HN genes). To this end, pCIneoHNl/2143 and pCIneoHNl/4143 were digested with Nhel and Smal  and the resulting fragments (containing the hybrid HNl/2143 and hybrid HNl/4143 genes) were cloned between the Nhel and Hpal site of plasmids pGEM-HNl/2NS and pGEM-HN1/4NS, resulting in pGEM+HN12 and pGEM+HN14, respectively. The latter plasmids were used to introduce the hybrid HN genes into the full length genomic cDNA clone of NDV. To this end, plasmids pGEM+HN12 and pGEM-t-HN14 were digested with NotI and Spel and the fragment containing either the HN12 or HN14 gene was used to replace the corresponding fragment of pNDFL+, yielding pNDFL+HNl/2143 Cm and pNDFL+HNl/4143 Cm, respectively. The Cm-gene was removed from these plasmids by digestion with Xbal followed by recircularization using T4 DNA ligase. In order to comply with the "rule-of-six", a<br><br>
linker was inserted into the unique Spel site of these plasmids by using self-complementary oligonucleotides. Linker H2 (5'-CTAGCGAGCGCTCG -3') was inserted in plasmid pNDFL+HNl/2'^' and linker H3 (5 •-CTAGCGAGCWGCTCG-3 ) was<br>
inserted in pNDFL+HNl/4143 yielding plasmids pNDFL+HNl/2143(H2) and pNDFL+HNl/4143 (H3) , respectively.<br>
Elimination of a specific epitope in the HN protein of NDV La Sota<br>
A specific epitope, i.e. amino acids 346 to 354 (PDEQDYQIR), in the HN protein of NDV LaSota that is recognized by MAb 4D6 (Long -et al. , 1986; Meulemans et al., 1986), was eliminated by replacing this sequence by the corresponding sequence of the HN proteins of either APMV-2 (NRTDIQQTI ) or APMV-4 (PDPLQDQIL) . To this end, plasmid pClneoHN (see section: Cloning and expression of individual NDV genes) was used as template to create overlapping PCR fragments. For the APMV-2 sequence the first PCR fragment was generated by using primers IVOl (5 ' -GTAGACGCGTAAGAGAGGCCGCCCCTCAAT-3 ' ) and primer 3HN2 (5' GATAGTTTGCTGTATATCAGTCCGATTGCATGTGTCATTGTATC-GCTTGTATATCAC-3 ' ) . The second PCR was generated by using the primers 5HN2 (5' -AATCGGACTGATATACAGCAAACTATCATGGCCAAGTCTT-CGTATAAGCCTGGAGCC-3 ' ) and NDV3-HN (5 ' -CGAGCCCGGGCCGGCATT-CGGTTTGATTCTTG- 3'). The resulting fragments were combined and used as template for a third PCR by using the primers IVOIB (5'-GTAGGAATTCAAGAGAGGCCGCCCCTCAAT-3') and primer NDV3-HN. For the APMV-4 sequence the first PCR fragment was generated by using primers IVOl and primer 3HN4 (5' TAAGATC-TGATCTTGCAGCGGGTCAGGGCATGTGTCATTGTATCGCTTGTATATCAC-3 ' ) . The second PCR was generated by using the primers 5HN4 (5*-CCTGA-CCGCTGCAAGATCAGATCTTAATGGCCAAGTCTTCGTATAAGCCTGGAGCC-3 ' ) and NDV3-HN. The resulting fragments were combined and used as template for a third PCR by using the primers IVOIB and NDV3-HN. Primers 3HN2 / 5HN2 and  3HN4/5HN4 are partly<br><br>
complementary and contain the genetic codes for the HN2 sequence (NRTDIQQTI) and HN4 sequence (PDPLQDQIL), respectively. The PCR reactions were performed by using the Expand Long Template PCR kit (Boehringer Mannheim) . The PCR consisted of  30 cycles of 10 sec 94°C, 30 sec 58°C and 2 min at 68 °C, followed by 1 cycle of 4 min 68 °C. The PCR fragments were digested with EcoNI and Bsu36I, and cloned between the EcoNI and Bsu36I sites of pCIneoHN. The resulting plasmids were designated pCIneoHNl(HN2e) and pCIneoHNl(HN4e) respectively. Transient expression studied indicated that the modified HN proteins were correctly expressed and transported to the cell surface as judged from haemadsorbtion studies using chicken erythrocytes. Furthermore, MAb 6D4 which is directed against a linear epitope of HN of NDV and which consists of (or at least includes, amino acids 346-354) did not react with the modified HN proteins.<br>
Plasmids pCIneoHNl(HN2e) and pCIneoHNl(HN4e) were digested with Narl and Spel and the fragments containing the modified HN genes were cloned between the Narl and Spel sites of pGEM-HN1/2NS and pGEM-HNl/4NS, respectively. The resulting plasmids, designated pGEM-HNl {HN2e) and pGEM-HNl(HN4e) , were digested with NotI and Spel, and used to replace the Notl-Spel fragment in pNDFL+. The resulting plasmids were designated pNDFL-HN(HN2e)Cm and pNDFL-HN(HN4e)Cm, respectively. The Cm-gene was removed from these plasmids by digestion with Xbal followed by religation. The resulting plasmids were designated pNDFL-HN(HN2e) and pNDFL-HN (HN4e), respectively.<br>
RESULTS<br>
Nucleotide sequence of the 3' and 5' terminal ends of the genome of NDV strain LaSota<br>
The sequence of a putative 3' end of the NDV genome has been published (Ishida et al., 1986) albeit from another NDV<br><br>
strain (D26) than the one used here (LaSota). Yusoff et al. (1987) have published a sequence of the L-gene and a relatively large non-coding region behind the L-gene of NDV strain Beaudette C. However, as shown herein, this sequence did not include the full terminal 5' end of the viral genome which makes it impossible to generate infectious copy NDV. The 3' and 5' terminal ends of the genome of negative-strand RNA viruses fulfil an essential function in replication and transcription (Lamb and Kolakofsky, 1996). Thus, in order to generate a full-length NDV cDNA which can be used to generate infectious virus by means of reverse genetics (Conzelmann, 1996), it is absolutely essential to include the correct 3' and 5' ends of the viral genome. Therefore, we determined the exact nucleotide sequence of both the 3' and 5' ends of the genomic RNA of NDV strain LaSota by using 3' and 5' RACE procedures (rapid amplification of cDNA ends). The 5' end was recovered by means of PCR after ligation of a single-stranded anchorprimer (ALG3) to single-stranded cDNA which was generated by reverse transcription of the 5' end of the genomic RNA. By using a primer (ALG4) that is complementary to the anchorprimer and an NDV-specific primer, PCR products were generated which contained the 5' end.<br>
To clone the 3' end of NDV, the single-stranded achorprimer ALG3 was ligated to the 3 ' end of viral RNA by using T4 RNA ligase and amplified by means of PCR by using primer ALG4 and an NDV-specific primer (method I). Alternatively, the 3' and 5' ends of the NDV RNA were ligated to each other by using T4 RNA ligase and the resulting concatenated RNA was used for RT-PCR by using NDV-specific primers that flanked the ligation point (method II). The 3' and 5' RACE products were cloned in T-vector pBluescriptll-TSK (Ichihara and Kurosawa, 1993) or in pGEM4Z and several independent clones were isolated and sequenced. The results are compiled in Table 2. To enable the direct comparison of the 3' and 5' terminal ends, the sequences are shown in the form of DNA and the 3' end of the genomic strand is represented as the 5' end of the antigenomic strand. At the genomic RNA level the sequence of<br><br>
the 3'-end reads 3 '-UGGUUUGUCUCUUAG whereas the sequence of the 5'-end reads UUUAGAAACAAACCA-5' . The sequence of the 3' end is almost similar to the published 3' terminal sequence of NDV strain D26 (Ishida et al. , 1986). However,'the sequence of the 5' end showed that NDV strain LaSota contains 64 additional nucleotides in comparison with the published sequence of the L-gene of NDV strain Beaudette C (Yusoff et al., 1987). (Figure 6.)<br>
Replication of NDV minicrenomes by helpervirus<br>
To determine whether the 3' and 5' ends of NDV are functional in replication and transcription, minigenomes were constructed which consisted of the 3' end of NDV (nt 1-119), a reporter gene encoding secreted alkaline phosphatase (SEAP), and the 5' end of NDV (nt 14973-15186) (Fig. 2). These minigenomes were cloned in both orientations in transcription vector pOLTVB, generating plasmids pOLTV535 and pOLTV553, respectively (for details of the construction see Materials and Methods). Plasmid pOLTVS (Fig. 1) contains the T7 RNA polymerase promoter followed by unique StuI and Smal restriction sites, the autocatalytic ribozyme from hepatitis delta virus (HDV) and the transcription termination signal from bacteriophage T7 (Pattnaik et al. , 1992) . In vivo or in vitro transcription using T7 RNA polymerase of plasmid pOLTV535 gives rise to antigenomic RNA (or [+]-RNA), whereas transcription of plasmid pOLTV553 gives rise to genomic RNA (or [-]-RNA) (Fig. 5).<br>
To examine whether the minigenome RNA's generated by plasmids pOLTV535 and pOLTV553 could be replicated and expressed by using NDV as helpervirus, we used CER cells which expressed T7 RNA polymerase either constitutively (CER-C9 cells, see Materials and Methods) , or after infection with fowlpox recombinant fpEFLT7pol (Britton et al. , 1995; hereafter called FPV-T7) that expresses T7 RNA polymerase. CER-C9 cells and FPV-T7 infected CER cells were transfected with the minigenome plasmids pOLTV535 or pOLTV553 and after incubation<br><br>
for 3h at 37°C the cells were either infected with NDV for Ih, or left uninfected. Approximately 24h after transfection, a sample was taken from the medium and assayed for SEAP activity. The results showed that SEAP expression;was very high in FPV-T7 infected cells which were transfected with pOLTV535. This is not surprising since transcription by T7 RNA polymerase generates antigenomic [+] -RNA which is capped by fowlpox enzymes and which is efficiently translated by the host cell- In cells transfected with pOLTV553, transcription by T7 RNA polymerase generates genomic [-]-RNA which must be converted into [+] -RNA by helperviru.s in order to be translated into SEAP protein (cf. Fig. 5) . In both cases, no increase in SEAP expression could be observed in NDV infected cells in comparison to non-infected cells. On the contrary, SEAP-expression in NDV infected cells was consistently approximately two times lower than in uninfected cells (results not shown). For pOLTV535-transfected cells this may be explained by the already very high level of SEAP expression by transcripts generated by T7 RNA polymerase. However, in pOLTV553-tranfected cells, where efficient expression of SEAP is dependent on the conversion of genomic [-]-RNA into antigenomic [ + ]-RNA or mRNA by the viral polymerase complex, we would have expected an increase in SEAP expression after NDV infection.<br>
We could think of two reasons why the luinigenomes could not be expressed and replicated by NDV. First, the size of the minigenome RNA's does not conform to the so-called "rule-of-six" (Calain and Roux, 1993; Kolakofsky et al., 1998). According to this rule, paramyxovirus genomes are only replicated efficiently when they are a multiple 6 nt in length. Second, the two extra G residues which are present at the 5' end of the minigenome RNA*s might interfere with correct replication and/or transcription by the viral polymerase complex. To find out whether replication of the genomes was dependent on the rule-of-six, we inserted a series of short self-complementary oligonucleotides which increased 1 nt in size in the unique Clal-site in plasmids<br><br>
pOLTV535 and pOLTV553 (Fig, 2). The resulting plasmids (pOLTV535N0 to -N5 and pOLTV553N0 to -N5) differ in size from 1 to 6 nt and therefore one of them should generate a minigenome RNA which conforms to the rule-of-six.The plasmids were used to transfect CER cells or FPV-T7 infected CER-C9 cells as described above. The results showed that only plasmids pOLTV535N3 and pOLTV553N3 gave rise to an enhanced SEAP activity after NDV infection. The length of the minigenome RNA's generated from these plasmids by T7 RNA polymerase were calculated to be 6n+2. Since two extra G residues are present at the 5' end of the minigenome RNA's, these results suggest that only the size of the RNA sequence which is located between the authentic 3' and 5' ends of the minigenome RNA's is relevant for the rule-of-six. This was verified by constructing minigenome plasmids in which the transcription start of T7 RNA polymerase was changed so that the first nucleotide which was incorporated into RNA was the first nucleotide of the 3' or 5' end of NDV (see Materials and Methods) . Transfection of these plasmids indicated that only minigenome RNA's generated by plasmids pOLTV735N3 and pOLTV753N3 are replicated by helpervirus (results not shown). These findings again indicate that replication of NDV is strictly dependent on the rule-of-six. Furthermore, these findings indicate that the presence of two extra G residues at the 5'-end of the minigenome RNA's does not interfere with correct replication. Similar results have been obtained with minigenome plasmids (or DI plasmids) from other paramyxoviridae (Pattnaik et al., 1992; Harty and Palese, 1995) .<br>
Packaqinq of NDV minigenomes by helpervirus<br>
To determine whether minigenome RNA's could be packaged by NDV helpervirus, the medium of the transfected cells was transferred to fresh monolayers and after Ih of adsorbtion, the monolayers were washed three times with PBS and further incubated in complete medium. After 24h of incubation, the<br><br>
SEAP activity in the medium was measured. The results showed that SEAP activity was present only in cells which had been treated with the medium from cells transfected with minigenome plasmid pOLTV553N3 (Table 4). This finding indicates that minigenome RNA's can be packaged into NDV envelopes and that these particles are able to infect cells. Furthermore, these results show that packaging is dependent on replication which indicates that only RNA molecules which are complexed with the viral NP, P and L proteins are packaged into virus-like particles.<br>
Replication of NDV minigenomes by plasmids expressing the NP. P, and L proteins<br>
To determine whether the minigenome RNA's could also be replicated by plasmids encoding the essential NP, P, and L proteins, we performed co-transfection experiments in cells infected with FPV-T7. Cells were transfected with a combination of plasmids consisting of the minigenome plasmid and plasmids pCIneoNP, -P, and -L(c), respectively. As a negative control, pClneoL{c), which encodes the essential L protein, was replaced by the vector plasmid pCIneo. The results (Table 5) indicated that indeed plasmids encoding NP, P, and L are able to replicate minigenome RNA's. The results furthermore show that, similar to minigenome replication by helpervirus, also replication by the NP, P, and L proteins is dependent on the rule-of-six.<br>
Nucleotide sequence of the complete genome of NDV strain LaSota<br>
Sub-genomic cDNA fragments spanning the entire NDV genome were constructed by means of RT-PCR (Fig. 4) . To keep the number of PCR errors to a minimum, a proofreading enzyme-mix (Expand Long Template; Boehringer Mannheim) was used in combination with a limited number of PCR cycles (15 cycles) . Primer 3UIT which is complementary to the 3 ' end of NDV RNA<br><br>
was used for reverse transcription, and gene-specific primers were used for PCR. To identify possible PCR errors, three<br>
independent RT reactions were performed and used to generate three independent sets of subgenomic cDNA's. The cDNA's, which varied in size from approximately 4 to 7 kb, were cloned in pGEM-T. The nucleotide sequence of two sets of cDNA's was determined by using primers which were either deduced from published NDV sequences, or by primers derived from the NDV sequence that was deduced during this sequencing project (Table 1). Remaining ambiguities were resolved by sequencing the relevant regions of the third set of cDNA clones. The genome of NDV strain LaSota consists of 15186 nt (Fig. -3), which makes it the smallest of all paramyxovirus genomes from which the entire sequence has been established to date (Kolakofsky et al. , 1998) .<br>
Construction of a full-length NDV cDNA clone in transcription plasmid pOLTVS<br>
To construct a full-length cDNA clone of NDV strain LaSota, overlapping cDNA clones spanning the entire NDV genome were joined at shared restriction sites according to the strategy shown in Fig. 4. The entire NUV  cDNA was assembled in the minigenome plasmid pOLTV535 (see bove) which is derived from transcription plasmid pOLTVB. As can be seen in Fig, 4B, the last step in the assembly of the complete NDV cDNA was the cloning of an approximately 8.8 kb Clal (nt 3521-12355) fragment from pGEM-B into p535-DI which contained the NDVsequences flanking the Clal site at either-side (i.e., nt 1-3521 and 12355-15186, respectively). This step proved to be quite difficult since we repeatedly failed in generating the correct clones. Therefore, the Clal fragment of pGEM-B was tagged with the chloramphenicol-resistance (Cm) gene from plasmid pACYC184. The Clal fragment harboring the Cm-gene was isolated and cloned in the Clal-site of p535-DI and transformants were selected for<br><br>
resistance against both Cm. Since transformants grew poorly, the antibiotic selection was reduced to 15 µg/ml Cm and 10 µg/ml Km and the incubation temperature was reduced from 37°C to 3 2°C. Finally, the Cm-gene was removed from this plasmid by digestion with BsiWI followed by recircularization by using T4 DNA ligase. After transformation of E-coli, cells harboring the desired plasmid were identified phenotypically by screening for Km-resistance and Cm-sensitivity. The resulting plasmid which consisted of the full-length NDV cDNA cloned between the Smal and StuI sites of transcription plasmid pOLTVB was designated pNDFL+.<br>
Generation of infectous NDV from full-length cDNA<br>
To generate infectious NDV entirely from cloned cDNA, plasmid pNDFL+ was used in co-transfection experiments with pCIneoNP, -P, and -L(c), as described above for the minigenome plasmids, Transfection of CER and CEF cells was monitored by using minigenome plasmid pOLTV553N3 and by measuring SEAP expression. As a negative control^ pCIneoL(c) was replaced by pCIneo. After co-transfection, the cells were incubated for 3 to 6 days in medium containing 5% allantoic fluid. The addition of allantoic fluid is neccessary because CER or CEF cells lack the appropriate proteases which are required to cleave the F protein of NDV strain LaSota. Cleavage of the F protein is absolutely required for cell-to-cell spread and for the generation of infectious virus. After 3 days of incubation, we performed an immunological staining of the fixed monolayers by using a monoclonal antibody against the F protein. The results showed that cells that were stained with the antibody were only present in, monolayers which had been co-transfected with pNDFL(+), pCIneoNP, -P, and L(c). These results indicated that genome replication and expression was occuring in these cells. No staining cells were observed when pCIneoL(c) was replaced by pCIneo in the co-transfection experiments.<br><br>
To recover infectious virus, the supernatant of transfected CEF monolayers was injected into the allantoic cavity of embryonated eggs. Four days later the allantoic fluid was<br>
harvested, analyzed in a haemagglutination assay, and passaged further in eggs. The results showed that only the supernatant of cells transfected with a combination of pNDFL+ and pCIneoNP, -P, and -L(c) yielded a positive reaction in the haemagglutination assay. Allantoic fluid which showed a positive haemagglutination reaction was subsequently analyzed in a haemagglutination inhibiton assay by using monoclonal antibodies 7B7, 8C11, 5A1, 7D4, and 4D6 (Long et al., 1986) which can be used to differentiate between different NDV strains. The results of this assay indicated that the NDV strain which was revovered from the inoculated eggs showed the same reactivity as the original LaSota strain. The virus which was recovered from the inoculated eggs was designated NDFL to distinguish it from the original LaSota strain.<br>
Generation of genetically modified NDV from full-length cDNA<br>
To show unambiguously that the co-transfection system could be used to recover infectious virus from cloned full-length NDV cDNA, a genetic tag was introduced in plasmid pNDFL( + ) . To this end, the amino acid sequence of the protease cleavage site in the Fo protein was changed from that of the LaSota strain (GGRQGR | L) to the consensus sequence of virulent NDV strains (GRRQRR j F) by means of PCR mutagenesis (for details see Materials and Methods). The resulting plasmid, pNDFL+[Fwt] , was used to generate virus by using the co-transfection system described above. Infectious virus, designated NDFL[Fwt],  was recovered from the allantoic fluid of embryonated eggs which had been inoculated with the medium of co-transfected CEF cells. In an HI test, all MAb's including 7D4, which is specific for the LaSota strain, showed the same reactivity with the newly generated virus as with the original LaSota strain. The nucleotide<br><br>
sequence of the region encoding the protease cleavage site of the F protein was determined by means of RT-PCR. The results showed that the nucleotide sequence contained the exact nucleotide changes which were introduced in the mutagenic primer which was used to modify the original LaSota sequence. This finding shows that the virus was derived from plasmid pNDFL+CFwt] and demonstrates that (genetically modified) NDV can be generated entirely from cloned full-length NDV cDNA.<br>
The protease cleavage site of the Fo protein of NDV is a key determinant for virulence<br>
It is generally assumed that the amino acid sequence of the protease cleavage site of the Fo protein is a key determinant for virulence of different NDV strains. The generation of a genetically modified LaSota strain in which the amino acid sequence of the protease cleavage site was changed from a lentogenic (non-virulent) to that of a velogenic (virulent) NDV strain offered the unique oppertunity to test this assumption. Therefore, we determined the intracerebral pathogenicity index (ICPI) "of the newly generated virus NDFL [Fwt] and compared it with that of strain NDFL and of the original LaSota strain (clone E13-1). The results showed that the ICPI of strain I-TDFL[Fwt] was 1.3 which is far above the value for strains NDFL (ICPI = 0.0) and clone E13-1 (ICPI=0.3). These results show that, as expected, the virulence of NDV is largely determined by the amino acid sequence of the protease cleavage site of the Fo protein.<br>
Introduction of serological marker<br>
The envelope glycoproteins F and HN of NDV are the most immunogenic proteins of the' virus. After infection, both the F and HN protein elicit a strong neutralizing antibody response. The induction of such a neutralizing antibody response is the basis of successful vaccination by non-virulent NDV strains (such as the widely used LaSota strain).<br><br>
However, the antibody response against NDV vaccine strains cannot be distinguished from the antibody response against virulent NDV field strains. Thus, infections with virulent<br>
field virus cannot be traced by serological methods. This situation is undesirable since field virus infections are masked by vaccination and clinical signs which are caused by field strains may be overlooked or are even attributed to the vaccine. Since successful differentiation between vaccination and infection is essential for eradication of NDV, we set out to develop genetically modified NDV strains which can be used for vaccination and which can be serologically distinguished from NDV field strains (so called marker vaccines). In order to develop an NDV marker vaccine, the virus has to be genetically modified such that one or several immunodominant epitopes of one of the (major) antigens are either deleted or modified. Deletion of part(s) of an essential protein may lead to the loss of the biological function of that protein. Therefore, we chose to modify one of the immunodominant envelope proteins of NDV in such a way that the biological function of the protein was retained whereas the antibody repertoire against the modified protein differed from that against the original protein. For reasons specified below, we chose for one embodiment of the invention to modify the HN protein of NDV. Infection of NDV is initiated by fusion of the virion envelope with the plasma membrane of the host cell. For this process, both the F protein and the HN protein are required. It has been shown that the F and HN proteins physically interact and that this interaction is required for membrane fusion (Deng et al. , 1995). Furthermore, it has been shown that the.interaction is type specific, i.e., the F and HN proteins must be derived from the same virus in order to shown fusion activity. The interacting domain of the HN protein of NDV has been localized to the so-called stalk- or stem-region of the protein, comprising the first 92 amino acid residues of the ectodomain of the HN protein (Deng et al., 1995). Hybrid HN<br><br>
proteins consisting of aa 1-141 of NDV and aa 141-572 of human parainfluenza virus type-3 (hPIV3) were shown to retain fusion activity when co-expressed with the NDV F protein. These finding suggests that genetically modified NDV strains which harbour a hybrid HN protein which consists of the stem region of NDV followed by the globular head of the HN protein of a different avian paramyxovirus serotype may be viable. Furthermore, such strains would elicit an anti-HN antibody response which is different from that of NDV. Since the neutralizing antibody response against the F protein is sufficient to allow efficient protection against challenge virus infection, such genetically modified NDV strains meet the two essential requirements of a marker vaccine, i.e., protection against disease and serological differentiation. Hybrid HN genes were constructed which consisted of a fusion of either aa 1-141 of NDV and aa 142-580 of avian paramyxovirus type-2 (APMV2) (designated HNl/2141) or aa 1-143 of NDV and aa 144-580 of APMV2 (designated HNl/2143) . Similary, hybrid HN genes were constructed which consisted either of aa 1-141 of NDV and aa 143-569 of AMPV4 (designated HNl/4141) or aa 1-143 of NDV and aa 145-569 of APMV4 (designated HNl/4143) The hybrid genes were cloned in the eukaryotic expression vector pCIneo and used in co-transfection experiments with a plasmid harboring the NDV F protein. To this end, the F protein was modified such that the amino acid sequence of the proteolytic cleavage site between F2 and Fl was changed from the LaSota sequence to that of the consensus sequence of virulent NDV strains (FwtS see Materials and Methods section). Co-transfection experiments in CER cells and QM5 cells indicated that both • HNl/2141 and HNl/2143 as well as HNl/4141 and HNl/4143 induced cell fusion when co-expressed with the Fwt protein. These results indicated that the complexes between the hybrid HN proteins and the F protein were biologically active. The hybrid HN proteins HNl/2143 and HNl/4143 were used to replace the original HN gene in the full-length cDNA clone pNDFL+, yielding pNDFL-HNl/2'143and pNDFL-HNl/4143. The latter two<br><br>
plasmids were subsequently used for the generation of infectious virus by using the co-transfection system described above. Viable recombinant viruses (designated NDFL-HNl/2143 and NDFL-HNl/4143) could be isolated from the allantoic fluid of embryonated eggs which had been inoculated with the supernatant of transfected monolayers.<br>
The presence of the hybrid HN gene in each of two recombinants was verified by means of RT-PCR. Haemagglutination inhibition tests showed that monoclonal antibodies and polyvalent antisera against NDV were unable to inhibit agglutination of chicken erythrocytes by the recombinant viruses NDFL-HNl/2143 and NDFL-HNl/4143. These results indicate that strains NDFL-HNl/2143 and NDFL-HNl/4143 may be used as vaccines that can be serologically distinguished from classical NDV vaccines.<br>
Expression of a heterologous protein from recombinant NDV<br>
To examine whether foreign genes can be inserted into the NDV genome, we constructed a recombinant virus that carried the SEAP reporter gene. The SEAP gene was derived from plasmid pOLTV53 5 and was modified to include the typical transcriptional stop and start boxes of NDV. A DNA fragment containing the SEAP gene followed by the transcriptional stop and start boxes was inserted into the XmnI site (nt 109) in plasmid pNDFL+[Fwt] . Infectious virus, designated NDFL-AP, was generated by means of the co-transfection system, and the presence of the SEAP gene was verified by means of RT-PCR. Cells infected with strain NDFL-AP expressed very high levels of the SEAP protein. By using the specific activity of the SEAP protein, we calculated that x% of the proteins expressed in cells infected with NDFL-AP consisted of SEAP protein. These results show that heterologous genes can be expressed to very high levels from ■recombinant NDV.<br><br>
Generation of an NDV deletion mutant on a trans-complementing cell line<br>
In order to abrogate expression of the M protein of NDV, a large part of the M gene was deleted by digestion of pNOFL+LFwt] with BsaAI (nt 3087) follwed by partial digestion with Hindlll (nt 4252). After filling in the Hindlll end with Klenow DNA polymerase, the fragment was recicularized by using T4 DNA ligase and used to transform E. coli. The resulting plasmid, designated pNDFL+[Fwt] dM, was used to generate virus by means of the co-transfection system in trans-complementing CER-M cells that expressed the NDV M protein. The supernatant of transfected monolayers was passaged three times on CER-M cells and analyzed for the presence of virus. Virus was obtained as evidenced by the fact that the culture supernatant of the third passage yielded positive results in hemagglutination (HA) and hemagglutination-inhibition (HI) tests. The virus was designated NDFL-dM. When NDFL-dM was used to infect monolayers of CEF cells, the virus was still able to spread by cell-to-cell transmission as seen in an IPMA by using a monoclonal antibody against the F protein. As expected, expression of the M protein could not be demonstrated in an IPMA by using a monoclonal antibodies against the M protein. When the supernatant was used to infect either CEF cells or CER-M cells, we were unable to show the presence of replicating virus in these monolayers by means of IPMA. This finding indicates that infectious virus could not be generated in non-complementing CEF cells. This finding was confirmed by the observation that inoculation of embryonated eggs with, supernatant from j.nfected CEF cells did not result in the generation of progeny virus when tested in HA or HI tests.<br>
The need for better NDV vaccines, and especially the need for NDV marker vaccines, prompted us to develop a reverse genetics system which would allow the genetic modification of NDV. In this document we describe the<br><br>
generation of infectious NDV entirely from cloned full-length cDNA. We show that the virulence of NDV can be dramatically changed by modifying only 3 nucleotides which determine the specificity of the protease cleavage site of the F protein. In this case the protease cleavage site was changed from that of the LaSota strain to that of the consensus cleavage site of virulent NDV strains. By generating this genetically modified NDV strain we deliver the formal proof that the cleavability of the F protein is the key determinant (but not the only determinant) for virulence of NDV. By using the same reverse genetics approach, the cleavage site can be modified, at .will, to any other amino acid sequence. This may lead to the generation of a series of NDV strains which display a spectrum of virulence levels.<br>
in vivo<br>
As already mentioned above, it has been shown that, besides the cleavability of the F and HN proteins, other viral factors may contribute to pathogenicity. Alterations in transcription and translation can modulate growth and cell-to-cell spread of the virus and/or cytopathogenicity. The availability of an infectious cDNA of NDV allows for the systematic modification of sequences which are involved in transcription and replication. This may lead to the design of new NDV vaccines which sport optimal immunogenicity to virtually non-existing virulence.<br>
Safety is one of the most important properties of live vaccines. However, for many live vaccines, including NDV, immunogenicity is often inversely related to virulence. Therefore, further attenuation of live vaccines without losing immunogenicity is one of the most desired alterations for which genetic modification could be used. In this respect it is worthwhile mentioning that it has been shown that elimination of expression of the V protein of Sendai virus resulted in a markedly reduced in  vivo  pathogenicity for mice<br><br>
(Kato et al., 1997). Similar to Sendai virus, NDV also generates a V protein by a mechanism known as RNA editing (Steward et al,, 1993). It is predictable that elimination of expression of the V protein of NDV may also result in an attenuated phenotype in vivo.<br>
Apart from changing the virulence of NDV, we show that it is possible to modify the antigenic make-up of NDV in such a way that strains can be generated which can be serologically discriminated from NDV field strains. These, so called, marker vaccines are an invaluable tool to assess the prevalence of NDV in commercial flocks around the world. Furthermore, the large-scale application of such marker vaccines may ultimately lead to the complete eradication of NDV by a process of intensive screening and stamping out of infected flocks. In this document we show that foreign genes can be inserted into the genome of NDV. These foreign genes can be expressed to very high levels in infected cells. This shows that NDV can be used as a vaccine vector for the expression of antigens from other {poultry') pathogens. Several properties make NDV an ideal vaccine vector for vaccination against respiratory or intestinal diseases. 1) NDV can be easily cultured to very high titres in embryonated eggs. 2) Mass culture of NDV in embx^yonated eggs is relatively cheap. 3) NDV vaccines are relatively stable and can be simply administered by mass application methods such as addition to drinking water or by spraying or aerosol formation. 4) The natural route of infection of NDV is by the respiratory and/or intestinal tract which are also the major natural routes of infection of many other poultry pathogens. 5) NDV can induce local immunity despite the presence of circillating maternal antibody.<br>
Finally, we show that viable NDV deletion mutants can be generated by using trans-complementing cell lines. An NDV deletion mutant was generated which is unable to express the matrix (M) protein which is involved in budding of NDV at the inner cell membrane. We show that a phenotypically<br><br>
complemented NDV strain that is unable to express the M protein is still able to infect cells and spread by means of cell-to-cell transmission. However, the mutant virus is unable to generate infectious progeny on non-complementing cells. This finding shows that phenotypically complemented NDV deletion mutants may be used as safe self-restricted vaccines which are unable to spread into the environment. Such a non-transmissible vaccine combines the most important advantage of live vaccines, i.e., efficacy, with the most important advantage of killed vaccines, i.e., safety.<br><br>
LEGENDS Fig.l.<br>
Transcription vector pOLTVB is a derivative of the transcription vector described by Pattnaik et al. (1992) . See text for details of the construction. The plasmid contains the T7 DNA-dependent RNA polymerase promoter (shown in boldface) followed by unique StuI and Smal restriction sites and the autocatalytic ribozyme from hepatitis delta virus (HDV) . DNA fragments can be cloned between the StuI and Smal sites and can be transcribed either in vitro or in vivo by using T7 RNA polymerase. The 5' end of the resulting transcripts contains two extra G-residues which are not encoded by the insert. Due to the action of the ribozyme, the 3 ' end of the transcripts exactly corresponds to the last nucleotide of the insert.<br>
Fig. 2.<br>
Structure of the minigenome plasmids pOLTV535 (Fig. 2A) and pOLTV553 (Fig. 2B) . The minigenome plasmids are based on transcription plasmid pOLTVS (cf. Fig. 1) and contain the 3'-region (nt 1-119) and 5'-region (nt 14970-15186) of NDV strain LaSota flanking the gene encoding secreted alkaline phosphatase (SEAP) . Transcription of pOLTV535 by T7 RNA polymerase yields antigenomic RNA (or [+]-RNA) whereas transcription of pOLTV553 yields genomic RNA (or [-]-RNA). The start (S) and end (E) boxes, which are viral transcription initiation and termination signals, are indicated. The start codon of the SEAP gene is underlined. The sequences of the insertions (N0-N5) in the Clal-site which generate minigenome plasmids which each differ 1 nt in length (pOLTV53 5N0-N5 and pOLTV553N0-N5, respectively) are also shown.<br><br>
Fig. 3.<br>
Nucleotide sequence of the genome of NDV strain LaSota and deduced amino acid sequence of the NDV genes. The sequence shown corresponds to the antigenomic strand and is shown in the 5' to 3' direction in the form of ssDNA. The sequence shown in this figure is that of the consensus sec[uence which was determined by completely sequencing two independent sets of overlapping subgenomic cDNA's which span the entire NDV genome. Remaining ambiguities (probably as a result of PCR errors) were resolved by sequencing relevant regions of a third independent set of clones. The sequence of the full length cDNA clone pNDFL+ which was assembled from overlapping subgenomic cDNA clones (see Fig. 4) , differs from that of the consensus NDV sequence at the following positions (consensus sequence between parentheses): nt 1755, G (A); nt 3766, A (G); nt 5109, G (A); nt 6999, T (C); nt 7056, G (A); nt 9337, G (A); nt 9486, A (T); nt 10195, T (C); nt 13075, A (G). These differences results in 3 amino acid changes (consencus sequence between parentheses): F protein, R189 (Q) ; HN protein S200 (P) L-protein N369 (I).<br>
Fig. 4.<br>
(A) Overall strategy used for the assembly of the full-length NDV cDNA from subgenomic overlapping cDNA clones. The cDNA was assembled in plasmid pOLTV535 which already contained the 3' and 5' ends of NDV strain LaSota (cf. Fig. 2) . The resulting plasmid, designated pNDFL+, vias used for the generation of infectious NDV.<br>
(B) Detailed cloning procedure for the assembly of the full-length NDV cDNA from subgenomic overlapping cDNA clones. Cm denotes the chloramphenicol-resistance gene which was temporarily introduced as a phenotypical tag (see text for details).<br><br>
(C) Detailed cloning procedure for the generation of genetically modified full-length NDV cDNA. The modification consists of 3 nucleotide changes which were introduced in the F gene and which result in the modification of the amino acid sequence of the proteolytic cleavage site of the F protein (see text for details).<br>
Fig. 5.<br>
(A) pOLTV535-series.<br>
Transcription by means of T7 RNA polymerase yields antigenomic RNA (or [+]-RNA) which can be directly translated into SEAP protein by the cell. After infection of cells by helpervirus (or after co-transfection of plasmids encoding NP, P, and L), the antigenomic RNA is used by the viral polymerase complex for the synthesis of genomic RNA (or [-] -RNA). The genomic RNA is then used by the viral polymerase complex for the synthesis of both mRNA (by using the specific transcription start [S] and end [E] boxes) and antigenomic RNA.<br>
(B) pOLTV553-series.<br>
Transcription by means of T7 RNA polymerase yields genomic RNA (or [-]-RNA) which cannot be translated into SEAP protein. After infection of cells by helpervirus (or after co-transfection of plasmids encoding NP, P, and L), the genomic RNA is used by the viral polymerase complex for the synthesis of both mRNA (by using the specific transcription start [S] and end [E] boxes) and antigenomic RNA.<br><br>
Figure 6.<br>
Alignment of nucleic acid sequences 5' terminal ends of NDV LaSota and other paramyxoviruses are given as sequence comparison of NDV across the four members of the 'Rubulavirus genus, three members of the Paramyxovirus  genus, and three members of the Morbillivirus  genus. The sequences are presented from the L gene end box to the 5' end {3' - 5'<br>
cDNA) .<br>
NDV, new castle disease virus; hPIV2, human parainfluenza virus 2; MuV, mumps virus; SV5 and SV41, simmian virus 5 and 41 respectively; SeV, sendai virus;<br>
bPIV3, and hPIVB bovine and human parainfluenza virus respectively; CDV, canine distemper virus; MeV measles virus RPV, rinderpest virus. Nucleotide (nt} sequences of the entire genomes were obtained as follows (accession no.): NDV (AF077761); hPIV2(X57559); MuV{AB000388); SV5(AF052755); SV41(X64275) ; bPIV3(D84095) ; hPIV3(Z11575): CDV(L13194); MeV(X16565); RPV(Z30697).<br><br>
REFERENCES<br>
Alexander, D.J. (1993) Paramyxovirus infections. in Virus infections of birds. McFerran, J.B. and McNulty, M.S. (eds), pp 321-340, Elsevier Science Publishers B.V., Amsterdam.<br>
Antin, P.B and Ordahl, C,P. (1991) Isolation and characterization of an avian myogenic cell line. Dev.<br>
Biol.   143: 111-121.<br>
Baron, M.D. and Barrett, T, (1997) Rescue of rinderpest virus from cloned cDNA. J". Virol.   71: 1265-1271.<br>
Beach,- J.R. (1944) The neutralization in vitro of avian pneumoencephalitis virus by Newcastle disease immune serum. Science  100: 361-362.<br>
Beard, C.W. and Hanson, R.P, (1984) Newcastle disease. In M.S. Hofstad et al. (eds) Disease of Poultry, 8th Ed., pp. 452-470. Iowa State University Press, Ames. Beaudette, F.R., Bivins, J.A, and Miller, B.R. (1949) Newcastle disease immunization with live virus. Cornell<br>
Vet.   39: 302-334.<br>
Boursnell, M.E.G., Green, P.P., Samson, A.C.R-, Campbell, J.I.A., Deuter, A., Peters, R.W., Millar, N.S., Emmerson, P.T. and Binns, M.M. (1990) A recombinant fowlpox virus expressing the hemagglutinin-neuraminidase gene of Newcastle disease virus (NDV) protects chickens against challenge by NDV. Virology<br>
178: 297-300.<br>
Britton, P., Green, P., Kottier, S., Mawditt, K.L., Penzes, Z., Cavanagh, D. and Skinner, M. (1996) Expression of bacteriophage T7 RNA polymerase in avian and mammalian cells by a recombinant fowlpox virus. J.<br>
Gen.   Virol.   77: 963-967.<br><br>
Calain, P. and Roux, L. (1993) The rule of six, a basic feature for efficient replication of Sendai virus defective interfering RNA, J. Virol.   67: 4822-4830.<br>
Chambers, P., Millar, N.S., Bingham, R.W. and Emmerson, P.T. (1986) Molecular cloning of complementary DNA to Newcastle disease virus and nucleotide sequence analysis of the junction between the genes encoding the haemagglutinin-neuraminidase and the large protein. J.<br>
Gen.   Virol.   67: 475-486<br>
Chang, A,C.Y. and Cohen, S.N. (1978) Construction and characterization of amplifiable multicopy DNA cloning vehicles derived from the P15A cryptic miniplasmid. J.<br>
Bacteriol. 134: 1141-1156.<br>
Cho, B.R. (1982) Cytopathic effects and focus formation by reticuloendotheliosis viruses in a quail fibroblast cell line. Avian Diseases  27: 261<br>
Collins, P.L., Hil, M,G., Camargo, E. , Grosfeld, H. , Chanock, R.M. and Murphy, B.R. (1995) Production of infectious human respiratory syncytial virus from cloned cDNA confirms an essential role for transcription elongation factor from the 5' proximal open reading frame of the M2 mRNA in gene expression and provides a capability for vaccine development. Proc.   Natl,   Acad.<br>
Sci.    USA   92: 11563-11567.<br>
Conzelmann, K.-K. (1996) Genetic manipulation of non-segmented negative-strand RNA viruses. J.   Gen.   Virol.<br>
77: 381-389.<br>
Cowen, B.S. and Braune, M.O. (1988) The propagation of avian viruses in a continuous cell line (QT35) of Japanese quail origin. Avian Diseases  32: 282-297.<br>
Deng, R., Wang, Z., Mirza, A.M. and lorio, R.M. (1995) Localization of a domain on the paramyxovirus attachment protein required for the promotion of cellular fusion by<br><br>
fusion and hemagglutinatinin-neuraminidase antigens. Avan Dis,   40: 110-111.<br>
Heuschele, W.P. and Easterday, B.C. (1970) Local immunity and persistence of virus in the tracheas.of chickens following infection with Newcastle disease virus. II. Immunofluorescent and histopathological studies- J.   Inf.   Dis.   121: 497-504.<br>
Hitchner, S.B. and Johnson, E.P. (1948) A virus of low virulence for immunizing fowls against Newcastle disease (avian pneumoencephalitis). Vet. Med.   43: 525-530.<br>
Hoffman, M.A, and Banerjee, A.K. (1997) An infectious clone of human parainfluenza virus type 3. J". Virol.   71:<br>
4272-4277.<br>
Hofstad, M.S. (1953) Immunization of chickens against<br>
newcastle disease by formalin-inactivated vaccine. Am.<br>
J. Vet. Res.   14: 586-589.<br>
Ichihara, Y., and Kurosawa, Y. (1993) Construction of new T vectors for direct cloning of PCR products. Gene<br>
130: 153-154.<br>
Ishida, N., Taira, H., Omata, T., Mizumoto, K., Hattori, S,, Iwasaki, K. and Kawakita, M. (1986) Sequence of 2617 nucleotides from the 3' end of Newcastle disease virus genome RNA and the predicted amino acid sequence of the viral NP protein. Nucl.   Acids Res.   14: 6551-6564.<br>
Kaleta, E.F, and Baldauf, C. (1988) Newcastle Disease in free-living and pet birds. In D.J. Alexander (ed.), Newcastle Disease, pp. 197-246. Kluwer Academic Publ., Boston.<br>
Kant, A., Koch, G., van Roozelaar, D.J., balk, F. and -ter Huurne, A. (1997) Differentiation of virulent and non-virulent strains of Newcastle disease virus within 24 hours by polymerase chain reaction. Avian  Pathol.   26:<br>
837-849.<br>
Kolakofsky, D., Pelet, T., Garcin, D., Hausmann, S.,<br>
Curran, J. and Roux, L. (1998) Paramyxovirus RNA<br><br>
fusion and hemagglutinatinin-neuraminidase antigens. Avan Dis.   40: 770-777.<br>
Heuschele, W.P. and Easterday, B.C. (1970) Local immunity and persistence of virus in the tracheas.of chickens following infection with Newcastle disease virus. II. Immunofluorescent and histopathological studies. J". Inf.   Dis.   121: 497-504.<br>
Hitchner, S.B. and Johnson, E.P. (1948) A virus of low virulence for immunizing fowls against Newcastle disease (avian pneumoencephalitis). Vet,   Med.   43: 525-530.<br>
Hoffman, M.A. and Banerjee, A.K, (1997) An infectious clone of human parainfluenza virus type 3. J". Virol,   71:<br>
4272-4277.<br>
Hofstad, M.S. (1953) Immunization of chickens against<br>
newcastle disease by formalin-inactivated vaccine. Am.<br>
J,   Vet.   Res.   14: 586-589.<br>
Ichihara, Y., and Kurosawa, Y. (1993) Construction of new T vectors for direct cloning of PCR products. Gene<br>
130: 153-154.<br>
Ishida, N., Taira, H., Omata, T., Mizumoto, K., Hattori, S., Iwasaki, K. and Kawakita, M. (1986) Sequence of 2617 nucleotides from the 3' end of Newcastle disease virus genome RNA and the predicted amino acid sequence of the viral NP protein. Nucl.   Acids  Res,   14: 6551-6564.<br>
Kaleta, E.F. and Baldauf, C. (1988) Newcastle Disease in free-living and pet birds. In D.J. Alexander (ed.), Newcastle Disease, pp. 197-246. Kluwer Academic Publ., Boston.<br>
Kant, A., Koch, G., van Roozelaar, D.J., balk, F. and ter Huurne, A. (1997) Differentiation of virulent and non-virulent strains of Newcastle disease virus within 24 hours by polymerase chain reaction. Avian  Pathol,   26:<br>
837-849.<br>
Kolakofsky, D., Pelet, T., Garcin, D., Hausmann, S.,<br>
Curran, J. and Roux, L. (1998) Paramyxovirus RNA<br><br>
synthesis and the requirement for hexamer genome length: the rule of six revisited. J.   Virol.   72: 891-899.<br>
Kraneveld, F.C. (1926) A poultry disease in the Dutch East Indies. Wed. Indisch Bl.   Diergeneesk.   38: 448-450.<br>
Lamb, R.A. and Lolakofsky,   D. (1996) Paramyxoviridae:<br>
the viruses and their replication, in: Fundamental<br>
Virology (Fields et al,, eds), Chapter 20, p577-604, Lipincott-Raven Publishers, Philadelphia. Lawson, N.D., Stillman, E.A., Whitt, M.A. and Rose, J.K. (1995) Recombinant vesicular stomatitis virus from DNA. Proc. Natl. Acad.   Sci.   USA  92: 4477-4481.<br>
Long, L., Portetelle, D, Ghysdael, J., Gonze, M., Burny, A. and Meulemans, G. (1986) Monoclonal antibodies to haemagglutinin-neuraminidase and fusion glycoproteins of Newcastle disease virus: relationship between glycosylation and reactivity. J. Virol. 57: 1198-1202. Madansky, C.H. and Bratt, M.A, (1978) Noncytopathic mutants of Newcastle disease virus.J. Virol.   26: 724-729<br>
Madansky, C.H. and Bratt, M.A, (1981a) Noncytopathic mutants of Newcastle disease virus are defective in virus-specific RNA synthesis. J. Virol.   37: 317-327.<br>
Madansky, C.H. and Bratt, M.A. (1981b) Relationships among virus spread, cytopathogenicity, and virulence as revealed by the noncytopathic mutants of Newcastle disease virus.J. Virol. 40: 691-702.<br>
Meulemans, G., Gonze, M., earlier, M.C., Petit, P., Burny, A, and Long, L. (1986) Protective effects of HN and F glycoprotein-specific monoclonal antibodies on experimental Newcastle disease. Avian  Pathgl,   15: 761-<br>
768.<br>
Millar, N.S., Chambers, P. and Emmerson, P.T. (1988) Nucleotide sequence of the fusion and haemagglutinin-neuraminidase gene of Newcastle disease virus, strain Ulster: Molecular basis for variations in pathogenicity between strains. J.   Gen,   Virol. 69: 613-620.<br><br>
Morgan, R.W., Gelb Jr., J., Schreurs, C.S., Lutticken, D., Rosenberger, J.K. and Sondermeijer, P. (1992) Protection of chickens from Newcastle and Marek's diseases with a recombinant herpesvirus of turkeys vaccine expressing the Newcastle disease virus fusion protein. Avian Dis.   36: 858-870.<br>
Morgan, R.W., Gelb Jr., J., Pope, C.R. and Sondermeijer, P. (1993) Efficacy in chickens of a herpesvirus of turkeys recombinant vaccine containing the fusion gene of Newcastle disease virus: onset of protection and effect of maternal antibodies.<br>
Moscovici, C. , Moscovici, M.G., Jimenez, H., Lai, M.M., Haymann, M.J. and Vogt, P.K, (1977) Continuous tissue culture cell lines derived from chemically induced tumors of Japanese quail. Cell  11: 95-103.<br>
Pattnaik, A.K., Ball, L.A., LeGrone, A.W. and Wertz, G.W. (1992) Infectious defective interfering particles of VSV from transcripts of a cDNA clone. Cell   69: 1011-1020.<br>
Peeples, M.E. (1988) Newcastle disease virus replication.<br>
In D.J. Alexander (ed.), Newcastle Disease, pp. 45-78.<br>
Kluwer Academic Publ., Boston.<br>
Peeters, B., N. de Wind, M. Hooisma, F. Wagenaar, A. Gielkens,<br>
and R. Moormann. (1992) Pseudorabies virus envelope<br>
glycoproteins gp50 and gll are essential for virus<br>
penetration, but only gll is involved in membrane fusion. J.<br>
Virol. 66: 894-905.<br>
Radecke, F., Spielhofer, P. , Schneider, H., Kaelin, K.,<br>
Huber, M., Dotsch, C., Christiansen, G. and Billeter,<br>
M.A. (1995) Rescue of measles virus from cloned DNA.EMBO<br>
J. 14: 5773-5784.<br>
Rott, R. and Klenk, H.-D. (1988) Molecular basis of infectivity and pathogenicity of Newcastle disease virus. In D.J. Alexander (ed.), Newcastle Disease, pp. 98-112. Kluwer Academic Publ., Boston.<br><br>
Russell, P.H., Griffiths, P.C., Goswami, K.K.A., Alexander, D.J., Cannon, M.J. and Russell, W.C. (1983) The characterization of monoclonal antibodies to Newcastle disease virus. J*. Gen.   Virol.   64: 2069-2072.<br>
Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular cloning, a laboratory manual. Cold Spring Harbor Laboratories, Cold Spring Harbor, NY.<br>
Schneider, H., Spielhofer, P., Kaelin, K., Dotsch, C., Radecke, P., Sutter, G. and Billeter, M.A. (1997) Rescue of measles virus using a replication-deficient vaccinia-TV vector. J.   Virol,   math.   64: 57-64.<br>
Schnell, M.J., Mebatsion, T, and Conzelmann, K.-K. (1994) Infectious rabies viruses from cloned cDNA. EMBO J,   13:<br>
4195-4203.<br>
Schiitze, H., Enzmann, P.-J., Kuchling, R., Mundt. , E., Niemann, H. and Mettenleiter, T.C. (1995) Complete genomic sequence of the fish rhabdovirus infectious haematopoietic necrosis virus, J". Gen.   Virol.   76: 2519-<br>
2527.<br>
Smith, A.L., Tignor, G.H., Mifune, K., and Motohashi, T. (1977) Isolation and assay of rabies serogroup viruses in CER cells. Intervirlogy  8: 92-99.<br>
Spradbrow, P.B. (1988) Geographical distribution. In D.J. Alexander (ed.), Newcastle Disease, pp. 247-255. Kluwer Academic Publ., Boston.<br>
Staiiber, N. , Brechtbiihl, K. , Bruckner, L. and Hofmann, M.A. (1995) Detection of Newcastle disease virus in poultry vaccines using the polymerase chain reaction and direct sequencing of amplified DNA. Vaccine  13: 360-364<br>
Steward, M., Vipond, I.E., Millar, N.S. and Emmerson, P.T. (1993) RNA editing in Newcastle disease virus. J. Gen.    Virol.   74 : 2 53 9-2547.'<br>
Taylor, J., Edbauer, C., Rey-Senelonge, A., Bouquet, J., Norton, E., Goebel, S., Desmettre, P. and Paoletti, E. (1990) Newcastle disease virus fusion protein expressed<br><br>
in a fowlpox virus recombinant confers protection in chickens. J.   Virol,   64: 1441-1450.<br>
Tessier, D.C., Brousseau, R., and Vernet, T. (1986) Ligation of single-stranded oligodeoxyribonucleotides by T4 RNA ligase. Anai • Biochem.   158.: 171-178.<br>
Vieira, J,, and Messing, J. (1991) New pUC-derived cloning vectors with different selectable markers and DNA replication origins. Gene  100: 189-194.<br>
Vindevogel, H. and Duchatel, J.P. (1988) Panzootic Newcastle disease virus in pigeons. In D.J. Alexander (ed.), Newcastle Disease, pp. 184-196. Kluwer Academic Publ., Boston.<br>
Whelan, S.P.J., Ball, L.A., Barr, J.N. and Wertz, G.T.W. (1995) Efficient recovery of infectious vesicular stomatitis virus entirely from cDNA clones. Proc. Natl.<br>
Acad.   Sci.    USA  92: 8388-8392.<br>
Wensvoort, G., Terpstra, C.,  Bonstra, J.,   Bloemraad, M. and Van Zaane, D. (1986) Production of monoclonal antibodies against swine fever virus and their use in laboratory diagnosis. Vet. Microbiol. 12: 101-108.<br>
Yusoff, K. , Millar, N.S., Chambers, P., and Emmerson, P.T. (1987) Nucleotide sequence analysis of the L gene of<br>
Newcastle disease virus: homologies with Sendai and vesicular stomatitis viruses. NucI. Acids Res.   15: 3961-3976.<br><br>
Table 1.<br>
3 • UIT	ACCAAACAGAGAATCCGTGAGTTA	1-24<br>
P368+	GTGATGAGGAACCATGTTGC	368-387<br>
P800+	GTCCGCATCTTCTTGGTTAG	800-819<br>
P1201+	GAGACTTGGAGTAGAGTACG	1201-1220<br>
P1279+	AGCAGCAATGAAGGGCCTGG	1279-1298<br>
P1356+	AAATCGGAGTCCTCACTGGG	1356-1375<br>
P1683+	CTCTATATGACCACACCCTC	1664-1683<br>
PRTl	CAAAGAATTCAGAAAAAAGTACGG	1785-1814<br>
GTAGAAG<br>
P2357+	GGAAACAGTCAGGAAAGACC	2358-2377<br>
P2599+	TAAGTAAAGTTGACTATCAG	2599-2618<br>
P2852+	GGCACTTAATAAACTTTCGC	2852-2871<br>
P3496+	GAATGAAGAAGCCACTGTCG	3496-3515<br>
P3587+	CGGAGATCTTGTTGAGTTGG	3589-3608<br>
P4267+	CATTATCCAAGCAGGTACCC	4270-4299<br>
NDV5 - F	ACGGGCTAGCGATTCTGGATCCCG	4498-4526<br>
GTTGG<br>
P4731+(LS)	AAGCTCCTCCCGAATCTGCC	4733-4752<br>
P4958+	AGCTCTGATACAAGCCAAAC	4960-4979<br>
P5266+(LS)	CTGGTGGGAATATGGATTAC	5267-5286<br>
P5591+(LS)	AGTAACGTTCCCTATGTCCC	5593-5612<br>
P5616+	GTATTTATTCCTGCTTGAGC	5616-5635<br>
P6000	AATACCCTTGATCAGATGAGAGCC	6166-6190<br>
NDV5-HN	GTAGGCTAGCAAGAGAGGCCGCCC	6325-6354<br>
CTCAAT<br>
P6693+(L)	CATTGTTAAAAACTGAGACC	6695-6714<br>
P7110+(L)     ATCGGAAGTCTTGCAGTGTG	7112-7131<br>
P7501+(L)	TGGTGGGAAACGCATCCAGC	7503-7522<br>
P7900+(LS)	AAGACTTAATCCTACGTCTG	7902-7921<br>
P8590+	AACTCGGAAGGGCAGTACAC	8592-8611<br>
L9000	TTTGTCACTCCTGAACTTGTCATT	9008-9031<br>
P9359+	CAATGATATAGCAGAATCCG	9361-9380<br><br>
Table 1 continued<br>
P9371+ I GCAGAATCCGTGACTCATGC I 9371-9411<br>
P9390+ ATAGCTACTGTATTCTCTGG 9392-9411<br>
P9686+	TCACACGATATCATGTTGAG      9686-9705<br>
P9799+	CACACCCTAACGATAATTGG      9801-9820<br>
P10198+       ATAAGAAACGTATCACTGAC      10200-<br>
10219 P10601+       TTGTCGCGTTGCCTGTATGG      10603-<br>
10622 P11006+        GCAGACATACTTTGACTCTG      11008-<br>
11027 P11393+        TCCCTTATTGTCTGGAGTGC      11395-<br>
11414 P11798+        TGATACGATAGAACTCGTAG      11800-<br>
11819<br>
L12000	CATATGTCGCCACATGTGAAGGCT  12008-<br>
12031 P12373+        CAACCAGGACATATGATGAG      12375-<br>
12394 P12796+        TCGACTGTTCTTACCAACTC      12798-<br>
12817 P12978+        CACACCAACTTGCAGATACG      12978-<br>
12997 P13236+        GAGTATCTACTGTCGGATGC      13238-<br>
13257 P13601+        ATACTTGTTCAGAGGAATAG      13603-<br>
13622 P13943+        GACCTGACCTCAGATAAAGC      13946-<br>
13965 P14002+        TATCATTGCTGCATTGTGAC      14004-<br>
14023<br><br>
Table 1 continued<br>
P360	I GGCGATGTAATCAGCCTAGTGCTT I 14756-<br>
14779<br>
P14812+       ACTAAGGACATACTTGAAGC      14812-<br>
14831<br>
P230-	CCGGGACTTCTACTTTTAAG	230-211<br>
P998-   [	TTTGGATATCGCCTGAGAGG	998-979<br>
P1B98-	AAAGGTGGCCATGTTTGTCC	1898-1879<br>
P2617-	TGATAGTCAACTTTACTTAC	2617-2598<br>
P3328-	GCAGAATCAAAGTACAGCCC	3330-3311<br>
P3610-	CTTGCCAACTCAACAAGATC	3612-3593<br>
P3990-	GATTAGCATAGTATCCACTG	3992-3973<br>
NDV3-M	TCTCCCCGGGGCAGCTTATTTCTT  4400-4368<br>
AAAAGGAT<br>
P4593-	GACAGATGCAACTCAGTACC	4625-4606<br>
P4618-(LS)	ATGCAACTCAGTACCAGCGC	4620-4601<br>
P5390-	GTAGAGTTACCTGTATACCC	5411-5392<br>
NDV3-F	ACTACCCGGGAAACCTTCGTTCCT  6238-6212<br>
CAT<br>
P6710-(LS) TCTCAGTTTTTAACAATGCC 6712-6693<br>
P7093- (LS) GTTGATGGAACGCAGAGTAG 7095-7076<br>
P7522-(LS) CTGCTGGATGCGTTTCCCAC 7524-7505<br>
P367	AGGGACCTCAATACTAGCCAGTTC  8692-8666<br>
P9905-	CTCTATCAAGAGGCGATTAG      9907-9888<br>
P10320-        TAAGACAGTACTTTTGCAGG      10322-<br>
10303 P10684-        GATGCAACTGTGTCAACACC     10687-<br>
10706 P11122-        AATTGGGCAGGAGTCAGAAC      11124-<br>
11105 P11510-        TGCCTCCATGATAGCATGCG      11512-<br>
11493<br>
	     	'<br><br>
Table 1 continued<br>
P11903-        ATTGCTTGGAAGATGGAACC     I 11905-<br>
11886 P12717-        TGTCATACACTATTATGGCG      12719-<br>
12700<br>
P13141	CAAAGAGTACCGTGTACAGACAGC  13172-<br>
ATAACC	13143<br>
P13281-        GACATGATAGAGCTCACCTG      13302-<br>
13283 P14101-        ACGGAATGCATGGCAATCAG      14163-<br>
14144 P14522-        GCTCACCAAACTCTCTGCAC      14524-<br>
14505 P14687-        AGGATCTGTCTCGTGCACTG      14709-<br>
14690<br>
P377	TTTCCTTAAGTTTGGTAATACCTA  14888-<br>
GGAC	14861<br>
P359	CACCAAGTCGACAATTGGCCAGAA  15046-<br>
AAGGAG	15017<br>
5NDV	ACCAAACAAAGATTTGGTGAATGA  15186-<br>
CGA	15159<br><br>
Table 2. Sequence of 3' and 5' terminal ends of the genome of NDV strain La Sota<br>
A.	Sequence of 3' terminal end (shown as 5' end of<br>
antigenomic DNA strand)<br>
method I.          clone	sequence<br>
4	ACCAAACAGAGAATC<br>
5	ACCAAACAGAGAATC 13	ACCAAACAGAGAATC 21	ACCAAACAGAGAATC<br>
method II.              clone	sequence<br>
26	ACCAAACAGAGAATC<br>
2	8	ACCAAACAGAGAATC<br>
3	0	ACCAAACAGAGAATC 31	GCCAAACAGAGAATC 32	ACCAAACAGAGAATC 3	ACCAAACAGAGAATC<br>
Consensus	ACCAAACAGAGAATC<br>
B.	Sequence of 5' terminal end (shown as DNA)<br>
pBluescriptll-TSK clones clone	sequence<br>
r3101-13	AC CAAACAAAGATTT<br>
r 3101-14	ACCAAACAAAGATTT<br>
r 3101-15	ACCAAACAAAGATTT<br>
r2601-17	ACCAAACAAAGATTT<br>
r 2 6 01-18	ACCAAACAAAGATTT<br><br>
(table 2 continued)<br>
r 2 6 01-19	ACCAAACAAAGATTT<br>
r 2 6 01-2 0	AACAAGGTGAAGATA<br>
r2601-21	ACCAAACAAAGATTT<br>
pGEM4Z clones      clone	sequence<br>
r3101-16	ACCAAACAAAGATTT<br>
r3101-17	ACCAAACAAAGATTT<br>
r3101-18	ACCAAACAAAGATTT<br>
r3101-19	ACCAAACAAAGATTT<br>
r3101-22	ACCAAACAAAGATTT<br>
Consensus	ACCAAACAAAGATTT<br><br>
Table 3. Minigenome replication by NDV helpervirus<br>
A.	SEAP activity (cps) after transfection of CER-c96 cells<br>
with the pOLTV535 and pOLTVB53-series of plasmids.<br>
plasmid       +NDV	-NDV	ratio<br>
pOLTV535N0	3.5 X	lO4       7.1 x 104	0.49<br>
pOLTV535Nl	5.9	12.1	0.49<br>
POLTV535N2	2.4	6.2	0.3 9<br>
POLTV53 5N3	7.6	5.2	1.4 6<br>
POLTV535N4	1.8	4.1	0.44<br>
POLTV5-3 5N5	1.5	3.0	0.50<br>
pOLTV553N0	5.5	x 103      9.6 x 103	0.57<br>
POLTV553N1	9.6           27.6	0.3 5<br>
POLTV553N2	2.4            3.5	0.68<br>
pOLTV553N3	15.1            9.5	1.59<br>
P0LTV553N4	3.4            7.9	0.43<br>
POLTV553N5	2.9            4.8	0.60<br>
B.	SEAP activity (cps) after transfection of FPV-T7 infected<br>
CER cells with the pOLTV553 -series of plasmids.<br>
Plasmid	+NDV	-NDV          ratio<br>
pOLTV553N0	7.2 x lO4	8.3	x 104        0.86<br>
POLTV553N1	8.4	12.0	0.70<br>
POLTV553N2	8.9	12.6	0.71<br>
POLTV553N3	27.4	8.6	3.19<br>
POLTV553N4	9.7	10.4	0.93<br>
POLTV553N5	8.5	8.1	1.05<br><br>
Table 4.<br>
Transfer of SEAP activity (cps) after treatment of CER cells with the supernatant of FPV-T7 infected CER cells'which had been transfected with the pOLTV553-series of plasmids and which had been superinfected with NDV (see Table 3) .<br>
Plasmid<br>
pOLTV553N0	2.4 x lO3<br>
POLTV553N1	6.2<br>
pOLTV553N2	2.0<br>
pOLTVS53N3	20.6<br>
pOLTV553N4	2.0<br>
POLTV553N5	2.1<br><br>
Table 5.<br>
SEAP activity (cps)	after co-transfection of CER cells with<br>
the pOLTV553-series	of plasmids and plasmids pCIneoNP,<br>
pCIneoP and pCIneoL(c) (or pCIneo as a negative control).<br>
Plasmid	NP, P &amp; L          NP, P &amp; pCIneo<br>
ratio<br>
POLTV553N0	3.1 x 10*           2.7 x 103        11.7<br>
pOLTV553Nl	4.1                5.2              7.9<br>
POLTV553N2	3.1                3.1              10.0<br>
pOLTVS53N3	35.9                3.6            100.8<br>
POLTV553N4	1.9                4.6               4.1<br>
POLTV553N5	1.0                4.1              2.5<br><br><br><br><br>
WE   CLAIMS<br>
1.	Avian-paramyxovirus cDNA at least comprising a nucleic acid sequence corresponding to the 5'-terminal end of the genome of avian-paramyxovirus allowing generating an infectious copy of avian-paramyxovirus.<br>
2.	A cDNA according to claim 1 comprising a full-length cDNA.<br>
3.	A cDNA at least comprising a nucleic acid sequence corresponding to the 5'-terminal end of the genome of avian-paramyxovirus allowing generating an replicating avian-paramyxovirus minigenome.<br>
4.	A cDNA according to claims 1, 2 or 3 at least partly derived from Newcastle Disease Virus.<br>
5.	A cDNA according to claim 4 wherein said Newcastle Disease Virus is a lentogenic virus, preferably derived from a vaccine strain.<br>
6.	A cDNA according to claim 5 wherein said vaccine strain is a LaSota strain ATCC VR-699.<br><br>
7	. A cDNA according to anyone of claims 1 to 6 additionally provided with a modification in a nucleic acid.<br>
8	. A cDNA according to claim 7 wherein said modification comprises a nucleic acid encoding a modified protease cleavage site.<br>
9.	A cDNA according to claim 8 wherein said cleavage site is<br>
a protease cleavage site of the fusion (F) protein.<br>
10.	A cDNA according to claim 7 wherein said modification comprises a nucleic acid encoding a hybrid viral protein.<br>
11.	A cDNA according to claim 10 wherein said protein is a hemaglutinin-neuraminidase (HN) protein.<br>
12.	A cDNA according to claim 7 wherein.said modification comprises a deletion in a nucleic acid encoding a viral protein.<br>
13.	A cDNA according to claim 12 wherein said viral protein is a matrix (M) protein.<br><br>
14. A cDNA according to anyone of claims 1 to 13 additionally<br>
provided with a nucleic acid encoding an heterologous<br>
antigen.<br>
15- A cDNA according to claim 14 wherein said antigen is<br>
derived from a poultry pathogen,<br>
16.	A cDNA according to claim 14 or 15 additionally provided with a nucleic acid encoding an immune-stimulatory protein or part thereof.<br>
17.	An RNA obtained from a cDNA according to anyone of claims 1 to 16.<br>
18.	A method for generating infectious copy avian-paramyxovirus comprising transfecting at least one cell with cDNA according to anyone of claims 1 to 16.<br>
19.	A method according to claim 18 wherein said cell is at least capable of expressing viral nucleocapsid (NP), phospo-(P) or large polymerase (L) protein.<br>
20.	A method according to claim 18 or 19 further comprising allowing cleavage of the fusion protein of said virus.<br>
21.	A method according to anyone of claims 18 to 20 further comprising incubating said cell in growth medium comprising proteolytic activity.<br>
22.	A method according to claim 21 wherein said growth<br>
medium comprises allantoic fluid comprising said proteo.lytic<br>
activity.<br>
23.	A method according to anyone of claims 18 to 22 wherein said cell is derived from a chicken cell.<br>
24.	An infectious copy avian-paramyxovirus obtainable by a method according to any one of claims 18 to 23.<br>
25.	A vaccine comprising a virus according to claim 24.<br>
26.	A live vaccine according to claim 25.<br>
27.	A vaccine according to claim 25 or 26 wherein said infectious copy avian-paramyxovirus is at least partly derived from a Newcastle Disease Virus (NDV).<br>
28- A method for distinguishing unvaccinated animals or animals vaccinated with a NDV vaccine according to claim 27 from animals infected with wild-type NDV or vaccinated with<br><br>
an unmodified mesogenic or lentogenic NDV strain comprising taking a least one sample from said animal and determining in said sample the presence of antibodies directed against an immunodominant epitope or marker expressed by said wild-type or unmodified NDV but not by said vaccine.<br>
29.	A method according to claim 28 wherein said antibodies<br>
are directed against the HN or F protein of NDV.<br>
30.	A method according to claim 28 or 29. wherein said animal<br>
is selected from the group composed of poultry, preferably of<br>
chickens-<br>
31.A diagnostic kit for use in a method according to any one<br>
of claims 28 to 30.<br>
Avian-peramyxovirus cDN A, substantially as hereniabove   described and illustrated with reference to the acconmpaying drawings.<br>
33. A method for generating infectious copy avian-paramyx:ovirus, substantially as hereinabove described and ilhustrated with reference to the accompanying drawing.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1jbGFpbXMgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1jbGFpbXMgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-description complete filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1kZXNjcmlwdGlvbiBjb21wbGV0ZSBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-description complete granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1mb3JtIDE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtODQ4LWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-848-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="211148-a-process-for-the-preparation-of-optionally-substituted-p-hydroxymandelic-compounds-and-derivatives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="211150-a-process-for-the-recovery-of-fluorinated-emulsifier-acids-from-wastewater.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>211149</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2000/848/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Oct-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Dec-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. ID-LELYSTAD, INSTITUUT VOOR DIERHOUDERIJ EN DIERGEZONDHEID B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Edelhertweg 15, 8219 PH Lelystad</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PEETERS, Bernardus, Petrus, Hubertus</td>
											<td>Golfpark 18, NL-8241 AB Lelystad</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DE LEEUW, Olav, Sven</td>
											<td>Johan Westerveldstraat 4, NL-1318 CC Almere</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KOCH, Guus</td>
											<td>Wildbaan 12, NL-8222 AG Lelystad</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GIELKENS, Arnoud, Leonard, Josef</td>
											<td>Boeier 04-76, NL-8242 CL Lelystad</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/45</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL99/00377</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-06-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>98202054.7</td>
									<td>1998-06-19</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/211149-an-avian-paramyxovirus-cdna-and-a-method-for-generating-infectious-copy-avian-paramyxovirus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:17:37 GMT -->
</html>
