<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/211566-novel-subtilase-enzymes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:17:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 211566:NOVEL SUBTILASE ENZYMES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NOVEL SUBTILASE ENZYMES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Subtilases exhibiting a low inhibition by substances present in eggs, such as the trypsin inhibitor type IV-0 are disclosed. These subtilases are useful in cleaning or detergent compositions, such as laundry detergent compositions and dishwash compositions, including automatic dishwash compositions, exhibiting excellent wash performance on egg stains. Isolated nucleic acid sequences encoding the subtilases, nucleic acid constructs, recombinant expression vectors, host cells comprising the nucleic acid construct, and methods for producing and using these subtilases are also disclosed together with cleaning and detergent compositions comprising the subtilases and their use for removal of egg stains.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
SUBTILASE ENZYMES<br>
TECHNICAL FIELD<br>
The present invention relates to novel subtilases having a reduced tendency towards inhibition by substances present in eggs, such as the trypsin inhibitor type IV-0. These subtilases are useful exhibiting excellent or improved wash performance on egg stains when used in e . g. cleaning or detergent compositions, such as laundry detergent compositions and dishwash compositions, including automatic dishwash compostions-<br>
The present invention also relates to isolated nucleic agid sequences encoding the subtilases, nucleic acid constructs, recombinant expression vectors, host cells comprising the nucleic acid construct, and methods for producing and using the subtilases of the invention. Further, the present invention relates "to cleaning and detergent compositions comprising the subtilase enzymes of the invention as well as to use of such enzymes in detergent compositions and for removal of egg stains.<br>
BACKGROUND OF THE INVENTION<br>
In the detergent industry enzymes have for more than 30 years been implemented in washing formulations. Enzymes used in such formulations comprise proteases, lipases, amylases, cellulases, as well as other enzymes, or mixtures thereof. Commercially most important enzymes are proteases.<br>
An increasing number of commercially used proteases are protein engineered variants of naturally occurring wild type proteases, e.g. DURAZYM® (Novozymes A/S) , RELASE® (Novozymes A/S) , MAXAPEM® (Gist-Brocades N.V.), PURAFECT® (C3enencor International, Inc.).<br><br>
However even though a number of useful proteases and protease' variants have been described/ there is still a need for new improved proteases or protease variants for a number of industrial uses.<br>
In particular the problem of removing egg stains from e.g. laundry or hard surfaces has been pronounced due to the fact that substances present in the egg white inhibit many serine proteases- Examples of such substances include trypsin inhibitor type IV-0 (Ovo-inhibitor) and trypsin inhibitor type III-O   (Ovomucoid) .<br>
Therefore, an object of the present invention, is to provide improved subtilase enzymes, which are not, or which are only to a limited extent, inhibited by such substances. A further object of the present invention is to provide improved subtilase enzymes that are suitable for removal of egg stains from,   for example,   laundry and/or hard surfaces.<br>
SUMMARY   OF  THE   INVENTION<br>
Thus, in a first aspect the present invention relates to a subtilase enzyme selected from the group consisting of<br>
(a)	a subtilase having an amino acid sequence which has at least 95%<br>
identity with the amino acid sequence shown as amino acids 1 to<br>
269 of SEQ ID N0:2;  and<br>
(b)	a subtilase which is encoded by a nucleic acid sequence which<br>
hybridizes under low stringency conditions with<br>
(i)        a   corrplementary   strand   of   the   nucleic   acid   sequence<br>
shown as nucleotides 334 to 1140 of SEQ ID N0:1,   or (ii)      a subsequence of  (i) of at least 100 nucleotides.<br><br>
1<br>
In a second aspect the present invention relates to an isolated nucleic acid sequence comprising a nucleic acid sequence that en¬codes  for the  subtilases according to the invention.<br>
In a third aspect the present invention relates to an isolated nucleic acid sequence encoding a subtilase, selected from the group consisting of<br>
(a)	a nucleic acid sequence having at  least  85% identity with the nucleic acid sequence  shown as nucleotides 334  to 1140  of  SEQ  ID N0:1;   and<br>
(b)	a nucleic acid sequence which hybridizes under low strin¬gency conditions with<br>
(i)   a  conplementaxy  strand of  the  nucleic   acid  sequence<br>
shown as nucleotides 334 to 1140 of SEQ ID N0:1,  or (ii) a subsequence of  (i)  of at least 100 nucleotides.<br>
In a fourth aspect the present invention relates to a nucleic	acid con¬<br>
struct conprising the nucleic acid sequence according to the	invention<br>
cperably linked to one or more control sequences capable of	directing<br>
the expression of the subtilase in a suitable host.<br>
In a fifth aspect the present invention relates to a recombinant expres¬sion vector corrprising the nucleic acid construct according to the in¬vention,  a promoter, and transcriptional and translational stop signals.<br>
In a sixth aspect the preset invention relates to a recombinant host cell conprising the nucleic acid construct of the invention,<br>
In a seventh aspect the present invention relates to a method for producing the subtilase according to the invention, the method comprising:<br><br>
(a)	cultivating a recombinant host cell according to the invention<br>
under conditions conducive to the production of the subtilase; and<br>
(b)	recovering the subtilase.<br>
In an eight aspect the present invention relates to a method for producing the subtilase according to the invention, the method comprising:<br>
(a) cultivating a strain from the genus Bacillus, prefera¬bly from the species Bacillus clausii, such as Bacillus elausii DSM 13 585, to produce a supernatant comprising the subtilase; and<br>
(b) recovering the subtilase.<br>
In a ninth aspect the present invention relates to a	cleaning<br>
or detergent composition, preferably a laundry or	dishwash<br>
composition, comprising the subtilase according	to the<br>
invention.<br>
Further aspects of the present invention relate to use of the subtilases according to the invention in a cleaning or detergent composition; use of the subtilases or the compositions according to the invention for removal of egg stains; a method for cleaning or washing, including a method for removal of egg stains from, a hard surface or laundry comprising contacting the hard surface or the laundry with the composition of the invention.<br>
Concerning alignment and numbering, reference is made to Fig. 1 which shows an alignment between subtilisin BPN' (a) (BASBPN) and the novel subtilase of the invention (b) .<br><br>
This  alignment  is  in	this patent  application used as  a<br>
reference for numbering	the residues,<br>
DEPINITONS<br>
Prior to discussing this invention in further detail,  the following terms and conventions will first be defined.<br>
NOMENCLATURE OF AMINO ACIDS<br>
A      =  Ala    ~	Alanine<br>
V	=  Val    =    Valine<br>
L      =  Leu    =    Leucine<br>
I      =  He    =    Isoleucine<br>
P      =  Pro    =    Proline<br>
F      =  Phe    =    Phenylalanine<br>
W      =  Trp    =    Tryptophan<br>
M      =  Met    =    Methionine<br>
G      =  Gly    -	Glycine<br>
S      -       Ser    =	Serine<br>
T      =  Thr    =    Threonine C      =  Cys    =    Cysteine<br>
Y	=  Tyr    =    Tyrosine<br>
N      =  Asn    =    Asparag ine<br>
Q      =  Gin    =    Glutamine<br>
D      =  Asp    =    Aspartic Acid<br>
E      =  Glu    =    Glutamic Acid<br>
K      =  Lys    =    Lysine<br>
R      =  Arg    -	Arginine<br>
H      ~      His    =    Histidine<br>
X      =  Xaa    =    Any amino acid<br>
NOMENCLATURE OF NUCLEIC	ACIDS<br>
A      =  Adenine<br>
G      =  Guanine<br><br>
C      =  Cytosine<br>
T      =  Thymine (only in DNA)<br>
U      =  Uracil  (only in RNA)<br>
NOMENCLATURE AND CONVENTIONS FOR DESIGNATION OF VARIANTS In describing the various subtilase enzyme variants produced or contemplated according to the invention, the following nomen¬clatures  and  conventions  have  been  adapted  for  ease  of reference:<br>
A frame of reference is first defined by aligning the isolated or parent enzyme with subtilisin BPN' (BASBPN).<br>
The alignment can be obtained by the GAP routine of the GCG package version 9,1 to number the subtilases using the following parameters: gap creation penalty = 8 and gap extension penalty = 8 and all other parameters kept at their default values.<br>
Another method is to use known recognized alignments between subtilases, such as the alignment indicated in WO 91/00345. In most cases the differences will not be of any importance.<br>
Such an alignment between subtilisin BPN' (BASBPN) and the novel subtilase of the invention is indicated in Fig. 1.<br>
Thereby a number of deletions and insertions will be defined in relation to BASBPN, In Fig. 1, the novel subtilase according to the invention has 6 deletions in positions 36, 58, 159, 162, 163 and 164 in comparison to BASBPN. These deletions are in Fig. 1 indicated by asterixes (*) .<br>
The various modifications performed -in a parent enzyme is indicated in general using three elements as follows:<br><br>
Original  amino acid position substituted amino acid<br>
The   notation   G195E   thus   means   a   substitution   of   a   glycine   in<br>
position 195  with a glutamic acid.<br>
In the case where the original amino acid residue may be any amino acid 'residue, a short hand notation may at times be used indicating only  the position and substituted amino acid:<br>
Position substituted amino acid<br>
Such    a    notation    is    particular    relevant    in    connection    with<br>
modification(s)   in homologous subtilases   {vide infra).<br>
Similarly when the identity of the substituting amino acid residue (s)   is  immaterial:<br>
Original  amino acid position<br>
VJhen both the original amino acid(s) and substituted amino acid(s) may comprise any amino acid, then only the position is indicated,   e.g. :   170.<br>
When the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), then the  selected amino acids are  indicated inside brackets:<br>
Original   amino  acid position   {substituted  amino  acid1<br>
substituted amino  acidn}<br>
For specific variants the specific three or one letter codes are used, including the codes Xaa and X to indicate any amino acid residue.<br>
SUBSTITUTIONS:<br>
The   substitution   of   Glutamic   acid   for  glycine   in  position   195<br>
is designated as:<br><br>
Glyl95Glu  or  G195E<br>
or the substitution of any amino acid residue acid for glycine in position 195 is designated as:<br>
Glyl95Xaa  .or  G195X or<br>
Glyl95     or  G195<br>
The substitution of serine for any amino acid residue in position 170 would thus be designated<br>
Xaa170Ser  or  X170S. or<br>
170Ser     or  170S<br>
Such a notation is particular relevant in connection with modification(s) in homologous subtilases {vide infra). 170Ser is thus meant to comprise e.g. both a Lysl70Ser modification in BASBPN and Argl70Ser modification in the subtilase according to the invention {cf. Fig. 1),<br>
For a modification where the original amino acid(s) and/or substituted amino acid(s) may comprise more than one, but not all amino acid(s), the substitution of glycine, alanine, serine or threonine for arginine in position 170 would be indicated by<br>
Argl70{Gly,Ala,Ser,Thr}    or R170{G,A,S/T}<br>
to indicate the variants<br>
R170G, R170A, R170S, and R170T.<br><br>
DELETIONS:<br>
A deletion of glycine in position 195 will be indicated by:<br>
Glyl95*     or   G195*<br>
Correspondingly, the deletion of more than one amino acid residue, such as the deletion of glycine and leucine in positions 195 and 196 will be designated<br>
Glyl95*+Leul96*  or  G195*+L196*<br>
INSERTIONS:<br>
The insertion of an additional amino acid residue such as e.g.<br>
*■<br>
a lysine after G195 is indicated by:<br>
Glyl95GlyLys     or  G195GK;<br>
or, when more than one amino acid residue is inserted, such as e.g.   a Lys and Ala after G195 this will be indicated as:<br>
Glyl95GlyLysAla  or  G195GKA<br>
In such cases the inserted amino acid residue(s) are numbered by the addition of lower case letters to the position number of the amino acid residue preceding the inserted amino acid residue (s). In the above example the sequences 194 to 196 would thus be:<br>
194 195 195 BLSAVI      A - G - L<br>
X94 195 195a 195b 196 Variant     A-G-K-  A-  L<br><br>
In cases where an amino acid residue identical to the existing amino acid residue is inserted it is clear that a degeneracy in the nomenclature arises. If for example a glycine is inserted after the glycine in the above example this would be indicated by G195GG. The same actual change could just as well be indicated as A194AG for the change from<br>
194 195 196 BLSAVI      A - G - L<br>
to<br>
194 195  195a 196 Variant    A - G -  G -  L<br>
194 194a 195  196<br>
Such instances will be apparent to the skilled person, and the indication G195GG and corresponding indications for this type of insertions are thus meant to comprise such equivalent degenerate indications,<br>
FILLING A GAP:<br>
Where a deletion in an enzyme exists in the reference comparison with the subtilisin BPN' sequence used for the numbering, an insertion in such a position is indicated as:<br>
*36Asp      or   *36D<br>
for the insertion of an aspartic acid in position 3 6<br>
MULTIPLE MODIFICATIONS:<br>
Variants comprising multiple modifications are separated by<br>
pluses, e.g:<br><br>
Argl70Tyr+Glyl95Glu   or  RIVOY+GISSE<br>
representing modifications in positions 170 and 195 substituting tyrosine and glutamic acid for arginine and glycine, respectively.<br>
Thus,	Tyrl67{Gly,Ala,Ser,Thr}+Argl70{Gly,Ala,Ser,Thr}<br>
designates the following variants:<br>
Tyrl67Gly+Argl70Gly,	Tyrl67Gly+Argl70Ala,<br>
Tyrl67Gly+Argl70Ser,	Tyrl67Gly-HArgl70Thr,<br>
Tyrl67Ala+Argl70Gly,	Tyrl67Ala+Argl70Ala,<br>
Tyrl67Ala+Argl70Ser,	Tyrl67Ala+Argl70Thr,<br>
Tyrl67Ser+Argl70Gly,	Tyrl67Ser-Argl70Ala,<br>
Tyrl67Ser+Argl70Ser,	Tyrl67Ser+Argl70Thr,<br>
Tyrl67Thr+Argl70Gly,	Tyrl67Thr+Argl70Ala,<br>
Tyrl67Thr+Argl70Ser,   and Tyrl67Thr-i-Argl70Thr.<br>
This nomenclature is particular relevant relating to<br>
modifications aimed at substituting, replacing, inserting or<br>
deleting amino acid residues having specific common properties,<br>
such as residues of positive charge (K, R, H) , negative charge<br>
(D, E), or conservative amino acid modification(s) of e.g.<br>
Tyrl67{Gly,Ala,Ser,Thr}+Argl70{Gly,Ala,Ser,Thr},	which<br>
signifies substituting a small amino acid for another small amino acid. See section ^'Detailed description of the invention" for further details-<br>
Proteases<br>
Enzymes cleaving the amide linkages in protein substrates are classified as proteases, or (interchangeably) peptidases (see Walsh, 1979, Enzymatic Reaction Mechanisms, W,H- Freeman and Company, San Francisco, Chapter 3),<br><br>
Numbering of amino acid positions/residues<br>
If nothing else is mentioned the amino acid numbering used herein correspond to that of the subtilase BPN'  (BASBPN) sequence. For further description of the BPN' sequence, see Fig. 1 or Siezen et al.,   Protein Engng.   4 (1991) 719-737.<br>
Serine proteases<br>
A serine protease is an enzyme which catalyzes the hydrolysis of peptide bonds, and in which there is an essential serine residue at the active site (White, Handler and Smith, 1973 "Principles of Biochemistry," Fifth Edition, McGraw-Hill Book Company, NY, pp. 271-272) ,<br>
The bacterial serine proteases have molecular weights in the 20,000 to 45,000 Dalton range. They are inhibited by diisopro-pylfluorophosphate. They hydrolyze simple terminal esters and are similar in activity to eukaryotic chymotrypsin, also a serine protease. A more narrow term, alkaline protease, covering a sub-group, reflects the high pH optimum of some of the serine proteases, from pH 9.0 to 11.0 (for review, see Priest (1977) Bacteriological    Rev.   41 711-753).<br>
Subtilases<br>
A sub-group of the serine proteases tentatively designated subtilases has been proposed by Siezen et al,, Protein Engng. 4 (1991) 719-737 and Siezen et al. Protein Science 6 (1997) 501-523. They are defined by homology analysis of more than 170 amino acid sequences of serine proteases previously referred to as subtilisin-like proteases. A subtilisin was previously often defined as a serine protease produced by Gram-positive bacteria or fungi, and according to Siezen et al. now is a subgroup of the subtilases. A " wide - variety of subtilases have been identified, and the amino acid sequence of a number of subtilases has been determined. For a more detailed description<br><br>
of such subtilases and their amino acid sequences reference is made to Siezen et  al(1997).<br>
One subgroup of the subtilases, I-Sl or "true" subtilisins, comprises the "classical" subtilisins, such as subtilisin 168 (BSS168),  subtilisin BPN',  subtilisin Carlsberg  (ALCALASE*, Novozymes A/S), and subtilisin DY (BSSDY).<br>
A further subgroup of the subtilases, I-82 or high, alkaline subtilisins, is recognized by Siezen et al. {supra). Sub-group I-82 proteases are described as highly alkaline subtilisins and comprises enzymes such as subtilisin PB92 (BAALKP) (MAXACAL®, Gist-Brocades NV) , subtilisin 309 (BLSAVI, BLS3 09) (SAVINASE®, Novozymes A/S), subtilisin 147 (BLS147) (ESPERASE®, Novozymes A/S),   and alkaline elastase YaB (BSEYAB).<br>
Parent subtilase<br>
The term "parent subtilase" describes a subtilase defined according to Siezen et al. (1991 and 1997), For further details see description of "SUBTILASES" immediately above. A parent subtilase may also be a subtilase isolated from a natural source, wherein subsequent modifications have been made while retaining the characteristic of a subtilase. Furthermore, a parent subtilase may also be a subtilase which has been prepared by the DNA shuffling technique, such as described by J.E. Ness et al., Nature Biotechnology, 17, 893-896 (1999). Alternatively the term "parent subtilase" may be termed "wild type subtilase" .<br>
Modification(s) of a subtilase<br>
The term "modification (s)" used herein is defined to include chemical modification of. a subtilase as well as genetic manipulation of the DNA encoding a subtilase. The modification (s) can be replacement (s) of the amino acid side<br><br>
chain(s), substitution(s), deletion(s) and/or insertions in or at the amino acid(s) of interest.<br>
Subtilase variant<br>
In the context of this invention, the term subtilase variant or mutated subtilase means a subtilase that has been produced by an organism which is expressing a mutant gene derived from a parent microorganism which possessed an original or parent gene and which produced a corresponding parent enzyme, the parent gene having been mutated in order to produce the mutant gene from which said mutated subtilase protease is produced when expressed in a suitable host. Analogously, the mutant gene may also be derived from a parent gene produced by DNA shuffling technique.<br>
Homologous subtilase secpaences<br>
In the present context the homology between two amino acid<br>
sequences is described by the parameter "identity" .<br>
In order to determine the degree of identity between two subti-lases the GAP routine of the GCG package version 9,1 can be ap¬plied (infra) using the same settings. The output from the rou¬tine is besides the amino acid alignment the calculation of the "Percent Identity" between the two sequences.<br>
Based on this description it is routine for a person skilled in the art to identify suitable homologous subtilases and corresponding homologous active site loop regions, which can be modified according to the invention.<br>
Isolated nucleic acid sequence<br>
The term "isolated nucleic acid sequence" as used herein refers<br>
to a nucleic . acid sequence,  which has been isolated and<br>
 purified and is thus in a form suitable for use within<br><br>
genetically  engineered  protein  production  systems.  Such' isolated molecules may be those that are separated from their natural environment and include cDNA and genomic clones as well as  nucleic  acid  sequences  derived  from  DNA  shuffling experiments  or  from site-directed mutagenisis  experiments. Isolated nucleic acid sequences of the present invention are free of other genes with which they are ordinarily associated, but may include 5 '  and 3'  untranslated regions such as promoters and terminators.    The identification of associated regions will be evident to one of ordinary skill in the art (see for example, Dynan and Tijan, Nature 316:774-78, 1985). The term "isolated nucleic acid sequence" may alternatively be termed "isolated DNA sequence, 'cloned nucleic acid sequence" or "cloned DNA sequence".<br>
Isolated protein<br>
When applied to a protein, the term "isolated" indicates that<br>
the protein has been removed from its native environment.<br>
In a preferred form, the isolated protein is substantially free of other proteins, particularly other homologous proteins (i.e, "homologous impurities" (see below)).<br>
An isolated protein is more than 10% pure, preferably more than 20% pure, more preferably more than 30% pure, as determined by SDS-PAGE. Further it is preferred to provide the protein in a highly purified form, i.e., more than 40% pure, more than 60% pure, more than 80% pure, more preferably more than 95% pure, and most preferably more than 99% pure, as determined by SDS-PAGE.<br>
The term "isolated protein" may alternatively be termed "purified protein" ,<br><br>
Homologous impurities<br>
The term "homologous impurities" means any impurity (e.g. an¬other polypeptide than the subtilase of the invention) , which originate from' the homologous cell where the subtilase of the invention is originally obtained from.<br>
Obtained from<br>
The term "obtained from" as used herein in connection with a specific microbial source, means that the polynucleotide and/or subtilase produced by the specific source, or by a cell in which a gene from the source has been inserted.<br>
Substrate<br>
The term "substrate" used in connection with a substrate for a protease should be interpreted in its broadest form as comprising a compound containing at least one peptide bond susceptible to hydrolysis by a subtilisin protease.<br>
Product<br>
The term "product" used in connection with a product derived<br>
from a protease enzymatic reaction should, in the context of<br>
the present invention, be interpreted to include the products<br>
of a hydrolysis reaction involving a subtilase protease. A<br>
product  may be  the substrate  in  a subsequent hydrolysis<br>
reaction.<br>
Wash Performance<br>
In the present context the term "wash performance" is used as<br>
an enzyme's ability to remove egg stains present on the object<br>
to the cleaned during e. g, wash or hard surface cleaning, See<br>
also the "Model Detergent Wash Performance Test" in Example 2,<br>
herein.<br><br>
Performance Factor<br>
The term "Performance Factor" is defined with respect to the<br>
below formula<br><br>
wherein P is the Performance Factor, Rsubtilase is the reflectance of the test material after being treated with a subtilase enzyme of the invention as described in the "Model Detergent Wash Performance Test", and Reavinase is the reflectance of the test material after being treated with Savinase® as described in the "Model Detergent Wash Performance Test" . For further details, see the "Model Detergent Wash Performance Test" in Example 2, herein.<br>
Residual Activity<br>
The term "Residual Activity" is defined as described in the<br>
"Ovo-inhibition Assay" herein (see Example 3) .<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Fig. 1 shows an alignment between subtilisin BPN' (a) and the amino acid sequence of the novel subtilase of the invention (b) using the GAP routine mentioned above.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In a first interesting aspect of the present invention, the subtilase enzyme is an isolated subtilase which has at least 95% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO:2 (i.e. the mature subtilase). In an interesting embodiment of the invention the subtilase has at least 96%, preferably at least 97%, more preferably at least 98%, in particular at least 99% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID N0;2 • (hereinafter "homologous subtilases") • In another interesting embodiment of the invention the isolated<br><br>
subtilase comprises or consists of the amino acid sequence shown as amino acids 1 to 269 of SEQ ID N0:2.<br>
Alignments of sequences and calculation of identity scores can be done using the GAP routine (and the same settings) mentioned previously.<br>
By performing such alignments, the following identities (in percentage) between the amino acid sequences of the subtilase having the amino acid sequence of SEQ ID NO;2 and various known subtilases were found:<br><br>
In another interesting embodiment of the invention the isolated sub¬tilase is encoded by a nucleic acid sequence which hybridizes under low stringency conditions, preferably under medium strin¬gency conditions, more preferably under high stringency condi¬tions with (i) a complementary strand of the nucleic acid se¬quence shown as nucleotides 334 to 1140 of SEQ ID N0:1, or (ii) a subsequence of (i) of at least 100 nucleotides (J. Sambrook, E,F, Fritsch, and T. -Maniatus, 1989, Molecular Cloning, A Labo¬ratory Manual,   2d edition, Cold Spring Harbor, New York).<br>
The subsequence of the complementary strand of the nucleic acid sequence shown as nucleotides 334 to 114 0 of SEQ ID N0:1 may be at least 100 nucleotides or preferably at least 200 nucleo¬tides. Moreover, the subsequence should encode a subtilase fragment, which has proteolytic activity. The subtilases may<br><br>
also be allelic variants or fragments of the subtilases that' have proteolytic activity.<br>
The nucleic acid sequence of SEQ ID N0:1 or a subsequence thereof, as well as the amino acid sequence of SEQ ID NO: 2 or a fragment thereof, may be used to design a nucleic acid probe to identify and clone DNA encoding subtilases having proteolytic activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blot¬ting procedures, in order to identify and isolate the corre¬sponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 15, preferably at least 25, and more preferably at least 35 nucleotides in length- Longer probes can also be used. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with 32P, 3H, 35S, biotin, or avidin) . Such probes are encompassed by the present invention-<br>
Thus, a genomic DNA or cDNA library prepared from such other organisms may be screened for DNA, which hybridizes with the probes described above and which encodes a subtilase according to the invention. Genomic or other DNA from such other organ¬isms may be separated by agarose or polyacrylamide gel electro¬phoresis, or other separation techniques known by the skilled person. DNA from the libraries or the separated DNA may be transferred to and immobilized on nitrocellulose or other suit¬able carrier materials. In order to identify a clone or DNA which is homologous with SEQ ID NO:1 or a subsequence thereof, the carrier material is used in a Southern blot. For purposes of the present invention, hybridization indicates that the nu¬cleic acid sequence hybridizes to a labeled nucleic acid probe corresponding to the nucleic acid sequence shown in SEQ ID<br><br>
N0:1, its complementary strand, or a subsequence thereof, undef low to high stringency conditions. Molecules to which the nu¬cleic acid probe hybridizes under these conditions are detected using X-ray film.<br>
For long probes of at least 10 0 nucleotides in length, low to high stringency conditions are defined as prehybridization and<br>
hybridisation at 42°C in 5X SSPE, 0,3% SDS, 200 µg/ml sheared and denatured salmon sperm DNA, and either 25% formamide for low stringency, 35% formamide for medium stringency, or 50% formamide for high stringency, following standard Southern blotting procedures.<br>
For long probes of at least 100 nucleotides in length, the car¬rier material is finally washed three times each for 15 minutes using 2 x SSC, 0.2% SDS preferably at least at 50°C (low strin¬gency) , more preferably at least at 55°C (medium stringency) , even more preferably at least at 65°C (high stringency) .<br>
For short probes, which are about 15 nucleotides to about 70 nucleotides in length, stringency conditions are defined as prehybridization, hybridization, and washing post-hybridization at 5°C to 10°C below the calculated Tm, using the calculation according to Bolton and McCarthy (1962, Proceedings of the Na¬tional Academy of Sciences USA 48:1390) in 0.9 M NaCl, 0.09 M Tris-HCl pH 7.6, 6 mM EDTA, 0,5% NP-40, IX Denhardt's solution, 1 mM sodium pyrophosphate, 1 mM sodium monobasic phosphate, 0.1 mM ATP, and 0.2 mg of yeast RNA per ml, following standard Southern blotting procedures.<br>
For short probes, which are about 15 nucleotides to about 70 nucleotides in length, the carrier material is washed once in 6X sec plus 0,1% SDS for 15 minutes and twice each for 15 min¬utes using 6X SSC at 5°C to lO°C below the calculated Tm.<br><br>
It is well known in the art that a so-called conservative substitution of one amino acid residue to a similar amino acid residue is expected to produce only a minor change in the characteristic of the enzyme.<br>
Table I below list groups of conservative amino acid substitutions.<br><br>
Table I<br>
Conservative amino acid substitutions<br>
Common Property	Amino Acid<br>
Basic (positive charge)        K = lysine<br>
H = histidine Acidic (negative charge)       E = glutamic acid<br>
D = aspartic acid<br>
Polar	Q = glutamine<br>
N = asparagine<br>
Hydrophobic	L -  leucine<br>
I = isoleucine<br>
V = valine<br>
M = methionines<br>
Aromatic	F = phenylalanine<br>
W = tryptophan<br>
y = tyrosine<br>
Small	G = glycine<br>
A = alanine<br>
S = serine<br>
T -  threonine<br>
Therefore, in a further interesting embodiment of the inven¬tion, the novel subtilase having the amino acid sequence of SEQ ID NO:2 is modified by the substitution, deletion and/or inser¬tion of one or more amino acid residues.<br>
Therefore, in a further interesting embodiment of the invention, the subtilase is a variant of the subtilase having the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO: 2, said variant comprises at least one modification, i.e. substitution, deletion and/or insertion, as compared to the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO: 2. Preferably, the number of modifications is at the most 18, such as at the most 17, e.g. at the most 16 or at the most<br><br>
15. In a more preferred embodiment the number of modifications is at the most 14, e.g. at the most 13, at the most 12, at the most 11, at the most 10, at the most 9, at the most 8, at the most 7, at the most 6, or at the most 5, In particular interesting embodiments of the invention, the number of modifications is at the most 4, preferably at the most 3, e.g. at the most 2.<br>
A modified subtilase of the invention has preferably an identity to the amino acid sequence of SEQ ID- N0:2 of 95% or more.<br>
Especially, combinations with other modifications known in the art to provide improved properties to the enzyme are envisaged. The art describes a number of subtilase variants with different improved properties and a number of those are mentioned in the "Background of the invention" section herein (vide supra) .<br>
Thus, modification of the amino acid sequence SEQ ID NO:2 in one or more of the following positions are contemplated as be¬ing of particular relevance (in BASBPN numbering) ;<br>
27, 36, 56, 76, 87, 96, 97, 98, 99, 100, 101, 103, 104, 120, 123, 129, 131, 132, 133, 143, 159, 167, 170, 192, 194, 206, 217, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274.<br>
In particular, the following variants of the subtilase of the invention are considered appropriate for combination (in BASBPN numbering):<br>
K27R,  *36D,   T56P,   N76D,   NB7S,   A97N,   A98AT,   A98AS,   N99ND,	N99NR,<br>
N99A,    N99T,    RIOIG,    P103A,. V104A,    V104I,    V104N,    V104Y,	D120H,<br>
N123S,    P129K,    P131H,   A133P,   A133D,   A133E,    T143K,    *159D,	*159E,<br>
Y167X,    yi67A,    R170X,   R170S,   A194P,    Q206E,    F217R,   N218S,	M222S,<br><br>
M222A,  T224S, A232V, K235L, Q236H, Q245R, N248D, N252K and* T274A.<br>
Of further particular interest are variants of the subtilase of the invention, wherein the modifications comprise any of the modifications V104N+R101G, K27R+V104Y+N123S+T274A, N76D+V104A, or R101G+P103A+V104I+*159D+A232V+Q236H+Q245R+N24 8D+N252K; or other combinations of these modifications (K27R, N76D, RIOIG, P103A, V104I, V104N, V104A, V104Y, N123S, *159D, A232V, Q236H, Q245R, N248D, N252K T274A) , in combination with any one or more of the modifications indicated above or below.<br>
Furthermore, it is contemplated that insertion of at least one additional amino acid residue in the active site (b) loop re¬gion, corresponding to insertion of at least one additional amino acid residue from position 95 to position 103 (BASBPN numbering) , will confer additional wash performance to the sub¬tilase of the invention. In particular, it is preferred to in¬sert at least one additional amino acid residue, such' as one additional amino acid residue, in the following positions: be¬tween positions 98 and 99. (BASBPN numbering) , and between posi¬tions 99 and 100 (BASBPN .numbering) ,<br>
As mentioned above, the subtilases of the invention are only inhibited by trypsin inhibitor type IV-0 to a limited extent and, consequently, they exhibit excellent wash performance on egg stains. Therefore, in order to enable the skilled person -at an early stage of his development work - to select effective and preferred subtilases for this purpose, the present inventors have provided a suitable preliminary test, which can easily be carried out by the skilled person in order to initially assess the performance of the subtilase in question.<br><br>
Thus, the "Ovo-inhibition Assay" disclosed in Example 3 herein' may be employed to initially assess the potential of a selected subtilase. In other words, the "Ovo-inhibit ion Assay" may be employed to assess whether a selected enzyme will be inhibited, and to what extent, by the trypsin inhibitor type IV-0. Using this test, the suitability of a selected subtilase to remove egg stains can be assessed, the rationale being that if a selected subtilase is strongly inhibited by trypsin inhibitor type IV-0, it is normally not necessary to carry out further test experiments.<br>
Therefore, a subtilases which is particular interesting for the purposes described herein, is a subtilases which - when tested in the "Ovo-inhibition Assay" described in Example 3 herein -has a Residual Activity of at least 15%, such as at least 20%, preferably at least 25%, such as at least 3 0%, more preferably at least 35%,<br>
Evidently, it is preferred that the subtilase of the invention fulfils the above criteria on at least the stated lowest level, more preferably at the stated intermediate level and most preferably on the stated highest level.<br>
Alternatively, or in addition to the above-mentioned assay, the suitability of a selected subtilase may be tested in the "Model Detergent Wash Performance Test" disclosed in Example 2 herein. The "Model Detergent Wash Perfomance Test" may be employed to assess the ability of a subtilase, when incorporated in a standard detergent composition, to remove egg stains from a standard textile as compared to a reference system, in this case Savinase® (incorporated in the same model detergent system and tested under identical conditions). Using this test, the suitability of a selected subtilase to remove egg stains can be initially investigated, the rationale being that if a selected<br><br>
subtilase does not show a significant improvement in the test' compared to Savinase®, it is normally not necessary to carry out further test experiments-<br>
Therefore, subtilases which are particular interesting for the purposes described herein, are such subtilases which, when tested in a model detergent composition comprising<br>
6.2%	IAS (Nansa SOS)<br>
2%	Sodium salt of  C16-C18  fatty acid<br>
4%	Non-ionic  surfactant   (Plurafax LF404)<br>
22%	Zeolite  P<br>
10.5%	Na2CO3<br>
4%	Na2Si2O5<br>
2%	Carboxymethylcellulose   (CMC)<br>
6-8%	Acrylate  liquid  CP5  40%<br>
2 0%	Sodium perborate   (empirical  formula NaB02.H202)<br>
0.2%	EDTA<br>
21%	Na2S04<br>
.	Water   (balance)<br>
as described in the "Model Detergent Wash Performance Test" herein, shows an improved wash performance on egg stains as compared to Savinase® tested under identical conditions.<br>
The improvement in the wash performance may be quant i fied by employing the so-called "Performance Factor" defined in Example 2, herein.<br>
In a very interesting embodiment of the invention, the subtilase of the invention, when tested in the "Wash Performance Test" has a Performance Factor of at least 1, such as at least 1,5, e,g. at least 2, preferably at least 2.5, such<br><br>
as at least 3, e.g. at least 3,5, in particular at least 4, J such as at least 4.5, e,g. at least 5.<br>
Evidently, it is preferred that the subtilase of the invention fulfils the above criteria on at least the stated lowest level, more preferably at the stated intermediate level and most preferably on the stated highest level.<br><br><br>
In an interesting embodiment, the subtilase is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces di-astaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis  or Saccharomyces oviformis  subtilase.<br>
In another interesting embodiment, the subtilase is an Asper¬gillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graiainea-rum, Fusarium graminum, Fusarium heterosporum, Fusarium ne-gundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium saroochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium tojrulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola inso-lens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermo-phila, Neurospora crassa, Penicillium purpurog-enum, Trichoderma harzianum, Trichoderma koningii, Trichoderma 1ongibrachiatum, Trichoderma reesei,   or rrichoderma viride subtilase,<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognise the identity of appropriate equiva¬lents.<br>
Strains of these species are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC) , Deutsche Sammlung von Mikroorganis-men und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmel-cultures (CBS), and Agricultural Research Service Patent Cul¬ture Collection, Northern Regional Research Center (NRRL)-<br><br>
A particular suitable Bacillus strain from which the subtilasd of the invention may be isolated is the strain Bacillus clausii HSBIO (alkaline Bacillus HS433) which was deposited according to the Budapest Treaty on the International Recognition of the Deposits of Microorganisms for the Purpose of Patent Procedures on 5 July 2 000 at the Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and designated the accession No. DSM 13 585, The de¬posit was made by Novo Nordisk A/S and has been assigned to No-vozymes A/S.<br>
Furthermore, such subtilases may be identified and obtained from other sources including microorganisms isolated from nature (e,g, soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The nucleic acid sequence may then be derived by similarly screening a genomic or cDNA library of another microorganism. Once a nucleic acid sequence encoding a subtilase has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e.g., Sambrook et al., 1989, supra).<br>
Moreover, the subtilase of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (see WO 95/22625 and J.E. Ness et al, Nature Biotechnology, 17, 893-896 (1999)) .<br>
Many methods for cloning a subtilase of the invention and for introducing insertions into genes (e.g. subtilase genes) are well known in the art, of. the references cited in the "BACK¬GROUND OF THE INVENTION" section.<br><br>
In general standard procedures for cloning of genes and intro-ducing insertions (random and/or site directed) into said genes may be used in order to obtain a subtilase enzyme of the inven¬tion. For further description of suitable techniques reference is made to Examples herein {vide infra) and (Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Har¬bor lab.. Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology". John Wiley and Sons, 1995; Harwood, C. R., and Cutting, S. M. (eds.) "Molecular Bio¬logical Methods for Bacillus". John Wiley and Sons, 1990); and WO 96/34946.<br>
Further, a subtilase enzyme of the invention may be constructed by standard techniques for artificial creation of diversity, such as by DNA shuffling of different subtilase genes (WO 95/22625;  Stemmer  WPC,  Nature  370:389-91  (1994)).  It  is contenplated that DNA shuffling of e.g. the gene encoding<br>
Savinase® with one or more partial subtilase sequences identified in nature will, after subsequent screening for improved wash performance, provide subtilases according to the invention.<br>
NUCLEIC ACID SEQUENCES<br>
The present invention also relates to an isolated nucleic acid<br>
sequence, which encodes a subtilase of the present invention.<br>
In one interesting embodiment, the nucleic acid sequence has at least 85% identity with the nucleic acid sequence shown as nu¬cleotides 3 34 to 1140 of SEQ ID N0:1. Preferably, the nucleic acid sequence has at least 86%, such as at least 87%, e.g. at least 88%, more preferably at least 8 9%, such as at least 90%, e.g. at least 91%, even more preferably at least 92%, such as at least 93%, e.g. at least 94%, most preferably at least 95%, such as at least 96%, e.g. at least 97%, in particular at least<br><br><br><br>
tion from genomic DNA, preparation from cDNA, or a combination thereof. The cloning of the nucleic acid sequences of the pre sent invention from such genomic DNA can be effected, e.g., by using the well knovm polymerase chain reaction (PCR) or anti body screening of expression libraries to detect cloned DN . fragments with shared structural features. See, e.g., Innis es al,, 1990, PCR: A Guide to Methods and Application, Academis Press, New York* Other nucleic acid amplification procedures such .as ligase chain reaction (LCR) , ligated activated trans scription (LAT) and nucleic acid sequence-based amplification (NASBA) may be used.<br>
An isolated nucleic acid sequence can, for example, be obtaine by standard cloning procedures used in genetic engineering to relocate the nucleic acid sequence from its natural location to a different site where it will be reproduced. The cloning proa cedures may involve excision and isolation of a desired nucleis acid fragment comprising the nucleic acid sequence encoding the subtilase, insertion of the fragment into a vector molecule and incorporation of the recombinant vector into a host cell<br>
where multiple copies or clones of the nucleic acid sequence will be replicated. The nucleic acid sequence may be of get nomic, cDNA, RNA, semisynthetic, synthetic origin, or any com binations the:feof.<br>
For purposes of the present invention, the degree of identity<br>
between two nucleic acid sequences is determined as described<br>
above.	<br>
Modification of a nucleic acid sequence encoding a subtilase of the present invention may be necessary for the synthesis of subtilases substantially' similar to the subtilase. The term<br>
"substantially similar" to the subtilase refers to non-naturally occurring forms of the subtilase. These subtilases<br><br><br><br>
Biology 224: 899-904; Wlodaver et al, , 1992, FEBS Letters  3ol 59-64),<br>
NUCLEIC ACID CONSTRUCTS<br>
The present invention also relates to nucleic acid constructs<br>
comprising a nucleic acid sequence of the present invention of<br>
erably linked to one or more control sequences capable of di<br>
recting the expression of the polypeptide in a suitable host<br>
cell.<br>
An isolated nucleic acid sequence encoding a subtilase of the present invention may be manipulated in a variety of ways the provide for expression of the subtilase. Manipulation of the nucleic acid sequence prior to its insertion into a vector may be desirable or necessary depending on the expression vector is The techniques for modifying nucleic acid sequences utilizind recombinant DNA methods are well known in the art.<br>
The control sequences include all components, which are necesB sary or advantageous for the expression of a subtilase of the present invention. Each control sequence may be native or form eign to the nucleic acid sequence encoding the subtilase, Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, promoter, signal peptide sequence, and transcription terminator. At a minimum<br>
the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific<br>
restriction sites facilitating ligation of the control se-<br><br>
quences with the  coding region of  the nucleic acid sequence en¬coding a  subtilase.<br>
The  control   sequence may be  an appropriate promoter sequence,   a nucleic   acid   sequence   which   is   recognized   by   a   host   cell   for<br><br>
expression of the nucleic acid sequence• The promoter sequence' contains transcriptional control sequences which mediate the expression of the subtilase, The promoter may be any nucleic acid sequence which shows transcriptional activity in the host cell of choice including mutanty truncated, and hybrid promot¬ers, and may be obtained from genes encoding extracellular or intracellular subtilases either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, espe¬cially in a bacterial host cell, are the promoters obtained from the B. coli lac operon, Streptomyces coelicolor agarase gene (dagrA) , Bacillus subtilis levansucrase gene (sacB) , Bacil¬lus licheniformis alpha-amylase gene (axnyL) , Bacillus stearothermophilus maltogenic amylase gene (amyM) , Bacillus amyloliquefaciens alpha-amylase gene (amyQ), Bacillus licheni-formis penicillinase gene (penP) , Bacillus subtilis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et al., 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et al., 1983, Proceedings of the National Academy of Sciences USA 80: 21-25), Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook et al.,   1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamori glucoamylase (glaA) , Rhizomucor miehei lipase,  Aspergillus oryzae     alkaline  protease,  Aspergillus<br><br>
oryzae triose phosphate isomerase, Asperg-illus nidulanh acetamidase, and Fusarium oxysporum trypsin-like protease (WO 96/00787), as well as the NA2-tpi promoter (a hybrid of the promoters from the genes for Aspergillus niger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase), and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae galactokinase (GALl), Saccharomyces cerevisiae alco¬hol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP), and Saccharomyces cerevisiae 3-phosphoglycerate kinase. Other useful promoters for yeast host cells are de¬scribed by Romanes et al.,   1992, Yeast 8: 423-488.<br>
The control sequence may also be a suitable transcription ter¬minator sequence, a sequence recognized by a host cell to ter¬minate transcription. The terminator sequence is operably linked to the 3' terminus of the nucleic acid sequence encoding the subtilase- Any terminator which is functional in the host cell of choice may be used in the present invention,<br>
Preferred terminators for filamentous fungal host cells are ob¬tained from the genes for Aspergillus oryzae TAKA amylase, As¬pergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, Aspergillus nigrer alpha-glucosidase, and Fusarium ox¬ysporum  trypsin-like protease-<br>
Preferred terminators for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYCl); and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase- Other useful termi¬nators for yeast host cells are described by Romanes et al., 1992, supra.<br><br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for transla¬tion by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleic acid sequence encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred leaders for filamentous fungal host cells are ob¬tained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase. Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomy-ces cerevisiae 3-phosphoglycerate kinase. Saccharomyces cere¬visiae alpha-factor/ and Saccharomyces cerevisiae alcohol dehy-drogenase/glyceraldehyde-3-phosphate dehydrogenase {ADH2/GAP) .<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 3' terminus of the nucleic acid sequence and which, when transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulaixs anthranilate synthase, Fusarium oxysporum trypsin-like prote¬ase, and Aspersrillus niger  alpha-glucosidase.<br>
Useful polyadenylation sequences for yeast host cells are de¬scribed by Guo and. Sherman, 1995, Molecular Cellular Biology 15: 5983-5990.<br><br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino ter¬minus of a subtilase and directs the encoded subtilase into the cell's secretory pathway. The 5' end of the coding sequence of the nucleic acid sequence may inherently contain a signal pep¬tide coding region naturally linked in translation reading frame with the segment of the coding region which encodes the secreted subtilase. Alternatively, the 5' end of the coding se¬quence may contain a signal peptide coding region which is for¬eign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not natu¬rally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the . natural signal peptide coding region in order to enhance secre-tion of the subtilase. However, any signal peptide coding re¬gion which directs the expressed subtilase into the secretory pathway of a host cell of choice may be used in the present in¬vention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Ba.cillus NCIB 11837 maltogenic amylase, Bacillus stearotheinnophilus alpha-amylase. Bacillus licheniformis sub¬tilisin. Bacillus licheniformis beta-lactamase, Bacillus stearothermophilus neutral proteases {jiprT, nprS, nprM) , and Bacillus subtilis prsA, Further signal peptides are described by Simonen and Palva, 1993, Microbiological Reviews SI i 109-137.<br>
Effective signal peptide coding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus ni-ger  neutral amylase, Aspergrillus niger glucoamylase, Rhizomucor<br><br>
miehei aspartic proteinase, Humicola insolens   cellulase, and' Humicola lanuginosa  lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Sac-charomyces cerevisiae invertase. Other useful signal peptide coding regions are described by Romanos et al.,   1992, supra.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino terminus of a subtilase. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases) . A propolypeptide is generally inactive and can be converted to a ^ mature active polypeptide by catalytic or autocatalytic cleav¬age of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus sub-tilis alkaline protease (aprE) , Bacillus subtilis neutral pro¬tease inprT) , Saccharomyces cerevisiae alpha-factor, Rhizomucor miehei aspartic proteinase, and Afyceiiophthora thermophila lac¬tase (WO 95/33836),<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a subtilase, the propeptide region is po¬sitioned next to the amino terminus of a subtilase and the sig¬nal peptide region is positioned next to the amino terminus of the propeptide region.<br>
It may also be desirable to add regulatory sequences which al¬low the regulation of the expression of the polypeptide rela¬tive to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimu¬lus, including the presence of a regulatory compound. Regula¬tory systems in prokaryotic systems include the lac, tac, and<br><br>
trp operator systems. In yeast, the ADH2 system or GALl system* may be used. In filamentous fungi, the TAKA alpha-amylase pro¬moter, Aspergillus niger glucoamylase promoter, and Aspergillus oryza^e glucoamylase promoter may be used as regulatory se¬quences. Other examples of regulatory sequences are those which allow for gene amplification. In eukaryotic systems, these in¬clude the dihydrofolate reductase gene which is amplified in the presence of methotrexate, and the metallothionein genes which are amplified with heavy metals. In these cases, the nu¬cleic acid sequence encoding the polypeptide would be operably linked with the regulatory sequence.<br><br>
EXPRESSION VECTORS<br>
The present invention also relates to a recombinant expression vector comprising the nucleic acid construct of the invention, a promoter, and transcriptional and translational stop signals.<br>
The recombinant expression vector comprising the nucleic acid construct encoding the enzyme of the invention may be any vector which may conveniently be subjected to recombinant DNA procedures.<br>
The choice of vector will often depend on the host cell into which it is to be introduced. Thus, the vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid.<br>
Alternatively, the vector may be one that on introduction into a host cell is integrated into the host cell genome in part or in its entirety and replicated together with the chromosome (s) into which it has been integrated.<br>
The vector is preferably an expression vector in which the DNA sequence encoding the enzyme of the invention is operably linked to additional segments required for transcription of the DNA. In general, the expression vector is derived from plasmid or viral DNA, or may contain elements of both. The term, "operably linked" indicates that the segments are arranged so that they function in concert for their intended purposes, e.g. transcription initiates in a promoter and proceeds through the DNA sequence coding for the enzyme.<br>
The promoter may be any DNA sequence which shows transcrip¬tional activity in the host cell of choice and may be derived<br><br>
from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for use in bacterial host cells include the promoter of the Bacillus stearothermophilus maltogenic amylase gene, the Bacillus licheniformis alpha-amylase gene, the Bacillus amyloliquefaciens alpha-amylase gene, the Bacillus subtilis alkaline protease gen, or the Bacillus pumilus xylosidase gene, or the phage Lambda PR or PL promoters or the E. coli lac, trp or tac promoters.<br>
The DNA sequence encoding the enzyme of the invention may also, if necessary, be operably connected to a suitable terminator.<br>
The recombinant vector of the invention may further comprise a DNA sequence enabling the vector to replicate in the host cell in question.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, or a gene encoding resistance to e.g. antibiotics like kanamycin, chloramphenicol, erythromycin, tetracycline, spectinomycine, or the like, or resistance to heavy metals or herbicides*<br>
To direct an enzyme of the present invention into the secretory pathway of the host cells, a secretory signal sequence (also known as a leader sequence, prepro sequence or pre sequence) may be provided in the recombinant vector. The secretory signal sequence is j oined to the DNA sequence encoding the enzyme in the correct reading frame. Secretory signal sequences are commonly positioned 5* to the DNA sequence encoding the ensyme. The secretory signal - sequence may be that normally associated with the enzyme or may be from a gene encoding another secreted protein.<br><br>
The procedures used to ligate the DNA sequences coding for the present enzyme, the promoter and optionally the terminator and/or secretory signal sequence, respectively, or to assemble these sequences by suitable PCR amplification schemes, and to insert them into suitable vectors containing the information necessary for replication or integration, are well known to persons skilled in the art (cf., for instance, Sambrook et al., op.cit.).<br>
HOST CELL<br>
The present invention also relates to a recombinant host cell<br>
comprising the nucleic acid construct of the invention.<br>
The DNA sequence encoding the present enzyme introduced into the host cell may be either homologous or heterologous to the host in question. If homologous to the host cell, i.e, produced by the host cell in nature, it will typically be operably connected to another promoter sequence ,or, if applicable, another secretory signal sequence and/or terminator sequence than in its natural environment. The term "homologous" is intended to include a DNA sequence encoding an enzyme native to the host organism in question. The teirm "heterologous" is intended to include a DNA sequence not expressed by the host cell in nature. Thus, the DNA sequence may be from another organism, or it may be a synthetj-C sequence.<br>
The host cell into which the DNA construct or the recombinant vector of the invention is introduced may be any cell which is capable of producing the present enzyme and includes bacteria, yeast, fungi and higher eukaryotic cells including plants.<br>
Examples of bacterial host cells which, on cultivation, are capable of producing the enzyme of the invention are gram-<br><br>
positive bacteria such as strains of Bacillus, such as strains' of B. subtiliS/ B. licheniformis, B. cla.u3ii, B. brevis, B. stearothermophilus, B. alkalophilus B. amyloliquefaciens, S. coagulans, B. circulans, B. lautus, B, megatherium or B, thuringiensis, in particular B. clausii, or strains of Streptomyces, such as S. lividans or S. murinus or gram-negative bacteria such as Echerichia  coli.<br>
The  transformation  of  the  bacteria may be  effected	by<br>
protoplast transformation, electroporation, conjugation, or	by<br>
using competent cells in a manner known per se (of. Sambrook	et<br>
al., supra).<br>
When expressing the enzyme in bacteria such as E, coli, the enzyme may be retained in the cytoplasm, typically as insoluble granules (known as inclusion bodies) , or may be directed to the periplasmic space by a bacterial secretion sequence. In the former case, the cells are lysed and the granules are recovered and denatured after which the enzyme is refolded by diluting the denaturing agent. In the latter case, the enzyme may be recovered from the periplasmic space by disrupting the cells, e.g. by sonication or osmotic shock, to release the contents of the periplasmic space and recovering the enzyme •<br>
When expressing the enzyme in gram-positive bacteria such as Bacillus or Streptomyces strains, the enzyme may be retained in the cytoplasm, or may be directed to the extracellular medium by a bacterial secretion sequence. In the latter case, the enzyme may be recovered from the medium as described below,<br>
METHOD OF PRODUCING A SUBTIIASES OF THE INVENTION<br>
The present invention further relates to a method for producing<br>
a subtilase of the invention, the method comprising:<br><br>
a)	cultivating a recombinant host cell of the invention under condi; tions conducive to the production of the subtilase; and<br>
b)	recovering the subtilase.<br>
When an expression vector comprising a DNA sequence encoding the enzyme is transformed into a heterologous host cell it is possible to enable heterologous recombinant production of the enzyme of the invention.<br>
Thereby it is possible to make a highly purified subtilase composition, characterized in being free from homologous impurities.<br>
In this context homologous impurities means any impurities (e.g. other polypeptides than the enzyme of the invention) which originate from the homologous cell where the enzyme of the invention is originally obtained from.<br>
The medium used to culture the transformed host cells may be any conventional medium suitable for growing the host cells in question. The expressed subtilase may conveniently be secreted into the culture medium and may be recovered therefrom by well-known procedures including separating the cells from the medium by centrifugation or filtration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulfate, followed by chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
The present invention also relates to methods for producing a subtilase of the present invention, the method comprising<br>
(a) cultivating a strain from the genus Bacillus to  produce a supernatant comprising the subtilase; and<br><br>
(b) recovering the subtilase.<br>
Preferably, the strain is of the species Bacillus clausii, and more preferably Bacillus clausii  DSM 13585.<br>
In the production methods of the present invention, the cells are cultivated in a nutrient medium suitable for production of the subtilase using methods known in the art. For example, the cell may be cultivated by shake flask cultivation, small-scale or large-scale fermentation (including continuous, batch, fed-batch, or solid state fermentations) in laboratory or indus¬trial fermentors performed in a suitable medium and under con¬ditions allowing the subtilase to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium com¬prising carbon and nitrogen sources and inorganic salts, using procedures known in the art. Suitable media are available from commercial suppliers or may be prepared according to published compositions {e.g., in catalogues of the American Type Culture Collection), If the subtilase is secreted into the nutrient me¬dium, the subtilase can be recovered directly from the medium. If the polypeptide is not secreted, it can be recovered from cell lysates.<br>
The sxibtilase may be detected using methods known in the art that are specific for the polypeptides. These detection methods may include use of specific antibodies, formation of an enzyme product, or disappearance of an enzyme substrate. For example, an enzyme assay may be used to determine the activity of the polypeptide as described herein.<br>
The resulting subtilase may be recovered by methods known in the art. For example, the subtilase may be recovered from the nutrient medium by conventional procedures including, but not<br><br>
limited to,  centrifugation,  filtration,  extraction,  spray- ' drying, evaporation, or precipitation.<br>
The subtilases of the present invention may be purified by a variety of procedures known in the art including, but not lim¬ited to, chromatography (e,gr., ion exchange, affinity, hydro¬phobic, chromatofcausing, and size exclusion), electrophoretic procedures (e.g., preparative isoelectric focusing), differen¬tial solubility (e.gr., ammonium sulfate precipitation) , SDS-PAGE, or extraction (see, e,g., Protein Purification, J. -C. Janson and Lars Ryden, editors, VCH Publishers, New York, 1989) ,<br>
USE OF A SUBTIIASE OF THE INVENTION<br>
A subtilase enzyme of the invention may be used for a number of industrial applications, in particular within the detergent industry. Thus, the present invention also relates to a cleaning or detergent composition, preferably a laundry or dishwash composition, in particular an automatic dishwash composition, comprising the subtilase enzyme of the invention.<br>
In general, cleaning and detergent compositions are well described in the art and reference is made to WO 96/3494 6; WO 97/072 02; WO 95/30011 for further description of suitable cleaning and detergent compositions,<br>
Furthermore the examples herein demonstrate the improvements in wash performance on egg stains for the subtilases of the invention.<br>
Detergent Compositions<br>
The subtilase of the.invention may be added to and thus become<br>
a component of a cleaning or detergnent composition.<br><br>
The detergent composition of the invention may for exartple be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations/ or be formulated for hand or machine dishwashing operations.<br>
In a specific aspect, the invention provides a detergent addi¬tive comprising the subtilase enzyme of the invention. The de¬tergent additive as well as the detergent composition may com¬prise one or more other enzymes such as another protease, a li¬pase , a cutinase, an amylase, a carbohydrase, a cellulase, a pectinase, a mannanase, an arabinase, a galactanase, a xy-lanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme (s) should be compatible with the selected detergent, (i.e. pH-optimum, com¬patibility with other enzymatic and non-enzymatic ingredients, etc.),  and the enzyme(s)  should be present  in effective' amount s.<br>
Proteases: Suitable proteases include those of animal, vegetable or microbial origin. Microbial origin is preferred. Chemically modified or protein engineered mutants are included. The protease may be a serine protease or a metallo protease, preferably an alkaline microbial protease or a trypsin-like protease, Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carlsberg, subtilisin 309, subtilisin 147 and subtilisin 168 (described in WO 89/06279) . Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine origin) and the Fusarium protease described in WO 89/06270 and WO 94/25583.<br><br>
Examples of useful proteases are the variants described in WO 92/19729, WO 98/20115, WO 98/20116, and WO 98/34946, especially the variants with substitutions in one or more of the following positions: 27, 36, 57, 76, 87, 97, 101, 104, 120, 123, 167, 170, 194, 206, 218, 222, 224, 235 and 274.<br>
Preferred  commercially  available  protease  enzymes  include<br>
Alcalase™, Savinase™, Primase™, Duralase™, EsperaseTM, and<br>
KannaseTM  (Novozymes A/S) ,  Maxatase™,  Maxacal"™,  Maxapem™,<br>
Properase™,  Purafect™,  Purafect  OxP™,  FN2™,  and  FN3™ (Genencor International Inc).<br>
Lipases: Suitable lipases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful lipases include lipases from Humicola (synonym Thermomyces) , e.g. from H. lanuginosa (T. lanuginosus) as described in EP 258 068 and EP 305 216 or from H. insolens as described in WO 96/13580, a Pseudomonas lipase, e.g, from P. alcaligenes or P. pseudoalcaligenes (EP 218 272), P. cepacia (Ep 331 376), P. stutzeri (GB 1,372,034), P. fluorescenSf Pseudomonas sp. strain SD 705 (WO 95/06720 and WO 96/27002), P. wisconsinensis (WO 96/12012), a Bacillus lipase, e.g. from B, subtilis (Dartois et al. (1993), Biochemica et Biophysica Acto, 1131, 253-360), B. stearothermophilus (JP 64/744992) or B.   pumilus   (WO 91/16422).<br>
Other examples are lipase variants such as those described in WO 92/05249, WO 94/01541, EP 407 225, EP 260 105, WO 95/35381, WO  96/00292,  WO  95/3 0744,  WO  94/25578,  WO  95/14783,  WO 95/22615, WO 97/04079 and WO 97/07202.<br><br>
I<br>
Preferred commercially available lipase enzymes include Lipolase™ and Lipolase UltraTM (Novozymes A/S) .<br>
Amylases: Suitable amylases (a and/or p) include those of bac¬terial or fungal origin• Chemically modified or protein engineered mutants are included. Amylases include, for example,<br>
a-amylases obtained from Bacillus, e.g. a special strain of B. licheniformis,   described in more detail in GB 1,296,839.<br>
Examples of useful amylases are the variants described in WO 94/02597, WO 94/18314, WO 96/23873, and WO 97/43424, especially the variants with substitutions in one or more of the following positions: 15, 23, 105, 106, 124, 128, 133, 154, 156, 181, 188, 190, 197, 202, 208, 209, 243, 264, 304, 305, 391, 408, and 444.<br>
Commercially available amylases are DuramylTM, TermamylTM, Fun-gamylTM and BAN™  (Novozymes A/S),  Rapidase™ and PurastarTM (from Genencor International Inc.),<br>
Cellulases: Suitable cellulases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included- Suitable cellulases include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonium, e,g the fungal cellulases produced from Humicola insolens, Myceliophthora thermophila and Fusarium oxysporum disclosed in US 4,435,3 07, US 5,648,263, US 5,691,178, US 5,776,757 and WO 89/09259.<br>
Especially suitable cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellu¬lases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/293 97, WO 98/08940, Other examples are cellulase variants such as those described in WO 94/07998, EP 0<br><br>
531 315, US 5,457,046, US 5,686,593, US 5,763,254, WO 95/24471,* WO 98/12307 and PCT/DK98/00299,<br>
Commercially  available  cellulases  include  Celluzyme'^^,  and<br>
Carezyme™  (Novozymes  A/S) ,  Clazinase™,  and  Puradax HA™<br>
(Genencor International Inc.) , and KAC-^500 (B)™ (Kao Corporation).<br>
Peroxidases/Oxidases: Suitable peroxidases/oxidases include those of plant, bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g. from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257,<br>
Commercially available peroxidases include GuardzyraeTM (Novozymes A/S) .<br>
The detergent enzyme(s) may be included in a detergent composi¬tion by adding separate additives containing one or more en¬zymes, or by adding a combined additive comprising all of these enzymes. A detergent additive of the invention/ i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Preferred detergent addi¬tive formulations are granulates, in particular non-dusting granulates, liquids, in particular stabilized liquids, or slur¬ries.<br>
Non-dusting granulates may be produced, e.g., as disclosed in US 4,106,991 and 4,661,452 and may optionally be coated by methods known in the art. Examples of waxy coating materials are poly(ethylene oxide)  products  (polyethyleneglycol,  PEG) with mean molar weights of 1000 to 20000; ethoxylated nonyl-<br><br>
phenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 car¬bon atoms and in which there are 15 to 8 0 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglyc¬erides of fatty acids. Examples of film-forming coating materi¬als suitable for application by fluid bed techniques are given in GB 1483591. Liquid enzyme preparations may, for instance, be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to estab-lished methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any con-venient form, e.g,, a bar, a tablet, a powder, a granule, a paste or a liquid, A liquid detergent may be aqueous, typically containing up to 70% water and 0-30% organic solvent, or non¬aqueous ,<br>
The detergent composition typically comprises one or more sur¬factants, which may be non-ionic including semi-polar and/or anionic and/or cationic and/or zwitterionic. The surfactants are typically present at a level of from 0,1% to 60% by weight. When included therein the detergent will usually contain from about 1% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsulfonate, alkyl sulfate (fatty alcohol sulfate), alcohol ethoxysulfate, secondary al-kanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuccinic acid or soap.<br>
When included therein the detergent will usually contain from about 0.2% to about 40% of a non-ionic surfactant such as alco¬hol ethoxylate, nonylphenol ethoxylate, alkylpolyglycoside, al-kyldimethylamineoxide, ethoxylated fatty acid monoethanolamide, fatty acid monoethanolamide, polyhydroxy alkyl  fatty acid am-<br><br>
ide,   or  N-acyl  N-alkyl   derivatives   of   glucosamine "glucamides") .<br>
The detergent may contain 0-55% of a detergent builder or com-plexing agent such as zeolite, diphosphate, triphosphate, phos-phonate, carbonate, citrate, nitrilotriacetic acid, ethyl-enediaminetetraacetic acid, diethylenetriaminepentaacetic acid, alkyl" or alkenylsuccinic acid, soluble silicates or layered silicates (e.g SKS-6 from Hoechst),<br>
The detergent may comprise one or more polymers. Examples are carboxymethylcellulose, poly(vinylpyrrolidone), poly (ethylene glycoD, poly(vinyl alcohol) , poly(vinylpyridine-N-oxide), poly(vinylimidazole), polycarboxylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br>
The detergent may contain a bleaching system which may comprise a H2O2 source such as perborate or percarbonate whigh may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxybenzenesulfoliate. Al¬ternatively, the bleaching system may comprise peroxyacids of e.g. the amide, imide, or sulfone type.<br>
The enzyme (s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g., a polyol such as propylene glycol or glycerol, a suga:t or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g. , an aromatic borate ester, or a phenyl boronic acid de¬rivative such as 4-formylphenyl boronic acid, and the com¬position may be formulated as described in e.g. WO 92/19709 and WO 92/19708.<br><br>
The detergent may also contain other conventional detergent in¬gredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil redeposition agents, dyes, bacte¬ricides , optical brighteners, hydrotropes, tarnish inhibitors, or perfumes,<br>
It is at present contemplated that in the detergent composi¬tions any enzyme, in particular the enzyme of the invention^ may be added in an amount corresponding to 0.01-100 mg of en¬zyme protein per liter of wash liquor, preferably 0.05-5 mg of enzyme protein per liter of wash liquor, in particular 0.1-1 mg of enzyme protein per liter of wash liquor.<br>
The enzyme of the invention may additionally be incorporated in The detergent formulations disclosed in WO 97/07202 which is hereby incorporated as reference-<br>
The invention is described in further detail in the following examples which, are not in any way intended to limit the scope of the invention as claimed.<br>
In the detergent compositions, the abbreviated component iden¬tifications have the following meanings:<br>
LiAS;       Sodium linear C12 alkyl benzene sulphonate<br>
TAS;       Sodium tallow alkyl sulphate<br>
XYAS:      Sodium C1x - C1y alkyl sulfate<br>
SS:        Secondary, soap surfactant of formula 2-butyl oc-<br>
tanoic acid<br><br>
25EY:      A C12-C15 predominantly linear primary alcohol con-'<br>
densed with an average of Y moles of ethylene oxide<br>
45EY;      A C14-C15 predominantly linear primary alcohol con¬densed with an average of Y moles of ethylene oxide<br>
XYEZS:     Cix-CiY sodium alkyl sulfate condensed with an aver¬age of Z moles of ethylene oxide per mole<br>
Nonionic:  C13-C15 mixed ethoxylated/propoxylated fatty alcohol<br>
with an average degree of ethoxylation of 3.8 and an average degree of propoxylation of 4,5 sold un¬der the tradename Plurafax LF404 by BASF GmbH<br>
CFAA:      C12-C14 alkyl N-methyl glucamide<br>
TFAA:      C16-C18 alkyl N-methyl glucamide<br>
Silicate:  Amorphous Sodium Silicate (SiO2:Na2O ratio =2.0)<br>
NaSKS-6:   Crystalline layered silicate of formula 5-Na2Si2O5<br>
Carbonate: Anhydrous sodium carbonate<br>
Phosphate:  Sodium tripolyphosphate<br>
MA/AA:     Copolymer of 1:4 maleic/acrylic acid/ average mo¬lecular weight about 80,000<br>
Polyacrylate: Polyacrylate homopolymer with an average mo¬lecular weight of 8,000 sold under the trade¬name PA30 by BASF Gmbh<br><br>
Zeolite A:  Hydrated   Sodium  Aluminosilicate   of   formula<br>
Nai2 (Al02Si02)i2-2 7H20 having a primary particle size in the range from 1 to 10 micrometers<br>
Citrate;   Tri-sodium citrate dihydrate<br>
Citric:    Citric Acid<br>
Perborate: Anhydrous sodium perborate monohydrate bleach, em¬pirical formula NaBO2.H202<br>
PB4:       Anhydrous sodium perborate tetrahydrate<br>
Percarbonate:    Anhydrous sodium percarbonate bleach of em¬pirical formula 2Na2C03, 3H2O2<br>
TAED:      Tetraacetyl ethylene diamine<br>
CMC:       Sodium carboxymethyl cellulose<br>
DETPMP:    Diethylene  triamine  penta  (methylene phosphonic<br>
acid) , marketed by Monsanto under the Tradename De-quest 2060<br>
PVP:       Polyvinylpyrrolidone polymer<br>
EDDS:      Ethylenediamine-N, N-disuccinic acid, [S,S] isomer<br>
in the form of the sodium salt<br>
Suds       25% paraffin wax Mpt 50°C, 17% hydrophobic silica, Suppressor: 58% paraffin oil<br>
Granular Suds    12% Silicone/silica, 18% stearyl alcohol, 70%<br><br>
suppressor: starch in granular form<br>
Sulphate;  Anhydrous sodium sulphate<br>
HMWPEO:    High molecular weight polyethylene oxide<br>
TAE 25:    Tallow alcohol ethoxylate (25)<br><br><br>
1<br><br>
Detergent Example III<br>
Granular fabric cleaning compositions in accordance with the invention which are especially useful in the laundering of col¬oured fabrics were prepared as follows:<br><br><br>
Enzyme	0.10       0.05<br>
Silicate	2.5<br>
Sulphate	5.2        3-0<br>
PVP	0,5<br>
Poly (4-vinylpyridine)-N-	-	0.2<br>
Oxide/copolymer of vinyl-<br>
imidazole and vinyl-<br>
pyrrolidone<br>
Perborate	1.0        <br>
Phenol sulfonate	0.2<br>
Water/Minors	Up to 100%<br>
Detergent Example IV<br>
Granular fabric cleaning compositions in accordance with the invention which provide "Softening through the wash^* capability may be prepared as follows:<br>
45AS	-	10,0<br>
LAS	1.6<br>
68AS	1.3<br>
45E7	4.0<br>
25E3	-	5.0<br>
Coco-alkyl-dimethyl hydroxy-  1.4        1.0<br>
ethyl ammonium chloride<br>
Citrate	5.0        3.0<br>
Na-SKS-6	-	11.0<br>
Zeolite A	15.0        15.0<br>
MA/AA	4.0	4.0<br>
DETPMP	0.4	0.4<br>
perborate	15.0<br>
Percarbonate	-	15.0<br>
TAED	5-0	5 - 0 ■<br><br>
Smectite clay	10,0       10.0<br>
HMWPEO	-           0.1<br>
Enzyme	0.10       0.05<br>
Silicate	3.0        5.0<br>
Carbonate	10*0       10.0<br>
Granular suds suppressor	1.0        4-0<br>
CMC	0.2        0.1<br>
Water/Minors	Up to 100%<br>
Detergent Example V<br>
Heavy duty liquid fabric cleaning compositions in accordance<br>
with the invention may be prepared as follows:<br>
LAS acid form	-                       25.0<br>
Citric acid	5.0        2.0<br>
25AS acid form	8.0<br>
25AE2S acid form	3.0<br>
25AE7	8.0<br>
CFAA	5<br>
DETPMP	1,0         1.0<br>
Fatty acid	8<br>
Oleic acid	-          1.0<br>
Ethanol	4.0        6.0<br>
Propanediol	2.0        6.0<br>
Enzyme	0.10       0.05<br>
Coco-alkyl  dimethyl	-                          3.0<br>
hydroxy ethyl  ammonium<br>
chloride<br>
Smectite clay	-          5-0<br>
PVP	2.0<br>
Water / Minors	Up to 100%<br>
Powder automatic dishwash composition X<br><br><br><br><br><br><br>
Powder and liquid dishwash composition with cleaning surfactant system VI<br><br><br><br>
Non-aqueous liquid automatic dishwshing composition VII<br><br><br><br><br><br><br><br>
XII: Automatic dishwashing compositions as described in I, II, III, IV, VI and X, wherein perborate is replaced by per-carbonate.<br>
XIII: Automatic dishwashing compositions as described in I-VI, which additionally contain a manganese catalyst. The manganese catalyst may, e,g., be one of the compounds described in "Effi¬cient manganese catalysts for low-temperature bleaching", Na~ ture,    (1994), 369, 637-639.<br>
MATERIALS AND METHODS<br>
PROTEOLYTIC ACTIVITY<br>
In the context of this invention proteolytic activity is<br>
expressed in Kilo NOVO Protease Units (KNPU) . The activity is<br>
determined relatively to an enzyme standard (SAVINASE ) , and the determination is based on the digestion of a dimethyl casein (DMC) solution by the proteolytic enzyme at standard conditions, i .e, 50°C, pH 8.3, 9 min, reaction time, 3 min. measuring time. A folder AF 220/1 is available upon request to Novozymes A/S, Denmark, which folder is hereby included by reference.<br>
A GU is a Glycine Unit, defined as the proteolytic enzyme activity which, under standard conditions/ during a 15 minutes'<br>
incubation at 40°C, with N-acetyl casein as svibstrate, produces an amount of NH2-group equivalent to 1 mmole of glycine.<br><br>
Enzyme   activity can also be measured using the PNA assay/ according to reaction with the soluble substrate succinyl-alanine - alanine-proline-phenyl-alanine-para-nitro-phenol, which is described in the Journal of American Oil Chemists Society, Rothgeb, T,M., Goodlander, B,D., Garrison, P.H., and Smith, L.A,, (1988) .<br><br>
EXAMPLE la - Isolation and Purification of Bacillus  strain<br>
The Bacillus clausii strain (DSM Accession No.: DSM 13585) was isolated from a soil sample collected in Sedona, AZ, USA, The soil was spread upon the surface of an agar plate with TY-agar (see below) and 0.1 M sodium sesquicarbonate and colonies aris¬ing after incubation for 2 days at 37°C were re-isolated and the pure culture was transferred to 500 ml shake flasks with 100 ml PS-1 medium (see below) and 0-1 M sodium sesquicarbon-ate.<br>
The shake flasks were incubated for 4 days at 30°C on a rotat¬ing table at 300 rpm. Subsequently, the subtilase was harvested and purified (see Example lb below) .<br>
TY-agar:<br>
Trypticase	20 g<br>
Yeast extract	5 g<br>
FeCl2/4 H2O	0.6 ml of a 1% solution (w/v)<br>
MnCl2,4 H2O	0.1 ml of a 1% solution (w/v)<br>
MgS04,4 H2O	1.5 ml of a 1% solution (w/v)<br>
Destined water	1000 ml<br>
Merck agar	20 g<br>
The pH was adjusted to 7.3 with 4 N KOH before autoclaving for 2 0 minutes.<br>
PS-1 medium:<br>
Saccharose	100 g<br>
Soy bean flour	40 g<br>
Na2HP04,12 H2O	10 g<br>
Pluronic PE 6100	0 .1 ml<br>
Tap water	1000 ml<br><br>
The ingredients were mixed and homogenised. 100 ml medium was then filled into 500 ml baffled shake flasks and autoclaved,<br>
» EXAMPLE lb - Isolation and Purification of the Stibtilase<br>
Approximately 1.6 liters of fermentation broth (from Example<br>
la) were centrifuged at 5000 rpm for 35 minutes in 1 liter<br>
beakers. The supernatants were adjusted to pH 7 using 10%<br>
acetic acid and filtered through a Seitz Supra SlOO filter<br>
plate.<br>
At room temperature, the filtrate was applied to a 100 ml Bacitracin agarose affinity column (Upfront Chromatography A/S) equilibrated with 0.01 M dimethylglutaric acid, 0^1 M boric acid and 0.002 M calcium chloride adjusted to pH 7 with sodium hydroxide (Buffer A), After washing the column with Buffer A to remove unbound protein, the subtilase was eluted from the Bacitracin column using Buffer A supplemented with 25% 2-propanol and 1 M sodium chloride.<br>
The fractions with protease activity from the Bacitracin purification step were combined and applied to a 750 ml Sephadex G25 column (Amersham Pharmacia Biotech) equilibrated with Buffer A.<br>
Fractions with proteolytic activity from the Sephadex G25 column were combined and the pH was adjusted to pH 6 with 10% acetic acid and applied to a 150 ml CM Sepharose CL 6B cation exchange column (Amersham Pharmacia Biotech) equilibrated with a buffer containing 0.01 M dimethylglutaric acid, 0.1 M boric acid, and 0.002 M calcium chloride adjusted to pH 6 with sodium hydroxide.<br>
Tbe subtilase was eluted using a linear gradient of 0-0.2 M sodium chloride in 2 liters of the same buffer. Finally, the<br><br>
protease containing fractions from the CM Sepharose column were' combined and filtered through a 0.2fx filter-<br>
EXAMPLE Ic - Determination of Sequence<br>
The DNA coding for the s\ibtilase of the invention has been multiplied from DNA isolated from a Bacillus clausii HSBIO strain (DSM Accession No,: DSM 13585) by PCR using oligonucleotides with homology to the aprH309 gene of B, clausii NCIB 10309 described in WO 89/06279, (25 cycles of PCR were carried out with denaturation temperature for 60 seconds<br>
at 92°C, annealing for 3 0 seconds at SB°C, and elongation for 90 seconds at 72°C .<br>
The N-terminal primer (5'-AAT AGA GCT CAC GAG CTT GGA CAA GTT GG-3') anneal 20 to 40 basepair upstream for the ATG start codon and the C-terminal primer (5'-TTT GGA TCC ATA CAC AAA AAA ACG CTG TGC CC-3') anneal 30 to 50 bp downstream for the TAA stop codon<br>
The DNA and protein sequences were deduced from the PCR segment (SEQ ID NOS. 1 and 2) -<br>
Example Id - Expression in B.Clausii.<br>
The expression of the subtilase of the invention was optimised by inserting the pro- and mature part of the gene in frame with the signal sequence of the aprH3 09 gene of B. clausii (formerly B. lentus) (WO 89/06279) in the chromosome on a derivative of Bacillus clausii NCIB 10309, The integration was made in such a way that the pro sequence and the mature part of the original apr gene were deleted from the B. clausii chromosome and re-placed by the pro sequence and the mature part of the gene en¬coding the enzyme of the invention.<br><br>
By flanking the new apr gene with DNA fragments of the aprH3 0^ gene the insertion into the chromosome was made. The two DNA segments from aprH309 consist of 500 bp directly upstream for the signal cutting site (in frame with the pro sequence of the enzyme of the invention) and 500 bp directly below the TAA stop codon. The subtilase segment flanked by the two 500 bp aprH309 sequences was inserted into a cat derivative of the temperature sensitive pE194 plasmid. This recombinant plasmid was trans¬formed by protoplast transformation to B. clausii (Akamatzu, T et all. Agric, Biol. Chem, 1984 vol 48: p. 651-655) (The pH in the HCP 1-5 regeneration plates was adjusted to pH 9 by addi¬tion of sodium carbonate buffer to 0.05M). After regeneration of the B.   clausii   transformants the plasmid was inserted into<br>
the chromosome by homologous recombination at 48°C selecting<br>
for resistance to 10 µg/ml chloramphenicol. By lowering the<br>
temperature to 30°C without chloramphinicol selection the plas¬mid will occasionally be lost from the chromosome leaving the subtilase inset behind. Among colonies with protease phenotype and without resistance towards 10 microgram/ml chloramphenicol cells was isolated and analysed and selected by PCR with spe¬cific primers for the subtilase gene. By PCR with primer (5'-AAT AGA GCT CAC GAG CTT GGA CAA GTT GG-3') and primer (5'-TTT GGA TCC ATA CAC AAA AAA ACG CTG TGC CC-3') a DNA fragment cov¬ering the total coding region was made and sequenced. The de¬duced amino acid sequence was identical to the mature subtilase of the invention,<br>
EXAMPLE 2 - The "Model Detergent Wash Performance Test" In order to asses the wash performance of subtilases in a standard detergent composition, standard washing experiments may be performed using the below experimental conditions:<br>
-    Detergent:	Model detergent<br><br>
Detergent dosage       4.0 g/1<br>
PH	10.1<br>
Wash time	20 min<br>
Temperature:	3 0°C<br>
Water hardness:	15°dH<br>
Enzyme concentration:	10 nm (in the detergent solution)<br>
Test system:	10 ml beakers with a stirring rod<br>
Textile/volume:	5 textile pieces (0 2.5 cm)/50 ml<br>
detergent solution<br>
Test material:	WFKION (egg stains)<br>
The composition of the model detergent is as follows:<br>
6.2%	LAS (Nansa SOS)<br>
2%	Sodium salt of Cig-Cie  fatty acid<br>
4%	Non-ionic  surfactant   (Plurafax LF404)<br>
22%	Zeolite P<br>
10.5%	NaaCOa<br>
4%	NasSiaOs<br>
2%	Carboxymethylcellulose (CMC)<br>
6.8%	Acrylate liquid CP5 40%<br>
20%	Sodium perborate (empirical formula NaB02.H202)<br>
0.2%	EDTA<br>
21%	NaaSOi<br>
Water (balance)<br>
pH of the detergent solution is adjusted to 10.1 by addition of HCl or NaOH. Water hardness is adjusted to 15°dH by addition of CaCla and MGCI2 (Ca:Mg = 4:1) to the test system. After washing the textile pieces are flushed in tap water and air-dried.<br>
Measurement of the reflectance (Rsubtiiase) on the test material is performed at 460 nm using a Macbeth ColorEye 7000 photometer<br><br>
(Macbeth, Division of Kollmorgen Instruments Corporation/ Germany) . The measurements are performed accordance with the manufacturer's protocol.<br>
In order to determine a blank value, a similar wash experiment is performed without addition of enzyme. The subsequent measurement of the reflectance (Rbiank) is performed as described right above.<br>
A reference experiment is then performed as described above, wherein the wash performance of Savinase® is tested. The subsequent measurement of the reflectance" (Rsavinase) is performed as described right above.<br>
The wash performance is evaluated by means of the Performance Factor (P) which is defined in accordance with the below formula:<br><br>
EXAMPLE 3 - The ^^Ovo-inhibition Assay"<br>
The below inhibition assay is based on the principle that the subtilase to be tested will catalyse the hydrolysis of a pep-tide-pNA bond, thereby releasing the yellow pNA, which may con¬veniently be followed at 405 nm. The amount of released pNA af¬ter a given period of time is a direct measure of the subtilase activity. By carrying out such hydrolysis experiments with and without inhibitor, respectively, it is possible to obtain a quantitative measure for the degree to which a certain subti--lase is inhibited.<br><br><br>
Cone, of trypsin inhibitor type IV-0:	0.0015 mg/ml<br>
Initial substrate concentration:	0.81 mM<br>
Reaction time:	11 min<br>
Assay temperature:	25°C<br>
Assay pH:	8.6<br>
Absorbance measured at:	405 nm<br>
Assay solutions:<br>
Substrate solution (2 mM) : 500 mg Suc-Ala-Ala-Pro-Phe-pNA is dissolved in 4 ml DMSO (200 mM). This solution is diluted 100 times with the buffer solution described below. The concentra¬tion of substrate in the resulting substrate solution is 2 mM.<br>
Inhibitor solution (0.005 mg/ml): 5 mg trypsin inhibitor type IV-0 (Sigma T-1886) is dissolved in 10 ml water. This solution. is dissolved 100 times with the buffer solution described be¬low. The concentration of inhibitor in the resulting inhibitor solution is 0.005 mg/ml.<br>
Enzyme solution (0.001 mg/ml): 1 mg enzyme is dissolved in 10 ml water. This solution is dissolved 100 times with the buffer solution described below. The concentration of enzyme in the resulting enzyme solution is 0.001 mg/ml-<br>
Buffer solution (pH 8.6) : 15.7 mg Tris is dissolved in an ap¬propriate amount of water and 0.75 ml 30% (w/v) BRIJ (BRIJ 35 polyoxyethylenelaurylether; 30% (w/v), Sigma Cat. No. 430AG-6) is added. The pH is adjusted to 8.6 with 4 M NaOH and the solu¬tion is diluted to 1 liter with water.<br>
Assay with inhibitor<br>
1 volume unit (e.g. SO µl} inhibitor solution is mixed with 1<br>
volume unit (e.g. 80 µl) enzyme solution in an appropriate re¬action vessel (e.g. a spectrophotometer cell or a micro titer<br><br>
plate) and equilibrated at 25°C for 15 min, 1,375 volume units (e.g. 110 |il) substrate solution is added to the reaction ves¬sel after which the absorbance at 405 nm is followed for 11 min (e.g. by measuring every 10th or 3 0th second) . The slope of the absorbance curve is calculated using linear regression analy¬sis. The slope of the absorbance curve is denoted ainhibitor<br>
Assay without inhibitor<br>
1 volume unit (e.g. 80 p,l) buffer solution is mixed with 1 vol¬ume unit (e.g. 80 /il) enzyme solution in an appropriate reac¬tion vessel (e.g. a spectrophotometer cell or a micro titer<br>
plate) and equilibrated at 25°C for 15 min. 1.375 volume units<br>
(e.g. 110 µl) substrate solution is added to the reaction ves¬sel after which the absorbance at 405 nm is followed for 11 min<br>
(e.g. by measuring every 10th or 30th second). The slope of the absorbance curve is calculated using linear regression analy¬sis. The slope of the absorbance curve is denoted a.<br>
Blank<br>
1 volume unit (e.g. 80 µl) inhibitor solution is mixed with 1<br>
volume unit (e.g. 80 µl) buffer solution in an appropriate re¬action vessel (e.g. a spectrophotometer cell or a micro titer plate) and equilibrated at 25°C for 15 min. 1.375 volume units<br>
(e.g. 110 µl) substrate solution is added to the reaction ves¬sel after which the absorbance at 405 nm is followed for 15 min. These measurements are not used in the calculations, but merely serve as a control that no enzyme has been added to the buffer and/or substrate solution.<br>
Calculation of Residual Activity (RA)<br>
The residual enzyme activity (RA) is calculated according to<br>
the below formula:<br><br>
RA   =     (ainhibitor/a)     X   100%<br>
Using the above test, the following results were obtained;<br><br>
As it appears, the subtilase according to the invention is inhibited to a much lesser extent than the structurally similar subtilase Savinase®,<br>
DEPOSIT OF BIOLOGICAL MATERIAL<br>
The following biological material has been deposited under the<br>
terms of the Budapest Treaty with the Deutsche Sammlung von<br>
Mikroorganismen und Zellkulturen GmbH, Mascheroder Weg 1 B, D-<br>
38124 Braunschweig, Germany, and given the following accession<br>
number:<br><br><br><br><br>
CLAIMS<br>
1.  A subtilase enzyme selected from the group consisting of<br>
(b) a subtilase having an amino acid sequence which has at least 95% identity with the amino acid sequence shown as amino acids 1 to 269 of SEQ ID N0:2;   and<br>
(b)       a subtilase which is encoded by a nucleic acid sequence which hybridizes under low stringency conditions with<br>
(i)        a   complementary   strand   of   the   nucleic   acid   sequence<br>
shown as nucleotides 334 to 1140 of SEQ ID NO; 1,  or (ii)      a subsequence of  (i)  of at least 100 nucleotides.<br>
2.  A subtilase according to claim 1 having an amino acid sequence which has at least 96%, preferably at least 97%, more preferably at least 98%, most preferably at least 99% identity with the amine acid sequence shown as amino acids 1 to 269 of SEQ   ID N0:2.<br>
3. A subtilas according to claim 2, which comprises the amino acid sequence  shown as  amino acids  1  to 269  of  SEQ  ID N0:2,<br>
4. A subtilase according to claim 3, which consists of the amino acid sequence shown as amino acids 1 to 269 of SEQ ID NO: 2 ,<br>
5. A subtilase according to any of the preceding claims, wherein the subtilase is a variant of a subtilase having the amino acid sequence shown as amino acids 1 to 2 69 of SEQ ID NO:2 comprising a substitution, deletion, and/or insertion of one or more amino acid residues.<br><br>
6.	A subtilase according to claim 5, comprising at least one modification in one of the positions 27, 36, 56, 76, 87, 96, 97, 98, 99, 100, 101, 103, 104, 120, 123, 129, 131, 132, 133, 143, 1S9, 167, 170, 192, 194, 206, 217, 218, 222, 224, 232, 235, 236, 245, 248, 252 and 274 (BASBPN numbering),<br>
7.	A subtilase according to claim 6, wherein the modification is selected from the group consisting of K27R, *36D, T56P, N76D, N87S, A97N, A98AT, A98AS, N99ND, N99NR, N99A, N99T, RIOIG, P103A, V104A, V104I, V104N, V104Y, D120H, N123S, P129K, P131H, A133P, A133D, A133E, T143K, *159D, *159E, Y167X, yi67A, R170X, R170S, A194P, Q206E, F217R, N218S, _M222S, M222A, T224S, A232V, K23 5L, Q236H, Q245R, N24 8D, N252K and T274A (BASBPN numbering),<br>
8.	A subtilase according to claim 7, wherein the modifications comprise any of the modifications V104N4-R101G, IC27R+V104Y+N123S+T274A,	N76D+V104A,	or R101G+P103A+V104I+*159D+A232V+Q236H+Q245R+N248D+N252K; or other combinations of these modifications (K27R, N76D, R101G, P103A, V10.4I, V104N, V104A, V104y, N123S, *159D, A232V, Q236H, Q245R, N248D, N252K T274A) , in combination with any one or more of the modifications indicated in claims 6 or 7.<br>
9. A subtilase according to any of the preceding claims, which is encoded by a nucleic acid sequence which hybridizes under medium stringency conditions, preferably under high stringency conditions, with<br>
(i)   a corrplementary strand of the nucleic acid sequence<br>
shown as nucleotides 334 to 1140 of SEQ ID NO;1, or (ii)  a subsequence of (i) of at least 100 nucleotides.<br><br>
10.	A subtilase according to any of the preceding claims, where the subtilase - when tested in the "Ovo-inhibition Assay" - has a Residual Activity of at least 15%.<br>
11.	A subtilase according to claim 10, where the subtilase has a Residual Activity of at least 20%, preferably at least 25%, such as at least 30%, more preferably at least 35%-<br>
12- An isolated nucleic acid sequence comprising a nucleic acid sequence which encodes for the subtilase defined in any of claims 1-11.<br>
13.	An isolated nucleic acid sequence encoding a subtilase,<br>
selected from the group consisting of<br>
(a)	a nucleic acid sequence having at least 85% identity with the nucleic acid sequence shown as nucleotides 334 to 1140 of SEQ ID N0:1; and<br>
(b)	a nucleic acid sequence which hybridizes under low strin¬gency conditions with<br>
(i) a complementary strand of the nucleic acid sequence<br>
shown as nucleotides 334 to 1140 of SEQ ID N0:1, or (ii) a subsequence of (i) of at least 100 nucleotides.<br>
14,	A nucleic acid sequence according to claim 13, having a nucleic<br>
acid sequence which has at least 86%, such as at least 87%,<br>
e.g. at least 88%, preferably at least 89%, such as at least<br>
90%, e.g. at least 91%, more preferally at least 92%, such as<br>
at least 93%, e.g. at least 94%, most preferably at least 95%,<br>
such as at least 96%,. e,g. at least 97%, in particular at least<br>
98%, preferably at least 99% identity with the nucleic acid se¬<br>
quence shown as nucleotides 334 to 1140 of SEQ ID N0;1.<br><br>
15.	A nucleic acid construct comprising the nucleic acid sequence of any of claims 12-14 operably linked to one or more cc5ntrol sequences capable of directing the egression of the subtilase in a suitable host.<br>
16.	A recombinant expression vector corprising the nucleic acid con¬struct of claim 15, a promoter, and transcriptional and translational stop signals,<br>
17.	A recorbinant host cell comprising the nucleic acid construct of claim 15,<br>
18. A host cell according to claim 17, which is a bacterium.<br>
19,	A host according to claim 18, wherein the bacterium is a bacillus,<br>
especially a B.   clausii,<br>
20.	A host cell according to claim 17, which is a fungus or yeast, pref*<br>
erably a filamentous fungus, especially an Aspergillus.<br>
21,	A method for producing the subtilase according to any of claims 1-<br>
11, the method comprising:<br>
(a)	cultivating a recombinant host cell as defined in any of claims<br>
17-20 under conditions conducive to the production of the subtilase;<br>
and<br>
(b)	recovering the subtilase.<br>
22.	A method for producing the subtilase according to any of<br>
claims 1-11, the method comprising:<br>
(a) cultivating a strain from the genus Bacillus,   prefera¬bly from the species Bacillus   clausii,    such as Bacillus<br><br>
clausii  DSM 13585; to produce a supernatant comprising the' subtilase; and<br>
(b) recovering the subtilase,<br>
23. A cleaning or detergent composition, preferably a laundry or dishwash composition, comprising the subtilase according to any of claims l-11.<br>
24. A composition according to claim 23, which additionally comprises  a  cellulase,  lipase,  cutinase,  oxidoreductase, another protease, an amylase or a mixture thereof-<br>
25. Use of a subtilase as defined in any of claims 1-11 in a cleaning or detergent composition-<br>
26.	Use of a subtilase as defined in any of claims 1-11 for removal of egg stains.<br>
27.	Use of a cleaning or detergent composition as defined in any of claims 23-24 for removal of egg stains.<br>
28.	A method for cleaning or washing a hard surface or laundry, the method comprising contacting the hard surface or the laundry with the composition defined in claims 23-24.<br>
29- A method for removal of egg stains from a hard surface or from laundry, the method comprising contacting the egg stain-containing hard surface or the egg stain-containing laundry with the composition defined in claims 23-24.<br><br>
30.	A subtilase enzyme substantially as herein described with reference to<br>
the accompanying drawing.<br>
31.	A method for removal of egg stains from a hard surface or from<br>
laundry substantially as herein described with reference to the<br>
accompanying drawing.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtY2xhaW1zIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">185-chenp-2003-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">185-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZGVzY3JpcHRpb24oY29tcGxldGUpIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-description(complete) filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZGVzY3JpcHRpb24oY29tcGxldGUpIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">185-chenp-2003-description(complete) granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">185-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">185-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg1LWNoZW5wLTIwMDMtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">185-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="211565-a-method-of-type-based-access-control-list-for-printing-access.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="211567-method-and-composition-for-the-treatment-of-epidermal-irritations-and-infections.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>211566</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>185/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Nov-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Jan-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsvaerd,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OUTTRUP, Helle</td>
											<td>Skovhaven 89, DK-3500 Vaerloese,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PEDERSEN, Poul, Erik</td>
											<td>Marienborg Alle, 33, DK-2860 Soeborg,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SORENSEN, Marianne, Vind</td>
											<td>Gulddyssevaenget 9, Gundsoemagle, DK-4000 Roskilde,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C11D 11/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK2001/000551</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-08-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2000 01232</td>
									<td>2000-08-21</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/211566-novel-subtilase-enzymes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:17:39 GMT -->
</html>
