<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/211787-gene-encoding-acetolactate-synthase by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:00:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 211787:GENE ENCODING ACETOLACTATE SYNTHASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">GENE ENCODING ACETOLACTATE SYNTHASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A gene encoding an ALS protein which shows an extremely high resistance against PC-based herbicides. Namely, a gene which encodes the following protein (a) or (b). (a) A protein comprising the amino acid sequence represented by SEQ ID NO:1. (b) A protein comprising an amino acid sequence derived from the amino acid sequence represented by SEQ ID NO:1 by substitution, deletion or addition of at least one amino acid and having a resistance against pyrimidinylcarboxy-based herbicides and having an acetolactic acid synthase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Gene Encoding Acetolactate Synthase<br>
FIELD OF THE INVENTION<br>
The present invention relates to a gene encoding acetolactate synthase which is a rate-determining enzyme in the branched-chain amino acid biosynthetic pathway.<br>
BACKGROUND OF THE INVENTION<br>
Acetolactate synthase (hereinafter referred to as "ALS") is a rate-determining enzyme in the biosynthetic pathway of branched chain amino acids, such as leucine, valine and isoleucine, and is known as an essential enzyme for the growth of plants. ALS is also known to be present in a wide variety of higher plants. In addition, ALS is found in various microorganisms, such as yeast (Saccharomyces cerevisiae), Escherichia coli, and Salmonella typhimurium.<br>
Three types of isoenzymes of ALS are known to be present in Escherichia coli and Salmonella typhimurium. Each of these isoenzymes is a hetero oligomer consisting of catalytic subunits with a large molecular weight that governs catalytic activity of the enzyme and regulatory subunits with a small molecular weight that functions as regulatory proteins by binding of branched-chain amino acids (Chipman et al., Biochim. Biophys. Acta. 1385, 401-419, 1998). Catalytic subunits are located at Ilv IH, Ilv GM and Ilv BN operons, respectively. On the other hand, ALS in yeast is a single enzyme, which comprises a catalytic subunit and a regulatory subunit, as is the case in bacteria (Pang et aL, Biochemistry, 38, 5222-5231, 1999). The catalytic protein subunit is located at the locus ILV2.<br><br>
In plants, ALS is known to comprise catalytic subunit(s) and regulatory subunit(s) as is the case in the above microorganisms (Hershey et al,, Plant Molecular Biology. 40, 795-806, 1999). For example, the catalytic subunit of ALS in tobacco (dicotyledon) is encoded by two gene loci, SuRA and SuRB (Lee et al., EMBO J. 7, 1241-1248, 1988); that in maize is encoded by two gene loci, als 1 and als 2 (Burr et al, Trendsin Genetics 7, 55-61, 1991; Lawrence et al-, Plant Mol. Biol. 18, 1185-1187, 1992). The nucleotide sequences of genes encoding a catalytic subunit have been completely determined for dicotyledonous plants including tobacco, Arabidopsis, rapeseed, cotton, Xanthium, Amaranthus and Kochia (See Chipman et al., Biochim. Biophys, Acta. 1385, 401-419, 1998 and domestic re-publication of PCT international publication for patent applications WO97/08327). However, maize is the only monocotyledonous plant whose nucleotide sequence has been completely determined.<br>
Herbicides, for example, sulfonylurea herbicides, imidazolinon herbicides, triazolopyrimidine herbicides and pyrimidinyl carboxy herbicides (hereinafter referred to as "PC herbicides) are known to suppress the growth of a plant by inhibiting ALS (Ray, Plant Physiol. 75, 827-831, 1984; Shaner et al., Plant Physiol.76, 545-546, 1984; Subramanian et al., Plant Physiol. 96, 310-313, 1991; Shimizu et al., J. Pestic. Sci.19, 59-67, 1994).<br>
Known plants having resistance to these herbicides contain a gene encoding ALS that includes substitution of one or two nucleotides which induces substitution of one or two amino acids in a region conserved among different species. Examples of such a gene include a gene encoding ALS having resistance specific to sulfonylurea herbicides (see Kathleen et al., EMBO J. 7,1241-1248, 1988; Mourad et al., Planta, 188, 491-497, 1992; Guttieri et ah, Weed Sci. 43,175-178, 1995; Bernasconi et al., J. Biol Chem. 270, 17381 -17385, 1995 and Japanese Patent Application Laying-Open (kokai) No. 63-71184); a gene encoding ALS having resistance specific to imidazolinon herbicides (Mourad et al., Planta, 188, 491-497, 1992; Lee et al, FEBS Lett 452,341-345, 1999<br><br>
and Japanese Patent Application Laying-Open (kokai) No.5-227964); and a gene encoding ALS having resistance to both sulfonylurea and imidazolinon herbicides (see Kathleen et al., EMBO J. 7,1241-1248, 1988; Bernasconi et aL, J. Biol. Chem. 270, 17381-17385, 1995; Hattori et al., Mol. Gen. Genet. 246,419-425, 1995; Alison et al., Plant Physiol. Ill, 1353, 1996; Rajasekarau et al., Plant Sci. 119, 115-124, 1996, Japanese Patent Application Laying-Open (kokai) No.63-71184, Japanese Patent Application Laying-Open (kokai) No.4-311392 and Bernasconi et al., US Patent 5633437, 1997). ALS showing resistance to both sulfonylurea and imidazolinon herbicides is also known to show cross resistance to PC herbicides and triazolopyrimidine herbicides (Bernasconi et al, J. Biol. Chem. 270, 17381-17385, 1995), Further, the production of a plant body showing resistant to both sulfonylurea and imidazolinon herbicides has been attempted by crossing a plant having ALS showing resistance specific to sulfonylurea herbicides with a plant having ALS showing resistance specific to imidazolinon herbicides (Mourad et al, Mol, Gen, Genet, 243, 178-184, 1994). Furthermore, artificial alteration of a gene encoding ALS into a herbicide resistance gene has been attempted (Ott et al., J, Mol. Biol. 263, 359-368, 1996, Japanese Patent Application Laying-Open (kokai) No.63-71184, Japanese Patent Application Laying-Open (kokai) No.5-227964, Japanese Patent Application Laying-Open (kohyo) No.11-504213) so that it has been found that a single amino acid deletion causes ALS to show resistance to both sulfonylurea and imidazolinon herbicides (see Japanese Patent Application Laying-Open (kokai) No.5-227964).<br>
As described above, ALSs having resistance to herbicides and genes encoding ALS have been aggressively studied. However, there has been no report directed toward resistance to PC herbicides and concerning a mutant ALS gene having resistance specific to a PC herbicide.<br><br>
The purpose of the present invention is to provide a gene encoding an ALS protein showing extremely high resistance to PC herbicides, the ALS protein encoded by the gene, a recombinant vector having the gene, a transformant having the recombinant vector, a plant having the gene, a method for rearing the plant, and a method for selecting a transformant cell using the gene as a selection marker.<br>
As a result of thorough studies to achieve the above purpose, we have completed the present invention by finding that a mutant ALS, which is derived from the wild type ALS by substituting a certain amino acid residue of the wild type ALS with a certain amino acid, shows extremely high resistance to PC herbicides.<br>
(1)	The present invention is a gene which encodes the following protein (a) or (b):<br>
(a)	a protein comprising an amino acid sequence of SEQ ID NO: 1;<br>
(b)	a protein comprising an amino acid sequence derived from the amino acid sequence of SEQ ID NO: 1 by substitution, deletion or addition of at least one or more amino acids, which has resistance to pyrimidinyl carboxy herbicides and has acetolactate synthase activity.<br><br>
(2)	Further, the present invention is an acetolactate synthase protein which is encoded by the gene of (1).<br>
(3)	Furthermore, the present invention is a recombinant vector which has the gene of (!)■<br>
(4)	Further, the present invention is a transfomiant which has the recombinant vector of (3).<br>
(5)	Moreover, the present invention is a plant which has the gene of (1) and has<br><br>
resistance to pyrimidinyl carboxy herbicides,<br>
(6)	The present invention is a method for rearing the plant of (5) which comprises rearing the plant in the presence of a pyrimidinyl carboxy herbicide.<br>
(7)	The present invention is a method for selecting a transformant cell having the gene of (1) using this gene as a selection marker.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows a comparison between the amino acid sequence of a mutant ALS protein and that of a wild type ALS protein.<br>
Figure 2 shows a comparison between the nucleotide sequence of a mutant ALS gene and that of a wild type ALS gene.<br>
Figure 3 is a characteristic figure showing sensitivity of the resistant mutant to bispyribac-sodium.<br>
Figure 4  is a characteristic figure  showing sensitivity of the wild type to bispyribac-sodium.<br>
Figure 5 is a characteristic figure showing sensitivity of the wild type to chlorsulfuron.<br>
Figure 6 is a characteristic figure showing sensitivity of the resistant mutant to chlorsulfuron.<br>
Figure 7 is a characteristic figure showing negative ion exchange chromatogram<br><br>
of the crude ALS protein extracted from the wild type.<br>
Figure 8 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant protein to bispyribac-sodium.<br>
Figure 9 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant protein to chlorsulfuron.<br>
Figure 10 is a characteristic figure showing sensitivity of the wild type ALS protein and the mutant protein to imazaquin.<br>
Figure 11 shows a comparison of nucleotide sequences between Nippon-bare EST and maize ALS gene.<br>
Figure 12 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to bispyribac-sodium.<br>
Figure 13 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to chlorsulfuron.<br>
Figure 14 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to pyrithiobac-sodium.<br>
Figure 15 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to pyriminobac.<br>
Figure 16 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to bensulfuron-methyl.<br><br>
Figure 17 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to pyrazosulfuron-ethyl.<br>
Figure 18 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to imazosulfuron.<br>
Figure 19 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to imazaquin.<br>
Figure 20 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 4 to imazapyr.<br>
Figure 21 is a characteristic figure showing sensitivity observed for the wild-type crude ALS? the wild-type purified GST-ALS and the wild-type GST-freed ALS obtained in Example 4 to bispyribac-sodium.<br>
Figure 22 is a characteristic figure showing sensitivity observed for the mutant crude ALS and the mutant purified GST-ALS obtained in Example 4, to bispyribac-sodium.<br>
Figure 23 is a characteristic figure showing sensitivity observed for the mutant crude ALS and the mutant purified GST-ALS obtained in Example 4, to chlorsulfuron.<br>
Figure 24 is a characteristic figure showing sensitivity observed for the mutant crude ALS and the mutant purified GST-ALS obtained in Example 4, to imazaquin.<br>
Figure 25 shows the concept for generating the ALS gene having W548L mutation only.<br><br>
Figure 26 is continued from Fig. 25 and shows the concept for generating the ALS gene having W548L mutation only.<br>
Figure 27 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 6 (2) to bispyribac-sodium.<br>
Figure 28 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 6 (2) to chlorsulfuron.<br>
Figure 29 is a characteristic figure showing sensitivity observed for the crude ALS obtained in Example 6 (2) to imazaquin.<br>
Figure 30 shows the concept for constructing the binary vector constructed in Example 7.<br>
Figure 31 is a photograph of electrophoresis by which the ALS gene insert was confirmed in the plasmid isolated from the single colony of E. coli (JM109), wherein lane 1 indicates XHind III/100 bp DNA marker, lane 2 indicates a plasmid derived from E. coli that has been transformed with pMLH7133 retaining the mutant ALS gene, lane 3 indicates a plasmid derived from E, coli that has been transformed with pMLH7133 retaining the wild type ALS gene, lane 4 indicates pMLH7133 retaining a GUS gene (control group), and arrows indicate the insert DNA.<br>
Figure 32 is a photograph of electrophoresis showing the result of PCR using plasmid DNAs as a template, wherein lane 1 indicates XHind III/100 bp DNA marker, lanes 2 and 5 indicate a pBI 121 plasmid (control group) retaining the wild type ALS gene, lanes 3 and 6 indicate pMLH7133 which is thought to retain the mutant ALS gene, lanes 4 and 7 indicate pMLH 7133 which is thought to retain a wild type ALS gene, and arrows indicate PCR products.<br><br>
Figure 33 is a photograph of electrophoresis showing the result of confirming the insertion orientation of ALS gene, wherein lane 1 indicates XHind III/100 bp DNA marker, lane 2 indicates pMLH7133 retaining a mutant ALS gene, lane 3 indicates pMLH7133 retaining a wild type ALS gene, lane 4 indicates pMLH7133 (control group) retaining a GUS gene, and arrows indicate PCR products.<br>
Figure 34 is a flow chart showing the method for transforming rice plants (Nippon-bare) using Agrobacterium.<br>
Figure 35 is a photograph showing rice plants (Nippon-bare) under redifferentiation from calli.<br>
Figure 36 is a characteristic figure showing bispyribac-sodium sensitivity of the callus of a rice plant (Nippon-bare) transformed with a mutant ALS gene.<br>
Figure 37 is a characteristic figure showing bispyribac-sodium sensitivity of the callus of the wild type rice plant (Nippon-bare).<br>
Figure 38 is a photograph of electrophoresis showing the results of PCR performed using genomic DNA prepared from a transformant callus, wherein lane 1 indicates XHind III/100 bp DNA marker, lanes 2 to 7 show the respective results of PCR using genomic DNAs derived from different calli, and arrow indicates the PCR products.<br>
Figure 39 is a characteristic figure showing the result of sequencing of the PCR product resulting from PCR using the genomic DNA prepared from the transformant callus as a template, wherein A shows the result of analysis of the periphery of an amino acid residue 548 using a sense primer (3-1-4), B shows the result of analysis of the<br><br>
periphery of an amino acid residue 548 using an antisense primer (ALS-Rsp2), C shows the result of analysis of the periphery of an amino acid residue 627 using a sense primer (3-1-4), and D shows the result of analysis of the periphery of an amino acid residue 627 using an antisense primer (ALS-Rsp2).<br>
Figure 40 is a photograph showing the result of measuring bispyribac-sodium sensitivity of a rice plant (Nippon-bare) transformed with the mutant ALS gene, wherein A shows a rice plant transformed with the mutant ALS gene, and B shows a rice plant transformed with the wild type ALS gene.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
A more detailed explanation will be given of the present invention.<br>
The acetolactate synthase protein of the present invention (hereinafter referred to as "mutant ALS protein") can be obtained by mutating a certain site in a wild type ALS protein expressed in a rice plant. The amino acid sequence of the mutant ALS protein of the present invention is represented by SEQ ID NO: 1. The nucleotide sequence of the gene encoding the ALS protein of the present invention is represented by SEQ ID NO: 2.<br>
The mutant ALS protein is derived from a wild type ALS protein by substituting tryptophan 548 with leucine and substituting serine 627 with isoleucine. Figure 1 shows the result of comparison of the amino acid sequence of the mutant ALS protein and that of the wild type ALS protein. In Fig. 1, the amino acid sequence in the first row shows the mutant ALS protein and that in the second row shows the wild type ALS protein.<br>
Compared to a gene encoding the wild type ALS protein, a gene (SEQ ID NO: 2) encoding the mutant ALS protein contains substitution of a codon encoding tryptophan<br><br>
548 with a codon encoding leucine and substitution of a codon encoding a serine 627 with a codon encoding isoleucine. Figure 2 shows the result of comparison of the nucleotide sequence of the gene encoding the mutant ALS protein and that encoding the wild type ALS protein. In Fig. 2, the nucleotide sequence in the first row shows the mutant ALS protein and that in the second row shows the wild type ALS protein.<br>
A gene encoding the mutant ALS protein can be obtained by introducing a mutation as described above into a gene encoding a wild type ALS protein which is present in the genome DNA of japonica type rice variety, Kinmaze. To introduce mutations, known standard techniques can be employed. For example, site-directed mutagenesis can be used. Site-directed mutagenesis can be performed using a commercial kit, e.g. Mutan-K (Takara Shuzo), Gene Editor (Promega) or Exsite (Stratagene).<br>
In addition, a gene encoding the mutant ALS protein can be obtained by culturing wild type culture cells sensitive to PC herbicides in the presence of a PC herbicide and then selecting mutant culture cells that emerge and show resistance to the PC herbicide.<br>
First, mRNAs is prepared from mutant culture cells resistant to a PC herbicide, cDNAs are synthesized, and then a cDNA library (which is thought to have heterozygosity for resistance property), of A,gt 11 phage is produced. Then, the library is screened using a nucleic acid probe [EST (Expression Sequence Tag, Genbank, DDBJ and EMBL Accession No. C72411) obtained from japonica type rice variety, Nippon-bare] containing part of a gene encoding the wild type ALS protein. Next, the insert DNA of the resulting positive clone is subcloned into pBluescript II SK+, to determine the nucleotide sequence. Selection of a clone having the insert DNA which encodes the amino acid sequence of SEQ ID NO: 1 enables a gene encoding a mutant ALS protein to be obtained. In addition, a plasmid comprising pBluescript II SK+ into which a gene encoding mutant ALS protein had been incorporated was deposited to<br><br>
Patent and Bio-Resource Center, National Institute of Advanced Industrial Science and Technology (Chuo-6, 1-1-1, Higashi, Tsukuba-shi, Ibaraki, JAPAN) as Mutant ALS cDNA in pBluescript II SK+ (FERM BP-7348) on November 2, 2000 under the Budapest Treaty.<br>
Compared to the wild type ALS protein, the mutant ALS protein possesses good resistance not only to PC herbicides, but also to sulfonylurea and imidazolinon herbicides. Particularly, the mutant ALS protein possesses a high level of resistance to a PC herbicide. It can be determined by incorporating a gene encoding the mutant ALS protein into an expression vector of e.g. E. coli, and examining sensitivity to the herbicide of E. coli transformed with the expression vector.<br>
Examples of a PC herbicide include bispyribac-sodium, pyrithiobac-sodium and pyriminobac as represented by the following chemical formula 1.<br><br><br>
Examples of a sulfonylurea herbicide include chlorsulfuron, bensulfuron-methyl, pyrazosulfuron-ethyl, and imazosulfuron as represented by the following chemical formula 2.<br><br>
Transformation of a target plant using a gene encoding the mutant ALS protein can impart resistance to PC herbicides to the plant. Known standard techniques can be used for transformation of a plant. For example, a foreign gene can be introduced into a target plant cell using Agrobacterhtm tiimefaciens.<br><br>
More specifically, a gene encoding the mutant ALS protein is inserted into a binary vector containing T-DNA sequence of a Ti plasmid of Agrobacterium. The Ti plasmid is transformed into E. coli and the like. Then, the binary vectors containing the gene encoding the mutant ALS protein replicated by e.g. E. coli are transformed into Agrobacteria which contain helper plasmids. Target plants are infected with the Agrobacteria, and then the transformed plants are identified. When the identified transformed plant is a culture cell, the plant cell can be regenerated into a complete plant by known standard techniques.<br>
To transform a target plant with a gene encoding the mutant ALS protein, the gene can be directly introduced using known standard techniques. Examples of a method which transforms an expression vector containing a gene encoding the mutant ALS protein include the polyethylene glycol method, electroporation, and particle gun method.<br>
A gene encoding the mutant ALS protein may be transformed into any type of plants, such as monocotyledonous and dicotyledonous plants. Examples of a target crop into which a gene encoding the mutant ALS protein is transformed include rice, maize, wheat, barley, soybean, cotton, rapeseeds, sugar beet and tobacco. In addition, turf grass, trees and the like can be transformed by introducing a gene encoding the mutant ALS protein.<br>
In any of the above cases, transformation of a plant using a gene encoding the mutant ALS protein can impart resistance to PC herbicides, sulfonylurea herbicides, and imidazolinon herbicides to the plant.<br>
Moreover, a gene encoding the mutant ALS protein can also be used as a selection marker in an experiment for transformation of a plant. For example to transform a plant cell using a gene of interest, a vector which contains a gene encoding the mutant ALS<br><br>
protein and a gene of interest is introduced into the plant cell, followed by culturing of the plant cell under the presence of a PC herbicide. Therefore, a plant cell which survives in the presence of a PC herbicide indicates that it contains a gene encoding the mutant ALS protein and the gene of interest introduced therein. Further, whether a gene of interest and a gene encoding the mutant ALS protein are incorporated into the chromosome of a plant cell can be confirmed by observing the phenotype of the plant and then examining the presence of these genes on the genome by genome southern hybridization or PCR.<br>
EXAMPLE<br>
Now, the present invention will be further described by the following examples, but the technical scope of the invention is not limited by these examples.<br>
Example 1: Production of rice (Kinmaze) culture cells resistant to a PC herbicide<br>
Chaff was removed from rice seeds (variety; Kinmaze, scientific name: Oryza sativa var. Kinmaze). The seeds were immersed in 70 % ethanol for 5 min, and then immersed in about 5 % antiformin for 20 min, followed by washing several times with sterile distilled water. Then, the seeds were static-cultured on a medium with a composition as shown in Table 1.<br>
Table 1<br><br><br>
In the above medium composition, 2,4-D is synthesized auxin. To prepare the medium, first, a medium with the above composition was placed in a 11 beaker, and distilled water was added to the beaker to 1000 ml. Next, the solution was adjusted to pH 5.7, and supplemented with 3 g of Gelrite. The Gelrite was dissolved well by heating with a microwave oven, and then the mixture was added 30 ml at a time to culture flasks using a pipetter. Next, three sheets of aluminum foil were laid over the culture flask, followed by heating for sterilization in an autoclave at 121 °C for 15 to 20 min. Finally the solution was cooled to room temperature so that the media for static culture of the above seeds were prepared.<br>
Next, endosperm portions were removed from the callus induced on the medium, and then subculture was performed. Then, part of the obtained calli was sub-cultured, that is, cultured successively once per two weeks in a liquid medium (the composition is the same as in that shown in Table 1, but not supplemented with Gelrite) supplemented with 1 µM bispyribac-sodium (one of PC herbicides). Two to 6 weeks later, the culture cells started to wither. About 2 months later, a plurality of non-discolored cell masses that were thought to be conducting cell division were obtained from among culture cell populations most of which had died and became discolored brown. These cell masses were isolated and cultured, so that a plurality of cell lines that can proliferate in the presence of 2 µM bispyribac-sodium were obtained.<br>
Subsequently, the resulting plurality of cell lines were cultured while elevating the concentration of bispyribac-sodium in an orderly manner. As a result, cell lines that can proliferate in the presence of 100 µM bispyribac-sodium were obtained. The bispyribac-sodium resistant culture cells (hereinafter referred to as resistant mutant) were sub-cultured on MS-2,4-D solid media supplemented with 100 µM bispyribac-sodium. Part of the sub-cultured resistant mutant was sampled, added into<br><br>
MS-2,4-D liquid media not supplemented with bispyribac-sodium, and then subjected to suspended cell culture at a cycle of 8 to 10 days.<br>
Approximately 1,5 g (wet weight) of the resistant mutant was transplanted in 200 ml of a Erlenmeyer flask supplemented with 50 ml of a MS-2,4-D liquid medium and bispyribac-sodium at an appropriate concentration, followed by culturing at approximately 27 °C for an appropriate period. The wet weight of the callus was measured periodically. The relative amount of increase was determined based on the wet weight of the transplanted resistant mutant. In addition, the experiment was performed three times with different bispyribac-sodium concentrations, and the standard error was calculated. Figure 3 shows the relation of changes in bispyribac-sodium concentration and the relative weight on day 8 in the resistant mutant. As a control, a similar experiment was conducted using the wild type (Kinmaze). Figure 4 shows the result of measuring the relation of bispyribac-sodium concentration and relative weight on day 8.<br>
As shown in Fig. 4, the growth of the wild type was not inhibited in a group supplemented with 1 nM bispyribac-sodium, but was inhibited in a group supplemented with 10 nM or more bispyribac-sodium. However as shown in Fig. 3, almost none of the growth of the resistant mutant was affected even in a group supplemented with 10 µM bispyribac-sodium.<br>
The growth rates of the wild type and the resistant mutant were measured as described above except for the use of chlorsulfuron instead of bispyribac-sodium. Figure 5 shows the relation of changes in chlorsulfuron concentration and relative weight on day 9 in the wild type. Figure 6 shows the relation of changes in chlorsulfuron concentration and relative weight on day 9 in the resistant mutant.<br>
As shown in Fig. 5, the growth of the wild type was inhibited by addition of 1 nM<br><br>
chlorsulfuron, showing that the wild type has higher sensitivity to chlorsulfuron than to bispyribac-sodium. However, as shown in Fig. 6, the growth of the resistant mutant was strongly inhibited by addition of 1 µM chlorsulfuron. Sensitivity to bispyribac-sodium and chlorsulfuron remained almost unchanged in both the wild type and the resistant mutant, even with longer culture duration. The growth rate was almost the same in the wild type and the resistant mutant.<br>
These results revealed that the resistant mutant possesses resistance specific to bispyribac-sodium. Moreover, the resistant mutant was shown to have improved resistance to chlorsulfuron compared to the wild type.<br>
Example 2: Herbicide sensitivity of ALS enzyme partially purified from the resistant mutant<br>
In this example, mutant ALS protein was partially purified from the resistant mutant obtained in Example 1, and then herbicide sensitivity of the obtained mutant ALS protein was examined. The mutant ALS protein was partially purified as follows.<br>
First, 200 g or more of resistant mutant was prepared by a liquid culture method (no supplementation with bispyribac-sodium), the culture having a composition as shown in Table 1 but without Gelrite. Then, about 150 g of the resistant mutant was homogenized using Hiscotron in a volume of buffer-1 [100 mM potassium phosphate buffer (pH 7.5) containing 20 % (v/v) glycerol, 0.5 mM thiamin pyrophosphate (TPP), 10 µM flavin adenine dinucleotide (FAD), 0.5 mM MgCl2, and a volume of polyvinyl polypyrrolidone one-tenth that of tissue volume] 3-fold greater than tissue volume. The homogenate was filtered through nylon gauze, and then centrifuged at 15,000 xg for 20 min. Ammonium sulfate was added to the centrifuged supernatant to 50 % saturation, and then allowed to stand in ice for approximately 1 hour. The mixture was again centrifuged at 15,000 xg for 20 min, and then the precipitated fraction was dissolved in about 30 ml of buffer-2 [10 mM Tris hydrochloric acid buffer (pH 7.5) containing 20 %<br><br>
(v/v) glycerol, 0.5 mM TPP and 0.5 mM MgCl2]. The mixture was again centrifuged at 15,000 xg for 20 min, and then the supernatant fraction was applied to a Sephadex G-25 (Amersham Pharmacia Biotec). About 40 ml of the fraction that had passed through the column was collected as a crude enzyme solution.<br>
Next, the protein concentration of the crude enzyme solution was measured by the Bradford method according to the manual of Bio-Rad Protein Assay. The crude enzyme solution was then filtered through a Whatman filter (Whatman), and then the crude enzyme solution in an appropriate protein amount (10 to 15 ml) was applied to three vertically-connected HiTrap Q columns (Amersham Pharmacia Biotec) using a FPLC device (Amersham Pharmacia Biotec). After protein component was adsorbed using HiTrap Q, unadsorbed fractions were washed out using buffer-2 having a volume 3 to 5 fold greater than the bed volume. Then, the adsorbed protein component was eluted using an eluate having a volume 10 fold greater than the bed volume (150 ml). Here, the eluate was prepared by dissolving KC1 with a linear concentration gradient (0 to 0.4 M) into buffer-2. The eluate containing the eluted protein component was apportioned, 5 ml each, into a plurality of test tubes for apportion. Further, to stabilize ALS enzyme contained in the eluted protein component, 0.5 ml of buffer-2 containing 20 mM sodium pyruvate had been previously added to each test tube for apportion.<br>
ALS activity resulting from the mutant ALS protein contained in the eluted fractions apportioned into each test tube for apportion was measured as follows. A reaction solution to be used in a measurement reaction was prepared by mixing an eluted fraction to be measured with a solution comprising 20 mM sodium pyruvate, 0.5 mM TPP, 0.5 mM MgCl2, 10 µM FAD and 20 mM potassium phosphate buffer (pH 7.5). One ml of this reaction solution was used. After the eluted fraction to be measured was added, the measurement reaction was performed at 30 °C for 40 to 60 min. Then, the reaction was stopped by addition of 0.1 ml of 6N sulfuric acid (or 0.25 N sodium hydroxide).<br><br>
After the reaction was stopped, the reaction solution was incubated at 60 °C for 10 min, thereby converting acetolactate contained in the reaction solution to acetoin.<br>
Then, to quantify acetoin contained in the reaction solution, 1ml of 0.5 % (w/v) creatine and 1ml of 5 % (w/v) a-naphthol dissolved in 2,5 N sodium hydroxide was added to the reaction solution, followed by incubation at 37 °C for 10 min. Acetoin was then quantified by color comparison of the absorbance (at 525 nm) of the reaction solution, thereby evaluating ALS activity. In addition, since the reaction solution contained a small amount of sodium pyruvate, reaction time 0 was used as control.<br>
As a result, absorbance at OD525 nm was as high as approximately 7 per 0.2 ml of the reaction solution. However, when the above measurement reaction was ceased with sodium hydroxide, and acetoin generation activity due to activity other than ALS activity was examined, nearly 80 % of the apparent ALS activity resulted from direct acetoin generation activity which was not due to activity of the mutant ALS protein. Accordingly, the mutant ALS protein and the other proteins were separated for acetoin generation activity by FPLC using negative ion exchange resin. As a result, two activity peaks were detected as shown in Fig, 7.<br>
To determine which one of the two activity peaks corresponded to the mutant ALS protein, acetoin generation activity was examined for the two peaks. Thus it was found that a fraction shown by the earlier peak corresponded to the mutant ALS protein.<br>
Using the enzyme solution containing the mutant ALS protein, sensitivity of the mutant ALS protein to bispyribac-sodium, chlorsulfuron and iniazaquin was examined. Sensitivity to each of these herbicides was evaluated by measuring ALS activity in the same manner as in the above measurement reaction, except that a herbicide was added to a certain concentration before addition of the enzyme solution. For comparison, the<br><br>
wild type ALS protein was separated and purified in the same manner and used for the experiment. In addition, bispyribac-sodium was prepared as an aqueous solution, and chlorsulfuron and imazaquin were prepared as acetone solutions. The final concentration of acetone in the reaction mixture was 1 %.<br>
Figure 8 shows the relation of ALS activity inhibition rate and bispyribac-sodium concentration. Figure 9 shows the relation of ALS activity inhibition rate and chlorsulfuron concentration. Figure 10 shows the relation of ALS activity inhibition rate and imazaquin concentration. A line connecting white squares denotes the wild type ALS protein, a line connecting black squares denotes the mutant ALS protein in Figs. 8 to 10.<br>
A herbicide concentration which inhibits 50 % of ALS activity (150) was calculated according to probit analysis, thereby calculating the ratio of 150 for the mutant ALS protein vs. 150 for the wild type ALS protein. Table 2 shows the results.<br><br>
a)	Concentration for 50 % inhibition<br>
b)	Calculated from the inhibition activity in Table 1; 150 (resistant mutant) /150 (wild type)<br>
As shown in Figs. 8 to 10 and Table 2, the mutant ALS protein showed a relatively high ALS activity even in the presence of the herbicide, when compared to the wild type ALS protein. In particular, the most significant difference between the mutant and the wild type ALS protein was sensitivity to bispyribac-sodium herbicide. That is,<br><br>
the mutant ALS protein possesses good resistance to bispyribac-sodium.<br>
Example 3: Cloning of mutant ALS gene<br>
Probes used for cloning a gene (mutant ALS gene) encoding the mutant ALS protein from the resistant mutant were prepared as follows. The partial cDNA used as a probe in this example was derived from rice (Nippon-bare) showing high homology with the ALS gene of maize.<br>
(1) Determination of the nucleotide sequence of a partial cDNA derived from rice (Nippon-bare) showing high homology with the ALS gene of maize<br>
As a part of the Ine Genome Project conducted by the Society for Techno-innovation of Agriculture, Forestry and Fisheries, and National Institute of Agrobiological Sciences, partial nucleotide sequences of cDNAs of rice (Nippon-bare) had been determined and a partial nucleotide sequence database of cDNAs had already been established. A cDNA clone (Accession No. C72411) which is known as a nucleotide sequence of approximately 350 bp contained in this database showed high homology to the ALS gene of maize. The ALS gene of maize had been completely sequenced.<br>
This cDNA clone (Accession No. C72411) was obtained from National Institute of Agrobiological Sciences, and the nucleotide sequence was determined as follows. Here, the cDNA clone comprised an ALS homolog gene inserted within pBluescript II SK+, and it was capable of autonomous replication in E, coli.<br>
First, an ALS homolog-retaining plasmid vector was transformed into E. coli (DH5oc). White colonies obtained from a plate were cultured in liquid, and then plasmids were extracted from the cells by standard techniques. Since the insert DNA had been inserted between Sail and Notl (restriction enzymes of multi-cloning sites in the plasmid vector), the vector was digested with the two enzymes. The insert was<br><br>
confirmed by agarose electrophoresis. Then, the obtained ALS homolog-retaining plasmid vector was purified by standard techniques using e.g. RNaseA, PEG and LiCl, followed by sequencing reaction using primers and an ABI BigDyeTerminator Cycle Sequencing Kit. Conditions for PCR reaction followed the manufacture's protocol. Primers used herein were Ml3 primers and synthesized primers designed from the determined nucleotide sequence. The resulting PCR product was purified by ethanol precipitation, and then the nucleotide sequence thereof was determined by an ABI PRISM 310 sequencer.<br>
The ALS homolog-retaining plasmid vector is known to contain an insert DNA with a length of 1,6 kb. The obtained ALS homolog-retaining plasmid vector was digested with restriction enzymes Sa 11 and Not I, and then subjected to electrophoresis. Therefore, a band of about 3 kbp corresponding to pBluescript II SK+ and a band of about 1.6 kbp corresponding to the insert DNA fragment were detected (not shown). The entire nucleotide sequence of the insert DNA portion was determined, and its homology to the nucleotide sequence of maize was searched. As shown in Fig. 11, 84.7 % homology was found. Since the ALS homolog was determined to be a partial cDNA of the ALS gene of the var. Nippon-bare, the insert DNA digested with Sal I and Not I was used as a probe. Further in Fig. 11, the first row is a nucleotide sequence of the partial cDNA of the ALS gene of the var. Nippon-bare; the second row is that of the ALS gene of maize.<br>
(2) Preparation of mRNA from resistant mutant<br>
First, the resistant mutant frozen with liquid nitrogen was crushed with a mortar and pestle, and then finely crushed with a mixer for 30 sec. The crushed powder was suspended in an extraction buffer [(100 mM Tris-HCl pH 9.0, 100 mM NaCl, 1 wt % SDS, 5 mM EDTA): (p-mercaptoethanol) : (Tris saturated phenol) = 15:3:20], and then stirred thoroughly. This solution was centrifuged at 12,000 xg for 15 min, and then the supernatant  was  collected.   Two  hundred  ml  of PCI   [(Tris   saturated  phenol)   :<br><br>
(chloroform) : (isoamylalcohol) = 25:24:1] was added to the supernatant, shaken at 4 °C for 10 min, centrifuged at 12,000 xg for 15 min, and then the supernatant was collected. The procedure was repeated twice. A 1/20 volume of 5 M NaCl and a 2.2-fold volume of ethanol were added to the obtained supernatant, and then the mixture was allowed to stand at -80 °C for 30 min. The precipitate was collected by centrifugation at 12,000 xg for 5 min. The precipitate was washed with 70 % ethanol, dried, and then dissolved in 10 mM p-mercaptoethanol solution. Next, the solution was centrifuged at 27,000 xg for 10 min to remove insoluble fraction. A 1/4 volume of 10 M LiCl was added to the solution, which was then allowed to stand on ice for 1 hour. Further, the solution was centrifuged at 27,000 xg for 10 min to collect precipitate, dissolved in 4 ml of H2O, and then absorbance at 260 nm was measured to find the concentration of RNA. A 1/20 volume of 5 M NaCl and a 2.2-fold volume of ethanol were added to the solution, which was then allowed to stand at -80 °C for 30 min. Subsequently the solution was centrifuged at 27,000 xg for 10 min to collect the precipitate, followed by washing with 70 % ethanol, and drying. The resulting product was dissolved in an appropriate amount of H2O to obtain a total RNA solution. Here, centrifugation was performed at 4 °C.<br>
mRNA was separated and purified from total RNA by the following method, A 2x binding buffer (20 mM Tris-HCl pH 7.5, 10 mM EDTA, 1 M NaCl) in a volume equivalent to that of the extracted total RNA solution was added to the extracted total RNA solution. A column filled with 0.1 g of oligo dT cellulose (Amersham Pharmacia Biotec) was washed with a lx binding buffer, and then the total RNA solution was applied to the column. After the column was washed with a lx binding buffer, an elution buffer (10 mM Tris-HCl pH 7.5, 5 mM EDTA) was applied, and the eluate collected 0.5 ml at a time. Fractions that had passed through the column were applied to another oligo dT cellulose (Amersham Pharmacia Biotec) column, and treated in the same manner. After the concentration of eluted mRNA was calculated based on the absorbance of each fraction, a 1/10 volume of 10 M LiCl and a 2.5-fold volume of ethanol were added to<br><br>
the products, and then the mixtures were allowed to stand at -80 °C for 30 min. Next, the mixtures were centrifuged and the precipitated fractions were dried, and dissolved in 100 µM of H2O, The thus obtained mRNA was subjected to size fractionation by sucrose density gradient centrifugation.<br>
The separated and purified mRNA was applied to a centrifuge tube with density gradient given by a 25 % sucrose solution and 5 % sucrose solution, and then ultracentrifuged at 27,000 rpm for 15 hours at 4 °C using a swing rotor. After centrifugation, 0.5 ml of each fraction was collected in order of density gradient. Absorbance of each fraction was measured, the concentration of the collected mRNA was calculated, and the presence of ALS mRNA was confirmed by hybridization using an ECL kit (ECL direct nucleic acid labeling and detection system, Amersham Pharmacia Biotec). Hybridization was performed using a probe prepared in (1) above at 42 °C for 16 hours. After hybridization, washing at 42 °C for 5 min was performed twice using a primary washing buffer provided with the kit, and then washing at 42 °C for 5 min was performed once using 2x SSC solution. The washed film was wrapped with a transparent plastic film to keep it immersed in an attached luminous reagent provided with the kit, and then exposed to an X-ray film.<br>
Approximately 35 mg of total RNA was extracted from the resistant mutant and approximately 4 mg of mRNA could be extracted by the above procedures. Further, in sucrose density gradient centrifugation, a hybridization-positive spot was found for a fraction expected to be positive.<br>
When the wild type was used, approximately 95 mg of total RNA was extracted in addition to approximately 7 mg of mRNA. When mRNA was extracted from the wild type, the above method was applied except that the wild type was used instead of the resistant mutant.<br><br>
(3) Construction of cDNA library derived from resistant mutant<br>
Using 2 µM of mRNA purified in (2) above and a cDNA synthesis kit (Amersham Pharmacia Biotec), cDNA was synthesized, so that a cDNA library derived from the resistant mutant was constructed.<br>
First, RTase provided with the kit was used for a reverse transcription reaction; T4 DNA polymerase provided with the kit was used for a subsequent complementary chain<br>
in<br>
elongation reaction. Upon complementary chain elongation reaction, P-dCTP was added to calculate the yield of cDNA synthesis. After an adaptor was added, the synthesized cDNA was incorporated into X phage by in vitro packaging method.<br>
The adaptor added to cDNA was an Eco Rl~Not l-Bam HI adaptor (Takara Shuzo). Adapters with a molar concentration 50-fold greater than that of cDNA were added to a solution containing cDNA. Then, T4 DNA Ligase (Pharmacia) was added to the mixture followed by ligation reaction at 4 °C overnight. The reaction solution was applied to HPLC using an AsahiPak GS 710 column (Asahi Chemical Industry Co., Ltd.), followed by monitoring the eluate with ultraviolet rays at a wavelength of 260 nm. The eluate was fractionated into 25 fractions of 0.5ml each. Each fraction was measured with a Cerenkov counter, and 3 to 4 fractions with a high count were collected. The 5' terminus of the adaptor contained in the fraction was phosphorylated using T4 polynucleotide kinase (Takara Shuzo), and then A,gt 11 Eco RI arm was added to perform ligation. GigaPack Gold III (Stratagene) was added to the solution, and then ligation reaction was performed at room temperature for 2 hours. After reaction, 200 µM of a SM buffer and 8 µM of chloroform were added to the reaction solution, thereby preparing a phage solution. This phage solution was diluted 10-fold, One µM of the diluted solution was infected with E. coli (Y-1088), to which 0.7 % top agar was added, and then the solution was inoculated over a LB plate. The number of plaques that had emerged on the plate 4 to 8 hours later was counted, thereby measuring the titer.<br><br>
Synthesis of approximately 74 ng of cDNA derived from the resistant mutant was confirmed by the result of DE 81 paper and Cerenkov counting, The result of Cerenkov counting after ligation of a vector with an adaptor added thereto revealed that approximately 22 ng of XDNA containing the insert inserted therein was obtained for the resistant mutant. The ADNA was packaged into the phage, thereby preparing a cDNA library derived from the cells of the resistant mutant. The titer of the library solution was 16,600 pfu/µl.<br>
When a cDNA library was prepared using mRNA extracted from the wild type according to the above-mentioned method, approximately 38 ng of cDNA derived from the wild type was synthesized. Further, approximately 5 ng of XDNA containing the insert inserted therein was obtained for the wild type. Furthermore, the titer of the cDNA library solution derived from the wild type was 18,160 pfu/^il.<br>
(4) Screening of cDNA containing the ALS gene<br>
To form about 20,000 plaques on the plates, the library solution prepared in (3) above was diluted, and then thereon phages derived from the wild type and those derived from the resistant mutant were separately inoculated over 10 plates, respectively. Plaques were transferred to a nitrocellulose membrane (Schleicher &amp; Schnell, PROTORAN BA85, pore size 0.45µM). The nitrocellulose membrane was immersed in a denaturation solution (0,5 M"Nu@k 1.5 M NaCl), and then in a neutralization solution +—<br>
KfojOH<br>
(1.5 M NaCl, 0.5 M Tris-HCl pH 7.5, 1 mM EDTA) for approximately 20 sec. Excess water was removed using a filter paper from the nitrocellulose membrane, and then the nitrocellulose membrane was baked at 80 °C for 2 hours. In addition, the baking step was omitted when Hybond-N+ (Amersham Pharmacia Biotec) was used instead of a nitrocellulose membrane, and immobilization was performed with 0.4 M NaOH for 20 min.<br><br>
The insert DNA prepared in (1) above was labeled by two types of method, RI and non-RI, and then used as a probe DNA, Labeling with RI and hybridization were performed by the following method. First, approximately 200 to 500 ng of probe DNA was thermally denaturated, and then labeled using a BcaBEST DNA labeling kit (Takara Shuzo Co., Ltd), Upon this labeling reaction, a buffer, random primers and P-dCTP provided with the kit were added. Next, BcaBEST was added, followed by incubation at 65 °C for 30 min. Subsequently, EDTA was added to stop the reaction. The reaction solution was applied to nitrocellulose membranes, so that 8 sheets of the membrane contained approximately 100 ng of probes. Hybridization was performed at 42 °C overnight with weakly shaking. After hybridization, the membranes were washed three times with 2xSSC, 0.1 % SDS solution, followed by exposure for about 1 hour to an imaging plate of a BAS 2000 imaging analyzer (Fuji Photo Film Co., Ltd.). Following exposure, positive clones were detected using the imaging analyzer,<br>
Labeling with non-RI was performed by the following method. Following thermal denaturation of approximately 200 to 500 ng of probe DNA, DNA labeling reagent (peroxidase) and glutaraldehyde which are provided in an ECL direct DNA/RNA labeling and detection system (Amersham Pharmacia Biotech) were added, followed by incubation at 37 °C. In this case, the labeled probe DNA was applied to nitrocellulose membrane, so that 8 sheets of the membrane contained approximately 100 ng of the labeled probe DNA. Hybridization was performed at 42 °C overnight with weakly shaking. After hybridization, the membrane was washed three times with a primary washing buffer at room temperature for 10 min, and then once with 2xSSC at room temperature for 10 min. The membrane was immersed in a luminous solution provided with the ECL kit, and then exposed to an X-ray film for 30 min to 3 hours.<br>
Positive phages obtained by hybridization (primary screening) were scraped off together with top agar using a sterile toothpick, and then suspended in 200 µL of SM buffer,  thereby obtaining a phage solution.  Phage solutions of each clone were<br><br>
appropriately diluted, infected with E. coli strain Y-1088, and then inoculated over LB plates. Using these newly prepared plates, hybridization (secondary screening) was performed similarly. Positive phages were suspended in 200 \x\ of a SM buffer, thereby obtaining single phages. If no single phage was isolated by secondary screening, another dilution was performed, followed by inoculation over LB plates. Subsequently, hybridization (the third screening) was performed, so that single phages were obtained.<br>
Next, A,DNA was prepared from the single phages by the following methods. X phages collected with a bamboo brochette or a toothpick from plaques of positive clones were inoculated in 200 |il of a 2xYT medium (containing 10 mM MgCl2 and 0.2 % maltose) containing 5 ^1 of a suspension of fresh host E. coli (Y1088), The product was allowed to stand and incubated at 42 °C overnight. Then, the medium was inoculated again in 1ml of a 2xYT medium (containing 10 mM MgCb and 0,2 % maltose) containing 25 \x\ of a suspension of host E. coli (Y1088), and then shake-cultured overnight (these steps compose a pre-culturing process). The pre-cultured solution (10 to 50 \i\) was inoculated in a 12 ml of 2xYT medium containing 10 mM MgCb and 0.5 ml of E. coli Y1088 suspension. Then, incubation was performed at 42 °C overnight with relatively strong shaking, until turbidity increased after lysis. After culturing, 50 \x\ of chloroform and 1.2 ml of 5 M NaCl were added, and then incubation was performed at 42 CC for 10 min while shaking. The product was centrifuged at 27,000 xg for 10 min, and then the supernatant was newly transferred to a centrifugation tube. Five ml of 50 % PEG was added to the supernatant, and then incubated on ice for 1 hour or more. The product was centrifuged at 27,000 xg for 10 min, and then the supernatant was discarded. Next, another centrifugation was performed at 27,000 xg, and then the liquid portion was discarded. The precipitated fraction was suspended in 300 µM of a 30 mM Tris hydrochloric acid buffer (pH 7.5) containing 4 |_ig of DNase I, 20 j.tg of RNase A and 10 mM MgCl2. The suspension was transferred to a 1.5 ml tube. After incubation of the suspension at 37 °C for 30 min, 7.5 nl of 20 %SDS, 3^1 of proteinase K (10 mg/ml), and 12 \A of 0.5 M EDTA were added to the suspension, followed by further<br><br>
incubation at 55 C for 15 min. Subsequently, 150 jal of phenol was added to the product, and then stirred vigorously. Then the mixture was centrifuged at 15,000 rpm for 3 min using a TOMY Microcentrifuge MR-150 (TOMY DIGITAL BIOLOGY CO., LTD.), and an aqueous layer collected. 800 \x\ of ethyl ether (to which distilled water had been added to remove peroxide) was added to the collected aqueous layer. The mixture was stirred vigorously, and then centrifuged at 15,000 rpm for 10 sec and the ether layer was discarded. After the ether extraction step was repeated, ether remaining in the aqueous layer was removed with nitrogen gas. Thirty [i\ of 5 M NaCl and 875 j^l of ethanol were added to the aqueous layer, so that precipitated A,DNA was rapidly collected. The collected XDNA was rinsed with approximately 1ml of 70 % ethanol, and then dried under reduced pressure for approximately 1 min, thereby removing ethanol. The product was dissolved in 20 jal to 50 jal of a TE buffer (pH 8.0), thereby preparing a A-DNA solution.<br>
Subcloning and sequencing of the insert DNA in the obtained AX)NA were performed by the following method. The obtained ADNA solution (1 [il) was digested with Not I so as to excise the insert DNA. The composition of a reaction solution (for cleavage reaction) followed the Manual attached to the restriction enzyme. After reaction at 37 °C for approximately 2 hours, the insert size was confirmed by electrophoresis using 1 % agarose gel. X.DNA (10 (il to 20 |il) containing the insert DNA was digested with Not I, so as to excise the insert DNA. The insert DNA was separated using agarose gel for sampling, the corresponding band was cleaved from the gel, and then the insert DNA was purified by standard techniques. The insert DNA was mixed with a vector following BAP treatment (dephosphorylation using alkaline phosphatase derived from a shrimp) at molar ratio of 1:1, followed by ligation reaction with T4 DNA ligase at 16 °C for 2 hours or more. Here, since the insert DNA cleaved with Not I was used as material, BAP treatment was performed for vectors cleaved with Not I. Following ligation, part of the solution was mixed with competent cells (DH5ot), and then allowed to stand on ice for 30 min. Next, the mixture was subjected to heat shock<br><br>
at 42 °C for 30 sec, and then allowed to stand on ice again for 2 min. Then, SOC was added to the mixture, incubated at 37 °C for 1 hour, inoculated over a LB medium plate on which a mixture of 100 jal of 2xYT (containing 50 ng/ml ampicillin), 30 jal of 3 % X-Gal and 3 |d of 1 M IPTG had been previously added uniformly, and then cultured at 37 °C for 10 hours or more. The transformed white colonies were each inoculated on 2 ml of an LB medium containing ampicillin or a 2xYT medium, and then cultured at 37 °C overnight. From the culture solution, plasmids were prepared by standard techniques and dissolved in H2O. The DNA concentration thereof was quantified, and then the plasmids were subjected to PCR reaction for sequencing. PCR reaction and sequencing were performed by methods described above.<br>
Therefore, the ALS cDNA with an incomplete length of approximately 2.2 kb was obtained by the above experiment. Since an Sma I site was present at a position approximately 250 bp from the 5' side of the DNA, a new probe was prepared by the following method. pBluescript II SK+ containing the approximately 2.2 kbp was amplified with host E. coli JM109, and then plasmids were extracted using an automated isolation system (KURABO PI-100). The plasmid was directly digested with Sma I. The generated fragment of approximately 250 bp was separated and purified by 1 % agarose electrophoresis, and then the concentration was calculated, thereby preparing a probe. Using the probe, the library was screened again by the above method employing RI. XDNA was prepared from the thus obtained single phages, the X.DNA solution (1 jutl) was digested with Eco RI, and then size was confirmed by electrophoresis, followed by immobilization onto a nitrocellulose membrane. Following electrophoresis, the gel was immersed in 0.5 M NaOH solution containing 1.5 M NaCl, and then shaken lightly for 15 min. The gel was then washed with water, immersed in 0.5 M Tris-HCl (pH 7.5) containing 3 M NaCl, and then neutralized while shaking for approximately 15 min. Approximately 5 thick, industrial filter papers were piled up to make a base. The base was placed in 20xSSC spread over a stainless bat. Subsequently, the neutralized gel, a membrane (which had been cut into a certain size, immersed in<br><br>
distilled water and then immersed in 20xSSC for another 10 min), and two-ply filter papers were placed in order on the seat, on which a paper towel with a thickness of 3 cm to 4 cm was further placed. A glass plate and then a light weight were placed on the product, followed by blotting for approximately 5 min. After confirmation of there being no bubble entrapped between the gel and the membrane, blotting was performed for approximately 10 min. Following blotting, the membrane was subjected to UV treatment with a trans-illuminator, and then baked at 80 °C for approximately 15 min to 30 min. After baking, hybridization (hybridization buffer composition: 5xSSPE, 0.5 % SDS, 5x Denharlts, solum sperm DNA, 50 % formamide) was performed with the above 250 bp probe DNA labeled with 32P. Radiation of the hybridized band was transferred to an imaging plate, and the result was analyzed with BAS-2000. Among inserts positive in hybridization, those showing a relatively large size were prepared in large quantity, and then sub-cloned (FERM BP-7348) into pBluescript II SK+ that had been digested with Eco RI and then treated with BAP by the above method. The product was transformed into E, coli (JM 105). The obtained transformants were subjected to liquid culture, and then plasmids were prepared by standard techniques. Thus, the nucleotide sequence was determined by the above methods.<br>
As a result, the cDNA containing a full length mutant ALS gene could be obtained. The nucleotide sequence of the cDNA is shown in SEQ ID NO: 2. Further, one type of cDNA containing a full length ALS gene could be obtained from a cDNA library derived from the wild type. Figure 1 shows the result of a homology comparison between the mutant ALS gene and the wild type ALS gene.<br>
Example 4: Expression of mutant ALS protein<br>
The plasmid obtained in Example 3 (4) was digested with Eco RI, the cDNA containing the wild type ALS gene and the cDNA containing the mutant ALS gene were cut out, and then they were separately incorporated into pGEX expression vector. The cDNA containing the wild type ALS gene had a 5' non-translation region which was 47<br><br>
nucleotides longer than the initiation codon, and was incorporated into Eco RI site of pGEX-2T. Further, the cDNA containing the mutant ALS gene contained a 5? non-translation region which was 31 nucleotides longer than the initiation codon, and it was incorporated into Eco RI site of pGEX-4T-3.<br>
These vectors were separately transformed into E. coli (JM 105). The colonies obtained by transformation were subjected to liquid culture, then plasmids were extracted, and then insertion orientation of the insert DNA was confirmed by the restriction enzyme cleavage pattern and sequencing. Clones where the orientation of the insert DNA was as expected, were each selected, and then shake-cultured in 2 ml of a LB medium containing ampicillin at 27 °C. Using 1 ml of this pre-culture solution, culturing was performed in 50 ml or 250 ml of a LB medium containing ampicillin. After culturing overnight, 1 mM IPTG was added, and then expression of GST fusion protein was induced for 3 to 4 hours.<br>
Table 3 shows the results of measuring ALS activity after addition of IPTG followed by culturing for 3 to 4 hours.<br><br>
As shown in Table 3, E, coli having the wild type ALS gene and E, coli having the mutant ALS gene showed specific activity approximately 3 to 4-fold higher than that of the control group containing no plasmid. The transformant E, coli was cultured for 3 to 4 hours after addition of IPTG, and then stored at -80 °C. Using the transformant E.<br><br>
coli, GST fusion ALS protein was purified.<br>
Preparation and purification of ALS from E. coli were performed by the following method. First, a pellet of the transformant E. coli stored at -80 °C was suspended in ALS extraction buffer (potassium phosphate buffer (pH 7.5) containing 30 % glycerol and 0.5 mM MgCk). Specifically, 2.5 ml of the buffer was added to the pellet obtained from 50 ml of the culture solution. The suspension was subjected to ultrasonication (Heat Systems-Ultrasonics, Sonicator W-225R, micro chip, output control 8, interval of approximately 1 sec, twice (40 sec each)), and subjected to centrifugation at 15000 xg, 4 °C for 20 min, thereby obtaining the supernatant as a crude enzyme solution.<br>
GST fusion ALS protein (hereinafter referred to as GST-ALS) was purified from<br>
the crude enzyme solution by the following method. The crude enzyme solution (12.5<br>
ml) that had been prepared from the pellet (obtained from 250 ml of the culture<br>
solution) was applied to a glutathione sepharose 4B affinity column (Amersham<br>
Pharmacia Biotec, the bed volume, about 1ml, that had been previously equilibrated<br>
with 10 ml of lxPBS (0.14 M NaCl, 2.7 mM KC1, 10.1 mM Na2HP04, 1.8 mM KH2P04,<br>
pH 7.3)). The column was then washed with 10 ml of lxPBS, and then the adsorbed<br>
GST-ALS was eluted (4 times) with 2 ml of 10 mM glutathione solution. GST activity<br>
and ALS activity of each of the fractions were measured, and then fractions with high<br>
activity were collected. In addition, since the adsorption rate to the above column was<br>
poor when the ALS extraction buffer was used, an experiment, which extracts enzyme<br>
with lxPBS and allows it to be adsorbed to the above column, was also conducted.<br>
Measurement of ALS activity and quantification of protein were performed according to<br>
the above method. Measurement of GST activity was performed by a method using<br>
1 -chloro-2,4-dinitrobenzene	(secondary	name:	abbreviated	as<br>
2,4-dichloronitrobenzen(CDNB)) as a substrate. Three ml of reaction solution, and a reaction composition (1 mM CDNB, 1 mM glutathione (reduced), enzyme solution, 100 mM potassium phosphate buffer (pH 6,5)) were used. After addition of enzyme solution,<br><br>
a rate assay was performed at 30 °C for increased absorbance at 340 nm. CDNB was prepared as an ethanol solution having 100-fold concentration, and then added to the reaction solution.<br>
Activities (ALS activity and GST activity) that were not adsorbed to the above affinity column were greatly detected. However, most of the activity adsorbed to the affinity column was detected in fractions eluted with the first and the second glutathione elution. Those fractions from the first and the second elution were combined to prepare purified GST-ALS.<br>
GST-ALS digested overnight at 4 °C with thrombin protease (25 unit) was determined as ALS freed from GST. Table 4 shows ALS activity, protein amount and specific activity after chromatography using crude enzyme solution and the above affinity column, and after thrombin protease treatment.<br><br>
The prepared ALS enzyme contains ALS activity derived from E. coli. However as shown in Table 5, since ALS activity derived from E. coli results from ALS isozyme I, it can be almost completely inhibited by addition of 1 mM valine.<br><br><br>
Example 5: Herbicide sensitivity of mutant ALS protein<br>
Herbicide sensitivity of mutant ALS protein was examined using the crude enzyme solution obtained in Example 4. Herbicide sensitivity test was performed in the manner same as that in Example 2. In the herbicide sensitivity test, three herbicides, bispyribac-sodium, pyrithiobac-sodium, and pyriminobac were used as PC herbicides; four drugs, chlorsulfuron, bensulfuron-methyl, pyrazosulfuron-ethyl and imazosulfuron were used as sulfonylurea herbicides; and two drugs, imazaquin, and imazapyr were used as imidazolinon herbicides.<br>
Solutions containing these herbicides at certain concentrations (bispyribac-sodium and pyrithiobac-sodium were aqueous solutions, others were acetone solutions) were added to reaction solutions before addition of the mutant ALS protein. The final concentration of acetone was 1 %. In addition, since an ALS protein from the rice cDNA expressed within E. coli is insusceptible to valine (Kil et al., J. Biochem. MoLBioL 31 287-295,1998), a herbicide sensitivity test was performed under conditions in which ALS activity derived from E. coli was inhibited with valine.<br>
Figure 12 shows the resulting sensitivity when bispyribac-sodium was used. Figure 13 shows the resulting sensitivity when chlorsulfuron was used. As shown in Fig. 12, wild type ALS activity was inhibited by bispyribac-sodium, while mutant ALS activity was not inhibited at all. For chlorsulfuron as shown in Fig. 13, the mutant ALS protein maintained its activity at a higher level to a certain concentration compared to<br><br>
the that of the wild type ALS protein, however when compared to the case of bispyribac-sodium, the resistance of the mutant ALS protein was moderate.<br>
Figure 14 shows the resulting sensitivity when pyrithiobac-sodium was used as a herbicide; Figure 15 shows the resulting sensitivity when pyriminobac was used as a herbicide; Figure 16 shows the resulting sensitivity when bensulfuron-methyl was used as a herbicide; Figure 17 shows the resulting sensitivity when pyrazosulfuron-ethyl was used as a herbicide; Figure 18 shows the resulting sensitivity when imazosulfuron was used as a herbicide; Figure 19 shows the resulting sensitivity when imazaquin was used as a herbicide; and Figure 20 shows the resulting sensitivity when imazapyr was used as a herbicide. In line graphs of Figs. 12 to 20, the lines joining black triangles indicate the crude enzyme solution containing the wild type ALS protein, and those joining black squares indicate the crude enzyme solution containing the mutant ALS protein.<br>
As shown in the Figs, 14 to 20, the mutant ALS protein has good resistance to each of the herbicides, superior to that of the wild type ALS protein. Particularly, the mutant ALS protein has good resistance to PC herbicides. Further comparison of Figs 12, 14 and 15 indicates that the mutant ALS protein has the best resistance to bispyribac-sodium among PC herbicides.<br>
Since these results can be thought to correlate with the result of Example 2, it was suggested that the factor which improved the resistance to PC-based herbicides in Example 2 is the mutant ALS gene.<br>
Furthermore, a herbicide sensitivity test was performed using the purified GST-ALS and the freed ALS prepared in Example 4. Figure 21 shows a result indicating resistance of the wild type ALS protein to bispyribac-sodium. Figure 22 shows a result indicating resistance of the mutant ALS protein to bispyribac-sodium. In addition, Figure  23   shows  a  result  indicating  resistance  of the  mutant  ALS  protein  to<br><br>
chlorsulfuron. Figure 24 shows a result indicating resistance of the mutant ALS protein to imazaquin. Herbicide sensitivity tests whose results were shown in Figs 21 to 24 were performed without adding valine. Further in a line graph of Fig, 21, the line joining black triangles indicate the crude enzyme solution containing the wild type ALS protein; the line joining black squares indicates the wild type GST-ALS; and the line joining black circles indicates the freed ALS protein. In line graphs of Figs. 22 to 24, the lines joining black triangles indicate the rough enzyme solution containing the mutant ALS protein, and the lines joining black squares indicate the mutant GST-ALS.<br>
As shown in Figs 21 to 24, the purified GST-ALS also showed herbicide sensitivity similar to that in a case when the crude enzyme solution was used.<br>
Example 6: Relation between mutated portion in mutant ALS gene and drug sensitivity<br>
The mutant portions in the mutant ALS gene determined in Example 3(4) were W548L mutation in which tryptophan (W) at position 548 of the wild type had been mutated to lysine (L) and S627I mutation in which serine (S) at position 627 of the wild type had been mutated to isoleucine (I) as shown in Fig. 1. To study the effect of these mutations on herbicide sensitivity, a mutant ALS gene containing only either one of the W548L or S627I mutations was prepared, and herbicide sensitivity of an ALS protein having either one of the mutations was examined.<br>
(1) Preparation of mutant gene<br>
For example, a mutant ALS gene (hereinafter referred to as "W548L mutant ALS gene") having W548L mutation only was prepared as shown in Figs. 25 and 26. First, PCR was performed using a sense primer named ALS-Rsp6 (5'-CATCACCAACCACCTCTT-3*: SEQ ID NO: 3) and an antisense primer named ALS-RspF (5'-ACACGGACTGCAGGAATA-3(: SEQ ID NO: 4)? and using pBluescript II SK+ (FERM BP-7348) retaining the double-point mutant ALS gene as a template, thereby amplifying a DNA fragment having W548L mutation but having no S627I<br><br>
mutation (Fig. 25). Meanwhile, a DNA fragment containing a region encoding serine at position 627 was amplified using a sense primer named ALS-RspE (5'-TTACAAGGCGAATAGGGC-3'; SEQ ID NO: 5) and a M13R antisense primer (5*-GGAAACAGCTATGACCATG-3': SEQ ID NO: 6) and using pBluescript II SK+ retaining the wild type ALS gene as a template (Fig. 25).<br>
Next, a large DNA fragment was obtained from the two DNA fragments by linking the two DNA fragments using the SPR method (Gene preparation method: Self Polymerase Reaction in which two single-stranded DNAs are bound at the end portions, where sequences correspond to one another, and the DNAs are replicated by DNA polymerase using the DNAs as templates for each other to form a double-stranded DNA) (Fig. 26). The obtained DNA fragment was digested with Ace I and Eco RI, thereby obtaining an Ace l-Eco RI fragment. After pGEX-2T plasmid (pGEX-2T-wALS) containing a wild type ALS gene incorporated therein was digested at Ace I site, partial digestion was performed with Eco RI at 37 °C for 1 min, thereby obtaining a partial fragment of pGEX-2T plasmid (Fig. 26). Then, ligation of Ace l-Eco RI fragment and the partial fragment of pGEX-2T plasmid resulted in a plasmid having W548L mutant ALS gene.<br>
In addition, a mutant ALS gene having only S627I mutation (hereinafter referred to as "S627I mutant ALS gene") could be obtained according to the method by which the above W548L mutant ALS gene was prepared. That is, S627I mutant ALS gene could be obtained in the same manner as in the method employed for the above W548L mutant ALS gene except that ALS-RspE and M13R were used as primers in PCR performed using pBluescript II SK+ (FERM BP-7348) as a template, and ALS-Rsp6 and ALS-RspF were used as primers in PCR performed using pBluescript II SK+ retaining the wild type ALS gene as a template.<br>
A plasmid having W548L mutant ALS gene and a plasmid having S627I mutant<br><br>
ALS gene were separately transformed into E. coli strain JM109, Full sequencing of the ALS gene was perfomied for the obtained E. coli colonies, so that the single-point mutation of W548L mutation and that of S627I mutation could be confirmed.<br>
(2) Herbicide resistance of respective mutant proteins<br>
A plasmid having W548L mutant ALS gene or a plasmid having S627I mutant ALS gene was transformed into E. coli strain JM 109. The colonies obtained from transformation were subjected to liquid culture, expression of GST induction protein was induced in the same manner as in Examples 4 and 5, a crude enzyme solution was prepared, and then herbicide sensitivity of ALS was examined in the presence of 1 mM valine.<br>
Figure 27 shows the result of the use of bispyribac-sodium. Figures 28 and 29 show the result of the use of chlorsulfuron and imazaquin. As shown in Fig. 27 for bispyribac-sodium, each single-point mutation of W548L mutation and of S627I mutation imparted resistance against bispyribac-sodium, and the degree of resistance was higher in the single-point mutation of only W548L than that in the single-point mutation of only S627L.<br>
However, the degrees of resistance among these single-point mutations were all shown to be several magnitudes lower than those of the double-point mutations shown in Example 5. In other words, coexistence of W548L and S627L mutations resulted in significantly strong resistance to bispyribac-sodium which could not be predicted from the case of either W548L or S627L mutation alone.<br>
When chlorsulfuron was used (Fig. 28), W548L mutation alone imparted resistance to chlorsulfuron, but S627I mutation alone imparted no obvious resistance. In addition, a comparison of the degree of resistance to chlorsulfuron between W548L mutation alone, and W548L mutation and S627I mutation together revealed that they<br><br>
shared the same degree of resistance. Moreover, when imazaquin was used (Fig. 29), results were similar to those for bispyribac-sodium. However in the concentration range employed in this example, increased resistance, being a multiplier effect due to co-existence of two mutations, could not be confirmed.<br>
As described above, a novel S627I mutation which was found this time was shown to drastically enhance bispyribac-sodium resistance due to W548L mutation. In conclusion, ALS gene having W548L and S627I mutations is a gene imparting high resistance specific to bispyribac-sodium.<br>
Example 7: Production of transformant plant<br>
A rice plant (Nippon-bare) was transformed with the mutant ALS gene determined in Example 3(4), and then bispyribac-sodium resistance of the transformant rice plant was examined.<br>
(1) Construction of binary vector with the mutant ALS gene incorporated therein<br>
The mutant ALS gene obtained in Example 3(4) was incorporated into a binary vector, pMLH7133 (Mitsuhara et al., Plant Cell Physiol 37 49-59, 1996), which had been developed for transformation of a rice plant.<br>
First as shown in Fig. 30, the mutant ALS gene incorporated in pBluescript II SK+ (FERM BP-7348) was cleaved out using the restriction enzyme cleavage site of a multicloning site and the cleavage site of an adaptor (digestion with Sac I and partial digestion with Bam HI). Then, pMLH7133 was digested with Sac I and Bam HI. GUS region in pMLH7133 was excised and pMLH7133 was linearized. A DNA fragment containing mutant ALS gene and pMLH7133 in the shape of a straight chain were ligated, thereby obtaining pMLH7133-ALS.<br>
The obtained pMLH7133-ALS was transformed into E, coli strain JM105. The<br><br>
resulting single colony was subjected to liquid culture, and then a plasmid was prepared. The presence or absence of the insert was confirmed by digesting the prepared plasmid using Sac I and Bam HI (Fig. 31), so that clones having mutant ALS genes were selected, PCR was performed using plasmids shown to contain the insert as templates and using two sets of primers which allow specific amplification of at least part of the mutant ALS gene. The primer sets used herein were a set of an ALS-Rspl sense primer (5'-GCTCTGCTACAACAGAGCACA-3':SEQ ID NO: 7) and a 4-83-3 antisense primer (5'-GCTTTGCCAACATACAG-3':SEQ ID NO: 8); and a set of a 3-1-4 sense primer (5'-AGGTGTCACAGTTGTTG-3':SEQ ID NO: 9) and a 3-1-1 antisense primer (5'-GCATCTTCTTGAATGGCG-3': SEQ ID NO: 10). Thus, specific fragment bands were detected (Fig. 32), suggesting that the mutant ALS gene obtained in Example 3(4) was incorporated into the pMLH7133 vector.<br>
Further, whether or not the mutant ALS gene had been incorporated normally was confirmed by PCR to learn the insertion orientation of the insert and to determine the sequence. The insertion orientation of the insert was confirmed by PCR as described below. pMLH7133-ALS was completely digested with Eco RI, thereby removing TNOS portion and linearizing. PCR was performed using the product as a template and using the above set of ALS-Resl and 4-83-3 and the set of 3-1-4 and 3-1-1. As a result (Fig. 33), bands for the PCR products were confirmed at positions as predicted, suggesting that the mutant ALS gene was inserted in forward orientation. If the mutant ALS gene is inserted in reverse orientation, PCR product would not be detected since the mutant ALS gene portion is cut out from pMLH7133-ALS by Eco RI digestion and no DNA fragment is amplified.<br>
Meanwhile, determination of the sequence was performed using the prepared pMLH7133-ALS as a template for the linkage region of the 5' side of a mutant ALS gene and pMLH7133. In addition, since pMLH7133-ALS leads to low number of copies in E. coli, preparation was performed by the alkali SDS method from a culture<br><br>
solution in a volume 20 fold greater than the standard volume (corresponding to 2 ml of culture solution x 20). Therefore, mutant ALS genes were shown that they have been inserted in forward orientation.<br>
(2)	Introduction of binary vector into Agrobacterium<br>
The binary vector (pMLH7133-ALS) obtained in (1) above was introduced into Agrobacteria as follows. The competent cells of Agrobacterium tumefaciens EHA 105 stored at -80 °C were thawed in ice and used. pMLH7133-ALS was added to a solution containing the competent cells of Agrobacterium tumefaciens EHA 105, and allowed to stand in ice for 15 min, followed by heat shock at 37 °C for 5 min. Then, the mixture was allowed to stand in ice for 2 min5 and 1ml of a SOC liquid medium was added thereto, followed by gentle shaking at 28 °C for 2 to 4 hours. Next, centrifugation was performed, and most of the supernatant was discarded. The precipitated cells were suspended in the remaining supernatant. The suspension was coated over a LB plate containing 50 ppm each of kanamycin and hygromycin, followed by incubation at 28 °C for 2 to 3 days, thereby obtaining single colonies.<br>
(3)	Transformation of rice plant with pMLH 7133-ALS-introduced Agrobacterium<br>
A 1/2 spoonful of the Agrobacteria obtained in (2) was taken with a sterilized micro spatula, and then suspended well in 30 ml of an A AM medium (Falcon tube) supplemented with acetosyringone at a concentration of 10 mg/1 as shown in Table 6.<br><br><br>
(Numerical figures are shown with mg/1)<br>
Pipetting  was  performed  well  with  a  Pasteur  pipette  so  that  no  mass  of Agrobacteria remained. The obtained suspension was put into a 9 cm Schale.<br>
Calli which had been induced from seeds (Nippon-bare) and pre-cultured in a N6D medium shown in Table 7 were put into a stainless mesh (or a tea strainer). Then the mesh containing the calli was dipped in the suspension of Agrobacteria for 1.5 to 2 min. At this time, the mesh was dipped such that the calli were entirely immersed in the bacterial solution, followed by gentle agitation with a spatula. Subsequently, the stainless mesh was placed on a sterilized filter paper to remove excessive bacterial solution. The thus treated calli were placed on a 2N6AS solid medium (10 mg/1 acetosyringone, 16 calli per petri dish) as shown in Table 8, sealed with a surgical tape,<br><br>
and cultured under dark conditions at 28 °C for 2 to 3 days, until the calli were covered with a thin layer of the cells (due to the growth of the cells).<br><br><br><br>
After the calli were coated with a thin layer of the cells as a result of cocultivation for 3 days, the calli were washed to remove the Agrobacteria by the following method. The calli treated as described above were put into a stainless mesh (or a tea strainer), and the mesh containing the calli was dipped in a N6D liquid medium (containing 500 mg/1 carbenicillin). Exchanging of the petri dish was continued until the medium was no longer cloudy. Agrobacteria attached to the calli were washed away by this treatment. Then, the mesh was placed on a sterilized filter paper to remove excessive moisture. The calli were placed on a N6D solid medium (containing 500 mg/1 carbenicillin and 50 mg/1 hygromycin), and then cultured under light conditions at 28 °C for 2 to 3 weeks. In addition, when Agrobacteria proliferated again after the procedure, washing was performed in the same manner and then the calli were placed on a new medium.<br><br>
After selection by culturing under light conditions at 28 °C for 2 to 3 weeks, the calli were transplanted, 9 calli per petri dish, on redifferentiation media (containing 250 mg/1 carbenicillin and 50 mg/1 hygroniycin) and then cultured under light conditions at 28 °C for 2 to 3 weeks. When portions of leaves, stems and roots differentiated respectively 1cm or more, the calli were transplanted, 2 to 3 calli per petri dish onto hormone free media (containing 50 mg/1 hygromycin). Subsequently when the obtained plant bodies proliferated to spread over the petri dish enough, the plants were potted into culture soil (bon sol) and allowed to acclimate. Upon transplantation, media attached to roots were completely washed away in water. Figure 34 shows the flow of the above procedures.<br>
Simultaneously with the above described potting, part of the calli which had not redifferentiated (9 calli) (Fig. 35) was sub-cultured on solid media for production of a bispyribac-sodium resistant callus used in Example 1. The solid media contained 10 |iM bispyribac-sodium. Thus, 6 out of 9 calli grew normally. One of the 6 calli that grew normally and showed bispyribac-sodium resistance was subjected to liquid culture, and then the drug sensitivity to bispyribac-sodium was examined in detail. Therefore, as shown in Fig, 36, the callus showed bispyribac-sodium resistance, almost identical to that of bispyribac-sodium resistant Sr line, from which the double-point mutant gene had been derived. Further, as shown in Fig. 37, the callus of the wild type rice plant (Nippon-bare) showed bispyribac-sodium sensitivity. Accordingly, 6 calli grown on a solid medium containing 10 |iM bispyribac-sodium were all shown to have strong bispyribac-sodium resistance.<br>
Genomic DNA was prepared from these bispyribac-sodium resistant calli using a DNeasy Plant Kit (QIAGEN). Then, PCR was performed using the genomic DNA as a template and using a set of a sense primer (3-1-4: SEQ ID NO: 9) and an antisense primer  (4-83-3:   SEQ  ID  NO:   8)  that  flank  the  double-point  mutation  portion<br><br>
therebetween. Thus, as shown in Fig. 38, a DNA fragment was amplified as expected. The amplified DNA fragment was purified, and then the nucleotide sequence was examined (a primer (3-1-4: SEQ ID NO: 9) was used to read the sense strand; and a primer (ALS-Rsp2 : 5'-AGTCCTGCCATCACCATCCAG-3': SEQ ID NO: 11) was used to read the antisense strand).<br>
Figure 39A and B show the results of analysis of the nucleotide sequence in the vicinity of the nucleotides encoding an amino acid at position 548 of the ALS protein. Figure 39C and D show the results of analysis of the nucleotide sequence in the vicinity of the nucleotides encoding an amino acid at position 627 of the ALS protein. As shown in Fig. 39 A to D, heterologous sequences of wild type and mutant were found for the double-point mutation of these lines. Hence, the mutant ALS gene was shown to be incorporated in the genome. In addition, 3 of 9 calli did not grow well in the presence of 10 JJM bispyribac-sodium. However, this was thought to be caused by low expression amount of the double-point mutant ALS gene introduced.<br>
Of 80 hygromycin resistant transformant rice plants potted, those which grew normally at the leaf stage 5 were 27 plants to which the mutant ALS gene had been introduced and 46 plants to which the wild type ALS gene had been introduced. Plants containing the wild type ALS gene introduced therein tended to grow healthier than those containing the mutant ALS gene therein. At this time, 4 plants containing the mutant ALS gene introduced therein and 5 plants containing the wild type ALS gene introduced therein were appropriately selected, and 1kg of a.i./ha bispyribac-sodium salt supplemented with 0.2 % surfactant K was sprayed over the leaves and stems. The results of a growth survey conducted about 40 days after spraying are shown in Fig. 40. In Fig. 40, the plant length A was about 90 cm.<br>
As shown in Fig. 40, an individual plant containing the mutant ALS gene introduced therein grew almost normally (A), and the plant was shown to have<br><br>
bispyribac-sodium resistance. Genomic DNA was prepared from the bispyribac-sodium resistant transformant rice plant using DNeasy Plant Kit (QIAGEN). Then PCR was performed using the genomic DNA as a template and using a set of a sense primer (3-1-4: SEQ ID NO: 9) and an antisense primer (4-83-3: SEQ ID NO: 8) that flank the double-point mutation portion therebetween. The nucleotide sequence of the resulting DNA fragment was examined in the same manner as in the above callus. Hence, the presence of double-point mutation in the transformant rice plant was confirmed.<br>
All publications, patents and patent applications cited herein are incorporated by reference in their entirety.<br>
Effect of the invention<br>
As described in detail above, the present invention can provide ALS gene encoding ALS enzyme which has good resistance to all ALS-inhibiting herbicides and shows extremely high level of resistance to PC herbicides.<br><br><br><br><br>
What is claimed is:<br>
1.	A gene which encodes the following protein (a) or (b):<br>
(a)	a protein which comprises an amino acid sequence of SEQ ID NO: 1;<br>
(b)	a protein which comprises an amino acid sequence derived from the amino acid sequence of SEQ ID NO: 1 by substitution, deletion or addition of at least one or more amino acids, has resistance to a pyrimidinyl carboxy herbicide and has acetolactate synthase activity.<br><br>
2.	An acetolactate synthase protein which is encoded by the gene of claim 1.    <br>
3.	A recombinant vector which has the gene of claim 1.<br>
A, A transformant which has the recombinant vector of claim 3.<br>
5.	A plant which has the gene of claim 1 and has resistance to a pyrimidinyl carboxy herbicide.<br>
6.	A method for rearing the plant of claim 5 which comprises rearing the plant in the presence of a pyrimidinyl carboxy herbicide.<br>
7.	A method for selecting a transformant cell having the gene of claim 1 using the gene as a selection marker.<br><br>
Claim 8<br>
A gene comprising a nucleotide sequence encoding an acetolactate synthase protein which is substituted serine corresponding to serine at position of 627 of a wild type rice acetolactate synthase protein with isoleucine and is resistant to a pyrimidinyl carboxy herbicide.<br>
Claim 9<br>
A gene comprising a nucleotide sequence encoding an acetolactate synthase protein which is substituted tryptophan corresponding to tryptophan at position of 548 of a wild type rice acetolactate synthase protein with leucine and is resistant to a pyrimidinyl carboxy herbicide.<br><br>
9. A gene  substantially as herein described with reference to the<br>
accompanying drawings. 10.A method for selecting a transformant cell substantially as herein<br>
described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWNsYWltcyBmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKWZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWRlc2NyaXB0aW9uKGNvbXBsZXRlKWdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWZvcm0gMTkucGRm" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLW90aGVyIGRvY3VtZW50cy5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyOC1jaGVucC0yMDAzLXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1028-chenp-2003-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="211786-a-power-formulation-comprising-powdery-medicament-and-erythritol-and-a-method-for-producing-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="211788-vehicle-body-cover-structure-in-motorcycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>211787</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1028/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2007</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Dec-2007</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Nov-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Jun-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>M/S. KUMIAI CHEMICAL INDUSTRY CO., LTD</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>4-26, Ikeno-hata 1-chome Taito-ku, Tokyo 110-8782,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHIMIZU, Tsutomu</td>
											<td>114, Hirao, Kikugawa-cho Ogasa-gun, Shizuoka 439-0031,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2001/010014</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-16</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2000-362630</td>
									<td>2000-11-29</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/211787-gene-encoding-acetolactate-synthase by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:00:34 GMT -->
</html>
