<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/211957-a-process-for-preparing-a-protein-having-an-enzymatic-activity-for-converting-carotene-into-zeaxanthin-or-canthaxanthin-into-astaxanthin by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:53:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 211957:A PROCESS FOR PREPARING A PROTEIN HAVING AN ENZYMATIC ACTIVITY FOR CONVERTING β-CAROTENE INTO ZEAXANTHIN OR CANTHAXANTHIN INTO ASTAXANTHIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PREPARING A PROTEIN HAVING AN ENZYMATIC ACTIVITY FOR CONVERTING β-CAROTENE INTO ZEAXANTHIN OR CANTHAXANTHIN INTO ASTAXANTHIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a process for preparing a protein which has an enzymatic activity for converting ~-carotene into zeaxanthin or canthaxanthin into astaxanthin and comprising the amino acid sequence SEQ ill NO.2 or a sequence derived from this sequence by substitution, insertion or deletion of amino acids as described in the sequence listing and having a homology of at least 50 % at the amino acid level with the sequence SEQ ill NO.2, by gene expression of the appropriate nucleic acid which encodes the said protein, in a known manner and to processes for preparing xanthophyll derivatives.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Carotene hydroxylase and process for preparing xanthophyll derivatives<br>
The present invention relates to proteins which have an enzymatic activity for converting β-carotene into zeaxanthin or canthaxanthin into astaxanthin, to nucleic acids which encode these proteins, to nucleic acid constructs comprising these nucleic acids, to genetically manipulated organisms where the genetic manipulation causes or increases the gene expression of this nucleic acid by comparison with a wild-type, and to processes for preparing xanthophyll derivatives.<br>
Xanthophylls are oxygen-containing carotenoids of animal, plant or microbial origin. Xanthophylls such as lutein, zeaxanthin or astaxanthin are important additives in the human and livestock diet as pigmenting substances and precursors of vitamin A derivatives. In addition, xanthophylls have a health-promoting action such as enhancing the immune response and, by reason of their antioxidant properties, a cancer-preventing action, which makes their use as nutraceuticals of interest. An economic process for preparing xanthophylls and foodstuffs with an increased xanthophyll content are therefore of great importance. Particularly economic processes for preparing xanthophylls are biotechnological processes which make use of proteins and biosynthesis genes of xanthophyll biosynthesis from xanthophyll-producing organisms.<br>
Prokaryotic β-carotene hydroxylases which catalyze the enzymatic conversion of β-carotene into zeaxanthin via β-cryptoxanthin, and the genes which encode these proteins are known from the bacteria Erwinia  uredovora   (Misawa et al., J. of Bacteriology 1990, 6704-6712; EP 393690 Bl), Erwinia herbicola   (WO 9113078), Agrobacterium aurantiacuin (Misawa et al., J. of Bacteriology 1995, 6575-6584; EP 735 137 Al), Alcaligenes  sp.   PC-l   (EP 735 137 Al), Fiavobacteriu/n sp.   strain R1534   (Pasamontes et al.. Gene<br>
1997,	185, 35-41; EP 747483 A2) and from the Cyanobacterium Synechocystis sp. PCC6803 (Masamoto et al.. Plant Cell Physiol.<br>
1998,	39(5), 560-564).<br>
It is also known that the prokaryotic β-carotene hydroxylases from Agrobacterium aurantiacum,  Alcaligenes and Erwinia  uredovora  are additionally able to covert canthaxanthin via adonirubin in astaxanthin (Misawa et al., J. of Bacteriology 1995, 6575-6584; Eraser et al., J. Biol. Chem. 1997, 272, 6128-6135).<br><br>
From eukaryotic sources, three plant β-carotene hydroxylases are known to catalyze the enzymatic conversion of β-carotene into zeaxanthin via β-cryptoxanthin. The corresponding cDNAs have been isolated from Arabidopsis thaliana   (Cunningham et al, J. Biol. Chem. 1996, 271, 24349-24352, WO 9736998), and from Capsicum an-nuum  L. (Bouvier et al., Biochimica et Biophysica Acta 1998, 1391, 320-328).<br>
Genes of eukaryotic origin have the advantage over prokaryotic genes that they are expressed better in higher transgenic organisms such as plants. Nevertheless, there is still a need to improve and increase the xanthophyll productivity for an economic process for preparing xanthophyll derivatives or foodstuffs with an increased xanthophyll content by incorporating eukaryotic nucleic acids into organisms.<br>
In addition, the appropriate eukaryotic β-carotene hydroxylases in the prior art have the disadvantage that they have only a narrow substrate range so that there is a build up of metabolic products which cannot be converted by the hydroxylases and may exert an inhibiting effect on the hydroxylases.<br>
It is an object of the present invention to remedy the described deficiencies of the prior art and to provide a eukaryotic β-carotene hydroxylase with improved properties.<br>
We have found that this object is achieved by a protein which has an enzymatic activity for converting β-carotene into zeaxanthin or canthaxanthin into astaxanthin, comprising the amino acid sequence SEQ ID NO. 2 or a sequence derived from this sequence by substitution, insertion or deletion of amino acids and having a homology of at least 50% at the amino acid level with the sequence SEQ ID NO.2.<br>
Carotene hydroxylases hereinafter mean the proteins according to the invention, i.e. proteins which have an enzymatic activity for converting β-carotene into zeaxanthin or canthaxanthin into astaxanthin, comprising the cunino acid sequence SEQ ID NO. 2 or a sequence derived from this sequence by substitution, insertion or deletion of amino acids and having a homology of at least 50% at the amino acid level with the sequence SEQ ID NO.2.<br>
The cimino acid sequence depicted in SEQ ID NO. 2 is derived from translation of the cDNA sequence depicted in SEQ ID NO. 1.<br>
The proteins according to the invention are able to catalyze the conversion of a β-ionone structural element into a 3-hydroxy-β-ionone structural element, such as the conversion of β-carotene into zeaxanthin, β-carotene into β-cryptoxanthin,<br><br>
β-cryptoxanthin into zeaxanthin, echinenone into 3'-hydroxyechinenone, 3-hydroxyecvhinenone into adonixanthin (4-ketozeaxanthin), a-carotene into a-cryptoxanthin or other chemical compounds which have up to 40 C atoms and contain a β-ionone ring into the corresponding 3-hydroxy-β-ionone compounds or the conversion of a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element, such as the conversion of canthaxanthin into astaxanthin, canthaxanthin into phoenicoxanthin (adonirubin), phoenicoxanthin (adonirubin) into astaxanthin, echinenone into 3-hydroxyechinenone, 3'-hydroxyechinenone into adonixanthin (4-ket02eaxanthin) or other chemical compounds which have up to 40 C atoms and contain a 4-keto-β-ionone ring into the corresponding 3-hydroxy-4-keto-β-ionone compounds.<br>
For illustration of the xanthophylls derived from β-carotene, reference is made to the biosynthesis scheme in Misawa et al., J. Biotechnol. 1998, 59, page 174 (top). The biosynthesis of lutein takes place starting from a-carotene via a-cryptoxanthin. The proteins which comprise a sequence which is derived from amino acid sequence SEQ ID NO. 2 by substitution, insertion or deletion of amino acids and which have an homology of at least 50% at the amino acid level witn the sequence SEQ ID NO.2 have an enzymatic activity for the conversion of β-carotene into zeaxanthin or canthaxanthin into astaxanthin, preferably in an activity comparable to that of the protein comprising the sequence SEQ ID NO. 2.<br>
Substitution means replacement of one or more amino acids by one or more amino acids. The replacements are preferably those called conservative, in which the replaced amino acid has a similar property to the original amino acid, for exctmple replacement of Glu by Asp, Gin via Asn, Val by lie. Leu by lie, Ser by Thr.<br>
Deletion is the replacement of an amino acid by a direct linkage. Preferred positions for deletions are the termini of the polypeptide and the linkages between the individual protein domains.<br>
Insertions are introductions of amino acids into the polypeptide chain, there formally being replacement of a direct linkage by one or more amino acids.<br>
The homology between two proteins means identity of the amino acids over the entire length of each protein, which is calculated by comparison with the aid of the computer program GAP (UWGCG, University of Wisconsin, Genetic Computer Group, program<br><br>
algorithm of Needleman and Wunsch, J. Mol. Biol. 1970, 48, 443-453), setting the following parameters:<br>
Gap Weight:	12<br>
Length Weight:	4<br>
Average Match:	2.912<br>
Average Mismatch:	-2.003<br>
A protein which has a homology of at least 50% at the amino acid level with the sequence of SEQ ID NO.2 accordingly means a protein which, in comparison of its sequence with the sequence SEQ ID NO,2 using the above program algorithm with the above set of parameters, has an identity of at least 50%, preferably 60%, particularly preferably 70%.<br>
The carotene hydroxylase according to the invention has a homology with the known prokaryotic β-carotene hydroxylases of 29.9% (Flavobacterium), 36.8% (Erwinia uredovora), 38.5% (Erwinia herbicola), 35,0% (Alcaligenes) and 35.6% (Agrobacterium aurantiacum), and a homology of 41.2% with the known eukaryotic β-carotene hydroxylase from Arabidopsis thaliana.<br>
A preferred protein has an enzymatic activity for the conversion of β-carotene into zeaxanthin and an enzymatic activity for the conversion of canthaxanthin into astaxanthin and comprises the cimino acid sequence SEQ ID NO. 2 or a protein which is derived from this sequence by substitution, insertion or deletion of amino acids and which has a homology of at least 50% at the amino acid level with the sequence of SEQ ID NO.2.<br>
These preferred proteins are able to catalyze the conversion of a β-ionone structural element into a 3-hydroxy-β-ionone structural element, such as the conversion of β-carotene into zeaxanthin, β-carotene into β-cryptoxanthin, β-cryptoxanthin into zeaxanthin, echinenone into 3'-hydroxyechinenone, 3-hydroxyechinenone into adonixanthin (4-ketozeaxanthin), a-carotene into a-cryptoxanthin or other chemical compounds which have up to 40 C atoms and contain a β-ionone ring into the corresponding 3-hydroxy-β-ionone compounds and the conversion of a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element, such as the conversion of canthaxanthin into astaxanthin, canthaxanthin into phoenicoxanthin (adonirubin), phoenicoxanthin (adonirubin) into astaxanthin, echinenone into 3-hydroxyechinenone, 3'-hydroxyechinenone into adonixanthin (4-ketozeaxanthin) or other chemical compounds which have up to<br><br>
40 C atoms and contain a 4-keto-β-ionone ring into the corresponding 3-hydroxy-4-keto-β-ionone compounds.<br>
A particularly preferred protein is the eucaryotic carotene hydroxylase from the green alga Haematococcus pluvialis  Flotow NIES-144 having the sequence SEQ ID NO, 2. This particularly preferred protein is able to catalyze the conversion of a β-iono.ie structural element into a 3-hydroxy-β-ionone structural element, such as the conversion of (3-carotene into zeaxanthin, β-carotene into β-cryptoxanthin, β-cryptoxanthin into zeaxanthin, echinenone into 3'-hydroxyechinenone, 3-hydroxyechinenone into adonixanthin (4-ketozeaxanthin), α-carotene into a-cryptoxanthin or other chemical compounds which have up to 40 C atoms and contain a β-ionone ring into the corresponding 3-hydroxy-β-ionone compounds and the conversion of a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element, such as the conversion of canthaxanthin into astaxanthin, canthaxanthin into phoenicoxanthin (adonirubin), phoenicoxanthin (adonirubin) into astaxanthin, echinenone into 3-hydroxyechinenone, 3'-hydroxyechinenone into adonixanthin (4-ketozeaxanthin) or other chemical compounds which have up to 40 C atoms and contain a 4-keto-β-ionone ring into the corresponding 3-hydroxy-4-keto-β-ionone compounds-<br>
The carotene hydroxylases can be prepared, as described hereinafter, by gene expression of the appropriate nucleic acids which encode these proteins from natural or genetically manipulated organisms.<br>
The invention further relates to nucleic acids, referred to as carotene hydroxylase genes hereinafter, which encode the proteins according to the invention described above. Suitable nucleic acid sequences can be obtained by back-translation of the polypeptide sequence in accordance with the genetic code- The codons preferably used for this purpose are those frequently used in accordance with the organism-specific codon  usage. The codon usage  can easily be found by means of computer analyses of other, known genes in the relevant organism.<br>
If, for example, the protein is to be expressed in a plant, it is often advantageous to use the codon usage of the plant in the back-translation.<br>
A preferred nucleic acid has the sequence SEQ ID NO. 1- This nucleic acid is a eucaryotic cDNA from the green alga Haematococcus pluvialis  Flotow NIES-144, which encodes the carotene hydroxylase of sequence SEQ ID NO. 2. Since the reading frame of the cDNA is open toward the 5' end, SEQ ID NO. 1<br><br>
possibly does not represent the complete sequence of the cDNA. Expression of this cDNA leads to a functional protein. Any partial sequence missing out the 5' end can be made up in a manner known per se by analyzing overlapping cDNA fragments of the cDNA library from Hae/natococcus pluvialis  Flotow NIES-144.<br>
It is known that the green alga Haematococcus pluvialis produces large cunounts of astaxanthin under unfavorable environmental conditions such as a phosphate or nitrogen deficit or where the light intensity is high;, to protect from photooxidative stress (Kobayashi et al., Appl. Environ. Microbiol. 1993, 59, 867-873; Boussiba et al.. Methods Enzymol 1992, 213, 386-391). This process is normally accompanied by a morphological change in which the vegetative cells of the green algae develop into cyst cells.<br>
Addition of sodium acetate and FeSO4 and an increase in the light intensity induced astaxanthinbiosynthesis and cyst cell formation in a suspension culture of Haematococcus pLuvialis  Flotow NIES-144, The RNA was isolated from Haematococcus pluvialis  in this stage to construct a cDNA library. The cDNA having the sequence SEQ ID NO 1 was isolated from this cDNA library.<br>
All the aforementioned carotene hydroxylase genes can be prepared in a manner known per se by chemical synthesis from the nucleotide building blocks such as, for example, by fragment condensation of individual overlapping, complementary nucleic acid building blocks of the double helix. Chemical synthesis of oligonucleotides is possible, for example, in a known manner by the phosphoramidite method (Voet, Voet, 2nd edition, Wiley Press New York, pages 896-897). Addition of synthetic oligonucleotides and filling in gaps using the Klenow fragment of DNA polymerase and ligation reactions, and general cloning methods, are described in Sambrook et al. (1989), Molecular cloning: A laboratory manual. Cold Spring Harbor Laboratory Press. The invention further relates to nucleic acid constructs comprising one of the carotene hydroxylase genes according to the invention described above, which are linked to one or more regulatory signals to increased gene expression. These regulatory sequences are, for example, sequences to which inducers or repressors bind and thus regulate the expression of the nucleic acid. In addition to these new regulatory sequences or in place of these sequences it is possible for the natural regulation of these sequences still to be present in front of the actual structural genes and, where appropriate, have been genetically modified so that the natural regulation has been switched off and expression of the genes has been increased.<br><br>
However, the nucleic acid construct may also have a simpler structure, that is to say no additional regulatory signals are inserted in front of the aforementioned carotene hydroxylase genes, and the natural promoter with its regulation is not deleted. Instead, the natural regulatory sequence is mutated so that regulation no longer takes place, and gene expression is increased. These modified promoters may also be placed alone in front of the natural genes to increase the activity. The nucleic acid construct may additionally advantageously contain one or more so-called enhancer sequences functionally linked to the promoter, which make increased expression of the nucleic acid sequence possible. It is also possible to insert additional advantageous sequences at the 3' end of the DNA sequences, such as further regulatory elements or terminators. The aforementioned carotene hydroxylase genes may be present in one or more copies in the gene construct.<br>
Advantageous reg;ulatory sequences for the nucleic acid constructs according to the invention, for the process described below for preparing xanthophylls and for the genetically modified organisms described below are present, for example, in promoters such as cos, tac, trp, tet, trβ-tet, Ipp, lac, Ipβ-lac, lacl^, T7, T5, T3, gal, trc, ara, SP6, X-ΛR or in the X-ΛL promoter, which are advantageously used in Gram-negative bacteria.<br>
Further advantageous regulatory sequences are present, for example, in the Gram-positive promoters amy and SP02, in the yeast or fungal promoters ADCl, MFa , AC, β-60, CYCl, GAPDH, TEF, rp28, ADH or in the plant promoters CaMV/35S [Franck et al.. Cell 21(1980) 285-294], PRPl [Ward et al.. Plant.Mol. Biol.22(1993)], SSU, OCS, leb4, usp, STLSl, B33, nos or in the ubiquitin or phaseolin promoter.<br>
Particularly advantageous are those plant promoters which ensure specific expression in tissues or plant parts in which the biosynthesis of carotenoids or precursors thereof takes place or in which the products advantageously accumulate.<br>
Particular mention should be made of promoters for the whole plant on the basis of constitutive expression, such as, for example, the CaMV promoter, the OCS promoter from Agrobacterium (octopine synthase), the NOS promoter from Agrobacterium (nopaline synthase), the ubiquitin promoter, promoters of vacuolar ATPase subunits or the promoter of a proline-rich protein from wheat (WO 9113991),<br><br>
seed-specific promoters such as, for example, the phaseolin promoter and the USP promoter from vicia faba, the promoter of the legumin gene from Vicia (leb4) or the Bce4 gene promoter from Brassica (WO 9113980),<br>
specific promoters for green tissues such as, for example, the SSU promoter (small subunit) of rubisco (ribulose-1,5-bisphosphate carboxylase) or the STLSl promoter (Solanum tuberosum, light harvesting system 1 from potato),<br>
mesophyll-specific promoters such as, for example, the FBPase promoter of the cytosolic FBPase from potato (WO9705900),<br>
specific promoters for tubers, storage roots or roots, such as, for example, the patatin promoter class I (B33), the promoter of the cathepsin D inhibitor from potato, the promoter of starch synthase (GBSSl) or the sporamin promoter,<br>
fruit-specific promoters such as, for example, the fruit-specific promoter from tomato (EP409625),<br>
fruit ripening-specific promoters such as, for example, the fruit ripening-specific promoter from tomato (WO 9421794),<br>
flower-specific promoters such as, for example, the phytoene synthase promoter (W09216635) or the promoter of the β-rr gene (W09822593),<br>
specific plastid or chromoplast promoters such as, for example, the RNA polymerase promoter (WO9706250) or<br>
pathogen- or chemically inducible promoters such as, for example, the PRPl promoter, a benzenesulfonamide-inducible (EP 388186), a tetracycline-inducible (Gatz et al., (1992) Plant J. 2,397-404), an abscisic acid-inducible (EP335528) or an ethanol- or cyclohexanone-inducible (W09321334) promoter.<br>
It is also possible and advantageous to use the promoter of phosphoribosyl-pyrophosphate amidotransferase from Glycine max (see also Genbank accession number U87999) or another node-specific promoter as in EP 249676.<br>
It is possible in principle to use all natural promoters with their regulatory sequences like those mentioned above for the process according to the invention. It is additionally possible and advantageous to use synthetic promoters.<br><br>
The nucleic acid construct may also contain other genes which are to be introduced into the organisms. These genes may be under separate regulation or under the same regulatory region as the above-described carotene hydroxylase genes. Examples of these genes are other biosynthetic genes of carotenoid biosynthesis which make increased synthesis possible. Particular mention may be made of other genes of carotenoid biosynthesis which are biochemically limiting, such as the genes encoding phytoene synthase, phytoene desaturase, isopentyl-pyrophosphate isomerase or β-cyclase.<br>
Genes encoding an isopentyl-pyrophosphate isomerase are known, for exconple, from the organisms Phaffia rhodozyma and Haematococcus pluvialis (EP 769 551) or Arabidopsis thaliana and Tagetes (Marigold) (WO 9736998).<br>
Genes encoding a phytoene synthase are known, for example, from the organisms Erwinia uredovora (EP 393690), Erwinia herbicola (WO 9113078), tomato (WO 9109128), melon (WO 9602650), Flavobacterium (EP 747483) or Nicotiana (US 5705624).<br>
Genes encoding a phytoene desaturase are known, for example, from the organisms Erwinia uredovora (EP 393690), Erwinia herbicola (WO 9113078), Nicotiana (US 5539093) or Flavobacterium (EP 747483) .<br>
Genes encoding a β-cyclase are known, for example, from the organisms Erwinia uredovora (EP 393690), Erwinia herbicola (WO 9113078), Flavobacterium (EP 747483), tobacco and tomato (WO 9628014) or Capsicum annuum (WO 9636717).<br>
The invention further relates to a process for preparing the genetically modified organisms described below, wherein the carotene hydroxylase genes according to the invention or the nucleic acid constructs according to the invention are introduced into the genome of the initial organism. By initial organisms are meant the organisms before the genetic modification according to the invention.<br>
The carotene hydroxylase genes according to the invention or the nucleic acid constructs according to the invention can in principle be introduced by all methods known to the skilled worker into the initial organisms described below, which are genetically modified thereby.<br><br>
They are advantageously introduced into the initial organisms or cells thereof by transformation, transfection, electroporation, using the so-called particle gun or by microinjection.<br>
The skilled worker can find appropriate methods for microorganisms in the textbooks by Sambrook, J. et al. (1989) Molecular cloning: A laboratory manual. Cold Spring Harbor Laboratory Press, by F.M. Ausubel et al. (1994) Current protocols in molecular biology, John Wiley and Sons, by D.M. Glover et al., DNA Cloning Vol. 1, (1995), IRL Press (ISBN 019-963476-9), by Kaiser et al. (1994) Methods in Yeast Genetics, Cold Spring Harbor Laboratory Press or Guthrie et al. Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, 1994, Academic Press.<br>
Excimples of advantageous methods which may be mentioned are those such as the introduction of the DNA by homologous or heterologous recombination, for example using the ura-3 gene, specifically the ura-3 gene from Ashbya, as described in the German Application DE 19801120.2, and/or by the REMI method (= "restriction enzyme mediated integration) which is described below.<br>
The REMI technique is based on the cotransformation of a linear DNA construct which has been cut at both ends with the same restriction endonuclease, together with the restriction endonuclease which was used for this restriction of the DNA construct, into an organism. The restriction endonuclease then cuts the genomic DNA of the organism into which the DNA construct has been introduced together with the restriction enzyme. This leads to an activation of the cell's own repair mechanisms. These repair mechanisms repair the strand breaks in the genomic DNA which have been caused by endonuclease, and during this also incorporate with a certain frequency the cotransformed DNA construct into the genome. Ordinarily, the restriction cleavage sites are retained at both ends of the DNA during this.<br>
This technique was described by Bolker et al- (Mol Gen Genet, 248, 1995: 547-552) for the insertion mutagenesis of fungi. The method was used by Von Schiestl and Petes (Proc. Natl. Acad. Sci. USA, 88, 1991: 7585-7589) to find out whether there is heterologous recombination in Saccharomyces. The method has been described by Brown et al. (Mol. Gen. Genet. 251, 1996: 75-80) for the stable transformation and regulated expression of an inducible reporter gene.<br><br>
It is possible using the REMI method to position the nucleic acid fragments according to the invention or the aforementioned carotene hydroxylase genes according to the invention at transcriptionally active sites in the genome.<br>
It is possible and advantageous to clone the nucleic acids together with at least one reporter gene into a DNA construct, which is introduced into the genome. This reporter gene ought to make detectability easy by a growth, fluorescence, chemo- or bioluminesence assay or by a photometric measurement. Examples which may be mentioned of reporter genes are antibiotic resistance genes, hydrolase genes, fluorescent protein genes, bioluminescence genes, glucosidase genes, the luciferase gene, β-galactosidase gene, gfp gene, lipase gene, esterase gene, peroxidase gene, β-lactamase gene, acetyl-, phospho- or adenyltransferase gene. These genes make it possible easily to measure and quantify the transcription activity and thus the expression of the genes. This means that it is possible to identify sites in the genome which have a productivity differing by up to a factor of 2.<br>
If it is intended to introduce a plurality of genes, such as, for example, further crt genes of carotenoid biosynthesis, into the organism, they can all be introduced together with a reporter gene in a single vector, or each individual gene with a reporter gene can be introduced in one vector in each case, into the organism, it being possible to introduce the various vectors at the same time or successively. It is also possible to insert gene fragments coding for the respective activities in the REMI techniques.<br>
Restriction enzymes suitable in principle for integrating the carotene hydroxylase genes or nucleic acid constructs according to the invention into the genome of initial organisms are all known ones. Restriction enzymes which recognize only 4 base pairs as restriction cleavage site are less preferred because they cut too often in the genome or in the vector to be integrated, and preferred enzymes recognize 6, 7, 8 or more base pairs as cleavage site, such as BamHI, EcoRI, Bglll, SphI, Spel, Xbal,<br>
' Xhol, Ncol, Sail, Clal, Kpnl, Hindlll, Sad, PstI, Bpnl, NotI, Srfl or Sfil, to mention only a few of the possible enzymes. It is advantageous if the enzymes used no longer have cleavage sites in the DNA to be introduced; this increases the efficiency of integration. Ordinarily, 5 to 500 U, preferably 10 to 250,<br>
i particularly preferably 10 to 100 U of the enzymes are used in the REMI mixture. The enzymes are advantageously employed in an aqueous solution which contains substances for osmotic<br><br>
stabilization, such as sugars such as sucrose, trehalose or glucose, polyols such as glycerol or polyethylene glycol, a buffer with an advantageous buffering in the range of pH 5 to 9, preferably 6 to 8, particularly preferably 7 to 8, such as tris, MOPS, HEPES, MES or PIPES and/or substances to stabilize the nucleic acids, such as inorganic or organic salts of Mg, Cu, Co, Fe, Mn or Mo. It is also possible where appropriate for the substances to be present, such as EDTA, EDDA, DTT, β-mercaptoethanol or nuclease inhibitors• However, it is also possible to carry out the REMI technique without these additions-<br>
The process is carried out at a temperature in the range from 5 to 80°C, preferably from 10 to 60°C, particularly preferably from 20 to 40°C. Other known methods for destabilizing cell membranes are suitable for the process, such as, for example, electroporation, fusion with loaded vesicles or destabilization with various alkali metal or alkaline earth metal salts such as lithium, rubidium or calcium salts, with lithium salts being preferred.<br>
The nucleic acids can be used for the reaction according to the invention directly after isolation or after purification.<br>
The introduction of the carotene hydroxylase genes according to the invention or the nucleic acid constructs according to the invention into plants can in principle take place by all methods known to the skilled worker.<br>
The transfer of foreign genes into the genome of a plant is referred to as transformation. In this case, the described methods for the transformation and regeneration of plants from plant tissues or plant cells for transient or stable transformation are utilized.<br>
Suitable methods are protoplast transformation by polyethylene glycol-induced DNA uptake, the use of a gene gun, electroporation, incubation of dry embryos in DNA-containing solution, microinjection and Agrobacterium-mediated gene transfer. The methods mentioned are described, for example, in B. Jenes et al.. Techniques for Gene Transfer, in: Transgenic Plants, Vol. 1, Engineering and Utilization, edited by S-D. Kung and R. Wu, Academic Press (1993) 128-143, and in Potrykus Annu. Rev. Plant Physiol. Plant Molec. Biol. 42 (1991) 205-225.<br>
The construct to be expressed is preferably cloned into a vector which is suitable for transforming Agrobacterium tumefaciens,   for example pBinl9 (Bevan et al., Nucl. Acids Res. 12 (1984) 8711).<br><br>
Transformation with Agrobacterium tumefaciens is described, for exciinple, by Hofgen and Willmitzer in Nucl. Acids Res. (1988) 16, 9877-<br>
Agrobacteria transformed with an expression vector according to the invention can likewise be used in a known manner for transforming plants, in particular tagetes, sunflower, arabidopsis, tobacco, red pepper, soybean, tomato, aubergine, paprika, carrot, potato, corn, lettuce and brassica species, oats, rye, wheat, triticale, millet, rice, alfalfa, flax, brassicaceae such as, for example, oilseed rape or canola, sugar beet, sugar cane or wood plants such as, for example, aspen or yew, for example by bathing wounded leaves or pieces of leaves in a solution of agrobacteria and then cultivating in suitable media.<br>
The genetically modified plant cells can be regenerated by all methods known to the skilled worker. Appropriate methods can be found in the abovementioned publications by S.D. Kung and R. Wu, Potrykus or Hofgen and Willmitzer.<br>
There is a large number of possibilities for increasing the enzymic activity of the carotene hydroxylase gene products in the cell.<br>
One possibility is to modify the endogenous carotene hydroxylase genes so that they code for enzymes having a carotene hydroxylase activity which is higher than the initial enzymes. Another increase in the enzymic activity can be achieved, for example, by modifying the catalytic sites to increase substrate conversion or by aborting the effect of enzyme inhibitors, that is to say they have an increased specific activity or their activity is not inhibited. The enzymic activity can also be increased in a further advantageous embodiment by increasing enzyme synthesis in the cell, for example by eliminating factors which repress enzyme synthesis or by increasing the activity of factors or regulatory elements which promote enhanced synthesis, or preferably by introducing further gene copies. This measure increases the total activity of the gene products in the cell without altering the specific activity. It is also possible to use a combination of these methods, that is to say increasing the specific activity plus increasing the total activity. These modifications can in principle be introduced by all methods known to the skilled worker into the nucleic acid sequences of the genes, regulatory elements or promoters thereof. It is possible for this purpose to subject the sequences, for example, to a mutagenesis such as a site directed mutagenesis as described in D.M. Glover et al., DNA<br><br>
Cloning Vol.1, (1995), IRL Press (ISBN 019-963476-9), chapter 6, page 193 et seq,<br>
Spee et al. (Nucleic Acids Research, Vol. 21, No. 3, 1993: 777 -778) describes a PCR method using dITP for random mutagenesis.<br>
The use of an in vitro recombination technique for molecular evolution is described by Stemmer (Proc. Natl. Acad. Sci. USA, Vol. 91, 1994: 10747 - 10751).<br>
Moore et al. (Nature Biotechnology Vol. 14, 1996: 458 - 467) describe combination of the PCR and recombination methods.<br>
The modified nucleic acid sequences are subsequently returned to the organisms via vectors.<br>
For increasing enzymic activities it is also possible to place modified promoter regions in front of the natural genes so that expression of the genes is increased and thus the activity is eventually raised. Sequences can also be introduced at the 3' end, for example to increase the stability of the mRNA and thus make increased translation possible. This likewise leads to higher enzymic activity.<br>
In a further preferred embodiment, the carotene hydroxylase genes or nucleic acid constructs according to the invention are, for expression in one of the organisms described below, inserted into a vector such as, for example, a plasmid, a phage or other DNA which makes optimal expression of the genes possible in the prokaryotic or eukaryotic organisms. This vector may contain the ATG start codon for expression of the protein according to the invention.<br>
Examples of suitable plasmids in E. coli are pLG33 8, pACyC184, pBR322, pUC18, pUC19, pKC30, pRep4, pHSl, pHS2, pPLc236, pMBL24, pLG200, pUR290, pIN-III13.Bl, Xgtll, pBdCI, the pET vector series (Novagen and Stratagene), pMAL or the pQE vector series (Qiagen), in fungi pALSl, pIL2 or pBB116, in yeasts 2 \i,   pAG-1, yEp6, YEpl3 or pEMBLYe23 or<br>
in plants pLGV23, pGHlac+, pBINl9, pAK2004, pDH51, or derivatives of the aforementioned plasmids.<br>
Said plasmids represent a small selection of the possible plasmids. Further plasmids are well known to the skilled worker and can be found, for example, in the book Cloning Vectors (Eds. Pouwels P. H. et al. Elsevier, Amsterdam-New York-Oxford, 1985, ISBN 0 444 904018). Suitable plant vectors are described inter<br><br>
alia in "Methods in Plant Molecular Biology and Biotechnology" (CRC Press), chapters 6/7, pp.71-119.<br>
It is advantageous for expression of the other genes present if the nucleic acid fragment also contains in addition 3' and/or 5' terminal regulatory sequences to increase expression, which are selected for optimal expression depending on the selected host organism and gene or genes.<br>
These regulatory sequences are intended to make specific expression of the genes and protein expression possible. This may mean, for example, depending on the host organism that the gene is expressed and/or overexpressed only after induction, or that it is immediately expressed and/or overexpressed.<br>
The regulatory sequences or factors may moreover preferably have a beneficial effect on expression of the introduced genes, and thus increase it. Enhancement of the regulatory elements is thus advantageously possible at the level of transcription by using strong transcription signals such as promoters and/or enhancers. However, it is also possible to enhance translation by, for example, improving the stability of the mRNA.<br>
In a further embodiment of the vector, the nucleic acid construct according to the invention may also advantageously be introduced in the form of a linear DNA into the host organisms and be integrated into the genome of the host organism by heterologous or homologous recombination. This linear DNA may consist of a linearized plasmid or only of the nucleic acid fragment as vector.<br>
It is also possible to use as vector any plasmid which undergoes autonomous replication in the cell, but also, as described above, a linear DNA fragment which integrates into the genome of the host. This integration can take place by heterologous or homologous recombination, but preferably, as mentioned, by homologous recombination (Steiner et al., Genetics, Vol. 140, 1995: 973-987). It is moreover possible for the aforementioned carotene hydroxylase genes to be present singly in the genome at various sites or on various vectors or to be present together in the genome or on one vector.<br>
The invention further relates to the use of the carotene hydroxylase genes according to the invention for producing genetically modified organisms.<br><br>
The invention further relates to a correspondingly genetically<br>
modified organism, with the expression of the carotene<br>
hydroxylase genes according to the invention<br>
being increased by comparison with a wild type in the case where<br>
the initial organism contains a carotene hydroxylase gene<br>
according to the invention,<br>
or being caused in the case where the initial organism does not<br>
contain a carotene hydroxylase gene according to the invention,<br>
by the genetic modification.<br>
A genetically modified organism means an organism in which the carotene hydroxylase genes or nucleic acid constructs according to the invention have been inserted, preferably by one of the methods described above.<br>
The genetically modified organism contains at least one carotene hydroxylase gene according to the invention or at least one nucleic acid construct according to the invention. Depending on the initial organism, the nucleic acid may be present inside or outside the chromosome.<br>
Carotenoid metabolism in the genetically modified organisms is preferably altered by comparison with the wild type.<br>
Suitable genetically modified organisms are in principle all organisms able to synthesize xanthophylls.<br>
Preferred initial organisms are those naturally able to synthesize xanthophylls. However, initial organisms able to synthesize xanthophylls because of the introduction of carotenoid biosynthesis genes are also suitable.<br>
Initial organisms mean prokaryotic or eukaryotic organisms such as, for example, microorganisms or plants. Preferred microorganisms are bacteria, yeasts, algae or fungi.<br>
Bacteria which can be used are both bacteria which are able, because of the introduction of carotenoid biosynthesis genes of a carotenoid-producing organism to synthesize xanthophylls, such as, for example, bacteria of the genus Escherichia, which contain, for example, crt genes from Erwinia, and bacteria which are intrinsically able to synthesize xanthophylls, such as, for example, bacteria of the genus Erwinia, Agrobacterium, Flavobacterium, Alcaligenes or Cyanobacteria of the genus Synechocystis. Preferred bacteria are Escherichia  colif  Erwinia herbicola,  Erwinia  uredovora,  Agrobacterium aurantiacum.<br><br>
Alcaligenes sp.  PC-1,  Flavobacterium  sp. strain R1534  or the Cyanobacterium Synechocystis sp. PCC6803.<br>
Preferred uses are Candida, Saccharomyces, Hansenula or Pichia.<br>
Preferred fungi are Aspergillus, Trichoderma, Ashbya, Neurospora, Fusarium or other fungi described in Indian Chem Engr. Section E. Vol 37, No- 1,2 (1995) on page 15, Table 6.<br>
Preferred algae are green algae such as, for example, algae of. the genus Haematococcus,  Phaedactylum tricornatum,   Volvox  or Dunaliella. Particular preferred algae are Haematococcus pluvialis  or Dunaliella bardawil.<br>
In a preferred embodiment, plants are used as initial organisms and, accordingly, also as genetically modified organisms. Examples of preferred plants are tagetes, sunflower, arabidopsis, tobacco, red pepper, soybean, tomato, aubergine, paprika, carrot, potato, corn, lettuce and brassica species, oats, rye, wheat, triticale, millet, rice, alfalfa, flax, brassicaceae such as, for example, oilseed rape or canola, sugar beet, sugar cane or woody plants such as, for example aspen or yew.<br>
Particular preference is given to Arabidopsis thaliana,   Tagetes erecta, oilseed rape, canola, potatoes, and oilseeds and typical carotenoid producers such as soybean, sunflower, paprika, carrot, pepper or corn.<br>
The invention further relates to a process for the preparation of xanthophyll derivatives, which comprises converting a β-ionone structural element into a 3-hydroxy-β-ionone structural element and/or a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element in the presence of the protein according to the invention.<br>
Xanthophyll derivatives mean xanthophylls, preferably xanthophylls containing at least one hydroxyl group, such as, for example, zeaxanthin, β-cryptoxanthin, 3'-hydroxyechinenone, 3-hydroxyechinenone, adonixanthin (4-ketozeaxanthin), astaxanthin, phoenicoxanthin (adonirubin), a-cryptoxanthin or lutein or derivatives thereof having up to 40 C atoms and containing at least one 3-hydroxy-β-ionone or at least one 3-hydroxy-4-keto-β-ionone structural element in the molecule, such as, for example, 3-hydroxy-6-vinyl-β-ionone, 3-hydroxy-4-keto-6-vinyl-P"ionone, 3-hydroxyretinol, 3-hydroxy-4-ketoretinol, 3-hydroxyretinal,<br><br>
3-hydroxy-4-ketoretinal, 3-hydroxyretinoic acid or 3-hydroxy-4-ketoretinoic acid.<br>
Preferred xanthophyll derivatives are zeaxanthin, lutein and astaxanthin.<br>
In the process according to the invention there is conversion in the presence of the proteins according to the invention of a β-ionone structural element into a 3-hydroxy-β-ionone structural element, such as β-carotene into zeaxanthin, β-carotene into β-cryptoxanthin, β-cryptoxanthin into zeaxanthin, echinenon into 3'-hydroxyechinenone, 3-hydroxyechinenone into adonixanthin (4-ketozeaxanthin), α-carotene into α-cryptoxanthin or a chemical compound having up to 40 C atoms and containing a β-ionone ring into the corresponding 3-hydroxy-β-ionone compound or a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element, such as canthaxanthin into astaxanthin, canthaxanthin in phoenicoxanthin (adonirubin), phoenicoxanthin (adonirubin) into astaxanthin, echinenone into 3-hydroxyechinenone, 3'-hydroxyechinenone into adonixanthin (4-ketozeaxanthin) or a chemical compound having up to 40 C atoms and containing a 4-keto-β-ionone ring into the corresponding 3-hydroxy-4-keto-β-ionone compound.<br>
In a preferred embodiment of the process, an aforementioned genetically modified organism according to the invention is cultivated, the latter organism is harvested, and subsequently the xanthophyll derivatives are isolated from the organism.<br>
Cultivation of the genetically modified organism according to the invention takes place in a manner known per se, such as cultivation of the appropriate wild type, for example in the case of microorganisms in a suitable medium such as, for example, on agar plates or in suspension culture, or in the case of plants in soil or appropriately suitable nutrient media. By harvesting is meant in the case of microorganisms the isolation of the microorganisms, and in the case of plants the cutting off of the plant or, where appropriate, particular plant parts containing the xanthophyll derivatives.<br>
The xanthophyll derivatives are isolated in a manner known per se, for example by disruption of the organism cells, extraction of the xanthophyll derivatives and subsequent purification of the xanthophyll derivatives by chemical or physical separation methods such as extraction or chromatography.<br><br>
The invention further relates to the use of the carotene hydroxylase according to the invention or of the carotene hydroxylase genes according to the invention for preparing xanthophyll derivatives.<br>
The following examples illustrate the invention<br>
Example 1<br>
Incorporation of the carotene hydroxylase gene into a β-carotene producing E. coli, fermentation of the transgenic organism and isolation of the xanthophylls<br>
General methods<br>
Isolation of the carotenoids<br>
To isolate the carotenoids (carotenes and xanthophylls), the E. coli cells were collected by centrifugation, and the resulting cell material was freeze dried for 24 h. The freeze-dried cells were resuspended in acetone and extracted twice with acetone at 55°C for 15 min. The combined extracts were washed with a diethyl ether/petroleum ether (boiling point 35-80°C) mixture (1:9, v/v) in a separating funnel and concentrated to dryness under nitrogen in a rotary evaporator.<br>
HPLC analysis<br>
The extracts were fractionated using a Nucleosil 100-5 C18 column<br>
(Macherey-Nagel) with an eluent flow rate of 1.5 ml/min.<br>
The eluent used to fractionate the β-carotene and the hydroxylated<br>
xanthophylls in Excimple 1 was an acetonitrile/methanol/2-propanol<br>
mixture (85:10:5, v/v/v; flow ).<br>
To fractionate the xanthophylls with keto groups in Example 2, an<br>
acetonitrile/methanol/H20 mixture (50:44:6, v/v/v) was used as<br>
eluent 1 for 22 min, and methanol was used as eluent 2. Detection<br>
took place directly using the Waters 994  diode array  detector. A<br>
comparison standard for the HPLC analysis were β-carotene,<br>
astaxanthin and zeaxanthin purchased from Sigma or Roth.<br>
1.1 Production of β-carotene-producing E. coli<br>
The organism used comprised E. coli cells of the strain JM101. The plasmid pACCAR16DcrtX contains the bacterial carotenoid biosynthesis genes crtE, crtB, crtI and crtY from Erwinia uredovora and results in the biosynthesis of β-carotene (N. Misawa et al., J. Bacterid. 172 (1990) 6704-6712; Biochem. Biophys. Res. Commun. 209 (1995) 867-876)<br><br>
To prepare this plasmid, a 6,0 kb Asp718(KpnI)-EcoRI fragment of the carotenoid biosynthesis gene cluster from Erwinia  uredovora (Plasmid pCARlSdelB) was cloned into the EcoRI site of the plasmid pACY184 (R.E. Rose, Nucl. Acids Res. 16 (1988) 355). The plasmid pCAR16delB contains a frameshift mutation in the β-carotene hydroxylase ORF  (open reading- frame)   (Misawa et al., Biochem. Biophys. Res. Commun. 209 (1995) 867-876). It is therefore impossible for the resulting β-carotene to be hydroxylated to zeaxanthin.<br>
Insertion of the plasmid pACCAR16DcrtX into E. coli and preparation and isolation of transformed E. coli cells took place in a manner known per se, as described in Sambrook et al., Molecular cloning: a laboratory manual, 2nd edn. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
1.2 Construction of a Haematococcus pluvialis XcDNA expression library and isolation of the plasmid containing the carotene hydroxylase gene<br>
Haematococcus pluvialis  Flotow NIES-144 originated from the National Institute for Environmental Studies   (NIES), Tsukuba, Japan. The basal medium (pH 6.8) in the growth phase of Haematococcus pluvialis  contained per liter 1.2 g of sodium acetate, 2.0 g of yeast extract, 0.4 g of L-asparagine, 0.2 g of MgCl2*6H2O, 0.01 g of FeSO4*7H2O, and 0.02 g of CaCl2*2H2O-Haematococcus pluvialis  was grown at 20°C with a dark/light cycle of 12 h light (20 µE/m2s) and 12 h dark for 4 days. To induce astaxanthin biosynthesis and cyst cell formation, after 4 days sodium acetate and FeSO4 were added to a concentration of 45 mM and 450 µM  respectively. After the addition, the light conditions were changed to continuous light (125 (lE/m2s) as described in Kajiwara et al., Plant Mol. Biol, 29 (1995) 343-352. After induction of cyst cell' formation for 8 h, the RNAs of Haematococcus pluvialis were isolated to construct a cDNA library from the cyst cells.<br>
The poly(A)RNA was purified using oligo(dT)-cellulose (Biolabs). Synthesis of the cDNAs and construction of the XZAP expression library took place with the aid of the cDNA Synthesis and ZAβ-cDNA Gigapack III Gold Cloning  Ki,t (Stratagene). The first strand of the cDNA was synthesized using MMLV* reverse transcriptase and a poly(dT) primer containing an Xhol restriction enzyme recognition site in accordance with the Stratagene instructions for use. The second strand was synthesized correspondingly using DNA polymerase I. Blunt DNA ends were produced by filling in with Pfu DNA polymerase, and EcoRI adapters were ligated thereto. The resulting cDNA fragments were then fractionated according to size and subsequently ligated<br><br>
into the EcoRI - Xhol recognition site of the Uni-ZAP XR vector. After isolation and purification of the positive carotene hydroxylase plaque, the pBluescript phagemid containing the carotene hydroxylase cDNA was recovered by in vivo cutting out by use of the ExAssist helper phage and the SOLR E. coli strain, in accordance with the Stratagene instructions for use. The resulting plasmid contains in addition to the short adapter sequences at the 5'- (5'-AATTCGGCACGAG-3') and 3' (5'-TCGAG-3') ends according to DNA sequence analysis a 1608 bβ-long cDNA fragment ligated into the EcoRI and Xhol restriction site of the multiple cloning site. This plasmid was used for insertion of the carotene hydroxylase gene into the β-carotene-producing E. coli cells as described under 1.1,<br>
1.3. Incorporation of the carotene hydroxylase gene into β-carotene-producing E. coli cells, cultivation of the transformed cells and isolation of the xanthophylls zeaxanthin and β-cryptoxanthin<br>
Transformation of the β-carotene-producing E. coli cells containing the plasmid pACCAR16DcrtX, described under 1.1., with the plasmid containing the carotene hydroxylase gene, described under 1.3., took place in a manner known per se as described in J. Sambrook et al.. Molecular cloning: a laboratory manual, 2nd edn. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
The transformed E. coli cells were cultured in LB medium at 28°C with addition of ampicillin (50 µg/ml; for the plasmid containing the carotene hydroxylase gene) and chloramphenicol (30 ^ig/ml; plasmid pACCAR16DcrtX) for 48 h.<br>
The carotenoids were isolated as described under the general methods. Fig. 1 shows the HPLC diagram for the carotenoids extracted from<br>
(A)	E, coli cells containing the plasmid pACCAR16DcrtX from 1-1 and<br>
(B)	E. coli cells containing the plasmid pACCARlSDcrtX together with the carotene hydroxylase gene from 1.2.<br>
The peak numbers in Fig. 1 mean<br><br>
3	zeaxanthin<br>
5	β-cryptoxanthin<br>
6	β-Carotene<br>
As is evident from Fig. 1, the transformed E. coli cells produce, due to incorporation of the carotene hydroxylase gene according to the invention, the hydroxylated xanthophylls zeaxanthin and β-cryptoxanthin, whereas only β-carotene is produced without the transformation. The carotene hydroxylase according to the invention is accordingly able to convert β-carotene into β-cryptoxanthin, β-cryptoxanthin into zeaxanthin or β-carotene into zeaxanthin.<br>
Example 2<br>
Incorporation of the carotene hydroxylase gene and the β-carotene ketolase gene (bkt) from Haematococcus pluvialis  into β-carotene-producing E. coli cells, cultivation of the transformed cells and isolation of the xanthophylls astaxanthin, canthaxanthin, adonixanthin, zeaxanthin and β-cryptoxanthin<br>
2.1	Preparation of a plasmid containing the β-carotene ketolase<br>
gene (bkt) from Haematococcus pluvialis<br>
To prepare the plasmid pRKbktl (Kajiwara et al.. Plant Mol. Biol. 29 (1995) 343-352) containing the β-carotene ketolase gene from Haematococcus pluvialis,   a 1 kb product of a PvuII partial digest of the pUC19bkt plasmid was fractionated by agarose gel electrophoresis (Breitenbach et al., FEMS Microbiol. Lett. 140 (1996) 241-246). This DNA fragment contains the lacZ promoter together with the ORF (open reading frame) of the ketolase and was subcloned into the plasmid pRK404 which had previously been digested with Hindlll and treated with the Klenow enzyme.<br>
The resulting plasmid pRKbktl was inserted once alone and once together with the carotene hydroxylase gene plasmid described under 1.2 into the β-carotene-producing E. coli cells.<br>
2.2	Incorporation of the plasmid pRKbktl into β-carotene-producing<br>
E. coli cells, cultivation of the transformed cells and isolation<br>
of the xanthophyll canthaxanthin<br>
Transformation of the β-carotene-producing E. coli cells containing the plasmid pACCAR16DcrtX, described under 1.1., with the plasmid pRKbktl containing the β-carotene ketolase gene (bkt), described under 2.1., took place in a manner known per se as described in J. Sambrook et al.. Molecular cloning: a laboratory<br><br>
manual, 2nd edn. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
The transformed E. coli cells were cultured in analogy to the description under 1.3. in LB medium at 28 °C, but with the addition of chloramphenicol (30 µg/ml; plasmid pACCARlSDcrtX), tetracycline (10µg/ml; plasmid pRKbktl) and isopropyl β-D-thio-galactopyranoside (0.5 mM) for 48 h.<br>
The carotenoids were isolated as described under the general methods.<br>
Fig. 2 (A) shows HPLC-diagram of the carotenoids extracted from E. coli cells containing the plasmid pACCAR16DcrtX together with the plasmid pRKbktl, containing the β-carotene ketolase gene (bkt).<br>
Incorporation of the plasmid pRKbktl results in the transformed E. coli cells producing the xanthophyll with a keto group canthaxanthin.<br>
2.3 Incorporation of the plasmid pRKbktl and the carotene hydroxylase gene into β-carotene-producing E. coli cells, cultivation of the transformed cells and isolation of the xanthophylls, astaxanthin, canthaxanthin, adonixanthin, zeaxanthin and β-cryptoxanthin<br>
Transformation of the β-carotene-producing E. coli cells containing the plasmid pACCAR16DcrtX, described under 1.1., with the plasmid pRKbktl containing the β-carotene ketolase gene (bkt), described under 2.1., with the plasmid containing the carotene hydroxylase gene, described under 1.3, took place in a manner known per se as described in J. Sambrook et al.. Molecular cloning: a laboratory manual, 2nd edn. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
The transformed E. coli cells were cultured in analogy to the description under 1.3. in LB medium at 28°C, but with the addition of ampicillin (50 µg/ml; plasmid containing the carotene hydroxylase gene), chloramphenicol (30 µg/ml; plasmid pACCARl6DcrtX), tetracycline (10 µg/ml; plasmid pRKbktl) and isopropyl β-D-thio-galactopyranoside (0.5 mM) for 48 h.<br>
The carotenoids were isolated as described under the general methods. Fig. 2 shows the HPLC diagrams for the carotenoids extracted from<br><br>
(A)	E. coli cells containing the plasmid pACCAR16DcrtX from 1.1 together with the plasmid pRKbktl and<br>
(B)	E. coli cells containing the plasmid pACCAR16DcrtX together with the plasmid pRKbktl and the carotene hydroxylase gene from 1.2.<br>
Fig. 2 (C) shows astaxanthin as comparison standard.<br>
The peak numbers in Fig. 2 mean<br><br>
1	astaxanthin<br>
2	adonixanthin<br>
3	zeaxanthin<br>
4	canthaxanthin<br>
5	β-cryptoxanthin<br>
6	β-carotene<br>
As can be seen in Fig. 2,   the transformed E. coli cells produce, due to incorporation of the carotene hydroxylase gene according to the invention together with the plasmid pRKbktl containing the β-carotene ketolase gene (bkt), the hydroxylated and/or keto-containing xanthophylls astaxanthin, canthaxanthin, adonixanthin, zeaxanthin and β-cryptoxanthin, whereas only canthaxanthin is produced without the carotene hydroxylase gene according to the invention.<br>
In enzyme studies on the β-carotene ketolase from Haematococcus pluvialis it was shown that the enzyme mainly converts β-carotene into canthaxanthin, whereas hydroxyl-containing xanthophylls such as zeaxanthin and β-cryptoxanthin are scarcely converted, and only to adonixanthin and not to astaxanthin (T. Lotan, J- Hirschberg, FEBS Lett. 364 (1995) 125-128; J. Breitenbach, N. Misawa, S. Kajiwara, G. Sandmann, FEMS Microbiol. Lett. 140 (1996) 241-246; P.D. Eraser, H. Shimada, N. Misawa, Eur. J. Biochem. 252 (1998) 229-236) .<br>
The fact that the E. coli cells transformed in Example 2.3 are able to produce astaxanthin proves that the carotene hydroxylase according to the invention is able to convert canthaxanthin via phoenicoxanthin (adonirubin) into astaxanthin.<br>
The carotene hydroxylase according to the invention is accordingly able to catalyze the conversion of a β-ionone structural element into a 3-hydroxy-β-ionone structural element, such as the conversion of β-carotene into zeaxanthin, β-carotene into β-cryptoxanthin, β-cryptoxanthin into zeaxanthin, echinenone<br><br>
into 3'-hydroxyechinenone, 3-hydroxyechinenone into adonixanthin (4-ketozeaxanthin), α-carotene into α-cryptoxanthin or other chemical compounds which have up to 40 C atoms and contain a β-ionone ring into the corresponding 3-hydroxy-β-ionone compounds or the conversion of a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element, such as the conversion of canthaxanthin into astaxanthin, canthaxanthin into phoenicoxanthin (adonirubin), phoenicoxanthin (adonirubin) into astaxanthin, echinenone into 3-hydroxyechinenone, 3'-hydroxyechinenone into adonixanthin (4-ketozeaxanthin) or other chemical compounds which have up to 40 C atoms and contain a 4-keto-β-ionone ring into the corresponding 3-hydroxy-4-keto-β-ionone compounds.<br><br>
SEQUENCE LISTING<br> BASF Aktiengesellschaft<br> Carotene hydroxylase and process for preparing xanthophyll derivatives<br> OZ 0050/49896<br><br> 2<br> Patentln Vers. 2.0<br>	1<br>	1607<br>	DNA<br>	Haematococcus pluvialis<br><br> CDS<br> (3)..(968)<br> 1<br>
ct aca ttt cac aag ccc gtg age ggt gca age get ctg ece cae	ate   47<br>
Thr Phe His Lys Pro Val Ser Gly Ala Ser Ala Leu Pro His	Ile<br>
15	10	15<br>
ggc eca cet cct cat etc eat egg tea ttt get get ace aeg- atg ctg  95 Gly Pro Pro Pro His Leu His Arg Ser Phe Ala Ala Thr Thr Met Leu<br>
20	25	30<br>
teg aag ctg cag tea ate age gte aag gee ege egc gtt gaa eta gee  143 Ser Lys Leu Gin Ser Ile Ser Val Lys Ala Arg Arg Val Glu Leu Ala<br>
35	40	45<br>
egc gae ate acg egg ccc aaa gte tgc etg eat get cag egg tge teg  191<br>
Arg Asp Ile Thr Arg Pro Lys Val Cys Leu His Ala Gin Arg Cys Ser<br>
50	55	60<br>
tta gtt egg etg ega gtg gca gea eca cag aca gag gag gcg etg gga  239<br>
Leu Val Arg Leu Arg Val Ala Ala Pro Gin Thr Glu Glu Ala Leu Gly<br>
65	70	75<br>
ace gtg cag get gee ggc gcg ggc gat gag cac ago gee gat gta gea  2 87<br>
Thr Val Gin Ala Ala Gly Ala Gly Asp Glu His Ser Ala Asp Val Ala<br>
80	85	90	95<br>
etc eag cag ctt gac egg get ate gea gag cgt cgt gee egg egc aaa   335<br><br>
Leu Gin Gin Leu Asp Arg Ala He Ala Glu Arg Arg Ala Arg Arg Lys<br>
100	105	110<br>
egg gag cag ctg tea tac eag get gee gee att gca gea tea att ggc  383 Arg Glu Gin Leu Ser Tyr Gin Ala Ala Ala He Ala Ala Ser He Gly<br>
115	120	125<br>
gtg tea ggc att gee ate tte gcc aec tae ctg aga ttt gcc atg cac  431<br>
 Val Ser Gly He Ala He Phe Ala Thr Tyr Leu Arg Phe Ala Met His<br>
130	135	140<br>
atg ace gtg ggc ggc gca gtg eca tgg ggt gaa gtg get ggc act etc  479<br>
Met Thr Val Gly Gly Ala Val Pro Trp Gly Glu Val Ala Gly Thr Leu<br>
145	150	155<br>
etc ttg gtg gtt ggt ggc gcg etc ggc atg gag atg tat gee cge tat  527<br>
Leu Leu Val Val Gly Gly Ala Leu Gly Met Glu Met Tyr Ala Arg Tyr<br>
160	165	170	175<br>
gca cac aaa gee ate tgg cat gag tcg cct ctg ggc tgg ctg ctg cac  575 Ala His Lys Ala He Trp His Glu Ser Pro Leu Gly Trp Leu Leu His<br>
180	185	190<br>
aag age cac cac aca cct cgc act gga ccc ttt gaa gee aac gae ttg  623 Lys Ser His His Thr Pro Arg Thr Gly Pro Phe Glu Ala Asn Asp Leu<br>
195	200	205<br>
ttt gca ate ate aat gga ctg ccc gee atg etc ctg tgt ace ttt ggc  671<br>
Phe Ala He He Asn Gly Leu Pro Ala Met Leu Leu Cys Thr Phe Gly<br>
210	215	220<br>
tte tgg ctg cec aac gtc ctg ggg gcg gee tge ttt gga gcg ggg ctg  719<br>
Phe Trp Leu Pro Asn Val Leu Gly Ala Ala Cys Phe Gly Ala Gly Leu<br>
225	230	235<br>
ggc ate aeg eta tac ggc atg gca tat atg ttt gta cac gat ggc ctg   767<br>
Gly He Thr Leu Tyr Gly Met Ala Tyr Met Phe Val His Asp Gly Leu<br>
240	245	250	255<br>
gtg cac agg cgc ttt cec aec ggg ecc ate get ggc ctg ecc tae atg  815 Val His Arg Arg Phe Pro Thr Gly Pro He Ala Gly Leu Pro Tyr Met<br>
260	265	270<br>
aag cge ctg aca gtg gee cac cag eta cac cac age ggc aag tae ggt   863 Lys Arg Leu Thr Val Ala His Gin Leu His His Ser Gly Lys Tyr Gly<br>
275	280	285<br>
ggc gcg ecc tgg ggt atg tte ttg ggt eca cag gag ctg eag cac att  911<br>
Gly Ala Pro Trp Gly Met Phe Leu Gly Pro Gin Glu Leu Gin His He<br>
290	295	300<br><br>
cca ggt gcg gcg gag gag gtg gag cga ctg gtc ctg gaa ctg gac tgg  959<br>
Pro Gly Ala Ala Glu Glu Val Glu Arg Leu Val Leu Glu Leu Asp Trp<br>
305	310	315<br>
tec aag egg tagggtgcgg aaccaggcac gctggtttca cacctcatgc	1008<br>
Ser Lys Arg<br>
320<br>
ctgtgataag gtgtggctag agegatgegt gtgagacggg	tatgteacgg	tegactggte	1068<br>
tgatggeeaa tggcatcggc catgtctggt cateacgggc	tggttgcctg	ggtgaaggtg	1128<br>
atgcacatea tcatgtgcgg ttggagggge tggcacagtg	tgggctgaae	tggagcagtt	1188<br>
gtecaggctg gegttgaatc agtgagggtt tgtgattggc	ggttgtgaag	caatgactec	1248<br>
geccatattc tatttgtggg agetgagatg atggcatgct	tgggatgtgc	atggateatg	1308<br>
gtagtgcagc aaactatatt caeetaggge tgttggtagg	atcaggtgag	gecttgeaca	1368<br>
ttgcatgatg taetegtcat ggtgtgttgg tgagaggatg	gatgtggatg	gatgtgtatt	1428<br>
cteagaegta gaecttgact ggaggcttga tcgagagagt	gggccgtatt	ctttgagagg	1488<br>
ggaggetegt gceagaaatg gtgagtggat gaetgtgacg	ctgtacattg	caggcaggtg	1548<br>
agatgcactg tctcgattgt aaaataeatt cagatgcaaa	aaaaaaaaaa	aaaaaaaaaa	1608<br> 2<br> 322<br> PRT<br> Haematoeoccus pluvialis<br> 2<br>
Thr Phe His Lys Pro Val Ser Gly Ala Ser Ala Leu Pro His Ile Gly<br>
15	10	15<br>
Pro Pro Pro His Leu His Arg Ser Phe Ala Ala Thr Thr Met Leu Ser<br>
20	25	30<br>
Lys Leu Gin Ser Ile Ser Val Lys Ala Arg Arg Val Glu Leu Ala Arg<br>
35	40	45<br>
Asp Ile Thr Arg Pro Lys Val Cys Leu His Ala Gin Arg Cys Ser Leu<br>
50	55	60<br>
Val Arg Leu Arg Val Ala Ala Pro Gin Thr Glu Glu Ala Leu Gly Thr<br>
65	70	75	80<br>
Val Gin Ala Ala Gly Ala Gly Asp Glu His Ser Ala Asp Val Ala Leu<br><br>
85	90	95<br>
Gin Gin Leu Asp Arg Ala Ile Ala Glu Arg Arg Ala Arg Arg Lys Arg<br>
100	105	110<br>
Glu Gin Leu Ser Tyr Gin Ala Ala Ala Ile Ala Ala Ser Ile Gly Val<br>
115	120	125<br>
 Ser Gly Ile Ala Ile Phe Ala Thr Tyr Leu Arg Phe Ala Met His Met<br>
130	135	140<br>
Thr Val Gly Gly Ala Val Pro Trp Gly Glu Val Ala Gly Thr Leu Leu<br>
145	150	155	160<br>
Leu Val Val Gly Gly Ala Leu Gly Met Glu Met Tyr Ala Arg Tyr Ala<br>
165	170	175<br>
His Lys Ala Ile Trp His Glu Ser Pro Leu Gly Trp Leu Leu His Lys<br>
180	185	190<br>
Ser His His Thr Pro Arg Thr Gly Pro Phe Glu Ala Asn Asp Leu Phe<br>
195	200	205<br>
Ala Ile Ile Asn Gly Leu Pro Ala Met Leu Leu Cys Thr Phe Gly Phe<br>
210	215	220<br>
Trp Leu Pro Asn Val Leu Gly Ala Ala Cys Phe Gly Ala Gly Leu Gly<br>
225	230	235	240<br>
He Thr Leu Tyr Gly Met Ala Tyr Met Phe Val His Asp Gly Leu Val<br>
245	250	255<br>
His Arg Arg Phe Pro Thr Gly Pro He Ala Gly Leu Pro Tyr Met Lys<br>
260	265	270<br>
Arg Leu Thr Val Ala His Gin Leu His His Ser Gly Lys Tyr Gly Gly<br>
275	280	285<br>
Ala Pro Trp Gly Met Phe Leu Gly Pro Gin Glu Leu Gin His He Pro<br>
290	295	300<br>
Gly Ala Ala Glu Glu Val Glu Arg Leu Val Leu Glu Leu Asp Trp Ser<br>
305	310	315	320<br>
Lys Arg<br><br><br><br><br>
We claim:<br>
1.	A protein which has an enzymatic activity for converting β-carotene into zeaxanthin or canthaxanthin into astaxanthin, comprising the amino acid sequence SEQ ID NO. 2 or a sequence derived from this sequence by substitution, insertion or deletion of amino acids and having a homology of at least 50% at the amino acid level with the sequence SEQ ID NO.2.<br>
2.	A protein as claimed in claim 1, wherein the protein has an enzymatic activity for converting p-carotene into zeaxanthin and an enzymatic activity for converting canthaxanthin into astaxanthin-<br>
3.	A nucleic acid encoding a protein as claimed in either of claims 1 or 2.<br>
4.	A nucleic acid as claimed in claim 3, which consists of the sequence depicted in SEQ ID NO. 1.<br>
5.	A nucleic acid construct comprising a nucleic acid as claimed in claim 3 or 4 which is functionally linked to one or more regulation signals to increase gene expression.<br>
6.	A nucleic acid construct as claimed in claim 5, wherein the nucleic acid as claimed in claim 3 or 4 is inserted into a vector which is suitable for expression of the gene in a prokaryotic or eukaryotic organism.<br>
7.	A genetically manipulated organism, where the gene expression of a nucleic acid as claimed in claim 3 or 4<br>
in the case where the initial organism comprises a nucleic acid as claimed in claim 3 or 4 is increased by comparison with a wild type or<br>
in the case where the initial organism does not comprise a nucleic acid as claimed in claim 3 or 4 is caused by the genetic manipulation.<br>
8.	A genetically manipulated organism as claimed in claim 7,<br>
whose carotenoid metabolism is different from that of the<br>
wild type.<br><br>
9.	A genetically manipulated organism, as claimed in claim 7 or 8, wherein the organism used is a eukaryotic organism.<br>
10.	A genetically manipulated organism as claimed in claim 9, wherein a plant is used as eukaryotic organism.<br>
11.	A process for producing genetically manipulated organisms as claimed in any of claims 7 to 10, which comprises introducing a nucleic acid as claimed in claim 3 or 4 or a nucleic acid construct as claimed in claim 5 or 6 into the genome of the initial organism.<br>
12.	The use of the nucleic acid as claimed in claim 3 or 4 for producing genetically manipulated organisms.<br>
13- A process for the preparation of xanthophyll derivatives,<br>
which comprises converting a β-ionone structural element into a 3-hydroxy-β-ionone structural element and/or a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element in the presence of a protein as claimed in either of claims 1 or 2.<br>
14.	A process for preparing xanthophyll derivatives as claimed in claim 13, wherein an organism as claimed in any of claims 7 to 10 is cultivated, the organism is harvested, and the xanthophyll derivatives are then isolated from the organism.<br>
15.	The use of the protein as claimed in claim 1 or 2 for preparing xanthophyll derivatives.<br>
16.	The use of the nucleic acid as claimed in claim 3 or 4 for preparing xanthophyll derivatives.<br><br><br><br>
We claim:<br>
1.	A protein which has an enzymatic activity for converting β-carotene into zeaxanthin or canthaxanthin into astaxanthin, comprising the amino acid sequence SEQ ID NO. 2 or a sequence derived from this sequence by substitution, insertion or deletion of amino acids and having a homology of at least 50% at the amino acid level with the sequence SEQ ID NO.2.<br>
2.	A protein as claimed in claim 1, wherein the protein has an enzymatic activity for converting p-carotene into zeaxanthin and an enzymatic activity for converting canthaxanthin into astaxanthin-<br>
3.	A nucleic acid encoding a protein as claimed in either of claims 1 or 2.<br>
4.	A nucleic acid as claimed in claim 3, which consists of the sequence depicted in SEQ ID NO. 1.<br>
5.	A nucleic acid construct comprising a nucleic acid as claimed in claim 3 or 4 which is functionally linked to one or more regulation signals to increase gene expression.<br>
6.	A nucleic acid construct as claimed in claim 5, wherein the nucleic acid as claimed in claim 3 or 4 is inserted into a vector which is suitable for expression of the gene in a prokaryotic or eukaryotic organism.<br>
7.	A genetically manipulated organism, where the gene expression of a nucleic acid as claimed in claim 3 or 4<br>
in the case where the initial organism comprises a nucleic acid as claimed in claim 3 or 4 is increased by comparison with a wild type or<br>
in the case where the initial organism does not comprise a nucleic acid as claimed in claim 3 or 4 is caused by the genetic manipulation.<br>
8.	A genetically manipulated organism as claimed in claim 7,<br>
whose carotenoid metabolism is different from that of the<br>
wild type.<br><br>
9.	A genetically manipulated organism, as claimed in claim 7 or 8, wherein the organism used is a eukaryotic organism.<br>
10.	A genetically manipulated organism as claimed in claim 9, wherein a plant is used as eukaryotic organism.<br>
11.	A process for producing genetically manipulated organisms as claimed in any of claims 7 to 10, which comprises introducing a nucleic acid as claimed in claim 3 or 4 or a nucleic acid construct as claimed in claim 5 or 6 into the genome of the initial organism.<br>
12.	The use of the nucleic acid as claimed in claim 3 or 4 for producing genetically manipulated organisms.<br>
13- A process for the preparation of xanthophyll derivatives,<br>
which comprises converting a β-ionone structural element into a 3-hydroxy-β-ionone structural element and/or a 4-keto-β-ionone structural element into a 3-hydroxy-4-keto-β-ionone structural element in the presence of a protein as claimed in either of claims 1 or 2.<br>
14.	A process for preparing xanthophyll derivatives as claimed in claim 13, wherein an organism as claimed in any of claims 7 to 10 is cultivated, the organism is harvested, and the xanthophyll derivatives are then isolated from the organism.<br>
15.	The use of the protein as claimed in claim 1 or 2 for preparing xanthophyll derivatives.<br>
16.	The use of the nucleic acid as claimed in claim 3 or 4 for preparing xanthophyll derivatives.<br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtY2xhaW1zIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZGVzY3JpcHRpb24oY29tcGxldGUpZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZm9ybSAxOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-form 19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ4Ni1jaGUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1486-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="211956-a-system-and-method-for-reducing-wireless-telecommunications-network-resources.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="211958-a-method-and-a-device-for-changing-shears-for-transverse-cutting-of-strips-or-sheet-metal-in-the-rolling-or-transport-line.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>211957</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1486/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-Jan-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Nov-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Oct-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF AKTIENGESELLSCHAFT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>67056 LUDWIGSHAFEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LINDEN, Hartmut</td>
											<td>St. Johanngasse 4, D-78462 Konstanz,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SANDMANN, Gerhard</td>
											<td>H. Steinhäuser Strasse 18, D-63065 Offenbach,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP00/02711</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-03-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>199 16 140.2</td>
									<td>1999-04-09</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/211957-a-process-for-preparing-a-protein-having-an-enzymatic-activity-for-converting-carotene-into-zeaxanthin-or-canthaxanthin-into-astaxanthin by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:53:21 GMT -->
</html>
