<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/212621-a-soluble-t-cell-receptor-and-a-method-for-obtaining-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 212621:A SOLUBLE T-CELL RECEPTOR AND A METHOD FOR OBTAINING THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A SOLUBLE T-CELL RECEPTOR AND A METHOD FOR OBTAINING THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a soluble T cell receptor (sTCR), which comprises (i) all for part of a TCR a chain, except the transmembrane domain thereof, and (ii) all or part of a TCR p chain, except the transmembrane domain thereof (i) and (ii) each comprises a functional variable domain and at least a part of the constant domain of the TCR chain, and are linked by a disulphide bond between constant domain residues which is not present in native TCR.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to soluble T cell receptors (TCRs).<br>
As is described in WO 99/60120, TCRs mediate the recognition qf specific Major Histocompatibility Complex (MHC)-peptide complexes by T cells and, as such, are essential to the functioning of the cellular arm of the immune system.<br>
Antibodies and TCRs are the only two types of molecules which recognise antigens in a specific manner, and thus the TCR is the only receptor for particular peptide antigens presented in MHC, the alien peptide often being the only sign of an ahnormality within a cell. T cell recognition occurs when a T-cell and an antigen presenting cell (APC) are in direct physical contact, and is initiated by ligation of antigen-specific TCRs withpMHC complexes.<br>
The TCR is a heterodimeric cell surface protein of the immunoglobulin superfamily which is associated with invariant proteins of the CD3 complex involved in mediating signal transduction. TCRs exist in a(3 and y5 forms, which are structurally similar but T cells expressing them have quite distinct anatomical locations and probably functions. The extracellular portion of the receptor consists of two membrane-proximal constant domains, and two membrane-distal variable domains bearing polymorphic loops analogous to the complementarity determining regions (CDRs) of antibodies. It is these loops which form the binding site of the TCR molecule and determine peptide specificity. The MHC class I and class II ligands are also immunoglobulin superfamily proteins but are specialised for antigen presentation, with a polymorphic peptide binding site which enables them to present a diverse array of short peptide fragments at the APC cell surface.<br>
Soluble TCRs are useful, not only for the purpose of investigating specific TCR-pMHC interactions, but also potentially as a diagnostic tool to detect infection, or to detect autoimmune disease markers. Soluble TCRs also have applications in staining, for example to stain cells for the presence of a particular peptide antigen presented in<br><br>
the context of the MHC. Similarly, soluble TCRs can be used to deliver a therapeutic agent, for example a cytotoxic compound or an immunostimulating compound, to cells presenting a particular antigen. Soluble TCRs may also be used to inhibit T cells, for example, those reacting to an auto-immune peptide antigen.<br>
Proteins which are made up of more than one polypeptide subunit and which have a transmembrane domain can be difficult to produce in soluble form because, in many cases, the protein is stabilised by its transmembrane region. This is the case for the TCR, and is reflected in the scientific literature which describes truncated forms of TCR, containing either only extracellular domains or extracellular and cytoplasmic domains, which can be recognised by TCR-specific antibodies (indicating that the part of the recombinant TCR recognised by the antibody has correctly folded), but which cannot be produced at a good yield, which are not stable at low concentrations and/or which cannot recognise MHC-peptide complexes. This literature is reviewed in WO 99/60120.<br>
A number of papers describe the production of TCR heterodimers which include the native disulphide bridge which connects the respective submits (Garboczi, et aL, (1996), Nature 384(6605): 134-41; Garboczi, et al., (1996), J Immunol 157(12): 5403-10; Chang et al, (1994), PNAS USA 91:11408-11412; Davodeau et aL, (1993), J. Biol. Chem. 268(21): 15455-15460; Golden et al., (1997), 1 Imm. Meth. 206:163-169; US Patent No. 6080840). However, although such TCRs can be recognised by TCR-specific antibodies, none were shown to recognise its native ligand at anything other than relatively high concentrations and/or were not stable.<br>
In WO 99/60120, a soluble TCR is described which is correctly folded so that it is capable of recognising its native ligand, is stable over a period of time, and can be produced in reasonable quantities. This TCR comprises a TCR a or y chain extracellular domain dimerised to a TCR p or 5 chain extracellular domain respectively, by means of a pair of C-terminal dimerisation peptides, such as leucine zippers. This strategy of producing TCRs is generally applicable to all TCRs.<br><br>
Reiter et al, Immunity, 1995, 2:281-287, details the construction of a soluble molecule comprising disulphide-stabilised TCR a and β variable domains, one of which is linked to a truncated form of Pseudomonas exotoxin (PE38). One of the stated reasons for producing this molecule was to overcome the inherent instability of single-chain TCRs. The position of the novel disulphide bond in the TCR variable domains was identified via homology with the variable domains of antibodies, into which these have previously been introduced (for example see Brinkrmann, et al. (1993), Proc. Natl. Acad. Sci. USA 90: 7538-7542, and Reiter, et al. (1994) Biochemistry 33: 5451- 5459). However, as there is no such homology between antibody and TCR constant domains, such a technique could not be employed to identify appropriate sites for new inter-chain disulphide bonds between TCR constant domains.<br>
Given the importance of soluble TCRs, it would be desirable to provide an alternative way of producing such molecules.<br>
Accordingly the present invention provides a soluble T cell receptor (sTCR), which comprises (i) all or part of a TCR a chain, except the transmembrane domain thereof, and (ii) all or part of a TCR(3 chain, except the transmembrane domain thereof, wherein (i) and (ii) each comprise a functional variable domain and at least a part of the constant domain of the TCR chain, characterised in that (i) and (ii) are linked by a disulphide bond between cysteine residues substituted for:<br>
Thr48ofexon 1 ofTRAC*01 and Ser 57 of exon 1 ofTRBCl*01 orTRBC2*01; Thr45 of exon 1 ofTRAC*01 and Ser 77 of exon 1 ofTRBCl*01 orTRBC2*01; Tyr 10 of exon 1 ofTRAC*01 and Ser 17 of exon 1 ofTRBCl*01 orTRBC2*01; Thr45ofexon 1 of TRAC*01 and Asp 59 of exon 1 ofTRBCl*01 orTRBC2*01; or Ser 15 of exon 1 ofTRAC*01 and Glu 15 of exon 1 ofTRBCl*01 orTRBC2*01;<br>
In another aspect, the invention provides a soluble aP-form T cell receptor (sTCR), wherein a covalent disulphide bond links a residue of the immunoglobulin region of the constant domain of the a chain to a residue of the immunoglobulin region of the constant domain of the p chain.<br>
The sTCRs of the present invention have the advantage that they do not contain<br>
heterologous polypeptides which may be immunogenic, or which may result in the sTCR<br>
being cleared quickly from the body. Furthermore, TCRs of the present invention have a<br>
three-dimensional structure which is highly similar to the native TCRs from which they are<br>
derived and, due to this structural similarity, they are not	<br><br>
likely to be immunogenic. sTCRs in accordance with the invention may be for recognising Class IMHC-peptide complexes or Class II MHC-peptide complexes.<br>
TCRs of the present invention are soluble. In the context of this application, solubility is defined as the ability of the TCR to be purified as a mono disperse heterodimer in phosphate buffered saline (PBS) (KCL 2.7mM, KH2P04 1.5mM, NaCl 137mM and Na2P04 8mM, pH 7.1-7.5. Life Technologies, Gibco BRL) at a concentration of lmg/ml and for &gt;90% of said TCR to remain as a mono disperse heterodimer after incubation at 25 °C for 1 hour. In order to assess the solubility of the TCR, it is first purified as.described in Example 2. Following this purification, 100μg of the TCR is analysed by analytical size exclusion chromatography e.g. using a Pharmacia Superdex 75 HR column equilibrated in PBS. A further 100μg of the TCR is incubated at 25°C for 1 hour and then analysed by size exclusion chromatography as before. The size exclusion traces are then analysed by integration and the areas under the peaks corresponding to the mono disperse heterodimer are compared. The relevant peaks may be identified by comparison with the elution position of protein standards of known molecular weight. The mono disperse heterodimeric soluble TCR has a molecular weight of approximately 50 kDa. As stated above, the TCRs of the present invention are soluble. However, as explained in more detail below, the TCRs can be coupled to a moiety such that the resulting complex is insoluble, or they may be presented on the surface of an insoluble solid support.<br>
The numbering of TCR amino acids used herein follows the IMGT system described in The T Cell Receptor Factsbook, 2001, LeFranc &amp; LeFranc, Academic Press. In this system, the a chain constant domain has the following notation: TRAC*01, where "TR" indicates T Cell Receptor gene; "A" indicates a chain gene; C indicates constant region; and "*01" indicates allele 1. The p chain constant domain has the following notation: TRBC1*01. hi this instance, there are two possible constant region genes "CI" and "C2". The translated domain encoded by each allele can be made up from the genetic code of several exons; therefore these are also specified. Amino acids are numbered according to the exon of the particular domain in which they are present.<br><br>
The extracellular portion of native TCR consists of two polypeptides ( or y$) each of which has a membrane-proximal constant domain, and a membrane-distal variable domain (see Figure 1). Each of the constant and variable domains includes an intra-chain disulphide bond. The variable domains contain the highly polymorphic loops analogous to the complementarity determining regions (CDRs) of antibodies. CDR3 of the TCR interacts with the peptide presented by MHC, and CDRs 1 and 2 interact with the peptide and the MHC. The diversity of TCR sequences is generated via somatic rearrangement of linked variable (V), diversity (D), joining (J), and constant genes. Functional a chain polypeptides are formed by rearranged V-J-C regions, whereas P chains consist of V-D-J-C regions. The extracellular constant domain has a membrane proximal region and an immunoglobulin region. The membrane proximal region consists of the amino acids between the transmembrane domain and the membrane proximal cysteine residue. The constant immunoglobulin domain consists of the remainder of the constant domain amino acid residues, extending from the membrane proximal cysteine to the beginning of the joining region, and is characterised by the presence of an immunoglobulin-type fold. There is a single a chain constant domain, known as Cal or TRAC*01, and two different p constant domains, known as Cpl orTRBCl*01 andCp2orTRBC2*01. The difference<br>
between these different p constant domains is in respect of amino acid residues 4,5<br>
F<br>
and 37 of exon 1. Thus, TRBC1*01 has 4N, 5K and 37/in exori 1 thereof, and TRBC2*01 has 4K, 5N and 37Y in exon 1 thereof. The extent of each of the TCR extracellular domains is somewhat variable.<br>
In the present invention, the disulphide bond is introduced between residues located in the constant domains (or parts thereof) of the respective chains. The respective chains of the TCR comprise sufficient of the variable domains thereof to be able to interact with its pMHC complex. Such interaction can be measured using a BIAcore 3000™ or BIAcore 2000™ instrument as described in Example 3 herein or in WO99/6120 respectively.<br><br>
In one embodiment, the respective chains of the sTCR of the invention also comprise the intra-chain disulphide bonds thereof- The TCR of the present invention may comprise all of the extracellular constant Ig region of the respective TCR chains, and preferably all of the extracellular domain of the respective chains, i.e. including the membrane proximal region. In native TCR, there is a disulphide bond linkingjhe conserved membrane proximal regions of the respective chains. In one embodiment of the present invention, this disulphide bond is not present. This may be achieved by mutating the appropriate cysteine residues (amino acid 4, exon 2 of the TRAC*01 gene and amino acid 2 of both the TRBC1*01 and TRBC2*01 genes respectively) to another amino acid, or truncating the respective chains so that the cysteine residues are not included. A preferred soluble TCR according to the invention comprises the native a and (3 TCR chains truncated at the C-terminus such that the cysteine residues which form the native interchain disulphide bond are excluded, i.e. truncated at the residue 1,2, 3,4,5,6,7, 8, 9 or 10 residues N-terminal to the cysteine residues. It is to be noted however that the native inter-chain disulphide bond may be present in TCRs of the present invention, and that, in certain embodiments, only one of the TCR chains has the native cysteine residue which forms the native interchain disulphide bond. This cysteine can be used to attach moieties to the TCR,<br>
However, the respective TCR chains may be shorter. Because the constant domains are not directly involved in contacts with the peptide-MHC ligands, the C-terminal truncation point maybe altered substantially without loss of functionality.<br>
Alternatively, a larger fragment of the constant domains maybe present than is preferred herein, i.e. the constant domains need not be truncated just prior to the cysteines forming the interchain disulphide bond. For instance, the entire constant domain except the transmembrane domain (i.e. the extracellular and cytoplasmic domains) could be included. It may be advantageous in this case to mutate one or more of the cysteine residues forming the interchain disulphide bond in the cellular TCR to another amino acid residue which is not involved in disulphide bond formation, or to delete one or more of these residues.<br><br>
The signal peptide may be omitted if the soluble TCR is to be expressed in prokaiyotic cells, for example Rcoli, since it does not serve any purpose in the mature TCR for its Hgand binding ability, and may in some circumstances prevent the formation of a functional soluble TCR. In most cases, the cleavage site at which the signal peptide is removed from the mature TCR chains is predicted but not experimentally determined. Engineering the expressed TCR chains such that they are a few, i.e. up to about 10 for example, amino acids longer or shorter at the N-terminal end may have no significance for the functionality (i.e. the ability to recognise pMHC) of the soluble TCR. Certain additions which are not present in the original protein sequence could be added. For example, a short tag sequence which can aid in purification of the TCR chains could be added, provided that it does not interfere with &gt; the correct structure and folding of the antigen binding site of the TCR.<br>
For expression in E.colt, a methionine residue may be engineered onto the N-terminal starting point of the predicted mature protein sequence in order to enable initiation of translation.<br>
Far from all residues in the variable domains of TCR chains are essential for antigen specificity and functionality. Thus, a significant number of mutations can be introduced in this domain without affecting antigen specificity and functionality.  Far from all residues in the constant domains of TCR chains are essential for antigen specificity and functionality. Thus, a significant number of mutations can be introduced in this region without affecting antigen specificity.<br>
The TCR (3 chain contains a cysteine residue which is impaired in the cellular or native TCR. It is preferred if this cysteine residue is removed or mutated to another residue to avoid incorrect intrachain or interchain pairing. Substitutions of this cysteine residue for another residue, for example serine or alanine, can have a significant positive effect on refolding efficiencies in vitro.<br>
The disulphide bond may be formed by mutating non-cysteine residues on the respective chains to cysteine, and causing the bond to be formed between the mutated<br><br>
residues. Residues whose respective p carbons are approximately 6 A (0.6 nra) or less, and preferably in the range 3.5 A (0.35 nm) to 5.9 A (0.59 nm) apart in the native TCR are preferred, such that a disulphide bond can be formed between cysteine residues introduced in place of the native residues. It is preferred if the disulphide bond is between residues in the constant immunoglobulin region, although it could be between residues of the membrane proximal region. Preferred sites where cysteines can be introduced to form the disulphide bond are the following residues in exon 1 of TRAC*01 for the TCR a chain and TRBC1*01 or TRBC2*01 for the TCR (3 chain:'.<br><br>
One sTCR of the present invention is derived from the A6 Tax TCR (Garboczi et al, Nature, 1996,384(6605): 134-141). In one embodiment, the sTCR comprises the whole of the TCR of chain which is N-tenninal of exon 2, residue 4 of TRAC*01 (amino acid residues 1-182 of the a chain according to the numbering used in Garboczi et al) and the whole of the TCR /3 chain which is N-tenninal of exon 2, residue 2 of both TRBC1*01 and TRCB2*01 (amino acid residues 1-210 of the 
It is to be noted that, in Figures 3a and 3b, residue 1 (according to the numbering used in Garboczi et al) is K and N respectively. The N-terminal methionine residue is not present in native A6 Tax TCR and, as mentioned above, is sometimes present when the respective chains are produced in bacterial expression systems.<br><br><br><br>
β Chain Glu 15:	VFPPEVAVF|PSEAEISHTQ(amino acids 6- 25 ofexon 1 of<br>
theTRBCl*01 &amp; TRBC2*01 genes)<br>
In other species, the TCR chains may not have a region which has 100% identity to the above motifs. However, those of skill in the art will be able to use the above motifs to identify the equivalent part of the TCR a or p chain and hence the residue to be mutated to cysteine. Alignment techniques may be used in this respect. For example, ClustalW, available on the European Bioinformatics Institute website (http://www.ebi.ac.uk/index.html) can be used to compare the motifs above to a particular TCR chain sequence in order to locate the relevant part of the TCR sequence for mutation.<br>
The present invention includes within its scope human disulphide-linked ap TCRs, as well as disulphide-linked aP TCRs of other mammals, including, but not limited to, mouse, rat, pig, goat and sheep. As mentioned above, those of skill in the art will be able to determine sites equivalent to the above-described human sites at which cysteine residues can be introduced to form an inter-chain disulphide bond. For example, the following shows the amino acid sequences of the mouse Ca and C/S soluble domains, together with motifs showing the murine residues equivalent to the human residues mentioned above that can be mutated to cysteines to form a TCR interchain disulphide bond (where the relevant residues are shaded):<br>
Mouse Ca soluble domain:<br>
PYIQNPEPAVYQLKDPRSQDSTLCLFTDFDSQI KSNGAIAWSNQTSFTCQDIFKETNATYPS SDVP<br>
Mouse Cj8 soluble domain:<br><br>
DPQAYKESNYSYCLSSRLRVSATFWHNPRNHFRCQVQFHGLSEEDKWPEGSPKPVTQN<br>
ISAEAWGRAD<br>
Murine equivalent of human a Chain Thr 48: ESGTFrTDKfVLDMKAMDSK<br><br>
Murine equivalent of human a Chain Thr 45: KTMESGTFI|DKTVLDMKAM<br>
Murine equivalent of human a Chain Tyr 10: YIQNPEPA\^QLKDPRSQDS<br>
Murine equivalent of human a Chain Ser 15: AVYQLKDPR|QDSTLCLFTD<br>
Murine equivalent of human p Chain Ser 57; NGREVHSGVfrDPQAYKESN<br>
Murine equivalent of human p Chain Ser 77; KESNYSYClJfSRIJlVSATFW<br>
Murine equivalent of human p Chain Ser 17: PPKVSLFEP|KAEIANKQKA<br>
Murine equivalent of human p Chain Asp 59: REVHSGVSlfPQAYKESNYS<br>
Murine equivalent of human p Chain Glu 15:.VTPPKVSLFfPSKAEIANKQ<br>
In a preferred embodiment of the present invention, (i) and (ii) of the TCR each comprise the functional variable domain of a first TCR fused to all or part of the constant domain of a second TCR, the first and second TCRs being from the same species and the inter-chain disulphide bond being between residues in said respective all or part of the constant domain not present in native TCR. In one embodiment, the first and second TCRs are human. In other words, the disulphide bond-hnked constant domains act as a framework on to which variable domains can be fused. The resulting TCR will be substantially identical to the native TCR from which the first TCR is obtained. Such a system allows the easy expression of any functional variable domain on a stable constant domain framework.<br>
The constant domains of the A6 Tax sTCR described above, or indeed the constant domains of any of the mutant ap TCRs having a new interchain disulphide bond described above, can be used as framework onto which heterologous variable domains<br><br>
can be fused. It is preferred if the fusion protein retains as much of the conformation of the heterologous variable domains as possible. Therefore, it is preferred that the heterologous variable domains are linked to the constant domains at any point between the introduced cysteine residues and the N terminus of the constant domain. For the A6 Tax TCR, the introduced cysteine residues on the a and P chains are preferably located at threonine 48 of exon 1 in TRAC*01 (threonine 158 of the a chain according . to the numbering used in Garboczi et al) and serine 57 of exon 1 in both TRBC1*01 and TRBC2*01 (serine 172 of the P chain according to the numbering used in Garboczi et al) respectively. Therefore it is preferred if the heterologous a and p chain variable domain attachment points are between residues 48 (159 according to the numbering used in Garboczi et al) or 58 (173 according to the numbering used in Garboczi et al) and the N terminus of the a or p constant domains respectively.<br>
The residues in the constant domains of the heterologous a and (5 chains corresponding to the attachment points in the A6 Tax TCR can be identified by sequence homology. The fusion protein is preferably constructed to include all of the heterologous sequence N-terminal to the attachment point.<br>
As is discussed in more detail below, the sTCR of the present invention may be derivatised with, or fused to, a moiety at its C or N terminus. The C terminus is preferred as this is distal from the binding domain. In one embodiment, one or both of the TCR chains have a cysteine residue at its C and/or N terminus to which such a moiety can be fused.<br>
A soluble TCR (which is preferably human) of the present invention may be provided in substantially pure form, or as a purified or isolated preparation. For example, it may be provided in a form which is substantially free of other proteins.<br>
A plurality of soluble TCRs of the present invention may be provided in a multivalent complex. Thus, the present invention provides, in one aspect, a multivalent T cell receptor (TCR) complex, which comprises a plurality of soluble T cell receptors as described herein.  Each of the plurality of soluble TCRs is preferably identical.<br><br>
-In.another aspect, the invention provides a method for detecting MHC-peptide complexes which method comprises:<br>
(i)       providing a soluble T cell receptor or a multivalent T cell receptor complex as described herein;<br>
(ii)      contacting the soluble T cell receptor or multivalent TCR complex with the MHC-peptide complexes; and<br>
(iii)     detecting binding of the soluble T cell receptor or multivalent TCR complex to the MHC-peptide complexes.<br>
In the multivalent complex of the present invention, the TCRs may be in the form of multimers, and/or may be present on or associated with a lipid bilayer, for example, a liposome.<br>
In its simplest form, a multivalent TCR complex according to the invention comprises a multimer of two or three or four or more T cell receptor molecules associated (e.g. covalently or otherwise linked) with one another, preferably via a linker molecule. Suitable linker molecules include, but are not limited to, multivalent attachment molecules such as avidin, streptavidin, neutravidin and extravidin, each of which has four binding sites for biotin. Thus, biotinylated TCR molecules can be formed into multimers of T cell receptors having a plurality of TCR binding sites. The number of TCR molecules in the multimer will depend upon the quantity of TCR in relation to the quantity of linker molecule used to make the multimers, and also on the presence or absence of any other biotinylated molecules. Preferred multimers are dimeric, trimeric or tetrameric TCR complexes.<br>
Structures which are a good deal larger than TCR tetramers may be used in tracking or targeting cells expressing specific MHC-peptide complex. Preferably the structures are in the range 1 Onm to 10μm in diameter. Each structure may display multiple TCR molecules at a sufficient distance apart to enable two or more TCR molecules on the structure to bind simultaneously to two or more MHC-peptide complexes on a cell and thus increase the avidity of the multimeric binding moiety for the cell.<br><br>
Suitable structures for use in the invention include membrane structures such as<br>
liposomes and solid structures which are preferably particles such as beads, for<br>
example latex beads. Other structures which may be externally coated with T cell<br>
receptor molecules are also suitable. Preferably, the structures are coated with T cell<br>
receptor multimers rather than with individual T cell receptor molecules.<br>
&gt;<br>
In the case of liposomes, the T cell receptor molecules or multimers thereof may be attached to or otherwise associated with the membrane.   Techniques for this are well known to those skilled in the art.<br>
A label or another moiety, such as a toxic or therapeutic moiety, may be included in a multivalent TCR complex of the present invention. For example, the label or other moiety may be included in a mixed molecule multimer. An example of such a multimeric molecule is a tetramer containing three TCR molecules and one peroxidase molecule. This could be achieved by mixing the TCR and the enzyme at a molar ratio of 3:1 to generate tetrameric complexes, and isolating the desired complex from any complexes not containing the correct ratio of molecules. These mixed molecules could contain any combination of molecules, provided that steric hindrance does not compromise or does not significantly compromise the desired function of the molecules. The positioning of the binding sites on the streptavidin molecule is suitable for mixed tetramers since steric hindrance is not likely to occur.<br>
Alternative means of biotinylating the TCR maybe possible. For example, chemical biotinylation may be used. Alternative biotinylation tags may be used, although certain amino acids in the biotin tag sequence are essential (Schatz, (1993). Biotechnology NY11(10): 1138-43). The mixture used for biotinylation may also be varied. The enzyme requires Mg-ATP and low ionic strength, although both of these conditions may be varied e.g. it may be possible to use a higher ionic strength and a longer reaction time. It may be possible to use a molecule other than avidin or streptavidin to form multimers of the TCR. Any molecule which binds biotin in a multivalent manner would be suitable. Alternatively, an entirely different linkage<br><br>
could be devised (such as poly-histidine tag to chelated nickel ion (Quiagen Product Guide 1999, Chapter 3 "Protein Expression, Purification, Detection and Assay" p. 35-37). Preferably, the tag is located towards the C-terminus of the protein so as to minimise the amount of steric hindrance in the interaction with peptide-MHC complexes.<br>
One or both of the TCR chains may be labelled with a detectable label, for example a label which is suitable for diagnostic purposes. Thus, the invention provides a method for detecting MHC-peptide complexes which method comprises contacting the MHC-peptide complexes with a TCR or multimeric TCR complex in accordance with the invention which is specific for the MHC-peptide complex; and detecting binding of the TCR or multimeric TCR complex to the MHC-peptide complex. In tetrameric TCR formed using biotinylated heterodimers, fluorescent streptavidin (commercially available) can be used to provide a detectable label. A fluorescently-labelled tetramer is suitable for use in FACS analysis, for example to detect antigen presenting cells carrying the peptide for which the TCR is specific.<br>
Another manner in which the soluble TCRs of the present invention may be detected is by the use of TCR-specific antibodies, in particular monoclonal antibodies. There are many commercially available anti-TCR antibodies, such as aFl and PF1, which recognise the constant regions of the a and p chain, respectively.<br>
The TCR (or multivalent complex thereof) of the present invention may alternatively or additionally be associated with (e.g. covalently or otherwise linked to) a therapeutic agent which may be, for example, a toxic moiety for use in cell killing, or an immunostimulating agent such as an interleukin or a cytokine. A multivalent TCR complex of the present invention may have enhanced binding capability for a pMHC compared to a non-multimeric T cell receptor heterodimer. Thus, the multivalent TCR complexes according to the invention are particularly useful for tracking or targeting cells presenting particular antigens in vitro or in vivo, and are also useful as intermediates for the production of further multivalent TCR complexes having such<br><br>
uses. The TCR or multivalent TCR complex may therefore be provided in a<br>
pharmaceutical acceptable formulation for use in vivo.	<br>
The invention also provides a method for delivering a therapeutic agent to a target cell, which method comprises contacting potential target cells with a TCR or multivalent TCR complex in accordance with the invention under conditions to allow attachment of the TCR or multivalent TCR complex to the target cells said TCR or multivalent TCR complex being specific for the MHC-peptide complexes and having the therapeutic agent associated therewith.<br>
In particular, the soluble TCR or multivalent TCR complex can be used to deliver therapeutic agents to the location of cells presenting a particular antigen. This would be useful in many situations and, in particular, against tumours. A therapeutic agent could be delivered such that it would exercise its effect locally but not only on the cell it binds to. Thus, one particular strategy envisages anti-tumour molecules linked to T cell receptors or multivalent TCR complexes specific for tumour antigens.<br>
Many therapeutic agents could be employed for this use, for instance radioactive compounds, enzymes (perforin for example) or chemotherapeutic agents (cis-platin for example). To ensure that toxic effects are exercised in the desired location the toxin could be inside a liposome linked to streptavidin so that the compound is released slowly. This will prevent damaging effects during the transport in the body and ensure that the toxin has maximum effect after binding of the TCR to the relevant antigen presenting cells.<br>
Other suitable therapeutic agents include:<br>
•	small molecule cytotoxic agents, i.e. compounds with the ability to kill<br>
mammalian cells having a molecular weight of less than 700 daltons. Such compounds could also contain toxic metals capable of having a cytotoxic effect. Furthermore, it is to be understood that these small molecule cytotoxic agents also include pro-drugs, i.e. compounds that decay or are converted under physiological conditions to release cytotoxic agents. Examples of such agents include cis-platin,<br><br>
maytansine derivatives, rachelmycin, calicheamicin, docetaxel, etoposide, gemcitabine, ifosfamide, irinotecan, melphalan, mitoxantrone, sorfimer sodiumphotofrin II, temozolmide, topotecan, trimetreate glucuronate, auristatin E vincristine and doxorubicin;<br>
•	peptide cytotoxins, i.e. proteins or fragments thereof with the ability to kill mammalian cells. Examples include ricin, diphtheria toxin, pseudomonas bacterial exotoxin A, DNAase and KNAase;<br>
•	radio-nuclides, i.e. unstable isotopes of elements which decay with the concurrent emission of one or more of a or P particles, or y rays. Examples include iodine 131,rhenium 186, indium 111, yttrium 90, bismuth 210 and 213, actinium 225 and astatine 213;<br>
•	prodrugs, such as antibody directed enzyme pro-drugs;<br>
•	immuno-stimulants, i.e. moieties which stimulate immune response. Examples include cytokines such as IL-2, chemokines such as BL-8, platelet factor 4, melanoma growth stimulatory protein, etc, antibodies or fragments thereof, complement activators, xenogeneic protein domains, allogeneic protein domains, viral/bacterial protein domains and viral/bacterial peptides.<br>
Soluble TCRs or multivalent TCR complexes of the invention may be linked to an enzyme capable of converting a prodrug to a drug. This allows the prodrug to be converted to the drug only at the site where it is required (i.e. targeted by the sTCR).<br>
Examples of suitable MHC-peptide targets for the TCR according to the invention include, but are not limited to, viral epitopes such as HTLV-1 epitopes (e.g. the Tax peptide restricted by HLA-A2; HTLV-1 is associated with leukaemia), HTV epitopes, EBV epitopes, CMV epitopes; melanoma epitopes (e.g. MAGE-1 HLA-Al restricted epitope) and other cancer-specific epitopes (e.g. the renal cell carcinoma associated antigen G250 restricted by HLA-A2); and epitopes associated with autoimmune disorders, such as rheumatoid arthritis. Further disease-associated pMHC targets, suitable for use in the present invention, are listed in the HLA Factbook (Barclay (Ed) Academic Press), and many others are being identified.<br><br>
A multitude of disease treatments can potentially be enhanced by localising the drug through the specificity of soluble TCRs.<br>
Viral diseases for which drugs exist, e.g. HIV, SIV, EBV, CMV, would benefit from the drug being released or activated in the near vicinity of infected cells. For cancer, the localisation in the vicinity of tumours or metastasis would enhance the effect of toxins or immunostimulants. In autoimmune diseases, immunosuppressive drugs could be released slowly, having more local effect over a longer time-span while minimally affecting the overall immuno-capacity of the subject. In the prevention of graft rejection, the effect of immunosuppressive drugs could be optimised in the same way. For vaccine delivery, the vaccine antigen could be localised in the vicinity of antigen presenting cells, thus enhancing the efficacy of the antigen. The method can also be applied for imaging purposes.<br>
The soluble TCRs of the present invention maybe used to modulate T cell activation by binding to specific pMHC and thereby inhibiting T cell activation. Autoimmune diseases involving T cell-mediated inflammation and/or tissue damage would be amenable to this approach, for example type I diabetes. Knowledge of the specific peptide epitope presented by the relevant pMHC is required for this use.<br>
Medicaments in accordance with the invention will usually be supplied as. part of a sterile, pharmaceutical composition which will normally include a pharmaceutically acceptable carrier. This pharmaceutical composition may be in any suitable form, (depending upon the desired method of administering it to a patient). It may be provided in unit dosage form, will generally be provided in a sealed container and may be provided as part of a kit Such a kit would normally (although not necessarily) include instructions for use. It may include a plurality of said unit dosage forms.<br>
The pharmaceutical composition may be adapted for administration by any appropriate route, for example by the oral (including buccal or sublingual), rectal, nasal, topical (including buccal, sublingual or transdermal), vaginal or parenteral (including subcutaneous, intramuscular, intravenous or intradermal) route. Such compositions may<br><br>
De prepared by any method known in the art of pharmacy, for example by admixing the active ingredient with the carrier(s) or excipient(s) under sterile conditions.<br>
Pharmaceutical compositions adapted for oral administration may be presented as discrete units such as capsules or tablets; as powders or granules; as solutions, syrups or suspensions (in aqueous or non-aqueous liquids; or as edible foams or whips; or as emulsions). Suitable excipients for tablets or hard gelatine capsules include lactose, maize starch or derivatives thereof, stearic acid or salts thereof.' Suitable excipients for use with soft gelatine capsules include for example vegetable oils, waxes, fats, semisolid, or liquid polyols etc.<br>
For the preparation of solutions and syrups, excipients which maybe used include for example water, polyols and sugars. For the preparation of suspensions oils (e.g. vegetable oils) maybe used to provide oil-in-water or water in oil suspensions. Pharmaceutical compositions adapted for transdermal administration maybe presented as discrete patches intended to remain in intimate contact with the epidennis of the recipient for a prolonged period of time. For example, the active ingredient may be delivered from the patch by iontophoresis as generally described in Pharmaceutical Research, 3(6):318 (1986). Pharmaceutical compositions adapted for topical administration may be formulated as ointments, creams, suspensions, lotions, powders, solutions, pastes, gelss sprays, aerosols or oils. For infections of the eye or other external tissues, for example mouth and skin, the compositions are preferably applied as a topical ointment or cream. When formulated in an ointment, the active ingredient may be employed with either a paraffinic or a water-miscible ointment base. Alternatively, the active ingredient maybe formulated in a cream with an oil-in-water cream base or a water-in-oil base. Pharmaceutical compositions adapted for topical administration to the eye. include eye drops wherein the active ingredient is dissolved or suspended in a suitable carrier, especially an aqueous solvent. Pharmaceutical compositions adapted for topical administration in the mouth include lozenges, pastilles and mouth washes.<br>
Pharmaceutical compositions adapted for rectal administration may be presented as suppositories or enemas. Pharmaceutical compositions adapted for nasal administration<br><br>
wherein the carrier is a solid include a coarse powder having a particle size for example in the range 20 to 500 microns which is administered in the manner in which snuff is taken, i.e. by rapid inhalation through the nasal passage from a container of the powder held close up to the nose. Suitable compositions wherein the carrier is a liquid, for administration as a nasal spray or as nasal drops, include aqueous or oil solutions of the active ingredient. Pharmaceutical compositions adapted for administration by inhalation include fine particle dusts or mists which may be generated by means of various types of metered dose pressurised aerosols, nebulizers or insufflators. Pharmaceutical compositions adapted for vaginal administration may be presented as pessaries, tampons, creams, gels, pastes, foams or spray formulations. Pharmaceutical compositions adapted for parenteral administration include aqueous and non-aqueous sterile injection solution which may contain anti-oxidants, buffers, bacteriostats and solutes which render the formulation substantially isotonic with the blood of the intended recipient; and aqueous and non-aqueous sterile suspensions which may include suspending agents and thickening agents. Excipients which maybe used for injectable solutions include water, alcohols, polyols, glycerine and vegetable oils, for example. The compositions maybe presented in unit-dose or multi-dose containers, for example sealed ampoules and vials, and may be stored in a freeze-dried (lyophilized) condition requiring only the addition of the sterile liquid carried, for example water for injections, immediately prior to use. Extemporaneous injection solutions and suspensions maybe prepared from sterile powders, granules and tablets.<br>
The pharmaceutical compositions may contain preserving agents, solubilising agents, stabilising agents, wetting agents, emulsifiers, sweeteners, colourants, odourants, salts (substances of the present invention may themselves be provided in the foim of a pharmaceutically acceptable salt), buffers, coating agents or antioxidants. They may also contain therapeutically active agents in addition to the substance of the present invention.<br>
Dosages of the substances of the present invention can vary between wide limits, depending upon the disease or disorder to be treated, the age and condition of the individual to be treated, etc. and a physician will ultimately determine appropriate dosages to be used. The dosage may be repeated as often as appropriate. If side effects<br><br>
develop the amount and/or frequency of the dosage can be reduced, in accordance with normal clinical practice.<br>
Gene cloning techniques may be used to provide a sTCR of the invention, preferably in substantially pure form. These techniques are disclosed, for example, in J. Sambrook et al Molecular Cloning 2nd Edition, Cold Spring Harbor Laboratory Press (1989). Thus, in a further aspect, the present invention provides a nucleic acid molecule comprising a sequence encoding a chain of the soluble TCR of the present invention, or a sequence complementary thereto. Such nucleic acid sequences may be obtained by isolating TCR-encoding nucleic acid from T-cell clones and making appropriate mutations (by insertion, deletion or substitution).<br>
The nucleic acid molecule may be in isolated or recombinant form. It may be incorporated into a vector and the vector may be incorporated into a host cell. Such vectors and suitable hosts form yet further aspects of the present invention.<br>
The invention also provides a method for obtaining a TCR chain, which method comprises incubating such a host cell under conditions causing expression of the TCR chain and then purifying the polypeptide.<br>
Accordingly the present invention also provides a method for obtaining a soluble T cell receptor (sTCR), which method comprises:<br>
incubating a host cell which comprises a vector comprising a nucleic acid molecule encoding (i) all or part of a TCR a chain, except the transmembrane domain thereof, and a host cell which comprises a vector comprising a nucleic acid molecule encoding (ii) all or part of a TCR p chain, except the transmembrane domain thereof under conditions causing expression of (i) and (ii) wherein (i) and (ii) each comprise a functional variable domain .and at least a part of the constant domain of the TCR chain;<br>
purifying (i) and (ii); and<br><br>
mixing (i) and (ii) under refolding conditions such that they are linked by a disulphide bond between constant domain residues which is not present in native TCR..<br>
The soluble TCRs of the present invention may obtained by expression in a bacterium such as E. coli as inclusion bodies, and subsequent refolding in vitro.<br>
Refolding of the TCR chains may take place in vitro under suitable refolding conditions. In a particular embodiment, a TCR with correct conformation is achieved by refolding solubilised TCR chains in a refolding buffer comprising a solubilising agent, for example urea. Advantageously, the urea may be present at a concentration of at least O.IM or at least IM or at least 2.5M, or about SM. An alternative solubilising agent which may be used is guanidine, at a concentration of between O.IM and 8M, preferably at least IM or at least 2.5M. Prior to refolding, a reducing agent is preferably employed to ensure complete reduction of cysteine residues. Further denaturing agents such as DIT and guanidine may be used as necessary. Different<br><br>
denaturants and reducing agents may be used prior to the refolding step (e.g. urea, p-mercaptoethanol). Alternative redox couples maybe used during refolding, such as a cystamine/cysteamine redox couple, DTT or P-mercaptoethanol/atmospheric oxygen, and cysteine in reduced and oxidised forms.<br>
Folding efficiency may also be increased by the addition of certain other protein components, for example chaperone proteins, to the refolding mixture. Improved refolding has been achieved by passing protein through columns with immobilised mini-chaperones (Altamirano, et aL (1999). Nature Biotechnology 17:187-191; Altamirano, etal (1997). Proc Nail Acad Sci USA 94(8): 3576-8).<br>
Alternatively, soluble TCR the present invention may obtained by expression in a eukaryotic cell system, such as insect cells.<br>
Purification of the TCR may be achieved by many different means. Alternative modes of ion exchange may be employed or other modes of protein purification may be used such as gel filtration chromatography or affinity chromatography.<br>
Soluble TCRs and multivalent TCR complexes of the present invention also find use in screening for agents, such as small chemical compounds, which have the ability to inhibit the binding of the TCR to its pMHC complex. Thus, in a further aspect, the present invention provides a method for screening for an agent which inhibits the binding of a T cell receptor to a peptide-MHC complex, comprising monitoring the binding of a soluble T cell receptor of the invention with a peptide-MHC complex in the presence of an agent; and selecting agents which inhibit such binding.<br>
Suitable techniques for such a screening method include the Surface Plasmon Resonance-based method described in WO 01/22084. Other well-known techniques that could form the basis of this screening method are Scintillation Proximity Analysis (SPA) and Amplified Luminescent Proximity Assay.<br><br>
Agents selected by screening methods of the invention can be used as drugs, or as the basis of a drug development programme, being modified or otherwise improved to have characteristics making them more suitable for administration as a medicament. Such medicaments can be used for the treatment of conditions which include an unwanted T cell response component. Such conditions include cancer (e.g. renal, ovarian, bowel, head &amp; neck, testicular, lung, stomach, cervical, bladder, prostate or melanoma), autoimmune disease, graft rejection and graft versus host disease.<br>
Preferred features of each aspect of the invention are as for each of the other aspects mutatis mutandis. The prior art documents mentioned herein are incorporated to the fullest extent permitted by law.<br>
Examples<br>
The invention is further described in the following examples, which do not limit the scope of the invention in any way.<br>
Reference is made in the following to the accompanying drawings in which:<br>
Figure 1 is a schematic diagram of a soluble TCR with an introduced inter-chain disulphide. bond in accordance with the invention;<br>
Figures 2a and 2b show respectively the nucleic acid sequences of the a and p. chains of a soluble A6 TCR, mutated so as to introduce a cysteine codon. The shading<br>
indicates the introduced cysteine codon;<br>
	—-	~——,_-_._....,..,,<br>
Figure 3a shows the A6 TCR a chain extracellular amino acid sequence, including the T48 —&gt; C mutation (underlined) used to produce the novel disulphide inter-chain bond, and Figure 3b shows the A6 TCR P chain extracellular amino acid sequence, including the S57 -&gt; C mutation (underlined) used to produce the novel disulphide inter-chain bond;<br><br>
Figure 4 i^trace obtained after anion exchange chromatography of soluble A6 TCR, showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 5 y^C. Reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 4, as indicated. B. Non-reducing SDS-PAGE (Cpomassie-stained) of fractions from column run in Figure 4, as indicated. Peak 1 clearly contains mainly non-disulphide linked (3-chain, peak 2 contains TCR heterodimer which is inter-chain disulphide linked, and the shoulder is due XoE;coli contaminants, mixed in with the inter-chain disulphide linked sTCR, which are poorly visible on this reproduction;<br>
Figure $ is a trace obtained from size-exclusion chromatography of pooled fractions from peak 1 in Figure 5. The protein elutes as a single major peak, corresponding to the heterodimer;<br>
Figure 7 is a BIAcore response curve of the specific binding of disulphide-Iinked A6 soluble TCR to HLA-A2-tax complex. Insert shows binding response compared to control for a single injection of disulphide-Iinked A6 soluble TCR;<br>
Figure 8a shows the A6 TCR a chain sequence including novel cysteine residue mutated to incorporate a BamHl restriction site. Shading indicates the mutations introduced to form the BamHl restriction site. Figures 8b and 8c show the DNA<br>
sequence of a and p chain of the JM22 TCR mutated to include additional cysteine<br>
^^-~..^,...       — *—•*<br>
residues to form a non-native disulphide bond;<br>
Figures 9a and 9b show respectively the JM22 TCR a and (3 chain extracellular amino acid sequences produced fromJhe DNA sequences of Figures 8a and 8b;<br>
Figure 10 is a trace obtained after anion exchange chromatography of soluble<br>
^	<br>
disulphide-Iinked JM22 TCR showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br><br>
Figure 11a shows a reducing SDS-PAGE (Coomassie-stained) of fractions from . column run in Figure 10, as indicated and Figure 1 lb shows a non-reducing.SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 10, as indicated. Peak 1 clearly contains TCR heterodimer which is inter-chain disulphide linked.<br>
Figure 12 is a trace obtained from size-exclusion chromatography of pooled fractions from peak 1 in figure 10. The protein elutes as a single major peak, corresponding to the heterodimer. Yield is 80%;<br>
Figure 13 - A. BIAcore response curve of the specific binding of disulphide-linked JM22 soluble TCR to HLA-Flu complex. B. Binding response compared to control for a single injection of disulphide-linked JM22 soluble TCR;<br>
Figures 14a and 14b show the DNA sequence of a and p chain of the NY-ESO mutated to include additional cysteine.residues to form a non-native disulphide bond;<br>
Figures 15a and 15b show respectively the NY-ESO TCR a and p chain extracellular amino acid sequences produced from the DNA sequences of Figures 14a and 14b<br>
Figure 16 is a trace obtained from anion exchange chromatography of soluble NY-ESO disulphide-linked TCR showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 17 -A. Reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 16, as indicated. B. Non-reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 16, as indicated. Peak 1 and 2 clearly contain TCR heterodimer which is inter-chain disulphide linked;<br>
Figure 18. Size-exclusion chromatography of pooled fractions from peak 1 (A) and peak 2 (B) in figure 17. The protein elutes as a single major peak, corresponding to the heterodimer;<br><br>
Figure 19 shows a BIAcore response curve of the specific binding of disulphide-linked NY-ESO soluble TCR to HLA-NYESO complex. A. peak 1, B. peak 2;<br>
Figures 20a and 20b show respectively the DNA sequences of the a and p chains of a soluble NY-ESO TCR, mutated so as to introduce a novel cysteine cocjon (indicated by shading). The sequences include the cysteine involved in the native disulphide inter-chain bond (indicated by the codon in bold);<br>
Figures 21a and 21b show respectively the NY-ESO TCR a and p chain extracellular amino acid sequences produced from the DNA sequences of Figures 20a and 21b;<br>
Figure 22 shows a trace obtained from anion exchange chromatography of soluble NY-ESO TCRoP fiys showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 23 shows a trace obtained from anion exchange chromatography of soluble NY-ESO TCRGP showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 24 shows a trace obtained from anion exchange chromatography of soluble NY-ESO TCR^8 showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 25 shows a reducing SDS-PAGE (Coomassie-stained) of NY-ESO TCRcP /P, TCRof **, and TCR/S03* fractions from anion exchange column runs in Figures 22-24 respectively. Lanes 1 and 7 are MW markers, lane 2 is NYESOdsTCRlg4 a-cys )3 peak (EB/084/033); lane 3 is NYESOdsTCRlg4 a-cys j8 small peak (EB/084/033), lane 4 isNYESOdsTCRlg4 a /3-cys (EB/084/034), lane 5 is NYESOdsTCRlg4 a-cys )3-cys small peak (EB/084/035), and lane 6 is NYESOdsTCRlg4 a-cys 0-cys peak (EB/084/035);<br><br>
Figure 26 shows a non-reducing SDS-PAGE (Coomassie-stained) of NY-ESO TCRcP jS0*8, TCRof &gt;*, and TCRjS0*8 fractions from anion exchange column runs in Figures 22-24 respectively. Lanes 1 and 7 are MW markers, lane 2 is NYESOdsTCRlg4 a-cys (3 peak (EB/084/033); lane 3 is NYESOdsTCRlg4 a-cys fi small peak (EB/084/033), lane 4 is NYESOdsTCRlg4 a (3-cys (EB/084/034), lane 5 is NYESOdsTCRlg4 a-cys j3-cys small peak (EB/084/035), and lane 6 is NYESOdsTCRlg4 a-cys 0-cys peak (EB/084/035);<br>
Figure 27 is a trace obtained from size exclusion exchange chromatography of soluble NY-ESO TCRJDP6 0^ showing protein elution of pooled, fractions from Figure 22. The protein elutes as a single major peak, corresponding to the heterodimer;<br>
Figure 28 is a trace obtained from size exclusion exchange chromatography of soluble NY-ESO TCRcP* showing protein elution of pooled fractions from Figure 22. The protein elutes as a single major peak, corresponding to the heterodimer,<br>
Figure 29 is a trace obtained from size exclusion exchange chromatography of soluble NY-ESO TCR^ showing protein elution of pooled fractions from Figure 22. The protein elutes as a single major peak, corresponding to the heterodimer;<br>
Figure 30 is a BIAcore response curve of the specific binding of NY-ESO TCRof^ /f5" to HLA-NY-ESO complex;<br>
Figure 31 is a BIAcore response curve of the specific binding of NY-ESO TCRoP5 to HLA-NY-ESO complex;<br>
Figure 32 is a BIAcore response curve of the specific binding of NY-ESO TCRjS0^ to HLA-NY-ESO complex;<br>
Figures 33a and 33b show respectively the DNA sequences of the a and P chains of a soluble AH-1.23 TCR, mutated so as to introduce a novel cysteine codon (indicated by<br><br>
snadingj. "I'Jie sequences include the cysteine involved in the native disulphide interchain bond (indicated by the codon in bold);<br>
Figures 34a and 34b show respectively the AH-1.23 TCR a and p chain extracellular amino acid sequences produced from the DNA sequences of Figures 33a and 33b;<br>
Figure 35 is a trace obtained from anion exchange chromatography of soluble AH-1.23 TCR showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 36 is a reducing SDS-PAGE (10% Bis-Tris gel, Coomassie-stained) of AH-1.23 TCR fractions from anion exchange column run in Figure 35. Proteins examined are the anion exchange fractions of TCR 1.23 S-S from refold 3. Lane 1 is MW markers, lane 2 is B4, lane 3 is C2, lane 4 is C3, lane 5 is C4, lane 6 is C5, lane 7 is C6, lane 8 is C7, lane 9 is C8, and lane 10 is C9;<br>
Figure 37 is anon-reducing SDS-PAGE (10% Bis-Tris gel, Coomassie-stained) of AH-1.23 TCR fractions from anion exchange column run in Figure 35. Proteins examined are the anion exchange fractions of TCR 1.23 S-S from refold 3. Lane 1 is MW markers, lane 2 is B4, lane 3 is C2, lane 4 is C3, lane 5 is C4, lane 6 is C5, lane 7 is C6&gt; lane 8 is C7, lane 9 is C8, and lane 10 is C9Figure 38 is a trace obtained from size exclusion exchange chromatography of soluble AH-1.23 TCR showing protein elution of pooled fractions from Figure 35. The protein elutes as a single major peak, corresponding to the heterodimer;<br>
Figures 39a and 39b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 48 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br><br>
Figures 40a and 40b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 45 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 41a and 41b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 61 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 42a and 42b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 50 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 43a and 43b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 10 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 44a and 44b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 15 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 45a and 45b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 12 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br><br>
Figures 46a and 46b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 22 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
* Figures 47a-and 47b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 52 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 48a and 48b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 43 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 49a and 49b show respectively the DNA and amino acid sequences of the a chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 57 in exon 1 of TRAC*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 50a and 50b show respectively the DNA and amino acid sequences of the p chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 77 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 51a and 51b show respectively the DNA and amino acid sequences of the (3 chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 17 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br><br>
Figures 52a and 52b show respectively the DNA and amino acid sequences of the P chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 13 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 53a and 53b show respectively the DNA and amino acid sequences of the P chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 59 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 54a and 54b show respectively the DNA and amino acid sequences of the P chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 79 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the "underlined amino acid indicates the introduced cysteine;<br>
Figures 55a and 55b show respectively the DNA and amino acid sequences of the P chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 14 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 56a and 56b show respectively the DNA and amino acid sequences of the P chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 55 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 57a and 57b show respectively the DNA and amino acid sequences of the p chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 63 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br><br>
Figures 58a and 58b show respectively the DNA and amino acid sequences of the p chain of a soluble A6 TCR, mutated so as to introduce a novel cysteine at residue 15 in exon 1 of TRBC2*01. The shaded nucleotides indicate the introduced novel cysteine codon and the underlined amino acid indicates the introduced cysteine;<br>
Figures 59-64 are traces obtained from anion exchange chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between: residues 48 of exon 1 of TRAC*01 and 57 of exon 1 of TRBC2*01; residues 45 of exon 1 of TRAC*01 and 77 of exon 1 of TRBC2*01; residues 10 of exon 1 of TRAC*01 and 17 of exon 1 of TRBC2*01; residues 45 of exon 1 of TRAC*01 and 59 of exon 1 of TRBC2*01; residues 52 of exon 1 of TRAC*01 and 55 of exon 1 of TRBC2*01; residues 15 of exon 1 of TRAC*01 and 15 of exon 1 of TRBC2*01, respectively, showing protein elution from a POROS 50 column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figures 65a and 65b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 48 of exon 1 of TRAC*01 and 57 of exon 1 of TRBC2*01, fractions run were collected from anion exchange column run in Figure 59;<br>
Figures 66a and 66b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 45 of exon 1 of TRAC*01 and 77 of exon l.of TRBC2*01, fractions run were collected from anion exchange column run in Figure 60;<br>
Figures 67a and 67b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain bond between residues lOof exon 1 of TRAC*01 and 17 exon 1 of TRBC2*01, fractions run were collected from anion exchange column run in Figure 61;<br>
Figures 68a and 68b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain<br><br>
bond between residues 45 of exon 1 of TRAC*01 and 59 of exon 1 of TRBC2*01, fractions run were collected from anion exchange column run in Figure 62;<br>
Figures 69a and 69b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 52 of exon 1 of TRAC*01 and 55 of exon 1 of TRBC2*01, fractions run were collected from anion exchange column run in Figure 63;<br>
Figures 70a and 70b are, respectively, reducing and non-reducing SDS-PAGE (Coomassie-stained) of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 15 of exon 1 of TRAC*01 and 15 of exon 1 ofTRBC2*01, fractions run were collected from anion exchange column run in Figure 64;<br>
Figure 71 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 48 of exon 1 of TRAC*01 and 57 of exon 1 of TRBC2*01, showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 59;<br>
Figure 72 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 45 of exon 1 of TRAC*01 and 77 of exon 1 of TRBC2*01, showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 60;<br>
Figure 73 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 10 of exon 1 of TRAC*01 and 17 of exon 1 of TRBC2*01, showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 61;<br><br>
Figure 74 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 45 of exon 1 of TRAC*01 and 59 of exon 1 of TRBC2*01, showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 62;<br>
Figure 75 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 52 of exon 1 of TRAC*01 and 55 of exon 1 of TRBC2*01, showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 63;<br>
Figure 76 is a trace obtained from size exclusion chromatography of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 15 of exon 1 of TRAC*0I and 15 of exon 1 of TRBC2*01s showing protein elution from a Superdex 200 HL gel filtration column. Fractions run were collected from anion exchange column run in Figure 64; and<br>
Figures 77-80 are BIAcore response curves showing, respectively, binding of soluble A6 TCR containing a novel disulphide inter-chain bond between: residues 48 of exon 1 of TRAC*01 and 57 of exon 1 of TRBC2*01; residues 45 of exon 1 of TRAC*01 and 77 of exon 1 of TRBC2*01; residues 10 of exon 1 of TRAC*01 and 17 of exon 1 of TRBC2*01; and residues 45 of exon 1 of TRAC*01 and 59 of exon 1 of TRBC2*01 to HLA-A2-tax pMHC.<br>
Figure 81 is a BIAcore trace showing non-specific binding of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 52 of exon 1 of TRAC*01 and 55 of exon 1 of TRBC2*01 to HLA-A2-tax and to HLA-A2-NY-ESO<br>
pMHC;<br><br>
Figure 82 is a BIAcore response curve showing binding of soluble A6 TCR containing a novel disulphide inter-chain bond between residues 15 of exon 1 of TRAC*01 and 15 of exon 1 of TRBC2*01 to HLA-A2-tax pMHC;<br>
Figure 83a is an electron density map around the model with 1BD2 sequence (Chain A Thrl64, Chain B Ser 174). Map contoured at 1.0,2.0 and 3.0 a. Figure 83b is an electron density map after refinement with Cys in the two positions A164 and B174.<br>
The map is contoured at the same a levels as for Fig 83a;<br>
t	......<br>
Figure 84 compares the structures of 1BD2 TCR with an NY-ESO TCR of the present invention by overlaying said structures in ribbon and coil representations;<br>
Figures 85a and 85b show the DNA and amino acid sequences respectively of the /3 chain of the NY-ESO TCR incorporating a biotin recognition site. The biotin recognition site is highlighted;<br>
Figures 86a and 86b show the DNA and amino acid sequences respectively of the 0 chain of the NY-ESO TCR incorporating the hexa-hisitidine tag. The hexa-hisitidine tag is highlighted;<br>
Figure 87 illustrates the elution of soluble NY-ESO TCR containing a novel disulphide bond and a biotin recognition sequence from a POROS 50HQ anion exchange column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 88 illustrates the elution of soluble NY-ESO TCR containing a novel disulphide bond and a hexa-histidine tag from a POROS 50HQ anion exchange columns using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 89 is a protein elution profile from gel filtration chromatography of pooled fractions from the NY-ESO-biotin tagged anion exchange column run illustrated by Figure 87;<br><br>
Figure 90 is a protein elution profile from gel filtration chromatography of pooled fractions from the NY~ESO-hexa-histidine tagged anion exchange column run illustrated by Figure 88;<br>
Figures 91a-h are FACS histograms illustrating the staining intensity produced from 25,000 events forHLA-A2 positive EBV transfonned B cell line (PP LCL) incubated with the following concentrations of NY-ESO peptide and fluorescent NY-ESO TCR tetramers respectively: NYESO 0 TCR 5^g, NYESO lO^M TCR 5p&amp; NYESO 10"5M TCR 5pg, NYESO 10"^ TCR 5p&amp; NYESO 0 TCR lOpg, NYESO 10^M TCR lOpg, NYESO 10*5M TCR lOjig, NYESO lC^M TCR lOjig;<br>
Figure 92 is the DNA sequence of the beta-chain of A6 TCR incorporating the TRBCl *01 constant region;<br>
Figure 93 is an anion exchange chromatography trace of soluble A6 TCR incorporating the TRBCl *01 constant region showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 94 - A. Reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 93, as indicated. B. Non-reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 93, as indicated.;<br>
7igure 95 - Size-exclusion chromatography of pooled fractions from peak 2 in figure )3. Peak 1 contains TCR heterodimer which is inter-chain disulphide linked;<br>
?igure 96 - A. BIAcore analysis of the specific binding of disulphide-linked A6 oluble TCR to HLA-Flu complex. B.Binding response compared to control for a ingle injection of disulphide-linked A6 soluble TCR;<br>
igure 97 shows the nucleic acid sequence of the mutated beta chain of the A6 TCR lcorporating the 'free5 cysteine;<br><br>
Figure 98 - Anion exchange chromatography of soluble A6 TCR incorporating the 'free5 cysteine.showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 99 - A. Reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 98, as indicated. B. Non-reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 98, as indicated;<br>
Figure 100 - Size-exclusion chromatography of pooled fractions from peak 2 in figure 98. Peak 1 contains TCR heterodimer which is inter-chain disulphide linked;<br>
Figure 101 - A. BIAcore analysis of the specific binding of disulphide-linked A6 soluble TCR incorporating the 'free' cysteine to HLA-Flu complex. B. Binding response compared to control for a single injection of disulphide-linked A6 soluble TCR;<br>
Figure 102 shows the nucleic acid sequence of the mutated beta chain of the A6 TCR -incorporating a serine residue mutated in for the 'free' cysteine;<br>
Figure 103 — Anion exchange chromatography of soluble A6 TCR incorporating a serine residue mutated in for the 'free' cysteine showing protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line;<br>
Figure 104 - A. Reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 103, as indicated. B. Non-reducing SDS-PAGE (Coomassie-stained) of fractions from column run in Figure 103, as indicated. Peak 2 clearly contains TCR heterodimer which is inter-chain disulphide linked;<br>
Figure 105 - Size-exclusion chromatography of pooled fractions from peak 2 in Figure 103. Peak 1 contains TCR heterodimer which is inter-chain disulphide linked;<br><br>
Figure 106 - A. BIAcore analysis of the specific binding of disulphide-linked A6 soluble TCR incorporating a serine residue mutated in for the 'free' cysteine to HLA-Fhi complex. B.Binding response compared to control for a single injection of disulphide-linked A6 soluble TCR;<br>
Figure 107 shows the nucleotide sequence of pYX112;<br>
Figure 108 shows the nucleotide sequence of pYX122;<br>
Figure 109 shows the DNA and protein sequences of pre-pro mating factor alpha fused to TCR a chain;<br>
Figure 110 shows the DNA and protein sequence of pre-pro mating factor alpha fused to TCR P chain;<br>
Figure 111 shows a Western Blot of soluble TCR expressed in S. cerevisiae strain SEY6210. Lane C contains 60ng of purified soluble NY-ESO TCR as a control. Lanes 1 and 2 contain the proteins harvested from the two separate TCR transformed yeast cultures;<br>
Figure 112 shows the nucleic acid sequence of the Kpnl to EcoRI insert of the pEX172 plasmid. The remainder of the plasmid is pBlueScript IIKS-;<br>
Figure 113 is a schematic diagram of the TCR chains for cloning into baculovirus;<br>
Figure 114 shows the nucleic acid sequence of disulphide A6 a TCR construct as a BamHI insert for insertion bxto pAcAB3 expression plasmid;<br>
Figure 115 shows the disulphide A6 j8 TCR construct as a BamHI for insertion into pAcAB3 expression plasmid; and<br><br>
Figure 116 shows a Coomassie stained gel and Western Blot against the bacterially-produced disulphide A6 TCR and the Insect disulphide A6 TCR.<br>
In all of the following examples, unless otherwise stated, the soluble TCR chains produced are truncated immediately C-terminal to the cysteine residues which form the native interchain disulphide bond.<br>
Example J — Design of primers and mutagenesis ofA6 Tax TCR a and fi chains<br>
For mutating A6 Tax threonine 48 of exon 1 in TRACOlto cysteine, the following primers were designed (mutation shown in lower case):<br>
5'-C ACA GAC AAA tgT GTG CTA GAC AT 5 '-AT GTC TAG CAC Aca TTT GTC TGT G<br>
For mutating A6 Tax serine 57 of exon 1 in both TRBC1*01 and TRBC2*01 to cysteine, the following primers were designed (mutation shown in lower case):<br>
5'-C AGT GGG GTC tGC ACA GAC GC 5'-GG GTC TGT GCa GAC CCC ACT G<br>
PCR mutagenesis:<br>
Expression plasmids containing the genes for the A6 Tax TCR a or P chain were mutated using the a-chain primers or the j3-chain primers respectively, as follows. 100 ng of plasmid was mixed with 5 ul 10 mM dNTP, 25 \xl lOxPfii-buffer (Stratagene), 10 units Pfu polymerase (Stratagene) and the final volume was adjusted to 240 jil with H2O. 48 \xl of this mix was supplemented with primers diluted to give a final concentration of 0.2 yM in 50-(J final reaction volume. After an initial denaturation step of 30 seconds at 95°C, the reaction mixture was subjected to 15 rounds of denaturation (95°C, 30 sec), annealing (55°C, 60 sec), and elongation (73°C, 8 min.) in a Hybaid PCR express PCR machine. The product was then digested for 5 hours at 37°C with 10 units of Dpnl restriction enzyme (New England Biolabs).<br><br>
10 JLLI of the digested reaction was transformed into competent XLl-Blue bacteria and grown for 18 hours at 37°C. A single colony was picked and grown over night in 5 ml TYP + ampicillin (16 g/1 Bacto-Tryptone, 16 g/1 Yeast Extract, 5 g/1 NaCl, 2.5 g/1 K2HPO4,100 mg/1 Ampicillin). Plasmid DNA was purified on a Qiagen mini-prep column according to the manufacturer's instructions and the sequence was verified by automated sequencing at the sequencing facility of Department of Biochemistry, Oxford University. The respective mutated nucleic acid and amino acid sequences are shown in Figures 2a and 3a for the a chain and Figures 2b and 3b for the J3 chain.<br>
Example 2 - Expression, refolding and purification of soluble TCR<br>
The expression plasmids containing the mutated a-chain and p-chain respectively were transformed separately into E.coli strain BL21pLysS, and single ampicillin-resistant colonies were grown at 37°C in TYP (ampicillin 100|ug/ml) medium to OD^oo of 0.4 before inducing protein expression with 0.5mM IPTG. Cells were harvested three hours post-induction by centriiugation for 30 minutes at 4000ipm in a Beckman J-6B. Cell pellets were re-suspended in a buffer containing 50mM Tris-HC^ 25% (w/v) sucrose, ImMNaEDTA, 0.1%(w/v)NaAzide, 10mMDTT,pH8.0. After an overnight freeze-thaw step, re-suspended cells were sonicated in 1 minute bursts for a total of around 10 minutes in a Milsonix XL2020 sonicator using a standard 12mm diameter probe. Inclusion body pellets were recovered by centrifugation for 30 minutes at 13000rpm in a Beckman J2-21 centrifuge. Three detergent washes were then carried out to remove cell debris and membrane components. Each time the inclusion body pellet was homogenised in a Triton buffer (50mM Tris-HC? 0.5% Triton-XlOO, 200mM NaC£ lOmM NaEDTA, 0.1% (w/v) NaAzide, 2mM DTT, pH 8.0) before being pelleted by centrifugation for 15 minutes at 13000rpm in a Beckman J2-21. Detergent and salt was then removed by a similar wash in the following buffer: 50mMTris-HCl, ImMNaEDTA, 0.1% (w/v)NaAzide, 2mM DTT, pH 8.0. Finally, the inclusion bodies were divided into 30 mg aliquots and frozen at -70°C. Inclusion body protein yield was quantitated by solubilising with 6M guanidine-HCl and measurement with a Bradford dye-binding assay (PerBio).<br><br>
Approximately 30mg (i.e. 1 pinole) of each solubilised inclusion body chain was thawed from frozen stocks, samples were then mixed and the mixture diluted into 15ml of a guanidine solution (6 M Guanidine-hydrochloride, lOmM Sodium Acetate, * lOmM EDTA), to ensure complete chain de-naturation. The guanidine solution containing fully reduced and denatured TCR chains was then injected into 1 litre of the following refolding buffer: 1 OOmM Tris pH 8.5,400mM L-Arginine, 2mM EDTA, 5mM reduced Glutathione, 0.5mM oxidised Glutathione, 5M urea, 0.2mM PMSF. The solution was left for 24 hrs. The refold was then dialysed twice, firstly against 10 litres of lOOmM urea, secondly against 10 litres of lOOmM urea, lOmM Tris pH 8.0. Both refolding and dialysis steps were carried out at 6-8°C.<br>
sTCR was separated from degradation products and impurities by loading the dialysed refold onto a POROS 50HQ anion exchange column and eluting bound protein with a gradient of 0-500mM NaQ?over 50 column volumes using an Akta purifier (Pharmacia) as in Figure 4. Peak fractions were stored at 4°C and analysed by Coomassie-stained SDS-PAGE (Figure 5) before being pooled and concentrated. Finally, the sTCR was purified and characterised using a Superdex 200HR gel filtration column (Figure 6) pre-equilibrated in HBS-EP buffer (10 mM HEPES pH 7.4,150 mM NaCl, 3.5 mM EDTA, 0.05% nonidet p40). The peak eluting at a relative molecular weight of approximately 50 kDa was pooled and concentrated prior to characterisation by BIAcore surface plasmon resonance analysis.<br>
Example 3 — BIAcore surf ace plasmon resonance characterisation ofsTCR binding to specific pMHC<br>
A surface plasmon resonance biosensor (BIAcore 3000™ ) was used to analyse the binding of a sTCR to its peptide-MHC ligand. This was facilitated by producing single pMHC complexes (described below) which were immobilised to a streptavidin-coated binding surface in a semi-oriented fashion, allowing efficient testing of the binding of a soluble T-cell receptor to up to four different pMHC (immobilised on separate flow cells) simultaneously. Manual injection of HLA complex allows the precise level of immobilised class I molecules to be manipulated easily.<br><br>
Such immobilised complexes are capable of binding both T-cell receptors and the coreceptor CD8aa, both of which may be injected in the soluble phase. Specific binding of TCR is obtained even at low concentrations (at least 40|ig/ml), implying the TCR is relatively stable. The pMHC binding properties of sTCR are observed to be qualitatively and quantitatively similar if sTCR is used either in the soluble or ■ immobilised phase. This is an important control for partial activity of soluble species and also suggests that biotinylated pMHC complexes are biologically as active as non-biotinylated complexes.<br>
Biotinylated class IHLA-A2 - peptide complexes were refolded in vitro from bacterially-expressed inclusion bodies containing the constituent subunit proteins and synthetic peptide, followed by purification and in vitro enzymatic biotinylation (O'Callaghau et al (1999) Anal Biochem. 266:9-15). HLA-heavy chain was expressed with a C-terminal biotinylation tag which replaces the transmembrane and cytoplasmic domains of the protein in an appropriate construct. Inclusion body expression levels of ~75 mg/litre bacterial culture were obtained. The HLA light-chain or p2-microglobulin was also expressed as inclusion bodies in Exoli from an appropriate construct, at a level of-500 mg/litre bacterial culture.<br>
E. coli cells were lysed and inclusion bodies are purified to approximately 80% purity. Protein from inclusion bodies was denatured in 6 M guanidine-HCl, 50 mM Tris pH 8.1,100 mM NaCl, 10 mM DTT, 10 mM EDTA, and was refolded at a concentration of 30 mg/litre heavy chain, 30 mWlitre j32m into 0.4 M L-Arginine-HCl, 100 mM Tris<br>
6*6    JB~<br>
pH 8.1,3.7 mM cystamineJmJVWcystcamine, 4 mg/ml peptide (e.g. tax 11-19), by addition of a single pulse of denatured protein into refold buffer at 
Buffer was exchanged by dialysis in 10 volumes of 10 mM Tris pH 8.1. Two changes of buffer were necessary to reduce the ionic strength of the solution sufficiently. The protein solution was then filtered through a 1.5(jm cellulose acetate filter and loaded onto a POROS 50HQ anion exchange column (8 ml bed volume). Protein was eluted with a linear 0-500 mM NaCl gradient. HLA-A2-peptide complex eluted at<br><br>
approximately 250 mM NaCl, and peak fractions were collected, a cocktail of protease inhibitors (Calbiochem) was added and the fractions were chilled on ice.<br>
Biotinylation tagged HLA complexes were buffer exchanged into 10 mM Tris pH 8.1, 5 mM NaCl using a Pharmacia fast desalting column equilibrated in the same buffer. Immediately upon elution, the protein-containing fractions were chilled on ice and protease inhibitor cocktail (Calbiochem) was added. Biotinylation reagents were then added: 1 mM biotin, 5 mM ATP (buffered to pH 8), 7.5 mM MgC12, and 5 |ig/ml BirA enzyme (purified according to O'Callaghan et al. (1999) Anal Biochem. 266: 9-15). The mixture was then allowed to incubate at room temperature overnight.<br>
Biotinylated HLA complexes were purified using gel filtration chromatography. A Pharmacia Superdex 75 HR 10/30 column was pre-equilibrated with filtered PBS and 1 ml of the biotinylation reaction mixture was loaded and the column was developed with PBS at 0.5 ml/min. Biotinylated HLA complexes eluted as a single peak at approximately 15 ml. Fractions containing protein were pooled, chilled on ice, and protease inhibitor cocktail was added. Protein concentration was determined using a Coomassie-binding assay (PerBio) and aliquots of biotinylated HLA complexes were stored frozen at -20°C. Streptavidin was immobilised by standard amine coupling methods.<br>
The interactions between A6 Tax sTCR containing a novel inter-chain bond and its ligand/ MHC complex or an irrelevant HLA-peptide combination, the production of which is described above, were analysed on a BIAcore 3000™ surface plasmon resonance (SPR) biosensor. SPR measures changes in refractive index expressed in response units (RU) near a sensor surface within a small flow cell, a principle that can be used to detect receptor ligand interactions and to analyse their affinity and kinetic parameters. The probe flow cells were prepared by immobilising the individual HLA-peptide complexes in separate flow cells via binding between the biotin cross linked onto P2m and streptavidin which have been chemically cross linked to the activated surface of the flow cells. The assay was then performed bypassing sTCR over the surfaces of the different flow cells at a constant flow rate, measuring the SPR response<br><br>
in doing so. Initially, the specificity of the interaction was verified bypassing sTCR at a constant flow rate of 5 pi min-1 over two different surfaces; one coated with ~5000 RU of specific peptide-HLA complex, the second coated with -5000 RU of nonspecific peptide-HLA complex (Figure 7 insert). Injections of soluble sTCR at constant flow rate and different concentrations over the peptide-HLA complex were used to define the background resonance. The values of these control measurements were subtracted from the values obtained with specific peptide-HLA complex and used to calculate binding affinities expressed as the dissociation constant, Kd (Price &amp; Dwek, Principles and Problems in Physical Chemistry for Biochemists (2nd Edition) 1979, Clarendon Press, Oxford), as in Figure 7.<br>
The Kd value obtained (1.8pM) is close to that reported for the interaction between A6 Tax sTCR without the novel di-sulphide bond and pMHC (0.91 pM - Ding et al, 1999, Inmmunity 11:45-56).<br>
Example 4 - Production of soluble JM22 TCR containing a novel disulphide bond.<br>
The P chain of the soluble A6 TCR prepared in Example 1 contains in the native sequence a Bglll restriction site (AAGCTT) suitable for use as a ligation site.<br>
PCR mutagenesis was carried as detailed below to introduce a BamHl restriction site (GGATCC) into the a chain of soluble A6 TCR, 5' of the novel cysteine codon. The sequence described in Figure 2a was used as a template for this mutagenesis. The following primers were used:<br>
JBamHI | 5' -ATATCCAGAACCCgGAtCCTGCCGTGTA-3' 5' -TACACGGCAGGAaTCcGGGTTCTGGATAT-3'<br>
100 ng of plasmid was mixed with 5 \il 10 mM dNTP, 25 pi lOxPfu-buffer (Stratagene), 10 units Pfu polymerase (Stratagene) and the final volume was adjusted to 240 |jJ with H2O. 48 pi of this mix was supplemented with primers diluted to give a<br><br>
final concentration of 0.2 piM in 50 jxl final reaction volume. After an initial denaturation step of 30 seconds at 95°C, the reaction mixture was subjected to 15 rounds of denaturation (95°C, 30 sec), annealing (55°C, 60 sec), and elongation (73°C, 8 min.) in a Hybaid PCR express PCR machine. The product was then digested for 5 hours at 37°C with 10 units of Dpnl restriction enzyme (New England Biolabs). 10 fxl of the digested reaction was transformed into competent XLl-Blue bacteria and grown for 18 hours at 37°C. A single colony was picked and grown over night in 5 ml TYP + ampicillin (16 g/1 Bacto-Tryptone, 16 g/\ Yeast Extract, 5 g/1 NaCl, 2.5. g/1 K2HPC&gt;4&gt; 100 mg/1 Ampicillin). Plasmid DNA was purified on a Qiagen mini-prep column according to the manufacturers instructions and the sequence was verified by automated sequencing at the sequencing facility of Department of Biochemistry, Oxford University. The mutations introduced into the a chain were "silent", therefore the amino acid sequence of this chain remained unchanged from that detailed in Figure 3 a. The DNA sequence for the mutated a chain is shown in Figure 8a.<br>
In order to produce a soluble JM22 TCR incorporating a novel disulphide bond, A6 TCR plasmids containing the a chain BamHl and P chain Bglll restriction sites were used as templates. The following primers were used:<br>
|  Ndel   | 5' -GGAGATATACATATGCAACTACTAGAACAA-3'<br>
5' -TACACGGCAGGATCCGGGTTCTGGATATT-3'<br>
|  BamHl)<br>
|Ndel  ) 5' -GGAGATATACATATGGTGGATGGTGGAATC-3'<br>
5' - CCCAAGCTTAGTCTGCTCTACCCCAGGCCTCGGC - 3' |BglIl|<br>
JM22 TCR a and P-chain constructs were obtained by PCR cloning as follows. PCR reactions were perfonned using the primers as shown above, and templates containing the JM22 TCR chains. The PCR products were restriction digested with<br><br>
the relevant restriction enzymes, and cloned into pGMT7 to obtain expression plasmids. The sequence of the plasmid inserts were confirmed by automated DNA sequencing. Figures 8b and 8c show the DNA sequence of the mutated a and j8 chains of the JM22 TCR respectively, and Figures 9a and 9b show the resulting amino acid sequences.<br>
The respective TCR chains were expressed, co-refolded and purified as described in Examples 1 and 2. Figure 10 illustrates the elution of soluble disulphide-linked JM22 TCR protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figure 11 shows the results of both reducing SDS-PAGE (Coomassie-stained) and non-reducing SDS-PAGE (Coomassie-stained) gels of fractions from the column run illustrated by Figure 10. Peak 1 clearly contains TCR heterodimer which is inter-chain disulphide linked. Figure 12 shows protein elution from a size-exclusion column of pooled fractions from peak 1 in Figure 10.<br>
A BIAcore analysis of the binding of the JM22 TCR to pMHC was carried out as described in Example 3.  Figure 13a shows BIAcore analysis of the specific binding of disulphide-linked JM22 soluble TCR to HLA-Flu complex. Figure 13b shows the binding response compared to control for a single injection of disulphide-linked JM22 soluble TCR. The Kd of this disulphide-linked TCR for the HLA-flu complex was determined to be 7.9 + 0.51(jM<br>
Example 5 - Production of soluble NY-ESO TCR containing a novel disulphide bond<br>
cDNA encoding NY-ESO TCR was isolated from T cells supplied by Enzo Cerundolo (Institute of Molecular Medicine, University of Oxford) according to known techniques. cDNA encoding NY-ESO TCR was produced by treatment of the mKNA with reverse transcriptase.<br>
In order to produce a soluble NY-ESO TCR incorporating a novel disulphide bond, A6 TCR plasmids containing the a chain BamHI and P chain Bglll restriction sites were used as templates as described in Example 4. The following primers were used:<br><br>
Ndel 5' -GGAGATATACATATGCAGGAGGTGACACAG-3<br>
5' -TACACGGCAGGATCCGGGTTCTGGATATT-3'<br>
BamHI<br>
Ndel 5' -GGAGATATACATATGGGTGTCACTCAGACC-3'<br>
5' - CCCAAGCTTAGTCTGCTCTACCCCAGGCCTCGGC - 3' iBglll.!<br>
NY-ESO TCR a and P-chain constructs were obtained by PCR cloning as follows.* PCR reactions were performed using the primers as shown above, and templates containing the NY-ESO TCR chains. The PCR products were restriction digested with the relevant restriction enzymes, and cloned into pGMT7 to obtain expression plasmids. The sequence of the plasmid inserts were confirmed by automated DNA sequencing. Figures 14a and 14b show the DNA sequence of the mutated a and (3 chains of the NY-ESO TCR respectively, and Figures 15a and 15b show the resulting amino acid sequences.<br>
The respective TCR chains were expressed, co-refolded and purified as described in Examples 1 and 2, except for the following alterations in protocol:<br>
Denaturation of soluble TCRs; 30mg of the solubilised TCR /3-chain inclusion body and 60mg of the solubilised TCR a-chain inclusion body was thawed from frozen stocks. The inclusion bodies were diluted to a final concentration of 5mg/ml in 6M guanidine solution, and DTT (2M stock) was added to a final concentration of lOmM. The mixture was incubated at 37°C for 30 min.<br>
Refolding of soluble TCRs: 1 L refolding buffer was stirred vigorously at 5°C ± 3°C. The redox couple (2-mercaptoethylamine and cystamine (to final concentrations of 6.6mM and 3.7mM, respectively) were added approximately 5 minutes before addition<br><br>
of the denatured TCR chains. The protein was then allowed to refold for approximately 5 hours =fc 15 minutes with stirring at 5°C ± 3°C. Dialysis of refolded soluble TCRs: The refolded TCR was dialysed in Spectrapor 1 membrane (Spectrum; Product No. 132670) against 10 L 10 mM Tris pH 8.1 at 5°C ± 3°C for 18-20 hours. After this time, the dialysis buffer was changed to fresh 10 mM Tris pH 8.1 (10 L) and dialysis was continued at 5°C ± 3°C for another 20-22 hours.<br>
Figure 16 illustrates the elution of soluble NY-ESO disulphide-linked TCR protein elution from a POROS 50HQ column using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figure 17 shows the results of both reducing SDS-PAGE (Coomassie-stained) and non-reducing SDS-PAGE (Coomassie-stained) gels of fractions from the column run illustrated by Figure 16. Peaks 1 and 2 clearly contain TCR heterodimer which is inter-chain disulphide linked. Figure 18 shows size-exclusion chromatography of pooled fractions from peak 1 (A) and peak 2 (B) in Figure 17. The protein elutes as a single major peak, corresponding to the heterodimer.<br>
A BIAcore analysis of the binding of the disulphide-linked NY-ESO TCR to pMHC was carried out as described in Example 3. Figure 19 shows BIAcore analysis of the specific binding of disulphide-linked NY-ESO soluble TCR to HLA-NYESO complex. A. peak 1, B. peak 2.<br>
The Kd of this disulphide-linked TCR for the HLA-NY-ESO complex was determined to be 9.4 ± 0.84pM.<br>
Example 6-Production of soluble NY-ESO TCR containing a novel disulphide interchain bond, and at least one of the two cysteines required to form the native disulphide inter-chain bond<br>
In order to produce a soluble NY-ESO TCR incorporating a novel disulphide bond and at least one of the cysteine residues involved in the native disulphide inter-chain bond,<br><br>
plasmids containing the a chain BamHI and P chain Bglll restriction sites were used as a framework as described in Example 4. The following primers were used:<br>
Ndel 5' -GGAGATATACATATGCAGGAGGTGACACAG-3'<br>
5' -CCCAAGCTTAACAGGAACTTTCTGGGCTGGGGAAGAA-3' HindiII<br>
|   Ndel 5' -GGAGATATACATATGGGTGTCACTCAGACC-3'<br>
5' - CCCAAGCTTAACAGTCTGCTCTACCCCAGGCCTCGGC   - 3 ' iBglll<br>
NY-ESO TCR a and p-chain constructs were obtained by PCR cloning as follows. PCR reactions were performed using the primers as shown above, and templates containing the NY-ESO TCR chains. The PCR products were restriction digested with the relevant restriction enzymes, and cloned into pGMT7 to obtain expression plasmids. The sequence of the plasmid inserts were confirmed by automated DNA sequencing. Figures 20a and 20b show the DNA sequence of the mutated a and f3 chains of the NY-ESO TCR respectively, and Figures 21a and 21b show the resulting amino acid sequences.<br>
To produce a soluble NY-ESO TCR containing both a non-native disulphide interchain bond and the native disulphide inter-chain bond, DNA isolated using both of the above primers was used. To produce soluble NY-ESO TCRs with a non-native disulphide inter-chain bond and only one of the cysteine residues involved in the native disulphide inter-chain bond, DNA isolated using one of the above primers together with the appropriate primer from Example 5 was used.<br>
The respective TCR chains were expressed, co-refolded and purified as described in Example 5.<br><br>
Figures 22-24 illustrate the elution of soluble NY-ESO TCRcP ft** (i.e. with non-native and native cysteines in both chains), TCRoP (with non-native cysteines in both chains but the native cysteine in the a chain only), and TCR^S0^ (with non-native cysteines in both chains but the native cysteine in the /3 chain only) protein elution from POROS 50HQ anion exchange columns using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figures 25 and 26 respectively show the results of reducing SDS-PAGE (Coomassie-stained) and non-reducing SDS-PAGE (Coomassie-stained) gels of fractions from the NY-ESO TCRotys /S0*1, TCRo^, and TCRjS^ column runs illustrated by Figures 22-24. These clearly indicate that TCR heterodimers which are inter-chain disulphide linked have been formed. Figures 27-29 are protein elution profiles from gel filtration chromatography of pooled fractions from the NY-ESO TCRcf ** jS0^, TCRcP5, and TCR/f*8 anion exchange column runs illustrated by Figures 22-24 respectively. The protein elutes as a single major peak, corresponding to the TCR heterodimer.<br>
A BIAcore analysis of sTCR binding to pMHC was carried out as described in Example 3.  Figures 30-32 show BIAcore analysis of the specific binding of NY-ESO TCRo?*8 jS03*, TCRoP, and TCR/?*5 respectively to HLA-NYESO complex.<br>
TCRoPys/3cyshadaK<jofl8.08 tcrofys had a ka of jim and tcrjs1 kd pm.></jofl8.08>
Example 7-Production of soluble AH-J.23 TCR containing a novel disulphide interchain bond<br>
cDNA encoding AH-1.23 TCR was isolated from T cells supplied by Hill Gaston (Medical School, Addenbrooke's Hospital, Cambridge) according to known techniques. cDNA encoding NY-ESO TCR was produced by treatment of the mRNA with reverse transcriptase.<br><br>
hi order to produce a soluble AH-1.23 TCR incorporating a novel disulphide bond, TCR plasmids containing the a chain BamHI and p chain BgUI restriction sites were used as a framework as described in Example 4. The following primers were used:<br>
| Ndei  | 5' -GGGA?VGCTTACATATGAAGGAGGTGGAGCAGAATTCTGG-3'<br>
5' -TACACGGCAGGATCCGGGTTCTGGATATT-3'<br>
|   BamHI|<br>
| Ndei | 5' -TTGGAATTCACATATGGGCGTCATGCAGAACCCAAGACAC-3'<br>
5' -CCCAAGCTTAGTCTGCTCTACCCCAGGCCTCGGC-3' |BglIl|<br>
AH-1.23 TCR a and P-chain constructs were obtained by PCR cloning as follows. PCR reactions were performed using the primers as shown above, and templates containing the AH-1.23 TCR chains. The PCR products were restriction digested with the relevant restriction enzymes, and cloned into pGMT7 to obtain expression plasmids. The sequence of the plasmid inserts were confirmed by automated DNA sequencing. Figures 33a and 33b show the DNA sequence of the mutated a and )3 chains of the AH-1.23 TCR respectively, and Figures 34a and 34b show the resulting amino acid sequences.<br>
The respective TCR chains were expressed, co-refolded and purified as described in Example 5.<br>
Figure 35 illustrates the elution of soluble AH-1.23 disulphide-linked TCR protein elution from a POROS 50HQ anion exchange column using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figures 36 and 37 show the results of reducing SDS-PAGE (Coomassie-stained) and non-reducing SDS-PAGE (Coomassie-stained) gels respectively of fractions from the column run illustrated by Figure 35. These gels clearly indicate the presence of a TCR heterodimer which is inter-chain disulphide linked. Figure 38 is the elution profile from a Superdex 75 HR gel filtration column of pooled fractions from the anion exchange column run illustrated in<br><br>
Figure 35. The protein elutes as a single major peak, corresponding to the heterodimer.<br>
Example 8-Production of soluble A6 TCRs containing a novel disulphide inter-chain bond at alternative positions within the immunoglobulin region of the constant domain<br>
The following experiments were carried out in order to investigate whether it was possible to form functional soluble TCRs which include a novel disulphide bond in the TCR immunoglobulin region at a position other than between threonine 48 of exon 1 in TRAC*01 and serine 57 of exon 1 in both TRBC1*01 / TRBC2*01.<br>
For the mutating the A6 TCR ochain, the following primers were designed (the numbers in the primer names refer to the position of the amino acid residue to be mutated in exon 1 of TRAC*01, mutated residues are shown in lower case):<br>
T48-»C Mutation<br>
5 '-CACAGACAAAtgTGTGCTAGAC AT-3'<br>
5 5-ATGTCTAGCACAcaTTTGTCTGTG-3»<br>
Yl 0-»C Mutation<br>
5 '-CCCTGCCGTGTgCCAGCTGAGAG-3 "	-<br>
5 '-CTCTCAGCTGGcACACGGCAGGG-3'<br>
L12-^C Mutation<br>
5'-CCGTGTACCAGtgcAGAGACTCTAAATC-3'<br>
S'-GATTTAGAGTCTCTgcaCTGGTACACGG-S*<br>
S15^C Mutation<br>
5 '-CAGCTGAGAGACTgTAAATCCAGTGAC-3'<br>
S'-GTCACTGGATTTAcAGTCTCTCAGCTG-S'<br><br>
V22-»C Mutation<br>
5 '-CAGTGACAAGTCTtgCTGCCTATTCAC-3'<br>
5 '-GTGAATAGGCAGcaAGACTTGTCACTG-3'<br>
Y43-&gt;C Mutation<br>
5'-GATTCTGATGTGTgTATCACAGACAAAT-3' 5'-ATTTGTCTGTGATAcACACATCAGAATC-3'<br>
T45-»C Mutation<br>
5 '-CTGATGTGTATATCtgtGACAAAACTGTGC-3'<br>
5'-GCACAGTTTTGTCacaGATATACACATCAG-3'<br>
L50-»C Mutation<br>
5 '-AGACAAAACTGTGtgtGACATGAGGTCT-3'<br>
5 '-AGACCTCATGTCacaCACAGTTTTGTCT-3'<br>
M52-+C Mutation<br>
5'-ACTGTGCTAGACtgtAGGTCTATGGAC-3'<br>
5 '-GTCCATAGACCTacaGTCTAGCACAGT-3'<br>
S61-»C Mutation<br>
5 '-CTTCAAGAGCAACtGTGCTGTGGCC-3'<br>
5 '-GGCCACAGCACaGTTGCTCTTGAAG-3'<br>
For mutating the TCR A6 /3-chain, the following primers were designed (the numbers in the primer names refer to the position of the amino acid residue to be mutated in exon 1 of TRBC2*01. Mutated residues are shown in lower case):<br>
S57-&gt;C Mutation<br>
5 '-CAGTGGGGTCtGCACAGACCC-3' 5 '-GGGTCTGTGCaGACCCCACTG-3'<br><br>
V13-»C Mutation<br>
5 '-CCGAGGTCGCTtgtTTTGAGCCATCAG-3'<br>
5 '-CTGATGGCTCAAAacaAGCGACCTCGG-3'<br>
F14-»C Mutation<br>
5 '-GGTCGCTGTGtgtGAGCCATCAGA-3'<br>
5'-TCTGATGGCTCacaCACAGCGACC-3'<br>
S17-»C Mutation<br>
5 '-GTGTTTGAGCCATgtGAAGCAGAGATC-3'<br>
5 '-GATCTCTGCTTCacATGGCTCAAACAC-3'<br>
G55-&gt;C Mutation<br>
5 '-GAGGTGCACAGTtGtGTCAGCACAGAC-3'<br>
5 '-GTCTGTGCTGACaCaACTGTGCACCTC-3'<br>
D59-»C Mutation<br>
5 '-GGGTCAGCACAtgCCCGCAGCCC-3'<br>
5 '-GGGCTGCGGGcaTGTGCTGACCC-3'<br>
L63-&gt;C Mutation<br>
5 '-CCCGCAGCCCtgCAAGGAGCAGC-3'<br>
5 '-GCTGCTCCTTGCaGGGCTGCGGG-3'<br>
S77-»C Mutation<br>
5 '-AGATACGCTCTGtGCAGCCGCCT-3'<br>
5 '-AGGCGGCTGCaCAGAGCGTATCT-3'<br>
R79-»C Mutation 5'-CTCTGAGCAGCtGCCTGAGGGTC-3'<br><br>
5 '-GACCCTCAGGCaGCTGCTCAGAG-3'<br>
E15-»C Mutation<br>
5r -GCTGTGTTTtgtCCATCAGAA- 3'<br>
5, .TTCTGATGGacaAAACACAGC- 3'<br>
PCR mutagenesis, a and /3 TCR construct amplification, ligation and plasmid purification was carried out as described in Example 1 using the appropriate combination of the above primers in order to produce soluble TCRs including novel disulphide inter-chain bonds between the following pairs of amino acids:<br><br>
Figures 39 to 58 show the DNA and amino acid sequences of the mutated A6 TCR chains amplified by the above primers. The codons encoding the mutated cysteines are highlighted.<br><br>
The respective TCR chains were expressed, co-refolded and purified as described in Example 5. Following purification on POROS 50HQ anion exchange column, the resulting proteins were run on SDS-Page gels in order to assess whether any correctly-refolded soluble TCR had been formed. These gels were also assessed to ascertain the presence or absence of any disulphide-linked protein of the correct molecular weight in the purified material. TCRs under investigation containing the following novel disulphide inter-chain bonds failed to produce disulphide-linked protein of the correct molecular weight using this bacterial expression system and these were not further assessed. However, alternative prokaryotic or eukaryotic expression systems are available. <br>
Figures 59 to 64 respectively illustrate the elution of soluble TCRs containing novel disulphide interchain bonds between the following residues: Thr 48-Ser 57, Thr 45-Ser 77, Tyr 10-Ser 17, Thr 45-Asp 59, Met 52-Gly 55 and Ser 15-Glu 15 from a POROS 200HQ anion exchange column using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figures 65 to 70 show the results of reducing SDS-PAGE (Coomassie-stained) and non-reducing SDS-PAGE (Coomassie-stained) gels respectively of fractions from the column runs illustrated by Figures 59 to 64. These gels clearly ndicate the presence of TCRheterodimers that are inter-chain disulphide linked.<br>
figures 71 to 76 are elution profiles from aSuperdex 200 HR gel filtration column of &gt;ooled fractions from the anion exchange column runs illustrated in Figures 59 to 64.<br><br>
A BIAcore analysis of the binding of the TCRs to pMHC was carried out as described in Example 3. Figures 77- 82 are BIAcore traces demonstrating the ability of the purified soluble TCRs to bind to HLA-A2 tax pMHC complexes.     /<br>
Thr 48-Ser 57 had a Ka of 7.8 pM, Thr 45-Ser 77 had a K^ of 12.7 \M9 Tyr 10-Ser 17 had aKd of 34 juM, Thr 45-Asp 59 had a Kd of 14.9 pM, and Ser 15-Glu 15 had alQ of 6.3 |oM. Met 52-Gly 55 was capable of binding to its native "target", the HLA-A2 tax complex, although it also bound in a similar manner to an "irrelevant" target, the HLA-A2-NY-ESO complex (see Figure 81)<br>
Example 9 - X-ray crystallography of the disulphide-linked NY-ESO Tcell receptor\ specific for the NY-ESO-HLA-A2 complex.<br>
The NY-ESO dsTCR was cloned as described in Example 5, and expressed as follows.<br>
The expression plasmids containing the mutated a-chain and p-chain respectively were transformed separately into E.coli strain BL21 pLysS, and single ampicillin-resistant colonies were grown at 37°C in TYP (ampicillin lOO^g/ml) medium to ODeoo of 0.7 before inducing protein expression with 0.5mM IPTG. Cells were harvested 18 hours post-induction by centrifugation for 30 minutes at 4000rpm in a Beckman J-6B. Cell pellets were resuspended in lysis buffer containing lOmM Tris-HCfpH 8.1,10 mM MgCl2,150 mM NaCl, 2 mM DTT, 10% glycerol. For every 1 L of bacterial culture 100 nl of lysozyme (20 mg/ml) and 100 jil of Dnase I (20 fig/ml) were added. After incubation on ice for 30 minutes, the bacterial suspension was sonicated in 1 minute bursts for a total of 10 minutes using a Milsonix XL2020 sonicator with a standard 12mm diameter probe. Inclusion body pellets were recovered by centrifugation for 30 minutes at 13000rpm in aBeckman J2-21 centrifuge (4 °C). Three washes were then carried out in Triton wash buffer (50mM Tris-HCIpH 8*1, 0.5%Triton-X100, lOOmMNaCg lOmMNaEDTA, 0.1% (w/v), 2mMDTT) to remove cell debris and membrane components. Each time, the inclusion body pellet was homogenised in Triton wash buffer before being pelleted by centrifugation for 15 minutes at 13000rpm in a Beckman J2-21. Detergent and salt was then removed by a<br><br>
similar wash in Resuspension buffer (50mM Tris-HCl pH 8.1 1 OOmM NaCl, 1 OmM NaEDTA, 0.1% (w/v) NaAzide, 2mM DTT), Finally, the inclusion bodies were solubilised in 6 M guanidine buffer (6 M Guanidine-hydrochloride, 5 OmM Tris pH 8.1, lOQmMNaCl, lOmM EDTA, lOmM DTT), divided into 120mg aliquots and frozen at -70°C. Inclusion bodies were quantitated by solubilising with 6M guanidine-HC1 and measurement with a Bradford dye-binding assay (PerBio).<br>
Approximately 60mg (i.e, 2.4 jxmole) of frozen solubilised alpha chain was mixed with 30 mg (i.e. 1.2 jimole) of frozen solubilised beta chain. The TCR mixture was diluted to a final volume of 18ml with 6 M guanidine buffer and heated to 37 °C for 30 min to ensure complete chain denaturation. The guanidine solution containing fully reduced and denatured TCR chains was then mixed into I litre of cold refolding buffer (lOOmM Tris pH 8.1,400niM L-Arginine-HCl, 2mM EDTA, 6.6 mM 2-mercapthoethylamine, 3.7 mM Cystamine, 5M urea) with stirring. The solution was left for 5 hrs in the cold room (5°C ± 3°C) to allow refolding to take place. The refold was then dialysed against 12 litres of water for 18-20 hours, followed by 12 litres of lOmM Tris pH 8.1 for 18-20 hours (5°C ± 3°C). Spectrapor 1 (Spectrum Laboratories product no. 132670) dialysis membrane that has a molecular weight cut off of 6-8000kDa was used for this dialysis process. The dialysed protein was filtered through 0.45 jam pore size filters (Schleicher and Schuell, Ref. number, 10 404012) fitted to a Nalgene filtration unit.<br>
The refolded NY-ESO TCR was separated from degradation products and impurities by loading the dialysed refold onto a POROS 50HQ (Applied Biosystems) anion exchange column using an AKTA purifier (Amersham Biotech). A POROS 50 HQ column was pre-equilibrated with 10 column volumes of buffer A (10 mM Tris pH 8.1) prior to loading with protein. The bound protein was eluted with a gradient of 0-500mM NaCI over 7 column volumes. Peak fractions (1 ml) were analysed on denaturing SDS-PAGE using reducing and non-reducing sample buffer. Peak fractions containing the heterodimeric alpha-beta complex were further purified using a Superdex 75HR gel filtration column pre-equilibrated in 25 mM MES pH 6.5. The protein peak eluting at a relative molecular weight of approximately 50 kDa was<br><br>
pooled, concentrated to 42 mg/ml in Ultrafree centrifugal concentrators (Millipore, part number UFV2BGC40) and stored at -80 °C.<br>
Crystallisation of NY-ESO TCR was performed by hanging drop technique at 18 °C using 1 ^1 of protein solution (8.4 mg/ml) in 5 mM Mes pH 6.5 mixed with an equivalent volume of crystallisation buffer. Crystals appeared under several different conditions using Crystal Screen buffers (Hampton Research). Single cubic crystals (
Crystals of the NY-ESO TCR were flash-frozen and tested for diffraction in the X-ray beam of the Daresbury synchrotron. The crystals diffracted to 0.25mn (2.5A) resolution. One data set was collected and processed to give a 98.6% complete set of amplitudes that were reasonable to around 0.27 mn (2.7A), but usable up to 0.25 nm (2.5A). The merging R-factors i.e. the agreement between multiple measurements of crystallographically equivalent reflections, was 10.8% for all the data. This is marginal at the highest resolution shell. The space group was P2i, with cell dimensions a=4.25 nm (42.5A), b=5.95 nm (59.5A), c=8.17 nm (81.7 A), p=91.5°. The cell dimensions and symmetry meant there were two copies in the cell. The asymmetric unit, au or the minimum volume that needs to be studied, has only 1 molecule, and the other molecule in the cell is generated by the 2\ symmetry operation. The positioning of the molecule in the au is arbitrary in the y-direction. As long as it is in the correct position in the x-z plane, it can be translated at will in the y-direction. This is referred to as a free parameter, in this 'polar1 space group.<br>
The PDB data base has only one entry containing an A/B heterodimeric TCR, 1BD2. This entry also has co-ordinates of the HLA-cognate peptide in complex with the TCR. The TCR chain B was the same in NY-ESO, but chain A had small differences in the C-domain and significant differences in the N-domain. Using the 1BD2 A/B model for molecular replacement, MR, gave an incorrect solution, as shown by extensive overlap with symmetry equivalent molecules. Using the B chain alone gave a better solution, which did not have significant clashes with neighbours. The<br><br>
correlation coefficient was 49%, the crystallographic R-factor 50%, and the nearest approach (centre-of-gravity to c-o-g) was 0.49 run (49A). The rotation and translation operation needed to transform the starting chain B model to the MR equivalent, was applied to chain A. The hybrid MR solution thus generated, packed well in the cell, with minimal clashes.<br>
Electron density maps generally agreed with the model, and allowed its adjustment to match the sequence of the NY-ESO TCR. But the starting model had many gaps, specifically missing side-chains, that are characteristic of poorly ordered portions of the model. Many of the hair-pin loops in between strands had very low density, and were difficult to model. The crystallographic R-factor of the model is 30%. TheR-factor is a residual, i.e. it is the difference between the calculated and observed amplitudes.<br>
As Figures 83a and 83b demonstrate, the input sequence from 1BD2 do not match up with the density very well. Changing the model for Cys at positions 164 in chain A, and 174 in chain B, followed by further refinement, showed clearly that this sequence assignment is much better fitted to the density. But the differences in terms of size of the side chain are minimal, so there was little perturbation in the model. The electron density in that region is little changed.<br>
The most important aspect of this work is that the new TCR is very similar in structure to the published model (1BD2). The comparison could include all of the TCR, the constant domains, or the small part near the mutation point.<br>
The r.m.s deviation values are listed in the table below. The comparison of structures is shown in Figure 84.<br><br><br>
The short stretch refers to the single strand from Chain A (A157 to A169) and the single strand from Chain B (B170 to B183) that are now joined by the disulphide bridge. The deviations were calculated for only the main chain atoms.<br>
These results show that the introduction of the disulphide bond has minimal effect on the local structure of the TCR around the bond. Some larger effects are observed when comparing the TCR to the published structure (1BD2) of the A6 TCR, but the increase in RMS displacement is largely due to differences in loop conformations (see Figure 84). These loops do not form part of the core structure of the TCR, which is formed by a series of P-sheets which form a characteristic Ig fold. The RMS deviation for the whole a-chain is particularly large because of the difference in the sequence of the variable domains between the A6 (1BD2) and the NY-ESO TCRs. However, the A6 and NY-ESO TCRs have the same variable P-domain and the RMS deviations for the whole P-chain show that the structure of this variable domain is also maintained in the TCR with the new disulphide bond. These data therefore indicate that the core structure of the TCR is maintained in the crystal structure of the TCR with the new disulphide bond.<br>
Example 10—Production of soluble NY-ESO TCRs containing a novel disulphide inter-chain bond, and C- terminalj3 chain tagging sites.<br>
In order to produce a soluble NY-ESO TCR incorporating a novel disulphide bond, A6 TCR plasmids containing the a chain BamHI and P chain Bgin restriction sites were used as frameworks as described in Example 4.<br><br>
NY-ESO TCR P-chain constructs were obtained by PCR cloning as follows. PCR reactions were performed using the primers as shown below, and templates containing the NY-ESO TCR chains.<br>
| Ndel | Fwd5' -GGAGATATACATATGGGTGTCACTCAGAAC-3'<br>
Rev5' -CCACCGGATCCGTCTGCTCTACCCCAGGC-3 '<br>
| BamHl|<br>
The PCR products were restriction digested with the relevant restriction enzymes, and cloned into pGMT7 containing the biotin recognition sequence to obtain expression plasmids. The sequence of the plasmid inserts were confirmed by automated DNA sequencing. Figure 85a shows the DNA sequence of the j8 chain of the NY-ESO TCR incorporating the biotin recognition site, and Figure 85b shows the resulting amino acid sequence.<br>
The a chain construct was produced as described in Example 5. The respective TCR chains were expressed, co-refolded and purified as described in Example 5.<br>
In order to produce a soluble NY-ESO TCR containing a non-native disulphide interchain bond and a hexa-histidine tag on the C- terminus of the P chain, the same primers and NY-ESO template were used as above. The PCR products were restriction digested with the relevant restriction enzymes, and cloned into pGMT7 containing the hexa-histidine sequence to obtain expression plasmids. Figure 86a shows the DNA sequence of the j8 chain of the NY-ESO TCR incorporating the hexa-histidine tag, and Figure 86b shows the resulting amino acid sequence.<br>
Figure 87 illustrates the elution of soluble NY-ESO TCR containing a novel disulphide bond and the biotin recognition sequence from a POROS 50HQ anion exchange column using a 0-500 mM NaCl gradient, as indicated by the dotted line. Figure 88 illustrates the elution of soluble NY-ESO TCR containing a novel<br><br>
disulphide bond and the hexa-histidine tag from a POROS 50HQ anion exchange columns using a 0-500 mM NaCl gradient, as indicated by the dotted line.<br>
Figures 89 and 90 are protein elution profiles from gel filtration chromatography of pooled fractions from the NY-ESO-biotin and NY-ESO-hexa-histidine tagged anion exchange column runs illustrated by Figures 87 and 88 respectively. The protein elutes as a single major peak, corresponding to the TCR heterodimer.<br>
A BIAcore analysis of sTCR binding to pMHC was carried out as described in Example 3. The NY-ESO-biotin TCR had a Kd of 7.5 /JM, The NY-ESO-hexa-histidine tagged TCR had a Kd of 9.6 fM<br>
Example 11 - Cell staining using fluorescent labelled tetramers of soluble NY-ESO TCR containing a novel disulphide inter-chain bond.<br>
TCR Tetramer preparation<br>
The NY-ESO soluble TCRs containing a novel disulphide bond and a biotin recognition sequence prepared as in Example 10 were utilised to form the soluble TCR tetramers using required for cell staining. 2.5 ml of purified soluble TCR solution (~ 0.2 mg/ml) was buffer exchanged into biotinylation reaction buffer (50 mM Tris pH 8.0,10 mM MgCl2) using a PD-dO column (Pharmacia). The eluate (3.5 ml) was concentrated to 1 ml using a centricon concentrator (Amicon) with a 10 kDa molecular weight cut-off. This was made up to lOmM with ATP added from stock (0.1 g/ml adjusted to pH 7.0). A volume of a cocktail of protease inhibitors was then added (protease inhibitor cocktail Set 1, Calbiochem Biochemicals ), suflBcient to give a final protease cocktail concentration of 1/100th of the stock solution as supplied, followed by 1 mM biotin (added from 0.2M stock) and 20 ng/ml enzyme (from 0.5 mg/ml stock). The mixture was then incubated overnight at room temperature. Excess biotin was removed from the solution by size exclusion chromatography on a S75 HR cloumn. The level of biotinylation present on the NY-ESO TCR was determined via a size exclusion HPLC-based method as follows. A 50ul aliquot of the biotinylated NY-ESO TCR (2mg/ml) was incubated with 50ul of streptavidin coated agarose beads<br><br>
(Sigma) for 1 hour. The beads were then spun down, and 50 pi of the unbound sample was run on a TSK 2000 SW column (Tosoohaas) using a 0.5ml/min flowrate (200mM Phosphate Buffer pH 7.0) over 30 minutes. The presence of the biotinylated NY-ESO TCR was detected by a UV spectrometer at both 214nm and 280nm. The biotinylated NY-ESO was run against a non-bioninylated NY-ESO TCR control. The percentage of biotinylation was calculated by subtracting the peak-area of the biotinylated protein from that of the non-biotinylated protein.<br>
Tetramerisation of the biotinylated soluble TCR was achieved using neutravidin-phycoerythrin conjugate (Cambridge Biosciences, UK). The concentration of biotinylated soluble TCR was measured using a Coomassie protein assay (Pierce), and a ratio of soluble TCR 0.8 mg/mg neutravidin-phycoerthrin conjugate was calculated to achieve saturation of the neutravidin-PE by biotinylated TCR at a ratio of 1:4. 19.5pl of a 6.15mg/ml biotinylated NY-ESO soluble TCR solution in phosphate buffered saline (PBS) was added slowly to 150 pi of a lmg/ml neutravidin-PE soluble over ice with gentle agitation. 100.5 |il of PBS was then added to this solution to provide a final NY- ESO TCR tetramer concentration of 1 mg/ml.<br>
Staining Protocol<br>
Four aliquots of 0.3xl06 HLA-A2 positive EBV transformed B cell line (PP LCL) in 0.5ml of PBS were incubated with varying concentrations (0,104, 10'5 and 10"6 M) of HLA-A2 NYESO peptide (SLLMWITQC) for 2 h at 37°C. These PP LCL cells were then washed twice in Hanks buffered Saline solution (HBSS) (Gibco, UK).<br>
Each of the four aliquots were divided equally and stained with biotinylated NY-ESO disulphide linked TCR freshly tetramerised with neutravidin-phycoerythrin. Cells were incubated with either 5 or 10 /ig of phycoerythrin labelled tetrameric dsTCR complexes on ice for 30 minutes and washed with HBSS. Cells were washed again, re-suspended in HBSS and analysed by FACSVantage. 25,000 events were collected and data analysed using WinMIDI software.<br><br>
Results	..        .<br>
Figures 91a-h illustrate as histograms the FACS Vantage data generated for each of the samples prepared as described above. The following table lists the percentage of positively stained cells observed for each of the samples:<br><br>
These data clearly indicate that the proportion of the cells labelled by the NY-ESO TCR tetramers increases in a manner correlated to the concentration of the peptide (SLLMWITQC) in which they had been incubated. Therefore, these NY-ESO TCR tetramers are moieties suitable for specific cell labelling based on the expression of the HLA-A2 NY-ESO complex.<br>
In the present example, a fluorescent conjugated NY-ESO TCR tetramer has been used. However, similar levels of cell binding would be expected if this label were replaced by a suitable therapeutic moiety.<br>
Example 12—Production of soluble A6 TCR with a novel disulphide bond incorporating the C/31 constant region.<br>
All of the previous examples describe the production of soluble TCRs with a novel disulphide bond incorporating the Cp2 constant region. The present example<br><br>
demonstrates that soluble TCRs incorporating the Cpi constant region can be produced successfully.<br>
Design of primers for PCR stitching ofA6 TCR fi-chain V-domain to Cfil. For PCR construct of A6 TCR p-chain V-domain, the following primers were<br>
designed:<br>
59-GGAGATATACATATGAACGCTGGTGTCACT-3' 55-CCTTGTTCAGGTCCTCTGTGACCGTGAG-3'<br>
For PCR construct of Cpi, the following primers were designed:<br>
5 '-CTCACGGTCACAGAGGACCTGAACAAGG-3»<br>
5' .CCCAAGCTTAGTCTGCTCTACCCCAGGCCTCGGC-3»<br>
Beta VTCR construct and Cpl construct were separately amplified using standard PCR technology. They were connected to each other using a stitching PCR. Plasmid DNA was purified on a Qiagen mini-prep column according to the manufacturer's instructions and the sequence was verified by automated sequencing at the sequencing facility of Department of Biochemistry, Oxford University. The sequence for A6+Cpl is shown in Figure 92.<br>
Consequently, the A6+CP1 chain was paired to A6 alpha TCR by inter-chain disulphide bond after introducing cysteine in C-domain of both chains.<br>
The soluble TCR was expressed and refolded as described in Example 2.<br>
Purification of refolded soluble TCR:<br>
sTCR was separated from degradation products and impurities by loading the dialysed refold onto a POROS 50HQ anion exchange column and eluting bound protein with a gradient of 0-500mM NaCI over 50 column volumes using an Akta purifier<br><br>
(Pharmacia) as in Figure 93. Peak fractions were stored at 4°C and analysed by Coomassie-stained SDS-PAGE (Figure 94) before being pooled and concentrated. Finally, the sTCR was purified and characterised using a Superdex 200HR gel filtration column (Figure 95) pre-equilibrated in HBS-EP buffer (10 mM HEPES pH 7.4,150 mM NaCl, 3.5 mM EDTA, 0.05% nonidet p40). The peak eluting at a relative molecular weight of approximately 50 kDa was pooled and concentrated prior to characterisation by BIAcore surface plasmon resonance analysis.<br>
A BIAcore analysis of the binding of the disulphide-linked A6 TCR to pMHC was carried out as described in Example 3. Figure 96 shows BIAcore analysis of the specific binding of disulphide-linked A6 soluble TCR to its cognate pMHC.<br>
The soluble A6 TCR with a novel disulphide bond incorporating the Cpi constant region had a K^ of 2.42 ± 0.55 \xM for its cognate pMHC. This value is very similar to the K<j of determined for the soluble a6 tcr with a novel disulphide bond incorporating cp2 constant region as in example></j>
Example 13— Production of soluble A6 TCR with a novel disulphide bond incorporating the "free" cysteine in the j8 chain<br>
The 0 chain constant regions of TCRs include a cysteine residue (residue 75 in exon 1 of TRBC1*01 and TRBC2*01) which is not involved in either inter-chain or intra-chain disulphide bond formation. All of the previous examples describe the production of soluble TCRs with a novel disulphide bond in which this "free" cysteine has been mutated to alanine in order to avoid the possible formation of any "inappropriate" disulphide bonds which could result in a reduced yield of functional TCR. The present example demonstrates that soluble TCRs incorporating this "free" cysteine can be produced.<br><br>
Design of primers and mutagenesis of TCR |8 chain<br>
For mutating TCR /5-chain alanine (residue 75 in exon 1 of TRBC1*01 and<br>
TRBC2*01) to cysteine, the following primers were designed (mutation shown in<br>
lowercase):<br>
5'-T GAC TCC AGA TAC tgT CTG AGC AGC CG 5 J-CG GCT GCT CAG Aca GTA TCT GGA GTC A<br>
PCR mutagenesis, expression and refolding of the soluble TCR was carried out as described in Example 2.<br>
Purification of refolded soluble TCR:<br>
sTCR was separated from degradation products and impurities by loading the dialysed refold onto a POROS 50HQ anion exchange column and eluting bound protein with a gradient of 0-500mM NaCI over 50 column volumes using an Akta purifier (Pharmacia) as in Figure 98. Peak fractions were stored at 4°C and analysed by Coomassie-stained SDS-PAGE (Figure 99) before being pooled and concentrated. Finally, the sTCR was purified and characterised using a Superdex 200HR gel filtration column (Figure 100) pre-equilibrated in HBS-EP buffer (10 mM HEPES pH 7.4,150 mM NaCI, 3.5 mM EDTA, 0.05% nonidet p40). The peak eluting at a relative molecular weight of approximately 50 kDa was pooled and concentrated prior to characterisation by BIAcore surface plasmon resonance analysis.<br>
A BIAcore analysis of the binding of the disulphide-linked A6 TCR to pMHC was carried out as described in Example 3. Figure 101 shows BIAcore analysis of the specific binding of disulphide-linked A6 soluble TCR to its cognate pMHC.<br>
The soluble A6 TCR with a novel disulphide bond incorporating the "free" cysteine in the 0 chain had a Kd of 21.39 ± 3.55 ^M for its cognate pMHC.<br><br>
Example 14 - Production of soluble A6 TCR with a novel disulphide bond wherein "free" cysteine in the fi chain is mutated to serine.<br>
The present example demonstrates that soluble TCRs with a novel disulphide bond in which the "free" cysteine in the /3 chain (residue 75 in exon 1 of TRBC1*01 and TKBC2*01) is mutated to serine can be successfully produced.<br>
Design of primers and mutagenesis of TCR j8 chain<br>
For mutating TCR (3-chain alanine that had previously been substituted for the native cysteine (residue 75 in exon 1 of TRBC1*01 and TRBC2*01) to serine, the following primers were designed (mutation shown in lower case):<br>
5'-T GAC TCC AGA TAC tCT CTG AGC AGC CG 5'-CG GCT GCT CAG AGa GTA TCT GGA GTC A<br>
PCR mutagenesis (resulting in a mutated beta chain as shown in Figure 102), expression and refolding of soluble TCR was carried out as described in Example 2.<br>
Purification of refolded soluble TCR:<br>
sTCR was separated from degradation products and impurities by loading the dialysed refold onto a POROS 50HQ anion exchange column and eluting bound protein with a gradient of 0-500mM NaCI over 50 column volumes using an Akta purifier (Pharmacia) as shown in Figure 103. Peak fractions were stored at 4°C and analysed by Coomassie-stained SDS-PAGE (Figure 104) before being pooled and concentrated. Finally, the sTCR was purified and characterised using a Superdex 200HR gel filtration column (Figure 105) pre-equilibrated in HBS-EP buffer (10 mM HEPES pH 7.4,150 mM NaCl, 3.5 mM EDTA, 0.05% nonidet p40). The peak eluting at a relative molecular weight of approximately 50 kDa was pooled and concentrated prior to characterisation by BIAcore surface plasmon resonance analysis.<br><br>
A BIAcore analysis of the binding of the disulphide-linked A6 TCR to pMHC was carried out as described in Example 3. Figure 106 shows BIAcore analysis of the specific binding of disulphide-linked A6 soluble TCR to its cognate pMHC.<br>
The soluble A6 TCR with a novel disulphide bond in which the "free" cysteine in the /J chain was mutated to serine had a Ka of 2.98 ± 0.27 pM for its cognate pMHC. This value is very similar to the Kd of 1.8 |iM determined for the soluble A6 TCR with a novel disulphide bond in which the "free" cysteine in the j8 chain was mutated to alanine as determined in Example 3.<br>
Example 15 - Cloning ofNY-ESO TCR a and J3 chains containing a novel disulphide bond into yeast expression vectors<br>
NY-ESO TCR a and p chains were fused to the C-terminus of the pre-pro mating factor alpha sequence from Saccharomyces cerevisiae and cloned into yeast expression vectors pYX122 and pYX112 respectively (see Figures 107 and 108).<br>
The following primers were designed to PCR amplify pre-pro mating factor alpha sequence from S. cerevisiae strain SEY6210 (Robinson et ah (1991), Mol Cell Biol 11(12):5813-24) for fusing to the TCR a chain.<br>
5'-TCT GAA TTC ATG AGA TTT CCT TCA ATT TTT AC-3' 5'-TCA CCT CCT GGG CTT CAG CCT CTC TTT TAT C -3'<br>
The following primers were designed to PCR amplify pre-pro mating factor alpha sequence from S. cerevisiae strain SEY6210 for fusing to the TCR P chain.<br>
5'-TCT GAA TTC ATG AGA TTT CCT TCA ATT TTT AC-3' 5'-GTG TCT CGA GTT AGT CTG CTC TAC CCC AGG C-3'<br>
Yeast DNA was prepared byre-suspending a colony of S. cerevisiae strain SEY6210 in 30/xl of 0.25% SDS in water and heating for 3 minutes at 90°C. Thepre-pro mating<br><br>
factor alpha sequences for fusing to the TCR a and p chains were generated by PCR amplifing 0.25/d of yeast DNA with the respective primer pairs mentioned above using the following PCR conditions. 12.5pmoles of each primer was mixed with 200JUM dNTP, 5/xl of 1 Ox Pfu buffer and 1.25units of Pfu polymerase (Stratagene) in a final volume of 50/d. After an initial denaturation step of 30 seconds at 92°C, the reaction mixture was subjected to 30 rounds of denaturation (92°C, 30 sec), annealing (46.9°C, 60 sec), and elongation (72°C&gt; 2 min.) in a Hybaid PCR express PCR machine.<br>
The following primers were designed to PCR amplify the TCR a chain to be fused to the pre-pro mating factor alpha sequence mentioned above.<br>
5'-GGC TGA AGC CCA GGA GGT GAC ACA GAT TCC-3' 5'-CTC CTC TCG AGT TAG GAA CTT TCT GGG CTG GG-3'<br>
The following primers were designed to PCR amplify the TCR p chain to be fused to the pre-pro mating factor alpha sequence mentioned above.<br>
5'-GGC TGA AGC CGG CGT CAC TCA GAC CCC AAA AT-3' 5'-GTG TCT CGA GTT AGT CTG CTC TAC CCC AGG C-3'<br>
The PCR conditions for amplifying the TCR a and P chains were the same as mentioned above except for the following changes: the DNA template used for amplifying the TCR a and p chains were the NY-ESO TCR a and p chains respectively (as prepared in Example 5); and the annealing temperature used was 60.1°C.<br>
The PCR products were then used in a PCR stitching reaction utilising the complementary overlapping sequences introduced into the initial PCR products to create a full length chimeric gene. The resulting PCR products were digested with the restriction enzymes EcoR I and Xho I and cloned into either pYX122 or pYXl 12 digested with the same enzymes. The resulting plasmids were purified on a Qiagen™<br><br>
mini-prep column according to the manufacturers instructions, and the sequences verified by automated sequencing at the sequencing facility of Genetics Ltd, Queensway, New Milton, Hampshire, United Kingdom. Figures 109 and 110 show the DNA and protein sequences of the cloned chimeric products.<br>
Example 16- Expression of soluble NY-ESO TCR containing a novel disulphide bond in yeast<br>
The yeast expression plasmids containing the TCR a and (3 chains respectively produced as described in Example 15 were co-transformed into S. cerevisiae strain SEY6210 using the protocol by Agatep et aL (1998) (Technical Tips Online (http://tto.trends.com) 1:51 :P01525). A single colony growing on synthetic dropout (SD) agar containing Histidine and Uracil (Qbiogene, HUrirch, France) was cultured overnight at 30°C in 10ml SD media containing Histidine and Uracil. The overnight culture was sub-cultured 1:10 in 10ml of the fresh SD media containing Histidine and Uracil and grown for 4 hours at 30°C. The culture was centrifuged for 5 minutes at 3800rpm in a Heraeus Megafiige 2.0R (Kendro Laboratory Products Ltd, Bishop's Stortford, Hertfordshire, UK) and the supernatant harvested. 5/xl StratClean Resin (Stratagene) was mixed with the supematent and kept rotating in a blood wheel at 4°C overnight. The StrataClean resin was spun down at 3800rpm in a Heraeus Megafuge 2.0R and the media discarded. 25/d of reducing sample buffer (950/xl of Laemmli sample buffer (Biorad) containing 50fil of 2M DTT) was added to the resin and the samples heated at 95°C for 5 minutes and then cooled on ice before 20/il of the mix was loaded on a SDS-PAGE gel at 0.8mA constant /cm2 of gel surface for 1 hour. The proteins in the gel were transferred to hnmuno-Blot PVDF membranes (Bio-Rad) and probed with TCR anti a chain antibody as described in Example 17 below except for the following changes. The primary antibody (TCR anti a chain) and secondary antibodies were used at 1 in 200 and 1 in 1000 dilutions respectively. Figure 111 shows a picture of the developed membrane. The result shows that there is a low level of TCR secretion by the yeast culture into the media.<br><br>
Example 17 - Bisulphide A6 Tax TCR a and /? chain expression in Baculovirus<br>
Strategy for cloning<br>
The a and p chains of the disulphide A6 Tax TCR were cloned from pGMT7 into a pBlueScript KS2- based vector called the pEX172. This vector was designed for cloning different MHC class II P-chains, for insect cell expression, using the leader sequence from DKB1*0101, an Agel site for insertion of different peptide-coding sequences, a linker region, and then Mlul and Sail sites to clone the DRp chains in front of the Jun Leucine zipper sequence. The sequence where pEX 172 differs from pBlueScript IIKS-, located between the Kpnl and EcoRI sites of pBlueScript IIKS-, is shown in Figure 112. For the purposes of cloning TCR chains in insect cells, this pEX172 was cut with Agel and Sail to remove the linker region and Mlul site, and the TCR chains go in where the peptide sequence would start. The TCR sequences were cloned from pGMT7 with a BspEI site at the 5' end (this had Agel compatible sticky ends) and a Sail site at the 3 s end. In order to provide the cleavage site for the removal of the DRP leader sequence, the first three residues of the DRp chain (GDT) were preserved. In order to prevent the Jim Leucine zipper sequence being transcribed, it was necessary to insert a stop codon before the Sail site. For a schematic of this construct, see Figure 113. Once the TCR chains are in this plasmid, the BamHI fragment were cut out and subcloned into the pAcAB3 vector, which has homology recombination sites for Baculovirus. The pAcAB3 vector has two divergent promoters, one with a BamHI site and one with a BglEf cloning site. There is a Bgin site in the A6 TCR P-chain, so the A6 TCR a-chain was inserted into the Bgin site, and the P-chain was then subcloned into the BamHI site.<br>
In accordance with the above cloning strategy, the following primers were designed (homology to the vectors is in uppercase):<br>
A6ct: F: 5' -gtagtccggagacaccggaCAGAAGGAAGTGGAGCAGAAC<br>
R: 5'-gtaggtcgacTAGGAACTTTCTGGGCTGGG A6P: F: 5' -gtagtccggagacaccggaAACGCTGGTGTCACTCAGA<br><br>
R:   5'-gtaggtcgacTAGTCTGCTCTACCCCAGG<br>
PCR, cloning and sub-cloning:<br>
Expression plasmids containing the genes for the disulphide A6 Tax TCR a or P chain were used as templates in the following PCR reactions. lOOng of a plasmid was mixed with lp.1 lOmM dNTP, 5|il lGxPfu-buffer (Stratagene), 1.25 units Pfu polymerase (Stratagene), 50pmol of the A6a primers above, and the final volume was adjusted to 50^1 with H2O. A similar reaction mixture was set up for the P chain, using the P plasmid and the pair of P primers. The reaction mixtures were subjected to 35 rounds of denaturation (95°C, 60 sec), annealing (50°C, 60 sec.)/and elongation (72°C, 8 min.) in a Hybaid PCR express PCR machine. The product was then digested for 2 hours at 37°C with 10 units of BspEI restriction enzyme then for a further 2 hours with 10 units of Sail (New England Biolabs). These digested reactions were ligated into pEX172 that had been digested with Agel and Sail, and these were transformed into competent XLl-Blue bacteria and grown for 18 hours at 37°C. A single colony was picked from each of the a and P preps and grown over night in 5 ml TYP + ampicillin (16 g/1 Bacto-Tryptone, 16 g/1 Yeast Extract, 5 g/1 NaCl, 2.5 g/1 K2HP04,100 mg/1 Ampicillin). Plasmid DNA was purified on a QIAgen mini-prep column according to the manufacturer's instructions and the sequence was verified by automated sequencing at the sequencing facility of Genetix. The amino acid sequences of the BamHI inserts are shown in Figures 114 and 115 for the a chain and P chain, respectively.<br>
These a and P disulphide A6 Tax TCR chain constructs in pEX172 were digested out for 2 hours at 37°C with BamHI restriction enzyme (New England Biolabs). The a chain BamHI insert was ligated into pAcAB3 vector (Pharmingen-BD Biosciences: 21216P) that had been digested with BgUI enzyme. This was transformed into competent XLl-Blue bacteria and grown for 18 hours at 37°C. A single colony was picked from this plate and grown overnight in 5 ml TYP + ampicillin and the plasmid DNA was purified as before. This plasmid was then digested with BamHI and the P chain BamHI insert was ligated in, transformed into competent XLl-Blue bacteria,<br><br>
grown overnight, picked to TYP-ampicillin, and grown before miniprepping as before using a QIAgen mini-prep column. The correct orientation of both the a and P chains were confirmed by sequencing using the following sequencing primers:<br>
pAcAB3 a forwards: 5'-gaaattatgcatttgaggatg pAcAB3 p forwards: 5'-attaggcctctagagatccg<br>
Transfection, infection, expression and analysis ofA6 TCR in insect cells The expression plasmid containing the a-chain and P-chain was transfected into sf9 cells (Pharmingen-BD Biosciences: 21300C) grown in serum free medium (Pharmingen-BD Biosciences: 551411), using the Baculogold transfection kit (Pharmingen-BD Biosciences: 21100K) as per the manufacturers instructions. After 5 days at 27°C? 200nl of the medium these transfected cells had been growing in was added to 100ml of High Five cells at IxlO6 cells/ml in serum free medium. After a further 6 days at 27°C, 1ml of this medium was removed and centrifbged at 13,000RPM in a Hereus microfuge for 5 minutes to pellet cell debris.<br>
lOpl of this insect A6 disulphide linked TCR supernatant was run alongside positive controls of bacterial A6 disulphide linked TCR 5|ig and lOjig on a pre-cast 4-20% Tris/glycine gel (Invitrogen: EC60252). Reduced samples were prepared by adding 10p,l of Reducing sample buffer (950\xl of Laemmli sample buffer (Bio-Rad: 161-0737) 50^1 of 2M DTT) and heating at 95°C for 5 minutes, cooling at room temperature for 10 minutes then loading 20|il. Non-reduced samples were prepared by adding lOjul of Laemmli sample buffer, and loading 20JJ1<br>
The gel was ruft at 150 volts for 1 hour in a Novex - Xcell gel tank after which the gel was stained in 50ml of Coomassie gel stain for 1 hour with gentle agitation (l.lg Coomassie powder in 500ml of methanol stir for 1 hour add 100ml acetic acid make up to 1 litre with H2O and stir for 1 hour then filter through 0.45pM filter). The gel was de-stained three times for 30 mins with gentle agitation in 50ml of de-stain (as Coomassie gel stain but omitting the Coomassie powder).<br><br>
Western Blots were performed by running SDS-PAGE gels as before but the proteins were transferred to Immuno-Blot PVDF membranes (Bio-Rad: 162-0174) rather than staining the gels with Coomassie. Six filter papers were cut to the size of the gel and soaked in transfer buffer (2.39g Glycine, 5.81g of Tris Base, 0.77g DTT dissolved in 500mls of H20? 200mls of methanol added then made up to lOOOmls with H20). The PVDF membrane was prepared by soaking in methanol for 1 minute and then in transfer buffer for 2 minutes. Three filter papers were placed on the anode surface of the hnmno-blot apparatus (Pharmacia - Novablot) then the membrane was placed on top followed by the gel and then finally three more filter papers on the cathode side. The Immuno-blot was run for 1 hour at 0.8mA constant /cm2 of gel surface.<br>
After blotting, the membrane was blocked in 7.5mls of blocking buffer (4 Tris-buffered saline tablets (Sigma: T5030), 3g non-fat dried milk (Sigma: M7409), 30p,l of Tween 20 made up to 30mls with H2O) for 60 mins with gentle agitation. The membrane was washed three times for 5 niins with TBS wash buffer (20 TBS tablets, 150|a.l Tween 20 made up to 300ml with H20). The membrane was then incubated in primary antibody 1 in 50 dilution of anti TCR a chain clone 3A8 (Serotec: MCA987) or anti TCR p chain clone 8A3 (Serotec: MCA988) in 7.5ml blocking buffer for I hour with gentle agitation. The membrane was washed as before in TBS wash buffer. Next, a secondary antibody incubation of HRP labelled goat anti-mouse antibody (Santa Cruz Biotech: Sc-2005) 1 in 1000 dilution in 7.5ml of blocking buffer was carried out for 30 min with gentle agitation. The membrane was washed as before and then washed in 30ml of H20 with 2 TBS tablets.<br>
The antibody binding was detected by Opti-4CN colourmetric detection (Biorad: 170-8235) (1.4ml Opt-4CN diluent, 12.6ml H20,0.28ml Opti-4CN substrate). The membranes were coloured for 30 minutes and then washed in H20 for 15 minutes. The membranes were dried at room temperature, and scanned images were aligned with an image of the coomassie stained gel (Figure 116).<br><br>
Results<br>
It can be seen from Figure 116 that both disulphide TCRs are formed as a heterodimer that is stable in the SDS gel. They both break into the a and P chains upon reduction. The insect disulphide TCR heterodimer has a slightly higher molecular weight that the bacterially produced version, presumably because of the glycosylation from the insect cells. It can be seen that in this instance the insect cells are producing a chain in excess, and free a chain can be seen in the non-reduced lane of the anti-a western blot.<br>
These data clearly demonstrate that the baculovirus expression system described above provides a viable alternative to prokaryotic expression of soluble TCRs containing novel disulphide bonds.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	A soluble T cell receptor (sTCR), which comprises (i) all or part of a TCR a<br>
chain, except the transmembrane domain thereof, and (ii) all or part of a TCRp chain,<br>
except the transmembrane domain thereof, wherein (i) and (ii) each comprise a<br>
functional variable domain and at least a part of the constant domain of the TCR<br>
chain, characterised in that (i) and (ii) are linked by a disulphide bond between<br>
cysteine residues substituted for:<br>
Thr48ofexon 1 ofTRAC*01 and Ser 57 of exon 1 ofTRBCl*01 orTRBC2*01; Thr45ofexon 1 ofTRAC*01 and Ser 77 of exon 1 ofTRBCl*01 orTRBC2*01; Tyr 10 of exon 1 of TRAC*01 and Ser 17 of exon 1 of TRBC1*01 or TRBC2*01; Thr45ofexon 1 ofTRAC*01 and Asp 59 of exon 1 ofTRBCl*01 or TRBC2*01; or Ser 15 of exon 1 of TRAC*01 and Glu 15 of exon 1 ofTRBCl*01 orTRBC2*01;<br>
2.	The sTCR as claimed in claim 1, wherein one or both of (i) and (ii) comprise all of the extracellular constant Ig domain of the TCR chain.<br>
3.	The sTCR as claimed in claim 1 or claim 2, wherein one or both of (i) and (ii) comprise all of the extracellular domain of the TCR chain.<br>
4.	A soluble aP-form T cell receptor (sTCR), wherein a covalent disulphide bond links cysteine residues substituted for:<br>
Thr48ofexon 1 ofTRAC*01 and Ser 57 of exon 1 ofTRBCl*01 orTRBC2*01; Thr45ofexon 1 ofTRAC*01 and Ser 77 of exon 1 ofTRBCl*01 orTRBC2*01; Tyr 10 of exon 1 of TRAC*01 and Ser 17 of exon 1 of TRBC1*01 or TRBC2*01; Thr 45 of exon 1 of TRAC*01 and Asp 59 of exon 1 of TRBC1*01 or TRBC2*01; or Ser 15 of exon 1 of TRAC*01 and Glu 15 of exon 1 of TRBC1*01 or TRBC2*01;<br><br>
5.	A soluble T cell receptor (sTCR), which comprises (i) all or part of a TCR a chain, except the transmembrane domain thereof, and (ii) all or part of a TCRP chain, except the transmembrane domain thereof, wherein (i) and (ii) each comprise a functional variable domain and at least a part of the constant domain of the TCR chain, and are linked by a disulphide bond between constant domain residues which is not present in native TCR and wherein an interchain disulphide bond in native TCR is not present.<br>
6.	The sTCR as claimed in claim 5, wherein one or both of (i) and (ii) comprise all of the extracellular constant Ig domain of the TCR chain.<br>
7.	The sTCR as claimed in claim 5 or claim 6, wherein one or both of (i) and (ii) comprise all of the extracellular domain of the TCR chain.<br>
8.	A soluble ap-form T cell receptor (sTCR), wherein a covalent disulphide bond<br>
links a residue of the immunoglobulin region of the constant domain of the a chain to<br>
a residue of the immunoglobulin region of the constant domain of the p chain, wherein an interchain disulphide bond in native TCR is not present.<br>
9.	The sTCR as claimed in any one of claims 5 to 8, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for residues whose P carbon atoms are less than 0.6 nm apart in the native TCR structure.<br>
10.	The sTCR as claimed in any one of claims 5 to 9 wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Thr 48 ofexon 1 ofTRAC*01 and Ser 57 of exon 1 ofTRBCl*01 orTRBC2*01.<br>
11.	The sTCR as claimed in any one of claims 5 to 9, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Thr 45 ofexon 1 of TRAC*01 and Ser77ofexon 1 of TRBC1*01 orTRBC2*01.<br><br>
12.	The sTCR as claimed in any one of claims 5 to 9, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Tyr 10 of exon 1 of TRAC*01 and Ser 17 of exon 1 of TRBC1*01 or TRBC2*01.<br>
13.	The sTCR as claimed in any one of claims 5 to 9, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Thr 45 of exon 1 ofTRAC*01 and Asp 59 of exon 1 ofTRBCl*01 orTRBC2*01.<br>
14.	The sTCR as claimed in any one of claims 5 to 9, wherein disulphide bond which is not present in native TCR is between cysteine residues, substituted for Ser 15 of exon lofTRAC*01 and Glu 15 of exon 1 of TRBC1*01 or TRBC2*01.<br><br>
15.	The sTCR as claimed in any one of claims 1 to 4, wherein an interchain disulphide bond in native TCR is not present.<br>
16.	The sTCR as claimed in any one of claims 5 to 15, wherein native a and P TCR chains are truncated at the C-terminus such that the cysteine residues which form the native interchain disulphide bond are excluded.<br>
17.	The sTCR as claimed in any one of claims 5 to 15, wherein cysteine residues<br>
which form the native interchain disulphide bond are substituted to another residue.<br>
18.	The sTCR as claimed in claim 17, wherein cysteine residues which form the<br>
native interchain disulphide bond are substituted to serine or alanine.<br>
19.	The sTCR as claimed in any preceding claim, wherein an unpaired cysteine residue present in native TCR p chain is not present.<br>
20.	The sTCR as claimed in any one of claims 1,2,5,6 and 9 to 19, wherein (i) and (ii) each comprise the functional variable domain of a first TCR fused to all or part of the constant domain of a second TCR, the first and second TCRs being from the same species.<br><br>
21.	The sTCR as claimed in claim 20, wherein the constant domains of the second TCR are truncated N-terminal to the residues which form the non-native interchain disulphide bond.<br>
22.	The sTCR as claimed in any preceding claim, wherein one or both of the chains are derivatised with, or fused to, a moiety at its C or N terminus.<br>
23.	The sTCR as claimed in any preceding claim wherein one or both of the chains have a cysteine residue at its C and/or N terminus to which a moiety can be fused.<br>
24.	The sTCR as claimed in any preceding claim comprising a detectable label.<br>
25.	The sTCR as claimed in any preceding claim associated with a therapeutic agent.<br>
26.	The multivalent T cell receptor (TCR) complex comprising a plurality of sTCRs as claimed in any preceding claim.<br>
27.	A complex as claimed in claim 26, comprising a sTCR multimer.<br>
28.	The complex as claimed in claim 27, comprising two or three or four or more T cell receptor molecules associated with one another, preferably via a linker molecule.<br>
29.	The complex as claimed in claim 26, 27 or 28, wherein the sTCRs or sTCR<br>
multimers are present in a lipid bilayer or are attached to a particle:<br>
30.	A method for detecting MHC-peptide complexes, which comprises:<br>
(i)       providing a soluble TCR as claimed in any one of claims 1 to 25 or a multivalent T cell receptor complex as claimed in any one of claims 26 to 29;<br><br>
(ii) contacting the soluble TCR or multivalent TCR complex with the MHC-peptide complexes; and<br>
(iii) detecting binding of the soluble TCR or multivalent TCR complex to the MHC-peptide complexes.<br>
31.	A pharmaceutical formulation comprising a sTCR as claimed in any one of claims 1 to 25, and/or a multivalent TCR complex as claimed in any one of claims 26 to 29, together with a pharmaceutical^ acceptable carrier.<br>
32.	A nucleic acid molecule comprising a sequence encoding (i) or (ii) of a sTCR as claimed in any one of claims 1 to 25, or a sequence complementary thereto.<br>
33.	A vector comprising a nucleic acid molecule as claimed in claim 32.<br>
34.	A host cell comprising a vector as claimed in claim 33.<br>
35.	A method for obtaining (i) or (ii) as defined in any one of claims 1 to 25, which method comprises incubating a host cell as claimed in claim 34 under conditions causing expression of the peptide and then purifying the polypeptide.<br>
36.	A method as claimed in claim 35, further comprising mixing (i) and (ii) under suitable refolding conditions.<br>
37.	A method for obtaining a soluble T cell receptor (sTCR), which method comprises:<br>
incubating a host cell which comprises a vector comprising a nucleic acid molecule encoding (i) all or part of a TCR a chain, except the transmembrane domain thereof, and a host cell which comprises a vector comprising a nucleic acid molecule encoding (ii) all or part of a TCR p chain, except the transmembrane domain thereof under conditions causing expression of (i) and (ii) wherein (i) and (ii) each comprise a<br><br>
functional variable domain and at least a part of the constant domain of the TCR chain;<br>
purifying (i) and (ii); and<br>
mixing (i) and (ii) under refolding conditions such that they are linked by a disulphide bond between constant domain residues which is not present in native TCR.<br>
38.	A method as claimed in claim 37, wherein one or both of (i) and (ii) comprise all of the extracellular constant Ig domain of the TCR chain.<br>
39.	A method as claimed in claim 37 or claim 38, wherein one or both of (i) and (ii) comprise all of the extracellular domain of the TCR chain.<br>
40.	A method for obtaining a soluble ap-form T cell receptor (sTCR), which method comprises:<br>
incubating a host cell which comprises a vector comprising a nucleic acid molecule encoding aTCR a chain and a host cell which comprises a vector comprising a nucleic acid molecule encoding a TCR p chain under conditions causing expression of the respective TCR chains;<br>
purifying the respective TCR chains; and<br>
mixing the respective TCR chains under refolding conditions such that a covalent disulphide bond links a residue of the immunoglobulin region of the constant domain of the a chain to a residue of the immunoglobulin region of the constant domain of the p chain.<br>
41.	A method as claimed in any one of claims 37 to 40, wherein an interchain disulphide bond in native TCR is not present.<br>
42.	A method as claimed in claim 41, wherein native a and P TCR chains are truncated at the C-terminus such that the cysteine residues which form the native<br><br>
interchain disulphide.bond are excluded.<br>
43.	A method as claimed in claim 41, wherein cysteine residues which form the native interchain disulphide bond are substituted to another residue.<br>
44.	A method as claimed in claim 43, wherein cysteine residues which form the native interchain disulphide bond are substituted to serine or alanine.<br>
45.	A method as claimed in any one of claims 37 to 44, wherein an unpaired cysteine residue present in native TCR p chain is not present.<br>
46.	A method as claimed in any one of claims 37 to 45, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for residues whose p carbon atoms are less than 0.6 nm apart in the native TCR structure.<br>
47.	A method as claimed in any one of claims 37 to 46, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Thr48ofexon 1 ofTRAC*01 and Ser 57 of exon 1 ofTRBCl*01 orTRBC2*01.<br>
48.	A method as claimed in any one of claims 37 to 46, wherein the disulphide<br>
bond which is not present in native TCR is between cysteine residues substituted for<br>
Thr 45 of exon 1 of TRAC*01 and Ser 77 of exon 1 of TRBC1*01 or TRBC2*01.<br>
49.	A method as claimed in any one of claims 37 to 46, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Tyr 10 of exon 1 ofTRAC*01 and Ser 17 of exon 1 ofTRBCl*01 orTRBC2*01.<br>
50.	A method as claimed in any one of claims 37 to 46, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Thr 45 of exon 1 of TRAC*01 and Asp 59 of exon 1 of TRBC1 *01 or TRBC2*01.<br><br>
51.	A method as claimed in any one of claims 37 to 46, wherein the disulphide bond which is not present in native TCR is between cysteine residues substituted for Ser 15ofexon 1 ofTRAC*01 and Glu 15 of exon 1 ofTRBCl*01 orTRBC2*01.<br>
52.	A method as claimed in any one of claims 37, 38 and 41 to 51, wherein (i) and (ii) each comprise the functional variable domain of a first TCR fused to all or part of the constant domain of a second TCR, the first and second TCRs being from the same species.<br>
53.	A method as claimed in claim 52, wherein the constant domains of the second TCR are truncated N-terminal to the residues which form the non-native interchain disulphide bond.<br>
54.	A method as claimed in any one of claims 37 to 53, wherein one or both of the chains are derivatised with, or fused to, a moiety at its C or N terminus.<br>
55.	A method as claimed in any one of claims 37 to 54, wherein one or both of the chains have a cysteine residue at its C and/or N terminus to which a moiety can be fused.<br>
56.	A method as claimed in anyone of claims 37 to 55, wherein the sTCR further comprises a detectable label.<br>
57.	A method as claimed in any one of claims 37 to 57, wherein the STCR is associated with a therapeutic agent.<br>
58.	A method as claimed in any preceding claim, further comprising combining a plurality of sTCRs to form a multivalent T cell receptor (TCR) complex.<br>
59.	A method as claimed in claim 58, wherein the sTCRs are combined to form a sTCR multimer.<br><br>
60.	A method as claimed in claim 59, wherein two or three or four or more T cell receptor molecules are associated with one another, preferably via a linker molecule.<br>
61.	A method as claimed in claim 58, 59 or 60, wherein the sTCRs or sTCR multimers are combined in a lipid bilayer or are attached to a particle.<br>
62.	A method for detecting MHC-peptide complexes, which comprises:<br>
J (i) providing a soluble TCR produced by the method of any one of claims 37 to 57 or a multivalent T cell receptor complex produced by the method of any one of claims 58 to 61;<br>
(ii) contacting the soluble TCR or multivalent TCR complex with the MHC-peptide complexes; and<br>
(iii) detecting binding of the soluble TCR or multivalent TCR complex to the MHC-peptide complexes.<br>
63.	A pharmaceutical formulation comprising a sTCR. produced by the method of any one of claims 37 to 57, and/or a multivalent TCR complex produced by the method of any one of claims 58 to 61, together with a pharmaceutically acceptable carrier.<br>
64.	A soluble T cell receptor (sTCR) substantially as herein described with reference to the accompanying drawings.<br>
65.	A method for obtaining a soluble T cell receptor (sTCR) substantially as herein described with reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">647-chenp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtYXNzaWduZW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">647-chenp-2004-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">647-chenp-2004-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">647-chenp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">647-chenp-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtZm9ybTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-form18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQ3LWNoZW5wLTIwMDQtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">647-chenp-2004-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="212620-a-package-companion-user-interactive-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="212622-adaptors-for-inhalers-to-improve-performance.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>212621</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>647/CHENP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Feb-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Dec-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Mar-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVIDEX LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>57C MILTON PARK, ABINGDON, OXON OX14 4RX,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GLICK MEIR,</td>
											<td>63 GOLDEN STREET , STOUGHTON, MA 02072,USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JAKOBSEN, BENT , KARSTEN</td>
											<td>57C MILTON PARK, ABINGDON, OXON OX14 4RX, GREAT BRITAIN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/705</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB02/03986</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-08-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0219146.8</td>
									<td>2002-08-16</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0121187.9</td>
									<td>2001-08-31</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/404,182</td>
									<td>2002-08-16</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/212621-a-soluble-t-cell-receptor-and-a-method-for-obtaining-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:49:39 GMT -->
</html>
