<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/212758-a-process-for-producing-riboflavin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:20:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 212758:A PROCESS FOR PRODUCING RIBOFLAVIN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR PRODUCING RIBOFLAVIN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a process for producing riboflavin. This process includes providing a fennentation medium containing a recombinantly- produced microorganism that over-produces riboflavin and contains a mutation which causes auxotrophic growth of the microorganism wherein the auxotrophy within the microorganism does not compromise the ability of the microorganism to produce riboflavin. The medium is then supplied in excess with all substrates required for production of riboflavin, glucose a substrate complementing the auxotrophy from a group consisting of biotin, aden ine tryptophan and lysine. Riboflavin is subsequently isolated in a known manner. I</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention relates to a process for producing a target fermentation product.   More particularly,'the present invention relates to a process for over-producing a target fermentation product in a microorganism having a mutation which causes auxotrophic growth of the microorganism, but that does not compromise its ability to over-produce the target fermentation product.   Host cells and polynucleotide sequences used in the process are also provided.<br>
Many commercially valuable products are produced by fermentation reactions.   For example, riboflavin, which is an essential vitamin that is required by all bacteria, animals, and plants, is synthesized by plants and bacteria, however, it cannot be produced by higher animals, which must acquire it from their diet.<br>
Riboflavin is produced commercially for use as a food and feed additive by, for example, fermentation reactions using Ashbya gossypii<br>
Eremothecium ashbyiiy or Candida flareri cells.   {See e.g., Ainsworth, G.C. and Sussman, A.S., The Fungi, Academic Press, New York (1965); Heefner, D.L., et al., WO 88/09822; Hickey, R.J., Production of Riboflavin by Fermentation, in Industrial Fermentation (Underkofler, L.A. and Hickey, R.J., eds.) pp. 157-190, Chemical Publishing Co., New York (1954); and Perlman, D. et al.y Fermentation Ind. Eng. Chem. 44:1996-2001 (1952).<br>
The enzymes required to catalyze the biosynthesis of riboflavin from guanosine triphosphate (GTP) and ribulose-5-phosphate are encoded by four genes {ribG, ribB, ribA, and ribH) in 5. subtilis.   See, FIG. lA.   These genes are located in an operon, the gene order of which differs from the order of the enzymatic reactions catalyzed by the enzymes.   For example, GTP cyclohydrolase II, which catalyzes the first step in riboflavin biosynthesis is encoded by the third gene in the operon, ribA.   See, FIG. 2.   The ribA protein also encodes a second<br><br>
enzymatic activity, i.e., DHB synthase, which catalyzes the conversion of ribulose-5-phosphate to the four-carbon unit DHB.   Deaminase and reductase are encoded by the first gene of the operon, ribG,  The bi-functionality of the ribA and ribG gene products may facilitate a coordinated riboflavin precursor flux.   The penultimate step in riboflavin biosynthesis is catalyzed by lumazine (Lum) synthase, the product of the last rib gene, ribH.   Riboflavin synthase, which controls the last step of the pathway, is encoded by the second gene of the operon, ribB.   The function of the gene X (FIG. 1) located at the 3' end of the rib operon is, at present, unclear, however, its gene product is not required for riboflavin synthesis.<br>
Transcription of the riboflavin operon from the ribP\ promoter is controlled by an attenuation mechanism involving a regulatory leader region located between ribP\ and ribG. RibO mutations within this leader region result in deregulated expression of the riboflavin operon.   Deregulated expression is also observed in strains containing missense mutations in the ribC gene.   The ribC gene has recently been shown to encode the flavin kinase/FAD synthase of B. subtilis.   See, Mack, M, et al.y J. Bact.y   180:950-55(1998).   Deregulating mutations reduce the flavokinase activity of the ribC gene product resulting in reduced intracellular concentrations of flavinmononucleotide (FMN), the effector molecule of the riboflavin regulatory system.<br>
Recently, a Bacillus subtilis microorganism was genetically engineered to produce high yields of riboflavin during a short fermentation cycle.   See, Perkins, J.B., U.S. Patent No. 5,837,528 ("Perkins "528"), which is hereby incorporated by reference as if recited in full herein.   This approach combined classical genetic mutant selection and fermentation improvement with genetic engineering of the riboflavin biosynthetic genes by deregulating and increasing the level of gene expression.   In this system, the expression of the rib genes was increased by mutating the flavokinase encoding ribC gene, by linking the rib genes to strong, constitutive promoters, and by increasing the copy number of the rib genes.<br>
For example, in the engineered rib operon present in the plasmid pRF69 disclosed by Perkins '528, the entire ribPi promoter and most<br><br>
of the regulatory leader region were deleted and replaced with a constitutive phage SPOl promoter, P15.   See, FIG. IB.   In addition, the phage promoter was introduced between the ribB and ribA genes to further increase the transcription of the corresponding downstream genes.   Finally, pRF69 was provided with a chloramphenicol resistance gene downstream of the rib genes.   pRF69 was targeted by single cross¬over transformation into the rib operon of the host microorganism RB50, which contained mutations deregulating purine biosynthesis and which contained a mutation in the ribC gene deregulating riboflavin biosynthesis.<br>
The genomic structure resulting from single cross-over transformation of RB50 with pRF69 includes a chloramphenicol resistance gene flanked by the wild-type rib operon at one end and by the engineered rib operon of pRF69 at the other end.   The iterative elements within this structure originate increased copy numbers of the resistance gene and of the flanking rib operon upon selection of the pRF69 transformed bacteria for increased chloramphenicol resistance.<br>
Enhanced transcription of the rib genes in RB50 containing multiple (n) copies of the modified rib operon of pRF69 (z.e., RB50::[pRF69]n) has been confirmed by Northern blot analysis. Unlike wild-type B. subtilisy which accumulated very small amounts of RNA transcript that covered the entire rib operon, RB50::[pRF69]n accumulated large amounts of shorter transcripts that covered primarily the first two genes of the operon.   The second P15 promoter engineered upstream of ribA gave rise to significant accumulation of RNA transcripts that covered the three downstream genes of the rib operon.   See, Perkins, J.B., et a/., /. Ind. Microbiol. BiotechnoLy 22:8-18 (1999).<br>
In a riboflavin fed batch fermentation reactor containing, for example, B. subtilis RB50::[pRF69]n&gt; biomass and riboflavin are produced from a common fermentation substrate, glucose.   The rate by which glucose is pumped into the reactor ("glucose feeding rate") is critical to its utilization in the production of biomass and riboflavin, respectively.   A fast glucose feeding rate allows the culture to grow at elevated rates causing an excess of biomass formation and a<br><br>
reduction of the riboflavin yield.   Glucose feeding rates that are too slow, however, while lowering biomass production, result in low riboflavin productivity.   Because low yield, low productivity, or both increase riboflavin production costs, a balance must be struck between biomass and riboflavin production by carefully regulating the glucose feeding rate in commercial riboflavin fermentation reactors.<br>
In view of the deficiencies noted above, it would be desirable to optimize production of a target fermentation product, such as riboflavin, while concurrently maintaining biomass production at a level that is most efficient for the size and type of reactor used.<br>
One embodiment of the present invention is a process for producing a target fermentation product.   This process includes providing a fermentation medium containing a recombinantly-produced microorganism that over-produces a target fermentation product and that contains a mutation which causes auxotrophic growth of the microorganism, wherein the auxotrophy within the microorganism does not compromise the ability of the microorganism to produce the target fermentation product.   The medium is then supplied with all substrates required for production of the fermentation product, a substrate for the target fermentation product, and a substrate complementing the auxotrophy.   The former substrate(s) is/are provided in excess, ensuring maximal productivity. The later substrate (i.e., substrate complementing the auxotrophy) is supplied in limited amounts to maintain biomass formation at a low rate.<br>
Another embodiment of the present invention is a process for decoupling production of a target fermentation product from biomass production in a fermentation medium.   This process includes providing a recombinantly produced microorganism that has been engineered to contain a polynucleotide sequence which encodes the biosynthetic enzymes for a target fermentation product.   In this process, the maximal production of the target fermentation product is dependent on an unlimited supply of a target fermentation product substrate for the microorganism.   Next, an auxotrophy is introduced into the microorganism to control biomass production by limiting the<br><br>
concentration of a substrate complementing the auxotrophy in the fermentation medium. A fermentation production microorganism made by the process set forth above is also provided.<br>
The present invention also includes as a further embodiment a polynucleotide, which is selected from SEQ ID NO: 1, an auxotrophy-introducing homolog or fragment of SEQ ID NO: 1, or a polynucleotide sequence containing an insertion, deletion, or substitution of SEQ ID NO: 1, which polynucleotide retains the ability to cause an auxotrophy in a host cell.<br>
Another embodiment of the present invention is a host cell transformed with a polynucleotide sequence including SEQ ID NO: 1, a homolog or a fragment of SEQ ID NO: 1 which retains its ability to cause an auxotrophy in the host cell, or a polynucleotide sequence containing an insertion, deletion, or substitution of SEQ ID NO: 1, which polynucleotide retains the ability to cause an auxotrophy in the host cell.<br>
The present invention also provides as another embodiment a riboflavin production microorganism RB50 containing multiple copies of the engineered rib operon pRF69 and transformed with the polynucleotide sequence SEQ ID NO: 1.<br>
The present invention includes a process for producing a target fermentation product.   In this process, a fermentation medium is provided containing a recombinantly produced microorganism that over-produces a target fermentation product.   The microorganism also contains a mutation that causes auxotrophic growth of the microorganism, wherein the auxotrophy does not compromise the ability of the microorganism to produce the fermentation product.<br>
As used herein, the phrase "recombinantly-produced microorganism" means any microorganism modified by recombinant DNA techniques to produce commercially useful target fermentation products, such as for example, riboflavin.   For example, a microorganism according to the present invention may include bacterial cells.   The microorganism may be selected from Escherichia,<br><br>
Bacillus, Cyanobacter, Streptomyces, and Corynebacteria cells. Preferably, the microorganism is selected from E. coli, B. subtilis, B. amyloliquefaciens, B. licheniformisy C, glutamicum, or B. ammoniagenes.<br>
In the present invention, the microorganism is modified using recombinant DNA techniques to increase production of the target fermentation product above wild-type production levels as set forth in more detail in the examples.   As used herein, "target fermentation product" means a compound produced by fermentation, such as for example riboflavin, pantothenic acid, biotin, thiamin, folic acid, pyridoxine, and amino acids.<br>
For example, when the target fermentation product is riboflavin, the recombinantly-produced microorganism is a B. subtilis cell, such as for example, the B. subtilis production microorganism designated as RB50::[pRF69]n containing multiple (n) copies (for example about 5 to about 20 copies) of pRF69 encoding a rib operon modified with the strong phage SPOl promoter (Pis) to enhance transcription of the rib genes.   This recombinantly-produced microorganism produces significantly more riboflavin than wild-type microorganisms.   See, Perkins '528.<br>
The Bacillus subtilis microorganism RB50 used in the present invention was deposited with the Agricultural Research Culture Collection (NRRL), Peoria, IL, under the terms of the Budapest Treaty on May 23, 1989, and was assigned accession number B 18502. Plasmid pRF69 used in the present invention was deposited with the American Type Culture Collection (ATCC), Rockville, MD, on June 6, 1990, and was assigned accession number ATCC 68338.<br>
In the present invention, the recombinantly-produced microorganism contains a mutation that causes auxotrophic growth. As used herein the term "mutation" refers to an alteration in the genomic sequence of the microorganism, which may be introduced by any convenient means including, for example, chemical and UV mutagenesis, followed by screening or selection for a desired phenotype, construction of dysfunctional genes in vitro by<br><br>
recombinant techniques used to replace the intact counterparts of the genes in the genome of the microorganism, by single and double cross¬over recombinations, and other well known techniques.   See, Sambrook, et aL, Molecular Cloning, A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press (1989) and, Harwood and Cutting, Molecular Biology Methods For Bacillus, John Wiley and Sons (1990), pp. 27-74.<br>
The terms "auxotroph," "auxotrophic," and "auxotrophy" are used interchangeably herein and refer to a microorganism that has been modified, by e.g.   a mutation, to require the addition of an exogenous compound to grow, that prior to the mutation, the microorganism could produce itself.   Thus, "auxotrophic growth" refers to the ability of a microorganism that has been rendered auxotrophic for a particular substrate to grow in a defined fermentation media.<br>
The exogenous compound required for auxotrophic growth is referred to herein as "a substrate complementing the auxotrophy" or "the complementing substrate."   Examples of a substrate complementing the auxotrophy in the present invention include amino acids, nucleotides, and vitamins.   In the present invention, a microorganism may be an auxotroph for biotin, tryptophan, lysine, and/or adenine.   The microorganism may also be engineered to contain more than one such auxotrophy.   The selection of a particular auxotrophy is not critical to the present invention, so long as the auxotrophy decouples production of a target fermentation product from biomass production and the auxotrophy does not compromise the ability of the microorganism to produce the target fermentation product.<br>
In certain microorganisms used in fermentation reactors, such as B. subtilisy various substrates are used as sources for carbon, nitrogen, and oxygen.   Such substrates are required to produce both the target fermentation product, as well as the biomass.   Auxotrophic microorganisms also require a supply of "complementing" substrates as set forth above.<br><br>
The phrase "maximal productivity" as used herein means the maximum amount of a target fermentation product that a microorganism is able to produce when all substrates required or beneficial for target fermentation product formation (e.g., sources for carbon, oxygen, nitrogen, etc.) are available in excess, at given chemo-physical parameters, such as pH and temperature.   Maximal productivity is measured by: gm product produced/gm biomass/hr.<br>
The phrase "maximal growth rate" as used herein means the highest growth rate that is achieved by a microorganism when provided with an excess of all substrates required for growth at given chemo-physical parameters, such as pH and temperature.  Thus, the maximal growth rate of a microorganism is the amount of relative increase of the microorganism's biomass per time.   Maximal growth rate and maximal productivity of a microorganism may be determined by those skilled in the art using, e.g., continuous culture fermentations.<br>
If one of the substrates required for production of the fermentation product is not provided in excess, this substrate will become the production rate limiting substrate and its supply will determine the rate by which the target fermentation product is produced (i.e., the productivity of the process).   Likewise, if one of the substrates required for growth of the biomass is not provided in excess, this substrate will become the growth limiting substrate and its supply will determine the growth rate.<br>
Biomass and target fermentation product production are said to be "coupled" if the limited supply of a substrate determines both the growth rate of the biomasss and productivity for the target fermentation product.   In a coupled process, the same substrate is the limiting substrate for growth and production.   For a riboflavin fermentation system, glucose is used by the microorganism as the major carbon source required for biomass and product formation. Glucose limitation will result in a "coupled" process.   An increase or decrease in the rate that glucose is supplied to the fermentor (and thus the microorganism) determines whether both riboflavin and biomass production are up- or downregulated, respectively.<br><br>
Biomass and target fermentation product production are said to be "decoupled" if the limited supply of one substrate (substrate 1) determines the growth rate of the microorganism, whereas the supply of another substrate (substrate 2) determines the productivity for the target fermentation product.   In a decoupled process, the unlimited supply of substrate 2 will result in maximal productivity of the microorganism.   Thus, in a decoupled process, such as for example, one using an auxotrophic microorganism of the present invention, glucose (substrate 2) may be supplied to a fermentation reactor at a non-limiting rate to achieve maximal productivity of the target fermentation product, whereas the substrate complementing the auxotrophy (substrate 1) is supplied to the fermentation reactor at a rate that prevents the biomass from increasing at its maximal growth rate and thus limits biomass production.<br>
The presence of an auxotrophy in a microorganism according to the present invention and the growth limiting supply of the corresponding substrate complementing the auxotrophy must not compromise the ability of the microorganism to produce the target fermentation product.   In the present invention, an auxotrophy "compromises" the microorganism's ability to produce the target fermentation product if the limited supply of the substrate complementing the auxotrophy results in less than 50% of maximal productivity for the target fermentation product.   The productivity of the target fermentation product is also said to be "compromised" by the auxotrophy if the maximal productivity of a production microorganism carrying that auxotrophy is less than 50% of the maximal productivity of an otherwise identical microorganism lacking the auxotrophy.<br>
In the present invention, the presence of an auxotrophy in a microorganism is confirmed by determining biomass production in a suitable fermentation medium in the presence or absence of the corresponding substrate complementing the auxotrophy.   See, Example 1.   As used herein, "biomass production" or "biomass growth" means the ability of a particular microorganism to grow and divide.   In the present invention, biomass production is determined by standard<br><br>
microbiology methods, such as for example, by weighting the total cell dry mass or by measuring the turbidity of a fermentation sample at a particular wavelength between, e.g., 550-660 nm.   See, Example 3. Alternatively, the ability of a microorganism to grow and divide, i.e., produce biomass, may be assessed by colony formation on an agar plate.<br>
Where applicable, the presence of a mutation in the genome of a microorganism leading to an auxotrophy may be confirmed by standard molecular biology techniques, such as for example Southern hybridization, PCR (as in Example 1), or DNA sequencing.   Such techniques are readily available to one skilled in the art.   See, for example, chpts. 8-14 of Sambrook, et al. Molecular Cloning, A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press (1989) and chpt. 15 of Ausubel et ah, Eds., Current Protocols in Molecular Biology, John Wiley and Sons, Inc. (1998).<br>
As set forth above, a microorganism according to the present invention preferably is a biotin auxotroph.   Biotin or vitamin Bg is required as a prosthetic group for a number of enzymes including pyruvate carboxylase and acetyl CoA carboxylase.   Biotin is bio-synthesized from pimelic acid involving a number of enzymatic steps. See, FIG. 3.   Genes for biotin synthesis are clustered in a single operon in B. subtilis.   See, FIG. 4.   A microorganism that is an auxotroph for biotin is unable to grow without supplementation with biotin, i.e., the substrate complementing the auxotroph.   In a similar manner, cells that are lysine, tryptophan, and adenine auxotrophs are likewise unable to grow without supplementation of the fermentation medium with the respective substrate complementing the auxotroph.<br>
In commercial fermentation processes, it is desirable to limit the growth rate of biomass to, e.g.,   reduce consumption of costly fermentation substrates and to keep the oxygen demand and the heat development of the metabolically active biomass within the limits of the fermentation reactor's oxygen transfer and cooling capacities.   In a "coupled" process, limitation of biomass production through limiting the supply of a target fermentation substrate will reduce the productivity of the microorganism for the target fermentation<br><br>
product.   In a "decoupled" process using an auxotrophic microorganism according to the present invention, biomass production may be limited by restricting the supply of the substrate complementing the auxotrophy, whereas the target fermentation product is produced at the microorganism's maximal productivity for that product because all substrates required for product formation are provided in excess.<br>
In the present invention, decoupling target fermentation product production from biomass production is accomplished by introducing an auxotrophy into a microorganism that has already been modifed to over-express a target fermentation product, e.g. riboflavin.  Thus, in the present invention, maximal production of the target fermentation product is achieved when the microorganism is synthesizing the target fermentation product at its maximal productivity.<br>
Accordingly, the auxotrophic microorganism of the present invention is cultured in a fermentation medium containing a substrate complementing the auxotroph at a concentration sufficient to maintain biomass at a defined growth rate.   All other substrates required by the microorganism are supplied to the fermentation medium at concentrations that do not limit the ability of the microorganism to produce the target fermentation product at its maximal productivity.<br>
The particular concentrations of the substrate complementing the auxotroph and of the other substrates required to achieve maximal productivity for the target fermentation product that are in or supplied to the fermentation medium will vary depending on the particular auxotrophy selected, the target fermentation product, the reactor used, the production microorganism, and other well known fermentation variables.   For example, in a riboflavin production culture containing a production microorganism auxotrophic for biotin, the amount of glucose {i.e., the target fermentation substrate) added to the fermentation medium must be sufficient to over-produce riboflavin (i.e., the target fermentation product).   As used herein, the term "over-produce" means that the microorganism produces the target fermentation product from a substrate that is used as a carbon<br><br>
source above at least 0.1% (w/w) yield, preferably above 1% (w/w) yield, such as for example, above 4% (w/w) yield.<br>
In the present invention, the ratio of the concentration of the substrate for the target fermentation product to the substrate complementing the auxotrophy in the fermentation medium is from about 1:10,000,000 to about 1:10, preferably from about 1:1,000,000 to about 1:100.<br>
In the present invention, the target fermentation product may be isolated from the microorganism and/or the medium.   As used herein, the term "isolated" means that the target fermentation product is purified, or at least partially purified.   The target fermentation product may be purified by methods known in the art.   Such methods include for example, filtration, centrifugation, and/or extraction.   The target fermentation product may be further purified by re-crystallization from aqueous or organic solvents or applying other methods known in the art, such as for example, ion-exchange, size-exclusion, or hydrophobic interaction chromatography. For a detailed description of the procedures for riboflavin isolation and purification from a fermentation broth, see Kupfer E., European Patent Application EP 0 730 034 Al.<br>
In a preferred embodiment, a recombinantly produced microorganism that over-produces riboflavin is produced.   This microorganism is further modified to contain a biotin auxotrophy that decouples riboflavin production from biomass production.   Thus, the substrate complementing the auxotroph is biotin and the substrate for the fermentation product is glucose.   For example, a microorganism according to the present invention is the B. subtilis riboflavin production microorganism RB50 containing multiple copies of the engineered rib operon pRF69.   See, Perkins '528.   In this embodiment, a biotin auxotrophy is introduced into RB50 cells containing the engineered rib operon pRF69, which decouples riboflavin production from biomass production.   This biotin auxotrophic riboflavin production microorganism is designated as RB50::[pRF69] Bio".   When cultivated with biotin as the growth limiting substrate, the specific<br><br>
riboflavin productivity of this strain is enhanced about three-fold compared to a glucose limited culture.   See,  Example 3.<br>
The present invention also includes derivatives of RB50::[pRF69] Bio".   As used herein, a "derivative" of RB50;:[pRF69] Bio" is any B. subtilis cell which contains the engineered rib operon of pRF69 or a polynucleotide sequence that is at least 25% identical to the engineered rib operon of pRF69, preferably at least 50% identical to the engineered rib operon of pRF69, and that is a biotin auxotroph, such as for example, other Bacillus microorganisms with recombinantly engineered rib operons in which riboflavin production is decoupled from biomass production.   In the present invention, the percent identity of the polynucleotide sequence are determined using the BLAST program and the server at the National Center of Biotechnology Information (Bethesda, MD, USA).<br>
In the present invention, the microorganism may contain first a polynucleotide sequence coding for one or more polypeptides with enzymatic activities for producing riboflavin and one or more transcription elements which are not naturally associated with, but which are now transcriptionally linked to this polynucleotide sequence.<br>
As used herein, "transcription element" includes enhancers, promoters, natural or synthetic ribosomal binding sites, and/or terminators as are known to those of skill in the art.   See, Perkins '528..   In the present invention, the polynucleotide sequence may contain more than one transcription element as set forth above. Preferably, the polynucleotide sequence includes at least one promoter.<br>
The present invention also includes a process for decoupling production of a target fermentation product from biomass production in a fermentation medium.   This process includes providing a recombinantly produced microorganism that has been engineered to contain a polynucleotide sequence which encodes the biosynthetic enzymes for a target fermentation product, the maximal production of which is dependent upon an unlimited supply of a substrate for the<br><br>
target fermentation product.   An auxotrophy is then introduced within the microorganism to control biomass production by limiting the concentration of a substrate complementing the auxotrophy in the fermentation medium.<br>
For purposes of the present invention, the auxotrophy is "introduced" into the microorganism using any convenient means. For example, the auxotrophy may be introduced into the microorganism using classical mutagenesis techniques, as well as, recombinant biological techniques.   Preferably, a polynucleotide sequence that encodes a defective gene or a set of defective genes, whose intact counterparts within the microorganism are required to produce an essential compound for biomass production, is introduced into the microorganism by transformation.<br>
The term "transformation" as used herein refers to the introduction of the polynucleotide sequence into a microorganism and the recombination of the polynucleotide sequence with the genomic DNA of the microorganism by a single or double cross-over mechanism thereby replacing the corresponding intact gene or set of genes.   See, Harwood and Cutting, Molecular Biology Methods For Bacillus, John Wiley and Sons (1990), pp. 27-74.   Introduction of the polynucleotide sequence into the microorganism may be achieved by any convenient means well-known to those skilled in the art, such as for example, transformation of linearized or circular polynucleotide sequences into natural competent recipient cells or protoplasts, generalized transduction, transfection, lipofection, electroporation, particle bombardment and the like.   See, Id,   and Sambrook et al., Molecular Cloning A Laboratory Manual (2nd Ed.) Cold Spring Harbor Laboratory Press (1989).<br>
The polynucleotide sequence of the present invention may contain deletion-insertion mutations within a bioFDB gene cassette of Bacillus subtilis as set forth in more detail in the examples.   Preferably, the polynucleotide sequence is SEQ ID NO: 1.   In the present invention, SEQ ID NO: 1 may be modified at its 3'- and 5' ends with extension sequences, each of which are several hundred base pairs in length, to increase the transformation efficiency of SEQ ID NO: 1.<br><br>
The extension sequences are random sequences, which should be less than 80% homologous to DNA sequences of the recipient cells to prevent recombination at undesired loci.   Such a polynucleotide sequence is then used to transform a microorganism capable of over¬producing a target fermentation product-<br>
Transformants positive for the deletion-insertion mutation, i.e., which are now auxotrophs, are selected using standard selection protocols.   See, Id,   For example, the polynucleotide sequence used to transform the microorganism may include various selection markers, including for example antibiotic resistance markers, color producing markers, etc.   Preferably, the marker is a neomycin resistance marker, and selection for the desired transformation includes identifying microorganisms capable of growing in fermentation media supplemented with neomycin, and which over-produce the target fermentation product, such as riboflavin, as set forth in more detail in the Example 1.<br>
In another embodiment of the present invention, a polynucleotide sequence is provided.   Bacillus subtilis or a closely related species, e.g., Bacillus amyloliquefaciens or Bacillus licheniformis are made auxotrophic for biotin upon transformation with this polynucleotide sequence.   Preferably, the polynucleotide sequence is SEQ ID NO: 1 or an auxotrophy-introducing homolog of SEQ ID NO: 1.   Preferably, a Bacillus host cell is transformed with SEQ ID NO: 1, or an auxotrophy-introducing homolog thereof, which retains its ability to cause an auxotrophy in a host cell.<br>
In the present invention, a polynucleotide is considered an "auxotrophy-introducing homolog" of SEQ ID NO: 1 if the polynucleotide contains sequences that are more than 70% identical to the partial bioF and bioB sequences present within SEQ ID NO: 1 as determined by BLAST.<br>
The present invention also includes host cells transformed with a polynucleotide sequence containing SEQ ID NO: 1 or a homolog of SEQ ID NO: 1, which causes an auxotrophy in the host cell, or a polynucleotide sequence containing one or more insertions, deletions,<br><br>
and/or substitutions of SEQ ID NO: 1, which sequence retains the ability to cause an auxotrophy in the host cell.<br>
The host cell may be any microorganism capable of producing a target fermentation product according to the present invention.   For example, the host cell may be selected from Escherichiuy BacilluSy Cyanobacter, Streptomyces, and Corynehacteria cells.   Preferably, the microorganism is selected from E. coliy B. subtilis, B. amyloliqtiefacienSy B. licheniformisy C. glutamicunij or B. ammoniagenes.   More preferably, the host cell is a B. subtilts cell, such as for example RB50 containing multiple copies of the engineered rib operon pRF69.<br>
Some of the most important results of the present invention are summarized in the Figures.<br>
Figure 1 shows the B. subtilis wild-type riboflavin operon and the engineered rib operon of pRF69 in accordance with the present invention.   (A) shows the location of the ribO regulatory site and the structural genes, ribG, ribB, ribAy and rzfcH, and gene X.   The upstream sigma A promoter ribP1, and the putative internal promoter ribPz are marked.   The rho-independent transcription terminator downstream of gene X and the transcription attenuator upstream of ribG are depicted as well.   (B) shows the structure of the engineered rib operon of pRF69 with the location of the DNA sequences containing the constitutive phage SPOl promoter P15.<br>
Figure 2 shows the riboflavin biosynthetic pathway of B. subtilis.<br>
Figure 3 shows the biotin biosynthesis pathway of Bacillus.<br>
Figure 4 shows the biotin {bio) biosynthetic operon of B. subtilis.<br>
The following examples are set forth to illustrate the processes and compositions of the present invention.   These examples are illustrative only and are not intended to limit the scope of the invention in any way.   For example, the present invention may be varied by carrying out a decoupled process in large scale industrial fermentors, varying the dilution rate from 0.3 1/h to 0.001 1/h,<br><br>
increasing the concentration of the components in the fermentation medium, increasing glucose concentration up to 400 g/1, and carrying out a decoupled process according to the present invention in a batch or fed batch fermentation reactor.   The media components for all of these variations, including biotin, would be determined and adjusted by one skilled in the art.<br>
Examples<br>
Example 1.   Construction of Bacillus subtilis auxotrophic mutants<br>
In the following examples, a biotin auxotrophy-introducing polynucleotide sequence was first constructed in E. coli. Transformation of a natural competent B. subtilis microorganism with the polynucleotide sequence resulted in a biotin auxotrophic B. subtilis mutant.   A PBSl phage lysate prepared from this mutant was then used to introduce the auxotrophy, via generalized transduction, into the production microorganism RB50 containing multiple copies of the engineered rib operon pRF69.   Standard recombinant DNA techniques were used for the construction of the polynucleotide sequence and the Bacillus subtilis mutants.   See, for example, Sambrook et al.. Molecular Cloning A Laboratory Manual (2nd Ed.) Cold Spring Harbor Laboratory Press (1989) and Harwood and Cutting, Molecular Biology Methods For Bacillus, John Wiley and Sons (1990).<br>
To construct a bioFDB deletion-insertion mutation, a 2938 bp DNA fragment containing the complete bioF, bioD, and bioB genes, was amplified by PCR using genomic DNA from B, subtilis microorganism 1012 (Saito et al., Mol.Gen.Genet.   170:117-122 (1979)) as a template and the following primers:<br>
BioF+1 (5'-GAGAGGATCCACGAGGTTACGAGC-3') (SEQ ID NO: 2)<br>
BioB-1 (5'-GCGACGAATTCGACATCATACCGATTGC-3') (SEQ ID NO: 3).<br><br>
The reaction conditions for the PCR reaction consisted of 25 cycles of denaturation at 95°C for 1 minute, annealing at 55°C for 1 minute, and extension at 72°C for 3 minutes.   The PCR product was purified using the Wizard PCR purification kit (Promega Corp.) and was doubly-digested with BamHl and EcoRl,   The digested PCR product was cloned into (1) a BamHI-EcoRI-digested pUC19, resulting in plasmid pNMR3 and into (2) a B5amHI-EcoRI-digested pBluescriptII SK+, resulting in plasmid pNMR4.<br>
The 1.2-kb neomycin-resistance cassette from plasmid pBESTSOl (Itaya et al., Nucleic Acid Res.   17:4410 (1989)) was amplified using the following primers in a PCR reaction consisting of 25 cycles of denaturation at 95°C for 1 minute, annealing at 55°C for 1 minute, and extension at 72°C for 3 minutes:<br>
pBESTBstBl+1 (5'-GCGCTTCGAAGCTTGGGCAGCAGGTCG-3') (SEQ ID NO: 4)<br>
pBESTBstBI-1 (5'-GCGCTTCGAATTCAAAATGGTATGCG-3') (SEQ ID NO: 5)<br>
Both pNMR3 and pNMR4 were digested with BstBl which removed 1019 bp, encompassing parts of the bioF and bioB genes and the entire bioD gene.   The amplified neomycin-resistant cassette was purified and digested with BstBly and was cloned into 55fBI-digested pNMR3 and pNMR4.<br>
The following plasmids were then created:   pNMR5, containing the neomycin-resistant cassette inserted into the bioFDB genes in the same orientation as bio transcription in pUC19; pNMR6 containing the neomycin-resistant cassette inserted into the bioFDB genes in the opposite orientation to bio transcription in pUC19; and pNMR7 containing the neomycin-resistant cassette inserted into bioFDB genes in the opposite orientation as bio transcription in pBluescriptll SK+. All three plasmids were linearized with Xbal and transformed into natural competent B. subtilis 1012 cells.   Transformants were selected on TBAB plates containing neomycin at a final concentration of 5 µg/ml.   Approximately 250 transformants were observed, from which<br><br>
30 were patched onto Spizen's Minimal Medium (SMM) in the presence or absence of 0.1 mg/ml biotin.   23 of 30 colonies were auxotrophic for biotin.   6 colonies were analyzed by PCR analysis of the fusion junctions, and 2 clones (designated B, suhtilis NMl and NM2, respectively) were kept for further use.<br>
B. suhtilis microorganism NM2 was used as a donor microorganism for preparation of PBSl phage lysate.   This lysate was used to transduce the riboflavin production microorganism RB50 provided with the modified riboflavin operon pRF69.   RB50 refers to the host microorganism of B. subtilisy which contains several mutations introduced to improve production of nucleotides and riboflavin.   pRF69 refers to a rih operon modified by the introduction of strong phage promoters which was introduced at the rih locus of pRF50.   The modified operon pRF69 was amplified to high copy numbers.   A detailed description of the microorganism RB50 and the modified rih operon pRF69 is presented in Perkins '528.   A number of neomycin-resistant colonies were obtained which were unable to grow on SMM in the absence of exogenous biotin.   Three of these clones were analyzed by PCR and Southern hybridization, and were shown to contain the bioFDB::neo mutation.   One of these clones designated NM9 was selected and renamed RB50::[pRF69] Bio".   Southern blot hybridization revealed the presence of pRF69.<br>
RB50::[pRF69] Bio' was cultivated in a rich, complex medium (VY medium, DSMZ Medium 577) supplemented with 10 |ig/ml chloramphenicol to an optical density OD 660 = 1.   One milliliter of this broth was transferred into 20 ml VY medium supplemented with 30 pg/ml chloramphenicol and after reaching an OD of 1, again 1 ml of culture was transferred into 20 ml VY medium supplemented with 60 µg/ml chloramphenicol-   The same passage was repeated using VY medium containing 80 µg/ml chloramphenicol.   After reaching an OD of 1, this culture was supplemented with 15% (vol/vol) glycerol and 1 ml aliquots at -80°C.   The stepwise increase in the antibiotic concentration was used to select for bacteria with increased copy number of the modified rib operon pRF69.   See Perkins '528.<br><br>
Example 2.   Continuous culture fermentations<br>
Decoupling of growth and production was achieved and resulted in the desired positive effect on the riboflavin productivity of RB50::[pRF69] Bio" as described in detail below using continuous chemostat cultures.  According to standard textbooks, see e,g. Neidhardt et al.y Physiology Of The Bacterial Cell, Sinauer Associates, Inc. (1990), the growth rate of the cells within a continuous fermentation culture, which has reached steady state conditions (chemostat), equals the dilution rate at which the fermentor is operated.   The concentration of the biomass within such a fermentor is correlated to the concentration of the rate limiting substrate.<br>
Fermentations were carried out in New Brunswick bio-reactors Model Bioflow 3000 (3 1 total volume) equipped with blade stirrers. Continuous chemostat cultivation was used with an inlet pump that controlled the flow rate and an overflow that controlled the liquid level in the reactor. The fermentation variables were set as follows:<br>
Liquid volume: 1200 ml<br>
Dilution rate: 0.15<br>
Temperature: 37°C<br>
pH: 6.75<br>
Aeration: 1 1/min compressed air<br>
Stirrer speed: 1000 rpm<br>
The dissolved oxygen concentration was at every stage of the cultivation above 20% of air saturation.<br>
The fermentation medium used in the batch phase and as feeding medium contained the following components at the given final concentrations:   0.25 g/1 Na-glutamate, 1.57 g/1 KH2PO4, 1.57 g/1 K2HPO4, 2.74 g/1 Na2HPO4 x 12 H2O, 4.00 g/1 NH4CI, 0.1 g/1 citric acid, 6.80 g/1 (NH4)2SO4, 22 g/1 glucose x H2O, 0.2 ml/1 antifoam<br><br>
(silicon based), 14.1 mg/1 FeS04 x 7 H2O;   10.5 mg/1 CaChl2 x 2 H2O, 9.4 mg/1 MnSO4 x 1 H2O, 2.7 mg/1 COCl2 x 6 H2O, 1.0 mg/1 (NH4)6HMo7O24 X 4 H2O, 0.67 mg/1 AlCl3 x 6 H2O, 0.50 mg/1 Cul2 x 2 H2O;   6.7 g/1 MgSO4 X 7 H2O, 2.68 mg/1 ZnSO4 x 7 H2O.<br>
Na-glutamate, KH2PO4, K2HPO4, Na2HP04 x 12 H2O, NH4CI, citric acid, and (NH4)2SO4 were dissolved in 85% of the end volume, the pH was adjusted to pH 4 by adding hydrochloric acid and the solution was autoclaved.   Glucose was dissolved in 10% of the end volume and autoclaved separately.   Antifoam was autoclaved separately as a concentrate.   The FeSO4 x 7 H2O solution was prepared freshly as a 500-fold concentrate for each batch of medium and sterilized by filtration.   The other salts were prepared as 500-fold concentrates and sterilized by filtration in the following groups: group 1: CaCl2 x 2 H2O;   group 2: MnSO4 x 1 H2O, C0CI2 x 6 H2O, (NH4)6HMo7024 X 4 H2O, AICI3 X 6 H2O, CuCl2 x 2 H2O;   group 3: MgSO4 X 7 H2O, ZnSO4 x 7 H2O.   The separately sterilized solutions were combined under sterile conditions and sterile water was added in order to reach the final volume.<br>
The fermentors were inoculated with 40 ml of a seed culture prepared as follows:   One aliquot of the frozen RB50:: [pRF69] Bio" bacterial suspension of Example 1 was thawed and transferred into 100 ml VY medium supplemented with 60 |ig/ml chloramphenicol.   The culture was incubated at 37oC until reaching OD = 1 (typically after 12 to 15 hours).<br>
The batch phase of the fermentation, i.e. the phase during which the glucose in the fermentation medium was used up by the growing bacteria, lasted for about 24 hours.   After glucose depletion was reached as indicated by a sharp rise in the dissolved oxygen value, the fermentations where switched to continuous mode (start of the inlet and outlet pumps) at a dilution rate of 0.15 per hour.   The fermentation media, that were administered to the fermentors, were the media described above (containing 20 g/1 glucose) complemented with either 10 |ig/l biotin (fermentation A of Example 3) or 3µg/l biotin (fermentation B of Example 3).   Fermentation samples were<br><br>
taken and analyzed after the cultures had reached the steady state, i.e. after the fermentor volume had been exchanged more than 5 times.<br>
Example 3.   Biomass and riboflavin production in coupled and decoupled processes<br>
20 fil of 40% NaOH solution was added to a 1 ml fermentation sample of Example 2 immediately after the collection from the fermentation reactor.   The sample was incubated for 20 seconds at room temperature to dissolve riboflavin crystals within the sample. An aliquot of this suspension was diluted and neutralized with 0.1 molar potassium phosphate buffer pH 7.0.   Biomass content in the suspension was measured by determination of the turbidity at 660 nm. The dilution of the sample was adjusted to achieve readings between 0.05 and 0.3 absorption units.<br>
As a confirmation, the biomass content in the suspension was determined by weighting the dry cell mass.   A 1 ml aliquot of the suspension obtained from above was transferred into pre-weighed Eppendorf vials and the bacteria were collected by centrifugation (14,000 rpm, 5 minutes).   The bacteria were washed once with 1 ml deionized water and dried in vacuo at 80°C until constancy of weight was achieved.   The dry cell mass was determined gravimetrically.<br>
The riboflavin concentration was determined by HPLC analysis from a cell free supernatant of the suspension obtained from above.   A Hewlett-Packard 1100 System equipped with a binary pump, a column thermostat and a diode array detector was used.   The sample was fractionated over a stainless-steel Supelcosil LC-8-DB column (150 x 4.6 mm, 3 |im particle size).   A gradient elution of solvent A (4 mmol/1 sulfuric acid solution in water) and solvent B (methanol) according to the following time profile was used:<br><br>
Time [min]	%A	%B<br>
0	94	6<br>
2	94	6<br>
15	50	50<br>
20	50	50<br>
The column temperature was set to 20oC, and the flow rate was 1.0 ml / minute.   The UV absorption was recorded at 280 nm and the riboflavin peak was detected at about 11 minutes (total run time 20 minutes).   The riboflavin concentrations were calculated by comparing the integrated peak of the sample to those of riboflavin standards (Sigma, St. Lous, MO, USA).<br>
The results of the fermentation runs described in Example 2 are summarized in TABLE 1 (values represent the means obtained from 3 samples taken between 45 hours and 71 hours after start of the continuous mode).<br><br><br>
In fermentation B (TABLE 1) with 3 µg/l biotin and 20 g/1 glucose in the fermentation medium 3.36 g/1 biomass were produced.   Upon increase of biotin to 10 |ig/l while keeping glucose at 20 g/1 biomass production increased to 5.87 g/1 biomass (fermentation A, TABLE 1). Further increase of the biotin supply did not result in higher biomass production.   Thus, in fermentation A glucose (the fermentation substrate) is the growth limiting substrate.   In fermentation B glucose is supplied at a non growth-limiting rate.   Rather, biotin (the complementing substrate) limits biomass growth.   Hence, fermentations A and B of TABLE 1 represent coupled and decoupled processes as defined herein, respectively.<br>
The results of this example show that in a decoupled process with biotin as the growth limiting substrate and glucose as the fermentation substrate (fermentation B), the productivity of the biomass is significantly increased (3-fold) over a coupled process (fermentation A).   In addition, the product yield, i.e. the amount of riboflavin<br><br>
produced on consumed glucose is 33% higher in the decoupled process compared to the coupled process.<br>
The invention being thus described, it will be obvious that the same may be varied in many ways.   Such variations are not to be regarded as a departure from the spirit and scope of the invention and all such modifications are intended to be included within the scope of the following claims.<br><br>
SEQUENCE LISTING<br> F. Hoffmann-La Roche AG<br> Process for producing a target fermentation product<br> Case 20606<br><br> USSN 09/633,927  2000-08-08<br> 5<br> Patentin Ver. 2.1<br> 1<br> 3156<br> DNA<br> Bacillus subtilis<br> 1<br>
ggatccacga ggttacgagc	cttgaagatt	gattcctggt	taaacgagcg	gttagacaga	60<br>
atgaaagaag ccggcgtaca	tcgtaacctg	cggtcaatgg	atggagcgcc	ggttccagag	120<br>
aggaatattg atggcgaaaa	tcaaacggtc	tggtcctcaa	acaattattt	agggctcgca	18 0<br>
agcgatagac gtttgatcga	tgcagcccaa	acagcattgc	agcaatttgg	gacaggaagc	240<br>
agcggttcac gtttaacgac	aggcaattcg	gtctggcatg	aaaagctaga	aaagaagatt	300<br>
gccagcttta aactgacaga	agcggccctg	ctgttttcga	gcggttactt	ggccaatgtc	360<br>
ggtgtccttt catccttgcc	agaaaaggaa	gatgtcattt	taagtgacca	gctcaatcat	420<br>
gcaagtatga tcgacggctg	ccgactttct	aaggctgata	cagttgttta	tcggcatatt	480<br>
gatatgaatg atcttgaaaa	caagctgaat	gaaacacagc	gttatcagcg	ccgttttatc	54 0<br>
gtaacagacg gagtattcag	catggatggc	acaatcgccc	ctcttgatca	gatcatctca	600<br>
cttgcgaaac gctatcatgc	cttcgtggtc	gttgatgatg	cccacgcaac	aggagttttg	660<br>
ggcgattcgg gacaaggaac	gagtgaatac	tttggtgttt	gtcccgacat	tgttatcggc	720<br>
accttaagca aagctgttgg	cgcggaagga	ggttttgcgg	caggatcagc	ggtcttcatc	780<br>
gactttttgc tgaaccatgc	cagaacattt	atctttcaaa	ccgctattcc	gccagccagc	840<br>
tgtgcggctg ctcacgaggc	tttcaacatc	attgaagcca	gcagggaaaa	acgacagctt	900<br>
ttattttctt atatcagcat	gatcagaacc	agtctgaaga	atatgggtta	tgtggtgaaa	960<br>
ggagatcaca caccgattat	tcctgtagtc	attggcgatg	cccataaaac	ggtcctattt	102C<br>
gctgaaaaac tgcagggcaa	gggaatttat	gctcctgcca	ttcggccgcc	aaccgttgcg	1080<br>
ccgggtgaaa gccggattcg	aagcttgggc	agcaggtcga	gatcagggaa	tgagtttata	114C<br>
aaataaaaaa agcacctgaa	aaggtgtctt	tttttgatgg	ttttgaactt	gttctttctt	1200<br>
atcttgatac atatagaaat	aacgtcattt	ttatttttat	tttagttgct	gaaaggtgcg	1260<br>
ttgaagtgtt ggtatgtatg	tgttttaaag	tattgaaaac	ccttaaaatt	ggttgcacag	132C<br>
aaaaacccca tctgttaaag	ttataagtga	ctaaacaaat	aactaaatag	atgggggttt	1380<br>
cttttaatat tatgtgtcct	aatagtagca	tttattcaga	tgaaaaatca	agggttttag	1440<br>
tggacaagac aaaaagtgga	aaagtgagac	catgtgctta	ggaagacgag	ttattaatag	1500<br>
ctgaataaga acggtgctct	ccaaatattc	ttatttagaa	aagcaaatct	aaaattatct	156C<br>
gaaaagggaa tgagaatagt	gaatggacca	ataataatga	ctagagaaga	aagaatgaag	1620<br>
attgttcatg aaattaagga	acgaatattg	gataaatatg	gggatgatgt	taaggctatt	1680<br>
ggtgtttatg gctctcttgg	tcgtcagact	gatgggccct	attcggatat	tgagatgatg	1740<br>
tgtgtcatgt caacagagga	agcagagttc	agccatgaat	ggacaaccgg	tgagtggaag	1800<br>
gtggaagtga attttgatag	cgaagagatt	ctactagatt	atgcatctca	ggtggaatca	18 60<br>
gattggccgc ttacacatgg	tcaatttttc	tctattttgc	cgatttatga	ttcaggtgga	192C<br>
tacttagaga aagtgtatca	aactgctaaa	tcggtagaag	cccaaacgtt	ccacgatgcg	1980<br>
atttgtgccc ttatcgtaga	agagctgttt	gaatatgcag	gcaaatggcg	taatattcgt	204C<br>
gtgcaaggac cgacaacatt	tctaccatcc	ttgactgtac	aggtagcaat	ggcaggtgcc	2100<br>
atgttgattg gtctgcatca	tcgcatctgt	tatacgacga	gcgcttcggt	cttaactgaa	2160<br>
gcagttaagc aatcagatct	tccttcaggt	tatgaccatc	tgtgccagtt	cgtaatgtct	222C<br><br>
ggtcaacttt ccgactctga gaaacttctg gaatcgctag agaatttctg gaatgggatt 2280<br>
caggagtgga cagaacgaca cggatatata gtggatgtgt caaaacgcat accattttga 2340<br>
attcgaaagc gccgattgag tcttaccgga tggtgaataa ggaaacgctg cttgaaggcg 2400<br>
cgaagcgggc gcacgatctg aatatcggca catattgtat cgtggcaagc ggcagaggtc 2460<br>
cgtctaacag agaagtggat caggtcgtag atgcggttca ggaaattaaa gagacgtatg 2520<br>
gactgaagat ttgtgcatgt cttggactgt tgaagccaga gcaggcgaag cggctcaaag 2580<br>
atgcaggagt agaccgctat aatcataatt tgaatacgtc acagagaaac cattcaaaca 2640<br>
tcacaacctc acatacatac gatgacagag tcaatacggt tgaaatcgca aaagaatcgg 2700<br>
ggctgtctcc gtgttcaggc gccattatcg ggatgaagga gacgaaacag gatgtcattg 2760<br>
acatcgccaa aagcttgaag gctcttgacg cggattccat tcctgtgaat tttttgcatg 2820<br>
caattgatgg cacgccgtta gaaggcgtca acgaattaaa cccgctgtat tgtttaaaag 2880<br>
tgctggcgct gttccgtttt atcaatccat caaaagaaat tcgcatttcc ggaggaagag 2940<br>
aggtcaatct ccgcacattg cagccattag ggctttacgc cgcaaactcc atttttgtcg 3000<br>
gagactactt aacaactgcc gggcaagagg agacggagga tcataaaatg ctgagtgatt 3060<br>
taggctttga agttgaatca gtcgaagaaa tgaaggctag tttaagtgcg aaaagctgaa 3120<br>
agaatcaata aaagcaatcg gtatgatgtc gaattc	3156<br> 2<br> 24<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer Sequence BioF+1<br><a00>   2<br>
gagaggatcc acgaggttac gage<br> 3<br> 28<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer Sequence BioB-1<br> 3<br>
gcgacgaatt cgacatcata ccgattgc<br> 4<br> 27<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer Sequence pBESTstBI+1<br> 4<br>
gcgcttcgaa gcttgggcag caggtcg<br> 5<br> 26<br> DNA<br> Artificial Sequence<br><br><br> Description of Artificial Sequence: Primer<br>
Sequence pBESTstBI-1<br> 5<br>
gcgcttcgaa ttcaaaatgg tatgcg<br><br><br><br><br><br>
Claims<br>
1.   A process for producing a target fermentation product comprising:<br>
(a)	providing a fermentation medium comprising a recombinantly-produced microorganism that over-produces a target fermentation product and comprises a mutation which causes auxotrophic growth of the microorganism, wherein the auxotrophy within the microorganism does not compromise the ability of the microorganism to produce the target fermentation product; and<br>
(b)	supplying the medium with all substrates required for the production of the target fermentation product, a substrate for the target fermentation product, and a substrate complementing the auxotrophy.<br>
2. A process according to claim 1 further comprising isolating the target fermentation product from the microorganism and/or the fermentation medium.<br>
3.   A process according to claim 1 or 2 wherein the substrate complementing the auxotrophy is provided to the fermentation medium at a concentration sufficient to maintain biomass growth at a defined growth rate, and the substrates required for production of the fermentation product are provided to the fermentation medium at concentrations that do not limit the ability of the microorganism to produce the fermentation product.<br>
4.   A process according to claim 3 wherein the ratio of the target fermentation substrate:substrate complementing the auxotrophy is from 1:10,000,000 to 1:10, especially from 1:1,000,000 to 1:100.<br><br>
5.   A process according to any one of claims 1 to 4 wherein the target fermentation product is selected from the group consisting of riboflavin, pantothenic acid, thiamin, folic acid, pyridoxine, and amino acids.<br>
6.   A process according to any one of claims 1 to 4 wherein the microorganism comprises one or more copies of a polynucleotide sequence coding for one or more polypeptides with enzymatic activities for producing riboflavin and one or more transcription element(s) which is/are not naturally associated with, but which is/are transcriptionally linked with the polynucleotide sequence in the microorganism.<br>
7.   A process according to claim 6 wherein the transcription element comprises at least one promoter.<br>
8.   A process according to any one of claims 1 to 7 wherein the auxotrophy in the microorganism is selected from the group consisting of biotin, adenine, tryptophan, lysine, and combinations thereof.<br>
9.   A process according to any one of the preceding claims wherein the substrate complementing the auxotrophy is biotin and one of the substrates for the fermentation product is glucose.<br>
10.   A process according to claim 9 wherein the microorganism is a biotin auxotroph and the target fermentation product is riboflavin.<br><br>
11.   A process according to any one of claims 1 to 10 wherein the microorganism is selected from the group consisting of Escherichia, Bacillus, Cyanohacter, Streptomyces, and Corynebacteria, especially selected from the group consisting of £. coli, B. subtilis B. amyloliqtiefacienr, B. licheniformisy C. glutamicum, and B. ammoniagenes.<br>
12.   A process according to any one of claims 1 to 10 wherein the microorganism is a B. subtilis bacterium, especially RB50::[pRF69] Bio" or derivative thereof which retains the ability to over-produce riboflavin.<br>
13.   A process for decoupling production of a target fermentation product from biomass production in a fermentation medium comprising:<br>
(a)	providing a recombinantly produced microorganism that has been engineered to comprise a polynucleotide sequence which encodes the biosynthetic enzymes for a target fermentation product, the maximal production of the target fermentation product being dependent on an unlimited supply of a target fermentation product substrate; and<br>
(b)	introducing an auxotrophy into the microorganism to control biomass production by limiting the concentration of a substrate complementing the auxotrophy in the fermentation medium.<br>
14.   A process according to claim 13 wherein step (b) comprises introducing a polynucleotide comprising a deletion-insertion mutation into the genome of the microorganism to disrupt the microorganism's ability to produce a compound required for biomass production.<br><br>
15.   A process according to claim 14 wherein the polynucleotide comprises deletion-insertion mutations within a bioFDB gene cassette.<br>
16.   A process according to any one of claims 13 to 15 comprising transforming the microorganism with a polynucleotide sequence comprising a bioFDB deletion-insertion mutation or SEQ ID NO: 1.<br>
17.   A process according to claim 13 to 16 wherein the target fermentation product is riboflavin.<br>
18.   A microorganism provided by the process of any one of claims 13 to 16.<br>
19.   A polynucleotide sequence which is selected from the group consisting of SEQ ID NO: 1 or an auxotrophy-introducing homolog of SEQ ID NO: 1, and polynucleotide sequences containing insertions, deletions, and substitutions of SEQ ID NO: 1, which retain the ability to cause an auxotrophy in a host cell.<br>
20.   A host cell transformed with a polynucleotide sequence comprising SEQ ID NO: 1 or a fragment thereof or a homolog of SEQ ID NO: 1 which retains its ability to cause an auxotrophy in the host cell, or a polynucleotide sequence containing insertions, deletions, and substitutions of SEQ ID NO: 1 which retains the ability to cause an auxotrophy in the host cell.<br>
21.   A host cell according to claim 20 wherein the host cell is a B subtilis cell.<br><br>
22.   A riboflavin production microorganism RB50 containing<br>
multiple copies of the engineered rib operon pRF69 transformed with<br>
the polynucleotide sequence of SEQ ID NO: 1.<br>
  1<br></a00></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWNsYWltcyBmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWNsYWltcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWNvcnJlc3BvbmRuZWNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWRlc2NyaXB0aW9uKGNvbXBsZXRlKWZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWRlc2NyaXB0aW9uKGNvbXBsZXRlKWdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">642-mas-2001-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gNC5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLWZvcm0gNi5wZGY=" target="_blank" style="word-wrap:break-word;">642-mas-2001-form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjQyLW1hcy0yMDAxLW90aGVyIGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">642-mas-2001-other document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="212757-a-powder-conditioning-system-and-a-powder-dispersion-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="212759-an-improved-process-for-the-preparation-of-high-purity-sumatriptan.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>212758</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>642/MAS/2001</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Feb-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Dec-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-Aug-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1,6411 TE HEERLEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HANS-PETER HOHANN</td>
											<td>OVERLOON 1,6411 TE HEERLEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 P 25/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/633,927</td>
									<td>2000-08-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/212758-a-process-for-producing-riboflavin by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:20:46 GMT -->
</html>
