<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/212814-a-variant-of-a-parent-fungal-cutinase-and-a-method-of-producing-a-cutinase-variant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:33:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 212814:A VARIANT OF A PARENT FUNGAL CUTINASE AND A METHOD OF PRODUCING A CUTINASE VARIANT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A VARIANT OF A PARENT FUNGAL CUTINASE AND A METHOD OF PRODUCING A CUTINASE VARIANT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A variant of a parent fungal cutinase, which variant: comprises substitution of one or more amino acid residues at a position which is located: within 12 A from the location of the N-terminal amino acid (as calculated from amino acid residues in a crystal structure), and/or within 20 positions from the N-terminal amino acid, and is more themostable than the parent cutinase.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Accordingly, the invention provides a variant of a parent fungal cuttings comprising substitution of one or more amino acid residues which is located:<br>
a)	within 17 A from the location of the N-terminal amino acid (as calculated from amino acid residues in a crystal structure), and/or<br>
b)	within 20 positions from the N-terminal amino acid.<br>
The invention also provides a DNA sequence encoding the variant, an expression vector comprising the DNA sequence, a transformed host cell harboring the DNA sequence or the expression vector, a method of producing the variant, processes using the variant and a detergent composition comprising the variant.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 gives the coordinates for the 3D structure of the cutinase of H. inso-lens.<br>
Fig. 2 is a computer model showing the three-dimensional structures of the cutinases from F. solani pisi (left) and R insolens (right). Different colors have been used to identify the N-terminal amino acid and zones of 12 A and 17 A diameter around this.<br>
Figs. 3-6 illustrate the hydrolysis of c3ET. Details are given in the Examples.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Fungal cutinase<br>
The parent cutinase is a fungal cutinase, such as a filamentous fungal cutinase, e.g. native to a strain of Hum cola or Fusarium, specifically H. insolens or F. solani posy, more specifically H. insolens strain DSM 1800.<br>
The amino acid sequence of the cutinase of H. insolens strain DSM 1800 i and the DNA sequence encoding it are shown as SEQ ID NO: 2 and SEQ ID NO: 1 of US 5,827,719* The numbering system used herein for the H, insolens cutinase is based on the mature peptide, as shown in said SEQ ID NO: 2,<br>
The amino acid sequence of the cutinase of F. solani pisi is shown as the mature peptide in Fig, ID of WO 94/14964, The numbering system used herein for<br><br>
the F. solani pisi cutinase is that used in WO 94/14964; it includes the pro-sequence shown in said Fig. 1D; thus, the mature cutinase is at positions 16-214, The parent cutinase may have an amino acid sequence which is at least 50 % (particularly at least 70 % or at least 80 %) homologous to the cutinase of K inso-lens strain DSM 1800. The parent cutinase may particularly be one that can be aligned with the cutinase of H. insolens strain DSM 1800.<br>
Nomenclature for amino acids and alterations<br>
The specification and claims refer to amin9 acids by their one-letter codes. A particular amino acid in a sequence is identified by its one-letter code and its position, e.g. Q1 indicates Gin (glutamine at position 1, i.e. at the N-terminal.<br>
The nomenclature used herein for defining substitutions is basically as described in WO 92/05249. Thus, R51P indicates substitution of R (Arg) at position 51 with P (Pro).<br>
Homology and alignment<br>
For purposes of the present invention, the degree of homology may be suitably determined according to the method described in Needleman, S.B, and Wunsch, CD,, (1970), Journal of Molecular Biology, 48, 443-45, with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1. The determination may be done by means of a computer program known such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711).<br>
Two given sequences can be aligned according to the method described in Needleman (supra) using the same parameters. This may be done by means of the GAP program (supra).<br>
Three-dimensional structure of cutinase<br>
The structure of the cutinase of H. insolens was solved in accordance with the principle for X-ray crystallographic methods as given, for example, in X-Ray Tincture Determination, Stout, G.K. and Jensen, L.H., John Wiley &amp; Sons, Inc. NY, 1989. The structural coordinates for the solved crystal structure at 2.2 A resolution<br><br>
using the isomorphous replacement method are given in Fig. 1 in standard PDB format (Protein Data Bank, Brookhaven National Laboratory, Brookhaven, CT).<br>
The structure of the cutinase of F. solani pisi is described in Martinez et al. (1992) Nature 356, 615-618. The 3D structures of the cuirass of F. solani pisi and H. insolens are compared as a computer model in Fig. 2.<br>
It should be noted that the overall three-dimensional structures of fungal cutinases are very similar and have been shoved by X-ray crystallography to be highly homologous. The similarities between the cutinases from F. solani pisi and H. insolens are dearly apparent from the computer model in Fig. 2. Therefore, modifications of the type indicated for one fungal cutinase will also be functional for other fungal cutinases.<br>
Substitution near N-terminal<br>
The variant of the invention has one or more amino acid substitutions in the vicinity of the N-terminal. The substitution is within a distance of 17 A (e.g. within 12 A) and/or within 20 positions (e.g. within 15 positions) of the N-terminal. The distance from the N-terminal is to be calculated between the Ca atom of the amino acids, and is calculated from an amino acid in a crystal structure (i.e. visible in the X-ray structure).<br>
In the cutinase of H. insolens strain DSM 1800, the two N-terminal amino acids (Q1 and L2. i.e. Gin and Leu at positions 1 and 2) are not visible in the X-ray structure, so the distance is to be calculated from amino acid G3. Amino acids within 17 A include positions 3-12, 18, 20^60, 62^64, 82, 85-86, 100-108, 110-111, 130-132, 174, 176-182, 184-185, 188, and 192, Those within 12 A include positions 3-8, 22-27, 30-47, 53-59, 102, 177, and 180-181.<br>
In the cutinase of F. solani pisi, the N-terminal amino acid G17 is visible in the X-ray structure. Amino acids within 17 A include positions 17-26, 34-75, 77-79, 101, 115, 117-119, 147, 191-197, 199-200, and 203. Those within 12 A include positions 17-22, 38, 40, 45-58, 60, 65, and 70-72.<br>
The variants of the invention have improved thermostability compared to the parent enzyme. The thermostability may be determined from the denaturation tern-<br><br>
premature by DSC (differential scanning calorimetry), e,g. as described in an example, e.g. at pH 8.5 vita a scan rate of 90 K/hr The variants may have a denaturation temperature which is at least 5°C higher than the parent enzyme.<br>
The total number of substitutions in the above regions is typically 1-10, e,g. 1-5 substitutions in the above regions. In addition, the cutinase variant of the invention may optionally include other modifications of the parent enzyme, typically 10 or fewer, e.g. 5 or fewer alterations (substitutions, deletions or insertions) outside of the above regions. Thus, the total amino acid sequence of the variant typically 1-20, e.g. 1-10 alterations compared to the parent cutinase.<br>
Solvent accessible surface<br>
One or more of the substitutions may be made at an exposed amino acid residue, i.e. an amino acid residue having a solvent accessible surface. This can be calculated by the "dssp" program (version October 1988) described in W. Kabsch and C, Sander, Biopolymers, 22 (1983) pp. 2577-2637.<br>
In the cutinase of R insolens strain DSM 1800, the following amino acids lie within 17 A of G3 at the N-terminal and have a solvent accessible surface greater than 0; 3-12, 18. 26-33, 36-38, 40-45, 47-56, 59-60, 62-64, 82, 85^86, 104-105, 174, 176-179,181-182, 192.<br>
Specific substitutions<br>
The substitution near the N-terminal may specifically be one that increases the electrical charge, i.e, a substitution of a negatively charged amino acid with a neutral or positively charged amino acid or substitution of a neutral amino acid with a positively charged amino acid. Thus, a negative amino acid residue at a position corresponding to position E6, E10, E30, E47.D63, E82 and/or E179 in the cutinase of Humicola insolens strain DSM 1800 may be substituted by a neutral or positive amino acid, e.g. R, K, Y, H, Q or N. Some specific substitutions are those corresponding to E6Q/N, E10Q/N, E47K/R or E179Q/N. Also, a neutral amino acid residue at a position corresponding to N7, S11, N44 or N52 in the H. insolens cutinase may be substituted by a positive amino acid (R, K or H).<br><br>
Another example of a substitution near the N-termini is substitution with a Pro residue, e.g. a substitution corresponding to A14P or R51P in the cutinase of Humicola insolens strain DSM 1800.<br>
Specific variants<br>
The following are some examples of variants in the H. insolens cutinase. Corresponding variants may be made on the basis of other parent cutinases. R51P<br>
E6N/Q+L138I A14P+ E47K E179N/Q<br>
E6N/Q+ E47K+ R51P<br>
A14P+E47K+E179N/Q<br>
E47K+ E179N/Q<br>
E47K+ D63N	 <br>
E6N/Q+ E10N/Q+ A14P+ E47K+ R51P+ E179N/Q E6N/Q+ A14P-*- E47K+ R51P+ El79N/Q Q1P+ L2V+ S11C+ N15T+ F24Y+ L461+ E47K<br>
Use of cutinase variant<br>
The cutinase variant of the invention may be used, e.g., for the enzymatic hydrolysis of cyclic oligomers of poly(ethylene terephthalate), such as cyclic tri(ethylene terephthalate), abbreviated as c3ET.<br>
In particular, this may be used to remove such cyclic oligomers from polyester containing fabric-or yam by treating the fabric or yam with the cutinase variant, optionally followed by rinsing the fabric or yam with an aqueous solution having a pH in the range of from about pH 7 to about pH 11. The treatment of polyester is conveniently carried out above the glass transition temperature of c3ET (about 55°C) and below the glass transition temperature of polyester (about 70°C). Thus, the treatment may suitably be carried out at 50-80°C, e.g. at 60-75°C. The process may be carried out in analogy with WO 97/27237.   "<br><br>
The cutinase variant may be used to treat polyester-containing textile, e.g. PET (polymer of ethyleneglycoi and terephthalic acid), P3GT (polymer of 1,3-propanediol and terephthalic acid) or a polyester/cotton blend. The treatment may provide benefits to the polyester textile such as improved wear and comfort, increased water permeability, reduced antistatic behavior, improve handle and softness, changed reed-position characteristics and/or color clarification.<br>
The cutinase variant may be used to improve the functional finish of a PET-containing yarn or fabric by a treatment with the cutinase variant, followed by a treatment with a finishing agent such as a softener, an anti-crease resin, an antistatic agent, an anti-soiling agent or agents to impair wrinkle-free, permanent press ior fire resistance effects. The treatment with the cutinase variant may increase the number of functional groups in the surface, and this can be used to attach the functional finish. Examples of finishing agents are described in "SENSHOKU SIAGEKAKO BENRAN" published 1998-10-15 by Nihon Sentimental KK.<br>
The cutinase variant of the invention is also useful in detergents, where it may be incorporated to improve the removal of fatty soiling, as described in WO 94/03578 and WO 94/14964. The addition of the cutinase variant to laundry detergent may reduce malodor from cloth which is accumulated during several wash/wear-cycles.<br>
The cutinase variant may also be used for degradation and recycling of polyester such as polycaprolactone (PCL), poly-ethyleneglycol-terephthalate (PET), polylactic acid, polybutylenesuccinate, and poly(hydroxybutiric acid)-co-(hydroxyvaIeric acid), e.g. film and bottles, e.g. as described in JP-A 5-344897.<br>
The cutinase variant may also be used for other known applications of lipases and cutinases, for example, in the baking industry (e.g. as described in WO 94/04035 and EP 585988), in the papermaking industry (e.g. for pitch removal, see EP 374700), and in the leather, wool and related industries (e.g. for degreasing of animal hides, sheepskin or wool), and for other applications involving degreas-ing/defatting. It may be used in immobilized form in the fat and oil industry, as a catalyst in organic synthesis (e.g. desertification, transesterification or ester hydrolysis reactions).<br><br>
Dyeing polyester<br>
The invention provides a process for dyeing polyester fabric or yarn. In this process, the fabric or yam is first treated with a cutinase, e.g. 12-48 hours at 50-70°C or 65"70'C, pH 7-10, followed by dyeing with dye, e.g. a reactive dye, a disperse dye or a cationic dye. The reactive dye may be one that reacts with OH or COOH groups, e.g. having the tincture Chromophore-NHPh-S02CH2CH20S03Na. The dyeing may be conducted at 40-80^C, e.g. for 20-60 minutes.<br>
The cutinase may be a thermos table cutinase having a thermal denaturation temperature, Td, at pH 8.5 which is at least 5° higher than the parent cutinase, e.g. 7-10*" higher, e.g. a value of 65°C or higher. The measurement may be made by DSC as described in an Example of this specification.<br>
Surfactant<br>
In the treatment of fabric or yarn, a conventional wetting agent and/or a dispersing agent may be used to improve the contact with the enzyme. The wetting agent may be a nonionic surfactant, e.g. an ethoxyiated fatty alcohol. A very useful wetting agent is an ethoxyiated and propoxylated fatty acid ester such as Berol 087 (product of Akzo Nobel, Sweden).<br>
The dispersing agent may suitably be selected from nonionic, anionic, cationic, ampholytic or zwitterionic surfactants. More specifically, the dispersing agent may be selected from carboxymethylcellulose, hydroxypropylcellulose, alkyl aryl sulfonates, long-chain alcohol sulfates (primary and secondary alkyl sulfates), sulfonated olefins, sulfated monoglycerides, sulfated ethers, sulfosuccinates, sulfonated methyl ethers, alkane sulfonates, phosphate esters, alkyl isothionates, acyi-sarcosides, alkyltaurides, fluorosurfactants, fatty alcohol and alkylphenol condensates, fatty acid condensates, condensates of ethylene oxide with an amine, condensates of ethylene oxide with an amide, sucrose esters, sorbitan esters, alky-loamides, fatty amine oxides, ethoxyiated monoamines, ethoxyiated diamines, alcohol ethoxylate and mixtures thereof A very useful dispersing agent is an alcohol ethoxylate such as Berol 08 (product of Akzo Nobel, Sweden),<br><br>
Methods      for     preparing      cutinase    variants<br>
The cutinase variant of the invention can be prepared by methods known in the art, e.g. as described in WO 94/14963 or WO 94/14964 (Unilever), The follov^ing describes methods for the cloning of cutinase-encoding DNA sequences, followed by methods for generating mutations at specific sites within the cutinase-encoding sequence.<br>
Cloning a DNA sequence encoding a cutinase<br>
The DNA sequence encoding a parent cutinase may be isolated from any cell or microorganism producing the cutinase in question, using various methods well known in the art. First, a genomic DNA and/or cDNA library should be constructed using chromosomal DNA or messenger RNA from the organism that produces the cutinase to be studied. Then, if the amino acid sequence of the cutinase is known, labeled oligonucleotide probes may be synthesized and used to identify cutinase-encoding clones from a genomic library prepared from the organism in question. Alternatively, a labeled oligonucleotide probe containing sequences homologous to another known cutinase gene could be used as a probe to identify cutinase-encoding clones, using hybridization and washing conditions of lower stringency.<br>
Yet another method for identifying cutinase-encoding clones would involve inserting fragments of genomic DNA into an expression vector, such as a piasmid, transforming cutinase-negative bacteria with the resulting genomic DNA library, and then plating the transformed bacteria onto agar containing a substrate for cutinase {i.e. maltose), thereby allowing clones expressing the cutinase to be identified.<br>
Alternatively, the DNA sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamidite method described S.L. Beaucage and M.H. Caruthers, (1981), Tetrahedron Letters 22, p. 1859-1869, or the method described by Matthes et aK, (1984), EMBO J. 3, p. 801-805. In the phosphoroamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in appropriate vectors.<br>
Finally, the DNA sequence may be of mixed genomic and synthetic origin, mixed synthetic and-cDNA origin or mixed genomic and cDNA origin, prepared by<br><br>
ligating fragments of synthetic, genomic or cDNA origin (as appropriate, the fragments corresponding to various parts of the entire DNA sequence), in accordance with standard techniques. The DNA sequence may also be prepared by polymerase chain reaction (PCR) using specific primers, for instance as described in US 4,683,202 or R.IC Saiki et al., (1988), Science 239. 1988, pp. 487-491.<br>
Site-directed mutagenesis<br>
Once a cutinase-encoding DNA sequence has been isolated, and desirable sites for mutation identified, mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites. In a specific method, a single-stranded gap of DNA, the cutinase-encoding sequence, is created in a vector carrying the cutinase gene. Then the synthetic nucleotide, bearing the desired mutation, is annealed to a homologous portion of the single-stranded DNA. The remaining gap is then filled in with DNA polymerase I (Klenow fragment) and the construct is ligated using T4 ligase. A specific example of this method is described in Morinaga at al., (1984), Biotechnology 2, p. 646-639. US 4,760,025 discloses the introduction of oligonucleotides encoding multiple mutations by performing minor alterations of the cassette. However, an even greater variety of mutations can be introduced at any one time by the Morinaga method, because a multitude of oligonucleotides, of various lengths, can be introduced.<br>
Another method for introducing mutations into cutinase-encoding DNA sequences is described in Nelson and Long, (1989), Analytical Biochemistry 180, p. 147-151, It involves the 3-step generation of a PCR fragment containing the desired mutation introduced by using a chemically synthesized DNA strand as one of the primers in the PCR reactions. From the PCR-generated fragment, a DNA fragment carrying the mutation may be* isolated by cleavage with restriction endonucleases and reinserted into an expression plasmid.<br>
Expression of cutinase variants<br>
According to the invention, a DNA sequence encoding the variant produced by methods described above, or by any alternative methods known in the art, can be expressed, in enzyme form, using an expression vector which typically includes con-<br><br>
trol sequences encoding a promoter, operator, ribosome binding site, translation initiation signal, and, optionally, a repressor gene or various activator genes.<br>
Expression vector<br>
The recombinant expression vector carrying the DNA sequence encoding a cutinase variant of the invention may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which it is to be introduced. The vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated. Examples of suitable expression vectors include pMT838.<br>
Promoter<br>
In the vector, the DNA sequence should be operably connected to a suitable promoter sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the DNA sequence encoding a cutinase variant of the invention, especially in a bacterial host, are the promoter of the lac operon of E.coli, the Streptomyces coelicolor agarase gene dagA promoters, the promoters of the Bacillus licheniformis a-amylase gene {amyL), the promoters of the Bacillus stearothermophilus maltogenic amylase gene (amyM), the promoters of the Bacillus amylollquefaciens a-amylase {amyQ), the promoters of the Bacillus subtiiis xylA and xylB genes etc. For transcription in a fungal host, examples of useful promoters are those derived from the gene encoding A. oryzae TAKA amylase, the TPI (those phosphate isomerase) promoter from S. cere-visiae (Alberet al. (1982), J. Mol. Appl. Genet 1, p. 419-434, Rhizomucor miehei as-partic proteinase, A. niger neutral a-amylase, A. niger acid stable a-amylase, A. ni-ger glucoamylase, Rhizomucor miehei Wpase, A. oryzae alkaline protease, A. oryzae those phosphate isomerase or A. nidulans acetamidase.<br><br>
Expression vector<br>
The expression vector of the invention may also comprise a suitable transcription terminator and, in eukaryotes, polyadenylation sequences operably connected to the DNA sequence encoding the a-amylase variant of the invention. Termination and polyadenylation sequences may suitably be derived from the same sources as the promoter<br>
The vector may further comprise a DNA sequence enabling the vector to replicate in the host cell in question. Examples of such-sequences are the origins of replication of plasmids pUC19, pACYC177, pUB110, pE194, pAMB1 and plJ702.<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the dal genes from 6. subtilis or S, licheniformis, or one which confers antibiotic resistance such as ampicillin, kanamycin,'chloramphenicol or tetracyclin resistance. Furthermore, the vector may comprise Aspergillus selection markers such as amdS, argB, niaD and sC, a marker giving rise to hygromycin resistance, or the selection may be accomplished by co-transformation, e.g. as described in WO 91/17243.<br>
The procedures used to llgate the DNA construct of the invention encoding a cutinase variant, the promoter, terminator and other elements, respectively, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et aL, Molecular Cloning; A Laboratory Manual, 2nd Ed., Cold Spring Harbor, 1989).<br>
Host Cells<br>
The cell of the invention, either comprising a DNA construct or an expression vector of the invention as defined above, is advantageously used as a host cell in the recombinant production of a cutinase variant of the invention. The cell may be transformed with the DNA construct of the invention encoding the variant, conveni-ently by integrating the DNA construct (in one or more copies) in the host chromosome. This integration is generally considered to be an advantage as the DNA sequence is more likely to be stably maintained in the cell. Integration of the DNA constructs into the host .chromosome may be performed according to conventional<br><br>
methods, e.g. by homologous or heterologous recombination. Alternatively, the cell may be transformed with an expression vector as described above in connection with the different types of host cells. .<br>
The cell of the invention may be a cell of a higher organism such as a mammal or an insect, but is preferably a microbial cell, e.g. a bacterial or a fungal (including yeast) cell.<br>
Examples of suitable bacteria are Gram positive bacteria such as Bacillus subtilis, Bacillus licheniformis, Bacillus lentus, Bacillus brevis, Bacillus stearothermo-philus, Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus coagulans, Bacillus circulans, Bacillus lautus, Bacillus megaterium, Bacillus thuhngiensis, or Streptomy-ces lividans or Streptomyces murinus, or gramnegative bacteria such as E.coli. The transformation of the bacteria may, for instance, be effected by protoplast transformation or by using competent cells in a manner known perse.<br>
The yeast organism may favorably be selected from a species of Saccharo-myces or Schizosaccharomyces, e.g. Sacctiaromyces cerevisiae.<br>
The host cell may also be a filamentous fungus e.g. a strain belonging to a species of Aspergillus, most preferably Aspergillus oryzae or Aspergillus niger, or a strain of Fusarium, such as a strain of Fusarium oxysporium, Fusarium graminearum (in the perfect state named Gribberella zeae, previously Sphaeria zeae, synonym with Gibberella roseum and Gibberella roseum f. sp. cerealis), or Fusarium sul-phureum (In the prefect state named Gibberella puricaris, synonym with Fusarium trichothecioides, Fusarium bactridioides, Fusarium sambucium, Fusarium roseum, and Fusarium roseum van graminearum), Fusarium cerealis ("synonym with Fusarium crokkwellnse), or Fusarium venenatum.<br>
In a preferred embodiment of the invention the host cell is a protease deficient or protease minus strain.<br>
This may for instance be the protease deficient strain Aspergillus oryzae JaL 125 having the alkaline protease gene named "alp" deleted. This strain is described in WO 97/35956 (Novo Nordisk).<br>
Filamentous fungi cells may be transformed by a process involving protoplast formation and transformation of the protoplasts followed by regeneration of the cell wall in a mannerknown per se. The use of Aspergillus as a host micro-organism<br><br>
is described in EP 238 023 (Novo Nordisk    A/S), the contents of which are hereby incorporated by reference.<br>
Production of cutinase variant by cultivation of transformant<br>
The invention relates, inter alia, to a method of producing a cutinase variant of the invention, which method comprises cultivating a host cell under conditions conducive to the production of the variant and recovering the variant from the cells and/or culture medium.<br>
The medium used to cultivate the cells may be any conventional medium suitable for growing the host cell in question and obtaining expression of the cutinase variant of the invention. Suitable media are available from commercial suppliers or may be prepared according to published recipes (e.g. as described in catalogues of the American Type Culture Collection).<br>
The cutinase variant secreted from the host cells may conveniently be recovered from the culture medium by well-known procedures, including separating the cells from the medium by centrifugation or filtration, and precipitating proteinaceous components of the medium by means of a salt such as ammonium sulphate, followed by the use of chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
Expression of variant in plants<br>
The present invention also relates to a transgenic plant, plant part or plant<br>
cell which has been transformed with a DNA sequence encoding the variant of the<br>
invention so as to express and produce this enzyme in recoverable quantities. The<br>
enzyme may be recovered from the plant or plant part. Alternatively, the plant or<br>
^plant part containing the recombinant enzyme may be used as such.<br>
The transgenic plant can be dicotyledonous or monocotyledonous, for short a dicot or a monocot. Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa), forage grass such as festuca, iolium, temperate grass, such as Agrostis, and cereals, e,g. wheat, oats, rye, bariey, rice, sorghum and maize (com).<br><br>
Examples of dicot plants are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis thaliana.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, and tubers. In the present context, also specific plant tissues, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes and cytoplasm are considered to be a plant part. Furthermore, any plant cell, whatever the tissue origin, is considered to be a plant part.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing the variant of the invention may be constructed in accordance with methods known in the art. In short the plant or plant cell is constructed by incorporating one or more expression constructs encoding the enzyme of the invention into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell<br>
Conveniently, the expression construct is a DNA construct which comprises a gene encoding the enzyme of the invention in operable association with appropriate regulatory sequences required for expression of the gene in the plant or plant part of choice. Furthermore, the expression construct may comprise a selectable marker useful for identifying host cells into which the expression construct has been integrated and DNA sequences necessary for introduction of the constmct into the plant in question (the latter depends on the DNA introduction method to be used).<br>
The choice of regulatory sequences, such as promoter and terminator sequences and optionally signal or transit sequences is determined, eg on the basis of when, where and how the enzyme is desired to be expressed. For instance, the expression of the gene encoding the enzyme of the invention may be constitutive or inducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific tissue or plant part such as seeds or leaves. Regulatory sequences are eg described by Tague et al, Plant, Phys., 86, 506, 1988.<br>
For constitutive expression the 35S-CaMV promoter may be used (Franck et al,, 1980. Cell 21: 285-294). Organ-specific promoters may eg be a promoter from<br><br>
storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet. 24: 275-303), or from metabolic sink tissues such as meristems (Ito et al,, 1994, Plant Mol. Biol. 24: 863-878), a seed specific promoter such as the glutelin, prolamin, globulin or albumin promoter from rice (Wu et al., Plant and Cell Physiology Vol. 39, No. 8 pp. 885-889 (1998)), a V7c/a faba promoter from the legumin B4 and the unknown seed protein gene from Vicia faba described by Conrad U. et al, Journal of Plant Physiology Vol. 152, No. 6 pp. 708-711 (1998), a promotter from a seed oil body protein (Chen et al., Plant and cell physiology vol. 39, No. 9 pp. 935-941 (1998), the storage protein napA promoter from Brassica napus, or any other seed specific promoter known in the art, eg as described in WO 91/14772. Furthermore, the promoter may be a leaf specific promoter such as the rbcs promoter from rice or tomato (Kyozuka et al., Plant Physiology Vol. 102, No. 3 pp. 991-1000 (1993), the chlorella virus adenine methyltransferase gene promoter (Mitra, A. and Higgins, DW, Plant Molecular Biology Vol, 26, No. 1 pp. 85-93 (1994). or the aldP gene promoter from rice (Kagaya et al., Molecular and General Genetics Vol. 248, No. 6 pp. 668-674 (1995), or a wound inducible promoter such as the potato pin2 promoter (Xo et al. Plant Molecular Biology Vol. 22, No. 4 pp. 573-588 (1993),<br>
A promoter enhancer element may be used to achieve higher expression of the enzyme in the plant. For instance, the promoter enhancer element may be an in-tron which is placed between the promoter and the nucleotide sequence encoding the enzyme. For instance, Xu et al. op cit disclose the use of the first intron of the rice actin 1 gene to enhance expression.<br>
The selectable marker gene and any other parts of the expression construct may be chosen from those available in the art.<br>
The DNA construct is incorporated into the plant genome according to conventional techniques known in the art, including Agrofeacfer/iy/77-mediated transformation, virus-mediated transformation, micro injection, particle bombardment, biolistic transformation, and electroporation (Gasser et al, Science, 244, 1293; Potrykus, Bio/Techn. 8, 535, 1990; Shimamoto et al, Nature, 338, 274, 1989).<br>
Presently, Agrobacterium tumefaciens mediated gene transfer is the method of choice for generating transgenic dicots (for review Hooykas &amp; Schilperoort, 1992,<br><br>
Plant Mol. Biol. 19: 15-38), however it can also be used for transforming monocots, although other transformation methods are generally preferred for these plants. Presently, the method of choice for generating transgenic monocots is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calli or developing embryos (Christou, 1992. Plant J. 2: 275-281; Shimamoto, 1994. Curr. Opin. Biotechnol. 5: 158-162; Vasil et a!., 1992. Bio/Technology 10: 667-674). An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh S, et al., Plant Molecular biology Vol. 21, No. 3 pp. 415-428 (1993).   .<br>
Following transformation, the transformants having incorporated the expression construct are selected and regenerated into whole plants according to methods well-known in the art.<br>
MATERIALS AND METHODS<br>
Plasmids<br>
PJSO026<br>
This is a S. cerevisiae expression plasmid described in WO 97/07205 and in<br>
J.S.Okkels, (1996) "A URA3-promoter deletion in a pYES vector increases the expression level of a fungal lipase in Saccharomyces cerevisiae. Recombinant DNA Biotechnology III: The Integration of Biological and Engineering Sciences, Vol. 782 of the Annals of the New York Academy of Sciences).<br>
pFuku83<br>
This is a yeast and E. coli shuttle vector for expression of the H. insolens cu-<br>
tinase under the control of a TPI promoter, constructed from pJSO026.<br>
Substrate<br>
BETEB<br>
Terephthalic acid bis(2-hydroxyethyl)ester dibenzoate is herein abbreviated<br>
as BETEB (benzoyl-ethylene-terephthalic-ethelene-benzoate). It was prepared from<br>
terephthalic acid bis (2-hydroxyethyl) ester and benzoic acid.<br><br>
Lipase activity (LU)<br>
A substrate for lipase is prepared by emulsifying tributyrin (glycerin tribu-tyrate) using gum Arabic as emulsifier. The hydrolysis of tributyrin at 30 "C at pH 7 is followed in a pH-stat titration experiment. One unit of lipase activity (1 LU) equals the amount of enzyme capable of releasing 1 pmol butyric acid/min at the standard conditions.<br>
Differential scanning calorimetry (DSC)<br>
Sample and reference solutions are carefully degassed immediately prior to loading of samples into the calorimeter (reference: buffer without enzyme). Sample and reference solutions (approx. 0.5 ml) are thermally pre-equillibrated for 20 minutes at S'C. The DSC scan is performed from 5 C to 95 C at a scan rate of approx. 90 K/hr. Denaturation temperatures are determined at an accuracy of approx. +/-1 C. A VP-DSC from MicroCal Inc. is suitable for the experiments.<br>
Methods<br>
PCR conditions<br>
step 1: 94° C, 120 sec.<br>
step 2: 94° C, 60 sec<br>
step 3: 50° C, 60 sec<br>
step 4: 72° C, 150 sec.<br>
Go to step 2, 35 cycles<br>
step 5: 72° C, 480 sec.<br>
Step 6: 4° C, for ever<br>
EXAMPLES<br>
Example 1: Preparation of cutinase variants<br>
A DNA sequence encoding H. insolens cutinase was obtained as described in US 5,827,719 (Novo Nordisk) and was found to have the DNAsequence shown in SEQ ID NO: 1 therein.-<br><br>
Variants were prepared by localized random mutagenesis and selection of positive clones by incubation at 60^C for 1 day on BETEB plates. The BETEB plates contained 200 ml/I of 500 mM glycine buffer (pH 8.5), 1.25 g/l of BETEB (dissolved in hot ethanol) and 20 g/l of agar.<br>
Three positive variants were isolated, and their amino acid sequence was determined. They were found to have the following modifications, compared to the parent H. insolens cutinase:<br>
A14P + E47K<br>
E47K<br>
E179Q<br>
Example 2: Site directed mutation<br>
A variant of the H. insolens cutinase having the substitutions E6Q+ E47K+ R51P was prepared as follows:<br>
A pair of PCR primers were designed so as to introduce amino acid substitutions, making use of the existed' restriction enzyme sites nearby, as follows (an asterisk indicates an introduced mutation):<br>
Upper primer: ESQ F<br>
egg caq cto gga gcc ate c*ag aac PvuW<br>
Lower primer: E47K,R51P<br>
cgc cct gga tec aga tgt teg* gga tgt ggg act t*aa ggc BamH I<br>
PCR was run using these primers and pFukuNL83 as a template under the PCR condition described above.<br>
The obtained PCR fragment was purified by Clontech Spincolumn and digested with Pvu II and BamH I.<br>
The resultant fragment was gel-purified and iigated to pFukuNL83 which had been digested with the same restriction enzyme sites.<br><br>
Example 3: Thermostabiiity of cutinase     variants<br>
Variants<br>
The thermostability was tested as described below for the H. insolens cutinase and the following variants thereof:<br>
A14P+ E47K<br>
E47K<br>
E179Q<br>
E6Q+E47K+R51P<br>
A14P+E47K+E179Q<br>
E6Q+A14P+ E47K+ R51P+ E179Q<br>
E6Q+E10Q+A14P+E47K+R51P+E179Q<br>
Differential Scanning Calorimetn/ fPSC)<br>
Thermostability of cutinase variants was investigated by means of DSC at<br>
pH 4.5 (50 mM acetate buffer) and pH 8.5 (SOmM glycyl-glycine buffer). The themial denaturation temperature, Td, was taken as the top of denaturation peak (major en-dothermic peak) in thermograms (Cp vs. T) obtained after heating of enzyme solutions at a constant programmed heating rate.<br>
The parent cutinase was found to have Td of 63°C at pH 8.5. Six of the above variants were found to have Td of 70-73°C, i.e. an improvement of 7-10°.<br>
The parent cutinase was found to have Td of 61 °C at pH 4.5. Five of the above variants were found to have Td of 64-66°C, i.e. an improvement of 3-5°.<br>
Hydrolysis of BETEB<br>
The thermostability of the H, insolens cutinase and two of the above variants<br>
9<br>
was measured by hydrolysis of BETEB at elevated temperature. For each cutinase, the following mixture was incubated for 17 hours at various temperatures in the range 55-70'C:<br>
0.1 ml 0.5 M glycyl-glycine buffer (pH 8.5)<br>
0.1 ml 0.5 % BETEB dissolved in ethanoi<br>
0.1 ml enzyme solution (approx. 25 LU/ml)<br>
0.7 ml Milli Q water<br><br>
The   degree   of  hydrolysis   was    measured  after  the   incubation.  The results are shown in the table below.<br><br>
These results clearly show that the variants have improved thermostability . compared to the parent cutinase.<br>
Hydrolysis of BETEB<br>
The thermostability of the H. insolens cutinase and three of the above variants was measured by hydrolysis of BETEB at 60^*0 for 2 hours. The hydrolysis was carried out at the above conditions, except that the temperature was fixed at 60°C I and the cutinase dosage was varied. The results below are shown in the table below.<br><br>
The results show a much faster hydrolysis at 60°C with the variants than with the parent cutinase.<br><br>
Example 4: Hydrolysis of c3ET<br>
The H. insolens cutinase and five of the above variants were tested in hydrolysis of c3ET at elevated temperature. For each cutinase, the following mixture was incubated for 2 hours at various temperatures.<br>
0.115mg c3ET (0.1ml of 2mM c3ET dissolved in HFIP was taken in reaction vessel. Solvent was removed under vacuum, then dried up at 70°C over night)<br>
0.1ml      0.5M glycyl-glycine buffer (pH8.5)<br>
0.1ml      enzyme solution (approx. 600LU/ml)<br>
0.8ml      Milli Q water<br>
After the incubation, 2ml of 1,1,1,3,3,3-Hexafluoro-2-propanol (HFIP) was added to each reaction mixture, then hydrolysis ratio was measured by HPLC. The results shown in Fig 3 clearly indicate that the variants have improved thermostability compared to the parent cutinase.<br>
Example 5: Hydrolysis of c3ET on yam<br>
The thermostability of the H. insolens cutinase five of the above variants was tested using polyester yarn containing c3ET as by product. The following substrate mixture was preincubated at 60 or SS^C:<br>
0.1 g      polyester yarn<br>
0.2ml     0.5M glycyl-glycine buffer (pH8.5)<br>
1.7ml    Milli Q water<br>
After preincubation, 0.1ml enzyme solution (approx. 1000 LU/ml) was added to each reaction vessel and incubated for 17 hours. Then 2ml HFIP was added and left for 30 minutes to extract and hydrolyze c3ET sitting on the surface of the polyester yam; then the hydrolysis ratio was measured. The results are shown in Fig. 4.    '<br>
It is seen that the variants are more effective than the parent cutinase for hydrolyzing c3ET on polyester yarn. One variant gives higher hydrolysis ratio at 65°C than at 60°C.<br><br>
Example 6: Treatment of yarn with	cutinase variant<br>
Time courses of c3ET hydrolysis on polyester yarn at different temperature or dosage were examined. Time course at different temperatures is shown in Fig 5. It is seen that the optimum temperature is 65**C. At 70'='C there is still about half of the activity left. Time course with increased enzyme dosage is shown in Fig 6. The curves at dosage 275 and 550 LU/ml are seen to be the same, indicating that the hydrolysis ratio reached to plateau between dosage of 100 to 275 LU/ml. Presumably 200LU/ml is enough.<br>
Example 7: Dyeing polyester with reactive dye<br>
The following polyester fabrics were treated:<br>
woven fabric; ca. 2 x 2 cm, 34mg<br>
knitted fabric; ca. 1,5 x 1.5 cm, 50mg<br>
Each fabric was soaked in 0.9 ml, 50 mM GlyGly (glycyl-glycine) buffer (pH 8,5) and 0.1 ml solution of a variant of the H. insolens cutinase (1100 LU/ml), and incubated at 65 or 70^*0. After one day, another 0.1 ml enzyme solution was added, incubation was continued for two more days, the fabrics were then taken out and rinsed in water. A comparative experiment was made with the parent cutinase, and a blank was treated in the same manner without enzyme.<br>
The fabrics were stirred in a mixture of 9 g 120 g Na2S04 and 60 g Na2C03 in 3 liter deionized water at 60 ^C for 30 min, and then rinsed with running warm water. The reactive dye was Celmazol Brilliant Blue B (product of Mitsui Chemical Co., Japan), which has the structure Chromophore-NHPh-S02CH2CH20S03Na.<br>
In all four experiments, (woven and knitted, 65 and 70°C), the fabrics were uniformly dyed.<br>
Example 8: SoiubUization of polyester fragments from knitted textile<br>
A 1x1 cm sample of knitted polyester textile (PET, polymer of ethylenegiycol and terephthalic acid) was incubated for 1 hour in 1 ml of buffer at pH 10, eo^'C with 0,01 mg of a variant of H. insolens cutinase. The reaction mixture was separated, and the release of terephthalic acid was found by measuring OD at 250 nm (ex-<br><br>
pressed as OD25o/mg PET), comparative    experiments       were   made   without enzyme   or   with   the   parent   cutinase.    Results:<br><br>
The results show that the variant is effective in solubilizing polyester.<br>
In another experiment, the cutinase variant was tested for 2 hours at 65°C with and without the addition of a non-ionic surfactant (alcohol ethoxylate, product name Softanol 50), using various amounts of the variant from 0.5 to 200 LU/ml The results showed more solubilization in the presence of non-ionic surfactant<br>
Example 9: Hydrolysis of poiycaproiactone and polyester film<br>
About 0.1 g of poiycaproiactone or polyester film were put in tubes. They were soaked in 5ml of 50mM GlyGly buffer (pH 8.5) with or without a variant of H. insolens cutinase (450 LU), They were incubated at 70*^0 for 5 hours. After the reaction we observed a thin layer of hydrolysate on the surface of the tubes with enzyme, both with poiycaproiactone and with polyester film. On the other hand no change was observed in controls without enzyme. In the case of poiycaproiactone there was 10% of weight loss. We see no weight change of polyester<br>
Example 10: cPET hydrolysis<br>
The performance of a cutinase variant was compared with the parent enzyme (H. insolens cutinase). The trials were done as follows:<br>
An oligomer-stained swatch of (blacik) PET-fabric (app. 4cm x 13cm) is subjected to the enzyme-treatment at relatively low agitation in a so-called mini-tergitometer apparatus. The PET-fabric is mounted onto a cylindrical, perforated holder (radius ca.2 cm, height ca 6 cm), that rotates around its axis, and with the oligomer stained side of the PET fabric facing the exterior of the cylinder.<br>
The fabric is immersed in a 150ml glass-beaker containing 100ml of the treatment solution at-a given temperature (here BS^'C). After a given treatment time<br><br>
[here   90minutes)   the   PET   swatch   is    removed from the bath and rinsed in deionized water and air dried.<br>
After conditioning the swatches are visually ranked (with respect to oligomer stain removal) on the iside having the oligomer-staining. The rating being as follows:<br>
-2:        Sample significantly worse than blank (no enzyme)<br>
-1:        Sample slightly worse than blank (no enzyme)<br>
0:        Sample can not be distinguished from blank<br>
1:	Sample slightly improved vs blank<br>
2:	Sample significantly improved over blank<br>
Also, the swatches are read spectrofotometrically (apparatus: Hunteriab Re-flectometer) to quantify the color strength (K/S-value at 600nm).<br>
The table below summarizes the test-conditions for a trial comparing the performance the enzymes under similar conditions:<br>
Temperature:	65**C<br>
Buffer/pH:	50 mM glycine buffer, pH 10.3<br>
Treatment time (min)	90<br>
Dosage of Enzyme (LU/g)	30000<br><br>
From this set of experiments it thus appears that the parent enzyme provides no or only very limited effect at the given test conditions (probably because the temperature is too high for the enzyme to retain activity), while the cutinase variant provides a substantial removal of the oligomer staining from the PET-fabric.<br><br>
Example 11: cPET hydrolysis<br>
The pH and temperature profile of a variant of H. insolens cutinase was tested in a model disperse dyeing experiment The trials were performed as follows:<br>
An oligomer-stained swatch of (black) PET-fabric is subjected to the conditions of a typical disperse dyeing sequence in a Werner Mathis Labomat. In overview of the process, the swatch is added to a buffer solution, heated to 130*^0, cooled down to the treatment temperature. Enzyme or buffer is added and then held at the desired temperature for 30 minutes. The solution is cooled down to room temperature and turbidity in the wash liquor is measured. The reduction in turbidity is a direct measure of the cutinase activity, corresponding to hydrolyzed cPET oligomers.<br>
Detailed description of the experiment:<br>
A black PET (app. 4cm x 13cm) swatch is added 140 ml 100 mM Britton-<br>
Robinson buffer containing 0.2 g/l Lutensol AT11 (BASF) and loaded in the Labomat<br>
(32 rotation per minute).<br>
The Labomat is heated to 130X at a gradient of S^'C/minute, and held for 10 minutes.<br>
The beakers are cooled to run temperature (according to table below) at a gradient of g^'C/minute, and held for 1 minute.<br>
10 mL enzyme solution (100 LU/ml of the variant) or buffer solution. (0 LU/ml) at appropriate pH is injected to the beakers.<br>
The Labomat is re-heated to temperature at a gradient of 2*'C/minute, and held for 30 minutes.<br>
The swatches are removed, and the wash liquor is cooled down to room temperature.<br>
Turbidity of the wash liquors are measured.<br>
Evaluation: Turbidity is measured on Hach 18900 Ratio Turbidimeter (standardized with 1.8, 18, and 180 NTU Turbidity Standards). Enzyme performance is calculated relative to a blank as the difference between turbidity of blank liquor (no enzyme) and turbidity of enzyme treated liquor<br>
The relative performance (reduction in turbidity) of the cutinase variant is calculated, and the results are shown in the following table. When a negative num-<br><br><br>
The results show that the cutinase variant is active over a broad pH and i temperature range, with optimum oligomer removal in the current set up around pH 9 and yS^'C. Inactivation seems to occur at or above 85**C.<br>
Example 12: cPET hydrolysis<br>
The effect of treatment time was investigated for a variant of H. insolens cutinase in a model disperse dyeing experiment. The trials were performed as follows;<br>
)	An oligomer-stained swatch of (black) PET-fabric is subjected to the condi-<br>
tions of a typical disperse dyeing sequence in a Werner Mathis Labpmat. In overview of the process, the swatch is added to a buffer solution, heated to 130**C, cooled down to the treatment temperature. Enzyme or buffer (100 mM Britton-Robinson pH 9) is added, and then held at 75X for 0-40 minutes. The solution is<br>
i cooled down to room temperature and turbidity in the wash liquor is measured. The reduction in turbidity is a direct measure of the cutinase activity, corresponding to hydrolyzed cPET oligomers.<br>
Detailed description of the experiment:<br>
A black PET (app. 4cm x 13cm) swatch is added to 140 ml 100 mM Britton-<br>
) Robinson buffer containing 0.2 g/l Lutensol AT11 (BASF) and loaded in the Labomat<br>
(32 rotation per minute).<br>
The Labomat is heated to 130*'C at a gradient of 9**C/minute. and the<br>
temperature is held for 10 minutes.<br>
The beakers are cooled to 75°C at a gradient of 9*C/minute, and held for 1<br>
5 minute.<br><br>
10 mL enzyme solution (100 LU/ml of variant) or 100 mM Britton-Robinson buffer pH 9.0 (0 LU/ml) is injected into the    beakers.<br>
The Labomat is re-heated to 75°C at a gradient of 2*'C/minute, and held for the appropriate number of minutes (0-40 minutes, see table below).<br>
The swatches are removed, and the wash liquor is cooled down to room temperature.<br>
Turbidity of the wash liquors are measured.<br>
Evaluation; Turbidity is measured on Hach 18900 Ratio Turbidimeter (standardized with 1.8, 18, and 180 NTU Turbidity Standards), Enzyme performance is calculated relative to a blank at time equal to zero: Turbidity of blank liquor at time zero (no enzyme) subtracted turbidity of enzyme treated liquor (at a given time).<br>
The relative performance (reduction in turbidity) of the cutinase variant was calculated, and the results are shown in the following table.<br><br>
The results show that the effect of the enzyme is increased over time. At the current enzyme dose and oligomer concentration, it seems to level off above approx. 20 minutes.<br>
Example 13: Fiber modification<br>
The effect on wetting characteristics of a disperse dyed polyester fabric was investigated by treating the fabric with a variant of H. insolens cutinase prior to dye-<br><br>
ing. The experiment therefore consistea or    TWO     pnases,     the     actual     fiber modification and the disperse dyeing pro-    cedure.<br>
Phase 1 ■ Fiber Modification:<br>
Equipment:	Atlas Launder-0-meter LP2<br>
Fabric:	knit 100 % scoured polyester from Testfabrics<br>
pH:	50 mM potassium phosphate buffer, pH 7<br>
Abrasives:	5 big steel balls<br>
Beaker Vol.:	120 mL<br>
Treatment:	2 hours 65°C then ramped up to 90°C and held for 1 hour<br>
Swatch Prep:	Cut 3* 1.5 g swatch of fabric, 3 per beaker = 4.5 g<br>
Rinse: Rinse in deionized water.<br>
Phase 2 - Dyeing - disperse dye:<br>
Dve Solution:<br>
Add together with deionized water to make liquor ratio 1:20-<br>
0.4 % Dianix Red (DyStar) SE-CB (owf)<br>
pH to 4.5-5<br>
Dyeing Procedure:<br>
1.	One swatch per treatment from the fiber modification is used for the dyeing (1.5 g/swatch is used for the liquor ratio calculation).<br>
2.	Make dyebath according to the recipe above. Add the cold dye solution to the Labomat beakers and heat to 55°C at a gradient of 3.5°C/minute. Run for 5 minutes once temperature has been reached.<br>
3.	Add the fabric to the beaker.<br>
4.	Raise temperature to 130X at a gradient of 1.5°C/minute. Dye for 30 minutes.<br>
5.	Cool to 70°C at a gradient of 5°C/minute. Drop bath, but collect, and rinse fabric hot (60°C) for 10 minutes. Follow the hot rinse with a room temperature overflow rinse until all bleeding had stopped.<br>
6.	Let air dry overnight.<br><br>
Tests/Analysis:<br>
AATCC Test Method 61 - Colorfastness to washing<br>
Percent Dyebath Exhaustion - Spectrophotometer<br>
K/S and L* - Refiectometer<br>
AATCC TM-79 Drop Test<br>
Results:<br><br>
The results show that the treatment of polyester with the variant increases the wetting substantially. No adverse effects are noticed on the dyeability with the disperse dye in the current set-up.<br>
Example 14: Malodor reduction in textiles soiled with human sweat/sebum by<br>
use of a cutinase variant in laundry<br>
t.<br>
The performance of cutinase, with respect to malodor reduction, can be tested in a one-cycie washing trial carried out in a Terg-O-tometer<br>
Experimental conditions:<br>
Washing liquor: 1000 ml per beaker<br>
Swatches: 100 % polyester (interlock knitted, previously cleaned by Soxhlet extraction), 24 swatches (3,3 &gt;« 3.5 cm) per beaker<br>
Soil: Human male axillary sweat and sebum applied by wiping the armpits after exercise.<br>
Detergent: 5 g/L of a standard color detergent. No pH adjustment.<br>
Water hardness: 3.2 mM Ca^VMg^^ (in a ratio of 5:1)<br>
Wash Temperature: 30°C<br>
Wash time; 30 min<br><br>
Rinse:   15 minutes in running tap    water<br>
Evaluation:<br>
After wash the wet swatches are placed in closed, tinted 200 ml glasses. A<br>
trained sensory panel (9-11 judges) evaluates the odor by sniffing the headspace<br>
over the wet samples and evaluates the total odor intensity. The odor intensity is<br>
noted by placing a mark on an unstructured line scale measuring 15 cm, with word<br>
anchors at each end ('nothing' at the beginning of the scale and Very strong' at the<br>
end). All evaluations are performed twice. The swatches are evaluated on day 1, 2<br>
and 3 after wash (swatches are kept in the glasses at all times).<br><br><br><br><br><br>
CLAIMS<br>
1.	A variant of a parent fungai cutinase, which variant:<br>
a)	comprises substitution of one or more' amino acid residues at a position<br>
which is boated:<br>
i)       within 17 A from the location of the N-terminal amino acid (as calculated from amino acid residues in a crystal structure), and/or ii)      within 20 positions from the N-termini amino acid, and<br>
b)	is more thermo stable than the parent cutinase.<br>
2.	. The variant of the preceding claim which comprises substitution of one or<br>
more amino acid residues at a position which Is located:<br>
i)       within 12 A foam the location of the N4erminal amino acid (as calculated from amino add residues in a crystal structure), and/or ii)      within 15 positions from the N-terminal amino acid.<br>
3.	The variant of either preceding calm wherein the parent cutinase is native to a filamentous fungus, preferably a strain of Humicola. or Fusarium, preferably H. inso-hen$ or F. solani pisi, most preferably H. inions strain DSM 1800.<br>
4.	The variant of any preceding claim wherein the parent cutinase has an amino acid sequence which can be aligned with the cutinase of H. inions strain DSM 1800.<br>
5.	The variant of any preceding claim wherein the parent cutinase has an amino acid sequence which is at least 50 % homologous to the cutinase of H. insolens strain DSM 1600, preferably at least 70 % homologous, more preferably at least 80 % homologous.<br>
6.	The variant of any preceding cam which comprises substitution of one or more amino acids having a solvent accessible surface.<br><br>
7.	The variant of any preceding claim wherein one or more substitutions Is substitution of a negatively charged amino acid with a neutral or positively charged amino acacia or substitution of a neutral amino acid with a positively charged amino acid.<br>
8.	The variant of the preceding dim wherein one or more substitutions is at a position corresponding to position E6, E10, E30, E47, D63, E82 and/or El79 in the cutinase of Humicola jnsol0ns strain DSM 1800, preferably a substitution with R/KA'/H/Q/N, more preferably a substitution corresponding to E6N/Q, E10N/Q, E47K/R and/cry E179N/Q {H. insolens cutinase numbering).<br>
9.	The variant of any preceding claim wherein one or more substitutions is substitution with a Pro residue, preferably at apposition corresponding to position A14 and/or R51,<br>
10.	The variant of any preceding claim which has one, two, three, four, five or six of said substitutions.<br>
11.	The variant of any preceding claim which has substitutions corresponding to one of the following in the cutinase of Humicola insolens strain DSM 180C<br><br>
a)	R51P<br>
b)	E6N/Q + L138I<br>
c)	A14P + E47K<br>
d)	E47K<br>
e)	E179N/Q<br>
f)	E6N/Q + E47K + R51P<br>
g)	A14P + E47K + E179N/Q<br>
h)	E47K + E179N/Q<br>
i)	E47K + D53N<br>
j)	E6N/Q + A14P + E47K + R51P + E179N/Q<br>
k)	E6WQ + EION/Q + A14P + E47K + R51P + E179N/Q. or<br>
I)	Q1P + L2V + S11C + N15T •+ F24Y + L46I + E47K<br><br>
12- The variant of any preceding claim which has hydrolytic activity towards tere-phthalic acid esters, particularly towards cyclic tri(ethylene terephthalate) and/or Terephthalic acid bats{2-hydroxyethy()esterd[benzoate (BETEB).<br>
13.	The variant of any preceding claim which has a denaturatron temperature which Is at least 5" higher than the parent cutlass, preferably measured at pH 8.5<br>
14.	A DNA sequence encoding the variant of any preceding claim.<br>
15.	A vector comprising the DNA sequence of the preceding claim.<br>
16.	A transformed host cell harboring the DNA sequence of claim -14 or the vector of claim 15.<br>
17.	A method of producing the variant of any of claims 1-13 comprising<br><br>
a)	cultivating the cell of claim 16 so as to express and preferably secrete the variant, and<br>
b)	recovering the variant.<br>
18.	A method of constructing a cutinase variant, which method comprises:<br>
a)	selecting a parent fungal cutinase,<br>
b)	identifying one or more amino acid residues in the parent cutinase at positions which are:<br>
i)       within 17 A from the location of the N-terminal amino acid (as calculated from amino acid residues in a crystal stmcture), and/or li)      within 20 positions from the N-termlnal amino acid, and<br>
c)	making alterations each of which is an insertion, a deletion or a substitution of the amino acid residue,<br>
d)	optionally, making alterations each of which Is an insertion, a deletion or a substitution of an amino acid residue at one or more positions other than b).<br>
e)	preparing the variant resulting from steps b-d,<br>
f)	. testing the thermostability of the variant,<br>
g)	optionally repeating steps b-l and<br>
5677.204-WO. SLK PCT/DK 99/00678<br><br>
1,<br>
h)      selecting a variant having higher themiostabiiity than the parent cuti-<br>
nase.<br>
19.	A method of producing a cutinase variant, which method comprises:<br>
a)	selecting a parent fungal cutlnase,<br>
b)	identifying one or more amino acid residues in the parent cutlnase at positions which are:<br>
1)       within 17 A from the location of the N-terminal amino acid (as calculated from amino acid residues in a crystal structure), and/or ii)       within 20 positions from the N-terminal amino acid, and<br>
c)	making alterations each of which Is an Insertion, a deletion or a substitution of the amino acid residue,<br>
d)	optionally, making alterations each of which is an insertioa, a deletion or a substitution of an amino acid residue at one or more positions other than b),<br>
e)	preparing the variant resulting from steps b-d,<br>
f)	testing the thennostability of the variant,<br>
g)	optionaiJy repeating steps b-f,<br>
h)      selecting a variant having higher thermostability than the parent cutlnase, and i)       producing the variant to obtain the cutinase variant.<br>
20.	A process for enzymatic hydrolysis of a cyclic oligomer of poly(ethyiene<br>
terephthalate), which process comprises treating the cyclic oligomer with the fungal cu<br>
tinase variant of any of claims 1-13,<br>
21 The process of the preceding claim, In which the cyclic oligomer is cyclic tri{ethyfene terephthalate).<br>
22,	The process of daim 20 or 21 wherein the treatment Is done at SO-SCC, pref<br>
erably at 65-75°C.<br>
23.	The process of any of claims 20-22 wherein the cyclic oligomer Is present in and<br>
on the fibers of a polyester containing fabric or yarn.<br>
5877.204-WO, SLK PCT/DK 99/00678-<br><br>
24.	The process of any of claims 20-23 which fuhher comprises subsequently rinsing the fabric or yarn, preferably rinsing with an aqueous solution having a pH in the rarige of from about pH 7 to about pH 11.<br>
25.	A process for dyeing polyester fabric or yarn, comprising:<br><br>
a)	treating the fabric or yarn with the fungal cutinase variant of any of claims 1-13; and<br>
b)	dyeing the treated fabric with a reactive dye or a disperse dye.<br><br>
26.	A detergent composition comprising a surfactant and the variant of any of claims 1-13.<br>
27.	A prcwess for improving the functional finish of a PET-containlng yarn or fabric<br>
comprising<br>
a)	treating the yam or fabric with the variant of any of claims 1 -13, and<br>
b)	subsequently the yam or fabric with a finishing agent selected from the group consisting of softeners, anti-crease resins, anti-static agents, anti-soiling agents.<br>
28.	A variant of a parent fungal cutinase comprising substitution of one or more<br>
amino acid residues which is located:<br>
a)	within 17 A from the location of the N-terminal amino acid (as calculated from amino acid residues In a crystal staicture), and/or<br>
b)	within 20 positions from the N-terminal amino acid,<br>
with the proviso that it is not a variant of the cutinase of Fusadum solanipisi having one of the substitutions R17, T18. T19V, D21N, 124E, Y38F, R40, G41A. S42, T43, E44, T45, G46, N47R. G49, T50, L51, P53, S54, A56C, S57. N58R, S61, A62E. K65A. D66S, G67D, W69Y, I70C, G74, G75, R78, Y119, G192. P193, D194R, A195, R196, G197V, or A199C {Fusarium so/an/p/s/cutinase numbering).<br>
29.	A variant of a parent fungal cutinase comprising substitution of one or more<br>
amino acid residues which:<br>
a)	has a solvent accessible surface, and<br>
b)	is located:<br>
5677.204-WO, SLK PCTJDK 99/00678 -<br><br>
I)        within 17 A from the locatior*i of ttie N-terminal amino acid (as calculated frc&gt;m amino acid residues in a crystal structure), and/or ii)       within 20 positions from the N-terminal amino acid, with the proviso that it is not a variant of the cutinase of Fusarium solan] pisi having one of the substitutions T18, Y3SF. R40, G41A, S42, T43, E44. T45, N47R. G49, T50, L51. P53, S54, A56C, A62E or G192 {Fusarium solani pis! cutinase numbering),<br>
30.	A variant of a parent fungal cutinase comprising substitution of one or more<br>
amino acid residues which is located:<br>
a)	less than 12 A from the location of the N-terminal amino group (as calculated from amino acid residues in a crystal stmcture), and/or<br>
b)	within 15 positions from the N-terminal amino acid,<br>
with the proviso that the variant is not the cutinase of Fusan'um solani pisi having one of the substitutions R17, T18, T19V, D21N, Y38F. R40, T45, G46. N47R, G49, T60, L51, P63. S54, A56C, S57. N58R. K65A or I70C (Fusarium solani pisi cutinase numbering).<br>
31.	A variant of a parent fungal cutinase from Humicola insolens which comprises<br>
substitution of one or more amino acid residues located:<br>
a)	within 17 A from the location of the N-tennlnal amino acid (as calculated from amino acid residues in a crystal structure), and/or<br>
b)	within 20 positions from the N-terminal amino acid.<br>
32.	A process for enzymatic hydrolysis of a cyclic oligomer of polyi(ethylene<br>
terephthalate), which process comprises treating the cyclic oligomer with a variant of a<br>
parent fungal cutinase, which variant comprises substitution of one or more amino<br>
acid residues at a position which is located;<br>
i)       within 17 A from the location of the N-terminal amino acid (as calculated fnsm amino acid residues in a crystal structure), and/or ii)      within 20 positions from the N-temrilnal amino acid.<br>
33.	A process for dyeing polyester fabric or yam. comprising:<br>
&amp;677.204-WO, SLK PCTIDK 99/00678<br><br>
a)	treating the fabric or yarn with a cutinase having a thermal denaturation temperature of 65°C or higher at pH 8.5; and<br>
b)	dyeing the treated fabric with a reactive dye or a disperse dye,<br>
iS4. A method for detecting cutinase activity in a sannple, comprising incubating the sample wfth terephthalic acid bis(2-hydroxye1hyl)ester dibenzoate and detecting hydrolysis of said ester.<br><br>
35.	A variant of a parent fungal cutinase substantially as herein described with reference to the accompany drawings.<br>
36.	A method of producing the variant substantially as herein described wade reference to the accompanying drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1jbGFpbXMgZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1jbGFpbXMgZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1jb3JyZXNwb25kbmVjZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1kZXNjcmlwdGlvbihjb21wbGV0ZSlmaWxlZC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1kZXNjcmlwdGlvbihjb21wbGV0ZSlncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1vdGhlciBkb2N1bWVudHMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNzYyLWNoZS1wY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-762-che-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="212813-sheathed-element-glow-plug-and-a-method-for-producing-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="212815-n-substituted-2-cyanopyrrolidines.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>212814</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/762/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Feb-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Dec-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-May-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Krogshoejvej 36, DK-2880 Bagsværd</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ABO, Masanobu</td>
											<td>2-5-3, Kouyadai, Funabashi-shi, Chiba-ken 274</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FUKUYAMA, Shiro</td>
											<td>Ariyoshi-cho, 166-2, Midori-ku, Chiba-shi, Chiba 211-0012</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SVENDSEN, Allan</td>
											<td>Bakkeledet 28, DK-3460 Birkerød</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MATSUI, Tomoko</td>
											<td>Higashi-Ohwada 1-6-9-A303, Ichikawa, Chiba 272-0026</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK99/00678</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-12-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 1998 01604</td>
									<td>1998-12-04</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 1999 00330</td>
									<td>1999-03-09</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/212814-a-variant-of-a-parent-fungal-cutinase-and-a-method-of-producing-a-cutinase-variant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:33:50 GMT -->
</html>
