<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/213110-transcriptional-activator-gene-for-genes-involved-in-cobalamin-biosynthesis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:36:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 213110:TRANSCRIPTIONAL ACTIVATOR GENE FOR GENES INVOLVED IN COBALAMIN BIOSYNTHESIS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">TRANSCRIPTIONAL ACTIVATOR GENE FOR GENES INVOLVED IN COBALAMIN BIOSYNTHESIS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a transcriptional activator gene for genes involved in cobalamin biosynthesis. More precisely, it relates to a process for amplifying the production of cobalamins and, more specifically, of coenzyme B12 by means of recombinant DNA techniques.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br>
The present invention relates to a transcriptional activator gene for genes involved in cobalumin biosynthesis. More precisely, it relates to a process for amplifying the  produc¬tion of cobalamins and, more specifically, of coenzyme B12 by means of recombinant DNA techniques.<br>
Vitamin B12 is a member of molecules termed cobalamins and whose structure is pre¬sented, in particular, in W091/11518. It has been used as agent for treating pernicious anemia, nervous diseases, methylmalonic aciduria, and so on, and feed additives for fowls and domestic animals. The industrial production of vitamin B12 by chemical synthetic methods is difficult because of its complicated structure. Fermentation processes for in¬dustrial production use microorganisms selected from Pseudomonas demtrtficans, Propionibacterium shermanii and Propionibacterium freudcrtreiehii. The complicated biosynthetic pathway of vitamin B12 has been studied well especially in P. demtnficans, P. shermanii, Salmonella typhimurium, and Bacillus megaterutm. No less than 22 cob genes are involved in cobalamin biosynthesis (W091/11518). However, the gene furan enzyme catalyzing reduction of cob(H)yrinic acid a,c-diamide has not been cloned and functions of polypeptides encoded by cobE, cobw, and cobX have not been identified.<br>
Other genes involved in the biosynthesis of building blocks for cobalamins such as (R)-l-amino-2-propanol or 5,6-dimethylbenzimidazole (DMBI) were not identified in P. denitri-ficans although such genes were isolated from the photosynthetic bacterium Rhodobacter capsulatus (US 6,156,545), There are reports on a 189-kb segment of the chromosome of R. capsulatus, assigned as cob genes by analogy in this strain were also clustered and the blu genes involved in l-amino-2-propanol and DMBI synthesis were found within the large cluster.<br><br>
The vitamin B12 productivity of P. denitrificans was gradually improved from 0,6 mg/1 to 60 mg/1 through repeated classical mutagenesis and random screening. The vitamin B12 production process by P. denitrificans is characterized in that the fermentation is performed under aerobic condition from the first to the end contrary to the anaerobic fermentation or periodic fermentation from anaerobic to aerobic condition by Propiortibacterium. In addition, it appears that glycine betaine (betaine) is an essential substance for overproduction of vitamin B12 in P. denitrificans. Betaine is generally known as an osmoprotectant, but it was metabolized as N-source, C-source, and energy source in P. denitrificans as reported for the family Rhizobiaceae. Betaine is converted to dimethylglycine and methionine by transferring a methyl group to homocysteine by betaine homocysteine transmethylase. S-Adenosyl methionine synthesized from methionine is used as a methyl donor in the introduction of eight methyl groups to the corrinoid ring in vitamin B12 synthesis. On the other hand, dimethylglycine synthesized from betaine is converted to glycine via sarcosine possibly by dimethylglycine dehydrogenase and sarcosine oxidase. Glycine is used for synthesizing 5-aminoIevurinic acid (ALA), which is a common precursor for heme and vitamin Bl2, with succinyl-CoA through C4-pathway (Shemin pathway). Thus, betaine is considered as a possible precursor for vitamin B)2 synthesis. But, the true role of betaine in cobalamin overproduction could be of a regulatory nature because the incorporation of radioactivity from [Me-14C]betaine to cobalamin was indirect. Moreover, blockage of betaine metabolism improved production of vitamin B12. DNA fragments including genes involved in betaine metabolism were identified but the genes in the fragments were not identified. Therefore, they were no longer limited to the structural genes of enzymes in¬volved in betaine metabolism such as betaine homocysteine transmethylase, dimethyl¬glycine dehydrogenase, or sarcosine oxidase. A regulatory gene for the structural genes or genes involved in betaine uptake is also one of candidates.<br>
The present invention provides a gene designated cobR, which encodes a transcriptional activator, designated CobR for genes involved in vitamin B12 biosynthesis in P. denitrificans and the use of the gene or polypeptide for improving vitamin B12 production or for screening of genes involved in vitamin B12 synthesis.<br>
The present invention additionally provides the use of CobR as a transcriptional actlva'tor for genes involved in betaine metabolism.<br>
Although cobR was found to be an essential gene for betaine metabolism in P. denitrificans, restriction sites of cobR suggested that cobR was not included in the fragments 3s disclosed in US 5,691,163, Therefore, the present invention has made it possible to improve produc-<br><br>
tion of vitamin B12 bv appropriate expression of cobR without reducing betaine meta¬bolism.<br>
As described above, most genes involved in vitamin B12 synthesis in P. denitrificans have been discovered and well characterized but some genes have remained to be discovered. To discover such genes we performed random mutagenesis and obtained several mutants that could not produce vitamin B12- Among these mutants, there was a characteristic mutant of CEEX6, which neither produces vitamin B12 nor utilizes betaine. By using this mutant, we found a novel gene cobR in P. denitrificans, which could make CEEX6 to pro¬duce vitamin B12 and consume betaine simultaneously. CEEX6 has a point mutation in cobR and that cobR gene disruption lead to no production of vitamin B12 and no consump¬tion of betaine representing that cobR is an essential gene for vitamin B12 synthesis and betaine consumption. These results suggested that CobR could be a transcriptional activa¬tor for genes involved in vitamin B12 synthesis and betaine consumption although there were no examples of transcriptional activator for such genes in AraC/XylS family. Then, we demonstrated that CobR was a-transcriptional activator for genes involved in vitamin B12 synthesis by determining the promoter activity of cobE operon, which was a part of genes involved in vitamin B12 synthesis, with the help of a reporter gene. CobR also inter¬acts with upstream regions ofcobE, cobQ and cobF. Appropriate expression of cobR in¬creases vitamin Bj2 production in P. denitrificans.<br>
The present invention is directed to DNA sequences comprising a DNA sequence which encodes CobR, which is a transcriptional activator for genes involved in vitamin B12 syn¬thesis, as disclosed in the sequence listing, as well as their complementary strands, or those which include these sequences,JDNA sequences which hybridize, preferably under standard conditions with such sequences or fragments thereof and DNA sequences, which because of the degeneration of the genetic code, do not hybridize under standard conditions with such sequences but which code for polypeptides having exactly the same amino acid sequence.<br>
Protocols for identifying DNA sequences by means of hybridization are known to the per¬son skilled in the art. The hybridization may take place under stringent conditions, that is to say only hybrids in which the probe and target sequence, i.e. the polynucleotides treated -with the probe, are at least 70% identical are formed. It is known that the stringency of the hybridization, including the washing steps, is influenced or determined by varying the buffer composition, the temperature and the salt concentration. The hybridization reac¬tion is preferably carried out under a relatively low stringency compared with the washing steps.<br><br>
A 5x SSC buffer at a temperature of approx. 50-68°C, for example, can he employed for the hybridization reaction. Probes can also hybridize here with polynucleotides which are less than 70 % identical to the sequence of the probe. Such hybrids are less stable and are removed by washing under stringent conditions. This can be achieved, for example, by lowering the salt concentration to 2x SSC and subsequently 0.5x SSC at a temperature of approx. 50-68°C being established. It is optionally possible to lower the salt concentration to O.lx SSC. Polynucleotide fragments, for example, at least 70% or at least 80% or at least 90% to 95% identical to the sequence of the probe employed can be isolated by increasing<br>
the hybridization temperature stepwise in steps of approx. 1-2°C.<br>
"Stringent conditions" in the context of this invention mean that hybridization in a buffer, for example, consisting of 5 x SSC, 0.1%(w/v) N-lauroylsarcosine, 0.02%(w/v) SDS, 1%<br>
blocking reagent (Roche Diagnostics, Cat. No. 1096 176) at 50°C overnight and two times of washing with 2 x SSC, 0.]%(w/v) SDS for 5 min at room temperature and following two times of washing with 0.1 x SSC,0.1%(w/v) SDS for 15 min at 68°C in the washing step of hybridization.<br>
The present invention also includes DNA sequences which are at least 80% to 85%, prefer¬ably at least 86% to 90%, and particularly preferably more than 90% identical to the DNA sequence according to S£Q ID NQ:1 or a fragment thereof.<br>
The present invention is also directed to functional derivatives of the polypeptides of the present invention. Such functional derivatives are defined on the basis of the amino acid sequences of the present invention by addition, insertion, deletion and/or substitution of one or more amino acid residues of such sequences wherein such derivatives still have the activity of the transcriptional activator for genes involved in vitamin Biosynthesis. Such functional derivatives can be made either by chemical peptide synthesis known in the art or by recombinant means on the basis of the DNA sequence as disclosed herein by methods known in the state of the art. Amino acid exchanges in proteins and peptides which do not generally alter the activity of such molecules are known in the state of the art. The most commonly occuring exchanges are: Ala/Ser, Val/Ile, Asp/Glu, Thr/Ser, Ala/Gly, Ala/Thr, Ser/Asn, Ala/Val, Ser/Gly, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/Ile, Leu/Val, Ala/Glu, Asp/gly as well as these in reverse.<br>
Furthermore, polypeptides according to the invention include a polypeptide according to SEQ ID NO:2, in particular those with the biological activity of the transcriptional activa¬tor CobR for genes involved in cobalamin biosynthesis, and also those which are at least<br><br>
80% to 85%, preferably at least 86% to 90%, more preferably at least 91% to 95%, and particularly preferably more than 95% identical to the polypeptide according to SEQ ID NO: 2 and have the activity mentioned.<br>
To express the cobR gene or generally speaking the CobR activity encoding DNA sequence efficiently, various promoters can be used; for example, the original promoter existing up-stream of the cobR gene, promoters of antibiotic gene such as the kanamycin resistance<br>
gene of  Tn5, the ampicillin resistencegene of pBR322, and of the β-galactosidase of E, coli (lac), the trp-, tac-» trc-promoter, promoters of lambda phage and any promoters which can be functional in the hosts consisting of microorganism including bacteria such as £. coliy P. denitrificans, P. putida, Agrobacterium tumefaciens, A. radiobacter, and Sinorhizo-bium mcliloti, mammalian cells, and plant cells.<br>
For the object above, other regulatory elements such as a Shine-Dalgarno (SD) sequence (for example, AGGAGG, and so on, including natural and synthetic sequence operable in the host cell) and a transcriptional terminator (inverted repeat structure including any natural and synthetic sequences operable in the host cell) which are operable in the host cell into which the coding sequence will be introduced and used with the above described promoter.<br>
A wide variety of host/cloning vector combinations may be employed in cloning the double-stranded DNA. Cloning vector is generally a plasmid or phage which contains a replication origin, regulatory elements, a cloning site including a multiple cloning site and selection markers such as antibiotic resistance genes including resistance genes for ampi¬cillin, tetracycline, chloramphenicol, kanamycin, streptomycin, gentamicin, spectinomycin etc.<br>
Examples for a vector for the expression of the gene in £ coli are selected from any vectors usually used in £ coli, such as pBR322 or its derivatives including pUCl8 and pBlue-scriptll, pACYCl77 and pACVC184 and their derivatives, and a vector derived from a broad host range plasmid such as RK2 and RSFIOIO. A preferred vector for the expression of the object gene in P. denitrificans, P. putida, A, tumefaciens, A. radiobacter, or S. meliloti is selected from any vectors which can replicate in these microorganisms, as well as in a preferred cloning organism such as £ coli. The preferred vector is a broad host range vector such as a cosmid vector like pVKlOO and its derivatives and RSFIOIO and its deriva¬tives, and a vector containing a replication origin functional in P. denitrificans, P. putida, A. tumefaciens,  A. radiobacter, or S. meliloti and another origin functional in E. coli. Copy number and stability of the vector should be carefully considered for stable and efficient<br><br>
expression of the cloned gene and also for efficient cultivation of the host cell carrying the cloned gene. DNA sequences containing transposable element such as Tn5 can also be used as a vector to introduce the object gene into the preferred host, especially on a chromosome. DNA sequences containing any DNAs isolated from the preferred host to¬gether with the object gene are also useful to introduce the desired DNA sequence into the preferred host, especially on a chromosome. Such DNA sequences can be transferred to the preferred host by transformation, transduction, transconjugation or electroporation.<br>
As a preferable microorganism, there may be mentioned bacteria such as £ coli, P. denitri-ficans, P. putida, A. tumefaciens, A, radiobacter, S. meliloti, and any Gram-negative bacteria which are capable of producing recombinant CobR, Mutants of said microorganism can also be used in the present invention.<br>
The DNA sequence encoding CobR of the present invention may be ligated into a suitable vector containing a regulatory region such as a promoter and a ribosomal binding site and a transcriptional terminator operable in the host cell described above by well-known methods in the art to produce an expression vector.<br>
To construct a host cell carrying an expression vector, various DNA transfer methods in¬cluding transformation, transduction, conjugal mating, and electroporation can be used. The method for constructing a transformed host cell may be selected from the methods well known in the field of molecular biology. Usual transformation system can be used for £ coli and Pseudomonas. Transduction system can also be used for £ coli. Conjugal mating system can be widely used in Gram-positive and Gram-negative bacteria including £ coli, P. putida, S. rneliloti and P. dcnitrifxcans.<br>
Furthermore, the present invention relates to a process for screening of genes involved in vitamin B12 synthesis by using the polypeptide according to SEQ ID NO: 2, in particular those with the transcriptional activator activity mentioned. Because a transcriptional acti¬vator activates transcription of a gene by interacting with DNA upstream the gene to be activated, a DNA fragment interacting with the transcriptional activator can be isolated from genomic DNA fragments using a polypeptide of transcriptional activator. It can be expected that genes involved in vitamin B12 synthesis not only cob genes already isolated and identified are isolated by using affinity of polypeptide according to SEQ ID NO: 2 or its C-terminal part including DNA binding domain.<br>
The following examples are offered by way of illustration and not by way of limitation.<br>
Strains and plasmids<br><br>
Pseudomonas denitrtficans PF1-48 is derived from strain MB580 (US 3,018,225) and it was deposited at DSMZ under the accession number DSM 15208 under the Budapest Treaty.<br>
Strain CEEX6 is a mutant, which does not consume betaine nor produce vitamin B12<br>
i<br>
derived from PFl-48, and it was deposited at DSMZ under the accession number DSM 15207. Plasmid pGUS02, which was used in Example 3-(l), was constructed as follows. A<br>
promoter-less (3-glucuIonidase gene (gus) was prepared from pBHOl (CLONTECH Laboratories, Inc. CA) by PCR with the following primers; N-GUS: SEQ ID NO:3 (tagged with Sma I and Nde I sites) and C-GUS: SEQ ID NO:4 (tagged with Xho I and Sac I). Then, the Sma l-Sac 1 fragment of gus was introduced between the Sma I and Sac I sites in pBluescriptll KS"(Stratagene, La Jolla, CA). At last, a cassette consisting of SD and three stop codons in three frames with Pst /, BamH I, and Nde I sites (annealed product of SEQ ID NO: 5 and SEQ ID NO; 6) was introduced between Pst I and the constructed Nde I in the resulting plasmid.<br>
Plasmid pETcobR, which was used in Example 3-(2), was constructed as follows. To make Nde I site at the initiation codon of cobR, PCR with primers COBRNde: SEQ ID NO: 7, and COBR135R: SEQ ID NO: 8 against pCRIIcobRB (pCRII::4.5 kb BamHl) was per¬formed. Then, the 0.1 kb Nde I-Sph 1 fragment from the PCR product was cloned into pUCl8 between Nde I and Sph I sites and a clone that had the correct sequence was selected. Next, the 0.1 kb Nde l-Sph I fragment and a LI kb Sph I-EcoR 1 fragment from pCRIIcobRB were cloned simultaneously into pUCl8 between Nde I and EcoR 1 sites. At last, a L2 kb Nde I-£toR I fragment was excised from the resulting plasmid and cloned into a modified pETlla (Stratagene) of which BamH I site was converted to EcoR I site by using EcoR I linker: pGGAATTCC.<br>
Conjugation<br>
Conjugations between £ coli ED8767 or DH5α and P. denitriftcans strains were carried out by tri-parental conjugation method as follows, £. coli DH5 carrying pRK20l3 having tra genes were used as a helper for conjugal transfer. E. coli strains were cultivated in LB with<br>
appropriate antibiotics at 37°C, and P. denitriftcans strains were cultivated in LB at 30°C overnight. The portion of £ coli cultures were transferred to LB without antibiotics at 1% of inoculation size and cultivated for 6 hours. Cultures of P, denitrificans, donor £. coli, and helper £. coli were mixed at a ratio of 2 : 1 : 1 and the mixture was spotted onto nitro¬cellulose filter put on the LB agar plate. The plate was incubated at 30°C overnight for the mating. Then, transconjugants of P. denitnficans were selected from the cells grown on the filter by plating on LB agar with 100µg/ml of streptomycin and 200 µg/ml of neomycin or 2.5 µg/ml of tetracycline.<br><br>
Gus assay<br>
Cells from a portion (4 ml) of culture broth were harvested by centrifugation at 3,000 x g<br>
for 10 min at 4°C. The precipitate was washed with 3 ml of a buffer containing 25 mM Tris-HG, pH 7,4 once and resuspended in the same buffer. After measurement of protein concentration of the cell suspension using BCA protein assay kit (Pierce, Rockford, ILL) with bovine serum albmin as a standard, the suspension was diluted with the buffer at the concentration of 1 mg protein/ml. The diluted suspension was dispensed into an Eppen-<br>
dorf tube by 50 µ1 and was frozen at -80°C once. The frozen sample was thawed at 4°C and 0.64 ml of Z buffer (60 mM Na2HPO4, 40 mM NaH:PO4, 10 mM KCl, 50 mM 2-mercaptoethanoI (2-ME), 1 mM MgSO47aq, (add 2-ME just before use)) and 0.16 ml of Lysozyme solution (2.5 mg/ml Z buffer) were added to it. After mixing by vortex for 1 sec, the sample was incubated at 37°C for 5 min. Then, it was mixed for 1 sec. after addition of 8µl of 10% Triton X-100 and was placed on ice. Next, it was pre-'incubated at 30°C for 5 min and 400 ^1 of p-nitrophenyl-p-glucuronide (PNPG) solution (4 mg/ml Z buffer) was<br>
added. It was incubated at 30°C for 2 min. and 200µl of 1M Na2CO3 was added quickly to stop the reaction. To remove cell debris, the reaction mixture was centrifuged at 12,000 rpm for 5 min and absorbanceat 415 nm of the supernatant was measured. The molecular extinction coefficient of p-nitrophenol is assumed to be 14,000. One unit is defined as the<br>
amount of enzyme that produces one nmol of product per minute at 30°C.<br>
Assay of vitamin B12 by HPLC<br>
100µl of 0.88 M acetate buffer (pH 3.5) with 0.11% KCN and 2.2% NaN02 were added to 1 ml of broth in a 2 ml Eppendorf tube with safe-lock. After mixing well with vortex, the mixture was autoclaved for 5 min with a tabletop steam sterilizer for cyanidation. The autoclaved sample was centrifuged for 5 min at 12,000 rpm and the supernatant was<br>
applied for HPLC assay. Ten µ1 of the sample was injected into Kanto Chemical Mightysil<br>
RP-18 GP (4.6 mm x 250 mm, 5 µm) column and eluted with 0.05 M KH2PO4 : MeOH (75 : 25), pH 4.5 at flow rate of 0.8 ml/min. Vitamin B]2 eluted at about 10 min was detected by absorption at 361 nm.<br>
Assay of betaine by HPLC<br>
After centrifugation of broth to remove cells, the supernatant was filtered with Ekicrodisc<br>
25 (pore size 0.45 nm, Pall) and applied for HPLC assay. Ten µl of sample was injected<br>
into YMC-Pack PA (4.6 mm x 250 mm, 5 µm) column and eluted with Acetonitrile : H2O<br><br><br>
Example 1:      Isolation of the cobR gene from P, denitrificans<br>
(1)	Construction of a genomic library of P. denitrificans PF1-48<br>
Chromosomal DNA prepared from P. denitrificans PFI-48 was partially digested with Sal I and 15-35 kb fragments of the partial digests were isolated from preparative agarose gel electrophoresis (agarose; 0.7%); the gel piece containing the objective fragments was cut out and the fragments were electro-eluted into TAE buffer consisting of 40 mM Tris-<br>
acetate and 2 mM EDTA. In parallel, 5 µg of the cosmid vector pVKlOO (ATCC37156) was completely digested with Sal I and the digest was treated with a shrimp alkaline phos¬phatase (United States Biochemical Corporation, Cleveland, USA) according to the supplier's recommendation. The dephosphorylated pVKlOO (130 ng) was ligated with the 15-35 kb fragments (33 ng) by the DNA ligation kit ver.l (Takara Shuzo, Co. Ltd., Kyoto,<br>
Japan) at 26°C for 10 min. Then, the ligate was used for packaging by phage coat protein<br>
using λ  in vitro packaging kit (Amersham Biosciences AB, Uppsala, Sweden) according to the supplier's protocol. The resulting phage particles were used for infection of E. coli ED8767&gt; and about 400 kanamycin resistant colonies were obtained. These colonies were<br>
individually cultivated and the grown cells we-re-stored at -80°C in 15% glycerol until use.<br>
(2)	Screening of the genomic library for gene(s) complementing CEEX6<br>
When CEEX6 is cultivated in CCM-1 medium, the culture broth does not become auburn because of no production of vitamin B12. Using this characteristic of CEEX6, genomic library was screened for gene that makes CEEX6 produce auburn pigment. About 400<br><br>
cosmid clones described above were individually transferred into CEEX6 from E. coli ED8767 by conjugation and the resulting transconjugants were cultivated in the 100 µl of<br>
CCM-1 in micro-titer plate with shaking at 30°C for 1 to 2 days. As a result, one clone CEEX6 carrying pS4F6 could make the medium auburn. Then, the vitamin B12 produc¬tivity of the transconiugant in CCM-1 was evaluated in shaking flask cultivation. CEEX6 carrying pS4F6 recovered vitamin B12 production and betaine consumption simultaneous-<br>
ly.<br>
(3) Subcloning of a gene complementing CEEX6 in pS4F6<br>
The localisation of the gene complementing CEEX6, i.e. cobR, in pS4F6 is shown in Fig. 1 where it is depicted with EcoR I sites. Subclones from pS4F6 are also shown with comple¬mentation ability of CEEX6 on pigment and vitamin B12 production and betaine consumption. Each EcoR I fragment (E1-E6) does not complement CEEX6 in pigmentation ability (E7 was not tested). B: BamH I, E: EcoR I, Et; E coT22 I, H: Hind III, P: Pst IS: Sal I,X: XhoL<br>
Several Sal I fragments from pS4F6 were deleted by partial digestion of pS4F6 with Sal I and re-Iigation. 6 of EcoR I fragments (El: 8 kb including 5.5 kb EcoR I-Sal I fragment from pVK100, E2: 5 kb, E3: 2.7 kb, E4: 1.8 kb, E5: 1.3 kb, E6: 1 kb) excised from pS4F6 were individually subcloned into pVKlOO. Existence of E7 (0.4 kb) was not recognized at this moment. About 5 kb fragment was estimated to remain at the vector which was not excised with EcoR I. Constructed deletion mutants with Sal I and pVKlOO carrying each EcoR I fragment were transferred into CEEX6 and the resulting transconjugants were evaluated on pigment production in CCM-1. As a result, each EcoR I fragment did not give CEEX6 pigmentation ability. On the other<br>
hand, one of deletion mutants with Sal I, pS4F6AS94, which had about 7 kb insert and only one EcoR I site in it, could make CEEX6 produce auburn pigment in the medium. However, pS4F6AS39, which was different from pS4F6AS94 in shortage of a 1 kb Sal I fragment, did not give the pigmentation ability to CEEX6. Then, further deletion mutants<br>
from pS4F6AS94 were constructed and analyzed. Plasmid pS4F6ASHO, which had only<br>
the 1 kb Sal I fragment, did not show the pigmentation ability, while pS4F6AS132, which had the 1 kb Sal 1 fragment witlj the adjacent 3.2 kb one, showed the pigmentation ability. Therefore, the gene complementing CEEX6 should exist across the two Sal I fragments. Then, a 6.5 kb Eco722 I fragment was cloned, which completely included these two Sal I fragments, from pS4F6 between two £coT22 I sites in kanamycin resistance gene on pVKlOO (pVKcobREt). In addition, 4.5 kb BamH I, 2.3 kb Xho I, and 1.9 kb Pst I frag¬ments were prepared including the Sal I site between the two Sal I fragments using BamH<br><br>
I, Xho I, or Pst 1 site in the 3.2 kb Sal I fragment. The 4.5 kb BamH I fragment was re¬placed with the 1.7 kb BglII fragment including cos site of pVKlOG to construct pVKcobRB. The Xho I and Pst I fragments were introduced respectively at Xho I site and between two EcoT22 I sites in the kanamycin resistance gene on pVKlOO to construct pVKcobRX and pVKcobRP. These clones were introduced into CEEX6 and assayed for vitamin B12 productivity and betaine consumption activity. As shown in Table 1, all of the clones except for the clone carrying pVKcobRX and pVKlOO recovered vitamin Bi2 pro¬duction and betaine consumption of CEEX6.<br><br>
(4) Determination of the nucleotide sequence of the gene complementing CEEX6 In order to define the gene complementing CEEX6, the nucleotide sequence was deter¬mined with a focus on the 1.9 kb Pst I fragment, because it was the shortest fragment com¬plementing CEEX6. We found one ORF of the gene which we designated cobR slightly across the Pst I site at the C-terminal. The ORF consisted of 1002 bp (SEQ ID NO:l: between nucleotide position 303 and 1304) encoding a polypeptide of 334 amino acids (SEQ ID NO:2). An Xho I site was found inside the ORF and it was consistent with the fact that pVKcobRX did not complement CEEX6. The gene product CobR could be a transcriptional activator belonging to AraC/XylS family because it had the helix-turn-helix motif found by Motifs progoram in GCG(Genetics Computer Group, University Research<br><br><br>
fragment between two EcoT22 I sites in pVK100, 4 amino acids at C-terminal of CobR (QN4AR) would be replaced with 6 amino acids derived from the vector sequence (HHQEYG). This modified CobR could be functional to complement CEEX6. Homology search for cobR performed with Blast (NCBI, Bethesda, Md. USA) showed that it was homologous to a gene SMc04I69 in the genome sequence of Sinorhizobium meliloti 1021 (Accession No. AL591688). Identity of cobR to SMc04169 was 86.6% at nucleotide level and 93.7% at amino acid level. The function of the gene product of SMc04169 was estimated as a transcriptional activator but it is hard to imagine that it can be an activator for genes involved in vitamin B12 synthesis.<br>
Example 2:      Confirmation of the relationship between the cobR mutation and<br>
deficiency in vitamin B12 production and betaine consumption<br>
(1)	Determination of a point mutation in the cobR from CEEX6<br>
Introduction of cobR gene certainly complemented CEEX6, but it does not exactly mean CEEX6 has mutation in cobR, In order to confirm that the cobR mutation causes no con¬sumption of betaine and no production of vitamin B12, the cobR gene from CEEX6 was cloned and its nucleotide sequence determined as follows. A 2,1 kb fragment including the whole ORF of cobR was amplified from chromosome of strain PFl-48 by PCR, and it was used as a probe after labeling with DIG. Because we found that the 4.5 kb BamH I frag¬ment also existed in CEEX6 by Southern blotting analysis, the fragment was cloned by using colony hybridization method with the probe described above. Then, we confirmed that introduction of this 4.5 kb BamH 1 fragment with pVKlOO did not complement CEEX6. When a 2.1 kb BamH 1-Hind III fragment, which includes an N-terminal half of CobR, was replaced with a corresponding fragment from the BamH I fragment from PFl-48, the resulting fragment also did not show the complementing activity. Therefore, we predicted that a mutation existed within the residual 2.4 kb Hind III-BamH 1 fragment encoding a C-terminal half of CobR. In fact, we found a point mutation in the cobR gene of CEEX6 in the C-terminal region within the conserved region among AraC/XylS family transcriptional activator; Cys299(TGC) was changed to Arg299(CGC) in the cobR gene from CEEX6.<br>
(2)	Random Tn mutagenesis against the 6,5 kb £coT22I fragment including cobR<br>
Using an in vitro Tn mutagenesis kit GSP-1 (New England BioLabs, Inc., Beverly, MA<br>
pCRII vector (Invitrogen Corporation, Carlsbad, CA) carrying the 6.5 kb EcoT22 I<br>
fragment was mutated with pGPS2,l(CAT gene in the Tn). From the mutated sample, 8<br>
kb EcoT22 I fragments including Tn were excised and introduced into pVKlOO to trans-<br>
form Exoli DH5a As a result, we obtained 128 Cm resistant transformants carrying<br><br>
various mutated fragments. Then, these plasmids were individually transferred into CEEX6 for testing complementation ability of pigmentation. Thirteen of 128 transconju-gants showed no complementation ability. As summarized in Fig. 2, all of 6 negative clones carrying pVKTn2, 5, 9, 26, 46 or 84 analyzed out of 13 had a Tn in the ORF of cobR. On the other hand, Tn insertion surrounding cobR ORF such as Tn30 (81 bp down¬stream), Tn28 (about 300 bp downstream of cobR within a probable next ORF), or Tn35 (about 340 bp upstream) did not lose complementation ability. Plasmid pVKTn3, which exceptionally showed weak complementation ability, was inserted just at the Pst I site. Although substitution of 4 amino acid residues at C-terminal with 6 residues was possible in pVKcobRP, substitution of two (AR) with other 8 residues (WADNKVLN) was not acceptable.<br>
Figure 2 shows random mutagenesis against 6.5 kb EcoT22 1 fragment: Tn insertion sites investigated were mapped on the 6.5 kb Ecolll I fragment. The ORF of cobR was shown as open box. Complementation ability of each fragment with Tn on pVK100 against CEEX6 were expressed as plus or minus in parenthesis with mapped Tn insertion site. 13: BamH I, E: EcoR I, Et: EcoT22 1, H: Hind III, P: Pst I, S: Sal 1, X: Xho I. (3) Construction of a cobR-null mutant<br>
As described above, several fragments carrying Tn in cobR were obtained. Plasmid pVKTn26, which had a Tn insertion at the center of cobR between Hind III and Sal I (Fig. 2) was used for null mutant construction. The 1.9 kb Pst I fragment with Tn (3.3 kb in<br>
total) from pVKTn26 was inserted at the Pst I site of a suicide vector pSUP202A£, which was constructed by filling in the cohesive end of EcoR I site in pSUP202 to disrupt CAT gene on the plasmid. Then, using Sal I sites the CAT gene inside Tn together with a next Sal I fragment was replaced with gentamicin (Gm) resistance gene in the suicide plasmid. The resulting plasmid was introduced into PFl-48 and a Gm resistant cobR-null mutant Tn26 was obtained. Strain Tn26 did not produce vitamin B12 nor consume betaine in CCM'l medium at all as well as CEEX6. It means that cobR is one of essential genes for vitamin B12 synthesis in P. denitnficans.<br>
Example 3:      Demonstration of CobR involvement in activation of cob genes<br>
Analyses of nucleotide sequence and deduced amino acid sequence suggested that cobR gene product CobR could be a transcriptional activator belonging to AraC family. Becaus'e cobR deficient mutants could not consume betaine nor produce vitamin B12, we predicted that genes involved in betaine metabolism and vitamin B12 synthesis were activated by CobR. In addition, betaine may be an effector molecule for CobR.<br><br>
(1) Estimation of cob gene expression level with or without CobR and betaine We applied a promoter probe vector, pGus02, which was constructed using gus as a reporter gene, to estimate expression level of cob genes. We picked up cobEABCD operon as a representative of cob genes because cob A in the operon encodes the enzyme catalyzing the first step reaction in the vitamin 812 synthetic pathway. In addition, we investigated expression level of cobR itself to know whether cobR was autorcgulated or not. As a frag, ment including promoter for cobEABCD operon, a 0.7 kb Hind HI fragment covering 0.7 kh Bgl II-Hind III fragment corresponding to 59th-792nd nucleotide in the sequence of cobEABCD genes resistered by Crouzet et al in GenBank (Accession No. M59236), which has about 500 bp of S'-noncoding region and about 200 bp of coding region of cobE, was inserted at Hind III site of pGus02. On the other hand, a 1.1 kb Xho I fragment, which has about 0.9 kb of 5'-noncoding region and 0,2 kb of coding region of cobR, was used as a fragment probably including a cobR promoter. The LI kb Xho 1 fragment was inserted between two Sal I sites of pGus02a, which was constructed by insertion of multiple cloning sites from pUC19 between Hind 111 and BamH I sites of pGus02. Then, units of pcobE-gus and pcobR-gus were respectively excised as 2.6 kb and 3 kb Xho 1 fragments from the vector and introduced at Xho 1 site of pVKlOO in the reverse direction to kanamycin resistance gene on the plasmid to construct pVKpcobEgus and pVKpcobRgus, respectively. Then, the constructed plasmids were introduced into PF1-48 and CEEX6, and resulting transconjugants were cultivated in CCM-1 (betaine including medium) and CCM-1X (betaine-less but glutamate amplified from 1 to 10 g/1), in which B12 was not produced though cells grew well. After two days cultivation, cells were collected and applied for gus assay. As shown in Table 2, we found that cobE promoter was repressed when cells were grown in the betaine-less medium or in the cobR deficient strain CEEX6, On the other hand, cobR promoter was not so repressed even in the betaine-less medium, although the activity in the cobR wild strain PFM8 in CCM-1 was the highest. These results suggest that CobR activated cobE promoter along with betaine because cobE promoter was re¬pressed though cobR could be expressed in the betaine-less medium and that CobR further activated own cobR promoter with betaine to some extent.<br><br><br><br>
(2) Demonstration of CobR function by using gel shift assay (CobR expressed in E. coti) As described in the previous section, assay of promoter activity suggested that CobR activated cobE promoter with betaine. To get more direct evidence for CobR involvement in activation of cob genes, we conducted a gel shift assay.<br>
Instead of purified CobR, we used a cell free extract (CFE) off. coli BL21(D£3)/p£TcobR expressing CobR by the pET system (Stratagene) together with CFE of E. coli carrying only a pET vector as a negative control. E. coli cells cultivated in 2 x 10 ml of LB supplemented with ampicitlin overnight were transferred to 2 x 200 ml of the same medium and cultiva¬ted for 4 hours at 37°C with shaking at 180 rpm. Cells were harvested by centrifugation at 4,000 x g for 10 min and washed once with 50 mM Tris-HCl (pH 7.5), 5 mM MgCI2. The washed cells were resuspended in the same buffer at a concentration of 0.2 g of wet cell/ml. Then, CFE from the cell suspension was prepared by passing through French pressure cell twice at 1500 kg/cm2 and removing cell debris by centrifugation at 3,000 x g for 10 min. On the other hand, we prepared a 290 bp (18 to 307 from the initiation codon) fragment upstream of cobE (cobEFlRl)by using PCR and labeled it at the 3'-end with DIG-H-ddUTP by terminal transferase included in DIG Gel Shift Kit (Roche Diagnostics, Basel, Switzerland).<br>
According to the protocol recommended by the supplier, the gel shift assay was performed with the labeled fragment and the CFE, One ng of labeled fragment was incubated with<br>
CFE including 1.2 mg of protein in 15 µl of 20 mM HEPES, pH 7.6, 1 mM EDTA, 10 mM (NH4)2SO4, 1 mM DTT,0.2%(w/v) Tween 20, 30 mM KCI&gt; 50 ng/µl PoIy[d(I-C)], 5 ng/µl Poly L-lysine for 15 min at room temperature for binding reaction. After addition of 5µl<br><br>
or 0.25 x 60% TBE buffer, 40% glycerol, 10 µl of the sample mixture was applied onto polyacrylamide gel electrophoresis (6% acrylamide in 0.25 x TBE, running buffer: 0.25 x TBE, constant voltage at 80 V for 2 hours), DNA and DNA-protein complex were electro-blotted onto positively charged nylon membrane (Boehringer Manheim GmbH Cat. No. 1209 299) by Trans-Blot SD (Bio-Rad Laboratories, Hercules, CA) using 0.25 x TBE as a transfer buffer at 25V for 45 min. Transferred DNA molecules were fixed on the mem¬brane by baking the membrane at 120°C for 15 min and were detected according to the protocol recommended by the supplier. As a result, we observed retardation of the labeled fragment when it was reacted with the CFE including CobR, In addition, existence of 100-fold excess amount of non-labeled cobEFlRl inhibited the retardation. On the contrary, the CFE from the vector control did not cause shift of the DNA fragment. These results strongly suggest that CobR directly binds to the regulatory region of cobEABCD operon to activate the transcription of the operon.<br>
(3) Demonstration of CobR involvement in activation of cob genes (CobR expressed in P.<br>
denitrificans)<br>
We found that the CFE off. colt including CobR was applicable for gel shift assay. It<br>
means we could demonstrate that CobR directly bound to the regulatory region of the<br>
cobEABCD operon. Then, CFE of the wild P. denitrificanfstrain PFl-48 was applied for gel<br>
shift assay in comparison with CFEs of CEEX6 {cobR with a point mutation) and Tn26<br>
(cobR-null mutant) as negative controls. These strains were cultivated in 200 ml of LB for<br>
16 hours at 30°C with shaking at 220 rpm, and CFE was prepared as well as E coli cells described in Example 3-(2). Besides cobEABCD operon, region upstream the other cob genes were examined additionally. We prepared 208 bp (from T in the initiation codon) fragment upstream of cobF (cobFFlRl) and 281 bp (10 to 290 from the initiation codon) fragment upstream of cobQ (cobQFlRl) by PCR based on the sequences registered in Entrez (Accession No. A30008 and M62866). These fragments were labeled with DIG-11-ddUTP by terminal transferase and applied for gel shift assay.<br>
As a result, we found that fragments from upstream region of cobE and cobQ clearly shifted only by CFE of PFl-48. On the other hand, although only a small part of fragments from upstream region of cobF shifted, the shift occurred when 10 mM of betaine was added into the binding reaction.<br>
Example 4:      Purification of a truncated CobR for preparation of anti-serum against<br>
CobR<br><br>
Although it was difficult to express cobR efficiently in £ coli as well as other transcriptional activators belonging to the AraC/XylS family even by using strong promoters such as tac or T7 in PET system, we found that deletion of N-terminal side at Hind III site<br>
(pCRIlcobRBAH) of a clone carrying the 4.5 kb BarnH I fragment in pCRlI (pCRIIcobRB)<br>
could express truncated CobR fused with N-terminal of LacZ (20 a.a.) coming from the<br>
multiple cloning site (CobRANHd: 20+186/334 (CobR)=206 a.a., 23,4 kDa) efficiently as<br>
an inclusion body.without IPTG induction. Then, CobRANHd was purified from the in¬clusion body by preparative SDS-PAGE after solubilization with 8M urea as follows. By cultivating in 1L of LB medium supplemented with kanamycin, we obtained 3.3 g of wet cells of the recombinant E. coli. The cells were suspended in 16.5 ml of buffer A consisting of 50 mM Tris/HCl, 5 mM MgCl: (pH 7.5) and the suspension was passed through French pressure cell at 1500 kg/cm" twice. Next, inclusion body with cell debris was recovered from the solution by centrifugation at 3,000 x g for 10 min. The precipitate was washed with buffer A once, then solubilized with 2.5 ml of buffer B consisting of 100 mM NaH2PO4 10 mM Tris, 8M urea (pH 8.0) by gentle mixing for one hour. The solubilized -proteins were recovered by centrifugation at 3,000 x g for 10 min. At last, 125 mg protein in 2.5 ml buffer B was obtained. Two mg of protein was applied for preparative SDS-PAGE (15% acrylamide). The major protein band of CobRANHd was cut out from the gel, and the gel slice was provided to prepare rabbit anti-CobR antiserum.<br>
Example 5:      DNA binding ability of the truncated CobR<br>
Transcriptional activators that belong to AraC/XylS family have two HTH motifs near C-terminal. Sequence conservation at the first HTH motif is low and the highly divergent each other whereas the second HTH motif does not have such variation. The conserved sequence at the second HTH motif is a determining factor in classification of transcrip¬tional activator into AraC/XylS family. As a matter of course CobR has the motifs in the C-terminal so that CobRANHd, which has C-terminal half of CobR, kept the DNA bind¬ing motifs. For the Bacillus thermoglucosidasium Hrc repressor it is reported that addition of any species of DNA appear to be effective for renaturation, and DNA containing Hrc binding site (CIRCE; controlling inverted repeat of chaperon expression) was far more effective "than other nonspecific DNA. Then, the solubilized CobRANHd with 8 M urea in Example 4 was refolded under existence of DNA, which has CobR binding region. As such a DNA, we prepared pCRIIcobApd, which carries a 2.3 kb Bgl U-EcoR V fragment in-<br>
eluding cobEA genes in pCRII vector. Eighty µg  of solubilized CobRANHd, 5 µg of pCRIIcobApd in 100 nl of 20 mM Tris-HCl (pH 7.5), 5 mM EDTA, 3M urea was dialyzed<br><br>
against 20 mM Tris-HCl (pH 7.5), 5 mM EDTA. The dialyzate was diluted with 50 mM Tris-HCl (pH 7,5), 5 mM MgCh to 1% (8 M-g protein/ml). Using Iµl of the solution instead of CFE and the labeled cobQFJRl prepared in Example 3-(3) in the binding reac¬tion described in Example 3-(2) gel shift assay was performed. As a result, we observed<br>
retardation of the fragment by renaturated CobRANHd. This result suggests that the N-<br>
terrninal-truncated CobR or CobRANHd also has binding activity against regulatory region of cob genes.<br>
Example 6:      Vitamin Bn production by cobR-amplified PF1-48<br>
PFl-48 carrying pVKcobRB and PFl-48 carrying pVK100 were cultivated in seed culture medium #688 for 2 days at 30°C The portion (1 ml) of seed culture broth was inoculated into 30 ml of CCM-1 medium in a 500 ml EMF. These main fermentations were per¬formed at 30°C, 220 rpm on a rotary shaker for 4 days. At the second day, 5 ml of feed<br>
medium consisting of 450 g of sucrose, 120 g of betaine, 3 gof MgSO4-7aq, and 0.105 g of 5,6-dimethyibenzimida/ole/L were added and the cultivation was continued for further two days. After conization of coenzyme B12, quantity of vitamin Bv2 produced was measured by HPLC. As a result, the former strain produced 6% higher amount of vitamin B12than the latter did.<br><br><br><br><br><br><br><br><br>
Claims:<br>
1.	A process for the biological production of cobalannin which comprises introducing an<br>
isolated DNA into an appropriate host organism, cultivating the host organism under the<br>
condition conductive to the production of cobalamin and recovering cobatamin from the<br>
culture, said isolated DNA comprising a nucleotide sequence that encodes CobR. which is a<br>
transcriptional activator for genes involved in vitamin B12 synthesis, selected from the group<br>
consisting of:<br>
(a)	a DNA sequence identified by SEQ ID N0:1 or the complementary strand thereof;<br>
(b)	a DNA sequence which hybridizes under stringent conditions to the DNA sequence complementary to the DNA sequence defined in (a) or a fragment thereof, and encodes a polypeptide having the activity of the transcriptional activator CobR for genes involved in vitamin 812 synthesis;<br>
(c)	a DNA sequence which codes for a polypeptide having the amino acid sequence encoded by the DNA sequence of (a) or (b);<br>
(d)	a DNA sequence which is identical to the extent of at least 80% to a DNA which codes for a polypeptide which comprises the amino acid sequence of SEQ ID NO:2;<br>
(e)	a DNA sequence which is identical to the extent of at least 90% to a DNA which codes for a polypeptide which comprises the amino acid sequence of SEQ ID N0:2;<br>
(f)	a DNA which codes for a polypeptide which comprises an amino acid sequence which is identical to the extent of at least 80% to the amino acid sequence of SEQ ID N0:2, the polypeptide preferably having the activity of the transcriptional activator CobR for genes involved in vitamin B12 synthesis;<br>
(g)	a DNA which codes for a polypeptide which comprises an amino acid sequence which is identical to the extent of at least 95% to the amino acid sequence of SEQ ID N0:2, the polypeptide preferably having the activity of the transcriptional activator CobR for genes involved in vitamin B12 synthesis.<br>
2,	The process of claim 1, wherein said host organism is Pseudomonas denitrificans,<br>
Agrobacterium radiobacter, or Sinorhizobium meliloti.<br><br>
3	The process of claim 2, wherein said host organism is Pseudomonas denitrificans CEEX6 or Pseudomonas denitrificans PF1-48, both deposited with the DSM2 under the Budapest Treaty.<br>
4	A process for discovering genes involved in vitamin B12 biosynthesis by using binding activity of a polypeptide encoded by the isolated DNA of any one of (a) to (g) in claim 1 against vitamin B12 biosynthesis genes.<br>
5	An isolated DNA comprising a nucleotide sequence that encodes CobR» which is a transcriptional activator for genes involved in vitamin B12 synthesis, selected from the group consisting of:<br><br>
(a)	a DNA sequence identified by SEQ ID NOfi or the complementary strand thereof;<br>
(b)	a DNA sequence which is more than 90% identical to the DNA sequence according to SEQ ID NO: 1;<br>
(c)	a DNA sequence which codes for a polypeptide having the amino acid sequence encoded by the DNA sequence of (a);<br>
(d)	a DNA sequence which is identical to the extent of at least 90% to a DNA which codes for a polypeptide which comprises the amino acid sequence of SEQ ID N0:2;<br>
(e)	a DNA which codes for a polypeptide which has an amino acid sequence which is identical to the extent of at least 95% to the amino acid sequence of SEQ ID N0;2. the polypeptide preferably having the activity of the transcriptional activator CobR for genes involved in vitamin 612 synthesis.<br><br>
6	A vector or plasmid comprising the isolated DNA of any one of (a) to (e) in claim 5.<br>
7	A host organism transformed or transfected by the isolated DNA as claimed in any one of (a) to (e) in claim 5 or by the vector or plasmid as claimed in claim 6.<br>
8	A polypeptide encoded by the isolated DNA as claimed in any one of (a) to (e) in claim 5.<br>
9	A process for the production of the polypeptide as claimed in claim 8 having activity of transcriptional activator for genes involved in vitamin B12 synthesis, which comprises culturing the host cell as claimed in claim 7 under the conditions conductive to the<br><br>
production of said polypeptide, wherein the host cell is selected from the group consisting of Pseudomonas denitrificans, Agrobacterium radiobacter, Agrobacterium tumefaciens.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">466-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtY2xhaW1zIGZpbGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-claims filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtY2xhaW1zIGdyYW50ZWQucGRm" target="_blank" style="word-wrap:break-word;">466-chenp-2005-claims granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtY29ycmVzcG9uZG5lY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">466-chenp-2005-correspondnece-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZGVzY3JpcHRpb24oY29tcGxldGUpZmlsZWQucGRm" target="_blank" style="word-wrap:break-word;">466-chenp-2005-description(complete)filed.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZGVzY3JpcHRpb24oY29tcGxldGUpZ3JhbnRlZC5wZGY=" target="_blank" style="word-wrap:break-word;">466-chenp-2005-description(complete)granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">466-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">466-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtb3RoZXIgZG9jdW1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-other documents.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY2LWNoZW5wLTIwMDUtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">466-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="213109-method-and-apparatus-for-storing-subscriber-data.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="213111-a-method-of-producing-a-remote-ac-signal.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>213110</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>466/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Dec-2007</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Mar-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Het Overloon 1, 6411 TE Heerlen,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HOSHINO, Tatsuo</td>
											<td>2-18-14 Fueta, Kamakura-shi, Kanagawa-ken, Kanagawa 248-0027,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SETOGUCHI, Yutaka</td>
											<td>4-27-2 Kataseyama, Fujisawa-shi, Kanagawa-ken, Kanagawa 251-0033</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TOMIYAMA, Noribumi</td>
											<td>1-1-27-305 Kawana, Fujisawa-shi, Kanagawa-ken, Kanagawa 251-0015</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/21</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP03/10572</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-09-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>02021603.2</td>
									<td>2002-09-27</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/213110-transcriptional-activator-gene-for-genes-involved-in-cobalamin-biosynthesis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:36:14 GMT -->
</html>
