<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/214302-molecular-antigen-array by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:00:55 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 214302:MOLECULAR ANTIGEN ARRAY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MOLECULAR ANTIGEN ARRAY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a composition useful in production of vaccines compnsmg: (a) a non-natural molecular scaffold comprising: (i) a core particle selected from the group consisting of: (1) a core particle of non-natural origin; and (2) a core particle of natural origin; and (ii) an organizer, such as herein described, comprising at least one first attachment site, wherein said organizer is connected to said core particle by at least one covalent bond; (b) an antigen or antigenic determinant with at least one second attachment site, wherein said antigen or antigenic determinant is amyloid beta peptide (A&amp;#946;1-42) or a fragment thereof, and wherein said second attachment site being selected from the group consisting of: (i) an attachment site not naturally occurring with said antigen or antigenic r determinant; and (ii) an attachment site naturally occurring with said antigen or antigenic determinant, (c) optionally a heterobifunctional cross-linker, wherein said second attachment site is capable of association through at least one non-peptide bond to said first attachment site; and wherein said antigen or antigenic determinant and said scaffold interact through said association to form an ordered and repetitive antigen array.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the Invention<br>
The present invention is about a composition useful in production of vaccine. More particularly about molecular antigen array related to the fields of molecular biology, virology, immunology and medicine. The invention provides a composition comprising an ordered and repetitive antigen or antigenic determinant array. The invention also provides a process for producing an antigen or antigenic determinant in an ordered and repetitive array. The ordered and repetitive antigen or antigenic determinant is useful in the production of vaccines for the treatment of infectious diseases, the treatment of allergies and as a pharmaccine to prevent or cure cancer and to efficiently induce self-specific immune responses, in particular antibody responses.<br>
Background Art<br>
WO 00/3227 describes compositions and processes for the production of ordered and repetitive antigen or antigenic determinant arrays. The compositions are useful for the production of vaccines for the prevention of infectious diseases, the treatment of allergies and the treatment of cancers. The compositions comprise a core particle, such as a virus or a virus-like particle, to which at least one antigen or one antigenic determinant, is associated by way of at least one non-peptide bond leading to the ordered and repetitive antigen array.<br>
Virus-like particles (VLPs) are being exploited in the area of vaccine production because of both their structural properties and their non-infectious nature. VLPs are supermolecular structures built in a symmetric manner from many protein molecules of one or more types. They lack the viral genome and, therefore, are noninfectious. VLPs can often be produced in large quantities by heterologous expression and can be easily be purified.<br><br>
Examples of VLPs include the capsid proteins of Hepatitis B viius (Ulrich, et al..<br>
Vims Res. 50:141-182 (1998)), measles virus CWames, et aL, Gene 760:173-178 (1995)),<br>
Sindbis virus, rotavirus (US 5,071,651 and US 5,374,426), foot-and-mouth-disease virus<br>
(Twomey, et al.. Vaccine ii:1603-1610, (1995)), Norwalk virus (Jiang, X., et al.. Science<br>
250:1580-1583 (1990); Matsui, SJvl., et aL, J. ain. Invest. <1456-l461 the></1456-l461>
retrovira! GAG protein (WO 96/30523), the retrotransposon Ty protein pi, the surface protein<br>
of Hepatitis B virus (WO 92/11291) and human papilloma virus (WO 98/15631).<br>
It is generally difficult to induce imraune responses against self-molecules due<br>
to immunological tolerance. Specifically, lymphocytes with a specificity for self-molecules are usually hypo- or even unresponsive if triggered by conventional vaccination strategies.<br>
The amyloid B peptide (APi has a central role in the neuropathology of Alzheimers disease. Region specific, extracellular accumulation of Ap peptide is acconqjanied by microgliosis, cytoskeletal changes, dystrophic neuritis and synaptic loss. These pitfhological alterations are thought to be linked to the cognitive decline that defines the disease.<br>
In a mouse model of Alzheim disease, transgenic animals engineered to produce Ai,42 (PDAPP-nace), develop plaques and neuron damage in their brains. Recent work, has shown immunization of young PDAPP-mice, using Api-Â«2, resulted in inhibition of plaque formation and associated dystrophic neuritis (Schenk, D. et al.. Nature 400:113-71 (1999)).<br>
Furthermore immunization of older PDAPP mice that had already developed AD-like neuropathologies, reduced the extent and progression of the neuropathologies. The immunization protocol for these studies was as follows; peptide was dissolved in aqueous buffer and mixed 1:1 with complete Freunds adjuvant (for primary dose) to give a peptide concentration of 100/ig/dose. Subsequent boosts used incomplete Freunds adjuvant. Mice received 11 immimizations over an 11 month period Antibodies tities greater than 1:10 000 were achieved and maintained. Hence, immunization may be an effective prophylactic and therapeutic action against Aizheirnet disease.<br>
In another study, peripherally administered antibodies raised against Apj,<br>
were able to cross the blood-brain barrier, bind A peptide, and induce clearance of<br>
pre-existing amyloid (Bard, F. et al.. Nature Medicine 6:916-19 (2000)). This study<br>
utilized either polyclonal antibodies raised against APM2, or monoclonal antibodies<br>
raised against synthetic ftgments derived from different regions of Ap. Thus<br>
induction of antibodies can be considered as a potential therapeutic treatment for<br>
Alzheimer disease.<br>
It is well established that the administration of purified proteins alone is<br>
usually not sufficient to elicit a strong immune response; isolated antigen generally<br><br>
 must be given together with helper substances called adjuvants. Within these adjuvants, the administered antigen is protected against rapid degradation, and the adjuvant provides an extended release of a low level of antigen.<br>
As indicated, one of the key events in Alzheimer"s Disease (AD) is the<br>
deposition of amyloid as insoluble fibrous masses (amyloidogenesis) resulting in<br>
extracellular nenritic plaques and deposits around the walls of cerebral blood vessels<br>
(for review see Selkoe, D. J. (1999) Nature. 399, A23-31). The major constituent of<br>
the neuritic plaques and congophilic angiopathy is amyloid B (Afi), although these<br>
deposits also contain other proteins such as glycosaminoglycans and apolipoproteins.<br>
AB is proteolyticaily cleaved from a much larger glycoprotein known as Amyloid<br>
Precursor Proteins (APPs), which comprises isoforms of 695-770 amino acids with a<br>
single hydrophobic transmembrane region. A6 fonns a group of peptides up to 43<br>
amino acids in length showing considerable amino- and carboxy-terminal<br>
heterogeneity (tmncation) as well as modifications (oher, A. E., Palmer, K. C,<br>
Chau, v., &amp; BaU, M. J. (19S8) J. Cell Biol. 107, 2703-2716.	Roher, A. E.,<br>
Palmer, K- C, Yurewicz, E. C, Ball, M. J., &amp; Greenberg, B. D. (1993) J. Neurochem. 61,1916-1926). Prominent isoforms are A" 1-40 and 1-42. It has a high propensity to form 6-sheets aggregating into fibrils, which ultiniately leads to the amyloid. Recent studies demonstrated that a vaccination-induced reduction in train amyloid deposits resulted in cognitive improvements (Schenk, D., Barbour, R., Dunn, W., Gordon, G., Grajeda, H., Chiido, T., Hu, K., Huang, J., Johnson-Wood, K., Khan, K., et al. (1999) Nature. 400,173-177).<br>
We have surprisingly found that self-molecules or self-antigens presented in a highly ordered and repetitive array were able to efficiently induce self-specific inmaune responses, in particular antibody responses. Moreover, such responses could<br>
, even be induced in the absence of adjuvants that otherwise non-specifically activate antigen presenting cells and other immune cells.<br><br>
BRIEF SUMMARY OF THE INVENTION<br>
The present invention provides compositions, which comprises highly ordered and repetitive antigen or antigenic determinant arrays, as well as the processes for their production and their uses. Thus, the compositions of the invention are useful for the production of vaccines for the prevention of infectious diseases, the treatment of allergies and cancers, and to efficiently incce self-specific immune responses, in particular antibody responses.<br>
In a first aspect, the present invention provides a novel composition comprising, oi alteroativeiy consisting of, (A) a tion-natural molecular scaffold and (B) an antigen or antigenic determinant. The non-natural molecular scaffold comprises, or altEinatively consists of, (i) a core particle selected from the group consisting of (1) a core particle of non-natural origin and (2) a core particle of natural origin; and (ii) an organizer comprising at least one first attachment site, wherein said organizer is connected to said core particle by at least one covalent bond. The antigen or antigenic determinant is a self antigen or a fragment thereof and has at least one second attachment site which is selected from the group consisting of (i) an attachment site not naturally occurring with sad antigen or antigenic determinant; and (ii) an attachment site naturally occurring with said antigen or antigenic determinant TTie invention provides for an ordered and repetitive self antigen array through an association of the second attachment site to the first attachment site by way of at least one non-peptide bond. "Hius, the self antigen or self antigenic detenninant and the non-natural molecular scaffold are brought together through this association of the first and the second attachment site to form an ordered and repetitive antigen array.<br>
In a second aspect, the present invention provides a novel composition comprising, or alternatively consistii of, (A) a non-natural molecular scaffold and (B) an antigen or antigenic determinant. The non-natural molecular scaffold comprises, or alternatively consists of, (i) a core particle and (ii) an organizer comprising at least one first attachment site, wherein said core particle is a virus-like particle comprising<br><br>
-""recombinant proteins, or fragments thereof, of a bacteriophage, and wherein said oianizo" is connected to said core particle by at least one covalent bond. The antigen or antigenic detenninant has at least one second attachment site which is selected from the group consisting of (i) an attachment site not naturally occurring with said antigen or antigenic determinant; and (ii) an attachment site naturally occurring with said antigen or antigenic determinant. The invention provides for an ordered and repetitive antigen array through an association of the second attachment site to the first attachment site by way of at least one non-peptide bond.<br>
In a third aspect, the present invention provides a novel conqwsition comprising, or alteraatively consisting of, (A) a non-natural molecular scaffold and (B) an antigen or antigenic determinant The non-natural molecular scaffold comprises, or altematively consists of, (i) a core particle selected from the group consisting of (1) a core particle of non-natural origin and (2) a core particle of natm origin; and (ii) an organizer conqmsing at least one first attachment site, wherein said organizer is connected to said core particle by at least one covalent bond. The antigen or antigenic determinant is an amyloid beta peptide (ABi-42) or a fragment thereof, and has at least one second attachment site which is selected from the group consisting of (i) an attachment site not naturally occmring with said antigen or antigenic determinant; and (ii) an attachment site naturally occurring with smd antigen or antigenic determinant. The invention provides for an ordered and repetitive antigen array through an association of the second attachment site to the first attachment site by way of at IsasX. one non-peptide bond.<br>
In a fourth aspect, the present invention provides a novel composition<br>
comprising, or alternatively consisting of, (A) a non-natural molecular scaffold and (B)<br>
an antigen or antigenic determinant. The non-natural molecular scaffold comprises, or<br>
,1	&gt;<br>
altematively consists of, (i) a gore particle selected from the group consisting of (1) a core particle of non-natural origin and (2) a core particle of natural origin; and (ii) an organizer comprising at least one first attachment site, wherein said organizer is connected to said core particle by at least one covalent bond. The antigen or antigenic determinant is. an anti-idiotypic antibody or an anti-idiotypic antibody fragment and has at least one second attachment site which is selected from the group consisting of (i) an attachment site not naturally occurring with said antigen or antigenic determinant; and (ii) an attachment site naturally occurring with said antigen or<br><br>
"antigenic determinant. Tlie invention provides for an ordered and repetitive antigen array through an association of the second attachment site to the first attachment site by way of at least one non-peptide bond.<br>
Further aspects as well as preferred embodiments and advantages of the present invention will become apparent in the following as well as, in particular, in the light of the defied description., the examples and the accompanying claim&amp;.<br>
In a preferred embodiment of the present invention, the core particle is a virusÂ¬like particle comprising recombinant proteins of a RNA-phage, preferably selected from the group consisting of a) bacteriophage QP; b) bacteriophage R17; c) bacteriophage fr; d) bacteriophage GA; e) bacteriophage SP; f) bacteriophage MS2; g) bacteriophage Mil; h) bacteriophage MXl; i) bacteriophage NL95; k) bacteriophage f2; and 1) bacteriophage PP7. Most preferred are bacteriophage Q3 and bacteriophage fr.<br>
In another preferred embodiment of the invention, the recombinant proteins of the RNA-phages comprise wild type coat proteins.<br>
In further preferred embodiment of the invention, the recombinant proteins of the RNA-phages comprise mutant coat proteins.<br>
In yet another embodiment, the core particle comprises, or alternatively consists of, one or more different Hepatitis core (capsid) proteins (HBcAgs). In a related embodiment, one or more cysteine residues of these HBcAga are either deleted or substituted with another amino acid residue (e.g., a serine residue), hi a specific embodiment, the cysteine residues of the HBcAg used to prepare compositions of the invention which correspond to amino acid residues 48 and 107 in SEQ ID NO: 134 are either deleted or substituted with another amino acid residue {e.g., a serine residue).<br>
Further, the HBcAg variants used to prepare compositions of the invention will generally be variants which retain the ability to associate with other HBcAgs to form dimeric or multimeric structures that present ordered and repetitive antigen or antigenic determinant arrays.<br>
Jn another embodiment, the non-natural molecular scaffold comprises, or altCTuatively consists of, pili or pilus-like structures that have been either produced fix&gt;m pilin proteins or harvested from bacteria. When pih or pilus-hke structures are used to prepare compositions of the invention, they may be formed from products of pilin genes which are naturally resident in the bacterial cells bat have been modified<br><br>
by genetically engineered ie.g., by homologous recombination) or pilin genes which have been introduced into these cells.<br>
In a related embodiment, the core particle comprises, or alternatively consists of, pili or pilus-like structures that have been either prepared from pilin proteins or harvested from bacteria. Tliese core particles may be formed from products of pilin genes naturally resident in the bacterial cells.<br>
hi a particular embodiment, the oranizK" may comprise at least one first<br>
attachment site. The first and the second attachment sites are particularly important<br>
elements of compositions of the invention. In various embodiments of the invention,<br>
the first and/or the second attachment site may be an antigen and an antibody or<br>
antibody fragment thereto; biotin and avidin; strepavidin and biotin; a receptor and its<br>
ligand;   a   ligand-binding   protein   and   its   ligand;   intK"acting   leucine   zipper<br>
polypeptides; an amino group and a chemical group reactive thereto; a carboxyl group<br>
and a chemical group reactive thereto; a sulfliydiyl group and a chemical group<br>
reactive thereto; or a combination thereof.<br>
In a further preferred embotMment, ftie composition further comprises an<br>
amino acid linker. Preferably the amino acid hnker comprises, or alternatively<br>
consists of, the second attachment site. The second attachment site mediates a<br>
directed and ordered association and binding, respectively, of the antigen to the core<br>
particle. An important function of the amino acid linker is to further ensure proper<br>
display and accessibility of the second attachment site, and thus to facilitate the<br>
binding of the antigen to the core particle, in particular by way of chemical cross-<br>
Unldng. Another important property of ihe amino acid linker is to further ensure<br>
optimal accessibility and, in particular, reactivity of the second attachment site. These<br>
properties of the amino acid linker are of even more importance for protein antigens.<br>
In another preferred embodiment, the amino acid linker is selected from the group consisting of (a) CGG; (b) N-terminal gamma I-Iinker; (c) Nterminal gamma 3-Uiikfir, (d) Ig hinge regions; (e) N-terminSl glycine linkers; (f) (G)kC (G)i, with ii=fl-12 and k=0-5; (g) N-termin&amp;I glycine-serine linkers; (h) (G)iC(G)n,(S)i(GGGGS)n with n=0-3, k=0"5, m=0-10,1=0-2; (i) GGC; (k) GGC-NH2; (1) C-tenninal gamma 1-linker; (m) C-lerminal gamma 3-linker, (n) C-teiminal glycine linkers; (o) {G)iiCtG)i: with n=0-12 and kHD-5; (p) C-terminal glycine-serine linkers; (q) (G)o(S)i(GGGGS)n(G)aC(G)k with n=0-3, k=0-5, m=0-10,1=0-2, and o=K)-8.<br>
An important property of glycine and glycine serine linkers is their flexibility, in particular their structural flexibility, allowing a wide range of conformations and disfavoring folding into structures precluding accessibility of the second attachment<br><br>
â site. As glycine and glycine serine linkers contain either no or a limited amount of side chain residues, they have limited tendency for engagement into extensive interactions with the antigen, thus, farther ensuring accessibility of the second attachment site. Serine residues within the glycine serine linkers confer improved solubility properties to these linkers. Accordingly, the insertion of one or two amino acids either in tandem or isolation, and in particular of polar or charged amino acid residues, in the glycine or glycine serine amino acid linker, is also encompassed by the teaching of the invention.<br>
In a further preferred embodiment, the amino acid Unker is either GGC-NH2, GGC-NMe, GGC-N(Me)2, GGCNHET or GGC-N(Et)2, in which the C-terminus of the cysteine residue of GGC is amidated. Tliese amino acid linkers are preferred in particular for peptide antigens, and in particular for embodiments, in which die antigen or antigenic determinant with said second attachment site comprises AP peptides or fragments theerof. Particular preferred is GGC-NH2.In another embodiment, the amino acid Bnker is an Immunoglobulin (Ig) hinge region. Fragments of Ig hinge regions are also within the scope of the invention, as well as Ig hinge reons modified with glycine residues. Preferably, the Ig hinge regions contain only one cysteine residue. It is to be understood, that the single cysteine residue of the Ig hinge region amino acid linker can be located at several posidons within the linker sequence, and a man skilled in the art would know how to select them with the guidance of the teachings of this invention.<br>
In one embodiment, the invention provides the coupling of almost any antigen of choice to the surface of a virus, bacterial pilus, structure formed from bacterial pilin, bacteriophage, virus-like particle or viral capsid particle. By bringiag an antigen into a quasi-crystalline "virus-like" structure, th? invention exploits the strong antiviral immuna-reaction of a host for the pioducdcsi of a Mghly efficient immune response, f.e., a vaccination, against the displayed andgen.<br>
fci yet anotiier embodiment, the aigen may be selected from the group consisting of: (1) a protein suited to induce an immune response against cancer cells; (2) a protein suited to induce an immune response against infectious diseases; (3) a protein suited to induce an innnune response against allergens; (4) a protein suited to induce an iiiroved response against seVf-antigens; and (5) a |ffotein suited to induce an inmvune response in farm animals or pets. In another embodiment, the first attachment site and/or the second attachment site are selected from the group coicsisg". (1) a geiffitically engineered lysine residue and (2&gt; a .   genetically engineered cysteine residue, two residues that may be chemically linked together.<br><br>
In a yet further preferred embodiment, ursi aiiatnmeni siie compnses or is an amino group and said second attachment site comprises or is a sulfhydryl group. Preferably, the first attachment site comprises or is a lysine residue and said second attachment site comprises or is a cysteine residue.<br>
The invention also includes embodiments where the organizer particle has only a single first attachment site and the antigen or andgenic determinant has only a single second attachment site. Thus, when an ordered and repetitive antigen array is prepared using such embodiments, each organizer will be bound to a single antigen or antigenic determinant<br>
In a further aspect, the invention provides composidons conrising, or altemadvely consisdng of, (a) a non-natural molecular scaffold con:q)rising (i) a core particle selected &amp;om tbe group consisting of a core particle of non-natxKal origin and a core paidcle of natural origin, and (ii) an organizer comprising at least one first attachment site, wherein the core particle conrises, or alternatively consists of, a virus-like particle, a bacterial pilus, a pilus-lilffi structure, or a modified HBcAg, or fragment thereof, and wherein the organizer is connected to the core particle by at least one covalent bond, and (b) an antigen or antigenic detemunant with al least one second attachment site, the second attachment site being selected from the group consisting of (i) an attachment site not naturally occuiring with the antigen or antigenic determinant and (ii) an attachment site naturally occurring with the antigen or antigenic determinant, wherein the second attachment site is capable of association through at least one non-peptidc bond to tbe first attachment site, and wherein the antigen or antigenic detemunant and the scaffold interact tiirougb the association to form an ordered and repetitive antigen array.<br>
Other embodiments of the invention include processes for the production of conositions of die invention and a mtethods of medical treatment using vaccine compositions described herdn.<br>
It is to be understood that both the foregoing general description and the following detailed description are exemplary and explanatory only and are intended to provide further explanation of the invention as claimed<br>
In a still further aspect, the present mvention provides a coii5)osition comprising a bacteriophage Qp coat protein attached by a covalent bond to phospholipase A2 protein, or a fragment thereof. In a prefrared embodiment, the phospholipase A2 protein, or a fragment thweof, and the bacteriophage Qp coat protein interact thmugh the covalent bond to form an ordered and repetitive antigen array. In another preferred embodiment, the covalent bond is not a peptide bond In another prefacred embodiment, the phospholipase A? protein includes an amino acid selected from the group consisting of the amino acid sequence of SEQ ID NO; 168, the amino acid sequence of SEQ ID NO:169, the amino acid sequence of SEQ ID NO:170, the amino acid sequence of SEQ ID N0:171, the amino acjd sequence of SEQ ID<br><br>
NO: 172, the amino acid sequence of SEQ ID NO: 173, the amino add sequence of SEQ ID NO:174, and the amino acid sequence of SEQ ID NO:175.<br>
The present invention also provides a method of maidng the composition con5)rising combining the bacteriophage QP coat protein and the phospholipase Ai protein, wherein the bacteriophage Qp coat protem and the phospholipase A2 proton interact to form an antigen array. .<br>
In another aspect, the present invention also provides a conapositiois conqjiising a non-natural molecular scaffold consing a bacteriophage Q3 coat protein, and an CH"garuzer conqirising at least one first attachment site, wherein die organizer is connected to the bacteriophage Q&amp; coat picrtan by at least one covalenl bcmd; and phospholipase A proton, or a fragnnt thereof, or a variant thoÂ«of with at least one second attachment site, the second attachment site being selected from the group consisting of: an attachment site not naturally occurring with the a phosphoUpase A2 protein, or a fragment thereof; and an attachment site naturally occuning with the a phospholipase Az protein, or a fragment thereof, wherein the second attachment site associates through at least one non-peptide bond to the first attachment site, and wherein the antigen or antigenic detemiinant and the scaffold interact through the associaticm to form an ordered and repetitive antigen array. In a preferred anbodiment, fee phospholipase A; protein includes an amino acid selected from the group consisting of the amino acid sequence of SEQ ID NO: 168, the anuno acid sequence of SEQ ID NO: 169, the amino acid sequence of SEQ ID NO: 170, the amino acid sequence of SEQ ID NO: 171, the anuno acid sequence of SEQ ID NO: 172, the amino acid sequence of SEQ ID NO; 173, the amino acid sequence of SEQ ID NO: 174, and the amino acid sequence of SEQ ID NO: 175.<br>
The presemt invention also provides a nthod of making the con5)DSitioii conq)rising combining the bactedchage Qp coat protein and the phospholipase A protein, wherein the bacteriophage Q coat protein and the phospholipase A2 protem intact to form an antigen array. Preferably, the antigen array is ordered and/or repetitive.<br>
The presuit invention also provides a pharmaceutical con5)osition conrising a phospholipase Aa protein, and a pharmaceutically acceptable canio:. The. present invention also provides a vaccine conosition conrising a phospholipase A3 protein, hi a preferred embodiment, the vaccine conqjosition of claim 31, further comprising at least one adjuvant.<br>
The iffesent invmtion also provides a method of treating an aller to bee venom, consing administering the pharmaceutical composition or the vaccine composition to a subject. As a result of such administration the. subject exhibits a decreased immune response to the venom.<br>
The invention also relates to &amp; vaccine for the prevention of prion-mediated diseases by induction of anti-lyirholoxinp, anti-lymphotoxina or anti-<br><br>
" lymphotoxinP-receptoT antibodies. The vaccine contains protein carries foreign to the immunized human or animal coupled to lymphotoxinP or fragments thefeof, iymphotoxina or fragments thereof or the lymphotoxinp receptor or fragments thereof The vaccine is injected in humans or animals in order to induce antibodies specific for endogenous lymphotoxinP, Iymphotoxina or lymphotoxinp receptor. These induced anti-lymphotoxinp, Iymphotoxina or anti-lymphotoxinp receptor antibodies reduce or eliminate the pool of follicular dendritic cells present in lymphoid organs. Since prion-replication in lymphoid organs and transport to the central nervous system are impaired in the absence of follicular dendritic cells, this treatment inhibits progression of prion-mediated disease. In addition, blocking lymphotoxinp is beneficial for patients with autoimmune diseases such as diabetes type I.<br>
STATEMENT OF THE INVENTION<br>
An embodiment of the present invention relates to a composition useful in production of vaccine comprising:<br>
(a)	a non-natural molecular scaffold comprising:<br>
(i)        a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and<br>
(ii)       an organizer, such as herein described, comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at least one covalent bond;<br>
(b)	an antigen or antigenic determinant with at least one second attachment site,<br>
wherein said antigen or antigenic determinant is amyloid beta peptide (Api-42)<br>
or a fragment thereof, and wherein said second attachment site being selected<br>
from the group consisting of<br>
(i)        an attachment site not naturally occurring with said antigen or antigenic determinant; and<br><br>
(ii)       an attachment site naturally occurring with said antigen or antigenic determinant,<br>
(c)       optionally a heterobifimctional cross-linker,<br>
wherein said second attachment site is capable of association through at least one non-peptide bond to said first attachment site; and<br>
wherein said antigen or antigenic determinant and said scaffold interact through said association to form an ordered and repetitive antigen array.<br>
Another embodiment of the present invention relates to a process for producing a non-naturally occurring, ordered and repetitive antigen array comprising:<br>
(a)	providing a non-natural molecular scaffold comprising:<br>
(i)        a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and<br>
(ii)       an organizer, such as herein described, comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at least one covalent bond; and<br>
(b)	providing an antigen or anfigenic determinant with at least one second<br>
attachment site, wherein said antigen or antigenic determinant is amyloid beta<br>
peptide (APi. 42) or a fragment thereof, and wherein said second attachment<br>
site being selected from the group consisfing of:<br>
(i)        an attachment site not naturally occurring with said antigen or<br>
antigenic determinant; and (ii)       an attachment site naturally occurring with said antigen or antigenic<br>
determinant, wherein said second attachment site is capable of association through at least one non-peptide bond to said first attachment site; and<br>
(c)	combining said non-natural molecular scaffold and said antigen or anfigenic<br>
determinant, wherein said anfigen or antigenic determinant and said scaffold<br>
interact through said associafion to form an ordered and repetitive antigen<br>
array.<br><br>
BRIEF DESCRIPTION OF THE ACCOMPANYING FIGURES<br>
FIG. lA-IC    Modular eukaryotic expression vectors for expression of antigens<br>
according to the invention; FIG. 2A-2C    Cloning, expression and coupling of resistin to QP capsid protein; FIG. 3A-3B    Cloning and expression of lymphotoxin-p constructs for coupling to<br>
virus-like particles and pili. FIG. 4A-4B    Cloning, expression and coupling of MIF constructs to Qp capsid<br>
protein.<br>
FIG. 4C	ELISA analysis of IgG antibodies specific for MIF in sera of mice<br>
immunized against MIF proteins coupled to QP capsid protein.<br>
FIG. 5	Coupling of MIF constructs to fr capsid protein and to HBcAg- lys-<br>
2cys-Mut capsid protein analyzed by SDS-Page.<br>
FIG. 6	Cloning and expression of human-C-RANKL.<br>
FIG. 7	Cloning and expression of prion protein.<br>
FIG. 8A.	ELISA analysis of IgG antibodies specific for "Angio F" in sera of<br>
mice immunized against angiotensin peptides coupled to QP capsid<br>
protein.<br>
FIG. 8B.	ELISA analysis of IgG antibodies specific for "Angio II" in sera of<br>
mice immunized against angiotensin peptides coupled to Qp capsid<br>
protein.<br><br>
HG. 8C.      EliSA analysis of IgG antibodies specific for "Angio HI" in sera<br>
of mice immunized against angiotensin peptides coupled to QP<br>
capsid protein. FIG. 8D.      ELISA analysis of IgG antibodies specific for "Angio IV" in sera<br>
of mice immunized against angiotensin peptides coupled to QP<br>
capsid protein. FIG. 9A.      ELISA analysis of IgG antibodies specific for "Der p I p52" in<br>
sera of mice immunized against Der p I peptides coupled to QP<br>
capsid protein. HG. 9B.      ELISA analysis of IgG antibodies specific for for "Der p I pll7"<br>
in sera of mice immunized against Der p I peptides coupled to<br>
Qp capsid protein. FIG. lOA.    ELISA analysis of IgG antibodies specific for human VBGFR n<br>
peptide in sera of mice inmiunized against human VEGFR H<br>
peptide and the extracellular domain of human VEGFR n both<br>
coupled to Type-1 pili protein. FIG. lOB.    ELISA analysis of IgG antibodies specific for the extracellular<br>
domain of human VEGro. II in s-a of mice immunized against<br>
human VEGKl II peptide and extracellular domain of human<br>
VEGFR    H    both     coupled     to    Type-1     pili     protein. FIG. 11.       ELISA analysis of IgG antibodies specific for anti-IKFa protein<br>
in sera of mice immunized against full length HBc-TNF. FIG. 12.        ELISA analysis of IgG antibodies specific for anti-TNFa protein<br>
in sera of mice immunized against 2cysLys-mut HBcAgl-149<br>
coupled to the 3"TNF H peptide HG. 13A.    SDS-PAGE analysis of coupling of "AP1-15" to QP capsid<br>
protein using the cross-linker SMPH. HG. 13B.     SDS-PAGE analysis of coupling of "Ap33-*2" to Qp csid<br>
protein using the cross-linker SMPH. Habc.   "SDS-PAGE analysis of "coupling"of "Apl-27" to Qp capsid<br>
protein using the cross-linker SMPH. HG. 13D.    SDS-PAGE analysis of coupling of "Apl-15" to QP capsid<br>
protein using the cross-Unksr Sulfo-GMBS. no. 13E.     SDS-PAGE analysis of couphng of "Apl-15" to Qp capsid<br>
protein using the cross-linker Sulfo-MBS.<br><br>
FIG. 14A.    ELISA analysis of IgG antibodies specific for "Apl-15" in sera<br>
of mice immunized against "Apl-15" coupled to QP capsid<br>
protein. FIG. 14B.    EUSA analysis of IgG antibodies specific for "Api-27" in sera<br>
of mice immunized against "Api-27" coupled to Qp capsid<br>
protein. HG. 14C.    EUSA analysis of IgG antibodies specific for "Ap33-42" in sera<br>
of mice immunized against "Ap33-42" coupled to Qp capsid<br>
protein. HG. 15A.    SDS-PAGE analysis of coupling of pCC2 to QP capsid protein. FIG. 15B.    SDS-PAGE analysis of coupling of pCA2 to Qp capsid protein.<br>
FIG. 15C.    SDS-PAGE analysis of coupling of pCB2 to Qp capsid protein. HG. 16        Coupling of prion peptides to QP capsid protein; SDS-Page<br>
analysis. FIG. 17 A.    SDS-PAGE analysis of expression of IL-5 in bacteria HG. 17 B.   Western-Blot  analysis of expression  of IL-5  and IL-I3  in<br>
eukaryotic cells FIG. 18 A.   SDS-PAGE analysis of coupling of murine VEGFR-2 peptide to<br>
PiU. HG. 18 B.   SDS-PAGE analysis of coupling of murine VEGFR-2 peptide to<br>
Qp capsid protein. HG. 18 C.   SDS-PAGE analysis of coupling of murine VEGFR-2 peptide to<br>
HBcAg-lys-2cys-Mut HG 18 D.    EUSA analysis of IgG antibodies specific for murine VEGFR-2<br>
peptide in sera of mice immunized against murine VEGER-2<br>
peptide coupled to Pili. HG 18.E.     EUSA analysis of IgG antibodies specific for murine VEGFR-2<br>
peptide in sera of mice immunized against murine VEGFR-2<br>
peptide coupled to Qp capsid protein. HG 18 F.     EUSA analysis of IgG antibodies specific for murine VEGFR-2<br>
peptide in sera of mice immunized against murine VEGFR-2<br>
peptide coupled to HBcAg-lys-2cys-Mut. HG.19 A.    SDS-PAGE analysis of coupling of Ap 1-15 peptide to HBcAg-<br>
lys-2cys-Mut and fi- capsid protein.<br><br>
FIG.I9 B. ELISA analysis of IgG antibodies specific for Ap 1-15 peptide in sera of mice inimunized against Ap 1-15 peptide coupled to HBcAg-lys-2cys-Mut or fr capsid protein.<br>
FIG.20	ELISA analysis of IgG antibodies specific for human Ap in sera<br>
of transgenic APP23 mice immunized with human Ap peptides<br>
coupled to Q3 capsid protein. HG. 21        SDS-PAGE analysis of coupling of an Fab antibody fragment to<br>
QP csid protein. FIG. 22 A.   SDS-PAGE analysis of coupling of flag peptide coupled to<br>
mutant Qp capsid protein with cross-iinlcer sulfo GMBS FIG. 22 B.    SDS-PAGE analysis of coupling of flag peptide coupled to<br>
mutant QP capsid protein with cross-linker sulfo MBS FIG. 22 C.   SDS-PAGE analysis of coupling of flag peptide coupled to<br>
mutant Qp capsid protein with cross-linker SMPH FIG. 22 D.   SDS-PAGE analysis of coupling of PLA2-cys protein coupled to<br>
mutant Qp capsid protein with cross-linker SMPH<br>
FIG.23	ELISA analysis of immunization with M2 peptide coupled to<br>
mutant QP capsid protein and fr capsid<br>
HG. 24	SDS-PAGE analysis of coupling of DER pl,2 peptide coupled to<br>
mutant QP capsid protein FIG. 25 A    Desensitization of allergic mice with PLA2 coupled to QP capsid<br>
protein: temperature measurements FIG. 25 B    Desensitization of allergic mice with PLA2-cys coupled to<br>
Qp capsid protein; IgG 2A and Ig E titers FIG. 26        SDS-PAGE Analysis and "Western-blot analysis of coupling of<br>
PlA2-cys to Qp capsid protein FIG. 27 A    ELISA analysis of IgG antibodies specific for M2 peptide in sera<br>
of mice immunized against M2 peptide coupled to HBcAg-Iys-<br>
2cys-Mut, QP capsid protein, fr capsid protein, HBcAg-lys-1-183<br>
and M2eptide fused to HBcAg 1-183<br><br>
FIG. 28 A    SDS-PAGE   Analysis   of   couphng   of   anti-idiotypic   I<br>
mimobody VAE051 to QP capsid protein HG. 28 B.   ELISA analysis of IgG antibodies specific for anti-idiotypic<br>
antibody VAE051 and Htiraan IgE in sera of mice immunized<br>
against VAE051 coupled to Qp capsid protein<br>
DETAILED DESdUFTION OF THE INVENTION 1.        Definitions<br>
Alphavirus: As used herein, the tenn "alphavirus" refers to any of the RNA viruses included within the genus Alphavirus. Descriptions of the members of this genus are contained in Strauss and Strauss, Microbiol. Rev., 58:491-562 (1994). Examples of alphaviruses inciude Aura virus, Bebaru virus, Cabassou \im&amp;, Chikungunya virus, Easter equine encephalomyelitis virus. Fort morgan virus, Getah virus, Kyzylagach virus, Mayoaro virus, Middleburg vims, Mucambo virus, Ndumu virus, Pixuna virus, Tonate virus, Triniti virus, Una virus, Westem equine encephalomyelitis virus, Whataroa virus, Sindbis virus (SJN), Semliki forest virus (SFV), Venezuelan equine encephalomyelitis virus (VEE), and Ross River virus.<br>
Antigen: As used herein, the tenn "antigen" is a molecule capable of being bound by an antibody. An antigen is additionally capable of inducing a humoral immune response and/or cellular immune response leading to the production of B-and/or T-lymphocytes. An antigen may have one or more epitopes (B- and T-epitoes). The specific reaction refened to above is meant to indicate that the antigen will react, in a highly selective manner, with its corresponding antibody and not with the multitude of other antibodies which may be evoked by other antigens.<br>
Antigenic detenninant: As used herein, the terra" antigenic determinant" is meant to refer to that portion of an antigen that is specifically recognized by either B-or T-Iymphocytes. B-lymphocytes respond to foreign antigenic determinants via antibody production, whereas T-lymphocytes are die mediator of cellular immunity. Thus, antigenic determinants or epitopes are those parts of an antigen that are recognized by antibodies, or in the context of an MHC, by T-cell receptors.<br><br>
Association; As used herein, the term "association" as it applies to the first and second attachment sites, refers to at least one non-peptide bond. The nature of the association may be covalent ionic, hydrophobic, polar or any combinatioa thereof.<br>
Attachment Site, First: As used herein, the phrase "first attachment site" refers to an element of the "organizer", itself bound to the core particle in a non-random fashion, to which the second attachment site located on the antigen or antigenic determinant may associate. Tlie first attachment site may be a protein, a polypeptide, an amino acid, a peptide, a sugar, a polynucleotide, a natural or synthetic polymer, a secondary metabolite or compound (biotin, fluorescein, retinol, digoxigenin, metal ions, phenylmethylsulfonylfluonde), or a combination thereof, or a chemically reactive group thereof. Multiple first attachment sites are present on the surface of the non-natural molecular scaffold in a repetitive configuration.<br>
Attachment Site, Second; As used harein, the phrase "second attachment site" refers to an element associated with the antigen or antigenic determinant to which the first attachment site of the "organizer" .located on the surface of the non-natural molecular scaffold may associate. The second attachment site of the antigen or antigenic determinant may be a protein, a polypeptide, a peptide, a sugar, a polynucleotide, a natural or synthetic polymer, a secondary metabolite or compound (biotin, fluorescein, retinol, digoxigenin, metal ions, phenylmelhylsulfonylfluoride), or a combination thereof, or a chemically reactive group thereof. At least one second attachment site is present on the antigen or antigenic determinant. The term "antigen or antigenic determinant with at least one second attachment site" refers, therefore, to an antigen or antigenic construct comprising at least the antigen or antigenic detenninant and the second attachment site. However, in particular for a second attachment site, which is not naturally occurring within the antigen or antigenic determinant, these antigen or antigenic constructs comprise an "amino acid linker". Such an amino acid linker, or also just termed "linker" within this specification, either associates the antigen or antigenic detenninant with the second attachment site, or more prefrably, already comprises or contains the second attachment site, typically -but not necessarily - as one amino acid residue, preferably as a cysteine residue. The term "amino acid linker" as used herein, however, does not intend to imply that such an amino acid linker consists exclusively of amino acid residues, even if an amino acid linker consisting of amino acid residues is a preferred embodiment of the present invention. The amino acid residues of the amino acid linker is, preferably, composed of naturally occuring amino acids or unnatural amino acids known in the art, all-L or ali-D or mixtures thereof. However, an amino add linker comprising a molecule with a sulfhydryl group or cysteine residue is also encompassed within the invention. Such a molecule comprise preferably a C1-C6 alkyl-, cycloalkyl (C5,C6), aryl or heteroaryl moiety. Association between the antigen or antigenic determinant or optionally the<br><br>
second attachment site and the amino acid linker is preferably by way of at least one cQvalent bond, more preferably by way of at least one peptide bond.<br>
Bound: As used herein, the term "bound" refers to binding or attachment that may be covalent, e.g., by chemically coupling, or non-covalent, e.g., ionic interactions, hydrophobic interactions, hydrogen bonds, etc. Covalent bonds can be, for example, ester, ether, phosphoester, amide, peptide, imide, carbon-sulfur bonds, caibon-phosphorus bonds, and the like. The term "bound" is broader than and includes tenns such as "coiqiled," "fused" and "attached".<br>
Core particle: As used herein, the term "core particle" refers to a rigid structure with an inherent repetitive organization that provides a foundation for attachment of an "organizer". A core particle as used herein may be the product of a synthetic process or the product of a biological process.<br>
Coat protein(s): As used herein, the term "coat protein(s)" refers to the protein(s) of a bacteriophage or a RNA-phage capable of being incorporated within the capsid assembly of the bacteriophage or the RNA-phage. However, when refening to the specific gene product of the coat protein gene of RNA-phages the term "CP" is used. For example, the specific gene product of the coat protein gene of RNA-phage QP is referred to as "Qp CP", whereas the "coat proteins" of bacteriophage Qb comprise the "Qp CP" as well as the Al protein.<br>
Cw-acting: As used herein, the phrase "cis-acting" sequence refers to nucleic acid sequences to which a replicase binds to catalyze the RNA-dependent replication of KNA molecules. These replication events result in the replication of the full-length and partial RNA molecules and, thus, the aJpahvirus subgenomic promoter is also a "cw-acting" sequence. Cw-acting sequences may be located at or near the 5" end, 3" end, or both ends of a nucleic acid molecule, as well as internally.<br>
Fusion: As used herein, the term "fusion" refers to the combination of amino acid sequences of different origin in one polypeptide chain by in-frame combination of their coding nucleotide sequences. The term "fusion" explicitly encompasses internal fusions, i.e., insertion of sequences of different,origin within a polypeptide chain, in addition to fusion to 6m of its tennini-<br>
Hetefologous sequence: As used herein, the term "heterologous sequence" refers to a second nucleotide sequence present in a vector of the invention. The term "heterologous sequence" also refers to any amino acid or RNA sequence encoded by a heterologous DNA sequence contained in a vector of the invention. Heterologous nucleotide sequences can encode proteins or RNA molecules normally expressed in the cell type in which they are present or molecules not normally expressed therein (e.g., Sindbis structural proteins).<br><br>
Isolated: As used herein, when the term "isolated" is used in reference to a molecule, the term means that the molecule has been removed ftom its native environment. For example, a polynucleotide or a polypeptiite naturally present in a living animal is not "isolated," but the same polynucleotide or polypeptide separated from the coexisting materials of its natural state is "isolated" Further, recombinant DNA molecules contained in a vector are considered isolated for the purposes of the present invention. Isolated RNA molecules include in vivo or in vitro RNA replication products of DNA and RNA molecules. Isolated nucleic acid molecules further include synthetically produced molecules. Additionally, vector molecules contained in recombinant host cells are also isolated. Thus, not all "isolated" molecules need be "purified."<br>
Immunotherjeutic; As used herein, the term "immunotherapeutic" is a composition for the treatment of diseases or disorders. More specifically, the term is used to refer to a method of treatment for allergies or a method of treatment for cancer.<br>
Individial: As used herein, the term "individual" refers to multicellular oianisms and includes both plants and animals. Preferred multicellular organisms are animals, more preferred are vertebrates, even more preferred are mammals, and most preferred are humans.<br>
Low or undetectable: As used herein, the phrase "low or undetectable," when used in reference to gene expression level, refers to a level of expression which is either significanfly lower than that seen when the gene is maximally induced (e.g., at least five fold lower) or is not readily detectable by the methods used in the following examples section.<br>
Lectin: As used herein, proteins obtained particularly fnim the seeds of leguminous plants, but also from many other plant and animal sources, that have binding sites for specific mono- or oligosaccharides. Examples include concanavalin A and wheat-germ agglutinin, wlucii are widely used as analytical and preparative agents in the study of glycoprotdn.<br>
Mimotppe: As used herein, the term"niimotope" refers to a substance which induces an immune response to an antigen or antigeruc determinant Generally, the tam mimotope will be used with reference to a particular antigen. For example, a peptide which elicits the production of antibodies to a phospholipase A2 (PLA2) is a mimotope of the antigenic detertoinant to which the antibodies bind. A mimotope may or may not have substantial stmctural similarity to or share structural properties with an antigen or antigenic determinant to which it induces an immune response. Methods for generating and identifying mimotopes which induce immune responses to particular antigens or antigenic determinants are known in the art and are described elsewhere herein.<br><br>
Natural origin: As used herein, the term "natural origin" means that the whole or parts thereof are not synthetic and exist or are produced in nature.<br>
Non-natural; As used herein, the term generally means not from nature, more specifically, the term means fttim the hand of man.<br>
Non-natural origin: As used herein, the tenn "non-natural origin" generally means synthetic or not from nature; more specifically, the term means from the hand of man.<br>
Non-natural molecular scaffold: As used herein, the phrase "non-natural molecular scaffold" refers to any product made by the hand of man that may serve to provide a rid and repetitive array of first attachment sites. Ideally but not necessarily, these first attachment sites are in a geometric order. "Hie non-natural molecular scaffold may be organic or non-organic and may be synthesized chemically or through a biological process, in part or in whole. The non-natural molecular scaffold is comprised of: (a) a core particle, either of natural or non-natural origin; and (b) an otganiiei, which itself comprises at least one first attachment site and is connected to a core particle by at least one covalent bond. In a particular embodiment, the non-natural molecular scaffold may be a virus, virus-like particle, a bacterial pilus, a virus capsid particle, a phage, a recombinant form thereof, or synthetic particle.<br>
Ordered and repetitive antigen or antigenic determinant array. As used heran, the term "ordered and repetitive antigen or antigenic determinant array" generally refers to a repeating pattern of antigen or antigenic determinant, characterized by a uniform spacial arrangement of the antigens or antigenic determinants with respect to the non-natural molecular scaffold. In one embodiment of the invention, the repeating pattern may be a geometric pattem. Examples of smtable ordered and repetitive antigen or antigenic determinant arrays are those which possess strictiy repetitive paracrystalline orders of antigens or antigenic deteiminants with spacings of 5 to 15 nanometers.<br>
Organizer As used herein, the term "organizer" is used to refer to an element bound to a core particle in a non-raridom fashion that provides a nucleation site for creating an ordered and repetitive antigen array. An organizer is any element comprising at least one first attachment site that is bound to a core particle by at leat one covalent bond. An organizer may be a protein, a polypeptide, a peptide, an amino acid (i.e., a residue of a protein, a polypeptide or peptide), a sugar, a polynucleotide, a natural or synthetic polymer, a secondary metabolite or compound (biotin, fluorescein, retinol, digoxigenin, metal ions, phenylmethylsulfonylfluoride), or a combination thereof, or a chemically reactive group thereof. Therefore, the organizer further ensures fonnation of an ordered and repetitive antigen array in<br><br>
accordance with the present invention. In typical embodiments of the invention, the core particle is modified, e.g. by way of genetic engineering or chemical reaction, so as to generate a non-naural molecular scaffold comprising tiie core particle and the oianizer, the latter being connected to the core particle by at least one covalent bond. In certain embodiments of the invention, however, the organizer is selected as being part of the core particle. Therefore, for those embodiments modification of the core particle is not necessarily needed to generate a non-natural molecular scaffold comprising the core particle and the organizer and to ensure the formation of an ordered and repetitive antigen array.<br>
Permissive temperature: As used herein, the phrase "permissive temperature" refers to temperatures at which an enzyme has relatively high levels of catalytic activity.<br>
Pili: As used herein, the term "piU" (singular being "pilus") refers to extracellular structures of bacterial cells composed of protein monomers (e.g., pilin monomers) which are organized into ordered and repetitive patterns. Further, pili are structures which are involved in processes such as the attachment of bacterial cells to host cell surface receptors, inter-cellular genetic exchanges, and cell-cell recognition. Examples of piU include Type-1 pili, P-piU, FlC pili, S-pili, and 987P-pih. Additional examples of pili are set out below.<br>
Pilus-like structure: As used herein, the phrase "pilus-like structure" refers to structures having characteristics similar to tiiat of pili and composed of protein monomers. One example of a "pilus-like structure" is a structure formed by a bactCTiai ceil which expresses modified pilin proteins that do not form ordered and repetitive arrays that are essentially identical to those of natural pih.<br>
Polypeptide; As used herein the term "polypeptide" refers to a polymer composed of amino acid residues, generally natural amino acid residues, linked together throu peptide bonds. Although a polypeptide may not necessarily be limited in size, the tgm polypeptide is often used in conjunction with peptide of a size of aboiit ten to about 50 amino acids.<br>
Protein: As used herein, the term protein refers to a polypeptide generally of a size of above 20, more particularly of above 50 amino acid residues. Proteins generally have a defined three dimensional structure although they do not necessarily need to, and are often referred to as folded, in opposition to peptides and polypeptides which often do not possess a defined three-dimensional structure, but rather can adopt a large number of different conformations, and are referred to as unfolded. The defined three-dimensional structures of proteins is especially important for the association between the core particle and the antigen, mediated by the second<br><br>
attachment site, and in particular by way of chemical cross-linking between the first and second attachment site using a chemical cross-linker. The amino acid linker is also intimately related to the structural properties of proteins in some aspects of the invention.<br>
Purified: As used herein, when the term "purified" is used in reference to a molecule, it means that the concentration of the molecule being purified has been increased relative to molecules associated with it in its natural environment. Naturally associated molecules include proteins, nucleic acids, lipids and sugars but generally do not include water, buffers, and reagents added to maintain the integrity or facilitate the purification of the molecule being purified. For example, even if mRNA is diluted with an aqueous solvent during oiigo dT column chromatography, mRNA molecules are purified by this chromatogrhy if naturally associated nucleic acids and other biological molecules do not bind to the column and are separated from the subject mRNA molecules.<br>
Receptor; As used herein, the term "receptor" refers to proteins or glycoproteins or fragments thereof capable of interacting with another molecule, called die ligand. The ligand may belong to any class of biochemical or chemical compounds. The receptor need not necessarily be a membrane-bound protein. Soluble protein, like e.g., maltose binding protein or retinol binding protein are receptors as well.<br>
Residue: As used herein, the term "residue" is meant to mean a specific amino acid in a polypeptide backbone or side chain.<br>
Recombinant host cell: As used herein, the term "recombinant host cell" refers to a host cell into which one ore more nucleic acid molecules of the invention have been introduced.<br>
Recombinant virus: As used herein, the phrase "recombinant virus" refers to a virus that is netically modified by the hand of man. The phrase covers any virus known in the art More specifically, the phrase refers to a"an alphavirus genetically modified by the hand of man, and most specifically, the phrase refers to a Sinbis virus genetically modified by the hand of man.<br>
Restrictive temperature: As used herein, the phrase "restrictive temperature" refers to temperatures at which an enzyme has low or undetectable levels of catalytic activity. Both "hot" and "cold" sensitive mutants are known and, thus, a restrictive temperature may be higher or lower than a permissive temperature.<br>
RNA-dependent RNA replication event: As used herein, the phrase "RNA-dependent RNA replication event" refers to processes which result in the fonnation of an RNA molecule using an RNA molecule as a template.<br><br>
f	RNA-Dependent RNA polymerase:  AS usea nerem, me phrase "KNA-<br>
Dependent RNA polymerase" refers to a polymerase which catalyzes the production<br>
of an RNA molecule from another RNA molecule.    This tenn is used herein<br>
synonymously with the term "replicase."<br>
RNA-phage: As used herein, the term "RNA-phage" refers to RNA viruses<br>
infecting bacteria, preferably to single-stranded positive-sense RNA viruses infecting<br>
bacteria.<br>
Self antigen : As used herein, the tern "self antigen" refers to proteins encoded<br>
by the host"s DNA and products generated by proteins or RNA encoded by the host"s<br>
DNA are defined as self. In addition, proteins that result from a combination of two or<br>
several sdf-molecules or that represent a fraction of a self-molecule and proteins that<br>
have a high homology two self-molecules as defined above (&gt;95%)may also be<br>
considered self.<br>
Temperature-sensitive: As used herein, the phrase "temperature-sensitive" refers to an enzyme which readily catalyzes a reaction at one temperature but catalyzes the same reaction slowly or not at all at another temperature. An example of a temperature-sensitive enzyme is the replicase protein encoded by the pCYTts vector, which has readily detectable replicase activity at temperatures below 34""C and has low or undetectable tivity at 37"C.<br>
Transcription: As used herein, the term "transcription" refers to the production of RNA molecules from DNA templates catalyzed by RNA polymerase.<br>
Untranslated RNA: As used herein, the phrase "untranslated RNA" refers to an RNA sequence or molecule which does not encode an open reading frame or encodes an open reading frame, or portion thoeof, but in a format in which an amino acid sequence will not be produced (e.g., no initiation codon is present). Examples of such molecules are tRNA molecules, rRNA molecules, and ribozymes.<br>
Vector: As used herein, the term "vector" refers to an agent (s.g., a plasmid or virus) used to transmit genetic material to a host cell. A vector may be composed of eitiierDNAorRNA.<br>
Virus-like particle; As used herein, the term "virus-like particle" refers to a structure resembling a virus particle. Moreover, a virus-like particle in accordance with the invention is non replicative and noninfectious since it lacks all or part of the vira! genome, in particular the replicative and infectious components of the viral genome, A virus-like particle in accordance with the invendoti may contain nucleic acid distinct from their genome.<br><br>
Vims-like particle of a bacteriophage: As used herein, the term "virus-like particle of a bacteriophage" refers to a virus-like particle resembling the stmcture of a bacteriophage, being non replicatjve and noninfectious, and lacking at least the gene or genes encoding for the replication machinery of the bacteriophage, and typically also lacking the gene or genes encoding the protein or proteins responsible for viral attachment to or entry into the host. This definition should, however, also encompass vima-Uke particles of bacteriophages, in which the aforementioned gene or genes are still present but inactive, and, therefore, also leading to non-replicative and noninfectious virus-like particles of a bacteriophage.<br>
Virus particle: The term "virus particle" as used herem refers to die morphological form of a vims. In some vims types it comprises a genome surrounded by a protein capsid; others have additional structures (e.g., envelopes, tails, etc.).<br>
one, a, or an: When the terms "one," "a," or "an" are used in this disclosure, they mean "at least one" or "one or more," unless otherwise indicated.<br>
2.        Compositions  of Ordered  and  Repetitive  Antigen  or  Antigenic Determinant Arrays and Methods to Make the Same<br>
The disclosed invention provides compositions comprising an ordered and repetitive antigen or antigenic determinant array. Furthermore, the invention conveniently enables the practitioner to construct ordered and repetitive antigen or antigenic determinant arrays for various treatment purposes, which includes the prevention of infectious diseases, the treatment of allergies and the treatment of cancers.<br>
Compositions of the invention essentially comprise, or alternatively consist of, two elements: (1) a non-natural molecular scaffold; and (2) an antigen or antigenic determinant with at least, one second attachment site capable" of association through at least one non-peptide bond to said first attachment site.<br>
Compositions of the invention also comprise, or altematively consist of, bacterial pilus proteins to which antigens or antigenic determinants are directiy linked.<br>
The non-natural molecular scaffold comprises, or altematively consists of: (a) a core particle selected from the group consisting of (1) a core particle of non-natural origin and (2) a core particle of natural origin; and (b) an organizer comprising at<br><br>
least one firat attachment site, wherein said organizer is connected to said core particle by at least one covalent bond.<br>
Compositions of the invention also comprise, or alternatively consist of, core particles to which antigens or antigenic detramiiiants are directly linked.<br>
The antigen or antigenic determinant has at least one second attachment site which is selected from the group consisting of (a) an attachment site not naturally occurring with said antigen or antigenic determinant; and (b) an attachment site naturally occurring with said antigen or antigenic determinant<br>
The invention provides for an ordered and repetitive antigen array through an association of the second attachment site to the first attachment site by way of at least one non-peptide bond. "Dius, the antigen or antigenic detenninant and the non-natural molecular scaffold are brought togettier through this association of the first and the second attachment site to form an ordered and r&gt;etitive antigen array.<br>
The practioner may specifically design the antigen OT antigenic determinant and the second attachment site such that the arrangement of all the antigens or antigenic determinants bound to the non-natural molecular scaffold or, in certain eaibodiments, the core particle will be uniform. For example, one may place a single second attachment site on the antigen or antigenic determinant at the carboxyl or amino terminus, thereby ensuring through design that all antigen or antigenic determinant molecules that are attached to the non-natural molecular scaffold are positioned in a uniform way. Thus, the invention provides a convenient means of placing any antigen or antigenic detenninant onto a non-natural molecular scaffold in a defined order and in a maimer which forms a repetitive pattern.<br>
As will be clear to those skilled in the art, certain embodiments of the invention involve the use of recombinant nucleic acid technologies such as cloning, polymerase chain reaction, the purification of DNA and RNA, the expression of recombinant proteins in prokaryotic and eukaryotic ceDs, etc. Such methodologies are well known to those skilled in the art and may be conveniently found in published laboratory methods manuals (e.g., Sambrook,. J. et al., eds.. MOLECULAR CLONING, A LABORATORY MANUAL, 2nd. edition. Cold Spring Harbor Laboratory Press, Cold Spring Ifaibor, N.Y. (1989); Ausubel, F. et at, eds.. CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John H. Wiley &amp; Sons, Inc. (1997)). Fundamental laboratory techniques for working with tissue culture cell lines (Cells, J., ed.. CELL BIOLOGY, Academic Press, 2" edition, (1998)) and antibody-based technologies (Harlow, E. and Lane, D., "Antibodies; A Laboratory Manual," Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. (1988); Deutscher, M.P., "Guide to Protein Purification," Meth. Enzymol. 128, Academic Press San Diego (1990); Scopes, R.K., "Protein Purification Principles and Practice," 3"* ed., Springer-Veriag, New York (1994)) are<br><br>
also adequately described in the literature, all of which are incorporated herein by reference.<br>
A.       Core Particles and Non-Natural Molecular Scaffolds<br>
One element in certain compositions of the invention is a non-natural molecular scaffold comprising, or alternatively consisting of, a core particle and an organizer. As used herein, the phrase "non-natural molecular scaffold" refers to any product made by the hand of man that may serve to provide a rigid and repetitive array of first attachment sites. More specifically, the non-natural molecular scaffold comprises, or alternatively consists of, (a) a core particle selected from the group consisting of (1) a core particle of non-natural origin and (2) a core particle of natural origin; and (b) an organizer comprising at least one first attachment site, wherein said organizer is connected to said core particle by at least one covalent bond<br>
As will be readily apparent to diose skilled in the art, the core particle of the non-natural molecular scaffold of the invention is not limited to any specific form. The core particle may be organic or non-organic and may be synthesized chemically or through a biological process.<br>
In one embodiment, a non-natural core particle may be a synthetic polymer, a lipid micelle or a metal. Such core particles are ioiown in the art, providing a basis from which to build the novel non-natural molecular scaffold of the invention. By â way of example, synthetic polymer or metal core particles are described in U.S. Patent No. 5,770,380, which discloses the use of a calixarene organic scaffold to which is attached a plurality of peptide loops in the creation of an "antibody mimic", and U.S. Patent No. 5,334,394 describes nanocrystalline particles used as a viral decoy that are composed of a wide variety of inorganic materials, including metals or ceramics. Suitable metals include chromium, rubidium, iron, zinc, selenium, nickel, gold, silver, platinum. Suitable ceramic materials in this embodiment include silicon dioxide, titanium dioxide, aluminum oxide, ruthenium oxide and tin oxide. The core particles of this embodiment may be made from organic materials including carbon (diamond). Suitable polymers include polystyrene, nylon and nitrocellulose. Fortius type of nanocrystalline particle, particles made fix)m tin oxide, titanium dioxide or carbon (diamond) are may also be used. A lipid micelle may be prepared by any means known in the art For example micelles may be prepared according to the procedure of Baiselle and Millar (Biophys. Chem. 4:355-361 (1975)) or Corti et al. iChem. Phys. Lipids 55:197-214 (1981)) or Lopez et al {FEES Utt. -25:314-318 (1998)) or Topchieva and Karezin (J. Colloid Interface Sci. 213:29-35 (1999)) or<br><br>
Morein et ol., {Nature 503:457-460 (1984)), which are all incorporated lierein by reference.<br>
TTie core particle may also be produced through a biological process, which may be natural or non-natural. By way of example, this type of embodiment may includes a core particle comprising, or alternatively consisting of, a vims, vims-like particle, a bacterial pUus, a phage, a viral capsid particfe or a recombinant form thereof. In a more specific embodiment, the core particle may comprise, or alternatively consist of, recombinant proteins of Rotavirus, recombinant proteins of Norwalk virus, recombinant proteins of Alphavims, recombinant proteins which form bacterial pili or pilus-Uke structures, recombinant proteins of Foot and Mouth Disease vims, recombinant proteins of Retrovims, recombinant proteins of Hepatitis B virus (e.g., a HBcAg), recombinant proteins of Tobacco mosaic virus, recombinant proteins of Flock House Virus, and recombinant proteins of human Papillomavirus. The core particle may further comprise, or alternatively consist of, one or more fragments of such proteins, as well as variants of such proteins which retain the ability to associate with each other to fona ordered and repetitive antigen or antigenic determinant arrays.<br>
As explained in more below, variants of proteins which retain the ability to associate witti each other to fonn ordered and repetitive antigen or antigenic detemainant arrays may share, for example, at least 80%, 85%, 90%, 95%, 97%, or 99% identity at the amino acid level with their wild-type countMpaits. Using the HBcAg having the amino acid sequence shown in SEQ ID NO:89 for illustration, the invention includes vaccine compositions comprising HBcAg polypeptides comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%, 90%, 95%, 97%, or 99% identical to the amino acid sequence shown in SEQ ID N0:S9, and forms of this protein which have been processed, where appropriate, to remove N-terminal leader sequence. These variants will generally be capable of associating to form dimeric or multimeric stmctures. Methods which can be used to" determine whether proteins form such structures comprise gel filtration, agarose-gel electrophoresis, sucrose gradient centrifugation and electron microscopy (e.g., Koschel, M. et ai, J. Virol 73:2153-2160 (1999)).<br>
Fragments of proteins which retain the ability to associate with each other to form ordered and repetitive antigen or antigenic detenninant arrays may comprise, or altCTnatively consist of, polypeptides which are 15,20,25, 30, 35,40,45, 50, 55,60, 65, 70, 75, 80. 85, 90, 95, 100, 110, 120, 130, 140, 150, 160, 170, 180. 190, or 200 amino acids in length. Examples of such protein fragments include fragments of proteins discussed herein which are suitable for the preparation of core particles and/or non-natural molecular scaffolds."<br><br>
/	Whether natural or non-natural, the core particle of the invention will<br>
generally have an organizer that is attached to the natural or non-natural core particle by at least one covalent bond. The organizer is an element bound to a core particle in a non-random fashion that provides a nucleation site for creating an ordered and repetitive antigen array. Ideally, but not necessarily, the organizer is associated with the core particle in a geometric order. Minimally, the organizer comprises a first attachment site.<br>
In some embodiments of the invention, the ordered and repetitive array is fomied by association between (1) either core particles or non-natural molecular scaffolds and (2) either (a) an antigen or antigenic determinant or (b) one or more antigens or antigenic determinants. For example, bacterial pili or pilus-like structures are fonned from proteins which are organized into ordered and repetitive structures. Thus, in many instances, it will be possible to form ordered arrays of antigens or antigenic determinants by linking these constituents to bacterial pili or pilus-like structures either directly or through an organizer.<br>
As previously stated, the organizer may be any element comprising at least one first attachment site that is bound to a core particle by at least one covalent bond. An organizer may be a protein, a polypeptide, a peptide, an amino acid (i.e., a residue of a protein, a polypeptide or peptide), a sugar, a polynucleotide, a natural or synthetic polymer, a secondary metabolite or compound OiotJn, fluorescein, retinol, digoxigenin, metal ions, phenylmethylsulfonylfluoride), or a combination thereof, or a chemically reactive group thereof. Iti a more specific embodiment, the organizer may comprise a first attachment site comprising an antigen, an antibody or antibody fragment, biotin, avidin, strepavidin, a receptor, a receptor ligand, a Ugand, a ligand-ttniBng protean, an inter;ting leucine apper polypeptide, an amino group, a chemical group reactive to an amino group; a carboxyl group, chemical group reactive to a carboxyl group, a sulfliydryl group, a chemical group reactive to a sulfhydryl group, or a combination thereof.<br>
In one embodiment, the core particle of the non-natural molecular scaffold comprises a virus, a bacterial pilus, a stmcture fonned finm bacterial pilin, a bacteriophage, a_ virus-like particle, a viral capsid particle or a recombinant form thereof. Any vims known in the art having an ordered and repetitive coat and/or core protein structure may be selected as a non-natural molecular scaffold of the invention; examples of suitable viruses include sindbis and other alpfaaviruses, riiabdoviruses (e.g. vesicular stomatitis virus), picomaviruses (e.g., hixman rliino virus, Aichi virus), togaviruses (e.g., rubella virus), orthomyxoviruses (e.g., Thogoto virus, Balken virus, . fowl plague virus), polyomaviruses {e.g., polyomavirus BK, polyomavirus JC, avian polyomavirus BEDV), parvoviruses, rotaviruses, bacteriophage QP, bacteriophage R17, bacteriophage Mil, bacteriophage MXl, bacteriophage NL95, bacteriophage fr.<br><br>
bacteriophage GA, bacteriophage SP, bacteriophage MS2, bacteriophage f2, bacteriophage PP7, Nonvalk virus, foot and mouth disease virus, a retrovirus. Hepatitis B virus. Tobacco mosaic virus, Flock House Virus, and human Papilomavirus (for example, see Table 1 in Bachman, M.F. and Zinkemagel, R.M., Immunol. Today i7:553-558 (1996)).<br><br> one embodiment, the invention utilizes genetic engineering of a virus to create a fusion between an ordered and repetitive viral envelope protein and an organizer comprising a heterologous protein, peptide, antigenic determinant or a reactive amino acid residue of choice. Other genetic manipulations known to those in the art may be included in the construction of the non-natural molecular scaffold; for example, it may be desirle to restrict the replication abUity of the recombinant virus through genetic mutation. The viral protein selected for fusion to the organizer (i.e., first attachment site) protein should have an organized and repetitive structure. Such an organized and repetitive structure include paracrystalline organizations with a spring of 5-15 nm on the surface of the virus. The creatjon of this type of fusion protein will result in multiple, ordered and repetitive organizers on the surface of the virus. Thus, the ordered and repetitive organization of the first attachment sites resulting therefiitim will reflect the normal organization of the native viral protein.<br>
As will be discussed in more detail herein, in another embodiment of the invention, the non-natural molecular scaffold is a recombinant alphavirus, and more specifically, a recombinant Sinbis virus. Alphaviruses are positive stranded RNA viruses that replicate their genomic RNA entirely in the cytoplasm of the infected cell and witiiout a DNA intennediate (Strauss, J. and Strauss, E., Microbiol. Rev. 55:491-562 (1994)). Several members of the alphavirus family, Sindbis (Xiong, C. et al.. Science243:\\2,i"ll9\ (1989); Schlesinger, S., TrendsBiotechnol 77:18-22 (1993)), SemEM Forest Virus (SFV) (Liijestrem, P. &amp; Garoff, H., Bio/Technology 9:1356-1361 (1991)) and others (Davis, N.L. et al.. Virology 777:189-204 (1989)), have received considerable attention for use as virus-based expression vectors for a variety of different protgns (Lundstrom, K., Curr. Opin. Biotechnol. S:578-582 (1997); liijestrom. P., .Curr". Opin. Biotechnol. 5:495-500 (1994Â»"and as candidates for vaccine development Recently, a number of patents have issued directed to the use of alphaviruses for the expression of hetwologous proteins and the development of vaccines {see U.S. Patent Nos. 5,766,602; 5,792,462; 5,739,026; 5.789,245 and 5,814,482). Te construction of the aiphaviral scaffold of the invention may be done by means generally known in the art of recombinant DNA technology, as described by the aforementioned articles, which are incorporated herein by reference.<br>
A variety of diffwent recombinant host cells can be utilized to produce a viral-based core particle for antigen or antigenic determinant attachment.   For example,<br><br>
Aipnavmises are Known to have a wide host range; Sindbis virus infects cultured niainmalian, reptilian, and amphibian cells, as well as some insect cells (Clark, H., J. Natl. Cancer Inst. 57:645 (1973); Uake, C, /. Gen. Virol. 35:3Z5 (1977); StoUar, V. in THE TOGAVIRUSES, R.W. Schlesinger, Ed., Academic Press, (1980), pp.583"62I). Thus, numerous recombinant host cells can be used in the practice of the invention. BHK, COS, Vero, HeLa and CHO cells are particularly suitable for the production of heterologous proteins because they have the potential to glycosylate heterologous proteins in a manner similar to human cells (Watson, E. et al., Glycobiology 4:221, (1994)) and can be selected (Zang, M. et al., Bio/Technology ]3:389 (1995)) or genetically engineered (Renner W. et al., Biotech. Bioeng. 4:416 (1995); Lee K. et al. Biotech. Bioeng. 50:336 (1996)) to grow in serum-free medium, as well as in suspension.<br>
Introduction of the polynucleotide vectors into host cells can be effected by methods described in standard laboratory manuals {see, e.g., Sambrook, J. et al, eds.. MOLECULAR CLONING, A LABORATORY MANUAL, 2nd. edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989). Chapter 9; Ausubel, E et al, eds., CuERHT PROTOCOLS IN MOLECULAR BIOLOGY, John H. Wiley &amp; Sons, Inc. (1997), Chapter 16), including methods such as electroporation, DEAE-dextran mediated transfection, transfection, microinjection, cationic lipid-mediated transfection, transduction, scrape loading, ballistic introduction, and infection. Methods for the introduction of exogenous DNA sequences into host cells are discussed in Feigner, P. et al., U.S. Patent No. 5,580,859.<br>
Packaged RNA sequences can also be used to infect host cells. These packaged RNA sequences can be introduced to host cells by adding them to the culture meiMum. For example, tiie preparation of non-infective alpahviial paiticles is described in a number of sources, including "Sindbis Expression System", Version C ilnvitrogen Catalog No, K750-1).<br>
When mammalian cells are used as recombinant host cells for the production of viral-based core particles, these cells will generally be grown in tissue culture. Methods for growing cells iri culture are well known in the art {see, e.g., Celis, J., ed.. CELL BIOLOGY, Academic Press, 2Â° edition, (1998); Sambrook, J. et al., eds.. MOLECULAR CLONiNa, A LABORATORY MANUAL, 2nd. edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989); Ausubel, F. et al., eds., CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John H. Wiley &amp; Sons, Lie. (1997); Freshney, R., CULTURE OF ANIMAL CELLS, Alan R. Uss, Inc. (1983)).<br>
As will be understood by those in the art, the first attachment site may be or be a part of any suitable protain, polypeptide, sugar, polynucleotide, peptide (amino acid), natural or synthetic polymer, a secondary metabolite or combination thereof<br><br>
that may serve to specifically attach the antigen or antigenic determinant of choice to the non-natural molecular scaffold. In one embodiment, the attachment site is a protein or peptide that may be selected from those known in the art For example, the first attachment site may selected from the following group: a Hgand, a receptor, a lectin, avidin, streptavidin, biotin, an epitope such as an HA or T7 tag, Myc, Max, immunoglobulin domains and any other amino acid sequence known in the art that would be usefiil as a first attachment site.<br>
It should be further understood by those in the art that with another embodiment of the invention, the first attachment site may be created secondarily to the orgamzer (Le., protein or polypeptide) utilized in constructing the iu-frame fusion to the capsid protein. For example, a protein may be utilized for fusion to the envelope protein with an amino add sequence known to be glycosylated in a specific fashion, and the sugar moiety added as a result may then serve at the first attachment site of the viral scaffold by way of binding to a lectin swng as the secondary attachment site of an antigen. Alternatively, the oiganizw sequence may be biotinylated in vivo and the biotin moiety may serve as the first attachment site of the invention, or the organizer sequence may be subjected to chemical modification of distinct amino acid residues in vitro, the modification serving as the first attachment te.<br>
In another embodiment of the invention, the first attachment site is selected to be the JUN-FOS leucine zipper protein domain that is fused in fime to the Hepatitis B csid (core) protein (HBcAg). However, it will be clear to all individuals in the art that other viral capsid proteins may be utilized in the fusion protein constmct for locating the first attachment site in the non-natural molecular scaffold of the invention.<br>
Bi another embodiment of the invention, the first attachment site is selected to be a lysine or cysteine residue tiiat is fused in frame to the HBcAg. However, it will be clear to all individuals in the art that other viral capsid or virus-like particles may be utilized in the fusion protein construet. for locating the first attachment in the non-natinral roolKiular scaffold of the invention.<br>
The JUN amino acid sequence utilized for the first attachment site is the following:<br>
CGGIUARIJEEKVKTLKAQNSELASTANMLREQVAQLKQKVMNHVGC<br>
(SEQ ID NO:59)<br>
In this instance, the anticipated second attachment site on the antigen would<br>
be the FOS leucine zipper protein domain and the amino acid sequence would<br>
be the following:<br><br>
CGGLTDTLQAETDQVEDEKSALQTEIANIJJCBKEKLEFILAAHGGC (SEQ ID NO:60)<br>
These sequences are derived from the transcription factors JVN and FOS, each flanked with a short sequence containing a cysteine residue on both sides. TTiese sequences are known to interact with each other. The oiigina] hypothetical structure proposed for the JUN-FOS dimer assumed that the hydrophobic side chains of one monomer interdigjtate with the respective ade chmns of the other monomer in a zippra--like manner (Landschulz et al, Science 24(?;1759-1764 (1988)). However, this hypothesis proved to be wrong, and these proteins are known to form an a-helical coiied coil (O"Shea et al., Science 2-3:538-542 (1989); O"Shea et al., Cell (5S:699-708 (1992); Cohen &amp; Parry, Trends Biochem. Set. 7i:245-248 (1986)). Thus, the term "leucine zipper" is frequently used to refer to these protein domains for more historical than stmctural reasons. Throughout this patent, the term "leucine zipper" is \Ked to refer to the sequences depicted above or sequences essentially similar to the ones depicted above. Tlie terms JUN and FOS are used for the respective leucine zipper domains rathH" than the entire JUN and FOS proteins.<br>
As previously stated, the invention includes viral-based core particles which comprise, or alternatively consist of, a virus, virus-like particle, a phage, a viral csid particle or a recombinant form thereof. Skilled artisans have the knowledge to produce such core particle and attach organizers thereto. The production of Hepatitis B virus-like particles and measles viral capsid particles as core particles is disclosed in Examples 17 to 22 of WO 00/32227, which is explicitly incorporated by reference. In such embodiments, the JUN leucine zipper protein domain or FOS leucine zipper protMn domain may be used as an organizer, and hence as a first attachment site, for the non-natural molecular scaffold of the invention.<br>
Examples 23-29 provide details of the production of Ifcpatitis B core particles<br>
carrying an in-frame fiised peptide with a reactive lysine residue and antigens<br>
carrying a genetically fused cysteine residue, as first and second attachmMt site.<br>
respectively.	â         .<br>
1	In other embodiments, the core particles used in compositions of the<br>
invention are composed of a Ifcpatitis B capsid (core) protein (HBcAg), a franent of a HBcAg, or other protein or peptide which can form ordered arrays, which have been modified to either eliminate or reduce the number of free cysteine residues. Zhou et al. {J. Virol. 66:5393-5398 (1992)) demonstrated that HBcAgs which have been modified to remove the naturally resident cysteine residues retain the ability to associate and form multimeric structures. Thus, core particles suitable for use in compositions of the invention include those comprising modified HBcAgs, or<br><br>
fragments thereof, in which one or more of the naturally resident cysteine residues have been either deleted or substituted with another amino add residue (e.g., a serine residue).<br>
2	The HBcAg is a protein generated by the processing of a Hepatitis B<br>
core antigen precursor protein. A number of isotypes of the HBcAg have been identified. For example, the HBcAg protein having the amino acid sequence shown in SEQ ID NO:132 is 183 amino acids in length and is generated by the processing of a 212 amino acid Hepatitis B core antigen precmor protein. This processing results in the removal of 29 amino acids from the N-terminus of the Hepatitis B core antigen precursor protein. Similarly, the HBcAg protein having the amino acid sequence shown in SEQ ID NO:134 is 185 amino acids in length and is generated by the processing of a 214 amino acid Hepatitis B core antigen precursor protein. The amino acid sequence shown in SEQ ID NO: 134, as compared to tiie amino acid sequence shown in SEQ ID NO: 132, contains a two amino acid insert at positions 152 andl53inSEQIDNO:134.<br>
In most instances, vaccine compositions of the invention will be prepared using the processed form of a HBcAg (i.e., a HBcAg from which the N-tenninal leader sequence (e.g., the first 29 amino acid residues shown in SEQ ID NO:134) of the Hepatitis B core antigen precursor protein have been removed).<br>
Further, when HBcAgs are produced under conditions where processing will not occur, the HBcAgs will generally be expressed in "processed" form. For example, bacterial systems, such as E. coli, generally do not remove the leader sequences, also referred to as "signal peptides," of proteins which are normally expressed in eukaryotic cells. Thus, when an E. coli expression system is used to produce HBcAgs of the invention, these proteins will generally be exprsed such that the N-terminal leader sequence of the Hepatitis B core antigen precursor protein is not present.<br>
In one embodiment of the inventjon, a "tnodified HBcAg comprising the amino, acid sequence shown in SEQ ID NO: 134, or subportjon thereof, is used to prepare non-natural molecular scaffolds, hi particular, modified HB6Ags suitable for use in the practice of the invention include proteins m which one or more of the cysteine residues at positions corresponding to positions 48, 61, 107 and 185 of a protein having the amino acid sequence shown in SEQ ID NO:134 have been either deleted or substituted with other amino acid residues (e.g., a serine residue). As one skilled in the art would recognize, cysteine residues at similar locations in HBcAg variants having amino acids sequences which differ from that shown in SEQ ID NO: 134 could also be deleted or substituted with<br><br>
other amino acid residues. The modified HBcAg variants can then be used to prepare vaccine compositions of the invention.<br>
The present invention also includes HBcAg variants which have been modified to delete or substitute one or more additional cysteine residues which are not found in polypeptides having the amino acid sequence shown in SEQ ID NO:134. Examples of such HBcAg variants have the amino acid sequences shown in SEQ ID NOs:90 and 132. These variant contain cysteines residues at a position corresponding to amino acid residue 147 in SEQ ID NO:134. Thus, the vacdne compositJOTis of the invention include compositions comimsing HBcAgs in which cysteine residues not present in the amino acid sequence shown in SEQ ID NO:134 have been deleted.<br>
Under certain circumstances {e.g., when a heterobifunctional cross-linking reagent is used to attach antigens or antigenic determinants to the non-natural molecular scaffold), the presence of free cysteine residues in the HBcAg is believed to lead to covalent coupling of toxic components to core particles, as well as the cross-linjdng of monomers to foim undefined species.<br>
Further, in niany instances, these toxic components may not be detectable with assays performed on compositions of fee invention. This is so because covalent coupling of toxic components to the non-natural molecular scaffold would result in the formation of a population of diverse species in which toxic components are linked to different cysteine residues, or in some cases no cysteine residues, of the HBcAgs. In other words, each free cysteine residue of each HBcAg will not be covalently linked to toxic components. Further, in many instances, none of the cysteine residues of particular HBcAgs will be linked to toxic components. TTius, the presence of these toxic conwnents may be difficult to detect because they would be present in a mixed population of molecules. Hie administration to an individual of HBcAg species containing toxic components, however, could lead to a potentially serious adverse reaction.<br>
:,	It is well known in the art that free cysteine residues can be involved in<br>
a numbw of chemical sitie reactions. These side reactions include disulfide exchanges, reaction with chemical substances or metabolites that are, for example, injected or formed in a combination therapy with other substances, or direct oxidation and reaction with nucleotides upon exposure to UV light Toxic adducts could thus be generated, especially considering the fact that HBcAgs have a strong tendency to bind nucleic acids. Detection of such toxic products in antigen-capsid conjugates would be difficult using capsids prepared using HBcAgs containing free cysteines and heterobifunctional cross-linkers, since a distribution of products with a broad range of molecular weight would be<br><br>
generated. The toxic adducts would thus be distributed between a multiplicity of species, which individually niay each be present at low concentration, but reach toxic levels when together.<br>
bi view of the above, one advantage to the use of HBcAgs in vaccine compositions which have been modified to remove naturally resident cysteine residuM is that sites to which toxic species can bind when antigens or antigenic determinants are attached to the non-natural molecular scaffold would be reduced in number or eliminated altogefeer. Further, a high concentration of cross-linko: can be used to produce highly decorated particles without the drawback of generating a plurality of undefined cross-linked species of HBcAg monomers (i.e., a diverse mixture of cross-linked monomeric HbcAgs).<br>
A number of naturally occumng HBcAg variants suitable for use in the practice of the present invention have been identified. Yuan et cU., (J. Virol. Z?;10122-10128 (1999)), for example, describe variants in which the isoleucine residue at position corresponding to position 97 in SEQ ID NO:134 is replaced with either a leucine residue or a phenylalanine residue. The amino acid sequences of a number of HBcAg variants, as well as several Hepatitis B core antigen precursor variants, are disclosed in GenBank reports AAF121240 (SEQ ID NO:89), AF121239 (SEQ ID NO:90), X85297 (SEQ ID NO:91), X02496 (SEQ ID NO:92), X85305 (SEQ ID NO:93), X85303 (SEQ ID NO:94), AF151735 (SEQ ID NO:95), X85259 (SEQ ID NO:96), X85286 (SEQ ID NO:97), X85260 (SEQ ID NO:98), X85317 (SEQ ID NO:99), X85298 (SEQ ID NO; 100), AF043593 (SEQ ID NO: 101), M20706 (SEQ ID NO: 102), X85295 (SEQ ID NO; 103), X80925 (SEQ ID NO: 104), X85284 (SEQ ID NO: 105), X85275 (SEQ ID NO:106), X72702 (SEQ ID NO;107), X85291 (SEQ ID NO;108), X65258 (SEQ ID N0:109), X85302 (SEQ ID NO:110), M32138 (SEQ ID N0:111), X85293 (SEQ ID N0:112), X85315 (SEQ Â© NO:113), U95551 (SEQ ID N0:114), X85256 (SEQ ID N0:n5), X85316 (SEQ ID N0:116), X85296 (SEQ ID N0:117), AB033559 (SEQ ID" 110:118), X59795 (SEQ ID N0;119), X85299 (SEQ ED NOfl20&gt;. X85367 (SEQ ID N0;121), X65257 (SEQ ID N0:122). X85311 (SEQ ID NO:123), X85301 (SEQ ID N0:124), X85314 (SEQ ID NO;X25), X85287 (SEQ ID NO:126). X85272 (SEQ ID NO:127), X85319 (SEQ ID NO: 128), AB010289 (SEQ ID NO: 129), X85285 (SEQ ID NO;130), AB010289 (SEQ ID N0:131), AF121242 (SEQ ID NO;132), M9052p (SEQ ID NO:135), P03153 (SEQ ID NO:136), AF110999 (SEQ ID NO:137), and M95589 (SEQ ID NO:138), the disclosures of each of which are incorporated herein by reference. These HBcAg variants differ in amino acid sequence at a number of positions, including amino acid residues which corresponds to the<br><br>
ammo acid resiaues located at positions 12, 13, 21, 22, 24, 29, 32, 33, 35, 38, 40, 42, 44, 45, 49, 51, 57. 58, 59, 64, 66, 67, 69, 74, 77, 80, 81, 87, 92, 93, 97, 98, 100, 103, 105, 106, 109, 113, 116, 121, 126, 130, 133, 135, 141, 147, 149, 157, 176.178,182 and 183 in SEQ ID NO:134.<br>
HBcAgs suitable for use in the present invention nay be derived from any organism so long as they are able to associate to form an ordered and repetitive antigen array.<br>
As noted above, generally processed HBcAgs (i.e., those which lack leader sequences) will be used in the vaccine compositions of the invention. Thus, when HBcAgs having amino acid sequence shown in SEQ ID NOs;136, 137, or 138 are used to prepare vaccine compositions of the invention, generally 30,35-43, cff 35-43 amino add residues at the N-terminus, respectively, of each of these proteins wiU be omitted.<br>
The present invention includes vaccine compositions, as well as methods for using these conqwsitions, which employ the above described variant HBcAgs for the preparation of non-natural molecular scaffolds.<br>
Farther included withing the scope of the invention are additional HBcAg variants which are capable of associating to form diraraic or multimeric structures. Thus, the invention further includes vaccine compositions comprising HBcAg polypeptides comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%, 90%, 95%, 97%. or 99% identical to any of the amino acid sequences shown in SEQ ID NOs;89-132 and 134-138, and forms of these proteins which have been processed, where appropriate, to remove the N-terminal leader sequence.<br>
Whether the amino acid sequence of a polyptide h an amino acid sequence that is at least 80%, 85%, 90%, 95%, 97%, or 99% identical to one of the amino acid sequences shown in SEQ ID NOs:89-I32 and 134-138, or a subportion thereof, can be detennined conventionally using known computer programs such the Bestfit program. When using Bfestfit or any other sequence â alignment program to determine whether a particular sequence is, for instance, 95% identical to a reference amino acid sequence according to the present invention, the parameters are set such that the percentage of identity is calculated over the full length of the reference amino acid sequence and liiat gs in homology of up to 5% of the total number of amino acid residues in the reference sequence are allowed.<br>
The HBcAg variants and precursors having the amino acid sequences set out in SEQ ID NOs:89-132 and 134-136 arc relatively similar to each other. Thus, reference to an amino acid residue of a HBcAg variant located at a position<br><br>
which corresponds to a particular position in SEQ ID NO:134, refers to the amino acid residue which is present at that position in the amino add sequence shown in SEQ ID NO: 134. The homology between these HBcAg variants is for the most part high enough among Hepatitis B viruses that infect mammals so that one skilled in the art would have little difficulty reviewing both the amino acid sequence shown in SEQ ID NO:134 and that of a particular HBcAg variant and identifying "corresponding" amino acid residues. For exanjle, the HBcAg amino acid sequence shown in SEQ ID NO:135, which shows the amino acid sequence of a HBcAg derived from a virus which infect woodchucks, has enough homology to the HBcAg having the amino acid sequence shown in SEQ ID NO: 134 that it is readily apparent that a three amino acid residue insert is present in SEQ ID NO:135 between amino acid residues 155 and 156 of SEQ ID NO: 134.<br>
The HBcAgs of Hepatitis B viruses which infect snow geese and ducks<br>
differ enough from the amino acid sequences of HBcAgs of Hepatitis B viruses<br>
which infect mammals that aUgnment of these forms of this protein with the<br>
amino acid sequence shown in SEQ ID NO: 134 is difficult However, the<br>
invention includes vaccine compositions which comprise HBcAg variants of<br>
Hepatitis B viruses which infect birds, as wells as vaccine compositions which<br>
comprise fragments of these HBcAg variants. HBcAg fragmraits suitable for use<br>
in preparing vaccine compositions of tiie invention include compositions which<br>
contain polypeptide fragments comprising, or alternatively consisting of, amino<br>
acid residues selected from the group consisting of 36-240, 36-269, 44-240,<br>
44-269, 36-305, and 44-305 of SEQ ID NO:137 or 36-240, 36-269, 44-240,<br>
44-269. 36-305, and 44-305 of SEQ ID NO: 138. As one skilled in the art would<br>
recognize, one, two, three or more of the cysteine residues naturally present in<br>
these polypeptides (e.g., the cysteine residues at position 153 is SEQ ID NO:137<br>
or positions 34,43, and 196 in SEQ ID NO:138) could be either substituted with<br>
anothCT amino acid residue or deleted prior to their inclusion in vaccine<br>
compositions of the invention.	.     <br>
In one embodiment, the cysteine residues at positions 48 and 107 of a protein having the amino acid sequence shown in SEQ ID NO: 134 are deleted or substituted with another amino acid residue but the cysteme at position 61 is left in place. Further, the modified polypeptide is then used to prepare vaccine compositions of the invention.<br>
As set out below in Example 31, the cysteine residues at positions 48 and 107, which are accessible to solvent, may be removed, for example, by site-directed mutagenesis. Further, the inventors have found that the Cys-48-Ser,<br><br>
Cys-107-Ser HBcAg double mutant constructed as described in Example 31 can be expressed in E. coli.<br>
As discussed above, the elimination of free cysteine residues reduces the number of sites where toxic components can bind to the HBcAg, and also elioMiates sites where cross-linking of lysine and cysteine residues of the same or of neighboring HBcAg molecules can occur. The cysteine at position 61, which is involved in dimer formation and fonns a disulfide bridge willi the cysteine at position 61 of another HBcAg, will normally be left int:t for stabilization of HBcAg dimers and multimers of the invention.<br>
As shown in Example 32, cross-linking experiments peEformed with (1) HBcAgs containing free cysteine residues and (2) HBcAgs whose free cysteine residues have been made uxuitactive with iodacetamide, indicate that free cysteine residues of the HBcAg are responsible for cross-linking between HBcAgs through reactions between heterobifunctional cross-linker derivatized lysine side chains, and free cysteine residues. Example 32 also indicates that cross-linking of HBcAg subunits leads to the formation of high molecular weight species of undefined size which cannot be resolved by SDS-pclyacrylamide gel electrophoresis.<br>
When an antigen or antigenic detenninant is linked to the non-natural molecular scaffold through a lysine residue, it may be advantageous to either substitute or delete one or both of the naturally resident lysine residues located at positions couesponding to positions 7 and 96 in SEQ ID NO:134, as well as other lysine residues present in HBcAg variante. The elimination of these lysine residues results in the removal of binding sites for antigens or antigenic determinants which could disrupt the ordered array and should improve the quality and uniformity of the final vaccine composition.<br>
In many instances, when both of the naturally resident lysine residues at positions corresponding to positions 7 and 96 in SEQ ID NO: 134 are eliminated, another lysine will be introduced into the HBcAg as air attachment site for an antigen or antigenic detenninant. Methods for inserting such a lysine residue are set out, for example, in Example 23 below. It will often be advantageous to introduce a lysine residue into the HBcAg when, for example, both of the naturally resident lysine residues at positions corresponding to positions 7 and 96 in SEQ ID NO:134 are altered and one seeks to attach the antigen or antigenic determinant to the non-natural molecular scaffold using a heterobifunctional cross-linking agent<br>
The C-terminus of the HBcAg has been shown to direct nuclear localization of this protein.    (Eckhardt et al., J. Virol 65:575-582 (1991).)<br><br>
Furflier, this region of the protein is also believed to confer upon the HBcAg the ability to bind nucleic acids.<br>
In some embodiments, vaccine compositions of the invention will contain HBcAgs which have nucleic acid binding activity {e.g., which contain a naturally resident HBcAg nucleic acid binding domain). HBcAgs containing one or more nucleic acid binding domains are useful for preparing vaccine compositions which exhibit enhanced T-cell stimulatory activity. Thus, the vaccine compositions of the invention include compositions which contain HBcAgs having nucleic acid binding activity. Further inclu{ted are vaccine compositions, as well as the use of such compositions in vaccination protocols, where HBcA are bound to nucleic acids. These HBcAgs may bind to the nucleic acids prior to administration to an individual or may bind the nucleic acids after administration.<br>
In other embodiments, vaccine compositions of the invention will contain HBcAgs from which the C-terminal region (e.g., amino acid residues 145-185 or 150-185 of SEQ ID NO:134) has been removed and do not bind nucleic acids. Thus, additional modified HBcAgs suitable for use in the practice of the present invention include C-terminal tmncation mutants. Suitable truncation mutants include HBcAgs where 1,5, 10, 15, 20, 25, 30, 34, 35, 36, 37, 38,39 40,41,42 or 48 amino acids have been removed from the C-terminus.<br>
HBcAgs suitable for use in the practice of the present invention also include N-tenninal truncation mutants. Suitable truncation mutants include modified HBcAgs where 1, 2,5,7,9, 10, 12,14, 15, or 17 amino acids have been removed from the N-teraiinus.<br>
Further HBcAgs suitable for use in the practice of the present invention include N- and C-terminal truncation mutants. Suitable truncation mutants include HBcAgs where 1, 2, 5, 7, 9, 10, 12, 14, 15, and 17 amino acids have been removed from the N-terminus and 1, 5, 10, 15, 20, 25, 30, 34, 35, 36, 37, 38, 39 40,41,42 or 48 amino acids havg been removed from the C-tenninus.<br>
The invention further includes vaccine compositions comprising HBcAg polypeptides. comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%, 90%, 95%, 97%, or 99% identical to the above described truncation mutants.<br>
As discussed above, in certain embodiments of the invention, a lysine residue is introduced as a first attachment site into a polypeptide which fonns the non-natural molecular scaffold. In preferred embodiments, vaccine compositions of the invention are prepared using a HBcAg comprising, or alternatively consisting of, amino acids 1-144 or amino acids 1-149 of SEQ ID NO:134 which<br><br>
is modified so that the amino acids corresponding to positions 79 and 80 are replaced with a peptide ha\ing the amino acid sequence of Gly-Gly-Lys-Gly-Gly (SEQ ID NO:158) and the cysteine residues at positions 48 and 107 are either deleted or substituted with another amino acid residue, while the cysteine at position 61 is left in place. The invention further includes vaccine compositions comprising coiresponing fragments of polypeptides having anuno acid sequences shown in any of SEQ ID NOs;89-132 and 135-136 which also have the above noted amino acid alterations.<br>
The invention further includes vaccine compositions comprising ftagments of a HBcAg comprising, or alternatively consisting of, an amino Kad sequence othra" than thai shown in SEQ ID NO: 134 from which a cysteine residue not present at corresponding location in SEQ ID NO: 134 has been deleted. One example of such a fragment would be a polypeptide comprising, or alternatively consisting of, amino acids amino acids 1-149 of SEQ DD NO:132 where the cysteine residue at position 147 has been either substituted with another amino acid residue or deleted.<br>
The invention further includes vaccine compositions comprising HBcAg polypeptides comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%, 90%, 95%, 97%. or 99% identical to amino acids 1-144 or 1-149 of SEQ ID NO: 134 and corresponding subportions of a polypeptide comprising an amino acid sequence shown in any of SEQ ID NOs:89-132 or 134-136, as well as to amino acids 1-147 or 1-152 of SEQ ID N0:158.<br>
The invention also includes vaccine compositions comprising HBcAg polypeptides comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%, 90%, 95%, 97%, or 99% identical to amino acids 36-240, 36-269, 44-240, 44-269, 36-305, and 44-305 of SEQ ID NO: 137 or 36-240,36-269,44-240,44-269,36-305, and 44-305 of SEQ ID NO:138.<br>
Vaccine compositions of the invention -may -comprise mixtures of different HBcAgs. Thus, these vaccine compositions may be composed of HBcAgs which differ in amino acid sequence. For example, vaccine compositions could be prepared comprising a "wild-type" HBcAg and a modified HBcAg in which one or more amino acid residues have been altered {e.g., deleted, inserted or substituted), hi most applications, however, only one type of a HBcAg, or at least HBcAgs having essentially equivalent first attachment sites, will be used because vaccines prepared using such HBcAgs will present highly ordered and reptive arrays of antigens or antigenic determinants.   Further,<br><br>
preferred vaccine compositions of the invention are those which present highly ordered and repetitive antigen array<br>
The invention further includes vaccine compositions where the non-natural molecular scaffold is prepared using a HBcAg fused to another protein. As discussed above, one example of such a fusion protein is a BBcAg/FOS fusion. Other examples of HBcAg ftision proteins suitable for use in vaccine compositions of the invention include fusion proteins where an amino acid sequence has been added which aids in the formation and/or stabilization of HBcAg dimers and multimers. This additional amino acid sequence may be fused to either the N- or C-traminus of the HBcAg. One example, of such a fusion protein is a fusion of a HBcAg with the GCN4 helix region of Saccharomyces cerevisiae (C5enBank Accession No. P03069 (SEQ ID NO:154)).<br>
The helix domain of the GCN4 protein forms homodimers via non-covalent interactions which can be used to prepare and stabilize HBcAg dimers and nmltimers.<br>
In one embodiment, the invention provides vaccine compositions prepared using HBcAg fusions proteins comprising a HBcAg, or fragment thereof, with a GCN4 polypeptide having the sequence of amino acid residues 227 to 276 in SEQ ID NO: 154 fused to the C-terminus. ITiis GCN4 polypeptide may also be fused to the N-terminus of the HbcAg.<br>
HBcAg/src homology 3 (SH3) domain fusion proteins could also be used to prepare vaccine compositions of the invention. SH3 domains are relatively small domains found in a number of proteins which confer the ility to interact with specific proline-rich sequences in protein binding partners {see McPhereon, Cell Signal ii:229-238 (1999). HBcAg/SH3 fusion proteins could be used in several ways. First, the SH3 domain could form a first attachment site which interacts with a second attachment site of the antigen or antigenic determinant. Similarly, a proline rich amino acid sequence could be added to the HBcAg and used as a first attachment site for an SH3 domain second attachment site of an antigen or antigenic deteiminaiit. Second, the SHJ domain could associate with prohne rich regions introduced into HBcAgs. Thus, SH3 domains and proline rich SH3 interaction sites could be inserted into either the same or different HBcAgs and used to form and stabilized dimers and multimers of the invention,<br>
hi other embodiments, a bacterial pilin, a subportion of a bacterial pilin, or a fusion protein which contains either a bacterial pilin or subportion thereof is used to prepare vaccine compositions of the invention. Examples of pilin  proteins  include  pilins   produced  by  Escherichia   coli,   Haemophilus<br><br>
influenzae. Neisseria meningitidis. Neisseria gonorrhoeae, Caulobacter crescenius, Pseudomonas stutzeri, and Pseudomoruxs aeruginosa. The amino add sequences of pilin proteins suitable for use with the present invention include those set out in GenBank reports AJ000636 (SEQ ID NO: 139), AJ132364 (SEQ ID N0:14fl), AF229646 (SEQ ID NO.Hl), AF0518I4 (SEQ ID NO:142), AF051815 (SEQ ID NO:143), and X0O981 (SEQ ID NO:155), the entire disclosures of which are incorporated herein by reference.<br>
Bacteria] pilin proteins are generally processed to remove N-terminal leader sequences prior to export of the proteins into the bacterial periplasm. Further, as one skilled in the art would recognize, bacterial pilin proteins used to prepare vaccine compositions of the invention will generally not have the naturally present leader sequence.<br>
One specific example of a pilin protein suitable for use in the present invention is the P-pilin of E. coli (GenBank report AF237482 (SEQ ID NO:144)). An example of a T&gt;pe-1 E. coli pilin suitable for use with the invention is a pilin having the aniino acid sequence set out in GenBank report P04128 (SEQ ID NO: 146), which is encoded by nucleic acid having the nucleotide sequence set out in GenBank report M27603 (SEQ ID NO:145). The entire disclosures of these GenBank reports are incoiporated herein by reference. Again, the mature form of the above referenced protein would generally be used to prepare vaccine compositions of the invention.<br>
Bacterial pilins or pilin subportions suitable for use in the practice of the present invention will generally be able to associate to form non-natural molecular scaffolds.<br>
Methods for preparing pili and pilus-like structures in vitro are known in the art Bullitt et al., Proc. Natl. Acad. Set USA 95:12890-12895 (1996), for example, describe the in vitro reconstitution of E. coli P-pili subunits. Further, Eshdat et al.. J. Bacterial 148:308-314 (1981) describe methods suitable for dissociating Type-1 pili of E. coli and the reconstitution of pili. In brief, these methods are as-foUows; pili are dissociated by incubation at 37Â°C in saturated guanidine hydrochloride. Pilin proteins are then purified by chromatography, after which pilin dimers are formed by dialysis against 5 mM tris(hydroxymethyl) aminomethane hydrochloride (pH 8.0). Eshdat et al. also found that pilin dimers reassemble to form pili upon dialysis against the 5 roM tris(hydroxymethy!) aminomethane (pH 8.0) containing 5 mM MgCb.<br>
Further, using, for example, conventional genetic engineering and protein modification methods, pilin proteins may be modified to contain a first attachment site to which an antigen or antigenic detemainant is linked through a<br><br>
second attachment site. Alternatively, antigens or antigenic determinants can be directly linked through a second attachment site to amino acid residues which are naturally resident in these proteins. Hiese modified pilin proteins may then be used in vaccine compositions of the invention.<br>
Bacterial pilin proteins used to prepare vaccine compositions of the invention may be modified in a manner similar to that described herein for HBcAg. For example, cysteine and lysine residues may be either deleted or substituted with other amino add residues and first attachment sites may be added to these proteins. Further, pilin proteins may either be expressed in modified form or may be chemically modified after expression. Similarly, intact pili may be harvested from bacteria and then modified chemically.<br>
In another embodiment, pili or pilus-like structures are harvested fiom bacteria {e.g., E. coli) and used to form vaccine compositions of the invention. One example of pali suitable fox preparing vaccine compositions is the Type-1 pilus of K coli, which is formed from pilin monomers having the amino acid sequence set out in SHJ ID NO: 146.<br>
A number of mediods for harvesting bacterial pili are known in the art. Bullitt and Makowski iBiophys. J. 74:623-632 (1998)), for example, describe a pilus purification method for harvesting P-pili horn E. coli. According to this method, pili are sheared from hyperpiliated E. coli containing a P-pilus plasmid and purified by cycles of solubilization and MgCl2 (1.0 M) precipitation. A similar purification method is set out below in Example 33.<br>
Once harvested, pili or pilus-Uke structures may be modified in a variety of ways. For example, a first attachment site can be added to the jali to wch antigens or antigen detenninants may be attached through a second attachment site. In other words, bacterial piU or pilus-like structures can be harvested and modified to form non-naiural molecular scaffolds.<br>
Pili or pilus-hke stnictures may also be modified by the attachment of antigens or antigenic detCTminants in the absence of a non-natural organizer For example, antigens or antigenic determinants could be linked to naturally occurring cysteine resides or lysine residues. In such instances, the high order and repetitiveness of a naturally occurring amino acid residue would guide the coupling of the antigens or antigenic determinants to the pili or pfius-ltke structures. For example, the pili or pilus-like stractvires could be linked to the second attachment sites of the antigens or antigenic determinants using a heterobifunctional cross-linking agent.<br>
When structures which are naturally synthesized by organisms {e.g., pili) are used to prepare vaccine compositions of the invention, it will often be<br><br>
advantageous to genetically engineer these organisms so that they produce structures having desirable characteristics. For example, when Type-1 pili of E. coli are used, the E. coli from which these pili are harvested may be modified so as to produce structures with specific characteristics. Examples of possible mofications of pilin. proteins include ttie insertion of one or more lysine residues, the deletion or substitution of one or more of the naturally resident lysine residues, and the deletion or substitution of one or more naturally resident cysteine residues {e.g., the cysteine residues at positions 44 and 84 in SEQ ID N0:14Â«).<br>
Further, additional modifications can be made to pilin genes which result in the expression products containing a first attachment site other than a lysine residue (e.g., a FOS or J17N domain). Of course, suitable first attachment sites will generally be limited to those which do no prevent pilin proteins fix&gt;m foraiing pili or pilus-like structure suitable for use in vaccine compositions of the invention.<br>
Pilin genes which naturally reside in bacterial cells can be modified in vivo (e.g., by homologous recombination) or pilin genes with particular characteristics can be inserted into these cells. For examples, pilin genes could be introduced inte bacterial cells as a component of rattier a replicable cloning vector or a vector which inserts into the bacterial chromosome. The inserted pilin genes may also be linked to expression regulatory control sequences ie.g., a lac operator).<br>
In most instances, the pili or pilus-like structures used in vaccine conwsitions of the invention wiD be composed of single type of a pilin subunit Pili or pilus-like structures composed of identical subunits will generally be used because they are expected to form structures which present highly ordered and repetitive antigen arrays.<br>
However, the compositions of the invention also include vaccines comprising pili or pilus-like structures formed ftum heterogenous pilin subunits. The pUin subunits which form these pili or pilus-like structures can be expressed fium genes naturally resident in the bacterial cell or may be introduced into the ceUs. When a naturally resident pilin gene and an introduced gene are both expressed in a cell which forms piB or pilus-like structures, the result will generally be structures formed from a mixture of these piUn proteins. Further, when two or more pilin genes are expressed in a bacterial cell, the relative expression of each piUn gene will typically be the factor which determines the ratio of the different piHn subunits in the pili or pilus-like structures.<br><br>
When pili or pilus-like structures having a particular composition of mixed pilin subunits is desired, the expression of at least one of the pilin genes can be regulated by a heterologous, inducible promoter. Such promoters, as well as other genetic elements, can be used to regulate the relative amounts of different pilin subunits proceed in the bacterial cell and, hence, the composition of the pili or pilus-lifce structures.<br>
In additional, while in most instances the antigen or antigenic determinant will be linked to bacterial piH or pilus-like structures by a bond which is not a peptide bond, bacterial cells which produce pili or pilus-lifce stmctures used in the compositions of the invention can be genetically engineered to generate pilin proteins which are fused to an antigen or antigenic determinant. Such fusion proteins which form pili or pilus-like structures are suitable for use in vaccine compositions of the invention.<br>
As already discussed, viral capsids may be used for (I) the presentation or antigen or antigenic determinants and (2) the preparation of vaccine compositions of the invention. Particularly, useful in the practice of the invention are viral capsid proteins, also referred to herein as "coat proteins," which upon expression form capsids or capsid-Iike structures. Thus, these csid proteins can form core particles and non-natural molecular scaffolds. Generally, these capsids or capsid-Uke structures form ordered and repetitive arrays which can be used for the presentation of antigens or antigenic determinants and the preparation of vaccine compositions of the invention.<br>
One or more (e.g., one, two, three, four, five, etc.) antigens or antigenic determinants may be attached by any number of means to one or more (e.g., one, two, three, four, five, etc.) proteins which form viral capsids or capsid-like structures (e.g., bacteriophage coat proteins), as well as other proteins. For example, antigens or antigenic determinants may be attached to core particles using first and second attachment sites. Further, one or more (e.g., one, two, three, four, five, etc.) heterobifunctional crosslinkers can be used to attach antigens or antigenic determinants to one or more proteins which fonn viral capsids or capsid-like structures.<br>
Viral capsid proteins, or fi:gments thereof may be used, for example, to prepare core particles and vaccine compositions of the invention. Bacteriophage Q coat proteins, for example, can be expressed recombinantly in E. coli. Further, upon such expression these proteins spontaneously form capsids. Additionally, these capsids form ordered and repetitive antigen or antigenic determinant arrays which can be used for antigen presentation and the preparation of vaccine compositions.  As described below in Example 38, bacteriophage Qp<br><br>
coat proteins can be used to prepare vaccine compositions which elicit immunological responses to antigenic determinants.<br>
Specific examples of bacteriophage coat proteins which can be used to prepare compositions of the invention include the coat proteins of RNA bacteriophages such as bacteriophage Qp (SEQ ID NO:159; PIR Database, Accession No. VCBPQ3 refeiring to Qp CP and SEQ ID NO: 217; Accession No. AAA16663 referring to Q|3 Al protein), bacteriophage R17 (SEQ ED NO:160; PIR Accession No. VCBPR7), bacteriophage fr (SEQ ID N0:161; PIR Accession No. VCBPFR), bacteriophage GA (SEQ ID NO: 162; GenBank Accession No. NP-040754), bacteriophage SP (SEQ ID NO:163; GenBank Accession No. CAA30374 referring to SP CP and SEQ ID NO: 254; Accession No. referring to SP Al protein), bacteriophage MS2 (SEQ ID NO:I64; FIR Accession No. VCBPM2), bacteriophage Mil (SEQ ID NO:165; GenBank Accession No. AAC06250), bacteriophage MXl (SEQ ID NO:I66; GenBank Accession No. AAC14699), bacteriophage NL95 (SEQ ID NO;167; GenBank Accession No. AAC14704), bacteriophage f2 (SEQ ID NO: 215; GenBank Accession No. P03611), bacteriophage PP7 (SEQ ID NO: 253), As one skilled in the art would recognize, any protein which forms capsids or capsid-like structures can be used for the IHeparation of vaccine compositions of the invention. Furthermore, the Al protein of bacteriophage Q|3 or C-temnnal truncated forms missing as much as 100, 150 or 180 amino acids from its C-terminus may be incorporated in a capsid assembly of QP coat proteins. The Al protein may also be fused to an organizer and hence a first attachment site, for attachment of Antigens containing a second attachment site. Generally, the percentage of A1 protein relative to Qp CP in the capsid assembly will be UiiHted, in order to insure csid formation. Al protein accession No. AAA16663 (SEQ ID NO: 217).<br>
QP coat protein has also been found to self-assemble into capsids when expressed in JE, coli (KozlovskaTM. et al", GENE W". 133-137 (1993)). The obtained capsids or virus-Hke particles showed an icosahedral phage-like capsid structure with a diameter of 25 nm and T=3 quasi symmetry. Further, the crystal structure of phage QP has been solved. The capsid contains 180 copies of the coat protein, which are linked in covalent pentamers and hexamers by disulfide bridges (Golmohammadi, R. et cd.. Structure 4:543-5554 (1996)). Other RNA phage coat proteins have also been shown to self-assemble upon expression jn a bacterial host (Kastelein, RA. et al.. Gene 23:245-254 (1983), Kozlovskaya, TM. et al, Dokl. Akad. Nauk SSSR 287:452-455 (1986), Adhin, MR. et al.. Virology 170: 238-242<br><br>
*(1989), Ni, CZ.. et al.. Protein Sci. 5: 2485-2493 (1996), Priano, C. et al., J. Mol. Biol. 249; 283-297 (1995)). The QP phage capsid contains, in addition to the coat protein, the so called read-through protein Al and the maturation protein A2. Al is generated by suppression at the UGA stop codon and has a length of 329 aa. The capsid of phage QP recombinant coat protein used in the invention is devoid of the A2 lysis protein, and contains RNA from the host. The coat protein of RNA phages is an RNA binding protein, and into-acts with the stem loop of the ribosomal binding site of the replicase gene acting as a translational repressor during the life cycle of the vims. The sequence and structural elements of the interaction are known (Witherell, GW. &amp; Uhlenbeck, OC. Biochemistry 28: 71-76 (1989); Lim F. et al., J. Biol. Chem. 271:31839-31845 (1996)). The stem loop and RNA in general are known to be involved in the virus assembly (Golmohammadi, R. et al.. Structure 4: 543-5554 (1996))<br>
Proteins or protein domains may affect the structure and assembly of the particle even more then a short peptide. As an example, proper folding of antigens comprising disulfide bridges will generally not be possible in the cytoplasm of E. coli, where the QP particles are expressed. Likewise, glycosylation is generally not possible in prokaryotic expression systems. It is therefore an advantage of the contemplated invention described here to attach the antigen to the particle by starting with the already assembled particle and the isolated antigen. This allows expression of both the particle and the antigen in an expression host guaranteeing prop folding of the antigen, and proper folding and assembly of the particle.<br>
It is a finding of this invention, that one or several several antigen molecules maybeattachedtoonesubunit of the capsid of RNA phages coat proteins. A specific feature of the capsid of the coat protein of RNA phages and in particular of QP csid â  is thus the possibility to couple several antigens po" subunit This allows for the generation of a dense antigen array. Other viral capsids used for covalent attachment of antigens by way of chemical cross-linking, such for example a HBcAg modified with a lysine residue in its major immunodominant region (MIR; WO 00/32227), show coupling density of maximally 0.5 antigens per subunit The distance between the spikes (corresponding to the MIR) of HBcAg is 50 A (Wynne, SA. et al., Mol. Cell 3:771-780 (1999)), and therefore an antigen array with distances shorter than 50 A cannot be generated<br><br>
-4-;-Csids of QP coat protein display a defined number of lysine residues on<br>
their surface, with a defined topology with three lysine residues pointing towards the<br>
interior of the capsid and interacting with the RNA, and four other lysine residues<br>
exposed to the exterior of the capsid. These defined properties favor the attachment of<br>
antigens to the exterior of the particle, and not to the interior where the lysine residues<br>
interact with RNA. Capsids of other RNA phage coat proteins also have a defined<br>
number of lysine residues on thdr surface and a defined topology of tiiese lysine<br>
residues. Another advantage of the csids derived fiom RNA phages is their high<br>
expression yield in bacteria, that allows to produce large quantities of material at<br>
affordable cost.<br>
Another feature of the capsid of QP coat protein is its stability. QP subunits are bound via disulfide bridges to each other, covalently linking the subunits. Qp capsid protein also shows unusual resistance to organic solvents and denaturing agents. Surprisingly, we have observed that DMSO and acetonitrile concentrations as high as 30%, and Guanidinium concentrations as high as 1 M could be used without affecting the stability or the ability to form antigen arrays of the capsid. Thus, theses (ffganic solvents may be used to couple hydrophobic peptides. The high stability of the capsid of Qp coat protein is an important feature pertaining to its use for immunization and vaccination of mammals and humans in particular. The resistance of the capsid to organic solvent allows the coupling of antigens not soluble in aqueous buffers.<br>
Insertion of a cysteine residue into the N-teiminal P-haiipin of the coat protein of the RNA phage MS-2 has been described in the patent application US/5,698,424. We note however, that the presence of an exposed free cysteine residue in the capsid may lead to oligomerization of csids by way of disulfide bridge formation. Other " attachments contemplated in patent application tJS/5,698,424 involve the formation of disulfide bridges between the antigen and the Qp particle. Such attachments are labile to sulfhydryl-moiety containing molecules.<br>
The reaction between an initial disulfide bridge formed with a cys-residue on QP, and the antigen containing a free sulfhydryl residue releases sulfhydryl containing species other than the antigen. These newly formes sulfhydryl containing species can react again with other disulfide bridges present on the particle, thus establishing an equilibrium. Upon reaction with the disulfide bridge formed on the<br><br>
particle, the antigen may either form a disulfide bridge with the cys-residue from the particle, or with the cys-residue of the leaving group molecule which was forming the initial disulficte bridge on the particle. Moreover, the other method of attachment descaibed, using a hetero-bifunctional cross-linker reacting with a cysteine on the QP particle on one side, and with a lysine residue on tiie antigen on the other side, leads to a random orientation of the antigens on the particle.<br>
We further note that, in contrast to the capsid of the QP and Fr coat proteins, recombinant MS-2 described in patent plication US/5,698,424 is essentially free of nucleic acids, while RNA is packaged inside the two capsids mentioned above.<br>
We describe new and inventive compositions allowing the formation of robust antigen arrays with variable antigen density. We show that much higher epitope density can be achieved than usuaUy obtained with other VLPs. We also disclose compositions with simultaneous display of several antigens widi appropriate spacing, and compositions wherein the addition of accessory molecules, enhancing solubility or modifiying the capsid in a suitable and desired way.<br>
The preparation of compositions of capsids of RNA phage coat proteins with a high epitope density is disclosed in this application. As a skilled artisan in the Art would know, the conditions for the assembly of the ordered and repetitive antigen array depend for a good part on the antigen and on the selection of a second attachment site on the antigen. In the case of the absence of a useful second attachment site, such a second attachment has to be engineered to the antigen.<br>
A prerequisite in designing a second attachment site, is the choice of the position at which it should be fijsed, inserted or generally engineered. A skilled artisan would know how to find guidance in selecting the position of the second attachment site. A crystal structure of die antigen may provide information on the availability of the C- or N-termini of the molecule (determined for example from their accessibility to solvent), or on the exposure to solvent of residues suitable for use as second attachment sites, such as cysteine residues. Exposed distilfide bridges, as is the case for Fab fragments, may also be a soiue of a second attachment site, since they can be generally converted to single cysteine residues through mild reduction. In general, in the case where immunization vnth a self-antigen is aiming at inhibiting the interaction of this self-antigen with its natural Ugands, the second attachment site will<br><br>
i<br>
be added such that it allows generation of antibodies against the site of interaction with the natural ligands. Thus, the location of the second attachment site will selected such, that steric hindrance from the second attachment site or any amino acid linker containing it, is avoided Li fiirther embodiments, an antibody response directed at a site distinct ftxim the interaction site of the self-antigen with its natural ligand is desired In such embodiments, the second attachment site may be selected such that it prevents generation of antibodies against the intertion site of the self-antigen with its natural ligands.<br>
Other criteria in selecting the position of the second attachment site include the oligomerization state of the antigen, the site of oligomerization, the presence of a cofactor, and llie availability of experimental evidence disclosing sites in the antigen structure and sequence where modification of the antigen is compatible with the function of the seff-antigen, or with the generation of antibodies recognizing the self-antigen.<br>
In some embodiments, engineering of a second attachment site onto the antigen requires the fusion of an amino acid linker containing an amino acid suitable as second attachment site according to the disclosures of this invention. In a preferred embodiment, the amino acid is cysteine. Tlie selection of the amino acidd linker will be dependent on the nature of the self-antigen, on its biochemical properties, such as pi, charge distribution, glycosylation. In general, flexible amino acid linkers are favored Examples of amino acid linkers are the hinge region of Immunoglobulins, glycine serine linkera (GGGGS),,, and glycine linkers (G)n all further containing a cysteine residue as second attachment site and optionally further glycine residues. (In the following arc examples of said amino acid linkers :<br>
N-tenninal gammal: CGDKTHTSPP<br>
C-terminal gamma 1; DKTHTSPPCG<br>
N-terminal gamma 3: CGGPKPSTPPGSSGGAP<br>
C-tenninal gamma 3: PKPSTPPGSSGGAPGGCG<br>
N-teiminal glycine linker: GCGGGG<br>
C-tenninal glycine linker: GGGGCG)<br><br>
For peptides, GGCG linkers at the C-terminus of the peptide, or CGG at its N"tenninus have shown to be useful. Li general, glycine residues will be inserted between bulky amino acids and the cysteine to be used as second attachment site.<br>
A particularly favored method of attachment of antigens to VLPs, and in particular to capsids of RNA phage coat proteins is the linldng of a lysine rraidue on the surface of the capsid of RNA phage coat proteins with a cysteine residue on the antigen. To be effective as second attachment site, a sulfliydryl group must be available for coupling. Thus, a cysteine residue has to be in its reduced state, that is a ftee cysteine or a cysteine residue with a free sulfliydryl group has to be available. In the instant where the cysteine residue to function as second attachment site is in an oxidized fonn, for example if it is forming a disulfide bridge, reduction of this disulfide bridge with e.g. DTT, TCEP or p-mercaptoethanol is required.<br>
It is a finding of this application that epitope density on the capsid of RNA phage coat proteins can be modulated by the choice of cross-hnker and other reaction conditions. For example, the cross-linkers Sulfo-GMBS and SMPH allow reaching higher epitope density than the cross-linker Sulfo-MBS under the same reaction conditions. Derivatization is positively influenced by high concentration of reactands, and manipulation of the reaction conditions can be used to control the number of antigens coupled to RNA images capsid proteins, and in particular to QP csid protein.<br>
From theoretical calculation, the maximally achievable number of globular protein antigens of a size of 17 kDa does not exceed 0.5. TTius, several of the lysine residues of the capsid of QP coat protein will be derivatized with a cross-linker molecule, yet be devoid of antigen. This leads to the disappearance of a positive charge, which may be detrimental to the solubility and stability of the conjugate. By replacing siome of the lysine residues with arginines, such is the case in the disclosed QP coat protein mutant, we prevent the excessive disappearance of positive charges since the arginine residues do not react with the cross-linkra-.<br>
In furthra" embodiments, we disclose a QP mutant coat protein with additional lysine residues, suitable for obtaining high density arrays of antigens.<br>
The  crystal  structure of several RNA bacteriophages  has  been determined (Golmohammadi, R. et al., Structure "#;543-554 (1996)).  Using such<br><br>
information, one skilled in the art could readily identify surface exposed residues and modify bacteriophage coat proteins such that one or more reactive amino acid residues can be inserted. Thus, one skilled in the art could readily generate and identify modified forms of bacteriophage coat proteins which can be used in the practice of the invention. Thus, variants of proteins which form capsids or capsid-Iike structures (e.g., coat proteins of bacteriophage QP, bacteriophage R17, bacteriophage fc, bacteriophage GA, bacriophage SP, and bacteriophage MS2) can also be used to prepare vaccine compositions of the invention.<br>
Although the sequence of the variants proteins discussed above will diffH- from their wild-type counterparts, these variant proteins will generally retain the ability to form cids or capsid-like structures. Thus, the invention further includes vaccine compositions which contain variants of proteins which form capsids or capsid-like structures, as well as methods for preparing such vaccine compositions, individual protein subunits used to prepare such vaccine compositions, and nucleic acid molecules which encode diese protein subunits. Hius, included within the scope of the invention are variant forms of wild-type proteins which form ordered and repetitive antigen arrays (e.g., variants of proteins which form capsids or capsid-like structures) and retain the ability to associate and form capsids or capsid-like structures.<br>
As a result, the invention further includes vaccine compositions coursing proteins comprising, or alternatively consisting of, amino acid sequences which are at least 80%, 85%. 90%, 95%, 97%, or 99% identical to wild-type proteins which form ordered arrays. In many instances, these proteins will be processed to remove signal peptides (e.g., heterologous signal peptides).<br>
Further included within the scope of the invention are nucleic acid molecules which encode proteins used to prepare vaccine compositions of the invention.<br>
In particular embodiments, the invention fuilher includes vaccine compositions comprising proteins pomprising, or alternatively consisting of, amino"scid sequences which are at least 80%, 85%, 90%, 95%, 97%, or 99% identical to any of the amino acid sequences shown in SEQ ID NOs:159-167, and forms of these proteins which have been processed, where appropriate, to remove the N-terminal leader sequence.<br>
Proteins suitable for use in the practice of the present invention also include C-terminal truncation mutants of proteins which form capsids or capsid-like structures, as well as other ordered arrays. Specific examples of such truncation mutants include proteins having an amino acid sequence shovra in any of SEQ ID NOs:159-167 where 1, 2, 5, 7, 9, 10, 12, 14, 15, or 17 ammo acids<br><br>
have been removed from the C-terminus. Normally, C-terminal truncation mutants used in the practice of the invention will retain the ability to form csids or capsid-Uke structures.<br>
Further proteins suitable for use in the practice of the present invention also include N-terminal truncation mutants of proteins which form capsi(k or capsid-like structures. Specific examples of such truncation mutants include proteins having an amino acid sequence shown in any of SEQ ID NOs:159-167 where 1,2, 5,7, 9,10,12,14, 15, or 17 amino acids have been removed from the N-tenninus. Normally, N-terminal truncation mutants used in the practice of the invention will retain the ability to form capsids or capsid-like structures.<br>
Additional proteins suitable for use in the practice of the presKit invention include - and C-terminal truncation mutants which form capsids w capsid-Uke structures. Suitable truncation mutants include proteins having an amino acid sequence shown in any of SEQ DD NOs:159-167 where 1, 2, 5, 7, 9, 10, 12,14, 15, or 17 amino acids have been removed from the N-terminus and 1, 2, 5, 7, 9, 10, 12, 14, 15, or 17 amino acids have been removed from the C-terminus. Normally, N-terminal and C-terminal truncation mutants used in the practice of the invention will retain the ability to form capsids or capsid-like structures.<br>
The invention further includes vaccine compositions comprising proteins comprising, or altematively consisting of, amino acid sequences which are at let 80%, 85%, 90%, 95%, 97%, or 99% identical to the above described truncation mutants.<br>
TTie invention thus includes vaccine coniositions prepared from proteins which form ordered arrays, methods for preparing vaccine compositions from individual protein subunits, methods for preparing these individual protan subunits, nucleic acid molecules which encode these subunits, and methods for vaccinating and/or eliciting inmmnological responses in individuals using vaccine compositions of the invention.<br>
B.       Construction of an Antigen or Antigenic Determinant with a Second Attachment Site<br><br>
The second element in the compositions of the invention is an antigen or antigenic determinant possessing at least one second attachment site enable of association through at least one non-peptide bond to the first attachment site of the non-natural molecular scaffold. The invention provides for compositions that vary according to the antigen or antigenic determinant selected in consideration of the desired therieutic effect. Other compositions are provided by varying the molecule selected for the second attachment site.<br>
However, when bacterial pili, or pilus-Uke structures, pilin proteins are used to prepare vaccine compositions of the invention, antigens or antigenic determinants may be attached to pilin proteins by the expression of pilin/antigen fusion proteins. Similarly, when proteins other than pilin proteins (e.g., viral capsid proteins) are used to prepare vaccine compositions of the invention, antigens or antigenic determinants may be attached to these non-pilin proteins fay the expression of non-pilin protein/antigen fusion proteins. Antigens or antigenic determinants may also be attached to bacterial pili, pilus-Uke structures, pilin proteins, and other proteins which form ordered arrays through non-peptide bonds.<br>
Antigens of the invention may be selected from the group consisting of the following: (a) proteins suited to induce an immune response against cancer cells; (b) proteins suited to induce an immune response against infectious diseases; (c) proteins suited to induce an immune response against allergens ,(d) proteins suited to induce an immune response in farm animals, and (e) fragments (e.g., a domain) of any of the proteins set out in (a)-(d).<br>
In one specific embodiment of the invention, the antigen or antigenic dstenmaant is one that is useful for the prevention of infectious disease. Such treatment will be usefiil to treat a wide variety of infectious diseases affecting a wide range of hosts, e.g,, human, cow, sheep, pig. dog, cat, other manamalian species and noD-mammalian species as well. Treatable infectious diseases are well known to those skilled in the art, examples include infections of viral etiology such as HTV, influenza, Herpes, viral hepatitis, Epstein Bar, polio, viral encephalitis, measles, chicken pox, etc.; or infections of bacterial etiology such as pneumonia, tuberculosis, syphilis, etc.; or infections of parasitic etiology such as malaria, trypanosomiasis, leishmaniasis, trichomoniasis, amoebiasis, etc. Thus, antigens or antigenic determinants selected for the compositions of the invention will be well known to those in the medical ait; examples of antigens or antigenic determinants include the following", the HIV antigens 140 and gpl60; the inflnenaza antigens hemaggintinin, M2 protein and neuraminidase. Hepatitis B surface antigen, circumsporozoite protein of malaria.<br>
In specific embodiments, the invention provides vaccine compositions suitable for use in methods for preventing and/or attenuating diseases or conditions which are<br><br>
caused or exacerbated by "self gene products {e.g., tumor necrosis factors). Thus, vaccine compositions of the invention include compositions which lead to the production of antibodies that prevent and/or attenuate diseases or conditions caused or exacerbated by "self" gene products. Examples of such diseases or conditions include graft versus host disease, IgE-mediated allergic reactions, anaphylaxis, adult respiratory distress syndrome, Crohn"s disease, allergic asthma, acute lymphoblastic leukemia (ALL), non-Hodgkin"s lymphoma (NHL), Graves" disease, systemic lupus aythematosus <sle inflammatory autoimmune diseases myasthenia gravis iramunoproliferative disease lymphadenopathy angioimmunoproliferative immunoblastive rtieumatoid arthritis diabetes multiple sclerosis alzheimer and osteoporosis.></sle>
In related specific embodiments, compositions of the invention are an immunotherapeutic that may be used for the treatment of allergies or cancer.<br>
Ths selection of antigens or antigenic determinants for the preparation of conqrasitions and for use in methods of treatment for allergies would be known to those skilled in the medical arts treating such disorders. Representative examples of such antigens or antigenic determinants include the following: bee venom phospholipase Az, Bet v I (birch pollen allergen), 5 Dol m V (white-faced hornet venom allergen), Mellitin and Der p I (House dust mite allergen), as well as fragments of each which can be used to elicit immunological responses.<br>
As indicated, a preferred antigen or antigenic determinant is Der p I. Der p I is a 25kD protease found in house dust mite faecal particles. Der p I represents the major allergic molecule of house dust mite. Accordingly, 80% of mite allergic patients have anti-Der p I IgE antibodies. In particular, the peptides p52-72 and pi 17-133, among others, are known to comprise epitopes, which are recognized by antibodies specific for the native Der p I. IgE antibodies raised in a polyclonal response to the whole antigen bind with high affinity to the peptide region 59-94 (L. Pierson-Mullany et al. (2000) Molecular Immunology). Other regions also bind IgE with high affinity. The peptide pi 17-133 contains"a free cysteine at its N-terminus, preferably representing the second attachment site in accordance with the invention. 3D model assigns peptides p52-72 andpll7-133 to the surface of the whole protein. However, other fragments of the Der p I protein may comprise B cell epitopes being preferably suitable for the present invention.<br>
The selection of antigens or antigenic determinants for compositions and methods of treatment for cancer would be known to those skilled in the medica! arts treating such disorders.    Representative examples of such types of antigens or<br><br>
antigenic detaminants include llie following: Her2 (breast cancer), GD2 (neuroblastoma), EGF-R (malignant glioblastoma), CEA (medullary thyroid cancer), and CD52 Oeukemia), human melanoma protein gplOO, human, melanoma protein melan-A/MART-1, tyrosinase, NA17-A nt protein, MAGE-3 protein, p53 protein, HPV16 E7 protein, as well as fragments of each which can be used to elicit immunological responses.Further preferred antigenic determinants useful for compositions and methods of treatment for cancer are molecules and antigenic determinants involved in angiogenesis. Angiogenesis, the formation of new blood vessels, plajs an essential role in physiological and pathophysiological processes such as wound healing and solid tumor growth, respectively olkman, J. (1995) Nat medicine 1, 27-31; Folkman, J., and Klagsbrun, M. (1987) Sciaicc 235, 442W6; Martiny-Baron, G., and Marm6, D. (1995) Curr. Opin. Biotechnol. 6,675-680; Risau, W. (1997) Nature 386, 671-674). Rapidly growing tumors initiate and depend on the formation of blood vessels to provide the required blood supply. Thus, antiangiogenic agents might be effective as an anticancer therapy.<br>
Among several putative angiogenic factors that have been identified so far vascular endothelial growth factor (VEGF) is a potent endothelial cell specific mitogen and a primary stimulant of the vascularization of many solid tumors. Although recent fintUngs impUcate that a set of angiogenic factors must be perfectly orchestrated to form functional vessels, it seems that the blockade of even a angle growth factor might limit disease-induced vascular growth. Thus, blockade of VEGF may be a premium target for intervention in tumor induced angiogenesis. To target the endothelium rather than the tumor itself has recently emerged as a novel strategy to fit tumors (Millauer, B., Shawver, L. K., Plate, K. H., Risau, W., and Ulkich, A. (1994) Nahtte 367, 575-579; Kim, J., Li, B., Winer, J., Armanini, M., Gillett, N., Phillip, H. S., Ferrara, N. (1993) Nature 362, 841-844). In contrast to tumors, which easily mutate target structures reccTgnized by the immune system, endothelial cells do not usually escape the immune system or other therapeutic regimens.<br>
An anti-VEGFR-H antibody (MC-lCl 1) and an and-VEGF antibody have been disclosed (Lu, D., Kussie, P., Pytowsld, B., Persaud, K., Bohlen, P.. Witte, L., Zhu, Z. (2000) J. Biol. Chem. 275,14321-14330; Presta, L.G, Chen, H., O"Connor, SJ., Chisholm. V., Meng, YG., Krummen, L.. Winkler, M., Ferrara N. (1997) Cancer Res. 47.4593-4599). The forma: neutralizing monoclonal anti-VEGFR-2 antibody<br><br>
recognizes an epitope that has been identified as putative VEGFA"EGFR-II binding site (Piossek, C, Schneidra--Mergener, J., Schimer, M., Vakalopoulou, E., Gemieroth, L., Thierauch, K.H. (1999) J Biol Chem. 274,5612-5619).<br>
ThuE, in another preferred embodiment of the invention, the antigen or antigenic determinant is a peptide derived from the VEGFR-II contact site. ITiis provides a composition and a vaccine composition in accordance with the invention, which may have antiangiogenic propatieE useful for the treatment of cancer. Inhibition of tumor growth in mice using sera specific for VEGFR-2 has been demonstrated (Wei, YQ-, Wang, QR., aao, X., Yang, L., Tian, L.. Lu, Y., Kang, B., Lu, CJ., Huang, MJ., Lou, YY., Xiao, E, He, QM., Shu, JM., Xie, XJ., Mao, YQ., Lei, S., Luo, F., Zhou, LQ.. Liu, CE., Zhou, a, Jiang, Y.. Peng, F., Yuan, LP., li, Q., Wu, Y., liu, JY. (2000) Nature Medicine 6,1160-1165). Therefore, further preferred antigenic determinants suitable for inventive compositions and antianogenic vaccine compositions in accordance with the invention comprise either the human VEGF!R-n derived peptide with the sequence CTARTELNVGIDFNWEYPSSKHQHKK:, and/or the murine VEGFR-II derived peptide having the sequence<br>
CTARTELNTVGLDFTWHSPPSKSHHKK, and/or the relevant extracellular globular domains 1-3 of tiie VEGFR-H.<br>
TTierefore, in a preferred embodiment of the invention, the vaccine composition comprises a core particle selected from a virus-like particle or a bacterial pilus and a VEGFR-H derived peptide or a fragment thereof as an antigen or antigenic detominant in accordance with the present invention.<br>
The selection of antigens or antigenic detemiinants for compositions and methods of treatment for other diseases or conditions associated with self antigens would be also known to those skilled in the mechcal arts treating such (tisorders. Representative examples of such antigens or antigenic detemiinants are, for example, lynhotoxins (e.g. Lymphotoxin a (LT a), Lympholoxin p (LT p)), and lymphotoxin receptors. Receptor activator of nuclear factor kB ligand (RANKL), vascular endothelial growfti factor (VEGF), vascular endothelial growth factor receptor (VEGF-R), Interleukin-5, Interleukin-17, iiterleukin-13, CCL21, CXCL12, SDF-1, MCP-1, Endoglin, Resistin, GHRH, IHRH, TRH, MIF, Eotaxin, Bradyltiflin, BLC, Tumor Necrosis Factor a and amyloid beta peptide (APM2) (SEQ ID NO: 220), as well as fragments of each which can be used to elicit immunological responses. In a<br><br>
preferred embodiment, the antigen is the amyloid beta peptide (APi-42) (DAEFRHDSGYEVHHQKL VFFAEDVGSNKGAnGLMVGGWIA (SEQ ID NO: 220), or a fragment thereof. Tlie amyloid beta protein is SEQ ID NO: 218. The amyloid beta precursor protein is SEQ ID NO: 219.<br>
In another preferred embodiment of the invention, the antigen or antigenic determinant is an angiotensin peptide or a fragment thereof. The term "anotensin peptide" as used herein, shall encompass any peptide comprising the sequence, or fragments thereof, of angiotensinogen, angiotensin I or angiotensin IL The sequences are as follows: Angiotensinogen: DRVYIHPFHLVIHN; Angiotensin L DRVYIHPEHL; Angiotensin II: DRVYIHPF. Typically, one or more additional amino acids are added either at the C- or at the N-terminus of the angiotensin peptide sequences. The sequence of the angiotensin peptides corresponds to the human sequence, which is identical to the murine sequence. Therefore, immunization of a human or a mouse with vaccines or compositions, respectively, comprising such angiotensin peptides as antigenic determinant in accordance with the invention, is a vaccination against a self-antigen. Those additional amino acids are, in particular, valuable for an oriented and ordered association to the core particle.<br>
Preferably, the angiotensin peptide has an amino acid sequence selected from the group consisting of a) the amino acid sequence of CGGDRVYIHPF; b) the amino acid sequence of CGGDRVYIHPFHL; c) the amino acid sequence of DRVYffiPFHLGGC; and d) the amino acid sequence of CDRVYIHPFH. Angiotensin I is cleaved from angiotensinogen (14aa) by the kidney-derived enzyme Renin. Angiotensin I is a biologically inactive peptide of 10 aa. It is further cleaved at die N-tenninus fay angiotensin converting enzyme (ACE) into die biologically active 8aa angiotensin U. Tliis peptide binds to the antgiotensin receptors ATII and AT2 which leads to vasoconstriction and aldosterone release.<br>
A vaccine in accordance with the present invention comprising at least one angiotensin peptide may be used for the treatment of hypertension.<br>
In a particular embodiment of the invention, the antigen br antigenic determinant is selected from the group consisting of: (a) a recombinant protein of HIV, (b) a recombinant protein of Influenza vims (e.g., an Influenza virus M2 protein or a fragment thereof), (c) a recombinant protein of Hepatitis C virus, (d) a recombinant protein of Toxoplasma, (e) a recombinant protein of Plasmodium falciparum, (f) a recombinant protein of Plasmodium vivax (g) a recombinant protein of Plasmodium ovale, (h) a recombinant protein of Plasmodium malariae, (i) a recombinant protein of breast cancer cells, (j) a recombinant protein of kidney cancer cells, (k) a recombinant protein of prostate cancer cells,. (1) a recombinant protein of<br><br>
skin cancer cells, (m) a recombinant protein of brain cancer cells, (n) a recombinant protein of leukemia cells, (o) a recombinant profiling, (p) a recombinant protein of bee sting allergy, (q) a recombinant proteins of nut allergy, (r) a recombinant proteins of food allergies, (s) recombinant proteins of asthma, (t)a recombinant protein of Chlamydia, and (u) a fragment of any of the proteins set out in (a)-(t).<br>
Once the antigen or antigenic determinant of the composition is chosen, at least one second attachment site may be added to the molecule in preparing to construct the organized and repetitive array associated with the non-natural molecular scaffold of the invention. Knowledge of what will constitute an ipropriate second attachment site will be known to those skilled in the art. Representative examples of second attachment sites include, but are not limited to, the following: an antigen, an antibody or antibody fragment, biotin, avidin, strepavidin, a receptor, a receptor ligand, a hgand, a ligand-binding protein, an interacting leucine zipper polypeptide, an amino group, a chemical group reactive to an amino group; a carboxyl group, chemical group reactive to a carboxyl group, a sulfhydryl group, a chemical group reactive to a sulfhydryl group, or a combination thereof.<br>
The association between the first and second attachment sites will be detemiined by the characteristics of the respective molecules selected but will comprise at least one non-peptide bond. Depending upon the combination of first and second attachment sites, the nature of the association may be covalent, ionic, hydrophobic, polar, or a combination thereof.<br>
In one embodiment of the invention, the second attachment site may be the FOS leucine zipper protein domain or the JUN leucine zipper protein domain.<br>
In a more specific embodiment of the invention, the second attachment site selected is the FOS leucine zippo- proton domain, which associates specifically with the JUN leucine zipper protein domain of the non-natural molecular scaffold of the invention. The association of the JUN and FOS leucine zipper protein domains provides a basis for the formation of an organized and repetitive antigen or antigenic determinant array on the surface of the scaffold. The FOS leucine zipper proton domain may be fused in frame to the antigen or antigenic determinant of choice at either the amino terminus, carboxyl terminus or internally located in the protein if desired.<br>
Several FOS fusion constructs are provided for exemplary purposes. Human growth hormone (Example 4), bee venom phospholipase A2 (PLA2) (Example 9), ovalbumin (Example 10) and HTV gpl40 (Example 12).<br>
In order to simplify the generation of FOS fusion constructs, several vectors are disclosed that provide options for antigen or antigenic detwininant design and construction (see Example 6). The vectore pAVM wae designed for the expression<br><br>
of FOS fusion in E. coli; the vectors pAV5 and pAV6 were designed for the expression of FOS fusion proteins in eukaryotic cells. Properties of these vectors are briefly described:<br>
1-	pAVl: This vector was designed for the secretion of fusion proteins with FOS at the C-teiminus into the E. coli penplasmic space. The gene of interest (g,o.i.) may be hgated into the StuI/NotI sites of the vector.<br>
2-	pAV2: This vector was designed for the secretion of fusion proteins with FOS at the N-tenninus into the E. coli penplasmic space. The gene of intMst ($.0.1.) ligated into the NotKEcoRV (or Notl/Hjndlll) sites of the vector.<br><br>
3.	pAV3: This vector was designed for the cytoplasmic production of fusion proteins with FOS at the C-tenninus in E. coli. The gene of interest (g.o.i.) may be ligated into the EcoRV/NotI sites of the vector.<br>
4.	pAV4: This vector is designed for the cytoplasmic production of fusion proteins with FOS at the N-terminus in E. coli. The gene of interest (g.o.i.) may be ligated into the NotKEcoRV (or Noti/ffindlD) sites of the vectw. The N-tenninal methionine residue is proteolytically removed upon protein synthesis (HLrel et al, Proc. Natl. Acad. Set USA S(5:8247-8251 (1989)).<br>
5.	pAV5: This vector was designed for the eukaryotic production of fusion proteins with FOS at the C-terminus. The gene of interest (g.o.i.) may be inserted between the sequences coding for the hGH signal sequence and the FOS domain by ligation into the Eco47III/NotI sites of the vector. Alternatively, a gene containing its own signal sequence may be fused to the FOS coding region by ligation into the Stul/NotI sites.<br>
6.	pAV6: This vector was designed for the eukaryotic production of fiision proteins with FOS at the N-teiminus. "Hie gene of interest (g.o.i.) may be ligated into the Notl/StuI (or Notl/IBndlll) sites of the vector.<br>
As will be understood by those skilled in the art, the construction of a FOS-antigen or -antigenic determinant fusion protein may include the addition of certain genetic elements to facilitate production of the recombinant protein. Example 4 provides guidance for the addition of certain E. coli regulatory elements for translation, and Example 7 provides guidance for the addition of a eukaryotic signal sequence. Other genetic elements may be selected, depending on the specific needs of the practioner.<br>
The invention is also seen to ioclude the production of the FOS-antigen or F05-antigenic determinant fusion protein either in bacterial (Example 5) or eukaryotic cells (Example 8). The choice of which cell type in which to express the fusion protein is within the knowledge of the skilled artisan, depending on factors<br><br>
such as whether post-translationai modifications are an inrtant consideration in the design of the composition.<br>
As noted previously, the invention discloses various methods for the construction of a FOS-antigen or FOS-antigenic determinant fusion protein through the use of the pAV vectors. In addition to enabling prokaryotic and eukaryotic expression, these vectors allow the practitioner to choose between N- and C-terminal addition to the antigen of the FOS leucine zipper protein domain. Specific examples are provided wherein N- and C-tenninal FOS fusions are made to PLAj (Example 9) and ovalbumin (Example 10). Exanle 11 demonstrates tfie purification of the PLA2 and ovalbumin FOS fiision proteins.<br>
In a more specific embodiment, the invention is drawn to an antigen or antigenic determinant encoded by the HIV genome. More specifically, the HIV antigen or antigenic determinant is gpl40. As provided for in Exanles 11-15, HIV gpl40 may be created with a FOS leucine zipper protein domain and the fusion protem synthesized and purified for attachment to the non-natural molecular scaffold of the invention. As one skilled in the art would know, other HIV antigens or antigenic determinants may be used in tiie creation of a composition of the invention.<br>
in another more specific embodiment, the invention is drawn to vaccine compositions conrising at least one antigen or antigenic determinaut encoded by an Influenza viral nucleic acid, and the use of such vaccine compositions to elicit immune responses. In an even more specific embodiment, the Mluenza antigen or antigenic determinant may be an M2 protein ifi.g., an M2 protein having the amino acids shown in SEQ ID NO:2l3, GenBank Accession No. P06g21, or in SEQ ID NO: 212, PIR Accession No. MFIV62, or fragment thereof (e.g., amino acids from about 2 to about 24 in SEQ ID NO:213. the ammo acid sequence in SEQ ID NO:212). Further, influenza antigens or antigenic determinants may be coupled to non-natural molecular scaffolds or core particles through either peptide or non-peptide bonds. When Influenza antigens or antigenic determinants are coupled to non-natural molecular scaffolds or core particles throu peptide bonds, the molecules which form order and repetitive arrays will generally be prepared as fusion protein expression products. The more preferred embodiment is however a composition, wherein the M2 peptide is coupled by chemical cross-Unking, to Qp capsid protein HBcAg capsid protein or Pili according to the disclosures of the invention.<br>
Portions of an M2 protein (e.g., an M2 protein having the amino acid sequence in SHJ ID NO:213), as well as other proteins against wMch an immunological response is sought, suitable for use with the invention may comprise, or alternatively consist of, peptides of any number of amino acids in length but will generally be at least 6 amino acids in length {e.g., peptides 6, 7, 8, 9,10, 12,15,18,<br><br>
20, 25, 30, 35, 40, 45. 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, or 91 amino acids in length).<br>
In another specific embodiment of the invention, the second attachment site selected is a cysteine residue, which associates specifically with a lysine residue of the non-natural molecular scaffold or core particle of the invention, or the second attachment site selected is a lysine residue, which associates specifically with a cysteine residue of the non-natural molecular scaffold or core particle of the invention. The chemical linkage of the lysine residue (Lys) and cysteine residue (Cys) provides a basis for the formation of an organized and repetitive antigen or antigenic determinant array on the surface of the scaffold or core particle. The cysteine or lysine residue may be engineered in frame to the antigen or antigenic determinant of choice at either the amino terminus, carboxyl terminus or internally located in the protein if desired. By way of example, PIA2 and HIV gpl40 are provided with a cysteine residue for linkage to a lysine residue first attachment site. In additional specific embodiments, the invention provides vaccine compositions suitable for use in methods for preventing and/or attenuating allergic reactions, such as allergic reactions which lead to anaphylaxis. Thus, vaccine compositions of the invention include compositions which lead to the production of antibodies that prevent and/or attenuate allergic reactions. Tlius, in certain embodiments, vaccine compositions of the invention include compositions which eUcit an immunological response against an allergen. Examples of such allergens include phospholipases such as the phospholipase A2 (PLA2) proteins of Apis mellifera (SEQ ID NO:168, GenBank Accession No. 443189; SEQ ID NO: 169, GenBank Accession No. 229378), Apis dorsata (SEQ ID NO: 170, GenBank Accession No. B59055), Jis cerana (SEQ ID NO:171, GenBank Accession No. A59055), Bombus permsylvanicus (SEQ ID NO:172 GenBank Accession No. B56338), and Heloderma suspectum (SEQ ID NO:173, GenBank Accession No. P800O3; SEQ ID NO:174, GenBank Accession No. S14764; SEQ ID NO:175, GenBank Accession No. 226711).<br>
Using the amino acid sequence of a PLA2 protein of Apis mellifera (SEQ ID NO: 168) for illustration, peptides of at least about 60 amino acids in length, which represent any portion of the whole PIA2 sequence, may also be used in compositions for preventing and/or attenuating allergic reactions. Examples of such peptides include peptides which comprise amino acids 1-60 in SEQ ID NO:168, amino acids 1-70 in SEQ ID NO:168, ammo acids 10-70 in SEQ ID N0:16S, amino acids 20-80 in SEQ ID NO:168, amino acids 30-90 in SEQ ID NO:168, amino acids 40-100 in SEQ ID NO:168, amino acids 47-99 in SEQ ID NO:168, amino acids 50-UO m SEQ ID NO:168, amino acids 60-120 in SEQ ID NO:168, ammo acids 70-130 in SEQ ID NO: 168, or amino acids 90-134 in SEQ ID NO: 168, as well corresponding portions of<br><br>
other PIA2 proteins (e.g., PLAj proteins described above). Further examples of such peptides include peptides which comprise amino acids 1-10 in SEQ ID NO:168, amino acids 5-15 in SEQ ID NO:168, amino acids 10-20 in SEQ ID NO:168, amino acids 20-30 in SEQ ID NO:168, amino acids 30-40 in SEQ ID NO:168, amino acids 40-50 in SEQ ID NO:168, amino acids 50-60 in SEQ ID NO:168, amino acids 60-70 in SEQ ID NO;168, amino acids 70-80 in SEQ ID NO:l68, amino acids 80-90 in SEQ ID NO;168, amino acids 90-100 in SEQ ID NO:168, amino acids 100-110 in SEQ ID NO:168, amino acids 110-120 in SEQ ID NO:I68, or amino acids 120-130 in SEQ ID NO:168, as well coiresponding portions of other PLA2 proteins (e.g., PLA2 proteins described above).<br>
Portions of PLA2, as well as portions of other proteins against which an immunological response is sout, suitable for use with the invention may comprise, or alternatively consist of, peptides which are generally at least 6 amino acids in length (e.g.. peptides 6, 7, 8, 9, 10, 12, 15,18, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70,75, 80, 85,90,95, or 100 amino acids in length).<br>
PLA2 peptides (e.g., the full length PLA2 proteins discussed above, as well as subportions of each) may also be coupled to any substance (e.g., a Q capsid protein or fragment thereof) which allows for the formation of ordered and repetitive antigen arrays.<br>
In another aspect of the preseait invention, the invention provides compositions being paiticulariy suitable for treating and/or preventing conditions caused or exacerbated by "self gene products.<br>
In a preferred embodiment of the invention, the antigenic determinant is RANKL (Receptor activator of NFkB ligand). RANKL is also known as TRANCE (TNF-related activation induced cytokine), ODF (Osteoclast differentiation factor) or OPGL (Osteoprotegerin ligand). TTie amino acid sequence of the extracellular part of human RANKL is shown in SEQ ID No: 221 (RANKL_human: TrEMBL: 014788), while the amino acid sequence of a human isoform is shown in SEQ ID No: 222. Sequences for the extracellular part of murine RANKL and an isoform are shown in SEQ ID No.223 (RANKL_mouse: TrEMBL:035235), and in SEQ ID No.224 (RANKL_mouse spUce forms: TrEMBL:Q9JJK8 and TrEMBL:Q9JJK9), respectively.<br>
It has been shown that RANKL is an essential factor in osteoclastogenesis. Inhibition of the interaction of RANKL with its receptor RANK can lead to a suppression of osteoclastogenesis and thus provide a means to stop excessive bone resorption as seen in osteoporosis and other conditions. The RANKL/RANK interaction was inhibited either by a RANK-Fc fusion protein or the soluble decoy receptor of RANKL, termed osteoprotegerin OPG.<br><br>
In the immune system RANKL is expressed on T cells while RANK is found on antigen-presenting cells. The RANKL-RANK interaction wajs shown to be critical for CD40L-independent T-helper cell activation (Bachmann et al, J. Exp. Med. 7: 1025 (1999)) and enhance the longevity and adjuvant properties of dendritic cells (Josien el al.. J Exp Med. 191:495 (2000)).<br>
In bone RANKL is expressed on stromal cells or osteoblasts, while RANK is expressed on the osteoclast precursor. The interaction of RANK and RANKL is crucial for the development of osteoclast precursors to mature osteoclasts. The interaction can be blocked by osteoprotegerin.<br>
OPG-deficient mice develop osteoporosis tfiat can be rescued by injection of recombinant OPG. Tliis means that OPG is able to reverse osteopOTOsis. Thus, inhibition of the RANK-RANKL interaction by way of injecting this specific embodiment of the invention may reverse osteoporosis.<br>
In addition, arterial calcification was observed in OPG k.o. mice which could be reversed by injection of OPG (Min et al., J. Exp. Med. 4: 463 (2000)). In an adjuvant-induced arthritis model OPG injection was able to prevent bone loss and cartilage destruction, but not inflammation (paw swelling). It is assumed that activated T cells lead to a RANKL-mediated increase of osteoclastogenesis and bone loss. OPG inhibits prostate cancer-induced osteoclastogenesis and prevents prostate tumor p-owth in the bone of mice. OPG diminishes advanced bone cancer pain in mice.<br>
RANKL is a transmembrane protein of 245 aa belonging to the TNF-superfamily. Part of the extracellular region (178 aa) can be shed by a TACE-like protease (Lum et al, J Biol Chem. 274:13613 (1999)). In addition splice variants lacking the transmembrane domain have been described (Ikeda et al., Endocrmologyl42: 1419 (2001)). The shed part contains the domain highly homologous to soluble TNF-OL This extracellular domain of RANKL forms homotrimers as seen for TNF-o. The C-terminus seems to be involved in the trimer contact site. One cysteine is present in this region of the sequence.<br>
We have built a model for the 3-4imensional stracture of the corresponding region of RANKL and found that the naturally present cysteine may not be accessible in the folded structure for interaction with a first attachment site on the carrier in accordance with the present invention. The N-tenninus is preferred for attaching a second attachment site comprising an amino acid linker with an additional cysteine residue. A human-RANKL construct with an N tennjnal amino acid linker containing a cysteine residue fused to the extracellular part of RAJNKL is a very preferred embodiment of the invention. However, an amino-acid linker containing a cysteine residue as second attachment site and being fused at the C-terminus of the RANKL<br><br>
sequence or the extracellular part of RANKL leads to further preferred embodiments<br>
of the invention.<br>
Human-RANKL constructs, such as the one identified in SEQ ID NO:320, are generated according to the teachings disclosed in EXAMPLE 6, and the man skilled in the art are able to compare murine and human RANKL sequences in a protein sequence alignment to identify the part of the sequence of hunian-RANKL to be cloned in the vectors described in EXAMPLE 6. Bragments containing amino acids 138-317 and corresponding to the C-terminal region of the extracellular domain of human RANKL, are particularly favored embodiments of the invention, and can be modified for coupling to VLPs and PiU as required according to the teaching of the present invention. However, other suitable vectors may also be used for expression in the suitable host described below. Further human-RANKL constructs, and in particular, the ones comprising the part of the extracellular region (178 aa), - or fragments thereof - that can be shed by a TACE-hke protease (Lum et al., J Biol Chem. 274:13613 (1999)), or comprising the sequence corresponding to the alternative splice variants lacking the transmembrane domain, as well as conservative fragments thereof, are intended to be encompassed within the scope of the present invention. Human C-terminal fragments comprising amino acids 165-317 are also embodiments of the invention. Alternatively, fragments which encompass the entire extracellular region (amino acids 71-317) and can be modified for coupling to VLPs and Pili as required according to the teaching of the present mvention, are also within the scope of the invention.<br>
JtANKL has been expressed in different systems (E.coli, insect cells, mammalian cells) and shown to be active, and therefore several expression systems can be used for production of the antigen of the composition. In the case where expression of the protein is directed to the periplasm of E. coli, the signal peptide of RANKL, or of RANKL constructs consisting of the extracellular part of the protein, and both possibly modified to comprise a second attachment site in accordance with the invention, is replaced with a bacterial signal peptide. For expression of the protein in the cytoplasm of E. coli, RANKL constructs are devoid of signal peptide.<br>
In another preferred embodiment of the invention, the antigenic determinant is MIF or a fragment thereof. MIF is a cytokine that has been first described in 1966 by its function as an inhibitor of macrophage migration. It is also known as delayed early response protein 6 (DER6), glyoDsylalion inhibiting factor or phenylpyruvate tautomerase. The latter name originates from enzymatic activity of MIF, however the endogenous substrate has not been identified.<br><br>
MIF has been shown to be implicated in a wide range of conditions. MIF (mRNA and protein) is upregulated in delayed type hypersensitivity (DTH) reaction induced by tuberculin, and anti-MlF antibody inhibits this DTH reaction. MIF is also upregulated in renal allograft rejection, In a model for ocular autoimmune disease, experimental autoimmune uveoretinitis (EAU), anti-MIF treatment caused delay of EAU development. In patients, there is an increase in serum of MIF, which is also the case in Behcet"s disease patients and patients suffering from iridocyclitis. Immunization against MIF may provide a way of treatment against rheumatoid arthritis.<br>
High senmi MIF concentration has been found in atopic dermatitis patients. In skin lesions, MIF is diffusely expressed instead of being found in the bal ceU layer in controls. MIF concentration is decreasing after steroid treatment, consistent with a role of MIF in inflammation. MIF has also been found to contribute to the establishment of glomerulonephritis. Animals treated with anti-MIF Antibody show significantly reduced glomerulonephritis. MIF is pituitary derived, secreted e.g. upon LPS stimulation, and potentiates endotoxemia. Accordingly, anti-MIF mAb inhibits endotoxemia and septic shock, while recombinant MEF markedly increases lethality of peritonitis. MIF is also a glucocorticoid-induced modulator of cytokine production, and promotes inflammation.<br>
MIF is produced by T-cells (Th2), supports proliferation of T-cells, and anti-MIF-treatment reduces T-cell proliferation and IgG levels. There is an increased MIF concentration in the cerebrpspinai fluid of multiple sclerosis and neuro-Behcet" s disease patients. Hgh MTF levels were also found in sera of patients with extended psoriasis. High MIF levels are found in sera of ulcerative colitis patients but not Crohn"s disease patients.<br>
High MCF levels have been found in sera of patients with bronchic asthma. MIF is also upregulated in synovial fluid of iheumatoid arthritis patients. Anti-MIF treatment was effectivly decreasing rfieumatoid arthritis in mouse and rat models (Mikulowska et ai, J. Immunol. ;55:5514-7(1997); Leech et at. Arthritis Rheum. 41:910-1 (1998), Leech et al. Arthntis Rheum. 45:827-33 (2000), Santos et al., Clin. Exp. Immunol. 725:309-14 (2001)). Thus, treatment directed at inhibiting MIF activity using a composition comprising MIF as an antigenic determinant may be beneficial for the conditions mentioned above.<br><br>
MIF from mouse, rat and human consists of 114 amino acid and contains three conserved cysteines, as shown in SEQ ID No 225 (MlF_rat: SwissProt), in SEQ ID No 226 (MIFmouse: SwissProt) and in SEQ ID No 227 (MIF.human: SwissPiot). Three subunits form a homotrimer that is not stabilized by disulfide bonds. The X-ray structure has been solved and shows three free cysteines (Sun et al, PNAS 93: 5191-96 (1996)), while some litK-ature data claim the presence of a disulfide bond-Nonetheless, none of the cysteines are exposed enough for optimal interaction with 8 possible first attachment site present on the carrier. Thus, as tiie C-terminus of the protein is exposed in the trimer structure, an amino acid linker containing a free cysteine residue is, preferably, added at the C-tMminus of the protein, for generation of the second attachment site in this preferred embodiment of the invention, as exemplarily described in EXAMPLE 4 for rat-MIF.<br>
There is only one amino acid change between mouse- and rat-MIF, and similarly a very high sequence homology (about 90 % sequence identity) between human- and rat-MIF or human- and mouse-MIF. Human- and mouse-MIF constructs according to the invention are described and can be generated as disclosed in EXAMPLE 4. Bi order to demonstrate the high potency to induce a self-specific immune response of MIF protein, or fragments thereof, associated to a core particle in accordance with the present invention, rat-MIF constructs coupled to QP capsid protem were injected in mice. The high antibody titers obtained by immunizing mice with rat-MIF show that tolance towards immunization with self-antigens was overcome by immunizing with MIF constructs coupled to virus-like particles, and in particular to QP capsid protein (EXAMPLE 4). Therefore, compositions in accordance with the present invention comprising human-MIF protein associated to a cere particle, preferably to pih or a virus-like particle, and more preferably to a virusÂ¬like particle of a RNA-phage, and even more preferably to RNA-phage Qp or fr, represent VM preferred embodiments of the present invention.<br>
However, an amino acid linker containing a free cysteine that is added at the N-terminus of the sequence of MIF leads to further prefeired embodiments of the present invention. MIF has been expressed in E.coU, purified and shown to be fully functional (Bemhagen et al., Biochemistry 33: 14144-155 (1994). Thus, MIF may be, preferably, expressed in E. coli for generating the preferred embodiments of the invention.<br><br>
Tautomerase activity of MIF is inhibited, if the start methionine is not cleaved from the construct. MIF constructs expressed in Kcoli and described in EXAMPLE 4 show tautomerase activity. Mutants of MIF where the start methionine is cleaved and where the proline residue right after the start methionine in the sequence is mutated to alanine also do not show tautomerase activity represent further embodiments of the invention and are intended to be encompassed within the scope of the invention. B some specific embodiments, immunization with MIF mutants devoid of tautomerase activity is envisaged.<br>
In another preferred embodiment of the invention, the antigenic determinant is Interleukin-17 (IL-17). Himian 1117 is a 32-kDa, disulfide-linked, homodimeritJ protein with variable glycosylation (Yao, Z. et al., J. Immunol 155: 5483-5486 (1995); Fossiez, F. et al., J. Exp. Med. 183: 2593-2603 (1996)). The protein comprises 155 amino acids and includes an N-terminal secretion signal sequence of 19-23 residues. The amino acid sequence of IL-17 is similar only to a Herpesvirus protein (HSV13) and is not similar to other cytokines or known proteins. The amino acid sequence of human IL-17 is shown in SEQ ID No: 228 (ACCESSION #: AAC50341), The mouse protein sequence is shown in SEQ ID No: 229 (ACCESSION #: AAA37490). Of die large number of tissues and cell lines evaluated, the mRNA transcript encodingE17 has been detected only in activatedT cells and phorbol 12-myristate 13-acetate/ionomycin-stimulated peripheral blood mononuclear cells (Yao, Z. et al., J. Immunol. 155: 5483-5486 (1995); Fossiez, F. et al., J. Exp. Med. 183: 2593-2603 (1996)). Botii human and mouse sequences contain 6 cysteine residues.<br>
The receptor for IL-17 is widely expressed in many tissues and cell types (Yao, Z. etal. Cytokine 9: 794-800 (1997)). Although the amino acid sequence of the human IL-17 receptor (866 aa) predicts a protein with a single trans-membrane domain and a long, 525 aa intracellular domain, the realtor sequence is unique and is not similar to that of any of the receptors from the cytokine/growth factor receptor family. This coupled with the lack of similarity of IL-17 itself to other known proteins indicates that IL-17 and its receptor may be part of a novel family of signalling protein and receptors. Clinical stu(es intcate IL-17 may be involved in many inflammatory diseases. IL-17 is secreted by synovial T cells fixjm ibeumatoid arthritis patients and stimulates the production of inflammatory mediators (Chabaud, M. et al., /. Immunol. 161: 409-414 (1998); Chabaud, M. et al., Arthntis Rheum. 42: 963-970 (1999)). H levels of 1117 have been reported in patients with rheumatoid arthritis (ZiolkowskaM.e/aZ.,J/mmurao/. 7(5:2832-8(2000)).<br>
	Interleukjn 17 has been shown to have an effect on proteoglycan degradation<br>
in murine knee joints (Dudler J. et al. Ann Rheum Dis. 59: 529-32 (2000)) and contribute to destruction of the synovium matrix (Chabaud M. et al., Cytoidne. i2:1092-9 (2000)). There are relevant arthritis models in animals for testing the effect of an immunization against MIF (Chabaud M. et al, Cytoidne. i2:1092-9 (2000)). Elevated levels of IL-17 mRNA have been found in mononuclear cells from patients with multiple sclerosis (Matusevicius, D. et al.. Mult. Scler. 5: 101-104 (1999)). Elevated serum levels of IL-17 are observed in patients suffering Systemic Lupus Eythematosus (Wong CX. et al.. Lupus 9: 589-93 (2000)). In addition, IL-17 mRKA levels are increased in T cells isolated from lesional psoriatic skin (Teunissen, M. B. et al. J. Invest. Dermatol 111: 645-649 (1998)).<br>
The involvement of IL-17 in rejection of kidney graft has also been demonstrated (Fossiez F. et al.. Int. Rev. Immunol 16:541-51 (1998)). Evidence for a role of IL-17 in organ allograft rejection has also been presented by Antonysamy et al. (J. Immunol 162:577-84 (1999)) who showed IL-17 promotes the fimctional differentiation of dendritic cell progenitors. Their findings suggest a role for IL--17 in allogeneic T cell proliferation that may be mediated in part via a maturation-inducing effect on DCs. Furthermore the same group reports (Tang J.L. et al. Transplantation 72:348-50 ( 2001)) a role for IL-17 in the immunopathogenesis of acute vascular rejection where Interleukin-17 antagonism inhibits acute but not chronic vascular rejection. IL-17 appears to have potential as a novel target for therapeutic intervention in allograft rejection.<br>
The above findings suggest IL-17 may play a pivotal role in the initiation or maintenance of an inflammatory response (Jovanovic; D. V. et al., J. lnmunol. 160: 3513-3521 (1998)).<br>
The anti-IL-17 monoclonal antibody mAb5 (Schering-Plough Research Institute) was able to completely inhibit the production of IL-6 Irom rheumatoid arthritis (RA) synovium supematants following induction by 50 ng/ml of IL-17. An irrelevant mAb MXl had no effect in this assay. mAb5 is a mouse IgGl obtained after immunization with human rIL-17 (r = recombinant). A concentration of 1 p,g/ml of mAb5 was able to completely inhibit the IL-6 production in the assay system (Chabaud, M. et al., J. Immunol 161: 409-14 (1998)). Thus, immunization against IL-17 provides a way of treatment for the various conditions described above.<br>
In another preferred embodiment of the invention, thus, the composition comprises a linker containing a second attachment site and being fused to the C-terminus of recombinant IL-17. In further prefared embodiments of the invention, however, an amino acid linker containing a free cysteine is fused to the N-taminus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-terminus of the sequence of the mature form of the protein, C-terminally of the<br><br>
signal peptide. For eukaryotic expression systems, the signal peptide of the IL-17 gene, as it is the case for the other self-antigens indicated herein, may be replaced by another signal peptide if required. For expression in bacteria, the signal peptide is either replaced by a bacterial signal peptide for soluble expression in the periplasm, or deleted for expression in the cytoplasm. Constructs of human IL-17 devoid of signal peptide will preferably comprise residues 24-155, 22-155, 21-155 or 20-155. Constructs of mouse IL-17 devoid of signal peptide will preferably comprise residues 26-158, 25-158, 24-158 or 27-155. Human IL-17 may be expressed in CVl/EBNA cells; recomlanant hIL-17 has been shown to be secreted in both glycosylated and nonglycosylated fonns (Yao, Z. et al., J. Immunol. 155: 5483-5486 (1995)). IL-17 can also be expressed as hIL-17/Fc fusion protein, with subsequent cleavage of the IL-17 protein from the fusion protein. IL-17 may also be expressed in the yeast Pichia pastoris (Murphy K.P. et. al.. Protein Expr Purif. 12: 208-14 (1998)). Human IL-17 may also be expressed in E. coll. When expression of IL-17 in E. coli is directed to the periplasm, the signal peptide of IL-17 is replaced by a bacterial signal peptide. For expression of the protein in the cytoplasm of E. coli, IL-17 constructs are devoid of signal peptide.<br>
In another preferred embodiment of the invention the antinic detraminant is Interleukin-13 CIL-13). IL-13 is a cytokine that is secreted by activated T lymphocytes and primarily impacts monocytes, macrophages, and B cells. The amino acid sequence of precursor human IL-13 is shown in SEQ ID No: 230 and the amino acid sequence of processed human IL-13 is shown in SEQ ED No: 231. The first 20 amino acids of the precursor protein correspond to the signal peptide, and are absent of the processed protein. The mouse sequence has also been described, and the processed amino acid sequence is shown in SEQ ID No: 232 (Brown K.D. et al.. J. Immunol. 142:619-681 (1989)). Depending on the expression host, the IL-13 construct will comprise the sequence of the precursor protein, e.g. for expression and secretion in eukaryotic hosts, or consist of the mature protein, e.g. for cytoplasmic expression in E.coli. For expression in the periplasm of E. coli, the signal peptide of IL-13 is replaced by a bacterial signal peptide.<br>
IL-13 is a T helper 2-derived cytokine (like IL-4, K5) that has recently been implicated in allergic airway responses (asthma). Upregulation of IL-13 and IL-13 -receptor has been found in many tumour types (e.g. Hodgkin lymphoma). Interleukin 13 is secreted by and stimulates the growth of Hodgkin and Reed-Stemberg cells (Kapp U et al, J Exp Med. 189:1939A6 (1999)). Thus, immunization against IL-13 provides a way of treating among others the conditions described above, such as Asthma or Hodgkins Lymphoma.<br><br>
preferably, the composition comprises an amino add linker containing a free cysteine residue and being fused to the N or C-tenninus of the sequence of mature IL-13 to introduce a second attachment site within the protein. In further preferred embodiments, an amino acid linker containing a free cysteine is added to the N-tenninus of the mature form of IL-13, since it is freely accessible according to the NMR structure of IL-13 (Eisenmesser, E. Z. ei at., J.MolBiol 310: 231 (2001)). In again further preferred embodiments, the amino acid linker containing a free cysteine is fused to the N-tenninus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-terminus of the sequence of the mature form of the protein, C-terminally of the signal peptide. In still further preferred embodiments, an amino acid linker containing a free cysteine residue is added to the C-terminus of the protein.<br>
IL-13 may be expressed in E.coli CEisenmesser E.Z. el al.. Protein Ejqir. Purif. 20:lS6-95 (2000)), or in NS-0 cells (eukaryotic cell line) (Cannon-Carlson S. et al, Protein Expr. Purif. 12:239AS (1998)). EXAMPLE 9 describes constructs and expression of constructs of murine IL-13, fused to an amino acid linker containing a cysteine residue, in bacterial and eukaryotic hosts.Human IL-13 constructs can be generated according to the teachings of EXAMPLE 9 and yielding the proteins human C-IL-13-F (SBQ ID NO:330) and human C-IH3-S (SEQ ID NO:331) after expression of the fusion proteins and cleavage with Factor Xa, and enterokinase respectively. The so generated proteins can be coupled to VLPs and Pili, leading to preferred embodiments of the invention.<br>
In yet another embodiment of the invention, the antigenic determinant is Interleukin-5 (IL-5). IL-5 is a lineage-specific cytokine for eosinophilopoiesis and plays an important part in diseases associated with increased number of eosinophils, such as asthma. The sequence of precursor and processed human IL-5 is provided in SEQ ID No: 233 and in SEQ ID No: 234, respectively, and the processed mouse amino acid sequence is shown in SEQ ED No: 235,<br>
The biological function of IL-5 has been shown in several studies (Coffman R.L. et al.. Science 245: 308-10 (1989); Kopf et al.. Immunity 4:15-24 (1996)), which point to a beneficial effect of inhibiting IL-5 function in diseases mediated through eosinophils. Inhibition of the action of IL-5 provides thus a way of treatment against asthma and other diseases associated with eosinophils.<br>
IL-5 forms a dimer, covalentiy linked by a disulfide bridge. A singje chain (sc) construct has been reported wherein two monomers of IL-5 are linked by a peptide linker.<br>
In preferred embodiments of the invention, a peptide linker containing a free cysteine is added at the N-terminus of the sequence of the processed form of IL-5.<br><br>
Addition of a linker containing a free cysteine is also, preferably, envisaged at the N-terminus of the sequence of the processed foim of a scIL-5. In further preferred embodiments, the amino acid Hnker containing a free cysteine is fused to the N-tenniniis of the sequence corresponding to the sequence of the processed protein, or inserted at the N-terminus of the sequence of the mature form of the protein, C-terminally of the signal peptide.<br>
In again further preferred embodiments, a linker containing a free cysteine is fused to the C- teiminus of the sequence of IL-5, or to the C-terminus of a scIL-5 sequence.<br>
A number of expression systems have been described for IL-5 and can be used in preparing the compositions of the invention. A bacterial expression system using E.coa has been described by Proudfoot et al, (Biochem J. 270:357-61 (1990)). In the case where IL-5 is expressed in the cytoplasm of E. coh, the IL-5 construct is devoid of a signal peptide. Insect cells may also be used for producing IL-5 constracts for making the compositions of the invention (Pierrot C. et al, Biochem. Biophys. Res. Commun. 255:756-60 (1998)). Likewise, Baculovirus expression systems (sf9 cells; higley E. et al.. Eur. J. Biochem. 196:623-9 (1991) and Brown P.M. et al.. Protein Expr. Purif. 6: 63-71 (1995)) can also be used. Hnally, mammalian expression systems have also been reported (CHO cells) and can be used in preparing these compositions of the invention (Kodama S et al., J. Biochem. (Tokyo) 110:693-701 (1991)).<br>
Baculovirus expression systems CCtchell et al, Biochem. Soc. Trans. 27:332S (1993); Kunimoto DY et al. Cytokine 5:224-30 (1991)) and a mammalian cell expression system using CHO cells (Kodama S et al., Glycobiology 2:419-27 (i992)) have also been described for mouse IL-5.<br>
EXAMPLE 10 describes the expression of murine IL-5 coiwtructs wherein the IL-5 sequence is fused at its N-terminus to amino acid linkers containing a cysteine residue for coupling to VLPs and Pili. Human constructs can be generated according to the teaching of EXAMPLE 10 and yield the proteins human C-IL-5-E (SEQ ID NO:335), human C-ES-F (SEQ ID NO:336) and human C-IL-5-S: (SEQ ID NO:337) suitable for coupling to VLPs and PUi and leading to preferred embodiments of the invention.<br>
Jn another preferred embodiment of the invention, the antigenic determinant is CCL-21. CCL-21 is a chemokine of the CC subfamily that is also known as small inducable cytokine A21, as exodus-2, as SLD (secondary lynhocyte cytokine), as TCA4 (thymus-deiived chemotactic agent 4) or 6Ckine.<br><br>
CCL21 inhibitis hemopoiesis and stimulates chemotaxis for thymocytes, activated T-cells and dendritic cells, but not for B cells, macrophages or neutrophiles. It shows preferential activitiy towards naive T cells. It is also a potent mesangial cell chemoattractant CCL2i binds to chemokine receptors CCR7 and to CXCR3 (dependent on species). It can trigger rapid integrin-dependent arrest of lymphocytes rolling under physiological shear and is highly expressed by high endothelial venules.<br>
Murine CCL21 inhibited tumor growth and angiogenesis in a human lung cancer SCID mouse model (Arenberg et al.. Cancer Immunol. Immunother. 49: 587-92 (2001)) and a colon carcinoma tumor model in mice (Vicari et al., J. Immunol. 165: 1992-2000 (2001)). The anostatic activity of murine CCL21 was also detected in a rat corneal micropocket assay (Soto et al., Proc. Natl. Acad. Sci. USA95: 8205-10 (1998).<br>
It has been shown that chemokine receptors CCR7 and CXCR4 are upregulated in breast cancer cells and that CCL21 and CXCL12, the respective ligands, are highly expressed in organs representing the first destinations of breast cancer metastasis MtiUer et al. (Nature 410: 50-6 (2001)). In vitro CCL21-mediated chemotaxis could be blocked by neutrali2dng anti-CCL21 antibodies as was CXCR4-mediated chemotaxis by the respective antibodies. Thus, immunization against CCL21 provides a way of treatment against metastatis spread in cancers, more specifically in breast cancer.<br>
Secreted CCL21 consist of 110 or HI aa in mice and humans, respectively. The respective sequences ace shown in SEQ ID No: 236 (Swissprot: SY21_human) and in SEQ ID No: 237 (Swissprot: SY21_mouse). In contrast to other CC cytokines does CCL21 contain two more cysteines within an extended region at the C-tenninus. It is assumed that all cysteines are engaged in disulfide bonds.<br>
In the following, constructs and expression systems are described for making compositions of the invention comprising the CCL21 antigenic determinant In the NMR structure of the homologous protein eotaxin, both N- and C-traminus are exposed to the solvent. In some specific embodiments, an amino acid linkra" containing a free cysteine residue as a second attachment site is added at the C-terminus of the protein. A fusion protein with alkaline phosphatase (at the C-terminus of CCL21) has been expressed and was shown to be functional, showing that fusions at the C-terminus of CCL21 are compatible with receptor binding. In other specific embodiments, the amino acid linker containing a fi-ee cysteine is fused to the N-teaminus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-termrnus of the sequence of the mature form of the protein, C-terminally of the signal peptide.<br><br>
Several expression systems have been described for production of CCL21 (e.g. Hedrick et al., J Immunol. 159: 1589-93 (1997)). For example, it may expressed inabaculovirussystem(Nagiraetal.,/. Biol. Chem. 272:19518-24(1997)).<br>
In a related preferred embodiment, the antigenic determinant is Stromal derived factor-1 (SDF-1), now termed CXCL12. CXCL12 is a chemoldne produced by bone marrow stromal cells and was originally identified as a stimulatory factor for pre-B cells.<br>
As already stated above, it has been shown that chranokire receptors CCR7 and CXCR4 are upregulated in breast cancer cells and that CCI-21 and SDF-1, the respective ligands, are highly expressed in organs representing the first destinations of breast cancer metastasis Miiller et al. (Nature 410: 50-6 (2001)). In vitro SDF-1 / CXCR4-mediated chemotaxis could be inhibitied by neutralizing anti-SDF-1 and anti-CXCR4 antibodies.<br>
In a breast cancer metastasis model in SCID mice using the human MDA-MB-231 breast cancer cell line, a sigmficant decrease in lung metastasis was observed when mice were treated with anti-CXCR4 antibodies. In the draining lymph nodes a reduction of metastasis to the inguinal and axillary lymph nodes (38% instead of 100% metastasis in controls) was observed. Thus, immunization against CXCL12 provides a way of treatment against metastatis of cancers, more specifically of breast cancers.<br>
The SDF-1 / CXCR4 chemokine-receptor pair has been shown to increase the efficacy of homing of more primitive hematopoietic progenitor cells to be bone marrow. In addition, CXCR4 and SDF-1 are supposed to influence the distribution of chronic lymphocytic leukemia cells. These cells invariably infiltrate the bone marrow of patients and it was shown that their migration in the bone manow was CXCR4 dependent. Chronic lymphocytic leukemia cells undergo optosis unless they are cocultured with stiomal cells. SDF-i blocking antibodies could inhibit this protective effect of stromal cells (Burger et al.. Blood 96: 2655-63 (2000)). Immunizing against CXCH2 thus provides a way of treatment against chronic lymphocytic leukemia.<br>
CXCR4 has been shown to be a coreceptor for entry of HIV into T-cells. SDF-1 inhibits infection of CD4+ cells by X4 (CXCR4-dependent) HIV sQns (Oberiin et al., Nature 382-.833-5 (1996); Bleui et al., Nature 3S2;829-33 (1996), Rusconi et al., Antivir. TJier. 5:199-204 (2000)). Synthetic peptide analogs of SDF-1 have been shown to effectively inhibit HIV-1 entry and infection via the CXCR4 receptor(WO059928Al). Thus, immunization against CXCL12 provides a way to block HIV entry in T-cells, and therefore a way of treating AIDS.<br>
SDF-1-CXCR4 interactions were also reported to play a central role in CD4+ T  cell   accumulation in riieumatoid  arthritis  synovium (Nanki  et  al.,  2000).<br><br>
Inununization against SDF-1 thus provides a way of treatment against rheumatoid arthritis.<br>
Human and murine SDF-l are known to arise in two fonns, SDF-la and SDF-ip, by difTerential splicing from a single gene. They differ in four C-terminal amino acids that are present in SDF-lp (74 aa) and absent in SDF-la (70 aa). The sequence of human is shown in SEQ ID No: 238 (Swissprot: SDFl_human) and the sequence mouse SDF-1 is shown in SEQ ID No: 239 (Swissprot SDFl_mouse). SDF-1 contains four conserved cysteines that form two ihtra-molecular disulfide bonds. The crystal structure of SDF shows a, non covalently-Iinked dimer (Dealwis et al. PNAS 95:6941-46 (1998)). The SDF-1 structure also shows a long N-terminal extension.<br>
Alanine-scanning mutagenesis was used to identify (part of) the receptor-binding site on SDF-1 (Ohnishi et al, J. Interferon Cytokine Res. 20: 691-700 (2000)) and Elisseeva et al (J. Biol Chem. 275:26799-805 (2000)) and Heveker et al. {Curr. Biol 5:369-76 (1998)) described SDF-1 derived peptides inhibiting receptor binding (and HTV entry).<br>
In the following, constructs and expression systems suitable in the generation of the compositions of the invention related to SDF-1 are described. Tlie N- and C-terminus of SDF-1 are exposed to the solvent. In specific embodiments, an amino add linka: containing a cysteine as second attachment site is thus fused to the C-terminus of the protein sequence, while in other specific embodiments an amino acid linker containing a cysteine as second attachment site is fused to the N-terminus of the protein sequence. The amino acid linker containing a free cysteine is fused to the N-terminus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-terminus of the sequence of the mature form of the protein, C-tenninally of the signal [Kptide.The genes coding for these specific constructs may be cloned in a suitable expression vector.<br>
Expression of SDF-1 in a sendai virus system in chicken embryonic fibroblasts (Moriya et al.. FEBS Lett. "#25:105-11 (1998)) has been described as well as expression in E.coU (Hohnes et al., Prot. Ejr. Purif. 21: 367-77 (2001)) and chemical synthesis of SDF-1 (Dealwis et al, PNAS 95: 6941-46 (2001)).<br>
In yet another embodiment of the invention, the antigenic determinant is BLC. B-lymphocyte chemoattractant (BLC, CXCL13) is expressed in die spleen, Peyer"s patches and lymph nodes (Gunn et al., 1998). Its expression is strongest in the germinal centres, where B cells undergo somatic mutation and affinity maturation. It belongs to the CXC chemokine family, and its closest homolog is GROa_(Gunn et al, Nature 391:799-803 (1998)). Human BLC is 64% homologous to murine BLC. Its receptor is CXCR5. BLC also shares homology with IL-8. BLC recruits B-cells to follicles in secondary lymphoid organs such as the spleen and peyer"s patches, BLC is<br><br>
also required for recruitment of B-cells to compartment of the lymph nodes rich in follicular Dendritic Cells (FDCs) (Ansel et al.. Nature 406:309-314 (2000)). BLC also induces increased expression of Lymphotoxinaip2 (LT?aip2) on the recruited B-cells. This provides a positive feed-back loop, since LT?aip2 promotes BLC expression (Ansel et al. Nature â #05:309-314 (2000)). BLC has also been shown to be able to induce lymphoid neogenesis (Luther et al.. Immunity 72:471-481(2000)). It appears that FDCs also express BLC. Thus immunization against BLC may provide a way of treatment against autoimmune diseases where lymphoid neogenesis is involved, such as Rheumatoid synovitis and Rheumatoid arthritis or Type I diabetes. A construct of BLC bearing a C-terminal his-tag has been described, and is functional (Ansel, K.M. etal., J. Exp. Med. 190: 1123-1134 (1999)).<br>
Thus, in a preferred embodiment of the present invention, the composition comprises a linker containing a cysteine residue as second attachment site and being fused at the C-tenninus of the BLC sequence.<br>
In IL-8, which is homologous to BLC, both N- and C-termini are free. In a further preferred embodiraent, addition of an amino acid linker containing a cysteine residue as second attachment site is, therefore, done to the N-tenninus of BLC for generation of this specific composition of the invention.<br>
In further preferred embodiments of the present invention, the composition comprises an amino acid linker containing a free cysteine and being fused to the N-terminus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-terminus of the sequence of the mature form of the protein, C-terminaliy of the signal peptide. The genes coding for these specific constructs may be cloned in a suitable expression vector and expressed accordingly. The sequence of human BLC is shown in SEQ ID No: 240 (Accession: NP_006410). Amino acids 1-22 of the sequence are the signal peptide. The mouse sequence is shown in SEQ ID No: 241 (AccKsion NP_061354). Amino acids 1-21 are the signal peptide. Compositions of the invention with BLC as the antigenic determinant, preferably, use the mature fonn of the protein for generating the compositions of the invention.<br>
hi another specific embodiment, the antigenic detenninant is Eotaxin. Eotaxin is a chemokine specific for Chemokine receptor 3, present on eosinophils, basophils and Th2 cells. Eotaxin seems however to be highly specific for Eosinophils itnmennan et al, J. Immunol 165: 5839-46 (2000)). Eosinophil migration is reduced by 70% in the eotaxin-1 knock-out mouse, which however can still develop eosinophilia (Rothenberg et al., J. Exp. Med. 185: 785-90 (1997)). IL-5 seems to be responsible for the migration of eosinophils fix)m bone-marrow to blood, and eotaxin for the local migration in the tissue (Humbles et al, J. Exp. Med. 186: 601-12 (1997)).<br><br>
ine numan genome uoniains :&gt; eotaxin genes, eotaxjni-3. They share 30% homology to each other. Two genes are known so far in the mouse: eotaxin 1 and eotaxin 2 (Zimmerman et al., I immunol 165: 5839-46 (2000)). They share 38% homology. Murine eotaxin-2 shares 59% homology with human eotaxin-2. hi the mouse, eotaxin-1 seems to be ubiquitously expressed in the gastro-intestinal tract, while eotaxin-2 seems to be predominantly expressed in the jejunum (Zimmerman et al, J. Immunol 165: 5839-46 (2000)). Eotaxin-1 is present jn broncho-alveolar fluid (Teixeira et al., J. Clin. Invest. 100: 1657-66 (1997)). Tlie sequence of human eotaxin-1 is shown in SEQ ID No.: 242 (aa 1-23 corresponds to the signal peptide), the sequence of human eotaxin-2 is shown in SEQ ID No.: 243 (aa 1-26 corresponds to the signal peptide), the sequence of human eotaxin-3 is shown in SEQ ID No.: 244 (aa 1-23 corresponds to the signal peptide), the sequence of mouse eotaxin-1 is shown in SEQ ID No.: 245 (aa 1-23 corresponds to the signal peptide), and the sequence of mouse eotaxin-2 is shown in SEQ ID No.: 246 (aa 1-23 COTresponds to the signal peptide).<br>
Eotaxin has a MW of 8.3 kDa. It is in equilibrium between monomers and dimers over a wide range of conditions, witii an estimated Kd of 1.3 mM at 37""C (Cramp et al., J. Biol Chem. "273: 22471-9 (1998)). The monomer fonn is however predominant Tlie structure of Eotaxin has been elucidated by NMR spectroscopy. Binding site to its receptor CCR3 is at the N-tenninus, and the region preceding the first cysteine is cracial (Crump et al., J. Biol Chem. 273: 22471-9 (1998)). Peptides of chemoldne receptors bound to Eotaxin confirmed this finding. Eotaxin has four cysteines forming two disulfide bridges. Therefore, in a preferred embodiment, the inventive composition conqirises an amino-acid linker containing a cysteine residue as second attachment site and being, preferably, fused to the C-tMminus of the Eotaxin sequence. In other preferred embodiments, an amino acid linker containing a free cysteine is fused to the N-tenninus of the sequence conesponding to the sequence of the processed protein, or inserted at the N-tenninus of the sequence of the mature form of the protein, C-terminally of the signal peptide. The genes coding for these specific constructs arc cloned in a suitable expression vector.<br>
Eotaxin can be chemically synthesized (Qark-Lewis et al.. Biochemistry 30:3128-3135 (1991)). Expression in E. coli has also been described for Eotaxin-1, in the cytoplasm (Crump et al.. J. Biol Chem. 273: 22471-9 (1998)). Expression in E. coli as inclusion bodies with subsequent refolding (Mayer et al. Biochemistry 39: 8382-95 (2000)), and iisect ceU expression (Forssmann et al., J. Exp. Med. 185: 2171-6 (1997)) have been described for Eotaxin-2, and may, moreover, be used to arrive at the specific embodiments of the invention.<br>
Jn yet another specific embodiment of the invention, the antigenic determinant is Macrophage colony-stimulating factor (M-CSF or CSF-1). M-CSF or CSF-1 is a<br><br>
regulator of proliferation, differentiation and survival of macrophages and their bonc" manow progenitors. The receptor for M-CSF is a cell surface tyrosine kinase receptor, encoded by the protooncxigene cfins. An elevated expression of M-CSF and its receptor has been associated with poor prognosis in several epithelial cancers such as breast, uterine and ovarian cancer. Tumor progression has been studied in a mouse strain resulting from the crossing of a transgenic mouse susceptible to mammary cancer (PyMT) with a mouse containing a recessive null mutation in csf-I gene. TTiese mice show attenuated late stage invasive carcinoma and pulmonary metastasis compared to the PyMT mouse (Lin et aJ., J. Exp. Med. 193:127-739 (2001)). The cause seems to be the absence of macrophage recruitment to neoplastic tissues-Subcutaneous growth of Lewis lung cancer is also impaired in csf.l null mice. It is postulated that the mecamsm of macrophage enhancement of tumor growth would be ttirough angiogenic factora, growth factors and proteases produced by the macrophages.<br>
Structural data on the soluble form of M-CSF are available (crystal structure: Pandit et al.. Science 25S:1358-*2 (1992)), and show that both the N- and C-termini of the iffotein are accessible. However, the N-tenninus is close to the site of interaction with the receptor. In addition, M-CSF is present both in a soluble and cell surface form, where the transmembrane region is at its C-terminus. llierefore, in a preferred embodiment of the present invention, the inventive composition comprises an amino acid linker containing a cysteine and being, preferably, added at the C-terminus of M-CSF or fragments thereof, or preferably at the C-terminus of the soluble form of M-CSF. In furtho: preferred embodiments, the amino acid Unker containir a free cysteine is fused to the N-terminus of the sequence corresponding to the sequence of the processed,protein or of the soluble form of the protein, or inserted at the N-terminus of the sequence of the mature form of the protein or of the soluble form of the protein, C-terminally of the signal peptide. M-CSF is a dimer, where the two monomers are linked via an interchain disulfide bridge.<br>
An expression system in E. coli has been described for an N-tenninal 149 amino acid fragment (functional) of M-CSF (Koths et al., Mol. Reprod. Dev. 46:31-yi (1997)). This fragment of M-CSF, preferably modified as outlined above, represents a preferred antigenic determinant in accordance with the invention.<br>
The human sequence is shown in SEQ ID No: 247 (Accession: NP_000748). Further preferred antigenic determinants of the present invention comprise the N-tetminal fragment consisting of residue 33 -181 or 33 -185 of SEQ ID No: 247, corresponding to the soluble form of the receptor.<br>
The mouse sequence (Accession. NP_031804) is shown in sequence ID No: 248- The mature sequence starts at amino acid 33. Thus, a preferred antigenic<br><br>
determinant in accordance with the present invention comprises amino-acid 33 -181 or 33-185.<br>
In another specific embodiment, the antigenic determinant is Resistin (Res). Passive immunization studies were performed with a rabbit polyclonal antibodies generated against a fusion protein of mouse Resistin (mRes) fused to GST, expressed in bacteria. This passive immunizaticn lead to improved glucose uptake in an animal obesity/ Type H diabetes model (Steppan et al.. Nature 409: 307-12 (2001)).<br>
Resistin (Res) is a 114 aa peptide hormone of approximately 12 KD. It contains 11 cysteine of which the most N-tenninal one was shown to be responsible for the dimerisation of the protein and the other 10 are beheved to be involved in intramolecular disulfide bonds (Banerjee and Lazar, X Biol. Chem. 276: 25970-3 (2001)). Mutation of the first cysteine to alanine abolishes the dimerisation of mRes.<br>
It was shown, that mRes with a FLAG tag at its C-tenninus still rranains active in an animal model (Steppan et al.. Nature 409: 307-12 (2001)), similarly a C-tenninally HA taged (Haemagglutinin tag) version of resistin was shown to be active in a tissue culture assay (Kim et al, J. Biol. Chem. 276: 11252-6 (2001)), suggesting that the C-tenninus is not veay sensitive to introduced modifications. Thus, in a prafened embodiment, the inventive composition comprises an amino-acjd linker containing a cysteine residue as second attachment site and being fused at the C-terminus of the resistin sequence, in further prefened embodiments, the amino acid linker containing a fi:ee cysteine is fused to the N-terminus of the sequence corresponding to the sequence of the processed protein, or inserted at the N-traminus of the sequence of the mature form of the protein, C-terminally of the signal peptide.<br>
For a preferred embodiment of the present invention, MRes or huRes may also be expressed as Fc fusion molecules with a protease cleavage site inserted between Resistin and the Fc part of the construct, preferably C-terminaUy of one or more cysteine residue of the hinge region of the Fc part of the fusion protein in a eukaryotic expression system, or more preferably according to the descriptions and disclosures of EXAMPLE 2. Cleavage of the fusion protein releases Resistin additionally ccnnising either an aminoacid linker containing a cysteine residue as described in EXAMPLE 2, or part or all of the hinge region of the Fc part of the fusion protein which comprises a cysteine residue at its C-terminus, which is suitable for coupling to VLPs or Pili. The human Resistin sequence is shown in SEQ ID No: 249 (Accession AF323081). The mouse sequence is shown in SEQ ID No: 250 (Accession AF323080). A favored embodiment of the invention is human resistin protein fused at its C-terminus to an amino acid linker containing a cysteine residue. Human resistin construct can be genorated according to the teachings disclosed in EXAMPLE 2, and by comparing murine and human Resistin sequences m a protein<br><br>
7<br>
sequence alignment to identify the part of the sequence of human Resistin to be cloned in the vectors described in EXAMPLE 1 and EXAMPLE 2 according to the teachings of EXAMPLE 2, or in other suitable expression vectors. Example of human resistin constructs suitable for generating compositions of the inventions are human resistin-C-Xa: (SEQ ID NO:325), human resistin-C-EK: (SEQ ID NO:326) and human resistin-C: (SEQ ID NO:327).<br>
Human Resistin constructs so genraated are a preferred embodiment of the invention. Vaccination against Resistin using the aforementioned compositions of the invention may thus provide a way of treating Type II Diabetes and obesity.<br>
In another embodiment the antigenic determinant is Lymphotoxin-p. Immunization against lymphotoxin-p may be useful in treating Prion mediated disease. Scrapie (a prion-mediated disease) agent repMcation is believed to take mainly place in lymphoid tissues and was shown to depend on prion-protein expressing follicular dendritic cells (FDCs) (Brown er aZ., Atoure Afed. 11: 1308-1312 (1999)). It was subsequently shown that mice lacking flmctional follicular dendrite cells show an impaired prion rephcation in spleens and a (small) retardation of neuroinvasion (Montrasio et al.. Science 288: 1257-1259 (2000)). This was achieved by injecting the mice with a soluble lymphotoxin-P receptor-Fc-fusion protein (LTR-Fc). This soluble receptor construct inhibits the development of FDCs by interfering with the crucial interaction of lymphotoxin-p on T, B or NK cells with the lymphotoxin-p receptor on the FDC precursor cells. TTius, vaccination against lymphotoxin-P (also called TNFy) may provide a vaccine for treatment or prevention of Creutzfeld-Jakob (variant form) or other prion-mediated diseases and thus prevent prion replication and neuroinvasion.<br>
Immunization against Lymphotoxin-P may also provide a way of treating diabetes. Transgene expression of soluble LTR-Fc fusion protein in nonobese diabetic NOD mice blocked diabetes development but not insulitis (Ettingar et al., J. Exp. Med. 193: 13330 K (2001)). Wu et al. (J. Exp. Med. 193: 1327-32 (2001)) also used NOD mice to study the involvement of lymphotoxin-p, but instead of transgenic animals they did inject the LTPR-Fc fusion protein. They saw a strong inhibition of diabetes development and inhibition of insulitis. Most interestingly, they could even reverse preexisting insulitis by the fusion protein treatment In the pancreas the formation of lymphoid follicular structures could thus be reversed. Vaccination against lymphotoxin-P may thus provide a way of treatment against tje-I diabetes.<br>
The sequence of the extracellular domain of human lymphotoxin-P is shown in SEQ ID No: 250 (TNFC_human) and the sequence of the extracellular domain of murine lymphotoxin-p is shown in SEQ ID No: 251 (TNFC_mouse).<br><br>
In a further preferred embodiment, the inventive composition comprises an amino acid linker containing a free cysteine and being added to the N-terminus of the sequence corresponding to the processed form of lymphotoxin-P, or inserted between the N-teiminus of the sequence corresponding to the mature form of the protein, and the signal peptide, C-tenninally to the signal peptide. Jxi further preferred embodiments of the invention, the extracellular part of lymphotoxin-P is expressed as a fusion protein either with Glutathion-S-transferase, fused N-terminally to lymphotoxin-P, or with a 6 histidine-tag followed by a myc-tag, fused again N-terminally to the extracellular part of lymphotoxin-p. An amino acid spacer containing a protease cleavage site as well as a Unker sequence containing a free cysteine as attachment site, C-temiinaUy to the protease cleavage site, are fused to the N-terminus of the sequence of the extracellular part of lymphotoxin-P. Preferably, the extracellular part of iynhotoxin-p consists of fragments corresponding to amino acids 49-306 or 126-306 of lymphotoxin-P. These specific consitions of the invention may be cloned and expressed in the pCEP~Pu eukaryotic vector. In further preferred embodiments, the inventive compositions comprise an aminoacid Unker containing a free cysteine residue suitable as second attachment site, and being fused to the C-tenninus of lymphotoxin-P or lymphotoxin-p fragments. In a particularly favored embodiment, the amino acid sequence LACGG, compiising the amino acid linker ACGG which itself contains a cysteine residue for coupling to VLPS and Pili is fused to the N-tenninus of the extracellular part of lymphotoxin-p ; or of a fragment of the extracellular part of lymphotoxia-p, yielding the proteins human C-LT* 49.306 (SEQ ID NO:346) and human C-LT* 126-306 (SEQ ID NO:347) after cleavage with enterokinase of the corresponding fusion proteins expressed rather in vector pCEP-SP-GST-EK or vector pCP-SP-his-myc-EK as described in EXAMPLE 3.<br>
In a prcfdied endxxUment, the antigen or antigenic determinant is the prion protein, fragments thereof and in particular ptides of the imon proteb. In one embodiioent the prion proldn is the hnman prion protein. Guidance on how to modify human prion jffolein toi association with the cpre particle is given tiirougbout the application and in particular in EXAMPLE 7. Mouse prion protem constrocts are disclosed, and human prion protein constructs can also be generated and have, for fjcraplf-, the sequence of SEQ IP NO: 348, Further constructs comprise flic whole human prion prot sequence, and other Â£ragioents of the buman prion protein, "Â»Uch are furthn conqiodtion of the invention. Immunization against prion protein may provide a way of beatment or prevention of CrcutzMd-Jakob (variant form) or otha prion-mcdiated diseases. Immunization using the consilions of the invention coo:q&gt;rising the prion protdn may iovide a way of treatment against prion mediated diseases in other animals, and (he conesponding sequences of bovine and sbe prion protein constructs arc given in SEQ ID NO:349 and SEQ ID NO:350, respectively. Ihe peptides of the human prion protein corresponding to the murine peptides described in EXAMPLE 8, and of amino acid sequence CSAMSRPIIHFGSDYEDRYYRENMHR<br><br>
"human cprplong") and CGSDYEDRYYRENMHR ("human cprpshort") lead to preferred embodiments of the invention. These peptides comprise an N-tenninal cysteine residue added for coupling to VLPs and Pili. Conesponding bovine and sheep peptides are CSAMSRPUHFGNDYEDRYYRENMHR ("bovine cprplong") andCGNDYEDRYYRENMHR ("bovine cprpshort") CSAMSRPLTHFGNDYEDRYYRENMYE ("sheep cprplong") and CGNDYEDRYYRENMYR ("sheep cprpshort"), all leading to embodhnents of the invention.<br>
bi a further preferred embodiment of the invention, the antigenic determinant is tumor necrosis factor a (TNFHX), fragments thereof or peptides of TNF-a. In particular, peptides or fragments of TNF-a can be used to induce a self-specific immune response directed towards the whole protein by immunizing a human or an animal with vaccines and compositions, respectively, comprising such peptides or fragments in accordance with the invention. Preferably, VLPs, bacteriophages of bacterial pili are used as core particle, to which TNF-a, peptides or fragments thereof are attached according to the invention.<br>
The following murine peptides are the murine homologs to human peptides that have been shown to be bound by antibodies neutralizing the activity of TNF-ot(Yone et al J. Biol. Chem.270: 19509-19515) and were, in a further preferred embodiment of the invention, modified with cysteine residues for coupling to VLPs, bacteriophages or bacterial pili.<br>
MuTNFa peptide: the sequence CGG was added at the N-terminus of the epitope consisting of amino acid residues 22-32 of mature murine TNF-cc: CGGVEEQIWLSQR.<br>
3"TNF n peptide; the sequence GGC was fused at the C-terminus of the epitope consisting of amino acid residues 4-22 of mature murine TNF-a and glotamine 21 was mutated to glycine. The sequence of the resulting peptide is: SSQNSSDKPVAHWANHGVGGC.<br>
5"n n peptide: a cysteine residue was fused to the N-terminus of the epitope consisting of amino acid residues 4-22 of mature murine TNF-a and<br><br>
glutamine 21 was mutated to glycine. The sequence of the resulting peptide is: CSSQNSSDKPVAHWANHGV.<br>
The corresponding human sequence of the 4-22 epitope is SSRTPSDKPVAHWANPQAEGQ. like for the murine sequence a cysteine is, preferably, fused at the N-tenninus of the epitope, or the sequence GGC is fused at the C-terminus of the epitope for covalent coupling to VLPs, bacteriophages or bacterial pili according to the invention. It is, however, within the scope of the present invention that other cysteine containing sequences are fused at the N- or C-termini of the epitopes. In general, one or two glycine residues are preferably inserted between the added cysteine residue and the sequence of the epitope. Other amino acids may, however, also be inserted instead of glycine residues, and these amino acid residues will preferably be small amino acids such as serine.<br>
Tlie human sequence corresponding to amino acid residues 22-32 is QLQWLNRRANA. Preferably, the sequence CGG is fused at the N-terminus of the epitope for covalent coupling to VLPs or bacterial pili according to the invention. Other TNF-cc_epitopes suitable for using in the present invention have been described and are disclosed for example by Yone el a/. (J.Biol. Chem.270: 19509-19515).<br>
The invention further includes compositions which contain mimotopes of the antigens or antigenic determinants described herein.<br>
The specific composition of the invention comprises an antibody or preferably an antibody fragment presented on a virus-like particle or pjlus for induction of an immune rraponse against said antibody. Antibodies or antibody fragments which are produced by lymphoma cells, may be selected for attachment to the virus-like particle and immunization, in order to induce a protective immune response against the lymphoma.<br>
In other further embodiments, an antibody or antibody fragment mimicking an antigen is attached to the particle. The mimicking antibody or antibody fragment may be generated by immunization and subsequent isolation of the mimicking antibody or antibody fragment by any known method known to the art such as e.g. hybridoma technology (Gherardi, E. et al., J. Inmiunol. Metiiods 126; 61-68 (1990)), phage display (Harrison et al. Methods Enzymol. 267: 83-109 (1996)), ribosome display (Hanes, J. et al, Nat. Biotedmol 18: 1287-1292 (2000), yeast two-hybrid (Visintin, M.etal.,Proc. Natl Acad. Sci. USA96: 11723-11728 (1999)), yeast surface display (Boder, ET. &amp; Wittrup, KD. Methods. Enzym. 328:430-444 (2000)), bacterial surface display (Daugherty, PS. et al.. Protein Eng. 12: 613-621 (1999)). The mimicking<br><br>
antibody may also be isolated from an antibody library or a nmVe antibody library using methods known to the art such as the methods mentioned above, for example.<br>
hi a further embodiment, an antibody recognizing the combining site of another antibody, i.e. an anti-idiotypic antibody, further called the immunizing antibody, may be used. The antibody recognized by the and-idiotypic antibody will be further referred to as the neutralizing antibody. Tims, by immunizing against the anti-idiotypic antibody, molecules wilh the specificity of fee neutralizing antibody are genered in situ; we will further refer to these generated antibodies as the induced antibodies. In anothest preferred embodiment, the immunizing antibody is selected to interact with a Ugand molecule of the target molecule against which immunization is seeked. The Egand molecule nmy be any molecule interacting with the farg molecule, but will preferentially interact with the site of the target molecule against which antibodies should be generated for inhibition of its function. The ligand molecule may be a natm ligand of the target molecule, or may be any engineered, designed or isolated Mgand having suitable binding properties.<br>
The immunizing antibodies may be of human origin, such as isolated from a naive or inmiune human antibody library, or may have been isolated from a Ubrary generated from another animal source, for example of murine origin.<br>
Coupling of the antibody or antibody fragment to the VLP cff pilus is achieved either by limited reduction of exposed disulfide bridges (for example of the interchain disulfide bridge between CHI and CK or C in a Fab fragment) or by fusion of a linker containing a fi-ee cysteine residue at the C-terminus of the antibody or antibody fragment. In a further embodiment, a linker containing a free cysteine residue is fused to the N-terminus of the antibody or antibody fragment for attachment to a VIP or pilus protein.<br>
A number of vaccine compositions which employ mimotopes are known in the art, as are methods for generating and identifying mimotopes of particular epitopes. For example, Amon et al.. Immunology J0J:555-562 (2000), the entire disclosure of which is incorporated herein by reference, describe mimotope peptide-based vaccines against Schistosoma mansoni. The mimotopes uses in these vaccines were obtained by screening a solid-phase 8mer random peptide library to identify mimotopes of an epitope recognized by a protective monoclonal antibody against Schistosoma mansoni. Similarly, Olszewska et al.. Virology 272:98-105 (2000), the entire disclosure of which is incorporated hain by refCTence, describe the<br><br>
identification of synthetic peptides which mimic an epitope of the measles virus fusion protein and the use of these peptides for the immunization of mice. In addition, Zuercherera/., Â£ur. /. Immunol. 50:128-135 (2000), the entire disclosure of which is incorporated herein by reference, describe compositions and methods for oral anti-IgE immunization using epitope-displaying phage. In particular, epitope-displaying M13 bacteriophages are employed as carriers for an oral anti-IgE vaccine. The vaccine compositions tested contain mimotopes and epitopes of the monoclonal anti-IgE antibody BSW17.<br>
The invention thus includes vaccine compositions which contain mimotopes that elicit immunological responses against particular antigens, as well as individual mimotope/corc particle conjugates and individual niimotope/non-naturaUy occurring molecular scaffold conjugate which make up these vaccine coirositions, and the use of these vaccine compositions to elicit immunological responses against specific antigens or antigenic determinants. Mimotopes may also be polypeptides, such as anti-idiotypic antibodies. Therefore, in a further prefened embodiment of the invention, the antigen or antigenic determinant is an anti-idiotypic antibody or anti-idiotypic antibody firagment.<br>
The invention further includes compositions which contain mimotopes of the antigens or antigenic determinants described herein.<br>
Mimotopes of particular antigens may be generated and identified by any number of means including the screening of random peptide phage display libraries (see, e.g., PCX Publication No. WO 97/31948, the entire disclosure of which is incorpOTated herein by reference). Screening of such libraries will often be perfomied to identify peptides which bind to one or more antibodies having specificity for a particular antigen.<br>
Mimotopes suitable for use in vaccine compositions of the invention may be linear or circular peptides. Mimotopes which are linear or circular peptides may be linked to non-natural molecular scaffolds or core particles by a bond which is not a peptide bond.<br>
As suggested above, a number of human IgE mimotopes and epitopes have been identified which elicit immunological responses against human IgE molecules. (See. eg., per Publication No. WO 97/31948.) Thus, in certain embodiments, vaccine compositions of the invention include compositions which elicit an immunological response against immunoglobin molecules (e.g., IgE molecules).<br>
Peptides which can be used to elicit such immunological responses include proteins, protein subunits, domains of IgE molecules, and mimotopes which are capable of eliciting production of antibodies having specificity for IgE molecules.<br><br>
Generally, portions of IgE molecules used to prepare vaccine compositions will be derived from IgE molecules of the species from which the composition is to be administered. For example, a vaccine composition intended for administration to humans will often contain one or more portions of the human IgE molecule, and/or one or more mimotopes which are capable of eliciting immunological responses against human IgE molecules.<br>
In specific embodiments, vaccine compositions of the invention intended for administration to humans wiU contain at least one portion of the constant reon of the IgE heavy chain set out in SEQ ID NO; 176; Accession No. AAB59424 (SEQ ID NO: 176). In more specific embodiments, IgE peptides used to prepare vaccine compositions of the invention comprise, or alternatively consist of, peptides having the foUowing amino acid sequences: CGGVNLTWSRASG (SEQ ID NO:178).<br>
In additional specific embodiments, vaccine compositions of the invention will contain at least one mimotope which is capable of eliciting an immune rwponse that results in the production of antibodies having specificity for a particular antigen.<br>
Examples of mimotopes of IgE suitable for use in the preparation of vaccine<br>
compositions of the invention include peptides having the following amino<br>
acid sequences:<br><br><br><br>
The invention provides novel compositions and methods for the construction of ordered and repetitive antigen arrays. As one of skill in the art would know, the conditions foi the assembly of the ordered and repetitive antigen array depend to a large extent on the specific choice of the first attachment site of the non-natural fflolecuiar scaffold and the specific choice of the second attachment site of the antigen or antigmic detenninant. Thus, practitioner choice in the design of the composition (i.e., selection of the first and second attachment sites, antigen and non-natural molecular scaffolcD will determine the specific conditions for the assembly of the AlphaVaccine particle (the ordered and repetitive antigen array and non-natural molecular scaffold combined). Information relating to assembly of the AlphaVaccine particle is well within the working knowledge of the practitioner, and numerous references exist to aid the practitioner (e.g., Sambrook, J. et al., eds.. MOLECULAR CLONING, A LABORATORY MANUAL, 2nd- edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. (1989); Ausubel, R et al., eds.. CURRENT PRCfTOCOLS IN MOLECULAR BIOLOGY, John H. Wiley &amp; Sons, Inc. (1997); Celis, J., ed., CELL BIOLGY, Academic Press, 2" edition, (1998); Harlow, E. and Lane, D., "Antibodies: A Laboratory Manual," Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. (1988), all of which are incorporated herein by reference.<br><br>
In a specific embodiment of the invention, the /tW and FOS leucine zipper protein domains are utilized for the first and second attachment sites of the invention, respectively. In the preparation of AlphaVaccine particles, antigen must be produced and purified under conditions to promote assembly of the ordered and repetitive antigen array onto the non-natural molecular scaffold. In the particular JUN/FOS leucine zipper protein domain embodiment, the FOS-antigsn or F05-antigemc determinant should be treated with a reducing agent (e.g., Dithiothreitol (DTT)) to reduce or eliminate the incidence of disulfide bond formation (Example 15).<br>
For the preparation of the non-natural molecular scaffold {Le., recombinant Sinbis virus) of the JUNIFOS leucine zipper protein domain embodiment, recombinant E2-JUN viral particles should be concentrated, neutralized and treated with reducing agent (see Exanle 16).<br>
Assembly of the ordered and repetitive antigen array in the JUN/FOS embodiment is done in the presence of a redox shuffle. "E2-JUN viral particles are combined with a 240 fold molar excess of FOS-antigen or FOLS-antigenic determinant for 10 hours at 4"*C. Subsequently, the AlphaVaccine particle is concentrated and purified by chromatography (Example 16).<br>
1	In another embodiment of the invention, the coupling of the non-<br>
natural molecular scaffold to the antigen or antigenic determinant may be accomplished by chemical cross-linking. In a specific embodiment, the chemical agent is a heterobifunctional cross-linking agent such as e-maleimidocaproic acid N-hydroxysuccinimide ester (Tanhnori et al~, J. Pharm. Dyn. 4".812 (1981); Fujiwara zt ai, J. Immunol. Meth. 45:195 (1981)), which contains (1) a succinimide group reactive with amino groups and (2) a maleimide group reactive with SH groups. A heterologous protein or polypeptide of the first attachment site may be engineered to contain one or more lysine residues that will serve as a reactive moiety for the succinimide portion of the heterobifunctional cross-linking agent. Once chMcdcaHy coupled to the lysine residues of the heterologous protein, the maleimide group of the heterobifunctional cross-linldng agent will be available to react with the SH group of a cysteine residue on the antigen or antigenic determinant Antigen or antigenic determinant preparation in this instance may require the engineering of a cysteine residue into the protein or polypeptide chosen as the second attachment site so that it may be reacted to the free maleimide function on the cross-linking agent bound to the non-natural molecular scaffold first attachment sites. Tlius, in such an instance, the heterobifunctional cross-linking agent binds to a first attachment site of the non-natural molecular scaffold and connects the scaffold to a second binding site of the antigen or antigenic determinant<br><br>
3.        Compositions, Vaccines, and the Administration Thereof, and Methods of Treatment<br>
The invention provides vaccine compositions which may be used for preventing and/or attenuating diseases or conditions. The invention further provides vaccination methods for preventing and/or attenuating diseases or conditions in individuals.<br>
In one embodiment, the invention provides vaccines for the prevention of infectious diseases in a wide range of species, particularly mammalian species such as human, monkey, cow, dog, cat, horse, pig, etc. Vaccines may be designed to treat infÂ«:tions of viral etiology such as HIV, influenza. Herpes, viral hepatitis, Epstein Bar, polio, viral encephalitis, measles, chicken pox, etc.; or infections of bacterial etiology such as pneumonia, tuberculosis, syphilis, etc.; or infections of parasitic etiology such as malaria, trypanosomiasis, leishmaniasis, trichomoniasis, amoebiasis, etc.<br>
In another embodiment, the invention provides vaccines for the pmvKition of cancer in a wide range of species, particularly mammalian species such as human, monkey, cow, dog, cat, horse, pig, etc. Vaccines may be designed to treat all types of cancer: lymphomas, carcinomas, sarcomas, melanomas, etc.<br>
IJa another embodiment of the invention, compositions of the invention may be used in the designof vaccines for the treatment of allK"gies. Antibodies of the IgE isotype are important conqwnents in allergic reactions. Mast cells bind IgE antibodies on their surface and release histamines and other mediators of allergic response upon bintUng of specific antigen to the IgE molecules bound on the mast cell surface. Inhibiting production of IgE antibodies, therefore, is a promising target to protect against allergies, lliis should be possible by attaining a desired T helper cell response. T helper cell responses can be divided into type 1 (THI) and type 2 CTH2) T helper cell responses (Romagnani, Immunol Today iS:263-266 (1997)). THI cells secrete interferon-gamma and other cytokines which trigger B cells to produce IgGl-3 antibodies. In contrast, a critical cytokine produced by TH2 cells is TLA, which drived B cells to produce IgG4 and IgE. In many experimental systems, the development of THI and TH2 responses is mumally exclusive sinceTnl cells suppress the induction of TH2 cells and vice versa. Thus, antigens that trigger a strong TRI response simultaneously suppress the development of TH2 responses and hence fee production of IgE antibodies, interestingly, virtually all viruses induce a THI response in the host and fail to trigger the production of IgE antibodies (Coutelier et al, J. Exp. Med. 165:64-69 (1987)).   This isotype pattern is not restricted to live<br><br>
viruses but has also been observed for inactivated or recombinant viral particles (Lo-Man et al., Eur. J. Immunol 28:1401-1407 (1998)). TTius, by using the processes of the invention (e.g., AlphaVaccine Technology), viral particles can be decorated with various alleiens and used for immunization. Due to the resulting "viral structure" of the allergen, a THI response will be elicited, "protective" IgGl-3 antibodies will be produced, and the production of IgE antibodies which cause allergic reactions will be prevented. Since the allergen is presented by viral particles which are recognized by a different set of helper T cells than the allergen itself, it is likely that the allergen-specific IgGl-3 antibodies will be induced even in allergic individuals harboring preÂ¬existing TH2 cells specific for the allergen. The presence of hi concentrations of IgG antibodies may prevent binding of allergens to mast cell bound IgE, thereby inhibiting the release of histamine. Thus, presence of IgG antibodies may protect from IgE mediated allergic reactions. Typical substances causing allergies include: grass, ragweed, birch or mountain cedar pollens, house dust, mites, animal danders, mold, insect venom or drugs (e.g.., penicillin). Tlius, immunization of individuals with allCTgen-decorated viral particles should be beneficial not only before but also after the onset of allergies.<br>
hi specific embodiments, the invention provides methods for preventing and/or attenuating diseases or conditions which are caused or exacerbated by "self" gene products (e.g., tumor necrosis factors), i.e. "self antigens" as used herein. In related embodiments, the invention provictes methods for inducing immunological responses in individuals which lead to the production of antibodies that prevent and/or attenuate diseases or conditions are caused or exacerbated by "self gene products. Examples of such diseases or conditions include graft versus host disease, IgE-mediated allergic reactions, anhylaxis, adult respiratory distress syndrome, Crohn"s disease, allergic asthma, acute lymphoblastic leukemia (ALL), non-Hodgkin"s lymphoma (NHL), Graves" disease, inflammatory autoimmune diseases, myasthenia gravis, systemic lupus erythematosus (SLE), immunoproliferative disease lymphadenopathy (IPL), angioimmunoproUferative lymphadenopathy (AIL), immunoblastive lymphadenopathy (IBL), rheumatoid arthritis, diabetes, multiple sclerosis, osteoporosis and Alzheimer"s disease.<br>
As would be understood by one of ordinary skill in the art, when compositions of the invention are administered to an individual, they may be in a composition which contains salts, buffers, adjuvants, or other substances which are desirable for improving the. efficacy of the composition. Examples of materials suitable for use in preparing pharmaceutical compositions are provided in numerous sources including REMINGTON"S PHABMACEUTFCAL ScaENCES (Osol, A, ed. Mack PubhshingCo.,(1990)).<br><br>
Compositions of the invention are said to be "pharmacologically acceptable" if their administration can be tolerated by a recipient individual. Further, the compositions of the invention will be administered in a "thapeutically effective amount" {i.e., an amount that produces a desired physiological effect).<br>
The compositions of the present invention may be adininistered by various methods known in the art, but will normally be administered by injection, infusion, inhalation, oral administration, or other suitable physical methods. The compositions may alternatively be administered intramuscularly, intravenously, or subcataneously. Components of compositions for administration include staile aqueous (e.g., {diysiologicai saline) or non-aqueous solutions and suspensions. Examples of non-aqueous solvents are propylene glycol, polyethylene glycol, vegetable oils such as olive oil, and injectable organic esters such as ethyl oleate. Carriers or occlusive dressings can be used to increase skin permeability and enhance antigen absorption.<br>
Prion-mediated diseases are an increasing threat for society. Specifically, prion-induced BSE in cattle represents a disease that has long been neglected and may affect a great number of animals throuout Europe. Moreover, a variant form of CJD is attributed to infection of humans after consumption of meat of prion-infected cattle. Althou the number of infected people has been relatively low so far, it seems possible that the disease may become epidemic. However, long-term prognosis for the development of vCJD may be particular difficult, since incubation times between infection and overt disease are very long (an estimated 10 years).<br>
Prions are cellular proteins existing in most mammahan species. Prion proteins exist in two forms, a normally folded form that is usxy present in healthy individuals (PrP" and a misfolded form that causes disease (Ptp). The current prion hypotheses postulates that the misfolded prion form Prp can catalyse the refolding of healthy prion PrP into disease causing Prp " (A. Aguzzi, Haematologica 85, 3-10 (2000)). \B. some rare instances, this transition inay also occur spontaneously, causing classical CJD in humans. Some mutations in PrP are associated with an increase in this spontaneous transition, causing the various forms of familial CJD. However, Pip* may also be infectious and may be transmitted by blood transfusion or via the food chain. The latter form of prion mediated disease is known as Kuru Kuru and used to occur in human cannibals. However, since species that are feeding on their own individuals are not abundant, this form of orally transmitted disease was too rare to be documented for other species.<br>
The massive feeding of cows with beef-products throughout Europe now<br><br>
changed the situation and numbers of cows infected with a transmissible form of BSE-causing Pip , dramatically increased in recent years, afflicting hundreds of thousands of cows. This sudden appearance of massive numbers of BSE-diseased cows caused great fear in the human population that a similar disease may be induced in humans. Indeed, in 1996, the first case of a variant form of CJD was reported that could be attributed to the consumption of Prp"" infected beef. Until now, this fear has further increased, since the number of infected humans has constantly increased during the following years and no cure is in sight. Moreover, since sheep succumb to a prion-mediated disease called scrie and ance other mammalian species can be infected with Pip<br>
Experimentally, it is possible that BSE-like diseases may occur also in other species. The mechanism of prion transmission has been studied in great detail. It is now clear that prions first replicate in the lymphoid organs of infected mice and are subsequently transported to the central nervous system. Follicular dendritic cells (FDCs), a rare cell population in lymphoid organs, seems to be essential for both replication of prion proteins in the lymphoid organs and transport into the central nervous system (S. Brandner, M. A. Klein, A. Aguzzi, Transfus Clin Biol 6, 17-23 (1999); F. Montrasio, et al.. Science 288, 1257-9 (2000)). FDCs are a poorly studied cell type but it is now clear that they depend upon the production of lymphotoxin and/or TNF by B cells for their development (F. Mackay, J. L, Browning, Nature 395, 26-27 (1998)). Indeed, mice deficient for lymphotoxin do not exhibit FDCs (M S. Matsumoto, et al.. Science 264. 703-707 (1996)). Moreover, they fail to be productively infected with prions and do not succumb to disease. Jn addition to FDCs, antibodies may also play a role in disease progression (S. Brandner, M. A. Klein, A. Aguzzi, Transfus Clin Biol 6, 17-23 (1999)).<br>
Recently, it was shown that blocking the LTb pathway using a Ltb receptor Fc fusion molecule not only eliminates FDCs in mice but also blocks infection witii PrP (F. Monttasio, et al., Science 288, 1257-9 (2000). Thus, a vaccine that induces antibodies specific for LTb or its receptor may be able to block transmission of PrP from one individual to another or fix&gt;m the periphery to the central nervous system.<br>
However, it is usually difficult if not impossible to induce antibody responses to self-molecules by conventional vaccination. One way to improve the efficiency of vaccination is to increase the degree of repetitiveness of the antigen applied: Unlike isolated proteins, viruses induce prompt and efficient immune responses in the absence of any adjuvants both with and without T -cell help (Bachmann  &amp; Zinkemagel,Ann. Rev.  Immunol:   15:235-270 (1991)).  Although<br><br>
viruses often consist of few proteins, they are able to trigger much stronger immune responses than their isolated components. For B-cell responses, it is known that one crucia] factor for ihe immunogenicity of viruses is the repetiliveness and order of surface epitopes. Many viruses exhibit a quasi- crystalline surface that displays a regular array of epitopes which efficiently crosslinks epitope-specific immunoglobulins on B cells (Bachmann &amp; Zinkemagel, Immunol. Today 17:553-558 (1996)). Tliis crosslinking of surface immunoglobulins on B cells is a strong activation signal that directly induces ceU- cycle progression and the production of IgM antibodies. Further, such triggered B cells are able to activate T helper cells, which in Uira induce a switch fcom IgM to IgG antibody productioa in B cells and the generation of long-lived B cell memory - the goal of any vaccination (Bachmann &amp; Zinkemagel, Ann. Rev. Immunol. 15:235-270 (1997)). Viral structure is even linked to the generation of anti-antibodies in autoimmune disease and as a part of the natural response to paUiogens (see Fehr, T., et al., J Exp. Med. 185:1785-1792 (1997)). TTius, antibodies presented b)" a highly organized viral surface are able to induce strong anti-antibody responses.<br>
ITie immune system usually fails to produce antibodies against self-derived stmctures. For soluble antigens present at low concentrations, this is due to tolerance at the Hi cell level. Under these conditions, coupling the self-antigen to a carrier that can deliver T help may break tolerance. For soluble proteins present at high concentrations or membrane proteins at low concentration, B and Th cells may be tolerant. However, B cell tolerance may be reversible (anergy) and can be broken by administration of the antigen in a hiy organized fashion coupled to a foreign carrier (Bachmann &amp; Zinkemagel, Ann. Rev. Immunol. 15:235-270 (1997). Thus, LTb, LTa or LTb receptor as highly organized as a virus, a virus like particle or a bacterial pilus may be able to break B cell tolerance and to induce antibodies specific for these molecules.<br>
The present invention is related to the fields of molecular biology, virology, immunology and medicine. The invention provides a method that facilitates induction of antibodies specific for endogenous lymphotoxin (LT)b, LTa or LTb receptor. The invention also provides a process for producing an antigen or antigenic determinant that is able to elicit antibodies specific for LTb, LTa or LTb receptor which is useful for the prevention and therapy of prion-mediated diseases such as variant Creutzfeld-Jacob disease (vCJD) or bovine spongioform encephalopalliy (BSE) and elimination of lymphoid organ like strucmres in autoimmune diseased tissues.<br>
TTie object of the invention is to provide a vaccine that is able to induce antibodies specific for LTb, LTa or LTb receptor thereby eliminating FDCs<br><br>
from lymphoid organs. This treatment may allow preventing infection with PrP" or spread of PrP" from the periphery to the central nervous system. In addition, this treatment blocks generation of lymphoid organ like structures in organs targeted by autoimmune disease and may even (Ussolve such existing structures, ameliorating disease symptoms.<br>
LTb, LTa or LTb receptor or fragments thereof are coupled to a protein carrier that is foreign to the host In a preferred embodiment of the invention, LTb, LTa or LTb receptor or fragments thereof will be coupled to a highly organized structure in order to render these molecules highly repetitive and organized The hiy organized structure may be a bacterial pilus, a virus like particle (VLP) generated by recombinant proteins of the bacteriophage QP, recombinant proteins of Rotavirus, recombinant proteins of Norwalkvirus, recombinant proteins of Alphavirus, recombinant proteins of Foot and Mouth Disease virus, recombinant proteins of Retrovirus, recombinant proteins of Hepatitis B virus, recombinant proteins of Tobacco mosaic virus, recombinant proteins of Flock House Virus, and recombinant proteins of human Papillomavirus. In order to optimize the three-dimensional arrangement of LTb, LTa or LTb receptor or fragments thereof on the highly organized structure, an attachment site, such as a chemically reactive amino-acid, is introduced into the highly organized structure (unless it is naturally there) and a binding site, such as a chemically reactive amino acid, will be introduced on the LTb, LTa or LTb receptor or fragments (unless it is naturally there). The presence of an attachment site on the highly organized structure and a binding site on the LTb, LTa or LTb receptor or fragments thereof will allow to couple these molecules to the repetitive structure in an oriented and ordered fashion which is essential for the induction of efficient B cell responses.<br>
In an equally preferred embodiment, the attachment site introduced in the repetitive structure is biotin that specifically binds sfreptavidin. Biotin may be introduced by chemical modification. LTb, LTa or LTb receptor or fragments thereof may be fiised or linked to streptavidin and bound to the biotinylated repetitive structure.<br>
Other embodiments of the invention include processes for the production of the compositions of the invention and methods of medical treatment using said compositions. It is to be understood that both the foregoing general description and the following detailed description are exemplary and explanatory only and are intended to provide further explanation of the invention as claimed.<br>
In addition to vaccine technologies, other enodiments of the invention are drawn to methods of medical treatment for cancer and allergies.<br><br>
All patents and publications referred to herein are expressly incoiporated by reference in their entirety.<br>
EXAMPLES<br>
Enzymes and reagents used in the experiments that follow included: T4 DNA ligase obtained from New England Biolabs; Taq DNA Polymerase, QIAprep Spin Plasraid Kit, QIAGEN Plasmid Midi Kit, QiaExH Gel Extraction Kit, QIAquick PCR Purification Kit obtained from QIAGEN; QuickPrep IvCcro mRNA Purification Kit obtained from Pharmacia; SupeiScript One-step RT PCR Kit, fetal calf serum (PCS), bacto-tryptone and yeast extract obtained from Gibco BRL; Oligonucleotides obtained from Microsynth (Switzerland); restriction endonucleases obtained from Boehringer Manhheim, New England Biolabs or MBI Fennentas; Pwo polymerase and dNTPs obtained from Boehringer Mannheim. HP-1 medium was obtained from Cell culture technologies (Glattbrugg, Switzerland). All standard chemicals were obtained from Fluka-Sigma-Aldrich, and all cell culture materials were obtained from TPP.<br>
DNA manipulations were carried out using standard techniques. DNA was prepared according to manufacturer instruction either from a 2 ml bacterial culture using the QIAprep Spin Plasmid Kit or from a 50 ml culture using the QIAGEN Plasmid Midi Kit. For restriction enzyme digestion, DNA was incubated at least 2 hours with the appropriate restriction enzyme at a concentration of 5-10 units (U) enzyme per mg DNA under manufacturer recommended conditions (buffer and temperature). Digests with more than one enzyme were performed simultaneously if reaction conditions were appropriate for all enzymes, otherwise consecutively. DNA fragments isolated for further manipulations were separated by electi-ophoresis in a 0.7 to 1.5% agarose gel, excised from the gei and purified with the QiaExH Gel Extraction Kit according to the instructions provided by the manufacturer. For ligation of DNA fragments, 100 to 200 pg of purified vector DNA were incubated overnight with a threefold molar excess of the insert fragment at 16Â°C in the presence of 1 U T4 DNA ligase in the buffer provided by the manufacturer (total volume: 10-20 fiY). An aliquot (0.1 to 0.5 fil) of the ligation reaction was used for transfonnation of E. coli XLl-Blue (Stratagene). Transformation was done by electroporation using a Gene Pulser (BioRAD) and 0.1 cm Gene Pulser Cuvettes (BioRAD) at 200 Ohm, 25 iF, 1.7 kV. After electioporation, the cells were incubated with shaking for 1 h in 1 ml S-O.B. medium (MiUer, 1972) before plating on selective S.O.B. agar.<br><br>
EXAMPLE 1 Modular eukaiyotic expression system for coupling of antigens to VLPs<br>
This system was generated in order to add various amino acid linker sequences containing a cysteine residue to antigens for chemical coupling to VLPs.<br>
A. Construction of an EBNA derived expression system encoding a cysteine-containing amino acid linker and cleavable Fc-Tag:<br>
pCep-Pu (Wuttke et al J. Biol. Chem. 276: 36839-48 (2001)) was digested with Kpn I and Bam HI and a new multiple cloning site was introduced with the annealed oligonucleotides PH37 (SEQ ID NO:270) and PH38 (SEQ ID NO:271) leading to pCep-MCS.<br>
A modular system containing a free cysteine flanked by several glycines, a protease cleavage site and the constant regjon of the human IgGl was generated as follows. pSec2/Hygro B (Invitrogen Cat. No. V910-20) was digested with Bspl20I and Hind HI and ligated with the annealed oligonucleotides SU7 (SEQ ID NO:278) and SU8 (SBQ ID NO:279) leading to construct pSec-B-MCS. pSec-B-MCS was then digested with Nhe I and Hind HI and ligated with the annealed oligonucleolides PH29 (SEQ ID NO:264) and PH30 (SEQ ID NO:265) leading to constmct pSec 29/30. The construct pSec-FL-EK-Fc* was generated by" a three fragment ligation of the following fragments; first pSec 29/30 digested with Eco RI and Hind HI, the annealed oligonucleotides PH31 (SEQ ID NO:266) and PH32 (SEQ ID NO. 267) and the Bgi I/EcoRI fragment of a plasraid (pSP-Fc*-Cl) containing a modified vMion of the human IgGl constant region (for details of the hu IgGl sequence see the sequence of the final construct pCep-Xa-Fc* see HG. lA-lC). The complete sequence of pCep-Xa-Fc* is given in SEQ ID NO:283. Tlie resulting construct was named pSec-FL-EK-Fc*. From this plasmid the linker region and the human IgGl Fc part was excised by Nhe I, Pme I digestion and cloned into pCep-MCS digested with Nhe I and Pme I leading to construct pCep-FL-EK-Fc*. Thus a modular vector, was created where the hnker sequence and the protease cleavage site, which are located between the Nhe I and Hind III sites, can easily be exchanged with annealed oligonucleotides, For the generation of cleavable fusion protein vectors pCep-FL-EK-Fc* was digested with<br><br>
Nhe I and Hind EI and the Factor Xa cleavage site N-tenninally flanked with amino acids GGGGCG was introduced with the annealed oligonuclotides PH35 (SEQ TO NO:268) and PH36 (SEQ ID NO:269) and the enterokinase site flanked n-terminally with GGGGCG was introduced with the annealed oligonucleotides PH39 (SEQ ID NO:272) and PH40 (SEQ ID NO;273) leading to die constructs pCep-Xa-Fc* (see FIG. lA) and pCep-EK-Fc* (see FIG. IB) respectively. The construct pCep-SP-EK-Fc* (see FIG. IC) which in addition contains a eukaryotic signal peptide was generated by a three fragment ligation of pCep-EK-Fc* digested Kpn V Bam HI, the annealed oiigos PH41 (SEQ ID NO-.274) and PH42 (SEQ ID NO:275) and the annealed oUgos PH43 (SEQ ID NO:276) and PH44 (SEQ ID NO:277).<br>
B.	Large Scale production of fiision proteins;<br>
For the large scale production of the different fusion proteins 293-EBNA cells (Invitrogen) were transfected with the different pCep expression plasmids with Lipofectamine 2000 reagent (life technologies) according to the manufacturer"s recommendation. 24-36 h post transfection the cells were split at a 1 to 3 ratio under puromycin selection (lp.g/ml) in DMEM supplemented with 10 % FCS. The resistant cells were then expanded in selective medium. For the harvesting of the fusion proteins the resistant cell population were passed onto poly-L-lysine coated dishes. Once the cells had reached confluence, they were washed 2 times with PBS and serum free medium (DMEM) was added to the plates. The tissue culture supernatant were harvested every 2 to 4 days and replaced with fresh DMEM medium during a period of up to one month. "Die harvested supematants were kept at 4 Â°C.<br>
C.	Purification of the fusion proteins:<br>
The recombinant Fc~fusion proteins were purified by affinity chromatography using protein A sepharose C!L-4B (Amersham Pharmacia Biotech AG). Briefly cbromatography coiranns were packed with 1-3 ml protein A resin and fee tissue culture supematants containing the recombinant proteins were applied to the column with a peristaltic pump at a flow rate of 0.5 - 1.5 ml/roiiv. The column was then washed with 20-50 ml PBS. Depending on the fusion protein the protease cleavage was performed .on the column or the protein was eluted as described below. Recombinant fusion proteins were eluted with a citrate phosphate buffer (pH 3.8)<br><br>
supplemented with 150 mM NaCl and the fractions containing the protein were pooled and concentrated with ultrafree centrifugal filters (Millipore).<br>
D. Protease cleavage of recombinant fusion proteins (Factor Xa, enterokinase):<br>
Eluted recombinant fusion proteins containing the enterokinase (EK) cleavage site were cleaved using the EKmax system (Invitrogen) according to the manufacturer"s recommendation. The cleaved Fc part of the fusion protein was removed by incubation with protein A. The enterokinase was then removed with the EK-Away system (Sivitrogen) according to the manufacturers recommendation. Similarly fusion proteins containing the factor Xa (Xa) cleavage site were cleaved using the restriction protease factor Xa cleavage and removal kit (Roche) according to the manufacturer"s recommendation. The cleaved Fc part was removed by incubation with protein A and the protease was removed with the streptavidin resio provided with the kit.<br>
The different fusion proteins were concentrated with ultraftee centrifugal filters (MUlipore), quantitated by UV spectrophotometrie and used for subsequent coupling reactions.<br>
FIG. lA-lC shows partial sequences of the different eukaryotic expression vectors used. Only the modified sequences are shown.<br>
FIG lA: pCep-Xa-Pc*: the sequence is shown from the Bam HI site onwards and different features are shown above the translated sequence. The arrow indicates the cleavage site of the factor Xa protease.<br>
FIG IB: pCep-EK-Fc*: the sequence is shown from the Bam HI site onwards and different features are shown above the translated sequence. Tlie arrow indicates the cleavage site of the enterokinase- "Hie sequence downstream of the Hind HI site is identical to the one shown in FIG 1 A.<br>
FIG. IC: pCep-SP-EK-Fc*: the sequence is shown from the begiiuiing of the signal peptide on and different features are shown above the translated sequence. The signal peptide sequence which is cleaved of by the signal peptidase is shown in bold The arrow indicates the cleavage site of the enterokinase. The sequence downstream of the Hind HI site is identical to the one shown in FIG lA.<br><br>
EXAMPLE 2 Eukaryotic expression and coupling of mouse resistin to VLPs and Pili<br>
A. Cloning of mouse Resistin:<br>
Total RNA was isolated from 60 mg mouse adipose tissue using a Qiagen RNeasy kit according to the manufacturer"s recommendation. TTie RNA was eluted in 40 111 H2O. This total RNA <br>a&amp; than used for IIK reverse transcription with an oligo dT primer using the "DiermoScript RT-PCR System (Life Technologies) according to the manufacturer"s recommendation. The sample was incubated at 50 Â°C for Ih, heated to 85 Â°C for 5 minutes and treated for 20 minutes at 37 "C with RNAseH.<br>
2 (il of the RT reaction were used for the PCR amplification of mouse resistin. The PCR was performed using Platinium TAQ (Life Technologies) according to the manufacturer"s recommendation using primers PH19 (SEQ ID NO:260) and PH20 (SEQ ID NO:261). Primra- PH19 (SEQ ID NO:260) conesponds to positions 58-77 and primer PH20 (SEQ ID NO:261) to positions 454-435 of the mouse Resistin sequence. The PCR mix was first denatured at 94 Â°C for 2 minutes and than 35 cycles were performed as follows: 30 seconds 94 Â°C, 30 seconds 56 "C and 1 minute 72 Â°C, at the end the samples were left for 10 minutes at 72 Â°C. The PCR fragment was purified and subcloned by TA cloning into the pGEMTeasy vector (hivitrogen) leading to pGEMT-mRes. In order to add appropriate restriction sites a second PCR was performed on pGHvU-raRes witii the primers PH21 (SEQ ID NO:262) and PH22 (SEQ ID NO. 263) primers using die same cycling program as described above. The forward primer (PH21 (SEQ ID NO:262)) contains a Bam HI site and nucleotides 81-102 of the mouse Resistin sequence- The reverse primer (PH22 (SEQ ID NO:263)) contains an Xba I site and nucleotides 426-406 of the mouse Resistin sequence. The indicated positions refer to the mouse resistin sequence Gene Accession No. AF323080. The PCR product was purified and digested witii Bam HI and Xba I and subcloned into pcmv-Fc*-Cl digested with Bam HI and Xba I leading to the construct pcmv-roRes-Fc*.<br>
The Resistin open reading frame was excised from pcmv-Res-Fc* by Bam HI/ Xba I digestion and cloned into pCep-Xa-Fc* and pCep-EK-Fc* (see EXAMPLE 1,<br><br>
section B) digested with Bam HI and Nhe I leading to the constructs pCep-mRes-Xa-Fc* and pCep-mRes-EK-Fc* respectively.<br>
B.	Production, purification and cleavage of Resistin<br>
pCep-mRes-Xa-Fc* and pCep-mRes-EK-Fc* constructs were then used 10 transfect 293-EBNA cells for the production of recombinant proteins as described in EXAMPLE 1, section B. The tissue culture supematants were purified as described in EXAMPLE 1, section C. TTie purified proteins were then cleaved as desoibed in EXAMPLE 1, section D. The resulting recombinant proteins were termed "lesistin-C-Xa" or "Res-C-Xa" and "resistin-C-EK" or "Res-C-EK" according to the vector used (see FIG. 2A and HG. 2B).<br>
FIG. 2A and FIG. 2B show sequence of recombinant mouse Resistin proteins used for expression and fiirther coupling. Res-C-Xa (FIG. 2A) and Res-C-EK (FIG. 2B) are shown as a translated DNA sequences. The resistin signal sequence which is cleaved upon protein secretion by the signal peptidase is shown in italic. The amino acid sequences which result form signal peptidase and specific protease (factor Xa or enterokinase) cleavage are shown bold. The bold sequences correspond to the actual protein sequence which was used for coupling, i.e. SEQ ID NO:280, SEQ ID NO:281. SEQ ID NO:282 corresponds to an alternative resistin protein construct, which can also be used for coupling to virus-like particles and pili in accordance with the invention.<br>
C.	Coupling of resistin-C-Xa and resistin-C-EK to Qp capsid protein<br>
A solution of 0.2 mi of 2 nml Qp capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.4 was reacted for 30 minutes with 5.6 fil of a solution of lOOmM SMPH (Pierce) in DMSO at 25 Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 Â°C. 8 pi of the dialyzed QP reaction mixture was then reacted with 32 \il of resistin-C-Xa solution (resulting in a final concentration of resistin of 0.39 mg/ml) and 13 |il of the QP reaction mixture was reacted with 27 jiJ resistin-C-EK solution (resultinginafinalconcentrationof resistin of 0.67 mg/ml) for four hours at 25 "Con a rocking shaker. Coupling products were analysed by SDS-PAGE (see FIG. 2C). An<br><br>
additional band of 24 kDa is present in the coupling reaction, but not in derivatized Qp and resistin, respectively. The size of 24 kDa corresponds to the expected size of<br>
24	kDa for the coupled product (14 kDa for Qp plus 10 kDa for resistin-C-Xa and<br>
resistin-C-EK, respectively).<br>
i<br>
FIG. 2C shows coupling results of resistin-C-Xa and lesistin-C-EK to Qp. Coupling products were analysed on 16% SDS-PAGE gels under reducing conditions. Lane 1: Molecular weight marker. Lane 2; resistin-C-EK before coupEng, Lane 3: resistin-C-EK- Qp after coupling. Lane 4: Qp derivatized. Lane 5: resistin-C-Xa before coupling. Lane 6: resistin-C-Xa- QP after coupling. Molecular weights of marker proteins are given on the left margin. Coupled band is indicated by the arrow.<br>
D.	Coupling of resistin-C-Xa and resistin-C-EK to fir capsid protein<br>
A solution of 0.2 ml of 2 mg/ml fr capsid protein in 20 mM Hepes, 150 jnM NaCl pH 7.4 is reacted for 30 minutes with 5.6 /il of a solution of lOOmM SMPH (Pierce) in DMSO at 25 "C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 roM Hepes, 150 mM NaCl, pH 7.4 at 4 Â°C. 8 pi of the dialyzed ir capsid protein reaction mixture is then reacted with 32 fil of resistin-C-Xa solution (resulting in a final concentration of resistin of 0.39 mg/ml) and 13 jiJ of the ft- capsid protein reaction mixture is reacted with 27 [jJ resistin-C-EK solution (resulting in a final concentration of resistin of 0.67 mg/ml) for four hours at<br>
25	"C on a rocking shaker. Coupling products are analysed by SDS-PAGE under<br>
reducing conditions.<br>
E.	Coupling of resistin-C-Xa and resistin-C-EK to HBcAg-Lys-2cys-Mut<br>
A solution of 0.2 ml of 2 mg/ml HBcAg-Lys-2cys-Mut in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with 5.6 1 of a solution of lOOmM SMPH (Pierce) in DMSO at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. 8 (il of the dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with 32 nl of resistin-C-Xa solution and 13 jd of the HBcAg-Lys-2cys-Mut reaction mixture is reacted with 27 yl resistin-C-EK solution for four hours at 25 Â°C on a rockJug shaker. Coupling products are analysed by SDS-PAGE.<br><br>
F. Coupling of resistin-C-Xa and resistin-C-EK to Pili<br>
A solution of 400 ill of 2.5 mg/ml Type-1 pili of E.coli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with a 50-fold molar excess of cross-linker SMPH diluted fium a stock solution in DMSO (fierce) at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech). TTie protein-containing fractions eluating from the column are pooled, and 8 I of the desalted derivatized pili protein is reacted with 32 il of resistin-C-Xa solution and 13 pj of the desalted derivatized pili protein is reacted with 27 fU resistin-C-EK solution for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
EXAMPLES A. Introduction of cys-containing linkers, expression and purification of mouse<br>
lymphotoxin-p<br>
The extracellular part of mouse lymphotoxin-*  (LT-" ) was recombinantly expressed with a CGG amino acid linker at its N-terminus. The linkCT contained one cysteine for coupling to VLP. A long (aa 49-306) and a short version (aa 126-306) of the protein were fused at their N-terminus to either glutathione S-transferase (GST) or a histidin-myc tag for purification. An enterokinase (EK) cleavage-site was inserted for cleavage of the tag.<br>
ConstiuctiDnofC-LTÂ« 49-306 and C-LTÂ« 126-306.<br>
Mouse LT" 49-306 was amplified by PCR with oligos 5"LTÂ« and 3"LT" from a mouse spleen cDNA library inserted into pFB-LIB. For the PCR reaction, 0.5 g of each primer and 200 ng of the template DNA was used in the 50 â¢ 1 reaction mixture (1 unit of PEX Platmum polymerase, 0.3 mM dNTPs and 2 mM MgS04). The temperature cycles were as follows: 94Â°C for 2 minutes, followed by 25 cycles of 94Â°C (15 seconds), 68Â°C (30 seconds), 68Â°C (1 minute) and followed by 68Â°C for 10 minutes. The PCR product was phosphorylated with T4 Kinase and ligated into pEntrylA (Life technoloes) which has been cut with EcoRV and has been dephosphorylated. The resulting plasroid was named pEntryl A-LTÂ» 49-306.<br><br>
A second PCR reaction was pertormea with ongos 3 LI* long-iVhel ana 3"LT* stop-NotI resp. 5"LT* short-A"Ael and 3XT" stop-WofI using pEntrylA-LT" 49-306 as a template. Oligos 5"LTÂ« long-W?icI and 5"LTÂ« short-A"iel had an internal Nhel site and contained codons for a Cys-GIy-Gly linker and 3"LTÂ» stop-Notl had an internal Notl site and contained a stop codon. For the second PCR reaction, 0.5 /ig of each primer and 150 ng of the template DNA was used in the 50 â¢ I reaction mixture (1 unit of FFX Platinum polymerase, 0.3 mM dNTPs and 2 mM MgS04). The temperature cycles were as follows: 94Â°C for 2 noinutes, followed by 5 cycles of 94Â°C (L5 seconds), SCC (30 seconds), eSC (1 minute), followed by 20 cylces of 94""C (15 seconds), 64"=C (30 seconds), eSC (1 minute) and followed by 68Â°C for 10 minutes.<br>
The PCR products were digested with Nhel and Notl and inserted into either pCEP-SP-GST-EK or pCEP-SP-his-myc-EK (Wuttke et al J. Biol Chem. 276: 36839-18 (2001)). Resulting plasmids were named pCEP-SP-GST-EK-C-LT- 49 306, pCEP-SP-GST-EK-C-LT- 126-306, pCEP-SP-his-myc-EK-C-LT- 49-306, pCEP-SP-his-myc-EK-C-LT" 126-306, respectively. GST stands for glutathione-S transferase, EK for enterokinase, his for a hexahistidine tag and myc for anti c-myc epitope. The C indicates the CGG linker containing the additional cysteine.<br>
All other steps were performed by standard molecular biology protocols.<br>
Sequence of the oligonucleotides:<br>
5"LT- : 5"-CTT GOT GCC OCA GGA TCA G-3" (SEQ ID NO:284)<br>
3"LT" : 5"-CAG ATG OCT GTC ACC CCA C-3" (SEQ ID NO:285)<br>
5"LTÂ« long-Miel: 5"-GCC CGC TAG CCT GCG GTG GTC AGG ATC AGG GAC GTC G-3" (SEQ ID NO:286)<br>
5"LTÂ« short-JVftel: 5"-GCC CGC TAG CCT GCG GTG GTT CTC CAG CTG CGG ATT C -3" (SEQ ID NO:287)<br>
3 XT* stoji-Notl<br><br>
5"-CAA TGA CTG CGG CCG CIT ACC CCA CCA TCA CCG -3" (SEQ ID NO:288)<br>
Expression and production of GST-EK-C-LT* 49.306. GST-EK-C-LT" 126-306, his-myc-EK-C-LT* 49.306 and his-myc-EK-C-LT" 126-306<br>
The plasmids pCEP-SP-GST-EK-C-LTÂ» 49-306, pCH"-SP-GST-EK-C-LT" 126-306, pCEP-SP-his-myc-EK-C-LTÂ» 49-306 and pCEP-SP-his-myc-EK-C-LT* 126-306 were transfected into 293-EBNA cells (Invitrogen) for protein production as described in EXAMPLE 1. The resulting proteins were named GST-EK-C-LT- 49-306. GST-EK-C-LT- 126-306, his-myc-EK-C-LT- 49-306 and his-myc-EK-<br>
C-LTÂ»  126-306-<br>
The protein sequences of the LT*   fusion proteins were translated from the cDNA sequences:<br>
GST-EK-C-LT* 49-306: SEQ ID NO:289<br>
GST-EK-C-LT" 126.306: SEQ ID NO:290<br>
his-myc-EK-C-LT- 49-306: SEQ ID NO:29i<br>
his-myc-EK-C-LTÂ« 12306: SEQ ID NO;292<br>
The fusion proteins were analysed on 12% SDS-PAGE geis und reducing conditions. Gels we blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a monoclonal mouse anli-myc antibody or with an anti-GST antibody. Blots were subsequently incubated with horse radish peroxidase-conjugated goat anti-mouse IgG or horse radish peroxidase-conjugated rabbit anti-goat IgG. The results are shown in FIG. 3. GST-EK-C-LT- 49-306 and GST-EK-C-LT- 126-306 could be detected with the anti-GST antibody at a molecular weight of 62 kDa and 48 kDa, respectively. his-myc-EK-C-LT- 49-306 and his-myc-EK-C-LT- 126-306 could be detected with the anti-myc antibody at 40-56 kDa and 33-39 kDa, respectively.<br>
FIG. 3 A and FIG. 3B show the result of the expression of LT- fusion proteins, LT- fusion proteins were analysed on 12% SDS-PAGE gels under reducing conditions. Gels were blotted onto nitrocellulose membranes. Membranes wa blocked, incubated either with a monoclonal mouse anti-myc antibody (dilution<br><br>
1:2000) (FIG. 3A) or with an anti-GST antibody (dilution 1:2000) (FIG. 3B). Blots were subsequently incubated with horse radish peroxidase-conjugated goat anti-mouse IgG (dilutions 1:4000) (FIG. 3A) or horse radish peroxidase-conjugated rabbit anti-goat IgG (dilutions 1:4000) (FIG. 3B). A: Lane 1 and 2: his-myc-EK-C-LT- ,2 306: Lane 3 and 4: his-myc-EK-C-LT- 49.306- B: Lane 1 and 2: GST-EK-C-LT" 126-306. Lane 3 and 4: GST-EK-C-LT" 49-306- Molecular weights of marker proteins are given on the left margin.<br>
B.	Purification of GST-EK-C-LT- 49-306, GST-EK-C-LT" 126-306. his-myc-EK-C-<br>
LT" 49-306 and his-myc-EK-C-LT" 126-306<br>
GST-EK-C-LT" 49-306 and GST-EK-C-LT" 126306 are purified on glutathione-sepharose column and his-myc-EK-C-LT" 49-306 and his-myc-EK-C-LT- 126-306 are purified on ffi-NTA sepharose column using standard purification protocols. Tlie purified proteins are cleaved with enterokinase and analysed on a 16% SDS-PAGE gel under reducing conditions<br>
C.	Coupling of C-LT" 49.306 and C-LT" 126-30S to QP capsid protein<br>
A solution of 120 pM QP capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. ITie reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. The dialyzed QP reaction mixture is then reacted with the C-LT" 49.306 and C-LT" 126-306 solution (end concentrations: 60 fiM QP, 60 [AM C-LT" 49.306 and C-LT" 126-306) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of C-LT" 49.306 and C-LT" 126-306 to fr capsid protein<br>
A solution of 120 pM fr capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. The dialyzed fr capsid protein reaction mixture is then reacted with the C-LT" 49-306 and C-LT" 126-306 solution (end concentrations: 60 jtM fr, 60 )AM C-LT" 49-306 and C-LT" 126-306) for four hours at 25 Â°C on a rocking shaker. Coupling<br><br>
products are analysed by SDS-PAGE under reducing conditions.<br>
E.	Coupling of C-LT"49-306 andC-LTÂ« 126-306 to HBcAg-Lys-2cys-Mut<br>
A solution of 120 tM HBcAg-Lys-2cys-Mut capsid in 20 mM Hepes, 150 n NaCl pH 7.2 is reacted for 30 minutes â with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C en a rocking shaker. Tlie reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 Â°C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with the C-LT* 49-306 and C-LT" 126-306 solution (end concentrations: 60 jiM HBcAg-Lys-2cys-Mut, 60 fiM C-LTÂ» 49.306 and C-LT* 126-306) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
F.	Coupling of C-LT* 49-306 and C-LT* 136-306 to Pili<br>
A solution of 125 (JM Type-1 pili of E.coli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with a 50-fold molar excess of crass-linker SMPH, diluted from a stock solution in DMSO (Pierce), at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 column (Ameisham-Phannacia Biotech). The protein-containing fractions eluating from the column are pooled, and the desalted derivatized pili protein is reacted with the C-LT* 49-305 and C-LT* 126-306 solution (end concentrations: 60 pM pili, 60 ]M. C-LT* 49.306 and C-LT* 126-306) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE under reducing conditions.<br>
EXAMPLE 4<br>
A. totrodiiction of cys-containing links, expression, purification and coupling of rat<br>
macrophage migration inhibitory factor MIF to QP<br><br>
Rat macrophage migration inhibitory factor (rMlF) was recombinantly expressed with tiiree different amino acid linkers CI, C2 and C3 fused at its C-tenninus. Each of the linker contained one cysteine for coupling to VO*.<br>
Construction of rMIF-Cl, rMIF-C2, and rMIF-C3.<br>
The MCS of pET22b(+) (Novagen, Inc.) was changed to GTrTAACnT AAGAAGGAGATATACATATGGATCCGGCTAGCGCTCGAGGGTTTAAACGG CGGCCGCATGCACC by replacing the original sequence from the Ndel site to Xhol site with annealed oligos primerMCS-lF and primerMCS-lR (aimealing in 15 mM TrisHCl pH 8 buffrar). The resulting plasmid was termed pModOO, which had Ndel, BamHI, Nhel, Xhol, Pmel and NotI restriction sites in its MCS. The annealed pair of oligos Bamhis6-EK-Nhe-F and Bainhis6-EKNhe-R and the annealed pair of oligolF-C-glycine-linker and oligolR-C-glycine-Iinker were together ligated into BamHI-Notl digested pModOO plasmid to get pModECl, which had an N terminal hexahistidine tag, an enterokinase cleavage site and a C-terminal amino acid glycine linker containing one cysteine residue, llie annealed pair of oligos Bamhis6-EK-Nhe-F and Bamhi6-EKNhe R together with the annealed pair of oligolF-C-gammal-linker and oUgolR-C-gammal-linker were ligated into BarnHf-Notl digested pModOO plasmid to get pModEC2, which had an N terminal hexahistidine tag, an enterokinase cleavage site and a C-terminal â¢ 1 linker, derived from the hinge region of human inmiunoglobuIinYl, containing one cysteine residue. Tlie annealed pair of oligos Bamhis6-EK-Nhe-F and Bamhis6-EK-Nhe-R, the annealed pair of oligolFA-C-gammaS-linker and oligolRA-C-gamina3-linker, and the annealed pair of oligolFB-C-gammaS-linker and oligolRB-C-gamma3-linker were together hgated into BamHI-Notl digested pModOO to get pModEC3, which had an N terminal hexahistidine tag, an enterokinase cleavage site and a C terminal â¢ 3 hnker, containing one cysteine residue, derived from the hinge region of mouse immunoglobulin* 3.<br>
pBS-rMIF, which contains the rat MIF cDNA, was amplified by PCR with oligos rMIF-F and rMIF-Xho-R. rMIF-F had an internal Ndel site and rMIF-Xho-R had an intemal Xhol site. The PCR product was digested with Ndel and Xhol and ligated into pModECl, pModEC2 and pModECS digested with the same enzymes. Resulting plasmids were named pMod-rMIF-Cl, pMod-rMIF-C2 and pMod-rivUF-C3, respectively.<br><br>
For the PCR reaction, 15 pmol ot each oligo and 1 ng of the template DNA was used in the 50 â¢ ! reaction mixture (2 units of PFX polymerase, 0.3 mM dNTPs and 2 mM MgS04). The temperature cycles were as follows: 94Â°C for 2 minutes, followed by 30 cycles of 94Â°C (30 seconds), 60*C (30 seconds), 68C (30 seconds) and followed by 68Â°C for 2 minutes.<br>
AH other steps were perfonned by standard molecular biology protocols.<br>
Sequence of the oligonucleotides:<br>
.    primerMCS-lF: 5"-TAT GGA TCC GGC TAG CGC TCG AGG GTT TAA ACG GCG GCC GCA T-3"(SEQIDNO:293)<br>
primerMCS-lR: 5"-TCG AAT GCG GCC GCC GTT TAA ACC CTC GAG CGC TAG CCG GAT CCA-3" (SEQ ID NO:294)<br>
BarDhis6-EK-Nhe-F: 5"-GAT CCA CAC CAC CAC CAC CAC CAC GGT TCT GGT GAG GAC GAT GAC AAA GCG CTA GCC C-3" (SEQ ID NO:295)<br>
Bambis6-EK-Nhe-R: 5"-TCG AGG GCT AGC GCT TTG TCA TCG TCG TCA CCA GAA CCG TGG TGG TGG TGG TGG TGT G-3" (SEQ ID NO:296)<br>
oligo IF-C-gJycine-linker: 5"-TCG AGG GTG GTG GTG GTG GTT GCG GTT AAT AAG TIT AAA CGC-3" (SEQ ID NO:297)<br>
oligo IR-C-glycine-linker: 5"-GGC CGC GTT TAA ACT TAT TAA CCG CAA CCA CCA CCA CCA CCC-3" (SEQlDNO:298)<br>
oligo IF-C-gammal -linker: 5"-TCG AGG ATA AAA CCC ACA CCT CTC CGC CGT GTG GTT AAT AAG TTT AAA CGC-3" (SEQ ID NO:299)<br>
oligo IR-C-gamma 1-linker: 5"-GGC CGC GTT TAA ACT TAT TAA CCA CAC GGC GGA GAG GTG TGG GTT TTA TCC-3" (SEQ ID N0:300)<br>
oligolFA-C-gamma3-liriker:<br><br>
5"-TCG AGC CGA AAC CGT CTA CCC CGC CGG GTT CTT CTG-3" (SEQ ID NO:301)<br>
oligoIRA-C-gamma3-Unken 5"-CAC CAC CAG AAG AAC CCG GCG GGG TAG AGO GTT TCG GC-3" (SEQ ID NO:302)<br>
oligo2FB-C-gamma3-linker 5"-GTG GTG CTC CGG GTG GTT GCG GTT AAT AAG TTT AAA CGC-3" (SEQ ID NO: 303)<br>
oligo2RB-C-gamma3-liiiker: 5"-GGC CGC GTT TAA ACT TAT TAA CCG CAA CCA CCC GGA G-3" (SEQ ID NO:304)<br>
rMIF-F: 5"-GGA ATT CCA TAT GCC TAT GIT CAT CGT GAA CAC-3"  (SBQ ID NO:305)<br>
rMIF-Xho-R: 5"-CCC GOT CGA GAG CGA AGG TGG AAC CGT TC-3" (SEQ ID NO:306)<br>
Expression and Purification of rMIF-Cs Competent E. coli BL21 (DE3) cells were transfoiTQed with plasmids pMod-rMIF-Cl, pMod-rMIF-C2 and pMod-rMIF-C3. Single colonies from ampicilUn (Amp)-containing agar plates were expanded in liquid culture (SB with 150mM MOPS, pH 7.0, 200ug/ml Amp. 0.5% glucose) and incubated at 30Â°C with 220 ipm shaking overnight. 1 1 of SB (150 mM MOPS, pH 7.0, 200ug/ml Amp) was then inoculated 1:50 v/v with the overnight culture and grown to OD600=2.5 at 30""C. Expression was induced with 2 mM IPTG. Cells were harvested after overnight culture and centrifuged at 6000 rpm. Cell pellet was suspended in lysis buffer (lOmM NaaHPO-f, 30mM NaCI, lOmM EDTA and 0.25% Tween-20) with 0.8 mg/ml lysozyme, sonicated and treated with benzonase. 2ml of the lysate was then run through a 20 ml Q XL- and a 20 ml SP XL-column. The proteins rMIF-Cl, rMIF-C2 and rMIF-C3 were in the flow through.<br>
The protein sequences of the rMIF-Cs were translated from the cDNA sequences. rMIF-Cl: SEQ ID NO:307 rMIF-C2:SEQIDNO:308<br><br>
rMIF-C3: SEQ ID NO:309<br>
Couplingof rMIF-Cl to QÂ» capsid protein A solution of 1.48 ml of 6 mg/ml QÂ« capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 was reacted for 30 minutes with 14.8 fil of a SMPH (Pierce) (from a 100 mM stock solution dissolved in DMSO) at 25Â°C. The reaction solution was subsequently dialyzed twice for 3 hours against 21 of 20 mM Hepes, 150 mM NaCl, pH 7.0 at 4 "C. A solution of 1.3 ml of 3.6 mg/ml rMIF-â¬l protran in 20 mM Hepes, 150 mM NaCl pH 7.2 was reacted for 1 hour with 9.6 jil of a TCEP (Pierce) (fiom a 36 mM stock solution dissolved in H2O) at 25Â°C. 130 y\ of the derivatized and dialyzed QÂ» was then reacted with 129 \il of reduced rMIF-Cl in 241 jd of 20 mM Hepes, 150 mM NaQ, pH 7.0 over night at 25Â°C.<br>
Coupling of rMIF-C2 to Q* capsid protein A solution of 0.9 ml of 5.5 mg/ml Q* capsid protein in 20 mM Hepes, 150 mM NaCI pH 7.2 was reacted for 30 minutes with 9 il of a SMPH (Pierce) (from a 100 mM stock solution dissolved in DMSO) at 25""C. The reaction solution was suhsequenfly dialyzed twice for 2 hours against 21 of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. A solution of 850 pi of 5.80 mg/ml rMIF-C2 protein in 20 mM Hepes, 150 mM NaCI pH 7.2 was reacted for 1 hour with 8.5 ill of a TCEP (Pierce) (from a 36 mM stock solution dissolved in H2O) at RT. 80 pi of the derivatized and dialyzed QÂ» was then reacted with 85 pi of reduced iMIF-C2 in 335 pJ of 20 mM Hepes, 150 mM NaCl, pH 7.2 over night at 25""C.<br>
Coupling of rMIF-C3 to Q* csid protein A solution of 1.48 ml of 6 mg/ml Q" capsid protein in 20 mM Hepes, 150 mM NaQ pH 7.2 was reacted for 30 minutes with 14.8 pi of a SMPH (Pierce) (from a 100 mM stock solution dissolved in DMSO) at 25Â°C. The reaction solution was subsequently dialyzed twice for 3 hours against 21 of 20 mM Hepes, 150 mM NaCl, pH 7.0 at 4 *C. A solution of 720 pi of 5.98 mg/ml rMIF-C3 proteui in 20 mM Hepes, 150 mM NaCl pH 7.2 was reacted for 1 hour with 9.5 fil of a TCEP (Pierce) (from a 36 mM stock solution dissolved in H2O) at 25Â°C. 130 pi of the derivatized<br><br>
and dialyzed QÂ»   was then reacted with 80 \jii of reduced rMIF-C3 in 290 jil of 20 mM Hepes, 150 mM NaCl, pH 7.0 over night at 25Â°C.<br>
Ail three coupled products were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were either stained with Coomassie Brilliant Blue or blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a polyclonal rabbit anti-Qb antiserum (dilution 1:2000) or a purified rabbit anti-MIF antibody (Toirey Pines Biolabs, Inc.) (dilution 1:2000). Blots were subsequently incubated with horse radish perojudase-conjugated goat anti-rabbit IgG (dilutions 1:2000). The results are shown in FIG 4A and FIG. 4B. Coupled product could be detected in the Coomassie-stained gels (HG. 4A) and by both â¢ anti-Qp" antiserum and ttie anti-MIF antibody (HG. 4B) clearly demonstrated the covalent coupling of all three rMIF variants to Qfl"  capsid protein.<br>
FIG 4A shows the coupling of the MIF constructs to Qp.Coupling products were analysed on 16% SDS-PAGE gels under reducing conditions. The gel was stained with Coomassie Brilliant Blue. Molecular weights of marker proteins are given on the left margin.<br>
FIG. 4B shows the coupling of MIF-Cl to Qp. Coupling products were analysed on 16% SDS-PAGE gels under reducing conditions. Lane 1; MIF-Cl before coupling Lane 2; derivatized QP before coupling. Lane 3-5: QP-MIF-Cl Lanes 1-3 were stained with Coomassie Brilliant Blue. Lanes 4 and 5 represent western blots of the coupling reaction developped with an anti-MIF antiserum and an anti-Qp antiserum, respectively. Molecular weights of marlar proteins are given on the left margin.<br>
B.Immunizationof mice with MIF-Cl coupled to QP capsid protein<br>
Female Balb/c mice were vaccinated with MIF-Cl coupled to Qp capsid protein without the addition of adjuvants. 25 jig of total protein of each sample was diluted in PBS to 200 u] and injected subcutaneously (100 ml on two ventral sides) on day 0 and day 14. Mice were bled retroorbitally on day 31 and their serum was analyzed using a MIF-specific HI ISA.<br><br>
C.EUSA<br>
ELISA plates were coated with MIF-Cl at a concentration of 5 jig/ml. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As fl control, preimmune serum of the same mice was also tested. The results are shown in FIG. 4C. There was a clear reactivity of the mouse sera raised against MIF-Cl coupled to QP capsid protein, while the pre-immune sera did not react with MIF (FIG. 4C and data not shown). From the dilution series with the antisera against MIF-Cl coupled to QP capsid protein, a half-maximal titer was reached at 1 :S4000..<br>
Shown on FIG. 4C are the ELISA signals obtained with the sera of the mice vaccinated with MIF-Cl coupled to QP capsid protein. Female Balb/c mice were vaccinated subcutaneously with 25 (ig of vaccine in PBS on day 0 and day 14. Serum IgG against MIF-Cl were measured on day 31. As a control, pre-immune sora from one of the mice were analyzed. Results for indicated serum dilutions are shown as optical density at 450 imi. All vaccinated mice made high IgG antibody titers. No MEF-specific antibodies were detected in control (pre-immune mouse).<br>
EXAMPLES Coupling of rMIF-Cl to fr capsid protein and HBcAg-lys-2cys-Mut capsid protein<br>
Coupling of TMIF-CI to fr capsid protein<br>
A solution of 100 |J1 of 3.1 mg/ml fr capsid protran in 20 mM Hepes, 150 mM NaCI pH 7.2 was reacted for 30 minutes with 3 fil of a 100 mM stock solution of SMPH (Fierce) dissolved in DMSO at 25Â°C. In a parallel reaction, fr capsid protein was first alkylated using iodoacetamid and then reacted with SMPH using the same reaction conditions described above. The reaction solutions were subsequently dialyzed twice for 2 hours against 21 of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4Â°C. A solution of 80 \J1 of 5.7 mg/ml rMIF-Cl protein in 20 mM Hepes, 150 mM NaCI pH 7.2, was reacted for 1 hour with 1 jil of a 36 mM TCEP (Pierce) stock solution dissolved in H2O, at 25Â°C. 50 1 of the derivatized and dialyzed fr capsid protein and<br><br>
50 )jJ of the derivatized, alkylated and dialyzed fir, capsid protein were then reacted each with 17 id of reduced rMIF-Cl for two hours at 25Â°C.<br>
Coupling products were analysed on 16% SDS-PAGE gels (FIG. 5). An additional band of the expected size of 27 kDa (rMIF-Cl: apparent MW 13 kDa, fr capsid protein apparent MW 14 kDa) and 29 kDa (rMIF-Cl: apparent MW 13 kDa, HBcAg-lys-2cys-Mut: apparent MW 15 kDa) can be detected in the coupling reaction but not in the fr capsid protein and rMIF-Cl solutions, clearly demonstrating coupling.<br>
Coupling of rMIF-Cl to hepatitis HBcAg-lys-2cys-Mut capsid protein:<br>
A solution of 100 \ii. of 1.2 mg/ml HBcAg-lys-2cys-Mut CEsid protein in 20 mM Hepes, 150 mM NaQ pH 7.2 was reacted for 30 minutes witii 1.4 1 of a SMPH (Pierce) (from a 100 mM stock solution dissolved in DMSO) at 25Â°C. The reaction solution was subsequently dialyzed twice for 2 hours against 21 of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4Â°C. A solution of 80 1 of 5.7 mg/ml rMIF-Cl protein in 20 mM Hepes, 150 mM NaCI, pH 7.2 was reacted for 1 hour with I pj of a TCEP (Pierce) (from a 36 mM stock solution dissolved in H2O) at 25Â°C. 60 1 of the derivatized and dialyzed HBcAg-lys-2cys-Mut capsid protein was then reacted with 20 jil of reduced rMIP-Cl for two hours at 25Â°C.<br>
Coupling products were analysed on 16% SDS-PAGE gels (FIG. 5) under reducing conditions. An additional band of the expected size of about 28 kDa (rMIF-Cl: apparent MW 13 kDa, HBcAg-lys-2cys-Mut: apparent MW 15 kDa) can be detected in the coupling reaction but not in derivatized HBcAg-lys-2cys-Mut or rMIF-Cl, clearly demonstrating coupling.<br>
The samples loaded on the gel of FIG. 5 were the following: Lane 1: Molecular weight marker. Lane 2: rMIF-Cl before coupling. Lane 3: iMIF-Cl-fr capsid protein after coupling. Lane 4: derivatized fr capsid protein. Lane 5: rMIF-Cl-fr after coupling to alkylated fr capsid protein. Lane 6: alkylated and derivatized fr capsid protein. Lane?: rMIF- HBcAg-lys-2cys-Mut after coupling. Lane<br><br>
8 and 9: derivatized HBcAg-lys-2cys-Mut. The gel was stained with Coomassie Brilliant Blue. Molecular weights of marker proteins are given on the left margin.<br>
EXAMPLE6 A. Introduction of amino acid linkers containing a cysteine residue, expression and<br>
purification of mouse RANKL<br>
A fragment of the receptor activator of nuclear factor kpa b Ugand (RANKL), which has also been termed osteoclast differentiation factor, osteoprotegerin ligand and tumor necrosis factor-related activation-induced cytokine was recombinantly expressed with an N-terminal linker containing one cysteine for coupling to VLP.<br>
Construction of expression plasmid<br>
The C-terminal coding region of the RANKL gene was amplified by PCR with oligos RANKL-UP and RANKL-DOWN. RANKL-UP had an internal Apal site and RANKL-DOWN had an internal Xhol site. The PCR product was digested with Apal and Xhol and ligated into pGEX-6pI (Amersham Pharmacia). The resulting plasmid was named pGEX-RAKKL. All steps were performed by standard molecular biology protocols and the sequence was verified. The plasmid pGEX-RANKL codes for a fusion protein of a glutathione S-transfere-Prescission cleavage site-cysteine-containing amino acid linker-RANKL (GST-PS-C-RANKL). The cysteine-containing amino acid linker had the sequence GCGGG. The construct also contains a hexa-histidine tag between the cysteine containing amino acid linlr and the RANKL sequence.<br>
Oligos:<br>
RANKL-UP:<br>
5"CTGCCAGGGGCCCGGGTGCGGCGGTGGCCATCATCACCACCATCACCAG<br>
CGCTTCTCAGGAG-3" (SEQffi NO:316)<br>
RANKL-DOWN: 5"-CCGCTCGAGTTAGTCrATGTCCTGAACTrrGAAAG-3" (SEQ ID NO:317)<br><br>
Protein of GST-PS-C-RANKL (SEQ ID NO:318) and cDNA sequence of GST-PS-C-RANKL (SEQ ID NO:319)<br>
IMSEILGYWKIKGLVQFTHLLLBYLE<br>
1  BtgtcccctatactaggttattggooaattaagggccttgtgcaacocBctcgacttcttttffgaatatcttgaa<br>
26EKYBEHLYERDEGDKWRNKKFELGL<br>
76 gaaaaatatgaagagcatttgtatgagcgcgatgaaggtgotaaatggcgaaacaaaaagtttgaattgggtttg<br>
BIEFPNLPYYIDGDVKLTQSMAI     IRYI<br>
151 gagtttcccaatcttccttattatattgatggtgatgttaaattaacacagtctatggccatcatacgttatata<br>
76fiDKHNMLGGCPKERAEISMLEGAVL 226 gctgacaagcacaacatgttgggtggttgtccaaaagagcgtgcagagatttcaatgcttgaaggageggttttg lOlDIRYQVSRIAYSKDFETLKVDFLSK 301 gatattagatacggtgtttcgagaottgcataCagtaaBgactttgaaaetctcaaagttgattttcttagcaag 125  LPEHLKMFEDRLCHKTYLNGDHVTH 376 ctacctgaaatgctgaaaatgttcgaagaccgtttatgtcataaaacatatttaaatggtgatcatgcaacccat ISl  PDFMLYDALDVVLYMDPMCLDRFPK 451 cctgacttcatgttgtatgacgctcttgatgttgctttatacatggacccaatgtgcctggatgcgtteccaaaa 176  LVCFKKRIEAI     PQIDKYLKSSKYIA 525 ttagtctgttttaaaaaacgcatcgaagctatcccacaaattgataagcacttgaaatccagcaagtatatagca 201WPI.QQWQATFGGGDHPPKSDLEVLF 601 tggcctttgcagggctggcaagccBogtttggtggtggcgaccatcotccaaaatcggatctggaagttctgtCc 226   QGPGCQQGHHRHHHQRFS0APRHH3 676c agGGEKCCGGGTIKBOCQGTQGCCATCaTCACICC ATCACC AGCGCTTCTCAGGAGCTCCfiGCTATG ATK lElQSWLnVftQRGKPEAQPFRHLTINSA 751 GQCrCATGGTTGGATtJTGOCCCAGCGAGGCAAGCCrroAIKCCCaaCCAlTrQCACACCTGACXXTC 276  STPSGSHKVTLSSWYHDRGWIKISN B2 6 AGCATCCCXTCGGGTTCCCATAAAGTCaCTCTGTCCTCTTGGTRCCRCGATCQAGOCTGGGCXyiAGATCTCTAAC 301   MTLSHGKLRVNQQGFYYLYANICFB 901  ATGACGTTAAGCAACGGAAAACT"AAGGGrrA&amp;CCAAGATQGCTTCTATTRCCTGTACaCCAACaTTTGCTl 326  HHETaGSVPT-DYLOLMVYVVKTSIK 976  CTCATGAARCATCGGGAAGCGTRCCTACAGACTATCTTCAGCTGATGGmOTATGTCGTTAAAACCAGCATCAAA 351   IPSSHNLHF.     GQSTKNWSGNSEFHFY 1051  ATCaAftHTTCTCATAACCTQATGAAAGaAGGGAGCaCGaftAAACTGGTaJGGGAArTCTGAAI<br>
375   SINVGnFFKLRaGBEI     SigVSKPSL 1126 TCCATAAATGlrrGGQaOATTTlT"CftJWSCTCCQfiGCTGGTGAAGAAATTAGCATTCAGGTGTCCaiA<br>
401  LDPDQDATVpGRFKVQDID* 1201   CrreGATCCGGATCRAGATCKGACGTACTTTGGGGCTTTCAAAGTTCAGGACATAGACTAACTCGAGCGG<br>
Expression and Purification of C-RANKL<br>
Competent E. coli BL21 (DE3) Gold pLys cells were transformed with the plasmid pGEX-RANKL. Single colonies from kanamycin and chloramphenicol-containing agar plates were expanded in liquid culture (LB medium, 30p.g/ml kanamycin, 50|Ag/mi chloramphenicol) and incubated at 30Â°C with 220 rpm shaldng overnight. 11 of LÂ£ (with SOug/ml kanamycin) was then inoculated 1:100 v/v with the overnight culture and grown to OD600=1 at 24""C. Expression was induced with 0.4 mM IPTG. Cells were harvested after 16 h and centrifuged at 5000 rpm. Cell pellet was suspended in lysis buffer (50 mM Tris-HCl, pH=8; 25 % sucrose; 1 mM EDTA, 1% NaNs; 10 mM DTT; 5 mM Mga2; 1 mg/ml Lysozyme; 0.4u/ml DNAse) for 30 min. Then 2.5 volumes of buffer A (50 mM Tris-HCl, pH=8.0; 1% Triton XlOO; 100 mM NaCl; 0,1% NaNs; 10 mM DTT;1 mM PMSF) were added and<br><br>
" incubated at 37Â°C for 15 min. The cells were sonicated and pelleted at 9000 rpm for 15 min. The supernatant was immediately used for GST-affmity chromatography.<br>
A column GST-Trap FF of 5 ml (Amersham Pharmacia) was equilibrated in PBS, pH 7.3 (140 mM NaCI, 2.7 mM KCl, 10 mM Na2HP04,1-8 mM KHzPO*). The supematant was loaded on the 5 ml GST-Trap FF column and subsequently the column was rinsed with 5 column volumes of PBS. The protein GST-PS-C-RANKL was eluled with 50 mM Ttis-HCl, pH=8.0 containmg GSH 10 mM.<br>
The purified GST-PS-C-RANKL protein was digested using the protease PieScission (Amersham Phannacja). The digestion was performed at 37Â°C for 1 hour using a molar ratio of 500/1 of GST-PS-C-RANKL to PreScJssion.<br>
Furthermore, the reaction of protease digestion was buffer exchanged using a HiPrep 26/10 desalting column (Amersham Phaimacia), the fractions containing the proteins were pooled and immediately used for another step of GST affinity chromatography using the same concKtions reported before. Purification of C-RANKL was analysed on a SDS-PAGE gel under reducing conditions, shown in Fig.6. Molecular weights of marker proteins are given on the left margin of the gel in the figure. The gel was stained with Coomassie Brilliant Blue. The cleaved C-RANKL is present in the flow-through (unbound fraction) while the uncleaved GST-PS-C-RANKL, the cleaved GST-PS and the PreScission remain bound to the column. C-RANKL protein of the expected size of 22 kDa was obtained in high purity.<br>
The samples loaded on the gel of HG. 6 were the following: Lane 1: Low molecular wdght marker. Lanes 2 and 3: the supernatant of the cell lysates of the BL21/DE3 cells transformed with the empty vector pGEX6pl and pGEX-RANKL respectively, after sixteen hours of induction with IPTG 0.4 mM. Lane 4: the purified GST-PS-C-RANKL protein after GST-Trap FF column. Lane 5: the GST-Trap FF column unbound fraction. Lane 6: the purified GST-PS-C-RANKL protein after the cleavage with the Rescission protease. Lane 7; the unbound ftaction of the GST-Trap FF column loaded with the GST-RANKL digestion, which contains the purified C-RANKL. Lane 8: the bound firaction of the GST-Trap FF column loaded with the GST-PS-C-RANKL digestion and eluted with GSH.<br>
B.       Coupling of C-RANKL to Qp capsid protein<br>
A solution of 120 \JM Qp capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is<br><br>
F<br>
reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 Â°C. The dialyzed Qp reaction mixture is then reacted with the C-RANKL solution (end concentrations: 60 M QP, 60 iM C-RANKL) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
C.	Coupling of C-RANKL to fr capsid protein<br>
A solution of 120 M fr csid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 houn against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. The dialyzed fr capsid protein reaction mixture is then reacted with the C-RANKL solution (end concentrations: 60 M fr capsid protein, 60 jiM C~ RANKL) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of C-RANKL to HBcAg-Lys-2cys-Mut<br>
A solution of 120 pM HBcAg-Lys-2cys-Mut capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with the C-RANKL solution (end concentrations: 60 pM HBcAg-Lys-2cys-Mut, 60 [oM C-RANKL) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
E.	Coupling of C-RANKL to Pili<br>
A solution of 125 \M Type-1 pili of E.coli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with a 50-fold molar excess of cross-linker SMPH, diluted from a stock solution in DMSO, at RT on a rocking shaker. TTie reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech). The protein-containing fractions eluating from the column are pooled, and the desalted derivatized pili<br><br>
protein is reacted with the C-RANKL solution (end concentrations: 60 pM pili, 60 pM C-RANKL) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
EXAMPLE?<br>
A. Introduction of amino acid linker containing a cysteine residue, expression and<br>
purification of a truncated form of the mouse prion protein<br>
A truncated form (aa 121-230) of the mouse prion protein (termed mPrPO was recombinantly expressed with a GGGGCG amino acid linker fused at its C-termihus for coupling to VLPs and Pili. The protein was fiised to the N-tenninus of a human Fc-iragment for purification. An enterokinase (EK) cleavage-site was introduced behind the EK cleavage site to cleave the Fc- part of the fusion protein after purification.<br>
Constniction of mPrPrEK-Fc*.<br>
Mouse PrP, was amplified by PCR with the primer yPrP-BamHl and 3"PrP-Nhel using the plasmid pBPÂ°PrP-Fc as a template. pBPÂ°**PrP-Fc contamed the wild-type sequence of the mouse prion protein. S"PrP-BoTnfll had an internal BamHl site and contained an ATG and 3"PrP-NheI had an internal Nhel site.<br>
For the PCR reaction, 0.5 fig of each primer and 200 ng of the template DNA was used in the 50 " 1 reaction mixtuie (1 unit of PFX Platinum polymerase, 0.3 mM dNTPs and 2 mM MgS04). The temperature cycles were as follows: 94Â°C for 2 minutes, followed by 5 cycles of 94Â°C (15 seconds), 50Â°C (30 seconds), 68Â°C (45 seconds), followed by 20 cycles of 94Â°C (15 seconds), 64Â°C {30 seconds), 68Â°C (45 seconds) and followed by 68Â°C for 10 minutes.<br>
The PCR product was digested with BamHl and Nhel and inserted into pCEP-SP-EK-Fc* containing the GGGGCG linker sequence at the 5"end of the EK cleavage sequence. The resulting pliismid was named pCEP-SP-mPrPrEK-Fc*.<br>
All other steps were performed by standard molecular biology protocols.<br>
OKgos:<br><br>
Primer 5"PrP-5afnHI 5"-CGG GAT CCC ACC ATG GTG GGG GGC CTT GG -3" (SEQ ID NO:321)<br>
Primer 3"PrP-A7wI 5"-CTA GCT AGC CTG GAT CTT CTC CCG -3" (SEQ ID NO:322)<br>
Expression and Purification of mPrPi-EK-Fc*<br>
Plasraid pCEP-SP-mPrPi-EK-Fc* was transfected into 293-EBNA cells (Invitrogen) and purified on a Protein A-sepharose column as described in EXAMPLE I.<br>
The protein sequence of the mPrPt-EK-Fc* is identified in SEQ ID NO;323. mPrPt aftM- cleavage has the sequence as identified in SEQ ID NO:324 with the GGGGCG linker at its C-terminus.<br>
The purified fusion protein mPrP|-EK-Fc* was cleaved with enterokinase and analysed on a 16% SDS-PAGE gel under reducing conditions before and after enterokinase cleavage. The gel was stained with Coomassie Brilliant Blue. The result is shown in FIG. 7. Molecular weits of marker proteins are given on the left margin of the gel in the figure. Hie mPrPrBK-Fc* fusion protein could be detected as a 50 kDa band. The cleaved mPrPt protein containing the GGGGCG amino acid linker fused to its C-terminus could be detected as a broad band between 18 and 25 kDa. Tlie identity of mPrPt was confirmed by western blotting (data not shown). Thus, mPrPt with a C-tenninal amino acid linker containing a cysteine residue, could be expressed and purified to be used for coupling to VLPs and RU.<br>
The samples loaded on the gel of FIG. 7 were the following. Iane 1: Molecular weight marker. Lane 2: mPrPt-EK-Fc* before cleavage. Lane 3; mPrP, after cleavage.<br>
B-       Coupling of mPrPi to QP capsid<br>
A solution of 120 pM Qp capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is rented for 30 minutes with a 25 fold molai excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl,<br><br>
pH 7.2 at 4 "C. The dialyzed Q reaction mixture is then reacted with the mPrPt solution (end concentrations: 60 (iM QP, 60 MM mPrPO for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
C.	Coupling of mPrPt to fr capsid protein<br>
A solution of 120 |AM fi: capsid protein in 20 mM Hepes, 150 mM NaCl pH 12 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. ITie dialyzed fr reaction mixture is then reacted with the mPrPt solution (end concentrations: 60 M fr, 60 pM mPrPO for four hours at 25 "C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of mPrPt to HBcAg-Lys-2cys-Mut<br>
A solution of 120 M HBcAg-Lys-2cys-Mut capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM HepK, 150 mM NaCl, pH 7.2 at 4 "C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with the mPrPt solution (end concentrations". 60 nM HBcAg-Lys-2cys-Mut, 60 pM mPrPt) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
E.	Coupling of mPrP, to Pili<br>
A solution of 125 jiM Type-1 pili of E.coli in 20 mM Efcpes, pH 7.4, is reacted for 60 minutes with a 50-fold molar excess of cross-linker SMPH (Pierce), diluted from a stock solution in DMSO, at RT on a rocking shaker- The reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech). The protein-containing fractions eluating from the column are pooled, and the desalted derivatized pili protein is reacted with the mPrPt solution (end concentrations: 60 loM pili, 60 pM mPrPt) for four hours at 25 ""C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br><br>
EXAMPLES A. Coupling of prion peptides to Qp capsid protein: prion peptide vaccines<br>
The following prion peptides were chemically synthesized: CSAMSRPMIHFGIWEDRYYKENMYR ("cpiplong") and CGNDWEDRYYRENMYR ("cprpshort"), which comprise an added N-tenninal cysteine residue for coupling to VLPs and Ptli, and used for chemical coupling to QP as described in the following.<br>
A solution of 5 ml of 140 /iM Qp capsid protein in 20 mM Hepes. 150 mM NaCl pH 7.4 was reacted for 30 minutes with 108 I of a 65 mM solution of SMPH (Pierce) in H2O at 25Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 5 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4Â°C. 100 |il of the dialyzed reaction mixture was then reacted either with 1.35 fil of a<br>
2	mM stock solution (in DMSO) of the peptide cprpshort (1:2 peptide/Q* capsid<br>
protem ratio) or with 2.7 |J1 of the san stock solution (1:1 peptide/Q" ratio). 1 jil of<br>
a 10 mM stock solution (in DMSO) of the peptide cprplong was reacted with 100 fjJ<br>
of the dialyzed reaction mixture. TTie coupling reactions were performed over night at<br>
15 Â°C in a water bath. The reaction mixtures were subsequently dialyzed 24 h against<br>
2x 5 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4Â°C.<br>
The coupled products were centrifuged and supematants and pellets were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were stained with Coomassie Brilliant Blue. The results are shown in FIG. 16. Molecular weights of marker proteins are ven on the left mar of the gel in the figure. The bands at a molecular weight between 16.5 and 25 kDa clearly demonstrated the covalent coupling of the peptides cprpshort and cprplong to QÂ«  capsid protein.<br>
Tlie samples loaded on the gel of FIG. 16 A were the following: Lane 1: purified QÂ» capsid protein. Lane 2: derivatized QP capsid protein beftwe coupling. Lanes 3-6: QP capsid protein-cprpshort couplings with a 1:2 peptide/Q* ratio (lanes 3 and 4) and 1:1 peptide/Q* ratio (lanes 5 and 6). Soluble fractions (lanes<br>
3	and 5) and insoluble fractions (lanes 4 and 6) are shown.<br><br>
The samples loaded on the gel of FIG. 16 B were the following: Lane 1; Molecular weight marker. Lane 2: derivatized Qp capsid protein before coupling. Lane 3 and 4: QP capsid protein-cprplong coupling reactions. Soluble fraction (lane 3) and insoluble fraction {lane 4) are shown.<br>
B.	Coupling of prion peptides to fr capsid protein<br>
A solution of 120 iiM fr capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 10 fold molar excess of SMPH (Kerce)), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 *C. The dialyzed fr reaction mixture is then reacted with equimolar concentration of peptide cprpshort or a ration of 1:2 cprplong/fr overnight at 16 "C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
C.	Coupling of prion peptides to HBcAg-Lys-2cys-Mut<br>
A solution of 120 pM HBcAg-Lys-2cys-Mut in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 10 fold molar excess of SMPH (Pierce) ), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with equimolar concentration of peptide cprpshort or a ration of 1:2 cprplong / HBcAg-Lys-2cys-Mut over night at 16 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of prion peptides to PiU<br>
A solution of 125 fiM Type-1 pili of E.coli in 20 mM Hepes. pH 7.4, is reacted for 60 minutes with a 50-foid raolar excess of cross-linkra- SMPH (Pierce), diluted from a stock solution in DMSO, at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 column (Anaersham-Pharmacia Biotech). Tlie protein-containing fractions eluating from the column are pooled, and the desalted derivatized pili protein is reacted with the prion peptides in equimolar or in a ratio of 1:2 peptide pih over nit at 16 "C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br><br>
Example 9 Cloning, expression and purification of IL-13 to VLPs and Pili<br>
A.       Cloning and expression of InterleuKn 13 (IL-13) with an N-terminal amino acid linker containing a cysteine residue for coupling to VLPs and Pili a) Cloning of mouse IL-13 (HEK.-293T) for expression in mammalian cells as Fc fusion protein The DNA for the cloning of IL-13 was isolated by RT-PCR from in vitro activated splenocytes, wich were obtained as following: CD4+ T cells were isolated from mouse spleen cells and incubated 3 days in EMDM (+5% PCS + 10 ng/ml IL4) in 6 well plates which have been previously coated with anti-CD3 and anti-CD28 antibodies. The RNA from these cells was used to amplify IL13 by one-step RT-PCR (Qiagen one-step PCR Idt). Primer XhoIL13-R was used for the reverse transccription of the RNA and the primers NheIL13-F (SEQ ID NO:338) and XhoIL13-R (SEQ ID NO:339) were used for the PCR amplification of the IL13 cDNA. Amplified IL13 cDNA was ligated in a pMOD vector using the Nhel/Xhol restriction sites (giving the vector pMODB 1-IL13). pMODB 1-113 was digested BamHI/XhoI and the fragment containing IL13 was ligated in the pCEP-SP-XA-Fc*(Axho) vector, an analogue of pCEP-SP-XA-Fc* where a Xhol site at the end of the Fc sequence has been removed, which had been previously digested with BamHI/XhoL Tlie plasmid resulting from this ligation (pCEP-SP-IL13-Fc) was sequenced and used to transfect HEK-293T cells. The resulting IL13 construct enaided by this plasroid had the aroino acid sequence ADPGCGGGGGLA fused at the N-terminus of the IL-13 mature sequence. This sequence comprises the amino acid linker sequence GCGGGOG flanked by additional amino acids introduced during the cloning procedure. IL13-Fc could be purified with Protein-A resin from the supernatant of the ceUs transfected with pCEP-SP-IL13-Fc. The result of the expression is shown on HG. 17 B (see EXAMPLE 10 for description of the samples). Mature IL-13 fused at its N-terminus with the aforementioned anuno acid sequence is released upon cleavage of the fusion protein with Factor-Xa, leading to a protein called hereinafter "mouse C-IL-13-F" and having a sequence of SEQ ID NO:328. The result of FIG. 17 B clearly demonstrates expression of the IL-13 construct.<br><br>
b) Cloning of mouse IL-13 (HEK-293T)forexpressioninmaminahan cells with GST (Glutathion-S-transferase) fused at its N-terminus The cDNA used for cloning lL-13 with an N-tenninal GST originated from the cDNA of TH2 actiated T-cells as described above (a.). IL-13 was amplified ftom this cDNA using the primers NheIinklIL13-F and IL13StopXhoNot-R. The PCR product was digested with Nhel and Xhol and ligated in the pCEP-SP-GST-EK vector previously digested with Nhel/Xhol. The plasmid which could be isolated from the ligation (pCEP-SP-GST-IL13) was used to transfecl HEK-293T cells. The resulting IL 13 construct encoded by this plasmid had the amino acid sequence LACGGGGG fused at the N-tMminus of the IL-13 mature sequence. This sequence comprises the ansino acid linker sequence ACGGGGG flanked by an additional amino acid introduced during the cloning procedure. The culture supernatant of the cells transfected with pCEP-SP-GST-IL13 contained the fusion protein GST-IL13 which could be purified by Glutathione affinity chromatography according to standard protocols. Mature IL-13 fused at its N-terminus with aforementioned amino acid sequence is released upon cleavage of the fusion protein with enterokinase, leading to a protein called hereinafter "mouse C-IL-13-S" and having a sequence of SEQ ID NO:329.<br>
B.	Coupling of mouse C-IL-13-P, mouse C-IL-13-S to QP capsid protein<br>
A solution of 120 fiM Qp capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Rerce), diluted ftom a stock solution in DMSO, at 25 "C on a rocking shaker. The reaction solution is subsequently dialy7Â«d twice for 2 hours against IL of 20 roM Hepes, 150 roM NaCl, pH 7.2 at 4 Â°C. The dialyzed Qp reaction mixture is then reacted with the mouse C-IL-13-F or mouse C-IL-13-S solution (end concentrations: 60 jjM Qp capsid protein, 60 pM mouse C-IL-13-F or mouse C-1L13-S) for four hours at 25 "C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
C.	Coupling of mouse C-IL-13-F, mouse C-1113-S to fr capsid protein<br>
A solution of 120 pM fr capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 roinutes with a 25 fold molar excess of SMPH (Pierce), diluted<br><br>
from a stock solution in DMSO, at 25 Â°C on a rocking shaker. "Hie reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. The dialyzed fir reaction mixture is then reacted with the the mouse C-II13-F or mouse C-IL-13- solution (end concentrations: 60 M fircapsid protein, 60 nM mouse C-IL-13-F or mouse C-IL-13-S} for four houra at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of mouse C-IL-13-F or mouse C-]IL-i3-S solution to HBcAg-Lys-<br>
2cys-Mut<br>
A solution of 120 \JM HBcAg-Lys-2cys-Mut capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molai excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Ifepes, 150 mM NaCl, pH 7.2 at 4 Â°C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is flien reacted with the mouse C-IL-13-F or mouse C-IL-13-S solution (end concentrations: 60 fiM HBcAg-Lys-2cys-Mut, 60 nM mouse C-n--13-F or mouse C-IL-13-S) for four hoars at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
E.	Coupling of mouse C-IL-13-F or mouse C-IL-13-S solution to Pili<br>
A solution of 125 [iM Type-1 pili of E.coli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with a 50-foId molar excess of cross-linker SMPH, diluted from a stock solution in DMSO, at RT on a rocking shaker. Tlie reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech). The protein-containing fractions eluating from the column are pooled, and the desalted derivatized pili protein is reacted with the mouse C-IL-13-F or mouse C-IL-13-S solution (end concentrations: 60 nM pili, 60 \iM mouse C-IL-13-F or mouse C-IL-13-S) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br><br>
Cloning and expression of Interleuliin 5 (IL-5) with an N-terminal amino acid linker containing a cysteine residue for coupling to YLFs and IMli<br>
A. Cloning of IL-5 for expression as Inclusion bodies in E. coli<br>
lL-5 was amplified from an ATCC clone (pniIL5-4G; ATCC number: 37562) by PCR using the following two primers: Spelinker3-Fl (SEQ ID NO:340) and USStopXho-R (SEQ ID NO:342). The product of this PCR was used as template for a second PCR with the primers SpeNlinker3-F2 (SEQ ID NO:34i) and USStopXho-R. The insert was digested with Spel and Notl. This insert was ligated into a pET vector derivative (pMODEC3-8 vector), previously digested with Nhel and Notl (not dephosphorylated), and transformed into E.coli TGI cells. The IL5 construct generated by cloning into pMODBC3-8 vector contains at its N-teiminus a hexa-hiatidine tag, followed by an enteroldnase site, an N-tecffiinal gamma 3 amino acid linker containing a cysteine residue, flanked C-terminaUy by the sequence AS and N-terminally by the sequence ALV, and the mature fonn of the IL 5 gene. The protein released by cleavage with enteroldnase is called "mouse C-IL-5-E" (SEQ ID NO:332). Plasmid DNA of resulting clone pMODC6-IL5.2 (also caUed pMODC6-IL5), whose sequence had been confinned by DNA sequencing, was transformed into Kco/i strain BL21.<br>
Clone pMODC6-IL5/BL21 WM grown over night in 5 ml UB containing 1 mg/L AmpiciUin. 2 ml of this culture were diluted in 100 nil terrific broth (TB) containing Img/L AmpiciUin. The culture was induced by adding 0.1 ml of a IM solution of Ispropyl P-D-Thiogalactopyranoside (IPTG) when the culture reached an optical density OD600=0.7. 10 ml samples were taken every 2h. The samples were centrifugated 10 min at 4000 x g. The pellet was resuspended in 0.5 ml Lysis buffer containing 50 mM Tris-HCl, 2 mM EDTA, 0.1% triton X-100 (pH8). After having added 20 pi of Lysozyme (40n:ml) and having incubated the tube 30 min at 4Â°C, the cells were sonicated for 2 min. 100 1 of a 50 mM MgCl2 solution and 1 ml of benzonase were added. The ceUs were then incubated 30 rriin at room temperature and centrifugated 15 rain at 13000 x g.<br>
The supernatant was discarded and the pellet was boiled 5 min at 98Â°C in 100 Vil of SDS loading buffet. 10 il of the samples in loading buffer were analyzed fay<br><br>
SDS-PAGE under reducing conditions (FIG. 17 A). The gel of FIG. 17 A clearly demonstrates expression of the 115 construct. The samples loaded on the gel of FIG. 17 A were the following:<br>
Lane M: Marker (NEB, Broad range prestained marker). Lane 1; cell exctract of Iml culture before induction. Lane 2: cell extract of 1 ml culture 4h after bduction.<br>
B.      Qoning of IL-5 for expression in mammalian cells (HEK-293T)<br>
a)IL-5 fused at its N-tenninus to an anuno add linker containing a cysteine residue and fused at its C-terminus to the Fc fragment<br>
The template described under (A) (ATCC clone 37562) was used for the cloning of the following construct. The plasmid pM0DBl-IL5 (a pET derivative) was digested with BamHI/XhoI to yield a small fragement encoding IL5 fused to an N terminal amino acid linker containing a cysteine. Tiiis fragment was ligated in the vector pCBP-SP-XA--Fc*(AXho) wliich had previously been digested with BamHI and Xhol. The ligation was electroporated into E.coli strain TGI and plasmid DNA of resulting clone pCEP-SP-IL5-Fc.2, whose sequence had been confirmed by DNA sequencing, was used to transfect HEK-293T cells. The resulting 115 construct encoded by this plasmid had the amino acid sequence ADPGCGGGGGLA fused at the N-tenninus of the TLr5 mature sequence. This sequence comprises ttie amino acid linker sequence GCGGGGG containing a cysteine and flanked by additional amino acids introduced during the cloning procedure. The IL-5 protein released by cleavage of the fusion protein with Factor-Xa is named hereinafter "mouse C-IL-5-F" (SEQ ID NO:333).<br>
After transfection and selection on Puromycin the culture supernatant was analyzed by Western-Blot (FIG. 17 B) using an anti-Ks (mouse) and an anti-mouse IgG antibody conjugated tb Horse raddish peroxidase. The anti-mouse IgG antibody conjugated to Horse raddish peroxidase also detects Fc-fosion proteins. Purification of the protein was performed by affinity chromatography on Protein-A resin. The result of FIG. 17 B clearly demonstrates expression of the IL-5 construct. The samples loaded on the Westem-Blot of FIG. 17 B were the following:<br><br>
Lane 1: supernatant of HEK culture expressing IL5-Fc (20nl). SDS-PAGE was perfonned under reducing conditions. Lane 2: supernatant of HEK culture expressing IL13-Fc (20|AI). SDS-PAGE was performed under non reducing conditions. Lane 3: supernatant of HEK culture expressing IL5-Fc (30(il). SDS-PAGE was perfonned under non reducing conditions.<br>
b) IL-5 cloned with GST (Glutathion-S-transferase) and an amino acid linker containing a cysteine residue fused at its N-terminus<br>
IL-5 (ATCC 37562) was amplified with the primers Nhe-linkl-IL13-F and IL5StopXho-R. After digestion with Nhel and Xhol the insert was ligated into pCEP-SP-GST-EK which had been previously digested with Nhel and Xhol. The resulting plasmid pCEP-SP-GST-IL5 was sequenced and used for transfection of HEK-293T cells. The resulting lL-5 construct encoded by this plasmid had the amino acid sequence LACGC3GGG fused at the N-terminus of the IL-5 mature sequence. This sequence comprises the amino acid linter sequence ACGGGGG containing a cysteine residue and flanked by additional amino acids introduced during the cloning procedure. The protein released by cleavage with enterokinase was named hereinafter "mouse C-IL-5-S" (SEQ ID NO:334). The purification of the resulting protein was performed by affinity chromatography on Glutathione affinity resin.<br>
C.	Coupling of mouse C-IL-5-F oi mouse C-IL-5-S to Qp capsid protein<br>
A solution of 120 M QP capsid protein in 20 mM Ifepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted fix&gt;m a stock solution in DMSO, at 25 Â°C on a rockiag shaker. The reaction solution is subsequently dialyzed twice for 2 houK against 1L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. The dialyzed Q3 reaction mixture is then reacted with the mouse C-IL-5-F or mouse C-IL-5-S solution (end concentrations: 60 (4M Qp capsid protein, 60 pM moose C-IL-5-F or moose C-IL-5-S) foi four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
D.	Coupling of mouse mouse C-IL-S-F or mouse C-IL-5-S to fr capsid protein<br>
A solution of 120 pM fi: capsid prottmi in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted<br><br>
rrom a stock; solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 Â°C. Tlie dialyzed fr reaction mixture is then reacted with the the mouse C-IL-5-F or mouse C-IL-5-S solution (end concentrations; 60 nM fir cap&amp;id protein, 60 \M mouse C-IL-5-F or mouse C-IL-5-S) for four hours at 25 "C on a rocking shaker. Coupling products are analysed by SDS-PAGE.<br>
E.	Coupling of mouse C-IL-5-F or mouse C-IL-5-S solution to HBcAg-Lys-<br>
2cys-Mut<br>
A solution of 120 M HBcAg-Lys-2cys-Mut capsid in 20 mM Hepes, 150 mM NaCl pH 7.2 is reacted for 30 minutes with a 25 fold molar excess of SMPH (Pierce), diluted from a stock solution in DMSO, at 25 Â°C on a rocking shaker. The reaction solution is subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. The dialyzed HBcAg-Lys-2cys-Mut reaction mixture is then reacted with the mouse mouse C-IL-5-F or mouse C-II.5-S solution (end concentrations: 60 pM HBcAg-Lys-2cys-Mut, 60 M mouse C-IL-5-F or mouse C-E--5-S) for four hours at 25 Â°C on a rockmg shaker. Coupling products are analysed by SDS-PAGE.<br>
F.	Coupling of mouse C-IL-5-F or mouse C-IL-5-S solution to Pili<br>
A solution of 125 M Type-1 pili of E.coli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with a 50-foId molar excess of cross-IinkCT SMPH, diluted fiiom a stock solution in DMSO, at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 colunon (Amersham-Pharmacia Biotech). The protein-containing fractions eluating ftora. the column are pooled, and the desalted derivatized pili protein is reacted with the mouse mouse C-IL-5-F or mouse C-IL-5-S solution (end concentrations: 60 flM pili, 60  mouse C-IL-5-F or mouse C-IL-5-S) for four hours at 25 Â°C on a rocking shaker. Coupling products are analysed by SDS-=PAGE.<br>
EXAMPLE 11 Introduction of an anuno acid linker contsuoing a cystine residue, expresaon,<br><br>
r<br>
purification and coupling of a murine vascular endothelial growth factor -2<br>
(mVEGFR-2, FLKl) fragment<br>
A construct of the murine vascular endothelial growth factor-2 (mVEGFR-2, FLK-1) comprising its second and third extracellular domains was recombinantly expressed as a Fc-fusion protein with an amino acid linker containing a cysteine residue at its C-terminus for coupling to VLPs and Pili. The protein sequences of the mVEGFll-2(2-3) was translated from the cDNA sequences of mouse FLK-1 ((Matthews et ai, Proc. Natl. Acad. Sci. USA 88: 9026-9030 (1991)): Accession no.: X59397; Ig-hke C2-type domain 2: amino acid 143-209; Ig-like C2-type domain 3: amino acid 241-306). The mVEGFR-2 (2-3) construct comprises the sequence of mVEGFR-2 from amino acid proIinel26 to lysine329 (in the numbering of the precQTSor protein)- The construct also comprises, in addition to the Immunoglobulin-like C2-type domains 2 and 3, flanking regions preceding domain 2 and following domain 3 in the sequence of mVEGFR-2, to add amino acid spacer moieties. An amino acid linker containing a cysteine residue was fused to the C-terminus of the mVEGFR-2 sequence through cloning into pCEP-SP-EK-Fc* vector (EXAMPLE 1). The fragment of mVEGFR-2 cloned into pCEP-SP-EK-Fc* vector encoded the following amino acid sequence (SBQ ID NO:345):<br>
PFIAS VSDQHGIVYI TENKNKTWI PCRGSISNLN VSLCARYPEK RFVPDGNRIS WDSEIGFTLP SYMISYAGMV FCEAKINDET YQSIMYIVW VGYRTiDVIL SPPHEIELSA GEKLVLNCTA RTELNVGLDF TWHSPPSKSH HKKIVNRDVIC PFPGTVAKMF LSTLTIESVT KSDQCYTCV ASSGRMIKRN RTFVRVHTKP<br>
Expression of recombinant mVEGFR-2(2-3) in eukaryotic cells Recombinant mVEGFR-2(2-3) was expressed in NA 293 cells usmg the pCEP-SP-EK-Fc* vector. The pCEP-SP-EK-Fc* vector has a BamHI and an Nhel sites, encodes an amino acid linker containing one cysteine residue, an enteroMnase cleavage site, and C-terminally a human Fc region. The mVEGFR-2(2-3) v/as amplified by PCR with the primer pair BamHl-FLKl-F and Nhel-FLKl-B from a mouse 7-day embryo cDNA (Marathon-Ready cDNA, Clontech). For the PCR reaction, 10 pmol of each oligo and 0.5 ng of the cDNA (mouse 7-day embryo cDNA<br><br>
Marathon-Ready cDNA, Qontech) was used in the 50 â¢ 1 reaction mixture (1 â¢ 1 of Advantage 2 polymerase mix (SOx), 0.2 mM dNTPs and 5 â¢ 1 lOx cDNA PCR reaction buffer). The temperature cycles were as follows; 5 cycles a 94Â» C for 1 minute, 94* C for 30 seconds, 54* ,C for 30 seconds, 72Â» C for 1 minute followed by 5 cycles of 94- C (30 seconds), 54Â» C (30 seconds), 70Â» C (1 minute) and followed by 30 cycles 94* C (20 seconds), 54" C (30 seconds) and 68* C (1 minute). The PCR produa was digested with BamHl and Nbel and inserted into the pCEP-SP-EK-Fc* vector digested with the same enzymes. Resulting plasmid was named mVEGFR-2(2~ 3)-pCep-EK-Fc. All other steps were performed by standard molecular biology protocols.<br>
Oligos:<br>
1.	PcimerBamHl-FLKl-F 5"-CGCGGATCCATTCATCGCCTCTGTC-3" (SEQ ID NO:343)<br>
2.	Primer Nhel-FLKl-B 5"-CTAGCTAGCTrTGTGTGAACTCGGAC-3" (SEQ ID NO:344)<br>
Transfection and expression of recombinant mVEGFR-2(2-3) EBNA 293 cells were transfected with the mVEGFR-2(2-3)-pCep-Ek-Fc construct described above and serum free supernatant of cells was harvested for purification as described m EXAMPLE 1.<br>
Purification of recombinant mVEGFR-2(2-3) Protein A purification of the expressed Fc-EK-mVEGFR-2(2-3) proteins was performed as described in EXAMPLE 1. Subsequently, after binding of Ihe fusion protein to Protein A, mVEGER-2(2-3) was cleaved from the Fc portion bound to protein A using enterokinase (EnterokinaseMax, Inviliogen). Digestion was conducted over night at 37" C (2,5 units enterokinase/100 nl Protein A beads with bound fusion protein). The released VEGFR-2(2-3) was separated fix)m the Fc-porhon still bound to protein A beads by short caitrifugation in chromatography columns (Micro Bio Spin, Biorad). In order to remove the enterokinase the flow through was treated with enterokinase away (Invitrogen) according to the instmctions of the manufacturer.<br><br>
Example 12 Coupling of murine VEGFR-2 peptide to Q6 capsid protran, HbcAg-lys-2cys-Mut and PiU and immunization of mice witti VLP-peptide and Pili-<br>
peptide Taccines<br>
A.       Coupling of murine VEGFR-2 peptides to VLPs and pili<br>
The following peptides was chemically synthesized (by Eurogentec, Belgium): murine VEGFR-2 peptide CTARTELNVGLDFTWHSPPSKSHHKK and used for chemical coupling to Pili as described below.<br>
Coupling of murine VEGI-2 peptides to pili: A solution of 1400 fil of 1 mg/ml pili protein in 20 mM Hepes, pH 7.4, was reacted for 60 minutes with 851 of a 100 mM Sulfo-MBS (Pierce) solution in (H2O) at RT on a rocking shaker. The reaction mixture was desalted on a PD-10 column (Amersham-Phamiacia Biotech), The protein-containing fractions eluting from the colimm were pooled (containing approximately 1,4 mg protein) and reacted with a 2.5-fold molar excess (final volmne) of murine VEGFR-2 peptide respectively. For example, to 200 1 eluate containing approximately 0,2 mg derivatized pih, 2.4 1 of a 10 mM peptide solution (in DMSO) were added. The mixture was incubated for four hours at 25 "C on a rocking shaker and subsequently dialyzed against 2 liters of 20 mM Hepes, pH 7.2 overnight at 4*0. Coupling results were analyzed by SDS-PAGE under reducing conditions and are shown in FIG. 18 A. Supernatant (S) and pellet (P) of each sample were loaded on the gel, as well pili and pili derivatized with Sulfo-MBS cross-linker (Pierce). The samples loaded on the gel of HG. 18 A were the following; Lane 1: Marker proteins; lane 2-5: coupled samples (Pih murine: Pili coupled to murine peptide; Pili human: Pili coupled to human peptide); lane 6: pili derivatized with Sulfo-MBS cross-linker, lane 7-9; three fractions of the eluate of the PD-10 column. Fraction 2 is the peak fraction, fraction 1 and 3 are fractions taken at the border of ttie peak. Coupling bands were clearly visible on the gel, demonstrating the successful coupHng of murine VEGFR-2 to pili.<br><br>
Coupling of murine VEGFR-2 peptide to Op capsid protein: A solution of 1 ml of<br>
1	mg/ml QP capsid protein in 20 mM Hepes, 150 mM NaQ pH 7.4 was reacted for<br>
45 minutes with 20 1 of 100 mM Sulfo-MBS (Pierce) solution in (IftO) at RT on a<br>
rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours<br>
against 2 L of 20 mM Hepes, pH 7.4 at 4 *C. 1000 /tl of the dialyzed reaction<br>
mixture was then reacted with 12 ft] of a 10 mM peptide solution (in DMSO) for four<br>
hours at 25 "C on a rocking shaker. The reaction mixture was subsequently dialyzed<br>
2x2 hours against 2 liters of 20 mM Hepes, pH 7.4 at 4 *C, Coupling results were<br>
analyzed by SDS-PAGE under reducing conditions and are shown in FIG. 18 B.<br>
Supernatant (S) of each sample was loaded on the gel, as well as Q6 capsid protein<br>
and Q6 capsid protein derivatized with Sulfo-MBS cross-linker. Coupling was done<br>
in duplicate. The following samples were loaded on the gel;<br>
Lane 1: Marker proteins; lane 2, 5: Q6 capsid protein; lane 3, 6 Q6 capsid protein dmvatized with Sulfo-MBS; lane 4, 7: QB capsid protein coupled to murine VEGER-<br>
2	peptide. Coupling bands were clearly visible on the gel, demonstrating the<br>
successful coupling of murine VEGFR-2 to Q6 capsid protein.<br>
Coupling of murine "VEGFR-2 peptide to Hix:Ae-lvs-2cvs-Mut: A solution of 3 ml of 0.9 mg/ml cys-free HbcAg csid protein (EXAMPLE 31) in PBS, pH 7.4 was reacted for 45 minutes with 37,5 fil of a 100 mM Sulfo-MBS (Pierce) solution in (HjO) at RT on a rocking shaker. The reaction solution was subsequently dialyzed overnight against 2 L of 20 mM Hepes, pH 7.4. After buffer exchange the reaction solution was dialyzed for another 2 hours against the same buffer. The dialyzed reaction mixture was then reacted with 3 1 of a 10 mM peptide solution (in DMSO) for 4 hours at 25 *C on a rocking shaker. TTie reaction mixture was subsequently dialyzed against 2 liters of 20 mM Hepes, pH 7.4 overnight at 4 "C followed by buffer exchange and another 2 hours of dialysis against the same buffer. Coupling results were analyzed by SDS-PAGE under reducing conditions and are shown in FIG. 18 C. The supernatant (S) of each sanle was loaded on the gel, as well as HbcAg-lys-2cys-Mut protein and HbcAg-lys-2cys-Mut protein derivatized with Sulfo-MBS cross-linkfir. Coupling was done in dupUcate. Coupling reactions were conducted in a 2.5 fold and 10 fold molar excess of peptide. The following samples were loaded on the gel:<br><br>
Lane 1: Marker proteins; lane 2, 4, 6, 8: Supernatant (S) and pellet (P) of coupling<br>
reactions performed with 10 fold molar excess of peptide; lane 3,5, 7. 9: Supernatant<br>
(S) and pellet (?) of coupling reactions performed with 2.5 fold molar excess of<br>
peptide; lane 10: HbcAg-lys-2cy8"Mut derivatized with Sulfo-MBS; lane 11: HbcAg-<br>
lys-2cys-Mut<br>
Coupling bands were clearly visible on the gel, demonstrating the successful coupling<br>
of murine VEGFR-2 to HbcAg-lys-2cys-Mut protein.<br>
B.       Immunization of mice:<br>
Pili-peptide vaccine: Female C3H-HeJ (Toll-like receptor 4 deficient) and C3H-HeN (wild-type) mice were vaccinated with the murine VEGFR-2 peptide coupled to pili protein without the addition of adjuvants. Approximately 100 (ig of total protein of each sample was diluted in PBS to 200 1 and injected subcutaneously on day 0, day 14 and day 28. Mice were bled retroorbitally on day 14, 28 and day 42 and serum of day 42 was analyzed using a human VEGFR-2 specific ELISA.<br>
OB capsid protein-peptide vaccine : Female Black 6 mice were vaccinated with the murine VEGFR-2 peptide coupled to QB capsid protein with and without flie addition of adjuvant (Aluioiniuinhydroxid). Approximately 100 /xg of total protein of each sample was diluted jn PBS to 200 fil and injected subcutaneously on day 0, day 14 and day 28. Mice were bled retroorbitally on day 14, 28 and day 42 and serum of day 42 was analyzed using a human VEGFR-2 specific EUSA.<br>
HbcAE--lvs-2cvs-Mut vaccines: Female Black 6 mice were vaccinated with the murine VEGFR-2 peptide coupled to HbcAg-]ys-2cys-Mut protein with and without the addition of adjuvant (Aluminiumhydroxid). Approximately 100 /ig of total protein of each sample was diluted in PBS to 200 fi\ and injected subcutaneously on day 0, day 14 and day 28. Mice were bled retroorbitally on day 14, 28 and day 42 and seruin of day 42 was<br><br>
analyzed using a human VEGFR-2 specific ELISA.<br>
C. EUSA Sera of immunized mice were tested in ELISA with immobilized murine VEGFR-2 peptide. Murine VEGER-2 peptide was coupled to bovine RNAse A using the chemical cross-linker Sulfo-SPDP. ELISA plates were coated with coupled RNAse A at a concentration of 10 /ig/ml. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse 1 antibody. As a control, preunmune sera of the same mice wwe also tested. Control HJSA experiments using sera from mice immunized with uncoupled carrier showed that the antibodies detected were specific for the respective peptide. Tlie results are shown in Figure 4-6.<br>
Pili-peptide vaccine: The result of the EUSA is shown in FIG. 18 D. Results for indicated serum dilutions are shown as optical density at 450 um. The average of three mice each (including standard deviations) are shown. All vaccinated mice made IgG antibody titers against the murine VEGFR-2 peptide. No difference was noted between mice deficiait for the Toll-like receptor 4 and wild-type mice, demonstrating the iromunogenicity of the self-antigen murine VEGFR-2 peptide, when coupled to pili, in mice. TTie vaccines injected in the mice are designating the corresponding analyzed sera.<br>
QB capsid protein-peptide vaccine: Results for indicated serum dilutions are shown in FIG. 18 E as optical density at 450 nm. The average of two mice each (including standard deviations) are shown. AU vaccinated mice made IgG antibody titers against the murine VEGFR-2 peptide, demonstrating the immunogenicity of the self-antigen murine VEGFR-2 peptide, when coupled to QB capsid protein, in mice. The vaccines injected in the mice are designating the corresponding analyzed sera.<br>
E[bcAg-lvs-2cvs-Mut vaccine: Results for indicated serum dilutions are shown in FIG. 18 F as optical density at<br><br>
450 nm. The average of three mice each (including standard deviations) are shown. All vaccinated mice made IgG antibody liters against the murine VEGFR-2 peptide, demonstrating the immunogenicity of the self-antigen murine VEGFR-2 peptide, when coupled to QB capsid protein, in mice. Tlie vaccines injected in the mice are designating the corresponding analyzed sera.<br>
EXAMPLE 13 Coupling of AP1-15 peptides to HBc-Ag-I}"s-2cj8&gt;Mut and fr capsid protein<br>
The following AP peptide was chemically synthesized (DAEFRHDSGYEVHHQGGC) , a peptide which comprises the amino acid sequence from residue 1-15 of human AP, fused at its C-teiminus to the sequence GGC for coupling to VUPs and Pili.<br>
A.       a.) Coupling of Ap 1-15 peptide to HBc-Ag-Iys-2cys-Mut using the cross-UnkerSMPH.<br>
A solution of 833.3 pi of 1.2 mg/ml HBc-Ag-lys-2cys-Mut protein in 20 mM Hepes 150 mM NaCl pH 7.4 was reacted for 30 minutes with 17 1 of a solution of 65 mM SMPH (Pierce) in H2O, at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 1 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 *C in a dialysis tubing with Molecular Weight cutoff 10000 Da. 833.3 \JX of the dialyzed reaction raixtiire was then reacted with 7.1 Ml of a 50 mM peptide stock solution (peptide stock solution in PMSO) for two hours at 15Â°C on a rocking shalrer. The reaction mixture was subsequentiy dialyzed overnight against 1 liters of 20 mM pes, 150 mM NaCl, pH 7.4 at 4 ""C. The sample was then frozen in aliquots in liquid Nitrogen and stored at -80Â°C until immunization of the mice.<br><br>
b) Coupling of AP 1-15 peptide to fr capsid protein using the cross-linker SMPH..<br>
A solution of 500 jU of 2 mg/ml fr capsid protein in 20 mM ifepes 150 mM NaCl pH 7.4 was reacted for 30 minutes with 23 /il of a solution of 65 mM SMPH (Pierce) in H2O, at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 1L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 *C in a dialysis tubing with Molecular Weight cutoff 10000 Da. 500pJ of the dialyzed reaction mixture was then reacted with 5.7 fil of a 50 mM pepticte stock solution (peptide stock solution in DMSO) for two hours at 15""C on a rocking shaker. Tlie reaction mixture was subsequentiy dialyzed overnight against 1 liter of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 "C. The sample was then frozen in aliquots in liquid Nitrogen and stored at -80Â°C until immunization of the mice. Samples of the coupling reaction were analyzed by SDS-PAGE under reducing conditions.<br>
The results of the coupling experinKints were analyzed by SDS-PAGE, and are shown in FIG. 19 A. Clear coupling bands corresponding to the coupling of Ap 1-15 either to fr capsid protein or to HBc-Ag-Iys-2cys-Mut were visible on the gel, and are indicated by arrows in the figure, demonstrating successful coupling of Ap 1-15 to fr capsid protein and to HBc-Ag-lys-2cys-Mut capsid protein. MultinIe coupling bands were visible for the coupling to fr capsid protein, while noainly one coupling band was visible for HBc-Ag-lys-2cys-Mut.<br>
The following samples were loaded on the gel of HG. 19 A. 1: Protein Marker (kDa Marker 7708S BioLabs. Molecular weight marker bands from the top of the gel: 175, S3, 62, 47.5, 32.5, 25, 16.5, 6.5 kDa). 2: derivatized HBc-Ag-lys-2cys-MuL 3: HBc-Ag-lys-2cys-Mut coupled with Apl-15, supematant of the sample taken at the end of the coupling reaction, and centrifuged. 4: HBc-Ag-lys-2cys-Mut coupled with Apl-15, peUet of the sample taken at the end of the coupling reaction, and centrifuged. 5: derivatized fr capsid protein. 6: fr capsid protein coupled with Apl-15, supernatant of the sample taken at the end of the couoline reaction, and<br><br>
centrifiiged. 4: fr capsid protein coupled with Api-15, pellet of the sample taken at the end of the coupling reaction, and centrifuged.<br>
B.	Immunization of Balb/c mice<br>
Female Ba)b/c mice were vaccinated twice on day 0 and day 14 subcutaneously with either 10 fig of fr capsid protein coupled to AB 1-15 (Fr-AB 1-15) or 10 ng of HBc-Ag-lys-2cys-Mut coupled to to A6 1-15 (HBc-Apl-15) diluted in sterile PBS. Mice were bled retroorbitally on day 22 and sera were analysed in an AB-l-15-specific EUSA.<br>
C.	EUSA<br>
The AP 1-15 peptide was coupled to bovine RNAse A using the chemical cross-linker sulfo-SPDP. ELISA plates were coated with AB 1-15-RNAse conjugate at a concentration of 10 fig/ml. The plates were blocked and then incubated with serially diluted serum samples. Bound antibodies were detected with enzymatically labeled anti-mouse IgG. As a control, serum from a naive mouse was also tested.<br>
Shown on HG. 19 B are the ELISA signals obtained on day 22 with the seni of the mice immunized with vaccines Fr-AB 1-15, and HBc-Apl-l5 respectively. A control senmi from a naive mouse (preimmune serum) was also included. Results from different serum dilutions are shown as optical density at 450 mn. Average results from three vaccinated mice each are shown. All vaccinated mice had AB I-15-specific IgG antibodies in their soimi.<br>
EXAMPLE 14 Coupling of AP 1-15, Ap 1-27 and Ap 33-42 peptides to Type I PUi<br>
Coupling of AP 1-15, AP 1-27 and AP 33-42 peptides to Pili using the cross-linker SMPH.<br>
The following Ap peptides were chemically synthesized: DAEFRHDSGYEVHHQGGC ("Ap 1-15"), a peptide which comprises the amino acid sequence from residue 1-15 of human Ap, fused at its C-terminus to the sequence GGC for coupling to Pili and VXPs,<br>
DAEFRHDSGYEVHHQKLVFFAEDVGSNGGC ("Ap 1-27") a peptide which comprises the amino acid sequence from residue 1-27 of human Ap, fused at its C-terminus to the sequence GGC for coupling to Pili and VIPs, and CGHGNKSGLMVGGWIA ("AP 33-42") a peptide which comprises the amino acid sequence from residue 33-42 of AP, fused at its N-terminus to the sequence CGHGNKS for coupling to Pili and VLPs. All three peptides were used for chemical coupling to Pili as described in the following.<br>
Asolution of 2ml of 2mg/ml Pili in 20 mM Hepes 150mM NaCl pH 7.4 was reacted for 45 minutes with 468 1 of a solution of 33.3 mM SMPH (Pierce) in H2O, at 25 *C on a rocking shaker. The reaction solution was loaded on a PD10 column (Pharmacia) and eluted with 6 X 500 pu of 20 mM Hepes 150mM NaCl pH 7.4. Fractions were analyzed by dotting on a Nitrocellulose (Schleicher &amp; Schuell) and stained with Amddoblack. Fractions 3-6 were pooled. The samples were then frozen in aliquots in liquid Nitrogen and stored at -80Â°C until coupling.<br>
200 u1 of the thawed desalted reaction mixture was then mixed with 200 fil DMSO and 2.5 \il of each of the corresponding 50 mM peptide stock solutions in DMSO, for 3.5 hours at RT on a rocking shaker. 400 JJJ of the reaction mixture was subsequently dialyzed three times for one hour against 1 liter of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 "C in a dialysis tubing with Molecular Weight cutoff 10000 Da. The samples were then frozen in aliquots in liquid Nitrogen and stored at -80Â°C<br>
Sample preparation for SDS-Page was performed as follows: 100 )il of<br><br>
the dialyzed coupling reaction was incubated for 10 minutes in 10 % TCA on ice and subsequently centrifiiged. The pellet was resuspended in 50 fl 8.5 M Guanidine-HCl solution and incubated for 15 minutes at 70Â°C. The samples were then precipitated with ethanol, and after a second centrifugation step, the pellet was resuspended in sample buffer.<br>
The results of the coupling experiments were analyzed by SDS-PAGE under reducing conditions. Clear coupling bands were visible for all three peptides, demonstrating coupling of AP peptides to Pili.<br>
EXAMPLE 15 Vaccination of APP23 mice with A peptides coupled to QP capsid<br>
protein<br>
A.	Immunization of APP23 nuce<br>
Three different AS peptides (AB l-27-Gly-Gly-Cys-NH2; H-Cys-Gly-His-Gly-Asn-Lys-Ser-A6 33-42; A6 1-15-Gly-Gly-Cys-NH2) were coupled to Qp capsid protein. The resulting vaccines were termed "Qb-Ab 1-15", "Qb-Ab 1-27" and "Qb~ Ab 33-42". 8 months old female APP23 mice which carry a human APP transgene (Sturchler-Pierrat er a/.. Proc.Natl.Acad.Sci. USA94: 13287-13292(1997)) were used for vaccination. The mice were injected subcutaneously with 25 fig vaccine diluted in sterile PBS and 14 days later boosted with the same amount of vaccine. Mice were bled from the tail vein before the start of immunization and 7 days after the booster injection. The sera were analyzed by ELIS A.<br>
B.	EUSA<br>
AP 1-40 and AP 1-42 peptide stocks were made in DMSO and diluted in coating buffer before use. ELJSA plates were coated with 0.1 fig /well Ap 1-40 or Ap 1-42 peptide. The plates were blocked and then incubated with serially diluted mouse serum. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, sera obtained before vaccination were also included. The<br><br>
serum dilution showing a mean three standard deviations above baseline was calculated and defined as "ELISA titer". All three vaccines tested were immunogenic in APP23 mice and induced high antibody titers against the A6 peptides 1-40 and/or AB 1-42. TTie results are shown in FIG. 20. No specific antibodies were detected in preimmune sera of the same mice (not shown).<br>
Shown on FIG. 20 are the EUSA signals obtained on day 22 with the sera of the mice immunized with vaccines Pr-AB 1-15, and HBc-Apl-15 respectively. A control serum from a nai"ve mouse (preimmune serum) was also included. Results fit&gt;m different serum dilutions are shown as optical density at 450 nm. Average results from three vaccinated mice each are shown.<br>
Mice A21-A30 received the vaccine Qb-Ab 1-15, mice A31-A40 received Qb-Ab 1-27 and mice A41-49 received Qb-Ab 332. For each mouse, Ap 1*0 and Ap 1-42 peptide-specific serum antibody titers were determined on day 21 by ELISA. The ELSIA titers defined as the serum dilution showing a mean three standard deviations above baseline are shown for individual mice. Mice vaccinated with Qb-Ab 1-15 or Qb-Ab 1-27 made high antibody titers against botii Ap 1-40 and Ap 1-42 whereas mice vaccinated with Qb-Ab 33-42 had only high antibody titers against the AP 1-42 peptide.<br>
EXAMPLE 16<br>
Coupling of Fab antibody fragmraits to QP capsid proton<br>
A solution of 4.0 mg/ml QP capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 was reacted fCM- 30 minutes with a 2.8 mM SMPH (Pierce) (firom a stock solution dissolved in DMSO) at 25Â°C on a rocldng shaker. The reaction solution was sutsequentiy dialyzed twice for 2 hours against 21 of 20 mM Ifepes, 150 mM NaQ, pH7.2at4Â°C.<br>
The Fab fragment of human IgG, produced by papain digestion of human IgG, waspurchasedfrom Jackson Immunolab. This solution (11. Img/ml) was diluted to a concentration of 2.5mg/ml in 20 mM Hepes, 150 mM NaCl pH 7.2 and allowed to react with different concentrations (0-1000 \JM) of either dithiothreitol (DTT) or tricarboxyethylphosphine (TCEP) for 30 minutes at 25Â°C.<br><br>
Coupling was induced by mixing the derivatized and dialysed QP capsid protein solution with non-reduced or reduced Fab solution (final concentrations: 1.14 mg/ml Q3 and 1.78 mg/ml Fab) and proceeded ovemit at 25 Â°C on a rocking shaker.<br>
The reaction products were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were stained with Coomassie Brilliant Blue. The results are shown in, FIG. 21.<br>
A coupling product of about 40 kDa could be detected in samples in which the Fab had been reduced before coupling by 25-1000 \JM TCEP and 25 - 100 pM DTT (FIG. 21, arrow), but not at 10 fiM TCEP, 10 pM DTT or 1000 \iM DTT. The coupled band also reacted with an anti-QP antiserum (data not shown) clearly demonstrating the covalent coupling of the Fab fragment to Qp capsid protein.<br>
Ilie samples loaded on the gel of HG. 21wMie the foUowing:-<br>
Lane 1: Molecular weight marker. Lane 2 and 3: derivatized QP capsid protein before coupling. Lane 4-13: QP-Fab coupling reactions sftet reduction of Fab with 4: QP-Fab coupling reactions after reduction of Fab with 10 pM TCEP. 5: QP-Fab coupUng reactions after reduction of Fab with 25 pM TCEP. 6: Qp-Fab coupling reactions after reduction of Fab with 50 pM TCEP, 7: Qp-Fab coupHng reactions after reduction of Fab with 100 pM TCEP. 8: Qp-Fab coupling reactions after reduction of Fab with 1000 pM TCEP. 9: QP-Fab coupling reactions after reduction of Fab with 10 pM DTT. 10: QP-Fab coupling reactions after reduction of Fab with 25 pM DTT. 11: QP-Fab coupling reactions after reduction of Fab with 50 pMDTT. 12: Qp-Fab coupUng reactions after reduction of Fab with 100 pM DTT. 13: QP-Fab coupling reactions after reduction of Fab with 1000 pM DTT. L-ane 14: Fab before coupling. Tte gel was stained with Coomassie Brilliant Blue. Molecular weights of marker proteins are given on the left margin. Tlie arrow indicates the coupled band.<br><br>
EXAMPLE 17 Vaccination of APP23 mice wiUi A peptides coupled to Qp capsid protein<br>
A.	Immunization of APP23 mice<br>
Three different A6 peptides (A6 l-27-GIy-Gly-Cys-NH2; H-Cys-GIy-His-Gly-Asn-Lys-Ser-AS 33-42; A6 1-15-Gly-Gly-Cys-NH2) were coupled to Qp capsid protein. The resulting vaccines were teimed "Qb-Ab 1-15", "Qb-Ab 1-27" and "Qb-Ab 33-42". 8 months old female APP23 mice which carry a human APP transgene (Sturchler-Pierrat et al, ProcNatl Acad.ScL USA 94:13287-13292 (1997)) were used for vaccination. The mice were injected subcutaneously with 25 ig vaccine diluted in sterile PBS and 14 days later boosted with the same amount of vaccine. Mice were bled from the tail vein before the start of immunization and 7 days after the booster injection. The sera were analyzed by ELISA.<br>
B.	EUSA<br>
AP 1-40 and AP 1-42 peptide stocks were made in DMSO and diluted in coating buffer before use. EUSA plates were coated with 0.1 p.g /well Ap 1-40 or AP 1-42 peptide. The plates were blocked and then incubated with serially diluted mouse serum. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, sera obtained before vaccination were also included. The serum dilution showing a mean three standard deviations above baseline was calculated and defined as "EUSA titer". All three vaccines tested were immunogenic in APP23 mice and induced high antibody titers against the A6 peptides 1-40 and/or AB 1-42. The results are shown in HG. 20. No specific antibodies were detected in preimmune sera of the same mice (not shown).<br>
Shown on FIG. 20 are the ELISA signals obtained on day 22 with the sera of the mice immunized with vaccines Qb-Ab 1-15, Qb-Ab 1-27 and Qb-Ab 33-42, respectively. Mice A21-A30 received Ihe vaccine Qb-Ab 1-15, mice A31-A40 received Qb-Ab 1-27 and mice A41-49 received Qb-Ab 33-42. For each mouse, AP 1-40 and AP 1-42 peptide-specific serum antibody titers were determined on day 21 by EUSA. TTie ELSIA titers defined as the serum dilution showing a mean three standard deviations above baseline are shown for individual mice. Mice vaccinated<br><br>
	-1<br>
with Qb-Ab 1-15 or Qb-Ab 1-27 made high antibody titers against both Ap 1-40 and AP 1-42 whereas mice vaccinated with Qb-Ab 33-42 had only high antibody titers against the AP 1-42 peptide. The very strong immune responses obtained with the human A6 peptides in the transgenic mice expressing human A6 transgene, demonstrate that by coupling AP peptides to QP capsid protein, tolerance towards the self-antigen can be overcome.<br>
F.YA\fPT.F1Â«<br>
Construction, e]q)rÂ«9on and purification of mutant QP coat proteins Construction of pQp-240<br>
The plasmid pQplO (Kozlovska, TM, et al.. Gene ii7:133-137) was used as an initial plasmid for the construction of pQp-240. The mutation Lysl3-&gt;Arg was created by inverse PCR. The inverse primes were designed in inverted tail-to-tail directions:<br>
5"-GGTAACATCGGTCGAGATGGAAAACAAACrCTGGTCC-3" and<br>
5"-GGAGCAGAGTTTGTnTCCATCTCGACCGATGTTACC-3".<br>
The products of the first PCR were used as templates for the second PCR reaction, in which an upstream primer<br>
5"-AGCrCGCCCGGGGATCCTCTAG-3" and a downstream inimer<br>
5"-CGATGCATITCATCCTrAGTrATCAATACGCrGGGTrCAG-3" were used. The product of the second PCR was digested with Xbal and MphllOSI and cloned into the pQplO expression vector, which was cleaved by the same restriction enzymes. The PCR reactions were performed with PCR kit reagents and according to producer protocol (MBI Fermentas, Vilnius, lithuania).<br>
Sequencing using the direct label incorporation method veiified the desired mutations. E.coli cells haibouring pQP-240 supported efficient synthesis of 14-kD protein co migrating upon PAGE with control Qp coat protan isolated from Qp phage particles.<br><br>
Resulting amino acid sequence: (SEQ ID NO: 255) AIOTVTLGNIGRDGKQTLVLNPRGVNPTNGVASLSQAGAVP AIKRVTVSVSQPSRNRKNYKVQVKIQKPTACTANGSCDPSVTRQ K YAP VTPSFTQ YSTPEERAFVRTELAALLASPT .T .TDAIDQLNPA Y Construction of pQP-243<br>
The plasmid pQPlO was used as an initial plasmid for the construction of pQP-243.The mutation AsnlO-Lys was created by inverse PCR. Tlie inverse primers were designed in inverted taU-to-tail directions:<br>
5"-GGCAAAATTAGAGACTGTTACnTAGGTAAGATCGG -3" and 5"-CCGATCrrACCTAAAGTAACAGTCTCTAArnTGCC -3". The products of the first PCR were used as templates for the second PCR reaction, in which an upstream primer<br>
5"-AGCTCGCCCGGGGATCCTCTAG-3" and a downstream primer 5"-CGATGCATrTCATCCTrAGTTATCAATACGCTGGGTTCAG-3" were used. The product of the second PCR was digested with Xbal and Mphll03I and cloned into the pQ 10 expression vector, which was cleaved by the same restriction enzymes. The PCR reactions were performed with PCR kit reagents and according to producer protocol (MBI Fennentas, Vilnius, Lithuania).<br>
Sequencing using the direct label incorporation method verified the desired mutations. B.coli cells harbouring pQp-243 supported efficient synthesis of 14-kD protein co migrating upon PAGE with control QP coat protein isolated from QP phage particles.<br>
Resulting amino acid sequence: (SEQ ID NO: 256}<br>
AKLETVTLGKIGKDGKQTLVLNPRGVNPTNGVASLSQAGAVP ALEKRVTVSVSQPSRNRKNYKVQVKIQNPTACTANGSCDPSVTRQ KYADVTFSFTQYSTDEERAFVRTELAALLASPLUDAIDQIJAY Construction of pQP-250<br>
The plasmid pQP-240 wÂ£ used as an initial plasmid for the construction of pQP-250. The mutation Lys2-*Arg was created by site-directed mutagenesis. An<br><br>
â <br>
upstream primer<br>
5"-GGCCATGGCACGACTCGAGACTGTTACnTAGG-3" and a downstream primer 5"-GATTTAGGTGACACTATAG-3" were used for the synthesis of the mutant PCR-fragment, which was introduced into the pQp-185 expression vector at the unique restriction sites Ncol and Hindlll. The PCR reactions were performed with PCR kit reagents and according to producer protocol (MBI Fermentas, Vihiius, Lithuania).<br>
Sequencing using the direct label incorporation method verified the desired mutations. Kcoli cells harbouring pQP-250 supported efficient synthesis of 14-kD protein co migrating upon PAGE with control Qp coat protein isolated from QP phage particles.<br>
Resulting amino acid sequence: (SEQ ID NO: 257)<br>
ARLETVTLGKIGRDGKQTLVLNPRGVNPTWGVASLSQAGAVP ALEKRVTVSVSQPSRNKKNYKVQVKIQKPTACTANGSCDPSVTRQ KYADVTF SPTQYSTDEERAFVRTEIiAALLASPLLlDAIDQIjKEAy<br>
Construction of pQP-25i<br>
The plasmid pQPlO was used as an initial plasmid for the construction of pQP-251. The mutation Lysi6-*Arg was created by inverse PCR. The inverse primers were designed in inverted tail-to-tail directions:<br>
5"-GATGGACGTCAAACrCrGGTCCTCAATCCGCGTGGGG -3" and<br>
5"-CCCCACGCGGATrGAGGACCAGAGTrrGACGTCCATC-3".<br>
The products of the first PCR were used as templates for the second PCR reaction, in which an upstream primra"<br>
5"-AGCTCGCCCGGGGATCCTCTAG-3" and a downstream primer<br>
S"-CGATGCArrTCATCCTTAGTrATCAATACGCrGGGrrCAG-S" were used. The product of the second PCR was digested with Xbal and Mphll03l and cloned into the pQplO expression vector, which was cleaved by the same restriction enzymes. The PCR reMons were performed with PCR kit reagents and according to producer protocol (MBI Fermentas, Vilnius, Lithuania).<br>
Sequencing using the direct label incorporation method verified the desired<br><br>
mutations. Kcoli cells harbouring pQP-251 supported efficient synthesis of 14-ld) protein co migrating upon PAGE with control Qp coat protein isolated from Q3 phage particles. The resulting amino acid sequence encoded by this construct is shown in SEQ. ID NO: 259.<br>
Construction of pQp-259<br>
Hie plasmid pQP-251 was used as an initial plasmid for the construction of pQp-259. The mutation Lys2-&gt;Arg was created by site-directed mutagenesis. An upstream primer<br>
5"-GGCCATGGCACGACTCGAGACTGTrACTTrAGG~3" md a downstream primer 5"-GAlTTAGGTGACACTATAG-3" were used for the synthesis of the mutant PCJl-fragment, which was introduced into the pQp-185 expression vector at the unique restriction sites Ncol and Bindlll. The PCR reactions were perfonned with PCR kit reagents and according to producer protocol (MBI Fermentas, Vilnius, Lithuania).<br>
Sequencing using the direct label incorporation method verified the desired mutations. Kcoli cells harbouring pQP-259 supported efficient synthesis of l4-kD protein co migrating upon PAGE with control Qp coat protein isolated from Qp phage particles.<br>
Resulting amino acid sequence: (SEQ ID NO: 258)<br>
AKLETVTLGNIGKDGKQTLVLMPRGVNPTNGVASLSQAGAVP ALEKRVTVSVSQPSRMRKNYKVQVKIQNPTACTANGSCDPSVTRQ KYADVTF SFTQYSTDEERAFVRTELAALLAS PLLIDAIDQLNPAY<br>
General procedures for EiqiressioD and purification of Qp and Qp<br>
mutants<br>
Expression<br><br>
Transform Exoli JM109 with Q-beta expression plasmids. Inoculate 5 ml of LB liquid medium with 20 â¢ g/ml ampicillin with clones transfonned with Q-beta expression plasmids. Incubate at 37 Â°C for 16-24 h without shaking.<br>
Inoculate 100-300 ml of LB medium, containing 20* g/ml, 1:100 with the prepared inoculum. Incubate at 37 Â°C overnight without shaking. Inoculate M9 + I % Casamino acids + 0.2 % glucose medium in flasks with the prepared inoculum 1:50, incubate at 37 "C overnight under shaking.<br>
Purification<br>
Solutions and buffers for the purification procedure; 1. Lysis buffer Lg<br>
SOmM Tris-HCl pH8,0 with 5mM EDTA , 0,1% tritonXlOO and fresh! prepared PMSF till Smicrograms per ml.Without lysozyme and DNAse.<br>
2.SAS<br>
Saturated ammonium sulphate in water<br>
3.	Buffer NET.<br>
20 mM Tris-HCl, pH 7.8 with 5mM EDTA and ISOmMNaQ.<br>
4.	PEG<br>
40% (w/v) polyelhylengiycol 6000 in NET<br>
Disruption and lyses<br>
Frozen cells were resuspended in LB at 2 ml/g cells. The mixture was sonicated â with 22 kH five times forlS seconds, with intervals oi Innn to cool the<br><br>
solution on ice. The lysate was then centrifuged at 14 000 rpm, for Ih using a Janecki K 60 rotor. The centrifugation steps described below were all performed using the same rotor, except otherwise stated. The supernatant was stored at 4Â° C, while cell debris were washed twice with LB, Aftercentrifugation, thesupematantsof thelysate and wash fractions were pooled.<br>
Fractionation<br>
A saturated ammomum sulphate solution was added dropwise under stirring to the above pooled lysate. The volume of the SAS was adjusted to be be one fifth of total volume, to obtain 20% of saturation. The solution was left standing overnight, and was centrifuged the next day at 14 000 rpm, for 20 min. The pellet was washed with a small amount of 20% ammonium sulphate, and centrifuged again . The obtained supematants were pooled, and SAS was added dropwise to obtain 40% of saturation. The solution was left standing overnight, and was centrifuged the next day at 14 000 rpm, for 20 min. The obtained pellet was solubilised in NET buffer.<br>
Chromatography<br>
The capsid protein resolubilized in NET buffer was loaded on a Sepharose CL- 4B column. Ttiree peaks eluted during chromatogrhy. The firet one mainly contained membranes and membrane fragments, and was not collected Capsids were contained in the second peak, while the third one contained other E.coli proteins.<br>
The peak fractions were pooled, and the NaCI concentration was adjusted to a final concentration of 0.65 M. A volume of PEG solution corresponding to one half of the pooled peak fraction was added dropwise under stirring. The solution was left to stand overnight without stirring. The capsid protein was sedimented by centrifugation at 14 000 rpm for 20 min. It was then solubilized in a minimal volume of NET and loaded again on the Sepharose CL- 4B column. The peak fi-actions were<br><br>
pooled, and precipitated with ammonium sulphate at 60% of saturation (w/v). After centrifugation and resolubilization in NET buffer, capsid protein was loaded on a Sepharose CL-6B column for lechromatography.<br>
Dialysis and drying<br>
The peak fractions obtained above were pooled and extensively dialysed against sterile water, and lyophilized for storage.<br>
Expression and purification Qp-240<br>
Cells (E. coli JM 109, transformed with the plasmid pQP-240) were resuspended in LB, sonicated five times for 15 seconds (water ice jacket) and centrifiiged at 13000 rpm for one hour. The supernatant was stored at 4""C until further processing, while the debris were washed 2 times with 9 ml of LB, and finally with 9 ml of 0,7 M urea in LB. All supematants were pooled, and loaded on the Sepharose CL-4B column. The pooled peak fi-actions were precipitated with ammonium sulphate and centrifuged. The resolubilized protein was then purified fiuther on a Sepharose 2B colmnn and finally on a Sepharose 6B column. Tlie capsid peak was finally extensively dialyzed against water and lyophilized as described above. The assembly of the coat protein into a csid was confirmed by electron microscopy.<br>
Expression and purification Qp-243<br>
Cells (Â£â . coli RRl) were resuspended in LB and processed as described in the general procedure. The protein was purified by two successive gel filtration steps on the sepharose CL-4B column and finally on a sepharose CL-2B column. Peak fractions were pooled and lyophilized as described above. The assembly of the coat<br><br>
protein mto a capsid was continned by electron microscopy.<br>
Expression and purification of 0-250<br>
Cells {Â£â . coli JM 109, transformed with pQP-250) were resuspended in LB and processed as described above. The protein was purified by gel filtration on a Sepharose CL-4B and finally on a Sepharose CL-2B column, and lyophilized as described above. The assembly of the coat protein into a capsid was confirmed by electron microscopy.<br>
Expression and puriflcatioa of QP-259<br>
Cells (E. coli  JM 109, transformed with pQP-259 ) were resuspended in LB and sonicated. The debris were washed once with 10 ml of LB and a second time with 10 ml of 0,7 M urea in LB. The protein was purified by two gel-fiJtration chromatogaphy steps, on a Sepharose CLA B column. The protein was dialyzed and lyophilized, as described above. The assembly of the coat protein into a capsid was confirmed by electron microscopy,<br>
EXAMPLE 19 Desensitizatioii of allergic mice with PLA2 coupled to Qp csid protein<br>
C.      Desensitization of allM"gic mice by vaccination<br>
Female CBA/J mice (8 weeks old) were sensitized with PLA2: Per mouse, 0.1 ug PLA2 from Latoxan (France) was adsoibed to 1 mg Alum (Imject, Pierce) in a total volume of 66 ul by vortexing for 30 min and then injected subcutaneously. This procedure was repeated every 14 days for a total of four times. This treatment led to the development of PLA2-specific serum IgE but no IgG2a antibodies. 1 month after the last sensitization, mice were injected subcutaneously with 10 \i% vaccine consisting of recombinant PLA2 coupled to Qp capsid protein. One and 2 weeks later they were again treated with the same amount of vaccine. One week after the last<br><br>
treatment, mice were bled and then challenged intraperitoneally with 25 |xg PLA2 (Latoxan) and rectal temperature was measured for 60 min using a calibrated digital thermometer. As a control sensitized mice which had not been treated with QP capsid protein-PLA2 were used. Whereas all control mice experienced an anaphylactic response reflected in a dramatic drop in rectal temperature after PLA2 challenge, vaccinated mice were fully or at least partially protected. Results are shown in FIG 25 A.<br>
B.      ELISA<br>
ELISA plates (Maxisorp, Nunc) were coated with PLA2 (Latoxan) at 5 Hg/ml. The plates were blocked and then incubated witti serially diluted serum. For the detection of IgE antibodies, serum was pretreated with protein G beads (Pharmacia) for 60 min on a shaker at room temperature. The beads were removed by centrifiigation and the supernatant was used for EUSA. Antibodies bound to P1A2 were detected with enzymatically labeled anti-mouse IgG2a or IgE antibodies. ELISA titers were determined at half maximal optical density (OD50%) and expressed as -logS of 100-fold prediltued sera for IgG2a and as -log5 of 10-fold prediluted sera for IgE. For all mice, PLA2-specific IgG2a and IgE in serum were deteamined before and at the end of Ae vaccine treatment Vaccination led to a dramatic increase of PLA2-specific IgG2a whereas no consistent changes in IgE titers were noted. These results indicate that the vaccination led to an induction of a Thl-Uke immune response (reflectedby the production of IgG2a). Results are shown in FIG. 25 B.<br>
The Anaphylactic response in vaccinated and non-vaccinated mice is shown in FIG. 25A.<br>
Mice were sensitized to PLA2 and then treated 3x subcutaneously with 10 \ig vaccine consisting of PLA2 coupled to QP capsid protein. Control mice were sensitized but not vaccinated. One week after the last vaccination all mice were challenged intraperitoneally with 25 ng PLA2 and the anaphylactic response was monitored by measuring the rectal temperature for 60 min. Whereas all control mice showed a dramatic drop in body temperature, vaccinated mice were fiilly or at least<br><br>
partially protected from an anaphylactic reaction.<br>
The induction of PLA2-Specific IgGla by vaccination is shown in FIG. 25 B.<br>
Mice were sensitized to PLA2 and then treated 3x with 10 ug vaccine consisting of PLA2 coupled to Qp capsid protein. Control mice were sensitized but not vaccinated. Serum was taken from sensitized mice before the start of the treatment and after completion of treatment, before challenge. In vaccinated mice 0eft hand of panel) a dramatic increase of PLA2-specific IgG2a was observed.<br>
EXAMPLE 20 Expression, refolding, purification and coupling of HarCys (also called FLA2 fusion protein)<br>
Expression and preparation of inclusion bodies<br>
The pETlla Plasraid containing the PLA2-Cys gene of example xxx was transformed into E. coli BL21DE3Rill (Stratagene). An overnight culture was grown in dYT medium containing 100 |i,g/ml Ampicillin and 15 g/ml Chloramphenicol. The culture was diluted in fresh dYT medium containing Ampicillin and Chloranhenicol, and grown at 3TC until OD ajo iun= 1 was reached, ITie culture was induced with 1 mM IPTG, and grown for another 4 hours. Cells were collected by centrifugation, and resuspended in PBS buffer containing 0.5 mg/ml Lysozyme. After incubation on ice, cells were sonicated on ice, and MgCl2 added to a concentration of 10 mM. 6 jiJ of Benzonase (Merck) were added to the cell lysate, and the lysate was incubated 30 minutes at RT. Triton was added to a final concentration of 1 %, and the lysate was further incubated for 30 minutes on ice. The inclusion body (IB) pellet was collected by centrifugation for 10 minutes at 13000 g. The inclusion body pellet was washed in wash buffer containing 20 mM Tris, 23% sucrose, I mM EDTA, pH 8.0. The IBs were solubilized in 6 M Guanidinium-HCl, 20 mM Tris, pH 8.0, containing 200 mM DTT. Hie solubilized IBs were centrifuged at 50000 g and the supernatant dialyzed against 6 M Guanidinium-HCl, 20 mM Tris, pH 8.0 and subsequently against fee same buffer containing 0.1 mM DTT. Oxidized glutathion<br><br>
was added to a final concentration of 50 mM, andihe solubilized .IBs were incubated for 1 h. at RT. The solubilized IBs were dialyzed against 6 M Guanidinium-HCL, 20 mM Tris, pH 8.0. The concentration of the EB solution was estimated by Bradford analysis and SDS-PAGE.<br>
B.	Refolding and purification<br>
The IB solution was added slowly in three portions, every 24 h., to a final concentration of 3 jiM, to the refolding buifer containing 2 mM EDTA, 0.2 mM Benzamidin, 0.2 mM 6 aminocapronic acid, 0.2 mM Guanidinium-HCl, 0.4 M L-Arginin, pH 6.8, to which 5 mM reduced Glutathion and 0.5 mM oxidized Glutathion were added prior to initiation of refolding at 4Â°C. The refolding solution was concentrated to one half of its volume by Ultrafiltration using a YMIO membrane (Millipore) and dialyzed against PBS, pH 7.2, containing 0.1 mM DTP. The protein was further concentrated by ultraiiltration and loaded onto a Superdex G-75 column (Phaimacia) equilibrated in 20 mM Hepes, 150 mM NaQ, 0.1 mM DTT, 4 Â°C for purification. The pH of the equilibration buffer was adjusted to 7.2 at RT. The monomeric firactions were pooled.<br>
C.	Coupling<br>
A solution of 1.5 mg Qp in 0.75mL 20mM Hepes,150mM NaCl, pH 7.4<br>
was reacted with 0.06mL Sulfo-SMPB (Pierce; 31 mM Stock in H20) for 45 min. at RT. Ttie reaction mixture was dialyzed overnight against 20mM Hepes,150mM NaCl, pH 7.4 and 0.75 mL of this solution were mixed with 1.5 mL of a PLAz-Cys solution in 0.1 mM DTT (62 pM) and 0.43 mL of 20mM Hepes, 150mM NaCI, 137 pM DTT, pH 7.4 adjusted at RT. The coupling reaction was left to proceed for 4 h. at RT, and the reaction mixture was dialyzed ovemit against 20mM Hepes.lSOmM NaCl, pH 7.4 using Spectra For dialysis tubing, MW cutoff 300 GOO Da (Spectrum). The coupling reaction was analyzed by SDS-PAGE and coomassie staining, and Western<br><br>
blotting, using either a rabbit anti-bee venom antiserum (diluted 1:10(X)0), developed with a goat anti-rabbit alkaline phosphatase conjugate (diluted 1:10000), or a rabbit anti-Qp antiserum (1:5000), developed with a goat anti-rabbit alkaline phosphatase conjugate (diluted 1:10000). Samples were nm in both cases under reducing conditions.<br>
Hie result of the coupling reaction is shown in FIG. 26. Bands corresponding to the coupling product of Qp capsid protein to PLAj-Cys aie clearly visible in the coomassie stained SDS-PAGE (left panel), the anti-QP Western Blot (center panel) and the anti-PLA2 Western blot (right panel) of the coupling reactions between QP capsid protein and PLA2-Cys, and are indicated by an arrow in the figure. 15 |JI of the coupling reactions and 50 (il of the dialyzed coupling reactions were loaded on the gel.<br>
Lane 1: Protein marker. 2: Dialyzed coupling reaction 1. 3: Couphng reaction 1. 4: CoupUng reaction 2. 5: coupling reaction 2.6: Couphng reaction 1. 7: Dialyzed coupling reaction 1. 8: Protein Maiker. 9: Couphng reaction 2. 10: Coupling reaction 1.11: Dialyed coupling reaction 1.12: Protein Marker.<br>
EXAMPLE 21 Coupling of anti-idiotypic IgE mimobody VAE051 to Qp, immunization of mice and testing of antisera<br>
A solution of 4.0 mg/ml QP capsid protein in 20 mM Hes, 150 inM NaCl pH 7.2 was reacted for 30 minutes with 10 fold molar excess SMPH (Pierce) (from a 100 raM stock solution dissolved in DMSO) at 25 *C on a rocking shaker. The reaction solution w subsequently dialyzed twice for 2 hours against 2 1 of 20 mM Hepes, 150 mM NaCi, pH 7.2 at 4 "C. The VAE051 solution (2.4 mg/ml) was reducted with an equimolar concentration ofTCEP for 60 min at 25 "C.<br>
46 (il of the dialyzed Qp reaction mixture was then reacted with 340 \)1 of the TCH*-tieated VAE051 solution (2.4 mg/ml) in a total volume of 680 jil of 50 mM sodium acetate buffer at<br>
16 "C for 2 h on a rocking shaker.<br><br>
The reaction products were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were either stained with Coomassie Brilliant Blue. The two additional band in the coupling reactions (which are absent in VAE or Qp solutions) represent the heavy chain and the light chain of the VAE05I coupled to QP (FIG. 28 A). Identity of the bands were confinned by Western blotting with antibodies specific for heavy and light chains, respectively.<br>
Immunization of mice<br>
The QP-VAE051 coupling solution was dialysed ainst 20 mM Hepes, 150 mM NaCl, pH 7.2 using a membrane with a cut-off of 300000 Da. 50 \ig of die Q-VAE051 were injected intreritoneal in two female Balb/c mice at day 0 and day 14. Mice were bled rctroorbitally on day 28 and their serum was analyzed using IgE- and VAE051-specific BUS As.<br>
EUSA<br>
ELISA plates were coated with human IgE at a concentration of 0.8 p.g/ml or with 10 p.g/ml VAE051. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody (FIG. 28 B).<br>
Both mice showed high reactivity to VAFX)51 as well as the human IgE. Preimmune sera of the same mice did not show any reactivity against VAE051 and IgE (FIG. 28 B). TTiis demonstrates that antibodies against the anti-idiotypic IgE mimobody VAE051 have been produced which also recognize the "parent" molecule IgE.<br><br>
EXAMPLE 22<br>
High occupancy coupling of DerpI peptide to wt QP capsid protein using tiie cross-linlter SMPH<br>
The Derp 1,2 peptide, to which a cysteine was added N-tenninally for coupling, was chemically synthesized and had the following sequence: H2N-CQrYPPNANKIREALAQTHSA-COOH. This peptide was used for chemical coupling to wt Qp capsid protein and as described in the following.<br>
D.     Couphng of Hag peptide to Qp capsid protein<br>
QP capsid protein in 20 mM Hepes, 150 mM NaCI, pH 7.2, at a concentration of 2 mg/ml, was reacted with a 5- or 20- fold excess of the cross-Unker SMPH (Pierce) for 30 min. at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours gainst 2 L of 20 mM Hepes, 150 mM NaQ. pH 7.2 at 4 *C. The dialyzed reaction mixture was then reacted with a 5-foId excess of Derp,1,2 peptide for two hours at 25 Â°C on a rocking shaker.<br>
TTie result of the coupling reaction can be seen on FIG. 24. Coupling bands corresponding to 1, 2 and 3 peptides per subunit, respectively, are clearly visible on the gel, and are indicated by arrows. An average of two peptides per subunit were displayed on the capsid.<br>
The samples loaded on the gel of FIG. 24 were the following: Lane 1: Protein Marker. 2: Qp capsid protein dMlvatized with a 5-fold excess of SMPH. 3: Qp capsid protein detivatized with a 20-fold excess of SMPH. 4: Coupling reaction of 5-fold derivatized Qp capsid protein. 5: Coupling reaction of 20-fold derivatized QP capsid protein.<br>
EXAMPLE 23 Insertion of a peptide containing a Lysine residue into the c/el epitope of<br>
HBcAg(I-149)<br><br>
The c/el epitope (residues 72 to 88) of HBcAg is located in the tip region on the surface of the Hepatitis B virus csid (HBcAg). A part of this region (Proline 79 and Alanine 80) was genetically replaced by die peptide Gly-Gly-Lys-Gly-Gly (HBcAg-Lys construct). The introduced Lysine residue contains a reactive amino group in its side chain that can be used for intermolecular chemical crosslinking of HBcAg particles with any antigen containing a free cysteine group.<br>
HBcAg-Lys DNA, having the amino acid sequence shown in SEQ ID<br>
NO:158, was generated by PCRs: The two fragments encoding HBcAg fragments<br>
(amino acid residues 1 to 78 and 81 to 149) were amplified separately by PCR. Tlie<br>
primers used for these PCRs also introduced a DNA sequence encoding the Gly-GIy-<br>
Lys-GIy-Gly peptide. The HBcAg (I to 78) fragment was amplified from pEco63<br>
using primers EcoR]HBcAg(s) and Lys-HBcAg(as). The HBcAg (81 to 149)<br>
fragment was amplified from pBco63 using primers Lys-HBcAg(s) and HBcAg(l-<br>
149)Hind(as).	Primers     Lys-HBcAg(as)    and    Lys-HBcAg(s)    introduced<br>
complementary DNA sequences at the ends of the two PCR products allowing fusion of the two PCR products in a subsequent assembly PCR. The assembled frijnents were amplified by PCR using primers EcoRiHBcAg(s) and HbcAg(l-149)Hind(as).<br>
For the PCRs, 100 pmol of each oligo and 50 ng of the template DNAs were used in the 50 ml reaction mixtures with 2 units of Pwo-polymerase, 0.1 mM dNTPs and 2 mM MgS04. For both reactions , temperature cycling was carried out as foUows: 94Â°C for 2 minutes; 30 cycles of 94Â°C (1 minute), 50Â°C (1 minute), 72""C (2 minutes).<br>
Primer sequences:<br>
EcoRIHBcAg(s}: (5"-CCGGAATrCATGGACATTGACCCTTATAAAG-3") (SEQ ID NO:79);<br>
Lys-HBcAg(as):<br>
(5"-<br>
CCTAGAGCCACCTTTGCCACCATCTTCTAAATTAGTACCCACCCAG GTAGC-3") (SEQ ID NO:80);<br>
Lys-HBcAg(s):<br>
(5"-<br>
GAAGATGGTGGCAAAGGTGGCTCTAGGGACCTAGTAGTCAGTTAT<br>
GTC-3")(SEQlDNO:8l);<br><br>
HBcAg(l-149)Hind(as):<br>
(5"-CGCGTCCCAAGCTrCTAAACAACAGTAGTCTCCGGAAG-3")(SEQ ID NO:82).<br>
For fusion of the two PCR fragments by PCR 100 pmol of primers EcoRIHBcAgCs) and HBcAg(l-149)Hind(as) were used with 100 ng of the two purified PCR firagments in a 50 ml reaction mixture containing 2 units of Pwo polymerase. 0.1 mM dNTPs and 2 mM MgS04. PCR cycling conditions were: 94""C for 2 minutes; 30 cycles of 94Â°C (1 minute), 50Â°C (1 minute), 72Â°C (2 minutes). The assembled PCR product was analyzed by agarose gel electrophoresis, purified and digested for 19 hours in an propriate buffer with EcoRI and HindDI restriction enzymes. The digested DNA fragment was ligated into EcoRI/Hindni-digested pKK vector to generate pKK-HBcAg-Lys expression vector. Insertion of the PCR product into the vector was analyzed by EcoRI/HindHI restriction analysis and DNA sequencing of the insert.<br>
EXAMPLE 24 Expression and partial purification of HBcAg-Lys<br><br>
E. coli strain XL-1 blue was transformed with pKK-HBcAg-Lys. 1 ml of an overnight culture of bacteria was used to innoculate 100 ml of LB medium containing 100 /ig/ml ampicillin. This culture was grown for 4 hours at 37Â°C until an OD at 600 nm of approximately 0.8 was reached. Induction of the synthesis of HBcAg-Lys was performed by addition of IPTG to a final concentration of 1 mM. After induction, bacteria were fUrther shaken at 37Â°C for 16 hours. Bacteria were harvested by centrifugation at 5000 x g for 15 minutes. TTie pellet was frozen at -20Â°C. The pellet was thawed and resuspended in bacteria lysis buffer (10 mM Na2HP04, pH 7.0, 30 mM NaQ, 0.25% Tween-20, 10 mM EDTA, 10 mM DTT) supplemented with 200 fig/ml lysozyme and 10 (i\ of Benzonase (Merck). Cells were incubated for 30 minutes at room temperature and disrupted using a French pressure cell. Triton X-100 was added to the lysate to a final concentration of 0.2%, and the lysate was incubated for 30 minutes on ice and shaken occasionally. E. coli cells hartioring pKK-HBcAg-Lys expression plasmid or a control plasmid were used for induction of HBcAg-Lys expression with IPTG. Prior to the addition of lETG, a sample was removed fi:om the bacteria culture carrying the pKK-HBcAg-Lys plasmid and from a culture carrying the control plasmid. Sixteen hours after addition of IPTG, samples were again, removed from the culture containing pKK-HBcAg-Lys and from the control culture. Protein expression was monitored by SDS-PAGE followed by Coomassie staining.<br>
The lysate was then centrifuged for 30 minutes at 12,000 x g in order to remove insoluble cell debris. TTie supernatant and the pellet were analyzed by Western blotting using a monoclonal antibody against HBcAg (YVS1841, purchased from Accurate Chemical and Scientific Corp., Westbury, NY. USA), indicating that a significant amount of HBcAg-Lys protein was soluble. Briefly, lysates from E. coli cells expressmg HBcAg-Lys and from control cells were centrifuged at 14,000 x g for 30 minutes. Supernatant (= soluble fraction) and pellet (= insoluble firaction) were separated and diluted with SDS sample buffer to equal volumes. Samples were analyzed by SDS-PAGE followed by Western blotting with anti-HBcAg monoclonal antibody YVS 1841.<br>
The cleared cell lysate was used for step-gradient centrifugation using a sucrose step gradient consisting of a 4 ml 65% sucrose solution overlaid with 3 ml 15% sucrose solution followed by 4 ml of bacterial lysate. Ths sample was centrifuged for 3 hrs with 100,000 x g at 4Â°C. After centrifrigation, 1 ml fractions from the top of the gradient were collected and analyzed by SDS-PAGE followed by Coomassie staining. The HBcAg-Lys protein was delected by Coomassie staining.<br>
The HBcAg-Lys protein was enriched at the intoface between 15 and 65% sucrose indicating that it had formed a capsid particle.  Most of the bacterial<br><br>
proteins remained in the sucrose-free upper layer of the gradient, therefore step-gradient centrifugatiion of the HBcAg-Lys particles led both to enrichment and to a partial purification of the particles.<br>
EXAMPLE 25<br>
Chemical coupling of FLAG peptide to HbcAg-Lys using the<br>
heterobifunctional cross-linker SPDP<br>
Synthetic FLAG peptide with a Cysteine residue at its amino terminus (amino acid sequence CGGDYKDDDDK (SEQ ID NO; 147)) was coupled chemically to purified HBcAg-Lys particles in order to elicit an immune response against the FLAG peptide. 600 ml of a 95% pure solution of HBcAg-Lys particles (2 mg/ml) were incubated for 30 minutes at room temperature with the heterobifunctional cross-linker N-Succinimidyl 3-
The FLAG decorated particles were injected into mice.<br>
EXAMPLE 26 Construction of pMPSV-gpl40cys<br>
The gpl40 gene was amplified by PCR from pCytTSgpl40FOS using oligos gpl40CysEcoRI and Sallgpl40. For the PCRs, 100 pmol of each oligo and 50 ng of the template DNAs were used in the 50 ml reaction mixtures with 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM MgS04. For both reactions, temperature cycling was carried out as follows: 94""C for 2 minutes; 30 cycles of 94Â°C (0.5 minutes). 55Â°C (0.5 minutes), 72""C (2 minutes).<br>
The PCR product was purified using QiaEXn kit, digested with SaU/EcoRI and ligated into vector pMPSVHE cleaved with the same enzymes.<br><br>
Oligo sequences:<br>
Gpl40CysEcoRI;<br>
S"-GCCGAATTCCTAGCAGCTAGCACCGAATITATCTAA-S"   (SEQ  ID NO:83);<br>
SaUgpl40-.<br>
5"-   GGTTAAGTCGACATGAGAGTGAAGGAGAAATAT-S"    (SEQ   ID<br>
NO:84).<br>
EXAMPLE 27 Expression of pMPSVgpMOCys<br>
pMPSVgpl40Cys (20 fig) was linearized by restriction digestion. The reaction was stopped by phenol/chloroform extraction, followed by an isopropanol precipitation of the linearized DNA. The restriction digestion was evaluated by agarose gel eletrophoresis. For the transfection, 5.4 fig of linearized pMPSVgpl40-Cys was mixed with 0.6 fig of linearized pSV2Neo in 30 110 and 30 /il of 1 M CaChsolution was added. After addition of 60 /il phosphate buffer (50 mM HEPES, 280 mM NaCl, 1.5 mM Nai HPO4, pH 7.05), the solution was vortexed for 5 seconds, followed by an incubation at room temperature for 25 seconds. The solution was immediately added to 2 ml HP-1 medium containing 2% PCS (2% PCS medium). The medium of an 80% confluent BHK21 cell culture (6-well plate) was then replaced by the DNA containing medium. After an incubation for 5 hours at 37Â°C in a CO2 incubator, the DNA containing medium was removed and replaced by 2 ml of 15% glycerol in 2% PCS medium. The ycerol containing medium was removed after a 30 second incubation phase, and the cells were washed by rinsing with 5 ml of HP-1 medium containing 10% PCS. Fmally 2 ml of fresh HP-1 medium containing 10% PCS was added.<br>
Stably transfected cells were selected and grown in selection medium (HP-1 medium supplemented witii G418) at 37""C in a CO2 incubator. When the mixed population was grown to confluency, the culture was split to two dishes, followed by a 12 h growtii period at 37Â°C. One dish of the cells was shifted to 30Â°C to induce the expression of soluble GP140-FOS. The other dish was kept at 37""C.<br>
The expression of soluble GP140-Cys was determined by Western blot analysis. Culture media (0.5 ml) was methanol/chloroform precipitated, and the pellet was resuspended in SDS-PAGE sample buffo-. Samples were heated for 5 minutes at<br><br>
95""C before being applied to a 15% acrylamide gel. After SDS-PAGE, proteins were transferred to Protan nitrocellulose membranes (Schleicher &amp; Schuell, Germany) as described by Bass and Yang, in Creighton, T.E., ed., Protein Function: A Practical Approach, 2nd Edn., IRL Press, Oxford (1997), pp. 29-55. The membrane was blocked with 1 % bovine albumin (Sigma) in TBS (lOxTBS per liter: 87.7 g NaCl, 66.1 g Trizma hydrochloride (Sigma) and 9.7 g Tiizma base (Sigma), pH 7.4) for 1 hour at room temperature, followed by an incubation with an anti-GP140 or GP-160 antibody for 1 hour. The blot was washed 3 times for 10 minutes with TBS-T (TBS with 0.05% Tween20), and incubated for 1 hour with an alkaline-phosphatase-anti-mouse/rabbit/monkey/human IgO conjugate. After washing 2 times for 10 minutes with TBS-T and 2 times for 10 minutes with TBS, the development reaction was carried out using alkaline phosphatase detection reagents (10 ml AP buffer (100 mM Tris/HCl, 100 mM Naa. pH 9.5) with 50 /il NET solution (7.7% Nitro Blue Tetrazolium (Sigma) in 70% dimethylformamide) and 37 fi\ of X-Phosphate solution (5% of 5-bromo-4-chloro-3-indolyl phosphate in dimethylformamide).<br>
EXAMPLE 28 Purification of gpl40Cys<br>
An anti-gpl20 antibody was covalently coupled to a NHS/EDC activated dextran and packed into a chromatography column. The supernatant, containing GP140Cy5 is loaded onto the column and after sufficient washing, GP140Cy5 was eluted using 0.1 M HCl. The eluate was directly neutralized during collection using 1 M Tris pH 7.2 in the collection tubes.<br>
Disulfide bond fonnation might occur during purification, therefore the collected sample is treated with 10 mM DTT in 10 mM Tris pH 7.5 for 2 hours at 25Â°C.<br>
DTT is remove by subsequent dialysis against 10 mM Mes; 80 mM NaCl pH 6.0. Hnally GP140Cys is mixed with alphavirus particles containing the JUN residue in E2 as described in Example 16.<br>
EXAMPLE 29 Construction of PLA2-Cys<br><br>
The PLA3 gene was amplified by PCR from pAV3PLAfos using oligos EcoRIPLA and PLA-Cys-hind. For the PCRB, 100 pmol of each oligo and 50 ng of the template DNAs were used in the 50 ml reaction mixtures with 2 units of Pwo polymerase, 0.1 mM dNTPs and 2 mM MgS04. For the reaction, temperature cycling was carried out as follows: 94Â°C for 2 minutes; 30 cycles of 94Â°C (0.5 minutes), 55Â°C (0.5 minutes), 72Â°C (2 minutes).<br>
The PCR product was purified using QiaEXn kit, digested with EcoRI/Hindin and ligated into vector pAV3 cleaved with the same enzymes.<br>
Oligos<br>
EcoRIPLA:<br>
5"-TAACCGAATrCAGGAGGTAAAAAGATATGG-3" (SEQ ID NO:85)<br>
PLA Cys-hind:<br>
5"-GAAGTAAAGCTnTAACCACCGCAACCACCAGAAG-3"    (SEQ   ID NO:86).<br>
EXAMPLE 30 Expression and Purification of PLA2-Cys<br><br>
For cytoplasmic production of Cys tagged proteins, E. coli XL-1-Blue strain was transformed with the vectors pAV3::PLA and pPIA-Cys. The culture was incubated in rich medium in the presence of ampicillin at 37Â°C with shaking. At an optical density (550nm) of, 1 mM IPTG was added and incubation was continued for another 5 hours. The cells were harvested by centrifiigation, resuspended in an appropriate buffer {e.g., Tris-HCl, pH 7.2, 150 mM NaCl) containing DNase, RNase and lysozyme, and disrupted by passage through a french pressure cell. After centrifiigation (Sorvall RC-5C, SS34 rotor, 15000 ipm, 10 niin, 4Â°C), the pellet was resuspended in 25 ml inclusion body wash buffer (20 mM tris-HCl, 23% sucrose, 0.5% Triton X-100, 1 mM EDTA, pH8) at 4Â°C and recentrifuged as described above. This procedure was repeated until the supernatant after centrifugation was essentially clear. Inclusion bodies were resuspended in 20 ml solubilization buffer (5.5 M guanidinium hydrochloride, 25 mM tris-HCl, pH 7.5) at room temperature and insoluble material was removed by centrifugation and subsequent passage of the supernatant through a sterile filter (0.45 fxm). The protein solution was kept at 4Â°C for at least 10 hours in the presence of 10 mM EDTA and 100 mM DTT and then dialyzed three times against 10 volumes of 5.5 M guanidinium hydrochloride, 25 mM tris-HCl, 10 mM EDTA, pH 6. The solution was dialyzed twice against 51 2 M urea, 4 mM "SUVA, 0.1 M NHiCl, 20 mM sodium borate (pH 8.3) m die presence of an )propriate redox shuffle (oxidized glutathione/reduced glutathione; cystine/cysteine). The refolded protein was then applied to an ion exchange chromatography. The protein was stored in an appropriate buffer with a pH above 7 in the presence of 2-10 mM DTT to keep the cysteine residues in a reduced form. Prior to coupling of the protein with die alphavirus particles, DTT was removed by passage of the protein solution through a Sephadex G-25 gel filtration column.<br>
EXAMPLE 31 Construction of a HBcAg devoid of fiee cysteine residues and containing<br>
an inserted lysine residue<br>
A Hepatitis core Antigen (HBcAg), referred to herein as HBcAg-lys-2cys-Mut, devoid of cysteine residues at positions corresponding to 48 and 107 in SEQ E) NO:134 and containing an inserted lysine residue was constructed using the following methods.<br>
The two mutations were introduced by first separately amplifying three ftagments of the HBcAg-Lys gene prepared as described above in Example 23 with the following PCR primer combinations. PCR methods essentially as described in Example 1 and conventional cloning techniques were used to prepare the HBcAg-lys-2cys-Mut gene.<br><br>
In brief, the following primers were used to prq)are fragment 1: Primer 1: EcoRIHBcAg(s) CCGGAATTCATOGACATTGACCCTTATAAAG (SEQ ID NO: 148)<br>
Primer 2:48as<br>
GTGCAGTATGGTGAGGTGAGGAATGCTCAGGAGACTC     (SEQ    ID NO: 149)<br>
The following primers were used to prepare fragment 2: Primer 3:4Ss<br>
GSGTCTCCTGAGCATTCCTCACCTCACCATACTGCAC      (SEQ      ID NO: 150)<br>
Primer 4: 107 as<br>
CTTCCAAAAGTGAGGGAAGAAATGTGAAACCAC (SEQ ID N0:151)<br>
TTie following primers were used to prepare fragment 3: Prima: 5: HBcAgl49hind-as<br>
CGCGTCCCAAGCTTCTAAACAACAGTAGTCTCCGGAAGCGTTGATA G (SEQ ID NO: 152)<br>
Primer 6:107s GTGGTITCACATITCTTCCCTCACrrTrGGAAG(SEQIDNO:153)<br>
Fragments 1 and 2 were then combined with PCR primers EcoRIHBcAg(s) and 107as to give fragnaent 4. Fragment 4 and fragnnt 3 were then combined with primers EcoRIHBcAg(s) and HBcAgl49hind-as to produce the full length gene. The full length gene was then digested with the EcoRI (GAATTC) and Hindm (AAGCTT) enzymes and cloned into the pKK vector (Pharmacia) cut at the same restriction sites.<br>
EXAMPLE 32 Blockage of free cysteine residues of a HBcAg followed by cross-Hnking<br>
The free cysteine residues of the HBcAg-Lys prepared as described above in Example 23 were blocked using lodacetamide. The blocked HBcAg-Lys was then cross-linked to the FLAG peptide with the hero-bifunctional cross-linker m-maleimidonbenzoyl-N-hydroxysuccinimide ester (Sulfo-MBS).<br><br>
ITie methods used to block the free cysteine residues and cross-link the HBcAg-Lys are as follows. HBcAg-Lys (550 g/ml) was reacted for 15 minutes at room temperature with lodacetamide (Fluka Chemie, Brugg, Switzerland) at a concentration of 50 mM in phosphate buffered saline (PBS) (50 mM sodium phosphate, 150 mM sodium chloride), pH7.2, in a total volume of 1 ml. The so modified HBcAg-Lys was then reacted iramediately with Sulfo-MBS (Piense) at a concentration of 330 iiM directly in the reaction mixture of step 1 for 1 hour at room temperature. TTie reaction mixture was then cooled on ice, and dialyzed against 1000 volumes of PBS pH 7.2. The dialyzed reaction mixture was finally reacted with 300 ;iM of the FLAG peptide (CGGDYKDDDDK (SEQ ID NO: 147)) containing an N-terminal free cysteine for coupling to the activated HBcAg-Lys, and loaded on SDS-PAGE for analysis.<br>
Tlie resulting patterns of bands on the SDS-PAGE gel showed a clear additional band migrating slower than the control HBcAg-Lys derivatized with the cross-UnkCT, but not reacted with the FLAG peptide. Reactions done under the same conditions without prior derivatization of the cysteines with lodacetamide led to complete cross-linking of monomers of the HBcAg-Lys to higher molecular weight species.<br>
EXAMPLE 33<br>
Isolation and chemical coupling of FLAG peptide to Type-1 pili of<br>
Escherichia coli using a heterobifunctional cross-linker<br>
A.       Introduction<br>
Bacterial piU or fimbriae are filamentous surface organelles produced by a wide range of bacteria. These organelles mediate the attachment of bacteria to surface receptors of host cells and are required for the establishment of many bacterial infections like cystitis, pyelonephritis, new bom meningitis and diarrhea.<br>
PiU can be divided in different classes with respect to their receptor specificity (agglutination of blood cells from different species), their assembly pathway (extracellular nucleation, general secretion, chaperone/usher, altemate chaperone) and their morphological properties (thick, rigid pili; thin, flexible pili; atypical structures including capsule; curli; etc). Examples of thick, rigid pili forming a rit handed helix that are assembled via the so called ch)erone/usher pathway and mediate adhesion to host glycoproteins include Type-1 pili, P-pili, S-pili, FlC-pili, and 987P-pili). The most prominent and best characterized member of this class of pili are P-pili and Type-1 pili (for reviews on adhesive stmctures, their assembly and the associated diseases see Soto, G. E. &amp; Hultgren, S. J., /. Bacteriol i8J:1059-1071<br><br>
(1999); BulUtt &amp; Makowski, Biophys. J. 74:623-632 (1998); Hung, D. L. &amp; Hultgren, S. J., /. Struct, Biol 224:201-220 (1998)).<br>
Type-1 pili are long, filamentous polymeric protein structures on the surface of E. coli. They possess adhesive properties that allow for binding to mannose-containing receptors present on the surface of certain host tissues. Type-1 piU can be expressed by 70-80% of all E. coli isolates and a single E. coli cell can tear up to 500 pili. Type- pili reach a length of typically 0.2 to 2 fiM with an average number of 1000 protein subunits that associate to a right-handed helix with 3.125 subunits per turn with a diameter of 6 to 7 mu and a central hole of 2.0 to 2.5 nm.<br>
The main Type-1 pilus component, FimA, which represents 98% of the total pilus protein, is a J5.8 kDa protein. The minor pilus components FimF, FimG and FimH are incorporated at the tip and in regular distances along the pilus shaft (Klemm, P. &amp; Krogfelt, K. A., "Type I fimbriae of Escherichia coli," in: Fimbriae. Klemm, P. (ed.), CRC Press Inc., (1994) pp. 9-26). FimH, a 29.1 kDa protein, was shown to be the mannose-binding adhesin of Type-1 pih (Krogfelt, K. A., et al, Infect. Immun. 5S:1995-1998 (1990); Klemm, P., et al., Mol. Microbiol. 4:553-56Q (1990); Hanson. M. S. &amp; Brinton, C. C. J., Nature I7:265~26S (1988)), and its incorporation is probably facilitated by FmnG and FimF (Klemm, P. &amp; Christiansen, G., Mol. Gen. Genetics 206:439-445 (1987); Russell, P. W. &amp; Omdorff. P. B., /. Bacterial. 774:5923-5935 (1992)). Recently, it was shown that FimH might also fOTm a thin tip-fibrillum at the end of the pili (Jones, C. H-, et al. Proc. Nat. Acad. Sci. USA 92:2081-2085 (1995)). The order of major and minor components in the individual mature piU is very similar, indicating a highly ordered assembly process (Soto, G. E. &amp; Hultgren, S. L, J. Bacterial 181:1059-1011 (1999)).<br>
P-piU of E. coli are of very similar architecture, have a diameter of 6.8 nm, an axial hole of 1.5 nm and 3.28 subunits per turn (Bullitt &amp; Makowski, Biophys. J. 74:623-632 (1998)). The 16.6 M5a PapA is the main component of this pilus type and shows 36% sequence identity and 59% similarity to FimA (see Table 1). As in Tjfpe-l pili the 36.0 kDa P-pilus adhesin PapG and specialized adapter proteins make up only a tiny fraction of total pilus protem. The most obvious difference to Type-1 piB is the absence of the adhesin as an integral part of the pilus rod, and its exclusive localization in the tip fibrillium that is connected to the pilus rod via specialized adapter proteins that Type-1 pili lack (Hultgren, S. J., et al., Cell 75:887-901 (1993)).<br><br>
Table 1:	Similarity and identity between several structural pilus proteins<br>
of Type-1 and P-pili (in percent). The adhesins were omitted.<br>
Similarity<br>
FimA PapA FimI RmF FimG PapE PapK PapH PapF<br>
FimA       59  57   56  44  50  44  46  46<br>
PapA   36        49   48   41   45   49   49   47<br>
Identity	FimI  35   31       56  46  40  47  48  48<br>
FimF 34 26 30 40 47 43 49 48 FimG 28 28 28 26 39 39 41 45 PapE 25 23 18 28 22 43 47 54 PapK 24 29 25 28 22 18 49 53 PapH 22 26 22 22 23 24 23 41 PapF   18  22   22   24   28   27   26   21<br>
Type-1 pili are extraordinary stable hetero-oligomeric complexes. Neither SDS-treatment nor protease digestjons, boiling or addition of denaturing agents can dissociate Type-1 pili into their individual protein components. The combination of different methods like incubation at 100Â°C at pH 1.8 was initially found to allow for the depolymeri2ation and separation of the components (Eshdat, Y., etal., J. Bacteriol i4S:308-314 (1981); Brinton, C.C. J., Trans. N. Y. Acad Sci, 27:1003-1054 (1965); Hanson, A. S., et al., J. Bacteriol, 770:3350-3358 (1988); Hemm, P. &amp; Krogfelt, K. A., "Type I fimbriae of Escherichia coli," in: Fimbriae. Hemm, P. (ed,), CRC Press Inc., (1994) pp. 9-26). Interestingly, Type-1 pili show a tendency to break at positions where FmiH is incorporated upon mechanical agitation, resulting in fragments that present a FimH adhesin at their tips. This was interpreted as a mechanism of the bacterium to shorten pili to an effective length under mechanical stress (Klemm, P. &amp; Krogfelt, K. A., "Type I fimbriae of Escherichia coli," in: Fimbriae. Klemm, P. (ed), CRC Press Inc., (1994) pp. 9-26). Despite their extraordinary stability, Type-1 pili have been shown to unravel partially in the presence of 50% glycerol; they lose their helical structure and form an extended and flexible, 2 nm wide protein chain (Abraham, S. N., et al, J. Bacteriol. i74:5145-5148 (1992)).<br>
P-pili and Type-1 pili are encoded by single gene clusters on the E. coli chromosome of approximately 10 kb (Klemm, P. &amp; &amp;ogfelt, K. A., "Type I fimbriae of Escherichia coli" in: Fimbriae. Klemm, P. (ed.), CRC Press Inc., (1994) pp. 9-26; Omdoiff, P. E, &amp; Falkow, S., /. Bacteriol 160:61-66 (1984)). A total of nine genes are found in the Type-1 pilus gene cluster, and 11 genes in the P-pUus<br><br>
cluster (Hultgren, S. J., et al.. Adv. Prot. Chem. 44:99-123 (1993)). Both clusters are organized quite similarly.<br>
The first two m-genes, fimB and fimE, code for recombinases involved in the regulation of pilus expression CNlcClain, M. S., et al., J. Bacteriol. 775:5308-5314 (1991)). The main structural pilus protein is encoded by the next gene of the cluster, yimA (Klemm. P., Euro. J. Biochem. 743:395-400 (1984); Omdorff, P. E. &amp; Falkow, S., /. Bacteriol 160:61-66 (1984); Omdorff, P. E. &amp; FaUcow, S., J. Bacteriol 162:454-451 (1985)). TTie exact role of fiml is unclear. It has been reported to be incorporated in the pilus as well (Klemin, P. &amp; Krogfelt, K. A., "Type I fimbriae of Escherichia coU" in: Fimbriae. Klemm, P. (ed.), CRC Press Inc., (1994) pp. 9-26). The adjacent fimC codes not for a structural component of the mature pUus, but for a so-called pilus chierone that is essential for the pilus assembly (Klemm, P., Res. Microbiol 743:831-838 (1992); Jones, C. H., et al, Proc. Nat. Acad Sci. C/SA 90:8397-8401 (1993)).<br>
Hie assembly platform in the outer bacterial membrane to which the mature pilus is anchored is encoded by JlmD (Klemm, P. &amp; Christiansen, G., Mol. Gen, Genetics 220:334-338 (1990)). The three niinor components of the Type-1 pili, FimF, FimG and RmH are encoded by the last three genes of the cluster (Klemm, P. &amp; Christiansen, G., Mol Gen. Genetics 205:439-445 (1987)). Apart from fimB and fimE, all genes encode precursor proteins for secretion into the periplasm via the sec-pathway.<br>
The similarities between different pili following the chaperone/usher pathway are not restricted to their morphological properties. Their genes are also arranged in a very similar manner. Generally the gene for the main structural subunit is found directly downstream of the regulatory elements at the beginning of the gene cluster, followed by a gene for an additional structural subunit (ftml in the case of Type-1 piliandpfgsTJinthecaseof P-pili). PapHwas shown and Kml is supposed to terminate pilus assembly (Hultgren, S. J., et al., Cell 75:887-901 (1993)). The two proteins that guide the process of pilus formation, namely the specialized pilus chaperone and the outer membrane assembly platform, are located adjacently downstream. At the end of the clusters a variable number of minor pilus components including the adhesins are encoded. The similarities in morphological structure, sequence (see Table 1), genetic organization and regulation indicate a close evolutionary relationship and a similar assembly process for these cell oi;ganeUes.<br>
Bacteria producing Type-1 pili show a so-called phase-variation. Either the bacteria are fully piliated or bald. This is achieved by an inversion of a 314 bp genomic DNA fragment containing the fimA promoter, thereby inducing an "all on" or "all off" expression of the pilus genes (McClain, M. S., et al., J. Bacteriol<br><br>
775:5308-5314 (1991)). The coupling of the expression of the other structural pilus genes to fimA expression is achieved by a still unknown mechanism. However, a wide range of studies elucidated the mechanism that influences the switching between the two phenotypes.<br>
The first two genes of the Type-1 pilus cluster, fimB and fimE encode recombinases that recognize 9 bp DNA segments of dyad symmetry that flank the invertable jimA promoter. Whereas FimB switches pilation "on", FimE turns the promoter in the "off" orientation. The up- or down-regulation of cithNfimB OTJWIE exjHtssion therefore controls the position of the so-called "_m-switch" (McOain, M. S.. et al.. J. Bactenol. iZ?:5308-5314 (1991); Blomfield, L C. et al.. J. Bacterial 173:529-5301 (1991%<br>
TTie two regulatory proteins fimB and fimE are transcribed from distinct promoters and their transcription was shown to be influenced by a wide range of different factors including the integration host factor (IHF) (Blomfield, I. C, et cd., Mol. Microbiol. 25:705-717 (1997)) and the leucine-tesponsive regulatory protein (UIP) (Blomfield, I. C, et al., J. Bactenol i 75:27-36 (1993); Gaily, D. L., et al. J. Bacterial 175:6186-6193 (1993); Gaily, D. L., et al., Microbiol 2i:725-738 (1996); Roesch, R. L. &amp; Blomfield, 1. C, Mol Microbiol 27:751-761 (1998)). Mutations in the former lock the bacteria either in "on" or "off" phase, whereas LRP mutants switch with a reduced frequency. In addition, an effect of leuX on pilus biogenesis has been shown. This gene is located in the vicinity of die fim-genes on the chromosome and codes for the minor leucine tRNA species for the UUG codon. WhencasfimB contains five UUG codons,yOTLE contains only two, and enhanced leuX transcription might favor FimB over FimE expression (Buroff, R. L., et al, Infect. Imnmn. 6i:1293-1300 (1993); Newman, J. V., et al, FEMS Microbiol Lett. 122:281-287 (1994); Ritter, A., et oL. Mol Microbial, 25:871-882 (1997)).<br>
Furthermore, temperature, medium conKJsition and other environmental factors were shown to influence the activity of FimB and FimE. Finally, a spontaneous, statistical switching of the fimA promoter has been reported. The frequency of this spontaneous switching is approximately 10" per generation (Eisenstein, B. L, Science 214:337-339 (1981); Abraham, S. M., et al, Proa. Nat. Acad. Sci, USA 52:5724-5727 (1985)), but is strongly influenced by the above mentioned factors.<br>
The genes _7Â«i andmC are also transcribed from the/imA promoter, but iMrectly downstream of fimA a DNA segment with a strong tendency to form secondary structure was identified which probably represents a partial transcription terminator (Klemm, P., Euro. J. Biochem. 75:395-400 (1984)); and is therefore supposed to severely reducem/ and mC transcription.   At the 3" end offimC an<br><br>
additional promoter controls the fimD transcription; at the 3" end at fimD the last known fim promoter is located that regulates the levels of RmF, FimG, and FimH. Thus, all of the minor Type-1 pili proteins are transcribed as a single mRNA (Klemm, P. &amp; Krogfelt, K. A., "Type I fimbriae of Escherichia coii," in: Fimbriae. Klemm, P. (ed.), CRC Press Inc., (1994) pp. 9-26). This ensures a 1:1:1 stoichiometry on mRNA-level, which is probably maintained on the protein level.<br>
In the case of P-pili additional regulatory mechanisms were found when the half-life of mRNA was determined for different P-pilus genes. ITie mRNA for papA was extraordinarily long-lived, whereas the mRNA for papB, a regulatory pilus protein, was encoded by short-lived mRNA (Naureckiene, S. &amp; Uhlin. B. E., Mot Microbiol. 27:55-68 {1996); Nilsson, P., et al., J. Bacterial. 77S:683-690 (1996)).<br>
In the case of Type-1 pili, the gene for die Type-1 pilus chaperone FimC starts with a GTG instead of an ATG codon, leading to a reduced translation efficiency. Finally, analysis oftheimH gene revealed atendency of the yimH mRNA to form a stem-loop, which might severely hamper translation. In summary, bacterial pilus biogenesis is regulated by a wide range of different mechanisms acting on all levels of protein biosynthesis.<br>
Periplasmic pilus proteins are generally synthesized as precursors, contwning a N-terminal signal-sequence that allows translocation across the inner membrane via the Sec-apparatus. After translocation the precursors are normally cleaved by signal-peptidase I. Structural Type-1 pilus subunits normally contain disulfide bonds, their formation is catalyzed by DsbA and possibly DsbC and DsbQ gene products.<br>
The Type-1 pilus chaperone FimC lacks cysteine residues. In contrast, the chaperone of P-pili, PapD, is tiie only membCT of the pilus chaperone family that contains a disulfide bond, and the dependence of P-pili on DsbA has been shown explicitly (Jacob-Dubuisson, F., et al.. Proc. Nat. Acad. Sci USA 97:21552-11556 (1994)). PapD does not accumulate in the periplasm of a AdsbA strain, indicating that the disturbance of the P-pilus assembly machinery is caused by the absence of tiifc chaperone (Jacob-Dubuisson, F., et al., Proc. Nat. Acad. Sci. USA 97:11552-11556 (1994)). This is in accordance with the finding that Type-1 pili are still assembled in a AdsbA strain, albeit to reduced level (Hultgren, S. J., et al., "Bacterial Adhesion and Their Assembly", in: Escherichia colt and Salmonella, Neidhardt, F. C. (ed.) ASM Press, (1996) pp. 2730-2756).<br>
Type-1 pili as well as P-pili are to 98% made of a single or main structural subunit termed FimA and PapA, respectively. Both proteins have a size of ~15.5 kDa. The additional minor components encoded in the pilus gene clusters are<br><br>
very similar (see Table 1). The similarities in sequence and size of the subunits with the exception of the adhesins suggest that all share an identical folding motif, and differ oivly with respect to their affinity towards each other. Especially the N- and C-terminal regions of these proteins are well conserved and supposed to play an important role in chaperone/subunit interactions as well as in subunit/subunit interactions within the pilus (Soto, G. E. &amp; Hultgren, S. J., / Bacteriol. 181:1059-1071 (1999)). Interestingly, the conserved N-terminal segment can be found in the naiddle of the pilus adhesins, indicating a two-domain organization of the adhesins where the proposed C-terminal domain, starting with the conserved motif, corresponds to a structural pilus subunit whereas the N-terminal domain was shown to be responsible for recognition of host cell receptors (Hultgren, S. J., et al., Proc. Nat. Acad. Sci. USA S6;4357-4361 (1989); Haslam, D. B., etal., Mol. Microbiol. 14:399-409 (1994); Soto, G. E., et al., EMBO J. 17:6155-6161 (1998)). The different subunits were also shown to influence the morphological properties of the pili. The removal of several genes was reported to reduce the number of Type-1 or P-pili or to . increase their length, {fimH, papG, papK, fimF, fimG) (Russell, P. W. &amp; Omdorff, P. E., J. Bacteriol. 174:5923-5935 (1992); Jacob-Dubuisson, R., et al, EMBO J. i2:837-847 (1993); Soto, G. E. &amp; Hultgren, S. J., J. Bactenol 757:1059-1071 (1999)); combination of the gene deletions amplified these effects or led to a total loss of pilation (Jacob-Dubuisson, R., et al, EMBO J. 72:837-847 (1993)).<br>
In non-fmibrial adhesive cell organelles also assembled via cherones/usher systems such as Myf fimbriae and CS3 pili, the conserved C-terminal region is different. This indirectly proves the importance of these C-tenninal subunit segments for quaternary interactions (Hultgren, S. I., et al., "Bacterial Adhesion and Their Assembly", in: Escherichia coli and Salmonella, Neidhardt, F. C. (ed.) ASM Press, (1996) pp. 2730-2756).<br>
Gene deletion studies proved that removal of the pilus chaperones leads to a total loss of piliation in P-pili and Type-1 pili (Lindberg, F., et al, J. Bactenol 777:6052-6058 (1989); Hemm, P., Res. Microbiol 745:831-838 (1992); Jones, C. H., et al., Proc. Nat. Acad Sd. USA 90:8397-8401 (1993)). Periplasmic extracts of a AfimC strain showed the accumulation of the main subunit RmA, but no pili could be detected (Klemm, P., Res. Microbiol 745:831-838 (1992)). Attempts to over-express individual P-pilus subunits failed and only proteolytically degraded forms could be detected in the absence of PapD; in addition, the P-pilus adhesin was purified with the inner membrane fiaction in the absence of the chaperone (Lindberg, F., et al., J. Bacteriol 777:6052-6058 (1989)). However, co-expression of the structural pilus proteins and their chaperone allowed the detection of chjerCHie/subunit complexes fiwrn the periplasm in the case of the FimC/FimH<br><br>
conlex as well as in the case of different Pap-proteins including the adhesin PapG and the main subunit PapA (Tewari, R., et al., J. Biol Chem. 265:3009-3015 (1993); Lindberg, F., et al.. J. Bactenol. J7i:6052-6058 (1989)). The affinity of chaperone/subunit complexes towards their assembly platform has also been investigated in vitro and was found to differ strongly (Dodson et al, Proc. Natl Acad. Sci. USA 90:3670-3674 (1993)). From diese results the following functions were suggested for the pilus chaperones.<br>
They are assumed to recognize unfolded pilus subunits, prevent dieir aggregation and to provide a "folding template" that guides the formation of a native structure.<br>
The folded subunits, which after folding display surfaces that allow subunit/subunit interactions, are then expected to be shielded from interacting with other subunits, and to be kept in a monomelic, assembly-competent state.<br>
Finally, the pilus chaperones are supposed to allow a triggered relee of the subunits at the outer membrane assembly location, and, by doing so with different efficiency, influence the composition and order of the mature pili (see also the separate section below).<br>
After subunit release at the outer membrane, the cherone is free for another round of substrate binding, folding assistance, subunit transport through the periplasm and specific delivery to the assembly site. Since the periplasm lacks energy sources, like ATP, the whole pilus assembly process must be thennodynamicaUy driven (Jacob-Dubuisson, F., et al.. Proc. Nat. Acad Sci. USA 9J:11552-11556 (1994)). The wide range of different ftmctions attributed to the pilus chaperones would implicate an extremely fine tuned cascade of steps.<br>
Several findings, however, are not readily explained with the model of pilus chaperone function outlined above. One example is the existence of multimeric chaperone/subunit complexes (Striker, R. T., et al.. J. Biol Chem. 259:12233-12239 (1994)), where one chaperone binds subunit dimers or trimers. It is difficult to imagine a folding template that can be "double-booked". The studies on the molecular details of chaperone/subunit interaction (see below) partially supported the functions smnmarized above, but also raised new questions.<br>
AH" 31 periplasmic chaperones identified by genetic studies or sequence analysis so far are proteins of jproximately 25 kDa with conspicuously high pi values around 10. Ten of these chaperones assist the assembly of rod-like pili, four are involved in the formation of thin pili, ten are important for the biogenesis of atypically thin structures (including capsule-like structures) and two adhesive structures have not been determined so far (Holmgren, A., et al., EMBO J. Ji:1617-1622 (1992); Bonci, A., et al., J. Mot Evolution 44:299-309 (1997); Smyth,<br><br>
C. J., el al, FEUS Immun. Med Microbiol. 26:127-139 (1996); Hung, D. L. &amp; Hultgren, S. J., J. Struct, Biol. 724:201-220 (1998)). The pairwise sequence identity between these chaperones and PapD ranges from 25 to 56%, indicating an identical overall fold (Hung, D. h., et al.. EMBO J. 75:3792-3805 (1996)).<br>
The first studies on the mechanism of chaperone/substrate recognition was based on the observation that the C-tennini of all known pilus chaperones are extremely similar. Synthetic peptides conesponding to the C-termini of the P-pilus proteins were shown to bind to PapD in EUSA assays (Kuehn, M. J., et al. Science 262:1234-1241 (1993)). Most importantly, the X-ray structures of two complexes were solved in which PapD was co-crystallized with 19-residue peptides corresponding to the C-tennini of either the adhesin P)G or the minor pilus component PapK (Kuehn, M. J., et al., Science 262:1234-1241 (1993); Soto, G. E., et al, EMBO J. 77:6155-6167 (1998)). Both peptides bound in an extended conformation to a p-strand in the N-terminal chaperone domain that is oriented towards the inter-domain cleft, thereby extending a P-sheet by an additional strand. The C-teiminal carboxylate groups of the peptides were anchored via hydrogen-bonds to Arg8 and Lysll2, these two residues are invariant in the family of pilus chaperones. Mutagenesis studies confirmed their importance since their exchange against alanine resulted in accumulation of non-functional pilus chierone in the periplasm (Slonim, L. N., et al, EMBO J. 77:4747756 (1992)). The crystal structure of PapD indicates that neither ArgS nor Lysll2 is involved in stabilization of the chaperone, but completely solvent exposed (Holmgren, A. &amp; Branden, C. I., Nature 542:248-251 (1989)). On the substrate side the exchange of C-tenninal PapA residues was reported to abohsh P-pilus formation, and similar experiments on the conserved C-tenninal segment of the P-pilus adhesin PapG prevented its inccaporation into the P-pilus (Hultgren, S. J., et al., "Bacterial Adhesion and Their Assembly", in: Escherichia coli and Salmonella, Neidhardt, F. C. (ed.) ASM Press, (1996) pp. 2730-2756). All evidence therefore indicated pilus subimit recognition via the C-terminal segments of the subunits.<br>
A more recent study on C-terminal amino acid exchanges of the P-pilus adhesin PapG gave a more detailed picture. A range of amino acid substitutions at the positions -2, -4, -6, and -8 relative to the C-terminus were tolerated, but changed pilus stability (Soto, G. E., et al., EMBO J. 77:6155-6167 (1998)).<br>
Still, cffltain problems arise when this mode! is examined more closely. Adhesive bacterial structures not assembled to rigid, rod-like pili lack the conserved C-terminal segments (Hultgren, S. J., et al., "Bacterial Adhesion and Their Assembly", in: Escherichia coli and Salmonella, Neidhardt, F. C. (ed.) ASM Press, (1996) pp. 2730-2756), even though they are also dependent on the presence of<br><br>
related pilus chaperones. "Hiis indicates a different general role for the C-terminal segments of pilus subunits, namely the mediation of quaternary interactions in the mature pilus. Moreover, the attempt to solve the structure of a C-terminal peptide in complex with the chaperone by NMR was severely hampered by the weak binding of the peptide to the chaperone (Walse, B., et al, FEBS Lett. 412:115-120 (1997)); whereas an essential contribution of the C-termlnal segments for chaperone recognition implies relatively high affinity interactions.<br>
An additional problem arises if the variability between the different subunits are taken into account Even though the C-terminal segments are conserved, a wide range of conservative substitutions is found. For example, 15 out of 19 amino acid residues differ between the two peptides co-crystallizfid with PD (Soto, G. E., et at., EMBO J. 77:6155-6167 (1998)). This has been explained by the kind of interaction between chaperone and substrate, that occurs mainly via backbone interactions and not specifically via side-chain interactions. Then again, the specificity of the chaperone for certain substrates is not readily explained. On the contrary to the former argument, the conserved residues have been taken as a proof for the specificity (Hultgren, S. J., et al., "Bacterial Adhesion and Their Assembly", in: Escherichia coli and Salmonella, Neidhardt, F. C. (ed.) ASM Press, (1996) pp. 2730-2756).<br>
The outer membrane assembly platform, also termed "ushta" in the literatures, is fonned by homo-oligomers of FiniD or P)C, in the case of Type-1 and P-pili, respectively (Klemm, P. &amp; Christiansen, G., Mol Gen, Genetics 220:34-338 (1990); Thanassi, D. G., et al., Proc. Nat. Acad. Sei. USA 95:3146-3151 (1998)). Studies on the elongation of Type-1 fimbriae by electron microscopy demonstrated an elongation of the pilus from the base (Lowe, M. A., et al., J. Bacterial 765:157-163 (1987)). In contrast to the secretion of unfolded subunits into the periplasmic space, the fuUy folded jMoteins have to be translocated through the outer membrane, possibly in an oligomeric form (Tlianassi, D. G., et al., Proc. Nat. Acad. Sei. USA 95:3146-3151 (1998)). This requires first a membrane pore wide enough to allow the passage and second a transport mechanism that is thermodynamically driven (Jacob-Dubuisson,F.,era/., / Biol Chem. 269:12447-12455 (1994)).<br>
FimD expression alone was shown to have a deleterious effect on bacterial growth, the co-expression of pilus subunits could restore normal growth behavior (KlEaran, P. &amp; Christiansen, G., Mol. Gen, Genetics 220:334-338 (1990)). Based on this it can be concluded that the ushers probably form pores that are completely filled by the pilus. Electron microscopy on membrane vesicles in which PapC had been incorporated confirmed a pore-forming structure with an inner diameta: of 2 nm (Thanassi, D. G., et al, Proc. Nat. Acad. Sei USA 95:3146-3151<br><br>
(1998)), Since the inner diameter of the pore is too small to allow the passage of a pilus rad, it has been suggested that the helical airangement of the mature pilus is formed st the outside of the bacterial surface. The finding that glycerol leads to unraveling of pill which then form a protein chain of Eproximately 2 ran is in good agreement with this hypothesis, since an extended chain of subunits mit be formed in the pore as a first step (Abraham, S. N., et al., J. Bacterial. i74:5145-5148 (1992); Thanassi, D. G., et al., Proc. Nat. Acad. Set USA 95:3146-3151 (1998)). The formation of the helical pilus rod at the outside of the bacterial membrane might then be the driving force responsible for translocation of the growing pilus through the membrane.<br>
It has also been demonstrated that the usher proteins of Type-1 and P-pili form ternary complexes with chaperone/subunit complexes with different afBnities (Dodson, K. W.. et al, Proc. Nat. Acad. Set USA 90:3670-3674 (1993); Sauliiro, E. T., et al., EMBO J. i7-.2177-2185 (1998)). This was interpreted as "kinetic partitioning" that allows a defined order of pilus proteins in the pilus. Moreover, it has been suggested that stmctural proteins mit present a binding surface only compatible with one other type of pilus protein; this would be another mechanism to achieve a highly defined order of subunits in the mature pilus (Saulino, E. T., etal., EMBO J. 77:2177-2185 (1998)).<br><br>
B.       Production of Type-1 pili from Escherichia coli<br>
E. coli strain WSllO was spread on LB (10 g/L tryptone, 5 g/L yeast extract, 5 g/L NaCI, pH 7.5. 1 % agar (w/v)) plates and incubated at 37Â°C overnight. A single colony was then used to inoculate 5 ml of LB starter culture (10 g/L tryptone, 5 g/L yeast extract, 5 g/L NaCl, pH 7.5). After incubation for 24 hours under conditions that favor bacteria that produce Type-1 pili (37Â°C, without agitation) 5 shaker flasks containing 1 liter LB were inoculated with one milliliter of the starter culture. The bacterial cultures were then incubated for additional 48 to 72 houra at 37Â°C without agitation. Bacteria were then harvested by centrifugation (5000 rpm, 4""C, 10 minutes) and the resulting pellet was resuspended in 250 milliliters of 10 mM Tris/HCl, pH 7.5. Pili were detached from the bacteria by 5 minutes agitation in a conventional mixer at 17.000 rpm-After centrifugation for 10 minutes at 10,000 rpm at 4Â°C the pili containing supernatant was collected and 1 M MgCl2 was added to a final concentration of 100 mM. The solution was kept at 4Â°C for 1 hour, and the precipitated pili were then pelleted by centrifugation (10,000 rpm, 20 minutes, 4Â°C). The peUet was then resuspended in 10 mM HEPES, pH 7.5, and the pilus solution was then clarified by a final centrifugation step to remove residual cell debris.<br>
C. Coupling of FLAG to purified Type-1 pUi of E. coli using m-Maleimidonbenzoyl-N-hydroxysulfosuccinimide ester (sulfo-MBS)<br>
600 I of a 95% pure solution of bacterial Type-I pili (2 mg/ml) were incubated for 30 minutes at room temperature with the heterobifunctional cross-Unker sulfo-MBS (0.5 mM). Thereafter, the mixture was dialyzed overnight against 1 liter of 50 mM Phosphate buffer (pH 7.2) with 150 mM NaQ to remove free sulfo-MBS. Then 500 1 of the derivatized pili (2 mg/ml) were mixed with 0.5 mM FLAG peptide (containing an amino-terminal Cysteine) in the presence of 10 mM H)TA to prevent metal-catalyzed sufhydryloxidation. The non-coupled peptide was removed by size-exclusion-chromatogrhy.<br>
EXAMPLE 34 Construction of an expression plasmid for the expression of Type-1 pili of<br>
Escherichia coli<br><br>
The DNA sequence disclosed in GenBank Accession No. U14O03, the entire disclosure of which is incorporated herein by reference, contains all of tiie Escherichia coli genes necessary for the production of type-1 pili from nucleotide number 233947 to nucleotide number 240543 (theyim gene cluster). This part of the sequences contains the sequences for the genes fimA., find, flmC, finiD, finiF, fimG, and fimR. Three different PCRs were employed for the amplification of this part of the E. coli genome and subsequent clomng into pUC19 (GenBank Accession Nos. L09137 and X02514) as described below.<br>
The PCR tenaplate was prepared by mixing 10 ml of a glycerol stock of the Â£ constrain W3110 with 90 ml of water and boiling of the mixture for 10 minutes at 95""C, subsequent centrifugation for 10 minutes at 14,000 rpm in a bench top centrifuge and collection of the supernatant.<br>
Ten ml of the supernatant were then mixed with 50 pmol of a PCR primra- one and 50 pmol of a PCR primer two as defined below. Then 5 ml of a lOX PCR buffer, 0.5 ml of Taq-DNA-Polymerase and water up to a total of 50 ml were added. All PCRs were carried out according to the following scheme; 94Â°C for 2 minutes, then 30 cycles of 20 seconds at 94""C, 30 seconds at 55Â°C, and 2 minutes at 12Â°C. The PCR products were then purified by 1% agarose gel-electrophoresis.<br>
Oligonucleotides with the following sequences with were used to amplify the sequence from nucleotide number 233947 to nucleotide number 235863, comprising tbefimKfind and_mC genes:<br>
TAGATGATrACGCCAAGCTTATAATAGAAATAOTTTnTGAAAG<br>
GAAAGCAGCATG (SEQ ID NO:196)<br>
and<br>
GTCAAAGGCCrrGTCGACGTrATTCCATTACGCCCGTCATnTG<br>
0 (SEQ ID NO: 197)<br>
These two oligonucleotides also contained flanking sequences that allowed for cloning of the amplification product into pucI9 via the restriction sites ffinrfm and Sail. The resulting plasmid was termed pFIMAIC (SEQ ID NO;198). Oligonucleotides with the following sequences with were used to amplify the sequence fix)m nucleotide number 235654 to nucleotide number 238666, comprising thefimD gene:<br>
AAGATCTTAAGCTAAGCnGAATTCTCTGACGCTGATTAACC<br>
(SEQ m NO: 199)<br>
and<br>
ACGTAAAGCATTTCTAGACCGCGGATAGTAATCGTGCTATC<br>
(SEQIDNO-.200).<br><br>
These two oligonucleotides also contained flanking sequences that allowed for cloning of the amplification product into pucl9 via the restriction sites Hindm and Xbal, the resulting plasmid was termed pFIMD (SEQ ID NO:201).<br>
Ohgonucleotides with the following sequences with were used to amplify the sequence from nucleotide number 238575 nucleotide number 240543, comprising thefimB,fimG, and fimH gene:<br>
AATTACGTGAGCAAGCTTATGAGAAACAAACCTrnTATC (SEQ<br>
ID NO:202)<br>
and<br>
GACTAAGGCCnTCTAGATTATTGATAAACAAAAGTCACGC<br>
(SEQIDNO:203).<br>
Tliese two oligonucleotides also contained flanking sequences that allowed for cloning of the amplification product into pucl9 via the restriction sites Hindm and Xbal\ the resulting plasmid was tenned pFIMFGR (SEQ ID NO:204).<br>
The following cloning procedures were subsequently carried out to generate a plasmid containing all die above-mentioned_/ini-genes:<br>
pFIMAIC was digested EcoRI and HindSi (2237-3982). pFIMD was<br>
digested EcdRl and SstJl (2267-5276), pFIMFGH was digested SstU and<br>
HindiR (2327-2231).  The fiagments were then ligated and the resulting<br>
plasmid, containing all the fim-genes necessary for pilus formation, was<br>
termed pHMAlCDFGH (SEQ ID NO;205).<br>
EXAMPLE 35 Construction of an expression plasmid for Escherichia coli type-1 pili that lacks the adhesion FimH<br>
The plasmid pHMAICDFGH (SEQ ID NO:205) was digested witii Kpnl, after which a fragment consisting of nucleotide numbers 8895-8509 was isolated by 0.7% agarose gelelectrophoresis and circularized by self-ligation. The resulting plasmid was tenned pFIMAICDFG (SEQ ID NO: 206), lacks the fimH gene and can be used for the production of FIMH-free type-1 pili.<br>
EXAMPLE 36 Expression of type-1 pili using the plasmid pHMAICDFGH<br><br>
Â£,. coil Strain W3110 was transformed with pFIMAICDFGH (SEQ ID NO:205) and spread on LB (10 g/L tryptone, 5 g/L yeast extract, 5 g/L NaCl, pH 7.5,<br>
1	% agar (w/v)) plates containing 100/ig/ml ampiciUin and incubated at 3"7Â°C<br>
overnight. A single colony was then used to inoculate 50 ml of LB-glucose starter<br>
culture (10 g/L tryptone, 5 g/L yeast extract, 1% (w/v) glucose, 5 g/L NaQ, pH 7.5,<br>
lOOmg/m] ampicillin). After incubation for 12-16 hours at 37Â°C at 150 rpm, a 5 liter<br>
shaker flasks containing 2 liter LB-glucose was inoculated with 20 milliliter of the<br>
starter culture. The bacterial cultures were then incubated for additional 24 at 37Â°C<br>
with agitation (150 rpm). Bacteria were then harvested by centrifugation (5000 rpm,<br>
4Â°C, 10 minutes) and the resulting pellet was resuspended in 250 milliliters of 10 mM<br>
Tris/HCl, pH 8. Pili were detached from the bacteria by agitation in a conventional<br>
mixer at 17,000 ipm for 5 minutes. After centrifugation for 10 minutes at 10,000<br>
rpm, 1 hour, "C the supernatant containing pili was collected and 1 M MgClj was<br>
added to a final concentration of 100 mM. The solution was kept at 4Â°C for 1 hour,<br>
and precipitated pih were then pelleted by centrifugation (10,000 rpm, 20 minutes,<br>
4Â°C). The pellet was then resuspended in 10 mM HEPES, 30 mM EDTA, pH 7.5, for<br>
30 minutes at room temperature, and the pilus solution was then clarified by a final<br>
centrifugation step to remove residual ceil debris. The preparation was then dialyzed<br>
against 20 mM HEPES, pH 7.4.<br>
EXAMPLES? Coupling of IgE epitopes and mimotopes to Type-1 pili of cherichia coli<br>
A 66 (il aliquot of a 100 ufA solution of the heteiobifuBttiDnal cross-linker sulfa-MBS was added to 400 1 of a 95% pure solution of bacterial Type-1 pili (2.5 mg/ml, 20 mM HEPES, pH 7.4) and subsequently incubated for 45 rrrinutes at room temperature with agitation. Ttereafter, the excess of sulfa-MBS was removed by size exclusion chromatography using a PD-10 column. Altematively, liie cross-linker can be removed by dialysis. Then either 1.3 fil of a solution containing 1.1 mg/ml peptide Ce3epi (CGGVNLTWSRA SG (SEQ ID NO:207)), or peptide CeSMim (CGGVNLPWSFGLE (SEQ ID NO:208) was added to 1 ml aliquots of the derivatized pili (1-1.25 mg/ml, 20 mMHEPES pH7.4). The samples were incubated at room temperature for 4 h and non-
2	times 2 1 of a buffer containing 20 mM HEPES (pH 7.4). Alternatively, the non-<br>
coupled peptide can be removed by size-exclusion chromatography.<br><br>
EXAMPLE 38<br>
Immunization of mice with a bee venom phospholipase Aj (PLAO<br>
fusion protein coupled to Qp capsid protein<br>
A. Preparation of an alternative vector for cytoplasmic expression of the catalytically inactive variant of the PLA2 gene fused to the amino acid sequence AAASGGCGG (SEQ ID NO: 209)<br>
The PLA2 gene construct of example 9 was amplified by PCR from pAVSPLAfos using oligos ecori_Ndel_j&gt;Ia (sequence below) and PLA-Cys-hind (Example 29). For the reaction, 100 pmol of each oHgo, and about 1 /ig of PAVSPLAfos DNA were used in the 50/il reaction mixtures with 1.2 units of Pfx DNA polymerase (Gibco), 1 mM MgSOa, 200 jiM dNTPS and Pfx Mihancer solution (Gibco) diluted ten times. For the reaction, temperature cycling was carried out as foUows: 94Â°C for 2 minutes, 5 cycles of 92Â°C (0.5 minutes), 58C (0.5 minutes), 68Â°C (1 minute); 25 cycles of 92Â°C (0.5 minutes), 63Â°C (0.5 minutes), eSC (1 minute). The PCR product was purified by agarose gel electrophoresis and subsequent isolation of tiie fragment using the Qiagen Qiaquick Kit, digested with enzymes Ndel and ffindlTI. and cloned into the PETUa vector (Novagen) digested with the same enzymes.<br>
Oligos:	ecorl_Ndelj)la:<br>
TAACCGAATTCAGGAGGTAAAAACATATGGC TATCATCTACC (SEQ ID NO: 214).<br>
The vector encoded a fusion protein having the amino acid sequence NtAIIYPGTLWCOHGNKSSGPNELGRFKHTDACCRTQDMCPDVMSAG ESKHGLTNTASinmaJCDDKFYDCLKNSADTISSYFVGKMYENLIDTK CYKLEHPVTGCGERTEGRCLHYTVDKSKPKVYQWFDLRKYAAASGGCG G(SEQIDNO:210).<br>
Coupling of PI-A2fusiDn protein to Q capsid protein<br><br>
A solution of 600/il of QP capsid protein (2 mg/ml in 20 mM Hepes,<br>
pH 7.4) was reacted with 176 fil Sulfo-MBS (13 mg/ml in 10) for 60 minutes at<br>
room temperature, and dialyzed against 1 L of 20 mM Hepes pH 7.4 0/N at 4Â°C.<br>
The next day, 500 1 of a PLA2 solution (2.5 mg/ml) containing O.I mM DTT<br>
were desalted over a 5 ml Hi-Trap column (Phannacia).  Reduced and desalted<br>
PLA2 (60 til, of a solution of approx. 0.5 mg/ml) was mixed with activated and<br>
dialyzed Qp capsid (25 (j.\ of a 1.5 mg/ml solution) and reacted for four hours at<br>
room temperature.<br>
1	Capsids  of 25-30  nm  diameter   are  clearly  visible  in  electron<br>
microscopy images of Qp capsid protein taken both before and after coupling to<br>
PIA2.<br>
C.        Immunization of mice with PLA2 coupled to QP capsid protein<br>
Female Balb/c mice were immunized intravenously on day 0 with 50 fig Qp capsid coupled to PLA2, and boosted on day 14 with the same amount of antigen. Mice were bled on day 20 and sera analyzed in an EOS A. A titer of 1:5000 against PLA2 was obtained.<br>
EXAMPLE 39 Coupling of IgE mimotopes and epitopes to QP capsid protein<br>
Human IgE epitopes having the following amino acid sequences were coupled to Qp capsid protein using the N-terminal cysteine residue: Ce3epitope: CGGVNLTWSRASG (SEQ ID NO:207) Ce3mimotope: CGGVNUWSK3LE (SEQ ID NO:208)<br>
The coupling reaction was performed using Qp capsid protein activated with Sulfo-MBS and subsequently dialyzed to remove excess crosslinker. The respective epitope or mimotope was diluted into the reaction mixture containing the activated Qp capsid, and left to react for 4 hours at room temperature. The reaction mixture was finally dialyzed for 4 hours against PBS, and injected into mice.<br>
The following circular mimotope was also coupled to QP capsid protein: Ce4mimotope: GEFCINHRGYWVCGDPA (SEQ ID N0:211).<br>
The mimotope was first reacted with the chemical group N-sucdnimidyl-S-acetylthioacetate (SATA), in order to introduce a protected sulfhydryl group into the miinotope. The protecting group was subsequently removed by treatment with hydroxylamine,  and immediately reacted with<br><br>
activated QP capsid protein, for 4 hours at room temperature.   The reaction mixture was finally dialyzed for 4 hours, and injected into mice.<br>
EXAMPLE 40 Immunization of mice with HBcAg-Lys coupled to M2 peptide<br>
A.       Coupling of M2 peptide to HBcAg-Lys capsid protein<br>
Synthetic M2 peptide, corresponding to an N-terminal fragment of the Influenza M2 protein with a cysteine residue at its C-teiminus (SLLXEVETPIRNEWGCRCNGSSDGGGC (SEQ ID NO:212)) was chemically coupled to purified HBcAg-Lys particles in order to elicit an immune response against the M2 peptide. Sulfo-MBS (232 fi\, 3 mM) was reacted with a solution of 1.4 ml HBcAg-Lys (1.6 mg/ml) in PBS. The mixture was dialyzed overnight against phosphate buffered saline (PBS). M2 peptide was diluted to a concentration of 24 mg/ml in DMSO; 5 1 of tiiis solution was diluted in 300 ;il PBS, 188 fi\ of which was added to 312 I of the dialyzed activated HBcAg-Lys solution. EDTA (10 (il of a 1 M solution) w also added to the reaction mixture, after which the reaction was allowed to proceed for 4 hours at room temperature.<br>
Immunization of mice with HBcAg-Lys coupled to M2 peptide<br>
Female Baib/c mice were immunized intravenously on day 0 with 50 /ig HBcAg-Lys-M2 or M2 peptide alone and boosted 10 days later with the same amount of antigen. After another 10 days, the mice were infected intranasally with Influenza virus (50 pfu, PR/8) and survival of infected mice was monitored. In addition, viral titers were determined in the lung. Mice primed with M2-HBcAg-Lys were fully protected and had eliminated the virus by day 7.<br>
EXAMPLE 41<br>
Coupling of M2 peptide to pili, QP and cys-free HbcAg-capsid protein and<br>
comparison of the antibody titer obtained by immunization of mice with these<br>
coupled pili and capsids with the titer obtained by immunizing mice with an<br>
N-tominal fusion protdln of the M2 peptide to HbcAgl-183<br>
A.       Coupling of M2 peptide to pili, QP- and cys-free HbcAg-capsid protein<br><br>
up:    A solution of 1 ml of 1 mg/ml Qp capsid protein in 20 mM<br>
Hepes. 150 mM NaC] pH 7,2 was reacted for 30 minutes with 93 /il of a solution of 13 mg/ml Sulfo-MBS (Pierce) in H2O at RT on a rocking shaker. The reaction solution was subsequently dialyzed overnight against 2 L of 20 mM hepes, 150 mM NaCl, pH 7.2. llie dialyzed reaction mixture was then reacted with 58.8 I of a 25 mM stock solution of M2 peptide (SEQ ID NO:212) in DMSO for four hours at RT on a rocking shaker. The reaction mixture was subsequently dialyzed against 2 liters of 20 mM ts, 150 mM NaCl, pH 7.2 overnight at 4Â°C.<br>
Cvs-free HbcAg: A solution of 1,25 ml of 0.8 mg/ml cys-free HbcAg capsid protein (example 31) in PBS, pH 7.2 was reacted for 30 minutes with 93 fil of a solution of 13 mg/ml Sulfo-MBS (Pierce) in HjO at RT on a rocking shaker. The reaction solution was subsequently dialyzed ovemit against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.2. The dialyzed reaction mixture was then reacted with 58.8 1 of a 25 mM stock solution of M2 peptide (SEQ ID NO:212) in DMSO for four hours at RT on a rocking shaker, llie reaction mixture was subsequently dialyzed against 2 liters of 20 mM hepes, 150 mM NaCl, ph 7.2 overnight at 4""C.<br>
Pili: A solution of 400 /il of 2.5 nml pili protein in 20 mM Hepes, pH 7.4, was reacted for 45 minutes with 60/il of a 100 mM Sulfo-MBS (Pierce) solution in (HjO) at RT on a rocking shaker. The reaction mixture was desalted on a PD-10 column (Amersham-Pharmacia Biotech), and tiie second fraction of 500 111 protein elating ftom the column (containing )proximately 1 g protein) was reacted with 58.8 (il of a 25 mM stock solution of M2 peptide (SEQ ID NO:212) in DMSO for four hours at RT on a rocking shaker. The reaction mixture was subsequently dialyzed against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.2 overnight at 4Â°C.<br>
Genetic fusion of the M2 peptide to HbcAgI-183<br>
M2 genetically fused to Hbc: M2 was cloned at the N-tenninus of Hbc as published by Neirynck et. al. Nature Medicine 5: 1157 (1999). MD-HBc was expressed in E. coli and purified by gel chromatography. The presence of the M2 peptide at the N-terminus of N-HBc was confirmed by Edman sequencing.<br>
Immunization of mice:<br><br>
Female Balb/c mice were vaccinated with M2 peptide coupled to pili, QP and cys-free HbcAg protein and with M2 peptide genetically fused to Hbc inimunogen without the addition of adjuvants. 35 {ig protein of each sample were injected intrapeiitoneally on day 0 and day 14. Mice were bled on day 27 and their serum analyzed using a M2-peptide specific ELISA.<br>
EUSA<br>
10 ng/vdi M2 peptide coupled to RNAse was coated on an EOSA plate. The plate was blocked then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, preimraune sera, were also tested. Control ELISA experiments using sera from mice immunized with unrelated peptides crosslinked to Ifl)c or other carriers showed the antibodies detected were specific for the M2 peptide. The results are shown in HG. 27 A and B.<br>
EXAMPLE 42<br>
Couphng of angiotensin I and angiotensin II peptides to Qfi and immunization<br>
of mice with QP - angiotensin peptide vaccines<br>
A.Coupling of angiotensin I and angiotensin n peptides to QP csid protein<br>
The following angiotensin peptides were chemically synthesized: CGGDRVYIHPF ("Angio T), CGGDRVYIHPFHL ("Angio H"), DRVYIHPEEILGGC ("Angio HI"), CDRVYIHPFKDL ("Angio IV") and used for chemical coupling to QP as described in the following.<br>
A solution of 5 ml of 2 mg/ml QP capsid protein in 20 mM Ifepes. 150 mM NaCl pH 7.4 was reacted for 30 minutes with 507 1 of a solution of 13 mg/ml Sulfo-MBS (Pierce) in H2O at 25Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 houK against 2 L of 20 mM Hepes, 150 noM NaCl, pH 7.4 at 4""C. 665 ml of the dialyzed reaction mixture was then reacted with 2.8 ml of each of the corresponding 100 mM peptide stock solution (in DMSO) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl,pH7.4at4""C.<br>
Immunization of mice:<br>
Female Balb/c mice vsre vaccinated with one of the four angiotensin peptides coupled to QP capsid protein without the addition of adjuvants. 50 /ig of<br><br>
total protein of each sample was diluted in PBS to 200 ml and injected subcutaneously (100 ml on two ventral sides) on day 0 and day 14. Mice were bled retroorbitally on day 21 and their serum was analyzed using a antgiotensin-specific ELISA.<br>
EUSA<br>
All four angiotensin peptides were individually coupled to bovine RNAse A using the chemical cross-linker sulfo-SPDP. EUSA plates were coated with coupled RNAse preparations at a concentration of 10 mg/ml. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, preimmune sera of the same mice were also tested. Control ELISA experiments using sera from mice immunized with unrelated peptides crosslinked to Qor other carriers showed ttiat the antibodies detected were specific for the respective peptide. The results are shown in FIG. 8A-8D.<br>
FIG. 8A, 8B, 8C and 8D, respectively, show ELISA analyses of IgG antibodies specific for "Angio I", "Angio II", "Angio HI", and "Anglo IV", respectively, in sera of mice immunized against Angio I-IV coupled to Qpcwid protein. QP-Angio I, QP-Angio H, QP-Angio HI and QP-Angjo IV, as used in the figures, stand for the vaccine injected in the mice, from which the sera are derived in accordance with above definition of the angiotensin peptides.<br>
Female Balb/c mice were vaccinated subcutaneously with 50 mg of vaccine in PBS on day 0 and day 14. IgG antibodies in sera of mice vaccinatecl with QP-Angio I, Qp-Angio JL, Qp-Angio in and QP-Angio IV were measured on day 21 against all four peptides (coupled to RNAse A), i.e. against "Angio I" ( HG. 8A), "Angio H" ( FIG. 8B), "Angio IH" ( HG. 80), and "Angio IV" (HG. 8D) . As a control, pre-immune SCTa from the same mice were analyzed. Results for indicated serum dilutions are shown as optical density at 450 nm. The average of three mice each (including standard deviations) is shown. AJl vaccinated mice made high IgG antibody titers against all four" peptides tested. No angiotensin-specific antibodies were detected in the controls (pre-immune mice).<br>
EXAMPLE 43<br>
Coupling of angiotensin I and angiotensin H peptides to HBcAp-149-lV5-<br>
2cvs-Mut. i.e. cys-free HBcAg.<br><br>
The following angiotensin peptides were chemically synthesized: CGGDRVYIHPF ("Angio I"), CGGDRVYIHPFHL ("Angio IT"), DRVYIHPEHLGGC ("Angio HI"), CDRVYIHPFHL ("Angio IV") and are used for chemical coupling to HBcAg-149-Iys-2cys-Mut, i.e. cys-free HBcAg.<br>
A solution of 1.25 ml of 0.8 mg/ml HBcAg-149-Iys-2c)Â«-Mut capsid protein (cf. Example 31) in PBS, pH 7.4 is reacted for 30 minutes with 93 (il of a solution of 13 mg/ml Sulfo-MBS ff"ierce) in HzO at 25Â°C on a rocking shaker. The reaction solution is subsequently dialyzed ovemit against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4. After buffer exchange the reaction solution is dialyzed for another 2 hours. The dialyzed reaction mixture is then reacted with 1.8 /il of a 100 mM peptide stock solution (in DMSO) for 2 hours at 25Â°C on a rocking shaker. The reaction mixture is subsequently dialyzed against 2 liters of 20 mM Hepes, 150 mM NaCl, ph 7.4 overnight at 4Â°C followed by buffer exchange and another 2 hours of dialysis.<br>
EXAMPLE 44 Coupling of angiotensin I and angiotensin II peptides to Type-1 pill of<br>
E.coli.<br>
Tte following angiotensin peptides were chemically synthesized: CGGDRVYIHPF ("Angio I"), CGGDRVYIHPFHL ("Angio IT"), DRVYIHPFHLGGC ("Angio m"). CDRVYIHPFHL ("Angio IV") and are used for chemical coupling to Type-1 pili oiKcoli.<br>
A solution of 400 i of 2.5 mg/ml Type-1 pili of Kcoli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with 601 of a 100 mM Sulfo-MBS (PiMce) solution in (H2O) at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech), "Hie protein-containing fractions eluating from the column are pooled (containing approximately 1 mg protein, i.e. derivatized pili) and reacted with a three-fold molar excess of peptide. For example, to 500 ul eluate containing approximately 1 mg derivatized pili, 2.34 ul of a 100 mM peptide stock solution (in DMSO) is added. The mixture is incubated for four hours at 25Â°C on a rocking shaker and subsequently dialyzed against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.2 overnight at 4Â°C.<br>
EXAMPLE 45<br>
Coupling of Der p I peptides to Qp and immunization of mice with<br>
Qp - Der p I vaccines<br><br>
Coupling of Der p I peptides to QP capsid protein<br>
The following peptides derived from the house dust mite allergen Der p I were chemically synthesized: CGNQSLDLAEQELVDCASQHGCH ("Der p I p52"; aa 52-72, with an additional cysteine-glycine hnker at the N terminus), CQIYPPNANKIREAIAQTHSA ("Der p 1 pll7"; aa 117-137). These peptides were used for chemical coupling to QP as described below.<br>
1ml of a solution consistiiig of 2 mg/ml Qp capsid protein in 20 mM Hepes, 150 mM NaCl, pH 7.4 was reacted for 30 minutes with 102 jiil of a solution of 13 mg/ml Sulfo-MBS (Pierce) in H2O at 25Â°C on a rocking shaker. The reaction solution was subsequentiy dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaC), pH 7.4 at 4""C. 440 /il of the dialyzed reaction mixture was then reacted with 1.9 ill of a 100 mM peptide stock solution (in DMSO) for two hours at 25 "C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl,pH7.4at4Â°C.<br>
Immunization of mice:<br>
Female Balb/c mice were vaccinated with one of the two Der p I peptides coupled to QP capsid protein without the addition of adjuvants. Two mice for each vaccine were used. 30 fig of total protein of each sairle was diluted in FBS to 200 1 and injected subcutaneously on day 0 and day 14. Mice weare bled retroorfaitally on day 21 and their serum was analyzed using a Der p I peptide-specific ELISA.<br>
EUSA<br>
The Der p I peptides "Der p I p52" and "Der p I pi 17" were individually coupled to bovine RNAse A using the chemical cross-linker sulfo-SPDP. ELISA plates were coated with coupled RNAse preparations at a concentration of 10 nral. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, preimmune sera of the same mice were also tested. Control ELISA experiments using sera from mice immunized with unrelated peptides crosslinked to QP or other carriers showed that the antibodies detected were specific for the respective peptide. The results are shown inHGS. 9Aand9B.<br><br>
HG.9A and FIG. 9B show ELISA analyses of IgG antibodies specific for "Der p I p52" (HG. 9A) and specific for "Der p I pll7" (FIG. 9B) in sera of mice immunized against the Der p I peptides coupled to Qp capsid protein. "p52" and "pll?", as used in FIGS. 9A and 9B, stand for the vaccine injected in the mice, from which the sera are derived.<br>
As a control, pre-immune sera from the same mice were analyzed (day 0). Results for indicated serum dilutions are shown as optical density at 450 nm. On day 21, all vaccinated mice made specific IgG antibodies against the Der p I peptide they were vaccinated with but not against the other Der p I peptide. No Der p I peptide-specific antibodies were detected before vaccination (day 0).<br>
Both Der p I peptide vaccines were hiy immunogenic in the absence of adjuvants. All vaccinated mice made good antibody responses specific for the peptide in the vaccine preparation.<br>
EXAMPLE 46 Coupling of Der p 1 peptides to HBcAg-149-lys-2cys-Mut, ix. cys-ftee<br>
HBcAg.<br>
The following peptides derived from the house dust mite allergen Der p I were cheroically synthesized: Der p I p52 (aa 52-72, with an additional cysteine-glycine linker at the N terminus): CGNQSLDLAEQELVDCASQHGCH, Der p I pU? (aa 117-137): CQIYPPNANKIREALAQTHSA. These peptides are used for chemical coupling to HBcAg-149-lys-2cys-Mut, i.e. cys-free HBcAg.<br>
A solution of 1.25 ml of 0.8 mg/ml HBcAg-149-lys-2cys-Mut capsid protein (Example 31) in PBS, pH 7.4 is reacted for 30 minutes with 93 1 of a solution of 13 mg/ml Sulfo-MBS (Pierce) in H2O at 25 "C on a rocking shaker. The reaction solution is subsequently dialyzed overnight against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.4. After buffer exchange the reaction solution is dialyzed for another 2 hours. The dialyzed reaction mixture is then reacted with 1.8 nl of a 100 mM peptide stock solution (in DMSO) for 2 hours at 25Â°C on a rocking shaker. The reaction mixture is subsequently dialyzed against 2 liters of 20 mM Hepes, 150 mM NaCI, ph 7.4 overnight at 4Â°C followed by buffer exchange and another 2 hours of dialysis.<br><br>
EXAMPLE 47 Coupling of Der p I peptides to Type-1 pili of E.coli<br>
The following peptides derived from the house dust mite allergen Der<br>
pi were chemically synthesized: Der p I p52 (aa 52-72, with an additional<br>
cysteine-glycine	linker	at	the	N	terminus)	and<br>
CGNQSLDLAEQELVDCASQHGCH, Derp 1 pll7 (aa 117-137): CQIYPPNANKIREAIAQTHSA. These pdes are used for chemical coupling to Type-1 pili of E.coli.<br>
A solution of 400 /il of 2.5 mg/ml Type-1 pili of Kcoli in 20 mM Hepes, pH 7.4, is reacted for 60 minutes with 60;xl of a 100 mM Sulfo-MBS (Pierce) solution in (H2O) at RT on a rocking shaker. The reaction mixture is desalted on a PD-10 column (Amersham-Pharmacia Biotech), The protein-containing fractions eluating from ihe column are pooled (containing approximately 1 mg protein, i.e. derivatized pili) and reacted with a three-fold molar excess of peptide. For example, to 500 ul eluate containing approximately 1 mg derivatized pili, 2.34 ul of a 100 mM peptide stock solution (in DMSO) is added. The mixture is incubated for four hours at 25Â°C on a rocking shaker and subsequently dialyzed against 2 Uten of 20 mM Hepes, 150 mM NaCl, pH 7.2 overnight at 4Â°C.<br>
EXAMPLE 48<br>
Coupling of HumanVEGFR-n Peptide to Type-1 pili of E.coli and<br>
Immunization of Mice with Vaccines Comprising Type-1 pili-<br>
HumanVEGFR-n Peptide Arrays<br>
Coupling of humanVEGKR-n peptide to Type-1 pili of E.coli<br>
Tte human VEGFR n peptide with the sequence CTARTELNVGIDFNWEYPSSKHQHKK was chemically synthesized and used for chemical coupling to"Type-1 pili of E.coli.<br>
A solution of 1400 (il of 1 mg/ml pili protein in 20 mM Ifepes, pH 7.4, was reacted for 60 minutes with SSjil of a 100 mM Sulfo-MBS (Pierce) solution in (H2O) at RT on a rocking shaker. The reaction mixture was desalted on a PD-10 column (Amersham-Phaimacia Biotech). The protein-containing fractions eluting from the column were pooled (containing approximately 1,4 mg protein) and reacted with a 2.5-foJd molar excess (final volume) of human VEGER H peptide. For example,, to 200 fil eluate containing approximately 0,2 mg derivatized pili, 2.4 1 of a 10 mM peptide solution (in DMSO) was added. The<br><br>
mixture was incubated for four hours  at 25Â°C  on  a rocking  shaker and subsequently dialyzed against 2 liters of 20 mM Hepes, pH 7.2 overnight at 4-Â°C.<br>
Immunization of mice<br>
Female C3H-HeJ (Toll-hke receptor 4 deficient, LPS non-responder mice) and C3H-HeN (wild-type) mice were vaccinated with the human VEGFR-H peptide coupled to Type-1 pili protein without the addition of adjuvants. Approximately 100 ng of total protein of each sample was diluted in PBS to 200 jUl and injected subcutaneously on day 0, day 14 and day 28. Nfice were bled retroorbitally on day 14, 28 and day 42 and serum of day 42 was analyzed using a human VEGFR-H specific EUSA<br>
ELISA<br>
Sera of immunized mice were tested in ELISA with immobilized human VEGER-H peptide and the extracellular domain of the human VEGFR-H (R&amp;D Systems GmbH, Wiesbaden).<br>
Human VEGFR-H peptide was coupled to bovine RNAse A using the chemical cross-linker sulfo-SPDP. ELISA plates were coated with coupled RNAse A at a concentration of 10 fig/mi. The human extracellular domain of VEGI-II was adsorbed to the plates at a concentration of 2 (Lg/wl. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. As a control, preimmune sera of the same mice were also tested. Control ELISA experiments using sera firom mice immunized with uncoupled caniar showed that the antibodies detected were specific for the respective peptide. The results for human VEGFR II peptide coupled to Type-1 pili are shown in Figure 10. In particular, FIG.IOA. and FIG. lOB show ELISA analyses of IgG antibodies specific for human VEGFR II peptide and extracellular domain of human VEGFR II, respectively, in sera of mice immunized against human VEGFR n peptide and the extracellular domain of human VEGFR n each coupled to Type-1 pili protein.<br>
Female C3H-HeJ (Toll-like receptor 4 deficient, LPS-nonresponder) and C3H-HeN (wild-type) mice were vaccinated subcutaneously with 100 ug of vaccine in PBS on day 0, 14 and 28. Serum IgG against the peptide (coupled to RNAse A) and the extracellular domain of human VEGFR U were measured on day 42. As a control, preimmune sera from the same mice were analyzed. Results for indicated serum dilutions are shown as optical density at 450 mn. The average of three mice each (including standard deviations) are shown. All vaccinated mice<br><br>
made high IgG antibody titers against the human VEGFR-II peptide as well as the extracellular domain of human VEGFR-II (KDR) and no difference was noted between mice deficient for the Toll-like receptor 4 and wild-type mice. The latter is remarkable since it demonstrates that formation of high IgG antibody titers against the human VEGFR-II peptide as well as the extracellular domain of human VEGFR-II is independent of endotoxin contaminations.<br>
EXAMPLE 49<br>
Coupling of HumanVEGFR-II Peptide to QP Capsid Protein and<br>
Immunization of Mice with Vaccines Comprising QP Capsid Protein -<br>
HumanVEGFR-n Peptide Airays<br>
Coupling of HumanVEGFR-n Peptide to Qp Capsid Protein<br>
"Hie human VEGFR II peptide with the sequence CTARTELNVGIDFNWEYPSSKHQHKK was chemically synthesized and is used for chemical coupling to QP csjd protein.<br>
A solution of 1 ml of 1 mg/ml QP capsid protein in 20 mM Hepes, 150 mM Naa pH 7.4 was reacted for 45 minutes with 20 jil of 100 mM Sulfo-MBS (Pierce) solution in (H2O) at RT on a rocking shaker. Tlie reaction solution was subsequently dialyzed twice for 2 hours in 2 L of 20 mM Hepes, pH 7.4 at 4 "C. 1(K&gt;0 1 of the dialyzed reaction mixture was then recited with 12 fil of a 10 mM human VEGFR n peptide solution (in DMSO) for four hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x2 hours against 2 liters of 20 mM Hepes, pH 7.4 at 4 "C.<br>
1ml of a solution consisting of 2 mg/ml QP csid protein in 20 mM Hepes, 150 mM NaCl, pH 7.4 was reacted for 30 minutes with 102 /tl of a solution of 13 mg/ml Sulfo-MBS (Pierce) in H2O at 25""C on a rocking shaker. TTie reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4""C. 440 fi\ of the dialyzed reaction mixture was then reacted with 1-9 jil of a 100 mM peptide stock solution (in DMSO) for two hours at 25 Â°C on a rocking shaker. Tlie reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM Naa.pH7.4at4X.<br><br>
Immunization of Mice<br>
C57BL/6 mice are vaccinated with the human VEGFR-II peptide coupled to QP protein without the addition of adjuvants. Approximately 50 [xg of total protein of each sample is diluted in PBS to 200 ul and injected subcutaneously on day 0, day 14 and day 28. Mice are bled retrooibitally on day 14, 28 and day 42 and serum of day 42 is analyzed using a human VEGFR-H specific ELISA<br><br>
EXAMPLJE50<br>
Coupling of HumanVEGFR-E Peptide to HBcAg-149-lys-2cys-Mut<br>
Capsid Protein, i.e. cys-free HBcAg, and Immunization of Mice with<br>
Vaccines Comprising HBcAg-149-lys-2cys-Mut Capsid Protein -<br>
humanVEGFR-n Peptide Arrays<br>
Coupling   of HumanVEGFR-U  Peptide  to  HBcAg-149-lys-2cys-Mut Capsid Protein<br>
The human VEGER II peptide with the sequence CTARTELNVGIDFNWEYPSSKHQHKK was chemically synthesized and is used for chemical couphng to HBcAg-i49-lys-2cys-Mut capsid protein.<br>
A solution of 3 ml of 0.9 mg/ml cys-free HbcAg capsid protein (cf. Example 31) in PBS, pH 7.4 is reacted for 45 minutes with 37.5 ]il of 100 mM Sulfo-MBS (Pierce) solution in (H2O) at RT on a rocking shaker. The reaction solution is subsequently dialyzed overnight against 2 L of 20 mM Hepes, pH 7.4. AftM buffer exchange the reaction solution is dialyzed for another 2 hours. The dialj"zed reaction mixture is then reacted with 3 pi of a 10 mM human VEGFR H peptide solution (in DMSO) for 4 hours at 25 C on a rocking shaker. The reaction mixture is subsequently dialyzed against 2 litCTS of 20 mM Hepes, pH 7.4 ovemight at 4 C followed by buffer exchange and another 2 hours of dialysis.<br>
EXAMPLES! Construction of HBcAgl-183Lys<br>
Hepatitis core Antigen (HBcAg) 1-183 was modified as described in Example 23. A part of the c/el epitope (residues 72 to 88) region (Proline 79 and Alanine 80) was genetically replaced by the peptide Gly-Gly-Lys-Gly-Gly (HBcAgl-183Lys construct). The introduced Lysine residue contains a reactive amino group in its side chain that can be used for intermolecular chemical crosslinldng of HBcAg particles with any antigen containing a free cysteine group. PCR methods essentially as described in Example 1 and conventional cloning techniques were used to prepare the HBcAgl-183Lys gene.<br>
The Gly-Gly-Lys-Gly-Gly sequence was inserted by ampling two separate fragments of the HBcAg gene ficom pEco63, as described above in Example<br><br>
23 and subsequently fusing the two fragments by PCR to assemble the full length gene. The following PCR primer combinations were used:<br>
fragment 1:<br>
Primer 1: EcoRIHBcAg(s) (see Example 23) Primer 2: Lys-HBcAg(as) (see Example23)<br>
fragment 2: Primer 3: Lys-HBcAg(s) (see Example23) Primer 4: HBcAgwtHindllll CGCGTCCCAAGCTrCTAACATTGAGATTCCCGAGATTG<br>
Assembly: Primer i: EcoRIHBcAg(s) (see example 23) Primer 2: HBcAgwtHindUn<br>
The assembled fall length gene was then digested with the EcoRI (GAATTC) and Hindlll (AAGCTT) enzymes and cloned into the pKK vector (Phannacia) cut at the same restriction sites.<br>
EXAMPLE 52<br>
Coupling of muTNFa Peptide to HBcAgl-183Lys and Immunization of Mice with<br>
Vaccines Comprising HBcAgl-183Lys - muTNFa Peptide Arrays<br>
A.       Coupling of muTNFa Peptide to HBcAgl-183Lys<br>
HBcAgl-183Lys at a concentration of 0.6 mg/ml (29 fiM) was treated with iodacetamide as described in Example 32. HBcAgl-183Lys was then reacted with a fifty-fold excess of the cross-linker Sulfo-MBS, as described in Example 32, and dialyzed overnight against 20mM Hepes, pH 7.2, at 4""C. Activated (derivatized) HBcAgl-183Lys was reacted with a five-fold molar excess of the peptide muTNFa (sequence: CGGVEEQLEWLSQR, diluted directiy into the HBcAgl-183Lys solution fit)m a 100 mM stock solution in DMSO) at RT for 4 hours. The coupling reaction (about 1 ml solution) was dialyzed against 2x 2 liters of 20mM HEPES pH 7.2, at 4Â°C, for 4 hours. The dialyzed coupling reaction was frozen in aliquots in liquid nitrogen and stored at -80Â°C until immunization of the mice.<br>
Immunization<br><br>
Two mice (female Balb/c) were immunized intravenously at day 0 and 14 with 100 /ig HBcAgI-183Lys coupled to the muTNFa peptide, per animal, without adjuvant Antibodies specific for the muTNFa peptide (coated as a Ribonuclease A conjugate) and for native TNFa protein (Sigma) in the serum were determined at day 21 by ELISA.<br>
ELISA<br>
Murine TNFa protein (Sigma) was coated at a concentration of 2 /ig/ml. As a control, preimmune sera from the same mice used for immunization were tested. FIG. 14 shows the result of the ELISA experiment, demonstrating that inmranization with.HBcAgl-183Lys coupled to tiie muTNFa peptide (FuU length HBc-TNF) generated an immune response specific for the murine TNFa protein. The sera from mice bled on day 0 (preimmune) and 21 were tested at three different dilutions. Each bar is the average of the signal obtained with sera from two mice. Thus, vaccination with HBcAgl-183Lys coupled to the muTNFa peptide induced an immune response against a self-antigen, since the amino acid sequence of the muTNFa peptide is derived from the sequence of mouse TNFa protan.<br>
EXAMPLE 53<br>
Coupling of 3"TNF n Peptide to 2cysLys-mut HBcAgl-149 and<br>
Immunization of Mice with Vaccines Comprising 2cysLys-mut HBcAgl-149<br>
- 3"TNF n Peptide Arrays<br>
Coupling of the 3"TNF n peptide to 2cysLys-mut HBcAgl-149<br>
2cysLys-mut HBcAgl-149 was reacted at a concentration of 2 mg/ml<br>
for 30 min. at RT with a fifty-fold excess of cross-linker in 20 mM Hopes, 150<br>
mM NaCl, pH 7.2. Excess cross-linker was removed by dialysis overnight, and<br>
activated (derivatized) 2cysLys-mut HBcAgl-149 capsid protein was reacted with<br>
a	ten-fold	excess	of        3"TNF	n	peptide        (SEQ:<br>
SSQNSSDKPVAHWANHGVGGC, dUuted from a 100 mM stock solution in DMSO) for 4 hours at RT. The reaction mixture was then dialyzed overnight in a dialysis tubing with a molecular weight cutoff of 50000 Da, ftxizen in liquid nitrogen and stored at -80Â°C until immunization of the mice.<br>
Immunization of Mice<br><br>
3 Female C3H/HeN mice, 8 weeks of age were vaccinated with die 3"TNF n peptide coupled to 2cysLys-mut HBcAgl-149 without die addition of adjuvants. 50 \ig of total protein was diluted in PBS to 200 fi\ and injected subcutaneously (100 1 on two inguinal sides) on day 0 and day 14. Mce were bled retroorbitally on day 0 and 21, and their serum were analyzed in an KITS A specific for murine TKFa protein.<br>
ELISA<br>
Murine TNFa protein (Sigma) was coated at a concentration of 2 /ig/ml. As a control, preimmune sera from the same mice used for immunization were tested. FIG. 15 shows the result of the ELISA, demonstrating that immunization with 2cysLys-mut HBcAgI-149 coupled to the 3"TNF H peptide generated an immune response specific for the murine IHPa protein. The sera fim mice bled on day 0 (preimmune) and 21 were tested at three different dilutions. Each bar is the average of the signal obtained with sera from 3 mice. Thus, vaccination with 2cysLys-mut HBcAgl-149 coupled to the 3"TNF E peptide induced an immune response specific for a self-antigen, since the amino acid sequence of the 3"TNF n peptide is derived fixim the sequence of murine TNFa protein.<br>
EXAMPLE 54<br>
Coupling of Ap 1-15, AP 1-27 and AP 33-42 peptides to QP and immunization of mice with vaccines comprising QP - Ap peptide arrays<br>
A.       Coupling of Ap 1-15 and AP 33-42 peptides to Qp capsid protein using the cioss-Unker SMPH.<br>
The following AP peptides were chemically synthesized: DAEFRHDSGYEVHHQGGC (abbreviated as "Ap 1-15"), a peptide which comprises the amino acid sequence from residue 1-15 of human Ap, fused at its C-terminus to the sequence GGC for coupUng to QP capsid protein and CGHGNKSGLMVGGWIA (abbreviated as "Ap 33-42") a peptide which conrises the amino acid sequence from residue 33-42 of AP, fused at its N-tenninus to the sequence CGHGNKS for<br><br>
coupling to QP capsid protein. Both peptides were used for chemical coupling to Qp as described in the following.<br>
A solution of 1.5 ml of 2 mg/ml QP capsid protein in 20 mM Hepes 150 mM NaCl pH 7.4 was reacted for 30 minutes with 16.6 1 of a solution of 65 naM SMPH (Pice) in H2O, at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 Â°C in a dialysis tubing with Molecular Weight cutoff 10000 Da. 450 pi of the dialyzed reaction mixture, which contains activated (derivatized) QP, was then reacted with 6.5 \il of each of the corresponding 50 mM peptide stock solution (in DMSO) for two hours at 15Â°C on a rocking shaker. 200 |xl of the reaction mixture was subsequently dialyzed overnight against 2 liters of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 Â°C, and the next morning for another two hours after change of buffer. The reaction mixture was then frozen in aliquots in liquid Nitrogen and stored at -80Â°C until immunization of the mice.<br>
The results of the coupling experiments were analyzed by SDS-PAGE, and are shown in FIG.13A and F1G.13B. The arrows point to the band corresponding to one, respectively two peptides coupled to one ( subimit CFIG.13A), or one peptide coupled to one Qpsubunit (FIG.13B). Molecular weights of maÂ±er proteins are given on the left margin of FIG. ISAandHG.DB.<br>
The samples loaded on the gel of FIG.13A are the following: 1: derivatized Qp; 2: Qp coupled with "Api-15", supernatant of the sample taken at the end of the coupling reaction, and centrifuged; 3: Qp coupled with "Api-15", pellet of the sample taken at the end of the coupling reaction, and centrifuged. 4: QP coupled with "Api-i5", supernatant of a sample left to stand 24 hours at 4 "C, undialyzed and centrifuged. 5: QP coupled with "Apl-15", pellet of a sample left to stand 24 hours at 4 Â°C, undialyzwi and centrifuged.<br><br>
6: QP coupled with "Apl-15", supernatant of the sample taken after dialysis of the coupling reaction, and centrifuged.<br>
The samples loaded on the gel of HG.13B are the following: 1: derivatized QP 2: Qp coupled with "Ap33-42", supernatant of the sample taken at the end of the coupling reaction, and centrifuged. 3; Qp coupled with "Ap33-42", pellet of the sample taken at die end of the coupling reaction, and centrifaged. 4: Qp coupled with "AP33-42", supernatant of a sample left to stand 24 hours at 4 Â°C, undialyzed and centrifuged. 5: QP coupled with "AP332", pellet of a sample left to stand 24 hours al 4 Â°C, undialyzed and centrifuged.<br>
6: Qp coupled with "AP33-42", supernatant of the sample taken after dialysis of the coupling reaction, and centrifuged.<br>
B.       Coupling of "ApI-27" peptide to QP capsid protdn using the cross-linker SMPH.<br>
The following AP peptide ("Apl-27") was chemically synthesized DAEFRHDSGYEVHHQKLVFFAEDVGSNGGC. This peptide comprises the amino acid sequence from residue 1-27 of human AP, fused at its C-terminus to the sequence GGC for coupling to Qp capsid protein.<br>
A first batch of "Api-27" coupled to QP capsid protein, in the following abbreviated as "QP-Api-27 batch 1" was prepared as follows:<br>
A solution of 1.5 ml of 2 mg/ml Qp capsid protein in 20 mM Hepes 150 mM NaCI pH 7.4 was reacted for 30 minutes with 16.6 1 of a solution of 65 mM SMPH (Pierce) in HzO, at 25 Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 "C in a dialysis tubing with Molecular "Weight cutoff 10000 Da, 450 ul of the dialyzed reaction mixture was then reacted with 6.5 pi of a 50 mM peptide stock solution (in DMSO) for two hours at 15Â°C on a rocking shaker. 2O0 fil of the sample was then aliquoted, frozen in liquid Nitrogen and stored at -BOC until immunization of the mice.<br><br>
A second batch of "A 1-27" coupled to Qp capsid protein, in the following abbreviated as "QP-AP 1-27 batch 2" was prepared as follows:<br>
500 lii of QP capsid protein in 20 mM Hepes 150 mM NaQ pH 7.4 was reacted for 30 minutes with 11.3 /il of a solution of 32.5 mM SMPH (Pierce) in H2O, at 25 Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 roM NaCl, pH 7.4 at 4 Â°C in a dialysis tubing with Molecular Weight cutoff 3500 Da (SnakeSkin, Pierce). The dialyzed reaction mixture was then reacted with 3.6 nl of a 50 mM peptide stock solution (in DMSO) for two hours at 15Â°C on a rocking shaker. Tlie reaction mixture was then dialyzed 2X against 1 1 20 mM Hepes, 150 mM NaCI, pH 7.4 for 1 hour and ovemight after a last change of buffer, using a dialysis membrane with a 50000 Da cutoff (Spectnqwr, spectrum). The reaction mixture was then fix)zen in aliquots in liquid nitrogen and stored at -SOC until immunization of the mice. "QP-Ap 1-27 batch 1" was used for the first immunization, wiiile "QP-AP 1-27 batch 2" was used for the boost<br>
The result of the coupling experiment was analyzed by SDS-PAGE, and is shown in FIG. 13C. The arrow points to the band corresponding to one peptide coupled to one QP subunit.<br>
The samples loaded on the gel of FIG.13C are the following: M: protein marker. 1: QP capsid protein 2: derivatized QP, supernatant of the sample taken at the end of ttie derivatization reaction, and centrifuged. 3: derivatized Qp, pellet of the sample taken at the end of the derivatization reaction, and centrifuged. 4: QP coupled with "APl-27", supernatant of the sample taken at the end of the coupling reaction, and centrifuged. 5: QP coupled with "Apl-27", pellet of the sample taken at the end of the coupling reaction, and centrifuged. 6: Qp coupled with "APl-27", supernatant of the sanyjle taken after dialysis of the couphng reaction, and centrifuged. 7: Qp coupled with "Api-27", pellet of the sample taken after dialysis of the couphng reaction, and centrifuged.<br><br>
C.       Coupling of "AP 1-15" peptide to QP capsid protein using the cross-linker Sulfo-GMBS<br>
A solution of 500 /il of 2 mg/ml QP csid protein in 20 mM Hepes 150 mM NaCl pH 7.4 was reacted for 30 minutes with 5.5 /xl of a solution of 65 mM SMPH (Pierce) in H2O, at 25 Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 "C in a dialysis tubing with Molecular Weight cutoff 10 000 Da. 500 nl of the dialyzed reaction mixture was then reacted with 6.5 (il of the 50 naM peptide stock solution (in DMSO) for two hours at 15Â°C on a rocking shaker. 200 pi of the reaction mixture was subsequently dialyzed overnight against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 C, and the next morning for another two hours after change of buffer. The reaction mixture was then frozen in aliquots in liquid Nitrogen and stored at â80Â°C until immunization of the mice.<br>
The result of the coupling experiment was analyzed by SDS-PAGE, and is shown in HG. 13D. The airow points to the band corresponding to one, two and three peptides, respectively, coupled to one Qp subunit.<br>
The samples loaded on the gel of FIG.13D are the foDowing: M: protein marker. 1: derivatized Qp 2; QP coupled with "Api-15". supMnatant of the sample taken at the end of the coupling reaction, and centiifuged. 3: Qp coupled with "Api-15", pellet of the sample taken at the end of the coupling reaction, and centrifuged, 4: QP coupled with "Api-15", supernatant of a sample left to stand 24 hours at 4 Â°C, undialyzed and centrifuged. 5: QP coupled with "APl-15", pellet of a sample left to stand 24 hours at 4 Â°C, undialyzed and centrifiiged. 6: Qp coupled with "Api-15", supernatant of the sample taken after dialysis of the couphng reaction, and centrifuged.<br><br>
Â»<br><br>
D.	Coupling of "Ap 1-15" to Q capsid protein using the cross-linker Sulfo-<br>
MBS.<br>
500 MJ of Qp capsid protein in 20 mM Hepes 150 mM NaQ pH 7.4 was reacted for 30 minutes with 14.7 (li of a solution of 100 mM Sulfo-MBS (Pierce) in HzO, at 25 C on a rocking shalar. The reaction solution w subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 inM NaCI, pH 7.4 at 4 Â°C in a dialysis tubing (SnakeSkin. Pierce) with Molecular Weit cutoff 3500 Da. The dialyzed reaction mixture was then reacted with 7.2 jxl of a 50 mM peptide stock solution (in DMSO) for two hours at 15Â°C on a rocking shaker. The reaction mixture was then dialyzed 3 X over 4 hours against 2 120 mM Hepes, 150 mM NaCI, pH 7.4 using a dialysis membrane with a 50000 Da cutoff (Spectrior, spectrum). The reaction mixture was then frozen in aliquots in liquid nitrogen and stored at -SOC until immumzation of Che mice.<br>
The result of the coupling experiment was analyzed by SDS-PAGE, and is shown in HG.13E. The arrow points to the band corresponding to one peptide coupled to one QP subunit.<br>
The samples loaded on the gel of FIG.ISE are the following: I: QP capsid protein 2: derivatized QP, supematant of die sample taken at the end of the derivatization reaction, and centrifiiged. 3: derivatized Qp, pellet of the sample taken at the end of the derivatizatiOD reaction, and centrifuged. 4: derivatized Qp, supematant of the sample taken at the end of tbe dialysis of the derivatization reaction, and centrifuged, 5: derivatized QP, pellet of the sample taken at the end of the dialysis of the derivatization reaction, and centrifuged. 6: Qp coupled with "Api-15", supematant of the sample taken at the end of the coupling reaction, and centrifuged. 7: Qp coupled with "Api-15", pellet of the sample taken at the end of the coupling reaction, and centrifuged. 8: Qp coupled with "Apl-15", supematant of the sample taken after dialysis of the coupling reaction, and centrifuged.<br>
E.	Immumzation of mic<br><br>
Five groups of female C57BL/6 mice, three mice per group, 8 weeks of age were vaccinated each with one of the five Ap peptiite-QP capsid protein conj"ugates without the addition of adjuvant. 25 fig of total protein of each sample was diluted in PBS to 200 nl and injected subcutaneously on day 0 and day 14. Mice were bled retroorbitally on day 0 (preimmune) and 21 and their serum was analyzed in an EUSA. "Ap 1-15" peptide was coupled to Qp with three different cross-linkers, resulting in three different vaccine preparations ("Qb-Apl-15 SMPH", "QP-Abl-15 SMBS", "Qb-Api-15 SGMBS"; see ELISA section for the results).<br>
F.        EUSA<br>
AH three Ap peptides were individually coupled to bovine RNAse A using the chemical cross-linker SPDP as follows: a solution of 10 mg RNAse A in 2mL PBS (SOniM Phoshate buffer, 150mM NaCl pH 7.2) was reacted with 100  of a 20 mM SPDP solution in DMSO, at 25Â°C for 60 min. on a rocking shaker. Excess cross-hnker was separated from activated (deiivatized) RNAse A by gel filtration using a PD 10 column (Pharmacia). The protein containing fractions were pooled and concentrated to a volume of 2 ml using centrifugal filters (5000 MWCO), A sample of 333 yd of the derivatized RNAse A solution was reacted with 2 il of the peptide stock solution (50 mM in DMSO). The coupling reaction was foUwed spectrophotometrically.<br>
ELISA plates were coated with RNAse A coupled to peptide at a concentration of 10 fig/ml. The plates were blocked and then incubated with serially diluted mouse sera. Bound antibodies were detected with enzymatically labeled anti-mouse IgG antibody. Preimmune sera or control sera from mice immunized with unrelated peptides conjugated to QP, showed that the antibodies detected were specific for the respective peptide. FIG. 14A, FIG. 14B and FIG.14C, respectively, show ELISA analyses of IgG antibodies specific for "Ap 1-15", "Ap 1-27" and "Ap 33-<br><br>
42", respectively, in sera of mice immunized against "A 1-15", "Ap i-27" and "Ap 33-42", respectively, coupled to QP capsid protein. TTie denominations on the abscissa stand for the vaccine injected in the mice from which the sera are derived, and describe the peptide and the cross-Unker used to make the respective vaccine. All sera were measured against the three peptides coupled to RNAse A, and the results show that while there is cross-reactivity between the antibodies raised against peptide 1-15 and 1-27, no such cross reactivity is observed against peptide 33-42, demonstrating the specificity of the immune response. Likewise, The ELISA titers obtained, expressed as the dilution of the serum yielding an EUSA signal three standard deviations above background, were very high, and ranged from 60*000 to 600"000. No A peptide-specific antibodies were detected in the controls (pre-immune mice).<br>
EXAMPLE 55<br>
Introduction of cys-containing linkers, expression, and purification of anti-<br>
idiotypic IgE mimobodies and their coupling to Qp capsid protein<br>
A.       Construction of plasmids for the expression of mimobodies for coupling to QP capsid protein<br>
Plasmids were based on the expression plasmid VAE051-pASKl 16. This plasmid contains the coding reons for the heavy chain and for the light chain of the mimobody. The following primers were used to introduce cys-containing linkers at the C-tenninus of the heavy chain:<br>
Primer CA2F: CGGCTCGAGCATCACCATCACCATCACGGTGAAGTTAAACTGCAGCTG GAGTCG<br>
Primer CAIR: CATGCCATGGTTAACCACAGGTGTGGGTrrrATCACAAGATTTGGGCT CAAC<br><br>
mmerCiiiK:<br>
CATGCCATGGTTAACCACACGGCGGAGAGGTGTGGGTTTTATCACAAG ATTTOGGCTCAAC<br>
Primer CCIR: CCAGAAGAACCCGGCGGGGTAGACGGTITCGGGCTAGCACAAGATTT GGGCTCAACTC<br>
Primer CCIF; CGCCGGGTTCTTCTGGTGGTGCTCCGGGTGGTTGCGGTTAACCATGGA GAAAATAAAGTG<br>
Primer CCR2: CTCCCGGGTAGAAGTCAC<br>
A. 1.     Construction of pCA2:<br>
Primers CA2F and CAIR were used to amplify a 741 bp fragment encoding part of the heavy chain with an extension encoding the cj-containing linker sequence. VAE-pASK116 served as template for the Pfe polymerase (Roche) in the PCR cycler (Robo) at (initial denaturation at 92Â°C, cycling: 92Â°C, 30 s; 48Â°C, 30 s; 68Â°C, 60s) for 5 cycles followed by 30 cycles with 92Â°C, 30 s; 58Â°C, 30 s; 68Â°C, 1 min. The PCR product of the appropriate size was purified using the Qiagen PCR purification idt and digested with Xhol and Ncol according to the recommendation of the manufacturer (Gibco). The product was purified from an agarose gel with the Qiagen gel extraction kit. Plasmid VAE-pASKll6 was in parallel cleaved with Xhol and Ncol and a 3.7 kb band purified from agarose gels. Appropriate aliquots of the Xhol-Ncol digested PCR product and the plasmid were ligated overnight at 16Â°C using T4 DNA ligase according to the manufacturer"s protocoll (Gibco). The ligation product was transfonned into competent E.coli XL-1 cells which were plated on agarose plates containing chloramphenicol. Single colonies were expanded in LB/chloramphenicol medium, plasmid was prepared (Qiagen mini plasmid kit) and tested for the presence of the tpropriate XhoI-NcoI insert size after digestion with the corresponding enzymes. A correspondingly positive plasmid termed pCA2 was sent for sequencing<br><br>
on both strands which confirmed the identity of the plasmid including the cys-containing linker.<br>
A.2.    Construction of pCB2:<br>
Primers CA2F and CBIR were used to introduce linker 2 at the 5" end of the heavy chain coding sequence and the same conditions as described in section Al. The resulting PCR product was 750 bp and cloned into VAE051-pASK116 as described in section.l.<br>
A.3.     Construction of pCC2:<br>
Plasmid pCC2 was constructed in a two step procedure: A first PCR product of 754 bp was amplified using primers CA2F and CCIR . A second PCR product of 560 bp was produced using primers CCIF and CC2R. For both PCRs VAED51-pASKI 16 was used as template and conditions were as described in section Al. Both PCR products were isolated from agarose gels, mixed with primMS CA2F and CC2R and a third PCR was performed that resulted in a 1298 bp fragment This fragment was isolated and digested with Xhol and Ncol. The resulting 780 bp fragment was cloned into VAE-pASKlOO as described in section A.1.<br><br>
B.	Expression of mimobodies<br>
Competent E. coli W3110 cells were transformed with plasmids pCA2, pCB2 and pCC2. Single colonies from chloramphenical agarose plates wee expanded m liquid culture (LB +15 ng/ml chloramphenicol) overnight at 3TC. 11 of TB medium was then inoculated 1:50 v/v with the overnight culture and grown to OD600=3 at 28Â°C. Expression was induced with 1 mg/I anhydrotetracyclin. Cells were harvested after overnight culture and centrifiiged at 6000 ipm. Periplasma was isolated from cell pellets by incubation in lysis buffer supplemented with polymyxin B sulfate for 2 h at 4Â°C. Spheroblasts were separated by centrifugalion at 6000 ipm. The resulting supernatant contained the miroobody and was dialysed against 20 mM Tris, pH 8.0.<br>
C.	Purification of mimobodies<br>
The inbuced his6-tag allowed the purification of mimobody pCA2 and pCB2 by chromatography on Ni-NTA fast flow (Qiagen) according the recommendations of the manufacture". If necessary, a polishing step on a protein G fast flow column (Amersham Pharmacia Biotech) followed. Mimobodies were eluted with 0.1 M glycine pH 2.7, immediately neutralized by addition of NaOH and dialysed against 20 mM Hepes, 150 mM NaQ, pH 7.2.<br>
pCC2 was purified by affinity chromatography on protein G only. Purity was analysed by SDS-PAGE.<br>
The protein sequences of the mimobodies were translated from the cDNA sequences. N-terminal sequences were confirmed by Edman sequencing of pCA2 and pCB2.<br>
The sequence of the light chains of pCA2, pCB2 and pCC2 is the same and as follows:<br>
DIELVVTQPASVSGSPGQSrnSCTGTRSDVGGYNY\"SWYQQHPGKAPKL<br>
MIYDVSNRPSGVSNRFSGSKSGNTASLTISGLQAEDEADyYCSSYTSSSTL<br>
GVFGGGTK1.TVLGQPKAAPSVT1J?PPSSEEIJ3AKKA.TLVCUSDFYPGAVT<br>
VAWKADSSPVKAGVETTTPSKQSNNKYAASSYLSLTPEQWKSHKSYSCQ<br>
VTHEGSTVEKTVAPTECS<br>
The sequence of the heavy chain of pCA2 is: EVKIX3LEHHHHHHGEVKLX3I-ESGPGLVKPSETLSLTCTVSGGSISSGGYY WIWmQRPGKGIWIGYiyYSGSTSYNPSLKSRVTMSVDTSKNQFSLRLT SVTAADTAVYYCARERGETGLYYPYYYIDVWGTGTTVTVSSASTKGPSV FPIJ1J*SSKSTSGGTAAIXJCLVKDYFPH&gt;VTVSWNSGALTSGVHTFPAVLQ SSGLYSlSVVTWSSSITQTYICNVNHKPSNTKVDKRVEPKSCDKTHrC G<br>
The sequence of the heavy chain of pCB2 is: EVK1IÂ£HHHHHHGEVKLQIÂ£SGPGLVKPSETLSLTCTVSGGSISSGGYY WTWmQiaÂ»GKGLEWIGYlVYSGSTSYNPSIJCSRVTMSVDTSKNQFSLRLT SVTAADTAVYYCARERGETGLYYPYYYIDVWGTGTTVTVSSASTKGPSV FPlJU"SSKSTSGGTAAUjCLVKDYITEPVTVSWNSGALTSGVHrFPAVLQ SSGLYSlSVVTVPSSSIXjTQTYtaWNHKPSNTKVDKRVEPKSCDKIHTS PPCG<br>
The sequence of the heavy chain of pCC2 is: EVKLQLEEIHHHHHGEVKLQLESGPGLVKPSEILSLTCTVSGGSISSGGYY WTVratQRPGKGLEWIGYlYYSGSTSYNPSIJCSRVTMSVDTSKNQFSLRLT SVTAADTAVYYCARERGGLYYPYYYIDVWGTGTrVTVSSASTKGPSV FPLAPS SKSTSGGTAALGCtVKD YFPEP VTVSWNSGALTSGVHTFPAVLQ SSGLYSISVVTWSSSLGX3TYICAHKPS}TKVDKRVEPKSCASPKPS TPPGSSGGAPGGC<br>
D.       Coupling of mimobodies to Qp capsid protein D.l.    Couphng of mimobody pCC2 to Qp capsid protein;<br><br>
A solution of 1.25 ml of 4.5 mg/ml QP capsid protein in 20 mM Hepes, 150 mM NaCl pH 7.2 was reacted for 30 minutes with 40 I of a SMPH solution (Pierce) (from a 100 mM stock solution dissolved in DMSO) at 25 Â°C on a rocking shaker. The reaction solution w subsequently dialyzed twice for 2 hours against 2 I of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. 6 fjJ of the dialyzed reaction mixture was then reacted with 30 \j} of the pCC2 solution (2.88 mg/ml) for at 25 "C over night on a rocking shaker.<br>
The reaction products were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were either stained with Coomassie Brilliant Blue or blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a polyclonal rabbit anti-Qb antiserum (dilution 1:2000) or a mouse monoclonal anti-Fab-mAb (Jackson ImmunoResearch) (dilution 1:2000). Blots ware subsequently incubated with horse radish peroxidase-conjugated goat anti-rabbit IgG or horse radish peroxidase-conjugated goat anti-mouse IgG (dilutions 1:7000), respectively<br>
The results are shown in FIG ISA. Coupling products and educts were analysed on 16% SDS-PAGE gels under reducing conditions. In FIG. 13A "pCC2" coiresponds to the mimobody before coupling. "Qp deriv" stands for derivatized QP before coupling, "QP-pCC2" for the product of the coupling reaction. Gels were either stmned with Coomassie Brilliant Blue or blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a polyclonal rabbit anti-Qp antiserum (dilution 1:2000) or an mouse monoclonal anti-Fab-mAb (Jackson ImmunoResearch) (dilution 1:2000). Blots were subsequently incubated with hole radish peroxidase-conjugated goat anti-rabbit IgG or horse radish peroxidase-conjugated goat anti-mouse IgG (dilutions 1:7000), respectively. Enhanced chemoluminescence (Amersham Pharmacia ELC kit) was used to visualize the immunoreactive bands. Molecular weights of markra: proteins are given on the left margin.<br>
A coupling product of about 40 kDa could be detected (FIG. 13A, arrows). Its reactivity with the  anti-Qp antiserum and the anti-Fab<br><br>
antibody recognizing the mimobody clearly demonstrated the covalent coupling of the mimobody to QP.<br>
D.2.     Coupling of mimobodies pCA2 and pCB2 to Qp capsid protein:<br>
A solution of 1.25 ml of 4.5 mg/ml QP capsid protein in 20 mM Hepes, 150 mM NaC! pH 7.4 was reacted for 30 minutes with 40 td of a SMPH erce) (from a 100 mM stock solution dissolved in DMSO) at 25 Â°C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 1 of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. pCA2 (1.2 mg/ml) was reduced with 20 mM TCEP for 30 min at 25Â°C, pCB2 (4.2 mg/ml) with 50 mM raercaptoethylamine at 37Â°C. Both mimobodies were then dialyzed twice against 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 Â°C. Coupling was performed by adding 6 1 of derivatized QP to 30 ] of mimobody at 25Â°C over night on a rocking shaker.<br>
The reaction products were analysed on 16% SDS-PAGE gels under reducing conditions. Gels were either stained with Cooraassie Brilliant Blue or blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a polyclonal rabbit anti-Qb antiserum (Cytos, dilution 1:2000) or an mouse monoclonal anti-his6-mAb (Qiagen) (dilution 1:5000). Blots wem subsequently incubated with horse radish peroxidase-conjugated goat anti-rabbit IgG or horse radish peroxidase-conjugated goat anti-mouse IgG (dilutions 1:5000), respectively.<br>
The results are shown in FIG. 13B and FIG. 13C. Couphng products and educts were analysed on 16% SDS-PAGE gels under reducing conditions. In HG.ISA and HG.15B "pCA2" and "pCB2" corresponds to the mimobodies before coupling. "Qb deriv" stands for derivatized Qp before coupling and "Qp-pCA2" and "Qp-pCA2" for the products of the coupling reaction. Gels were either stained with Coomassie Brilliant Blue or blotted onto nitrocellulose membranes. Membranes were blocked, incubated with a polyclonal rabbit anti-Qb antiserum (dilution<br><br>
1:2000) or an mouse monoclonal anti-his6-roAb (Qiagen) (dilution 1:5000). Blots were subsequently incubated with horse radish peroxidase-conjugated goat anti-rabbit IgG or horse radish peroxidase-conjugated goat anti-mouse IgG (dilutions 1:5000), respectively. Enhanced chemoluminescence (Amersham Pharmacia ECL kit) was used to visualize the immunoreactive bands. Molecular weights of marker proteins are given on the left margin.<br>
Coupling products of about 40 kDa could be detected for both the pCA2 and the pCB2 coupling (FIG.15A and HG.15B, aiiows). Its reactivity with the anti-QP antiserum and the anti-his6 antibody recognizing the heavy chain of the mimobody clearly demonstrated the covalent coupling of the mimobody to Qp.<br>
EXAMPLE 56<br>
Coupling of Flag peptides to wt and mutant Q capsid protefai using tbe cross-linker Sulfo-<br>
â     GMBS<br>
The Flag peptide, to which a CGG sequence was added N-tenninally for coupling, was chemically synthesized and had the following sequence: CGGDYKDDDDK. This peptide was used for chemical coupling to wt QP capsid protein and the QP mutants capsid protein as described in the following.<br>
E.   Coupling of Flag peptide to Qp capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP capsid protein in 20 mM Hepes. 150<br>
mM NaCl pH 7.2 was reacted for 60 minutes with 7 /il of a solution of 65 mM Sulfo-GMBS (Pierce) in H2O at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaQ, pH 7.2 at 4 "C. 100 pi of the dialyzed reaction mixture was then reacted with 0.58 \il of 100 mM Flag peptide stock solution (in H2O) for two hours at 25 "C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 *C.<br>
B.   Coupling of Hag peptide to QP-240 capsid protein<br>
	A solution of 100 ul of 2 mg/ml Q240 capsid p-otein in 20 mM Hepes.<br>
150 mM NaCl pH 7.2 was reacted for 60 minutes with 7 /tl of a solution of<br><br>
65 mM Sulfo-GMBS (Pierce) in H2O at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 ""C. 100 Ml of the dialyzed reaction mixture was then reacted with 0.58 I of 100 mM Flag peptide stock solution (in H:0) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 *C.<br>
C.   Coupling of Flag peptides to QP-250 capsld protein<br>
	A solution of 100 ul of 2 mg/ml Qp-250 capsid protein in 20 mM<br>
Ifcpes. 150 roM NaCl pH 7.4 was reacted for 60 minutes with 7 /tl of a solution of 65 mM Sulfo-GMBS (Pierce) in HjO at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 *C. 100 <br>i. of the dialyzed reaction mixture was then reacted with 0.58 \il of 100 mM Flag peptide stock solution (in H2O) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 lifers of 20 mM Hepes, 150 mM NaQ, pH 7.2 at 4 "C.<br>
D. Coupling of Flag peptides to Q&amp;-259 capsid protein<br>
	A solution of 100 ul of 2 mg/ml Qp-259 capsid protein in 20 mM<br>
Hepes. 150 mM NaCl pH 7.4 was reacted for 60 minutes with 7 /il of a solution of 65 mM Sulfo-GMBS (Pierce) in H2O at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaQ, pH 7.4 at 4 "C. 100 I of the dialyzed reaction mixture was then reacted with 0.58 JAI of 100 mM Flag peptide stock solution (in H2O) for two hours at 25 "C on a rocking shaker. The reaction mixture was subsequentiy dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 "C.<br>
The results of the coupling reactions of the QP mutants 240, 250 and 259 to Hag peptide analyzed by SDS-PAGE are shown in FIG. 22 A. The loading pattern yas the following:<br><br>
1. Derivatized Qp-240 2. QP-240 coupled to the Flag peptide 3. Derivatized Qp-250 4. Qp-250 coupled to the Flag peptide 5. Daivatized Qp-259 6. Qp-259 coupled to the Flag peptide 7. Draivatized wt QP   8.   wt Qp coupled to the Flag peptide 9. Protein<br>
Marker.<br>
Comparison of the daivatized reaction with tiie coupling reactions shows that for all the mutants and wt, coupling bands corresponding to 1 and 2 peptides per subumt are visible. The band corresponding to the uncoupled QP subunit is very weak, indicating that nearly all subunits have reacted with al least one Hag peptide. For the QP-250 mutant and wt Qp, a band corresponding to three peptides per subumt is visible. The ratio of the intensities of the band corresponding to two peptides per subunit and the band coiresponding to 1 peptide per sulwnit is strongest for wt, with a ratio of 1:1. this ratio is stUl high for the QP-250 mutant, while it is significantiy weaker for the Qp-240 mutant and weakest for the Qp-259 mutant.<br>
EXAMPLES? Coupling of Flag peptide to Qp cqidd protein using tlie cross-linker Solfo-MBS<br>
The Bag peptide, to which a CGG sequence was added N-terminally for coupling, was chemically synthesized and had the following sequence: CGGDYKDDDDK. This peptide was used for chemical coupling to wt QP capsid protein and the Qp mutant capsid protein as described in the following.<br>
F.   Coupling of Flag peptides to Qp capsid protein<br>
_A solution of 100 ul of 2 mg/ml QP capsid protein in 20 mM Hepes. 150 mM NaCl pH 7.2 was reacted for 60 minutes with 7 ftl of a solution of 65 mM Sulfo-MBS (Pierce) in H2O at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 "C. 100 pi of tiie dialyzed reaction mixture was then reacted with 0.58 \tl of 100 mM Hag peptide stock solution (in H2O) for two hours at 25  on a rocking shaker. The reaction mixture was subsequenfly dialyzed 2x 2 hours against 2 liters of<br><br>
20 mM Hepes, 150 mM NaCI, pH 7.2 at 4 *C.<br>
B.   Coupling of Hag peptide to Qp-240 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-240 capsid protein in 20 mM Hepes.<br>
150 mM NaCl pH 7.2 was reacted for 60 minutes with 7 /il of a solution of 65 mM Sulfo-MBS (Pioce) in H2O at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 *C. 100  of the diaJyzed reaction mixture was then reacted with 0.58 \xi of 100 mM Flag peptide stock solution (in H2O) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 *C.<br>
C.   Coupling of Flag peptide to QP-250 capsid protein<br>
_A solution of 100 ul of 2 mg/ml QP-250 capsid protein in 20 mM<br>
Hepes. 150 mM NaCl pH 7.2 was reacted for 60 minutes with 7 /il of a solution of 65 mM Sulfo-MBS (Pierce) in H2O at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaQ, pH 7.2 at 4 *C. 100 pi of the dialyzed reaction mixture was then reacted with 0.58 \il of 100 mM Flag peptide stock solution (in H2O) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequeaitly dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.2 at 4 *C.<br>
D.   Coupling of Flag pqrtides to QP-259 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-259 capsid protein in 20 mM<br>
Hepes. 150 mM NaCl pH 7.2 was reacted for 60 minutes with 7 /il of a solution of 65 mM Sulfo-MBS erce) in H2O at 25 *C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, i50mMNaCl,pH7.2at4*C. 100 fit of the dialyzed reaction mixture was then reacted with 0.58 |jJ of 100 mM Flag<br><br>
peptide stock solution (in H2O) for two hours at 25 Â°C on a rocking shaker. The reaction tnixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaQ, pH 7.2 at 4 *C.<br>
The results of the coupling reactions of the QP mutants 240, 250 and 259 to Flag peptide analyzed by SDS-PAGE are shown in Figure 1. The loading pattern was the following:<br>
1. Protein Marker 2. Derivatized QP-240 3. QP-240 coupled to the Flag peptide 4. Derivatized Qp-250 5. Qp-250 coupled to the Flag peptide 6. Derivatized Qp-259 7. Qp-259 coupled to \he Flag peptide 8. Derivatized wt Qp   9.  wt QP coupled to the Hag peptide.<br>
Comparison of the derivatized reaction with the coupling reactions shows that for all the mutants and wt, a coupling band corresponding to 1 peptide per subunit is visible. Bands corresponding to 2 peptides per subunit are also visible for the mutant Qp-250 and wt Qp. The ratio of the intensities of the band corresponding to 1 peptide per subunit and to the uncoupled subunit, respectively, is higher for the Qp-250 mutant and wt QP. A weak band corresponding to two peptides per subunit is visible for the Qp-240 mutant<br>
EXAMPLE 58 Coupling of Flag peptides to QP mutants using the cross-linker SMPH<br>
The Flag peptide, to which a CGG sequence was added N-terminaDy for coupling, was chemically synthesized and had the following sequence: CGGDYKDDDDK. This peptide was used for chemical coupling to the QP mutants as described in the following.<br>
A   Coupling of Flag peptides to QfS-240 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-240 capsid protein in 20 mM Hepes.<br>
150 mM NaCl pH 7.4 was reacted for 30 minutes with 2.94 {il of a solution of 100 mM SMPH (Pierce) in DMSO at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 "C. 90 d of the<br><br>
dialyzed reaction mixture was then reacted with 1.3 fjJ of 50 mM Hag peptide stock solution (in DMSO) for two houK at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 hters of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 *C.<br>
B.   Coupling of Flag ptide to QP-250 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-250 capsid protein in 20 mM<br>
Hepes. 150 mM NaCI pH 7.4 was reacted for 30 minutes with 2.94 1 of a solution of 100 mM SMPH (Pierce) in DMSO at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI. pH 7.4 at 4 "C. 90 \il of the dialyzed reaction mixture was then reacted witii 1.3 jil of 50 mM Flag peptide stock solution (in DMSO) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaQ, pH 7.4 at 4 "C.<br>
C.   Coupling of Flag ptidc to QP-259 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-259 capsid protein in 20 mM<br>
Hepes. 150 mM NaCI pH 7.4 was reacted for 30 minutes with 2.94 /il of a solution of 100 mM SMPH (Pierce) in DMSO at 25 "C on a rocking shalcer. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 *C. 90 d of the dialyzed reaction mixture was then reacted with 1.3 fil of 50 mM Flag peptide stock solution (in DMSO) for two hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 Uters of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 "C.<br>
The results of the coupling reactions of the QP mutants 240, 250 and 259 to Flag peptide analyzed by SDS-PAGE are shown in Figure 1. The loading pattern was the following. 1. Protein Marker 2. Qp-240 coupled to Flag, pellet of the coupling reaction 3. Q|3-240 coupled to Flag, Supematant of the coupling reaction 4. QP-240 derivatized with SMPH 5.<br><br>
QP-250 coupled to Flag, pellet of the coupling reaction 6. Q250 coupled to Flag, supernatant of the coupling reaction 7. QP-250 derivatized with SMPH 8. Qp-259 coupled to Flag, pellet of the coupling reaction 9. Qp-259 coupled to Flag, supernatant of the coupling reaction 10. QP-259 derivatized with SMPH.<br>
Comparison of the derivatized reaction with the coupling leactions shows that for all the mutants, coupling bands corresponding to 1, respectively 2 peptides per subunits are visible. Bands corresponding to three, respectively four peptides per subunit are also visible for the mutant QP-250.<br>
EXAMPLE 59 Coupling of PLAj-Cys proteio to mntant Qp capsid proteins<br>
Lyophilized mutant QP capsid proteins were swollen overnight in 20 mM Hcpes, 150 mM NaCl, pH 7.4.<br>
A,   Coupling of PLAi-Cys protein to Qp-240 capsid protein<br>
	A solution of 100 ul of 2 mg/ml QP-240 capsid protein in 20 mM Hepes.<br>
150 mM NaCI pH 7,4 was reacted for 30 minute with 2.94 I of a solution of 100 mM SMPH (Pierce) in DMSO at 25 "C on a rocking shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 *C. 90 (il of the dialyzed reaction mixture was mixed with 146 ul 20 mM Hepes, 150 mM NaCl, pH 7.4 and reacted with 85.7 ul of 2.1 mg/ml FLAz-Cys stock solution for four hours at 25 "C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 *C.<br>
B. Coupling of PLAi-Cys protein to QP-250 capsid protein<br>
A soJutioD of 100 ui of 2 mg/ml QP-250 capsid protein in 20 mM Hepes. 150 mM NaCl pH 7.4 was reacted for 30 minutes with 2.94 /il of a solution of 1(X) mM SMPH (Pierce) in DMSO at 25 *C on a rocking<br><br>
shaker. The reaction solution was subsequently dialyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 *C. 90 fil of the dialyzed reaction mixture was mixed with 146 ul 20 mM Hepes, 150 mM NaCl, pH 7.4 and reacted with 85.7 ul of 2.1 mg/ml PLA2-<:ys stock solution for four hours at on a rocking shaker. the reaction toixture was subsequently dialyzed against liters of mm hepes nacl ph></:ys>
C.   Coiqjling of PLAj-Cys protcio to Q]3-259 capsid protein<br>
A solution of 100 ul of 2 mg/ml Qp-259 capsid protein in 20 mM Hepes. 150 mM NaCl pH 7.4 was reacted for 30 minutes with 2.94 /il of a solution of 100 mM SMPH (Pierce) in DMSO at 25 *C on a rocking shalrer. The reaction solution was subsequently cUalyzed twice for 2 hours against 2 L of 20 mM Hepes, 150 mM NaCI, pH 7.4 at 4 "C. 90 fd of the dialyzed reaction mixture was mixed with 146 1 20 mM Hepes, 150 mM NaCl, pH 7.4 and reacted with 85.7 1 of 2.1 mg/ml PLAa-Cys stock solution for four hours at 25 Â°C on a rocking shaker. The reaction mixture was subsequently dialyzed 2x 2 hours against 2 liters of 20 mM Hepes, 150 mM NaCl, pH 7.4 at 4 "C.<br>
The results of the coupling experiment analyzed by SDS-PAGE are shown in Figure 1. The loading pattern was the following; 1. Protein Marker 2. daivatized Qp-240 3. Qp-240 coupled to PIa2Cys. supematant of the coupling reaction 4. Qp-240 coupled to PLAa-Cyg, pellet of the coupling reaction 5. derivatized QP-250 6. QP-250 coupled to PLArCys, supematant of the coupling reaction 7. Q-250 coupled to PI2-Cys, pellet of the coupling reaction 8. derivatized QP-259 9. Qp-259 coupled to PLA2-Cys, supematant of the coupling reaction 10. QP-259 coupled to PLA2-C!ys, pellet of the coupling reaction 11. PLAj-Cys.<br>
Coupling bands (indicated by the arrow in the figure) were visible for aU the mutants, showing that PLAj-Cys protein could be coupled to all of the mutant Qp capsid proteins.<br><br>
1	All   patents   and   publications   referred   to   herein   are   expressly<br>
incorporated by reference.<br><br>
The entire disclosure of U.S. Application No. 09/449,631 and WO 00/3227, both filed November 30, 1999, are herein incorporated by reference in their entirety. All publications and patents mentioned hereinabove are hereby incorporated in their entireties by reference.<br><br>
EEQXreMCE LISTIN<br> Cytos Biotechnology Novartis Pharroa AG<br>
Renner, Wolfgang A_ Bachmann, Martin Tissot, Alain Maurer, Patrick Lechner, Franziska Sebbel, Peter Piossek, Christine Ortmann, Rainer Luond, Rainer Staufenbiel, Matthias Frey, Peter<br> Molecular Antigen Array<br> 17D0.019PC05<br> (To be assigned)  2002-01-18<br> US 60/262,379  2001-01-19<br> US 60/288,549  2001-05-04<br> US 60/326,998  2001-10-05<br><i50> US 60/331,045  2001-11-07<br> 350<br> Patentin Ver. 2.1<br> 1<br> 41<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 1<br>
ggggacgcgt gcagcaggta accaccgtta aagaaggcac c<br> 2<br> 44<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 2<br><br>
"ggtggttac ctgctgcacg cgttgcttaa gcgacatgta gcgg	44<br> 3<br> 20<br> DNA<br> Artificial Sequence<br><br> Description of Artificial SeqMeaoe: Primer<br> 3<br>
ccatgaggcc tacgataccc	20<br> 4<br> 25<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Priflier<br> 4<br>
ggcactcacg gcgcgcttta caggc	25<br> 5<br> 47<br> DMA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Prisier<br>  5<br>
ccttctttaa cggtggttac ctgctggcaa ccaacgtggt tcatgac	47<br> 6<br> 40<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>   6<br>
aagcatgctg cacgcgtgtg cggtggtcgg atcgcccggc	40<br> 7<br> 90<br> DNA<br> Artificial Sequence<br><br> Pescription of Artificial Sequence:   Primer<br> 7<br>
gggtctagat tcccaaccat tcccttatcc aggctttttg acaacgctat gctccgcgcc 50<br>
catcgtctgc accagctggc ctttgacacc	90<br><br>
!:210&gt; B<br> 108<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>   8<br>
gggtctagaa ggaggtaaaa aacgatgaaa aagacagcta tcgcgattgc agtggcactg 60<br>
gctggtttcg ctaccgtagc gcaggccttc ccaaccattc ccttatcc	10Â£<br> 9<br> 31<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 9<br>
cccgaattcc tagaagccac agctgccctc c	31<br> 10<br> 24<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 10<br>
cctgcggtgg tctgaccgac accc	24<br> 11<br> 41<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 11<br>
ccgcggaaga gccaccgcaa ccaccgtgtg ccgccaggat g	41<br> 12<br> 33<br> DNA<br> Artificial Seqpience<br><br> Description of Artificial Sequence: Primer<br>  12<br>
ctatcatcta  gaatgaatag  aggattcttt   aac	33<br>  13   15<br><br>
â  DNA<br> Artificial  Seguence<br><br>
â =:223&gt; Description of Artificial Sequence;  Modified ribosome binding site<br> 13<br>
aggaggtaaa aaaog	15<br> 14<br> 21<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence;   signal peptide<br>  14<br>
Met Lys Lys Thr Ala lie Ala lie Ala Val Ala Leu Ala Gly Phe Ala<br>
15	10	15<br>
Thr Val Ala Gin Ala 20<br> 15<br> 46<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence:  modified Fos construct<br> 15<br>
Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala Glu Thr Asp Gin Val Glu<br>
15	10	15<br>
Asp Glu Lys Ser Ala Leu Gin Thr Glu lie Ala Asn Leu Leu Lys Glu<br>
20	25	30<br>
Lys Glu Lys Leu Glu P&amp;e He Leu Ala Ala His Gly Gly Cys<br>
35	40	45<br> 16<br> 5<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence;  peptide linker<br>
Â«100&gt; 16<br>
Ala Ala Ala  Ser Gly Gly<br>
1	5<br>   17    6   PRT<br><br> Artificial Seguence<br><br> Description of Artificial Sequence: peptide linker<br> 17<br>
Gly Gly Ser Ala Ala Ala<br>
1	5<br> 18<br> 256<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fog fusion construct<br> 18<br>
gaattcagga ggtaaaaaac gatgaaaaag acagctatcg	cgattgcagt ggcactggct 60<br>
ggtttcgcta ccgtagcgca ggcctgggtg ggggcggccg	cttctggtgg ttgcggtggt 120<br>
ctgaccgaca ccctgcaggc ggaaaccgac caggtggaag	acgaaaaatc cgcgctgcaa 180<br>
accgaaatcg cgaacctgct gaaagaaaaa gaaaagctgg	agttcatcct ggcggcacac 240<br>
ggtggttgct aagctt	256<br> 19<br> 52<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br> 19<br>
Ala Ala Ala Ser Gly Gly Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala<br>
5    .	10	15<br>
Glu Thr Asp Gin Val Glu Asp Glu Lys Ser Ala Leu Gin Thr Glu lie<br>
20	25	30<br>
Ala Asn Leu Leu Lys Glu Lys Glu Lys Leu Glu Phe He Leu Ala Ala<br>
35	40	45<br>
His Gly Gly Cys 50<br> 20<br> 261<br> DHA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Pos fusion construct<br><br> CDS<br> (22}..(240)<br> 20<br><br>
gaattcagga ggtaaaaaac g atg aaa aag aca get ate gcg att gca gtg  51<br>
Met Lys Lys Thr Ma He Ala He Ala Val<br>
15	10<br>
gca ctg get ggt ttc get aec gta gcg cag gee tgc ggt ggt ctg ace  99<br>
Ala Leu Ala Gly Phe Ala Thr Val Ala Gin Ala Cys Gly Gly Leu Thr<br>
15	20	25<br>
gac aec ctg cag gcg gaa ace gac cag gtg gaa gac gaa aaa tec gcg  147<br>
Asp Thr Leu Gin Ala Glu Thr Asp Gin Val Glu Asp Glu Lys Ser Ala<br>
30	35	40<br>
ctg eaa aec gaa ate gcg aac ctg ctg aaa gaa aaa gaa aag ctg gag  195<br>
Leu Gin Thr Glu He Ala Asn Leu Leu Lys Glu Lys Glu Lys Leu Glu<br>
45	50	55<br>
ttc ate ctg gcg gca cac ggt ggt tgc ggt ggt	tct gcg gcc get      240<br>
Phe He Leu Ala Ala His Gly Gly Cys Gly Gly	Ser Ala Ala Ala<br>
60                 65	70<br>
gggtgtgggg atatcaagct t	261<br> 21<br> 73<br> PET<br> ArtiEicial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br> 21<br>
Met Lys Lys Thr Ala He Ala He Ala Val Ala Leu Ala Gly Phe Ala<br>
15	10	15<br>
Thr Val Ala Gin Ala Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala Glu<br>
20	25	30<br>
Thr Asp Gin Val Glu Asp Glu Lys Ser Ala Leu Gin Thr Glu He Ala<br>
35	40	45<br>
Asn Leu Leu Lys Glu Lys Glu Lys Leu Glu Phe He Leu Ala Ala His<br>
50	55	60<br>
Gly Gly Cys Gly Gly Ser Ala Ala Ala<br>
65	70<br> 22<br> 195<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br><br> CDS<br> (34)..[IBS)<br> 22<br><br>
gaattcagga ggtaaaaaga tatcgggtgt ggg gcg gcc get tct ggt ggt tgc 54<br>
Ala Ala Ala Ser Gly Gly Cys<br>
1	5<br>
ggt ggt ctg ace gac ace ctg cag gcg gaa ace gac cag gtg gaa gac  102<br>
Gly Gly Leu Thr Asp Thr Leu Gin Ala Glu Thr Asp Gin Val Glu Asp<br>
10	15	20<br>
gaa aaa tec gcg ctg caa ace gaa ate gcg aac ctg ctg aaa gaa aaa  150<br>
Glu Lys Ser Ala Leu Gin Thr Glu lie Ala Asn Leu Leu Lys Glu Lys<br>
25	30	35<br>
gaa aag ctg gag ttc ate ctg gcg gea cac ggt ggt tgc taagctt      196<br>
Glu Lys Leu Glu Phe lie Leu Ala Ala His Gly Gly Cys<br>
40	45	50<br> 23<br> 52<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br> 23<br>
Ala Ala Ala Ser Gly Gly Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala<br>
15	10	15<br>
Glu Thr Asp Gin Val Glu Asp Glu Lya Ser Ala Leu Gin Thr Glu lie<br>
20	25	30<br>
Ala Asn Leu Leu Lys Glu Lys Glu Lys Leu Glu Phe lie Leu Ala Ala<br>
35	40	45<br>
His Gly Gly Cys 50<br> 24<br> 204<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br> 24<br>
gaattcagga ggtaaaaaac	gatggcttgc ggtggtctga ccgacaccct gcaggcggaa 60<br>
aocgaccagg tggaagacga	aaaatccgcg ctgcaaaccg aaatcgcgaa cctgctgaaa 120<br>
gEiaaaagaaa agctggagtt	catcctggcg gcacacggtg gttgcggtgg ttctgcggcc 180<br>
gctgggtgtg gggatatcaa	gctt	204<br> 25<br> 56<br>
â <:212> PRT<br> Artificial Sequence<br><br><br> Description of Artificial Sequence: Fos fusion construct<br> 25<br>
Lys Thr Met Ala Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala Glu Thr<br>
15	10	15<br>
Asp Gin Val Glu Asp Glu Lys Ser Ala Leu Gin Thr Glu lie Ala Asn<br>
2D	25	30<br>
Leu Leu Lys Glu Lys Glu Lys Leu Glu Phe lie Leu Ala Ala His Gly<br>
35	40	45<br>
Gly Cys Gly Gly Ser Ala Ala Ala<br>
50	55<br> 25<br> 2 6<br> PRT<br> Homo sapiens<br> 26<br>
Met Ala Thr Gly Ser Arg Thr Ser Leu Leu Leu Ala Phe Gly Leu Leu<br>
15	10	15<br>
Cys Leu Pro Trp Leu Gin Glu Gly Ser Ala<br>
20	25<br> 27<br> 262<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Fos fusion construct<br> 27<br>
gaattcaggc ctatggctac aggctcccgg acgtccctgc tcctggcttt tggcctgctc 60<br>
tgcctgccct ggcttcaaga gggcagcgct gggtgtgggg cggccgcttc tggtggttgc 120<br>
ggtggtctga ccgacaccct gcaggcggaa accgaccagg tggaagacga aaaatccgcg 180<br>
ctgcaaaccg aaatcgcgaa cctgctgaaa gaaaaagaaa agctggagtt catcctggcg 240<br>
gcacacggtg gttgctaagc tt	2S2<br> 28<br> 52<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequenfce: Fos fusion construct<br> 28<br>
Ala Ala Ala Ser Gly Gly Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala<br>
5	10	15<br>
Glu Thr Asp Gin Val Glu Asp Glu Lys Ser Ala Leu Gin Thr Glu, lie<br>
20	25	30<br>
Ala Asn Leu Leu Lys Glu Lys Glu Lys Leu Glu Phe lie Leu Ala Ala<br>
35	40	45<br><br>
50	55	60<br>
He Leu Ala Ala His Gly Gly Cys Gly Gly Ser Ala Ala Ala<br>
65	70	75<br> 31<br> 44<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 31<br>
cctgggtggg ggcggccgct tctggtggtt gcggtggtct gacc	44<br> 32<br> 44<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 32<br>
ggtgggaatt caggaggtaa aaagatatcg ggtgtggggc ggcc	44<br> 33<br> 47<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 33<br>
ggtgggaatt caggaggtaa aaaacgatgg cttgcggtgg tctgacc	47<br> 34<br> 18<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 34<br>
gcttgcggtg gtctgacc	18<br> 35<br> 27<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br><br>
ccaccaagct tagcaaccac cgtgtgc	27<br> 36<br> 54<br> DMA<br> Artificial Sequence<br><br>  Description of Artificial Sequence:   Primer<br>   36<br>
ccaccaagct tgatatcccc acacccagcg gccgcagaac caccgcaacc accg      54<br> 37<br> 32<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>   37<br>
ccaccaagct taggcctccc acacccagcg gc	32<br>  38<br>  29<br>  DNA<br> Artificial  Sequence<br><br> Description of Artificial Sequence:   Primer<br>  38<br>
ggtgggaatt caggaggtaa aaaacgatg	29<br> 39<br> 32<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 39<br>
ggtgggaatt caggcctatg gctacaggct cc	32<br> 40<br> 27<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence;   Primer<br> 40<br>
ggtgggaatt catggctaca ggctccc	27<br><br> 41<br> 59<br> DHA<br> Artificial Se(3uence<br><br> Description of Artificial  Seguence:   Primer<br>  41<br>
gggtctagaa tggctacagg ctcccggacg tccctgctcc tggcttttgg cctgctctg  59<br> 42<br> 58<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  SeQuence:   Primer<br>  42<br>
cgcaggcctc ggcactgccc tcttgaagcc agggcaggca gagcaggcca aaagccag  58<br> 43<br> 402<br> DHA<br> Artificial Seguence<br><br> Description of Artificial Sequence: Modified bee venom phospholipase A2<br><br> CDS<br> (1)..(402)<br> 43<br>
ate ate tac cca ggt act etg tgg tgt ggt cac ggc aac aaa tct tct  48<br>
He He Tyr Pro Gly Thr Leu Trp Cys Gly His Gly Asn Lys Ser Ser<br>
15	10	15<br>
ggt ccg aac gaa etc ggc egc ttt aaa cac acc gac gca tgc tgt egc  96<br>
Gly Pro Asn Glu Leu Gly Arg Phe Lys His Thr Asp Ala Cya Cys Arg<br>
20	25	30<br>
ace cag gac atg tgt ccg gac gtc atg tct get ggt gaa tct aaa cac  144<br>
Thr Gin Asp Met Cys Pro Asp Val Met Ser Ala Gly Glu Ser Lys His<br>
35	40	45<br>
ggg tta act aac acc get tct cac acg cgt etc age tgc gac tgc gac  192<br>
Gly Leu Thr Asn Thr Ala Ser His Thr Arg Leu Ser Cys Asp Cys Asp<br>
50	5S	60<br>
gac aaa ttc tac gac tgc ctt aag aac tec gcc gat acc ate tct tct  240<br>
Asp Lys Phe Tyr Asp Cys Leu Lys Asn Ser Ala Asp Thr He Ser Ser<br>
65	70	75	80<br>
tac ttc gtt ggt aaa atg tat ttc aac ctg ate gat acc  aaa tgt tac  288<br>
Tyr Phe Val Gly Lys Met Tyr Phe Asn Leu He Asp Thr Lys Cys Tyr â <br>
85	90	95<br><br>
aaa cCg gaa cac ccg gta ace ggc tgc ggc gaa cgt ace gaa ggt cgc  336<br>
Lys Leu Glu His Pro Val Thr Gly Cys Gly Glu Arg Thr Glu Gly Arg<br>
100	105	110<br>
tgc ctg cac tac ace gtt gac aaa tct aaa ccg aaa gtt tac cag tgg  384<br>
Cys Leu His Tyr Thr Val Asp Lys Ser Lys Pro Lys Val Tyr Gin Trp<br>
115	120	125<br>
ttc gac ctg cgc aaa tac	402<br>
Phe Asp Leu Arg Lys Tyr 130<br> 44<br> 134<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence: Modified bee venom phospholipase A2<br> 44<br>
lie lie Tyr Pro Gly Thr Leu Trp Cys Gly His Gly Asn Lys Ser Ser<br>
15	10	15<br>
Gly Pro Asn Glu Leu Gly Arg Phe Lys His Thr Asp Ala Cys Cys Arg<br>
20	25	30<br>
Thr Gin Asp Met Cys Pro Asp Val Met Ser Ala Gly Glu Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asn Thr Ala Ser His Thr Arg Leu Ser Cys Asp Cys Asp<br>
50	55	60<br>
Asp Lys Phe Tyr Asp Cys Leu Lys Asn Ser Ala Asp Thr lie Ser Ser<br>
65	70	75	80<br>
Tyr Phe Val Gly Lys Met Tyr Phe Asn Leu lie Asp Thr Lys Cys Tyr<br>
85	90	95<br>
Lys Leu Glu His Pro Val Thr Gly Cys Gly Glu Arg Thr Glu Gly- Arg<br>
100	105	110<br>
Cys Leu His Tyr Thr Val Asp Lys Ser Lys Pro Lys Val Tyr Gin Trp<br>
115	120	125<br>
Phe Asp Leu Arg Lys Tyr 13 0<br> 45<br> 19<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>  45<br>
ccatcatcta cccaggtac	19<br><br> 46<br> 34<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 46<br>
cccacaccca gcggccgcgt atttgcgcag gtcg	34<br> 47<br> 36<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>   47<br>
cggtggttct gcggccgcta tcatctaccc aggtac	36<br> 48<br> 19<br> DKA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>  48<br>
ttagtatttg cgcaggtcg	19<br> 49<br> 18<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 49<br>
ccggctccat cggtgcag	18<br> 50<br> 36<br> DKA<br> Artificial Sequence<br><br> Description oÂ£ Artificial Sequence: Primer<br>   50<br>
accaccagaa gcggccgcag gggaaacaoa tctgcc	36<br> 51   35<br><br> DNA<br> Artificial  Sequence<br><br> Description of Artificial  Sequence:   Primer<br>   51<br>
cggtggttct gcggccgctg gctccatcgg tgcag	35<br> 52<br> 21<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Priiner<br>  52<br>
ttaaggggaa acacatctgc c	21<br> 53<br> 33<br> DHA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>  53<br>
actagtctag aatgagagtg aaggagaaat ate	33<br> 54<br> 42<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Priiner<br> 54<br>
tagcatgcta gcaccgaatt tatctaattc caataattct tg	42<br> 55<br> 51<br> DIiIA<br>
Â«:213&gt; Artificial Sequence<br><:220><br> Description of Artificial   Sequence:   Priiner<br><:400> 55<br>
gtagcaccca ccaaggcaaa gctgaaagct acccagctcg agaaactggc a        51<br> 56  48<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 56<br>
caaagctcct attcccactg ccagtttctc gagctgggta gctttcag	48<br> 57<br> 36<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 57<br>
ttcggtgcta gcggtggctg cggtggtctg  accgac	36<br>   5B<br>   37<br>  DMA<br> Artificial Sequence<br><br> I&gt;escxiptiDn of Artificial Sequence: Primer<br> 58<br>
gatgctgggc ccttaaccgc aaccaccgtg tgccgcc	37<br> 59<br> 46<br> PRT<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   JUN amino  acid sequence<br> 59<br>
Cys Gly Gly Arg lie Ala Arg Leu Glu Glu Lys Val Lys Thr Leu Lys<br>
15	10	15<br>
Ala Gin Asn Ser Glu Leu Ala Ser Thr Ala Asn Met Leu Arg Glu Gin<br>
20 .	25	30<br>
Val Ala Gin Leu Lys Gin Lys Val Met Asn His Val Gly Cys<br>
35	40	45<br> 60<br> 46<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence:  FOS amino acid sequence<br> 60<br>
Cys Gly Gly Leu Thr Asp Thr Leu Gin Ala Glu Thr Asp Gin Val Glu<br>
15	10	15<br><br>
Asp Glu Lys Ser Ala Leu Gin Thr Glu lie Ala Asn Leu Leu Lys Glu<br>
20	25	30<br>
Lys Glu Lys Leu Glu Phe lie Leu Ala Ala His Gly Gly Cys<br>
35	40	45<br> 61<br> 33<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 61<br>
ccggaattca tgtgcggtgg tcggatcgcc egg	33<br> 62<br> 39<br> DHA<br> Artificial Sequence<br><br> Description of Artificial Seguence: Primer<br> 62<br>
gtcgctaccc gcggctccgc aaccaacgtg gttcatgac	39<br>  63<br> 50<br> DHA<br> Artificial  Sequence<br><br> Description of Artificial Seguence; Primer<br> 63<br>
gttggttgcg gagccgcggg tagcgacatt gacccttata aagaatttgg	50<br> 64<br> 38<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 64<br>
cgcgtcccaa gcttctacgg aagcgttgat aggatagg	38<br> 65<br> 33<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br><br> 55<br>
ctagccgcgg gttgcggtgg tcggatcgcc egg	33<br>
, 66<br>
" 3 8<br> DHA<br> Artificial Segaence<br><br> Description of Artificial Sequence: Primer<br> 66<br>
cgcgtcccaa gcttttagca accaacgtgg ttcatgac	38<br> 67<br> 31<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 67<br>
ccggaattca tggacattga cccttataaa g	31<br>	6B<br>	45<br>	DNA<br>	Artificial Sequence<br><br> Description of Artificial Sequence; Primer<br> 68<br>
ccgaccaccg caacccgcgg ctagcggaag cgttgatagg atagg	45<br> 69<br> 47<br> DHA<br> Artificial Sequence<br><br> Description of Artificial Sequence; Primer<br> 69<br>
Ctaatggatc cggtgggggc tgcggtggtc ggatcgcccg gctcgag	47<br> 70<br> 3 9<br> DHA<br> Artificial Sequence<br><:220><br> Description of Artificial Sequence: Primer<br>  70<br>
gtcgctaccc gcggctccgc aaccaacgtg gttcatgac	39<br><br> 71<br> 31<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>  71<br>
ccggaattca tggacattga cccttataaa g	31<br> 72<br> 48<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Seq[uence:   Primer<br>   72<br>
ccgaccaccg cagcccccac cggatccatt agtacccacc caggtagc	48<br> 73<br> 45<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>   73<br>
gttggttgcg gagccgcggg tagcgaccta gtagtcagtt atgtc	45<br> 74<br> 3 8<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Primer<br> 74<br>
cgcgtcccaa gcttctacgg aagcgttgat aggatagg	38<br> 75<br> 33<br> DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>  75<br>
ctagccgcgg gttgcggtgg tcggatcgcc egg	33<br> 76  38<br><br>
i212&gt; DNA<br> Artificial Sequence<br><br> Description of Artificial  Sequence:   Primer<br>  76<br>
cgcgtcccaa gcttttagca accaacgtgg ttcatgac	38<br> 77<br> 3D<br> DNA<br> Artificial  Sequence<br><br> Description  of  Artificial  Sequence:   primer<br>  77<br>
ccggaattca tggccacact tttaaggagc	30<br> 78<br> 3B<br> DHA<br> Artificial Sequence<br><br> Description  of Artificial  Sequence:   Primer<br>  78<br>
cgcgtcccaa gcttttagca accaacgtgg ttcatgac	38<br> 79<br> 31<br> DHA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br> 79<br>
ccggaattca tggacattga cccttataaa g	31<br> 80<br> 51<br> DNA<br> Artificial Sequence<br><br> Description  oE Artificial  Sequence:   Primer<br> 80<br>
cctagagcca cctttgccac catcttctaa attagtaccc acccaggtag c        51<br> 81<br> 43<br> DHA<br> Artificial Sequence<br><br>
T<br><br> Description of Artificial Sequence: Primer<br> 81<br>
gaagatggtg gcaaaggtgg ctctagggac ctagtagtca gttatgtc	48<br> 82<br> 38<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>  82<br>
cgcgtcccaa gcttctaaac aacagtagtc tccggaag	38<br> 83<br> 3 6<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>  83<br>
gccgaattcc  tagcagctag caccgaattt atctaa	36<br>  84<br>
211&gt; 33<br>
-:212&gt; DKR<br>
-:213&gt; Artificial  Sequence<br>
-:220&gt;<br> Description of Artificial Sequence:   Primer<br>
400&gt;  84<br>
ggttaagtcg acatgagagt gaaggagaaa tat	33<br> 85<br>  30<br><:212> DNS<br><:213> Artificial Sequence<br><br><:223> Description of Artificial Sequence: Primer<br><:400> 85<br>
taaccgaatt caggaggtaa aaagatatgg	30<br><:210> 86<br> 35<br> DNA<br> Artificial Sequence<br>
220&gt;<br>
â¢:223&gt; Description of Artificial Sequence:   Primer<br><br>
"<:400> 86<br>
g-aagtaaagc ttttaaccac cgcaaccacc agaag	35<br> 87<br> 33<br> DMA<br> Artificial Sequence<br><br> Description of Artificial Sequence:   Primer<br>  87<br>
tcgaatgggc cctcstcttc gtgtgctagt cag	33<br> 88<br> 4<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence: Foa fusion construct<br> 88<br>
Glu Phe Arg Arg<br>
1<br> 89  183<br> PRT<br> Hepatitis B virus<br> 89<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro He<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin IjeM Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
13 0	135	140<br><br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	ISO<br>
Pro Ser Pro Arg Axg Arg Arg Ser Gin Sex Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Gly Ser Gin Cys 180<br> 90<br> 1B3<br> PRT<br> Hepatitis B virus<br> 90<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
1	5	10      .	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35 .	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro Thr<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Thr Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Cys Val He Arg Arg Arg Gly Axg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160.<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Gly Ser Gin Cys 180<br> 91<br> 212<br> PRT<br> Hepatitis B virus<br> 91 ,<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys pro Thr<br>
1	5	10    â       â       15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br><br>
ZV	25	3D<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
B5	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro lie Ser Arg Asp<br>
IDD	105	110<br>
Leu Val Val Ser Tyr Val Asn	Thr Asn Met	Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser	Cys Leu Thr	Phe Gly Arg Glu Thr Val<br>
130                135	140<br>
He Glu Tyr  Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val arg Arg Arg Gly ftrg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Jirg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 92<br> 212<br>
â e212&gt; PRT<br><:213> Hepatitis B virus<br> 92<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cyg Pro Thr<br>
1	5	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro TVr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Asn Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Le-u Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro He Ser Arg Asp<br><br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 93<br> 183<br> PRT<br> Hepatitis B virus<br>
Met Asp He Asp Pro Tyr Lys Glu Plie Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Thr Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Txp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Cys Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys<br><br>
180<br>	94<br>	212<br>	PRT<br>	Hepatitis B virus<br> 94<br>
Met Gin Leu Phe His Leu Cys u lie lie Ser Cys Ser Cys Pro Thx<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro Val Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr phe Gly Arg Glu Thr Val<br>
. 130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
, 165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br>
â =210&gt; 95<br> 212<br> Hepatitis B virus<br>   95<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys  Leu Cys  Leu Gly Trp Leu Trp Asp Met Asp   He<br>
20	25	30<br><br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Tbr Ala Ser<br>
50	55	GO<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
S5	7D	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp Leu Met Thr<br>
65	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro Val Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys Pbe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	150<br>
Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 96<br> 212<br> PRT<br> Hepatitis B virus<br> 96<br>
Met Gin Leu phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Rla Thr Val Glu Leu Leu Set Phe Leu<br>
35	40	45<br>
pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro Gin<br>
65	70	75	BO<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Txp Val Gly Gly Asn Leu Glu Asp Pro lie Ser Arg Asp<br><br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Ai-g Thr Pro Pro Ala<br>
145	15D	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro* He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 97<br> 212<br> PRT<br> Hepatitis B virus<br> 97<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu" Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro Hig<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thx Pro Pro Ala<br>
145	150	155	150<br>
Tyr Lys Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br><br>
â 1-BU	1.S5	ISO<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Gly Ser Gin Cys 210<br> 98<br> PRT<br> Hepatitis B virus<br>
Met Asp He Asp Pro Tyx Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
1	5	10	15 ,<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Phe Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala Ha Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asii Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Asp Thr Val He Glu Tyr Leu Val Ser Phe Gly ,Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Ser Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Cys Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 99<br> 183<br> PRT<br> Hepatitis B virus<br> 99<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp 20<br><br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
55	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 100  212  PRT  Hepatitis B virus<br> 100<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Pha Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
55	70	75	80<br>
His Thr Ala Leu Arg His Ala He Leu Cys Trp Gly Asp Leu Arg Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro He Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
-   . -7<br><br>
130	135	140<br>
He Glu Tyr Leu Val Ser Pbe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
155	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
190	IBS	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 101  212  PRT  Hepatitis B virus<br>
Ket Gla Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Asp Het Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Phe Arg Asp Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Ala Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Het Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Gin Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Cys<br>
155	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br><br>
Glu Ser Gin Cys 210<br> 102  183  PRT  Artificial Sequence<br>
â t:220&gt;<br>
â¢:223&gt; Description of Artificial Sequence: synthetic human Hepatitus B construct<br>
â¢:400&gt; 102<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu<br>
50	55	50<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
S5	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Tip Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val Leu Glu Tyx Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	no	i"?5<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 103  212  PRT  Hepatitis B virus<br> 103<br>
Met Gin Leu Phe His  Leu Cys  Leu He  He  Ser Cys  Ser Cys   Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyx Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser  Phe Leu<br><br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp Leu Met Ser<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro lie Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp phe His lie Ser Cys Leu Thr Phe	Gly Arg Glu	Thjr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp lie	Arg Thr Pro	pro Ala<br>
145                150                155	160<br>
Tyr Arg Pro pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 104<br> 183  PRT  Hepatitis B virus<br> 104<br>
Met Asp He Asp Pro Tyr Lyd Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
1	5	,       iO	15<br>
Ser Phe Leu Fro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr, Trp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
65	70	75	BO<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br><br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	150<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys<br>
leo<br> 105 
 105<br>
Met Asp lie Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
65	7D	75	80<br>
Ser Arg Asp Leu Val val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	9S<br>
Phe Arg Gin Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val lie Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro ser Pro Arg .Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 106  133<br> PRT<br> Hepatitis B virus<br> 106<br><br>
Met Asp lie Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu<br>
50	55	â¬0<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Ala Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val lie Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Thr Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Axg Arg Arg Ser<br>
155	170	175<br>
Gin Ser Arg Glu Ser Gin Cys ISO<br> 107  212  PRT  Hepatitis B virus<br> 107<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser tys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	-25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe phe Pro Ser Vsl Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	BO<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br><br>
Leu Val Val Ser Tyr Val Asn Thr Asn MeC Gly Leu Lys Phe Arg Gin<br>
115	12 0	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	150<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> lOa  212  PRT  Hepatitis B virus<br> 108<br>
Met Gin Leu Phe His Leu Cys Leu He He Ssr Cys Ser Cys Pro Thr<br>
1	-5	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
2D	25	3D<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr VaJ Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	50<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Atg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br><br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br>
210&gt; 109 -:211&gt; 212 <:212> PRT â i213&gt; Hepatitis B virus<br> 109<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Thr Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyx Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe- Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr  Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ale Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Aan Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	136	140<br>
He Glu Tyr Leu Val Ala Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 110  212  PRT<br> Hepatitis B virus<br><br><iuu> 111)<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Phe Glu Cys Ser Glu His Cys Ser Pro His<br>
65     .	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro lie Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asm Thr Asn Met Gly Leu Lys phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu ser Gin Cys 210<br> 111  212  PRT  Hepatitis B virus<br><br> UNSURE<br> [28)<br> May be any amino acid.<br> 111<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
1	5	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Xaa Asp Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br><br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	S5	60<br>
Ala Leu Tyr Ai-g Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp Leu He Thr<br>
85	90	95<br>
Leu Ser Thr Trp Val Gly Gly Asn Leu Glu Asp Pro Thr Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Pbe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Thr Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 112  212  PRT  Hepatitis B virus<br> 112<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Asn Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Tip Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser â Xyr Val ASn Thr Asn Met Gly Leu Lys Phe Arg Gin<br><br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
lis	ISO	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 113  212  PRT  Hepatitis B virus<br> 113<br>
Met Gin. Leu Phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thx<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	"120	125<br>
Leu Leu Trp Phe His He Cys Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br><br>
195	200	205<br>
Glu Ser Gin Cys 210<br><:210> 114 <:211> 212  PRT â i213&gt; Hepatitis B virus<br> 1X4<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Ket Gly Leu Lys Phe Tirg Gin<br>
115	120	125<br>
Leu Leu Trp phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin ser Arg<br>
195	200	205<br>
Glu Pro Gin Cys 210<br> 115  212  PRT  Hepatitis B virus<br> 115<br>
Met Gin Leu Phe His J.eu Cys Leu lie He Ser Cys Ser Cys PÂ£0 Thx<br>
1	5	10   -	15<br><br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala TJir Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Ser Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
S5	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 116  212  PRT  Hepatitis B virus<br> 116<br>
Met Gin Leu Phe His Leu Cys Leu He lie Ser Cys Ser cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br><br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Leu Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg	Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Axg Arg Arg Arg Ser Gin Ser	Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> in<br> 212  PRT  Hepatitis B virus<br> 117<br>
Met Gin Leu Phe His Leu Cys Leu lie He Ser Cys Ser Cys Pro Thr<br>
1	5	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Dys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp, Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Lys Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br><br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
leo	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 118  212  PRT  Hepatitis B virus<br> il8<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ala<br>
50	â    55	60<br>
Ala LeU Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
55	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Thr Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130 â 	135	140<br>
Leu Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	1*75<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 119  183<br><br> PRT<br> Hepatitis B virus<br> 119<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Ser Met Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Tyr Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Thr Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
5D	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asn Leu Gin Asp Pro Thr<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His Val Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val Val Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Gin Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Cys Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ssr<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 120  183  PRT  Hepatitis B virus<br> 120<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg His Val Phe Leu Cys Trp Gly Asp<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro Thr<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br><br>
85	90	-  95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 121  212  PUT  Hepatitis B virus<br> 121<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser CyS Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu Hia Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp Leu Thr Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Sar Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	no	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br><br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 122  212  PRT  Hepatitis B virus<br> 122<br>
Met Gin Leu Phe His Leu Cys Leu lie He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu He Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
lao	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 123  183  PRT  hepatitis B virus<br>   123<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br><br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Tfar Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp<br>
50	55	50<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Val<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val lie Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Ala Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br> 124  212  PRT  Hepatitis B virus<br> 124<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	00<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp Leu Met Asn<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp pro Val Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Gly Tyr Val Asn Thr Thr Val Gly Leu Lys Phe Arg Gin<br><br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
13D	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	no	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 125  183  PRT  Hepatitis B virus<br> 125<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	â   25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp<br>
50	55	eo<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala<br>
65	70 .	75	80<br>
Ser Arg" Asp Leu Val Val Ser Tyr Val Asn Thr Asn Mat Gly Leu Lys<br>
35	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Tir Phe Gly Arg<br>
100	IDS	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Thr Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys 180<br><br> 126  212  PRT  Hepatitis B virus<br> 126<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Ala Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	TO	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
lie Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 127  212  PRT  Hepatitis B virus<br> 127<br>
Met Gin Leu Phe His Leu Cys Leu He He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Tip Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40 â 	45<br><br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp Leu Met Thr<br>
85	90	95<br>
Leu Ala Thi  Trp Val Gly Val Asn Leu Glu Asp Pro Ala Thr Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	-  175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 128  212  PET  Hepatitis B virus<br> 128<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	BO<br>
His Thr Ala Leu Arg Gin Arg lie Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys Phe Arg Gin<br><br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
lie Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Thr Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br>	129<br>	212<br>	PRT<br>	Hepatitis B virus<br> 129<br>
Met Gin Leu Phe His Leu Cys Leu Val He Ser Cys Ser Cys Pro Thr<br>
15	10	15<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp He<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Jeu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ala<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu Leu Met Thr<br>
85	90	95<br>
Leu Ala Thr Trp Val Gly Asn Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Asn Tyr Val Asn Thr Asn Met Gly Leu Lys He Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Giy Arg Glu Thr Val<br>
130	135	140<br>
Leu Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Pro Ala<br>
145	150	155	160<br>
Tyr Arg Pro pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Glh Ser Arg<br><br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 130  212  PRT  Hepatitis B virus<br> 130<br>
Met Gin Leu Phe His Leu Cys Leu lie lie Ser cys Ser Cys Pro Thr<br>
15	10	IS<br>
Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Gly Met Asp lie<br>
20	25	30<br>
Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu Ser Phe Leu<br>
35	40	45<br>
Pro Ser Ala Phe Phe Pro Ser Val Arg Asp Leu Lsu Asp Thr Ala Ser<br>
50	55	60<br>
Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser Pro His<br>
65	70	75	80<br>
His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Asp Leu Met Thr<br>
85	90	95<br>
Lau Ala Thr Trp Val Gly Val Asn Leu Glu Asp Pro Ala Ser Arg Asp<br>
100	105	110<br>
Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lya Phe Arg Gin<br>
115	120	125<br>
Leu Leu Trp Phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val<br>
130	135	140<br>
He Glu Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala<br>
145	150 ,	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr<br>
165	170	175<br>
Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr Pro Ser -Pro<br>
180	185	190<br>
Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
195	200	205<br>
Glu Ser Gin Cys 210<br> 131  183  PRT  Hepatitis B virus<br>   131<br>
Met Asp  He Asp  Pro Tyr Lys Glu Phe Gly Ala Thr Val  Glu Leu Leu<br>
15	10	15<br><br>
I<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ala Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Asn Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Asn Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
lie Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val Leu Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Glu Ser Gin Cys IBO<br> 132  183  PRT  Hepatitis B virus<br> 132<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Glu<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Gly Asn Leu Glu Asp Pro He<br>
65	70 â 	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val He Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br><br>
115	120	125<br>
Pro Pro Ala Tyr Arg pro Pro Asn Ala Pro lie Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Cys Val Val Arg Arg Arg Gly Arg Ser Pro Arg Arg Arg Thr<br>
145	150	155	160<br>
Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser<br>
165	170	175<br>
Gin Ser Arg Gly Ser Gin Cys 180<br> 133<br> 3221<br> DNA<br> Hepatitis B virus<br><br> CDS<br> (1901)..(2458)<br> 133<br>
ttccactgcc ttccaccaag ctctgcagga ccccagagtc aggggtctgt attttcctgo 60<br>
tggtggctcc agttcaggaa cagtaaaccc tgctccgaat attgcctctc scatctcgtc 120<br>
aatctccgcg aggactgggg accctgtgac gaacatggag aacatcacat caggattcct 180<br>
sggacccctg ctcgtgttac aggcggggtt tttattgttg acaagaatcc tcacaatacc 240<br>
gcagagtcta gactcgtggt ggactCctct caattttata gggggatcac ccgtgtgtct 300<br>
tggccaaaat tcgcagtccc caacctccaa tcactcacca acctcctgtc ctccaatttg 360<br>
tcctggttat cgctggatgt gtctgcggcg ttttatcata ttcotcttca toctgctgct 420<br>
atgcctcatc ttcttattgg ttcttctgga ttatcaaggt atgttgcccg tttgtcctct 480<br>
aattccagga tcaacaacaa ccagtacggg accatgcaaa acctgcacga ctcctgctoa 540<br>
aggcaactct atgtttccct catgttgctg tacaaaacct acggttggaa attgcacctg 600<br>
tattcccatc ccatcgtcct gggctttcgc aaaataccta tgggagtggg cctcagtccg 660<br>
tttctcttgg ctcagtttac tagtgdcatt tgttcagtgg ttcgtagggc tttcccccac 720<br>
tgtttggctt tcagctatat ggatgatgtg gtattggggg ccaagtctgt acagcatcgt 780<br>
gagtcccttt ataccgctgt taccaatttt cttttgtctc tgggtataca tttaaaccct 840<br>
aacaaaacaa aaagatgggg ttattcccta aacttcatgg gttacataat tggaagttgg 900<br>
ggaacattgc cacaggatca tattgtacaa aagatcaaac actgttttag aaaacttcct 960<br>
gtt.aacaggc ctattgattg gaaagtatgt caaagaattg tgggtctttt gggctttgct 1020<br>
gctccattta cacaatgtgg atatcctgcc ttaatgcctt tgtatgcatg tatacaggct 1080<br>
aaacaggctt tcactttctc gccaecttac aaggcctttc taagtaaaca gtacatgaac 1140<br>
ctttaccccg ttgctcggca acggcctggt ctgtgccaag tgtttgctga cgcaaecccc 1200<br><br>
T<br>
actggttggg gcttggccat aggccatcag cgcatgagtg gaacctttgt ggctcctctg 1260<br>
ccgatccata ctgcggaact cctagccgct tgtattgctc gcagccggtc tggagcaaag 1320<br>
CtcaCcggaa ctgacaattc tgtcgtcctc tcgcggaaat atacatcgtt tccatggctg 13S0<br>
ctaggctgta ctgccaactg gatccttcgc gggacgtcct ttgtttacgt cccgtcggcg 1440<br>
ctgaatcccg cggacgaccc ctctcggggc cgcttgggac tctatcgtcc ccttctccgt 1500<br>
ctgccgttcc agccgaccac ggggcgcacc tctctttacg cggtctcccc gtctgtgcct 1560<br>
tctcstctgc cggtccgtgt gcacttcgct tcacctctgc acgttgcatg gagaccaccg 1620<br>
tgaacgccca tcagatcctg cccaaggtct tacataagag gactcttgga ctcccagcaa 1680<br>
tgtcaacgac cgaccttgag gcctacttca aagactgtgt gtttaaggac tgggaggagc 1740<br>
tgggggagga gattaggtta aaggtctttg tattaggagg ctgtaggcat aaattggtct 1800<br>
gcgcaccagc accatgcaac tttttcacct ctgcctaatc atctcttgta catgtcccac 1860<br>
tgttcaagcc tccaagctgt gccttgggtg gctttggggc atg gac att gac cct  1915<br>
Met Asp lie Asp Pro<br>
1	5<br>
tat aaa gaa ttt gga get act gtg gag tta etc teg ttt ttg cct tct  1963<br>
Tyr Lys Glu Phe Gly Ala Thr Val Glu Iisu Leu Sex Phe Leu Pro Ser<br>
10	15	20<br>
gac ttc ttt cct tec gtc aga gat etc eta gac ace gee tea get etg  2011<br>
Asp phe Phe Pro Ser Val Arg Asp Leu Leu Asp Thr Ala Ser Ala Leu<br>
25	30	35<br>
tat cga gaa gcc tta gag tct ccC gag cat tge tea cct cae cat act  2 059<br>
Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys Ser pro His His Thr<br>
40	45	50<br>
gea etc agg caa gcc att etc tge tgg ggg gaa ttg atg act eta get  2107<br>
Ala Leu Arg Gin Ala lie Leu CyS Tip Gly Glu Leu Met Thr Leu Ala<br>
55	60	65<br>
ace_ tgg gtg ggt aat aat ttg gaa gat cca gea tec agg gat eta gta  2155<br>
Thr Trp Val Gly Asn Asn Leu Glu Asp Pro Ala Ser Arg Asp Ieu Val<br>
70	75	80	85<br>
gtc aat tat gtt aat act aac atg ggt tta aag ate agg caa eta ttg  2203<br>
Val Asn Tyr Val Asn Thr Asn Met Gly Leu Lys He Arg Gin Leu Leu<br>
90	95	100<br>
tgg ttt cat ata tct tgc ctt act ttt gga aga gag act gta ctt gaa  2251<br>
Trp phe His lie Ser Cys Leu Thr Phe Gly Arg Glu Thr Val Leu Glu<br>
105	110	115<br>
tat ttg gtc tct tte gga gtg tgg att egc act cct cca gcc tat aga  2299<br>
Tyr Leu Val Ser Phe Gly Val Trp lie Arg Thr Pro Pro Ala Tyr Arg<br>
120	125	130<br>
cca cca aat gcc ect ate tta tea aca ctt ceg gaa act act gtt gtt  2347<br>
Pro pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu Thr Thr Val Val<br>
135	140	145<br><br>
J<br>
dga cga egg gac cga ggc agg tec cct aga aga aga act ccc teg cct  2395<br>
Arg Arg Arg Asp Axg Gly Arg Ser Pro Arg Rxg Arg Thx Pro Ser Pro<br>
150	155	160	165<br>
cgc aga cgc aga tct caa teg ccg cgt cgc aga aga tct caa tct egg  2443<br>
Axg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg Arg Ser Gin Ser Arg<br>
170	17 5	leO<br>
gaa tct caa tgt tag tattccttgg aeteataagg tgggaaactt tactgggctt  2498 Glu Ser Gin Cys 185<br>
tattcetcta eagtacctat ctttaatcct gaatggcaaa ctecttectt tcctaagatt 2558<br>
eatttacaag aggacattat tgataggtgt caacaatttg tgggeectct caetgtaaat 2618<br>
gaaaagagaa gattgaaatt aattatgcct gctagattct atcctaccca cactaaatat 2676<br>
ttgceettag acaaaggaat taaaccttat tatccagatc aggtagttaa tcattacttc 2738 caaaecagac attatttaea tacCctttgg aaggctggta ttctatataa gagggaaacc 2798<br>
acacgtagcg catcattttg cgggtcacca tattcttggg aacaagagct acagcstggg 2858<br>
aggttggtca ttaaaacctc gcaaaggcat ggggacgaat ctttetgttc ccaaccctct 2918<br>
gggattcttt ccegatcate agttggaeec tgcattegga gccaactcaa aeaatccaga 2978<br>
ttgggacttc aaccccatea aggaccactg gccagcagcc aaccaggtag gagtgggagc 3038<br>
attcgggcca gggctcaecc ctcdacacgg cggtattttg gggtggagce ctcaggctca 3098<br>
gggcatattg accacagtgt caacaattcc tcctcctgcc tccaccaatc ggcagtcagg 3158<br>
aaggcagcet acteccatct ctccacctct aagagacagt catectcagg ccatgcagtg 3216<br>
gaa	3221<br> 134  185  PRT  Hepatitis E vims<br> 134<br>
Met Asp lie Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
1	5	10	15<br>
Ser Phe Lsu Pro Ser Asp Phe Ptie Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ser Ala Leu Tyr Arg Glu Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala lie Leu Cys Trp Gly Glu<br>
50	55	5Q<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Asn Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Asn Tyr Val Asn Thr Asn Met Gly Leu Lys<br>
85	90	95<br>
He Axg Gin, Leu Leu Trp Piie His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val Leu Glu Tyr Leu Val ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyi: Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val Arg Arg Arg Asp Arg Gly Arg Ser Pro Arg Arg<br><br>
i
Arg Thr Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg<br>
165	170	175<br>
Arg Ser Gin Ser Arg Glu Ser Gin Cys<br>
leO	IBS<br> 135  IBS  PRT  Woodchuck hepatitis B virus<br> 135<br>
Met Asp lie A3P Pro Tyr Lys Glu Phe Gly Ser Ser Tyr Gin Leu Leu<br>
15	10	15<br>
Asn Phe Leu Pro Leu Asp Phe Phe Pro Asp Leu Asn Ala Leu Val Asp<br>
20	25	30<br>
Thr Ala Thr Ala Leu Tyr Glu Glu Glu Leu Thr Gly Arg Glu His Cys<br>
35	40	45<br>
ser Pro His His Thr Ala He Arg Gin Ala Leu Val Cys Trp Asp Glu<br>
50	55	60<br>
Leu Thr Lys Leu lie Ala Trp Met Ser Ser Asn He Thr Ser Glu Gin<br>
65	70	75	80<br>
Val Arg Thr He He Val Asn His Val Asn Asp Thr Trp Gly Leu Lys<br>
85	90	95<br>
Val Arg Gin Ser Leu Trp Phe His Leu Ser Cys Leu Thr Phe Gly Gin<br>
100	105	110<br>
His Thr Val Gin Glu Phe Leu Val ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Ala Pro Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu His Thr Val He ZLrg Arg Arg Gly Gly Ala Arg Ala Ser Arg Ser<br>
145	150	155	160<br>
Pro Arg Arg Arg Thr Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro<br>
165	170	175<br>
Arg Arg Arg Arg Ser Gin Ser Pro Ser Thr Asn Cys<br>
180	185<br> 136  217  PRT  Ground squirrel hepatitis virus<br> 136<br>
Ket  Tyr Leu Phe His Leu Cys Leu val Phe Ala Cys Val Pro Cys Pro<br>
15	10	15<br>
Thr Val Gin Ala Ser Lys Leu Cys Leu Gly Trp Leu Trp Asp Met Asp<br>
20	25	30<br>
He Asp Pro Tyr Lys Glu Phe Gly Ser Ser Tyr Gin Leu Leu Asn Phe<br>
35	40	45<br>
Leu Pro Leu Asp Phe Phe Pro Asp Leu Asn Ala Leu Val Asp Thr Ala<br>
50	55	60<br>
Ala Ala Leu Tyr Glu Glu Glu Leu Thr Gly Arg Glu His Cys Ser Pro<br>
65	70	75	80<br>
His His Thr Ala He Arg Gin Ala Leu Val Cys Trp Glu Glu Leu Thr<br>
85	90	95<br>
Arg Leu He Thr Trp Met Ser Glu Asn Thr Thr Glu Glu Val Arg Arg<br>
100	105	110<br>
lie He Val Asp His Val Asn Asn Thr Trp Gly Leu Lys Val Arg Gin<br>
115	120	125<br>
Thr Leu Trp Phe His Leu Ser Cys Leu Thr Phe Gly Gin His Thr Val<br>
130        -       135	140<br>
Gin Glu Phe Leu Val Ser Phe Gly Val Trp He Arg Thr Pro Ala Pro<br>
145	150	155	160<br>
Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro Glu His Thr<br>
165	170	175<br>
Val He Arg Arg Arg Gly Gly Ser Arg Ala Ala Arg Ser Pro Arg Arg<br>
180	185	190<br>
Arg Thr Pro Ser Pro Arg Arg Arg Arg Ser Gin Ser Pro Arg Arg Arg<br>
195	200	2D5<br>
Arg Ser Glu Ser Pro Ala Ser Asn Cys<br>
210	215<br> 137<br> 262  PRT  Snow Goose Hepatitis B Virus<br> 137<br>
Met Asp Vai Asn Ala Ser Arg Ala Leu Ala Asn Val Tyr Asp Le\i Pro<br>
15	10	15<br>
Asp Asp Phe Phe Pro Lys He Glu Asp Leu Val Arg Asp Ala Lys Asp<br>
20	25	30<br>
Ala Leu Glu Pro Tyr Trp Lys Ser Asp Ser He Lys Lys His Val Leu<br>
35	40	45<br>
He Ala Thr His Phe Val Asp Leu He Glu Asp Phe Trp Gin Thr Thr<br>
50	55	60<br>
Gin Gly Met His Glu He Ala Glu Ala He Arg Ala Val He Pro pro<br>
55	70	75	80<br>
Thr Thr Ala Pro Val Pro Ser Gly Tyr Leu He Gin His Asp Glu Ala<br>
85	90	95<br>
Glu Glu He Pro Leu Gly Asp Leu Phe Lys Glu Gin Glu Glu Arg lie<br>
100	105	110<br>
Val Ser Phe Gin Pro Asp Tyr Pro He Thr Ala Arg He His Ala His<br><br>
;       115	120	125<br>
Leu Lys Ala Tyr Ala Lys He Asn Glu Glu Ser Leu Asp Arg Ala Arg<br>
130	135	140<br>
Arg Leu Leu Trp Trp His Tyr Asn Cys Leu Leu Trp Gly Glu Ala Thr<br>
145	150	155	160<br>
Val Thr Asn Tyr lie Ser Arg Deu Arg Thr Trp Leu Ser Thr Pro Glu<br>
165	170	175<br>
Lys Tyr Arg Gly Arg Asp Ala Pro Thr He Glu Ala He Thr Arg Pro<br>
180	185	190<br>
He Gin Val Ala Gin Gly Gly Arg Lys Thr Ser Thr Ala Thr Arg Lys<br>
195	200	205<br>
Pro Arg Gly Leu Glu Pro Arg Arg Arg Lys Val Lys Thr Thr Val Val<br>
210	215	220<br>
Tyr Gly Arg Arg Arg Ser Lys Ser Arg Glu Arg Arg Ala Ser Ser Pro<br>
225	230	235	240<br>
Gin Arg Ala Gly Ser Pro Leu Pro Arg Ser Ser Ser Ser His His Arg<br>
245	250	255<br>
Ser Pro Ser Pro Arg Lys 260<br>
â¢:210&gt;	138<br>
-:211&gt;	305<br>
â¢:212&gt;	PRT<br>	Duck hepatitis B virus<br> 138<br>
Met Trp Asp Leu Arg Leu His Pro Ser Pro Phe Gly Ala Ala Cys Gin<br>
15	10	15<br>
Gly He Phe Thr Ser Ser Leu Leu Leu Phe Leu Val Thr Val Pro Leu<br>
20	25	30<br>
Val Cys Thr He Val Tyr Asp Ser Cys Leu Cys Met Asp He Asn Ala<br>
35	40	45<br>
Ser Arg Ala Leu Ala Asn Val Tyr Asp Leu pro Asp Asp Phe Phe Pro<br>
50	55	60<br>
Lys He Asp Asp Leu Val Arg ASP Ala Lys Asp Ala Leu Glu Pro Tyr<br>
65	70	75	80<br>
Trp Arg Asn Asp Ser He Lys Lys His Val Leu He Ala Thr His Phe<br>
85	90	95<br>
Val Asp Leu He Glu Asp Phe Trp Gin Thr Thr Gin Gly Met His Glu<br>
100	105	110<br>
He Ala Glu Ala Leu Arg Ala He He Pro Ma Thr Thr Ala Pro Val<br>
115	120	125<br>
, pro Gin Gly Phe Leu Val Gin His Glu Glu Ala Glu Glu lie Pro Leu<br>
130	135	140<br>
Gly Glu Leu Phe Arg Tyr Gin Glu Glu Arg Leu Thr Asn Phe Gin Pro<br><br>
1<br>
i5	150	155	150<br>
Asp Tyr Pro Val Thr Ala Arg He His Ala His Leu Lys Ala Tyr Ala<br>
155	170	175<br>
Lys He Asn Glu Glu Ser Leu Asp Arg Ala Arg Arg Leu Leu Trp Trp<br>
180	185	190<br>
His Tyr Asn Cys Leu Leu Trp Gly Glu Pro Asn Val Tbr Asn Tyr He<br>
195	200	205<br>
Ser Arg Leu Arg Thr Trp Leu Ser Thr Pro Glu Lys Tyr Arg Gly Lys<br>
210	215	220<br>
Asp Ala Pro Thr He Glu Ala He Thr Arg Pro He Gin Val Ala Gin<br>
225	230	235	240<br>
Gly Gly Arg Asn Lys Thr Gin Gly Val	Arg Lys Ser Arg Gly Leu Glu<br>
245	250	255<br>
Pro Arg Arg Arg Arg Val Lys Thr Thr	He Val Tyr Gly Arg Arg Arg<br>
260                265	270<br>
Ser Lys Ser Arg Glu Arg Arg Ala Pro Thr Pro Gin Arg Ala Gly Ser<br>
275	260	285<br>
Pro Leu Pro Arg Thr Ser Arg Asp His His Arg Ser Pro Ser Pro Arg<br>
290	295	300<br>
Glu 305<br> 139  212  PRT  Haemophilus influenzae<br> 139<br>
Met Lys Lys Thr Leu Leu Gly Ser Leu He Leu Leu Ala Phe Ala Gly<br>
15	ID	15<br>
Asn Val Gin Ala Ala Ala Asn Ala Asp Thr Ser Gly Thr Val Thr Phe<br>
20	25	30<br>
Phe Gly Lys Val Val Glu Asn Thr Cys Gin Val Asn Gin Asp Ser Glu<br>
35	40	45<br>
Tyr Glu Cya Asn Leu Asn Asp Val Gly Lys Asn His Leu Ser Gin Gin<br>
50	55	60<br>
Gly Tyr Thr Ala Met Gin Thr Pro Phe Thr He Thr Leu Glu Asn Cys<br>
65	70	75	80<br>
Asn Val Thr Thr Thr Asn Asn Lys Pro Lys Ala Thr Lys Val Gly Val<br>
85	90	95<br>
Tyr Phe Tyr Ser Trp Glu He Ala Asp Lys Asp Asn Lys Tyr Thr Leu<br>
lOQ	105	110<br>
Lys Asn He Lys Glu Asn Thr Gly Thr Asn Asp Ser Ala Asn Lys Val<br>
115	120	125<br>
Asn He Gin Leu Leu Glu Asp Asn Gly Thr Ala Glu He Lys Val Val<br><br>
130	135	140<br>
Gly Lys Thr Thr Thr Asp Phe Thr Ser Glu Asn His Asn Gly Ala Gly<br>
145	150	155	160<br>
Ala Asp Pro Val Ala Thr Asn Lys His lie Ser Ser Leu Thr Pro Leu<br>
165	170	175<br>
Asn Asn ijln Asn "Ser lie Asn Leu His Tyr lie Ala Gin Tyr Tyr Ala<br>
180	185	190<br>
Thr Gly Val Ala Glu Ala Gly Lys Val Pro Ser Ser Val Asn Ser Gin<br>
195	200	205<br>
lie Ala Tyr Glu 210<br> 140  139  PRT  Pseudoinonas stutzeri<br> 140<br>
Met Lys Ala Gin Met Gin Lys Gly Phe Thr Leu lie Glu Leu Met lie<br>
15	10	15<br>
Val Val Ala lie lie Gly lie Leu Ala Ala lie Ala Leu Pro Ala Tyr<br>
20	25	30<br>
Gin Asp Tyr Thr Val Arg Ser Asn Ala Ala Ala Ala Leu Ala Glu lie<br>
35	40	45<br>
Thr Pro Gly Lys lie Gly Phe Glu Gin Ala lie Asn Glu Gly Lys Thr<br>
50	55	60<br>
Pro Ser Leu Thr Ser Thr Asp Glu Gly Tyr lie Gly lie Thr Asp Ser<br>
65	70	75	. BO<br>
Thr Ser Tyr Cys Asp Val Asp Leu Asp Thr Ala Ala Asp Gly His He<br>
85	90	95<br>
Glu Qys Thr Ala Lys Gly Gly Asn Ala Gly Lys Phe Asp Gly Lys Thr<br>
100	105	110<br>
He Thr Leu Asn Arg Thr Ala Asp Gly Glu Trp Ser Cys Ala Ser Thr<br>
115	120	125<br>
Leu Asp Ala Lys Tyr Lys Pro Gly Lys Cys Ser<br>
130	135<br> 141<br> 59<br> PRT<br> Caulobacter crescentus<br>   141<br>
Met Thr Lys  Phe Val Thr Arg Phe Leu Lys Asp Glu Ser Gly Ala Thr<br>
15	10	15<br>
Ala  He Glu Tyr Gly Leu He Val Ala Leu  He Ala Val Val He Val<br>
20	25	30<br><br>
Thr Ala Val Thr Thr Leu Gly Thr Asn Leu Arg Thr Ala Phe Thr Lys<br>
35	40	45<br>
Ala Gly Ala.Ala Val Ser Thr Ala Ala Gly Thr<br>
50	55<br> 142  173  PRT  Escherichia coli<br> 142<br>
Met Ala Val Val Ser Phe Gly Val Asn Ala Ala Pro Thr lie Pro Gin<br>
15	10	15<br>
Gly Gin Gly Lys Val Thr Phe Asn Gly Thr Val Val Asp Ala Pro Cys<br>
20	25	30<br>
Ser He Ser Gin Lys Ser Ala Asp Gin Ser He Asp Phe Gly Gin Leu<br>
35	40	45<br>
Ser Lys Ser Phe Leu Glu Ala Gly Gly Val Ser Lys Pro Met Asp Leu<br>
50	55	60<br>
Asp He Glu Leu Val Asn Cys Asp He Thr Ala Phe Lys Gly Gly Asn<br>
65	70	75	80<br>
Gly Ala Gin Lys Gly Thr Val Lys Leu Ala Phe Thr Gly Pro He Val<br>
85	90	95<br>
Asn Gly His Ser Asp Glu Lea Asp Thr Asn Gly Gly Thr Gly Thr Ala<br>
100	105	110<br>
lie Val Val Gin Gly Ala Gly Lys Asn Val Val Phe Asp Gly Ser Glu<br>
115	120	125<br>
Gly Asp Ala Asn Thr Leu Lys Asp Gly Glu Asn Val Leu His Tyr Thr<br>
130	135	140<br>
Ala Val Val Lys Lys Ser Ser Ala Val Gly Ala Ala Val Thr Glu Gly<br>
145	150	155	160<br>
Ala Phe Ser Ala Val Ala Asn Phe Asn Leu Thr Tyr Gin<br>
165	170<br> 143  173  PRT  Escherichia coli<br> 143<br>
Met Ala Val Val Ser phe Gly Val Asn Ala Ala Pro Thr He Pro Glji<br>
1	5	10	15<br>
Gly Gin Gly Lys Val Thr Phe Asn Gly Thr Val Val Asp Ala Pro Cys<br>
20	25	30<br>
Ser He Ser Gin Lys Ser Ala Asp Gin Ser He Asp Phe Gly Gin Leu<br>
35	40	45<br>
Ser Lys Ser Phe Leu Glu Ala Gly Gly Val Ser Lys Pro Met Asp Leu<br><br>
50	55	60<br>
Asp He Glu Leu Val Asn Cys Asp He Thr Ala Phe Lys Gly Gly Asn<br>
65	70	75	80<br>
Gly Ala Gin Lys Gly Thr Val Lys Leu Ala Phe Thr Gly Pro He Val<br>
85	90	95<br>
Asn Gly His Ser Asp Glu Leu Asp Thr Asn Gly Gly Thr Gly Thr Ala<br>
100	105	110<br>
He Val Val Gin Gly Ala Gly Lys Asn Val Val Phe Asp Gly Ser Glu<br>
115	120	125<br>
Gly Asp Ala Asn Thr Leu Lys Asp Gly Glu Asn Val Leu His Tyr  Thr<br>
130	135	140<br>
Ala Val Val Lys Lys Ser Ser Ala Val Gly Ala Ala Val Thr Glu Gly<br>
145	150	155	160<br>
Ala Phe Ser Ala Val Ala Asn Phe Asn Leu Thr Tyr Gin<br>
165	no<br>	144<br>	172<br>	PRT<br>	Escherichia coli<br> 144<br>
Met Ala Val Val Ser Phe Gly Val Asn Ala Ala Pro Thr Thr pro Gin<br>
15	10	15<br>
Gly Gin Gly Arg Val Thr Phe Asn Gly Thr Val Val Asp Ala Pro Cys<br>
20	25	30<br>
Ser He Ser Gin Lys Ser Ala Asp Gin Ser He Asp Phe Gly Gin Leu<br>
35	40	45<br>
Ser Lys Ser Phe Leu Ala Asn Asp Gly Gin Ser Lys Pro Met Asn Leu<br>
50	55	60<br>
Asp He Glu Leu Val Asn Cys Asp He Thr Ala Phe Lys Asn Gly Asn<br>
65.	70	75	80<br>
Ala Lys Thr Gly Ser Val Lys Leu Ala Phe Thr Gly Pro Thr Val Ser<br>
85	90	95<br>
Gly His Pro Ser Glu Leu Ala Thr Asn Gly Gly Pro Gly Thr Ala He<br>
100	105	110<br>
Met He Gin Ala Ala Gly Lys Asn val Pro Phe Asp Gly Thr Glu Gly<br>
115	120	125<br>
Asp Pro Asn Leu Leu Lys Asp Gly Asp Asn Val Leu His Tyr Thr Thr<br>
130	135	140<br>
Val Gly Lys Lys Ser Ser Asp Gly Asn Ala Gin He Thr Glu Gly Ala<br>
145	150	155	160<br>
Phe Ser Gly Val Ala Thr Phe Asn Leu Ser Tyr Gin<br>
165	170<br> 145  853<br><br> DNA<br> Escherichia coli<br><br> CDS<br> (281). . (829)<br> 145<br>
acgtttctgt ggctcgacgc atcttcctca ttcttctctc caaaaaccac ctcatgcaat 60<br>
ataaacatct ataaataaag ataacaaata gaatattaag ccaacaaata aactgaaaaa 120<br>
gtttgtccgc gatgctttac ctctatgagt caaaatggcc ccaatgtttc atcttttggg 160<br>
ggaaactgtg cagtgttggc agtcaaactc gttgacaaac aaagtgtaca gaacgactgc 240<br>
ccatgtcgat ttagaaatag ttttttgaaa ggaaagcagc atg aaa att aaa act  295<br>
Met Lys lie Lya Thr<br>
1	5<br>
ctg gca ate gtt gtt ctg teg get etg tec etc agt tct acg acg get  343<br>
Leu Ala He Val Val Leu Ser Rla Leu Ser Leu Ser Ser Thr Thr Ala<br>
10	15	20<br>
ctg gee get gcc acg acg gtt aat ggt ggg ace gtt cac ttt aaa ggg  391<br>
Leu Ala Ala Ala Thr Thr Val Asn Gly Gly Thr Val His Phe Lys Gly<br>
25	30	35<br>
gaa gtt gtt aae gcc get tgc gca gtt gat gca ggc tct gtt gat caa  439<br>
Glu Val Val Asn Ala Ala Cys Ala Val Asp Ala Gly Ser Val Asp Gin<br>
40	45	50<br>
ace gtt cag tta gga eag gtt cgt ace gca teg etg gca eag gaa gga  487<br>
Thr Val Gin Leu Gly Gin Val Arg Thr Ala Ser Leu Ala Gin Glu Gly<br>
55	60	65<br>
gca ace agt tct get gtc ggt ttt aac att cag ctg aat gat tgc gat  535<br>
Ala Thr Ser Ser Ala Val Gly phe Asn lie Gin Leu Aan Asp Cys Asp<br>
70	75	80	85<br>
ace aat gtt gca tct aaa gee get gtt gcc ttt tta ggt acg gcg att  583<br>
Thr Asn Val Ala Ser Lys Ala Ala Val Ala Phe Leu Gly Thx Ala He<br>
90	95	100<br>
gat gcg ggt cat ace aac gtt ctg get ctg cag agt tea get gcg ggt   631<br>
Asp Ala Gly His Thr Asn Val Leu Ala Leu Gin Ser Ser Ala Ala Gly<br>
105	110	115<br>
age gca aca aac gtt ggt gtg eag ate ctg gac aga acg ggt get gcg  679<br>
Ser Ala Thr Asn Val Gly Val Gin lie Leu Asp Arg Thr Gly Ala Ala<br>
120	125	130<br>
ctg acg ctg gat ggt gcg aca ttt agt tea gaa aca ace etg aat aac  727<br>
Leu Thr Leu Asp Gly Ala Thr phe Ser Ser Glu Thr Thr Leu Asn Asn<br>
135	140	145<br>
gga ace aat ace att ccg ttc cag gcg cgt tat ttt gca ace ggg gee  775<br>
Gly Thr Asn Thr He Pro Phe Gin Ala Arg Tyr Phe Ala Thr Gly Ala<br>
150	155	160	165<br>
gca ace ccg ggt get get aat gcg gat gcg ace tte aag gtt cag tat  823<br>
Ala Thr Pro Gly Ala Ala Asn Ala Asp Ala Thr phe Lys Val Gin Tyr<br>
170	175	180<br><br>
caa taa cctacctagg ttcagggscg ttca	853<br>
Gin<br>	146<br>	182<br>	PRT<br>	Escherichia coli<br> 146<br>
Met Lys lie Lys Thr Leu Ala He Val Val Leu Ser Ala Leu Ser Leu<br>
15	10	15<br>
Ser Ser Thr Thr Ala Leu Ala Ala Ala Thr Thr Val Asn Gly Gly Thr<br>
20	25	30<br>
Val His Phe Lys Gly Glu Val Val Asn Ala Ala Cys Ala Val Aap Ala<br>
35	40	45<br>
Gly Ser Val Asp Gin Thr Val Gin Leu Gly Gin Val Arg Thr Ala Ser<br>
50	55	60<br>
Leu Ala Gin Glu Gly Ala Thr Ser Ser Ala Val Gly Phe Asn lie Gin<br>
65	70	75	80<br>
Leu Asn Asp Cys Asp Thr Asn Val Ala Ser Lys Ala Ala Val Ala Phe<br>
85	90	95<br>
Leu Gly Thr Ala lie Asp Ala Gly His Thr Asn Val Leu Ala Leu Gin<br>
100	105	110<br>
Ser Ser Ala Ala Gly Ser Ala Thr Asn Val Gly Val Gin lie Leu Asp<br>
115	120	125<br>
Arg Thr Gly Ala Ala Leu Thr Leu Asp Gly Ala Thr Phe Ser Ser Glu<br>
130	135	140<br>
Thr Thr Leu Asn Asn Gly Thr Asn Thr He Pro Phe Gin Ala Arg Tyr<br>
145	150	155	160<br>
Phe Ala Thr Gly Ala Ala Thr Pro Gly Ala Ala Asn Ala Asp Ala Thr<br>
165	170	175<br>
Phe Lys Val Gin Tyr Gin IBO<br> 147<br> 11<br> PRT<br> Artificial Sequence<br><br> Description of Artificial Sequence: FLAG peptide<br> 147<br>
Cys Gly Gly Asp Tyr Lys Asp Asp Asp Asp Lys<br>
15	10<br> 148<br> 31<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: primer<br>   148<br>
ccggaattca tggacattga cccttstaaa g	31<br>   149<br><br>
f<br><:211> 37<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence; primer<br> 149<br>
gtgcagtatg gtgaggtgag gaatgctcag gagactc	37<br> 150<br> 37<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: primer<br> 150<br>
gsgtctcctg agcattcctc acctcaccat actgcac	37<br> 151<br> 33<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence: primer<br> 151<br>
cttccaaa.ag tgagggaaga aatgtgaaac cac	33<br> 152<br> 47<br> DNA<br> Artificial Sequence<br><br> Description oÂ£ Artificial Sequence: primer<br> 152<br>
cgcgtcccaa gcttctaaac aacagtagtc tccggaagcg ttgatag	47<br> 153<br> 33<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: primfer<br> 153<br>
gtggtttcac atttcttccc tcactcttgg aag	33<br> 154  281  PRT  Saccharoices cerevisise<br><br> 154<br>
Met Ser Glu Tyr Gin Pro Ser Leu Phe Ala Leu Asn Pro Met Gly Phe<br>
15	10	15<br>
Ser Pro Leu Asp Gly Ser Lys Ser Thr Asn Glu Asn Val Ser Ala Ser<br>
20	25	30<br>
Thr Ser Thr Ala Lys Pro Met Val Gly Gin Leu lie Phe Asp Lys Phe<br>
35	40	45<br>
lie Lys Thr Glu Glu Asp Pro lie He Lys Gin Asp Thr Pro Ser Asn<br>
50	55	60<br>
Leu Asp Phe Asp Phe Ala Leu Pro Gin Thr Ala Tlir Ala Pro Asp Ala<br>
55	70	75	80<br>
Lys Thr" Val. Leu Pro	He Pro Glu Leu Asp Asp Ala Val Val Glu Ser<br>
85	90	95<br>
Phe Phe Ser Ser Ser	Thr Asp Ser Thr Pro Met Phe Glu Tyr Glu Asn<br>
100	105	110<br>
Leu Glu Asp Asn Ser Lys Glu Trp Thr Ser Leu Phe Asp Asn Asp He<br>
115	120	125<br>
Pro Val Thr Thr Asp Asp Val Ser Leu Ala Asp Lys Ala He Glu Ser<br>
130	135	140<br>
Thr Glu Glu Val Ser Leu Val Pro Ser Asn Leu Glu Val Ser Thr Thr<br>
145	150	155	160<br>
Ser Phe Leu Pro Thr Pro Val Leu Glu Asp Ala Lys Leu Thr Gin Thr<br>
155	170	175<br>
Arg Lys Val Lys Lys Pro Asn Ser Val Val Lys Lys Ser His His Val<br>
180	185	190<br>
Gly Lys Asp Asp Glu Ser Arg Leu Asp His Leu Gly Val Val Ala Tyr<br>
195	200	205<br>
Asn Arg Lys Gin Arg Ser lie Pro Leu Ser Pro He Val Pro Glu Ser<br>
210	215	220<br>
Ser Asp Pro Ala Ala Leu Lys Arg Ala Arg Asn Thr Glu Ala Ala Arg<br>
225	230	235	240<br>
Arg Ser Arg Ala Arg Lys Leu Gin Arg Met Lys Gin Leu Glu Asp Lys<br>
245	250	255<br>
Val Glu Glu Leu Leu Ser Lys Asn Tyr His Leu Glu Asn Glu Val Ala<br>
260	265	27Q<br>
Arg Leu Lys Lys Leu Val Gly Glu Arg<br>
275	280<br>	155<br>	181<br>	PRT<br>	Escherichia coli<br>   155<br>
Met Lys   He Lys  Thr Leu Ala  He Val Val  Leu Ser Ala Leu Ser Leu<br>
15	10	15<br><br>
Ser Ser Thr Ala Ala Leu Ala Ala Ala Thr Thr Val Asn Gly Gly Thr<br>
20	25	30<br>
Val His Phe Lys Gly Glu Val Val Asn Ala Ala Cys Ala Val Asp Ala<br>
35	40	45<br>
Gly Ser Val Asp Gin Thx Val Gin Leu Gly Gin Val Arg Thr Ala Ser<br>
50	55	60<br>
Leu Ala Gin Glu Gly Ala Thr Ser Ser Ala Val Gly Phe Asn lie Gin<br>
65	70	75	80<br>
Leu Asn Asp cys Asp Thr Asn Val Ala Ser Lys Ala Ala Val Ala Phe<br>
85	90	95<br>
Leu Gly Thr Ala lie Asp Ala Gly His Thr Asn Val Leu Ala Leu Gin<br>
100	105	110<br>
Ser Ser Ala Ala Gly Ser Ala Thr Asn Val Gly Val Gin He Leu Asp<br>
115	120.	125<br>
Arg Thr Gly Ala Ala Leu Thr Leu Asp Gly Ala Thr Phe Ser Ser Glu<br>
130	135	140<br>
Thr Thr Leu Asn Asn Gly Thr Asn Thr He Pro Phe Gin Ala Arg Tyr<br>
145	150	155	160<br>
Phe Ala Gly Ala Ala Thr Pro Gly Ala Ala Asn Ala Asp Ala Thr Phe<br>
1S5	170	175<br>
Lys Val Gin Tyr Gin. 180<br> 156  447  DNA  Hepatitis B<br><br> CDS<br> (1)..(447)<br> 156<br>
atg gac att gae cet tat aaa gaa ttt gga get act gtg gag tta etc  48<br>
Het Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
teg ttt ttg cct tct gac ttc ttt cct tec gta cga gat ctt eta gat  96<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
ace gcc gca get ctg tat egg gat gee tta gag tct cct gag cat tgt  144<br>
Thr Ala Ala Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	30	45<br>
tea cct eac eat act gca etc agg caa gca att ctt tgc tgg gga gac  192<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp<br>
50	55	60<br>
tta atg act eta get ace tgg gtg ggt act aat tta gaa gat cca gca  240<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Thr Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br><br>
tct agg gac eta gta gtc agt tat gte aae act aat gtg ggc eta aag  288<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys<br>
85	90	95<br>
ttc aga caa tta ttg tgg ttt cac att tct tgt etc act ttt gga aga  336<br>
Phe Arg Gin Leu Lieu Trp Phe His He Ser Cys Leu Ttir Phe Gly Arg<br>
100	105	110<br>
gaa acg gtt eta gag tat ttg gtc tct ttt gga gtg tgg att cgc act  384<br>
Glu Thr Val Leu Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
cet cea gcc tat aga cca cca aat gcc cct ate eta tea acg ctt ecg  432<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu ser Thr Leu Pro<br>
130	135	140<br>
gag act aet gtt gtt	447<br>
Glu Thr Thr Val Val<br>
145<br> 157  149  PRT  Hepatitis B<br> 157<br>
Met Asp He Asp pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ala Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp<br>
50	55	60<br>
Leu Met Thr Leu Ala Thr Trp Val Gly Thr Asn Leu Glu Asp Pro Ala<br>
65	70	75	80<br>
Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Val Gly Leu Lys<br>
85	90	95<br>
Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr Phe Gly Arg<br>
100	105	110<br>
Glu Thr Val Leu Glu Tyr Leu Val Ser Phe Gly Val Trp He Arg Thr<br>
115	120	125<br>
Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser Thr Leu Pro<br>
130	135	140<br>
Glu Thr Thr Val Val 145<br> 158  152  PRT<br><br> Hepatitis B<br>   158<br>
Met Asp He Asp Pro Tyr Lys Glu Phe Gly Ala Thr Val Glu Leu Leu<br>
15	10	15<br>
Ser Phe Leu Pro Ser Asp Phe Phe Pro Ser Val Arg Asp Leu Leu Asp<br>
20	25	30<br>
Thr Ala Ala Ala Leu Tyr Arg Asp Ala Leu Glu Ser Pro Glu His Cys<br>
35	40	45<br>
Ser Pro His His Thr Ala Leu Arg Gin Ala He Leu Cys Trp Gly Asp<br>
50	55	60<br>
Leu Met Ihr Leu Ala Thr Trp Val Gly Thx Asn Leu Glu Asp Gly Gly<br>
65	70	75	SO<br>
Lys Gly Gly Ser Arg Asp Leu Val Val Ser Tyr Val Asn Thr Asn Val<br>
85	90	95<br>
Gly Leu Lys Phe Arg Gin Leu Leu Trp Phe His He Ser Cys Leu Thr<br>
100	105	110<br>
Phe Gly Arg Glu Thr Val Leu Glu Tyr Leu Val Ser Phe Gly Val Trp<br>
115	120	125<br>
He Arg Thr Pro Pro Ala Tyr Arg Pro Pro Asn Ala Pro He Leu Ser<br>
130	135	140<br>
Thr Leu Pro Glu Thr Thr Val Val<br>
145	150<br>	159<br>	132<br>	PRT<br>	Bacteriophage Q Beta<br>     159<br>
Ala Lys  Leu Glu Thr Val Thr Leu Gly Asn He Gly Lys  Asp Gly Lys<br>
15	10	15<br>
Gin Thr Leu Val Leu Asn Pro Arg Gly Val Asn Pro Thr Asn Gly Val<br>
20	25	30<br>
Ala Ser Leu Ser Gin Ala Gly Ala Val Pro Ala Leu Glu Lys Arg Val<br>
35	40	45<br>
Thr Val Ser Val Ser Gin Pro Ser Arg Asn Arg Lys Asn Tyr Lys Val<br>
50	55	60<br>
Gin Val Lys He Gin Asn Pro Thr Ala Cys Thr Ala Asn Gly Ser Cys<br>
65	70	75	80<br>
Asp Pro Ser Val Thr Arg Gin Ala Tyr Ala Asp Val Thr Phe Ser Phe<br>
85	90	95<br>
Thr Gin Tyr Ser Thr Asp Glu Glu Arg Ala phe Val Arg Thr Glu Leu<br>
100	1D5	IID<br>
Ala Ala Leu Leu Ala Ser Pro Leu Leu He Asp Ala He Asp Gin Leu<br>
115	120	125<br><br>
f<br>
ftsn Pro Ala Tyr 130<br>	160<br>	129<br>	PRT<br>	Bacteriophage R 17<br>  160<br>
Ala Ser Asn Phe Thr Gin Phe Val Leu Val Asn Asp Gly Gly Thr Gly<br>
15	10	15<br>
Asn Val Thr Val Ala Pro Ser Asn Phe Ala Asn Gly Val Ala Glu Trp<br>
20	25	30<br>
He Ser Ser Asn Ser Arg Ser Gin Ala Tyr Lys Val Thr Cys Ser Val<br>
35	40	45<br>
Arg Gin Ser Ser Ala Gin Asn Arg Lys Tyr Thr He Lys Val Glu Val<br>
50	55	60<br>
Pro Lys Val Ala Thr Gin Thr Val Gly Gly Val Glu Leu Pro Val Ala<br>
65	70	75	80<br>
Ala Trp Arg Ser Tyr Leu Asn Met Glu Leu Thr He Pro He Phe Ala<br>
85	90	95<br>
Thr Asn Ser Asp Cys Glu Leu He Val Lys Ala Met Gin Gly Leu Leu<br>
100	105	IID<br>
Lys Asp Gly Asn Fro He Pro Ser Ala He Ala Ala Asn Ser Gly He<br>
115	120	125<br>
Tyr<br>	161<br>	130<br>	PRT<br>	Bacteriophage fr<br>  161<br>
Met Ala Ser Asn Phe Glu Glu Phe Val Leu Val Asp Asn Gly Gly Thr<br>
15	10	15<br>
Gly Asp Val Lys Val Ala Pro Ser Asn Phe Ala Asn Gly Val Ala Glu<br>
20	25	30<br>
Trp He Ser Ser Asn Ser Arg Ser Gin Ala Tyr Lys Val Thr Cys Ser<br>
35	40	45<br>
Val Arg Gin Ser Ser Ala Aan Asn Arg Lys lyz  Thr Val Lys Val Glu<br>
50	55	60<br>
*<br>
Val Pro Lys Val Ala Thr Gin Val Gin Gly Gly Val Glu Leu Pro Val<br>
65	70	75	80<br>
Ala Ala Trp Arg Ser Tyr Met Asn Met Glu Leu Thr He Pro Val Phe<br><br>
85	90	95<br>
Ala Thr Asn Asp Asp Cys Ala Leu He Val Lys Ala Leu Gin Gly Thr<br>
100	105	110<br>
Phe Lys Thr Gly Asn Pro He Ala Thr Ala He Ala Ala Asn Ser Gly<br>
115	120	125<br>
He Tyr 130<br>.	162<br>	130<br>	PRT<br>	Bacteriophage GA<br>  162<br>
Met Ala Thr Leu Arg Ser Phe Val Leu Val Asp Asn Gly Gly Thr Gly<br>
15	10	15<br>
Asn Val Thx Val Val Pro Val Ser Asn Ala Asn Gly Val Ala Glu Trp<br>
20	25	30<br>
Leu Ser Asn Asn Ser Arg Ser Gin Ala Tyr Arg Val Thr Ala Ser Tyr<br>
35	40	45<br>
Arg Ala Ser Gly Ala Asp Lys Arg Lys Tyr Ala He Lys Leu Glu Val<br>
50	55	60<br>
Pro Lys He Val Thr Gin Val Val Asn Gly Val Glu Leu Pro Gly Ser<br>
65	70	75	80<br>
Ala Trp Lys Ala Tyr Ala Ser He Asp Leu Thr He Pro He Phe Ala<br>
85	90	95<br>
Ala Thr Asp Asp Val Thr Val He Ser Lys Ser Leu Ala Gly Leu Phe<br>
100	105	110<br>
Lys Val Gly Asn Pro He Ala Glu Ala He Ser Ser Gin Ser Gly Phe<br>
115	120	125<br>
Tyr Ala 130<br>	163<br>	132<br>	PRT<br>	Bacteriophage SP<br> 163<br>
Met Ala Lys Leu Asn Gin Val Thr.Leu Ser Lys He Gly Lys Asn Gly<br>
15	10	15<br>
Asp Gin Thr Leu Thr Leu Thr Pro Arg Gly Val Asn Pro Thr Asn Gly<br>
20	25	30<br>
Val Ala Ser Leu Ser Glu Ala Gly Ala Val Pro Ala Leu Glu Lys Arg<br>
35	40	45<br><br>
Val Thr Val Ser Val Ala Glu Pro Ser Arg Asn Arg Lys Asn Phe Lys<br>
50	55	60<br>
Val Gin lie Lys Leu Gin Asn Pro Thr Ala Cys Thr Arg Asp Ala Cys<br>
S5	70	75	SO<br>
Asp Pro Ser Val Thr Arg Ser Ala Phe Ala Asp Val Thr Leu Ser Phe<br>
85	90	95<br>
Thr Ser Tyr Ser Thr Asp Glu Glu Arg Ala Leu lie Arg Thr Glu Leu<br>
100	105	110<br>
Ala Ala Leu Leu Ala Asp Pro Leu lie Val Asp Ala lie Asp Asn Leu<br>
115	120	125<br>
Asn Pro Ala Tyr 130<br>	164<br>	130<br>	PRT<br>	Bacteriophage MS2<br>  164<br>
Met Ala Ser Asn Phe Thr Gin Phe Val Leu Val Asp Asn Gly Gly Thr<br>
15	10	15<br>
Gly Asp Val Thr Val Ala Pro Ser Asn Phe Ala Asn Gly Val Ala Glu<br>
20	25	30<br>
Trp lie Ser Ser Asn Ser Arg Ser Gin Ala Tyr Lys Val Thr Cys Ser<br>
35	40	45<br>
Val Arg Gin Ser Ser Ala Gin Asn Arg Lys Tyr Thr lie Lys Val Glu<br>
50	55	60<br>
Val Pro Lys Val Ala Thr Gin Thr Val Gly Gly Val Glu Leu Pro Val<br>
65	70	75	80<br>
Ala Ala Trp Arg Ser Tyr Leu Asn Met Glu Leu Thr lie Pro lie Phe<br>
85	90	95<br>
Ala Thr Asn Ser Asp Cys Glu Leu lie Val Lys Ala Met Gin Gly Leu<br>
100	105	110<br>
Leu Lys Asp Gly Asn Pro lie Pro Ser Ala lie Ala Ala Asn Ser Gly<br>
115	120	125<br>
He Tyr 130<br>	165<br>	133<br>	PRT<br>	Bacteriophage Mil<br>  165<br>
Met Ala Lys Leu Gin Ala lie Thr Leu Ser Gly lie Gly Lys Lys Gly<br>
15	10	15<br><br>
T<br>
Asp Val Thr Leu Asp Leu Asn Pro Arg Gly val Asn Pro Thr Asn Gly<br>
20	25	30<br>
Val Ala Ala Leu Ser Glu Ala Gly Ala Val pro Ala Leu Glu Lys Arg<br>
35	40	45<br>
Val Thr lie Ser Val Ser Gin Pro Ser Arg Asn Arg Lys Asn Tyr Lys<br>
50	55	50<br>
Val Gin Val Lys lie Gin Asn Pro Thr Ser cys Thr Ala Ser Gly Thr<br>
65	70	75	80<br>
Cys Asp Pro Ser Val Thr Arg Ser Ala Tyr Ser Asp Val Thr Phe Ser<br>
85	90	95<br>
Pbe Thr Gin Tyr Ser Thr Val Glu Glu Arg Ala Leu Val Arg Thr Glu<br>
100	105	110<br>
Leu Gin Ala Leu Leu Ala Asp Pro Met ieu Val Asn Ala lie Asp Asn<br>
115	120	125<br>
Leu Asn Pro Ala Tyr 130<br>	166<br>	133<br>	PRT<br>	Bacteriophage MXl<br>  166<br>
Met Ala Lys Leu Gin Ala lie Thr Leu Ser Gly lie Gly Lys Asn Gly<br>
15	10	15<br>
Asp Val Thr Leu Asn Leu Asn Pro Arg Gly Val Asn Pro Thr Asn Gly<br>
20	25	30<br>
Val Ala Ala LGU Ser Glu Ala Gly Ala Val Pro Ala Leu Glu Lys Arg<br>
35	40	45<br>
Val Thr lie Ser Val Ser Gin Pro Ser Arg Asn Arg Lys Asn Tyr Lys<br>
50	55	60<br>
Val Gin Val Lys lie Gin Asn Pro Thr Ser Cys Thr Ala Ser Gly Thr<br>
65	70    "	75	80<br>
Cys Asp Pro Ser Val Thr Arg Ser Ala Tyr Ala Asp Val Thr Phe Ser<br>
85	90	95<br>
Phe Thr Gin Tyr Ser Thr Asp Glu Glu Arg Ala Leu Val Arg Thr Glu<br>
100	105	110<br>
Leu Lys Ala Leu Leu Ala Asp Pro Met Leu He Asp Ala lie Asp Asn<br>
115	120   . "	125<br>
Leu Asn Pro Ala Tyr 130<br> 167  330   PRT<br><br>
r<br>    Bacteriophage NL95<br>    157<br>
Met Ala Lys Leu Asn Lys Val Thr Leu Thr Gly He Gly Lys Ala Gly<br>
15	10	15<br>
Asn Gin Thr Leu Thr Leu Thr Pro Arg Gly Val Asn Pro Thr Asn Gly<br>
20	25	30<br>
Val Ala Ser Leu Ser Glu Ala Gly Ala Val Pro Ala Leu Glu Lys Arg<br>
35	40	45<br>
Val Thr Val Ser Val Ala Gin Pro Ser Arg Asn Arg Lys Asn Tyr Lys<br>
50	55	60<br>
Val Gin He Lys Leu Gin Asn Pro Thr Ala Cys Thr Lys Asp Ala Cys<br>
65	70	75	80<br>
Asp Pro Ser Val Thr Arg Ser Gly Ser Arg Asp Val Thr Leu Ser Phe<br>
85	90	95<br>
Thr Ser Tyr Ser Thr Glu Arg Glu Arg Ala Leu He Arg Thr Glu Leu<br>
100	105	110<br>
Ala Ala Leu Leu Lys Asp Asp Leu lie Val Asp Ala He Asp Asn Leu<br>
115	120	125<br>
Asn Pro Ala Tyr Trp Ala Ala Leu Leu Ala Ala Ser Pro Gly Gly Gly<br>
130	135	140<br>
Asn Asn Pro Tyr Pro Gly Val Pro Asp Ser Pro Asn Val Lys Pro Pro<br>
145	150	155	160<br>
Gly Gly Thr Gly Thr Tyr Arg Cya Pro Phe Ala Cys Tyr Arg Arg Gly<br>
165	170	175<br>
Glu Leu He Thr Glu Ala Lys Asp Gly Ala Cys Ala Leu Tyr Ala Cys<br>
180	185	190<br>
Gly Ser Glu Ala Leu Val Glu Phe Glu Tyr Ala Leu Glu Asp Phe Leu<br>
195	200	205<br>
Gly Asn Glu Phe Trp Arg Asn Trp Asp Gly Arg Leu Ser Lys Tyr Asp<br>
210	215	220<br>
He Glu Thr His Arg Arg Cys Arg Gly Asn Gly Tyr Val Asp Leu Asp<br>
225	230	235	240<br>
Ala Ser Val Met Gin Ser Asp Glu Tyr Val Leu Ser Gly Ala Tyr Asp<br>
245	250	255<br>
Val Val Lys Met Gin Pro Pro Gly Thr Phe Asp Ser Pro Arg Tyr Tyr<br>
260	265	270<br>
Leu His Leu Met Asp Gly He Tyr Val Asp Leu Ala Glu Val Thr Ala<br>
275	280	285<br>
Tyr Arg Ser Tyr Gly Met Val He Gly Phe Trp Thr Asp Ser Lys Ser<br>
290	295	300<br>
Pro Gin Leu Pro Thr Asp Phe Thr Arg Phe Asn Arg His Asn Cys Pro<br>
305	310	315	320<br><br>
Wal Gin Thr Val lie Val lie Pro Ser Leu<br>
325	330<br>	168<br>	134<br>	PRT<br>	Apis irellifera<br>  158<br>
lie lie Tyr Pro Gly Thr Leu Trp Cys Gly His Gly Asn Lys Ser Ser<br>
15	10	15<br>
Gly Pro Asn Glu Leu Gly Arg Phe Lys His Thr Asp Ala Cys Cys Arg<br>
20	25	30<br>
Thr His Asp Met Cys Pro Asp Val Met Ser Ala Gly Glu Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asn Thr Ala Ser His Thr Arg Leu Ser Cys Asp Cys Asp<br>
50	55	60<br>
Asp Lys Phe Tyr Asp Cys Leu Lys Asn Ser Ala Asp Thr lie Ser Ser<br>
65	70	75	80<br>
Tyr Phe Val Gly Lys Met Tyr  Phe Asn Leu lie Asp Thr Lys Cys Tyr<br>
85	90	95<br>
Lys Leu Glu His Pro Val Thr Gly CyS Gly Glu Arg Thr Glu Gly Arg<br>
100	105	110<br>
Cys Leu His Tyr Thr Val Asp Lys Ser Lys Pro Lys Val Tyr Gin Trp<br>
115	120	125<br>
Phe Asp Leu Arg Lys Tyr 130<br>	169<br>	129<br>	PRT<br>	;is mellifera<br>     169<br>
He lie Tyr Pro Gly Thr Leu Trp Cys Gly His  Gly Asn Lys  Ser  Ser<br>
15	10	15<br>
Gly Pro Asn Glu Leu Gly Arg Phe Lys His Thr Asp Ala Cys Cys Rrg<br>
20	25	30<br>
Thr His Asp Met Cys Pro Asn Val Met Ser Ala Gly Glu Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asp Thr Ala Ser Arg Leu Ser Cys Asn Asp Asn Asp Leu<br>
50	55	60<br>
Phe Tyr Lys Asp Ser Ala Asp Thr He Ser Ser Tyr Phe Val Gly Lys<br>
55	70	75	80<br>
Met Tyr Phe Asn Leu He Asn Thr Lys Cys Tyr Lys Leu Glu His Pro<br>
85	90	95<br>
Vai Thr Gly Cys Gly Glu Arg Thr Glu Gly Arg Cys Leu His Tyr Thr<br>
100	105	110<br><br>
Val Asp Lys Ser Lys Pro Lys Val Tyr Gin Trp Phe Asp Leu Arg Lys<br>
115	120	125<br>
Tyr<br>	170<br>	134<br>	PET<br>	Apia dorsata<br> 170<br>
He He Tyr Pro Gly Thr Leu Trp Cys Gly His  Gly Asn Val  Ser Ser<br>
15	10	15<br>
Ser Pro Asp Glu Leu Gly Arg Phe Lys Hig Thr Asp Ser Cys Cys Arg<br>
20	25	30<br>
Ser His Asp Met Cys Pro Asp Val Met Ser Ala Gly Glu  Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asn Thr Ala Ser His Thr Arg Leu Ser Cys Asp Cys Asp<br>
50	55	60<br>
Asp Lys Phe Tyr Asp Cys Leu Lys Asn Ser Ser Asp Tlir He Ser Ser<br>
65	70	75	80<br>
Tyr Phe Val Gly Glu Met Tyr Phe Asn He Leu Asp Thr Lys Cys Tyr<br>
85	90	95<br>
Lys Leu Glu His Pro Val Thr Gly Cys Gly Lys Arg Thr Glu Gly Arg<br>
100	105	110<br>
Cys Leu Asn Tyr Thr Val Asp Lys Ser Lys Pro Lys Val Tyr Gin Trp<br>
lis	120	125<br>
Phe Asp Leu Arg Lys Tyr 130<br>	171<br>	134<br>	PRT<br>	Apis cerana<br>     171<br>
He He  Tyr Pro Gly Thr Leu Trp Cys Gly His  Gly Asn Val  Ser  Ser<br>
15	10	15<br>
Gly Pro Asn Glu Leu Gly Arg Phe Lys His Thr Asp Ala Cys Cys Arg<br>
20	25	30<br>
Thr His Asp Met Cys Pro Asp Val Met Ser Ala Gly Glu Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asn Tfar Ala Ser His Thr Arg Leu Ser Cys Asp Cys Asp<br>
50	55	60<br>
Asp Thr Phe Tyr Asp Cys Leu Lys Asn Ser Gly Glu Lys He Ser Ser<br>
65	70	75	80<br>
Tyr Phe Val Gly I-ys Met Tyr Phe Asn teu He Asp Thr Lys Cys Tyr<br>
85	90	95<br><br>
Lys Leu Glu His Pro Val Thr Gly Cys Gly Glu Arg Thr Glu Gly Arg<br>
100	105	110<br>
Cys Leu Arg Tyr Thr Vsl Asp Lys Ser Lys Pro Lys Val Tyr Gin Tip<br>
il5	12 0	125<br>
Phe Asp Leu Arg Lys Tyr 130<br>	172<br>	136<br>	PBT<br>	Bombus pennsylvanicus<br>  172<br>
lie He Tyr Pro Gly Thr Leu Trp Cys Gly Asn Gly Asn He Ala Asn<br>
15	10	â "       15<br>
Gly Thr Asn Glu Leu Gly Leu Trp Lys Glu Thr Asp Ala Cys Cys Arg<br>
20	25	30<br>
Thr His Asp Met Cys Pro Asp He He Glu Ala His Gly Ser Lys His<br>
35	40	45<br>
Gly Leu Thr Asn Pro Ala Asp Tyr Thr Arg Leu Asn Cys Glu Cys Asp<br>
50	55	60<br>
Glu Glu phe Arg His Cys Leu His Asn Ser Gly Asp Ala Val Ser Ala<br>
65	70	75	80<br>
Ala Phe val Gly Arg Thr Tyr Phe Thr He Leu Gly Thr Gin Cys Phe<br>
85	90	95<br>
Arg Leu Asp Tyr Pro He Val Lys Cys Lys Val Lys Ser Thr He Leu<br>
100	105	110<br>
Arg Glu Cys Lys Glu Tyr Glu Phe Asp Thr Asn Ala Pro Gin Lys Tyr<br>
115	120	125<br>
Gin Trp Phe Asp Val Leu Ser Tyr<br>
130	135<br>	173<br>	142<br>	PRT<br>	Helodenna suspectum<br>     173<br>
Gly Ala  Phe He Met Pro Gly Thr Leu Trp Cys   Gly Ala Gly Asn Ala<br>
15	10	15<br>
Ala Ser Asp Tyr Ser Gin Leu Gly Thr Glu Lys Asp Thr Asp Met Cys<br>
20	25	30<br>
Cys Arg A5P His Asp His Cys Ser Asp Thr Met Ala Ala Leu Glu Tyr<br>
35	iO	45<br>
Lys His Gly Met Arg Asn Tyr Arg Pro His Thr Val Ser His Cys Asp<br>
50	55	60<br><br>
Cys Asp Asn Gin Phe Arg Ser Cys Leu Met Asn Val Lys Asp Arg Thr<br>
65	70	75	80<br>
Ala Asp Leu Val Gly Met Thr Tyr Phe Thr Val Leu Lys lie Ser Cys<br>
85	90	95<br>
Phe Glu Leu Glu Glu Gly Glu Gly Cys Val Asp Asn Asn Phe Ser Gin<br>
100	105	110<br>
Gin Cys Thr Lys Ser Glu lie Met Pro Val Ala Lys Leu Val Ser Ala<br>
115	120	125<br>
Ala Pro Tyr Gin Ala Gin Ala Glu Thr Gin Ser Gly Glu Gly<br>
130	135	140<br>	174<br>	143<br>	PRT<br>	Heloderma suspectura<br>  174<br>
Gly Ala Phe lie Met Pro Gly Thr Leu Trp cys Gly Ala Gly Asn Ala<br>
15	10	15<br>
Ala Ser Asp Tyr Ser Gin Leu Gly Thr Glu Lys Asp Thr Asp Met Cys<br>
20	25	30<br>
Cys Arg Asp His Asp His Cys Glu Asn Trp lie Ser Ala Leu Glu Tyr<br>
35	40	45<br>
Lys His Gly Met Arg Asn Tyr Tyr Pro Ser Thr lie Ser His Cys Asp<br>
50	55	60<br>
Cys Asp Asn Gin Phe Arg Ser Cys Leu Met Lys Leu Lys Asp Gly Thr<br>
65	70	75	80<br>
Ala Asp Tyr Val Gly Gin Thr Tyr Phe Asn Val Leu Lys lie Pro Cys<br>
B5	90	95<br>
Phe Glu Leu Glu Glu Gly Glu Gly Cys Val Asp Trp Asn Phe Trp Leu<br>
100	105	110<br>
Glu Cys Thr Glu Ser Lys He Met Pro Val Ala Lys Leu Val Ser Ala<br>
115	120	125<br>
Ala Pro Tyr Gin Ala Gin Ala Glu Thr Gin Ser Gly Glu Gly Arg<br>
130	135	140<br>	175<br>	142<br>	PRT<br>	Heloderma suspectum<br>  175<br>
Gly Ala Phe lie Met Pro Gly Thr Leu Trp Cys Gly Ala Gly Asn Ala<br>
1	5	10	15<br>
Ala Ser Asp Tyr Ser Gin Leu Gly Thr Glu Lys Asp Thr Asp Met Cys<br>
20	25	30<br><br>
f<br>
Cys Arg Asp His Asp His Cys Glu Asn Trp He Ser Ala Leu Glu Tyr<br>
35	40	45<br>
Lys His Gly Met Arg Asn Tyr Tyr Pro Ser Thr He Ser His Cys Asp<br>
50	55	60<br>
Cys Asp Asn Gin Phe Arg Ser Cys Leu Met Lys Leu Lys Asp Gly Thr<br>
65	70	75	80<br>
Ala Asp Tyr Val Gly Gin Thr Tyr Phe Asn Val Leu Lys He Pro Cys<br>
85	90	95<br>
Phe Glu Leu Glu Glu Gly Glu Gly Cys Val Asp Trp Asn Phe Trp Leu<br>
100	105	110<br>
Glu Cys Thr Glu Ser Lys He Met Pro Val Ala Lys Leu Val Ser Ala<br>
115	120	125<br>
Ala Pro Tyr Gin Ala Gin Ala Glu Thr Gin Ser Gly Glu Gly<br>
130	135	140<br>	176<br>	574<br>	PRT<br>	IgE heavy chain<br> ,176<br>
Met Asp Trp Thr Trp He Leu Phe Leu Val Ala Ala Ala Thr Arg Val<br>
15	10	15<br>
His Ser Gin Thr Gin Leu Val Gin Ser Gly Ala Glu Val Arg Lys Pro<br>
20	25	30<br>
Gly Ala Ser Val Arg Val Ser Cys Lys Ala Ser Gly Tyr Thr Phe He<br>
35	40	45<br>
Asp Ser Tyr He His Trp He Arg Gin Ala Pro Gly His Gly Leu Glu<br>
50	55	60<br>
Trp Val Gly Trp He Asn Pro Asn Ser Gly Gly Thr Asn Tyr Ala Pro<br>
65	70	75	80<br>
Arg Phe Gin Gly Arg Val Thr Met Thr Arg Asp Ala Ser Phe Ser Thr<br>
85	90	95<br>
Ala Tyr Met Asp Leu Arg Ser Leu Arg Ser Asp Asp Ser Ala Val Phe<br>
100	105	110<br>
Tyr Cys Ala Lys Ser Asp Pro Phe Trp Ser Asp Tyr Tyr Asn Phe Asp<br>
115	120	125<br>
Tyr Ser Tyr Thr Leu Asp Val Trp Gly Gin Gly Thr Thr Val Thr Val<br>
130	135	140<br>
Ser Ser Ala Ser Thr Gin Ser Pro Ser Val Phe Pro Leu Thr Arg Cys<br>
145	150	155	160<br>
Cys Lys Asn He Pro Ser Asn Ala Thr Ser Val Thr Leu Gly Cys Leu<br>
165	170	175<br>
Ala Thr Gly Tyr Phe Pro Glu Pro Val Met Val Thr Trp Asp Thr Gly<br><br>
"	180	185	190<br>
Ser Leu Asn Gly Thr Thr Met Thr Leu Pro Ala Thr Thr Leu Thr Leu<br>
195	200	205<br>
Ser Gly His Tyr Ala Thr lie Ser Leu Leu Thr Val Ser Gly Ala Trp<br>
210	215	220<br>
Ala Lys Gin Met Phe Thr Cys Arg Val Ala His Thr Pro Ser Ser Thr<br>
225	230	235	240<br>
Asp Trp Val Asp Asn Lys Thr Phe Ser Val Cys Ser Arg Asp Phe Thr<br>
245	250	255<br>
Pro Pro Thr Val Lys lie Leu Gin Ser ser Cys Asp Gly Gly Gly His<br>
260	265	270<br>
Phe Pro Pro Thr Xle Gin Leu Leu Cys Leu val Ser Gly Tyr Thr Pro<br>
275	280	285<br>
Gly Thr He Asn lie Thr Trp Leu Glu Asp Gly Gin Val Met Asp Val<br>
290	295	300<br>
Asp Leu Ser Thr Ala Ser Thr Thr Gin Glu Gly Glu Leu Ala Ser Thr<br>
305	310	315	320<br>
Gin Ser Glu Leu Thr Leu Ser Gin Lys His Trp Leu Ser Asp Arff Thr<br>
325	330	335<br>
Tyr Thr Cys Gin Val Thr Tyr Gin Gly His Thr Phe Giu Asp Ser Thr<br>
340	345	350<br>
Lys Lys Cys Ala Asp Ser Asn Pro Arg Gly Val Ser Ala Tyr Leu Ser<br>
355	360	365<br>
Arg Pro Ser Pro Phe Asp Leu phe He Arg Lys Ser Pro Thr He Thr<br>
370	375	380<br>
Cys Leu Val Val Asp Leu Ala pro Ser Lys Gly Thr Val Asn Leu Thr<br>
385	390	395 .	400<br>
Trp Ser Arg Ala Ser Gly Lys Pro Val Asn His Ser Thr Arg Lya Glu<br>
405	410	415<br>
Glu Lys Gin Arg Asn Gly Thr Leu Thr Val Thr Ser Thr Leu Pro Val<br>
420	425	430<br>
Gly Thr Arg Asp Trp He Glu Gly Glu Thr Tyr Gin Cys Arg Val Thr<br>
435	440	445<br>
His Pro Hie Leu Pro Arg Ala Leu Met Arg Ser Thr Thr Lys Thr Ser<br>
450	455	460<br>
Gly Pro Arg Ala Ala Pro Glu Val Tyr Ala Phe Ala Thr Pro Glu Trp<br>
465	470	475	480<br>
Pro Gly Ser Arg Asp Lys Arg Thr Leu Ala Cys Leu He Gin Asn Phe<br>
485	490	â  495<br>
Met Pro Glu Asp lie Ser Val Gin Trp Leu His Asn Glu Val Gin Leu<br>
500	505	510<br>
Pro Asp Ala Arg His Ser Thr Thr Gin Pro Arg Lys Thr Lys Gly Ser<br><br>
515	520	525<br>
Gly Phe Phe Val Phe Ser Arg Leu Glu Val Tbr Arg Ala Glu Trp Glu<br>
530	535	540<br>
Gin Lys Asp Glu Phe He Cys Arg Ala Val His Glu Ala Ala Ser Pro<br>
545	550	555	560<br>
Ser Gin Thr Val Gin Arg Ala Val Ser Val Asn Pro Gly Lys<br>
555	570<br>
â¢;210&gt; 177<br><:400>  177 000<br>	178<br><:211>	13<br>	PRT<br>  178<br>
Cys Gly Gly Val Asn Leu Thr Trp Ser Arg Ala Ser Gly<br>
15	10<br>	179<br>	8<br>	PRT<br>	IgE Kimotype<br>	179<br>
lie Asn His Arg Gly Tyr Trp Val<br>
1	5<br>	180<br>	a<br>	PRT<br>	IgE Mimotype<br>	180<br>
Arg Aan His Arg Gly Tyr Trp Val<br>
1	5<br>	181<br>	10<br><:212>	PRT<br>	IgE tsimotype<br>  181<br>
Arg Ser Arg Ser Gly Gly Tyr Trp Leu Trp<br>
15	10<br>  182   10   PRT<br><br>
-r213&gt;    igE Mimotype<br>     182<br>
Val Asn Leu Thr Trp Ser Arg Ala Ser Gly<br>
15	10<br>	183<br>	10<br>	PRT<br>	IgE Mimotype<br>  183<br>
Val Asn Leu Pro Trp Ser Arg Ala Ser Gly<br>
15	10<br>
â¢:210&gt;	184<br>	10<br>	PRT<br>	IgE Mimotype<br><:400>  184<br>
Val Asn Leu Thr Trp Ser Phe Gly Leu Glu<br>
15	10<br>	185<br>	10<br>	PRT<br>	IgE Mimotype<br>  185<br>
Val Asn Leu Pro Trp Ser Phe Gly Leu Glu<br>
15	10<br>	186<br>	10<br>	PRT<br>	IgE Mimotype<br>  186<br>
Val Asn Arg Pro Trp Ser Phe Gly Leu Glu<br>
15	10<br>	187<br>	10<br>	IgE Mimotype<br>  1B7<br>
Val Lys Leu Pro Trp Arg Phe Tyr Gin val<br>
15	10<br>	188<br>	10<br>	PRT<br>	IgE Mimotype<br>	188<br><br>
Val Trp Thr Ala Cys Gly Tyr Gly Arg Met<br>
1	5	â        10<br>	189<br>	7<br>	PRT<br>	IgE Mimotype<br>	189<br>
Gly Thr Val Ser Thr Leu Ser<br>
1	5<br>	190<br>	7<br>	PRT<br>	IgE Mimotype<br>	190<br>
Leu Leu Asp Ser Arg Tyr Trp<br>
1	5<br>	191<br>	7<br>	PRT<br>	IgE Mimotype<br>  191<br>
Gin Pro Ala His Ser Leu Gly<br>
1	5<br>
c210&gt;	192<br>	7<br>	PRT<br>	IgS Mimotype<br>	192<br>
Leu Trp Gly Met Gin  Gly Arg<br>
1	5<br>
.i210&gt;	193<br>
211&gt;	15<br>
i212&gt;	PRT<br>	IgE Mimotype<br>
â =400&gt;  193<br>
Leu Thr Leu Ser His Pro His Trp Val Leu Asn His Phe Val Ser<br>
15	10	15<br>
Â«:210&gt;	194<br>
-:211&gt;	9<br>
-:212&gt;	PRT<br>
":213&gt;	IgE Mimotype<br>
400&gt;  194<br>
Ser Met Gly Pro Asp Gin Thr Leu Arg<br><br>	195<br>	6<br>	PRT<br>	IgE Mimotype<br>	195<br>
Val Asn Leu Thr Trp Ser<br>
1	5<br>	196<br>	56<br>	DKR<br>	Oligonucleotide Primer<br>	196<br>
tagatgatta cgccaagctt ataatagaaa tagttttttg aaaggaaagc agcatg       56<br>	197<br>	45<br>	DNA<br>	Oligonucleotide Primer<br>	197<br>
gtcaaaggcc ttgtcgacgt tattccatta cgcccgtcat tttgg	45<br>	198<br>	4623<br>	DNft<br>	pFIKAIC<br>     198<br>
agacgaaagg gcctcgtgat acgcctattt  ttataggtta atgtcatgat aataatggtt	60<br>
tcttagacgt caggtggcac ttttcgggga aatgtgcgcg gaacccctat ttgtttattt	12Q<br>
ttctaaatac attcaaatat gtatccgctc atgagacaat aaccctgata aatgcttcaa	180<br>
taatattgaa aaaggaagag tatgagtatt  caacatttcc gtgtcgccct tattcccttt	240<br>
tttgcggcat  tttgccttcc tgtttttgct  cacccagaaa cgctggtgaa agtaaaagat	300<br>
gctgaagatc agttgggtgc acgagtgggt   tacatcgaac  tggatctcaa cagcggtaag	360<br>
atccttgaga gttttcgccc cgaagaacgt  tttccaatga tgagcacttt  tasagttctg	420<br>
ctatgtggcg cggtattatc ccgtaCtgac gccgggcaag agcaactcgg tcgccgcata	480<br>
cactattctc  agaatgactt ggttgagtac   tcaccagtca  cagaaaagca  tcttacggat	540<br>
ggcatgacag taagagaatt  atgcagtgct  gccataacca tgagtgataa cactgcggcc	600<br>
aacttacttc  tgacaacgat cggaggaccg aaggagctaa ccgctttttt gcacaacatg	S60<br><br>
ggggatcatg tsactcgcct  tgatcgttgg gaaccggagc  tgaatgaagc caCaccaaac	720<br>
gacgagcgtg acaccacgat  gcctgtagca atggcaacaa cgttgcgcaa actattaact	780<br>
ggcgaactac ttactctagc  ttcccggcaa caattaatag actggatgga ggcggataaa	840<br>
gttgcaggac cacttctgcg ctcggcoctt ccggctggct ggtttattgc tgataaatct	900<br>
ggagccggtg agcgtgggtc  tcgcggtatc attgcagcac  tggggccaga tggtaagccc	960<br>
tcccgtatcg tagttatcta cacgacgggg agtcaggcaa ctatggatga acgaaataga	1020<br>
cagatcgctg agataggtgc  ctcactgatt aagcattggt  aactgtcaga ccaagtttac	1080<br>
tcatatatac tttagattga  tttaaaactt catttttaat  ttaaaaggat ctaggtgaag	1140<br>
atcctttttg ataatctcat  gaccaaaatc ccttaacgtg agttttcgtt ccactgagcg	1200<br>
tcagaccccg tagaaaagat  caaaggatct tcCtgagatc ctttttttct gcgcgtaatc	1260<br>
tgctgcttgc aaacaaaaaa accaccgcta ccageggtgg tttgtttgcc ggatcaagag	1320<br>
ctaccaactc tttttccgaa  ggtaactggc  ttcagcagag cgcagatacc aaatactgtc	1380<br>
cttctagtgt agccgtagtt aggccaccac ttcaagaact ctgtagcacc gcctacatac	1440<br>
ctcgctctgc taatcctgtt accagtggct gctgccagtg gcgataagtc gtgtcttacc	1500<br>
gggttggact caagacgata gttaccggat aaggcgcagc ggtcgggctg aacggggggt	1560<br>
tcgtgcacac agcccagctt ggagcgaacg acctacaccg aactgagata cctacagcgt	1620<br>
gagctatgag aaagcgccac gcttcccgaa gggagaaagg cggacaggta tccggtaagc	1680<br>
ggcagggtcg gaacaggaga  gcgcacgagg gagcttccag ggggaaacgc  ctggtatctt	1740<br>
tatagtcctg tcgggtttcg ccacctctga cttgagcgtc gatttttgtg atgctcgtca	1800<br>
ggggggcgga gcctatggaa aaacgccagc aacgcggcct  ttttacggtt  cctggccttt	1860<br>
tgctggcctt ttgctcacat grttctttcct gcgttatccc ctgattctgt ggataaccgt	1920<br>
attaccgcct ttgagtgagc  tgataccgot pgccgcagcc gaacgaccga gcgcagcgag	1990<br>
tcagtgagcg aggaagcgga agagcgccca atacgcaaac cgcctctccc cgcgcgttgg	2040<br>
ccgattcatt aatgcagctg gcacgacagg  ttCcccgact  ggaaagcggg cagtgagcgc	2100<br>
aacgcaatta atgtgagtta gctcactcat taggcacccc aggctttaca ctttatgctt	2160<br>
ccggctcgta tgttgtgtgg aattgtgagc ggataacaat ttcacacagg aaacagctat	2220<br>
gaccatgatt acgccaagct  tataatagaa atagtttttt gaaaggaaag cagcatgaaa	22 80<br>
attaaaactc  tggcaatcgt  tgttctgtcg gctctgtccc  tcagttctac agcggctctg	2340<br>
gccgctgcca cgacggttaa tggtgggacc gttcacttta aaggggaagt  tgttaacgcc	2400<br>
gcttgcgcag ttgatgcagg  ctctgttgat  caaaccgttc  agttaggaca  ggttcgtacc	2460<br>
gcatcgctgg cacaggaagg agcaaccagt  tctgctgtcg gttttaacat tcagctgaat	2520<br>
gattgcgata ccaatgttgc  atctaaagcc gctgttgcct  ttttaggtac ggcgattgat	2580<br><br>
r<br>
gcgggtcata ccaacgttct ggctctgcag agttcagctg cgggtagcgc aacaaacgtt	2 640<br>
ggCgtgcaga tcctggacag aacgggtgct gcgctgacgc tggatggtgc gacatttagt	2700<br>
tcagaaacaa ccctgaataa cggaaccaat accattccgt tccaggcgcg ttattttgca	2760<br>
accggggccg caaccccggg tgctgctaat gcggatgcga ccttcaaggt tcagtatcaa	2820<br>
taacctaccc aggttcaggg acgtcattac gggcagggat gcccaccctt gtgcgataaa	2880<br>
aataacgatg aaaaggaaga gattatttct attagcgtcg ttgctgccaa tgtttgctct	2940<br>
ggccggaaat aaatggaata ccacgttgcc cggcggaaat atgcaatttc agggcgtcat	3OD0<br>
tattgcggaa acttgccgga ttgaagccgg tgataaacaa atgacggtca atatggggca	3060<br>
aatcagcagt aaccggtttc atgcggttgg ggaagatagc gcaccggtgc cttttgttat	3120<br>
tcatttacgg gaatgtagca cggtggtgag tgaacgtgta ggtgtggcgt ttcacggtgt	3180<br>
cgcggatggt aaaaatccgg atgtgctttc cgtgggagag gggccaggga tagccaccaa	3240<br>
tattggcgta gcgttgtttg atgatgaagg aaacctcgta ccgsttaatc gtcctccagc	3300<br>
aaactggaaa cggctttatt caggctctac ttcgctacat ttcstcgcca aatatcgtgc	3360<br>
taccgggcgt cgggttactg gcggcatcgc caatgcccag gcctggttct ctttaaccta	3420<br>
tcagtaattg ttcagcagat aatgtgataa caggaacagg acagtgagta ataaaaacgt	3480<br>
caatgtaagg asatcgcagg aaataacatt ctgcttgctg gcaggtatcc tgatgttcat	3540<br>
ggcaatgatg gttgccggac gcgctgaagc gggagtggcc ttaggtgcga ctcgcgtaat	3600<br>
ttatccggca gggcaaaaac aagagcaact tgccgtgaca aataatgatg aaaatagtac	3660<br>
ctatttaatt caatcatggg tggaaaatgc cgatggtgta aaggatggtc gttttatcgt	3720<br>
gacgcctcct ctgtttgcga tgaagggaaa aaaagagaat accttacgta ttcttgatgc	3760<br>
aacaaataac caa.ttgccac aggaccggga aagtttattc tggatgaacg ttaaagcgat	3840<br>
tccgtcaatg gataaatcaa aattgactga gaatacgcta cagctcgcaa ttatcagccg	3900<br>
cattaaactg tactatcgcc cggctaaatt agcgttgcca cccgatcagg ccgcagaaaa	3960<br>
attaagattt cgtcgtagcg cgaattctct gacgctgatt aacccgacac cctattacct	4020<br>
gacggtaaca gagttgaatg ccggaacccg ggttcttgaa aatgcattgg tgcctccaat	4060<br>
gggcgaaagc acggttaaat tgccttctga tgcaggaagc aatattactt accgaacaat	4140<br>
aaatgattat ggcgcactta cccccaaaat gacgggcgta atggaataac gtcgactcta	4200<br>
gaggatcccc gggtaccgag ctcgaattca ctggccgtcg ttttacaacg tcgtgactgg	4260<br>
gaaaaccctg gcgttaccca acttaatcgc cttgcagcan atcccccttt cgccagctgg	4320<br>
cgtaatagcg aagaggcccg caccgatcgc ccttcccaac agttgcgcag cctgaatggc	4380<br>
gaatggcgcc tgatgcggta ttttctcctt acgcatctgt gcggtatttc acaccgcata	4440<br>
tggegcactc tcagtacaat ctgctctgat gccgcatagt taagccagcc ccgacacccg	4500<br><br>
cCaacacccg ctgacgcgcc ctgacgggct tgtctgctcc cggcatccgc ttacagacaa 4560<br>
gctgtgaccg tctccgggag ctgcatgtgt cagaggtttt caccgtcatc accgaaacgc 4620<br>
gcg	4623<br>	199<br>	42<br>	DNA<br>	Oligonucleotide Primer<br>  199<br>
aagatcttaa gctaagcttg aattctctga cgctgattaa cc	42<br>	200<br>	41<br>	DNA<br>	Oligonucleotide Primer<br>  200<br>
acgtaaagca tttctagacc gcggatagta atcgtgctat c	41<br><:210>	201<br>	5681<br>	DKfi.<br>	pFIMD<br>  201<br>
tcaccgtcat caccgaaacg cgcgagacga aagggcctcg tgatacgcct atttttatag	60<br>
gttaatgtca tgataataat ggtttcttag acgtcaggtg gcacttttcg gggaaatgtg	120<br>
cgcggaaccc ctatttgttt attttCCtaa atacattcaa atatgtatcc gctcatgaga	180<br>
caataaccct gataaatgct tcaataatat tgaaaaagga agagtatgag tattcaacat	240<br>
ttccgtgtcg cccttattcc cttttttgcg gcattttgcc ttcctgtttt tgctcaccca	300<br>
gaaacgctgg tgaaagtaaa agatgctgaa gatcagttgg gtgcacgagt gggttacatc	3 60<br>
gaactggatc tcaacagcgg taagatcctt gagagttttc gccccgaaga acgttttcca	420<br>
atgatgagca cttttaaagt tctgcCatgt ggcgcggtat tatcccgtat tgacgccggg	480<br>
caagagcaac tcggtcgccg catacactat tctcagaatg acttggttga gtactcacca	540<br>
gtcacagaaa agcatcttac ggatggcatg acagtaagag aattatgcag tgctgccata	600<br>
accatgagtg ataacactgc ggccaactta cttctgacaa cgatcggagg accgaaggag	660<br>
ctaaccgctt ttttgcacaa catgggggat catgtaactc gccttgatcg ttgggaaccg	720<br>
gagctgaatg aagccatacc aaacgacgag cgtgacacca cgatgcctgt agcaatggca	780<br><br>
scaacgttgc gcaaactatt aactggcgaa ctacttactc tagcttcccg gcaacaatta	840<br>
atagactgga tggaggcgga taaagttgca ggaccacttc tgcgctcggc ccttccggct	900<br>
ggctggttta ttgctgataa atctggagcc ggtgagcgtg ggtctcgcgg tatcattgca	960<br>
gcactggggc cagatggtaa gccctcccgt atcgtagtta tctacacgac ggggagtcag	1020<br>
gcaactatgg atgaacgaaa tagacagatc gctgagatag gtgcctcact gattaagcat	1O80<br>
tggtaactgt cagaccaagt ttactcatat atactttaga ttgatttaaa acttcatttt	1140<br>
taatttaaaa ggatctaggt gaagatcctt tttgataatc tcatgaccaa aatcccttaa	1200<br>
cgtgagtttt cgttccactg agcgtcagac cccgtagaaa agatcaaagg atcttcttga	1260<br>
gatccttttt ttctgcgcgt aatctgctgc ttgcaaacaa aaaaaccacc gctaccagcg	1320<br>
gtggtttgtt tgccggatca agagctacca actctttttc cgaaggtaac tggcttcagc	1380<br>
agagcgcaga taccaaatac tgtccttcta gtgtagccgc agttaggcca ccacttcaag	1440<br>
aactctgtag caccgcctac atacctcgct ctgctaatcc tgttaccagt ggctgctgcc	1500<br>
agtggcgata agtcgtgtct taccgggttg gactcaagac gatagttacc ggataaggcg-	1560<br>
cagcggtcgg gctgaacggg gggttcgtgc acacagccca gcttggagcg aacgacctac	1620<br>
accgaactga gatacctaca gcgtgagcta tgagaaagcg ccacgcttcc cgaagggaga	1680<br>
aaggcggaca ggtatccggt aagcggcagg gtcggaacag gagagcgcac gagggagctt	1740<br>
ccagggggaa acgcctggta tctttatagt cctgfcgggt ttcgccacct ctgacttgag	1800<br>
cgtcgatttt cgtgatgctc gtcagggggg cggagcctat ggaaaaacgc cagcaacgcg	1860<br>
gcctttttac ggttcctggc cttttgctgg ccttttgctc acatgttctt tcctgcgtca	1920<br>
tcccctgatt ctgtggataa ccgtattacc gcctttgagt gagctgatac cgctcgccgc	1980<br>
agccgaacga ccgagcgcag cgagtcagtg agcgaggaag cggaagagcg cccaatacgc	2040<br>
aaaccgcctc cccccgcgcg ttggccgatt cattaatgca gctggcacga caggtttccc	2100<br>
gactggaaag cgggcagtga gcgcaacgca attaatgtga gttagctcac tcattaggca	2160<br>
ccccaggctt cacactttat gcttccggct cgtatgttgt gtggaattgt gagcggataa	2220<br>
caatttcaca caggaaacag ctatgaccat gattacgcca agcttgaatt ctctgacgct	2280<br>
gattaacccg acaccctatt acctgacggt aacagagttg aatgccggaa cccgggttct	2340<br>
tgaaaatgca ttggtgcctc caatgggcga aagcacggtt aaattgcctt ctgatgcagg	2400<br>
aagcaatatt acttaccgaa caataaatga ttatggcgca cttaccccca aaatgacggg	2460<br>
cgtaatggaa taacgcaggg ggaatttttc gcctgaataa aaagaattga ctgccggggt	2520<br>
gattttaagc cggaggaata atgtcatatc tgaatttaag actttaccag cgaaacacac	2580<br>
aatgcttgca tattcgtaag catcgtttgg ctggtttttt tgtccgactc gttgtcgcct	2540<br>
gtgcttttgc cgcacaggca cctttgtcat ctgccgacct ctattttaat ccgcgctttt	2700<br><br>
tagcggatga tccccaggct gtggccgatt tatcgcgttt tgaaaatggg caagaattac	2760<br>
cgccagggac gtatcgcgtc gatatctatt tgaataatgg ttatatggca acgcgtgatg	2820<br>
tcacatttaa tacgggcgac agtgaacaag ggattgttcc ctgcctgaca cgcgcgcaac	2880<br>
tcgccagtat ggggctgaat acggcttctg tcgccggtat gaatctgctg gcggatgatg	2940<br>
cctgCgtgcc attaaccaca atggtccagg acgctactgc gcatctggat gttggtcagc	3000<br>
agcgactgaa cctgacgatc cctcaggcat ttatgagtaa tcgcgcgcgt ggttatattc	3060<br>
ctcccgagtt atgggatccc ggtattaatg ccggattgct caattataat ttcagcggaa	3120<br>
atagtgtaca gaatcggatt gggggtaaca gccattatgc atatttaaac ctacagagtg	3180<br>
ggttaaatat tggtgcgtgg cgtttacgcg acaataccac ctggagttat aacagtagcg	3240<br>
acagatcatc aggtagcaaa aataaatggc agcatatcaa tacctggctt gagcgagaca	3300<br>
taataccgtt acgttcccgg ctgacgctgg gtgatggtta tactcagggc gatattttcg	33 60<br>
atggtattaa ctttcgcggc gcacaattgg cctcagatga caatatgtta cccgatagtc	3420<br>
aaagaggatt tgccccggtg atccacggta ttgctcgtgg taotgcacag gtcactatta	3480<br>
aacaaaatgg gtatgacatt tataatagta cggtgccacc ggggcctttt accatcaacg	3540<br>
atatctatgc cgcaggtaat agtggtgact tgcaggtaac gatcaaagag gctgacggca	3600<br>
gcacgcagat ttttaccgta ccctattcgt cagtcccgct tttgcaacgt gaagggcata	3660<br>
ctcgttattc cattacggca ggagaatacc gtagtggaaa tgcgcagcag gaaaaaaccc	3720<br>
gctttttcca gagtacatta ctccacggcc ttccggctgg ctggacaata tatggtggaa	3780<br>
cgcaactggc ggatcgttat cgtgctttta atctcggtat cgggaaaaac atgggggcac	3840<br>
tgggcgctct gtctgtggat atgacgcagg ctaattccac acttcccgat gacagtcagc	3900<br>
atgacggaoa atcggtgcgt tttctctata acaaatcgct caatgaatca ggcacgaata	3960<br>
ttcagttagt gggttaccgt tattcgacca gcggatattt taatttcgct gatacaacat	4020<br>
acagtcgaat gaatggctac aacattgaaa cacaggacgg agttattcag gttaagccga	4080<br>
aattcaccga ctattacaac ctcgcttata acaaacgcgg gaaattacaa ctcaccgtta	4140<br>
ctcagcaact cgggcgcaca tcaacactgt atttgagtgg tagccatcaa acttattggg	4200<br>
gaacgagtaa tgtcgatgag caattccagg ctggattaaa tactgcgttc gaagatatca	4260<br>
actggacgct cagctatagc ctgacgaaaa acgcctggca aaaaggscgg gatcagatgt	4320<br>
tagcgcttaa cgtcaatatt cctttcagcc actggctgcg ttctgacagt aaatctcagt	4380<br>
ggcgacatgc cagtgccagc tacagcatgt cacacgatct caacggtcgg atgaccaatc	4440<br>
tggctggtgt atacggtacg ttgctggaag aoaacaacct cagctatagc gtgcaaaccg	4500<br>
gctatgccgg gggaggcgat ggaaatagcg gaagtacagg ctacgccacg ctgaattatc	4560<br>
goggtggtta cggcaatgcc aatatcggtt acagccatag cgatgatatt aagcagctct	4620<br><br>
attacggagt  cagcggtggg	gtactggctc aCgccaatgg	cgtaacgctg gggcagccgt	4680<br>
taaacgatac ggtggtgctt	gttaaagcgc  ctggcgcaaa	agatgcaaaa gtcgaaaacc	4740<br>
agacgggggt gcgtaccgac	tggcgtggtt  atgccgtgct	gccttatgcc  actgaatatc	4800<br>
gggaaaatag agtggcgctg	gataccaata  ccctggctga	taacgtcgat  ttagataacg	4B60<br>
cggttgctaa cgttgttccc	actcgtgggg cgatcgtgcg	agcagagttt  aaagcgcgcg	492 0<br>
ttgggataaa actgctcatg	acgctgaccc acaataataa	gccgctgccg tttggggcga	4980<br>
tggtgacatc agagagtagc	cagagtagcg gcattgttgc	ggataatggt caggtttacc	5040<br>
tcagcggaat gcctttagcg	ggaaaagttc aggtgaaatg	gggagaagag gaaaaCgctc	5100<br>
actgtgtcgc caattatcaa ctgccaccag agagtcagca  gcagttatta acccagctat       5160 cagctgaatg tcgttaaggg ggcgtgatga gaaacaaacc  tttttatctt  ctgtgcgctt       5220<br>
ttttgtggct  ggcggtgagt	cacgctttgg  ctgcggatag	cacgattact  atccgcggtc	5280<br>
tagaggatcc ccgggtaccg	agctcgaatt  cactggccgt	cgttttacaa cgtcgtgact	5340<br>
gggaaaaccc tggcgttacc	caacttaatc gccttgcagc	acatccccct ttcgccagct	5400<br>
ggcgtaatag  cgaagaggcc	cgcaccgatc  gcccttccca	acagttgcgc agcctgaatg	5460<br>
gcgaatggcg cctgatgcgg	tattttctcc  ttacgcatct	gtgcggtatt tcacaccgca	5520<br>
tatggtgcac tctcagtaca	atctgctctg atgccgcata	gttaagccag ccccgacacc	5580<br>
cgccaacacc  cgctgacgcg	ccctgacggg  cttgtctgct	cccggcatcc gcttacagac	5640<br>
aagctgtgac cgtctccggg	agctgcatgt gtcagaggtt	t	5681<br>	202<br>	40<br>	DNA<br>	Oligonucleotide Primer<br>  202<br>
aattacgtga gcaagcttat gagaaacaaa cctttttatc	40<br>	203<br>	41<br>	DKA<br>	Oligonucleotide Primer<br>  203<br>
gactaaggcc tttctagatt attgataaac aaaagtcacg c	41<br>	204<br>	4637<br>	DNA<br>	pFIMFGH<br><br>  204<br>
aaagggcctc gtgatacgcc tatttttata ggttaatgtc atgataataa tggtttctta	60<br>
gacgtcaggt ggcacttttc ggggaaatgt gcgcggaaco cctatttgtt tatttttcta	120<br>
aatacattca aatatgtatc cgctcatgag acaataaccc tgataaatgc ttcaataata	180<br>
ttgaaaaagg aagagtatga gtattcaaca tttccgtgtc gcccttattc ccttttttgc	240<br>
ggcattttgc cttcctgttt ttgctcaccc agaaacgctg gtgaaagtaa aagatgctga	300<br>
agatcagttg ggtgcacgag tgggttacat cgaactggat ctcaacagcg gtaagatcct	3 60<br>
tgagagtttt cgccccgaag aacgttttcc aatgatgagc acttttaaag ttctgctatg	420<br>
tggcgcggta ctatcccgta ttgacgccgg gcaagagcaa cccggtcgcc gcatacacta	480<br>
ttctcagaat gacttggttg agtactcacc agtcacagaa aagcatctca cggatggcat	540<br>
gacagtaaga gaattatgca gtgctgccat aaccatgagt gataacactg cggccaactt	600<br>
acttctgaca acgatcggag gaccgaagga gctaaccgct tttttgcaca acatggggga	660<br>
tcatgtaact cgccttgatc gttgggaacc ggagctgaat gaagccatac caaacgacga	720<br>
gcgtgacacc acgatgcctg tagcaatggc aacaacgttg cgcaaactat taactggcga	780<br>
actacttact ctagcttccc ggcaacaatt aatagactgg atggaggcgg ataaagttgc	840<br>
aggaccactt ctgcgctcgg cccttccggc tggctggttt attgctgata aatctggagc	300<br>
cggtgagcgt gggtctcgcg gtatcattgc agcactgggg ccagatggta agccctcccg	960<br>
tatcgtagtt atctacacga cggggagtca ggcaactatg gatgaacgaa atagacagat	1020<br>
cgctgagata ggtgcctcac tgattaagca ttggtaactg tcagaccaag tttactcata	1080<br>
tatactttag attgatttaa aacttcattt ttaatttaaa aggatctagg tgaagatcct	1140<br>
ttttgataat ctcatgacca aaatccctta acgtgagttt tcgttccact gagcgtcaga	1200<br>
ccccgcagaa aagatcaaag gatcttcttg agatcctttt tttctgcgcg taatctgctg	1260<br>
cttgcaaaca aaaaaaccac cgctaccagc ggtggtttgt ttgccggatc aagagctacc	1320<br>
aactcttttt ccgaaggtaa ctggcttcag cagagcgcag ataccaaata ctgtccttct	1380<br>
agtgtagccg tagttaggcc accacttcaa gaactctgta gcaccgccta catacctcgc	1440<br>
tctgctaatc ctgttaccag tggctgctgc cagtggcgat aagtcgtgtc ttaccgggtt	1500<br>
ggactcaaga cgatagttac cggataaggc gcagcggtcg ggctgaacgg ggggttcgtg	1560<br>
cacacagccc agcttggagc gaacgaccta caccgaactg agatacctac agcgtgagct	1620<br>
atgagaaagc gccacgcttc ccgaagggag aaaggcggac aggtatccgg taagcggcag	1680<br>
ggtcggaaca ggagagcgca cgagggagct tccaggggga aacgcctggt atctttatag	1740<br>
tcctgtcggg tttcgccacc tctgacttga gcgtcgattt ttgtgatgct cgtcaggggg	1800<br>
gcggagccta tggaaaaacg ccagcaacgc ggccttttta cggttcctgg ccttttgctg	i860<br><br>
gccttttgct cacatgttct  ttcctgcgtt atcccctgat  tctgtggata accgtattac	1920<br>
cgcctttgag tgagctgata ccgctcgccg cagccgaacg accgagcgca gcgagtcagt	1980<br>
gagcgaggaa gcggaagagc gcccaatacg caaaccgcct  ctccccgcgc gttggccgat	2040<br>
tcattaatgc agctggcacg acaggtttcc  cgactggaaa gcgggcagtg agcgcaacgc	2100<br>
aattaatgtg agttagctca ctcattaggc  accccaggct  ttacacttta tgcttccggc	2160<br>
tcgtatgttg tgtggaattg tgagcggata acaatttcac  acaggaaaca gctatgacca	2220<br>
tgattacgcc aagcttatga gaaacaaacc  tttttatctt ctgtgcgctt ttttgtggct	2280<br>
ggcggtgagt cacgctttgg ctgcggatag cacgattact atccgcggct atgtcaggga	2340<br>
taacggctgt  agtgtggccg ctgaatcaac  caattttact  gttgatctga tggaaaacgc	2400<br>
ggcgaagcaa tttaacaaca  ttggcgcgac  gactcctgtt   gttccatttc gtattttgct	2460<br>
gtcaccctgt  ggtaatgccg tttctgccgt   aaaggttggg  tttactggcg ttgcagatag	2520<br>
ccacaatgcc aacctgcttg cacttgaaaa tacggtgtca gcggcttcgg gactgggaat	2580<br>
acagcttctg aatgagcagc aaaatcaaat accccttaat gctccatcgt ccgcgctttc	2640<br>
gtggaegacc ctgacgccgg gtaaaccaaa  tacgctgaat  ttttacgccc ggctaatggc	2700<br>
gacacaggtg cctgtcactg cggggcatat   caatgccacg gctaccttca ctcttgaata	2760<br>
tcagtaactg gagatgctca tgaaatggtg caaacgtggg tatgtattgg cggcaatatt	2820<br>
ggcgctcgca agtgcgacga  tacaggcagc  cgatgtcacc atcacggtga acggtaaggt	2880<br>
cgtcgccaaa ccgtgtacgg tttccaccac  caatgccacg gttgatctcg gcgatcttta	2940<br>
ttctttcagt cttatgtctg ccggggcggc  atcggcctgg catgatgttg cgcttgagtt	3000<br>
gactaattgt ccggtgggaa cgtcgagggt cactgccagc ttcagcgggg cagccgacag	3 060<br>
taccggatat  tataaaaacc aggggaccgc  gcaaaacatc  cagttagagc tacaggatga	3120<br>
cagtggcaac acattgaata ctggcgcsac  caaaacagtt caggtggatg attcctcaca	3180<br>
atcagcgcac ttcccgttac aggtcagagc  attgacagta aatggcggag ccactcaggg	3240<br>
aaccattcag gcagtgatta gcatcaccta tacctacagc  tgaacccgaa gagatgattg	3300<br>
taatgaaacg agttattacc  ctgtttgctg  tactgctgat   gggctggtcg gtaaatgcct	3360<br>
ggtcattcgc  ctgtaaaacc  gccaatggta  ccgctatccc   tattggcggt  ggcagcgcca	3420<br>
atgtttatgt aaaccttgcg cccgtcgtga atgtggggca aaacctggtc gtggatcttt	3480<br>
cgacgcaaat cttttgccat aacgattatc cggaaaccat  tacagactat gtcacactgc	3540<br>
aacgaggctc  ggcttatggc  ggcgtgttat   ctaatttttc   cgggaccgta aaatatagtg	3 500<br>
gcagtagcta tccatttcct  accaccagcg  aaacgccgcg  cgttgtttat  aattcgagaa	3 560<br>
cggataagcc gtggccggtg gcgctttatt  tgacgcctgt gagcagtgcg ggcggggtgg	3720<br>
cgattaaagc  tggctcatta attgccgtgc  ttattttgCg acagaccaac aactataaca	3780<br><br>
gcgatgattt ccagtttgtg tggaatattt acgccaataa tgatgtggtg gtgcctactg	3840<br>
gcggctgcga tgtttctgct cgtgatgtca ccgttactct gccggactac cctggttcag	3900<br>
tgccaattcc tcttaccgtt tattgtgcga aaagccaaaa cctggggtat tacctctccg	395"0<br>
gcacaaccgc agatgcgggc aactcgattt tcaccaatac cgcgtcgttt tcacctgcac	402D<br>
agggcgtcgg cgtacagttg acgcgcaacg gtacgattat tccagcgaat aacacggtat	4080<br>
cgttaggagc atagggact tcggcggtga gtctgggatt aacggcaaat tatgcacgta	4140<br>
ccggagggca ggtgactgca gggaatgtgc aatcgattat tggcgtgact tttgtttatc	4200<br>
aataatctag aggatccccg ggtaccgagc tcgaattcsc tggccgtcgt tttacaacgt	4260<br>
cgtgactggg aaaaccctgg cgttacccaa cttaatcgcc ttgcagcaca tccccctttc	4320<br>
gccagctggc gtaatagcga agaggcccgc accgatcgcc cttcccaaca gttgcgcagc	4380<br>
ctgaatggcg aatggcgcct gatgcggtat tttctcctta cgcatctgtg cggtatttca	4440<br>
caccgcatat ggtgcactct cagtacaatc tgctctgatg ccgcatagtt aagccagccc	45D0<br>
cgacacccgc caacacccgc tgacgcgocc tgacgggctt gtctgctccc ggcatccgct	4560<br>
tacagacaag ctgtgaccgt ctccgggagc tgcatgtgtc agaggttttc accgtcatca	4620<br>
ccgaaacgcg cgagacg	4637<br>  205<br>  9299 â¢<br> DKa<br> pFIMAICDFGH<br>  205<br>
cgagacgaaa gggcctcgtg atangcctat tttcataggt taatgtcatg ataataatgg	60<br>
tttcttagac gtcaggtggc acttttcggg gaaatgtgcg cggaacccct atttgtttat	120<br>
ttttctaaat acattcaaat atgtatccgc tcatgagaca ataaccctga taaatgcttc	180<br>
aataatattg aaaaaggaag agtatgagta ttcaacattt ccgtgtcgcc cttattccct	240<br>
tttttgcggc attttgcctt cctgtttttg ctcacccaga aacgctggtg aaagtaaaag	300<br>
atgctgaaga tcagttgggt gcacgagtgg gttacatcga actggatctc aacagcggta	360<br>
agatccttga gagttttcgc cccgaagaac gttttccaat gatgagcact tttaaagttc	420<br>
tgctatgtgg cgcggtatta tcccgtattg acgccgggca agagcaactc ggtcgccgca	480<br>
tacactsttc tcagaatgac ttggttgagt actcaccagt cacagaaaag catcttacgg	540<br>
atggcatgac agtaagagaa ttatgcagtg ctgccataac catgagtgat aacactgcgg	600<br>
ccaacttBCt tctgacaacg atcggaggac cgaaggagct aaccgctttt ttgcacaaca	660<br><br>
tgggggatca tgtaactcgc cttgatcgtt gggaaccgga gctgaatgaa gccatacoaa	720<br>
acgacgagcg tgacaccacg atgcctgtag caatggcaac aacgttgcgc aaactattaa	780<br>
ctggcgaact acttactcta gcttcccggc aacasttast agactggatg gaggcggata	840<br>
aagttgcagg accacttctg cgctcggccc ttccggctgg ctggtttatt gctgataaat	900<br>
ctggagccgg Cgagcgtggg tctcgcggta tcattgcagc actggggcca gatggtaagc	960<br>
cctcccgtat cgtagttatc tacacgacgg ggagtcaggc aactat:ggat gaacgaaata	1020<br>
gacagatogc tgagataggt gcctcactga tcaageattg gtaacCgtca gaccaagttt	1080<br>
actcatatat actttagatt gatttaaaac ttcattttta atttaaaagg atctaggtga	1140<br>
agatcctttt cgataatctc atgaccaaaa tcccttaacg tgagttttcg ttccactgag	1200<br>
cgtcagaccc cgtagaaaag atcaaaggat cCtcttgaga tccttttttc ctgcgcgtaa	12G0<br>
tctgctgctt gcaaacaaaa aaaccaccgc taccagcggt ggtttgtttg ccggatcaag	1320<br>
agctaccaac tctttttccg aaggtaactg gcttcagcag agcgcagata ccaaatactg	1380<br>
tccttctagt gtagccgtag ttaggccacc acttcaagaa ctctg-agca ccgcctacat	1440<br>
acctcgctct gctaatcctg ttaccagtgg ctgctgccag tggcgataag tcgtgtctta	1500<br>
ccgggttgga ctcaagacga tagttaccgg ataaggcgca gcggtcgggc tgaacggggg	1560<br>
gttcgtgcac acagcccagc ttggagcgaa cgacctacac cgaactgaga tacctacagc	1620<br>
gtgagctatg agaaagcgcc acgcttcccg aagggagaaa ggcggacagg tatccggtaa	1680<br>
gcggcagggt cggaacagga gagcgcacga gggagcttcc agggggaaac gcctggtatc	1740<br>
tttatagtcc tgtcgggttt cgccacctct gacttgagcg tcgatttttg tgstgctcgt	1800<br>
caggggggcg gagcctatgg aaaaacgcca gcaacgcggc ctttttacgg ttcctggcct	1860<br>
tttgctggcc ttttgctcac atgttctttc ctgcgttatc ccctgattct gtggataacc	1920<br>
gtattaccgc ctttgagtga gctgataccg ctcgccgcag ccgaacgacc gagcgcagcg	1980<br>
agtcagtgag cgaggaagcg gaagagcgcc caatacgcaa accgcctctc cccgcgcgtt	2040<br>
ggccgattca ttaatgcagc tggcacgaca ggtttcccga ctggaaagcg ggcagtgagc	2100<br>
gcaacgcaat taatgtgagt tagctcactc attaggcacc ccaggcttta cactttatgc	2160<br>
ttccggctcg tatgttgtgt ggaattgtga gcggataaca atttcacaca ggaaacagct	2220<br>
atgaccatga ttacgccaag cttataatag aaatagtttt ttgaaaggaa agcagcatga	2280<br>
aaattaaaac tctggcaatc gttgttctgt cggctctgtc cctcagttct acagcggctc	2340<br>
tggocgctgc cacgacggtt aatggcggga ccgttcactt taaaggggaa gttgtt;aacg	2400<br>
ccgcttgcgc agttgatgca ggctctgttg atcaaaccgt tcagttagga caggttcgta	2460<br>
ccgcatcgct ggcacaggaa ggagcaacca gttctgctgt cggttttaac attcagctga	2520<br>
atgattgcga taccaatgtt gcatctaaag ccgctgttgc ctttttaggt acggcgattg	2580<br><br>
atgcgggtca taccaacgtt  ctggctctgc agagttcagc tgcgggtagc gcaacaaacg	2S40<br>
ttggtgtgca gatcctggac agaacgggtg ctgcgctgac gctggatggt gcgacattta	2700<br>
gttcagaaac aaccctgaat aacggaacca ataccattcc gttccaggcg cgttattttg	2760<br>
caaccggggc cgcaaccccg ggtgctgcta atgcggatgc gaccttcaag gttcagtatc	2820<br>
aataacctac ccaggttcag ggacgtcatt acgggcaggg atgcccaccc ttgtgcgata	2880<br>
aaaataacga tgaaaaggaa gagattattt ctattagcgt cgttgctgcc aatgtttgct	2940<br>
ccggccggaa ataaatggaa taccacgttg cccggcggaa atatgcaatt tcagggcgtc	3000<br>
attattgcgg aaacttgccg gattgaagcc ggCgataaac aaatgacggt caatatgggg	3060<br>
csaatcagca gtaaccggtt tcatgcggtt ggggaagata gcgcaccggt gccttttgtt	3120<br>
aCtcatttac gggaatgtag cacggtggtg agtgaacgtg taggtgtggc gtttcacggt	3180<br>
gtcgcggatg gtaaaaatcc ggatgtgctt  tccgtgggag aggggccagg gatagccacc	3240<br>
aatattggcg tagcgttgtt tgatgatgaa ggaaacctcg taccgattaa tcgtcctcca	3300<br>
gcaaactgga aacggcttta ttcaggctct acttcgctac atttcatcgc caaatatcgt	3360<br>
gctaccgggc  gtcgggttac  tggcggcatc  gccaatgccc   aggcctggtt ctctttaacc	3420<br>
tstcagtaat  tgttcagcag ataatgtgat aacaggaaca ggacagtgag taataaaaac	3480<br>
gtcaatgtaa ggaaatcgca ggaaataaca ttctgcttgc tggcaggtat cctgatgttc	3540<br>
atggcaatga tggttgccgg acgcgctgaa gcgggagtgg ccttaggtgc gactcgcgta	3600<br>
atttatccgg cagggcaaaa acaagagcaa cttgccgtga caaataatga tgaaaatagt	366O<br>
acctatttaa ttcaatcatg ggtggaaaat gccgatggtg taaaggatgg tcgttttatc	3720<br>
gtgacgcctc ctctgtttgc gatgaaggga aaaaaagaga ataccttacg tattcttgat	3780<br>
gcaacaaata accaattgcc  acaggaccgg gasagtttat   tctggatgaa cgttaaagcg	3840<br>
attccgtcaa tggataaatc  aaaattgact gagaatacgc  tacagctcgc aattatcagc	3900<br>
cgcattaaac tgtactatcg cccggctaaa ttsgcgttgc cacccgatca ggccgcagaa	3950<br>
asattaagat  ttcgtcgtag cgcgaattct  ctgacgctga  ttaacccgac accctattac	4020<br>
ctgacggtaa cagagttgaa tgccggaacc cgggttcttg aaaatgcatt ggtgcctcca	4O8O<br>
atgggcgaaa gcacggttaa attgccttct gatgcaggaa gcaatattac  ttaccgaaca	4140<br>
ataaatgatt atggcgcact tacccccaaa atgacgggcg taatggaata acgcaggggg	4200<br>
aatttttcgc ctgaataaaa agaattgact gccggggtga ttttaagccg gaggaataat	4260<br>
gtcatatctg aatttaagac  tttaccagcg aaacacacaa tgcttgcata ttcgtaagca	4320<br>
tcgtttggct ggtttttttg tccgactcgt tgtcgcctgt gcttttgccg cacaggcacc	4380<br>
tttgtcatct gccgacctct attttaatcc gcgcttttta gcggatgatc cccaggctgt	4440<br>
ggccgattta tcgcgttttg aaaatgggca agaattaccg ccagggacgt atcgcgtcga	4500<br><br>
tatctatttg aataatggct atatggcaac  gcgtgatgtc  acatttaata cgggcgacag	4560<br>
tgaacaaggg attgttccct gcctgacacg  cgcgcaactc  gccagtatgg ggctgaatac	4620<br>
ggcttctgtc gccggtatga atctgctggc  ggatgatgcc  tgtgtgccat taaccacaat	4680<br>
ggtccaggac gctactgcgc atctggatgt  tggtcagcag cgactgaacc tgacgatccc	4740<br>
tcaggcattt  atgagtaatc gcgcgcgtgg  ttatattcct  cctgagttat gggatcccgg	4300<br>
tattaatgcc ggattgctca attataattt  cagcggaaat  agtgtacaga atcggattgg	4850<br>
gggtaacagc cattatgcat atttaaacct  acagagtggg ttaaatattg gtgcgtggcg	4920<br>
tttacgcgac aataccacct ggagttataa cagtagcgac agatcatcag gtagcaaaaa	4980<br>
taaatggcag catatcaata cctggcttga gcgagacata ataccgttac gttcccggct	5040<br>
gacgctgggt  gatggttata ctcagggcga   tattttcgat   ggtattaact  ttcgcggcgc	5100<br>
acaattggcc  tcagatgaca atatgttacc  cgatagtcaa agaggatttg ccccggtgat	5160<br>
ccacggtatt gctcgtggta ctgcacaggt  cactattaaa caaaatgggt atgacattta	5220<br>
taatagtacg gtgccaccgg ggccttttac  catcaacgat  atctatgccg caggtaatag	5260<br>
tggtgacttg caggtaacga tcaaagaggc   tgacggcagc   acgcagattt   ttaccgtacc	5340<br>
ctattcgtca gtcccgcttt  tgcaacgtga agggcatact  cgttattcca ttacggcagg	5400<br>
agaatacogt agtggaaatg cgcagcagga aaaaacccgc  tttttccaga gtacattact	5460<br>
ccacggcctt ccggctggct ggacaatata  tggtggaacg caactggcgg atcgttatcg	5520<br>
tgcttttaat ttcggtatcg ggaaaaacat  gggggcactg ggcgctctgt ctgtggatat	5580<br>
gacgcaggct aattccacac  ttcccgatga cagtcagcat gacggacaat cggtgcgttt	5640<br>
tctctataac aaatcgctca atgaatcagg cacgaatatt  cagttagtgg gttaccgtta	5700<br>
ttcgaccagc ggatatttta atttcgctga  tacaacatac agtcgaatga atggctacaa	5760<br>
cattgaaaca caggacggag ttattcaggt  taagccgaaa ttcaccgact  attacaacct	5820<br>
cgcttataac aaacgcggga aattacaact  caccgttact  cagcaactcg ggcgcacatc	5880<br>
aacactgtat ttgagtggta gccatcaaac  ttattgggga acgagtaatg tcgatgagca	5940<br>
attccaggct ggattaaata ctgcgttcga  agatatcaac  tggacgctca gctatagcct	6000<br>
gacgaaaaac gcctggcaaa aaggacggga  tcagatgtta gcgcttaacg tcaatattcc	6060<br>
tttcagccac tggctgcgtt ctgacagtaa  atctcagtgg cgacatgcca gtgccagcta	6120<br>
cagcatgtca cacgatctca acggtcggat  gaccaatctg gctggtgtat acggtacgtt	6180<br>
gctggaagac aacaacctca  gctatagcgt   gcaaaccggc  tatgccgggg gaggcgatgg	6240<br>
aaatagcgga agtacaggct acgccaggct gaattatcgc  ggtggttacg gcaatgccaa	6300<br>
tatcggttac agccatagcg atgatattaa gcagctctat tacggagtca gcggtggggt	6360<br>
actggctcat gccaatggcg taacgctggg gcagccgtta aacgatacgg tggtgcttgt	6420<br><br>
taaagcgcct ggcgcaaaag atgcaaaagt cgaaaaccag acgggggtgc gtaccgactg	6480<br>
gcgtggttst  gccgtgctgc cttatgccac  tgaatatcgg gaaaatagag tggcgctgga	6540<br>
taccaatacc  ctggctgata acgtcgattt  agataacgcg gttgctaacg ttgttcccac	6600<br>
tcgtggggcg atcgtgcgag cagagtttaa agcgcgcgtt gggataaaac tgctcatgac	6560<br>
gctgacccac aataataagc cgctgccgtt  tggggcgatg gtgacatcag agagtagcca	6720<br>
gagtagcggc attcpttgcgg ataatggtca ggtttacctc agcggaatgc ctttagcggg	6780<br>
aaaagttcag gtgaaatggg gagaagagga aaatgctcac tgtgtcgcca attatcaact	6840<br>
gccaccagag  agtcagcagc agttattaac  ccagctatca  gctgaatgtc  gttaaggggg	6900<br>
cgtgatgaga aacaaacctt tttatcttct gtgcgctttt  ttgtggctgg cggtgagtca	6960<br>
cgctttggct gcggatagca cgattactat ccgcggctat gtcagggata acggctgtag	7020<br>
tgtggccgct  gaatcaacca attttactgt  tgatctgatg gaaaacgcgg cgaagcaatt	7080<br>
taacaacatt ggcgcgacga ctcctgttgt tccatttcgt attttgctgt caccctgtgg	7140<br>
taatgccgtt  tctgccgtaa aggttgggtt  tactggcgtt gcagatagcc acaatgccaa	7200<br>
cctgcttgca  cttgaaaata cggtgtcagc  ggcttcggga  ctgggaatac  agcttctgaa	7260<br>
tgagcagcaa aatcaaatac cccttaatgc tccatcgtcc gcgctttcgt ggacgaccct	7320<br>
gacgccgggt aaaccaaata cgctgaattt ttacgcccgg ctaatggcga cacaggtgcc	7380<br>
tgtcactgcg gggcatatca atgccacggc  taccttcact cttgaatatc agtaactgga	7440<br>
gatgctcatg aaatggtgca aacgtgggta  tgtattggcg gcaatattgg cgctcgcaag	7500<br>
tgcgacgata caggcagccg atgtcaccat cacggtgaac ggtaaggtgg tcgccaaacc	7560<br>
gtgtacggtt  tccaccacca atgccacggt  tgatctcggc gatctttatt  ctttcagtct	7620<br>
tatgtctgcc ggggcggcat cggcctggca tgatgttgcg cttgagttga ctaattgtoc	7680<br>
ggtgggaacg tcgagggtca ctgccagctt cagcggggca gccgacagta ccggatatta	7740<br>
taaaaaccag gggaccgcgc aaaacatcca gttagagcta caggatgaca gtggcaacac	7800<br>
attgaatact  ggcgcaacca aaacagttca ggtggatgat  tcctcacaat  cagcgcactt	7860<br>
cccgttacag gtcagagcat tgacagtaaa tggcggagcc actcagggaa ccattcaggc	7920<br>
agtgattagc atcacctata cctacagctg aacccgaaga gatgattgta atgaaacgag	7980<br>
ttattaccct gtttgctgta ctgctgatgg gctggtcggt aaatgcctgg tcattcgcct	8040<br>
gtaaaaccgc caatggtacc gctatcccta ttggcggtgg cagcgccaat gtttatgtaa	8100<br>
accttgcgcc   cgtcgtgaat gtggggcaaa acctggtcgt  ggatctttcg  acgcaaatct	8160<br>
tttgccataa  cgattatccg gaaaccatta  cagactatgt  cacactgcaa  cgaggctcgg	8220<br>
cttatggcgg cgtgttatct aatttttccg ggaccgtaaa atatagtggc agtagctatc	8280<br>
catttcctac caccagcgaa acgccgcgcg ttgtttataa ttcgagaacg gataagccgt	8340<br><br>
ggccggtggc gctttatttg acgcctgtgs gcagtgcggg cggggtggcg attaaagctg	8400<br>
gctcattaat tgccgtgctt attttgcgac agaccaacaa ctataacagc gatgatttcc	8460<br>
agtttgtgtg gaatatttac gccaataatg atgtggtggt gcctactggc ggctgcgatg	8520<br>
tttctgctcg tgatgtcacc gttactctgc cggactaccc tggttcagtg ccaattcctc	8580<br>
ttaccgttta ttgtgcgaaa agccaaaacc tggggtatta cctctccggc acaaccgcag	S640<br>
atgcgggcaa ctcgattttc accaataccg cgtcgttttc acctgcacag ggcgtcggcg	8700<br>
tacagttgac gcgcaacggt acgattattc cagcgaataa cacggtatcg ttaggagcag	8760<br>
tagggacttc ggcggtgagt ctgggattaa cggcaaatta tgcacgtacc ggagggcagg	6820<br>
tgactgcagg gaatgtgcaa tcgattattg gcgtgacttt tgtttatcaa taatctagaa	8880<br>
ggatccccgg gtaccgagct cgaattcact ggccgtcgtt ttacaacgtc gtgactggga	8940<br>
aaaccctggc gttacccaac ttaatcgcct tgcagcacat ccccctttcg ccagctggcg	9000<br>
taatagcgaa gaggcccgca ccgatcgccc ttcccaacag ttgcgcagcc tgaatggcga	9050<br>
atggcgcctg atgcggtatt ttctccttac gcatctgtgc ggtatttcac accgcatatg	9120<br>
gtgcactctc agtacaatct gctctgatgc cgcatagtta agccagcccc gacacccgcc	9180<br>
aacacccgct gacgcgccct gacgggcttg tctgctcccg gcatccgctt acagacaagc	9240<br>
tgtgaccgtc tccgggagct gcatgtgtca gaggttttca ccgtcatcac cgaaacgcg	9299<br>  206   8464   DMA   pFIMMCDFG<br>  206<br>
cgagacgaaa gggcctcgtg atacgcctat ttttataggt taatgtcatg ataataatgg	60<br>
tttcttagac gtcaggtggc acttttcggg gaaatgtgcg cggaacccct atttgtttat	120<br>
ttttctaaat acattcaaat atgtatccgc tcatgagaca ataaccctga taaatgcttc	180<br>
aataatattg aaaaaggaag agtatgagta ttcaacattt ccgtgtcgcc cttattccct	240<br>
tttttgcggc attttgcctt cctgtttttg ctcacccaga aacgctggtg aaagtaaaag	300<br>
atgctgaaga tcagttgggt gcacgagtgg gttacatcga actggatctc aacagcggta	360<br>
agatccttga gagttttcgc cccgaagaac gttttccaat gatgagcact tttaaagttc	420<br>
tgctatgtgg cgcggtatta tcccgtattg acgccgggca agagcaactc ggtcgccgca	480<br>
tacactattc tcagaatgac ttggttgagt actcaccagt cacagaaaag catcttacgg	540<br>
atggcatgac agtaagagaa ttatgcagtg ctgccataac catgagtgat aacactgcgg	600<br><br>
ccaacCtact  tctgacaacg atcggaggac  cgaaggagct  aaccgctttt  ttgcacaaca	560<br>
tgggggatca tgtaactcgc cttgatcgtt  gggaaccgga gctgaatgaa gccataccaa	720<br>
acgacgagcg tgacaccacg atgcctgtag  caatggcaac  aacgttgcgc  aaactattaa	780<br>
ctggcgaact acttaotcta gcttcccggc  aacaattaat agactggatg gaggcggata	840<br>
aagttgcagg accacttctg cgctcggccc  ttccggctgg ctggtttatt  gctgataaat	900<br>
ctggagccgg tgagcgtggg tctcgcggta  tcattgcagc actggggcca gatggtaagc	960<br>
cctcccgtat cgtagttatc tacacgacgg ggagtcaggc aactatggat gaacgaaata	1020<br>
gacagatcgc tgagataggt gcctcactga ttaagcattg gtaactgtca gaccaagttt	1080<br>
actcatatat  actttagatt gatttaaaac  ttcattttta atttaaaagg atctaggtga	1140<br>
agatcctttt  tgataatctc atgaccaaaa tcccttaacg tgagttttcg ttccactgag	12 DO<br>
cgtcagaccc cgcagaaaag atcaaaggat cttcttgaga tccttttttt  ctgcgcgtaa	1260<br>
tctgctgctt  gcaaacaaaa aaaccaccgc  taccagcggt ggtttgtttg ccggatcaag	1320<br>
agctaccaac  tctttttccg aaggtaactg gcttcagcag agcgcagata ccaaatactg	1380<br>
tccttctagt gtagccgtag ttaggccacc acttcaagaa ctctgtagca ccgcctacat	1440<br>
acctcgetct gctaatcctg ttaccagtgg ctgctgccag tggcgataag tcgtgtctta	1500<br>
ccgggttgga  ctcaagacga  tagttaccgg  ataaggcgca  gcggtcgggc   tgaacggggg	1650<br>
gttcgtgcac acagcccagc ttggagcgaa  cgacctacac cgaactgaga tacctacagc	1620<br>
gtgagctatg agaaagcgcc acgcttcccg aagggagaaa ggcggacagg tatccggtaa	1680<br>
gcggcagggt  cggaacagga gagcgcacga  gggagcttcc  agggggaaac  gcctggtatc	1740<br>
tttatagtcc tgtcgggttt cgccacctct gacttgagcg tcgatttttg tgatgctcgt	IBOO<br>
caggggggcg gagcctatgg aaaaacgcca gcaacgcggc ctttttacgg ttcctggcct	i860<br>
tttgctggcc ttttgctcac atgttctttc ctgcgttatc ccctgattct  gtggataacc	1920<br>
gtattaccgc ctttgagtga gctgataccg ctcgccgcag ccgaacgacc gagcgcagcg	1980<br>
agtcagtgag cgaggaagcg gaagagcgcc  caatacgcaa accgcctctc cccgcgcgtt	2040<br>
ggccgattca ttaatgcagc tggcacgaca ggtttcccga ctggaaagcg ggcagtgagc	2100<br>
gcaacgcaat  taatgtgagt tagctcactc  attaggcacc ccaggcttta cactttatgc	2i60<br>
ttccggctcg tatgttgtgt ggaattgtga gcggataaca atttcacaca ggaaacagct	2220<br>
atgaccatga ttacgccaag cttataatag aaatagtttt  ttgaaaggaa agcagcatga	2280<br>
aaattaaaac tctggcaatc gttgttctgt  cggctctgtc cctcagttct  acagcggctc	2340<br>
tggccgctgc cacgacggtt aatggtggga  ccgttcactt taaaggggaa gttgttaacg	2400<br>
ccgcttgcgc  agttgatgca ggctctgttg  atcaaaccgt  tcagttagga caggttcgta	2460<br>
ccgcatcgct ggcacaggaa ggagcaacca gttctgctgt cggttttaac attcagctga	2520<br><br>
atgattgcga taccaatgtt gcatctaaag ccgctgttgc ctttttaggt acggcgattg	2580<br>
atgcgggtca taccaacgtt ctggctctgc agagttcagc tgcgggtagc gcaacaaacg	2640<br>
ttggtgtgca gatcctggac agaacgggtg ctgcgctgac gctggatggt gcgacattta	2700<br>
gttcagaaac aaccctgaat aacggaacca ataccattcc gttccaggcg cgttattttg	2760<br>
caaccggggc cgcaaccccg ggtgctgcta atgcggatgc gaccttcaag gttcagtatc	2820<br>
aataacctac ccaggttcag ggacgtcatt acgggcaggg atgcccaccc ttgtgcgata	2880<br>
aaaataacga tgaaaaggaa gagattattt ctattagcgt cgttgctgcc aatgtttgct	2940<br>
ctggccggaa ataaatggaa taccacgttg cccggcggaa atatgcaatt tcagggcgtc	3o00<br>
attattgcgg aaacttgccg gattgaagcc ggtgataaac aaatgacggt caatatgggg	3o60<br>
caaatcagca gtaaccggtt tcatgcggtt ggggaagata gcgcaccggt gccttttgtt	3l20<br>
attcatttac gggaatgtag cacggtggtg agtgaacgtg taggtgtggc gtttcacggt	3180<br>
gtcgcggatg gtaaaaatcc ggatgtgctt tccgtgggag aggggccagg gatagccacc	3240<br>
aatattggcg tagcgttgtt tgatgatgaa ggaaacctcg taccgattaa tcgtcctcca	3300<br>
gcaaactgga aacggcttta ttcaggctct acttcgctac atttcatcgc caaatatcgt	3360<br>
gctaccgggc gtcgggttac tggcggcatc gccaatgccc aggcctggtt ctctttaacc	3420<br>
tatcagtaat tgttcagcag etaatgtgat aacaggaaca ggacagtgag taataaaaac	3430<br>
gtcaatgtaa ggaaatcgca ggaaataaca ttctgcttgc tggcaggtat cctgatgttc	3540<br>
atggcaatga tggttgccgg acgcgctgaa gcgggagtgg ccttaggtgc gactcgcgta	3600<br>
atttatccgg cagggcaaaa acaagagcaa cttgccgtga caaataatga tgaaaatagt	3660<br>
acctatttaa ttcaatcatg ggtggaaaat gccgatggtg taaaggatgg tcgttttatc	3720<br>
gtgacgcctc ctctgtttgc gatgaaggga aaaaaagaga ataccttacg tattcttgat	3780<br>
gcaacaaata accaattgcc acaggaccgg gaaagtttat tctggatgaa cgttaaagcg	3840<br>
attccgtcaa tggataaatc aaaattgact gagaatacgc tacagctcgc aattatcagc	3900<br>
cgcattaaac tgtactatcg cccggctaaa ttagcgttgc cacccgatca ggccgcagaa	3960<br>
aaattaagat ttcgtcgtag cgcgaattct ctgacgctga ttaacccgac accctattac	4020<br>
ctgacggtaa cagagttgaa tgccggaacc cgggttcttg aaaatgcatt ggtgcctcca	4080<br>
atgggcgaaa gcacggttaa attgccttct gatgcaggaa gcaatattac ttaccgaaca	4140_<br>
ataaatgatt atggcgcact tacccccaaa atgacgggcg taatggaata acgcaggggg	42 00<br>
aatttttcgc ctgaataaaa agaattgact gccggggtga ttttaagccg gaggaataat	4260<br>
gtcatatctg aatttaagac tttaccagcg aaacacacaa tgcttgcata ttcgtaagca	4320<br>
tcgtttggct ggtttttttg tccgactcgt tgtcgcctgt gcttttgccg cacaggcacc	4380<br>
tttgtcatct gccgacctct attttaatcc gcgcttttta gcggatgatc cccaggctgt	4440<br><br>
IQO	IDS	110<br>
Gly Arg Cys Leu His Tyr Thr Val Asp Lys Ser Lys Pro Lys Val Tyr<br>
115	120 ,	125<br>
Gin Trp Phe Asp Leu Arg Lys Tyr Ala Ala Ala Ser Gly Gly Cys Gly<br>
130	135	140<br>
Gly 145<br>	211<br>	17<br>	PRT<br>	Ce4iniiiiotope<br>  211<br>
Gly Glu Phe Cys lie Asn His Arg Gly Tyr Trp Val Cys Gly Asp Pro<br>
1	5	10      .	15<br>
Ala<br>	212<br>	27<br>	PRT<br>	Synthetic M2 Peptide<br>  212<br>
Ser Leu Leu Thr Glu Val Glu Thr Pro lie Arg Asn Glu Trp Gly Cys<br>
15	10	15<br>
Arg Cys Asn Gly Ser Ser Asp Gly Gly Gly Cys<br>
20	25<br>	213<br>	97<br>	PRT<br>	Matrix protein M2<br>  213<br>
Met ser Leu Leu Thr Glu Val Glu Thr Pro lie Arg Asn Glu Trp Gly<br>
15	10	15<br>
Cys Arg Cys Asn Gly Ser Ser Asp Pro Leu Ala lie Ala Ala Asn lie<br>
20	25	30<br>
lie Gly lie Leu His Leu lie Leu Trp lie Leu Asp Arg Leu Phe Phe<br>
35	40	45<br>
Lys Cya lie Tyr Arg Arg Phe Lys Tyr Gly Leu Lys Gly Gly Pro Ser<br>
50  .	55	60<br>
Thr Glu Gly Val Pro Lys Ser Met Arg Glu Glu Tyr Arg Lys Glu Gin<br>
65	70	75	80<br>
Gin Ser Ala Val Asp Ala Asp Asp Gly His Phe Val Ser lie Glu Leu<br>
85	90	95<br>
Glu<br><br>	214<br>	42<br>	DNA<br>	Oligonucleotide<br>     214<br>
taaccgaatt caggaggtaa aaacatatgg ctatcatcta cc	42<br>	215<br>	129<br>	PRT<br>	Bacteriophage f2<br>  215<br>
Ala Ser Asn Phe Thr Gin Phe Val Leu Val Asn Asp Gly Gly Thr Gly<br>
15	10	15<br>
Asn Val Thr Val Ala Pro Ser Asn Phe Ala Asn Gly Val Ala Glu Trp<br>
20	25	3D<br>
He Ser Ser Asn Ser Arg Ser Gin Ala Tyr Lys Val Thr Cys Ser Val<br>
35	40	45<br>
Arg Gin Ser Ser Ala Gin Asn Arg Lya Tyr Thr lie Lys Val Glu Val<br>
50	55	60<br>
Pro Lys Val Ala Thr Gin Thr Val Gly Gly Val Glu Leu Pro Val Ala<br>
65	70	75	80<br>
Ala Trp Arg Ser Tyr Leu Asn Leu Glu Leu Thr lie Pro lie Phe Ala<br>
85	90	95<br>
Thr Asn Ser Asp Cys Glu Leu He Val Lys Ala Met Gin Gly Leu Leu<br>
100	105	110<br>
Lys Asp Gly Asn Pro lie Pro Ser Ala He Ala Ala Asn Ser Gly lie<br>
115	120	125<br>
Tyr<br>	216<br>	17<br>	PRT<br>	Circular Mimotope<br><br><iqq>     216<br>
Gly Glu Phe Cys lie Asn His Arg Gly Tyr Trp Val Cys Gly Asp Pro<br>
15	10	15<br>
Ala<br>	217<br>	329<br>	PRT<br>	Bacteriophage Q-beta<br>  217<br>
Met Ala.  Lys Leu Glu Thr Val Thx Leu Gly Asn lie Gly Lys Asp Gly<br>
15	10	15<br>
Lys Gin Thr Leu Val Leu Asn Pro Arg Gly Val Asn Pro Thr Asn Gly<br>
20	.       25	30<br>
Val Ala Ser Leu Ser Gin Ala Gly Ala Val Pro Ala Leu Giu Lys Arg<br>
3-5	40	45<br>
Val Thr Val Ser Val Ser Gin Pro Ser Arg Asn Arg Lys Asn Tyr Lys<br>
50	55	60<br>
Val Gin Val Lys lie Gin Asn Pro Thr Ala Cys Thr Ala Asn Gly Ser<br>
65	70	75	80<br>
Cys Asp Pro Ser Val Thr Arg Gljn Ala Tyr Ala Asp Val Thr Phe Ser<br>
85	90	95<br>
Phe Thr Gin Tyr Ser Thr Asp Glu Glu Arg Ala Phe Val Arg Thr Glu<br>
100	105	110<br>
Leu Ala Ala Leu Leu Ala Ser Pro Leu Leu lie Asp Ala lie Asp Gin<br>
115	120	125<br>
Leu Asn Pro Ala Tyr Trp Thr Leu Leu lie Ala Gly Gly Gly Ser Gly<br>
130	135	140<br>
Ser Lys Pro Asp Pro Val lie Pro Asp Pro Pro He Tp Pro Pro Pro<br>
145	150	155	160<br>
Gly Thr Gly Lys Tyr Thr Cys Pro Phe Ala lie Trp Ser Leu Glu Glu<br>
165	170	175<br>
Val Tyr Glu Pro Pro Thr Lys Asn Arg Pro Trp Pro He Tyr Asn Ala<br>
180	185	190<br>
Val Glu Leu Gin Pro Arg Glu Phe Asp Val Ala Leu Lys Asp Leu Leu<br>
195	200	205<br>
Gly Asn Thr Lys Trp Arg Asp Trp Asp Ser Arg Leu Ser Tyr Thr Thx<br>
210	215	220<br>
Phe Arg Gly Cys Arg Gly Asn Gly Tyr He Asp Leu Asp Ala Thr Tyr<br><br>
Ai5	230	235	240<br>
Leu Ala Thr Asp Gin Ala Met Arg Asp Gin Lys Tyr Asp lie Arg Glu<br>
245	250	255<br>
Gly Lys Lys Pro Gly Ala Phe Gly Asn lie Glu Arg Phe lie Tyr Leu<br>
260	265	270<br>
Lys Ser lie Asn Ala Tyr Cys Ser Leu Ser Asp lie Ala Ala Tyr His<br>
275	280	285<br>
Ala Asp Gly Vai lie Val Gly Phe Trp Arg Asp Pro Ser Ser Gly Gly<br>
290	295	300<br>
Ala lie Pro Phe Asp Phe Thr Lys Phe Asp Lys Thr Lys Cys Pro lie<br>
305	310	315	320<br>
Gin Ala Val lie Val Val Pro Arg Ala 325<br>	219<br>	770<br>	PRT<br>	Amyloid-Beta Protein {Homo Sapiens)<br>  218<br>
Met Leu Pro Gly Leu Ala Leu Leu Leu Leu Ala Ala Trp Thr Ala Arg<br>
15	10	15<br>
Ala Leu Glu Val Pro Thr Asp Gly Asn Ala Gly Leu Leu Ala Glu Pro<br>
20	25	30<br>
Gin lie Ala Met Phe Cys Gly Arg Leu Asn Met His Met Asn Val Gin<br>
35	40	45<br>
Asn Gly Lys Trp Asp Ser Asp Pro Ser Gly Thr Lys Thr Cys lie Asp<br>
50	55	60<br>
Thr Lys Glu Gly lie Leu Gin Tyr Cys Gin Glu Val Tyr Pro Glu Leu<br>
65	70	75	80<br>
Gin He Thr Asn Val Val Glu Ala Asn Gin Pro Val Thr He Gin Asn<br>
85	90	95<br>
Trp Cys Lys Arg Gly Arg Lys Gin Cys Lys Thr His Pro His Phe Val<br>
lOp	105	110<br>
He Pro Tyr Arg Cys Leu Val Gly Glu Phe Val Ser Asp Ala Leu Leu<br>
115	120	125<br>
Val Pro Asp Lys Cys Lys Phe Leu His Gin Glu Arg Met Asp Val Cys<br>
130	135	140<br>
Glu Thr His Leu Hia Trp His Thr Val Ma Lys Glu Thr Cya Ser GXu<br>
145	150	155	160<br>
Lys Ser Thr Asn Leu His Asp Tyr Gly Met Leu Leu Pro Cys Gly He<br><br>
165	170	175<br>
Asp Lys Phe Arg Gly Val Glu Phe Val Cys Cys Pro Leu Ala Glu Glu<br>
180	1B5	190<br>
Ser Asp Asn Val Asp Ser Ala Asp Ala Glu Glu Asp Asp Ser Asp Val<br>
195	200	205<br>
Trp Trp Gly Gly Ala Asp Thr Asp Tyr  Ala Asp Gly Ser Glu Asp Lys<br>
210	215	Z2Q<br>
Val Val Glu Val Ala Glu Glu Glu Glu Val Ala Glu Val Glu Glu Glu<br>
225	230	235	240<br>
Glu Ala Asp Asp Asp Glu Asp Asp Glu Asp Gly Asp Glu Val Glu Glu<br>
245	250	255<br>
Glu Ala Glu Glu Pro IVr Glu Glu Ala Thr Glu Arg Thr Thr Ser lie<br>
2S0	265	270<br>
Ala Thr Thr Thr Thr Thr Thr Thr Glu Ser Val Glu Glu Val Val Arg<br>
275	280	285<br>
Glu Val Cys Ser Glu Gin Ala Glu Thr Gly Pro Cys Arg Ala Met lie<br>
290	295	300<br>
Ser Arg Trp Tyr Phe Asp Val Thr Glu Gly Lys Cys Ala pro Phe Phe<br>
305	310	315	320<br>
Tyr Gly Gly Cys Gly Gly Asu Arg Asn Asn Phe Asp Thr Glu Glu Tyr<br>
325	330	335<br>
Cys Met Ala Val Cys Gly Ser Ala Met Ser Gin Ser Leu Leu Lys Thr<br>
340	345	350<br>
Thr Gin Glu pro Leu Ala Arg Asp Pro Val Lys Leu Pro Thr Thr Ala<br>
355	360	365<br>
Ala Ser Thr Pro Asp Ala Val Asp Lys Tyr Leu Glu Thr Pro Gly Asp<br>
370	375	380<br>
Glu Asn Glu His Ala His Phe Gin Lys Ala Lys Glu Arg Leu Glu Ala<br>
385	390	395	400<br>
Lys His Arg Glu Arg Met Ser Gin Val Met Arg Glu Trp Glu Glu Ala<br>
405	410	415<br>
Glu Arg Gin Ala Lys Asn Leu Pro Lys Ala Asp Lys Lys Ala Val lie<br>
420	425	430<br>
Gin His Phe Gin Glu Lys Val Glu Ser Leu Glu Gin Glu Ala Ala Asn<br>
435	440	445<br>
Glu Arg Gin Gin Leu Val Glu Thr His Met Ala Arg Val Glu Ala Met<br>
450	455	460<br>
Leu Asn Asp Arg Arg Arg Leu Ala Leu Glu Asn Tyr lie Thr Ala Leu<br>
465	470	475	480<br>
Gin Ala Val Pro Pro Arg Pro Arg His Val Phe Asn Met Leu Lys Lys<br>
485	490	495<br>
Tyr Val Arg Ala Glu Gin Lys Asp Arg Gin His Thr Leu Lys His Phe<br><br>
500	505	510<br>
Glu His Val Arg Met Val Asp Pro Lys Lys Ala Ala Gin He Arg Ser<br>
515	520	525<br>
Gin Val Met Thr His Leu Arg Val lie Tyr Glu Arg Met Asn Gin Ser<br>
530	535	540<br>
Leu Ser Leu Leu Tyr Asn Val Pro Ala Val Ala Glu Glu He Gin Asp<br>
545	550	555	560<br>
Glu Val Asp Glu Leu Leu Gin Lys Glu Gin Asn Tyr Ser ASp Asp Val<br>
565	570	575<br>
Leu Ala Asn Met He Ser Glu Pro Arg He Ser Tyr Gly Asn Asp Ala<br>
5B0	585	590<br>
Leu Met Pro Ser Leu Thr Glu Thr Lys Thr Thr Val Glu X-eu Leu Pro<br>
595	600	605<br>
Val Asn Gly Glu Phe Ser Leu Asp Asp Leu Gin Pro Trp Hid Ser Phe<br>
61D	615	620<br>
Gly Ala Asp Ser Val Pro Ala Asn Thr Glu Asn Glu Val Glu Pro Val<br>
625	630	635	640<br>
Asp Ala Arg Pro Ala Ala Asp Arg Gly Leu Thr Thr Arg pro Gly Ser<br>
645	650	655<br>
Gly Leu Thr Asn He Lys Thr Glu Glu He Ser Glu Val Lys Met Asp<br>
660	655	670<br>
Ala Glu Phe Arg His Asp Ser Gly Tyr Glu Val His His Gin Lys Leu<br>
675	680	685<br>
Val Phe Phe Ala Glu Asp Val Gly Ser Asn Lys Gly Ala He He Gly<br>
690	695	700<br>
Leu Met Val Gly Gly Val Val He Ala Thr Val He Val He Thr Leu<br>
705	710	715	720<br>
Val Met I-eu Lys Lys Lys Gin Tyr Thr Ser He His His Gly Val Val<br>
725	730	735<br>
Glu Val Asp Ala Ala Val Thr Pro Glu Glu Arg His Leu Ser Lys Met<br>
740	745	750<br>
Gin Gin Asn Gly Tyr Glu Asn Pro Thr Tyr Lys Phe Phe Glu Gin Met<br>
755	760	765<br>
Gin Asn 770<br>	219<br>	82<br>	PET<br>	Beta-Amyloid Peptide Precuxsor (Homo Sapiens)<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
We Claim:<br>
1.	A composition useful in production of vaccine comprising:<br>
(a)	a non-natural molecular scaffold comprising:<br>
(i)        a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and<br>
(ii)       an organizer, such as herein described, comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at least one covalent bond;<br>
(b)	an antigen or antigenic determinant with at least one second attachment site,<br>
wherein said antigen or antigenic determinant is amyloid beta peptide (AÎ²1-42) or<br>
a fragment thereof, and wherein said second attachment site being selected from<br>
the group consisting of:<br>
(i)        an attachment site not naturally occurring with said antigen or antigenic<br>
determinant; and (ii)       an attachment site naturally occurring with said antigen or antigenic<br>
determinant,<br>
(c)	optionally a heterobifunctional cross-linker,<br>
wherein said second attachment site is capable of association through at least one non-peptide bond to said first attachment site; and<br>
wherein said antigen or antigenic determinant and said scaffold interact through said association to form an ordered and repetitive antigen array.<br>
2.	The composition as claimed in claim 1, wherein said association is by way of at least one covalent non-peptide bond.<br>
3.	The composition as claimed in claim 1, wherein said core particle is selected from the group consisting of:<br>
i)	a virus;<br><br>
ii)	a virus-like particle;<br>
          iii)	a bacteriophage;<br>
iv)	a bacterial pilus;<br>
v)	a viral capsid particle; and<br>
vi)	a recombinant form of (i), (ii), (iii), (iv) or (v).<br>
4.	The composition as claimed in claim 3, wherein said organizer is a polypeptide, a peptide or an amino acid and said second attachment site is a polypeptide, a peptide or an amino acid.<br>
5.	The composition as claimed in claim 1 or claim 3, wherein said core particle is a virus-like particle.<br>
6.	The composition as claimed in claim 5, wherein said virus-like particle is a dimer or multimer of a polypeptide comprising amino acids 1-147 of SEQ ID NO: 158.<br>
7.	The composition as claimed in claim 6, wherein said virus-like particle is a dimer or multimer of a polypeptide comprising amino acids 1-152 of SEQ ID NO: 158.<br>
8.	The composition as claimed in claim 7, wherein said first attachment site comprises or is an amino group and said second attachment site comprises or Is a sulfhydryl group.<br>
9.	The composition as claimed in claim 5, wherein said virus-like particle is a Hepatitis B virus capsid protein.<br>
10.	The composition as claimed in claim 9, wherein said first attachment site comprises or is a lysine residue and said second attachment site comprises or is a cysteine residue.<br><br>
11. The composition as claimed in claim 10, wherein one or more cysteine residues of  said Hepatitis B virus capsid protein have been either deleted or substituted with another amino acid residue.<br>
12. The composition as claimed in claim 10, wherein said Hepatitis B virus capsid protein comprises an amino acid sequence selected from the group consisting of;<br>
a)	the amino acid sequence of SEQ ID NO:89;<br>
b)	the amino acid sequence of SEQ ID NO:90;<br>
c)	the amino acid sequence of SEQ ID NO:93;<br>
d)	the amino acid sequence of SEQ ID NO:98;<br>
e)	the amino acid sequence of SEQ ID NO:99;<br>
f)	the amino acid sequence of SEQ ID NO: 102;<br>
g)	the amino acid sequence of SEQ ID NO: 104; h) the amino acid sequence of SEQ ID NO: 105; i) the amino acid sequence of SEQ ID NO:106; j) the amino acid sequence of SEQ ID NO: 119; k) the amino acid sequence of SEQ ID NO: 120; 1) the amino acid sequence of SEQ ID NO: 123; m) the amino acid sequence of SEQ ID NO: 125; n) the amino acid sequence of SEQ ID NO: 131; o) the amino acid sequence of SEQ ID NO: 132; p)  the amino acid sequence ofSEQ ID NO: 134;<br>
q) the amino acid sequence of SEQ ID NO:157; and r) the amino acid sequence of SEQ ID NO: 158.<br>
13. The composition as claimed in claim 12, wherein one or more cysteine residues of said Hepatitis B virus capsid protein have been either deleted or substituted with another amino acid residue.<br>
14. The composition as claimed in claim 13, wherein the cysteine residues corresponding to amino acids 48 and 107 in SEQ ID NO: 134 have been either deleted or substituted with another amino acid residue.<br><br>
15. The composition as claimed in claim 12, wherein one or more lysine residue of said Hepatitis B virus capsid protein have been either deleted or substituted with another amino acid residue.<br>
16.	The composition as claimed in claim 5, wherein said virus-like particle comprising<br>
recombinant proteins, or fragments thereof, being selected from the group consisting<br>
of:<br>
(a)	recombinant proteins of Hepatitis B virus;<br>
(b)	recombinant proteins of measles virus;<br>
(c)	recombinant proteins of Sindbis virus;<br>
(d)	recombinant proteins of Rotavirus;<br>
(e)	recombinant proteins of Foot-and-Mouth-Disease virus; (0 recombinant proteins of Retrovirus;<br>
(g) recombinant proteins of Norwalk virus;<br>
(h) recombinant proteins of Alphavirus;<br>
(i) recombinant proteins of human Papilloma virus;<br>
(j) recombinant proteins of Polyoma virus;<br>
(k) recombinant proteins of bacteriophages; and<br>
(1) recombinant proteins of RNA-phages;<br>
(m)recombinant proteins of QP-phage;<br>
(n) recombinant proteins of GA-phage;<br>
(o) recombinant proteins of fr-phage; and<br>
(p) recombinant proteins of Ty.<br>
17.	The composition as claimed in claim 5, wherein said virus-like particle comprising, or<br>
alternatively essentially consisting of, recombinant proteins, or fragments thereof, of<br>
a RNA phage.<br><br><br><br>
18.	The composition as claimed in claim 5, wherein said virus-like particle comprising, or<br>
    alternatively essentially consisting of, recombinant proteins, or fragments thereof, of<br>
a RNA phage being selected from the group consisting of:<br>
(a)	bacteriophage QÎ²;<br>
(b)	bacteriophage R 17;<br>
(c)	bacteriophage fr;<br>
(d)	bacteriophage GA;<br>
(e)	bacteriophage SP;<br>
(f)	bacteriophage MS2;<br>
(g)	bacteriophage M11; (h) bacteriophage MX 1; (i) bacteriophage NL95; (k) bacteriophage f2; and (1) bacteriophage PP7.<br><br>
19.	The composition as claimed in claim 5, wherein said virus-like particle comprising, or alternatively essentially consisting of, recombinant proteins, or fragments thereof, of bacteriophage Qp.<br>
20.	The composition as claimed in claim 19, wherein said virus-like particle comprises or alternatively consists essentially of coat proteins having the amino acid sequence of SEQIDN0:I59.<br>
21.	The composition as claimed in claim 5, wherein said virus-like particle comprising, or alternatively essentially consisting of, recombinant proteins, or fragments thereof, of bacteriophage fr.<br>
22.	The composition as claimed in claim I, wherein said core particle is selected from the group consisting of<br>
i)	a virus-like particle;<br>
ii)        a bacterial pilus; and<br><br>
iii)       a virus-like particle of a RNA-phage,<br>
23.	The composition as claimed in claim 22, wherein said core particle is a virus-like particle of a RNA-phage.<br>
24.	The composition as claimed in claims 5, 9, 12, 17-21, wherein said second attachment site does not naturally occur within said antigen or antigenic determinant.<br>
25.	The composition as claimed in claim 24, wherein said composition comprises an amino acid linker.<br>
26.	The composition as claimed in claim 25, wherein said amino acid linker is bound to said antigen or said antigenic determinant by way of at least one covalent bond.<br>
27.	The composition as claimed in claim 26, wherein said covalent bond is a peptide bond.<br>
28.	The composition as claimed in 25, wherein said amino acid linker comprises, or alternatively consist of. said second attachment site.<br>
29.	The composition as claimed in claim 28, wherein said amino acid linker comprises a sulfhydryl group.<br>
30.	The composition as claimed in claim 28, wherein said amino acid linker comprises a cysteine residue.<br>
31.	The composition as claimed in claim 28, wherein said amino acid linker is selected from the group consisting of:<br><br>
(a)	CGG<br>
(b)	N-terminal gamma 1-linker<br>
(c)	N-terminal gamma 3-linker<br>
(d)	Ig hinge regions;<br><br>
(e) N-termina! glycine linkers; y- (0(G)kC(G)n with n=0-12 and k=0-5; (g) N-terminal glycine-serine linkers<br>
(h) (G)kC(G)â,(S)i(GGGGS)n with n=0-3, k=0-5, m=0-10, 1=0-2; (i) GGC G) GGC- NH2<br>
(k) C-terminal gamma 1-linker (1) C-terminal gamma 3-linker (m) C-terminal glycine linkers (n) (G)âC(G)k with n=0-12 and k=0-5; (o) C-terminal glycine-serine linkers (p) (G)n,(S)i(GGGGS)n(G)oC(G)k with n=0-3, k=0-5, m=O-10, 1=0-2, and o=0^8.<br>
32.	The composition as claimed in claim 1, wherein said first attachment site comprises or is an amino group and said second attachment site comprises or is a sulfhydryl group.<br>
33.	The composition as claimed in claim 1, wherein said first attachment site comprises or is a lysine residue and said second attachment site comprises or is a cysteine residue.<br>
34.	The composition as claimed in claim 1, wherein said amyloid beta peptide (AÎ²1-2) or a fragment thereof is selected from the group consisting of<br><br>
(a)	Ap 1-15;<br>
(b)	AP 1-27;<br>
(c)	Ap 1-40; (d)Api-42;<br><br>
(e)	Ap 33-40; and<br>
(f)	Ap 33-42.<br><br>
35.	The composition as claimed in claim I or claim 34 comprising a heterobifunctional<br>
 cross linker, preferably selected from the group consisting of:<br>
(a)	SMPH;<br>
(b)	Sulfo-MBS; and<br>
(c)	Sulfo-GMBS.<br>
36.	The composition as claimed in claim 1, wherein said amyloid beta peptide (AÎ²1-42) or<br>
fragment thereof with said second attachment site has an amino acid sequence<br>
selected from the group consisting of<br>
(a)	the amino acid sequence of DAEFRHDSGYEVHHQGGC;<br>
(b)	the amino acid sequence of CGHGNKSGLMVGGVVIA; and<br>
(c)	the amino acid sequence of DAEFRHDSGYEVHHQKLVFFAEDVGSNGGC.<br>
37.	The composition as claimed in claim 36, wherein said core particle is selected from<br>
the group consisting of:<br>
(a)	a virus-like particle comprising, alternatively consisting of, recombinant proteins, or fragments thereof of bacteriophage Qp;<br>
(b)	a virus-like particle comprising, alternatively consisting of, recombinant proteins, or fragments thereof of bacteriophage fr;<br>
(c)	a virus-like particle of HBcAg-lys-2cys-Mut;<br>
(d)	a bacterial pilus; and<br>
(e)	a Type-I pilus of Escherichia coli.<br><br>
38.	The composition as claimed in claim 34, wherein said first attachment site comprises or is an amino group and said second attachment site comprises or is a sulfhydryl group.<br>
39.	The composition as claimed in claim 34. wherein said first attachment site comprises or is a lysine residue and said second attachment site comprises or is a cysteine residue.<br><br>
0) GGC-NH2, GGC-NMe, GGC-N(Me)2, GGC-NHET or GGC-N{Et)2;      (k) C-terminal gamma 1 -linker (1) C-terminal gamma 3-linker (m) C-terminal glycine linkers (n) {G)nC(G)k with n=0-12 and k=0-5; (o) C-terminal glycine-serine linkers (P) (G)m(S)i{GGGGS)n(G)o C(G)k with n=0-3, k=0-5, m=0-10, 1=0-2, and o=0-8.<br>
48.	The composition as claimed in claim 34, wherein said amino acid linker is selected<br>
from the group consisting of:<br>
(a)	CGG<br>
(b)	CGKR;<br>
(c)	CGHGNKS;<br>
(d)	GGC;<br>
(e)	GGC-NH2;<br>
49.	A pharmaceutical composition comprising:<br>
(a)	the composition of anyone of the claims 1-48; and<br>
(b)	an acceptable pharmaceutical carrier.<br><br>
50.	A vaccine composition comprising the composition in anyone of the claims 1-48.<br>
51.	A process for producing a non-naturally occurring, ordered and repetitive antigen array comprising:<br>
(a) providing a non-natural molecular scaffold comprising: (i)       a core particle selected from the group consisting of:<br>
(1)	a core particle of non-natural origin; and<br>
(2)	a core particle of natural origin; and<br>
(ii)       an organizer, such as herein described, comprising at least one first<br>
attachment site, wherein said organizer is connected to said core particle by at least one covalent bond;and<br><br>
(b)	providing an antigen or antigenic determinant with at least one second attachment<br>
      site, wherein said antigen or antigenic determinant is amyloid beta peptide (AÎ²1.<br>
42) or a fragment thereof, and wherein said second attachment site being selected<br>
from the group consisting of:<br>
(i)        an attachment site not naturally occurring with said antigen or antigenic<br>
determinant; and (ii)       an attachment site naturally occurring with said antigen or antigenic<br>
determinant, wherein said second attachment site is capable of association through at least one non-peptide bond to said first attachment site; and<br>
(c)	combining said non-natural molecular scaffold and said antigen or antigenic<br>
determinant, wherein said antigen or antigenic determinant and said scaffold<br>
interact through said association to form an ordered and repetitive antigen array.<br>
52.	A composition useful in production of vaccine substantially such as herein described with reference to the accompanying drawings and as illustrated in the foregoing examples.<br>
53.	A process for producing a non-naturally occurring, ordered and repetitive antigen array substantially such as herein described with reference to the accompanying drawings and as illustrated in the foregoing examples.</iqq></:210></:400></:212></:211></:400></:211></:211></:210></iuu></:212></:213></:400></:210></:400></:223></:213></:212></:220></:400></:220></:212></i50></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGFic3RyYWN0IGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGNsYWltcyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgMi5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete) 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgMy5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete) 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgNC5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete) 4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkgNS5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete) 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlIDIucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete)-duplicate 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlIDMucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete)-duplicate 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSktZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRyYXdpbmdzIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIG90aGVycyBkdXBsaWNhdGUgMi5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 others duplicate 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIG90aGVycyBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 others duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTEwNi1jaGVucC0yMDAzIHBldGl0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">1106-chenp-2003 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="214301-method-and-system-for-determining-a-lattice-of-points-suitable-for-a-simulation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="214303-a-method-of-moulding-a-plastic-material-a-mould-and-a-method-of-injection-compression-moulding.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>214302</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1106/CHENP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Feb-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Jul-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CYTOS BIOTECHNOLOGY AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Wagistrasse 25, CH-8952 Zurich-Schlieren,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RENNER, Wolfgang, A (ET. AL)</td>
											<td>Grenzsteig 9, CH-8802 Kilchberg,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2002/000168</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-01-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/288,549</td>
									<td>2001-05-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/262,379</td>
									<td>2001-01-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/326,998</td>
									<td>2001-10-05</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/331,045</td>
									<td>2001-11-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/214302-molecular-antigen-array by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:00:56 GMT -->
</html>
