<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/215520-methods-and-materials-for-the-production-of-organic-products-in-cells-of-candida-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:38:11 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 215520:METHODS AND MATERIALS FOR THE PRODUCTION OF ORGANIC PRODUCTS IN CELLS OF CANDIDA SPECIES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS AND MATERIALS FOR THE PRODUCTION OF ORGANIC PRODUCTS IN CELLS OF CANDIDA SPECIES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to biocatalysts that are cells, optimally of the Crabtree-negative phenotype, comprising expression vectors encoding genes heterologous to the cell that enable increased production of organic products. More specifically, lthe invention relates to genetically modified Candida cells, methods for making the Candida cells, and their use in production of organic products, particularly lactic acid.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHODS AND MATERIALS FOR THE PRODUCTION OF ORGANIC<br>
PRODUCTS IN CELLS OF CANDIDA SPECIES<br>
This application is a continuation-in-part of U.S. Patent Application Serial<br>
No. 09/992,430, filed November 23, 2001, which claims priority to U.S.<br>
Provisional Application Serial No. 60/252,541, filed November 22, 2000.<br>
BACKGROUND OF THE INVENTION<br>
The use of microorganisms for synthesizing industrially important organic<br>
products is well known. Biosynthetic approaches for producing organic products<br>
can be extremely efficient when compared to large-scale chemical synthesis.<br>
Advantages a biosynthetic approach may have over a chemical synthetic approach<br>
for manufacturing an organic product include more rapid and more efficient<br>
product yield, isomeric purity, and reduced cost (see Thomas et al., 2002, Trends<br>
Biotechnol. 20: 238-42).<br>
Lactic acid has wide industrial applicability, including uses in chemical<br>
processing and synthesis, cosmetics, pharmaceuticals, plastics, and food<br>
production. Lactic acid is a relatively simple organic molecule, and can be<br>
produced either by chemical synthesis or by fermentation in microorganisms<br>
(biosynthesis). As genetic manipulation of microorganisms has become more<br>
advanced, fermentation processes for lactic acid production have become<br>
commercially preferred over chemical synthesis. One reason for this preference is<br>
that using genetically modified microorganisms enables production of optically<br>
pure (i.e., either the l(+) or D(-) isomer) product. Such methods obviate the need<br>
for separating racemic product mixtures, thereby reducing cost.<br>
Nevertheless, the use of microorganisms for producing organic products<br>
has certain limitations. For example, bacteria can produce large quantities of<br>
organic products under fermentation conditions, but the accumulation of organic<br>
products within the bacteria itself and in the growth medium can inhibit<br>
proliferation of the bacteria, or cause cell death. Even when more robust<br>
organisms are engineered and used for production, such as the acidophilic yeast<br>
Saccharomyces cerevisiae, organic products can lead to cell growth suppression,<br>
reducing overall yield of organic product. Thus, there remains a need in the art for<br>
robust microorganisms that are amenable to genetic manipulation, for use in<br>
bioreactors and with other biosynthetic methods for producing industrially<br>
important organic products.<br>
SUMMARY OF THE INVENTION<br>
This invention provides methods and reagents, particularly cells and<br>
recombinant cells, for producing organic products by biosynthesis. The invention<br>
specifically provides recombinant nucleic acid constructs encoding at least one<br>
protein useful for the synthesis of an organic product, cells comprising said<br>
constructs, particularly Crabtree-negative cells, methods for making such cells,<br>
methods for culturing such cells, and methods and reagents for synthesizing<br>
numerous organic products in vivo.<br>
In one aspect, the invention provides recombinnant nucleic acid constructs<br>
comprising a sequence encoding at least one protein useful for the synthesis of an<br>
organic product. In a preferred embodiment, the recombinant nucleic acid<br>
construct encodes lactate dehydrogenase. In one embodiment of this aspect, the<br>
recombinant nucleic acid construct comprises a promoter operably linked to the<br>
nucleic acid encoding a protein useful for synthesis of an organic product, wherein<br>
the promoter is a promoter from a Candida species, preferably the Candida<br>
species that comprises the recombinant nucleic acid construct.<br>
In another aspect, the invention provides a transformed Crabtree-negative<br>
cell from the genera Candida, comprising the recombinant nucleic acid construct<br>
encoding at least one protein useful for the synthesis of an organic product. In a<br>
preferred embodiment, the recombinant nucleic acid construct encodes lactate<br>
dehydrogenase. In one embodiment of this aspect, the recombinant nucleic acid<br>
construct comprises a promoter operably linked to the nucleic acid encoding a<br>
protein useful for synthesis of an organic product, wherein the promoter is a<br>
promoter from a Candida species, preferably the Candida species that comprises<br>
the recombinant nucleic acid construct. In another aspect, the invention provides<br>
a cell of a Candida species genetically manipulated so that it has reduced<br>
efficiency in metabolizing pyruvate to ethanol. In preferred embodiments of this<br>
aspect of the invention, the cell farther comprises a recombinant nucleic acid<br>
construct of the invention encoding at least one protein useful for the synthesis of<br>
an organic product. In a preferred embodiment, the recombinant nucleic acid<br>
construct encodes lactate dehydrogenase. In one embodiment of this aspect, (he<br>
recombinant nucleic acid construct comprises a promoter operably linked to the<br>
nucleic acid encoding a protein useful for synthesis of an organic product, wherein<br>
the promoter is a promoter from a Candida species, preferably the Candida<br>
species that comprises the recombinant nucleic acid construct.<br>
In another aspect, the invention provides methods for producing organic<br>
products comprising fermenting a Crabtree-negative cell from the genera Candida<br>
comprising a recombinant nucleic acid construct of the invention under conditions<br>
that allow for the biosynthesis of said organic products. In preferred embodiments<br>
of this aspect of the invention, the organic product is lactic acid. In a preferred<br>
embodiment, the recombinant nucleic acid construct encodes lactate<br>
dehydrogenase. In one embodiment of this aspect, the recombinant nucleic acid<br>
construct comprises a promoter operably linked to the nucleic acid encoding a<br>
protein useful for synthesis of an organic product, wherein the promoter is a<br>
promoter from a Candida species, preferably the Candida species that comprises<br>
the recombinant nucleic acid construct.<br>
It is an advantage of this invention that the transformed cells provided<br>
herein exhibit the "Crabtree negative" phenotype. Crabtree-negative organisms<br>
are characterized by the ability to be induced into an increased fermentative scale.<br>
Both naturally occurring organisms and genetically modified organisms can be<br>
characterized as Crabtree-negative. The Crabtree effect is defined as oxygen<br>
consumption inhibition in a microorganism when the microorganism is cultured<br>
under aembic conditions in the presence of a high concentration of glucose (e.g.<br>
&gt;5 mM glucose). Crabtree-posmye organisms continue to ferment (rather than<br>
respire) irrespective of oxygen availability in the presence of glucose, while<br>
Crabtree-negative organisms do not exhibit glucose-mediated inhibition of oxygen<br>
consumption. This characteristic is useful for organic product synthesis, since it<br>
permits cells to be grown at high substrate concentrations but to retain the<br>
beneficial energetic effects of oxidative phosphorylation. Many yeests and fungi<br>
have the Crabtree-negative phenotype including the non-limiting examples of<br>
genera Kluyveromyces, Pichia, Hansenula, Torulopsis. Yamadazyma, and<br>
Candida.<br>
Candida species, which are variously characterized as yeasts and<br>
dimorphic fungi in the art, can exhibit the Crabtree-negative phenotype (Franzblau<br>
&amp; Sinclair, 1983, Mycopathologia 82: 185-190). Certain species can ferment<br>
glucose, as well as alternative carbon sources, can grow at elevated temperatures<br>
(i.e., greater than 37°C), and can tolerate low pH stress. Candida species have<br>
several of the desirable characteristics of an organism to be used in biosynthetic<br>
methods of organic product manufacture: amenability to genetic manipulation,<br>
ability to process a variety of carbon sources, Crabtree-negative phenotype, and<br>
ability to proliferate under various environmental stresses.<br>
Specific preferred embodiments of the present invention will become<br>
evident from the following more detailed description of certain preferred<br>
embodiments and the claims.<br>
ACCOMPANYING<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 is a schematic diagram of the vector described as pM1260,<br>
comprising the G418 resistance-coding gene driven by the PGK promoter (S.<br>
cerevisiae) and linked to the GAL 10 (S. cerevisiae) terminator.<br>
FIG. 2 is a schematic diagram of the vector described as pMI268,<br>
comprising the G418 resistance-coding gene driven by the PGK promoter (C.<br>
sonorensis) and linked to the GAL 10 (S. cerevisiae) terminator.<br>
FIG. 3 is a schematic diagram of the vector described as pMI269,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C<br>
sonorensis) and linked to the GAL 10 (S. cerevisiae) terminator.<br>
FIG. 4 is a schematic diagram of the vector described as pMI270,<br>
comprising the hygromycin resistance-coding gene driven by the PGK promoter<br>
(C sonorensis) and linked to the GAL10 (S. cerevisiae) terminator.<br>
FIG. 5 is a schematic diagram of the vector described as pMI234,<br>
comprising the MEL5 (S. cerevisiae) gene driven by the PGK promoter (C.<br>
sonorensis).<br>
FIG. 6 is a schematic diagram of the vector described as pMI238,<br>
comprising the MEL5 (S. cerevisiae) gene driven by the TDH promoter (C.<br>
sonorensis).<br>
FIG. 7 is a schematic diagram of the vector described as pM1271,<br>
comprising the hygromycin resistance-coding gene driven oy the TDH promoter<br>
(C. sonorensis) and linked to the GAL10(S. cerevisiae) terminator.<br>
FIG 8 is a schematic diagram of the vector described as pMI246,<br>
composing the MEL5 (S. cerevisiae) and LDH (L. helveticus) genes each driven<br>
by the PGK promoter (C sonorensis). The LDH gene is linked to the CYCl<br>
terminator, which is upstream from an S26 rRNA (C. sonorensis) region<br>
FIG. 9 is a schematic diagram of the vector described as pMI247,<br>
comprising the MEL5 (S. cerevisiae) gene driven by the TDH promoter (C.<br>
sonorensis) and the LDH (L helveticus) gene driven by the PGK promoter (C<br>
sonorensis). The LDH gene is linked to the CYC1 terminator, which is upstream<br>
from an S26 rRNA (C sonorensis) region.<br>
FIG. 10 is a schematic diagram of the vector described as pM1257,<br>
comprising the MEL5 (5. cerevisiae) gene driven by the PGK promoter (C.<br>
sonorensis) and the LDH (L helveticus) gene driven by the PGK promoter (C<br>
sonorensis). The LDH gene is linked to the CYCl terminator. This entire<br>
expression cassctte is inserted between the PDC) promoter and terminator (C<br>
sonorensis).<br>
FIG. 11 is a schematic diagram of the vector described as pMI265,<br>
composing the MEL5 (S cerevisiae) gene driven by the PGK romoter (C<br>
sonorensis) and the LDH (B. megatenum; from vector pVR24) gene driven by the<br>
PGK promoter (C sonorensis). The LDH gene is linked to the PDC1 (C.<br>
sonorensis) teminator. This entire expression cassette is inserted between the<br>
PDCl promoter and terminator (C sonorensis).<br>
FIG. 12 is a schematic diagram of the vector described at pMI266.<br>
comprising the MEL5 (5. cerevisiae) gene driven by the PGK promoter (C<br>
sonorensis) and the LDH (k. oryzae; from vector pVR27) gene driven by the PGK<br>
promoter (C sonorensis). The LDH gene is linked to the PDC1 (C sonorensis)<br>
terminator. This entire expression cassette is inserted between the PDCl promoter<br>
and terminator (C sonorensis).<br>
FIG. 13 is a schematic diagram of the vector described as pMI267,<br>
comprising tfie MEL5 (S. cerevisiae) gene driven by the PGK promoter (C<br>
sonorensis). This expression cassette is inserted between the PDCl promoter and<br>
terminator (C. sonorensis).<br>
FIG. 14 is a schematic diagram of the vector described as pML778,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C.<br>
sonorensis), operatively Jinked to the MEL5 terminator, and the LDH {B.<br>
megaterium) gene driven by the PGK promoter (C. sonorensis). The LDH gene is<br>
linked to the GAL10 (S. cerevisiae) terminator.<br>
FIG, 15 is a schematic diagram of the vector described as pM1286,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C.<br>
sonorensis), operahvely linked to the MEL5 (S. cerevisiae) terminator, and the<br>
LDH (B. megaterium) gene driven by the PGK promoter (C sonorensis). The<br>
LDH gene is linked to the GALIO {S. cerevisiae) terminator. This entire<br>
expression cassette is inserted between the PDC2 promoter and terminator (C<br>
sonorensis).<br>
FIG, 16 is a schematic diagram of the vector described as pMI287,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C.<br>
sonorensis), operatively linked to the MEL5 (S. cerevisiae) terminator. This<br>
expression cassette is inserted between the PDC2 promoter and terminator (C<br>
sonorensis).<br>
FIG. 17 is a schematic diagram of the vector described as pMI288,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C.<br>
sonorensis), operatively linked to the MEL5 (S. cerevisiae) terminator, and the<br>
LDH (L helveticus) gene driven by the PGK promoter (C. sonorensis). The LDH<br>
gene is linked to the CYC1 terminator. This entire expression cassette is inserted<br>
between the PDC2 promoter and terminator (C. sonorensis).<br>
Fig, 18 is a schematic diagram of the vector described as pM1256,<br>
comprising the MEL5 (S. cerevisiae) gene driven by the PGK promoter (C<br>
sonorensis) and the LDH (L Helvetian) gene driven by the PGK prompter (C.<br>
sonorertsis). The LDH gene is linked to the CYCl terminator. This entire<br>
expression cassette is inserted upstream of the PDCl terminator (C. sonorensis).<br>
FIG. 19 is a schematic diagram of the vector described as pMI277,<br>
comprising the PDC2 promoter (C, sonorensis),<br>
FIG. 20 is a schematic diagram of the vector described as pMI279,<br>
comprising the G418 resistance-coding gene driven by the TDH promoter (C<br>
sonorensis), operatively linked to the MEL5 (S. cerevisiae) terminator, and the<br>
LDH (B. megaterium) gene driven by the PGK promoter (C. sonorensis). The<br>
LDH gene is linked to the GAL10 terminator (S. cerevisiae). This entire<br>
expression cassette is inserted downstream of the PDC2 promoter (C. sonorensis).<br>
FIG. 21 is a schematic diagram of the vector described as pVR24.<br>
FIG. 22 is a schematic diagram of the vector described as pVR27.<br>
FIG. 23 A-C are schematic diagrams of the vectors including pMI214,<br>
pMI203, pMl205, pMI227, pMI233, and pMI234.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As used herein, "organic product" is any compound containing a carbon<br>
tom. Non-limiting examples of organic products include carboxylates (e.g.<br>
lactate, acrylate, citrate, isocitrate, alpha-ketoglutarate, succinate, fumarate,<br>
malate, oxaloacetate), carbohydrates (e.g. D-xylose), alditols (e.g. xylitol, arabitol,<br>
ribitol), amino acids (e.g. glycine, tryptophan, glutamate), lipids, esters, vitamins<br>
(e.g., L-ascorbate), polyols {e.g. glycerol, 1,3-propanediol, erythritol), aldehydes,<br>
alkenes, alkynes, and lactones. Thus, an organic product can contain one, two,<br>
three, four, five, six, seven, eight, nine, ten, or more carbon atoms. In addition,<br>
organic products can have a molecular weight that is less than about 1,000 (e.g.<br>
less-than about 900, 800, 700, 600, 500, 400, 300, 200, or 100) daltons. For<br>
example, D-xylose (C5H10O5) is an organic product that has a molecular weight of<br>
150 daltons. Further, organic products can be fermentation products.<br>
The term "fermentation product" as used herein refers to any organic<br>
compound that is produced by a fermentation process. Generally, a fermentation<br>
process can involve the anaerobic enzymatic conversion of organic compounds<br>
{e.g. carbohydrates) to compounds such as ethyl alcohol, producing energy in the<br>
form of ATP. Cellular fermentation differs from cellular respiration in that<br>
organic products rather than molecular oxygen are used as electron acceptors.<br>
Non-limiting examples of fermentation products are acetate, ethanol, butyrate, and<br>
lactate.<br>
The organic products can also be derived from pyruvate. A "pyruvate-<br>
derived product," as used herein, refers to any compound that is synthesized from<br>
pyruvate within no more than fifteen enzymatic steps. One enzymatic step is<br>
considered to be any chemical reaction or series of reactions catalyzed by a<br>
polypeptide having enzymatic activity. Such polypeptides are any polypeptide that<br>
catalyzes a chemical reaction of other substances without itself being destroyed or<br>
altered upon completion of the reaction or reactions. These polypeptides can have<br>
any type of enzymatic activity including the non-limiting examples of activities<br>
associated with aconitase, isocitrate dehydrogenase, ketoglutarate dehydrogenase,<br>
succinate thiokinase, succinate dehydrogenase, fumarase, malate dehydrogenase,<br>
citrate synthase, 2,5-dioxovalerate dehydrogenase, 5-dehydro-4-deoxy-D-glucarate<br>
dehydrogenase, glucarate dehydratase, aldehyde dehydrogenase, glucuronolactone<br>
reductase, L-gulonolactone oxidase, 2-dehydro-3-deoxy-D-pentanoate aldolase,<br>
xylonate dehydratase, xylonolactonase, D-xylose dehydrogenase, lactate<br>
dehydrogenase, CoA-transferase, lacyl-CoA dehydratase, or acrylyl-CoA<br>
hydratase.<br>
The carboxylate products of the invention can be in the free acid or salt<br>
form, and can be referred to interchangeably (e.g. "lactic acid" or "lactate"). Use<br>
of either of the terms is taken to encompass the other, unless specifically noted<br>
otherwise. In preferred embodiments, the invention provides the carboxylates in<br>
free acid form.<br>
The term "nucleic acid sequence" or "nucleic acid molecule" refers to a<br>
DNA or RNA molecule. The term encompasses molecules formed from any of<br>
the known base analogs of DNA and RNA such as, but not limited to 4-<br>
acetylcytosine, 8-hydroxy-N6-methyladenosine, aziridinyl-cytosine,<br>
pseudoisocytosine, 5-(carboxyhydroxylmethyl) uracil, 5-fluorouracil, 5-<br>
bromouracil, 5-carboxymethylaminomethyl-2-thiouracil, 5-carboxy-<br>
methylaminomethyluracil, dihydrouracil, inosine, N6-iso-pentenyladenine, 1-<br>
methyladenine, 1-methylpseudouracil, 1-methylguanine, 1-methylinosine, 2,2-<br>
dimethyl-guanine, 2-methyladenine, 2-methylguanine, 3-methylcytosine, 5-<br>
methylcytosine, N6-methyladenine, 7-methylguanine, 5-methylaminomethyluracil,<br>
5-methoxyamino-methyl-2-thiouracil, beta-D-mannosylqueosine, 5"<br>
mcthcxycarbonyl-methyluracil, 5-methoxyuracil, 2-methylthio-N6-<br>
isopentenyladenine, uracil-5-oxyacetic acid methylester, uracil-5-oxyacetic acid,<br>
oxybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-methyl-2-thiouracil, 2-<br>
thiouracil, 4-thiouracil, 5-methyluracil, N-uracil-5-oxyacetic acid methylester,<br>
uracil-5-oxyacetic acid, pseudouracil, queosine, 2-thiocytosine, and 2,6-<br>
diaminopurine.<br>
The term "vector" is used to refer to any molecule (e.g., nucleic acid,<br>
plasmid, or virus) used to transfer protein-coding information to a host cell.<br>
The term "expression vector" refers to a vector that is suitable for<br>
transformation of a host cell and contains nucleic acid sequences that direct and/or<br>
control expression of inserted heterologous nucleic acid sequences. Expression<br>
includes, but is not limited to, processes such as transcription, translation, and<br>
RNA splicing, if introns are present.<br>
The term "operably linked" is used herein to refer to an arrangement of<br>
sequences wherein the sequences are joined together and configured or assembled<br>
so as to perform their usual function. Thus, a sequence operably linked to a<br>
sequence encoding a protein may flank the coding sequence and be capable of<br>
effecting replication and/or transcription of the coding sequence. For example, a<br>
coding sequence is operably linked to a promoter when the promoter is capable of<br>
directing transcription of that coding sequence. A flanking sequence need not be<br>
contiguous with the coding sequence, so long as it functions correctly. Thus, for<br>
example, intervening untranslated yet transcribed sequences can be present<br>
between a promoter sequence and the coding sequence and the promoter sequence<br>
can still be considered "operably linked" to the coding sequence.<br>
The term "host cell" is used to refer to a cell into which has been<br>
introduced or transformed, or is capable of being transformed with a nucleic acid<br>
sequence and then of expressing a selected gene of interest. The term includes the<br>
progeny of the parent cell, whether or not the progeny is identical in morphology<br>
or in genetic make-up to the original parent.<br>
The term "endogenous" as used herein refers to genomic material that is<br>
not exogenous, that is, which has not been introduced into the cell. Such<br>
endogenous genomic material usually develops within an organism, tissue, or cell<br>
and is not inserted or modified by recombinant technology. Endogenous genomic<br>
material encompasses naturally occurring variations.<br>
The term "exogenous" or "heterologous" as used herein refers to genomic<br>
material that is not endogenous, that is, material that has been introduced into the<br>
cell. Typically such material is inserted or modified by recombinant technology.<br>
As used herein, the term "genetically modified" refers to an organism<br>
whose genome has been modified by methods including the non-limiting<br>
examples of addition, substitution, or deletion of genetic material. Such methods<br>
of genetic manipulation are well known in the art and include, but are not limited<br>
to, random mutagenesis, point mutations, including insertions, deletions, and<br>
substitutions of one or a plurality of individual nucleotide residues, knock-out<br>
technology, and transformation of an organism with a nucleic acid sequence using<br>
recombinant technology, including both stable and transient transformants.<br>
The terms "anaerobic" and "anaerobic conditions" are taken to mean that<br>
the amount of dissolved oxygen in a solution, typically a culture medium, is not<br>
detectable (i.e., about 0%), or alternatively the amount of oxygen in the<br>
atmosphere is from about 0% to 2%.<br>
Vectors and Host Cells<br>
A nucleic acid molecule encoding the amino acid sequence of a<br>
polypeptide useful for synthesis of organic products of interest is inserted into an<br>
appropriate cloning or expression vector using standard ligation techniques (see,<br>
for example, Sambrook et al., 2001, Molecular Cloning: A Laboratory<br>
Manual, 3rd ed., Cold Spring Harbor Laboratory Press, New York). The vector is.<br>
typically selected to be functional in the particular host cell employed (i.e., the<br>
vector is compatible with the host cell machinery such that replication,<br>
amplification and/or expression of the gene can occur). A nucleic acid molecule<br>
encoding the amino acid sequence of a polypeptide useful for synthesis of organic<br>
products of interest can be amplified in any appropriate cell and expressed in any<br>
host cell, most preferably a Crabtree-negative host cell.<br>
Preferred Crabtree-negative host cells include those from genera Candida,<br>
including the non-limiting examples of C. sonorensis, C. methonosorbosa, C.<br>
diddensiae, C. parapsilosis, C. naeodendra, C. krusei, C. blankii, and C.<br>
entomophila.<br>
Flanking sequences (including promoters and terminators) may be<br>
homologous (i.e., from the same species and/or strain as the host cell),<br>
heterologous (i.e., from a species other than the host cell species or strain), hybrid<br>
(i.e., a combination of flanking sequences from more than one source), or<br>
synthetic, or the flanking sequences may be native sequences that normally<br>
function to regulate expression of the gene of interest. As such, the source of a<br>
flanking sequence may be any prokaryotic or eukaryotic organism, any vertebrate<br>
or invertebrate organism, or any plant, provided that the flanking sequence is<br>
functional in, and can be activated by, the host cell machinery.<br>
Flanking sequences useful in the vectors of this invention may be obtained<br>
by any of several methods well known in the art. Typically, flanking sequences<br>
useful herein will have been previously identified by mapping and/or by restriction<br>
endonuclease digestion and can thus be isolated from a biological source using the<br>
appropriate restriction endonucleases. In some cases, the complete nucleotide<br>
sequence of a flanking sequence may be known. In such cases, the flanking<br>
sequence may be synthesized using methods well known to those of skill in the<br>
art, as well as those described herein, for nucieic acid synthesis or cloning.<br>
Where all or only a portion of the flanking sequence is known, the full<br>
extent of the functional flanking sequence may be obtained using in vitro<br>
amplification technique such as polymerase chain reaction (PCR) and/or by<br>
screening a genomic library with a suitable oligonucleotide and/or flanking<br>
sequence fragment from the same or another species. Where the flanking<br>
sequence is not known, a fragment of DNA containing a flanking sequence may<br>
be isolated from a larger piece of DNA that may contain, for example, a coding<br>
sequence or even another gene or genes. Isolation may be accomplished by<br>
restriction endonuclease digestion to produce the proper DNA fragment followed<br>
by isolation using agarose gel purification, Qiagen* column chromatography<br>
(Chatsworth, CA), or other methods known to the skilled artisan. The selection of<br>
suitable enzymes to accomplish this purpose will be readily apparent to one of<br>
ordinary skill in the art.<br>
A selectable marker gene or element encodes a protein necessary for<br>
survival and growth of a host cell grown in a selective culture medium. Useful<br>
selection marker genes encode proteins that (a) confer resistance in host cells to<br>
antibiotics or other toxins, e.g., ampicillin, tetracycline, or kanamycin; (b)<br>
complement auxotrophic deficiencies of the host cell, such as Leu2 deficiency; or<br>
(c) supply critical nutrients not available from complex media. Preferred<br>
selectable markers include the non-limiting examples of zeocin resistance gene,<br>
G418 resistance gene, and the hygromycin resistance gene.<br>
Other selection genes may be used to amplify the gene that will be<br>
expressed. Amplification is the process wherein genes that are in greater demand<br>
for the production of a protein critical for growth are reiterated in tandem within<br>
the chromosomes of successive generations of recombinant cells. Examples of<br>
suitable selectable markers for mammalian cells include dihydrofolate reductase<br>
(DHFR) and promoterless thymidine kinase. The mammalian cell transformants<br>
are placed under selection pressure wherein only the transformants are uniquely<br>
adapted to survive by virtue of the selection gene present in the vector. Selection<br>
pressure is imposed by culturing the transformed cells under conditions in which<br>
the concentration of selection agent in the medium is progressively increased,<br>
thereby leading to the amplification of both the selection gene and DNA that<br>
encodes a polypeptide useful for synthesizing an organic product.<br>
Expression and cloning vectors of the present invention will typically<br>
contain a promoter that is recognized by the host organism and operably linked to<br>
the molecule encoding the polypeptide useful for synthesizing an organic product.<br>
Promoters are untranscribed sequences located upstream (i.e., 5") to the translation<br>
start codon of a structural gene (generally within about 100 to 1000 bp) and<br>
control transcription of the structural gene. Promoters are conventionally grouped<br>
into one of two classes: inducible promoters and constitutive promoters. Inducible<br>
promoters initiate increased levels of transcription from DNA under their control<br>
in response to some change in culture conditions, such as the presence or absence<br>
of a nutrient or a change in temperature. Constitutive promoters, on the other<br>
hand, initiate continual gene product production; that is, there is little or no<br>
regulation of gene expression. A large number of promoters of both promoter<br>
types, recognized by a variety of potential host cells, are well known in the art, A<br>
suitable promoter is operably linked to DNA encoding a polypeptide useful for<br>
synthesizing an organic product by removing the promoter from the source DNA<br>
by restriction enzyme digestion or producing a promoter fragment by in vitro<br>
amplification and inserting the desired promoter sequence into the vector. Native<br>
promoter sequences may be used to direct amplification and/or expression of a<br>
nucleic acid molecule that encodes a polypeptide useful for synthesizing an<br>
organic product. A heterologous promoter is preferred, however, if it permits<br>
greater transcription and higher yields of the expressed protein compared to the<br>
native promoter, and if it is compatible with the host cell system that has been<br>
selected for use.<br>
Suitable promoters for use with yeast host cells are also well known in the<br>
art, and include the non-limiting examples of promoters from yeast genes<br>
including phosphoglycerate kinase (PGK), triose dehydrogenase (TDH), pyruvate<br>
decarboxylase (PDC), triose phosphate isomerase (TPI), and alcohol<br>
dehydrogenase (ADH). Preferred promoters of the invention include PGK and<br>
TDH promoters. Yeast enhancers, sequences that increase expression when placed<br>
in relative proximity to a promoter are advantageously used with yeast promoters.<br>
Methods of transforming cells are well known in the art, and can include<br>
such non-limiting examples as electroporation and calcium chloride or lithium<br>
acetate based transformation methods.<br>
Several of the vectors disclosed in the Examples of this invention have<br>
been previously constructed and are described in application PCT/US01/44041.<br>
Briefly vectors pMI234, pMI238, pMI246, pMI247, and the PDC2 in lambda were<br>
constructed as follows.<br>
C. sonorensis gene isolation (PDC2 in lambda): Genomic DNA of C.<br>
sonorensis (ATCC Accession No. 32109) was isolated from cells grown overnight<br>
in YPD using the Easy DNA kit (Invitrogen). DNA was partially digested with<br>
Sau3A and size fractionated by sucrose gradient centrifugation (Sambrook et al.<br>
Id.,). DNA fragments of about 22 kb were ligated to BamHI digested, phosphatase<br>
treated lambda DASH? vector arms (Stratagene) and the ligation mixture was<br>
packaged into lambda particles using Gigapack II Gold Packaging Extract<br>
(Stratagene). The lambda particles were used to infect E. coli MRA P2.<br>
Probes used for isolation of C. sonorensis genes from the library were<br>
prepared by PCR amplification using the Dynazyme EXT polymerase (Finnzymes,<br>
Espoo, Finland), sequence specific primers and genomic DNA of S. cerevisiae, C.<br>
albicans or C. sonorensis as a template as follows,<br>
• Oligonucleotides TGT CAT CAC TGC TCC ATC TT (SEQ ID No. 17) and<br>
TTA AGC CTT GGC AAC ATA TT (SEQ ID No. 18) corresponding to the S.<br>
cerevisiae TDH1 gene were used to amplify a fragment of the TDH gene from<br>
genomic S. cerevisiae DNA.<br>
• Oligonucleotides GCG ATC TCG AGG TCC TAG AAT ATG TAT ACT AAT<br>
TTG C (SEQ ID No. 19) and CGC GAA TTC CCA TGG TTA GTT TTT GTT<br>
GGA AAG AGC AAC (SEQ ED No. 20) corresponding to the C. albicans<br>
PGKl gene were used to amplify a fragment of the PGKl gene from genomic C.<br>
albicans DNA.<br>
• Oligonucleotides TGG ACT AGT AAA CCA ACA GGG ATT GCC TTA GT<br>
(SEQ ID No. 21) and CTA GTC TAG AGA TCA TTA CGC CAG CAT CCT<br>
AGG (SEQ ID No. 22) corresponding to the C. sonorensis 26 S rRNA were<br>
used to amplify a fragment of the 26S rDNA gene from C. sonorensis genomic<br>
DNA.<br>
• Oligonucleotides CCG GAA TTC GAT ATC TGG GCW GGK AAT GCC<br>
AAY GAR TTR AAT GC (SEQ ID No. 2,3) and CGC GGA TTC AGG CCT<br>
CAG TAN GAR AAW GAA CCN GTR TTR AAR TC (SEQ ID No. 24) were<br>
designed based on portions of pyruvate decarboxylase amino acid sequence<br>
WAGNANELNA (SEQ ID No. 25) and DFNTGSFSYS (SEQ ID No. 26), that<br>
are conserved between 5. cerevisiae PDCl, Pichia stipitis PDC1 and PDC2,<br>
and incomplete sequences of Candida albicans PDC1 and PDC3. These<br>
primers were used were used to amplify a fragment of the PDC gene(s) from C.<br>
sonorensis genomic DNA. PCR reaction with these primers produced two<br>
fragments of different nucleotide sequence termed PDCl and PDC2.<br>
• Oligonucleotides TCTGTTMCCTACRTAAGA (SEQ ID No. 27) and<br>
GTYGGTGGTCACGAAGGTGC (SEQ ID No. 28) were designed based on<br>
conserved regions found in fungal alcohol dehydrogenase sequences. These<br>
primers were used to amplify a fragment of the ADH gene(s) from C. sonorensis<br>
genomic DNA. PCR reaction with these primers produced three fragments of<br>
different nucleotide sequences termed ADHl, ADH2, and ADH3.<br>
The library was screened with PCR fragments produced as described above,<br>
and products were labeled with 32P a-dCTP using the Random Primed Labeling Kit<br>
(Boehringer Mannheim). Hybridization with the radioactive probes was performed<br>
by incubation overnight at 42°C in a solution containing 50% formamide, 5x<br>
Denhardt"s, 5x SSPE, 0.1% SDS, 100 Mg/mL herring sperm DNA, 1 µg/mL polyA<br>
DNA. For TDH1, PGK1, and PDC1 probes, filters were washed after<br>
hybridization at room temperature in a solution of 2x SSC for 5 min and repeated,<br>
followed by two 30 min washes in a solution of 1x SSC - 0.1% SDS at 68°C. The<br>
post hybridization washes foT rDNA and PDC2 probes were performed twice for S<br>
min at room temperature in 2x SSC, followed by two 30 min. washes in 0.1x SSC<br>
-0.1%SDS at 68°C.<br>
Positive plaques were isolated and purified according to manufacturers<br>
instructions (Stratagene). Bacteriophage were purified using conventional methods<br>
(Sambrook et al., Id.), modified by eliminating DNAseI treatment and precipitating<br>
phage particles released from lysed host cells using PEG6000. Said phage particles<br>
were then dissolved in SM buffer and extracted with chloroform, pelleted by<br>
centrifugation at 25,000 rpm in Kontron TST41.14 rotor for 2 h, and again dissolved<br>
in SM buffer. Lambda DNA was isolated by digesting the phage particles with<br>
proteinase K followed by phenol extraction and ethanol precipitation.<br>
C. sonorensis genomic PNA inserts were partially sequenced using<br>
sequence-specific primers. The nucleotide sequences and the amino acid<br>
sequences deduced therefrom were compared against sequence databases in order<br>
to identify genes encoded in whole or part by the phage insert, using homology to<br>
known genes or proteins. The sequences obtained had significant similarity to<br>
fungal rPNA, phosphoglycerate kinases, glyceraldehyde-3-phosphate<br>
dehydrogenases, or pyruvate decarboxylases depending on the probe used for<br>
isolating each clone. The start and end points of the open reading, frames, encoding<br>
sequences of C. sonorensis PGK1, FDC1 and TDH1 were identified thereby.<br>
"Building-block" vectors. pMI203. PMI205 (Zeocin resistance vectors for<br>
C sonorensis), pVR24, and pVR27<br>
These plasmids are used in the construction of the vectors described in the<br>
Examples and are described in PCT application PCT/US01/44041. Briefly, the<br>
construction of these vectors is described.<br>
The plasmid pTEFl/Zeo (Invitrogen) containing the zeocin resistance<br>
marker under control of S. cerevisiae TEFl promoter was modified by adding a C<br>
sonorensis rDNA fragment to provide a target for homologous recombination.<br>
The following oligonucleotide primers:<br>
TGG ACT AGT AAA CCA ACA GGG ATT GCC TTA GT (SEQ ED No. 29)<br>
and<br>
CTA GTC TAG AGA TCA TTA CGC CAG CAT CCT AGG (SEQ ID No. 30),<br>
which correspond to C. sonorensis 26 S rRNA (Genbank Accession No. U70185),<br>
were used to amplify C. sonorensis genomic DNA to provide a PCR-amplified<br>
fragment of the 26S rDNA gene. The resulting PCR product fragment was digested<br>
with restriction enzymes SpeI and XbaI and ligated with pTEF/Zseo plasmid<br>
digested with XbaI. The resulting plasmid was designated pMI203 (FIG 23 B).<br>
The TEF1 promoter contained in pMI203 was replaced by a promoter of a<br>
gene from another Candida species, the C. albicans PGKl promoter. The<br>
following oligonucleotide primers:<br>
GCG ATC TCG AGG TCC TAG AAT ATG TAT ACT AATTTGC (SEQ ID<br>
No. 31)<br>
and<br>
ACT TGG CCA TGG TGA TAG TTA TTC TTC TGC AATTGA (SEQ ID No.<br>
32)<br>
were designed based on the available C. albicans PGKl sequence (Genbank<br>
Accession No. U25180). These primers were used to amplify a 700 bp fragment<br>
from the region upstream of the C. albicans PGKl open reading frame, using C.<br>
albicans genomic DNA as the template. Restriction sites Xbal and Spel<br>
(underlined above) were added to the primers to facilitate cloning of the fragment.<br>
After amplification, the fragment was isolated and digested with restriction enzymes<br>
XhoI and NcoI and then ligated to plasmid pMI203 digested with XhoI and NcoI.<br>
The resulting plasmid was designated pM1205 (FIG. 23 B).<br>
PVR24 and pVR27: Plasmid pBFY004 (proprietary, NREL) was digested<br>
with NotI restriction enzyme (Invitrogen), resulting in a 1235bp fragment (SEQ ID<br>
No: 33). The fragment was isolated and ligated to a NotI digested pGEM5zF(+)<br>
(Promega North, Madison, WI). E. coli (top 10) (Invitrogen) was transformed with<br>
the ligation mixture using standard electroporation protocols (Sambrook, Id.). The<br>
resultant plasmid was designated pNC002.<br>
B. megaterium DNA encoding the LDH gene was isolated as follows. B.<br>
megaterium was obtained from the American Type Culture Collection (ATCC<br>
Accession #6458) and grown under standard conditions. Genomic DNA was<br>
purified from these cells using an Invitrogen "Easy-DNA" kit according to the<br>
manufacturer"s protocol. Primers were designed on the basis of the available<br>
sequence in Genbank for the L-LDH from B. megaterium (Genbank accession #<br>
M22305). PCR amplification reactions were performed using standard<br>
techniques, with each reaction containing B. megaterium genomic DNA (6 ng/mL),<br>
the 4 dNTPs (0.2 mM), and the amplification primers BM1270 and BM179 (1 µM<br>
in each). The primers have the sequences:<br>
BM1270 CCTGAGTCCACGTCATTATTC (SEQ ED<br>
No:34<br>
and<br>
BM179 TGAAGCTATTTATTCTTGTTAC (SEQ ID<br>
No:35)<br>
Reactions were performed according to the following themocycling conditions: an<br>
initial incubation for 10 min at 95°C, followed by 35 cycles consisting of 30 sec at<br>
95°C, 30 sec. at 50°C, 60 sec at 72°C. A strong product fragment of 1100 base<br>
pairs (bp) was gel purified using conventional procedures, cloned, and sequenced.<br>
The resulting sequence could be translated into a polypeptide that exhibited<br>
excellent homology to known L-LDH-encoding genes.<br>
The coding sequence for the B. megaterium LDH-encoding disclosed<br>
herein was operatively linked to a promoter from the PGK1 gene and a<br>
transcriptional terminator from the GAL10 gene, both from the yeast<br>
Saccharomyces cerevisiae. Two oligonucleotide primers, Bmeg5" and Bmeg3",<br>
were designed based on this sequence to introduce restriction sites at the ends of<br>
the coding sequence of the gene:<br>
Bmeg5" GCTCTAGATGAAAACACAATTTACACC (SEQ ID<br>
No:36) and<br>
Bmeg3" ATGGATCCTTACACAAAAGCTCTGTCGC (SEQ ID<br>
No:37)<br>
This amplification reaction was performed using dNTP and primer concentrations<br>
described above using Pfu Turbo polymerase (Stratagene) in a buffer supplied by<br>
the manufacturer. Thermocycling was done by initially incubating the reaction<br>
mixture for 3 min at 95°C, then by 20 cycles of 30 sec at 95°C, 30 sec at 50°C, 60<br>
sec at 72°C, followed by a final incubation for 9 min at 72°C. The product was<br>
digested with restriction enzymes Xbal and BamHI and then ligated into the Xbal<br>
and BamHI sites of plasmid pNC002. This ligation resulted in the PGK promoter<br>
and GAL 10 terminator becoming operably linked to the B. megaterium LDH<br>
coding sequence (pVR24; FIG. 21).<br>
Construction of pVR27 (FIG. 22) was performed to create a vector<br>
containing R. oryzae LDH for its expression under the control of the S. cerevisiae<br>
PGK1 promoter. LDH was isolated from Rhizopus oryzae from genomic DNA<br>
purified ("Easy-DNA" kit, Invitrogen) from cells (ATCC Accession #9363) grown<br>
under standard conditions. Primers were designed on the basis of the available<br>
sequence in Genbank for the LDH from R. oryzae (Genbank accession #<br>
AF226154). PCR amplification reactions were performed using standard<br>
techniques, with each reaction containing R. oryzae genomic DNA (6 ng/mL), each<br>
of 4 dNTPs (0.2 mM), and each of the amplification primers Ral-5"and Ral-3" (1<br>
µM). The amplification primers had the sequence:<br>
Ral-5" CTTTATTTTTCTTTACAATATAATTC (SEQ ID<br>
No:38)and<br>
Ral-3" ACTAGCAGTGCAAAACATG (SEQ ID<br>
No:39)<br>
Reactions were performed according to the following cycling conditions: an initial<br>
incubation for 10 min at 95°C, followed by 35 cycles consisting of 30 sec at 95°C,<br>
30 sec. at 41°C, 60 sec at 72°C. A strong product fragment of 1100 bp was gel<br>
purified, cloned in TA vector (Invitrogen, Carlsbad, CA) and sequenced. The<br>
resulting sequence could be translated into a polypeptide that exhibited excellent<br>
homology to known Rhizopus oryzae LDH-encoding gene sequence in Genbank<br>
(Accession # AF226154).<br>
The coding sequence for the R. oryzae LDH-encoding gene disclosed<br>
herein was operatively linked to a promoter from the PGK1and a transcriptional<br>
terminator from the GAL10 gene, both from the yeast 5. cervisiae. In making this<br>
construct, the following oligonucleotides were prepared and used to amplify the<br>
coding sequence from the plasmid containing the Rhizopus LDH insert. Two<br>
oligonucleotide primers, Rapgk5 and Papgk3", were designed based on this<br>
sequence to introduce restriction sites at the ends of the coding sequence of the<br>
gene.<br>
Rapgk5 GCTCTAGATGGTATTACACTCAAAGGTCG (SEQ ID<br>
No:40) and<br>
Papgk3 GCTCTAGATCAACAGCTACTTTTAGAAAAG (SEQ ID<br>
No:41)<br>
This amplification reaction was performed using dNTP and primer concentrations<br>
as described above using Pfu Turbo polymerase (Stratagene) in a buffer supplied<br>
by the manufacturer. Thermocycling was done by initially incubating the reaction<br>
mixture for 3 min at 95°C, then by 20 cycles of 30 sec at 95°C, 30 sec at 53°C, 60<br>
sec at 72°C, followed by a final incubation for 9 min at 72°C. The product was<br>
digested with restriction enzymes XbaI and then ligated into the XbaI site of<br>
plasmid pNC002.<br>
This ligation resulted in the PGK promoter and GAL10 terminator<br>
becoming operably linked to the R. oryzae LDH coding sequence (pVR27; FIG.<br>
22)<br>
pMI234 and pMI238: In order to develop a positive selection for C.<br>
sonorensis transformants, the S. cerevisiae MEL5 gene (Naumov et al., 1990,<br>
MGG 224: 119-128; Turakainen et al., 1994, Yeast 10: 1559-1568; Genbank<br>
Accession No. Z37511) was obtained as the 2160 bp EcoRI-SpeI fragment from<br>
plasmid pMEL5-39 and ligated to pBluescript II KS(-) (Stratagene) digested with<br>
EcoRI and SpeI. The EcoRI site in the MEL5 gene is located 510 bp upstream of<br>
the initiator ATG, and the Spel site is located 250 bp downstream of the stop<br>
codon of MEL5. The resulting plasmid was designated pMI233 (FIG. 23 C).<br>
The 1500 bp PGKl promoter of C. sonorensis was amplified with primers<br>
having the sequence: GCG ATC TCG AGA AAG AAA CGA CCC ATC CAA<br>
GTG ATG (SEQ ID No. 5) and TGG ACT AGT ACA TGC ATG CGG TGA<br>
GAA AGT AGA AAG CAA ACA TTG TAT ATA GTC TTT TCT ATT ATT<br>
AG (SEQ ID No. 42) using DNA from the PGK1 lambda clone isolated above as<br>
template. The 3" primer can create a fusion between the C. sonorensis PGKl<br>
promoter and S. cerevisiae MEL5, since it corresponds to nucleotides present in the<br>
PGKl promoter immediately upstream of the open reading frame and nucleotides<br>
corresponding to the 5" end of MEL5 open reading frame. The resulting amplified<br>
fragment was digested with restriction enzymes SphI and XhoI and ligated to<br>
plasmid pMI233 (FIG. 23 C) digested with SphI and XhoI. The resulting construct<br>
in the plasmid contains C. sonorensis PGKI promoter upstream of and operatively<br>
linked to the MEL5 open reading frame, and is identified as pMI234 in FIG. 5.<br>
In a similar fashion, a 650 bp of the C. sonorensis TDHI promoter was<br>
amplified with primers having the sequence: GCG ATC TCG AGA AAA TGT<br>
TAT TAT AAC ACT ACA C (SEQ ID No. 3) and TGG ACT AGT ACA TGC<br>
ATG CGG TGA GAA AGT AGA AAG CAA ACA TTT TGT TTG ATT TGT<br>
TTG TTT TGT TTT TGT TTG (SEQ ID No. 43) using DNA from the TDHl<br>
lambda clone isolated above as the template. The 3" primer can create a fusion<br>
between C. sonorensis TDHI promoter and S. cerevisiae MEL5, since it corresponds<br>
to nucleotides present in the TDHI promoter immediately upstream of the open<br>
reading frame and nucleotides corresponding to the 5" end of MEL5 open reading<br>
frame. The amplified fragment was digested with SphI and XhoI and ligated to<br>
plasmid pMI233 (FIG. 23 C) digested with SphI and XhoI. The resulting plasmid,<br>
identified as pMI238 in FIG. 6, contains C. sonorensis TDHI promoter upstream of<br>
and operatively linked to the MEL5 open reading frame.<br>
pMI246 and pMI247: Plasmid pMI205 was used to produce a plasmid<br>
containing the MEL5 gene as a selectable marker and the LDH gene for enabling<br>
production of lactic acid in G sonorensis. In the resulting plasmid, the zeocin<br>
resistance gene in pMI205 was replaced by the L. helveticus LDH gene.<br>
A 1329 bp NcoI-BamHI fragment of pVR1 containing the LDH gene and<br>
the CYC1 terminator was ligated to the 3413 bp NcoI-BamHI fragment of pMI205<br>
(FIG. 23 B) bringing the L. helveticus LDH gene under control of the C. albicans<br>
PGK1 promoter; the resulting plasmid was named pMI214. In a second step the<br>
C. albicans PGK1 promoter was replaced by the C. sonorensis PGK1 promoter.<br>
The C. sonorensis PGK1 promoter was isolated by amplification from an isolated<br>
lambda clone as described above using primers having the sequence: GCG ATC<br>
TCGAGA AAG AAA CGA CCC ATC CAA GTG ATG (SEQ ID No. 5) and ACT<br>
TGG CCA TGG TAT ATA GTC TTT TCT ATT ATT AG (SEQ ED No. 44), and<br>
the PCR product was digested with XhoI and NcoI and ligated into pMI214<br>
digested with XhoI and NcoI. This plasmid was designated pMI277 and is shown<br>
in FIG. 19.<br>
The LDH expression cassette from pMI227 and MEL5 marker cassette<br>
from pMI234 were combined into the same vector by ligating a 3377 bp AvrII-<br>
NheI fragment of pMI227 (FIG. 23 A) with SpeI-digested pMI234 (FIG. 23 C).<br>
The resulting plasmid was designated pMI246 and is shown in FIG. 8.<br>
The LDH expression cassette from pMI227 and the MEL5 marker cassette<br>
from pMI238 were combined into the same vector by ligating a 3377 bp AvrII-<br>
NheI fragment of pMI227 with Spel-digested pMI238. The resulting plasmid was<br>
designated pMI247 and is shown in FIG. 9.<br>
In one embodiment, the invention provides recombinant nucleic acid<br>
constructs comprising a nucleotide sequence that encodes a polypeptide useful for<br>
the biosynthesis of an organic product, which is operatively linked to a promoter<br>
that is functional in the genera Candida.<br>
In related embodiments the nucleotide sequence encodes a lactate<br>
dehydrogenase gene. In preferred embodiments, the lactate dehydrogenase gene is<br>
heterologous to the Candida yeast cell into which it is introduced. In most<br>
preferred embodiments the lactate dehydrogenase gene is from a microorganism<br>
such as, for example, a bacterium or fungus, and the organic product produced<br>
according to the methods of the invention is lactic acid (or lactate).<br>
Typically, the methods of the invention for producing lactic acid can yield<br>
(based on grams of lactic acid produced / gram of a carbohydrate substrate<br>
consumed) about 60% or more, preferably about 70% or more, more preferably<br>
about 80% or more, and most preferably about 90% or more, when the<br>
carbohydrate substrate is a hexose, for example, glucose.<br>
The methods of the invention for producing lactic acid can result in lactic<br>
acid titers of about 75 grams/L or more, preferably about 90 grams/L or more, and<br>
most preferably about 100 grams/L or more. The cells of the invention have a<br>
specific productivity of lactic acid production (in terms of grams of lactic acid<br>
produced / gram of dry cell weight per hour) of about 0.20 or more, preferably<br>
about 0.30 or more, and most preferably about 0.50 or more, when a hexose<br>
carbohydrate substrate, such as glucose, is used for production.<br>
In one embodiment, the Crabtree-negative cells of the invention can<br>
catabolize starch, either naturally or because of a genetic modification. In<br>
additional embodiments, the cells are genetically modified to catabolize<br>
cellulosics through the addition of such molecules as fungal-based cellulases.<br>
In related embodiments, the cells of the invention can metabolize sugars<br>
other than glucose or other monosaccharide hexoses, in particular pentoses<br>
including the non-limiting examples of xylose and L-arabinose.<br>
The Crabtree-negative cells of the invention are preferably selected from<br>
the Candida strains C. sonorensis, C. methanosarbosa, C. diddensiae, C.<br>
pdrapsilosis, C. naeodendra, C. krusei, C. blankii, and C. entomophila. In<br>
preferred embodiments the cells C. sonorensis and C. methanosorbosa cells.<br>
Methods for isolating organic products produced by the cells of the<br>
invention are well known in the art. In particular, methods for separating lactic<br>
acid from a fermentation mixture, including low pH fermentation mixtures, are<br>
disclosed by Eyal et al. (International Patent Application, Publication No. WO<br>
99/39290, published April 22, 2999). Such methods for isolating lactic acid<br>
include extraction, adsorption, distillation/vaporization, separation via a<br>
membrane, crystallization, and phase splitting. {See also: Vickroy, 1985,<br>
Comprehensive Biotechnollgy, (Moo-Young, ed.), Volume 3, Chapter 38<br>
Pergamon Press, Oxford; Datta et al, 1995, FEMS Microbiol. Rev. 16: 221-231;<br>
U.S. Patent 4,771,001; U.S. Patent 5,132,456; U.S. Patent 5,510,526; and U.S.<br>
Patent 5,420,304).<br>
Fermentation Conditions<br>
Various fermentation processes can be used with the various aspects of the<br>
instant invention. (See, e.g., Wolf, 1996, Nonconventional Yeasts in<br>
Biotechnology. Springer-Verlag Berlin, and Walker, 2000, Yeast Physiology and<br>
Biotechnology. John Wiley &amp; Sons, England). Those of skill in the art will<br>
recognize that fermentation conditions can be varied to improve various aspects of<br>
the fermentation, including product yield, culture productivity, and culture health<br>
(among others), depending on the specific host organism and desired product. It is<br>
particularly advantageous to use the favorable characteristics of Candida in<br>
adjusting the fermentation conditions. Thus, the pH can have a range during<br>
various stages of processing from about 2.5 to about 9.0. Oxygen levels can vary<br>
from about 0% to about 100% (relative to the oxygen content found in air), as<br>
measured in the atmosphere above the medium or dissolved in the medium.<br>
Oxygen levels can be measured or calculated by any common methods including<br>
partial pressure, O2 electrode, volume/volume, or gas flow rate (VVM).<br>
Temperature ranges can span from about ambient temperature (23°C) to about<br>
40°C and above (e.g. to about 45°C).<br>
Preferred fermentation conditions include maintenance of a pH range from<br>
about 4 to about 5. It is especially preferred to maintain a pH of about 5 during<br>
biomass production and during lactic acid production. Preferably, the pH is<br>
maintained throughout the entirety of the fermentation process by automated<br>
addition of a base, for example Ca(OH)2. The temperature during biomass<br>
production is preferably maintained at about 35°C. Preferably the biomass is<br>
produced under aerobic conditions wherein the culture medium is preferably<br>
agitated and supplied an airflow, until an adequate cell mass for lactic acid<br>
production is attained. During production of lactic acid, the agitation rate and<br>
airflow are preferably slowed, relative to their rate during biomass production.<br>
Examples<br>
Example 1: G418 resistance vectors and use of G418 for selection of C.<br>
sohorensis transformants<br>
Vectors conferring G418 resistance on transformed yeast cells, which<br>
permit selection of yeast cell transformants comprising a recombinant nucleic acid<br>
construct encoding a protein useful for synthesis of an organic product, were<br>
prepared as follows. The G418 resistance marker was cloned to be under the<br>
transcriptional control of either the C. sonorensis PGK1 or TDH1 promoter and<br>
the constructs were designated as pMI268 (FIG. 2) and pMI269 (FIG. 3),<br>
respectively. The S. cerevisiae GAL10 terminator was used in both cases.<br>
The G418 resistance gene was amplified by polymerase chain reaction<br>
(PCR) using the Dynazyme EXT Polymerase (Finnzymes, Espoo, Finland) using a<br>
pair of oligonucleotide primers having the sequence: CTAGTCTAGA ACA ATG<br>
AGC CAT ATT CAA CGG GAA ACG (G418 5"; SEQ ID NO:1) and CGC<br>
GGATCC GAA TTC TTA GAA AAA CTC ATC GAG CAT CAA ATG (G418<br>
3"; SEQ ID NO:2). The plasmid pPIC9K (obtained from Invitrogen) was used as<br>
template. PCR was performed by initially incubating the reaction mixture for 5<br>
min at 95°C, followed by 29 cycles of 45 sec at 95°C, 45 sec at 55°C, and 2 min at<br>
72°C, with a final incubation for 5 min at 72°C. The PCR product was digested<br>
with restriction enzymes BamHI and XbaI and an 800 bp fragment was isolated.<br>
This fragment was ligated to the 4226 bp BamHI-XbaI fragment of pNC101<br>
(obtained from Eric Jarvis at NREL). Plasmid pNClOl was constructed from the<br>
phosphoglycerate kinase promoter (pPGK) and the GAL10 terminator sequences<br>
from S. cerevisiae, using standard cloning techniques (see, e.g., Sambrook et al.,<br>
Id.). This plasmid also harbors an LDH gene from K. thermotolerans inserted<br>
between XbaI and EcoRI sites, which, along with a BamHI site, are contained in a<br>
polylinker region found between the yeast promoter and terminator sequences.<br>
This plasmid permits expression of various genes or selectable markers, under the<br>
control of the yeast promoter and terminator.<br>
The plasmid resulting from these manipulations contains the G418<br>
resistance gene between the S. cerevisiae PGK1 promoter and the S. cerevisiae<br>
GAL10 terminator, and was named pMI260. The structure of this plasmid is<br>
shown schematically in FIG. 1.<br>
The 600 bp TDH1 promoter of C. sonorensis was amplified by PCR using<br>
the Dynazyme EXT Polymerase with a pair of oligonucleotide primers having the<br>
sequence: GCG ATC TCG AGA AAA TGT TAT TAT AAC ACT AC A C (5441;<br>
SEQ ID NO:3) and CTAGTCTAGATT TGT TTG ATT TGT TTG TTT TGT<br>
TTT TGT TTG (Cs1; SEQ ID NO:4) using pMI238 as a template (see above<br>
"Vectors and Host Cells"; shown in FIG. 6). PCR was performed by initially<br>
incubating the reaction mixture for 5 min at 95°C, followed by 29 cycles of 45 sec<br>
at 95°C, 45 sec at 55°C, 2 min at 72°C, with a final incubation for 5 min at 72°C.<br>
The PCR product was made blunt ended with Klenow polymerase and each of the<br>
4 dNTPs and then digested with the restriction enzyme XbaI. The resulting 600bp<br>
fragment was ligated with the 4216 bp PstI (made blunt ended with T4<br>
polymerase)-XbaI fragment of pMI260. The resulting plasmid contains the G418<br>
resistance gene operatively linked to the C. sonorensis TDH1 promoter and the S.<br>
cerevisiae GAL10 terminator and was named pMI269. The structure of this<br>
plasmid is shown schematically in FIG. 3.<br>
The 1500 bp C. sonorensis PGK1 promoter was amplified by PCR using<br>
the Dynazyme EXT Polymerase with a pair of oligonucleotide primers having the<br>
sequence: GCG ATC TCG AGA AAG AAA CGA CCC ATC CAA GTG ATG<br>
(5423; SEQ ID NO:5) and CTA GTC TAG ATG TAT ATA GTC TTT TCT ATT<br>
ATT AG (Cs2;SEQ ID NO:6) using pMI234 as the template {see above "Vectors<br>
and Host Cells"; FIG. 5). PCR was performed by initially incubating the reaction<br>
mixture for 5 min at 95°C, followed by 29 cycles of 45 sec at 95°C, 45 sec at<br>
55°C, 2 min at 72°C, with a final incubation for 10 min at 72°C. The 1500 bp<br>
PCR product fragment was made blunt ended with Klenow polymerase and each<br>
of the 4 dNTPs and then digested with the restriction enzyme XbaI. The 1500 bp<br>
PGK1 promoter fragment was ligated with the 4216 bp PstI (made blunt ended<br>
with T4 polymerase)-XbaI fragment of pMI260. The resulting plasmid contains<br>
the G418 resistance gene operatively linked to the C. sonorensis PGK1 promoter<br>
and the S. cerevisiae GAL10 terminator, and was named pMI268. The structure of<br>
this plasmid is shown schematically in FIG. 2.<br>
The two constructs pMI268 and pMI269 were digested with restriction<br>
enzymes SaiI and NotI and transformed into C. sonorensis using the chemical<br>
method according to Gietz et al. (1992, Nucleic Acids Res. 20:1425). This<br>
transformation technique was used throughout these Examples, and is described<br>
briefly as follows.<br>
Cells from an overnight culture of C. sonorensis grown to an OD600 of<br>
0.8-1.5 were collected by centrifugation, and were washed first with an excess of a<br>
solution of 10 mM Tris-HCl, 1 mM EDTA (pH 7.5), followed by washing with an<br>
excess of a solution of 100 mM lithium acetate (LiAc), 10 mM Tris-HCl, 1 mM<br>
EDTA (pH 7.5), and then resuspended in 2 mL of a solution of 100 mM LiAc, 10<br>
mM Tris-HCl, 1 mM EDTA (pH 7.5). Cells were mixed (about 50 µL of the 2mL<br>
suspension) with about 10 µg of transforming DNA and 300 µL of a solution of<br>
40% PEG4000, 100 mM LiAc, 10 mM Tris-HCl, 1 mM EDTA (pH 7.5). The<br>
cells were incubated at 30°C for 30 min with slow shaking. Dimethyl sulfoxide<br>
(DMSO; 40 µL) was added and the cells were incubated in a 42°C water bath for<br>
15 min. The cells were collected by centrifugation, washed with an excess of a<br>
solution of 10 mM Tris-HCl, 1 mM EDTA (pH 7.5), resuspended and incubated at<br>
30°C in YPD medium (comprising 10g/L yeast extract, 20g/L peptone and 20 g/L<br>
glucose) for 3-7 h. Optionally, the YPD incubation can be continued overnight.<br>
Before applying selection the cells were incubated in liquid YPD for at<br>
least 3 h or overnight. The transformants were grown on YPD agar plates<br>
(comprising 10g/L yeast extract, 20g/L peptone, 20 g/L glucose and 2% agar)<br>
supplemented with G418 antibiotic at a concentration of either 100 µg/mL or 200<br>
µg/mL. The plates were incubated at 30°C for 2-5 days and the transformants were<br>
then restreaked onto fresh selection plates. Southern analysis of total DNA<br>
isolated from the G418 resistant colonies showed that the G418 resistance gene<br>
was integrated in the genome of the transformants.<br>
These results showed that the G418 resistance gene can be expressed from<br>
the constructs prepared as described herein and is a suitable selection for C.<br>
sonorensis transformation.<br>
Example 2: Hygromycin resistance (hgh) vectors and use of hygromycin B<br>
for selection of C. sonorensis transformants<br>
Vectors conferring hygromycin resistance on transformed yeast cells,<br>
which permit selection of yeast cell transformants comprising a recombinant<br>
nucleic acid construct encoding a protein useful for synthesis of an organic<br>
product, were prepared as follows. The hygromycin resistance marker (E. coli<br>
hph) was cloned under the transcriptional control of either the C. sonorensis<br>
PGK1 and TDH1 promoter and the constructs were designated as pMI270 (FIG. 4)<br>
and pMI271, respectively. The S. cerevisiae GAL10 terminator was used in both<br>
cases.<br>
The E. coli hph gene that confers resistance to hygromycin B was obtained<br>
from the plasmid pRLMex30 (Mach et al. 1994, Curr. Genet. 25, 567-570).<br>
pRLMex30 was linearized with the restriction enzyme NsiI and made blunt ended<br>
with T4 DNA polymerase and then digested with XbaI.<br>
The pMI268 plasmid prepared in Example 1 was digested with EcoRI and<br>
was made blunt ended with Klenow polymerase and each of the 4 dNTPs and then<br>
digested with XbdI. The resulting 4900 bp fragment was ligated with the 1035 bp<br>
hph fragment from pRLMex30. This ligation produced a plasmid that contains the<br>
hygromycin resistance gene operatively linked to the C. sonorensis PGK1<br>
promoter and the S. cerevisiae GAL10 terminator, and was named pM1270. The<br>
structure of this plasmid is shown schematically in FIG. 4.<br>
The pMI269 plasmid prepared in Example 1 was digested with EcoRI and<br>
was made blunt ended with Klenow polymerase and each of the 4 dNTPs and then<br>
digested with XbaI. The resulting 4000 bp fragment was ligated with the 1035 bp<br>
hph fragment of pRLMex30. This produced a plasmid that contains the<br>
hygromycin resistance gene operatively linked to the C. sonorensis TDH1<br>
promoter and the S. cerevisiae GAL10 terminator, and was named pMI271. The<br>
structure of this plasmid is shown schematically in FIG. 7.<br>
Yeast cells were transformed using the chemical method according to Gietz<br>
et al. (1992, Nucleic Acids Res. 20: 1425) as described in Example 1 above. The<br>
two constructs pMI270 and pMI271 were digested with the restriction enzymes<br>
XhoI and NotI. The transformation mixture was incubated in YPD at 30°C for 3h<br>
before plating onto selective plates. The transformants were grown at 30°C for 2-<br>
5 days on YPD agar plates supplemented with hygromycin B (Calbiochem) at<br>
concentrations of 150-300 µg/mL. Transformants were restreaked onto fresh<br>
selection plates. The presence of the transformed DNA in the genome of the<br>
hygromycin resistant transformants was verified by PCR using a pair of<br>
oligonucleotide primers having the sequence: CCGGACTA GTT GGT ACA<br>
GAG AAC TTG TAA ACA ATT CGG (ScerGallOt; SEQ ID NO:7) and TAT<br>
AAA TAC TTA TCA TTT CCTCC (5436; SEQ ID NO:8). PCR was performed<br>
by initially incubating the reaction mixture for 3 min at 94°C, followed by 29<br>
cycles of 45 sec at 94°C, 45 sec at 55°C, 2 min at 72°C, with a final incubation for<br>
10 min at 72°C.<br>
These results show that the E. coli hph gene can be expressed using the<br>
constructs described herein, functions in C. sonorensis and that hygromycin B can<br>
be used to select C. sonorensis transformants.<br>
Example 3: Vectors for expression of the L. helveticus LDH and for targeted<br>
integration of the transformed DNA into the PDC1 locus<br>
Vectors comprising a L. helveticus LDH gene for targeted integration into<br>
the C sonorensis PDC1 gene locus were prepared as follows. The pMI246 vector<br>
contains the MEL5 expression cassette and the L. helveticus LDH expression<br>
cassette, shown schematically in FIG. 8 (see above "Vectors and Host Cells"). In<br>
order to construct a vector that enables targeted integration into the C. sonorensis<br>
PDC1 locus and replacement of the PDC1 protein-coding region, DNA fragments<br>
corresponding to sequences immediately upstream and downstream of the PDC1<br>
protein-coding region were added into pMI246.<br>
The PDC1 terminator was amplified by PCR using the Dynazyme EXT<br>
Polymerase (Finnzymes, Espoo, Finland) with oligonucleotide primers having the<br>
sequence: GGG ACT AGT GGA TCC TTG AAG TGA GTC AGC CAT AAG<br>
GAC TTA AATTCACC (Cs7; SEQ ID NO:9) and AAGGCCT TGT CGA CGC<br>
GGC CGC TTG GTT AGA AAA GGT TGT GCC AAT TTA GCC (Cs8; SEQ ID<br>
NO: 10), using C. sonorensis genomic DNA as a template. PCR was performed by<br>
initially incubating the reaction mixture for 5 min at 95°C, followed by 29 cycles<br>
of 45 sec at 95°C, 45 sec at 55°C, 2 min at 72°C, with a final incubation for 10<br>
min at 72°C. The 920 bp PCR product fragment was digested with restriction<br>
enzymes BamHI and NotI and the 920 bp fragment was purified and ligated with<br>
the 8900 bp BamHI-NotI fragment from pMI246. The resulting plasmid was<br>
named pMI256, and is shown schematically in FIG. 18.<br>
The PDC1 promoter was amplified from C. sonorensis with a pair of<br>
oligonucleotide primers having the sequence: GGG ACG GGC CCG CGG CCG<br>
CTA CAA GTG ATT CAT TCA TTC ACT (Cs5; SEQ ID NO:11) and CCC<br>
TGG GCC CCT CGA GGA TGA TTT AGC AAG AAT AAA TTA AAA TGG<br>
(Cs6; SEQ ED NO: 12) using genomic C. sonorensis DNA as a template. PCR was<br>
performed by initially incubating the reaction mixture for 5 min at 95°C, followed<br>
by 29 cycles of 45 sec at 95°C, 45 sec at 55°C, 2 min at 72°C, with a final<br>
incubation for 10 min at 72°C. The PCR product fragment was digested with<br>
Apal and the 800 bp fragment was purified and ligated with the 9760 bp Apal<br>
linearized pMI256 (see above "Vectors and Host Cells"; FIG. 18). The resulting<br>
plasmid was named pMI257, and is shown schematically in FIG. 10. pMI257<br>
contains, in order, the C. sonorensis PDC1 promoter, the C. sonorensis PGK1<br>
promoter operatively linked to the 5. cerevisiae MELS gene, the C. sonorensis<br>
PGK1 promoter operatively linked to the L. helveticus LDH and the S. cerevisiae<br>
CYCl terminator followed by the C. sonorensis PDC1 terminator.<br>
pMI257 was digested with Noil to excise the 7300 bp fragment containing<br>
the MEL5 and LDH expression cassettes flanked by the PDC1 5" and 3" regions.<br>
This 7300 bp fragment was used to transform C. sonorensis by the method<br>
described in Example 1 above, and the transformants were screened based on<br>
expression of the MEL 5 marker. The transformants were grown on YPD agar<br>
plates supplemented with the chromogenic substrate of a-galactosidase, 5-bromo-<br>
4-chloro-3-indolyl-a-D-galactopyranoside (X-a-gal; ICN Biochemicals) at a<br>
concentration of 40 µg/mL. The plates were incubated at 30°C for 1-3 days and<br>
then transferred to 4°C. In the presence of X-a-gal yeast colonies transformed<br>
with a functional MEL5 expression cassette turned blue, whereas the<br>
untransformed colonies were white. Blue colonies were purified by restreaking<br>
them onto fresh indicator plates. The transformants originating from the<br>
transformation of C. sonorensis with Nod digested pMI257 were designated as<br>
257-1 through 257-15, 257-41, 257-42, and 257-45.<br>
Southern blot analysis of genomic DNA isolated from the pMI257<br>
transformants was carried out with the C. sonorensis PDC1 probe to identify<br>
transformants in which the anticipated replacement of the PDC1 open reading<br>
frame by the transformed pMI257 DNA had occurred. The absence of a PDC1<br>
hybridizing band in transformants 257-3, 257-9, 257-12, 257-15, and 257-41<br>
indicated that PDC1 gene was deleted. Other pMI257 transformants and C.<br>
sonorensis gave a positive signal in the PDC1 hybridization. Hybridization with<br>
the L. helveticus LDH probe showed that the LDH gene was present in one copy in<br>
the PDC1 deletants. Transformants 257-6 257-7, and 257-8 contained two copies<br>
of the L. helveticus LDH randomly integrated into the genome. Other pMI257<br>
transformants had one copy of LDH randomly integrated in the genome.<br>
These results show that targeted integration of the transformed pMI257<br>
DNA into the PDC1 locus occurred through homologous recombination between<br>
PDC1 promoter and terminator sequences. These results also show that PDC1 is a<br>
single copy gene in C. sonorensis. In addition, integration events outside the<br>
PDC1 locus occurred. In some transformants the LDH gene was integrated in<br>
more than one copy into the genome.<br>
Example 4: Vectors for expression of the B. megaterium LDH and for<br>
targeted integration of the transformed DNA into the PDC1 locus<br>
Vectors comprising a B. megaterium LDH gene for targeted integration<br>
into the C sonorensis PDC1 gene locus were prepared as follows. In these<br>
vectors, the L helveticus LDH in pMI257 was replaced by the B. megaterium<br>
LDH.<br>
pMI257 was linearized with Ncol and the 5" overhangs were partially filled<br>
in with DNA polymerase I, Klenow fragment, and a mixture of dATP, dCTP, and<br>
dTTP, omitting dGTP from the reaction. This was followed by removal of the<br>
single stranded extensions by treatment with mung bean nuclease. The DNA was<br>
then digested with BamHI and the 9200 bp fragment was isolated from a 0.8%<br>
agarose gel after electrophoretic separation. Vector pVR24 containing B.<br>
megaterium LDH was generated from B. megaterium genomic DNA, and is<br>
shown in FIG. 21. The LDH gene was cloned from the genomic DNA by PCR<br>
using primers designed from accession no. M22305, and ligated into a<br>
commercially-available vector (see above "Vectors and Host Cells")- The 976 bp<br>
fragment containing the B. megaterium LDH was excised from pVR24. by XbaI<br>
digestion followed by fill-in of the 5" overhangs by DNA polymerase I, Klenow<br>
fragment and each of the 4 dNTPs, and digestion by BamHI. The 9200 bp Ncol<br>
(blunt)-BamHI fragment from pMI257 and the 976 bp XbaI(blunt)-BamHI<br>
fragment from pVR24 were ligated and the resulting plasmid was designated as<br>
pMI265, shown in FIG. 11. pM1265 contains, in order, the C. sonorensis PDC1<br>
promoter, the C. sonorensis PGK1 promoter operatively linked to the S. cerevisiae<br>
MEL5 gene, the C sonorensis PGK1 promoter operatively linked to the B.<br>
megaterium LDH and the C. sonorensis PDC1 terminator. pMI265 was digested<br>
with NotI to excise the 7300 bp fragment that consisted of the MEL5 and LDH<br>
expression cassettes flanked by the PDC1 5" and 3" regions. This 7300 bp<br>
fragment was used to transform C sonorensis as described in Example 1 and the<br>
transformants were screened on YPD plates supplemented with X-a-gal at a<br>
concentration of 40 mg/mL. The transformants were grown on YPD agar plates<br>
supplemented with X-a-gal (40 µg/ml-). The transformants originating from the<br>
transformation of C. sonorensis with NotI-digested pMI265 were designated as<br>
265-1 through 265-60.<br>
Southern blot analysts of genomic DNA isolated from the pMI265<br>
transformants was carried out with the C. sonorensis PDC1 probe to identify<br>
transformants in which the anticipated replacement of the PDC1 open reading<br>
frame by the transformed pMI265 DNA had occurred. The absence of a PDC1<br>
hybridizing band in transformants 265-5, 265-7, 265-15, 265-17, 265-33, 265-34,<br>
265-35. 265-38, 265-39, 265-42, 265-43, 265-47. 265-48, 2G5-49, 265-5], and<br>
265-60 indicated that the PDC1 gene was deleted. Other pMI265 transformants<br>
and untransformed C. sonorensis gave a positive signal for PDC1 hybridization.<br>
Hybridization with the B. megaterium LDH probe showed that the LDH gene was<br>
present in one copy in the pdcl deletants. Positively PDC1-hybridizing<br>
transformants 265-14, 265-22 and 26S-23 contained two copies and 265-56<br>
contained three copies of the LDH gene randomly integrated into the genome.<br>
Other pMl265 transformants had one copy of LDH randomly integrated in the<br>
genome.<br>
These results showed that targeted integration of the transformed pMI265<br>
DNA into the PDC1 locus occurred through homologous recombination between<br>
PDC1 promoter and terminator sequences. These results also confirmed that<br>
PDC1 is a single copy gene in C. sonorensis. The transformants deleted of pdc1<br>
were viable, indicating that PDC1 is not an essential gene in C sonorensis. In<br>
addition to PDC1 -deleting integrations, integration events outside the PDC1 locus<br>
occurred in certain transformants. In some transformants the LDH gene was<br>
integrated in more than one copy into the genome<br>
Example 5: Victors for expression of R. orytae LDH and for targeted<br>
integration of the transformed DNA into the PDC1 locus<br>
Vectors comprising a r. oryzae LDH gene for targeted integration into the<br>
C sonorensis PDC1 gene focus were prepared as follows. In these vectors, the L.<br>
helveticus LDH encoding sequences in pMI257 were replaced by R. oryzae LDH.<br>
The pMI2S7 plasmid described in Example 3 above was linearised with<br>
NcoI and the 5" overhangs were partially filled in with DNA polymerase I, Klenow<br>
fragment, and a mixture of dATP, dCTP, and dTTP, omitting dGTP from the<br>
reaction. This was followed by removal of the single stranded extensions by<br>
treatment with mung bean nuclease. The DNA was then digested with BamHI and<br>
the 9200 bp fragment was isolated from a 0.8% agarose gel after electrophoretic<br>
separation. The coding sequence of R. oryzae LDH-encoding DNA was<br>
operatively linked to the C. sonorensis PGK1 promoter and the transcriptional<br>
terminator of the C. sonorensis PDC1 gene. Vector pVR 27 containing R. oryzae<br>
LDH was generated from R. oryzae genomic DNA, and is shown in FIG. 22. The<br>
LDH gene was cloned from the genomic DNA by PCR using primers designed<br>
from accession no. AF226154, and ligated into a commercially-available vector<br>
{see above "Vectors and Host Cells"). The 978 bp fragment containing the R.<br>
oryzae LDH was excised from pVR27 by XbaI digestion followed by fill in of the<br>
5" overhangs by DNA polymerase I, Klenow fragment and each of the 4 dNTPs<br>
and digestion by BamHI. The 9200 bp Ncol (blunt)-SamHI fragment from<br>
pMI257 and the 978 bp XbaI(blunt)-BamHI from pVR27 were ligated and the<br>
resulting plasmid containing was designated as pMI266, shown schematically in<br>
FIG. 12. pM1266 contains, in order, the C. sonorensis PDC1 promoter, the C.<br>
sonorensis PGK1 promoter operatively linked to the S. cerevisiae MEL5 gene, the<br>
C. sonorensis PGK1 promoter operatively linked to the R. oryzae LDH A and the<br>
C. sonorensis PDC1 terminator. pMI266 was digested with NotI to excise a 7300<br>
fragment that consisted of the MEL5 and LDH expression cassettes flanked by the<br>
PDC1 5" and 31 regions. This 7300 bp fragment was used to transform C.<br>
sonorensis by the method described in Example 1 above, and the transformants<br>
were screened on YPD plates supplemented with X-a-gal at a concentration of 40<br>
µg/mL. The transformants originating from the transformation of C. sonorensis<br>
with NotI-digested pMI266 were designated as 266-1 through 266-13.<br>
Southern blot analysis of genomic DNA isolated from the pMl266<br>
transformants was carried out with the C. sonorensis PDC1 probe to identify<br>
transformants in which the anticipated replacement of the PDC1 open reading<br>
frame by the transformed pMI266 DNA had occurred. The absence of a PDC1<br>
hybridizing band in transformants 266-1, 266-3, 266-4, and 266-11 indicated that<br>
PDC1 gene was deleted. In contrast, pMI266 transformants 266-2, 266-7 and<br>
266-8 and untransformed C. sonorensis gave a positive signal in the PDC1<br>
hybridization. Hybridization with the LDH probe showed that the R. oryzae LDH<br>
gene was present in one copy in ail the transforrnants.<br>
These results showed that targeted integration of the transformed pMI266<br>
DNA into the PDC1 locus occurred through homologous recombination between<br>
PDC1 promoter and terminator sequences. In addition, integration events outside<br>
the PDC1 locus occurred.<br>
Example 6: Vector for replacement of PDC1 without LDH<br>
Vectors were prepared for replacing PDC1 without introducing exogenous<br>
LDH-encoding sequences. The pMI257 plasmid described in Example 3 above<br>
was digested with Ncol and BamHI in order to remove the LDH gene and the S.<br>
cerevisiae CYC1 terminator. The 5" overhangs were filled in by DNA polymerase<br>
I, Klenow fragment, and each of the 4 dNTPs. The 9200 bp fragment was purified<br>
after agarose gel electrophoresis and recircularized by incubation at a<br>
concentration of 40 ng/mL in the presence of 400 U of T4 DNA ligase (New<br>
England Biolabs) and the appropriate buffer recommended by the manufacturer.<br>
The resulting plasmid was named pMI267, and shown schematically in FIG. 13.<br>
pM1267 contains, in order, the C. sonorensis PDC1 promoter, the C. sanorensis<br>
PGK1 promoter imperatively linked to the S. cerevisiae MELS gene, and the C.<br>
sonorenus PDC1 terminator<br>
pMI267 was digested with NatI to excise the 6300 bp fragment that<br>
consisted of the MEL5 cassette flanked by the PbCJ 5" and 3" regions. This 6300<br>
bp fragment was used to trarnsform C. sonorensis by the method describe above<br>
in Example 1 and the transformants were screened on YPD plates supplemented<br>
with X-a-gal at a concentration of 40 µg/ml,. The transformants originating, from<br>
transformation of C. sonorensis with NotI digested pMI267 were designated as<br>
267-1 through 267-10.<br>
Southern blot analysis of genomic DNA, isolated from the pMI267<br>
transformants was carried out with the C. sonorensis PDC1 probe to identify<br>
transformants in which PDCl open reading frame was deleted. The absence of a<br>
PDC1 hybridizing band in transformants 267-1 and 267-10 indicate that the<br>
PDC1 gene was deleted.<br>
These results showed that targeted integration of the transformed pMI267<br>
DNA into the PDC1 locus occurred through homologous recombination between<br>
the PDC1 promoter and terminator sequences. LDH expression was not required<br>
to maintain the viability of the pdcl-deleted strain. In addition, integration events<br>
outside the PDC1 locus occurred.<br>
Example 7: Construction of a C. sonorensis vector containing the B.<br>
megaterium LDH gene and the G418 marker<br>
A vector comprising the G418 resistance gene and B. megaterium LDH<br>
gene was prepared as follows. In these vectors, the B. megaterium LDH<br>
expression cassette from the plasmid pMI265 and the G418 resistance marker<br>
cassette from the plasmid pM1269 were combined into the same vector. The<br>
pMI269 plasmid described in Example 1 was digested with £coRI and the 5"<br>
overhangs were filled in by DNA polymerase I, Klenow fragment, and each of the<br>
4 dNTPs, followed by digestion of the DNA with BamHI. The 4800 bp<br>
EcoRI(blunt) -BamHI fragment of pMI269 was ligated with 2800 bp MscI -<br>
BamHI fragment from the pMI265 plasmid described in Example 4. The resulting<br>
plasmid was named pMI278 and contains, in order, the C. sonorensis TDH1<br>
promoter operatively linked to the G418 resistance gene and the MEL5 terminator<br>
followed by the C. sonorensis PGK1 promoter operatively linked to the B.<br>
megaterium LDH and the S. cerevisiae GAL10 terminator, and is shown<br>
schematically in FIG. 14.<br>
Example 8: Construction of C. sonorensis strains expressing the R. oryzae<br>
LDH and the B. megaterium LDH simultaneously<br>
The C. sonorensis transformant designated 266-3, in which the R. oryzae<br>
LDH is integrated into the PDC1 locus, was chosen as host for a second<br>
transformation with the B. megaterium LDH construct described in Example 4<br>
above. Transformant 266-3 was further transformed with SalI-NotI digested<br>
pMI278 and the transformants were selected on YPD agar plates supplemented<br>
with G418 antibiotic at a concentration of 200 mg/mL. The plates were incubated<br>
at 30°C for 2-5 days for selection; transformants were purified by restreaking onto<br>
fresh selection plates. The resulting transformants were designated as 278-1<br>
through 278-20. The presence of the B. megaterium LDH in the genome of 19 of<br>
these transformants was verified by Southern biot analysis of HindIII digested<br>
yeast DNA using the B. megaterium LDH gene as the probe. Some of the<br>
transformants had more than one copy of the B. megaterium LDH integrated in the<br>
genome. Southem blot analysis was repeated with the R. oryzae LDH gene as a<br>
probe to verify that the R. oryzae LDH was still present.<br>
This experiment showed that C. sonorensis could be transformed multiply<br>
and independently with different markers. In this way it was demonstrated to be<br>
possible to increase the copy number of the gene of interest (LDH) in the host<br>
genome.<br>
Example 9: Vectors for expression of B. megaterium LDH and for targeted<br>
integration of the transformed DNA into the PDC2 locus<br>
Vectors comprising a B. megaterium LDH gene for targeted integration<br>
into the C. sonorensis PDC2 gene locus were prepared as follows C. sonorensis<br>
PDC2 promoter was amplified by PCR using the Dynazyme EXT polymerase and<br>
a pair of oligonucleotide primers having the: sequence: GGG ACG GGC CCG<br>
CGG CCG CTT ACA GCA GCA AAC AAG TGATGCC (Cs26; SEQ ID NO: 13)<br>
and CCC TGG GCC CCT CGA GTT TGA TTT ATT TGC TTT GTA<br>
AAGAGAA (Cs27; SEQ ID NO: 14). The genomic copy of the C sonorensis<br>
PDC2 cloned in a lambda vector was used as the template (see above). PCR was<br>
performed by initially incubating the reaction mixture for 3 min at 94°C, followed<br>
by 29 cycles of 45 sec at 94°C, 45 sec at 55°C, 2 min at 72"C, with a final<br>
incubation for 10 min at 72°C. The 1000 bp PCR product was cloned into the<br>
TOPO TA vector (Invitrogen) and the resulting piasmjd was named pMI277,<br>
shown schematically in FIG. 19. The PDC2 promoter was released by EcoRI<br>
digestion and made blunt ended with the Klenow polymerase and each of the 4<br>
dNTPs.<br>
The pMI278 plasmid prepared as described in Example 7 was linearized by<br>
Sail and the 5" overhangs were filled in by Klenow polymerase and each of the 4<br>
dNTPs, then ligated to the 1000 bp EcoRI (blunt) fragment of the pMI277<br>
plasmid. The plasmid containing the insert in the desired orientation was named<br>
pMI279, shown schematically in FIG. 20.<br>
The PDC2 terminator was amplified by PCR using the Dynazyme EXT<br>
polymerase with a pair of oligonucleotide primers having the sequence:<br>
TGGACTAGTTAGATAG CAA TTC TTA CTT GAA AAA TTA ATT GAA<br>
GCA TTACC (Cs29; SEQ ID NO: 15) and GGC CCG CGG CCG CTA AAT<br>
ATA ATT ATC GCT TAG TTA TTA AAA TGG (Cs30; SEQ ID NO:16), using<br>
the genomic copy of the C. sonorensis PDC2 gene cloned in a lambda vector as<br>
the template. The pdc2 terminator fragment includes part of the open reading<br>
frame corresponding to the 239 C-terminal amino acids. Translation stop codons<br>
were introduced in the PCR oligonucleotide Cs29 in all three frames upstream of<br>
the nucleotides corresponding to the last 239 C-terminal amino acids protein in the<br>
terminator fragment. PCR was performed by initially incubating the reaction<br>
mixture for 3 min at 94°C, followed by 29 cycles of 45 sec at 94°C, 45 sec at<br>
55°C, 2 min at 72°C, with a final incubation for 10 min at 72°C. The PCR<br>
product was made blunt ended with the Klenow polymerase and each of the 4<br>
dNTPs, and purified with a Qiaquick column (Qiagen). The PCR product was<br>
phosphorylated with T4 polynucleotide kinase and rATP at a concentration of 1<br>
mM under standard conditions (see Sambrook et al., Id.). The 800 bp PDC2<br>
terminator fragment was purified after agarose gel electrophoresis and ligated with<br>
NcoI (blunt) digested pMI279 that was dephosphorylated with calf intestinal<br>
phosphatase. The resulting plasmid was named pMI286 and contains, in order,<br>
the C. sonorensis PDC2 promoter, the C. sonorensis TDH1 promoter operatively<br>
linked to the G418 resistance gene and the S. cerevisiae MEL5 terminator, the C.<br>
sonorensis PGK1 promoter operatively linked to the B. megaterium LDH gene,<br>
the -S. cerevisiae GAL10 terminator followed by the C. sonorensis PDC2<br>
terminator. This construct is shown schematically in FIG. 15.<br>
The pMI286 plasmid was digested with NotI to excise the 6400 bp<br>
fragment that consisted of the G418 resistance and LDH expression cassettes<br>
flanked by the PDC2 5" and 3" regions. This 6400 bp fragment was used to<br>
transform C. sonorensis by the method described in Example 1 above. The<br>
transformants were grown on YPD agar plates supplemented with G418 antibiotic<br>
at a concentration of 200 µg /mL. The plates were incubated at 30aC for 2-5 days<br>
and the transformants were then restreaked onto fresh selection plates. The<br>
transformants were designated as 286-1 through 286-40.<br>
Southern blot analysis of genomic DNA isolated from the pMI286<br>
transformahts was carried out with the C. sonorensis PDC2 probe (corresponding<br>
to nucleotides in the deleted area) to identify transformants in which B.<br>
megaterium LDH was integrated into the PDC2 locus. The absence of a PDC2<br>
hybridizing band in transformants 286-1, 286-2, 286-4, 286-19, and 286-30<br>
indicated that PDC2 gene was deleted. Other pMI286 transformants and<br>
untransformed C. sonorensis gave a positive signal in the PDC2 hybridization.<br>
Hybridization with the B. megaterium LDH probe showed the LDH was present in<br>
one copy in the pdc2 deletants. The frequency of targeted integration into the<br>
PDC2 locus was 15%.<br>
These results showed that targeted integration of the transformed pMI286<br>
DNA into the PDC2 locus occurred through homologous recombination between<br>
PDC2 promoter and PDC2 terminator sequences. These results also show that the<br>
PDC2 is a single copy gene in C. sonorensis. In addition, integration events<br>
outside the PDC2 occurred. In some transformants the LDH gene was integrated<br>
in more than one copy into the genome.<br>
Example 10: Construction of C. sonorensis strains deleted of PDC1 and<br>
disrupted in pdc2 and harboring two copies of B. megaterium LOH integrated<br>
in the genome in the PDC1 and pdc2 loci<br>
The C. sonorensis transformant 265-15 having B. megaterium LDH<br>
integrated in the PDC1 locus was chosen as host for a second transformation with<br>
B. megaterium LDH. Transformant 265-15 was further transformed with NotI<br>
digested pMI286 using the methods described in Example 1 above, and the<br>
transformants were selected on YPD agar plates supplemented with G418<br>
antibiotic at a concentration of 200 µg/mL. The plates were incubated at 30°C for<br>
2-5 days for selection, and transformants obtained thereby were purified by<br>
restreaking them onto fresh selection plates. The transformants were designated<br>
as C44/286-1 through C44/286-40.<br>
Disruption of the pdc2 gene was verified using the PDC2 probe<br>
(corresponding to nucleotides in the deleted area). T he absence of PDC2<br>
hybridizing band in transformants C44/286-10, C44/286-26, C44/286-27,<br>
C44/286-28, C44/286-29, C44/286-30, C44/286-31, C44/286-32, and C44/286-33<br>
indicated that the PDC2 gene was deleted. The presence of B. megaterium LDH<br>
in the genome in two copies in the pdc1, pdc2 double deletants was verified by<br>
Southern analysis of HindIII digested yeast DNA using the B. megaterium LDH<br>
gene as the probe.<br>
These results showed that targeted integration of the transformed pMI286<br>
DNA into the PDC2 locus occurred through homologous recombination between<br>
PDC2 promoter and PDC2 terminator sequences. These resuJts also confirm that<br>
the PDC2 is a single copy gene in C. sonorensis, and that integration events<br>
outside the PDC2 locus can occur. In some transformants the LDH gene was<br>
integrated in more than one copy into the genome. T he transformants<br>
simultaneously deleted of pdc1 and disrupted in pdc2 are viable.<br>
This Example also confirmed that C. sonorensis can be transformed<br>
multiply and independently when different markers are used. In this way it is also<br>
possible to increase copy number of the gene of interest (LDH) in the host<br>
genome.<br>
Example 11: Ethano) production by the pdc1-pdc2- strains<br>
Ethanol production in Candida strains bearing deletions or disruptions in<br>
the PDC1 and/or PDC2 genes was assayed as follows. Transformants designated<br>
C44/286-10, C44/286-26, and C44/286-33 and four other strains included as<br>
controls were grown in 50 mL of YP + 5% glucose in 250 mL shaker flasks at 250<br>
rpm shaking and at a temperature of 30°C Samples were withdrawn daily and<br>
cells were removed by centrifugation. Culture supernatant samples taken 56 h<br>
after inoculation were analyzed for ethanol by the ethanol UV method of<br>
Boehringer Mannheim (Table 1). These results showed that ethanol production by<br>
the transformants deleted of both pdc1 and pdc2 ethanol is reduced more than ten-<br>
fold compared to the strains containing an intact PDC1 or PDC2 gene.<br>
These results demonstrated that both PDC1 and PDC2 encode functional<br>
pyruvate decarboxylases, since a drastic reduction in ethanol production is only<br>
observed when both of the genes are simultaneously deleted. The results also<br>
indicated that PDC2 disruption removing approximately 60% of the PDC2 open<br>
reading frame abolished PDC2 function.<br>
Example 12: Vector for disruption of PDC2 without LDH<br>
Vectors were prepared for replacing PDC2 without introducing exogenous<br>
LDH-encoding sequences. The B. megaterium LDH gene was removed from the<br>
pMI286 plasmid described in Example 9 as a 1276 bp SpeI - XbaI fragment.<br>
pMI286 was digested with SpeI and the linearized molecule partially digested with<br>
XbaI. The 8100 bp SpeI-XbaI fragment was isolated after gel electrophoresis and<br>
recircularized. The resulting plasmid termed pMI287 consists, in order, of the C.<br>
sonorensis PDC2 promoter, the C. sonorensis TDH1 promoter operatively linked<br>
to the G418 resistance gene and the S. cerevisiae MEL5 terminator, the C.<br>
sonorensis PGK1 promoter followed by the C. sonorensis PDC2 terminator, and<br>
is shown schematically in FIG. 16..<br>
pMI287 was digested with NotI to excise the 5100 bp fragment that<br>
consisted of the G418 expression cassette flanked by the PDC2 5" and 3* regions.<br>
This 5100 bp fragment -was used to transform C. sonorensis by the methods<br>
described in Example 1 above. Transformants were grown on YPD agar plates<br>
supplemented with G418 antibiotic at a concentration of 200 mg/mL. The plates<br>
were incubated at 30°C for 2-5 days and the transformants were then restreaked<br>
onto fresh selection plates.<br>
Transformants were designated as 287-1 through 287-57. Southern blot<br>
analysis of genomic DNA isolated from the pMI287 transformants was performed<br>
using a PDC2 probe that corresponded to nucleotides in the deleted region, in<br>
order to identify successful transformants. No PDC2 hybridizing band was<br>
observed in the transformants 287-6 and 287-16, indicating that the PDC2 gene<br>
was deleted.<br>
Example 13: Vectors for expression of L helveticus LDH and for targeted<br>
integration of the transformed DNA into the PDC2 locus<br>
In order to replace sequences encoding B. megaterium LDH in pMI286 by<br>
L. helveticus LDH-encoding DNA, the pMI286 described in Example 9 was<br>
digested with the restriction enzyme SpeI and made blunt ended with DNA<br>
polymerase I, Klenow fragment, and each of the four dNTPs and then digested<br>
with BspMI. Plasmid pMI247 shown in FIG. 9 was digested with BamHI and<br>
made blunt ended with DNA polymerase I, Klenow fragment, and each of the four<br>
dNTPs and then digested with BspMI. The 6800 bp SpeI(blunt)- BspMI fragment<br>
of pMI286 and the 2700 bp BamHI(blunt)- BspMI fragment of pMI247 were<br>
ligated. The resulting plasmid termed pMI288 consists, in order, of the C.<br>
sonorensis PDC2 promoter, the C. sonorensis TDH1 promoter operatively linked<br>
to the G418 resistance gene and the S. cerevisiae MEL5 terminator, the C.<br>
sonorensis PGK1 promoter operatively linked to the L. helveticus LDH gene and<br>
the S. cerevisiae CYC1 terminator followed by the C. sonorensis PDC2<br>
terminator, and is shown schematically in FIG. 17.<br>
pMI288 was digested with NotI to excise the 6400 bp fragment that<br>
consisted of the G418 resistance and LDH expression cassettes flanked by the<br>
PDC2 5" and 3" regions. The C. sonorensis transformant designated 257-3 having<br>
the L. helveticus LDH integrated in the PDC1 locus was chosen as host for a second<br>
transformation with L. helveticus LDH. Transformant 257-3 was further<br>
transformed with the 6400 bp NotI fragment of pMI288 by the methods described<br>
in .Example 1 above. Transformants were selected on YPD agar plates<br>
supplemented with G418 antibiotic at a concentration of 200 µg/mL. The plates<br>
were incubated at 30°C for 2-5 days, and transformants obtained thereby were<br>
purified by restreaking them onto fresh selection plates. These transformants were<br>
designated as C40/288-1 through C40/288-40.<br>
Disruption of the pdc2 gene was verified using a PDC2 probe<br>
corresponding to nucleotides in the deleted area of the locus. The absence of a<br>
PDC2 hybridizing band in transformants C40/288-2, C40/288-11, C40/288-29,<br>
C40/288-34, and C40/288-38, indicated that the PDC2 gene was deleted. The<br>
presence of L. helveticus LDH in the genome in two copies in the PDC1, pdc2<br>
double deletants was verified by Southern blot analysis of HindIII digested yeast<br>
DNA using the L helveticus LDH gene as the probe.<br>
These results demonstrated that targeted integration of exogenous LDH<br>
sequences into C, sonorensis PDC2 locus was achieved, and provided cells with<br>
disrupted PDC2 loci.<br>
Example 14: Production of L-iactic acid in defined or rich glucose medium<br>
in aerobic test tube cultures by C. sonorensis harboring L hetveticus or B.<br>
megaterium LDH gene integrated into the genome.<br>
C. sonorensis cells and the transformarnts disclosed in the Examples above<br>
(namely, 246-27, 247-11, 265-03, 265-05, 265-06, 265-07, 265-11, 265-12, 265-<br>
14, 265-15, 265-17, 265-18, 265-22, 26S-23, 265-29, 265-33, 265-34, 265-35,<br>
265-38, 265-39, 265-42, 265-43, 265-44, 265-45, 265-4$, 265-47, 265-48, 265^49,<br>
265-51, 265-52, 265-55, 265-56, 265-57, and 265-60) were cultivated in YPD<br>
medium (YP supplemented with 5% glucose and 0.5 M MES pH 5.5) or YD<br>
medium (yeast nitrogen base without amino acids supplemented with 2% glucose<br>
and 0.5 M MES pH 5.5). Two independent colonies from each transformant were<br>
inoculated into a 14 mL disposable plastic cube containing 5 mL of YPD or YD<br>
medium and cultivated with 250 rpm shaking at 30°C. Samples were withdrawn<br>
during cultivation, OD600 measured, and cells removed by centrifugation and the<br>
culture supernatant analyzed by HPLC for lactic acid, glucose and ethanol. HPLC<br>
analyses were carried out with Waters 510 HPLC pump, Waters 717+<br>
autosampler, and Water System Interfase Module liquid chromatography complex<br>
with refractive index detector (Waters 410 Differential refractometer) and UV-<br>
detector (Waters 2487 dual ? UV detector). An Aminex HPX-87H Ion Exclusion<br>
Column (300 mm x 7.8 mm, Bio-Rad) was used and was equilibrated with 5 mM<br>
H2SO4 in water at 35°C, and samples were eluted with 5 mM H2SO4 in water at a<br>
flow rate of 0.6 mL/min. Data acquisition and control were performed using<br>
Waters Millennium software. Values are averaged from two independent samples.<br>
These results are shown in Table 2 and 3.<br>
After 13 hours of cultivation in defined medium, transformants 246-27 and<br>
247-11 harboring the L. helveticus LDH gene produced 0.1 - 0.4 g/L lactic acid;<br>
1.8 - 3.9 g/L lactic acid was produced after 19 hours.<br>
After 13 hours of cultivation in defined medium, transformants 265-03,<br>
265-06, 265-11, 265-12, 265-18, 265-29, 265-44, 265-45, 265-46, 265-52, 265-55<br>
and 265-57 harboring the B. megaterium LDH gene integrated in an unknown site<br>
in the genome in one copy produced 0.5 - 1.9 g/L lactic acid; 4.0 - 6.3 g/L lactic<br>
acid were produced after 19 hours.<br>
After 13 hours of cultivation in defined medium, transformants 265-14,<br>
265-22 and 265-23 harboring two copies of the B. megaterium LDH gene<br>
integrated in an unknown site in the genome produced 0.5 -1.2 g/L lactic acid; 3.8<br>
- 6.1 g/L lactic acid were produced after 19 hours .<br>
After 13 hours of cultivation in defined medium, transformant 265-56<br>
harboring three copies of the B. megaterium LDH gene produced 0.7 g/L lactic<br>
acid; 5.2 g/L lactic acid were produced after 19 hours.<br>
After 13 hours of cultivation in defined medium, transformants 265-05,<br>
265-07, 265-15, 265-17, 265-33,265-34, 265-35, 265-38, 265-39, 265-42, 265-43,<br>
265-47, 265-48, 265-49, 265-51 and 265-60 harboring the B. megaterium LDH<br>
gene integrated into the pdc1 gene locus (pdc1- genotype) produced 0.4 - 2.7 g/L<br>
lactic acid; 3.4 - 7.5 g/L lactic acid were produced after 19 hours.<br>
After 12 hours cultivation in rich medium, transformants 246-27 and 247-<br>
11 harboring the L. helveticus LDH gene produced 0.5 - 1.7 g/L lactic acid, and<br>
produced 3.7 - 6.1 g/L lactic acid after 17 hours. In comparison, the host strain<br>
produced 0.1 g/L lactic acid after 17 hours of cultivation.<br>
After 12 hours cultivation in rich medium, the transformants 265-03, 265-<br>
06, 265-11, 265-12, 265-18, 265-29, 265-44, 265-45, 265-46, 265-52, 265-55 and<br>
265^57 harboring the B. megaterium LDH gene produced 1.4 - 4.3 g/L lactic acid,<br>
and produced 7.2 - 9.8 g/L lactic acid after 17 hours.<br>
After 12 hours of cultivation in rich medium, transformants 265-14, 265-<br>
22 and 265-23 harboring two copies of the B. megaterium LDH gene produced 2.1<br>
-1.9 g/L lactic acid, and produced 6.3 - 6.8 g/L lactic acid after 17 hours.<br>
After 12 hours of cultivation in rich medium, transformant 265-56<br>
harboring three copies of the B. megaterium LDH gene produced 2.6 g/L lactic<br>
acid, and produced 7.5 g/L lactic acid after 17 hours.<br>
After 12 hours of cultivation in rich medium, the transformants 265-05,<br>
265-07, 265-15, 265-17, 265-33, 265-34, 265-35, 265-38, 265-39, 265-42, 265-43,<br>
265-47, 265-48, 265-49, 265-51 and 265-60 harboring the B. megaterium LDH<br>
gene integrated into the pdc1 gene locus (PDC1- genotype) produced 2.0 - 4.7 g/L<br>
lactic acid, and produced 7.1 -10.7 g/L lactic acid after 17 hours.<br>
These results show that the LDH transformants produced lactic acid when<br>
the host strain did not, B. megaterium and L. helveticus LDHs were shown to be<br>
active in C. sonorensis. These heterologous LDHs can thus effectively compete<br>
for pyruvate in tht presence of PDC. The PDC1 deletion did not seem to have an<br>
effect on the overall yield and production of lactate. Residual glucose was higher<br>
and ethanol concentration was lower in transformants containing two (265-14,<br>
265-22, 265-23) or three (265-56) copies A higher LDH copy number also<br>
resulted in a higher lactic acid yield from glucose, less ethanol production, and a<br>
higher ratio of lactic acid to ethanol. The biomass (OD600) increased less in strains<br>
containing more than one copy of B. megaterium LDH.<br>
Example 15: Production of L-lactic acid in defined or rich glucose medium<br>
in aerobic test tube cultures by C. sonorensis harboring L. helveticus, B.<br>
megaterium or R. oryzae LDH gene integrated into the genome.<br>
C. sonorensis cells and the transformants disclosed above (namely, 246-27,<br>
247-11, 265-39, 265-5, 265-15, 265-44, 266-1, 266-2, 266-4, 266-6, 266-7, 266-8,<br>
266-11, 278-2, 278-3, 278-4, 278-6, 278-7, 278-8, 278-9, 278-11, 278-12, 278-13,<br>
278-14, 278-15, 278-17, 278-18, 278-19, 278-20, 257-3, 257-5, 257-6, 257-8,<br>
257-8, 257-9, 257-10, 257-11, and 257-12) were cultivated in YPD (YP<br>
supplemented with 5% glucose and 0.5 M MES pH 5.5) or YD -medium (yeast<br>
nitrogen base without amino acids supplemented with 2% glucose and 0.5 M MES<br>
pH 5.5). A colony from each transformant was inoculated into a 14 mL<br>
disposable plastic tube containing 5 mL of YPD or YD medium and cultivated<br>
with 250 rpm shaking at 30°C. Samples were withdrawn during cultivation at<br>
time points 12 and 17 hours, OD600 measured, and cells harvested by<br>
centrifugation and the culture supernatant analyzed by HPLC as described above<br>
for lactic acid, glucose and ethanol. HPLC analyses were carried out as detailed<br>
above in Example 14. These results are shown in Tables 4 and 5.<br>
After 12 hours of cultivation in defined medium, transformants harboring<br>
the L. helveticus LDH gene produced 0.1 - 0.7 g/L lactic acid. In rich medium 0.9<br>
- 2.7 g/L lactic acid was produced by these cells.<br>
After 12 hours of cultivation in defined medium, transformants harboring<br>
the B. megaterium LDH gene produced 0.1 - 0.5 g/L lactic acid. In rich medium<br>
1.9 - 3.2 g/L lactic acid was produced by these cells.<br>
After 12 hours of cultivation in defined medium, transformants harboring<br>
the R. oryzae LDH gene produced 0.2 - 0.6 g/L lactic acid. In rich medium 0.9 -<br>
2.7 g/L lactic acid was produced by these cells.<br>
After 12 hours of cultivation in defined medium, transformants harboring<br>
both the R. oryzae LDH gene integrated into PDC1 gene locus and the B.<br>
megaterium LDH gene produced 0.1 - 0.9 g/L lactic acid. In rich medium 1.0 - 3.3<br>
g/L lactic acid was produced by these cells.<br>
After 17 hours of cultivation in defined medium, transformants harboring<br>
the L. helveticus LDH gene produced 0.9 - 2.1 g/L lactic acid. In rich medium 6.6<br>
- 9.9 g/L lactic acid was produced by these cells<br>
After 17 hours of cultivation in defined medium, transformants harboring<br>
the B. megaterium LDH gene produced 0.8 - 1.7 g/L lactic acid. In rich medium<br>
8.7 -11.0 g/L lactic acid was produced by these cells.<br>
After 17 hours of cultivation in defined medium, transformants harboring<br>
the R. oryzae LDH gene produced 0.7 - 1.3 g/L lactic acid. In rich medium 7.3 -<br>
9.5 g/L lactic acid was produced by these cells.<br>
After 17 hours of cultivation in defined medium, transformants harboring<br>
both the R. oryzae LDH gene integrated into PDC1 gene locus and the B.<br>
megaterium LDH gene produced 0.7 - 3.0 g/L lactic acid. In rich medium 5.0 -<br>
10.7 g/L lactic acid was produced by these cells.<br>
These results showed that all three heterologous LDHs were active in C.<br>
sonorensis and could be used for producing lactic acid. These LDHs can<br>
effectively compete for pyruvate in the presence of PDC. Expression of any of<br>
these LDH genes reduced glucose utilization, growth and ethanol production,<br>
especially in rich medium. The reduction in glucose utilization rate and growth<br>
were strongest in strains containing L. helveticus LDH and mildest in strains<br>
containing R. oryzae LDH, while B. megaterium LDH transformants showed<br>
intermediate behavior. The effects were masked by the presence of the B.<br>
megaterium LDH in the transformants containing LDHs of two origins.<br>
Example 16: Production of L-lactic acid in defined glucose medium with or<br>
without buffering in microaerobic shake flask cultures by C. sonorensis<br>
harboring B. megaterium or R. oryzae LDH gene integrated into the genome.<br>
The C. sonorensis transformants harboring the B. megaterium LDH gene<br>
(namely 265-23 and 265-55) or the R. oryzae LDH gene (266-8) were cultivated in<br>
defined glucose medium. Precultures were grown in YD medium (yeast nitrogen<br>
base without amino acids supplemented with 5% glucose and 0,5 M MES pH 5.5),<br>
cells collected by centrifugation and resuspended in 50 mL of YD medium (yeast<br>
nitrogen base without amino acids supplemented with 10% glucose) to an OD600<br>
of 15 for the cultivation experiments. Yeasts were cultivated in 250 mL<br>
Erlenmeyer flasks with or without 4 g CaCO3 with 100 rpm shaking at 30°C.<br>
Samples were withdrawn during cultivation, OD600 measured from the cultures<br>
without CaCO3, and cells harvested by centrifugation and the culture supernatant<br>
analyzed for L-lactic acid (by the L-lactic acid UV method of Boehringer<br>
Mannheim, Roche) and glucose (by the glucose/ GOD-Perid method of<br>
Boehringer Mannheim, Roche). These results are shown in Table 6.<br>
After 24 hours of cultivation, transformant 265-55 harboring B.<br>
megaterium LDH gene produced 35.7 g/L lactic acid with CaCO3 buffering and<br>
6.16 g/L lactic acid without buffering when the pH dropped to 2.75. Transformant<br>
265-23 harboring two copies of B. megaterium LDH gene produced 38.2 g/L lactic<br>
acid with CaCO3 buffering and 6.81 g/L lactic acid without buffering when the pH<br>
dropped to 2.68 (24 hours of cultivation). Transformant 266-8 harboring R.<br>
oryzae LDH gene produced 35.4 g/L lactic acid with CaCO3 buffering and 3.05<br>
g/L lactic acid without buffering when the pH dropped to 2.83 (24 hours of<br>
cultivation).<br>
These results demonstrated that in the presence of CaCO3 at pH 6.5, lactic<br>
acid production and glucose utilization were higher than in unbuffered conditions<br>
below pH 3. Higher lactic acid titers were reached in the presence of CaCO3.<br>
Example 17: Intracellular lactic acid in CaCO3-buffered and unbuffered<br>
cultivation<br>
Cell pellets fronn C. sonorensis transformants harboring the B. megateriwn<br>
LDH gene (namely 265-23 and 265-55) or the R. oryzae LDH gene (266-8)<br>
cultivated in defined glucose medium, as described above in Example 16, were<br>
analyzed to determine intracellular lactic acid concentration. Samples (2 mL)<br>
were withdrawn during cultivation at 8h and 24h, OD600 measured and cells<br>
harvested by centrifugation. The supernatant was discarded and each of the pellets<br>
was washed with 1 mL of ice-cold 10 mM K2HPO4/KH2PO4, pH 7.5,<br>
supplemented with 2 mM EDTa. Washed cell pellets were resvispended in 0.5<br>
mL of the same buffer and stored at -70°C. Samples were thawed and washed (1<br>
mL) once in I M Tris-HCI, pH 9.0, and centrifuged at 13,000 rpm for 1 min. The<br>
pellet was suspended into 1 mL ice cold 5% trichloroacetic acid (TCA) and<br>
vortexed 1 min. After vortexing, the sample was kept on ice fpr about 50 min.<br>
After incubation on ice, the sample was vortexed for 1 min and centrifuged at<br>
13,000 rpm for 30 mm at 4°C. Lactic acid levels were measured in the collected<br>
supernatant. Lactic acid concentration was analyzed from the sample by using an<br>
enzymatic method (L-lactic acid UV method., Boehringer Mannheim, Roche) or by<br>
HPLC (as in Example 14). Intracellular concentration of lactic acid was<br>
calculated as follows;<br>
1. The intracellular volume of the cells (in the sample):<br>
Dry weight of the culture (g/L) * volume of the sample (L) * 2 mL/g cell = cell<br>
volume (mL).<br>
Cell volume is converted into liters by multiplying by 0.001. One gram of cell (dry<br>
weight) corresponds to 2 mL cell volume (Gancedo &amp; Serrano, 1989, "Energy<br>
Yielding Metabolism," in The yeasts. (Rose &amp; Harrison, eds.), Vol 3. Academic<br>
Press: London).<br>
2. The lactic acid amounts in the cells:<br>
Measured lactic acid concentration (g/L) • volume of used 5% TCA (L) = lactic<br>
acid amount (g) in the sample. To calculate lactic acid concentration in the cell:<br>
divide lactic acid amount in the sample (g) by cell volume (L).<br>
After 24 hours of cultivation transformant 265-55 harboring the B.<br>
megaterium LDH gene had an intracellular concentration of 28.2 g/L lactic acid<br>
with CaCO3 buffering and 7.2 g/L of lactic acid without buffering. Transformant<br>
265-23 harboring two copies of the B. megaterium LDH gene had an intracellular<br>
concentration of 46.1 g/L lactic acid with CaCO3 buffering and 8.2 g/L of lactic<br>
acid without buffering, after 24 hours of cultivation. Transformant 266-8<br>
harboring R. oryzae LDH gene had an intracellular concentration of 45.4 g/L of<br>
lactic acid with CaCO3 buffering and 4.9 g/L of lactic acid without buffering (24<br>
hours cultivation). These results are shown in Table 7.<br>
These results showed that after 8h of cultivation intracellular lactic acid<br>
levels were twice as high as extracellular levels in transformants 265-55 and 265-<br>
23 when grown in unbuffered culture. At 8h of cultivation for the other<br>
transformants, the difference between intra- and extracellular levels was small,<br>
about 10%. When CaCO3 was included in the cultures, the intracellular and<br>
extracellular lactic acid levels in all strains were higher than cultures without<br>
CaCO3. The intra- and extracellular lactic acid concentrations in all strains<br>
increased from 8 to 24h in the CaCO3-buffered culture. The intracellular lactic<br>
acid concentrations in the unbuffered cultures are similar at 8h and at 24h. The<br>
intracellular lactic acid levels of strain 266-8 are lower than the levels of the other<br>
strains.<br>
Table 7. Intracellular lactic acid concentration (g/L). "C" indicates the presence of<br>
CaCO3 in the cultivation.<br>
Example 18: Enzyme activities of lactate dehydrogenase and pyruvate<br>
decarboxylase in C. sonorensis harboring L. helveticus or B. megaterium LDH<br>
gene integrated into the genome.<br>
The C. sonorensis transformants (namely, 246-27, 247-11, 257-3, 257-12,<br>
257-6, 247-9, 246-27, 247-11, 265-39, 265-15, 265-44, 265-55, 265-23, 265-22,<br>
265-56, 278-14, 278-17, 286-4, 286-30, and 286-1) were cultivated in 50 mL of<br>
YD -medium (yeast nitrogen base without amino acids supplemented with 5%<br>
glucose and 0.5 M MES pH 5.5), in 250 mL Erlenmeyer flasks with 250 rpm<br>
shaking to an OD600 of 10 at 30°C. Cells were harvested by centrifugation and the<br>
culture supernatant was analyzed by HPLC. Cell samples to be used for enzyme<br>
activity measurements (2 mL) were collected by centrifugation and washed with 1<br>
mL of ice-cold 10 mM K2HPO4/KH2PO4, pH 7.5 supplemented with 2 mM<br>
EDTA. Washed cell pellets were resuspended in 0.5 mL of the same buffer and<br>
stored at -70°C. Samples were thawed at room temperature and washed (1 mL)<br>
once in sonication buffer (10 0 mM KH2PO4/K2HPO4, pH 7.5 supplemented with<br>
2 mM MgCl2 and 10 mM DTT). Washed samples were resuspended in 0.5 mL of<br>
sonication buffer and homogenized with 0.5 mL of glass beads with a Bead Beater<br>
homogenizer for 1 minute. After homogenization samples were centrifuged at<br>
14,000 rpm for 30 min at 4°C. Supernatant samples were collected and lactate<br>
dehydrogenase activity was determined spectrophotometrically (A340) with Cobas<br>
MIRA automated analyzer at 30°C in sodium acetate buffer (50 mM Na-acctate<br>
pH 5.2) (Lactobacillus helveticus LDH) or in imidazole buffer (40 mM imidazole-<br>
HCl, pH 6.5) (Bacillus megaterium LDH) containing 0.4 mM NADH, 5 mM<br>
fructose- 1,6-diphosphate, 1 mM glyoxylic acid and 2 mM pyruvate. The protein<br>
concentrations were determined by the Lowry method (Lowry et a!., 1951, J. Biol.<br>
Chem.\93: 265-275). Bovine serum albumin (Sigma) was used as a protein<br>
standard. Pyruvate decarboxylase activity was determined spectrophotometrically<br>
(A340) with Cobas MIRA automated analyzer at 30°C in imidazole buffer (40 mM<br>
imidazole-HCl pH 6.5) containing 0.2 mM NADH, 50 mM MgCl2, 0.2 mM<br>
thiamin pyrophosphate (cocarboxylase), 90 units of ADH and 50 mM pyruvate. 1<br>
U of enzyme activity was defined as the amount of activity converting 1 µmol of<br>
NADH to NAD+ per min. These results are shown in Table 8.<br>
This Example demonstrated that intracellular LDH activity correlated with<br>
the copy number of the LDH genes in the genome. The calculated LDH activity in<br>
strains harboring one copy of the L. helveticus LDH was 8 U/mg total cellular<br>
protein, and the activity in strains harboring two copies was 15 or 35 U/mg total<br>
cellular protein. Lactic acid titers and yields from glucose were greater in the<br>
strains containing multiple copies of the LDH gene, however the ethanol titers<br>
were lower than in strains containing only one copy of the LDH gene. Calculated<br>
LDH activity in strains harboring one copy of the B. megaterium LDH was 2-3<br>
U/mg total cellular protein, the activity in strains harboring 2 copies was 10 U/mg,<br>
and the activity in strains harboring 3 copies was 40 U/mg.<br>
Pyruvate decarboxylase activity was typically 2-4 U/mg total cellular<br>
protein in strains containing an intact PDC2 gene. When pdc2 was disrupted, PDC<br>
activity dropped below 0.4 U/ mg total cellular protein. If both PDC1 and pdc2<br>
were deleted or disrupted (strain C44/286-10) PDC activity decreased to 0.07<br>
U/mg total cellular protein.<br>
Table 8. LDH and PDC enzyme activities; glucose, lactic acid, and ethanol<br>
concentrations; lactic acid yield in the culture supernatant measured from cultures<br>
grown on YD -medium, n.d.: not determined, 1x, 2x, and 3x indicate the LDH<br>
gene copy number.<br>
Example 19," Production of L-lactic acid in defined glucose medium by C.<br>
sonorensis harboring the L helveticus, B. megaterium or R. oryzae LDH<br>
encoding gene or both B. megaterium and R. oryiae LDH genes integrated<br>
into the genome.<br>
C sonorensis cells and the transfonnants (namely 266-7, 266-8, 246-27,<br>
247-11, 257-3, 257-12, 257-6, 247-9, 265-39, 265-15, 265-44, 265-55, 265-23,<br>
265-22, 265-56, 266-3, 278-14, 278-17, 286-4, 286-30, 286-1) were cultivated in<br>
YD medium (yeast nitrogen base without amino acids, pH 5.5, supplemented with<br>
5% glucose and 0.5 M MES), and collected by centrifugation. The cells were<br>
resuspended in 50 mL of YD (yeast nitrogen base without amino acids<br>
supplemented with 10 % glucose) to an OD600 of 15 for the cultivation<br>
experiments. The cells were cultivated in 250 mL Erlenmeyer flasks containing 4<br>
g CaCO3 with 100 rpm shaking at 30°C. Samples were withdrawn during<br>
cultivation, the cells were harvested by centrifugation, and the growth medium<br>
was analyzed for lactic acid, glucose, and ethariol, by HPLC as described above<br>
(Example 14). These results are shown in Tables 9-13.<br>
The maximal lactic acid titers in the culture supernatants were typically<br>
reached at 72 h or later in the cultivation after all glucose had been consumed. The<br>
maximal lactic acid titers and yields reached as classified on the basis of the<br>
different genetic backgrounds were as follows:<br>
- 1 copy of R. oryzae LDH (strain 266-7): 81 g/L and 79% yield at 96 h<br>
- 1 copy of B. megaterium LDH (strain 265-55): 85 g/L and 82 % yield at 96 h<br>
- 1 copy of L helveticus LDH (strain 257-3): 85 g/L and 84% yield at 96 h<br>
- 2 copies of B. megaterium LDH (strain 265-22): 87 g/L and 84 % yield at 72 h<br>
- 3 copies of B. megaterium LDH (strain 265-56): 83 g/L and 80 % yield at 72 h<br>
- 2 copies of L. helveticus LDH (strain 247-9): 90 g/L and 89% yield at 72 h<br>
- 1 copy of R. oryzae LDH and 1 copy of B. megaterium LDH (strain 278-17):<br>
79 g/L and 76% yield at 72 h<br>
- 1 copy of/?, oryzae LDH and 2 copies of B. megaterium LDH (strain 278-14):<br>
89 g/L and 86 % yield at 96 h<br>
After all glucose was consumed a calcium lactate precipitate was formed in<br>
the following cultures: strains 246-27, 247-11, 265-39, 265-15, 265-44, 265-23,<br>
265-22, 278-14, 278-17, 286-4, 286-30, and 286-1. The precipitate formation also<br>
indicated that very high lactic acid titers were obtained.<br>
These results demonstrated that C. sonorensis overexpressing L.<br>
helveticus, R. oryzae or B. megaterium LDH reached high final lactic acid titers<br>
(&gt;80 g/L) and yields (&gt;8O%) from glucose in CaCO3 buffered defined medium at<br>
pH 6.5. L. helveticus and B. megaterium LDH transformants performed<br>
essentially equally well, and better than R. oryzae LDH transformants that gave<br>
slightly lower lactic acid titers and yields. LDH copy number especially affected<br>
byproduct formation: a higher LDH copy number and LDH activity resulted in less<br>
ethanol and acetate production. Both L. helveticus and B. megaterium LDH<br>
transformants produced less ethanol and acetate than R. oryzae LDH<br>
transformants. Other measured byproducts, including glycerol and pyruvate were<br>
present in negligible amounts, and did not significantly differ between the PDC+,<br>
pdc1- or pdc2- genotypes.<br>
Example 20: Production of L-lactic acid in defined glucose medium in<br>
nitrogen sparged tubes by C. sonorensis harboring L. helveticus or R. oryzae<br>
LDH encoding gene integrated into the genome.<br>
Production of L-lactic acid in transformed C. sonorensis cells was<br>
demonstrated as follows. C. sonorensis cells and the transformants harboring the<br>
I. helveticus LDH gene (namely, 246-14, 246-14, 246-18, 246-23, 246-27, 247-7,<br>
247-8, 247-11, and 257-3) or the R. oryzae LDH gene (266-3 and 266-4) were<br>
cultivated in YD medium (yeast nitrogen base without amino acids supplemented<br>
with 12% glucose and 0.4 M MES pH 5.5). Precultures were grown in 50 mL of<br>
YD medium (yeast nitrogen base without amino acids supplemented with 6.5%<br>
glucose and 0.4 M MES, pH 5.5) in 250 mL Erlenmeyer flasks with 250 rpm<br>
shaking at 30°C. Cells were collected by centrifugation and washed once with<br>
0.9% NaCl, then resuspended in 50 mL of YD medium to an OD600 of 11 for the<br>
cultivation experiments. Yeasts were cultivated in 50 mL disposable plastic tubes<br>
sparged with nitrogen with 250 rpm shaking at 30°C ((nearly) anaerobic<br>
conditions). Samples were withdrawn during cultivation, and after that the tubes<br>
were sparged with nitrogen. OD600 was measured, and cells harvested by<br>
centrifugation and the culture supernatant analyzed by HPLC as described above<br>
for lactic acid, glucose and ethanol. These results are shown in Tables 14-20.<br>
After 94 hours of cultivation the transformants harboring L. helveticus<br>
LDH gene produced 6.9 - 7.2 g/L lactic acid (equivalent to 66 - 84% yield) and 1-<br>
1.4 g/L ethanol, whereas the host strain produced 0.1 g/L lactic acid and 40 g/L<br>
ethanol. The transformants harboring R. oryzae LDH gene produced 7.2-8.8 g/L<br>
lactic acid (equivalent to 13-18% yield) and 17-28 g/L ethanol after 94 hours of<br>
cultivation. Glucose consumption and ethanol production by the R. oryzae LDH<br>
transformants were faster than those of the L. helveticus transformants.<br>
These results showed that C. sonorensis transformed with L. helveticus<br>
LDH or R. oryzae LDH produced lactic acid from glucose in nitrogen sparged tube<br>
cultures.<br>
Example 21; Production of L-lactic acid in rich glucose medium without<br>
buffering in microaerobic shake flask cultures by C. sonorensis harboring L.<br>
helveticus or B. megaterium LDH gene integrated into the genome.<br>
Production of L-lactic acid in transformed C. sonorensis cells was<br>
demonstrated as follows. The C. sonorensis transformants harboring the B.<br>
megaterium LDH gene (namely, 265-23 and 286-1) and L helveticus LDH gene<br>
(246-27 and 247-11) disclosed above were cultivated in 50 mL of YD medium<br>
(yeast nitrogen base without amino acids supplemented with 5% glucose and 0.5<br>
M MES, pH 5.5) in 250 mL Erlenmeyer flasks with 250 rpm shaking at 30ºC.<br>
Cells were collected by centrifugation and then resuspended in 50 mL of YP<br>
supplemented with 5% glucose to an OD600 of 15 for the cultivation experiments.<br>
Cells were cultivated in 250 mL Erlenmeyer flasks with 100 rpm shaking at 30°C.<br>
Samples were withdrawn during cultivation, OD600 measured, and cells were<br>
harvested by centrifugation. The culture supernatant analyzed for L-lactic acid (by<br>
the L-lactic acid UV method of Boehringer Mannheim, Roche), for glucose (by<br>
the glucose/ GOD-Perid method of Boehringer Mannheim, Roche), for acetate (by<br>
the acetic acid UV method of Boehringer Mannheim, Roche), and for ethanol (by<br>
the ethanol UV method of Boehringer Mannheim, Roche). These results are<br>
shown in Tables 15-20.<br>
Transformants 246-27 and 247-11 harboring L helveticus LDH gene<br>
integrated randomly into the yeast genome (PDC+ genotype) produced 7.8-9.0 g/L<br>
lactic acid (equivalent to 24-29% yield) after 24 hours of cultivation. The<br>
transformant 286-1 harboring B. megaterium LDH gene integrated into the pdc2<br>
gene locus (pdc2- genotype) produced 8.9 g/L lactic acid (equivalent to 31% yield)<br>
after 24 hours of cultivation. Transformant 265-23 harboring two copies of B.<br>
megaterium LDH gene integrated randomly into genome (PDC+ genotype)<br>
produced 9.1 g/L lactic acid (equivalent to 30% yield) after 24 hours of<br>
cultivation. After 24 hours of cultivation the transformants harboring B.<br>
megaterium LDH gene produced 8.9 - 9.1 g/L lactic acid, equivalent to 30-31%<br>
yield from glucose. Transformants harboring the L. helveticus LDH gene<br>
produced 7,8-9.0 g/L lactic acid, which is equivalent to 24-29% yield from<br>
glucose. Although some glucose was unconsumed at 24 h, all glucose was<br>
eventually consumed (at 120 h). No further increase in lactic acid concentration<br>
occurred after 24 h, however. Glucose consumption by all strains was very<br>
similar. The pH of the culture medium was between 3.4 - 3.8 during this<br>
experiment. The transformant 265-23 containing two copies of B. megaterium<br>
LDH produced less ethanol and acetate early in the cultivation whereas the pdc2-<br>
transformant 286-1 produced less ethanol and acetate towards the end of the<br>
cultivation than the other stTains.<br>
These results demonstrated that C. sonorensis transformed with L.<br>
helveticus LDH or B, megaterium LDH was capable of producing lactic acid from<br>
glucose under microaerobic conditions at low pH up to 9 g/L.<br>
Example 22: Production of L-lactic acid in a bioreactor in rich glucose<br>
medium by C sonorensis harboring B. megaterium LDH gene integrated into<br>
the genome.<br>
C sonorensis transformants designated 265-55, 286-30 and 265-15,<br>
described above, were cultivated in aerobic bioreactors. Batch cultivation was<br>
performed at 35°C in a laboratory bioreactor (Biostat CT-DCU3, Braun, Germany)<br>
with a working volume of 2 L. During the production phase the pH was<br>
maintained at 5.0 ±0.1 or increased to 6.0 ± 0.1 after 48 hours of cultivation by<br>
automated addition of 5 M potassium hydroxide (KOH). Biomass was produced<br>
with YP medium supplemented with 150 g/L glucose. The biomass production<br>
phase was inoculated with 20 mL of culture stored in 23% (w/v) glycerol at -80°C<br>
to an initial OD600 of 0.7-]. The bioreactor was flushed with 100% air at a flow<br>
rate of 1.0 L/min and stirred at 800 rpm during this phase. After 23.5 hours of<br>
cultivation 10-21 g/L cell mass was produced (dry weight) (equivalent to 0.2-0.3 g<br>
dry weight per used gram of glucose). After the 24 hour biomass production, the<br>
bioreactor was emptied and cells were collected by centrifugation (4000 rpm,<br>
20ºC, 10 min). The medium for lactate production (YP supplemented with 100<br>
g/L glucose) was pumped into the bioreactor and was inoculated with the cells<br>
collected from the biomass production phase, to a density corresponding to 5 g/L<br>
dry weight. The bioreactor was flushed with 10% air - 90% nitrogen gas at a flow<br>
rate of 1.0 L min-1 and stirred at 500 rpm.<br>
Samples were withdrawn during cultivation. For each sample, dry cell<br>
weight was determined, the OD600 was measured, and the cells were harvested by<br>
centrifugation. Culture supematants were analyzed by HPLC as described above<br>
for lactic acid, glucose, ethanol, and acetate. These results are shown in Tables 21<br>
and 22.<br>
The transformant harboring the B. megaterium LDH gene integrated<br>
randomly into the genome (265-55, PDC+ genotype) produced 28 g/L lactic acid<br>
(equivalent to 67% yield) at pH 5.0, after 52 hours of cultivation in the lactate<br>
production phase. The same transformant produced 28 g/L lactic acid (equivalent<br>
to 60% yield) at pH 6.0 after 72 hours of cultivation in the production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the pdc1 gene locus (265-15, pdc1- genotype) produced 23 g/L lactic acid<br>
(equivalent to 66% yield) at pH 5.0 after 51 hours of cultivation in the lactate<br>
production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the pdc2 gene locus (286-30, pdc2- genotype) produced 27 g/L lactic acid<br>
(equivalent to 54% yield) at pH 5.0 after 46 hours of cultivation in the lactate<br>
production phase.<br>
After 46 to 52 hours of lactic acid production phase, the transformants<br>
produced 23 - 28 g/L lactic acid (equivalent to a 54 - 67% yield).<br>
These results demonstrated that C. sonorensis overexpressing a<br>
heterologous lactate dehydrogenase encoding gene produced lactic acid from<br>
glucose in batch fermentation under microaerobic condition (e.g. 0%-2% O2 in the<br>
atmosphere).<br>
Intracellular lactic acid and pyruvate<br>
Intracellular lactic acid and pyruvate concentrations were determined as<br>
described above in Example 17, except that the sample volume was 1 mL and the<br>
cell pellet was washed (1 mL) in 1 M Tris-HCl, pH 9.0, centrifuged at 13,000 rpm<br>
for 1 min., and stored at -70°C. After thawing, the pellet was directly suspended<br>
into 1 mL of ice-cold 5% TCA. Intracellular pyruvate concentration was analyzed<br>
from the sample enzymatically (pyruvate kit, Sigma Diagnostics). These results<br>
are shown in Tables 24-27.<br>
The transformant harboring the B. megaterium LDH gene integrated<br>
randomly into the genome (265-55, PDC+ genotype) produced 60.9 g/L of lactic<br>
acid in the cells at 52 hours of cultivation in the lactate production phase, at pH<br>
5.0. The same transformant produced 38.7 g/L of lactic acid, at pH 6.0, at 72<br>
hours of cultivation in the lactate production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the PDC1 locus (265-15, PDC1- genotype) produced 13.4 g/L of lactic acid in the<br>
cells at 51 hours of cultivation in the lactate production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the pdc2 locus (286-30, pdc2- genotype) produced 14.3 g/L lactic acid at 49 hours<br>
of cultivation in the lactate production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated<br>
randomly into the genome (265-55, PDC+ genotype) produced 0.1 g/L of<br>
pyruvate in the cells during the cultivation at pH 5.0 and at pH 6.0.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the pdc1 locus (265-15, pdc1- genotype) or into the pdc2 locus (286-30, pdc2-<br>
genotype) produced 0.3 g/L pyruvate in the cells during the cultivations.<br>
These results showed that deletion of pdc1 and disruption of pdc2 caused<br>
an increase in intracellular pyruvate levels. In the PDC+ strain intracellular lactic<br>
acid levels increased towards the end of the cultivations, although this trend was<br>
not as clear in the pdc1- and in the pdc2- strains.<br>
Lactate dehydrogenase and pyruvate decarboxylase activities<br>
Lactate dehydrogenase and pyruvate decarboxylase activities were<br>
determined as follows. Samples for enzyme activity measurements (2 mL) were<br>
collected by centrifugation and the cell pellets were washed with 1 mL of ice-cold<br>
10 mM K2HPO4/ KH2PO4, pH 7.5 supplemented with 2 mM EDTA. Washed<br>
pellets were resuspended in 0.5 mL of the same buffer and stored at -70ºC.<br>
Samples were thawed at room temperature and washed (1 mL) once in<br>
homogenization buffer (100 mM KH2PO4/ K2HPO4, pH 7.5, supplemented with 2<br>
TnM MgCl2 and 10 mM DTT). Washed samples were resuspended in 0.5 mL of<br>
homogenization buffer and homogenized with 0.5 mL of glass beads with a Bead<br>
Beater homogenizer for 1 minute. After homogenization samples were<br>
centrifuged 14,000 rpm for 30 min at 4ºC. Supernatant samples were collected<br>
and lactate dehydrogenase and pyruvate decarboxylase activities were determined<br>
spectrophotometrically (A340) as described above in example 18, except that<br>
glyoxylic acid was not used. These results are shown in Tables 28-31.<br>
The transformant harboring B. megateriwn LDH gene integrated randomly<br>
into the genome (265-55, PDC+ genotype) produced lactate dehydrogenase<br>
activity of 1.4 U/mg total cellular protein and pyruvate decarboxylase activity of<br>
0.8 U/mg total cellular protein at 52 hours of cultivation in the lactate production<br>
phase at pH 5.0. The same transformant produced lactate dehydrogenase activity<br>
of 1.2 U/mg total cellular protein and pyruvate decarboxylase activity of 0.4 U/mg<br>
total cellular protein, at pH 6.0, at 72 hours of cultivation in the lactate production<br>
phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the PDC1 locus (265-15, PDC1- genotype) produced lactate dehydrogenase activity<br>
of 1.5 U/mg total cellular protein and pyruvate decarboxylase activity of 0.5 U/mg<br>
total cellular protein at 51 hours of cultivation in the lactate production phase.<br>
The transformant harboring the B. megaterium LDH gene integrated into<br>
the pdc2 locus (286-30, pdc2- genotype) produced lactate dehydrogenase activity<br>
of 0.7 U/mg total cellular protein and pyruvate decarboxylase activity of 0.1 U/mg<br>
total cellular protein, at 49 hours of cultivation in the lactate production phase.<br>
These results demonstrated that LDH activity is similar in all strains that<br>
contain one copy of the B. megaterium LDH integrated in the genome. LDH<br>
activity was higher than PDC activity (U/mg total cellular protein) and thus LDH<br>
could compete efficiently with PDC for pyruvate. The pdc2- strain 286-30 has<br>
clearly reduced PDC activity compared with the wild type. The observed effect of<br>
the pdc1 deletion on PDC activity in the pdc1- strain 265-15 was a more gradual<br>
decrease in activity over time.<br>
Table 28. Lactate dehydrogenase and pyruvate decarboxylase activities (265-55<br>
(PDC+,pH 5.0))<br>
Example 23: Anaerobic production of L-lactic acid in a bioreactor in rich<br>
glucose medium by C. sonorensis harboring the B. megaterium LDH gene<br>
integrated into the genome.<br>
The C. sonorensis transformant designated 265-55 described above was<br>
cultivated in a bioreactor. Batch cultivation was carried out at 35°C in a<br>
laboratory bioreactor (Biostat CT-DCU3, Braun, Germany) with a working<br>
volume of 2 L. Biomass was produced aerobically on YP medium supplemented<br>
with 150 g/L glucose. The biomass production phase was inoculated with 20 mL<br>
of culture stored in 23% (w/v) glycerol at -80ºC. The bioreactor was flushed with<br>
100 % air at a flow rate of 1.0 L/min, and stirred at 800 rpm. The dissolved-<br>
oxygen concentration was continuously monitored with an oxygen electrode<br>
(Mettler Toledo). After 22.5 hours of biomass production the bioreactor was<br>
emptied and cells were collected by centrifugation (4000 rpm, 20°C, 10 min).<br>
Medium for lactic acid production (YP supplemented with 100 g/L glucose) was<br>
pumped into the bioreactor, and was inoculated with the centrifuged biomass to a<br>
density equivalent of 4.5 g/L cell dry weight. The bioreactor was flushed with<br>
100% nitrogen at a flow rate of 1.0 L/min and stirred at 500 rpm. The pH was<br>
maintained at 5.0 ±0.1 by automated addition of 5 M potassium hydroxide (KOH).<br>
Samples were withdrawn during cultivation. Cell dry weight was<br>
determined, OD600 was measured, and cells were harvested by centrifugation. The<br>
culture supernatants were analyzed for L-lactic acid (by the L-lactic acid UV<br>
method of Boehringer Mannheim) and glucose content (by the glucose/ GOD-<br>
Perid method of Boehringer Mannheim). These results are shown in Tables 32<br>
and 33.<br>
After 120 h of cultivation 4.7 g/L lactic acid was produced from glucose<br>
(equivalent to a 52% yield).<br>
This example demonstrated that C. sonorensis overexpressing a<br>
heterologous lactate dehydrogenase was capable of producing lactic acid from<br>
glucose under anaerobic batch fermentation.<br>
Table 32: Aerobic biomass production in a bioreactor cultivation in YPD medium<br>
with 150 g/L glucose.<br>
Example 24: Production of L-lactic acid in a bioreactor in Ca(OH)2 -<br>
buffered rich glucose medium by C. sonorensis harboring B. megaterium<br>
LDH gene integrated into the genome.<br>
The C. sonorensis transformant designated 265-55 described above was<br>
cultivated by batch cultivation in a bioreactor (Biostat CT-DCU3, Braun,<br>
Germany) at 35°C as described in Example 23. After 18.5 hours of biomass<br>
production the bioreactor was emptied and cells were collected by centrifugation<br>
(4000 rpm, 20°C, 10 min). The medium for lactic acid production (YP<br>
supplemented with 100 g/L glucose) was pumped into the bioreactor and<br>
inoculated with the centrifuged biomass to a density equivalent of 6.7 g/L cell dry<br>
weight. The bioreactor was flushed with 90% nitrogen and 10% air at a flow rate<br>
of 1.0 L/min and stirred at 500 rpm. The pH was maintained at S.O ±0.1 by<br>
automated addition of 2.5 M calcium hydroxide (Ca(OH)2).<br>
Samples were withdrawn during cultivation. Cell dry weight was<br>
determined, OD600 was measured and cells were harvested by centrifugation. The<br>
culture supematants were analyzed for L-lactic acid (by the L-lactic acid UV<br>
method, Boehringer Mannheim) and glucose content (by the glucose/GOD-Perid<br>
method, Boehringer Mannheim). These results are shown in Tables 34 and 35.<br>
After 53 hours of cultivation, 26 g/L lactic acid was produced from glucose<br>
(equivalent to a 67% yield).<br>
These results demonstrated that C. sonorensis overexpressing B.<br>
megaterium lactate dehydrogenase was capable of producing lactic acid from<br>
glucose in a microaerobic batch fermentation, (2% O2) with calcium hydroxide<br>
buffering.<br>
Example 25: Production of L-lactic acid in a bioreactor in rich glucose<br>
medium by C. sonorentsis harboring L. helveticus LDH gene integrated into<br>
the genome.<br>
The C. sonorensis transformants designated 247-5 and 247-11 described<br>
above were cultivated by batch cultivation in a laboratory bioreactor (Biostat CT-<br>
DCU3, Braun, Germany) at 30°C (strain 247-11) or 35°C (strain 247-5) with a<br>
working volume of 2 L. The cultivation medium was YP supplemented with 40<br>
g/L glucose. Precultures were grown on YPD medium to an OD600 of 11-16, cells<br>
were collected by centrifugation and the bioreactor was inoculated to an OD600 of<br>
L Cultivation continued until all glucose was consumed. The pH was maintained<br>
at 5.0 ±0.1 by automated addition of 5 M potassium hydroxide (KOH). The<br>
bioreactor was flushed with 5% air and 95 % nitrogen gas at a flow rate of 0.5<br>
L/min and stirred at 500 rpm. The dissolved-oxygen concentration was<br>
continuously monitored with an oxygen electrode (Mettler Toledo).<br>
Samples were withdrawn during cultivation. Cell dry weight was<br>
determined, OD600 was measured, and the cells were harvested by centrifugation.<br>
The culture supematants were analyzed by HPLC as described above for lactic<br>
acid, glucose, ethanol and acetate. These results are shown in Tables 36 and 37.<br>
After 52 to 69 hours of cultivation, the transformants produced 26-29 g/L<br>
of lactic acid (equivalent to a 67 -72 % yield) from glucose.<br>
These results demonstrated that C. sonorensis overexpressing the L.<br>
helveticus lactate dehydrogenase gene was capable of producing lactic acid from<br>
glucose in a microaerobic batch fermentation.<br>
Example 26: Production of L-lactic acid in defined xylose medium by C.<br>
sonorensis cells harboring L. helveticus LDH gene integrated into the genome.<br>
C. sonorensis cells and the transformants disclosed above (specifically,<br>
246-1, 246-10, 247-2, and 247-5) were cultivated in YX -medium (yeast nitrogen<br>
base without ammonium sulfate and amino acids supplemented with 0.3% urea<br>
and 5% xylose). Precultures were grown in YPD-medium, cells were collected by<br>
centrifugation, washed once with YX -medium and then resuspended in 50 mL of<br>
YX medium to an OD600 of 14-22 for the cultivation experiments. Yeast cells<br>
were cultivated in 100 mL Erlenmeyer flasks with 100 rpm shaking at 30°C.<br>
When the pH reached approximately 3.5, 0.2% solid calcium carbonate was<br>
added. Samples were withdrawn during cultivation, OD600 measured, cells were<br>
harvested by centrifugation, and the culture supernatant was analyzed by HPLC, as<br>
described above. These results are shown in Table 38.<br>
After 71 hours of cultivation, the transformants harboring L. helveticus<br>
LDH produced 3.6-5.0 g/L lactic acid, equivalent to 18 - 34 % yield from xylose,<br>
whereas C. sonorensis host did not produce detectable lactic acid. The biomass<br>
increased less than 10% during the 167 hour experiment. The transformants<br>
utilized 10-30 g/L xylose and produced 4-9 g/L of lactic acid. One third of used<br>
xylose was converted into lactic acid by the transformants 246-10 and 247-5.<br>
These results demonstrated that C. sonorensis overexpressing a<br>
heterologous lactate dehydrogenase gene was capable of producing lactic acid<br>
from xylose.<br>
Example 27: Production of L-lactic acid in defined L-arabinose medium by<br>
C. sonorensis harboring L. helveticus LDH gene integrated into the genome.<br>
C. sonorensis cells and the trans form ants described above (specifically<br>
246-1, 246-10, 247-2, and 247-5) were cultivated in YA-medium (yeast nitrogen<br>
base without ammonium sulfate and amino acids supplemented with 0.3% urea<br>
and 2% L- arabinose). Precultures were grown in YPD-medium, cells were<br>
collected by centrifugation, washed once with YA-medium and resuspended on<br>
the 50 mL of YA-medium to an OD600 of 14-20 for the cultivation experiments.<br>
Yeast cells were cultivated in 100 mL Erlenrneyer flasks with 100 rpm shaking<br>
(microaerobic conditions) at 30°C. When the pH reached approximately 3.5,<br>
0.2% solid calcium carbonate was added. Samples were withdrawn during<br>
cultivation, OD600 measured, the cells were harvested by centrifugation, and the<br>
culture supernatant analyzed for lactic acid and xylose by HPLC as described<br>
above. These results are shown in Table 39.<br>
After 71 hours of cultivation the transformants harboring the L. helveticus<br>
LDH produced 2.3 - 3.2 g/L lactic acid equivalent of 14 - 34% yield from<br>
arabinose, whereas the control strain did not produce detectable lactic acid. The<br>
biomass increased 20-60% during the 167h experiment. The transformants used<br>
almost all the 20 g/L arabinose initially provided and produced 3.3-4.5 g/L of<br>
lactic acid. About 20% of used arabinose was converted into lactic acid by<br>
transformants 246-10 and 247-5.<br>
This example showed that C. sonorensis expressing a heterologous LDH<br>
gene produced lactic acid from arabinose.<br>
Example 28: Transformation of C. methanosorbosa and production of lactic<br>
acid by strains harboring B. megaterium LDH integrated in the genome<br>
C. methanosorbosa was transformed with the C. sonorensis vector pMI278<br>
described above for lactic acid production. pMI278 was digested with SalI and<br>
NotI. Lithium acetate transformation according to a modification of the method of<br>
Gietz et al. (1992, Nucleic Acids Res. 20: 1425) described above in Example 1.<br>
Cells from an overnight culture of C. methanosorbosa grown to OD600 = 0.9-1.1<br>
were collected by centrifugation, washed first with an excess of a solution of 10<br>
mM Tris-HCl, 1 mM EDTA (pH 7.5), and then with an excess of a solution of 100<br>
mM LiAc / 10 mM Tris-HCl, 1 mM EDTA (pH 7.5), and resuspended in 2 mL<br>
100 mM LiAc / 10 mM Tris-HCl, 1 mM EDTA (pH 7,5). 50 µL of cells was<br>
mixed with 10 µg of transforming DNA and 50 µg of heat-denatured herring<br>
sperm DNA. To the cells was added 300 µL, of a 40% PEG-4000 solution in 100<br>
mM LiAc / 10 mM Tris-HCl, 1 mM EDTA (pH 7.5) and the cells were then<br>
incubated at 30°C for 30 min with slow shaking. DMSO was then added (40 µL)<br>
and the cells were incubated in a 42°C water bath for 15 min. Cells were collected<br>
by centrifugation, washed with an excess of a solution of 10 mM Tris-HCl, 1 mM<br>
EDTA (pH 7.5), resuspended in YPD and incubated at 30°C overnight. Cells<br>
were spread onto solid YPD medium containing 200 µg/mL G418 and incubated<br>
at 30°C for three to five days. Transformants were streaked onto fresh selection<br>
plates twice. The transformants were designated as Cm/278-1 through Cm/278-<br>
74.<br>
Transformants were tested for their ability to produce L-lactic acid as<br>
follows. 5 mL of YPD in a 10 mL plastic tube was inoculated with a colony<br>
grown on G418 plates and incubated with shaking at 250 rpm at 30°C overnight.<br>
The cells were removed by centrifugation and the supernatant was analyzed for L-<br>
lactic acid using the L-lactic acid UV method of Boehringer Mannheim. L-lactic<br>
acid was produced at 2.3-4.3 g/L. The presence of a single copy of B. megaterium<br>
LDH gene in the genome was verified by Southern blot analysis of HindIII<br>
digested yeast DNA using the B. megaterium LDH gene as the probe.<br>
These results showed that B. megaterium LDH was able to function in C.<br>
methanosorbosa and produced lactic acid from glucose. The B megaterium LDH<br>
is operatively linked to C. sonorensis PGK1 promoter that is able to drive<br>
expression of a heterologous gene in C. methanosorbosa. Furthermore, the C.<br>
sonorensis TDH1 promoter that is operatively linked to the G418 resistance gene<br>
is also able to function in C. methanosorbosa.<br>
Example 29: Production of L-lactic acid in rich glucose medium without<br>
buffering by C methanosorbosa harboring the B. megaterium LDH gene<br>
integrated into the genome.<br>
One of the C. methanosorbosa transformants disclosed above (Cm/278-1)<br>
was cultivated in YD-medium (yeast nitrogen base without amino acids<br>
supplemented with 5% glucose and 0.5 M MES pH 5.5). Cells were then<br>
collected by centrifugation and resuspended in 50 mL of YP supplemented with<br>
5% glucose to an OD600 of 16. Yeast cells were cultivated in 250 mL Erlenmeyer<br>
flasks with 100 rpm shaking at 30°C. Samples were withdrawn during cultivation,<br>
OD600 was measured, the cells were harvested by centrifugation, and the culture<br>
supernatant was analyzed for L-lactic acid (by the L-lactic acid UV method of<br>
Boehringer Mannheim, Roche), glucose (by the glucose/ GOD-Perid method of<br>
Boehringer Mannheim, Roche), for acetate (by the acetic acid UV method of<br>
Boehringer Mannheim, Roche), and ethanol (by the ethanol UV method of<br>
Boehringer Mannheim, Roche). These results are shown in Table 40.<br>
After 24 hours of cultivation the transformant produced 8.1 g/L lactic acid<br>
(equivalent to 19 % yield) from glucose and the pH dropped to 3.5.<br>
These results demonstrated that C. methanosorbosa overexpressing a<br>
heterologous LDH produced lactic acid from glucose in rich medium at low pH.<br>
Example 30: Production of L-lactic acid in CaCO3-buffered defined glucose<br>
medium by C. methanosorbosa harboring the B. megaterium LDH gene<br>
integrated into the genome.<br>
The transformed C. methanosorbosa cells disclosed above (specifically,<br>
transformants designated Cm/278-1 and Cm/278-14) and the untransformed host<br>
strain (Cm) were cultivated in YD-medium (yeast nitrogen base without amino<br>
acids supplemented with 5% glucose and 0.5 M MES, pH 5.5). The cells were<br>
then collected by centrifugation and resuspended in 50 mL of YD medium (yeast<br>
nitrogen base without amino acids supplemented with 10 % glucose) to an OD600<br>
of 15 for the cultivation experiments. Yeast cells were cultivated in 250 mL<br>
Erlenmeyer flasks containing 4 g CaCO3 with 100 rpm shaking at 30°C. The pH<br>
of the culture medium throughout the cultivation was 6.5. Samples were<br>
withdrawn during cultivation, cells harvested by centrifugation and the culture<br>
supernatant analyzed by HPLC for lactic acid, glucose and ethanol, as described<br>
above. These results are shown in Tables 41-44.<br>
The transformants had consumed glucose at 96 hours of cultivation and<br>
had produced 63-65 g/L of lactic acid (equivalent to 63-64% yield) and 6.5-6.9 g/L<br>
of ethanol. The host strain (Cm) had used all glucose by 120 hours of cultivation<br>
and it had produced 23 g/L of ethanol and no lactic acid.<br>
These results demonstrated that C. methanosorbosa cells overexpressing a<br>
heterologous LDH gene produced lactic acid from glucose in defined medium at<br>
neutral pH. High lactic acid titers, 63-65 g/L, and yields 63-64 % were achieved.<br>
Example 31: Enzyme activities of lactate dehydrogenase and pyruvate<br>
decarboxylase and production of L-lactic acid in defined glucose medium by<br>
C. methanosorbosa harboring the B megaterium LDH gene integrated into the<br>
genome.<br>
The C. methanosorbosa transformants disclosed above (Cm/278-1 and<br>
Cm/278-14) were cultivated in 50 mL of YD-medium (yeast nitrogen base without<br>
amino acids supplemented with 5% glucose and 0.5 M MES, pH 5.5). Yeast cells<br>
were cultivated in 250 mL Erlenmeyer flasks with 250 rpm shaking to an OD600 of<br>
10 at 30°C. Samples were collected (2 mL) and cells were harvested by<br>
centrifugation. The culture supernatant was analyzed by HPLC.<br>
For enzyme activity measurements the cell pellet was washed with 1 mL of<br>
ice-cold 10 mM K2HPO4/KH2PO4, 2 mM EDTA (pH 7.5). Washed pellets were<br>
resuspended with 0.5 mL of the same buffer and stored at -70°C. Samples were<br>
thawed at room temperature and washed once with 1 mL of sonication buffer (100<br>
mM KH2PO4/K2HPO4, 2 mM MgCl2, 10 mM DTT, pH 7.5). Washed samples<br>
were resuspended to 0.5 mL of sonication buffer and homogenized with 0.5 mL of<br>
glass beads in a Bead Beater homogenizer for 1 min. After homogenization, the<br>
samples were centrifuged at 14,000 rpm for 30 min at 4°C. Supernatant samples<br>
were collected and lactate dehydrogenase and pyruvate decarboxylase activities<br>
were determined spectrophotometrically (A340) as described above in example 18.<br>
These results are shown in Table 45.<br>
At 20 h of cultivation, transformants 278-1 and 278-14 produced 0.69 and<br>
0.33 g/L lactic acid (equivalent to 7 and 4 % yield from glucose), respectively. At<br>
that time point, lactate dehydrogenase activity was 0.05 and 0.16 U/mg total<br>
cellular protein, and pyruvate decarboxylase activity was 0.71 and 0.53 U/mg total<br>
cellular protein in the transformant 278-1 and 278-14, respectively.<br>
These results demonstrated that lactate dehydrogenase activity is detected<br>
in C. methanosorbosa cells overexpressing a heterologous LDH gene and<br>
confirmed that the cells were capable of producing lactic acid from glucose. The<br>
lower activity could be attributed to a lower starting OD600 and higher aeration<br>
(250 rpm), resulting in predominantly cell growth and small amount of lactate<br>
production.<br>
Example 32. Production of lactic acid in defined xylose medium by C.<br>
sonorensis harboring the L. helveticus or the B. megaterium LDH encoding<br>
gene and by C. methanosorbosa harboring the B. megaterium LDH encoding<br>
gene integrated in the genome.<br>
Lactic acid was produced from xylose in CaCO3 buffered cultures of C.<br>
sonorensis and C. methanosorbosa cultivated on defined medium. The cell<br>
biomass was generated either on glucose or on xylose in two stages before transfer<br>
into the xylose-containing production medium.<br>
A) Biomass generation on glucose and lactate production on xvlose<br>
5 mL of YP+5% glucose medium was inoculated with a yeast colony<br>
(strain C40/288-34) grown on YPD plates. The culture was incubated overnight<br>
with 250 rpm shaking at 30°C. 50 mL of YD -medium (yeast nitrogen base, no<br>
amino acids supplemented, 5% glucose, and 0.5 M MES, pH 5.5) in 250 mL<br>
Erlenmeyer flasks was inoculated into an initial OD600 of 0.1 and incubated with<br>
250 rpm shaking overnight at 30°C until an OD600 of 10 was reached. The cells<br>
were resuspended in 50 mL of YX- medium (yeast nitrogen base without amino<br>
acids supplemented with 5 % xylose) to an OD600 of 11-13. The cells were<br>
cultivated in 250 mL Erlenmeyer flasks containing 2 g CaCO3 with 100 pm<br>
shaking at 30°C. Samples were withdrawn during cultivation. The cells were<br>
removed by centrifugation, and the culture supernatant was analyzed for lactic<br>
acid and xylose by HPLC as described above (Example 14). Two independent<br>
experiments were carried out and the results are shown in Table 46.<br>
The C. sonorensis transformant C40/288-34 consumed 50 g/L of xylose in<br>
7-8 days and produced 13-16 g/L of lactic acid, corresponding to 28-32% lactic<br>
acid yield from xylose.<br>
B) Biomass generation on xylose and lactate production on xylose<br>
Transformants 265-55 and 265-44 (C. sonorensis) harboring the B.<br>
megaterium LDH, transformants C40/288-34, C40/288-36, 257-3, and 246-27 (C.<br>
sonorensis) harboring the L. helveticus LDH, and transformants Cm/278-1 and<br>
Cm/278-42 (C. methanosorbosa) harboring the B. megaterium LDH were used.<br>
50 mL of YP+5% xylose medium in a 250 mL shake flask were inoculated<br>
with a yeast colony grown on YP+2% xylose plates. The culture was incubated<br>
overnight with 250 rpm shaking at 30°C until an OD600 of 10 was reached, then 50<br>
mL of YX -medium (yeast nitrogen base, no amino acids supplements, 5% xylose,<br>
and 0.5 M MES, pH 5.5) in a 250 mL Erlenmeyer flasks was inoculated to an<br>
initial OD600 of 0.2. The cells were incubated with 250 rpm shaking overnight at<br>
30°C until an OD600 of 7-10 was reached. The cells were resuspended in 50 mL of<br>
YX- medium (yeast nitrogen base, no amino acids supplements, and 5 % xylose)<br>
to an OD600 of 11-12. The cells were cultivated in 250 mL Erlenmeyer flasks<br>
containing 2 g CaCO3 with 100 rpm shaking at 30°C. Samples were withdrawn<br>
during cultivation. The cells were removed by centrifugation, and the culture<br>
supernatant was analyzed for lactic acid and xylose by HPLC as described above<br>
(Example 14). The results are shown in Table 47.<br>
34 converted xylose into lactic acid, after transfer into xylose-containing medium,<br>
at approximately 30% yield. In comparison, when the biomass was generated on<br>
xylose, the same transformant converted xylose into lactic acid with a much higher<br>
yield (63-76%), after transfer into xylose-containing medium. The xylose-grown<br>
biomass also consumed xylose faster than the glucose-grown biomass under lactic<br>
acid production conditions. The data suggests that increased lactic acid yields can<br>
be obtained when the cells are "adapted" to sugars other than glucose, for example<br>
xylose, by growth on xylose-containing medium, prior to their transfer to the<br>
xylose-containing lactic acid production medium.<br>
Example 33. Production of L-lactic acid in defined glucose medium by C.<br>
sonorensis comprising a deleted pdc1 gene and a disrupted pdc2 gene and<br>
harboring the L. helveticus LDH encoding gene integrated into the genome.<br>
The C. sonorensis transformants designated 257-3, C40/288-2, C40/288-<br>
34 and C40/28S-11 (described above in Example 13) were cultivated and assayed<br>
as described in Example 19, with the exception that the cells were suspended to an<br>
OD600 = 18 for the lactate production phase. The culture supernatant was analyzed<br>
for lactic acid, glucose, and ethanol as described above. These results are shown<br>
in Table 48.<br>
The PDC1- strain 257-3 (where PDC1 is deleted) produced 89 g/L of lactic<br>
acid in 48 h, corresponding to a 93% yield from glucose (g/g). The PDC1-<br>
(deleted) pdc2- (where pdc2 is disrupted) strains C40/288-2, C40/288-34 and<br>
C40/288-11 produced 86-87 g/L of lactic acid in 72 h, corresponding to 89-90%<br>
yield from glucose (g/g). No ethanol was detected at these time points.<br>
It is to be understood that while the invention is described in conjunction<br>
with the foregoing detailed description and examples, they are intended to<br>
illustrate and not limit the scope or the spirit of the invention, which is defined by<br>
the appended claims. Other aspects, advantages, and modifications are within the<br>
scope of the claims.<br>
CLAIMS:<br>
1. A recombinant nucleic acid construct comprising a nucleic acid<br>
having a nucleotide sequence encoding a lactate dehydrogenase protein, wherein,<br>
the lactate dehydrogenase protein is operatively linked to a promoter functional in<br>
a yeast cell from a species of genera Candida.<br>
2. The recombinant nucleic acid construct, as claimed in claim 1, wherein the.<br>
nucleotide sequence encodes a lactate dehydrogenase protein from Bacillus<br>
megaterium.<br>
3. The recombinant nucleic acid construct, as claimed in claim 2, wherein the<br>
nucleotide sequence encodes a lactate dehydrogenase protein from Lactobacillus<br>
helveticus<br>
4. The recombinant nucleic acid construct, as claimed in claim 1, wherein the<br>
nucleotide sequence encodes a lactate dehydrogenase protein from Rhizopus<br>
oryzae.<br>
5. The recombinant nucleic acid construct, as claimed in claim 1. wherein the<br>
promoter is from a Candida species that is Candida sonorensis, Candida<br>
parapsilosis, Candida naeodendra, Candida methanosorbosa, Candida<br>
entomophila, Candida krusei, Candida blankiit or Candida diddensiae.<br>
6. The recombinant nucleic acid construct, as claimed in claim 1, further<br>
comprising a gene coding for resistance to a selective agent.<br>
7. The recombinant nucleic acid construct, as claimed in claim 6, wherein the gene<br>
coding for resistance to a selective agent is a bacteria) neomycin resistance gene,<br>
kanamycin resistance gene, hygrornycin resistance gene, or zeocin resistance gene.<br>
8. The recombinant nucleic acid construct, as claimed in claim 1, further<br>
 comprising a gene that encodes a protein that processes carbon sources other than<br>
monosaccharide hexoses.<br>
9. The recombinant nuclecic acid construct, as claimed in claim 8, wherein the gene<br>
encodes an alpha-galactosidase.<br>
10. The recombinant nucleic acid construct, as claimed in claim 9, wherein the<br>
gene is yeast MEL5.<br>
11. A genetically modified cell from genera Candida, comprising at least<br>
one exogenous LDH gene.<br>
12. A genetically modified cell from genera Candida transformed with the<br>
recombinant nucleic acid construct as claimed in any of claims 1 to 10, wherein the<br>
cell expresses the lactate dehydrogenase protein.<br>
13. The genetically modified cell from genera Candida, as claimed in claim<br>
11, wherein the cell further expresses reduced pyruvate decarboxylase (PDC) activity.<br>
14. The genetically modified cell from genera Candida as claimed in claim<br>
13, wherein the reduced PDC activity results from deletion of at least one pyruvate<br>
decarboxylase gene.<br>
15. The genetically modified cell from genera Candida as claimed in claim<br>
13, wherein the reduced PDC activity results from genetic disruption of at least one<br>
pyruvate decarboxylase gene.<br>
16. The genetically modified cell from genera Candida as claimed in claim<br>
11, that is a Candida sonorensis, Candida parapsilosis, Candida naeodendra, Candida<br>
methanosorbosa, Candida entomophila, Candida krusei, Candida blankii, or Candida<br>
diddensiae cell.<br>
17. The genetically modified cell from genera Candida, as claimed in claim<br>
12, wherein the promoter is a promoter from the Candida species of the cell.<br>
18. The genetically modified cell from genera Candida as claimed in claim<br>
12, wherein the promoter is a promoter from a species other than the Candida species<br>
of the cell.<br>
19. A genetically modified cell from genera Candida comprising a deletion<br>
at a PDC1 gene locus, a disruption at a pdc2 gene locus and two or more copies of<br>
lactate dehydrogenase genes in the cellular genome at each of the PDC1 and pdc2 loci.<br>
20. The genetically modified cell from genera Candida as claimed in claim<br>
19, wherein the two or more copies of lactate dehydrogenase genes are each operably<br>
linked to a promoter that is transcriptionally active in the Candida cell.<br>
21. The genetically modified cell from genera Candida contains non-<br>
functional or deleted PDC1 or pdc2 gene, characterized by at least a 10-fold reduction<br>
of ethanol production when cultured in the presence of a defined glucose or rich<br>
glucose medium.<br>
22. The genetically modified cell from genera Candida as claimed in claim<br>
21, further comprising a gene encoding for a lactate dehydrogenase.<br>
23. The genetically modified cell from genera Candida, as claimed in claim<br>
22, wherein the lactate dehydrogenase is operably linked to a pdc1 or pdc2 promoter.<br>
24. The genetically modified cell from genera Candida as claimed in claim<br>
11, wherein the cell has increased lactic acid dehydrogenase activity relative to the<br>
Candida cell that is untransformed.<br>
25. A method for producing lactic acid comprising the steps of<br>
a) culturing a cell, as claimed in claim 11, under conditions that allow the cell to<br>
proliferate; and<br>
b) fermenting the cell culture, as claimed in (a), in a nutrient medium comprising a<br>
sugar, under conditions whereby the amount of the sugar converted by the cell to<br>
lactic acid is increased, relative to the amount of the sugar converted to lactic acid<br>
by an untransformed Candida cell<br>
26. The method, as claimed in claim 25, wherein the lactic acid is L-lactic<br>
acid.<br>
27. The method, as claimed in claim 25, wherein the cell is a Candida sonorensis<br>
cell.<br>
28. The method, as claimed in claim 27, wherein the cell comprises at least one<br>
lactate dehydrogenase gene that is a L. helveticus, B. megaterium, or R. oryzae<br>
lactate dehydrogenase gene, or combinations thereof.<br>
29. The method, as claimed in claim 25, wherein the cell is a Candida<br>
methanosorbosa cell.<br>
30. The method, as claimed in claim 29, wherein the cell comprises at least one<br>
lactate dehydrogenase gene that is a L. helveticus, B. megaterium, or R. oryzae<br>
lactate dehydrogenase gene, or combinations thereof.<br>
31. The method, as claimed in claim 25, wherein the cells are cultivated in a<br>
medium that is a buffered medium, wherein the medium is buffered to maintain a<br>
pH in the nutrient medium ranging from pH 5 to pH 9.<br>
32. The method, as claimed in claim 25, wherein the final pH of the culture<br>
medium after lactic acid production is ranging from pH 2.6 to pH 5.<br>
33. The method, as claimed in claim 25, wherein fermenting step is performed under<br>
an atmosphere that contains no more than 2% oxygen.<br>
34. The method, as claimed in claim 25, wherein the fermenting step is performed<br>
under anaerobic conditions.<br>
35. The method, as claimed in claim 25,, wherein the sugar in the nutrient medium is<br>
one or a plurality of hexoses, one or a plurality of pentoses, or combinations<br>
thereof.<br>
36. The method, as claimed in claim 25, wherein the sugar in the nutrient medium is<br>
glucose, xylose, or L-arabinose, or combinations thereof.<br>
37. The method, as claimed in claim 36, wherein the sugar in the nutrient medium is<br>
glucose, and wherein the yield of lactic acid relative to the amount of glucose<br>
consumed by the cell is at least 60% by weight.<br>
38. The method, as claimed in claim 36, wherein the sugar in the nutrient medium is<br>
xylose, and. wherein the yield of lactic acid relative to the amount of xylose<br>
consumed by the cell is at least 15% by weight.<br>
39. The method, as claimed in claim 36, wherein the sugar in the nutrient medium is<br>
L-arabinose, and wherein the yield of lactic acid relative to the amount of L-.<br>
arabinose consumed by the cell is at least 20% by weight.<br>
40. The method, as claimed in claim 25, wherein the Candida cell is a Candida<br>
diddensiae, Candida parapsilosis, Candida naeodendra, Candida krusei, Candida<br>
blankii, Candida methanosorbosa or Candida entomophila cell.<br>
41. A method to reduce pyruvate decarboxylase activity in a cell from<br>
genera Candida comprising transforming the cell with the a recombinant nucleic<br>
acid construct, wherein the nucleic acid construct comprises a selectable gene<br>
flanked by 5" and 3" flanking sequences from at least one pyruvate decarboxylase<br>
gene native to the genera Candida.<br>
42. The method, as claimed in claim 41, wherein the at least one pyruvate<br>
decarboxylase gene is selected from the group consisting of pyruvate<br>
decarboxylase 1 (pdc1), pyruvate decarboxylase 2 (pdc2), or both pdc1 and pdc2,<br>
43. The method, as claimed in claim 41, wherein the flanking sequences are a<br>
promoter and a terminator for at least one pyruvate decarboxylase gene native to<br>
the genera Candida.<br>
44. A genetically modified Candida cell made by the method, as claimed in<br>
any of claims 41, 42 or 43.<br>
The present invention relates to biocatalysis that are cells, optimally of the Crabtrce-negative phenotype comprising<br>
expression vectors encoding genes heterologous to the cell that enable increased production of organic products. More specifically,<br>
the invention relates to genetically modified Candida cells. methods for making the Candida cells, and their use in production of<br>
organic products, particularly lactic acid.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1sZXR0ZXIgcGF0ZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjgzLWtvbG5wLTIwMDQtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">683-kolnp-2004-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="215519-immunogenic-composition-vaccine-made-therefrom-and-kit-containing-said-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="215521-aqueous-composititions-containing-metronidazole.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>215520</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00683/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Feb-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Feb-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-May-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NATUREWORKS LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>12700 WHITEWATER DRIVE, MINNETONKA, MINNESOTA 55343</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RAJGARHIA VINEET</td>
											<td>APARTMENT # 1103, 5455 SMETANA DRIVE, HOPKINS, MN 55343</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PENTTILA MERJA</td>
											<td>VATTUNIEMENKATU 2 A 72, FIN-00210 HELSINKI</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RUOHONEN LAURA</td>
											<td>VAINAMOISENKATU 15A 4, FIN-00100 HELSINKI</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ILMEN MARJA</td>
											<td>SANKARITIE 7 A 9, FIN-00320 HELSINKI</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SUOMINEN PIRKKO</td>
											<td>7801 KINGSVIEW LANE N, MAPLE GROVE, MN 55311</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KOIVURANTA KARI</td>
											<td>MAKITOPANTIE 17 D 39, FIN-00640 HELSINKI</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/16223</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-05-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/992, 430</td>
									<td>2001-11-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/215520-methods-and-materials-for-the-production-of-organic-products-in-cells-of-candida-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:38:12 GMT -->
</html>
