<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/215963-a-method-for-determining-the-ability-of-a-test-compound-to-alter-phosphorylation-and-degradation-of-human-period-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:07:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 215963:A METHOD FOR DETERMINING THE ABILITY OF A TEST COMPOUND TO ALTER PHOSPHORYLATION AND DEGRADATION OF HUMAN PERIOD PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR DETERMINING THE ABILITY OF A TEST COMPOUND TO ALTER PHOSPHORYLATION AND DEGRADATION OF HUMAN PERIOD PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is directed to methods to identify test compounds that alter circadian rhythms of mammals, and more specifically, directed to methods for determining the ability of a test compound to alter hCKI &amp;#948; and/or e phosphorylation of a human Period protein. The present invention is also directed to a method for determining the ability of a test compound to selectively alter phosphorylation, interaction with, or alternatively degradation, of one or more human Period proteins relative to its ability to alter phosphorylation, interaction with, or alternatively degradation, of a different human Period protein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention relates to a method for determining the ability of a test compound to alter phosphorylation of atleast one human period proteins and a method for determining the ability of a test compound to alter degradation of a human period protein.<br>
BACKGROUND OF THE Envelop<br>
Circadian rhythms generated by endogenous biological pacemakers are present in a number of diverse organisms including humans, fungi, insects and bacteria (Dunlap, J. C. (1999) Cell, 96, 271-290; Hastings, J. W., et al., (1991) in Neural and Integrative Animal Physiology, ed. Prosser, C. L. (New York: Wiley-Liss), pp.435-546; Allada, R., et al., (1998) Cell, 93, 791-804; Kondo, T., et al., (1994) Science, 266, 1233-1236; Crosthwaite, S. K., et al., (1997) Science, 276, 763-769). Circadian clocks are essential in maintaining biological rhythms. They are self-sustaining and constant even under conditions of total darkness but can be entrained by environmental signals such as light and temperature changes. Endogenous clocks control patterns of activity including daily fluctuations in behavior, food intake and sleep/wake cycle as well as physiological changes such as hormone secretion, and fluctuations in body temperature (Hastings, M., (1997) Trends Neurosci. 20, 459-464; Kondo,<br><br>
T., et al., (1993) Proc. Natl. Acad. Set USA, 90, 5672-5676.; Report, S.M., &amp; Weaver, D.R. (1997)Ce//, 89, 487-490).<br>
Genetic and molecular studies in Drosophila have allowed for the elucidation of some of the genes involved in circadian rhythmicity. What has emerged from these studies is a pathway closely auto-regulated and comprised of a transcription/translation-based negative feed back loop punlap, J.C. (1999) Cell, 96, 271-290; Dunlap, J.C. (1996) Annie. Rev. Genet. 30, 579-601; Hall, J.C. (1996) Neuron. 17, 799-802). Two critical components of the central clock are molecules termed Period or PER and Timeless or TIM.<br>
The per locus, first discovered in Drosophila, is a necessary element in controIHng circadian rhythms in adult eclosion behavior and locomotor activity (Konopka, R.J., &amp; Benzer, S. (1971) Proc. Natl. Acad. Sci. USA 68, 2112-2116). Missense mutations of PER can either shorten (per ) or lengthen (per ) the period of circadian rhythms, while nonsense mutations (per"^ cause arrhythmicity in their behaviors (Hall, J.C. (1995) Trends Neurosci. 18, 230-240). In the suprachiasmatic nuclei (SCN) of mammals, three PER homologues, peri, per2, and per3 have been identified. The protein products of these mamma an genes share several regions of homology to each other (Zylka, M.J., et al., (1998) Neuron 20,1103-1110; Albrecht, U., et al., (1997) Cell 91,1055-1064.). Per mRNA and protein levels oscillate during the daily cycle, but only PERI and PER2 oscillate in response to light (Zylka, M.J., et al., (\99S) Neuron 20, 1103-1110., Shearman, L.P., et al, (1991) Neuron 19,1261-1269).<br>
All PER proteins contain a protein/protein interacting region called the PAS domain that is necessary for dimer formation (Zylka, M.J., et al., (1998) Neuron 20,1103-1110.). Another PAS containing protein, TIM was isolated by a yeast two-hybrid genetic screen using PER as a bait (Gekakis, N., et al., (1995) Science 270, 811-815). As PER protein levels increase, PER forms heterodimers with TM. TIM/PER heterodimer formation is required for PER regulation because mutations in tim, cause a loss in circadian rhythm which is<br><br>
accompanied by a loss of per niRNA oscillation and the inability of PER to undergo nuclear translocation (Sangoram, A.M., et al., (1998) Neuron 21,1101-1113; Zylka, M.J., et al., (1998) Neuron 21,1115-1122).<br>
Recently, several additional molecular components of circadian rhythmicity including CLOCK and BMALA40P3 have been discovered using genetic screening and biochemical characterization (Gekakis, N., et al., (1998) Science 280,1564-1569; King, D.P., et al., (1997) Cell 89, 641-653; Allada, R., et al, (1998) Cell 93, 791-804).<br>
Subsequent studies shed light on how PER is regulated at transcriptional levels. CLOCK and BMAL/M0P3, both contain basic-helix-loop-helix domain, a PAS domain, and form heterodimers to each other. Once PER is transcribed, translated and accumulated, PER translocates to the nucleus and binds to CLOCK through their common PAS domains and down regulates its own transcription, forming a negative feedback loop (Allada, R., et al., (1998) Cell 93, 791-804; Darlington, T.K., et al, (1998) Science 280, 1599-1603; Hao, H., et al., (1997) Mol. Cell. Biol. 17, 3687-3693; Jin, X., et al., (1999) Cell 96, 57-68.).<br>
In addition, PER is modified and regulated at post-translational levels. Both PER and TIM appear to undergo phosphorylation which is effected by circadian oscillation (Edery, I., et al., (1994) Proc. Natl. Acad. Sci. USA 91, 2260-2264; Lee, C, et al., (1998) Neuron 21, 857-867). A Drosophila kinase termed double time (DBT) was recently cloned (Price, J.L., et al., (1998) Cell 94, 83-95, Kloss, B., et al., (1998) Cell 94, 97-107). Mutations in DBT cause either shortened or lengthened period of the behavioral rhythm. A P-element insertion mutation in DBT abolishes the circadian oscillations of PER in larval brain, indicating that DBT is an essential component of the Drosophila clock. PER accumulates in these mutants to high levels and is hypophosphorylated. DBT has not been shown to directly phosphorylate PER. CKIs is a closely related homologue of DBT in mammals (Kloss, B., et al., (1998) Cell 94, 97-107).   CKIE and DBT are 86% homologus at the amino acid level in the kinase<br><br>
domain. hCKIs, first identified by Fish et al, is one of several CKI isoforms (a, P, y, 5) which has serine/threonine protein kinase activity (Fish, K.J., et al., (1995) J. Biol. Chem. 270, 14875-14883; Rowles, J., et al, (1991) Proc. Natl. Acad. Sci. USA 88, 9548-9552). CKIs are involved in regulation of cellular DNA metabolism. Saccharomyces mutants with defective a HRR25 gene, a homologue to mammalian CKI, show sensitivity to double-stranded DNA breaks (Hoekstra, M.F., et al., (1991) Science 253, 1031-1034). Several in vitro substrates for hCKI have been identified which include RNA polymerases I and 11, p53, BcBa, and simian virus 40 large T antigen.   However, very little evidence exist which correlates hCKI phosphorylation to changes in substrate function, and to date, no clock genes have been shown to be hCKI 5 and s substrates.<br>
Circadian rhythms are controlled by sequential phosphorylation of, and alterations of protein levels of, certain key proteins in the circadian pathway. Period (PER), a central component of the circadian clock pathway, undergoes daily oscillation in abundance and phosphorylation state. PER genes have been identified in Drosophila PER, designated dPER, mouse PER, designated mPER, and human PER, designated hPER. In Drosophila there is only one PER, which has most homology to the PERI proteins. Both humans and mice have three PERs, designated PER 1, 2 and 3. BRIEF SUMMARY OF THE INVENTION<br>
The present invention is directed to discovery that hCKI 6 and s phosphorylate human Period proteins and that phosphorylated human Period proteins are degraded. As a result, the present invention is directed to methods to identify test compoimds that alter circadian rhythms of mammals, and more specifically, directed to methods for determining the ability of a test compound to aher hCKI 5 and s phosphorylation of a human Period protein. The present invention is also directed to a method for determining the ability of a test compound to alter degradation of a phosphorylated human Period protein.   The present invention is also<br><br>
directed to a method for determing the ability of a test compound to selectively aher phosphorylation, or alternatively degradation, of one or more human Period proteins relative to its ability to alter phosphorylation, or alternatively degradation, of a different human Period protein and subsequently alter the circadian rhythm of a mammal.<br>
Accordingly the present invention provides a method for determining the ability of a test compound to alter phosphorylation of atleast one human Period proteins, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein and atleast one human Period proteins selected from the group consisting of hPERl, hPER2 and hPER3, and<br>
(2)	determining the level of phosphorylation of human Period protein.<br>
Accordingly the present invention also provides a method for determining the ability of a test compound as herein described to alter degradation of a human Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 6 and/or 8 protein<br>
and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition of the test compound, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br><br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure   1.  In   Vitro  Phosphorylation  of Casein,  IkB.aIpha.,  and  hPERl  by Recombinant hCMIs.<br>
Purification of recombinant casein kinase(s) and kinase assays conditions are described in the Materials and Methods, below. (A) hCKIe (lanes 1 to 3), hCKIs-K38R8 (lanes 4-6), or buffer control (lanes 7 and 8) is incubated either alone (lanes 1 and 4), with casein (lanes 2 and 5), or with IkBa (lanes 3 and 6), and kinase assays are performed as described in Material and Methods. Molecular weight markers are indicated to the left. (B) Lysates from 293T cells transfected with vector (lanes 1, 4 and 8), luciferase (lanes 2, 5 and 9) or hPERl (lanes 3, 6, and 10) are prepared and immunoprecipitated using the M2 anti-Flag mAb and kinase assays are performed with hCKIs (lanes 1 to 6), hCKlE-K38R8 (lane 7) or buffer control (lanes 8-10). Immunoprecipitates are heat-inactivated at 65"C for 30 min prior to the kinase assay (lanes 4-6). Samples are resolved by 12% SDS-PAGE. The gel is stained by Coomassie R-250, dried and autoradiographed.<br>
FIG. 2. (A and B) Western Blot Analysis of hPERl and hCKIe.<br>
293T cells are transfected with hPERl and vector (lane 1), hPERl and hCKIe (lane 2), hPERl and hCKl8-K38R (lane 3), vector and hCKIs (lane 4), or vector and hCKIs-K38R (lane 5). At 24 hr post-transfection, cells are harvested and lysates are prepared as described in the Materials and Methods. 40 iig of total 293T lysate are loaded onto a 3-8% gradient NU-PAGE. Proteins are transferred to PVDF membranes and Western blotted using the M2 anti-Flag mAb (1:1000) or anti-hCKIs mAb (1:750). (C) Lambda phosphatase treatment of<br><br>
iPERl. 293T cells are transfected with hPERl and vector (lanes 1 and 4), hPERl and hCKJs lanes 2 and 5), or hPERl and hCKlE-K38R (lanes 3 and 6) and labeled with [^^S]methionine ^250 jiCi/ml). Lysate is immunoprecipitated using the M2 anti-Flag mAb and then either treated with recombinant lambda phosphatase (lanes 4, 5 and 6) or mock treated (lanes 1, 2 and 3). Figure 3. Pulse-chase labeling of hPERl co-transfected with hCKIe.<br>
293T cells are co-transfected with either hPERl and vector (panel A) or hPERl and hCKIs (panel B).   Three hours post transfection, 293T cells are pulse labeled with [^^Sjmethionine and cysteine (1000 i^Ci/ml) for 30 min and then chased for the times indicated at the top of each gel. Cells are lysed and immunoprecipitated with M2 anti-Flag mAb and iiPERl is resolved on 8% SDS-PAGE.    Molecular weight markers are indicated to the left. (C) Bar graph representing a phosphoimaging scan of the area surrounding and including the hPERl band from each lane. Bars (2-30 hr) are based on the percentage of total counts per minute (cpm) as compared to counts of the zero time point. Solid bars indicate hPERl co-transfected with vector. Cross-hatched bars indicate hPERl co-transfected with hCKIe. Figure 4. Protein interaction between hPERl and hCKls. (A, B, C and D)<br>
293T cells are transfected with vector and hCKIe (lane 1), vector and hPERl (lane 2), and hCKIe and hPERl (lane 3). 24 hr post-transfection, cells are harvested and lysates are prepared. Lysates are immunoprecipitated with M2 anti-Flag mAb (panels A and C), HA mAb (panel B), or with anti-hCKIs mAb (panel D) and Western blotted with anti-HA mAb (panel A), M2 anti-Flag mAb (panel B and D), or anti-hCKIs mAb (panel C). In lane 4, Western blot - analysis is performed on crude lysates before immunoprecipitation with the mAb indicated. All proteins are resolved on a 10% SDS-PAGE.<br><br>
Figure 5. Mapping of the hCKIe phosphorylation sites.<br>
(A) Schematic representation of the recombinant truncated hPERl mutants. Truncation mutants are constructed as described in the Materials and Methods, below. ORF indicates the complete open reading frame of hPERl, amino acid residues 1 to 1289. Restriction sites used for the generation of Nl, N2, N3, N4, and C5 are indicated in Table 1. Open bars represent mutants that did not show molecular mass shifts. Solid bars represent mutants that did show molecular mass shifts. Crossed-hatched area of ORF, N2, N3, and N4 represent the region of putative phosphorylation of hPERl by hCKIs. (B) Western blot analysis of co-transfected hPERl truncation mutants from 293T lysates. 293T cells are co-transfected with hPERl and vector (lane 1), hPERl and hCKIe (lane 2), or hPERl and hCKIs-K38R (lane 3) and hPERl are analyzed by Western blot analysis using M2 anti-Flag mAb. Molecular weight markers are indicated on the left. Arrows indicate the position of migration of each truncated mutant protein. hPERI ORF and mutants Nl, and N2 are routinely resolved on a 10% SDS-PAGE, while hPERl mutants N3, N4, CI, C2, C5, and C6 are routinely resolved on a 12% SDS-PAGE. FIGURE 6 Period mRNA concentration<br>
Time is given in hours on the bottom axis; the other axis is the activity, body temperature and Per mRNA levels over time. Per mRNA levels oscillate over a twenty-four hour period, and are inversely correlated with period length in Drosophila. Similar oscillations are observed in normal Per mice; however. Per knockout mice have an altered circadian rhythm. FIGURE 7     CLOCK PROTEIN PATHWAY<br>
This is a schematic representation of the Clock protein pathway. hCKI 5 and/or z phosphorylate Period, as represented by PO4, resulting in its degradation. Clock and BMal<br><br>
interract in the PAS domain, and initiate transcription of Period mRNA, resulting in increased<br>
levels of Period protein.<br>
FIGURE 8     PERIOD PROTEIN AND PHOSPHATE LEVELS OVER TIME<br>
The bottom axis is time in hours over a twenty-four hour period; the other axis is Period protein (ii^icated in whiteyfind phosphorylation levels (indicated in colorP At the beginning of the circadian cycle, Period protein level is low, and relatively unphosphorylated. As Period protein levels increase to peak around 8 pm, relative phosphorylation of the Period protein also increases, and continues as Period protein levels decrease. FIGURE 9     CKIs/HUMAN PERIOD 1 CO-TRANSFECTION<br>
The top right panel shows hPerl phosphorylation. The first column shows 1 uM of test compound; the second column shows 10 uM of test compound; the third column shows 30 uM of test compound; the fourth column shows Per alone; the fifth and last column shows Per and hCKIs. The top right panel shows a control test compound. The middle panel shows CKI £ inhibitor test compound S943166 and the bottom right panel shows CKI s inhibitor test compound W0236. These results demonstate that inhibition of Per phosphorylation results in increased protein stability and levels. FIGURE 10   HUMAN PERIOD mRNA LEVELS<br>
The top graph provides a pictorial representation of the data presented in the bottom graph. The bottom axis is time in hours; the other axis is the relative real-time endogenous mRNA levels of hPERl either Rati fibroblast or Rat SCN (suprachiasmatic nucleus) as determined by RT-PCR. Two test compounds are represented; the test compound represented by boxes is "C" which is added to the cells at lOuM, the second test compound is represented by triangles is "F"(Fluoxitin) which is added to the cells at 10 uM. The circadian rhythm of cultured cells decreases over time, hence the amplitude of the response decreases over time.<br><br>
These results show that the control compound do not aher CKI activity also do not alter the<br>
circadian rhythm of cells.<br>
FIGURE 11   HUMAN PERIOD mRNA LEVELS<br>
The top graph provides a pictorial representation of the data presented in the bottom graph. The bottom axis is time in hours; the other axis is the relative real-time mRNA levels of hPERl Rati fibroblast or Rat SCN cells as determined by RT-PCR. Two test compounds are represented; the test compound represented by boxes is "C" which is added to the cells at 10 uM, the second test compound is represented by triangles is S943166 which is added to the cells at 10 uM. These results show that the the test compound, S943166, alters the circadian rhythm of cells by shifting the mRNA oscillation of hPERl, and shorten the circadian rhythm to about 20 hrs.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention is directed to methods to identify test compounds that alter circadian rhythms of mammals, and more specifically, directed to methods to determine the ability of a test compound to alter hCKI 5 and/or s phosphorylation of a human Period protein, preferably human Period 1, human Period 2 and/or human Period 3. In addition, the present invention is directed to a method for determining the ability of a test compound to alter degradation of a phosphorylated human Period protein.   The present invention is also directed to a method for determing the ability of a test compound to selectively alter phosphorylation, or alternatively degradation, of one or more human Period proteins relative to its ability to alter phosphorylation, or alternatively degradation, of a different human Period protein.<br>
The present invention relates to a method for determining the ability of a test compound to alter hCK I 5 and/or E phosphorylation of hPERl, hPER2 and/or hPER 3, and methods of identifying compounds that alter degradation of hCKl 5 and/or £ phosphorylated hPERl, hPER2 and/or hPER 3 in a cell. The present invention also relates to a method for<br><br>
determining the ability of a test compound to alter the stability of hPERl, hPER2 and/or hPER 3, or and increasing protein degradation of hPERl, hPER2 and/or hPER 3. The present invention provides a method method for determining the ability of a test compound to alter circadian rhythm of a mammal.<br>
An aspect of the present invention is to determine the ability of a test compound to alter phosphorylation of hPERl, hPER2 and hPER 3 by hCKI 5 and/or s. Another aspect of this invention is determine the ability-of a test compound to inhibit phosphorylation of hPERl, hPERZ and hPER 3 by hCKI 5 and/or z, comprising adding a test compound to a screening system comprising a human Period protein selected from the group consisting of hPERl, hPER2 and hPER 3, and hCKI 5 and/or s under conditions which permit phosphorylation, and determinign the level of phosphorylation of the human Period protein. In one preferred embodiment, the screening system comprises a source of phosphate. A preferred source of phosphate is ATP.<br>
The term "amino acid" refers to the meaning including either of optical isomers, i.e., aix L-isomer and a D-isomer of naturally-occurring and non-naturally-occurring amino acids. Thus, the term "peptide" refers to the meaning including not only peptides constituted by L-amino acids solely but also peptides comprising D-amino acids partially or totally.<br>
Furthermore, the term "amino acid" includes only twenty naturally-occurring amino acid residues which constitute natural proteins, as well as other alpha-amino acids, beta.-, gamma- and delta-amino acids, and non-naturally-occurring amino acids, and the like. Thus, the proteins, human Period and human hCKI 5 and/or z, may be modified with one or more amino acid residues conservative amino acid residues, for example, one having a similar charge, polarity or other property of one of the alpha-amino acid residue which constitute natural proteins, as well as other alpha-amino acids residues, and beta-, gamma- and delta-amino acid residues, non-natural amino acid residues, and the like. Examples of suitable beta-,<br><br>
gamma- and delta-amino acids include beta-alanine, gamma-aminobutyric acid and ornithine. Examples of other amino acid residues other than those constituting natural proteins or the non-natural amino acids include 3,4-dihydroxyphenylalamne, phenylglycine, cyclohexylglycine, l,2,3,4-tetrahydroisoquinolin-3-carboxylic acid or nipecotinic acid.<br>
The terms "hPERl", "hPER2", "hPERS", " hCKI 5" and "hCKIs" includes full length proteins of human Period 1, human Period 2, human Period 3, human Casein Kinase I 5 and human Casein Kinase I s, respectively, alleles and derivatives of hPERl, hPER2, hPER 3, and hCKl 5 and/or s proteins. Derivatives include alternation from naturally-occurring forms of these proteins by one or more different amino acids, truncated proteins, and fusion proteins of the full length or truncated protein containing either 3" or 5"-"tags", as well as naturally-occurring and non-naturally-occurring mutant sequences provided in the literature cited above and submitted to public databases such as in GeneBank. Derivatives of these proteins include proteins which contain a leader, epitope or other protein sequence, such as a Myc   -tagged, his-tagged, or a Flag^^ epitope tag sequence. Human Period 1 sequence is accessible under Gene Bank Accession AB002107, NE) g2506044, submitted by H. Tei on March 24,1997. Human Period 1 sequence was also published in Tei, H., et al.. Nature 389:512-516(1997). Human Period 2 sequence is accessible under Gene Bank Accession NM003894, NID g4505710, submitted by T. Nagase et al. Human Period 2 sequence was also published in Nagase, T., et al., DNA Res. 4(2):141-150 (1997) and in Shearman, LP., et al., Neuron 19(6): 1261-1269(1997).   Human Period 3 genomic sequence is accessible under Gene Bank Accession Z98884. Human Casein Kinase I 5 sequence is accessible under Gene Bank Accession U29171. Human Casein Kinase I delta was also pubHshed in Kusda, J., et al, Genomics 32:140-143(1996). Human Casein Kinase I E sequence is accessible under Gene Bank Accession L37043. Human Casein Kinase I epsilon was also published in Fish, K.J., et<br><br>
ah, J. Biol. Chem. 270:14875-14883(1995). The c-MYC tagged CKI S was a gift form Dr. David Virshup.<br>
The term "base sequence" refers to RNA sequences as well as DNA sequences encoding hPERl, hPER2, hPER3, or hCKIe, including derivatives thereof<br>
The proteins"hCKI5" and "hCKIs" according to the present invention is a protein, or<br>
derivative thereof, having substantially similar phosphorylation activity on a human Per<br>
protein as described herein. The proteins hCKI 6 and/or s are proteins having substantially<br>
similar activity of naturally-occurring hCKI S and/or s, alleles and derivatives thereof hCKI 5<br>
and/or z includes other mammalian Casein Kinase I proteins which retain its kinase activity<br>
with respect to hPERl, hPER2, and/or hPER3, or have been modified in such a manner that its<br>
ability to phosphorylate hPERl, hPER2, and/or hPER3 is not essentially altered. Human<br>
forms of hCKI 5 and/or s are preferred. However, use of other mammalian forms of hCKI 5<br>
and/or s would be acceptable because, for example, human hCKI 5 and rat hCKI 5 are 97%<br>
homologous, and their sequences in the kinase domain (284 amino acid residues) were<br>
completely identical. Modified proteins include a truncated forms of hCKI 5 and/or s,<br>
derivatives of hCKJ 5 and/or s containing amino acid substitions, deletions, additions and the<br>
like, which retain the ability to phosphorylate hPERl, hPER2 and/or hPER3. Derivatives of<br>
Casein Kinase I include proteins which contain a leader, epitope or other protein sequence,<br>
such as a Myc^^-tagged, his-tagged, or a Flag ^^epitope tag sequence and have hPERl<br>
phosphorylating activity. Such derivatives faciliate purification or enable attachment to<br>
Sepharose beads or permit easy detection.<br>
Derivatives of hPERl, hPER2 and/or hPER3 include proteins which contain a leader, epitope or other protein sequence, such as a Myc^^-tagged, his-tagged, or a Flag    epitope tag sequence, which retain the ability to be phosphorylated by hCKIs. Such derivatives faciliate purification or enable attachment to Sepharose beads or permit easy detection.   Preferred<br><br>
human Period proteins comprise proteins with one or more hCKIs consensus phosphorylation sequence "DXXS", where D is a glutamic acid residue, X is any amino acid residue, and S is a serine residue. Phosphorylation occurrs at serines fitting the S-Xn-S motif, where n is 1, 2, 3 or 4 and may result in hyperphosphorylation. Phosphorylation preferences for casein kinase I are characterized in Flotow, H. and Roach, P.J., J. Biol. Chem. 266(6):3724-3727(1991).   In a preferred embodiment of the invention, the human Period protein is capable of hyperphosphorylation. Phosphorvlation sites in hPERl occur at between amino acids 743 and 889 of hPERl, preferably between amino acids 800 and 820 of hPERl and most preferably between amino acids 808 and 815 of hPERl, or alternatively for the disruption of the putative CKI interaction domain for human PERI at IQELSEQIHRLLLQPVH, at amino acids 486-503, for human PER2 at IQELTEQIHRLLLQPVH, amino acids 460-477, and/or for human PER3 at ITELQEQIYKLLLQPVH. In one embodiment of the invention, preferred derivatives of hPERl, liPER2 and/or hPER3 comprise a phosphorylation site selected from the group consisting of hPERl amino acids 743 and 889, or for disruption of the putative CKI interaction domain on hPERl at amino acids 486-503 (IQELSEQIHRLLLQPVH,), for human PER2 at amino acids 460-477 (IQELTEQIHRLLLQPVH), and for human PER3 at ITELQEQIYKLLLQPVH.<br>
The term "protein having protein kinase activity" refers to a protein which is evaluated<br>
by one skilled in the art to have protein kinase activity, e.g., a protein which is capable of<br>
phosphorylating one or more human Period protein in a screening system. The screening<br>
system may be the same, or substantially similar, conditions as set forth in any one of<br>
examples below.   However, methods of setting up phosphorylation, degradation, or circadian<br>
rhythm assays, are well known in the art and the present invention is not intended to be<br>
limited to the specific embodiments provided herein.<br>
Proteins to be used in the present invention may be obtained, for example, from human tissue, recombinantly expressed by standard recombinant techniques, and/or optionally chemically modified. Recombinant expression of the proteins is preferred.<br>
"Derivatives" of proteins includes proteins in which an amino group at an amino terminal (N-terminal) or all or a part of amino groups of side chains of amino acids, and/or a<br><br>
carboxyl group at a carboxyl terminal (C-terminal) or all or a part of carboxyl groups of side chains of amino acids, and/or functional groups other than the amino groups and carboxyl groups of the side chains of the amino acids such as hydrogen, a thiol group or an amido group have been modified by appropriate other substituents. The modification by the appropriate other substituents is carried out in order to, for example, protect functional groups in the protein, improve safety or facilitate assaying, such as addition of fianctional groups to attach a protein to a Sepharose bead. .An example is addition of a Flag ^"""epitope tag sequence added to the primers at the 5" end or his-tagged. The derivatives of the proteins include:<br>
(1)	proteins in which one or more hydrogen atoms of the amino group at the amino terminal (N-terminal) or a part or all of the amino groups of the side chains of the amino acids are replaced by substituted or unsubstituted alkyl groups (which may be straight chain or branched chain or cyclic chain) such as a methyl group, an ethyl group, a propyl group, an isopropyl group, an isobutyl group, a butyl group, a t-butyl group, a cyclopropyl group, a cyclohexyl group or a benzyl group, substituted or unsubstituted acyl groups such as a formyl group, an acetyl group, a caproyl group, a cyclohexylcarbonyl group, a benzoyl group, a phthaloyl group, a tosyl group, a nicotinoyl group or a piperidincarbonyl group, urethane-type protective groups such as a p-nitrobenzyloxycarbonyl group, a p-methoxybenzyloxycarbonyl group, a p-biphenylisopropyl-oxycarbonyl group or a t-butoxycarbonyl group, or urea-type substituents such as a methylaminocarbonyl group, a phenylcarbonyl group or a cyclohexylaminocarbonyl group;<br>
(2)	proteins in which the carboxyl groups at the carboxyl terminal (C-terminal) or a part or all of the side chains of the amino acids are esterified (for example, the hydrogen atom(s) are replaced by methyl, ethyl, isopropyl, cyclohexyl, phenyl, benzyl, t-butyl or 4-picolyl), or amidated (for example, unsubstituted amides or CpCe alkylamide such as an methylamide, an<br><br>
ethylamide or an isopropylamide are formed; or<br>
(3) proteins in which a part or all of the functional groups other than the amino groups and the carboxyl groups of the side chains of the amino acids such as hydrogen, a thiol group or an amino group are replaced by the substituents described in (1) or a trityl group.<br>
The term "altering" refers to the abiUty of a test compound to inhibit or enhance phosphorylation of hPERl, hPER2, and/or hPER3 by hCKI 5 and/or s relative to the phosphorylation in the absence of the test compound. Alternatively, "altering" also refers to the ability of a test compound to inhibit or enhance phosphorylation of hPERl, hPER2, and/or hPER3 by hCKI 5 and/or s relative to the phosphorylation of the different compound, such as a standard.   It is preferred that the ability of a compound to inhibit or enhance phosphorylation of hPERl, hPER2, and/or hPER3 is determined with respect to a naturally-occuring form of hCKI 5 and/or s protein.<br>
The term "screening system" refers to a set of conditions suitable to permit phosphorylation of of hPERl, hPER2, and/or hPER3 by hCKI S and/or s. Generally, a screening system contains a ready source of phosphate. A preferred source of phosphate is a ready source of ATP. The screening system may be cell-based or in vitro. Cell-based screening systems include the use of cells which express any or each of hPERl, hPER2, hPER3 and/or hCKI 5 and/or E. A method for screening may be either a cell or a cell-free system. Suitable cell systems include yeast cells, such as S. cerevisia, bacterial cells, such as E. colt, insect cells, such as those used in bacculoviral expression systems, nematode cells, mammalian cells such as COS cells, lymphocytes, fibroblasts (3Y1 cells, NIH/3T3 cells, Rati cells, Balb/3T3 cells, etc.), human embryonic kidney cells, such as 293T cells, CHO cells, blood cells, tumor cells, smooth muscle cells, cardiac muscle cells, brain cells. Preferred cell systems are suprachiasmatic nuclei cells, nerve cells, myelocytes, gliacytes and astrocytes. In a cell based system, if the cell system does not express the human Period protein and/or<br><br>
hCKIe, then the cell must be transfected or transformed to express one or both human Period protein and/or and hCKIs. Alternatively, a cell-free system may be used. Partially purified, or purified hPERl, hPER2, and/or hPER3, and hCKI 5 and/or s may be obtained from recombinant sources which express hPERl, hPER2, and/or hPER3, respectively, and hCKI 5 and/or s, or whereby the underlying base sequence of the original mRNA encoding the protein is modified.<br>
Recombinant expression of a human Period protein and/or hCKI 5 and/or g in a cell may be the result of transfection with one or more suitable expression vectors containing, for example, a promoter and cDNA encoding hPERl, hPER2, hPER3 and/or hCKI 5 and/or £. Cell-based screening systems also include the use of cells in which the human Period protein and/or hCKI 5 and/or s is transuded or transduced into the cell as a fiision protein with a transduction or transducing sequence such as TAT protein obtained from HIV, Antennepedia transduction fragment, or any other means of introducing exogenous protein into a cell.<br>
Preferred in vitro screening systems include aqueous compositions comprising a ready source of phosphate. Preferred in vitro screening systems comprise ATP.<br>
Examples of methods for determining the level of phosphorylation of a human Period protein includes standard methods of detecting the amount of protein phosphorylation, such as use of radiolabeled phosphorous and autoradiography, or indirectly by comparing the amount of radiolabeled phosphorous added and the resulting amount of unbound phosphorous. Alternatively, colormetric or other detection means may be used to determine the level of phosphorylation. Another suitable method for determining the level of phosphorylation of a human Period protein includes a cell-free system using glutathione Sepharose beads where either the human Period protein or hCKIe is bound to a solid support such as to Sepharose<br><br>
beads, and either the hCKIs or human Period protein is added. In addition, numerous alternative methods for determining the amount of human Period protein after are available, and include the use of ^^S-labeled human Period protein degradation, colormetric assays, elution of bound human Period protein and the like.<br>
The screening methods disclosed herein are particularly useful in that they can be automated, which allows for high through-put screening of large number of test compounds, either randomly designed test compounds or rationally-designed test compounds, in order to identify those test compounds that effectively modulate or alter the level of phosphorylation and/or degradation of the human Period protein, and hence alter the circadian rhythm of a mammal.<br>
The term "mammal" refers to human, primate, canine, porcine, bovine and other higher organisms. Humans and primates are more preferred mammals. Humans are most preferred.<br>
Test compounds for use in the present invention include any biological or small molecule chemical compounds, such as a simple or complex organic molecules, peptides, analogues of peptides, proteins, oligonucleotides, compounds obtained from microorganism culture, naturally-occurring or synthetic organic compounds, and/or naturally-occurring or synthetic inorganic compounds.   The choice of test compound to be screened is well within v_J the skill of the art.<br>
The present invention also provides a method for determining the ability of a test compoimd to alter phosphorylation of one or more human Period proteins, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein and one or more human Period proteins selected from the group consisting of hPERl, hPER2 and hPER3, and<br>
(2)	determining the level of phosphorylation of human Period protein.<br><br>
\I0<br><br>
a"^<br><br>
It is also understood that the present includes a method for determining the ability of a test compound to alter phosphorylation of a human "Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein and two or more different hPER proteins selected from the group consisting of hPERl, hPER2 and hPER3, and<br>
(2)	determining the level of phosphorylation of human Period protein. Alternatively, the present invention includes a method for determining the ability of a<br>
test compound to selectively alter phosphorylation of a human Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein<br>
and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
and<br>
(2)	adding a test compound to a screening system comprising hCKI 5 and/or E protein and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3, where the hPER protein selected in (2) is not the hPER protein selected in (1);<br>
(3)	determining the level of phosphorylation of human Period protein in (1) and (2); and<br>
(4)	comparing the results obtained in (3) for each human Period protein to determine if the test compound is selective for ahering phosphorylation of hPERl, hPER2, and/or hPER3.<br>
Alternatively, the present invention includes a method for determining the ability of a test compound to alter degradation of a human Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or e protein and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition of the test compound, and<br><br>
(3) comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br>
Alternatively, the present invention includes a method for determining the ability of a test compound to alter degradation of a human Period protein, comprising:<br>
(1)	adding a test compound and hCKI 5 and/or s protein to a screening system comprising ahPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition of the test compound and hCKI 5 and/or s protein, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br>
Alternatively, the present invention includes a method for determining the ability of a test compound to aher degradation of a human Period protein, comprising:<br>
(1)	adding hCKI 5 and/or z protein to a screening system comprising a test compound and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition hCKI 5 and/or £ protein, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the<br>
amovmt of human Period protein in the screening system.<br>
The present invention includes a method of altering degradation of hPERl by an compound which alters the ability of hCKI 5 and/or e protein to phosphorylate hPERl at a site between amino acids 743 and 889 of hPERl, preferably between amino acids 800 and 820 of hPERl and most preferably between amino acids 808 and 815 of hPERl, or alternatively for disniption of the putative CKI interaction domain of human PERI at IQELSEQIHRLLLQPVH, at amino acids 486-503, for human PER2 at<br><br>
IQELTEQIHRLLLQPVH, amino acids 460-477, and/or for human PER3 at ITELQEQIYKLLLQPVH.<br>
As described below, phosphorylated hPERl protein is rapidly degraded, therefore the<br>
screening method according to the present invention can be used to identify test compounds<br>
that selectively activate or inhibit degradation of hPERl. Since phosphorylated hPER2 and<br>
hPER3 protein is rapidly degraded, the present method can be used to identify test compounds<br>
that selectively activate or inhibit degradation of hPER2 or hPER3, respectively. The present<br>
method also provides for a method of determining compounds that selectively activate or<br>
inhibit phosphorylation hPERl, hPER2 and/or hPERS, by determining the effect of that<br>
compound on activation or inhibition of phosphorylation any of hPERI, hPER2 and/or<br>
hPER3, and comparing the results obtained with the same, or a different, test compound.<br>
Also, as phosphorylated hPERl protein is rapidly degraded, the present method of the invention can be used to identify test compounds that selectively increase or reduce the level of a human Period protein in a cell relative to the level of the same or a different human Period protein in the absence of the test compound. In one preferred embodiment of the invention, the method is used to identify test compounds that selectively increase or reduce the level of a hPERl in a cell relative to the level hPERl in the absence of the test compound. In an alternative preferred embodiment of the invention, the method is used to identify test compounds that selectively increase or reduce the level of a hPER2 in a cell relative to the level hPER2 in the absence of the test compound.<br>
In addition, the present invention includes a method to identify test compounds that selectively inhibit the amount of hPERl degradation in a cell relative to the amount of hPER2 degradation in the presence of the test compound. In an alternative embodiment of the invention, the method is used to to identify test compounds that selectively inhibit the amount<br><br>
of hPER2 degradation in a cell relative to the amount of hPERl degradation in the presence of the test compound.<br>
Alternatively, the present invention can be used to used to identify test compoimds that selectively increase or reduce the level of hPER2 and/or hPER3 in a cell relative to the level of hPER2 and/or hPER3, respectively, in the absence of the test compound, or alternatively, relative to the level of a different human Period protein. The present method can be used to identify compounds that selectively increase or reduce the levels of hPERl, liPER2 and/or hPER3 in cell relative to its native level. Comparision of the results of different test compounds on the level of human Period protein may also be after a biological or chemical treatment, such as addition, inhibition, or alteration of endogenous and/or exogenous stimuli, such as light, growth factors, transcription factors, and the like.<br>
Phosphorylated hPER proteins are known to be closely involved in the regulation of the circadian cycle of mammals. Therefore, the present invention can be used to used to identify test compounds that affect, modulate or otherwise change the physiological response of the circadian cycle of a mammal in the absence of a test compound or stimuU. Modulation of circadian cycle of a mammal includes the prevention of the alteration of the normal circadian cycle of a mammal in response to a stimuli in the absence of the test compoimd. Thus, the present invention includes methods of identifying test compounds capable of preventing alteration of circadian rhythms of mammals in response to stimuH that normally alter the circadian rhythms of a mammal.<br>
The following Examples which demonstrate the effect of human casein kinase I s (hCKIs) on phosphorylation of human Period 1 may be modified to substitute human Period 2 and/dr human Period 3. Similar results are obtained with human casein kinase I hCKJ 5.<br>
Purified recombinant hCKIs, but not a kinase negative mutant of hCKIs (hCKIs-K38R), phosphorylates hPERl in vitro. When co-transfected with wild-type hCKlE in 293T<br><br>
cells, hPERl shows a significant increase in phosphorylation as evidenced by a shift in molecular mass. hPERl protein could also be co-immundprecipitated with transfected hCKIs as well as endogenous hCKIs, indicating physical association between hPERl and hCKIs proteins in vivo. Furthermore, phosphorylation of hPERl by hCKIs causes a decrease in protein stability in hPERl. Unphosphorylated hPERl remains stable in the cell throughout a 24-hr cycle, whereas phosphorylated hPERl has a half-life of approximately 12 hr. Using various hPERl truncation mutants, potential phosphorylation sites in hPERl are amino acids 743 through 889, which contain a CKI consensus phosphorylation site.<br>
To investigate whether hCKIs, the mammalian homologue of Drosophila DBT could phosphorylated hPERl, recombinant his-tagged wild-type hCKIs is expressed in from E. coli, purified and assayed for its ability to phosphorylate a pair of known substrates, casein and GST-DcBa, as well as hPERl. Recombinant hCKIs phosphorylates both casein and DeBa substrates (Fig. lA, lanes 2 and 3). Purified wild-type hCKIe autophosphorylates. The ability to autophosphorylate indicates hCKIs activity (Fig. 1 A, lanes 1 and 2). Phosphorylation is not observed when recombinant hCKIs is absent (Fig. 1 A, lanes 7 and 8).<br>
A kinase negative mutant of hCKIs-K38R, in which lysine 38 in the ATP binding domain is mutated to an arginine, is assayed for phosphorylation of both casein and IkBa substrates. hCKIe-K38R does not have autophosphorylating activity and does not phosphorylate either casein or GST-IkBa substrates (Fig. 1 A, lanes 4-6). This demonstrates that the previous phosphorylating activity is specific to wild-type hCKIs.<br>
Recombinant hCKIe is also shown to phosphorylate hPERl in vitro. As shown in Fig. IB, no phosphorylation is observed in the absence of recombinant hCKIs (lanes 8-10). The presence of hCKIs results in phosphorylation of hPERl, but not Flag-tagged luciferase in vitro (lanes 2 and 3). Phosphorylation of hPERl is not due to hPERl associated kinase<br><br>
activity as hCKIs also phosphorylated heat inactivated hPERl immunoprecipitates (lanes 6). Furthermore, hCKl£-K38R has no kinase activity towards hPERl (lane 7). Therefore, hCKIs directly phosphorylates hPERl in vitro.<br>
hCKIs specifically phosphorylates hPERl in 293T cells co-transfected with flag-tagged hPERl and either vector control, wild-type hCKIe or hCKIs-K38R. Cells are lysed 24 hr after transfection and the lysates separated on a 3-8% SDS NU-PAGE followed by Western blot analysis. Figure 2A shows that in cells co-transfected with wild-type hCKIs and hPERl, a significant shift in molecular mass of the hPERl protein is observed as compared to cells co-transfected with either vector control or hCKl£-K38R (lanes 1-3). Similar shifts in hPERl molecular mass are always observed in several co-transfection experiments using different percentages of SDS-PAGE. Western blot analysis showed that both wide type hCKIs and hCKIs-K38R proteins are expressed at equivalent levels (Fig. 2B, lanes 2 - 5).<br>
Co-transfected 293T cells with Flag-tagged hPERl and either vector control, hCKIs or the kinase-negative mutant of hCKlE-K38R and radiolabeled with [^^S]methionine and cysteine demonstrates the cause for the change in hPERl molecular mass after phosphorylation. ""^S-labeled hPERl is immunoprecipitated and either treated or not treated with purified recombinant lambda phosphatase. As shown in Fig. 2C, immunoprecipitated ^^S-radiolabeled hPERl shows a shift in molecular mass when cells are co-transfected with wild-type hCKIs, but not with vector or kinase-negative hCKIe-K38R controls (lanes 1,2, and 3). The shift in molecular mass of the protein from co-transfected hPERl and wild-type hCKIe cells is significantly reduced after 1 hr treatment with lambda phosphatase. This demonstrates that the shift in mobility of hPERl is due to phosphorylation (Fig. 2C, lanes 2 and 5). After 1 hr treatment with lambda phosphatase, the mobility of all hPERl from hCKIs<br><br>
co-transfected cells verses vector control and kinase negative co-transfected cells are essentially indistinguishable from each other (Fig. 2C, lanes 4, 5 and 6).<br>
The lambda phosphatase mobility shift is not due to contaminating proteases. Addition of 50mM sodium fluoride (a phosphatase inhibitor) to the lambda phosphatase reaction blocked the reduction of the mobility shift of hPERl. No other higher molecular mass forms of hPERl are present in the immunoprecipitates; indicating that the post-translational mobility shift of hPERl is due to phosphorylation.<br>
During the circadian cycle, PER protein accumulates and this accumulation leads to its subsequent degradation (Edery, I., et al. (1994) Proc. Natl. Acad. Sci. USA 91, 2260-2264, Dembinska, M.E., et al. (1997) J. Biol. Rhythms 12, 157-172). During the phase when PER protein accumulates, there is a significant shift in molecular mass that might be due to the phosphorylation of the protein. The mobility shift reaches its maximum just before PER disappears (Edery, I., et al. (1994) supra). Co-transfected 293T cells with expression plasmids encoding both hPERl and wild-type hCKIs, or vector control are used to demonstrate that phosphorylation of hPERl results in its instability in cells. Approximately 20 hrpost-transfection, the cells are pulse labeled for 30 min with [^^S]methionine/cysteine and then chased for 0-30 hr. After the appropriate times, hPERl is harvested, immunoprecipitated and analyzed by SDS-PAGE. As shown in Fig. 3A, cells co-transfected with hPERl and vector alone showed very little shift in mobility throughout the time course (lanes 1-7). After 12 hr there appeared to be a slight shift in molecular mass as indicated by a smearing of the protein which increased slightly at the 30 hr time point (lanes 1, 5, and 7). The amount of hPERl present in the control cell remains relatively constant throughout the time courses. At 2 hr post-radiolabeling, approximately 50% hPERl protein is still present in the cell and this level remains constant throughout the time course (Fig. 3C, solid bars).   In contrast to the vector control, cells co-transfected with hPERl and wt hCKIs showed a shift in mobility as soon as 2<br><br>
hr post-radiolabeling (Fig. 3B, lanes 1 and 2). This shift in molecular mass continued to become more pronounced throughout the time course with the maximum shift occurring between 24 and 30 hr (Fig. 3B, lanes 2-7). In contrast to the vector control, hPERl from 293T cells co-transfected with hCKIs showed a decrease in protein stability. Similar to vector control, at the 2-hr time point 50% of total hPERl from CKIs co-transfected cells is present in the cell. (Fig. 3B, lane 2, and Fig. 3C, crossed-hatched bars). Unlike vector control, only one-half of phosphorylated hPERl remained in the cell after 12 hr. At 24 hr approximately 14% of phosphorylated hPERl is present (Fig. SB, lanes 5 and 6 and Fig 3C, crossed-hatched bars). This experiment is repeated three times with similar results. Phosphorylation of hPERl by wild-type hCKIs results in decreased protein stability, and subsequently, in its degradation.<br>
hPERl and hCKIe physically interact in 293T Cells. 293T cells are co-transfected with Flag-tagged hPERl and either vector alone or HA-tagged hCKIs. Transfected 293T cells are lysed and hPERl immunoprecipitated with anti-Flag mAb and then immunoblotted with anti-HA mAb. Alternatively, hCKIs is immunoprecipitated with anti-HA mAb and then immunoblotted with anti-Flag mAb. Figures 4A and 4B demonstrate that recombinant hCKIs co-precipitated with hPERl, indicating that hCKIs directly associates with hPERl. 293T cells are transfected with hPERl only in order to demonstrate that hPERl. Cells are lysed and hPERl immunoprecipitated with anti-Flag mAb and then immunoblotted with anti-hCKIs mAb. Alternatively, endogenous hCKJs are immunoprecipitated with anti-hCKIs mAb and then immunoblotted with anti-Flag mAb. Endogenous hCKIs co-precipitated with hPERl indicating a physical association between the two proteins (Fig. 4C and 4D).<br>
hCKIs phosphorylates hPERl between amino acids 621 and 889. Fig. 2A shows that the shift in molecular mass of hPERl is due to phosphorylation by hCKIs. hi order to identify the phosphorylation site(s) of hPERl phosphorylated by hCKIs, truncated versions of hPERl<br><br>
are prepared as described in Fig. 5A and Materials and Methods, below. These constructs are transfected into 293T cells along with either vector, hCKIs, or hCKlE-K38R, and assayed for a shift in molecular mass. As shown in Fig. 5B, lanes 2, cells co-transfected with both hCKIe and either full length open reading frame hPERl (ORF), N2, N3 or N4 showed a shift in molecular mass of the protein. Lambda phosphatase treatment of truncated hPERl protein results in a disappearance of the shift, which is due to phosphorylation by hCKIs hCKIs co-transfected with Nl or C-terminal constructs CI, C2, C5 or C6 did not show a shift in molecular mass of the proteins (Fig. 5B and 5C, lanes 2). Truncated constructs that showed a shift in molecular mass (ORF, N2, N3, and N4) share a region of homology from amino acids 621 through 889 (see Fig. 5A). Because CI, which contains amino acids 584-743, did not show a shift, hCKIs is phosphorylated hPERl between amino acids 743 and 889. Several CKI phosphorylation consensus sequences are located throughout hPERl, including one within the region of hPERl showed above to be phosphorylated by hCKIs, specifically je sequence encompassing amino acids 808 - 815: DSSSTAPS. All the serines and the threonine could serve as substrate for hCKIe, which might account for the dramatic mobility shift that observed. MATERIALS AND METHODS Example 1 Plasmid Construction, Expression, and Purification of Proteins.<br>
The cDNA encoding wild-type hCKIs is isolated from a human placental cDNA library using previously described methods, Fish, K.J., et al., (1995) J. Biol. Chem. 270, 14875-14883. Cloning of hCKIe into bacterial vector (pRST-B-CKIe) and mammaHan expression vector (pCEP4-CKl£) is as described previously (Cegielska, A., et al., (1998)J. BioT. Chem. 273,1357-1364, Rivers, A., et al., (1998) J. Biol. Chem. 273,15980-15984). A hemagglutinin (HA) epitope tag (YPDYDVPDYA) is added at the 5"end of hCKIe in pCEP4-<br><br>
CKIs. Full-length hPERl (Tel, H., et al., (1997) Nature 389, 512-516) is cut with £co/2/and Sail and ligated into plasmid vector pCMV-Tag"™ (Stratagene) to create an in-frame frision with the Flag tag. Truncated N-terminal mutants (Nl, N2, N3, N4, C5) are generated by cutting PERI vvith EcoRI/EcoRV, EcoRI/XhoI. EcoRV/XhoI, PvuII/XhoI, orBamHI/Sall. respectively, and ligation into the same vector. To construct mutants CI, C2 and C6, oligonucleotide primers are used in PCR reactions to amplify DNA fragments encoding amino acids 584 to 743, 998 to 1160, or 1161 to 1289, respectively, using hPERl cDNA as template. The resulting fragments are summarized in Table 1.<br>
Table 1 Full Length and Truncated forms of hPERl<br><br>
Designation	First Amino Acid	Last Amino Acid<br>
RF	1	1289<br>
Nl	1	485<br>
N2	1	889<br>
N3	486	889<br>
N4	621	889<br>
CI	584	743<br>
C2	998	1160<br>
C5	1127	1289<br>
C6	1161	1289<br>
A Flag ^^epitope tag sequence is added to the primers at the 5" end. The PCR products are cloned into the mammalian expression vector pcDNA3 Topo vector"™ (Invitrogen). Bacterially expressed histidine-tagged hCKIs and hCKlE-K38R are expressed and purified as described in Cegielska, A., et al., (1998) J. Biol. Chem. 213,1357-1364. The c-MYC tagged CKI § was provided as a gift from Dr. David Virshup. A protein of greater than 90% homogeneity and with an approximate molecular weight of 54 kDa is purified.<br>
Example 2 Transfection and Radiolabeling of 293T Cells.<br>
Human embryonic kidney cells 293T are grown in 6 well plate in DMEM supplemented with 10% fetal bovine calf serum (Hyclone). Cells are transfected at a density<br><br>
of approximately 80%, with 2 |ig of DNA, using the lipofectAMINE^M reagent (Life Technologies) according to the manufacturers instructions. Transient transfection efficiencies of 293T cells are typically 30-50% as monitored by GFP control plasmid transfection.<br>
293T cells are radiolabeled 16 hr post-transfection with 0.5 )j.Ci/ml [^^Sjmethionine/cysteine for 30 min in methionine and cysteine deficient media. Thereafter, cells are ished and cultured in regular DMEM for the time indicated. Cells are lysed using lysis buffer (20 mM tris, 1% Triton X^lOO™, 0.5% Igepal ™, 150 mM NaCl, 20 mM NaF, 0.2 mM Na2V04, 1 mM EDTA, 1 mM EGTA, Complete protease inhibitor cocktail [Boeringer Mannheim], pH 7.5). Lysates are cleared of cellular debris by centrifugation at 12,000 rpm. Supematants are collected and stored at -70°C until use.<br>
Example 3 Immunoprecipitations and Western Blot Analysis.<br>
Lysates containing equal amounts of protein (100 |ig total) are mixed with 5 \JLI of a 1 ;500 dilution either of anti-Flag, anti-HA, or anti-hCKIs mAb and incubated overnight at 4°C. Aiter incubation with the antibody, 30 |il of a 1:1 slurry of G-protein sepharose beads is added and incubated for an additional 2-4 hr. The beads are washed five times in lysis buffer and subsequently resuspended in 30 ^1 of SDS sample buffer with 5 mM DTT, boiled, and analyzed by SDS-PAGE. Western blotting of proteins is performed on either supematants or immunoprecipitated proteins from transfected 293T cells using either anti-Flag""*^ (Sigma) at a 1:1000 dilution, anti-HA (Invitrogen) at 1:1000 dilution, or anti-hCKIs (Transduction Laboratories) at a 1:750 dilution as described previously (Yao, Z., et al., (1997) J. Biol. Chem. 272, 32378-32383).<br>
Example 4 Kinase and Phosphatase Assays.<br>
hCKIs is assayed for activity using either casein or GST-DcBa as substrate. Casein or GST-DcBa (0.5 ^ig) is combined with hCKIs (0.1 jig) and 5 ^Ci [y-^^PjATP (Amersham) in<br><br>
PBS containing 200 mM ATP, 10 mM MgCb, 0.6 mM EGTA, and 0.25 mM DTT. Reactions<br>
are incubated for 30 minutes at room temperature, stopped by the addition of SDS sample<br>
buffer, and then analyzed by SDS-PAGE. Because GST-EcBa migrated at a similar position<br>
in SDS-PAGE as hCKIs, a sUght modification of the protocol was performed. After the 30<br>
min incubation, GST-DcBa is removed from the kinase reaction by the addition of<br>
Glutathione-sepharose beads. The beads are washed five times in lysis buffer to remove any<br>
contaminating hCKIs before the addition of SDS sample buffer. Gels are stained with<br>
Coomassie blue R-250, dried, and autoradiographed.<br>
Immunoprecipitation of    S-Iabeled hPERl is as described above. Beads containing<br>
hPERl are washed three additional times in phosphatase buffer (100 mM MES, 0.5 mM<br>
dithiothreitol DTT, 0.2 mM phenyhnethylsulfonyl fluoride, 20 |ig/ ml aprotinin, 10 |ig/ ml<br>
leupeptin, 10 |ig/ ml pepstatin A, pH 6.0) and resuspended in 20 p-l of phosphatase buffer.<br>
The phosphatase treatment is initiated by the addition of a 1 Ox solution of reaction (50 mM<br>
Tris-HCL, 0.1 mM EDTA, 5 mM DTT 0.01% Brij 35, 2 mM MnCb pH 7.0), and 40 units of<br>
purified lambda phosphatase. The reaction is allowed to proceed for 1 hr at 37°C. Inhibition<br>
of phosphatase activity is achieved by the addition of 50 mM sodium fluoride. After the<br>
appropriate incubations, the reaction is stopped by the addition of SDS sample buffer. The<br>
proteins are separated using SDS-PAGE; the gel is dried and autoradiographed. The image is<br>
visualized using a Molecular Dynamics Phospholraager""^"^.<br>
Example 5 hCKIe interaction with and phosphorylation of tiuman PERI<br>
The following Examples demonstrate the Materials and Methods used to demonstrate that hCKIe interacts with and phosphorylates human PER2. To summarize these result, when co-transfected with hCKIs in 293T cells, hPER2 shows a significant increase in phosphorylation state as evidenced by "^"P incorporation, as well as a shift in molecular mass. Furthermore, like hCKIs and<br><br>
hPERl, hCKIs co-immunoprecipitates with transfected hPER2. Treatment of transfected cells with the hCKIs inhibitor, CKI-7, resuhs in a decrease of hPERl and hPER2 phosphorylation. Pulse/chase studies reveal that increased<br>
*<br>
phosphorylation of hPER2 by transfected hCKIs caused hPER2 to be degraded. These data indicate a physical association between hCKIs and the human period proteins in vivo between CKI and human PERI at IQELSEQIHRLLLQPVH, at amino acids 486-503, for human PER2 at IQELTEQIHRLLLQPVH, amino acids 460-477, and/or presumably for human PER3 at ITELQEQIYKLLLQPVH, and a regulation of period stability through hPERl and hPER2 phosphorylation. Materials and Methods for hPerl<br>
Example 6 Plasmid Construction, Expression, and Purification of Proteins.<br>
Full length open reading frame (ORF) human period 2 is cloned by PCR from a human brain cDNA library from Clonetech using forward primer ATCTAGATCTAGAATGAATGGATACGCGGAATTTCCG and reverse primer TCTGCTCGAGTCAAGGGGGATCCATTTTCGTCTT. The ORF encodes a 1246 amino acid protein. The DNA is subcloned into the pYGFP living color vector (Clonetech) creating a hPER2-C-terminal YGFP protein. Bacterially expressed histidine-tagged hCKIe are expressed and purified as described above. A protein of greater than 90% homogeneity and with an approximate molecular weight of 54 kDa was purified.<br>
Example 7 Transfection of 293T Cells.<br>
Transfections of human embryonic kidney cells 293T is done using the methods and materials described above, with the substitution of human Period 2 DNA for that of human Period 1 DNA. Lysates and supematants are collected and stored as described above.<br>
Example 8 Immunoprecipitations and Western Blot Analysis. Lysates are used for immunopH-ecipitation and Western Blot analysis using the methods and materials described above, with the substitution of hPER2 lysate for that of hPERl lysate. The results are shown below in Table 2. After co-transfection of hPER2 and hCKIs in 293T cells, cells are lysed 24 h after transfection, immunoprecipitated, and lysates separated on 8% SDS-PAGE followed by Western blot analysis. As shovm in Table 2, immunoprecipitation of HA- hCKIs followed by Western blot analysis shows that hCKIs interacts with<br><br>
hPER2 as well as hPERl. Positive denotes interaction, a negative denotes no interaction.<br>
Example 9 hCKIe associates with and phosphorylates hPER2<br>
The consequence of PERI phosphorylation is instability and degradation of the protein. Therefore, in order to determine if hCKIs phosphorylates PER2, 293 cells were co-transfected with CKIe and hPER2 or hPERl as control and proteins were visualized by Western blot analysis. As shown in Table 2 in cells co-transfected with hCKIs and hPER2, a shift in molecular mass of the protein is observed, which is similar to the results seen with hCKIs and hPERl.<br>
Table 2<br><br>
	hCKIe al one	hPER2 alone	hCKIs and hPER2	hCKlE (K38A) andhPER2<br>
Interaction with CKIs	-	-	+	+<br>
Shift in hPER2 migration	not<br>
determine<br>
d		+	<br>
To determine if the shift in mobility was due to phosphorylation of hPER2, we performed p32 labeling experiments and assayed for incorporation of p32 label into hPER2. As shown in Table 3, co-transfection of hPER2 or hPERl with CKle resulted in the incorporation of p32 into both PER proteins. The amount of p32 incorporation appeared to be greater in hPERl than hPER2. This difference in hPERl phosphorylation verses hPER2 could be due to an enhanced kinetic rate of phosphorylation of hPERl by CKI verses hPER2. Another explanation is that hPERl has a greater nimiber of CKI consensus phosphorylation sites than hPER2, (9 on hPERl verses 7 on hPER2).<br>
Table 3<br><br>
	hCKIe alo ne	hPER2 alone	hCKIsa<br>
nd<br>
hPER2	hCKIs (K38 A) and hPER2<br>
P32 incorporation into hPER2 in cpm"	1000	750	5000	3000<br>
Phosphorylation ofhPERl leads to protein instability: Phosphorylation of hPERl results in protein instability and degradation.   Since hPER2 is similarly<br><br>
phosphorylated by hCKIs , in order to determine the effect of phosphorylation on hPER2 protein stability, HEK 293 cells are transfected with cDNA encoding either PER2 alone or PERI alone, or co-transfected with cDNAs encoding both hCKIs and PER2, or hCKIe and PERI. Cells are pulsed with 35-S Methionine and immunoprecipitated at the times summarized in Table 4, for 32 hours. As shown in Table 4, single transfection of either hPER2 and hPERl resuhs in either hPERl or hPER2 being phosphorylated by endogenous kinase and degraded. The half-lives of each protein is approximately 14 hours for hPERl and 4 hours for hPER2. However, co-transfection of either hPERl or hPER2 with hCKIe results in a hyperphosphorylation of both proteins. Furthermore, this hyperphosphorylation results in a slight shift and shortening of the protein half-lives of approximately 2-4 hours. hPER2 appears not to be any more stable than hPERl after phosphorylation with hCKIs even though it appears to be phosphorylated to a lesser degree than hPERl.<br>
Table 4<br><br>
Time in hours	hPERl alone*	hPER2 alone*	hCKls and hPERl*	hCKIe and hPER2*<br>
0	100	100	100	100<br>
4	59	69	33	44<br>
6	41	37	24	28<br>
8	48	29	28	26<br>
14	43	19	22	10<br>
18	32	11	15	10<br>
24	16	7	8	6<br>
32	13	7	2	7<br>
* S-35 labeled hPERl or hPER2 in cpm over time to determine the half-life of the protein. Example 10 Assay to for hCKI 6 and £ Inhibitors<br>
The following assay is used to test compound for their ability to alter phosphorylation of hPERl, thereby increasing hPERl levels in co-transfected cells and alter rat PERI cellular mRNA oscillation. Using a hCKls-Perl co-transfection (transient) assay, HEK293T cells are grown in six well plates to about 80% confluence and then co-transfected with hCKls and Perl or Per2 using Lipofectamine plus reagent (Gibco BRL). After 16 hours, the transfection medium is removed and the cells are dosed with I, 10, or 30 uM CKI inhibitory and non-actfve analogue compounds for 16hr. After an additional 16 hours, the medium is removed and the cells are washed two times with PBS, lysed, centrifuged and supematents are run on 8-16% or 8% tris glycine gels. Western blots are performed for Flag-tagged hPERl or GFP-<br><br>
tagged hPer2. The presence of hCKls is detected in each sample by Western blotting with anti-HA antibodies.<br>
As shown in Fig 9 cells co-transfected with hCKls and hPERl and exposed to an increasing concentrations of a control test compound, for example, one that does not inhibit hCKls, no increase in hPERI levels are observed. However, co-transfected cells treated with hCKls inhibitors show a relative increase in hPERl levels in a dose dependent manner. This increase in hPERl levels is due to an inhibition of CKI phosphorylation activity and a relative decrease in hPERl phosphorylation followed by an increase in protein stability. If CKI inhibitors alter PERI protein stability-and half-life, it can be reasoned that increasing the cellular PERI levels will have some effect on the circadian oscillation or cellular cycle.<br>
To test the effect of CKI inhibitors in altering rat PERI oscillation by quantitative PCR using TaqMan RT-PCR (Perkin Elmer Biosystems), rat-1 fibriblasts are grown in Dulbecco"s modified Eagle medium supplimented with 5% fetal calf serum and a mixture of penicillin-streptomycin-glutatamine. SCN cells are grown in Dulbecco"s Minimum Eagle medium supplimented with 10% fetal calf serum penicillin-streptomycin-glutamine and 2% glucose. Approximately 5x10^ cells are plated in 10 cm petri dishes 3-5 days prior to the experiment. Once the plates are confluent, which is designated as time =0, the medium is exchanged with serum rich medium, i.e. serum containing 50% horse serum. After 2 hours of the serum shock in 50% horse serum, this medium is replaced with with serum-free medium. At indicated times, the dishes are washed with PBS and kept frozen at -80°C until the extraction of whole cell mRNA. hCKls inhibitor or control is added at the time when serum-free medium is added.<br>
Whole cell mRNA is extracted by means of RNeasy Midi kit or Rneasy 96 kit (Qiagen) and Dnase treated (Ambion DNA-free). Quantitative PCR is performed with real¬time Taq-Man technology (PE Biosystems) [ C.A. Heid et al.. Genomes Res. 6, M (1996)] and analysed on an ABI PRISM 7700 (T. Takumi et al.. Genes Cells 4, 67: 1999).   The primers for rPerl are as follows: Forward 5"-TCTGGTTAAGGCTGCTGACAAG-3"; Reverse, 5"-GTGTAGCCCCAACCCTGTGA-3", and the TaqMan, probe 5"-TCCAAATCCCAGCTGAGCCCGA-3". As an internal confrol for the RNA, expression of rActin is examined under the same conditions. Ratios of rPerl to rActin were calculated and normalized.<br><br>
As shown in Fig 10, cells treated with no compound or a test compound that is an inactive hCKls small molecule analog show a normal circadian cycle of approximately 24 hr as indicated by PERI mRNA oscillation. However, cells treated with a test compound that is a CKI inhibitor show an altered daily oscillatory circadian cycle Fig 11. PERI mRNA levels in these cells demonstrate a shorten rhythm of about 18 to 20 hours instead of the nomial 24 hour cycle. The shortened cycle is the is due to CKI inhibition of phosphorylation, resulting in lower levels of PER phosphorylation. Less phosphorylated PER leads to increased PER protein stablilty and increased cellular levels of PER, which alters the circadian rhythm of a mammal.<br>
All references cited above are hereby incorporated by reference into this specification.<br>
The above examples are not intended to be limiting and are merely illustrative of the<br>
specific embodiments of the invention.<br><br><br>
WE CLAIM <br>
1.	A method for determining the ability of a test compound as herein described to<br>
alter phosphorylation of at least one human Period protein, comprising:<br>
(1)	adding said test compound to a screening system comprising hCKI 5 and/or 8 protein and atleast one human Period proteins selected from the group consisting of hPERl, hPER2 and hPER3, and subsequently,<br>
(2)	determining the level of phosphorylation of human Period protein in a known, manner.<br><br>
2.	The method as claimed in claim 1, wherein said test compound alters phosphorylation of human Period protein by hCKI 8.<br>
3.	The method as claimed in claim 1, wherein said test compound enhances phosphorylation of the human Period protein by hCKI 6.<br>
4.	The method as claimed in claim 1, wherein the screening system is a cell system or a cell-free system.<br>
5.	The method as claimed in claim 4, wherein the screening system is a cell-free system.<br>
6.	The method as claimed in claim 5, wherein the cell-free system uses partially purified or purified human Period protein, hCKI 8 or hCKI 8.<br>
7.	The method as claimed in claim 6, wherein the human Period protein and hCKI 8 are obtained from recombinant sources.<br>
8.	The method as claimed in claim 4, wherein the screening system is a cell-based system.<br><br>
9.	The method as claimed in claim 8, wherein the cell-based system is a prokaryotic cell.<br>
10.	The method as claimed in claim 9, wherein the prokaryotic cell is a bacterial cell.<br>
11.	The method as claimed in claim 4, wherein the cell-based system is a eukaryotic cell.<br>
12.	The method as claimed in claim 11, wherein the eukaryotic cell is a yeast cell.<br><br><br>
14.	The method as claimed in claim 11, wherein the cell-based system is an insect cell.<br>
15.	The method as claimed in claim 11, wherein the cell-based system is a mammalian cell.<br>
16.	The method as claimed in claim 15, wherein the mammalian cell is a human cell.<br>
17.	The method as claimed in claim 15, wherein the mammalian cell is a lymphocyte cell, fibroblast cell, tumor cell, smooth muscle cell, cardiac muscle cell, embryonic kidney cell, brain cell, nerve cell, myelocyte cell, gliacyte cell or astrocyte cell.<br><br>
18. A method for determining the ability of a test compound as herein described to selectively alter phosphorylation of a human Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3, and<br>
(2)	adding a test compound to a screening system comprising hCKI 5 and/or e protein and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3, where the hPER protein selected in (2) is not the hPER protein selected in (1);<br>
(3)(;detennining)the level of phosphorylation of human Period protein in (1) and (2);<br>
and<br>
(4) comparing the results obtained in (3) for each human Period protein to detentions if<br>
the test compound is selective for altering phosphorylation of hPERl, hPER2, and/or<br>
hPER3.<br>
19.	The   method   as   claimed   in   claim   18,   wherein   the   compound   alters phosphorylation of human Period protein by hCKI s.<br>
20.	The  method  as  claimed  in  claim   18,  wherein the  compound  enhances phosphorylation of the human Period protein by hCKI 6.<br>
21.	The method as claimed in claim 18, wherein the screening system is a cell-based system or a cell-free system.<br>
22.	The method as claimed in claim 21, wherein the screening system is a cell-free system.<br>
23.	The method as claimed in claim 22, wherein the cell-free system uses partially purified or purified human Period protein, hCKI 5 or hCKI s.<br><br>
24.	The method as claimed in claim 23, wherein the human Period protein and hCKI 8 are obtained from recombinant sources.<br>
25.	The method as claimed in claim 21, wherein the screening system is a cell-based system.<br>
26.	The method as claimed in claim 25, wherein the cell-based system is a prokaryotic cell.<br>
27.	The method as claimed in claim 26, wherein the prokaryotic cell is a bacterial cell.<br>
28.	The method as claimed in claim 21, wherein the cell-based system is a eukaryotic cell.<br>
29.	The method as claimed in claim 28, wherein the eukaryotic cell is a yeast cell.<br>
30.	The method as claimed in claim 29, wherein the yeast cell is S. crevice.<br>
31.	The method as claimed in claim 21, wherein the cell-based system is an insect cell.<br>
32.	The method as claimed in claim 21, wherein the cell-based system is a mammalian cell.<br>
33.	The method as claimed in claim 32, wherein the mammalian cell is a human cell.<br>
34.	The method as claimed in claim 32, wherein the mammalian cell is a lymphocyte cell, fibroblast cell, tumor cell, smooth muscle cell, cardiac muscle cell, embryonic kidney cell, brain cell, nerve cell, myelocyte cell, gliacyte cell or astrocyte cell.<br><br>
35.	A method for determining the ability of a test compound as herein described to<br>
alter degradation of a human Period protein, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or e protein<br>
and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition of the test compound, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br>
36.	A method for determining the ability of a test compound as herein described to<br>
alter degradation of a human Period protein, comprising:<br>
(1)	adding a test compound and hCKI 5 and/or e protein to a screening system comprising a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition of the test compound and hCKI 5 and/or 8 protein, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br>
37.	A method for determining the ability of a test compound as herein described to<br>
alter degradation of a human Period protein, comprising:<br>
(1)	adding hCKI 8 and/or 8 protein to a screening system comprising a test compound and a hPER protein selected from the group consisting of hPERl, hPER2 and hPER3,<br>
(2)	determining the amount of human Period protein after addition hCKI 8 and/or s protein, and<br>
(3)	comparing the amount of human Period protein obtained in step (2) with the amount of human Period protein in the screening system.<br><br>
38.	The method as claimed in claim 1, wherein the human Period protein is human Period 1.<br>
39.	The method as claimed in claim 1, wherein the human Period protein is human Period 2.<br>
40.	The method as claimed in claim 1, wherein the human Period protein is human<br>
Period 3.<br>
41.	A method for determining the ability of a test compound as herein described to<br>
alter the circadian rhythm of a mammal, comprising:<br>
(1)	adding a test compound to a screening system comprising hCKI 5 and/or s protein and one or more human Period proteins selected from the group consisting of hPERI, hPER2 and hPER3, and<br>
(2)	determining the effect on the circadian rhythm of the mammal relative to the circadian rhythm of the mammal in the absence of the test compound.<br>
42.	A method for determining the ability of a test compound to alter<br>
phosphorylation of atleast one human period proteins such as herein described and<br>
exemplified with reference to accompanying drawings.<br>
43.	A method for determining the ability of a test compound to alter degradation of<br>
a human period proteins such as herein described and exemplified with reference to accompanying drawings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS1kdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che description (complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgcGN0IHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTY3MC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1670-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="215962-device-for-in-vivo-determining-blood-viscosity.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="215964-a-process-for-elimination-of-sodium-oxalate-in-an-industrial-facility-implementing-the-bayer-alumina-hydrate-production-process-and-using-industrial-water.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>215963</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1670/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Nov-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVENTIS PHARMACEUTICALS INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Route 202-206 P.O. Box 6800, Bridgewater, NJ 08807,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KEESLER, George</td>
											<td>99 Perrine Pike Somerville, NJ 08876,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MONDADORI, Cesare</td>
											<td>44 Hartly Lane, Basking Ridge, NJ 07920,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YAO, Zhengbin</td>
											<td>2 Delaware Road, Whitehouse Station, NJ 08889,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CAMACHO, Fernando</td>
											<td>308 Wayne Terrace, Union, NJ 07083,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/15633</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-06-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/327,745</td>
									<td>1999-06-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/215963-a-method-for-determining-the-ability-of-a-test-compound-to-alter-phosphorylation-and-degradation-of-human-period-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:07:40 GMT -->
</html>
