<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/215958-a-method-of-enhancing-the-rate-of-novel-gene-discovery by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:07:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 215958:A METHOD OF ENHANCING THE RATE OF NOVEL GENE DISCOVERY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF ENHANCING THE RATE OF NOVEL GENE DISCOVERY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a highly efficient, high-throughput method for the identification and elimination of redundancy in a population of nucleic acid molecules using microarrays. This method involves a reiterative subtraction protocol using a library that becomes more biased toward unknown genes with each successive round. The removal of repetitive and previously characterized nucleic acids from the library allows the identification of low-abundance mRNA from sources of interest and enhances the rate of novel gene discovery. The present invention is also useful for the removal of contaminating nucleic acids from cloning libraries.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
The present invention relates to nucleic acids and more particularly to methods for the identification and isolation of non-redundant mRNAs and novel genomic sequences.<br>
BACKGROUND OF THE INVENTION<br>
The elucidation of the mechanisms that dictate the normal functioning of living cells requires a detailed understanding of the information encoded in all of the complete genome. Messenger RNA (mRNA) sequences are typically used to map and sequence the genes contained in the genomes of different organisms. The sequence information is used to evaluate the genetic makeup of a particular cell or organism of interest. However, mRNAs are produced at different levels within different cell types and during different points in development. The distribution of mRNA types, their developmental and cell-type specific regulated expression, and their translation into protein produce the unique character of a particular cell type.<br>
Populations of nucleic acid molecules, such as messenger RNAs, are typically studied using cloned nucleic acid libraries. Libraries can include genomic libraries, which can be constructed by placing randomly cleaved DNA fragments of an entire genome into a suitable cloning vector. For libraries made from DNA of mammalian genomes, most random genomic clones contain non-coding DNA, highly repeated DNA, or both.<br>
A second type of library is made from complementary DNA (cDNA) molecules. These libraries are constructed from DNA that is reverse transcribed from mRNA isolated from a source of interest. Accordingly, cDNA libraries primarily contain DNA that codes for genes.<br>
Different species of mRNA are not equally represented in a given cell. Instead, mRNA molecules are distributed into three frequency classes: (1) superprevalent (consisting of approximately 10-15 mRNAs which, together, represent 10-20% of the total mRNA mass); (2) intermediate (consisting of approximately 1-2,000 mRNAs which, together,  represent 40-45% of the total mRNA mass), and (3) complex<br><br>
(consisting of approximately 15-20,000 mRNAs which, together, represent 40-45% of the total mRNA mass). Davidson and Brittan, SCIENCE 204: 1052-1059 (1979).<br>
The existence of superprevalent and intermediate complexity RNA molecules within a cell can be a significant obstacle to the identification and sequencing of low-abundance mRNA species. In the creation of nucleic acid libraries suitable for sequencing, superprevalent mRNAs impede the isolation and analysis of lower abundance mRNAs. Since the majority of clones isolated from a cDNA library will be from superprevalent and intermediate prevalent mRNAs, significant time and effort is spent resequencing previously known prevalent mRNA species, and large numbers of mRNA species must be sequenced in order to isolate and sequence low-abundance mRNA species. Thus, the rate of gene discovery from libraries can be limited by the redundant nature of mRNAs present in a given cell. The presence of highly abundant mRNAs also hinders the comparison of differences in active genes observed in different cells of related tissue types, cells in varying stages of development, the effect of stimuli, and differential gene expression between normally functioning and abnormal cells (e.g., cells from normal tissue compared to tumor tissues).<br>
SUMMARY OF THE INVENTION<br>
The present invention is based in part on the discovery of a highly efficient, high-throughput method for the identification and elimination of redundancy in a population of nucleic acid molecules.<br>
The method includes providing a random sample of nucleic acid fragments, immobilizing the random sample of nucleic acid fragments on a microarray, and hybridizing one or more labeled probes corresponding to previously arrayed or sequenced fragments. Fragments that hybridize to the labeled probes are detected, and at least one fragment not hybridized or weakly hybridized to the labeled probes is identified and, if desired, is sequenced.<br>
The nucleic acid fragments may be RNA or DNA, and may be optionally cloned into a vector. In some embodiments, the nucleic acid fragments are members of a cDNA or genomic library. The library may be normalized or non-normalized. In<br><br>
other embodiments,  the nucleic acid fragments are PCR fragments.     In some embodiments, the nucleic acid fragments are amplified, e.g., by PCR.<br>
The nucleic acid fragments are then immobilized to a solid surface, e.g., in a microarray. The solid surface is preferably glass. Labeled probes that correspond to previously arrayed or sequenced fragments (i.e., the subtraction probe) are hybridized to the immobilized nucleic acid fragments. Nucleic acid labels may be fluorescent, luminescent, or radioactive labels, biotinylated, hyphenated, or other chemical tags that allow for easy detection of labeled probes. Generally, the no hybridized probes are removed. Nucleic acid fragments that are not hybridized or are weakly hybridized to a labeled probe are isolated and are then pooled with the previous set of probes to generate a new, larger probe set. Usually, the newly isolated fragments are sequenced and their sequences compared to those found in a sequence database.<br>
The methods of the present invention involve a subtraction protocol that identifies and isolates non-redundant nucleic acid fragments from a population of nucleic acid molecules. If desired, the protocol is reiterated, in order to create a set of fragments that becomes more biased toward previously uncharacterized genes with each successive iteration. Accordingly, with each round of subtraction, probes corresponding to the newly isolated fragments are labeled and added to the previous subtraction probe, and this new subtraction probe is hybridized to the next microarray containing randomly picked nucleic acid fragments. This procedure is repeated several times, preferably accompanied by adding the newly identified sequences to the previous subtraction probe. Thus, the method allows the identification and isolation of non-redundant or minimally overlapping nucleic acid fragments from sources of interest and enhances the rate of novel gene discovery. In a preferred embodiment, the non-redundant clones that are isolated using the methods of the invention are identified, selected, and immobilized to a new microarray to produce a unsigned set.<br>
Numerous applications can be envisioned for the methods described herein. For example, the method can be used in any application in which enrichment for sequences of interest is desired. Alternatively, the method can be used to remove undesired nucleic acid fragments from a population of nucleic acid molecules. A non-limiting set of uses includes:<br><br>
I.	A microarray-based method for enhancing the rate of discovery of expressed mRNA/cDNA sequences and facilitating construction a set of nucleic acid molecules for identifying messages present in low amounts in a population of nucleic acid molecules. This method allows for increasing discovery of expressed cDNAs and expedited construction of sets of uniquely cDNAs.<br>
II.	A microarray-based method for enhancing the rate of discovery of genomic sequences and facilitating isolation of a DNA fragments corresponding to a whole genome or sub regions of interest. In this application, the method provides for increased discovery of genomic clones, expedited construction of a set of non-redundant or minimally tiled genomic clones. Also provided is increased discovery of clones mapping to a region of interest, expedited construction of a set of genomic clones in a region of interest in the genome, and expedited filling of gaps in genomic maps. This can be useful in e.g. facilitating disease gene mapping and disease gene identification.<br>
III.	A microarray-based method for enrichment and/or isolation of DNA sequences that are unique to a population compared to another population. The invention also allows for identification of sequences unique or novel to one organism versus another, including nucleic acid molecules from different strains, e.g., pathogenic vs. nonpathogenic, and different species. The sequences can be, e.g., mRNA/cDNA, genomic, extra chromosomal, plasmid, or viral nucleic acids.<br>
IV.	A microarray-based method for increasing discovery of related DNA sequences. Conversely, the method allows for identification of related sequences among closely or distantly related organisms. The sequences can be, e.g., mRNA/cDNA, genomic, extra chromosomal, plasmid, and viral nucleic acids.<br>
V.	A microarray-based method for enhancing the rate of removal of undesired sequences from a population of nucleic acid molecules. In another embodiment, the invention provides for removal of undesired DNA sequences, including contaminating DNA sequences and sequences closely related to those previously identified.<br>
VI.	A microarray-based method for identifying changes in copy number of one or more DNA sequences in two different populations of nucleic acids.<br><br>
Accordingly the present invention provides a method of enhancing the rate of novel gene discovery in a population of previously-uncharacterized nucleic acid molecules, comprising<br>
a.	contacting a first ordered array of nucleic acid samples from said population with an<br>
initial complex subtraction probe pool, said probe pool comprising a labeled probe<br>
corresponding to a redundant or known sequence;<br>
b.	identifying a nucleic acid, said nucleic acid being characterized by weak or no<br>
hybridization to said initial probe pool;<br>
c.	sequencing said nucleic acid to yield a newly-identified sequence;<br>
d.	adding said newly-identified sequence to said initial pool to generate a second<br>
complex subtraction probe pool, said second pool comprising a larger number of<br>
labeled probes compared to said initial pool;<br>
e.	contacting a second ordered array of nucleic acid samples from said population with<br>
said second subtraction probe pool; and<br>
f repeating steps b.-e., wherein each repeat of steps b.-e. reduces the redundancy of said population of nucleic acid molecules, thereby enhancing the rate of novel gene discovery.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, suitable methods and materials are described below. AH publications, patent applications, patents, and other references mentioned herein are incorporated by reference in their entirety. In the case of conflict, the present specification, including definitions, will control. In addition, the materials, methods, and examples are illustrative only and not intended to be limiting.<br>
Other features and advantages of the invention will be apparent from the following detailed description and claims.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 is a flow chart depicting an overview of the gene discovery strategy.<br><br>
FIG. 2 is a representation of a microarray used in the identification of novel genes. FIG. 3 is a flow chart depicting the construction of a mouse calvaria unigene chip. FIG. 4 is a line graph depicting the removal of contaminating mitochondrial genes from a library. DETAILED DESCRIPTION OF THE INVENTION<br>
For convenience, the intended meaning of certain terms and phrases used herein are provided below.<br>
"Known genes" or "known sequences" are those whose sequences can be found in a public or private sequence database.<br>
"Novel genes" or "novel sequences" are nucleic acid molecules whose sequences are not publicly available, e.g., they are not found in any public or private sequence database.<br>
"Heterogeneous" means a set of nucleic acid molecules wherein the set includes at least two nucleic acid molecules that differ in sequence. Heteregeneous nucleic acids include, e.g., RNA populations isolated from cells, tissues, and/or organisms.<br>
"Weakly" hybridizing, as defined herein, refers to a hybridization signal that has a signal-to-noise (S/N) ratio of less than 0.5, as identified using the data analysis system of ArrayVision software. S/N is a measure of the background subtracted intensity of the spot signal divided by the standard deviation of the background intensity.<br>
"Non-redundant nucleic acid" is a nucleic acid that has not been previously identified in a population of nucleic acid molecules, e.g., a heterogeneous population of nucleic acid molecules. In some embodiments, the non-redundant nucleic acid is a sequence present in a low copy number in a population of nucleic acid molecules. "Unique gene set" is a set of genes wherein each member of the set corresponds to a sequence represented in a low copy number in a population of cells. For example, a unique gene set corresponds to sequences present in a few copies per cell. A unique gene set can alternatively be referred to as a reduced abundance gene set. In some embodiments, the unique gene set includes members present in 100, 50, 25, 15, 10, 5, 2 copies, or even in 1 copy per cell.   Thus, a unique gene set from a particular cell<br><br>
type or population will, in some embodiments, contain one copy of each gene expressed in that cell type.<br>
"Complex probe" is one that contains many different DNA or RNA molecules. As used herein, the complex probe contains nucleic acid sequences that are complementary to known sequences.<br>
The present invention provides a highly efficient, high-throughput method for the identification and elimination of redundancy in a population of nucleic acid molecules using reiterative selection involving subtractive hybridization with a complex probe pool and microarray technology. Randomly-chosen nucleic acid fragments chosen from a cloning library are microarrayed and exposed, under hybridizing conditions, to a complex labeled probe generated from known sequences (e.g. sequences obtained from a public database such as Genbank, pubEST). Only clones that hybridize weakly, or do not hybridize, to this first probe pool are sequenced; clones that hybridize to this first probe pool are not sequenced. The sequences obtained in this step are compared to public databases of previously-identified nucleic acid molecules (e.g. BLAST, pubEST, Genbank NR). These sequences are added to the initial probe pool to generate a second, subtractive probe pool. This subtractive probe pool is hybridized to a second group of randomly-chosen clones from the library. The probe hybridizes to the nucleic acids complimentary to the nucleic acids of the initial, publicly known sequences, as well as to the nucleic acids identified in the first round of hybridization. The procedure of expanding the probe pool after successive roimds of hybridization is repeated until essentially all randomly-chosen clones hybridize to a labeled probe, indicating that all of the previously uncharacterized clones have been sequenced. Accordingly, the methods of the present invention eliminate the need to re-sequence known clones - a problem that reduces the efficiency of other methods known in the art.<br>
The first step involves isolation of a heterogeneous population of nucleic acid molecules. The nucleic acid molecules can be fragments of full-length RNA or DNA molecules. In some embodiments, the heterogeneous population of nucleic acid molecules can include randomly-chosen nucleic acid fragments from a population of nucleic acid molecules.    These fragments may be DNA or RNA, and they may<br><br>
optionally be cloned into vectors. In a preferred embodiment, the nucleic acid molecules include cDNA from a nucleic acid library of unknown sequences. In one embodiment, the library is a genomic library. In another embodiment, the library is a cDNA library. In either embodiment, the library can be a normalized library, or a non-normalized library. Alternatively, the population of nucleic acid fragments may not be part of a library, e.g., PCR fragments.<br>
In some embodiments, the nucleic acids are amplified using PCR or other amplification methods. The DNA is then immobilized in a microarray on a surface, preferably glass. In some embodiments, the arrayed DNA includes oligonucleotides synthesized on glass. In a preferred embodiment, the PCR fragments are arrayed using a microarray spotter.<br>
Immobilized DNA molecules are then allowed to contact a labeled probe pool containing nucleic acid molecules of known or defined sequences under conditions that permit hybridization. Standard labeling protocols for nucleic acids are described, e.g., in Sambrook et al.; Kambara et al., BIOTECHNOLOGY 6: 816-821 (1988); Smith et al., Nuc. ACIDS RES. 75: 2399-2412 (1985). Nucleic acid labels may be fluorescent, luminescent, or radioactive labels, biotinylated, haptenated, or other chemical tags that allow for easy detection of labeled probes. Fluorescent labels are advantageous for the methods described herein, as these are routinely used with automated instrumentation for simultaneous high throughput analysis of multiple samples. Metzker and Gibbs have recently disclosed a family of fluorescently tagged nucleotides based on the Cy fluorophores with improved spectral characteristics. See, e.g., U.S. Patent No. 5,728,529, incorporated herein by reference. Alternative sets of fluorophores include rhodamine based fluorophores, TARAM, ROX, JOE, and FAM; the BigDye® fluorophores (Applied Biosystems, Inc.), the dansyl group, fluorescein and substituted fluorescein derivatives, acridine derivatives, coumarin derivatives, pthalocyanines, tetramethylrhodamine, Texas Red®, 9-(carboxyethyl)-3-hydroxy-6-oxo-6H-xanthenes, DABCYL® and BODIPY® fluorophores (Molecular Probes, Eugene, OR).<br>
Subsequent to the hybridization step, the amount of label is detected at each position in the   a DNA microarrav.   See Brown and Botstein, NAT.<br><br>
GENET. 21: 33-37 (1999). Microarrays are an ordered array of double stranded or single stranded DNA molecules positioned on a support material in a spatially separated organization. In contrast to filter "macroarrays", which are typically large sheets of nitrocellulose, microarrays position the DNA more densely packed organization such that up to 10,000 DNA molecules can be fit into a region typically 1-4 square centimeters. Microarrays typically use coated glass as the solid support, in contrast to the nitrocellulose-based material of filter arrays. By having an ordered array of DNA samples, the position of each sample can be tracked and linked to the original sample from which the DNA on the array was generated. Methods and apparatus for preparing a microarray have been described. See, e.g.. United States Patents 5,445,934 and 5,800,992, both incorporated herein by reference.<br>
The process of creating a microarray involves a series of steps in which a set of bacterial colonies containing a cDNA library are processed to obtain an amplified fragment of purified DNA derived from the insert of the vector held in any one bacterial colony. The series of steps can be carried out either manually or through the use of robotic workstations, such as PCR machines, and liquid handling robots. In the preferred method, the steps are carried out using multiwell plates made up of 96 wells, 384 wells, or 1536 wells. However, other configurations are possible. The microarray is constructed using a robotic spotter such as the instrument developed by Molecular Dynamics or in the laboratory of Patrick Brown, Stanford University (ibid). Another feature of the microarray process is that the data analysis is performed automatically using data analysis software, such as ArrayVision. The entire process from clone tracking, microarray construction, to results is fully integrated so that bacterial colonies to be processed for sequencing can be readily identified. In various embodiments, a microarray according to the invention includes 100; 1,000; 10,000, or even 100,000 or more DNA samples.<br>
The DNA samples on the microarray are hybridized with RNA or DNA probes that have been labeled, e.g., fluorescently labeled, to identify whether the probe sample contains a molecule that is similar or identical to the DNA sample on the microarray.   In preferred embodiments, a complex probe can be hybridized to the<br><br>
microarray.   A complex probe is one that contains many different DNA or RIN/\ molecules.<br>
Probe molecules are hybridized to a DNA molecule on the microarray under conditions that allow for formation of hybrids between probe molecules and identical or nearly identical DNA molecule or molecules on the microarray. The presence of DNA-probe hybrid molecules is detected, e.g., by a fluorescence detection instrument. If the hybridization signal is weak or non-existent at a particular DNA site, then the corresponding DNA or RNA molecule in the probe is absent. In various embodiments, 1, 2, 3, or 4 or more probe molecules are used simultaneously.<br>
In some embodiments, iterative rounds of microarray construction, hybridization, and analysis accomplish the process. The method reduces amount of sequencing to obtain a unique gene set of a given size. In various embodiments, it can be used to remove repeated, e.g., duplicate cDNAs, samples with bacterial or mitochondrial DNA, and samples without any cDNA insert.<br>
There are a number of advantages of microarray-based subtraction versus filter array methods. For example, the present methods dispense with the need to propagate clones in bacteria prior to sequencing. For example, filter methods generally array bacterial colonies in which the cloned cDNA is contained. The colonies must be grown up over several days, lysed to release DNA and fix DNA onto the filter. Hybridization to filter arrays of colonies can be hindered because of bacterial debris and low amounts of DNA released fi"om the colony.<br>
A second advantage of the present methods is that the iterations can be performed more quickly with microarrays than with filter assays using bacterial colonies. Propagating and analyzing clones in bacteria can be require up to several days. In contrast, probing of a subsequent microarray can begin less than 24 hr after analysis of an array is completed.<br>
Another advantage of microarrays is the ability to use fluorescently labeled probes. This provides for a non-radioactive method for hybridization detection. In contrast, filter hybridization generally uses probes labeled with radioactive phosphorus   or   sulfur.      Microarrays  can  be  hybridized  with  multiple  probes<br><br>
simultaneously.   In contrast, filter arrays can only be hybridized with one probe at a time.<br>
A further advantage of microarrays is in their reproducibility and sensitivity of hybridization signals. Typically, hybridization signals are higher and sensitivity is greater on microarrays versus filter arrays. This allows for the complexity of the probe to be higher and still get a positive hybridization signal on the microarray. In addition, filter arrays often exhibit spurious background signals that are unrelated to productive hybridization between the probe and DNA on the filter.<br>
There are in addition a number of advantages of constructing a unique gene set by microarray-based subtraction versus other methods. For example, methods using selective hybridization in solution, as used in cDNA normalization or subtraction procedures, typically result in the loss of certain low abundant clones. In contrast, using the iterative microarray-based subtraction approach, all clones in the library are capable of being identified. A low abundant cDNA clone will be found after multiple iterative rounds of hybridization. Another advantage of the microarray-based approach for preparation of unique sets of cDNAs is that the resulting cDNA clones are more likely to be near full length. In contrast, other methods for subtraction and normalization often enrich short, incomplete cDNAs.<br>
A weak signal or no signal at all at a particular position of the array generally corresponds to the presence of DNA that has not hybridized or has weakly hybridized to sequences represented in the probe pool. Clones with a weak signal or no signal at all are sequenced. Sequencing can be performed using DNA sequencing methods well known in the art. See, e.g., Maxam and Gilbert, PROC NATL ACAD SCI USA 74: 560-564 (1977); Sanger et al, PROC NATL ACAD Sci USA 74: 5463-5467 (1977); and United States Patent No. 5,821,058, incorporated herein by reference.<br>
Automated analysis of fragments in gels or capillaries has significantly reduced the labor involved in collecting and processing sequence information. See, e.g.. Prober et al. SCIENCE 238: 336-341 (1987); Smith et al, NATURE 321: 674-679 (1986); Luckey et al, NUCLEIC AciDS RES 75: 4417-4421(1990); Dovichi, ELECTROPHORESIS 18: 2393-2399 (1997). The nucleic acid fragments isolated and sequenced in this step are added into the initial probe pool. Often, these fragments are<br><br>
sequenced first to determine the presence of non-redundant nucleic acid molecules in the microarray. Their sequences are then compared to public databases of previously-identified nucleic acid molecules (e.g., by comparing sequences in publicly available databases).<br>
The second subtractive probe pool is used in the next round of hybridization involving a second microarray generated from a different group of randomly-chosen clones from the library. By adding the newly identified nucleic acid fragments to the subtractive probe pool, subsequent microarrays of randomly-chosen clones from the library are exposed to a larger number labeled probes. Accordingly, methods of the present invention eliminate the re-sequencing of known clones<br>
As the number of labeled probed increases with successive iterations, the number of labeled "known" clones will increase and fewer clones will be unlabeled. Eventually, with sufficient iterations, all of the clones in microarrays of randomly-chosen clones from a given library will be labeled, as all of the clones will have been sequenced.<br>
In one aspect of the present invention, a cloning library enriched in low abundance mRNAs is generated. In an extension of this method, the low-abundance sequences are used in the construction of a chip containing nucleic acids corresponding to low abundance RNAs. These chips can be used in subsequent experimentation (e.g., differential display experiments). An overview of the strategy for the discovery of novel genes is depicted in FIG. 1. In an additional embodiment of the present invention, contaminating nucleic acids are removed from a cloning library. In another aspect, the invention provides method for reducing redundancy in a sample of nucleic acid molecules. The method includes providing a heterogeneous sample of nucleic acid molecules and immobilizing the sample of nucleic acid molecules on a microarray. One or more labeled probes corresponding to previously arrayed or sequenced nucleic acid molecules are then hybridized to the nucleic acid molecules in the immobilized sample. Preferably, the probes include sequences known to be present in the sample of nucleic acid molecules. The method also includes identifying at least one immobilized molecule that hybridizes to the labeled probes and providing a second nucleic acid molecule which specifically recognizes the immobilized nucleic<br><br>
acid or acids which hybridize to the sample. If desired, the second nucleic acid molecule can be labeled and used as a probe molecule, and the sample can be rehybridized with a probe molecule or molecules that includes the newly added probe molecule. The process can be repeated, as desired, to progressively eliminate nucleic acid molecules that have already been identified in the population of nucleic acid molecules.<br>
Also included in the invention is a method of identifying a plurality of non-redundant sequences in a population of nucleic acids by providing a heterogeneous sample of nucleic acid molecules, immobilizing the sample of nucleic acid molecules on a microarray, and hybridizing one or more labeled probes to the sample. The labeled probes correspond to one or more previously identified nucleic acid molecules in the sample. At least one immobilized nucleic acid molecule in the immobilized sample that hybridizes weakly or does not hybridize to the labeled probes is identified. The process is repeated, if desired, until a plurality of non-redundant sequences has been identified. The invention also includes the sequences identified according tot this method, as well as a microarray that includes the plurality of sequences. The invention may be adapted to any purpose in which the practitioner desires to enrich for sequences of interest or remove sequences that are not of interest. Exemplary uses include, but are not limited to, the following:<br>
I. A MICROARRAY-BASED METHOD FOR ENHANCING THE RATE OF DISCOVERY OF EXPRESSED mRNA/cDNA SEQUENCES AND FACILITATING CONSTRUCTION OF A SET OF SEQUENCES CORRESPONDING TO UNIQUE OR LOW ABUNDANCE TRANSCRIPS<br>
In one aspect, the invention provides for increased novel gene discovery and expedited construction of a set from expressed cDNAs. This can be accomplished by:<br>
a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of nucleic acid fragments;<br>
b)	immobilizing the random nucleic acids on a solid surface in a microarray format;<br><br>
(c)	hybridizing labeled probes from a DNA source to the immobilized, microarrayed DNA fragments;<br>
(d)	detecting DNA fragments hybridized to a labeled probe and identifying at least one fragment that does not hybridize or hybridizes weakly (i.e., subtraction);<br>
(e)	determining the identity of the DNA fragment by DNA sequencing, hybridization or other analytic approaches; and<br>
(f) reiterating steps (b) or (c) through (e) with previously identified sequences in the probe set in order to identify additional sequences and increase the unique gene set.<br>
II. A MICROARRAY-BASED METHOD FOR ENHANCING THE RATE OF DISCOVERY OF GENOMIC SEQUENCES AND FACILITATING ISOLATION OF A DNA FRAGMENTS CORRESPONDING TO A WHOLE GENOME OR SUBREGIONS OF INTEREST<br>
In the case of genomic sequences, it may be desired to construct a set of "minimally tiled" clones, i.e., clones with the smallest amount of overlapping sequence possible. Thus, the invention allows for increased discovery of genomic clones, expedited construction of a set of non redundant or minimally tiled genomic clones and increased discovery of clones mapping to a gene of interest. The method also allows expedited construction of set of genomic clones in a region of interest in the genome. The ability to easily map clones to a region of interest in the genome further enables the practitioner to easily fill gaps in a genomic map, facilitating disease gene mapping and disease gene identification. The method includes:<br>
a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of genomic nucleic acid fragments;<br>
b)	immobilizing the random nucleic acids on a solid surface in a microarray format;<br>
c)	hybridizing labeled probes (pooled or single) from a DNA source to the immobilized, microarrayed DNA fragments (probes can be cDNA/mRNA or genomic sequences);<br>
d)	detecting DNA fragments that hybridize to a labeled probe;<br><br>
e)	determmmg tne identity oi tne JJJNA rragment oy UJNA sequencing, hybridization or other analytic approaches; and<br>
f)	reiterating steps (b) or (c) through (e) with previously identified sequences in the probe set in order to identify additional sequences and increase the unique gene set.<br>
In particular, end regions of previously identified sequences can be used as probes to "walk" and identify flanking clones.<br>
III. A MICROARRAY-BASED METHOD FOR ENRICHMENT AND/OR ISOLATION OF DNA SEQUENCES (MRNA/CDNA, GENOMIC, EXTRACHROMOSOMAL, PLASMID AND ALL OTHER) THAT ARE UNIQUE TO A POPULATION COMPARED TO ANOTHER POPULATION.<br>
Another use of the invention is for the identification of sequences (expressed cDNA or genomic) unique or novel to one organism versus another. This includes identification of nucleic acids unique to one strain vs. another (i.e., pathogenic vs. non-pathogenic), as well as comparison of unique gene sets from closely related species and more distantly related species.<br>
Since nucleic acid molecules from more than one source organism are used in this application, the specific steps of the method are somewhat different and include:<br>
a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of nucleic acid fragments;<br>
b)	mmobilizing the random nucleic acids on a solid surface in a microarray format;<br>
c)	hybridizing labeled probes from a first source and a second source to the immobilized, microarrayed DNA fragments (probes can be cDNA/mRNA or genomic sequences);<br>
d)	detecting DNA fragments which hybridize to a labeled probe from a first source but not a second source or visa versa; and<br>
e)	determining the identity of the DNA fragment by DNA sequencing, hybridization or other analytic approaches.<br><br>
IV.	A MICRO ARRAY-BASED METHOD FOR INCREASING DISCOVERY OF<br>
RELATED (OR CONSERVED) DNA SEQUENCES (MRNA/CDNA,<br>
GENOMIC, EXTRACHROMOSOMAL, PLASMID AND ALL OTHER)<br>
The converse to the above method allows discovery of related sequences (rather than differences) in various species, including different bacterial strains and distantly related organisms. The method includes:<br>
a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of nucleic acid fragments;<br>
b)	immobilizing the random nucleic acids on a solid surface in a microarray format;<br>
c)	hybridizing labeled probes (single or pooled) to the immobilized, microarrayed DNA fragments, particularly at decreased hybridization stringencies (probes can be cDNA/mRNA or genomic sequences);<br>
d)	detecting DNA fragments that hybridize to a labeled probe from a first source but not a second source or visa; and<br>
e)	determining the identity of the DNA fragment by DNA sequencing, hybridization or other analytic approaches;<br>
V.	A MICROARRAY-BASED METHOD FOR ENHANCING THE RATE OF<br>
REMOVAL OF UNDESIRED SEQUENCES<br>
The invention also provides for removal of undesired DNA sequences (cDNA or genomic) from any population of arrayed DNA. This may include contaminating DNA, as well as any DNA not of interest, e.g., sequences closely related to those already identified. This method includes:<br>
(a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of nucleic acid fragments;<br>
(b)	immobilizing the random nucleic acids on a solid surface in a microarray format;<br>
(c)	hybridizing labeled probes, which are sequences targeted for removal, to the immobilized, microarrayed DNA fragments;<br><br>
(d)	detecting DNA fragments hybridized to a labeled probe and identifying at least one fragment that does not hybridize or hybridizes weakly (i.e., subtraction);<br>
(e)	determining the identity of the DNA fragment by DNA sequencing, hybridization or other analytic approaches; and<br>
(f)	reiterating steps (a), (b) or (c) through (e) with previously identified sequences in the probe set, as deemed necessary, in order to eliminate unwanted sequences from the population of fragments.<br>
VI. A MICROARRAY-BASED METHOD FOR IDENTIFYING CHANGES IN COPY NUMBER (UNDER OR OVER REPRESENTED) OF DNA SEQUENCES (GENOMIC, EXTRACHROMOSOMAL, PLASMID AND ALL OTHER) BETWEEN DIFFERENT SOURCES OF NUCLEIC ACIDS.<br>
The present invention also provides a microarray-based method for identifying changes in copy number (under or over represented) of DNA sequences (genomic, extrachromosomal, plasmid and all other) between different sources of nucleic acids. The method includes:<br>
(a)	amplifying (by PCR or nucleic acid isolation procedures) and providing a random sample of nucleic acid fragments from a given source;<br>
(b)	immobilizing the random nucleic acids on a solid surface in a microarray format;<br>
(c)	hybridizing labeled probes (single or pooled; any type of nucleic acid -mRNA/cDNA, genomic, extrachromosomal, plasmid and all other - generally corresponding to the type of nucleic acid arrayed) from another source to the immobilized, microarrayed DNA fragments;<br>
(d)	detecting DNA fragments which show absent, significantly lesser or significantly greater hybridization to a labeled probe. Such alterations in signal intensity will reflect changes in abundance; and<br>
(e)	determining the identity of the DNA fragment(s) by DNA sequencing, hybridization or other analytic approaches.<br><br>
Example 1    cDNA Microarray Construction<br>
1.1      cDNA Amplification<br>
A cDNA library was generated by techniques well known in the art. cDNA inserts were isolated from bacterial clones using Polymerase Chain Reaction (PCR). Twenty five μl of PCR mix was added to each well of a 384 well microtiter plate from a PCR master mix containing 1000 μl l0X PCR Buffer, 800 μl of 2.5 mM dNTPs, 400 ^1 T3 primer (5 pmoles/μl), 400 μ1 of T7 primer (5 pmoles/μl), and 10μl recombinant Taq polymerase. The PCR plate was inoculated from the overnight growth plate using a 384 pin tool to transfer approximately 1 μl. Plates were sealed with MicrosealA and placed in alpha units of MJ Research tetrad PCR machines. Reactions were preheated at 95°C for 4 min, followed by 35 cycles of amplification: 45 sec at 95°C, 1 min at 55°C, and 2.5 min at 72°C.<br>
Amplification of the cDNA inserts was verified by agarose gel electrophoresis in which 2 μl of each PCR reaction was loaded onto a 1% agarose gel containing 500 ng/μl ethidium bromide. 500 ng of molecular weight standard (1 Kb ladder, Promega) was added to each gel for size determination and quantitation of amplification. Samples were electrophoresed in IX TAE (Tris Acetate, EDTA) at 150 milliamps for 30 min.<br>
Amplified cDNA inserts were purified from unincorporated nucleotides and primers in 384 well glass fiber filter plates as follows: 70 μl of 5M guanidinium isothiocyanate (Sigma) was added to each plate using a 96 pin Cyclone liquid handler; 25 μl of PCR reaction was transferred from the PCR microtiter plate to the glass fiber filter plate and incubated for 2 min at room temperature. The filter plate was placed on a vacuum manifold and dried. The filter plate was washed 2 times with 70μl of 80% isopropanol and dried on a vacuum manifold for two min. The purified PCR product was eluted from the glass fiber plate into a 384-well collection plate by addition of 50 μl of water, incubation for 5 min at room temperature, and centrifugation at 3000 rpm for 5 min.   Purified PCR products were lyophilized to<br><br>
dryness in a speed-vac on high for 45 min to  1 hour.    Purified products were resuspended in 30 μl of 50% DMSO/water. 1.2     Immobilization of the Sample<br>
A maximum of 4,608 different PCR fragments were spotted onto glass slides in duplicate using a Generation III (Gen III) microarray spotter from Amersham/Molecular Dynamics. Twelve 384-well U bottom microliter plates were placed in the   hotel of  the Gen III spotter.   Up to 36 glass slides were placed on the<br>
spotter and spotted with DNA while the spotting humidity of the spotting chamber was maintained at 55%. Six slides were spotted per source plate access. Prior to accessing the next set of PCR templates, the 12 pen cassette was washed as follows: 1 sec in 0.2 M KOH, 1 sec in 95% ethanol, and 2 sec in distilled water. After spotting, slides were air dried for 1 hour at room temperature. The PCR fragments were cross linked to the glass by UV crosslinking in a Stratalinker (Stratagene) at 5000 joules. Example 2    Probe Synthesis<br>
The cDNA inserts for sequence verified clones were amplified in 96-well PCR plates as previously described for 384-well plates, with the exception that the PCR reaction volume was 50 μl. Amplification was verified by agarose gel electrophoresis as previously described. PCR fragments were purified using 96-well carbon fiber filter plates and a DNA binding resin as follows: 100 μl of Wizard PCR purification resin (Promega) was added to each well of a 96-well filter plate using a 12 charmel pipettor; the 50 μl PCR- reaction was added to the resin, and incubated at room temperature for 1 min; the plate was then placed on a vacuum manifold for 1 min, until dry. Each plate was briefly washed with 200μl of 80% isopropanol before the solution was removed by vacuum manifold, and the filter plate was well dried by vacuum. Purified PCR product was eluted by addition of 50 μl of sterile, distilled water followed by centrifugation at 3000 rpm for 5 min into a 96-well microliter plate. The purified PCR reactions were pooled and 100 μg of the pooled PCR reaction was purified over Qiagen PCR purification columns. The repurified PCR pool was verified by gel electrophoresis as previously described.    The concentration of the<br><br>
pooled PCR was determined spectrophotometrically and the concentration adjusted to 100 ng/μl using sterile, distilled water.<br>
The purified PCR fragment pool was used as a template for RNA synthesis in an in vitro transcription reaction. Purified PCR reaction (500 ng) was added to a 20 μl reaction containing IX transcription buffer, 0.5 mM of each rNTP, 37 mM DTT, and 10 units of T7 or T3 RNA polymerase. The reaction was incubated at 37°C for 90 min. The DNA template was removed by addition of 1 unit of RQ1 DNase and incubation at 37°C for 30 min. The unincorporated nucleotides were removed by purification over an RNA Easy Column (Qiagen).<br>
A fluorescent probe was synthesized from the in vitro synthesized RNA template in a first strand labeling reaction as follows: 100 ng of RNA template was incubated with 0.5 μg random hexamers in a final volume of 10 μl for 10 min at 70°C. The reaction was chilled on ice for 5 min, followed by the addition a reaction mix containing Ix Reverse transcription buffer (20 mM Tris pH 8.4, 50 mM KCl, 2.5 mM MgCb), 10 mM DTT, 100 μM dGTP, dTTP, dATP, 50 μM dCTP, 50 μM Cye3 dCTP or cye5 dCTP, and 200 units of Superscript II reverse transcriptase (Gibco BRL, #18089-011). The reaction was incubated at 42°C for 90 min. The RNA template was hydrolyzed by the addition of 1 μl of 5 M NaOH and incubation at 37°C for 10 min. The NaOH was neutralized by adding 10 μl of 2M MOPS (free acid). Unincorporated nucleotides and primers were removed by purification over GFX columns (Pharmacia). Probes were lyophilized down and resuspended in 30 μl of hybridization buffer containing 50% formamide, 5XSSC, 0.2% SDS, IX Denhardt"s, 100 μg/ml salmon sperm DNA, and 1 μg oligo-dA(80). Example 3 Identification of Novel Genes The procedure for identifying novel genes is depicted in FIG. 2. 3.1     Hybridization<br>
Probes were denatured at 100°C for 10 min and added to the microarray slide. Slides were covered with glass cover slips (Coming) and incubated for 18 to 24 hours at 42°C in a humidified chamber. Hybridized slides were washed twice in 2X SSC containing 0.1% SDS for 10 min at room temperature, followed by 2 washes in 0.1 X<br><br>
SSC containing 0.1% SDS at 42°C for 10 min. Slides were dipped several times in distilled water and dried under filtered high pressure air. Hybridization of fluorescent probes was detected by scanning the slides in a Generation III (Gen III) confocal scanner (Amersham/Molecular Dynamics).<br>
3.2	Data Acquisition<br>
Slide images were analyzed with Array Vision image analysis software (Imaging Research) for spot finding analysis, localized background determination, distribution of signal intensities in a spot, and signal to noise ratios. Data was exported as a tab-delimited file and exported to an Oracle database. Normalizatior and statistical quality assessment was performed with a Web-based set of data analysis tools and data was analyzed with Web browser tools developed in the Hoechst-Ariac Genomics Center. Data was sorted based on signal to noise ratio, and DNAs which had a signal above background but below a threshold signal to noise ratio were further characterized by EST sequencing.<br>
3.3	Sequencing and Comparison to Known Libraries<br>
Clones which did not hybridize in step 3.1 were targeted for EST (expressec sequence tag) sequencing, using techniques well-known in the art. Adams et al. SCIENCE 252: 1651-1656 (1991). Sequences were searched against the public dbEST mouse EST, and Lifeseq (Incyte) databases. All clones that were newly characterize( or identified as previously-known sequences were added to the probe (subtraction pool to prevent multiple identification of the same gene during subsequen hybridizations.<br>
3.4	Reiterative Subtractive IHybridization<br>
The next microarray containing randomly picked clones was hybridized to i new probe that included all the previous clones, and in addition contained any nev clones identified from EST sequencing. The clones identified from this methodolog; were used for building unique gene sets (Ermolaeva et al., NAT. GENET. 20: 19-2! (1998)) containing clones previously identified from other sources {e.g., publii databases) as well as novel clones which had not been previously described.<br><br>
Example 4    Selection of Low Abundance mRNAs<br>
A reiterative-based subtraction protocol was used where microarrays were constructed which contained 1500 different cDNAs randomly picked from a cDNA library. The first microarray was hybridized with a subtraction probe containing cDNAs encoding 64 housekeeping and ribosomal genes. Non-hybridizing clones were analyzed by EST sequencing. Any clones that were not present in the previous subtraction probe were added to the existing subtraction probe and hybridized to the next microarray containing randomly picked cDNA clones. This procedure was repeated 17 times, so that a total of 26,112 cDNAs were microarrayed. Based on the hybridization data, only 7,700 clones were chosen for EST sequencing. After clustering, 4,400 different cDNA clones were identified. This group of clones was highly enriched for low abundance transcripts. Additionally, by removing the redundant high abundance messenger RNAs, the microarray-based subtraction decreased the EST sequencing effort by 70%.<br>
Example 5    Construction of a Chip for Identifying Unique or Low Abundant Messages, use in Differential Display<br>
A chip for identifying messenger RNA present in low copy number in mouse calivaria was constructed according to the procedure outlined in FIG. 3. Arrays were generated of 27,648 clones from normalized mouse calvaria libraries. All 27,648 clones were PCR amplified and arranged into 18 microarrays. As a result of subtractive hybridization (as outlined above), 7,790 cDNAs were sequenced following 18 rounds of hybridization. Upon comparison of these sequences to databases containing known sequences, 4,608 clones were chosen for amplification and placement on a microarray. This chip was then used in a differential gene expression experiment to identify genes modulated by BMP2 (bone morphogenic protein 2). Example 6    Removal of Library Contamination<br>
The subtractive hybridization protocol as described by the current invention was demonstrated in the removal of contamination of highly abundant genes from a cloning library. As shown in FIG. 4, the initial library was contaminated by 35%) of mitochondrial genes.  After nine rounds of subtractive hvbridization, this amount was<br><br>
reduced to 2-3%.   This demonstrates the usefulness of the invention in removing highly abundant sequences from a starting population of nucleic acids. EQUIVALENTS<br>
From the foregoing detailed description of the specific embodiments of the invention, it should be apparent that a unique method of gene isolation has been described. Although particular embodiments have been disclosed herein in detail, this has been done by way of example for purposes of illustration only, and is not intended to be limiting with respect to the scope of the appended claims that follow.<br><br><br>
WE CLAIM;<br>
 A method of enhancing the rate of novel gene discovery in a population of previously-uncharacterized nucleic acid molecules, comprising<br>
a.	contacting a first ordered array of nucleic acid samples from said population with an<br>
initial complex subtraction probe pool, said probe pool comprising a labeled probe<br>
corresponding to a redundant or known sequence;<br>
b.	identifying a nucleic acid, said nucleic acid being characterized by weak or no<br>
hybridization to said initial probe pool;<br>
c.	sequencing said nucleic acid to yield a newly-identified sequence;<br>
d.	adding said newly-identified sequence to said initial pool to generate a second<br>
complex subtraction probe pool, said second pool comprising a larger number of<br>
labeled probes compared to said initial pool;<br>
e.	contacting a second ordered array of nucleic acid samples from said population with<br>
said second subtraction probe pool; and<br>
f repeating steps b.-e., wherein each repeat of steps b.-e. reduces the redundancy of said population of nucleic acid molecules, thereby enhancing the rate of novel gene discovery.<br>
  The method as claimed in claim 1, wherein said redundant or known sequence is selected from the group consisting of a public database member gene, a housekeeping gene or ribosomal gene.<br>
 The method as claimed in claim 1, wherein said first and said second ordered array comprise DNA immobilized on a glass surface.<br>
 . The method as claimed in claim 1, comprising repeating steps b.-e. to generate a Unsigned set.<br>
   The method as claimed in claim 1, wherein said weak hybridization comprises a signal-to-noise ration of less than 0.5.<br><br>
6. A method of enhancing the rate of novel gene discovery in a population of previously-uncharacterized nucleic acid molecules substantially as hereinbefore described with reference to the accompanying drawings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgYWJzdHJhY3QgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che abstract duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgY2xhaW1zIGR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che claims duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgY29ycmVzcG9uZGVuY2Ugb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgY29ycmVzcG9uZGVuY2UgcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKSBkdXBsaWNhdGUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che description (complete) duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZHJhd2luZ3MgZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che drawings duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTYyMC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1620-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="215957-a-method-for-scouring-and-dyeing-cellulosic-fibers-in-a-single-bath.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="215959-centrifugal-separator-with-improved-injection-of-a-solids-laden-water-stream.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>215958</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1620/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Nov-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVENTIS PHARMACEUTICALS INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Route 202-206, P.O. Box 6800 Bridgewater, NJ 08807-0800,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PERRIN, Steven</td>
											<td>120 Adams Avenue, Newton, MA 02465,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CALL, Katherine</td>
											<td>99 Woodland Road, Malden, MA 02148,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CONNOLLY, Timothy</td>
											<td>74 Pond Street, Belmont, MA 02178,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/13789</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-05-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/314,698</td>
									<td>1999-05-19</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/215958-a-method-of-enhancing-the-rate-of-novel-gene-discovery by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:07:20 GMT -->
</html>
