<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216458-an-isolated-nucleic-acid-sequence-comprising-a-polynucleotide-encoding-a-plant-sterol-cholesterol-acyltransferase-like-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:40:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216458:AN ISOLATED NUCLEIC ACID SEQUENCE COMPRISING A POLYNUCLEOTIDE ENCODING A PLANT STEROL: CHOLESTEROL ACYLTRANSFERASE-LIKE POLYPEPTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED NUCLEIC ACID SEQUENCE COMPRISING A POLYNUCLEOTIDE ENCODING A PLANT STEROL: CHOLESTEROL ACYLTRANSFERASE-LIKE POLYPEPTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is directed to lecithin: cholesterol acyltransferase-like polypeptides (LCAT) and acyl CoA: cholesterol acyltransferases-like polypeptides (ACAT). The invention provides polynucleotides encoding such cholesterol: acyltransferases-like polypeptides, polypeptides encoded by such polynucleotides, and the use of such polynucleotides to alter sterol composition and oil production in plants and host cells. Also provided are oils produced by the plants and host cells containing the polynucleotides and food products, nutritional supplements, and pharmaceutical composition containing plants or oils of the present invention. The polynucleotides of the present invention include those derived from plant sources.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This application claims priority to U.S. provisional application Serial No. 60/152,493, filed August 30, 1999 and herein incorporated by reference in its entirety for all purposes.<br>
BACKGROUND<br>
Technical Field<br>
The present invention is directed to plant acyltransferase-like nucleic acid and amino acid sequences and constructs, and methods related to their use in altering sterol composition and/or content, and oil composition and/or content in host cells and plants.<br>
Related An<br>
Through the development of plant genetic engineering techniques, it is now possible to produce transgenic varieties of plant species to provide plants which have novel and desirable characteristics. For example, it is now possible to genetically engineer plants for tolerance to environmental stresses, such as resistance to pathogens and tolerance to herbicides. It is also possible to improve the nutritional characteristics of the plant, for example to provide improved fatty acid, carotenoid, sterol and tocopherol compositions. However, the number of useful nucleotide sequences for the engineering of such characteristics is thus far limited.<br>
There is a need for improved means to obtain or manipulate compositions of sterols from biosynthetic or natural plant sources. The ability to increase sterol production or alter the sterol compositions in plants may provide for novel sources of sterols for use in human and animal nutrition.<br>
Sterol biosynthesis branches from the famesy! diphosphate intermediate in the isoprenoid pathway. Sterol biosynthesis occurs via a mevalonate dependent pathway in mammals and higher plants (Goodwin,(1981) Biosynthesis of Isoprenoid Compounds, vol \ (Porter, J.W, &amp; Spurgeon, S.L., eds) pp.443-480, John Wiley and Sons, New York), while in green algae sterol biosynthesis is thought to occur via a mevalonate independent pathway (Schwender, et al. (1997) Physiology, Biochemistry, and Molecular Biology of<br><br>
Plant Lipids, (Williams, J.Pâ€ž Khan, M.U., and Lem, N.W., eds) pp. 180-182, Kluwer Academic Publishers. Norweil, MA).<br>
The solubility characteristics of sterol esters suggests that this is the storage form of sterols (Chang, era/., 0997)/(n/iu. Rev. Biochem., 66:612-638). Sterol O-acyltransferase enzymes such as acyl CoA:cholesterol acyltransferase (ACAT) and lecithin:cholesterol acyltransferase (LCAT) catalyze the formation of cholesterol esters, and thus are key to controlling the intracellular cholesterol storage. In yeast, it has been reported that overexpression ofLROl, a homolog of human LCAT, and phospho lipid; di acyl glycerol acyltransferase increased lipid synthesis (Oelkers et al., (2000) J. Biol. Chem., 26:15609-15612; Dahiqvist et al., (2000) Proc. Natl Acad. Sci. USA, 97-,6487-6492),<br>
The characterization of various acylfransferase proteins is useful for the fiirther study of plant sterol synthesis systems and for the development of novel and/or alternative sterol sources. Studies of plant mechanisms may provide means to fixriher enhance, control, modify, or otherwise alter the sterol composition of plant ceils. Funhennore, such alterations in sterol content and/or composition may provide a means for obtaining tolerance to stress and insect damage. Of particular interest are the nucleic acid sequences of genes encoding proteins which may be useful for applications in genetic engineering.<br>
SUMMARY OF THE INVENTION<br>
The present invention is directed to lecithin; cholesterol acyltransferase-like polypeptides (also refened to herein as LCAT) and acyl CoAicholesterol acyliransferase-like polypeptides (also referred to herein as ACAT). In particular the invention is related to polynucleotides encoding such sterol :acyltransferases, polypeptides encoded by such polynucleotides, and the use of such polynucleotides to alter sterol composition and oil production. The polynucleotides of the present invention include those derived from plant sources.<br>
One aspect of the invention, therefore, is an isolated nucleic acid sequence encoding a plant lecithin:cholesterol acyltransferase-like polypeptide, a fragment of a plant lecithin:cholesterol acyltransferase-like polypeptide, a plant acyl CoA:cholesterol acyltransferase-like polypeptide or a fragment of a plant acyl CoAicholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of SEQ ID NO: 2, 4, 6, 8, 10-29, 43-51, 73 or 75. Also provided is an isolated nucleic acid sequence consisting of SEQ ID NO; 2,4, 6, 8,10-29, 43-51, 73 or 75.<br><br>
Still another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 3 or SEQ ID NO: 3 with at least one conservative amino acid substitution; SEQ ID NO: 2; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 2; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 2; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 2 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Still another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula 5" X-(R,)â€ž-(R2)n-(R3}â€ž-Y 3" where X is a hydrogen, Y is a hydrogen or a metal, R, and Rj are any nucleic acid, n is an integer between 0-3000, and R, is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO". 3 or SEQ ID NO: 3 with at least one conservative amino acid substitution; SEQ ID NO: 2; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 2; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 2; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 2 and encodes a plant lecithin;cholesteroI acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID N0:5 or SEQ ID NO: 5 with at least one conservative amino acid substitution; SEQ ID NO: 4; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 4; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 4; an isolated polynucleotide complementaiy to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 4 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula 5" X-{R,X-(RjV(R3)â€ž-Y 3" where X is a hydrogen, Y is a hydrogen or a metal, R; and Rj are any nucleic acid, n is an integer between 0-3000, and R; is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 5 or SEQ ID NO: 5 with at least one conservative amino acid<br><br>
substitution; SEQ ID NO: 4; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ H) NO: 4; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 4; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 4 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ED N0:7 or SEQ ID NO: 7 with at least one conservative amino acid substitution; SEQ ID NO: 6; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 6; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ED NO: 6; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 6 and encodes a plant iecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the fomiula 5" X-(R|)â€ž-(R2)â€ž-(R3VY 3" where X is a hydrogen, Y is a hydrogen or a metal, R, and Rj are any nucleic acid, n is an integer between 0-3000, and Rj is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 7 or SEQ ID NO: 7 with at least one conservative amino acid substitution; SEQ ID NO: 6; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ED NO: 6; an isolated polynucleotide of at least 10 amino acids that hybridizes under sttingent conditions to SEQ ID NO: 6; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 6 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID N0:9 or SEQ ID NO: 9 with at least one conservative amino acid substimtion; SEQ ID NO: 8; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 8; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 8; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that<br><br>
hybridizes under stringent conditions to SEQ ID NO: 8 and encodes a plant lecithin-.cholesterol acyitransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula 5" X-(R,X"(RX-(RiX-Y 3" where X is a hydrogen, Y is a hydrogen or a metal, R, and Rj are any nucleic acid, n is an integer between 0-3000, and R is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 9 or SEQ ID NO: 9 with at least one conservative amino acid substitution; SEQ ID NO: 8; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence idenrity with SEQ ID NO: 8; an isolated polynucleotide of al least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 8; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ iD NO: 8 and encodes a plant lecithin;cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO:74 or SEQ ED NO: 74 with at least one conservative amino acid substitution; SEQ ID NO: 73; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 73; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 73; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 73 and encodes a plant lecithinxholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula 5" X-(R|)â€ž-{R2)â€ž-(R3)â€ž-Y 3" where X is a hydrogen, Y is a hydrogen or a metal, Ri and R are any nucleic acid, n is an integer between 0-3000, and R; is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 74 or SEQ ID NO: 74 with at least one conservative amino icid substitution; SEQ ID NO: 73; an isolated polynucleotide that has at least 70%, 80%, ?0%, or 95% sequence identity with SEQ ID NO: 73; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 73; an isolated jolynucleotide complementary to any of fiie foregoing; and an isolated polynucleotide that lybridizes under stringent conditions to SEQ ID NO: 73 and encodes a plant 2cithin:choleslerol acyitransferase-like polypeptide.<br><br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO:76 or SEQ ID NO: 76 with at least one conservative amino -acid substitution; SEQ ID NO: 75; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 75; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 75; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 75 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide ofthe formula 5" X-(R,)n-(Rj)â€ž-(R3X-Y3" where X is a hydrogen. Visa hydrogen or a metal, Rj and Rj are any nucleic acid, n is an integer between 0-3000, and R, is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 76 or SEQ ID NO: 76 with at least one conservative amino acid substimtion; SEQ ID NO: 75; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 75; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 75; an isolated polynucleotide complementary to any ofthe foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 75 and encodes a plant lecithin:cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of SEQ ID NO: 42 or a degenerate variant thereof; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 42; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 42; an isolated polynucleotide compiementary to any ofthe foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 42 and encodes an acyl Co A; cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula 5" X-(R,)â€ž-(Ri)n-(R3)n-Y 3" where X is a hydrogen, Y is a hydrogen or a metal, R and Rj are any nucleic acid, n is an integer between 0-3000, and R3 is selected from the group consisting of SEQ ID NO: 42 or a degenerate variant thereof; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 42; an isolated polynucleotide of at least 10 amino acids that hybridizes<br><br>
under stringent conditions to SEQ ID NO: 42; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 42 and encodes a acyl CoA:cholesterol acyltransferase-like . polypeptide.<br>
Also provided is a recombinant nucleic acid construct comprising a regulatory sequence operably linked to a polynucleotide encoding a lecithin:choleslerol acyltransferase-like polypeptide and/or an acyl Co A: cholesterol acyltransferase-like polypeptide. In one embodiment, the sterol acyl transferases are plant sterol acyl transferases. In another embodiment, the recombinant nucleic acid constructs further comprises a termination sequence. The regulatory sequence can be a constitutive promoter, an inducible promoter, a developmentally regulated promoter, a tissue specific promoter, an organelle specific promoter, a seed specific promoter or a combination of any of the foregoing. Also provided is a plant containing this recombinant nucleic acid construct and the seed and progeny from such a plant. This recombinant nucleic acid construct can also be introduced into a suitable host cell to provide yet another aspect of the invention. If the host cell is a plant host ceil, the cell can be used to generate a plant to provide another aspect of the invention. Further aspects include seed and progeny from such a plant.<br>
Yet another aspect is a purified polypeptide comprising, SEQ ID NO: 3, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 9, SEQ ID NO: 74, SEQ ID NO: 76, or any of the preceding sequences with at least one conservative amino acid substitution.<br>
Still another aspect provides a purified immunogenic polypeptide comprising at least 10 consecutive amino acids fiom an amino acid sequence selected from the group consisting of SEQ ID NO: 3, 5, 7, 9, 74, 76 and any of the preceding sequences containing at least one conservative amino acid substitution. Also provided are antibodies, either polyclonal or monoclonal, that specifically bind the preceding immunogenic polypeptides.<br>
One aspect provides a method for producing a lecithin:cholesterol acyltransferase-like polypeptide or an acyl Co A: cholesterol acyltransferase-like polypeptide comprising culturing a host cell containing any recombinant nucleic acid construct of the present invention under condition permitting expression of said lecithin :choIesterol acyltransferase-like polypeptide or acyl CoAicholesterol acyltransferase-like polypeptide.<br>
Another aspect provides a method for modiiying the sterol content ofa host cell, comprising transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithin:cholesteroI<br><br>
acyltransferase-Uke polypeptide and culturing said host cell under conditions wherein said host-cell expresses a lecithinxholesterol acyitransferase-like polypeptide such that said host cell has a modified sterol composition as compared to host cells without the recombinant construct.<br>
An additional aspect is a method for modifying the sterol content of a host cell comprising transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoA; cholesterol acyltransferase-Iike polypeptide and culturing said host cell under conditions wherein said host cell expresses an acyl CoA; cholesterol acyltransferase-Iike polypeptide such that said host cell has a modified sterol composition as compared to host cells without the recombinant construct.<br>
A fiirther aspect is a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithinxholesterol acyltransferase-Uke polypeptide wherein expression of said recombinant coi\struct results in modified sterol composition of said plant as compared to the same plant without said recombinant construct.<br>
Another aspect provides a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoAxholesteroi acyltransferase-Uke polypeptide wherein expression of said recombinant construct results in modified sterol composition of said plant as compared to the same plant without said recombinant construct.<br>
In a further aspect is provided an oil obtained from any of the plants or host cells of the present invention.<br>
In still another aspect is provided a method for producing an oil with a modified sterol composition comprising providing any of the plants or host ceils of the present invention and extracting oil from the plant by any known method. Also provided is an oil produced by the preceding method.<br>
Still another aspect provides a method for altering oil production by a host cell comprising, transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithinxholesterol acyltransferase-Uke polypeptide and culturing the host cell under conditions wherein the host cell expresses a lecithinxholesterol acyltransferase-Uke polypeptide such that the host cell has an altered oil production as compared to host cells without the recombinant construct.<br><br>
Another aspect provides a method for ahering oil production by a host ceil comprising, transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl Co A: cholesterol acyltransferase-like polypeptide and culturing the host cell under conditions wherein the host cell expresses an acyl CoAxholesterol acyltransferase-like polypeptide such that the host ceil has an altered oil production as compared to host cells without the recombinant construct.<br>
Also provided is a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a leciihinxholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in an altered production of oil by said plant as compared to the same plant without said recombinant construct.<br>
In a fiirther aspect is provided a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoAxholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in an altered production of oil by said plant as compared to the same plant without said recombinant construct.<br>
Additional aspects provide a food, food ingredient or food product comprising any oil, plant or host cell of the present invention; a nutritional or dietary supplement comprising any oil, plant or host cell of the present invention; and a pharmaceutical composition comprising any oil, plant or host cell of the present invention along with a suitable diluent, carrier or excipient.<br>
Additional aspects will be apparent from the descriptions and examples that follow.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
These and other features, aspects, and advantages of the present invention will become better understood with regard to the following description, appended claims and accompanying figures where;<br>
Figure 1 shows an alignment of yeast, human and rat lecithin:cholesterol acyltransferase protein sequences with Arabidopsis LCATl, LCAT2, LCAT3, and LCAT4 deduced amino acid sequences.<br>
Figure 2 shows the results of NMR sterol ester analysis on T2 seed from plant expressing LCAT4 under the control of the napin promoter (pCGN9998).<br><br>
Figure 3 shows the results of hi"U vMS sterol analysis on oil extracted from T2 seed from control lines (pCGN8640) and lines expressing LCAT3 (pCGN9968) under the control of the napin proraoter.<br>
Figure 4 shows the results of HPLC/MS sterol analysis on oil extracted from T2 seed from control lines (pCGN8640), and plant line expressing LCATl (pCGN9962), LCAT2 (pCGN99S3), LCAT3 (pCGN9968), and LCAT4 {pCGN9998) under the control of the napin promoter. Additionally, data from 3 lines expressing LCAT4 under the control of the 35S promoter (pCGN9996) are sho&gt;wn.<br>
Figure 5 shows the results of Nir analysis of the oil content of T2 seed from control lines (pCGN8640), and plant lines expressing LCATl (pCGN9962), LCAT2 (pCGN9983), and LCAT3 (pCGN9968) under the control of the napin promoter. Additionally, data from 16 lines expressing LCAT2 under the control of the 35S promoter {pCGN9981) are shown.<br>
DETAILED DESCRIPTION<br>
The following detailed description is provided to aid those skilled in the art in practicing the present invention. Even so, this detailed description should not be construed to unduly hmii the present invention as modifications and variations in the embodiments discussed herein can be made by those of ordinary skill in the art without departing from the spirit or scope of the present inventive discovery.<br>
All publications, patents, patent applications and other references cited in this application are herein incorporated by reference in their entirety as if each individual publication, patent, patent application or other reference were specifically and individuaUy indicated to be incorporated by reference.<br>
The present invention relates to lecithinxholesterol acyltransferase, particularly the isolated nucleic acid sequences encoding lecithin: cholesterol-like polypeptides (LCAT) from plant sources and acyl CoAxhoieslerohacyltransferase, particularly the isolated nucleic acid sequences encodmg acyl CoAxholesterol acyltransferase-like polypeptides (ACAT) from plant sources. Lecithinxholesierol acyltransferase-Iike as used herein includes any nucleic acid sequence encoding an amino acid sequence from a plant source, such as a protein, polypeptide or peptide, obtainable fitjm a ceil source, which demonstrates the ability to utilize lecithin (phosphatidyl choline) as an acyl donor for acylation of sterols or glycerides to fonn esters tmder enzyme reactive conditions along with such proteins polypeptides and peptides. Acyl Co A: cholesterol acyltransferase-like<br><br>
as used herein includes any nucleic acid sequence encoding an amino acid sequence from a plant source, uch as a protein, polypeptide or peptide, obtainable from a ceil source, which demonstrates the ability to utilize acyl CoA as an acyl donor for acylation of sterols or glycerides to form esters under enzyme reactive conditions along with such proteins polypeptides and peptides. By "enzyme reactive conditions" is meant that any necessary conditions are available in an environment (i.e., such factors as temperature, pH, lack of inhibiting substances) which will permit the enzyme to ftmction.<br>
The term "sterol" as apphed to plants refers to any chirai tetracyclic isopentenoid which may be formed by cycUzation of squaiene oxide through the transition state possessing stereochemistry similar to the trans-syn~trans-anti-trans-anti configuration, for example, protosteroid cation, and which retains a polar group at C-3 (hydroxyl or keto), an all-trans-anii stereochemistry in the ring system, and a side-chain 20R-configuration (Parker, etal. (1992)/n Nes, e/a/., Eds., Regulation of Isopentenoid Metabolism, ACS Symposium Series No. 497, p. 110; American Chemical Society, Washington, D.C).<br>
Sterols may or may not contain a C-5-C-6 double bond, as this is a feature introduced late in the biosynthetic pathway. Sterols contain a Cj-Cio side chain at the C-17 position.<br>
The term "phytosterol," which applies to sterols found uniquely in plants, refers to a sterol containing a C-5, and in some cases a C-22, double bond. Phytosterols are fiirther characterized by alkylation of die C-17 side-chain with a methyl or ethyl substituent at the C-24 position. Major phytosterols include, but are not limited to, sitosterol, stigmasterol, campesterol, brassicasterol, etc. Cholesterol, which lacks a C-24 methyl or ethyl side-chain, is found in plants, but is not unique thereto, and is not a "phytosterol."<br>
"Phytostanols" are saturated forms of phytosterols wherein the C-5 and, when present, C-22 double bond(s) is (are) reduced, and include, but are not Ihnited to, sitostanol, campestanol, and 22-dihydrobrassicastanol-<br>
"Sterol esters" are further characterized by the presence of a fatty acid or phenolic acid moiety rather than a hydroxyl group at the C-3 position.<br>
The terra "sterol" includes sterols, phytosterols, phytosterol esters, phytostanols, and phytostanol esters.<br>
The term "sterol compounds" includes sterols, phyotsterols, phytosterol esters, phytostanols, and phytostanol esters.<br>
The term "phytosterol compound" refers to at least one phytosterol, at least one phytosterol ester, or a mixture thereof.<br><br>
The term "phytostanol compound" refers to at least one phytostanol, at least one phytostanol ester, or a mixture thereof.<br>
The temi "giyceride" refers to a fatty acid ester of glycerol and includes mono-, di-, and tri- acylgiycerols.<br>
As used herein, "recombinant construct" is defined either by its method of production or its structure. In reference to its method of production, e.g., a product made by a process, the process is use of recombinant nucleic acid techniques, e.g., involving human intervention in the nucleotide sequence, typically selection or production. Alternatively, in terms of structure, it can be a sequence comprising ftision of two or more nucleic acid sequences which are not naturally contiguous or operatjveiy linked to each other<br>
As used herein, "regulatory sequence" means a sequence of DNA concerned with controlling expression of a gene; e.g. promoters, operators and attenuators. A " heterologous regulatory sequence" is one which differs from the regulatory sequence naturally associated with a gene.<br>
As used herein, "polynucleotide" and "oligonucleotide" are used interchangeably and mean a polymer of at least two nucleotides joined together by a phosphodiester bond and may consist of either ribonucleotides or deoxynucleotides.<br>
As used herein, " sequence" means the linear order in which monomers appear in a polymer, for example, the order of amino acids in a polypeptide or the order of nucleotides in a polynucleoride.<br>
As used herein, "polypeptide", "peptide", and "protein" ar: used interchangeably and mean a compound that consist of two or more amino acids that are linked by means of peptide bonds.<br>
As used herein, the tenns "complementary" or "complementarity" refer to the pairing of bases, purines and pyrimidines, that associate through hydrogen bonding in double stranded nucleic acids. For example, the following base pairs are complementary: guanine and cyiosine; adenine and thymine; and adenine and uracil. The terms, as used herein, include complete and partial complementarity.<br><br>
Isolated proteins. Polypeptides and Polynucleotides<br>
A first aspect of the present invention relates to isolated LCAT polynucleotides. The polynucleotide sequences of the present invention include isolated polynucleotides that encode the polypeptides of the invention having a deduced amino acid sequence selected from the group of sequences set forth in the Sequence Listing and to other polynucleotide sequences closely related to such sequences and variants thereof<br>
The invention provides a polynucleotide sequence identical over its entire length to each coding sequence as set forth in the Sequence Listing. The invention also provides the coding sequence for the mature polypeptide or a fragment thereof, as well as the coding sequence for the mature polypeptide or a fragment thereof in a reading frame with other coding sequences, such as those encoding a leader or secretory sequence, a pie-, pro-, or prepro- protein sequence. The polynucleotide can also include non-coding sequences, including for example, but not limited to, non-coding 5" and 3" sequences, stjch as the transcribed, untranslated sequences, termination signals, ribosome binding sites, sequences that stabilize mRNA, introns, polyadenylation signals, and additional coding sequence that encodes additional amino acids. For example, a marker sequence can be included to facilitate the purification of the fused polypeptide. Polynucleotides of the present invention also include polynucleotides comprising a structural gene and the tiaturally associated sequences that control gene expression.<br>
The invention also includes polynucleotides of the formula: X-(R,).-(R-{R3).-Y wherein, at the 5" end, X is hydrogen, and at the 3" end, Y is hydrogen or a metal, R, and R3 are any nucleic acid residue, n is an integer between 0 and 3000, preferably between 1 and 1000 and Rj is a nucleic acid sequence of the invention, particularly a nucleic acid sequence selected from the group set forth in the Sequence Listing and preferably SEQ ID NOs: 2, 4, 6, 8, 10-29, 33, 42-51, 73 and 75. In the formula, R is oriented so that its 5" end residue is at the left, bound to R,, and its 3" end residue is at the rit, bound to R3. Any stretch of nucleic acid residues denoted by either R group, where R is greater than I, may be either a heteropolymer or a homopolymer, preferably a heteropolymer.<br>
The invention also relates to variants of the polynucleotides described herein that encode for variants of the polypeptides of the invention. Variants that are fragments of the polynucleotides of the invention can be used to synthesize full-length polynucleotides of the invention. Preferred embodiments are polynucleotides encoding polypeptide variants wherein 5 to 10, 1 to 5,1 to 3,2,1 or no amino acid residues of a polypeptide sequence of<br><br>
the invention are substituted, added or deleted, in any combination. Particularly preferred are substitutions, additions, and deletions that are silent such that they do not alter the properties or activities of the polynucleotide or polypeptide.<br>
Further preferred embodiments of the invention that are at least 50%, 60%, or 70% identical over their entire length to a polynucleotide encoding a polypeptide of the invention, and polynucleotides that are complementary to such polynucleotides. More preferle are polynucleotides that comprise a region that is at least 80% identical over its entire length to a polynucleotide encoding a polypeptide of the invention and polynucleotides that are complementary thereto. In this regard, polynucleotides at least 90% identical over their entire length are particularly preferred, those at least 95% identical are especially preferred. Further, those with at least 91% identity are highly preferred and those with at least 98% and 99% identity are particularly highly preferred, with those at least 99% being the most highly preferred.<br>
Preferred embodiments are polynucleotides that encode polypeptides that retain substantially the same biological function or activity as determined by the methods described herein as the mature polypeptides encoded by the polynucleotides set forth in the Sequence Listing.<br>
The invention further relates to polynucleotides that hybridize to the above-described sequences. In particular, the invention relates to polynucleotides that hybridize under stringent conditions to the above-described polynucleotides. An example of stringent hybridization conditions is overnight incubarion at 42Â°C in a solution comprising 50% formamide, 5x SSC (150 mM NaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardt"s solution, 10% dextran sulfate, and 20 micrograms/milliliter denatured, sheared salmon sperm DNA, followed by washing the hybridization support in Olx SSC at approximately 65Â°C. Also included are polynucleotides that hybridize under a wash stringency of 0. IX SSC or O.IX SSPE (at SO"C- Other hybridization and wash conditions are well known and are exemplified in Sambrook, et ai. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY (1989), particularly Chapter 11.<br>
The invention also provides a polynucleotide consisting essentially of a polynucleotide sequence obtainable by screening an appropriate library containing the complete gene for a polynucleotide sequence set for in the Sequence Listing under stringent hybridization conditions with a probe having the sequence of said polynucleotide<br><br>
seiquence or a fragment thereof; and isolating said polynucleotide sequence. Methods for screening libraries are well known in the art and can be found for example in Sambrook, ei ai. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY (1989), particularly Chapter 8 and Ausubel et al. Short Protocols in Molecular Biology, 3" ed, Wiley and Sons, 1995, chapter 6. Nucleic acid sequences useful for obtaining such a polynucleotide include, for example, probes and primers as described herein and in particular SEQ ID NO: 2, 4, 6, 8,10-29, 33,42-51, 73 and 75. These sequences are particularly useful in screening libraries obtained from Arabidopsis, soybean and com for sequences encoding lecithin:cholesterol acyltransferase and lecithin:cholesterol acyitransferase-Uke polypeptides and for screening libraries for sequences encoding acyt CoA:cholesterol acyl transferase and acyl CoAxhoIesterol acyl transferase-like polypeptides.<br>
As discussed herein regarding polynucleotide assays of the invention, for example, polynucleotides of the invention can be used as a hybridization probe for RNA, cDNA, or genomic DNA to isolate full length cDNAs or genomic clones encoding a polypeptide and to isolate cDNA or genomic clones of other genes that have a high sequence similarity to a polynucleotide set forth in the Sequence Listing and in particular SEQ ID NO: 2, 4, 6, S, 10-29,33, 42-51, 73 and 75. Suchprobes will generEdly comprise at least 15 bases. Preferably such probes will have at least 30 bases and can have at least 50 bases. Particularly preferred probes wiH have between 30 bases and 50 bases, inclusive.<br>
The coding region of each gene that comprises oris comprised by a polynucleotide sequence set forth in the Sequence Listing may be isolated by screening using a DNA sequence provided in the Sequence Listing to synthesize an oUgonucleolide probe. A labeled oligonucleotide having a sequence complementary to that of a gene of the invention is then used to screen a library of cDNA, genomic DNA or mRNA to identify members of the library which hybridize to the probe. For example, synthetic oligonucleotides are prepared which correspond to the LCAT EST sequences. The oiigonucieotides are used as primers in polymerase chain reaction (PCR) techniques to obtain 5" and 3" terminal sequence of LCAT genes. Alternatively, where oligonucleotides of low degeneracy can be prepared from particular LCAT peptides, such probes may be used directly to screen gene libraries for LCAT gene sequences. In particular, screening of cDNA libraries in phage vectors is useful in such methods due to lower levels of background hybridization.<br><br>
Typically, a LCAT sequence obtainable from the use of nucleic acid probes will show 60-70% sequence identity between the target LCAT sequence and the encoding sequence used as a probe. However, lengthy sequences with as little as 50-60% sequence identity may also be obtained. The nucleic acid probes may be a lengthy fragment of the nucleic acid sequence, or may also be a shorter, oligonucleotide probe. When longer nucleic acid fragments are employed as probes (greater than about 100 bp), one may screen at lower stringencies in order to obtain sequences from the target sample which have 20-50% deviation (i.e., 50-80% sequence homology) from the sequences used as probe. Oligonucleotide probes can be considerably shorter than the entire nucleic acid sequence encoding an LCAT enzyme, but should be at least about 10, preferably at least about 15, and more preferably at least about 20 nucleotides. A higher degree of sequence identity is desired when shorter regions are used as opposed to longer regions. It may thus be desirable to identify regions of highly conserved amino acid sequence to design oligonucleotide probes for delecting and recovering other related LCAT genes. Shorter probes are often particularly useful for polymerase chain reactions (PCR), especially when highly conserved sequences can be identified. (See, Gou\d, et al., PNAS USA (1989) 56:1934-1938.).<br>
Another aspect of the present mvention relates to LCAT polypeptides. Such polypeptides include isolated polypeptides set forth in the Sequence Listing, as well as polypeptides and fragments thereof, particularly those polypeptides which exhibit LCAT activity and also those polypeptides which have at least 50%, 60% or 70% identity,  â–  preferably at least 80% identity, more preferably at least 90% identity, and most preferably at least 95% identity to a polypeptide sequence selected from the group of sequences set forth in the Sequence Listing, and also include portions of such polypeptides, wherein such portion of the polypeptide preferably includes at least 30 amino acids and more prefersdaly includes at least 50 amino acids.<br>
"Identity", as is well understood in the art, is a relationship between two or more solypeptide sequences or two or more polynucleotide sequences, as determined by ;omparing the sequences. In the art, "identity" also means the degree of sequence â– elatedness between polypeptide or polynucleotide sequences, as determined by the match jetween strings of such sequences. "Identity" can be readily calculated by known methods ncluding, but not limited to, those described in Computational Molecular Biology, Lesk, KM., ed., Oxford University Press, New York (1988); Biocomputing: Informatics and Jenome Projects, Smith, D.W., ed.. Academic Press, New York, 1993; Computer Analysis<br><br>
of Sequence Data, Part I, Griffin, A.M. and Griffin, H.G., eds., Humana Press, New Jersey (1994); Sequence Analysis in Molecular Biology, von Heinje, G., Academic Press (1987); Sequence Analysis Primer, Gribskov, M. and Devereux, J., eds., Stockton Press, New York (1991); and Carillo, H., and Lipman, D., SIAM J Applied Math, 48:1073 (1988). Methods to determine identity are designed to give the largest match between the sequences tested. Moreover, methods to determine identity are codified in publicly available programs. Computer programs which can be used to determine identity between two sequences iBcIude, but are not limited to, GCG (Devereux, J., et al.. Nucleic Acids Research 12(1):387 (1984); suite of five BLAST programs, three designed for nucleotide sequences queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology, J2: 76-80 (1994); Birren, et al.. Genome Analysis, J: 543-559 (1997)). The BLAST X program is publicly available from NCBI and other sources (BLASTManual, Altschul, S., el al., NCBINLM NIH, Bethesda, MD 20894; Altschul, S., et al., J. Mol. Biol., 215:403-410 (1990)). The well known Smith Waterman algorithm can also be used to detennine identity.<br>
Parameters for polypeptide sequence comparison typically include the following:<br>
Algorithm: Needleman and Wunsch, J". Mol. Biol. 48:443-453 (1970)<br>
Comparison matrix: BLOSSUM62 from Hentikoff and Hentikoff, Proc. Natl. Acad. Sci USA 89:10915-10919 (1992)<br>
Gap Penalty: 12<br>
Gap Length Penalty: 4<br>
A program which can be used with these parameters is publicly available as the "gap" program from Genetics Computer Group, Madison Wisconsin. The above parameters along with no penalty for end gap are the default parameters for peptide comparisons.<br>
Parameters for polynucleotide sequence comparison include the following:<br>
Algorithm: Needleman and Wunsch, ]. Mol. Biol. 48:443-453 (1970)<br>
Comparison matrix: matches = +10; mismatches = 0<br>
Gap Penalty: 50<br>
Gap Length Penalty: 3<br>
A program which can be used with these parameters is publicly available as the "gap" program from Genetics Computer Group, Madison Wisconsin. The above parameters are the default parameters for nucleic acid comparisons.<br><br>
The invention also includes polypeptides of the fonnula;<br>
x-(R,)n-{R:)-(R3VY<br>
wherein, at the amino terminus, X is hydrogen, and at the carboxyl temiinus, Y is hydrogen or a metal, R, and R, are any amino acid residue, n is an integer between 0 and 1000, and Rj is an amino acid sequence of the invention, particularly an amino acid sequence selected from the group set forth in the Sequence Listing and preferably SEQ ID NOs: 3, 5, 7, 9, 74 and 76. In the formula, R, is oriented so that its amino terminal residue is at the left, bound to Rj, and its carboxy terminal residue is at the right, bound to R3. Any stretch of amino acid residues denoted by either R group, where R is greater than 1, may be either a heteropolymer or a homopolymer, preferably a heteropoiymer.<br>
Polypeptides of die present invention include isolated polypeptides encoded by a polynucleotide comprising a sequence selected from the group of a sequence contained m SEQ ID NOs: 2, 4, 6, 8, 73 and 75.<br>
The polypeptides of the present invention can be mature protein or caii be part of a fusion protein.<br>
Fragments and variants of the polypeptides are also considered to be a part of the invention. A fragment is a variant polypeptide which has an amino acid sequence that is entirely the same as part but not all of the amino acid sequence of the previously described polypeptides. The fragments can be "free-standing" or comprised within a lafger polypeptide of which the fragment forms a part or a region, most preferably as a single continuous region. Preferred fragments are biologically active fragments which are those fragments that mediate activities of the polypeptides of the invention, including those with similar activity or improved activity or with a decreased activity. Also included are those polypeptides and polypeptide fragments that are antigenic or immunogenic in an animal, particuiarly a human and antibodies, either polyclonal or monoclonal that specifically bind the antigenic fragments. In one preferred embodiment, such antigenic or immunogenic fragments comprise at least 10 consecutive amino acids from the amino acid sequences disclosed herein or such sequences with at least one conservative amino acid substitution. In additional embodiments, such antigenic or immunogenic fragments comprise at least 15, at least 25, at least 50 or at least 100 consecutive amino acids from the amino acid sequences disclosed herein or such sequences with at least one conservative amino acid substitution. Methods for the production of antibodies from polypeptides and polypeptides conjugated to carrier molecules are well known in the art and can be found<br><br>
for example in Ausubel et al.. Short Protocols in Molecular Biology, 3"* ed., Wiley &amp; Sons, 1995, particularly chapter 11.<br>
Variants of the polypeptide also include polypeptides that vary from the sequences set forth in the Sequence Listing by conservative amino acid substitutions, substitution of a residue by another with like characteristics. Those of ordinary skill in the art are aware that modifications in the amino acid sequence of a peptide, polypeptide, or protein can result in equivalent, or possibly improved, second generation peptides, etc., that display equivalent or superior fimctional characteristics when compared to the original amino acid sequence. The present invention accordingly encompasses such modified amino acid sequences. Alterations can include amino acid insertions, deletions, substitutions, truncations, fusions, shuffling of subunit sequences, and the like, provided that the peptide sequences produced by such modifications have substantially the same functional properties as the naturally occurring counterpart sequences disclosed herein.<br>
One factor that can be considered in making such changes is the hydropathic index of amino acids. The importance of the hydropathic amino acid index in conferring interactive biological fimction on a protein has been discussed by Kyte and Doohttle (J. Mol. BioL, 157: 105-132,1982). It is accepted that the relative hydropathic character of amino acids contributes to the secondary structure of the resultant protein. This, in turn, affects the interaction of the protein with molecules such as enzymes, substrates, receptors, DNA, antibodies, antigens, etc.<br>
Based on its hydrophobicity and charge characteristics, each amino acid has been assigned a hydropathic index as follows: isoleucine (+4.5); valine (+4.2); leucine (+3.8); phenylalanine (+2.8); cysteine/cystine (+2.5); methionine (+1.9); alanine (+1.8); glycine (-0.4); threonine (-0.7); serine (-0.8); tryptophan (-0.9); tyrosine (-1.3); proline (-1.6); histidine (-3.2); glutamate/glulamine/aspartate/asparagine (-3.5); lysine (-3.9); and arginine (-4.5).<br>
As is known in the art, certain amino acids in a peptide or protein can be substituted for other amino acids having a similar hydropathic index or score and produce a resultant peptide or protein having similar biological activity, i.e., which still retains biological functionality. In making such changes, it is preferable that amino acids having hydropathic indices within Â±2 are substituted for one another. More preferred substitutions are those wherein the amino acids have hydropathic indices within Â±1. Most preferred substitutions are those wherein the amino acids have hydropathic indices within Â±0.5.<br><br>
Like amino acids can also be substituted on the basis of hydrophilicity. U.S. Patent No. 4,554,101 discloses that the greatest local average hydrophilicity ofa protein, as governed by the hydrophihcity of its adjacent amino acids, correlates with a biological property of the protein. The following hydrophilicity values have been assigned to amino acids: arginine/lysine (+3.0); aspartate/glulamate (+3.0 Â±1); serine (+0.3); asparagine/glutamine (+0.2); glycine (0); threonine (-0.4); proline (-0.5 Â±1); alanine/histidine (-0.5); cysteine (-1.0); methionine (-1.3); valine (-1.5); leucine/isoleucine (-1.8); tyrosme (-2.3); phenylalanine (-2.5); and tryptophan (-3.4). Thus, one amino acid in a peptide, polypeptide, or protein can be substituted by another amino acid having a similar hydrophilicity score and still produce a resultant protein having similar biological activity, i.e., still retaining correct biological fimction. In making such changes, amino acids having hydropathic indices within Â±2 are preferably substituted for one another, those within Â±1 are more preferred, and those within iO.5 are most preferred.<br>
As outlined above, amino acid substitutions in the peptides of the present invention can be based on e relative similarity of the amino acid side-chain substituents, for example, their hydrophobicity, hydrophilicity, charge, size, etc. Exemplary substitutions that take various of the foregoing characteristics into consideration in order to produce conservative amino acid changes resulting in silent changes within the present peptides, etc., can be selected from other members of the class to which the naturally occurring amino acid belongs. Ar. ino acids can be divided into the following four groups: (1) acidic amino acids; (2) basic amino acids; (3) neutral polar amino acids; and (4) neutral non-polar amino acids. Representative amino acids within these various groups include, but are not limited to: (1) acidic (negatively charged) amino acids such as aspartic acid and glutamic acid; (2) basic (positively charged) amino acids such as arginine, histidine, and lysine; (3) neutral polar amino acids such as glycine, serine, threonine, cysteine, cystine, tyrosine, asparagine, and glutamine; and (4) neutral non-polar amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan, and methionine. It should be noted that changes which are not expected to be advantageous can also be useful if these result in the production of functional sequences.<br>
Variants that are fragments of the polypeptides of the invention can be used to produce the corresponding full length polypeptide by peptide synthesis. Therefore, these variants can be used as iniemiediates for producing the full-length polypeptides of the invention.<br><br>
The polynucleotides and polypeptides of the invention can be used, for example, in the transformation of host cells, such as plant cells, animal cells, yeast cells, bacteria, bacteriophage, and viruses, as fimher discussed herein.<br>
The invention also provides polynucleotides that encode a polypeptide that is a mature protein plus additional amino or carboxyl-terminai amino acids, or amino acids within the mature polypeptide (for example, when the mature form of the protein has more than one polypeptide chain). Such sequences can, for example, play a role in the processing of a protein from a precursor to a mature form, allow protein transport, shorten or lengthen protein half-life, or facilitate manipulation of the protein in assays or production. It is contemplated that cellular enzymes can be used to remove any additional amino acids from the mature protein.<br>
A precursor protein, having the mature fonn of the polypeptide fused to one or more prosequences may be an inactive form of the polypeptide. The inactive precursors generally are activated when the prosequences are removed. Some or all of the prosequences may be removed prior to activation. Such precursor protein are generally called proproteins.<br>
Preparatioa of Expression Constructs and Methods of Use<br>
Of interest is the use of the nucleotide sequences in recombinant DNA constructs to direct the transcription or transcription and translation (expression) of the acyltransferase sequences of the present invention in a host cell. Of particular interest is the use of the polynucleotide sequences of the present invention in recombinant DNA constructs to direct the transcription or transcription and translation (expression) of the acyldransferase sequences of the present invention in a host plant cell.<br>
The expression constructs generally comprise a regulatory sequence functional in a host cell operably linked to a nucleic acid sequence encoding a lecithin:cholesterol acyltransferase-like polypeptide or acyl CoA:cholesterol acyltransferase-like polypeptide of the present invention and a transcriptional termination region fionctional in a host plant cell. Of particular interest is the use of promoters (also referred to as transcriptional initiation regions) functional in plant host cells.<br>
Those skilled in the art will recognize that there are a number of promoters which are fimctional in plant cells, and have been described in the Hterature including constitutive, inducible, tissue specific, organelle specific, developmentally regulated and environmentally regulated promoters. Chloroplast and plastid specific promoters,<br><br>
chioropiasl or plastid functional promoters, and chloroplast or plastid operable promoters are also envisioned.<br>
One set of promoters are constitutive promoters such as the CaMV35S or FMV35S promoters that yield high levels of expression in most plant organs. Enhanced or duplicated versions of the CaMV35S and FMV35S promoters are useful in the practice of this invention (Odell, el ai. (1985) Nature 313:810-812; Rogers, U.S. Patent Number 5,378, 619). Other usefiil corutitutive promoters include, but are not limited to, the mannopine synthase (mas) promoter, the nopaline synthase (nos) promoter, and the octopine synthase (ocs) promoter.<br>
Useiiil inducible promoters include heat-shock promoters (Ou-Lee et ai. (1986) Proc. Natl. Acad. Sci. USA S3: 6815; Ainley et al. (1990) Plant Mol BioL 14: 949), a nitrate-inducible promoter derived from the spinach nitrite reductase gene (Baclc el al.<br>
(1991)	Plant Mol. Biol. 17: 9), hormone-inducible promoters (Yanmguchi-ShinozEdd et al<br>
(1990) Plant Mol. Biol. 15: 905; Kares et al. (1990) Plant Mol. Biol 13: 905), and<br>
lighl-inducible promoters associated with the small subunit of RuBP carboxylase and<br>
LHCP gene families (Kuhlemeier et al. (1989) Plant Cell \: All; Feinbaum et al. (1991)<br>
Moi Gen. Gener. 226: 449; Weisshaar etai. (1991)Â£Jl/5C)7: 10: 1777; Lam and Chua<br>
(1990) Science 248: 471; Castresana el al. (1988) EMSOJ. 7:1929; Schuize-Lefert et al.<br>
{199) EMBO J. %: 651).<br>
In addition, it may also be preferred to bring about expression of the acyitransferase gene in specific tissues of the plant, such as leaf, stem, root, tuber, seed, fruit, etc., and the promoter chosen should have the desired tissue and developmental specificity. Examples of useful tissue-specific, developmenlally-regulated promoters include fruit-specific promoters such as the E4 promoter (Cordes et al. (1989) Plant Cell 1:1025), the E8 promoter (Deikman et al. (1988) EMBO J. 7: 3315), the kiwifruit actinidin promoter (Lin et al. (1993) PNAS 90: 5939), the 2A11 promoter (Houck et al., U.S. Patent 4,943,674), and the tomato pZI30 promoter (U.S. Patents 5,175, 095 and 5,530,185); the p-congiycinin 7S promoter (Doyle et al. (1986) J. Biol. Chem. 261: 9228; Shghton and Beachy (1987) Planta 172: 356), and seed-specific promoters (Knufzon et al.<br>
(1992)	Proc. Natl. Acad Sci. USA 89: 2624; Bnstos et al. (1991) EMBO J. 10: 1469; Lam<br>
and Chua (1991) J. Biol. Chem. 266:17131; Stayton et al. (1991) Aust. J. Plant. Physiol.<br>
18: 507). Fruit-specific gene regulation Is discussed in U.S. Patent 5,753,475. Other<br>
useful seed-specific promoters include, but are not limited to, the napin, phaseoiin, zein,<br>
soybean trypsin inhibitor, 7S, ADR12, ACP, stearoyl-ACP desaturase, oleosin,<br><br>
Lasquerella hydroxylase, and barley aldose reductase promoters (Bartels (1995) Plant J. 7: 809-822), the EA9 promoter (U.S. Patent 5,420,034), and the Bce4 promoter (U.S. Patent 5,530,194). Useful embryo-specific promoters include the com globulin 1 and oieosin promoters- Useful endosperm-specific promoters include the rice glutelin-l promoter, the promoters for the low-pl 3 amylase gene (Amy32b) (Rogers et al. (1984) J. Biol Chem. 259: 12234), the high-pl j3 amylase gene (Amy 64) (Khurseed et al. (1988) J. Biol. Chem. 263:18953), and the promoter for abarley thiol protease gene ("Aleurain") (Whittier et al. (1987) Nucleic Acids Res. 15:2515).<br>
Of particular interest is the expression of the nucleic acid sequences of the present invention from transcription initiation regions which are preferentiaily expressed in a plant seed tissue. Examples of such seed preferential transcription initiation sequences include those sequences derived from sequences encoding plant storage protein genes or from genes involved in fatty acid biosynthesis in oilseeds. Examples of such promoters include the 5" regulatory regions from such genes asnapin (Kridl ei al., Seed Set Res. /;209:219 (1991)), phaseolin, zein, soybean trypsin inhibitor, ACP, slearoyl-ACP desaturase, soybean a" subunitof P-conglycinin (soy 7s, (Chen era/., Proc. Natl. Acad. Sci., 83:8560-8564 (1986))) and oieosin. Seed-specific gene regulation is discussed in EP 0 255 378 BI and U.S. Patents 5,420,034 and 5,608,152 . Promoter hybrids can also be constructed to enhance transcriptional activity (Hoffinan, U.S. Patent No. 5,106,739), or to combine desired transcriptional activity and tissue specificity.<br>
It may be advantageous to direct the localization of proteins conferring LCAT to a particular subcellular compartment, for example, to the mitochondrion, endoplasmic reticulum, vacuoles, chloroplast or other plastidic compartment. For example, where the genes of interest of the present invention will be targeted to plastids, such as chloroplasts, for expression, the constructs will also employ the use of sequences to direct the gene to the plastid. Such sequences are referred to herein as chloroplast transit peptides (CTP) or plastid a-ansit peptides (PTP). hi this manner, where the gene of interest is not directly inserted into the plastid, the expression construct will additionally contain a gene encoding a transit peptide to direct the gene of interest to the plastid. The chloroplast transit peptides may be derived from the gene of interest, or may be derived from a heterologous sequence having a CTP. Such transit peptides are known in the art. See, for example, Von Heijnee/a/. (1991)PlantMoL Biol. Rep. 9:104-126; Clarke?a/. {19S9) J. Biol. Chem. 264:17544-17550; della-Cioppa et al. (1987) Plant Physiol. 54:965-968; Romer el al.<br><br>
(1993) Biochem. Biophys. Res Commun. /96:1414-1421; and. Shah et al. (1986) Science 253:478-481.<br>
Depending upon the intended use, the constructs may contain the nucleic acid sequence which encodes the entire LCAT protein, a portion of the LCAT protein, the entire ACAT protein, or a portion of the ACAT protem. For example, where antisense inhibition of a given LCAT or ACAT protein is desired, the entire sequence is not required. Furthermore, where LCAT or ACAT sequences used in constructs are intended for use as probes, it may be advantageous to prepare constructs containing only a particular portion of a LCAT or ACAT encoding sequence, for example a sequence which is discovered to encode a highly conserved region.<br>
The skilled artisan will recognize that there are various methods for the inhibition of expression of endogenous sequences in a host cell. Such methods include, but are not limited to antisense suppression (Smith, et al. (1988) Nature 334:724-726), co-suppression (NapoU, et a/. (1989) P/ant Ce//2-.279-289), ribozymes (PCT Publication WO 97/10328), and combinations of sense and antisense Waterhouse, el al. (1998) Proc. Natl. Acad. Sci. USA 95:13959-13964.   Methods for the suppression of endogenous sequences in a host cell typically employ the transcription or transcription and translation of at least a portion of the sequence to be suppressed. Such sequences may be homologous to coding as well as non-coding regions of the endogenous sequence.<br>
Regulatory transcript termination regions may be provided in plant expression constructs of this mvention as well. Transcript termination regions may be provided by the DNA sequence encoding the diacylglycerol acyllransferase or a convenient transcription termination region derived from a different gene source, for example, the n-anscript termination region which is naturally associated with the transcript initiation region. The skilled artisan will recognize that any convenient transcript termination region which is capable of terminating transcription in a plant cell may be employed in the constructs of the present invention.<br>
Alternatively, constructs may be prepared to direct the expression of the LCAT or ACAT sequences directly from the host plant cell plastid. Such constructs and methods are known in the art and are generally described, for example, in Svab, et al. (1990) Proc. Natl. Acad. Sci. USA 87:8526-8530 and Svab and Maliga (1993) Proc. Natl. Acad Sci. USA 90:913-917 and in U.S. Patent Number 5,693,507.<br>
A plant cell, tissue, organ, or plant into which the recombinant DNA coi\stnicts containing the expression constructs have been introduced is considered transformed.<br><br>
transfected, or transgenic. A transgenic or transformed cell or plant also includes progeny of the cell or plant and progeny produced from a breeding program employing such a transgenic plant as a parent in a cross and exhibiting an altered phenotype resulting from the presence of a LCAT nucleic acid sequence.<br>
Plant expression or transcription constructs having a plant LCAT as the DNA sequence of interest for increased or decreased expression thereof may be employed with a wide variety of plant life, particularly, plant life involved in the production of vegetable oils for edible and industrial uses. Most especially preferred are temperate oilseed crops. Plants of interest include, but are not limited to, rapeseed (Canola and High Erucic Acid varieties), sunflower, safilower, cotton, soybean, peanut, coconut and oil palms, and com. Depending on the method for introducing the recombinant constructs into the host cell, other DNA sequences may be required. Importantly, this invention is applicable to dicotyledyons and monocotyledons species alike and will be readily applicable to new and/or improved transformation and regulation techniques.<br>
Of particular interest, is the use of plant LCAT and ACAT constructs in plants to produce plants or plant parts, including, but not limited to leaves, stems, roots, reproductive, and seed, with a modified content of lipid and/or sterol esters and to alter the oil production by such plants.<br>
Of particular interest in the present invention, is the use of ACAT genes in conjunction with the LCAT sequences to increase the sterol content of seeds. Thus, overexpression of a nucleic acid sequence encoding an ACAT and LCAT in an oilseed crop may find use in the present invention to increase sterol levels in plant tissues and/or increase oil production.<br>
It is contemplated that the gene sequences may be synthesized, either completely or in part, especially where it is desirable to provide plant-preferred sequences. Thus, all or a portion of the desired structural gene (that portion of the gene which encodes the LCAT or ACAT protein) may be synthesized using codons preferred by a selected host. Host-preferred codons may be determined, for example, from the codons used most frequently in the proteins expressed in a desired host species.<br>
One skilled in the art will readily recognize that antibody preparations, nucleic acid probes (DNA and RNA) and the like may be prepared and used to screen and recover "homologous" or "related" sequences from a variety of plant sources. Homologous sequences are found when there is an identity of sequence, which may be determined upon-comparison of sequence information, nucleic acid or amino acid, or through hybridization<br><br>
reactions between a known LCAT and a candidate source, Conservative clianges, such as Glu/Asp, Val/rie, Ser/Thr, Arg/Lys and Gln/Asn may also be considered in determining sequence homology. Amino acid sequences are considered homologous by as lirtie as 25% sequence identity between the two complete mature proteins. (See generally, Doolittle, R.F., OF URFS and ORFS (Umversity Science Books, CA, 1986.)<br>
Thus, other LCATs may be obtained &amp;om the specific sequences provided herein. Furthermore, it will be apparent that one can obtain natural and synthetic sequences, including modified amino acid sequences and starting materials for synthetic-protein modeling from the exemplified LCAT and ACAT sequences and from sequences which are obtained through the use of such exemplified sequences. Modified amino acid sequences include sequences which have been mutated, truncated, increased and the like, whether such sequences were partially or wholly synthesized. Sequences which are actually purified fiom plant preparations or are identical or encode identical proteins thereto, regardless of the method used to obtain the protein or sequence, are equally considered naturally derived.<br>
For immimologicai screening, antibodies to the protein can be prepared by injecting rabbits or mice with the purified protein or portion thereof, such methods of preparing antibodies being well known to those in the art. Either monoclonal or polyclonal antibodies can be produced, although typically polyclonal antibodies are more useful for gene isolation. Western analysis may be conducted to determine that a related protein is present in a crude extract of the desired plant species, as detertnined by cross-reaction with the antibodies to the encoded proteins. When cross-reactivity is observed, genes encoding the related proteins are isolated by screening expression libraries representing the desired plant species. Expression hbraries can be constructed in a variety of commercially available vectors, including lambda gtl 1, as described in Sambrook, et ai (Molecular Cloning: A Laboratory Manual, Second Edition (1989) Cold Spring Harbor Laboratory, Cold Spring Harbor, New York).<br>
To confirm the activity and specificity of the proteins encoded by the identified nucleic acid sequences as acyltransferase enzymes, in vitro assays are performed in insect cell cultures using baculovinis expression systems. Such baculovirus expression systems are known in the art and are described by Lee, et ai U.S. Patent Number 5,348,886, the entirety of which is herein incorporated by reference.<br>
In addition, other expression constructs may be prepared to assay for protein activity utilizing different expression systems. Such expression constructs are transformed<br><br>
into yeast or prokaryotic host and assayed for acyltransferase activity. Such expression systems are known in the art and are readily available through commercial sources.<br>
The method of transfomiation in obtaining such transgenic plants is not critical to the instant invention, and various methods of plant transfonnation are currently available. Furthermore, as newer methods become available to transform crops, they may also be directly applied hereunder. For example, many plant species naturally susceptible to Agrobacterium infection may be successfully transformed via tripartite or binary vector methods of Agrobacierium mediated transformation, hi many instances, it will be desirable to have the construct bordered on one or both sides by T-DNA, particularly having the left and right borders, more particularly the right border. This is particularly useful when the construct uses A. tumefaciens ox A. rhizogenes as a mode for transformation, although the T-DNA borders may find use with other modes of transformation, hi addition, techniques of microinjection, DNA particle bombardment, and electroporation have been developed which allow for the transformation of various monocot and dicot plant species.<br>
Normally, included with the DMA construct will be a structural gene having the necessary regulatory regions for expression in a host and providing for selection of transformani ceils. The gene may provide for resistance to a cytotoxic agent, e.g. antibiotic, heavy metal, toxin, etc., complementation providing prototrophy to an auxotrophic host, viral immimity or the like. Depending upon the number of different host species the expression construct or components thereof are introduced, one or more markers may be employed, where different conditions for selection are used for the different hosts.<br>
Non-hmiting examples of suitable selection markers include genes that confer resistance to bleomycin, gentamycin, glyphosate, hygromycin, kanamycin, methotrexate, phleomycin, phosphinotricin, spectinomycin, streptomycin, sulfonamide and sulfonylureas. Maliga et al.. Methods in Plant Molecular Biology, Cold Spring Harbor Laboratory Press, 1995, p. 39. Examples of markers include, but are not limited to, alkaline phosphatase (AP), myc, hemagglutinin (HA), p glucuronidase (GUS), luciferase, and green fluorescent protein (GFP).<br>
Where Agrobacterium is used for plant cell transformation, a vector may be used which may be introduced Into the Agrobacterium host for homologous recombination with T-DNA or the Ti- or Ri-plasmid present in the Agrobacterium host. The Ti- or Ri-plasmid containing the T-DNA for recombination may be armed (capable of causing gall<br><br>
formation) or disarmed (incapable of causing gall formation), the latter being pemiissible, so long as the vir genes are present in the transformed Agrobacterium host. The armed plasmid can give a mixture of normal plant ceils and gall.<br>
In some instances where Agrobacterium is used as the vehicle for transforming ?     host plant cells, the expression or transcription construct bordered by the T-DNA border region(s) will be inserted into a broad host range vector capable of replication in E. coii and Agrobacterium, there being broad host range vectors described in the literature. Conamonly used is pRK2 or derivatives thereof See, for example, Ditta, et al, (Proc. Nat. Acad. ScL. U.S.A. (1980) 77:7347-7351) and EPA 0 120 515, which are incorporated I      herein by reference. Alternatively, one may insert the sequences to be expressed in plant cells into a vector containing separate replication sequences, one of which stabilizes the vector in E. coli, and the other in Agrobacterium. See, for example, McBride and Summerfelt (Plant Mol. Biol. (1990) 14:269"276), wherein the pRiHRI (Jouanin, et al.. Mol. Gen. Genet. (1985) 201:370-374) origin of replication is utilized and provides for added stability of the plant expression vectors Ln host Agrobacterium cells.<br>
Included with the expression construct and the T-DNA can be one or more markera, which allow for selection of transformed Agrobacterium and transformed plant cells. A<br>
number of markers have been developed for use with plant cells, such as resistance to chloramphenicol, kanamycin, the aminoglycoside G4-I8, hygromycin, or the like. The particular marker employed is not essential to this invention, one or another marker being prefened depending on the particular host and the manner of construction.<br>
For transformation of plant cells using Agrobacterium, explants may be combined and incubated with the transformed Agrobacterium for sufficient time for transformation, the bacteria killed, and the plant ccUs cultured in an appropriate selective medium. Once callus forms, shoot formadon can be encouraged by employing the appropriate plant hormones in accordance with known methods and the shoots transferred to rooting medium for regeneration of plants. The plants may then be grown to seed and the seed used to estabUsh repetitive generations and for isolation of vegetable oils.<br>
Thus, in another aspect of the present invention, methods for modifying the sterol and/or stanol composition of a host cell. Of particular interest are methods for modifying the sterol and/or stanol composition of a host plant cell. In general the methods involve eidier increasing the levels of sterol ester compounds as a proportion of the total sterol<br><br>
compounds. The method generally comprises the use of expression constructs to direct the expression of the polynucleotides of the present invention in a host ceil.<br>
Also provided are methods for reducing the proportion of sterol ester compounds as a percentage of total sterol compounds in a host plant cell. The method generally comprises the use of expression constructs to direct the suppression of endogenous acyltransferase proteins in a host cell.<br>
Of particular interest is the use of expression constructs to modify the levels of sterol compounds in a host plant cell. Most particular, the methods find use in modifying the levels of sterol compounds in seed oils obtained from plant seeds.<br>
Also of interest is the use of expression constructs of the present invention to alter oil production in a host cell and in particular to increase oil production. Of particular interest is the use of expression constructs containing nucleic acid sequences encoding LCAT and/or ACAT polypeptides to transform host plant cells and to use these host cells to regenerate whole planis having increase oil production as compared to the same plant not containing the expression construct.<br>
The oils obtained from transgenic plants having modified sterol compound content find use in a wide variety of applications. Of particular interest in the present invention is the use of Uie oils containing modified levels of sterol compounds in applications involved in improving human nutrition and cardiovascular health. For example, phytostanols are beneficial for lowering serum cholesterol (Ling, et al (1995) Life Sciences 57:195-206).<br>
Cholesterol-lowering compositions comprise the oils and sterol ester compound compositions obtained using the methods of the present invention. Such cholesterol lowering compositions include, but are not limited to foods, food products, processed foods, food ingredients, food additive compositions, or dietary/nutritional supplements that contain oils and/or fats. Non-limiting examples include margarines; butters; shortenings; cooking oils; frying oils; dressings, such as salad dressings; spreads; mayonnaises; aiKi vitamin/mineral supplements. Patent documents relating to such compositions include, U.S. Patents 4,588,717 and 5,244,887, and PCT International Publication Nos. WO 96/38047, WO 97/42830, WO 98/06405, and WO 98/06714. Additional non-limiting examples include toppings; dairy products such as cheese and processed cheese; processed meat; pastas; sauces; cereals; desserts, mcludlng frozen and shelf-stable desserts; dips; chips; baked goods; pastries; cookies; snack bars; confections; chocolates; beverages; unextracted seed; and unextracted seed that has been<br><br>
ground, cracked, milled, rolled, extruded, pelleted, defatted, dehydrated, or oiherwise processed, but which still contains the oils, etc., disclosed herein.<br>
The cholesterol-lowering compositions can also take the form of pharmaceuricai compositions comprising a cholesterol-lowering effective amount of the oils or sterol compound compositions obtained using the methods of the present invention, along with a pharmaceutically acceptable carrier, excipient, or diluent. These pharmaceutical compositions can be in the form of a liquid or a solid. Liquids can be solutions or suspensions; solids can be in the form of a powder, a granule, a pill, a tablet, a gel, or an extrudate. U.S. Patent 5,270,041 relates to sterol-containing pharmaceutical compositions.<br>
Thus, by expression of the nucleic acid sequences encoding acyltransferase-iike sequences of the present mvention in a host cell, it is possible to modify the lipid content and/or composition as well as the sterol content and/or composition of the host ceil.<br>
The inventioa now being generally described, it will be more readily understood by reference to the following examples which are included for purposes of illustration only and are not intended to limit the present invention.<br>
EXAMPLES Example 1: RKA Isolations<br>
Total RNA from the inflorescence and developing seeds of Arabidopsis thaUana was isolated for ixse in construction of complementary (cDNA) libraries. The procedure was an adaptation of the DNA isolation protocol of Webb and Knapp (D.M. Webb and S.J. Knapp, (1990) Plant Molec. Reporter, 8,180-185). The following description assumes the use of Ig fresh weight of tissue. Frozen seed tissue was powdered by grinding under liquid nifrogen. The powder was added to 10ml REC buffer (50mM Tris-HCl, pH 9, 0.8M NaCl, lOmM EDTA, 0.5% w/v CTAB (cetyltrimethyl-ammonium bromide)) along wiih 0.2g insoluble polyvinylpolypyrrolidone, and ground at room temperature. The homogenate was centrifiiged for 5 minutes at 12,000 xg to pellet msoluble material. The resulting supernatant fraction was extracted with chlorofonn, and the top phase was recovered.<br>
The RNA was then precipitated by addition of 1 volume RecP (SOmM Tris-HCL pH9, lOmM EDTA and 0.5% (w/v) CTAB) and collected by brief centrifiigation as before. The RKA pellet was redissoWed in 0.4 ml of IM NaCl. The RNA pellet was redissolved in water and extracted with phenol/chloroform. Sufficient 3M potassium acetate (pH 5) ws added to make the mixture 0.3M in acetate, followed by addition of two volumes of<br><br>
ethanoi to precipitate the RNA. After washing with ethanol, this final RNA precipitate was dissolved in water and stored frozen.<br>
Alternatively, total RNA may be obtained using TRIzol reagent (BRL-Lifetechnologies, Gaithersburg, MD) following the manufacturer"s protocol. The RNA precipitate was dissolved in water and stored frozen.<br>
Example 2: Identification of LCAT Sequences<br>
Searches were performed on a Silicon Graphics Unix computer using additional Bioaccellerator hardware and Gen Web software supplied by Compugen Ltd. This software and hardware enabled the use of the Smith-Waterman algorithm in searching DNA and protein databases using profiles as queries. The program used to query protein databases was profilesearch. This is a search where the query is not a single sequence but a profile based on a multiple alignment of amino acid or nucleic acid sequences. The profile was used to query a sequence data set, i.e., a sequence database. The profile contained all the pertinent information for scoring each position in a sequence, in effect replacing the "scoring matrix" used for the standard query searches. The program used to query nucleotide databases with a protein ptofiie was tprofilesearch. Tprofilesearch searches nucleic acid databases using an amino acid profile query. As the search is running, sequences in the database are translated to amino acid sequences in six reading fi-ames. The output file for tprofilesearch is identical to the output file for profilesearch except for an additional column that indicates liie frame in which the best alignment occurred.<br>
The Smith-Waterman algorithm, (Smith and Waterman (1981)/ Molec. Biol. 147:195-197), was used to search for similarities between one sequence from the query and a group of sequences contained in the database.<br>
A protein sequence of Lecithin: cholesterol acyltransferase from human (McLean J, etal. (1986) Nucleic Acids Res. 14(23):93 97-406 SEQ ID N0:1)) was used to search the NCBI non-redundant protein database using BLAST. Three sequences were identified from Arabidopsis, GenBank accessions AC004557 (referred to herein as LCATl, SEQ ID N0:2), AC003027 (referred to herein as LCAT2, SEQ ID N0:4), and AL024486 (referred to herein as LCAT3, SEQ K) N0:6). The deduced amino acid sequences are provided in SEQ ID NOs: 3, 5, and 7, respectively.<br>
The profile generated fiom the queries using PSI-BLAST was excised from the hyper text markup language (html) file. The worldwide web (www)/html interface to<br><br>
psiblast at ncbi stores the current generated profile matrix in a hidden field in the html file that is returned after each iteration of psiblast. However, this matrix has been encoded into string62 (s62) format for ease of transport through html. String62 format is a simple conversion of the values of the matrix into html legal ascii characters.<br>
The encoded matrix width (x axis) is 26 characters, and comprise the consensus characters, the probabilities of each amino acid in the order A3,C J&gt;,E,F,G Ji,I,K.,L,M,N, P,Q,R,S,T,V,W,X,Y,Z (where B represents D and N, and Z represents Q and E, and X represents any amino acid), gap creation value, and gap extension value.<br>
The length (y axis) of the matrix corresponds to the length of the sequences identified by PSI-BLAST. The order of the amino acids corresponds to the conserved amino acid sequence of the sequences identified using PSI-BLAST, widi the N-terminal end at the top of the matrix. The probabilities of otiier amino acids at that position are represented for each amino acid along the x axis, below the respective single letter amino acid abbreviation.<br>
Thus, each row of the profile consists of the highest scoring (consensus) amino acid, followed by the scores for each possible amino acid at that position in sequence matrix, the score for opening a gap that that position, and the score for continuing a gap at that position.<br>
The string62 file is converted back into a profile for use in subsequent searches. The gap open field is set to U and the gap extension field is set to 1 along the x axis. The gap creation and gap extension values are known, based on the settings given to the PSIÂ¬BLAST algorithm. The matrix is exported to the standard GCG profile fonn. This format can be read by Gen Web.<br>
The algorithm used to convert the string62 formatted file to the matrix is outlined<br>
in Table 1.	"<br><br>
Table 1<br>
1.	if encoded characrer 2 then the value is blast score min<br>
2.	if encoded character Z then the value is blast score max<br>
3.	else if the encoded character is uppercase then its value is (64-(asczi # of char))<br>
4.	else if the encoded character is a digit the value is ((ascii # of char)-48)<br>
5.	else if the encoded character is not uppercase then the value is ((ascii # of char) - 87)<br>
6.	ALL B positions are set to min of D and N amino acids at that row in sequence matrix<br>
7.	ALL Z positions are set to min of Q amd E amino acids at that row in sequence matrix<br>
8.	ALL X positions are set to min of all amino acids at that row in sequence matrix<br>
9.	kBLAST_SCORE_MAX=999;<br>
10.	kBLAST_SCORE_MIN=-999;<br>
U. all gap opens are set to 11<br>
12. all gap lens are set to 1<br>
The protein sequences ofLCATl, LCAT2, and LCAT3 as well as the PSI-BLAST profile were used to search public and proprietary databases for additional LCAT sequences. Two EST sequences were identified which appear to be identical to LCATi and LCAT3, respectively. One additional Arabidopsis sequence was identified from the proprietary databases, LCAT4 (SEQ ID NO".S). The deduced protein sequence of LCAT4 is provided in SEQ 10 N0-.9. Two additional genomic sequences were identified using the PSI-BLAST profile from libraries of Arabidopsis ecotypes Columbia and Landsberg, LCAT7 (SEQ ID NO-.IO) and LCAT8 (SEQ ID NO:Ii). The LCAT7 sequence was present in both the Columbia and Landsberg genomic libraries, while the LCAT8 sequence was only present in the Columbia library.<br>
An open reading frame was predicted from the genomic sequence of LCAT7 in the Arabidopsis public database and this sequence was called MSH12 (referred to herein as LCAT5, SEQ ID NO; 73). The deduced protein sequence of LCAT5 is provided in SEQ ID NO". 74.<br>
The PSI-BLAST profile and the LCAT sequences were used to query the pubhc yeast database and proprietary libraries containing com and soy EST sequences. The yeast genome contains only one gene, LROl (LCAT Related Open reading frame, YNR008W, Figure I) with distinct similarity to the human LCAT. The DNA sequence of LROl is<br><br>
provided in SEQ ID NO: 75 and the protein sequence is provided in SEQ ID NO: 76. Seven EST sequences were identified fi-om soybean libraries as being LCAT sequences. Two sequences from soy (SEQ ID NOs: 12 and 13) are most closely related to the Arabidopsis LCATl sequence, a single sequence was identified as being most closely related to LCAT2 (SEQ ID NO; 14), three were closely related to LCAT3 (SEQ ID NOs: 15-17), and an additional single sequence was identified (SEQ ID NO: 18). A total of 11 com EST sequences were identified as being related to \hs Arabidopsis LCAT sequences. Two com EST sequences (SEQ ID NOs: 19 and 20) were most closely related to LCATl, two sequences were identified as closely related to LCAT2 (SEQ ID NOs: 21 and 22), four com EST sequences were identified as closely related to LCAT3 (SEQ ID NOs: 23-26), and an additional three com EST sequences were also identified (SEQ ID NOs: 27-29).<br>
Example 3: Identification of AC AT Sequences<br>
Since plant ACATs are unknown in the art, searches were performed to identify known and related ACAT sequences fi"om mammahan sources fi-om public databases. These sequences were then used to search public and proprietary EST databases to identify plant ACAT-like sequences.<br>
A public database containing mouse Expressed Sequence Tag (EST) sequences (dBEST) was searched for ACAT-like sequences. The search identified two sequences (SEQ ID 30and3I) which were related (approximately 20% identical), but divergent, to known ACAT sequences.<br>
In order to identify ACAT-like sequences fim other organisms, the two mouse<br>
ACAT sequences were used to search public and proprietary databases containing EST<br>
/ sequences fi"om human and rat tissues. Results of the search identified several sequences<br>
fi-om the human database and from the rat database which were closely related to the<br>
mouse sequences. The human and rat ACAT-like EST sequences were assembled, using<br>
the GCG assembly program, to construct a complete inferred cDNA sequence by<br>
identifying overlapping sequences (SEQ ID NOs: 32 and 33, respectively).<br>
/	The protein sequence of the human ACAT-like sequence was aligned with known<br>
ACAT sequences from human (Chang, ei al. (1993) J. Biol. Chem. 268:20747-20755, SEQ ID NO:34), mouse (Uehnen, et al. (1995) J. Biol Chem. 270:26192-26201 SEQ ID NO;35) and yeast (Yu, et al (1996) J. Biol Chem. 271:24157-24163, SEQ ID NO:36 and Yang, et al (1996) Science 272:1353-1356, SEQ ED NO:37) using MacVector (Oxford Molecular, Inc.). Results of the alignment demonstrated that the sequence was related to<br><br>
the known sequences, however the related sequence was only about 25% similar to the known sequences.<br>
The protein sequence of the human sterol O-acyltransferase (ACAT, Acyl Co A; Cholesterol acyltransferase, Accession number A48026) related sequence was used to search protein and nucleic acid Genbank databases. A single plant homologue was identified in the public Arabidopsis EST database (Accession A042298, SEQ ID NO:38). The protein sequence (SEQ ID NO:39)was translated from the EST sequence, and was found to contain a peptide sequence conserved in both mammalian and yeast ACATs (Chang et ai, (1997) Ann. Rev. Biochem., 66:613-638).<br>
To obtain the entire coding region corresponding to the Arabidopsis ACAT-like EST, synthetic oligo-nucleotide primers were designed to amplify the 5" and 3" ends of partial cDNA clones containing ACAT-like sequences. Primers were designed according to the Arabidopsis ACAT-like EST sequence and were used in Rapid Amplification of cDNA Ends (RACE) reactions (Frohman et ai (1988) Proc. Natl. Acad. Sci. USA 85:8998-9002).<br>
Primers were designed (5"-TGCAAATTGACGAGCACACCAACCCCTTC-3" (SEQ ID NO:40) and 5"-AAGGATGCTTTGAGTTCCTGACAATAGG-3" (SEQ ID N0:4I)) to amplify the 5" end from the Arabidopsis ACAT EST sequence. Amplification of flanking sequences from cDNA clones were performed using the Marathon cDNA Amplification kit (Clontech, CA).<br>
The sequence derived from the 5"-RACE amphfication was used to search proprietary v4ra6ic?op5w EST libraries. A smgle EST accession, LIB25-088-C7 (SEQ ID NO:42), was identified which contamed a sequence identical to the 5"-RACE sequence. Furthermore, LIB25-088-C7 was found to contain the complete putative coding sequence for the Arabidopsis ACAT-like product.<br>
The nucleic acid as well as the putative translation product sequences of A042298 were used to search public and proprietary databases. Four EST sequences were identified in both soybean (SEQ ID NOs:43-46) and maize (SEQ ID NOs:47-50) proprietary databases, and a single ACAT-like sequence was identified fixjm Mortierrella alpina EST sequences (SEQ ID N0:51).<br>
Sequence alignments between ACAT sequences from several different sources were compared to identify the similarity between the sequences. Nucleotide sequences from known human and mouse ACATs, as well as nucleotide sequences from known yeast ACATs were compared to the ACAT-like EST sequences fiim human and Arabidopsis,<br><br>
Analysis of the sequence ahgmnents revealed several classes of ACATs based on sequence similarity. The known human and mouse ACATs, being 88% similar in the nucleotid"e sequence, formed one class of ACATs. Another class of ACATs included the yeast ACATs which are less than 20% similar to the known human and mouse class ACATs.<br>
The final class of ACATs included the Arabidopsis and human sequences disclosed in the present invention. This class is approximately 22%o similar to the known human and mouse ACAT class and approximately 23% similar to the yeast class of ACATs. Thus, the ACAT sequences disclosed in the present invention represent a novel class of ACAT enzymes. Partial mouse sequences of this class ate also provided.<br>
Example 4: Expression Construct Preparation<br>
Constructs were prepared to direct expression of the LCATI, LCAT2, LCAT3, LQAT4, LCAT5 and the yeast LROl sequences in plants and cultured insect cells. The entire coding region of each LCAT was amplified from the appropriate EST clone or an Arabidopsis genomic cDNA library using the following oligonucleotide primers in a polymerase chain reactions (PCR).   The LCATI coding sequence was amplified from the EST clone Lifa25-082-Ql-El-G4 using the primers<br>
5"-GGATCCGCGGCCGCACAATGAAAAAAATATCTTCACATTATTCGG-3"(SEQ ID NO:52) and 5"-GGATCCCCTGCAGGTCATTCATTGACGGCATTAACATTGG-3" (SEQ ID NO:53). The LCAT2 coding sequence was amplified fim an Arabidopsis genomic cDNA library using the synthetic oligo nucleotide primers 5" -GGATCCGCGGCCGCACAATGGGAGCGAATTCGAAATCAGTAACG-3" (SEQ ID NO:54) and 5"-GGATCCCCTGCAGGTTAATACCCACTrrTATCAAGCTCCC-3" (SEQ ID NO:55). The LCAT3 coding sequence was amplified fix)m the EST clone LIB22-004-Q1-E1-B4 using the synthetic oligo nucleotide primers 5"-GGATCCGCGGCCGCACAATGTCTCTATTACTGGAA GAGATC-3" (SEQ ID NO:56) and 5"-GGATCCCCTGCAGGTTATGCATC AACAGAGACACTTACAGC-3" " (SEQ ID NO-.57) . The LCAT4 coding sequence was amplified from the EST clone LIB23-007-Q1-E1-B5 using the synthetic oligo nucleotide primers 5"-GGATCCGCGGCCGCACAATGGGCTGGATTCCGTGTCCGTGC-3" (SEQ ID NO:58) and 5"-GGATCCCCTGCAGGTTAACCAGAATCAACTACTTTGTG-3" (SEQ ID NO:59). The LCAT5 coding sequence was amplified from LIB23-053-Q1-E1-E3 using the synthetic oUgo nucleotide primers<br><br>
5"-GGATCCGCGGCCGCACAATGCCCCTTATTCATCGG-3" (SEQ ID NO;77) and 5"-GGATCCCCTGCAGGTCACAGCTTCAGGTCAATACG-3" (SEQ ID NO:78).<br>
The yeast LROI coding sequence was amplified from genomic yeast DNA using the synthetic oligo nucleotide primers<br>
5"GGATCCGCGGCCGCACAATGGGCACACTGTTTCGAAG3" (SEQ ID NO:79) and 5"GGATCCCCTGCAGGTTACATTGGGAAGGGCATCTGAG3" (SEQ ID NO:80).<br>
The entire coding region ofHicArabidopsis ACAT sequence (SEQ ID NO: 42) was amplified from the EST clone LIB25-088-C7 using oligonucleotide primers 5"-TCGACCTGCAGGAAGCTTAGAAATGGCGATTTTGGATTC-3" (SEQ ID NO: 60) and S"-GGATCCGCGGCCGCTCATGACATCGATCCTTTTCGG-S" (SEQ ID NO: 61) in a polymerase chain reaction (PCR).<br>
Each resulting PCR product was subcloned into pCR2.1Topo (Invitrogen) and labeled pCGN9964 (LCATl), pCGN9985 (LCAT2), pCGN9965 (LCAT3), pCGN9995 (LCAT4), pCGN10964 (LCAT5), pCGN10963 (LROI), and pCGN8626 (ACAT). Double stranded DNA sequence was obtained to verify that no errors were introduced by the PCR amplification.<br>
4A. Baculovinis Expression Constracts<br>
Constructs are prepared to direct the expression of the Arabidopsis LCAT and yeast LCAT sequences in cultured insect cells. The entire coding region of the LCAT proteins was removed from the respective constructs by digestion with Notl and 5se8387I, followed by gel electrophoresis and gel purification. The fragments containing the LCAT coding sequences were cloned into Noil and Pstl digested baculovinis expression vector pFastBl (Gibco-BRL, Gaithersburg, MD). The resuUing baculovinis expression constructs were referred to as pCGN9992 (LCATl), pCGN9993 (LCAT2), pCGN9994 (LCAT3), pCGN10900 (LCAT4), pCGNI0967 (LCAT5), and pCGNl0962 {LROI).<br>
4B. Plant Expression Construct Preparation<br>
A plasmid containing the napin cassette derived from pCGN3223 (described in U.S. Patent No. 5,639,790, the entirety of which is incorporated herein by reference) was modified to make it more usefiil for cloning large DNA fiagments containing multiple restriction sites, and to allow the cloning of multiple napin fusion genes into plant binary transformation vectors. An adapter comprised of the self annealed oligonucleotide of sequence 5"-<br><br>
/"<br><br>
CGCGATTTAAATGGCGCGCCCTGCAGGCGGCCGCCTGCAGGGCGCGCCATTTA AAT-3" (SEQ ID NO:62) was ligated into the cloning vector pBC SK+ (Stratagene) after digestion with the restriclion endonuclease BssHU to construct vector pCGN7765. Plamids pCGN3223 and pCGN7765 were digested with Noa and ligaied together. The resultant vector, pCGN7770, contained the pCGN7765 backbone with the napin seed specific expression cassette from pCGN3223.<br>
The cloning cassette, pCGN7787, contained essentially the same regulatory elements as pCGN7770, with the exception of the napin regulatory regions of pCGN7770 have been replaced with the double CAMV 35S promoter and the tml polyadenylation and transcriptional termination region.<br>
A binary vector for plant transfonnation, pCGN5139, was constructed from pCGNl558 (McBride and Summerfeit, (1990) Plant Molecular Biology, 14:269-276). In pCGN5139, the polylinker of pCGN1558 was replaced as a HindIII/Asp718 fragment with a polylinker containing unique restriction endonuclease sites, AscI, Pad, Xbal, Swal, BamHI,and Notl. The Asp718 and HindHI restriction endonuclease sites are retained in pCGN5139.<br>
A series of turbo binary vectors was constructed to allow for the rapid cloning of DNA sequences into binary vectors containing transcriptional initiation regions (promoters) and transcriptional termination regions.<br>
The plasmid pCGNS618 was constructed by ligaling oligonucleotides 5"-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3" (SEQrDNO:63) and 5"-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCC-3" (SEQ ID NO:64) into Sall/XhoI-digested pCGN7770. A fragment containing the napin promoter, polylinker and napin 3" region was excised from pCGN86l8 by digestion with Asp71SI; the fragment was blunt-ended by filling in the 5" overhangs with Klenow fragment then ligated into pCGN5139 that had been digested with Asp718I and HindUI and blunt-ended by filling in the 5" overhangs with Klenow fragment. A plasmid containing the insert oriented so that the napin promoter was closest to the blunted Asp71 SI site ofpCGN5139 and the napin 3" was closest to the blunted Hindlll site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8622.<br>
The plasmid pCGN8619 was constructed by ligating oligonucleotides 5"-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCC-3" (SEQ ID NO:65) and<br><br>
5"-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3" (SEQ ID NO:66) into Sall/XhoI-digested pCGN7770. A fragment containing the napin promoter, poiylinker and napin 3" region was removed from pCGN86I9 by digestion with Asp718I; the fragment was blunt-ended by filling in the 5" overhangs with Klenow fragment then ligated into pCGN5139 that had been digested with Asp718I and Hindlll and blunt-ended by filling in the 5" overhangs with KJenow fragment. A plasmid containing the insen oriented so that the napin promoter was closest to the blunted Asp7181 site of pCGN5139 and the napin 3" was closest to the blunted HindlH site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8623.<br>
The plasmid pCGN8620 was constructed by ligating oligonucleotides 5"-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGGAGCT -3" (SEQ ID NO:67) and 5"-CCTGCAGGAAGCTTGCGGCCGCGGATCC-3" (SEQ ID NO:68) into Sall/Sacl-digested pCGN7787. A fragment containing the d35S promoter, poiylinker and tml 3" region was removed from pCGNS620 by complete digestion with Asp7181 and partial digestion with Notl. The fragment was blunt-ended by filling in the 5" overhangs with Klenow fragment then ligated into pCGN5139 that had been digested with Asp7181 and Hindm and blunt-ended by filling in the 5" overiiangs with Klenow fragment. A plasmid containing tiie insert oriented so that the d35S promoter was closest to the blunted Asp7181 site of pCGN5139 and the tml 3" was closest to the blunted Hindlll site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning jimctions. The resulting plasmid was designated pCGNS624.<br>
The plasmid pCGN8621 was constructed by ligating oligonucleotides 5"-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCCAGCT -3" (SEQ ID H0:â‚¬9) and 5"-GGATCCGCGGCCGCAAGCTTCCTGCAGG-3" (SEQ ID NO-.70) into Sall/Sacl-digested pCGN7787. A fragment containing the d35S promoter, poiylinker and tml 3" region was removed from pCGN8621 by complete digestion with Asp718I and partial digestion with Notl. The fragment was blunt-ended by filling m the 5" overhangs with Klenow fragment then ligated into pCGN5139 that had been digested with Asp718I and Hindlll and blunt-ended by filling in the 5" overhangs with Kienow fragment. A plasmid containing the insert oriented so that the d35S promoter was closest to the blunted Asp7I8I site of pCGN5139 and the tml 3" was closest to the blunted HindHI site was subjected to sequence analysis to confirm both the insert orientation and the integrity of :loning junctions. The resulting plasmid was designated pCGN8625.<br><br>
The plasmid construct pCGN8640 is a modification of pCGN8624 described above. A 938bp PstI fragment isolated from iransposon Tn7 which encodes bacterial spectinomycin and streptomycin resistance {Fling et al. (1985), Nucleic Acids Research 13(19);7095-7106), a determinant for E. coli and Agcobacterium selection, was blunt ended with Pfii polymerase. The blunt ended fragment was ligated into pCGN8624 that had been digested with Spel and blunt ended with Pfii polymerase. The region containing the PstI fragment was sequenced to confimi both the insert orientation and the integrity of cloning junctions.<br>
The spectinomycin resistance marker was introduced into pCGN8622 and pCGN8623 as follows. A 7.7 Kbp Avrll-SnaBI fragment from pCGN8640 was ligated to a 10.9 Kbp AvrU-SnaBI fragment from pCGN8623 or pCGN8622, described above. The resulting plasmids were pCGN8641 and pCGN8643, respectively.<br>
The plasmid pCGN8644 was constructed by hgating oligonucleotides 5"-GATCACCTGCAGGAAGCTTGCGGCCGCGGATCCAATGCA-3" (SEQ ID N0:71) and 5"-TTGGATCCGCGGCCGCAAGCTTCCTGCAGGT-3" (SEQ ID NO:72) into BamHI-PstI digested pCGN8640.<br>
4C. Plant LCAT Expression Construct Preparation<br>
The codmg sequence of LCATl was cloned from pCGN9964 as dNotV SseZZZll fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9960, pCGN996U pCGN9962, and pCGN9963, respectively. The construct pCGN9960 was designed to express the LCATl coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN996I was designed to express the LCATl coding sequence in the anlisense orientation from the napin promoter. The construct pCGN9962 was designed to express the LCATl coding sequence in the sense orientation from the napin promoter. The construct pCGN9963 was designed to express the LCATl coding sequence in the antisense orientation from the constitutive promoter CaMV 35S.<br>
The coding sequence of LCAT2 was cloned from pCGN9985 as zNotV &amp;e8387I fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9981, pCGN9982, pCGN9983, and pCGN9984, respectively. The construct pCGN9981 was designed to express the LCAT2 coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN9982 was designed to express the LCAT2 coding sequence in the antisense orientation from the<br><br>
napin promoter. The construct pCGN9983 was designed to express the LCAT2 coding sequence in the sense orientation from the napin promoter. The construct pCGN9984 was designed to express the LCAT2 coding sequence in the antisense orientation from the constitutive promoter CaMV 35S.<br>
The coding sequence of LCAT3 was cloned from pCGN9965 as a NotV SseZIZll fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9966, pCGN9967, pCGN9968, and pCGN9969, respectively. The construct pCGN9966 was designed to express the LCAT3 coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN9967 was designed to express the LCAT3 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9%8 was designed to express the LCAT3 coding sequence in the sense orientation fim the napin promoter. The construct pCGN9969 was designed to express the LCAT3 coding sequence in the antisense orientation from the constitutive promoter CaMV 35S.<br>
The coding sequence of LCAT4 was cloned from pCGN9995 as aA"o/I/ &amp;e8387I fragment into pCGN8640, pCGN864l, pCGN8643, and pCGN8644 to create the expression constructs pCGN9996, pCGN9997, pCGN9998, and pCGN9999, respectively. The construct pCGN9996 was designed to express the LCAT4 coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN9997 was designed to express the LCAT4 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9998 was designed to express the LCAT4 coding sequence in the sense orientation from the napin promoter. The construct pCGN9999 was designed to express the LCAT4 coding sequence in the antisense orientation from the constitutive promoter CaMV 35S.<br>
The coding sequence of LCAT5 was cloned from pCGN10964 asa-NotU Sse%Z%li fragment into pCGN9977 and pCGN9979, to create the expression constructs pCGN10965, and pCGN10966, respectively. The construct pCGNl0965 was designed to express the LCAT5 coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN10966 was designed to express the LCAT5 coding sequence in the sense orientation from the napin promoter.<br>
The coding sequence of LROl was cloned from pCGN10963 as a NotV &amp;e8387I fragment into pCGN9977 and pCGN9979, to create the expression constructs pCGNl0960, and pCGN10961, respectively. The construct pCGN10960 was designed to express the LPOJ coding sequence in the sense orientation fim the constitutive promoter<br><br>
CaMV35S. The construct pCGNl0961 was designed to express the iO/ coding sequence in the sense orientation from the napin promoter.<br>
4D. Plant ACAT Expression Construct Preparation<br>
A fragment containing the Arabidopsis ACAT-Iike coding region was removed fi-om pCGN8626 by digestion with Sse8387I and Not I. The fragment containing the ACAT-Iike sequence was ligated into Pstl-Not I digested pCGN8622. The resulting plasmid was designated pCGN8627. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis ACAT-Iike coding region (SEQ ID NO: 42) was removed from pCGN8626 by digestion with Sse8387I and Not I. The fragment was ligated into Pstl-Not I digested pCGN8623. The resulting plasmid was designated pCGN8628. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis ACAT-Iike coding region was removed from pCGN8626 by digestion with Sse8387 and Not I. The fragment was ligated into Pstl-Not I digested pCGN8624. The resulting plasmid was designated pCGN8629. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis .ACAT-Iike coding region was removed from pCGN8626 by digestion with SseS387 and Not I. The fragment was Ugated into Pstl-Nor I digested pCGN8625. The resulting plasmid was designated gCGN8630. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
An additional expression construct for the suppression of endogenous ACAT-Iike activity was also prepared. The construct pCGN8660 was constructed by cloning approximately 1 Kb of the Arabidopsis ACAT-Iike coding region from pCGNS626 in the sense orientation, and the fiiU-length Arabidopsis ACAT-Iike coding region in the antisense orientation under the regulatory control of the napin transcription initiation sequence.<br>
For expression of the rat ACAT-!ike sequence in plants, the NotI-Sse8387I fragment of pCGN8592 was cloned into Notl-PstI digested binary vectors pCGN862I, pCGN8622, and pCGN8624 to yield plasmids, pCGN 9700, pCGN9701, andpCGN9702, respectively. Plasmid pCGN9700 expresses a sense transcript of the rat ACAT-Iike cDNA under control of a napin promoter, plasmid pCGN9701 expresses an antisense transcript of the rat ACAT-Iike cDNA under control of a napin promoter, and plasmid pCGN9702 expresses a sense transcript of the rat ACAT-Iike cDNA under control of a double 35S<br><br>
promoter. Ptasmids pCGN 9700, pCGN970l, and pCGN9702 were introduced in Agrobacterium tumefaciens EPIAIOI.<br>
Constructs were prepared to direct the expression of the rat ACAT-like sequence in the seed embryo of soybean and the endosperm of com. For expression of the rat ACAT-like DNA sequence in soybean, a 1.5 kb NotVSsefitll fragment from pCGN8592 containing the coding sequence of the rat ACAT-Uke sequence was blunt ended using Mung bean nuclease, and ligated into the Smal site of the turbo 7S binary/cloning vector pCGN8809 to create the vector pCGN8817 for transformation into soybean by particle bombardnient. The vector pCGN8817 contained the operably linked components of the promoter region of the soybean a" subunit of p-conglycinin (7S promoter, (Chen ei a!., (1986), Proc. Natl. Acad. Sci., 83:8560-8564), the DNA sequence coding for the entire rat ACAT-iike protein, and the transcriptional tennination region of pea RuBisCo small subunit, referred to as E9 3" (Coruzzi, et al (1984) EMBOJ. 3:1671-1679 and Morelli, et al. (1985) Nature 315:200-204). This construct further contained sequences for the selection of positive transformed plants by screening for resistance to glyphosate using the CP4 EPSPS (U.S. Patent 5,633,435) expressed under the control of the figwort mosaic viriis (FMV) promoter (U.S. Patent Number 5,378,619) and the transcriptional termination region of E9.<br>
For expression of the rat AC AT-like sequence in the com endosperm, a 1.5 kb NotVSse"&amp;ZZll fragment from pCGN8S92 containing the coding sequence of the rat ACAT-like sequence was blunt ended using Mung bean nuclease, and Ugated into the BamYQ. site of the rice pGtl expression cassette pCGNS592 for expression from the pGtl promoter (Leisy, D.J. et al.. Plant Mol. Biol. 14 (1989) 41-50) and the HSP70 intron sequence (U.S. Patent Number 5,593,874). This cassette also included the transcriptional termination region downstream of the cloning site of nopaline synthase, nos 3" (Depicker et al., J. Molec. Appl. Genet. (1982) 1: 562-573). A 7.5 kb fragment containing the pGtl promoter, the DNA sequence encoding the rat ACAT-like protein, and the nos transcriptional termination sequence was cloned into the binary vector pCGN8816 to create the vector pCGN8818 for transformation into com. This construct also contained sequences for the selection of positive transfonnants with kanamycin using the kanamycin resistance gene from Tn5 bacteria under the control of the CAMV 358 promoter and tml transcriptional termination regions.<br><br>
Example 5: Expression in Insect Cell Culture<br>
A bacuiovirus expression system was used to express the LCAT cDNAs in cultured insect cells.<br>
The bacuiovirus expression constructs pCGN9992, pCGN9993, pCGN9994, pCGN10900, pCGNl0962, and pCGN10967 were transformed and expressed using the BAC-to-BAC Bacuiovirus Expression System (Gibco-BRL, Gaithersburg, MD) according to the manufacturer"s directions.<br>
The transformed insect cells were used to assay for acyltransferase activities using methods known in the art (see Example 8).<br>
Example 6: Plant Transformation<br>
A variety of methods have been developed to insert a DNA sequence of interest into the genome of a plant host to obtain the transcription or transcription and translation of the sequence to effect phenotypic changes. Transgenic plants were obtained by Agrobacterium-mediated transformation as described by Radke et al. {Theor. Appl. Genet. (1988) 75:685-694; Plant Cell Reports (1992) 77:499-505). Alternatively, microprojectile bombardment methods, such as described by Klein et al. {Bio/Technology 70:286-291) may also be used to obtein nuclear transformed plants. Other plant species may be similarly transformed using related techniques.<br>
The plant binary constructs described above were used in plant transformation to direct the expression of the sterol acyltransferases in plant tissues. Binary vector constructs were transformed into strain EHAlOl Agrobacterium cells (Hood et al., J. Bacterial (1986) 7(55:1291-1301), by the method of Holsters etal. (Mol. Gen. Genet. (1978) 7(55:181-187). Transgenic Arabidopsis thaliana plants were obtained by Agrobacterium-mtizi&amp;d transformation as described by Valverkens et al., (Proc. Nat. Acad. Sci. (1988) 55:5536-5540), Bent et al. ((1994), Science 265:1856-1860), and Bechtold et al. ((1993), C. }i.Acad. Sci., Life Sciences 316:1194-1199).<br>
Example 7: Plant Assays for Modified Sterol Content/Profile 7A: NMRofT2Â£eed<br>
Seed from plants expressing LCAT 1 through 4 under the control of the napin promoter were analyzed by NMR. Arabidopsis seeds from transgenic plants were placed directly into wide-mouth MAS NMR sample tubes.<br><br>
High-resolution spectra were measured at II.7 T (1H=500 MHz, 13C=125 mHz) using Varian NMR Instruments (Palo Alto, CA) Inovaâ„¢ NMR spectrometers equipped with carbon-observe MAS Nanoprobesâ„¢. The 13C spectra were acquired without a field-frequency lock at ambient temperahire (prox. 21-22""C) for 14 hours using the following conditions: spectral width = 29.996 kHz, acquisition time = 2.185 seconds, p/2 pulse (3.8 ms) with no relaxation delay, IH g B2 = 2.5 kHz with Waltz decoupling. Data processing conditions were typically: digital resolution = 0.11 Hz, 0.3 to 1.5 Hz line broadening and time-reversed linear prediction of the first three data points. Chemical shifts were referenced by adding neat tetramethylsilane (TMS) to Arabidopsis seeds and using the resulting referencing parameters for subsequent spectra. The 13C resolution was 2-3 Hz for the most narrow seed resonances. Spectral resolution was independent of MAS spinningspeeds(1.5-3.5kHz)anddata were typically obtained with 1.5 kHz spinning speeds. Spinning sidebands were approx. 1% of the main resonance. PhytosteroI13C assignments were based on model samples composed of triolein, P-sitosterol and cholesterol oleate. Triacylglycerol 13C assignments were made fiiam comparison with literature assignments or with shifts computed fi"om a 13C NMR database (Advanced Chemical Development, Inc., version 3.50, Toronto Canada).<br>
The results of these analyses are displayed in Figure 2 and show that there was a trend of an approximately 2 fold increase of phytosterols in the seeds derived from plant line 5 expressing the LCAT 4 gene (pCGN9998) under the control of the napin promoter. During the course of this analysis it was also noted that the average oil content of seed from plants expressing the LCAT2 construct (pCGN9983) under the control of the napin promoter was higher than that of controls. This is the first in planta evidence supporting the concept that ovcrexpression of a nucleotide sequence encoding a lecithinxholesterol acyltransferase-like polypeptide can increase oil content.<br>
7B;HPLC/MSofT2seed<br>
Seed oil from T2 plants expressing LCATl through 4 under the control of the napin promoter was extracted using an accelerated solvent extractor (ASE) method. Seed samples were ground, using a mortar and pestle, to achieve a fine homogeneous meal. Oil was obtained using a Dionex Accelerated Solvent Extractor (ASE). Clean ground seed w added to an equal amount of diatomÂ£Â«;eous earth- The ground seed sample and the diatomaceous earth were thoroughly mixed until a homogeneous texture was achieved.<br><br>
The sample was then loaded into the instrument and oil extraction was achieved using hexane under validated laboratory protocols.<br>
Oil from these seed samples was then analyzed for sterol ester analysis using EIPLC/MS for &amp;ee campesterol, stigmasterol, and sitosterol and their fatty acid esters. To the autosampler vial containing approximately 0.1 grams oil was added 0.3 mLs CDCI3. One-hundred microliters of this solution was added to 900 microliters CHCI3. Five microliters of this diluted sample was subsequently injected into an HPLC/MS with positive ion atmospheric pressure ionization. The individual components in the oils were separated using two 4.6 x 50 mm Cg Zorbax columns in series and a gradient using acetonitrile and acetonitrile with 40% CHClj. The sterol concentrations were calculated assuming each sterol and its fatty acids have the same molar responses. This was observed to be the case widi cholesterol and its esters and was assumed to be the case for campesterol, stigmasterol, and sitosterol. In the present study, the sterol identified as stigmaslerol was actually an isomer of this compound.<br>
The results of these analyses are displayed in Figures 3 and 4 and show that there were sterol ester enhancements on the order of 50%. in the seeds derived from six out of seven T2 plant lines expressing LCAT3 (pCGN9968) under the control of the napin promoter.<br>
Example 8: Baculovirus Insect Cell Culture for Sterol Esterification Activity<br>
Baculovirus expression construct pCGN9992, pCGN9993, pCGN9994 and pCGN10900 (see Example 4) were transfomied and expressed using the BAC-TOBAC Baculovirus Expression System (Gibco-BRL, Gaithersburg, MD) according to the manufacturer"s instruction except harvesting of recombinant viruses was done 5 days post-transfection. The supemalant from the transfection mixture was used for generating virus stock which in turn was used for infecting Sf9 cells used in the assay.<br>
The transformed cells were assayed for lecithin:sterol acyltransferase activities using the method described herein. Insect cells were cennifuged and the resulting cell pellet was either used immediately or stored at -70 C for later analysis. Cells were resuspended in iMedium A (100 mM Tricine/NaOH, pH 7.8,10% (w/v) glycerol, 280 mM NaCl with : 0.1 nM Aprorinin, 1 M Leupeptin, and 100 nM Pefabloc (all from Boehringer Mannheim, Germany) and lysed by sonication (2 x 10 sec). Cell walls and odier debris were pelleted by centrifugation (14,000 x g , 10 min, 4""C). The supernatant<br><br>
was transferred to a new vial and membranes pelleted by centrifugation (100,000 x g, Ti 70.1 rotor, 46,000 rpm for 1 hour at 4Â°C). Total membranes were resuspended in Medium A. Lecithin:sterol acyltransferase activity was assayed in a 0.1 m! reaction mixture containing 100 mM Tris/HCl, pH 7, 28 mM NaCI, 0.03% Triton X-lOO, 0.1 mM sitosterol, 20 iM l,2-["""C]-palinitoyl-phosphalidyi choline (246420 dpm/nmole), and 0.05-20 mg of membrane protein. After 15 minutes at 30 Â°C, thereaction was tenninaied by addinon of a 0.5 ml solution of methylene chloride:methanol (4:1, v/v) containing 100 jig cholesterol and cholesterol ester as cold carriers. A portion (0.1 ml) of the bottom organic layer was removed and evaporated under nitrogen gas. The concentrated extract was resuspended in 30 p.i of hexane and spotted onto a silica gel-G thin layer chromatographic plate. The plate was migrated m hexane:diethyl etherracetic acid (80:20:1) to the lop, then air dried. Radioactivity was determined by exposure to a Low Energy Phosphor-imaging Screen. Following exposure, the screen was read on a phosphorimager.<br>
The LCAT 4 protein from pCGN10900 in baculovinis membranes showed a radioactive spot in the region of the TLC plate where cholesterol ester migrates indicating that LCAT 4 has the ability to catalyze the transfer of an acyl group from lecithin (PC) to sitosterol to make a sitosterol ester.<br>
Example 9: Plant Assay for Modified Lipid Content<br>
Nir (near infixed spectroscopy spectral scanning) can be used to determine the total oil content of Arabidopsis seeds in a non-destructive way provided that a specfral calibration curve has been developed and validated for seed oil content. A seed oil spectral calibartion curve was developed using seed samples from 85 Arabidopsis plants. Seed was cleaned and scanned using a Foss NIR model 6500 (Foss-Nirs Systems, Inc.). Approximately 50 to 100 milligrams of whole seeds, per sample, were packed in a mini sample ring cup with quartz lens [ IH-0307 ] consisting a mini-insert [ IH-0337 ] and scanned in reflectance mode to obtain the spectral data. The seed samples were then ground, using a mortar and pestle, to achieve a fine homogeneous meal. The ground samples were measured for oil using an accelerated solvent extractor (ASE).<br>
Measurement for the total oil content was performed on the Dionex Accelerated Solvent Extractor (ASE). Approximately 500 mg of clean ground seed was weighed to the nearest 0.1 mg onto a 9 x 9 cm weigh boat. An equal amount of diatomaceous earth was added using a top-loading balance accurate to the nearest 0.01 g. The ground seed sample<br><br>
and the diatomaceous earth were thoroughly mixed until a homogeneous texture was achieved. The sample was loaded on to the instrument and oil extraction was achieved using hexane under validated Laboratory protocols. Standard Rapeseed samples were obtained from the Community Bureau of Reference (BCR). The ASE extraction method was validated using the BCR reference standards. A total percent oil recovery of 99% to 100% was achieved. "As-is" oil content was calculated to the nearest 0.01 mass percentage using the formula:<br>
Oil Content = 100% x (vial plus extracted oil wt - initial vial w[) / (sample wt)<br>
The analytical data generated by ASE were used to perform spectral calibrations. Nir calibration equations were generated using the built-in statistical pkage within the NirSytems winisi software. The spectral calibration portion of the software is capable of calibration and self-validation. From a total of 85 samples, 57 samples were used to generate the total percent oil calibration. The remaining samples were used to validate the oil calibrations. Optimized smoothing, derivative size, and mathematical treatment (modified partial least square) was utilized to generate the calibration. The samples that were not used in building respective calibrations were used as a validation set. Statistical tools such as correlation coefficient (R), coefficient of determination (R), standard error of prediction ( SEP ), and the standard error of prediction corrected for bias (SEPC) were used to evaluate the calibration equations.<br>
T2 seeds fi"om plants that had been transformed with the LCAT genes were cleaned and scanned using a Foss NIR model 6500 (Foss-Nirs Systems, Inc.). Approximately 50 to 100 miUigrams of whole seeds, per sample, were packed in a mini sample ring cup with quartz lens [ IH-0307 ] consisting a mini-insert [ IH-0337 ] and scanned in reflectance mode to obtain the spectral data. Oil percentage in each seed sample was determined using the seed oil spectral cahbration curve detailed above.<br>
The results of these analyses are found in Figure 5 and Table 2 and show that there was a significant increase in the oil level in seed from T2 plants expressing the LCAT2 gene. This increase in oil was seen in plants when LCAT2 was driven by either the 35S constitutive promoter or the seed-specific napin promoter. These results show that overexpression of a nucleic acid sequence encoding a lecithinxholesterol acyltransferase-like polypeptide can increase seed oil production in plants.<br><br><br><br><br>
In light of the detailed description of the invention and the examples presented above, it can be appreciated that the several aspects of the invention are achieved.<br>
It is to be underetood that the present invention has been described in detail by way of illustration and example in order to acquaint others skilled in the art with the invention, its principles, and its practical application. Particular formulations and processes of the present invention are not limited to the descriptions of the specific embodiments presented, but rather the descriptions and examples should be viewed in tenns of the claims that follow and their equivalents. While some of the examples and descriptions above include some conclusions about the way the invention may function, the inventors do not intend to be bound by those conclusions and functions, but put them forth only as possible explanations.<br>
It is to be further understood that the specific embodiments of the present invention as set forth are not intended as being exhaustive or limiting of the invention, and that many alternatives, modifications, and variations will be apparent to those of ordinary skill in the art in light of the foregoing examples and detailed description. Accordingly, this invention is intended to embrace all such alternatives, modifications, and variations that fall within the spirit and scope of the following claims.<br><br>
SEQUENCE LISTING  Monsanto Company  PLANT STEROL ACYLTRANSFERASES  MTC671S<br>
;   <br> 60/152,493  1999-08-30<br>   80<br> Patentin Ver. 2.1<br> 1<br> 440<br> PRT<br> 1<br>
Met Gly Pro Pro Gly Ser Pro Trp Gin Trp Val Thr Leu Leu Leu Gly<br>
15	10	IS<br>
Leu Leu Leu Pro Pro Ala Ala Pro Phe Trp Leu Leu Asn Val Leu ?he<br>
20	25	30<br>
Pro Pro His Thr Thr Pro Lys Ala Glu Leu Ser Asn His Thr Arg Pro<br>
35	40	45<br>
Val lie Leu Val Pro Gly Cys Leu Gly Asn Gin Leu Glu Ala Lys Leu<br>
50	55	60<br>
Asp Lys Pro Aep Val Val Asn Trp Met Cys Tyr Arg Lys Thr Glu Asp<br>
65	70	75	aO<br>
Phe Phe Thr He Trp Leu Asp Leu Asn Met Phe Len Pro Leu Gly Val<br>
85	90	95<br>
Asp Cys Trp He Asp Asn Thr Arg Val Val Tyr Asn Arg Ser Ser Gly<br>
100	105	110<br>
Leu Val Ser Asn Ala pro Gly Val Gin He Arg Val Pro Gly Phe Gly<br>
115	120	125<br>
Lys Thr Tyr Ser Val Glu Tyr Leu Asp Ser Ser Lys Leu Ala Gly Tyr<br>
130	135	140<br>
Leu His Thr Leu Val Gin Asn Leu Val Asn Asn Gly Tyr Val Arg Asp<br>
145	150	155	ISO<br>
Glu Thr Val Arg Ala Ala Pro Tyr Asp Trp Arg Leu Glu Pro Gly Gin<br>
165	170	175<br><br>
Gin Glu Glu Tyr Tyr Arg Lys Leu Ala Gly Leu Val Glu Glu Met His<br>
130	185	190<br>
Ala Ala Tvr Gly Lys Pro Val Phe Leu lie Gly His Ser Leu Gly Cys<br>
195	200	205<br>
Leu His Leu Leu Tyr Phe Leu Leu Arg Gin Pro Gin Ala Tirp Lys Asp<br>
210	215	220<br>
Arg Phe He Asp Gly Phe He Ser Leu Gly Ala Pro Trp Gly Gly Ser<br>
225	230	235	20<br>
He Lys Pro Met Leu Val Leu Ala Ser Gly Asp Asn Gin Gly He Pro<br>
245	250	255<br>
He Met Ser Ser He Lys Leu Lys Glu Glu Gin Arg He Thr Thr Thr<br>
260	265	270<br>
Ser Pro Trp Met Phe Pro Ser Arg Met Ala Trp Pro Glu Asp His Val<br>
275	2a0	285<br>
Phe He Ser Thr Pro Ser Phe Asn Tyr Thr Gly Arg Asp Phe Gin Arg<br>
290	295	300<br>
Phe Phe Ala Asp Leu His Phe Glu Glu Gly Trp Tyr Met Trp Leu Gin<br>
305	3X0	315	320<br>
Ser Arg Asp Leu Leu Ala Gly Leu Pro Ala Pro Gly Val Glu Val Tyr<br>
325	330	335<br>
Cys Leu Tyr Gly Val Gly Leu Pro Thr Pro Arg Thr Tyr He Tyr Asp<br>
340	345	350<br>
His Gly Phe Pro Tyr Thr Asp Pro Val Gly Val Leu Tyr Glu Asp Gly<br>
355	360	365<br>
Asp Asp Thr Val Ala Thr Arg Ser Thr Glu Leu Cys Gly Leu Trp Gin<br>
370	375	330<br>
Gly ftxg Gin Pro Gin Pro Val His Leu Leu Pro Leu His Gly He Gin<br>
385	390	395	400<br>
His Leu Asn Met: val Phe Ser Asn Leu Thr Leu Glu His He Asn Ala<br>
405	410	415<br>
He Leu Leu Gly Ala Tyr Arg Gin Gly Pro Pro Ala Ser Pro Thr Ala<br>
420	425	430<br>
Ser Pro Glu Pro Pro Pro Pro Glu<br>
435	440<br><br> 2<br> 1299<br> DNA<br> Arabidopsis thaliana<br> 2<br>
atgaaaaaaa tatcttcaca tcattcggCa gtcatagcga tacCcgttgt ggtgacgatg 60<br>
acctcgatgt gtcaagctgt gggcagcaac gtgtaccctt tgattctggt Cccaggaaac 120<br>
ggaggtaacc agctagaggt acggctggac agagaataca agccaagtag tgcctggtgt ISO<br>
agcagctggt tataCccgat tcataagaag agtggcggac ggtttaggct atggtccgat 240<br>
gcagcagtgt tattgccccc cttcaccagg tgcctcagcg atcgaatgat gctgtacCaC 300<br>
gaccctgatc tggatgatta ccaaaacgct cctggtgtcc aaacccgggt tcctcatttc 360<br>
ggttcgacca aatcacttcc acaccccgac cctcgtctcc gagatgccac atcttacatg 420<br>
gaacatttgg cgaaagctct agagaaaaaa tgcgggtatg ttaacgacca aaccatccta 480<br>
ggagctccat aCgatttcag gtacggcctg gctgcttcgg gccacccgtc ccgtgtagcc 540<br>
tcacagttcc tacaagaccc caaacaattg gtggaaaaaa ctagcagcga gaacgaagga 600<br>
aagccagtga tactcctctc ccatagccta ggaggacttt tcgCcctcca tttccccaac 660<br>
cgtaccaccc cttcatggcg ccgcaagtac aCcaaacact ttgttgcact cgctgcgcca 720<br>
tggggtggga cgacctctca gatgaagaca tttgcttctg gcaacacact cggtgtccct 7B0<br>
ttagttaacc ctttgcCggt cagacggcat cagaggaccc ccgagagtaa ccaatggcta 840<br>
cttccatcta ccaaagtgtt tcacgacaga actaaaccgc ttgtcgtaac tccccaggtt 900<br>
aactacacag cctacgagac ggaccggctc tttgcagaca ttggattccc acaaggagtt 960<br>
gtgccttaca agacaagagt gttgccttta acagaggagc tgatgactcc gggagtgcca 1020<br>
gtcacttgca tatatgggag aggagttgat acaccggagg ttttgatgta tggaaaagga 10 8 0<br>
ggattcgata agcaaccaga gattaagtat ggagatggag atgggacggt taatttggcg 1140<br>
agcttagcag ctttgaaagt cgatagcttg aacaccgtag agattgatgg agtttcgcat 12 00<br>
acatctatac ttaaagacga gatcgcactt aaagagatta tgaagcagat ttcaattatt 12 60<br>
aattatgaat Cagccaatgt taatgccgtc aatgaatga	1299<br> 3<br> 432<br> PRT<br> Arabidopsis thaliana<br> 3<br>
Met Lys Lys lie Ser Ser His Tyr Ser Val Val He Ala lie Leu Val<br>
1	S	10	15<br>
Val Val Thr Met Thr Ser Met Cys Gin Ala Val Gly Ser Asn Val Tyr<br>
20	25	30<br>
Pro Leu He Leu Val Pro Gly Asn Gly Gly Asn Gin Leu Glu Val Arg<br>
35	40	45<br>
Leu Asp Arg Glu Tyr Lys Pro Ser Ser Val Trp Cys Ser Ser Trp Leu<br>
SO	55	60<br>
Tyr Pro He His Lys Lys Ser Gly Gly Trp Phe Arg Leu Trp Phe Asp<br>
65	70	75	aO<br>
Ala Ala Val Leu Leu Ser Pro Phe Thr Arg Cys Phe Ser Asp Arg Met,<br>
S5	90	95<br>
Met Leu Tyr Tyr Asp Pro Asp Leu Asp Asp Tyr Gin Asn Ala Pro Gly<br>
100	105	110<br><br>
Val Gin Thr Arg Val Pro His Phe Gly Ser Thr Lys Ser Leu Leu Tyr<br>
115	120	125<br>
Leu Asp pro Arg Leu Arg Asp Ala Thr Ser Tyr Met Glu His Leu Val<br>
130	135	140<br>
Lys Ala Leu Glu Lys Lys Cys Gly Tyr Val Asn Asp Gin Thr lie Leu<br>
145	150	15S	160<br>
Gly Ala Pro Tyr Asp Phe Arg Tyr Gly Leu Ala Ala Ser Gly His pro<br>
165	170	175<br>
Ser Arg Val Ala Ser Gin Phe Leu Gin Asp Leu Lys Gin Leu Val Glu<br>
180	185	190<br>
Lys Thr Ser Ser Glu Asn Glu Gly Lys Pro Val lie Leu Leu Ser His<br>
195	200	205<br>
Ser Leu Gly Gly Leu Phe Val Leu His Phe Leu Asn Arg Thr Thr Pro<br>
210	215	220<br>
Ser Trp Arg Arg Lys Tyr lie Lys His Phe Val Ala Leu Ala Ala Pro<br>
225	230	235	240<br>
Trp Gly Gly Thr lie Ser Gin Met Lys Thr Phe Ala Ser Gly Asn Thr<br>
245	250	255<br>
Leu Gly Val Pro Leu Val Asn Pro Leu Leu Val Arg Arg His Gin Arg<br>
2S0	265	270<br>
Thr Ser Glu Ser Asn Gin Trp Leu Leu Pro Ser Thr Lys Val Phe His<br>
275	280	2B5<br>
Asp Arg Thr Lys Pro Leu Val Val Thr Pro Gin Val Asn Tyr Thr Ala<br>
290	295	300<br>
Tyr Glu Met Asp Arg Phe Phe Ala Asp lie Gly Phe Ser Gin Gly Val<br>
305	310	315	320<br>
Val Pro Tyr Lys Thr Arg Val Leu Pro Leu Thr Glu Glu Leu Met Thr<br>
325	330	335<br>
Pro Gly Val Pro Val Thr Cys lie Tyr Gly Arg Gly Val Asp Thr Pro<br>
340	345	350<br>
Glu Val Leu Met Tyr Gly Lys Gly Gly Phe Asp Lys Gin Pro Glu lie<br>
355	360	365<br>
Lys Tyr Gly Asp Gly Asp Gly Thr Val Asn Leu Ala Ser Leu Ala Ala<br>
370	375	380<br>
Leu Lys Val Asp Ser Leu Asn Thr Val Glu lie Asp Gly Val Ser Hia<br>
335	390	395	400<br>
Thr Ser lie Leu Lys Asp Glu He Ala Leu Ly3 Glu He Met Lys Gin<br>
405	410	415<br><br>
lie Ser lie lie Asn Tyr Glu Leu Ala Asn Val Asn Ala Val Asn Glu<br>
420	425	430<br> 4<br> 1641<br> DNA<br> Arabidopsis thaliana<br> 4<br>
atgggagcga actcgaaatc agtaacggct tccttcaccg tcatcgccgt ttttttcttg 60<br>
atttgcggtg gccgaactgc ggtggaggac gagaccgagt ttcacggcga cCactcgaag 120<br>
ctatcgggta taatcattcc gggatttgcg tcgacgcagc tacgagcgtg gtcgatcctt 180<br>
gactgtccac acactccgtt ggacttcaat ccgctcgacc tcgtatggct agacaccact 240<br>
aagcttcttt ctgctgtcaa ccgctggttt aagtgtatgg tgctagatcc ttataatcaa 300<br>
acagaccacc ccgagtgCaa gccacggcct gacagtggtc tttcagccat cacagaattg 360<br>
gatccaggtt acataacagg tcctctttct actgtctgga aagagtggct taagtggtgt 42 0<br>
gttgagtttg gCatagaagc aaaCgcaatt gtcgctgttc catacgatCg gagattgtca 480<br>
ccaaccaaat tggaagagcg tgacctttac tttcacaagc tcaagttgac ctttgaaact 54 0<br>
gctttaaaac Cccgtggcgg cccttctata gtattcgccc attcaatggg taacaatgtc 500<br>
ctcagatact ttccggaatg gctgaggcca gaaattgcac caaaacatta tttgaagtgg 660<br>
cttgatcagc ataCccatgc ttatttcgct gctggagctc ctcttcttgg ttctgttgag 720<br>
gcaatcaaat ctaccctctc tggcgtaacg tttggccttc ctgtttctga gggaactgct 7S0<br>
cggttgttgc ccaattcttt tgcgtcgtca ttgcggctta tgccatttcc aaagaatcgc 840<br>
aagggtgata acacatcctg gacgcatttt tctgggggtg ctgcaaagaa agataagcgc 90 0<br>
gCaCaccact gtgatgaaga ggaatatcaa tcaaaatatt ctggctggcc gacaaatatt 960<br>
actaacattg aaattccctc cactagcgtt acagaaacag ctctagtcaa catgaccagc 1020<br>
atggaatgtg gccttcccac ccctttgtct ttcacagccc gtgaactagc agatgggact 1030<br>
cttitccaaag caatagaaga cCatgaccca gatagcaaga ggatgttaca ccagccaaag 1140<br>
aagctgtatc atgatgaccc tgttttcaat cctctgactc cttgggagag accacctata 1200<br>
aaaaatgcat tttgcatata tggtgctcat ctaaagacag aggttggtta ttactttgcc 1260<br>
ccaagtggca aaccttatcc tgataaCtgg atcatcacgg ataccactta cgaaactgaa 1320<br>
ggttccctcg tgtcaaggtc tggaactgtg gttgatggga acgctggacc tataactggg 1380<br>
gatgagacgg Caccctatca tccactctct tggtgcaaga attggctcgg accCaaagtt 1440<br>
aacaCaacaa tggctcccca gccagaacac gatggaagcg acgtacatgt ggaactaaat 1500<br>
gctgatcatg agcatgggCc agacatcata gctaacatga caaaagcacc aagggttaag 1560<br>
tacataacct tttatgaaga ccctgagagc attccgggga agagaaccgc agtctgggag 1620<br>
cttgataaaa gtgggtatta a	1641<br> 5<br> S46<br> PRT<br> Arabidopsis thaliana<br> 5<br>
Met Gly Ala Asn Ser Lys Ser Val Thr Ala Ser Phe Thr Val He Ala<br>
15	10	15<br>
Val Phe Phe Leu lie Cys Gly Gly Arg Thr Ala Val Glu Asp Glu Thr<br>
20	25	30<br>
Glu Phe His Gly Asp Tyr Ser Lys Leu Ser Gly He He lie Pro Gly<br>
35	40	45<br><br>
Phe Ala Ser Thr Gin Leu Arg Ala Trp Ser lie Leu Asp Cys Pro Tyr<br>
50	55	SO<br>
Thr Pro Leu Asp Phe Asn Pro Leu Asp Leu Val Trp Leu Asp Thr Thr<br>
65	70	75	SO<br>
Lys Leu Leu Ser Ala Val Asn Cys Trp Phe Lys Cys Met Val Leu Asp<br>
85	90	95<br>
Pro Tyr Asn Gin Thr Asp His Pro Glu Cys Lys Ser Arg Pro Asp Ser<br>
100	105	110<br>
Gly Leu Ser Ala lie Thr Glu Leu Asp Pro Gly Tyr lie Thr Gly Pro<br>
115	120	125<br>
Leu Ser Thr Val Trp Lys Glu Trp Leu Lys Trp Cys Val Glu Phe Gly<br>
130	135	140<br>
lie Glu Ala Asn Ala lie Val Ala Val Pro Tyr Asp Trp Arg Leu Ser<br>
145	150	1S5	160<br>
Pro Thr Lys Leu Glu Glu Arg Asp Leu Tyr phe His Lys Leu Lys Leu<br>
165	170      "	175<br>
Thr Phe Glu Thr Ala Leu Lys Leu Arg Gly Gly Pro Ser lie Val Phe<br>
180	185	190<br>
Ala His Ser Met Gly Asn Asn Val Phe Arg Tyr Phe Leu Glu Trp Leu<br>
195	200	205<br>
Arg Leu Glu lie Ala Pro Lys His Tyr Leu Lys Trp Leu Asp Gin His<br>
210	215	220<br>
He His Ala Tyr Phe Ala Val Gly Ala Pro Leu Leu Gly ser Val Glu<br>
225	230	235	240<br>
Ala lie Lys Ser Thr Leu Ser Gly Val Thr Phe Gly Leu Pro Val Ser<br>
245	250	255<br>
Glu Gly Thr Ala Arg Leu Leu Ser Asn Ser Phe Ala Ser ser Leu Trp<br>
260	265	270<br>
Leu Met Pro Phe Ser Lys Asn Cys Lys Gly Asp Asn Thr Ser Trp Thr<br>
275	280	285<br>
His Phe Ser Gly Gly Ala Ala Lys Lys Asp Lys Arg Val Tyr His Cys<br>
290	295	300<br>
Asp Glu Glu Glu Tyr Gin Ser Lys Tyr Ser Gly Trp Pro Thr Asn lie<br>
305	310	315	320<br>
He Asn He Glu He Pro Ser Thr Ser Val Thr Glu Thr Ala Leu Val<br>
325	330	335<br>
Asn Met Thr Ser Met Glu Cys Gly Leu Pro Thr Leu Leu Ser Phe Thr<br>
340	345	350<br><br>
Ala Arg Glu Leu Ala Asp Gly Thr Leu Phe Lys Ala lie Glu Asp Tyr<br>
35S	360	365<br>
Asp Pro Asp Ser Lys Arg Met Leu His Gin Leu Lys Lys Leu Tyr His<br>
370	375	380<br>
Asp Asp Pro Val Phe Asn Pro Leu Thr Pro Trp Glu Arg Pro Pro lie<br>
385	390	395	400<br>
Lys Asn Val Phe Cys lie Tyr Gly Ala His Leu Lys Thr Glu Val Gly<br>
405	410	415<br>
Tyr Tyr Phe Ala Pro Ser Gly Lys Pro Tyr Pro Asp Asn Trp lie lie<br>
420	425	430<br>
Thr Asp lie lie Tyr Glu Thr Glu Gly Ser Leu Val Ser Arg Ser Gly<br>
435	440	445<br>
Thr Val Val Asp Gly Asa Ala Gly Pro lie Thr Gly Asp Glu Thr Val<br>
450	455	4S0<br>
Pro Tyr His Ser Leu Ser Trp Cys Lys Asn Trp Leu Gly Pro Lys Val<br>
465	470	475	480<br>
Asn lie Thr Met Ala Pro Gin Pro Glu Hia Asp Gly Ser Asp Val His<br>
485	490	495<br>
Val Glu Leu Asn Val Asp His Glu His Gly Ser Asp lie lie Ala Asn<br>
500	505	510<br>
Met Thr Lys Ala Pro Arg Val Lys Tyr lie Thr Phe Tyr Glu Asp Ser<br>
515	520	525<br>
Glu Ser lie Pro Gly Lys Arg Thr Ala Val Trp Glu Leu Asp Lys Ser<br>
530	535	540<br>
Gly Tyr 545<br> 6<br> 1608<br> DNA<br> Arabidopsis thaliana<br>
â– :400&gt; 6<br>
atgcctctat tactggaaga gatcactaga tcagtagagg ccttgctgaa gctcagaaat 60 cggaatcaag aaccctaCgt tgacccgaat ctaaacccgg ttcttctagt tccaggaatc 12 0 gctggatcaa ttctaaacgc cgttgaCcat gagaacggga acgaagaacg tgtttgggtt 18 0 aggatcttcg gtgctgatca tgagtttcga acaaagaCgt ggtctcgatt tgatccttca 240 actggtaaaa cgatatctct tgatccaaaa acgagtattg ttgttcctca agacagagct 300 gggctacatg caattgatgt cCtagaccct gaCaCgaCCg CCggccgCga gtctgCgCac 3 60 caCCtccaCg agaCgaCCgt tgagatgaCc ggaCggggaC tCgaagaagg gaaaaccctt 420 tttggtttcg gtCatgaCtt ccgccaaagc aacagactgc aggaaacgtt ggaccagtct 480 gctaaaaagc tggaaacCgt ttataaagcc tcaggagaga agaagattaa tgCCaCtagC 540 caCCctaCgg gaggacCaCt ggCgaaatgC ctcatgggtc tgcatagtga tatatCcgag 600<br><br>
aagtatgtac agaattggat tgctattgct gctccatttc gaggtgctcc tggatatatc 660<br>
acatcgactt tattgaatgg aacgtcgttt gtcaatggtt gggaacagaa cttttttjgtc 720<br>
tctaagCgga gcatgcatca gcCgcttatt gagtgtccat ccatatatga gccgatgtgt 780<br>
tgcccgcact ttaaarggga gctccctccc gtcttagagc tgtggagaga gaaagagagc 840<br>
aatgatggag ttggaacctc tgatgttgtt ctcgagtctt accgtagcct ggagagtfctt 900<br>
gaagttttta cgaaacctct ctcgaataat acagctgatt attgtggaga gtcgatcgac 960<br>
cttcctttta actggaagat-catggagtgg gctcacaaaa ccaagcaagt attagcCtcc 1020<br>
gccaagctgc ctccgaaagt taaattctat aacatatatg ggaccaatct agaaacccct 1080<br>
catagtgttt gccatgggaa tgagaagatg cccgttaaag atctaacgaa tctaagatac 1140<br>
ttccagccga catatatatg cgtggatggt gaCggcacag tcccgatgga atctgccatg 1200<br>
gcggatgggc ttgaagcagt agcaagagtt ggagtccctg gtgagcaccg aggaatcctc 1260<br>
aacgatcacc gtgtcttccg aatgctcaaa aaatggctaa atgtaggcga accagacccg 1320<br>
ttctacaacc cagtaaacga ttatgtcatc cttcccacca catatgaatt tgagaattc 1380<br>
catgagaatg gactcgaggt tgctcccgtg aaagaatcgt gggacatcat atcagacgac 1440<br>
aacaatatcg gcacaaccgg gtcaaccgtg aactccatat cagtctctca acccggagat 1500<br>
gatcaaaacc ctcaagctga agctcgtgca accttaaccg tccaaccaca aagcgacggc 1560<br>
agacaacacg tagagctcaa tgctgtaagt gtctctgttg atgcataa	1608<br> 7<br> 535<br> PHT<br> Arabidopsis thaliana<br> 7<br>
Met Ser Leu Leu Leu Glu Glu lie lie Arg Ser Val Glu Ala Leu Leu<br>
15	10	15<br>
Lys Leu Arg Asn Arg Asn Gin Glu Pro Tyr Val Asp Pro Asn Leu Asn<br>
20	25	30<br>
Pro Val Leu Leu Val Pro Gly He Ala Gly Ser He Leu Asn Ala Val<br>
35	40	45<br>
Asp Hia Glu Asn Gly Asn Glu Glu Arg Val Trp Val Arg lie Phe Gly<br>
50	55	60<br>
Ala Asp His Glu Phe Arg Thr Lya Met Trp Ser Arg Phe Asp Pro Ser<br>
65	70	75	go<br>
Thr Gly Lys Thr He Ser Leu Asp Pro Lys Thr Ser He Val Val Pro<br>
85	90	95<br>
Gin Asp Arg Ala Gly Leu His Ala lie Asp Val Leu Asp Pro Asp Met<br>
100	105	110<br>
He Val Gly Arg Glu Ser Val Tyr Tyr Phe His Glu Met He Val Glu<br>
115	120	125<br>
Met He Gly Trp Gly Phe Glu Glu Gly Lye Thr Leu Phe Gly Phe Gly<br>
130	135	140<br>
Tyr Asp Phe Arg Gin Ser Asn Arg Leu Gin Glu Thr Leu Asp Gin Phe<br>
145	150	155	160<br><br>
Ala Lys Lys Leu Glu Thr Val Tyr Lys Ala Ser Gly Glu Lys Lys He<br>
165	170	175<br>
Asn Val He Ser His Ser Met Gly Gly Leu Leu Val Lys c?ys Phe Mec<br>
laO	185	190<br>
Gly Leu His Ser Asp lie Phe Glu Lys Tyr Val Gin Asn Trp He Ala<br>
195	200	205<br>
He Ala Ala Pro Phe Arg Gly Ala Pro Gly Tyr He Thr Ser Thr Leu<br>
210	215	220<br>
Leu Asn Gly Met Ser Phe Val Aan Gly Trp Glu Gin Asn Phe Phe Val<br>
225	230	235	240<br>
Ser Lys Trp Ser Met His Gin Leu Leu He Glu Cys Pro Ser He Tyr<br>
245	250	255<br>
Glu Leu Met Cys Cys Pro Tyr Phe Lys Trp Glu Leu Pro Pro Val Leu<br>
260	255	270<br>
Glu Leu Trp Arg Glu Lys Glu Ser Asn Asp Gly Val Gly Thr Ser Asp<br>
275	280	285<br>
Val Val Leu Glu Ser Tyr Arg Ser Leu Glu Ser Leu Glu Val Phe Thr<br>
290	295	300<br>
Lys Ser Leu Ser Asn Asn Thr Ala Asp Tyr CyS Gly Glu Ser He Asp<br>
305	310	315	320<br>
Leu Pro Phe Asn Trp Lys He Met Glu Trp Ala His Lys Thr Lys Gin<br>
325	330	335<br>
Val Leu Ala Ser Ala Lys Leu Pro Pro Lys Val Lys Phe Tyr Asn He<br>
340	345	350<br>
Tyr Gly Thr Asn Leu Glu Thr Pro His Ser Val Cya Tyr Gly Asn Glu<br>
355	360	365<br>
Lys Met Pro Val Lys Asp Leu Thr Asn Leu Arg Tyr Phe Gin Pro Thr<br>
370	375	380<br>
Tyr He Cys Val Asp Gly Asp Gly Thr Val Pro Met Glu Ser Ala Met<br>
335	390	395	400<br>
Ala Asp Gly Leu Glu Ala Val Ala Arg Val Gly Val pro Gly Glu His<br>
405	410	415<br>
Arg Gly He Leu Asn Asp His.Arg Val Phe Arg Met Leu Lys Lys Trp<br>
420	425	430<br>
Leu Asn Val Gly Glu Pro Asp Pro Phe Tyr Asn Pro Val Asn Asp Tyr<br>
435	440	445<br>
Val He Leu Pro Thr Thr Tyr Glu Phe Glu Lys Phe His Glu Asn Gly<br>
450	455	460<br><br>
Leu Glu val Ala Ser Val Lys Glu Ser Trp Asp lie lie Ser Asp Asp<br>
465	470	475	480<br>
Asn Asn lie Gly Thr Thr Gly Ser Thr Val Asn Ser lie Ser Val Ser<br>
485	490	495<br>
Gin Pro Gly Asp Asp Gin Asn Pro"Gln Ala Glu Ala Arg Ala Thr Leu<br>
500	505	510<br>
Thr Val Gin Pro Gin Ser Asp Gly Arg Gin His Val Glu Leu Asn Ala<br>
515	520	525<br>
Val Ser Val Ser Val Asp Ala<br>
530	535<br> 8<br> 1344<br> DKA<br> 8<br>
atgggctgga ttccgtgtcc gtgctgggga accaacgacg atgaaaacgc cggcgaggtg 60<br>
gcggatcgcg atccggtgcc tccagtacct ggaattggag gctctattct gcattctaag 120<br>
aagaagaatt caaagtccga aactcgggtt tgggtccgaa tatttctagc taaccttgcc 180<br>
ttcaagcaga gcctctggtc tctctataat cccaaaactg gctatacaga gccgttggat 240<br>
gataatattg aagtatcggc ccctgatgat gaccatggac tctatgcaat tgacattcta 300<br>
gatccccctt ggtttgtaaa gctttgtcac ttgacggagg tttatcactt Ccacgatatg 360<br>
atagaaacgc tggttggatg cggttataag aaggggacta cattattcgg ttatggttac 420<br>
gatttccgtc aaagcaatag gatcgatcta ctcatactag gtctgaagaa gaagctggaa 480<br>
actgcatata aacgttcagg ggggagaaaa gtcactatca tctcccatcc aacgggagga 54 0<br>
cttatggctt catgtttcat gtatctccat ccggaggcat tttccaagta tgtaaataaa 500<br>
tggattacaa ttgcaacacc tttccaagga gcaccagggt gcatcaacga ttcaatcttg 660<br>
actggagtgc aatttgtgga agggttagaa agtttcttct ttgtgtcacg ttggacgatg 720<br>
caccaactgt tggtcgaatg cccatctata cacgagatga tggcaaatcc agactttaag 780<br>
tggaaaaagc aaccagagac tcgagtttgg cgCaagaaat ctgaaaacga cgttgatact 840<br>
tctgCagaac tggaatcatt tggcttaatc gagagtattg atctattcaa cgatgcatta 900<br>
aaaaataacg agctaagcta tggtgggaat aaaacagctt tgccctctaa ctctgctatc 960<br>
ctcgactggg ctgctaagac aagagaaatt ctcaacaaag cgcaacCtcc tgatggagtg 1020<br>
tccttctata acatatatgg agtgtcactt aatacaccct tcgacgttcg ttatggcaca loao<br>
gagacttctc cgatagacga tttgtctgaa atatgtcaaa ctatgcctga gtatacatat 1140<br>
gtagatggag atggaactgt ccctgctgaa tcagctgcag ctgctcagtt taaagcagtt 1200<br>
gctagcgtag gagtttcggg tagccaccgc gggcttctcc gtgatgaaag agtgtttgag 1260<br>
ctcattcaac aatggttagg agttgagccc aagaaggcta aacggaagca tttaaggact 1320<br>
cacaaagtag ttgattctgg ttaa	1344<br> 9<br> 447<br> PRT<br> Arabidopsis thaliana<br> 9<br>
Met Gly Trp lie Pro Cys Pro Cys Trp Gly Thr Asn Asp Asp Glu Asn<br>
15	10	15<br><br>
Ala Gly Glu Val Ala Asp Arg Asp Pro Val Leu Leu Val Ser Gly lie<br>
20	25      "	30<br>
Gly Gly Ser lie Leu His Ser Lys Lys Lys Asn Ser Lyg Ser Glu lie<br>
35	40	45<br>
Arg Val Trp Val Arg He Phe Leu Ala Asn Leu Ala Phe Lys Gin Ser<br>
50	55	60<br>
Leu Trp Ser Leu Tyr Asn Pro Lys Thr Gly Tyr Thr Glu Pro Leu Asp<br>
65	70	75	80<br>
Asp Asn He Glu Val Leu Val Pro Asp Asp Asp His Gly Leu Tyr Ala<br>
85	90	95<br>
He Asp He Leu Asp Pro Ser Trp Phe Val Lys Leu Cys His Leu Thr<br>
100	105	110<br>
Glu Val Tyr His Phe His Asp Met He Glu Met Leu Val Gly Cys Gly<br>
115	120	125<br>
Tyr Lys Lys Gly Thr Thr Leu Phe Gly Tyr Gly Tyr Asp Phe Arg Gin<br>
130	135	140<br>
Ser Asn Arg He Asp Leu Leu He Lau Gly Leu Lys Lys Lys Leu Glu<br>
145	ISO	155	150<br>
Thr Ala Tyr Lys Arg Ser Gly Gly Arg Lys Val Thr He He Ser Kis<br>
165	170	175<br>
Ser Met Gly Gly Leu Met Val Ser Cys Phe Met Tyr Leu His Pra Glu<br>
180	laS	190<br>
Ala Phe Ser Lys Tyr Val Asn Lys Trp He Thr Ha Ala Thr Pro Phe<br>
195	200	205<br>
Gin Gly Ala Pro Gly Cys He Asn Asp Ser He Leu Thr Gly Val Gin<br>
210	215	220<br>
Phe Val Glu Gly Leu Glu Ser Phe Phe Phe Val Ser Arg Trp Thr Met<br>
225	230	235	240<br>
His Gin Leu Leu Val Glu Cys Pro Ser He Tyr Glu Met Met Ala Asn<br>
245	250	255<br>
Pro Asp Phe Lys Trp Lys Lys Gin Pro Glu He Arg Val Trp Arg Lys<br>
2S0	2S5	270<br>
Lys Ser Glu Asn Asp Val Asp Thr Ser Val Glu Leu Glu Ser Phe Gly<br>
275	230	255<br>
Leu He Glu Ser He Asp Leu Phe Asn Asp Ala Leu Lys Asn Asn Glu<br>
290	295	300<br>
Leu Ser Tyr Gly Gly Asn Lys He Ala Leu Pro Phe Asn Phe Ala He<br>
305	310	315	320<br><br>
Leu Asp Tr? Ala Ala Lys Thr Arg Glu He Leu Asn Lys Ala Gin Leu<br>
325	330	335<br>
Pro Asp Gly Val Ser Phe Tyr Asn lie Tyr Gly Val Ser Leu Asn Thr<br>
340	345	350<br>
Pro Phe Asp Val Cys Tyr Gly Thr Glu Thr Ser Pro He Asp Asp Leu<br>
355	360	365<br>
Ser Glu He Cys Gin Thr Met Pro Glu Tyr Thr Tyr Val Asp Gly Asp<br>
370	375	360<br>
Gly Thr Val Pro Ala Glu Ser Ala Ala Ala Ala Gin Phe Lys Ala Val<br>
385	390	395	400<br>
Ala Ser Val Gly Val Ser Gly Ser His Arg Gly Leu Leu Arg Asp Glu<br>
405	410	415<br>
Arg Val Phe Glu Leu lie Gin Gin Trp Leu Gly Val Glu Pro Lys Lys<br>
420	425	430<br>
Ala Lys Arg Lys His Leu Arg Thr His Lys Val Val Asp Ser Gly<br>
435	440	445<br>
";210&gt; 10<br> 3107<br> DNA<br> Arabidopsis thaliana<br> 10<br>
cctttttgat ctttcagctc aaCgagcttt tcicaatttt ttgggggaac Cgaatatgtg 60 aatttcaaag tttccacatc gagtttattc acacgtcttg aatttcgtcc atcctcgttc 120 cgttatccag ctttgaactc ctcccgaccc tgctatggat atattaaaaa aaaagtgttt ISO tgtgggctgc atctttgtta cgatccgcat ctccttcttt cggctcagtg ctcatgcttt 240 tgctaCggta gagatgggca atgttattgt tgatggtaac agtggtatag ttgacagtaC 300 cctaactaat caattatctc tttgattcag gcctctatgt tgggtggaac acatgtcact 360 tgacaatgaa accgggttgg atccagctgg tattagagct cgagctgtat caggactcgt 420 ggctgccgac tactttgctc ctggccactt tgtctgggca gtgctgattg ctaaccctgc 480 acatattgga Catgaagaga aaaatatgta catggctgca catgactggc ggctttcgct 540 tcagaacaca gaggctcttt tctcatcgtt ctttctatta ttctgttcca tgccacgCtt 600 ctttcctcat tacttaaggc ttaaatatgt ttcatgttga attaataggt acgtgaccag 660 actcttagcc gtatgaaaag taatatagag ttgatggtct ctaccaatgg cggaaaaaaa 720 gcagttatag ttccgcattc catgggggtc ttgtattttc tacattttat gaagtgggtt 780 gaggcaccag ctcctctggg tggcgggggC gggccagatt ggCgtgcaaa gtatattaag S40 gcggtgacga acactggtgg accacttctt ggtgttccaa aagccgctgc agggcttttc 900 tctgctgaag caaaggatgt tgcagttgcc aggtattgaa tatctgctta tacttttgat 960 gatcagaacc ttggctctgg aactcaaagc tattctacta aatatcaatt ctaataacat 1020 tgctatatta tcgctgcaac cgacattggt tgattatttt gctgcttatg caactgaaac 1080 tctcttgaga ttagacaaat gatgaattga taattcttac gcactgctcc gtgatgacca 1140 gtttcttagc ttcgacgata acatttgtca tactgtcttt Cggagggcat tgaattctgc 1200 tatggaaagc gctggagctc ccatgcttgc attctttacc aattagcatt attctgcttc 12S0 tttcaatttt cttgtatatg catctatggt cttttatttc ttcttaatta aagactcgtt 1320 ggagtagttg ctctattagt cgcttggttc cttaatatag aacttcactc tcttcgaaaa 1380 ttgcagagcg attgccccag gattcttaga caccgacata tttagacttc agaccttgca 1440 gcatgtaatg agaatgacac gcacatggga ctcaacaatg tctatgttac cgaagggagg 1500<br><br>
tgacacaata tggggcgggc ttgattggtc accggagaaa ggccacacct	gttgCgggaa ISSO<br>
aaagcaaaag aacaacgaaa cttgtggtga agcaggtgaa aacggagttt	ccaagaaaag 1620<br>
Ccctgttaac Catggaagga tgatatcttt cgggaaagaa gtagcagagg	ctgcgccatc 1680<br>
tgagactaac aatattgatt tccgagcaag gacatataaa ccataataaa	ccttgtacac 1740<br>
ttcgtgattg tatgatgaat atctgtacat tttatctggt gaagggcgct	gtcaaaggtc 1800<br>
agagtatccc aaatcacacc tgtcgcgacg tgcggacaga gtaccatgac	atgggaattg 1860<br>
ctgggatcaa agctatcgct gagCaCaagg tctacactgc tggtgaagct	atagatctac 1920<br>
Cacactatgt tgctcctaag atgatggcgc gcggtgccgc tcatctctct	catgggattg 1980<br>
ctgatgattt ggatgacacc aagtatcaag aCcccaaata ctggccaaat	ccgttagaga 2040<br>
caaagtaagt gatttctcga ttccaaccgt atccttcgcc ctgacgcatc	atcagtcttt 2100<br>
ttgttttcgg tcttgttgga Cacggttttc agctcaaagc ttacaaagct	gtrcctgagc 2160<br>
ctttctcaaa aaggcctgct cagttatatt gaggtgctaa agttgaCaca	tgtgacccct 2220<br>
gctcataaat cctccgcttg gtttgttctg ctctttcaga ttaccgaatg	ctcctgagat 22S0<br>
ggaaatctac tcattatacg gagcggggat accaacggaa cgagcaCacg	tatacaagct 2340<br>
taaccagtct cccgacagtt gcatccccct tcagaCattc acttctgctc	acgaggagga 2400<br>
cgaagatagc CgCctgaaag caggagttta caatgtggat ggggatgaaa	cagcacctgt 2460<br>
cctaagtgcc gggCacatgt gCgcaaaagc gtggcgCggc aagacaagac	Ccaacccttc 2520<br>
cggaatcaag acttacaCaa gagaatacaa tcactctccg ccggctaacc	tgttggaagg 2580<br>
gcgcgggacg cagagtggtg cccatgttga tatcaCggga aactctgctt	tgatcgaaga 2640<br>
tatcatgagg gtCgccgccg gaggtaacgg gCctgatata ggacatgacc	aggtccactc 2700<br>
tggcatattt gaatggCcgg agcgCattga cctgaagctg tgaacatcat	gatctcttta 2760<br>
agctgtcctg tcagctcatg cgaatccaat actttgaaag agagaCcacc	atcaactcac 2820<br>
catcatcgtc atcatcatga tgctcaactc acaaagaagc ccgagaatga	tactttggtg 2B80<br>
cgaaattctc aatacctcct caaCattctt attgaaCgta aattaCacaa	ccctatetaa 2940<br>
tgtctgaacg ataacgcaaa actCgcCgcg ccatgcttgt ttgtcttgCc	aaaagcatca 3000<br>
atttgtgggt catacgtagt gcagaggatg attcaaattt gcgacaaatt	tggtaatcaa 3060<br>
agttaattct gaaaatgcaa caccacatga actaCgtcac taaggcc	3107<br> 11<br> 1680<br> DNA<br> Arabidopeis thaliana<br>
=220&gt;<br>  unsure     (694)   n=uiikown<br>
":400&gt; 11<br>
cgcataaggt gttcgagtgt ttgcagcttg agaagttccg	agtccaagag	accCggagcc 60<br>
aaagaCctga accaCaaaaa CgaccaaCca aaaCccatta	agccaattca	aacattcact 12 0<br>
aaaaatgtta cagttctcac gaatactaac ataacaagtg	aaagtaaact	taaaaacgct 180<br>
catggaccta acctggcgta acggtatgtc tctgccctca	gcagaaagCa	aattactgac 240<br>
ggccctaggg acaccCaaaa aggcgggccc aatgttgacg	acggatttga	tgtgtttggc 300<br>
acaccaacct ggaccacccc caccgccCcc atcaggaaga	ggtgttCcCa	cccattcaag 360<br>
gaagtgaagg aaatagatag cccccaCCga atgcggaacc	accacaactt	tcttaaaccc 420<br>
actggtggca tacattagct cgaCtttgct cttcagCcta	cttaacgact	ggccacgcac 480<br>
ctgctcggtt tcaatccaaa aacCaCagat tagtccaaag	ctctacaaca	atatgtaatt 540<br>
acatacacta aagtagccaa tcatggaggC cttatagtat	atcattatca	Ccattctcta 600<br>
gaccaccagC gtCgCcaacg Cgatcataca ggcattaata	acgactaatc	tgagcacacc 660<br>
CcggtgttaC ggaaagagag cctccaaCca Caangaggcc	atgtgaaggC	ccttgccttc 720<br>
ataCccaaCC Cttgccaaat cctctatgag aactgcccaa	gcaaagcagc	atggtgcgaa 780<br>
atagtctgca gccactagtc ctgggactgc tcggacacgg	actcccggtg	gatcgagacc 840<br>
ggtctcactg cctagagata agtgccccaa ceagcacaat	ggcccataaa	tcaaatcaca 900<br>
acaattaaac gaccaagtat acacttcaaa cCaaCtcaga	attgagaaaa	tcgaaatgcc 960<br><br>
aaccagaaaa	tcatgtaaaC caaaaaccgt aacaafccaat atatataCat atattttcca 1020<br>
gaatccaCgC	taaaaccata accaaaaata tatgaaaatt tagaaatacc aaaataatat 1080<br>
gttaaaactg	atattctaaa tttagtaagt tttaaaatgc aatgaaatcg tcattcatgt 1140<br>
tttgaacata	aatatattct atagttttgt aggacgattt tctacttcct atatagaaat 1200<br>
caaaacttac	ggtttccatt tccaaattcg aatgacattt aaaaacatat cccaaaaatc 1260<br>
acgattaatt	attaatttcc taaaaccacc catcattact tagaaaataa" tattttcata 1320<br>
aactagctgc	aaaacaataa caaaacccaa agaaccatct ccacccacta accaaaacga 1380<br>
aaatccaaag	accatccata acaacaacag tataacacta cgtaaagcca attcaagaag 1440<br>
aaaccaagct	taaccaatta tacatacccc taaccagacg aaccaaacca atatctgacc 1500<br>
gggtctataa	aaatatctcg aaccgaacat aacggtctaa tgtgttacct tctaagaatc 1560<br>
tcggagaagc	tagcacccca aagacgttca cgaaagagtc cttcagcgca aggccgacct 1620<br>
tcccaaagct	cgagcccgcc ggttacaacc cccggaacaa gaatcaccgg atgaaacgcc 1680<br> 12<br> 264<br> DNA<br>  Glycine max<br><br>   unsure     (39)   n=untaown<br><br><:2>  unsure     {175)   n=unknown<br><br> unsure  (241)  n=unknown<br> 12<br>
ccaagaactc gatgattact tcaacactcc tggggttgng acccgggtcc ctcactttgg	50<br>
ttccaccaac tctcttctct catctcaacc ctcgtctcaa gcatatcscc ggatacatgg	120<br>
cacccctggt agattcatta caaaagcttg gctacgctga tggtgagact ctgtntggag	180<br>
ccccttatga ctttagatat ggtctagctg ctgaaggtca cccttcacaa gtgggttcca	240<br>
ngttcctcaa agatctaaag aact	264<br> 13<br> 273<br> DNA<br> Glycine max<br><br>	unsure<br>	(12)<br>	n=i.iIvknown<br><br>	unsure<br>	(33)<br>	n=Uilknown<br><br><br>  unsure     (252)   n=uniaiown<br><br>  unsure   n=unlaiQwn<br><br> unsure  (272)  n=unknown<br> 13<br>
ccaacatctg anaggggcag agtaggtatt Ccnacctatt atccatatcg	tgatgaagaa 60<br>
ggaacaagaa gagggtctca agattgaggt tgctacactc acagttacag	tagttgttgt 120<br>
gacgctgCca ttgctacgca catgtggggc aagcaacctc gaccctttga	tcctaatacc ISO<br>
aggtaacgga gggaaccaac tagaagcaag gttgaccaat cagtacaagc	cctctacntt 24 0<br>
cacctgcgac cntggtaccc tcccannaag ana	273<br> 14<br> 419<br> DNA<br> Glycine max<br><br> unsure     (99)    n=unknown<br><:220><br> unsure<br>  n=unknown<br><br>  unsure     (352)   n=unknown<br><br> unsure  (392) e223&gt; n=unknown<br><br> unsure  (405)  n=unknown<br><br><br> unsure     (418)   n=uiiknQwn<br>  14<br>
gctgcatatg attggagaac agcacctcag aacactgagg tgagggatca aacactaagt 60 cggataaaaa gcaacacaga acttatggtt gctactaang gtggaaataa ggcagctatt 120 attccacact caatgggggt cttgtacttc ctacatttta tgaaatgggt tgaagcacca 180 gctccaatgg gtggtggggg aggaccagat tggtgctcca aatatataaa ggcagttgta 240 aacatcggtg gaccattttt aggtgttccc aaggctatag cagggctact ctcagctgag 300 gccaaggata ctgctgttgc caggacgata gctccaggat ttttanataa cnatctgttt 360 ccgcattcaa acccttgcaa catgtaatga anatgaaccc gttcnttggg actcaacna 419<br> 272<br> DNA<br> Glycine max<br><br>	unsure<br>	(1) . . 1272)<br>
â– !:223&gt;	n=uiiknown<br>   15<br>
Cganttgatc ntgngaagtn attctgtgta ttanttccat gacatgaccg ttnnagatnc 60<br>
gtaagtgang ggtntgaaga gggaaagacg ccttttggtn ttngatatga ttttcgccaa 120<br>
agcaacaggt tgcaggaaac aatggatcgg ttggctgcna agtcagaatc aanttataat 180<br>
gccgcaggnn ggaagacaat aaacattata nctcattcta tgggcggtct tttccnngan 240<br>
atgtttcntg tgcctgcaaa gcgatatttt ga	272<br> 16<br> 237<br> DNA<br> Glycine max<br><br> unsure c222&gt;    (1).-(237) â– c223&gt;  n=ucknown<br> 16<br>
gattttcgcc aaagcaacag gttgcaggaa acaatggatc ggttggctgc aaagttagaa 50 tcaatntata atgcngcagg agggaagana ataaacatta taactcattc tatgggcggt 12 0 cttttggtga aatgnttcac gtgcctgcaa agcgatattt ttgagaaata Cgttaagaat 180 tgggttgcaa ttcgtgcgcc attccagggt gcaccaggaa ccatcaattc naccttt   237<br> 17<br> 244<br><:212> DNA<br> Glycine max<br><br><br>   unsure     (1)..(244)   n=unknown<br>   17<br>
gattttcgcc aaagcaacag gttgcaggaa	acaatggatc	ggtnggctgc aaagCCagaa 60<br>
Cgcaatttat aatgctgcag gagggaagaa	aataaacatt	ataactcacc ctacgggcgg 120<br>
tcttttggcg aaatgtttca tgtgcctgca	aagcgatacc	tttgagaaat atgccaagaa 180<br>
ttgggttgca actcgcgcgc catcccaggg	tgcaccagga	accatcaact cCacccttct 240<br>
aaat	244<br> 18<br> 263<br> DNA<br> Glycine max<br>
â– ;400&gt; IB<br>
gacgaaacca aaccgtgggc gaccaagctt	gtttactcgg	CtgactcaCg gcaagaccaa 60<br>
gttcgCCgct tcatagaaga ggCcattggc	gaaccagtct	atcccgtggg caactcacCa 120<br>
ggaggactgg ctgcattgca ttccgcggca	aacasccctc	atccagcgaa aggcgtcgca 130<br>
ttgcttaagc aacacctttt- tgggggCttc	tgccaaaCcc	cataaaaagt ccaagactag 240<br>
;gaaaatact tccatgggcc gga	263<br>
:210&gt; 19 ;211&gt; 311 ;212&gt; DNA ;213&gt; Zea mays<br>
220&gt;<br>
221&gt; unsure 222&gt;    (11 . . (311! 223&gt;  n=uiiknown<br>
;400&gt; 19<br>
:ggacgctgg ncatgtCcgg agccccctac gacttccgct acgcgccgcc gtcccccggc 60<br>
:agacgCccg aggcgtactc ccgctacttc aaggagctga Cggagctggt cgaggccgcg 12 0<br>
Lgcgagagga cccggaagaa ggccgtcatc cCcggccaca gccccggcgg cacggCcgcg 180<br>
iccgagttcg Cccggaacac Cccgccggcg Cggcggcgcg agcacaCcga gcgcctcgtc 240<br>
icggtcgcgc cgacgctccc cggcgggttc ctggagccgg Cgcgcaactt cgcgtccggg 300<br>
â– cggacatcc t	311<br>
210&gt; 20 211&gt; IISS 212&gt; DNA 213&gt; Zea mays<br>
â– 100&gt; 20<br>
cgacccacg cgCccggcca caagaaccct cCcaagccag actggtgcct cggaaagcCg 60 gagccgcac Cggaagacat gggataccga gacggagaca ccatgttcgg agccccctac 120 acctccgcc acgcgccgcc gtcccccggc cagacgtccg aggtgtactc ccgctacttc 180 aggagccga tggagcCggt cgaggccgca agcgagagga cccggaagaa ggccgtcatc 240 Ccggccaca gcttcggcgg catggCcgcg cCcgagcccg Cccggaacac Cccgccggcg 300<br><br>
tggcggcgcg agcacaccga gcgcctcgtc ctggtcgcgc cgacgctccc cggcgcgttc 360<br>
ctggagccgg tgcgcaacct cgcgtccggg acggacatcc tmtacgcgcc agcgacgacg 420<br>
ccgctggcca cgcgagccat gcgragragc ttcgagagcg ccatcgtgaa cctcccgtcg 430<br>
ccggccgcgt tcgggcgcct gcaggcgccg ctcgtggtca ccagggagcg gaactactcc 540<br>
gcgcccgcgc acgacacgga gcgcctcctc gccgccgtcg gccccggcga ggccgcggag 600<br>
cccttcagga gacgggccgt ccccaagacg ggcagcctcg cggcgccgat ggtgcccatg 660<br>
acgtacatca gcggggtcgg caacaggacg ccgccgcggc Cggtgctctg gggcgacgac 720<br>
ttcgacgcgg ccccggaggt ggcggcgcac ggggacggag atggcaagat caattcgatc 7S0<br>
agcgtcttgg cgtctgagaa ggagatgcgt cggcagccgg agcagaagaa gcsgttcaaa 840<br>
tccatcaaga tcgataaggc ccagcattcc acgatcgtca cggatgattt tgccctgcac 900<br>
agggtcattc aagaaattgt tgaggccaat aatcagaaga ttccatccta aattcctcat 9S0<br>
gtcatgtatg cattaccgag ctgtgggggc caatagtggg ttgggagttg ggacaccggc 1020<br>
tccgtgctta aaacggtcgt ggtgtggtct caattcaatc gattagttaC ttgctaacgc 1080<br>
caactgcttg cctcaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa aaaaaaaaaa 1140<br>
aaaaaaaaar gggcg	1155<br> 21  328  DNA  Zea mays<br> 21<br>
gttggaatgc tcttcaactt tcttacgctc atttacaCtg cttttctctg cggtaaattg 60<br>
ctggcctaaa tgcacgctgc ttgaacccta taatcagata gaccatcccg aatgcaagCc 120<br>
aaggcctgat agtggtcttc CgcaaCtaca gagccggacc ctggttatac aacaggtcct ISO<br>
ctctcttcag tatggaaaga atgggtcaaa tggtgtgtag agtttggcac tgaagctaat 240<br>
gcaattaccg ctgtCccgta tgactggaga ctgcccccat caacgctcga ggagagagat 300<br>
ccgtactttc acaattaaac aggatcag	328<br> 22  356  DNA  Zea mays<br> 22<br>
gtctttctgc aattacagag ctggaccctg gttatataac aggtttcagg tcctctctct SO tcagtatgga aagaatgggC caaatggtgt gtagagtttg gcaCCgaagc taaCgcaatt 12 0 atcgctgttc cgtatgattg gagactgccc ccatcaatgc ttgaggagag agatctgtac ISO tttcacaaat taaagtttgt aacacttgcc tcaacttgtt atgaagcaac caaCgctata 240 caCcCgtCag gatcagCaag agttaacggc ccatgacgga Ctcaggttcc tgcCcaccaa 300 cagatcccac aagcatacgg Ccaccgccaa tgcctgcagt tggacagtac caaccc    356<br> 23  1552  DNA  Zea mays<br> 23<br>
tcgacccacg cgCccgcaga catgatcatC ggtgatgaca ctgCgtacta ctatcatgac 60 atgatagCgg aaatgattaa atggggatat caagaaggaa aaactctctt tggaCCtggt 12 0 tacgatttcc gtcaaagcaa caggctctca gagacacttg acagactttc taaaaagctg 180 gagCcagtgt acacagcttc tggtggaaag aagatcaatd tcattactca ttcaatgggg 24 0 ggaCtacttg tgaaatgttt caCctcactg cacagCgata tatttgaaaa atacgtcaar 300<br><br>
agttggatcg caattgctgc accattccar ggcgcccctg ggcamataac taccak-ytig 360 ctgaaCggaa tgncttttgc craaggacgg gaaycaagat tcttratctc caaawkgkgt 420 atgcascaat tgccactcga gtgcccatca atctacgagk tgctgscaam ccctaactct 4S0 ccagtggaga gacatcccac tgctacagat tcggagagag aacttggata mcagtggcaa 540 gaaaagtgcc ccgttagagt cgtatgagcc cgaggaagca acaaagatga ttaaagaggc 600 tctttccagC aatgagatca ttgctgatgg catgcatatt ccggtgcccc ttaatttgga 660 tatattgaat tgggcaaaga aacttatgat cctttatgca gcacaaagcc tccggaatca 720 gtgaaattct acaacattta tgggaCtgat tatgatactc cacatactgt ctgctacggc 780 agtgaacagc agccggcttc aagtcttagc agcctcctat atgctcaggg aaaatacgcc 840 tatgttgacg gcgacggatc tgttcccgca gaatcagcaa aggctgacgg aCttaatgca 900 gtggcaaggg ttggggttgc tcccgaccac cggggaatcg tgcgcagtcg ccgcgcgttc 960 cggatcgtcc agcactggct gcacgccgga gaacctgacc cgttctacga cccgctgagc 1020 gactatgtca tactcccaac acgcttacga aatcgagaag catcgtgaga aacacgggga 1080 tgtcacgtca gtagcggagg actgggagat catctcccct aacgacggca agaccatrrg 1140 gccaggcgag cttcctccta tggtcagcac actgaccacg agccgggaag gcaaggaggg 1200 agcactggaa gaggcgcatg ccaccgtggt cgttcacccg gagaagaagg gacggcagca 12 60 tgtgcaagtt agggctgtgg gtgtcagcca tggtggctaa agccgtagga gccacgttgg 1320 ctgcctactc tatccagcag tagcagctat acctctgtgc acgcaccgta aaattggatg 1380 tacatatatg gctatgacct ctgtagggat ctggttttag aagtataaat gggcaccctg 1440 cctgcttgta aatgctcaga accgaaaaca caggccctgt tctttttttt cctttttaaa 1500 aaaaataaaa agatggtaaa ggattccatt aaaaaaaaaa aaaaaaaagg eg        1552<br>	24<br>
";211&gt;	227<br>	DMA<br>	Zea mays<br><br> unsure  (1)..(227)  n=unknown<br> 24<br>
ttggttatga	tttccgtcaa agcaacaggc cctcagagac acttgacaga ttttctaaaa 60<br>
agctggagtc	agtgtacaca gctcccggtg gaaagaagat caatctcatt, actcattcaa 120<br>
tggggggatt	acttgtgaaa tgtntcatct cactgcacag tgatatatnt gaaaaatatg ISO<br>
tcaagagctg	gntcgcaatt gcngcaccat Cccaaggtgc ccctggg	227<br> 25  1587  DNA c213&gt; Zea mays<br><br>
e221&gt;  unaure     11).-(1587)   n=ankiiown<br> 25<br>
ggagattgtc gtgccggagg acgaccacgg cctgtttgcc atcgacattc ttgatcctcc 50 ctggtttgta gaactcgacc cacgcgtccg cccaccgtcc gggagattgt cgtgccggag 120 gacgaccacg gcctgtttgc catcgacatt cttgaccctt cctggtttgt agaacttctc 180 catctgtcta tggtgtatca cttccatgat atgattgata tgctcataaa ctgtggatat 240 gagaaaggga ccacactatt tggatatggt tatgattttc gccaaagcaa caggatagac 300<br><br>
aaagcgatgg ccggttcgag agcaaaactt	gagacagctc ataagacctc tggagggaaa 3S0<br>
aaagttaatc caaccccaca ctctatgggt	ggacrgctag tacgctgctt catgcctatg 420<br>
aatcacgatg tattcactaa gtatgtcaac	aaatggattt gcatcgcttg tccactccaa 480<br>
ggtgcccccg gaCgcaccaa cgactctcta	cttactggat tgcaatttgt ttatggtttt 540<br>
gagagcttct ttttcgtatc cagatgggca	atgcaccaat tgcttgtcga atgcccatca â‚¬00<br>
atccaCgaaa tgttaccaaa cccagaattc	aagtggaagg aaaaaccaat tattcaggtt 660<br>
tggcgtaaga accctgaaaa ggatggaact	gtggagcttg ttcaatatga agcaactgat 72 0<br>
tgtgtgtcct cgctcgaaga agctttaagg	aataatgagc tcacgtataa cggaaagaaa 780<br>
gtagcactac cattcaatat gtcagtcttc	aaatgggcca ccaagactcg ccaaatccta 840<br>
gacaacgctg aattaccaga cactgtgagc	CtttacaaCa tatacgggac atcttacgaa 900<br>
actccatacg atgcatgcta cggctcagaa	agctccccga Ctggagattt gtcagaagtg 960<br>
tgtcacacag tgccggcata cacttaCgtg	gatggagatt gcacggttcc catagaatcg 1020<br>
gcacgggctg atgggttctc tgcgaaagaa	agagttggcg tcaaggcgga ccaccgtggc lOao<br>
ctgctgtccg atgagaacgt attcaagctt	ctcaagaaat ggctcggtgt gagcgagaag 1140<br>
aagccagagt ggcgttgcgt gcctaaatcc	cactccaaag tgacctaatt gggttgcctg 1200<br>
tagttcttca ggaagactgt tatcttggcc	cctcctcctg aagagaagat gaaacaaaac 1260<br>
CccggtgaCt gtattgcatg cctgcacgat	gtaaatctct gcaagctgca cggaacaagg 1320<br>
gactagtgcc cttgtacgat gtatcattgg	caggcatttn cttttgaacc tangggcata 1360<br>
ttcntttgnc cttccactct ggacntagta	aagaacatnt gaatcgacct tanttnnaan 1440<br>
nngtctgnnn nnjumimrinii nnnnnnnnnn	ruimmnnnim nnnnnnrmnn nnimnnnnnn 1500<br>
nrmnimrmnn niiaaaaaaaa awgkgaagcc	gntnntnntt tnaaaagnnt tttnnnaaaa 1550<br>
aaaaaaaaaa aaaaaaaaaa aaaaaaa	1587<br> 26  300  DNA  Zea mays<br><br> unsure  (1)..(300)  n=unknown<br>
gacaaagcga tggctggttt gagagcaaaa cttgagacag cccataagac ctctggaggg 60 aaaaaagtta atttaatctc acattctatg ggtggattgc tagtacgctg cttcatgtct 120 atgaatcatg atgtgagttt tcatgttttc tgtgCttttt ttgcttttgc ataaatatcc 180 atgtcaattt cccccacttt ctaggtattc actangtatg tcaacaaatg gatctgcatt 240 gcttgtccat tccaaggtaa cttatgggac acttcaattg tttattanat natggggncc 300<br> 27  1240  DNA  Zea mays<br>
=:220&gt;<br>
Â«221&gt; unsure =:222&gt;    (1) . . (1240) i;223&gt;  n=un)aiown<br>
;400&gt; 27<br>
:cgacccacg cgtccggttc	ccagttccca ccgtgtagat ggttctggta taaaatgtat 60<br>
-.gccatattt gtaacacaga	ttactatata caggttcgtg atcaaacttt gagcagaata 120<br>
iagascaaca ttgaactcat	agwagsgaca aatggtggaa atagggcggt ggiangatccc 180<br><br>
acnactccat  ggggtcnttn  attctntgcn  ttctacgnaa  tggnccgaag  ccczcctccg  240<br>
tgggggcagt "gggtccgaac  tggntgtaga  accatataaa gcCgtaaCga  atattggagg  300<br>
acctttccta  ggagttccta  aggctgttgc  tgggcttttt   ttcttctaag caaaagacgc  360<br>
tgccggttgc  taggtataag  taacgattca  tccatttaaa  gcaaaaggga  atagcaaaag  420<br>
aacgaacatt  attggatgct  cgacaagctt gcggagcttt  tgctcccaag ccatctcctg  430<br>
gaccccacaa gtccagggag  tgcctgcctc  tgatcctcat  catcaggaac  aggcCcaagc   540<br>
atgcaccgac  ggtaccgcga ggccatttct  atcctgatgc  aacaccacgt  acttgttgat  600<br>
ggcaaggcca  ggactgacaa gacctaccct  gctgggCtca  tggaCgtcat  ttccatcccc  660<br>
aagacaaacg  agaactacag gccgctttcg  Ccttcaccca  aCcagggatg  aggatgccaa  720<br>
gttcaagctc  cacaaggtga ggtctgttca gtttggccag  aaagacaccc  cctatctgaa  780<br>
cacctacgac  gaccgcacca  tccgctaccc  cgacccgctc  atcaaggcca acgacaccat  340<br>
caagaccgat  ctggagacca  acaagatcat ggacttcacc  atgtttgacg  tcggcaacgt   900<br>
ggtcacggtg  atcggcagga ggaataccgg gcgtgtagga  gtgatcaara  taagggagaa  960<br>
gcataagggc  aactccgaga  ccatccacgt  gctgctcgra gctttttgct  atgtctagct:   1020<br>
ttctcctatt  tgttgtacag gaaaacaCag  aatgaaattc  aaatttggtg gccacaaaag  1080<br>
tgtggagact  tgatttcata  taaagttagg  cttaacatta gtgcaaacag  ttgtatttta  1140<br>
gtttagattt  agagtacact  atgtatgcgt  tgtttgacaa  tgcttattta  tgatacattg  1200<br>
aatggtactt  aCttatatta attaattaaa aaaaaaaaaa	1240<br>	28<br>	324<br>	DNA<br>	Zea mays<br>
cgaatgctcc tgacatggaa atattttcca tgtacggagt aggcattcct actgaaaggg 60<br>
catatgtcta taagttggcc ccacaggcag aatgttatat acctttccga attgacacct 120<br>
cggctgaagg cggggaggaa aatagctgct tgaaaggggg tgtttactta gccgatggtg 180<br>
acgaaactgt tccagttctt agtgcgggct acatgtgtgc aaaaggatgg cgtggcaaaa 240<br>
ctcgtttcaa ccctgccggc agcaagactt acgtgagaga atacagccat tcaccaccct 300<br>
ctactctcct  ggaaggcagg ggca	324<br>   29    254   DNA   Zea mays<br>   29<br>
gaataaagag  caacattgaa ctcatggtag	caacaaatgg  tggaaatagg gtggtggtga  60<br>
tcccacactc  catgggggtc  ctctattttt	tgcattttat  gaaatgggtc gaagcacctc  120<br>
ctcccatggg gggtggcggt ggtccagact	ggtgtgagaa gcatattaaa  gctgtaatga  180<br>
atattggagg  acctttctta  ggagttccta	aggctgttgc  tggccttCtc  tcatctgaag  240<br>
ccaaagatgt tgcc	254<br>   30<br>   513<br>  DNA<br> Mus  musculua<br>   30<br>
tggaggacaa  cgcggggtct gatacgactc  actataggga  atttggccct	cgagcagtag  60<br>
attcggcacg acgggcacga ggactccatc  atgttcctca agctttattc	ctaccgggaC  120<br>
gtcaacctgt ggtgccgcca gcgaagggtc  aaggccaaag ctgtctctac	agggaagaag  ISO<br><br>
gtcagtgggg ctgccgcgag caagctgtga gctatccaga caacctgacc taccgagatc	240<br>
tcgatcactt cacctttgct cccactttgt gttatgaact caactttcct cggccccccc	300"<br>
gaaCacgaga gcgctttctg ctacgacgag tCcttgagat gctctttttc acccagctcc	360<br>
aagtggggct gatccaacag cggatggtcc ctactatcca gaactccatg gaagcccttt	420<br>
caagagcttc tgcagcttcg gagaccgcga gctctacaga gactggtgga atgctgagtc	480<br>
tgccaccgac ttttggcaga actggaacat ccccgcgg	51B<br> 31<br> 299<br> DNA<br> Mus musculus<br> 31<br>
ccatgatggc tcaggtccca ctggcctgga tcgtgggccg attcttccaa gcgaactatg 60 gcaatgcagc tgtgtgggcg acactcatca ttgggcaacc ggtggctgcc tcacgtatgt 120 ccacgactac tacgtgctca actacgatgc cccagtgggt catgagctac tgccaaaggc 180 agcccCccct aacctgggcc tggagttctg gaggggttcc tggctgcctg cacactcctc 240 ctagtctggg aggcctctct gccccCatgc gctactcctg ctcttgggga tggcattcg 299<br> 32  1395 
 Artificial Sequence<br><br> Description of Artificial Sequence: Inferred cDNA sequence<br><br> unsure  (1).,(1895)  n=unknovm<br>
c400&gt; 32<br>
gcctggtgtg aCggggacag ggagggactt ccccCtaccc agcactggtg ttggctgagg 60 tgggtgccga gtcccagagc ttggcatgga gaccagacag ggctgggtct gcaagcctga 120 ggctgccgcc ctgagctcgg gctgggacgt gcccagaggt gttgggagga tctggggcga 180 gcaccctgtg gccaggacta aaggggctnc acccccctgt ccaCccctcg cagatcttga 240 gcaatgcccg gttatttctg gagaacctca tcaagtatgg catcctggtg gaccccatcc 300 aggtggtctc tctgttcctg aaggatccct atagctggcc cgccccatgc ctggttattg 360 cggccaangt ctttgctgtg gctgcattcc aggttgagaa gcgcctggcg gtgggtgccc 420 tgacggagca ggcgggactg ctgccgcacg tggccaacct ggccaccatt ctgtgcttcc 480 cagcggccgt ggtcttactg gttgagtcta tcactccagt gggctccctg ctggcgctga 540 tggcgcacac catcctcttc ctcaagctct tcccctaccg cgacgtcaac tcatggtgcc 600 gcagggccag ggccaaggct gcctcCgcag ggaagaaggc cagcagtgct gctgccccgc 660 acaccgtgag ctacccggac aatctgaccc accgcgatcc ctactacttc ctcttcgccc 720 ccaccttgtg ctacgagctc aaccttcccc gctctccccg catccggaag cgctttctgc 780 tgcgacggat ccctgagatg ctgttcttca cccagctcca ggtggggctg atccagcagc 840 ggatggtccc caccatccag aactccatga agcccttcaa ggacatggac tacccacgca 900 tcatcgagcg cctcctgaag ctggcggtcc ccaatcacct catctggcCc aCcttcttct 960 accggctctt ccactcctgc ctgaaCgccg Cggctgagct catgcagttt ggagaccggg 102 0 agttctaccg ggactggtgg aactccgagt ctgtcaccta cttctggcag aactggaaca 1080 tcccCgtgca caagtggtgc atcagacact tctacaagcc catgctCcga cggggcagca 1140 gcaagCggaC ggccaggaca ggggCgCCcc CggccCcggc cctcttccac gagtacctgg 1200<br><br>
tgagcgtccc tctgcgaatg ttccgcctct gggcgttcac gggcatgaCg gcccagatcc 12S0<br>
cactggcctg gttcgtgggc cgctttttcc agggcaacta tggcaacgca gctgtgtggc 1320<br>
cgccgctcaC catcggacag ccaatagccg Ccctcatgta cgtccacgac tacCacgtgc 1330<br>
tcaactat-ga ggccccagcg gcagaggcct gagctgcacc tgagggcccg gcttctcact 1440<br>
gccacctcac acccgctgcc agagcccacc tctcctccta ggcctcgagt gctggggatg 1500<br>
ggcctggctg cacagcatcc tcctctggtc ccagggaggc ctctctgccc ctatggggct 1560<br>
ctgtcctgca cccctcaggg atggcgacag caggccagac acagtctgat gccagctggg 1620<br>
agtcttgctg accctgcccc gggtccgagg gtgtcaataa agtgctgtcc agcgacctct 1580<br>
tcagcctgcc aggggcctgg ggcctggtgg ggggtatggc cacacccaca agggcgagcg 1740<br>
ccagagccgt gtggacagct gtcccaggac ctgccgggga gcagcagctc cactgcagca laoo<br>
gggcgggcat ggccggtagg gggagtgcaa ggccaggcag acgcccccat tccccacact 1860<br>
cccctaccta gaaaagctca gctcaggcgt cctct	1895<br> 33<br> l76o<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Inferred cDNA sequence<br> 33<br>
cacgactggg ccgcgacgtg gtgcgggccg aagccatggg cgaccgcgga ggcgcgggaa 60<br>
gctctcggcg tcggaggacc ggctcgcggg tttccatcca gggtggtagt gggcccatgg 12 0<br>
tagacgaaga ggaggtgcga gacgccgctg tgggccccga cttgggcgcc gggggtgacg 190<br>
ctccggctcc ggctccggtt ccggctccag cccacacccg ggacaaagac cggcagacca 240<br>
gcgtgggcga cggccactgg gagctgaggt gccatcgtct gcaagactct ttgttcagct 300<br>
cagacagcgg tttcagcaat taccgtggta tcctgaattg gtgcgtggtg atgctgatcc 360<br>
tgagtaatgc aaggttattt ttagagaatc ttatcaagta tggcatcctg gCggatccca 420<br>
tccaggtggc gtctctgttt ctgaaggacc cctacagctg gcctgcccca tgcttgatca 480<br>
ttgcatccaa tatctttact gtggctacat ttcagattga gaagcgcctg tcagtgggtg 540<br>
ccctgacaga gcagatgggg ctgctgctac atgtggttaa cctggccaca attatctgct 600<br>
tcccagcagc tgtggcctta ctggttgagt ctatcactcc agtgggttcc ctgtttgctc 650<br>
tggcatcata ctccatcatc ttcctcaagc ttttctccta ccgggatgtc aatctgtggt 72 0<br>
gccgccagcg aagggtcaag gccaaagctg tgtctgcagg gaagaaggtc agtggggctg 780<br>
ctgcccagaa cactgtaagc tatccggaca acctgaccta ccgagatctc tattactcca 840<br>
tctttgctcc tactttgtgt tatgaactca actttcctcg atccccccga atacgaaagc 900<br>
gctttctgct acggcgggtt cttgagatgc tctttttcac ccagcttcaa gtggggctga 960<br>
tccagcagtg gatggtccct actatccaga actccatgaa gcccttcaag gacatggact 1020<br>
attcacgaac cattgagcgt ctcttaaagc tggcggtccc caaccatctg atacggctca 1080<br>
tcttcttcta ttggcttttc cactcatgtc tcaatgctgt ggcagagctc ctgcagtttg 1140<br>
gagaccgcga gttctacagg gactggCgga atgctgagtc tgtcacctac ttttggcaga 1200<br>
actggaataC ccccgtgcac aagtggtgca tcagacactt ctacaagcct atgctcagac 1260<br>
tgggcagcaa caaacggatg gccaggactg gggtcttttt ggcgtcagcc ttcttccatg 1320<br>
agtacctagC gagcattccc ctgaggatgt tccgcctctg ggcattcaca gccatgatgg 1380<br>
ctcaggtccc actggcctgg attgtgaacc gcttcttcca agggaactat ggcaatgcag 1440<br>
ctgtgtgggc gacactcatc attgggcaac cggtggctgt gctcatgtat gtccacgact IS 00<br>
actacgtgct caactatgat gccccagtgg gggcctgagc tactgccaaa ggccagccct 1560<br>
Gcctaacctg ggcctggagt tctggagggc ttcctggctg cctgcacact cctcctagtc 1620<br>
tgggaggcct ctctgcccct atggggccta ctcctgctct tggggatggc acctgagtcc 1680<br>
agctggtatg agccagtgct gggagtctgt gctgaccagg ggctgaggat atcaataaag 1740<br>
agctatctaa aaaaaaaaaa aaaaaa	1766<br><br>
c210&gt; 34<br> PRT<br> Homo sapiens<br> 34<br>
Arg Arg Ser Leu Leu Asp Glu Leu Leu Glu Val Asp His He Arg Thr<br>
IS	10	15<br>
He Tyr His Met Phe He Ala Leu Leu He Leu Phe He Leu Ser Thr<br>
20	25	30<br>
Leu Val Val Asp Tyr He Asp Glu Gly Arg Leu Val Leu Glu Phe Ser<br>
35	40	45<br>
Leu Leu Ser Tyr Ala Phe Gly Lys Phe Pro Thr Val Val Trp Thr Trp<br>
50	55	60<br>
Trp He Met Phe Leu Ser Thr Phe Ser Val Pro Tyr Phe Leu Phe Gin<br>
65	70	75	80<br>
His Trp Arg Thr Gly Tyr Ser Lys Ser Ser His Pro Leu lie Arg Ser<br>
as	90	95<br>
Leu Phe His Gly Phe Leu Phe Met He Phe Gin He Gly val Leu Gly<br>
100	105	110<br>
Phe Gly Pro Thr Tyr Val Val Leu Ala Tyr Thr Leu Pro Pro Ala Ser<br>
115	120	125<br>
Arg Phe He He He Phe Glu Gin He Arg Phe Val Het Lys Ala His<br>
130	135	140<br>
Ser Phe Val Arg Glu Asn Val Pro Arg Val Leu Asn Ser Ala Lys Glu<br>
145	150	155	160<br>
Lys Ser Ser Thr Val Pro He Pro Thr Val Asn Gin Tyr Leu Tyr Phe<br>
165	170	175<br>
Leu Phe Ala Pro Thr Leu He Tyr Arg Asp Ser Tyr Pro Arg Asn Pro<br>
180	185	190<br>
Thr Val Arg Trp Gly Tyr Val Ala Met Lys Phe Ala Gin Val Phe Gly<br>
195	200	205<br>
Cys Phe Phe Tyr Val Tyr Tyr He Phe Glu Arg Leu Cys Ala Pro Leu<br>
210	215	220<br>
Phe Arg Asn He Lys Gin Glu Pro Phe Ser Ala Arg Val Leu Val Leu<br>
225	230	235	240<br>
Cys Val Phe Asn Ser He Leu Pro Gly Val Leu He Leu Phe Leu Thr<br>
245	250	255<br>
Phe Phe Ala Phe Leu His Cys Trp Leu Asn Ala Phe Ala Glu Met Leu<br>
260	265	270<br><br>
Arg Phe Gly Asp Arg Met Phe Tyr Lys Asp Trp Trp Asn Ser Thr Ser<br>
275	280	285<br>
Tyr Ser Asn Tyr Tyr Arg Thr Trp Asn Val Val Val His Asp Trp Leu<br>
290	295	300<br>
Tyr Tyr Tyr Ala Tyr Lys Asp Phe Leu Trp Phe Phe Ser Lys Arg Phe<br>
305	310	315	320<br>
Lys Ser Ala Ala Met Leu Ala Val Phe Ala Val Ser Ala Val Val His<br>
325	330	335<br>
Glu Tyr Ala Leu Ala Val Cys Leu Ser Phe Phe Tyr Pro Val Leu Phe<br>
340	345	350<br>
Val Leu Phe Met Phe Phe Gly Met Ala Phe Asn Phe He val Asn Asp<br>
355	360	365<br>
Ser Arg Lys Lys Pro He Trp Asn Val Leu Met Trp Thr Ser Leu Phe<br>
370	375	330<br>
Leu Gly Asn Gly Val Leu Leu Cys Phe Tyr Ser Gin Glu Trp Tyr Ala<br>
385	390	395	40o<br>
Arg Arg His Cys Pro Leu Lys Asn Pro 405<br> 35<br> 409<br> PRT<br> Mus tnusculus<br> 35<br>
Arg Gin Ser Leu Leu Asp Glu Leu Phe Glu Val Asp His He Arg Thr<br>
15	10	15<br>
He Tyr His Met Phe He Ala Leu Leu He Leu Phe Val Leu Ser Thr<br>
20	25	30<br>
He Val Val Asp Tyr He Asp Glu Gly Arg Leu Val Leu Glu Phe Aan<br>
35	40	45<br>
Leu Leu Ala Tyr Ala Phe Gly Lys Phe Pro Thr Val He Trp Thr Trp<br>
50	55	60<br>
Trp Ala Met Phe Leu Ser Thr Leu Ser He Pro Tyr Phe Leu Phe Gin<br>
65	70	75	ao<br>
Pro Trp Ala His Gly Tyr Ser Lys Ser Ser His Pro Leu He Tyr Ser<br>
35	90	95<br>
Leu Val His Gly Leu Leu Phe Leu Val Phe Gin Leu Gly Val Leu Gly<br>
100	105	110<br><br>
Phe Val Pro Thr Tyr Val Val _Leu Ala Tyr Thr Leu Pro Pro Ala Ser<br>
115	""i20 "	125<br>
Arg Phe lie Leu lie Leu Glu Gin lie Arg Leu lie Met Lys Ala His<br>
130	135	140<br>
Ser Phe Val Arg Glu Asn lie Pro Arg Val Leu Asn Ala Ala Lys Glu<br>
145	150	155	160<br>
Lys Ser Ser Lys Asp Pro Leu Pro Thr Val Asn Gin Tyr Leu Tyr Phe<br>
165	170	175<br>
Leu Phe Ala Pro Thr Leu lie Tyr Arg Asp Asn"Tyr Pro Arg Thr Pro<br>
130	laS	190<br>
Thr Val Arg Trp Gly Tyr Val Ala Met Gin Phe Leu Gin Val Phe Gly<br>
195	200	205<br>
Cys Leu Phe Tyr Val Tyr Tyr lie Phe Glu Arg Leu Cys Ala Pro Leu<br>
210	215	220<br>
Phe Arg Asn lie Lys Gin Glu Pro Phe Ser Ala Arg Val Leu Val Leu<br>
225	230	235	240<br>
Cys Val Phe Asn Ser lie Leu Pro Gly Val Leu lie Leu Phe Leu Ser<br>
245	250	255<br>
Phe Phe Ala Phe Leu His Cys Trp Leu Asn Ala Phe Ala Glu Met Leu<br>
260	265	270<br>
Arg Phe Gly Asp Arg Met Phe Tyr Lys Asp Trp Trp Asn Ser Thr Ser<br>
275	280	285<br>
Tyr Ser Asn Tyr Tyr Arg Thr Trp Asn Val Val Vai His Asp Trp Leu<br>
290	295	300<br>
Tyr Tyr Tyr Val Tyr Lys Asp Leu Leu Trp Phe Phe Ser Lys Arg Phe<br>
305	310	315	320<br>
Lys Ser Ala Ala Met Leu Ala Val Phe Ala Leu Ser Ala Val Val His<br>
325	330	335<br>
Glu Tyr Ala Leu Ala lie Cys Leu Ser Tyr Phe Tyr Pro Val Leu Phe<br>
340	345	350<br>
Val Leu Phe Met Phe Phe Gly Met Ala Phe Asn Phe He Val Asn Asp<br>
355	360	365<br>
Ser Arg Lys Arg Pro He Trp Asn He Met Val Trp Ala Ser Leu Phe<br>
370	375	380<br>
Leu Gly Tyr Gly Leu He Leu Cys Phe Tyr Ser Gin Glu Trp Tyr Ala<br>
385	390	395	400<br>
Arg Gin His Cys Pro Leu Lys Asn Pro 405<br><br> 36<br> 429<br> PRT<br> Saccharomyces cerevisiae<br>
Asp Lys Ala Asp Ala Pro Pro Gly Glu Lys Leu Glu Ser Asn Phe Ser<br>
15	10	IS<br>
Gly He Tyr Val Phe Ala Trp Met Phe Leu Gly Trp lie Ala He Arg<br>
20	25	30<br>
Cys Cys Thr Asp Tyr Tyr Ala Ser Tyr Gly Ser Ala Trp Asn Lys Leu<br>
35	40	45<br>
Glu He Val Gin Tyr Met Thr Thr Asp Leu Phe Thr He Ala Met Leu<br>
50	55	60<br>
Asp Leu Ala Met Phe Leu Cys Thr Phe Phe Val Val Phe Val His Trp<br>
65	70	75	80<br>
Leu Val Lys Lys Arg He He Asn Trp Lys Trp Thr Gly Phe Val Ala<br>
85	30	95<br>
Val Ser He Phe Glu Leu Ala Phe He Pro Val Thr Phe Pro He Tyr<br>
100	105	110<br>
Val Tyr Tyr Phe Asp Phe Asn Trp Val Thr Arg He Phe Leu Phe Leu<br>
115	120	125<br>
His Ser Val Val Phe Val Met Lys Ser His Ser Phe Ala Phe Tyr Asn<br>
130	135	140<br>
Gly Tyr Leu Trp Asp He Lys Gin Glu Leu Glu Tyr Ser Ser Lys Gin<br>
145	150	155	160<br>
Leu Gin Lys Tyr Lys Glu Ser Leu Ser Pro Glu Thx Arg Glu He Leu<br>
165	170	175<br>
Gin Lys Ser Cys Asp Phe Cys Leu Phe Glu Leu Asn Tyr Gin Thr Lys<br>
180	IBS	190<br>
Asp Asn Asp Phe Pro Asn Asn He Ser Cys Ser Asn Phe Phe Met Phe<br>
195	200	205<br>
Cys Leu Phe Pro Val Leu Val Tyr Gin He Asn Tyr Pro Arg Thr Ser<br>
210	215	220<br>
Arg He Arg Trp Arg Tyr Val Leu Glu Lys Val Cys Ala He He Gly<br>
225	230	23S	240<br>
Thr He Phe Leu Met Met Val Thr Ala Gin Phe Phe Met His Pro Val<br>
245	250	255<br>
Ala Met Arg Cys He Gin Phe His Asn Thr Pro Thr Phe Gly Gly Trp<br>
260	265	270<br><br>
lie Pro Ala Thr Gin Glu Trp Phe His Leu Leu Phe Asp Met He Pro<br>
275 â– 	280	2SS<br>
Gly Phe Thr Val Leu Tyr Met Leu Thr Phe Tyr Met He Trp Asp Ala<br>
290	295	300<br>
Leu Leu Asn Cys Val Ala Glu Leu Thr Arg Phe Ala Asp Arg Tyr Phe<br>
305	310	315	320<br>
Tyr Gly Asp Trp Trp Asn Cys Val Ser Phe Glu Glu Phe Ser Arg lie<br>
325	330	335<br>
Trp Asn Val Pro Val His Lys Phe Leu Leu Arg His Val Tyr His Ser<br>
340	345	350<br>
Ser Met Gly Ala Leu His Leu Ser Lys Ser Gin Ala Thr Leu Phe Thr<br>
355	3S0	365<br>
Phe Phe Leu Ser Ala Val Phe His Glu Met Ala Met Phe Ala He Phe<br>
370	375	380<br>
Arg Arg Val Arg Gly Tyr Leu Phe Met Phe Gin Leu Ser Glr Phe Val<br>
385	390	395	400<br>
Trp Thr Ala Leu Ser Asn Thr Lys Phe Leu Arg Ala Arg Pro Gin Leu<br>
405	410	415<br>
Ser Asn Val Val Phe Ser Phe Gly Val Cys Ser Gly Pro<br>
420	425<br>	37<br>	432<br>	PRT<br> 37<br>
Glu Thr Val Val Thr Val Glu Thr Thr He He Ser Ser Asn Phe Ser<br>
15	10	15<br>
Gly Leu Tyr Val Ala Phe Trp Met Ala He Ala Phe Gly Ala Val Lys<br>
20	25	30<br>
Ala Leu He Asp Tyr Tyr Tyr Gin His Asn Gly Ser Phe Lys Asp Ser<br>
35	40	45<br>
Glu He Leu Lys Phe Met Thr Thr Asn Leu Phe Thr Val Ala Ser Val<br>
50	55	60<br>
Asp Leu Leu Met Tyr Leu Ser Thr Tyr Phe Val Val Gly He Gin Tyr<br>
65	70	75	80<br>
Leu Cys Lys Trp Gly Val Leu Lys Trp Gly Thr Thr Gly Trp He Phe<br>
85	90	95<br><br>
Thr Ser He Tyr Glu Phe Leu Phe Val He Phe Tyr Met Tyr Leu Thr<br>
100	105	110<br>
Glu Asn He Leu Lys Leu His Trp Leu Ser Lys He phe Leu Phe Leu<br>
115	120	125<br>
His Ser Leu Val Leu Leu Met Lys Met His Ser Phe Ala Phe Tyr Asn<br>
130	135	140<br>
Gly Tyr Leu Trp Gly He Lys Glu Glu Leu Gin Phe Ser Lys Ser Ala<br>
145	150	155	ISO<br>
Leu Ala Lys Tyr Lys Asp Ser He Asn Asp Pro Lys Val He Gly Ala<br>
165	170	175<br>
Leu Glu Lys Ser Cys Glu Phe Cys Ser Phe Glu Leu Ser Ser Gin Ser<br>
180	185	190<br>
Leu Ser Asp Gin Thr Gin Lys Phe Pro Asn Asn He Ser Ala Lys Ser<br>
195	200	205<br>
Phe Phe Trp Phe Thr Met Phe Pro Thr Leu He Tyr Gin He Glu Tyr<br>
210	215	220<br>
Pro Arg Thr Lys Glu lie Arg Trp Ser Tyr Val Leu Glu Lys He Cya<br>
225	230	235	240<br>
Ala He Phe Gly Thr He Phe Leu Met Met He Asp Ala Gin He Leu<br>
245	250	255<br>
Met Tyr Pro Val Ala Met Arg AJa Leu Ala Val Arg Asn Ser Glu Trp<br>
260	265	270<br>
Thr Gly He Leu Asp Arg Leu Leu Lys Trp Val Gly Leu Leu Val Asp<br>
275	280	285<br>
He Val Pro Gly Phe He Val Met Tyr He Leu Asp Phe Tyr Leu He<br>
290	295	300<br>
Trp Asp Ala He Leu Asn Cys Val Ala Glu Leu Thr Arg Phe Gly Asp<br>
305	310	315	320<br>
Arg Tyr Phe Tyr Gly Asp Trp Trp Asn Cys Val Ser Trp Ala Asp Phe<br>
325	330	335<br>
Ser Arg He Trp Asn He Pro Val His Lys Phe Leu Le- Arg His Val<br>
340	345	350<br>
Tyr His Ser Ser Met Ser Ser Phe Lys Leu Asn Lys Ser Gin Ala Thr<br>
355	360	365<br>
Leu Met Thr Phe Phe Leu Ser Ser Val Val His Glu Leu Ala Met Tyr<br>
370	375	380<br>
Val He Phe Lys Lys Leu Arg Phe Tyr Leu Phe Phe Phe Gin Met Leu<br>
385	390	395	400<br><br>
Gin Met Pro Leu Val Ala Leu Thr Asn Thr Lys Phe Met Arg Asn Arg<br>
405	410	415<br>
Thr lie He Gly Asn Val He Phe Trp Leu Gly He Cys Met Gly Pro<br>
420	425	430<br> 38<br> 1942<br> Arabidopsia thaliana<br> 38<br>
ctctcgtgaa tcctttttcc tttcttcttc ttcttctctt cagagaaaac tttgcttctc 60<br>
tttctataag gaaccagaca cgaatcccat tcccaccgat ttcttagctt cttccttcaa 120<br>
tccgctcttt ccctctccat tagattctgt ttcctctttc aatttcttct gcatgctcct 180<br>
cgattctctc tgacgcctct tttctcccga cgctgtttcg tcaaacgctt ttcgaaacgg 240<br>
cgattttgga ttctgctggc gttactacgg tgacggagaa cggtggcgga gagttcgtcg 300<br>
aCcttgatag gcttcgtcga cggaaatcga gatcggattc ttctaacgga cttcttccct 360<br>
ctggtCccga taataattct ccttcggatg atgttggagc tcccgccgac gttagggatc 420<br>
ggattgattc cgctgttaac gatgacgctc agggaacagc caatttggcc ggagataata 430<br>
acggtggtgg cgataataac ggtggtggaa gaggcggcgg agaaggaaga ggaaacgccg 540<br>
atgctacgct tacgtatcga ccgtcggttc cagctcaCcg gagggcgaga gagagtccac 600<br>
ttagccccga cgcaatcttc aaacagagcc atgccggatt attcaacctc tgtgtagcag 660<br>
ttcttattgc tgtaaacagt agacCcaCca tcgaaaatct tatgaagtat ggttggttga 720<br>
tcagaacgga tttctggttt agttcaagat cgctgcgaga ttggccgctt ttcatgtgtt 780<br>
gtacatcccc ttcgatcttt cctttggctg cctttacggt tgagaaattg gcacttcaga 840<br>
aatacatacc agaacctgct gtcatctttc ttcatattat tatcaccatg acagaggctc 900<br>
tgtatccagc ttacgtcacc ctaaggtgtg attctgcttt tttatcaggt gtcacttcga 960<br>
tgctcctcac ttgcattgtg tggctaaagt tggtttctta tgctcatacC agctatgaca 102 0<br>
taagacccct agccaatgca gctgataagg ccaatcctga agtctcctac tacgttagct 1080<br>
tgaagagctt ggcatatttc atggtcgctc ccacattgtg ttatcagcca agttatccac 1140<br>
gttctgcatg catacggaag ggttgggtgg ctcgtcaatt Cgcaaaactg gtcatatcca 1200<br>
ccggattcat gggatttata atagaacaat atataaatcc tattgtcagg aactcaaagc 1260<br>
atcctttgaa aggcgatctt ctatatgcta ttgaaagagt gttgaagctt tcagttccaa 1320<br>
atttatatgt gtggctctgc atgttctact gcttcttcca cctttggtta aacatatcgg 1380<br>
cagagcttct ctgcttcggg gatcgtgaat tctacaaaga ttggtggaat gcaaaaagtg 1440<br>
tgggagatta ctggagaatg tggaatatgc ctgttcataa atggatggtt cgacatatat 1500<br>
acttcccgtg cttgcgcagc aagataccaa agacactcgc cattatcatt gctttcccag 1560<br>
tctctgcagt ctttcatgag cCatgcatcg cagttccttg tcgtctcttc aagctatggg 1620<br>
cttttcttgg gattatgttt caggtgcctt tggtcttcat cacaaactat ctacaggaaa 1680<br>
ggtttggctc aacggtgggg aacatgatct tctggttcat cttctgcatt ttcggacaac 1740<br>
cgatgtgtgt gcttctttat taccacgacc tgatgaaccg aaaaggatcg atgtcatgaa 1300<br>
acaactgttc aaaaaatgac tttcttcaaa catctatggc ctcgttggat ctccgttgat 1860<br>
gttgtggtgg ttctgatgct aaaacgacaa atagtgttat aaccattgaa gaagaaaaga 1920<br>
caattagagt tgttgtatcg ca	1942<br> 39<br> 520<br> PRT<br> Arabidopsis thaliana<br><br> 39<br>
Met Ala He Leu Asp Ser Ala Gly Val Thr Thr Val Thr Glu Asn Gly<br>
15	10	15<br>
Gly Gly Glu Phe Val Asp Leu Asp Arg Leu Arg Arg Arg Lys Ser Arg<br>
20	25	30<br>
Ser ASp Ser Ser Asn Gly Leu Leu Leu Ser Gly Ser Asp Asn Asn Ser<br>
35	40	45<br>
Pro Ser Asp Asp Val Gly Ala Pro Ala Asp Val Arg Asp Arg He Asp<br>
SO	55	60<br>
Ser Val Val Asn Asp Asp Ala Gin Gly Thr Ala Asn Leu Ala Gly Asp<br>
65	70	75	80<br>
Asn Asn Gly Gly Gly Asp Asn Asn Gly Gly Gly Arg Gly Gly Gly Glu<br>
85	90	95<br>
Gly Arg Gly Asn Ala Asp Ala Thr Phe Thr Tyr Arg Pro Ser Val Pro<br>
100	105	110<br>
Ala His Arg Arg Ala Arg Glu Ser Pro Leu Ser Ser Asp Ala He Phe<br>
115	12 0	125<br>
Lys Gin Ser His Ala Gly Leu Phe Asn Leu Cys Val Val Val Leu He<br>
130	135	140<br>
Ala Val Asn Ser Arg Leu He He Glu Asn Leu Met Lys Tyr Gly Trp<br>
1-15	150	155	160<br>
Leu He Arg Thr Asp Phe Trp Phe Ser Ser Arg Ser Leu Arg Asp Trp<br>
165	170	175<br>
Pro Leu Phe Met Cys Cys He Ser Leu Ser He Phe Pro Leu Ala Ala<br>
180	185	190<br>
Phe Thr Val Glu Lys Leu Val Leu Gin Lys Tyr He Ser Glu Pro Val<br>
195	200	205<br>
Val He Phe Leu His He He He Thr Met Thr Glu Val Leu Tyr Pro<br>
210	215	220<br>
Val Tyr Val Thr Leu Arg Cys Asp Ser Ala Phe Leu Ser Gly Val Thr<br>
225	230	235	240<br>
Leu Met Leu Leu Thr Cys He Val Trp Leu Lys Leu Val Ser Tyr Ala<br>
245	250	255<br>
His Thr Ser Tyr Asp He Arg Ser Leu Ala Asn Ala Ala Asp Lys Ala<br>
260	265	270<br>
Asn Pro Glu Val Ser Tyr Tyr Val Ser Leu Lys Ser Leu Ala Tyr Phe<br>
275	280	285<br><br>
Met Val Ala Pro Thr Leu Cys Tyr Gin Pro Ser Tyr Pro Arg Ser Ala<br>
290	"   295	300<br>
Cys He Arg Lys Gly Trp Val Ala Arg Gin Phe Ala -Lys Leu Val He<br>
305	310	315	320<br>
Phe Thr Gly Phe Met Gly Phe He He Glu Gin Tyr He Asn Pro He<br>
325	330	335<br>
Val Arg Asn Ser LyS His Pro Leu Lys Gly Asp Leu Leu Tyr Ala He<br>
340	34S	350<br>
Glu Arg Val Leu Lys Leu Ser Val Pro Asn Leu Tyr Val Trp Leu Cys<br>
355	3fi0	365<br>
Met Phe Tyr Cys Phe Phe His Leu Trp Leu Asn He Leu Ala Glu Leu<br>
370	375	380<br>
Leu Cys Phe Gly Asp Arg Glu Phe Tyr Lys Asp Trp Trp Asn Ala Lys<br>
385	390	395	400<br>
Ser Val Gly Asp Tyr Trp Arg Met Trp Asn Met Pro Val His Lys Trp<br>
405	410	415<br>
Met Val Arg His He Tyr phe Pro Cys Leu Arg Ser Lys He Pro Lys<br>
420	425	430<br>
Thr Leu Ala He He He Ala Phe Leu Val Ser Ala Val Phe His Glu<br>
435	440	445<br>
Leu Cys He Ala Val Pro Cys Arg Leu Phe Lys Leu Trp Ala Phe Leu<br>
450	455	460<br>
Gly He Met Phe Gin Val Pro Leu Val Phe He Thr Aan Tyr Leu Gin<br>
465	470	475	480<br>
Glu Arg Phe Gly Ser Thr val Gly Asn Met He Phe Trp Phe lie Phe<br>
485	490	495<br>
Cys He Phe Gly Gin Pro Met Cys Val Leu Leu Tyr Tyr His Asp Leu<br>
500	505	510<br>
Met Asn Arg Lys Gly Ser Met Ser<br>
515	520<br> 40<br> 29<br> DNA<br> Artificial Sequence<br><br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 40<br>
tgcaaattga cgagcacacc aaccccttc<br> 41  26  DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonuclotide primer<br> 41<br>
aaggatgctt tgagttcctg acaatagg<br> 42<br> 1942<br> DNA<br> Arabidopsis thaliana<br> 42<br>
ctctcgtgaa tcctttttcc tttcttcttc ttcttctctt cagagaaaac tttgcttctc 60 tttctataag gaaccagaca cgaatcccat tcccaccgat ttcttagctc cttccttcaa 120 tccgctcttt ccctctccat tagattctgt ttcctctttc aatttcttcc gcatgcttct 180 cgattctctc tgacgcctct tctctcccga cgctgtttcg tcaaacgctt ttcgaaatgg 240 cgactttgga ttctgctggc gttactacgg tgacggagaa cggtggcgga gagttcgtcg 300 atcctgatag gcttcgtcga cggaaatcga gatcggattc ttctaacgga cttcttctct 360 ctggttccga taataattct ccttcggatg-atgttggagc tcccgccgac gttagggatc 420 ggattgattc cgttgttaac gatgacgctc agggaacagc caatttggcc ggagataaCa 480 acggtggtgg cgataataac ggtggtggaa gaggcggcgg agaaggaaga ggaaacgccg 540 atgctacgtt tacgCatcga ccgtcggttc cagctcatcg gagggcgaga gagagtccac 600 ctagctccga cgcaatcttc aaacagagcc atgccggatt attcaacctc tgtgtagtag 660 ttcttattgc tgtaaacagt agactcatca tcgaaaatct taCgaagtat ggttggttga 720 tcagaacgga tttctggttt agttcaagat cgctgcgaga ttggccgctt ttcatgtgtt 780 gtatatccct ttcgatcttt cctttggctg cctttacggt tgagaaattg gtacttcaga 840 aatacatatc agaacctgtt gccatctttc ttcatattat taccaccatg acagaggttt 900 tgtatccagt ttacgtcacc ccaaggtgtg attctgcttt tttatcaggt gtcaccttga 960 tgctcctcac ttgcattgtg tggctaaagt Cggtttctta tgctcatact agctatgaca 1020 taagatccct agccaatgca gctgataagg ccaatcctga agtctcctac tacgttagct 1080 tgaagagctt ggcatatttc acggtcgctc ccacattgtg ttatcagcca agttatccac 114 0 gttctgcatg tatacggaag ggttgggtgg ctcgtcaatt tgcaaaactg gtcatattca 1200 ccggattcat gggatttata atagaacaat atataaatcc tattgtcagg aactcaaagc 1260 atccttcgaa aggcgatctt ctatatgcta ttgaaagagt gttgaagctt tcagttccaa 1320 atttatatgt gtggctctgc acgttctact gcttcttcca cctttggtta aacatattgg 1380 cagagcttct ctgcttcggg gatcgtgaat tctacaaaga ttggtggaat gcaaaaagtg 1440 tgggagatta ctggagaatg tggaatatgc ctgttcataa atggatggtt cgacatatat 1500 acttcccgtg cttgcgcagc aagataccaa agacactcgc cattatcatt gctttcctag 15S0 tctctgcagt ctttcatgag ctatgcatcg cagttccttg tcgtctcttc aagctatggg 1620 cttttcttgg gattatgttt caggtgcctt tggtcttcat cacaaactat ctacaggaaa 1630 ggtttggctc aacggtgggg aacatgatct tctggctcat cttctgcatt ttcggacaac 1740<br><br>
cgatgtgtgt gcStctttat cacqacgacc tgatgaaccg aaaaggatcg acgtcatgaa laoo<br>
acaactgttc aaaaaatgac cttctccaaa catctatggc ctcgttggat ctccgttgat 1860<br>
gttgtggtgg ttctgatgct aaaacgacaa atagtgCtat aaccattgaa gaagaaaaaa 1920<br>
caattagagt tgttgtatcg ca	1942<br>	43<br>	234<br>	UNA<br>	Glycine max<br><br> unsure  (1),.(234)  n=unknown<br> 43<br>
gtaagcttca agagcttagc atanttcctg gttgccccta	ncattatgtt accagccaan 60<br>
ctatcctcgc acaccttata ctcgaaaggg ttggctgttt	cgccaacttg tcaactgata 120<br>
atatttacag gagttatggg atttataata gaacaataca	ttaatcccat tgtacaaaat 180<br>
tcacagcatc ctctcaaggg aaaccttctt tacgccatcg	agagagttct gaag      234<br> 44<br> 267<br> Glycine max<br> 44<br>
ctgcttttgt atctggtgtc acgttgatgc	tactaacttg	cattgtgtgg ttaaaattgg 60<br>
tgccatatgc acatacaaac tatgatatga	gagcacttac	tgtttcgaac gaasagggag 120<br>
aaacattacc caatactttg atatggagta	tccgtacacc	gtgaccttca ggagtttggc 180<br>
atacttcatg gttgctccta cattatgcta	tcagacaagc	tatcctcgca caccttcagt 240<br>
tcgaaagggt tgggtgcttc gtcaact	267<br> 45<br> 275<br> DMA<br> Glycine max<br><br> unsure  (1)..(275)  n=unknown<br> 45<br>
gtggaatgcc aaaactgttg aagattattg gaggatgtgg aatatgcctg ttcacaaatg 60<br>
gatgatccgc cacctatatt ttccatgttt aaggcacggt ataccaaagg ccgttgctct 120<br>
tttaattgcc ttcctggttc tgctttattc catgagctgt gcatcgctgc tccttgccca 180<br>
catattcaag tngtgggttt cngnggaatt nagtttcagg tnccttgggt ttcnaccnna 240<br>
attimtnggc naaaaaattc cnngaacccc ggggg	275<br><br>	4S<br>	257<br>	DNA<br>	Glycine max<br> 46<br>
aacggaactg agactccaga gaatatgcca aaatgtatta ataattgtca caacttggaa 60<br>
ggcttttgga aaaactggca tgcttccttc aacaagtggc ttgtgaggta tatatacatt 12 0<br>
cctcttgggg gatctaagaa aaagctacta aatgtgtggg ttgttttcac acttgttgca lao<br>
atctggcatg atttagagtg gaagcttctt tcatgggcat ggttgacgtg tctattcttc 240<br>
atccctgagt tggtttt	257<br> 47  253  DNA 

agaaaatgga acatgcctgt gcataaatgg attgttcgtc atatatattt tccttgcatg 60<br>
cgaaatggta tatcaaagga agttgctgtt tttatatcgt tcttgtttct gctgtacttc 120<br>
atgagttatg tgttgctgtt ccctgccaca tactcaagtt ctgggctttt tttaggaatc 130<br>
atgcttcaga ttcccctcat catattgaca tcatacctca aaaataaatt cagtgacaca 240<br>
atggttggca ata	253<br>
tgaagtatgg cttattaata agatctggct cttggtttaa tgctacatca	ttgcgagact 60<br>
ggccactgct aatgtgttgc cttagtctac ccatatttcc ccttggtgca	tttgcagtcg 120<br>
aaaagttggc attcaacaat ctcattagtg atcctgctac tacctgtttt	cacatccttt ISO<br>
ttacaacatt tgaaattgta tatccagtgc tcgtgattct taagtgtgat	tctgcagttt 240<br>
tatcaggctt tgtg	254<br> 49  262  DNA  Zea mays<br> 49<br>
gaagtatggc ttattaataa gatctggctt ttggtttaat gctacatcat tgcgagactg 60<br>
gccactgcta atgtgttgcc ttagtctacc catatttccc cttggtgcat ttgcagtcga 120<br>
aaagttggca ttcaacaatc tcattagtga tcctgctact acctgttttc acatcctttt 180<br>
tacaacattt gaaattgtat atccagtgct cgtgattctt aagtgtgatt ctgcagtttt 240<br>
acaggctttg tgttgatgtt ta	262<br>	SO<br>	325<br>	DHA<br>	Zea mays<br><br><br>  unsure     {1}..(325)   n=unknown<br>   50<br>
taatcnaacc tcgntncngg ctcagctgta tnccacgaga tatgtaatgc ggtgccgtgc 60<br>
cacatantca nacctnggca tnncngggat catngctcag ataccgntgg nactcctgac 12 0<br>
aagatatctc caCgctacgt tcaagcaCgt aatggtgggc aacatgatan ttcggntcnn 180<br>
cagtatagtc ggacagccga tgtimimima tctatactac catgacgtca tgaacaggca 240<br>
ggcccaggca agtagatagt ncggcagaga catgtacttc aacatcganc atcagnagca 300<br>
nacngagcga gcggcangaa ncagc	32 5<br> 51<br> 519<br> DNA<br> Mortierrella alpina<br>  unsure     (1)..{519)   n=uiiknown<br>  51<br>
gagnnnngna acgtttagcc tnccgtagcc gccaaaatcc aagggncnac cnaccctncg 60<br>
ttanactnaa ttngaaaatn crmncccaac ttnaggnact tnnagncccc ccnacttgac 12 0<br>
aacggagcac tatatttacc ccgtggtngt tcaacccagc catctcaccc ttgcgagcat 180<br>
tggtgctgct ctCgataccc ttcatgctta actatctcat gatcttttac atcattttcg 240<br>
agtgcatctg csacgccttt gcggaactaa gttgctttgc ggatcgcaac ttttacgagg 300<br>
atcggtggaa ctgcgtcagc tttgatgagt gggcacgcaa atggascaag cctgtgcaac 360<br>
acttcttgct ccgccacgtg tacgacccga gcatccgagt ccttccactt gtccgaaatc 420<br>
caatgccgcn aattgcaaac gttccttccc ggtcgtcaat gcgttcaacg aacctgggtg 480<br>
aagaatgggt ggtgacaacg ttaaagtgcg cccggtatc	513<br> 52<br> 45<br>
c212&gt; DSA<br> Artificial Sequence<br><br>  Description of Artificial  Sequence: Oligonucleotide primer<br>   52<br>
ggatccgcgg ccgcacaatg aaaaaaatat cttcacatta ttcgg<br> 53<br> 40<br> DNA<br>
c213&gt; Artificial Sequence<br><br><br> Description of Artificial Sequence: Oligonucleotide primer<br> S3<br>
ggatcccctg caggtcattc attgacggca ttaacattgg<br> 54<br> 44<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence; Synthetic oligonucleotide primer<br> 54<br>
ggatccgcgg ccgcacaatg ggagcgaatt cgaaatcagt aacg<br> 55  40  DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 55<br>
ggatcccctg caggttaata cccactttta tcaagctccc<br> 56<br>
":211&gt; 41<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence-. Synthetic oligonucleotide primer<br> 56<br>
ggatccgcgg ccgcacaatg tctctattac tggaagagat c<br> 57<br> 41<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br><br>
c400i 57<br>
ggatcccctg caggttatgc atcaacagag acacttacag c<br> 58<br> 41<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 53<br>
ggatccgcgg ccgcacaatg ggctggattc cgtgtccgtg c<br> 59<br> 38<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 59<br>
ggatcccctg caggttaacc  agaatcaact  actctgtg<br>
c210&gt;   60<br>   39<br>   DNA<br> Artificial   Sequence<br><br> Description of Artificial Sequence; Oligonucleotide primer<br>
c400&gt; 60<br>
ccgacctgca ggaagcttag aaatggcgat tttggattc<br> 61<br> 36<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence; Oligonucleotide primer<br>
c400&gt; 61<br>
ggatccgcgg  ccgctcatga catcgatcct  tttcgg<br><br> 62  56<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence; Annealed oligonucleotide adapter<br> 62<br>
cgcgatttaa atggcgcgcc ctgcaggcgg ccgcctgcag ggcgcgccat ttaaac<br> 63<br> 32<br> DKA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 63<br>
tcgaggatcc gcggccgcaa gcttcctgca gg<br> 64<br> 32<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 64<br>
tcgacctgca ggaagcttgc ggccgcggat cc<br>	65<br>	32<br>	DHA<br>	Artificial Sequence<br>
â– ;220&gt;<br> Description of Artificial Sequence: Ligating oligonucleotide<br> 65<br>
tcgacctgca ggaagcttgc ggccgcggat cc<br><br> 66<br> 32<br> DNA<br> Artificial Sequence<br>
t;220&gt;<br> Description of Artificial Sequence; Ligating oligonucleotide<br> 66<br>
tcgaggaccc gcggccgcaa gcttcctgca gg<br> 67<br> 36<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 67<br>
tcgaggatcc gcggccgcaa gcttcctgca ggagct<br> 6a<br> 28<br> DNA<br> Artificial Sequence<br><:220><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 68<br>
cctgcaggaa gcttgcggcc gcggatcc<br> 69<br> 36<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Ligating o1i gonuc1eot i de<br> 63<br>
tcgacctgca ggaagcttgc ggccgcggat ccagct<br><br> 70<br> 2B<br> DHA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 70<br>
ggatccgcgg ccgcaagctc cctgcagg<br> 71<br><:211> 39<br> DKA<br> Artificial Sequence<br> Description of Artificial Sequence: Ligating oligonucleotide<br> 71<br>
gatcacctgc aggaagcttg cggccgcgga tccaatgca<br>	72<br>	31<br>	DNA<br>	Artificial Sequence<br><br> Description of Artificial Sequence: Ligating oligonucleotide<br> 72<br>
ttggatccgc ggccgcaagc ttcctgcagg t<br> 73<br> 2013<br> DNA<br>
Â«:213&gt; Arabidopsis thaliana<br> 73<br>
atgcccctta ttcatoggaa aaagccgacg gagaaaccat	cgacgccgcc atctgaagag 60<br>
gtggtgcacg atgaggattc gcaaaagaaa ccacacgaat	cttccaaatc ccaccataag 120<br>
aaatcgaacg gaggagggaa gtggtcgtgc atcgactctt	gttgttggtt cattgggCgt 130<br>
gtgtgtgtaa cctggtggtt tcttctcttc ctttacaacg	caatgcctgc gagcttccct 240<br>
cagtatgtaa cggagcgaat cacgggtcct ttgcctgacc	cgcccggtgt taagctcaaa 300<br>
aaagaaggtc ttaaggcgaa acatcctgtt gtcttcattc	ctgggattgt caccggtggg 360<br>
ctcgagcttu gggaaggcaa acaatgcgct gatggtttat	ttagaaaacg tttgtggggt 420<br>
ggaacttttg gtgaagtcta caaaaggcct ctatgttggg	tggaacacat gtcacttgac 480<br>
aatgaaactg ggttggatcc agctggtatt agagttcgag	ctgtatcagg actcgtggct S40<br>
gctgactact ttgctcctgg ctacttgtc tgggcagtgc	tgattgctaa ccttgcacat SCO<br>
attggatatg aagagaaaaa tatgtacatg gctgcatatg	actggcggct ttcgtttcag 6â‚¬0<br><br>
aacacagagg tacCgatca gactcttagc cgcatgaaaa gtaatataga gttgatggct TZQ.<br>
tctaccaacg gtg"gaaaaaa agcagttaca getccgcatt ccatgggggt cttgtacctt 780<br>
ctacatccta tgaagtgggt cgaggcacca gctcccctgg gtggcggggg cgggccagat 640<br>
tggtgtgcaa agtatattaa ggcggtgatg aacattggtg gaccatttcc tggtgtccca 900<br>
aaagctgttg cagggctttt ctctgctgaa gcaaaggatg ttgcagttgc cagagcgatt 960<br>
gccccaggat Ccttagacac cgacatattt agacttcaga ccttgcagca tgcaatgaga 1020<br>
atgacacgca catgggactc aacaatgtct atgctaccga agggaggtga cacgatatgg 1O80<br>
ggcgggcttg attggtcacc ggagaaaggc cacacctgtt gtgggaaaaa gcaaaagaac 1140<br>
aacgaaactt gtggtgaagc aggtgaaaac ggagtttcca agaaaagtcc tgttaactac 1200<br>
ggaaggatga tatcttttgg gaaagaagta gcagaggctg cgccatctga gactaataat 12 60<br>
atugattttc gaggtgctgt caaaggtcag agtatcccaa atcacacctg tcgtgacgtg 1320<br>
tggacagagt accatgacat gggaattgct gggatcaaag ctatcgctga gcataaggtc 1330<br>
tacactgctg gtgaagctat agatctacta cattatgttg ctcctaagat gatggcgcgt 1440<br>
ggtgccgctc atttctctta tggaattgct gatgatttgg atgacaccaa gcatcaagat 1500<br>
cccaaatact ggtcaaatcc gttagagaca aaattaccga atgctcctga gatggaaatc 1560<br>
tactcactat acggagtggg gataccaacg gaacgagcat acgtatacaa gcttaaccag 1620<br>
tctcccgaca gttgcatccc ctttcagata ttcacttctg cccacgagga ggacgaagat 1680<br>
agctgtctga aagcaggagt ttacaatgtg gatggggatg aaacagtacc cgtcctaagt 174 0<br>
gccgggtaca tgtgtgcaaa agcgtggcgt ggcaagacaa gattcaaccc ttccggaatc laoo<br>
aagacttata taagagaata caatcactct ccgccggcta acctgctgga agggcgcggg 1860<br>
acgcagagtg gtgcccatgt tgatatcatg ggaaactttg ccttgatcga agatatcatg 1920<br>
agggttgccg ccggaggtaa cgggtctgac ataggacatg accaggtcca ctctggcata 1980<br>
tttgaatggt cggagcgtat tgacctgaag ctg	2013<br> 74<br> 671<br> PRT<br> ArabidopsiG thaliana<br> 74<br>
Met Pro Leu lie His Pg Lys Lys Pro THr GLu Lys Pro Ser Thr Pro<br>
15	10	IS<br>
Pro Ser Glu Glu Val Val His Asp Glu Asp Ser Gin Lys Lys Pro His<br>
20	25	30<br>
Glu Ser Ser Lys Ser His His Lys Lys Ser Asn Gly Gly Gly Lys Trp<br>
35	40	45<br>
Ser Cys lie Asp Ser Cys Cys Trp Pbe He Gly Cys Val Cys Val Thr<br>
50	55	60<br>
Trp Trp Phe Leu Leu Phe Leu Tyr Asn Ala Met Pro Ala Ser Phe Pro<br>
55	70	75	ao<br>
Gin Tyr Val Thr Glu Arg lie Thr Gly Pro Leu Pro Asp Pro Pro Gly<br>
85	90	95<br>
Val Lys Leu Lys Lys Glu Gly Leu Lys Ala Lys His Pro Val Val Phe<br>
100	lOS	110<br>
lie Pro Gly lie Val Thr Gly Gly Leu Glu Leu Trp Glu Gly Lys Gin<br>
115	120	125<br><br>
Cys Ala Asp Gly Leu ?he Arg Lys Arg Leu Trp Gly Gly Thr Phe Gly<br>
130	135	140<br>
Glu Val Tyr Lys Arg Pro Leu Cys Trp Val Glu His Met Ser Leu Asp<br>
145	150	155	160<br>
Asn Glu Thr Gly Leu Asp Pro Ala Gly lie Arg Val Arg Ala Val Ser<br>
165	170	175<br>
Gly Leu val Ala Ala Asp Tyr Phe Ala Pro Gly Tyr Phe Val Trp Ala<br>
180	1B5	190<br>
Val Leu lie Ala Asn Leu Ala His lie Gly Tyr Glu Glu Lys Asn Me<br>
195	200	205<br>
Tyr Met Ala Ala Tyr Asp Trp Arg Leu Ser Phe Gin Asn Thr Glu Val<br>
210	215	220<br>
Arg Asp Gin Thr Leu Ser Arg Met Lys Ser Asn lie Glu Leu Met Val<br>
225	230	235	240<br>
Ser Thr Asn Gly Gly Lys Lys Ala Val lie Val Pro His Ser Met Gly<br>
245	250	255<br>
Val Leu Tyr Phe Leu His Phe Met Lys Trp Val Glu Ala Pro Ala Pro<br>
260	265	270<br>
Leu Gly Gly Gly Gly Gly Pro Asp Trp Cys Ala Lys Tyr He Lys Ala<br>
275	280	235<br>
Val Met Asn lie Gly Gly Pro Phe Leu Gly Val Pro Lys Ala Val Ala<br>
290	295	3O0<br>
Gly Leu Phe Ser Ala Glu Ala Lys Asp Val Ala val Ala Arg Ala He<br>
305	310	315	320<br>
Ala Pro Gly Phe Leu Asp Thr Asp He Phe Arg Leu Gin Thr Leu Gin<br>
325	330	335<br>
His Val Met Arg Met Thr Arg Thr Trp Asp Ser Thr Met Ser Met Leu ""<br>
340	345	350<br>
Pro Lys Gly Gly Asp Thr He Trp Gly Gly Leu Asp Trp Ser Pro Glii<br>
355	360	365<br>
Lys Gly His Thr Cys Cys Gly Lys Lys Gin Lys Asn Asn Glu Thr Cys<br>
370	375	380<br>
Gly Glu Ala Gly Glu Asn Gly Val Ser Lys Lys Ser Pro Val Asn Tyr<br>
385	390	395	400<br>
Gly Arg Met He Ser Phe Gly Lys Glu Val Ala Glu Ala Ala Pro Ser<br>
405	410	415<br>
Glu He Asn Asn He Asp Phe Arg Gly Ala Val Lys Gly Gin Ser He<br>
420	425	430<br><br>
Pro Asn His Thr Cys Arg .Asp Val Trp Thr Glu Tyr His Asp MeC Gly<br>
435	440	445<br>
lie Ala Gly He Lys Ala He Ala Glu Tyr Lys Val Tyr Thr Ala Gly<br>
450	455	460<br>
Glu Ala rie Asp Leu Leu His Tyr Val Ala Pro Lys Met Met Ala Arg<br>
4S5	470	475	480<br>
Gly Ala Ala His Phe Ser Tyr Gly lie Ala Asp Asp Leu Asp Asp Thr<br>
485	490	495<br>
Lys Tyr Gin Asp Pro Lys Tyr Trp Ser Asn Pro Leu Glu Thr Lys Leu<br>
500	505	SIO<br>
Pro Asn Ala Pro Glu Met Glu lie Tyr Ser Leu Tyr Gly Val Gly He<br>
515	520	525<br>
Pro Thr Glu Arg Ala Tyr Val Tyr Lys Leu Asn Gin Ser Pro Asp Ser<br>
530	535	540<br>
Cys He Pro Phe Gin He Phe Thr Ser Ala His Glu Glu Asp Glu Asp<br>
545	550	555	560<br>
Ser Cys Leu Lys Ala Gly Val Tyr Asn Val Asp Gly Asp Glu Thr Val<br>
565	570	575<br>
Pro Val Leu Ser Ala Gly Tyr Met Cys Ala Lys Ala Trp Arg Gly Lys<br>
530	585	590<br>
Thr Arg Phe Asn Pro Ser Gly He Lys Thr Tyr He Arg Glu Tyr Asn<br>
595	600	605<br>
His Ser Pro Pro Ala Asn Leu Leu Glu Gly Arg Gly Thr Gin Ser Gly<br>
610	615	620<br>
Ala His Val Asp He MeC Gly Asn Phe Ala Leu He Glu Asp He Met<br>
625	630	635	640<br>
Arg Val Ala Ala Gly Gly Asn Gly Ser Asp He Gly His Asp Gin Val<br>
645	650	655<br>
His Ser Gly He Phe Glu Trp Ser Glu Arg He Asp Leu Lys Leu<br>
560	665	670<br>
=:210&gt; 75<br>
c211&gt; 1986<br>
!:212&gt; DNA<br>
213&gt; Saccharomyces cerevisiae<br>
:400&gt; 75<br>
itgggcacac tgtttcgaag aaatgtccag aaccaaaaga gtgattctga tgaaaacaat	60<br>
laagggggtt ctgttcataa caagcgagag agcagaaacc acactcatca tcaacaggga	120<br>
:taggc:cata agagaagaag gggtaccagt ggcagtgcaa aaagaaatga gcgtggcaaa	180<br>
jacttcgaca ggaaaagaga cgggaacggt agaaaacgtt ggagagatcc cagaagaccg	240<br><br>
attttcaCtc	ttggtgcatt  cttaggtgta  ctttCgccgt   tcagctttgg  cgcttaccaC   300<br>
gttcacaata	gcgatagcga  ctcgtctgac  aactttgcaa  attttgattc  acctaaagtg  3S0<br>
tatttggatg	actggaaaga  tgctctccca  caaggtataa gctcgtttat  cgatgacacc  420<br>
caggctggta	actactccac  atcttcttta gatgatctca gtgaaaattt  tgccgttggt  480<br>
aaacaacCcc	tacgtgatta  taatatcgag gccaaacatc  ctgttgtaac  ggtccctggc  540<br>
gtcatttcta	cgggaattga aagctgggga  gttattggag  acgatgagtg  cgatagctct  600<br>
gcgcaCtttc	gtaaacggct gtggggaagt  ttttacatgc  tgagaacaat  ggttatggat  660<br>
aaagtttgtt	ggttgaaaca  tgtaatgtta  gatcctgaaa  caggtctgga  cccaccgaac  720<br>
ttcacgctac	gcgcagcaca gggcttcgaa  tcaactgatt  actccatcgc  agggtactgg  780<br>
atttggaaca	aagttttcca aaatctggga  gtaattggct  atgaacccaa  taaaatgacg  340<br>
agcgctgcgt	acgattggag gcttgcatat  ttagatctag aaagacgcga  caggtacttt   900<br>
acgaagccaa	aggaacaaac  cgaactgttt   catcaattga  gtggtgaaaa  agtttgCtta  960<br>
attggacatt	ctaCgggttc  tcagattatc   ttccacttta  tgaaatgggt  cgaggctgaa  1020<br>
ggccctcctt	acggcaatgg tggccgtggc  tgggctaacg  aacacataga  ttcattcatt   1080<br>
aatgcagcag	ggacgcttct gggcgctcca  aaggcagttc  cagctcCaat  tagtggtgaa  1140<br>
acgaaagata	ccactcaatt  aaatacgtta  gccacgtacg  gtttggaaaa  gttcctctca  1200<br>
agaattgaga	gagtaaaaat gctacaaacg  tggggtggta  taccatcaat gccaccaaag  1260<br>
ggagaagagg	tcatttgggg ggataCgaag  tcatcttcag aggatgcatt gaataacaac   1320<br>
actgacacat	acggcaattt  cattcgattt  gaaaggaata  cgagcgatgc  tttcaacaaa  1380<br>
aatttgacaa	tgaaagacgc  cattaacatg  acattatcga  tatcacctga  atggctccaa  1440<br>
agaagagcac	atgagcagta  ctcgttcggc  tattccaaga  atgaagaaga gttaagaaaa  1500<br>
aatgagctac	accacaagca  ctggtcgaat  ccaacggaag taccacttcc  agaagctccc  1560<br>
cacatgaaaa	tctaCCgtat  atacggggtg  aacaacccaa  ctgaaagggc  aCaCgtatat  162 0<br>
aaggaagagg	atgacccctc  tgctctgaat  ttgaccatcg  actacgaaag  caagcaacct   1680<br>
gCattcctca	ccgaggggga  cggaaccgct  ccgctcgtgg  cgcactcaat gtgtcacaaa  174 0<br>
tgggcccagg	gtgcttcacc gtacaaccct  gccggaatta  acgttactat  tgtggasatg  ISOO<br>
aaacaccagc	cagatcgatt  tgaCatacgt  ggtggagcaa  aaagcgccga  acacgtagac   1860<br>
aCcctcggca	gcgcggagtt gaacgattac  atcttgaaaa  ttgcaagcgg  taatggcgat   1920<br>
ctcgtcgagc	cacgccaatt gtctaatttg  agccagtggg  tttctcagat  gcccttccca  1980<br>
aCgtaa	1986<br>   76<br>   661<br>   PRT<br>  Saccharomyces  cerevisiae<br>   76<br>
Met Gly Thr Leu Phe Arg Arg Asn Val Gin Asn Gin Lys Ser Asp Ser<br>
IS	10	15<br>
Asp Glu Asn Asn Lys Gly Gly Ser Val His Asn Lys Arg Glu Ser Arg<br>
20	25	30<br>
Asn His He His His Gin Gin Gly Leu Gly His Lys Arg Arg Arg Gly<br>
35	40	45<br>
He Ser Gly Ser Ala Lys Arg Asn Glu Arg Gly Lys Asp Phe Asp Arg<br>
SO	55	60<br>
Lys Arg Asp Gly Asn Gly Arg Lys Arg Trp Arg Asp Ser Arg Arg Leu<br>
65	70	75	80<br>
He Phe He Leu Gly Ala Phe Leu Gly Val Leu Leu Pro Phe Ser Phe<br>
85	90	95<br><br>
Gly Ala Tyr His Val His Asn Ser Asp Ser Asp Leu Phe Asp Asn Phe<br>
100	105	110<br>
Val Asn Phe Asp  Ser Leu Lys Val Tyr Leu Asp Asp Trp Lys Asp Val<br>
115	120	125<br>
Leu Pro Gin Gly He Ser Ser Phe He Asp Asp He Gin Ala Gly Asn<br>
130	135	140<br>
Tyr Ser Thr Ser Ser Leu Asp Asp Leu Ser Glu Asn Phe Ala Val Gly<br>
145	150	155	160<br>
Lys Gin Leu Leu Arg Asp Tyr Asn He "Glu Ala Lys His Pro Val Val<br>
165	170	175<br>
Met Val Pro Gly Val He Ser Thr Gly He Glu Ser Trp Gly Val He<br>
180	IBS	190<br>
Gly Asp Asp Glu Cys Asp Ser Ser Ala His Phe Arg Lys Arg Leu Trp<br>
195	200	205<br>
Gly Ser Phe Tyr Met Leu Arg Thr Met Val Met Asp Lys Val Cys Trp<br>
210	215	220<br>
Leu Lys His Val Met Leu Asp Pro Glu Thr Gly Leu Asp Pro Pro Asn<br>
225	230	235	240<br>
Phe Thr Leu Arg Ala Ala Gin Gly Phe Glu Ser Thr Asp Tyr Phe He<br>
245	250	255<br>
Ala Gly Tyr Trp He Trp Asn Lys Val Phe Gin Asn Leu Gly Val He<br>
260	265	270<br>
Gly Tyr Glu Pro Asn Lys Met Thr Ser Ala Ala Tyr Asp Trp Arg Leu<br>
275	280	285<br>
Ala Tyr Leu Asp Leu Glu Arg Arg Asp Arg Tyr Phe Thr Lys Leu Lys<br>
290	295	300<br>
Glu Gin He Glu Leu Phe His Gin Leu Ser Gly Glu Lys Val Cys Leu<br>
305	310	315	320<br>
He Gly His Ser Met Gly Ser Gin He He Phe Tyr Phe Met Lys Trp<br>
325	330	335<br>
Val Glu Ala Glu Gly Pro Leu Tyr Gly Asn Gly Gly Arg Gly Trp Val<br>
340	345	350<br>
Asn Glu His He Asp Ser Phe He Asn Ala Ala Gly Thr Leu Leu Gly<br>
355	3S0	365<br>
Ala Pro Lys Ala Val Pro Ala Leu He Ser Gly Glu Met Lys Asp Thr<br>
370	375	380<br>
He Gin Leu Asn Thr Leu Ala Met Tyr Gly Leu Glu Lys Phe Phe Ser<br>
385	390	395	400<br><br>
Arg lie Glu Arg Val Lys Mec Leu Gin Thr Trp Gly Gly lie Pro Ser<br>
40S	410	415<br>
Mec Leu Pro Lys Gly Glu Glu Val lie Trp Gly Asp Met Lys Ser Ser<br>
420	425	430<br>
Ser Glu Asp Ala Leu Asn Asn Asn Thr Asp Thr Tyr Gly Asn Phe He<br>
435	440	445<br>
Arg Phe Glu Arg Asn Thr Ser Asp Ala Phe Asn Lys Asn Leu Thr Met<br>
450	455	460<br>
Lys Asp Ala He Asn MeC Thr Leu Ser He Ser Pro Glu Trp Leu Gin<br>
465	470	475	480<br>
Arg Arg Val His Glu Gin Tyr Ser Phe Gly Tyr Ser Lys Asn Glu Glu<br>
485	490	495<br>
Glu Leu Arg Lys Asn Glu Leu His His Lys His Trp Ser Asn Pro Met<br>
500	505	510<br>
Glu Val Pro Leu Pro Glu Ala Pro His Met Lys He Tyr Cys He Tyr<br>
515	520	525<br>
Gly Val Asn Asn Pro Thr Glu Arg Ala Tyr Val Tyr Lys Glu Glu Asp<br>
530	535	540<br>
Asp Ser Ser Ala Leu Asn Leu Thr He Asp Tyr Glu Ser Lys Gin Pro<br>
545	550	S5S	560<br>
Val Phe Leu Thr Glu Gly Asp Gly Thr Val Pro Leu Val Ala His Ser<br>
565	570	575<br>
Met Cys His Lys Trp Ala Gin Gly Ala Ser Pro Tyr Asn Pro Ala Gly<br>
580	585	590<br>
He Asn Val Thr He Val Glu Met Lys His Gin Pro Asp Arg Phe Asp<br>
595	600	605<br>
lie Arg Gly Gly Ala Lys Ser Ala Glu His Val Asp He Leu Gly Ser<br>
610	615	620<br>
Ala Glu Leu Asn Asp Tyr He Leu Lys He Ala Ser Gly Asn Gly Asp<br>
625	630	635	640<br>
Leu Val Glu Pro Arg Gin Leu Ser Asn Leu Ser Gin Trp Val Ser Gin<br>
645	S50	655<br>
Met Pro Phe Pro Met 660<br><br> 77  35 <:212> DNA<br> Arrificial Sequence<br><br>  Description of  Artificial  Sequence:     Synthetic oligonucleotide primer<br>  77<br>
ggacccgcgg ccgcacaatg ccccttattc atcgg<br> 73<br>
e211&gt; 35<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 78<br>
ggatcccctg caggtcacag cttcaggtca atacg<br> 79<br> 37<br> DNA<br> Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 79<br>
ggatccgcgg ccgcacaatg ggcacactcc ttcgaag<br> ao<br>
c211&gt; 39<br> DNA<br>
e213&gt; Artificial Sequence<br><br> Description of Artificial Sequence: Synthetic oligonucleotide primer<br> 80<br>
ggatcccctg caggttacac tgggcacact gtttcgaag<br><br><br>
WE CLAIM:<br>
1.	A recombinant nucleic acid construct, for altering the sterol content in host cells and plants, having a coding sequence and a heterologous regulatory sequence Operably linked together, wherein said coding sequence comprises a polynucleotide having sequence selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 73, 75 and degenerate variants thereof and encodes a plant sterol: cholesterol acyltransferase-like polypeptide or fragment thereof, said sterol: cholesterol acyltransferase-like polypeptide is a lecithin: cholesterol acyltransferase-like polypeptide or acyl CoA: cholesterol acyltransferase-like polypeptide and said regulatory sequence is a sequence functional in plants.<br>
2.	The recombinant nucleic acid construct as claimed in claim 1, wherein said plant lecithin: cholesterol acyltransferase-like polypeptide is irom a plant selected from the group consisting of Arabidopsis, soybean and com.<br>
3.	The recombinant nucleic acid construct as claimed in claim 1, wherein the polynucleotide is selected from the group consisting of:<br><br>
a)	a polynucleotide encoding a polypeptide of SEQ ID NO 3, 5, 7, 9, 74 or 76 or SEQ ID NO 3, 5, 7, 9, 74 or 76 with at least one conservative amino acid substitution;<br>
b)	SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br><br>
c)	a polynucleotide that has at least 70% sequence identity to SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br>
d)	a polynucleotide that has at least 80% sequence identity to SEQ ID NO:2,4, 6, 8, 73 or 75;<br>
e)	a polynucleotide that has at least 90% sequence identity to SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br>
f)	a polynucleotide that has at least 95%) sequence identity to SEQ ID NO:2,4, 6, 8, 73 or 75;<br><br>
g)   a   polynucleotide   of   at   least    10   nucleic   acids   that   hybridizes   under<br>
stringent conditions to SEQ ID NO: 2, 4, 6, S, 73 or 75;<br>
h)   a   polynucleotide   complementary   to   a   polynucleotide   of   (a),   (b),   (c),<br>
(d),(e),(f)or(g);and<br>
i)    a    polynucleotide    that    hybridizes    under    stringent    conditions    to    SEQ<br>
ID NO: 2, 4, 6, 8, 73 or 75 and encodes a plant lecithin; cholesterol acyltransferase-<br>
like polypeptide.<br>
4. The recombinant nucleic acid construct as claimed in claim 1, wherein the polynucleotide is of the formula 5"X-(Ri)n-CR2)-{R3)n-Y3", where X is hydrogen, Y is hydrogen or a metal, R1 and R3 are any nucleic acid, n is an integer between 0-3000, and R2 is selected from the group consisting of:<br>
a)	a polynucleotide encoding a polypeptide of SEQ ID NO: 3, 5, 7, 9, 74 or 76 or SEQ ID NO: 3, 5, 7, 9, 74 or 76 with at least one conservative amino acid substitution;<br>
b)	SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br><br>
c)	a polynucleotide that has atleast 70% sequence identity to SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br>
d)	a polynucleotide that has at least 80% sequence identity to SEQ ID N0:2, 4, 6, 8, 73 or 75;<br>
e)	a poiynudeotide that has at least 90% sequence Identity to SEQ ID NO;2,4,6, 8, 73 or 75;<br>
f)	a polynucleotide that has at least 95% sequence identity to SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br>
g)	a polynucleotide of at least 10 nucleic acids that hybridizes under stringent conditions to SEQ ID NO: 2, 4, 6, 8, 73 or 75;<br>
h)   a   polynucleotide   complementary   to   a   polynucleotide   of   (a),    (b),   (c),<br>
(d),(e),(f)or{g);and<br>
i)    a    polynucleotide    that    hybridizes    under    stringent    conditions    to    SEQ<br>
ID NO; 2, 4, 6, 8, 73 or 75 and encodes a plant lecithin; cholesterol acyltransferase-<br>
like polypeptide.<br><br>
5.	The recombinant nucleic acid construct as claimed in claim 1, wherein the<br>
polynucleotide is selected from the group consisting of:<br>
a)	SEQ ID NO: 42 or a degenerate variant thereof;<br>
b)	a polynucleotide having at least 70% sequence identity with SEQ ID NO: 42;<br>
c)	a polynucleotide having at least 80% sequence identity with SEQ ID NO: 42;<br>
d)	a polynucleotide having at least 90% sequence identity with SEQ ID NO: 42;<br>
e)	a polynucleotide having at least 95% sequence identity with SEQ ID NO: 42;<br>
f)	a polynucleotide of at least 10 nucleic acids that hybridizes under stringent conditions to SEQ ID NO: 42;<br>
g)	a polynucleotide complementary to a polynucleotide of (a), (b), (c), (d), (e), or(f); and<br>
h) a polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 42 and encodes an acyl CoA: cholesterol acyltransferase-like polypeptide.<br>
6.	The recombinant nucleic acid construct as claimed in claim I, wherein the<br>
polynucleotide is of the formula 5"X-(R,V(R2)-(R3)n-Y3", where X is hydrogen, Y is<br>
hydrogen or a metal R1 and R3 are any nucleic acid, n is an integer between 0 and<br>
3000, and R2 is selected from the group consisting of:<br>
a)	SEQ ID NO: 42 or degenerate variants thereof;<br>
b)	a polynucleotide having at least 70% sequence identity to SEQ ID NO: 42;<br>
c)	a polynucleotide having at least 80% sequence identity to SEQ ID NO: 42;<br>
d)	a polynucleotide having at least 90% sequence identity to SEQ ID NO: 42;<br><br>
e)	a polynucleotide having at least 95% sequence identity to SEQ ID NO: 42;<br>
f)	a polynucleotide of at least 10 nucleic acids that hybridizes under stringent conditions to SEQ ID NO: 42;<br>
g)	a polynuc]eotide complementary to a polynucleotide of (a), (b), (c), (d), (e), or(f); and<br>
h) a polynucleotide that hybridizes under stringent conditions to SEQ ID NO; 42 and encodes an acyl CoA: cholesterol acyltransferase-like polypeptide.<br>
7.	The recombinant nucleic acid construct as claimed in claim 1, wherein said lecithin: cholesterol acyltransferase-like polypeptide is a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
8.	The recombinant nucleic acid construct as claimed in claim 1, wherein said acyl CoA: cholesterol acyltransferase-like polypeptide is a plant acyl CoA: cholesterol acyltransferase-like polypeptide.<br><br>
9.	The recombinant nucleic acid construct as claimed in claim 1, comprising a termination sequence as herein described.<br>
10.	The recombinant nucleic acid construct as claimed in claim I, wherein said regulatory sequence comprises a constitutive promoter.<br>
11.	The recombinant nucleic acid construct as claimed in claim 1, wherein said regulatory sequence comprises an inducible promoter.<br>
12.	The recombinant nucleic acid construct as claimed in claim 1, wherein said regulatory sequence is selected from the group consisting of a tissue specific promoter, a developmentally regulated promoter, an organelle specific promoter, and a seed specific promoter.<br><br>
13.	An isolated polypeptide obtained by culturing a host cell selected from yeast, bacteria, bacteriophage or viruses containing the recombinant nucleic acid as claimed in claim 1, comprising an amino acid sequence selected from the group consisting of SEQ ID NO: 3, SEQ ID NO: 3 with at least one conservative amino acid substitution, SEQ ID NO: 5, SEQ ID NO: 5 with at least one conservative amino acid substitution, SEQ ID NO: 7, SEQ ID NO: 7 with at least one conservative amino acid substitution, SEQ ID NO: 9, SEQ ID NO; 9 with at least one conservative amino acid substitution, SEQ ID NO: 74, SEQ ID NO: 74 with at least one conservative amino acid substitution, SEQ ID NO: 76 and SEQ ID NO: 76 with at least one conservative amino acid substitution.<br>
14.	An isolated polypeptide as claimed in claim 13, which is immunogenic and contains at least 10 consecutive amino acids.<br>
15.	A method of producing a polypeptide as claimed in claim 13, comprising the steps of culturing the host cell under conditions permitting expression of said sterol: cholesterol acyl transferase- like polypeptide and Isolating the polypeptide in a known manner.</:212></:211></:220></:212></:220></:2></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgY2xhaW1zLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che claims-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZGVzY3JpcHRpb24oY29tcGxldGUpLWR1cGxpY2F0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che description(complete)-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZHJhd2luZ3MtZHVwbGljYXRlLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che drawings-duplicate.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS00LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDItMDQzMy1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2002-0433-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216457-cold-storage-appliance.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216459-mixtures-of-fluorescent-whitening-agents.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216458</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2002/433/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Mar-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 North N Lindbergh Boulevard St, Louis, MO 63167,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LASSNER, Michael</td>
											<td>515 Galveston Drive, Redwood City, CALIFORNIA 94063,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN EENENNAAM, Alison</td>
											<td>856 Burr Street, Davis, CA 95616,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/23863</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-08-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/152,493</td>
									<td>1999-08-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216458-an-isolated-nucleic-acid-sequence-comprising-a-polynucleotide-encoding-a-plant-sterol-cholesterol-acyltransferase-like-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:40:22 GMT -->
</html>
