<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216568-a-chimeric-gene-comprising-a-nucleic-acid-fragment-conferring-disease-resistance-to-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:47:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216568:A CHIMERIC GENE COMPRISING A NUCLEIC ACID FRAGMENT CONFERRING DISEASE RESISTANCE TO PLANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A CHIMERIC GENE COMPRISING A NUCLEIC ACID FRAGMENT CONFERRING DISEASE RESISTANCE TO PLANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The preparation and use of an isolated nucleic acid fragment which confers a $i(Pi-ta) resistance gene-mediated defense response in plants against disease caused by fungal pathogens is described. Genes incorporating such nucleic acid fragments either alone or in combination with an $i(AVR-Pita) isolated nucleic acid fragment or functionally equivalent subfragments thereof and suitable regulatory sequences can be used to create transgenic plants which can produce a $i(Pi-ta) resistance gene-mediated defense response against a variety of fungal pathogens, in particular, the rice blast fungus.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970 [39 OF 1970]<br>
THE PATENTS RULES, 2003 COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
"A CHIMERIC GENE COMPRISING A NUCLEIC ACID FRAGMENT CONFERRING<br>
DISEASE RESISTANCE TO PLANTS"<br>
E. I. DU PONT DE NEMOURS AND COMPANY, located at 1007 Market Street, Wilmington, Delaware 19898, United States of America,<br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br><br>
11-9-2007<br><br>
The present invention relates to a chimeric gene comprising a nucleic acid fragment conferring disease resistance to plants.<br>
A     The invention relates to the preparation and use of an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response in plants against disease caused by fungal pathogens. Genes incorporating such nucleic acid fragments either alone or in combination with znAVR-Pita isolated nucleic acid fragment or functionally equivalent subfragments thereof and suitable regulatory sequences can be used to create transgenic plants which can produce a Pi-ta resistance gene-mediated defense response against a variety of fungal pathogens, in particular, the rice blast fungus.<br>
BACKGROUND OF THE INVENTION Plants can be damaged by a wide variety of pathogenic organisms including viruses, bacteria, fungi and nematodes. The invasion of a plant by a potential pathogen can result in a range of outcomes: the pathogen can successfully proliferate in the host, causing associated disease symptoms, or its growth can be halted by the host defenses. In some plant-pathogen interactions, the visible hallmark of an active defense response is the so-called hypersensitive response (HR). The HR involves rapid necrosis of cells near the site of the infection and may include the formation of a visible brown fleck or lesion. Eathogens which elicit an HR on a given host are said to be avirulent (AVR) on that host, thg.host is said to be resistant, and the plant-pathogen interaction "*s said to be incompatible. Strains which proliferate and cause disease on a particular host are said to be virulent, in this case the host is said to be susceptible, and the plant-pathogen interaction is said to be compatible.<br>
Genetic analysis has been used to help elucidate the genetic basis of plant-pathogen recognition for those cases in which a. series of strains (races) of a particular fungal or bacterial pathogen are either virulent or avirulent on a series of cultivars of a particular host species. In many such cases, genetic analysis of both the host and the pathogen revealed that many avirulent fungal and bacterial strains differ from virulent ones by the possession of one or more avirulence ("avr" or "AVR") genes that have corresponding "resistance" (R) genes in the host.<br>
This avirulence gene-resistance gene model is termed the "gene-for-gene" model (Crute et al. (1985) pp 197-309 in: Mechanisms of Resistance of Plant Disease. R. S. S. Fraser, ed.; Ellingboe, (1981) Annu. Rev. Phytopathol. 19:125-143; Flor, (1971) Annu. Rev. Phythopathol. 9:275-296). According to a simple formulation of this model, plant resistance genes encode specific receptors for molecular signals generated by avr genes. Signal transduction pathway(s) then carry the signal to a set of target genes that initiate the host defenses. Despite this simple predictive model, the molecular basis of the avr-resistance gene interaction is still unknown.<br>
2<br><br>
5<br>
The first R-gene cloned was the Hml gene from corn {Zea mays), which confers resistance to specific races of the fungal pathogen Cochliobolus carbonum (Johal et al.. 1992, Science 258:985-987). Hml encodes a reductase that detoxifies a toxin produced by the pathogen. Next to be cloned was the Pto gene from tomato (Lycopersicon pimpimllifollium) (Martin et al, 1993, Science 262:1432-1436; U.S. Patent No. 5,648,599). Pto encodes a serine-threonine protein kinase that confers resistance in tomato to strains of the bacterial pathogen Pseudomonas syringae pv. tomato that express the avrPto avirulence gene. Taking center stage now are R-genes that encode proteins containing leucine-rich-repeats (LRRs) (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv. Plant Pathol. 24:89-167). Two classes of membrane anchored proteins with extracellular LRRs have been identified. One subclass includes R-gene products that lack a cytoplasmic serine/threonine kinase domain such as the tomato Cf-9 gene for resistance to the fungus Qadosporium fulvum (Jones et al., 1994, Science 266:789-793, WO95/38230), and the other subclass includes an R-gene product with a cytoplasmic serine/threonine kinase domain, the rice Xa-21 gene for resistance to the bacterial pathogen Xanthomonas oryzae (Song et al.. 1995, Science 270:1804-1806; U.S. Patent No. 5,859,339). The largest class of R-genes includes those encoding proteins with cytoplasmic LRRs such as the Arabidopsis R-genes RPS2 (Bent et al., 1994, Science 265:1856-1860; Mindrinos et al., 1994, Cell 78:1089-1099) and RPM1 (Grant et al., 1995, Science 269:843-846). These R-proteins also possess a putative nucleotide binding site (MBS), and either a leucine zipper (LZ) motif or a sequence homologous to the Toll/lnterleukin-1 receptor (TIR). Table 1 has been reproduced in part from Baker et al. (1997, Science 276:726-733) as a concise summary of classes of R-genes cloned to date and examples of cloned genes within each class.<br><br><br><br>
"This Table has been reproduced in part from Baker et al. (1997, Science 276:726-733). References for each of the listed R-genes can be found in this review article.<br>
Nucleotide binding sites (NBS) are found in many families of proteins that are critical for fundamental eukaryotic cellular functions such as cell growth, differentiation, cytoskeletal organization, vesicle transport, and defense. Key examples include the RAS group, adenosine triphosphatases, elongation factors, and heterotrimeric GTP binding proteins called G-proteins (Saraste at al., 1990, Trends in Biochem. Science 15:430). These proteins have in common the ability to bind ATP or GTP (Traut, 1994, Eur. J. Biochem. 229:9-19).<br>
It has long been hypothesized that the rice blast system represented a classical gene-for-gene system as defined by H. H. Flor (Flor, 1971, Annu. Rev. Phytopathol. 19:125-143). Genetic analyses needed to identify A VR-genes in the rice blast pathogen, Magnaporthe %risea, has been hampered by the low fertility that typifies M. grisea field isolates thai infect rice. Genetic crosses between poorly fertile M. grisea rice pathogens and highly fertile M. grisea pathogens of other grasses (such as weeping lovegxass, Eragrostis curvula, and finger millet, Eleusine coracana) have provided laboratory strains of the fungus with the level of sexual fertility required for identifying AVR-genes (Valent et al., 1991, Genetics 127:87-101). Rare fertile rice pathogens have since allowed demonstration of a one-to-one genetic or functional correspondence between blast fungus AVR-genes and particular rice R-genes (Silue et al., 1992, Phytopathology 82:577-580).<br>
Interest in the rice blast pathosystem is keen because rice blast disease, caused world¬wide by the fungal pathogen Magnaporthe grisea (Hebert) Barr (anamorph Pyricularia grisea Sacc), continues as the most explosive and potentially damaging disease of the rice crop despite decades of research towards its control. Manipulation of blast resistance genes remains one of the primary targets in all rice breeding programs, as fungal populations evolve to defeat deployed resistance strategies (See The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford).<br>
Commercial fungicide usage to supplement genetic control strategies began around 1915 when rice farmers used inorganic copper-based fungicides (Chapter 29 in The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford). The<br>
4<br><br>
Fungicides used to control blast disease have changed through time, with some compounds, such as the organomercurials used in the 1950s, causing major environmental damage. The :ontrol of rice blast with fungicides currently represents a cost of more than $500 million per year to farmers. This expense for blast control is the largest segment of the world rice fungicide market, which totaled $752 million in 1998 (Wood Mackenzie). Expectations are that the disease problems will intensify as the world rice requirements increase by an estimated 1.7% annually between 1990 and 2025 (See The Rice Blast Disease, 1994: ed. Zeigler, Leong and Teng. CAB International, Wallingford). This estimated need for an additional 13 million tons of rough rice per year to feed the growing population must come from intensification of production on decreasing available land. Rice blast disease is favored by agronomic production practices aimed at high yields, and thus the disease will continue, and most likely increase, as a constraint to rice crop yields unless durable genetic resistance against rice blast disease can be engineered into rice. This invention is among one of the first rice blast resistance genes to be cloned, represents a critical first step to the long term goal of engineering durable genetic resistance to rice blast disease.<br>
The fungus M. grisea has a large host range including species of different tribes within the grass family, Triticeae (e.g., wheat), Oryzeae (e.g., rice), Clorideae (e.g., finger millet), Paniceae (e.g., pearl millet), Andropogoneae (e.g., Sorghum) and Maydeae (e.g., maize). Molecular analyses have now defined 8 host species-specific subpopulations of M. grisea, each with a restricted set of host species specificities (Reviewed by Valent, 1997, The Mycota V, Plant Relationships, Carroll/Tudzynski, eds., Springer-Verlag Berlin Heidelberg pp 37-54). Table 2 gives a current view of pathogen subpopulations according to mitochondrial DNA (mtDNA) type. This view is strongly supported by separate analyses of ribosomal DNA (rDNA) polymorphisms (including both Restriction Fragment Length Polymorphism (RFLP) and Internal Transcribed Spacer (ITS) sequences) and of polymorphisms in both repetitive DNAs and single copy sequences.<br>
The pathogens of rice, wheat, finger millet, barley and corn (mtDNA types la-e) appear closely related, while pathogens of Digitaria spp. and Pennisetum spp. (mtDNA types II-IV) are highly divergent from the previous groups and from each other. However, M. grisea strains throughout this broad host range can cause significant crop damage. This pathogen has been shown to be the main cause of yield loss of finger millet (Eleusine coracana) in Africa, while infections in wheat (Triticum aestivum; Urashima et al., 1993, Plant Disease 77:1211-1216) and pearl millet (Pennisetum glaucum; Hanna et al., 1989, J. Heredity 80:145-147), although less widespread, can be severe under humid weather conditions. The disease has been documented on barley and com (See refs. In Urashima etal., 1993, Plant Disease 77:1211-1216).<br><br><br>
* Designated by mitochondrial-DNA haplotype.<br>
Pathogenicity to the "Defining Host Species" appears conserved within the subpopulation. Some hosts, such as barley, are infected by members of two or more subpopulations.<br>
Knowledge of pathogenicity and host specificity for plant pathogenic fungi is not as advanced as for bacterial and viral pathogens, and likewise, less is known about the molecular basis of resistance in cereal crop plants than in dicot crops or in dicot model systems such as Arabidopsis (Baker et al., 1997, Science 276:726-733). Sasaki reported the first results on the inheritance of resistance to rice blast disease from studies begun in Japan in 1917 (Sasaki, 1922, Japanese Genetics, Japan 1, 81-85).<br>
Since this me, over 30 R-genes have been defined through extensive genetic analysis worldwide, and many of these blast resistance genes have been mapped to rice chromosomes (See Refs. In Takahashi, 1965, The Rice Blast Disease, Johns Hopkins Press, Baltimore, 303-329; Causse et al., 1994, Genetics 138:1251-1274). These R-genes include 20 major resistance genes and 10 putative quantitative trait loci (QTLs). Kiyosawa has described 13 major resistance genes with 9 of these genes found as multiple alleles at 3 loci; 5 at the Pi-k locus on chromosome 11,2 at the Pi-z locus on chromosome 6 and 2 at the Pi-la locus on chromosome 12 (Kiyosawa, 1984, Rice Genetics Newsletter 1:95-97). Recent studies in Japan (Ise, 1992, International Rice Research Newsletter 17:8-9) and at the International Rice Research Institiute (IRRI) (Mackill et al., 1992, Phytopathology 82:746-749) have produced near isogenic rice lines (NILs) for use as "differential" rice varieties for determining which resistance genes are effective in controlling individual strains of the fungus. The IRRI NILs, which provide indica differentials for the blast fungus populations in tropical regions, have been analyzed for genetic relationships between their resistance genes and those present in Kiyosawa Differentials (Inukai et al., 1994, Phytopathology 84:1278-1283).<br>
Molecular markers (or "tags") tightly linked to R-genes have utility for efficient introgression and manipulation of those R-genes in breeding programs. By comparing<br><br>
genotypic patterns of near-isogenic lines, their donors, and their recurrent parents, Yu et a!. (1987, Phytopathology 77:323-326) were able to identify five restriction fragment length polymorphic (RFLP) markers linked to three blast resistance genes and to map them to rice chromosomes using segregating populations. RFLP markers linked to the R-genes have been reported (Yu et al., 1991, Theor Appl Genet 81:471-476). Molecular cloning of agronomjcally important R-genes represents a further advance to the ability of researchers to combine R-genes with other input and output traits in key crop varieties.<br>
In the course of the above mentioned investigations on the inheritance of resistance, Sasaki discovered physiological races of the rice blast pathogen by observing that different field isolates of the blast fungus vary in their ability to cause disease on different varieties of rice (Sasaki, 1922, Journal of Plant Protection 9:631-644; Sasaki, 1923, Journal of Plant Protection 10:1-10). Instability, or "breaking down" under field conditions, of major R-gene resistance to the rice blast fungus has resulted in identification of numerous races, or pathotypes, defined according to virulence spectra on differential rice varieties (Chapters 13 and 16 in The Rice Blast Disease, 1994, ed. Zeigler, Leong and Teng, CAB International, Wallingford). Pathogen populations are dynamic in response to deployment of a new resistance gene, sometimes resulting in new races that overcome the resistance gene within one or two years after deployment in the field.<br>
Researchers have not yet characterized in depth an avirulence gene/R-gene pair in the rice blast pafhosystem.<br>
U.S. Patent No. 5,648,599, issued to Tanksley and Martin on July 15,1997, describes an isolated gene fragment from tomato which encodes the Pto serine/threonine kinase, conferring disease resistance to plants by responding to an avirulence gene in a bacterial plant pathogen.<br>
WO 95/28423, which published on October 26,1995, describes resistance due to the Pseudomonas syringae RPS2 gene family, primers, probes and detection methods. This published international application includes broad claims to genes encoding proteins with particular NH2-terminal motifs, NBS motifs and leucine rich repeats for protecting plants against pathogens. There axe some unique features of the Pi-ta protein. The Pi-ta gene product has a unique amino terminus, lacking either the potential leucine zipper motif of the RPS2 gene-product subfamily (Bent et al., 1994, Science 265:1856-1860; Mindrinos et a!., 1994, Cell 78:1089-1099) or the Toll/Interleukin-1 receptor homology encoded by the N gene subfamily (Whitman et al., 1994, Cell 78:1101-1115). Most importantly, the carboxy terminal portion of the Pi-ta gene product is leucine rich, but it does not fit the consensus sequences for leucine-rich repeats reported for R-gene products (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv. Plant Pathol. 24:89-167).<br>
U.S. Patent No. 5,571,706, issued to Baker et al. on November 5,1996, covers plant virus resistance conferred by the N gene.<br>
7<br><br>
U.S. Patent No. 5,859,351, issued to Staskawicz et al. on January 12,1999, describes<br>
the PRF protein and nucleic acid sequence, which is involved in disease resistance in tomato.<br>
U.S. Patent No. 5,859,339, issued to Ronald et al. on January 12, 1999, describes the<br>
first resistance gene cloned from rice, Xa-21, which encodes an integral membrane, protein<br>
with both LRR and serine/threonine kinase domains, and confers resistance in rice to<br>
bacterial blight.<br>
WO 91/15585 which published on October 17,1991 and U.S. Patent No. 5,866,776<br>
issued to de Wr"t et al. on February .1,1999 describe a method for the protection of plants<br>
against pathogens using a combination of a pathogen avirulence gene and a corresponding<br>
plant resistance gene.<br>
.  U.S. Patent No. 5,674,993 ("993 patent), issued to Kawasaki et al. on October 7,<br>
1997, describes nucleic acid markers that co-segregate with the rice blast resistance genes<br>
Pi-b, Pi-ta and Pi-la2 and the suggestion that rice blast resistance genes could be isolated and<br>
cloned by using these nucleic acid markers. However, no nucleotide sequences are provided<br>
for any rice blast resistance genes iri the "993 patent. It should be noted that a putative<br>
sequence for the Pi-b rice blast resistance gene is now available in Genbank (accession<br>
number AB013448).<br>
in addition, Kawasaki et al. have also published two papers. The first paper,<br>
Rybka et al., MPMI, 10(4):517-524 (1997), is entitled "High Resolution Mapping of the fcidica-Derived Rice Blast Resistance Genes. II. Pi-ta2 and Pi-ta and a Consideration of Their Origin." The sequence for the RAPD primer that is set forth at the top of column 2 on page 519 is not the same as the RAPD primer set forth in SEQ ID NO:2 in the "993 patent. The sequence for the primer in the paper is TCCCCAGCCA CSEOIDNO:75). The sequence for the primer in the "993 patent is "        TCGCCAGCCA (SEP ID NO:76). It is not clear which sequence is correct. Notwithstanding this, it is clear that this paper does not set forth any nucleotide sequences for any rice blast genes. The second paper is Nakamura et al., Mol. Gen. Genet. 254:611-62 (1997). Thispaper describes the construction of an 800-kb contig in the near-centromeric region of the rice blast resistance gene Pi-ta2 using a rice BAC library. Again, no nucleotide sequence for any rice blast genes is disclosed. Thus, it is believed that no one hereto has cloned and sequenced a Pi-ta resistance gene from rice with demonstrated utility in controlling Magnaporthe grisea that would have significant value as crop protection tools for other graminaceous crops as well. Accordingly, the invention described herein represents the first molecular characterization of a Pi-ta avirulence gene/R-gene pair in the rice blast pathosystem.<br>
SUMMARY OF THE INVENTION This invention relates to an isolated nucleic acid fragment which confers a Pi-ta ■esistance gene-mediated defense response against diseases caused by fungal pathogens<br><br>
wherein said nucleic acid fragment correpoisnhstanlialy to the nucleotide sequence sei forth in SEQ ID NOS: 1 or 68 or a functionally "equivalent subfragment thereof.<br>
In another aspect this invention concerns genes comprising  isolatedjiucleic acid which confers a Pi-ta resistance gene-mediated defense response against diseases caused by fungal pathogens wherein said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent* subfragment thereof wherein these fragments are operably linked to suitable regulatory sequences.<br>
In still another aspect, this invention concerns a method of conferring Pi-ta resistance gene mediated defense response in plants against disease caused by a fungal pathogen which comprises introducing into the plant a gene comprising isolated nucleic acid which confers a Pi-ta resistance gene-mediated defense response against diseases caused by fungal pathogens wherein said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof wherein these fragments are operably linked to suitable regulatory sequences.<br>
In an even further aspect, this invention concerns a method of conferring a resistance gene-mediated defense response in plants at the site of infection by a fungal pathogen, comprising transforming the plant with a recombinant expression construct which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response against diseases caused by fungal pathogens wherein said nucleic fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof and (2) an AVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NO:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to regulatory sequences and further wherein said construct confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection.<br>
This invention also concerns a recombinant expression construct which confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response against diseases caused by fungal pathogens wherein said gene fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof and (2) an AVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NO:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to regulatory sequences.<br>
9<br><br>
r<br>
Also of interest are-pJantsjransforrned with such a gene or recombinant expression   <br>
construct and seeds obtained from such plants.	 <br>
BIOLOGICAL DEPOSIT<br>
The fungal strain 0-137 has been deposited with the American Type Culture Collection (ATCC), 10801 University Boulevard, Manassas, VA 20110-2209, and bears the following designation, accession number and date of deposit. Fungal strain G-213, a pathogen isolated from DigUaria smuisii in Japan, was obtained from the collection of Jean Loup Notteghem, Laboratoire de phy topathologie, Institu. de Recherches Agronomiques Tropicales et des Cultures Vivrieres, Centre de Cooperation Internationale en Recherche Agronomique pour le Developpement (CIRAD), BP 5035, 34032 Montpellier Cedex 1, France, and is available under the name JP34. This strain has also been deposited with the ATCC.<br>
Organism	Accession Number	Date of Deposit<br>
M grisea 0-137	ATCC 74457	August 3,1998<br>
M. grisea G-213	PTA-191	June 8, 1999<br>
BRIEF DESCRIPTION OF THE SEQUENCE LISTINGS AND FIGURES<br>
The invention can be more fully understood from the following detailed description and the Sequence Descriptions which form a part of this application. The Sequence Descriptions contain the three letter codes amino acids as " Ined in 37 C.F.R. 1.822 which are incorporated herein by reference. The nucleotide sequences read from 5" to 3".<br>
3<br>
SEQ ID NO:l is the 5757 nucleotide sequence of the genomic clone of the Pi-ta gene from Oryza saliva variety Yashiro-mochi.<br>
SEQ ID NO:2 is the predicted protein sequence encoded by the Pi-ta gene set forth in SEQ ID NO: 1.<br>
SEQ ID NO:3 is the 5113 nucleotide sequence of the genomic clone of the, pi-ta gene from susceptible Oryza sativa variety Tsuyuake.<br>
SEQ ID NO:4 is the predicted protein sequence encoded by the pi-ta gene set forth in SEQ ID NO:3.<br>
SEQ ID NO:5 is the 672 nucleotide coding sequence of an AVR-Pita (formerly called AVR2-YAMO) cDNA from M. grisea rice pathogen 0-137.<br>
SEQ ID NO:6 is the predicted protein sequence encoded by the sequence set forth in SEQ ID NO:5.<br>
SEQ ID NO:7 is the 675 nucleotide coding sequence of an A VR-Pita cDNA from M. grisea pathogen G-213 that infects Digitaria species.<br>
SEQ ID NO:8 is the predicted protein sequence encoded by the sequence set forth in SEQIDNO:7.<br>
10<br><br>
SEQ ID NOS:9-24 set forth the sequences of oligonucleotide (RAPD) primers used in the genetic mapping and cloning of the Pi-ta region.<br>
SEQ ID NOS:25-31 are oligonucleotide primers used in PCR amplification ofBAC insert ends.<br>
SEQ ID NOS.-32 and 33 are PCR primers used to verify the location of RAPD marker SP4B9.<br>
SEQ ID NOS:34-36 are partial sequences obtained from BAC clone RB142E8 which contains a single copy of the Pi-ta candidate gene PRG2.<br>
SEQ ID NOS:37-64 are sequencing primers used in the full sequencing of the genomic clone for Pi-ta (PRG2) set forth in SEQ ID NO:l.<br>
SEQ ID NO:65 is an oligonucleotide primer used to obtain a Pi-ta nucleic acid fragment.<br>
SEQ ID NOS:66 and 67 are PCR primers used to amplify A VR-Pita]76 an A VR-Pita nucleic acid fragment that directly encodes the putative mature protease.<br>
SEQ ID NO.-68 is the 5222 nucleotide sequence of an EcoW-HMm fragment that contains 2425 bp of the native Pi-ta promoter (nucleotides 1 to 2425) and Pi-ta cDNA (nucleotides 2426 5211 2426-5212). y :<br>
SEQ ID NOS :69 and 70 are oligonucleotide primers used to generate a linker fragment in constructing pML135.<br>
SEQ ID NOS:71 and 72 are PCR primers used to amplify the Adhl -6 intron from maize genomic DN \.<br>
SEQ ID NOS:73 and 74 are PCR primers used to amplify the A VR-Pita nucleic acid fragment (SEQ ID NO:5) in the process of constructing pAVR3.<br>
Figure 1A-B. Molecular map of the chromosome region containing the R-gene.<br>
A. Each of 1440 RAPD primers was tested for identification of polymorphisms linked to the R-gene derived from the rice variety Yashiro-mochi. This Figure shows the linkage map produced using sixteen primers that identified polymorphic DNA fragments segregating with the R-gene in 119 members of the doubled-hapldid (DH) population. The most likely order of markers (as defined by the computer program Mapmaker, version2) is given, with the distance between markers shown in centimorgans (cM) on the left. Note that the map is not drawn to scale. Markers SP7C3, SP7H8 and SP8C6 showed no recombination with the R-gene in this population, while markers SP8B8 and SP3G6 mapped to distal ends of the linkage group.<br>
B. Linkage map resolution was increased by testing the 16 markers on an additional 151 DH lines and on 720 F2 progeny. In the larger total population of 990 individuals, markers SP7C3, SP7H8 and SP8C6 again showed no recombination with the R-gene. However, this analysis resolved flanking markers that were previously clustered. The markers SP4B9 and SP9F3 most immediately flank the R-gene region.<br><br>
Figure 2. Positional cloning of Pi-ta. Chromosome walking experiments were begun with the single copy polymorphic sequence originally identified with RAPD marker SP7C3. One step to the left identified one recombination border, that is BAC clones containing the marker SP4B9, which was separated from the R-gene by a recombination event in the F2 progeny line K25. Open boxes indicate locations of the BAC probes used to identify overlapping BAC clones, and black boxes indicate probes used to verify the overlaps. BAC clones that were analyzed by random genomic sequencing are indicated by the broken lines. The Pi-ta gene candidate (PRG2). as well as the DNA ma -kers from 179G5R vhrough 77D8-12 (shown above the line representing the genome region), failed to recombine with the marker SP7C3 and the phenotypically-defined R-gene (determined by infection assays) in members of the rice mapping population. Marker SP9F3 that defines the right border showed recombination with the R-gene in one DH line, YT171.<br>
Figure 3. Restriction map of the R-gene showing the locations of the first three subclones obtained (dotted lines) through random genomic sequencing of BAC RB142E8. These were bad42e8.pk0001.fi 2 (fl2: SEQ ID NO:34), bacl42e8.pk0001.b7 (b7; SEQ ID NO:35), and bacl42e8.pk0001.f8 (f8; SEQ ID NO:36). Subsequent sequencing identified overlapping subclones bacl42e8.pk0001.el0 (elO), and bacl42e8.pk0005.cl0 (clO) from the "no hits class". A 5.3 kb fragment from the BAC RB142E8 (#7) was subcloned into plasmid bacl42e8.pkO O01.f8 to give pCB1641 containing the entire R-gene as described in Figure 4A. On the topmost line, the two thick black bars indicate the Pi-ta coding sequence, with the intervening thin black line representing the single intron in the gene, and thi. arrow representing direction of gene transcription. Restriction sites included are BamHI (B) and EcoRI (E).<br>
Figure 4. Construction of plasmidspCB 1641 and pCB 1645. Restriction sites included are BamHI" (B), EcoRI (E), Hindlll (H), Sstll and Sail. Italicized restriction sites (E.Sall) are from the vector and not from the rice genomic sequence.<br>
A.	Plasmid pCB 164 L containing the genomic full length PRQ2 coding sequence<br>
with 1255 bp of promoter sequence, a 1463 bp intron and 2 kb of 3" untranslated sequence,<br>
was constructed by ligating the 5.3 kb EcoRI rice genomic fragment from Clone #7 in place<br>
of the (201 bp) EcoRI fragment from the 2.2 kb rice DNA insert in bacl42e8.pk0001 .f8.<br>
B.	Plasmid pCB 1645 containing the 3" 1028 bp of the PRG2 coding sequence<br>
(positions 4477-5505) was constructed by amplifying this region from plasmid pCB1641 by<br>
PCR using primers GB46 (SEQ ID NO: 63) and GB47 (SEQ ID NO: 64). These primers<br>
contained restriction sites for Sail and EcoRI respectively. The PCR fragment was digested<br>
with Sail and £coRI and cloned into the corresponding sites of pGAL4-AD (Stratagene).<br>
Figure 5. Relative primer locations in the PRG2 nucleic acid fragment contained in plasmid pCB1641. These primers were used to amplify genomic DNA fragments from resistant and sensitive rice varieties and for DNA sequencing.<br>
12,<br><br>
Figure 6. Deduced Pi-ta protein sequence. Five amino acid differences found in sensitive variants are indicated above the R-protein sequence. Features of the R-protein discussed in the text are underlined: the P-loop domain between amino acids 236 and 244. a kinase 2a domain between amino acids 314 and 323, a kinase 3a domain between amino acids 342 to 353, a hydrophobic domain between amino acids 407 to 415. Four potential N-glycosylation sites at positions 339, 556, 654 and 838 are marked with a double underline.<br>
Figure 7. Diagram of constructs inplasmids pCB1947, pML63 and pCBl926.<br>
A.	pCB1947 contains a construct of the AVR-Pita ioslated nucleic acid fragment<br>
(formerly called AVR2-YAMO) engineered to encode directly the putative mature protease<br>
for the transient expression experiments. This processed form of the isolated nucleic acid<br>
fragment, designated^ VR-Pita]76, encodes a polypeptide comprised of amino acids 48 to<br>
223 as set forth in SEQ ID NO:6. An initiation methionine was added to the construct<br>
through the Ncol site used in the cloning process. The AVR-Pita coding sequence was first<br>
amplified by PCR using primers YL20 (SEQ ID NO:66) containing an in-frame Ncol site<br>
and YL37 (SEQ ID NO:67) containing a Kpril site, and cloned into the Ncol-Kpnl site of<br>
pML142, resulting in vector pCB1947. The maize Adhl-6 intron inserted downstream of the<br>
35S promoter results in enhanced expression in monocots. This intron is described in<br>
Mascarenhas D. et al. (1990) Plant Mol Biol. 15:913-920.<br>
B.	pML63 contains the uidA gene (which encodes the GUS enzyme) operably linked<br>
to the CaMV35S promoter and 3" NOS sequence. pML63 is modified from pMH40 to<br>
produce a minimal 3" NOS terminator fragment. pMH 40 is described in WO 98/16650<br>
which published on April 23,1998, the disclosure of which are hereby incorporated by<br>
reference. Using standard techniques familiar to those skilled in the art, the 770 base pair<br>
terminator sequence contained in pMH40 was replaced with a new 3" NOS terminator<br>
sequence comprising nucleotides 1277 to 1556 of the sequence published by Depicker et al.<br>
(1982, J. Appl. Genet. 1:561-574).<br>
C.	pCB1926 contains the Pi-ta cDNA construct that was created by first amplifying a<br>
2.1 kb partial Pi-ta cDNA nucleic acid fragment from first strand cDNA using primers F12-1<br>
(SEQ ID NO:44) and GB67 (SEQ ID NO:65). A synthetic full-length cDNA was generated<br>
by incorporating a 706 bp Ncol-BamUl fragment containing the 5" end of the genomic Pi-ta<br>
gene from pCB1641, resulting in plasmid pCB1906. A 3.1 kb EcoTU fragment from<br>
pCB1649 containing 2425 bp of the native Pi-ta promoter sequences (pPi-ta) and 736 bp of<br>
the 5" Pi-ta coding sequence was then inserted into the EcoRl sites of pCBl 906 to replace<br>
the 736 bp 5" end of the synthetic cDNA nucleic acid fragment, producing pCB1926.<br>
Figure 8. Particle bombardment transient assay for Pi-ta function in rice plants. These rice leaves show the results of biolistic transient expression of GUS and the Pi-ta and AVR-Pita cDNAs in rice using the constructs in Figure 7. All seedlings were co-bombarded with 35S::GUS (pML63) and 35S/adh::A VR-Pita (pCB1947). Leaves 3 and 4 were taken<br>
13<br><br>
from seedlings that were also bombarded with the Pi-ta cDNA under the control of the native 2.425 kb Pi la promoter (pCB1926). Presence (Pi-ta) or absence (-) of the endogenous Pi-ta gene in rice varieties YT14 and YT16 is indicated. GUS expression is indicated by the presence of dark spots in the leaf in the Figure, which are most prominent in the leftmost leaf. The absence of blue GUS staining was used as an indicator of the HR resistance response. The cloned PRG2 candidate nucleic acid fragment substitutes for the endogenous Pi-ta gene in this assay. YT14 and YT16 indicate the source of the rice leaves; both are DH rice lines, but YT14 contains an endogenous Pi-ta nucleic acid fragment while YT16 does not.<br>
Figure 9. DNA gel blot analysis on primary transformants to determine presence of the intact Pi-ta nucleic acid fragment. Genomic DNA was digested with EcoRJ (left hand side of blot) or EcoRV (right hand side of blot) and probed with 1028 bp £coRI-Sa/l 3" end fragment of the Pi-ta nucleic acid fragment cloned in plasmid pCB1645 (Figure 4B). The bands corresponding to the endogenous susceptible pi-ta allele in the Nipponbare transformants and non-transformed Nipponbare parent are indicated. AH additional bands represent copies of the Pi-ta nucleic acid fragment from plasmid pCBl641. Primary transformants 22-6-1-1,22-6-3-1 and 22-6-11-4 are resistant while 22-2-7-2; 22-3-1-1 and 22-4-1-1 are intermediate resistant.<br>
Figure 10. DNA gel blot analysis of 31 hygromycin resistant Rl individuals from the intermediate resistant primary transformant 22-6-10-1 to determine presence of the intact Pi-ta nucleic acid fragment. Genomic DNA was digested with EcoRV and probed with 860 bp EcoKV-Hindlll 3" end fragment of the Pi-ta nucleic acid fragment isolated from plasmid pCB1926. The band corresponding to the endogenous susceptible pi-ta allele in the Nipponbare transformants and non-transformed Nipponbare parent is indicated. The additional upper band represents a copy of the Pi-ta nucleic acid fragment from plasmid pCB1641. The R0 primary transformant 22-6-10-1 (lane 14) has the same RFLP as the J1 Rl progeny (A-K). The poor DNA quality from this plant resulted in distorted and uneven bands in lane 14 on the Southern blot. All 11 Rl progeny contain the same RFLP, and are intermediate disease resistant indicating stable inheritance of the Pi-ta nucleic acid fragment and the disease resistance phenotype.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In the context of this disclosure, a number of terms shall be utilized.<br>
The term "disease resistance gene" means a gene encoding a polypeptide capable of triggering a defense response in a plant cell or plant tissue. The terms "disease resistance gene", "resistance (R) gene" and "R" gene are used interchangeably herein. The resistance that results from a defense response can take several forms. For example, in some genetic backgrounds resistance may take the form of no visible symptoms on inoculated plant tissue, and in others resistance may include small brown spots from which the fungus does not<br><br>
                                                                 14<br><br>
spomlate and does not reinitiate infection. In both cases, the fungal pathogen does not complete its life cycle and disease development is stopped. In some cases disease resistance may take the form of smaller-sized lesions that do not produce the quantity of fungal spores typical of the full disease.<br>
A "defense response" is a specific defensive reaction produced by a host, e.g., a plant, to combat the presence of an infectious agent or pathogen.<br>
A "Pi-ta resistance gene" is a disease resistance gene encoding a polypeptide capable of triggering a defense response in a plant cell or plant tissue against a fungal pathogen such as Magnaporthe grisea.<br>
The term "Pi-ta resistance gene mediated defense response" means a defense response due to the production of the polypeptide encoded by the Pi-ta resistance gene and elicited by the presence of a fungal pathogen.<br>
An "AVR-Pita isolated nucleic acid fragment" is a nucleic acid fragment isolated from a pathogen wherein the nucleic acid fragment encodes a polypeptide whose direct or indirect interaction with the Pi-ta resistance protein is responsible for triggering the Pi-ta resistance gene mediated defense response.<br>
An "isolated nucleic acid fragment" is a polymer of RNA or DNA that is single- or double-stranded, containing synthetic, non-natural or altered nucleotide bases. An isolated nucleic acid fragment in the form, of a polyrner of DNA may be comprised of one or more segments of cDNA, genomic DNAJ orsyntnetre DNA.   % O<br>
The terms "subfragment that is functionally equivalent" and "functionally equivalent subfragment" are used interchangeably herein. These terms refer to a portion or subsequence of an isolated nucleic acid fragment in which the ability to alter gene expression or produce a certain phenotype is retained whether or not the fragment or subfragment encodes an active enzyme. For example, the fragment or subfragment can be used in the design of chimeric genes to produce the desired phenotype in a transformed"<br><br><br>
The terms "substantially similar" and "corresponding substantially" as used herein refer to nucleic acid fragments wherein changes in one or more nucleotide bases does not affect the ability of the nucleic acid fragment to mediate gene expression or produce a certain phenotype. These terms also refer to modifications of the nucleic acid fragments of the instant invention such as deletion or insertion of one or more nucleotides that do not substantially alter the functional properties of the resulting nucleic acid fragment relative to the initial, unmodified fragment. It is therefore understood, as those skilled in the art will appreciate, that the invention encompasses more than the specific exemplary sequences.<br>
15<br><br>
Moreover, the skilled artisan recognizes that substantially similar nucleic acid sequences encompassed by this invention are also defined by their ability to hybridize, under moderately stringent conditions (for example, 0.5 X SSC, 0.1% SDS, 60°C) with the sequences exemplified herein, or to any portion of the nucleotide sequences reported herein and which are functionally equivalent to the promoter of the invention. Preferred substantially similar nucleic acid sequences encompassed by this invention are those sequences that are 80% identical to the nucleic acid fragments reported herein or which are.80% identical ,to any portion of the nucleotide sequences reported herein. More preferred are nucleic acid fragments which are 90% identical to the nucleic acid sequences reported herein, or which are 90% identical to any portion of the nucleotide sequences reported herein. Most preferred are nucleic acid . fragments which are 95% identical to the nucleic acid sequences reported herein, or which are 95% identical to any portion of the nucleotide sequences reported herein. Sequence alignments and percent similarity calculations may be determined using the Megalign program of tbe"LASARGENE bioinformatics computing suite (DNASTAR Inc., Madison, WI). Multiple alignment of the sequences are performed using the Clustal method of alignment (Higgins and Sharp (1989) CABIOS. 5:151-153) with the   default parameters (GAP PENALTY=10, GAP LENGTH PENALTY=I0). Default parameters for pairwise alignments and calculation of percent identiy of protein sequences using the Clustal.method are KTUPLE^l, GAP PENALTY=3, WINDOW=5 and DIAGONALS SAVED=5. For nucleic acids these parameters are GAP PENALTY=10, GAP LENGTH PENALTY=10, KTUPLE=2, GAP PENALTY=5, WINDQW=4 and DIAGONALS SAVED=4. A "substantial portion" of an amino acid or nucleotide sequence comprises enough of the amino acid sequence of a polypeptide or the nucleotide" sequence of a gene to afford putative identification of that polypeptide or gene, either by manual evaluation of the sequence by one skilled in the art, or by computer-automated sequence comparison and identification using algorithms such as BLAST (Altschul, S. F., et ah, (1993; J. Mol. Biol. 275:403-410) and Gapped Blast (Altschul, S.-F: et al., (1997) Nucleic Acids Res. J?5.33R9-3402^:sco also www.ncbi.nlm.nih.gov/BLAST/).<br>
"Gene" refers to a nucleic acid fragment that expresses a specific protein, including regulatory sequences preceding (5" non-coding sequences) and following (3* non-coding sequences) the coding sequence. "Native gene" refers to a gene as found in nature with its own regulatory sequences. "Chimeric gene" refers to any gene that is not a native gene, comprising regulatory and coding sequences that are not found together in nature. Accordingly, a chimeric gene may comprise regulatory sequences and coding sequences that are derived from different sources, or regulatory sequences and coding sequences derived from the same source, but arranged in a manner different than that found in nature. "Endogenous gene" refers to a native gene in its natural location in the genome of an organism. A "foreign" gene refers to a gene not normally found in the host organism, but<br>
-14<br><br>
that is introduced into the host organism by gene transfer. Foreign genes can comprise native Tmsertedlnto a non-native prganisirk.pi; chimeric genes. „As"transgene"is a gene that has been introduced into the .genome by a transformation procedure.<br>
"Coding sequence" refer? to a DNA sequence that codes for a specific amino acid sequence. "Regulatory sequences" refer to nucleotide sequences located upstream (5" non-coding sequences), within, or downstream (31 non-coding sequences) of a coding sequence, and which influence the transcription, RNA processing or stability, or translation of the associated coding sequence. Such sequences can be native or non-native. Regulatory sequences may include, but are tfot limited to, promoters, translation leader sequences, introns, and polyadenylation recognition sequences.<br>
"Pathogen" refers to an organism or an infectious agent whose infection around or inside the cells of viable plant tissue elicits a disease response.<br>
"Promoter" refers to a DNA sequence capable of controlling the expression of a coding sequence or functional RNA, The promoter sequence consists of proximal and more distal upstream elements, the latter elements often referred to as enhancers. Accordingly, an "enhancer" is a DNA sequence which can stimulate promoter activity and may be an innate element of the promoter or a heterologous element inserted to enhance the level or tissue-specificity of a promoter. Promoters may be derived in their entirety from a native gene, or be composed of different elements derived from different promoters found in nature, or even comprise synthetic DNA segments. It is understood by those skilled in the an that different promoters may direct the expression of a gene in different tissues or cell types, or at different stages of development, or in response to different environmental conditions. Promoters which cause a gene to be expressed in most cell types at most times are commonly referred to as "constitutive promoters". New promoters of various types useful in plant cells are constantly being discovered; numerous examples may be found in the compilation by Okamuro and Goldberg, (1989) Biochemistry of Plants 75:1-82. It is further recognized that since in most cases the exact boundaries of regulatory sequences have not been completely defined, DNA fragments of some variation may have identical promoter activity.<br>
An "intron" is an intervening sequence in a gene that does not encode a portion of the protein sequence. Thus, such sequences are transcribed into RNA but are then excised and are not translated. The term is also used for the excised RNA sequences. An "exon" is a portion of the sequence of a gene that is transcribed and is found in the mature messenger RNA derived from the gene, but is not necessarily a part of the sequence that encodes the final gene product.<br>
The "translation leader sequence" refers to a DNA sequence located between the promoter sequence of a gene and the coding sequence. The translation leader sequence is present in the fully processed mRNA upstream of the translation start sequence. The translation leader sequence may affect processing of the primary transcript to mRNA,<br>
17<br><br>
mRNA stability or translation efficiency. Examples of translation leader sequences have been described (Turner, R. and Foster. G. D. (1995) Molecular Biotechnology 3:225).<br>
The "3" non-coding sequences" refer to DNA sequences located downstream of a coding sequence and include polyadenylation recognition sequences and other sequences encoding regulatory signals capable of affecting mRNA processing or gene expression. The polyadenylation signal is usually characterized by affecting the addition of polyadenylic acid tracts to the 3" end of the mRNA precursor. The use of different 3" non-coding sequences is exemplified by Ingelbrecht et al., (1989) Plant Cell 7:671-680.<br>
"RNA transcript1" refers to the product resulting from RNA polymerase-catalyzed transcription of a DNA sequence. When the RNA transcript is a perfect complementary copy of the DNA sequence, it is referred to as the primary transcript or it may be a RNA sequence derived from posttranscriptional processing of the primary transcript and is referred to as the mature RNA. "Messenger RNA (mRNA)" refers to the RNA that is without introns and that can be translated into protein by the cell. "cDNA" refers to a DNA that is complementary to and synthesized from a mRNA template using the enzyme reverse transcriptase. The cDNA can be single-stranded or converted into the double-stranded form using the klenow fragment of DNA polymerase I. "Sense" RNA refers to RNA transcript that includes the mRNA and so can be translated into protein within a cell or in vitro. "Antisense RNA" refers to a RNA transcript that is complementary to all or part of a target primary transcript or mRNA and that blocks the expression of a target gene (U.S. Patent No. 5,107,u65). The complementarity of an antisense RNA may be with any pan of the specific gene transcript, i.e., at the 5" non-coding sequence, 3" non-coding sequence, introns, or the coding sequence. "Functional RNA" refers to antisense RNA, ribozyme RNA, or other RNA that may not be translated but yet has an effect on cellular processes.<br>
The term "operably linked" refers to the association of nucleic acid sequences on a single nucleic acid fragment so that the function of one is affected by the other. For example, a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence (i.e., that the coding sequence is under the transcriptional control of the promoter). Coding sequences can be operably linked to regulatory sequences in sense or antisense orientation.<br>
The term "expression", as used herein, refers to the production of a functional end-product. Expression or overexpression of a gene involves transcription of the gene and translation of the mRNA into a precursor or mature protein.   "Antisense inhibition" refers to the production of antisense RNA transcripts capable of suppressing the expression of the target protein. "Overexpression" refers to the production of a gene product in transgenic organisms that exceeds levels of production in normal or non-transformed organisms. "Co-suppression" refers to the production of sense RNA transcripts capable of suppressing<br>
18<br><br>
the expression of identical or substantially similar foreign or endogenous genes (U.S. Patent No. 5,231,020).<br>
"Altered expression" refers to the production of gene product(s) in transgenic organisms in amounts or proportions that differ significantly from that activity in comparable tissue (organ and of developmental type) from wild-type organisms.<br>
"Mature" protein refers to a post-translationally processed polypeptide; i.e., one from which any pre- or propeptides present in the primary translation product have been removed. "Precursor" protein refers to the primary product of translation of mRNA; i.e., with pre- and propeptides still present. Pre- and propeptides may be but are not limited to intracellular localization signals.<br>
A "chloroplast transit peptide" is an amino acid sequence which is translated in conjunction with a protein and directs the protein to the chloroplast or other plastid types present in the cell in which the proteins made. "Chloroplast transit sequence" refers to a nucleotide sequence that encodes a chloroplast transit peptide. A "signal peptide7 is an amino acid sequence which is translated in conjunction with a protein and directs the protein to the secretory system (Chrispeels, J. J., (1991) Ann. Rev. Plant Phys. Plant Mol. Biol. 42:21-53). If the protein is to be directed to a vacuole, a vacuolar targeting signal (supra) can further be added, or if to the endoplasmic reticulum, an endoplasmic reticulum retention signal (supra) may be added. If the protein is to be directed to the nucleus, any signal peptide present should be removed and instead a nuclear localization signal included (Raikhel (1992) Plant Phys. 100:1627-1632).<br>
"Transformation" refers to the transfer of a nucleic acid fragment into the genome of a host organism, resulting in genetically stable inheritance. Host organisms containing the transformed nucleic acid fragments are referred to as "transgenic" organisms. The preferred method of cell transformation of rice, com and other monocots is the use of particle-accelerated or "gene gun" transformation technology (Klein et al., (1987) Nature (London) 327:70-73; U.S. Patent No. 4,945,050), or an Agrobacterium-mediated method using an appropriate Ti plasmid containing the transgene (Ishida Y. et al., 1996, Nature Biotech. 14:745-750).<br>
Standard recombinant DNA and molecular cloning techniques used herein are well known in the art and are described more fully in Sambrook, J., Fritsch, E.F. and Maniatis, T. Molecular Cloning: A Laboratory Manual; Cold Spring Harbor Laboratory Press: Cold Spring Harbor, 1989 (hereinafter "Sambrook").<br>
"PCR" or "Polymerase Chain Reaction" is a technique for the synthesis of large quantities of specific DNA segments, consists of a series of repetitive cycles (Perkin Elmer Cetus Instruments, Norwalk, CT). Typically, the double stranded DNA is heat denatured, the two primers complementary to the 3" boundaries of the the target segment are annealed at<br><br>
                                                                       19<br><br>
low temperature and then extended at an intermediate temperature. One set of these three consecutive steps comprises a cycle.<br>
An "expression construct"as used herein comprises any of the isolated nucleic acid fragments of the invention used either alone or in combination with each ortier as discussed herein and further may be used in conjunction with a vector or a subfragment thereof.  If a vector is used then the choice of vector is dependent upon the method that will be used to transform host plants as is well known to those skilled in the art.  For example, a plasmid \ector can be u.ced. The skilled artisan is well aware of the genetic elements that must be present on the vector in order to successfully transform, select and propagate host cells comprising any of the isolated nucleic acid fragments of the invention. The skilled artisan will also recognize that different independent transformation events will result in different levels and patterns of expression (Jones et al., (1985) EMBO J. 4:2411-2418; De Almeida et al., (1989) Mol. Gen. Genetics 275:78-86), and thus that multiple events must be screened in order to obtain lines displaying the desired expression level and pattern.  Such screening may be accomplished by Southern analysis of DNA, Northern analysis of mRNA expression. Western analysis of protein expression, or phenotypic analysis. The terms "expression construct" and "recombinant expression construct" are used interchangeably herein.<br>
The present invention concerns an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response against disease caused by a fungal pathogen whereu. said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof.<br>
Also of interest is a gene comprising these isolated nucleic acid fragment(s) operably linked to suitable regulatory sequences, plants comprising such genes in their genome, and seeds of such plants. For example, such a gene could be native or chimeric.<br>
In another aspect, this invention also concerns a method of conferring a Pi-ta resistance gene-mediated defenr.e response against disease caused by a fungal pathogen in plants which comprises introducing into the genome of the plant any of the aforementioned genes.<br>
In still another aspect, this invention concerns a method of conferring a resistance gene-mediated defense response in plants, comprising transforming a plant with a recombinant expression construct which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene mediated defense response against diseases caused by fungal pathogens wherein said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof and (2) an AVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ED NO:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to<br>
20<br><br>
regulatory sequences for expression at the site of infection and further wherein said construct confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection.<br>
In still an even further aspect, this invention concerns a recombinant expression construct which confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene mediated defense response against diseases caused by fungal pathogens wherein said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof and (2) an AVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NO:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to regulatory sequences. Also of interest are plants transformed with such an expression construct and seeds obtained from such transformed plants.<br>
Cloned R-genes can be used to facilitate the construction of crop plants that are resistant to pathogens. In particular, transformation technology can be used to stack multiple single genes into an agronomic germplasm without linked genomic sequences that accompany genes transferred by classical breeding techniques. Cloned R-genes also can be used to overcome the inability to transfer disease resistance genes between plant species by classical breeding.<br>
The present invention provides an isolated nucleic acid fragment ti*at has utility in controlling rice blast disease, caused by the fungus Magnaporthe grisea, in rice. The isolated nucleic acid fragments of the invention are effective when the pathogen comprises an AVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NOS:5 or 7 or a functionally equivalent subfragment thereof. Alternatively, the isolated nucleic acid fragments of the invention which confer a Pi-ta resistance gene mediated defense response against diseases caused by fungal pathogens are effective when the AVR-Pita avinilence nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NOS:5 or 7 or a functionally equivalent subfragment thereof, are introduced into the genome of a host plant as a foreign nucleic acid fragment operably linked to a promoter that expresses the A VR-Pita nucleic acid fragment at the site of infection.<br>
As discussed below, other subpopulations of M. grisea possess AVR-genes that are homologous to that contained in strains that elicit the Pi-ta specific defense response in rice. Demonstrations that AVR-genes which trigger a Pj-ta-resistance gene-mediated defense response can be present in M. grisea rice pathogens in subpopulation I, in Digitaria pathogens in subpopulation II, and in Pennisetum pathogens in subpopulation III, support<br>
21<br><br>
broad utility for this gene in controlling M. grisea on the range of graminaceous hosts infected by this fungus.<br>
Thus, it is believed that the isolated nucleic acid fragments of the invention will have utility in controlling diseases caused by M. grisea on other cereal crops including, but not limited to, wheat, barley, com. finger millet, and pearl millet.<br>
The rice Pi-ta resistance nucleic acid fragment was cloned by a map-based cloning strategy that is familiar to those skilled in the art as is discussed below. It is shown that intri"ducuon of the clont d Pi-ta nucleic acid fragment into susceptible rice confers resistance to pauhogen strains in a manner consistent with that expected for the Pi-ta resistance nucleic acid fragment.<br>
The Pi-ta resistance gene product has a novel structure compared to all known classes of resistance gene products. The Pi-ta protein bears no resemblance to the class of membrane anchored proteins with extracellular LRRs. Although the Pi-ta gene product has some features in common with R-gene products with cytoplasmic LRRs such as the Arabidopsis R-genes RPS2 (Bent et aP., 1994, Science 265:1856-1860; Mindrinos et al., 1994, Cell 78:1089-1099) and RPM1 (Grant et al., 1995, Science 269:843-846), it lacks the LRR structure characteristic of this class of R-gene products. The Pi-ta protein also lacks either a leucine zipper motif or a Toll/Interleukin-1 receptor motif that further subdivides the cytoplasmic LRR class of R-gene products (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv. Plant Pathol. 24:89-167). Thus, the Pi-ta resistance gene product falls into a different class from any other R-gene y, duct currently identified!<br>
Japonica rice varieties Yashiro-mochi (Yamada et al., 1976, Ann. Phytppath. Soc. Japan 42:216) and Kl (Kiyosawa, 1984, Rice Genetics Newsletter 1:95) have been designated as differential rice varieties for the blast resistance gene Pi-ta. While the Pi-ta gene in variety Kl was introgressed into the Japonica background from the indica rice variety Tadukan, the origin of Pi-ta in Yashiro-mochi appears to be an upland rice variety called Okaine (Rybka et al., 1997, Mol. Plant Microbe Interact. 10:517-524 and references therein). One other resistance gene derived from variety Tadukan, the Pi-ta2 gene, appeared to be allelic or closely linked to Pi-ta, although the relationship between Pi-ta and Pi-ta2 appears more complex than would be expected for a pair of independent allelic R-genes (Kiyosawa, 1967, Japan. J. Breeding 17:165-172; Kiyosawa, 1971, JARQ 6:73-80). The R-gene named Pi-4(t) in IRRI NIL differential rice varieties is closely linked to, and may indeed be the Pi-ta gene (Inukai et al., 1994, Phytopathology 84:1278-1283). In addition, the chromosome 12 centromeric region with Pi-ta appears to contain other resistance genes, the major R-gene Pi-6(t) (Abadassi et al., 1990, Rice Genetics II, IRRI, Los Banos, the Philippines, 746-748). a blast QTL (Wang et al., 1994, Genetics 136:1421-1434). and virus and insect R-genes.<br>
22<br><br>
The genetic cross (cross 4360) that identified the PWL2 gene, an AVR-gene controlling host species specificity on weeping lovegrass (Sweigard et al., 1995, The Plant Cell 7:1221), also segregated for an additional fungal gene that determined the ability of rice pathogens to infect rice variety Yashiro-mochi. This second AVR-gene, A VR-Pita (formerly called AVR2-YAMO for Yashiro-mochi) was inherited from the parental strain, 4224-7-8, and was derived from the Chinese field isolate 0-137 (collected in 1985 at the China National Rice Research Institute in Hangzhou). In each of five complete tetrads derived from cross 4360, four of eight ascospore progeny were able to infect Yashiro-mochi and the others were not. Random spore analysis of cross 4360 and subsequent crosses confirmed segregation of AVR-Piia. Avirulent progeny from cross 4360 frequently produced a few fully pathogenic lesions on Yashiro-mochi. It was speculated that these rare lesions might be due to spontaneous mutations occurring at the A VR-Pita locus. Mutants that had lost function of AVR-Pita were isolated as described in Sweigard et al (1995, The Plant Cell 7:1221). These mutants that were now fully virulent toward Yashiro-mochi retained morphological and fertility characteristics as well as the MGR586 DNA fingerprinting profiles of the presumptive parent. The host specificities of the mutants toward rice varieties with other R-genes were unchanged.<br>
Although dominance is not easy to assess for genes in predominantly haploid fungi like M. grisea, the occurrence of virulent mutants suggested that the expressed form of this AVR gene functions to stop infection of Yashiro-mochi, as predicted by the gene-for-gene hypothesis. The genetic instability of the A VR-Pita gene aided in its cloning. The AVR-gene was found to cosegregate with a cluster of physical markers including the telomeric repeat sequence at the end of a linkage group in the M. grisea RFLP map produced from cross 4360 (Sweigard et al., 1993, Genetic Maps, edited by S.J. O"Brien, Cold Spring Harbor Laboratory, pp 3.112-3.117). Spontaneous mutants that had become virulent on Yashiro-mochi rice showed structural changes in telomeric restriction fragments that mapped with the avirulence gene, suggesting the gene resided within 1 to 2 kb of the tip of the chromosome (Valent and Chumley, 1994, The Rice Blast Disease, edited by Zeigler, Leong and Teng, CAB International, Wallingford). Southern analysis of genomic DNA from wild type avirulent strains and from spontaneous mutants that had acquired deletions at the chromosome end, identified the sizes of the terminal chromosome fragment produced by digestion of genomic DNA with various restriction enzymes. This analysis suggested that the AVR-gene resided within a telomeric 6.5 kb Bglll fragment that corresponded to the chromosome end. Cloning of the corresponding telomeric fragment allowed demonstration that it did indeed contain the A VR-Pita gene, which functioned to transform virulent pathogens of rice cultivar Yashiro-mochi into avirulent strains on Yashiro-mochi.<br>
The AVR-Pita nucleic acid fragment of the invention was isolated from the Chinese rice pathogen 0-137 encodes a protein with 223 amino acids (SEQ ID NO:5). Amino acids<br>
25<br><br>
173-182 form a characteristic motif of a neutral zinc metalloproteinase and natural or in vitro mutation of the motif residues destroys AVR-gene activity, that is. it no longer transforms virulent strains of the pathogen to avirulence on rice variety Yashiro-mochi. The predicted amino acid sequence has low levels of homology to other metalloproteinases characterized from fungi (Genbank Accession numbers L37524 and SI6547). The best characterized secreted fungal metalloprotease, NpII fxom Aspergillus oryzae, contains a 175 amino acid prepro-region that precedes a 177 amino acid mature region (Tatsumi et al., 1991, Mo!. Gen. Genet. 228:97-103). The predicted AVR-Pita amino acid sequence exhibits 35% homology and 29% identity with NpII, with the most significant homology confined to the mature 177 amino acid form of NpII. In addition, alignment of the amino acid sequences of AVR-Pita and NpII showed conservation of the cysteines involved in disulphide bonds in the mature NpII protein. It was anticipated that the AVR-Pita isolated nucleic acid fragment encodes a preproprotein that is processed to a mature metalloprotease containing 176 amino acids. Based on this prediction, an A VR-Pha!7 expression construct was engineered to produce directly the putative mature protease for functional analyses.<br>
M. grisea rice pathogens do not appear to require an A VR-Pita gene for fitness in the laboratory or under field conditions. For example, the well-characterized Chinese field isolate 0-135 (Valent et al., 1991, Genetics 127:87-101), which was collected at the same time as the source strain 0-137, lacks homology to this AVR-nucleic acid fragment. Rice pathogens such as 0-135 are not currently affected by the Pi-ta mediated defense response because they fail to elicit the response in Pi-ta cou.aining rice.<br>
Functional A VR-Pita nucleic acid fragments have been cloned from M. grisea strains that infect host plants other than rice, and are distantly related to rice pathogens in subpopulation la. including a Digitaria pathogen (JP34, isolated in Japan) from subpopulation III, and a Pennisetum pathogen (BF17, isolated in Burkina Faso) from subpopulation IV. The A VR-Pita nucleic acid fragment cloned from the Digitaria pathogen (SEQ ID N0:7, the G-213 AVR sequence) oorresponds to a translated amino acid sequence with 87.9% similarity and 84.7% identity to the 0-137 AVR-Pita amino acid sequence when compared by the Bestfit algorithm of the University of Wisconsin Computer group package 9.1 (Devereux et al., 1984, Proc Natl Acad Sci USA 12:387-395). The G-213 AVR-Pita (avirulence) nucleic acid fragment has the most divergent sequence identified which retains the ability to transform virulent rice pathogens into avirulent strains that elicit a Pi-ta resistance gene mediated defense response. Conservation of AVR-gene function between distantly related M. grisea strains that infect different grass species suggests that a cloned Pi-ta resistance gene will be effective in controlling the blast fungus on its other host plants, in addition to rice.<br>
The present invention also concerns a gene comprising an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response against disease<br>
24<br><br>
caused by a fungal pathogen wherein said nucleic acid fragment corresponds substantially to the nucleotide coding sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof wherein this fragment is operably linked to suitable regulatory sequences. Such a gene can be an isolated native gene or a chimeric gene. can be constructed using For example, a<br>
chimeric gene can be made which comprises the coding sequence of the nucleotide sequence of SEQ ID NO:68 operably linked to a native Pi-ta promoter fragment (nucleotides 1 to 2425) as set forth in SEQ ID NO:68.<br>
Transgenic plants can be made which are capable of mounting a Pi-ia resistance gene-mediated defense response against a fungal pathogen. Examples of plants which can be transformed with an isolated native gene or a chimeric gene as described herein include, but are not limited to, monocots. Preferably the monocot is a cereal. Most preferably, the<br>
«B"-SH*»"iSWS!H«!r™**SilBB!ffl^^<br>
rhonocoTisrice, wheat, barley, corn, fmger millet or pearl millet. Seeds of such transgenic<br>
	,<br>
Thus, this invention also concerns a method of conferring a Pi-ta resistance gene-mediated defense response against disease caused by a fungal pathogen in plants which comprises introducing into the plant any of the genes of the present invention as described herein. PZ-ta-mediated defenses are generated in the crop plant in response to A VR-Pita avirulence gene expression, either the native AVR-gene as expressed by the fungal pathogen. or a chimeric AVR-transgene expressed by the plant in response to the pathogen.<br>
In another aspect, this indention also concerns a method of conferring a resistance gene-mediated defense response in plants, comprising fransforming the plant with a recombinant expression construct which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene mediated defense response against diseases caused by fungal pathogens wherein said nucleic acid fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent subfragment thereof and (2) anAVR-Pita isolated nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NOS:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to regulatory sequences and further wherein said expression construct confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection.<br>
In still another, aspect this invention concerns a recombinant expression construct which confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection which comprises (1) an isolated nucleic acid fragment which confers a Pi-ta resistance gene-mediated defense response against diseases caused by fungal pathogens wherein said gene fragment corresponds substantially to the nucleotide sequence set forth in SEQ ID NOS:l or 68 or a functionally equivalent<br><br>
subfragment thereof and (2) a nucleic acid fragment corresponding substantially to the nucleotide sequence set forth in SEQ ID NO:5 or 7 or a functionally equivalent subfragment thereof, wherein said nucleic acid fragments are operably linked to regulatory sequences.<br>
Also of interest are plants transformed with such a recombinant expression construct and seeds obtained from such plants.  Such an expression construct can be prepared using techniques well known to those skilled in the art as is discussed above. Introduction of transgenes into plants, i.e.. transformation is well known to those skilled in the art. A preferred method of plant cell transformation is the use of particle-accelerated or "gene gun" transformation technology (Klein et al. (1978) Nature (London) 327.70-73; U.S. Patent No. 4,945,050). Examples of plants that can be transformed with such recombinant expression constructs include, but are not limited to, monocots. Preferably, the monocot is a cereal. Most preferably, the monocot is rice, wheat, barley, corn, finger millet or pearl millet,<br>
**    EXAMPLES<br>
The present invention is further defined in the following Examples. From the above discussion and these Examples, one skilled in the art can ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make various changes and modifications of the invention to adapt it to various usages and conditions.<br>
Unless otherwise stated, all parts and percentages are by weigtu and degrees are Celsius. Techniques in molecular biology were typically performed as described in Sambrook, J., Fritsch, E. F. and Maniatis, T. 1989. Molecular cloning - A Laboratory Manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor. New York.<br>
EXAMPLE 1<br>
Pathogen Strain Development for Identification of the<br>
Corresponding R-Gene<br>
An avirulent isolate of the pathogen is necessary to identify a host resistance gene, and the use of new pathogen strains may identify resistance genes that have so far escaped detection. Two fungal strains that show virulence (that is, they fail to elicit an R-gene mediated defense response) toward a rice variety with a particular R-gene are considered to lack the corresponding functional AVR-gene. However, although an avirulent strain of the fungus with a corresponding AVR-gene is necessary for identifying a host resistance gene. when two strains are avirulent toward a particular rice variety, they could have the same AVR-gene, or different AVR-genes triggering different R-gene mediated defense responses. Use of new or uncharacterized fungal strains may identify previously unidentified R-genes. This caution is especially required when working with field isolate rice pathogens such as 0-137, the source of the A VR-Pita gene. In addition, rice blast infection assays are<br>
26<br><br>
notoriously sensitive to environmental conditions, especially if the rice varieties have some level of general resistance to blast, or if the pathogens used are unstable in pathogenicity and morphology characteristics. Therefore, the use of thoroughly characterized pathogen strains is key to success.<br>
Multiple pathogen resources, including genetic populations segregating for the AVR-Pita gene, facilitated cloning the R-gene. Progeny strains such as 4360-R-l 7. 4360-R-27, 4360-R-30, and subsequent generation strain 4375-R-26 (Sweigard et al.. 1995, The Plant Cell 7:1221) had improved characteristics for these studies. Because the rice crosses in these experiments had at least two R-genes segregating, Pi-la and Pi-km, and because pathogen cross 4360 was segregating for AVR-genes corresponding to both known R-genes, there was a need to obtain genetically characterized pathogen strains bearing single AVR-genes. Such strains were needed to identify the effects of individual R-gene/AVR gene interactions on lesion development. There was also the need to maximize phenotypic differences between the resistant and susceptible interactions in order to assure accuracy of scoring the resistance phenotype among the rice progeny in the mapping population. Twenty-eight progeny strains were obtained from cross 4360 that contained A VR-Pita, but which did not contain A VR1-TSUY. It is believed that A VR1-TSUY is the AVR-gene corresponding to the blast resistance gene Pi-km in the japonica variety Tsuyuake. These 28 progeny were screened for stable ability to produce fully susceptible disease symptoms (Type 4 - Type 5 lesions) on one parental rice variety, and typical resistance responses (Type 0 - Type 1 lesions) on the other rice variety (Valent et al., 1991, Genetics 127:87). M. grisea isolate 4360-R-62 was chosen as a test strain which contained only A VR-Pita. Out of an additional 9 strains screened, 4360-R-67 was chosen as an excellent pathogen that contains only A VR1-TSW. Most of the R-gene mapping that is described below was done with these two fungal strains.<br>
In addition, mutants that have lost AVR-gene function are useful for comparison to the parental strain from which they were derived. For example, strain CP987 was derived from strain 4360-R-l 7 by isolation of a spontaneous virulent mutant. The AVR-Pita gene in CP987 was inactivated by a small deletion. Virulent mutants can easily be obtained from avirulent field isolates or laboratory strains as described in Sweigard et al. (1995) The Plant Cell 7:1221. In the opposite direction of pathogen trait alteration, field isolate Guyl 1, which is virulent on Yashiro-mochi, became avirulent on this host when it was transformed with a cloned AVR-Pita gene to produce strain CP3285. Fungal transformation in the rice blast system is routine, as described in Sweigard et al. (1995) The Plant Cell 7:1221.<br>
EXAMPLE 2 Plant Infection and Evaluation<br>
Infection assays were performed as previously described (Valent et al., 1991, Genetics 127:87). Conidia were collected from cultures grown on oatmeal agar plates by<br><br>
washing with a sterile 0.25% gelatin solution. Four to five individuals from each rice variety were sown in Metro-Mix® potting medium within plastic pots in a growth chamber. Plants were grown on a day cycle of 14 hours of light, 28 degrees and 70 to 85% relative humidity. Night conditions were 22 degrees and 85% relative humidity. Pots with two-week-old plants were placed into plastic bags in order to maintain 95-100% relative humidity required for pathogen penetration, and inoculated with pathogen strains such as 4360-R-62 or 4360-R-67 described in Example 1. A four ml aqueous suspension, containing 2.5 X 105 conidia per ml, was sprayed onto the plants using an artist"s air brush (Pasha, size 1). The plastic bags containing the inoculated plants were then closed with a twist-tie, and were incubated in low light conditions at room temperature for 24 hours. After 24 hours, the plants were removed from the bags, and were placed back in the growth chamber. Infection types, i.e., lesion symptoms, were scored 7 days after inoculation. A scale of 0-5 was used to classify infection phenotypes (Valent et al., 1991, Genetics 127:87-101). Lesion types of 0 and 1 were scored as resistant while lesion types 3 to 5 were scored as susceptible.<br>
1    EXAMPLE 3 Identification and Mapping of the R-gene Corresponding to the Rice Blast AVR-gene Development of Rice Mapping Populations. Two plant populations were used to identify physical markers linked to the R-gene. First, a doubled-haploid (DH) population was generate for our use at CIAT (Centro Internacional de Agricultura Tropical; Apdo aereo 6713, Cah, Colombia). In order to minimize problems"with segregation distortion that are known to occur in crosses between japonica and indica varieties (Wang et al., 1994, Genetics 136:1421 -1434), a decision was made to work with a cross between two japonica varieties, Yashiro-mochi (with the R-gene Pi-ta) and Tsuyuake (with the R-gene Pi-kni). Although the levels of polymorphism that exist between two japonica varieties are too low for purposes of mapping an entire rice genome, it was reasoned that because the Pi-ta gene originated from indica rice, there would be sufficient polymorphism in this R-gene region to allow mapping of the disease-resistance trait. Bulked segregant analysis of the results of this cross, described below, focused mapping efforts on particular regions of the genome.<br>
Production of DH rice lines was done according to standard technology to those skilled in the art. First, haploid callus was obtained from the pollen of Fl plants derived from reciprocal crosses between rice varieties Yashiro-mochi and Tsuyuake. Rice plants were then regenerated from this callus. Fertile plants derived from this procedure have undergone spontaneous diploidization of the haploid genome. Individual plants from reciprocal crosses were subjected to 7 selling generations to produce a total of 429 independent DH lines. Of these, 119 lines were used as the initial mapping population in this study.<br>
28<br><br>
For fine structure mapping, an F2 population was developed at DuPont by crossing two DH lines, YT4, which contained the active Pi-ta R-gene from variety Yashiro-mochi and YT10, which did not. Techniques for F2 analysis are well known to those skilled in the art.<br>
Segregation Analysis Identified a R-Gene Corresponding XoAVR-Pita. Reactions of 119 DH lines to the two fungal strains 4360-R-62 and 4360-R-67 were as follows: 35 resistant to both strains, 24 resistant to 4360-R-62, 29 resistant to 4360-R-67 and 31 susceptible to both strains. Progeny in these four phenotypic classes fit an expected 1: 1: 1: 1 ratio (X2 = 2.20; P &gt; 0.50), indicating, as expected, that Pi-ta and Pi-km were independently segregating R-genes. An additional 115 DH lines were also inoculated with strain 4360-R-62 alone, of which 62 lines exhibited resistant types and 53 lines exhibited susceptible types. Thus, the total ratio of resistant to susceptible lines with strain 4360-R-62 was 126 to 108, which did not deviate from the expected 1:1 ratio based on chi-square test (X2 = 1.38, P &gt; 0.20).<br>
Analysis of Fl and F2 progeny Yrom the fme structure mapping cross between rice lines YT10 and YT4 further supported the segregation of a dominant R-gene corresponding to the A VR-Pita avirulence gene in the pathogen. Four F j hybrids were infected with 4360-R-62 in order to determine if the R-gene phenotype was dominant or semi-dominant, as predicted. All the Fj hybrids exhibited reaction type 0 or type 1, similar to the resistant parent, indicating that the allele for blast resistance was dominant to the allele for susceptibility in rice. Seven hundred twenty F2 progeny were also infected with 4360-R-62. Infection assays for the F2 population were essentially the same as described in Example 2 with exceptions that plants were sown in plastic trays, 60 plants per tray, and inoculum was reduced to 1 X 105 conidia per ml. About 30 ml of conidial suspension was applied to each tray. F2 plants segregated as 541 resistant and 179 susceptible which nearly perfectly fit a 3:1 ratio (0.007; P &gt; 0.90) indicating a single dominant gene. For use in fme structure mapping studies, the susceptible plants from this analysis were carefully trimmed to remove diseased tissue, and transplanted into fresh potting medium.<br>
RAPD Screening To Identify DNA Markers Linked to the R-Gene. In order to identify physical markers linked to the R-gene, Random Amplified Polymorphic DNA (RAPD) markers (Williams et al., 1990, Nucleic Acids Research 18:6531-6535) were employed in an early stage of mapping. Polymorphisms, DNA segments that amplify from one parent and not the other, are detected based on amplification of random DNA segments with single primers of arbitrary nucleotide sequence. In order to focus on markers linked to the R-gene, bulked segregant analysis was employed. This is a method of assaying polymorphisms between two pools of individuals, one in which all individuals have the gene of interest and the other in which no individuals have the gene of interest. (Michelmore et al., 1991, Proc Natl Acad Sci USA 88:9828-9832). DNA segments that are not linked appear equally in both pools, while DNA segments that are linked will appear in one pool and not the other.<br>
29<br><br>
Rice total genomic DNA was isolated from individual DH lines using a method described previously (Dellaporta et al., 1983, Plant Mol. Biol. Reporter 1:19-21). After estimating DNA concentration by electrophoresis in gels vs. standard quantities, aliquois of DNAs from 10 DH lines that were resistant to 4360-R-62 and aliquots of DNAs from 10 DH lines that were susceptible to 4360-R-62 were combined to form two sets of "bulked" DNAs. Bulked DNAs w;ere used as templates to evaluate the amplification of RAPD markers. RAPD reactions were carried out in volumes of 20 ul. containing 80 mM Tris-HCl (pH 9.0); 20 mM (NH4)2S04; ?.5 mM MgCl2; 100 p.M each of dGTP, dATP, dCTP and dTTP; 0.2 uM primer; 10 ng of DNA; and 1 unit of Tag DNA polymerase (Perkin Elmer Cetus). The DNA fragments that were amplified with this procedure were separated on 2% agarose (New England BioLabs) gels in 1 X TAE buffer for 5 hours. Polymorphic fragments, either present or absent in resistant and susceptible pools, were further confirmed by using DNA templates from both parents, from the resistant and susceptible pools, and from each of the individuals forming the two pools.<br>
The pooled DNAs. either from the resistant or from the susceptible lines, were used as templates to screen 1440 RAPD primers. More than 95 percent of the primers gave rise to one or more amplified products. The average number of discrete bands per primer was approximately 8. From these amplifications, RAPD primers SP1B8, SP2C12, SP3F4, SP3G6, SP4A5, SP4B9, SP6C2, SP6G10, SP7C3, SP7H8, SP8B8, SP8C6, SP8E2, SP8FL SP9F3 and SP15B12 (SEQ ID NOS: 9-24) identified polymorphisms linked to the R-gene corresponding to A VR-Pita.<br><br>
SP1B8:	5"-TCAGCGCCT-3"	(SEQ ID NO:9)<br>
SP2C12:	5"-CCAATCGGAC-3"	(SEQ ID NO: 10)<br>
SP3F4:	5"-TAATGGGCGG-3"	(SEQIDNO:ll)<br>
SP3G6:	5"-GTCGCTACTG-3"	(SEQ ID NO: 12)<br>
SP4A5:	5"-ACAGCGCCTT-3"	(SEQ ID NO: 13)<br>
SP4B9:	5"-AGGCGTCTTC^"	(SEQ ID NO: 14)<br>
SP6C2:	5"-TAGCCAGACC-3"	(SEQ ID NO: 15)<br>
SP6G10:	5"-TCTATGCCCC-3"	(SEQIDNO:16)<br>
SP7C3:	5"-ArGGCAGATG-3"	(SEQ ID NO: 17)<br>
SP7H8:	5"-CGAGTCAACT-3"	(SEQ ID NO: 18)<br>
30<br><br>
SP8B8:	5"-GTAGAAGCCT-3"	(SEQ  ID  NO: 19)<br>
SP8C6:	5"-TCATGCGGAG-3"	(SEQ  ID  NO:20)<br>
SP8E2:	S"-CCATTTCCGT-S"	(SEQ ID N0:21)<br>
SP8F1:	5"-GGGAGGACTT-3"	(SEQ ID N0.22)<br>
SP9F3:	S"-AAAGGCAGTG^1	(SEQ ID NO:23)<br>
SP15B12:	5"-TGTGCAACGG-3"	(SEQ  ID NO:24)<br>
Initial mapping of markers linked to the R-gene. Segregation of linked RAPD molecular markers were scored for each of 119 individual DH rice lines. Linkage analysis between these RAPD markers and the R-gene,was performed with the computer program MAPMAKER (version 2.0) (Konieczny and Ausubel, 1993, The Plant Journal 4:403). A LOD score &gt; 3 was used to establish linkage between RAPD markers and the R-gene.<br>
Two-point analysis revealed strong linkage among the identified RAPD markers, and between these markers and the R-gene in the DH population. The order of these markers relative to the R-gene was determined by multiple point analysis. SP7C3, SP7H8 and SP8C6 mapped closest to the R-gene with no recombination observed among the 119 progeny, while SP8B8 and SP3G6 mapped to distal ends of the mapping region, with genetic distances of 8.6 and 7.3 cM from the R-gene, respectively. The most probable order of these markers is presented in Figure 1A. However, the order of a number of markers could not be determined unambiguously because the probability of any particular order was not greater than 100:1 relative to other alternative orders based on this population size. Fine Structure Mapping of the R-Gene. Increased mapping resolution was accomplished by scoring the segregation of the R-gene and the linked markers in a larger mapping population. Additional DH rice lines were included to bring the total to 270 lines in the mapping population. In addition, 720 F2 progeny from the genetic cross between DH lines YT4 and YT10 were included in this analysis. As described, 720 F2 plants were inoculated with fungal strain 4360-R-62 (containing only AVR-Pita) to determine which contained the corresponding R-gene. The susceptible plants from this analysis were recovered by removal of diseased tissue and by transplanting to fresh potting medium. As is well understood, F2 segregation analysis of RAPD markers is complicated because the dominance of these markers prevents discrimination between heterozygous and homozygous progeny. This difficulty can be overcome in some cases by isolating codominant markers from the the RAPD markers. For example, isolation of RFLP markers from RAPD markers is described in Example 4. Additional differentiation of heterozygous versus homozygous F2 rice lines<br><br>
was accomplished by checking for segregation of the resistance phenotype in the F3 generation. The fine structure map produced in this study is shown in Figure IB.<br>
EXAMPLE 4 Evidence that the R-gene Recognizing A VR-Pita is Pi-ta Three lines of evidence were obtained that the R-gene corresponding to the cloned avirulence gene is the blast resistance gene previously identified as Pi-ta. Mapping of blast resistance genes to rice chromosome 12. A recombinant inbred (RI) population of 192 individuals (prepared by crossing lines LH422 and 9024; Xiao et al., 1995, Genetics 140:745) was used to map the R-gene region to its rice chromosomal location. The R-gene was mapped indirectly using flanking RFLP markers that had been derived from polymorphic RAPD markers. RAPD fragments were eluted from gels, cleaned using Geneclean II (Bio 101, San Diego, CA) and subcloned into the TA cloning vector (Invitrogen, San Diego, CA). The subcloned DNA inserts were amplified by using T7 (SEQ ID NO:27) and SP6 (SEQ ID NO: 28) promoter primers and used as probes to survey the two parents of the mapping population. Probes P4A5, P1B8 , P7C3 and P8C6 showed polymorphisms with restriction enzymes Xbal, EcoRl, EcoRV and Dral, respectively. Allelic patterns for the four RFLP markers were scored for the RI population members and mapped using the MAPMAKER (version 2.0) program. All the markers were placed with a LOD score of greater than 3. Two point analysis established that all these markers were linked to a number of RFLP markers on chromosome 12. Multiple point analysis placed P7C3 and P8G6 in the intervals between RG341 and RZ670 with a geneuo distance of 1.3 cM and 3.2 cM respectively (Causse et al., 1994, Genetics 138:1251-1274). Markers P1B8 and P4A5 mapped between RG9 and RG869. Thus the R-gene corresponding to A VR-Pita maps to the same chromosomal region as the Pi-ta gene.<br>
AVR-Pita Mutants and Segregating Progeny Show Identical Specificity Toward Other Rice Varieties Reported to Contain Pi-ta. Although the R-gene maps to the chromosomal region containing the Pi-ta resistance gene, it might still be an unknown gene or one of the other R-genes mapping to this same region. A series of pathogen strains was identified, including 0-137, the Chinese field isolate that is the source of the AVR-Pita gene; 4360-R-17 and 4360-R-27, progeny of cross 4360 that contain the AVR-gene; 4360-R-30, a progeny of cross 4360 that does not contain the AVR-gene; and CP987 and CP983, mutants that have lost AVR-gene function. These isolates of the pathogen were used in infection assays on other rice varieties that contain the Pi-ta gene, specifically, Tadukan and Kl. The identical avirulence spectrum shown by these pathogen strains on Yashiro-mochi, Kl and Tadukan (Table 3) suggests that the R-gene corresponding to AVR-Pita is the previously identified gene Pi-ta. The rice variety Sariceltik which lacks the Pi-ta gene is susceptible to all of these strains of the pathogen.<br><br><br>
                                                                     32<br><br>
	table 3. Lesion Type on Rice Cultivar		<br>
Fungal Strain	AVR-Pita Functional	Sariceltik	Yashiro-mochi	Tadukan	Kl<br>
0-137	Yes	S	R	R	R<br>
4360-R-17	Yes	S	R	R	R<br>
CP987	no	S	S	S	S<br>
CP983	no	S	S	S	S<br>
4360-R-27	Yes	S	R	R	R<br>
4360-R-30	no	s	S	S	S<br>
S: Susceptible, Lesion Types 3-5	R: Resistant, Lesion Types 0-1<br>
infections with Japanese Differential Pathogen Strains Confirm the Specificity. Finally, inoculations were conducted with Japanese pathogen strains Ken60-19, Ken54-20, Ina72, Inal68, Ken54-04 (Kiyosawa, 1976, §ABRAO Journal 8:53-67; Yamada et al., 1976, Ann. Phytopath. Soc. Japan 42:216); these strains were used in the identifications of various blast resistance genes in rice, including Pi-ta. These strains were inoculated on DH progeny rice lines and differential rice varieties along with strains 4360-R-62 and 4360-R-67 obtained from this study. All of these field isolates, except Inal68, were avirulent on both differential rice varieties known to contain the Pi-ta R-gene, and DH progeny rice lines that contain the R-gene corresponding to AVR-Pita, thus demonstrating specificity of infection and disease that is consistent with the R-gene being Pi-ta.<br>
EXAMPLE 5 Construction of a BAC Library Methods for constructing large insert libraries are now routine for those familiar with the art (Wang et al., 1995, The Plant Journal 7:525). Standard techniques were employed for the construction of a bacterial artificial chromosome (BAC) library using genomic DNA from protoplasts of leaves of the DH rice line YT14 that carries the R-gene. Protoplasts were isolated from young leaves and sheaths as previously described (Wu and Tanksley, 1993, Plant Mol. Biol. 23:243). The DNA was partially digested with Hindm in agarose plugs and fractionated on CHEF gels. DNA fragments ranging from 100 to 150 kb were ehited and subjected to a second size selection to remove small molecules. Size-selected DNA was then recovered from agarose by treating with GELase (Epicentre Tech) and ligated into the HindlTl site of pBeloBACII vector (Shizuya et al., 1992, Proc. Natl. Acad. Sci. USA 89:8794). Transformation of E. coli DH10B cells was carried out by electroporation using Cell Porator Electroporation System I (BRL)- BAC colonies were selected on LB plates with chloramphenicol at 12.5 u.g/ml. A total of 20,160 independent colonies were picked and stored in 96-well microther plates containing 100 p.1 LB with chloramphenicol and 10%<br>
33<br><br>
glycerol. The average insert size of clones in the library is 110 kb based on analysis of 28 randomly selected colonies. The library contains approximately 5 haploid genome equivalents of rice DNA.<br>
EXAMPLE 6 Chromosome Walking and Identification of Pi-ta R-gene Candidate PRG2 The BAC library, contained in 210 microtiter plates, was spotted onto high density filters using a Biomek robot (Beckman Instruments). The entire library was arrayed onto 14 filters with a density of 16 plates spotted per filter. The set of filters was probed with the single copy KFLP probe, p7C3, obtained from the RAPD marker SP7C3 as described in Example 3. This marker cosegregated with the Pi-ta gene in the entire DH and F2 mapping populations. Four overlapping BAC clones were identified in this hybridization: 5C1, 22H9,136F8 and 179G5 (See Figure 2). With these materials and information as a starting point, two methods were used to isolate probes from the ends of the BAC rice inserts, and these probes were used to screen the library filters again. BAC DNAs were isolated using a midi-prep procedure based on standard alkaline lysis plasmid purification methods (Sambrook). The method for isolating the insert border sequences for one side of the vector involved digestion of the DNA with either BamHl or Sphl to remove the majority of the insert DNA. The remaining vector was religated and the insert sequences were isolated by PCR with either the M13-20 (SEQ ID NO:25) and reverse (SEQ ID NO:26) primers or the T7 (SEQ ID NO:27) and SP6 (SEQ ID NO:28) primers.<br>
M13-20:	5"-GTAAAACGACGGCCAGT-3*	(SEQ ID NO;25)<br>
M13-REV:	5"-GGAAACAGCTATGACCATG-3"	(SEQ ID NO:26)<br>
T7:	5"-GTAATACGACTCACTATAGGGC-3"	(SEQ ID NO:27)<br>
SP6:	5"-ATTTAGGTGACACTATAG-3"	(SEQ ID N0.28)<br>
The opposite border fragment was isolated by one of two methods. In the first method BAC clones were digested with Seal which deleted the majority of the genomic insert DNA. The remaining BAC clone was religated and the remainder of the insert amplified by PCR using the T7 (SEQ ID NO:27) and Belo840 (SEQ ID NO:29) primers. The second method involved an inverse PCR technique (Sambrook; Ochman H. et al. (1988) Genetics 120:621-623) after digestion of the BAC clone with BamHl and religation in which the opposite border fragment was amplified by PCR using primers IBAC1 (SEQ ID NO:30) and IBAC2 (SEQ ID NO:31). The border fragments obtained by PCR were subcloned from an agarose gel, and used as probes for isolating overlapping BAC clones.<br>
Belo840:     5,-TTTGTGATGGCTTCCATGTC-3,	(SEQ ID NO:29)<br>
34<br><br>
IBAC1:        5"-GTCGACTCTAGAGGATCC-3"	(SEQ ID NO:30)<br>
IBAC2:       5"-CTGCAGGCATGCAAGCTT-3"	(SEQ ID N0:31)<br>
Walking One Step to the Left Defines the Left Recombination Border. One insert border fragment of BAC clone 179G5, named 179G5R, appeared to be a single copy sequence using Southern gel blot analysis, and was used to probe the BAC library filters. This fragment identified four of the previously identified BAC clones, 5C1,22H9,136F8 and 179G5, as well as three new clones, 31H3,107F10 and 157A9 (Figure 2). An insert end fragment, named 31H3R, hybridized to new BAC clones 70F1 and 147G7. The overlapping rice BAC clones 31H3, 107F10,157A9, 70F1 and 147G7 were subsequently shown to contain the DNA sequence corresponding to the RAPD marker SP4B9 that defines one border of the R-gene region (Recombination has occurred between SP4B9 and the PUta gene in one progeny rice line out of the entire mapping population tested). The SP4B9 sequence was shown to reside at this location in the BAC contig using specific PCR primers GB24 and GB25 (SEQ ID NO:32 and 33) produced by subcloning and sequencing the linked SP4B9 PCR fragment from an agarose gel. This result defined the left border in the BAC contig of Figure 2, indicating that the R-gene is located to the right in this figure.<br>
GB24     S"-AGGCGTCTTCAGTTTTGTAATA-S"	(SEQ ID NO:32)<br>
GB25     S"-AGGCGTCTTCCGGAAAGCAGCG-S"	(SEQ ID NO:33)<br>
Walking to the Right: The border fragment from the opposite end of BAC clone 179G5, called G179G5L, hybridized to itself on the BAC library filters as well as to new BAC clones 34H9, 50E12,108E4 and 158C2. The BAC end fragment 158C2L hybridized to 158C2 as well as new BAC clones 35B3,41F2 and 77D8. Because both the right and left border fragments of 77D8 contain repetitive DNA sequences, a single copy fragment internal to the BAC insert, called 77D8-12, was cloned from an agarose gel and used to probe the BAC library. This fragment identified 77D8 and 41F2, as well as new BAC clones 18D1, 6D5 and 142E8.<br>
Random Genomic Sequencing of BAC Inserts to Identify R-Gene Candidates. Overlapping BAC clones 31H3,179G5,34H9, 77D8 and 142E8 were nebulized by high pressure treatment with nitrogen gas (20 psi) in a reaction chamber (Nebulization tube, order #4207, IPI Medical Products, Chicago) for 90 seconds. The ends of the sheared and reprecipitated DNA fragments were blunt-ended by high fidelity DNA polymerase (PFU, Stratagene) -mediated fill-in reaction. DNA fragments in the size range of 1000-2000 bp were isolated from agarose gels using the QIAEXII Gel Extraction Kit (Qiagen, Chatsworth, CA)<br><br>
                                                                    35<br><br>
sutxioned into the Smal site of PUCI, ana transiormed into DHIOB by electroporation. Clones containing BAC DNA inserts were sequenced in one direction using the universal reverse primer. Approximately 300 to 400 clones were sequenced for each BAC in order to have a reasonable chance of obtaining sequence from and identifying all genes contained in the set of clones. Similarity searches were done using the BlastX program (Gish, W. and States, D.J. (1993) Nature Genetics 3:266-272). For convenience, the P-value (probability) of observing a match of a sequence to a sequence contained in the searched databases merely by chance ;is calculated by BLAST is reported herein as "pLog" values, which represent the negative of the logarithm of the reported P-value. Accordingly, the greater the pLog value, the greater the likelihood that the sequence and the BLAST "hit" represent similar proteins. The clones were divided into three groups: those with highly significant hits (pLog&gt;10); those with weakly significant hits (1
PRG2 was identified by three sequences falling in the weakly significant hits category. For example, the amino acid sequence encoded by the 586 bp DNA fragment sequenced in ba. .42e8.pk0001.fl2 (SEQ ID NO:34; "fl2" in Figure 3) possessed moderate similarity (pLog = 5.80) to the protein PRF (GenBank accession U653391) that is required for R-gene function in tomato (Salmeron et al., 1996, Cell 86:123-133). It also was related [pLog = 5.43) to the.amino acid sequence encoded by the Arabidopsis R-gene RPMJ [X87851; Grant et al., 1995, Science 269:843-846). The 598 bp sequenced from :&gt;acl42e8.pkO001.b7 (SEQ ID NO:35; "b7" in Figure 3) identified the PRF gene and the Arabidopsis myosin heavy chain homolog (U19619), and the 605 bp insert of 3ac 142e8.pk0001 .f8 (SEQ ID NO:36; "f8" in Figure 3) showed a 44 amino acid region with lomology to the P-loop containing regions of RPM1 and PRF. Primers from these genomic lones were made for further genomic sequencing in this region from the resistant rice variety Yashiro-mochi, described below. As the sequencing progressed, additional verlapping subclones were identified from the original set of 239 BAC fragments showing o significant hits (Figure 3).<br>
EXAMPLE 7<br>
Isolation of Genomic and cDNA Clones For the Pi-ta Candidate Gene<br>
The nucleic acid fragments subcloned from BAC 142E8, described.in<br>
Example-6 as comprising the putative Pi-ta locus, were accurately sequenced in order<br>
to design the strategy for cloning the full length gene. In particular, the rice genomic<br>
fragment in bacl42e8.pk0001.f8 was 2 kb in size and contained 1255 bp of promoter<br>
36<br><br>
sequence before the initiation codon (Figure 3). Southern analyses of Yashiro-mochi<br>
genomic DNA and BAC 142E8 using agarose gel purified rice insert fragments from<br>
clones bacl42e8.pk0001.b7 and bacl42e8.pk0001.fl2 showed that BAC RB142E8<br>
contained a single copy of PRG2, This analysis also identified a 5.3 kb BamHI<br>
fragment from 142E8 that contained the remainder of the gene plus approximately<br>
2 kb of 3" untranslated region. This 5.3 kb BamBl fragment was subcloned from the<br>
BAC DNA into the vector pBluescriptIISK+ (Stratagene) and was designated Clone<br>
#7 (Figure 3). The entire gene was assembled by digesting both bacl42e8.pk0001 ,f8<br>
and .Clone #7 with EcoBl, and ligating the 5.3 kb .gcoRI fragment from Clone #7 in place of the small EcoRI fragment in bacl42e8.pko66i.f8 (Figure 4). The resulting"<br>
vector, pCB 1641, contained a 7.5 kb insert of rice DNA in the 2.7 kb Vector, This<br>
rice clone included the full length PRG2 coding sequence along with 1255 bp of<br>
promoter sequence and approximately 2 kb of 3" untranslated sequence. The DNA<br>
sequence of 5757 bp of the genomic clone (SEQ ID NO:l, including the entire 5"  .<br>
sequence, the coding sequence and intron, and 246 252 bp past the termination codon)<br>
in pCB1641 was determined for both strands by the primer walk method, with the use<br>
Df DyeDeoxy terminator cycle sequencing and a niodel-377A DNA sequencing<br>
system (Applied Biosystems, Foster City, CA). Relative locations of some of the<br>
jrimers used are diagramed in Figure 5 (SEQ ID NOS:37-64).<br><br>
F8-2:	5"-CCATCGGTGACCATGGG-3"	(SEQ ID NO:37)<br>
1641-1:	5*-CCCTCTCCGTTGTTCCCATGG-3"	(SEQ ID NO:38)<br>
F8-4:	5"-GGGTTGGAGAATGCATGG-3"	(SEQ ID NO:39)<br>
1641-2:	5"-CCATGCATTCTCCAACCC-3"	(SEQ ID NO:40)<br>
F8-3:	5"-CATGGATGGAGACCTCTGC-3*	(SE"Q ID NO:41)<br>
F8-5:	5"-TCCTCAGAGGCGATCTCC-3"	(SEQ ID NO:42)<br>
F8-1:	5"-GAACAGGGTGACCTCGTCG-3"	(SEQ ID NO:43)<br>
F12-1:	S"-GTGGCTTCCATTGTTGGATC-S"	(SEQ ID NO:44)<br>
F12-5:	5"-CGAACGGCGCATCCAACC-3"	(SEQ ID NO:45)<br>
F12-6:	S"-GTGTCCTCTATTAGTAAATAC-S"	(SEQ ID NO:46)<br>
F12-7:	S"-GTATTTACTAATAGAGGACAC-S"	(SEQ ID NO:47).<br>
37<br><br>
1641-3:	5"-GGGCCTCCCTTGTTCGG-3"	(SEQ ID NO:48)<br>
1641-4:	5"-CCGAACAAGGGAGGCCC-3"	(SEQ ID NO.-49)<br>
F12-4:	5"-CCTACAGATCTGTAGCCAGC-3"	(SEQ ID NO:50)<br>
F12-3:	5"-GCTGGCTACAGATCTGTAGG-3"	(SEQIDNO:51)<br>
FP.-2:	5"- GCTGL CTACAGATCTGTAGG -3"	(SEQ ID NO:52)<br>
B7-1:	5"- GCTGGCTACAGATCTGTAGG -3"	(SEQ ID NO:53)<br>
B7-3:	S"-GTGATTCCATTGCTGCATTCC-S"	(SEQIDNO:54)<br>
B7-4:	5"-CATGTGTAGTGCACCACATGG-3"	(SEQ ID NO:55)<br>
B7-2:	5"-CTGTCGTCTGAGAATGATCC-3"	(SEQ ID NO:56)<br>
C10-1:	5"-CCAAGGACTACAACACTTGC-3*	(SEQ ID NO:57)<br>
173C12-5:	5"-GCATCCCCAAATGGACTGG-3"	(SEQ ID NO:58)<br>
1641-5:	S"-CCAGTCCATTrGGGGATGC-S"	(SEQ ID NO:59)<br>
173-6:	5"-CAAGCAT(XGACGCCGAGC-3"	(SEQ ID NO:60)<br>
173-7:	5"-TTGTATATCAACCATAAGAGTGC-3"	(SEQ ID N0:61)<br>
1641-6:	5*-GrrC7TTGATCCAAGTGTTAGG-3*	(SEQ ID NO:62)<br>
GB46:        5"-CATTAAAGTCGACCTCAAACAATCATCAAGTCAGGT-3"<br>
(SEQ ID NO:63)<br>
GB47:        S"-AATGCAGAATTCACAACACCACTAGCAGGTTTGO"<br>
(SEQIDNO.-64)<br>
DNA sequence analysis of the rice genomic fragment (SEQ ID N0:1; 5757 mcleotides) subcloned in pCB1641 identified an open reading frame of 2784 nucleotides &gt;eginning at nucleotide 1256 and interrupted by one 1463 bp intron between nucleotides !199 and 3663. The coding sequence corresponds to a deduced polypeptide with 928 amino icids (Figure 6). The putative protein has a relative molecular weight of 105 kDa and a pi of r.05. Four amino acid sequences matching theN-glycosylation consensus [NX(S/T) with X<br>
38<br><br>
not being P] (Kornfeld et aL 1985, Ann. Rev. Biochem. 54:631) are present in the PRG2 protein.<br>
To determine MPRG2 is present in a single copy or in multiple copies in the rice genome, blots of total genomic rice DNA were probed using nucleic acid fragments encoding either the amino-terminal portion of the gene (639 bp fragment amplified by primers F8-5 and F12-5) or the carboxy-terminal portion (nucleotides 4477 to 5503, amplified by primers GB47 and GB46). Blots were washed at 50° to 55° in 2 x SSPE with 0.1% SDS, for low stringency hybridizations, and at 65° in 0.1 x SSPE with 0.1% SDS for high stringency hybridization. Even under the high stringency conditions, the N-terminal probe (nucleic acid fragment comprising bases 1401 to 2040) identified a single copy against a general smear of background hybridization. Probing the very same blots with the C-terminal nucleic acid fragment (subcloned as pCBl 645, described below) identified one copy with no background smear. It was concluded that PRG2 is a single copy gene, but that there may be related genes in the ricgenome.<br>
The single copy C-terminal probe was subcloned for use as a hybridization probe. Primer GB47 was synthesized based on the sequence of this region from nucleotides 4465 through 4497, except that residues between 4471 and 4476 were modified to incorporate an EcoRI restriction site. Likewise, primer GB46 was the reverse-complement of nucleotides 5519 through 5484, except that the residues corresponding to nucleotides 5512 and 5506 were modified to incorporate a Sail restriction site. Primers GB46 and GB47 were used to amplify the gene fragment. After purification *rom an agarose gel as described in Example 6, and following digestion with the indicated enzymes, the nucleic acid fragment was subcloned into the EcoBlfSaE sites of the vector pBD-GAL4 Cam (Stratagene) to produce plasmid pCB 1645 (Figure 4B). The subcloned fragment was sequenced to confirm its structure.<br>
A full-length Pi-ta cDNA fragment was cloned using reverse transcriptase (RT) PCR and subcloning. The approach involved isolating mRNA from transgenic Nipponbare line 27-4-8-1 which contained the transgene comprising a genomic Pi-ta nucleic acid fragment operabiy linked to the CaMV 35S promoter (Example 10, expression construct 3) and which was shown on a northern blot to overexpress Pi-ta. First strand cDNA was synthesized using the isolated mRNA fraction as template and the oligonucleotide GB67 (SEQ ID NO:65) as primer.<br>
GB67:     S"-CCATTAAGCTTGGTTTCAAACAATC-S"    (SEQIDNO:65)<br>
A partial Pi-ta cDNA (2.1 kb) was amplified from first strand cDNA using primers F12-1 (SEQ ID NO:44) and GB67 (SEQ ID NO:65). It was then cloned into pSLI 180 (Pharmacia) using the BamHI (restriction site present in the Pi-ta nucleic acid fragment) and HINDlll<br><br><br><br>
(restriction site present in GB67 sequence) cloning sites. To obtain a full-length synthetic cDNA, a 706 bp Ncol-BamHl fragment containing the 5" end of the Pi-ta coding sequence was isolated from pCBl649 and cloned into the Ncol-BamHl site upstream of the 2090 bp BamUl-Hindlll partial Pi-ta cDNA fragment to create a full-length promoter-less Pi-ta cDNA. DNA sequence analysis determined that there was a 2 bp deletion present at codon 796 (probably a PCR artifact) resulting in a frameshift mutation that would have truncated the predicted Pi-ta protein by 119 amino acids. This was corrected by replacing a 1400 bp Sphl-BgRl fragment with the ctiresponding fragment from pCBl 649 which contained the correct sequence, to create pCB1906. DNA sequence analysis also determined that the predicted intron was precisely spliced in this synthetic cDNA. A native Pi-ta promoter fragment (2425 bp) was added by cloning a 3173 bp £coRl fragment from pCB1649 into the EcdRJ sites of pCB1906, resulting in plasmid pCB1926 which contained the final Pi-ta cDNA construct (Figure 7C).<br>
»   EXAMPLE 8 Cloning of the Pi-ta Candidate Gene From Susceptible Rice Variety Tsuyuake and Sequence Analysis for Comparison of Resistant and Sensitive Alleles<br>
A rice genomic library was constructed in bacteriophage lambda using DNA from the susceptible variety Tsuyuake (Sambrook). Genomic DNA was prepared using a procedure described by Dellaporta et al., (1983) Plant Mol. Biology Reporter 7:19-27. The genomic DNA was subjected to partial digestion with the restriction enzyme Tsp509l (cuts at 5"-AATT-3"). DNA fragments in the Size range of 7 to 1 v. :b were purified from an agarose gel and subcloned into the EcoRI site of vector XZapII (Stratagene). Screening the Tsuyuake library by hybridization with the DNA insert of pCB1641 identified 8 overlapping clones with sequences related to PRG2. Two clones that overlapped by 1 kb and included the entire coding sequence, T2-9 and T2-2, were sequenced using the primers in Figure 5. The sequence of the cloned genomic fragments from Tsuyuake were identical to sequences obtained from PCR-amplified genomic DNA of the susceptible DH rice line YT16. Comparison of Resistant and Sensitive Alleles From Multiple Rice Lines, The functional Pi-ta coding sequence from resistant Yashiro-mochi rice (contained in SEQ ID NOS: 1 and 68) differs by 7 nucleotides from the sequence of the nonfunctional coding sequence from sensitive Tsuyuake rice (contained in SEQ ID NO:3). The 7 basepair substitutions in the coding region result in 5 amino acid differences between the two proteins (Figure 6). Six additional basepair changes occur in the intron sequences. Most of the nucleotide differences in the coding region occur in the region preceding the P-loop motif. Both genes encode proteins with identical NBS motifs and with the 4 potential N-glycosylation sites at positions 339,556,654 and 838.<br>
As determined by RNA gel blot analysis with polyA+-RNA, both the Pi-ta gene from Yashiro-mochi and the susceptible allele from Tsuyuake are transcribed with an estimated<br>
40<br><br>
transcript size of 3.3 kb. Further, RNA blot analysis demonstrates that both alleles are constitutively expressed. This analysis was done by probing polyA+-RNA blots with the C-terminal Pi-ta fragment that had been cloned in pCB1645. PolyA+-mRNA was prepared from 2 week old leaves of resistant rice variety YT14, or from susceptible rice variety YT16 by the method of Perry and Francki (1992, J. Gen. Virol. 73:2105-2114). RNA gel blot analysis was performed according to the method of Ausubel et al. (1987 Protocols in Molecular Biology, New York, John Wiley and Sons).<br>
41<br>
The sequences of genomic clones of Pi-ta nucleic acid fragments from additional varieties of rice were obtained in order to see if the differences in nucleic acid sequences were conserved in resistant versus susceptible forms of the Pi-ta gene. Using primers listed in Figure 5, PCR products of the Pi-ta gene were sequenced from the resistant varieties Tadukan and Kl. These varieties are believed to have acquired a Pi-ta resistance gene in a manner independent of that which resulted in the presence of the gene in Yashiro-mochi. The same PCR primers were used to^eguence the nucleic acid fragment from an additional susceptible Japanese rice variety, Nipponbare, and from the unrelated susceptible variety, Sariceltik. To determine if the sequence differences were due to a difference between japonica versus indica alleles, the nucleic acid fragment was sequenced from the susceptible indica variety CI01A51. The result shows a strong correlation between nucleic acid sequence and resistance to strains of the rice blast pathogen containing A VR-Pita; over the region sequenced, Tadukan and Kl have an identical sequence to Pi-ta from Yashiro-mochi (Table 4), and the gene." from Nipponbare and Sariceltik have the identical sequence to the susceptible pi-ta gene from Tsuyuake. The nucleotide sequence of the indica CI 01A51 allele suggests that a single amino acid difference, alanine versus serine at position 918, determines the specificity of Pi-ta function.<br><br><br>
EXAMPLE 9 Identification ofPi-ta Function bv Transient Expression in Rice<br>
As is standard practice to those skilled in the art, high velocity biolistic bombardment of plant tissue with particles coated with recombinant expression constructs of interest results in transient expression of the nucleic acid fragments from the introduced plasmids. Function of disease resistance genes can be demonstrated in transient leaf bombardment experiments using reporter gene expression to assay foi triggering of the hypersensitive cell death resistance response. Such assays have demonstrated utility for identifying R-genes for bacterial pathogens because the corresponding AVR-gene required for triggering resistance responses can be supplied by uniform infiltration of the plant tissue with the bacterial pathogen (Mindnnos et ah, 1994, Cell 78:1089-1099). Uniform pathogen infiltration is not possible with fungal pathogens such as M. grisea that directly penetrate through the plant cuticle and produce localized lesions attthe point of penetration.<br>
In order to overcome the obstacle to uniform incorporation of the fungal AVR-gene within the plant tissue, introduction of an AVR-Pita expression construct was tested by co-bombardment along with the GUS reporter gene. In particular, a construct was engineered that expresses the putative mature protease (AVR-Pita J75) under control of the 35S promoter for constitutive expression in plant cells. First, pML63 (Figure 7B) was cut with Ncol, and blunted with SI nuclease. The GUS nucleic acid fragment was then removed by further digestion with Aspll 8. A linker fragment generated with primers MJL  J2 (SEQ ID:69) and ML133 (SEQ ID: 70) was cloned into the blunted Ncol and Aspll 8 sites to produce pML 135.<br>
ML132:	5"-CACGTGGAATTCCCCGGGG-3"	(SEQIDNO-.69)<br>
ML133:	5"-GTACCCCCGGGGAATTCCACGTG-3"	(SEQIDNO:70)<br>
pML135 was digested with Pmll m&amp;Asp7\%, and the Adhl-6 intron amplified by PCR from maize genomic DNA using primers ML111 (SEQ ID:71) and ML135 (SEQ ID:72) was cloned into pMLl35 to create pML 141.<br>
ML 111:	5"-CCCGGGGAATTCCTGCAGAAGGTG<br>
CAAGGATTGCTGGAGCG-3,	(SEQ ID NO:71)<br>
ML 135:	5"-TTTAAAGGTACCCCATGGCACGTGCCGGC<br>
TTGTTGTGGTCTTTTGGGTTCAC-3,	(SEQ ID NO:72)<br>
42<br><br>
pML142 was finally created by digested pML141 with BamHl and Ncol and the 1.9 kb fragment containing the 35S/Adhl-6 sequence frompMLUl was cloned into the BarnHI/Ncol sites of pML63.<br>
Plasmid pAVR3 contained nucleotides 139-672 of the A VR-Pita nucleic acid fragment (SEQ ID NO:5) encoding the predicted mature protease plus one additional N-terminal amino acid (AVR-Pita! 177, beginning with Ile-47 of the preproprotein) and a start codon met fused to the 35S/Adhl-6 promoter in vector pML142. The AVR-Pita nucleic acid fragment (SEQ ID NO:5) was amplified by PCR from A VR-Pita cDNA using primers A V1 (SEQ ID NO:73) and AV3 (SEQ ID NO:74), digested with Pmtl and Kpnl, blunted with Klenow polymerase and cloned into pML142 that had also been cut with PmR and Kpnl and blunted with Klenow polymerase, resulting in pAVR3.<br>
AVI:   5"-GCCGGCACGTGCCATGATTGAACGCTATTCCCAATG-3"<br>
	(SEQIDNO:73)<br>
AV3:   5"-GCCGGGATCCCCCTCTATTGnAGATTGAC-3"      (SEQIDNO:74)<br>
The coding sequence for the predicted mature protease (beginning with Glu-48 of the preproprotein) was obtained by PCR-amplification from the A VR-Pita nucleic acid fragment (SEQ ID NO:5) in plasmid pAVR3 using oligonucleotides YL30 contaimng an in-frame Nol site (SEQ ID NO:66) and YL37 (SEQ ID NO:67). The PCR fragment was cloned Ncol/Kpnl into pML142 resulting in vector pCB1947. This was done to eliminate the Ile-47 codon (art) in pAVR3 and generate AVR-Pita]76. Additional details are described in the Legend to Figure 7A.<br>
YL30: S"-ACAACAAGCCGGCACGTGCCATGGAACGCT^"    (SEQ ID NO:66)<br>
YL37: 5"-TCCTTCTTTAGGTACCGCTCTCTC-3"	(SEQIDNO:67)<br>
To determine if expression of the A VR-Pita176 construct triggers Pi-ta mediated defense responses when introduced into P/-/a-containing plant cells, seedlings from Pi-ta-plants (Yashiro-mochi and YT14) and plants that lack Pi-ta (Nipponbare and YT16) were co-bombarded with the 35S/Adhl-6::A VR-PitaJ76 gene construct and the 35S::GUS reporter gene (Figure 7). Seeds were germinated on leaf assay media: 1/2 strength MS medium (Murashige and Skoog, 1962, Physiol. Plant. 15:473-497) supplemented with 100 mg casein hydrolysate and 0.5% agarose for a week in an incubator at 25°C for 48 hours in 12 hr photoperiod with a 100 u£nr2s_1 of cool, white light Two-leaf seedlings were excised from the agar medium using a surgical razor and placed in a petri dish containing a prewetted filter<br>
43<br><br>
paper. Plantlets were labeled at the base with a permanent marker for identification. Biolistic bombardment of the seedlings was performed using Bio-Rad PDS-1000/He apparatus and 1150-psi rupture disks. Gold particles, (0.6 um diameter), were prepared according to the instructions provided by the manufacturer. For each cobombardment, 1 ug of gold particles was coated with 1.5 ug of 35S/GUS and 1 ug of other plasmids. After bombardment, seedlings were maintained at 25°C for 48 hours in Petri dishes containing prewetted filter paper. Leaves were cleared in 70% ethanol and histochemically assayed for P-Glucuronidase (GUS) activity using 5-bromo-4-chloro-3-indolyl glucuroude (X-gluc) as a substrate (Jefferson, 1987, Plant Mol. Bio. Rep. 5:387-405).<br>
The functional AVR-Pita JJ^ expression construct mediated a striking decrease in GUS expression (suggesting cell death prior to expression), when co-bombarded with the GUS construct into leaves containing the endogenous Pi-ta gene. As predicted, the A VR-Pitaj76 expression construct showed significantly higher activity than a similar expression construct containing the ful| length/4 VR-Pita coding sequence. Early experiments revealed a hypersensitive-like response upon bombardment of leaves of intact Yashiro-mochi seedlings, based on observation of typical Type I necrotic lesions and a decrease in GUS expression, but necrotic lesion formation was not as reproducible as the decrease in GUS activity. Mutant forms of AVR-Pita 176 that fail to confer avirulerice also fail to decrease GUS expression. Nor is the effect seen in rice varieties that lack an endogenous Pi-ta gene. Thus, expression of the AVR-Pita coding "sequence triggers the rice defense response when introduced directly into rice leaves containing the endogenous Pi-ta gene. PRG2 functions in place of the endogenous Pi-ta gene in the transient expression assav. Resistant (YT14) and susceptible (YT16) rice leaves were co-bombarded with the A VR-Pham (pCB1947; Figure 7A), GUS (pML63; Figure 7B) and the Pi-ta cDNA (pCB1926; Figure 7C) expression constructs in order to see if exogenously supplied PRG2 can complement endogenous Pi-ta function. Typical leaves from this assay are shown in Figure 8. Decreased levels of GUS expression were observed in resistant (YT14) leaves whether or not the Pi-ta expression construct was co-bombarded with A VR-Pita 176 and GUS. In susceptible (YT16) rice leaves, however, a marked decrease in GUS expression was noted only if the Pi-ta expression construct was co-bombarded with A VR-Pita176, and GUS. Thus, PRG2 functions in place of the endogenous Pi-ta gene in recognizing AVR-Pita 176and inducing localized cell death. Control leaves of susceptible YT16 seedlings shot with A VR-Pita j y6 and GUS only had high levels of GUS expression. The sensitive allele from variety C101A51, which differs by a single amino acid from Pi-ta, did not result in a decrease in GUS expression when co-bombarded with AVR-Pita /;,$ and GUS, indicating that specificity is maintained in this assay. These experiments are consistent with a conclusion that PRG2 is, in fact, a Pi-ta gene.<br><br>
EXAMPLE 10 Production of Transgenic Rice Containine the Putative R-gene As is standard practice in molecular plant pathology, identification of R-gene function involves stable introduction (i.e., transformation) of a transgene into susceptible plants, and the demonstration that this transgene confers a resistance phenotype to the host plant that was previously characterized as susceptible. One method for transforming DNA into eukaryotic cells of higher plants that is available to those skilled in the art is high-velocity ballistic bombardment using metal particles coated with the nucleic acid constructs of interest (see Klein et al. Nature (1987) (London) 327:70-73, and see U.S. Patent No. 4,945,050). A Biolistic PDS-1000/He (BioRAD Laboratories, Hercules, CA) was used for these experiments. The particle bombardment technique was used to transform the susceptible rice variety Nipponbare with four different expression constructs containing the putative Pi-ta nucleic acid fragments:<br>
1)	pCB1641 - plasmid containing thet7 kb Pi-ta nucleic acid fragment including a 1255 bp native promoter region (Figure 4A, SEQ ID NO:l).<br>
2)	pCB1649 - plasmid containing an 8.4 kb Pi-ta nucleic acid fragment including a 3 kb native promoter. This vector is identical to pCB1641 except that it contains a longer promoter region. To make pCBl649, pCB1641 was cut with HindXWSacll to excise a<br>
1.58 kb fragment. This was replaced by a 4.98 kb HindHl/Sadl subclone of BAC142E8 to create pCB1649. This cloning step essentially replaced the 1255 bp promoter in pCB1641 with a longer 4656 bp promoter fragment.<br>
3)	35S/Pi-ta - plasmid containing a 5.36 kb Pi-ta nucleic acid fragment operably linked to a<br>
1.9 kb modified 35S promoter fragment. This vector was constructed by cloning a<br>
5.36 kb NcoVKpril fragment from pCB1641 into the Ncol/Kpril sites of plasmid pML63 which contained the 35S promoter fragment.<br>
4)	pCB1926 - plasmid containing Pi-ta cDNA nucleic acid fragment and a 2425 bp native<br>
promoter region described in Figure 7C, SEQ ID NO:68.<br>
The bacterial hygromycin B phosphotransferase (Hpt II) gene from Streptomyces hygroscopicus that confers resistance to the antibiotic was used as the selectable marker for rice transformation. In the vector was used, pMLl 8, the Hpt II gene was engineered with the 35S promoter from Cauliflower Mosaic Virus and the termination and polyadenylation signals from the octopine synthase gene of Agrobacterium tumefaciens, pML18 is described in WO 97/47731, which was published on December 18,1997, the disclosure of which is hereby incorporated by reference.<br>
Embryogenic callus cultures derived from the scutellum of germinating Nipponbare seeds served as source material for transformation experiments. This material was generated by germinating sterile rice seeds on a callus initiation media (MS salts, Nitsch and Nitsch vitamins, 1.0 mg/12,4-D and 10 yM AgN03) in the dark at 27-28° C. Embryogenic callus<br>
45<br><br>
proliferating from the scutellum of the embryos was the transferred to CM media (N6 salts. Nitsch and Nitsch vitamins, 1 mg/12,4-D, Chu et al., 1985, Sci. Sinica 18: 659-668). Callus cultures were maintained on CM by routine sub-culture at two week intervals and used for transformation within 10 weeks of initiation.<br>
Callus was prepared for transformation by subculturing 0.5-1.0 mm pieces approximately 1 mm apart, arranged in a circular area of about 4 cm in diameter, in the center of a circle of Whatman #541 paper placed on CM media. The plates with callus were incubated in the dark at 27-28° C for 3-5 days. Prior to bombardment, the filters with callus were transferred to CM supplemented with 0.25 M mannitol and 0.25 M sorbitol for 3 hr. in the dark. The petri dish lids were then left ajar for 20-45 minutes in a sterile hood to allow moisture on tissue to dissipate.<br>
Circular plasmid DNA from two different plasmids, for example, pMLl 8 containing the selectable marker for rice transformation and pCB1641 containing the putative R-gene, were co-precipitated onto the surface of gold particles. To accomplish this, a total of 10 ug of DNA at a 2:1 ratio of trait:selectable marker DNAs were added to 50 ul aliquot of gold particles that had been resuspended at a concentration of 60 mg ml*1. Calcium chloride (50 u.1 of a 2.5 M solution) and spermidine (20 ul of a 0.1 M solution) were then added to the gold-DNA suspension as the tube was vortexing for 3 min. The gold particles were centrifuged in a microfuge for 1 sec and the supernatant removed. The gold particles were then washed twice with 1 ml of absolute ethanol and then resuspended in 50 ul of absolute ethanol and sonicated (bath sonicator) for one second to disperse the gold particles. The gold suspension was incubated at -70°C for five minutes and sonicated (bath sonicator) if needed to disperse the particles. Six \i\ of the DNA-coated gold particles were then loaded onto mylar macrocarrier disks and the ethanol was allowed to evaporate.<br>
At the end of the drying period, a petri dish containing the tissue was placed in the chamber of the PDS-1000/He. The air in the chamber was then evacuated to a vacuum of 28-29 inches Hg. The macrocarrier was accelerated with a helium shock wave using a rupture membrane that bursts when the He pressure in the shock tube reaches 1080-1100 psi. The tissue was placed approximately 8 cm from the stopping screen and the callus was bombarded two times. Five to seven plates of tissue were bombarded in this way with the DNA-coated gold particles. Following bombardment, the callus tissue was transferred to CM media without supplemental sorbitol or mannitol.<br>
Within 3-5 days after bombardment the callus tissue was transferred to SM media (CM medium containing 50 mg/1 hygromycin). To accomplish this, callus tissue was transferred from plates to sterile 50 ml conical tubes and weighed. Molten top-agar at 40° C was added using 2.5 ml of top agar/100 mg of callus. Callus clumps were broken into fragments of less than 2 mm diameter by repeated dispensing through a 10 ml pipet. Three ml aliquots of the callus suspension were plated onto fresh SM media and the plates were<br>
46<br><br>
incubated in the dark for 4 weeks at 27-28°C. After 4 weeks, transgenic callus events were identified, transferred to fresh SM plates and grown for an additional 2 weeks in the dark at 27-28°C.<br>
Growing callus was transferred to RMl media (MS salts, Nitsch and Nitsch vitamins, 2% sucrose, 3% sorbitol, 0.4% gelrite +50 ppm hyg B) for 2 weeks in the dark at 25°C. After 2 weeks the callus was transferred to RM2 media (MS salts, Nitsch and Nitsch vitamins, 3% sucrose, 0.4% gelrite + 50 ppm hyg B) and placed under cool white light (-40 uEnrV1) with a 12 hr photoperiod at 25°C and 30-40% humidity. After 2-4 weeks in the light, callus began to organize, and form shoots. Shoots were removed from surrounding callus/media and gently transferred to RM3 media (1/2 x MS salts, Nitsch and Nitsch vitamins, 1 % sucrose + 50 ppm hygromycin B) in phytatrays (Sigma Chemical Co., St. Louis, MO) and incubation was continued using the same conditions as described in the previous step.<br>
Plants were transferred from RM3 to 4" pots containing Metro mix 350 after 2-3 weeks, when sufficient root and shoot growth had occurred. Plants were grown using a 12 hr/12 hr light/dark cycle using ~30/18°Cday/night temperature regimen.<br>
EXAMPLE 11 Identification of Pi-ta bv Functional Complementation in Transgenic Rice Molecular Analysis of the Transformants. Allele specific PCR, as well as DNA gel blot analysis, was used to analyze transformants for the presence of R-gene sequences. High-stringency DNA gel blot analysis was performed, as described in Example 7, using the PRG2 DNA insert from pCBl 645 as the hybridization probe. Figure 9 shows a typical result from analysis of transformants produced with the PRG2 isolated nucleic acid fragment in pCB1641. Events ranged from insertion of single copies of the putative Pi-la gene to complex multi-copy insertions.<br>
47<br>
Assay for Gain of the Resistance Phenotvpe. Pathogenicity assays were done as described in Example 2 using the AVR-gene containing pathogen strains 4360-R-62 or 0-137 to test for gained resistance function. Table 5 shows the results from assays performed on primary transformants produced by co-bombardment with pCB1641 and the Hyg&amp; plasmid (pML18).<br><br><br>
For construct pCB164L 7 out of 42 transformants assayed were resistant and 13 out of 42 were intermediate. Intermediate resistance was characterized by uniformly lower numbers of lesions that were also smaller than produced by fungal strains lacking A VR-Pita. For the construct with the PRG2 coding sequence operably linked to the 35S promoter, 4 out of 41 showed intermediate resistance. It was not surprising that 22/42 primary transformants containing the Pi-ta transgene (pCB1641) were susceptible, this was likely a result of three different problems affecting biolistic plant transformation. Many integration events contained multiple copies of the Pi-ta transgene (Figure 9) including some that may have been fragmented. This can result in gene silencing events. Secondly in some integration events the transgene can be inserted into a chromosomal region that is unfavourable for gene expression. A third possible factor may have been gene inactivation by small deletions or frameshift mutations in the transgene that could not be detected on a Southern blot.<br>
R1 seedlings derived from a single RO transformant were divided into two sets and inoculated with either the avirulent sta^n 0-137 or a virulent mutant derived from 0-137 (CP3337) in order to determine if the resistance phenotype was stable through meiosis and if the resistance had specificity characteristic of the Pi-ta resistance gene. For example, DKA gel blot analysis (Figure 10) of Rl seedling family 22-6-10-1 shows stable inheritance of the transgene correlating with Pi-ta- specific resistance. Results of the Rl analysis demonstrate that PRG2 confers resistance with specificity characteristic of the Pi-ta gene and that this resistance is stably inherited. Thus, it was concluded that SEQ ID NO:l corresponds to the Pi-ta resistance gene.<br>
EXAMPLE 12 Characterization of the Pi-ta Resistance Gene Product<br>
The encoded polypeptide of a candidate Pi-b gene sequence (accession ABO 13448), now available in Genbank, displays 30% identity (49% similarity) with a subfragment corresponding substantially to amino acid residues 200 to 680 of the Pi-ta polypeptide set forth in SEQ ID NO:2. Using BESTFIT (Smith and Waterman, 1981, Advances in Applied Mathematics 2:482-489). the translated Pi-ta polypeptide sequence from amino acid residues 208 to 527 was found to show 30.3% identity (44.3% similarity) to the amino acid sequence of the RPM1 protein (Grant et al., 1995, Science 269:843-846). The same region shows 27.4% identity (38.8% similarity) to the tomato PRF protein, which is implicated in disease resistance signal transduction pathways (Salmeron et al., 1996, Cell 86:123-133), and 24.5% identity (38%similarity) fa the product of the Arabidopsis R-gene RPS2 (Bent et al., 1994, Science 265:1856-1860; Mindrinos et al., 1994, Cell 78:1089-1099). No significant similarity is seen to the leucine-rich regions of any of these previously cloned R-gene products.<br>
A prominent feature of the Pi-ta gene product is the NBS motif characteristic of numerous ATP- and GTP-binding proteins (Traut, 1994, Eur. J. Biochem. 229:9-19). Amino<br>
48<br><br>
acids 236 to 244 match the generalized consensus GXGXXG(R/k)V for the phosphate<br>
binding (P) loop. The probable companion kinase domains are located at amino acids<br>
314-323 (kinase domain 2a) and around amino acids 342 to 353 (kinase domain 3a). A<br>
conserved internal hydrophobic domain found in the two subclasses of cytoplasmic R-gene<br>
products (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv. Plant Pathol. 24:89-167) is<br>
partially present between amino acids 407 to 415. However, key features of the two R-gene<br>
subclasses are not found in Pi-ta. First, the Pi-la gene product has a unique amino terminus,<br>
lacking either the potential leucine zipper motif of the RPS2 gene-product subfamily (Bent<br>
et al., 1994, Science 265:1856-1860; Mindrinos et al., 1994, Cell 78:1089-1099) or the<br>
Toll/lnterleukin-1 receptor homology encoded by the N gene subfamily (Whitman et al.,<br>
1994, Cell 78:1101-1115). Perhaps most importantly, the carboxy terminal portion of the<br>
Pi-ta gene product is leucine rich, but it does not fit any consensus sequence for leucine-rich<br>
repeats reported for R-gene products (Jones and Jones, 1997, Adv. Bot. Res. Incorp. Adv.<br>
Plant Pathol. 24:89-167).	<br><br>
INDICATIONS RELATING TO A DEPOSITED MICROORGANISM<br>
(PCTRuk- 13ow)<br>
A.  The indications rfcadc below relate to the microorganism referred to in the description<br><br>
on page<br><br>
, line<br><br>
13<br><br><br><br>
.  IDENTIFICATION OF DEPOSIT<br>
Narr z of depositary ii stitution<br>
AMERICAN TYPE  CULTURE COLLECTION<br><br>
Further deposits are identified on an additional sheet    xj<br><br>
Address of depositary institution (including postal code and country)<br>
10801 University Blvd. Manassas, Virginia 20110-2209 USA<br><br>
Date of deposit<br><br>
Accession Number<br><br><br><br>
03 August 1998<br><br>
ATCC  74457<br><br>
C. ADDITIONAL INDICATIONS (leave blank if not applicable)	This information is continued on an additional sheet   f~]<br>
In respect of those designations in which a European patent is sought, a sample of the deposited microorganism will be made available until the publication of the mention of the grant of the European patent or until "he date on which the appi "■ nation has been refused or withdrawn or is v iemed to be withdrawn, only by the issue of such a sample to an expert nominated by the person requesting the sample. (Rule"28(4) EPC)<br>
D. DESIGNATED STATES FOR WHICH INDICATIONS ARE MADE (if the indications are not for alt designated States)<br>
E. SEPARATE FURNISHING OF INDICATIONS (leave blank if not applicable)<br>
The indications listed below will be submitted to the International Bureau later (specify the genera! nature of the indications e.g.. "Accession Number if Deposit")<br><br>
For receiving Office use only<br><br>
For International Bureau use oulv<br><br><br><br>
[    | This sheet was received with the international application<br><br>
|    | This sheet was received by the Internationa! Bureau on:<br><br><br><br>
Authorized officer<br><br>
Authorized officer<br><br><br><br>
Form PCT/RO/I3J (July 1992)<br><br>
50<br><br>
INDICATIONS RELATING TO A DEPOSITED MICROORGANISM<br>
(PCTRulc \2bis)<br>
A. The indications made below relate to the microorganism referred to in the description<br>
9		    .line	14	<br>
on page<br><br>
B. IDENTIFICATION OF DEPOSIT<br><br>
Further deposits are identified on an additional sheet  Pj<br><br>
Name of depositary institution<br>
AMERICAN TYPE CULTURE COLLECTION<br>
Address of depositary institution (includingpostal eodt and country)<br>
10801 University Blvd. Manassas, Virginia 20110-2209 USA<br><br>
Date of deposit<br><br>
08 June  1999<br><br>
Accession Number<br><br>
PTA-191<br><br>
C. ADDITIONAL INDICATIONS (Itavt blank if not applicable)	This information is continued on an additional sheet   Q]<br>
In respect of those designations in which a European patent is sought, a sample of the deposited microorganism will be made available until the publication of the mention of the grant of the European patent or until the date on which the application has been refused or withdrawn or. is deemed to be withdrawn, only by the issue of such a sample to an expert nominated by the person requesting the sample. (Rule 28(A) EPC)<br>
D. DESIGNATED STATES FOR WHICH INDICATIONS ARE MADE (if the indications art not for all designated States)<br>
E. SEPARATE FURNISHING OF INDICATIONS (leave blank if not applicable)<br>
The indications listed below will be submitted to the International Bureau bter (specify the general nature of the indications e.g., "Accession Number of Deposit")<br><br>
For International Bureau use only<br><br><br><br>
|"~) This sheet was received with the international applicatio <br>
For receiving Office use only<br>
n<br><br>
fl This sheet was received by the International Bureau on:<br><br><br><br>
Authorized officer<br><br>
Authorized officer<br><br><br><br>
Form PCT/RO/134 (July 1992)<br><br>
51<br><br>
WE CLAIM:<br>
1.	A chimeric gene comprising a nucleic acid fragment which confers a Pi-ta resistance gene mediated defense response against diseases caused by fungal pathogens comprising an AVR-Pita gene wherein said nucleic acid fragment encodes a polypeptide having a sequence identity of at least 90% when compared to a polypeptide sequence as set forth in SEQ ID NO.2, operably linked to regulatory sequences.<br>
2.	The chimeric gene as claimed in claim 1, wherein said nucleic acid fragment confers resistance to the rice blast fungus.<br>
3.	The chimeric gene as claimed in claim 1, wherein said nucleic acid fragment confers resistance to a fungal pathogen comprising an AVR-Pita gene wherein the AVR-Pita gene encodes a polypeptide having a sequence identity of at least 80% when compared to a polypeptide sequence as set forth in SEQ ID NOs: 6 or 8.<br>
4.	A method of conferring a resistance gene-meditated defense response in plant, comprising transforming the plants with a recombinant expression construct which comprises (1) the nucleic acid fragment of the chimeric gene as claimed in claim 1 and (2) and AVR-Pita isolated nucleic acid fragment wherein the AVR-Pita isolated nucleic acid fragment encodes a polypeptide having a sequence identity of at least 80% when compared to a polypeptide sequence as set forth in SEQ ID NOs: 6 or 8, wherein said nucleic acid fragments are operably linked to regulatory sequences and further wherein said construct confers a resistance gene-meditated defense response in plants against diseases caused by a fungal pathogen at the site of infection.<br>
5.	The method as claimed in claim 4, wherein resistance to rice blast fungus is conferred.<br>
6.	The method as claimed in claim 4, wherein the plant is a monocot.<br>
52<br><br>
7.       The method as claimed in claim 4, wherein the plant is a cereal.<br>
8.	The method as claimed in claim 7, wherein the cereal is selected from the group consisting of rice, wheat, barley, corn, fingef millet and pearl millet.<br>
9.	The method as claimed in claim 4, wherein the plant is a natural host for Magnaporthe grisea.<br>
10.	A recombinant expression construct which confers a resistance gene-mediated defense response in plants against diseases caused by a fungal pathogen at the site of infection which comprises (1), the nucleic acid fragment of the chimeric gene as claimed in claim 1 and (2) an AVR-Pita isolated nucleic acid fragment wherein the AVR-Pita isolated nucleic acid fragment encodes a polypeptide having a sequence identity of at least 80% when compared to a polypeptide sequence as set fourth in SEQ ID NOs: 6 or 8, wherein said nucleic acid fragments are operably linked to regulatory sequences.<br>
11.	The recombinant expression construct as claimed in claim 10, wherein said construct confers resistance to rice blast fungus.<br>
12.	A chimeric gene comprising a nucleic acid fragment which confers a Pi-ta resistance gene meditated defense response against diseases caused by fungal pathogens comprising an AVR-Pita gene, wherein said nucleic acid fragment has a sequence identity of at least 90% when compared to the nucleotide sequence set forth in SEQ ID NOs: 1 or 68.<br>
,13.      The chimeric gene as claimed in claim 12, wherein the sequence identity is at least 95%.<br>
Dated	this    17th   day    of       January,      2001.<br><br>
[DEEPAK KUMAR]<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPLIANT[S]<br>
53</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWFic3RyYWN0KDExLTA5LTIwMDcpLmRvYw==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-abstract(11-09-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWFic3RyYWN0KDExLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-abstract(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWNhbmNlbGxlZCBwYWdlcygxMS0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-cancelled pages(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWNsYWltcyhncmFudGVkKS0oMTEtMDktMjAwNykuZG9j" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-claims(granted)-(11-09-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWNsYWltcyhncmFudGVkKS0oMTEtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-claims(granted)-(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWNvcnJlc3BvbmRlbmNlKDAxLTExLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-correspondence(01-11-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWNvcnJlc3BvbmRlbmNlKGlwbyktKDExLTA5LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-correspondence(ipo)-(11-09-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWRyYXdpbmdzKDExLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-drawings(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMSgxMS0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 1(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMSgxNC0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 1(14-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMSgxNy0wMS0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 1(17-01-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMTMoMTEtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 13(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMTMoMTItMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 13(12-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMTgoMjgtMTItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 18(28-12-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMihncmFudGVkKS0oMTEtMDktMjAwNykuZG9j" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 2(granted)-(11-09-2007).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMihncmFudGVkKS0oMTEtMDktMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 2(granted)-(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMygxMS0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 3(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gMygxNy0wMS0yMDAxKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 3(17-01-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0gNSgxMS0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form 5(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0tcGN0LWlwZWEtNDA5KDE3LTAxLTIwMDEpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form-pct-ipea-409(17-01-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLWZvcm0tcGN0LWlzYS0yMTAoMTctMDEtMjAwMSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-form-pct-isa-210(17-01-2001).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLXBldGl0aW9uIHVuZGVyIHJ1bGUgMTM3KDExLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-petition under rule 137(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtNjItbXVtLXBvd2VyIG9mIGF0dG9ybmV5KDExLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-62-mum-power of attorney(11-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=c2VxdWVuY2UgbGlzdGluZyBvZiBpbi1wY3QtMjAwMS0wMDA2Mi1tdW0uZG9j" target="_blank" style="word-wrap:break-word;">sequence listing of in-pct-2001-00062-mum.doc</a></p>
		<br>
		<div class="pull-left">
			<a href="216567-annealing-control-primer-and-methods-involving-nucleic-acid-amplification-with-the-annealing-control-primer.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216569-image-processing-apparatus-and-method-and-image-capturing-apparatus.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216568</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00062/MUM</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jan-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>E.I.DU PONT DE NEMOURS AND COMPANY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1007 MARKET STREET, WILMINGTON, DELAWARE 19898, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BARBARA SUE VALENT</td>
											<td>800 HOPETON ROAD, WILMINGTON, DELAWARE 19807, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GREGORY T. BRYAN</td>
											<td>1215 Spruce Avenue Wilmington, DE 19805</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/29</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US99/17706</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-08-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/095,229</td>
									<td>1998-08-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/336,946</td>
									<td>1999-06-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216568-a-chimeric-gene-comprising-a-nucleic-acid-fragment-conferring-disease-resistance-to-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:47:40 GMT -->
</html>
