<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216567-annealing-control-primer-and-methods-involving-nucleic-acid-amplification-with-the-annealing-control-primer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:47:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216567:ANNEALING CONTROL PRIMER AND METHODS INVOLVING NUCLEIC ACID AMPLIFICATION WITH THE ANNEALING CONTROL PRIMER</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ANNEALING CONTROL PRIMER AND METHODS INVOLVING NUCLEIC ACID AMPLIFICATION WITH THE ANNEALING CONTROL PRIMER</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An annealing control primer for improving annealing specificity in nucleic acid amplification, which comprises: (a) a 3&#x27; -end portion having a hybridizing nucleotide sequence substantially complementary to a site on a template nucleic acid to hybridize therewith; (b) a 5&#x27; -end portion having a pre-selected arbitary nucleotide sequence; and (c) a regulator portion positioned between said 3&#x27; -end portion and said 5&#x27; -end portion comprising at least two universal bases or non-discriminatory base analogs, whereby said regulator portion is capable of regulating an annealing portion of said primer in association with annealing temperaturer.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
COMPLETE SPECIFICATION (Section 10; rule 13)<br>
"ANNEALING CONTROL PRIMER AND METHODS INVOLVING NUCLEIC ACID AMPLIFICATION WITH THE ANNEALING CONTROL PRIMER"<br>
We, SEEGENE INC., a Company incorporated in the Republic of Korea and having its principal place of business at 2th Fl., Myungji Bldg., 142-21, Samsung-dong, Kangnam-gu, 135-090, Seoul, Republic of Korea<br>
The following specification describes the nature of the invention and the manner in which it is to be performed.<br>
GRANTED<br><br>
ORIGINAL<br><br>
7-10-2005<br><br>
BACKGROUND OF THE INVENTION<br>
FIELD OF THE INVENTION<br>
The present invention relates to an annealing Control Primer and methods involving nucleic acid amplification with the annealing control primer. More particularly, the present invention relates to an annealing Control Primer and a method for amplifying a nucleic acid sequence from a DNA or a mixture of nucleic acids, a method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, a method for selectively amplifying a target nucleic acid sequence from an mRNA, a method for detecting DNA complementary to differentially expressed mRNA in two or more nucleic acid samples, a method for rapidly amplifying a target cDNA fragment comprising a cDNA region corresponding to the 3"- or 5"-end region of an mRNA, a method for amplifying a population of full-length double-stranded cDNAs complementary to mRNAs, a method for amplifying 5"-enriched double-stranded cDNAs complementary to mRNAs and a method for amplifying more than one target nucleotide sequence simultaneously.<br>
DESCRIPTION OF THE RELATED ART<br>
Nucleic acid amplification is a pivotal process for a wide variety of methods in molecular biology, so that various amplification methods have been proposed. For example, Miller, H. I. et al. (WO 89/06700) disclose a nucleic acid sequence amplification based on the hybridization of a promoter/primer sequence to a target single-stranded DNA ("ssDNA") followed by transcription of many RNA copies of the sequence. Other known nucleic acid amplification procedures include transcription-based amplification systems (Kwoh, D. et al., Proc. Natl Acad. Set U.S.A.,$6:l 173(1989); and Gingeras T.R. et al., WO 88/10315).<br>
Schemes based on ligation of two or more oligonucleotides in the presence of nucleic acid having the sequence of the resulting "di-oligonucleotide", thereby amplifying the di-oligonucleotide are also known (Wu, D.Y. et al., Genomics 4:560 (1989)), which are called "Ligation Chain Reaction" (LCR).<br>
Davey C. et al. (European Pat. Appln. Publication No.329,822) disclose a nucleic acid amplification process involving cyclically synthesizing single-stranded RNA ("ssRNA"), ssDNA, and double-stranded DNA (dsDNA). The ssRNA is a first template for a first primer oligonucleotide, which is elongated by reverse transcriptase (RNA-dependent DNA polymerase). The RNA is then removed from resulting DNA:RNA duplex by the action of ribonuclease H. The resultant ssDNA is a second template for a second primer, which also includes the sequences of an RNA polymerase promoter. This primer is then extended by DNA polymerase, resulting as a double-stranded DNA ("dsDNA") molecule, having a sequence identical to that of the original RNA between the primers and having additionally, at one end,a promoter sequence.This promoter sequence can be used by the appropriate RNA polymerase to<br><br><br>
produce many RNA copies of the DNA. These copies cart then re-enter the cycle leading to very rapid amplification.<br>
The most predominant process for nucleic acid amplification known as polymerase chain reaction (hereinafter referred to as "PCR"), is based on repeated cycles of denaturation of double-stranded DNA, followed by oligonucleotide primer annealing to the DNA template, and primer extension by a DNA polymerase (Mullis et al. U.S. Pat Nos. 4,683,195,4,683,202, and 4,800,159; Saiki et al. 1985). The oligonucleotide primers used in PCR are designed to anneal to opposite strands of the DNA, and are positioned so mat the DNA polymerase catalyzed extension product of one primer can serve as the template strand for the other primer. The PCR amplification process results in the exponential increase of discrete DNA fragments wb se length is defined by the 5" ends of the oligonucleotide primers.<br>
The success in the nucleic acid amplifications, in particular PCR amplification, relies on the specificity with which a primer anneals only to its target (and not non-target) sequences and therefore it is important to optimize this molecular interaction. Whether a primer can anneal only to its perfect complement or also to sequences that have one or more mismatches, depends critically upon the annealing temperature, hi general, the higher the annealing temperature, the more specific annealing of the primer to its perfect matched template and so the greater the likelihood of only target sequence amplification can be accomplished. The lower the temperature, the more mismatches between template and primer can be tolerated, leading to increased amplification of non-target sequences. Adjusting the annealing temperature can alter the specificity of pairing between template and primer. For examples, if there is no product, the temperature may be too high and can be reduced. If there are products in control where only one primer is present, this indicates that the single primer is annealing to more than one region of the template. In this case, the annealing temperature should be increased. Considering such effect of annealing temperature on primer annealing specificity, there remains a strong need for an annealing control primer system which is capable of controlling primer annealing in accordance with annealing temperature to enhance primer annealing specificity regardless of primer design.<br>
In addition to annealing temperature, several "primer search parameters" such as primer length, GC content and PCR product length (Dieffenbach et al., 1995) should be considered for primer annealing specificity. If a primer, which satisfies all such parameters, were employed, primer annealing would be specified, resulting in the significant enhancement of primer<br><br>
annealing specificity during target DNA amplification and the freedom from the problems such as backgrounds and non-specific products arising from primers used in the experiments. It is usual that well-designed primers can help avoid non-specific annealing and backgrounds as well as distinguish between cDNAs or genomic templates in RNA-PCR.<br>
Many approaches have been developed to improve primer annealing specificity and therefore accomplish the amplification of the desired product. Examples are touchdown PCR (Don et al., 1991), hot start PCR (D"Aquila et al., 1991), nested PCR (Mullis and Faloona, 1987) and booster PCR (Ruano et al., 1989). Another alternative approaches have been also reported that various "enhancer" compounds can improve the specificity of PCR. The enhancer compourds include chemicals mat increase the effective annealing temperature of the reaction, DNA ondir - . roteins and commercially available reagents. However, there is no •magic" additive that will ensure the success in every PCR and it is very tedious to test different additives under different conditions such as annealing temperature. Although these approaches have contributed to uie improvement of primer annealing specificity in some cases, they have not accessed fundamentally to a solution for the problems arising from primers used in the PCR amplification, such as not -specific products and high backgrounds.<br>
In many cases, the primer sequence does not need to be a perfect complement to the template sequence. The region of the primer that should be perfectly matched to the template is the 3"-end because this end is the region of the primer extended by the DNA polymerase and is therefore the most important for ensuring the specificity of annealing to the correct target sequence. The 5"-end of the primer is less important in detennining specificity of annealing to the target sequence and can be modified to carry additional sequence such as restriction sites and promoter sequences that are not complementary to the template (McPherson and Moller 2000). This notion is adapted to the design of the annealing control primers of this invention as described below.<br>
PCR-based techniques have been widely used not only for amplification of a target DNA sequence but also for scientific applications or methods in the fields of biological and medical research such as Reverse transcriptase PCR (RT-PCR), Differential Display PCR (DD-PCR), Cloning of known or unknown genes by PCR, Rapid amplification of cDNA ends (RACE) and PCR-based genomic analysis (McPherson and Moller, 2000). The followings are only representatives of PCR applications.<br>
Techniques designed to identify genes that are differentially regulated by cells under<br><br>
various physiological or experimental conditions (for example, differentiation, carcinogenesis, pharmacological treatment) have become pivotal in modem biology. One such method for screening differences in gene expression between various cell types or between different stages of cell development with the availability of PCR is known as Differential Display PCR (DD-PCR), described by Liang and Pardee in 1992. This method uses combinations of 10-mer arbitrary primers with anchored cDNA primers and generates fragments mat originate mostly from the poly(A) tail and extend about 50-600 nucleotide upstream. By combining 3" anchored Oligo(dT) primers and short 5" arbitrary primers, the subsets of the transcriptome are amplified, the resulting cDNA fragments are generally separated on denaturing polyacrylamide gel and visualized autoradiographically.<br>
Although this method is simple and rapid and oriy req&gt; V;. small amounts of total RNA, there are a number of disadvantages in the conventional DD-PCR methods. The differential banding patterns are often only poorly reproducible due to the use of short arbitrary primer so that many laboratories have had difficulty in obtaining reproducible losults with these methods. It has been shown that at least 40% of the differentially displayed bands are not reproducible between experiments even in well-trained hands (Bauer el al., 1994). Furthermore, the pattern of differential expression often cannot be reproducible on Northern blots and the percentage of these false positives can arise up to 90% (Sompayrac et al., 1995). As a modification used for an alternative, the use of longer random primers of, e.g. 20 bases in length does not satisfactorily solve the problem of reproducibility (Ito et al., 1994), There are another factors responsible for the relatively low reproducibility of DD-PCR such as an insufficient amount of starting material and very low concentration of dNTP (2-5 uM) employed to prepare the different banding patterns (Mate and Lukyanov, 1998). It is also difficult to detect rare transcripts with these methods (Mate and Lukyanov, 1998). In addition, because the cDNA fragments obtained from DD-PCR are short (typically 100-500 bp) and correspond to the 3"-end of the gene that represent mainly the 3" untranslated region, they usually do not contain a large portion of the coding region. Therefore, the labor-intensive full-length cDNA screening is needed unless significant sequence homology, information for gene classification and prediction of function is obtained (Mate and Lukyanov, 1998).<br>
Differential Display methods generally use radioactive detection techniques using denaturing polyacrylamide gels. The radioactive detection of the reaction products restricts the use of this technique to laboratories with the appropriate equipment. Relatively long exposure<br><br>
times and problems with the isolation of interesting bands from the polyactylamide gels arc additional drawbacks of Differential Display technique. Although modified non-radioactive Differential Display methods have recently been described, which include silver staining (Gottschlich et al. 1997; Kociok et al., 1998), fluorescent-labeled oligonucleotides (Bauer et aL 1993; Ito et al. 1994; Luehrsen et al, 1997; Smith et al„ 1997), the use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al., 1996) and ethidium bromide-stained agarose gels (Rompf andXaH 1997; Jefferies et al,, 1998; Gromova et al., 1999), these methods have met with only limited success. If the reaction products could be simply detected on etlridium bromide-stained agarose gel and the results were reproducible and reliable, it would greatly increase the speed of DD-PCR analysis and avoid the use of radioactivity.<br>
Another PCR-based approach called targeted differential display uses an oligonucleotide primer that directs the amplification of multigene family members with conserved protein domains. Oaie families are groups of genes which are often functionally characterized by a particular type of function undertaken by the gene products in a cell and which structurally have one or more conserved regions (domains) in common. Examples of gene families include the MADS-box and the homeogene family as well as further transcription factor families. The cyclin, cytokine and globin gene families are examples of medical interest. The Prosite Database provides a list of proteins that have common domains and sequence motifs. "Jlie. oligonucleotide used in the PCR can either be a specific primer that is used at a low annealing temperature or, as is more often the case, a degenerate primer mixture for use at higher stringencies (Stone and Wharton, 1994). However, amplifications using degenerate primers can sometimes be problematic and may require optimization. It is important to keep the annealing temperature as high as possible to avoid extensive nonspecific amplification and a good rule of thumb is to use 55°C as a starting temperature. In general, it is difficult to keep this rule because degenerate primers should be designed on the basis of amino acid sequences or conserved domain sequences as a precondition. In order to generate a satisfied relationship between degenerate primer and annealing temperature in this approach, it is required to use an annealing control primer which can tolerate the alternation of annealing temperature, particularly high temperature such as 68°C regardless of primer design.<br>
Sail another PCR-based technique is arbitrary primed PCR (AP-PCR) for UNA fingerprinting. One great strength of AP-PCR methods is their simplicity (Welsh and McClelland, 1991; Williams et al., 1990). AP-PCR uses a single primer or a pair of primers,<br><br>
wherein the primers are 10-mers or 18-mers as longer primer. This method has previously been used to provide DNA fingerprints of hybrid cell lines (Ledbetter et al., 1990) and particular genomic regions (Welsh and McClelland, 1990; Williams et al., 1990). It provides a very useful tool for genome analysis in bacteria, fungi and plant identification and population studies, where individual isolates can be compared rapidly. For example, they can be used as a tool to identify pathogens or the occurrence of particular strains or pathotypes. Commonly, AP-PCR uses a single primer to initiate DNA synthesis from regions of a template where the primer matches imperfectly. In order for this to work, the initial cycles have to be performed at low stringency (37-50°C), normally for the first five cycles, which allows primer annealing to imperfect sites throughout the genome. The stringency is then increased (55 °C) as for standard PCR amplification ar.J die f- tion is allowed for an additional 30-35 cycles. AP-PCR is not recommended for use in such applications as paternity testing where unequivocal results are demanded, because nonparental products are occasionally produced. Although alternative AP-PCR approaches including nested AP-PCR have been developed (McClelland et al., 1993; Ralph et al., 1993), the issue of reproducibility is still of main concern. One concern is that the patterns may vary from day to day or fron lab to lab (see, e.g., Meunier and Grimont, 1993).<br>
Still yet another PCR-based application is RACE (rapid amplification of cDNA end) technology. RACE is a procedure for amplification of cDNA regions corresponding to the 5 "-or 3"-end of mRNA (Frohman et al., 1988) aud it has been used to isolate rare transcripts successfully. The gene-specific primer may be derived from sequence data from a partial cDNA, genomic exon or peptide. In 3" RACE, the polyA tail of mRNA molecules is exploited as a priming site for PCR amplification. mRNAs are converted into cDNAs using reverse transcriptase and an Oligo-dT primer as known in the art. The generated cDNAs can then be directly PCR amplified using a gene-specific primer and a primer that anneals to the polyA region.<br>
The same principle as 3" RACE applies to 5" RACE but there is no polyA tail. Thus, 5" RACE is made by tagging the 5"-end of a cDNA by means of different methods (Fromont-Racine et al., 1993; Schaefer, 1995; Franz et al., 1999). Most approaches for the 5* RACE such as homopolymeric tailing and ligation anchored tailing require a set of enzymatic reactions after completion of first strand cDNA synthesis (Schaefer, 1995). Each enzymatic step has the potential to introduce failures and to destroy the integrity of the cDNA. Recently, an alternative has been introduced, die so-called CapFinder approach (Chenchik et al., 1998; Chenchik et al.<br><br>
U.S. Pat. Nos. 5,962,271 and 5,962,272). The technique relies on dual functions of the reverse transcriptases: one is the terminal transferase activity to add non-templated nucleotides to the 3"-end of a cDNA and the other is the template switching activity to switch a template to a second template. This property is utilized during the retroviral life cycle (Clark, 1988; Kulpa et al., 1997). Moloney murine leukemia virus (M-MLV) reverse transcriptase (RT) often adds three to four non-template-derived cytosine residues to the 3"-end of newly synthesized cDNAs in the presence of manganese or high magnesium (Schmidt and Mueller, 1999). This approach allows the amplification of full-lengtfi cDNAs because the M-MLV RT adds C residues preferentially to the cDNA if complete (capped) mRNA serves as template.<br>
However, the CapFinder approach for S"-RACE experiments could not be free from background problems such as DNA smear arising from the contar-j* Jitic , "the CapFinder and Oligo-dT primers, which are used in cDNA synthesis (Chenchik et al., 1998). Even residual amounts of these primers result in a high background because both ideally fit to all cDNAs present in the reaction mixture. In addition, 3"-RACE and full-length cDNA amplification have the same background problems due to the contamination of primers used for cDNA synthesis in which they generate non-specific products in PCR reaction (Chenchik et al., 19*&gt;8). New approaches to overcome the problems above have been recently introduced. One appioach is step-out PCR to suppress unwanted PCR products (Mate et al., 1999) but it has been pointed out that this approach still remains a smear of DNA rather than a single DNA (Schramm et al., 2000). Another approach which is introduced more recently is to use solid-phase cDNA synthesis and procedures to remove all contaminants used in cDNA synthesis (Schramm et al., 2000), but the major drawback of this technique is costly and time-consuming by requiring solid-phase cDNA synthesis and following procedures. Therefore, more effective, simple, rapid and inexpensive strategies are required to completely eliminate problems arising from contamination of the primers such as Oligo-dT or CapFinder primer used for cDNA synthesis.<br>
In addition to RACE technologies, in current technologies for cDNA library construction, the 5"-ends of genes tend to be under-represented in cDNA populations, especially where a poly(dT) primer is used during first cDNA strand synthesis and the starting material is limited. Although a number of different approaches have been developed to overcome this problem, most suffer from common limitations producing full-length cDNAs or 5"-enriched cDNAs with a number of inherent problems. These approaches are complex or costly and time-consuming by requiring multiple enzymatic steps and/or are not pronounced sensitive (Caimnci et al.,<br><br>
1997; Suzuki et al., 1997; Guegler et al. U.S. Pat. Nos. 6,083,727 and 6,326,175; Hayashizaki. U.S. Pat. No. 6,143,528). Therefore, there is continued interest in die development of improved methods for generating full-length or 5"-enriched cDNAs, particularly with the limited starting material.<br>
Multiplex PCR is another variant of PCR in which more tfian one target sequence can be simultaneously amplified with more than one pair of primers in the same reaction. Since its first description in 1988 (Chamberlain et al., 1988), mis method has been successfully applied in many areas of DNA testing, including analyses of gene deletion (Anonymous, 1992; Henegariu, et al., 1994), mutation and polymorphism analysis (Shuber et al., 1993; Mutirangura et al., 1993), quantitative analysis (Zimmermann et al., 1996), and RNA detection (Zou et al., 1998). In the field of infection diseases, the technique has been shown to be a valuable method for identification of viruses, bacteria, fungi, and/or parasites.<br>
However, the results obtained with multiplex PCR are frequently complicated by the aiufacts of the amplification procedure. These include "false-negative" results due to reaction failure and "false-positive" results such as the amplification of spurious products, which may be caured by annealing of the primers to sequences which are related to but distinct from the true recognition sequences. For use in multiplex PCR, a primer should be designed so that its predicted hybridization kinetics are similar to those of the other primers used in the sample multiplex reaction. While the annealing temperature and primer concentrations may be calculated to some degree, the conditions generally have to be empirically determined for each multiplex reaction. Since the possibility of non-specific priming increases with each additional primer pair, the conditions must be modified as necessary as individual primer sets are added. Moreover, the artifacts that result from competition for resources (e.g., depletion of primers) are augmented in multiplex PCR, since the differences in the yields of unequally amplified fragments are enhanced with each cycle. Thus, the optimization of the reaction conditions for multiplex PCR can become labor-intensive and time-consuming. Since the different multiplex PCRs may have unique reaction conditions, the development of new diagnostic tests can become very costly.<br>
Therefore, there is a need in the art for primers that allow multiplex PCR reactions to be designed and carried out without elaborate optimization steps, irrespective of the potentially divergent properties of the different primers used. Furthermore, there is a need in the art for primers that allow multiplex PCR reactions that, under the same reaction conditions,<br><br>
simultaneously produce equivalent amounts of each of many amplification products.<br>
Single nucleotide polymorphisms (SNPs), the most common genetic variations found in the human genome, are important markers for identifying disease-associated loci and for pharmaco-genetic studies (Landegren et al., 1998; Roses, 2000). SNPs appear in the human genome with an average of once every 1000 bp and totaling &gt;3 million. A variety of approaches have been used to detect SNPs. However, one of the key bottlenecks is the amplification of DNA. Most current assays include a step mat produces many copies of a short segment of the sample DNA spanning each target SNP. This amplification is usually necessary because only small amounts of DNA can be harvested from typical clinical samples. Also, the amplification improves the signs1-to-noise ratio of the assays, increasing the reliability of detection. Most genotyping t-x"jniq TJ accomplish this amplification using PCR. Most importantly, the specificity of PCR amplification is critical in the application of PCR in the SNP genotyping. Therefore, it would be beneficial if the methods for improving PCR specificity are available and applied to the development of SNP genotyping assay. It would also be beneficial if such methods are capable of providing multiple analyses in a single assay (multiplex assays).<br>
As described above, all these methods and techniques involving nucleic acid amplification, in particular PCR amplification, could not be completely free from the limitations and problems resulting from the non-specificity of primers used in each method, such as false positives, poor reproducibility, high backgrounds and so on, although improved approaches to each method has been continuously introduced: Therefore, there remains a need of novel primer for improving annealing specificity and methods, which can give rise to true results.<br>
Throughout this application, various patents and publications are referenced and citations are provided in parentheses. The disclosure of these patents and publications in their entities are hereby incorporated by references into this application in order to more fully describe this invention and the state of the art to which this invention pertains.<br>
SUMMARY OF THE INVENTION<br>
Endeavoring to resolve the problems of such conventional primer and various methods involving nucleic acid amplification, the present inventor has developed a novel annealing<br><br>
control primer that can permit nucleic acid amplification with much higher specificity and its unlimited applications in all fields of nucleic acid amplification-based technology.<br>
Accordingly, it is an object of this invention to provide an annealing control primer for improving annealing specificity in nucleic acid amplification.<br>
It is another object of this invention to provide a method for amplifying a nucleic acid sequence from a DNA or a mixture of nucleic acids as template.<br>
It is still another object of this invention to provide a method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids as template<br>
It is further object of this invention to provide a method for selectively amplifying a target nucleic acid sequence from an mRNA.<br>
It is still further object of this invention to provide a method for C&amp;G"^J DNA complementary to differentially expressed mRNA in two or more nucleic acid samples.<br>
It is another object of this invention to provide a method for rapidly amplifying a target cDNA fragment comprising a cDNA region corresponding to the 3"-end region of an mRNA.<br>
It is still another object of this invention to provide a method for amplifying a target cDNA fragment comprising a cDNA region corresponding to the 5"-end region of an mRNA.<br>
It is further object of this invention to provide a method for amplifying a population of full-length double-stranded cDNAs complementary to mRNAs.<br>
It is still further object of this invention to provide a method for amplifying 5"-enriched double-stranded cDNAs complementary to mRNAs.<br>
It is another object of this invention to provide a method for amplifying more than one target nucleotide sequence simultaneously,<br>
It is still another object of this invention to provide a method for producing a DNA fingerprint of gDNA.<br>
It is still another object of this invention to provide a method for producing a RNA fingerprint of an mRNA sample.<br>
It is further object of this invention to provide a method for identifying a conserved homology segment in a multigene family.<br>
It is still further object of this invention to provide a method for identifying a nucleotide variation in a target nucleic acid.<br>
It is another object of this invention to provide a method for mutagenesis in a target nucleic acid.<br><br>
It is still another object of this invention to provide a kit comprising an annealing control primer.<br>
It is further object of this invention to provide kits for a variety of methods involving nucleic acid amplification.<br>
It is still further object of this invention to provide a use of an annealing control primer for a process involving nucleic acid amplification.<br>
Other objects and advantages of the present invention will become apparent from the detailed description to follow taken in conjugation with the appended claims and drawings.<br>
BK IEF DESCRIPTION OF THE DRAWINGS<br>
FIGa. IA and IB show schematic representations for selectively amplifying a target nucleic acid of double-stranded DNA (IA) or mRNA (IB) using the ACP of the present invention,<br>
FIGs. 2A and 2B show schematic representations for identifying differentially expressed genes using the ACP of the present invention.<br>
FIG. 3 shows ., schematic representation for amplifying a target cDNA fragment comprising 3"-end region corresponding to the 3"-end of mRNA using the ACP of the present invention.<br>
FIGs. 4A and 4B show schematic representations for amplifying a target cDNA fragment comprising 5"-cnd region corresponding to the 5"-end of mRNA using the ACP of the present invention. The Oligo dT (4A) or random primer (4B) is used as a first-strand cDNA synthesis primer.<br>
FIG. 5 shows a schematic representation for amplifying full-length cDNA molecules complementary to the mRNA molecules using the ACP of the present invention.<br>
FIG. 6 shows a schematic representation for amplifying 5" enriched cDNA molecules complementary to the mRNA molecules comprising the 5"-end information using the ACP of the present invention.<br>
FIG. 7A shows a schematic representation for detecting single nucleotide polymorphism (SNP) using the ACP of the present invention.<br>
FIG.- 7B shows another schematic representation for detecting single nucleotide polymorphism (SNP) using the ACP of the present invention.<br><br>
FIG. 8 is an agarose gel photograph to show the effect of a deoxyinosine group positioned<br>
between the 3"- and 5"-end portions of ACP. The cDNA was amplified using total RNA<br>
isolated from conceptus tissues at E4.5 (lanes 1 and 4), El 1.5 (lanes 2 and 5), and El8.5 (lanes<br>
3 and 6), with a set of the dT,0-JYC2 (SEG ID NO. 29) and ACP10 (lanes 1-3) (SEG ID NO.<br>
5      13), and a set of the dTio-ACPl (SEG ID NO. 30) and ACP10 (lanes 4-6), respectively.<br>
FIG. 9 is an agarose gel photograph to show the effect of deoxyinosine residues positioned<br>
between the 3"- and 5"-end portions of ACP in association with the alteration of number of<br>
deoxyinosine during PCR The lanes 0, 2, 4, 6, and 8 represent the number of deoxyinosine<br>
residues, respectively.<br>
10	FIG.  10A is an agarose gel photograph to show the results of two stage PCR<br>
amplifications for Esxl using a set of EsxN" ? .id i "^: % primers (lane 1) and a set of EsxN7-ACP and EsxC6-ACP primers (lane 2).<br>
FIG.  10B is an agarose gel photograph to show the results of two stage PCR amplifications for Esxl using EsxNl (lane 1), EsxC2 (lane 2), a set of EsxNl-ACP and EsxC2 15      (lane 3), and a set of EsxNl -ACP and EsxC2-ACP (lane 4).<br>
FIG. 10C is an agarose gel photograph to show the results of two stage PCR amplifications for Esxl using a set of EsxN3 and EsxC5 (lanes 1 and 2) and a set of EsxN3-ACP and EsxC5-ACP (lane 3).<br>
FIG. 10D is an agrasoe gel photograph to show the results of non-stop two stage PCR 2 0      amplifications for Esxl using the primer EsxNl (lane 1), EsxC2 (lane 2), a pair of EsxNl and EsxC2 (lane 3) and a pair of EsxNl-ACP and EsxC2-ACP (lane 4).<br>
FIG. HA is a photograph of agarose gels to show examples of the ACP used for detecting differentially expressed mRNAs during embryonic development using different stages of mouse conceptus tissues. The cDNAs were amplified using total RNA isolated from conceptus 2 5 tissues at E4.5 (lane 1), El 1.5 (lane 2), and E18.5 (lane 3), with a set of ACP3 (SEG ID NO. 3) and dTio-ACPl. The bands indicated by arrows represent the cDNA fragments amplified from differentially expressed mRNAs. The numbers of the arrows indicate the cDNA fragments used as probes in the Northern blot analysis of FIG. 13.<br>
FIG. 1 IB is a photograph of agarose gels to show examples of the ACP used for detecting<br>
30      differentially expressed mRNAs during embryonic development using different stages of<br>
mouse conceptus tissues. The cDNAs were amplified using total RNA isolated from conceptus<br>
tissues at E4.5 (lanes 1-2 and 7-8), El 1.5 flanes 3-4 and 9-10), and El 8.5 (lanes 5-6 and 11-<br><br><br>
12), with a set of ACP5 (SEG ID NO. 5) and dTl0-ACPl (the lanes 1-6), and a set of ACP8 (SEG ID NO. 8) and dTi0-ACPl 0anes 7-12), respectively. The bands indicated by arrows represent the cDNA fragments amplified from differentially expressed mRNAs. The numbers of the arrows indicate the cDNA fragments used as probes in the Northern blot analysis of FIG. 5       13.<br>
FIG. 11C is an agarose gel photograph to show the amplified cDNA products obtained from different stages of mouse conceptus samples (E4.5: lanes 1 and 2; Ell.5: lanes 3 and 4; E18.5: lanes 5 and 6) using a set of ACP10 and dTto-ACP primers.<br>
FIG. 1 ID is an agarose gel photograph to show the amplified cDNA products obtained 10      from different stages of mouse conceptus samples (E4.5: lanes 1 and 2; Ell.S: lanes 3 and 4; E18.5: lanes 5 and 6) using a set of ACP14 and Tio-ACPl primers.<br>
FIG. 12A is an agarose gel photograph to show the amplified cDNA products obtained<br>
from JifFerent stages of mouse conceptus samples (E4.5: lane 1; El 1.5: lane 2; E18.5: lane 3)<br>
by one-stop two-stage PCR amplification using a set of ACP10 and JYC5-TiS-ACP primers.<br>
15	FIG. 12B is an agarose gel photograph to show the amplified cDNA products obtained<br>
from different stages of mouse conceptus samples (E4.5: lane 1; E11.5: lane 2; E18.5: lane 3) by non-stop two-stage PCR amplification using a set of ACP10 and JYC5-TiS-ACP primers.<br>
FIG. 13 shows Northern blot analysis of six cDNA fragments amplified from differentially expressed mRNAs during embryonic development. The six 32P-labeled fragments indicated by<br>
2	0      arrows in FIG. 11 were used as probes for Northern blot analysis. The arrows 1, 2, 3, 4, 5, and<br>
6 are DEG1 (FIG. 13A), DEG3 (FIG. 13B), DEG2 (FIG. 13C), DEG8 (FIG. 13D), DEG5 (FIG. 13E), and DEG7 (FIG. 13F), respectively, wherein the results of the DEG sequence analysis are shown in Table 1. DEG2 (SEG ID NO. 31) and DEG5 (SEG ID NO. 32) are turned out as novel genes (Table 2). The control panels (the lower part of each panel) show each gel before 25 blotting, stained with ethidium bromide and photographed under UV light, demonstrating similar levels of 18S and 28S rRNA as a loading control.<br>
FIG. 14 shows the expression patterns of a novel gene, DEG5, in a full stage of mouse conceptus. Northern blot analysis was performed using the radio-labeled DEG5 cDNA fragment as a probe. Total RNA (20 ug/lane) was prepared from mouse conceptuses at the<br>
3	0      gestation times as indicated. The control panel at the lower part shows a gel before blotting,<br>
stained with ethidium bromide and photographed under UV light, demonstrating similar levels of 18S and 28S rRNA as a loading control.<br><br><br><br><br><br>
FIG. 15 is an agarose gel photograph to show the difference between the conventional 3"-RACE (lane 1) and the ACP-based 3*-RACE (lane 2) with regard to beta-actin 3"-RACE,<br>
FIG. 16 is an agarose gel photograph to show the difference between CapFinder methods and ACP-based methods for mouse JunB (lanes 1 and 2) and beta-actin 5"-RACE (lanes 3 and 5      4) using the conventional primer (lanes 1 and 3) and ACP (lanes 2 and 4), respectively<br>
FIG. 17 is an agarose gel photograph to show the difference between CapFinder methods and ACP-based methods for mouse PLP-C alpha 5"-RACE using the conventional primer (lane 1) and ACP (lanes 2, 3, and 4), respectively.<br>
FIG. 18 shows the results of virtual Northern analysis by the CapFinder methods or ACP-10      based methods forf ic amplification of mouse full-length GAPDH cDNA,<br>
FIG. 19 sbor.* agji. .•&gt;" -. gel photographs to show the results of genomic fingerprintings of 7 mouse stains using two different sets of arbitrary ACPs.<br>
FIG. 20 shows agarose gel photographs to short the amplified products of multiplex PCR by the conventional methods (A) or ACP-based methods (B) for the amplification of three 15      target nucleic acids.<br>
FIG. 21 shows agarose gel pho^graphs to show the amplified products of multiplex PCR<br>
by the conventional methods (A) or ACP-based methods (B and C) for the amplification of four<br>
target nucleic acids. The ACP-based multiplex was conducted by one-stop (B) or non-stop (C)<br>
two-stage PCR amplification,<br>
20	FIG. 22 shows an agarose gel photograph to show the results of allele-specific<br>
amplification for a SNP in exon 4 of the human TP53 gene using ACP.<br>
FIG. 23 shows six agarose gel photographs which show the results of allele-specific<br>
amplifications using ACPs for six additional SNPs each present in different gene such as Beta-<br>
2 adrenergic receptor (ADRB2) (A), Chemokine (c-c motif) receptor 5 (CCR5) (B), Literleukin<br>
25      13 receptor (C), Leukocyte adhesion molecule-l (LAM-1) (D), Tachykinin receptor 3<br>
(TACR3) (E), and Ihterleukin 1, beta (DL1B) (F).<br>
DETAILED DESCRIPTION OF THIS INVETNION<br>
The present invention is generally directed to (a) an annealing control primer for the<br>
specificity of nucleic acid amplification and (b) its applications. The annealing control primer<br>
30      of this invention (hereinafter referred to as "ACP") allows primer annealing to be controlled in<br>
association with annealing temperature, such that the specificity of nucleic acid amplification<br><br>
(in particular, PCR) can be significantly improved. The principle of the ACP is based on the composition of an oligonucleotide primer having 3"- and 5"-ends distinct portions separated by at least one universal base or non-discriminatory base. The present inventor has discovered that the universal base or non-discriminatory base group positioned between the 3"- and 5"- end portions plays as a regulator in controlling primer annealing to a template nucleic acid in associated with annealing temperature during nucleic acid amplification. The presence of universal base or non-discriminatory base residue group positioned between the 3"- and 5"- end portions interrupts the annealing of the S"-end portion as well as limits primer annealing to the 3"-end portion at certain annealing temperature, which results in dramatic improvement of annealing specificity. A universal base group positioned betwc-n the 3"- and 5"-end portions of ACP is designed to define each portion. For these reasons, *J.Y. AC "* fiindamentally different from the conventional primers in terms of the function for improving primer annealing specificity under a particular stringency conditions during nucleic acid amplification.<br>
The ACP of this invention is significantly effective and widely accessible to nucleic acid amplification-based applications. Also, various problems related to primer annealing specificity in the conventional PCR techniques can be fundamentally solved by the /CP. The main benefits to be obtained from the use of the ACP during nucleic acid amplification (particularly PCR) are as follows:<br>
(a)	since the presence of an universal base residue group positioned between the 3"- and 5"-end portions restricts primer annealing portion to the 3"-end portion under such conditions that the 3"-end portion anneals to the template, the annealing sequence of a primer can be precisely controlled, which make it possible to design a primer with a desired number of annealing sequence. It is particularly useful when an annealing portion of a primer has to be limited (e.g., single nucleotide polymorphism (SNP) genotyping, DNA microarrary screening, and detection of differentially expressed genes);<br>
(b)	since the presence of an universal base residue group positioned between the 3"- and 5"-end portions interrupts the annealing of the 5"-end portion to the template under such conditions that the 3"-end portion anneals to the template, eventually the 5"-end portion not involved in the annealing provides the 3"-end portion with primer annealing specificity;<br>
(c)	the specificity of primer annealing is highly sensitive enough to detect even a single-base mismatching. Thus, it is particularly useful for the identification of a nucleotide variation<br><br><br>
in a target nucleic acid, including, for example, single nucleotide polymorphisms and point mutations;<br>
(d)	ACP is capable of providing a primer with a high tolerance in "primer search parameters" for primer design such as primer length, annealing temperature, GC content, and PCR product length;<br>
(e)	ACP system provides two-stage PCR amplifications which allow the products to be excluded from non-specific amplification;<br>
(f)	the efficiency of PCR amplification is increased, which makes it easier to detect rare mRNAsjand<br>
(g)	the reproducibility of PCR products is increased, which saves a great amount of time and cost.<br>
Principle of ACP<br>
In one aspect of this invention, there is provided an annealing control primer for improving annealing specificity in nucleic acid amplification, which comprises: (a) a 3*-end nortion having a hybridizing nucleotide sequence substantially complementary to a site on a template nucleic acid to hybridize therewith; (b) a 5"-end portion having a pro-selected arbitrary nucleotide sequence; and (c) a regulator portion positioned between said 3"-cud portion and said 5"-end portion comprising at least one universal base or non-discriminatory base analog, whereby said regulator portion is capable of regulating an annealing portion of said primer in association with annealing temperature.<br>
The principle of ACP is based on the composition of an oligonucleotide primer having 3"- and 5"-end distinct portions separated by a regulator portion comprising at least one universal base or non-discriminatory base and the effect of the regulator portion on the 3"-and 5"-end portions in the oligonucleotide primer. The presence of the regulator portion comprising at least one universal base or non-discriminatory base between the 3"- and 5"-end portions of ACP acts as a main factor which is responsible for the improvement of primer annealing specificity.<br>
The term "template" refers to nucleic acid. The term "nucleic acid" is a deoxyribonucleotide or ribonucleotide polymer in either single or double-stranded form, including known analogs of natural nucleotides unless otherwise indicated. Therefore, the ACP of this invention can be employed in nucleic acid amplification using single or double-stranded<br><br><br>
gDNA, cDNA or mRNA as template. The term "portion" used herein in conjunction with the primer of this invention refers to a nucleotide sequence separated by the regulator portion. The term "3"-end portion" or "5 "-end portion" refers to a nucleotide sequence at the 3""-end or 5"-end of the primer of this invention, respectively, which is separated by the regulator portion.<br>
The term "primer" as used herein refers to an oligonucleotide, whether occurring naturally or produced synthetically, which is capable of acting as a point of initiation of synthesis when placed under conditions in which synthesis of primer extension product which is complementary to a nucleic acid strand (template) is induced, i.e., in the presence of nucleotides and an agent for polymerization such as DNA polymerase and at a suitable temperature and pH. The primer is.preferably single stranded for maximum efficiency in amplification. Preferably, JS Tj.sr is an oligodeoxyribonucleotide. The primer of this invention can be comprised of naturally occurring dNMP (i.e., dAMP, dGM, dCMP and dTMP), modified nucleotide or non-natural nucleotide.. The primer can also include ribonucleotides. The primer must be sufficiently long to prime the synthesis of extension products in the presence of the agent for polymerization. The exact length of the primers will depend on many factors, including temperature, application and source of primer. The term "annealing" or "priming" as used herein refers U&gt; the apposition of an oligodeoxynucleotide or nucleic acid to a template nucleic acid, whereby said apposition enables the polymerase to polymerize nucleotides into a nucleic acid molecule which is complementary to the template nucleic acid or a portion thereof.<br>
The 3"-end portion of ACP has a nucleotide sequence substantially complementary to a site on a template nucleic acid molecule. The term "substantially complementary" in reference to primer is used herein to mean that the primer is sufficiently complementary to hybridize selectively to a template nucleic acid sequence under the designated annealing conditions, such that the annealed primer can be extended by polymerase to form a complementary copy of the template. Therefore, this term has a different meaning from "perfectly complementary" or related terms thereof. It will be appreciated that the 3"-end portion of ACP can have one or more mismatches to template to an extent that the ACP can serve as primer. Most preferably, the 3"-end portion of ACP has a nucleotide sequence perfectly complementary to a site on a template, i.e., no mismatches.<br>
The 3"-end portion of ACP may have a wide variety of nucleotide sequences depending on its applications as well as template sequence. For example, where the ACP is applied to the<br><br>
process involving reverse transcription such as differential display PCR, RACE, amplification of full-length cDNA, fingerprinting, identification of conserved homology segment and the like, its 3"-end portion may have the nucleotide sequence which hybridizes to the polyadenosine (poly A) tail of an mRNA, preferably at least 8 deoxymymidine nucleotides, more preferably at least 10 deoxythymidine nucleotides and the most preferably, at least 10 contiguous deoxythymidine nucleotides. For the process involving reverse transcription as above, in one embodiment, the 3"-end portion of ACP has at least 10 contiguous deoxythymidine nucleotides having 3"-V at its 3"-end; in which V is" one selected from the group consisting of deoxyadenosine, deoxycytidine and deoxyguanosine, in another embodiment, at least 10 contiguous deoxythymidine nucleotides having .V-NV at its 3"-end; in which V is one selected from the group consisting of deoxyadenof.tf e, t *«* • /cytidine and deoxyguanosine, and N is one selected from the group consisting of deoxyadenosine, deoxythymidine, deoxycytidine and deoxyguanosine.<br>
Furthermore, where the ACP is employed in amplification of a target nucieic acid sequence, its 3"-end portion comprises a nucleotide sequence substantially complementary to a target sequence; in differential display PCR, an arbitrary sequence substantially complement try to a site in a cDNA from an mRNA; in RACE, a gene-specific sequence substantially complementary to a site in a cDNA from an mRNA; in amplification of 5"-enriched c-DNAs, a random sequence of at least six nucleotides substantially complementary to sites in mRNAs; in identification of conserved homology segment, a nucleotide sequence substantially complementary to a consensus sequence found in a gene family or degenerate sequence selected from a plurality of combinations of nucleotides encoding a predetermined amino acid sequence; in identification of a nucleotide variation (e.g., allelic site) in a target nucleic acid, a nucleotide sequence comprising a nucleotide complementary to the corresponding nucleotide of a nucleotide variation; and in mutagenesis, a nucleotide sequence comprising at least one mismatch nucleotide to a target nucleic acid.<br>
The term "arbitrary" nucleotide sequence is used herein to mean the nucleotide sequence that is chosen without knowledge of the sequence of the target nucleic acids to be amplified. The term arbitrary should not to be confused with "random" in reference to primer which connotes a primer composed of a random population of primers each of different and random sequence. The term "degenerate" sequence in conjunction with ACP for identification of conserved homology segment refers to the nucleotide sequence that is deducted from amino<br><br>
acid sequence, so that the degenerate sequence can form a pool of the nucleotide sequences from one amino acid sequence due to degeneracy of genetic codon.<br>
According to a preferred embodiment of the ACP, the pre-selected arbitrary nucleotide sequence of me 5"-end portion is substantially not complementary to any site on the template 5      nucleic acid.<br>
According to a preferred embodiment, the annealing control primer of this invention can be represented by a general formula (1) of 5"-Xp-Yq-Zr-3", wherein Xp represents the S"-end portion having the pre-selected arbitrary nucleotide sequence substantially not complementary to any site on the template nucleic acid; Yq represents the regulator portion comprising at least<br>
10 one universal base or non-discriminatory base analog; Zr represents the 3 "-end portion having a : ucleotide sequence substantially complementary to a site on the template nucleic acid; wherein p, q and r represent the number of nucleotides; and wherein X, Y and Z is deoxyribonucleotide or ribonucleotide.<br>
The regulator portion comprising at least one universal base or non-discriminatory base<br>
15 analog is responsible for the main function of ACP in associated with alteration of annealing temperature during nucleic acid amplification. The term "universal base or non-discriminatory base analog" used herein refers to one capable of forming base pairs with each of the natural DNA/RNA bases with little discrimination between them.<br>
It has been widely known that nucleotides at some ambiguous positions of degenerate<br>
20 primers have been replaced by universal base or a non-discriminatory analogue such as deoxyinosine (Ohtsuka et al, 1985; Sakanari et al., 1989), l-(2"-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole (Nichols et al., 1994) and 5-nitroindole (Loakes and Brown, 1994) for solving the design problems associated with the degenerate primers because such universal bases are capable of non-specifically base pairing with all four conventional bases. However, there has<br>
25 not been any report that this universal base or a non-discriminatory analogue such as deoxyinosine, l-(2"-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole and 5-nitroindole is used to increase the specificity of primer annealing during PCR.<br>
The presence of universal base such as deoxyinosine, l-(2"-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole and 5-nitroindole in a primer generates low annealing temperatures due to its<br>
3 0 weaker hydrogen bonding interactions in base pairing. As an extension of this theory, the present inventor has induced that the presence of the contiguous universal bases between the 3"-end and 5"-end of a primer could generate a region which has lower melting temperature,<br><br>
forms a boundary to each of 3"-and 5"-end portions of the primer, and affect the annealing of each portion, respectively. This theory provides the basis of the annealing control primers of this invention.<br>
In a preferred embodiment, the ACP contains at least 2 universal base or non¬discriminatory base analog residues between the 3"- and 5"-end portion sequences, more preferably, at least 3 universal bases or non-discriminatory base analogs. Advantageously, the universal base residues between the 3"- and 5"-end portion sequences can be up to 15 residues in length. According to one embodiment, the ACP contains 2-15 universal base or non¬discriminatory base analog residues. Most preferably, the universal bases between the 3"- and 5"-end portion sequences are about 5 residues ir length.<br>
With reference to the optimum auro *r; of universal base, i.e., 5 residues, the minimum number of universal base residues between the 3"- and 5"-end portions of ACP is preferred in order to interrupt the annealing of the 5"-end portion to the template during nucleic acid amplification at certain annealing temperature. It is very likely that the length of universal base in the sequence (8-10 bases) does not make a significant difference on its own function in ACP.<br>
The use of universal base residues between the 3"- and 5"-end portion sequences is considered as a key feature in the present invention because it provides each portion (3"- and 5"-end) with a distinct annealing specificity in association with an annealing temperature during nucleic acid amplification, e.g. PCR.<br>
According to a preferred embodiment, the universal base or non-discriminatory base<br>
analog in the regulator portion includes deoxyinosinc, inosine, 7-deaza-2"-deoxyinosine, 2-aza-<br>
2"-deoxyinosine, 2"-OMe inosine, 2"-F inosine, deoxy 3-nitropyrrole, 3-nitropyrrole, 2"-OMe 3-<br>
nitropyrrole, 2"-F 3-nitropyrrole, l-(2Meoxy-beta-D-ribofuranosyl)-3-nitropyrrole, deoxy 5-<br>
nitroindole, 5-nitroindole, 2"-OMe 5-nitroindole, 2"-F 5-nitroindole, deoxy 4-<br>
nitrobenzimidazole, 4-nitrobenzimidazole, deoxy 4-aminobenzimidazole, 4-<br>
aminobenamidazole, deoxy nebularine, 2"-F nebularine, 2"-F 4-nitroben2imidazole, PNA-5-<br>
introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole,<br>
morpholino-5-nitroindole, morpholino-nebularine, morpholino-inosine, morphoIino-4-<br>
nitrobenzimidazole,	morphoIino-3-nitropyrrole,	phosphoramidate-5-nitroindole,<br>
phosphoramidate-nebularine,	phosphoramidate-inosrae,	phosphoramidate-4-<br>
nitrobenzimidazole, phosphoramidate-3-nitropyrrole, 2"-0-methoxyethyl inosine, 2"0-methoxyethyl   nebularine,   2"-0-methoxyethyl   5-nitroindole,   2"-0-methoxyethyl   4-nitro-<br>
benzimidazole, 2"-0-methoxyethyl 3-nitropyrrole and combinations thereof, but not limited to. More preferably, the universal base or non-discriminatory base analog is deoxyinosine, l-(2"-deoxy-oeta-D-ribofuranosyl)-3-nitropyiT0le or 5-nitroindole, most preferably, deoxyinosine.<br>
The preferred length of an oligonucleotide primer, as used herein, is determined from desired specificity of annealing and the number of oligonucleotides having the desired specificity that are required to hybridize to the template. For example, an oligonucleotide primer of 20 nucleotides is more specific than an oligonucleotide primer of 10 nucleotides because the addition of each nucleotide to an oligonucleotide increases the annealing temperature of the primer to the template.<br>
The lengths of the 3"- and 5"-end portion sequences of the ACP may vary and .Jep^nd in part on the objective of each application using ACP. In a preferred embodiment, the J ,nd portion of ACP is at least 6 nucleotides in length, which is considered a minimal requirement of length for primer annealing. More preferably, the 3"-end portion sequence is from 10 to 25 nucleotides and can be up to 60 nucleotides in length, m another embodiment, the 3" -end portion of ACP can include ribonucleotides as well as deoxyribonucleotides.<br>
In another preferred embodiment, the 5"-end portion of ACP contains at least 15 nucleotides in length, which is considered a minimal requirement of length for annealing under high stringent conditions. Preferably, the 5"-cnd portion sequence can be up to 60 nucleotides in length. More preferably, the 5"-end portion sequence is from 6 to 50 nucleotides, most "preferably, from 20 to 25 nucleotides in length. The entire ACP is preferably from 35 to 50 nucleotides in length, and can be up to 100 nucleotides in length.<br>
The 5"-end portion of ACP has a pre-selected arbitrary nucleotide sequence substantially not complementary to any site on the template nucleic acid and this nucleotide sequence can serves as a priming site for subsequent amplification. The term "pre-selected arbitrary" nucleotide sequence used herein refers as any defined or pre-selected deoxyribonucleotide, ribonucleotide, or mixed deoxyribonucleotide sequence which contains a particular sequence of natural or modified nucleotides. In some embodiment, me pre-selected arbitrary nucleotide sequence of the 5"-end portion can be composed of a universal primer sequence such as T3 promoter sequence, T7 promoter sequence, SP6 promoter sequence, and M13 forward or reverse universal sequence. Using a longer arbitrary sequence (about 25 to 60 bases) at the 5"-end portion of ACP reduces the efficiency of ACP, but shorter sequences (about 15 to 17 bases) reduce the efficiency of annealing at high stringent conditions of ACP. It is also a key feature<br><br><br><br><br><br><br>
of the present invention to use a pre-selected arbitrary nucleotide sequence at the 5"-end portion of ACP as a priming site for subsequent amplification.<br>
According to one embodiment of the present invention, some modifications in the 5"-end portion of ACP can be made unless the modifications abolish the advantages of the ACP, i.e., 5 improvement in annealing specificity. For example, the 5"-end portion can comprises a sequence or sequences recognized by a restriction endonuclease(s), which makes it feasible to clone the amplified product into suitable vector. In addition, the 5"-end portion am comprises at least one nucleotide with a label for detection or isolation of amplified product. Suitable labels include, but not limited to, fluorophores, chromophores, chemiluminescers, magnetic<br>
10 particles radioisotopes, mass labels, electron dense particles, enzymes, cofactors, substrates for enzyme", anc Viptens having specific binding partners, e.g., an antibody, streptavidin, biotin, digoxigcnin and chelating group. The 5"-end portion also comprises bacteriophage RNA polymerase promoter region.<br>
According to the preferred embodiment of this invention, the ACP is applied to PCR.<br>
15 More preferably, the PCR is performed under a first and a second annealing temperaturej i.e., under different stringent c ^nditions. The first annealing temperature may be equal to or lower than the second annealing temperature and preferably, the second annealing tempo aim c i-higher than the first annealing temperature. In the PCR process performed under two (iiff i 
2 0 first annealing temperature and the 5"-end of ACP incorporated into amplified product of In; i amplification stage serves as a priming site at the second annealing temperature. 3u flm; oa:;^ the advantages of ACP will be demonstrated in accordance with the following assumptions,:<br>
(1)	since a regulator portion of ACP is composed of at least one universal base or non¬<br>
discriminatory analogue which has lower Tm than other .portion in ACP due to its weal■*..*•<br>
2	5      hydrogen bonding interactions in base pairing, the regulator portion of ACP is not favorable, in<br>
annealing to the template nucleic acid under the conditions that the 3"-end portion of ACP anneals to a site of the template at a first annealing temperature. Consequently, the- presence of a regulator portion comprising at least one universal base or non-discriminatory analogue between the 3"- and 5"-end portions of ACP restricts primer annealing portion to the 3*«nd<br>
3	0      portion at first annealing temperature;<br>
(2)	the 5"-end portion which is not involved in the annealing under the first aimealing<br>
temperature keeps bothering the annealing of the 3 "-end portion to the template;<br><br><br>
(3) thus, the strength in which the specific annealing of the 3"-end portion sequence occurs<br>
is relatively stronger than the strength in which non-specific annealing occurs, under the first<br>
annealing temperature, which results in the improvement of primer annealing specificity at the<br>
3"-end portion;<br>
5	(4) where the 5"-end portion comprises a pre-selected arbitrary nucleotide sequence, the<br>
portion serves as a priming site at a second annealing temperature, which is high stringency conditions and also should be higher than the first annealing temperature, for subsequent amplification of reaction product generated from annealing and extension of the 3"-end portion sequence; and<br>
1 o	(5) consequently, only the reaction product generated from annealing and extension of the<br>
3"-end portion sequence can be amplified clce to V, theoretical optimum of a two-fold increase of product for each PCR cycle under the second annealing temperature.<br>
Therefore, the 3"-end portion of ACP acts only as annealing site to the template at the first annealing temperature and the 5"-end portion of ACP is used as a priming site at the second<br>
15 annealing temperature for the subsequent amplification of the product generated by contacting and extending the 3"-end portion of ACP to the template.<br>
It may be appreciated that the ACP of the present invention is very useful in a variety of primer-based nucleic acid amplification methods including the methods of Miller, H. I. (WO 89/06700) and Davey, C. et al. (EP 329,822), Ligase Chain Reaction (LCR, Wu, D.Y. et al,<br>
20 Genomics 4:560 (1989)), Polymerease Ligase Chain Reaction (Barany, PCR Methods and Applic, 1:5-16(1991)), Gap-LCR (WO 90/01069), Repair Chain Reaction (EP 439,182), 3SR (Kwoh et al., PNAS, USA, 86:1173(1989)) and NASBA (U.S. Pat. No. 5,130,238), but not limited to.<br>
25	In another aspect of this invention, there is provided a kit comprising the annealing control<br>
primer or the annealing control primer set according to the present invention. According to one embodiment of this invention, this kit further comprises a primer or a primer pair having a nucleotide sequence corresponding to the 5"-end portion of the ACP; in case that the 5"-end portion comprises universal primer sequence, it is more preferred that the kit comprises the<br>
3 0 universal primers. The present kits may optionally include the reagents required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5"-tripho$phates.   Optionally,   the   kits   may   also   include   various<br><br>
polynucleotide molecules, reverse transcriptase, various buffers and reagents, and antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary for performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current 5 disclosure. The kits, typically, are adapted to contain in separate packaging or compartments the constituents afore-described.<br>
The ACP of the subject invention can be applied to a variety of nucleic acid amplification-<br>
based technologies. Representative examples to prove the effect of ACP are:<br>
	I. Application to amplifying a nucleic acid sequence;<br>
n. Application to amplifying a target nucleic acid sequence;	<br>
EL Application to multiplex DNA amplification;	<br>
IV.	Application to the identification of differentially expressed genes;<br>
V.	Application to rapid amplification of cDNA ends (RACE);	15                VI. Application to amplifying full-length cDNA;	<br>
VII. Application to amplifying 5"-enriched cDNA; VHI. Application to DNA or RNA fingerprinting;<br>
DC Application to the identification of conserved homology segments in multigene<br>
families;<br>
20	X. Application to identification of a nucleotide sequence variation;<br>
XI. Application to mutagenesis; and XTI. Other applications.<br>
I. Application to Amplifying a (target) Nucleic Acid Sequence<br>
In still another of this invention, there is provided a method for amplifying a nucleic acid 25 sequence from a DNA or a mixture of nucleic acids, comprising performing an amplification reaction using primers, characterized in that at least one primer is derived from any one of ACP described above. Preferably, the primer according to fee structure of ACP is one having at its 3"end portion a hybridizing sequence substantially complementary to a region of the nucleic acid sequence to hybridize therewith.<br><br>
In a specific embodiment of this method, there is provided a method using two stage amplifications for amplifying a nucleic acid sequence from a DNA or a mixture of nucleic acids, which comprises:<br>
(a)	performing a first-stage amplification of the nucleic acid sequence at a first annealing<br>
5      temperature comprising at least two cycles of primer annealing, primer extending and<br>
denaturing, using the primer pair of any one of the ACP described above each having at its 3" end portion a hybridizing sequence substantially complementary to a region of the nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to the region of the nucleic acid sequence, whereby the amplification product of the nucleic acid 10      sequence is gene ated; an<l></l>
(b)	perfriT.Jng /.cond-stage amplification of the amplification product generated from<br>
step (a) at a second annealing temperature, which is high stringent conditions, comprising at<br>
least one cycle of primer annealing, primer extending and denaturing, using the same primers<br>
as used in step (a) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence<br>
15 corresponding to each 5" -end portion of the primers used in step (a), under conditions in which each primer anneals to the 3"- anc 5"-ends of the amplification product, respectively, whereby the amplification product is re-amplified.<br>
Where the method is applied to the amplification of a target nucleic acid sequence, the primer pair used has at its 3"-end portion a hybridizing sequence substantially complementary<br>
20 to a region of the target nucleic acid sequence to hybridize therewith. Therefore, in a further aspect of this invention, there is provided a method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, wherein the method comprises performing an amplification reaction using primers, characterized in that at least one primer is derived from the ACP described above. Preferably, the primer according to the structure of<br>
2 5 ACP is one having at its 3"end portion a hybridizing sequence substantially complementary to a region of the target nucleic acid sequence to hybridize therewith.<br>
In a specific embodiment of this method, there is provided a method using two stage amplifications for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, which comprises:<br>
(a) performing a first-stage amplification of the target nucleic acid sequence at a first annealing temperature comprising at least two cycles of primer annealing, primer extending and denaturing, using the primer pair of any one of the ACP described above each having at its<br><br>
3" end portion a hybridizing sequence substantially complementary to a region of the target<br>
nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to<br>
its target nucleotide sequence, whereby the amplification product of the target nucleotide<br>
sequence is generated; and<br>
5	(b) performing a second-stage amplification of the amplification product generated from<br>
step (a) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primers as used in step (a) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5" -end portion of the primers used in step (a), under conditions in which<br>
10 each primer anneals to the 3"- and 5"-ends of the amplification jroduct, respectively, whereby the amplification product is re-amplified.<br>
Where the template for amplification is mRNA, the production of cDNA is required prior to amplification. Therefore, in still further aspect of this invention, there is provided a method for selectively amplifying a target nucleic acid sequence from an mRNA, wherin the method<br>
15 comprises reverse transcribing the mRNA and performing an amplification reaction using primers, characterized in that at least one primer is derived from the ACP descibed above. Preferably, the primer according to the structure of ACP is one having at its 3"end portion a hybridizing sequence substantially complementary to a region of the target nucleic acid sequence to hybridize therewith.<br>
20	In a specific embodiment of this invention, there is provided a method using two stage<br>
amplifications for selectively amplifying a target nucleic acid sequence from an mRNA which comprises:<br>
(a)	contacting the mRNA with an oligonucleotide dT primer which is hybridized to potyA<br>
tail of the mRNA under conditions sufficient for template driven enzymatic deoxyribonucleic<br>
2	5      acid synthesis to occur;<br>
(b)	reverse transcribing the mRNA to which the oligonucloetide dT pirmer hybridizes to produce a first DNA strand that is complementary to the mRNA to which the oligonucloetide dT pinner hybridizes;<br>
(c)	performing a first-stage amplification of the target nucleic acid sequence from the first<br>
3	0      DNA strand obtained from step (b) at a first annealing temperature comprising at least two<br>
cycles of primer annealing, primer extending and denaturing, using the primer pair of ACP described   above  having  at   its   3"end  portion  a  hybridizing   sequence   substantially<br><br>
complementary to a region of the target nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to its target nucleotide sequence, whereby me amplification product of the target nucleotide sequence is generated; and<br>
(d) performing a second-stage amplification of the amplification product generated from 5 step (c) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primers as used in step (c) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5" -end portion of the primers used in step (c), under conditions in which each primer anneals to the 3"- and 5"-ends of the amplification product, respectively, whereby 10      the amplification product is re-amplified.<br>
Since the amplification methods of this invention employs the ACP of this invention, the common descriptions between them are omitted in order to avoid the complexity of this • specification leading to undue multiplicity.<br>
15	This application using ACP of the subject invention can provide an improved method for<br>
selectively amplifying a target nucleic acid sequence from a nucleic acid or a mixture of nucleic acids (DNA or mRNA) by performing nucleic acid amplifications, preferably, PCR. Since the effect of ACP provides the conventional primers with primer annealing specificity regardless of "primer search parameters" for primer design such as primer length, annealing<br>
2 0 temperature, GC content and product length, it is particularly recommended to use the ACP when the conventional primers used to amplify a target nucleic acid fragment are too sensitive to such parameters to generate specific nucleic acid amplification products.<br>
A schematic representation for selectively amplifying a target nucleic acid of double-stranded DNA using novel ACP system as described above is illustrated in FIG. 1A. FIG. IB<br>
2 5 illustrates a schematic representation for selectively amplifying a target nucleic acid of mRNA using novel ACP system. Referring to FIGs. 1A and IB, the present methods will be described in more detail.<br>
The present methods for amplifying a nucleic acid sequence may be carried out in accordance with various primer-based nucleic acid amplifications known in the art. Preferably,<br>
30 the methods are carried out according to the two stage amplifications developed by the present inventor, more preferably, the amplification is performed by polymerase chain reaction known in the art and most preferably, hot start PCR method.<br><br>
The methods of the present invention, for amplifying a nucleic acid sequence can be used to amplify any desired nucleic acid molecule. Such molecules may be either DNA or RNA. The molecule may be in either a double-stranded or single-stranded form, preferably, double-stranded. Where the nucleic acid as starting material is double-stranded, it is preferred to render 5 the two strands into a single-stranded, or partially single-stranded, form. Methods known to separate strands includes, but not limited to, heating, alkali, formamide, urea and glycoxal treatment, enzymatic methods (e.g., helicase action) and binding proteins. For instance, strand separation can be achieved by heating at temperature ranging from 80°C to 105°C. General methods for accomplishing this treatment are provided by Joseph Sambrook, et al., Molecular<br>
10 Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001).<br>
Where a mRNA is employed as starting material for amplification, a reverse transcription step is necessary prior to amplification, details of which are fount! in Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring<br>
15 Harbor, N.Y.(2001); and Noonan, K. F. et al., Nucleic Acids Res. 16:10366 (1988)). For reverse transcription, an oligonucleotide dT primer hybri&amp;sable to poly A tail of mRNA is used. The oligonucleotide dT primer is comprised of dTMPs, one or more of which may be replaced with other dNMPs so long as the dT primer can serve as primer. Reverse transcription can be done with a reverse transcriptase that has RNase H activity. If one uses an enzyme having RNase H<br>
2	0     activity, it may be possible to omit a separate RNase H digestion step, by carefully choosing the<br>
reaction conditions.<br>
The present methods do not require that the molecules to be amplified have any particular sequence or length. In particular, the molecules which may be amplified include any naturally occurring procaryotic, eukaryotic (for example, protozoans and parasites, fungi, yeast, higher 25 plants, lower and higher animals, including mammals and humans) or viral (for example, Herpes viruses, HTV, influenza virus, Epstcin-Barr virus, hepatitis virus, polio virus, etc.) or viroid nucleic acid. The nucleic acid molecule can also be any nucleic acid molecule which has been or can be chemically synthesized. Thus, the nucleic acid sequence may or may not be found in nature.<br>
3	0	The ACP used for the present invention is hybridized or annealed to a region on template<br>
so that double-stranded structure is formed. Conditions of nucleic acid hybridization suitable for forming such double stranded structures are described by Joseph Sambrook, et al.,<br><br><br>
Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001) and Haymes, B. D., et al., Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, D.C. (1985). The sequence of the 3" -end portion of ACP needs not to exhibit precise complementarity, but need only to be substantially complementary 5 in sequence to be able to form a stable double-stranded structure. Thus, departures from complete complementarity are permissible, so long as such departures are not sufficient to completely preclude hybridization to form a double-stranded structure. Hybridization of ACP to a region on template nucleic acid is a prerequisite for its template-dependent polymerization with polymerases. Factors (see Joseph Sambrook, et al., Molecular Cloning, A Laboratory<br>
10 Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001); and Haymes, B.D., et. al., Nucleic Acid Hybridization, A Practical Approach, IRL P/*JS, "* rL: aington, D.C.(1985)) which affect the base pairing of ACP to its complementary nucleic acids subsequently affect priming efficiency. The nucleotide composition of ACP can affect the temperature at which annealing is optimal and therefore can affect its priming efficiency.<br>
15	A variety of DNA polymerases can be used in the amplification step of the present<br>
methods, which includes "Klenow" fragment of E. coli DNA polymerase I, a thermostable DNA polymerase and bacteriophage T7 DNA polymerase. Preferably, the polymerase is a thermostable DNA polymerase such as may be obtained from a variety of bacterial species, including Thermus aquaticus CTaq), Thermus thermophilus (Tth), Ihermus filiformis, Thermis<br>
2 0 flavus, Thermococcus lUeralis, and Pyrococcus juriosus (Pfu). Many of these polymerases may be isolated from bacterium itself or obtained commercially. Polymerase to be used with the subject invention can also be obtained from cells which express high levels of the cloned genes encoding the polymerase. When a polymerization reaction, is being conducted, it is preferable to provide the components required for such reaction in excess in the reaction vessel. Excess in<br>
2	5      reference to components of the amplification reaction refers to an amount of each component<br>
such that the ability to achieve the desired amplification is not substantially limited by the concentration of that component. It is desirable to provide to the reaction mixture an amount of required cofactors such as Mg2*, and dATP, dCTP, dGTP and dTTP in sufficient quantity to support the degree of amplification desired.<br>
3	0	All of the enzymes used in this amplification reaction may be active under the same<br>
reaction conditions. Indeed, buffers exist in which all enzymes are near their optimal reaction<br><br>
conditions. Therefore, the amplification process of the present invention can be done in a single reaction volume without any change of conditions such as addition of reactants.<br>
It would be understood that the 5" -end portions of a set of ACPs used in the step of the first- stage amplification could comprise identical or different sequences; if they are identical, 5 one primer corresponding to the sequence of 5" -end portion will be used in the step of the second-stage amplification, whereas if they are different, two primers each corresponding to the sequence of each 5"-end portion of ACPs will be used in the step of the second-stage amplification.<br>
The present invention includes an alternative process for selectively amplifying a target<br>
10 * ucleic ac;d fragment from a nucleic acid or a mixture using ACP, wherein a set of primers ^om: " i ng an ACP and a conventional primer can be used in the first amplification step, instead of a set of ACP. The term "conventional primer" used herein refers to any primer having a structure different from ACP, especially, in terms of the presence of the regulator portion containing universal base. In this case, the conventional primer is added only the first<br>
15 amplification step with the ACP and only one pre-selected arbitrary primer corresponding to the 5" -end portia \ sequence of the ACP is added in the second amplification step. In preferred embodiment, the alternative process can be used when each 3" -portion of a pair of ACP to be used in the first amplification step has different melting temperature (Tm). "Tm" refers to the temperature at which half the primers are annealed to the target region.<br>
2 0	Two amplification steps of the present methods (in case of amplification from mRNA,<br>
including reverse transcritptation) are separated only in time. The first-stage amplification should be followed by the second-stage amplification. It would be understood mat the first-stage amplification reaction mixture could include the primers corresponding to the 5" -end portion which will be used to anneal to the sequences of the 5" -end portions of the ACPs in the<br>
2	5      second-stage amplification, which means that the primers corresponding to the 5" -end portion<br>
can be added to the reaction mixture at the time of or after the first-stage amplification step.<br>
As an alternative process, in the second-stage amplification step the complete sequences of the ACPs used in the first-stage amplification step, instead of the primers corresponding to the 5" -end portions of the ACPs, can be used as primers at the high stringent conditions for re-<br>
3	0      amplifying the product generated from the first-stage amplification step, wherein the 3" - and<br>
5" - ends of the product from the first amplification step which is generated from annealing and extension of the 3" -end portion sequence of the set of ACP to the template nucleic acid at the<br><br><br><br><br><br><br><br><br>
I<br><br>
low stringent conditions comprise the sequence or complementary sequence of ACP and also serve as perfect paring sites to the set of ACP. In this view, this alternative process is preferred because mis need not further add the primers corresponding to the 5" -end portions of the ACPs to the reaction mixture at the time of or after the first-stage amplification step. FIG. 1A also 5 illustrates a schematic representation for selectively amplifying a target nucleic acid by the alternative process stated above.<br>
Annealing or hybridization in the present methods is performed under stringent conditions that allow for specific binding between a nucleotide sequence and ACP. Such stringent conditions for annealing will be sequence-dependent and varied depending on environmental<br>
10 parameters. In the present methods, the seco&gt; ."-sfrge amplification is generally performed under higher stringent conditions than the fir,st stage «o/ jlification.<br>
In a preferred embodiment, the first annealing temperature ranges from about 30°C to 68°C for the first-stage amplification step, more preferably, 40°C to 65°C. It is preferred that the second annealing temperature ranges from about 50°C to 72°C for the second-stage<br>
15 amplification. According to a more preferred embodiment, the first annealing temperature is equal to or lower than the second annealing temperature. The lei gth or melting temperature (T„,) of the 3" -end portion sequence of ACP will determine the annealing temperature for the first-stage amplification. For example, in case that ACP comprises 10 arbitrary nucleotides at the 3" -end portion, preferably, the annealing temperature will be about between 45°C and 55°C<br>
2 0      for the first-stage amplification.<br>
According to the present methods, the first-stage amplification under low stringent conditions is carried out for at least 2 cycles of annealing, extending and denaturing to improve the specificity of primer annealing during the first-stage amplification, and through the subsequent cycles, the second-stage amplification is processed more effectively under high<br>
25 stringent conditions. The first-stage amplification can be carried out up to 30 cycles. In a preferred embodiment, the first-stage amplification is carried out for 2 cycles. Jh another embodiment, the second-stage amplification under high stringent conditions is carried out for at least one cycle (preferably, at least 5 cycles) and up to 45 cycles to amplify the first-stage product, m a more preferred embodiment, the second-stage amplification is carried out for 25-<br>
30 35 cycles. High and low stringent conditions may be readily determined from the standard known in the art. "Cycle" refers to the process which results in the production of a copy of<br><br><br><br><br><br><br>
target nucleic acid. A cycle includes a denaturing step, an annealing step, and an extending step.<br>
In the most preferable embodiment, the amplification is performed in accordance with<br>
PCR which is disclosed in U.S. Pat. Nos. 4,683,195,4,683,202, and 4,800,159.<br>
5	According to a preferred embodiment, when the first-stage amplification is carried out, the<br>
3" -end portion of the primer pair of ACP is involved in annealing at the first annealing temperature and when the second-stage amplification is carried out, the 5" -end portion of the primer pair serves as a priming site. Such alteration of the portion to involve in annealing is mainly ascribed to the ACP itself, in particular, the regulator portion of ACP. In the present<br>
10 methods, the regulator portion of ACP is capable of restricting the annealing portion of ACP to its 3"-end portion at the first annealing temperature, responsible for improving annealing specificity to a target sequence.<br>
The present methods may be combined with many other processes known in the art to achieve a specific aim. For example, the isolation (or purification) of amplified product may<br>
15 follow the second-stage amplification. This can be accomplished by gel electrophoresis, column chromatography, affinity chromatography or hybridization. In addition, the amplified product of this invention may be inserted into suitable vehicle for cloning. Furthermore, the amplified product of this invention may be expressed in suitable host harboring expression vector. In order to express the amplified product, one would prepare an expression vector mat<br>
2 0 carries the amplified product under the control of, or operatively linked to a promter. The promoter is originated from the vector itselt or the end portion of the amplified product, which may correspond to 5" -end portion of the ACP. Many standard techniques are available to construct expression vectors containing the amplified product and transcriptional/translational/control sequences in order to achieve protein or peptide expression<br>
25 in a variety of host-expression systems. The promoter used for prokaryotic host includes, but not limited to, phX promoter, trp promoter, lac promoter and T7 promoter. The promoter used for eukaryotic host includes, but not limited to, metallothionein promoter, adenovirus late promoter, vaccinia virus 7.5K promoter and the promoters derived from polyoma, adenovirus 2, simian virus 40 and cytomegalo virus. Certain examples of prokaryotic hosts are E. coli,<br>
30 Bacillus subtilis, and other enterobacteriaceae such as Salmonella typhimurium, Serratia marcescens, and various Pseudomonas species. In addition to microorganisms, cultures of cells derived from multicellular organisms may also be used as hosts. In principle, any such cell<br><br><br><br><br><br><br>
culture is workable, whether from vertebrate or invertebrate culture. In addition to mammalian cells, these include insect cell systems infected with recombinant virus expression vectors (e.g., baculovirus); and plant cell systems infected with recombinant virus expression vectors (e.g., cauliflower mosaic virus, tobacco mosaic virus) or transformed with recombinant plasmid 5 expression vectors (e.g., Ti plasmid) containing one or more coding sequences. The expressed polypeptide from the amplified product may be generally purified with a variety of purposes in accordance with the method known in the art.<br>
In another aspect of this invention, mere is provided a kit for nucleic acid amplification of the instant invention described previously, which comprises the annealing control primer or<br>
10      annealing control primer set indicated above.<br>
IP ^ ".ill L n
In further aspec; of this invention, there is provided a kit for selective amplification of a<br>
15 target nucleic acid sequence from mRNA described previously, which comprises the annealing control primer or annealing control primer set indicated above.<br>
According to one embodiment of this invention, these kits further comprises a primer or a primer pair each having a nucleotide sequence corresponding to the 5" -end portion of the ACP; in case that the 5" -end portion comprises universal primer sequence, it is more preferred that<br>
2 0 the kit comprises the universal primers. The present kits may optionally include the reagents required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5, -triphosphates. Optionally, the kits may also include various polynucleotide molecules, reverse transcriptase, various buffers and reagents, and antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary<br>
2 5 for performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or compartments the constituents afore-described.<br>
30      II. Application to Multiplex DNA Amplification<br>
This application using ACP of the subject invention can also provide an improved method for amplifying more than one target sequence using more than one pair of primers in the same<br><br><br>
reaction. In general, it is extremely difficult to set up PCR conditions to amplify more than 10 targets in parallel because an optimal PCR reaction is required to amplify even one specific locus without any unspecific by-products, so mat those researchers who have achieved multiplex PCR have had to work hard to optimize their systems. Since annealing needs to take 5 place at a sufficiently high temperature to allow the perfect DNA-DNA matches to occur in the reaction, the ACP of the subject invention is ideal in the optimization of multiplex DNA amplification due to its function of improving the specificity of amplification. "Multiplex PCR" as used herein refers to the simultaneous amplification of multiplex DNA targets in a single polymerase chain reaction (PCR) mixture.<br>
10	In still further aspect of this invention, there is provided a method for amplifying more<br>
than one target nucleotide sequence simultaneously v-»» ig r. or. than one pair of primers in the same reaction, wherein the method comprises performing an amplification reaction using primers, characterized in that at least one primer is derived from any one of ACP described above. Preferably, the primer according to the structure of ACP is one having at its 3"end<br>
15      portion a hybridizing sequence substantially complementary to a region of the target nucleic acid sequence to hybridize therewith.<br>
hi a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	performing a first-stage amplification of more than one target nucleotide sequence at<br>
20      a first annealing temperature comprising at least two cycles of primer annealing, primer<br>
extending and denaturing, using the primer pairs of any one of ACP above in which its 3" end portion each of the primer pairs has a hybridizing nucleotide sequence substantially complementary to a region of the target nucleic acid sequence to hybridize therewith, under conditions in which each of each primer pair anneals to its target nucleotide sequence, whereby 25      the amplification products of target nucleotide sequences are generated; and<br>
(b)	performing a second-stage amplification of the amplification products generated<br>
from step (a) at a second annealing temperature, which is high stringent conditions, comprising<br>
at least one cycle of primer annealing, primer extending and denaturing, using the same primer<br>
pairs as used in step (a) or primer pairs each comprising a pre-selected arbitrary nucleotide<br>
30      sequence corresponding to each 5"-end portion of the primer pairs used in step (a), under conditions in which each of each primer pair anneals to the 3"- and 5"-end sequences of the<br><br>
amplification products generated from step (a), respectively, whereby the amplification products are re-amplified in the same reaction.<br>
Since this application using the ACP of this invention is carried out in accordance with the<br>
present methods for amplification of nucleic acid sequence previously discussed, except for<br>
5      using more than one target nucleotide sequence and primer pairs, the common descriptions<br>
between them are omitted in order to avoid the complexity of mis specification leading to<br>
undue multiplicity.<br>
For instance, the composition and structure of ACP used and the conditions for amplification, are common between this process and the present methods for amplification of 10      nucleic acid sequence previously discussed.<br>
In a preferred embodiment, the amplified products from each of target nucleotide sequences are different in size for subsequent analysis.<br>
According to a preferred embodiment, the amplification products of multiplex target nucleotide sequences may be analyzed through size separation. The size separation comparison is performed using a variety of method known in the art, such as electrophoresis through a polyacrylamide gel matrix or agarose gel matrix and nucleotide sequencing. The nucleotide sequencing may be rapidly carried out with an automatic sequencer available from various manufacturor.<br>
As exemplified in Example below, the ACP of this invention permits the final amplified products to be free from the background problems as well as non-specificity arising from the conventional primers used in multiplex nucleic acid amplification methods known in the art.<br>
The advantage of the multiplex amplification is that numerous diseases or specific 15      nucleotide sequence alterations (e.g., single nucleotide polymorphism or point mutation) can be assayed in the same reaction.<br>
The number of analyses that can be run simultaneously is unlimited; however, the upper<br>
limit is probably about 20 and is likely to be dependent on the size difference required for<br>
resolution and methods mat are available to resolve the amplified product.<br>
20	In another aspect of this invention, there is provided a kit for amplifying more than one<br>
target nucleotide sequence simultaneously in the same reaction, which comprises the annealing control primer or annealing control primer set described above. According to one embodiment of this invention, these kits further comprises a primer or a primer pair having a nucleotide sequence corresponding to the 5"-end portion of the ACP; in case that the 5"-end portion<br><br>
comprises universal primer sequence, it is more preferred that the kit comprises the universal primers. The present kits may optionally include the reagents required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5"-triphosphates. Optionally, the kits may also include various 5 polynucleotide molecules, reverse transcriptase, various buffers and reagents, and antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary for performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or compartments 10     the constituents afor -describe i.<br>
The method rj&gt; J. kit " \ie present invention may be applied to the diagnosis of genetic and infectious diseases, gender determination, genetic linkage analysis, and forensic studies.<br>
III. Application to Identification of Differentially Expressed Genes<br>
This application using ACP of the subject invention can also provide an improved method for detecting and cloning cDNAs complementary to differentially expressed mRNAs in two or more nucleic acid samples.<br>
15	In still further aspect of this invention, there is provided a method for detecting DNA<br>
complementary to differentially expressed mRNA in two or more nucleic acid samples, wherein the method comprises reverse transcribing the mRNA and performing an amplification reaction using primers, characterized in that at least one primer is derived from any one of ACP described above. Preferably, the primer according to the structure of ACP is one having at its<br>
20      3"end portion a hybridizing sequence (more preferably, arbitrary sequence) substantially complementary to a region of cDNA strands generated from reverse transcription.<br>
In a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	providing a first sample of nucleic acids representing a first population of mRNA<br>
2 5     transcripts and a second sample of nucleic acids representing a second population of mRNA<br>
transcripts;<br>
(b)	separately contacting each of the first nucleic acid sample and the second nucleic<br>
acid sample with a first primer of any one of ACP described above, in which the 3" -end<br>
portion of the  first primer comprises a hybridizing nucleotide sequence  substantially<br><br><br>
complementary to a first site in the differentially expressed mRNA to hybridize therewith, under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(c) reverse transcribing the differentially expressed mRNA to which the first primer<br>
5 hybridizes to produce a first population of first cDNA strands that are complementary to the<br>
differentially expressed mRNA in the first nucleic acid sample to which the first primer<br>
hybridizes, and a second population of first cDNA strands that are complementary to the<br>
differentially expressed mRNA in the second nucleic acid sample to which the first primer<br>
hybridizes;<br>
10	(d)   purifying and quantifying each of the first and second populations of first cDNA<br>
strands;<br>
(e)	performing a first-stage amplification of each of the first and second population of<br>
first DNA strands obtained from step (d) at a first annealing temperature comprising at least<br>
one cycle of primer annealing, primer extending and denaturing, using a second primer of any<br>
15 one of ACP described above having at its 3" end portion a hybridizing sequence substantially complementary to a second site in the first and second populations of first cDNA strands, under conditions in which the second primer anneals to the second site in each population of the first cDNA strands, whereby first and second populations of second cDNA strands are generated;<br>
(f)	performing a second-stage amplification of each second cDNA strand generated from<br>
2	0      step (e) at a second annealing temperature, which is high stringent conditions, comprising at<br>
least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (b) and (e), respectively, or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5" -end portion of the first and second primers used in steps (b) and (e), respectively, under conditions in which each primer 25 anneals to the 3"- and 5"-end sequences of each second cDNA strand, respectively, whereby amplification products of the second cDNA strands are generated, and<br>
(g)	comparing the presence or level of individual amplification products in the first and<br>
second populations of amplification products obtained from step (f).<br>
Since this application using the ACP of this invention employs the present methods for<br>
3	0      amplification of nucleic acid sequence previously discussed, the common descriptions between<br>
them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br><br>
A schematic representation for identifying differentially expressed genes using novel ACP is illustrated in FIG. 2A.<br>
In the present method, the nucleic acid sample representing a population of mRNA transcripts can be obtained from a wide variety of biological materials. In general, the first 5 nucleic acid sample comprises mRNA expressed in a first cell and the second nucleic acid sample comprises mRNA expressed in a second cell. In particular, the first nucleic acid sample comprises mRNA expressed in a cell at a first developmental stage and the second nucleic acid sample comprises mRNA expressed in a cell at a second developmental stage. In addition, the first nucleic acid sample comprises mRNA expressed irt a tumorigenic cell and the second<br>
10     nucleic acid sample comprises mRNA expressed in a normal cell.<br>
Steps (e) and (f) of the subject application may occur in a single tube using the same reaction mixture except for primers, which means that steps (e) and (f) are separated only in time. It would be understood that the primers corresponding to the 5" -end portion could be added to the reaction mixture at the time of or after the second cDNA strand synthesis. In a<br>
15 preferred embodiment, the primers corresponding to the 5"-end portion are added to the reaction mixture right after step (e) is completed, followed by subsequent PCR amplification of sei ond cDNA strands.<br>
It would be also understood that the 5" -end portion sequences of the first and second ACPs used in steps (b) and (c), respectively, could be identical or different sequences; if they<br>
20 are identical, one primer corresponding to the sequence of 5" -end portion will be used in the step (f), whereas if they are different, two primers each corresponding to the sequence of each 5" -end portion of ACPs will be used in the step (f). In a preferred embodiment, the 5" -end portion sequences of the first and second ACPs used in steps (b) and (e) are different and thus, two primers each corresponding to the sequence of each 5" -end portion of ACPs are used in<br>
25      step(f).<br>
As an alternative process, in step (f) the complete sequences of the first and second ACPs used in steps (b) and (e), respectively, instead of the primers corresponding to the 5" -end portions of the ACPs, can be used as primers at the high stringent conditions for amplifying each second DNA strand obtained from step (e), wherein the 3" - and 5" -ends of the second<br>
3 0 DNA strands which are initially synthesized using the second ACP comprise the sequence of the first ACP and the complementary sequence of the second ACP, respectively, and also serve as perfect pairing sites to the first and second ACPs. In this view, this alternative process is<br><br>
preferred because there is no need to add the primers corresponding to the 5" -end portions of<br>
the ACPs to the reaction mixture at the time of or after first-stage PCR reaction. FIG. 2B<br>
illustrates a schematic representation for identifying differentially expressed genes by the<br>
alternative process stated above.<br>
5	The method of the subject application for detecting differences in gene expression uses<br>
only a single cDNA synthesis primer (the first ACP) to react with mRNA, unlike conventional Differential Display PCR which requires multiple cDNA synthesis anchor primers. In (he original differential display method outlined by Liang and Pardee in 1992, twelve anchor primers have been introduced. The anchor primers for example, having a sequence of TI2MN.<br>
10      where M is A, C, or G and N is A, 
. Recently, modified ancho; ?,rinv : lave been proposed by altering the number of nucleotides<br>
such as one or three instead of two at the 3" -end which can hybridize to a sequence that is<br>
immediately 5" to the poly A tail of mRNAs or by extending additional nucleotides at the 51-<br>
end while retaining the Oligo (dT)9.i2 MN tail resulting in at least 21 nucleotides in length<br>
15      (Villeponteau et al., 1996, Combates et al., 2000).<br>
The subject invention concerns the embo foments of the ACP used in this method for the identification of differentially expressed genes, wherein the first ACP used in step (b) is represented by the following general formula (2): 5" -dXp-dYq-dTr- 3" wherein dX is one of the four deoxyribonucleotides, A, C, G, or T; dY is a regulator portion comprising universal bases<br>
20 responsible for the main function of the ACP associated with alteration of annealing temperature during PCR; dT is a T deoxyribonucleotide; p, q, and r represent an integer, respectively, dXp represents the 5" -end portion and contains a pre-selected arbitrary nucleotide sequence; dYq contains at least 2 universal bases; dTr represents the 3" -end portion; the nucleotide sequence of the 3" -end portion should have lower Tm than that of the 5" -end<br>
25 portion. The fonnula (2) basically follows the rule of formula (1). The 3"-end portion of formula (2) consists of the sequences capable of annealing to the poly A tail of mRNA and serves as a cDNA synthesis primer for reverse transcription of mRNA.<br>
In a preferred embodiment, the 3" -end portion of the first ACP used in step (b) contains at least 6 T nucleotides in length, which is considered a minimal requirement of length for primer<br>
30 annealing. More preferably, the 3" -end portion sequence is from 10 to 20 T nucleotides and can be up to 30 T nucleotides in length. Most preferably, the 3" -end portion sequence is about 15 T nucleotides in length. This primer is named dT15 annealing control primer (dTiS-ACP). In<br><br>
a preferred embodiment, the first primer has a general formula of 5" -dXis.jo-dY2.io-dTio.io-3", wherein dX represents a deoxyribonucleotide and comprises a pre-selected arbitrary nucleotide sequence not substantially complementary to the first and second populations of mRNAs; dY represents the regulator portion comprising 2-10 universal bases or non-discriminatory base 5 analogs; and dT represents a contiguous deoxymymidine capable of annealing to the first site in the first and second populations of mRNAs.<br>
In one embodiment, the 3" -end portion of the first ACP used in step (b) may contain at least one additional nucleotide at the 3" -end that can hybridize to an mRNA sequence which is immediately upstream of the polyA tail. The additional nucleotides at the 3" end of the first<br>
10 ACP may be up to 3 in length. For example, dT may further comprise 3" -V at its 3" -end; in which V is one selected from the group consisting of deoxyadenoeifl;, d
15 and deoxyguanosine. Most preferably, the 3" -end portion sequence of the first ACP used in step (b) contains dTu only.<br>
In a preferred embodiment, the first entire ACP is about 40-45 nucleotides in length and comprises dT(J at the 3" -end portion, dXao.25 at the 5" -end portion and dYs between the 3" -and 5" -end portions. The first entire ACP can be up to 100 nucleotides in length. The first<br>
2 0      primer is exemplified by SEQ ID NOs: 30,39, 57 and 61-63.<br>
The first ACP described herein is hybridized to the poly A tail of the mRNA, which is present on all mRNAs, except for a small minority of mRNA. The use of the first ACP used in this invention results in only one reaction and produces only one cDNA population, in contrast to at least 3 to 64 separate cDNA populations generated by the conventional anchor primers of<br>
25 Differential Display technique. This greatly increases the efficiency of the method by generating a substantially standard pool of single-stranded cDNA from each experimental mRNA population.<br>
In the step (d), the standard pools of cDNAs synthesized by the first ACP should be purified and then quantitated by techniques well known to those of ordinary skill in the art such<br>
30 as spectrophotometry. This step is necessary to precisely control their inputs into the amplification step and then compare the final amplified poducts between two or more samples. Preferably, the amount of cDNA produced at this point in the method is measured. It is more<br><br><br><br><br><br>
preferred that this determination is made using ultraviolet spectroscopy, although any standard procedure known for quantifying cDNA known to those of ordinary skill in the art is acceptable for use for this purpose. When using the UV spectroscopy procedure, an absorbance of about 260 nm of UV light advantageously is used. By the measurement of cDNA quantity at this step, 5 therefore, the cDNA quantity can be standardized between or among samples in the following amplification reaction.<br>
After synthesis of the first cDNA strands using the first ACP, the second cDNA strands are synthesized using the second ACP primer under low stringent conditions, by at least one cycle comprising denaturing, annealing and primer extension, wherein the resultant first cDNA<br>
\ 0      stnnds are used as templates.<br>
The second ACP basically follows the rule of formula (1) and its 3"-end portion comprises a short arbitrary sequence, which preferably has lower Tro than that of the 5" -end portion. This primer is nPtned an arbitrary annealing control primer (AR-ACP). In a preferred embodiment, the 3" -end portion of the second ACP can have from 8 to 15 nucleotides in<br>
15 length. Most preferably, the 3" -end portion of the second ACP contains about 10 nucleotides in length.<br>
According to a preferred embodiment, the second ACP has the general formula of 5" -dX15.30-dY2-10-dZs.15-3", wherein dX represents a deoxyribonucleotide and comprises a pre¬selected arbitrary nucleotide sequence not substantially complementary to the first and second<br>
20 populations of the first cDNA strands; dY represents the regulator portion comprising 2-10 universal bases or non-discriminatory base analogs; dZ represents a hybridizing arbitrary nucleotide sequence capable of annealing to the second site in the first and second populations of DNA strands. More preferably, the entire second ACP is about 40-45 nucleotides in length comprising dZio at the 3" -end portion, 6X20.25 at the 5" -end portion and dYs between the 3" -<br>
2 5 and 5" -end portions. The second entire ACP can be up to 100 nucleotides in length. The second primer is exemplified by SEQ ID NOs: 1-9, 13-18 and 20-23.<br>
The second ACP described herein is different from a so-called long arbitrary primer, as used in the known modified Differential Display technique. For example, the conventional long arbitrary primers as described by Villeponteau et al. (1996) and Diachenko et al. (1996), having<br>
30 at least 21 or 25 nucleotides in length, comprise of only arbitrary nucleotides in (lie entire sequences. These conventional long arbitrary primers will hybridize in a non-predictable way under the low annealing temperature (about 40°C) which is required to achieve arbitrary<br><br><br>
priming in the early PCR cycle, such that it is impossible to design a representative set of primers rationally. Furthermore, many of the bands represent the same mRNA due to the ,v Stickiness" of long primers when used under such a low stringency.<br>
The advantages of the present method for detecting differentially expressed genes are 5 predominantly ascribed to the use of the second ACP. Since the second ACP is designed to limit the annealing of the second ACP to its 3" -end portion sequence, not to its 5" -end portion sequence, in association with annealing temperature, the resultant annealing will come out in a predictable way, such that it is possible to design a representative set of primers rationally. In addition, the use of the second ACP allows avoiding false positive problems caused by the<br>
10 "Stickiness" of the conventional lor i primers, under low stringent conditions as used in the previous Differential Display tech J"^ue,<br>
The annealing temperature used for the synthesis of second DNA strands under low stringency conditions used in step (e) is preferably about between 40°C to 65°C, more preferably, about between 45°C and 55°C and the most preferably, about 50°C. However,<br>
15 unlike Differential Display, which uses annealing temperatures between 35°C and 45°C, the annealing temperature of low stringency conditions use*" in the subject application is relatively higher than those used in die known classical or enhanced Differential Display teda^ucc with arbitrary primers.<br>
Another unique and significant features  of the subject application for  detecting<br>
2 0 differentially expressed genes is to amplify only the initially synthesized second DNA brands by the subsequent amplification, wherein the 3" - and 5" -ends of the second Di !A brands which have been initially synthesized using the second ACP comprise the complementary sequence of the first ACP and the sequence of the second ACP, respectively and fiiur ■-, the entire sequences of the first and second ACPs, or only their 5" -end portion sequences of OK SIS" and<br>
2	5      second ACPs, are used as 3" and 5" primer sequences for die amplification of the second DNA<br>
strands.<br>
Since the ACP in die subject application leads to me amplification of specific products, it can be possible to fundamentally eliminate the cause of major bottleneck problems puoh as false products and poor reproducibility, which result from non-specific anneaUui of tne<br>
3	0      conventional arbitrary and dT primers to first and second DNA strands as well as to amplified<br>
products during PCR in the known Differential Display methods.<br><br>
In a preferred embodiment, the synthesis of second DNA strands in step (e) is carried out by at least 1 cycle of amplification under low stringent conditions to achieve arbitrary priming, and through the subsequent cycles, the amplification is processed more effectively for the amplification of the resultant second DNA strands under high stringent conditions used in step 5 (f). Most preferably, the synthesis of second DNA strands in step (e) is carried out by one cycle of amplification under low stringent conditions.<br>
In a preferred embodiment, the amplification of the resultant second DNA strands synthesized by the step (e) is carried out under high stringent conditions using the complete sequences of the first and second ACPs used in steps (b) and (e), respectively, as primer<br>
10 sequences, wherein the 3" - and 5" -ends of the resultant second DNA strands provide perfect pairing sites to the first and second ACPs. However, it is interesting that the first ? ^ &amp; &gt; J&amp; ACPs are not involved in any other annealing to the template nucleic acid, except the annealing and extension of the 3" - and 5" - ends of the second DNA strands as a reaction unit at such a high stringent condition because their 3"-end portions require relatively low annealing<br>
15 temperature and the high stringent conditions do not allow them to anneal to any site of the template, except the 3" - and 5" -ends of the second DNA strands. Consequently, owing to this function of ACP, which is capable of selectively annealing to the template in associated with annealing temperature, the amplified products can be free from the problems of the high false positive rate, poor reproducibility and possible undcr-representation of minor mRNA fractions<br>
20 in the analysis which are the main problems of the known Differential Display. La this view, there is a significant difference between this subject method and the conventional Differential Display methods despite the fact that they are in common to use the same primers for high stringent conditions as well as for low stringent conditions.<br>
In a preferred embodiment, the annealing temperature of the amplification for high<br>
25 stringent conditions used in step (f) is preferably about between 5S0C and 72°C. Most preferably, the annealing temperature used for the high stringent conditions is about 65-68°C.<br>
In a preferred embodiment, the amplification under high stringent conditions used step (f) is carried out by at least 10 cycles and up to 50 cycles to amplify the resultant second DNA strands synthesized by step (e) during PCR Most preferably, the PCR amplification is carried<br>
3 0      out by 40-45 cycles.<br>
The second-strand cDNA is preferably synthesized by PCR, more preferably, hot start PCR method in which the procedure is to set up the complete reactions without the DNA<br><br>
polymerase and incubate the tubes in the thermal cycler to complete the initial denaturation step at &gt;90°C. Then, while holding fee tubes at a temperature above 70°C, the appropriate amount of DNA polymerase can be pipetted into the reaction. In a preferred embodiment, the addition of the primers for the second-stage amplification into the reaction mixture after the complete 5 reaction of the second-strand cDNA synthesis is also carried out under denaturation temperature such as &gt;90°C. Then, while holding the tubes at a temperature about 90 °C, the appropriate amount of the primers for the second-stage amplification can be pipetted into the reaction.<br>
An example of the second DNA strand synthesis and the subsequent amplification of the<br>
10 rest tant second DNA strands in a single tube using the pre-selected arbitrary sequence of the f" end v-w-"ions of the first and second ACPs is conducted under the following conditions: the second DNA strands arc synthesized under low stringent conditions by one cycle of the first-stage amplification comprising annealing, extending and denaturing reaction; the reaction mixture containing the first-strand cDNA, PCR reaction buffer (e.g., available from Roche),<br>
15 dNTP, and the second ACP is pre-heated at about 94°C, while holding the tube containing the reaction mixture at ab^ut 94°C, Taq polymerase (e.g., available from Roche) is added into the reaction mixture; the PCR reactions are as follows: one cycle of 94°C for 1 min, 50°C for 3 min, and 72°C for 1 min; followed by denaturing the amplification product at 94°C; after the complete reaction of the second DNA strand synthesis in step (e), 5" pre-selected arbitrary<br>
20 primer and 3" pre-selected arbitrary primer are added to the reaction mixture and then the second stage amplification is conducted as follows: 40 cycles of 94°C for 40 sec, 68°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final extension at 72°C.<br>
An alternative example of the second DNA strand synthesis and the subsequent amplification of the resultant second DNA strands in a single tube using the complete<br>
2	5      sequences of the first and second ACPs used in steps (b) and (e), respectively, instead of the<br>
pre-selected arbitrary sequences of the 5" -end portions of the first and second ACPs, is conducted under the following conditions: the second DNA strands are synthesized under low stringent conditions by one cycle of the first-stage amplification comprising annealing, extending and denaturing reaction; Ihe reaction mixture containing the first-strand cDNA, PCR<br>
3	0      reaction buffer (e.g., available from Roche), dNTP, the first ACP (dTi5-ACP), and the second<br>
ACP (AR-ACP) is pre-heated at about 94°C, while holding the tube containing the reaction mixture at about 94°C, Taq polymerase (e.g., a mailable from Roche) is added into the reaction<br><br><br>
mixture; the PCR reactions are as follows: one cycle of 94°C for 1 min, 50°C for 3 min, and<br>
72°C for 1 min; followed by the second-stage PCR amplification comprising annealing,<br>
extending and denaturing reaction; the PCR reactions are as follows: 40 cycles of 94°C for 40<br>
sec, 65°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final extension at 72°C.<br>
5	It should be noted that a proper concentration of arbitrary ACP (the second ACP) is used<br>
to synthesize the second-strand cDNAs by one cycle of the first-stage amplification. If the amount of the second ACP used in the step (e) is too low, the resultant amplified products are not reproducible. In contrast, the excess amount of the second ACP used in the step (e) generates backgrounds such as DNA smear during PCR. In a preferred embodiment, the<br>
10 concentration of the second ACP used in the ste -. (e) is aUmt between 0.1 pM and 1.0 uM. Most preferably, the concentration of the secorj .ACP - well as the first ACP is about 0.2 uM. In a preferred embodiment, the concentration of the primers used in the step (f) is about between 0.1 uM and 1 uM, most preferably, about 0.4 uM.<br>
Another significant feature of the subject application to the identification of differences in<br>
15 gene expression is the use of high annealing temperature in a method. High annealing temperature used in step (f) increases the specificity of primer am. waling during PCR, which results in eliminating false positive products completely and increasing reproducibility. Freedom from false positives which is one "major bottleneck remaining for the previous Differential Display technique is especially important in the screening step for the verification<br>
20      of the cDNA fragments identified by Differential Display.<br>
The step of comparing the presence or level of amplification products obtained from step (f) may be performed in accordance with various methods known in the art. In a preferred embodiment, each of the first and second populations of amplification products of step (f) are resolved by electrophoresis to identify differentially expressed mRNAs. More preferably, the<br>
25 resultant PCR cDNA fragments are detected on an ethidium bromide-stained agarose gel. Another prominent feature of this subject application is the use of ethidium bromide-stained agarose gel to identify differentially expressed mRNAs. In general, the conventional Differential Display methods use radioactive detection techniques using denaturing polyacrylamide gels. However, according to the present method, the significant amount of the<br>
30 amplified cDNA fragments obtained through two stage amplifications allows to use an ethidium bromide-stained agarose gel to detect the amplified cDNAs, which results in increasing the speed and avoiding the use of radioactivity.<br><br>
Alternatively, the resulting cDNA fragments can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et aL, 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
In another embodiment, it might be useful for diagnostic purposes to use an automatic system such as an automatic DNA sequencer together with any distinct labeling of the ACPs to detect or analyze the amplified products (Bauer, et al., 1993).<br>
Considering the features of ACP in this subject application, the present method for detecting and cloning differentially expressed genes differs fundamentally from the previour Differential Display techniques as described above.<br>
In conclusion, the use of the ACP in this method makes it possible to allow the amplification of only second DNA strands and the use of the sufficient amount of starting materials as well as the high concentration of dNTP, resulting in the following benefits: a) increasing primer annealing specificity, b) eliminating the problem of false positives which requires the subsequent labor-intenrive work to verify true positives, c) improving reliability and reproducibility, d) detecting rare mRNAs, e) generating long-distance PCR products ranging in size from 150 bp to 2.0 kb, f) allowing the use of ethidium bromide-stained agarose gel to detect products, g) increasing the speed of analysis, h) particularly, not requiring well-trained hands to conduct this method, and i) allowing the rational design of a representative set of primers.<br>
In further aspect of this invention, there is provided a kit for detecting DNA complementary to differentially expressed mRNA, which comprises the annealing control primer or annealing control primer set described above (the first and second primer). According to one embodiment of this invention, these kits further comprises a primer or a primer pair having a nucleotide sequence corresponding to the 5" -end portion of the ACPs; in case that the 5" -end portion comprises universal primer sequence, it is more preferred that the kit comprises the universal primers. The present kits may optionally include the reagents required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5" -triphosphates. Optionally, the kits may also include various polynucleotide molecules, reverse transcriptase, various buffers and reagents, and antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary for<br><br>
performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or compartments the constituents afore-described.<br>
IV. Application to Rapid Amplification of cDNA Ends fRACE"t<br>
This application using the ACP of the subject invention can provide an improved method<br>
for rapidly amplifying cDNA ends, so called RACE technologies. To be specific, the ACP of<br>
the subject application is adapted to the RACE technologies related to either of 3" - and 5" -end,<br>
5      and eliminates the background problems resulting from the primers used in the conventional<br>
RACE techn, i &gt;gie ■..<br>
In still further aspect of this invention, there is provided a method for rapidly amplifying a target cDNA fragment comprising a cDNA region corresponding to the 3" -end region of an mRNA, wherein the method comprises reverse transcribing said mRNA and performing an 10 amplification reaction using primers, characterized in that at least one primer is derived from any one of ACPs described above. Preferably, the primer according to the structure of ACP is one having at its 3" -end portion t gene-specific hybridizing nucleotide sequence substantially complementary to a site in cDNA generated from reverse transcription and/or one having at its 3" -end portion a hybridizing nucleotide sequence substantially complementary to poly A tails 15      of the mRNAs.<br>
In a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	contacting mRNAs with a first primer of any one of the ACP described above, in<br>
which the 3"-end portion of the primer comprises a hybridizing nucleotide sequence<br>
2 0      substantially complementary to poly A tails of the mRNAs to hybridize therewith, under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing the mRNAs to which the first primer hybridizes to produce a<br>
population of first cDNA strands that are complementary to the mRNAs to which the first<br>
primer hybridizes;<br>
2 5	(c)   performing a first-stage amplification of the first cDNA strands at a first annealing<br>
temperature comprising at least one cycle of primer annealing, primer extending and denaturing, using a second primer of any one of the ACP described above having at its 3" -end<br><br><br>
portion a gene-specific hybridizing nucleotide sequence substantially complementary to a site<br>
in one of the first cDNA strands to hybridize therewith, under conditions in which the second<br>
primer anneals to a gene-specific site on one of me first cDNA strands, whereby a gene-specific<br>
second cDNA strand is generated; and<br>
5	(d)   performing a second-stage amplification of the gene-specific second cDNA strand<br>
generated from step (c) at a second annealing temperature, which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (a) and (c), respectively, or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5" -end portion<br>
10 of the first and second primers used in steps (a) and (c), re r ec^vely, under conditions in which each primer anneals to the 3" - and 5" -end sequences of a gene pecific second cDNA strand, respectively, whereby an amplification product of a gene-specific cDNA strand is generated.<br>
Since this application using the ACP of this invention employs the present methods for amplification of nucleic acid sequence previously discussed, the common descriptions between<br>
15 them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br>
A schematic representation for amplifying a target cDNA fragment comprising 3" -end region corresponding to the 3" -end of mRNA using novel ACP system, called as ACP-based 3" RACE, is illustrated in FIG. 3.<br>
2 0	Steps (c) and (d) of the subject application may occur in a single tube using the same<br>
reaction mixture except for primers, which means that steps (c) and (d) are separated only in time. It would be understood that the primers corresponding to the 5" -end portion could be added to the reaction mixture at the time of or after second cDNA strand synthesis. In a preferred embodiment, the primers corresponding to the 5"-end portion are added to the<br>
2	5     reaction mixture right after step (2) is completed, followed by subsequent amplification of<br>
second cDNA strands.<br>
As an alternative process, in step (d) the complete sequences of the first and second ACPs, instead of the primers corresponding to the c" -end portions of the first and second ACPs, can be used as 3" and 5" primers for amplifying the second-strand cDNA obtained from step (c),<br>
3	0     wherein the 3" - and 5" -ends of the second-strand cDNA which are initially synthesized using<br>
the second ACP comprise the complementary sequence of the first ACP and the sequence of the second ACP, respectively, and also serve as perfect pairing sites to the first and second<br><br>
ACPs. FIG. 3 also illustrates a schematic representation for amplifying a target cDNA fragment comprising 3" -end region corresponding to the 3" -end of mRNA by the alternative process stated above.<br>
One of significant features of the present invention for 3" -RACE is that the first ACP<br>
5      comprising nucleotide sequence substantially complementary to poly A tail of mRNA is used<br>
as a cDNA synthesis primer and then the resultant cDNAs are directly used as templates for<br>
subsequent amplification without any additional purification steps to remove the oDNA<br>
synthesis primer.<br>
The annealing of the first ACP to the templates will be interrupted during subsequent by<br>
10 the effect of the regulator portion on the 3" - and 5" -end portions of the ACP under relatively high stringent conditions as described in the principle of ACP. As a result, the subject application to 3" -RACE simplifies the conventional RACE methods by reducing the step of purification and also, the ACP used in the subject application does not involve me background problems because the annealing of the 3" -end portion is specified by the presence of the<br>
15 regulator portion positioned between the 3"-and 5"-end portions in the ACPs, whereas the conventional cDNA synthesis primers such as Oligo-dT primers for 3"-RACE generate backgrounds during PCR, which is non-specific products. According to a preferred embodiment, the formula of the first ACP for the cDNA synthesis may be identical to the formula (2).<br>
2 0	When a gene-specific primer is used as 5" primer, the first amplification of a target cDNA<br>
fragment containing a 3"-end sequence in step (c) is carried out in accordance with conventional PCR methods as known in me art The term "gene-specific" in reference sequence used herein refers to a partial sequence of a specific gene or complement thereof that has been generally known or available to one skilled in the art. Therefore, (he gene- specific<br>
2	5     primer means one comprising the gene-specific sequence.<br>
The generated second cDNA strand is amplified by the second-stage amplification which is used in the application of the present invention for amplifying a target nucleic acid sequence above. Since the ACP described in this invention can generate stable Tm in a primer and also tolerate "primer search parameters" for primer design such as primer length, annealing<br>
3	0      temperature, GC content, and PCR product length, it is useful when the gene-specific primer<br>
sequences have low Tm or are too sensitive to such parameters to generate specific products.<br><br>
In another aspect of this invention, mere is provided a method for rapidly amplifying a target DNA fragment comprising a cDNA region corresponding to the 5" -end region of an mRNA, wherein the method comprises reverse transcribing the mRNA and performing an amplification reaction using primers, characterized in that at least one primer is derived from 5       any one of ACPs described above. Preferably, the primer according to the structure of ACP is one having at its 3" -end portion a gene-specific hybridizing nucleotide sequence substantially complementary to a site in cDNA generated from reverse transcription.<br>
In a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
10	(a)   contacting mRN \s with ?i oligonucleotide dT primer or random primer as a cDNA<br>
synthesis primer under c^adit :j sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur, in which the cDNA synthesis primer comprises a hybridizing nucleotide sequence substantially complementary to a region of an mRNA to hybridize therewith;<br>
15	(b)   reverse transcribing the mRNAs, using a reverse transcriptase, to which the cDNA<br>
synthesis primer hybridizes to produce a population of first cDNA strands that are complementary to the mRNAs to which the cDNA synthesis primer hybridiw-s, whereby mRNA-cDNA intermediates are generated;<br>
(c)	permitting cytosine residues to be tailed at the 3" -ends of the first cDNA strands in<br>
20      the form of the mRNA-cDNA intennediates by the terminal transferase reaction of reverse<br>
transcriptase;<br>
(d)	contacting the cytosine tails at the 3" -ends of the first cDNA strands generated from<br>
step (c) with an oligonucleotide which comprises a 3"-end portion and a 5"-end portion<br>
separated by a group of universal base or non-discriminatory base analog, wherein the. 3" end<br>
25 portion comprises at least three guanine residues at its 3"-end to hybridize with the cytokine tails at the 3" -ends of the first cDNA strands and the 5" -end portion comprises a pre.-sek.oted arbitrary nucleotide sequence, under conditions in which the 3-end portion of the oligonucleotide is hybridized to the cytosine tails;<br>
(e)	extending the tailed 3" -ends of the first cDNA strands to generate an additional<br>
30      sequence complementary to the oligonucleotide using reverse transcriptase, in which the<br>
oligonucleotide serves as a template in the extension reaction, whereby full-length first. cDNA strands are extended;<br><br>
(f) performing a first-stage amplification of the full-length first cDNA strands obtained from step (e) at a first annealing temperature comprising (i) and (ii) as follows:<br>
(i)   at least one cycle of primer annealing, primer extending and denaturing using a<br>
first primer comprising a nucleotide sequence substantially complementary to the 3" -<br>
5	end sequences of the full-length first cDNA strands under conditions in which the<br>
first primer anneals to the full-length first cDNA strands, under conditions in which the first primer anneals to the 3" - ends of the full-length first cDNA strands, whereby full-length second cDNA strands are generated;<br>
(ii)   at least one cycle of primer annealing, primer extending and denaturing using a<br>
10	second primer of any one of claims 1-25 having at its 3" -end portir a a gene ^specific<br>
hybridizing sequence substantially complementary to a region OT. on( ■&gt;" the full-<br>
length second cDNA strands to hybridize therewith, under conditions in which the<br>
second primer anneals to a gene-specific site on one of the full-length second cDNA<br>
strands, whereby a gene-specific cDNA strand is generated; and<br>
15	(g)   performing a second-stage amplification of the gene-specific cDNA strand at a<br>
second annealing temperature, which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (fHO m&amp; (0-00, respectively, or a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the first and second primers as<br>
2	0     used in steps (f)-(i) and (f)-(ti)&gt; respectively, under conditions in which each primer anneals to<br>
the 3" - and 5" -end sequences of a gene-specific cDNA strand, respectively, whereby an amplification product of a gene-specific cDNA strand is generated.<br>
The schematic representations for amplifying a target cDNA fragment comprising 5" -end region corresponding to the 5" -end of mRNA using novel ACP system, called as ACP-based 25      5" RACE, is illustrated in FIGs. 4A (using oligonucleotide dT primer) and 4B (using random primer).<br>
The descriptions of the oligonucleotide dT primer used in step (a) is identical to those used in the present method for amplification of a target nucleic acid sequence from an mRNA. Alternatively, when the size of a target mRNA is so large that the reverse transcriptase falls off<br>
3	0      before reaching the 5" complete sequences, the random primer is used as cDNA synthesis<br>
primer.<br><br>
According to a preferred embodiment, the step (c), permitting cytosine residues to be tailed is performed in the presence of manganese ion.<br>
Steps (f) and (g) of the subject application may occur in a single tube using the same reaction mixture except for primers, which means that steps (f) and (g) are separated only in 5 time. It would be understood that the priraer(s) used in each step (f) and (g) can be added to the reaction mixture at the time of or after each step. In a preferred embodiment, the primer(s) is(are) added to the reaction mixture right after each step is completed, followed by subsequent PCR amplification of second cDNA strands.<br>
When a gene-specific primer is used as 5" primer, the amplification of a target cDNA<br>
10 fragment containing a 5" -end sequence in step (f) is carried out under high stringent conditions i".i ace :: race with conventional PCR methods as known in the art.<br>
In a preferred embodiment, a target cDNA fragment containing a 5" -end sequence in step (f) is amplified using a second AC° comprising a gene-specific sequence at the 3" -end portion, by two stage FCR amplifications which are used in the application of the present invention for<br>
15 amplifying a target nucleic acid sequence above. Since the ACP described in this invention can generate stable Tm in a primer and also tolerate "primer search parameters" such as primer design, comprising primer length, annealing temperature, GC content, and PCR product length, it is particularly useful when the gene-specific primer sequences have low Tm or are too sensitive to such parameters to generate specific products. The formula of the second ACP is<br>
2 0      identical to the formula (1) in which the 3" -end portion contains a gene-specific sequence.<br>
The oligonucleotide for the step (d) is similar to CapFinder primer (Chenchik et al., 1998; Chenchik et al. U.S. Pat Nos. 5,962,271 and 5,962,272) in the senses both of them comprise at least three guanine residues at its 3" -end and use them as a template switching primer for the 3" -end extension of the first cDNA strand by reverse transcriptase, whereas they are clearly<br>
2 5 different from each other in terms of the function of a switch in controlling primer annealing to a template nucleic acid in associated with annealing temperature during PCR. CapFinder primer does not comprise univeasl base residue group which is responsible for regulating primer annealing in ACP, so that the CapFinder PCR method for 5" -RACE (Chenchik et al., 1998) can not be free from a high background such as DNA spear arising from contamination<br>
30 of the primers such as the CapFinder and Oligo-dT primers used in cDNA synthesis during PCR. On the other hand, the universal base resiude group of the first ACP plays a key role in regulating primer annealing, so that the subject method does not provide any cause for the<br><br>
background problems during subsequent PCR amplification; this is a key feature of the ACP application to 5" -RACE.<br>
Furthermore, when the ACP of the present invention is used in 5" -RACE technology, it is unnecessary to conduct the process of physical separation such as a solid-phase cDNA 5      synthesis and procedures which has been introduced as an alternative method to remove all contaminants used in cDNA synthesis (Schramm et al., 2000).<br>
In a preferred embodiment, the oligonucleotide to form a base-pair(s) with the cytosine tail • for 5" -RACE which has a similar structure to ACP, wherein the oligonucleotide is represented by the following general formula (3): 5" -dX15.30-dY2.10-dZu10-G3.5-3", in which dX represents 10 a deoxyribonucleotide and comprises a pre-se"ected arbitrary sequence; dY represents a regulatory portion comprising 2-10 universal crises ■»&gt; ion-discriminatory base analogs; dZ represents a deoxyribonucleotide and comprises a pre-selected arbitrary sequence; and G3.5 represents three to five guanines.<br>
Most preferably, the 3" -end portion sequence dZ is ubout 2-3 nucleotides in length. 15      Further, in one embodiment, the 5" -end portion dX can include a sequence that is recognized by a restriction endonuclease.<br>
The G3.5 may be three to five riboguanines or deoxyguanii.es, or a combination of riboguanine and deoxyriboguanine. In more preferred embodiment, the G3.5 comprises two riboguanines and one deoxyriboguanine (r(G)i-d(G)-3"), most preferably, three riboguanines.<br>
2	0	When the gene-specific primer in step (f) is used as 3" primer for 5" -RACE, a target<br>
cDNA fragment containing a 5" -end sequence is amplified under high stringency conditions by conventional PCR methods as known in the art<br>
In a preferred embodiment, a target cDNA fragment containing a 5" -end sequence is amplified using a second ACP which comprises a gene-specific sequence at the 3" -end portion, 25 by two stage PCR amplifications which is conducted in the application for amplifying a target nucleic acid sequence in the present invention. Since the ACP described in this invention can provide stable Tra in a primer and also tolerate "primer search parameters" for primer design such as primer length, annealing temperature, GC content, and PCR product lengdi, it is useful when the gene-specific primer sequences have low Tm or are too sensitive to such parameters to<br>
3	0      generate specific products. The formula of the second ACP is identical to the formula (1) in<br>
which the 3" -end portion contains a gene-specific sequence.<br><br>
The use of ACP in RACE technology significantly simplifies and improves the conventional RACE technologies with regard to the amplification of cDNA ends as described above. The vital feature of the subject method is to be free from the background problems arising from the primers used in conventional RACE methods. Consequently this method 5 described herein can be more effective, easier, less labor-intensive, and more reproducible man conventional RACE methods.<br>
In still another aspect of this invention, there is provided a kit for rapidly amplifying a target cDNA fragment comprising 3" -end region of mRNA, which comprises the annealing control primer or annealing control primer set described previously (including the first-*1 d second primer). According to one embodiment of this invention, these kits further compr"jc. a primer or a primer pair having a nucleotide sequence corresponding to the 5" -end portion of the ACPs; in case that the 5" -end portion comprises universal primer sequence, it is more preferred that the kit comprises the universal primers. The present kits may optionally include the reagents required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5" -triphosphates. Optionally, the kits may also include various polynucleotide molecules, reverse transcriptase, various buffers and reag.ents, and antibodies that inhibit DNA polymerase activity. The kits may also include lc^pcDfe necessary for performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having (lie benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or compartments the constituents afore-described.<br>
In further aspect of this invention, there is provided a kit for rapidly amplifying a target cDNA fragment comprising 5" -end region of mRNA, which comprises the annealing control primer or annealing control primer set described above (including the oligonucleotide dl primer and random primer for cDNA synthesis, the oligonucleotide to form a base-paii"(e) with the cytosine tail, the first primer and the second primer). According to one embodiment of this invention, these kits further comprises a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the first and second primers as used in steps (f)-(i) and (f)-(ii); in case that the 5"-end portion comprises universal primer sequence, it is more preferred that the kit comprises the universal primers.<br><br>
V. Application to Amplifying Full-length cDNA<br>
In further aspect of this invention, there is provided a method for amplifying a population of full-length double-stranded cDNAs complementary to mRNAs, wherein the method comprises reverse transcribing the mRNA and performing an amplification reaction using 5 primers, characterized in that at least one primer is derived from any one of ACP described above. Preferably, the primer having the structure of ACP is one having a hybridizing nucleotide sequence substantially complementary to poly A tails of mRNAs.<br>
hi a specific embodiment of this invention, there is provided the method comprises:<br>
(a)	contacting the mRNAs with a first primer of any one of ACP described above, in<br>
10      which the 3" -end portion of the first primer has a hybridizing nucleotide sequence substantially<br>
complementary   <.o j>V A tails of the mRNAs to hybridize therewith, under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing the mRNAs, usin°; a reverse transcriptase, to which the first<br>
primer hybridizes to product the population of first cDNA strands that are complementary to<br>
15      the mRNAs to which the primer hybridizes, whereby mRNA-cDNA intermediates are generated;<br>
(c)	permitting cytosine residues to be tailed at the 3" -ends of the first cDNA strands in<br>
the form of the mRNA-cDNA intermediates by the terminal transferase reaction of reverse<br>
transcriptase;<br>
2	0	(d)   contacting the cytosine tails at the 3" -ends of the first cDNA strands generated from<br>
step (c) with an oligonucleotide which comprises a 3" -end portion and a 5" -end portion separated by a group of universal base or non-discriminatory base analog, wherein the 3" -end portion comprises at least three guanine residues at its 3" -end to hybridize with the cytosine tails at the 3" -ends of the first cDNA strands and the 5" -end portion comprises a pre-selected 25 arbitrary nucleotide sequence, under conditions in which the 3-end portion of the oligonucleotide is hybridized to the cytosine tails;<br>
(e)	extending the tailed 3" -ends of the first cDNA strands to generate an additional<br>
sequence complementary to the oligonucleotide using reverse transcriptase, in which the<br>
oligonucleotide serves as a template in the extension reaction, whereby full-length first cDNA<br>
3	0      strands are extended; and<br>
(f)	performing an amplification of the full-length first cDNA strands generated from step<br>
(e) comprising at least two cycles of primer annealing, primer extending and denaturing, using<br><br>
a primer pair each comprising a nucleotide sequence corresponding to the same first primer and oligonucleotide as used in steps (a) and (d), respectively, or a primer pair each comprising a nucleotide sequence corresponding to each 5"-end portion of the first primer and oligonucleotide used in steps (a) and (d), respectively, under conditions in which each primer 5 anneals to the 3" - and 5" -end sequences of the full-length first cDNA strands, respectively, whereby amplification products of full-length cDNA strands complementary to the mRNAs are generated.<br>
Since this application using the ACP of this invention employs in principle the present methods for amplification of nucleic acid sequence previously discussed, the common<br>
10 descriptions between them are omitted in order to avoid the complexity of mis specification leading to undue multiplicity. In addition, the ACP described *ov
15      (f) are in principle identical to those for 5" -RACE of this invention discussed above.<br>
A schematic representation for amplifying full-length cDNA molecules of the present invention is illustrated in FIG. 5.<br>
The use of ACP significantly simplifies and improves the conventional technologies with regard to the amplification of full-length cDNAs as described above. The vital feature of the<br>
20 subject method is to be free from the background problems arising from the primers used in conventional methods. Consequently this method described herein can be more effective, easier, less labor-intensive, and more reproducible than conventional methods.<br>
In still further aspect of this invention, there is provided a kit for amplifying a full-length 25 double stranded cDNA complementary to mRNA, which comprises the annealing control primer or the annealing control primer set described above (including the oligonucleotide dT primer, the oligonucleotide to form a base-pair(s) with the cytosine tail, the primer(s) used in the step (f». According to one embodiment of this invention, these kits further comprises a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the 3 0 primer and oligonucleotide used in steps (a) and (d), respectively; in case that the 5" -end portion comprises universal primer sequence, it is more preferred that the kit comprises the universal primers. The present kits may optionally include the reagents required for performing<br><br><br>
PCR reactions such as buffers, DNA polymerase, DNA polymerase cofactors, and deoxyribonucleotide-5"-triphosphates. Optionally, the kits may also include various polynucleotide molecules, reverse transcriptase, various buffers and reagents, and antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary for 5 performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or compartments the constituents afore-described.<br>
10      VI. Application to Amplifying 5"-Enriched cDNA<br>
In another aspect of this invention, there is provided a method for amplifying a population of 5" -enriched double-stranded cDNAs comprising cDNA regions corresponding to the 5" -end regions of mRNAs, wherein the method comprises reverse transcribing the mRNA and performing an amplification reaction using primers, characterized in that at least one primer is 15 derived from any one of ACP described above. Preferably, the primer having the structure of ACP used for cDNA synthesis is one having at its 3" -end portion at least six random nuc, "jotide sequences.<br>
In a specific embodiment of this invention, there is provided the method comprises:<br>
(a)	contacting the mRNAs with a first primer of any one of ACP described above under<br>
2	0      conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur,<br>
wherein the 3" -end portion of the first primer has at least six random nucleotide sequences;<br>
(b)	performing the steps (b)-(e) of the method for amplifying a population of full-length<br>
double-stranded cDNAs, whereby 5" -enriched first cDNA strands are extended;<br>
(c)	performing an amplification of the 5" -enriched first cDNA strands generated from<br>
25      step (b) comprising at least two cycles of primer annealing, primer extending and denaturing,<br>
using a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the primer and oligonucleotide used in steps (a) and (b), respectively, under conditions in which each primer anneals to the 3" - and 5" -end sequences of the 5" -enriched first cDNA strands, respectively, whereby amplification products of 5" -enriched cDNA strands<br>
3	0      are generated.<br>
Since this application using the ACP of this invention employs in principle the present methods for amplification of nucleic acid sequence previously discussed, the common<br><br><br>
descriptions between them are omitted in order to avoid the complexity of mis specification<br>
leading to undue multiplicity. In addition, the oligonucleotide to form a base-pair(s) with the<br>
cytosine tail and the primer pair used in the step (c) are in principle identical to those for 5" -<br>
RACE of this invention discussed above.<br>
5	A schematic representation for the method for amplifying 5" -enriched double-stranded<br>
cDNAs complementary to mRNAs is illustrated in FIG. 6.<br>
"5" enriched cDNAs" refers to a significant portion of the cDNA constituents which contain the nucleotide sequence information of the 5" -end of the mRNAs from which the cDNAs are derived.<br>
10	The formula of the first primer is identical to the formula (1) in which the 3" -end portion<br>
comprises a random nucleoli- "&amp; st "<v- ice. in a preferred embodiment the portion of first primer used step contains at least six random deoxyribonucleotides. can includes sequence that is recognized by restriction endonucleas></v->
15	The conventional methods require more steps to amplify 5"enriched cDNA molecules<br>
complementary to the mRNA molecules than the subject method because the. conventional methods use the conventional primers which do not have the function of controlling prbno annealing. In contrast, this subject method is considerably a simple and effective approach due-to the function of regulating primer annealing generated by the effect of a universal baao<br>
2	0      residue group in ACP.<br>
In still another aspect of this invention, mere is provided a kit for amplifying 5" -enriched double-stranded cDNAs complementary to mRNAs, which comprises the annealing control primer or the annealing control primer set described above (the first primer, the oligonucleotide 25 to form a base-pair(s) with the cytosine tail). According to one embodiment of this invention, these kits further comprises a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the primer and oligonucleotide used in steps (a) and (b), respectively; in case that the 5" -end portion comprises universal primer sequence, it is more preferred that the kit comprises the universal primers. The present kits may optionally include the reagents<br>
3	0      required for performing PCR reactions such as buffers, DNA polymerase, DNA polymerase<br>
cofactors, and deoxyribonucleotide-5"-triphosphates. Optionally, the kits may also include various polynucleotide molecules, reverse transcriptase, various buffers and reagents, and<br><br>
antibodies that inhibit DNA polymerase activity. The kits may also include reagents necessary for performing positive and negative control reactions. Optimal amounts of reagents to be used in a given reaction can be readily determined by the skilled artisan having the benefit of the current disclosure. The kits, typically, are adapted to contain in separate packaging or 5      compartments the constituents afore-described.<br>
VII. Application to DNA or RNA Fingerprinting<br>
This application using ACP of the subject invention can provide an improved method for detecting polymorphisms in genomic DNA (DNA fingerprinting) or for detecting differential gene expression in mRNA (RNA fingerpruiting).<br>
10	m further aspect of mis invention, there is provided a method for  r; Jucing a DNA<br>
fingerprint of gDNA, wherein the method comprises performing an amplification reaction using primers, characterized in that at least one primer is derived from any one of ACPs described above. Preferably, the primer having the structure of ACP is one having at its 3" -end portion an arbitrary nucleotide sequence substantially complementary to sites on the gDNA.<br>
15	In a specific embodiment of this invention, there is provided the method using two stage<br>
amplifications, which comprises:<br>
(a) performing a first-stage amplification of the DNA fingerprint, which is a set of discrete DNA segments characteristic of genome, from the gDNA at a first annealing temperature comprising at least two cycles of primer annealing, primer extending and<br>
2	0      denaturing, using the primer or the primer pair of any one of ACPs described above, wherein<br>
each primer has at its 3"-end portion an arbitrary nucleotide sequence substantially<br>
complementary to sites on the gDNA to hybridize therewith, under conditions in which the<br>
primer or the primer pair anneals to the gDNA, whereby the set of discrete DNA segments<br>
characterized as a DNA fingerprint is produced; and<br>
25	(b)   performing a second-stage amplification of the set of discrete DNA segments<br>
generated from step (a) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primer or primer pair as used in step (a) or a primer or a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the primer or primer pair used in<br>
3	0      step (a), under conditions in which the primer or each of the primer pair anneals to the 3" - and<br><br><br><br><br><br>
5" -end sequences of the set of discrete DNA segments generated from step (a), respectively, whereby the set of discrete DNA segments is re-amplified.<br>
In still further aspect of tins invention, there is provided a method for producing a RNA<br>
5 fingerprint of an mRNA sample, wherein the method comprises reverse transcribing and<br>
performing an amplification reaction using primers, characterized in that at least one primer is<br>
derived from any one of ACPs. Preferably, the primer according to the structure of ACP is one<br>
having at its 3" -end portion an arbitrary nucleotide sequence substantially complementary to<br>
sites on cDNA strands generated from reverse transcription and/or one having at its 3" -end<br>
10 portion a hybridizing nucleotide sequence substantially complementary to poly A tails of the<br>
rnRNAs.	-<br>
In a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	contacting the mRNA sample with a first primer of any one of ACPs described<br>
15      above, in which the first primer has a hybridizing nucleotide sequence substantially<br>
complen aitary to poly A tails of the mRNA sample to hybridize therewith, under conditions sufficient tor template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing the mRNA sample to which the first primer hybridizes to<br>
produce a population of first cDNA strands that are complementary to the mRNA sample to<br>
2 0      which the first primer hybridizes;<br>
(c)	performing a first-stage amplification of the population of first cDNA strands<br>
generated from step (b) at a first annealing temperature comprising at least one cycle of primer<br>
annealing, primer extending and denaturing, using a second primer or primer pair of any one of<br>
ACPs described above, wherein each primer has at its 3" -end portion an arbitrary nucleotide<br>
2	5      sequence substantially complementary to sites on the first cDNA strands to hybridize therewith,<br>
under conditions in which the primer or primer pair anneals to the mRNA sample, whereby a set of discrete cDNA segments characterized as a RNA fingerprint is produced; and<br>
(d)	performing a second stage amplification of the set of discrete cDNA segments<br>
generated from step (c) at a second annealing temperature which is high stringent conditions,<br>
3	0      comprising at least one cycle of primer annealing, primer extending and denaturing, using the<br>
same primer or primer pair as used in step (c) or a primer or primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the primer or primer pair used in<br><br><br>
step (c), under conditions in which the primer or each of the primer pair anneals to the 3" - and 5" -end sequences of the set of discrete cDNA segments generated from step (c), respectively, whereby the set of discrete cDNA segments is re-amplified.<br>
5	Since this application using the ACP of this invention employs in principle the present<br>
methods for amplification of nucleic acid sequence previously discussed, the common descriptions between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity, m addition, the RNA fingerprinting in principle follows the present method for detecting DNA complementary to differentially expressed mRNA.<br>
10	The term " genomic DNA" as -""jd herein refers to a population of DNA that comprises<br>
the complete genetic component of a specie . Thus genomic DNA comprises the complete set of genes present in a pre-selected species. The complete set of genes in a species is also referred to as genome. The term DNA or RNA "fingerprinting" as used herein refers to a set of discrete DNA amplification products characteristic of a genome or a set of discrete cDNA<br>
15      segments characteristic of a sample of mRNA, respectively.<br>
In the previous arbitrarily primed PCR fingerprints, "-.ailed AP-PCR, short or long arbitrary primers have been used under non-stringent conditions for early 2-5 cycles of PCR amplification because a low annealing temperature is required to achieve arbitrary priming, such that a significant portion of isolated fragments is not still reproducible although effective<br>
2 0      amplification proceeds in the following cycles under high stringent condition.<br>
In contrast to AP-PCR, the ACP-based PCR for fingerprinting increases the specificity of primer annealing during PCR due to the function of a universal base residue group positioned between the 3" - and 5" -end portions of ACP, wherein the universal base residue group restricts the annealing site to the 3" -end portion of the ACP and also allows this 3" -end portion to<br>
2	5      anneal at a relatively high annealing temperature. Thus, the ACP-based PCR for fingerprinting<br>
completely eliminates false positive products and significantly increases reproducibility.<br>
In a preferred embodiment, the ACP contains an arbitrary sequence at the 3" -end portion with at least 6 nucleotides in length. More preferably, the 3"-end portion contains 8-15 nucleotides in length, most preferably, about 10 nucleotides in length.<br>
3	0	A single ACP or a pair of ACPs can be used for detecting polymorphisms in DNA<br>
fingerprinting. Preferably, a pair of ACPs is used for DNA fingerprinting because a pair of ACPs produces more products than a single arbitrary ACP does.<br><br>
An example of the DNA fingerprinting using ACP is conducted by two stages of PCR amplifications under the following conditions: amplification reactions are performed under low stringent conditions by two cycles of the first-stage PCR comprising annealing, extending and denaturing reaction; the reaction mixture containing genomic DNA, PCR reaction buffer, 5 MgCl2, dNTPs (dATP, dCTP, dGTP and dTTP), a pair of ACPs is pre-heated, Taq polymerase is added into the reaction mixture; the PCR reactions are performed, followed by denaturing the amplification product; after the complete reaction of the first-stage PCR, the pre-selected arbitrary primer JYC4 corresponding to the 5" -end portion of the ACPs are added to the reaction mixture and then the second stage PCR amplification is conducted.<br>
10	It should be noted that a proper concentration of ACP is used to prod"ce DNA<br>
fingerprinting. If the amount of the ACP is too low, the resultant amplified PCR p* jduc . &gt;re not reproducible. In contrast, the excess amount of the ACP generates backgrounds such as DNA smear during PCR. In a preferred embodiment, the concentration of the ACP is about between 0.1 pM and 2 uM. Most preferably, the concentration of the ACP is about 1.4 pM.<br>
15	In a preferred embodiment, the concentration of the primer corresponding to the 5" -end<br>
portion of the ACPs is about between 0.1 uM and 2 uM, most preferably, about 0.8 \.iM.<br>
The genomic DNA and mRNA samples may be obtained from a wide variety of biomaterials and conditions. For example, they may be obtained from plants, animal (human) and microbes and from different individual organisms.<br>
2 0	The amplified products can be analyzed by gel electrophoresis. In one embodiment, the<br>
resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (GottscMich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated<br>
2 5      primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
»<br>
In still further aspect of this invention, there is provided a kit for producing a DNA<br>
fingerprint by use of gDNA or mRNA, which comprises the annealing control primer or<br>
annealing control primer set described above. The descriptions of the kits for the amplification<br>
30      of nucleic acid sequence and for detecting DNA complementary to differentially expressed<br>
mRNA of this invention can be applied to the present kit.<br><br>
VID, Application to Identification of Conserved Homology Segments in Mcltigene Families<br>
This application using ACP of the subject invention can also provide an improved method<br>
for the identification of conserved homology segments in multigene families.<br>
5	In another aspect of this invention, there is provided a method for identifying conserved<br>
homology segments in a multigene family from an mRNA sample, wherein Hie method comprises reverse trariscribing and performing an amplification reaction using primers, characterized in that at least one primer is derived from any one of ACPs described above. Preferably, the primer having the structure of ACP is one having at its 3" -end portion a 10 hybridising seqvmee substantially complementary to a consensus sequence or a degenerate sequ"jp^e     .ing amino acid sequence of a conserved homology segment on cDNA strands generated from reverse transcription and/or one having at its 3" -end portion a hybridizing nucleotide sequence substantially compl~mentary to poly A tails of the mRNAs.<br>
In a specific embodiment of this invention, there is provided the method using two stage 15      amplifications, which comprises:<br>
(a)   contacting the nRNA sample with a first primer of any one of claims 1-29, in which<br>
the first primer has a hybridizing nucleotide sequence substantially complementary to poly A<br>
tails of the mRNA sample to hybridize therewith, under conditions sufficient for template<br>
driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
20	(b)   reverse transcribing the mRNA sample to which the first primer hybridizes to<br>
produce a population of first cDNA strands that are complementary to the mRNA sample to which the first primer hybridizes;<br>
(c)    performing a first-stage amplification of the population of first cDNA strands generated from step (b) at a first annealing temperature comprising at least one cycle of primer<br>
2	5      annealing, primer extending and denaturing, using a second primer of any one of claims 1-25<br>
having at its 3" end portion a hybridizing sequence substantially complementary to a consensus sequence or a degenerate sequence encoding amino acid sequence of a conserved homology segment on the first cDNA strands to hybridize therewith, under conditions in which the second primer anneals to the consensus sequence or degenerate sequence of first cDNA strands,<br>
3	0      whereby 3" -end cDNA segments having the consensus sequence or degenerate sequence are<br>
generated; and<br><br>
(d) performing a second stage amplification of the 3" -end cDNA segments generated from step (c) at a second annealing temperature which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (a) and (c) or a primer pair each comprising a nucleotide 5 sequence corresponding to each 5" -end portion of the first and second primers used in steps (a) and (c), respectively, under conditions in which each primer anneals to the 3" - and 5" -end sequences of the 3"-end cDNA segments, respectively, whereby the 3"-end conserved homology cDNA segments are amplified.<br>
This specific embodiment follows in principle, the present method for 3" RACE as 10      discussed previously except for the second primer use 1.<br>
In another specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	performing steps of (a)-(e) of the method for amplifying a population of full-length<br>
15      double-stranded cDNA, whereby full-length cDNA strands are generated;<br>
(b)	performing a first-stage amplification of the full-length first i/DNA strands obtained<br>
from step (a) at a first annealing temperature, which comprises the steps of:<br>
(i)   at least one cycle of primer annealing, primer extending and denaturing using a<br>
first primer comprising a nucleotide sequence substantially complementary to the 3" -<br>
2 0	end sequences of the full-length first cDNA strands under conditions in which the<br>
first primer anneals to the full-length first cDNA strands, under conditions in which the first primer anneals to the 3" - ends of the full-length first cDNA strands, whereby full-length second cDNA strands are generated; and<br>
(ii)   at least one cycle of primer annealing, primer extending and denaturing using a<br>
25	second primer of any one of claims 1-25 having at its 3" end portion a hybridizing<br>
sequence substantially complementary to a consensus sequence or a degenerate<br>
sequence encoding amino acid sequence of a conserved homology segment on the<br>
full-length second cDNA strands to hybridize therewith, under conditions in which the<br>
second primer anneals to the consensus sequence or degenerate sequence of full-length<br>
30	second cDNA strands, whereby 5"-end cDNA segments having the consensus<br>
sequence or degenerate sequence are generated; and<br><br>
(c) performing a second stage amplification of the 5" -end cDNA segments generated from step (b) at a second annealing temperature which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (b)-(i) and (b)-(ii), respectively, or a primer pair each 5 comprising a nucleotide sequence corresponding to each 5" -end portion of the first and second primers used in steps (b)-(i) and (b)-(ii), respectively, under conditions in which each primer anneals to the 3" - and 5" -end sequences of the 5" -end cDNA segments, respectively, whereby the 5" -end conserved homology cDNA segments are amplified.<br>
This specific embodiment follows in principle, the present method for 5" RACE as 10      discussed previously except for the second primer used.<br>
In further aspect of this invention, there is provided a method for identifying conserved homology segments in a multigene family from gDNA, wherein the method comprises performing an amplification reaction using primers, characterized in that at least one primer is<br>
15 derived from any one of ACPs described above. Preferably, the primer having the structure of ACP is one having at its 3" -end portion a hybridizing sequence substantially complementary to a consensus sequence or a degenerate sequence encoding amino acid sequence of a conserved homology segment on the gDNA.<br>
In a specific embodiment of this invention, there is provided the method using two stage<br>
2 0      amplifications, which comprises:<br>
(a) performing a first-stage amplification of the conserved homology segments from the gDNA at a first annealing temperature comprising at least two cycles of primer annealing, primer extending and denaturing, using the primer or the primer pair of any one of ACPs described above, wherein each primer has at its 3"end portion a hybridizing sequence<br>
25 substantially complementary to a consensus sequence or a degenerate sequence encoding amino acid sequence of a conserved homology segment on the gDNA to hybridize therewith, under conditions in which the primer or the primer pair anneals to the consensus sequence or degenerate sequence of gDNA, whereby genomic DNA segments having the consensus sequence or degenerate sequence are generated; and<br>
30	(b)   performing a second-stage amplification of the genomic DNA segments generated<br>
from step (a) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primer<br><br>
or primer pair as used in step (a) or a primer or a primer pair each comprising a nucleotide sequence corresponding to each 5" -end portion of the primer or primer pair used in step (a), under conditions in which the primer or each of the primer pair anneals to the 3" - and 5" -end sequences of the genomic DNA segments generated from step (a), respectively, whereby the 5      conserved homology genomic segments are amplified.<br>
The present method follows in principle, the present method for amplifying a target nucleic acid sequence from a DNA as discussed previously except for the primer used.<br>
Since this application using the ACP of this invention employs in principle the present<br>
10 methods for amplification of nucleic acid sequence previously discussed, the common<br>
descriptions betweftii ther "■; omitted in order to avoid the complexity of this specification<br>
leading to undue multiplicity. In addition, where an mRNA is used as starting material, the<br>
present methods for 3" or 5" RACE are in principle applied to the present methods for the<br>
identification of conserved homology segments in multigene families.<br>
15	The formula of ACP for the identification of conserved homology segments in multigene<br>
families is identical to the formula (1) ,-n which the 3" -end portion of ACP has a hybridizing sequence substantially complementary ..o a consensus sequence in a gene family or a degenerate sequence encoding amino acid sequence of a conserved homology.<br>
There are two principle approaches to the design of degenerate primer: (a) using peptide<br>
2 0      sequence data obtained from a purified protein; and (b) using consensus protein sequence data<br>
from alignments of gene families. If orthologs of the gene of interest have been cloned from<br>
other organisms, or if the gene is a member of a gene family, it will be possible to generate<br>
protein sequence alignments.<br>
These may reveal appropriate regions for the design of degenerate primers, for example,<br>
2	5      from consensus sequence of highly conserved regions. Amplifications using degenerate primers<br>
can sometimes be problematic and may require optimization. The first parameter is annealing temperature. It is important to keep the annealing temperature as high as possible to avoid extensive nonspecific amplification and a gocd rule of thumb is to use 55°C as a starting temperature. In general, it is difficult to keep this rule because degenerate primers should be<br>
3	0      designed based on amino acid sequences as a precondition. However, the ACP of the present<br>
invention does not have to satisfy this requirement because it allows a high annealing temperature such as 65°C at the second stage of PCR amplification regardless of primer design.<br><br>
According to a preferred embodiment, the second primer is a pool of primers each comprising a degenerate sequence selected from a plurality of the nucleotides coding for amino acid sequence of the consensus sequence.<br>
The term "conserved region" and more specifically "conserved region of a gene in a 5 mul ti gene family" as used herein refers to a segment of nucleotide sequence of a gene or amino acid sequence of a protein that is significantly similar between members of gene families. The degree of similarity can vary. Li some cases the conserved regions will be identical between family members. In some cases the nucleotide sequence may vary significantly but still encode for amino acid segments that are conserved between family members. The term "consensus 10 sequence" as used herein refers to the bases most often found at any given position when comparing a large number of similar nucleotide sequences.<br>
Alternatively, the present methods for the identification of conserved homology segments can be also combined with that for detecting differentially expressed mRNAs.<br>
15	In still further aspect of this invention, there is provided a kit for identifying a conserved<br>
homology segment in a multigene family by use of mRNA or gDNA, which comf rises the annealing control primer or annealing control primer set described above. The descriptions of the kits for the amplification of nucleic acid sequence, 3" RACE and 5" RACE of this invention can be applied to the present kit<br>
20      IX. Application to Identification of a Nucleotide Variation<br>
This application using ACP system of the subject invention can also provide an improved method for identifying a nucleotide variation in a target nucleic acid.<br>
In another aspect of this invention, there is provided a method for identifying a nucleotide variation in a target nucleic acid, wherein the method comprises performing an amplification<br>
2 5 reaction using primers, characterized in that at least one primer is derived from any one of ACPs described above. Preferably, the primer having the structure of ACP is (a) a first primer one having at its 3"-end portion a hybridizing sequence substantially complementary to a pre¬selected sequence at a first site of target nucleic acid, wherein each of the first primer and the first site comprises an interrogation position corresponding to the nucleotide variation, and/or<br>
30 (b) a second primer having a hybridizing sequence substantially complementary to a pre¬selected sequence at a second site of target nucleic acid.<br><br>
In a specific embodiment of this invention, there is provided the method using two stage amplifications, which comprises:<br>
(a)	performing a first-stage amplification to produce a first DNA strand complementary<br>
to the target nucleic acid including the nucleotide variation at a first annealing temperature<br>
5 comprising at least one cycle of primer annealing, primer extending and denaturing, using a first primer of any one of ACPs described above having at its 3" -end portion a hybridizing sequence substantially complementary to a pre-selected sequence at a first site of the target nucleic acid to hybridize therewith, wherein each of the first primer and the first site comprises an interrogation position corresponding to the nucleotide variation, whereby the first DNA 10 strand complementary to the target nucleic acid including the nucleotide variation is generated when the interrogation position is occupied by the complementary nucleotide of the first primer to its corresponding nucleotide of the first site; and<br>
(b)	performing a second-stage amplification of the first DNA strand generated from step<br>
(a) at a second annealing temperature, which is high stringent conditions, comprising the steps:<br>
15	(i)   at least one cycle of primer annealing, primer extending and denaturing using a<br>
second primer of any one of ACPs decribed above having at its 3"-end portion a hybridizing sequence substantially complementary to a pre-selected sequence at a second site of the target nucleic acid to hybridize therewith under conditions in which the second primer anneals to the second site of the target nucleic acid, whereby a<br>
2 0	second DNA strand complementary to the first DNA strand including the nucleotide<br>
variation is generated; and<br>
(ii) at least one cycle of primer annealing, primer extending and denaturing using the same first and second primers as used in steps (a) and (b)-(i) or a primer pair each having a hybridizing sequence complementary or corresponding to the 3" - and 5" -<br>
2 5	ends of the second DNA strand generated from step (b)-(i) to hybridize therewith,<br>
under conditions in which each primer anneals to the 3" - and 5" -end sequences of the second DNA strand, respectively, whereby the second DNA strand which comprises the first and second sites of the target nucleic acid at its 3" -and 5" -ends is amplified so that a short target nucleotide segment corresponding to the second DNA strand<br>
30	containing the nucleotide variation is generated.<br><br>
WO 03/050305<br><br>
PCT/KR02/01781<br><br>
Since this application using the ACP of this invention employs in principle the present<br>
methods for amplification of nucleic acid sequence previously discussed, the common<br>
descriptions between them are omitted in order to avoid the complexity of this specification<br>
leading to undue multiplicity.<br>
5	A   schematic   representation   of this   specific   embodiment   for   single   nucleotide<br>
polymorphism (SNP) genotyping using novel ACP is illustrated in FIG. 7A.<br>
The formula of ACP for the detection of a nucleotide variation, is identical to the formula (1) in which its 3" -end portion comprises a hybridizing sequence substantially complementary to a pre-selected sequence at a site of the target nucleic acid to hybridize therewith which<br>
10 contains the nucleotide variation, "/herein the nucleotide corresponding to the nucleotide variation and its complementary dacleouv , of the ACP occupy an interrogation position. The process for this application is carried out by two stage PCR amplifications using the genomic DNA obtained from samples such as patient blood or a short segment of the sample DNA, which includes a target nucleotide variation. The interesting nucleotide sample may be obtained<br>
15      from human nucleic acid and an organism that can cause an infectious disease.<br>
The method using two-stage PCR amplifications for detecting single nucleotide polymorphism (SNP) genotyping basically follows the process used for amplifying a target nucleic acid sequence using genomic DNA as a starting material, ui addition, the process for multiplex DNA amplification can be adapted to this application.<br>
20	To use a short segment of the sample DNA including a target nucleotide variation as a<br>
starting material for the above process, it is preferable that the target short segment is pre-amplified prior to step (a) using a primer pair in which each has a hybridizing sequence substantially complementary to the sample DNA to hybridize therewith. Furthermore, more than one target nucleotide segment each including a SNP can be prepared by the multiplex<br>
2	5     DNA amplification as described in Application II to be used as a starting material in the subject<br>
invention for multiple SNP screening.<br>
The first ACP used in step (a) for the detection of a polymorphic base is an allele-specific ACP which contains an interrogation position within its 3"-end portion occupied by a complementary nucleotide to the corresponding nucleotide of the nucleotide variation in a<br>
3	0      target nucleic acid. Preferably, (he interrogation position of the first primer is in the middle of<br>
its 3" -end portion. In a more preferred embodiment, the interrogation position of the allele-specific ACP is within about 10 bases of the 3" -end nucleotide. More advantageously, the<br><br><br>
interrogation position of the allele-specific ACP is within about 6 bases of the 3"-end nucleotide of the allele-specific ACP. In another preferred embodiment, the interrogation position of the allele-specific ACP is located within positions 4 and 6 from the 3" -end nucleotide. Most preferably, the interrogation position of the allele-specific ACP is located in 5 position 5 from the 3" -end nucleotide. The term "3" -end nucleotide" used herein refers to a nucleotide which is positioned at the 3" -end of ACP.<br>
In another embodiment, die 3" -end portion of the allele-specific ACP used in step (a) contains at least 6 nucleotides in length, which is a minimal requirement of length for primer annealing. Preferably, the 3" -end portion sequence is about 8 to 20 nucleotides in length. Most<br>
10 preferably, the 3"-end portion sequence is about 10 nucleotides in leng+h including an interrogation position.<br>
In one embodiment, at least one artificial mismatch can be also placed within the 3" -end portion of ACP using universal base or non-discriminatory analog that hydrogen-bonds minimally with all four bases without steric disruption of the DNA duplex. Although the<br>
15 position of the artificial mismatch can vary depending on experimental designs, it is preferred that the mismatch nucleotide is substantially adjacent the interrogation position of the first primer.<br>
hi a preferred embodiment, the first or second primers comprise at least one nucleotide with a label for detection or isolation.<br>
2 0	According to a preferred embodiment, the first DNA strand including nucleotide variation<br>
in step (a) is generated by one cycle of primer annealing, primer extending, and denaturing. It is preferred that the second DNA strand including nucleotide variation in step (o)-(i) is generated by one cycle of primer annealing, primer extending, and denaturing. Preferably, the second DNA strand including nucleotide variation in step (b)-(ii) is amplified by at least 5 cycles of<br>
2	5      primer annealing, primer extending, and denaturing.<br>
In another specific embodiment of this invention using amplified short DNA strand fragment containing the nucleotide variation, there is provided the method using two individual amplifications of a first and a second amplifications in which the second<br>
3	Q      acapUficaliow. is performed vsiftg two stage, amplifications, which cocapcisfcs;<br>
(a) performing the first amplification to produce a short DNA strand fragment containing the nucleotide variation between its ends comprising at least two cycles of primer<br><br>
annealing, primer extending and denaturing, using a primer pair each primer comprising a hybridizing sequence substantially complementary to a pre-selected sequence at a site of the target nucleic acid under conditions that the nucleotide variation is positioned between the pre-selected sequences, in which at least one primer of the primer set is any one of ACPs 5    described above having at its 3"-end portion the hybridizing sequence, whereby the short DNA strand fragment containing the nucleotide variation between its ends is amplified;<br>
(b)	performing a first-stage amplification of the second amplification to produce a first<br>
DNA strand complementary to the short DNA strand fragment including the nucleotide<br>
variation at a first annealing temperature comprising at least one cycle of primer annealing,<br>
10       primer extending and denaturing, using a first primer of any one of ACPs described above  hi"V:; at its 3"-end portion a hybridizing sequence substantially complementary to a pre¬selected sequence at a first site of the target nucleic acid to hybridize therewith, wherein each of the first primer and the firs* site comprises an interrogation position corresponding to the nucleotide variation, whereby the first DNA strand complementary to the target nucleic acid<br>
15          including the nucleotide variation is generated when the interrogation position is occupied by the complemen*ary nucleotide of the first primer to its corresponding nucleotide of the first site; and<br>
(c)	performing a second-stage amplification of the second amplification of the first DNA<br>
strand generated from step (a) at a second annealing temperature, which is high stringent<br>
2	0     conditions, comprising at least one cycle of primer annealing, primer extending and denatming<br>
using a primer pair in which amongst the primer pair one is the same as the primer of any one of ACPs used in step (a) the other is the same as the first primer used in step (b), or a primer pair each having a hybridizing sequence complementary or corresponding to the 3"- and 5"-ends of the first DNA strand generated from step (b) to hybridize therewith, under conditions in 25 which each primer anneals to the 3"- and 5"-end sequences of the first DNA strand, respectively, whereby the first DNA strand is amplified so that a short target nucleotide segment corresponding to the first DNA strand containing the nucleotide variation is generated.<br>
A schematic representation of another specific embodiment for single nucleotide<br>
3	0      polymorphism (SNP) genotyping using novel ACP is illustrated in FIG. 7B. Since this specific<br>
embodiment is carried out in a similar manner to above embodiment, the common descriptions<br><br>
WO 03/050305<br><br>
PCT/KR02/01781<br><br>
between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br>
The present method can be applied to a variety of nucleotide variations including single 5      nucleotide polymorphism and point mutation (substitution, deletion and insertion).<br>
The amplified products can be analyzed by gel electrophoresis. In one embodiment, the resulting PCR products can be also detected, on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 10 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 199"  Re io" et al., 1996).<br>
The amplified products generated by multiplex DNA amplification for multiple SNP<br>
screening can be compared through the size separation of the products. The size separation<br>
comparison  is  also performed by electrophoresis "trough  an  agarose  gel  matrix  or<br>
15      polyacrylamide gel matrix or sequencing. The products can be also detected by the use of<br>
fluoresenscent-labelled oligonucleotide primers for automatic analysis.<br>
The term * interrogation position" as used herein refers to the location of a specific nucleotide base of interest within a target nucleic acid. For example, in the analysis of SNPs, the "interrogation position" in the target nucleic acid is in position what would be different 2 0 from wild type. The interrogation position also includes the location of nucleotide sequence of a primer which is complementary to an interrogation position of the target nucleic acid. The interrogation position of the target nucleic acid is opposite the interrogation position of the primer, when the primer is hybridized with the target nucleic acid.<br>
The term "polymorphism" as used herein refers to the presence of two or more alternative 25 genomic sequences or alleles between or among different genomes or individuals. "Polymorphic" refers to the condition in which two or more variants of a specific genomic sequence can be found in a population. A "polymorphic site" is the locus at which the variation occurs. A single nucleotide polymorphism, or SNP, is a single base-pair variant, typically the substitution of one nucleotide by another nucleotide at the polymorphic site. Deletion of a 30 single nucleotide or insertion of a single nucleotide, also give rise to single nucleotide polymorphisms. Typically, between different genomes or between different individuals, the polymorphic site may be occupied by two different nucleotides. The term "allele" as used<br><br><br>
herein refers a specific member of a collection of naturally occurring sequence variants (detectable within a population of individuals) at a specific genomic locus or marker.<br>
In still another aspect of this invention, there is provided a kit for identifying a nucleotide variation in a target nucleic acid, which comprises the annealing control primer or annealing control primer set (including the first and second primers) described above. The descriptions of the kits for the amplification of nucleic acid sequence of this invention can be applied to the present kit.<br>
X. Application to Mutagenesis<br>
This application using ACP of the subject invention can also provide an improve" method 5     for mutagenesis. The ACP-based PCRprovides an excellent tool for mutagenes4J. inch, .r"g deletion, or insertion of sequences, the alteration of one or a few specific nucleotides, and the random mutation of nucleotide sequence.<br>
In further aspect of this invention, mere is provided a method for mutagenesis in a target nucleic acid, comprising performing an amplification reaction using primers, characterized in 10 that at least one primer is derived from any one of ACPs described above. Preferably, the primer having the structure of ACP is one having at its 3"-end portion a hybridizing sequence substantially complementary to a region of target nucleic acid sequence, wherein the hybridizing sequence has a nucleotide sequence responsible for mutagenesis.<br>
In a specific embodiment of this invention, there is provided the method using two stage 15      amplifications, which comprises:<br>
(a)	performing a first-stage amplification of the target nucleic acid sequence at a first<br>
annealing temperature comprising at least two cycles of primer annealing, primer extending<br>
and denaturing, using a primer pair of any one of ACPs described above each having at its<br>
3" end portion a hybridizing sequence substantially complementary to a region of the target<br>
2 0 nucleic acid sequence to hybridize merewith, wherein the hybridizing sequence has at least one mismatch nucleotide to generate site-directed mutation, under conditions in which the primer or primer pair anneals to its target nucleotide sequence, whereby an amplification product containing site-directed mutation site is generated; and<br>
(b)	performing a second-stage amplification of the amplification product generated from<br>
2 5      step (a) at a second annealing temperature, which is high stringent conditions, comprising at<br>
least one cycle of primer annealing, primer extending and denaturing, using the same primers<br><br>
as used in step (a) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence<br>
corresponding to each 5"-end portion of the primers used in step (a), under conditions in which<br>
each primer anneals to the 3"- and 5"-ends of the amplification product, respectively, whereby<br>
the amplification product containing site-directed mutation site is re-amplified.<br>
5	This specific embodiment relates to site-directed mutagenesis.<br>
Since this application using the ACP of this invention employs in principle the present methods for amplification of nucleic acid sequence previously discussed, the common descriptions between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br>
10	The formula of ACP for PCR mutagenesis is identical to the formula (1) in which the 3"-<br>
end portion comprises r f *qu* nc, for site-directed mutagenesis or for random mutation.<br>
In still further aspect of this invention, there is provided a kit for mutagenesis in a target nucleic acid, which comprises the annealing control primer or annealing control primer set described above. The descriptions of Ae kits for the amplification of nucleic acid sequence of<br>
15      this invention can be applied to the present kit.<br>
XI. Other Applications<br>
The ACP of the subject invention can be also useful in a variety of processes involving nucleic acid amplifications, particularly, PCR. For example, the processes include mixed oligonucleotide-prirned amplification of cDNA, long-range PCR, linear PCR, inverse PCR,<br>
20 quantitative t-CR, touchdown PCR, sequencing, in situ PCR, vectorette PCR and thermal asymmetric interlaced PCR The general procedures for these methods can be found in Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001) and M.J. McPherson, et al., PCR, Springer-Verlag New York Inc., N.Y,(2000).<br>
25	Therfore, the present invention encompasses all uses of the primer according to ACP for<br>
process involving nucleic acid amplification, particularly PCR.<br>
The ACP of this invention is significantly effective and widely accessible to nucleic acid<br>
amplification-based applications. Also, various problems related to primer annealing specificity<br>
3 0      remained in the previous nucleic acid amplification techniques can be fundamentally solved by<br><br><br>
the ACP and the methods of the present invention. The main benefits to be obtained from the use of the ACP during nucleic acid amplification are as follows:<br>
(a) since the presence of a regulator portion positioned between the 3"- and 5 "-end portions<br>
restricts primer annealing portion to the 3"-end portion under such conditions that the 3"-end<br>
5 portion anneals to the template, the annealing sequence of a primer can be precisely controlled,<br>
which makes it possible to design a primer caplable of having only a desired number of<br>
sequence annealed (or possible to design a primer capable of controlling an annealing portion<br>
thereof). It is particularly useful when an annealing portion of a primer has to be limited (e.g.,<br>
SNP genotyping, DNA microarrary screening, and detection of differentially expressed genes);<br>
10	(b) since the presence of a regulator portion positioned between the 3"- and 5"-end<br>
portions interrupts the annealing of the 5"-end porli- -n to ?:-$ template under such conditions that the 3 "-end portion anneals to the template, eventually the 5 "-end portion not involved in the annealing provides the 3"-end portion with primer annealing specificity;<br>
(c)	the specificity of primer annealing is highly sensitive enough to detect even a single-<br>
15      base mismatching. It is particularly useful for single nucleotide polymorphisms(SNPs)<br>
genotyping;<br>
(d)	the ACP is capable of providing a primer with a high tolerate in "primer search<br>
parameters" for primer design such as primer length, annealing temperature, GC content, and<br>
PCR product length;<br>
20	(e) the ACP provides two-stage nucleic acid amplifications which allow the amplified<br>
products to be excluded from non-specific amplification;<br>
(f)	the efficiency of nucleic acid amplification is increased, which makes it easier to detect rare mRNAs; and<br>
(g)	the reproducibility of nucleic acid amplification products is increased, which saves a<br>
2	5      great amount of time and cost.<br>
As much as the nucleic acid amplification technology such as PCR has influenced the biotechnological field, the use of ACP fundamentally alter the principles of the current existing nucleic acid amplification methods, as mentioned above, by exhibiting unlimited applicability, and have them significantly upgraded at one time. In consequence, the ACP and its various<br>
3	0      applications described herein provide a turning point to open a new biotechnological era since<br>
the introduction of nucleic acid amplification technology.<br><br><br>
The following specific examples are intended to be illustrative of the invention and should not be construed as limiting the scope of the invention as defined by appended claims.<br>
EXAMPLES<br>
5	In the experimental disclosure which follows, the following abbreviations apply to: M<br>
(molar), mM (millimolar), uM (micromolar), g (gram), fig (micrograms), ng (nanograms), 1 (liters), ml (milliliters), nl (microliters), °C (degree Centigrade); Promega (Promega Co., Madison, USA); Clontech (CLONTECH Laboratories, Palo Alto, USA); Roche (Roche Diagnostics, Mannheim, Germany); QIAGEN (QIAGEN GmbH, Hilden, Germany). 10     The primers used in the subject invention are shown in Table I.<br>
EXAMPLE 1: Evaluation of Universal Base Effect in ACP<br>
The effect of universal base residues positioned between the 3" - and 5" -end portions of ACP was evaluated by RT-PCR using mouse conceptus tissues.<br>
15	Total RNA was isolated from the entire conceptuses of mouse strain ICR at the day of 4.5,<br>
11.5 and 18.5 during gestation period using either Tri-reagent (Sigma), or the LiCl/Urea method (Hogan et al., 1994) as previously described (Chun et al., 1999; Hwang et al., 2000). Two individual experiments of cDNA amplifications using ACP were performed to examine the effect of universal base, particularly, deoxyinosine residues positioned between the 3" - and<br>
20       5" -end portions of ACP as follows: A. The effect of deoxyinosine residues positioned between the 3" - and 5" -end portions of ACP in comparison with ACP and the conventional primer not containing a dexoyinosine group; B. The effect of deoxyinosine residues positioned between the 3"- and 5"-end portions of ACP in association with the alteration of number of dexoyinosine.<br>
2	5	These experiments were conducted based on the following assumptions:<br>
(i) the presence of universal base residues which have lower Tm than other portions in ACP due to their weaker hydrogen bonding interactions in base pairing would not be involved in annealing to the template nucleic acid under the conditions that the 3" -end portion of ACP anneals to a site of the template at a first annealing temperature.<br>
3	0	(ii) the presence of at least one universal base residue between the 3" - and 5" -end<br>
portions of ACP would be capable of interrupting the annealing of the 5" -end portion and restricting a primer annealing portion to the 3" -end.<br><br>
(iii) the 3" -end portion of ACP would act only as a annealing portion to the template during PCR.<br>
(iv) the 3" -end portion of dT-ACP which is dT^ comprising 10 T nucleotides also has too<br>
low Tm to bind the template nucleic acid.<br>
5	(v) consequently, the dTw-ACP does not produce any PCR products under high annealing<br>
temperature.<br>
A. The effect of deoxyinosine residues positioned between the 3" - and 5" -end portions of<br>
ACP in comparison with ACP with the primer not containing a dexovmosine group<br>
10	(a) First-strand cDNA synthesis<br>
dTI0-JYC2 i - G V GACTACGATACTGTGCGATITITrTTTT -3" (SEQ ID NO:29) or dT10-ACPl 5"- GCTTGACTACGATACTGTGCGAUllirrrri"rrrrr-S" (SEQ ID NO:30) was used as a cDNA synthesis primer.<br>
Three micrograms of totai RNA and 2 ul of 10 uM dT10-JYC2 or 10 uM dTio-ACPl were 15 combined in a 20 ul final volume. The solution was heated at 65°C for 10 minutes, quenched on ice, and microcentriruged to collet solvent at the bottom. The following components were added sequentially to the annealed primer/template on ice: 0.5 ul (40 units/|iJ) of RNasin ribonuclease inhibitor (Promega), 4 ul of 5x reaction buffer (250 mM Tris-HCl, pH 8.3, 375 mM KC1, 15 mM MgCl2, 50 mM DTT; Promega), 5 ul of 2 mM each deoxynucleoude mix<br>
2	0      (dATP, dCTP, dGTP, dTTP), and 1 ul of Moloney-murine leukemia virus (M-MLV) reverse<br>
transcriptase (200 units/ul; Promega). The 20 ul of reaction mixture was incubated at 37°C for 90 min, microcentrifuged, and placed on ice for 2 min. The reaction was stopped by incubation at 94°C for 2 min.<br>
25	(b) cDNA amplification using ACPs<br>
The dTio-ACPl was used to examine the effect of a deoxyinosine group positioned between the 3"- and 5"-end portions during PCR. The dTi0-JYC2 not containing a deoxyinosine group was used as a control.<br>
The ACP 10 5"- GTCTACCAGGCATTCGCTTCATlIfflGCCATCGACC -3" (SEQ ID<br>
3	0     NO: 13) was used as 5" primer for this experiment.<br>
The PCR amplification was conducted in a 50 ul volume containing 50 ng of the first-strand cDNA, 5 ul of 10 x PCR buffer, 1 ul of 10 uM 5"primer (ACP 10), lul of 10 uM<br><br>
3" primer (dTi<rjyc2 or dti0-acpl ul of mm mgcl2 dntp taq polymerase units the pcr reactions were conducted under following conditions: min at followed by cycles for and a final extension amplified products analyzed electrophoresis in agarose gel ethidium bromide staining.></rjyc2>
As a result, FIG. 8 shows that the dTjo-ACPl containing a deoxyinosine group produced almost no products (lanes 4-6), whereas the dTi0-JYC2 not containing a deoxyinosine group produced a plurality of amplified cDNA products (lanes 1-3). Consistent with our assumption, the results clearly indicate that the deoxyinosine group positioned between the 3" -and 5" -end 10 portions affects the annealing of the 3" - and 5" -end portionr of the dTio-ACP to the template cDNA under such high annealing temperature, resulting ru AO F ^ Jet as stated in the above assumption.<br>
B. The effect of deoxyinosine residues positioned between the 3" - and 5" -end portions of<br>
ACP in association with the alteration of number of dexoyinosine<br>
15	(a) First-strand cDNA synthesis<br>
The first-strand cDNA was synthesized from total RNA of mouse concentucs using dTio-JYC2 as a cDNA synthesis primer as the above.<br>
(V) cDNA amplification using ACPs<br>
20	This experiment used four ACPs each comprising different number of deoxyinosine<br>
residues as follows, to examine the effect of deoxyinosine residues positioned between the- 3" -<br>
and 5" -end portions in association with the alteration of number of deoxyinosine, under a<br>
particular stringency conditions.<br>
ACP16 5" - GTCTACCAGGCATTCGCTTCATnGCCATCGACC -3" (SEQ ID NO:20); 25  ACP17 5" - GTCTACCAGGCATTCGCTTCATnnGCCATCGACC-3" (SEQ ID NO:21);<br>
ACP 18 5" - GTCTACCAGGCATTCGCTTCATmniGCCATCGACC -3" (SEQ ID NO:22);<br>
ACP19 5"- GTCTACCAGGCATTCGCTTCATimimGCCATCGACC -3" (SEQ ID<br>
NO.-23); and<br>
CRP2I0 5"- GTCTACCAGGCATTCGCTTCATGCCATCGACC -3" (SEQ ID NO: 19) not 3 0      containing a deoxyinosine group was used as a control.<br><br><br>
The resultant first-strand cDNA generated from step (A), which comprises the pre-selected<br>
arbitrary sequence of the dTjo-ACP at its 5" -end, was used as a template and the primer JYC2<br>
5" - GCTTGACTACGATACTGTGCGA -3" (SEQ ID NO: 10) corresponding to the 5" -end<br>
portion of the dTi0-ACP was used as 3" primer.<br>
5	The PCR amplification was conducted in a 50 ul volume containing 50 ng of the first-<br>
strand cDNA, 5 ul of 10 x PCR buffer, 1 ul of 10 pM 5"primer (ACP16, 17, 18, 19, or CRP2I0), lul of 10 uM 3" primer (JYC2), 3 ul of 25 mM MgCl2,5 ul of 2 mM dNTP, 0.5 ul of Taq polymerase (5 units/ul). The PCR reactions were comprised of: 5 min at 94°C, followed by 30 cycles of 94°C for I min, 57°C for 1 min, and 72°C for 1 min; followed by a 5 min final<br>
10  extension at 72°C. Amplified products were analyzed by electrophoresis in a 2% agarose gel followed by ethidium bromide staining.<br>
As a result, FIG. 9 shows that the CRP2I0 not containing any deoxyinosine residues produced a plurality of amplified cDNA products, whereas the ACPs containing at least two deoxyinosine residues generated the significant reduction of amplified cDNA products, and<br>
15  even more, the ACP containing eight deoxyinosine residues produced almost no products. Consistent with our assumption, the results clearly indicates that the annealing of the 3" -end p vtion of ACP to the template could be separated from the 5" -portion since a group of contiguous deoxyinosine residues separates the annealing of the 3"-end and 5"-end portions under high stringent conditions due to the property of deoxyinosine such as its weaker<br>
2 0      hydrogen bonding interaction in base pairing.<br>
EXAMPLE 2: Method for Amplifying a Target Nucleic Acid Sequence Using ACP<br>
The ACP of flie subject invention was applied to amplify target nucleotide sequences of mouse placenta-specific homeobox gene Esxl cDNA. The process and results for the<br>
2	5      amplification of the target nucleotide sequences of Esxl cDNA using ACPs are described<br>
herein. Total RNA (3 ug) obtained from mouse 18.5-day-old placenta was used as a starting material. First-strand cDNAs were prepared under the same conditions as used in the cDNA synthesis of Example 1, except that 01igo-dT15 was used as the first-strand cDNA synthesis primer.<br>
3	0      01igo-dT,s 5" -TTTTTTTrrTTTTTT-3" (SEQ ID NO:54)<br><br>
The resultant first-strand cDNAs were used as templates to amplify target cDNA<br>
fragments of Esxl using ACPs. These experiments conducted two stage PCR amplifications,<br>
which is one of unique features of the present invention.<br>
The conventional primers of Esxl used in the Example are: 5      EsxN7       5"-GCCGGTTGCAGAAGCACC-3" (SEQ ID NO:44);<br>
EsxC6      5" -GAACCATGTTTCTGAATGCC-3" (SEQ ID NO:45);<br>
EsxNl       5" -GAATCTGAAACAACTTTCTA-3" (SEQ ID NO:48);<br>
EsxC2      5" -GATGCATGGGACGAGGCACC-3" (SEQ ID NO:49);<br>
EsxN3       5" -CGCCGCAACCCCTGCX:CGCA.3" (SEQ ID NO:51); and 10      EsxC5       5" -GATGCATGGGACX5AGGCA-3" (SEQ ID NO:52).<br>
Three primer seV Es."N: and EsxC6, EsxNl and EsxC2, and EsxN3 and EsxC5,<br>
were used in the Example because they are known as the primer sets which generate high<br>
backgrounds as well as non-specific products in conventional PCR methods as known in the<br>
art.<br>
15	According to single-target PCR systems, primers with similar melting temperatures (Tm)<br>
should be chosen. However, a primer set of EsxNl (Tm 50.7°C) and EsxC2 (Tm 71.9°C) shows<br>
about 20°C of different melting temperatures Istwcen them, and a primer set of EsxN3 (Tm<br>
86.9°C) and EsxC5 (Tm 66.2°C) both has high melting temperatures. Also, a primer set of<br>
EsxN7 (Tm 68.2°C) and EsxC6 (T» 61.2°Q, which has relatively similar melting temperature, 20      are selected to observe the effect of ACP.<br>
The ACP of the subject invention was applied to these three conventional primer sets to<br>
demonstrate if the ACP system can overcome the main problems arising from these<br>
conventional primer sets, such as background and non-specific products.<br>
The following ACPs comprise the sequences of the above conventional primers at their 25     3"-end portions and were used as Esxl gene-specific primers for the first-stage PCR<br>
amplification:<br>
EsxN7-ACP 5" primer 5"-GTCTACCAGGCATTCXXrrTCATninGCCGGTTGCAGAAGC<br>
ACC-3" (SEQIDNO:46);<br>
EsxC6-ACP  3"   primer 5"-  GCTTGACTACGATACTGTGCGAimiGAACCATG"nTCT 3 0      GAATGCC-3" (SEQ ID NO:47);<br>
EsxNl-ACP 5" primer 5" ^TCTACCAGGCATTCXSCTTCATninGAATCTGAAACAACT<br>
TTCTA-3" (SEQIDNO:50);<br><br>
EsxC2-ACP 3" primer 5"-GCTTGACTACGATACrGTGCGAinnGATGCATGGGACGAG GCACC-3" (SEQE&gt;NO:55);<br>
EsxN3-ACP 5"  primer   5" -GTCTACCAGGCATrCGCTTCATTtniCGCCGCAACCCCTG CCCGCA-3" (SEQ ID NO:53); and 5      EsxC5-ACP 3" primer   5" -GCTTGACrACGATACTGTGCGAIiniGATGCATGGGACGA GGCA-3"(SEQIDNO:56).<br>
The 5"-end portion sequences of the ACPs were served as pre-selected arbitrary primer sequences only for the second-stage PCR amplification: JYC2 and JYC4 5"-GTCTACCAGGCATTCGCTTCAT -3" (SEQ ID NO:l2).<br>
10	During the first-stage PCR amplification, the primer set of EsxN7-ACP and EsxC 5-ACP<br>
was used as 5" and 3" primers, respectively, to generate the 520-bp fragment PC "jae IL/" cDNA, the primer set of EsxNl-ACP and EsxC2-ACP was used as 5" and 3" primers, respectively, to generate the 784-bp fragment of the Esxl cDNA, and the primer set of EsxN3-ACP and EsxC5-ACP was used as 5" and 3" primers, respectively, to generate the 483-bp<br>
15      fragment of the Esxl cDNA.<br>
During the second-stage PCR amplification, JYC4 and JYC2 were used as pre-selected arbitrary 5" and 3" primers, respectively (PROTOCOL A). As an alternative, the complete sequences of the ACPs, instead of the pre-selected arbitrary primers such as JYC4 and JYC2, can be used as 5" and 3" primers for the second-stage PCR amplification at the high stringent<br>
20      conditions. In this case, it is not necessary to add the pre-selected arbitrary primers to the reaction mixture at the time of or after the first-stage PCR reaction (PROTOCOL B).<br>
PROTOCOL A: One-stop two-stage PCR Amplifications<br>
(A) First-stage PCR Amplification<br>
The first-stage PCR amplification was performed by hot start PCR method in which the<br>
25      procedure is to set up the complete reactions without the DNA polymerase and incubate the<br>
tubes in the thermal cycler to complete the initial denaturation step at &gt;90°C, Then, while<br>
holding the tubes at a temperature above 70°C, the appropriate amount of DNA polymerase can<br>
be pipetted into the reaction.<br>
The first-stage PCR amplification was conducted by two cycles of PCR consisting of<br>
3 0      annealing, extending and denaturing reaction; the reaction mixture in a final volume of 49.5 \icontaining 50 ng of the first-strand cDNA, 5 ul of 10 x PCR reaction buffer (Promega), 5 jxl of<br><br><br><br><br><br><br>
25 mM MgCl2, 5 al of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), 1.35 ul of 5" ACP (1 uM) and 1.35 u.1 of 3" ACP (1 uM) is pre-heated at 94°C, while holding the tube containing the reaction mixture at the 94"C, 0.5 \il of Taq polymerase (5units/ul; Promega) is added into the reaction mixture; the PCR reactions comprise two cycles of 94°C for 40 sec, 60°C for 40 5     sec, and 72°C for 40 sec; followed by denaturing the amplification product at 94°C.<br>
(B) Second-Stage PCR Amplification<br>
The resultant cDNA product generated by the first-stage PCR amplification using Esxl gene-specific ACPs was then amplified by the following second-stage PCR amplification under higher annealing temperature. After the completion of the first-stage PCR amplification, each 1 10 ul of 10 uM pre-selected arbitrary primers, JYC4 and JYC2, was added into the reaction mixture obtained from the first-stage PCR amplification, under denaturing temperature such as at 94°C. The second stage-PCR reaction was as follows: 35 cycles of 94°C for 40 sec, 68°C for 40 sec, and 72"C for 40 sec; followed by a 5 min final extension at 72°C.<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected 15 by staining with ethidium bromide. The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al„ 1997; Smith et al., 1997), and the use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al., 20      1996).<br>
As shown in FIGs. 10A-C, the one-stop two-stage PCR amplifications for Esxl using each primer set of EsxN7-ACP and EsxC6-ACP, EsxNl-ACP and EsxC2-ACP, and EsxN3-ACP and EsxC5-ACP generated a single band which corresponds to the expected size, 520-bp (FIG. 10A, lane 2), 784-bp (FIG. 10B, lane 4), and 483-bp (FIG. 10C, lane 3) of Esxl cDNA<br>
2	5      fragments, respectively. Subsequent cloning and sequence analysis of the clones confirm that<br>
the band is Esxl cDNA fragments. In contrast, the conventional primer sets, which contain the sequences corresponding only to the 3" -end portions of each ACP sets, produced non-specific products as well as high backgrounds such as DNA smear (FIG. 10A, lane 1; FIG. 10B, lane 3; FIG. 10C, lanes 1 and 2). Since the PCR products using a ACP set comprise the pre-selected<br>
3	0      arbitrary sequences at their 5" - and 3" -ends, additional 54-bp sequences corresponding to the<br>
pre-selected arbitrary sequences and deoxyinosine residues were found.<br><br><br>
FIG. 10 A shows the amplified cDNA products generated by the following sets of primers;<br>
a set of EsxN7 and EsxC6 (lane 1), and a set of EsxN7-ACP and EsxC6-ACP (lane 2). PCR<br>
reactions using the conventional primer set EsxN7 and EsxC6 were as follows: 5 min at 94°C.<br>
followed by 30 cycles of 94°C for 40 sec, 60°C for 40 sec, and 72°C for 40 sec; followed by a 5<br>
5      min final extension at 72°C.<br>
FIG. 10B shows the amplified cDNA products generated by a single primer or a primer pair as follows: the primers, EsxNl and EsxC2, were used in lanes I and 2, respectively; a combination of EsxNi-ACP and conventional primer EsxC2 was used in lanes 3; two ACPs EsxNl-ACP and EsxC2-ACP were used in lane 4. When a conventional primer set, EsxNl and<br>
10 EsxC2, was used under high annealing temperature of 60°C, no specific-target product was produced. When a primer set compv jag OT ACP EsxNl-ACP and a conventional primer of EsxC2 was used, a target-specific product as well as non-specific products were amplified due to the non-specific binding of the conventional primer EsxC2 (lane 3). However, when a ACP set was used, only a single target-specific produc* was amplified (lane 4), which indicates (hat<br>
15 the ACP of the subject invention provides primers with tolerance to "primer design parameter" related to melting temperatures of general primers requested for single-target PCR systems.<br>
FIG. 10C shows the amplified cDNA products gen rated by using the following primer sets: a set of EsxN3 and EsxC5 was used in lanes 1 and 2, and a set of EsxN3-ACP and EsxC5-ACP was used in lane 3, PCR reactions using the conventional primer set of EsxN3 and BsxC5<br>
2 0 were as follows; 5 min at 94°C followed by 30 cycles of 94°C for 40 sec, 58°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final extension at 72°C (lane 1). The conventional primer set was also compared with the ACP set by conducting the same two stage PCR amplifications as used in the ACP, such mat its annealing temperature is increased from 60°C to 68°C (lane 2). These results also indicate that although the conventional primers including ones having<br>
25 high Tm are used in the same two stage PCRamphfication, they could not be free from the problems of non-specific products and background, whereas the ACP of the subject invention can help overcome such problems arising from these conventional primers.<br>
PROTOCOL B: Non-stop two-staee PCR Amplifications<br>
Alternatively, the complete sequences of the ACPs, instead of the pre-selected arbitrary<br>
30     primers such as JYC4 and JYC2, can be used as primers for the second-stage )3CR<br>
amplification at the high stringent conditions. In this case, it is not necessary to add the pre-<br><br>
selected arbitrary primers to the reaction mixture at the time of or after the first-stage PCR<br>
reaction.<br>
The process of the non-stop two-stage PCR amplifications is basically identical to<br>
Protocol A, except that the ACPs, 1 ul of 5" ACP (10 uM) and 1 nl of 3" ACP (10 uM), are 5      added at the first stage PCR amplification and the second stage PCR amplification immediately<br>
follows the first stage PCR amplification without any delay because there is no step of adding<br>
pre-selected arbitrary primers.<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected<br>
by staining with ethidium bromide. The resulting PCR products can be also detected on a 10      denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such<br>
as silver staining (Gottschlich et al., 1997; Kociok et ai, 1998), the use of fliv <ss ist nt-></ss>
labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith ec al.,<br>
1997), and the use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al.,<br>
1996).<br>
15	FIG. 10D shows the amplified cDNA products generated by the non-stop two-stage PCR<br>
Amplifications using the following single primer or a primer pair; the primers EsxNl and<br>
EsxC2 were used in lane 1 and 2, respectively; a pair of EsxNl and EsxC2 was used in lane 3;<br>
and a pair of EsxNl-ACP and EsxC2-ACP was used in lane 4. When a conventional primer set,<br>
EsxNl and EsxC2, was used, no specific-target product was produced. However, when a ACP<br>
2	0      set was used in non-stop two-stage PCR amplifications, only a single target-specific product<br>
was amplified (lane 4), which is consistent with the results of oue-stop two-stage PCR Amplifications (FIG. 10B).<br>
These examples illustrate that the ACP permits the products to be free from the background problems as well as non-specificity arising from the conventional primers used in 25      PCR methods as described in the art. It could be also understood that the ACP allows the generation of the specific products regardless of the design of gene-specific primers.<br>
EXAMPLE 3: Identification and Characterization of Differentially Expressed mRNAs during Mouse Embryonic Development Using ACP<br>
The ACP of the subject invention has been applied to detect differentially expressed<br>
3	0      mRNAs in embryonic developments. Specifically, three different procedures and results using<br><br>
different stages of conceptus total RNAs as starting materials are described herein. The primers used in the subject invention are shown in Table 1.<br>
Al. PROCEDURE 1<br>
Step (1): First-strand cDNA synthesis<br>
5	The first-strand cDNAs were prepared under the same conditions as used in the cDNA<br>
synthesis of Example 1 using the dT10-ACPl or JYC5-T,5-ACP as a cDNA synthesis primer. The resultant cDNAs were purified by a spin column (PCR purification Kit, QIAGEI^} to remove primers, dNTP, and the above reagents. It is necessary to perform the purification step prior to the determination of the cDNAs concentration using the UV spectroscopy at an 10 .*(• iorb ■«.; of 260 nm. The same amount of cDNAs from each sample was used for comparing their amplification patterns using the ACP system described herein.<br>
Step (2): x"irst-staee PCR amplification using ACP<br>
The following ACPs were used as arbitrary ACPs (AR-ACPs) for the first PCR 15      amplification:<br>
ACP3     5" - GTCTACCAGGCATTCGCTTCATfflnGCCATCGACS -3" (SEQ JD NO:3); ACP5     5" - GTCTACCAGGCATTaKJTTCATIIIIIAGGCGATGCS -3" (SEQ ID NO;5); ACP8     5" - GTCTAC^AGfGCATTCGCTTCATninCTCCGATGCS -3" (SEQ ID NO: 8); ACP10   5" NO:17).<br>
The 5"-end portion sequences of the dTio-ACPl and AR-ACPs serve as pre-selected arbitrary primer sequences only for the second-PCR amplification. The pre-seleoted arbitrary<br>
2	5     primers are JYC2 and JYC4.<br>
The first-strand cDNAs produced from step (1) were amplified by the following first-stage PCR amplification using one of AR-ACPs (ACP3, ACP5, ACP8, ACP10, AGP 13, or ACP14) and the dTi0-ACPl as 5" and 3" primers, respectively. The first-stage PCR amplification was conducted in a 50 ul volume containing 50 ng of the first-strand cDNA, 5 ul of 10 x PCR<br>
3	0      reaction buffer (Promega), 3 ul of 25 mM MgCl2, 5 ul of dNTP (0.2 mM each dATP, dCTP,<br>
dGTP, dTTP), 5 ul of 5" primer (1 uM), 5 ul of 3" primer (1 uM), and 0.5 ul of Taq<br><br><br>
polymerase (5units/ul; Promega). The PCR reactions were as follows: 5 min at 94°C followed by 20 cycles of 94°C for 1 min, 50°C for 1 min, and 72°C for 1 min; followed by a 5 min final extension at 72°C.<br>
The cycle of the first-stage PCR amplification can be varied depending on the types of 5      samples. For example, the 20 cycles of the first PCR amplification were used for mouse conceptus samples.<br>
Step (3): Second-stage PCR amplification using pre-selected arbitrary primers corresponding to the 5" -end portion sequences of ACPs<br>
10	The amplified cDNA products produced from step (2) are re-amplified by the following<br>
second-stage PCR amplification using two pif-"dcitH arbitrary primers, JYC4 and JYC2, each corresponding to the 5" -end portion sequences o/ AR-ACP and dTI0-ACPl, respectively. The second-stage PCR amplification was conducted in a 50 fxl volume containing 5 ftl of the first amplified cDNA products (50 ul), 5 pi of 10 x PCR reaction buffer (Promega), 3 ul of 25<br>
15 mM MgCl2, 5 ul of 2 mM dNTP, 1 pi of 5" primer (10 uM), 1 ul of 3" primer (10 uM), and 0.5 ul of Taq polymerase (Sunits/ul). The PCR reactions were as follows: 5 min at 94°C followed by 30 cycles of 94°C for 1 min, 65°C for 1 min, and 72°C for 1 min; followed by a 5 min final extension at 72°C.<br>
A2. PROCEDURE 2<br>
2	0	The alternative procedure composes the following steps of:<br>
(a)	providing a first sample of nucleic acids representing a first population of mRNA<br>
transcripts and a second sample of nucleic acids representing a second population of mRNA<br>
transcripts;<br>
(b)	contacting each of the first nucleic acid sample and the second nucleic acid sample<br>
25      with  a  first ACP,  wherein  the  first ACT  has  a  hybridizing  sequence  substantially<br>
complementary to a region of the first and second population of mRNA transcripts to hybridize therewith;<br>
(c)	reverse transcribing me mRNA to which the first ACP hybridizes to produce a first<br>
population of DNA strands that are complementary to the mRNAs in the first nucleic acid<br>
3	0      sample to which the first ACP hybridizes, and a second population of DNA strands that are<br><br>
complementary to the mRNA in the second nucleic acid sample to which the first ACP hybridizes;<br>
(d) purifying and quantifying the complementary DNA strands produced as a result of the<br>
reverse transcription step (c);<br>
5	(e) synthesizing a second DNA strand complementary to each of the first and second<br>
populations of DNA strands using a second ACP under low stringent conditions, by at least one<br>
PCR cycle comprising denaturing, annealing and primer extension, wherein the second ACP<br>
has a hybridizing sequence substantially complementary to the first and second populations of<br>
DNA strands;<br>
10	(f) amplifying each second DNA strand obtained from step (e) under high stringent<br>
conditions, by at least one PCR cycle comprising denaturing, annealing and primer extensor to<br>
generate first and second populations of amplification products using two pre-selected arbitrary<br>
primers each comprising a sequence corresponding to each 5" -end portion of the first and<br>
second annealing control primers; and<br>
15	(g) comparing the amount of individual amplification products in the first and second<br>
populations of amplification products.<br>
The first-strand cDNAs are synthesized using JYC5-T,5-ACP 5"-CTGTGAATGC7J GC GACTACGATllWl"lTrnTlTlTnTlT-S" (SEQ ED NO:61).<br>
The 5" -end portion sequence of the JYC5-TIS-ACP serves as a 3" pre-selected arbitra y 2 0     primer sequence to be used only for the second stage of PCR amplification: JYC5       5"-CTGTGAATGCTGCGACTACGAT-3" (SEQ ID NO:60). Step (1): First-strand cDNA synthesis<br>
I. Combine 3 fig total RNA and 2 ul of 10 uM JYC5-TIS-ACP in a sterile 0,2 ml<br>
microcentrifuge tube.<br>
25	2. Add sterile H20 to a final volume of 9.5 ul. Mix contents and spin the tube briefly in a<br>
microcentrifuge.<br>
3.	Incubate the tube at 80°C for 3 minutes or use a thermocycler for the same purpose.<br>
4.	Cool the tube on ice for 2 minutes. Spin down the contents of the tube briefly m a microcentrifuge.<br>
30	5. To the same reaction tube add the following reagents: 4 ul 5x First-strand buffer<br>
(Promega), 5 ul dNTP (2mM each dATP, dCTP, dGTP, dTTP), 0.5ul RNasin inhibitor (40 units/ul, Promega) and   1 ul M-MLV reverse transcriptase (200 U/uJ).<br><br><br>
6.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
7.	Incubate the tube at 42 °C for 90 min.<br>
8.	Incubate the tube at 94 °C for 2 minutes to terminate first-strand synthesis.<br>
9.	Place the tube on ice for 2 min.<br>
5	10. Purify the resultant cDNAs by a spin column (PCR purification Kit, QIAGEN) to<br>
remove primers, dNTP, and the above reagents.<br>
11.	Next, measure the concentration of the cDNAs using the UV spectroscopy at an<br>
absorbance of 260 run.<br>
12.	Process to step 2.<br>
10<br>
Step (2): Sec jd sp ad cDNA synthesis using ACP<br>
The same amount of cDNAs from each sample was used for the comparison of their amplification patterns using the ACPs described herein. The second-strand cDNA was synthesized using arbitrary ACP10 by hot start PCR method in which the procedure is to set up 15 the complete reactions without the DNA polymerase and incubate the tubes in the thermal cycler to complete the initial denaruration step at &gt;90°C. Then, while holding the tubes at a temperature above 90°C, the appropi-ate amount of DNA polymerase can be pipetted into the reaction.<br>
1.	Combine the following reagents in a sterile 0.2 ml microcentrifuge tube: 49.5 ul of the<br>
2 0      total volume containing 1 ul of first-strand cDNA (SO ng/ul) prepared by step 1, 5 pi of 10 x<br>
PCR buffer (Roche), 5 ul of 2 mM dNTP, 1 ul of 10 uM arbitrary ACP (5" primer) and 37.5 ul of sterile dH20.<br>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Place the tube in the preheated thermal cycler at 94°C.<br>
25	4. Add the 0.5 ul of Taq polymerase (5units/ul; Roche) into the reaction, while holding the<br>
tube at the temperature 94°C.<br>
5. Conduct PCR reaction under the following conditions: one cycle of 94°C for 5 min, 50 °C for 3 min, and 72°C for 1 min; followed by denaturing the first amplification produot at 94°C. 30<br>
Step (3): PCR Amplification of the second-strand cDNAs using pre-selectod arbitrary primers corresponding to the 5" -end portion sequences of ACPs<br><br><br>
1. After the completion of the first stage PCR amplification, while holding the tubes at a<br>
temperature above 94°C, add 2 ul of 10 uM JYC4 and 2 ul of 10 uM JYC5, in which each<br>
corresponds to the 5" -end portion sequences of both 5" and 3" ACPs, respectively, into the<br>
reaction mixture used in step (2).<br>
5	2. Conduct second stage PCR reactions under the following conditions: 40 cycles of 94°C<br>
for 40 sec, 68°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final extension at 72°C.<br>
A3, PROCEDURE 3<br>
As an alternative process, in the step (f) of PROCEDURE 2 the complete sequences of the first and second ACPs used in the steps (b) and (e) of PROCEDURE 2, instead of the pre-10 selected arbitrary sequences of the 5" -end portions of the f&gt; •. an&gt;: e.- jond ACPs, can be used as 3" and 5" primers, respectively, at the high stringent conditions for amplifying each second DNA strand obtained from the step (e) of PROCEDURE 2, wherein the 3" - and 5" nsnds of the second DNA strands which were initially synthesized using the second ACP comprise the sequence of the first ACP and the complementary sequence of the second ACP, respectively, 15 and also serve as perfect pairing sites to the first and second ACPs. In thh case, it is not necessary to add the pre-selected arbitrary primers to the reaction mixture at the une of or after first-stage PCR reaction.<br>
Step (1): First-strand cDNA synthesis<br>
The first-strand cDNAs were prepared under the same conditions as used in the cDNA 2 0      synthesis of PROCEDURE 2 using the JYC5-TiS-ACP as a cDNA synthesis primer.<br>
Step (2): Second-strand cDNA synthesis and amplification using non-stop two-jjfego ?CR, The same amount of cDNAs from each sample was used for the comparison of their amplification patterns using the ACPs described herein. The second-strand oDNA was 2 5 synthesized using arbitrary ACP10 by hot start PCR method in which the procedure is to set up the complete reactions without the DNA polymerase and incubate the tubes in the thermal cycler to complete the initial denaturation step at &gt;90°C. Then, while holding the tubes at a temperature above 90°C, the appropriate amount of DNA polymerase can be pipetted into the reaction.<br>
30	1. Combine the following reagents in a sterile 0.2 ml microcentrifuge tube: 49.5 |it of the<br>
total volume containing 1 ul of first-strand cDNA (50 ng/ul) prepared by step 1, 5 jil of 1.0 x<br><br><br>
PCR buffer (Roche), 5 ul of 2 mM dNTP, 1 ul of 10 uM arbitrary ACP10 (5" primer), 1 ul of 10 uM JYC5-Tls-ACP (3" primer) and 36.5 ul of sterile dH20.<br>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Place the tube in the preheated thermal cycler at 94°C.<br>
5	4. Add the 0.5 ul of Taq polymerase (5un|ts/ul; Roche) into the reaction, while holding the<br>
tube at the temperature 94°C.<br>
5. Conduct PCR reaction under the following conditions: one cycle of 94°C for 1 min, 50°C for 3 min, and 72°C for 1 min; followed by 40 cycles of 94°C for 40 sec, 65°C for 40 sec, and 72°C for 40 sec; and followed by a 5 min final extension at 72°C.<br>
10 B. Separation of amplified PCR products bv electrophoresis analysis and recovery of the differentially displayed bands<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected by staining with ethidium bromide. Several major bands differentially expressed during embryonic development (E4.5, El 1.5, and E18.5) were selected, excised and extracted from the<br>
15 gels using GENECLEAN II Kit (BIO 101). The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluorcscn."T( it-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al, 1997; Smith ci al., 1997), and the use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al.,<br>
20      1996).<br>
C.	Re-amplification of the recovered bands<br>
The bands obtained from step B were re-amplified using the same pre-selectcd arbitrary primers and PCR conditions as used in PROCEDURE 1,2 and 3.<br>
D.	Cloning and sequencing of the re-amplified fragments<br>
25	Each amplified fragment was cloned into the pGEM-T Easy vector (Piomega) and<br>
sequenced with the ABI PRISM 310 Genetic Analyzer (Perkin Elmer Biosystem) ueiii£, RigDye Terminator cycle sequencing kit (Perkin Elmer). Computer-assisted sequence analysis was carried out using the BLAST search program (Basic Local Alignment Search Tool).<br><br>
E. Northern analysis<br>
Twenty micrograms of total RNA from conceptus tissues were resolved on denaturing 1% agarose gels containing formaldehyde, transferred onto nylon membranes (Hybond-N, Amersham, USA), and hybridized with a 32P-Iabeled subcloned PCR product in QuikHyb solution (Stratagene, USA) overnight at 58CC as previously described (Chun et al., 1999; Hwang et al., 2000). Blots were washed at 65°C twice for 20 min in 2 x SSC, 0.1% SDS, twice for 20 min in 1 x SSC, 0.1% SDS, and twice for 20 min in 0.1 x SSC, 0.1% SDS. The membranes were exposed to Kodak X-Omat XK-1 film with a Fuji intensifying screen at -80°C.<br>
FIGs. 11A-D shows the amplified cDNA. products, wherein mouse conceptus samples obtained from different stages were amplilVd b rlOCEDURE 1 using the primer sets as follows; a set of ACP3 and dTl0-ACPl for the lanes 1-3 of FIG. 11A; a set of ACP5 and dT10-ACP1 for the lanes 1-6 and of FIG. 1 IB and a set of ACP8 and dT,0-ACPl for "he lanes 7-12 of FIG. 10B, respectively. FIG. 1 IB also shows additional results of the amplified cDNA products generated by using another ACP sets. FIGs. 11C-D shows the amplified products generated by using two primer sets of the ACP10 and dT10-ACPl(FIG. 11C), and ACP14 and dT,0-ACP1(FIG. 11D), respectively. Many differentially expressed bands in a specific stage were obtained, subcloned into the pGEM-T Easy vector (Promega), and sequenced. Sequence analysis reveals that all of the clones are known genes except two novel genes (Table 2). The expression patterns were confirmed by Northern blot analysis using mouse conceptus stage blot (Seegene, Inc., Seoul, Korea).<br>
FIG. 12A shows the amplified cDNA products, wherein mouse conceptus samples (E4.5: lane 1; E11.5: lane 2; E18.5: lane 3) obtained from different stages were amplified by PROCEDURE 2 using a set of ACP10 and JYC5-TiS-ACP. Many differentially expressed bands in a specific stage were obtained, subcloned into the pGEM-T Easy vector (Promega), and sequenced. Sequence analysis reveals that all of the clones are known genes except one DEG 2 (Table 2). The expression patterns were confirmed by Northern blot analysis using mouse conceptus stage blot (Seegene, Inc., Seoul, Korea).<br>
FIG. 12B shows the amplified cDNA products, wherein mouse conceptus samples at the different stages of (E4.5: lane 3; El 1.5: lane 4; E18.5: lane 5) were amplified by non-stop two-stage PCR amplifications using a set of ACP10 and JYCS-TJS-ACP as above mentioned in PROCEDURE 3. When a set of ACP10 and JYC5-T,S-ACP (lanes 3-5) was used, the resultant<br><br>
bands were identical to the bands which were obtained by PROCEDURE 2 comprismg one-stop two-stage PCR amplifications (FIG. 12A). However, no products were generated when a single primer, ACP10 (lane 1) or JYC5-TiS-ACP (lane 2) was used, which indicates that the amplified products were generated only when both ACP10 and JYC5-Tis-ACP as a set were 5      used for their specific bindings.<br>
FIG. 13 shows the results of Northern blot hybridization for representing six different clones using DEG1 (A; arrow 1 of FIG. 10A, FIG. 11, and FIG. 12), DEG2 (C), DEG3 (B; arrow 2 of FIG. 10A, FIG. 11, and FIG. 12), DEG5 (E), DEG7 (F), and DEG8 (D; arrow 4 of FIG. 10A, FIG. 11, and FIG. 12) as probes. The DEG1 probe was also hybridized to the<br>
10 alternative isoform of Tropomyosin 2 (arrow 1" of FIG. 11, an«* FIG. 1°), which was discovered by this present invention. Consistent with the result", jf a v se gel analysis, Northern blot analysis showed that the expression patterns of the clones are identical to the original bands on the agarose gels, indicating that all of the clones are true positive products. Thus, the ACP produces only positive products without any false positives, which means that<br>
15      the ACP eliminates the problem of false positives.<br>
FIG. 14 -shows the results of Northern blot hybridization for the expression of DEG5 during mouse embryonic development. DEG5, which is turned out as a novel gene by sequence analysis, shows an interesting expression patterns: after a strong expression appeared in the early pregnancy stage (E4.5), its expression was gradually reduced in the middle stages and<br>
2 0     gradually increased again in the late development stage (E17.5 and El 8.5).<br>
These results indicate that the method using ACP for isolating differentially expressed genes produces only real PCR products and completely eliminates false positive products. Freedom from false positives which have been one major bottleneck remaining for the previous Differential Display technique allows avoiding the subsequent labor-intensive work required<br>
2	5      for the verification of the cDNA fragments identified by Differential Display.<br>
EXAMPLE 5: Method for Rapid Amplification of 3"-Ends of cDNA (3"-RACE) Using ACP<br>
The present example compares the ACP-based 3" -RACE and the conventional 3" -RACE in order to demonstrate if the ACP of the present invention can exclude such background<br>
3	0      problems arising from conventional oligo-dT primers used in cDNA synthesis.<br><br><br>
In the conventional 3" -RACE, the poly(A) tail of mRNA molecules is exploited as a<br>
priming site for PCR amplification and thus the oligo-dT primer is used as a 3" primer for the<br>
conventional 3" -RACE, hi contrast, the ACP of the present invention uses the poly(A) tail of<br>
mRNA as a priming site only for the cDNA synthesis but not for the subsequent PCR<br>
5      amplification.<br>
Mouse first-strand cDNAs were prepared under the same conditions as used in the cDNA synthesis of Example 1 using Oligo VdTu-ACP 5"-GCTTGACTACGATACTGTGCGAinn TTTTTTrTTTTTTTTV-3" (SEQ ID NO:57) (V is A, C or G) as a cDNA synthesis primer and then, directly used as templates for the subsequent PCR amplification without the purification<br>
10      step for the removal of the cDNA synthesis primer.<br>
FJ. the. r..nventional 3"-RACE, the first-strand cDNAs were synthesized using the following cDNA synthesis primer;<br>
CDS m/3" 5" -ATTCTAGAGGCCGAGGCGGCCGACATG-(dT)3o-VN-3" (SEQ ID NO:35) (V is A, C or G;and N is A, C, T or G).<br>
15	This cDNA synthesis primer, CDS HI/3", was used as 3" primer for subsequent PCR<br>
amplification.<br>
The PCR amplification was conducted in a 50 ul volume containing 50 ng of the first-strand cDNA, 5 ul of 10 x PCR buffer (Promega), 1 ul of a gene-specific 5" primer (10 uM), lpl of pre-selected arbitrary 3" primer JYC2 (10 uM) or CDS m/3" (10 pM), 3 ul of 25 mM<br>
2 0 MgCl2, 5 ul of 2 mM dNTP, 0.5 ul Taq polymerase (5units/ul; Promega). The PCR reactions were conducted under the following conditions: 5 min at 94°C followed by 30 cycles of 94°C for 1 min, 65°C for 1 min, and 72°C for 1 min; followed by a 5 min final extension at 72°C. Amplified products were analyzed by electrophoresis in a 2% agarose gel followed by ethidium bromide staining. The resulting PCR products can be also detected on a denaturing<br>
2 5     polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver<br>
staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled<br>
oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997),<br>
and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
FIG. 15 shows the results of beta-actin 3"-RACE. The conventional 3"-RACE (lane I)<br>
30 was compared with ACP-based 3"-RACE (lane 2). The conventional 3"-RACE method produced non-specific products as well as DNA smear background, whereas the ACP-based 3" -RACE produced only a single band, which is the expected size of 348-bp. These results<br><br>
indicate that the ACP-based 3" -RACE can exclude the background problems such as DNA smear and non-specific products.<br>
EXAMPLE 6: Method for Rapid Amplification of 5"-End (5"-RACE) and Full-length cDNAs Using ACP<br>
5	The ACP of the subject invention was also used to amplify the 5" -ends of cDNA<br>
fragments. The first-strand cDNAs were synthesized using Oligo VdTl5-ACP, or Random dN«-ACP:<br>
Oligo VdTIS-ACP     5" -GCTTGACTAOTATACTGTGCGAlIIirrTI"rrrrrrrilllTV-a" (SEQ ID NO:57), wherein V can be A, ", or G;<br>
10      Random dN«-ACP   5" ■GCTTGACi ^CG ",: .CTGTGCGATOIINNNNNN-3" (SEQ ID NO: 58), wherein N can be A, C, G, or T.<br>
After the complete synthesis of the first strand cDNA sequences present in the form of mRNA-cDNA intermediates, cytosine residues are tailed at the 3" -end of the first strand cDNA sequences by the terminal transferase reaction of reverse transcriptase in the presence of<br>
15      manganese. The 3" -ends of the first strand cDNAs wen extended using the first strand oDN A 3"-end extending ACP (rG3-ACP, rG2-ACP, or dG3-ACP) and then, directly used ;s templates for the subsequent PCR amplification without a purification step for the removal of the first strand cDNA 3" -end extending ACP as well as the cDNA synthesis primer. The sequences of the first-strand cDNA 3" -end extending ACPs are:<br>
20      rG3-ACP 5" -GTCTA<xag id no:36 rg2-acp gtctaccaggcattcgcttcatmnggr no:3 rgl-acp gtctaccagxk no:59 or dg3-acp gtctaccag ib no:38 r and d represent ribonucleotide deoxyribonucleotide respectively></xag>
25      A. First-strand full-length cDNA synthesis<br>
PROTOCOL A: First-strand cDNA synthesis using the ACP of the subject invention<br>
1. Combine the followings in a sterile 0.2 ml microcentrifuge tube: 3 (j,g of total RNA and 2 ul of 10 uM of Oligo VdT,5-ACP or random dNfi-ACP.<br><br>
2.	Add sterile H20 to a final volume of 10 ul. Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Incubate the tube in a 65°C water bath for 15 minutes or use a thermocycler for the same purpose.<br>
5	4. Cool the tube on ice for at least 2 minutes. Spin down the contents of the tube briefly in<br>
a microcentrifuge.<br>
5.	Add the following reagents to the same reaction tube: 4 ul of 5x first-strand buffer<br>
(hvitrogen), 1 ul of 0.1 M DTT, 2 ul of BSA (lmg/ml), 2 ul of dNTP (10 mM each dATP,<br>
dCTP, dGTP, dTTP), 0.4 ul of 100 mM MnCl2 and 0.5ul of RNasin inhibitor (40 units/ul,<br>
10      Promega).<br>
6.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
7.	Incubate the tube at 42°C for 2 minutes in an incubator or thermocycler,<br>
8.	Add 1 fil of Superscript II reverse transcriptase (200 units/ul; Ihvitrogen).<br>
9.	Incubate the tube at 42°C for 1 hour in an incubator or thermocycler.<br>
15	10. Add 1 ul of 10 uM first strand cDNA 3" -end extending ACP (rG3-ACP, rG2-ACP, or<br>
dG3-ACP).<br>
11.	Add 0.3 ul of Superscript II reverse transcriptase (200 units/ul; Invitrogen).<br>
12.	Incubate the tube at 42°C for 30 minutes in an incubator or thermocycler.<br>
13.	Incubate the tube at 70°C for 15 minutes in an incubator or thermocycler to terminate 2 0      first-strand synthesis.<br>
14.	Place the tube on ice or can be stored at -20°C.<br>
PROTOCOL B: First-strand full-length cDNA synthesis bv CaoFinder method<br>
The following primers are used in the CapFinder method (Clontech): SMART IV™  Oligonucleotide  5"-AAGCAGTGGTATCAACGCAGAGTGGCCATTACG<br>
2	5      GCCr(GGG)-3" (SEQ ID NO:33); and<br>
5" PCR primer 5"-AAGCAGTGGTATCAACGCAGAGT-3" (SEQ ID NO:34), and CDS HI/3" PCR primer.<br>
1.	Combine the followings in a sterile 0.2 ml microcentrifuge tube: 3 ug of total<br>
RNA, 1 ul of 10 uM CDS m/3" PCR primer (Clontech) and 1 ul of 10 uM SMART IV<br>
3	0      Oligonucleotide (Clontech).<br>
2.	Add sterile H20 to a final volume of 5 ul. Mix contents and spin the tube briefly in a<br>
microcentrifuge.<br><br>
3.	Incubate the tube at 72°C for 2 minutes.<br>
4.	Cool the tube on ice for 2 minutes. Spin down the contents of the tube briefly in a microcentrifuge.<br>
5.	Add the following reagents to the same reaction tube: 10 ul of the total volume containing 2 pi of 5x first-strand buffer (Clontech), 1 ul of 20 mM DTT, 1 ul of dNTP (10 mM each dATP, dCTP, dGTP, dTTP) and 1 ul of PowerScript Reverse Transcriptase (Clontech).<br>
6.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
7.	Incubate the tube at 42°C for 1 hour<br>
8.	Place the tube on ice or can be stored at -20°C.<br>
s?. PC ** "a nplification<br>
PROTOCOL C: Amplification of a target 5"-end cDNA fragment using ACP system or<br>
conventional 5" -RACE method<br>
The present example compares the current CapFinder 5"-RACE technology and ACP-based 5" RACE method, wherein the current CapFinder 5"-RACE technology oould not exclude the high background due to residual amount of the primers during the process, hi order to demonstrate if the ACP of the present invention can eliminate such background puibb.ms arising from primers such as the CapFinder primer, SMART IV Oligonucleotide (doou eh), and cDNA synthesis primer, CDS DI/3" PCR primer (Clontech), used in cDNA synthesis, both the ACP-based 5"-RACE and the CapFinder 5"-RACE for the amplification of 5" -emb; of mouse JunB and beta-actin cDNAs were conducted in the same conditions. The moust ,binB mRNA is a relatively rare transcript in mouse 18.5-day-old placenta RNA, whereas mouse beta-actin is a relatively abundant.<br>
1.	Combine the following reagents in a sterile 0.2 ml microcentrifuge tube: 50 \xl of the total volume containing 1 ul of first-strand cDNA prepared from Protocol A or B, 5 | il of ) 0 x PCR buffer (Promega), 5 ul of 25 mM MgCl2) 5 ul of 2 mM dNTP, 1 ul of 10 )iM s\ene-specific 5" -RACE primer, 1 ul of 10 uM JYC2 or 5« PCR primer (Clontech), 0.5 |.U of Taq Polymerase (5 units/ul; Promega) and 31.5 ul of sterile dH20.<br>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Conduct PCR reaction under the following conditions: 5 min at 94°C, followed by 30 cycles of 94°C for 40 seconds, 58°C for 40 seconds, and 72°C for 1 min 30 sec; followed by a 5 min final extension at 72°C.<br>
^<br>
4. Analyze the amplified products by electrophoresis in a 2% agarose gel followed by ethidium bromide staining.<br>
The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
As shown in FIG. 16, the CapFinder methods for mouse JunB and beta-actin 5" -RACE using the 5" PCR primer (Clontech) and the gene-specific primer produced high backgrounds such as DNA smear (lanes 1 and 3) as described by many researchers (Chenchik et al., 1998; Mate et al., 1999; Schramm et al., 2000), v&gt;*eat t":: ACP-based 5"-RACE of the present invention generated only a single band which corresponds each to the expected size 155-bp or 319-bp of mouse JunB (lane 2) or mouse beta-actin (lane 4) 5"-end cDNA fragment, respectively. These examples illustrate that the ACP can bw used to fundamentally eliminate such background problems arising from contamination of primers used during cDNA synthesis, without the purification step for the removal of primers used in th"&gt; cDNA synthesis.<br>
FIG. 17 also shows that the ACP of the subject invention permits the no)w;p<fific products not to be formed which are generated by the capfinder method tin- jfist-strand cdna was synthesized either or acp and then directly used as template in me subsequent pcr amplification foj mouse prolactuvlike protein plp-c alpha alpha-specific primer is: id no:40 first-strand extending acps comprising three riboguanines lane deoxyriboguanines a combination of two find one deoxyriboguanine at cdnas so that single band corresponds expected size odna fragment produced from acp-based for></fific>
PROTOCOL D: Amplification of 5" enriched cDNA fragments using ACP<br>
The first-strand cDNAs are synthesized using Random dNg-ACP in Protocol A. J he PCR amplification was performed by hot start PCR method in which the procedure- is to set up the complete reactions without the DNA polymerase and incubate the tubes in the thermal cycler to<br><br>
complete the initial denaturation step at &gt;90°C. Then, while holding fbe tubes at a temperature above 70°C, the appropriate amount of DNA polymerase can be p^ettodirttottereaction.<br>
1 .Combine the following reagents in a sterile 0.2 ml miczocenfaMSatF lobe: 49.5 ul of the total volume containing 1 ul of first-strand cDNA prepared by Randan dM&amp;rACP in Protocol A, 5 ul of 10 x PCR buffer (Promega), 5 ul of 25 mM MgCl2, 5 pi off! wM <flw ul of um jyc2 primer jyc4 and sterile dbyol></flw>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Place the tube in the preheated thermal cycler at 94°C.<br>
4.	Add the 0.5 ul of Taq polymerase (5units/ul; Promega) intotteiffisolffiffliitwflDiteiokJing the tube at the temperature 94°C.<br>
5.	Conduct PCR reaction under the following conditions: 5 nn air 9MPffT ffidOnrai V 30 cycles of 94°C for 40 seconds, 68°C for 40 seconds, and 72°C for I anna 3© an; tidBmmcd by a 5 min final extension at 72°C.<br>
6.	Analyze the amplified products by electrophoresis in a 2% agarose jgdi fMkm 
The resulting PCR products can be also detected on a denaturing pifipwufQeiriilfe gel by autoradiography or non-radioactive detection methods such as sitwr -^WMif; ^GotnxWdh et al., 1997; Kociok et a!., 1998), me use of fluoresenscent-labelled aGgomiiidbotidles (Bauar ct aL 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and lie me of biottnfbted primers (Kom et al., 1992; Tagle ct al., 1993; Rosok et al., 1996).<br>
PROTOCOL E: Amplification of full-length enriched cDNAs usmp- ACT1<br>
The first-strand cDNAs are synthesized using Oligo VdTtrACF/ imKraftHHril A. The PCR amplification was performed by hot start PCR method as in Protocol D..<br>
1.	Combine the following reagents in a sterile 0.2 ml microcentrifuge: tfaftc: 4gJS ul of the total volume containing 1 fxl of first-strand cDNA prepared by OKgp Vdliris^iCFm Protocol A, S ul of 10 x PCR buffer (Promega), 5 ul of 25 mM MgCl2, 5 ul of 2mMcfiNI5P, 1 ul of 10 uM JYC2 (3" primer), 1 ul of 10 uM JYC4 (5" primer) and 31.5 ul of steriJfecffifeOt<br>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Place the tube in the preheated thermal cycler at 94°C.<br>
4.	Add the 0.5 ul of Taq polymerase (5units/ul; Promega, Ma&amp;sna, USA) into «K reaction, while holding the tube at the temperature 94°C.<br><br>
5.	Conduct PCR reaction under the following conditions: 5 min at 94°C followed by 30 cycles of94°C for 40 seconds, 68°C for 40 seconds, and 72°C for 1 min 30 sec; followed by a 5 min final extension at 72°C.<br>
6.	Analyze the amplified products by electrophoresis in a 2% agarose gel followed by ethidium bromide staining.<br>
The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich ct al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagh et al., 1°93; Rosok et al., 1996).<br>
To evaluate the efficierc* of /method using ACP in the amplification of full-length cDNAs, the full-length cDNAs amplified by either the above procedures of ACP method or the current CapFinder method were blotted to a Hybond-N membrane (Amersham/United States Biochemical). The mouse glyceraldehydes-i-phosphate dehydrogenase (GAPDH) cDNA was labeled with [alpha-32P]dCTP using a random labeling kit (Roche Diagnostics Co, Indianapolis, " USA) and used as a probe.<br>
As shown in FIG. 18, the GAPDH cDNA probe detected a single band which corresponds to the expected size 1,3-kb of full-length GAPDH cDNA. As expected, the signals of the PCR products generated by the above ACP method (lane 2) were several fold stronger than the ones by the CapFinder method (lane 1). This example illustrates that the ACP method of the present invention much more effectively amplifies full-length cDNAs than the CapFinder method does.<br>
EXAMPLE 7: Genomic Fingerprinting Using ACP-based Arbitrarily Primed PCR<br>
The ACP of the subject invention has been applied to detect polymorphisms in mouse. The genomic DNAs of mouse strains C57BL/6J, CBA, BALB/cJ, NOR, SPRETUS, PANCEVO, and Korean Wild Mouse were used starting materials. Genomic DNA was prepared from the liver of mice using the QIAamp Tissue Kit (QIAGEN, Hilden, Germany). The arbitrary ACPs used in the subject invention are:<br>
ACP 101    5" -GTCTACCAGGCATTCGCTTCATimiCCGGAGGATC-3" (SEQ ID NO:
<br><br>
ACPI 16   5" ^TCTACCAGGCATTCGCTTCATIiniCGGAGCATCC-3" (SEQ ID NO:66).<br>
A set of arbitrary ACPs, ACP101 and ACP109 (FIG. 19A), or ACP101 and ACPI 16 (FIG. 19B), was used as primers for mouse genomic fingerprinting. The PCR amplification was performed by hot start PCR method as described in Example 2. The genomic fingerprinting using ACP is conducted by two stages of PCR amplifications under the following conditions: amplification reactions are performed under low stringent conditions by two cycles of the first-stage PCR comprising annealing, extending and denaturing reaction; the reaction mixture in the final volume of 49.5 ul containing 50 ng of the genomic DNA, 5 uloflOx PCR reaction buffer (Promega), 5 ul of 25 mM MgCl2,5 ul of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), each 7 ul of a pair of ACPs (each 10 uM) is pre-heated at 94°C, while ho"ding the tube containing the reaction mixture at the 94°C, 0.5 ul of Taq polymera^ 4°C for 40 sec, 68°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final extension at 72°C.<br>
Amplification products were resolved and analyzed by electrophoresis in a 2.0 agarose gel which was stained with ethidium bromide and photographed. The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoreoenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al, 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 1993; Rosoketal., 1996).<br>
FIG. 19 shows the results of an experiment in which a pair of arbitrary ACPs were used to amplify segments of genomic DNA from a variety of mouse strains. To examine the reproducibility of genomic fingerprinting, the fingerprinting of each mouse strain was duplicated using two different sets of ACPs. The ACP -based PCR amplification produced several DNA segments from each set of primers and the results were reproducible. The polymorphisms were apparent between mice stains, indicating that mice stains can be distinguished through polymorphisms in genomic fingerprintings generated by ACP-based<br><br>
arbitrarily primed PCR. Thus, the ACP of the subject invention is useful to detect polymorphisms and construct genetic maps.<br>
EXAMPLE 8: Multiplex PCR Using ACP-based PCR<br>
To demonstrate the application of ACP in multiplex PCR, the portions containing single nucleotide polymorphisms of human leukocyte adhesion molecule I (ELAM1) and human p53 (TP53) genes were amplified with either conventional primers or ACP. The process and results for the multiplex PCR amplification using ACPs are described herein. DNA template was obtained from human placenta.<br>
The conventional primers for exon3 of ELAM1 (155bp) used in the Example are: ELAM1N1       5" -TTGCACACTGTTGATTCTAA-3" (SEQ ID NO:67); and ELAM1C1       5" -TTATTGATGGTCTCTACACA-3" (SEQ ID NO:68).<br>
The conventional primers for exonlO of ELAM1 (287bp) used in the Example are: ELAM1N2       5" -CCACTGAGTCCAACATTC-3" (SEQ ID NO:69); and ELAM1C2       5" -CTGAAACACTTCCCACAC-3" (SEQ ID NO:70).<br>
The conventional primers for exon4 of TP53 (349bp) used in the Example are:<br>
P."3N1	5" -:CTCTGACTGCTCTTTTCAC-3" (SEQ ID NO:71); and<br>
P53C1	5" -ATTGAAGTCTCATGGAAGCC-3" (SEQ ID NO:72).<br>
The conventional primers for exons7-8 of TP53 (750bp) used in the Example are:<br>
P53N2	5" -TGCTTGCCACAGGTCTC-3" (SEQ ID NO:73); and<br>
P53C2	5" -GCAGTGCTAGGAAAGAGG-3" (SEQ ID NO:74).<br>
These conventional primers used in the Example are known as the primers that generate non-specific products in conventional mutiplex PCR methods as known in the art.<br>
The ACPs of the subject invention were applied to these four conventional primer sets to demonstrate if the ACP can overcome the problems such as non-specific products resulting from the use of these conventional primer sets for multiplex PCR.<br>
The 3" -end portions of the ACPs comprise the sequences of the above conventional primers as follows and thus the size of ACPs is 26 bp or 27 bp bigger than that of the conventional primers:<br>
ELAM1N1-ACP    5" -GTCTACCAGGCATTCGCTTCATnnrrTGCACACTGTTGATTCTA A-3" (SEQIDNO:75);<br><br>
ELAM1C1-ACP 5" -TCACAGAAGTATGCCAAGCGAnmTTATTGATGGTCTCTACACA-<br>
3"(SEQIDNO:76);<br>
ELAM1N2-ACP    5" ^TCTACCAGGCATrCGCTTCATnmCCACTGAGTCCAACA"ITC-<br>
3"(SEQK&gt;NO:77);<br>
ELAM1C2-ACP 5"- TCACAGAAGTATGCCAAGCGAmnCTGAAACACTTCCCACAC-<br>
3" (SEQIDNO:78);<br>
P53N1-ACP 5"- GTCTACCAGGCATTCGC^TCATimCCTCTGACTGCTCnTTCAC-3"<br>
(SEQIDNO:79);<br>
P53C1-ACP 5" - TCACAGAAGTATGCCAAGCGAIIIIIATTGAAGTCTCATGGAAGCC-3"<br>
(SEQIDNO:80);<br>
P53N2-ACP     5" -    ( fC ^ CAGGCATTCGCTTCATninTGCTTGCCACAGGTCTC-3"<br>
(SEQE&gt;NO:81);and<br>
P53C2-ACP   5"-  TCACAGAAGTATGCCAAGCGAimiGCAGTGCTAGGAAAGAGG-3"<br>
(SEQIDNO:82).<br>
The 5" -end portion sequences of the ACPs comprise and serve as pre-selected arbitrary primer sequences only for the second-staf t PCR amplification:<br>
JYC3   5" - TCACAGAAGTATGCCAAGCGA -3" (SEQ ID NO: 11) and<br>
JYC4   5" - GTCTACCAGGCATTCGCTTCAT -3" (SEQ ID NO:12).<br>
Multiplex PCR amplifications were conducted by one-stop or non-stop two-stage PCR amplifications, which is a unique feature of the present invention. The PCR amplification was performed by hot start PCR method as described in Example 2.<br>
PROTOCOL A: One-stop two-stage PCR amplifications<br>
(A) First-stage PCR Amplification<br>
The first-stage PCR amplification was conducted by two cycles of PCR comprising of annealing, extending and denaturing reaction; the reaction mixture in the final volume of 49.5 ul containing 50 ng of human genomic DNA, 8 ul of 10 x PCR reaction buffer (Promega), 7 ul of 25 mM MgCl2, 5 ul of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), each 0.5 ul of each 5" ACP (10 uM) and 3" ACP (10 uM) set is pre-heated at 94°C, while holding the tube containing the reaction mixture at the 94°C, 0.5 ul of Taq polymerase (5units/ul; Promega) is added into the reaction mixture; the PCR reactions are as follows: two cycles of 94°C for 40<br><br>
sec, WC for 40 sec, and 72°C for 40 sec; followed by denaturing (he amplification product at 94°C.<br>
(B) Second-stage PCR amplification<br>
The resultant products generated by the first-stage PCR amplification using multiple sets of the ACPs were then amplified by the following second-stage PCR amplification under higher annealing temperature. After the completion of the first-stage PCR amplification, each 2 fj.1 of 10 uM pre-selected arbitrary primers, JYC3 and JYC4, was added into the reaction mixture obtained from the first-stage PCR amplification, under denaturing temperature such as at 94°C. The second stage-PCR reaction was as follows: 40 cycles of 94°C for 40 sec, 68°C for 40 sec, and 72°C for 1 min; followed by a 5 min final extension at 72CC.<br>
The amplified products were analyzed by electrophoresis in a 2% aga -v.. gel and detected by staining with ethidiurn bromide. The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al„ 1998), the use of flwresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al, 1997; Smith et al„ 1997), and the use of biotinylated primers (Kom et al., 1992; Tagle et al, 1993; Rosok et al., 1996).<br>
FIGs. 20 and 21 show the results of experiments in which three or four sets of primers were used to amplify mutiplex segments of genomic DNA at one reaction. The conventional primer sets generated non-specific products as well as specific-target products from three sets (FIG. 20A) or four sets (FIG. 21 A) of primers. In contrast, the three sets (FIG. 20B) or four sets (FIG. 21B) of ACPs produced only mutipiex target products. Thus, the ACP of the subject invention can be used for the application of multiplex PCR.<br>
PROTOCOL B: Non-stop two-stage PCR amplifications<br>
Alternatively, the complete sequences of each ACP set, instead of the pre-selected arbitrary primers such as JYC3 and JYC4, can be used as primers for the second-stage PCR amplification at the high stringent conditions. In this case, it is not necessary to add the reselected arbitrary primers to the reaction mixture at the time of or after the first-stage PCR reaction.<br>
The process of the non-stop two-stage PCR amplifications is basically identical to Protocol A, except that the second stage PCR amplification should immediately follow first<br><br><br><br><br>
stage PCR amplification without any delay because there is no step of adding pre-selected arbitrary primers and that the concentration of each ACP set, each 1 up of 5" ACP (10 ump) and 3" ACP (10 am) set, is added at the first stage PCR amplification.<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected by staining with ethidium bromide. The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Korn et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
Consistent with the results of one-stop two-stage PCR amplifications (FIG. 2 IB), non-stop two-stage PCR amplification also produced only target multiplex specific products (FIG. 21C). These examples illustrate that ACP permits the products to be free from the background problems as well as non-specificity arising from the conventional primers used in multiplex PCR methods as known in the art.<br>
EXAMPLE 9: Identification of Conserved Homology Segments in Multigene Families Using ACP<br>
The ACP of the subject invention was applied to detect and clone conserved homology segments in multigene families. In the present example, degenerate primers were designed to detect homeobox sequences. The homeobox genes are characterized by a conserved 180-bp nucleotide sequence known as the homeobox, which encodes a 60-aa DNA binding homeodomain. To isolate homeobox genes involved in mouse embryo development, total RNA obtained from three different stages of conceptus development, mouse 4.5-, 11.5-, and 18.5-day-old conceptuses, was used as a starting material. First-strand cDNAs were prepared under the same conditions as used in the cDNA synthesis of Example 3, wherein JYCS-Tu-ACP was used as the first-strand cDNA synthesis primer.<br>
The following ACPs comprise the degenerate sequences for homeobox sequence at their 3"-end portions and were used as degenerate homeobox-specific primers for the first-stage PCR amplification:<br>
JYC2-HD1 5"-GCrTGACTACGATACTGTGCGAimiGTNCRRGTGTGGTT-3" (SEQ ID <br><br>
JYC2-HD2   S"^GCTTGACTACGATACTGTGCGAIIinGTNCRRGTCTGGTr-a"   (SEQ  ID NO:84); and<br>
JYC2-HD3   5"■<k id no:85></k>
The PCR amplification was performed by hot start PCR method as described in Example 2 and conducted by one-stop or non-stop two-stage PCR amplifications. The following is an example of the process of one-stop two-stage PCR amplifications.<br>
1.	Combine the following reagents in a sterile 0.2 ml microcentrifuge tube: 49.5 ul of the total volume containing 1 ul of first-strand cDNA (50 ng/ul), 5 ul of 10 x PCR buffer (Roche), 5 ul of 2 mM dNTP, 1 ul of one of 10 uM JYC2-HD1, JYC2-HD2, or JYC2-HD3 (5" primer), I ul of 10 uM JYC5-T,5-ACP \? pn -; ) and 36.5 ul of sterile dH20.<br>
2.	Mix contents and spin the tube briefly in a microcentrifuge.<br>
3.	Place the tube in the preheated thermal cycler at 94°C.<br>
4.	Add 0.5 ul of Taq polymerase (Sunns/ul; Roche) into the reaction while holding the tube at the temperature 94°C.<br>
5.	Conduct PCR reaction under the following conditions: out cycle of 94°C for 1 min, 52°C for 3 min, and 72°C for 1 min; followed by 40 cycles of94°C for 40 sec, 65°C for 40 sec, and 72°C for 40 sec; and followed by a 5 min final extension at 72°C.<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected by staining with ethidium bromide. The resulting PCR products can be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al., 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrsen et al., 1997; Smith et al., 1997), and the use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al., 1996).<br>
Many differentially expressed bands in a specific stage were obtained, subcloned into the pGEM-T Easy vector (Promega), and sequenced. Sequence analysis reveals that some of the clones contain homeobox sequences. Northern blot or RT-PCR analysis shows that the clones are identical to the results of the expression patterns observed by the electrophoresis. These results indicate that the method using the ACP of the present invention for isolating conserved homology segments in multigene families produces only real PCR products. Freedom from false positives, which is one major bottleneck remaining in the previous PCR-based techniques<br>
for isolating conserved homology segments in multigene families, allows avoiding the subsequent labor-intensive work required for the verification of the amplified cDNA fragments.<br>
EXAMPLE 10: Single Nucleotide Polymorphism Genotyping Using ACP-based PCR<br>
To demonstrate the application of ACP in single nucleotide polymorphism genotyping, a portion containing a single nucleotide polymorphism (SNP) of human p53 (TP53) gene was amplified with either conventional primer or ACP. The process and results for the SNP genotyping using ACPs are described herein. DNA templates were obtained from human blood samples which have a SNP in exon 4 of the TP53 gene. This polymorphism is expressed as an Arg -&gt; Pro substitution at amino acid position 72 by replacing G with C. A 349 nt senuence between nucleotide 11991 and 12339 of the TP53 gene was amplified f.c .n c ■ type of template by a set of the following primers:<br>
P53N	5" <xjtctgachxktcttttcac-3 id no:86 and></xjtctgachxktcttttcac-3>
P53C-ACP    5" -TCACAGAAGTATGCCAAGCGAmHATTGAAGTCTCATGGAAGCC"3" (SEQBDNO:87).<br>
The amplified products containing the SNP between their ends were used as templates for detecting the SNP using allele-specific ACPs as follows:<br>
P53N1A-ACP     5"- GTCTACCAGGCATTCGCTTCATIIinCCCCGCGTGG-3"  (SEQ ID NO:88),<br>
P53N1B-ACP      5"-GTCTACCAGGCATTCGCTTCATnfflCCCCCCGTGG-3"  (SEQ ID NO:89),<br>
P53N2A-ACP      5"- GTCTACCAGGCATTCGCTTCATinnTCCCCGCGTG-3"  (SEQ ED NO:90),<br>
P53N2B-ACP      5"- GTCTACCAC
P53N3A-ACP      5"- GTCTACCAGGCATTCGCTTCATIiniCTCCCCGCGT-3"  (SEQ ID NO:92),<br>
P53N3B-ACP      5"- GTCTACCAG<attcgcttcatnnictccccccgt-3 id no:93></attcgcttcatnnictccccccgt-3>
P53N4A-ACP     5"- GTCTACCAGGCATTCGCTTCATIIinGCTCCCCGCG-3"  (SEQ ID NO:94),<br><br><br>
P53N4B-ACP      5"- GTCTACCAGGCATTCGCTTCA"niinGCTCCCCCCX3-3"  (SEQ ID<br>
NO:95),<br>
PS3N5A-ACP	5"- GTCTACCAGGCATTCGCITCATnniGCTCCCCG-3"   (SEQ ED<br>
NO-.96), and<br>
P53N5B-ACP       5" - GTCTACCAGGCATTCGCrTCATIIinQCTCCCCC-3" (SEQ ID NO:<br>
97).<br>
The polymorphic base is underlined at the 3" -end portion of each allele-specific ACP and the position of the polymorphic base is considered an interrogation position. The interrogation position is placed at several different positions from the 3" -end of allele-specific ACPs in order to letermuu. the most critical position in annealing specificity for detecting the SNP.<br>
1 rillele-specific ACPs were used as 5" primers. P53C-ACP and one of P53N1A-ACP,<br>
P53N2A-ACP, P53N3A-ACP, P53N4A-ACP and P53N5A-ACP were used for wild-type. A<br>
genotyping. P53C-ACP and one ;f P53NIB-ACP, P53N2B-ACP, P53N3B-ACP, P53N4B-<br>
ACP and P53N5B-ACP were used for variant-type B genotyping. The 5"-end portion<br>
sequences of the ACPs were served as pre-selected arbitrary primer sequences for the second-<br>
stage PCR amplifier tion:	<br>
JYC3 5" - TCACAGAAGTATGCCAAGCGA -3" (SEQ ID NO: 11) and<br>
JYC4            5" - GTCTACCAGGCATTCGCTTCAT -3" (SEQ ID NO: 12).<br>
(A) First-stage PCR Amplification<br>
The first-stage PCR amplification was conducted by one cycle of PCR consisting of annealing, extending and denaturing reaction; the reaction mixture in a final volume of 49.5 ul containing 1 uJ of the amplified target genomic segment containing the SNP in exon 4 of the TP53 gene, 5 ul of 10 x PCR reaction buffer (Promega), 5 ul of 25 mM MgCl2, 5 ul of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), and I ul of one of allele-specific ACPs (10 uM) is pre-heated at 94°C, while holding the tube containing the reaction mixture at the 94°C, 0.5 ul of Taq polymerase (5 units/ul; Promega) is added into the reaction mixture; for the allele-specific ACPs having 10 nucleotides at its 3" -end portion, the PCR reactions are as follows: one cycle of 94°C for 40 sec, 55°C for 40 sec, and 72°C for 40 sec; followed by denaturing the amplification product at 94°C; for the allele-specific ACPs having 8 nucleotides at its 3" -end portion, the PCR reactions are as follows: one cycle of 94°C for 40 sec, 50°C for 40 sec, and<br>
72°C for 40 sec; followed by denaturing the amplification product at 94°C. The resultant product is a first DNA strand complementary to the target genomic segment.<br>
(B) Second-stage PCR Amplification<br>
The resultant product generated by the first-stage PCR amplification was then amplified by the following second-stage PCR amplification at a higher annealing temperature than the first annealing temperature. After the completion of the first-stage PCR amplification, 1 jxl of 10 uM pre-selected arbitrary primer, JYC3, was added into the reaction mixture obtained from the first-stage PCR amplification, under denaturing temperature such as at 94°C. The second stage-PCR reaction was performed as follows: 30 cycles of °4°C for 40 sec, 68°C for 40 sec, and 72°C for 40 sec; followed by a 5 min final ex\c isio * ■: 72°C.<br>
The amplified products were analyzed by electrophoresis in a 2% agarose gel and detected by staining with ethidium bromide. The resulting PCR products can be also detroted on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods such as silver staining (Gottschlich et al., 1997; Kociok et al, 1998), the use of fluoresenscent-labelled oligonucleotides (Bauer et al. 1993; Ito et al. 1994; Luehrser. et al., 1997; Smith et al., 1997), and me use of biotinylated primers (Kom et al., 1992; Tagle et al., 1993; Rosok et al, 1996).<br>
FIG. 22 shows the results of allele-specific amplification using ACP. The pair of wild-type A-specific ACPs (P53N2A-ACP and P53C-ACP) generated a specific target product only from the samples having homozygous wild-type A (lane 1) or heterozygous genotyping (lane 3), but not from the samples having homozygous variant-type B genotyping (lane 5). The pair of variant-type B-specific ACPs (P53N2B-ACF and P53C-ACP) generated a specific target product only from the samples having homozygous variant-type B (lane 6) or heterozygous genotyping (lane 4), but not from the samples having homozygous wild-type A genotyping (lane 2). These results indicate that the ACP of the subject invention can be applied as an easy and economic method for detecting the genotype of SNPs since the use of fluorescent DNA probe nor post-PCR processing is not required in this approach. The allele-specific ACPs each having an interrogation position at its 3"-end portion showed improvement of annealing specificity. Moreover, when the allele-specific ACP has an interrogation position at the position 5 from the 3" -end (e.g., P53N2A-ACP and P53N2B-ACP), the annealing specificity is most critically accomplished.<br><br><br>
In order to verify if the position 5 from the actual 3" -end of the allele-specific ACP is the most appropriate for the interrogation position, additional six experiments were conducted using the same process as used in FIG. 22. DNA templates were obtained from human blood samples which have a SNP. Six short genomic fragments containing SNPs were amplified using each different primer set as follows:<br>
703N        5*- ATTCTGATGGTGTGGATTGTG -3" (SEQ ID NO:98) and SM703C   5"- TCACAGAAGTATGCCAAGCGAimiACCCTGGAGTAGACGAAGA -3" (SEQ ID NO:99) for Beta-2 adrenergic receptor (ADRB2), 028N        5"- CCrrCTGTGCTTGATGCrTrT -3" (SEQ ID NO: 102) and SM028C     5"- TCACAGAAGTATGCCAAGCGAmnCAGGAAGGATGAGCATTTAG  -3 "(SEQ ID NO: 103) for Chemokine (c-c motif) receptor 5 (CCR5), 695N:       5"- AGAAAAACCAGAGGCAGCTT-3" (SEQ ID NO:106) and SM695C   5"- TCACAGAAGTATGCCAAGCGAinn AGCACAAACCAAAGACACAGT -3" (SEQIDNO:107) for Merleukin 13 receptor,<br>
679N        5"- CTAGCTGCAAGTGACATCTCT -3" (SEQ ID NO: 110) and SM679C   5"- TCACAGAGTATCCAAGCGfflirrCAGTAAGAAGCCAGGAGAG -3"(SEQ ID NO: 111) for Leukocyte adhesion molecule-1 (LAM-1), 832N       5"- TTTTGGGTGGAGGCTAACAT -3 * (SEQ ID NO: 114) and SM832C: 5"- TCACAGAAGTATGCCAGCGAIimAACGATGCAGACACCACCA-S"CSEQ ID NO: 115) for Tachykinin receptor 3 (TACR3), and 880N      S"-CTTCCACCAATACTCTTTTCC-S" (SEQEDNO:118) and SM880C : 5"- TCACAGAAGTATGCCAGOGAmn GCATACACACAAGAGGCAGA -3"(SEQ ID NO: 119) for Interleukin 1, beta (ILIB).<br>
The amplified products containing the SNP between their ends were used as templates for detecting the SNPs wherein allele-specific ACPs were applied as follows: SM703-A 5"- GTCTACCAGGCATTCGCTTCATIffllGGTACAGGGC -3" (SEQ ID NO: 100) and SM703-B S"-GTCTACCAGGCATTCGCTTCATinnGGTACCGGGC -V (SEQ ID NO:101) for Beta-2 adrenergic receptor (ADRB2),<br>
SM028-A 5"- GTCTACCAGGCATTCGCTTCATimrrCCAAACCAA -3" (SEQ 3D NO: 104) and SM028-B 5"-GTCTACCAGGCATTCGCTTCATIimTCCAACCCAA -3" (SEQ ID NO: 105) for Chemokine (c-c motif) receptor 5 (CCR5),<br><br><br>
WO 03/050305<br><br>
PCT/KR02/01781<br><br>
SM695-A 5"- GTCTACCAGGCATTCGCTTCATmn CCATTTTAGG -3" (SEQ ID NO:108)<br>
and SM695-B 5"- GTCTACCAGGCATTCGCTTCATmn CCATTGTAGG -3" (SEQ ID<br>
NO: 109) for fciterleukin 13 receptor,<br>
SM679-A 5"- GTCTACCAGGCATTCXKnTCATnfflCCAGAACTTT -3" (SEQ ID NO:112)<br>
and SM679-B 5"- GTCTACCAGGCATTCGCTTCATIimCCAGACCTTT -3" (SEQ ID<br>
NO:l 13) for Leukocyte adhesion molecule-1 (LAM-1),<br>
SM832-A 5"- GTCrACCAGGCATTCGCTTCATnmGACTGGTAAA -3" (SEQ ID NO: 116)<br>
and SM832-B 5"- GTCTACCAGGCATTCGCTTCATIiniGACTGATAAA -3* (SEQ ID<br>
NO: 117) for Tachykinin receptor 3 (TACR3), and<br>
SM880-A5"- GTCTACC^GGCATTCGCTTCATimiAAAGCCATAA -3" (SEQ ID NO:120)<br>
and SM880-B ",   GT ^. iCCAGGCATTO}CriTCATnniAAAGCTATAA -3" (SEQ ID<br>
No-.121) for fcvterkukin 1, beta (IL1B).<br>
FIG. 23 shows the resuiis of allele-specific amplifications for six additional SNPs each present in different gene such as Beta-2 adrenergic receptor (ADRB2) (A), Chemokine (c-c motif) receptor 5 (CCR5) (B), In^rleukin 13 receptor (C), Leukocyte adhesion molecule-1 (LAM-1) (D), Tachykinin receptor 3 (TACR3) (E), and Interleukin 1, beta (IL1B) (J")-Consistant with the results of FIG. 22, me annealing specificity is critically accomplished when the allele-specific ACP has an interrogation position at the position 5 from the 3" -end. The pair of wild-type A-specific ACPs generated a specific target product only from the samples having homozygous wild-type A (lane 1) or heterozygous genotyping (lane 3), but not from the samples having homozygous variant-type B genotyping Gane 5). The pair of variant-type B-specific ACPs generated a specific target product only from the samples having homozygous variant-type B (lane 6) or heterozygous genotyping (lane 4), but not from the samples having homozygous wild-type A genotyping (lane 2),<br>
Having described a preferred embodiment of the present invention, it is to be understood that variants and modifications thereof falling within the spirit of the invention may become apparent to those skilled in this art, and the scope of this invention is to be determined by appended claims and their equivalents.<br><br><br>
TABLE 1<br><br><br><br><br><br><br><br><br>
SEQ	<br>
ID NO Designation<br>
1	ACPI<br>
2	ACP2<br>
3	ACP3<br>
4	ACP4<br>
5	ACP5<br>
6	ACP6<br>
7	ACP7<br>
8	ACP8<br>
9	ACP9<br>
10	JYC2<br>
11	JYC3<br>
12	JYC4<br>
13	ACP10<br>
14	ACP11<br>
15	ACP12<br>
16	ACP13<br>
17	ACP14<br>
18	ACP15<br>
19	CRP2I0<br>
20	ACP16<br>
21	ACP17<br>
22	ACPI 8<br>
23	ACP19<br>
24	dT-JYC3<br>
25	dT-JYC2<br>
26	JYC2-T13C<br>
27	JYC2-T13G<br><br>
Sequence Information<br>
5"-GTCTACCAGGCATTCGCTTCATimiCAGGAGTGG-3"<br>
5"-GTCTACCAGGCATTCGCTTCATmnGGCGACGATS-3"<br>
5 "-GTCTACCAGGCATTCGCTTCATinnGCCATCGACS-3"<br>
S"-GTCTACCAGGCATTCGCTTCATimiAGATGCCCGW-S"<br>
5"43TCTACCAGG
5"<jtctaccaggcarrcgcitcatinirrctcccggts-3></jtctaccaggcarrcgcitcatinirrctcccggts-3>
5"^TCTACCAGGCATTCGCTTC/   III fr"GTGGCGGS-3"<br>
S"-GTCTACCAGGCATTCGCTTCATinncrCCGATGCS-S"<br>
S"^TCTACCAGGCATTCGCTTCATinnCCTGCGGGTW^"<br>
5"- GCTrGACTACGATACTGTGCGA -3"<br>
5"- TCACAGAAGTATGCCAAGCGA-3"<br>
5"- GTCTACCAGGCATTCGCTTCAT -3"<br>
5"^TCTACCAGGCA^^aJCTTCAT^^IGCCATCGAOC-3,<br>
S"^TCrACCAGGCArrCGCTTCATimiGCCATCGACG-S"<br>
5"^TCTACCAGCK;AlTCGCTTCATinilAGGCGATGCC-3"<br>
5"^TCTACCAGGCATTa3C^TCATIIIIIAGGCGATGCG-3,<br>
5"^TCTACCAGGCATTCGCTTCATninCTCCGATGCC-3"<br>
5"^TCTACX:AGGCATTCGCTTCATIIinCTCCGATGCG-3"<br>
5"-GTCTACCAGGCATTCGCTTCATGCCATCGACC-3"<br>
5"-GTCTACCAGGCATTCGCTTCATnGCCATCGACC-3"<br>
5,-GTCTACCAGGCATTCGCTTCATmIGCCATCGACC-3•<br>
S"-GTCTACCAGGCATTCGCTTCATDiniGCCATCGACC-S"<br>
5"^TCTAO:AGGCATraKnTCATIIIIiniGCCATCGACC-3"<br>
5"^ACAGAAGTATGCCAAGCGACTCGAGTTTTTnTTT<br>
TTTTT-3"<br>
5 "•<xtitgacracgatactgtgcga cttgactacgatactgtgcgattttttttitrttc-3></xtitgacracgatactgtgcga><br>
TABLE 1-continued<br>
28	JYC2-T13A       5"-GCITGACTACGATACTGTGCGATTTTTTTTTTTTTA-3"<br>
29	dT,0-JYC2	S"-GCTTGACTACGATACTGTGCGATTTTTTTTTT-S"<br>
30	dT,0-ACPl	5"^CrTGACTACGATACTGTGCGAmnTTTTTTTTTT-^<br>
31	DEG2	GCCATCGACCCGTTTCTCTAGCCCCATCTTCATGTGT<br>
TTTAATGAGATGATATTAATTCATTACATTCATGGAT AATATGTCCCTGAGTACATTCTAATCTAGATTTAACT TCAAAAAAAAAAAAAAAAA<br>
32	DEG 5	AGGCGATGCGGGCTGTACTCTGGGTGGCTGCCACAGT<br>
CTCATGAGAAACCAAGGGCAAAGGACCAAGGAAAAG<br>
GGTCTCAGGCCCCTAAAGCAGTGGCTTTCAACCATCCTAA<br>
TGTTGTGACCTTTTAATACAGTTCCTCATGTTGTG<br>
TGACCCCCCAACCATAAAATGATTTTTGTTTCTACTTC<br>
AAAAAAAAAAAAAAAAAAAAAA<br>
33	SMART IV       S"-AAGCAGTGGTATCAACGCAGAGTGGCCATTACG<br>
GCCrCGGG)^" 3s       5"PCR Primer   S"-AAGCAGTGGTATCAACGCAGAGT-S"<br>
35	CDS ffl/3"	5"-ATTCTAGAGGCCGAGGCGGCCGACATG-(dT)3o<br>
VN-3"<br>
36	rG3-ACP	5,^TCTACCAGGCATTCGCTTCATIII^GGr(GGG)-3,<br>
37	rG2-ACP	5*^TCTACCAGGCATTCGCTTCATIfflIGGr(GG)d(G)-3"<br>
38	dG3-ACP	5,^TCTACCAGGCAT^CGCTTCAT^^IGGd(GGG)-3,<br>
39	Oligo dT„-ACp 5 "-GCTTGACTACGATACTGTGCGAminTTTTTTTT<br>
lTrnTOT-3"<br>
40	PLP-CQ<br>
41	JunB3<br>
42	P -actin 1<br>
43	P -actin 2<br>
44	EsxN7<br>
45	EsxC6<br>
46	EsxN7-ACP<br>
5 "-GAGAGGATAGTTTCAGGGAC-3" S"-CTCCGTGGTACGCCTGCnTTCTC-S" 5"-TCGTCACCCACATAGGAGTC-3" 5 "-CTAAGAGGAGGATGGTCGC-3" 5"-GCCGGTTGCAGAGCACC-3" S"-GAACCATGnTCTGAATGCC-S" S"-GTCTACCAGGCATTCGCTTCATinnGCCGGT TGCAGAGCACC-3"<br><br>
BLE 1-continued<br>
47	EsxC6-ACP      5"-GCTTGACTACGATACTGTGCGAHniGAACCAT<br>
GTTTCTGAATGCC-3"<br>
48	EsxNl	5"^AATCTGAAACAACITTCTA-3"<br>
49	EsxC2	S"-GATGCATGGGACGAGGCACCO"<br>
50	EsxNl-ACP      5"-GTCTACCAGGCATTCGCTTCATIIinGAATCT<br>
GAAACAACTTTCTA-3"<br>
51	EsxN3	5"-CGCCGCACCCCTGCCCGCA-3"<br>
52	EsxC5	5"-GATGCATGGGACGAGGCA-3"<br>
53	EsxN3-ACP      5"-GTCTACCAGGCATTCGCTTCATIimCGCCGC<br>
ACCCCTGCCCGCA-3,<br>
54	Oligo-dTis	5^riTTTn lTTTTTTT-3"<br>
55	EsxC2-ACP       S"-GCTTGACTACGATACTGTGCGAIIfflGATGCA<br>
TGGGACGAGGC ACC-3"<br>
56	EsxC5-ACP      5*.aCTTGACTACGATACTGTGCGAHniGATGCA<br>
TGGGACGAGGCA-3"<br>
57	OligoVdTis-ACP   5"-GCITGACTACG-VTACTGTGCGAII]IITTTTTT<br>
rrnrmrv-v<br>
58	dNs-ACP	5,•GCTTGA
59	iGl-ACP	5"-GTCTACCAGGCATTCGCTTCATimiGGr(G)d(GG)-3"<br>
60	JYC5	5"-CTGTGAATGCTGCGACTACGAT-3"<br>
61	JYC5-T,j-ACP        5"<rrgtgaatgctgcgacracgatmintitlt></rrgtgaatgctgcgacracgatmintitlt>
TTTTnTTT-3"<br>
62	JYC5-T15V-ACP       S"-CTGTGAATGCTGCGACTACGATIIIinTnTJT<br>
TTTTTTTTV-3"<br>
63	JYC5-T,sVN-ACP   S"-CTGTGAATGCTGCGACTACGATIIinTTmT<br><br>
64	ACP101	S"-GTCTACCAGGCATTCGCTTCATIimCCGCT AGC 3 ATC -3"<br>
65	ACP109	5 "-GTCTACCAGGCATTCGCTTCATinnCTnC^AGG A CG-3"<br>
66	ACPI 16	5 "-GTCTACCAGGCATTCGCTTCATnfflCGGAGCATC(&gt;3"<br>
67	ELAM1N1	5 "-TTGCACACTGTTGATTCTAA-S"<br>
68	ELAM1CI	5"-TTATTGATGGTCTCTACACA-3"<br><br><br>
TABLE l-continued<br>
69 70 71 72 73 74 75<br>
76<br>
77<br>
      81<br><br><br>
ELAM1N2	5"-CCACTGAGTCCAACATTC-3"<br>
ELAM1C2	S"-CTGAAACACTTCCCACAC-S"<br>
P53NI	5"-CCTCTGACTGCTCTnTCAC-3"<br>
P53C1	S"-ATTGAAGTCTCATGGAAGCC-S"<br>
P53N2	5"-TGCTTGCCACAGGTCTC-3*<br>
P53C2	5"-GCAGTGCTAGGAAAGAGG-3*<br>
ELAM1N1-ACP   S"-GTCTACCAGGCATTCGCTTCATimnTGCACACTGT<br>
TGATTCTAA-3* ELAM1C1-ACP   S"-TCACAGAAGTATGCCAAGCGAnnnTATTGA^GGT<br>
CTCTACACA-3* ELAM1N2-ACP   S"^rTCTACrAGGCATTCGCTTCATninCCACTGAGTCC<br>
76<br>
AACATTC-3" ELAM1C2-ACP   5"TCACAGAAGTATGCCAAGCGAIMCTGAAACACfT<br>
79<br>
CCCACAC-3"<br>
P53N1-ACP	5"^TCTACCAGGCATr
80<br>
•     TCTTTTCAC-3" P53C1-ACP        5 ^TCACAGAAGTATGCCAAGCGAIIIIIATTGAAGTCTC<br>
ATGGAAGCC-3" P53N2-ACP        5"^TCTACCAGGCATTCGCTrCATmirrGCrTGCCACA<br>
GGTCTC-3" P53C2-ACP        5 "-TCACAGAAGTATGCCAAGCGAmnGCAGTGCTAGG<br>
AAAGAGG-3"<br>
83	JYC2-HD1	S"-GCTTGACTACGATACTGTGCGAimrGTNCRRGTGTG<br>
GTT-3"<br>
84	JYC2-HD2	S"-GCTTGACTACGATACTGTGCGAmnGTNCRRGTCTG<br>
GTT-3"<br>
85	JYC2-HD3	5"-GCTTGACTACGATACTGTGCGAIimGTNCRRGTTTG<br>
GTT-3"<br>
86	P53N	5,-CCTCTGACTGCTCTTTTCAC-3,<br>
87	P53C-ACP	S"-TCACAGAAGTATGCCAAGCGAimiATTGAAGTCTC<br>
AGAAGCC-3"<br><br>
TABLE 1-continued<br>
88	P53N1A-ACP     5"-GTCTACCAGGCATTCGCTTCATimiCCCCGCGTGG-3"<br>
89	P53N1B-ACP     5"-GTCTACCAGGCATTCGCTTCATmnCCCCCCGTGG-3"<br>
90	P53N2A-ACP     S"-GTCTACCAGGCATTCGCTTCATimrrCCCCGCGTG-S"<br>
91	P53N2B-ACP     5"-GTCTACCAGGCATTCGCTTCATIIII^^CCCCCCGTG-3,<br>
92	P53N3A-ACP     5^GTCTACCAGGCATTCGC^^CAT^^ICTCCCCGCGT-3,<br>
93	P53N3B-ACP     5^GTCTACCAGGCArrCGCTTCATmiICTCCCCCCGT-3"<br>
94	P53N4A-ACP     5"-GTCTACCAGGCATTCGCTTCATII^IGCTCCCCGCG-3,<br>
95	P53N4B-ACP     S"-GTCTACCAGGCATTCGCTTCATimiGCTCCCCgCG-S"<br>
96	P53NSA-ACP       5^GTCTACCAGGCATTCGCTTCATII^IGCTCCCCG-3, 9"	P53N5B-ACP       5^GTCTACCAGGCATrCGCTTCATIIinGCTCCCCC-3"<br><br>
98	703N	5" - ATTCTGATGGTGTGGATTGTG -3"<br>
99	SM703C	5"-TCACAGAAGTATGCCAAGCGAIIinACCCTGGAGTAG<br>
ACGAAGA-3"<br>
100	SM703-A	5"- GTCTACCAGGCATTCGCTTCATIiniGGTACAGGGC-3"<br>
101	SM703-B	5" - GTCTACCAGGCATTCGCTTCATninGGTACCGGGC -3"<br>
102	028N	5" - CXrTTCTGTGCTTGATGCTTTT -3"<br>
103	SM028C	5"-TCACAGAAGTATGCCAAGCGADniCAGGAAGGATGA<br>
GCATTTAG-3"<br>
104	SM028-A	5"-GTCTACCAGGCATTCGCTTCATimrrCCAAACCAA-3"<br>
105	SM028-B	5" - GTCTACX:AGGCATTCGCTTCATIiniTCCAACCCAA -3"<br>
106	695N	S"-AGAAAAACCAGAGGCAGCTT-S"<br>
107	SM695C	5"-TCACAGAAGTATGCCAAGCGAranAGCACAAACCAA<br>
AGACACAGT-3"<br>
108	SM695-A        5" - GTCTACCAGGCATTCGCTTCATIffll CCATTTTAGG -3"<br>
109	SM695-B	5"-GTCTACCAGGCATTCGCTTCATIiniCCATTGTAGG-3"<br>
110	679N	5" - CTAGCTGCAAGTGACATCTCT -3"<br>
111	SM679C        5"-TCACAGAGTATCCAAGCGnnrrCAGTAAGAAG<br>
CCAGGAGAG-3"<br>
112	SM679-A       5"-GTCTACCAGGCATTCGCTTCATinnCCAGAACTTT-3"<br>
113	SM679-B       5" - GTCTACCAGGCATTCGCTTCATIiniCCAGACCTTT -3"<br>
114	832N	5" - TTTTGGGTGGAGGCTAACAT -3"<br><br>
TABLE 1-continued<br>
115	SM832C       5"-TCACAGAAGTATGCCAGCGAmnAACGATGCAG<br>
ACACCACCA-3"<br>
116	SM832-A     5"-GTCTACCAGGCATTCGCTTCATmnGACTGGTAAA-3"<br>
117	SM832-B     5"-GTCrACCAGGCATTCGCrTCATfflnGACTGATAAA-3"<br>
118	880N	5" - CTTCCACCAATACTCTTTTCC -3"<br>
119	SM880C     S"-TCACAGAAGTATGCCAGCGAUfflGCATACACACA<br>
AGAGGCAGA-3"<br>
120	SM880-A     5"-GTCTACCAGGCATTCGCTTCATnfflAAAGCCATAA.3"<br>
121	SM880-B     5"-GTCTACCAGGCATTCGCTTCATmiIAAAGCTATAA-3"<br>
S=GorC W = AorT V = A,G,orC N = A, G, C, orT I is deoxyinosine r is ribose d is deoxyribose<br><br>
20<br><br><br><br>
TABLE 2<br>
Differentially Expressed cDNA Fragments Cloned by the ACP of the Present Invention<br>
Nomenclature	Identity	Homology<br>
DEG1	Tropomyosin 2 (beta)	Mouse 92%<br>
DEG2	Novel	Novel<br>
DEG3	Hypothetical protein (Tes gene)	Mouse 99%<br>
DEG4	Protease-6	Mouse 92%<br>
DEG5	Novel	Novel<br>
DEG6	Cytochrome c oxidase, subunit Vb	Mouse 99%<br>
DEG7	Hydroxylacyl-Coenzyme A dehydrogense (Hadh)	Mouse 98%<br>
DEG8	Troponin T2, cardiac (Tnnt2)	Mouse 94%<br>
DEG9	RNA binding motif protein, X chromosome	Mouse 96%<br>
DEG10	Peroxiredoxin 6 (Prdx6)	Mouse 89%<br>
DEG11	11 days or 13 days embryo cDNA	Mouse 98%<br><br><br><br>
References<br>
Anonymous (1992) Diagnosis of Duchenne and Becker muscular dystrophies by polymerase chain reaction. A multicenter study. JAMA 267,2609-2615.<br>
Bauer, D., Muller, H., Reich, J., Ahrenkiel, V., Warthoe, P., Strauss, M. (1993) Identification of differentially expressed mRNA species by an improved display technique (DDRT-PCR). Nucleic Acids Res. 21,4272-4280.<br>
Bauer, D„ Warthoe, P., Rohde, M., Struss, M. (1994) PCR Methods &amp; App: Manual Supplement, pp. S97-S108. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY<br>
Carninci, P., Westover, A., Nishiyama, Y., Ohsumi, T., Itoh, M., Nagaoka, S., Sasaki, N., Okazaki, Y., Muramatsu, M., Hayashizaki, Y. (1997) High efficiency selection of full-length cDNA by improved biotinylated cap trapper. DNA Res. 4,61-66.<br>
Chamberlain, J.S., Gibbs, R.A., Ranier, J.E., Nguyen, P.N., Caskey, C.T. (1988) Deletion screening of the Duchenne muscular dystrophy locus via multiplex DNA amplification. Nucleic Acids Res. 16,11141-11156.<br>
Chenchik, A., Zhu, Y., Diatchenko, L., Li, R., Hill, J., Siebert, P. (1998) Generation and use of high-quality cDNA from small amounts of total RNA by Smart™ PCR. In Siebert, P. and Larrick, J. (eds), Gene Cloning and analysis by RT-PCR. Biotechniques Books, Natick, MA, pp. 305-319.<br>
Chenchik, A., Zhu, Y., Diatchenko, L., Siebert, P. Methods and compositions for generating full-length cDNA having arbitrary nucleotide sequence at the 3"-end. U.S. Pat. No. 5,962,271. Date of Patent: Oct. 5,1999.<br>
Chenchik, A., Zhu, Y., Diatchenko, L., Siebert, P. Methods and compositions for full-length cDNA cloning using a template-switching oligonucleotide. U.S. Pat. No. 5,962,272. Date of Patent: Oct 5,1999.<br>
Chun, J.Y., Han, Y.J., Ahn, K.Y. (1999) Psx homeobox gene is X-linked and specifically expressed in trophoblast cells o mouse placenta, Dev. Dyn, 216, 257-266<br>
Clark, J.M. (1988) Novel non-templated nucleotide addition reactions catalyzed by prokaryotic and eucaryotic DNA polymerases. Nucleic Acids Res. 16,9677-9686.<br>
Combates, N, Pardinas, J.R., Parimoo, S., Proucy, S.M., Stenn, K.S. Technique for differential display. U.S. Pat. No. 6,045,998. Date of Patent: Apr. 4,2000.<br>
D"Aquila, R.T., Bechtel, L.J., Videler, J.A., Eron, J.J., Gorczyca, P., Kaplan, J.C. (1991) Maximizing sensitivity and specificity of PCR by pre-amplification healing. Nucleic Acids<br><br>
Res., 19,3749.<br>
Diachenko, L.B., Ledesma, J., Chenchik, A.A., Siebert, P.D. (1996) Combining the technique of RNA fingerprinting and differential display to obtain differentially expressed mRNA. Biochem. Biophys. Res. Commun. 219, 824-828.<br>
Dieffenbach, C.W., Lowe, TJM.J, Dveksler, G.S. (1995) General concepts for PCR primer design. PCR primen a Laboratory Manual., pp. 133-142, Cold Spring Harbor Laboratory Press, Cold Spring Harbor. NY.<br>
Don, R.H., Cox, P.T., Wainwright, B.J., Baker, K., Mattick, J.S. (1991) "Touchdown" PCR to circumvent spurious priming during gene amplification. Nucleic Acids Res., 19,4008.<br>
Franr, O., Bruchhaus, I., Roeder, T. (1999) Verification of differential gene transcription using vLr .al i: "ir &gt; em blotting. Nucleic Acids Res., 27,1 -3.<br>
Frohman, M.A., Dush, MJC., Martin, G.R. (1988) Rapid production of full-length cDNAs from rare transcripts: amplification using ? single gene-specific oligonucleotide primer. Proc. Natl Acad. Sci. USA bJ, 8998-9002.<br>
Fromont-Racine, M., Bertrand, E., Pictet, R., Grange, T. (1993) A highly sensitive method for mapping the 5" termini f. f mRNAs. Nucleic Acids Res., 2I,1683-1684.<br>
Gottschlich, S., Goeoegli, t, Folz, B.J., Lippert, B.M., Werner, J.A. (1997) Optimized differential display and reamplification parameters for silver staining. Res. Commun, Mol. Path. Pharm. 97, 237-240.<br>
Gromova, I., Gromov, P., Celis, J.E. (1999) Identification of true differentially expressed mRNAs in a pair of human bladder transitional cell carcinomas sing an improved differential display procedure. Electrophoresis 20,241-248.<br>
Guegler, K., Tan, R., Rose, M.J. Methods and compositions for producing 5" enriched cDNA libraries. U.S. Pat. No. 6,083,727. Date of Patent: Jul. 4,2000.<br>
Guegler, K., Tan, R., Rose, M.J. Methods and compositions for producing full length cDNA libraries. U.S. Pat. No. 6,326,175. Date of Patent: Dec. 4,2001.<br>
Hogan, B., Bedding, R., Costantini, F., Lacy, E. (1994) Manipulating the moue embryo: a laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.<br>
Hwang, I.T., Lee, Y.H., Moon, B.C., Ahn, K.Y., Lee, S.W., Chun, J.Y. (2000) Identification and characterization of a new member of the placental prolactin-like protein-C (PLP-C) subfamily, PLP-Cp. Endocrinology 141, 3343-3352.<br>
Hayashizaki, Y. Method for forming full length cDNA libraries. U.S. Pat. No. 6,143,528.<br><br>
Date of Patent: Nov. 7,2000.<br>
Henegariu, O, Hirschmann, P., Killian, K., Kirch, S., Lengauer, C, Maiwald, R., Mielke, K., Vogt, P. (1994) Rapid screening of the Y chromosome in idiopathic sterile men, diagnostic for deletions in AZF, a genetic Y factor expressed during spermatogenesis. Andrologia, 26, 97-106.<br>
Ito, T., Kito, K., Adati, N., Mitsui, Y., Hagiwara, H., Sakaki, Y. (1994) Fluorescent differential display: arbitrarily primed RT-PCR fingerprinting on an automated DNA sequencer. FEBS Lett. 351,231-236.<br>
Jefferies, D., Botman, M. F., Lester, D, Whitehead, C. C, Thorp, B. H. (1998) Cloning differentially regulate genes from chondrocytes vising agarce gel differential display. Biochim. Biophys. Acta 1396,237-241.<br>
Kociok, N., Unfiled, K., Eser, P., Krott, R., Schraermeyer, U., Heimann, K. (1998) The nonradioisotopic representation of differentially expressed mRNA by a combination of RFA fingerprinting and differential display. Mol. Biotechnol. 9,25-33.<br>
Kern, B., Sedlacek, Z., Manca, A., Kioschis, P., Konecki, D., Lehrach, H., Poutska, A. (1992) A strategy for the selection of transcribed sequences in the Xq28 r-"gion. Hum. Mol. Genet. 1,235-242.<br>
Kulpa, D., Topping, R, Telesnitskt, A. (1997) Determination of the site of first strand transfer during Moloney murine leukemia virus reverse transcription and identification of strand transfer-associated reverse transcriptase errors. EMBO J. 16, 856-865.<br>
Landegren, U., Nilsson, M., Kwok, P.-Y. (1998) reading bits of genetic information: methods for singlc-nucleotide polymorphism analysis. Genome Res. 8,769-776.<br>
Liang, P., Pardee, A.B, (1992) Differential display of eukaryotic messenger RNA by means of the polymerase chain reaction. Science 257,967-971.<br>
Ledbetter, S.A., Nelson, D.L., Warren, ST., Ledbetter, D.H. (1990) Rapid isolation of DNA probes within specific chromosome regions by interspersed repetitive sequence polymerase chain reaction. Genomics 6,475-481.<br>
Loakes, D., Brown, D.M. (1994) 5-Nitroindole as an universal base analog. Nucleic Acids Res. 22,4039-4043.<br>
Loakes, D. The applications of universal DNA base analogues. Nucleic Acids Res. 29, 2437-2447.<br>
Luehrsen, K.R., Marr, L.L., Van Der Knaap, E., Cumberledge, S. (1997) Analysis of<br><br>
differential display RT-PCR products using fluorescent primers and genescan software. BioTechniques 22,168-174.<br>
Mate, M.V., Lukyanov, S.A, (1998) Different strategies of differential display: areas of application. Nucleic Acids Res. 26, 5537-5543.<br>
Mate, M., Shagin, D., Bogdanova, E., Britanova, O., Lukyanov, S., Diatchenko, L., Chenchik., A. (1999) Amplification of cDNA ends based on templateswitching effect and step-out PCR. Nucleic Acids Res. 27,1558-1560.<br>
McClelland, M., Chada, K., Welsh, J., Ralph, D. (1993). Arbitrary primed PCR fingerprinting of RNA applied to mapping differentially expressed genes. In Symposium on ~ DNA fingerprinting: State of the science, November, 1992 (ed. S.D. Pena et al.). Birkhauser Verlag, Basel, Switzerland.<br>
McPherson, MJ., Moller, S.G. (2000) PCR. BIOS Scientific Publishers, Springer-Verlag New York Eerlin Heidelberg, NY.<br>
Meunier, J.R., Grimont, PA.D. (1993) Factors affecting reproducibility of amplified polymorphic DNA fingerprints. Res. Microbiol. 144,373-379.<br>
Mullis, K.B., Faloona, F.A. (1987) Specific synthesis of DNA in vitro via a polymerase-catalyzed chain reaction. Methods Enzymol. 155,335-350.<br>
Mullis, K.B., Erlich, H.A, Amheim, N., Horn, G.T., Saiki, R.K., Scharf, S.J. Process for amplifying, detecting, and/or cloning nucleic acid sequences. U.S. Pat. No. 4,683s195. Date of Patent: Jul. 28,1987.<br>
Mullis, K.B. Process for amplifying nucleic acid sequences. U.S. Pat No. 4,683,202. Date of Patent: Jul. 28,1987.<br>
Mullis, K.B., Erlich, H.A, Amheim, N., Horn, G.T., Saiki, R.K., Scharf, SI. Process for amplifying, detecting, and/or cloning nucleic acid sequences. U.S. Pat. No. 4,800,159. Date of Patent: Jan. 24,1989.<br>
Mutirangura, A., Greenberg, F,, Butler, M.G., Malcolm, S., Nicholls, R.D., Chakravarti, A., Ledbetter, D.H. (1993) Multiplex PCR of three dinucleotide repeats in the Prader-Willi/Angelman critical region (15qll-ql3): molecular diagnosis and mechanism of uniparental disomy. Hum. Mol. Genet., 2, 143-151.<br>
Nichols, R., Andrews, P.C., Ahang, P., Bergstrom, D.E. (1994) A universal nucleoside for use at ambiguous sites in DNA primers. Nature 369,492-493.<br>
Ohtsuka, E., Matsuka, S., Ikehara M., Takahashi, Y., Matsubara K. (1985) An alternative<br><br>
approach to deoxyoligonucleotides as hybridization probes by insertion of deoxyinosine at ambiguous codon positions. J. Biol. Chem. 260,2605-2608.<br>
Ralph, D., Welsh, J., McClelland, M. (1993) RNA fingerprinting using arbitrary primed PCR identifies differentially regulated RNAs in Mink lung (MvlLu) cells growth arrested by TGF-p. Proc. Natl. Acad. Sci. 90,10710-10714.<br>
Rompf, R., Kahl, G. (1997) mRNA differential display in agarose gels. BioTechniques 23, 28-32.<br>
Roses, A.D. (2000) Pharmacogenetics and the practice of medicine. Nature, 405, 857-865.<br>
Rosok, O., Odeberg, J., Rode, M., Stofcke, T„ Funderud, S., Smeland, E. (1996) solid-phase method for differentially display of genes expressed in hematopoietic stem cells. BioTechniques 21,11+ 21.<br>
Ruano, G., Fenton,W., Kidd, KJC. (1989) Biphasic amplification of very dilute DNA samples via "booster PCR". Nucleic Acids Res. 17, 5407.<br>
Sailri, R.K., Scharf, S., Faloonn, F., Mullis, K.B., Horn, G.T., Erlich, H.A., Arnheim, N. (1985) Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia. Scienc ? 230,1350-1354.<br>
Sakanari, J.A., Staunton, C.E., Eakin, A.E., Craik, C.S. (1989) Serine proteases from nematode and protozoan parasites: Isolation of sequence homologs using generic molecular probes. Proc. Natl. Acad. Sci. 86,4863-4867.<br>
Schaefer, B.C. (1995) Revolutions in rapid amplification of cDNA ends: New strategies for polymerase chain reaction cloning of full-length cDNA ends. Anal. Biochem. 227, 255-273.<br>
Schmidt W.M., Mueller, M.W. (1999) CapSelect: A highly sensitive method for 5" CAP-dependent rerichment of full-length cDNA in PCR-mediated analysis of mRNAs. Nucleic Acids Res. 27,e31.<br>
Schramm, G., Bruchhaus, I., Roeder, T. (2000) A simple and reliable 5"RACE approach. Nucleic Acids Res. 28, e96.<br>
Shuber, A.P., Skoletsky, J., Stern, R., Handelin, B.L. (1993) Efficient 12 mutation testing in the CFTR gene: a general model for complex mutation analysis. Hum. Mol Gemt., 2, 153-158.<br>
Smith, N.R., Aldersley, M., Li, A., High, A.S., Moynihan, T.P., Markham, A.F., Robinson. P.A. (1997) Automated differential display using a fiuorescently labeled universal primer. BioTechniques 23,274-279.<br><br>
Sompayrac, L., Jane, S., Bum, T.C., Tene, D.Q., Danna, KJ. (1995) Overcoming limitations of the mRNA differential display technique. Nucleic Acids Res. 23,4738-4739.<br>
Stone, B., Wharton, W. (1994) Targeted RNA fingerprinting: the cloning of differentially-expressed cDNA fragments enriched for members of the zinc finger gene family. Nucleic Acids Res. 22,2612-2618.<br>
Suzuki, Y., Yoshitomo-Nakagawa, K., Maruyama, K., Suyama, A., Sugano, S. (1997) Construction and characterization of a full length-enriched and a S"-end-enriched cDNA library. Gene 200,149-156.<br>
Tagle, D.A., Swaroop, M., Lovett, M., Collins, F.S. (1993) Magnetic bead capture of expressed sequences encoded within large genomic segments. Nature i61. 751-753.<br>
Villeponteau, B„ Feng, J., Funk, W., Linskens, M.H.K. Metliod w.-1 kit for enhanced differential display. U.S. Pat. No. 5,580,726. Date of Patent: Dec. 3,1996.<br>
Welsh, J., McClelland, M. (1990) Fingerprinting genomes using PCR with arbitrary primers. Nucleic Acids Res, 18, 7213-7218.<br>
Welsh, J., McClelland, M. (1991) Genomic fingerprinting using arbitrarily primed PCR and a matrix of pairwise combinations of primers. Nucleic Acids Res. 19, 5275-5279.<br>
Williams, J.G.K., Kubelik, A.R., Livak, K.J., Rafalki, J.A., Tingey, S.V. (1990) DNA polymorphisms amplified by arbitrary primers are useful as genetic markers. Nucleic Acids Res. 18,6531-6535.<br>
Zimmermann, K., Schogl, D., Plaimauer, B., Mannhalter, J.W, (1996) Quantitative multiplex competitive PCR of HTV-1 DNA in a single reaction tube, BioTechniques 21, 480-484.<br>
Zou, S., Stanfield, C, Bridge, J. (1998) Identification of new influenza B virus variants by multiplex reverse transcription-PCR and the heteroduplex mobility assay. /. Clin. Microbiol.        36,1544-1548.<br><br><br>
We claim:<br>
1.	An annealing control primer for improving annealing specificity in nucleic acid<br>
amplification, which comprises: (a) a 3"-end portion having a hybridizing nucleotide<br>
sequence substantially complementary to a site on a template nucleic acid to hybridize<br>
therewith; (b) a 5"-end portion having a pre-selected arbitrary nucleotide sequence; and (c) a<br>
i regulator portion positioned between said 3"-end portion and said 5"-end portion comprising<br>
at least two universal bases or non-discriminatory base analogs, whereby sai"d regulator<br>
portion is capable of regulating an annealing portion of said primer in association with<br>
annealing temperature.<br>
2.	The annealing control primer according to claim 1, wherein said pre-selected arbitrary nucleotide sequence of said 5"-end portion is substantially not complementary to any site on said template nucleic acid.<br>
3.	The annealing control primer according to claim 1, wherein said template nucleic acid is gDNA, cDNA or mRNA<br>
4.	The annealing control primer of claim 3, wherein said gDNA or cDNA is single or double-stranded DNA.<br>
5 The annealing control primer according to claim 1, wherein said nucleic acid amplification is performed under a first and a second annealing temperatures.<br>
6. The annealing control pnmer according to claim 5, wherein said first annealing temperature in said nucleic acid amplification is identical to or lower than said second annealing temperature.<br>
7 The annealing control primer according to claim 5, wherein said 3"-end portion is involved in annealing at said first annealing temperature and said 5-end portion serves as a priming site at said second anneaiing temperature.<br><br>
8.	The annealing control primer according to claim 5, wherein said regulator portion is<br>
capable of restricting said annealing portion of said primer to said 3"-end portion at said first<br>
annealing temperature.<br>
9.	The annealing control primer according to claim 5, wherein said first annealing temperature is between about 30°C and 68°C.<br>
10.	The annealing control primer according to claim 5, wherein said second annealing temperature is between about 50°C and 72°C.<br>
11.	The annealing control primer according to claim 1, wherein said annealing control primer has a general formula of 5"-Xp-Yq-Zr-3", wherein Xp represents said 5"-end portion having said pre-selected arbitrary nucleotide sequence substantially not complementary to any site on the template nucleic acid; Yq represents said regulator portion comprising at least two universal bases or non-discriminatory base analogs; Zr represents said 3?-end portion having a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith; wherein p. q and r represent the number of nucleotides; and wherein X, Y and Z is deo.vy ribonucleotide or ribonucleotide.<br>
12.	The annealing control primer according to claim 11, wherein said universal base or non¬discriminatory base analog is capable of forming base-pairs with each of the natural DNA/RNA bases with little discrimination between said natural DNA/RNA bases.<br>
13.	The annealing control primer according to claim 11, wherein said universal base or non¬discriminatory base analog is selected from the group consisting of deoxyinosine. inosine. 7-deaza-2-deoxyinosine, 2-aza-2"-deoxyinosine. 2"-OMe inosine. 2"-F inosine. deoxy 3-nitropyrrole. 3-nitropyrrole. 2"-OMe 3-nitropyrrole. 2"-F 3-nitropyrrole. l-(2*-deoxy-beta-D-ribofuranosyl)-3-nitrop\Trole. deoxy 5-nitroindole. 5-nitroindole. 2-OMe 5-nitroindole. 2"-F 5-nitroindole, deoxy 4-nitroben/imidazole, 4-nitroben/imida/ole. deoxy 4-aminobenzimida/ole. 4-aminobenzimida/ole. deoxy nebularine. 2"-F nebularine. 2-F 4-<br><br><br>
nitrobenzimidazole, PNA-5-introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole, morpholino-5-nitroindole, moipholino-nebularine, morpholino-inosine, morpholino-4-nitrobenzimidazoIe, morpholino-3-nitropyrrole, phosphoramidate-5-nitroindole, phosphoramidate-nebularine, phosphoramidate-inosine, phosphoramidate-4- nitrobenzimidazole, phosphoramidate-3-nitropyrrole, 2"-0-methoxyethyl inosine, 2"0-methoxyethyl nebularine, 2"-0-methoxyethyl 5-nitroindole, 2"-0-methoxyethyl 4-nitro-benzimidazole, 2"-0-methoxyethyI 3-nitropyrrole, and combinations thereof.<br>
14.	The annealing control primer according to claim 11, wherein said universal base or non¬<br>
discriminatory base analog is deoxyinosine, l-(2"-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole<br>
or 5-nitroindole.<br>
15.	The annealing control primer according to claim 11. wherein said regulator portion<br>
comprises contiguous nucleotides having universal base or non-discriminatory base analog.<br>
16.	The annealing control primer according to claim 11, wherein said deoxyribonucleotide is naturally occurring dNMP, modified nucleotide and non-natural nucleotide.<br>
17.	The annealing control primer according to claim 11. wherein p represents an integer of 15 to 60.<br>
18.	The annealing control primer according to claim 11. \\ herein q is at least 3.<br>
19.	The annealing control primer according to claim 11. wherein q is at least 4.<br>
20.	The annealing control primer according to claim 11, wherein q represents an integer of 2 tol5.<br>
21.	The annealing control primer according to claim 11. wherein r represents an integer of 6 to 50.<br><br><br>
22.	The annealing control primer according to claim 11, wherein p is an integer of 15 to 60, q is an integer of 2 to 15 and r is an integer of 6 to 30.<br>
23.	The annealing control primer according to claim U, wherein Xp comprises a universal primer sequence.<br>
24.	The annealing control primer according to claim 11, wherein Xp includes a sequence or sequences recognized by a restriction endonuclease or restriction endonucleases.<br>
25.	The annealing control primer according to claim 11, wherein Xp comprises at least one nucleotide with a label for detection or isolation.<br>
26.	The annealing control primer according to claim 11, wherein Zr is a nucleotide sequence which hybridizes to the polyadenosine (polyA) tail of an mRNA.<br>
27.	The annealing control primer according to claim 26, wherein ZT comprises at least 10 contiguous deoxythymidine nucleotides.<br>
28.	The annealing control primer according to claim 26, wherein Zr comprises at least 10 contiguous deoxythymidine nucleotides having 3"-V at its 3"-end; in which V is one selected from the group consisting of deoxyadenosine, deoxycytidine and deoxyguanosine.<br>
29.	The annealing control primer according to claim 26, wherein Zr comprises at least 10 contiguous deoxythymidine nucleotides having 3"-NV at its 3-end: in which V is one selected from the group consisting of deoxyadenosine, deoxycytidine and deoxyguanosine. and N is one selected from the group consisting of deoxyadenosine, deoxythymidine. deoxycytidine and deoxyguanosine<br>
30.	The annealing control primer according to claim 11. wherein Zr is a nucleotide sequence substantially complementary to a target sequence in the template nucleic acid.<br><br><br>
31.	The annealing control primer according to claim 11, wherein Z, is a random nucleotide sequence.<br>
32.	The annealing control primer according to claim 11, wherein Zr is a nucleotide sequence substantially complementary to a consensus sequence found in a gene family.<br>
33.	The annealing control primer according to claim 11, wherein Zr is a degenerate<br>
nucleotide sequence selected from a plurality of combinations of nucleotides encoding a<br>
predetermined amino acid sequence.<br>
34.	The annealing control primer according to claim 11, wherein Zr comprises at least one ribonucleotide.<br>
35.	The annealing control primer according to claim 11, wherein Zr comprises at least one nucleotide complementary to allelic site.<br>
36.	The annealing control primer according to claim 11. wherein Zr comprises at least one mismatch nucleotide to a target nucleic acid for mutagenesis.<br>
37.	A method for amplifying a nucleic acid sequence from a DNA or a mixture of nucleic acids, wherein said method comprises performing an amplification reaction using primers, characterized in that at least one primer has the same structure as the primer of any one of claims 1-25.<br>
38.	The method according to claim 37. wherein said method is performed using two stage amplifications, which comprises:<br>
(a) performing a first-stage amplification of said nucleic acid sequence at a first annealing temperature comprising at least two cycles of primer annealing, pnmer extending and denaturing, using the primer pair of any one of claims 1 -25 each having at<br><br><br>
its 3-end portion a hybridizing sequence substantially complementary to a region of said nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to said region of said nucleic acid sequence, whereby the amplification product of said nucleic acid sequence is generated; and<br>
(b) performing a second-stage amplification of said amplification product generated from step (a) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primers as used in step (a) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5"-end portion of said primers used in step (a), under conditions in which each primer anneals to the 3"- and 5"-ends of said amplification product, respectively, whereby said amplification product is re-amplified.<br>
39.	A method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, wherein said method comprises performing an amplification reaction using primers, characterized in that at least one primer has the same structure as the primer of any one of claims 1 -25.<br>
40.	The method according to claim 39. wherein said method is performed using two stage amplifications, which comprises:<br><br>
(a)	performing a first-stage amplification of said target nucleic acid sequence at a first annealing temperature comprising at least two cycles of primer annealing, primer extending and denaturing, using the primer pair of any one of claims 1-25 each having at its 3 "end portion a hybridizing sequence substantially complementary to a region of said target nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to its target nucleotide sequence, whereby the amplification product of said target nucleotide sequence is generated; and<br>
(b)	performing a second-stage amplification of said amplification product generated from step (a) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing,<br><br><br>
using the same primers as used in step (a) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5"-end portion of said primers used in step (a), under conditions in which each primer anneals to the 3"- and 5"-ends of said amplification product, respectively, whereby said amplification product is re-amplified.<br>
41.	A method for selectively amplifying a target nucleic acid sequence from an mRNA,<br>
wherein said method comprises reverse transcribing said mRNA and performing an<br>
amplification reaction using primers, characterized in that at least one primer has the same<br>
structure as the primer of any one of claims 1-25.<br>
42.	The method according to claim 41, which comprises.<br>
(a)	contacting said mRNA with an oligonucleotide dT primer which is hybridized to polyA tail of said mRNA under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing said mRNA to which said oligonucloetide dT pirmer hybridizes to produce a first DNA strand that is complementary to said mRNA to which said oligonucloetide dT pirmer hybridizes;<br>
(c)	performing a first-stage amplification of said target nucleic acid sequence from said first DNA strand obtained from step (b) at a first annealing temperature comprising at least two cycles of primer annealing, primer extending and denaturing, using the primer pair of any one of claims 1-25 each having at its V end portion a hybridizing sequence substantially complementary to a region of said target nucleic acid sequence to hybridize therewith, under conditions in which each primer anneals to its target nucleotide sequence, whereby the amplification product of said target nucleotide sequence is generated; and<br>
(d)	performing a second-stage amplification of said amplification product generated from step (c) at a second annealing temperature, which is high stringent conditions, comprising at least one cycle of primer annealing, primer extending and denaturing, using the same primers as used in step (c) or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5"-end portion of said primers used<br><br><br>
in step (c), under conditions in which each primer anneals to the 3"- and 5"-ends of said amplification product, respectively, whereby said amplification product is re-amplified.<br>
43.	A method for detecting DNA complementary to differentially expressed mRNA in two<br>
or more nucleic acid samples, wherein said method comprises reverse transcribing said<br>
mRNA and performing an amplification reaction using primers, characterized in that at least<br>
one primer has the same structure as the primer ofany one of claims 1-25.<br>
44.	The method according to claim 43, which comprises:<br>
(a)	providing a first sample of nucleic acids representing a first population of mRNA transcripts and a second sample of nucleic acids representing a second population of mRNA transcripts;<br>
(b)	separately contacting each of said first nucleic acid sample and said second nucleic acid sample with a first primer ofany one of claims 1-29, in which the 3"-end portion of said first primer comprises a hybridizing nucleotide sequence substantially complementary to a first site in said differentially expressed mRNA to hybridize therewith, under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(c)	reverse transcribing said differentially expressed mRNA to which said first primer hybridizes to produce a first population of first cDNA strands that are complementary to said differentially expressed mRNA in said first nucleic acid sample to which said first primer hybridizes, and a second population of first cDNA strands that are complementary to said differentially expressed mRNA in said second nucleic acid sample to which said first primer hybridizes:<br>
(d)	purifying and quantifying each of said first and second populations of first cDNA strands:<br>
(e)	performing a firsi-stage amplification of each of said first and second population of first DNA strands obtained from step (d) al a first annealing temperature comprising at least one cycle of primer annealing, primer extending and denaturing, using a second primer of any one of claims 1-25 having at its 3" end portion a hybridizing sequence<br><br><br>
substantially complementary to a second site in said first and second populations of first cDNA strands, under conditions in which said second primer anneals to said second site in each population of said first cDNA strands, whereby first and second populations of second cDNA strands are generated;<br>
(0 performing a second-stage amplification of each second cDNA strand generated from step (e) at a second annealing temperature, which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (b) and (e), respectively, or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5"-end portion of said first and second primers used in steps (b) and (e), respectively, under conditions in which each primer anneals to the 3"- and 5"-end sequences of each second cDNA strand, respectively, whereby amplification products of said second cDNA strands are generated, and<br>
(g) comparing the presence or level of individual amplification products in said first and second populations of amplification products obtained from step (f)-<br>
45.	A method for rapidly amplifying a target cDNA fragment comprising a cDNA region corresponding to the 3"-end region of an mRNA, wherein said method comprises reverse transcribing said mRNA and performing an amplification reaction using primers, characterized in that at least one primer has the same structure as the primer of any one of claims 1-25.<br>
46.	The method according to claim 45, which comprises:<br><br>
(a)	contacting mRNAs with a first primer of any one of claims 1-29, in which the 3-end portion of said primer comprises a hybridizing nucleotide sequence substantially complementary to poly A tails of said mRNAs to hybridize therew ith, under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing said mRNAs to which said first primer hybridizes to produce a population of first cDNA strands that are complementary to said mRNAs to which said first primer hybridizes;<br><br><br>
(c)	performing a first-stage amplification of said first cDNA strands at a first annealing temperature comprising at least one cycle of primer annealing, primer extending and denaturing, using a second primer of any one of claims 1-25 having at its 3 "-end portion a gene-specific hybridizing nucleotide sequence substantially complementary to a site in one of said first cDNA strands to hybridize therewith, under conditions in which said second primer anneals to a gene-specific site on one of said first cDNA strands, whereby a gene-specific second cDNA strand is generated; and<br>
(d)	performing a second-stage amplification of said gene-specific second cDNA strand generated from step (c) at a second annealing temperature, which is high stringent conditions, comprising at least two cycles of primer annealing, primer extending and denaturing, using the same first and second primers as used in steps (a) and (c), respectively, or a primer pair each comprising a pre-selected arbitrary nucleotide sequence corresponding to each 5"-end portion of said first and second primers used in steps (a) and (c), respectively, under conditions in which each primer anneals to the 3"-and 5"-end sequences of a gene-specific second cDNA strand, respectively, whereby an amplification product of a gene-specific cDNA strand is generated.<br>
. A method for rapidly amplifying a target cDNA fragment comprising a cDNA region rresponding to the 5"-end region of an mRNA, wherein said method comprises reverse ascribing said mRNA and performing an amplification reaction using primers, aracterized in that at least one primer has the same structure as the primer of any one of tims 1-25.<br>
.   The method according to claim 47. which comprises<br>
(a)	contacting mRNAs w ith an oligonucleotide dT primer or random primer as a cDNA synthesis primer under conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur, in which said cDNA synthesis primer comprises a hybridizing nucleotide sequence substantially complementary to a region of an mRNA to hybridize therewith:<br>
(b)	reverse transcribing said mRNAs. using a reverse transcriptase, to which said cDNA<br><br><br>
synthesis primer hybridizes to produce a population of first cDNA strands that are complementary to said mRNAs to which said cDNA synthesis primer hybridizes, whereby mRNA-cDNA intermediates are generated;<br>
(c)	permitting cytosine residues to be tailed at the 3"-ends of said first cDNA strands in<br>
5            the form of said mRNA-cDNA intermediates by the terminal transferase reaction of<br>
reverse transcriptase;<br>
(d)	contacting the cytosine tails at the 3"-ends of said first cDNA strands generated from<br>
step (c) with an oligonucleotide which comprises a 3"-end portion and a 5"-end portion<br>
separated by a group of universal base or non-discriminatory base analog, wherein the<br>
10	3"-end portion comprises at least three guanine residues at its 3"-end to hybridize with<br>
said cytosine tails at the 3"-ends of said first cDNA strands and the 5"-end portion comprises a pre-selected arbitrary nucleotide sequence, under conditions in which said 3-end portion of said oligonucleotide is hybridized to said cytosine tails:<br>
(e)	extending the tailed 3-ends of said first cDNA strands to generate an additional<br>
15            sequence complementan. to said oligonucleotide using reverse transcriptase, in which<br>
said oligonucleotide serves as a template in the extension reaction, whereby full-length<br>
first cDNA strands are extended;<br>
(0 performing a first-stage amplification of said full-length first cDNA strands obtained<br>
from step (e) at a first annealing temperature, which comprises the steps of:<br>
20	(i) at least one cycle of primer annealing, primer extending and denaturing using a<br>
first primer comprising a nucleotide sequence substantially complementary to the 3*-<br>
end sequences of said full-length first cDNA strands .under conditions in which said<br>
first primer anneals to the 3- ends of said full-length first cDNA strands, whereby<br>
full-length second cDNA strands are generated;<br>
25	(ii) at least one cycle of primer annealing, primer extending and denaturing using a<br>
second primer of any one of claims 1-25 having at its 3"-end portion a gene-specific<br>
hybridizing sequence substantially complementan to a region on one of said full-<br>
length second cDNA strands to hybridize therewith, under conditions in which said<br>
second primer anneals to a gene-specific site on one of said full-length second cDNA<br>
30	strands, whereby a gene-specific cDNA strand is generated: and<br><br><br>
(g) performing a second-stage amplification of said gene-specific cDNA strand at a<br>
second annealing temperature, which is high stringent conditions, comprising at least<br>
two cycles of primer annealing, primer extending and denaturing, using the same first<br>
and second primers as used in steps (f)-(0 and (f)-(ii). respectively, or a primer pair each<br>
5	comprising a nucleotide sequence corresponding to each 5"-end portion of said first and<br>
second primers as used in steps (f)-(0 and (f)-(ii), respectively, under conditions in which each primer anneals to the 3"- and 5"-end sequences of a gene-specific cDNA strand, respectively, whereby an amplification product of a gene-specific cDNA strand is generated. 10<br>
49.	A method for amplifying a population of full-length double-stranded cDNAs<br>
complementary to mRNAs, wherein said method comprises reverse transcribing said mRNA<br>
and performing an amplification reaction using primers, characterized in that at least one<br>
primer has the same structure as the primer of any one of claims 1-25.<br>
15<br>
50.	The method according to claim 49, wherein said method comprises:<br>
(a)	contacting said mRNAs with a first primer of any one of claims 1-29. in which the<br>
3 "-end portion of said first primer has a hybridizing nucleotide sequence substantially<br>
complementary to poly A tails of said mRNAs to hybridize therewith, under conditions<br>
2 0	sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing said mRNAs. using a reverse transcriptase, to which said first<br>
primer hybridizes to produce said population of first cDNA strands that are<br>
complementary to said mRNAs to which said primer hybridizes, whereby mRNA-cDNA<br>
intermediates are generated;<br>
25	(c) permitting cytosine residues to be tailed at the 3"-ends of said first cDNA strands in<br>
the form of said mRNA-cDNA intermediates by the terminal transferase reaction of reverse transcriptase;<br>
(d) contacting the cytosine tails at the 3"-ends of said first cDNA strands generated from step (c) with an oligonucleotide which comprises a 3-end portion and a 5"-end portion<br>
30	separated by a group of universal base or non-discriminatory base analog, wherein the<br><br><br>
3"-end portion comprises at least three guanine residues at its 3"-end to hybridize with<br>
said cytosine tails at the 3"-ends of said first cDNA strands and the 5"-end portion<br>
comprises a pre-selected arbitrary nucleotide sequence, under conditions in which said 3-<br>
end portion of said oligonucleotide is hybridized to said cytosine tails;<br>
5	(e)  extending the tailed 3"-ends of said first cDNA strands to generate an additional<br>
sequence complementary to said oligonucleotide using reverse transcriptase, in which said oligonucleotide serves as a template in the extension reaction, whereby full-length first cDNA strands are extended; and (0  performing an amplification of said full-length first cDNA strands generated from<br>
10	step (e) comprising at least two cycles of primer annealing, primer extending and<br>
denaturing, using a primer pair each comprising a nucleotide sequence corresponding to the same first primer and oligonucleotide as used in steps (a) and (d), respectively, or a primer pair each comprising a nucleotide sequence corresponding to each 5"-end portion of said first primer and oligonucleotide used in steps (a) and (d), respectively, under<br>
15	conditions in which each primer anneals to the 3"- and 5"-end sequences of said full-<br>
length first cDNA strands, respectively, whereby amplification products of full-length cDNA strands complementary to said mRNAs are generated.<br>
51.	A method for amplifying 5"-enriched double-stranded cDNAs complementary to<br>
20      mRNAs, wherein said method comprises comprising reverse transcribing said mRNAs and<br>
performing an amplification reaction using primers, characterized in that at least one primer has the same structure as the primer of any one of claims 1-25.<br>
52.	The method according to claim 51, wherein said method comprises:<br>
2 5	(a)    contacting said mRNAs with a first primer of any one of claims 1-25 under<br>
conditions sufficient for template driven enzymatic deoxyribonucleic acid synthesis to occur, wherein the 3-end portion of said first primer has at least six random nucleotide sequences: (b) performing the steps (b)-(e) of claim 52, whereby 5"-enriched first cDNA strands are<br>
30	extended;<br><br><br>
(c) performing an amplification of said 5"-enriched first cDNA strands generated from<br>
step (b) comprising at least two cycles of primer annealing, primer extending and<br>
denaturing, using a primer pair each comprising a nucleotide sequence corresponding to<br>
each 5"-end portion of said primer and oligonucleotide used in steps (a) and (b),<br>
5	respectively, under conditions in which each primer anneals to the 3"- and 5"-end<br>
sequences of said 5"-enriched first cDNA strands, respectively, whereby amplification products of 5"-enriched cDNA strands are generated.<br>
53.	A method for amplifying more than one target nucleotide sequence simultaneously using<br>
10     more than one pair of primers in the same reaction, wherein said method comprises<br>
performing an amplification reaction using primers, characterized in that at least one primer has the same structure as the primer of any one of claims 1-25.<br>
54.	The method according to claim 53, wherein said method is performed using two stage<br>
15      amplifications, which comprises:<br>
(a)	performing a first-stage amplification of more than one target nucleotide sequence at<br>
a first annealing temperature comprising at least two cycles of primer annealing, primer<br>
extending and denaturing, using the primer pairs of any one of claims 1-25 in which its<br>
3" end portion each of each primer pair has a hybridizing nucleotide sequence<br>
20	substantially complementary to a region of said target nucleic acid sequence to hybridize<br>
therewith, under conditions in which each of said primer pairs anneals to its target nucleotide sequence, whereby the amplification products of target nucleotide sequences are generated; and<br>
(b)	performing a second-stage amplification of said amplification products generated<br>
25             from step (a) at a second annealing temperature, which is high stringent conditions,<br>
comprising at least one cycle of primer annealing, primer extending and denaturing,<br>
using the same primer pairs as used in step (a) or primer pairs each comprising a pre¬<br>
selected arbitrary nucleotide sequence corresponding to each 5 -end portion of said<br>
primer pairs used in step (a), under conditions in which each of each primer pair anneals<br>
30	to the 3"- and 5"-end sequences of said amplification products generated from step (a),<br>
respectively, whereby said amplification products are re-amplified in the same reaction.<br><br><br>
55. The method according to claim 38, 40,42, 46, 48, or 54, wherein said primer pair used in said first-stage amplification is a combination of an annealing control primer selected from the primers of claims 1-25 and an conventional primer.<br>
Dated this 19th day of September, 2004.<br><br>
(TEHEMTAN N. DARUWALLA)<br>
	CONSTITUTED ATTORNEY<br>
FOR THE APPLICANTS.<br>
To:<br>
The Controller of Patents.      The Patent Office, at Bombay.</.o></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtYXNzaWdpbWVudCg2LTctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-assigiment(6-7-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtY2FuY2VsbGVkIHBhZ2VzKDctMTAtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-cancelled pages(7-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtY2xhaW0oZ3JhbnRlZCktKDctMTAtMjAwNSkuZG9j" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-claim(granted)-(7-10-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtY2xhaW1zKGdyYW50ZWQpLSg3LTEwLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-claims(granted)-(7-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtY29ycmVzcG9uZGVuY2UoMTctMTAtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-correspondence(17-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LU1VTU5QLTIwMDQtQ09SUkVTUE9OREVOQ0UoMzEtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">326-MUMNP-2004-CORRESPONDENCE(31-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtY29ycmVzcG9uZGVuY2UoaXBvKS0oMTktMTAtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-correspondence(ipo)-(19-10-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAxOSgxNC03LTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 19(14-7-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAxYSg2LTctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 1a(6-7-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAyKGdyYW50ZWQpLSg3LTEwLTIwMDUpLmRvYw==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 2(granted)-(7-10-2005).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAyKGdyYW50ZWQpLSg3LTEwLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 2(granted)-(7-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAyNig2LTctMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 26(6-7-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAyNig3LTYtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 26(7-6-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAzKDE4LTEwLTIwMDUpLnBkZg==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 3(18-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAzKDE4LTItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 3(18-2-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSAzKDctMTAtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 3(7-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybSA1KDE4LTItMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form 5(18-2-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybS1wY3QtaXBlYS00MDkoNy02LTIwMDQpLnBkZg==" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form-pct-ipea-409(7-6-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtZm9ybS1wY3QtaXNhLTIxMCg3LTYtMjAwNCkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-form-pct-isa-210(7-6-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzcoMTgtMTAtMjAwNSkucGRm" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-petition under rule 137(18-10-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LW11bW5wLTIwMDQtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzcoOC0xMS0yMDA0KS5wZGY=" target="_blank" style="word-wrap:break-word;">326-mumnp-2004-petition under rule 137(8-11-2004).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzI2LU1VTU5QLTIwMDQtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzgoMzEtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">326-MUMNP-2004-PETITION UNDER RULE 138(31-3-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216566-composition-comprising-cysteamine-for-improving-lactation-in-dairy-animals.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216568-a-chimeric-gene-comprising-a-nucleic-acid-fragment-conferring-disease-resistance-to-plants.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216567</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>326/MUMNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SEEGENE INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2th MYUNGJI BLDG., 141-21, SAMSUNG DONG, KANGAM-GU, 135-090 SEOUL RUPUBLIC OF KOREA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHUN JONG-YOON</td>
											<td>319-1301 OLYMPIC SEONSUGIJACHON APARTMENTS, 89, BANGI-DONG, SONGPA-GU, 138-050 SEOUL</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/KR02/01781</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-09-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/KR01/02133</td>
									<td>2001-12-08</td>
								    <td>Republic of Korea</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PCT/KR02/00816</td>
									<td>2002-05-01</td>
								    <td>Republic of Korea</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216567-annealing-control-primer-and-methods-involving-nucleic-acid-amplification-with-the-annealing-control-primer by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:47:36 GMT -->
</html>
