<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216702-bacteria-attenuated-by-a-non-reverting-mutation-in-each-of-the-aroc-ompf-and-ompc-genes-useful-as-vaccines by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:56:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216702:&quot;BACTERIA ATTENUATED BY A NON-REVERTING MUTATION IN EACH OF THE AROC, OMPF AND OMPC GENES, USEFUL AS VACCINES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;BACTERIA ATTENUATED BY A NON-REVERTING MUTATION IN EACH OF THE AROC, OMPF AND OMPC GENES, USEFUL AS VACCINES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides a bacterium attenuated by a non-reverting mutation in each of the aroC gene, the ompF gene and the ompC gene. The bacterium is useful as a vaccine. The bacterium may, for example be an attenuated strain of E.coli useful in vaccination against diarrhoea.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ATTgNUATED_BACTERIA USEFUL IN VACCINES<br>
The invention relates to attenuated bacteria useful in vaccines.<br>
Background to the invention<br>
The principle behind vaccination is to induce an immune response in the host thus providing protection against subsequent challenge with a pathogen. This may be achieved by inoculation with a live attenuated strain of the pathogen, i.e. a strain having reduced virulence such that it does not cause the disease caused by the virulent pathogen.<br>
Clasically, Live attenuated vaccine strains of bacteria and viruses have been selected using one of two different methodologies. Mutants have been created either by treatment of the organism using mutagenic chemical compounds or by repeated passage of the organism in vitro. However, use of either method gives rise to attenuated strains in which the mode of attenuation is unclear. These strains are particularly difficult to characterise in terms of possible reversion to the wild type strain as attenuation may reflect single (easily reversible) or multiple mutation events.  Furthermore, it is difficult to obtain such strains having optimum immunogenic properties because of multiple mutation events, and multiple strains may need to be used to provide protection against the pathogen.<br>
Using modern genetic techniques, it is now possible to construct genetially defined attenuated bacterial strains in which stable attenuating deletions can be created. A number of site directed mutants of Salmonella have been created using this type of technology (2, 4, 5, 9, 12,<br><br>
lu, 17, 18) . KI
(e.g. aroA, aroC, aroD and aroE), pur, htrA, ompR, ompF, ompC, galE, cya, crp and phoP.<br>
Salmonella aroA mutants have now been well characterised<br>
and have been shown to be excellent live vaccines against salmonellosis in several animal species. In addition, in order to reduce the chances of a reversion to virulence by a recombination event, mutations have been introduced into two independent genes such as aroA/purA and aroA/aroC.  Identical mutations in host adapted strains of Salmonella such as S.typhi (man) and S.dublin (cattle) has also resulted in the creation of a number of candidate single dose vaccines which have proved successful in clinial (8, 11) and field trials (10) .<br>
A Salmonella typhimurium strain harboring stable mutations in both ompC and ompF is described in Chatfield et al (1991, ref. 21). When administered orally to BALB/c mice the strain was attenuated, with the 50% lethal dose (LD50) reduced by approximately 1,000-fold. However, the intravenous LD50 was reduced only by approximately 10-fold, demonstrating the importance of the porins in confering on the bacteria the ability to infect by the oral route.<br>
Expression of the ompC and ompF genes is regulated by ompR. Pickard et al (1994, ref. 13) describes the cloning of the ompB operon, comprising the ompR and envZ genes, from a Salmonella typhi Ty2 cosmid bank and characterisation by DNA sequence analysis. The DNA sequence data were used to identify appropriate restriction sites for generating a defined deletion of<br><br>
'"&gt; i "I bp with::) the open •• aciing frame of the ompR Tene. This deletion was introduced by homologous recombination into the chromosomes of two S.typhi strains which already harbored defined deletions in both the aroC and aroD genes. The S.typhi  ompR mutants displayed a marked decrease in ompC and ompF porin expression as demonstrated by examination of outer membrane preparations. It was also shown that the ompR-envZ two component regulatory system plays an important role in the regulation of Vi polysaccharide synthesis in S.typhi.<br>
In animal studies, attenuated S. typhimurium has been used as a vehicle for the delivery of heterologous antigens to the immune system (3, 6, 15). This raises the potential of the development of multivalent vaccines for use in man<br>
Summary of the Invention<br>
The invention provides a bacterium attenuated by a non-reverting mutation in each of the aroC gene, the ompF gene and the ompC gene. Tne invention also provides a vaccine containing the bacterium.<br>
It is believed that the aroC/ompF/ompC combination of mutations gives a vaccine having superior properties. For example, it is believed that the aroC/ompF/ompC combination may be superior to a aroC/ompR combination for two reasons:<br>
1.    The ompR mutation may cause higher levels of attenuation than the ompF/ompC combination of mutations because ompR may regulate a number of genes other than ompF and ompC which are important for survival of the bacterium in vivo.  Thus, the<br><br>
o;,'./-;'-'- :.nnpC comb i ;i.-;'. i on may al low the bacterium to survive in the vaccinated host for a longer time and at higher levels, resulting in better-protection.<br>
2.    The ompR mutation may cause reduced immunogenicity compared to the ompF/ompC combination of mutations because ompR may regulate the expression of antigens important for immunogenicity.<br>
Detailed Description of the Invention<br>
Bacteria useful in the Invention<br>
The bacteria that are used to make the vaccines of the invention are generally those that infect by the oral route. The bacteria may be those that invade and grow within eukaryotic cells and/or colonise mucosal surfaces.<br>
The bacteria are generally Gram-negative.<br>
The bacteria may be from the genera ' Es'cherichia, Salmonella, Vibrio, Haemophilus, Neisseria, Yersinia, Bordetella or Brucella. Examples of such bacteria are Escherichia coli - a cause of diarrhoea in humans; Salmonella typhimurium - the cause of salmonellosis in several animal species; Salmonella typhi - the cause of human typhoid; Salmonella enteritidis - a cause of food poisoning in humans; Salmonella choleraesuis - a cause of salmonellosis in pigs; Salmonella dublin - a cause of both a systemic and diarrhoel disease in cattle, especially of new-born calves; Haemophilus influenza  - a cause of meningitis; Neisseria gonorrhoeas - a cause of gonorrhoeae; Yersinia enterocolitica - the cause of a spectrum of diseases in humans ranging from gastroenteritis to fatal septicemic disease; Bordetella<br><br>
pertussis  - the cause of whooping cough; and Brucella abortus - a cause of abortion and infertility in cattle and a condition known as undulant fever in humans.<br>
Strains of E.coli and Salmonella are particularly useful in the invention. As well as being vaccines in their own right against infection by Salmonella, attenuated Salmonella can be used as carriers of heterologous antigens from other organisms to the immune system via the oral route. Salmonella are potent immunogens and are able to stimulate systemic and local cellular and antibody responses. Systems for driving expression of heterologous antigens in Salmonella in vivo are known; for example the nirB and htrA promoters are known to be effective drivers of antigen expression in vivo.<br>
The invention may be applied to enterotoxigenic E.coli ("ETEC").  ETEC is a class of E.coli that cause diarrhoea.  They colonise the proximal small intestine. A standard ETEC strain is ATCC H10407.<br>
Infections of ETEC are the single most frequent cause of travellers diarrhoea, causing 3-9 million cases per year amongst visitors to developing countries.  In endemic areas, ETEC infections are an important cause of dehydrating diarrhoea in infants and young children, resulting in 800,000 deaths a year in the under fives world-wide.  In developing countries, the incidence of ETEC infections leading to clinical disease decreases with age, indicating that immunity to ETEC infection can be acquired.  In contrast, naive adults from industrialized countries who visit endemic areas are highly susceptible to ETEC infections.  However, with prolonged or repeated visits to endemic areas susceptibility to ETEC infections diminishes, suggesting<br><br>
ihu; ,i live al t •••iiuat ed appru u.:ii to ETEC vaccination may prove successful.<br>
The inventors chose to work on a non-toxigenic strain of ETEC called E1392/75/2A. E1392/75/2A arose spontaneously from a toxic mutant by deletion of toxin genes. In human studies, oral vaccination with live E1392/75/2A gave 75° protection against challenge with toxin-expressing ETEC from a different serotype. However, approximately 15% of vaccinees experienced diarrhoea as a side effect of the vaccine. The strain needs further attenuation to reduce the side effects before it can be considered as a potential vaccine and the invention gives a means of achieving such attenuation.<br>
Seq Id No. 1 shows the sequence of the E.coli aroC gene, Seq Id No. 3 shows the sequence of the E.coli ompC gene and Seq. Id No. 5 shows the sequence of the E.coli  ompF gene .<br>
Further mutations<br>
One or more further mutations may be introduced into the bacteria of the invention to generate strains containing mutations in addition to those in arqC, ompC and ompF. Such a further mutation may be (i) an attenuating mutation in a gene other than aroC, ompC and ompF, (ii) a mutation to provide in vivo selection for cells maintaining a plasmid (e.g. a plasmid expressing a heterologous antigen), or (iii) a mutation to prevent expression of a toxin gene.<br>
The further attenuating mutation may be a mutation that is already known to be attenuating.  Such mutations<br><br>
iMclude mut.a;. ; - =n.s in aro .^iies (e.g. ar\)A, aroD and aroE), pur, htrA, ompR, galE, cya, crp, phoP and surA (see e.g. refs 2, 4, 5, 9, 12, 13, 16, 17 and 18).<br>
A mutation to provide selection for maintenance of a plasmid may be made by mutating a gene that is essential for the bacterium to survive.  A plasmid carrying the essential gene is then introduced into the bacterium, so that; only cells carrying the plasmid can survive.  This may be useful where the plasmid contains, for example, a heterologous antigen to be expressed by the bacterium.<br>
A mutation to prevent expression of a toxin gene may be made to reduce any side-effects caused by vaccination with the bacterium.  For example, in the case of vaccination with E.coli strains such as ETEC it may be desirable to mutate the heat labile toxin (LT) or heat stable toxin (ST) genes so that they are not expressed.<br>
The nature of the mutations<br>
The mutations introduced into the bacterial vaccine generally knock-out the function of the gene completely. This may be achieved either by abolishing synthesis of any polypeptide at all from the gene or by making a mutation that results in synthesis of non-functional polypeptide. In order to abolish synthesis of polypeptide, either the entire gene or its 5'-end may be deleted. A deletion or insertion within the coding sequence of a gene may be used to create a gene that synthesises only non-functional polypeptide (e.g. polypeptide that contains only the N-terminal sequence of the wild-type protein).<br><br>
The mutatii.'i.; an; non - L -•••:•.! r I ing mutations. These are mutations that show essentially no reversion back to the wild-type when the bacterium is used as a vaccine. Such mutations include insertions and deletions. Insertions and deletions are preferably large, typically at least 10 nucleotides in length, for example from 10 to 600 nucleotides.  Preferably, the whole coding sequence is deleted.<br>
The bacterium used in the vaccine preferably contains only defined mutations, i.e. mutations which are characterised. It is clearly undesirable to use a<br>
bacterium which has uncharacterised mutations in its genome as a vaccine because there would be a risk that the uncharacterised mutations may confer properties on the bacterium that: cause undesirable side-effects.<br>
The attenuating mutations may be introduced by methods well known to those skilled in the art (see ref. 14). Appropriate methods include cloning the DNA sequence of the wild-type gene into a vector, e.g. a plasmid, and inserting a selectable marker into the cloned DNA sequence or deleting a part of the DNA sequence, resulting in its inactivation. A deletion may be introduced by, for example, cutting the DNA sequence using restriction enzymes that cut at two points in or just outside the coding sequence and ligating together the two ends in the remaining sequence. A plasmid carrying the inactivated DNA sequence can be transformed into the bacterium by known techniques such as electroporation and conjugation. It is then possible by suitable selection to identify a mutant wherein the inactivated DNA sequence has recombined into the chromosome of the bacterium and the wild-type DNA sequence has been rendered non-functional by homologous<br><br>
Expression of heterologous antigens<br>
The•attenuated bacterium of the invention may be genetically engineered to express an antigen that is not expressed by the native bacterium (a "heterologous antigen"), so that the attenuated bacterium acts as a carrier of the heterologous antigen.  The antigen may be from another organism, so that the vaccine provides protection against the other organism. A multivalent vaccine may be produced which not only provides immunity against the virulent parent of the attenuated bacterium but also provides immunity against the other organism. Furthermore, the attenuated bacterium may be engineered to express more than one heterologous antigen, in which case the heterologous antigens may be from the same or different organisms.<br>
The heterologous antigen may be a complete protein or a par;' •• &gt; 1 ,i protein containing an epitope. The antigen may be from another bacterium, a virus, a yeast or a fungus. More especially, the antigenic sequence may be from E.coli (e.g. ETEC),tetanus, hepatitis A, B or C virus, human rhinovirus such as type 2 or type 14, herpes simplex virus, poliovirus type 2 or 3, foot-a'nd-mouth disease virus, influenza virus, coxsackie virus or Chlamydia trachomatis. Useful antigens include non-toxic components of E.coli heat labile toxin, E.coli K88 antigens, ETEC colonization factor antigens, P.69 protein from B.pertussis and tetanus toxin fragment C.<br>
The ETEC colonization factors and components thereof are prime candidates for expression as heterologous antigens. To instigate diarrhoeal disease, pathogenic strains of ETEC must be able to colonize the intestine and elaborate<br><br>
ent i-'r .-• .'X i i;..; .  For ;uost strain,  of ETEC colonization factojs (CF) that are required for adhesion to the intestinal mucosa have been identified.  In almost all cases CFs are expressed as fimbrae on the outer surface of the bacteria.  A large number of CFs have been identified, the most prevalent being CFAI, CRAII (includes CSI, CS2, CS3) and CFAIV (includes CS4, CSS, CS6) .<br>
A vaccine to ETEC will ideally give protection against a range of colonization factor antigens to ensure that protection against different strains is obtained.  In order to achieve this, it would be possible to express several colonization factors in one strain. Alternatively, the same attenuations could be made in a range of different ETEC strains, each with a different colonization factor.  This would involve deleting the t o x .i n s from s u c h strains.<br>
The [&gt;NA encoding the heterologous antigen is expressed from a promoter that is active in vivo. Two promoters that, have been shown to work well in Salmonella are the nirB promoter (19, 20) and the htrA promoter (20).  For expression of the ETEC colonization factor antigens, the wild-type promoters could be used.<br>
A DNA construct comprising the promoter operably linked to DNA encoding the heterologous antigen may be made and transformed into the attenuated bacterium using conventional techniques. Transformants containing the DNA construct may be selected, for example by screening for a selectable marker on the construct. Bacteria containing the construct, may be grown in vitro before being formulated for administration to the host for vaccination purposes.<br><br>
Formulation of the vaccine<br>
The vaccine may be formulated using known techniques for formulating attenuated bacterial vaccines. The vaccine is advantageously presented for oral administration, for example in a lyophilised encapsulated form. Such capsules may be provided with an enteric coating comprising, for example, Eudragate "S" (Trade Mark), Eudragate "L" (Trade Mark), cellulose acetate, cellulose phthalate or hydroxypropylmethyl cellulose. These capsules may be used as such, or alternatively, the lyophilised material may be reconstituted prior to administration, e.g. as a suspension. Reconstitution is advantageously effected in a buffer at a suitable pH to ensure the viability of the bacteria. In order to protect the attenuated bacteria and the vaccine from gastric acidity, a sodium bicarbonate preparation is advantageously administered before each administration of the vaccine. Alternatively, the vaccine may bo prepared for parerr..eral administration, intranasal -idiui ii j. s t rat i on or intramuscular administration.<br>
The vaccine may be used in the vaccination of a mammalian host, particularly a human host but also an animal host. An infection caused by a microorganism, especially a pathogen, may therefore be prevented by administering an effective dose of a vaccine prepared according to the invention. The dosage employed will ultimately be at the<br>
*<br>
discretion of the physician, but will be dependent on various factors including the size and weight of the host and the type of vaccine formulated. However, a dosage comprising the oral administration of from 101 to 1011 bacteria per dose may be convenient for a 70 kg adult human host.<br><br>
Examplejs<br>
The Examples described in this section serve to<br>
illustrate the invention.<br>
Brief description of the drawings<br>
Figure 1 shows a system for constructing defined deletions in target genes using splicing by overlay extension PCR mutagenesis.<br>
P'igure 2 shows the expected sequences of target genes after recombination and selection for deletions.<br>
Figure 3 shows the cloning of deletion cassettes into pi a sin id pCVD4 42 .<br>
Figure 4 shows an SDS-PAGE analysis of outer membranes prepared from ETEC strains under conditions of low (no salt L-bror;h'j and high (no salt L-broth + 15% sucrose) osmolari ' y.  M "; markers; Sample 1 = PTL010; Sample 2 = PTL002; Sample 3 = PTL003; Sample 4 = AaroCAompC; Sample 5 = AompF.<br>
Figure 5 shows expression of CS1 and CS3 in deletion strains after growth on CFA agar.  Equal numbers of cells from each strain were loaded on a 15% SDS-PAGE gel and Western blotted with monospecific anti-CSl or anti-CS3 polycional antibodies.  Controls for antibody specificity were whole cesll lysates of TGI cells expressing the majore pilin protein of CS1, or purified major pilin protein from CS3.  Lane M, rainbow low molecular mass markers; lane 1, induced TGI cells harbouring pKK223; lane 2, induced TGI cells harbouring pKKCsl; lane 3, CS1-ETEC strain; lane 4, PTL010; lane 5, PTL001; lane 6, PTL002; lane 7, PTL003; lane 8, purified CS3 major pilin<br><br>
Figure 6 shows a Southern blot of mutant loci. Chromosomal DNA was extracted from the wild-type ETEC<br>
(E1392/75-2A), PTL001 (htrA aroC), PTL002 (aroC ompR) and PTL003 (aroC ompC ompF) as indicated, digested with restriction endonuclease EcoRV, and pulsed field electrophoresed through 1% agarose.  DNA was blotted from the gel onto Hybond N+ nylon membranes (Amersham) and hybridised with DNA probes derived from the aroC, htrA, ompR, ompC, or ompF loci as shown.  The banding patterns are consistent with the mutant loci being deletions.<br>
Figure 7 shows the IgA responses in volunteers administered a vaccine according to the invention.<br>
EXAMPLE 1: CONSTRUCTION AND CHARACTERISATION OF STRAIN ACCORDING TO THE INVENTION<br>
Des_jgn___of_de_l_e_tions and construction of plasmids pCVDAAz-oC, pCVDZlOmpC and pCVDZlQmpF<br>
Deletions were designated to remove the entire open reading frame of the target gene. Using the E.coli genome sequence as a template, PCR primers were designed to amplify fragments of 500-600 base pairs flanking the target open reading frame (see Table 1 for primer sequences). Splicing by overlap extension using PCR was used to fuse the two flanking sequences, creating a PCR product with the entire gene deleted (Figure 1).  The wild-type sequences around the deletion site and the predicted sequences after deletion are depicted in Figure 2.<br>
For each gene two different restriction sites were<br><br>
int roduo.'d into Lhe :.;;Lice region (see Table 2 below) . These were used for identification of deletion clones. The PCR primers at either end of the PCR fragment introduced unique restriction sites that were used to clone the fragment into the multiple cloning site of pCVD442 (Figure 3) .<br>
PCR products were gel purified using a Qiagen (Trade Name) gel extraction kit and digested with the relevant restriction enzymes prior to ligation to the suicide plasmid pCVD442(22) digested with the same enzyme and treated with alkaline phosphatase to prevent vector self-ligation (Figure 3). The ligation mix was transformed into SY327Xpir and plated on L-Ampicillin (100 ug/ml) plates. Plasmids from Ampicillin resistant transformants were screened for the presence of the deletion cassettes by restriction digestion. The following plasmids were gene rater):<br>
pCVD/l/Vroi ' pCVDZlOmpC pCVDZlOmpF<br>
The suicide plasmid pCVD442 can only replicate in cells harboring the pir gene. On introduction into non-pir strains, pCVD442 is unable to replicate, and the Ampicillin resistance conferred by the plasmid can only be maintained if the plasmid is integrated in the chromosome by a single homologous recombination event. The plasmid also has a sacB gene, encoding levan sucrase, which is toxic to gram negative bacteria in the presence of sucrose. This can be used to select clones that have undergone a second recombination event, in which the suicide plasmid is excised. Such cells will be resistant to sucrose, but Ampicillin sensitive.<br><br>
Constructionand characterisation of AAroCAOmpCAOmpF strain<br>
Tliis section outlines the chronology of construction and history of a AAroCAOmpCAOmpF strain. In the section, "ETEC" refers specifically to strain E1392/75/2A or its deri va ti ves.<br>
AAroCAOmpCAOmpF deletions were introduced into E1392/75/2A in the following order: AAroC -* A AroCAOmpC-AAroCAOmpCAOmpF<br>
Construction of ETECAAroC<br>
1)	E1392/75/2A from original microbanked stock was<br>
p1a ted onto L-Ag a r.<br>
2)	Electroporation competent cells were prepared from<br>
these cells. 100 pi aliquots were frozen.<br>
3)	pCVDZMroC was purified from SY327pir cells using a<br>
Qiagen Qiafilter (Trade Name) midiprep. The plasmid<br>
was concentrated about 10-fold by ethanol<br>
precipitation. The construction of pCVDZl/lroC is<br>
described above.<br>
4)	5 pi of concentrated plasmid was mixed with 100 pi<br>
defrosted cells and electroporated. The whole<br>
transformation was plated on an L-Ampicillin plate<br>
(50 pg/ml) and incubated overnight at 37°C.<br>
5)	A single Ampicillin resistant colony grew.<br>
6}    The colony was streaked onto an L-Ampicillin plate (100 pg/ml) and grown overnight at 37°C ("merodiploid plate").<br>
7)    PCR using primers T?19 and TT20 (specific for the aroC gene)and a colony picked from the merodiploid plate amplified two bands, with sizes corresponding to that of the wild-type and AaroC genes. The sequences of the primers are shown in Table 1<br><br>
bo Low<br>
8}    A colony from the merodiploid plate was grown up for 7 hr in a) L-Ampicillin broth (100 ug/ml) and b) L-Broth. The colony grown on L-Ampicillin was microbanked.<br>
9)	Serial dilutions of the L-broth culture were set up<br>
on :<br>
a)	No salt L-agar<br>
b)	No salt L-agar + 5% sucrose.<br>
The plates were incubated overnight at 30°C.<br><br>
10)	Colony counts showed that 10" more colonies grew on<br>
L-agar than on L-agar + 5% sucrose, showing sucrose<br>
selection worked.<br>
11)	Sucrose resistant colonies were screened for the<br>
presence of AaroC gene by PCR. Colonies chosen for<br>
screening were picked onto an L-agar plate and<br>
grown overnight at 37°C. This plate was stored at<br>
4°C, whilst further tests were carried out.<br>
50-s of 90 colonies tested had AaroC only. Colonies were tested for growth on:<br>
a)	M-9 minimal media plates<br>
b)	M-9 minimal media + Aromix plates<br>
c)	L-Amp (100 ug/ml)<br>
AaroC colonies should not grow on M-9 minimal media<br>
without Aromix or on L-Amp.<br>
Aromix is a mix of aromatic compounds as follows   (Table Remove)<br><br><br>
Those i •ornpound;; i:" made in wild-type ba&gt;"t&gt;v i.i, b&gt;:-the ciroC mutation prevents their synthesis. 13/14 putative AAroC colonies required Aromix for growth on M-9 minimal media and were susceptible ' •.' Ampici1lin.<br>
15)	3 colonies (No. 1,2,3) were tested for the presence<br>
of the CS1 major pilin protein gene by PCR using<br>
primers MGR169 and MGR170. All 3 colonies gave PCR<br>
products of the expected size (700 bp.). The<br>
sequences of the primers are shown in Table 1.<br>
16)	Colonies 1, 2 and 3 from screening master plate<br>
were streaked onto L-Agar and grown overnight at<br>
37"C. Cells from these plates were used to<br>
inoculate microbank tubes.<br>
17)	Colony 1, stored in a microbank, was used for<br>
further work.<br>
IS)   For permanent storage, a bead from the microbank<br>
tray was inoculated into 1 ml L-broth, grown for 4 hi: with shaking at 37°C and used to make agar slopes which were used to make freeze dried stocks . The freeze dried stock of E1392/75/2AZlAroC was designated PTL004. 20 ml of L-broth was added to the rest of the 1 ml culture and the culture was incubated overnight at 30°C. 1 ml of the overnight culture was transferred to each of three cryovials and stored in liquid nitrogen.<br>
Construction of ETECZLflroC^Qmpr<br>
1)    Preparation of pCVDdOmpC plasmid DNA for<br>
electroporati on:<br>
A colony of SY327Apir harbouring pCVDAOmpC was grown overnight at 37°C in 100 ml L-Ampicillin broth<br>
(100 pg/ml). Plasmid DNA was purified using 2 Qiagen Qiafilter (Trade Name) midipreps. DNA was<br><br>
fii L'l h&gt;.&gt; i cc-ii-••-n 1.1 at.ed by • 'hanol precipitation. Th&gt;-construct'i on of pCVDAOmpC is described above.<br>
2)	Preparation of electrocornpetent cells:<br>
E'TECAAroC cells from the microbank tray produced in<br>
step 17 of the preceding section were streaked on<br>
L-agar, grown at 37°C overnight and then stored at<br>
4°C for no more than 1 week before being used to<br>
inoculate cultures for preparing electrocompetent<br>
cells .<br>
3)	ETECZJ/lroC cells were electroporated with 5 ul of<br>
concentrated pCVDAOmpC DNA, and each transformation<br>
plated on a single L-Ampicillin plate (50 pg/ml)<br>
and grown overnight at 37°C.<br>
4)	17 Ampicillin resistant colonies (putative<br>
ETECZl/iroC/ pCVDAOmpC merodiploids) were obtained.<br>
':&gt;)    These colonies were spotted onto a master L-<br>
Ampicillin (100 pg/ml) plate and used as templates for PCR with primers TT7/TT8. The master plate was grown at room temperature over the weekend. The sequences of the primers are given in Table 1 be i ow.<br>
6)	A single colony (No, 7) had the AompC gene.<br>
7)	The colony was grown for 5 hr in L-broth.<br>
8)	Serial dilutions of the L-broth culture were set up<br>
on:<br><br>
a)	No salt L-agar<br>
b)	No salt L-agar + 5% sucrose.<br>
The plates were incubated overnight at 30°C.<br>
9} Colony counts showed that 10" more colonies grew on L-agar than on L-agar + 5% sucrose, showing sucrose selection worked.<br>
10)   45 sucrose resistant colonies were screened for<br>
AompC by PCR using primers TT7 and TT8. 9 colonies had the AompC gene, but most had traces of w.t. ompC gene. The sequences of the primers are given<br><br>
i n Tab I «- ! ho low.<br>
11)	To further characterise putative ETECAAroCAOmpC<br>
colonies, they were grown in 1 ml L-Broth for 5 hr<br>
and plated on:<br>
a)	L-Agar -*- 100 ug/ml Ampicillin<br>
b)	L-Agar<br>
c)	L-Agar + 5% sucrose<br>
AOmpC colonies should be resistant to sucrose and sensitive to Ampicillin.<br>
12)	Only 1 colony (No. 1) was Ampicillin sensitive and<br>
sucrose resistant.<br>
13)	Colony 1 was checked for the presence of AaroC,<br>
AompC and CS1 genes by PCR with primers TT19/TT20,<br>
TT7/TT8 and MGR169 and 170. The sequences of the<br>
primers are given in Table 1 below.<br>
L4 :   Colony 1 cave single PCR products of the expected size for AaroC, AompC and CS1 genes. The colony was microbanked.<br>
For permanent storage, a bead from the microbank was inoculated into 1 ml L-broth, grown for 4 hr with shaking at 37°C and used to make agar slopes which were freeze dried. The freeze dried stock of E1392/75/2A^AroCZlO/npC was designated PTL008 . 20 ml of L-broth was added to the rest of the 1 ml culture and the culture was incubated overnight at 30°C. 1 ml of the overnight culture was transferred to each of three cryovials and stored in liquid nitrogen.<br>
Construction of ETEC&amp;AroCAOiry&gt;CAOznpF<br>
Conjugation was used to introduce pCVDZIQmpF into E1392/75/2AZ)AroC^O/npC.<br>
1)    Conjugation donor cells SMIOApir were transformed with pCVDzlO/npF. The construction of plasmid<br><br>
pCVD/]0/:;/.',:- i ,-: cii.s,-; ;:--•! above.<br>
,°)    ETEC,d/lroCZ]OmpC cells were conjugated with SMIOApir/ pCVDZlOmpF cells.  The pCVD442 plasmid includes a transfer origin which allows the plasmid to be transferred from a donor strain containing the RP4 transfer genes (e.g. SMIOApir) to a recipient strain (e.g. ETEC).  ETECAaroCAOmpC cells and E.colj strain SMIOApir harbouring the PcvdAompF recombinant were cross-streaked on L-agar plates so as to cover an area of approximately 10 cm2. Plates were incubated at 31° C for 20 h, then the growth washed off using 4 ml L-broth and the suspension plated onto McConkey agar (Difco) containing streptomycin at 20//g ml"1 and ampicillin at 300/^g ml"!.  Plates were incubated overnight at 37'C and resulting colonies were checked for merodiploidy by PCR using appropriate ol :i gonucleot ides as primers.<br>
!)    Putative ETEC transconjugants were screened. 10<br>
colonies were picked from McConkey agar plates and grown overnight on L-Ampicillin (100 ug/ml) agar. The presence of AompF gene was checked for by PCR with primers TT1/TT2. The sequences of the primers are given in Table 1 below.<br>
4)	The colonies were grown for 5 hr in L-Broth.<br>
5)	Serial dilutions of the L-broth culture were set up<br>
on:<br><br>
a)	No salt L-agar<br>
b)	No salt L-agar + 5% sucrose.<br>
The plates were incubated overnight at 30°C.<br><br>
6)	Colony counts showed 10b more colonies grew on L-<br>
agar than on L-agar + 5% sucrose, showing sucrose<br>
selection worked.<br>
7)	Sucrose resistant colonies were screened for AompF<br>
gene by PCR with primers TT1/TT2. The sequences of<br><br>
;-}-,,, p, imer.s are  ven in Table 1 below. The screened colonies were grown overnight on L-Agar. 3 colonies out of 47 had the AompF gene with no evidence of the wild-type ompF gene.<br>
8)	To further characterise putative<br>
ETECAAroCAOmpCAOmpF colonies, they were plated on:<br>
a)	L-Agar + 100 ug/ml Ampicillin<br>
b)	L-Agar<br>
c)	L-Agar + 5% sucrose<br>
AompF colonies should be resistant to sucrose and sensitive to Ampicillin.<br>
9)	All three AompF colonies were Ampicillin sensitive<br>
and sucrose resistant.<br>
10)	The colonies were microbanked and one colony was<br>
chosen as a ma s tei: stock .<br>
11)	For permanent storage, a bead from the master stock<br>
was inoculated into 1 ml L-broth, grown for 4 hr<br>
with shaking at 37°C and used to make agar slopes<br>
which were used to make freeze dried stocks. The<br>
freeze dried stock of E1392/75/ 2AAaroCAompCAompF<br>
was designated PTL003. 20 ml of L-broth was added<br>
to the rest of the 1 ml culture and the culture was<br>
incubated overnight at 30°C. 1 ml of the overnight<br>
culture was transferred to each of three cryovials<br>
and stored in liquid nitrogen.<br>
Characterisation of E1392/75/2AAflroCZlOng?CZlQny)F<br>
1)    Growth requirements:<br>
Cells taken from the master stock produced in step<br>
10 of the preceding section were streaked on L-Agar<br>
plate. At the same time 8 ml L-broth was inoculated<br>
for a chromosomal DNA prep for Southern blots. Both<br>
plate and liquid culture were grown overnight at<br>
37°C.<br>
Cells from the grown plate were streaked onto the<br><br>
Medium	Growth<br>
L-Amp	No<br>
M9 minimal	media	No<br>
M9 minimal	-f Aromix	Yes<br>
M9 + sulfathiazole (100 ug/ml)	No<br>
M9 + sulfathiazole (100 ug/ml)	+ Aromix  Yes<br>
L-Agar + 50 ug/ml streptomycin	Yes<br>
L-Agar + 5% sucrose	Yes<br>
As expected, the cells were Amp sensitive. The<br>
cells were resistant: to sucrose, streptomycin and<br>
sulfathiazole, but required Aromix to grow on<br>
minimal media.<br>
LPS ana 1ysis of PTL003:<br>
.-&gt;.)    A freeze dri^d via] of PTL003 was broken<br>
open. The culture was resuspended in L-Broth<br>
and plated on<br>
L-Agar for growth. Some cells were scraped<br>
off and stored in microbank. b)    More cells were scraped off and the LPS<br>
profile was analysed. There was no visible<br>
difference between the LPS profile of PTL003<br>
and original E',1392/75/2A strain. Confirmation of deletions by PCR:<br>
a)	A scrape of cells was taken from the plate<br>
made in in 2a and streaked onto L-Agar and<br>
grown overnight.<br>
b)	Freshly grown cells were used for PCR with<br>
primers that flank the following genes: aroC,<br>
htrA, ompC, ompF, ompR.<br>
c)	PTL003 was shown to have deletions in aroC,<br><br>
ompC and ompf gene:;, but not in hr.rA or ompR. A:i,ilysi.s o 1. outer membiai.-.; protein proLile oi FT LOO. 3:<br>
Outer membrane protein fractions were prepared from strains PTL010 (El 392/75/2A) and the deletion strains PTL002 and PTL003. A strain with a single ompF deletion and a strain with both aroC and ompC deletion were also analysed. Strains were grown under conditions of low osmolarity (no salt L-broth) and high osmolarity (no salt L-broth+15% sucrose) . The OmpF protein product is normally expressed at low osmolarity whereas the OmpC product, is expressed at high osmolarity. The OmpC and OmpF proteins have similar electroporetic mobilities. At both high and low osmolarities , the strain PTL003 lacks proteins in the OmpC/OmpF region when compared to the wild-type E1392/75/2A strain or to the AAroCAOmpC or AOmpF deletion<br>
Expression of CS1 and CS3 pili on CFA agar: The expression of CS1 and CS3 pili in the deletion strains was examined. Equal numbers (2 A600nm units) of bacteria strains PTL010, PTL001, PTL002 and PTL003 grown overnight at 37°C on CFA agar were subjected to SDS PAGE and analysed by Western blotting with monospecific polyclonal antibodies against CS1 or CS3 . CS1 and CS3 pili were expressed equally well in four strains (Figure 5) .<br>
A CFAII-negative derivative of E1392/75/2A was constructed for use as a control. This was done by specific curing of the CS encoding plasmids from ETEC strain E1392/75-2A. A short fragment of DNA was amplified from the cooB gene using PCR with<br><br>
•&gt; 1 i q^nuc ! '•"M i des C.CAO :  -<:id csa02 as primers and ligated into pgem-t easy plasmid vector name promega designed for the cloning of pcr products. fragment was subcloned pcvd442 by virtue sail sphi restriction enzyme sites. pcvd442-coob derivative introduced etec strain e1392 conjugation from smioapfr. ampicillin resistant transconjugants are most likely to be result fusion with coob-bearing plasmid. such transconjugates were then grown on l-agar supplemented sucrose select loss sacb gene pcvd442. resulting colonies tested sensitivity using csa01 primers. three included positive controls among these pcrs. two that qave no product streaked out onto fresh obtain pure cultures. in l-broth at approximately h microbanked cs1 encoding confirmed analysis profiles derivatives agarose gel electrophoresis. negative but still cs3></:id>
6)    Southern blotting of PTL003:<br>
Structure of deletion mutations.  Total DNA was extracted from cultures of the three deletion mutants grown from the microbanked stocks, digested with restriction endonuclease EcoRV, and the digested DNA subjected to pulsed field agarose gel electrophoresis.  DNA was blotted from the gels onto Hybond N+ (Trade Name) nylon membranes and<br><br>
hybridised with appropriate DNA probes according to standard procedures.  Results (Figure 6) show that the hybridising chromosomal DNA fragments of the mutants are shorter than the wild-type, consistent with the mutations being deletions.<br>
Confirmation of absence of Heat-Stable (ST) and Heat-Labile (LT) toxin genes in E.coli strain E1392/75-2A.  For this the ST and LT-AB genes were used as DNA probes against total DNA from E1392/75-2A.  Total DNA from the toxin positive ETEC strain E1393/75 was included as a positive control, while that from the laboratory E.coli strain JM109 was included as a negative.  Hybridised membranes were left under Hyperfilm-ECL (Trade Name) for 1 h to obtain the maximum amount of signal.  Probes were prepared using PCR with plasmid DNA extracted from E1392/'/5-2A as template and oligonucleotides EST01 and EiJ'i'02 as primers for ST, or LT-R1 and LT-03 for LT-AB.  There was no significant hybridisation with tota] DNA using either the LT-AB or the ST probe, despite obtaining a very intense signal from the positive control total DNA.<br>
Confirmation of absence of pCVD442 sequences from the chromosome of deletion mutants. ' The plasmid pCVD442 was labelled and hybridised to total DNA from deletion mutants PTL001, PTL002 and PTL003 digested with EcoRV.  Total DNA from ETEC strain E1392/75-2A was included as a control.  A complex pattern of hybridising DNA fragments was obtained. But, there was no significant difference between the pattern obtained for the wild-type and that for the mutants, indicating that probably no residual pCVD442 nucleotide sequences were left in the genomes of the mutants.  The complex pattern of<br><br>
hyb r : i i r, i nq f raqrv •:• t; s was most  likely due ' &gt; ".he pCVD442 probe hybridising with the plasrnid DNA components of the E1392/75-2A strain and mutant deri vatives.<br>
(Table Remove)<br><br><br><br>
(Table Remove)<br><br><br>
Table <br>
EXAMPLE 2 :  SAFETY AND IMMUNOGENICITY OF ATTENUATED<br>
OF  ENTEROTOXIGENIC E. COLI<br>
ompC/XlompF) IN HUMAN VOLUNTEERS<br>
The study wa; designed to evaluate a candidate live attenuated vaccine strain of  enterotoxigenic £. cola, namely the AaroC/AompC/AompF PTL003 described above.<br>
Preparation of the vaccine seed lots<br>
The bacterial strain was plated onto MacConkey agar for purity and for confirmation of identity, and 5 colonies used to inoculate a flask containing 200 ml of luria broth. After 8 hours incubation at +37°C, 30 ml of sterile glycerol was added to the broth culture and aliquots prepared (1 ml per vial) . One hundred such vials were frozen at -70°C. These vials constituted the seed lot for the vaccine strain.<br>
Purity -"if t. ho 5;'--.'-; lot VMS ensured by selecting ten random vials, and testing them for bacterial purity and freedom from fungi. An additional three vials were tested to determine the number of bacteria in the vials using standard plate count methods with serial dilutions of the culture broth.<br>
Preparation of the vaccine<br>
The vaccine was prepared fresh prior to each vaccination and all steps in the preparation of the inoculum carried out in a safety cabinet. The day prior to vaccination, 0.2 ml was spread onto the surface of luria agar plates using sterile cotton swabs to prepare the lawn of bacteria. The same culture broth was streaked onto MacConkey and luria agar plates for purity. The agar plates were incubated at 37 °C for 18 hours in a sealed container with tamper-resistant indicator tape to ensure that the plates were not tampered with during incubation. After incubation, che lawn of bacteria was harvested with 5 ml of sterile phosphate buffered saline (PBS), and the optical density of the suspension measured. The appropriate volume of this suspension, corresponding to the desired dose, was then placed into unit dose bottles with 30 ml of bicarbonate buffer and administered to the volunteers. An extra dose of vaccine was prepared and left in the laboratory, and immediately after the volunteers had been vaccinated the actual number of bacteria in each dose of vaccine was validated using standard colony count procedures with ten fold dilutions of vaccine.<br>
The procedure for diluting the bacteria was established during preliminary studies using lawn cultures prepared and incubated exactly as done for the vaccine preparations administered to volunteers. Suspensions were made and the<br>
;: ::;ii&gt;-: of vi ;: !'- ha c t i J .•: •.•numerated by colony counts of serial dilutions and related to the determined optical density. Based on these preliminary studies, a standard procedure was developed for preparing and validating the correct dilutions of bacteria in order to give the doses stated.<br>
Preparation of buffer<br>
A buffer consisting of sodium bicarbonate in water was used.  For each dose of vaccine 150 ml of deionised water containing 2 gram of sodium bicarbonate was prepared and filter sterilised. 30 ml of the buffer was placed into 50 ml sterile vials and the dose of vaccine bacteria was added to these vials. The remaining 120 ml of buffer was placed into separate sterile bottles. At the time of vaccination, the volunteers were first administered 120 ml of buffer, then a minute  later,  30 ml of buffer containing  the vaccine.<br>
Vaccination schedule<br>
Groups of volunteers were studied in a dose escalation manner. The first group of volunteers received a dose of approximately 5X107 bacteria, the second' a dose of approximately 5X109 and the third group a dose of approximately 5X108.<br>
The volunteers were given Ciprof loxacin 500 mg BID for three days beginning on day 4. They were discharged on day 6, having had a haematology and chemistry screen for safety. Serum was saved for antibody measurement.<br>
On days 9 and 14 the volunteers returned for follow-up outpatient visits at which time an interval history was<br>
done and .1 hlood .samp ' • • was obtained for so: logical assays. In total, bJood (40 ml) was collected for serology three times, prior to vaccination and on day 9 and day 14 after vaccination.<br>
Laboratory Assay Procedures<br>
Up to two faecal specimens were cultured each day while the volunteers were in hospital. For qualitative cultures, a faecal swab was placed into Gary Blair transport media and taken to the laboratory where it was inoculated directly onto MacConkey agar and onto MacConkey agar containing antibiotics selective for the vaccine strain. Up to five colonies were shown to be agglutinated using antisera specific for the vaccine strain. For quantitative culture (first; specimen each day only) faecal specimens were weighed and diluted in PBS, with serial 10-fold dilutions up to 10"4, and then 100 ul of each dilution was spread onto MacConkey agar with antibiotics. Suspected colonies were confirmed by agglutination with anti-0 serum.<br>
Serum was collected and saved for subsequent assay for antibody against CFA II antigens by ELISA and bactericidal antibody against the vaccine strain.<br>
Peripheral blood mononuclear cells were separated from whole blood collected into citrate and washed. These cells were cultured at a density of 107 cells per ml in RPMI tissue culture medium at 37°C for 48 hours. After 48 hours the supernatant was transferred to a cryovial and frozen at -20°C until it could be assayed for IgG and IgA antibody to CFA II by ELISA.<br><br><br>
Table<br>
Results:<br>
No symptoms were seen at all actual doses of 6.8 x 107 and 3.7 x 10a cfu. At the higher dose of 4.7 x 109 1/6 volunteers experienced diarrhoea and 2/6 had mild abdominal cramps. Bacterial shedding was seen in all volunteers at the 5X109 cfu dose level form day 1 post vaccination until, as per protocol, ciprofloxacin was started on day 4 after vaccination. This indicates good intestinal colonisation, which is indicative of the potential to induce a good immune response. At the two lower doses, vaccine strain was recovered from all volunteers on at least one time point following vaccination but the duration of the excretion was reduced compared to that seen at the highest dose.<br>
At the time of filing the application, the analysis of the immune responses generated by the vaccine was incomplete.<br>
However, the IgA &gt;int. L !,TA II responses in the culture supernatanLs of PBMNC purified from the blood of recipients of the highest dose of vaccine at day 0 (before vaccination) and days 7 and 10 post vaccination have been analysed (see Figure 7) , Supernatants were analysed by ELISA on assay plates coated with purified CFA II antigen. The OD values observed from the day 7 and day 10 samples were significantly higher than those from the pre-vaccination samples, demonstrating the induction of a specific IgA response at these time points. As expected, the responses show a peak at day 7 and are reduced at day 10, consistent with the homing of primed IgA secreting B-cells from the blood to the mucosal effector sites of the Gut Associated Lymphoid Tissue.<br>
Conclusions:<br>
The attenuated live strain of ETEC (AaroC/AompC/AompF) has been shown to be well tolerated in healthy adult-volunteers and to colonise the intestine in a manner-consistent with its utility as an oral vaccine to protect against travellers diarrhoea.  It has also been demonstrated to elicit a specific mucosal immune response.<br>
1.	Bacon, G.A., Burrows, T.W. and Yates, M. (1950)<br>
Br.J.Exp. Pathol., 31, 714-24<br>
2.	Chatfield, S.N., Charles, I.G., Makoff, A.J. et al<br>
(1992a) Biotech. 10, 888-892<br>
3.	Chatfield, S.N., Strahan, K., Pickard, D., Charles,<br>
T.G., Hormaeche, C . E . and Dougan, G. (1992b)<br>
Microbiol. Pathog., 12, 145-151<br>
4.	Curtiss III, R. and Kelly, S.M. (1987)<br>
Infect.Immun. 55, 3035-3043<br>
5.	Dougan, G., Chatfield, S., Pickard, D., Bester, J.,<br>
O'Callaghan, D. and Maskell, D. (1988) J.Inf.Dis.<br>
156, 1329-1335<br>
b.    Fairweather, N.F., Chatfield, S.N., Makoff, A.J. et al (1990) Infect.Immun. 58, 1323-1329<br>
7.	Gomaz-Uuarte, O.G., Galen, J., Chatfield, S.N.<br>
(1995) Vaccine, 13:1596-1602<br>
8.	Hohmann, E.L., Oletta, C.A., Killeen, K.P. and<br>
Miller, S.I. (1996) Vaccine 14, 19-24<br>
9.	Hone, D., Morona, R,, Attridge, S. and Hackett, J.<br>
(1987) J.Infect.Dis., 156, 167-1<br>
10.	Jones, P.W., Dougan, G., Haywood, C., MacKensie,<br>
N., Collins, P. and Chatfield, S.N. (1991) Vaccine<br>
9, 29-36<br>
11.	Levine, M.M., Galen, J., Barry, E. et al (1995)<br>
J.Biotech, 44, 193-196<br>
12.	Miller, S.I., Kukral, A.M. and Mekalanos, J.J.<br>
(1989), Proc.Nat.Acad.Sci., USA 86, 5054-5058<br>
13.	Pickard, D., Li, J.L., Roberts, M., Maskell, D.,<br>
Hone, I)., Levine, M., Dougas, G. and Chatfieid, S.<br>
(,L'J94)  election an..: ! :;i;;iuni ty 62, .3984-3993<br>
14.	Sarnbrook, J., Fntsch, E.F. and Maniatis, T.,<br>
(1989)<br>
Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA<br>
15.	Strugnell, R.A., Dougan, G., Chatfield, S.N. et al<br>
(1992) Infect.Immun., 60, 3994-4002<br>
16.	EP-B-0322237 (Dougan et al)<br>
17.	EP-B-0400958 (Dougan et al)<br>
18.	EP-B-0524205 (Dougan et al)<br>
19.	WO 92/15689 (Charles et al)<br>
20.	Everest., P., Allen, J., Papakonstantinopoulou, A.,<br>
Mar.troeni, P., Roberts, M. and Dougan, G. (1995)<br>
FEMS Microbiol. Letts., 126, 97-101<br>
2.1.   Chatfield, S.N., Dorman, C.J., Hayward, C. and<br>
Dougan, G. (1991) Infection &amp; Immunity 59, 449-452<br>
22.   Dormenberg, M.S. and Kaper, J.B. (1991) Infection and Immunity 59, 4310-4317<br><br>
CLAIMS<br>
A bacterium attenuated by a non-reverting mutation in each of the aroC gene, the ompF gene and the ompC gene.<br>
2.	A bacterium according to claim 1 which infects by<br>
the-1 oral route.<br>
3.	A bacterium according to claim 1 which is from the<br>
genera Escherichia, Salmonella, Vibrio,<br>
Haemophilus, Neisseria, Yersinia, Bordetella or<br>
Brucella.<br>
4.	A bacterium according to claim 3 which is a strain<br>
of Escherichia coli, Salmonella typhimurium,<br>
Salmonella typhi, Salmonella enteritidis,<br>
Salmonella choleraesuis, Salmonella dublin,<br>
Haemophilus influenzae, Neisseria gonorrhoeae,<br>
Yersinia enterocolitica, Bordetella pertussis or<br>
Brucella abortus.<br>
5.	A bacterium according to claim 4 which is a strain<br>
of enterotoxigenic E.coli (ETEC).<br>
6.	A bacterium according to any one of the preceding<br>
claims which is further attenuted by a mutation in<br>
a fourth gene.<br><br>
A •' "' ' t  r i um aC'-o rd i nq t.o claim 6 wherein t;he fourth gone is aroA, aroD, aroE, pur, htrA, galE, cya, ci'p, phoP or surA.<br>
8.    A bacterium according to any one of the preceding claims, wherein the mutation in each gene is a<br>
d e f. i n e d mu t a t i on .<br>
9.    A bacterium according to any one of the preceding claims, wherein the mutation in each gene is deletion of the entire coding sequence.<br>
10.   A bacterium according to any one of the preceding claims which has been genetically engineered to express a heterologous antigen.<br>
11.	A bacterium according to claim 10, wherein<br>
expression of the antigen is driven by the nirB<br>
promoter or the htrA promoter.<br>
12.	A vaccine comprising a bacterium as defined in any<br>
one of the preceding claims and a pharmaceutically<br>
acceptable carrier or diluent.<br>
13.   A bacterium as defined in any one of claims 1 to 11 for use in a method of vaccinating a human or an-imal.<br>
14.   An enterotoxigenic E.coli cell attenuated by a non-reverting mutation in each of the aroC gene, the<br><br>
• •::.;&gt;/' ' aii i ! ho omp(' ye:i'.;,  for use i :: .) method oi vaccinating a human or animal against, diarrhoea.<br>
15.	Use of a bacterium as defined in any one of claims<br>
] to 11 for the manufacture of a medicament for<br>
vaccinating a human or animal.<br>
16.	A method of raising an immune response in a<br>
mammalian host, which comprises administering to<br>
the host a bacterium attenuated by a non-reverting<br>
mutation in each of the aroC gene, the ompF gene<br>
and the ompC gene.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUFic3RyYWN0LSgwNS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Abstract-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUNsYWltcy0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Claims-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Correspondence-Others-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDA1LTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Description (Complete)-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLURyYXdpbmdzLSgwNS0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Drawings-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUZvcm0tMS0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Form-1-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUZvcm0tMTMtKDA1LTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Form-13-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUZvcm0tMi0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Form-2-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUZvcm0tMy0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Form-3-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUZvcm0tNS0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Form-5-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLUdQQS0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-GPA-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLVBldGl0aW9uLTEzNy0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Petition-137-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=SU4tUENULTIwMDAtMDAyMTYtREVMLVBldGl0aW9uLTEzOC0oMDUtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">IN-PCT-2000-00216-DEL-Petition-138-(05-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC0zMzEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-331.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC0zMzIucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-332.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC00MDIucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC1ub3RpZmljYXRpYW4ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-notificatian.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDAtZGVsLTAwMjE2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2000-del-00216-pct-search report.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216701-2-pyridone-derivatives-as-inhibitors-or-neutrophile-elastase.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216703-a-pharmaceutical-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216702</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2000/00216/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Sep-2000</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PEPTIDE THERAPEUTICS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>PETERHOUSE TECHNOLOGY PARK, 100 FULBOURN ROAD, CAMBRIDGE, CB1 9PT, UNITED KINGDOM,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STEVEN NEVILLE CHEATFIELD</td>
											<td>545 ESKDALE ROAD, WINNERSH TRIANGLE, WOKINGHAM, BERKSHIRE RG41 5TU, UNITED KINGDOM,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 39/02</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB99/00935</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-03-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>9806449.6</td>
									<td>1998-03-25</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216702-bacteria-attenuated-by-a-non-reverting-mutation-in-each-of-the-aroc-ompf-and-ompc-genes-useful-as-vaccines by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:56:28 GMT -->
</html>
