<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216872-methods-for-genomic-analysis-and-a-kit-therefor by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:07:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216872:METHODS FOR GENOMIC ANALYSIS AND A KIT THEREFOR.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS FOR GENOMIC ANALYSIS AND A KIT THEREFOR.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for identifying variations that occur in the human genome and relating these variations to the genetic basis of disease and drug response. In particular, the present invention relates to identifying individual SNPs, determining SNP haplotype blocks and patterns, and, further, using the SNP haplotype blocks and patterns to dissect the genetic bases of disease and drug response. The methods of the present invention are useful in whole genome analysis.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BACKGROUND OF THE INVENTION<br>
The DNA that makes up human chromosomes provides the instructions that direct<br>
the production of all proteins in the body. These proteins carry out the vital functions of<br>
life. Variations in the sequence of DNA encoding a protein produce variations or<br>
mutations in the proteins encoded, thus affecting the normal function of cells. Although<br>
environment often plays a significant role in disease, variations or mutations in the DNA<br>
of an individual are directly related to almost all human diseases, including infectious<br>
disease, cancer, and autoimmune disorders. Moreover, knowledge of genetics,<br>
particularly human genetics, has led to the realization that many diseases result from<br>
either complex interactions of several genes or their products or from any number of<br>
mutations within one gene. For example, Type I and II diabetes have been linked to<br>
multiple genes, each with its own pattern of mutations. In contrast, cystic fibrosis can be<br>
caused by any one of over 300 different mutations in a single gene.<br>
Additionally, knowledge of human genetics has led to a limited understanding of<br>
variations between individuals when it comes to drug response—the field of<br>
pharmocogenetics. Over half a century ago, adverse drug responses were correlated with<br>
amino acid variations in two drug-metabolizing enzymes, plasma cholinesterase and<br>
glucose-6-phosphate dehydrogenase. Since then, careful genetic analyses have linked<br>
sequence polymorphisms (variations) in over 35 drug metabolism enzymes, 25 drug<br>
targets and 5 drug transporters with compromised levels of drug efficacy or safety (Evans<br>
and Relling, Science 296:487-91 (1999)). In the clinic, such information is being used to<br>
prevent drug toxicity; for example, patients are screened routinely for genetic differences<br>
in the thiopurine methyltransferase gene that cause decreased metabolism of 6-<br>
mercaptopurine or azathiopurine. Yet only a small percentage of observed drug toxicities<br>
have been explained adequately by the set of pharmacogenetic markers validated to date.<br>
Even more common than toxicity issues may be cases where drugs demonstrated to be<br>
safe and/or efficacious for some individuals have been found to have either insufficient<br>
therapeutic efficacy or unanticipated side effects in other individuals.<br>
In addition to the importance of understanding the effects of variations in the<br>
genetic make up of humans, understanding the effects of variation in the genetic makeup<br>
of other non-human organisms-particularly pathogens-is important in understanding their<br>
effect on or interaction with humans. For example, the expression of virulence factors by<br>
pathogenic bacteria or viruses greatly affects the rate and severity of infection in humane<br>
that come into contact with such organisms. In addition, a detailed understanding of the<br>
genetic makeup of experimental animals, i.e., mice, rats, etc., is also of great value. For<br>
example, understanding the variations in the genetic makeup of animals used as model<br>
systems for evaluation of therapeutics is important for understanding the test results<br>
obtained using these systems and their predictive value for human use.<br>
Because any two humans are 99.9% similar in their genetic makeup, most of the<br>
sequence of the DNA of their genomes is identical. However, there are variations in<br>
DNA sequence between individuals. For example, there are deletions of many-base<br>
stretches of DNA, insertion of stretches of DNA, variations in the number of repetitive<br>
DNA elements in non-coding regions, and changes in single nitrogenous base positions in<br>
the genome called "single nucleotide polymorphisms" (SNPs). Human DNA sequence<br>
variation accounts for a large fraction of observed differences between individuals,<br>
including susceptibility to disease.<br>
Although most SNPs are rare, it has been estimated that there are 5.3 million<br>
common SNPs, each with a frequency of 10-50%, that account for the bulk of the DNA<br>
sequence difference between humans. Such SNPs are present in the human genome once<br>
every 600 base pairs (Kruglyak and Nickerson, Nature Genet. 27:235 (2001)). Alleles<br>
(variants) making up blocks of such SNPs in close physical proximity are often<br>
correlated, resulting in reduced genetic variability and defining a limited number of "SNP<br>
haplotypes", each of which reflects descent from a single, ancient ancestral chromosome<br>
(Fullerton, et al.,Am. J. Hum. Genet. 67:881 (2000)).<br>
The complexity of local haplotype structure in the human genome~and the<br>
distance over which individual haplotypes extend—is poorly defined. Empiric studies<br>
investigating different segments of the human genome in different populations have<br>
revealed tremendous variability in local haplotype structure. These studies indicate that<br>
the relative contributions of mutation, recombination, selection, population history, and<br>
stochastic events to haplotype structure vary in an unpredictable manner, resulting in<br>
some haplotypes that extend for only a few kilobases (kb), and others that extend for<br>
greater than 100 kb (A. G. Clark et al., Am. J. Hum. Genet. 63:595 (1998)).<br>
These findings suggest that any comprehensive description of the haplotype<br>
structure of the human genome, defined by common SNPs, will require empirical analysis<br>
of a dense set of SNPs in many independent copies of the human genome. Such whole-<br>
genome analyses would provide a fine degree of genetic mapping and pinpoint specific<br>
regions of linkage. Until the present invention, however, the practice and cost of<br>
genotyping over 3,000,000 SNPs across each individual of a reasonably sized population<br>
has made this endeavor impractical. The present invention allows for, among a wide<br>
variety of applications, whole-genome association analysis of populations using SNP<br>
haplotypes.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to methods for identifying variations that occur in<br>
the human genome and relating these variations to the genetic bases of phenotype such as<br>
disease resistance, disease susceptibility or drug response. "Disease" includes but is not<br>
limited to any condition, trait or characteristic of an organism that it is desirable to<br>
change. For example, the condition may be physical, physiological or psychological and<br>
may be symptomatic or asymptomatic. The methods allow for identification of variants,<br>
identification of SNPs, determination of SNP haplotype blocks, determining SNP<br>
haplotype patterns, and further, identification of informative SNPs for each pattern, which<br>
affords genetic data compression.<br>
Thus, one aspect of the present invention provides methods for selecting SNP<br>
haplotype patterns useful in data analysis. Such selection can be accomplished by<br>
isolating substantially identical (homologous) nucleic acid strands from a plurality of<br>
individuals; determining SNP locations in each nucleic acid strand; identifying the SNP<br>
locations in the nucleic acid strands that are linked, where the linked SNP locations form<br>
a SNP haplotype block; identifying isolate SNP haplotype blocks; identifying SNP<br>
haplotype patterns that occur in each SNP haplotype block; and selecting the identified<br>
SNP haplotype patterns that occur in at least two of the substantially identical nucleic acid<br>
strands. In one preferred embodiment, nucleic acid strands from at least about 10<br>
different individuals or origins are used. In a more preferred embodiment, nucleic acid<br>
strands from at least 16 different origins are used. In an even more preferred<br>
embodiment, nucleic acid strands from at least 25 different origins are used, and in a yet<br>
more preferred embodiment, nucleic acid strands from at least 50 different origins are<br>
used. Further, a more preferred embodiment would determine SNP locations in at least<br>
about 100 nucleic acid strands from different origins. In addition, this method may<br>
further comprise selecting the SNP haplotype pattern that occurs most frequently in the<br>
substantially identical nucleic acid strands; selecting the SNP haplotype pattern that<br>
occurs next most frequently in the substantially identical nucleic acid strands; and<br>
repeating the selecting until the selected SNP haplotype patterns identify a portion of<br>
interest of the substantially identical nucleic acid strands. In a preferred embodiment, the<br>
portion of interest is between 70% and 99% of the substantially identical nucleic acid<br>
strands, and, in a more preferred embodiment, the portion of interest is about 80% of the<br>
substantially identical nucleic acid strands. Alternatively, one may wish to limit the<br>
selection of SNP haplotype patterns to no more than about three SNP haplotype patterns<br>
per SNP haplotype block.<br>
In addition, the present invention provides a method for selecting a data set of<br>
SNP haplotype blocks for data analysis, comprising comparing SNP haplotype blocks for<br>
informativeness; selecting a first SNP haplotype block with high informativeness; adding<br>
the first SNP haplotype block to the data set; selecting a second SNP haplotype block<br>
with high informativeness; adding the second selected SNP haplotype block to the data<br>
set; and repeating the selecting and adding steps until the region of interest of a DNA<br>
strand is covered. In preferred embodiments, the SNP haplotype blocks selected are non-<br>
overlapping.<br>
The present invention further provides methods for determining at least one<br>
informative SNP in a SNP haplotype pattern, comprising first determining SNP haplotype<br>
patterns for a SNP haplotype block, then comparing each SNP haplotype pattern of<br>
interest in the SNP haplotype block to the other SNP haplotype patterns of interest in the<br>
SNP haplotype block, and selecting at least one SNP in each SNP haplotype pattern that<br>
distinguishes this SNP haplotype pattern of interest from the other SNP haplotype<br>
patterns of interest in the SNP haplotype block. The selected SNP (or SNPs) is an<br>
informative SNP for the SNP haplotype pattern.<br>
Also, the present invention allows for rapid scanning of genomic regions and<br>
provides a method for determining disease-related genetic loci or pharmacogenomic-<br>
related loci without a priori knowledge of the sequence or location of the disease-related<br>
genetic loci or pharmacogenomic-related loci. This can be done by determining SNP<br>
haplotype patterns from individuals in a control population, then determining SNP<br>
haplotype patterns from individuals in a experimental population, such as individuals in a<br>
diseased population or individuals that react in a particular manner when administered a<br>
drug. The frequencies of the SNP haplotype patterns of the control population are<br>
compared to the frequencies of the SNP haplotype patterns of the experimental<br>
population. Differences in these frequencies indicate locations of disease-related genetic<br>
loci or pharmacogenomic-related loci.<br>
An additional aspect of the present invention provides a method of making<br>
associations between SNP haplotype patterns and a phenotypic trait of interest<br>
comprising: building baseline of SNP haplotype patterns of control individuals by the<br>
methods of the present invention; pooling whole genomic DNA from a clinical population<br>
having a common phenotypic trait of interest; and identifying the SNP haplotype patterns<br>
that are associated with the phenotypic trait of interest. Thus, the present invention<br>
allows for genome scanning to identify multiple haplotype blocks associated with a<br>
phenotype, which is particularly useful when studying polygenic traits.<br>
Also, the present invention provides a method for identifying drug discovery<br>
targets comprising: associating SNP haplotype patterns with a disease; identifying a<br>
chromosomal location of the associated SNP haplotype patterns; determining the nature<br>
of the association of the chromosomal location and said disease; and using the gene or<br>
gene product of the chromosomal location as a drug discovery target.<br><br>
BRIEF DESCRIPTION OF THE ACCOMPANYING GURES<br>
The following figures and drawings form part of the present specification and are<br>
included to further demonstrate certain aspects of the patent invention. The invention<br>
may be better understood by reference to one or more of these drawings in combination<br>
with the detailed description of the specific embodiments presented herein.<br>
Figure 1 is a schematic of one embodiment of the methods of the present<br>
invention from identifying variant locations to associating variants with phenotype, to<br>
using the associations to identify drug discovery targets or as diagnostic markers.<br>
Figure 2 shows sample SNP haplotype blocks and SNP haplotype patterns<br>
according to the present invention.<br>
Figure 3 is a schematic showing one embodiment of a method for selecting SNP<br>
haplotype blocks.<br>
Figure 4 illustrates a simple employment of one embodiment of the method shown<br>
in Figure 3.<br>
Figure 5 A is a schematic of one embodiment of a method for choosing a final set<br>
of SNP haplotype blocks. Figure 5B is a simple employment of the method shown in<br>
Figure 5A. The "letter.number" designations in Figure 5B indicate "haplotype block<br>
ID:informativeness value" for each block.<br>
Figure 6 shows an example of how informative SNPs may be selected according<br>
to one embodiment of the present invention.<br>
Figure 7A is a schematic showing one embodiment for resolving variant<br>
ambiguities and/or SNP haplotype pattern ambiguities. Figure 7B illustrates a simple<br>
employment of the method shown in Figure 7A.<br>
Figure 8 is a schematic of one embodiment of using the methods of the present<br>
invention in an association study.<br>
Figure 9 shows an exemplary computer network system suitable for executing<br>
some embodiments of the present invention.<br>
Figure 10 is a schematic of the construction of somatic cell hybrids.<br>
Figure 11 is a table illustrating a portion of results obtained from screening<br>
hamster-human cell hybrids with the HuSNP genechip from Affymetrix, Inc.<br>
Figure 12 shows an example of various amplified genomic regions of human<br>
chromosome 22 and human chromosome 14 genomic DNA using long range PCR.<br>
Figure 13A is a bar graph showing the percentage of SNPs plotted against the .<br>
frequency of the minor allele (variant) of the SNP. Figure 13B is a graph of the<br>
percentage of 200kb intervals as a function of the nucleotide diversity in the interval.<br>
Figure 13C is a bar graph showing the percentage of all intervals plotted against interval<br>
length.<br>
Figure 14 shows the haplotype patterns for twenty independent globally diverse<br>
chromosomes defined by 147 common human chromosome 21 SNPs.<br>
Figure 15 is a plot of the fraction of chromosome covered as a function of the<br>
number of SNPs required for that coverage.<br>
The present invention relates to methods for identifying variations that occur in<br>
the human genome and relating these variations to the genetic basis of disease and drug<br>
response. In particular, the present invention relates to identifying individual SNPs,<br>
determining SNP haplotype blocks and patterns, and, further, using the SNP haplotype<br>
blocks and patterns to dissect the genetic bases of disease and drug response. The<br>
methods of the present invention are useful in whole genome analysis.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
It readily should be apparent to one skilled in the art that various embodiments<br>
and modifications may be made to the invention disclosed in this application without<br>
departing from the scope and spirit of the invention. All publications mentioned herein<br>
are cited for the purpose of describing and disclosing reagents, methodologies and<br>
concepts that may be used in connection with the present invention. Nothing herein is to<br>
be construed as an admission that these references are prior art in relation to the<br>
inventions described herein.<br>
As used in the specification, "a" or "an" means one or more. As used in the<br>
claim(s), when used in conjunction with the word "comprising", the words "a" or "an"<br>
mean one or more. As used herein, "another" means at least a second or more.<br>
As used herein, when the term "different origins" is used, it refers to the fact DNA<br>
strands from different organisms come from a different origin. Further, each DNA strand<br>
in a single organism"s genome come from different origins. In a diploid organism, an<br>
individual organism"s genome is made up of a set of pairs of substantially identical DNA<br>
strands. That is, a single individual would have substantially identical DNA strands from<br>
two different origins-one DNA strand of the pair is of maternal origin and one DNA<br>
strand of the pair is of paternal origin. Two or more nucleic acid sequences--for example,<br>
two or more DNA strands--are considered to be substantially identical if they exhibit at<br>
least about 70% sequence identity at the nucleotide level, preferably about 75%, more<br>
preferably about 80%, still more preferably about 85%, yet more preferably about 90%,<br>
even more preferably about 95% and even more preferably nucleic acid sequences are<br>
considered to be substantially identical if they exhibit at least about 98% sequence<br>
identity at the nucleotide level. The extent of sequence identity that is relevant between<br>
two or more nucleic acid sequences will depend on the host source of the nucleic acids.<br>
For example, a greater than 95% sequence identity may be relevant when looking at same<br>
species comparisons, whereas a sequence identity of 70% or even less may be relevant<br>
when making cross species comparisons. Of course, when one refers to DNA herein such<br>
reference may include derivatives of DNA such as amplicons, RNA transcripts, nucleic<br>
acid mimetics, etc.<br>
As used herein, "individual" refers to a specific single organism, such as a single<br>
animal, human insect, bacterium, etc.<br>
As used herein, "informativeness" of a SNP haplotype block is defined as the<br>
degree to which a SNP haplotype block provides information about genetic regions.<br>
As used herein, the term "informative SNP" refers to a genetic variant such as a<br>
SNP or subset (more than one) of SNPs that tends to distinguish one SNP haplotype<br>
pattern from other SNP haplotype patterns within a SNP haplotype block.<br>
As used herein, the term "isolate SNP block" refers to a SNP haplotype block that<br>
consists of one SNP.<br>
As used herein, the term "linkage disequilibrium", "linked" or "LD" refers to<br>
genetic loci that tend to be transmitted from generation to generation together; e.g.,<br>
genetic loci that are inherited non-randomly.<br>
As used herein, the term "singleton SNP haplotype" or "singleton SNP" refers to a<br>
specific SNP allele or variant that occurs in less than a certain portion of the population.<br>
As used herein, the term "SNP" or "single nucleotide polymorphism" refers to a<br>
genetic variation between individuals; e.g., a single nitrogenous base position in the DNA<br>
of organisms that is variable. As used herein, "SNPs" is the plural of SNP. Of course,<br>
when one refers to DNA herein such reference may include derivatives of DNA such as<br>
amplicons, RNA transcripts, etc.<br>
As used herein, the term "SNP haplotype block" means a group of variant or SNP<br>
locations that do not appear recombine independently and that can be grouped together in<br>
blocks of variants or SNPs:<br>
As used herein, the term "SNP haplotype pattern" refers to the set of genotypes<br>
for SNPs in a SNP haplotype block in a single DNA strand.<br>
As used herein, the term "SNP location" is the site in a DNA sequence where a<br>
SNP occurs.<br>
As used herein a "SNP haplotype sequence" is a DNA sequence in a DNA strand<br>
that contains at least one SNP location.<br>
Preparation of Nucleic Acids for Analysis<br>
Nucleic acid molecules may be prepared for analysis using any technique known<br>
to those skilled in the art. Preferably such techniques result in the production of a nucleic<br>
acid molecule sufficiently pure to determine the presence or absence of one or more<br>
variations at one or more locations in the nucleic acid molecule. Such techniques may be<br>
found, for example, in Sambrook, et al., Molecular Cloning: A Laboratory Manual (Cold<br>
Spring Harbor Laboratory, New York) (1989), and Ausubel, et al., Current Protocols in<br>
Molecular Biology (John Wiley and Sons, New York) (1997), incorporated herein by<br>
reference.<br>
When the nucleic acid of interest is present in a cell, it may be necessary to first<br>
prepare an extract of the cell and then perform further steps-i.e., differential precipitation,<br>
column chromatography, extraction with organic solvents and the like-in order to obtain<br>
a sufficiently pure preparation of nucleic acid. Extracts may be prepared using standard<br>
techniques in the art, for example, by chemical or mechanical lysis of the cell. Extracts<br>
then may be further treated, for example, by filtration and/or centrifugation and/or with<br>
chaotropic salts such as guanidinium isothiocyanate or urea or with organic solvents such<br>
as phenol and/or HCCl3 to denature any contaminating and potentially interfering<br>
proteins. When chaotropic salts are used, it may be desirable to remove the salts from the<br>
nucleic acid-containing sample. This can be accomplished using standard techniques in<br>
the art such as precipitation, filtration, size exclusion chromatography and the like.<br>
In some instances, it may be desirable to extract and separate messenger RNA<br>
from cells. Techniques and material for this purpose are known to those skilled in the art<br>
and may involve the use of oligo dT attached to a solid support such as a bead or plastic<br>
surface. Suitable conditions and materials are known to those skilled in the art and may<br>
be found in the Sambrook and Ausubel references cited above. It may be desirable to<br>
reverse transcribe the mRNA into cDNA using, for example, a reverse transcriptase<br>
enzyme. Suitable enzymes are commercially available from, for example, Invitrogen,<br>
Carlsbad CA. Optionally, cDNA prepared from mRNA may then be amplified.<br>
One approach particularly suitable for examining haplotype patterns and blocks is<br>
using somatic cell genetics to separate chromosomes from a diploid state to a haploid<br>
state. In one embodiment, a human lymphoblastoid cell line that is diploid may be fused<br>
to a hamster fibroblast cell line that is also diploid such that the human chromosomes ate<br>
introduced into the hamster cells to produce cell hybrids. The resulting cell hybrids are<br>
examined to determine which human chromosomes were transferred, and which, if any,<br>
of the transferred human chromosomes are in a haploid state (see, e.g., Patterson, et al.,<br>
Annal. N.Y. Acad. Of Sciences, 396:69-81 (1982)).<br>
A schematic of the procedure is shown in Figure 10. Figure 10 shows a diploid<br>
human lymphoblastoid cell line that is wildtype for the thymidine kinase gene being fused<br>
to a diploid hamster fibroblast cell line containing a mutation in the thymidine kinase<br>
gene. In a sub-population of the resulting cells, human chromosomes are present in<br>
hybrids. Selection for the human DNA-containing hybrid cells is achieved by utilizing<br>
HAT medium (selective medium). Only hybrid cells that have a stably-incorporated<br>
human DNA strand having the wildtype human thymidine kinase gene grow in cell<br>
culture medium containing HAT. Of the resulting hybrids, some hybrids may contain<br>
both copies of some human chromosomes, only one copy of a human chromosome or no<br>
copies of a particular human chromosome. For example, for a human chromosome 22<br>
having a locus with either an A or a B allele, the resulting hybrid cells may contain one<br>
human chromosome 22 variant (e.g., the "A" variant) or a portion thereof, some may<br>
contain the other human chromosome 22 variant (the "B" variant) or a portion thereof,<br>
some may contain both human chromosome 22 variants or portions thereof, and some<br>
hybrids may not contain any portion of a human chromosome 22 at all. In Figure 10,<br>
only two of the resulting hybrid populations are shown. Once the appropriate hybrids are<br>
selected, the nucleic acids from these hybrids may be isolated by, for example, the<br>
techniques described above and then subjected to SNP discovery, and haplotype block<br>
and pattern analyses of the present invention.<br>
Amplification Techniques<br>
It may be desirable to amplify one or more nucleic acids of interest before<br>
determining the presence or absence of one or more variations in the nucleic acid.<br>
Nucleic acid amplification increases the number of copies of the nucleic acid sequence of<br>
interest. Any amplification technique known to those of skill in the art may be used in<br>
conjunction with the present invention including, but not limited to, polymerase chain<br>
reaction (PCR) techniques. PCR may be carried out using materials and methods known<br>
to those of skill in the art.<br>
PCR amplification generally involves the use of one strand of a nucleic acid<br>
sequence as a template for producing a large number of complements to that sequence.<br>
The template may be hybridized to a primer having a sequence complementary to a<br>
portion of the template sequence and contacted with a suitable reaction mixture including<br>
dNTPs and a polymerase enzyme. The primer is elongated by the polymerase enzyme<br>
producing a nucleic acid complementary to the original template.<br>
For the amplification of both strands of a double stranded nucleic acid molecule,<br>
two primers may be used, each of which may have a sequence which is complementary to<br>
a portion of one of the nucleic acid strands. Elongation of the primers with a polymerase<br>
enzyme results in the production of two double-stranded nucleic acid molecules each of<br>
which contains a template strand and a newly synthesized complementary strand. The<br>
sequences of the primers typically are chosen such that extension of each of the primers<br>
results in elongation toward the site in the nucleic acid molecule where the other primer<br>
hybridizes.<br>
The strands of the nucleic acid molecules are denatured-for example, by heating-<br>
and the process is repeated, this time with the newly synthesized strands of the preceding<br>
step serving as templates in the subsequent steps. A PCR amplification protocol may<br>
involve a few to many cycles of denaturation, hybridization and elongation reactions to<br>
produce sufficient amounts of the desired nucleic acid.<br>
Although PCR methods typically employ heat to achieve strand denaturation and<br>
allow subsequent hybridization of the primers, any other means that results in making the<br>
nucleic acids available for hybridization to the primers may be used. Such techniques<br>
include, but are not limited to, physical, chemical, or enzymatic means, for example, by<br>
inclusion of a helicase, (see Radding, Ann. Rev. Genetics 16: 405-436 (1982)) or by<br>
electrochemical means (see PCT Application Nos. WO 92/04470 and WO 95/25177).<br>
Template-dependent extension of primers in PCR is catalyzed by a polymerase<br>
enzyme in the presence of at least 4 deoxyribonucleotide triphosphates (typically selected<br>
from dATP, dGTP, dCTP, dUTP and dTTP) in a reaction medium which comprises the<br>
appropriate salts, metal cations, and pH buffering system. Suitable polymerase enzymes<br>
are known to those of skill in the art and may be cloned or isolated from natural sources<br>
and may be native or mutated forms of the enzymes. So long as the enzymes retain the<br>
ability to extend the primers, they may be used in the amplification reactions of the<br>
present invention.<br>
The nucleic acids used in the methods of the invention may be labeled to facilitate<br>
detection in subsequent steps. Labeling may be carried out during an amplification<br>
reaction by incorporating one or more labeled nucleotide triphosphates and/or one or<br>
more labeled primers into the amplified sequence. The nucleic acids may be labeled<br>
following amplification, for example, by covalent attachment of one or more detectable<br>
groups. Any detectable group known to those skilled in the art may be used, for example,<br>
fluorescent groups, ligands and/or radioactive groups. An example of a suitable labeling<br>
technique is to incorporate nucleotides containing labels into the nucleic acid of interest<br>
using a terminal deoxynucleotidyl transferase (TdT) enzyme. For example, a nucleotide-<br>
preferably a dideoxy nucleotide-containing a label is incubated with the nucleic acid to<br>
be labeled and a sufficient amount of TdT to incorporate the nucleotide. A preferred<br>
nucleotide is a dideoxynucleotide-i.e., ddATP, ddGTP, ddCTP, ddTTP, etc-having a<br>
biotin label attached.<br>
Techniques to optimize the amplification of long sequences may be used. Such<br>
techniques work well on genomic sequences. The methods disclosed in pending US<br>
patent applications USSN 60/317,311, filed 9/5/01; USSN [unassigned], attorney docket<br>
number 1011N-1, filed 01/09/02 entitled "Algorithms for Selection of Primer Pairs"; and<br>
USSN [assigned], attorney docket number 1011N1D1, filed 01/09/02, entitled "Methods<br>
for Amplification of Nucleic Acids" are particularly suitable for amplifying genomic<br>
DNA for use in the methods of the present invention.<br>
Amplified sequences may be subjected to other post amplification treatments<br>
either before or after labeling. For example, in some cases, it may be desirable to<br>
fragment the amplified sequence prior to hybridization with an oligonucleotide array.<br>
Fragmentation of the nucleic acids generally may be carried out by physical, chemical or<br>
enzymatic methods that are known in the art. Suitable techniques include, but are not<br>
limited to, subjecting the amplified nucleic acids to shear forces by forcing the nucleic<br>
acid containing fluid sample through a narrow aperture or digesting the PCR product with<br>
a nuclease enzyme. One example of a suitable nuclease enzyme is Dnase I. After<br>
amplification, the PCR product may be incubated in the presence of a nuclease for a<br>
period of time designed to produce appropriately sized fragments. The sizes of the<br>
fragments may be varied as desired, for example, by increasing the amount of nuclease or<br>
duration of incubation to produce smaller fragments or by decreasing the amount of<br>
nuclease or period of incubation to produce larger fragments. Adjusting the digestion<br>
conditions to produce fragments of the desired size is within the capabilities of a person<br>
of ordinary skill in the art. The fragments thus produced may be labeled as described<br>
above.<br>
Methods for the Detection of SNPs (SNP Discovery)<br>
Determination of the presence or absence of one or more variations in a nucleic<br>
acid may be made using any technique known to those of skill in the art. Any technique<br>
that permits the accurate determination of a variation can be used. Preferred techniques<br>
will permit rapid, accurate determination of multiple variations with a minimum of<br>
sample handling required. Some examples of suitable techniques are provided below.<br>
Several methods for DNA sequencing are well known and generally available in<br>
the art and may be used to determine the location of SNPs in a genome. See, for<br>
example, Sambrook, et al, Molecular Cloning: A Laboratory Manual (Cold Spring<br>
Harbor Laboratory, New York) (1989), and Ausubel, et al., Current Protocols in<br>
Molecular Biology (John Wiley and Sons, New York) (1997), incorporated herein by<br>
reference. Such methods may be used to determine the sequence of the same genomic<br>
regions from different DNA strands where the sequences are then compared and the<br>
differences (variations between the strands) are noted. DNA sequencing methods may<br>
employ such enzymes as the Klenow fragment of DNA polymerase I, Sequenase (US<br>
Biochemical Corp, Cleveland, Ohio.), Taq polymerase (Perkin Elmer), thermostable T7<br>
polymerase (Amersham, Chicago, 111.), or combinations ef polymerases and proofreading<br>
exonucleases such as those found in the Elongase Amplification System marketed by<br>
Gibco/BRL (Gaithersburg, Md.). Preferably, the process is automated with machines<br>
such as the Hamilton Micro Lab 2200 (Hamilton, Reno, Nev.), Peltier Thermal Cycler<br>
(PTC200; MJ Research, Watertown, Mass.) and the ABI Catalyst and 373 and 377 DNA<br>
Sequencers (Perkin Elmer, Wellesley, MA).<br>
In addition, capillary electrophoresis systems which are commercially available<br>
may be used to perform variation or SNP analysis. In particular, capillary sequencing<br>
may employ flowable polymers for electrophoretic separation, four different fluorescent<br>
dyes (one for each nucleotide) which are laser activated, and detection of the emitted<br>
wavelengths by a charge coupled device camera. Output/light intensity may be converted<br>
to electrical signal using appropriate software (e.g. Genotyper and Sequence Naviagator,<br>
Perkin Elmer, Wellesley, MA) and the entire process from loading of samples to<br>
computer analysis and electronic data display may be computer controlled. Again, this<br>
method may be used to determine the sequence of the same genomic regions from<br>
different DNA strands where the sequences are then compared and the differences<br>
(variations between the strands) are noted.<br>
Optionally, once a genomic sequence from one reference DNA strand has been<br>
determined by sequencing, it is possible to use hybridization techniques to determine<br>
variations in sequence between the reference strand and other DNA strands. These<br>
variations may be SNPs. An example of a suitable hybridization technique involves the<br>
use of DNA chips (oligonucleotide arrays), for example, those available from Affymetrix,<br>
Inc. Santa Clara, CA. For details on the use of DNA chips for the detection of, for<br>
example, SNPs, see United States Patent No. 6,300,063 issued to Lipshultz, et al., and<br>
United States Patent No. 5,837,832 to Chee, et al., HuSNP Mapping Assay, reagent kit<br>
and user manual, Affymetrix Part No. 90094 (Affymetrix, Santa Clara, CA), all<br>
incorporated by reference herein.<br>
In preferred embodiments, more than 10,000 bases of a reference sequence and<br>
the other DNA strands are scanned for variants. In more preferred embodiments, more<br>
than 1x106 bases of a reference sequence and the other DNA strands are scanned for<br>
variants, even more preferably more than 2x106 bases of a reference sequence and the<br>
other DNA strands are scanned, even more preferably 1x107 bases are scanned, and more<br>
preferably more than 1x108 bases are scanned, and more preferably more than 1x109<br>
bases of a reference sequence and the other DNA strands are scanned for variants. In<br>
preferred embodiments at least exons are scanned for variants, and in more preferred<br>
embodiments both introns and exons are scanned for variants. In an even more preferred<br><br>
embodiment, introns, exons and intergenic sequences are scanned for variants. In<br>
preferred embodiments the scanned nucleic acids are genomic DNA, including both<br>
coding and noncoding regions. In most preferred embodiments, such DNA is from a<br>
mammalian organism such as a human. In preferred embodiments, more than 10% of the<br>
genomic DNA from the organism is scanned, in more preferred embodiments more than<br>
25% of the genomic DNA from the organism is scanned, in more preferred embodiments,<br>
more than 50% of the genomic DNA from the organism is scanned, and in most preferred<br>
embodiments, more than 75% of the genomic DNA is scanned. In some embodiments of<br>
the present invention, known repetitive regions of the genome are not scanned, and do not<br>
count toward the percentage of genomic DNA scanned. Such known repetitive regions<br>
may include Single Interspersed Nuclear Elements (SINEs, such as alu and MIR<br>
sequences), Long Interspersed Nuclear Elements (LINEs, such as LINE1 and LINE2<br>
sequences), Long Terminal Repeats (LTRs such as MaLRs, Retrov and MER4<br>
sequences), transposons, and MER1 And MER2 sequences.<br>
Briefly, in one embodiment, labeled nucleic acids in a suitable solution are<br>
denatured-for example, by heating to 95 °C-and the solution containing the denatured<br>
nucleic acids is incubated with a DNA chip. After incubation, the solution is removed,<br>
the chip may be washed with a suitable washing solution to remove un-hybridized nucleic<br>
acids, and the presence of hybridized nucleic acids on the chip is detected. The<br>
stringency of the wash conditions may be adjusted as necessary to produce a stable signal.<br>
Detecting the hybridized nucleic acids may be done directly, for example, if the nucleic<br>
acids contain a fluorescent reporter group, fluorescence may be directly detected. If the<br>
label on the nucleic acids is not directly detectable, for example, biotin, then a solution<br>
containing a detectable label, for example, streptavidin coupled to phycoerythrin, may be<br>
added prior to detection. Other reagents designed to enhance the signal level may also be<br>
added prior to detection, for example, a biotinylated antibody specific for streptavidin<br>
may be used in conjunction with the biotin, streptavidin-phycoerythrin detection system.<br>
In some embodiments, the oligonucleotide arrays used in the methods of the present<br>
invention contain at least 1 x 106 probes per array. In a preferred embodiment, the<br>
oligonucleotide arrays used in the methods of the present invention contain at least 10 x<br>
106 probes per array. In a more preferred embodiment, the oligonucleotide arrays used in<br>
the methods of the present invention contain at least 50 x 106 probes per array.<br>
Once variant locations have been determined (SNP discovery) by using, for<br>
example, sequencing or microarray analysis, it is necessary to genotype the SNPs of<br>
control and sample populations. The hybridization methods just described work well for<br>
this purpose, providing an accurate and rapid technique for detecting and genotyping<br>
SNPs in multiple samples. In addition, a technique suitable for the detection of SNPs in<br>
genomic DNA-without amplification-is the Invader technology available from Third<br>
Wave Technologies, Inc., Madison, WI. Use of this technology to detect SNPs may be<br>
found, e.g., in Hessner, et al, Clinical Chemistry 46(8):1051-56 (2000); Hall, et al,<br>
PNAS 97(15):8272-77 (2000); Agarwal, et al.,Diag. Molec. Path. 9(3):158-64 (2000);<br>
and Cooksey, et al, Antimicrobial and Chemotherapy 44(5): 1296-1301 (2000). In the<br>
Invader process, two short DNA probes hybridize to a target nucleic acid to form a<br>
structure recognized by a nuclease enzyme. For SNP analysis, two separate reactions are<br>
run—one for each SNP variant. If one of the probes is complementary to the sequence,<br>
the nuclease will cleave it to release a short DNA fragment termed a "flap". The flap<br>
binds to a fluorescently-labeled probe and forms another structure recognized by a<br>
nuclease enzyme. When the enzyme cleaves the labeled probe, the probe emits a<br>
detectable fluorescence signal thereby indicating which SNP variant is present.<br>
An alternative to Invader technology, rolling circle amplification utilizes an<br>
oligonucleotide complementary to a circular DNA template to produce an amplified<br>
signal (see, for example, Lizardi, et al., Nature Genetics 19(3):225-32 (1998); and Zhong,<br>
et al, PNAS 98(7):3940-45 (2001)). Extension of the oligonucleotide results in the<br>
production of multiple copies of the circular template in a long concatemer. Typically,<br>
detectable labels are incorporated into the extended oligonucleotide during the extension<br>
reaction. The extension reaction can be allowed to proceed until a detectable amount of<br>
. extension product is synthesized.<br>
In order to detect SNPs using rolling circle amplification, three probes and two<br>
circular DNA templates may be used. The first probe-the target specific probe-may be<br>
constructed to be complementary to a target nucleic acid molecule such that the 5"-<br>
terminus of the probe hybridizes to the nucleotide immediately adjacent 5" to the SNP site<br>
in the target nucleic acid. The site of the SNP is not base paired to the first probe.<br>
The other two probes-rolling circle probes-are constructed to have two 3"-<br>
terminals. This can be accomplished in various ways, for example, by introducing a 5"-5"<br>
linkage in the central portion of the probes resulting in a reversal of polarity of the<br>
nucleotide sequence at that point. One end of each of the probes has a sequence that is<br>
complementary to a portion of a different circular template molecule while the other end<br>
is complementary to a portion of the target nucleic acid sequence. The target-sequence-<br>
complementary terminal is constructed such that the 3"-most nucleotide aligns with the<br>
nucleotide at the SNP site. One of the probes may contain a nucleotide complementary to<br>
the nucleotide at the SNP site in the target nucleic acid while the other contains a<br>
nucleotide that is not complementary. In the instance where two or more variants of the<br>
SNP are present in the population, probes may be constructed to have 3"-nucleotides<br>
complementary to the variants to be detected.<br>
The probes-both target specific and rolling circle-may be hybridized to the target<br>
sequence and contacted with a ligase enzyme. When the 3"-most nucleotide of the rolling<br>
circle probe forms a base pair with the nucleotide at the SNP site, the two probes-the<br>
target specific and the rolling circle-are efficiently ligated together. When the 3"-most<br>
nucleotide of the rolling circle probe is not capable of base pairing with the nucleotide at<br>
the SNP site in the target, the probes are not ligated. The unligated probe is washed away<br>
and the sample is contacted with the template circles, polymerase and labeled nucleoside<br>
triphosphates.<br>
Another technique suitable for the detection of SNPs makes use of the 5"-<br>
exonuclease activity of a DNA polymerase to generate a signal by digesting a probe<br>
molecule to release a fluorescently labeled nucleotide. This assay is frequently referred to<br>
as a Taqman assay (see, e.g., Arnold, et al, BioTechniques 25(l):98-106 (1998); and<br>
Becker, et al., Hum. Gene Ther. 10:2559-66 (1999)). A target DNA containing a SNP is<br>
amplified in the presence of a probe molecule that hybridizes to the SNP site. The probe<br>
molecule contains both a fluorescent reporter-labeled nucleotide at the 5"-end and a<br>
quencher-labeled nucleotide at the 3"-end. The probe sequence is selected so that the<br>
nucleotide in the probe that aligns with the SNP site in the target DNA is as near as<br>
possible to the center of the probe to maximize the difference in melting temperature<br>
between the correct match probe and the mismatch probe. As the PCR reaction is<br>
conducted, the correct match probe hybridizes to the SNP site in the target DNA and is<br>
digested by the Taq polymerase used in the PCR assay. This digestion results in<br>
physically separating the fluorescent labeled nucleotide from the quencher with a<br>
concomitant increase in fluorescence. The mismatch probe does not remain hybridized<br>
during the elongation portion of the PCR reaction and is, therefore, not digested and the<br>
fluorescently labeled nucleotide remains quenched.<br>
Denaturing HPLC using a polystyrene-divinylbenzene reverse phase column and<br>
an ion-pairing mobile phase can be used to identify SNPs. A DNA segment containing a<br>
SNP is PCR amplified. After amplification, the PCR product is denatured by heating and<br>
mixed with a second denatured PCR product with a known nucleotide at the SNP<br>
position. The PCR products are annealed and are analyzed by HPLC at elevated<br>
temperature. The temperature is chosen to denature duplex molecules that are<br>
mismatched at the SNP location but not to denature those that are perfect matches. Under<br>
these conditions, heteroduplex molecules typically elute before homoduplex molecules.<br>
For an example of the use of this technique see Kota, et al., Genome 44(4):523-28 (2001).<br>
SNPs can be detected using solid phase amplification and microsequencing of the<br>
amplification product. Beads to which primers have been covalently attached are used to<br>
carry out amplification reactions. The primers are designed to include a recognition site<br>
for a Type II restriction enzyme. After amplification-which results in a PCR product<br>
attached to the bead-the product is digested with the restriction enzyme. Cleavage of the<br>
product with the restriction enzyme results in the production of a single stranded portipn<br>
including the SNP site and a 3"-OH that can be extended to fill in the single stranded<br>
portion. Inclusion of ddNTPs in an extension reaction allows direct sequencing of the<br>
product. For an example of the use of this technique to identify SNPs see Shapero, et al,<br>
Genome Research 11(11):1926-34 (2001).<br>
Data Analysis<br>
Figure 1 is a schematic showing the steps of one embodiment of the methods of<br>
the present invention. Once SNPs (variants) have been located or discovered by, e.g., the<br>
methods described supra (step 110 of Figure 1), SNP haplotype blocks, SNP haplotype<br>
patterns within each SNP haplotype block, and informative SNPs for the SNP haplotype<br>
patterns may be determined. One may use all SNPs or variants located; alternatively,<br>
one may focus the analysis on only a portion of the SNPs located. For example, the set of<br>
SNPs analyzed may exclude transition SNPs of the form Cg Tg or cG  cA. In<br>
addition, in one embodiment of the present invention, the focus is on common SNPs.<br>
Common SNPs are those SNPs whose less common form is present at a minimum<br>
frequency in a given population. For example, common SNPs are those SNPs that are<br>
found in at least about 2% to 25% of the population. In a preferred embodiment, common<br>
SNPs are those SNPs that are found in at least about 5% to 15% of the population. In a<br>
more preferred embodiment, common SNPs are those that are found in at least about 10%<br>
of the population. Common SNPs likely result from mutations that occurred early in the<br>
evolution of humans. Focusing on common SNPs minimizes systematic allele or variant<br>
differences between control and experimental populations that appear as disease or drug-<br>
response associated, yet result only from migratory history or mating practices; i.e.,<br>
focusing on common SNPs decreases the false positives that result from recent population<br>
anomalies. Moreover, common SNPs are relevant to a larger proportion of the human<br>
population, making the present invention more broadly applicable to disease and drug<br>
response studies. Along the same line, SNPs in which an variant is observed only once<br>
may be eliminated from analysis in some embodiments of the present invention (for<br>
example, singleton SNPs). However, certain analyses may be performed including some<br>
or all of these singleton SNPs, particularly when looking at specific sub-populations or<br>
populations that have been influenced by migratory practices and the like.<br>
In step 120 of Figure 1, the variants or SNPs of interest are assigned to haplotype<br>
blocks for evaluation. Variants or SNPs from a whole genome or chromosome may be<br>
analyzed and assigned to SNP haplotype blocks. Alternatively, variants from only a<br>
focused genomic region specific to some disease or drug response mechanism may be<br>
assigned to the SNP haplotype blocks.<br>
Figure 2 provides one illustration of showing how variants, usually SNPs, occur in<br>
haplotype blocks in a genome, and that more than one haplotype pattern can occur within<br>
each haplotype block. If SNP haplotype patterns were completely random, it would be<br>
expected that the number of possible SNP haplotype patterns observed for a SNP<br>
haplotype block of N SNPs would be 2N. However, it was observed in performing the<br>
methods of the present invention that the number of SNP haplotype patterns in each SNP<br>
haplotype block is smaller than 2N because the SNPs are linked (not 4N, as the variants<br>
will most commonly be biallelic, i.e., occur in only one of two forms, not all four<br>
nucleotide base possibilities). Certain SNP haplotype patterns were observed at a much<br>
higher frequency than would be expected in a non-linkage case. Thus, SNP haplotype<br>
blocks are chromosomal regions that tend to be inherited as a unit, with a relatively small<br>
number of common patterns. Each line in Fig. 2 represents portions of the haploid<br>
genome sequence of different individuals. As shown therein, individual W has an "A" at<br>
position 241, a "G" at position 242, and an "A" at position 243. Individual X has the<br>
same bases at positions 241, 242, and 243. Conversely, individual Y has a T at positions<br>
241 and 243, but an A at position 242. Individual Z has the same bases as individual Y at<br>
positions 241, 242, and 243. Variants in block 261 will tend to occur together. Similarly,<br>
the variants in block 262 will tend to occur together, as will those variants in block 263.<br>
Of course, only a few bases in a genome are shown in Figure 2. In fact, most bases will<br>
be like those at position 245 and 248, and will not vary from individual to individual.<br>
The assignment of SNPs to SNP haplotype blocks, step 120 of Figure 1, is, in one<br>
case, an iterative process involving the construction of SNP haplotype blocks from the<br>
SNP locations along a genomic region of interest. In one embodiment, once the initial<br>
SNP haplotype blocks are constructed, SNP haplotype patterns present in the constructed<br>
SNP haplotype blocks are determined (step 130 of Figure 1). In some specific<br>
embodiments, the number of SNP haplotype patterns selected per SNP haplotype block in<br>
step 130 is no greater than about five. In another specific embodiment, the number of<br>
SNP haplotype patterns selected per SNP haplotype block is equal to the number of SNP<br>
haplotype patterns necessary to identify SNP haplotype patterns in greater than 50% of<br>
the DNA strands being analyzed. In other words, enough SNP haplotype patterns are<br>
selected, for example, four patterns per block are selected, such that at least half of the<br>
DNA strands analyzed will have a SNP haplotype pattern that matches one of the four<br>
patterns selected in each SNP haplotype block. In a preferred embodiment, the number of<br>
SNP haplotype patterns selected per SNP haplotype.block is equal to the number of SNP<br>
haplotype patterns necessary to identify SNP haplotype patterns in greater than 70% of<br>
the DNA strands being analyzed. In one preferred embodiment, the number of SNP<br>
haplotype patterns selected per SNP haplotype block is equal to the number of SNP<br>
haplotype patterns necessary to identify SNP haplotype patterns in greater than 80% of<br>
the DNA strands being analyzed. In addition, in some embodiments of the present<br>
invention, SNP haplotype patterns that occur in less than a certain portion of DNA strands<br>
being analyzed are eliminated from analysis. For example, in one embodiment, if ten<br>
DNA strands are being analyzed, SNP haplotype patterns that are found to occur in only<br>
one sample out of ten are eliminated from analysis.<br>
Once the SNP haplotype patterns of interest are selected, informative SNPs for<br>
these SNP haplotype patterns are determined (step 140 of Figure 1). From this initial set<br>
of blocks, a set of candidate SNP blocks that fit certain criteria for informativeness is<br>
constructed (step 150 of Figure 1). Figures 4 and 5 illustrate steps 120, 130, 140 and 150<br>
in more detail.<br>
In Figure 3, step 310 provides that a new block of SNPs is chosen for evaluation.<br>
In one embodiment, the first block chosen contains only the first SNP in a SNP haplotype<br>
sequence; thus at step 320, the first, single, SNP is added to the block. At step 330,<br>
informativeness of this block is determined.<br>
"Informativeness" of a SNP haplotype block is defined in one embodiment as the<br>
degree to which the block provides information about genetic regions. For example, in<br>
one embodiment of the present invention, informativeness could be calculated as the ratio<br>
of the number of SNP locations in a SNP haplotype block divided by the number of SNPs<br>
required to distinguish each SNP haplotype pattern under consideration from other SNP<br>
haplotype patterns under consideration (number of informative SNPs) in that block.<br>
Another measure of informativeness might be the number of informative SNPs in the<br>
block. One skilled in the art recognizes that informativeness may be determined in any<br>
number of ways.<br>
Referring again to Figure 2, SNP haplotype block 261 contains three SNPs and<br>
two SNP haplotype patterns (AGA and TAT). Any one of the three SNPs present can be<br>
used to tell the patterns apart; thus, any one of these SNPs can be chosen to be the<br>
informative SNP for this SNP haplotype pattern. For example, if it is determined that a<br>
sample nucleic acid contains a T at the first position, the same sample will contain an A at<br>
the second position and a T at the third position. If it is determined in a second sample<br>
that the SNP in the second position is a G, the first and third SNPs will be A"s. Thus, by<br>
one measure of informativeness, the informativeness value for this first block is 3: 3 total<br>
SNPs divided by 1 informative SNP needed to distinguish the patterns from each other.<br>
Similarly, SNP haplotype block 262 contains three SNPs (two positions do not have<br>
variants) and two haplotype patterns (TCG and CAC). As with the previously-analyzed<br>
block, any one of the three SNPs can be evaluated to tell one pattern from the other; thus,<br>
the informativeness of this block is 3: 3 total SNPs divided by 1 informative SNP needed<br>
to distinguish the patterns. SNP haplotype block 263 contains five SNPs and two SNP<br>
patterns (TAACG and ATCAC). Again, any one of the five SNPs can be used to tell one<br>
pattern from the other; thus, the informativeness of this block is 5: 5 total SNPs divided<br>
by 1 informative SNP needed to distinguish the patterns.<br>
Figure 2 provides a simple example of genetic analysis. When several SNP<br>
haplotype patterns are present in a block, it may be necessary to use more than one SNP<br>
as informative SNPs. For example, in a case where a block contains, for example, six<br>
SNPs and two SNPs are needed to distinguish the patterns of interest, the informativeness<br>
of the block is 3: 6 total SNPs divided by 2 SNPs needed to distinguish the patterns.<br>
Generally speaking, as many as 2N distinct SNP haplotype patterns can be distinguished<br>
by using the genotypes of N suitably selected SNPs. Therefore, if there exist only two<br>
SNP haplotype patterns in the SNP haplotype block, a single SNP should be able to<br>
differentiate between the two. If there are three or four patterns, at least two SNPs would<br>
likely be required, etc.<br>
In step 340 of Figure 3, once the informativeness of a SNP haplotype block is<br>
determined, a test is performed. The test essentially evaluates the SNP haplotype blocks<br>
based on selected criteria (for example, whether a block meets a threshold measure of<br>
informativeness), and the result of the test determines whether, for example, another SNP<br>
will be added to the block for analysis or whether the analysis will proceed with a new<br>
block starting at a different SNP location. Figure 4 illustrates one embodiment of this<br>
process.<br>
In Figure 4, assume there is a DNA sequence with six SNP locations. The<br>
analysis of SNP haplotype blocks described above might be performed in the following<br>
manner: SNP haplotype block A is selected containing only the SNP at SNP position 1<br>
(steps 310 and 320 of Figure 3). The informativeness of this block is calculated (step<br>
330), and it is determined whether the informativeness of this block meets a threshold<br>
measure of informativeness (step 340). In this case, it "passes" and two things happen.<br>
First, this block of one SNP (SNP position 1) is added to the set of candidate SNP<br>
haplotype blocks (step 350). Second, another SNP (here, SNP position 2) is added to this <br>
block (step 320) to create a new block, B, containing SNP positions 1 and 2, which is then<br>
analyzed. In this illustration block B also meets the threshold measure of informativeness<br>
(step 340), so it would be added to the set of candidate SNP haplotype blocks (step 350),<br>
and another SNP (here, SNP position 3) is added to this block (step 320) to create new<br>
block C, containing SNP positions 1, 2 and 3, which is then analyzed. In this illustration,<br>
C also meets the threshold measure of informativeness and it is added to the set of<br>
candidate SNP haplotype blocks (step 350), and another SNP (here, SNP position 4) is<br>
added to this block (step 320) to create new block D, containing SNP positions 1, 2, 3,<br>
and 4, which is then analyzed. In the Figure 4 illustration, SNP block D does not meet<br>
the threshold measure of informativeness. SNP block D is not added to the set of<br>
candidate SNP haplotype blocks (step 350), nor does another SNP get added to block D<br>
for analysis. Instead, a new SNP location is selected for a round of SNP block<br>
evaluations.<br>
In Figure 4, after block D fails to meet the threshold measure of informativeness, a<br>
new block, E, is selected that contains only the SNP at position 2. Block E is evaluated<br>
for informativeness, is found to meet the threshold measure, is added to the set of<br>
candidate SNP haplotype blocks (step 350), and another SNP (here, SNP position 3) is<br>
added to this block (step 320) to create new block F, containing SNP positions 2 and 3,<br>
which is then analyzed, and so on. Note that block H fails to meet the threshold measure<br>
of informativeness, is not added to the set of candidate SNP haplotype blocks (step 350),<br>
nor does another SNP get added to block H for analysis. Instead, a new block, I, is<br>
selected that contains only the SNP at position 3, and so on.<br>
Once a set of candidate SNP blocks is constructed (step 350 of Figure 3), analysis<br>
is performed on the set to select a final set of SNP blocks (step 160 of Figure 1). The<br>
selection of the final set of SNP blocks can performed in a variety of ways. For example,<br>
referring back to Figure 4, one could select the largest block containing SNP position 1<br>
that passes the threshold test (block C, containing SNPs 1, 2 and 3), discard the smaller<br>
blocks that contain the same SNPs (blocks A and B). Then the next block selected might<br>
be the next block starting with SNP position 4 that is the largest block that meets the<br>
threshold test for informativeness (block G) and the smaller blocks that contain the same<br>
SNPs (blocks E and F) would be discarded. Such a method would give a set of final, non-<br>
overlapping SNP haplotype blocks that span the genomic region of interest, contain the<br>
SNPs of interest and that have a high level of informativeness. Thus, once all candidate<br>
SNP haplotype blocks are evaluated, the result may be, in a preferred embodiment, a set<br>
of non-overlapping SNP haplotype blocks that encompasses all the SNPs in the original<br>
set. Some groups, called isolates, may consist of only a single SNP, and by definition<br>
have an informativeness of 1. Other groups may consist of a hundred or more SNPs, and<br>
have an informativeness exceeding 30.<br>
An alternative method for selecting a final set of SNP haplotype blocks is shown<br>
in Figures 5A and 5B. Looking first at Figure 5A, in a first step 510, the candidate SNP<br>
haplotype block set (generated, for example, by the methods described in Figures 3 and 4<br>
herein) is analyzed for informativeness. In step 520, the candidate SNP haplotype block<br>
with the highest informativeness in the entire candidate set is chosen to be added to the<br>
final SNP haplotype block set (step 530). Once this candidate SNP haplotype block is<br>
chosen to be a member of the final SNP haplotype block set, it is deleted from the<br>
candidate block set (step 540), and all other candidate SNP haplotype blocks that overlap<br>
with the chosen block are deleted from the candidate SNP haplotype block set (step 550).<br>
Next, the candidate SNP haplotype blocks remaining in the candidate set are analyzed for<br>
informativeness (step 510), and the candidate SNP haplotype block with the highest<br>
informativeness is chosen to be added to the final SNP haplotype block set (steps 520 and<br>
530). As before, once this SNP haplotype block is chosen to be a member of the final<br>
SNP haplotype block set, it is deleted from the candidate block set (step 540), and all<br>
other candidate SNP haplotype blocks that overlap with the chosen block are deleted from<br>
the candidate SNP haplotype block set (step 550). The process continues until a final set<br>
of non-overlapping SNP haplotype blocks that encompasses all the SNPs in the original<br>
set is constructed.<br>
Figure 5B illustrates a simple employment of the method of selecting a final set of<br>
SNP haplotype blocks described in Figure 5A. In figure 5B, a sequence 5" to 3" is<br>
analyzed for SNPs, SNP haplotype patterns and candidate SNP haplotype blocks<br>
according to the methods of the present invention. Candidate SNP haplotype blocks<br>
contained within this sequence are indicated by their placement under the sequence, and<br>
are designated by a letter. In addition, after the letter, the informativeness of each block<br>
is indicated. For example, candidate SNP haplotype block A is located at the extreme 5"<br>
end of the sequence, and has an informativeness of 1. Candidate SNP haplotype block R<br>
is located at the extreme 3" end of the sequence, and has an informativeness of 2.<br>
According to figure 5A, in a first step 510, the candidate SNP haplotype blocks<br>
are analyzed for informativeness, and in step 520, the SNP haplotype block with the<br>
highest informativeness is chosen to be added to the final SNP haplotype block set (steps<br>
520 and530). In the case of figure 5B, candidate SNP haplotype block M with an<br>
informativeness of 6 would be the first candidate SNP haplotype block selected to be<br>
added to the final SNP haplotype block set. Once SNP haplotype block M is selected, it<br>
is deleted or removed from the candidate set of SNP haplotype blocks (step 540), and all<br>
other candidate SNP haplotype blocks that overlap with SNP haplotype block M (blocks<br>
J, N, K, L, O and P) are deleted frorn the candidate SNP haplotype block set (step 550).<br>
Next, the remaining blocks of the candidate SNP haplotype block set, namely SNP<br>
haplotype blocks A, B, C, D, E, F, G, H, I, Q and R are analyzed for informativeness, and<br>
in step 520, the remaining SNP haplotype block with the highest informativeness, I, with<br>
an informativeness of 5, is chosen to be added to the final SNP haplotype block set (530)<br>
and deleted or removed from the candidate set of SNP haplotype blocks (step 540). Next,<br>
in step 550, all other candidate SNP haplotype blocks that overlap with SNP haplotype<br>
block I, here, only block H, is deleted from the candidate SNP haplotype block set.<br>
Again, the remaining blocks of the candidate SNP haplotype block set, namely SNP<br>
haplotype blocks A, B, C, D, E, F, G, Q and R are analyzed for informativeness. In step<br>
520, the remaining SNP haplotype block with the highest informativeness, block F, with<br>
an informativeness of 4, is chosen to be added to the final SNP haplotype block set (530)<br>
and deleted or removed from the candidate set of SNP haplotype blocks (step 540). Next,<br>
all other candidate SNP haplotype blocks that overlap with SNP haplotype block F~here,<br>
blocks E, G, C and D--are deleted from the candidate SNP haplotype block set, and the<br>
remaining blocks of the candidate SNP haplotype block set, namely SNP haplotype<br>
blocks A, B, Q and R, are analyzed for informativeness, and so on.<br>
Other methods can be employed to select a final set of SNP haplotype blocks for<br>
analysis from the set of candidate SNP haplotype blocks (step 160 of Figure 1). For<br>
example, algorithms known in the art may be applied for this purpose. For example,<br>
shortest-paths algorithms may be used (see, generally, Cormen, Leiserson, and Rivest,<br>
Introduction to Algorithms (MIT Press) pp. 514-78 (1994). In a shortest-paths problem, a<br>
weighted, directed graph G=(V,E), With weight function w : E?R mapping edges to real-<br>
valued weights is given. The weight of path p = (v0, v1, ...vk) is the sum of the weights of<br>
its constituent edges:<br>
The shortest-path weight from u to v is defined by d(u,v) being equal to min w(p):u?v if<br>
there is a path from u to v; otherwise, d(u, v ) is equal to infinity. A shortest path from<br>
vertex u to vertex v is then defined as any path p with weight w(p) = d(u,v). Edge weights<br>
can be interpreted as various metrics: for example, distance, time, cost, penalties, loss, or<br>
any other quantity that accumulates linearly along a path that one wishes to minimize. In<br>
the embodiment of the shortest path algorithm used in applications of this invention, each<br>
SNP haplotype block would be considered a "vertex" with an "edge" defined for each<br>
boundary of the block. Each SNP haplotype block has a relationship to each other SNP<br>
haplotype block, with a "cost" for each edge. Cost is determined by parameters of choice,<br>
such as overlap (or the extent thereof) of the vertices or gaps between the vertices.<br>
Single-source shortest-paths problems focus on a given graph G=(V,E), where a<br>
shortest path from a given source vertex S V to every vertex v V is determined.<br>
Additionally, variants of the single source algorithm may be applied. For example, one<br><br>
may apply a single-destination shortest-paths solution where a shortest path to a given<br>
destination vertex t from every vertex v is found. Reversing the direction of each edge in<br>
the graph reduces this problem to a single-source problem. Alternatively, one may apply<br>
a single-pair shortest-path problem where the shortest path from u to v for given vertices<br>
u and v is found. If the single-source problem with source vertex u is solved, the single-<br>
source shortest path problem is solved as well. Also, the all-pairs shortest-paths approach<br>
may be employed. In this case, a shortest path from u to v for every pair of vertices u and<br>
v is found--a single-source algorithm is run from each vertex.<br>
One single-source shortest-path algorithm that may be employed in the methods<br>
of the present invention is Dijkstra"s algorithm. Dijkstra"s algorithm solves the single-<br>
source shortest-paths problem on a weighted, directed graph G=(V,E) for the case in<br>
which all edge weights are nonnegative. Dijkstra"s algorithm maintains a set of vertices,<br>
S, whose final shortest-path weights from a source s have already been determined. That<br>
is, for all vertices v being elements of S, d[v]=d(s,v). The algorithm repeatedly selects the<br>
vertex u as an element of V-S with the minimum shortest-path estimate, inserts u into S,<br>
and relaxes all edges radiating from u. In one implementation, a priority queue Q that<br>
contains all the vertices in V-S, keyed by their d values, is maintained. This<br>
implementation assumes that graph G is represented by adjacency lists.<br>
Dijkstra (G, w, s)<br>
1 INITIALIZE-SINGLE SOURCE (G,s)<br>
2 S??<br>
3 Q?V[G]<br>
4 while Q??<br>
5 do u ? Extract-Min (Q)<br>
6 S?SU{u}<br>
7 for each vertex v e Adj[u]<br>
8 do Relax (u,v,w)<br>
Thus, G in this case is the graph of linear coverage of the genomic sequence being<br>
analyzed and S is the set of vertices selected. Once one vertex is selected that covers a<br>
particular area of the genomic sequence, other vertices that overlap this sequence can be<br>
discarded.<br>
Other algorithms that may be used for selecting SNP haplotype blocks include a<br>
greedy algorithm (again, see, Cormen, Leiserson, and Rivest, Introduction to Algorithms<br>
(MIT Press) pp. 329-55 (1994)). A greedy algorithm obtains an optimal solution to a<br>
problem by making a sequence of choices. For each decision point in the algorithm, the<br>
choice that seems best at the moment is chosen. This heuristic strategy does not always<br>
produce an optimal solution. Greedy algorithms differ from dynamic programming in<br>
that in dynamic programming, a choice is made at each step, but the choice may depend<br>
on the solutions to subproblems. In a greedy algorithm, whatever choice seems best at<br>
the moment is chosen and then subproblems arising after the choice is made are solved.<br>
Thus, the choice made by a greedy algorithm may depend on the choices made thus far,<br>
but cannot depend on any future choices or on the solutions to subproblems. One<br>
variation of greedy algorithms is Huffman codes. A Huffman greedy algorithm<br>
constructs an optimal prefix code and the algorithm builds a tree T corresponding to the<br>
optimal code in a bottom-up manner. It begins with a set of |C | leaves and performs a<br>
sequence of |C |-1 "merging" operations to create the final tree. For example, assuming C<br>
is a set of n characters and that each character c e C is an object with a defined frequency<br>
f[c]; a priority queue Q, keyed on f, is used to identify the two least-frequent objects to<br>
merge together. The result of the merger of two objects is a new object whose frequency<br>
is the sum of the frequencies of the two objects that were merged. For example:<br>
1. n?|C|<br>
2. Q?C<br>
3. for i?l to n-1<br>
4. do z?Allocate-Node()<br>
5. x?left[z]?EXTRACT-MlN(Q)<br>
6. y?right[z]?EXTRACT-MIN(Q)<br>
7. f[z] ?f[x]+f[y]<br>
8. Insert (Q,z)<br>
9. return Extract-Min(0<br>
Line 2 initializes the priority queue Q with the characters in C. The for loop in<br>
lines 3-8 repeatedly extracts the two nodes x and y of lowest frequency from the queue,<br>
and replaces them in the queue with a new node z representing their merger. The<br>
frequency of z is computed as the sum of the frequencies of x and y in line 7. The node z<br>
has x as its left child and y as its right child. After n-l mergers, the one node left in the<br>
queue—the root of the code tree—is returned in line 9.<br>
Again, these methods result in a set of final, non-overlapping SNP haplotype<br>
blocks that encompasses all SNPs evaluated in a particular genomic region. An important<br>
result of selecting SNPs, SNP haplotype blocks and SNP haplotype patterns according to<br>
the methods of the present invention, is that in some embodiments during the calculation<br>
of informativeness of SNP haplotype blocks, informative SNPs for each SNP haplotype<br>
block and pattern are determined. Informative SNPs allow for data compression.<br>
In one embodiment of the present invention, the selection of at least log2p SNPs from<br>
each group containing p patterns (rounding up to the nearest integer) provides one set of<br>
informative SNPs which are unusually powerful for predicting genotype/phenotype<br>
associations. One skilled in the art recognizes that in other analyses it is not necessary to<br>
use spatially contiguous groups to determine such a subset. For example, in some<br>
embodiments of the present invention, it may be desirable to identify sets of non-adjacent<br>
SNPs that statistically are passed on in a fashion analogous to that of SNP haplotype<br>
blocks even though they are not spatially contiguous on the DNA strand.<br>
In order to determine SNP haplotype blocks that will be used in association<br>
studies accurately (build an accurate baseline of SNPs and SNP haplotype blocks and<br>
patterns), it is necessary to examine more than a few individual DNA strands. Figure 6<br>
illustrates the importance of examining at least about five different DNA strands for<br>
determining SNP haplotype blocks and for the selection of informative SNPs. The top<br>
portion of Figure 6 illustrates the sequence of a hypothetical stretch of DNA, with the<br>
variant positions indicated and variant block boundaries drawn; however, SNP haplotype<br>
block boundaries would not be known ab initio. Sequencing results 610 show the results<br>
of sequencing haploid DNA of three individuals. As shown, in general it is possible to<br>
have identified a large fraction of the common SNPs after a relatively small number of<br>
individuals have been sequenced. In the case in Figure 6, the SNPs at each location<br>
shown in the top portion of Figure 6 have been identified, as indicated by check marks.<br>
If, however, further individuals are not evaluated, the block boundaries would not<br>
be correctly identified at this stage. For example, while one could at this stage draw<br>
block boundaries between blocks 620 and 630 (note that the first C ?G variant predicts<br>
the first G?A variant, and the first C?T variant predicts the second C?T variant), it is<br>
not possible to distinguish between the blocks 630 and 640 at this stage. At this stage it<br>
appears that the first C?T variant would predict the first and second T?A variants.<br>
Accordingly, a more statistically significant sample set is required to draw the block<br>
boundaries. For example, in the methods of the present invention, the number of DNA<br>
strands analyzed to determine SNP haplotype blocks, SNP haplotype patterns, and/or<br>
informative SNPs is a plurality, for example, at least about five or at least about 10. In<br>
preferred embodiments, the number of DNA stands is at least 16. In more preferred<br>
embodiments, the number of DNA strands analyzed to determine SNP haplotype blocks,<br>
SNP haplotype patterns, and/or informative SNPs is at least 25. However, once relevant<br>
SNPs have been identified (i.e.,SNP discovery has been performed), it is possible to<br>
genotype only the variant positions in the remaining samples to complete the process of<br>
identifying block boundaries without sequencing the entire stretch of genomic DNA. For<br>
examples of such methods, see USSN 10/042,819, filed 01/06/02, attorney docket number<br>
1016N-1, entitled "Whole Genome Scanning".<br>
The results of performing a genotyping process on only the SNPs in another<br>
hypothetical genomic sample are shown in Fig. 6 at 650. As shown, by performing this<br>
additional genotyping step, it is now possible to see that blocks 630 and 640 are<br>
distinguishable. Specifically, it is now possible to see that the first C?T variant does not<br>
track with the first and second T?A variants, but instead, the first C?T variant can be<br>
used to predict only the second C?T variant (and vice versa) and the first T?A variant<br>
can be used only to predict the second T?A variant (and vice versa).<br>
In addition to the aspects of the present invention described above, a specific<br>
embodiment of the present invention is that it can be employed to resolve ambiguous SNP<br>
haplotype sequences for data analysis. For example, a SNP may be ambiguous because<br>
data from a gel sequencing operation or array hybridization experiment does not give a<br>
clear result. "Resolving" in this case may mean, e.g., resolving ambiguous SNP locations<br>
in a SNP haplotype sequence by matching the SNP haplotype sequence to the SNP<br>
haplotype pattern to which the SNP haplotype sequence most closely relates.<br>
Additionally, "resolving" may mean removing an ambiguous SNP haplotype sequence<br>
from data analysis.<br>
In one embodiment of resolving ambiguous SNP haplotype sequences, SNP<br>
haplotype sequences are placed in a data set for possible addition to a pattern set. The<br>
data set will contain all SNP haplotype sequences that are to be evaluated for possible<br>
assignment to a SNP haplotype pattern. Referring now to Figure 7A, in step 710, the<br>
SNP haplotype sequences in the data set are compared, one by one, to the pattern<br>
sequences in the pattern set. In some cases, there will be no patterns in the pattern set<br>
initially, though in other cases some or all pattern sequences may be known beforehand.<br>
In step 720, a query is made: is the SNP haplotype sequence from the data set consistent<br>
with a pattern sequence in the pattern set? If the answer is no, step 730 provides the SNP<br>
haplotype sequence being evaluated will be added to the pattern set. If the answer is yes,<br>
another query is made (740): is the SNP haplotype sequence from the data set consistent<br>
with more than one pattern sequence in the pattern set?<br>
If the answer is yes, the SNP sequence from the data set may be discarded or, in<br>
some embodiments, held for further or different analyses (step 750). If the answer to the<br>
second query is no, then, in step 760, the SNP sequence from the data set is compared to<br>
 the pattern sequence from the pattern set with which it is consistent. From these two<br>
sequences, the SNP sequence with the least number of ambiguities is selected and placed<br>
in the pattern set (770). The SNP sequence containing the more ambiguities may be<br>
discarded, or, in some embodiments, held for further or different types of analyses.<br>
The resolving process may be understood further by referring to Figures 7A and<br>
7B. In Figure 7B, a first SNP sequence, TTCGA, is compared to the sequences contained<br>
in the pattern set (step 710). At this point, there are no pattern sequences contained in the<br>
pattern set, thus TTCGA is not consistent with any pattern sequence in the pattern set.<br>
This occurrence of SNP sequence TTCGA is then removed from the data set (or is<br>
retained for different analyses), and added to the pattern set (730). The pattern set now<br>
has one pattern sequence, TTCGA.<br>
Looking again at Figure 7B, the second SNP sequence in the data set, T?C??, is<br>
compared to the sequence contained in the pattern set (step 710). Now there is one<br>
pattern sequence in the pattern set, TTCGA, and T?C?? is consistent with sequence (step<br>
720). The answer to the second query (740), whether SNP sequence T?C?? is consistent<br>
with more than one pattern sequence in the pattern set, is no, as currently there is only one<br>
pattern sequence, TTCGA, in the pattern set. In step 760, T?C?? is compared to TTCGA<br>
to determine which sequence has the more ambiguities. T?C?? clearly does; thus,<br>
TTCGA is retained in the pattern set (770) and T?C?? may be discarded or held for<br>
further analyses.<br>
The third sequence of the data set in Figure 7B is C????. C???? first is compared<br>
to TTCGA (step 710), is found not to be consistent with TTCGA (720), and is thus added<br>
to the pattern set (730). The fourth sequence in Figure 7B is CTACA. CTACA is<br>
compared to TTCGA and C???? (the pattern sequences in the pattern set, step 710), and is<br>
found to be consistent with C???? (720). The second query (740) now is made: is<br>
CTACA consistent with both C???? and TTCGA? The answer is no, so C???? and<br>
CTACA are then compared (760) and the sequence with the least number of ambiguities,<br>
in this case, CTACA, is held in the pattern set and C???? is discarded (removed from<br>
analysis), or held for further analyses (770).<br>
The fifth SNP sequence in the data set in Figure 7B is ?T??A. This SNP sequence<br>
is compared to pattern sequences TTCGA and CTACA (710) and is found to be<br>
consistent with both TTCGA and CTACA. Thus, the answer to query 740 is yes: ?T??A<br>
is consistent with more than one pattern sequence in the pattern set. In step 750, SNP<br>
sequence ?T??A is held for further analysis or discarded (removed from analysis).<br>
Another approach to resolving allows that if, for example, one pattern sequence is<br>
CCATT? and a SNP sequence from the data set is C?ATTG, the sequences are<br>
"combined" to solve the ambiguities (CCATTG), and the "combined" sequence is added<br>
to the pattern set. Additional array hybridizations, sequencing or other techniques known<br>
in the art may be employed to analyze ambiguous SNP nucleotide positions.<br>
Association of Phenotypes with SNP Haplotypes Blocks and Patterns<br>
The SNP haplotype blocks, SNP haplotype patterns and/or informative SNPs<br>
identified may be used for a variety of genetic analyses. For example, once informative<br>
SNPs have been identified, they may be used in a number of different assays for<br>
association studies. For example, probes may be designed for microarrays that<br>
interrogate these informative SNPs. Other exemplary assays include, e.g., the Taqman<br>
assays and Invader assays described supra, as well as conventional PCR and/or<br>
sequencing techniques.<br>
In some embodiments, as shown in step 170 of Figure 1, the haplotype patterns<br>
identified may be used in the above-referenced assays to perform association studies.<br>
This may be accomplished by determining haplotype patterns in individuals with the<br>
phenotype of interest (for example, individuals exhibiting a particular disease or<br>
individuals who respond in a particular manner to administration of a drug) and<br>
comparing the frequency of the haplotype patterns in these individuals to the haplotype<br>
pattern frequency in a control group of individuals. Preferably, such SNP haplotype<br>
pattern determinations are genome-wide; however, it may be that only specific regions of<br>
the genome are of interest, and the SNP haplotype patterns of those specific regions are<br>
used. In addition to the other embodiments of the methods of the present invention<br>
disclosed herein, the methods additionally allow for the "dissection" of a phenotype.<br>
That is, a particular phenotype may result from two or more different genetic bases. For<br>
example, obesity in one individual may be the result of a defect in Gene X, while the<br>
obesity phenotype in a different individual may be the result of mutations in Gene Y and<br>
Gene Z. Thus, the genome scanning capabilities of the present invention allow for the<br>
dissection of varying genetic bases for similar phenotypes. Once specific regions of the<br>
genome are identified as being associated with a particular phenotype, these regions may<br>
be used as drug discovery targets (step 180 of Figure 1) or as diagnostic markers (step<br>
190 of Figure 1).<br>
As described in the previous paragraph, one method of conducting association<br>
studies is to compare the frequency of SNP haplotype patterns in individuals with a<br>
phenotype of interest to the SNP haplotype pattern frequency in a control group of<br>
individuals. In a preferred method, informative SNPs are used to make the SNP<br>
haplotype pattern comparison. The approach of using informative SNPs has tremendous<br>
advantage over other whole genome scanning or genotyping methods known in the art to<br>
date, for instead of reading all 3 billion bases of each individual"s genome-or even<br>
reading the 3-4 million common SNPs that may be found-only informative SNPs from a<br>
sample population need to be determined. Reading these particular, informative SNPs<br>
provides sufficient information to allow statistically accurate association data to be<br>
extracted from specific experimental populations, as described above.<br>
Figure 8 illustrates an embodiment of one method of determining genetic<br>
associations using the methods of the present invention. In step 800, the frequency of<br>
informative SNPs is determined for genomes of a control population. In step 810, the<br>
frequency of informative SNPs is determined for genomes of a clinical population. Steps<br>
800 and 810 may be performed by using the aforementioned SNP assays to analyze the<br>
informative SNPs in a population of individuals. In step 820, the informative SNP<br>
frequencies from steps 800 and 810 are compared. Frequency comparisons may be made,<br>
for example, by determining the minor allele frequency (number of individuals with a<br>
particular minor allele divided by the total number of individuals) at each informative<br>
SNP location in each population and comparing these minor allele frequencies. In step<br>
830, the informative SNPs displaying a difference between the frequency of occurrence in<br>
the control versus clinical populations are selected for analysis. Once informative SNPs<br>
are selected, the SNP haplotype blocks that contain the informative SNPs are identified,<br>
which in turn identifies the genomic region of interest (step 840). The genomic regions<br>
are analyzed by genetic or biological methods known in the art (step 850), and the regions<br>
are analyzed for possible use as drug discovery targets (step 860) or as diagnostic markers<br>
(step 870), as described in detail below.<br>
Uses of Identified Genomic Sequences<br>
Once a genetic locus or multiple loci in the genome are associated with a<br>
particular phenotypic trait--for example, a disease susceptibility locus--the gene or genes<br>
or regulatory elements responsible for the trait can be identified. These genes or<br>
regulatory elements may then be used as therapeutic targets for the treatment of the<br>
disease, as shown in step 180 of Figure 1 or step 860 of Figure 8. The genomic sequences<br>
identified by the methods of the present invention may be genie or nongenic sequences.<br>
The term "gene" intended to mean the open reading frame (ORF) encoding specific<br>
polypeptides, intronic regions, as well as adjacent 5" and 3" non-coding nucleotide<br>
sequences involved in the regulation of expression of the gene up to about 10 kb beyond<br>
the coding region, but possibly further in either direction. The ORFs of an identified gene<br>
may affect the disease state due to their effect on protein structure. Alternatively, the<br>
noncoding sequences of the identified gene or nongenic sequences may affect the disease<br>
state by impacting the level of expression or specificity of expression of a protein.<br>
Generally, genomic sequences are studied by isolating the identified gene substantially<br>
free of other nucleic acid sequences that do not include the genic sequence. The DNA<br>
sequences are used in a variety of ways. For example, the DNA may be used to detect or<br>
quantify expression of the gene in a biological specimen. The manner in which cells are<br>
probed for the presence of particular nucleotide sequences is well established in the<br>
literature and does not require elaboration here, however, see, e.g., Sambrook, et al,<br>
Molecular Cloning: A Laboratory Manual (Cold Spring Harbor Laboratory, New York)<br>
(1989)<br>
In addition, the sequence of the gene, including flanking promoter regions and<br>
coding regions, may be mutated in various ways known in the art to generate targeted<br>
changes in expression level, or changes in the sequence of the encoded protein, etc. The<br>
sequence changes may be substitutions, insertions, translocations or deletions. Deletions<br>
may include large changes, such as deletions of an entire domain or exon. Techniques for<br>
in vitro mutagenesis of cloned genes are known. Examples of protocols for site specific<br>
mutagenesis may be found in Gustin, et al, Biotechniques 14:22 (1993); Barany, Gene<br>
37:111-23 (1985); Colicelli, et al, Mol Gen. Genet. 199:537-9 (1985); Prentki, et al,<br>
Gene 29:303-13 (1984); Sambrook, et al, Molecular Cloning: A Laboratory Manual<br>
(Cold Spring Harbor Press) pp. 15.3-15.108 (1989); Weiner, et al, Gene 126:35-41<br>
(1993); Sayers, et al., Biotechniques 13:592-6 (1992); Jones and Winistorfer,<br>
Biotechniques 12:528-30 (1992); and Barton, et al, Nucleic Acids Res. 18:7349-55<br>
(1990). Such mutated genes may be used to study structure/function relationships of the<br>
protein product, or to alter the properties of the protein that affect its function or<br>
regulation.<br>
The identified gene may be employed for producing all or portions of the resulting<br>
polypeptide. To express a protein product, an expression cassette incorporating the<br>
identified gene may be employed. The expression cassette or vector generally provides a<br>
transcriptional and translational initiation region, which may be inducible or constitutive,<br>
where the coding region is operably linked under the transcriptional control of the<br>
transcriptional initiation region, and a transcriptional and translational termination region.<br>
These control regions may be native to the identified gene, or may be derived from<br>
exogenous sources.<br>
The peptide may be expressed in prokaryotes or eukaryotes in accordance with<br>
conventional methods, depending upon the purpose for expression. For large scale<br>
production of the protein, a unicellular organism, such as E. coli, B. subtilis, S. cerevisiae,<br>
insect cells in combination with baculovirus vectors, or cells of a higher organism such as<br>
vertebrates, particularly mammals, e.g. COS 7 cells, may be used as the expression host<br>
cells. In many situations, it may be desirable to express the gene in eukaryotic cells,<br>
where the gene will benefit from native folding and post-translational modifications.<br>
Small peptides also can be synthesized in the laboratory. With the availability of the<br>
protein or fragments thereof in large amounts, the protein may be isolated and purified in<br>
accordance with conventional ways. A lysate may be prepared of the expression host and<br>
the proteins or fragments thereof purified using HPLC, exclusion chromatography, gel<br>
electrophoresis, affinity chromatography, or other purification techniques.<br>
An expressed protein may be used for the production of antibodies, where short<br>
fragments induce the expression of antibodies specific for the particular polypeptide<br>
(monoclonal antibodies), and larger fragments or the entire protein allow for the<br>
production of antibodies over the length of the polypeptide (polyclonal antibodies).<br>
Antibodies are prepared in accordance with conventional ways, where the expressed<br>
polypeptide or protein is used as an immunogen, by itself or conjugated to known<br>
immunogenic carriers, e.g. KLH, pre-S HBsAg, other viral or eukaryotic proteins, or the<br>
like. Various adjuvants may be employed, with a series of injections, as appropriate. For<br>
monoclonal antibodies, after one or more booster injections, the spleen is isolated, the<br>
lymphocytes are immortalized by cell fusion and screened for high affinity antibody<br>
binding. The immortalized cells, i.e. hybridomas, producing the desired antibodies may<br>
then be expanded. For further description, see Monoclonal Antibodies: A Laboratory<br>
Manual, Harlow and Lane, eds. (Cold Spring Harbor Laboratories, Cold Spring Harbor,<br>
N.Y.) (1988). If desired, the mRNA encoding the heavy and light chains may be isolated<br>
and mutagenized by cloning in E. coli, and the heavy and light chains mixed to further<br>
enhance the affinity of the antibody. Alternatives to in vivo immunization as a method of<br>
raising antibodies include binding to phage "display" libraries, usually in conjunction<br>
with in vitro affinity maturation.<br>
The identified genes, gene fragments, or the encoded protein or protein fragments<br>
may be useful in gene therapy to treat degenerative andother disorders. For example,<br>
expression vectors may be used to introduce the identified gene into a cell. Such vectors<br>
generally have convenient restriction sites located near the promoter sequence to provide<br>
for the insertion of nucleic acid sequences in a recipient genome. Transcription cassettes<br>
may be prepared comprising a transcription initiation region, the target gene or fragment<br>
thereof, and a transcriptional termination region. The transcription cassettes may be<br>
introduced into a variety of vectors, e.g. plasmid; retrovirus, e.g. lentivirus; adenovirus;<br>
and the like, where the vectors are able to be transiently or stably maintained in the cells.<br>
The gene or protein product may be introduced directly into tissues or host cells by any<br>
number of routes, including viral infection, microinjection, or fusion of vesicles. Jet<br>
injection may also be used for intramuscular administration, as described by Furth, et al.,<br>
Anal. Biochem, 205:365-68 (1992). Alternatively, the DNA may be coated onto gold<br>
microparticles, and delivered intradermally by a particle bombardment device, or "gene<br>
gun" as described in the literature (see, for example, Tang, et al., Nature, 356:152-54<br>
(1992)).<br>
Antisense molecules can be used to down-regulate expression of the identified<br>
gene in cells. The antisense reagent may be antisense oligonucleotides, particularly<br>
synthetic antisense oligonucleotides having chemical modifications, or nucleic acid<br>
constructs that express such antisense molecules as RNA. A combination of antisense<br>
molecules may be administered, where a combination may comprise multiple different<br>
sequences.<br>
As an alternative to antisense inhibitors, catalytic nucleic acid compounds, e.g.,<br>
ribozymes, anti-sense conjugates, etc., may be used to inhibit gene expression.<br>
Ribozymes may be synthesized in vitro and administered to the patient, or may be<br>
encoded on an expression vector, from which the ribozyme is synthesized in the targeted<br>
cell (for example, see International patent application WO 9523225, and Beigelman, et<br>
al., Nucl. Acids Res. 23:4434-42 (1995)). Examples of oligonucleotides with catalytic<br>
activity are described in WO 9506764. Conjugates of antisense oligonucleotides with a<br>
metal complex, e.g. terpyridylCu(II), capable of mediating mRNA hydrolysis are<br>
described in Bashkin, et al., Appl. Biochem. Biotechnol. 54:43-56 (1995).<br>
In addition to using the identified sequences for gene therapy, the identified nucleic acids<br>
can be used to generate genetically modified non-human animals to create animal models<br>
of diseases or to generate site-specific gene modifications in cell lines for the study of<br>
protein function or regulation. The term "transgenic" is intended to encompass<br>
genetically modified animals having an exogenous gene that is stably transmitted in the<br>
host cells where, for example, the gene may be altered in sequence to produce a modified<br>
protein, or may be a reporter gene operably linked to an exogenous promoter. Transgenic<br>
animals may be made through homologous recombination, where the endogenous gene<br>
locus is altered, replaced or otherwise disrupted. Alternatively, a nucleic acid construct<br>
may be randomly integrated into the genome. Vectors for stable integration include<br>
plasmids, retroviruses and other animal viruses, YACs, and the like. Of interest are<br>
transgenic mammals, e.g., cows, pigs, goats, horses, etc., and, particularly, rodents, e.g.,<br>
rats, mice, etc.<br>
Investigation of genetic function may also utilize non-mammalian models, particularly<br>
using those organisms that are biologically and genetically well-characterized, such as C.<br>
elegans, D. melanogaster and S. cerevisiae. The subject gene sequences may be used to<br>
knock-out corresponding gene function or to complement defined genetic lesions in order<br>
to determine the physiological and biochemical pathways involved in protein function.<br>
Drug screening may be performed in combination with complementation or knock-out<br>
studies, e.g., to study progression of degenerative disease, to test therapies, or for drug<br>
discovery.<br>
In addition, the modified cells or animals are useful in the study of protein function and<br>
regulation. For example, a series of small deletions and/or substitutions may be made in<br>
the identified gene to determine the role of different domains in enzymatic activity, cell<br>
transport or localization, etc. Specific constructs of interest include, but are not limited<br>
to, antisense constructs to block gene expression, expression of dominant negative genetic<br>
mutations, and over-expression of the identified gene. One may also provide for<br>
expression of the identified gene or variants thereof in cells or tissues where it is not<br>
normally expressed or at abnormal times of development. In addition, by providing<br>
expression of a protein in cells in which it is not normally produced, one can induce<br>
changes in cellular behavior that provide information regarding the normal function of the<br>
protein.<br>
Protein molecules may be assayed to investigate structure/function parameters. For<br>
example, by providing for the production of large amounts of a protein product of an<br>
identified gene, one can identify ligands or substrates that bind to, modulate or mimic the<br>
action of that protein product. Drug screening identifies agents that provide, e.g., a<br>
replacement or enhancement for protein function in affected cells, or for agents that<br>
modulate or negate protein function. The term "agent" as used herein describes any<br>
molecule, e.g. protein or small molecule, with the capability of altering, mimicking or<br>
masking, either directly or indirectly, the physiological function of an identified gene or<br>
gene product. Generally a plurality of assay mixtures are run in parallel with different<br>
concentrations of the agent to obtain a differential response to the various concentrations.<br>
Typically, one of these concentrations serves as a negative control, i.e., at zero<br>
concentration or below the level of detection.<br>
A wide variety of assays may be used for this purpose, including labeled in vitro protein-<br>
protein binding assays, protein-DNA binding assays, electrophoretic mobility shift assays,<br>
immunoassays for protein binding, and the like. Also, all or a fragment of the purified<br>
protein may be used for determination of three-dimensional crystal structure, which can<br>
be used for determining the biological function of the protein or a part thereof, modeling<br>
intermolecular interactions, membrane fusion, etc.<br>
Candidate agents encompass numerous chemical classes, though typically they are<br>
organic molecules or complexes, preferably small organic compounds, having a<br>
molecular weight of more than 50 and less than about 2,500 daltons. Candidate agents<br>
comprise functional groups necessary for structural interaction with proteins, particularly<br>
hydrogen bonding, and typically include at least an amine, carbonyl, hydroxyl or carboxyl<br>
group, and frequently at least two of the functional chemical groups. The candidate<br>
agents often comprise cyclical carbon or heterocyclic structures and/or aromatic or<br>
polyaromatic structures substituted with one or more of the above functional groups.<br>
Candidate agents are also found among biomolecules including, but not limited to:<br>
peptides, saccharides, fatty acids, steroids, purines, pyrimidines, derivatives, structural<br>
analogs or combinations thereof.<br>
Candidate agents are obtained from a wide variety of sources including libraries of<br>
synthetic or natural compounds. For example, numerous means are available for random<br>
and directed synthesis of a wide variety of organic compounds and biomolecules,<br>
including expression of randomized oligonucleotides and oligopeptides. Alternatively,<br>
libraries of natural compounds in the form of bacterial, fungal, plant and animal extracts<br>
are available or readily produced. Additionally, natural or synthetically produced<br>
libraries and compounds are readily modified through conventional chemical, physical<br>
and biochemical means, and may be used to produce combinatorial libraries. Known<br>
pharmacological agents may be subjected to directed or random chemical modifications,<br>
such as acylation, alkylation, esterification, amidification, etc., to produce structural<br>
analogs.<br>
Where the screening assay is a binding assay, one or more of the molecules may be<br>
coupled to a label, where the label can directly or indirectly provide a detectable signal.<br>
Various labels include radioisotopes, fluorescers, chemiluminescers, enzymes, specific<br>
binding molecules, particles, e.g., magnetic particles, and the like. Specific binding<br>
molecules include pairs, such as bio tin and streptavidin, digoxin and antidigoxin, etc. For<br>
the specific binding members, the complementary member would normally be labeled<br>
with a molecule that provides for detection, in accordance with known procedures.<br>
A variety of other reagents may be included in the screening assay. These include<br>
reagents like salts, neutral proteins, e.g., albumin, detergents, etc that are used to facilitate<br>
optimal protein-protein binding and/or reduce non-specific or background interactions.<br>
Reagents that improve the efficiency of the assay, such as protease inhibitors, nuclease<br>
inhibitors, anti-microbial agents, etc., may be used.<br>
Agents may be combined with a pharmaceutically acceptable carrier, including any and<br>
all solvents, dispersion media, coatings, anti-oxidant, isotonic and absorption delaying<br>
agents and the like. The use of such media and agents for pharmaceutically active<br>
substances is well known in the art. Except insofar as any conventional media or agent is<br>
incompatible with the active ingredient, its use in the therapeutic compositions and<br>
methods described herein is contemplated. Supplementary active ingredients can also be<br>
incorporated into the compositions.<br>
The formulation may be prepared for use in various methods for administration. The<br>
formulation may be given orally, by inhalation, or may be injected, e.g. intravascular,<br>
intratumor, subcutaneous, intraperitoneal, intramuscular, etc. The dosage of the<br>
therapeutic formulation will vary widely, depending upon the nature of the disease, the<br>
frequency of administration, the manner of administration, the clearance of the agent from<br>
the host, and the like. The initial dose may be larger, followed by smaller maintenance<br>
doses. The dose may be administered as infrequently as once, weekly or biweekly, or<br>
fractionated into smaller doses and administered daily, semi-weekly, etc., to maintain an<br>
effective dosage level. In some cases, oral administration will require a different dose;<br>
than if administered intravenously. Identified agents of the invention can be incorporated<br>
into a variety of formulations for therapeutic administration. More particularly, the<br>
complexes can be formulated into pharmaceutical compositions by combination with<br>
appropriate, pharmaceutically acceptable carriers or diluents, and may be formulated into<br>
preparations in solid, semi-solid, liquid or gaseous forms, such as tablets, capsules,<br>
powders, granules, ointments, solutions, suppositories, injections, inhalants, gels,<br>
microspheres, and aerosols. As such, administration of the agents can be achieved in<br>
various ways. Agents may be systemic after administration or may be localized by the<br><br>
use of an implant that acts to retain the active dose at the site of implantation.<br>
The following methods and excipients are merely exemplary and are in no way<br>
limiting. For oral preparations, an agent can be used alone or in combination with<br>
appropriate additives to make tablets, powders, granules or capsules, for example, with<br>
conventional additives, such as lactose, mannitol, corn starch or potato starch; with<br>
binders, such as crystalline cellulose, cellulose derivatives, acacia, corn starch or gelatins;<br>
with disintegrators, such as corn starch, potato starch or sodium carboxymethylcellulose;<br>
with lubricants, such as talc or magnesium stearate; and if desired, with diluents,<br>
buffering agents, moistening agents, preservatives and flavoring agents.<br>
Additionally, agents may be formulated into preparations for injections by<br>
dissolving, suspending or emulsifying them in an aqueous or nonaqueous solvent, such as<br>
vegetable or other similar oils, synthetic aliphatic acid glycerides, esters of higher<br>
aliphatic acids or propylene glycol; and if desired, with conventional additives such as<br>
solubilizers, isotonic agents, suspending agents, emulsifying agents, stabilizers and<br>
preservatives. Further, agents may be utilized in aerosol formulation to be administered<br>
via inhalation. The agents identified by the present invention can be formulated into<br>
pressurized acceptable propellants such as dichlorodifluoromethane, propane, nitrogen<br>
and the like. Alternatively, agents may be made into suppositories by mixing with a<br>
variety of bases such as emulsifying bases or water-soluble bases. Further, identified<br>
agents of the present invention can be administered rectally via a suppository. The<br>
suppository can include vehicles such as cocoa butter, carbowaxes and polyethylene<br>
glycols, which melt at body temperature, yet are solidified at room temperature.<br>
Implants for sustained release formulations are well-known in the art. Implants<br>
are formulated as microspheres, slabs, etc. with biodegradable or non-biodegradable<br>
polymers. For example, polymers of lactic acid and/or glycolic acid form an erodible<br>
polymer that is well-tolerated by the host. The implant containing identified agents of the<br>
present invention may be placed in proximity to the site of action, so that the local<br>
concentration of active agent is increased relative to the rest of the body. Unit dosage<br>
forms for oral or rectal administration such as syrups, elixirs, and suspensions may be<br>
provided wherein each dosage unit, for example, teaspoonful, tablespoonful, gel capsule,<br>
tablet or suppository, contains a predetermined amount of the compositions of the present<br>
invention. Similarly, unit dosage forms for injection or intravenous administration may<br>
comprise the compound of the present invention in a composition as a solution in sterile<br>
water, normal saline or another pharmaceutically acceptable carrier. The specifications<br>
for the novel unit dosage forms of the present invention depend on the particular<br>
compound employed and the effect to be achieved, and the pharmacodynamics associated<br>
with each active agent in the host.<br>
The pharmaceutically acceptable excipients, such as vehicles, adjuvants, carriers<br>
or diluents, are readily available to the public. Moreover, pharmaceutically acceptable<br>
auxiliary substances, such as pH adjusting and buffering agents, tonicity adjusting agents,<br>
stabilizers, wetting agents and the like, are readily available to the public.<br>
A therapeutic dose of an identified agent is administered to a host suffering from a<br>
disease or disorder. Administration may be topical, localized or systemic, depending on<br>
the specific disease. The compounds are administered at an effective dosage such that<br>
over a suitable period of time the disease progression may be substantially arrested. It is<br>
contemplated that the composition will be obtained and used under the guidance of a<br>
physician for in vivo use. The dose will vary depending on the specific agent and<br>
formulation utilized, type of disorder, patient status, etc., such that it is sufficient to<br>
address the disease or symptoms thereof, while minimizing side effects. Treatment may<br>
be for short periods of time, e.g., after trauma, or for extended periods of time, e.g., in the<br>
prevention or treatment of schizophrenia.<br>
The SNPs identified by the present invention may be used to analyze the<br>
expression pattern of an associated gene and the expression pattern correlated to a<br>
phenotypic trait of the organism such as disease susceptibility or drug responsiveness.<br>
The expression pattern in various tissues can be determined and used to identify<br>
ubiquitous expression patterns, tissue specific expression patterns, temporal expression<br>
patterns and expression patterns induced by various external stimuli such as chemicals or<br>
electromagnetic radiation. Such determinations would provide information regarding<br>
function of the gene and/or its protein product.<br>
The newly identified sequences also may be used as diagnostic markers, i.e., to<br>
predict a phenotypic characteristic such as disease susceptibility or drug responsiveness.<br>
In addition, the methods of the present invention may be used to stratify populations for<br>
clinical studies. As such, the genes or fragments thereof may be used as probes to<br>
determine whether the same nucleic acid sequence is present in the genome of an<br>
organism being tested. In addition, the probes may be used to monitor RNA or mRNA<br>
levels within the organism to be tested or a part thereof, such as a specific tissue or organ,<br>
so as to determine the expression level of the marker where the expression level can be<br>
correlated to a particular phenotypic characteristic of the organism. Likewise, the marker<br>
may be assayed at the protein level using any customary technique such as<br>
immunological methods-Western blots, radioimmune precipitation and the like-or<br>
activity based assays measuring an activity associated with the gene product. Moreover,<br>
when a phenotype cannot clearly distinguish between similar diseases having different<br>
genetic bases, the methods of the present invention can be used to identify correctly the<br>
disease.<br>
Also, it should be apparent that the methods of the present invention can be used<br>
on organisms aside from humans. For example, when the organism is an animal, the<br>
methods of the invention may be used to identify loci associated, e.g., with disease<br>
resistance/ or susceptibility, environmental tolerance, drug response or the like, and when<br>
the organism is a plant, the method of the invention may be used to identify loci<br>
associated with disease resistance/ or susceptibility, environmental tolerance and or<br>
herbicide resistance.<br>
It is to be understood that this invention is not limited to the particular<br>
methodology, protocols, cell lines, animal species or genera, and reagents described, as<br>
such may vary. It is also to be understood that the terminology used herein is for the<br>
purpose of describing particular embodiments only, and is not intended to limit the scope<br>
of the present invention, which will be limited only by the appended claims.<br>
Databases<br>
The present invention includes databases containing information concerning<br>
variations, for instance, information concerning SNPs, SNP haplotype blocks, SNP<br>
haplotype patterns and informative SNPs. In some embodiments, the databases of the<br>
present invention may comprise information on one or more haplotype patterns associated<br>
with one or more phenotypic traits. Databases may also contain information associated<br>
with a given variation such as descriptive information about the general genomic region<br>
in which the variation occurs, such as whether the variation is located in a known gene,<br>
whether there are known genes, gene homologs or regulatory regions nearby and the like.<br>
Other information that may be included in the databases of the present invention<br>
include, but are not limited to, SNP sequence information, descriptive information<br>
concerning the clinical status of a tissue sample analyzed for SNP haplotype patterns, or<br>
the clinical status of the patient from which the sample was derived. The database may<br>
be designed to include different parts, for instance a variation database, a SNP database, a<br>
SNP haplotype block or SNP haplotype pattern database and an informative SNP<br>
database. Methods for the configuration and construction of databases are widely<br>
available, for instance, see Akerblom et al., (1999) U.S. Patent 5,953,727, which is herein<br>
incorporated by reference in its entirety.<br>
The databases of the invention may be linked to an outside or external database.<br>
Figure 9 shows an exemplary computer network that is suitable for the databases and<br>
executing the software of the present invention. A computer workstation 902 is<br>
connected with the application/data server(s) 906 through a local area network (LAN),<br>
such as an ethemet 905. A printer 904 may be connected directly to the workstation or to<br>
the Ethernet 905. The LAN may be connected to a wide area network (WAN), such as<br>
the internet 908 via a gateway server 907 which may also serve as a firewall between the<br>
WAN 908 and the LAN 905. In preferred embodiments, the workstation may<br>
communicate with outside data sources, such as The SNP Consortium (TSC) or the<br>
National Center for Biotechnology Information 909, through the internet 908.<br>
Any appropriate computer platform may be used to perform the necessary<br>
comparisons between SNP haplotype blocks or patterns, associated phenotypes, any other<br>
information in the database or information provided as an input. For example, a large<br>
number of computer workstations are available from a variety of manufacturers, such has<br>
those available from Silicon Graphics. Client-server environments, database servers and<br>
networks are also widely available and are appropriate platforms for the databases of the<br>
invention.<br>
The databases of the invention may also be used to present information identifying<br>
the SNP haplotype pattern in an individual and such a presentation may be used to predict<br>
one or more phenotypic traits of the individual. Such methods may be used to predict the<br>
disease susceptibility/resistance and/or drug response of the individual. Further, the<br>
databases of the present invention may comprise information relating to the expression<br>
level of one or more of the genes associated with the variations of the invention.<br>
The following examples describe specific embodiments of the present invention<br>
and the materials and methods are illustrative of the invention and are not intended to<br>
limit the scope of the invention.<br>
Example 1: Preparation of Somatic Cell Hybrids<br>
Standard procedures in somatic cell genetics were used to separate human DNA<br>
strands (chromosomes) from a diploid state to a haploid state. In this case, a diploid<br>
human lymphoblastoid cell line that was wildtype for the thymidine kinase gene was<br>
fused to a diploid hamster fibroblast cell line containing a mutation in the thymidine<br>
kinase gene. A sub-population of the resulting cells were hybrid cells containing human<br>
chromosomes. Hamster cell line A23 cells were pipetted into a centrifuge tube containing<br>
10 ml DMEM in which 10% fetal bovine serum (FBS) + IX Pen/Strep + 10% glutamine<br>
were added, centrifuged at 1500 rpm for 5 minutes, resuspended in 5 ml of RPMI and<br>
pipetted into a tissue culture flask containing 15 ml RPMI medium. The lymphoblastoid<br>
cells were grown at 37° C to confluence. At the same time, human lymphoblastoid cells<br>
were pipetted into a centrifuge tube containing 10 ml RPMI in which 15% FBCS + lx<br>
Pen/Strep + 10% glutamine were added, centrifuged at 1500 rpm for 5 minutes,<br>
resuspended in 5 ml of RPMI and pipetted into a tissue culture flask containing 15 ml<br>
RPMI. The lymphoblastoid cells were grown at 37 °C to confluence.<br>
To prepare the A23 hamster cells, the growth medium was aspirated and the cells<br>
were rinsed with 10 ml PBS. The cells were then trypsinized with 2 ml of trypsin,<br>
divided onto 3-5 plates of fresh medium (DMEM without HAT) and incubated at 37 °C.<br>
The lymphoblastoid cells were prepared by transferring the culture into a centrifuge tube<br>
and centrifuging at 1500 rpm for 5 minutes, aspirating the growth medium, resuspending<br>
the cells in 5 ml RPMI and pipetting 1 to 3 ml of cells into 2 flasks containing 20 ml<br>
RPMI.<br>
To achieve cell fusion, approximately 8-10 x 106 lymphoblastoid cells were<br>
centrifuged at 1500 rpm for 5 min. The cell pellet was then rinsed with DMEM by<br>
resuspending the cells, centrifuging them again and aspirating the DMEM. The<br>
lymphoblastoid cells were then resuspended in 5 ml fresh DMEM. The recipient A23<br>
hamster cells had been grown to confluence and split 3-4 days before the fusion and were,<br>
at this point, 50-80% confluent. The old media was removed and the cells were rinsed<br>
three times with DMEM, trypsinized, and finally suspended in 5 ml DMEM. The<br>
lymphoblastoid cells were slowly pipetted over the recipient A23 cells and the combined<br>
culture was swirled slowly before incubating at 37 °C for 1 hour. After incubation, the<br>
media was gently aspirated from the A23 cells, and 2 ml room temperature PEG 1500<br>
was added by touching the edge of the plate with a pipette and slowly adding PEG to the<br>
plate while rotating the plate with the other hand. It took approximately one minute to<br>
add all the PEG in one full rotation of the plate. Next, 8 ml DMEM was added down the<br>
edge of the plate while rotating the plate slowly. The PEG/DMEM mixture was aspirated<br>
gently from the cells and then 8 ml DMEM was used to rinse the cells. This DMEM was<br>
removed and 10 ml fresh DMEM was added and the cells were incubated for 30 min. at<br>
37 °C. Again the DMEM was aspirated from the cells and 10 ml DMEM in which 10%<br>
FBCS and lx Pen/Strep were added, was added to the cells, which were then allowed to<br>
incubate overnight.<br>
After incubation, the media was aspirated and the cells were rinsed with PBS.<br>
The cells were then trypsinized and divided among plates containing selection media<br>
(DMEM in which 10% FBS + 1x Pen/Strep + 1x HAT were added) so that each plate<br>
received approximately 100,000 cells. The media was changed on the third day following<br>
plating. Colonies were picked and placed into 24-well plates upon becoming visible to<br>
the naked eye (day 9-14). If a picked colony was confluent within 5 days, it was deemed<br>
healthy and the cells were trypsinized and moved to a 6-well plate.<br>
DNA and stock hybrid cell cultures were prepared from the cells from the 6-well<br>
plate cultures. The cells were trypsinized and divided between a 100 mm plate containing<br>
10 ml selection media and an Eppendorf tube. The cells in the tube were pelleted,<br>
resuspended 200 µl PBX and DNA was isolated using a Qiagen DNA mini kit at a<br>
concentration of 
confluence, and the cells were either continued in culture or frozen.<br>
Example 2: Selecting Haploid Hybrids<br>
Scoring for the presence, absence and diploid/haploid state of human<br>
chromosomes in each hybrid was performed using the Affymetrix, HuSNP genechip<br>
(Affymetrix, Inc,. of Santa Clara, CA, HuSNP Mapping Assay, reagent kit and user<br>
manual, Affymetrix Part No. 900194), which can score 1494 markers in a single chip<br>
hybridization. As controls, the hamster and human diploid lymphoblastoid cell lines were<br>
screened using the HuSNP chip hybridization assay. Any SNPs which were heterozygous<br>
in the parent lymphoblastoid diploid cell line were scored for haploidy in each fusion cell<br>
line. Assume that "A" and "B" are alternative variants at each SNP location. By<br>
comparing the markers that were present as "AB" heterozygous in the parent diploid cell<br>
line to the same markers present as "A" or "B" (hemizygous) in the hybrids, the human<br>
DNA strands which were in the haploid state in each hybrid line was determined.<br>
Figure 11 shows results after two human/hamster cell hybrids (Hybrid 1 and<br>
Hybrid 2) are tested for selected markers on human chromosome 21. The first column<br>
lists the HuSNP chip marker designations. The second column reports whether a signal<br>
was obtained when the hamster cell nucleic acid (no fusion) was used for hybridization<br>
with a HuSNP chip. As expected, there was no signal for any marker in the hamster cell<br>
sample. The third column reports which variants for each marker were detected ("A",<br>
"B" or "AB") in the diploid parent human lymphoblastoid cell line, CPD17. In some<br>
instances, only an A variant was present, in some instances only a B variant was present,<br>
and in some cases the CPD17 cells were heterozygous ("AB") for the variants. The last<br>
two columns report the result when nucleic acid samples from two human/hamster<br>
hybrids (Hybrid 1 and Hybrid 2) are hybridized with the HuSNP chip. Note in cases<br>
where only A variants were present in the parent CPD17 cell line, only A variants were<br>
transferred in the fusion. In cases where only B variants were present in the parent<br>
CPD17 cell line, only B variants were transferred in the fusion. In cases where the<br>
CPD17 cell line was heterozygous, an A variant was transferred to some fusion clones,<br>
and a B variant was transferred to other fusion clones. It should be understood, however,<br>
that often only portions of chromosomes are present in the hybrid cell lines resulting from<br>
this fusion process, that some hybrids may be diploid for some human chromosomes or<br>
portions thereof, that some hybrids may be haploid for other human chromosomes or<br>
portions thereof, and some hybrids may not have either variant of some chromosomes.<br>
Hybrids containing only one variant of a particular human chromosome (for instance,<br>
chromosome 21) were selected for analysis. Even more preferably, hybrids containing a<br>
whole chromosome (as opposed to only a portion thereof) were selected for analysis.<br>
Example 3: Long Range PCR<br>
DNA from the hamster/human cell hybrids was used to perform long-range PCR<br>
assays. Long range PCR assays are known generally in the art and have been described,<br>
for example, in the standard long range PCR protocol from the Boehringer Mannheim<br>
Expand Long Range PCR Kit, incorporated herein by reference or all purposes.<br>
Primers used for the amplification reactions were designed in the following way: a<br>
given sequence, for example the 23 megabase contig on chromosome 21, was entered into<br>
a software program known in the art herein called "repeat masker" which recognizes<br>
sequences that are repeated in the genome (e.g., Alu and Line elements)(see, A. F. A.<br>
Smit and P. Green, www.genome.washington.edu/uwgc/analvsistools/repeatmask.<br>
incorporated herein by reference). The repeated sequences were "masked" by the<br>
program by substituting each specific nucleotide of the repeated sequence (A, T, G or C)<br>
with "N". The sequence output after this repeat mask substitution was then fed into a<br>
commercially available primer design program (Oligo 6.23) to select primers that were<br>
greater than 30 nucleotides in length and had melting temperatures of over 65 °C. The<br>
designed primer output from Oligo 6.23 was then fed into a program which then "chose"<br>
primer pairs which would PCR amplify a given region of the genome but have minimal<br>
overlap with the adjacent PCR products. The success rate for long range PCR using<br>
commercially available protocols and this primer design was at least 80%, and greater<br>
than 95% success was achieved on some portions of human chromosomes.<br>
An illustrative protocol for long range PCR uses the Expand Long Template PCR<br>
System from Boehringer Mannheim Cat.# 1681 834, 1681 842, or 1759 060. In the<br>
procedure each 50 µL PCR reaction requires two master mixes. In a specific example,<br>
Master Mix 1 was prepared for each reaction in 1.5 ml microfuge tubes on ice and<br>
includes a final volume of 19 µL of Molecular Biology Grade Water (Bio Whittaker,<br>
Cat.# 16-001Y); 2.5 µL 10 mM dNTP set containing dATP, dCTP, dGTP, and dTTP at<br>
10 mM each (Life Technologies Cat.# 10297-018) for a final concentration of 400 µM of<br>
each dNTP; and 50 ng DNA template.<br>
Master Mix 2 for all reactions was prepared and kept on ice. For each PCR<br>
reaction Master Mix 2 includes a final volume of 25 µL of Molecular Biology Grade<br>
Water (Bio Whittaker); 5 µL 10 x PCR buffer 3 containing 22.50 mM MgCl2 (Sigma,<br>
Cat.# M 10289); 2.5 µL 10 mM MgCl2 (for a final MgCl2 concentration of 2.75 mM);<br>
and 0.75 µL enzyme mix (added last)<br>
Six microliters of premixed primers (containing 2.5 µL of Master Mix 1) were<br>
added to appropriate tubes, then 25 µL of Master Mix 2 was added to each tube. The<br>
tubes were capped, mixed, centrifuged briefly and returned to ice. At this point, the PCR<br>
cycling was begun according to the following program: step 1: 94°C for 3 min to denature<br>
template; step 2: 94°C for 30 sec; step 3: annealing for 30 sec at a temperature appropriate<br>
for the primers used; step 4: elongation at 68°C for 1 min/kb of product; step 5: repetition<br>
of steps 2-4 38 times for a total of 39 cycles; step 6: 94°C for 30 sec; step 7: annealing for<br>
30 sec; step 8: elongation at 68°C for 1 min/kb of product plus 5 additional minutes; and<br>
step 9: hold at 4°C. Alternatively, a two-step PCR would be performed: step 1: 94°C for<br>
3 min to denature template; step 2: 94°C for 30 sec; step 3: annealing and elongation at<br>
68°C for 1 min/kb of product; step 4: repetition of steps 2-3 38 times for a total of 39<br>
cycles; step 5:94°C for 30 sec; step 6: annealing and elongation at 68°C for 1 min/kb of<br>
product plus 5 additional minutes; and step 7: hold at 4°C.<br>
Results of the long range PCR amplification reaction for various regions on<br>
human chromosomes 14 and 22 were visualized on ethidium bromide-stained agarose<br>
gels (Figure 12). The long range PCR amplification methods of the present invention<br>
routinely produced amplified fragments having an average size of about 8 kb, and<br>
appeared to fail to amplify genomic regions in only rare cases (see G11 on the<br>
chromosome 22 gel).<br>
Example 4: Wafer Design, Manufacture, Hybridization and Scanning<br>
The set of oligonucleotide probes to be contained on an oligonucleotide array<br>
(chip or wafer) was defined based on the human DNA strand sequence to be queried. The<br>
oligonucleotide sequences were based on consensus sequences reported in publicly<br>
available databases. Once the probe sequences were defined, computer algorithms were<br>
used to design photolithographic masks for use in manufacturing the probe-containing<br>
arrays. Arrays were manufactured by a light-directed chemical synthesis processes which<br>
combines solid-phase chemical synthesis with photolithographic fabrication techniques.<br>
See, for example, WO 92/10092, or U.S. Patent Nos. 5,143,854; 5,384,261; 5,405,783;<br>
5,412,087; 5,424,186; 5,445,934; 5,744,305; 5,800,992; 6,040,138; 6,040,193, all of<br>
which are incorporated herein by reference in their entireties for all purposes. Using a<br>
series of photolithographic masks to define exposure sites on the glass substrate (wafer)<br>
followed by specific chemical synthesis steps, the process constructed high-density areas<br>
of oligonucleotide probes on the array, with each probe in a predefined position. Multiple<br>
probe regions were synthesized simultaneously and in parallel.<br>
The synthesis process involved selectively illuminating a photo-protected glass<br>
substrate by passing light through a photolithographic mask wherein.chemical groups in<br>
unprotected areas were activated by the light. The selectively-activated substrate wafers<br>
were then incubated with a chosen nucleoside, and chemical coupling occurred at the<br>
activated positions on the wafer. Once coupling took place, a new mask pattern was<br>
applied and the coupling step was repeated with another chosen nucleoside. This process<br>
was repeated until the desired set of probes was obtained. In one specific example, 25-<br>
mer oligonucleotide probes were used, where the thirteenth base was the base to be<br>
queried. Four probes were used to interrogate each nucleotide present in each sequence-<br>
one probe complementary to the sequence and three mismatch probes identical to the<br>
complementary probe except for the thirteenth base. In some cases, at least 10 x 106<br>
probes were present on each array.<br>
Once fabricated, the arrays were hybridized to the products from the long range<br>
PCR reactions performed on the hamster-human cell hybrids. The samples to be analyzed<br>
were labeled and incubated with the arrays to allow hybridization of the sample to the<br>
probes on the wafer.<br>
After hybridization, the array was inserted into a confocal, high performance<br>
scanner, where patterns of hybridization were detected. The hybridization data were<br>
collected as light emitted from fluorescent reporter groups already incorporated into the<br>
PCR products of the sample, which was bound to the probes. Sequences present in the<br>
sample that are complimentary to probes on the wafer hybridized to the wafer more<br>
strongly and produced stronger signals than those sequences that had mismatches. Since<br>
the sequence and position of each probe on the array was known, by complementarity, the<br>
identity of the variation in the sample nucleic acid applied to the probe array was<br>
identified. Scanners and scanning techniques used in the present invention are known to<br>
those skilled in the art and are disclosed in, e.g., U.S. Patent No. 5,981,956 drawn to<br>
microarray chips, U.S. Patent No. 6,262,838 and U.S. Patent No. 5,459,325. U.S.S.N. In<br>
addition, 60/223,278 filed on August 3, 2000, and non-provisional application claiming<br>
priority to USSN 60/223,278 filed on August 3, 2001, drawn to scanners and techniques<br>
for whole wafer scanning, are also incorporated herein by reference in their entireties for<br>
all purposes.<br>
Example 5: Determination of SNP Haplotypes on Human Chromosome 21<br>
Twenty independent copies of chromosome 21, representing African, Asian, and<br>
Caucasian chromosomes were analyzed for SNP discovery and haplotype structure. Two<br>
copies of chromosome 21 from each individual were physically separated using a rodent-<br>
human somatic cell hybrid technique (Figure 10), discussed supra. The reference<br>
sequence for the analysis consisted of human chromosome 21 genomic DNA sequence<br>
consisting of 32,397,439 bases. This reference sequence was masked for repetitive<br>
sequences and the resulting 21,676,868 bases (67%) of unique sequence were assayed for<br>
variation with high density oligonucleotide arrays. Eight unique oligonucleotides, each<br>
25 bases in length, were used to interrogate each of the unique sample chromosome 21<br>
bases, for a total of 1.7 X 108 different oligonucleotides. These oligonucleotides were<br>
distributed over a total of eight different wafer designs using a previously described tiling<br>
strategy (Chee, et al., Science 274:610 (1996)). Light-directed chemical synthesis of<br>
oligonucleotides was carried out on 5 inch x 5 inch glass wafers purchased from<br>
Affymetrix, Inc. (Santa Clara, CA).<br>
Unique oligonucleotides were designed to generate 3253 minimally overlapping<br>
longe range PCR (LRPCR) products of 10 kb average length spanning 32.4 Mb of<br>
contiguous chromosome 21 DNA, and were prepared as described supra. For each wafer<br>
hybridization, corresponding LRPCR products were pooled and were purified using<br>
Qiagen tip 500 (Qiagen). A total of 280 µg of purified DNA was fragmented using 37 µl<br>
of 10X One-Phor-AU buffer PLUS (Promega) and 1 unit of DNAase (Life<br>
Technolgies/Invitrogen) in 370 µl total volume at 37°C for 10 min followed by heat<br>
inactivation at 99°C for 10 min. The fragmented products were end labeled using 500<br>
units of Tdt (Boehringer Manheim) and 20 nmoles of biotin-N6-ddATP (DuPont NEN) at<br>
37°C for 90 min and heat inactivated at 95°C for 10 min. The labeled samples were<br>
hybridized to the wafers in 10 mM Tris-HCL (pH 8), 3M Tetramethylammonium<br>
chloride, 0.01% Tx-100,10 µg/ml denatured herring sperm DNA in a total volume of 14<br>
ml per wafer at 50°C for 14-16 hours. The wafers were rinsed briefly in 4X SSPE,<br>
washed three times in 6X SSPE for 10 min each, stained using streptavidin R-<br>
phycoerythrin (SAPE, 5 ng/ml) at room temp for 10 min. The signal was amplified by<br>
staining with an antibody against streptavidin (1.25 ng/ml) and by repeating the staining<br>
step with SAPE.<br>
PCR products corresponding to the bases present on a single wafer were pooled<br>
and hybridized to the wafer as a single reaction. In total, 3.4 x 109 oligonucleotides were<br>
synthesized on 160 wafers to scan 20 independent copies of human chromosome 21 for<br>
DNA sequence variation. Each unique chromosome 21 was amplified from a rodent-<br>
human hybrid cell line by using long range PCR. LRPCR assays were designed using<br>
Oligo 6.23 primer design software with high-moderate stringency parameters. The<br>
resulting primers were typically 30 nucleotides in length with the melting temperature of<br>
&gt; 65°C. The range of amplicon size was from 3 kb-14 kb. A primer database for the<br>
entire chromosome was generated and software (pPicker) was utilized to choose a<br>
minimal set of non-redundant primers that yield maximum coverage of chromosome 21<br>
sequence with a minimal overlap between adjacent amplicons. Alternatively, the primer<br>
selection method described in Example 3, herein, was employed. LRPCR reactions were<br>
performed using the Expand Long Template PCR Kit (Boehringer Mannheim) with minor<br>
modifications. The wafers were scanned using a custom built confocal scanner.<br>
SNPs were detected as altered hybridization by using a pattern recognition<br>
algorithm. A combination of previously described algorithms (Wang, et al., Science<br>
280:1077 (1998)), was used to detect SNPs based on altered hybridization patterns. In<br>
total, 35,989 SNPs were identified in the sample of twenty chromosomes. The position<br>
and sequence of these human polymorphisms have been deposited in GenBank"s SNPdb.<br>
Dideoxy sequencing was used to assess a random sample of 227 of these SNPs in the<br>
original DNA samples, confirming 220 (97%) of the SNPs assayed. In order to achieve<br>
this low rate of 3% false positive SNPs, stringent thresholds were required for SNP<br>
detection on wafers that resulted in a high false negative rate. Approximately 65% of all<br>
bases present on the wafers yielded data of high enough quality for use in SNP detection<br>
with 35% being discarded as being false negatives. Consistent failure of long range PCR<br>
in all samples analyzed accounts for 15% of the 35% false negative rate. The remaining<br>
20% false negatives are distributed between bases that never yield high quality data<br>
(10%) and bases that yield high quality data in only a fraction of the 20 chromosomes<br>
analyzed (10%). In general, it is the sequence context of a base that dictates whether or<br>
not it will yield high quality data. The finding that approximately 20% of all bases give<br>
consistently poor data is very similar to the finding that approximately 30% of bases in<br>
single dideoxy sequencing reads of 500 bases have quality scores too low for reliable<br>
SNP detection (Altschuler, et al, Nature 407:513 (2000)). The power to discover rare<br>
SNPs as compared to more frequent SNPs is disproportionately reduced in cases where<br>
only a limited number of the samples analyzed yield high quality data for a given base.<br>
As a result, SNP discovery by this method is biased in favor of common SNPs.<br>
Figure 13A shows the distribution of minor allele frequencies of all 35,989 SNPs<br>
discovered in the sample of globally diverse chromosomes. Genetic variation,<br>
normalized for the number of chromosomes in the sample, was estimated with two<br>
measures of nucleotide diversity: n the average heterozygosity per site and 0 the<br>
population mutation parameter (see Hartl and Clark, Principles of Population Genetics<br>
(Sinauer, Massachusetts, 1997)). The 32,397,439 bases of finished genomic chromosome<br>
21 DNA were divided into 200,000 base pair segments, and the high-quality base pairs<br>
used for SNP discovery in each segment were examined. The observed heterozygosity of<br>
these bases was used to calculate an average nucleotide diversity (p) for each segment.<br>
The estimates of average nucleotide diversity for the total data set (p = 0.000723 and ? =<br>
0.000798), as well as the distribution of nucleotide diversity, measured in contiguous<br>
200,000 base pair bins of chromosome 21 (Fig. 13B), are within the range of values<br>
previously described (The International SNP Map Working Group, Nature 409:928-33<br>
(2001)).<br>
The extent of overlap of 15,549 chromosome 21 SNPs discovered by The SNP<br>
Consortium (TSC) was compared with the SNPs found in this study. Of the TSC SNPs,<br>
5,087 were found to be in repeated DNA and were not tiled on the wafers. Of the<br>
remaining 10,462 TSC SNPs, 4705 (45%) were identified. The estimate of 0 was<br>
observed to be greater than the estimate of p ? for 129 of the 162 200-kb bins of<br>
contiguous DNA sequence analyzed. This difference is consistent with a recent<br>
expansion of the human population and is similar to the finding of a recent study of<br>
nucleotide diversity in human genes (Stephens, et al., Science 293:489 (2001)). It was<br>
found that 11,603 of the SNPs (32%) had a minor allele observed a single time in the<br>
sample (singletons), as compared with the neutral model expectation of 43% singletons<br>
given the observed amount of nucleotide diversity (Fu and Li, Genetics 133:693 (1993)).<br>
The difference between the observed and expected values is likely attributable to the<br>
reduced power to identify rare as compared to common SNPs in this study as discussed<br>
above.<br>
Over all, 47% of the 53,000 common SNPs with an allele frequency of 10% or<br>
greater estimated to be present in 32.4 Mb of the human genome were identified. This<br>
compares with an estimate of 18-20% of all such common SNPs present in the collection<br>
generated by the International SNP Mapping Working Group and the SNP Consortium.<br>
The difference in coverage is explained by the fact that the present study used larger<br>
numbers of chromosomes for SNP discovery. To assess the replicability of the findings,<br>
SNP discovery was performed for one wafer design with nineteen additional copies of<br>
chromosome 21 derived from the same diversity panel as the original set of samples. A<br>
total of 7188 SNPs were identified using the two sets of samples. On average, 66% of all<br>
SNPs found in one set of samples were discovered in the second set, consistent with<br>
previous findings (Marth, et al., Nature Genet. 27:371 (2001) and Yang, et al., Nature<br>
Genet. 26:13 (2000)). As expected, failure of a SNP to replicate in a second set of<br>
samples is strongly dependent on allele frequency. It was found that 80% of SNPs with a<br>
minor allele present two or more times in a set of samples were also found in a second set<br>
of samples, while only 32% of SNPs with a minor allele present a single time were found<br>
in a second set of samples. These findings suggest that the 24,047 SNPs in the collection<br>
with a minor allele represented more than once are highly replicable in different global<br>
samples and that this set of SNPs is useful for defining common global haplotypes. In the<br>
course of SNP discovery, 339 SNPs which appeared to have more than two alleles were<br>
identified. These SNPs were not included in the present analysis.<br>
In addition to the replicability of SNPs in different samples, the distance between<br>
consecutive SNPs in a collection of SNPs is critical for defining meaningful haplotype<br>
structure. Haplotype blocks, which can be as short as several kb, may go unrecognized if<br>
the distance between consecutive SNPs in a collection is large relative to the size of the<br>
actual haplotype blocks. The collection of SNPs in this study was very evenly distributed<br>
across the chromosome, even though repeat sequences were not included in the SNP<br>
discovery process. Figure 13C shows the distribution of SNP coverage across 32,397,439<br>
bases of finished chromosome 21 DNA sequence. An interval is the distance between<br>
consecutive SNPs. There are a total of 35,988 intervals for the entire SNP set and a total<br>
of 24,046 intervals for the common SNP set (i.e. SNPs with a minor allele present more<br>
than once in the sample). The average distance between consecutive SNPs was 900 bases<br>
when all SNPs are considered, and 1300 bases when only the 24,047 common SNPs were<br>
considered. For this set of common SNPs, 93% of intervals between consecutive SNPs in<br>
genomic DNA, including repeated DNA, were 4000 bases or less (again, see Figure 13C).<br>
The construction of haplotype blocks or patterns from diploid data is complicated<br>
by the fact that the relationship between alleles for any two heterozygous SNPs is not<br>
directly observable. Consider an individual with two copies of chromosome 21 and two<br>
alleles, A and G, at one chromosome 21 SNP, as well as two alleles, A and G, at a second<br>
chromosome 21 SNP. In such a case, it is unclear if one copy of chromosome 21 contains<br>
allele A at the first SNP and allele A at the second SNP, while the other copy of<br>
chromosome 21 contains allele G at the first SNP and allele G at the second SNP, or if<br>
one copy of chromosome 21 contains allele A at the first SNP and allele G at the second<br>
SNP, while the other copy of chromosome 21 contains allele G at the first SNP and allele<br>
A at the second SNP. Current methods used to circumvent this problem include statistical<br>
estimation of haplotype frequencies, direct inference from family data, and allele-specific<br>
PCR amplification over short segments.<br>
To avoid these complexities, the present invention characterized SNPs on haploid<br>
copies of chromosome 21 isolated in rodent-human somatic cell hybrids were<br>
characterized, allowing direct determination of the full haplotypes of these chromosomes.<br>
The set of 24,047 SNPs with a minor allele represented more than once in the data set was<br>
used to define the haplotype structure are shown in Figure 14. The haplotype patterns for<br>
twenty independent globally diverse chromosomes defined by 147 common human<br>
chromosome 21 SNPs is shown. The 147 SNPs span 106 kb of genomic DNA sequence.<br>
Each row of colored boxes represents a single SNP. The black boxes in each row<br>
represent the major allele for that SNP, and the white boxes represent the minor allele.<br>
Absence of a box at any position in a row indicates missing data. Each column of colored<br>
boxes represents a single chromosome, with the SNPs arranged in their physical order on<br>
the chromosome. Invariant bases between consecutive SNPs are not represented in the<br>
figure. The 147 SNPs are divided into eighteen blocks, defined by black horizontal lines.<br>
The position of the base in chromosome 21 genomic DNA sequence defining the<br>
beginning of one block and the end of the adjacent block is indicated by the numbers to<br>
the left of the vertical black line. The expanded boxes on the right of the figure represent<br>
a SNP block defined by 26 common SNPs spanning 19 kb of genomic DNA. Of the<br>
seven different haplotype patterns represented in the sample, the four most common<br>
patterns include sixteen of the twenty chromosomes sampled (i.e. 80% of the sample).<br>
The black and white circles indicate the allele patterns of two informative SNPs, which<br>
unambiguously distinguish between the four common haplotypes in this block. Although<br>
no two chromosomes shared an identical haplotype pattern for these 147 SNPs, there are<br>
numerous regions in which multiple chromosomes shared a common pattern. One such<br>
region, defined by 26 SNPs spanning 19 kb, is expanded for more detailed analysis<br>
(again, see the enlarged region of Figure 14). This block defines seven unique haplotype<br>
patterns in 20 chromosomes. Despite the fact that some data is missing due to failure to<br>
pass the threshold for data quality, in all cases a given chromosome can be assigned<br>
unambiguously to one of the seven haplotypes. The four most frequent haplotypes, each<br>
of which is represented by three or more chromosomes, account for 80% of all<br>
chromosomes in the sample. Only two "informative" SNPs out of the total of twenty-six<br>
are required to distinguish the four most frequent haplotypes from one another. In this<br>
example, four chromosomes with infrequent haplotypes would be incorrectly classified as<br>
common haplotypes by using information from only these two informative SNPs.<br>
Nevertheless, it is remarkable that 80% of the haplotype structure of the entire global<br>
sample is defined by less than 10% of the total SNPs in the block. Several different<br>
possibilities exist in which three informative SNPs can be chosen so that each of the four<br>
common haplotypes is defined uniquely by a single SNP. One of these "three SNP"<br>
choices would be preferred over the two SNP combination in an experiment involving<br>
genotyping of pooled samples, since the two SNP combination would not permit<br>
determination of frequencies of the four common haplotypes in such a situation; thus, the<br>
present invention provides a dramatic improvement over the random selection method of<br>
SNP mapping.<br>
In summary, while the particular application may dictate the selection of<br>
informative SNPs to capture haplotype information, it is clear that the majority of the<br>
haplotype information in the sample is contained in a very small subset of all the SNPs. It<br>
is also clear that random selection of two or three informative SNPs from this block of<br>
SNPs will often not provide enough information to uniquely assign a chromosome to one<br>
of the four common haplotypes.<br>
One issue is how to define a set of contiguous blocks of SNPs spanning the entire<br>
32.4 Mb of chromosome 21 while minimizing the total number of SNPs required to<br>
define the haplotype structure. In one embodiment, an optimization algorithm based on a<br>
"greedy" strategy was used to address this problem. All possible blocks of physically<br>
consecutive SNPs of size one SNP or larger were considered. Ambiguous haplotype<br>
patterns were treated as missing data and were not included when calculating percent<br>
coverage. Considering the remaining overlapping blocks simultaneously, the block with<br>
the maximum ratio of total SNPs in the block to the minimal number of SNPs required to<br>
uniquely discriminate haplotypes represented more than once in the block was selected.<br>
Any of the remaining blocks that physically overlapped with the selected block were<br>
discarded, and the process was repeated until a set of contiguous, non-overlapping blocks<br>
that cover the 32.4Mb of chromosome 21 with no gaps, and with every SNP assigned to a<br>
block, was selected. Given the sample size of twenty chromosomes, the algorithm<br>
produces a maximum often common haplotype patterns per block, each represented by<br>
two independent chromosomes.<br>
Applying this algorithm to the data set of 24,047 common SNPs, 4135 blocks of<br>
SNPs spanning chromosome 21 were defined. A total of 589 blocks, comprising 14% of<br>
all blocks, contain greater than ten SNPs per block and include 44% of the total 32.4 Mb.<br>
In contrast, 2138 blocks, comprising 52% of all blocks, contain less than three SNPs per<br>
block and make up only 20% of the physical length of the chromosome. The largest<br>
block contains 114 common SNPs and spans 115 kb of genomic DNA. Overall, the<br>
average physical size of a block is 7.8 kb. The size of a block is not correlated with its<br>
order on the chromosome, and large blocks are interspersed with small blocks along the<br>
length of the chromosome. There are an average of 2.7 common haplotype patterns per<br>
block, defined as haplotype patterns that are observed on multiple chromosomes. On<br>
average, the most frequent haplotype pattern in a block is represented by 9.6<br>
chromosomes out of the twenty chromosomes in the sample, the second most frequent<br>
haplotype pattern is represented by 4.2 chromosomes, and the third most frequent<br>
haplotype patterns, if present, is represented by 2.1 chromosomes. The fact that such a<br>
large fraction of globally diverse chromosomes are represented by such limited haplotype<br>
diversity is remarkable. The findings are consistent with the observation that when<br>
haplotype pattern frequency is considered, 82% of the haplotype patterns observed in a<br>
collection of 313 human genes are observed in all ethnic groups, while only 8% of<br>
haplotypes are population specific (Stephens, et al., Science 293:489-93 (2001)).<br>
Several experiments were performed to measure the influence of parameters of the<br>
haplotype algorithm on the resulting block patterns. The fraction of chromosomes<br>
required to be covered by common haplotypes was varied, from an initial 80%, to 70%<br>
and 90%. As would be expected, requiring more complete coverage results in somewhat<br>
larger numbers of shorter blocks. Using only the 16,503 SNPs with a minor allele<br>
frequency of at least 20% in the sample resulted in somewhat longer blocks, but the<br>
numbers of SNPs per block did not change significantly. For one region of about 3 Mb, a<br>
deeper sample of 38 chromosomes for SNPs and common haplotype blocks with at least<br>
10% frequency was analyzed, so as to be comparable with the 20 chromosome analysis.<br>
The resulting distribution of block sizes closely matched the initial results. Also, a<br>
randomization test was performed in which the non-ambiguous alleles at each SNP were<br>
permuted, and then used for haplotype block discovery. In this analysis, 94% of blocks<br>
contained fewer than three SNPs, and only one block contained more than five SNPs.<br>
This confirms that the larger blocks seen in the data cannot be produced by chance<br>
associations or as artifacts of the block selection methods of the present invention.<br>
In an effort to determine if genes were proportionately represented in both large<br>
and small blocks, a determination was made of the number of exonic bases in blocks<br>
containing more than 10 SNPs, 3 to 10 SNPs, and less than 3 SNPs. Exonic bases are<br>
somewhat over-represented as compared to total bases in blocks containing 3 to 10 SNPs<br>
(p
Based on knowledge of the haplotype structure within blocks, subsets of the<br>
24,047 common SNPs can be selected to capture any desired fraction of the common<br>
haplotype information, defined as complete information for haplotypes present more than<br>
once and including greater than 80% of the sample across the entire 32.4 Mb. Figure 15<br>
shows the number of SNPs required to capture the common haplotype information for<br>
32.4 Mb of chromosome 21. For each SNP block, the minimum number of SNPs<br>
required to unambiguously distinguish haplotypes in that block that are present more than<br>
once (i.e., common haplotype information) was determined. These SNPs provide<br>
common haplotype information for the fraction of the total physical distance defined by<br>
that block. Beginning with the SNPs that provide common haplotype information for the<br>
greatest physical distance, the cumulative increase in physical coverage (i.e., fraction<br>
covered) is plotted relative to the number of SNPs added (i.e., SNPs required). Genie<br>
DNA includes all genomic DNA beginning 10 kb 5" of the first exon of each known<br>
chromosome 21 gene and extending 10 kb 3" of the last exon of that gene. For example,<br>
while a minimum of 4563 SNPs are required to capture all the common haplotype<br>
information, only 2793 SNPs are required to capture the common haplotype information<br>
in blocks containing three or more SNPs that cover 81% of the 32.4 Mb. A total of 1794<br>
SNPs are required to capture all the common haplotype information in genie DNA,<br>
representing approximately two hundred and twenty distinct genes.<br>
The present invention has particular relevance for whole-genome association<br>
studies mapping phenotypes such as common disease genes. This approach relies on the<br>
hypothesis that common genetic variants are responsible for susceptibility to common<br>
diseases (Risch and Merikangas, Science 273:1516 (1996), Lander, Science 274:536<br>
(1996)). By comparing the frequency of genetic variants in unrelated cases and controls,<br>
genetic association studies can identify specific haplotypes in the human genome that<br>
play important roles in disease. While this approach has been used to successfully<br>
associate single candidate genes with disease (Altschuler, et al., Nature Genet. 26:76<br>
(2000)), the recent availability of the human DNA sequence offers the possibility of<br>
surveying the entire genome, dramatically increasing the power of genetic association<br>
analysis (Kruglyak, Nature Genet. 22:139 (1999)). A major limitation to the<br>
implementation of this method has been lack of knowledge of the haplotype structure of<br>
the human genome, which is required in order to select the appropriate genetic variants<br>
for analysis. The present invention demonstrates that high-density oligonucleotide arrays<br>
in combination with somatic cell genetic sample preparation provide a high-resolution<br>
approach to empirically define the common haplotype structure of the human genome.<br>
Although the length of genomic regions with a simple haplotype structure is<br>
extremely variable, a dense set of common SNPs enables the systematic approach to<br>
define blocks of the human genome in which 80% of the global human population is<br>
described by only three common haplotypes. In general, when applying the particular<br>
algorithm used in this embodiment, the most common haplotype in any block is found in<br>
50% of individuals, the second most common in 25% of individuals, and the third most<br>
common in 12.5% of individuals. It is important to note that blocks are defined based on<br>
their genetic information content and not on knowledge of how this information<br>
originated or why it exists. As such, blocks do not have absolute boundaries, and may be<br>
defined in different ways, depending on the specific application. The algorithm in this<br>
embodiment provides only one of many possible approaches. The results indicate that a<br>
very dense set of SNPs is required to capture all the common haplotype information.<br>
Once in hand, however, this information can be used to identify much smaller subsets of<br>
SNPs useful for comprehensive whole-genome association studies.<br>
Those skilled in the art will appreciate readily that the techniques applied to<br>
human chromosome 21 can be applied to all the chromosomes present in the human<br>
genome. In a preferred embodiment of the present invention, multiple whole genomes of<br>
a diverse population representative of the human species are used to identify SNP<br>
haplotype blocks common to all or most members of the species. In some embodiments,<br>
SNP haplotype blocks are based on ancient SNPs by excluding SNPs that are represented<br>
at low frequency. The ancient SNPs are likely to be important as they have been<br>
preserved in the genome because they impart some selective benefit to organisms<br>
carrying them.<br>
Example 6: Using Associated Genes for Gene Therapy and Drug Discovery <br>
One example for using the methods of the present invention is outlined in this<br>
prophetic example. SNP discovery is performed on twenty haploid genomes, and fifty<br>
haploid genomes are analyzed by the methods of the present invention to determine SNP<br>
haplotype blocks, SNP haplotype patterns, informative SNPs and minor allele frequency<br>
for each informative SNP. These fifty haploid genomes comprise the control genomes of<br>
the present study (see step 1300 of Figure 13).<br>
Next, genomic DNA from 500 individuals having an obesity phenotype are<br>
assayed for variants by using long distance PCR and microarrays as described supra (see<br>
also, United States Patent No. 6,300,063 issued to Lipshutz, et ai, and United States<br>
Patent No. 5,837,832 to Chee, et al), and the frequency of the minor allele for each<br>
informative SNP is determined for this clinical population (see step 1310 of Figure 13).<br>
The minor allele frequencies of the informative SNPs for the two populations are<br>
compared, and the control and clinical populations are determined to have statistically<br>
significant differences in three informative SNP locations (steps 1320 and 1330). The<br>
SNP location with the largest difference in the minor allele frequency between the control<br>
and clinical populations is selected for analysis.<br>
The informative location selected is contained within a SNP haplotype block that<br>
is found to span 1 kb of noncoding sequence 5" of the coding region and 4 kb of the<br>
coding region of the leptin gene (step 1340). Analysis of the variations contained within<br>
this region indicates that a G at one SNP position in this region is responsible for<br>
destruction of the promoter for the leptin gene, with a commensurate lack of expression<br>
of the leptin protein.<br>
Fibroblasts are obtained from a subject by skin biopsy. The resulting tissue is<br>
placed in tissue-culture medium and separated into small pieces. Small pieces of the<br>
tissue are placed on the bottom of a wet surface of a tissue culture flask with medium.<br>
After 24 hours at room temperature, fresh media is added (e.g., Ham"s F12 media, with<br>
10% FBS, penicillin and streptomycin). The tissue is then incubated at 37°C for<br>
approximately one week. At this time, fresh media is added and subsequently changed<br>
every several days. After an additional two weeks in culture, a monolayer of fibroblasts<br>
emerges. The monolayer is trypsinized and scaled into larger flasks.<br>
The vector derived from the Moloney murine leukemia virus, which contains a<br>
kanamycin resistance gene, is digested with restriction enzymes for cloning a fragment to<br>
be expressed. The digested vector is treated with calf intestinal phosphatase to prevent<br>
self-ligation. The dephosphorylated, linear vector is fractionated on an agarose gel and<br>
purified. Leptin cDNA, capable of expressing active leptin protein product, is isolated.<br>
The ends of the fragment are modified, if necessary, for cloning into the vector. Equal<br>
molar quantities of the Moloney murine leukemia virus linear backbone and the leptin<br>
gene fragment are mixed together and joined using T4 DNA ligase. The ligation mixture<br>
is used to transform E. coli and the bacteria are then plated onto agar-containing<br>
kanamycin. Kanamycin phenotype and restriction analysis confirm that the vector has the<br>
properly inserted leptin gene.<br>
Packaging cells are grown in tissue culture to confluent density in Dulbecco"s<br>
Modified Eagles Medium (DMEM) with 10% calf serum, penicillin and streptomycin.<br>
The vector containing the leptin gene is introduced into the packaging cells by standard<br>
techniques. Fresh media is added to the packaging cells, and after an appropriate<br>
incubation period, media is harvested from the plates of confluent packaging cells. The<br>
media, containing the infectious viral particles, is filtered through a Millipore filter to<br>
remove detached packaging cells, then is used to infect fibroblast cells. Media is<br>
removed from a sub-confluent plate of fibroblasts and quickly replaced with the filtered<br>
media. Polybrene (Aldrich) may be included in the media to facilitate transduction. After<br>
appropriate incubation, the media is removed and replaced with fresh media. If the titer of<br>
virus is high, then virtually all fibroblasts will be infected and no selection is required. If<br>
the titer is low, then it is necessary to use a retroviral vector that has a selectable marker,<br>
such as neo or his, to select out transduced cells for expansion.<br>
Engineered fibroblasts then are introduced into individuals, either alone or after<br>
having been grown to confluence on microcarrier beads, such as cytodex 3 beads. The<br>
injected fibroblasts produce leptin product, and the biological actions of the protein are<br>
conveyed to the host.<br>
Alternatively or in addition, the leptin gene is isolated, cloned into an expression<br>
vector and employed for producing leptin polypeptides. The expression vector contains<br>
suitable transcriptional and translational initiation regions, and transcriptional and<br>
translational termination regions, as disclosed supra. Isolated leptin protein can be<br>
produced in this manner and used to identify agents which bind it; alternatively cells<br>
expressing the engineered leptin gene and protein are used in assays to identify agents.<br>
Such agents are identified by, for example, contacting a candidate agent with an isolated<br>
leptin polypeptide for a time sufficient to form a polypeptide/compound complex, and<br>
detecting the complex. If a polypeptide/compound complex is detected, the compound<br>
that binds to the leptin polypeptide is identified. Agents identified via this method can<br>
include compounds that modulate activity of leptin. Agents screened in this manner are<br>
peptides, carbohydrates, vitamin derivatives, and other small molecules or pharmaceutical<br>
agents. In addition to biological assays to identify agents, agents may be pre-screened by<br>
choosing candidate agents selected by using protein modeling techniques, based on the<br>
configuration of the leptin protein.<br>
In addition to identifying agents that bind the leptin protein, sequence-specific or<br>
element-specific agents that control gene expression through binding to the leptin gene<br>
are also identified. One class of nucleic acid binding agents are agents that contain base<br>
residues that hybridize to leptin mRNA to block translation (e.g., antisense<br>
oligonucleotides). Another class of nucleic acid binding agents are those that form a<br>
triple helix with DNA to block transcription (triplex oligonucleotides). Such agents<br>
usually contain 20 to 40 bases, are based on the classic phosphodiester, ribonucleic acid<br>
backbone, or can be a variety of sulfhydryl or polymeric derivatives that have base<br>
attachment capacity.<br>
Additionally, allele-specific oligonucleotides that hybridize specifically to the<br>
leptin gene and/or agents that bind specifically to the variant leptin protein (e.g., a<br>
variant-specific antibody) can be used as diagnostic agents. Methods for preparing and<br>
using allele-specific oligonucleotides and for preparing antibodies are described supra<br>
and are known in the art.<br>
All patents and publications mentioned in this specification are indicative of the<br>
levels of those skilled in the art to which the invention pertains. All patents and<br>
publications are herein incorporated by reference to the same extent as if each individual<br>
publication was specifically and individually indicated to be incorporated by reference.<br>
The present invention provides greatly improved methods for conducting genome-<br>
wide association studies by identifying individual variations, determining SNP haplotype<br>
blocks, determining haplotype patterns and, further, using the SNP haplotype patterns to<br>
identify informative SNPs. The informative SNPs may be used to dissect the genetic<br>
bases of disease and drug response in a practical and cost effective manner unknown<br>
previously. It is to be understood that the above description is intended to be illustrative<br>
and not restrictive. Many embodiments will be apparent to those skilled in the art upon<br>
reviewing the above description. The scope of the invention should, therefore, be<br>
determined not with reference to the above description, but should instead be determined<br>
with reference to the appended claims, along with the full scope of equivalents to which<br>
such claims are entitled.<br>
WE CLAIM:<br>
1. A method for selecting SNP haplotype patterns, comprising:<br>
(a) isolating a substantially identical nucleic acid strand from a plurality of<br>
different origins for analysis;<br>
(b) determining more than one SNP location in each nucleic acid strand;<br>
(c) identifying SNP locations in said nucleic acid strands that are linked, wherein<br>
said SNP locations that are linked form a SNP haplotype block thereby identifying said<br>
SNP haplotype block;<br>
(d) identifying isolate SNP haplotype blocks which have only one SNP therein;<br>
(e) identifying SNP haplotype patterns that occur in the SNP haplotype block identified in<br>
step (c) and the isolate SNP haplotype block identified in step (d); and<br>
(f) selecting each identified SNP haplotype pattern that occurs in at least two of<br>
said substantially identical nucleic acid strands.<br>
2. The method as claimed in claim 1, wherein the step of identifying SNPs that are<br>
linked is determined by a greedy algorithm or a shortest-paths algorithm.<br>
3. The method as claimed in claim 1, wherein said SNP haplotype block identified in<br>
step (c) and said isolate SNP haplotype block identified in step (d) are non-overlapping.<br>
4. The method as claimed in claim 1, wherein said plurality of different origins is at<br>
least between 10 to 100 different origins.<br>
5. The method as claimed in claim 4, wherein said plurality of different origins is at<br>
least 16 different origins.<br>
6. The method as claimed in claim 5, wherein the plurality of different origins is at<br>
least 25 different origins.<br>
7. The method as claimed in claim 6, wherein the plurality of different origins is at<br>
least 50 different origins.<br>
8. The method as claimed in claim 1, wherein said substantially identical nucleic<br>
acid strands are genomic DNA strands.<br>
9. The method as claimed in claim 1, wherein at least one of the plurality of different<br>
origins comprises at least ten percent of genomic DNA from an organism is isolated and<br>
analyzed.<br>
10. The method as claimed in claim 1, wherein at least 1 x 108 bases from said<br>
substantially identical nucleic acid strands are isolated and analyzed.<br>
11. The method as claimed in claim 1, wherein selected repeat regions from said<br>
substantially identical nucleic acid strands are not analyzed.<br>
12. The method as claimed in claim 1, further comprising:<br>
after step (b),<br>
excluding from analysis those SNP locations that occur only once in said<br>
substantially identical nucleic acid strands.<br>
13. The method as claimed in claim 1, further comprising:<br>
(g) selecting a SNP haplotype pattern that occurs most frequently in said substantially<br>
identical nucleic acid strands; and<br>
(h) selecting a SNP haplotype pattern that occurs next most frequently in said<br>
substantially identical nucleic acid strands; and<br>
(i) repeating step (h) until the SNP haplotype patterns selected in step (g) and step (h)<br>
characterise a portion of said substantially identical nucleic acid strands.<br>
14. The method as claimed in claim 13, wherein said portion is between 70% and<br>
99% of said substantially identical nucleic acid strands.<br>
15. The method as claimed in claim 14, wherein said portion is at least 80% of said<br>
substantially identical nucleic acid strands.<br>
16. The method as claimed in claim 13, wherein no more than three SNP haplotype<br>
patterns are selected from each of the SNP haplotype blocks identified in step (c) and the<br>
isolate SNP block identified in step (d).<br>
17. A method as claimed in claim 1 for selecting a data set of SNP haplotype blocks<br>
for characterising a region of interest in a nucleic acid strand, comprising:<br>
(a) comparing SNP haplotype blocks for informativeness wherein informativeness<br>
is the degree to which a SNP ahplotype block provides information about genetic regions;<br>
(b) selecting a first SNP haplotype block with a high informativeness;<br>
(c) adding said first SNP haplotype block to said data set;<br>
(d) selecting a second SNP haplotype block with a high informativeness;<br>
(e) adding said second selected SNP haplotype block to said data set; and<br>
(f) repeating said selecting and adding steps until a region of interest of a nucleic<br>
acid strand is covered.<br>
18. The method as claimed in claim 17, wherein the SNP haplotype blocks selected in<br>
step (b) are not overlapping.<br>
19. The method as claimed in claim 17, wherein the selecting of step (b) is<br>
determined by a greedy algorithm.<br>
20. A method for determining an informative SNP in a SNP haplotype pattern,<br>
comprising:<br>
(a) identifying SNP haplotype patterns in a SNP haplotype block according to the<br>
method as claimed in any of claims 1 to 16;<br>
(b) selecting SNP haplotype patterns of interest from amongst the SNP haplotype<br>
patterns identified in step (a);<br>
(c) comparing SNP haplotype patterns of interest in said SNP haplotype block;<br>
and<br>
(d) determining at least one SNP in a first SNP haplotype pattern of interest<br>
selected in step (b) that distinguishes one SNP haplotype pattern of interest from other<br>
SNP haplotype patterns of interest in said SNP haplotype block,<br>
wherein the at least one SNP is an informative SNP for said first SNP haplotype pattern<br>
in said SNP haplotype block.<br>
21. The method as claimed in claim 20, further comprising repeating step (d) until a<br>
sufficient number of informative SNPs are selected to distinguish a portion of SNP<br>
haplotype patterns of interest in said SNP haplotype block.<br>
22. The method as claimed in claim 21, wherein said selected portion of SNP<br>
haplotype patterns constitutes 70% to 99% of SNP haplotype patterns in said SNP<br>
haplotype block.<br>
23. The method as claimed in claim 21, wherein said selected portion of SNP<br>
haplotype patterns is associated with a disease of interest.<br>
24. The method as claimed in any of claims 17 to 19, wherein the informativeness of a<br>
SNP haplotype block is determined by:<br>
(a) determining a number of SNP locations in said SNP haplotype block;<br>
(b) determining a number of informative SNPs required to distinguish SNP haplotype<br>
patterns of interest in said SNP haplotype block; and<br>
(c) dividing said number of SNP locations by said number of informative SNPs to<br>
produce a quotient, wherein said quotient is said informativeness of said SNP haplotype<br>
block.<br>
25. The method as claimed in any of claims 17 to 19 wherein the informativeness of a<br>
SNP haplotype block is determined by:<br>
(a) determining a number of SNP locations in said SNP haplotype block; and<br>
(b) determining a number of informative SNPs required to distinguish SNP haplotype<br>
patterns of interest in said SNP haplotype block from each other,<br>
wherein said number of informative SNPs required to distinguish SNP haplotype patterns<br>
of interest is said informativeness of said SNP haplotype block.<br>
26. A method for determining disease-related genetic loci without a priori<br>
knowledge of a sequence or location of said disease-related genetic loci, comprising:<br>
(a) determining SNP haplotype patterns from individuals in a control population<br>
according to the method as claimed in any of claims 1 to 16;<br>
(b) determining SNP haplotype patterns from individuals in a diseased population;<br>
and<br>
(c) comparing frequencies of said SNP haplotype patterns of said control<br>
population with frequencies of said SNP haplotype patterns of said diseased population,<br>
wherein differences in said frequencies indicate locations of disease-related genetic loci.<br>
27. The method as claimed in claim 26, wherein said SNP haplotype patterns are<br>
determined in at least 50 individuals in a control population.<br>
28. The method as claimed in claim 26, wherein said SNP haplotype patterns from<br>
said populations are determined using informative SNPs.<br>
29. A method of making associations between SNP haplotype patterns and a<br>
phenotypic trait of interest comprising:<br>
(a) determining SNP haplotype patterns from a control population using the methods as<br>
claimed in any of claims 1 to 16;<br>
(b) pooling whole genomic DNA from a population having a common phenotypic trait of<br>
interest; and<br>
(c) identifying said SNP haplotype patterns that are associated with said phenotypic trait<br>
of interest.<br>
30. The method as claimed in claim 29, wherein informative SNPs are used for<br>
steps (a) and (c).<br>
31. A method of identifying diagnostic markers comprising;<br>
identifying informative SNPs according to the method as claimed in claim 20, wherein<br>
informative SNPs that are associated with a disease condition are diagnostic markers.<br>
32. A method for identifying drug discovery targets comprising:<br>
(a) identifying SNP haplotype patterns in a genome according to the method as claimed in<br>
any of claims 1 to 16 wherein said genome is associated with a disease;<br>
(b) identifying a chromosomal location of said SNP haplotype patterns;<br>
(c) determining association between said chromosomal location and said disease; and<br>
wherein the chromosomal location or expression product of the chromosomal location is<br>
a drug discovery target.<br>
33. The method as claimed in claim 32, wherein the chromosomal locations are<br>
prioritized as drug discovery targets based on a set of criteria that includes location in a<br>
highly conserved region and location in an intergenic region.<br>
34. The method as claimed in claim 32, wherein informative SNPs form the<br>
identifying of step (a).<br>
35. A kit for diagnosis of a disease, disease susceptibility, or therapy response<br>
comprising means for detecting a presence or absence of SNP haplotype patterns or<br>
informative SNPs in a sample of genomic DNA from a patient and a data set of<br>
associations of said SNP haplotype patterns or informative SNPs with one or more<br>
specific phenotypic traits on a computer-readable medium.<br>
36. A method for determining pharmacogenomic-related genetic loci without prior<br>
knowledge of a sequence or location of said pharmacogenomic-related genetic loci,<br>
comprising:<br>
(a) determining SNP haplotype patterns from individuals in a control population<br>
according to the method as claimed in any one of claims 1 to 16;<br>
(b) determining SNP haplotype patterns from individuals that react in an altered<br>
manner to administration of a substance; and<br>
(c) comparing frequencies of said SNP haplotype patterns of said control<br>
population with frequencies of said SNP haplotype patterns of said individuals that react<br>
in an altered manner to administration of a substance, wherein differences in said<br>
frequencies indicate locations of pharmacogenomic-related genetic loci.<br>
37. The method as claimed in claim 36, wherein the SNP haplotype patterns are<br>
determined in at least 50 individuals in a control population.<br>
38. The method as claimed in claim 36, wherein the SNP haplotype patterns from said<br>
populations are determined using informative SNPs, wherein informative SNPs are SNPs<br>
that distinguish one SNP haplotype pattern from other SNP haplotype patterns within a<br>
SNP haplotype block.<br>
The present invention relates to methods for identifying variations that occur in<br>
the human genome and relating these variations to the genetic basis of disease and drug<br>
response. In particular, the present invention relates to identifying individual SNPs,<br>
determining SNP haplotype blocks and patterns, and, further, using the SNP haplotype<br>
blocks and patterns to dissect the genetic bases of disease and drug response. The<br>
methods of the present invention are useful in whole genome analysis.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">00183-cal-2002-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">00183-cal-2002-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">00183-cal-2002-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">00183-cal-2002-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">00183-cal-2002-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">00183-cal-2002-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItbGV0dGVyIHBhdGVudC5wZGY=" target="_blank" style="word-wrap:break-word;">00183-cal-2002-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">00183-cal-2002-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItcHJpb3JpdHkgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">00183-cal-2002-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDAxODMtY2FsLTIwMDItcmVwbHkgZi5lLnIucGRm" target="_blank" style="word-wrap:break-word;">00183-cal-2002-reply f.e.r.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtbGV0dGVyIHBhdGVudC5wZGY=" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTgzLWNhbC0yMDAyLWdyYW50ZWQtdHJhbnNsYXRlZCBjb3B5IG9mIHByaW9yaXR5IGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">183-cal-2002-granted-translated copy of priority document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216871-an-apparatus-for-stripping-dirt-off-the-belts-in-a-conveyor-belt-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216873-a-process-of-preparation-of-nanosized-metal-oxide-particles.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216872</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>183/CAL/2002</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Mar-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PERLEGEN SCIENCES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2021 STIERLIN COURT, MOUNTAIN VIEW, CALIFORNIA (CA) 94043</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PATIL NILA</td>
											<td>C/O. PERLEGEN SCIENCES INC., 2021 STIERLIN COURT MOUNTAIN VIEW, CALIFORNIA (CA) 94043</td>
										</tr>
										<tr>
											<td>2</td>
											<td>COX DAVID R</td>
											<td>-DO-</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BERNO ANTHONY J</td>
											<td>-DO-</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HINDS DAVID A</td>
											<td>-DO-</td>
										</tr>
										<tr>
											<td>5</td>
											<td>FODOR STEPHEN P.A.</td>
											<td>-DO-</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/280,530</td>
									<td>2001-03-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216872-methods-for-genomic-analysis-and-a-kit-therefor by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:07:45 GMT -->
</html>
