<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216886-a-method-of-predicting-athletic-performance-in-an-individual by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:08:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216886:A METHOD OF PREDICTING ATHLETIC PERFORMANCE IN AN INDIVIDUAL.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF PREDICTING ATHLETIC PERFORMANCE IN AN INDIVIDUAL.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention concerns novel methods of selecting or matching a sport or sporting event to an individual (e.g. a sprint/power sport or an endurance sport) and predicting athletic performance, the methods involving assessing ACTN3 genotype. In alternative embodiments, training regimens may be optimally designed for athletes by assessing the ACTN3 genotypes. Certain embodiments concern combining the assessment of the ACTN3 genotype with other known fitness-related genes to better assess the athletic potential of an individual. In addition, the genotypic analysis of the ACTN3 gene may be combined with physiological tests, physical measurements and/or psychological assessments to more optimally design a training regimen for an individual athlete.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD OF PREDICTING ATHLETIC PERFORMANCE<br>
IN AN INDIVIDUAL<br>
[0001] This application claims the benefit of Australian Provisional Patent Application<br>
No. 2002951411 filed September 14,2002.<br>
BACKGROUND OF THE INVENTION<br>
[0002] The present invention relates to methods for selecting or matching a sport or<br>
sporting event to an individual (e.g., a sprint/power sport or an endurance sport) to<br>
increase their chances of success, optimizing the training programs of individuals, and<br>
for predicting the athletic performance of individuals. Certain embodiments of the<br>
invention relate to identifying specific gene(s) or alterations in the gene(s) that correlate<br>
with potential athletic performance. More particularly, the invention relates to methods<br>
of genotyping an individual with respect to the gene encoding the skeletal muscle<br>
protein, a-actinin-3 (ACTN3). In a specific embodiment, the ACTN3 genotype is<br>
determined for a single nucleotide polymorphism (SNP) site 1747 C&gt;T.<br>
DESCRIPTION OF RELATED ART<br>
[0003] In an increasingly competitive environment for athletic performance, talent<br>
search programs are on the rise to ensure that those with the potential to become an<br>
elite athlete are identified earlier in life to enable a head start in their efforts to reach<br>
their peak performance. These talent search programs are presently based on actual<br>
performance data and phenotypic predictors determined by the type of training to be<br>
undertaken, as well as the likely demands from the particular sport. One weakness of<br>
both current training programs and talent search criteria is the inability to determine<br>
whether an individual has already reached his/her performance potential, and so is<br>
unlikely to respond optimally to further training.<br>
[0004] Another weakness of the current talent search programs, which is particularly<br>
relevant in countries with a relatively small population base in a large geographic area,<br>
is the opportunity for selection. An individual brought up in a environment with<br>
widespread access to sporting and coaching facilities is more likely to achieve success,<br>
and therefore more likely to come to the attention of coaches and talent scouts than a<br>
young individual with potential who resides in a relatively isolated location or who<br>
might otherwise have an underprivileged background. Similarly, individuals with<br>
potential to excel in lower profile sports such as rowing may be overlooked simply<br>
because these sports programs are less available in most schools. Again, this<br>
diminishes the chances of early identification and participation, leading to subsequent<br>
overlook by coaches and talent scouts. These are dilemmas facing sporting<br>
organizations such as the Australian Institute of Sport (AIS), since potential elite<br>
athletes are preferably selected and inducted into relevant training programs at a young<br>
age.<br>
[0005] The possibility exists that linkages or associations of genotype or genotypic<br>
markers to certain physiological traits may contribute to or reduce performance in an<br>
elite athlete. Such methods may permit the development of DNA screens to assist in<br>
the selection of individuals with elite athlete potential. Such screens may help in<br>
overcoming some of the selection limitations of current talent search programs. In<br>
addition, such screening methods may assist in recognizing to whom and when a<br>
possibly small, but critical difference in an individual"s training program should be<br>
made.<br>
[0006] The a-actinins are a family of actin-binding proteins related to dystrophin and<br>
the spectrins (Blanchard, A. et al., Journal of Muscle Research &amp; Cell Motility, 10,<br>
280-289,1989). In skeletal muscle, the family members a-actinin-2 and q-actinin-g^are<br>
major structural components of sarcomeric Z-lines, where they function to anchor actin-<br>
containing thin filaments in a constitutive manner (Beggs, A. H. et ah, Journal of<br>
Biological Chemistry, 267,9281-9288,1992). However, recent studies suggest additional<br>
roles for the a-actinins in skeletal muscle.<br>
[0007] It has been found that sarcomeric a-actinins bind to other thin filament and Z-line<br>
proteins including nebulin, myotilin, CapZ and myozenin (Nave, R. et al., FEBS Letters,<br>
269, 163-166,1990, Papa, I. et al, Journal of Muscle Research &amp; Cell Motility, 20,187-<br>
197, 1999, and Salmikangas, P. et al., Human Molecular Genetics, 8, 1329-1336,1999),<br>
the intermediate filament proteins, synemin and vinculin (Bellin, R. M. et al., Journal of<br>
Biological Chemistry, 274, 29493-29499, 1999, and McGregor, A. et al., Biochemical<br>
Journal, 301,225-233,1994), and the sarcolemmal membrane proteins, dystrophin and pi<br>
integrin (Hance, J.E. et al., Archives of Biochemistry &amp; Biophysics, 365, 216-222,1999,<br>
and Otey, C. A. et al., Journal of Biological Chemistry, 268,21193-21197,1993). These<br>
binding studies suggest that the a-actinins play a role in thin filament organization and the<br>
interaction between the sarcomere cytoskeleton and the muscle membrane. In addition,<br>
sarcomeric a-actinin binds phosphatidylinositol 4,5-bisphophate (Fukami, K. et al.,<br>
Journal of Biological Chemistry, 269,1518-1522,1994), phosphatidylinositol 3 kinase<br>
(Shibasaki, F. et al., Biochemical Journal, 302, 551-557,1994) and PDZ-LJM adaptor<br>
proteins (Pomies, P. et al., Journal of Cell Biology, 139,157-168,1997, and Pomies, P. et<br>
al., Journal of Biological Chemistry, 274, 29242-29250), suggesting a role in the<br>
regulation of myofiber differentiation and/or contraction.<br>
[0008] In humans, the a-actinin-2 gene, ACTN2, is expressed in all skeletal muscle<br>
fibers, while expression of ACTN3, encoding oc-actinin-3, is limited to a subset of type 2<br>
(fast) fibers (North, K. N. et al., Nature Genetics, 21,353-354,1999). It has been recently<br>
demonstrated that a-actinin-3 is absent in ~18% of individuals in a range of human<br>
populations and that homozygosity for a premature stop codon (577X) accounts for all<br>
cases of true a-actinin-3 deficiency states identified to date. An additional polymorphism<br>
(523R) occurs in linkage disequilibrium with 577X, but does not appear to exert a<br>
deleterious effect when expressed in the heterozygous state in coupling with 577R.<br>
Further, absence of a-actinin-3 is not associated with an obvious disease phenotype,<br>
suggesting that ACTN3 is redundant in humans (North, K. N. et al., 1999 Nature Genetics<br>
21: 353-354).<br>
[0009] Functional redundancy occurs when two genes perform overlapping functions<br>
so that inactivation of one of the genes has little or no effect on the phenotype<br>
(reviewed in Nowak, M. A. et al., Nature, 388,167-171,1997). In human skeletal<br>
muscle, a-actinin-2 expression completely overlaps a-actinin-3. ACTN2 and ACTN3<br>
are also 80% identical and 90% similar (Beggs, A. H. et al., 1992, supra), and a-<br>
actinin-2 and a-actinin-3 are capable of forming heterodimers in vitro and in vivo,<br>
suggesting structural similarity and lack of significant functional differences between<br>
the two skeletal muscle a-actinin isoforms (Chan, Y. et al., Biochemical &amp; Biophysical<br>
Research Communications, 248, 134-139, 1998). It is hypothesised that a-actinin-2 is<br>
able to compensate for the absence of a-actinin-3 in type 2 (fast) fibers in humans.<br>
SUMMARY OF THE INVENTION<br>
[0010] Despite the apparent functional redundancy of ACTN3 and ACTN2 in humans,<br>
genotype screens of a pool of elite Australian athletes and noted Caucasian sprint<br>
athletes (particularly short distance runners, swimmers and cyclists) showed a very low<br>
frequency of homozygosity for the ACTN3 premature stop codon 577X mutation (i.e.<br>
an ACTN3 null mutation, 577XX) relative to the Australian Caucasian population at<br>
large. It is therefore considered that screening for ACTN3 genotype, would provide<br>
considerable assistance in the selection of young individuals with potential for elite<br>
performance in sprint-type sports and events. Also, the genotype screens showed that<br>
the frequency of the 577XX genotype was relatively higher in Caucasian elite<br>
endurance athletes. Thus, a screening procedure for ACTN3 577XX genotype, may<br>
also provide assistance in identifying young individuals with potential for elite<br>
performance in endurance sports and events.<br>
[0011] The present invention solves a need in the art by providing in vitro methods for<br>
screening individuals for athletic potential. In a one embodiment, the genotype of an<br>
individual may be determined for the gene ACTN3. In another embodiment, mRNA or<br>
protein is isolated from type 2 skeletal muscle and analyzed for the presence or absence<br>
of ACTN3. In another embodiment, individuals are identified by isolating DNA from<br>
blood or buccal swab samples and the DNA is amplified and analyzed for the presence<br>
or absence of a premature stop codon (577X) in the ACTN3 gene. Other embodiments<br>
provide methods for screening individuals for athletic potential by combining the<br>
screening of ACTN3 with other genetic, and physiological tests. In addition, the<br>
methods described provide for developing training program(s) better suited for an<br>
individual athlete by genetic assessments, physiological tests, physical measurements<br>
and/or psychological assessments.<br>
[0012] In another embodiment, the invention provides for screening individuals for<br>
elite athletic potential, the method for example is carried out by obtaining a suitable<br>
muscle cell sample from an individual and detecting in the sample, a-actinin-3 protein<br>
and/or messenger RNA encoding that protein.<br>
[0013] Particular embodiments of the invention relate to a method of predicting the<br>
presence or absence of a particular phenotype. The method comprises obtaining a<br>
nucleic acid sample from an individual and determining the identity of one or more<br>
bases (nucleotides) at specific (e.g., polymorphic) sites of nucleic acid molecules<br>
described herein, wherein the presence of a particular base at that site is correlated with<br>
a specified phenotype, thereby predicting the presence, absence, or likelihood of the<br>
presence or absence, of the phenotype in the individual.<br>
ACCOMPANYING<br>
BRIEF DESCRIPTION OF THEEDRAWINGS<br>
[0014] The following drawings form part of the present specification and are included<br>
to further demonstrate certain aspects of the present invention. The invention may be<br>
better understood by reference to one or more of these drawings in combination with<br>
the detailed description of specific embodiments presented herein.<br>
[0015] FIG. 1 illustrates the ACTN3 genotype frequency in controls, elite sprint/power<br>
athletes and elite endurance athletes.<br>
[0016] TABLE 1: represents the genotypes of the R577X SNP in ACTN3 in<br>
Caucasian elite athletes of specific disciplines.<br>
[0017] TABLE 2 represents a summary of individuals tested for number and frequency<br>
(%) of ACTN3 alleles in controls and elite sprint/power and endurance athletes.<br>
[0018] TABLE 3 represents SNPs identified in the ACTN3 gene thus far and compiled<br>
in a list from the NCBI SNP website.<br>
[0019] TABLE 4 represents symbols, full names, and cytogenic location of nuclear<br>
and mitochondrial genes of the 2002 Human Gene Map for Performance and Health-<br>
Related Fitness Phenotypes.<br>
[0020] TABLE 5 represents endurance phenotypes and case-control studies (DNA<br>
polymorphisms).<br>
[0021] TABLE 6 represents genotype and allele frequencies of ACTN3 577/R/X<br>
alleles in different human populations.<br>
Definitions<br>
[0022] As used herein in the specification, "a" or "an" may mean one or more. As used<br>
herein in the claim(s), when used in conjunction with the word "comprising", the words<br>
"a" or "an" may mean one or more than one. As used herein "another" may mean at<br>
least a second or more.<br>
[0023] "Elite athlete" or variants thereof, refers to athletes that perform at the very<br>
highest levels in terms of endurance, speed and/or strength (e.g. such that they are<br>
capable of competing at State, National and/or International levels in their sport).<br>
[0024] As used herein, the terms "SNPs" or "single nucleotide polymorphisms" refer to<br>
single base changes at a specific location in an organism"s (e.g., a human) genome.<br>
DETAILED DESCRIPTION<br>
[0025] In the following section, several embodiments of, for example, methods are<br>
described in order to exemplify various embodiments of the invention. It will be<br>
obvious though, to one skilled in the art that practicing the various embodiments does<br>
not require the employment of all or even some of the specific details outlined herein.<br>
In some cases, well known methods or components have not been included in the<br>
description.<br>
[0026] Methods and compositions to screen individuals for athletic potential are<br>
disclosed. In one embodiment of the invention, a method to screen individuals for the<br>
presence or absence of ACTN3 protein and/or mRNA is disclosed. In another<br>
embodiment of the invention, a method to screen individuals for the presence or<br>
absence of ACTN3 genotype variations is disclosed. In another embodiment of the<br>
invention, a method to screen individuals for the presence or absence of particular<br>
ACTN3 genotypes, such as 577RR, 577XR or 577XX is disclosed. Identification of<br>
ACTN3 protein may be accomplished by directly measuring the protein levels or by<br>
indirectly measuring protein levels (e.g. antibodies etc).<br>
ACTN3 Polymorphisms and Other Genetic Variations<br>
[0027] A common polymorphism in humans has been identified in the gene encoding<br>
the skeletal muscle protein, a-actinin-3 (ACTN3) that is only present in type 2 (fast)<br>
fibers. Three possible genotypes 577RR (wildtype - expresses oc-actinin-3), 577RX<br>
(heterozygous - a-actinin-3 present), and 577XX (homozygous null - no oc-actinin-3 in<br>
skeletal muscle), have been identified. The allelic frequency varies in different ethnic<br>
groups (i.e. about 18% of Caucasians are a-actinin-3 deficient compared to ~1% of<br>
African Zulus) (see Table 3)WEST AFRICANS and African Americans???. As<br>
discussed in the Examples below, in Caucasian elite sprint/power athletes, the<br>
frequency of the 577XX genotype is very low. Thus a screening procedure for ACTN3<br>
577RR genotype, may provide assistance in identifying for example young Caucasian<br>
individuals with potential for elite performance in sprint or power-type sports and<br>
events. In contrast, in Caucasian elite endurance athletes, the frequency of the 577XX<br>
genotype is relatively higher. Thus a screening procedure for ACTN3 577XX<br>
genotype, may also provide assistance in identifying for example young Caucasian<br>
individuals with potential for elite performance in endurance sports and events. In<br>
addition, Table 6 illustrates the genotype and allele frequencies of ACTN3 577R/X<br>
alleles in different human populations. In Table 6 and Table 2, the negroid Africans (ie<br>
Zulus) screened have an extremely low number of 577XX individuals. Thus, the<br>
screening of ACTN3 in negroid African populations (and, likely, the related West<br>
Africans and African-Americans) to detect 577XX genotypes may prove useful in<br>
identifying individuals with endurance potential. In one embodiment, a method for<br>
screening for an ACTN3 allele (e.g. 577R, 577X) alone or in combination with another<br>
screening methods may be used to select, or at least assist in the selection of, young<br>
individuals with elite sprint/power potential (e.g. potential as track sprinters, short<br>
distance swimmers, and track cyclists).<br>
[0028] Other genes may also have beneficial effects on sprint/power and/or endurance<br>
athletic performance. For example, angiotensin-converting enzyme (ACE) is reported<br>
to have two alleles, I and D, which have an effect on athletic performance. The I allele<br>
is associated with lower ACE activity in both serum and tissue (Reider et al.,<br>
"Sequence variation in the human angiotensin converting enzyme." Nat Genet, 1999<br>
vol. 22 pp59-62). It is reported that there is an increased frequency of the I allele in<br>
elite endurance athletes (Gayagay et al. 1998 "Elite endurance athletes and the ACE I<br>
allele; the role of genes in athletic performance". Hum Genet 103:48-50; Montgomery<br>
et al. 1998 Human gene for physical performance. Nature 393:221-222; Myerson et al.<br>
1999 Human angiotensin I-converting enzyme gene and endurance performance. J<br>
Appl Physiol 87:1313-1316; Nazarov et al. 2001 The angiotensin converting enzyme<br>
I/D polymorphism in Russian athletes Eur J Hum Genet 9:797-801). Conversely, an<br>
increased frequency of the ACE D allele has been associated with elite sprint<br>
performance (Myerson et al. 1999 Human angiotensin I-converting enzyme gene and<br>
endurance performance. J Appl Physiol 87:1313-1316; Nazarov et al. 2001 The<br>
angiotensin converting enzyme I/D polymorphism in Russian athletes Eur J Hum Genet<br>
9:797-801; Woods et al. 2001 Elite swimmers and the D allele of the ACE I/D<br>
polymorphism. Hum Genet 108: 230-232).<br>
[0029] It is possible that there is a tradeoff between sprint and endurance attributes that<br>
imposes limitations on the evolution of physical performance in humans and other<br>
vertebrates (Garland et al. 1990 "Heritability of locomotor performance and its<br>
correlates in a natural population" Experientia 46:530-533). This is supported by data<br>
from world-class decathletes, which demonstrate that performance in the 100-m sprint,<br>
shot-put, long-jump, and 110-m hurdles (relying on explosive power and fast fatigue-<br>
susceptible muscle fibers) is negatively correlated with performance in the 1,500-m<br>
race (requiring endurance and fatigue-resistant slow fiber activity). (Van Damme et al.<br>
2002 Performance constraints in decathletes. Nature 415:755-756). This suggests that<br>
an individual may be predisposed toward specialist performance in only one of the two<br>
areas (sprint/power vs. endurance). In particular embodiments of the invention,<br>
screening tests for ACTN3 may be combined with one or more genetic tests for other<br>
performance associated genes. Such tests may include any gene that is known in the art<br>
to be associated with sprint/power and/or endurance performance (e.g., Rankinen et al.<br>
2002, "The human gene map for performance and health-related fitness phenotypes: the<br>
2001 update" Med. Sci. Sports Exerc. 34: 1219-33; Perusse et al. 2003, "The human<br>
gene map for performance and health-related fitness phenotypes: the 2002 update"<br>
Med. Sci. Sports Exerc. 35: 1248-1264 incorporated herein by reference in their<br>
entirety).<br>
[0030] Two reports (Rankinen et al. 2002; Perusse et al. 2003) have summarized the<br>
results of studies of performance and health-related fitness phenotypes. A human<br>
performance and health-related fitness gene map is shown as figure 1 in the 2002<br>
article. The map includes all gene entries and QTL (quantitative trait loci) that have<br>
shown associations or linkages with exercise-related phenotypes. The chromosomes<br>
and their regions are from the Gene Map of the Human Genome web site, the National<br>
Center for Biotechnology Information (NCBI), National Institutes of Health, Bethesda,<br>
MD. The loci abbreviations and full names of the genes of potential use in conjunction<br>
with ACTN3 screening are summarized in TABLE 4. In one embodiment, analysis of<br>
one or more of the genes referenced in TABLE 4 may be used in combination with the<br>
evaluation of the ACTN3 gene of an individual to predict the elite athletic potential of<br>
that individual.<br>
[0031] TABLE 5 summarizes a study (Perusse et al., 2003) of alleles and genotype<br>
frequencies of the ADRA2A (Alpha-2A-adrenergic receptor) and ACE (Angiotensin 1<br>
converting enzyme) genes between endurance athletes and sedentary controls. TABLE<br>
5 illustrates the differences between endurance athletes and sedentary individuals. In<br>
one embodiment of the invention, the examination of the ACTN3 genotype of a<br>
potential elite athlete may be combined with the assessment of either the ADRA2A<br>
genotype and/or the ACE genotype in order to more accurately predict the athletic<br>
potential of an individual. In another embodiment, the assessment of the ACTN3<br>
genotype of an athlete may be combined with the assessment of either the ADRA2A<br>
genotype and/or the ACE genotype and/or other physiological assessments (eg VO2<br>
max etc.) to customize a training regimen for the athlete.<br>
Evolutionary Divergence of ACTN3 and ACTN2<br>
[0032] Genotyping of non-human primates indicates that the 577X null mutation has<br>
likely arisen in humans. The mouse genome contains four orthologues which all map<br>
to evolutionarily conserved regions for the four human genes. Murine ACTN2 and<br>
ACTN3 are differentially expressed, spatially and temporally, during embryonic<br>
development, and in contrast to humans, a-actinin-2 expression does not completely<br>
overlap a-actinin-3 in postnatal skeletal muscle, suggesting independent function.<br>
Furthermore, sequence comparison of human, mouse and chicken a-actinin genes<br>
demonstrates that ACTN3 has been conserved over a long period of evolutionary time,<br>
implying a constraint on evolutionary rate imposed by continued function of the gene.<br>
These observations provide a real framework in which to test theoretical models of<br>
genetic redundancy as they apply to human populations as well as other animals (Mills<br>
et al Differential Expression of the Actin-binding Proteins, a-actinin-2 and -3, in<br>
Different Species: Implications for the Evolution of Functional Redundancy" 2001<br>
Hum Mol Gene 13:1335-1346).<br>
[0033] To determine the origin of the 577X allele (and the 523R allele, which occurs in<br>
strong linkage disequilibrium with 577X), 36 unrelated baboons (diverged from human<br>
lineage 25 X 106 years ago) and 33 unrelated chimpanzees (diverged from human<br>
lineage 5 X 106 years ago) were genotyped. All 69 non-human primates were<br>
homozygous for the "wild-type" alleles in exons 15 (523Q) and 16 (577R), suggesting<br>
that the polymorphisms originated after the separation of the human and chimpanzee<br>
lineages, or that they have a very low frequency in non-human primates (Mills et al<br>
2001).<br>
[0034] As for mice, the similarity between mouse ACTN2 and ACTN3 is the same as<br>
between human ACTN2 and ACTN3, i.e. 88% similar and 79% identical. The mouse<br>
proteins are collinear and have the same functional domains as the human proteins - an<br>
N-terminal actinin-binding domain, four central repeat domains and C-terminal EF-<br>
hands (Mills et al 2001).<br>
[0035] There is only one skeletal muscle ACTN gene in the chicken .whereas the<br>
mouse genome contains four orthologues which all map to evolutionarily conserved<br>
syntenic regions for the four human genes. Sequence comparison between mouse and<br>
human ACTN2 and ACTN3 suggests that the evolution of the a-actinins has been slow<br>
relative to other genes. The low rate of substitution in ACTN3 appears not to be due to<br>
an intrinsically low mutation rate in this gene (Mills et al 2001).<br>
[0036] In other mammals, such as rabbits and pigs, there are also fast- and slow-<br>
muscle-specific isoforms of a-actinin, although the gene(s) responsible have not been<br>
isolated. The presence of two sarcomeric a-actinin genes may, however, be restricted<br>
to mammals.<br>
[0037] In mammals both copies of the gene have survived, and the comparison of the<br>
human and mouse ACTN2 and ACTN3 sequences shows that the genes have been<br>
highly conserved throughout mammalian evolution (Mills et al 2001).<br>
Elite Athletic Performance and Horses<br>
[0038] The horse is one of very few animals besides some dogs and camels that is bred,<br>
kept or sold for its athletic performance and therefore is another model for studying<br>
gene expression as it correlates with performance. For example, the conservation of the<br>
ACTN3, an athletic marker in humans for athletic potential, and ACTN2 gene<br>
throughout species has been previously demonstrated. Although the equivalent gene<br>
has not yet been identified in horses, it is highly probable that a gene like ACTN3<br>
exists in horses but has eluded detection. In certain embodiments of the invention,<br>
horses may be screened for an ACTN3-like gene. In other embodiments race horses<br>
such as the horses trained to compete in a derby may be screened for an ACTN3-like<br>
gene. Alternatively, horses required to sprint with enormous power such as polo ponies<br>
and barrel racing horses may also be screened for differential expression of an ACTN3-<br>
like gene. It is likely that the sprinting horses express a gene that is slightly different<br>
than an endurance horse and therefore analysis of the ACTN3-like gene may be an<br>
indicator of elite athletic potential in horses. Similar to what is seen in human athletes,<br>
screening a gene for a minor change, for example the presence or absence of a specific<br>
nucleotide sequence (eg. SNP site, deletion or insertion) may be a valuable indicator of<br>
elite athletic potential in an animal such as a horse. An ACTN3-like gene is a gene that<br>
has the same function as the ACTN3 in other species and/or it has sequence similarities<br>
to the ACTN3 gene.<br>
[0039] Previous studies indicate the equine angiotensin-converting enzyme gene might<br>
be an ideal candidate gene for athletic performance in horses. The human variant of the<br>
gene contains a polymorphic marker that is associated with increased athletic ability of<br>
elite endurance athletes and an increased anabolic response to training (Ellis et al,<br>
Characterization of the Equine Angiotensin-converting Enzyme" 7th World Congress<br>
on Genetics Applied to Livestock Production, August 19-23,2002, Montpellier, France<br>
Session 05. Horse breeding Abstract of N° 05-07 GENE. N:A. I. Tammen, F.W.<br>
Nicholas and H.W. Raadsma. ReproGen, University of Sydney, Camden, Australia).<br>
To date, a correlation in horses of the ACE expression and elite athletic performance<br>
has been unsuccessful. Other studies including a study of the myosin heavy-chain<br>
gene(MyHC) in equine gluteus medius muscle where diffential expression of the gene<br>
has been identified in foals but direct correlation of athletic abilities and presence or<br>
absence of the gene have not yet been correlated with performance (Eizema et al<br>
Differential Expression of Equine Myosin heavy-chain mRNA and Protein Isoforms in<br>
a Limb muscle" J Histochem Cytochem 2003 Sept; 51 (9):1207-1216).<br>
[0040] It is contemplated that the analysis of an ACTN3-like gene and other<br>
physiological and genetic parameters may be measured in horses in order to more<br>
accurately access the elite athletic ability of a horse at an early age. It is contemplated<br>
that horses may be pre-screened before using them for breeding purposes to identify a<br>
more satisfactory genetic match. In addition it is possible that a foal in utero may be<br>
screened in order to assess the athletic potential of the foal before it is born. The<br>
information generated from such screenings would save the breeders and investors of<br>
horses (camels, dogs) a tremendous amount of time and money as well as identify the<br>
potential ability of an animal at a early stage of development. As with humans, the<br>
information generated from genotypic screening of a horse as well as other parameters<br>
(bloodlines etc.) may help to identify a potential elite athlete and/or design a better<br>
training regiment for a specific animal (e.g., a polo pony).<br>
Single Nucleotide Polymorphisms (SNPs)<br>
[0041] Various embodiments of the invention provide for methods for determining a<br>
correlation between a polymorphism or genetic variation (e.g, a SNP) and a phenotype,<br>
comprising: a) providing: samples from one or more subjects; possibly medical records<br>
from one or more subjects, for determining a phenotype of the subject(s) and detection<br>
assays that detect a polymorphism; b) exposing the samples to detection assays under<br>
conditions such that the presence or absence of at least one polymorphism is revealed;<br>
and; c) determining a correlation between the at least one polymorphism and the<br>
phenotype of the subjects.<br>
[0042] Nucleic acids in the region of interest (e.g., the region containing the genetic<br>
variation of interest) may be assayed using any suitable method, including but not<br>
limited to manual sequencing using radioactive marker nucleotides, or automated<br>
sequencing. The sequence may be examined and the presence or absence of a given<br>
SNP or mutation determined. The particular SNP site(s) (e.g. 1747 C&gt;T of ACTN3) of<br>
a gene may be used to evaluate the presence, absence or change in a particular gene in<br>
order to assess the athletic potential of an individual or modify a training regimen for<br>
that individual. The known SNPs for ACTN3 are listed in TABLE 3. In various<br>
embodiments of the invention, screening for the 1747 C&gt;T SNP of the ACTN3 gene<br>
may be combined with screening for any other known polymorphism in the ACTN3<br>
gene, including but not limited to any SNP listed in TABLE 3.<br>
[0043] Other SNPs of potential use in the practice of the claimed methods are disclosed<br>
for example, in the Table of published U.S. patent application serial No. 801274,<br>
publication No. 20020032319, incorporated herein by reference in its entirety. Any one<br>
or more of these sites may be assayed in combination with 1747 C&gt;T SNP of the<br>
ACTN3 gene to predict the athletic potential of an individual, select or match a sport or<br>
sproting event o an individual (ie to increase the individual"s chances of success) and/or<br>
to optimize a training regimen.<br>
[0044] In alternative embodiments of the invention, screening for genetic variations<br>
may utilize other detection assays, such as an allele-specific hybridization assay. In a<br>
hybridization assay, the presence of absence of a given SNP or other genetic variation<br>
is determined based on the ability of the DNA from the sample to hybridize to a<br>
complementary DNA molecule (e.g., a oligonucleotide probe). A variety of<br>
hybridization assays using a variety of techniques for hybridization and detection are<br>
known in the art and any such known technique may be used in the claimed methods.<br>
Exemplary assays are disclosed below.<br>
[0045] In some embodiments, detection assays may utilize a DNA chip hybridization<br>
assay. In such assays, a series of oligonucleotide probes are affixed to a solid support.<br>
In some embodiments, the oligonucleotide probes are designed to be unique to a given<br>
SNP or mutation. The DNA sample of interest is contacted with the DNA "chip" and<br>
hybridization is detected. DNA chips, including customized DNA chips specific for<br>
particular SNP sequences, are available from commercial sources such as Affymetrix<br>
(Santa Clara, CA).<br>
[0046] In other exemplary embodiments, polymorphisms may be detected using a<br>
SNP-IT primer extension assay (Orchid Biosciences, Princeton, N.J.; e.g., U.S. Pat.<br>
Nos. 5,952,174 and 5,919,626). In this assay, SNPs are identified by using a specially<br>
synthesized DNA primer and a DNA polymerase to selectively extend the DNA chain<br>
by one base at the suspected SNP location. DNA in the region of interest is amplified<br>
and denatured. Polymerase reactions are then performed using microfluidic systems.<br>
Detection is accomplished by adding a label to the nucleotide suspected of being at the<br>
SNP or mutation location. Incorporation of the label into the DNA can be detected by<br>
any suitable method (e.g., if the nucleotide contains a biotin label, detection is via a<br>
fluorescently labeled antibody specific for biotin). Other commercial kits may be used<br>
to identify the presence or absence of one or more SNPs (e.g., Applied Biosystems:<br>
SNaPSHOT, Assay-on-Demand, Assay-By-Design, Pyrosequencing assays (see:<br>
http://wwwpyrosequencing.com/pages/products96hs.html).<br>
Nucleic Acids<br>
[0047] Various embodiments of the invention involve the isolation and analysis of<br>
nucleic acid molecules, such as DNA, mRNA or cDNA. Nucleic acids of interest may<br>
encode a portion or all of a targeted protein (eg ACTN3, ACE etc.). A "nucleic acid"<br>
as used herein includes single-stranded and double-stranded molecules, as well as<br>
DNA, RNA, chemically modified nucleic acids and nucleic acid analogs. It is<br>
contemplated that a nucleic acid within the scope of the present invention may be of 1,<br>
2, 3,4, 5, 6, 7, 8, 9, 10, 11,12, 13, 14, 15, 16, 17, 18, 19, 20, 21,22, 23, 24, 25, 26, 27,<br>
28, 29, 30, 31, 32, 33, 34, 35, 36, 37,38, 39, 40, 41, 42,43,44,45, 46, 47,48,49, 50,<br>
51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69,70, 71, 72,73,<br>
74,75, 76,77, 78,79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92,93, 94, 95, 96,<br>
97, 98, 99,100, about 110, about 120, about 130, about 140, about 150, about 160,<br>
about 170, about 180, about 190, about 200, about 210, about 220, about 230, about<br>
240, about 250, about 275, about 300, about 325, about 350, about 375, about 400,<br>
about 425, about 450, about 475, about 500, about 525, about 550, about 575, about<br>
600, about 625, about 650, about 675, about 700, about 725, about 750, about 775,<br>
about 800, about 825, about 850, about 875, about 900, about 925, about 950, about<br>
975,about 1000,about 1100, about 1200, about 1300,about 1400,about 1500, about<br>
1750, about 2000, about 2250, about 2500 or greater nucleotide residues in length, up<br>
to and including full-length chromosomal DNA.<br>
[0048] Methods for partially or fully purifying DNA and/or RNA from complex<br>
mixtures, such as cell homogenates or extracts, are well known in the art. (See, e.g.,<br>
Guide to Molecular Cloning Techniques, eds. Berger and Kimmel, Academic Press,<br>
New York, NY, 1987; Molecular Cloning: A Laboratory Manual. 2nd Ed., eds.<br>
Sambrook, Fritsch and Maniatis, Cold Spring Harbor Press, Cold Spring Harbor, NY,<br>
1989). Generally, cells, tissues or other source material containing nucleic acids are<br>
first homogenized, for example by freezing in liquid nitrogen followed by grinding in a<br>
mortar and pestle. Certain tissues may be homogenized using a Waring blender, Virtis<br>
homogenizer, Dounce homogenizer or other homogenizer. Crude homogenates may be<br>
extracted with detergents, such as sodium dodecyl sulphate (SDS), Triton X-100,<br>
CHAPS (3-[(3-cholamidopropyl)-dimethylammonio]-l-propane sulfonate),<br>
octylglucoside or other detergents known in the art. As is well known, nuclease<br>
inhibitors such as RNase or DNase inhibitors may be added to prevent degradation of<br>
target nucleic acids.<br>
[0049] Extraction may also be performed with chaotrophic agents such as guanidinium<br>
isothiocyanate, or organic solvents such as phenol. In some embodiments, protease<br>
treatment, for example with proteinase K, may be used to degrade cell proteins.<br>
Particulate contaminants may be removed by centrifugation or ultracentrifugation.<br>
Dialysis against aqueous buffer of low ionic strength may be of use to remove salts or<br>
other soluble contaminants. Nucleic acids may be precipitated by addition of ethanol at<br>
-20°C, or by addition of sodium acetate (pH 6.5, about 0.3 M) and 0.8 volumes of 2-<br>
propanol. Precipitated nucleic acids may be collected by centrifugation or, for<br>
chromosomal DNA, by spooling the precipitated DNA on a glass pipet or other probe.<br>
The skilled artisan will realize that the procedures listed above are exemplary only and<br>
that many variations may be used, depending on the particular type of nucleic acid to be<br>
analyzed.<br>
[0050] In certain embodiments, nucleic acids to be analyzed may be naturally occurring<br>
DNA or RNA molecules. Virtually any naturally occurring nucleic acid may be<br>
analyzed by the disclosed methods including, without limit, chromosomal,<br>
mitochondrial or chloroplast DNA or ribosomal, transfer, heterogeneous nuclear or<br>
messenger RNA. Nucleic acids may be obtained from either prokaryotic or eukaryotic<br>
sources by standard methods known in the art. Alternatively, nucleic acids of interest<br>
may be prepared artificially, for example by PCR™ or other known amplification<br>
processes or by preparation of libraries such as BAC, YAC, cosmid, plasmid or phage<br>
libraries containing nucleic acid inserts. (See, e.g., Berger and Kimmel, 1987; Sambrook<br>
et al, 1989.) The source of the nucleic acid is unimportant for purposes of analysis and<br>
it is contemplated within the scope of the invention that nucleic acids from virtually<br>
any source may be analyzed.<br>
Nucleic Acid Amplification<br>
In particular embodiments, nucleic acids to be analyzed for screening may first be<br>
amplified to increase the signal strength. Nucleic acid sequences to be used as a template<br>
for amplification may be isolated from cells contained in a biological sample (eg DNA or<br>
mRNA from skeletal muscle), according to standard methodologies. The nucleic acid<br>
may be genomic DNA or fractionated or whole cell RNA. Where RNA is used, it may be<br>
desired to convert the RNA to a complementary cDNA. In one embodiment, the RNA is<br>
whole cell RNA and is used directly as the template for amplification. In one example,<br>
the determination of the ACTN3 genotype is performed by amplifying (e.g., by PCR)<br>
the ACTN3 polynucleotide sequences, or more preferably a fragment thereof which<br>
includes the 1747 C&gt;T SNP (e.g., exon 16), and sequencing the amplification products<br>
or otherwise detecting the presence and/or absence of the 1747 C&gt;T SNP in the<br>
amplification products. In another example, it is known that the 577X allele contains a<br>
Ddel restriction site which can be readily detected by Ddel digestion of the<br>
amplification products and size fractionation of the digestion products (e.g. by gel<br>
electrophoresis). The size of the products may be used to genotype the ACTN3 locus<br>
in the individual. Various forms of amplification are well known in the art and any<br>
such known method may be used. Generally, amplification involves the use of one or<br>
more primers that hybridize selectively or specifically to a target nucleic acid sequence<br>
to be amplified.<br>
Primers<br>
[0051] The term primer, as defined herein, is meant to encompass any nucleic acid that is<br>
capable of priming the synthesis of a nascent nucleic acid in a template-dependent<br>
process. Typically, primers are oligonucleotides from ten to twenty base pairs in length,<br>
but longer sequences may be employed. Primers may be provided in double-stranded or<br>
single-stranded form, although the single-stranded form is preferred. Methods of primer<br>
design are well-known in the art, based on the design of complementary sequences<br>
obtained from standard Watson-Crick base-pairing (i.e., binding of adenine to thymine or<br>
uracil and binding of guanine to cytosine). Computerized programs for selection and<br>
design of amplification primers are available from commercial and/or public sources well<br>
known to the skilled artisan. Particular primer sequences of use in detecting genetic<br>
variants predictive of athletic performance, such as the 1747 C&gt;T SNP in ACTN3, are<br>
provided in the following Examples. The skilled artisan will realize that the specific<br>
sequences provided are exemplary only and that alternative primer and/or probe<br>
sequences may be used in the practice of the claimed methods.<br>
Amplification Methods<br>
[0052] A number of template dependent processes are available to amplify the marker<br>
sequences present in a given sample. One of the best known amplification methods is the<br>
polymerase chain reaction (referred to as PCR) which is described in detail in U.S. Patent<br>
Nos. 4,683,195,4,683,202 and 4,800,159.<br>
[0053] One embodiment of the invention may comprise obtaining a suitable sample<br>
from an individual and detecting a specific messenger RNA, such as an ACTN3<br>
mRNA. An exemplary sample for use in this method is a muscle tissue sample (e.g.<br>
muscle tissue biopsy, such as a punch biopsy). Once the tissue sample is obtained the<br>
sample may be prepared for isolation of the nucleic acids by standard techniques (eg<br>
single cell isolation, digestion of outer membranes, Oligo dT isolation of mRNA etc.)<br>
The isolation of the mRNA may also be performed using kits known to the art (Pierce,<br>
AP Biotech, etc). A reverse transcriptase PCR amplification procedure may be<br>
performed in order to quantify an amount of mRNA amplified. Methods of reverse<br>
transcribing RNA into cDNA are well known and described in Sambrook et al., 1989.<br>
Alternative methods for reverse transcription utilize thermostable DNA polymerases.<br>
These methods are described in WO 90/07641 filed December 21,1990.<br>
[0054] Another method for amplification of nucleic acids is the ligase chain reaction<br>
("LCR"), disclosed in European Application No. 320 308. In LCR, two complementary<br>
probe pairs are prepared, and in the presence of the target sequence, each pair will bind to<br>
opposite complementary strands of the target such that they abut. In the presence of a<br>
ligase, the two probe pairs will link to form a single unit. By temperature cycling, as in<br>
PCR, bound ligated units dissociate from the target and then serve as "target sequences"<br>
for ligation of excess probe pairs. U.S. Patent 4,883,750 describes a method similar to<br>
LCR for binding probe pairs to a target sequence.<br>
[0055] Qbeta Replicase, described in PCT Application No. PCT/US87/00880, may also<br>
be used as still another amplification method in the present invention. In this method, a<br>
replicative sequence of RNA that has a region complementary to that of a target is added<br>
to a sample in the presence of an RNA polymerase. The polymerase will copy the<br>
replicative sequence that may then be detected.<br>
[0056] An isothermal amplification method, in which restriction endonucleases and<br>
ligases are used to achieve the amplification of target molecules that contain nucleotide 5-<br>
[alpha-thio]-triphosphates in one strand of a restriction site may also be useful in the<br>
amplification of nucleic acids in the present invention (Walker et al., Proc. Nat! Acad. Sci.<br>
USA 89:392-396,1992).<br>
[0057] Strand Displacement Amplification (SDA) is another method of carrying out<br>
isothermal amplification of nucleic acids which involves multiple rounds of strand<br>
displacement and synthesis, i.e., nick translation. A similar method, called Repair Chain<br>
Reaction (RCR), involves annealing several probes throughout a region targeted for<br>
amplification, followed by a repair reaction in which only two of the four bases are<br>
present. The other two bases may be added as biotinylated derivatives for easy detection.<br>
A similar approach is used in SDA. Target specific sequences may also be detected using<br>
a cyclic probe reaction (CPR). In CPR, a probe having 3" and 5" sequences of non-specific<br>
DNA and a middle sequence of specific RNA is hybridized to DNA which is present in a<br>
sample. Upon hybridization, the reaction is treated with RNase H, and the products of the<br>
probe identified as distinctive products which are released after digestion. The original<br>
template is annealed to another cycling probe and the reaction is repeated.<br>
[0058] Still other amplification methods described in GB Application No. 2 202 328, and<br>
in PCT Application No. PCT/US89/01025 may be used in accordance with the present<br>
invention. In the former application, "modified" primers are used in a PCR like, template<br>
and enzyme dependent synthesis. The primers may be modified by labeling with a<br>
capture moiety (e.g., biotin) and/or a detector moiety (e.g., enzyme). In the latter<br>
application, an excess of labeled probes are added to a sample. In the presence of the<br>
target sequence, the probe binds and is cleaved catalytically. After cleavage, the target<br>
sequence is released intact to be bound by excess probe. Cleavage of the labeled probe<br>
signals the presence of the target sequence.<br>
[0059] Other nucleic acid amplification procedures include transcription-based<br>
amplification systems (TAS), including nucleic acid sequence based amplification<br>
(NASBA) and 3SR. Kwoh et al, Proc. Natl Acad. Sci. USA 86:1173 (1989); Gingeras et<br>
al, PCT Application WO 88/10315. In NASBA, the nucleic acids may be prepared for<br>
amplification by standard phenol/chloroform extraction, heat denaturation of a clinical<br>
sample, treatment with lysis buffer and mini spin columns for isolation of DNA and RNA<br>
or guanidinium chloride extraction of RNA. These amplification techniques involve<br>
annealing a primer which has target specific sequences. Following polymerization,<br>
DNA/RNA hybrids are digested with RNase H while double stranded DNA molecules are<br>
heat denatured again. In either case the single stranded DNA is made fully double<br>
stranded by addition of second target specific primer, followed by polymerization. The<br>
double-stranded DNA molecules are then multiply transcribed by a polymerase such as T7<br>
or SP6. In an isothermal cyclic reaction, the RNA"s are reverse transcribed into double<br>
stranded DNA, and transcribed once against with a polymerase such as T7 or SP6. The<br>
resulting products, whether truncated or complete, indicate target specific sequences.<br>
[0060] Davey et al., European Application No. 329 822 disclose a nucleic acid<br>
amplification process involving cyclically synthesizing single-stranded RNA ("ssRNA"),<br>
ssDNA, and double-stranded DNA (dsDNA), which may be used in accordance with the<br>
present invention. The ssRNA is a first template for a first primer oligonucleotide, which<br>
is elongated by reverse transcriptase (RNA-dependent DNA polymerase). The RNA is<br>
then removed from the resulting DNA:RNA duplex by the action of ribonuclease H<br>
(RNase H, an RNase specific for RNA in duplex with either DNA or RNA). The resultant<br>
ssDNA is a second template for a second primer, which also includes the sequences of an<br>
RNA polymerase promoter (exemplified by T7 RNA polymerase) 5" to its homology to<br>
the template. This primer is extended by DNA polymerase (exemplified by the large<br>
"Klenow" fragment of E. coli DNA polymerase I), producing a double-stranded DNA<br>
("dsDNA") molecule with a sequence identical to that of the original RNA between the<br>
primers and having additionally, at one end, a promoter sequence. This promoter<br>
sequence may be used by the appropriate RNA polymerase to make many RNA copies of<br>
the DNA. These copies may then re-enter the cycle leading to very swift amplification.<br>
With proper choice of enzymes, this amplification may be done isothermally without<br>
addition of enzymes at each cycle. Because of the cyclical nature of this process, the<br>
starting sequence may be chosen to be in the form of either DNA or RNA.<br>
[0061] Miller et al., PCT Application WO 89/06700 disclose a nucleic acid sequence<br>
amplification scheme based on the hybridization of a promoter/primer sequence to a target<br>
single-stranded DNA ("ssDNA") followed by transcription of many RNA copies of the<br>
sequence. This scheme is not cyclic, i.e., new templates are not produced from the<br>
resultant RNA transcripts. Other amplification methods include "race" and "one-sided<br>
PCR." Frohman, M.A., In: PCR PROTOCOLS: A GUIDE TO METHODS AND<br>
APPLICATIONS, Academic Press, N.Y. (1990) and Ohara et al, Proc. NatiAcad. Sci.<br>
USA, 86:5673-5677 (1989).<br>
[0062] Methods based on ligation of two (or more) oligonucleotides in the presence of<br>
nucleic acid having the sequence of the resulting "di-oligonucleotide", thereby amplifying<br>
the di-oligonucleotide, may also be used in the amplification step of the present invention.<br>
(e.g., Wu et al, Genomics 4:560 1989).<br>
Separation Methods<br>
[0063] Following amplification, it may be desirable to separate the amplification product<br>
from the template and the excess primer for the purpose of determining whether specific<br>
amplification has occurred. In one embodiment, amplification products are separated by<br>
agarose, agarose-acrylamide or polyacrylamide gel electrophoresis using standard<br>
methods. (E.g., Sambrook et al, 1989) Alternatively, chromatographic techniques may<br>
be employed to effect separation. There are many kinds of chromatography which may be<br>
used in the present invention (Freifelder, 1982).<br>
Identification Methods<br>
[0064] Various methods for detection of nucleic acid sequence variants are known in the<br>
art and any such known method may be used. Li one embodiment, detection may be by<br>
Southern blotting and hybridization with a labeled probe. The techniques involved in<br>
Southern blotting are well known to those of skill in the art (e.g., Sambrook et al, 1989).<br>
Briefly, amplification products are separated by gel electrophoresis. The gel is then<br>
contacted with a membrane, such as nitrocellulose, permitting transfer of the nucleic acid<br>
and non-covalent binding. Subsequently, the membrane is incubated with a chromophore-<br>
conjugated probe that is capable of hybridizing with a target amplification product.<br>
Detection is by exposure of the membrane to x-ray film or ion-emitting detection devices.<br>
One example of the foregoing is disclosed in U.S. Patent No. 5,279,721, which shows an<br>
apparatus and method for the automated electrophoresis and transfer of nucleic acids. The<br>
apparatus permits electrophoresis and blotting without external manipulation of the gel<br>
and is suited for carrying out methods according to the present invention.<br>
[0065] Methods and apparatus for detecting nucleic acid sequence variants are<br>
commercially available from a variety of sources, such as Third Wave, Pyrosequencing,<br>
Applied Biosystems, Affymetrix, Sequenom, Nanogen and others and any such<br>
commercial system may be used to detect sequence variants in ACTN3 or other<br>
performance related genes.<br>
Proteins and Peptides<br>
[0066] In certain embodiments, the disclosed methods may involve detecting and/or<br>
quantifying the amount of a specific protein (e.g. ACTN3) in samples to be screened.<br>
For convenience, the terms "protein," "polypeptide" and "peptide" are used<br>
interchangeably herein. Although a variety of methods of protein quantification are<br>
known in the art and may be used, antibody-based assays, such as ELISA, are<br>
particularly useful for protein quantification. The skilled artisan will realize that the<br>
following discussion is exemplary only and that any known techniques for protein<br>
identification/quantification may be used.<br>
[0068] In certain embodiments a protein or peptide may be isolated or purified. Protein<br>
purification techniques are well known to those of skill in the art. These techniques<br>
involve, at one level, the homogenization and crude fractionation of the cells, tissue or<br>
organ to polypeptide and non-polypeptide fractions. The protein or polypeptide of<br>
interest may be further purified using chromatographic and electrophoretic techniques<br>
to achieve partial or complete purification (or purification to homogeneity). Analytical<br>
methods particularly suited to the preparation of a pure peptide are ion-exchange<br>
chromatography, gel exclusion chromatography, HPLC (high performance liquid<br>
chromatography) FPLC (AP Biotech), polyacrylamide gel electrophoresis, affinity<br>
chromatography, immunoaffinity chromatography and isoelectric focusing. An<br>
example of receptor protein purification by affinity chromatography is disclosed in U.S.<br>
Patent No. 5,206,347, the entire text of which is incorporated herein by reference. One<br>
of the more efficient methods of purifying peptides is fast performance liquid<br>
chromatography (AKTA FPLC) or even HPLC.<br>
[0069] A purified protein or peptide is intended to refer to a composition, isolatable<br>
from other components, wherein the protein or peptide is purified to any degree relative<br>
to its naturally-obtainable state. An isolated or purified protein or peptide, therefore,<br>
also refers to a protein or peptide free from the environment in which it may naturally<br>
occur. Generally, "purified" will refer to a protein or peptide composition that has been<br>
subjected to fractionation to remove various other components, and which composition<br>
substantially retains its expressed biological activity. Where the term "substantially<br>
purified" is used, this designation will refer to a composition in which the protein or<br>
peptide forms the major component of the composition, such as constituting about<br>
50%, about 60%, about 70%, about 80%, about 90%, about 95%, or more of the<br>
proteins in the composition.<br>
[0070] In certain embodiments, the disclosed methods may involve purifying one or<br>
more proteins or peptides. It may be of use when purifying a protein or a DNA sample<br>
that magnetic beads be used (Dynal, Dyna beads) to isolate the molecule and<br>
subsequently identify or quantitate the amount of molecule in a sample the molecule.<br>
These techniques are known by those skilled in the art.<br>
Antibodies<br>
[0070] In certain embodiments, it may be desirable to make antibodies against<br>
particular proteins or peptides of interest (e.g. ACTN3). The appropriate protein, or<br>
portions thereof, may be conjugated, or chemically linked to one or more agents to<br>
enhance their immunogenicity, as is well known in the art. Preferred agents are the<br>
carriers are keyhole limpet hemocyanin (KLH) or bovine serum albumin (BSA).<br>
[0071] In one embodiment, the detection of a targeted protein may be by Western blot<br>
or immunocytochemistry using one or more specific antibodies to all or a portion of a<br>
target protein (e.g. ACTN3) with a specific antibody or fragment thereof (e.g. Fab<br>
fragment or a recombinant antibody fragment such as a scFv). One example of an<br>
antibody that may be used is anti-ACTN3 antibodies (as disclosed in North, K. N. et<br>
al., Neuromuscular Disorders, 6, 229-235, 1996). In another embodiment, the level of<br>
a targeted protein may be detected by obtaining a sample from an individual (e.g. a<br>
muscle biopsy) and exposing the sample to one or more antibodies directed to the<br>
targeted protein.<br>
[0072] The term "antibody" is used to refer to any antibody-like molecule that has an<br>
antigen binding region, and includes antibody fragments such as Fab", Fab, F(ab")2,<br>
single domain antibodies (DABs), Fv, scFv (single chain Fv), and the like. Techniques<br>
for preparing and using various antibody-based constructs and fragments are well<br>
known in the art. Means for preparing and characterizing antibodies are also well<br>
known in the art (See, e.g., Harlow and Lane, Antibodies: A Laboratory Manual, Cold<br>
Spring Harbor Laboratory, 1988; incorporated herein by reference).<br>
ELISA<br>
[0073] In certain preferred embodiments, the amount of a protein of interest, such as<br>
ACTN3, may be determined by various types of enzyme linked immunosorbent assays<br>
" (ELISAs) or radioimmunoassays (RIA) known in the art. Immunohistochemical detection<br>
using tissue sections is also particularly useful. However, it will be readily appreciated<br>
that detection is not limited to such techniques, and Western blotting, dot blotting, FACS<br>
analyses, and the like may also be used.<br>
[0074] In one exemplary ELISA, antibodies binding to the target proteins (e.g. ACTN3)<br>
are immobilized onto a selected surface exhibiting protein affinity, such as a well in a<br>
microtiter plate. A test composition suspected of containing the protein or portion of the<br>
protein is introduced to the well. After binding and washing to remove non-specifically<br>
bound immune complexes, the bound antigen (protein of interest) may be detected.<br>
Detection is generally achieved by the addition of a second antibody specific for the target<br>
protein that is linked to a detectable label. This type of ELISA is a "sandwich ELISA".<br>
Detection may also be achieved by the addition of a second antibody, followed by the<br>
addition of a third antibody that has binding affinity for the second antibody, with the third<br>
antibody being linked to a detectable label.<br>
[0075] In another exemplary ELISA, the samples suspected of containing the protein<br>
(antigen) are immobilized onto the well surface and then contacted with the antibodies of<br>
the invention. After binding and washing to remove non-specifically bound immune<br>
complexes, the bound antigen is detected. Where the initial antibodies are linked to a<br>
detectable label, the immune complexes may be detected directly. Alternatively, the<br>
immune complexes may be detected using a second antibody that has binding affinity for<br>
the first antibody, with the second antibody being linked to a detectable label.<br>
[0076] Another ELISA in which the proteins or peptides are immobilized, involves the<br>
use of antibody competition in the detection. In this ELISA, labeled antibodies are added<br>
to the wells, allowed to bind to the target protein, and detected by means of their label.<br>
The amount of target antigen in an unknown sample is then determined by mixing the<br>
sample with the labeled antibodies before or during incubation with coated wells. The<br>
presence of target antigen in the sample acts to reduce the amount of antibody available<br>
for binding to the well and thus reduces the ultimate signal. This is appropriate for<br>
detecting antibodies in an unknown sample, where the unlabeled antibodies bind to the<br>
antigen-coated wells and also reduces the amount of antigen available to bind the labeled<br>
antibodies.<br>
[0077] In coating a plate with either antigen or antibody, one will generally incubate the<br>
wells of the plate with a solution of the antigen or antibody, either overnight or for a<br>
specified period of hours. The wells of the plate will then be washed to remove<br>
incompletely adsorbed material. Any remaining available surfaces of the wells are then<br>
"coated" with a nonspecific protein that is antigenically neutral with regard to the test<br>
antisera. These include bovine serum albumin (BSA), casein and solutions of milk<br>
powder. The coating allows for blocking of nonspecific adsorption sites on the<br>
immobilizing surface and thus reduces the background caused by nonspecific binding of<br>
antisera onto the surface.<br>
[0078] In ELIS As, it is more customary to use a secondary or tertiary detection means<br>
rather than a direct procedure. Thus, after binding of a protein or antibody to the well,<br>
coating with a non-reactive material to reduce background, and washing to remove<br>
unbound material, the immobilizing surface is contacted with the control biological<br>
sample to be tested under conditions effective to allow immune complex<br>
(antigen/antibody) formation. Detection of the immune complex then requires a labeled<br>
secondary binding ligand or antibody, or a secondary binding ligand or antibody in<br>
conjunction with a labeled tertiary antibody or third binding ligand.<br>
[0079] "Under conditions effective to allow immune complex (antigen/antibody)<br>
formation" means that the conditions preferably include diluting the antigens and<br>
antibodies with solutions such as BSA, bovine gamma globulin (BGG) and phosphate<br>
buffered saline (PBS)/Tween. These added agents also tend to assist in the reduction of<br>
nonspecific background.<br>
[0080] The "suitable" conditions mean that the incubation is at a temperature and for a<br>
period of time sufficient to allow effective binding. Incubation steps are typically from<br>
about 1 to 2 to 4 hours, at temperatures preferably on the order of 25 C to 27 C, or may be<br>
overnight at about 4 C or so.<br>
[0081] Following all incubation steps in an ELIS A, the contacted surface is washed so as<br>
to remove non-complexed material. A preferred washing procedure includes washing<br>
with a solution such as PBS/Tween, or borate buffer. Following the formation of specific<br>
immune complexes between the test sample and the originally bound material, and<br>
subsequent washing, the occurrence of even minute amounts of immune complexes may<br>
be determined.<br>
[0082] To provide a detecting means, the second or third antibody will have an associated<br>
label to allow detection. Preferably, this will be an enzyme that will generate color<br>
development upon incubating with an appropriate chromogenic substrate. Thus, for<br>
example, one will desire to contact and incubate the first or second immune complex with<br>
a urease, glucose oxidase, alkaline phosphatase or hydrogen peroxidase-conjugated<br>
antibody for a period of time and under conditions that favor the development of further<br>
immune complex formation (e.g., incubation for 2 hours at room temperature in a PBS-<br>
containing solution such as PBS-Tween).<br>
[0083] After incubation with the labeled antibody, and subsequent to washing to remove<br>
unbound material, the amount of label is quantified, e.g., by incubation with a<br>
chromogenic substrate such as urea and bromocresol purple or 2,2-azido-di-(3-ethyl-<br>
benzthiazoline-6-sulfonic acid [ABTS] and H2O2, in the case of peroxidase as the enzyme<br>
label. Quantitation is then achieved by measuring the degree of color generation (e.g.,<br>
using a visible spectra spectrophotometer).<br>
Kits<br>
[0084] In still further embodiments, the present invention concerns detection kits for use<br>
with the nucleic acid or immunodetection methods described above. Depending upon the<br>
type of assay to be utilized, a kit may comprise one or more primer pairs for amplification<br>
of a target nucleic acid sequence, one or more probes, such as labeled probes, to detect a<br>
genetic variant, and one or more control target sequences to confirm amplification and/or<br>
probe binding conditions. Controls may include, for example, specific target sequences<br>
for each allele of the 1747 C&gt;T SNP in ACTN3. Probes may also be specific for<br>
hybridization to the 1747 C&gt;T SNP alleles. Various other reagents of use, such as<br>
buffer, nucleotides, and polymerase may also be included.<br>
[0085] In kits for immunoassay of protein, immunodetection kits may comprise, in<br>
suitable container means, a target protein or peptide, or a first antibody that binds to a<br>
target protein or peptide, and an immunodetection reagent. The kits may comprise a first<br>
antibody specific for the target protein or peptide and a labelled second antibody specific<br>
for the first antibody. Alternatively, kits may comprise a first and a second antibody<br>
specific or selective for a protein of interest, with the second antibody labelled.<br>
Alternatively, the first and second antibody may be unlabeled and a third antibody,<br>
specific for the second antibody, may be included. Other standard reagents, such as buffer<br>
and various substrates or reactants used to develop a labelled antibody may also be<br>
included.<br>
[0086] The container means of the kits will generally include at least one vial, test tube,<br>
flask, bottle, syringe or other container means, into which a sample may be placed, and<br>
preferably, suitably aliquoted. Where a second or third binding ligand or additional<br>
component is provided, the kit will also generally contain a second, third or other<br>
additional container into which this ligand or component may be placed. Such kits may<br>
include injection or blow-molded plastic containers into which the desired vials are<br>
retained.<br>
Performance Testing<br>
[0087] In certain embodiments, the screening methods of use may include, in addition to<br>
ACTN3 assays, one or more performance based tests. Such performance tests may be<br>
used in combination with, for example, ACTN3 SNP testing or ACTN3 protein or mRNA<br>
assays. Various exemplary performance tests are discussed below. The skilled artisan<br>
will realize that the examples are not limiting and any performance assay known in the art<br>
may be used.<br>
VO2 max testing<br>
[0088] VO2 max testing provides athletes with a direct measure of their physiological<br>
potential. Maximum oxygen consumption rates under conditions of vigorous exercise<br>
are determined by methods well known in the art. Data includes aerobic and anaerobic<br>
thresholds, heart rate and speed, ventilatory parameters, maximum heart rate and heart<br>
rate zones.<br>
Anaerobic Threshold Testing (Blood Lactate &amp; Ventilatory)<br>
[0089] Anaerobic Threshold refers to the point in exercise where lactic acid production<br>
is equal to removal. This intensity is equivalent to a 60-120 min run or cycle<br>
depending on fitness, technique and experience. The test is conducted by<br>
simultaneously measuring ventilation as well as blood lactate levels. Although the<br>
ventilatory and blood lactate methods produce very similar results, they both accurately<br>
determine anaerobic threshold. Information provided by this test include blood lactate<br>
threshold and ventilatory threshold, heart rates at anaerobic threshold and speed (run)<br>
or watts (cycle) at anaerobic threshold<br>
Anaerobic Power and Capacity Testing (Wingate Test)<br>
[0090] The Wingate test determines leg power and capacity and is designed for power<br>
sport athletes. The test is a 30 second all out effort on a cycle ergometer that<br>
determines peak power and ability to resist fatigue. Data collected from a Wingate test<br>
includes: (30 s test) peak power (watts), absolute, relative and fatigue index (how fast<br>
power drops off over the 30 s test) and work (joules) (energy expenditure).<br>
Critical Power (CP)<br>
[0091] The goal of CP tests is to determine what is the optimal workload that an athlete<br>
can sustain for a given time period or distance. The most common CP tests may<br>
include CP (60 - 180s), time frame dependant on sport; and CP Time Trial.<br>
Resting Metabolic Rate (RMR)<br>
[0092] RMR is also referred to as Resting Energy Expenditure (REE). It is a non-<br>
invasive method of determining the minimal amount of calories (Kcal) an individual<br>
utilizes in a day. The higher the RMR, the more calories an individual bums. The<br>
results are directly measured by O2 and CO2 inspiration and expiration. One test<br>
protocol consists of no food or alcohol for 12 hours, no stimulants for 24 hours such as<br>
coffee and no exercise for 24-36 hours. The test is most commonly recommended for<br>
early in the morning. The individual is connected to a metabolic measuring machine<br>
for 30 min while lying on his back in a rested state. During the test, the individual<br>
breathes into the metabolic measuring machine through a mouthpiece and fitted hose.<br>
At the completion of the test, the following information is gathered: Metabolic Rate<br>
(RMR) - Kcal/day • Respiratory Rate (RR), Respiratory Exchange Ratio<br>
(RER),Ventilation and heart rate at rest % of Carbohydrates and Fat utilized at rest<br>
Speed / Power Testing<br>
[0093] Speed / Power Testing consists most commonly of three tests: Running Speed:<br>
Infrared Timing Lights (5 - 50 meters); a Vertical Jump &amp; Leg Power: Vertec<br>
apparatus and Agility Tests: Standard and Sport Specific. These tests assist in the<br>
analysis of an individuals capabilities in, for example, power sports).<br>
Strength / Flexibility Testing<br>
[0094] Strength / Flexibility testing generally consists of RM (resting muscle) strength:<br>
squat, bench, dead-lift, leg press; Muscular Endurance: repeated repetitions at a<br>
specified weight; Olympic Lifts: Clean &amp; Jerk, Snatch, Power Cleans, Power Snatch;<br>
Flexibility: standard and sport-specific and abdominal and lower back strength.<br>
Body Composition<br>
[0095] A body composition test may consist of a Harpenden skinfold calliper test<br>
(pinching the skin in several sites on the body such as under the arm, hip etc.) and<br>
estimating the percent body fat as well as estimating lean muscle mass and fat mass.<br>
Another method involves immersion in water in a tank with deflated lungs. Body fat is<br>
measured by a special measuring device that determines water displacement.<br>
Applicability of Methods<br>
[0096] While the disclosed methods are particularly suitable for the prediction of<br>
athletic performance in sprint/power-type sports and events in Caucasian individuals,<br>
the methods may also be suitable for use in any other ethnic group which generally<br>
shows a high frequency (i.e. preferably at least 5%, more preferably at least 10%, and<br>
most preferably at least 15%) of the 577XX genotype. After analyzing multiple<br>
Caucasians and several other ethnic groups, the null genotype if absent from an<br>
individual athlete such as the Zulus and certain Caucasian females appears to correlate<br>
with the potential to be a sprint/power elite athlete versus an endurance athlete. For<br>
example, the null genotype is common within the Native American population (29%),<br>
Asian population (25 %) and White Europeans (20%), PNG Highlanders (15%),<br>
African American population (13%) and the Aboriginal Australian population (10%).<br>
[0097] Talent search programs may utilize the methods of the present invention by<br>
themselves or in combination with similar methods for genotyping individuals in<br>
respect of other genes linked to athletic performance. Other methods that may be<br>
combined with the methods disclosed are based upon performance data and phenotypic<br>
predictors (eg. height and build) and the like. Thus, the results of the methods of the<br>
present invention may be used to select, ox at least assist in the selection of, young<br>
individuals with elite athlete potential and/or to provide guidance on the type of sport<br>
that a young individual may choose to specialize.<br>
[0098] In another embodiment, training programs may be devised for a potential or<br>
current elite athlete that have greater chance of success, based on the knowledge of<br>
genetic factors that will predict a person"s training capability (e.g. levels of ACTN3<br>
protein or mRNA and/or SNP detection). Individualized training programs may focus<br>
on specific talents (determined from genetic makeup) by identifying the type of training<br>
that is most likely to be successful. This would help to narrow the small margin<br>
between success and failure at the elite level, avoid unnecessary fatigue from excessive<br>
training without the expected gains (eg. the genetic potential is not there); reduce<br>
wasted resources and premature "burn out"; and may enhance long-term goals and self<br>
esteem in an individual athlete. Resources are wasted every time an individual with<br>
elite athlete potential is removed from a program because he/she cannot achieve<br>
success. At a personal level, the effort and sacrifices already undertaken by such<br>
individuals can adversely affect their life goals and self esteem. In these situations,<br>
knowledge of the genetic makeup alone or in combination with other predictors may<br>
help to clarify why success has not been achieved, and will assist in directing the<br>
individual to more realistic life goals that may include a more appropriate sport.<br>
[0099] Therefore, in one embodiment, identifying an improved training program for an<br>
athlete may involve the determination of a specific genotype of a targeted gene (e.g.<br>
ACTN3 genotype) of an athlete. Another example of developing a training program foi<br>
a potential or current athlete may involve combining one or more tests for a targeted<br>
molecule with other performance assessing tests as indicated previously and analyzing<br>
the results of the two or more tests to develop a program.<br>
EXAMPLES<br>
Example 1: Screening for the ACTN3 null (577XX) genotype in elite athletes.<br>
Materials and Methods<br>
[0100] Human genomic DNA was isolated from blood from a pool of elite athletes<br>
(108 endurance athletes and 83 sprint athletes), 88 African Zulu individuals and 152<br>
control Australian Caucasian individuals, by phenolrchloroform extraction following<br>
cell lysis with Triton-XlOO and digestion with proteinase K. Exon 16 of ACTN3 was<br>
amplified from genomic DNA. The primers corresponding to adjacent intronic<br>
sequences for exon 16 were:<br>
forward 5"CTGTTGCCTGTGGTAAGTGGG3" (SEQ ID NO:1)<br>
reverse 5"TGGTCACAGTATGCAGGAGGG3" (SEQ ID NO:2)<br>
[0101] The PCR reaction cycle for the primers was: 35 cycles at 94°C for 30s and then<br>
72°C for 1 min, with a final extension of 94°C for 10 min. The R577X alleles (codons<br>
CGA and TGA respectively) can be distinguished by the presence (577X) or absence<br>
(577R) of a Dde I (C|TNAG) restriction site in Exon 16. 577R (wild type) PCR<br>
products have 205 bp and 86 bp fragments; while 577X PCR products have 108 bp, 97<br>
bp and 86 bp fragments. Digested PCR fragments were separated by 10%<br>
polyacrylamide gel electrophoresis and visualized by staining with ethidium bromide.<br>
Results and Discussion<br>
[0102] Results of the genotyping assays are shown in Table 2. ACTN3 genotyping<br>
was conducted in elite athletes (i.e. individuals who perform at the highest levels in<br>
terms of endurance, speed and/or strength). Compared to controls, elite sprint athletes<br>
had a low frequency of the ACTN3 null mutation 577XX (6% versus 18% in a control<br>
Caucasian population; p
Since, the force-generating capacity of type 2 muscle fibers at high velocity, the speed<br>
and tempo of movements, and the capacity of the individual to adapt to exercise<br>
training, all appear to be strongly genetically influenced, it is considered that ACTN3<br>
genotype is likely to be a factor influencing normal variation in muscle function in the<br>
general population. Based on these results, ACTN3 genotyping is shown to be of<br>
considerable potential in the selection, or at least to assist in the selection, of young<br>
individuals with elite athletic potential.<br>
Example 2<br>
Methods<br>
436 unrelated Caucasian controls were genotyped from three different sources<br>
(150 blood donors, 71 healthy children participating in an unrelated study, and 215<br>
healthy adults participating in a talent-identification program with the Australian<br>
Institute of Sport), through use of the genotyping methodology described by Mills et<br>
al.(2001). Sex was known for 292 female controls and 134 male controls. 429 elite<br>
Caucasian athletes were genotyped from 14 different sports. For the purposes of the<br>
example, athletes were defined as "elite" if they had represented Australia in their sport<br>
at the international level; 50 of the athletes had competed in Olympic Games.<br>
[0103] Given the localization of cc-actinin-3 in fast skeletal-muscle fibers, it was<br>
hypothesized that deficiency of a-actinin-3 would reduce performance in sprint/power<br>
events and would therefore be less frequent in elite sprint athletes. To test this<br>
hypothesis, the genotypes of a subset of 107 elite athletes (72 male and 35 female) were<br>
analyzed, classified a priori as specialist sprint/power athletes, blinded to genotyping<br>
results. This group comprised 46 track athletes competing in events of 800 m, 42<br>
swimmers competing in events of 200 m, 9 judo athletes, 7 short-distance track<br>
cyclists, and 3 speed skaters. For comparison, a subset of 194 subjects (122 male and<br>
72 female) classified independently as specialist endurance athletes and analyzed,<br>
including 77 long-distance cyclists, 77 rowers, 18 swimmers competing over distances<br>
of 400 m, 15 track athletes competing in events of 5,000 m, and 7 cross-country skiers.<br>
Thirty-two sprint athletes (25 male and 7 female) and 18 endurance athletes (12 male<br>
and 6 female) had competed at the Olympic level. Because of the stringency of the<br>
classification criteria, 128 of the elite athletes could not be unambiguously assigned<br>
into either the sprint/power or endurance groups and were excluded from subsequent<br>
analyses.<br>
[0104] To test for homogeneity of ACTN3 allele and genotype frequencies between<br>
athlete and control groups, the log-linear modeling approach was used as described by<br>
Huttley and Wilson (2000), implemented in the statistical programming language R<br>
(version 1.6.2), through use of a package (contributed by J. Maindonald; available from<br>
The R Project for Statistical Computing Web site). "X" 2 values were estimated using<br>
genotype numbers for comparisons between athletes and controls. The genotypic<br>
profiles of the three control groups (150 blood donors, 71 healthy children, and 215<br>
healthy adults) did not differ significantly from one another (x 2=0.19; P =.996) nor<br>
from a previously genotyped group of 107 white Europeans (Mills et al. 2001),<br>
suggesting that the genotype frequencies in the control cohort are representative of a<br>
broader Caucasian population. ACTN3 genotype frequencies did not vary significantly<br>
between male and female control subjects, and, overall, there was no significant<br>
deviation from Hardy-Weinberg (H-W) equilibrium.<br>
[0105] ACTN3 genotyping data from the control, sprint/power, and endurance groups<br>
are summarized in TABLE 2 and FIG.l. There were no significant allele or genotype<br>
frequency differences between the elite athlete group as a whole and the controls.<br>
However, when the athletes were divided into sprint/power and endurance groups and<br>
compared with controls, there was strong evidence of allele frequency variation (x2[df=s]<br>
= 23; P<.001 there were significant allele frequency differences between sprint></.001>
athletes and controls for both males( x2 [df=i] = 14.8; P<.001 females x2></.001>
P<.01 sprint athletes had a lower frequency of the null></.01>
genotype (6% vs. 18%), and no female elite sprint athletes or sprint Olympians were<br>
577XX. The sprint athlete group also had a higher frequency of the 577RR genotype<br>
(50% vs. 30%) and a lower frequency of the heterozygous 577RX genotype (45% vs.<br>
52%), compared with controls. Elite endurance athletes had a slightly higher frequency<br>
of the 577XX genotype (24%) than did controls (18%). More importantly, allele<br>
frequencies in sprint and endurance athletes deviated in opposite directions and differed<br>
significantly from each other in both males (x2 [df=i] = 13.3; P<.001 and females></.001>
[dfoi] = 5.8; P<.05 the differences between two groups effectively cancelled each></.05>
other out, explaining the lack of association when the entire elite athletic cohort was<br>
compared with the control group.<br>
[0106] Overall, there was also evidence of genotype variation that is not explained by<br>
allele frequency differences (x2[df=5] = 16.7; P
disequilibrium coefficients among groups, despite there being no evidence for<br>
departure from H-W equilibrium overall. The effect was restricted to female sprint<br>
(x2[df=1] = 7.4; P<.01 endurance p athletes with more></.01>
heterozygous female sprint athletes than expected at H-W equilibrium (20 vs. 15) and<br>
fewer than expected heterozygous female endurance athletes (25 vs. 36). The allele-<br>
frequency-independent genotype differences between female sprint and endurance<br>
athletes were highly significant (x2[df=1] = 13.8; P
suggesting that the effect of ACTN3 genotype on performance differs between males<br>
and females.<br>
[0107] These findings suggest that the ACTN3 577R allele provides an advantage for<br>
power and sprint activities. No female elite sprint athletes in the sample were a-<br>
actinin-3 deficient (compared with 8% of males). In males, the androgen hormone<br>
response to training is likely to make a significant contribution to improvements in<br>
performance, so that the relative effect of a-actinin-3 on muscle power may be reduced.<br>
Interestingly, all male Olympian power athletes in the cohort had at least one copy of<br>
the functional 577R allele of ACTN3 (associated with the presence of oc-actinin-3 in<br>
skeletal muscle), suggesting that "every variable counts" at the highest levels of<br>
sporting competition. Although at least 73 genetic loci have been associated with<br>
fitness and performance phenotypes (Rankinen et al. 2002 "The human gene map for<br>
performance and health-related fitness phenotypes: the 2001 update". Med Sci Sports<br>
Exerc 34:1219-1233), ACTN3 is the first structural skeletal-muscle gene for which<br>
such an association has been demonstrated.<br>
[0108] The a-actinin-3 protein may promote the formation of fast-twitch fibers or alter<br>
glucose metabolism in response to training. In addition, a-actinin-3 may be<br>
evolutionarily optimized for the minimization of damage caused by eccentric muscle<br>
contraction. The Z line in fast, glycolytic fibers is the structure most vulnerable to<br>
exercise-induced injury resulting in morphological damage and degradation of<br>
associated proteins, including the a-actinins (Friden and Lieber 2001, "Eccentric<br>
exercise-induced injuries to contractile and cytoskeletal muscle fiber components Acta<br>
Physiol Scand 171:321-326).<br>
[0109] If the 577XX genotype enhances endurance performance as the 577R allele<br>
appears to enhance sprint-ability, then the 577R and 577X alleles may be maintained in<br>
the population because they both confer selective advantages under different<br>
environmental conditions and are thus kept at high population frequencies by balancing<br>
selection.<br>
Example 3<br>
[0110] FIG. 1 represents a histogram compilation of ACTN3 genotype frequency in<br>
controls, elite sprint/power athletes, and endurance athletes. Compared with healthy<br>
Caucasian controls, there is a marked reduction in the frequency of the ACTN3 577XX<br>
genotype (associated with a -actinin-3 deficiency) in elite Caucasian sprint athletes;<br>
remarkably, none of the female sprint athletes or sprint athletes who had competed at<br>
the Olympic level (25 males and 7 females) were a-actinin-3 deficient. Conversely,<br>
there is a trend toward an increase in the 577XX genotype in endurance athletes,<br>
although this association reaches statistical significance only in females. Error bars<br>
indicate 95% CIs.<br>
[0111] It will be appreciated by persons skilled in the art that numerous variations<br>
and/or modifications may be made to the invention as shown in the specific<br>
embodiments without departing from the spirit or scope of the invention as broadly<br>
described. The present embodiments are, therefore, to be considered in all respects as<br>
illustrative and not restrictive.<br>
* * *<br>
[0112] All of the COMPOSITIONS, METHODS and APPARATUS disclosed and<br>
claimed herein can be made and executed without undue experimentation in light of the<br>
present disclosure. While the compositions and methods of this invention have been<br>
described in terms of preferred embodiments, it are apparent to those of skill in the art<br>
that variations may be applied to the COMPOSITIONS, METHODS and<br>
APPARATUS and in the steps or in the sequence of steps of the methods described<br>
herein without departing from the concept, spirit and scope of the invention. More<br>
specifically, it are apparent that certain agents that are both chemically and<br>
physiologically related may be substituted for the agents described herein while the<br>
same or similar results would be achieved. All such similar substitutes and<br>
modifications apparent to those skilled in the art are deemed to be within the spirit,<br>
scope and concept of the invention as defined by the appended claims.<br>
TABLE 4. Symbols, full names, and cytogenic location of nuclear and mitochondrial<br>
genes of the 2002 Human Gene Map for Performance and Health-Related Fitness<br>
Phenotypes.<br>
Gene or Locus Name Location<br>
AB<br>
ACADVL Acyl coenzyme A dehydrogenase, very long chain 17pl3-p11<br>
ACE Angiotensin I converting enzyme 17q23<br>
ADRA2A Alpha-2A-adrenergic receptor 10q24-q26<br>
ADRB1 Adrenergic, beta-1-, receptor 10q24-q26<br>
ADRB2 Beta-2-adrenergic receptor 5q31-q32<br>
ADRB3 Beta-3-adrenergic receptor 8pl2-pl 1.2<br>
AGT Angiotensinogen Iq42-q43<br>
ANG Angiogenin, ribonuclease, RNase A family, 5 14qll.l-qll.2<br>
APOE Apolipoprotein E 19ql3.2<br>
ATP1A2 ATPase, Na_/K_ transporting, alpha-2 polypeptide Iq21-q23<br>
ATP1B1 ATPase, Na_/K_ transporting, beta 1 polypeptide Iq22-q25<br>
BDKRB2 Bradykinin receptor B2 14q32.1-q32.2<br>
CDEFG<br>
CASO2 Calsequestrin 2 (cardiac muscle) Ipl3.3-p11<br>
CFTR Cystic fibrosis transmembrane conductance regulator, ATP-binding cassette (subfamily<br>
C, member 7) 7q31.2<br>
CKM Creatine kinase, muscle 19ql3.2-ql3.3<br>
CNTF Ciliary neurotrophic factor Ilql2.2<br>
CPT2 Camitine palmitoyltransferase 2 Ip32<br>
COL1A1 Collagen, type I, alpha 1 17q21.3-q22.1<br>
EDN1 Endothelin 1 6p24.1<br>
ENO3 Enolase 3, (beta, muscle) 17pter-pl 1<br>
FABP2 Fatty acid binding protein 2 4q28-q31<br>
FGA Fibrinogen, A alpha polypeptide 4q28<br>
FGB Fibrinogen, B beta polypeptide 4q28<br>
GDF8 (MSTN) Growth differentiation factor 8 (myostatin) 2q32.2<br>
GNB3 Guanine nucleotide binding protein (G protein), beta polypeptide 3 12pl3<br>
HIKLM<br>
HLA-A Major histocompatibility complex, class I, A 6p21.3<br>
HP Haptoglobin 16q22.1<br>
IGF1 Insulin-like growth factor 112q22-q23<br>
IGF2 Insulin-like growth factor 2 Ilpl5.5<br>
IL-6 Interleukin-6<br>
KCNQ1 K_voltage-gated channel, KQT-like subfamily, member 1 Ilpl5.5<br>
LDHA Lactate dehydrogenase A 1 Ipl5.4<br>
LPL Lipoprotein Iipase 8p22<br>
MTCO1 Cytochrome c oxidase I mtDNA 5904-7445<br>
MTCO3 Cytochrome c oxidase III mtDNA 9207-9990<br>
MTCYB Cytochrome b mtDNA 14747-15887<br>
MTND1 NADH dehydrogenase 1 mtDNA 3307-4262<br>
MTND4 NADH dehydrogenase 4 mtDNA 10760-12137<br>
MTND5 NADH dehydrogenase 5 mtDNA 12337-14148<br>
MTTE Transfer RNA, mitochondrial, glutamic acid mtDNA 14674-14742<br>
MTTITransfer RNA, mitochondrial, isoleucine mtDNA 4263^4331<br>
MTTK Transfer RNA, mitochondrial, lysine mtDNA 8295-8364<br>
MTTL1 Transfer RNA, mitochondrial, leucine 1 (UUR) mtDNA 3230-3304<br>
MTTL2 Transfer RNA, mitochondrial, leucine 2 (CUN) mtDNA 12266-12336<br>
MTTM Transfer RNA, mitochondrial, methionine mtDNA 4402-4469<br>
MTTTTransfer RNA, mitochondrial, threonine mtDNA 15888-15953<br>
MTTY Transfer RNA, mitochondrial, tyrosine mtDNA 5826-5891<br>
MyHC myosin Heavy-chain<br>
NOPQRSTUV<br>
NOS3 Nitric oxide synthase 3 (endothelial cell) 7q36<br>
NPY Neuropeptide Y 7pl5.1<br>
PAI1 Plasminogen activator inhibitor 1 7q21.3-q22<br>
PFKM Phosphofructokinase, muscle 12ql3.3<br>
PGAM2 Phosphoglycerate mutase 2 (muscle) 7pl3-pl2<br>
PGK1 Phosphoglycerate kinase 1 Xql3<br>
PHKA1 Phosphorylase kinase, alpha 1 (muscle) Xql2-ql3<br>
PON1 Paraoxonase 1 7q21.3<br>
PPARA Peroxisome proliferative activated receptor, alpha 22ql3,31<br>
PPARG Peroxisome proliferative activated receptor, gamma 3p25<br>
PYGM Phosphorylase, glycogen, muscle Ilql2-ql3.2<br>
RYR2 Ryanodine receptor 2 (cardiac) Iq42.1-q43<br>
SGCA Sarcoglycan, alpha (50kDa dystrophin-associated glycoprotein) 17q21<br>
S100A1 S100 calcium binding protein Al Iq21<br>
SUR Sulfonyl urea receptor lip 15.1<br>
TGFB1 Transforming growth factor beta 1 19ql3.2<br>
UCP2 Uncoupling protein 2 Ilql3<br>
UCP3 Uncoupling protein 3 Ilql3<br>
VDR Vitamin D (1,25-dihydroxyvitamin D3) receptor 12ql2-ql4<br>
The gene symbols, names and cytogenetic locations are from the Locus Link web site<br>
(http://www.ncbi.nlm.nih.gov/LocusLink) available from the National Center for<br>
Biotechnology<br>
Information (NCBI). For mitochondrial DNA, locations are from the human<br>
mitochondrial genome data base (http://www.mitomap.org).<br>
WE CLAIM:<br>
1. A method to predict athletic performance in an individual comprising :<br>
(a) performing an in vitro screening on a sample from the individual for the presence of one<br>
or more genetic variations in the a-actinin-3 (ACTN3) gene linked to athletic performance ; and<br>
(b) predicting athletic performance based on the presence of the one or more genetic<br>
variations,<br>
wherein the athletic performance is selected from at least one of sprint performance, enduring<br>
performance, power performance and strength performance.<br>
2. The method as claimed in claim 1, wherein the individual is a human.<br>
3. The method as claimed in claim 1, wherein the individual is a horse, a dog or a camel.<br>
4. The method as claimed in any one of claims 1 to 3, which involves screening the individual for<br>
a 1747 OT single nucleotide polymorphism (SNP) in the ACTN3 gene.<br>
5. The method as claimed in any one of claims 1 to 4, which involves genotyping the individual at<br>
the ACTN3 locus.<br>
6. The method as claimed in claim 5, wherein the presence of at least one copy of the 577R allele<br>
of the ACTN3 gene is positively associated with sprinting or power performance.<br>
7. The method as claimed in claim 6, wherein genotyping the individual as a 577RR genotype is<br>
positively associated with sprinting or power performance.<br>
8. The method as claimed in claim 6, wherein genotyping the individual as a 577XX genotype is<br>
negatively associated with sprinting or power performance.<br>
9. The method as claimed in claim 6, wherein genotyping the individual as a 577XX genotype is<br>
positively associated with endurance performance.<br>
10. The method as claimed in claim 6, wherein genotyping the individual as a 577RX genotype is<br>
positively associated with sprinting or power performance in female individuals.<br>
11. The method as claimed in claim 6, wherein genotyping the individual as a 577RX genotype is<br>
negatively associated with endurance performance in female individuals.<br>
12. The method as claimed in any one of claims 1 to 11, which involves measuring the amount of<br>
ACTN3 protein present in the individual"s skeletal muscle.<br>
13. The method as claimed in claim 12, wherein the amount of ACTN3 protein is measured using<br>
an antibody specific for the ACTN3 protein.<br>
14. The method as claimed in any one of claims 1 to 11, which involves measuring the amount of<br>
ACTN3 messenger RNA (mRNA) expressed in the individual"s skeletal muscle.<br>
15. The method as claimed in claim 4, which involves identifying the 1747 OT SNP alleles in the<br>
individual"s genomic DNA by DNA sequencing, allele - specific hybridization, allele - specific<br>
amplification or restriction fragment length polymorphism analysis.<br>
16. The method as claimed in claim 4 or 15, which involves screening the individual for the<br>
presence of one or more additional SNPs in the ACTN3 gene.<br>
17. The method as claimed in claim 16, wherein the one or more additional SNPs are selected from<br>
the group consisting of the SNPs listed in TABLE 3.<br>
18. The method as claimed in any one of claims 1 to 17, which involves screening the individual for<br>
the presence of one or more genetic variations in at least one other gene.<br>
19. The method as claimed in claim 18, wherein the at least one other gene is selected from the<br>
group consisting of the genes listed in TABLE 4.<br>
20. The method as claimed in claim 19, which involves screening the individual for the presence of<br>
the ACE (angiotensin-converting enzyme) I allele and the ACE D allele.<br>
21. The method as claimed in claim 20, wherein the ACE I allele is positively associated with<br>
endurance performance.<br>
22. The method as claimed in claim 20, wherein the ACE D allele is positively associated with<br>
sprinting or power performance.<br>
23. The method as claimed in claim 19, which involves screening the individual for the presence or<br>
absence of an ADRA2A (Alpha-2A-adrenergic receptor) allele.<br>
24. The method as claimed in any one of claims 1 to 23, which involves screening the individual<br>
using a test selected from the group consisting of VC2 maximum, anaerobic threshold test, Wingate<br>
test, critical power, resting metabolic rate, body composition, speed testing, power testing, strength<br>
testing, flexibility testing, muscle biopsy, fast twitch fiber test and slow twitch fiber test.<br>
25. A method of optimizing a training program comprising :<br>
(a) performing an in vitro screening on a sample from the individual for the presence of one<br>
or more genetic variations in the a-actinin-3 (ACTN3) gene ; and<br>
(b) selecting the individual"s training program to optimize strength performance, power<br>
performance or endurance performance.<br>
26. The method as claimed in claim 25, wherein the individual is a human, a horse, a dog or a<br>
camel.<br>
27. The method as claimed in claim 25 or 26, which involves screening the individual for a 1747<br>
C&gt;T single nucleotide polymorphism (SNP) in the ACTN3 gene.<br>
28. The method as claimed in any one of claims 25 to 27, which involves genotyping the individual<br>
at the ACTN3 locus.<br>
29. A method of selecting a sport or sporting event for an individual comprising :<br>
(a) performing an in vitro screening on a sample from the individual for the presence of one<br>
or more genetic variations in the a-actinin-3 (ACTN3) gene ; and<br>
(b) selecting a sprint / power type sport or event or, otherwise, and endurance sport or event<br>
on the basis of the result of the said screening.<br>
30. The method as claimed in claim 29, wherein the individual is a human, a horse, a dog or a<br>
camel.<br>
31. The method as claimed in claim 29 or 30, which involves screening the individual for a 1747<br>
C&gt;T single nucleotide polymorphism (SNP) in the ACTN3 gene.<br>
32. The method as claimed in any one of claims 25 to 27, which involves genotyping the individual<br>
at the ACTN3 locus.<br>
The present invention concerns novel methods of selecting or matching a sport or sporting event to an individual (e.g.<br>
a sprint/power sport or an endurance sport) and predicting athletic performance, the methods involving assessing ACTN3 genotype.<br>
In alternative embodiments, training regimens may be optimally designed for athletes by assessing the ACTN3 genotypes. Certain<br>
embodiments concern combining the assessment of the ACTN3 genotype with other known fitness-related genes to better assess the<br>
athletic potential of an individual. In addition, the genorypic analysis of the ACTN3 gene may be combined with physiological tests,<br>
physical measurements and/or psychological assessments to more optimally design a training regimen for an individual athlete.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1sZXR0ZXIgcGF0ZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1OTkta29sbnAtMjAwNS1yZXBseSBmaXJzdCBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">00599-kolnp-2005-reply first examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTk5LUtPTE5QLTIwMDUtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">599-KOLNP-2005-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTk5LUtPTE5QLTIwMDUtRk9STSAyNy5wZGY=" target="_blank" style="word-wrap:break-word;">599-KOLNP-2005-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTk5LUtPTE5QLTIwMDUtRk9STS0yNy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">599-KOLNP-2005-FORM-27-1.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216885-a-highly-weatherable-moisture-curable-one-componentpolyurethane-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216887-automatic-transfeer-switch-systems-and-controllers.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216886</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>599/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENETIC TECHNOLOGIES LIMITED.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>60 HANOVER STREET, FITZROY, VIC 3065</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NORTH , KATHRYN, NANCE.</td>
											<td>18A COOK STREET, GLEBE, NSW 2037</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2003/001202</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-09-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2002951411</td>
									<td>2002-09-16</td>
								    <td>Australia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216886-a-method-of-predicting-athletic-performance-in-an-individual by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:08:41 GMT -->
</html>
