<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216847-a-method-for-identifying-markers by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:06:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216847:A METHOD FOR IDENTIFYING MARKERS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR IDENTIFYING MARKERS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention is a method for identifying markers associated with the presence of a predetermined characteristic comprising accumulating spectral data from a sample known to have a predetermined characteristic, identifying a spectral feature which is indicative of the predetermined characteristic, and identifying a marker associated with the spectral feature (FIG.nil).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR IDENTIFYING MARKERS<br>
Field of the Invention<br>
The present invention relates to methods for identifying markers associated with the presence of a known characteristic in a sample. In particular, the markers are fluorescent molecules indicative of the presence of a disease or involved in the disease process.<br>
Background of the Invention<br>
In addition to genes and proteins, there exist categories of biological molecules, including carbohydrates and small organic molecules that hold major physiological significance. The variations in these molecules represent the complex interaction of the organism"s genome and proteome with environmental factors that include diseases. Alterations in a subject"s profile may have linkage to an acute disease or correlations with disease progression.<br>
Establishing the relationship between the profile of small organic molecules and specific diseases provides another pathway for building new approaches to early diagnosis and treatment of infectious, cancerous, and metabolic diseases. Perhaps of greatest importance is the potential power of this approach for the prospective detection of indicators of "subclinical" disease in healthy individuals and development of individualized disease prevention strategies. The analysis of these non-genetic molecules seeks to correlate the effects of the broadest range of environmental influences (i.e., infectious agents, diet, exposure to toxins) on the complete portfolio of biological molecules found in an organism over time.<br>
The invention exploits the high sensitivity and information content of natural fluorescence (or intrinsic fluorescence) as its primary approach to establishing disease spectral profiles. The natural portfolio of fluorescent molecules present in human cells and fluids include many structurally diverse molecular families with widely divergent biologic roles. The array of intrinsically fluorescent molecules that a host possesses therefore represents a broad view of the physiological status of the organism. Furthermore, aberrations of any biochemical pathway are likely to ultimately lead to a disruption of the normal physiologic level of one or more of these natural fluoresce. As markers of a disease state, these intrinsically fluorescent molecules hold great value as vehicles both to screen for disease directly and for diagnostic and therapeutic development.<br>
Summary of the Invention<br>
Spectra Molecular Informatics (SMI) is a method for identifying associations between specific molecuies and specific diseases. By applying SMI, a disease-related discriminatory spectral signal may be identified. The method preferably monitors intrinsic fluorescence for identification of the disease-related signal, although absorbance, phosphorescence, Raman spectroscopy, extrinsic fluorescence, chemically altered intrinsic fluorescence, or other optical signals could be exploited for these purposes.<br>
The method of the invention preferably includes accumulating spectral data, which is preferably multidimensional. The data preferably include intrinsic fluorescence arising from samples having a known disease state. The method further includes the identification of spectral signals, which are preferably indicative of the<br>
presence of disease, more preferably indicative of the presence of a specific disease. The identification step preferably includes subjecting the spectra to one or more data reduction steps. Based upon the spectral signals, preferably the spectral signals that are indicative of a specific disease, compounds associated with the appearance of the spectral signals are identified, preferably to at least the extent that some structural or physio-chemical properties useful for determining the presence of the molecule in a sample are characterized. For example, the spectral signals may be used in a separative technique such as chromatography to identify and preferably isolate the molecules carrying the discriminatory signal.<br>
In one embodiment, the identified markers are used to determine the presence of the disease based upon detection of the presence of the molecule in a sample acquired from an individual.<br>
In another embodiment the identified molecules are used to discover drugs or other treatment modalities useful in treating the disease. For example, in one embodiment a chemical library is searched for substances that interact with the identified molecule or modulate the activity of the molecule. By providing identified molecules that are known to be associated with a particular disease the present invention provides a disease-focused strategy to finding new drugs.<br>
In another embodiment the identification of molecules are used to identify biochemical pathways,, such as enzyme pathways, exhibiting aberrant behavior, such as up or down regulation, associated with the disease. This may include identificatign of precursor compounds associated with the disease.<br>
In one embodiment, the present invention relates to the identification of intrinsicallv fluorescent molecules as indicators of the presence of disease. This<br>
category of molecules typically includes smaller organic molecules containing, for example, well-defined ring structures and/or several complex bond structures. These fluorescently active biomolecules are important indicators of disrupted physiology in virtually any disease. The molecules identified by the invention are preferably distinct from the DNA and proteins that are the subject of Genomics and Proteomics respectively, and thus represent an import pathway to obtaining disease-specific information. Thus present invention address problems that are not easily approached by genomic or proteonomic methods.<br>
In another embodiment, the Spectra-Molecular Informatics of the invention is applied to any disease where additional diagnostic markers and therapeutic targets would have particular clinical value. The method includes the parallel identification of new markers for cancer tvoes. neurological diseases, heart disease, and other selected conditions.<br>
Yet another aspect of the invention relates to industries such as veterinary science, food and beverage quality control, chemical contaminant analysis, and the characterization and detection of biological and chemical weapons in which the informatics method of the invention is used to obtain data indicative of, for example, the purity or quality of a particular material.<br>
Detailed Description<br>
Animal and human plasma, serum, urine, cerebral spinal fluid and other biological fluids are complex mixtures of proteins, lipids and metabolites representing<br>
the immunologic, hormonal, metabolic and nutritional status of individuals. These fluids include molecules that are intrinsically fluorescent (i.e., can be excited to emit a<br>
spectrum of light without any added reagents). Due to the complex nature of these biological fluids, disease-specific fluorescent signatures may be partly or completely obscured by signals common to all individuals, and methods to enhance differential signals are commonly employed. Fluorescent molecules (fluorophores) in plasma exist both bound to proteins or free in solution. The invention uses selective sample extraction methods to permit analysis of both bound and free fluorphores and a spectral database for these preparations from normal and disease-infected individuals.<br>
In the invention, these sample preparation tools have been coupled to comprehensive spectral surveys in which the intrinsic fluorescence is analyzed over a range of excitation and emission wavelengths. The invention includes a spectral library from complementary preparation methods, which yields a high-resolution view of the fluorescent signature of a biological fluid sample.<br>
T00he invention utilizes algnrithms to analyse the spqotral databa.se. This process includes: 1) the building of mathematical models of fluorescence spectra from normal and infected individuals, 2) the objective testing of each model, and 3) the iterative modification of these models based upon the inclusion test sample spectra and reoptimization. This procedure is initiated by the extraction of spectral features from normal and infected fluid by multivariate statistical methods, including Principal and Independent Component Analysis, to identify the major parameters of the spectra that carry disease discrimination. These parameters become the components of linear and non-linear mathematical "discriminators" functions, which are models of disease-specific spectral differences.<br>
The fluorescence obtained from the fluid sample represents the aggregate spectra of many fluorescent molecules. The isolation and characterization of the<br>
specific molecules that give rise to fluorescence-based discrimination is an important complement to the spectral discrimination for both the development of molecule-directed diagnostics and therapeutics. The present invention exploits discriminatory spectral information to define appropriate conditions for molecular isolation from the effective sample preparation methods. This effort identifies the discriminatory molecules in effective fluorescent assays and provides the critical molecular components.<br>
The method of the present invention comprises:<br>
1. Spectral data accumulation;<br>
2. Identification of disease-specific spectral signals; and<br>
3. Identification of molecules carrying disease-specific spectral properties. Particular embodiments of the invention may also include:<br>
4. Development of prototype fluorescent assay for target molecule and validation of the molecular target; and<br>
5. Market assay development.<br>
The invention includes sample preparation methods to selectively amplify the signal from different classes of molecules. The sample preparation methods include, but not limited to, dilutions with varying formulations (e.g. pH, salts, buffers), acid/base extractions, organic solvent extractions (including 1,2 and 3 phase systems), temperature induced fractionation, size fractionation (by filtration, chromatography, ultracentrifugation, etc.), super-critical fluid extraction, differential extraction by chemical modification coupled to any of these methods, centrifugation or filtration coupled to any of these methods, and other known methods. A matrix of preparative methods and excitation wavelengths constitutes the Standard Spectral Survey and<br>
represents a level of database complexity necessary for comparative spectral testing between normal and diseased subjects. The excitation wavelengths that can be utilized by the invention is only limited by the irradiating sources available. The wavelengths most prevalent commercially today range from 190-1200nm.<br>
Based on the results of the Standard Spectral Survey phase, the invention utilizes methods with enhanced selectivity and spectral resolution in a subsequent Advanced Spectral Profile phase. These methods include, but are not limited to, for fluorescence: fine emission wavelength selection, focused spectral data collection, increased radiation power; or other methods including: infrared spectroscopy, Raman spectroscopy, dual photon fluorescence, phosphorescence, X-ray fluorescence.<br>
The invention subjects the compiled data of both the Standard Spectral Survey and Advanced Spectral Profile to multiple analytical strategies to identify spectral patterns and further characterize spectral differences between normal and disease subjects. These analytical strategies include, but are not limited to, simple processing (subtractions, normalization, user-specified computations), mathematical transformations of data (e.g. first, second, third and fourth derivatives, Fourier transformation)., Principal Component Analysis (PCA) and Independent Component Analysis (ICA). Some of the spectral comparative computational methods employ multivariate analytical strategies that isolate key differentiating features of the spectra. Comparative methods include comparison of selected spectral variables, comprehensive methods that fully analyze underlying spectral features such as, but not limited to, discrimination function development using linear and non-linear combinations of spectral data, statistical model development and testing, genetic algorithms, as well as computationally "intelligent" methods such as a neural net.<br>
Based on results of the Standard Specfral Survey and Advanced Spectral Profile, the conditions are defined for molecular identification. The wavelength providing the most significant discriminatory signal is selected for molecular detection coupled to chromatographic methods. The spectral patterns of the molecules eluted from systems reiterate the discrimination of the aggregate preparations. The confirmation of this effect by comparison of multiple diseased and control samples provides significant scientific validation of the molecular marker of the discriminatory signals.<br>
The molecular constituents that contribute to the discriminatory signal are then purified and structural identification by mass sriectrometrv is established.<br>
In some cases, the native fluorescence of the target molecule will permit its direct detection in patient samples, however, in other cases, physiological conditions will prevent the detection of the target molecule based on intrinsic fluorescence in sample preparations. The invention may utilize an assay for the subject target molecules that employs an alternative fluorogenic molecule linked to a second entity that will specifically interact with the target molecule. This approach will validate the presence of the target molecule in the disease process being studied and lead to a family of specific and highly sensitive test procedures that will efficiently utilize a common platform instrument such as that described in U.S. Patent No. 6,265,151.<br>
The application of SMI is typically directed to medical applications where the need for specific molecular information is well recognized, but has many additional applications in pharmaceutical process control, food and beverage processing, and for environmental detection of toxic substances, for example. For drug processing and food and beverage manufacturing, SMI technology will significantly reduce the extent of product recalls and reduce the economic impact of<br>
these events. Specific applications include but are not limited to: bacterial testing in meat and poultry, e.g., salmonella and E. coli, in process testing for beer and wine manufacture, fruit juice blending, and vegetable oil extraction. In each processed food product noted above, a standard color range is desirable for product release and progressive changes will occur throughout the production process that can be analyzed using SMI. The availability of real-time spectral data would be of great value in directly regulating final product quality.<br>
The present invention is particularly well suited for the rapid detection of small fluorescent molecules and can be further enhanced to detect non-fluorescent molecules with other specific spectral signals as well. Several classes of highly toxic molecules that are considered potential terrorist weapons, such as the neurotoxins: VX gas and Sarin, contain specific chemical structures that produce distinct spectral signals. The invention can be optimized to rapidly detect these types of molecules in a system that would permit the screening of solid objects and liquids in a high through-put format such as mail testing, luggage surveillance, or random analysis of packaged fluids. EXAMPLES<br>
The following examples are given to illustrate the scope of the present invention. Since these examples are given for illustrative purposes only, the present invention is not limited to the examples. Example 1<br>
For intrinsic fluorescence (IF), biological samples (urine, blood, plasma, CSF, etc.) are prepared by a number of different procedures for parallel analysis. These may include, but are not limited to, simple dilution, organic solvent extraction or precipitation, acid extraction or precipitation, and PEG precipitation. Following<br>
preparation, the IF of several normal and diseased samples are surveyed using several wavelengths of excitation light, preferably 210 nm to 1.2 urn until a difference in spectral signals is detected.<br>
Using ACN/TFA extracted plasma samples from normal and HCV infected subjects and illumination with light of 290 nm, a consistent IF spectral difference is noted (Fig la). Normalization of these spectra at a single wavelength provided a clearer view of the differences in spectral composition between normal and infected subjects (Fig 2a). The mean spectra for the two groups (Figs, lb, 2b) further clarified the specific spectral differences between the two groups which was magnified by the subtraction of the mean for all the samples and is referred to as the centered mean (Figs, lc, 2c)<br>
The full panel of spectra were analyzed by Principal Component Analysis (PC A) to identify spectral domains with the greatest level of variation. The initial analysis was limited to identification of the eight "factors" carrying the greatest amount of signal variation. The individual spectra were then reclassified according to their infection status and projected onto the first eight principal component factors (Fig 3). For both the normalized and unnormalized spectra, Factors 2 and 3 held a significant level of discriminatory information. For the normalized spectra (Fig. 3b), Factor 1 also held discriminatory information as indicated by the partial separation of the infected (o) and normal samples (*) along the diagonal axis of the Fl x Fl plot (upper left).<br>
When analyzed as 2 dimensional combinations the combination of Factors 2 and Factor 3 provided full discrimination between the two groups in the unnormalized data set (Fig 4a) and near full discrimination in the normalized data set (Fig 4b). The individual components were subjected to univariate stepwise discriminate analysis<br>
which showed that, for the unnormalized data set, the differences between the infected and normal groups were statistically significant in Factor 2 (pO.Ol) and Factor 3 (p
A chromatography strategy is used by the invention that utilizes the key Spectral parameters identified to isolate the molecules carrying the discriminatory signal. For this purpose, extracted samples from several normal and HCV-infected individuals were subjected to reverse phase high pressure liquid chromatography (RP-HPLC). Elution of discriminatory molecules was monitored by simultaneous UV absorption (290 nm) and fluorescence (ex 290nm/em 320nm or ex 290nm/em 440nm).<br>
Differences in fluorescence (ex 290nm/em 320nm) elution profiles for normal and infected samples were notable. Typical chromatographs are presented in Fig 5.<br>
Using excitation light of a wavelength (290 nm) that provided discriminatory signals in the bulk extract and a detection wavelength (320 nm) where differential spectral signals were notable, molecular peaks elutedreproducibly at around 5.3, 13.8 and 16.2 minutes. Chromatographic processing of a panel of infected and normal samples provided results permitting the determination of the quantitative relationship of the amounts of each peak (peak area) with disease and statistical analysis of any correlations established. Quantitative increases in the 5.3 and 13.4 minute peaks were associated HCV infection (Fig 6a). The increase in the 13.4 minute peak was statistically significant (p
of the 16.2 and 5.3 minute peaks was significantly (p
Based on the statistical analysis of the peak ratios, it is concluded that the molecules represented by these peaks carry the discriminatory information established previously by the spectral profiling strategy. The combination of the two significant ratios may provide enhanced discriminatory power suggesting that the levels of all three molecules may be altered during HCV infection.<br>
The material eluting from the RP-HPLC system noted above at 5.3, 13.4 and 16.2 minutes was collected and subjected to mass spectrometric (MS) analysis following rechromatography under similar conditions. Tandem MS revealed each chromatographic peak to contain several mass ions in the range of 250 to 2500 Daltons that represented a coherent set of fragment of 3 discrete molecular species, called SXI-18053, SXI-18134 and SXI-18162. Example 2<br>
Spectra-molecular Informatics<br>
Elucidation of Hepatitis C virus molecular markers by spectra-molecular surveillance Standard Spectral Survey<br>
Human plasma and serum samples from normal and HCV-infected patients were processed by several methods (e.g. simple dilution, ACN/TFA extraction), and scanned by standard spectroflorometric methods as well as exposed to excitation light of several selected wavelengths. The fluorescence spectra obtained from these studies revealed subtle differences between samples from infected and normal subjects that provided the foundation for more specific studies.<br>
Advanced Spectral Profile<br>
Based on the preliminary results noted above and specific biological knowledge relevant to the pathogenicity of the Hepatitis C virus, an alternative preparative method employing precipitation of plasma with polyethylene glycol (PEG) and additional wavelengths of excitation light were evaluated further in addition to the ACN/TFA extraction method. Such preparative methods are discussed in copending application No. 10/144,778, filed May 15, 2002, which is incorporated by reference. A.<br>
For samples extracted with ACN/TFA, excitation wavelengths of 260, 290, 320, 355, 380, 420, 580, 640 run were evaluated in preliminary studies. Comprehensive studies focused on the assessment of fluorescence spectra derived from excitation light of 290, 320, 355 and 380 nm due to the higher level of fluorescence intensity and spectral complexity associated with these wavelengths.<br>
Analysis of spectral data from sets of normal and HCV infected plasma donors revealed significant differences between these two groups in the fluorescence spectra obtained from 290 nm excitation, indicating the differential presence of one or more fluorescent molecules that could be detected with these specific conditions (i.e. 290 nm excitation, 300 — 500 nm emission). Identification of the "discriminatory spectral signal" is a critical element of the Spectra-molecular Informatics strategy, as it provides the information regarding instrument settings needed to isolate the "discriminatory molecule(s)". B.<br>
Both the supematants and resuspended pellets of samples that were precipitated with PEG were illuminated with light of multiple wavelengths (320, 355, 380 nm). The<br>
samples derived from the PEG pellets (a lipoprotein rich fraction depleted of albumin) showed significant differences between the fluorescence spectra of the normal and infected groups under 355 and 380 ran illumination. These additional discriminatory signals provide additional information concerning the biochemical nature of the discriminatory molecule and an alternative approach to its purification that would rely on these spectral features. Identification of HCVDiscriminatory Molecules<br>
Based on the results of the Advanced Spectral Profile, ACN/TFA extracted plasma samples were examined chromatographically in several solvent systems and using several chromatographic approaches, with fluorescence at 290 nm excitation as the detection criteria. Reverse phase HPLC, with a C-18 column and acetonitrile elution gradient separated multiple fluorescent peaks representing the molecular components of the mixture that offered the discriminatory spectral signal. Comparison of the chromatographic profiles between control and diseased samples revealed significant quantitative differences in specific peak areas between the two groups. Most notably, a significant increase in the area of a peaks eluting at approximately and 5.3 and 13.4 minutes was associated with HCV infection. A parallel decrease in the area of the peak eluting at 16.2 minutes provided a ratio (16.2 min/5.3 min) with substantial discriminatory power.<br>
The material eluting from the RP-HPLC system noted above at 5.3, 13.4 and 16.2 minutes was collected and subjected to mass spectrometric (MS) analysis following rechromatography under similar conditions. Tandem MS revealed each chromatographic peak to contain several mass ions in the range of 250 to 2500 Daltons<br>
that represented a coherent set of fragment of 3 discrete molecular species, called SXI-<br>
18053, SXI-18134 and SXI-18162.<br>
Development of prototype assays and validation ofHCV discriminatory markers<br>
While the identity of the discriminatory molecular species holds significant value, the relative importance of their contributions to the overall discriminatory signal offers additional information regarding their utility in disease diagnosis and treatment. To quantify the amount of each discriminatory molecule in HCV infected plasma samples, a biochemical assay is used based on the binding of the discriminatory molecule to a specific binding entity.<br>
In the case of SXI-18053, the assay employs a fluorescently tagged version of SXI-18053 (SXI-18053-FL23) that binds with similar affinity to untagged SXI-18053 to the selected protein. In. the absence of soluble SXI-18053, the protein binds the tagged molecule and can be immunoprecipitated or filtered to separate the complex from the free SXI-18053-FL23, In the presence of free SXI-18053, the tagged i molecule is displaced in proportion to their relative amounts, and a reduced amount of tagged molecule is associated with the binding agent after separation. Either dissociated or enzyme associated signal can be measured to quantify the amount of SXI-18053 in a test sample.<br>
The assay is a general competitive-binding assay that can alternatively employ an antibody, carbohydrate, nucleic acid or other molecule as the binding agent. The compound used to tag the discriminatory molecule for its specific analysis can include radioisotopes, fluorescent compounds, enzymes, avidin, biotin, and other detectable agents. For the assays described here, a common fluorescent tag (FL23) has been employed that is optimally excited and fluorescent in a spectra range distinct from the<br>
spectra of the subject molecules. These assays are extrinsic fluorescent systems that benefit from the use of a common instrument for their detection while employing target-specific reagents.<br>
These assays are utilized to evaluate the levels of SXT-1S053, SXI-18134 and SXI-18162 in plasma derived from normal and infected patients and provide quantitative confirmation of these molecules as valid independent markers of Hepatitis C infection. The relative levels of these molecules provide further information concerning biochemical pathways that are disrupted by HCV infection and would be important targets for therapeutic drug development.<br>
According, it should be readily appreciated that the methods of the present invention has many practical applications. Additionally, although the preferred embodiments have been illustrated and described, it will be obvious to those skilled in the art that various modifications can be made without departing from the spirit and scope of this invention. In example is that the invention is not limited to only biological samples, but can be utilized for any sample that provides for the practice of the invention. Such modifications are to be considered as included in the following claims.<br>
WE CLAIM:<br>
1. A method for identifying compounds associated with the presence of disease, comprising:<br>
obtaining a plurality of fluorescence spectra, each spectrum being<br>
obtained from a respective biological sample having a known disease<br>
state;<br>
identifying at least one spectral feature indicative of the presence of the<br>
disease;<br>
and<br>
identifying at least one compound based upon the spectral feature.<br>
2. The method as claimed in claim 1, wherein the identifying at least one compound step comprises subjecting at least a portion of a biological sample to a chromatographic separation.<br>
3. The method for identifying compounds as claimed in claim 1 wherein: The fluorescence spectra is a product of intrinsic fluorescence of the sample.<br>
4. The method for identifying compounds as claimed in claim 1, wherein: the identified compound is a molecule.<br>
5. The method for identifying compounds as claimed in claim 1, further comprising:<br>
preparing the sample to selectively amplify the identified spectral feature.<br>
6. The method for identifying compounds as claimed in claim 5, wherein: Sample extracting methods are used to analyze the sample.<br>
7. The method for identifying compounds as claimed in claim 1, wherein: The biological sample is selected from the group consisting of plasma, serum, urine and spinal fluid.<br>
B. The method for identifying compounds as claimed in claim 1, wherein: The fluorescent spectra is obtained from an excitation source having a wavelength of between 190 and 1200 nanometers.<br>
9. The method for identifying compounds as claimed in claim 8, wherein : the wavelength of the excitation source is between 260 and 640 nanometers.<br>
10.The method for identifying compounds as claimed in claim &amp;, wherein, the wavelength of the excitation source is about 290 nanometers.<br>
11. The method for identifying compounds as claimed in claim 1, wherein: Multivariate statistical methods are used to identify the spectral feature.<br>
12. The method for identifying compounds as claimed in claim 11, wherein: The multivariate statistical methods used are Principal Component Analysis and Independent Component Analysis.<br>
13. The method for identifying unknown markers associated with the presence of a predetermined characteristic comprising:<br>
Creating a first spectra) survey fo a sample with a predetermined characteristic and a sample without such characteristic based on predetermined preparative methods<br>
applied to the sample and predetermined excitation wavelengths used to irradiate the sample;<br>
creating a second spectral survey wherein further spectral resolution is acquired;<br>
subjecting the first and second surveys to analytical processing wherein a sample with the characteristic is differentiated from a sample without the characteristic based on its emission spectrum;<br>
isolating the wavelength which demonstrates the greatest differentiation; and<br>
determining the marker that demonstrates the greatest differentiation.<br>
14. A method for identifying markers associated with the presence of a predetermined characteristic comprising:<br>
accumulating spectral data comprising intrinsic fluorescence from a sample known to have a predetermined characteristic;<br>
identifying a spectral feature which is indicative of the predetermined characteristic;<br>
identifying a marker associated with the spectral feature.<br>
15. A method for identifying molecules associated with the presence of disease comprising:<br>
obtaining a biological sample having a known disease state;<br>
preparing the sample to amplify an intrinsic fluorescent signal from different molecules;<br>
analyzing the intrinsic fluorescence over a range of excitation and emission wavelengths;<br>
identifying a spectral feature that provides disease discrimination;<br>
identifying a molecule that produces the spectral feature.<br>
16. A method for the identification of disease specific molecules comprising: obtaining a sample having a known disease state;<br>
analyzing the intrinsic spectral features of the sample;<br>
identifying a spectral feature that provides specific disease discrimination;<br>
identifying a molecule that produces the spectral feature.<br>
17. A method for identifying disease related discriminatory spectral signals comprising:<br>
obtaining multidimensional spectral data from a sample having a known disease state;<br>
identifying the spectral data that indicates the presence of the disease; identifying compounds that express the identified spectral data.<br>
18.The method as claimed in claim 17, wherein:<br>
the multidimensional spectral data includes intrinsic fluorescence.<br>
19.The method as claimed in claim 17, wherein:<br>
the identified data indicates the presence of a predetermined disease.<br>
20. The method as claimed in claim 17, further comprising: Utilizing data reduction methods to amplify the spectral data.<br>
A method for identifying compounds associated with the presence of disease, comprising:<br>
obtaining a plurality of fluorescence spectra, each spectrum being obtained from<br>
a respective biological sample having a known disease state;<br>
identifying at least one spectral feature indicative of the presence of the disease;<br>
and<br>
identifying at least one compound based upon the spectral feature.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1sZXR0ZXIgcGF0ZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-letter patent.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MjYta29sbnAtMjAwNC1yZXBseSBmLmUuci5wZGY=" target="_blank" style="word-wrap:break-word;">00926-kolnp-2004-reply f.e.r.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216846-an-aerosol-composition-process-for-its-preparation-and-pressurized-can-comprising-it.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216848-a-keypade-system-for-providing-fast-and-efficient-text-entry-in-a-communication-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216847</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00926/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>21-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jul-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SEROPTIX , INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>82, CUMMI8NGS PARK, WOBURN, MA 01880. USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>THEODORE MAIONE</td>
											<td>9, AVON STREET USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/38463</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-12-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/334, 606</td>
									<td>2001-12-03</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216847-a-method-for-identifying-markers by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:06:05 GMT -->
</html>
