<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/216986-a-composition-comprising-a-polypeptide-having-the-sequence-of-seq-id-no-431-or-a-variant-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:15:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 216986:A COMPOSITION COMPRISING A POLYPEPTIDE HAVING THE SEQUENCE OF SEQ ID NO : 431 OR A VARIANT THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A COMPOSITION COMPRISING A POLYPEPTIDE HAVING THE SEQUENCE OF SEQ ID NO : 431 OR A VARIANT THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A composition comprising: (a) a polypeptide comprising the sequence of Seq ID No: 431 or a variant thereof having at least 90% sequence identity thereto, as determined by comparing the two optimally aligned sequences over a window of comparison of the whole length of the sequence of Seq ID No: 431; or (b) a polynucleotide encoding the polypeptide of (a) and physiologically acceptable carrier; for use in stimulating an immune response in a patient.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
"A COMPOSITION COMPRISING A POLYPEPTIDE HAVING THE SEQUENCE OF SEQ ID NO: 431 OR A VARIANT THEREOF"<br>
CORIXA CORPORATION, A corporation of the State of Delaware, of 1124 Columbia street, Suite 200, Seatle, Washington 98104, United States of America<br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br><br>
WO 02/08267<br><br>
PCT/US01/23121<br><br>
SEQ ID NO: 122 is the determined full-length DNA sequence for the C. trachomatis serovar E Cap1 gene CT529.<br>
SEQ ID NO: 123 is the predicted full-length amino acid sequence for the C. trachomatis serovar E Capl gene CT529.<br>
SEQ ID NO: 124 is the determined full-length DNA sequence for the C trachomatis serovar 1A Capl gene CT529.<br>
SEQ ID NO: 125 is the predicted full-length amino acid sequence for the C. trachomatis serovar 1A Capl gene CT529.<br>
SEQ ID NO: 126 is the determined full-length DNA sequence for the C trachomatis serovar G Capl gene CT529.<br>
SEQ ID NO: 127 is the predicted full-length amino acid sequence for the C. trachomatis serovar G Capl gene CT529.<br>
SEQ ID NO: 128 is the determined full-length DNA sequence for the C. trachomatis serovar F1 NII Cap1 gene CT529.<br>
SEQ ID NO: 129 is the predicted full-length amino acid sequence for the C. trachomatis serovar F1 NII Cap1 gene CT529.<br>
SEQ ID NO: 130 is the determined full-length DNA sequence for the C. trachomatis serovar L1 Cap1 gene CT529.<br>
SEQ ID NO: 131 is the predicted full-length amino acid sequence for the C. trachomatis serovar L1 Cap1 gene CT529.<br>
SEQ ID NO: 132 is the determined full-length DNA sequence for the C. trachomatis serovar L3 Cap1 gene CT529.<br>
SEQ ID NO: 133 is the predicted full-length amino acid sequence for the C. trachomatis serovar L3 Cap1 gene CT529.<br>
SEQ ID NO: 134 is the determined full-length DNA sequence for the C. trachomatis serovar Ba Cap1 gene CT529.<br>
SEQ ID NO: 135 is the predicted full-length amino acid sequence for the C. trachomatis serovar Ba Cap1 gene CT529.<br>
SEQ ID NO: 136 is the determined full-length DNA sequence for the C. trachomatis serovar MOPN Cap1 gene CT529.<br>
SEQ ID NO: 137 is the predicted full-length amino acid sequence for the C. trachomatis serovar MOPN Cap1 gene CT529.<br>
SEQ ID NO: 138 is the detennined amino acid sequence for the Cap1 CT529 ORP peptide #124-139 of C. trachomatis serovar L2.<br>
SEQ ID NO: 139 is the detennined amino acid sequence for the Capl CT529 ORF peptide #132-147 of C. trachomatis serovar L2.<br>
SEQ ID NO: 140 is the detennined amino acid sequence for the Cap1 CT529 ORP peptide #138-155 of C. trachomatis serovar L2.<br>
SEQ ID NO: 141 is the determined amino acid sequence for the Cap1 CT529 ORF peptide # 146-163 of C. trachomatis serovar L2.<br>
SEQ ID NO: 142 is the detennined amino acid sequence for the Cap1 CT529 ORF peptide #154-171 of C. trachomatis serovar L2.<br>
SEQ ID NO: 143 is the determined amino acid sequence for the Cap1 CT529 ORF peptide #162-178 of C. trachomatis serovar L2.<br>
SEQ ID NO: 144 is the determined amino acid sequence for the Cap1 CT529 ORF peptide #138-147 of C trachomatis serovar L2.<br>
SEQ ID NO: 145 is the determined amino acid sequence for the Cap1 CT529 ORF peptide #139-147 of C. trachomatis serovar L2.<br>
SEQ ID NO: 146 is the determined amino acid sequence for the Cap1 CT529 ORF peptide #140-147 of C. trachomatis serovar L2.<br>
SEQ ID NO: 147 is the determined amino acid sequence for the Cap1 CT529 ORF peptide #138-146 of C. trachomatis serovar L2.<br>
SEQ ID NO: 148 is the detennined amino acid sequence for the Capl CT529 ORF peptide #138-145 of C. trachomatis serovar L2.<br>
SEQ ID NO: 149 is the determined amino acid sequence for the Cap1 CT529 ORF peptide # F140-&gt;I of C. trachomatis serovar L2.<br>
SEQ ID NO: 150 is the determined amino acid sequence for the Cap1 CT529 ORF peptide # #S139&gt;Ga of C. trachomatis serovar L2.<br>
SEQ ID NO: 151 is the determined amino acid sequence for the Cap1 CT529 ORF peptide # #S139&gt;Gb of C. trachomatis serovar L2.<br>
SEQ ID NO: 152 is the determined amino acid sequence for the peptide<br>
#	2 C7.8-6 of the 216aa ORF of C. trachomatis serovar L2.<br>
SEQ ID NO: 153 is the determined amino acid sequence for the peptide<br>
#	2 C7.8-7 of the 216aa ORF of G trachomatis serovar L2.<br>
SEQ ID NO: 154 is the determined amino acid sequence for the peptide<br>
#	2 C7.8-8 of the 2l6aa ORF of C. trachomatis serovar L2.<br>
SEQ ID NO: 155 is the determined amino acid sequence for the peptide<br>
#	2 C7.8-9 of the 216aa ORF of C. trachomatis serovar L2.<br>
SEQ ID NO: 156 is the determined amino acid sequence for the peptide<br>
#	2 C7.8-10 of the 216aa ORF of C. trachomatis serovar L2.<br>
SEQ ID NO: 157 is the determined amino acid sequence for the 53 amino acid residue peptide of the 216aa ORF within clone 2C7.8 of C. trachomatis serovar L2.<br>
SEQ ID NO: 158 is the determined amino acid sequence for the 52 amino acid residue peptide of the CT529 ORF within clone 2C7.8 of C. trachomatis serovar L2.<br>
SEQ ID NO: 159 is the determined DNA sequence for the 5' (forward) primer for cloning full-length CT529 serovar L2.<br>
SEQ ID NO: 160 is the determined DNA sequence for the 5' (reverse) primer for cloning full-length CT529 serovar L2.<br>
SEQ ID NO: 161 is the determined DNA sequence for the 5' (forward) primer for cloning full-length CT529 for serovars other than L2 and MOPN.<br>
SEQ ID NO: 162 is the determined DNA sequence for the 5' (reverse) primer for cloning full-length CT529 serovars other than L2 and MOPN.<br>
SEQ ID NO: 163 is the determined DNA sequence for the 5' (forward) primer for cloning full-length CT529 serovar MOPN.<br>
SEQ ID NO: 164 is the determined DNA sequence for the 5' (reverse) primer for cloning full-length CT529 serovar MOPN:<br>
SEQ ID NO: 165 is the determined DNA sequence for the 5' (forward) primer for pBIB-KS.<br>
SEQ ID NO: 166 is the determined DNA sequence for the 5' (reverse) primer for pBIB-KS.<br>
SEQ ID NO: 167 is the determined amino acid sequence for the 9-mer epitope peptide Capl#l39-147 from serovar L2.<br>
SEQ ID NO: 168 is the determined amino acid sequence for the 9-mer epitope peptide Cap1#139-147 from serovar D.<br>
SEQ ID NO: 169 is the determined full-length DNA sequence for the C. trachomatis pmpI (CT874) gene.<br><br>
SEQ ID NO: 170 is the determined full-length DNA sequence for the C. trachomatis pmpG gene.<br>
SEQ ID NO: 171 is the determined full-length DNA sequence for the C. trachomatis pmpE gene.<br>
SEQ ID NO: 172 is the determined full-length DNA sequence for the C. trachomatis pmpD gene.<br>
SEQ ID NO: 173 is the determined full-length DNA sequence for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 174 is the determined full-length DNA sequence for the C. trachomatis pmpB gene.<br>
SEQ ID NO: 175 is the predicted full-length amino acid sequence for the C. trachomatis pmpl gene.<br>
SEQ ID NO: 176 is the predicted full-length amino acid sequence for the C. trachomatis pmpG gene.<br>
SEQ ID NO: 177 is the predicted full-length amino acid sequence for the C. trachomatis pmpE gene.<br>
SEQ ID NO: 178 is the predicted full-length amino acid sequence for the C. trachomatis pmpD gene.<br>
SEQ ID NO: 179 is the predicted full-length amino acid sequence for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 180 is the predicted full-length amino acid sequence for the C. trachomatis pmpB gene.<br>
SEQ ID NO: 181 is the determined DNA sequence minus the signal sequence for the C. trachomatis pmpI gene.<br>
SEQ ID NO:   182 is  a subsequently  determined full-length DNA sequence for the C. trachomatis pmpG gene.<br>
SEQ ID NO: 183 is the determined DNA sequence minus the signal sequence for the C. trachomatis pmpE gene.<br>
SEQ ID NO: 184 is a first determined DNA sequence representing the . carboxy terminus for the C. trachomatis pmpD gene.<br>
SEQ ID NO: 185 is a second determined DNA sequence representing the amino terminus minus the signal sequnce for the C. trachomatis pmpD gene.<br>
SEQ ID NO: 186 is a first determined DNA sequence representing the carboxy terminus for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 187 is a second determined DNA sequence representing the amino terminus minus the signal sequence for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 188 is the determined DNA sequence representing the C. pneumoniae serovar MOMPS pmp gene in a fusion molecule with Ral2.<br>
SEQ ID NO: 189 is the predicted amino acid sequence minus the signal sequence for the C. trachomatis pmpl gene.<br>
SEQ ID NO: 190 is subsequently predicted amino acid sequence for the C. trachomatis pmpG gene.<br>
SEQ ID NO: 191 is the predicted amino acid sequence minus the signal sequence for the C. trachomatis pmpE gene.<br>
SEQ ID NO: 192 is a first predicted amino acid sequence representing the carboxy terminus for the C trachomatis pmpD gene.<br>
SEQ ID NO: 193 is a second predicted amino acid sequence representing the Amino terminus minus the signal sequence for the C. trachomatis pmpD gene.<br>
SEQ ID NO: 194 is a first predicted amino acid sequence representing the Carboxy terminus for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 195 is a second predicted amino acid sequence representing the Amino terminus for the C. trachomatis pmpC gene.<br>
SEQ ID NO: 196 is the predicted amino acid sequence representing the C. pneumoniae serovar MOMPS pmp gene in a fusion molecule with Ra12.<br>
SEQ ID NO: 197 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpC gene in the SKB vaccine vector.<br>
SEQ ID NO: 198 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpC gene in the SKB vaccine vector.<br>
SEQ ID NO: 199 is the determined DNA sequence for the insertion sequence for cloning the C. trachomatis pmpC gene in the SKB vaccine vector.<br>
SEQ ID NO: 200 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpD gene in the SKB vaccine vector.<br>
SEQ ID NO: 201 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpD gene in the SKB vaccine vector.<br>
SEQ ID NO: 202 is the determined DNA sequence for the insertion sequence for cloning the C. trachomatis pmpD gene in the SKB vaccine vector.<br>
SEQ ID NO: 203 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpE gene in the SKB vaccine vector.<br>
SEQ ID NO: 204 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpE gene in the SKB vaccine vector.<br>
SEQ ID NO: 205 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpG gene in the SKB vaccine vector.<br>
SEQ ID NO: 206 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpG gene in the SKB vaccine vector.<br>
SEQ ID NO: 207 is the determined DNA sequence for the 5' oligo primer for cloning the amino terminus portion of the C. trachomatis pmpC gene in the pETl7b vector.<br>
SEQ ID NO: 208 is the determined DNA sequence for the 3' oligo primer for cloning the amino terminus portion of the C. trachomatis pmpC gene in the pETl7b vector.<br>
SEQ ID NO: 209 is the determined DNA sequence for the 5' oligo primer for cloning the carboxy terminus portion of the C. trachomatis pmpC gene in the pETl7b vector.<br>
SEQ ID NO: 210 is the determined DNA sequence for the 3' oligo primer for cloning the carboxy terminus portion of the C. trachomatis pmpC gene in the pETl7b vector.<br>
SEQ ID NO: 211 is the determined DNA sequence for the 5' oligo primer for cloning the amino terminus portion of the C. trachomatis pmpD gene in the pETl 7b vector.<br>
SEQ ID NO: 212 is the determined DNA sequence for the 3' oligo primer for cloning the amino terminus portion of the C. trachomatis pmpD gene in the pETl7b vector.<br>
SEQ ID NO: 213 is the determined DNA sequence for the 5' oligo primer for cloning the carboxy terminus portion of the C. trachomatis pmpD gene in the pETl 7b vector.<br>
SEQ ID NO: 214 is the determined DNA sequence for the 3' oligo primer for cloning the carboxy terminus portion of the C. trachomatis pmpD gene in the pETl7b vector.<br>
SEQ ID NO: 215 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpE gene in the pET17b vector.<br>
SEQ ID NO: 216 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpE gene in the pET17b vector.<br>
SEQ ID NO: 217 is the determined DNA sequence for the insertion sequence for cloning the C. trachomatis pmpE gene in the pET17b vector.<br>
SEQ ID NO: 218 is the amino acid sequence for the insertion sequence for cloning the C. trachomatis pmpE gene in the pETl7b vector.<br>
SEQ ID NO: 219 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpG gene in the pET17b vector.<br>
SEQ ID NO: 220 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpG gene in the pET17b vector.<br>
SEQ ID NO: 221 is the amino acid sequence for the insertion sequence for cloning the C. trachomatis pmpG gene in the pET17b vector.<br>
SEQ ID NO: 222 is the determined DNA sequence for the 5' oligo primer for cloning the C. trachomatis pmpl gene in the pET17b vector.<br>
SEQ ID NO: 223 is the determined DNA sequence for the 3' oligo primer for cloning the C. trachomatis pmpl gene in the pET17b vector.<br>
SEQ ID NO: 224 is the determined amino acid sequence for the C. pneumoniae Swib peptide 1-20.<br>
SEQ ID NO: 225 is the determined amino acid sequence for the C. pneumoniae Swib peptide 6-25.<br>
SEQ ID NO: 226 is the determined amino acid sequence for the C. pneumoniae Swib peptide 12-31.<br>
SEQ ID NO: 227 is the determined amino acid sequence for the C. pneumoniae Swib peptide 17-36.<br>
SEQ ID NO: 228 is the determined amino acid sequence for the C. pneumoniae Swib peptide 22-41.<br>
SEQ ID NO: 229 is the determined amino acid sequence for the C. pneumoniae Swib peptide 27-46.<br>
SEQ ID NO: 230 is the determined amino acid sequence for the C. pneumoniae Swib peptide 42-61.<br>
SEQ ID NO: 231 is the determined amino acid sequence for the C. pneumoniae Swib peptide 46-65.<br>
SEQ ID NO: 232 is the determined amino acid sequence for the C. pneumoniae Swib peptide 51-70.<br>
SEQ ID NO: 233 is the determined amino acid sequence for the C. pneumoniae Swib peptide 56-75.<br>
SEQ ID NO: 234 is the determined amino acid sequence for the C pneumoniae Swib peptide 61-80.<br>
SEQ ID NO: 235 is the determined amino acid sequence for the C. pneumoniae Swib peptide 66-87.<br>
SEQ ID NO: 236 is the determined amino acid sequence for the C. trachomatis OMCB peptide 103-122.<br>
SEQ ID NO: 237 is the determined amino acid sequence for the C. trachomatis OMCB peptide 108-127.<br>
SEQ ID NO: 238 is the determined amino acid sequence for the C. trachomatis OMCB peptide 113-132.<br>
SEQ ID NO: 239 is the determined amino acid sequence for the C. trachomatis OMCB peptide 118-137.<br>
SEQ ID NO: 240 is the determined amino acid sequence for the C. trachomatis OMCB peptide 123-143.<br>
SEQ ID NO: 241 is the determined amino acid sequence for the C. trachomatis OMCB peptide 128-147.<br>
SEQ ID NO: 242 is the determined amino acid sequence for the C. trachomatis OMCB peptide 133-152.<br>
SEQ ID NO:  243 is the determined amino acid sequence for the C. trachomatis OMCB peptide 137-156.<br>
SEQ ID NO: 244 is the determined amino acid sequence for the C. trachomatis OMCB peptide 142-161.<br>
SEQ ID NO: 245 is the determined amino acid sequence for the C. trachomatis OMCB peptide 147-166.<br>
SEQ ID NO: 246 is the determined amino acid sequence for the C. trachomatis OMCB peptide 152-171.<br>
SEQ ID NO: 247 is the determined amino acid sequence for the C. trachomatis OMCB peptide 157-176.<br>
SEQ ID NO: 248 is the determined amino acid sequence for the C. trachomatis OMCB peptide 162-181.<br>
SEQ ID NO: 249 is the determined amino acid sequence for the C. trachomatis OMCB peptide 167-186.<br>
SEQ ID NO: 250 is the determined amino acid sequence for the C. trachomatis OMCB peptide 171-190.<br>
SEQ ID NO: 251 is the determined amino acid sequence for the C. trachomatis OMCB peptide 171-186.<br>
SEQ ID NO: 252 is the determined amino acid sequence for the C trachomatis OMCB peptide 175-186.<br>
SEQ ID NO: 252 is the determined amino acid sequence for the C. trachomatis OMCB peptide 175-186.<br>
SEQ ID NO: 253 is the determined amino acid sequence for the C. pneumoniae OMCB peptide 185-198.<br>
SEQ ID NO: 254 is the determined amino acid sequence for the C. trachomatis TSA peptide 96-115.<br>
SEQ ID NO: 255 is the determined amino acid sequence for the C. trachomatis TSA peptide 101-120.<br>
SEQ ID NO: 256 is the determined amino acid sequence for the C. trachomatis TSA peptide 106-125.<br>
SEQ ID NO: 257 is the determined amino acid sequence for the C. trachomatis TSA peptide 111-130.<br>
SEQ ID NO: 258 is the determined amino acid sequence for the C. trachomatis TSA peptide 116-135.<br>
SEQ ID NO: 259 is the determined amino acid sequence for the C. trachomatis TSA peptide 121-140.<br>
SEQ ID NO: 260 is the determined amino acid sequence for the C. trachomatis TSA peptide 126-145.<br>
SEQ ID NO: 261 is the determined amino acid sequence for the C. trachomatis TSA peptide 131-150.<br>
SEQ ID NO: 262 is the determined amino acid sequence for the C. ti-achomatis TSA peptide 136-155.<br>
SEQ ID NO: 263 is the determined full-length DNA sequence for the C. trachomatis CT529/Cap 1 gene serovar I.<br>
SEQ ID NO: 264 is the predicted full-length amino sequence for the C. trachomatis CT529/Cap 1 gene serovar I.<br>
SEQ ID NO: 265 is the determined full-length DNA sequence for the C. trachomatis CT529/Cap 1 gene serovar K.<br>
SEQ ID NO: 266 is the predicted full-length amino sequence for the C. trachomatis CT529/Cap 1 gene serovar K.<br>
SEQ ID NO: 267 is the determined DNA sequence for the C. trachomatis clone 17-G4-36 sharing homology to part of the ORF of DNA-dirrected RNA polymerase beta subunit- CT315 in serD.<br>
SEQ ID NO: 268 is the determined DNA sequence for the partial sequence of the C. trachomatis CT016 gene in clone 2E10.<br>
SEQ ID NO: 269 is the determined DNA sequence for the partial sequence of the C. ti-achomatis tRNA syntase gene in clone 2E10.<br>
SEQ ID NO: 270 is the determined DNA sequence for the partial sequence for the C. trachomatis clpX gene in clone 2E10.<br>
SEQ ID NO: 271 is a first determined DNA sequence for the C. ti-achomatis clone CtL2gam-30 representing the 5'end.<br>
SEQ ID NO: 272 is a second determined DNA sequence for the C. ti-achomatis clone CtL2gam-30 representing the 3'end.<br>
SEQ ID NO: 273 is the determined DNA sequence for the C. trachomatis clone CtL2gam-28. <br>
SEQ  ID NO:  274 is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-27.<br>
SEQ  ID NO:  275  is the  determined  DNA  sequence for the  C. trachomatis clone CtL2gam-26.<br>
SEQ  ID NO:  276 is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-24.<br>
SEQ  ID NO:  277 is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-23.<br>
SEQ  ID NO:  278  is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-21.<br>
SEQ  ID NO:  279 is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-18.<br>
SEQ  ID  NO:  280  is the  determined  DNA  sequence for the  C<br>
trachomatis clone CtL2gam-17.<br>
		SEQ ID NO: 281 is a first determined DNA sequence for the C.<br>
trachomatis clone CtL2gam-15 representing the 5' end.<br>
SEQ ID NO: 282 is a second determined DNA sequence for the C. trachomatis clone CtL2gam-15 representing the 3' end.<br>
SEQ  ID NO:  283  is  the  determined  DNA  sequence  for the  C. trachomatis clone CtL2gam-13.<br>
SEQ  ID NO:  284 is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-10.<br>
SEQ  ID NO:  285  is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-8.<br>
SEQ ID NO: 286 is a first determined DNA sequence for the C trachomatis clone CtL2gam-6 representing the 5' end.<br>
SEQ ID NO: 287 is a second determined DNA sequence for the C. trachomatis clone CtL2gam-6 representing the 3' end.<br>
SEQ  ID NO:  288  is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-5.<br>
SEQ  ID NO:  289 is the  determined  DNA  sequence for the  C. trachomatis clone CtL2gam-2.<br>
SEQ  ID NO:  290  is the determined  DNA  sequence for the  C. trachomatis clone CtL2gam-l.<br>
SEQ ID NO: 291 is the determined full-length DNA sequence for the C. pneumoniae homologue of the CT529 gene.<br>
SEQ ID NO: 292 is the predicted full-length amino acid sequence for the C:pneumoniae homologue of the CT529 gene.<br>
SEQ ID NO: 293 is the determined DNA sequence for the insertion sequence for cloning the C. trachomatis pmpG gene in the SKB vaccine vector.<br>
SEQ ID NO: 294 is the amino acid sequence of an open reading frame of clone CT603.<br>
SEQ ID NO: 295 is the amino acid sequence of a first open reading frame of clone CT875.<br>
SEQ ID NO: 296 is the amino acid sequence of a second open reading<br>
frame of clone CT875.<br>
	SEQ ID NO; 297 is the amino acid sequence of a first open reading<br>
frame of clone CT858."<br>
SEQ ID NO; 298 is the amino acid sequence of a second open reading frame of clone CT858.<br>
SEQ ID NO: 299 is the amino acid sequence of an open reading frame of clone CT622.<br>
SEQ ID NO: 300 is the amino acid sequence of an open reading frame of clone CT610.<br>
SEQ ID NO: 301 is the amino acid sequence of an open reading frame of clone CT396.<br>
SEQ ID NO: 302 is the amino acid sequence of an open reading frame of clone CT318.<br>
SEQ ID NO: 304 is the amino acid sequence for C. trachomatis, serovar L2 rCt529cl-125 having a modified N-terminal sequence (6-His tag).<br>
SEQ ID NO: 305 is the amino acid sequence for C. trachomatis, serovar L2 rCt529cl-125.<br>
SEQ ID NO: 306 is the sense primer used in the synthesis of the PmpA(N-term) fusion protein.<br>
SEQ ID NO: 307 is the antisense primer used in the synthesis of the PmpA(N-term) fusion protein.<br>
SEQ ID NO: 308 is the DNA sequence encoding the PmpA(N-term) fusion protein.<br>
SEQ ID NO: 309 is the amino acid sequence of the PmpA(N-term) fusion protein.<br>
SEQ ID NO: 310 is the sense primer used in the synthesis of the PmpA(C-terrn) fusion protein.<br>
SEQ ID NO: 311 is the antisense primer used in the synthesis of the PmpA(C-term) fusion protein.<br>
SEQ ID NO: 312 is the DNA sequence encoding the PmpA(C-term) fusion protein.<br>
SEQ ID NO: 313 is the amino acid sequence of the PmpA(C-term) fusion protein.<br>
SEQ ID NO: 314 is the sense primer used in the synthesis of the PmpF(N-term) fusion protein.<br>
SEQ ID NO: 315 is the antisense primer used in the synthesis of the PmpF(N-term) fusion protein.<br>
SEQ ID NO: 316 is the DNA sequence encoding the PmpF(N-term) fusion protein.<br>
SEQ ID NO: 317 is the amino acid sequence of the PmpF(N-term) fusion protein.<br>
SEQ ID NO: 318 is the sense primer used in the synthesis of the PmpF(C-term) fusion protein.<br>
 SEQ ID NO: 319 is the antisense primer used in the synthesis of the PmpF(C-term) fusion protein.<br>
SEQ ID NO: 320 is the DNA sequence encoding the PmpF(C-term) fusion protein.<br>
SEQ ID NO: 321 is the amino acid sequence of the PmpF(C-term) fusion protein.<br>
SEQ ID NO: 322 is the sense primer used in the synthesis of the PmpH (CT412) (N-term) fusion protein.<br>
SEQ ID NO: 323 is the antisense primer used in the synthesis of the PmpH(N-term) fusion protein.<br>
SEQ ID NO: 324 is the DNA sequence encoding the PmpH(N-term) fusion protein.<br>
SEQ ID NO: 325 is the amino acid sequence of the PmpH(N-term) fusion protein.<br>
SEQ ID NO: 326 is the sense primer used in the synthesis of the PmpH(C-term) fusion protein.<br>
SEQ ID NO: 327 is the antisense primer used in the synthesis of the PmpH(C-term) fusion protein.<br>
SEQ ID NO: 328 is the DNA sequence encoding the PmpH(C-term) fusion protein.<br>
SEQ ID NO: 329 is the amino acid sequence of the PmpH(C-term) fusion protein.<br><br>
SEQ ID NO: 330 is the sense primer used in the synthesis of the PmpB(l) fusion protein.<br>
SEQ ID NO: 331 is the antisense primer used in the synthesis of the PmpB(l) fusion protein.<br>
SEQ ID NO: 332 is the DNA sequence encoding the PmpB(l) fusion protein.<br>
SEQ ID NO: 333 is the amino acid sequence of the PmpB(1) fusion protein.<br>
SEQ ID NO: 334 is the sense primer used in the synthesis of the PmpB(2) fusion protein.<br>
SEQ ID NO: 335 is the antisense primer used in the synthesis of the PmpB(2) fusion protein.<br>
SEQ ID NO: 336 is the DNA sequence encoding the PmpB(2) fusion protein.<br>
SEQ ID NO: 337 is the amino acid sequence of the PmpB(2) fusion protein.<br>
SEQ ID NO: 338 is the sense primer used in the synthesis of the PmpB(3) fusion protein.<br>
SEQ ID NO: 339 is the antisense primer used in the synthesis of the PmpB(3) fusion protein.<br>
SEQ ID NO: 340 is the DNA sequence encoding the PmpB(3) fusion protein.<br>
SEQ ID NO: 341 is the amino acid sequence of the PmpB(3) fusion protein.<br>
SEQ ID NO: 342 is the sense primer used in the synthesis of the PmpB(4) fusion protein.<br>
SEQ ID NO: 343 is the antisense primer used in the synthesis of the PmpB(4) fusion protein.<br>
SEQ ID NO: 344 is the DNA sequence encoding the PmpB(4) fusion protein.<br>
SEQ ID NO: 345 is the amino acid sequence of the PmpB(4) fusion protein.<br><br>
SEQ ID NO: 346 is the sense primer used in the synthesis of the PmpC(1) fusion protein.<br>
SEQ ID NO: 347 is the antisense primer used in the synthesis of the PmpC(1) fusion protein.<br>
SEQ ID NO: 348 is the DNA sequence encoding the PmpC(1) fusion protein.<br>
SEQ ID NO: 349 is the amino acid sequence of the PmpC(1) fusion protein.<br>
SEQ ID NO: 350 is the sense primer used in the synthesis of the PmpC(2) fusion protein.<br>
SEQ ID NO: 351 is the antisense primer used in the synthesis of the PmpC(2) fusion protein.<br>
SEQ ID NO: 352 is the DNA sequence encoding the PmpC(2) fusion protein.<br>
SEQ ID NO: 353 is the amino acid sequence of the PmpC(2) fusion protein.<br>
SEQ ID NO: 354 is the sense primer used in the synthesis of the PmpC(3) fusion protein.<br>
SEQ ID NO: 355 is the antisense primer used in the synthesis of the PmpC(3) fusion protein.<br>
SEQ ID NO: 356 is the DNA sequence encoding the PmpC(3) fusion protein.<br>
SEQ ID NO: 357 is the amino acid sequence of the PmpC(3) fusion protein.<br>
SEQ ID NO: 358 is the DNA sequence of the oppA1 protein, devoid of the first trans-membrane domain.<br>
SEQ ID NO: 359 is the full length DNA sequence of CT139.<br>
SEQ ID NO: 360 is the full length DNA sequence of ORF-3.<br>
SEQ ID NO: 361 is the full length DNA sequence of CT611.<br>
SEQ ID NO: 362 is the amino acid sequence of oppAl starting from amino acid 22.<br>
SEQ ID NO: 363 is the amino acid sequence of CT139.<br>
SEQ ID NO: 364 is the amino acid sequence of ORF-3.<br>
SEQ ID NO: 365 is the amino acid sequence of CT611.<br>
SEQ ID NO: 366 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0275, of the Chlamydia trachomatis gene CT190.<br>
SEQ ID NO: 367 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0407, of the Chlamydia trachomatis gene CT103.<br>
SEQ ID NO: 368 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0720, of the Chlamydia trachomatis gene CT659.<br>
SEQ ID NO: 369 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0716, of the Chlamydia trachomatis gene CT660.<br>
SEQ ID NO: 370 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0519, of the Chlamydia trachomatis gene CT430.<br>
SEQ ID NO: 371 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0520, of the Chlamydia trachomatis gene CT431.<br>
iSEQ ID NO: 372 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0078, of the Chlamydia trachomatis gene CT318.<br>
SEQ ID NO: 373 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0628, of the Chlamydia trachomatis gene CT509. <br>
SEQ ID NO: 374 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0540, of the Chlamydia trachomatis gene CT414.<br>
SEQ ID NO: 375 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, pmp20, of the Chlamydia trachomatis gene CT413.<br>
SEQ ID NO: 376 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0081, of the Chlamydia trachomatis gene CT315.<br>
SEQ ID NO: 377 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0761, of the Chlamydia trachomatis gene CT610.<br>
SEQ ID NO: 378 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0557, of the Chlamydia trachomatis gene CT443.<br>
SEQ ID NO: 379 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0833, of the Chlamydia trachomatis gene CT557.<br>
SEQ ID NO: 380 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0134, of the Chlamydia trachomatis gene CT604.<br>
SEQ ID NO: 381 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0388, of the Chlamydia trachomatis gene CT042.<br>
SEQ ID NO: 382 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn1028, of the Chlamydia trachomatis gene CT376.<br>
SEQ ID NO: 383 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0875, of the Chlamydia trachomatis gene CT734.<br>
SEQ ID NO: 384 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0908, of the Chlamydia trachomatis gene CT764.<br>
SEQ ID NO: 385 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0728, of the Chlamydia trachomatis gene CT622.<br>
SEQ ID NO: 386 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0275, of the Chlamydia trachomatis gene CT190.<br>
SEQ ID NO: 387 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0407, of the Chlamydia trachomatis gene CT103.<br>
SEQ ID NO: 388 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0720, of the Chlamydia trachomatis gene CT659.<br>
SEQ ID NO: 389 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0716, of the Chlamydia trachomatis gene CT660.<br>
SEQ ID NO: 390 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0519, of the Chlamydia trachomatis gene CT430.<br>
SEQ ID NO: 391 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0520, of the Chlamydia trachomatis gene CT431.<br>
SEQ ID NO: 392 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0078, of the Chlamydia trachomatis gene CT318.<br>
SEQ ID NO: 393 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0628, of the Chlamydia trachomatis gene CT509.<br>
SEQ ID NO: 394 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0540, of the Chlamydia trachomatis gene CT414.<br>
SEQ ID NO: 395 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, pmp20, of the Chlamydia trachomatis gene CT413.<br><br>
SEQ ID NO: 396 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0081, of the Chlamydia trachomatis gene CT315.<br>
SEQ ID NO: 397 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn076l, of the Chlamydia trachomatis gene CT610.<br>
SEQ ID NO: 398 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0557, of the Chlamydia trachomatis gene CT443.<br>
SEQ ID NO: 399 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0833, of the Chlamydia trachomatis gene CT557.<br>
SEQ ID NO: 400 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0134, of the Chlamydia trachomatis gene CT604.<br>
SEQ ID NO: 401 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0388, of the Chlamydia trachomatis gene CT042.<br>
SEQ ID NO: 402 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn1028, of the Chlamydia trachomatis gene CT376.<br>
SEQ ID NO: 403 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0875, of the Chlamydia trachomatis gene CT734.<br>
SEQ ID NO: 404 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0908, of the Chlamydia trachomatis gene CT764.<br>
SEQ ID NO: 405 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0728, of the Chlamydia trachomatis gene CT622.<br>
SEQ ID NO: 406 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT287.<br>
SEQ ID NO: 407 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT858.<br>
SEQ ID NO: 408 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT764.<br>
SEQ ID NO: 409 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT734.<br>
SEQ ID NO: 410 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT660.<br>
SEQ ID NO: 411 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT659.<br><br>
SEQ ID NO: 412 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT622.<br>
SEQ ID NO: 413 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT610.<br>
SEQ ID NO: 414 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT604.<br>
SEQ ID NO: 415 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT557.<br>
SEQ ID NO: 416 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT509.<br>
SEQ ID NO: 417 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT443.<br>
SEQ ID NO: 418 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT431.<br>
SEQ ID NO: 419 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT430.<br>
SEQ ID NO: 420 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT414.<br>
SEQ ID NO: 421 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT413.<br>
SEQ ID NO: 422 sets forth the full-length serovar D DNA seguence of the Chlamydia trachomatis gene CT396.<br>
SEQ ID NO: 423 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT376.<br>
SEQ ID NO: 424 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT318.<br>
SEQ ID NO: 425 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT315.<br>
SEQ ID NO: 426 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT104.<br>
SEQ ID NO: 427 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT103.<br>
SEQ ID NO: 428 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT102.<br>
SEQ ID NO: 429 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT098.<br>
SEQ ID NO: 430 sets forth the full-length serovar D DNA sequence of the Chlamydia trachomatis gene CT042.<br>
SEQ ID NO: 431  sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT858.<br>
SEQ ID NO: 432 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT764.<br>
SEQ ID NO: 433 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT734.<br>
SEQ ID NO: 434 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT660.<br>
SEQ ID NO: 435 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT659.<br>
SEQ ED NO: 436 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT622.<br>
SEQ ID NO: 437 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT610.<br>
SEQ ID NO: 438 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT604.<br>
SEQ ID NO: 439 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT557.<br>
SEQ ID NO: 440 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT509.<br>
SEQ ID NO: 441  sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT443.<br>
SEQ ID NO: 442 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT431.<br>
SEQ ID NO: 443 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT430.<br>
SEQ ID NO: 444 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT414.<br>
SEQ ID NO: 445 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT413.<br>
SEQ ID NO: 446 sets forth the full-length serovar D amino acid sequence of the Chlamydia trachomatis gene CT396.<br>
SEQ ID NO: 447 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT376.<br>
SEQ ID NO: 448 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT318.<br>
SEQ ID NO: 449 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT315.<br>
SEQ ID NO: 450 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT104.<br>
SEQ ID NO: 451  sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT103.<br>
SEQ ID NO: 452 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT102.<br>
SEQ ID NO: 453 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT098.<br>
SEQ ID NO: 454 sets forth the full length serovar D amino acid sequence of the Chlamydia trachomatis gene CT042.<br>
SEQ ID NO: 455 corresponds to the DNA sequence of CPn0894, which is the CP homologue of CT751 (amn), which was identified in clones CTL2-1, and CTL2-5.<br>
SEQ ID NO: 456 corresponds to the DNA sequence of CPn0074, which is the CP homologue of CT322 (tuf), which was identified in clone CTL2-2.<br>
SEQ ID NO: 457 corresponds to the DNA sequence of CPn0122, which is the CP homologue of CT032 (metG), which was identified in clones CTL2gam2, CTL2-3(5') and CTL2-4.<br>
SEQ ID NO: 458 corresponds to the DNA sequence of CPn0121, which is the C? homologue of CT031, which was identified in clone CTL2-3(5')(3').<br>
SEQ ID NO: 459 corresponds to the DNA sequence of CPn0120, which is the CP homologue of CT030 (gmK), which was identified in clones CTL2-3(3')and CTL2-21.<br>
SEQ ID NO: 460 corresponds to the DNA sequence of CPn0359, which is the CP homologue of CT064 (lepA), which was identified in clone CTL2gam5.<br>
SEQ ID NO: 461 corresponds to the DNA sequence of CPn0414, which is the CP homologue of CT265 (accA), which was identified in clone CTL2-6.<br>
SEQ ID NO: 462 corresponds to the DNA sequence of CPn0413, which is the CP homologue of CT264 (msbA), which was identified in clone CTL2-6.<br>
SEQ ID NO: 463 corresponds to the DNA sequence of CPn0394, which is the CP homologue of CT256 which was identified in clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 464 corresponds to the DNA sequence of CPn0395, which is the CP homologue of CT257 which was identified in clones CTL2gam6(5')<br>
and CTL2-11(5').<br>
SEQ ID NO: 465 corresponds to the DNA sequence of CPn0487, which is the CP homologue of CT384 which was identified in clones CTL2gam6(3') and CTL2-11(3').<br>
SEQ ID NO: 466 corresponds to the DNA sequence of CPn0592, which is the CP homologue of CT473, which was identified in clone CTL2-8b.<br>
SEQ ID NO: 467 corresponds to the DNA sequence of CPn0593, which is the CP homologue of CT474, which was identified in clone CTL2-8b.<br>
SEQ ID NO: 468 corresponds to the DNA sequence of CPn0197, which is the CP homologue of CT139 (oppA1), which was identified in clone CTL2-8b.<br>
SEQ ID NO: 469 corresponds to the DNA sequence of CPn0363, which is the CP homologue of CT060 (flhA), which was identified in clone CTL2-8b.<br>
SEQ ID NO: 470 corresponds to the DNA sequence of CPn0301, which is the CP homologue of CT242, which was identified in clone CTL2gam8.<br>
SEQ ID NO: 471 corresponds to the DNA sequence of CPn0302, which is the CP homologue of CT243 (IpxD), which was identified in clone CTL2gam8.<br>
SEQ ID NO: 472 corresponds to the DNA sequence of CPn0324, which is the CP homologue of CT089 (lcrE), which was identified in clones CTL2-9, CTL2gaml, CTL2gam17 and CTL2-19(5').<br>
SEQ ID NO: 473 corresponds to the DNA sequence of CPn0761, which is the CP homologue of CT610, which was identified in clone CTL2-10(5')(3').<br>
SEQ ID NO: 474 corresponds to the DNA sequence of CPn0760, which is the CP homologue of CT611, which was identified in clone CTL2-10(5').<br>
SEQ ID NO: 475 corresponds to the DNA sequence of CPn0329, which is the CP homologue of CT154, which was identified in clones CTL2gam10 and CTL2gam21.<br>
SEQ ID NO: 476 corresponds to the DNA sequence of CPn0990, which is the CP homologue of CT833 (infC), which was identified in clone CTL2-12.<br>
SEQ ID NO: 477 corresponds to the DNA sequence of CPn0984, which is the CP homologue of CT827 (nrdA), which was identified in clones CTL2-<br>
16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 478 corresponds to the DNA sequence of CPn0985 which is the CP homologue of CT828 (nrdB) which was identified in clones CTL2-16(3')CTL2gaml5(3').<br>
SEQ ID NO: 479 corresponds to the DNA sequence of CPn0349, which is the CP homologue of CT067 (ytgA), which was identified in clone CTL2gaml8.<br>
SEQ ID NO: 480 corresponds to the DNA sequence of CPn0325, which is the CP homologue of CT088 (sycE), which was identified in clone CTL2-19(5').<br>
SEQ ID NO: 481 corresponds to the DNA sequence of CPn0326, which is the CP homologue of CT087 (malQ), which was identified in clone CTL2-<br>
19(5').<br>
SEQ ID NO: 482 corresponds to the DNA sequence of CPn0793, which is the CP homologue of CT588 (rbsu), which was identified in clone CTL2gam23.<br>
SEQ ID NO: 483 corresponds to the DNA sequence of CPn0199, which is the CP homologue of CT199 (oppB1), which was identified in clone CTL2gam24.<br>
SEQ ID NO: 484 corresponds to the DNA sequence of CPn0666, which is the CP homologue of CT545 (dnaE), which was identified in clone CTL2-24.<br>
SEQ ID NO: 485 corresponds to the DNA sequence of CPn0065, which is the CP homologue of CT288, which was identified in clone CTL2gam27.<br>
SEQ ID NO: 486 corresponds to the DNA sequence of CPn0444, which is the CP homologue of CT413 (pmpB), which was identified in clone CTL2gam30(5')(3').<br>
SEQ ID NO: 487 corresponds to the DNA sequence of CPn-ORF5, which is the CP homologue of CT-ORF3, which was identified in clones CTL2gaml5(5')5 CTL2-16(5'), CTL2-18(5'), and CTL2-23.<br>
SEQ ID NO: 488 corresponds to the DNA sequence of CPn-ORF6, which is the CP homologue of CT-ORF4, which was identified in clone CTL2-18(3').<br>
SEQ ID NO: 489 corresponds to the DNA sequence of CP-ORF7, which is the CP homologue of CT-ORF5, which was identified in clone CTL2-18(3').<br>
SEQ ID NO: 490 corresponds to the amino acid sequence of CPn0894, which is the CP homologue of CT751 (amn), which was identified in clones CTL2-1 and CTL2-5.<br>
SEQ ID NO: 491 corresponds to the amino acid sequence of CPn0074, which is the CP homologue of CT332 (tuf), which was identified in clone CTL2-2.<br>
SEQ ID NO: 492 corresponds to the amino acid sequence of CPn0122, which is the CP homologue of CT032 (metG), which was identified in clones CTL2gam2, CTL2-3(5') and CTL2-4.<br>
SEQ ID NO: 493 corresponds to the amino acid sequence of CPn0121, which is the CP homologue of CT031, which was identified in clone CTL2-3(5')(3').<br>
SEQ ID NO: 494 corresponds to the amino acid sequence of CPn0120 which is the CP homologue of CT030 (gmK) which was identified in clones CTL2-3(3')and CTL2-21.<br>
SEQ ID NO: 495 corresponds to the amino acid sequence of CPn0359, which is the CP homologue of CT064 (lepA), which was identified in clone CTL2gam5.<br>
SEQ ID NO: 496 corresponds to the amino acid sequence of CPn0414, which is the CP homologue of CT265 (accA), which was identified in clone CTL2-6.<br>
SEQ ID NO: 497 corresponds to the amino acid sequence of CPn0413, which is the CP homologue of CT264 (msbA), which was identified in clone CTL2-6.<br>
SEQ ID NO: 498 corresponds to the amino acid sequence of CPn0394, which is the CP homologue of CT256, which was identified in clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 499 corresponds to the amino acid sequence of CPn0395, which is the CP homologue of CT257, which was identified in clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 500 corresponds to the amino acid sequence of CPn0487, which is the CP homologue of CT384, which was identified in clones CTL2gam6(3') and CTL2-11(3').<br>
SEQ ID NO: 501 corresponds to the amino acid sequence of CPn0592, which is the CP homologue of CT473, which was identified in clone CTL2-8b.<br>
SEQ ID NO: 502 corresponds to the amino acid sequence of CPn0593, which is the CP homologue of CT474, which was identified in clone CTL2-8b.<br>
SEQ ID NO: 503 corresponds to the amino acid sequence of CPn0197, which is the CP homologue of CT139 (oppA1), which was identified in clone CTL2-8b.<br>
SEQ ID NO: 504 corresponds to the amino acid sequence of CPn0363, which is the CP homologue of CT060 (flhA), which was identified in clone CTL2-8b.<br>
SEQ ID NO: 505 corresponds to the amino acid sequence of CPn0301, which is the CP homologue of CT242, which was identified in clone CTL2gam8.<br>
SEQ ID NO: 506 corresponds to the amino acid sequence of CPn0302, which is the CP homologue of CT243 (lpxD), which was identified in clone CTL2gam8.<br>
SEQ ID NO: 507 corresponds to the amino acid sequence of CPn0324, which is the CP homologue of CT089 (lcrE), which was identified in clones CTL2-9, CTL2gam1, CTL2gaml7 and CTL2-19(5').<br>
SEQ ID NO: 508 corresponds to the amino acid sequence of CPn0761, which is the CP homologue of CT610, which was identified in clone CTL2-10(5')(3').<br>
SEQ ID NO: 509 corresponds to the amino acid sequence of CPn0760, which is the CP homologue of CT611, which was identified in clone CTL2-<br>
10(5').<br>
SEQ ID NO: 510 corresponds to the amino acid sequence of CPn0329, which is the CP homologue of CT154, which was identified in clones CTL2gam10 and CTL2gam21.<br>
SEQ ID NO: 511 corresponds to the amino acid sequence of CPn0990, which is the CP homologue of CT833 (infC), which was identified in clone CTL2-12.<br>
SEQ ID NO: 512 corresponds to the amino acid sequence of CPn-ORF5, which is the CP homologue of CT ORF3, which was identified in clones CTL2gaml5(5'), CTL2-16(5'), CTL2-18(5'), and CTL2-23.<br>
SEQ ID NO: 513 corresponds to the amino acid sequence of CPn0984, which is the CP homologue of CT827 (nrdA) which was identified in clones CTL2-16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 514 corresponds to the amino acid sequence of CPn0985, which is the CP homologue of CT828 (nrdB) which was identified in clones CTL2-16(3') CIL2gaml5(3').<br>
SEQ ID NO: 515 corresponds to the amino acid sequence of CPn03495 which is the CP homologue of CT067 (ytgA), which was identified in clone CTL2gaml8.<br>
SEQ ID NO: 516 corresponds to the DNA sequence of CPn-ORF6, which is the CP homologue of CT-ORF4, which was identified in clone CTL2-18(3').<br>
SEQ ID NO: 517 corresponds to the DNA sequence of CP-ORF7, which is the CP homologue of CT-ORF5, which was identified in clone CTL2-18(3').<br>
SEQ ID NO: 518 corresponds to the amino acid sequence of CPn0326, which is the CP homologue of CT087 (malQ), which was identified in clone CTL2-19(5').<br>
SEQ ID NO: 519 corresponds to the amino acid sequence of CPn0325, which is the CP homologue of CT088 (sycE), which was identified in clone CTL2-19(5').<br>
SEQ ID NO: 520 corresponds to the amino acid sequence of CPn0793, which is the CP homologue of CT588 (rbsu), which was identified in clone CTL2gam23.<br>
SEQ ID NO: 521 corresponds to the amino acid sequence of CPn0199, which is the CP homologue of CT199 (oppB1), which was identified in clone CTL2gam24.<br>
SEQ ID NO: 522 corresponds to the amino acid sequence of CPn0666, which is the CP homologue of CT545 (dnaE), which was identified in clone CTL2-24.<br>
SEQ ID NO: 523 corresponds to the DNA sequence of CPn0065, which is the CP homologue of CT288, which was identified in clone CTL2gam27.<br>
SEQ ID NO: 524 corresponds to the DNA sequence of CPn0444, which is the CP homologue of CT413 (pmpB), which was identified in clone CTL2gam30(5')(3')-<br>
SEQ ID NO: 525 sets forth the full-length C trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT751 (amn) identified from the clones CTL2-1 and CTL2-5.<br>
SEQ ID NO: 526 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT322 (tuff) identified from the clone CTL2-2.<br>
SEQ ID NO: 527 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT032 (metG) identified from the clones CTL2gam2, CTL2-3(5') and CTL2-4.<br>
SEQ ID NO: 528 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C trachomatis LGV II sequence for CT031 identified from the clone CTL2-3(5')(3').<br>
SEQ ID NO: 529 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT030 (gmK) identified from the clones CTL2-3(3') and CTL2-21.<br>
SEQ ID NO: 530 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT064 (lepA) identified from the clone CTL2gam5.<br>
SEQ ID NO: 531 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT265 (accA) identified from the clone CTL2-6.<br>
SEQ ID NO: 532 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT624 (msbA) identified from the clones CTL2-6.<br>
SEQ ID NO: 533 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT256 identified from the clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 534 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT257 identified from the clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 535 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT384 identified from the clones CTL2gam6(3') and CTL2-11(3').<br>
SEQ ID NO: 536 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT473 identified from the clone CTL2-8b.<br>
SEQ ID NO: 537 sets forth the full-length C, trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT474 identified from the clones CTL2-8b.<br>
SEQ ID NO: 538 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT139 (oppAl) identified from the clones CTL2-8b.<br>
SEQ ID NO: 539 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT060 (flhA) identified from the clone CTL2-8b.<br>
SEQ ID NO: 540 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT242 identified from the clone CTL2gam8.<br>
SEQ ID NO: 541 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT243 (lpxD) identified from the clone CTL2gam8.<br>
SEQ ID NO: 542 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT089 identified from the clones CTL2-9, CTL2gaml, CTL2gaml7, and CTL2-19(5').<br>
SEQ ID NO: 543 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT610 identified from the clone CTL2-10 (5')(3').<br>
SEQ ID NO: 544 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT611 identified from the clone CTL2-10(5').<br>
SEQ ID NO: 545 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT154 identified from the clones CTL2gam10 and CTL2gam21.<br>
SEQ ID NO: 546 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT833 (infC) identified from the clone CTL2-12.<br>
SEQ ID NO: 547 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT827 (nrdA) identified from the clones CTL2-16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 548 sets forth the full-length C trachomatis serovar D DNA sequence homologous to the C. ti-achomatis LGV II sequence for CT828 (nrdB) identified from the clones CTL2-16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 549 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. ti-achomatis LGV II sequence for CT067 (ytgA) identified from the clone CTL2gaml 8.<br>
SEQ ID NO: 550 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT088 (sycE) identified from the clones CTL2-19(5').<br>
SEQ ID NO: 551 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT087 identified from the clone CTL2-19(5').<br>
SEQ ID NO: 552 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT588 (rsbu) identified from the clone CTL2gam23.<br>
SEQ ID NO: 553 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT199 (oppB1) identified from the clone CTL2gam24.<br>
SEQ ID NO: 554 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT545 (dnaE) identified from the clone CTL2-4.<br>
SEQ ID NO: 555 sets forth the full-length C trachomatis serovar D DNA sequence homologous to the C. ti-achomatis LGV II sequence for CT288 identified from the clones CTL2gam27.<br>
SEQ ID NO: 556 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT413 (pmpB) identified from the clone CTL2gam30(5')(3').<br>
SEQ ID NO: 557 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT-ORF3 identified from the clones CTL2gaml5(5'), CTL2-16(5'), CTL2-18(5') and CTL2-23.<br>
SEQ ID NO: 558 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for pCT-ORF4 identified from the clone CTL2-18(3').<br>
SEQ ID NO: 559 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT-ORF5 identified from the clones CTL2-18(3').<br>
SEQ ID NO: 560 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT751 (amn) identified from the clones CTL2-1 and CTL2-5.<br>
SEQ ID NO: 561 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT322 (tuff) identified from the clone CTL2-2.<br>
SEQ ID NO: 562 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV E sequence for CT032 (metG) identified from the clones CTL2gam2, CTL2-3(5') and CTL2-4. <br>
SEQ ID NO: 563 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV H sequence for CT031 identified from the clone CTL2-3(5')(3').<br>
SEQ ID NO: 564 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT030 (gmK) identified from the clones CTL2-3(3') and CTL2-21.<br>
SEQ ID NO: 565 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT064 (lepA) identified from the clone CTL2gam5.<br>
SEQ ID NO: 566 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT265 (accA) identified from the clone CTL2-6.<br>
SEQ iD NO: 567 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT624 (msbA) identified from the clones CTL2-6.<br>
SEQ ID NO: 568 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT256 identified from the clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 569 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT257 identified from the clones CTL2gam6(5') and CTL2-11(5').<br>
SEQ ID NO: 570 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV H sequence for CT384 identified from the clones CTL2gam6(3') and CTL2-11(3').<br>
SEQ ID NO: 571 sets forth the full-length G trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT473 identified from the clone CTL2-8b.<br>
SEQ ID NO: 572 sets forth the full-length G trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV E sequence for CT474 identified from the clones CTL2-8b.<br>
SEQ ID NO: 573 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT139 (oppA1) identified from the clones CTL2-8b.<br>
SEQ ID NO: 574 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT060 (flhA) identified from the clone CTL2-8b.<br>
SEQ ID NO: 575 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT242 identified from the clone CTL2gam8.<br>
SEQ ID NO: 576 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT243 (IpxD) identified from the clone CTL2gam8.<br>
SEQ ID NO: 577 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT089 identified from the clones CTL2-9, CTL2gam1, CTL2gaml7, and CTL2-19(5').<br>
SEQ ID NO: 578 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT610 identified from the clone CTL2-10 (5')(3').<br>
SEQ ID NO: 579 sets forth the full-length C trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT611 identified from the clone CTL2-10(5').<br>
SEQ ID NO: 580 sets forth the full-length C. ti-achomatis serovar D amino acid sequence homologous to the C trachomatis LGV II sequence for CT154 identified from the clones CTL2gaml0 and CTL2gam21.<br>
SEQ ID NO: 581 sets forth the full-length C. trachomatis serovar D amino acid sequence homologous to the C. trachomatis LGV II sequence for CT833 (infC) identified from the clone CTL2-12.<br>
SEQ ID NO: 582 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT-ORF3 identified from the clones CTL2gaml5(5'), CTL2-16(5'), CTL2-18(5') and CTL2-23.<br>
SEQ ID NO: 583 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT827 (nrdA) identified from the clones CTL2-16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 584 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT828 (nrdB) identified from the clones CTL2-16(3') and CTL2gaml5(3').<br>
SEQ ID NO: 585 sets forth the full-length C trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT067 (ytgA) identified from the clone CTL2gaml 8.<br>
SEQ ID NO: 586 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for pCT-ORF4 identified from the clone CTL2-18(3')<br>
SEQ ID NO: 587 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT-ORF5 identified from the clones CTL2-18(3').<br>
SEQ ID NO: 588 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV H sequence for CT087 identified from the clone CTL2-19(5').<br>
SEQ ID NO: 589 sets forth the full-length C trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT088 (sycE) identified from the clones CTL2-19(5').<br>
SEQ ID NO: 590 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT588 (rsbu) identified from the clone CTL2gam23.<br>
SEQ 3D NO: 591 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT199 (oppB1) identified from the clone CTL2gam24.<br>
SEQ ID NO: 592 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT545 (dnaE) identified from the clone CTL2-4.<br>
SEQ ID NO: 593 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT288 identified from the clones CTL2gam27.<br>
SEQ ID NO: 594 sets forth the full-length C. trachomatis serovar D DNA sequence homologous to the C. trachomatis LGV II sequence for CT413 (pmpB) identified from the clone CTL2gam30(5')(3').<br>
SEQ ID NO: 595 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0406, of the Chlamydia trachomatis gene CT102.<br>
SEQ ID NO: 596 sets forth the DNA sequence for the Chlamydia pneumoniae homologue, CPn0315, of the Chlamydia trachomatis gene CT098.<br>
SEQ ID NO: 597 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0406, of the Chlamydia trachomatis gene CT102.<br>
SEQ ID NO: 598 sets forth the amino acid sequence for the Chlamydia pneumoniae homologue, CPn0315, of the Chlamydia trachomatis gene CT098.<br>
SEQ ID NO: 599 sets forth the amino acid sequence for Chlamydia trachomatis serovar D CT287 protein.<br>
DESCRIPTION OF THE FIGURES<br>
Fig. 1 illustrates induction of INF-γ from a Chlamydia-specific T cell line activated by target cells expressing clone 4C9-18#2.<br>
Fig. 2 illustrates retroviral vectors pBIB-KS1,2,3  modified to contain a Kosak translation initiation site and stop codons.<br>
Fig. 3 shows specific lysis in a chromium release assay of P815 cells pulsed with Chlamydia peptides CtC7.8-12 (SEQ ID NO: 18)and'CtC7.8-13 (SEQ ID NO: 19).     Fig. 4 shows antibody isotype titers in C57B1/6 mice immunized with C. trachomatis SWIB protein.<br>
Fig. 5 shows Chlamydia-specific T-cell proliferative responses in splenocytes from C3H mice immunized with C. trachomatis SWIB protein.<br>
Fig. 6 illustrates the 5' and 3' primer sequences designed from C. pneumoniae which were used to isolate the SWIB and S13 genes from C. pneumoniae.<br>
Figs. 7A and 7B show induction of IFN-γ from a human anti-chlamydia T-cell line<br>
(TCL-8) capable of cross-reacting to C. trachomatis and C. pneumonia upon activation<br>
by monocyte-derived dendritic cells expressing chlamydial proteins.<br>
Fig. 8 shows the identification of T cell epitopes in Chlamydial ribosomal S13 protein<br>
with T-cell line TCL 8 EB/DC.<br>
Fig. 9A and B illustrate the proliferative response of CP-21 T-cells generated against C.<br>
pnuemoniae-infected dendritic cells to recombinant C. pneumonia-SWIB protein, but<br>
not C. trachomatis SWIB protein.<br>
Fig. 10 shows the C. trachomatis-specific SWIB proliferative responses of a primary T-<br>
cell line (TCT-10 EB) from an asymptomatic donor.<br>
Fig. 11 illustrates the identification of T-cell epitope in C. trachomatis SWIB with an<br>
antigen specific T-cell line (TCL-10 EB).<br>
Fig. 12 shows the C. trachomatis-specific proliferative responses of primary T cell lines<br>
generated from two patients against the CT specific antigens CT622, CT875 and CT<br>
EB.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As noted above, the present invention is generally directed to compositions and methods for the diagnosis and treatment of Chlamydial infection. In one aspect, the compositions of the subject invention include polypeptides that comprise at least one immunogenic portion of a Chlamydia antigen, or a variant thereof.<br>
In specific embodiments, the subject invention discloses polypeptides comprising an immunogenic portion of a Chlamydia antigen, wherein the Chlamydia antigen comprises an amino acid sequence encoded by a polynucleotide molecule disclosed herein, the complements of said nucleotide sequences, and variants of such sequences.<br>
As used herein, the term "polypeptide" encompasses amino acid chains of any length, including full length proteins (i.e., antigens), wherein the amino acid residues are linked by covalent peptide bonds. Thus, a polypeptide comprising an immunogenic portion of one of the inventive antigens may consist entirely of the immunogenic portion, or may contain additional sequences.  The additional sequences may be derived from the native Chlamydia antigen or may be heterologous, and such sequences may (but need not) be immunogenic.<br>
The term "polynucleotide(s)," as used herein, means a single or double-stranded polymer of deoxyribonucleotide or ribonucleotide bases and includes DNA and corresponding RNA molecules, including HnRNA and mRNA molecules, both sense and anti-sense strands, and comprehends cDNA, genomic DNA and recombinant DNA, as well as wholly or partially synthesized polynucleotides. An HnRNA molecule contains introns and corresponds to a DNA molecule in a generally one-to-one manner. An mRNA molecule corresponds to an HnRNA and DNA molecule from which the introns have been excised. A polynucleotide may consist of an entire gene, or any portion thereof. Operable anti-sense polynucleotides may comprise a fragment of the corresponding polynucleotide, and the definition of "polynucleotide" therefore includes all such operable anti-sense fragments.<br>
An "immunogenic portion" of an antigen is a portion that is capable of reacting with sera obtained from a Chlamydia-ivfected individual (i.e., generates an absorbance reading with sera from infected individuals that is at least three standard deviations above the absorbance obtained with sera from uninfected individuals, in a representative ELISA assay described herein). Such immunogenic portions generally comprise at least about 5 amino acid residues, more preferably at least about 10, and most preferably at least about 20 amino acid residues.   Methods for preparing and identifying immunogenic portions of antigens of known sequence are well known in the art and include those summarized in Paul, Fundamental Immunology, 3rd ed., Raven Press, 1993, pp. 243-247 and references cited therein. Such techniques include screening polypeptides for the ability to react with antigen-specific antibodies, antisera and/or T-cell lines or clones.   As used herein, antisera and antibodies are "antigen-specific" if they specifically bind to an antigen (i.e., they react with the protein in an ELISA or other immunoassay, and do not react detectably with unrelated proteins). Such antisera and antibodies may be prepared as described herein, and using well known techniques. An immunogenic portion of a native Chlamydia protein is a portion that reacts with such antisera and/or T-cells at a level that is not substantially less than the reactivity of the full length polypeptide (e.g., in an ELISA and/or T-cell reactivity assay).   Such immunogenic portions may react within such assays at a level that is similar to or greater than the reactivity of the full length polypeptide. Such screens may generally be performed using methods well known to those of ordinary skill in the art, such as those described in Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988. For example, a polypeptide may be immobilized on a solid support and contacted with patient sera to allow binding of antibodies within the sera to the immobilized polypeptide. Unbound sera may then be removed and bound antibodies detected using, for example, 125I-labeled Protein A.<br>
Examples of immunogenic portions of antigens contemplated by the present invention include, for example, the T cell stimulating epitopes provided in SEQ ID NO: 9, 10, 18, 19, 31, 39, 93-96, 98, 100-102, 106, 108, 138-140, 158, 167, 168, 246, 247 and 254-256. Polypeptides comprising at least an immunogenic portion of one or more Chlamydia antigens as described herein may generally be used, alone or in combination, to detect Chlamydial infection in a patient.<br>
The compositions and methods of the present invention also encompass variants of the above polypeptides and polynucleotide molecules. Such variants include, but are not limited to, naturally occurring allelic variants of the inventive sequences. In particular, variants include other Chlamydiae serovars, such as serovars D, E and F, as well as the several LGV serovars which share homology to the inventive polypeptide and polynucleotide molecules described herein. Preferably, the serovar homologues show 95-99% homology to the corresponding polypeptide sequence(s) described herein.<br>
A polypeptide "variant," as used herein, is a polypeptide that differs from the recited polypeptide only in conservative substitutions and/or modifications, such that the antigenic properties of the polypeptide are retained. In a preferred embodiment, variant polypeptides differ from an identified sequence by substitution, deletion or addition of five amino acids or fewer. Such variants may generally be identified by modifying one of the above polypeptide sequences, and evaluating the antigenic properties of the modified polypeptide using, for example, the representative procedures described herein. In other words, the ability of a variant to react with antigen-specific antisera may be enhanced or unchanged, relative to the native protein, or may be diminished by less than 50%, and preferably less than 20%, relative to the native protein. Such variants may generally be identified by modifying one of the above polypeptide sequences and evaluating the reactivity of the modified polypeptide with antigen-specific antibodies or antisera as described herein. Preferred variants include those in which one or more portions, such as an N-terminal leader sequence or transmembrane domain, have been removed. Other preferred variants include variants in which a small portion (e.g., 1-30 amino acids, preferably 5-15 amino acids) has been removed from the N- and/or C-terminal of the mature protein.<br>
As used herein, a "conservative substitution" is one in which an amino<br>
acid is substituted for another amino acid that has similar properties, such that one<br>
skilled in the art of peptide chemistry would expect the secondary structure and<br>
hydropathic nature of the polypeptide to be substantially unchanged. Amino acid<br>
substitutions may generally be made on the basis of similarity in polarity, charge,<br>
solubility, hydrophobicity, hydrophilicity and/or the amphipathic nature of the residues.<br>
For example, negatively charged amino acids include aspartic acid, and glutamic acid;<br>
positively charged amino acids include lysine and arginine; and amino acids with<br>
uncharged polar head groups having similar hydrophilicity values include leucine,<br>
isoleucine and valine; glycine and alanine; asparagine and glutamine; and serine,<br>
threonine, phenylalanine and tyrosine. Other groups of amino acids that may represent<br>
conservative changes include: (1) ala, pro, gly, glu, asp, gln, asn, ser, thr; (2) cys, ser,<br>
tyr, thr; (3) val, ile, leu, met, ala, phe; (4) lys, arg, his; and (5) phe, tyr, trp, his.   A<br>
variant may also, or alternatively, contain nonconservative changes.   In a preferred<br>
embodiment, variant polypeptides differ from a native sequence by substitution,<br>
deletion or addition of five amino acids or fewer. Variants may also (or alternatively)<br>
be modified by, for example, the deletion or addition of amino acids that have minimal<br>
influence on the immunogenicity, secondary structure and hydropathic nature of the<br>
polypeptide. Variants may also, or alternatively, contain other modifications, including<br>
the deletion or addition of amino acids that have minimal influence on the antigenic<br>
properties, secondary structure and hydropathic nature of the polypeptide. For example,<br>
a polypeptide may be conjugated to a signal (or leader) sequence at the N-terminal end<br>
of the protein which co-translationally or post-translationally directs transfer of the<br>
protein. The polypeptide may also be conjugated to a linker or other sequence for ease<br>
of synthesis, purification or identification of the polypeptide (e.g., poly-His), or to<br>
enhance binding of the polypeptide to a solid support. For example, a polypeptide may be conjugated to an immunoglobulin Fc region.<br>
A polynucleotide "variant" is a sequence that differs from the recited nucleotide sequence in having one or more nucleotide deletions, substitutions or additions such that the immunogenicity of the encoded polypeptide is not diminished, relative to the native protein. The effect on the immunogenicity of the encoded polypeptide may generally be assessed as described herein. Such modifications may be readily introduced using standard mutagenesis techniques, such as oligonucleotide-directed site-specific mutagenesis as taught, for example, by Adelman et al. (DNA, 2:183, 1983). Nucleotide variants may be naturally occurring allelic variants as discussed below, or non-naturally occurring variants. The polypeptides provided by the present invention include variants that are encoded by polynucleotide sequences which are substantially homologous to one or more of the polynucleotide sequences specifically recited herein. "Substantial homology," as used herein, refers to polynucleotide sequences that are capable of hybridizing under moderately stringent conditions. Suitable moderately stringent conditions include prewashing in a solution of 5X SSC, 0.5% SDS, 1.0 mM EDTA (pH 8.0); hybridizing at 50°C-65°C, 5X SSC, overnight or, in the event of cross-species homology, at 45°C with 0.5X SSC; followed by washing twice at 65°C for 20 minutes with each of 2X, 0.5X and 0.2X SSC containing 0.1% SDS. Such hybridizing polynucleotide sequences are also within the scope of this invention, as are nucleotide sequences that, due to code degeneracy, encode a polypeptide that is the same as a polypeptide of the present invention.<br>
Two nucleotide or polypeptide sequences are said to be "identical" if the sequence of nucleotides or amino acid residues in the two sequences is the same when aligned for maximum correspondence as described below. Comparisons between two sequences are typically performed by comparing the sequences over a comparison window to identify and compare local regions of sequence similarity. A "comparison window" as used herein, refers to a segment of at least about 20 contiguous positions, usually 30 to about 75, 40 to about 50, in which a Sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned.<br>
Optimal alignment of sequences for comparison may be conducted using the Megalign program in the Lasergene suite of bioinformatics software (DNASTAR, Inc., Madison, WI), using default parameters. This program embodies several alignment schemes described in the following references: Dayhoff, M.O. (1978) A model of evolutionary change in proteins - Matrices for detecting distant relationships. In Dayhoff, M.O. (ed.) Atlas of Protein Sequence and Structure, National Biomedical Resarch Foundaiton, Washington DC Vol. 5, Suppl. 3, pp. 345-358; Hein J. (1990) Unified Approach to Alignment and Phylogenes pp. 626-645 Methods in Enzymology vol. 183, Academic Press, Inc., San Diego, CA; Higgins, D.G. and Sharp, P.M. (1989) Fast and sensitive multiple sequence alignments on a microcomputer CABIOS 5:151-153; Myers, E.W. and Muller W. (1988) Optimal alignments in linear space CABIOS 4:11-17; Robinson, E.D. (1971) Comb. Theor 11:105; Santou, N. Nes, M. (1987) The neighbor joining method. A new method for reconstructing phylogenetic trees Mol. Biol. Evol 4:406-425; Sneath, P.H.A. and Sokal, R.R. (1973) Numerical Taxonomy-the Principles and Practice of Numerical Taxonomy, Freeman Press, San Francisco, CA; Wilbur, W.J. and Lipman, DJ. (1983) Rapid similarity searches of nucleic acid and protein data banks Proc. Natl Acad, Sci. USA 80:726-730.<br><br>
Alternatively, optimal alignment of sequences for comparison may be conducted by the local identity algorithm of Smith and Waterman (1981) Add. APL. Math 2:482, by the identity alignment algorithm of Needleman and Wunsch (1970) J. Mol. Biol. 48:443, by the search for similarity methods of Pearson and Lipman (1988) Proc. Natl. Acad. Sci. (U.S.A.) 85: 2444, by computerized implementations of these algorithms (GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, WI), or by inspection.<br>
One illustrative example of algorithms that are suitable for determining percent sequence identity and sequence similarity are the BLAST and BLAST 2.0 algorithms, which are described in Altschul et al. (1977) Nuc Acids Res. 25:3389-3402 and Altschul et al. (1990) J. Mol. Biol. 215:403-410, respectively. BLAST and BLAST 2.0 can be used, for example with the parameters described herein, to determine percent sequence identity for the polynucleotides and polypeptides of the invention. Software for performing BLAST analyses is publicly available through the National Center for<br>
Biotechnology Information (http://www.ncbi.nlm.nih.gov/) In one illustrative example, cumulative scores can be calculated using, for nucleotide sequences, the parameters M (reward score for a pair of matching residues; always &gt;0) and N (penalty score for mismatching residues; always 
Preferably, the "percentage of sequence identity" is determined by comparing two optimally aligned sequences over a window of comparison of at least 20 positions, wherein the portion of the polynucleotide or amino acid sequence in the comparison window may comprise additions or deletions (i.e. gaps) of 20 percent or less, usually 5 to 15 percent, or 10 to 12 percent, as compared to the reference sequences (which does not comprise additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by detennining the number of positions at which the identical nucleic acid bases or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the reference sequence (i.e. the window size) and multiplying the results by 100 to yield the percentage of sequence identity.<br>
Therefore, the present invention provides polynucleotide and polypeptide sequences having substantial identity to the sequences disclosed herein, for example those comprising at least 50% or more sequence identity, preferably at least 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% or higher, sequence identity compared to a polynucleotide or polypeptide sequence of this invention using the methods described herein, (e.g., BLAST analyisis using standard parameters, as described below).    One skilled in this art will recognize that these- values can be appropriately adjusted to determine corresponding identity of proteins encoded by two polynucleotide sequences by taking into account codon degeneracy, amino acid similarity, reading frame positioning and the like.<br>
In additional embodiments, the present invention provides isolated polynucleotides or polypeptides comprising various lengths of contiguous stretches of sequence identical to or complementary to one or more of the sequences disclosed herein. For example, polynucleotides and polypeptides encompassed by this invention may comprise at least about 15, 20, 30, 40, 50, 75, 100, 150, 200, 300, 400, 500 or 1000 or more contiguous nucleotides of one or more of the disclosed sequences, as well as all intermediate lengths therebetween. It will be readily understood that "intermediate lengths", in this context, means any length between the quoted values, such as 16, 17, 18, 19, etc.; 21, 22, 23, etc.; 30, 31, 32, etc.; 50, 51, 52, 53, etc.; 100, 101, 102, 103, etc.; 150, 151, 152, 153, etc.; including all integers through the 200-500; 500-1,000, and the like.<br>
The polynucleotides of the present invention, or fragments thereof, regardless of the length of the coding sequence itself, may be combined with other DNA sequences, such as promoters, polyadenylation signals, additional restriction enzyme sites, multiple cloning sites, other coding segments, and the like, such that their overall length may vary considerably. It is therefore contemplated that a nucleic acid fragment of almost any length may be employed, with the total length preferably being limited by the ease of preparation and use in the intended recombinant DNA protocol. For example, illustrative DNA segments with total lengths of about 10,000, about 5000, about 3000, about 2,000, about 1,000, about 500, about 200, about 100, about 50 base pairs in length, and the like, (including all intermediate lengths) are contemplated to be useful in many implementations of this invention.<br>
Also included in the scope of the present invention are alleles of the genes encoding the nucleotide sequences recited in herein. As used herein, an "allele" or "allellic sequence" is an alternative form of the gene which may result from at least one mutation in the nucleic acid sequence. Alleles may result in altered mRNAs or polypeptides whose structure or function may or may not be altered. Any given gene may have none, one, or many allelic forms. Common mutational changes which give rise to alleles are generally ascribed to natural deletions, additions, or substitutions of nucleotides. Each of these types of changes may occur alone or in combination with the others, one or more times in a given sequence.<br>
In specific embodiments, the subject invention discloses polypeptides<br>
comprising at least an immunogenic portion of a Chlamydia antigen (or a variant of<br>
such an antigen), that comprises one or more of the amino acid sequences encoded by<br>
(a) a polynucleotide sequence selected from the group consisting of SEQ ID NO: 358-<br>
361, 407-430, 525-559, 582-598; (b)the complements of such DNA sequences or<br>
(c) DNA sequences substantially homologous to a sequence in (a) or (b). As discussed<br>
in the Examples below, several of the Chlamydia antigens disclosed herein recognize a<br>
T cell line that recognizes both Chlamydia trachomatis and Chlamydia pneumoniae<br>
infected monocyte-derived dendritic cells, indicating that they may represent an<br>
immunoreactive epitope shared by Chlamydia trachomatis and Chlamydia pneumoniae.<br>
The antigens may thus be employed in a vaccine for both C. trachomatis genital tract<br>
infections and for C. pneumonia infections. Further characterization of these Chlamydia<br>
antigens from Chlamydia trachomatis and Chlamydia pneumonia to determine the<br>
extent of cross-reactivity is provided in Example 6. Additionally, Example 4 describes<br>
cDNA fragments (SEQ ID NO: 15, 16 and 33) isolated from C. trachomatis which<br>
encode proteins (SEQ ID NO: 17-19 and 32) capable of stimulating a Chlamydia-<br>
specific murine CD8+T cell line.<br>
In general, Chlamydia antigens, and polynucleotide sequences encoding<br>
such antigens, may be prepared using any of a variety of procedures. For example, polynucleotide molecules encoding Chlamydia antigens may be isolated from a Chlamydia genomic or cDNA expression library by screening with a Chlamydia-specific T cell line as described below, and sequenced using techniques well known to those of skill in the art. Additionally, a polynucleotide may be identified, as described in more detail below, by screening a microarray of cDNAs for Chlamydia-associattd expression (i.e., expression that is at least two fold greater in Chlamydia-infected cells than in controls, as detennined using a representative assay provided herein). Such screens may be performed using a Synteni microarray (Palo Alto, CA) according to the manufacturer's instructions (and essentially as described by Schena et al., Proc. Natl. Acad. Sci. USA 93:10614-10619, 1996 and Heller et al., Proc. Natl. Acad Sci. USA 94:2150-2155,  1997).    Alternatively, polypeptides may be amplified from cDNA<br>
prepared from cells expressing the proteins described herein.. Such polynucleotides may be amplified via polymerase chain reaction (PCR). For this approach, sequence-specific primers may be designed based on the sequences provided herein, and may be purchased or synthesized.<br>
Antigens may be produced recombinantly, as described below, by inserting a polynucleotide sequence that encodes the antigen into an expression vector and expressing the antigen in an appropriate host. Antigens may be evaluated for a desired property, such as the ability to react with sera obtained from a Chlamydia-infected individual as described herein, and may be sequenced using, for example, traditional Edman chemistry. See Edman and Berg, Eur. J. Biochem. 80:116-132, 1967.<br>
Polynucleotide sequences encoding antigens may also be obtained by screening an appropriate Chlamydia cDNA or genomic DNA library for polynucleotide sequences that hybridize to degenerate oligonucleotides derived from partial amino acid sequences of isolated antigens. Degenerate oligonucleotide sequences for use in such a screen may be designed and synthesized, and the screen may be performed, as described (for example) in Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratories, Cold Spring Harbor, NY (and references cited therein). Polymerase chain reaction (PCR) may also be employed, using the above oligonucleotides in methods well known in the art, to isolate a nucleic acid probe from a cDNA or genomic library. The library screen may then be performed using the isolated probe.<br>
An amplified portion may be used to isolate a full length gene from a suitable library (e.g., a Chlamydia cDNA library) using well known techniques. Within such techniques, a library (cDNA or genomic) is screened using one or more polynucleotide probes or primers suitable for amplification. Preferably, a library is size-selected to include larger molecules. Random primed libraries may also be preferred for identifying 5' and upstream regions of genes. Genomic libraries are preferred for obtaining introns and extending 5' sequences.<br>
For hybridization techniques, a partial sequence may be labeled (e.g., by nick-translation or end-labeling with 32P) using well known techniques. A bacterial or bacteriophage library is then screened by hybridizing filters containing denatured bacterial colonies (or lawns containing phage plaques) with the labeled probe (see<br>
Sambrook et al., Molecular Cloning:, A Laboratory Manual, Cold Spring Harbor Laboratories, Cold Spring Harbor, NY, 1989). Hybridizing colonies or plaques are selected and expanded, and the DNA is isolated for further analysis. cDNA clones may be analyzed to determine the amount of additional sequence by, for example, PCR using a primer from the partial sequence and a primer from the vector. Restriction maps and partial sequences may be generated to identify one or more overlapping clones. The complete sequence may then be determined using standard techniques, which may involve generating a series of deletion clones. The resulting overlapping sequences are then assembled into a single contiguous sequence. A full length cDNA molecule can be generated by ligating suitable fragments, using well known techniques.<br>
Alternatively, there are numerous amplification techniques for obtaining a full length coding sequence from a partial cDNA sequence. Within such techniques, amplification is generally performed via PCR. Any of a variety of commercially available kits may be used to perform the amplification step. Primers may be designed using techniques well known in the art {see, for example, Mullis et al., Cold Spring Harbor Symp. Quant. Biol. 51:263, 1987; Erlich ed., PCR Technology, Stockton Press, NY, 1989), and software well known in the art may also be employed. Primers are preferably 22-30 nucleotides in length, have a GC content of at least 50% and anneal to the target sequence at temperatures of about 68°C to 72°C. The amplified region may be sequenced as described above, and overlapping sequences assembled into a contiguous sequence.<br>
One such amplification technique is inverse PCR {see Triglia et al., Nucl. Acids Res. 16:8186, 1988), which uses restriction enzymes to generate a fragment in the known region of the gene. The fragment is then circularized by intramolecular ligation and used as a template for PCR with divergent primers derived from the known region. Within an alternative approach, sequences adjacent to a partial sequence may be retrieved by amplification with a primer to a linker sequence and a primer specific to a known region. The amplified sequences are typically subjected to a second round of amplification with the same linker primer and a second primer specific to the known region. A variation on this procedure, which employs two primers that initiate extension in opposite directions from the known sequence, is described in WO 96/38591. Additional techniques include capture PCR (Lagerstrom et al., PCR Methods Applic. 1:111-19, 1991) and walking PCR (Parker et al., Nucl. Acids. Res. 19:3055-60, 1991). Transcription-Mediated Amplification, or TMA is another method that may be utilized for the amplification of DNA, rRNA, or mRNA, as described in Patent No. PCT/US91/03184. This autocatalytic and isothermic non-PCR based method utilizes two primers and two enzymes: RNA polymerase and reverse transcriptase. One primer contains a promoter sequence for RNA polymerase. In the first amplification, the promoter-primer hybridizes to the target rRNA at a defined site. Reverse transcriptase creates a DNA copy of the target rRNA by extension from the 3'end of the promoter-primer. The RNA in the resulting complex is degraded and a second primer binds to the DNA copy. A new strand of DNA is synthesized from the end of the primer by reverse transcriptase creating double stranded DNA. RNA polymerase recognizes the promoter sequence in the DNA template and initiates transcription. Each of the newly synthesized RNA amplicons re-enters the TMA process and serves as a template for a new round of replication leading to the expotential expansion of the RNA amplicon. Other methods employing amplification may also be employed to obtain a full length cDNA sequence.<br>
In certain instances, it is possible to obtain a full length cDNA sequence by analysis of sequences provided in an expressed sequence tag (EST) database, such as that available from GenBank. Searches for overlapping ESTs may generally be performed using well known programs (e.g., NCBI BLAST searches), and such ESTs may be used to generate a contiguous full length sequence. Full length cDNA sequences may also be obtained by analysis of genomic fragments.<br>
Polynucleotide variants may generally be prepared by any method known in the art, including chemical synthesis by, for example, solid phase phosphoramidite chemical synthesis. Modifications in a polynucleotide sequence may also be introduced using standard mutagenesis techniques, such as oligonucleotide-directed site-specific mutagenesis {see Adelman et al., DNA 2:183, 1983). Alternatively, RNA molecules may be generated by in vitro or in vivo transcription of DNA sequences encoding a Chlamydial protein, or portion thereof, provided that the DNA is incorporated into a vector with a suitable RNA polymerase promoter (such as T7 or SP6). Certain portions may be used to prepare an encoded polypeptide, as described herein. In addition, or alternatively, a portion may be administered to a patient such that the encodedpolypeptide is generated in vivo (e.g., by transfecting antigen-presenting cells, such as dendritic cells, with a cDNA construct encoding a Chlamydial polypeptide, and administering the transfected cells to the patient).<br>
A portion of a sequence complementary to a coding sequence (i.e., an antisense polynucleotide) may also be used as a probe or to modulate gene expression. cDNA constructs that can be transcribed into antisense RNA may also be introduced into cells of tissues to facilitate the production of antisense RNA. An antisense polynucleotide may be used, as described herein, to inhibit expression of a Chlamydial protein. Antisense technology can be used to control gene expression through triple-helix formation, which compromises the ability of the double helix to open sufficiently for the binding of polymerases, transcription factors or regulatory molecules (see Gee et al., In Huber and Carr, Molecular and Immunologic Approaches, Futura Publishing Co. (Mt. Kisco, NY; 1994)). Alternatively, an antisense molecule may be designed to hybridize with a control region of a gene (e.g., promoter, enhancer or transcription initiation site), and block transcription of the gene; or to block translation by inhibiting binding of a transcript to ribosomes.<br>
A portion of a coding sequence, or of a complementary sequence, may also be designed as a probe or primer to detect gene expression. Probes may be labeled with a variety of reporter groups, such as radionuclides and enzymes, and are preferably at least 10 nucleotides in length, more preferably at least 20 nucleotides in length and still more preferably at least 30 nucleotides in length. Primers, as noted above, are preferably 22-30 nucleotides in length.<br>
Any polynucleotide may be further modified to increase stability in vivo. Possible modifications include, but are not limited to, the addition of flanking sequences at the 5' and/or 3' ends; the use of phosphorothioate or 2' O-methyl rather than phosphodiesterase linkages in the backbone; and/or the inclusion of nontraditional bases such as inosine, queosine and wybutosine, as well as acetyl- methyl-, thio- and other modified forms of adenine, cytidine, guanine, thymine and uridine.<br>
Nucleotide sequences as described herein may be joined to a variety of other nucleotide sequences using established recombinant DNA techniques. For example, a polynucleotide may be cloned into any of a variety of cloning vectors, including plasmids, phagemids, lambda phage derivatives and cosmids. Vectors of particular interest include expression vectors, replication vectors, probe generation vectors and sequencing vectors. In general, a vector will contain an origin of replication functional in at least one organism, convenient restriction endonuclease sites and one or more selectable markers. Other elements will depend upon the desired use, and will be apparent to those of ordinary skill in the art.<br>
Synthetic polypeptides having fewer than about 100 amino acids, and generally fewer than about 50 amino acids, may be generated using techniques well known in the art. For example, such polypeptides may be synthesized using any of the commercially available solid-phase techniques, such as the Merrifield solid-phase synthesis method, where amino acids are sequentially added to a growing amino acid chain. See Merrifield, J. Am. Chem. Soc. 85:2149-2146, 1963. Equipment for automated synthesis of polypeptides is commercially available from suppliers such as Perkin Elmer/Applied BioSystems Division, Foster City, CA, and may be operated according to the manufacturer's instructions.<br>
As noted above, immunogenic portions of Chlamydia antigens may be prepared and identified using well known techniques, such as those summarized in Paul, Fundamental Immunology, 3d ed., Raven Press, 1993, pp. 243-247 and references cited therein. Such techniques include screening polypeptide portions of the native antigen for immunogenic properties. The representative ELISAs described herein may generally be employed in these screens. An immunogenic portion of a polypeptide is a portion that, within such representative assays, generates a signal in such assays that is substantially similar to that generated by the full length antigen. In other words, an immunogenic portion of a Chlamydia antigen generates at least about 20%, and preferably about 100%, of the signal induced by the full length antigen in a model ELISA as described herein.<br>
Portions and other variants of Chlamydia antigens may be generated by synthetic or recombinant means. Variants of a native antigen may generally be prepared using standard.mutagenesis techniques, such as oligonucleotide-directed site-specific mutagenesis. Sections of the polynucleotide sequence may also be removed using standard techniques to permit preparation of truncated polypeptides.<br>
Recombinant polypeptides containing portions and/or variants of a native antigen may be  readily prepared from  a polynucleotide  sequence encoding the polypeptide using a variety of techniques well known to those of ordinary skill in the art. For example, supematants from suitable host/vector systems which secrete recombinant protein into culture media may be first concentrated using a commercially available filter. Following concentration, the concentrate may be applied to a suitable purification matrix such as an affinity matrix or an ion exchange resin. Finally, one or more reverse phase HPLC steps can be employed to further purify a recombinant protein.<br>
Any of a variety of expression vectors known to those of ordinary skill in the art may be employed to express recombinant polypeptides as described herein. Expression may be achieved in any appropriate host cell that has been transformed or transfected with an expression vector containing a polynucleotide molecule that encodes a recombinant polypeptide. Suitable host cells include prokaryotes, yeast and higher eukaryotic cells. Preferably, the host cells employed are E. coli, yeast or a mammalian cell line, such as COS or CHO. The DNA sequences expressed in this manner may encode naturally occurring antigens, portions of naturally occurring antigens, or other variants thereof.<br>
In general, regardless of the method of preparation, the polypeptides disclosed herein are prepared in an isolated, substantially pure, form. Preferably, the polypeptides are at least about 80% pure, more preferably at least about 90% pure and most preferably at least about 99% pure.<br>
Within certain specific embodiments, a polypeptide may be a fusion protein mat comprises multiple polypeptides as described herein, or that comprises at least one polypeptide as described herein and an unrelated sequence, such as a known Chlamydial protein. A fusion partner may, for example, assist in providing T helper epitopes (an immunological fusion partner), preferably T helper epitopes recognized by humans, or may assist in expressing the protein (an expression enhancer) at higher yields than the native recombinant protein. Certain preferred fusion partners are both immunological and expression enhancing fusion partners. Other fusion partners may be selected so as to increase the solubility of the protein or to enable the protein to be targeted to desired intracellular compartments. Still further fusion partners include affinity tags, which facilitate purification of the protein. A DNA sequence encoding a fusion protein of the present invention may be constructed using known recombinant DNA techniques to assemble separate DNA sequences encoding, for example, the first and second polypeptides, into an appropriate expression vector. The 3' end of a DNA sequence encoding the first polypeptide is ligated, with or without a peptide linker, to the 5' end of a DNA sequence encoding the second polypeptide so that the reading frames of the sequences are in phase to permit mRNA translation of the two DNA sequences into a single fusion protein that retains the biological activity of both the first and the second polypeptides.<br>
A peptide linker sequence may be employed to separate the first and the second polypeptides by a distance sufficient to ensure that each polypeptide folds into its secondary and tertiary structures. Such a peptide linker sequence is incorporated into the fusion protein using standard techniques well known in the art. Suitable peptide linker sequences may be chosen based on the following factors: (1) their ability to adopt a flexible extended conformation; (2) their inability to adopt a secondary structure that could interact with functional epitopes on the first and second polypeptides; and (3) the lack of hydrophobic or charged residues that might react with the polypeptide functional epitopes. Preferred peptide linker sequences contain Gly, Asn and Ser residues. Other near neutral amino acids, such as Thr and Ala may also be used in the linker sequence. Amino acid sequences which may be usefully employed as linkers include those disclosed in Maratea et al., Gene 40:39-46, 1985; Murphy et al., Proc. Natl Acad. Sci. USA 83:8258-8562, 1986; U.S. Patent No. 4,935,233 and U.S. Patent No. 4,751,180. The linker sequence may be from 1 to about 50 amino acids in length. As an alternative to the use of a peptide linker sequence (when desired), one can utilize non-essential N-terminal amino acid regions (when present) on the first and second polypeptides to separate the functional domains and prevent steric hindrance.<br>
The ligated DNA sequences are operably linked to suitable transcriptional or translational regulatory elements. The Tegulatory elements responsible for expression of DNA are located only 5' to the DNA sequence encoding the first polypeptides. Similarly, stop codons required to end translation and transcription teimination signals are only present 3' to the DNA sequence encoding the second polypeptide.<br>
Fusion proteins are also provided that comprise a polypeptide of the present invention together with an unrelated immunogenic protein. Preferably the immunogenic protein is capable of eliciting a recall response. Examples of such proteins include tetanus, tuberculosis and hepatitis proteins {see, for example, Stoute et al. New Engl J. Med., 336:86-91, 1997).<br>
Within preferred embodiments, an immunological fusion partner is derived from protein D, a surface protein of the gram-negative bacterium Haemophilus influenza B (WO 91/18926). Preferably, a protein D derivative comprises approximately the first third of the protein (e.g., the first N-terminal 100-110 amino acids), and a protein D derivative may be lipidated. Within certain preferred embodiments, the first 109 residues of a Lipoprotein D fusion partner is included on the N-terminus to provide the polypeptide with additional exogenous T-cell epitopes and to increase the expression level in E. coli (thus functioning as an expression enhancer). The lipid tail ensures optimal presentation of the antigen to antigen presenting cells. Other fusion partners include the non-structural protein from influenzae virus, NS1 (hemaglutinin). Typically, the N-terminal 81 amino acids are used, although different fragments that include T-helper epitopes may be used.<br>
In another embodiment, the immunological fusion partner is the protein known as LYTA, or a portion thereof (preferably a C-terminal portion). LYTA is derived from Streptococcus pneumoniae, which synthesizes an N-acetyl-L-alanine amidase known as amidase LYTA (encoded by the LytA gene; Gene 43:265-292, 1986). LYTA is an autolysin that specifically degrades certain bonds in the peptidoglycan backbone. The C-terminal domain of the LYTA protein is responsible for the affinity to the choline or to some choline analogues such as DEAE. This property has been exploited for the development of E. coli C-LYTA expressing plasmids useful for expression of fusion proteins. Purification of hybrid proteins containing the C-LYTA fragment at the amino terminus has been described (see Biotechnology 70:795-798, 1992). Within a preferred embodiment, a repeat portion of LYTA may be incorporated into a fusion protein. A repeat portion is found in the C-terminal region starting at residue 178. A particularly preferred repeat portion incorporates residues 188-305.<br>
In another embodiment, a Mycobacterium tuberculosis-derived Ral2 polynucleotide is linked to at least an immunogenic portion of a polynucleotide of this invention. Ral2 compositions and methods for their use inenhancing expression of heterologous   polynucleotide   sequences   is   described   in  U.S.   Patent Application 60/158,585, the disclosure of which is incorporated herein by reference in its entirety. Briefly, Ral2 refers to a polynucleotide region that is a subsequence of a Mycobacterium tuberculosis MTB32A, nucleic acid. MTB32A is a serine protease of 32 KD molecular weight encoded by a gene in virulent and avirulent strains of M. tuberculosis. The nucleotide sequence and amino acid sequence of MTB32A have been described (U.S. Patent Application 60/158,585; see also, Skeiky et al., Infection and Immun. (1999) 67:3998-4007, incorporated herein by reference. In one embodiment, the Ral2 polypeptide used in the production of fusion polypeptides comprises a C-terminal fragment of the MTB32A coding sequence that is effective for enhancing the expression and/or immunogenicity of heterologous Chlamydial antigenic polypeptides with which it is fused. In another embodiment, the Ral2 polypeptide corresponds to an approximately 14 kD C-terminal fragment of MTB32A comprising some or all of amino acid residues 192 to 323 of MTB32A.<br>
Recombinant nucleic acids, which encode a fusion polypeptide comprising a Ral2 polypeptide and a heterologous Chlamydia polypeptide of interest, can be readily constructed by conventional genetic engineering techniques. Recombinant nucleic acids are constructed so that, preferably, a Ral2 polynucleotide sequence is located 5' to a selected heterologous Chlamydia polynucleotide sequence. It may also be appropriate to place a Ral2 polynucleotide sequence 3' to a selected heterologous polynucleotide sequence or to insert a heterologous polynucleotide sequence into a site within a Ral2 polynucleotide sequence.<br>
In addition, any suitable polynucleotide that encodes a Ral2 or a portion or other variant thereof can be used in constructing recombinant fusion polynucleotides comprising Ral2 and one or more Chlamydia polynucleotides disclosed herein. Preferred Ral2 polynucleotides generally comprise at least about 15 consecutive nucleotides, at least about 30 nucleotides, at least about 60 nucleotides, at least about 100 nucleotides, at least about 200 nucleotides, or at least about 300 nucleotides that encode a portion of a Ral2 polypeptide.<br>
Ral2 polynucleotides may comprise a native sequence (i.e., an endogenous sequence that encodes a Ral2 polypeptide or a portion thereof) or may comprise a variant of such a sequence. Ral2 polynucleotide variants may contain oneor more substitutions, additions, deletions and/or insertions such that the biological activity of the encoded fusion polypeptide is not substantially diminished, relative to a fusion polypeptide comprising a native Ral2 polypeptide. Variants preferably exhibit at least about 70% identity, more preferably at least about 80% identity and most preferably at least about 90% identity to a polynucleotide sequence that encodes a native Ra12 polypeptide or a portion thereof.<br>
In another aspect, the present invention provides methods for using one or more of the above polypeptides or fusion proteins (or polynucleotides encoding such polypeptides or fusion proteins) to induce protective immunity against Chlamydial infection in a patient. As used herein, a "patient" refers to any warm-blooded animal, preferably a human. A patient may be afflicted with a disease, or may be free of detectable disease and/or infection. In other words, protective immunity may be induced to prevent or treat Chlamydial infection.<br>
In this aspect, the polypeptide, fusion protein or polynucleotide molecule is generally present within a pharmaceutical composition or a vaccine. Pharmaceutical compositions may comprise one or more polypeptides, each of which may contain one or more of the above sequences (or variants thereof), and a physiologically acceptable carrier. Vaccines may comprise one or more of the above polypeptides and an immunostimulant, such as an adjuvant or a liposome (into which the polypeptide is incorporated). Such pharmaceutical compositions and vaccines may also contain other Chlamydia antigens, either incorporated into a combination polypeptide or present within a separate polypeptide.<br>
Alternatively, a vaccine may contain polynucleotides encoding one or more polypeptides or fusion proteins as described above, such that the polypeptide is generated in situ. In such vaccines, the polynucleotides may be present within any of a variety of delivery systems known to those of ordinary skill in the art, including nucleic acid expression systems, bacterial and viral expression systems. Appropriate nucleic acid expression systems contain the necessary polynucleotide sequences for expression in the patient (such as a suitable promoter and terminating signal). Bacterial delivery systems involve the administration of a bacterium (such as Bacillus-Calmette-Guerrin) that expresses an immunogenic portion of the polypeptide on its cell surface. In a<br><br>
preferred embodiment, the polynucleotides may be introduced using a viral expression system (e.g., vaccinia or other pox virus, retrovirus, or adenovirus), which may involve the  use   of a  non-pathogenic   (defective)  virus.      Techniques  for incorporating polynucleotides into such expression systems are well known to those of ordinary skill in the art. The polynucleotides may also be administered as "naked" plasmid vectors as described, for example, in Ulmer et al., Science 259:1745-1749, 1993 and reviewed by Cohen, Science 259:1691-l692, 1993. Techniques for incorporating DNA into such vectors are well known to those of ordinary skill in the art.  A retroviral vector may additionally transfer or incorporate a gene for a selectable marker (to aid in the identification or selection of transduced cells) and/or a targeting moiety, such as a gene that encodes a ligand for a receptor on a specific target cell, to render the vector target specific. Targeting may also be accomplished using an antibody, by methods known to those of ordinary skill in the art.<br>
Other formulations for therapeutic purposes include colloidal dispersion systems, such as macromolecule complexes, nanocapsules, microspheres, beads, and lipid-based systems including oil-in-water emulsions, micelles, mixed micelles, and liposomes. A preferred colloidal system for use as a delivery vehicle in vitro and in vivo is a liposome (i.e., an artificial membrane vesicle). The uptake of naked polynucleotides may be increased by incorporating the polynucleotides into and/or onto biodegradable beads, which are efficiently transported into the cells. The preparation and use of such systems is well known in the art.<br>
In a related aspect, a polynucleotide vaccine as described above may be administered simultaneously with or sequentially to either a polypeptide of the present invention or a known Chlamydia antigen. For example, administration of polynucleotides encoding a polypeptide of the present invention, either "naked" or in a delivery system as described above, may be followed by administration of an antigen in order to enhance the protective immune effect of the vaccine.<br>
Polypeptides and polynucleotides disclosed here in may also be employed in adoptive immunotherapy for the treatment of Chlamydial infection.    Adoptive immunotherapy may be broadly classified into either active or passive immunotherapy. In active immunotherapy, treatment relies on the in vivo stimulation of the endogenous host immune system with the administration of immune response-modifying agents (for example, vaccines, bacterial adjuvants, and/or cytokines).<br>
In passive immunotherapy, treatment involves the delivery of biologic reagents with established immune reactivity (such as effector cells or antibodies) that can directly or indirectly mediate anti-Chlamydia effects and does not necessarily depend on an intact host immune system.    Examples of effector cells include T lymphocytes (for example, CD8+ cytotoxic T-lymphocyte, CD4+ T-helper), killer cells (such as Natural Killer cells, lymphokine-activated killer cells), B cells, or antigen presenting cells (such as dendritic cells and macrophages) expressing the disclosed antigens. The polypeptides disclosed herein may also be used to generate antibodies or anti-idiotypic antibodies (as in U.S. Patent No. 4,918,164), for passive immunotherapy. The predominant method of procuring adequate numbers of T-cells for adoptive immunotherapy is to grow immune T-cells in vitro.   Culture conditions for expanding single antigen-specific T-cells to several billion in number with retention of antigen recognition in vivo are well known in the art. These in vitro culture conditions typically utilize intermittent stimulation with antigen, often in the presence of cytokines, such as IL-2, and non-dividing feeder cells.   As noted above, the immunoreactive polypeptides described herein may be used to rapidly expand antigen-specific T cell cultures in order to generate sufficient number of cells for immunotherapy.    In particular,   antigen-presenting   cells,   such   as   dendritic,   macrophage,   monocyte, fibroblast,   or   B-cells,   may   be   pulsed   with   immunoreactive   polypeptides,   or polynucleotide sequence(s) may be introduced into antigen presenting cells, using a variety of standard techniques well known in the art.   For example, antigen presenting cells may be transfected or transduced with a polynucleotide sequence, wherein said sequence contains a promoter region appropriate for increasing expression, and can be expressed as part of a recombinant virus or other expression system.   Several viral vectors may be used to transduce an antigen presenting cell, including pox virus, vaccinia virus, and adenovirus; also, antigen presenting cells may be transfected with polynucleotide sequences disclosed herein by a variety of means, including gene-gun technology, lipid-mediated delivery, electroporation, osmotic shock, and particlate delivery mechanisms,  resulting  in  efficient  and  acceptable expression  levels as determined by one of ordinary skill in the art. For cultured T-cells to be effective in herapy, the cultured T-cells must be able to grow and distribute widely and to survive long term in vivo. Studies have demonstrated that cultured T-cells can be induced to grow in vivo and to survive long term in substantial numbers by repeated stimulation with antigen supplemented with IL-2 (see, for example, Cheever, M., et al, "Therapy With Cultured T Cells: Principles Revisited," Immunological Reviews, 157:177, 1997).<br>
The polypeptides disclosed herein may also be employed to generate and/or isolate chlamydial-reactive T-cells, which can then be administered to the patient. In one technique, antigen-specific T-cell lines may be generated by in vivo immunization with short peptides corresponding to immunogenic portions of the disclosed polypeptides. The resulting antigen specific CD8+ or CD4+ T-cell clones may be isolated from the patient, expanded using standard tissue culture techniques, and returned to the patient.<br>
Alternatively, peptides corresponding to immunogenic portions of the polypeptides may be employed to generate Chlamydia reactive T cell subsets by selective in vitro stimulation and expansion of autologous T cells to provide antigen-specific T cells which may be subsequently transferred to the patient as described, for example, by Chang et al, (Crit. Rev. Oncol. Hematol, 22(3), 213, 1996). Cells of the immune system, such as T cells, may be isolated from the peripheral blood of a patient, using a commercially available cell separation system, such as Isolex™ System, available from Nexell Therapeutics, Inc. Irvine, CA. The separated cells are stimulated with one or more of the immunoreactive polypeptides contained within a delivery vehicle, such as a microsphere, to provide antigen-specific T cells. The population of antigen-specific T cells is then expanded using standard techniques and the cells are administered back to the patient.<br>
In other embodiments, T-cell and/or antibody receptors specific for the polypeptides disclosed herein can be cloned, expanded, and transferred into other vectors or effector cells for use in adoptive immunotherapy. In particular, T cells may be transfected with the appropriate genes to express the variable domains from chlamydia specific monoclonal antibodies as the extracellular recognition elements and joined to the T cell receptor signaling chains, resulting in T cell activation, specific lysis, and cytokine release. "This enables the T cell to redirect its specificity in an MHC-independent manner. See for example, Eshhar, Z., Cancer Immunol Immunother, 45(3-<br><br>
WO 02/08267<br><br>
PCT/US01/23121<br><br>
4): 131-6, 1997 and Hwu, P., et al, Cancer Res, 55(15):3369-73, 1995. Another embodiment may include the transfection of chlamydia antigen specific alpha and beta T cell receptor chains into alternate T cells, as in Cole, DJ, et al, Cancer Res, 55(4):748-52, 1995.<br>
In a further embodiment, syngeneic or autologous dendritic cells may be pulsed with peptides corresponding to at least an immunogenic portion of a polypeptide disclosed herein. The resulting antigen-specific dendritic cells may either be transferred into a patient, or employed to stimulate T cells to provide antigen-specific T cells which may, in turn, be administered to a patient. The use of peptide-pulsed dendritic cells to generate antigen-specific T cells and the subsequent use of such antigen-specific T cells to eradicate disease in a murine model has been demonstrated by Cheever et al, Immunological Reviews, 157:177,1997). Additionally, vectors expressing the disclosed polynucleotides may be introduced into stem cells taken from the patient and clonally propagated in vitro for autologous transplant back into the same patient.<br>
Within certain aspects, polypeptides, polynucleotides, T cells and/or binding agents disclosed herein may be incorporated into pharmaceutical compositions or immunogenic compositions {i.e., vaccines). Alternatively, a pharmaceutical composition may comprise an antigen-presenting cell {e.g. a dendritic cell) transfected with a Chlamydial polynucleotide such that the antigen presenting cell expresses a Chlamydial polypeptide. Pharmaceutical compositions comprise one or more such compounds and a physiologically acceptable carrier. Vaccines may comprise one or more such compounds and an immunostimulant. An immunostimulant may be any substance that enhances or potentiates an immune response to an exogenous antigen. Examples of immunostimulants include adjuvants, biodegradable microspheres {e.g., polylactic galactide) and liposomes (into which the compound is incorporated; see e.g., Fullerton, U.S. Patent No. 4,235,877). Vaccine preparation is generally described in, for example, M.F. Powell and MJ. Newman, eds., "Vaccine Design (the subunit and adjuvant approach)," Plenum Press (NY, 1995). Pharmaceutical compositions and vaccines within the scope of the present invention may also contain other compounds, which may be biologically active or inactive. For example, one or more immunogenic portions of other Chlamydial antigens may be present, either incorporated into a fusion polypeptide or as a separate compound, within the composition or vaccine.<br>
A pharmaceutical composition or vaccine may contain DNA encoding one or more of the polypeptides as described above, such that the polypeptide is generated in situ. As noted above, the DNA may be present within any of a variety of delivery systems known to those of ordinary skill in the art, including nucleic acid expression systems, bacteria and viral expression systems. Numerous gene delivery techniques are well known in the art, such as those described by Rolland, Crit. Rev. Therap. Drug Carrier Systems 75:143-198, 1998, and references cited therein. Appropriate nucleic acid expression systems contain the necessary DNA sequences for expression in the patient (such as a suitable promoter and terminating signal). Bacterial delivery systems involve the administration of a bacterium (such as Bacillus-Calmette-Guerrin) that expresses an immunogenic portion of the polypeptide on its cell surface or secretes such an epitope.<br>
In a preferred embodiment, the DNA may be introduced using a viral expression system {e.g., vaccinia or other pox virus, retrovirus, adenovirus, baculovirus, togavirus, bacteriophage, and the like), which often involves the use of a non¬pathogenic (defective), replication competent virus.<br>
For example, many viral expression vectors are derived from viruses of the retroviridae family. This family includes the murine leukemia viruses, the mouse mammary tumor viruses, the human foamy viruses, Rous sarcoma virus, and the immunodeficiency viruses, including human, simian, and feline. Considerations when designing retroviral expression vectors are discussed in Comstock et al. (1997).<br>
Excellent murine leukemia virus (MLV)-based viral expression vectors have been developed by Kim et al. (1998). In creating the MLV vectors, Kim et al. found that the entire gag sequence, together with the immediate upstream region, could be deleted without significantly affecting viral packaging or gene expression. Further, it was found that nearly the entire U3 region could be replaced with the immediately-early promoter of human cytomegalovirus without deleterious effects. Additionally, MCR and internal ribosome entry sites (IRES) could be added without adverse effects. Based on their observations, Kim et al. have designed a series of MLV-based expression vectors comprising one or more of the features described above.<br>
As more has been learned about human foamy virus (HFV), characteristics of HFV that are favorable for its use as an expression vector have been discovered. These characteristics include the expression of pol by splicing and start of translation at a defined initiation codon. Other aspects of HFV viral expression vectors are reviewed in Bodem et al. (1997).<br>
Murakami et al. (1997) describe a Rous sarcoma virus (RSV)-based replication-competent avian retrovirus vectors, IR1 and IR2 to express a heterologous gene at a high level. In these vectors, the IRES derived from encephalomyocarditis virus (EMCV) was inserted between the env gene and the heterologous gene. The IR1 vector retains the splice-acceptor site that is present downstream of the env gene while the IR2 vector lacks it. Murakami et al. have shown high level expression of several different heterologous genes by these vectors.<br>
Recently, a number of lentivirus-based retroviral expression vectors have been developed. Kafri et al. (1997) have shown sustained expression of genes delivered directly into liver and muscle by a human immunodeficiency virus (HIV)-based expression vector. One benefit of the system is the inherent ability of HIV to transduce non-dividing cells. Because the viruses of Kafri et al are pseudotyped with vesicular stomatitis virus G glycoprotein (VSVG), they can transduce a broad range of tissues and cell types.<br>
A large number of adenovirus-based expression vectors have been developed, primarily due to the advantages offered by these vectors in gene therapy applications. Adenovirus expression vectors and methods of using such vectors are the subject of a number of United States patents, including United States Patent No. 5,698,202, United States Patent No. 5,616,326, United States Patent No. 5,585,362, and United States Patent No. 5,518,913, all incorporated herein by reference.<br>
Additional adenoviral constructs are described in Khatri et al. (1997) and Tomanin et al (1997). Khatri et al. describe novel ovine adenovirus expression vectors and their ability to infect bovine nasal turbinate and rabbit kidney cells as well as a range of human cell type, including lung and foreskin fibroblasts as well as liver, prostate, breast, colon and retmal lines. Tomanin et al. describe adenoviral expression vectors containing the T7 RNA polymerase gene. When introduced into cells containing a heterologous gene operably linked to a T7 promoter, the vectors were able to drive gene expression from the T7 promoter. The authors suggest that this system may be useful for the cloning and expression of genes encoding cytotoxic proteins.<br>
Poxviruses are widely used for the expression of heterologous genes in mammalian cells. Over the years, the vectors have been improved to allow high expression of the heterologous gene and simplify the integration of multiple heterologous genes into a single molecule. In an effort to diminish cytopathic effects and to increase safety, vaccinia virus mutant and other poxviruses that undergo abortive infection in mammalian cells are receiving special attention (Oertli et al, 1997). The use of poxviruses as expression vectors is reviewed in Carroll and Moss (1997).<br>
Togaviral expression vectors, which includes alphaviral expression vectors have been used to study the structure and function of proteins and for protein production purposes. Attractive features of togaviral expression vectors are rapid and efficient gene expression, wide host range, and RNA genomes (Huang, 1996). Also, recombinant vaccines based on alphaviral expression vectors have been shown to induce a strong humoral and cellular immune response with good immunological memory and protective effects (Tubulekas et al, 1997). Alphaviral expression vectors and their use are discussed, for example, in Lundstrom (1997).<br>
In one study, Li and Garoff (1996) used Semliki Forest virus (SFV) expression vectors to express retroviral genes and to produce retroviral particles in BHK-21 cells. The particles produced by this method had protease and reverse transcriptase activity and were infectious. Furthermore, no helper virus could be detected in the virus stocks. Therefore, this system has features that are attractive for its<br>
use in gene therapy protocols.<br>
Baculoviral expression vectors have traditionally been used to express<br>
heterologous proteins in insect cells. Examples of proteins include mammalian<br>
cliemokine receptors (Wang et al, 1997), reporter proteins such as green fluorescent<br>
protein (Wu et al, 1997), and FLAG fusion proteins (Wu et al, 1997; Koh et al, 1997).<br>
Recent advances in baculoviral expression vector technology, including their use in<br>
virion display vectors and expression in mammalian cells is reviewed by Possee (1997).<br>
Other reviews on baculoviral expression vectors include Jones and Morikawa (1996)<br>
and O'Reilly (1997).<br>
Other suitable viral expression systems are disclosed, for example, in<br>
Fisher-Hoch et al., Proc. Natl. Acad. Sci. USA 86:317-321, 1989; Flexner et al., Ann.<br>
N.Y. Acad. Sci. 569:86-103, 1989; Flexner et al., Vaccine 8:11-21, 1990; U.S. Patent Nos. 4,603,112, 4,769,330, and 5,017,487; WO 89/01973; U.S. Patent No. 4,777,127; GB 2,200,651; EP 0,345,242; WO 91/02805; Berkner, Biotechniques 6:616-627, 1988; Rosenfeld et al., Science 252:431-434, 1991; Kolls et al., Proc. Natl. Acad, Sci. USA 91:215-219,1994; Kass-Eisler et al., Proc. Natl Acad. Sci. USA 90:11498-11502, 1993; Guzman et al., Circulation 55:2838-2848, 1993; and Guzman et al., Cir. Res. 75:1202-1207, 1993. Techniques for incorporating DNA into such expression systems are well known to those of ordinary skill in the art. In other systems, the DNA may be introduced as "naked" DNA, as described, for example, in Ulmer et al., Science 259:1745-1749, 1993 and reviewed by Cohen, Science 259:1691-1692, 1993. The uptake of naked DNA may be increased by coating the DNA onto biodegradable beads, which are efficiently transported into the cells.<br>
It will be apparent that a vaccine may comprise a polynucleotide and/or a polypeptide component, as desired. It will also be apparent that a vaccine may contain pharmaceutically acceptable salts of the polynucleotides and/or polypeptides provided herein. Such salts may be prepared from pharmaceutically acceptable non-toxic bases, including organic bases (e.g., salts of primary, secondary and tertiary amines and basic amino acids) and inorganic bases (e.g., sodium, potassium, lithium, ammonium, calcium and magnesium salts). While any suitable carrier known to those of ordinary skill in the art may be employed in the pharmaceutical compositions of this invention, the type of carrier will vary depending on the mode of administration. Compositions of the present invention may be formulated for any appropriate manner of administration, including for example, topical, oral, nasal, intravenous, intracranial, intraperitoneal, subcutaneous or intramuscular administration. For parenteral administration, such as subcutaneous injection, the carrier preferably comprises water, saline, alcohol, a fat, a wax or a buffer. For oral administration, any of the above carriers or a solid carrier, such as mannitol, lactose, starch, magnesium stearate, sodium saccharine, talcum, cellulose, glucose, sucrose, and magnesium carbonate, may be employed. Biodegradable microspheres (e.g., polylactate polyglycolate) may also be employed as carriers for the pharmaceutical compositions of this invention. Suitable biodegradable microspheres are disclosed, for example, in U.S. Patent Nos. 4,897,268 and 5,075,109.<br>
Such compositions may also comprise buffers (e.g., neutral buffered saline or phosphate buffered saline), carbohydrates (e.g., glucose, mannose, sucrose or dextrans), mannitol, proteins, polypeptides or amino acids such as glycine, antioxidants, bacteriostats, chelating agents such as EDTA or glutathione, adjuvants (e.g., aluminum hydroxide), solutes that render the formulation isotonic, hypotonic or weakly hypertonic with the blood of a recipient, suspending agents, thickening agents and/or preservatives. Alternatively, compositions of the present invention may be formulated as a lyophilizate. Compounds may also be encapsulated within liposomes using well known technology.<br>
Any of a variety of irnmunostimulants may be employed in the vaccines of this invention. For example, an adjuvant may be included. Most adjuvants contain a substance designed to protect the antigen from rapid catabolism, such as aluminum hydroxide or mineral oil, and a stimulator of immune responses, such as lipid A, Bortadella pertussis or Mycobacterium tuberculosis derived proteins. Suitable adjuvants are commercially available as, for example, Freund's Incomplete Adjuvant and Complete Adjuvant (Difco Laboratories, Detroit, MI); Merck Adjuvant 65 (Merck and Company, Inc., Rahway, NJ); AS-2 (SmithKline Beecham, Philadelphia, PA); aluminum salts such as aluminum hydroxide gel (alum) or aluminum phosphate; salts of calcium, iron or zinc; an insoluble suspension of acylated tyrosine; acylated sugars; cationically or anionically derivatized polysaccharides; polyphosphazenes; biodegradable microspheres; monophosphory] lipid A and quil A. Cytokines, such as GM-CSF or interleukin-2, -7, or -12, may also be used as adjuvants.<br>
Within the vaccines provided herein, under select circumstances, the adjuvant composition may be designed to induce an immune response predominantly of the Thl type or Th2 type. High levels of Thl-type cytokines (e.g., IFN-γ, TNFα, IL-2 and IL-12) tend to favor the induction of cell mediated immune responses to an administered antigen. In contrast, high levels of Th2-type cytokines (e.g., IL-4, IL-5, IL~6 and IL-10) tend to favor the induction of humoral immune responses. Following application of a vaccine as provided herein, a patient will support an immune response that includes Thl- and Th2-type responses. Within a preferred embodiment, in which a response is predominantly Thl-type, the level of Thl-type cytokines will increase to a greater extent than the level of Th2-type cytokines.  The levels of these cytokines may<br>
i'<br>
be readily assessed using standard assays. For a review of the families of cytokines, see Mosmann and Coffman, Ann. Rev. Immunol. 7:145-173, 1989.<br><br>
		Preferred adjuvants for, use in eliciting a predominantly Thl-type response include, for example, a combination of monophosphoryl lipid A, preferably 3-de-O-acylated monophosphoryl lipid A (3D-MPL), together with an aluminum salt. MPL adjuvants are available from Corixa Corporation (Seattle, WA; see US Patent Nos. 4,436,727; 4,877,611; 4,866,034 and 4,912,094). CpG-containing oligonucleotides (in which the CpG dinucleotide is unmethylated) also induce a predominantly Thl response. Such oligonucleotides are well known and are described, for example, in WO 96/02555 and WO 99/33488. Immunostimulatory DNA sequences are also described, for example, by Sato et al., Science 273:352, 1996.   Another preferred adjuvant is a<br>
t<br>
saponin, preferably QS21 (Aquila Biopharmaceuticals Inc., Framingham, MA), which may be used alone or in combination with other adjuvants. For example, an enhanced system involves the combination of a monophosphoryl lipid A and saponin derivative, such as the combination of QS21 and 3D-MPL as described in WO 94/00153, or a less reactogenic composition where the QS21 is quenched with cholesterol, as described in WO 96/33739. Other preferred formulations comprise an oil-in-water emulsion and tocopherol. A particularly potent adjuvant formulation involving QS21, 3D-MPL and tocopherol in an oil-in-water emulsion is described in WO 95/17210.<br>
Other preferred adjuvants include Montanide ISA 720 (Seppic, France), SAF (Chiron, California, United States), ISCOMS (CSL), MF-59 (Chiron), the SBAS series of adjuvants (e.g., SBAS-2 or SBAS-4, available from SmithKline Beecham, Rixensart, Belgium), Detox (Corixa Corporation; Seattle, WA), RC-529 (Corixa Corporation; Seattle, WA) and other arninoalkyl glucosaminide 4-phosphates (AGPs), such as those described in pending U.S. Patent Application Serial Nos. 08/853,826 and 09/074,720, the disclosures of which are incorporated herein by reference in their entireties.<br>
Any vaccine provided herein may be prepared using well known methods that result in a combination of antigen, immunostimulant and a suitable carrier or excipient. The compositions described herein may be administered as part of a sustained release formulation (i.e., a formulation such as a capsule, sponge or gel (composed of polysaccharides, for example) that effects a slow release of compound following administration). Such formulations may generally be prepared using well known technology (see, e.g., Coombes et al.,  Vaccine 74:1429-1438,  1996) and administered by, for example, oral,. rectal or subcutaneous implantation, or by implantation at the desired target site. Sustained-release formulations may contain a polypeptide, polynucleotide or antibody dispersed in a carrier matrix and/or contained within a reservoir surrounded by a rate controlling membrane.<br>
Carriers for use within such formulations are biocompatible, and may also be biodegradable; preferably the formulation provides a relatively constant level of active component release. Such carriers include microparticles of poly(lactide-co-glycolide), as well as polyacrylate, latex, starch, cellulose and dextran. Other delayed-release carriers include supramolecular biovectors, which comprise a non-liquid hydrophilic core {e.g., a cross-linked polysaccharide or oligosaccharide) and, optionally, an external layer comprising an amphiphilic compound, such as a phospholipid (see e.g., U.S. Patent No. 5,151,254 and PCT applications WO 94/20078, WO/94/23701 and WO 96/06638). The amount of active compound contained within a sustained release formulation depends upon the site of implantation, the rate and expected duration of release and the nature of the condition to be treated or prevented.<br>
Any of a variety of delivery vehicles may be employed within pharmaceutical compositions and vaccines to facilitate production of an antigen-specific immune response that targets Chlamydia-infected cells. Delivery vehicles include antigen presenting cells (APCs), such as dendritic cells, macrophages, B cells, monocytes and other cells that may be engineered to be efficient APCs. Such cells may, but need not, be genetically modified to increase the capacity for presenting the antigen, to improve activation and/or maintenance of the T cell response, to have anti-Chlamydia effects per se and/or to be immunologically compatible with the receiver (i.e., matched HLA haplotype). APCs may generally be isolated from any of a variety of biological fluids and organs, and may be autologous, allogeneic, syngeneic or xenogeneic cells.<br>
Certain preferred embodiments of the present invention use dendritic cells or progenitors thereof as antigen-presenting cells. Dendritic cells are highly potent APCs (Banchereau and Steinman, Nature 392:245-251, 1998) and have been shown to be effective as a physiological adjuvant for eliciting prophylactic or therapeutic immunity (see Timmerman and Levy, Ann. Rev. Med 50:507-529, 1999). In general, dendritic cells may be identified based on their typical shape (stellate in situ, with<br><br>
WO 02/08267<br><br>
PCT/US01/23121<br><br>
marked cytoplasmic processes (dendrites) visible in vitro), their ability to take up, process and present antigens with high efficiency, and their ability to activate nauїve T cell responses. Dendritic cells may, of course, be engineered to express specific cell-surface receptors or ligands that are not commonly found on dendritic cells in vivo or ex vivo, and such modified dendritic cells are contemplated by the present invention. As an alternative to dendritic cells, secreted vesicles antigen-loaded dendritic cells (called exosomes) may be used within a vaccine (see Zitvogel et al, Nature Med. 4:594-600, 1998).<br>
Dendritic cells and progenitors may be obtained from peripheral blood, bone marrow, lymph nodes, spleen, skin, umbilical cord blood or any other suitable tissue or fluid. For example, dendritic cells may be differentiated ex vivo by adding a combination of cytokines such as GM-CSF, IL-4, IL-13 and/or TNFa to cultures of monocytes harvested from peripheral blood. Alternatively, CD34 positive cells harvested from peripheral blood, umbilical cord blood or bone marrow may be differentiated into dendritic cells by adding to the culture medium combinations of GM-CSF, IL-3, TNFa, CD40 Iigand, LPS, flt3 ligand and/or other compound(s) that induce differentiation, maturation and proliferation of dendritic cells.<br>
Dendritic cells are conveniently categorized as "immature" and "mature" cells, which allows a simple way to discriminate between two well characterized phenotypes. However, this nomenclature should not be construed to exclude all possible intermediate stages of differentiation. Immature dendritic cells are characterized as APC with a high capacity for antigen uptake and processing, which correlates with the high expression of Fcγ receptor and mannose receptor. The mature phenotype is typically characterized by a lower expression of these markers, but a high expression of cell surface molecules responsible for T cell activation such as class I and class II MHC, adhesion molecules (e.g., CD54 and CD11) and costimulatory molecules (e.g., CD40, CD80, CD86 and 4-1BB).<br>
APCs may generally be transfected with a polynucleotide encoding a Chlamydial protein (or portion or other variant thereof) such that the Chlamydial polypeptide, or an immunogenic portion tnereof, is expressed on the cell surface. Such transfection may take place ex vivo, and a composition or vaccme comprising such transfected cells may then be used for therapeutic purposes, as described herein.<br><br>
Alternatively, a gene delivery vehicle that targets a dendritic or other antigen presenting cell may be administered to a patient, resulting in transfection that occurs in vivo. In vivo and ex vivo transfection of dendritic cells, for example, may generally be performed using any methods known in the art, such as those described in WO 97/24447, or the gene gun approach described by Mahvi et al., Immunology and cell Biology 75:456-460, 1997. Antigen loading of dendritic cells may be achieved by incubating dendritic cells or progenitor cells with the Chlamydial polypeptide, DNA (naked or within a plasmid vector) or RNA; or with antigen-expressing recombinant bacterium or viruses (e.g., vaccinia, fowlpox, adenovirus or lentivirus vectors). Prior to loading, the polypeptide may be covalently conjugated to an immunological partner that provides T cell help (e.g., a carrier molecule). Alternatively, a dendritic cell may be pulsed with a non-conjugated immunological partner, separately or in the presence of the polypeptide.<br>
Routes and frequency of administration of pharmaceutical compositions and vaccines, as well as dosage, will vary from individual to individual In general, the pharmaceutical compositions and vaccines may be administered by injection (e.g., intracutaneous, intramuscular, intravenous or subcutaneous), intranasally (e.g., by aspiration) or orally. Between 1 and 3 doses may be administered for a 1-36 week period. Preferably, 3 doses are administered, at intervals of 3-4 months, and booster vaccinations may be given periodically thereafter. Alternate protocols may be appropriate for individual patients. A suitable dose is an amount of polypeptide or DNA that, when administered as described above, is capable of raising an immune response in an immunized patient sufficient to protect the patient from Chlamydial infection for at least 1-2 years. In general, the amount of polypeptide present in a dose (or produced in situ by the DNA in a dose) ranges from about 1 pg to about 100 mg per kg of host, typically from about 10 pg to about 1 mg, and preferably from about 100 pg to about 1 μg. Suitable dose sizes will vary with the size of the patient, but will typically range from about 0.1 mL to about 5 mL.<br>
While any suitable carrier known to those of ordinary skill in the art may be employed in the pharmaceutical compositions of this invention, the type of carrier will vary depending on the mode of administiration. For parenteral administration, such as subcutaneous injection, the carrier preferably comprises water, saline, alcohol, a fat, a wax or a buffer. For oral administration, any of the above carriers or a solid carrier, such as mannitol, lactose, starch, magnesium stearate, sodium saccharine, talcum, cellulose, glucose, sucrose, and magnesium carbonate, may be employed. Biodegradable microspheres (e.g., polylactic galactide) may also be employed as carriers for the pharmaceutical compositions of this invention. Suitable biodegradable microspheres are disclosed, for example, in U.S. Patent Nos. 4,897,268 and 5,075,109.<br>
In general, an appropriate dosage and treatment regimen provides the active compound(s) in an amount sufficient to provide therapeutic and/or prophylactic benefit. Such a response can be monitored by establishing an improved clinical outcome in treated patients as compared to non-treated patients. Increases in preexisting immune responses to a Chlamydial protein generally correlate with an improved clinical outcome. Such immune responses may generally be evaluated using standard proliferation, cytotoxicity or cytokine assays, which may be performed using samples obtained from a patient before and after treatment.<br>
In another aspect, the present invention provides methods for using the polypeptides described above to diagnose Chlamydial infection. In this aspect, methods are provided for detecting Chlamydial infection in a biological sample, using one or more of the above polypeptides, either alone or in combination. For clarity, the term "polypeptide" will be used when describing specific embodiments of the inventive diagnostic methods. However, it will be clear to one of skill in the art that the fusion proteins of the present invention may also be employed in such methods.<br>
As used herein, a "biological sample" is any antibody-containing sample obtained from a patient. Preferably, the sample is whole blood, sputum, serum, plasma, saliva, cerebrospinal fluid or urine. More preferably, the sample is a blood, serum or plasma sample obtained from a patient. The polypeptides are used in an assay, as described below, to determine the presence or absence of antibodies to the polypeptide(s) in the sample, relative to a predetermined cut-off value. The presence of such antibodies indicates previous sensitization to Chlamydia antigens which may be indicative of Chlamydia-infection.<br>
In embodiments in which more than one polypeptide is employed, the polypeptides used are preferably complementary (i.e., one component polypeptide will tend to detect infection in samples where the infection would not be detected by another component polypeptide). Complementary polypeptides may generally be identified by <br>
using each polypeptide individually to evaluate serum samples obtained from a series of patients known to be infected with Chlamydia. After determining which samples test positive (as described below) with each polypeptide, combinations of two or more polypeptides may be formulated that are capable of detecting infection in most, or all, of the samples tested.<br>
A variety of assay formats are known to those of ordinary skill in the art for using one or more polypeptides to detect antibodies in a sample. See, e.g., Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988, which is incorporated herein by reference. In a preferred embodiment, the assay involves the use of polypeptide immobilized on a solid support to bind to and remove the antibody from the sample. The bound antibody may then be detected using a detection reagent that contains a reporter group. Suitable detection reagents include antibodies that bind to the antibody/polypeptide complex and free polypeptide labeled with a reporter group (e.g., in a semi-competitive assay). Alternatively, a competitive assay may be utilized, in which an antibody that binds to the polypeptide is labeled with a reporter group and allowed to bind to the immobilized antigen after incubation of the antigen with the sample. The extent to which components of the sample inhibit the binding of the labeled antibody to the polypeptide is indicative of the reactivity of the sample with the immobilized polypeptide.<br>
The solid support may be any solid material known to those of ordinary skill in the art to which the antigen may be attached. For example, the solid support may be a test well in a microtiter plate, or a nitrocellulose or other suitable membrane. Alternatively, the support may be a bead or disc, such as glass, fiberglass, latex or a plastic material such as polystyrene or polyvinylchloride. The support may also be a magnetic particle or a fiber optic sensor, such as those disclosed, for example, in U.S. Patent No. 5,359,681.<br>
The polypeptides may be bound to the solid support using a variety of techniques known to those of ordmaiy skill in the art. In the context of the present invention, the term "bound" refers to both noncovalent association, such as adsorption, and covalent attachment (which may be a direct linkage between the antigen and functional groups on the support or may be a linkage by way of a cross-linking agent). Binding by adsorption to a well in a microtiter plate or to a membrane is preferred. In such cases, adsorption may be achieved by contacting the polypeptide, in a suitable buffer, with the solid support for a suitable amount of time. The contact time varies with temperature, but is typically between about 1 hour and 1 day. In general, contacting a well of a plastic microtiter plate (such as polystyrene or polyvinylchloride) with an amount of polypeptide ranging from about 10 ng to about 1 μg, and preferably about 100 ng, is sufficient to bind an adequate amount of antigen.<br>
Cbvafent attachment of polypeptide to a solid support may generally be achieved by first reacting the support with a bifunctional reagent that will react with both the support and a functional group, such as a hydroxyl or amino group, on the polypeptide. For example, the polypeptide may be bound to supports having an appropriate polymer coating using benzoquinone or by condensation of an aldehyde group on the support with an amine and an active hydrogen on the polypeptide {see, e.g., Pierce Immunotechnology Catalog and Handbook, 1991, at A12-A13).<br>
In certain embodiments, the assay is an enzyme linked immunosorbent assay (ELISA). This assay may be performed by first contacting a polypeptide antigen that has been immobilized on a solid support, commonly the well of a microtiter plate, with the sample, such that antibodies to the polypeptide within the sample are allowed to bind to the immobilized polypeptide. Unbound sample is then removed from the immobilized polypeptide and a detection reagent capable of binding to the immobilized antibody-polypeptide complex is added. The amount of detection reagent that remains bound to the solid support is then determined using a method appropriate for the specific detection reagent.<br>
More specifically, once the polypeptide is immobilized on the support as described above, the remaining protein binding sites on the support are typically blocked. Any suitable blocking agent known to those of ordinary skill in the art, such as bovine serum albumin (BSA) or Tween 20TM (Sigma Chemical Co., St. Louis, MO) may be employed. The immobilized polypeptide is then incubated with the sample, and antibody is allowed to bind to the antigen. The sample may be diluted with a suitable difutent, such as phosphate-buffered safine (PBS) prior to incubation. In general, an appropriate contact time (i.e., incubation time) is that period of time that is sufficient to detect the presence of antibody within an HGE-infected sample. Preferably, the contact time is sufficient to achieve a level of binding that is at least 95% of that achieved at equilibrium between bound and unbound antibody. Those of ordinary skill in the art will recognize that the time necessary to achieve equilibrium may be readily determined by assaying the level of binding that occurs over a period of time. At room temperature, an incubation time of about 30 minutes is generally sufficient.<br>
Unbound sample may then be removed by washing the solid support with an appropriate buffer, such as PBS containing 0.1% Tween 20™. Detection reagent may then be added to the solid support. An appropriate detection reagent is any compound that binds to the immobilized antibody-polypeptide complex and that can be detected by any of a variety of means known to those in the art. Preferably, the detection reagent contains a binding agent (such as, for example, Protein A, Protein G, immunoglobulin, lectin or free antigen) conjugated to a reporter group. Preferred reporter groups include enzymes (such as horseradish peroxidase), substrates, cofactors, inhibitors, dyes, radionuclides, luminescent groups, fluorescent groups and biotin. The conjugation of binding agent to reporter group may be achieved using standard methods known to those of ordinary skill in the art. Common binding agents may also be purchased conjugated to a variety of reporter groups from many commercial sources (e.g., Zymed Laboratories, San Francisco, CA, and Pierce, Rockford, IL).<br>
The detection reagent is then incubated with the immobilized antibody-polypeptide complex for an amount of time sufficient to detect the bound antibody. An appropriate amount of time may generally be determined from the manufacturer's instructions or by assaying the level of binding that occurs over a period of time. Unbound detection reagent is then removed and bound detection reagent is detected using the reporter group. The method employed for detecting the reporter group depends upon the nature of the reporter group. For radioactive groups, scintillation counting or autoradiographic methods are generally appropriate. Spectroscopic methods may be used to detect dyes, luminescent groups and fluorescent groups. Biotin may be detected using avidin, coupled to a different reporter group (commonly a radioactive or fluorescent group or an enzyme). Enzyme reporter groups may generally be detected by the addition of substrate (generally for a specific period of time), followed by spectroscopic or other analysis of the reaction products,<br>
To determine the presence or absence of anti-Chlamydia antibodies in the sample, the signal detected from the reporter group that remams bound to the solid <br>
support is generally compared to a signal that corresponds to a predetermined cut-off value. In one preferred embodiment, the cut-off value is the average mean signal obtained when the immobilized antigen is incubated with samples from an uninfected patient. In general, a sample generating a signal that is three standard deviations above the predetermined cut-off value is considered positive for Chlamydia-infection. In an alternate preferred embodiment, the cut-off value is determined using a Receiver Operator Curve, according to the method of Sackett et al., Clinical Epidemiology: A Basic Science for Clinical Medicine, Little Brown and Co., 1985, pp. 106-107. Briefly, in this embodiment, the cut-off value may be determined from a plot of pairs of true positive rates (i.e., sensitivity) and false positive rates (100%-specificity) that correspond to each possible cut-off value for the diagnostic test result. The cut-off value on the plot that is the closest to the upper left-hand corner (i.e., the value that encloses the largest area) is the most accurate cut-off value, and a sample generating a signal that is higher than the cut-off value determined by this method may be considered positive. Alternatively, the cut-off value may be shifted to the left along the plot, to minimize the false positive rate, or to the right, to minimize the false negative rate. In general, a sample generating a signal that is higher than the cut-off value determined by this method is considered positive for Chlamydial infection.<br>
In a related embodiment, the assay is performed in a rapid flow-through or strip test format, wherein the antigen is immobilized on a membrane, such as nitrocellulose. In the flow-through test, antibodies within the sample bind to the immobilized polypeptide as the sample passes through the membrane. A detection reagent (e.g., protein A-colloidal gold) then binds to the antibody-polypeptide complex as the solution containing the detection reagent flows through the membrane. The detection of bound detection reagent may then be performed as described above. In the strip test format, one end of the membrane to which polypeptide is bound is immersed in a solution containing the sample. The sample migrates along the membrane through a region containing detection reagent and to the area of immobilized polypeptide. Concentration of detection reagent at the polypeptide indicates the presence of anti-Chlamydia antibodies in the sample. Typically, the concentration of detection reagent at that site generates a pattern, such as a linte, that can be read visually. The absence of such a pattern indicates a negative result. In general, the amount of polypeptide immobilized on the membrane is selected to generate a visually discernible pattern when the biological sample contains a level of antibodies that would be sufficient to generate a positive signal in an ELISA, as discussed above. Preferably, the amount of polypeptide immobilized on the membrane ranges from about 25 ng to about 1 μ,g, and more preferably from about 50 ng to about 500 ng. Such tests can typically be performed with a very small amount (e.g., one drop) of patient serum or blood.<br>
Of course, numerous other assay protocols exist that are suitable for use with the polypeptides of the present invention. The above descriptions are intended to be exemplary only. One example of an alternative assay protocol which may be usefully employed in such methods is a Western blot, wherein the proteins present in a biological sample are separated on a gel, prior to exposure to a binding agent. Such techniques are well known to those of skill in the art.<br>
The present invention further provides agents, such as antibodies and antigen-binding fragments thereof, that specifically bind to a Chlamydial protein. As used herein, an antibody, or antigen-binding fragment thereof, is said to "specifically bind" to a Chlamydial protein if it reacts at a detectable level (within, for example, an ELISA) with a Chlamydial protein, and does not react detectably with unrelated proteins under similar conditions. As used herein, "binding" refers to a noncovalent association between two separate molecules such that a complex is formed. The ability to bind may be evaluated by, for example, determining a binding constant for the formation of the complex. The binding constant is the value obtained when the concentration of the complex is divided by the product of the component concentrations. In general, two compounds are said to "bind," in the context of the present invention, when the binding constant for complex formation exceeds about 103 L/mol. The binding constant may be determined using methods well known in the art.<br>
Binding agents may be further capable of differentiating between patients with and without a Chlamydial infection using the representative assays provided herein. In other words, antibodies or other binding agents that bind to a Chlamydial protein will generate a signal indicating the presence of a Chlamydial infection in at least about 20% of patients with the disease, and will generate a negative signal indicating the absence of the disease in at least about 90% of individuals without infection. To determine whether a binding agent satisfies this requirement, biological samples (e.g., blood, sera, sputum urine and/or tissue biopsies ) frorn patients with and without Chlamydial infection (as determined using standard clinical tests) may be assayed as described herein for the presence of polypeptides that bind to the binding agent. It will be apparent that a statistically significant number of samples with and without the disease should be assayed. Each binding agent should satisfy the above criteria; however, those of ordinary skill in the art will recognize that binding agents may be used in combination to improve sensitivity.<br>
Any agent that satisfies the above requirements may be a binding agent. For example, a binding agent may be a ribosome, with or without a peptide component, an RNA molecule or a polypeptide. In a preferred embodiment, a binding agent is an antibody or an antigen-binding fragment thereof. Antibodies may be prepared by any of a variety of techniques known to those of ordinary skill in the art. See, e.g., Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988. In general, antibodies can be produced by cell culture techniques, including the generation of monoclonal antibodies as described herein, or via transfection of antibody genes into suitable bacterial or mammalian cell hosts, in order to allow for the production of recombinant antibodies. In one technique, an immunogen comprising the polypeptide is initially injected into any of a wide variety of mammals (e.g., mice, rats, rabbits, sheep or goats). In this step, the polypeptides of this invention may serve as the immunogen without modification. Alternatively, particularly for relatively short polypeptides, a superior immune response may be elicited if the polypeptide is joined to a carrier protein, such as bovine serum albumin or keyhole limpet hemocyanin. The immunogen is injected into the animal host, preferably according to a predetermined schedule incorporating one or more booster immunizations, and the animals are bled periodically. Polyclonal antibodies specific for the polypeptide may then be purified from such antisera by, for example, affinity chromatography using the polypeptide coupled to a suitable solid support.<br>
Monoclonal antibodies specific for an antigenic polypeptide of interest may be prepared, for example, using the technique of Kohler and Milstein, Eur. J. Immunol 6:511-519, 1976, and improvements thereto. Briefly, these methods involve the preparation of immortal cell lines capable of producing antibodies having the desired specificity (i.e., reactivity with the polypeptide of interest). Such cell lines may be produced, for example, from spleen cells obtained from an animal immunized as described above. The spleen cells are then immortalized by, for example, fusion with a myeloma cell fusion partner, preferably one that is syngeneic with the immunized animal. A variety of fusion techniques may be employed. For example, the spleen cells and myeloma cells may be combined with a nonionic detergent for a few minutes and then plated at low density on a selective medium that supports the growth of hybrid cells, but not myeloma cells. A prefen'ed selection technique uses HAT (hypoxanthine, aminopterin, thymidine) selection. After a sufficient time, usually about 1 to 2 weeks, colonies of hybrids are observed. Single colonies are selected and their culture supernatants tested for binding activity against the polypeptide. Hybridomas having high reactivity and specificity are preferred.<br>
Monoclonal antibodies may be isolated from the supernatants of growing hybridoma colonies. In addition, various techniques may be employed to enhance the yield, such as injection of the hybridoma cell line into the peritoneal cavity of a suitable vertebrate host, such as a mouse. Monocloqal antibodies may then be harvested from the ascites fluid or the blood. Contaminants may be removed from the antibodies by conventional techniques, such as chromatography, gel filtration, precipitation, and extraction. The polypeptides of this invention may be used in the purification process in, for example, an affinity chromatography step.<br>
Within certain embodiments, the use of antigen-binding fragments of antibodies may be preferred. Such fragments include Fab fragments, which may be prepared using standard techniques. Briefly!, immunoglobulins may be purified from rabbit serum by affinity chromatography on Protein A bead columns (Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988) and digested by papain to yield Fab and Fc fragments. The Fab and Fc fragments may be separated by affinity chromatography on protein A bead columns.<br>
Monoclonal antibodies of the present invention may be coupled to one or more therapeutic agents. Suitable agents in this regard include radionuclides, ' differentiation inducers, drugs, toxins, and derivatives thereof. Preferred radionuclides include 90Y, 123I J25I, I31,I, I86Re, 188Re, 211At, and 212Bi. Preferred drugs include methotrexate, and pyrimidine and purine analogs. Preferred differentiation inducers include phorbol esters and butyric acid.  Preferred toxins include ricin, abrin, diphtheria toxin, cholera toxin, gelonin, Pseudomonas exotoxin, Shigella toxin,, and pokeweed antiviral protein.<br>
A therapeutic agent may be coupled (e.g., covalently bonded) to a suitable monoclonal antibody either directly or indirectly (e.g., via a linker group). A direct reaction between an agent and an antibody is possible when each possesses a substituent capable of reacting with the other. For example, a nucleophilic group, such as an amino or sulfhydryl group, on one may be capable of reacting with a carbonyl-containing group, such as an anhydride or an acid halide, or with an alkyl group containing a good leaving group (e.g., a halide) on the other.<br>
Alternatively, it may be desirable to couple a therapeutic agent and an antibody via a linker group. A linker group can function as a spacer to distance an antibody from an agent in order to avoid interference with binding capabilities. A linker group can also serve to increase the chemical reactivity of a substituent on an agent or an antibody, and thus increase the coupling efficiency. An increase in chemical reactivity may also facilitate the use of agents, or functional groups on agents, which otherwise would not be possible.<br>
It will be evident to those skilled in the art that a variety of bifunctional or polyfunctional reagents, both homo- and hetero-functional (such as those described in the catalog of the Pierce Chemical Co., Rockford, IL), may be employed as the linker group. Coupling may be effected, for example, through amino groups, carboxyl groups, sulfhydryl groups or oxidized carbohydrate residues. There are numerous references describing such methodology, e.g., U.S. Patent No. 4,671,958, to Rodwell et al.<br>
Where a therapeutic agent is more potent when free from the antibody portion of the immunoconjugates of the present invention, it may be desirable to use a linker group which is cleavable during or upon internalization into a cell. A number of different cleavable linker groups have been described. The mechanisms for the intracellular release of an agent from these linker groups include cleavage by reduction of a disulfide bond (e.g., U.S. Patent No. 4,489,710, to Spitler), by irradiation of a photolahile bond (e.g., U.S. Patent No. 4,625,014, to Senter et al.), by hydrolysis of derivatized amino acid side chains (e.g., U.S. Patent No. 4,638,045, to Kohn et al.), by serum, complement-mediated hydrolysis (e.g., U.S. Patent No. 4,671,958, to Rodwell et al.), and acid-catalyzed hydrolysis (e.g., U.S. Patent No. 4,569,789, to Blattler et al.).<br><br>
It may be desirable to couple more than one agent to an antibody. In one embodiment, multiple molecules of an agent: are coupled to one antibody molecule. In another embodiment, more than one type of agent may be coupled to one antibody. Regardless of the particular embodiment, irnmunoconjugates with more than one agent may be prepared in a variety of ways. Fqr example, more man one agent may be coupled directly to an antibody molecule, or linkers which provide multiple sites for attachment can be used. Alternatively, a carrier can be used.<br>
A carrier may bear the agents in a variety of ways, including covalent bonding either directly or via a linker group. Suitable carriers include proteins such as albumins (e.g., U.S. Patent No. 4,507,234, to; Kato et al.), peptides and polysaccharides such as arninodextran (e.g., U.S. Patent No. 4,699,784, to Shih et al.). A carrier may also bear an agent by noncovalent bonding or by encapsulation,, such as within a liposome vesicle (e.g., U.S. Patent Nos. 4,429,008 and 4,873,088). Carriers specific for radionuclide agents include radiohalogenated small molecules and chelating compounds. For example, U.S. Patent No. 4,735,792 discloses representative radiohalogenated small molecules and their synthesis. A radionuclide chelate may be formed from chelating compounds that include those containing nitrogen and sulfur atoms as the donor atoms for binding the metal, or metal oxide, radionuclide. For example, U.S. Patent No. 4,673,562, to Davison et al. discloses representative chelating compounds and their synthesis.<br>
A variety of routes of administration for the antibodies and irnmunoconjugates may be used. Typically, administration will be intravenous, intramuscular, subcutaneous or in site-specific regions by appropriate methods. It will be evident that the precise dose of the antibody/immunoconjugate will vary depending upon the antibody used, the antigen density, and the rate of clearance of the antibody.<br>
Antibodies may be used in diagnostic tests to detect the presence of Chlamydia antigens using assays similar to those detailed above and other techniques well known to those of skill in the art, thereby providing a method for detecting Chlamydial infection in a patient.<br>
Diagnostic reagents of the present invention may also comprise DNA sequences encoding one or more of the above polypeptides, or one or more portions thereof.   For example, at least two oligomicleotide primers may be employed in a polymerase chain reaction (PCR) based assay to amplify Chla)7tydia-specific cDNA derived from a biological sample, wherein at least one of the oligonucleotide primers is specific for a DNA molecule encoding a polypeptide of the present invention. The presence of the amplified cDNA is then detected using techniques well known in the art, such as gel electrophoresis. Similarly, oligonucleotide probes specific for a DNA molecule encoding a polypeptide of the present invention may be used in a hybridization assay to detect the presence of an inventive polypeptide in a biological sample.<br>
As used herein, the term "oligonucleotide primer/probe specific for a DNA molecule" means an oligonucleotide sequence that has at least about 80%, preferably at least about 90% and more preferably at least about 95%, identity to the DNA molecule in question. Oligonucleotide primers and/or probes which may be usefully employed in the inventive diagnostic methods preferably have at least about 10-40 nucleotides. In a preferred embodiment, the oligonucleotide primers comprise at least about 10 contiguous nucleotides of a DNA molecule encoding one of the polypeptides disclosed herein. Preferably, oligonucleotide probes for use in the inventive diagnostic methods comprise at least about 15 contiguous oligonucleotides of a DNA molecule encoding one of the polypeptides disclosed herein. Techniques for both PCR based assays and hybridization assays are well known in the art (see, for example, Mullis et al. Ibid; Ehrlich, Ibid). Primers or probes may thus be used to detect Chlamydia-specific sequences in biological samples. DNA probes or primers comprising oligonucleotide sequences described above may be used alone or in combination with each other.<br>
The following Examples are ioffered by way of illustration and not by way of limitation.<br>
EXAMPLE 1 ISOLATION OF DNA SEQUENCES ENCODING CHLAMYDIA ANTIGENS<br>
Chlamydia antigens of the present invention were isolated by expression cloning of a genomic DNA library of Chlamydia trachomatis LGV II essentially as described by Sanderson et al. (J. Exp. Med., l995, 182:1751-1757) and were shown to induce PBMC proliferation and IFN-y in an immunoreactive T cell line.<br>
A Chlamydia-specific T cell pine was generated by stimulating PBMCs from a normal donor with no history of* chlamydial genital tract infection with elementary bodies of Chlamydia trachomatis LGV II. This T cell line, referred to as TCL-8, was found to recognize both Chlamydia trachomatis and Chlamydia pneumonia infected monocyte-derived dendritic cells.<br>
A randomly sheared genomic library of Chlamydia trachomatis LGV II was constructed in Lambda ZAP (Stratagerite, La Jolla, CA) and the amplified library plated out in 96 well microtiter plates at a density of 30 clones/well. Bacteria were induced to express recombinant protein in the presence of 2 mM IPTG for 3 h, then pelleted and resuspended in 200 μl of RPMI 10% FBS. 10 μl of the induced bacterial suspension was transferred to 96 well plateis containing autologous monocyte-derived dendritic cells. After a 2 h incubation, dendritic cells were washed to remove free E. coli and Chlamydia-specific T cells were added. Positive E. coli pools were identified by determining IFN-γ production and proliferation of the T cells in response to the pools.<br>
Four positive pools were identified, which were broken down to yield four pure clones (referred to as 1-B1-66, 4-D7-28, 3-G3-10 and 10-C10-31), with insert sizes of 481 bp, 183 bp, 110 bp and 1400 bp, respectively. The determined DNA sequences for 1-B1-66, 4-D7-28, 3-G3-10 and 10-C10-31 are provided in SEQ ID NO: 1-4, respectively. Clone 1-B1-66 is approximately in region 536690 of the C trachomatis genome (NCBI C. trachomatis database). Within clone 1-B1-66, an open reading frame (ORF) has been identified (nucleotides 115 - 375) that encodes a previously identified 9 kDa protein (Stephens, et al. Genbank Accession No. AE001320), the sequence of which is provided in SEQ ED NO: 5). Clone 4-D7-28 is a smaller region of the same ORF (amino acids 22-82 of 1-B1-66). Clone 3-G3-10 is approximately in region 74559 of the C. trachomatis genome. The insert is cloned in the antisense orientation with respect to its orientation in the genome. The clone 10-C10-31 contains an open reading frame that corresponds to a previously published sequence for S13 ribosomal protein from (Chlamydia trachomatis (Gu, L. et al. J. Bacteriology, 177:2594-2601, 1995). The predicted protein sequences for 4-D7-28 and 10-C10-31 are provided in SEQ ID NO: 6 and 12, respectively. Predicted protein sequences for 3-G3-10 are provided in SEQ ID NO: 7-11.<br>
In a related series of screening studies, an additional T cell line was used to screen the genomic DNA library of Chlamydia trachomatis LGV II described above. A Chlamydia-specific T cell line (TCT-1) was derived from a patient with a chlamydial genital tract infection by stimulating patient PBMC with autologous monocyte-derived dendritic cells infected with elementary bodies of Chlamydia trachomatis LGV II. One clone, 4C9-18 (SEQ ID NO: 21), containing a 1256 bp insert, elicited a specific immune response, as measured by standard proliferation assays, from the Chlamydia-specific T cell line TCT-1. Subsequent analysis revealed this clone to contain three known sequences: lipoamide dehydrogenase (Genbank Accession No. AE001326), disclosed in SEQ ID NO: 22; a hypothetical protein CT429 (Genbank Accession No. AE001316), disclosed in SEQ ID NO: 23; and part of an open reading frame of ubiquinone methyltransferase CT428 (Genbank Acce----n No. AE001316), disclosed in SEQ ID NO: 24.<br>
In further studies involving clone 4C9-18 (SEQ ID NO: 21), the full-length amino acid sequence for lipoamide dehydrognase (SEQ ID NO: 22) from C. trachomatis (LGV II) was expressed in clone CtL2-LPDA-FL, as disclosed in SEQ ID NO: 90.<br>
To further characterize the open reading frame containing the T cell stimulating epitope(s), a cDNA fragment containing nucleotides 1-695 of clone 4C9-18 with a cDNA sequence encoding a 6X-Histidine tag on the amino terminus was subcloned into the NdeI/EcoRI site of the pET17b vector (Novagen, Madison, WI), referred to as clone 4C9-18#2 BL21 pLysS (SEQ ID NO: 25, with the corresponding amino acid sequence provided in SEQ ID; NO: 26) and transformed into E. coli. Selective induction of the transformed E. coli with 2 mM IPTG for three hours resulted in the expression of a 26 kDa protein from clone 4C9-18#2 BL21 pLysS, as evidenced by standard Coomassie-stained SDS-PAGE., To determine the immunogenicity of the protein encoded by clone 4C9-18#2 BL21 pLysS, E. coli expressing the 26 kDa protein were titered onto 1 x 104 monocyte-derived djendritic cells and incubated for two hours. The dendritic cell cultures were washed arid 2.5 x 104 T cells (TCT-1) added and allowed to incubate for an additional 72 hours, at which time the level of IFN-y in the culture supernatant was determined by ELISA. As shown in Fig. 1, the T-cell line TCT-1 was found to respond to induced cultures as measured by IFN-g, indicating a<br>
Chlamydia-specific T-cell response against the lipoamide dehydrogenase sequence. Similarly, the protein encoded by clone 4C9-18#2 BL21 pLysS was shown to stimulate the TCT-1 T-cell line by standard proliferation assays.<br>
Subsequent studies to identify additional Chlamydia trachomatis antigens using the above-described CD4+ T-cell expression cloning technique yielded additional clones. The TCT-1 and TCL-8 Chlamydia-specific T-cell lines, as well as the TCP-21 T-cell line were utilized to screen the Chlamydia trachomatis LGVII genomic library. The TCP-21 T-cell line was derived from a patient having a humoral immune response to Chlamydiapnuemoniae. The TCT-1 cell line identified 37 positive pools, the TCT-3 cell line identified 41 positive pools and the TCP-21 cell line identified 2 positive pools. The following clones were derived from 10 of these positive pools. Clone 11-A3-93 (SEQ ID NO: 64), identified by the TCP-21 cell line, is a 1339 bp genomic fragment sharing homology to the HAD superfamily (CT103). The second insert in the same clone shares homology with the fab I gene (CT104) present on the complementary strand. Clone 11-C12-91 (SEQ ID NO: 63), identified using the TCP-21 cell line, has a 269 bp insert that is part of the OMP2 gene (CT443) and shares homology with the 60 kDa cysteine rich outer membrane protein of C. pnuemoniae.<br>
Clone 11-G10-46, (SEQ ID NO: 62), identified using the TCT-3 cell line, contains a 688 bp insert that shares homology to the hypothetical protein CT610. Clone 11-G1-34, (SEQ ID NO: 61), identified using the TCT-3 cell line, has two partial open reading frames (ORF) with an insert size of 1215 bp. One ORF shares homology to the malate dehydrogenase gene (CT376), and the other ORF shares homology to the glycogen hydrolase gene (CT042). Clone 11-H3-68, (SEQ ID NO: 60), identified using the TCT-3 cell line, has two ORFs with a total insert size of 1180 bp. One partial ORF encodes the plasmid-encoded PGP6-D virulence protein while the second ORF is a complete ORF for the L1 ribosomal gene (CT318). Clone 11-H4-28, (SEQ ID NO: 59), identified using the TCT-3 cell line, has an insert size of 552 bp and is part of the ORF for the dnaK gene (CT396). Clone 12-B3-95, (SEQ ID NO: 58), identified using the TCT-1 cell line, has an insert size of 463 bp and is a part of the ORF for for the lipoamide dehydrogenase gene (CT557). Clones 15-G1-89 and 12-B3-95 are identical, (SEQ ID NO: 55 and 58, respectively), identified using the TCT-1 cell line, has an insert size of 463 bp and is part of the ORF for the lipoamide dehydrogenase gene (CT557). Clone 12-G3-83, (SEQ ID NO: 57), identified using the TCT-1 cell line, has an insert size of 1537 bp and has part of the ORF for the hypothetical protein CT622.<br>
Clone 23-G7-68, (SEQ ID NO: 79), identified using the TCT-3 cell line,<br>
contains a 950 bp insert and contains a small part of the L11 ribosomal ORE, the entire<br>
ORP for L1 ribosomal protein and a part of the ORF for L10 ribosomal protein. In<br>
addition, this clone also identified the patient lines CT4, CT5, CT11, CT12, and<br>
CHH037.   Clone 22-F8-91, (SEQ ID NO: 80), identified using the TCT-1 cell line,<br>
contains a 395 bp insert that contains a part of the pmpC ORF on the complementary<br>
strand of the clone. Clone 21-E8-95, (SEQ ID NO: 81), identified using the TCT-3 cell<br>
line, contains a 2,085 bp insert which contains part of CT613 ORF, the complete ORF<br>
for CT612, the complete ORF for CT611 and part of the ORF for CT610. Clone 19-<br>
F12-57, (SEQ ID NO: 82), identified using the TCT-3 cell line, contains a 405 bp insert<br>
which contains part of the CT 858 ORF and a small part of the recA ORF. Clone 19-<br>
F12-53, (SEQ ID NO: 83), identified using the TCT-3 cell line, contains a 379 bp insert<br>
that is part of the ORF for CT455 encoding glutamyl tRNA synthetase. Clone 19-A5-<br>
54, (SEQ ID NO: 84), identified using the TCT-3 cell line, contains a 715 bp insert that<br>
is part of the ORF3 (complementary strand of the clone) of the cryptic plasmid. Clone<br>
17-E11-72, (SEQ ID NO: 85), identified using the TCT-1 cell line, contains a 476 bp<br>
insert that is part of the ORF for Opp__2 and pmpD. The pmpD region of this clone is<br>
covered by the pmpD region of clone 15-H2-76.  Clone 17-C1-77, (SEQ ID NO: 86),<br>
identified using the the patient cell lines CT3, CT1, CT4, and CT12, contains a 1551 bp<br>
insert that is part of the CT857 ORF, as well as part of the CT858 ORF. Clone 15-H2-<br>
76, (SEQ ID NO: 87), identified using the TCT-1 cell line, contains a 3,031 bp insert<br>
that contains a large part of the pmpD ORF, part of the CT089 ORF, as well as part of<br>
the ORF for SycE. Clone 15-A3-26, (SEQ ID NO: 88), contains a 976 bp insert that<br>
contains part of the ORF for CT858.  Clone 17-G4-36, (SEQ TD NO: 267), identified<br>
using the patient lines CL8, TCT-10, CT1, CT5, CT13, and CHH037, contains a 680 bp<br>
insert that is in frame with beta-gal in the plasmid and shares homology to part of the<br>
ORF for DNA-directed RNA polymerase beta subunit (CT315 in SerD).<br>
Several of the clones described above share homology to various polymorphic membrane proteins. The genomic sequence of Chlamydia trachomatis contains a family of nine polymorphic membrane protein genes, referred to as pmp. <br>
		These genes are designated pmpA, pmpB, pmpC, pmpD, pmpE, pmpF, pmpG, pmpH and pmpl. Proteins expressed from these genes are believed to be of biological relevance in generating a protective immune response to a Chlamydial infection. In particular, pmpC, pmpD, pmpE and pmpI contain predictable signal peptides, suggesting they are outer membrane proteins, and therefore, potential immunological targets.<br>
Based on the Chlamydia trachomatis LGVII serovar sequence, primer pairs were designed to PCR amplify the full-length fragments of pmpC, pmpD, pmpE, pmpG, pmpH and pmpL The resulting fragments were subcloned into the DNA vaccine vector JA4304 or JAL, which is JA4304 with a modified linker (SmithKline Beecham, London, England). Specifically, PmpC was subcloned into the JAL vector using the 5' oligo GAT AGG CGC GCC GCA ATC ATG AAA TTT ATG TCA GCT ACT GCT G and the 3' oligo CAG AAC GCG TTT AGA ATG TCA TAC GAG CAC CGC A, as provided in SEQ ID NO: 197 and 198, respectively. PCR amplification of the gene under conditions well known in the art and ligation into the 5' ASCI/3' Mlul sites of the JAL vector was completed after inserting the short nucleotide sequence GCAATC (SEQ ID NO: 199) upstream of the ATG to create a Kozak-like sequence. The resulting expression vector contained the full-length pmpC gene comprising 5325 nucleotides (SEQ ID NO: 173) containing the hypothetical signal sequence, which encodes a 187 kD protein (SEQ ID NO: 179). The pmpD gene was subcloned into the JA4304 vaccine vector following PCR amplification of the gene using the following oligos: 5' oligo-TGC AAT CAT GAG TTC GCA GAA AGA TAT AAA AAG C (SEQ ID NO: 200) and 3' oligo- CAG AGC TAG CTT AAA AGA TCA ATC GCA ATC CAG TAT TC (SEQ ID NO: 201). The gene was ligated into the a 5' blunted HIII/3' Mlul site of the JA4304 vaccine vector using standard techniques well known in the art. The CAATC (SEQ ID NO: 202) was inserted upstream of the ATG to create a Kozak-like sequence. This clone is unique in that the last threonine of the HindIII site is missing due to the blunting procedure, as is the last glycine of the Kozak-like sequence. The insert, a 4593 nucleotide fragment (SEQ ID NO: 172) is the full-length gene for pmpD containing the hypothetical signal sequence, which encodes a 161 kD protein (SEQ ID NO: 178). PmpE was subcloned into the JA4304 vector using the 5' oligo- TGC AAT CAT GAA AAA AGC GTT TTT CTT TTT C (SEQ ID NO: 203), and the 3' oligo- CAG AAC<br><br>
WO 02/08267<br><br>
PCT/US01/23121<br><br>
GCG TCT AGA ATC GCA GAG CAA TTT C (SEQ ID NO: 204).  Following PCR amplification, the gene was ligated into the 5' blunted HIII/3' MluI site of JA4304. To facilitate this, a short nucleotide sequence, TGCAATC (SEQ ID NO: 293), was added upstream of the initiation codon for creating a Kozak-Iike sequence and reconstituting the HindIII site. The insert is the full-length pmpE gene (SEQ ID NO: 171) containing the hypothetical signal sequence.   The pmpE gene encodes a 105 kD protein (SEQ ID NO: 177).  The pmpG gene was PCR amplified using the 5' oligo- GTG CAA TCA TGA TTC CTC AAG GAA TTT ACG ( SEQ ID NO: 205), and the 3' oligo- CAG AAC GCG TTT AGA ACC GGA CTT TAC TTC C (SEQ ID NO: 206) and subcloned into the JA4304 vector. Similar cloning strategies were followed for the pmpl and pmpK genes. In addition, primer pairs were designed to PCR amplify the full-length or overlapping fragments of the pmp genes, which were then subcloned for protein expression in the pET17b vector (Novagen, Madison, WI) and transfected into E. coli BL21 pLysS for expression and subsequent purification utilizing the histidine-nickel chromatographic methodology provided by Novagen. Several of the genes encoding the recombinant proteins, as described below, lack the native signal sequence to facilitate expression of the protein.  Full-length protein expression of pmpC was accomplished through expression of two overlapping fragments, representing the amino and carboxy termini.    Subcloning of the pmpC-amino terminal portion, which lacks the signal sequence, (SEQ ID NO: 187, with the corresponding amino acid sequence provided in SEQ ID NO: 195) used the 5' oligo- CAG ACA TAT GCA TCA CCA TCA CCA TCA CGA GGC GAG CTC GAT CCA AGA TC (SEQ ID NO: 207), and the 3' oligo- CAG AGG TAC CTC AGA TAG CAC TCT CTC CTA TTA AAG TAG G (SEQ ID NO: 208) into the 5' NdeI/3' KPN cloning site of the vector. The carboxy terminus portion of the   gene,   pmpC-carboxy  terminal  fragment   (SEQ   ID  NO:   186,  with  the corresponding amino acid sequence provided in SEQ ID NO: 194), was subcloned into the 5' NheI/3' KPN cloning site of the expression vector using the following primers: 5' oligo- CAG AGC TAG CAT GCA TCA CCA TCA CCA TCA CGT TAA GAT TGA GAA CTT CTC TGG C (SEQ ID NO: 209), and 3' oligo- CAG AGG TAC CTT AGA ATG TCA TAC GAG CAC CGC AG (SEQ ID NO: 210).   PmpD was also expressed as two overlapping proteins. The pmpD-amino terminal portion, which lacks the signal sequence, (SEQ ID NO: 185, with the corresponding amino acid sequence provided in<br>
SEQ ID NO: 193) contains the initiating codon of the pET17b and is expressed as a 80 kD protein. For protein expression and purification purposes, a six-histidine tag follows the initiation codon and is fused at the 28th amino acid (nucleotide 84) of the gene. The following primers were used, 5' oligo, CAG ACA TAT GCA TCA CCA TCA CCA TCA CGG GTT AGC (SEQ ID NO: 211), and the 3' oligo- CAG AGG TAC CTC AGC TCC TCC AGC ACA CTC TCT TC (SEQ ID NO: 212), to splice into the 5' NdeI/3' KPN cloning site of the vector. The pmpD-carboxy terminus portion (SEQ ID NO: 184) was expressed as a 92 kD protein (SEQ ID NO: 192).  For expression and subsequent purification, an additional methionine, alanine and serine was included, which represent the initiation codon and the first two amino acids from the pETl7b vector. A six-histidine tag downstream of the methionine, alanine and serine is fused at the 691st amino acid (nucleotide 2073) of the gene.   The 5' oligo- CAG AGC TAG CCA TCA CCA TCA CCA TCA CGG TGC TAT TTC TTG CTT ACG TGG (SEQ ID NO: 213) and the 3' oligo- CAG AGG TAC TTn AAA AGA TCA ATC GCA ATC CAG TAT TCG (SEQ ID NO: 214) were used to subclone the insert into the 5' NheI/3' KPN cloning site of the expression vector. PmpE was expressed as a 106kD protein (SEQ ID NO: 183 with the corresponding amino acid sequence provided in SEQ ID NO: 191). The pmpE insert also lacks the native signal sequence. PCR amplification of the gene under conditions well known in the art was performed using the following oligo primers: 5' oligo- CAG AGG ATC CAC ATC ACC ATC ACC ATC ACG GAC TAG CTA GAG AGG TTC (SEQ ID NO: 215), and the 3' oligo- CAG AGA ATT CCT AGA ATC GCA GAG CAA TTT C (SEQ ID NO: 216), and the amplified insert was ligated into a 5' BamHI/3' EcoRI site of JA4304. The short nucleotide sequence, as provided in SEQ ID NO: 217, was inserted upstream of the initiation codon for creating the Kozak-like sequence and reconstituting the HindIII site. The expressed protein contains the initiation codon and the downstream 21 amino acids from the pET17b expression vector, i.e., MASMTGGQQMGRDSSLVPSSDP (SEQ ID NO: 218). In addition, a six-histidine tag is included upstream of the sequence described above and is fused at the 28th amino acid (nucleotide 84) of the gene, which eliminates the hypothetical signal peptide.   The sequences provided in SEQ ID NO: 183 with the corresponding amino acid sequence provided in SEQ ID NO: 191 do not include these additional sequences. The pmpG gene (SEQ ID NO: 182, with the corresponding amino acid sequence provided in SEQ ID No; 190) was PCR amplified under conditions well known in the art using the following oligo primers: 5' oligo- CAG AGG TAC CGC ATC ACC ATC ACC ATC ACA TGA TTC CTC AAG GAA TTT ACG (SEQ ID NO: 219), and the 3' oligo- CAG AG.C GGC CGC TTA GAA CCG GAC TTT ACT TCC (SEQ ID NO: 220), and ligated into the 5' KPN/3' NotI cloning site of the expression vector. The expressed protein contains an additional amino acid sequence at the amino end, namely, MASMTGGQQNGRDSSLVPHHHHHH (SEQ ID NO: 221), which comprises the initiation codon and additional sequence from the pET17b expression vector. The pmpI gene (SEQ ID NO: 181, with the corresponding amino acid sequence provided in SEQ ID No; 189) was PCR amplified under conditions well known in the art using the following oligo primers: 5' oligo- CAG AGC TAG CCA TCA CCA TCA CCA TCA CCT CTT TGG CCA GGA TCC C (SEQ ID NO: 222), and the 3' oligo- CAG AAC TAG TCT AGA ACC TGT AAG TGG TCC (SEQ ID NO: 223), and ligted into the expression vector at the 5' NheI/3' SpeI cloning site. The 95 kD expressed protein contains the initiation codon plus an additional alanine and serine from the pET17b vector at the amino end of the protein. In addition, a six-histidine tag is fused at the 21st amino acid of the gene, which eliminates the hypothetical signal peptide.<br>
Clone 14H1-4, (SEQ ID NO: 56), identified using the TCT-3 cell line, contains a complete ORF for the TSA gene, thiol specific antioxidant - CT603 (the CT603 ORF is a homolog of CPn0778 from C. pnuemoniae). The TSA open reading frame in clone 14-H1-4 was amplified such that the expressed protein possess an additional methionine and a 6x histidine tag (amino terminal end). This amplified insert was sub-cloned into the Nde/EcoRI sites of the pET17b vector. Upon induction of this clone with IPTG, a 22.6 kDa protein was purified by Ni-NTA agarose affinity chromatography. The determined amino acid sequence for the 195 amino acid ORF of clone 14-H1-4 encoding the TSA gene is provided in SEQ ID NO: 65. Further analysis yielded a full-length clone for the TSA gene, referred to as CTL2-TSA-FL, with the full-length amino acid sequence provided in SEQ ID NO: 92.<br>
Further studies yielded 10 additional clones identified by the TCT-1 and TCT-3 T-cell lines, as described above. The clones identified by the TCT-1 line are: 16-D4-22, 17-C5-19; 18-C5-2, 20-G3-45 and 21-C7-66; clones identified by the TCT-3 cell line are: 17-C10-31, 17-E2-9, 22-A1-49 and 22-B3-53. Clone 21-G12-60 was recognized by both the TCT-1 and TCT-3 T cell lines. In addition, clone 20-G3-45, which contained sequence specific for pmpB, was identified against the patient lines CT1 and CT4. Clone 16-D4-22 (SEQ ID NO: 119), identified using the TCT-1 cell line contains a 953 bp insert that contains two genes, parts of open reading frame 3 (ORF3) and ORF4 of the C. trachomatis plasmid for growth within mammalian cells.   Clone 17-C5-19 (SEQ ID NO: 118), contains a 951 bp insert that contains part of the ORF for DT431, encoding for clpP_l protease and part of the ORF for CT430 (diaminopimelate epimerase).   Clone 18-C5-2 (SEQ ID NO: 117) is part of the ORF for S1 ribosomal protein with a 446 bp insert that was identified using the TCT-1 cell line. Clone 20-G3-45 (SEQ ID NO: 116), identified by the TCT-1 cell line, contains a 437 bp insert that is part of the pmpB gene (CT413).  Clone 21-C7-8 (SEQ ID NO: 115), identified by the TCT-1 line, contains a 995bp insert that encodes part of the dnaK like protein. The insert of this clone does not overlap with the insert of the TCT-3 clone 11-H4-28 (SEQ ID NO: 59), which was shown to be part of the dnaK gene CT396.  Clone 17-C10-31 (SEQ ID NO: 114), identified by the TCT-3 cell line, contains a 976 bp insert.  This clone contains part of the ORF for CT858, a protease containing IRBP and DHR domains.   Clone 17-E2-9 (SEQ ID NO: 113) contains part of ORFs for two genes, CT611 and CT610, that span a 1142 bp insert. Clone 22-A1-49 (SEQ ID NO: 112), identified using the TCT-3 line, also contains two genes in a 698 bp insert. Part of the ORF for CT660 (DNA gyrase{gyrA_2}) is present on the top strand where as the complete ORF for a hypothetical protein CT659 is present on the complementary strand. Clone 22-B3-53 (SEQ ID NO: 111), identified by the TCT-1 line, has a 267 bp insert that encodes part of the ORF for GroEL (CT110). Clone 21-G12-60 (SEQ ID NO: 110), identified by both the TCT-1 and TCT-3 cell lines contains a 1461 bp insert that contains partial ORFs for hypothetical proteins CT875, CT229 and CT228.<br>
Additional Chlamydia antigens were obtained by screening a genomic expression library of Chlamydia trachomatis (LGV II serovar) in Lambda Screen-1 vector (Novagen, Madison, WI) with sera pooled from several Chlamydia-infected individuals using techniques well known in the art. The following immuno-reactive clones were identified and the inserts containing Chlamydia genes sequenced: CTL2#1 (SEQ ID NO: 71); CTL2#2 (SEQ ID NO: 70); CTL2#3-5' (SEQ ID NO: 72, a first determined genomic sequence representing the 5' end); CTL2#3-3' (SEQ ID NO: 73, a second determined genomic sequence representing the 3' end); CTL2#4 (SEQ ID NO: 53); CTL2#5 (SEQ ID NO: 69); CTL2#6 (SEQ ID NO: 68); CTL2#7 (SEQ ID NO: 67); CTL2#8b (SEQ ID NO: 54); CTL2#9 (SEQ ID NO: 66); CTL2#10-5' (SEQ ID NO: 74, a first determined genomic sequence representing the 5' end); CTL2#10-3' (SEQ ID NO: 75, a second determined genomic sequence representing the 3' end); CTL2#11-5' (SEQ ID NO: 45, a first determined genomic sequence representing the 5' end); CTL2#11-3' (SEQ ID NO: 44, a second determined genomic sequence representing the 3' end); CTL2#12 (SEQ ID NO: 46); CTL2#16-5' (SEQ ID NO: 47); CTL2#18-5' (SEQ ID NO: 49, a first determined genomic sequence representing the 5' end); CTL2#18-3' (SEQ ID NO: 48, a second determined genomic sequence representing the 3' end); CTL2#19-5' (SEQ ID NO: 76, the determined genomic sequence representing the 5' end); CTL2#21 (SEQ ID NO: 50);CTL2#23 (SEQ ID NO: 51; and CTL2#24 (SEQ ID NO: 52).<br>
Additional Chlamydia trachomatis antigens were identified by serological expression cloning. These studies used sera pooled from several Chlamydia-infected individuals, as described above, but, IgA,and IgM antibodies were used in addition to IgG as a secondary antibody. Clones screened by this method enhance detection of antigens recognized by an early immune response to a Chlamydial infection, that is a mucosal humoral immune response. The following immunoreactive clones were characterized and the inserts containing Chlamydia genes sequenced: CTL2gam-l (SEQ ID NO: 290), CTL2gam-2 (SEQ ID NO: 289), CTL2gam-5 (SEQ ID NO: 288), CTL2gam-6-3' (SEQ ID NO: 287, a second determined genomic sequence representing the 3' end), CTL2gam-6-5' (SEQ ID NO: 286, a first determined genomic sequence representing the 5' end), CTL2gam-8 (SEQ ID NO: 285), CTL2gam-10 (SEQ ID NO: 284), CTL2gam-13 (SEQ ID NO: 283), CTL2gam-15-3' (SEQ ID NO: 282, a second determined genomic sequence representing the 3' end), CTL2gam-15-5' (SEQ ID NO: 281, a first determined genomic sequence representing the 5' end), CTL2gam-17 (SEQ ED NO: 280), CTL2gam-18 (SEQ ID NO: 279), CTL2gam-21 (SEQ ID NO: 278), CTL2gam-23 (SEQ ID NO: 277), CTL2gam-24 (SEQ ID NO: 276), CTL2gam-26 (SEQ ID NO: 275), CTL2gam-27 (SEQ ID NO: 274), CTL2gam-28 (SEQ ID NO: 273), CTL2gam-30-3'   (SEQ   ID   NO:   272,   a   second   determined   genomic   sequence <br>
representing the 3' end) and CTL2gam-30-5' (SEQ ID NO: 271, a first determined genomic sequence representing the 5' end).<br>
EXAMPLE 2<br>
INDUCTION OF T CELL PROLIFERATION AND INTERFERON-γ<br>
PRODUCTION BY CHLAMYDIA TRACHOMATIS ANTIGENS<br>
The ability of recombinant Chlamydia trachomatis antigens to induce T cell proliferation and interferon-γ production is determined as follows.<br>
Proteins are induced by EPTG and purified by Ni-NTA agarose affinity chromatograph (Webb et al., J. Immunology 757:5034-5041, 1996). The purified polypeptides are then screened for the ability to induce T-cell proliferation in PBMC preparations. PBMCs from C. trachomatis patients as well as from normal donors whose T-cells are known to proliferate in response to Chlamydia antigens, are cultured in medium comprising RPMI 1640 supplemented with 10% pooled human serum and 50 μg/ml gentamicin. Purified polypeptides are added in duplicate at concentrations of 0.5 to 10 μg/mL. After six days of culture in 96-well round-bottom plates in a volume of 200 μl, 50 μl of medium is removed from each well for determination of IFN-y levels, as described below. The plates are then pulsed with 1 μCi/well of tritiated thymidine for a further 18 hours, harvested and tritium uptake determined using a gas scintillation counter. Fractions that result in proliferation in both replicates three fold greater than the proliferation observed in cells cultured in medium alone are considered positive.<br>
IFN-y is measured using an enzyme-linked immunosorbent assay (ELISA). ELISA plates are coated with a mouse monoclonal antibody directed to human IFN-y (PharMingen, San Diego, CA) in PBS for four hours at room temperature. Wells are then blocked with PBS containing 5% (W/V) non-fat dried milk for 1 hour at room temperature. The plates are washed six times in PBS/0.2% TWEEN-20 and samples diluted 1:2 in culture medium in the ELISA plates are incubated overnight at room temperature. The plates are again washed and a polyclonal rabbit anti-human IFN-y serum diluted 1:3000 in PBS/10% nonnal goat serum is added to each well. The plates are then incubated for two hours at room temperature, washed and horseradish peroxidase-coupled anti-rabbit IgG (Sigma Chemical So., St. Louis, MO) is added at a 1:2000 dilution in PBS/5% non-fat dried milk. After a further two hour incubation at room temperature, the plates are washed and TMB substrate added. The reaction is stopped after 20 min with 1 N sulfuric acid. Optical density is determined at 450 nm using 570 nm as a reference wavelength. Fractions that result in both replicates giving an OD two fold greater than the mean OD from cells cultured in medium alone, plus 3 standard deviations, are considered positive.<br>
Using the above methodology, recombinant 1B1-66 protein (SEQ ID NO: 5) as well as two synthetic peptides corresponding to amino acid residues 48-67 (SEQ ID NO: 13; referred to as 1-B1-66/48-67) and 5S-77 (SEQ ID NO: 14, referred to as 1B1-66/58-77), respectively, of SEQ ID NO: 5, were found to induce a proliferative response and IFN-γ production in a Chlamydia-specific T cell line used to screen a genomic library of G trachomatis LGV II<br>
Further studies have identified a C. trachomatis-spccific T-cell epitope in the ribosomal S13 protein. Employing standard epitope mapping techniques well known in the art, two T-cell epitopes in the ribosomal S13 protein (rS13) were identified with a Chlamydia-specific T-cell line from donor CL-8 (T-cell line TCL-8 EB/DC). Fig. 8 illustrates that the first peptide, rS13 1-20 (SEQ ID NO: 106), is 100% identical with the corresponding C. pneumoniae sequence, explaining the cross-reactivity of the T-cell line to recombinant C. trachomatis- and C. pneumoniae-rS13. The response to the second peptide rS13 56-75 (SEQ ID NO: 108) is C. trachomatis-specific, indicating that the rS13 response in this healthy asymptomatic donor was elicited by exposure to C. trachomatis and not to C. pneumoniae, or any other microbial infection.<br>
As described in Example 1, Clone 11-C12-91 (SEQ ID NO: 63), identified using the TCP-21 cell line, has a 269 bp insert that is part of the OMP2 gene (CT443) and shares homology with the 60 kDa cysteine rich outer membrane protein of C. pneumoniae, referred to as OMCB. To further define the reactive epitope(s), epitope mapping was performed using a series of overlapping peptides and the immunoassay previously described. Briefly, proliferative responses were determined by stimulating 2.5 X 104 TCP-21 T-cells in the presence of 1 x 104 monocyte-derived dendritic cells with either non-infectious elementary, bodies derived from C. trachomatis and C. pneumoniae, or peptides derived from the protein sequence of C. 'trachomatis or C pneumoniae OMCB protein (0.1 μg/ml). The TCP-21 T-cells responded to epitopes CT-OMCB #167-186, CT-OMCB #171-190, CT-OMCB #171-186, and to a lesser extent, CT-OMCB #175-186 (SEQ ID NO: 249-252, respectively). Notably, the TCP-21 T-cell line also gave a proliferative response to the homologous C. pneumoniae peptide CP-OMCB #171-186 (SEQ ID NO: 253), which was equal to or greater than the response to the C. trachomatis peptides. The amino acid substitutions in position two (i.e., Asp for Glu) and position four (i.e., Cys for Ser) did not alter the proliferative response of the T-cells and therefore demonstrating this epitope to be a cross-reactive epitope between C. trachomatis and C. pneumoniae.<br>
To further define the epitope described above, an additional T-cell line, TCT-3, was used in epitope mapping experiments. The immunoassays were performed as described above, except that only peptides from C. trachomatis were tested. The T-cells gave a proliferative response to two peptides, CT-OMCB #152-171 and CT-OMCB #157-176 (SEQ ID NO: 246 and 247, respectively), thereby defining an additional immunogenic epitope in the cysteine rich outer membrane protein of C. trachomatis.<br>
Clone 14H1-4, (SEQ ID NO: 56, with the corresponding full-length amino acid sequence provided in SEQ ID NO: 92), was identified using the TCT-3 cell line in the CD4 T-cell expression cloning system previously described, and was shown to contain a complete ORP for the, thiol specific antioxidant gene (CT603), referred to as TSA. Epitope mapping immunoassays were performed, as described above, to further define the epitope. The TCT-3 T-cells line exhibited a strong proliferative response to the overlapping peptides CT-TSA #96-115, CT-TSA #101-120 and CT-TSA #106-125 (SEQ ID NO: 254-256, respectively) demonstrating an immunoreactive epitope in the thiol specific antioxidant gene of C. trachomatis serovar LGVH<br>
EXAMPLE 3 PREPARATION OF SYNTHETIC POLYPEPTIDES<br>
Polypeptides may be synthesized on a Millipore 9050 peptide synthesizer using FMOC chemistry with HPTU (0-Benzotriazole-N,N,N',N'-tetramemyluronium hexafluorophosphate) activation. A Gly-Cys-Gly sequence may be attached to the amino terminus of the peptide to provide a method of conjugating or labeling of the peptide. Cleavage of the peptides from the solid support may be carried out using the following cleavage mixture: trifluoroacetic acid:ethanedithiol:thioanisole:water:phenol (40:1:2:2:3). After cleaving for 2 hours, the peptides may be precipitated in cold methyl-t-butyl-ether. The peptide pellets may then be dissolved in water containing 0.1% trifluoroacetic acid (TFA) and lyophilized prior to purification by C18 reverse phase HPLC. A gradient of 0-60% acetonitrile (containing 0.1% TFA) in water (containing 0.1% TFA) may be used to elute the peptides. Following lyophilization of the pure fractions, the peptides may be characterized using electrospray mass spectrometry and by amino acid analysis.<br>
EXAMPLE 4<br>
ISOLATION AND CHARACTERIZATION OF DNA SEQUENCES ENCODING<br>
CHLAMYDIA ANTIGENS USING RETROVIRAL EXPRESSION VECTOR<br>
SYSTEMS AND SUBSEQUENT IMMUNOLOGICAL ANALYSIS<br>
A genomic library of Chlamydia trachomatis LGV II was constructed by limited digests using BamHI, BgIII, BstYi and MboI restriction enzymes. The restriction digest fragments were subsequently ligated into the BamHI site of the retroviral vectors pBIB-KS1,2,3. This vector set was modified to contain a Kosak translation initiation site and stop codons in order to allow expression of proteins from short DNA genomic fragments, as shown in Fig. 2. DNA pools of 80 clones were prepared and transfected into the retroviral packaging line Phoenix-Ampho, as described in Pear, W.S., Scott, M.L. and Nolan, G.P., Generation of High Titre, Helper-free Retroviruses by Transient Transfection.   Methods in Molecular Medicine: Gene<br><br><br>
		Therapy Protocols, Humana Press, Totowa, NJ, pp. 41-57. The Chlamydia library in retroviral form was then transduced into H2-Ld expressing P815 cells, which were then used as target cells to stimulate an antigen specific T-cell line.<br>
A Chlamydia-specific, murine H2d restricted CD8+ T-cell line was expanded in culture by repeated rounds of stimulation with irradiated C. trachomatis-infected J774 cells and irradiated syngeneic spleen cells, as described by Starnbach, M., in J. Immunol., 153:5183,1994. This Chlamydia-spocific T-cell line was used to screen the above Chlamydia genomic library expressed by the retrovirally-transduced P815 cells. Positive DNA pools were identified by detection of IFN-γ production using Elispot analysis (SEE Lalvani et al., J. Experimental Medicine186 859-865,1997).<br>
Two positive pools, referred to as 2C7 and 2E10, were identified by IFN-y Elispot assays. Stable transductants of P815 cells from pool 2C7 were cloned by limiting dilution and individual clones were selected based upon their capacity to elicit IFN-y production from the Chlamydia-specific CTL line. From this screening process, four positive clones were selected, referred to as 2C7-8, 2C7-9, 2C7-19 and 2C7-21. Similarly, the positive pool 2E10 was further screened, resulting in an additional positive clone, which contains three inserts. The three inserts are fragments of the CT016, tRNA syntase and clpX genes (SEQ ID NO: 268-270, respectively).<br>
Transgenic DNA from these four positive 2C7 clones were PCR amplified using pBIB-KS specific primers to selectively amplify the Chlamydia DNA insert. Amplified inserts were gel purified and sequenced. One immunoreactive clone, 2C7-8 (SEQ ID NO: 15, with the predicted amino acid sequence provided in SEQ ID NO: 32), is a 160 bp fragment with homology to nucleotides 597304-597145 of Chlamydia trachomatis, serovar D (NCBI, BLASTN search; SEQ ID NO: 33, with the predicted amino acid sequence provided in SEQ ID NO: 34). The sequence of clone 2C7-8 maps within two putative open reading frames from the region of high homology described immediately above, and in particular, one of these putative open reading frames, consisting of a 298 amino acid fragment (SEQ ID NO: 16, with the predicted amino acid sequence provided in SEQ ID NO: 17), was demonstrated to exhibit immunological activity.<br>
Full-length cloning of the 298 amino acid fragment (referred to as CT529 and/or the Capl gene) from serovar L2 was obtained by PCR amplification using 5'-  <br>
ttttgaagcaggtaggtgaatatg (forward) (SEQ ID NO: 159) and 5'-ttaagaaatttaaaaaatccctta (reverse) (SEQ ID NO: 160) primers, using purified C. trachomatis L2 genomic DNA as template. This PCR product was gel-purified, cloned into pCRBlunt (Invitrogen, Carlsbad, CA) for sequencing, and then subcloned into the EcoR1 site of pBIB-KMS, a derivative of pBIB-KS for expression. The Chlamydia pnuemoniae homlogue of CT529 is provided in SEQ ID NO: 291, with the corresponding amino acid sequence provided in SEQ ID NO: 292.<br>
Full-length DNA encoding various CT529 serovars were amplified by<br>
PCR from bacterial lysates containing 100 IFU, essentially as described (Denamur, E.,<br>
C. Sayada, A. Souriau, J. Orfila, A. Rodolakis and J. Elion. 1991. J. Gen. Microbiol.<br>
137:2525). The following serovars were amplified as described: Ba (SEQ ID NO: 134,<br>
with the corresponding predicted amino acid sequence provided in SEQ ID NO: 135); E<br>
(BOUR) and E (MTW447) (SEQ ID NO: 122, with the corresponding predicted amino<br>
acid sequence provided in SEQ ID NO: 123); F (N11) (SEQ ID NO: 128, with the<br>
corresponding predicted amino acid sequence provided in SEQ ID NO: 129); G; (SEQ<br>
ID NO: 126, with the corresponding predicted arnino acid sequence provided in SEQ ID<br>
NO: 127); la (SEQ ID NO: 124, with the corresponding predicted amino acid sequence<br>
provided in SEQ ID NO: 125); L1 (SEQ ID NO: 130, with the corresponding predicted<br>
amino acid sequence provided in SEQ ID NO: 131); L3 (SEQ ID NO: 132, with the<br>
corresponding predicted amino acid sequence provided in SEQ ID NO: 133); I (SEQ ID<br>
NO: 263, with the corresponding predicted amino acid sequence provided in SEQ ID<br>
NO: 264); K (SEQ ID NO: 265, with the corresponding predicted amino acid sequence<br>
provided in SEQ ID NO: 266); and MoPn (SEQ ID NO: 136, with the corresponding<br>
predicted amino acid sequence provided in SEQ ID NO: 137).   PCR reactions were<br>
performed with Advantage Genomic PCR Kit (Clontech, Palo Alto, CA) using primers<br>
specific for serovar L2 DNA (external to the ORF).   Primers sequences were 5'-<br>
ggtataatatctctctaaattttg (forward-SEQ ID NO: 161) and 5'-agataaaaaaggctgtttc' (reverse-<br>
SEQ ID NO: 162)  except for MoPn which required 5'-ttttgaagcaggtaggtgaatatg<br>
(forward-SEQ ID NO: 163) and 5'-tttacaataagaaaagctaagcactttgt (reverse-SEQ ID NO:<br>
164). PCR amplified DNA was purified with QIAquick PCR purification kit (Qiagen,<br>
Valencia, CA) and cloned in pCR2.1 (Invitrogen, Carlsbad, CA) for sequencing.<br>
Sequencing of DNA derived from PCR amplified inserts of imrnunoreactive clones was done on an automated sequencer (ABI 377) using both a pBIB-KS specific forward primer 5'-ccttacacagtcctgctgac (SEQ ID NO: 165) and a reverse primer 3'-gtttccgggccctcacattg (SEQ ID NO: 166). PCRBlunt cloned DNA coding for CT529 serovar L2 and pCR2.1 cloned DNA coding for CT529 serovar Ba, E (BOUR), E (MTW447), F (N11), G, la, K, L1, L3 and MoPn were sequenced using T7 promoter primer and universal M13 forward and M13 reverse primers.<br>
To determine if these two putative open reading frames (SEQ ID NO: 16 and 20) encoded a protein with an associated immunological function, overlapping peptides (17-20 amino acid lengths) spanning the lengths of the two open reading frames were synthesized, as described in Example 3. A standard chromium release assay was utilized to determine the percent specific lysis of peptide-pulsed H2d restricted target cells. In this assay, aliquots of P815 cells (H2d) were labeled at 37° C for one hour with 100 μCi of 51Cr in the presence or absence of 1 μg/ml of the indicated peptides. Following this incubation, labeled P815 cells were washed to remove excess<br>
51Cr and peptide, and subsequently plated in duplicate in microculture plates at a concentration of 1,000 cells/well. Effector CTL (Chlamydia-specific CD8 T cells) were added at the indicated effector:target ratios. Following a 4 hour incubation, supematants were harvested and measured by gamma-counter for release of 51Cr into the supernatant. Two overlapping peptides from the 298 amino acid open reading frame did specifically stimulate the CTL line. The peptides represented in SEQ ID NO: 138-156 were synthesized, representing the translation of the L2 homologue of the serovar D open reading frame for CT529 (Capl gene) and 216 amino acid open reading frame. As shown in Fig. 3, peptides CtC7.8-12 (SEQ ID NO: 18, also referred to as Capl#132-147, SEQ ID NO: 139) and CtC7.8-13 (SEQ ID NO: 19, also referred to as Capl#138-155, SEQ ID NO: 140) were able to elicit 38 to 52% specific lysis, respectively, at an effector to target ratio of 10:1. Notably, the overlap between these two peptides contained a predicted H2d (Kd and Ld) binding peptide. A 10 amino acid peptide was synthesized to correspond to this overlapping sequence (SEQ ID NO: 31) and was found to generate a strong immune response from the anti-Chlamydia CTL line by elispot assay. Significantly, a search of the most recent Genbank database revealed no proteins have previously been described for this gene.   Therefore, the putative open reading frame encoding clone 2C7-8 (SEQ ID NO: 15) defines a gene which encompasses an antigen from Chlamydia capable of stimulating antigen-specific CD8+ T-cells in a MHC-I restricted manner, demonstrating this antigen could be used to develop a vaccine against Chlamydia.<br>
To confirm these results and to further map the epitope, truncated peptides (SEQ ID NO: 138-156) were made and tested for recognition by the T-cells in an IFN-g ELISPOT assay. Truncations of either Serl39 (Capl#140-147, SEQ ID NO: 146) or Leul47 (Capl#138-146, SEQ ID NO: 147) abrogate T-cell recognition. These results indicate that the 9-mer peptide Capl#139-147 (SFIGGITYL, SEQ ID NO: 145) is the minimal epitope recognized by the Chlamydia-specific T-cells.<br>
Sequence alignments of Capl (CT529) from selected serovars of C. trachomatis (SEQ ID NO: 121, 123, 125, 127, 129, 131, 133, 135, 137 and 139) shows one of the amino acid differences is found in position 2 of the proposed epitope. The homologous serovar D peptide is SIIGITYL (SEQ ID NO: 168). The ability of SFIGGITYL and SHGGITYL to target cells for recognition by the Chlamydia specific T-cells was compared. Serial dilutions of each peptide were incubated with P815 cells and tested for recognition by the T-cells in a 51Cr release assay, as described above. The Chlamydia-specific T-cells recognize the serovar L2 peptide at a minimum concentration of 1 nM and the serovar D peptide at a minimum concentration of 10 nM.<br>
Further studies have shown that a Capl#139-147-specific T-cell clone recognizes C. trachomatis infected cells. To confirm that Capl 139-147 is presented on the surface of Chlamydia infected cells, Balb-3T3 (H-2d) cells were infected with C. trachomatis serovar L2 and tested to determine whether these cells are recognized by a CD8+ T-cell clone specific for Capl#139-147 epitope (SEQ ID NO: 145). The T-cell clone specific for Capl#139-147 epitope was obtained by limiting dilution of the line 69 T-cells. The T-cell clone specifically recognized the Chlamydia infected cells. In these experiments, target cells were C. trachomatis infected (positive control) or uninfected Balb/3T3 cells, showing 45%, 36% and 30% specific lysis at 30:1, 10:1 and 3:1 effector to target ratios, respectively; or Capl#139-147 epitope (SEQ ID NO: 145) coated, or untreated P815 cells, showing 83%, 75% and 58% specific lysis at 30:1, 10:1 and 3:1 effector to target ratios, respectively (negative controls having less than 5% lysis in all cases). This data suggests that the epitope is presented during infection.<br>
In vivo studies show Capl#139-147 epitope-specific T-cells are primed during murine infection with C. trachomatis. To determine if infection with C. trachomatis primes a Cap1#139-147 epitope-specific T-cell response, mice were infected i.p. with 108 IFU of C. trachomatis serovar L2. Two weeks after infection, the mice were sacrificed and spleen cells were stimulated on irradiated syngeneic spleen cells pulsed with Capl#l39-147 epitope peptide. After 5 days of stimulation, the cultures were used in a standard 51Cr release assay to determine if there were Capl#139-147 epitope-specific T-cells present in the culture. Specifically, spleen cells from a C. trachomatis serovar L2 immunized mouse or a control mouse injected with PBS after a 5 days culture with Capl#139-147 peptide-coated syngeneic spleen cells and CD8+ T-cells.able to specifically recognize Capl#139-147 epitope gave 73%, 60% and 32% specific lysis at a30:l, 10:1 and 3:1 effector to target ratios, respectively. The control mice had a percent lysis of approximately 10% at a 30:1 effector to target ratio, and steadily declining with lowering E:T ratios. Target cells were Capl#139-147 peptide-coated, or untreated P815 cells. These data suggest that Capl#139-147 peptide-specific T-cells are primed during murine infection with C. trachomatis.<br>
Ct529 Localization<br>
Studies were performed demonstrating that Ct529 (referred to herein as Cap-1) localizes to the inclusion membrane of C. trachomatis-infected cells and is not associated with elementary bodies or reticulate bodies. As described above, Cap-1 was identified as a product from Chlamydia that stimulates CD8+ CTL. These CTL are protective in a murine model of infection, thus making Cap-1 a good vaccine candidate. Further, since these CTL are MHC-I restricted, the Cap-1 gene must have access to the cytosol of infected cells, which may be a unique characteristic of specific Chlamydial gene products. Therefore, determination of the cellular localization of the gene products would be useful in characterizing Cap-1 as a vaccine candidate. To detect the intracellular localization of Cap-1, rabbit polyclonal antibodies directed against a recombinant polypeptide encompassing the N-terminal 125 amino acids of Cap-1 (SEQ ID NO: 305, with the amino acid sequence including the N-terminal 6-His tag provided in SEQ ID NO: 304) were used to stain McCoy cells infected with Chlamydias.<br>
Rabbit-anti-Cap-1 polyclonal antibodies were obtained by hyper-immunization of rabbits with a recombinant polypeptide, rCt529cl-125 (SEQ ID NO: 305) encompassing the N-terminal portion of Cap-1. Recombinant rCt529el-125 protein was obtained from E. coli transformed with a pET expression plasmid (as described above) encoding the nucleotides 1-375 encoding the N-terminal 1-125 amino acids of Cap-1. Recombinant protein was purified by Ni-NTA using techniques well known in the art. For a positive control antiserum, polyclonal antisera directed against elementary bodies were made by immunization of rabbits with purified C. trachomatis elementary bodies (Biodesign, Sacco, Maine). Pre-immune sera derived from rabbits prior to immunization with the Cap-1 polypeptide was used as a negative control.<br>
Immunocytochemistry was performed on McCoy cell monolayers grown on glass coverslips inoculated with either C. trachomatis serovar L2 or C. psitacci, strain 6BC, at a concentration of 106 IFU (Inclusion Forming Units) per ml. After 2 hours, medium was aspirated and replaced with fresh RP-10 medium supplemented with cycloheximide (1.0 μg/ml). Infected cells were incubated at in 7% CO2 for 24 hours and fixed by aspirating medium, rinsing cells once with PBS and methanol fixation for 5 minutes. For antigen staining, fixed cell monolayers were washed with PBS and incubated at 37°C for 2 hours with 1:100 dilutions of specific or control antisera. Cells were rinsed with PBS and incubated for 1 hour with fluorescein  isothiocyanate (FITC)-labeled, anti-rabbit IgG (KPL, Gaithersburg) and stained with Evans blue (0.05%) in PBS. Fluorescence was observed with a 100X objective (Zeiss epifluorescence microscope), and photographed (Nikon UFX-11A camera).<br>
Results from this study show Cap-1 localizes to the inclusion membrane of C. trachomatis-infected cells. Cap-1 specific antibody labeled the inclusion membranes of C. trachomatis-infected cells, but not Chlamydial elementary bodies contained in these inclusions or released by the fixation process. Conversely, the anti-elementary body antibody clearly labeled the bacterial bodies, not only within the inclusions, but those released by the fixation process. Specificity of the anti-Cap-1 antibody is demonstrated by the fact that it does not stain C. psittaci-infected cells. Specificity of the Cap-1 labeling is also shown by the absence of reactivity in pre-immune sera. These results suggest that Cap-1 is released from the bacteria and becomes associated with the Chlamydial inclusion membrane. Therefore, Cap-1 is a gene product which may be useful for stimulating CD8+ T cells in the development of a vaccine against infections caused by Chlamydia.<br>
The relevance of the Cap-1 gene as a potential CTL antigen in a vaccine against Chlamydia infection is further illustrated by two additional series of studies. First, CTL specific for the MHC-I epitope of Cap-1 CT529 #138-147 peptide of C. trachomatis (SEQ ID NO: 144) have been shown to be primed to a high frequency during natural infection. Specifically, Balb/C mice were inoculated with 106 I.F.U. of C. trachomatis, serova L2. After 2 weeks, spleens were harvested and quantified by Elispot analysis for the number of IFN-γ secreting cells in response to Cap-1 #138-147 peptide-pulsed antigen presenting cells. In two experiments, the number of IFN-y-secreting cells in 105 splenocytes was about 1% of all CD8+ T-cells. This high frequency of responding CD8+ CTL to the MHC-1 epitope (Cap-1 CT529 #138-147 peptide) suggest that Cap-1 is highly immunogenic in infections.<br>
Results from a second series of studies have shown that the Cap-1 protein is almost immediately accessible to the cytosol of the host cell upon infection. This is shown in a time-course of Cap-1 CT529 #138-147 peptide presentation. Briefly, 3T3 cells were infected with C. trachomatis serovar L2 for various lengths of time, and then tested for recognition by Cap-1 CT529 #138-147 peptide-specific CTL. The results show that C. frachomatis-infected 3T3 cells are targeted for recognition by the antigen-specific CTL after only 2 hours of infection. These results suggest that Cap-1 is an early protein synthesized in the development of C. trachomatis elementary bodies to reticulate bodies. A CD8+ CTL immune response directed against a gene product expressed early in infection may be particularly efficacious in a vaccine against Chlamydia infection.<br>
EXAMPLE 5 GENERATION OF ANTIBODY AND T-CELL RESPONSES IN MICE IMMUNIZED<br>
WITH CHLAMYDIA ANTIGENS<br>
Immunogenicity studies were conducted to determine the antibody and CD4+ T cell responses in mice immunized with either purified SWIB or S1 3 proteins formulated with Montanide adjuvant, or DNA-based immunizations with pcDNA-3 expression<br>
 vectors containing the DNA sequences for SWIB or S13. SWIB is also referred to as<br>
clone 1-B1-66 (SEQ ID NO: 1, with the corresponding amino acid sequence provided in<br>
SEQ ID NO: 5), and S13 ribosomal protein is also referred to as clone 10-C10-31 (SEQ<br>
ID NO: 4, with the corresponding amino acid sequence provided in SEQ ID NO: 12).<br>
In the first experiment, groups of three C57BL/6 mice were immunized twice and<br>
monitored for antibody and CD4+ T-cell responses. DNA immunizations were<br>
intradermal at the base of the tail and polypeptide immunizations were administered by<br>
subcutaneous route. Results from standard 3H-incorporation assays of spleen cells from<br>
immunized mice shows a strong proliferative response from the group immunized with<br>
purified recombinant SWIB polypeptide (SEQ ID NO: 5). Further analysis by cytokine<br>
induction assays, as previously described, demonstrated that the group immunized with<br>
SWIB polypeptide produced a measurable IFN-y and IL-4 response. Subsequent<br>
ELISA-based assays to determine the predominant antibody isotype response in the<br>
experimental group immunized with the SWIB polypeptide were performed.   Fig. 4<br>
illustrates the SWIB-immunized group gave a humoral response that was predominantly<br>
IgG1.<br>
In a second experiment, C3H mice were immunized three times with 10 μg purified SWIB protein (also referred to as clone 1-B1 -66, SEQ ID NO: 5) formulated in either PBS or Montanide at three week intervals and harvested two weeks after the third immunization. Antibody titers directed against the SWIB protein were determined by standard ELISA-based techniques well known in the art, demonstrating the SWIB protein formulated with Montanide adjuvant induced a strong humoral immune response. T-cell proliferative responses were determined by a XTT-based assay (Scudiero, et al, Cancer Research, 1988, 48:4827). As shown in Fig. 5, splenocytes from mice immunized with the SWIB polypeptide plus Montanide elicited an antigen specific proliferative response. In addition, the capacity of splenocytes from immunized animals to secrete IFN-γ in response to soluble recombinant SWIB polypeptide was determined using the cytokine induction assay previously described. The splenocytes from all animals in the group immunized with SWIB polypeptide formulated with montanide adjuvant secreted IFN-y in response to exposure to the SWIB Chlamydia antigen, demonstrating an Chlamydia-specific immune response.<br>
In a further experiment, C3H mice were immunized at three separate<br>
time points at the base of the tail with 10 μg of purified SWIB or S13 protein (C<br>
trachomatis, SWIB protein, clone 1-B1-66, SEQ ID NO: 5, and S13 protein, clone 10-<br>
C10-31, SEQ ID NO: 4) formulated with the SBAS2 adjuvant (SmithKline Beecham,<br>
London, England). Antigen-specific antibody titers were measured by ELISA, showing<br>
both polypeptides induced a strong IgG response, ranging in titers from 1 x10'4 to 1 x10-<br>
5. The IgG1 and IgG2a components of this response were present in fairly equal<br>
amounts.  Antigen-specific T-cell proliferative responses, determined by standard 3H-<br>
incorporation assays on spleen cells isolated from immunized mice, were quite strong<br>
for SWIB (50,000 cpm above the negative control) and even stronger for s13 (100,000<br>
cpm above the negative control). The IFNy production was assayed by standard ELISA<br>
techniques from supernatant from the proliferating culture. In vitro restimulation of the<br>
culture with S13 protein induced high levels of IFNy production, approximately 25<br>
ng/ml versus 2 ng/ml for the negative control. Restimulation with the SWIB, protein<br>
also induced IFNy, although to a lesser extent.<br>
In a related experiment, C3H mice were immunized at three separate time points with 10 μg of purified SWIB or S13 protein (C. trachomatis, SWIB protein, clone 1-B1-66, SEQ ID NO: 5, and S13 protein, clone 10-C10-31, SEQ ID NO: 4) mixed with 10 μg of Cholera Toxin. Mucosal immunization was through intranasal inoculation. Antigen-specific antibody responses were determined by standard ELISA techniques. Antigen-specific IgG antibodies were present in the blood of SWIB-immunized mice, with titers ranging from 1 x10-3 to 1 x10-4, but non-detectable in the SB-immunized animals. Antigen-specific T-cell responses from isolated splenocytes, as measured by IFNy production, gave similar results to those described immediately above for systemic immunization.<br>
An animal study was conducted to determine the immunogenicity of the CT529 serovar LGVII CTL epitope, defined by the CT529 10mer consensus peptide (CSFIGGITYL - SEQ ID NO: 31), which was identified as an H2-Kd restricted CTL epitope. BALB/c mice (3 mice per group) were immunized three times with 25 μg of peptide combined with various adjuvants. The peptide was administered systemically at the base of the tail in either SKB Adjuvant System SBAS-2", SBAS-7 (SmithKline<br><br>
Beecham, London, England) or Montanide. The peptide was also administered intranasally mixed with l0ug of Cholera Toxin (CT). Naive mice were used as a control. Four weeks after the 3rd immunization, spleen cells were restimulated with LPS-blasts pulsed with l0ug/ml CT529 l0mer consensus peptide at three different effector to LPS-blasts ratios : 6, 1.5 and 0.4 at 1x106 cell/ml. After 2 restimulations, effector cells were tested for their ability to lyse peptide pulsed P815 cells using a standard chromium release assay. A non-relevant peptide from chicken egg ovalbumin was used as a negative control. The results demonstrate that a significant immune response was elicited towards the CT529 10mer consensus peptide and that antigen-specific T-cells capable of lysing peptide-pulsed targets were elicited in response to immunization with the peptide. Specifically, antigen-specific lytic activities were found in the SBAS-7 and CT adjuvanted group while Montanide and SBAS-2" failed to adjuvant the CTL epitope irnmunization.<br>
EXAMPLE 6 EXPRESSION AND CHARACTERIZATION OF CHLAMYDIA PNEUMONIAE<br>
GENES<br>
The human T-cell line, TCL-8, described in Example 1, recognizes Chlamydia trachomatis as well as Chlamydia pneumonia infected monocyte-derived dendritic cells, suggesting Chlamydia trachomatis and pneumonia may encode cross-reactive T-cell epitopes. To isolate the Chlamydia pneumonia genes homologous to Chlamydia trachomatis LGV II clones 1B1-66, also referred to as SWIB (SEQ ID NO: 1) and clone 10C10-31, also referred to as S13 ribosomal protein (SEQ ID NO: 4), HeLa 229 cells were infected with C. pneumonia strain TWAR (CDC/CWL-029). After three days incubation, the C. pneumonia-infected HeLa cells were harvested, washed and resuspended in 200 ul water and heated in a boiling water bath for 20 minutes. Ten microliters of the disrupted cell suspension was used as the PCR template.<br>
C. pneumonia specific primers were designed for clones 1B1-66 and 10C10-31 such that the 5' end had a 6X-Histidine tag and a Nde I site inserted, and the 3' end had a stop codon and a BamHI site included (Fig. 6). The PCR products were amplified and sequenced by standard techniques well known in the art. The C. pneumonia-speciGc PCR products were cloned into expression vector pET17B (Novagen, Madison, WI) and transfected into E. coli BL21 pLysS for expression and subsequent purification utilizing the histidine-nickel chromatographic methodology provided by Novagen. Two proteins from C. pneumonia were thus generated, a 10-11 kDa protein referred to as CpSWIB (SEQ ID NO: 27, and SEQ ID NO: 78 having a 6X His tag, with the corresponding amino acid sequence provided in SEQ ID NO: 28, respectively), a 15 kDa protein referred to as CpS13 (SEQ ID NO: 29, and SEQ ID NO: 77, having a 6X His tag, with the corresponding amino acid sequence provided in SEQ ID NO: 30 and 91, respectively).<br>
EXAMPLE 7<br>
INDUCTION OF T CELL PROLIFERATION AND INTERFERON-γ<br>
PRODUCTION BY CHLAMYDIA PNEUMONIAE ANTIGENS<br>
The ability of recombinant Chlamydia pneumoniae antigens to induce T cell proliferation and interferon-y production is determined as follows.<br>
Proteins are induced by IPTG and purified by Ni-NTA agarose affinity chromatography (Webb et al., J. Immunology 157:5034-5041, 1996). The purified polypeptides are then screened for the ability to induce T-cell proliferation in PBMC preparations. PBMCs from C. pneumoniae patients as well as from normal donors whose T-cells are known to proliferate in response to Chlamydia antigens, are cultured in medium comprising RPMI 1640 supplemented with 10% pooled human serum and 50 μg/ml gentamicin. Purified polypeptides are added in duplicate at concentrations of 0.5 to 10 μg/mL. After six days of culture in 96-well round-bottom plates in a volume of 200 \i\, 50 \il of medium is removed from each well for determination of IFN-y levels, as described below. The plates are then pulsed with 1 μCi/well of tritiated thymidine for a further 18 hours, harvested and tritium uptake determined using a gas scintillation counter. Fractions that result in proliferation in both replicates three fold greater than the proliferation observed in cells cultured in medium alone are considered positive.<br>
IFN-γ was measured using an enzyme-linked immunosorbent assay (ELISA). ELISA plates are coated with a mouse monoclonal antibody directed to human IFN-y (PharMingen, San Diego, CA) in PBS for four hours at room temperature. Wells are then blocked with PBS containing 5% (W/V) non-fat dried milk for 1 hour at room temperature. The plates are washed six times in PBS/0.2% TWEEN-20 and samples diluted 1:2 in culture medium in the ELISA plates are incubated overnight at room temperature. The plates are again washed and a polyclonal rabbit anti-human IFN-γ serum diluted 1:3000 in PBS/10% normal goat serum is added to each well. The plates are then incubated for two hours at room temperature, washed and horseradish peroxidase-coupled. anti-rabbit IgG (Sigma Chemical So., St Louis, MO) is added at a 1:2000 dilution in PBS/5% non-fat dried milk. After a further two hour incubation at room temperature, the plates are washed and TMB substrate added. The reaction is stopped after 20 min with 1 N sulfuric acid. Optical density is determined at 450 nm using 570 nm as a reference wavelength. Fractions that result in both replicates giving an OD two fold greater than the mean OD from cells cultured in medium alone, plus 3 standard deviations, are considered positive.<br>
A human anti-Chlamydia T-cell line (TCL-8) capable of cross-reacting to C. trachomatis and C. pneumonia was used to determine whether the expressed proteins described in the example above, (i.e., CpSWIB, SEQ ID NO: 27, and SEQ ID NO: 78 having a 6X His tag, with the corresponding amino acid sequence provided in SEQ ID NO: 28, respectively, and the 15 kDa protein referred to as CpS13 SEQ ID NO: 29, and SEQ ID NO: 77, having a 6X His tag, with the corresponding amino acid sequence provided in SEQ ID NO: 30 and 91, respectively), possessed T-cell epitopes common to both C. trachomatis and C. pneumonia. Briefly, E. coli expressing Chlamydial proteins were titered on 1 x 104 monocyte-derived dendritic cells. After two hours, the dendriticcells cultures were washed and 2.5 x 104 T cells (TCL-8) added and allowed to incubate for an additional 72 hours. The amount of INF-γ in the culture supernatant was then determined by ELISA. As shown in Figs. 7A and 7B, the TCL-8 T-cell line specifically recognized the S13 ribosomal protein from both C. trachomatis and C. pneumonia as demonstrated by the antigen-specific induction of IFN-γ, whereas only the SWIB protein from C. trachomatis was recognized by the T-cell line. To validate these results, the T cell epitope of C. trachomatis SWIB was identified by epitope mapping using target cells pulsed with a series of overlapping peptides and the T-cell line TCL-8. 3H-thymidine incorporation assays demonstrated that the peptide, referred to as C.t.SWIB 52-67, of SEQ ID NO: 39 gave the strongest proliferation of the TCL-8 line. The homologous peptides corresponding to the SWIB of C. pneumoniae sequence (SEQ ID NO: 40), the topoisomerase-SWIB fusion of C. pneumoniae (SEQ ID NO: 43) and C. trachomatis (SEQ ID NO: 42) as well as the human SWI domain (SEQ ID NO: 41) were synthesized and tested in the above assay. The T-cell line TCL-8 only recognized the C. trachomatis peptide of SEQ 3D NO: 39 and not the corresponding C. pneumoniae peptide (SEQ ID NO: 40), or the other corresponding peptides described above (SEQ ID NO; 41-43).<br>
Chlamydia-specific T cell lines were generated from donor CP-21 with a positive serum titer against C. pneumonide by stimulating donor PBMC with either C. trachomatis or C. pneumoniae-infacted monocyte-derived dendritic cells, respectively. T-cells generated against C. pneumoniae responded to recombinant C pneumoniae-SWIB but not C. trachomatis-SWIB, whereas the T-cell line generated against C. trachomatis did not respond to either C. trachomatis- or C. pneumoniae-SWIB (see Fig. 9). The C. pneumoniae-SWIB specific immune response of donor CP-21 confirms the C. pneumoniae infection and indicates the elicitation of C pneumoniae-SWIB specific T-cells during in vivo C. pneumoniae infection.<br>
Epitope mapping of the T-cell response to C. pneumoniae-SWIB has shown that Cp-SWIB-specific T-cells responded to the overlapping peptides Cp-SWIB 32-51 (SEQ ID NO: 101) and Cp-SWIB 37-56 (SEQ ID NO: 102), indicating a C pneumoniae-SWIB-specific T-cell epitope Cp-SWIB 37-51 (SEQ ID NO: 100).<br>
In additional experiments, T-cell lines were generated from donor CP1, also a C. pneumoniae seropositive donor, by stimulating PBMC with non-infectious elementary bodies from C. trachomatis and C. pneumoniae, respectively. In particular, proliferative responses were determined by stimulating 2.5 x 10 T-cells in the presence of 1 X 104 monocyte-derived dendritic cells and non-infectious elementary bodies derived from C. trachomatis and C. pneumoniae, or either recombinant C. trachomatis or C. pneumoniae SWIB protein. The T-cell response against SWIB resembled the data obtained with T-cell lines.from CP-21 in that C. pneumoniae-SWIB, but not C. trachomatis-SWIB elicited a response by the C. pneumoniae T-cell line.  In addition, the C. trachomatis T-cell line did not proliferate in response to either C. trachomatis or C. pneumoniae SWIB, though it did proliferate in response to both CT and CP elementary bodies.As described in Example 1, Clone 11-C12-91 (SEQ ID NO: 63), identified using the TCP-21 cell line, has a 269 bp insert that is part of the OMP2 gene (CT443) and shares homology with the 60 kDa cysteine rich outer membrane protein of C. pneumoniae, referred to as OMCB. To further define the reactive epitope(s), epitope mapping was performed using a series of overlapping peptides and the immunoassay previously described.  Briefly, proliferative responses were determined by stimulating 2.5 x 104 TCP-21 T-cells in the presence of 1 x 104 monocyte-derived dendritic cells with either non-infectious elementary bodies derived from C. trachomatis and C. pneumoniae, or peptides derived from the protein sequence of C. trachomatis or C. pneumoniae OMCB protein (0.1 μg/ml). The TCP-21 T-cells responded to epitopes CT-OMCB #167-186, CT-OMCB #171-190, CT-OMCB #171-186, and to a lesser extent, CT-OMCB #175-186 (SEQ ID NO: 249-252, respectively). Notably, the TCP-21 T-cell line also gave a proliferative response to the homologous C. pneumonide peptide CP-OMCB #171 -186 (SEQ ID NO: 253), which was equal to or greater than the response to the to the C. trachomatis peptides. The amino acid substitutions in position two (i.e., Asp for Glu) and position four (i.e., Cys for Ser) did not alter the proliferative response of the T-cells and therefore demonstrating this epitope to be a cross-reactive epitope between C. trachomatis and C. pneumoniae.<br>
EXAMPLE 8 IMMUNE RESPONSES OF HUMAN PBMC AND T-CELL LINES AGAINST<br>
CHLAMYDIA ANTIGENS<br>
The examples provided herein suggest that there is a population of healthy donors among the general population that have been infected with C. trachomatis and. generated a protective immune response controlling the C. trachomatis infection. These donors remained clinically asymptomatic and seronegative for C. trachomatis. To characterize the immune responses of normal donors against chlamydial antigens which had been identified by CD4 expression cloning, PBMC<br>
obtained from 12 healthy donors were tested against a panel of recombinant chlamydial antigens including C. trachomatis-, C. pneumonide-SWIB and C. trachomatis-, C. pneumoniae-S13. The data are summarized in Table I below. All donors were seronegative for C. trachomatis, whereas 6/12 had a positive C. pneumoniae titer. Using a stimulation index of &gt;4 as a positive response, 11/12 of the subjects responded to C. trachomatis elementary bodies and 12/12 responded to C. pneumoniae elementary bodies. One donor, AD104, responded to recombinant C. pneumonide-S13 protein, but not to recombinant C. trachomatis-S13 protein, indicating a C. pneumonide-specific response. Three out of 12 donors had a C. trachomatis-SWIB, but not a C pneumonide-SWIB specific response, confirming a C. trachomatis infection. C. trachomatis and C pneumoniae- S13 elicited a response in 8/12 donors suggesting a chlamydial infection. These data demonstrate the ability of SWIB and S13 to elicit a T-cell response in PBMC of normal study subjects.<br><br><br>
CT= Chlamydia trachomatis; CP= Chlamydia pneumoniae; EB= Chlamydia elementary bodies; Swib= recombinant Chlamydia Swib protein; S13= recombinant Chlamydia S13 protein; lpdA= recombinant Chlamydia lpdA protein; TSA= recombinant Chlamydia TSA protein.Values represent results from standard proliferation assays. Proliferative responses were determined by stimulating 3 x 105 PBMC with 1 x 104 monocyte-derived dendritic cells pre-iricubated with the respective recombinant antigens or elementary bodies (EB). Assays were harvested after 6 days with a 3H-thymidine pulse for the last 18h.<br>
SI: Stimulation index<br>
+/-:      SI ~	4<br>
+:        SI&gt;	4<br>
-H-:        SI	10-30<br>
+++:    SI&gt;	30<br>
In a first series of experiments, T-cell lines were generated from a healthy female individual (CT-10) with a history of genital exposure to C. trachomatis by stimulating T-cells with C. trachomatis LGV II elementary bodies as previously described. Although the study subject was exposed to C. trachomatis, she did not seroconvert and did not develop clinical symptoms, suggesting donor CT-10 may have developed a protective immune response against C. trachomatis. As shown in Fig. 10, a primary Chlamydia-specific T-cell line derived from donor CT-10 responded to C. trachomatis-SWIB, but not C. pnenmoniae-SWIB recombinant proteins, confirming the exposure of CT-10 to C. trachomatis. Epitope mapping of the T-cell response to C. trachomatis-SWIB showed that this donor responded to the same epitope Ct-SWIB 52-67 (SEQ ID NO: 39) as T-cell line TCL-8, as shown in Fig. 11.<br>
Additional T-cell lines were generated as described above for various C. trachomatis patients. A summary of the patients' clinical profile and proliferative responses to various C. trachomatis, and C. pneumoniae elementary bodies and recombinant proteins are summarized in Table II as follows:<br><br>
WO 02/08267<br><br>
PCT/US01/23121<br><br><br>
NGU=Non-Gonococcal Urethritis; BV=Bacterial Vaginosis; CT= Chlamydia trachomatis; CP= Chlamydia pneumoniae; EB= Chlamydia elementary bodies; Swib= recombinant Chlamydia Swib protein; S13= recombinant Chlamydia S13 protein; lpdA= recombinant Chlamydia IpdA protein; TSA= recombinant Chlamydia TSA protein<br>
Values represent results from standard proliferation assays. Proliferative responses were determined by stimulating 3 x 105 PBMC with the respective recombinant antigens or elementary bodies (EB). Assays were harvested after 6 days with a 3H-thymidine pulse for the last 18 hours.<br>
SI: Stimulation index<br>
+/-:        SI ~	4<br>
+:           SI&gt;	4<br>
++:        SI	10-30<br>
+++:     SI   &gt;	30<br>
Using the panel of asymptomatic (as defined above) study subjects and C. trachomatis patients, as summarized in Tables I and II, a comprehensive study of the immune responses of PBMC derived from the two groups was conducted. Briefly, PBMCs from C. pneumoniae patients as well as from normal donors are cultured in medium comprising RPMI 1640 supplemented with 10% pooled human serum and 50 μ g/ml gentamicin. Purified polypeptides, a panel of recombinant chlamydial antigens including C. trachomatis-, C. pneumoniae-SWIB and S13, as well as . C. trachomatis IpdA and TSA are added in duplicate at concentrations of 0.5 to 10 μg/mL. After six days of culture in 96-well round-bottom plates in a volume of 200 μl, 50 μl of medium is removed from each well for determination of IFN-y levels, as described below. The plates are then pulsed with 1 μCi/well of tritiated thymidine for a further 18 hours, harvested and tritium uptake determined using a gas scintillation counter. Fractions that result in proliferation in both replicates three fold greater than the proliferation observed in cells cultured in medium alone are considered positive.<br>
Proliferative   responses   to   the   recombinant   Chlamydiae   antigens<br>
demonstrated that the majority of asymptomatic donors and C. trachomatis patients<br>
recognized the C. trachomatis S13 antigen (8/12) and a majority of the C. trachomatis<br>
patients recognized the C. pneumonia S13 antigen (8/12), with 4/12 asymptomatic<br>
donors also recognizing the C. pneumonia S13 antigen. Also, six out of twelve of the<br>
C. trachomatis patients and four out of twelve of the asymptomatic donors gave a<br>
proliferative response to the IpdA antigen of C. trachomatis. These results demonstrate<br>
that the C. trachomatis and C. pneumonia S13 antigen, C. trachomatis Swib antigen and<br>
the C. trachomatis IpdA antigen are recognized by the asymptomatic donors, indicating<br>
these antigens were recognized during exposure to Chlamydia and an immune response<br>
elicited against them.    This implies these antigens may play a role in conferring<br>
protective immunity in a human host.    In addition,    the C. trachomatis and C.<br>
pneumonia S13 antigen is recognized equally well among the C. trachomatis patients,<br>
therefore indicating there may be epitopes shared between C. trachomatis and C.<br>
pneumonia in the S13 protein. Table III summarizes the results of these studies.<br><br><br>
A series of studies were initiated to determine the cellular immune response to short-term T-cell lines generated from asymptomatic donors and C. trachomatis patients. Cellular immune responses were measured by standard proliferation assays and IFN-y, as described in Example 7. Specifically, the majority of the antigens were in the form of single E. coli clones expressing Chlamydial antigens, although some recombinant proteins were also used in the assays. The single E. coli clones were titered on 1 X 104 monocyte-derived dendritic cells and after two hours, the culture was washed and 2.5 x 104 T-cells were added. The assay using the recombinant proteins were performed as" previously described. Proliferation was determined after four days with a standard 3H-thymidine pulse for the last 18 hours. Induction of IFN-y was determined from culture supernatants harvested after four days using standard ELISA assays, as described above. The results show that all the C. trachomatis antigens tested, except for C.T. Swib, elicited a proliferative response from one or more different T-cell lines derived form C. trachomatis patients. In addition, proliferative responses were elicited from both the G trachomatis patients and asymptomatic donors for the following Chlamydia genes, CT622, groEL, pmpD, CT610 and rS13.<br>
The 12G3-83 clone also contains sequences to CT734 and CT764 in addition to CT622, and therefore these gene sequence may also have imrnunoreactive rpitopes. Similarly, clone 21 G12-60 contains sequences to the hypothetical protein genes CT229 and CT228 in addition to CT875; and 15H2-76 also contains sequences <br>
from CT812 and CT088, as well as sharing homology to the sycE gene. Clone 11H3-61 also contains sequences sharing homology to the PGP6-D virulence protein.<br><br>
EXAMPLE 9 PROTECTION STUDIES USING CHLAMYDIA ANTIGENS 1. SWIB<br>
Protection studies were conducted in mice to determine whether immunization with chlamydial antigens can impact on the genital tract disease resulting from chlamydial inoculation. Two models were utilized; a model of intravaginal inoculation  <br>
that uses a human isolate containing a strain of Chlamydia psittaci (MTW447), and a model of intrauterine inoculation that involves a human isolate identified as Chlamydia trachomatis, serovar F (strain NI1).   Both strains induce inflammation in the upper genital tract, which resemble endometritis and salpingitis caused by Chlamydia trachomatis in women.   In the first experiment, C3H mice (4 mice per group) were immunized three times with 100 μg of pcDNA-3 expression vector containing C trachomatis SWIB DNA (SEQ ID NO: 1, with the corresponding amino acid sequence provided in SEQ ID NO: 5). Inoculations were at the base of the tail for systemic immunization. Two weeks after the last immunization, animals were progesterone treated and infected, either thru the vagina or by injection of the inoculum in the uterus. Two weeks after infection, the mice were sacrificed and genital tracts sectioned, stained and examined for histopathology. Inflammation level was scored (from + for very mild, to +++++ for very severe). Scores attributed, to each single oviduct/ovary were summed and divided by the number of organs examined to get a mean score of inflammation for the group. In the model of uterine inoculation, negative control-irnmunized animals receiving empty vector showed consistent inflammation with an ovary/oviduct mean inflammation score of 6.12, in contrast to 2.62 for the DNA-immunized group. In the model of vaginal inoculation and ascending infection, negative control-immunized mice had an ovary /oviduct mean inflammation score of 8.37, versus 5.00 for the DNA-immunized group. Also, in the later model, vaccinated mice showed no signs of tubal occlusion while negative control vaccinated groups had inflammatory cells in the lumen of the oviduct<br>
In a second experiment, C3H mice (4 mice per group) were immunized three times with 50 μg of pcDNA-3 expression vector containing C. trachomatis SWIB DNA (SEQ ID NO: 1, with the corresponding amino acid sequence provided in SEQ ID NO: 5) encapsulated in Poly Lactide co-Glycolide microspheres (PLG); immunizations were made intra-peritoneally. Two weeks after the last immunization, animal were progesterone treated and infected by inoculation of C. psittaci in the vagina. Two weeks after infection, mice were sacrificed and genital tracts sectioned, stained and examined for histopathology. Inflammation level was scored as previously described. Scores attributed to each single oviduct /ovary were summed and divided by the number of examined organs to get a mean of inflammation for the group.   Negative control-immunized animals receiving PLG-encapsulated empty vector showed consistent infammation with an ovary /oviduct mean inflammation score of 7.28, versus 5.71 for the PLG-encapsulated DNA immunized group. Inflammation in the peritoneum was 1.75 for the vaccinated group versus 3. 75 for the control.<br>
In a third experiment, C3H mice (4 per group) were immunized three times with 10 μg of purified recombinant protein, either SWIB (SEQ ID NO: 1, with the corresponding amino acid sequence provided in SEQ ID NO: 5, or S13 (SEQ ID NO: 4, with the corresponding amino acid sequence provided in SEQ ID NO: 12) mixed with Cholera Toxin (CT); the preparation was administred intranasally upon anaesthesia in a 20 uL volume.   Two weeks after the last immunization, animal were progesterone treated and infected, either by vaginal inoculation of C. psittaci or by injection of C. trachomatis serovar F in the uterus.    Two weeks after infection, the mice were sacrificed and genital tracts sectioned, stained and examined for histopathology.   The degree of inflammation was scored as described above. Scores attributed to each single oviduct /ovary were summed and divided by the number of examined organs to get a mean score of inflammation for the group. In the model of uterine inoculation, negative control- immunized animals receiving cholera toxin alone showed an ovary /oviduct mean inflammation score of 4.25 (only 2 mice analyzed; 2 other died) versus 5.00 for the sl3 plus cholera toxin-immunized group, and 1.00 for the SWIB plus cholera toxin. Untreated infected animals had an ovary /oviduct mean inflammation score of 7. In the model of vaginal inoculation and ascending infection, negative control-immunized mice had an ovary/oviduct mean inflammation score of 7.37 versus 6.75 for the s13 plus cholera toxin-immunized group and 5.37 for the SWIB plus cholera toxin-immunized group. Untreated infected animals had an ovary /oviduct mean inflammation score of 8. The three experiments described above suggest that SWIB-specific protection is obtainable. This protective effect is more marked in the model of homologous infection but is still present when in a heterologous challenge infection with C. psittaci.<br>
2. CT529/Capl<br>
CT529/Capl was identified earlier as a product from Chlamydia that stimulates CD8+ CTL. In this example, we sought to confirm that immunization with Cap1 would be protective in an animal model of chlamydia infection.<br>
To generate recombinant vaccinia virus for delivery of a Capl immunogenic fragment, a DNA fragment containing a modified Kozak sequence and base pairs 319-530 of the capl gene (CT529) was amplified from C. trachomatis L2 genomic DNA using PCR™ and ligated into pSC1 1ss (Earl PL, Koenig S, Moss B (1991) Biological and immunological properties of human immunodeficiency virus type 1  envelope glycoprotein:  analysis   of proteins  with truncations  and  deletions  expressed  by recombinant vaccinia viruses. J Virol. 65:31-41).   DNA digested with Sal1 and Stul. The portion of the capl gene ligated into pSC11ss encodes amino acids 107-176 of Capl protein, containing the previously identified CTL epitope of amino acids 139-147. The resulting plasmid was used to transfect CV-1 cells (ATCC# CCL-70; Jensen FC et al. (1964) Infection of human and simian tissue cultures with Rous Sarcoma Virus. Proc. Natl. Acad. Sci. USA 52: 53-59.) which were subsequently infected with wild-type vaccinia virus. Homologous recombination between the wild-type virus and plasmid DNA generated recombinant vaccinia viruses which were selected on the basis of both beta-galactosidase expression and the inactivation of thymidine kinase, as described  previously   (Chakrabarti   et   al,   Mol   Cell   Biol.   1985,   5(12):3403-9). Recombinant virus was plaque purified three times and titered after growth in human TK-143B cells. Virus preparations were treated with equal volume of 0.25 mg/ml trypsin for 30 mins. at 37°C and diluted in PBS prior to immunization of mice. Groups of 5 mice were used for all experimental and control groups. The data presented below are representative of three independent experiments.<br>
A group of mice was immunized with 106 of the recombinant vaccinia i.p. and was allowed to recover for 3 weeks. Negative control groups were immunized with either buffer alone or wild-type vaccinia. As a positive control, a group of mice was infected i.v. with 105 i.f.u. of C. trachomatis. The number of organisms given to the positive control group has been previously shown to be cleared within 2 weeks. After 3 weeks, animals in each of the groups were challenged i.v. with 106 i.f.u. of C. trachomatis. Three days after challenge the mice were sacrificed and the number of i.f.u. per spleen was determined.<br><br>
The mean number of organisms found in the spleens of animals immunized with the vaccinia virus expressing Capl (7.1xl04) was 2.6-fold fewer (p
EXAMPLE 10 Pmp/Ra12 FUSION PROTEINS<br>
Various Pmp/Ra12 fusion constructs were generated by first synthesizing PCR fragments of a Pmp gene using primers containing a Not I restriction site. Each PCR fragment was then ligated into the NotI restriction site of pCRX1. The pCRX1 vector contains the 6HisRa12 portion of the fusion. The Ra12 portion of the fusion construct encodes a polypeptide corresponding to amino acid residues 192-323 of Mycobacterium tuberculosis MTB32A, as described in U.S. Patent Application 60/158,585, the disclosure of which is incorporated herein by reference. The correct orientation of each insert was determined by its restriction enzyme pattern and its sequence was verified. Multiple fusion constructs were made for PmpA, PmpB, PmpC, PmpF and PmpH, as described further below:<br>
PmpA Fusion Proteins<br>
PmpA is 107 kD protein containing 982 aa and was cloned from serovar E. The PmpA protein was divided into 2 overlapping fragments, the PmpA(N-terminal) and (C-terminal) portions.<br>
PmpA(N-term) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCATGTTTATAACAAAGGAACTTATG (SEQ ID NO: 306) GAGAGCGGCCGCTTACTTAGGTGAGAAGAAGGGAGTTTC  (SEQ ID NO: 307)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 308, encoding a 66 kD protein (619aa) expressing the segment 1-473 aa of PmpA. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 309.<br>
PmpA(C-term) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCCATTCTATTCATTTCTTTGATCCTG (SEQ ID NO: 310) GAGAGCGGCCGCTTAGAAGCCAACATAGCCTCC (SEQ ID NO: 311)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 312, encoding a 74 kD protein (691aa) expressing the segment 438-982 aa of PmpA. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 313.<br>
PmpF Fusion Proteins<br>
PmpF is 112 kD protein containing 1034 aa and was cloned from the serovar E. PmpF protein was divided into 2 overlapping fragments, the PmpF(N- term) and (C-term) portions.<br>
PmpF(N-term) was amplified by the sense and antisense primers:<br>
I<br>
GAGAGCGGCCGCTCATGATTAAAAGAACTTCTCTATCC (SEQ ID NO: 314)<br>
GAGAGCGGCCGCTTATAATTCTGCATCATCTTCTATGGC (SEQ ID NO: 315)<br>
respectively. The resulting fusion has a DNA sequence set forth in SEQ ID NO: 316, encoding a 69 kD protein (646aa) expressing the segment 1-499 aa of PmpF. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 317.<br>
PmpF(C-term) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGACATACGAACTCTGATGGG (SEQ ID NO: 318)<br>
GAGAGCGGCCGCTTAAAAGACCAGAGCTCCTCC (SEQ ID NO: 319)<br>
respectively. The resulting fusion has a DNA sequence set forth in SEQ ID NO: 320, encoding a 77 kD protem (715aa) expressing the segment 466-1034aa of PmpF. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 321.<br>
PmpH Fusion Proteins<br>
PmpH is 108 kD protein containing 1016 aa and was cloned from the serovar E. PmpH protein was divided into 2 overlapping fragments, the PmpH(N-term)and (C-term)portions.<br>
PmpH(N-term) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCATGCCTTTTTCTTTGAGATCTAC (SEQ ED NO: 322) GAGAGCGGCCGCTTACACAGATCCATTACCGGACTG (SEQ ID NO: 323)<br>
respectively. The resulting fusion has a DNA sequence set forth in SEQ ID NO: 324, encoding a 64 kD protein (631aa) expressing the segment 1-484 aa of PmpH. The amino acid sequence of the fusion protein is set forth in SEQ ED NO: 325. The donor line CHH037 was found to be reactive against this protein.<br>
PmpH(C-term) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGATCCTGTAGTACAAAATAATTCAGC (SEQ ID NO: 326) GAGAGCGGCCGCTTAAAAGATTCTATTCAAGCC (SEQ ID NO: 327)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 328, encoding a 77 kD protein (715aa) expressing the segment 449-1016aa of PmpH. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 329. The patient line CT12 was found to be reactive hi response to this protein.<br>
PmpB Fusion Proteins<br>
PmpB is 183 kD protein containing 1750 aa and was cloned from the serovar E. PmpB protein was divided into 4 overlapping fragments, PmpB(l), (2), (3) and (4).<br>
PmpB(l) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCATGAAATGGCTGTCAGCTACTGCG (SEQ ED NO: 330)<br>
GAGAGCGGCCGCTTACTTAATGCGAATTTCTTCAAG (SEQ ID NO: 331)<br>
respectively. The resulting fusion has a DNA sequence set forth in SEQ ED NO: 332, and encodes is a 53 kD protein (518aa) expressing the segment 1-372 aa of PmpB. The amino acid sequence of the fusion protein is set forth in SEQ ED NO: 333.<br>
PmpB(2) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGGTGACCTCTCAATTCAATCTTC (SEQ ID NO: 334)<br>
GAGAGCGGCCGCTTAGTTCTCTGTTACAGATAAGGAGAC (SEQ ID NO: 335)<br>
respectively. The resulting fusipn has a DNA sequence set forth in SEQ ID NO: 336 and 5    encodes a 60 kD protein (585aa) expressing the segment 330-767 aa of PmpB. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 337. Ceil lines derived from patient lines CT1, CT3, CT4 responded to this recombinant pmpB protein.<br>
PmpB(3) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGACCAACTGAATATCTCTGAGAAC (SEQ ID NO: 338)<br>
GAGCGGCCGCTTAAGAGACTACGTGGAGTTCTG (SEQ ID NO: 339)<br>
respectively. The resulting fusion has a DNA sequence set forth in SEQ ID NO: 340 encodes a 67 kD protein (654aa) expressing the segment 732-1236 aa of PmpB. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 341<br>
PmpB(4) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGGAACTATTGTGTTCTCTTCTG (SEQ ED NO: 342)<br>
GAGAGCGGCCGCTTAGAAGATCATGCGAGCACCGC (SEQ ID NO: 343)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 344 encodes a 76 kD protein (700aa) expressing the segment 1160-1750 of PmpB. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 345.<br>
PmpC Fusion Proteins<br>
PmpC is 187 kD protein containing 1774 aa and was cloned from the serovar E/L2. PmpC protein was divided into 3 overlapping fragments, PmpC(1), (2) and (3).<br>
PmpC(l) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCATGAAATTTATGTCAGCTACTGC (SEQ ID NO: 346)<br>
GAGAGCGGCCGCTTACCCTGTAATTCCAGTGATGGTC (SEQ ID NO: 347) respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 348 and encodes a 51 kD protein (487aa) expressing the segment 1-340 aa of PmpC. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 349.<br>
PmpC(2) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGATACACAAGTATCAGAATCACC (SEQ ID NO: 350)<br>
GAGAGCGGCCGCTTAAGAGGACGATGAGACACTCTCG (SEQ ID NO: 351)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 352 and encodes a 60 kD protein (583aa) expressing the segment 305-741 aa of PmpC. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 353.<br>
PmpC(3) was amplified by the sense and antisense primers:<br>
GAGAGCGGCCGCTCGATCAATCTAACGAAAACACAGACG (SEQ ID NO: 354)<br>
GAGAGCGGCCGCTTAGACCAAAGCTCCATCAGCAAC (SEQ ID NO: 355)<br>
respectively. The resulting fusion construct has a DNA sequence set forth in SEQ ID NO: 356 and encodes a 70 kD protein (683aa) expressing the segment 714-1250 aa of PmpC. The amino acid sequence of the fusion protein is set forth in SEQ ID NO: 357.<br>
EXAMPE 11<br>
IMMUNOGENICITY OF CT622<br>
Chlamydia-specific T cells lines were generated from two patients with Chlamydia infections and the lines were designated CT1 and CT13. The T cell lines were either generated against monocyte-derived dendritic cells infected C. trachomatis serovar E for 72 hours (CT1-ERB) or against killed serovar E elementary bodies (EB) (CT13-EEB). Once generated, the lines were tested against the recombinant Chlamydia-specific protein, CT622 in a proliferation assay. Proliferation assays were performed by stimulating 2.5x104 T cells in the presence of lxl04 monocyte-derived dendritic cells with either recombinant CT antigens (2μg/ml) or Chlamydia EBs (lμg/ml). The assay was incubated for 4 days with a 3H-thymidine pulse for the last 18 hours.<br>
The cell line GT1-ERB demonstrated proliferative responses significantly above the media controls when stimulated with CT622, CT875, and CT EB. The cell line CT13-EEB demonstrated a proliferative response significantly above media controls when stimulated with CT622, CT875, and CT EB (see Figure 12).<br>
EXAMPLE 12<br>
CLONING AND EXPRESSION OF FULL LENGTH CHLAMYDIA<br>
TRACHOMATIS GENES CT611, ORF3 AND QppA1<br>
Recombinant protein expression of the full-length open reading frames was performed for clones containing genes CT611, ORF-3, and oppA1. The clones that contained the genes of interest were CtL2-8 (SEQ ID NO:285) which encoded 4 ORFs (CT474, CT473, CT060, and CT139), CtL2-10 (SEQ ID NO:284) which encoded the ORFs of CT610 and CT611, and clones 16CtL2-16 (SEQ ID NO:47), 16-D4-22 (SEQ ID NO: 119) and 19-A5-54 (SEQ ID NO: 84) which all contained sequences related to ORF-3. Sequences within CfL2-10 (Ct-610) and CtL2-16 (ORF-3) were also independently identified by the T-cell expression cloning approach. The clone CtL2-8 was further investigated as this clone had stimulated the proliferative responses and IFN-gamma production by two T cell lines generated against serovar E.<br>
Cloning and expression of clone sequences:<br>
CtL2-10 was found to encode two open reading frames (ORFs), CT610 and CT611 and these were found organized adjacent to each other within the genomic clone. The full length ORF of CT610 (containing a PQQ synthesis domain) was previously expressed and demonstrated to stimulate the proliferative responses of T cell lines generated against Chlamydia. To determine whether the second ORF, CT611, was also recognized by T cells, the full-length sequence of CT611 was PCR amplified and engineered for protein expression. The nucleotide sequence is disclosed in SEQ ID NO:361 with the corresponding amino acid sequence disclosed in SEQ ID NO:365.<br>
! The second serological clone, CtL2-8, was found to contain 4 ORFs (CT474, CT473, CT060, and CT139). Overlapping peptides to the three smallest predicted ORFs (CT474, CT473, and CT060) did not stimulate the proliferative responses of   <br>
cell lines. This suggested that the immunostimulatory antigen resides in the fourth ORF, CT139. The ORF of CT139 is approximately 450 nucleotides. The full-length nucleotide sequence is disclosed in SEQ ID NO:359 and the full-length amino acid sequence is disclosed in SEQ ID NO:363. Amino acid sequence comparison from Genbank revealed that it is an oligo-peptide binding protein (oppA1) as well as belonging to the peptide ABC transporter family. This protein is 462 amino acids long with a predicted.size of 48.3kDa and appears to contain 2 trans-membrane regions.<br>
To express the full-length sequence of oppAl, oligonucleotides were designed which specifically amplified sequences starting from amino acid residue 22 (devoid of the first transmembrane domain), the nucleotide sequence for which is disclosed in SEQ ID NO:358 and, the amino acid sequence of which is disclosed in SEQ ID NO:362. This was shown to express the protein in E. coli.<br>
The full-length cloning and recombinant protein expression of ORF-3 was also achieved. The nucleotide and amino acid sequences are disclosed in SEQ ID NOs:360 and 364, respectively.<br>
EXAMPLE 13 RECOMBINANT CHLAMYDIAL ANTIGENS RECOGNIZED BY T CELL LINES<br>
Patient T cell lines were generated from the following donors: CT1, CT2, CT3, CT4, CT5, CT6, CT7, CT8, CT9, CT10, CT11, CT12, CT13, CT14, CT15, and CT16, some of which were discussed above. A summary of their details is included in Table V.<br><br><br><br><br><br>
NGU=Non-Gonococcal Urethritis; BV=Bacterial Vaginosis; CT=Chlamydia trachomatis; Cp=Chlamydia pneumoniae; Eb=Chlamydia elementary bodies; HPV=human papiloma virus; Dx=diagnosis; PID=pelvic inflammatory disease; LCR=Ligase chain reaction.<br>
PBMC were collected from a second series of donors and T cell lines have been generated from a sub-set of these. A summary of the details for three such T cell lines is listed in the table below.<br><br>
Table III: Normal Donors<br>
Donor	Gender	Age	CT IgG Titer	CP IgG Titer<br>
CHH011	F	49	1:64	1:16<br>
CHH037	F	22	0	0<br>
CHH042	F	25	0	1:16<br>
Donor CHH011 is a heathly 49 year old female donor sero-negaitve for C. trachomatis. PBMC produced higher quantities of IFN-gamma in response to C. trachomatis elementary bodies as compared to C. pneumoniae elementary bodies, indicating a C. trachomatis-specific response. Donor CHH037 is a 22 year old healthy  female donor sero-negative for C. trachomatis. PBMC poruced higher quantities of IFN-gamma in response to C. trachomatis elementary bodies as compared to C. pneumoniae elementary bodies, indicating a C. trachomatis-specific response. CHH042 is a 25 year old healthy female donor with an IgG titer of 1:16 to C. pneumoniae. PBMC produced higher quantities of IFN-gamma in response to C. trachomatis elementary bodies as compared to C. pneumoniae elementary bodies, indicating a C. trachomatis-specific response.<br>
Recombinant proteins for several Chlamydia trachomatis genes were generated as described above. Sequences for MOMP was derived from serovar F. The genes CT875, CT622, pmp-B-2, pmpA, and CT529 were derived from serovar E and sequences for the genes gro-EL, Swib, pmpD, pmpG, TSA, CT610, pmpC, pmpE, S13, IpdA, pmpI, and pmpH-C were derived from DDL<br>
Several of the patient and donor lines described above were tested against the recombinant Chlamydia proteins. Table IV summarizes the results of the T cell responses to these recombinant Chlamydia proteins.<br><br><br><br>
Although the present invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, changes and modifications can be carried out without departing from the scope of the invention which is intended to be limited only by the scope of the appended claims.<br><br>
We claim:<br>
1.	A composition comprising:<br>
(a)	a polypeptide comprising the sequence of Seq ID No: 431 or a variant thereof having at least 90% sequence identity thereto, as determined by comparing the two optimally aligned sequences over a window of comparison of the whole length of the sequence of Seq ID No: 431; or<br>
(b)	a polynucleotide encoding the polypeptide of (a) and physiologically acceptable carrier; for use in stimulating an immune response in a patient.<br>
2.	A composition comprising:<br>
(a) a polypeptide comprising the sequence of Seq ID No:431 or a variant thereof having at least 90% sequence identity thereto, as determined by comparing the two optimally aligned sequences over a window of comparison of the whole length of the sequence of Seq ID No:431; or<br>
(b)     a polynucleotide encoding the polypeptide of (a);<br>
for prevention of chlamydial infection.<br>
3.	A composition as claimed in either claim 1 or 2 wherein said composition, comprising a polypeptide which comprises the sequence of Seq ID No: 431 or a variant thereof having at least 90% sequence identity thereto, as determined by comparing the two optimally aligned sequences over a window of comparison of the whole length of the sequence of Seq ID No: 431.<br>
4.	A composition as claimed in claim 3, wherein said composition comprising a polypeptide which comprises the sequence of Seq ID No: 431.<br>
5.	A composition as claimed in any one of claims 1 to 4, wherein said composition comprising a polypeptide which is a fusion polypeptide.<br>
6.	A composition as claimed in any one of claims 1 to 4, wherein said composition comprising a polypeptide which consists of the sequence of Seq ID No: 431 or a variant thereof having at least 90% sequence identity thereto, as determined by comparing the two optimally aligned sequences over a window of comparison of the whole length of the sequence of Seq ID No: 431.<br>
7.	A composition as claimed in either claim 1 or 2, wherein said composition comprising a polynucleotide which comprises the sequence of Seq ID No: 407.<br><br>
8.	A composition as claimed in claim 7, wherein said composition comprising a polynucleotide which consists of the sequence of Seq ID No: 407.<br>
9.	A pharmaceutical composition comprising a composition according to any one of claims 1 to 8 and a physiologically acceptable carrier.<br>
10.	A vaccine comprising a composition as claimed in any one of claims 1<br>
to 8 and an immunostimulant.<br>
Dated this 22nd January , 2003.<br>
(DEEPAK KUMAR)<br>
OF REMFRY &amp; SAGAR<br>
ATTORNEY FOR THE APPUCANT(S)<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtY2FuY2VsbGVkIHBhZ2VzKDE3LTAzLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-cancelled pages(17-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgxNy0wMy0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-claims(granted)-(17-03-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtY2xhaW1zKGdyYW50ZWQpLSgxNy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-claims(granted)-(17-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoMTctMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-correspondence(17-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtY29ycmVzcG9uZGVuY2UoaXBvKS0oMjQtMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-correspondence(ipo)-(24-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAxKDAzLTAxLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 1(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAxKDIyLTAxLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 1(22-01-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAxMygwMy0wMS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 13(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAxOCgzMC0xMS0yMDA1KS5wZGY=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 18(30-11-2005).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgxNy0wMy0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 2(granted)-(17-03-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAyKGdyYW50ZWQpLSgxNy0wMy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 2(granted)-(17-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAzKDAzLTAxLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 3(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybSAzKDIwLTAxLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form 3(20-01-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybS1wY3QtaXBlYS00MDkoMjItMDEtMjAwMykucGRm" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form-pct-ipea-409(22-01-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtZm9ybS1wY3QtaXNhLTIxMCgyMi0wMS0yMDAzKS5wZGY=" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-form-pct-isa-210(22-01-2003).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzcoMDMtMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-petition under rule 137(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtcGV0aXRpb24gdW5kZXIgcnVsZSAxMzgoMDMtMDEtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-petition under rule 138(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtcG93ZXIgb2YgYXV0aG9yaXR5KDAzLTAxLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-power of authority(03-01-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwLW11bW5wLTIwMDMtcG93ZXIgb2YgYXV0aG9yaXR5KDAzLTAzLTIwMDMpLnBkZg==" target="_blank" style="word-wrap:break-word;">100-mumnp-2003-power of authority(03-03-2003).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="216985-bearing-assembly-for-rotatably-supporting-a-quill-in-the-housing-of-rolling-mill-laying-heads.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="216987-a-process-for-preparing-a-carboxylic-ester.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>216986</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>100/MUMNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Jan-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CORIXA CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CSC, THE UNITED STATES CORPORATION, 2711 CENTERVILLE ROAD, SUITE 400, WILMINGTON, DELAWARE - 19808</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STEVEN P. FLING</td>
											<td>11414 PINYON AVENUE N.E., BAINBRIDGE ISLAND, WA 98110, USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PETER PROBST</td>
											<td>137 N.W. 77TH STREET, SEATLE, WA 98117, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>AJAY BHATIA</td>
											<td>1805 BELLEVUE AV, # 204,SEATLE, WA 98104, USA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YASIR A. W.</td>
											<td>SHEIKY OF 15106 SE 47TH PLACE, BELLEVUE, WA 98006, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C 12 N 15/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US01/23121</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-07-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/841,321</td>
									<td>2001-04-23</td>
								    <td>Argentina</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/620,412</td>
									<td>2000-07-20</td>
								    <td>Argentina</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/216986-a-composition-comprising-a-polypeptide-having-the-sequence-of-seq-id-no-431-or-a-variant-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:15:16 GMT -->
</html>
