<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/217105-a-method-for-producing-an-enzyme-and-modifying-a-composition by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:23:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 217105:&quot;A METHOD FOR PRODUCING AN ENZYME AND MODIFYING A COMPOSITION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR PRODUCING AN ENZYME AND MODIFYING A COMPOSITION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A novel enzyme with a microbial origin which has an activity of cleaving disaccharide glycosides (in particular $g(b)-primeveroside and/or disaccharide glycosides similar thereto) in disaccharide units; and a gene encoding this enzyme. Various components can be produced by treating disaccharide glycosides or modified glycosides with this enzyme.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL ENZYME COMPOSITION AND PRODUCTION METHOD AND<br>
USE THEREOF<br>
TECHNICAL FIELD<br>
This invention relates to a novel microorganism-derived enzyme having an enzyme activity to cut disaccharide glycosides, particularly B-primeveroside and/or its analogs, in disaccharide unit, a method for producing said enzyme, a gene which encodes said enzyme, a vector containing said gene, a transformant transformed with said vector and use of said enzyme.<br>
BACKGROUND ART<br>
Alcoholic aromas such as geraniol, linalool, benzyl alcohol, 2-phenyl alcohol and C13-norterpenoid alcohol as plant aroma components take an important role in the aroma formation of, for example, flowers, tea, fruits and wine.<br>
Among these aroma components, monosaccharide glycosides such as ß-D-glucopyranoside have been isolated and identified as aroma precursors of benzyl alcohol and (Z)-3-hexenol.<br>
Recently, the presence of a disaccharide glycoside p-primeveroside (6-O-ß-D-xylopyranosyl-ß-D-glucopyranoside) or its analogs has been confirmed as precursors of fragrant alcohols such as geraniol and linalool, which seem to be<br><br>
taking an important role regarding the aroma of flowers. The presence of the disaccharide glycoside ß-primeveroside and its analogs has also been revealed as precursors of other alcoholic aroma components described above.<br>
In addition to such aromas, the presence of the disaccharide glycoside ß-primeveroside or its analogs has also been found in certain physiologically active substances such as pigments and pharmacological components. For example/ it is known that macrozamin in a cycad, etc. is cut by ß-primeverosidase in disaccharide unit to form a toxin.<br>
On the other hand, such an enzyme having a function to cleave precursors of these aroma components and physiologically active components in disaccharide unit has been confirmed only in a small amount in, for example, tea leaves (JP-A-8-140675; the term "JP-A" as used herein means an "unexamined published Japanese patent application"), and almost no study on its application has been carried out. Recently, it became apparent that the aglycon cannot be sufficiently released from these disaccahride glycoside and analogues thereof by the action of the known glucosidase. In consequence, great concern has been directed toward the development of a method by which such an enzyme can be produced on an industrial scale at low cost without<br>
--*■--£ -<br><br>
depending on the prior art supply sources such as tea leaves.<br>
DISCLOSURE OF THE INVENTION<br>
With the aim of solving the aforementioned problems, the inventors of the present invention have conducted intensive studies searching for its supply source in microorganisms.  That is, as a result of efforts to screen a microorganism capable of producing an enzyme having the aforementioned function from a broad range of natural sources, we have found a microorganism which is suitable for fermentation culturing and has the ability to produce an enzyme having the action to cut the saccharide moiety of disaccharide glycosides such as ß-primeveroside in disaccharide unit, and have isolated and purified said enzyme and determined nucleotide sequence of a gene that encodes said enzyme. Thereafter, we have found the enzyme having the aforementioned action in a large number of microorganisms such as molds, yeast, bacteria and actinomycetes and thereby accomplished this invention.<br>
Accordingly, the invention relates to a novel microorganism-derived enzyme having an enzyme activity to cut disaccharide glycosides, particularly ß-primeveroside and/or its analogs, in disaccharide unit, a method for producing said enzyme, a gene which encodes said enzyme, a<br><br>
vector containing said gene, a transformant transformed with said vector and use of said enzyme.<br>
The enzyme of the present invention is characterized in that it has an activity to release saccharides in a disaccharide unit from the disaccharide glycoside by acting on the disaccharide glycoside which can hardly be utilized as the substrate by the known glycosidase.  In the present specification, the enzyme having such activity is called "diglycosidase". The diglycosidase of the present invention has not only the activity to act on the disaccharide glycoside to release the saccharides in a disaccharide unit but also cut the glycoside bonding of the monoglycoside. Moreover, it has an activity to cut the glycoside bonding of the modified monoglycoside (e.g., acetylglucoside, malonylglucoside, methylglucoside, phosphoglucoside, and amidoglucoside).<br>
The microorganism-derived enzyme of the present invention which has the activity to act upon a disaccharide glycoside and thereby release saccharides in disaccharide unit from said disaccharide glycoside is different from the plant-derived enzyme in terms of physicochemical properties and homology of gene sequences.<br><br>
Accordingly, tho pre,9«=&gt;nt invention relates to -opr microorganism-derived enzyme having an acfeivTEy^to act upon a disaccharide glycoside to release--s"accharides from said disaccharide glycosidejUt-tJisaccharHHo unit.<br>
The pr««ent invention a1so relates to a polytF^LiJu which comprises a polypeptide having the amino acid---s"equence of SEQ ID NO: 8 shown in the Sequence ListjjagTwherein one or more amino acid residues of the amino acid sequence are modified by at least one of deletd.oif; addition, insertion and substitution, anjd^also having an activity to act upon a disaccharide glycoside to release saccharides from said<br>
d^rS^Cfhjirl de   g1yrr»sirf«   in   Hisaf!cJi3""""Jc   unit.<br>
The proseni- inv«nHon =lso r«i3teg to a polyiiucj-aetide which encodes a microorganismj^ejiived_-poIypeptide having an activity toagjt---«p6nradi saccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide<br>
ent invention also relates to a method for producing an enzyme having an activity to act upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit, which comprises the steps of:<br>
a)	culturing, under aerobic condition with a pH in range of 3-8, a microorganism of the kind as herein described in a nutrient medium of the kind as herein described to effect production of the enzyme having an activity to act upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside, and<br>
b)	subsequently collecting/isolating in the usual way by combination of centrifugation, UF concentrating and salting out said enzyme from the resulting culture mixture to produce the said enzyme.<br>
The present invention also relates to a method for producing an enzyme having an activity to act upon a<br><br>
disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit, which comprises the steps of:<br>
a)	culturing, under aerobic condition with a pH in range of 3-8, the transformant of the kind such as herein described to effect production of the enzyme having an activity to act upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside, and<br>
b)	subsequently collecting/isolating in the usual way by combination of centrifugation, UF concentrating and salting out said enzyme from the resulting culture mixture to produce the said enzyme.<br>
The present invention also relates to a method for producing modified composition of a material containing a disaccharide glycoside, which comprises allowing an enzyme having an action to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside to react with said material.<br>
The present invention also relates to a method for producing modified composition of a material containing a modified monoglycoside, which comprises allowing an enzyme having an action to release a modified monosaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside from said modified monoglycoside to react with said material.<br>
Next, the present is described in detail.<br>
In this connection, results of the measurement of various enzyme activities carried out in the present<br>
7<br><br>
invention are shown by values obtained by the following methods unless otherwise noted.<br>
(1) Disaccharide glycoside degradation activity<br>
Measurement of the activity was carried out using an automatic chemical analyzer (TBA-30R, manufactured by TOSHIBA CORP.) . A 30 µl portion of each enzyme sample was mixed with 200 µl of acetate buffer solution (pH 5.5) containing 2 mM of p-nitrophenyl (pNP) primeveroside as the disaccharide glycoside substrate to carry out the reaction at 40°C and at a cycle time of 22.5 seconds, for 9.75 minutes, and then the reaction solution was mixed with 250 µl  of sodium carbonate to measure absorbance at 412 run. Measurement of the sample blank was carried out in the same manner using 20 mM acetate buffer (pH 5.5) instead of the substrate solution.<br>
One unit of the enzyme activity is defined as the amount of enzyme which increases the absorbance by a factor of 1 under these conditions.<br>
The pNP-primeveroside used herein can be synthesized for example by allowing pNP-glucoside (manufactured by Merck) to react with xylo-oligosaccharide (manufactured by Wako Pure Chamical Industries) using an enzyme xylosidase (manufactured by Sigma), thereby effecting transfer of one xylose residue to pNP-glucoside through ß~l,6-bonding.<br><br>
(2) ß-Glucosidase activity<br>
Measurement of the activity was carried out using an automatic chemical analyzer (TBA-30R, manufactured by TOSHIBA CORP.) . A 10 µl portion of each enzyme sample was mixed with 200 µl of acetate buffer solution (pH 5.5) containing 2 mM of p-nitrophenyl (pNP) glucoside as the substrate to carry out the reaction at 40°C and at a cycle time of 22.5 seconds, and then the reaction solution was mixed with 250 µl  of sodium carbonate to measure absorbance at 412 nm. Measurement of the sample blank was carried out in the same manner using 20 mM acetate buffer (pH 5.5) instead of the substrate solution.<br>
One unit of the enzyme activity is defined as the amount of enzyme which increases the absorbance by a factor of 1 under these conditions.<br>
In order to obtain a microorganism capable of producing an enzyme having a diglycosidase activity, the present inventors have examined a broad range of natural sources and found that several microbial strains isolated from the natural world can produce an enzyme having said activity.  The disaccharide glycosides analogous to ß~ primeveroside are disaccharides having glucose on the aglycon side, such as apiofuranosyl-ß-D-glucopyranoside and arabinofuranosyl-ß-D-glucopyranoside.<br><br>
The diglycosidase producing microorganisms of the present invention can be screened, for example, in the following manner. That is, a soil sample solution is inoculated into a separation liquid medium containing eugenylprimeveroside or the like compound as the sole carbon source to carry out enrichment culturing, the resulting culture broth is spread on a separation agar plate medium having the same composition, and the thus grown colonies are selected and isolated. A strain having an activity to release pNP from bypassing disaccharide (e.g., pNP-primeveroside or the like) can be selected by culturing the thus isolated strains in an appropriate liquid medium.<br>
A diglycosidase producing microorganism can be screened from the thus selected strains using pNP-primeveroside or the like compound as the substrate and release of disaccharide as the index.<br>
Main strains isolated by the present inventors were identified by examining their mycological properties in the light of the following references (1) to (3). References:<br>
(1) Raper, K.B. and Fennel1, D.I., 1965. "The genus Aspergillus", Williams &amp; Wilkins, Baltimore.<br>
^§&gt;	 /O<br><br>
(2)	Kozakiewicz, Z., 1989. Aspergillus species on stored products. Mycological Papers, No. 161, CAB International Mycological Institute.<br>
(3)	Al-Musallam, A., 1980. "Revision of the black Aspergillus species", University of Utrecht.<br>
Mycological properties are described in the following.<br>
Identification of strain A (1) Growth condition Growth condition<br>
•	Czapek agar medium<br>
Colony size is 48 to 50 mm in diameter (25°C, 7 days), its surface is velutinous to powdery, hypha is white, formation of conidia is slightly poor, dull green to grayish green, backside is light yellowish brown to brown.<br>
•	Malt extract agar medium<br>
Colony size is 78 to 80 mm in diameter (25°C, 7 days), its surface is velutinous powdery, hypha is white, formation of conidia is markedly good, dull green to grayish green, backside is colorless to yellowish white. Colony size at 37°C (3 days) is 73 to 75 mm in diameter. Good growth even at 45°C.<br><br><br>
(2) Morphology  Conidial heads:<br>
Strong columnar form, 48 to 128 µm in length, 16 to 52 µm  in diameter, dull green to grayish green.<br>
•	Conidiophores:<br>
Forms from substrate mycelium, 125 to 800 µm in length (mostly 500 µm or less) , 5 to 10 µm in diameter, straight or slight bending, smooth surface.<br>
•	Vesicles:<br>
Diameter from 10 to 25 µm, flask shape, forms phialide in upper 2/3.<br>
•	Metulae:<br>
Not formed.<br>
•	Phialides:<br>
5.6 ~ 12 x 2.4 ~ 3.2 µm<br>
•	Conidia:<br>
Diameter from 2.6 to 3.6 µm, globose to subglobose, echinulate surface.<br>
•	Ascospores:<br>
Not formed.<br>
The above results show that the strain A belongs to the Aspergillus fumigatus group, because the conidium forming cells are single columnar (metula is not formed), the conidial head is cylindrical and dull green to grayish green, the conidia are globose and ascospore is not formed.<br>
^  \o-     IX<br><br>
In addition, since the conidial head is strong columnar and does not form nodding appearance, the conidia have echinulate surface and most of the conidiophores are 500 µm or less, this strain is Aspergillus fumigatus. Identification of strains B, C and D (1) Growth condition<br><br>
Table	1-1			<br>
Medium	Item	Strain B	Strain C	Strain D<br>
	Colony diameter (25°C, 7 days)	45 to 48 mm	47 to 50 mm	46 to 48 mm<br>
	Colony diameter	80 mm or	80 mm or	80 mm or<br>
	(25°C, 14 days)	more	more	more<br>
Czapek	Hyphae layer	dense, white	dense, white	dense, white<br>
agar		to yellow		<br>
medium	Formation of conidia	good	good	good<br>
	Color of conidia	dull grayish	dull grayish	dull grayish<br>
		brown to	brown to	brown to<br>
		black brown	black brown	black brown<br>
	Backside color	white to yellow	white	white<br>
	Colony diameter	46 to 51 mm	53 to 55 mm	55 to 59 mm<br>
Malt	(25°C)			<br>
extract	Hyphae layer	thin and	thin and	thin and<br>
agax		flat,	flat,	flat,<br>
medium		colorless	colorless	colorless<br>
	Formation of conidia	very good	very good	very good<br>
	Color of conidia	black to	black to	black to<br>
		black brown	black brown	black brown<br>
	Backside color	colorless	colorless f	colorless<br><br>
-MT-<br><br>
(3<br><br>
(2) Morphology (Czapek agar medium)<br><br>
Table 2					<br>
Item		Strain B	Strain C		Strain D<br>
Conidial	Shape	spherical,	spherical,	radial,	spherical,<br>
heads		radial,	sometimes i	split	radial,<br>
		sometimes split	into cylindrical	sometimes split<br>
		into cylindrical	form when matured	into cylindrical<br>
		form when matured			form when matured<br>
	Size	120 to 560 (im	150 to 500	(im	125 to 350 um<br>
	Color	dull grayish	dull grayish brown	dull grayish<br>
		brown to black	to black brown	brown to black<br>
		brown			brown<br>
Conidio-	Origin	forms from	forms from		forms from<br>
phores		substrate	substrate mycelium	substrate<br>
		mycelium			mycelium<br>
	Length	350 um to 3 mm	350 pm to 3 ran	350 (im to 2.5 mm<br>
	Diameter	9 to 20nm	10 to 22.5	Um	12.5 to 20 fun<br>
	Surface	smooth	smooth		smooth<br>
Vesicles	Diameter	20 to 80 |im	15 (mostly 80 |im	35) to	30 to 80 fim<br>
	Shape	globose	globose		globose<br>
	He tula<br>
formation<br>
region	entire	entire		entire<br>
Metulae	Length	20 to 24 jim	12 to 22.4	Um	12.8 to 24 Mm<br>
	Diameter	5.6 to 7.2 Jim	4.8 to 6.8	(jm	5.6 to 8 um<br>
	Shape	globose to	globose to		globose to<br>
		subglobose	subglobose		subglobose<br>
	Surface	echinulate	echinulate		echinulate<br>
Ascospore		not formed	not formed		not formed<br>
Based on the above results, all of the strains B, C and D belong to the Aspergillus niger  group, because the conidia forming cells are double columnar (metulae and phialides are formed) and the conidial heads are globose and blackish.  In addition, since the colony diameter becomes 5 cm or more by 14 days on the Czapek agar medium, the conidial surface is echinulate (verrucose), the conidium is globose to subglobose shape of 6 \xm  or less and<br><br>
dull grayish brown to black brown and the conidiophore is 6 \sm  or less, these are strains of Aspergillus niger  var. niger.<br>
The present inventors also have selected strains belonging to the genus Aspergillus  from type cultures at random and examined their ability to produce diglycosidase. As a result, productivity of the enzyme was also found, for example, in Aspergillus niger  IFO 4407, Aspergillus niger IAM 2020 and Aspergillus fumigatus  IAM 2046, etc.  In addition, screening of various other microorganisms was also carried out for their ability to produce diglycosidase. As a result, the diglycosidase activity was found in various microorganisms such as those belonging to the genus Aspergillus,   the genus Penicillium,   the genus Rhizopus,   the genus Rhizomucor,   the genus Talaromyces,   the genus Mortierella,   the genus Cryptococcus, the genus Microbacterium, the genus Corynebacterium  and the genus Actinoplanes.<br>
The strains which can be used in the present invention are not limited to the strains described above, and any strain having diglycosidase productivity can be used.  In addition, mutants of the strains having diglycosidase productivity, or various microorganisms or various cells (e.g., yeast cells, bacterial cells, higher plant cells and animal cells) modified by recombinant DNA<br><br>
techniques to have an ability to produce diglycosidase, in particular, preferably those modified to produce diglycosidase in great quantity are also included in the production method which can be used in the present invention. When diglycosidase productivity is added by introducing a diglycosidase gene, the host microorganism may not have the diglycosidase productivity.<br>
When diglycosidase is produced using the aforementioned various microorganisms, methods and conditions suited for the culturing of said microorganisms can be selected, and such methods and conditions are not particularly limited.  For example, culturing method of the aforementioned various strains may be either liquid culturing or solid culturing, but liquid culturing is preferably used. For example, the liquid culturing can be carried out in the following manner.<br>
Any type of medium can be used, provided that a diglycosidase producing microorganism can be grown therein. For example, a medium to be used may contain carbon sources such as glucose, sucrose, gentiobiose, soluble starch, glycerol, dextrin, molasses and organic acids, nitrogen sources such as ammonium sulfate, ammonium carbonate, ammonium phosphate, ammonium acetate, peptone, yeast extract, corn steep liquor, casein hydrolysate, wheat bran and meat extract and inorganic salts such as potassium<br><br>
salts, magnesium salts, sodium salts, phosphates, manganese salts, iron salts and zinc salts.  In addition, various inducers can be added to the medium in order to produce and accumulate diglycosidase. Examples of the inducers to be used include saccharides, preferably gentose (e.g., Gentose #80, Nihon Shokuhin Kako), gentiobiose and gentio-oligosaccharide (e.g., Gentio-oligosaccharide, Wako Pure Chemicals) . Amount of these inducers to be added is not particularly limited, with the proviso that the amount is effective in increasing the productivity of diglycosidase to an intended level, but is added preferably in an amount of from 0.01 to 5%.<br>
The medium pH is adjusted to a level of approximately from 3 to 8, preferably from about 5 to 6, and the culturing is carried out under aerobic conditions at a culturing temperature of generally from about 10 to 50°C, preferably at about 30°C, for a period of from 1 to 15 days, preferably from 4 to 7 days. Regarding the culturing method, shaking culture and aerobic submerged culture by a jar fermentor can be used.  However, the aforementioned various culture conditions are optionally changed depending on the microorganisms or cells to be cultured as a matter of course, and such conditions are not particularly limited with the proviso that the diglycosidase of the present invention can be produced.<br>
, &amp;	11-<br><br>
Regarding the isolation and purification of diglycosidase from the thus obtained culture broth, purified primeverosidase can be obtained in the usual way by a combination of centrifugation, UF concentration, salting out and various types of chromatography such as of an ion exchange resin.<br>
The culture of the aforementioned microorganism as it is can be used as the enzyme composition of the present invention. Of course, the culture may be purified to an appropriate degree of purification depending on the intended use of the present invention.<br>
The following further describes a gene which encodes a microorganism-derived enzyme of the present invention having the activity to act upon a disaccharide glycoside and thereby release saccharides from said disaccharide glycoside in disaccharide unit, a recombinant vector which contains said gene, a transformant into which said vector is introduced and a method for producing said enzyme using said transformant.<br>
As the microorganism-derived enzyme of the present invention having the activity to act upon a disaccharide glycoside and thereby release saccharides from said disaccharide glycoside in disaccharide unit, all of the enzymes which can be obtained by the aforementioned production methods are included, in which particularly<br><br>
preferred one is a polypeptide which has the amino acid sequence of SEQ ID NO: 8 shown in the Sequence Listing, wherein one or more amino acid residues of the amino acid sequence may be modified by at least one of deletion, addition, insertion and substitution, and more preferred one is a polypeptide which has the amino acid sequence of SEQ ID NO: 8 shown in the Sequence Listing.<br>
Examples of the gene which encodes the enzyme of the present invention include a gene which can be obtained from a microorganism capable of producing said enzyme by cloning of said gene and a gene which has a certain degree of homology with said gene. Regarding the homology, a gene having a homology of at least 50% or more, preferably a gene having a homology of 80% or more and more preferably a gene having a homology of 95% or more can be exemplified. The following polynucleotide (DNA or RNA) is desirable as the gene which encodes the enzyme of the present invention.<br>
A polynucleotide which comprises a polynucleotide<br>
being selected from the following polynucleotides (a) to<br>
(g) and encoding a polypeptide having the activity to act<br>
upon a disaccharide glycoside and thereby release<br>
saccharides from said disaccharide glycoside in<br>
disaccharide unit;	"<br><br><br>
(a)	a polynucleotide which encodes a polypeptide having the amino acid sequence of SEQ ID NO: 8 shown in the Sequence Listing,<br>
(b)	a polynucleotide which encodes a polypeptide having the amino acid sequence of SEQ ID NO: 8 shown in the Sequence Listing, wherein one or more amino acid residues of the amino acid sequence are modified by at least one of deletion, addition, insertion and substitution,<br>
(c)	a polynucleotide which has the nucleotide sequence of SEQ ID NO: 7 shown in the Sequence Listing,<br>
(d)	a polynucleotide which has the nucleotide sequence of SEQ ID NO: 7 shown in the Sequence Listing, wherein one or more bases of the nucleotide sequence are modified by at least one of deletion, addition, insertion and substitution,<br>
(e)	a gene which hybridizes with any one of the aforementioned polynucleotides (a) to (d) under a stringent condition,<br>
(f)	a polynucleotide which has homology with any one of the aforementioned polynucleotides (a) to (d), and<br>
(g)	a polynucleotide which is degenerate with respect to any one of the aforementioned polynucleotides (a) to<br>
(f) •<br>
The gene which encodes the enzyme of the present invention can be prepared from the aforementioned<br><br>
microorganism capable of producing the enzyme of the present invention by carrying out cloning of said gene in the following manner. Firstly, the enzyme of the present invention is isolated and purified from a microorganism capable of producing the enzyme of the present invention by the aforementioned method and information on its partial amino acid sequence is obtained.<br>
Regarding the determination method of a partial amino acid sequence, it is effective to carry out a method in which purified enzyme is directly applied to an amino acid sequence analyzer (such as Protein Sequenser 476A, manufactured by Applied Biosysterns) by Edman degradation method [J.   Biol.   Chem., vol. 256, pp. 7990 - 7997 (1981)], or a method in which limited hydrolysis of the enzyme is carried out using a protein hydrolase, the thus obtained peptide fragments are isolated and purified and then amino acid sequences of the thus purified peptide fragments are analyzed.<br>
Based on the information of the thus obtained partial amino acid sequences, a gene which encodes the enzyme of the present invention is cloned.  In general, the cloning is carried out making use of a PCR method or a hybridization method.<br>
When a hybridization method is used, the method described in "Molecular Cloning, A Laboratory Manual"<br><br><br>
(edit, by T. Maniatis et al., Cold Spring Harbor Laboratory, 1989) may be used.<br>
When a PCR method is used, the following method may be used.<br>
Firstly, a gene fragment of interest is obtained by carrying out PCR reaction using genomic DNA of a microorganism capable of producing the enzyme of the present invention as the template and synthetic oligonucleotide primers designed based on the information of partial amino acid sequences.  The PCR method is carried out in accordance with the method described in "PCR Technology" (edit, by Erlich H.A., Stockton Press, 1989). When nucleotide sequences of the thus amplified DNA fragments are determined by a usually used method such as the dideoxy chain termination method, a sequence which corresponds to the partial amino acid sequence of the enzyme of the present invention is found in the thus determined sequences, in addition to the sequences of synthetic oligonucleotide primers, so that a part of the enzyme gene of interest of the present invention can be obtained. As a matter of course, a gene which encodes complete enzyme of the present invention can be cloned by further carrying out a cloning method such as the hybridization method using the thus obtained gene fragment as a probe.<br>
-AS"-<br><br>
In the following Examples, a gene coding for the enzyme of the present invention was determined by the PCR method using Aspergillus fvmigatus  IAM 2046. Complete nucleotide sequence of the gene coding for the enzyme of the present invention originated from Aspergillus fumlgatus is shown in the SEQ ID NO: 7, and the amino acid sequence encoded thereby was determined to be the sequence shown in the SEQ ID NO: 8.  In this connection, there are countless nucleotide sequences which correspond to the amino acid sequence shown in the SEQ ID NO: 8, in addition to the nucleotide sequence shown in the SEQ ID NO: 8, and all of these sequences are included in the scope of the present invention.<br>
The gene of interest can also be obtained by chemical synthesis based on the information of the amino acid sequence shown in the SEQ ID NO: 8 and the nucleotide sequence shown in the SEQ ID NO: 7 (cf. Gene, 60(1), 115 -127 (1987)).<br>
Regarding the gene of the object enzyme of the present invention, a polynucleotide which encodes a polypeptide having the amino acid sequence of SEQ ID NO: 8, wherein one or more amino acid residues of the amino acid sequence are modified by at least one of deletion, addition, insertion and substitution, a gene which hybridizes with said polynucleotide under a stringent<br><br>
condition, a polynucleotide which has homology with said polynucleotide and a polynucleotide which is degenerate with respect to said polynucleotide are also included in the present invention, with the proviso that the polypeptides encoded thereby have the enzyme activity of the present invention.<br>
The term "under stringent condition" as used herein means, for example, the following condition.  That is, 6 x SSC, 1.0% blocking agent, 0.1% N-lauroylsarcosine sodium, 0.02% SDS.<br>
By using the entire portion or a part of the enzyme gene of the present invention, whose complete nucleotide sequence has been revealed making use of Aspergillus fxmigatus  IAM2046, as a probe for hybridization, DNA fragments having high homology with the enzyme gene of the present invention shown in SEQ ID NO: 7 can be selected from genomic DNA libraries or cDNA libraries of microorganisms capable of producing other enzymes of the present invention.<br>
The hybridization can be carried out under the aforementioned stringent condition.  For example,a. genomic DNA library or a cDNA library obtained from a microorganism capable of producing an enzyme of the present invention is fixed on nylon membranes, and the thus prepared nylon membranes are subjected to blocking at 65°C in a pre-<br>
^-c&gt;9 -<br><br>
hybridization solution containing 6 x SSC, 0.5% SDS, 5 x Denhart"s and 100 jig/ml of salmon sperm DNA.  Thereafter, each probe labeled with 32P or digoxigemn is added thereto, followed by incubation overnight at 68°C.  The thus treated nylon membranes are washed in € x SSC containing 0.1% SDS at room temperature for 10 minutes, in 6 x SSC  containing 0.1% SDS 45°C for 30 minutes and then    subsequently subjecting the thus washed membranes to an auto-radiography to detect a DNA fragment which hybridizes with the probe m a specific fashion. Also, genes which show various degree of homology can be obtained by changing certain conditions such as washing or lowering the hybridization temperature (e.g., 45°C).<br>
On the other hand, primers for use m the PCR reaction can be designed from the nucleotide sequence of the gene of the present invention. By carrying out the PCR reaction using these primers, gene fragments having high homology with the gene of the present invention can be detected and the complete gene can also be obtained.<br>
In order to determine whether the thus obtained gene encodes a polypeptide having the enzyme activity of interest, the thus determined nucleotide sequence is compared with the nucleotide sequence coding for the enzyme of the present invention or with its ammo acid sequence, and the identity is estimated based on the gene structure<br><br>
and homology. Alternatively, it is possible to determine<br>
whether the gene encodes a polypeptide which has the enzyme<br>
activity of interest by producing a polypeptide of the gene<br>
and measuring its enzyme activity.	<br>
The following method is convenient for producing a polypeptide having the enzyme activity of the present invention using the enzyme gene of the present invention.<br>
Firstly, transformation of a host is carried out using a vector containing the object gene of the enzyme of the present invention and then culturing of the thus obtained transformant is carried out under generally used conditions, thereby allowing the strain to produce a polypeptide having the enzyme activity of the present invention.<br>
Examples of the host to be used include microorganisms, animal cells and plant cells.  Examples of the microorganisms include bacteria such as Escherichia coli  and other bacteria belonging to the genera Bacillus, Streptomyces, and Lactococcus, yeasts such as those belonging to the genera Saccharomyces,   Pichia  and Kluyveromyces  and filamentous fungi such as those belonging to the genera Aspergillus, Penicillium,   Trichoderma  and Rhizopus.   Examples of the animal cells include those which unitize the baculovirus expression system.<br>
-
 <br>
Confirmation of the expression and expressed product can be made easily by the use of an antibody specific for the enzyme of the present invention, and the expression can also be confirmed by measuring the enzyme activity of the present invention.<br>
As described in the foregoing, purification of the enzyme of the present invention from the transformant culture medium can be carried out by optional combination of centrifugation, UF concentration, salting out and various types of chromatography such as of ion exchange resins.<br>
In addition, since the primary structure and gene structure of the enzyme of the present invention have been revealed by the present invention, it is possible to obtain a gene coding for the amino acid sequence wherein one or more amino acid residues of the amino acid sequence are modified by at least one of deletion, addition, insertion and substitution, by introducing random mutation or site-specific mutation using the gene of the present invention. This method renders possible preparation of a gene coding for an enzyme of the present invention which has the enzyme activity of the present invention but its properties such as optimum temperature, temperature stability, optimum pH, pH stability and substrate specificity are slightly changed, and it also renders possible production of such<br><br>
enzymes of the present invention by means of genetic engineering techniques.<br>
Examples of the method for introducing random mutation include a chemical DNA modification method in which a transition mutation is induced to convert cytosine base into uracil base by the action of sodium hydrogensulfite [Proceedings of the National Academy of Sciences of the USA,   vol. 79, pp. 1408 - 1412 (1982)], a biochemical method in which base substitution is induced during the step of double strand formation in the presence of [a-S] dNTP [Gene,   vol. 64, pp. 313 - 319 (1988)] and a PCR method in which PCR is carried out by adding manganese to the reaction system to decrease fidelity of the nucleotide incorporation [Analytical Biochemistry, vol. 224, pp. 347 - 353 (1995)].<br>
Examples of the method for introducing site-specific mutation include a method in which amber mutation is employed [gapped duplex method; Nucleic Acids Research, vol. 12, no. 24, pp. 9441 - 9456 (1984)], a method in which recognition sites of restriction enzymes are used [Analytical Biochemistry,   vol. 200, pp. 81 - 88 (1992); Gene,   vol. 102, pp. 67 - 70 (1991)], a method in which mutation of dut (dUTPase) and ung (uracil DNA glycosylase) is used [Kunkel method; Proceedings  of the National Academy of Sciences  of the  USA,   vol. 82, pp. 488 - 492 (1985)], a<br>
-JZ5--<br><br>
method in which amber mutation is induced using DNA polymerase and DNA ligase [oligonucleotide-directed dual amber (ODA) method: Gene,   vol. 152, pp. 271 - 275 (1995); JP-A-7-289262], a method in which a host introduced with a DNA repair system is used (JP-A-8-70874), a method in which a protein capable of catalyzing DNA chain exchange reaction is used (JP-A-8-140685), a method in which PCR is carried out using two different primers for mutation use to which recognition sites of restriction enzymes are added (U.S. Patent 5,512,463), a method in which PCR is carried out using a double-stranded DNA vector having an inactivated drug resistance gene and two different primers [Gene,   vol. 103, pp. 73 - 77 (1991)] and a method in which PCR is carried out making use of amber mutation (WO 98/02535).<br>
Also, site-specific mutation can be introduced easily by the use of commercially available kits. Examples of such kits include Mutan*-G (manufactured by Takara Shuzo) in which the gapped duplex method is used, Mutan*-K (manufactured by Takara Shuzo) in which the Kunkel method is used, Mutan*-Express Km (manufactured by Takara Shuzo) in which the ODA method is used and QuickChange* Site-Directed Mutagenesis Kit (manufactured by STRATAGENE) in which primers for mutation use and Pyrococcus furiosus  DNA polymerase are used, as well as TaKaRa LA PCR in vitro Mutagenesis Kit (manufactured by Takara Shuzo) and Mutan*-<br><br>
Super Express Km (manufactured by Takara Shuzo) as kits in which PCR is used.<br>
Thus, the primary structure and gene structure of the enzyme of the present invention provided by the present invention render possible production of an inexpensive and high purity polypeptide having the enzyme activity of the present invention by means of genetic engineering techniques.<br>
In this connection, various literature and references are cited in the specification, and all of them are incorporated herein by references.<br>
Next, various applications of the enzyme composition of the present invention are described.<br>
Diglycosidase can be used for the improvement of various components such as aromas, colors and physiologically active contents of plant materials and for adjusting extraction efficiency of these components.  In consequence, it can be used in the production of food and drinks having increased aromas and of spices, perfumes and liquid scents having increased aromas, and it also can be used for the early stage release of unfavorable odor by optionally using it during a step of the just described productions. Regarding the colors, it can be used for the improvement of hues of plant materials, food and drinks, development of colors and production of pigments.<br>
-J2PC -<br><br>
In addition, similar to the case of aromatic components, it can be used for the degradation and removal of pigment precursors which are not desirable in view of qualities, and regarding the physiological activities, it can be used for the increase of pharmacological components and useful physiologically active components of crude drugs, herbs and other plant components or degradation and removal of undesirable components.<br>
That is, it is possible to produce the aforementioned actions by allowing the diglycosidase of the present invention to act upon various disaccharide glycoside components.<br>
In addition, the diglycosidase of the present invention may be administered with the aforementioned physiologically active substance, etc. after mixing or without mixing but by simultaneously or with a short interval administration, in order for the physiologically active substance to be absorbed efficiently into the body, etc.<br>
Examples of the materials containing disaccharide glycosides to be treated by the present invention include those which undergo the action of diglycosidase, such as foods, cosmetics, medicaments, quasi drugs, agricultural chemicals and feeds, more illustratively, it can also be applied to the production of industrial products having<br><br>
various aromas, such as foods, toiletries, woodworks and mats produced from plant materials.<br>
Food articles having aromatic components can be exemplified as materials to which the diglycosidase of the present invention is preferably applied. As illustrative examples, it can be used in the so-called "wilting" step during the production of oolong tea and jasmine tea and for the improvement of aromas of black tea (for tea pack by CTC method) and wine.  It can also be used for the maintenance of aromas of cosmetics and liquid scents and improvement of aromas and pharmacological effects of medicaments.<br>
The diglycosidase of the present invention is also useful in the production of pigments. For example, extraction of alizarine dye from Rubia  tinctoxrvun  L. ruberythric acid can be carried out more efficiently than the conventional method by the use of the enzyme.<br>
Also, it is possible to produce precursors of disaccharide components such as an aroma, a pigment, a physiologically active component and primeverose making use of the action of diglycosidase.  Improvement of the stability and keeping quality of these components, their detoxication and modification of pharmacological components for DDS can be expected by their glycosylation.<br>
In addition, diglycosidase can degrade modified glucosides such as acetylglucoside, malonylglucoside,<br>
-v-2«r-<br><br>
methylglucoside, phosphoglucoside, and amidoglucoside which can hardly be utilized by glucosidase as its substrate more efficiently than known glucosidase. Making use of this property, absorption and yield of isoflavone contained in soybean can be improved by converting acetylglucoside and malonylglucoside of isoflavone into their aglycon forms.<br>
The enzyme solution of the present invention may be sprayed to the cut flowers or may be abosorbed by the cut flowers to enhance the aroma of flowers.<br>
Regarding application methods of diglycosidase, its adding method, adding amount, reaction method and the like can be changed at will depending on the conditions of material to be treated.<br>
Regarding an illustrative application method, the diglycosidase of the present invention is added to a plant extract or fermentation product containing an aroma precursor, and the mixture is incubated.  The reaction conditions are not particularly limited, with the proviso that the diglycosidase of the present invention can act upon the precursor of an aroma, pigment or physiologically active component to release the aroma, pigment or physiologically active component, and such conditions can be set by those skilled in the art without undue efforts. Under such conditions, concentration of said component can be increased.<br><br>
Also, the enzyme of the present invention can be used for increasing concentration of an aroma, pigment or physiologically active component which is present in plants. That is, since plants contain precursors of these components, an aroma, pigment or physiologically active component in a plant can be increased by cultivating the plant with adding an effective amount of the diglycosidase of the present invention (including transgenic method) under such conditions that the precursor in said plant can be hydrolyzed.  In addition, the formation period of an aroma, pigment or physiologically active component can be controlled making use of the enzyme composition of the present invention.<br>
In this connection, it is possible to synthesize various types of glycoside by making use of reverse reaction of the diglycosidase of the invention. Best Mode for Carrying Out the Invention<br>
The invention is described further in detail in the following with reference to examples but, as a matter of course, the invention is not limited to the following examples without departing from its scope. Unless otherwise noted, the term % as used herein means w/v %. Example 1<br>
Each of AspBrgillus niger  IFO 4407 and Aspergillus niger  IAM 2020 was cultured overnight at 30°C on a shaker<br>
-j&amp;sr-	"<br><br>
in a pre-culture medium (composition; 0.2% yeast extract, 0.5% peptone, 2% glucose, 0.1% KH2P04, 0.05% MgS04-7H20, pH 5.7), the resulting culture broth in an amount of 1/100 was inoculated into a mam culture medium (4% soybean flour, 0.3% sodium chloride, 0.1% KH2P04, 0.05% MgS04-7H20, 2% soluble starch, 1% red bran, pH 5.6), and cultured for 6 days on a shaker, and then the cells were removed from the culture broth to obtain a crude enzyme solution.  Using this enzyme solution, diglycosidase activity and ß~ glucosidase activity were measured.<br>
As the result, the diglycosidase activity and (5-glucosidase activity m the strain IFO 4407 were 0.129 unit/ml and 4.34 units/ml, respectively, and the diglycosidase activity and (3-glucosidase activity in the strain IAM 2020 were 0.156 unit/ml and 5.97 units/ml, respectively. Example 2<br>
In accordance with Example 1, Aspergillus fxanigatus IAM 2046 was pre-cultured in the same manner, and the resulting culture broth was inoculated into a mam culture medium (2% soybean flour, 0.3% sodium chloride, 0.1% KH2P04, 0.05% MgS04-7H20, 3% soluble starch, 0.5% gentose<br>
(mfd. by Nihon Shokuhm Kako), pH 5.6) and cultured for#80 4 days to obtain a crude enzyme solution.  As the result, the<br>
-jaer-<br><br>
diglycosidase activity was 0.106 unit/ml and p-glucosidase activity was 0.320 unit/ml. Example 3<br>
Using Aspergillus fumlgatus  IAM 2046, effects of inducers on the production of diglycosidase were examined. Aspergillus fumlgatus  IAM 2046 was cultured for 6 days in a culture medium (2% soybean flour, 0.3% sodium chloride, 0.1% KH2PO4, 0.05% MgS04-7H20, 3% soluble starch) supplemented with 0.1% of each of various saccharides, and the diglycosidase activity was measured.  The results are shown in Table 3.<br><br>
Table 3				<br>
Inducers			Inducing	r ability (%)<br>
Not added				100<br>
Isomaltose				145<br>
Maltotriose				171<br>
Maltose				136<br>
Gentose #8D				235<br>
Gentiobiose				211<br>
Gentio-oligosaccharide			180<br>
Sucrose				116<br>
Trehalose				113<br>
Glucose				164<br>
Galactose				125<br>
Fructose				143<br>
Rhamnose				129<br>
Tulbose				116<br>
Maltitol				142<br>
Arabitol				112<br>
Galactitol				142<br>
Glucosamine hydrc	&gt;chlori	.de	2c	157<br><br>
As is evident from the above table, the producing ability of diglycosidase was increased by various saccharides. Particularly, markedly high inducing ability was found in gentose, gentiobiose and gentio-oligosaccharide.<br>
In addition, similar effects were also found when Aspergillus niger  IFO 4407 or Aspergillus niger  IAM 2020 was used. Example 4<br>
Each of the crude enzyme solutions obtained in Examples 1 and 2 was concentrated using an ultrafiltration membrane having a molecular weight cutoff of 6,000. Next, 1 ml of the concentrated solution was mixed with 1 ml of a 5 mg/ml pNP-primeveroside solution which had been prepared using 20 mM phosphate buffer (pH 6.0), and the mixture was incubated at 37°C. Samples were collected 1, 2, 4, 24 and 48 hours thereafter to confirm release of primeverose by a thin layer chromatography (TLC).<br>
As a result, a spot was observed by TLC at the same position of a disaccharide primeverose in all of the culture media of two Aspergillus niger  strains described in Example 1 and Aspergillus fumigatus described in Example 2. Such a spot was not observed in samples in which the crude enzyme concentration solutions were subjected to the same test after their heat treatment (100°C for 10 minutes).<br>
--34"-<br><br>
Thus, the presence of an enzyme capable of releasing<br>
primeverose in disaccharide unit from pNP-primeveroside was<br>
found in the crude enzyme concentration solutions.<br>
Example 5<br>
Screening of diglycosidase in various microorganisms<br>
a) Preparation of enzyme samples<br>
Each microorganism to be subjected to screening was pre-cultured and main cultured.  In the case of liquid culturing, the obtained culture broth was centrifuged at 10,000 min"1 for 10 minutes and the resulting supernatant was used as the enzyme sample.  In the case of solid culturing, the medium after completion of the culturing was extracted with water and the resulting extract was used as the enzyme sample.<br>
In this connection, intracellular enzyme was also examined in the case of bacteria.  In that case, the culture broth was centrifuged, and the thus obtained cells as the precipitate were washed with physiological saline, suspended in 10 times amount of 10 mM phosphate buffer (pH 7.0) based on the cell weight and then treated with ultrasonic wave to disrupt the cells.  The disrupted suspension was centrifuged at 12,000 min"1 for 20 minutes and the resulting supernatant was used as the intracellular enzyme sample.<br>
- -35"-<br><br>
Media and culture conditions regarding the culturing<br>
are shown in Tables 1 to 5.<br>
Table 1 Liquid culturing of mold and yeast<br>
Pre-culture: Medium composition<br>
Yeast extract (DIFCO)	0.2%<br>
Peptone (DIFCO)	0.5%<br>
Glucose (Katayama Chemical)	2.0%<br>
Potassium dihydrogenphosphate<br>
(Kanto Chemical)	0.1%<br>
Magnesium sulfate heptahydrate<br>
(Katayama Chemical)	0.05%<br>
The above composition was dissolved in purified water and adjusted to pH 5.7 with 1 M hydrochloric acid and 1 M sodium hydroxide.  The medium was dispensed in 100 ml portions into Sakaguchi flasks and then sterilized at 121°C for 20 minutes under 1 atmospheric pressure.<br>
Pre-culture: Culture conditions<br>
The culturing was carried out at a shaking speed of 140 min"1, with one loopful inoculum from a slant culture, for 1 day or more and at 30°C. Regarding yeast strains, the temperature condition was set to 25°C.<br>
Main culture: Medium composition<br>
Soya flower A (Nisshin)	2.0%<br>
Sodium chloride (Kanto Chemical)	0.3%<br>
Dipotassium hydrogenphosphate<br>
(Kanto Chemical)	0.1%<br><br><br>
Magnesium sulfate heptahydrate<br>
(Katayama Chemical)	0.05%<br>
Soluble starch (Wako Pure Chemical)      3.0%<br>
Gentose #80 (Nihon Shokuhin Kako)        0.5%<br>
The above composition was dissolved in purified water and adjusted to pH 5.6 with 1 M hydrochloric acid and 1 M sodium hydroxide.  The medium was dispensed in 100 ml portions into Sakaguchi flasks and then sterilized at 121°C for 20 minutes under 1 atmospheric pressure.<br>
The culturing was carried out using the medium both in the presence and absence of gentose #80.<br>
Main culture: Culture conditions<br>
The culturing was carried out at a shaking speed of 140 min"1, with 1 ml inoculum from the pre-culture broth, for 5 days and at 30°C. Regarding yeast strains, the temperature condition was set to 25°C. Table 2 Solid culturing of mold and yeast<br>
Pre-culture: Medium composition<br>
High starch bran<br>
(B Ohgi, Nippon Flower Milling)  8.3%<br>
The above composition was suspended in purified water, and the medium was dispensed in 9 ml portions into culture test tubes and then sterilized at 121°C for 20 minutes under one atmospheric pressure.<br>
Pre-culture:  Culture conditions<br><br>
The culturing was carried out at a shaking speed of 300 min-1, with one loopful inoculum from a slant culture, for 1 to 2 days and at 30°C.<br>
Main culture: Medium composition<br>
A 5.0 g portion of bran was suspended in 1.5 ml of purified water, dispensed into 100 ml capacity conical flasks and then sterilized at 121°C for 20 minutes under one atmospheric pressure.<br>
Main culture: Culture conditions<br>
The culturing was carried out with 1 ml inoculum from the pre-culture broth, for 3 days and at 30°C.<br>
Extraction<br>
By adding 90 ml of tap water, extracted overnight at 8°C or below. Table 3 Culturing of bacteria and actinomycetes<br>
Pre-culture: Medium composition<br>
Tryptic soy broth (DIFCO)<br>
BACTO Tryptone	1.7%    "&gt;<br>
BACTO Soytone	0.3%<br>
BACTO Dextrose	0.2%     ypH 7.3 + 0.2<br>
Sodium chloride        0.5%<br>
Dipotassium<br>
hydrogenphosphate      0.25%   J<br>
The above composition was dissolved in purified water, and the medium was dispensed in 100 ml portions into<br>
— -3o —<br><br>
Sakaguchi flasks and then sterilized at 121°C for 20 minutes under one atmospheric pressure. Pre-culture: Culture conditions<br>
The culturing was carried out at a shaking speed of 140 min-1, with one loopful inoculum from a slant culture, for 1 day or more and at 30°C.<br>
Main culture: Medium composition<br>
Polypeptone (Japan Pharmaceutical)       1.0%<br>
Yeast extract (DIFCO)	0.25%<br>
Ammonium sulfate (Wako Pure Chemical)    0.1% Dipotassium hydrogenphosphate<br>
(Kanto Chemical)	0.05%<br>
Magnesium sulfate heptahydrate<br>
(Katayama Chemical)	0.025%<br>
Calcium chloride (Wako Pure Chemical)    0.0001%<br>
Adekanol LG126 (Asahi Denka)	0.001%<br>
Gentose #80 (Ninon Shokuhin Kako)        0.5% The above composition was dissolved in purified water and adjusted to pH 7.0 with 1 M hydrochloric acid and 1 M sodium hydroxide.  The medium was dispensed in 100 ml portions into Sakaguchi flasks and then sterilized at 121°C for 20 minutes under one atmospheric pressure.<br>
The culturing was carried out using the medium both in the presence and absence of gentose #80.<br>
- 2?T-<br><br>
Main culture: Culture conditions<br>
The culturing was carried out at a shaking speed of 140 nun"1, with 1 ml inoculum from the pre-culture broth, for 5 days and at 30°C. Table 4 Culturing of Penicilllvm multicolor<br>
Pre-culture: Medium composition<br>
Defatted soybean "Soypro" (Hohnen Oil)	2.0%<br>
Glucose (Katayama Chemical)	3.0%<br>
Potassium dihydrogenphosphate<br>
(Kanto Chemical)	0.5%<br>
Ammonium sulfate (Wako Pure Chemical)	0.4%<br>
Dry yeast	0.3%<br>
Adekanol (Asahi Denka)	0.05%<br>
The above composition was dissolved in purified<br>
water, and the medium was dispensed in 100 ml portions into<br>
Sakaguchi flasks and then sterilized at 121°C for 20<br>
minutes under one atmospheric pressure. Pre-culture:  Culture conditions The culturing was carried out at a shaking speed of<br>
140 min-1, with one loopful inoculum from a slant culture,<br>
for 5 days or more and at 27°C.<br>
Main culture: Medium composition<br>
Gentose #80 (Nihon Shokuhin Kako)	3.0%<br>
Potassium dihydrogenphosphate<br>
(Kanto Chemical)	2.0%<br>
Ammonium sulfate (Wako Pure Chemical)	1.0%<br><br>
Meast PIG (Asahi Beer Food)	3.13%<br>
Adekanol LG126 (Asahi Denka)	0.05%<br>
The above composition was dissolved in purified water, and the medium was dispensed in 100 ml portions into Sakaguchi flasks and then sterilized at 121°C for 20 minutes under one atmospheric pressure. Main culture: Culture conditions<br>
The culturing was carried out at a shaking speed of 140 min-1, with 1 ml inoculum from the pre-culture broth, for 6 days and at 27°C.<br>
Table 5 Culturing of the genus CaryneJbacterium<br>
Pre-culture: Medium composition<br>
Glucose	0.2%<br>
Yeast extract	0.1%<br>
Ammonium nitrate	0.4%<br>
Potassium dihydrogenphosphate	0.15%<br>
Sodium hydrogenphosphate dodecahydrate   0.15%<br>
Magnesium sulfate heptahydrate	0.02%<br>
Ferrous sulfate heptahydrate	0.0001%<br>
Calcium chloride dihydrate	0.001%<br>
The above composition was dissolved in purified water and adjusted to pH 7.0 with 1 M hydrochloric acid and 1 M sodium hydroxide.  The medium was dispensed in 10 ml portions into culture test tubes and then sterilized at 121°C for 20 minutes under one atmospheric pressure.<br><br>
Pre-culture: Culture conditions<br>
The culturing was carried out at a shaking speed of<br>
300 min"1, with one loopful inoculum from a slant culture,<br>
for 2 days and at 30°C.<br>
Main culture: Medium composition<br>
Eugenyl-ß-primeveroside	0.2%<br>
Yeast extract	0.1%<br>
Ammonium nitrate	0.4%<br>
Potassium dihydrogenphosphate	0.15%<br>
Sodium hydrogenphosphate dodecahydrate   0.15%<br>
Magnesium sulfate heptahydrate	0.02%<br>
Ferrous sulfate heptahydrate	0.0001%<br>
Calcium chloride dihydrate	0.001%<br>
The above composition was dissolved in purified water<br>
and adjusted to pH 7.0 with 1 M hydrochloric acid and 1 M<br>
sodium hydroxide.  The medium was dispensed in 10 ml<br>
portions into culture test tubes and then sterilized at<br>
121°C for 20 minutes under one atmospheric pressure. Main culture: Culture conditions The culturing was carried out at a shaking speed of<br>
140 min-1, with 1 ml inoculum from the pre-culture broth,<br>
for 1 day and at 30°C.<br>
b) Preparation of substrate solution<br>
pNP-p-primeveroside was dissolved in 20 mM acetate<br>
buffer (pH 5.5) to a concentration of 5 mg/ml and used as a<br>
-JET-<br><br>
substrate solution A. Eugenyl-ß-primeveroside was dissolved in 20 mM acetate buffer (pH 5.5) to a concentration of 10 mg/ml and used as a substrate solution B.<br>
c)	Enzyme reaction<br>
A 100 µl portion of the substrate solution A was put into a micro-centrifugation tube and mixed with 100 |il of each enzyme sample to carry out 96 hours of the enzyme reaction in a water bath of 37°C. When the reaction reached the intended time, the reaction solution was treated at 100°C for 10 minutes to stop the enzyme reaction.  This was used as the enzyme reaction-completed solution.<br>
As a comparative control of the enzyme sample, the same enzyme sample was treated at 100°C for 10 minutes before the enzyme reaction and subjected to the same reaction.  The enzyme reaction was carried out on each of the substrate solutions A and B. When the substrate solution B was used, the reaction was carried out in the same manner as the case of the substrate solution A.<br>
d)	Thin layer chromatography<br>
20 µl of the enzyme reaction-completed solution was spotted on a thin layer of silica gel (Silica gel 60 F254 [1.05554], Merck) and dried.  This was developed twice with a developing solvent prepared by mixing ethyl acetate,<br><br>
acetic acid and purified water at a ratio of 3:1:1. After completion of the development, the thin layer was air-dried. Thereafter, a color developing reagent prepared by mixing sulfuric acid and methanol at a ratio of 20:80 was sprayed all over the thin layer after completion of the development, and the color was developed at 105°C for about 10 minutes.<br>
An enzyme sample by which the spot of primeverose was appeared on the thin layer after the color development was judged that the primeverosidase was present therein, and this producer strain was judged as a diglycosidase producing strain.<br>
e) Results of the screening of diglycosidase producing strains<br>
A summary of the diglycosidase producing strains found by the above evaluation method is shown in Table 6.<br><br><br>
Table 6 Strains in which the diglycosidase production was<br>
found<br>
Microorganisms Strain names<br>
Mold	Aspergillus oryzae  IAM 2769<br>
Aspergillus niger IAM 2020 IFO 4091 IFO 9455 IAM 2107<br>
Aspergillus aculeatus Penicillium rugolosum  IFO 7242 Penicillium lilacinum IFO 5350 Penicillium decnmbence  IFO 31297 Penicillium multicolor  IAM 7153 Rhizopus oryzae  JCM 5560 Rhizomucor pusillus  IAM 6122 Rhizomucor miehei  IFO 9740 Talaromyces emersonii  IFO 9747 Mortierella  vinacea  IFO 7875<br>
Yeast	Cryptococcus albidus  IAM 12205<br>
Bacteria       Microbacterium arborescens  JCM 5884<br>
Corynebacterium ammoniagenes IFO 12072 Corynebacterium ammoniagenes IFO 12612 Corynebacterium glutamicum  IFO 1318<br>
Actinomycetes Actinoplanes missouriensis  JCM 3121<br>
Example 6<br>
Purification  of diglycosidase derived from Aspergillus<br>
fumigatus<br>
As the pre-culture, Aspergillus fumigatus  IAM 2046 was inoculated into a glucose-peptone medium (0.2% yeast extract, 0.5% peptone, 2% glucose, 0.1% potassium dihydrogenphosphate, 0.05% magnesium sulfate, pH 5.7) and cultured at 30°C for 24 hours on a shaker.  The pre-culture<br><br>
broth was inoculated in an inoculum size of 1% into the main culture medium (2% Soya flower, 0.3% sodium chloride, 0.1% dipotassium hydrogenphosphate, 0.05% magnesium sulfate, 3% soluble starch, 1% gentose #80, pH 5.6) and cultured at 30°C for 6 days on a shaker.<br>
Cells were removed from the culture broth by filter paper filtration, and 8,600 ml of the resulting filtrate was concentrated to 710 ml using an ultrafiltration membrane of 6,000 molecular weight cutoff (AIP-1010, mfd. by Asahi Chemical Industry) . A 200 ml portion of the concentrated solution was centrifuged at 4°C and at 15,000 rpm for 10 minutes, and 192 ml of the supernatant was mixed with 55.9 g of ammonium sulfate (50% saturation) and stirred overnight at 4°C.  This was centrifuged at 4°C and at 15,000 rpm for 10 minutes, and the thus obtained precipitate was dissolved in 10 ml of 20% saturation ammonium sulfate/20 mM phosphate buffer (pH 6.0) and centrifuged at 4°C and at 15,000 rpm for 10 minutes to recover the supernatant. A 9.5 ml portion of the supernatant was applied to a Phenyl Sepharose column (16 x 100 mm, mfd. by Pharmacia) which had been equilibrated with 20% saturation ammonium sulfate/20 mM phosphate buffer (pH 6.0), and the adsorbed protein was released by an ammonium sulfate linear density gradient of from 20% to 0%.  The active peaks were recovered, buffer-exchanged to 25 mM<br>
un^ -^R<br>
-J&amp;r--<br><br>
triethanolamine buffer (pH 8.3) using 10DG column (mfd. by BIO-RAD), applied to an anion exchange Mono-P column (5 x 200 mm, mfd. by Pharmacia) which had been equilibrated with 25 mM triethanolamine buffer (pH 8.3) and then eluted with Polybuffer (pH 5.0, mfd. by Pharmacia), and the thus adsorbed protein was released by a pH linear density gradient of from pH 8.3 to pH 5.0 to obtain a purified enzyme preparation of diglycosidase. An SDS-PAGE analysis confirmed that the enzyme was purified as a single band of 47 kDa. Also, when it was treated with Endoglycosidase H (mfd. by BOEHRINGER MANNHEIM), changes in the band size were not found. Example 7<br>
Physicochemical properties of diglycosidase derived from Aspergillus fvmigatus<br>
Its optimum pH was measured in the following manner. A 400 ul portion of 2 mM pNP-primeveroside solution which had been adjusted to a respective pH value of from pH 2 to 5 with 20 mM secondary citric acid-HCl buffer was incubated at 37°C for 5 minutes. Next, this was mixed with 90 \xl  of the enzyme solution to carry out the reaction at 37°C for 20 minutes.  The reaction was stopped by adding 500 ul of 0.5 M sodium carbonate solution, and the activity measurement was carried out by measuring the absorbance at 420 nm. As a result, it was found that its optimum pH was<br><br>
from 2.5 to 3.0.  It was found that it shows sufficient activity at pH 3 which is more lower pH value than those of plant-derived enzymes having similar activity.<br>
Its optimum temperature was measured in the following manner. A 4 00 µl portion of 2 mM pNP-primeveroside<br>
solution prepared using 20 mM disodium citric acid-HCl buffer (pH 2.5) was mixed with 90 ul of the enzyme solution to carry out the reaction at 30 to 65°C for 20 minutes. The reaction was stopped by adding 500 ul of 0.5 M sodium carbonate solution, and the activity was determined by measuring the absorbance at 420 ran. It was found that sufficient activity is maintained, because diglycosidase derived from Aspergillus fvunlgatus  has 80% of the activity even at 60°C, in comparison with the plant-derived enzymes having similar activity.<br>
Its pH stability was measured in the following<br>
manner.  The purified enzyme preparation was diluted 100<br>
times with each of disodium citrate buffer of from pH 2 to<br>
5, phosphate buffer of from pH 6 to 8 or glycine NaCl-NaOH buffer of from pH 7 to 10 and treated at 37°C for 1 hour, and then a 90 ul portion thereof was mixed with 4 00 µl of 2 mM pNP-primeveroside solution (pH 2.5) which had been incubated at 37°C for 5 minutes, and the reaction was carried out at 37°C for 20 minutes.  The reaction was stopped by adding 500 µl of 0.5 M sodium carbonate<br>
^
solution, and the activity measurement was carried out by measuring the absorbance at 420 nm to calculate the residual activity. As a result, its pH stability was 100% at pH 8 and it was stable within a range of from pH 3 to 8. It was found that this enzyme is stable within broader pH range in comparison with plant-derived enzymes which have similar activity and are stable at pH 4 to 7.<br>
Thermal stability of the purified preparation was examined by diluting it 100 times with 20 mM glycine NaCl-NaOH buffer (pH 8), treating the dilution at each temperature of from 30 to 55°C for 1 hour and then measuring the residual activity. As a result, the activity was stable at a temperature of 50°C or below.  It was found that this enzyme was stable broader range of temperature, in comparison with plant-derived enzymes which have similar activity and are stable at 45°C or below.<br>
Physicochemical properties of diglycosidase derived from other microorganisms<br>
Diglycosidase samples, the production of which had been confirmed by the same method as described in Example 5, were examined for physicochemical properties. As a result, it was found that diglycosidase has an activity enough for practical use at pH 3 or less because of its optimum pH ranging from 3 to 6, that it has an activity enough for practical use at 50°C or more because of its<br><br>
optimum temperature ranging from 30°C to 60°C, and that it is stable at pH 3 to 8 and at 50°C or less.  Thus, diglycosidase derived from microorganisms can be used in relatively broad ranges of pH and temperature in comparison with similar enzymes derived from plants and is superior thereto in stability. Example 8<br>
Isolation of gene coding for the diglycosidase derived from Aspergillus fumlgatus<br>
Unless otherwise noted, gene manipulation techniques employed herein were carried out in accordance with a textbook (e.g., Molecular Cloning 2nd ed., Cold Spring Harbor Laboratory Press, 1989).<br>
a)	Isolation of chromosomal DNA<br>
Aspergillus fumlgatus  IAM 2046 was inoculated into a glucose-peptone medium (0.2% yeast extract, 0.5% peptone, 2% glucose, 0.1% potassium dihydrogenphosphate, 0.05% magnesium sulfate, pH 5.7) and cultured at 30°C for 3 days on a shaker.<br>
In accordance with the method of Michael J. Hynes (Molecular and Cellular Biology,   1983, Vol. 3, No. 8, 1430 - 1439), 0.2 ml of chromosomal DNA having a concentration of 12.6 mg/ml was obtained from 300 ml of the culture broth.<br>
b)	Determination of partial amino acid sequence<br><br>
The purified enzyme preparation of diglycosidase obtained in the Example was applied to a protein sequencer (mfd. by Hewlett Packard) to determine the 22 residue N-terminal amino acid sequence shown in SEQ ID NO: 1. Next, the purified enzyme preparation of diglycosidase obtained in the Example was subjected to reductive carboxylmethylation and then digested with lysyl endopeptidase.  The thus obtained digest was applied to a reverse phase liquid chromatography, and one of the digested peptide fractions was applied to the protein sequencer to determine the 22 residue internal amino acid sequence shown in SEQ ID NO: 2. SEQ ID NO: 1<br>
Ala-Ala-Ser-Ala-Ser-Ala-Tyr-Cys-Ser-Asn-Ser-Ala-Gly-Asn-Tyr-Lys-Leu-Ser-Ser-Ile-Ala-Ala SEQ ID NO: 2<br>
Leu-Met-Thr-Pro-Ala-Gly-Ala-Asn-Phe-Ala-Leu-Met-Arg-His-Thr-Ile-Gly-Ala-Ser-Asp-Leu-Ser c) Preparation of DNA probe by PCR<br>
Based on the N-terminal amino acid sequence and internal amino acid sequence, the following four mixed oligonucleotides were synthesized by a DNA synthesizer and used as PCR primers.<br><br>
SEQ ID NO: 3<br>
Sense primer:<br>
5"-ACGAATTCAA(TC) (TA) (CG) IGC (TCAG) GGIAA(TC) TA(TC)AA-3"<br>
SEQ ID NO: 4<br>
Sense primer:<br>
5"-CGGAATTCTA(TC)TG(TC)(TA)(CG)IAA(TC)(TA)(CG)IGC(TCAG)GG-3"<br>
SEQ ID NO: 5<br>
Antisense primer:<br>
5"-TCAAGCTTGC(AG)AA(AG)TTIGC(TCAG)CCIGC(TCAG)GG-3"<br>
SEQ ID NO: 6<br>
Antisense primer:<br>
5" -AGAAGCTTGCICC (TAG) ATIGT (AG) TG (TCAG) C (TG) CAT<br>
Using these primers and the Aspergillus fumigatus<br>
chromosomal DNA as the template, PCR reaction was carried<br>
out under the following conditions using GeneAmp PCR System<br>
9600 (Perkin Elmer).<br><pcr reaction solution><br>
10 x PCR reaction buffer (Perkin Elmer)  10 |il dNTP mixed solution (each 2 mM, Perkin Elmer)<br>
10 µ1<br>
25 mM MgCl2 (Perkin Elmer)	6 µl<br>
chromosomal DNA solution (100 jig/ml)      1 µl<br>
40 µM sense primer	2.5 µl<br>
40 µM antisense primer	2.5 µl<br>
sterilized water	67.5 µl<br>
Amplitaq Gold (5 U/µl, Perkin Elmer)      0.5 µl<br>
--52--<br><br><pcr reaction conditions><br>
Stage 1: denaturation (95°C, 9 minutes)      1 cycle Stage 2: denaturation (94°C, 45 seconds)    30 cycles annealing (55°C, 1 minute) elongation (72°C, 2 minutes) Stage 3: elongation (72°C, 10 minutes)       1 cycle<br>
When the thus obtained DNA fragment of about 0.27 kbp was cloned into pUC19 (TOYOBO) and then its nucleotide sequence was examined, a nucleotide sequence coding for the partial amino acid sequence described in the foregoing was found between just after the sense primer and just before the antisense primer. This DNA fragment was used as the DNA probe for the gene cloning.<br>
d)	Preparation of gene library<br>
By recovering total RNA from Asper-gillus fvaaigatus, Poly (A) RNA was prepared using Poly (A) Quick mRNA Isolation Kit (mfd. by STRATAGENE) . Next, cDNA was synthesized using ZAP-cDNA Synthesis Kit (mfd. by STRATAGENE) , ligated to A.ZAP II vector (mfd. by STRATAGENE) and then subjected to packaging using Gigapack III Gold (mfd. by STRATAGENE) to obtain a gene library.<br>
e)	Screening of gene library<br>
The 0.27 kbp DNA fragment obtained in the aforementioned step c) was labeled using DIG-High Prime (mfd. by BOEHRINGER MANNHEIM) .  Using this as the DNA probe, the gene library obtained in the step d) was<br><br>
screened by plaque hybridization. After recovering phage particles from the thus obtained positive plaques, a plasmid pAFPri containing a cDNA of about 1.7 kbp was obtained by the in vivo  excision method in accordance with the instruction of STRATAGENE. f) determination of nucleotide sequence<br>
A nucleotide sequence coding for the diglycosidase is shown in SEQ ID NO: 7. Also, an amino acid sequence encoded by the SEQ ID NO: 7 is shown in SEQ ID NO: 8. Since the N-terminal amino acid sequence (SEQ ID NO: 1) and internal amino acid sequence (SEQ ID NO: 2) determined in the step b) were found in this amino acid sequence, it was confirmed that this DNA fragment is a diglycosidase gene fragment.<br>
--S4T-<br><br>
SEQ ID NO: 7<br>
gccgcctctg cttcggctta ctgttccaac tcggccggca actacaagct gtcctccatc 60<br>
gcagctccgg ttcaaggggc cggaaacccc ggctcggaat cgacctggca attgaccgtt 120<br>
gacgacactt cgtccggtca caaacagacg atagttgggt tcggtgctgc tgtcactgat 180<br>
gccacggtca cctcgttcaa cactttgtcc gcctccgtgc tgcaagactt gctcaataaa 240<br>
ctgatgacac ctgccggggc gaactttgct ttgatgcgac atactattgg ggcttcggat 300<br>
ctgtccggtg acccagccta cacgtacgat gacaatggtg ggaaagcgga tccgtcactg 360<br>
tcgggattca acctggggga ccgcggaacg gctatggcca agatgttggc aacaatgaag 420<br>
tctctgcagc ccaacctcaa gatcctcggc tctccctgga gtgcaccagg atggatgaag 480<br>
ctgaacgggg tccttgatgg caatacgaac aacaacaact tgaacgatgg atacctaacc 540<br>
agtgggggaa ccggtagtac ggggtatgcc agtcaattcg cgcagtactt tgtcaagtac 600<br>
attcaggcct ataagaatct cggtgctcac gtcgacgcga ttaccatcca gaacgagccg 660<br>
ctgttcagct cagcgggcta tcccaccatg tatgtctacg attatgagtc ggcacagctg 720<br>
atccagaact acatcggccc cgctcttgcc agcgcggggc tagatacgga aatctgggct 780<br>
tatgaccaca acacagatgt cccgtcgtac ccccagactg tccttaacca ggccggtcag 840<br>
tacgtcaagt cggtggcctg gcactgctac gctcccaacg tcgactggac cgtgctcagc 900<br>
cagttccaca acacaaaccc tggagtgaag caatatatga ccgagtgctg gactccagca 960<br>
tctggcgcat ggcatcaggc ggcggacttc accatgggtc ccctgcagaa ctgggcctcg 1020<br>
ggagtggcag catggactct gggaaccaac gctcaggatg gtccgcatct gtccactggc 1080<br>
ggctgcgcga catgtcaagg cttggtgacc atcaacaacg gaggatacac gctcaacacc 1140<br>
gcatactaca tgatggcgca attcagcaag ttcatgccgc ctggtgcgat tgtgctcaat 1200<br>
ggcagtggca gctacacgta ctctggcgga ggcggtatcc agtccgtggc ttccttgaat 1260<br>
cccgatggaa cccgcactgt ggttattgaa aacacttttg gcaatgatgt ctatgtgact 1320<br>
gtcactatga agagcgggca gaagtggagt gggaacgccc ctagccaatc cgtgactacc 1380<br>
tgggttcttc catctgcttg a	1401<br><br><br>
SEQ ID NO: 8<br>
Ala Ala Ser Ala Ser Ala Tyr Cys Ser Asn Ser Ala Gly Asn Tyr Lys<br>
15	10	15<br>
Leu Ser Ser lie Ala Ala Pro Val Gin Gly Ala Gly Asn Pro Gly Ser<br>
20	25	30<br>
Glu Ser Thr Trp Gin Leu Thr Val Asp Asp Thr Ser Ser Gly His Lys<br>
35	40	45<br>
Gin Thr He Val Gly Phe Gly Ala Ala Val Thr Asp Ala Thr Val Thr<br>
50	55	60<br>
Ser Phe Asn Thr Leu Ser Ala Ser Val Leu Gin Asp Leu Leu Asn Lys<br>
65	70	75	80<br>
Leu Met Thr Pro Ala Gly Ala Asn Phe Ala Leu Met Arg His Thr Me<br>
85	90	95<br>
Gly Ala Ser Asp Leu Ser Gly Asp Pro Ala Tyr Thr Tyr Asp Asp Asn<br>
100	105	110<br>
Gly Gly Lys Ala Asp Pro Ser Leu Ser Gly Phe Asn Leu Gly Asp Arg<br>
115	120	125<br>
Gly Thr Ala Met Ala Lys Met Leu Ala Thr Met Lys Ser Leu Gin Pro<br>
130	135	140<br>
Asn Leu Lys Me Leu Gly Ser Pro Trp Ser Ala Pro Gly Trp Met Lys<br>
145	150	155	160<br>
Leu Asn Gly Val Leu Asp Gly Asn Thr Asn Asn Asn Asn Leu Asn Asp<br>
165	170	175<br>
Gly Tyr Leu Thr Ser Gly Gly Thr Gly Ser Thr Gly Tyr Ala Ser Gin<br>
180	185	190<br>
Phe Ala Gin Tyr Phe Val Lys Tyr Me Gin Ala Tyr Lys Asn Leu Gly<br>
195	200	205<br>
Ala His Val Asp Ala Me Thr Me Gin Asn Glu Pro Leu Phe Ser Ser<br>
210	215	220<br>
Ala Gly Tyr Pro Thr Met Tyr Val Tyr Asp Tyr Glu Ser Ala Gin Leu<br>
225	230	235	240<br>
Me Gin Asn Tyr Me Gly Pro Ala Leu Ala Ser Ala Gly Leu Asp Thr<br>
245	250	255<br>
Glu Me Trp Ala Tyr Asp His Asn Thr Asp Val Pro Ser Tyr Pro Gin<br>
260	265	270<br>
Thr Val Leu Asn Gin Ala Gly Gin Tyr Val Lys Ser Val Ala Trp His<br>
275	280	285<br>
-jser-      S~9<br><br>
Cys Tyr Ala Pro Asn Val Asp Trp Thr Val Leu Ser Gin Phe His Asn<br>
290	295	300<br>
Thr Asn Pro Gly Val Lys Gin Tyr Met Thr Glu Cys Trp Thr Pro Ala<br>
305	310	315	320<br>
Ser Gly Ala Trp His Gin Ala Ala Asp Phe Thr Met Gly Pro Leu Gin<br>
325	330	335<br>
Asn Trp Ala Ser Gly Val Ala Ala Trp Thr Leu Gly Thr Asn Ala Gin<br>
340	345	350<br>
Asp Gly Pro His Leu Ser Thr Gly Gly Cys Ala Thr Cys Gin Gly Leu<br>
355	360	365<br>
Val Thr Ile Asn Asn Gly Gly Tyr Thr Leu Asn Thr Ala Tyr Tyr Met<br>
370	375	380<br>
Met Ala Gin Phe Ser Lys Phe Met Pro Pro Gly Ala lie Val Leu Asn<br>
385	390	395	400<br>
Gly Ser Gly Ser Tyr Thr Tyr Ser Gly Gly Gly Gly Me Gin Ser Val<br>
405	410	415<br>
Ala Ser Leu Asn Pro Asp Gly Thr Arg Thr Val Val lie Glu Asn Thr<br>
420	425	430<br>
Phe Gly Asn Asp Val Tyr Val Thr Val Thr Met Lys Ser Gly Gin Lys<br>
435	440	445<br>
Trp Ser Gly Asn Ala Pro Ser Gin Ser Val Thr Thr Trp Val Leu Pro<br>
450	455	460<br>
Ser Ala 465<br><br><br>
The open reading frame of this gene is shown in SEQ ID NO: 9. As shown in SEQ ID NO: 10, the entire portion is coded as a preprotein of 488 amino acids, of which the N-terminal 22 residues are assumed to be the pre-region and the remaining 466 residues correspond to the mature protein (cf. SEQ ID NO: 8).<br>
The invention is not only particularly limited to a polypeptide having an activity to act upon a disaccharide glycoside to release saccharides from the disaccharide glycoside in disaccharide unit and a nucleotide which encodes the same, but also includes a more longer polypeptide comprising the former polypeptide (e.g., precursor) and a nucleotide which encodes the same.<br><br><br>
SEQ ID NO: 9<br>
ggcgacacca gaaagcaacc aagagcacga cacggactta tttctctttg aca atg  56<br>
Met<br>
cgt ata tct gtc ggt get ctg ctt gge ttg aca gee ctg agt cat gec  104<br>
Arg He Ser Val Gly Ala Leu Leu Gly Leu Thr Ala Leu Ser His Ala<br>
-20	-15	-10<br>
aca aca gag aaa cga gee gee tct get teg get tac tgt tec aac teg  152<br>
Thr Thr Glu Lys Arg Ala Ala Ser Ala Ser Ala Tyr Cys Ser Asn Ser<br>
-5	-11	5	10<br>
gee gge aac tac aag ctg tec tec ate gca get ccg gtt caa ggg gee  200<br>
Ala Gly Asn Tyr Lys Leu Ser Ser Me Ala Ala Pro Val Gin Gly Ala<br>
15	20	25<br>
gga aac ccc gge teg gaa teg ace tgg caa ttg ace gtt gac gae act  248<br>
Gly Asn Pro Gly Ser Glu Ser Thr Trp Gin Leu Thr Val Asp Asp Thr<br>
30	35	40<br>
teg tec ggt cac aaa cag acg ata gtt ggg ttc ggt get get gtc act  296<br>
Ser Ser Gly His Lys Gin Thr lie Val Gly Phe Gly Ala Ala Val Thr<br>
45	50	55<br>
gat gec acg gtc ace teg ttc aac act ttg tec gee tec gtg ctg caa  344<br>
Asp Ala Thr Val Thr Ser Phe Asn Thr Leu Ser Ala Ser Val Leu Gin<br>
60	65	70	75<br>
gac ttg etc aat aaa ctg atg aca cct gee ggg gcg aac ttt get ttg  392<br>
Asp Leu Leu Asn Lys Leu Met Thr Pro Ala Gly Ala Asn Phe Ala Leu<br>
80	85	90<br>
atg cga cat act att ggg get teg gat ctg tec ggt gac cca gec tac  440<br>
Met Arg His Thr He Gly Ala Ser Asp Leu Ser Gly Asp Pro Ala Tyr<br>
95	100	105<br>
acg tac gat gac aat ggt ggg aaa gcg gat ccg tea ctg teg gga ttc  488<br>
Thr Tyr Asp Asp Asn Gly Gly Lys Ala Asp Pro Ser Leu Ser Gly Phe<br>
110	115	120<br>
aac ctg ggg gac cgc gga acg get atg gec aag atg ttg gca aca atg  536<br>
Asn Leu Gly Asp Arg Gly Thr Ala Met Ala Lys Met Leu Ala Thr Met<br>
125	130	135<br>
aag tct ctg cag ccc aac etc aag ate etc gge tct ccc tgg agt gca  584<br>
Lys Ser Leu Gin Pro Asn Leu Lys lie Leu Gly Ser Pro Trp Ser Ala<br>
140	145	150	155<br>
cca gga tgg atg aag ctg aac ggg gtc ctt gat gge aat acg aac aac  632<br>
Pro Gly Trp Met Lys Leu Asn Gly Val Leu Asp Gly Asn Thr Asn Asn<br>
160	165	170<br>
aac aac ttg aac gat gga tac eta ace agt ggg gga ace ggt agt acg  680<br>
Asn Asn Leu Asn Asp Gly Tyr Leu Thr Ser Gly Gly Thr Gly Ser Thr<br>
175	180	185<br><br>
-
L°_ &amp;V^<br><br>
ggg tat gcc agt caa ttc gcg cag tac ttt gtc aag tac att cag gcc  728<br>
Gly Tyr Ala Ser Gin Phe Ala Gin Tyr Phe Val Lys Tyr Me Gin Ala<br>
190	195	200<br>
tat aag aat etc ggt get cac gtc gac gcg att ace ate cag aac gag  776<br>
Tyr Lys Asn Leu Gly Ala His Val Asp Ala Me Thr Me Gin Asn Glu<br>
205	210	215<br>
ccg ctg ttc age tea gcg ggc tat ccc ace atg tat gtc tac gat tat  824<br>
Pro Leu Phe Ser Ser Ala Gly Tyr Pro Thr Met Tyr Val Tyr Asp Tyr<br>
220	225	230	235<br>
gag teg gca cag ctg ate cag aac tac ate ggc ccc get ctt gcc age  872<br>
Glu Ser Ala Gin Leu lie Gin Asn Tyr Me Gly Pro Ala Leu Ala Ser<br>
240	245	250<br>
gcg ggg eta gat acg gaa ate tgg get tat gac cac aac aca gat gtc  920<br>
Ala Gly Leu Asp Thr Glu Me Trp Ala Tyr Asp His Asn Thr Asp Val<br>
255	260	265<br>
ccg teg tac ccc cag act gtc ctt aac cag gcc ggt cag tac gtc aag  968<br>
Pro Ser Tyr Pro Gin Thr Val Leu Asn Gin Ala Gly Gin Tyr Val Lys<br>
270	275	280<br>
teg gtg gcc tgg cac tgc tac get ccc aac gtc gac tgg ace gtg etc  1016<br>
Ser Val Ala Trp His Cys Tyr Ala Pro Asn Val Asp Trp Thr Val Leu<br>
285	290	295<br>
age cag ttc cac aac aca aac cct gga gtg aag caa tat atg ace gag  1064<br>
Ser Gin Phe His Asn Thr Asn Pro Gly Val Lys Gin Tyr Met Thr Glu<br>
300	305	310	315<br>
tgc tgg act cca gca tct ggc gca tgg cat cag gcg gcg gac ttc ace  1112<br>
Cys Trp Thr Pro Ala Ser Gly Ala Trp His Gin Ala Ala Asp Phe Thr<br>
320	325	330<br>
atg ggt ccc ctg cag aac tgg gcc teg gga gtg gca gca tgg act ctg  1160<br>
Met Gly Pro Leu Gin Asn Trp Ala Ser Gly Val Ala Ala Trp Thr Leu<br>
335	340	345<br>
gga ace aac get cag gat ggt ccg cat ctg tec act ggc ggc tgc gcg  1208<br>
Gly Thr Asn Ala Gin Asp Gly Pro His Leu Ser Thr Gly Gly Cys Ala<br>
350	355	360<br>
aca tgt caa ggc ttg gtg ace ate aac aac gga gga tac acg etc aac  1256<br>
Thr Cys Gin Gly Leu Val Thr Ile Asn Asn Gly Gly Tyr Thr Leu Asn<br>
365	370	375<br>
ace gca tac tac atg atg gcg caa ttc age aag ttc atg ccg cct ggt  1304<br>
Thr Ala Tyr Tyr Met Met Ala Gin Phe Ser Lys Phe Met Pro Pro Gly<br>
380	385	390	395<br>
gcg att gtg etc aat ggc agt ggc age tac acg tac tct ggc gga ggc  1352<br>
Ala Me Val Leu Asn Gly Ser Gly Ser Tyr Thr Tyr Ser Gly Gly Gly<br>
400	405	410<br><br>
ggt ate cag tec gtg get tec ttg aat ccc gat gga ace cgc act gtg  1400<br>
Gly Me Gin Ser Val Ala Ser Leu Asn Pro Asp Gly Thr Arg Thr Val<br>
415	420	425<br>
gtt att gaa aac act ttt ggc aat gat gtc tat gtg act gtc act atg  1448<br>
Val lie Glu Asn Thr Phe Gly Asn Asp Val Tyr Val Thr Val Thr Met<br>
430	435	440<br>
aag age ggg cag aag tgg agt ggg aac gee cct age caa tec gtg act  1496<br>
Lys Ser Gly Gin Lys Trp Ser Gly Asn Ala Pro Ser Gin Ser Val Thr<br>
445	450	455<br>
ace tgg gtt ctt cca tct get tga aaagagtgta gtttcagatg gttagatatg 1550<br>
Thr Trp Val Leu Pro Ser Ala<br>
460	465<br>
tattgaagag tagegcttgg agacatcaat agecttttte taattacatg tegtgeaget 1610<br>
tccaaaaaaa aaaaaaaaaa aaaaaaaaaa aactcga	1647<br>
SEQ ID NO: 10<br>
Met Arg lie Ser Val Gly Ala Leu Leu Gly Leu Thr Ala Leu Ser His<br>
15	10	15<br>
Ala Thr Thr Glu Lys Arg Ala Ala Ser Ala Ser Ala Tyr Cys Ser Asn<br>
20	25	30<br>
Ser Ala Gly Asn Tyr Lys Leu Ser Ser lie Ala Ala Pro Val Gin Gly<br>
35	40	45<br>
Ala Gly Asn Pro Gly Ser Glu Ser Thr Trp Gin Leu Thr Val Asp Asp<br>
50	55	60<br>
Thr Ser Ser Gly His Lys Gin Thr lie Val Gly Phe Gly Ala Ala Val<br>
65	70	75	80<br>
Thr Asp Ala Thr Val Thr Ser Phe Asn Thr Leu Ser Ala Ser Val Leu<br>
85	90	95<br>
Gin Asp Leu Leu Asn Lys Leu Met Thr Pro Ala Gly Ala Asn Phe Ala<br>
100	105	110<br>
Leu Met Arg His Thr Me Gly Ala Ser Asp Leu Ser Gly Asp Pro Ala<br>
115	120	125<br>
Tyr Thr Tyr Asp Asp Asn Gly Gly Lys Ala Asp Pro Ser Leu Ser Gly<br>
130	135	140<br>
Phe Asn Leu Gly Asp Arg Gly Thr Ala Met Ala Lys Met Leu Ala Thr<br>
145	150	155	160<br>
Met Lys Ser Leu Gin Pro Asn Leu Lys Me Leu Gly Ser Pro Trp Ser<br>
165	170	175<br>
Ala Pro Gly Trp Met Lys Leu Asn Gly Val Leu Asp Gly Asn Thr Asn<br>
180	185	190<br>
- Ur - <p></p>
 <br>
Asn Asn Asn Leu Asn Asp 61y Tyr Leu Thr Ser Gly Gly Thr Gly Ser<br>
195	200	205<br>
Thr Gly Tyr Ala Ser Gin Phe Ala Gin Tyr Phe Val Lys Tyr Me Gin<br>
210	215	220<br>
Ala Tyr Lys Asn Leu Gly Ala His Val Asp Ala Me Thr Me Gin Asn<br>
225	230	235	240<br>
Glu Pro Leu Phe Ser Ser Ala Gly Tyr Pro Thr Met Tyr Val Tyr Asp<br>
245	250	255<br>
Tyr Glu Ser Ala Gin Leu Me Gin Asn Tyr Me Gly Pro Ala Leu Ala<br>
260	265	270<br>
Ser Ala Gly Leu Asp Thr Glu Me Trp Ala Tyr Asp His Asn Thr Asp<br>
275	280	285<br>
Val Pro Ser Tyr Pro Gin Thr Val Leu Asn Gin Ala Gly Gin Tyr Val<br>
290	295	300<br>
Lys Ser Val Ala Trp His Cys Tyr Ala Pro Asn Val Asp Trp Thr Val<br>
305	310	315	320<br>
Leu Ser Gin Phe His Asn Thr Asn Pro Gly Val Lys Gin Tyr Met Thr<br>
325	330	335<br>
Glu Cys Trp Thr Pro Ala Ser Gly Ala Trp His Gin Ala Ala Asp Phe<br>
340	345	350<br>
Thr Met Gly Pro Leu Gin Asn Trp Ala Ser Gly Val Ala Ala Trp Thr<br>
355	360	365<br>
Leu Gly Thr Asn Ala Gin Asp Gly Pro His Leu Ser Thr Gly Gly Cys<br>
370	375	380<br>
Ala Thr Cys Gin Gly Leu Val Thr Me Asn Asn Gly Gly Tyr Thr Leu<br>
385	390	395	400<br>
Asn Thr Ala Tyr Tyr Met Met Ala Gin Phe Ser Lys Phe Met Pro Pro<br>
405	410	415<br>
Gly Ala Me Val Leu Asn Gly Ser Gly Ser Tyr Thr Tyr Ser Gly Gly<br>
420	425	430<br>
Gly Gly Me Gin Ser Val Ala Ser Leu Asn Pro Asp Gly Thr Arg Thr<br>
435	440	445<br>
Val Val Me Glu Asn Thr Phe Gly Asn Asp Val Tyr Val Thr Val Thr<br>
450	455	460<br>
Met Lys Ser Gly Gin Lys Trp Ser Gly Asn Ala Pro Ser Gin Ser Val<br>
465	470	475	480<br>
Thr Thr Trp Val Leu Pro Ser Ala 485<br>
- jssr-<br><br>
Example 9<br>
Confirmation of the expression of ß-primeverosidase in mold<br>
a) Construction of expression cassette<br>
In order to verify whether or not the cloned gene is the primeverosidase gene, expression of the thus obtained DNA was confirmed. Using an Aspergillus oryzae  Taka-amylase gene-containing plasmid pTG-Taa (Kato M, Aoyama A, Naruse F, Kobayashi T and Tsukagoshi N (1997), An Aspergillus nidulans  nuclear protein, An CP, involved in enhancement of Taka-amylase A gene expression binds to the CCAAT-containing taaG2, amdS  and gatA promoters., Mol.   Gen. Genet.,   254: 119 - 126) as the template, a fragment was obtained by amplifying it by PCR using a primer TAA5" SEQ ID NO: 10 sense primer:<br>
5"-GGGCCTGCAGGAATTCATGGTGTT-3" and a primer TP3" SEQ ID NO: 11 antisense primer:<br>
5"-CGAGCCGGGGTTTCCGTCCGCAGGCGTTGC-3". <pcr reaction solution><br>
template DNA solution (50 (j.g/ml)	1 µl<br>
50 µM sense primer	1 µ1<br>
50 µM antisense primer	1 µ1<br><br><br>
sterilized water	22 µl<br>
Premix Taq (EX Taq Version TaKaRa)	23 µl<br><pcr reaction conditions><br>
Stage 1: denaturation (95°C, 1 minute)      1 cycle Stage 2: denaturation (95°C, 1 minute)      30 cycles annealing (55°C, 1 minute) elongation (72°C, 1 minute) Stage 3: elongation (72°C, 5 minutes)        1 cycle Also, a fragment was obtained by amplifying it by PCR using the DNA-containing plasmid pAFPri as the template and using a primer dPC5" SEQ ID NO: 12 sense primer:<br>
5"-GCAACGCCTGCGGACGGAAACCCCGGCTCG-3" and a primer PC3" SEQ ID NO: 13 antisense primer:<br>
5"-GCGCAAGCTTGGAAGCTGCACGACATGTAA-3" . In addition, after recovering and mixing respective fragments, a fragment was obtained by amplifying it by PCR using the primer TAA5" and primer PC3" .  This fragment contains a sequence corresponding to a region of from the Aspergillus oryzae  Taka-amylase promoter to the N-terminal 5th amino acid of the mature protein and a sequence corresponding to a region of from the N-terminal 28th amino<br>
--w-      b r<br><br>
acid (glycine) to the C-terminal of the mature ß-primeverosidase protein. An Sse8387l site has been introduced into the upstream of the thus obtained fragment, and a Hindlll site into its downstream. The fragment was recovered by treating with restriction enzymes Sse8387I and HindllI.<br>
Regarding the terminator region, a fragment was obtained by amplifying it by PCR using pTG-Taa as the template and using a primer TAAH SEQ ID NO: 14 sense primer:<br>
5"-GCGCAAGCTTTGAAGGGTGGAGAGT-3" and a primer TAA3" SEQ ID NO: 15 antisense primer:<br>
5"-GCGCCCTGCAGGTCTAGAATTCCTAGTGGTT-3".<br>
A Hindlll site has been introduced into the upstream of the thus obtained fragment, and an Sse8387l site into its downstream.  The fragment was recovered by treating with restriction enzymes Hindlll and Sse8387I.<br>
A plasmid pTGl containing orotidine-5"-phosphate decarboxylase gene (pyr4) as a marker gene (Kato M (1997), Mol.   Gen.   Genet.,   254: 119 - 126) was treated with the restriction enzyme Sse8387I and with alkaline phosphatase and then recovered.<br><br>
Plasmids pAFPriEl (forward direction to the direction<br>
of the marker gene) and pAFPriE2 (reverse direction) were<br>
obtained by connecting these 3 fragments.<br>
b) Acquisition of transformant	<br>
An orotidine-5"-phosphate decarboxylase  (pyx C) producing strain Aspergillus nldulans  G191 (Kato M (1997), Mol.   Gen.   Genet.,   254: 119 - 126) was inoculated into a complete medium (2% malt extract, 0.1% peptone, 2% glucose, 0.1% uridine, 2 µg/ml p-aminobenzoic acid, pH 6.5) and cultured at 30°C for 18 hours on a shaker.  The cells were collected by filtration, suspended in a protoplast solution (0.8 M NaCl, 10 mM NaH2P04, 20 mM CaCl2, 3.75 mg/ml Novozyme 234) and then treated at 30°C for 1 hour on a shaker.  The resulting protoplasts were recovered by filtration and centrifuged at 1,500 rpm for 5 minutes to obtain the protoplasts as the precipitate.  This precipitate was suspended in 0.8 M NaCl solution and centrifuged at 1,500 rpm for 5 minutes to collect the precipitate.  This was again suspended in 0.8 M NaCl/50 mM CaCl2 solution and centrifuged at 1,500 rpm for 5 minutes to collect the precipitate. A protoplast solution was obtained by suspending this in an appropriate amount of 0.8 M NaCl/50 mM CaCl2 solution.  Next, 50 ul of this protoplast solution was mixed with 20 ug of a DNA solution and 12.5 ul of a PEG solution (25% PEG 6000/50 mM CaCl2/10 mM Tris-HCl (pH 7.5))<br><br>
and then allowed to stand for 20 minutes on ice. Next, 0.5 ml of PEG was added and then the mixture was allowed to<br>
A<br>
stand for 5 minutes on ice.  Next, 1 ml of 0.8 M NaCl/50 mM CaCl2 solution was added and mixed. A 0.5 ml portion of this mixed solution was mixed with 15 ml of 2% agar-containing regeneration medium (0.6% NaN03/ 11 mM KH2P04, 7 mM KC1, 1.2 M sorbitol, 0.05% MgS04-7H20, 1% glucose, 2 µg/ml p-aminobenzoic acid, pH 6.5) which had been incubated at 50°C in advance, solidified in Petri dishes and then cultured at 37°C for 3 days.<br>
This was carried out on the plasmid DNA of each of pTGl, pAFPriEl and pAFPriE2.<br>
Colonies formed on the plates were isolated by single spore separation. A total of 15 transformant strains were obtained from pTGl, and 23 strains from pAFPriEl and 13 strains from pAFPriE2. c) Evaluation of transformants<br>
Evaluation of transformants was carried out on 8 strains obtained from pTGl, 18 strains from pAFPriEl and 12 strains from pAFPriE2.  Each transformant was inoculated into an enzyme production confirming medium (1% polypeptone, 0.5% KH2P04, 0.1% NaN03, 0.05% MgS04-7H20, 2% maltose, 4 pig/ml p-aminobenzoic acid, 0.1% trace element solution) (Core DJ., Biochem. ,   Blophys.   Acta.,   1996, vol., 113, pp. 51 - 56) and cultured at 30°C for 96 hours on a<br>
-JPT-<br><br>
shaker. The culture broths were sampled after 48, 72, and 96 hours and filtered, and the resulting filtrates were checked for the activity. The enzyme activity was not found in the transformants obtained from pTGl and pAFPriE2 but the enzyme activity was confirmed in 5 transformant strains obtained from pAFPriEl. Example 10 Comparison with plant gene by a hybridization method<br>
Using the gene of an enzyme similar to the tea-derived diglycosidase as the probe, an examination was carried out to know if a gene having a similar structure is present on the chromosome of the microorganisms in which the presence of diglycosidase had been confirmed by us. Preparation of gene fragment of an enzyme similar to the tea-derived diglycosidase was carried out with reference to the report by Sakata, Mizutani et al.    (The 73rd Annual Meeting of Agricultural Chemical Society of Japan) and Japanese Patent Application No. Hei. 11-56299.<br>
Preparation of microorganism-derived chromosome was carried out in the following manner.<br>
Preparation of chromosomes from yeast and fungi was carried out in accordance with the method described in Molecular and Cellular Biology,  Vol. 3, pp. 1430 - 1439 (1983).  Preparation of chromosomal DNA from bacteria was carried out in accordance with the method of Saito and<br>
- *rt - *} ( — JJKT<br><br>
Mitsuura (Biochim.   Biophys.   Acta,  Vol. 72, pp. 619 - 629, 1963). Preparation of chromosomal DNA from actinomycetes was carried out in accordance with the method of Iefuji et al.    (Biosci.   Biotec.   Biochem.,  Vol. 60, pp. 1331 - 1338, 1996).<br>
A 10 fig portion of each of the thus obtained various chromosomal DNA preparations was digested with BamHI in the case of Aspergillus fumigatus, Aspergillus oryzae, Aspergillus niger,  Aspergillus aculaetus,   Penicillium lilacinum,   Penicillium decumbence,   Penicillium multicolar, Talaromyces emersonii,  Mortierella  vinacea,   Cryptococcus aUbidus,   Corynebacterium ammoniagenes,   Corynebacterium glutamicum,  Microbacterium arborescens  and Penicillium rugolosum,   or with ETcoRI in the case of Rhizopus oryzae, Khizomucor pusillus, Rhizomucor miehei  and Actinoplanes missouriensis,   and the resulting digest was applied to a 1% agarose gel electrophoresis. As a control, the gene fragment of an enzyme similar to the tea-derived diglycosidase used as the probe was also subjected to the same gel electrophoresis. After the electrophoresis, DNA samples were blotted on a nylon membrane and hybridization was carried out using a labeled gene fragment p of an enzyme similar to the tea-derived diglycosidase (structural gene moiety of matured plant primeverosidase gene) as the probe, using DIG System Kit (Boehringer Mannheim) in<br><br>
accordance with the instruction attached thereto. As a result, when the detection was carried out under hybridization conditions (5 x SSC, 1% blocking agent, 0.1% N-lauroylsarcosine sodium, 0.02% SDS, 68°C, overnight) and washing conditions (6 x SSC, 0.1% SDS, room temperature, 5 min. x 2 and 6 x SSC, 0.1% SDS, 45°C, 15 min. x 2), a signal was obtained at a position where the plant gene was blotted, but the signal was not observed at any other position where the microorganism-derived genome was blotted.  Thus, it is considered that the microorganism-derived diglycosidase gene has a structure which is different from the plant primeverosidase gene.<br>
On the other hand, using the AspBrgillus fumigatus IAM 2020 diglycosidase gene of the invention obtained in Example 8 as the probe, an examination was carried out by the same methods and conditions to know if a gene having a similar structure is present on the chromosome of the microorganisms in which the presence of diglycosidase had been confirmed by us. As a result, the signal was detected in these microorganisms.<br>
In addition, it was able to detect the signal in Aspergillus oryzae,  Aspergillus niger,   Aspergillus aculeatus,   Penicillium multicolar,   Penicillium. lilacinum, Corynebacterium ammoniagenes  and Corynebacterium glutamicum,   even under more stringent washing conditions (5<br>
-2-   ^<br><br>
x SSC, room temperature, 10 min. and 4 x SSC, 68°C, 30<br>
min.).<br>
Example 11<br>
Activity of the diglycosidase to hydrolyze isoflavone in<br>
isoflavone glycosides<br>
As shown in the following table, glucosides and modified glucosides of acetylglucosides and malonylglucosides, are present in isoflavone glycosides. The activity of diglycosidase to hydrolyze the acetyl type and malonyl type glucosides, namely the aglycon releasing activity, was examined.<br>
Isoflavone	M.   wt.	Rj.	R2	R3	Concentration<br>
Glycitin	446.4	H	OCH3	H	2  mM<br>
Genistin	432.4	OH	H	H	2  mM<br>
Daidzin	416.4	H	H	H	2  mM<br>
Acetylglycitin	458.4	H	OCH3	COCH3	2  mM<br>
Acetylgenistin	474.7	OH	H	COCH3	2  mM<br>
Acetyldaidzin	458.4	H	H	COCH3	2  mM<br>
Malonylglycitin	502.4	H	OCH3	COCH2COOH	2  mM<br>
Malonylgenistin	518.4	OH	H	COCH2COOH	2  mM<br>
Malonyldaidzin	502.4	H	H	COCH2COOH	2  mM<br>
Ri  to R3  correspond to  the  following  structural  formula.<br><br><br>
Each of acetylglycitin, acetylgenistin, acetyldaidzin, malonylglycitin, malonylgenistin and malonyldaidzin (produced by Fujicco, available from Nakalai Tesque) was allowed to react with a diglycosidase enzyme solution prepared from Asp.   fumigatus or Pen.   multicolar  or with an almond-derived glucosidase (mfd. by Sigma) under the following conditions.<br>
Each of the enzymes was diluted with 20 mM acetate buffer (pH 4.0) to adjust its activity to 1.88 AU/ml, and then 2 mM of each isoflavone (12.5 ul) , 20 mM of acetate buffer (87.5 µl)   and each enzyme solution (25 (µl)) were mixed to carry out the reaction at 55°C. After 1, 3 and 6 hours of the reaction, samples were taken out in 25 µl) portions, and each of the samples was mixed with 75 ul of methanol and 900 µl) of water, filtered through a filter (0.2 µm) and then further diluted 2.5 times with water. A 1 ml portion thereof was analyzed by HPLC (HPLC conditions; column: ODS 80TM (Tosoh), eluent: a mixed solution of<br><br><br>
acetonitrile and 10% acetic acid, separation under linear density gradient).<br>
As a result, it was revealed that the glucosidase (Sigma) hardly hydrolyzed the modified glucoside substrates, but both diglycosidase enzymes hydrolyzed all of the modified glucosides efficiently and released the aglycon. Example 12<br>
Preparation of an aroma component precursor, eugenyl primeveroside<br>
A 2 kg portion of fresh leaves of Camellia  sasanqua were extracted with hot water at 100°C for 10 minutes, and the extract was applied to a column packed with Diaion HP20 (mfd. by Mitsubishi Chemical) to adsorb eugenyl primeveroside thereon.  The column was washed with about 2 times the bed volume of deionized water and 20% methanol and then the adsorbed eugenyl primeveroside was eluted with 100% methanol.  Thereafter, the thus recovered methanol solution containing eugenyl primeveroside was concentrated to crystallize eugenyl primeveroside which was then recovered using a glass filter.<br>
A 1 ml portion of the crude enzyme concentrate obtained in Example 4 was mixed with 1 ml of the eugenyl primeveroside solution adjusted to 5 mg/ml with 20 mM phosphate buffer (pH 6.0), the mixture was incubated at<br>
-^-"  ^<br><br>
37°C for 24 hours and then the formation of aroma was examined by a sensory test (10 panel) . As a result, formation of the eugenol-specific aroma was confirmed in all of the cases in which the crude enzyme concentrates of two Aspergillus niger strains and Aspergillus  xumigatus were used.<br>
In a case in which the crude enzyme concentrates were used after their heat-treatment (100°C, 10 minutes), the eugenol-specific aroma was not observed. Accordingly, it was found that these crude enzyme extracts have a function to release the aroma component aglycon from glycosides such as eugenyl primeveroside. Example 13<br>
Release of disaccharide from pNP-primeveroside by purified enzyme<br>
A 0.3 AU portion of the purified enzyme solution obtained in Example 6 and the aforementioned pNP-primeveroside were incubated at 37°C for 24 hours to examine release of disaccharide by TLC. As a result, a spot was observed at the same position of primeverose so that release of a disaccharide was confirmed. Such a spot was not found by the heat-treated purified enzyme solution used as a control.  Thus, it was revealed that the purified enzyme has a function to release a disaccharide from a disaccharide glycoside.<br><br><br>
Example 14<br>
Release of disaccharide and formation of aroma from eugenyl<br>
primeveroside by purified enzyme<br>
A 0.3 AU portion of the purified enzyme solution obtained in Example 6 and the aforementioned pNP-primeveroside were incubated at 37°C for 24 hours to examine release of disaccharide by TLC. As a result, a spot was observed at the same position of primeverose so that release of a disaccharide was confirmed.  In addition, when the reaction solution was analyzed by a gas chromatography, release of eugenol as the aglycon of eugenyl primeveroside glycoside was confirmed, and the release of eugenol was also confirmed by a sensory test. These were not found in heat-inactivated enzyme solution. Thus, it was revealed that the purified enzyme forms aroma by acting upon an aroma precursor such as eugenyl primeveroside. Example 15 Release of disaccharide from pigment glycoside<br>
Release of disaccharide was examined using a disaccharide glycoside, ruberythric acid, as the substrate. Ruberythric acid was prepared by adsorbing a water extract of Rubia   tinctorum  L. root powder for staining use (available from Tanaka Senshoku Ten) to HP-20 column, washing the column with 50% methanol, and then eluting the<br><br>
compound with 100% methanol and evaporating the eluate to dryness using an evaporator. A substrate prepared by dissolving the thus recovered ruberythric acid m a phosphate buffer to a concentration of 5 mg/ml was mixed with the crude enzyme solution (0.3 AU) shown in Example 4 or the purified enzyme solution shown in Example 6 and incubated at 37°C for 24 hours, and then the reaction solution was analyzed by TLC. As a result, release of the disaccharide primeverose and the aglycon alizarin was observed by the crude enzyme solution and purified enzyme solution. Example 16 Hydrolysis of other disaccharide glycosides<br>
Using the diglycosidase preparations derived from various microorganisms shown m Example 5, their ability to hydrolyze various disaccharide glycosides was examined using TLC. As a result, it was revealed that the diglycosidase acts upon not only the primeveroside glycosides but also various other disaccharide glycosides analogous to the primeveroside glycosides, including<br>
rutmose glycosides such as naringm and rutin, gentiobiose<br>
glycosides, arabmofuranosyl glycosides and aprofuranosyl<br>
glycosides, and thereby releases disaccharides and produces respective free aglycons<br>
S-" ^<br><br>
Example 17<br>
Improvement of tea extract aroma<br>
Using the Aspergillus fumlgatus  enzyme solutions shown in Examples 4 and 6, their function to increase aroma components of green tea, black, tea and oolong tea was examined. Each tea extract was mixed with 1.88 AU of the enzyme and incubated at 55°C for 24 hours and then increase in the aroma was examined by a gas chromatography under the aforementioned conditions. As a result, it was found that aroma components such as 1-hexanol, 3-hexen-l-ol, benzaldehyde, linalool, methyl salicyanate, geraniol and benzyl alcohol were increased.  Increase in the aroma was also found by a sensory test. Example 18 Improvement of fruit juice aroma<br>
Using the Aspergillus fumlgatus  and Penlcllllum multicolor  enzyme solutions shown in Examples 4 and 5, 1.88 AU of each of the enzyme was added to a fruit juice such as of grape, orange, apple, prune or nectar and incubated at 37°C for 24 hours and then the aroma was analyzed by a gas chromatography. As a result, increase in the aroma components such as linalool was observed.  Improvement of the aroma was also found in the enzyme-treated fruit juices by a sensory test.<br>
-jrr-<br><br>
Example 19<br>
Improvement of wine aroma<br>
Using the various crude enzyme solutions shown in Example 4, 0.5 AU of each enzyme was added to red wine and white wine and incubated at 37°C for 24 hours to examine improvement of the aroma by a sensory test. As a result, improvement of the aroma was found in both cases. Example 20<br>
When 1 ml of each of the various crude enzyme concentrates obtained in Examples 4 and 5 was mixed with 1 ml of a grape juice (commercial product: 100% fruit juice, concentrated and reduced) and then incubated at 37°C overnight (14 hours) to examine the aroma, the aroma was clearly improved in comparison with a sample in which an acetate buffer was added instead of the enzyme preparation. In addition, this function was not found when the crude enzyme concentrate was heat-treated at 100°C for 10 minutes. Example 21<br>
A 1 ml portion of each of the crude enzyme concentrates obtained in Example 4 was mixed with 1 ml of a commercially available orange juice (reduced concentrate) and incubated at 37°C for 24 hours to examine formation of the aroma by a sensory test. As a result, improving effect of the aroma of the orange juice was found in the crude<br><br>
enzyme concentrates derived from the two Aspergillus nlger strains and Aspergillus fumlgatus. Such a function was not found when the crude enzyme concentrate was heat-treated (100°C, 10 minutes). Example 22<br>
An examination was carried out to find whether a sugar transfer in disaccharide unit is generated by diglycosidase.<br>
The purified enzyme of Example 6 was diluted with deionized water to adjust the activity to 1.0 AU/ml, and acetonitrile (200 µl) containing 2.5% phenethyl alcohol, 20 mM acetate buffer (250 µl)   containing 10% primeverose and the enzyme solution (50 ul) were mixed to carry out the reaction at 55°C.  The reaction was completed after 6 hours and the reaction solution was mixed with 500 ul of diethyl ether, stirred and then centrifuged to remove free aglycon which was transferred into the ether layer.  By applying the water layer to a Diaion HP-20 column and passing purified water through the column, free primeverose was removed.  The disaccharide glycoside adsorbed to the resin was eluted with methanol and concentrated to dryness.  This was dissolved in 100 µl   of deionized water, and a 20 µl portion thereof was spotted on a TLC plate to detect the reaction product (the developing solvent was ethyl acetate:acetic acid:deionized water = 3:1:1, acetic<br>
-to-   Z^<br><br>
acid:methanol = 1:4 solution was sprayed thereto after the development and allowed to stand at 105°C for 10 minutes) .<br>
As a result, it was revealed that ß-primeverosidase transfers the diglycoside to phenethyl alcohol and thereby forms a disaccharide glycoside. Industrial Applicability<br>
An enzyme having a function to cut P-primeveroside and/or analogous disaccharide glycoside in disaccharide unit or to hydrolyze modified glucosides can be provided by the invention as a novel enzyme using microorganisms as its supply source, and it can be broadly used in various types of food, medicaments, quasi drugs and the like by using the enzyme composition of the invention.  For example, the aroma, pigment and physiologically active component of food can be increased or decreased.<br><br><br><br><br><br><br>
WE CLAIM:<br>
1	A method for producing an enzyme having an activity to act<br>
upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit, which comprises the steps of:<br>
a)	culturing, under aerobic condition with a pH in range of 3-8, a microorganism of the kind as herein described in a nutrient medium of the kind as herein described to effect production of the enzyme having an activity to act upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside, and<br>
b)	subsequently collecting/isolating in the usual way by combination of centrifugation, UF concentrating and salting out said enzyme from the resulting culture mixture to produce the said enzyme.<br>
2	A method for producing an enzyme having an activity to act<br>
upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit as claimed in claim 1, wherein said process comprises the steps of:<br>
a)	culturing, under aerobic condition with a pH in range of 3-8, the transformant of the kind such as herein described to effect production of the enzyme having an activity to act upon a disaccharide glycoside to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside, and<br>
b)	subsequently  collecting/isolating  in   the  usual  way  by combination   of   centrifugation,   UF   concentrating   and<br>
salting   out   said   enzyme   from   the   resulting   culture mixture to produce the said enzyme.<br>
3	The method as claimed in claim 1 or 2, wherein said step of culturing is performed at 10-50°C.<br>
4	The method as claimed in claim 1, 2 or 3, wherein said step of culturing is performed for a period of 1 to 15 days.<br>
5	The method as claimed in claim 1, wherein said microorganism is selected from the genus Aspergillus, the genus Penicillium, the genus Rhizopus, the genus Rhizomucor, the genus Talaromyces, the genus Mortierella, the genus Cryptococcus, the genus Microbacterium, the genus Corynebacterium and the genus Actinoplanes.<br>
6	The method as claimed in claim 1 or 4, wherein said nutrient medium contains an inducer which induces production of an enzyme having an action to release saccharides from a disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside.<br>
7	The method as claimed in claim 6, wherein said inducer is a saccharide.<br>
8	The method as claimed in claim 7, wherein said saccharide is selected from the group consisting of gentose, gentibiose and gentio-oligosaccharide.<br>
9	A method for producing modified composition of a material containing a disaccharide glycoside, which comprises allowing in the manner such as herein described an enzyme having an action to release saccharides from said disaccharide glycoside in disaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside to react with said material.<br>
10	The method as claimed in claim 9, wherein said disaccharide glycoside is an aromatic component precursor or a pigment component precursor.<br>
11	The method as claimed in claim 9, wherein said disaccharide glycoside is ß-primeveroside and/or an analogous disaccharide glycoside.<br>
12	A method for producing modified composition of a material containing a modified monoglycoside, which comprises allowing in the manner such as herein described an enzyme having an action to release a modified monosaccharide unit and having an activity to cut the glycoside bonding of the monoglycoside and modified monoglycoside from said modified monoglycoside to react with said material.<br>
13	The method as claimed in claim 12, wherein said modified monoglycoside is at least one member selected from the group consisting of acetylglucoside, malonylglucoside, methylglucoside, phosphoglucosdie, and amidoglucoside.<br>
14	The method as claimed in claim 13, wherein said modified monoglycoside is acetylglucoside of isoflavone, malonylglucoside of isoflavone, and both thereof.<br>
15	A method for producing an enzyme substantially as herein described with reference to the forgoing examples.<br>
16	A method for producing modified composition of a material containing a disaccharide glycoside substantially as herein described with reference to the forgoing examples.<br>
17	A method for modifying composition of a material containing a modified monoglycoside substantially as herein described with reference to the forgoing examples.</pcr></pcr></pcr></pcr></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChhcyBmaWxlcykucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-complete specification (as files).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGV0ZCkucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-complete specification (grantetd).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtMzA4LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtMzM4LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-338.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wZXRpdGlvbi0xMzcucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMjQzLWRlbC1wZXRpdGlvbi0xMzgucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-243-del-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="217104-manually-operated-fluid-pump-or-compressor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="217106-pre-grinding-of-oilseeds-prior-to-mechanical-expelling-a-new-process-of-energy-saving.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>217105</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/00243/DEL</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2008</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Mar-2008</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Mar-2008</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Mar-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AMANO ENZYME INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2-7, NISHIKI 1-CHOME, NAKA-KU, NAGOYA-SHI, AICHI 460-0003, JAPAN.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHIGERU YAMAMOTO</td>
											<td>C/O AMANO ENZYME INC. 2-7, NISHIKI 1-CHOME, NAKA-KU, NAGOYA-SHI, AICHI 460-0003, JAPAN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MASAMICHI OKADA</td>
											<td>C/O AMANO ENZYME INC. 2-7, NISHIKI 1-CHOME, NAKA-KU, NAGOYA-SHI, AICHI 460-0003, JAPAN.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TAICHI USUI</td>
											<td>C/O FACULTY OF AGRICULTURE, SHIZUOKA UNIVERSITY 836 OHYA, SHIZUOKA-SHI 422-8017, JAPAN.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KANZO SAKATA</td>
											<td>C/O INSTITUTE FOR CHEMICAL RESEARCH, KYOTO UNIVERSITY KANYUUCHI, GOKASHOU, UJI-SHI, KYOTO 611-0011, JAPAN.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ATSUKI TOUMOTO</td>
											<td>10-17, TAIKOU 5-CHOME, NAKAMURA-KU, NAGOYA-SHI, AICHI 453-0801, JAPAN.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KAZUTAKA TSURUHAMI</td>
											<td>52-103, AZA KURAMAE, OAZA OKIMURA, NISHIHARU-CHO, NISHIKASUGAI-GUN, AICHI 481-0043, JAPAN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 19/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP99/05346</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>1999-09-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>HEI-10-294675</td>
									<td>1998-09-30</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/217105-a-method-for-producing-an-enzyme-and-modifying-a-composition by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:23:13 GMT -->
</html>
